---
pmid: '9922454'
title: 'Ordering the cytochrome c-initiated caspase cascade: hierarchical activation
  of caspases-2, -3, -6, -7, -8, and -10 in a caspase-9-dependent manner.'
authors:
- Slee EA
- Harte MT
- Kluck RM
- Wolf BB
- Casiano CA
- Newmeyer DD
- Wang HG
- Reed JC
- Nicholson DW
- Alnemri ES
- Green DR
- Martin SJ
journal: J Cell Biol
year: '1999'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2132895
doi: 10.1083/jcb.144.2.281
---

# Ordering the cytochrome c-initiated caspase cascade: hierarchical activation of caspases-2, -3, -6, -7, -8, and -10 in a caspase-9-dependent manner.
**Authors:** Slee EA, Harte MT, Kluck RM, Wolf BB, Casiano CA, Newmeyer DD, Wang HG, Reed JC, Nicholson DW, Alnemri ES, Green DR, Martin SJ
**Journal:** J Cell Biol (1999)
**DOI:** [10.1083/jcb.144.2.281](https://doi.org/10.1083/jcb.144.2.281)
**PMC:** [PMC2132895](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2132895/)

## Abstract

1. J Cell Biol. 1999 Jan 25;144(2):281-92. doi: 10.1083/jcb.144.2.281.

Ordering the cytochrome c-initiated caspase cascade: hierarchical activation of 
caspases-2, -3, -6, -7, -8, and -10 in a caspase-9-dependent manner.

Slee EA(1), Harte MT, Kluck RM, Wolf BB, Casiano CA, Newmeyer DD, Wang HG, Reed 
JC, Nicholson DW, Alnemri ES, Green DR, Martin SJ.

Author information:
(1)Molecular Cell Biology Laboratory, Department of Biology, National University 
of Ireland, Maynooth, Co. Kildare, Ireland.

Exit of cytochrome c from mitochondria into the cytosol has been implicated as 
an important step in apoptosis. In the cytosol, cytochrome c binds to the CED-4 
homologue, Apaf-1, thereby triggering Apaf-1-mediated activation of caspase-9. 
Caspase-9 is thought to propagate the death signal by triggering other caspase 
activation events, the details of which remain obscure. Here, we report that six 
additional caspases (caspases-2, -3, -6, -7, -8, and -10) are processed in 
cell-free extracts in response to cytochrome c, and that three others 
(caspases-1, -4, and -5) failed to be activated under the same conditions. In 
vitro association assays confirmed that caspase-9 selectively bound to Apaf-1, 
whereas caspases-1, -2, -3, -6, -7, -8, and -10 did not. Depletion of caspase-9 
from cell extracts abrogated cytochrome c-inducible activation of caspases-2, 
-3, -6, -7, -8, and -10, suggesting that caspase-9 is required for all of these 
downstream caspase activation events. Immunodepletion of caspases-3, -6, and -7 
from cell extracts enabled us to order the sequence of caspase activation events 
downstream of caspase-9 and reveal the presence of a branched caspase cascade. 
Caspase-3 is required for the activation of four other caspases (-2, -6, -8, and 
-10) in this pathway and also participates in a feedback amplification loop 
involving caspase-9.

DOI: 10.1083/jcb.144.2.281
PMCID: PMC2132895
PMID: 9922454 [Indexed for MEDLINE]

## Full Text

Numerous studies have implicated caspases (cysteine aspartate–specific proteases) as the molecular instigators of apoptosis (Yuan et al., 1993; Gagliardini et al., 1994; Kumar et al., 1994; Lazebnik et al., 1994; Wang et al., 1994; Nicholson et al., 1995; Tewari et al., 1995; Kuida et al., 1996). Caspases are a family of human proteases that cleave their substrates after aspartic acid residues, an uncommon substrate preference (Jacobson and Evan, 1994; Martin and Green, 1995; Alnemri et al., 1996; Chinnaiyan and Dixit, 1996; Henkart, 1996; Alnemri, 1997; Salvesen and Dixit, 1997). Caspases are typically constitutively present within cells as inactive zymogens that require proteolytic processing to achieve their active, two-chain configurations (Thornberry et al., 1992; Walker et al., 1994; Darmon et al., 1995; Gu et al., 1995; Duan et al., 1996; Schlegel et al., 1996; MacFarlane et al., 1997). In vitro, caspases are known to cleave a number of structural as well as RNA splicing and DNA repair-associated proteins and can also process other caspases (Casciola-Rosen et al., 1994, 1995; Brancolini et al., 1995; Emoto et al., 1995; Martin et al., 1995a; Tewari et al., 1995; Casiano et al., 1996; Fernandes-Alnemri et al., 1996; Hsu and Yeh, 1996; Kayalar et al., 1996; Takahashi et al., 1996; Weaver et al., 1996). The consequences of these cleavage events are now emerging and suggest that they are responsible for many of the phenotypic changes that occur during apoptosis. In addition, the observation that caspases can process other caspases suggests that there is likely to be a stepwise activation of caspases during apoptosis, similar to the clotting or complement cascades (Martin and Green, 1995).

Several studies suggest that receptor-associated adaptor proteins, such as FADD/MORT-1, that facilitate close association of certain caspases promote caspase autoprocessing (Boldin et al., 1996; Fernandes-Alnemri et al., 1996; Muzio et al., 1996; Ahmad et al., 1997; Duan and Dixit, 1997; Yang et al., 1998). Similar adaptor molecules, such as the recently described CED-4 homologue, Apaf-1, may play key roles in promoting apoptosis by clustering caspases at intracellular sites. Current evidence suggests that there are several distinct routes to caspase activation depending upon the stimulus that initiates the death program.

Many studies have shown that cytochrome c enters the cytosol during apoptosis, probably as a result of loss of this protein from mitochondria rather than as a consequence of failed import (Liu et al., 1996; Kluck et al., 1997a,b; Reed, 1997; Yang et al., 1997; Bossy-Wetzel et al., 1998). Cell death initiator or repressor proteins such as Bid and Bcl-2 have been shown to regulate this event, suggesting that this is a critical step in the death signaling cascade (Kluck et al., 1997a; Yang et al., 1997; Li et al., 1998; Luo et al., 1998). Studies using cell-free systems have shown that cytochrome c, in association with dATP, is capable of initiating apoptosis-like changes in cytosols derived from a variety of cell types (Liu et al., 1996; Kluck et al., 1997a,b; Deveraux et al., 1998; Pan et al., 1998a). The apoptosis-promoting activity of cytochrome c is due to its ability to interact with the CED-4 homologue Apaf-1 (Zou et al., 1997). Binding of cytochrome c to Apaf-1 enables this protein to recruit caspase-9 and to stimulate processing of the inactive caspase-9 zymogen to its active form (Li et al., 1997; Srinivasula et al., 1998). Once active, caspase-9 then presumably triggers a cascade of caspase activation events leading to apoptosis.

To explore more fully the range of caspase activation events that are triggered by cytochrome c, we have used a human cell-free system based on Jurkat postnuclear extracts. Here, we show that cytochrome c is capable of initiating processing of multiple caspases (-2, -3, -6, -7, -8, -9, and -10) in cell-free extracts, as well as a range of biochemical and morphological events characteristic of apoptosis. In contrast, activation of caspases-1, -4, and -5 was not observed in response to cytochrome c, suggesting that these caspases do not participate in apoptosis or do so upstream of the point of entry of cytochrome c into the cytosol. Strikingly, depletion of caspase-9 from cell extracts rendered all of the other caspases examined unresponsive to cytochrome c, suggesting that all of these caspase activation events lie on the same pathway, with caspase-9 at the apex of the cascade. Based on data generated by immunodepletion of specific caspases, we propose an order of the caspase activation events that lie downstream of caspase-9 in the cytochrome c–inducible pathway.

Materials and MethodsMaterialsAnti–caspase-3 and anti–caspase-9 polyclonal antibodies were generated by immunizing rabbits with GST-caspase-3 fusion protein or purified recombinant caspase-9, respectively; anti–caspase-3 and anti–caspase-7 mouse mAbs were purchased from Transduction Laboratories; purified rabbit polyclonal anti–caspase-6 antibody was purchased from Upstate Biotechnology; rabbit polyclonal anti–caspase-1 (ICE) was kindly provided by Dr. Douglas K. Miller; anti-U1snRNP and anti-PARP autoantibodies were derived from human subjects, as previously described (Casiano et al., 1996); anti–α-fodrin (nonerythroid spectrin) was purchased from Chemicon International; and anti–β-actin antibody was purchased from ICN. Ac-YVAD-CHO and Ac-DEVD-CHO peptides were purchased from BACHEM Bioscience; YVAD-pNA and DEVD-pNA peptides were purchased from Biomol Ltd. GST-CrmA fusion protein was kindly provided by Dr. David Pickup. Bovine heart cytochrome c was purchased from Sigma Chemical Co.
GST-Apaf-11-97 and GST-Apaf-11-401 fusion proteins were produced by PCR-mediated amplification of the relevant coding sequences from the full-length Apaf-1 cDNA (kindly provided by Dr. Xiaodong Wang), followed by subcloning of the resulting PCR products in-frame with the GST coding region of pGEX4TK2 (Pharmacia). Plasmids encoding GST and GST fusion proteins were transformed into Escherichia coli DH5α and bacteria were induced to express the recombinant proteins in the presence of 100 μM IPTG for 4 h at 30°C. GST and GST fusion proteins were subsequently purified using glutathione Sepharose (Pharmacia) according to standard procedures.In Vitro Association AssaysThe ability of caspases-1, -2, -3, -6, -7, -8, -9, and -10 to interact with GST-Apaf-1 fusion proteins was assessed as follows. [35S]Methionine-labeled caspases (5–15-μl aliquots of translation reactions) were brought to 200 μl in GST buffer (50 mM Tris, pH 7.6, 120 mM NaCl, 0.1% CHAPS, 100 μM PMSF, 10 μg/ml leupeptin, and 2 μg/ml aprotinin). 2-μl aliquots (∼6 μg protein) of glutathione Sepharose–immobilized GST or GST-Apaf-1 fusion proteins were then added, followed by incubation for 2 h at 4°C under constant rotation. Bead complexes were then washed several times in GST buffer and bound caspases were detected by SDS-PAGE/fluorography.Depletion of Caspases from Cell ExtractsCaspase-9 was depleted from cell extracts using either glutathione Sepharose–immobilized GST-Apaf-11-97 or protein A/G agarose–immobilized anti–caspase-9 antibody, as follows. For GST-Apaf-11-97 depletions, 40 μl of a 50% slurry of GST-Apaf-1 or GST was added to 100-μl aliquots of Jurkat cell extract which were incubated overnight at 4°C under constant rotation. Beads were then pelleted and extracts were used immediately. For antibody depletions, 40-μl aliquots of protein A/G agarose (Santa Cruz Biotechnology) were precoated with anti–caspase-9 rabbit polyclonal antibody by incubation with 50 μl of either anti–caspase-9 antiserum or a control (anti-RelA; Santa Cruz Biotechnology) rabbit polyclonal in a total volume of 300 μl in PBS, pH 7.2, for 3 h at 4°C under rotation. Antibody-coated beads were then washed three times before addition to Jurkat cell extracts (100 μl) which were incubated overnight under constant rotation at 4°C. Beads were then removed from the extracts before use. Caspase-3, -6, and -7 immunodepletions were performed in a similar manner with the exception that 5 μg of each antibody was used to precoat 40-μl aliquots of protein A/G agarose before depletion.Preparation of Cell-free ExtractsCell-free extracts were generated from Jurkat T lymphoblastoid cells or MCF-7 cells as previously described (Martin et al., 1995b, 1996), with the following modifications. Cells (2–5 × 108) were pelleted and washed twice with PBS, pH 7.2, followed by a single wash with 5 ml of ice-cold cell extract buffer (CEB;1 20 mM Hepes-KOH, pH 7.5, 10 mM KCl, 1.5 mM MgCl2, 1 mM EDTA, 1 mM EGTA, 1 mM dithiothreitol, 100 μM PMSF, 10 μg/ml leupeptin, 2 μg/ml aprotinin). Cells were then transferred to a 2-ml Dounce-type homogenizer, were pelleted, and two volumes of ice-cold CEB was added to the volume of the packed cell pellet. Cells were allowed to swell under the hypotonic conditions for 15 min on ice. Cells were then disrupted with 20 strokes of a B-type pestle. Lysis was confirmed by examination of a small aliquot of the suspension under a light microscope. Lysates were then transferred to Eppendorf tubes and were centrifuged at 15,000 g for 15 min at 4°C (S15 or postnuclear extracts). The supernatant was removed while taking care to avoid the pellet. Supernatants were then frozen in aliquots at −70°C until required.Cell-free ReactionsCell-free reactions were typically set up in 10- or 100-μl reaction volumes. For 100-μl scale reactions, 50 μl of cell extract (∼5 mg/ml) and 10 μl of rat liver nuclei were brought to a final volume of 100 μl in CEB, with or without peptides or proteins solubilized in the same buffer. Apoptosis was typically induced by addition of bovine heart cytochrome c to extracts at a final concentration of 50 μg/ml. Where necessary, dATP was also to a final concentration of 1 mM, although many extracts did not require addition of this nucleotide triphosphate. To initiate apoptosis, extracts were incubated at 37°C for periods of up to 3 h. At time points indicated in the text, 2-μl aliquots were removed for determination of percentages of apoptotic nuclei using Hoechst 33342 staining, as previously described (Martin et al., 1995b, 1996). Samples of extract (10–20 μl) were also removed at times indicated in the text and frozen at −70°C for subsequent SDS-PAGE/Western blot or fluorographic determination of substrate cleavage profiles or caspase activation.Coupled In Vitro Transcription/Translations[35S]Methionine-labeled caspases were in vitro transcribed and translated using the TNT kit (Promega), as previously described (Martin et al., 1996). For use in coupled in vitro transcription/translation experiments, plasmids encoding each of the caspases used were grown in E. coli DH5α strain and were purified using tip-100 Qiagen columns. Typically, 1 μg of plasmid was used in a 50 μl transcription/translation reaction containing 4 μl of translation grade [35S]methionine (1,000 μCi/ml; ICN).YVAD-pNA and DEVD-pNA Cleavage AssayAt times indicated in the text, 10-μl aliquots of cell-free reactions were removed and were diluted to 100 μl by the addition of ice-cold protease reaction buffer (PRB; 50 mM Hepes, pH 7.4, 75 mM NaCl, 0.1% CHAPS, 2 mM dithiothreitol). Samples were held on ice until completion of the experiment and were then divided into two separate 50-μl portions for the separate assessment of YVAD-p-nitroanalide (YVAD-pNA) and DEVD-pNA cleavage activity, respectively. To each 50-μl aliquot, 5.5 μl of a 10× stock of each peptide (500 μM) was added such that the final concentration of either peptide in the reaction was 50 μM. Reactions were then incubated for 30 min at 37°C, followed by addition of 950 μl ice-cold dH2O to stop the reaction. OD400 readings of each sample were then taken against a blank containing buffer and peptide alone (i.e., no extract).SDS-PAGE and Western Blot AnalysisProteins were subjected to standard SDS-PAGE at 60–70 V and were transferred onto 0.45 μM PVDF membranes (Bio-Rad) for 3 h at 50–75 mA, followed by probing for various proteins using the polyclonal antibodies described under materials. Bound antibodies were detected using appropriate peroxidase-coupled secondary antibodies (Amersham), followed by detection using the Supersignal chemiluminescence system (Pierce), all as previously described (Martin et al., 1996).

Materials and Methods

MaterialsAnti–caspase-3 and anti–caspase-9 polyclonal antibodies were generated by immunizing rabbits with GST-caspase-3 fusion protein or purified recombinant caspase-9, respectively; anti–caspase-3 and anti–caspase-7 mouse mAbs were purchased from Transduction Laboratories; purified rabbit polyclonal anti–caspase-6 antibody was purchased from Upstate Biotechnology; rabbit polyclonal anti–caspase-1 (ICE) was kindly provided by Dr. Douglas K. Miller; anti-U1snRNP and anti-PARP autoantibodies were derived from human subjects, as previously described (Casiano et al., 1996); anti–α-fodrin (nonerythroid spectrin) was purchased from Chemicon International; and anti–β-actin antibody was purchased from ICN. Ac-YVAD-CHO and Ac-DEVD-CHO peptides were purchased from BACHEM Bioscience; YVAD-pNA and DEVD-pNA peptides were purchased from Biomol Ltd. GST-CrmA fusion protein was kindly provided by Dr. David Pickup. Bovine heart cytochrome c was purchased from Sigma Chemical Co.
GST-Apaf-11-97 and GST-Apaf-11-401 fusion proteins were produced by PCR-mediated amplification of the relevant coding sequences from the full-length Apaf-1 cDNA (kindly provided by Dr. Xiaodong Wang), followed by subcloning of the resulting PCR products in-frame with the GST coding region of pGEX4TK2 (Pharmacia). Plasmids encoding GST and GST fusion proteins were transformed into Escherichia coli DH5α and bacteria were induced to express the recombinant proteins in the presence of 100 μM IPTG for 4 h at 30°C. GST and GST fusion proteins were subsequently purified using glutathione Sepharose (Pharmacia) according to standard procedures.

Anti–caspase-3 and anti–caspase-9 polyclonal antibodies were generated by immunizing rabbits with GST-caspase-3 fusion protein or purified recombinant caspase-9, respectively; anti–caspase-3 and anti–caspase-7 mouse mAbs were purchased from Transduction Laboratories; purified rabbit polyclonal anti–caspase-6 antibody was purchased from Upstate Biotechnology; rabbit polyclonal anti–caspase-1 (ICE) was kindly provided by Dr. Douglas K. Miller; anti-U1snRNP and anti-PARP autoantibodies were derived from human subjects, as previously described (Casiano et al., 1996); anti–α-fodrin (nonerythroid spectrin) was purchased from Chemicon International; and anti–β-actin antibody was purchased from ICN. Ac-YVAD-CHO and Ac-DEVD-CHO peptides were purchased from BACHEM Bioscience; YVAD-pNA and DEVD-pNA peptides were purchased from Biomol Ltd. GST-CrmA fusion protein was kindly provided by Dr. David Pickup. Bovine heart cytochrome c was purchased from Sigma Chemical Co.

GST-Apaf-11-97 and GST-Apaf-11-401 fusion proteins were produced by PCR-mediated amplification of the relevant coding sequences from the full-length Apaf-1 cDNA (kindly provided by Dr. Xiaodong Wang), followed by subcloning of the resulting PCR products in-frame with the GST coding region of pGEX4TK2 (Pharmacia). Plasmids encoding GST and GST fusion proteins were transformed into Escherichia coli DH5α and bacteria were induced to express the recombinant proteins in the presence of 100 μM IPTG for 4 h at 30°C. GST and GST fusion proteins were subsequently purified using glutathione Sepharose (Pharmacia) according to standard procedures.

In Vitro Association AssaysThe ability of caspases-1, -2, -3, -6, -7, -8, -9, and -10 to interact with GST-Apaf-1 fusion proteins was assessed as follows. [35S]Methionine-labeled caspases (5–15-μl aliquots of translation reactions) were brought to 200 μl in GST buffer (50 mM Tris, pH 7.6, 120 mM NaCl, 0.1% CHAPS, 100 μM PMSF, 10 μg/ml leupeptin, and 2 μg/ml aprotinin). 2-μl aliquots (∼6 μg protein) of glutathione Sepharose–immobilized GST or GST-Apaf-1 fusion proteins were then added, followed by incubation for 2 h at 4°C under constant rotation. Bead complexes were then washed several times in GST buffer and bound caspases were detected by SDS-PAGE/fluorography.

In Vitro Association Assays

The ability of caspases-1, -2, -3, -6, -7, -8, -9, and -10 to interact with GST-Apaf-1 fusion proteins was assessed as follows. [35S]Methionine-labeled caspases (5–15-μl aliquots of translation reactions) were brought to 200 μl in GST buffer (50 mM Tris, pH 7.6, 120 mM NaCl, 0.1% CHAPS, 100 μM PMSF, 10 μg/ml leupeptin, and 2 μg/ml aprotinin). 2-μl aliquots (∼6 μg protein) of glutathione Sepharose–immobilized GST or GST-Apaf-1 fusion proteins were then added, followed by incubation for 2 h at 4°C under constant rotation. Bead complexes were then washed several times in GST buffer and bound caspases were detected by SDS-PAGE/fluorography.

Depletion of Caspases from Cell ExtractsCaspase-9 was depleted from cell extracts using either glutathione Sepharose–immobilized GST-Apaf-11-97 or protein A/G agarose–immobilized anti–caspase-9 antibody, as follows. For GST-Apaf-11-97 depletions, 40 μl of a 50% slurry of GST-Apaf-1 or GST was added to 100-μl aliquots of Jurkat cell extract which were incubated overnight at 4°C under constant rotation. Beads were then pelleted and extracts were used immediately. For antibody depletions, 40-μl aliquots of protein A/G agarose (Santa Cruz Biotechnology) were precoated with anti–caspase-9 rabbit polyclonal antibody by incubation with 50 μl of either anti–caspase-9 antiserum or a control (anti-RelA; Santa Cruz Biotechnology) rabbit polyclonal in a total volume of 300 μl in PBS, pH 7.2, for 3 h at 4°C under rotation. Antibody-coated beads were then washed three times before addition to Jurkat cell extracts (100 μl) which were incubated overnight under constant rotation at 4°C. Beads were then removed from the extracts before use. Caspase-3, -6, and -7 immunodepletions were performed in a similar manner with the exception that 5 μg of each antibody was used to precoat 40-μl aliquots of protein A/G agarose before depletion.

Depletion of Caspases from Cell Extracts

Caspase-9 was depleted from cell extracts using either glutathione Sepharose–immobilized GST-Apaf-11-97 or protein A/G agarose–immobilized anti–caspase-9 antibody, as follows. For GST-Apaf-11-97 depletions, 40 μl of a 50% slurry of GST-Apaf-1 or GST was added to 100-μl aliquots of Jurkat cell extract which were incubated overnight at 4°C under constant rotation. Beads were then pelleted and extracts were used immediately. For antibody depletions, 40-μl aliquots of protein A/G agarose (Santa Cruz Biotechnology) were precoated with anti–caspase-9 rabbit polyclonal antibody by incubation with 50 μl of either anti–caspase-9 antiserum or a control (anti-RelA; Santa Cruz Biotechnology) rabbit polyclonal in a total volume of 300 μl in PBS, pH 7.2, for 3 h at 4°C under rotation. Antibody-coated beads were then washed three times before addition to Jurkat cell extracts (100 μl) which were incubated overnight under constant rotation at 4°C. Beads were then removed from the extracts before use. Caspase-3, -6, and -7 immunodepletions were performed in a similar manner with the exception that 5 μg of each antibody was used to precoat 40-μl aliquots of protein A/G agarose before depletion.

Preparation of Cell-free ExtractsCell-free extracts were generated from Jurkat T lymphoblastoid cells or MCF-7 cells as previously described (Martin et al., 1995b, 1996), with the following modifications. Cells (2–5 × 108) were pelleted and washed twice with PBS, pH 7.2, followed by a single wash with 5 ml of ice-cold cell extract buffer (CEB;1 20 mM Hepes-KOH, pH 7.5, 10 mM KCl, 1.5 mM MgCl2, 1 mM EDTA, 1 mM EGTA, 1 mM dithiothreitol, 100 μM PMSF, 10 μg/ml leupeptin, 2 μg/ml aprotinin). Cells were then transferred to a 2-ml Dounce-type homogenizer, were pelleted, and two volumes of ice-cold CEB was added to the volume of the packed cell pellet. Cells were allowed to swell under the hypotonic conditions for 15 min on ice. Cells were then disrupted with 20 strokes of a B-type pestle. Lysis was confirmed by examination of a small aliquot of the suspension under a light microscope. Lysates were then transferred to Eppendorf tubes and were centrifuged at 15,000 g for 15 min at 4°C (S15 or postnuclear extracts). The supernatant was removed while taking care to avoid the pellet. Supernatants were then frozen in aliquots at −70°C until required.

Preparation of Cell-free Extracts

Cell-free extracts were generated from Jurkat T lymphoblastoid cells or MCF-7 cells as previously described (Martin et al., 1995b, 1996), with the following modifications. Cells (2–5 × 108) were pelleted and washed twice with PBS, pH 7.2, followed by a single wash with 5 ml of ice-cold cell extract buffer (CEB;1 20 mM Hepes-KOH, pH 7.5, 10 mM KCl, 1.5 mM MgCl2, 1 mM EDTA, 1 mM EGTA, 1 mM dithiothreitol, 100 μM PMSF, 10 μg/ml leupeptin, 2 μg/ml aprotinin). Cells were then transferred to a 2-ml Dounce-type homogenizer, were pelleted, and two volumes of ice-cold CEB was added to the volume of the packed cell pellet. Cells were allowed to swell under the hypotonic conditions for 15 min on ice. Cells were then disrupted with 20 strokes of a B-type pestle. Lysis was confirmed by examination of a small aliquot of the suspension under a light microscope. Lysates were then transferred to Eppendorf tubes and were centrifuged at 15,000 g for 15 min at 4°C (S15 or postnuclear extracts). The supernatant was removed while taking care to avoid the pellet. Supernatants were then frozen in aliquots at −70°C until required.

Cell-free ReactionsCell-free reactions were typically set up in 10- or 100-μl reaction volumes. For 100-μl scale reactions, 50 μl of cell extract (∼5 mg/ml) and 10 μl of rat liver nuclei were brought to a final volume of 100 μl in CEB, with or without peptides or proteins solubilized in the same buffer. Apoptosis was typically induced by addition of bovine heart cytochrome c to extracts at a final concentration of 50 μg/ml. Where necessary, dATP was also to a final concentration of 1 mM, although many extracts did not require addition of this nucleotide triphosphate. To initiate apoptosis, extracts were incubated at 37°C for periods of up to 3 h. At time points indicated in the text, 2-μl aliquots were removed for determination of percentages of apoptotic nuclei using Hoechst 33342 staining, as previously described (Martin et al., 1995b, 1996). Samples of extract (10–20 μl) were also removed at times indicated in the text and frozen at −70°C for subsequent SDS-PAGE/Western blot or fluorographic determination of substrate cleavage profiles or caspase activation.

Cell-free Reactions

Cell-free reactions were typically set up in 10- or 100-μl reaction volumes. For 100-μl scale reactions, 50 μl of cell extract (∼5 mg/ml) and 10 μl of rat liver nuclei were brought to a final volume of 100 μl in CEB, with or without peptides or proteins solubilized in the same buffer. Apoptosis was typically induced by addition of bovine heart cytochrome c to extracts at a final concentration of 50 μg/ml. Where necessary, dATP was also to a final concentration of 1 mM, although many extracts did not require addition of this nucleotide triphosphate. To initiate apoptosis, extracts were incubated at 37°C for periods of up to 3 h. At time points indicated in the text, 2-μl aliquots were removed for determination of percentages of apoptotic nuclei using Hoechst 33342 staining, as previously described (Martin et al., 1995b, 1996). Samples of extract (10–20 μl) were also removed at times indicated in the text and frozen at −70°C for subsequent SDS-PAGE/Western blot or fluorographic determination of substrate cleavage profiles or caspase activation.

Coupled In Vitro Transcription/Translations[35S]Methionine-labeled caspases were in vitro transcribed and translated using the TNT kit (Promega), as previously described (Martin et al., 1996). For use in coupled in vitro transcription/translation experiments, plasmids encoding each of the caspases used were grown in E. coli DH5α strain and were purified using tip-100 Qiagen columns. Typically, 1 μg of plasmid was used in a 50 μl transcription/translation reaction containing 4 μl of translation grade [35S]methionine (1,000 μCi/ml; ICN).

Coupled In Vitro Transcription/Translations

[35S]Methionine-labeled caspases were in vitro transcribed and translated using the TNT kit (Promega), as previously described (Martin et al., 1996). For use in coupled in vitro transcription/translation experiments, plasmids encoding each of the caspases used were grown in E. coli DH5α strain and were purified using tip-100 Qiagen columns. Typically, 1 μg of plasmid was used in a 50 μl transcription/translation reaction containing 4 μl of translation grade [35S]methionine (1,000 μCi/ml; ICN).

YVAD-pNA and DEVD-pNA Cleavage AssayAt times indicated in the text, 10-μl aliquots of cell-free reactions were removed and were diluted to 100 μl by the addition of ice-cold protease reaction buffer (PRB; 50 mM Hepes, pH 7.4, 75 mM NaCl, 0.1% CHAPS, 2 mM dithiothreitol). Samples were held on ice until completion of the experiment and were then divided into two separate 50-μl portions for the separate assessment of YVAD-p-nitroanalide (YVAD-pNA) and DEVD-pNA cleavage activity, respectively. To each 50-μl aliquot, 5.5 μl of a 10× stock of each peptide (500 μM) was added such that the final concentration of either peptide in the reaction was 50 μM. Reactions were then incubated for 30 min at 37°C, followed by addition of 950 μl ice-cold dH2O to stop the reaction. OD400 readings of each sample were then taken against a blank containing buffer and peptide alone (i.e., no extract).

YVAD-pNA and DEVD-pNA Cleavage Assay

At times indicated in the text, 10-μl aliquots of cell-free reactions were removed and were diluted to 100 μl by the addition of ice-cold protease reaction buffer (PRB; 50 mM Hepes, pH 7.4, 75 mM NaCl, 0.1% CHAPS, 2 mM dithiothreitol). Samples were held on ice until completion of the experiment and were then divided into two separate 50-μl portions for the separate assessment of YVAD-p-nitroanalide (YVAD-pNA) and DEVD-pNA cleavage activity, respectively. To each 50-μl aliquot, 5.5 μl of a 10× stock of each peptide (500 μM) was added such that the final concentration of either peptide in the reaction was 50 μM. Reactions were then incubated for 30 min at 37°C, followed by addition of 950 μl ice-cold dH2O to stop the reaction. OD400 readings of each sample were then taken against a blank containing buffer and peptide alone (i.e., no extract).

SDS-PAGE and Western Blot AnalysisProteins were subjected to standard SDS-PAGE at 60–70 V and were transferred onto 0.45 μM PVDF membranes (Bio-Rad) for 3 h at 50–75 mA, followed by probing for various proteins using the polyclonal antibodies described under materials. Bound antibodies were detected using appropriate peroxidase-coupled secondary antibodies (Amersham), followed by detection using the Supersignal chemiluminescence system (Pierce), all as previously described (Martin et al., 1996).

SDS-PAGE and Western Blot Analysis

Proteins were subjected to standard SDS-PAGE at 60–70 V and were transferred onto 0.45 μM PVDF membranes (Bio-Rad) for 3 h at 50–75 mA, followed by probing for various proteins using the polyclonal antibodies described under materials. Bound antibodies were detected using appropriate peroxidase-coupled secondary antibodies (Amersham), followed by detection using the Supersignal chemiluminescence system (Pierce), all as previously described (Martin et al., 1996).

ResultsCytochrome c Initiates Multiple Features of Apoptosis in Jurkat Cell ExtractsAddition of purified cytochrome c to postnuclear (15,000 g; S15) extracts of Jurkat T lymphoblastoid cells was sufficient to initiate the whole spectrum of events characteristic of apoptosis in these extracts. Nuclei incubated in the extracts in the presence of cytochrome c rapidly exhibited apoptotic features (chromatin margination and nuclear fragmentation; Fig. 1 A) and chromatin also underwent fragmentation into ∼200-bp multiples (data not shown). Proteolysis of several caspase substrates (α-fodrin, U1sn- RNP, PARP) was also observed in response to cytochrome c (Fig. 1 B). Interestingly, although previous reports have shown that addition of dATP (or ATP) to cell extracts is required for the proapoptotic activities of cytochrome c, many extracts did not require addition of exogenous nucleotide triphosphates, presumably due to sufficiently high levels of ATP or dATP endogenous to these extracts. Cytochrome c–initiated Apoptosis Is Associated with Proteolytic Processing of Caspase-3, but Not Caspase-1Previous studies have shown that caspases-3 and -9 are activated in response to cytochrome c (Liu et al., 1996; Li et al., 1997; Kluck et al., 1997b; Zou et al., 1997; Pan et al., 1998a). We initially confirmed these observations before assessing the activation of other caspases in this context. Fig. 2 demonstrates that caspase-3 endogenous to Jurkat cell extracts was rapidly converted from the 36-kD proenzyme to the p17/p12 mature form in the presence of cytochrome c. Processing occurred in a two-step manner, with the initial appearance of a p24/p12 intermediate in the extracts, followed by accumulation of the mature p17/ p12 form of the enzyme (Fig. 2, A and C), reminiscent of the mechanism of activation of caspase-3 in response to granzyme B (Martin et al., 1996). This was further confirmed by addition of [35S]methionine-labeled caspase-3 to the extracts, which enabled detection of the caspase-3-p12 chain that was not recognized by the anti–caspase-3 polyclonal antibody used (Fig. 2 B). In direct contrast, conversion of caspase-1 (ICE) to its mature form was not detected in the same extracts over an identical time course (Fig. 2 A). To further confirm that processed caspases were active, we used synthetic tetrapeptide substrates that are preferentially cleaved by caspase-1–like (YVAD-pNA) or caspase-3–like (DEVD-pNA) proteases to assess the induction of caspase-1– or caspase-3–like proteolytic activity in response to cytochrome c. We observed a striking induction of DEVD-pNA cleaving activity within 15 min of addition of cytochrome c to the extracts, whereas YVAD-pNA cleaving activity did not rise above basal levels during the same time course (Fig. 2 D), in agreement with our observations on the absence of caspase-1 processing in this context (Fig. 2 A). Cytochrome c induced apoptotic changes in nuclei added to the extracts at concentrations of as little as 1–5 μg/ml, but had no direct effects on nuclei in the absence of cell extract (Fig. 3 A). Interestingly, at cytochrome c concentrations where only partial caspase-3 activation was observed (5 μg/ml), substrates such as fodrin, PARP, and U1snRNP were almost completely cleaved (Fig. 3 B), suggesting that only a small amount of the total caspase-3 pool is required in order to effect complete proteolysis of these substrates. An alternative explanation is that cytochrome c activates other caspases in the extracts that are capable of cleaving these substrates. Cytochrome c failed to trigger caspase-1 processing at any of the concentrations tested (Fig. 3 B). Cytochrome c Initiates a Cascade of Protease Activation Events Involving Caspases-2, -3, -6, -7, -8, -9, and -10Recent studies have shown that caspase-9 is activated in response to cytochrome c due to clustering of caspase-9 by Apaf-1 and that this results in activation of Caspases-3 (Li et al., 1997; Pan et al., 1998a; Srinivasula et al., 1998). To explore the full range of caspase activation events in the cytochrome c–initiated proteolytic cascade, we introduced [35S]methionine-labeled caspases-1, -2, -3, -4, -5, -6, -7, -8, -9, and -10 into Jurkat cell–free extracts and monitored processing of these proteases to their mature forms. Fig. 4 demonstrates that cytochrome c/dATP triggered maturation of caspases-2, -3, -6, -7, -8, -9, and -10, whereas none of the ICE subfamily proteases (caspases-1, -4, and -5) were processed under the same conditions. Significantly, cytochrome c failed to activate any of the caspases in the absence of cell extract, suggesting that a cytosolic factor such as Apaf-1 was required for all of these activation events. Caspases with long prodomains such as caspases-2, -8, and -10 are generally considered to be upstream or signaling caspases in the cell death pathway due to their ability to associate with cell surface death receptor molecules such as Fas/CD95 or TNFR1. Therefore, it was somewhat surprising that these caspases became activated in the presence of cytochrome c. However, it has been reported recently that thymocytes from APAF-1 null mice are impaired with respect to caspase-2 and caspase-8 activation in response to several proapoptotic stimuli (Yoshida et al., 1998). Similarly, dexamethasone-induced processing of caspases-2 and -8 was found to be impaired in mice deficient for caspase-9 (Hakem et al., 1998). These data suggest that these caspases are indeed activated in the Apaf-1 pathway in vivo. To explore the range of cytochrome c–inducible caspase activation events in more detail, we monitored the kinetics of activation of all caspases relative to each other in this system. Fig. 5 shows that detectable activation of most caspases, with the exceptions of caspases-8 and -10, appeared to occur contemporaneously, typically within 30 min of addition of cytochrome c to the extracts. In contrast, processing of caspases-8 and -10 were noticeably delayed relative to the other caspases, suggesting that these caspases might be activated late in this pathway. APAF-1 Selectively Binds to Caspase-9The observation that multiple caspases were activated in response to cytochrome c suggested either that all of these caspase activation events occurred downstream of caspase-9 (the only caspase known to directly associate with Apaf-1/ cytochrome c), or that a number of distinct caspase activation pathways could be instigated by cytochrome c, independent of caspase-9. To discriminate between these possibilities, we first explored whether Apaf-1 could directly bind caspases other than caspase-9. Fig. 6 demonstrates that a GST fusion protein comprising the CED-3 homology region of Apaf-1 (amino acid [aa] residues 1–97) selectively bound caspase-9 but did not bind any of the other caspases (-1, -2, -3, -6, -7, -8, and -10) tested. Similar results were obtained using a different GST fusion that spanned the CED-3 as well as the CED-4 homology regions of Apaf-1 (aa 1–412; Slee, E.A., and S.J. Martin, data not shown). These data demonstrate that Apaf-1 is highly selective for caspase-9, although they do not rule out the possibility that other caspases may become recruited to Apaf-1 via suitable adaptor molecules. Depletion of Caspase-9 from Jurkat Extracts Abrogates Processing of all Caspases in Response to Cytochrome cTo determine whether caspase-9 was required for activation of all other caspases in this context, we depleted this protease from Jurkat extracts before the addition of cytochrome c. Depletion of caspase-9 using Sepharose- immobilized GST-Apaf-11-97 (Fig. 7 A), or anti–caspase-9 polyclonal antibody (Fig. 7 B), rendered all caspases unresponsive to cytochrome c. In contrast, mock depletions performed using Sepharose-GST or control polyclonal antibody did not interfere with cytochrome c–induced activation of any of the caspases examined (Fig. 7, A and B). These data suggest that caspase-9 is critical for cytochrome c–initiated caspase activation events and cannot be substituted for by the other caspases present in the extracts. They also provide further support for the idea that Apaf-1 is selective for caspase-9 (Fig. 6) and fails to initiate the apoptotic program in its absence. Ordering the Cytochrome c–inducible Caspase Cascade Downstream of Caspase-9As a preliminary approach to ordering the sequence of caspase activation events triggered by cytochrome c, we investigated the effects of the tetrapeptide caspase inhibitors YVAD-CHO and DEVD-CHO, as well as the cowpox virus–derived caspase inhibitor CrmA, on the processing of all caspases downstream of caspase-9. The caspase-1–selective inhibitor YVAD-CHO had similar effects on all caspases, exhibiting little inhibition of caspase activation except at the highest concentration tested (Fig. 8). These data are consistent with YVAD-CHO directly inhibiting caspase-9 at high concentrations and terminating activation of all downstream caspases, in agreement with the results obtained by depletion of caspase-9 from the extracts (Fig. 7). Broadly similar effects on all caspases was also observed using GST-CrmA, with the exception that activation of caspases-8 and -10 were blocked at all concentrations of this inhibitor, whereas processing of all of the other caspases was seen at the lowest concentration tested (0.2 μM). This suggests that caspase-8 processing is downstream of the other caspases in the context of cytochrome c. Strikingly, very different inhibitory effects were observed using the caspase-3– and caspase-7–selective inhibitor DEVD-CHO. Whereas processing of caspases-2, -6, -8, and -10 was completely blocked at all concentrations of DEVD-CHO tested, very significant processing of caspases-3 and -7 was observed in the presence of 1–10 μM of this inhibitor. These data suggest that, after caspase-9, caspases-3 and -7 are the next caspases to become activated in the cytochrome c–initiated caspase cascade. Inhibition of caspase-3 and -7 activities by DEVD-CHO prevents further activation of any other caspases downstream of this point.Fig. 8 also demonstrates that maturation of caspase-3 in the presence of 1–10 μM DEVD-CHO was arrested at the p24/p12 intermediate. This confirms our earlier observation (Fig. 2, A–C) that cytochrome c–initiated caspase-3 processing was achieved via two distinct processing steps, with the first step involving a single cleavage of the proenzyme between the large and small subunits to produce a p24/p12 intermediate which then undergoes further processing via a distinct protease. The observation that caspase-3 maturation was completely arrested at the p24/ p12 intermediate in the presence of DEVD-CHO suggests that the second step (i.e., removal of the prodomain) is autocatalytic. A similar two-step processing mechanism has been reported for granzyme B–mediated activation of caspase-3 (Fernandes-Alnemri et al., 1996; Martin et al., 1996).Depletion of Caspase-3 from Cell Extracts Ablates Cytochrome c–induced Processing of Caspases-2, -6, -8, and -10 and Reveals a Feedback Loop Involving Caspase-9To confirm that caspase-3 was activated upstream of caspases-2, -6, -8, and -10 in response to cytochrome c and to ask whether caspase-3 was required for processing of any of the other caspases in this context, we immunodepleted caspase-3 from cell extracts (Fig. 9). Caspase-3–depleted extracts were compared with mock-depleted extracts for their ability to support cytochrome c–induced processing of caspases-2, -3, -6, -7, -8, -9, and -10. Strikingly, removal of caspase-3 from the extracts abrogated processing of caspases-2, -6, -8, and -10 but had only a marginal effect on the processing of caspase-7 in the presence of cytochrome c (Fig. 9 A). These data suggest that caspases-3 and -7 are directly processed downstream of caspase-9 in the cytochrome c/Apaf-1–inducible caspase cascade and that caspase-3 is necessary for the activation (either directly or indirectly) of caspases-2, -6, -8, and -10. Interestingly, although caspase-3 was activated downstream of caspase-9 in response to cytochrome c as expected (Fig. 7), partial inhibition of caspase-9 processing was observed in caspase-3–depleted, but not in mock- depleted, extracts (Fig. 9 B). Consistent with this, reintroduced 35S-labeled caspase-3 was processed less efficiently in caspase-3–depleted extracts as compared with mock- depleted extracts, suggesting that caspase-9 activation was less efficient in these extracts (Fig. 9 A). In normal or mock-depleted extracts, the processed form of caspase-9 could be resolved into two major bands migrating at ∼37 and ∼35 kD, as recently reported (Srinivasula et al., 1998). However, in caspase-3–depleted extracts the 37-kD cleavage product was not produced (Fig. 9 B), suggesting that caspase-9 activation in response to cytochrome c is partially achieved via a feedback loop involving caspase-3. To confirm this, we compared Jurkat postnuclear extracts with similar extracts prepared from MCF-7 cells which are devoid of caspase-3 due to a deletion in exon 3 of the CASP-3 gene (Jänicke et al., 1998). Using MCF-7 cell extracts we confirmed that, in the absence of caspase-3, caspase-9 is processed to a single 35-kD cleavage product, whereas both the 35- and 37-kD cleavage products were produced in Jurkat extracts (Fig. 9 B). These data are consistent with the interpretation that cytochrome c/Apaf-1–triggered processing of caspase-9 is initially autocatalytic, producing the p35 form via cleavage at Asp-315, but upon activation of caspase-3 is also achieved via a feedback loop in which caspase-3 processes caspase-9 at Asp-330 (Fig. 9 C; Srinivasula et al., 1998).Caspase-6 Is Required for Cytochrome c–induced Processing of Caspases-8 and -10We next depleted caspase-6 or caspase-7 from Jurkat extracts to ask whether either of these caspases was required for any of the other caspase activation events seen in the presence of cytochrome c (Fig. 10). Immunodepletion of caspase-6 failed to have any effect on the processing of caspases-3, -7, and -9 in response to cytochrome c, consistent with caspase-6 activation being downstream of the latter caspases (Fig. 10 A). Caspase-2 processing was also unaffected in the absence of caspase-6, suggesting that caspase-2 is directly processed, along with caspase-6, upon activation of caspase-3 in the extracts. However, processing of caspases-8 and -10 was largely abrogated in extracts devoid of caspase-6 (Fig. 10 A). These data suggest that, upon activation by caspase-3, caspase-6 in turn promotes the processing of caspases-8 and -10 further down the cascade. In contrast to the effects seen after depletion of caspases-9, -3, or -6, immunodepletion of caspase-7 from the extracts failed to have any detectable inhibitory effects on any of the caspase processing events observed in the presence of cytochrome c (Fig. 10 B). Taken together, these data suggest an order of caspase activation events that take place distal to entry of cytochrome c into the cytosol (Fig. 11).

Cytochrome c Initiates Multiple Features of Apoptosis in Jurkat Cell ExtractsAddition of purified cytochrome c to postnuclear (15,000 g; S15) extracts of Jurkat T lymphoblastoid cells was sufficient to initiate the whole spectrum of events characteristic of apoptosis in these extracts. Nuclei incubated in the extracts in the presence of cytochrome c rapidly exhibited apoptotic features (chromatin margination and nuclear fragmentation; Fig. 1 A) and chromatin also underwent fragmentation into ∼200-bp multiples (data not shown). Proteolysis of several caspase substrates (α-fodrin, U1sn- RNP, PARP) was also observed in response to cytochrome c (Fig. 1 B). Interestingly, although previous reports have shown that addition of dATP (or ATP) to cell extracts is required for the proapoptotic activities of cytochrome c, many extracts did not require addition of exogenous nucleotide triphosphates, presumably due to sufficiently high levels of ATP or dATP endogenous to these extracts.

Cytochrome c Initiates Multiple Features of Apoptosis in Jurkat Cell Extracts

Addition of purified cytochrome c to postnuclear (15,000 g; S15) extracts of Jurkat T lymphoblastoid cells was sufficient to initiate the whole spectrum of events characteristic of apoptosis in these extracts. Nuclei incubated in the extracts in the presence of cytochrome c rapidly exhibited apoptotic features (chromatin margination and nuclear fragmentation; Fig. 1 A) and chromatin also underwent fragmentation into ∼200-bp multiples (data not shown). Proteolysis of several caspase substrates (α-fodrin, U1sn- RNP, PARP) was also observed in response to cytochrome c (Fig. 1 B). Interestingly, although previous reports have shown that addition of dATP (or ATP) to cell extracts is required for the proapoptotic activities of cytochrome c, many extracts did not require addition of exogenous nucleotide triphosphates, presumably due to sufficiently high levels of ATP or dATP endogenous to these extracts.

Cytochrome c–initiated Apoptosis Is Associated with Proteolytic Processing of Caspase-3, but Not Caspase-1Previous studies have shown that caspases-3 and -9 are activated in response to cytochrome c (Liu et al., 1996; Li et al., 1997; Kluck et al., 1997b; Zou et al., 1997; Pan et al., 1998a). We initially confirmed these observations before assessing the activation of other caspases in this context. Fig. 2 demonstrates that caspase-3 endogenous to Jurkat cell extracts was rapidly converted from the 36-kD proenzyme to the p17/p12 mature form in the presence of cytochrome c. Processing occurred in a two-step manner, with the initial appearance of a p24/p12 intermediate in the extracts, followed by accumulation of the mature p17/ p12 form of the enzyme (Fig. 2, A and C), reminiscent of the mechanism of activation of caspase-3 in response to granzyme B (Martin et al., 1996). This was further confirmed by addition of [35S]methionine-labeled caspase-3 to the extracts, which enabled detection of the caspase-3-p12 chain that was not recognized by the anti–caspase-3 polyclonal antibody used (Fig. 2 B). In direct contrast, conversion of caspase-1 (ICE) to its mature form was not detected in the same extracts over an identical time course (Fig. 2 A). To further confirm that processed caspases were active, we used synthetic tetrapeptide substrates that are preferentially cleaved by caspase-1–like (YVAD-pNA) or caspase-3–like (DEVD-pNA) proteases to assess the induction of caspase-1– or caspase-3–like proteolytic activity in response to cytochrome c. We observed a striking induction of DEVD-pNA cleaving activity within 15 min of addition of cytochrome c to the extracts, whereas YVAD-pNA cleaving activity did not rise above basal levels during the same time course (Fig. 2 D), in agreement with our observations on the absence of caspase-1 processing in this context (Fig. 2 A). Cytochrome c induced apoptotic changes in nuclei added to the extracts at concentrations of as little as 1–5 μg/ml, but had no direct effects on nuclei in the absence of cell extract (Fig. 3 A). Interestingly, at cytochrome c concentrations where only partial caspase-3 activation was observed (5 μg/ml), substrates such as fodrin, PARP, and U1snRNP were almost completely cleaved (Fig. 3 B), suggesting that only a small amount of the total caspase-3 pool is required in order to effect complete proteolysis of these substrates. An alternative explanation is that cytochrome c activates other caspases in the extracts that are capable of cleaving these substrates. Cytochrome c failed to trigger caspase-1 processing at any of the concentrations tested (Fig. 3 B).

Cytochrome c–initiated Apoptosis Is Associated with Proteolytic Processing of Caspase-3, but Not Caspase-1

Previous studies have shown that caspases-3 and -9 are activated in response to cytochrome c (Liu et al., 1996; Li et al., 1997; Kluck et al., 1997b; Zou et al., 1997; Pan et al., 1998a). We initially confirmed these observations before assessing the activation of other caspases in this context. Fig. 2 demonstrates that caspase-3 endogenous to Jurkat cell extracts was rapidly converted from the 36-kD proenzyme to the p17/p12 mature form in the presence of cytochrome c. Processing occurred in a two-step manner, with the initial appearance of a p24/p12 intermediate in the extracts, followed by accumulation of the mature p17/ p12 form of the enzyme (Fig. 2, A and C), reminiscent of the mechanism of activation of caspase-3 in response to granzyme B (Martin et al., 1996). This was further confirmed by addition of [35S]methionine-labeled caspase-3 to the extracts, which enabled detection of the caspase-3-p12 chain that was not recognized by the anti–caspase-3 polyclonal antibody used (Fig. 2 B). In direct contrast, conversion of caspase-1 (ICE) to its mature form was not detected in the same extracts over an identical time course (Fig. 2 A). To further confirm that processed caspases were active, we used synthetic tetrapeptide substrates that are preferentially cleaved by caspase-1–like (YVAD-pNA) or caspase-3–like (DEVD-pNA) proteases to assess the induction of caspase-1– or caspase-3–like proteolytic activity in response to cytochrome c. We observed a striking induction of DEVD-pNA cleaving activity within 15 min of addition of cytochrome c to the extracts, whereas YVAD-pNA cleaving activity did not rise above basal levels during the same time course (Fig. 2 D), in agreement with our observations on the absence of caspase-1 processing in this context (Fig. 2 A).

Cytochrome c induced apoptotic changes in nuclei added to the extracts at concentrations of as little as 1–5 μg/ml, but had no direct effects on nuclei in the absence of cell extract (Fig. 3 A). Interestingly, at cytochrome c concentrations where only partial caspase-3 activation was observed (5 μg/ml), substrates such as fodrin, PARP, and U1snRNP were almost completely cleaved (Fig. 3 B), suggesting that only a small amount of the total caspase-3 pool is required in order to effect complete proteolysis of these substrates. An alternative explanation is that cytochrome c activates other caspases in the extracts that are capable of cleaving these substrates. Cytochrome c failed to trigger caspase-1 processing at any of the concentrations tested (Fig. 3 B).

Cytochrome c Initiates a Cascade of Protease Activation Events Involving Caspases-2, -3, -6, -7, -8, -9, and -10Recent studies have shown that caspase-9 is activated in response to cytochrome c due to clustering of caspase-9 by Apaf-1 and that this results in activation of Caspases-3 (Li et al., 1997; Pan et al., 1998a; Srinivasula et al., 1998). To explore the full range of caspase activation events in the cytochrome c–initiated proteolytic cascade, we introduced [35S]methionine-labeled caspases-1, -2, -3, -4, -5, -6, -7, -8, -9, and -10 into Jurkat cell–free extracts and monitored processing of these proteases to their mature forms. Fig. 4 demonstrates that cytochrome c/dATP triggered maturation of caspases-2, -3, -6, -7, -8, -9, and -10, whereas none of the ICE subfamily proteases (caspases-1, -4, and -5) were processed under the same conditions. Significantly, cytochrome c failed to activate any of the caspases in the absence of cell extract, suggesting that a cytosolic factor such as Apaf-1 was required for all of these activation events. Caspases with long prodomains such as caspases-2, -8, and -10 are generally considered to be upstream or signaling caspases in the cell death pathway due to their ability to associate with cell surface death receptor molecules such as Fas/CD95 or TNFR1. Therefore, it was somewhat surprising that these caspases became activated in the presence of cytochrome c. However, it has been reported recently that thymocytes from APAF-1 null mice are impaired with respect to caspase-2 and caspase-8 activation in response to several proapoptotic stimuli (Yoshida et al., 1998). Similarly, dexamethasone-induced processing of caspases-2 and -8 was found to be impaired in mice deficient for caspase-9 (Hakem et al., 1998). These data suggest that these caspases are indeed activated in the Apaf-1 pathway in vivo. To explore the range of cytochrome c–inducible caspase activation events in more detail, we monitored the kinetics of activation of all caspases relative to each other in this system. Fig. 5 shows that detectable activation of most caspases, with the exceptions of caspases-8 and -10, appeared to occur contemporaneously, typically within 30 min of addition of cytochrome c to the extracts. In contrast, processing of caspases-8 and -10 were noticeably delayed relative to the other caspases, suggesting that these caspases might be activated late in this pathway.

Cytochrome c Initiates a Cascade of Protease Activation Events Involving Caspases-2, -3, -6, -7, -8, -9, and -10

Recent studies have shown that caspase-9 is activated in response to cytochrome c due to clustering of caspase-9 by Apaf-1 and that this results in activation of Caspases-3 (Li et al., 1997; Pan et al., 1998a; Srinivasula et al., 1998). To explore the full range of caspase activation events in the cytochrome c–initiated proteolytic cascade, we introduced [35S]methionine-labeled caspases-1, -2, -3, -4, -5, -6, -7, -8, -9, and -10 into Jurkat cell–free extracts and monitored processing of these proteases to their mature forms. Fig. 4 demonstrates that cytochrome c/dATP triggered maturation of caspases-2, -3, -6, -7, -8, -9, and -10, whereas none of the ICE subfamily proteases (caspases-1, -4, and -5) were processed under the same conditions. Significantly, cytochrome c failed to activate any of the caspases in the absence of cell extract, suggesting that a cytosolic factor such as Apaf-1 was required for all of these activation events. Caspases with long prodomains such as caspases-2, -8, and -10 are generally considered to be upstream or signaling caspases in the cell death pathway due to their ability to associate with cell surface death receptor molecules such as Fas/CD95 or TNFR1. Therefore, it was somewhat surprising that these caspases became activated in the presence of cytochrome c. However, it has been reported recently that thymocytes from APAF-1 null mice are impaired with respect to caspase-2 and caspase-8 activation in response to several proapoptotic stimuli (Yoshida et al., 1998). Similarly, dexamethasone-induced processing of caspases-2 and -8 was found to be impaired in mice deficient for caspase-9 (Hakem et al., 1998). These data suggest that these caspases are indeed activated in the Apaf-1 pathway in vivo.

To explore the range of cytochrome c–inducible caspase activation events in more detail, we monitored the kinetics of activation of all caspases relative to each other in this system. Fig. 5 shows that detectable activation of most caspases, with the exceptions of caspases-8 and -10, appeared to occur contemporaneously, typically within 30 min of addition of cytochrome c to the extracts. In contrast, processing of caspases-8 and -10 were noticeably delayed relative to the other caspases, suggesting that these caspases might be activated late in this pathway.

APAF-1 Selectively Binds to Caspase-9The observation that multiple caspases were activated in response to cytochrome c suggested either that all of these caspase activation events occurred downstream of caspase-9 (the only caspase known to directly associate with Apaf-1/ cytochrome c), or that a number of distinct caspase activation pathways could be instigated by cytochrome c, independent of caspase-9. To discriminate between these possibilities, we first explored whether Apaf-1 could directly bind caspases other than caspase-9. Fig. 6 demonstrates that a GST fusion protein comprising the CED-3 homology region of Apaf-1 (amino acid [aa] residues 1–97) selectively bound caspase-9 but did not bind any of the other caspases (-1, -2, -3, -6, -7, -8, and -10) tested. Similar results were obtained using a different GST fusion that spanned the CED-3 as well as the CED-4 homology regions of Apaf-1 (aa 1–412; Slee, E.A., and S.J. Martin, data not shown). These data demonstrate that Apaf-1 is highly selective for caspase-9, although they do not rule out the possibility that other caspases may become recruited to Apaf-1 via suitable adaptor molecules.

APAF-1 Selectively Binds to Caspase-9

The observation that multiple caspases were activated in response to cytochrome c suggested either that all of these caspase activation events occurred downstream of caspase-9 (the only caspase known to directly associate with Apaf-1/ cytochrome c), or that a number of distinct caspase activation pathways could be instigated by cytochrome c, independent of caspase-9. To discriminate between these possibilities, we first explored whether Apaf-1 could directly bind caspases other than caspase-9. Fig. 6 demonstrates that a GST fusion protein comprising the CED-3 homology region of Apaf-1 (amino acid [aa] residues 1–97) selectively bound caspase-9 but did not bind any of the other caspases (-1, -2, -3, -6, -7, -8, and -10) tested. Similar results were obtained using a different GST fusion that spanned the CED-3 as well as the CED-4 homology regions of Apaf-1 (aa 1–412; Slee, E.A., and S.J. Martin, data not shown). These data demonstrate that Apaf-1 is highly selective for caspase-9, although they do not rule out the possibility that other caspases may become recruited to Apaf-1 via suitable adaptor molecules.

Depletion of Caspase-9 from Jurkat Extracts Abrogates Processing of all Caspases in Response to Cytochrome cTo determine whether caspase-9 was required for activation of all other caspases in this context, we depleted this protease from Jurkat extracts before the addition of cytochrome c. Depletion of caspase-9 using Sepharose- immobilized GST-Apaf-11-97 (Fig. 7 A), or anti–caspase-9 polyclonal antibody (Fig. 7 B), rendered all caspases unresponsive to cytochrome c. In contrast, mock depletions performed using Sepharose-GST or control polyclonal antibody did not interfere with cytochrome c–induced activation of any of the caspases examined (Fig. 7, A and B). These data suggest that caspase-9 is critical for cytochrome c–initiated caspase activation events and cannot be substituted for by the other caspases present in the extracts. They also provide further support for the idea that Apaf-1 is selective for caspase-9 (Fig. 6) and fails to initiate the apoptotic program in its absence.

Depletion of Caspase-9 from Jurkat Extracts Abrogates Processing of all Caspases in Response to Cytochrome c

To determine whether caspase-9 was required for activation of all other caspases in this context, we depleted this protease from Jurkat extracts before the addition of cytochrome c. Depletion of caspase-9 using Sepharose- immobilized GST-Apaf-11-97 (Fig. 7 A), or anti–caspase-9 polyclonal antibody (Fig. 7 B), rendered all caspases unresponsive to cytochrome c. In contrast, mock depletions performed using Sepharose-GST or control polyclonal antibody did not interfere with cytochrome c–induced activation of any of the caspases examined (Fig. 7, A and B). These data suggest that caspase-9 is critical for cytochrome c–initiated caspase activation events and cannot be substituted for by the other caspases present in the extracts. They also provide further support for the idea that Apaf-1 is selective for caspase-9 (Fig. 6) and fails to initiate the apoptotic program in its absence.

Ordering the Cytochrome c–inducible Caspase Cascade Downstream of Caspase-9As a preliminary approach to ordering the sequence of caspase activation events triggered by cytochrome c, we investigated the effects of the tetrapeptide caspase inhibitors YVAD-CHO and DEVD-CHO, as well as the cowpox virus–derived caspase inhibitor CrmA, on the processing of all caspases downstream of caspase-9. The caspase-1–selective inhibitor YVAD-CHO had similar effects on all caspases, exhibiting little inhibition of caspase activation except at the highest concentration tested (Fig. 8). These data are consistent with YVAD-CHO directly inhibiting caspase-9 at high concentrations and terminating activation of all downstream caspases, in agreement with the results obtained by depletion of caspase-9 from the extracts (Fig. 7). Broadly similar effects on all caspases was also observed using GST-CrmA, with the exception that activation of caspases-8 and -10 were blocked at all concentrations of this inhibitor, whereas processing of all of the other caspases was seen at the lowest concentration tested (0.2 μM). This suggests that caspase-8 processing is downstream of the other caspases in the context of cytochrome c. Strikingly, very different inhibitory effects were observed using the caspase-3– and caspase-7–selective inhibitor DEVD-CHO. Whereas processing of caspases-2, -6, -8, and -10 was completely blocked at all concentrations of DEVD-CHO tested, very significant processing of caspases-3 and -7 was observed in the presence of 1–10 μM of this inhibitor. These data suggest that, after caspase-9, caspases-3 and -7 are the next caspases to become activated in the cytochrome c–initiated caspase cascade. Inhibition of caspase-3 and -7 activities by DEVD-CHO prevents further activation of any other caspases downstream of this point.Fig. 8 also demonstrates that maturation of caspase-3 in the presence of 1–10 μM DEVD-CHO was arrested at the p24/p12 intermediate. This confirms our earlier observation (Fig. 2, A–C) that cytochrome c–initiated caspase-3 processing was achieved via two distinct processing steps, with the first step involving a single cleavage of the proenzyme between the large and small subunits to produce a p24/p12 intermediate which then undergoes further processing via a distinct protease. The observation that caspase-3 maturation was completely arrested at the p24/ p12 intermediate in the presence of DEVD-CHO suggests that the second step (i.e., removal of the prodomain) is autocatalytic. A similar two-step processing mechanism has been reported for granzyme B–mediated activation of caspase-3 (Fernandes-Alnemri et al., 1996; Martin et al., 1996).

Ordering the Cytochrome c–inducible Caspase Cascade Downstream of Caspase-9

As a preliminary approach to ordering the sequence of caspase activation events triggered by cytochrome c, we investigated the effects of the tetrapeptide caspase inhibitors YVAD-CHO and DEVD-CHO, as well as the cowpox virus–derived caspase inhibitor CrmA, on the processing of all caspases downstream of caspase-9. The caspase-1–selective inhibitor YVAD-CHO had similar effects on all caspases, exhibiting little inhibition of caspase activation except at the highest concentration tested (Fig. 8). These data are consistent with YVAD-CHO directly inhibiting caspase-9 at high concentrations and terminating activation of all downstream caspases, in agreement with the results obtained by depletion of caspase-9 from the extracts (Fig. 7). Broadly similar effects on all caspases was also observed using GST-CrmA, with the exception that activation of caspases-8 and -10 were blocked at all concentrations of this inhibitor, whereas processing of all of the other caspases was seen at the lowest concentration tested (0.2 μM). This suggests that caspase-8 processing is downstream of the other caspases in the context of cytochrome c.

Strikingly, very different inhibitory effects were observed using the caspase-3– and caspase-7–selective inhibitor DEVD-CHO. Whereas processing of caspases-2, -6, -8, and -10 was completely blocked at all concentrations of DEVD-CHO tested, very significant processing of caspases-3 and -7 was observed in the presence of 1–10 μM of this inhibitor. These data suggest that, after caspase-9, caspases-3 and -7 are the next caspases to become activated in the cytochrome c–initiated caspase cascade. Inhibition of caspase-3 and -7 activities by DEVD-CHO prevents further activation of any other caspases downstream of this point.

Fig. 8 also demonstrates that maturation of caspase-3 in the presence of 1–10 μM DEVD-CHO was arrested at the p24/p12 intermediate. This confirms our earlier observation (Fig. 2, A–C) that cytochrome c–initiated caspase-3 processing was achieved via two distinct processing steps, with the first step involving a single cleavage of the proenzyme between the large and small subunits to produce a p24/p12 intermediate which then undergoes further processing via a distinct protease. The observation that caspase-3 maturation was completely arrested at the p24/ p12 intermediate in the presence of DEVD-CHO suggests that the second step (i.e., removal of the prodomain) is autocatalytic. A similar two-step processing mechanism has been reported for granzyme B–mediated activation of caspase-3 (Fernandes-Alnemri et al., 1996; Martin et al., 1996).

Depletion of Caspase-3 from Cell Extracts Ablates Cytochrome c–induced Processing of Caspases-2, -6, -8, and -10 and Reveals a Feedback Loop Involving Caspase-9To confirm that caspase-3 was activated upstream of caspases-2, -6, -8, and -10 in response to cytochrome c and to ask whether caspase-3 was required for processing of any of the other caspases in this context, we immunodepleted caspase-3 from cell extracts (Fig. 9). Caspase-3–depleted extracts were compared with mock-depleted extracts for their ability to support cytochrome c–induced processing of caspases-2, -3, -6, -7, -8, -9, and -10. Strikingly, removal of caspase-3 from the extracts abrogated processing of caspases-2, -6, -8, and -10 but had only a marginal effect on the processing of caspase-7 in the presence of cytochrome c (Fig. 9 A). These data suggest that caspases-3 and -7 are directly processed downstream of caspase-9 in the cytochrome c/Apaf-1–inducible caspase cascade and that caspase-3 is necessary for the activation (either directly or indirectly) of caspases-2, -6, -8, and -10. Interestingly, although caspase-3 was activated downstream of caspase-9 in response to cytochrome c as expected (Fig. 7), partial inhibition of caspase-9 processing was observed in caspase-3–depleted, but not in mock- depleted, extracts (Fig. 9 B). Consistent with this, reintroduced 35S-labeled caspase-3 was processed less efficiently in caspase-3–depleted extracts as compared with mock- depleted extracts, suggesting that caspase-9 activation was less efficient in these extracts (Fig. 9 A). In normal or mock-depleted extracts, the processed form of caspase-9 could be resolved into two major bands migrating at ∼37 and ∼35 kD, as recently reported (Srinivasula et al., 1998). However, in caspase-3–depleted extracts the 37-kD cleavage product was not produced (Fig. 9 B), suggesting that caspase-9 activation in response to cytochrome c is partially achieved via a feedback loop involving caspase-3. To confirm this, we compared Jurkat postnuclear extracts with similar extracts prepared from MCF-7 cells which are devoid of caspase-3 due to a deletion in exon 3 of the CASP-3 gene (Jänicke et al., 1998). Using MCF-7 cell extracts we confirmed that, in the absence of caspase-3, caspase-9 is processed to a single 35-kD cleavage product, whereas both the 35- and 37-kD cleavage products were produced in Jurkat extracts (Fig. 9 B). These data are consistent with the interpretation that cytochrome c/Apaf-1–triggered processing of caspase-9 is initially autocatalytic, producing the p35 form via cleavage at Asp-315, but upon activation of caspase-3 is also achieved via a feedback loop in which caspase-3 processes caspase-9 at Asp-330 (Fig. 9 C; Srinivasula et al., 1998).

Depletion of Caspase-3 from Cell Extracts Ablates Cytochrome c–induced Processing of Caspases-2, -6, -8, and -10 and Reveals a Feedback Loop Involving Caspase-9

To confirm that caspase-3 was activated upstream of caspases-2, -6, -8, and -10 in response to cytochrome c and to ask whether caspase-3 was required for processing of any of the other caspases in this context, we immunodepleted caspase-3 from cell extracts (Fig. 9). Caspase-3–depleted extracts were compared with mock-depleted extracts for their ability to support cytochrome c–induced processing of caspases-2, -3, -6, -7, -8, -9, and -10. Strikingly, removal of caspase-3 from the extracts abrogated processing of caspases-2, -6, -8, and -10 but had only a marginal effect on the processing of caspase-7 in the presence of cytochrome c (Fig. 9 A). These data suggest that caspases-3 and -7 are directly processed downstream of caspase-9 in the cytochrome c/Apaf-1–inducible caspase cascade and that caspase-3 is necessary for the activation (either directly or indirectly) of caspases-2, -6, -8, and -10.

Interestingly, although caspase-3 was activated downstream of caspase-9 in response to cytochrome c as expected (Fig. 7), partial inhibition of caspase-9 processing was observed in caspase-3–depleted, but not in mock- depleted, extracts (Fig. 9 B). Consistent with this, reintroduced 35S-labeled caspase-3 was processed less efficiently in caspase-3–depleted extracts as compared with mock- depleted extracts, suggesting that caspase-9 activation was less efficient in these extracts (Fig. 9 A). In normal or mock-depleted extracts, the processed form of caspase-9 could be resolved into two major bands migrating at ∼37 and ∼35 kD, as recently reported (Srinivasula et al., 1998). However, in caspase-3–depleted extracts the 37-kD cleavage product was not produced (Fig. 9 B), suggesting that caspase-9 activation in response to cytochrome c is partially achieved via a feedback loop involving caspase-3. To confirm this, we compared Jurkat postnuclear extracts with similar extracts prepared from MCF-7 cells which are devoid of caspase-3 due to a deletion in exon 3 of the CASP-3 gene (Jänicke et al., 1998). Using MCF-7 cell extracts we confirmed that, in the absence of caspase-3, caspase-9 is processed to a single 35-kD cleavage product, whereas both the 35- and 37-kD cleavage products were produced in Jurkat extracts (Fig. 9 B). These data are consistent with the interpretation that cytochrome c/Apaf-1–triggered processing of caspase-9 is initially autocatalytic, producing the p35 form via cleavage at Asp-315, but upon activation of caspase-3 is also achieved via a feedback loop in which caspase-3 processes caspase-9 at Asp-330 (Fig. 9 C; Srinivasula et al., 1998).

Caspase-6 Is Required for Cytochrome c–induced Processing of Caspases-8 and -10We next depleted caspase-6 or caspase-7 from Jurkat extracts to ask whether either of these caspases was required for any of the other caspase activation events seen in the presence of cytochrome c (Fig. 10). Immunodepletion of caspase-6 failed to have any effect on the processing of caspases-3, -7, and -9 in response to cytochrome c, consistent with caspase-6 activation being downstream of the latter caspases (Fig. 10 A). Caspase-2 processing was also unaffected in the absence of caspase-6, suggesting that caspase-2 is directly processed, along with caspase-6, upon activation of caspase-3 in the extracts. However, processing of caspases-8 and -10 was largely abrogated in extracts devoid of caspase-6 (Fig. 10 A). These data suggest that, upon activation by caspase-3, caspase-6 in turn promotes the processing of caspases-8 and -10 further down the cascade. In contrast to the effects seen after depletion of caspases-9, -3, or -6, immunodepletion of caspase-7 from the extracts failed to have any detectable inhibitory effects on any of the caspase processing events observed in the presence of cytochrome c (Fig. 10 B). Taken together, these data suggest an order of caspase activation events that take place distal to entry of cytochrome c into the cytosol (Fig. 11).

Caspase-6 Is Required for Cytochrome c–induced Processing of Caspases-8 and -10

We next depleted caspase-6 or caspase-7 from Jurkat extracts to ask whether either of these caspases was required for any of the other caspase activation events seen in the presence of cytochrome c (Fig. 10). Immunodepletion of caspase-6 failed to have any effect on the processing of caspases-3, -7, and -9 in response to cytochrome c, consistent with caspase-6 activation being downstream of the latter caspases (Fig. 10 A). Caspase-2 processing was also unaffected in the absence of caspase-6, suggesting that caspase-2 is directly processed, along with caspase-6, upon activation of caspase-3 in the extracts. However, processing of caspases-8 and -10 was largely abrogated in extracts devoid of caspase-6 (Fig. 10 A). These data suggest that, upon activation by caspase-3, caspase-6 in turn promotes the processing of caspases-8 and -10 further down the cascade.

In contrast to the effects seen after depletion of caspases-9, -3, or -6, immunodepletion of caspase-7 from the extracts failed to have any detectable inhibitory effects on any of the caspase processing events observed in the presence of cytochrome c (Fig. 10 B). Taken together, these data suggest an order of caspase activation events that take place distal to entry of cytochrome c into the cytosol (Fig. 11).

DiscussionIn this study, we have shown that cytochrome c can initiate a complex series of caspase activation events, ultimately resulting in apoptotic changes (nuclear condensation and fragmentation, degradation of several caspase substrates, DNA fragmentation) in cell extracts. Surprisingly, cytochrome c initiated activation of several of the so-called signaling (caspases-2, -8, -9, and -10) as well as effector (-3, -6, -7) caspases. In contrast, no detectable processing of the ICE subfamily caspases (-1, -4, -5) was observed in this context. All of these events were abrogated by removal of caspase-9 from the extracts, confirming that this protease is indispensable for cytochrome c–initiated triggering of the death program and occupies an apical point in the caspase cascade. In line with these observations, the CARD domain of Apaf-1 was shown to bind selectively to caspase-9. Inhibitory profiles generated with the caspase inhibitor DEVD-CHO suggested that caspases-3 and -7 were activated downstream of caspase-9 and that these caspases then went on to propagate the caspase cascade by activating caspases-2, -6, -8, and -10. This interpretation was confirmed and extended by immunodepleting caspases-3, -6, or -7 from the extracts and assessing the impact of their removal on the other caspase activation events. Interestingly, removal of caspase-3 revealed that this caspase was required for four other caspase activation events and also revealed a feedback loop in this pathway involving caspase-9.In this study we have assessed caspase activation events in most cases (with the exception of caspases-1, -3, and -9) by adding 35S-labeled in vitro transcribed and translated caspases to the cell extracts. Clearly, this raises the issue of whether caspases endogenous to the extracts behave in the same way as their exogenously added counterparts. Where antibodies were available to us (caspases-2, -6, -7, and -8), we confirmed that the order and kinetics of endogenous caspase processing was essentially identical to that observed using exogenously added caspases (Slee, E.A., and S.J. Martin, data not shown). However, the use of radiolabeled caspases enabled us to track complete processing of each caspase whereas many of the available anticaspase antibodies recognized processed forms inefficiently or not at all.Although the initial report that cytochrome c could trigger caspase-3 processing was surprising (Liu et al., 1996), much evidence has accumulated to suggest that release of cytochrome c from mitochondria is an important control point in apoptosis (reviewed by Reed, 1997). It is still unclear exactly how cytochrome c release is achieved, although recent reports suggest that death-promoting members of the Bcl-2 family such as Bax or Bid may play a role in this, possibly due to their ability to form ion or small protein channels (Jurgensmeier et al., 1998; Li et al., 1998; Luo et al., 1998). Although opening of a permeability transition pore was proposed previously as one possible means of enabling cytochrome c escape to the cytosol (Kroemer et al., 1997), cytochrome c release has been observed in situations where either no loss in mitochondrial transmembrane potential was observed or where changes in transmembrane potential occurred after cytochrome c efflux (Kluck et al., 1997a; Yang et al., 1997; Bossy-Wetzel et al., 1998).Irrespective of the exact mechanism of release, much evidence now exists to suggest that cytochrome c plays an important role as an initiator of the death machinery in cases where cellular damage is general (i.e., radiation, heat shock, cytotoxic drugs), or as an amplifier of death signals in cases where caspase activation is initiated by a membrane receptor such as Fas (Kuwana et al., 1998; Li et al., 1998; Luo et al., 1998; Scaffidi et al., 1998). Perhaps the most compelling argument for a central role for cytochrome c in apoptosis is the finding of Wang and colleagues that cytochrome c binds to and activates Apaf-1, the first human CED-4 homologue to be discovered (Zou et al., 1997). Gene targeting experiments in mice have revealed that Apaf-1 plays a critical role in developmental-related cell death in the brain, as well as in cytotoxic drug-induced cell death in other cell lineages (Cecconi et al., 1998; Yoshida et al., 1998). At present, cytochrome c is the only known activator of Apaf-1, although it is possible that other pathways may harness the caspase-activating properties of this molecule.Although numerous studies have appeared documenting the activation of individual caspases in the context of many different death-promoting stimuli, it is still unclear whether caspases are activated sequentially or in parallel in many of these contexts. Here, we provide evidence for a stepwise series of caspase activation events occurring in response to cytochrome c. Caspase-9 appears to be the first caspase to become activated in this context, almost certainly due to clustering of this protease via Apaf-1. Clustering of caspase-9 results in partial activation of this protease in an autocatalytic manner (Pan et al., 1998a; Srinivasula et al., 1998). Caspase-9 then initiates processing of caspase-3 as well as caspase-7. The activation of caspase-3 in this context appears to occur in a partly autocatalytic manner since the caspase-3 inhibitor, DEVD-CHO, arrested maturation of caspase-3 at an incompletely processed intermediate stage. The pattern of caspase-3 breakdown suggests that caspase-9 attacks this molecule between the large and small subunits and that caspase-3 subsequently removes its own prodomain by autocatalysis. Activated caspase-3 in turn activates caspases-2 and -6 and also appears to be capable of acting in a feedback loop on caspase-9 to ensure complete activation of the latter. Somewhat surprisingly, caspase-6 was found to be required for the activation of caspases-8 and -10 in this context (Fig. 10).Clearly, further work is necessary to determine whether the sequence of cytochrome c–inducible caspase activation events that take place in cell extracts also takes place in intact cells. However, recent gene targeting studies provide support for our model (Hakem et al., 1998; Yoshida et al., 1998). CASP-9 null embryonic stem (ES) cells and embryonic fibroblasts were found to be resistant to multiple proapoptotic stimuli, but not to cytotoxic T lymphocyte or TNF-mediated killing, arguing that caspase-9 is required for forms of apoptosis that are thought to be routed along the mitochondrial pathway (Hakem et al., 1998). As further evidence of this, CASP-9
−/− ES cells were found to be resistant to UV-induced death, although cytochrome c release still took place (Hakem et al., 1998). Moreover, UV-induced caspase-3 and caspase-8 processing was impaired in CASP-9
−/− ES cells as was dexamethasone-induced processing of caspases-2, -7, and -8 in CASP-9
−/− thymocytes (Hakem et al., 1998), supporting our observations that these caspases are activated downstream of caspase-9 in the cytochrome c pathway. Similar observations have also been made with respect to etoposide-induced caspase-2 and caspase-8 activation in thymocytes from APAF1
−/− mice (Yoshida et al., 1998). Once again, these observations lend support to our observations that caspases-2 and -8 are activated downstream of cytochrome c/Apaf-1.We have confirmed the observation of Wang and colleagues (Li et al., 1997) that Apaf-1 directly binds to caspase-9 and have extended this observation to show that caspases-1, -2, -3, -6, -7, -8, and -10 do not bind to this molecule. These observations are at odds with recent findings that suggest that Apaf-1 can also form complexes with caspases-4 and -8 (Hu et al., 1998). However, in the latter study no direct interaction between these caspases was demonstrated since coimmunoprecipitation from cell lysates was the criteria used to determine interaction. Thus, binding could have been mediated by an adaptor protein. Using a Gal4-based yeast two-hybrid system we have confirmed the interaction between the CED-3-homologous region of Apaf-1 and caspase-9 but again failed to detect direct binding of Apaf-1 to caspase-8 (Harte, M.T., C. Adrain, and S.J. Martin, unpublished data). In addition, the observation that removal of caspase-9 from cell extracts abolished all caspase activating activity of cytochrome c suggests that this caspase is indispensable for this pathway, irrespective of the ability of Apaf-1 to complex with other caspases.Although it is generally believed that multiple caspases participate in the signaling and destruction phases of apoptosis, it is still unclear whether there is significant functional redundancy within this family of proteases. The observations that caspase-3, caspase-8, and caspase-9 knockout mice die in utero or soon after birth would argue against redundancy, at least in certain tissues (Kuida et al., 1996, 1998; Hakem et al., 1998; Varfolomeev et al., 1998). In addition, in vitro studies that have used dominant negative forms of caspase-9, as well as data available from CASP-9 null mice, suggest that this caspase occupies a critical position in a major subset of cell death pathways since apoptosis was abrogated in the absence of this caspase in a number of contexts (Hakem et al., 1998; Kuida et al., 1998; Pan et al., 1998b; Srinivasula et al., 1998).It is also commonly believed that caspases with long prodomains are upstream or signaling caspases whereas those with short prodomains are effector or executioner caspases. For example, studies on caspase-8 suggest that this protease is the most proximal caspase to become activated upon ligation of the CD95 (Fas/Apo-1) molecule since this caspase is directly recruited into the CD95 signaling complex upon receptor aggregation (Kischkel et al., 1995; Boldin et al., 1996; Fernandes-Alnemri et al., 1996; Muzio et al., 1996). However, recent studies have suggested that caspase-8 is not always activated early in the context of CD95 signaling (Scaffidi et al., 1998). This has led to the suggestion that two distinct cellular types exist with respect to CD95 signaling: type I, cells that activate caspase-8 early (within seconds) of CD95 receptor aggregation and type II, cells that activate caspase-8 late and in a mitochondrial-dependent fashion (Scaffidi et al., 1998).In this study we also observed caspase-8 activation late in the cytochrome c–inducible caspase cascade. It is possible that caspase-8 activation is merely a bystander event in this model of apoptosis, since the caspase substrates examined (fodrin, PARP, U1snRNP) were almost completely cleaved within 60 min of addition of cytochrome c to the extracts (Fig. 1 B) and nuclear destruction was largely complete by 90 min (Fig. 1 A). In contrast, only a small portion of the [35S]methionine-labeled caspase-8 that was added to the extracts had become processed by 60 min under similar conditions (Fig. 5). However, as previously discussed, caspase-8 activation was also found to be impaired in cells from CASP-9
−/− as well as APAF-1
−/− mice in certain contexts, suggesting that this caspase may be activated downstream in some situations (Hakem et al., 1998; Yoshida et al., 1998).In summary, our observations suggests that a branched cascade of caspase activation events, with at least one feedback loop, is initiated distal to entry of cytochrome c into the cytosol. Further studies are required to establish whether the sequence of caspase activation events we report is conserved between different cell types and in response to divergent death-promoting stimuli.

In this study, we have shown that cytochrome c can initiate a complex series of caspase activation events, ultimately resulting in apoptotic changes (nuclear condensation and fragmentation, degradation of several caspase substrates, DNA fragmentation) in cell extracts. Surprisingly, cytochrome c initiated activation of several of the so-called signaling (caspases-2, -8, -9, and -10) as well as effector (-3, -6, -7) caspases. In contrast, no detectable processing of the ICE subfamily caspases (-1, -4, -5) was observed in this context. All of these events were abrogated by removal of caspase-9 from the extracts, confirming that this protease is indispensable for cytochrome c–initiated triggering of the death program and occupies an apical point in the caspase cascade. In line with these observations, the CARD domain of Apaf-1 was shown to bind selectively to caspase-9. Inhibitory profiles generated with the caspase inhibitor DEVD-CHO suggested that caspases-3 and -7 were activated downstream of caspase-9 and that these caspases then went on to propagate the caspase cascade by activating caspases-2, -6, -8, and -10. This interpretation was confirmed and extended by immunodepleting caspases-3, -6, or -7 from the extracts and assessing the impact of their removal on the other caspase activation events. Interestingly, removal of caspase-3 revealed that this caspase was required for four other caspase activation events and also revealed a feedback loop in this pathway involving caspase-9.

In this study we have assessed caspase activation events in most cases (with the exception of caspases-1, -3, and -9) by adding 35S-labeled in vitro transcribed and translated caspases to the cell extracts. Clearly, this raises the issue of whether caspases endogenous to the extracts behave in the same way as their exogenously added counterparts. Where antibodies were available to us (caspases-2, -6, -7, and -8), we confirmed that the order and kinetics of endogenous caspase processing was essentially identical to that observed using exogenously added caspases (Slee, E.A., and S.J. Martin, data not shown). However, the use of radiolabeled caspases enabled us to track complete processing of each caspase whereas many of the available anticaspase antibodies recognized processed forms inefficiently or not at all.

Although the initial report that cytochrome c could trigger caspase-3 processing was surprising (Liu et al., 1996), much evidence has accumulated to suggest that release of cytochrome c from mitochondria is an important control point in apoptosis (reviewed by Reed, 1997). It is still unclear exactly how cytochrome c release is achieved, although recent reports suggest that death-promoting members of the Bcl-2 family such as Bax or Bid may play a role in this, possibly due to their ability to form ion or small protein channels (Jurgensmeier et al., 1998; Li et al., 1998; Luo et al., 1998). Although opening of a permeability transition pore was proposed previously as one possible means of enabling cytochrome c escape to the cytosol (Kroemer et al., 1997), cytochrome c release has been observed in situations where either no loss in mitochondrial transmembrane potential was observed or where changes in transmembrane potential occurred after cytochrome c efflux (Kluck et al., 1997a; Yang et al., 1997; Bossy-Wetzel et al., 1998).

Irrespective of the exact mechanism of release, much evidence now exists to suggest that cytochrome c plays an important role as an initiator of the death machinery in cases where cellular damage is general (i.e., radiation, heat shock, cytotoxic drugs), or as an amplifier of death signals in cases where caspase activation is initiated by a membrane receptor such as Fas (Kuwana et al., 1998; Li et al., 1998; Luo et al., 1998; Scaffidi et al., 1998). Perhaps the most compelling argument for a central role for cytochrome c in apoptosis is the finding of Wang and colleagues that cytochrome c binds to and activates Apaf-1, the first human CED-4 homologue to be discovered (Zou et al., 1997). Gene targeting experiments in mice have revealed that Apaf-1 plays a critical role in developmental-related cell death in the brain, as well as in cytotoxic drug-induced cell death in other cell lineages (Cecconi et al., 1998; Yoshida et al., 1998). At present, cytochrome c is the only known activator of Apaf-1, although it is possible that other pathways may harness the caspase-activating properties of this molecule.

Although numerous studies have appeared documenting the activation of individual caspases in the context of many different death-promoting stimuli, it is still unclear whether caspases are activated sequentially or in parallel in many of these contexts. Here, we provide evidence for a stepwise series of caspase activation events occurring in response to cytochrome c. Caspase-9 appears to be the first caspase to become activated in this context, almost certainly due to clustering of this protease via Apaf-1. Clustering of caspase-9 results in partial activation of this protease in an autocatalytic manner (Pan et al., 1998a; Srinivasula et al., 1998). Caspase-9 then initiates processing of caspase-3 as well as caspase-7. The activation of caspase-3 in this context appears to occur in a partly autocatalytic manner since the caspase-3 inhibitor, DEVD-CHO, arrested maturation of caspase-3 at an incompletely processed intermediate stage. The pattern of caspase-3 breakdown suggests that caspase-9 attacks this molecule between the large and small subunits and that caspase-3 subsequently removes its own prodomain by autocatalysis. Activated caspase-3 in turn activates caspases-2 and -6 and also appears to be capable of acting in a feedback loop on caspase-9 to ensure complete activation of the latter. Somewhat surprisingly, caspase-6 was found to be required for the activation of caspases-8 and -10 in this context (Fig. 10).

Clearly, further work is necessary to determine whether the sequence of cytochrome c–inducible caspase activation events that take place in cell extracts also takes place in intact cells. However, recent gene targeting studies provide support for our model (Hakem et al., 1998; Yoshida et al., 1998). CASP-9 null embryonic stem (ES) cells and embryonic fibroblasts were found to be resistant to multiple proapoptotic stimuli, but not to cytotoxic T lymphocyte or TNF-mediated killing, arguing that caspase-9 is required for forms of apoptosis that are thought to be routed along the mitochondrial pathway (Hakem et al., 1998). As further evidence of this, CASP-9
−/− ES cells were found to be resistant to UV-induced death, although cytochrome c release still took place (Hakem et al., 1998). Moreover, UV-induced caspase-3 and caspase-8 processing was impaired in CASP-9
−/− ES cells as was dexamethasone-induced processing of caspases-2, -7, and -8 in CASP-9
−/− thymocytes (Hakem et al., 1998), supporting our observations that these caspases are activated downstream of caspase-9 in the cytochrome c pathway. Similar observations have also been made with respect to etoposide-induced caspase-2 and caspase-8 activation in thymocytes from APAF1
−/− mice (Yoshida et al., 1998). Once again, these observations lend support to our observations that caspases-2 and -8 are activated downstream of cytochrome c/Apaf-1.

We have confirmed the observation of Wang and colleagues (Li et al., 1997) that Apaf-1 directly binds to caspase-9 and have extended this observation to show that caspases-1, -2, -3, -6, -7, -8, and -10 do not bind to this molecule. These observations are at odds with recent findings that suggest that Apaf-1 can also form complexes with caspases-4 and -8 (Hu et al., 1998). However, in the latter study no direct interaction between these caspases was demonstrated since coimmunoprecipitation from cell lysates was the criteria used to determine interaction. Thus, binding could have been mediated by an adaptor protein. Using a Gal4-based yeast two-hybrid system we have confirmed the interaction between the CED-3-homologous region of Apaf-1 and caspase-9 but again failed to detect direct binding of Apaf-1 to caspase-8 (Harte, M.T., C. Adrain, and S.J. Martin, unpublished data). In addition, the observation that removal of caspase-9 from cell extracts abolished all caspase activating activity of cytochrome c suggests that this caspase is indispensable for this pathway, irrespective of the ability of Apaf-1 to complex with other caspases.

Although it is generally believed that multiple caspases participate in the signaling and destruction phases of apoptosis, it is still unclear whether there is significant functional redundancy within this family of proteases. The observations that caspase-3, caspase-8, and caspase-9 knockout mice die in utero or soon after birth would argue against redundancy, at least in certain tissues (Kuida et al., 1996, 1998; Hakem et al., 1998; Varfolomeev et al., 1998). In addition, in vitro studies that have used dominant negative forms of caspase-9, as well as data available from CASP-9 null mice, suggest that this caspase occupies a critical position in a major subset of cell death pathways since apoptosis was abrogated in the absence of this caspase in a number of contexts (Hakem et al., 1998; Kuida et al., 1998; Pan et al., 1998b; Srinivasula et al., 1998).

It is also commonly believed that caspases with long prodomains are upstream or signaling caspases whereas those with short prodomains are effector or executioner caspases. For example, studies on caspase-8 suggest that this protease is the most proximal caspase to become activated upon ligation of the CD95 (Fas/Apo-1) molecule since this caspase is directly recruited into the CD95 signaling complex upon receptor aggregation (Kischkel et al., 1995; Boldin et al., 1996; Fernandes-Alnemri et al., 1996; Muzio et al., 1996). However, recent studies have suggested that caspase-8 is not always activated early in the context of CD95 signaling (Scaffidi et al., 1998). This has led to the suggestion that two distinct cellular types exist with respect to CD95 signaling: type I, cells that activate caspase-8 early (within seconds) of CD95 receptor aggregation and type II, cells that activate caspase-8 late and in a mitochondrial-dependent fashion (Scaffidi et al., 1998).

In this study we also observed caspase-8 activation late in the cytochrome c–inducible caspase cascade. It is possible that caspase-8 activation is merely a bystander event in this model of apoptosis, since the caspase substrates examined (fodrin, PARP, U1snRNP) were almost completely cleaved within 60 min of addition of cytochrome c to the extracts (Fig. 1 B) and nuclear destruction was largely complete by 90 min (Fig. 1 A). In contrast, only a small portion of the [35S]methionine-labeled caspase-8 that was added to the extracts had become processed by 60 min under similar conditions (Fig. 5). However, as previously discussed, caspase-8 activation was also found to be impaired in cells from CASP-9
−/− as well as APAF-1
−/− mice in certain contexts, suggesting that this caspase may be activated downstream in some situations (Hakem et al., 1998; Yoshida et al., 1998).

In summary, our observations suggests that a branched cascade of caspase activation events, with at least one feedback loop, is initiated distal to entry of cytochrome c into the cytosol. Further studies are required to establish whether the sequence of caspase activation events we report is conserved between different cell types and in response to divergent death-promoting stimuli.
