---
pmid: '26871637'
title: Widespread Expansion of Protein Interaction Capabilities by Alternative Splicing.
authors:
- Yang X
- Coulombe-Huntington J
- Kang S
- Sheynkman GM
- Hao T
- Richardson A
- Sun S
- Yang F
- Shen YA
- Murray RR
- Spirohn K
- Begg BE
- Duran-Frigola M
- MacWilliams A
- Pevzner SJ
- Zhong Q
- Wanamaker SA
- Tam S
- Ghamsari L
- Sahni N
- Yi S
- Rodriguez MD
- Balcha D
- Tan G
- Costanzo M
- Andrews B
- Boone C
- Zhou XJ
- Salehi-Ashtiani K
- Charloteaux B
- Chen AA
- Calderwood MA
- Aloy P
- Roth FP
- Hill DE
- Iakoucheva LM
- Xia Y
- Vidal M
journal: Cell
year: '2016'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4882190
doi: 10.1016/j.cell.2016.01.029
---

# Widespread Expansion of Protein Interaction Capabilities by Alternative Splicing.
**Authors:** Yang X, Coulombe-Huntington J, Kang S, Sheynkman GM, Hao T, Richardson A, Sun S, Yang F, Shen YA, Murray RR, Spirohn K, Begg BE, Duran-Frigola M, MacWilliams A, Pevzner SJ, Zhong Q, Wanamaker SA, Tam S, Ghamsari L, Sahni N, Yi S, Rodriguez MD, Balcha D, Tan G, Costanzo M, Andrews B, Boone C, Zhou XJ, Salehi-Ashtiani K, Charloteaux B, Chen AA, Calderwood MA, Aloy P, Roth FP, Hill DE, Iakoucheva LM, Xia Y, Vidal M
**Journal:** Cell (2016)
**DOI:** [10.1016/j.cell.2016.01.029](https://doi.org/10.1016/j.cell.2016.01.029)
**PMC:** [PMC4882190](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4882190/)

## Abstract

1. Cell. 2016 Feb 11;164(4):805-17. doi: 10.1016/j.cell.2016.01.029.

Widespread Expansion of Protein Interaction Capabilities by Alternative 
Splicing.

Yang X(1), Coulombe-Huntington J(2), Kang S(3), Sheynkman GM(4), Hao T(4), 
Richardson A(4), Sun S(5), Yang F(6), Shen YA(4), Murray RR(7), Spirohn K(4), 
Begg BE(4), Duran-Frigola M(8), MacWilliams A(7), Pevzner SJ(9), Zhong Q(7), 
Wanamaker SA(7), Tam S(7), Ghamsari L(7), Sahni N(4), Yi S(4), Rodriguez MD(7), 
Balcha D(4), Tan G(10), Costanzo M(10), Andrews B(11), Boone C(11), Zhou XJ(12), 
Salehi-Ashtiani K(7), Charloteaux B(4), Chen AA(4), Calderwood MA(4), Aloy 
P(13), Roth FP(14), Hill DE(4), Iakoucheva LM(15), Xia Y(16), Vidal M(17).

Author information:
(1)Genomic Analysis of Network Perturbations Center of Excellence in Genomic 
Science (CEGS), Dana-Farber Cancer Institute, Boston, MA 02215, USA; Center for 
Cancer Systems Biology (CCSB) and Department of Cancer Biology, Dana-Farber 
Cancer Institute, Boston, MA 02215, USA; Department of Genetics, Harvard Medical 
School, Boston, MA 02115, USA; Department of Obstetrics and Gynecology, Nanfang 
Hospital, Southern Medical University, Guangzhou 510515, China.
(2)Department of Bioengineering, McGill University, Montreal, QC H3A 0C3, 
Canada.
(3)Department of Psychiatry, University of California, San Diego, La Jolla, CA 
92093, USA.
(4)Genomic Analysis of Network Perturbations Center of Excellence in Genomic 
Science (CEGS), Dana-Farber Cancer Institute, Boston, MA 02215, USA; Center for 
Cancer Systems Biology (CCSB) and Department of Cancer Biology, Dana-Farber 
Cancer Institute, Boston, MA 02215, USA; Department of Genetics, Harvard Medical 
School, Boston, MA 02115, USA.
(5)Donnelly Centre, University of Toronto, Toronto, ON M5S 3E1, Canada; 
Department of Molecular Genetics, University of Toronto, Toronto, ON M5S 3E1, 
Canada; Lunenfeld-Tanenbaum Research Institute, Mt. Sinai Hospital, Toronto, ON 
M5G 1X5, Canada; Department of Medical Biochemistry and Microbiology, Uppsala 
University, SE-75123 Uppsala, Sweden.
(6)Donnelly Centre, University of Toronto, Toronto, ON M5S 3E1, Canada; 
Department of Molecular Genetics, University of Toronto, Toronto, ON M5S 3E1, 
Canada; Lunenfeld-Tanenbaum Research Institute, Mt. Sinai Hospital, Toronto, ON 
M5G 1X5, Canada.
(7)Center for Cancer Systems Biology (CCSB) and Department of Cancer Biology, 
Dana-Farber Cancer Institute, Boston, MA 02215, USA; Department of Genetics, 
Harvard Medical School, Boston, MA 02115, USA.
(8)Institute for Research in Biomedicine (IRB Barcelona), The Barcelona 
Institute of Science and Technology, Barcelona 08028, Catalonia, Spain.
(9)Center for Cancer Systems Biology (CCSB) and Department of Cancer Biology, 
Dana-Farber Cancer Institute, Boston, MA 02215, USA; Department of Genetics, 
Harvard Medical School, Boston, MA 02115, USA; Department of Biomedical 
Engineering, Boston University, Boston, MA 02215, USA; Boston University School 
of Medicine, Boston, MA 02118, USA.
(10)Donnelly Centre, University of Toronto, Toronto, ON M5S 3E1, Canada.
(11)Donnelly Centre, University of Toronto, Toronto, ON M5S 3E1, Canada; 
Department of Molecular Genetics, University of Toronto, Toronto, ON M5S 3E1, 
Canada.
(12)Molecular and Computational Biology Program, Department of Biological 
Sciences, University of Southern California, Los Angeles, CA 90089, USA.
(13)Institute for Research in Biomedicine (IRB Barcelona), The Barcelona 
Institute of Science and Technology, Barcelona 08028, Catalonia, Spain; 
Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona 08010, 
Catalonia, Spain.
(14)Genomic Analysis of Network Perturbations Center of Excellence in Genomic 
Science (CEGS), Dana-Farber Cancer Institute, Boston, MA 02215, USA; Center for 
Cancer Systems Biology (CCSB) and Department of Cancer Biology, Dana-Farber 
Cancer Institute, Boston, MA 02215, USA; Donnelly Centre, University of Toronto, 
Toronto, ON M5S 3E1, Canada; Department of Molecular Genetics, University of 
Toronto, Toronto, ON M5S 3E1, Canada; Lunenfeld-Tanenbaum Research Institute, 
Mt. Sinai Hospital, Toronto, ON M5G 1X5, Canada; Canadian Institute for Advanced 
Research, Toronto, ON M5G 1Z8, Canada.
(15)Department of Psychiatry, University of California, San Diego, La Jolla, CA 
92093, USA. Electronic address: lilyak@ucsd.edu.
(16)Center for Cancer Systems Biology (CCSB) and Department of Cancer Biology, 
Dana-Farber Cancer Institute, Boston, MA 02215, USA; Department of 
Bioengineering, McGill University, Montreal, QC H3A 0C3, Canada. Electronic 
address: brandon.xia@mcgill.ca.
(17)Genomic Analysis of Network Perturbations Center of Excellence in Genomic 
Science (CEGS), Dana-Farber Cancer Institute, Boston, MA 02215, USA; Center for 
Cancer Systems Biology (CCSB) and Department of Cancer Biology, Dana-Farber 
Cancer Institute, Boston, MA 02215, USA; Department of Genetics, Harvard Medical 
School, Boston, MA 02115, USA. Electronic address: marc_vidal@dfci.harvard.edu.

Comment in
    Nat Methods. 2016 Apr;13(4):291. doi: 10.1038/nmeth.3828.

While alternative splicing is known to diversify the functional characteristics 
of some genes, the extent to which protein isoforms globally contribute to 
functional complexity on a proteomic scale remains unknown. To address this 
systematically, we cloned full-length open reading frames of alternatively 
spliced transcripts for a large number of human genes and used protein-protein 
interaction profiling to functionally compare hundreds of protein isoform pairs. 
The majority of isoform pairs share less than 50% of their interactions. In the 
global context of interactome network maps, alternative isoforms tend to behave 
like distinct proteins rather than minor variants of each other. Interaction 
partners specific to alternative isoforms tend to be expressed in a highly 
tissue-specific manner and belong to distinct functional modules. Our strategy, 
applicable to other functional characteristics, reveals a widespread expansion 
of protein interaction capabilities through alternative splicing and suggests 
that many alternative "isoforms" are functionally divergent (i.e., "functional 
alloforms").

Copyright © 2016 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.cell.2016.01.029
PMCID: PMC4882190
PMID: 26871637 [Indexed for MEDLINE]

## Full Text

Graphical Abstract

Alternatively-spliced isoforms of proteins exhibit strikingly different
interaction profiles and thus in the context of global interactome networks
appear to behave as if encoded by distinct genes, rather than as minor variants
of each other.

INTRODUCTION

Humans are more complex than worms or fruit flies, yet they appear to have
roughly the same number of protein-coding genes ( Blencowe, 2006 ). One way to address this apparent paradox is to
investigate the extent to which functionally different polypeptides can be encoded
by individual genes in various species.

Eukaryotic genes can encode multiple protein ‘forms’ via
alternative transcription, splicing, 3′ end formation, translation, and
post-translational modification. Alternative splicing produces transcript
‘isoforms’ for most human genes ( Pan
et al., 2008 ; Wang et al., 2008 ),
providing functional diversity at the level of enzymatic activities, subcellular
localizations, as well as protein-protein, protein-DNA, and protein-ligand physical
interactions ( Kelemen et al., 2013 ). An
isoform may exhibit dominant negative effects over other isoforms encoded by the
same gene, be up- or down-regulated instead of constitutively active, or even have
opposing cellular functions. For example, two isoforms encoded by the BCL2L1 gene have opposite functions in apoptosis, the longer
isoform inhibits the process while the shorter one promotes it ( Schwerk and Schulze-Osthoff, 2005 ). In another
example, ubiquitous alternative splicing of D. melanogaster Dscam1 generates thousands of different polypeptides, each with different binding
specificities to enable self-recognition of neurons ( Wojtowicz et al., 2007 ). Altogether, several hundred human genes are
known to encode alternatively spliced isoforms with distinct functional
characteristics ( Kelemen et al., 2013 ).

What remains unclear is how widespread this phenomenon is at the scale of
the whole proteome, which is of much higher complexity than originally anticipated
( Tran et al., 2011 ). As many as 100,000
distinct isoform transcripts could be produced from the ~20,000 human
protein-coding genes ( Pan et al., 2008 ),
collectively leading to perhaps over a million distinct polypeptides obtained by
post-translational modification of products of all possible transcript isoforms
( Smith and Kelleher, 2013 ). How such
proteomic complexity relates to global cellular processes is essentially unknown. To
what extent are pairs of isoforms encoded by a common gene functionally different
from each other? How widespread is isoform-specific functional diversity in any
given species? How might such functional diversity vary between species? What role
does this diversity play in evolution? Altogether, the central challenge is to
determine the extent to which two distinct, yet non-mutually exclusive, models might
apply: 1) alternative isoforms tend to mediate similar functions, i.e., they mostly
behave as ‘functional isoforms’; and 2) alternative isoforms tend to
display distinct functions, i.e., they should mostly be considered as
‘functional alloforms’ ( Figure
1A ).

So far, investigations into the role of alternative splicing have focused on
the functions alternative protein isoforms can or cannot perform, relative to their
so-called ‘reference’ counterpart ( Buljan et al., 2012 ; Ellis et al.,
2012 ). To begin addressing the questions outlined above in a systematic
and unbiased manner, large-scale functional profiling approaches are needed to
quantify the extent to which all isoforms encoded by large numbers of genes are
functionally similar or different from each other, taking all pairwise combinations
of isoforms encoded by the same gene into consideration. This, in turn, requires
novel methodologies to identify, clone, and exogenously express full-length open
reading frames (ORFs) for all isoforms across a wide range of genes.

Contemporary attempts at systematically discovering alternatively spliced
isoforms genome-wide have been based on next-generation sequencing (NGS) methods.
For example, RNA-Seq provides relatively deep sampling ( Pan et al., 2008 ; Wang et al.,
2008 ). However, the short length of RNA-Seq reads has hampered the
discovery of contiguous exon connectivity for full-length alternatively spliced
isoforms. Full-length sequencing of single cDNA molecules, or
“Iso-seq” ( Eid et al., 2009 ),
has proven successful in generating improved models of full-length transcript
isoforms ( Sharon et al., 2013 ). Another
strategy captures co-association of distant alternatively spliced exons by limiting
the number of RNA molecules in the pools used to generate sequencing libraries
( Tilgner et al., 2015 ). However, none of
the above strategies provide the large-scale physical clone collections needed to
systematically express and study the function of alternative isoforms.

Here, we apply a new strategy, “ORF-Seq”, to discover,
characterize, exogenously express, and functionally investigate large numbers of
alternatively spliced full-length ORFs. We have applied this strategy to the study
of binary protein-protein interactions (PPIs) and identified widespread interaction
differences due to alternative splicing ( Figure
1A ). Alternatively spliced protein isoforms tend to behave
like completely distinct genes in interactome networks rather than minor variants of
each other. Thus, a sizable proportion of alternative isoforms in the human proteome
are “functional alloforms” ( Figure
1A ).
