---
pmid: '20705607'
title: Turning the RING domain protein MdmX into an active ubiquitin-protein ligase.
authors:
- Iyappan S
- Wollscheid HP
- Rojas-Fernandez A
- Marquardt A
- Tang HC
- Singh RK
- Scheffner M
journal: J Biol Chem
year: '2010'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2963378
doi: 10.1074/jbc.M110.115113
---

# Turning the RING domain protein MdmX into an active ubiquitin-protein ligase.
**Authors:** Iyappan S, Wollscheid HP, Rojas-Fernandez A, Marquardt A, Tang HC, Singh RK, Scheffner M
**Journal:** J Biol Chem (2010)
**DOI:** [10.1074/jbc.M110.115113](https://doi.org/10.1074/jbc.M110.115113)
**PMC:** [PMC2963378](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2963378/)

## Abstract

1. J Biol Chem. 2010 Oct 22;285(43):33065-33072. doi: 10.1074/jbc.M110.115113.
Epub  2010 Aug 12.

Turning the RING domain protein MdmX into an active ubiquitin-protein ligase.

Iyappan S(1), Wollscheid HP(1), Rojas-Fernandez A(1), Marquardt A(2), Tang 
HC(1), Singh RK(3), Scheffner M(4).

Author information:
(1)From the Department of Biology and Konstanz Research School Chemical Biology, 
78457 Konstanz, Germany.
(2)Proteomics Facility, University of Konstanz, 78457 Konstanz, Germany.
(3)Department of Chemistry and Biochemistry, Center for Biomolecular Structure 
and Organization, University of Maryland, College Park, Maryland 20742-3360.
(4)From the Department of Biology and Konstanz Research School Chemical Biology, 
78457 Konstanz, Germany. Electronic address: martin.scheffner@uni-konstanz.de.

The related RING domain proteins MdmX and Mdm2 are best known for their role as 
negative regulators of the tumor suppressor p53. However, although Mdm2 
functions as a ubiquitin ligase for p53, MdmX does not have appreciable 
ubiquitin ligase activity. In this study, we performed a mutational analysis of 
the RING domain of MdmX, and we identified two distinct regions that, when 
replaced by the respective regions of Mdm2, turn MdmX into an active ubiquitin 
ligase for p53. Mdm2 and MdmX form homodimers as well as heterodimers with each 
other. One of the regions identified localizes to the dimer interface indicating 
that subtle conformational changes in this region either affect dimer stability 
and/or the interaction with the ubiquitin-conjugating enzyme UbcH5b. The second 
region contains the cryptic nucleolar localization signal of Mdm2 but is also 
assumed to be involved in the interaction with UbcH5b. Here, we show that this 
region has a significant impact on the ability of respective MdmX mutants to 
functionally interact with UbcH5b in vitro supporting the notion that this 
region serves two distinct functional purposes, nucleolar localization and 
ubiquitin ligase activity. Finally, evidence is provided to suggest that the 
RING domain of Mdm2 not only binds to UbcH5b but also acts as an allosteric 
activator of UbcH5b.

DOI: 10.1074/jbc.M110.115113
PMCID: PMC2963378
PMID: 20705607 [Indexed for MEDLINE]

## Full Text

Abstract

The related RING domain proteins MdmX and Mdm2 are best known for their role as negative regulators of the tumor suppressor p53. However, although Mdm2 functions as a ubiquitin ligase for p53, MdmX does not have appreciable ubiquitin ligase activity. In this study, we performed a mutational analysis of the RING domain of MdmX, and we identified two distinct regions that, when replaced by the respective regions of Mdm2, turn MdmX into an active ubiquitin ligase for p53. Mdm2 and MdmX form homodimers as well as heterodimers with each other. One of the regions identified localizes to the dimer interface indicating that subtle conformational changes in this region either affect dimer stability and/or the interaction with the ubiquitin-conjugating enzyme UbcH5b. The second region contains the cryptic nucleolar localization signal of Mdm2 but is also assumed to be involved in the interaction with UbcH5b. Here, we show that this region has a significant impact on the ability of respective MdmX mutants to functionally interact with UbcH5b in vitro supporting the notion that this region serves two distinct functional purposes, nucleolar localization and ubiquitin ligase activity. Finally, evidence is provided to suggest that the RING domain of Mdm2 not only binds to UbcH5b but also acts as an allosteric activator of UbcH5b.

Introduction

Genetic experiments in mice have shown that the related proteins Mdm2 and MdmX are critical regulators of the tumor suppressor protein p53. Knock-out of the mdm2 or the mdm4 gene (encoding MdmX) causes embryonic lethality, which is rescued by concomitant deletion of the p53 gene ( 1 – 4 ). Furthermore, analysis of tissue-specific Mdm2 and MdmX null mice indicates that the presence of MdmX is required to keep p53 in check in most but not all tissues, whereas Mdm2 appears to be essential in all tissues studied ( 5 – 11 ). Together with biochemical studies ( 12 – 14 ), these findings suggest that Mdm2 and MdmX act synergistically in p53 regulation but that Mdm2 also has MdmX-independent functions in p53 regulation and vice versa.

Mdm2 and MdmX share significant structural similarity, with the most conserved regions being the p53-binding domain within the N terminus of the respective protein, a central zinc- binding domain of unknown function, and a C-terminal RING domain ( 11 , 15 , 16 ). In many cases, RING or RING-like domains have been shown to represent interaction sites for ubiquitin-conjugating enzymes (E2), and thus, the presence of a RING domain is commonly assumed to be indicative for proteins with the function of an E3 ubiquitin-protein ligase ( 17 – 19 ). Indeed, Mdm2 has E3 ligase activity and in concert with members of the UbcH5 subfamily of E2s targets p53 for ubiquitination and degradation ( 20 – 24 ).

Unlike Mdm2, MdmX has no or only little E3 activity. It rather acts as a stimulator of the E3 activity of Mdm2 via heterocomplex formation, which is mediated by the respective RING domains of Mdm2 and MdmX ( 12 , 13 , 25 , 26 ). Indeed, di- or multimeric forms of Mdm2 rather than Mdm2 monomers have E3 activity ( 14 , 27 ). Furthermore, Mdm2-MdmX heteromers appear to be thermodynamically more stable than Mdm2 homomers providing a possible explanation for the observation that MdmX stimulates Mdm2 activity ( 25 ). In addition, Mdm2 but not MdmX contains amino acid sequence motifs for nuclear localization as well as nuclear export ( 11 , 16 , 24 ). Consequently, Mdm2 can shuttle between the nucleus and the cytosol, whereas the subcellular localization of MdmX depends on its interaction with Mdm2 or p53 ( 28 , 29 ). Finally, there is evidence to indicate that Mdm2 controls the levels of MdmX and vice versa ( 13 , 28 , 30 – 32 ). In conclusion, the available data indicate that Mdm2 and MdmX cooperate to control the turnover rate of each other as well as that of p53.

Structural studies identified the amino acid residues of the RING domain of Mdm2 and MdmX that are involved in Mdm2 homodimer formation and Mdm2-MdmX heterodimer formation, respectively, and did not reveal any significant differences between the Mdm2 homodimer and the Mdm2-MdmX heterodimer (the structure of the RING domain of MdmX alone has not yet been reported) ( 27 , 33 ). Furthermore, in silico comparisons with structures of RING domains of other E3s with their cognate E2 enzymes indicate that the amino acid residues of the Mdm2 RING domain that are presumably involved in E2 interaction are at least in part conserved in the MdmX RING domain ( 27 , 33 ). Thus, to identify the regions within the RING domain of Mdm2 that render it an active E3 ligase for p53, we performed a mutational analysis of the RING domain of MdmX. The results obtained indicate that a region at the interface of the RING-RING dimer and a region overlapping with the cryptic nucleolar localization signal of Mdm2 ( 34 , 35 ) are critical for E3 activity. In addition, the latter region affects the ability of the isolated RING domain of respective MdmX mutants to stimulate UbcH5b activity.

DISCUSSION

In this study, we show that substitution of two regions within the MdmX RING domain by the respective regions of Mdm2 turn MdmX into an active E3 ligase for p53. The first region encompasses amino acids 448–453, and a single amino acid substitution in this region (N448C; Figs. 1 B and 7 ) results in an MdmX mutant with a significantly increased ability to ubiquitinate p53 in vitro . For ubiquitination and degradation of p53 within cells, however, concomitant substitution of the region encompassing amino acid residues 465–480 is required ( Figs. 2 C and 7 ). In view of the finding that the RING domain of MdmX can interact with UbcH5b ( supplemental Fig. 1 ), these data suggest that a physical contact between a RING domain and its cognate E2 does not suffice to facilitate E2-mediated ubiquitination of substrate proteins and that for functional interaction additional constraints must be met.

Structural analyses of the Mdm2 RING domain homodimer and the Mdm2-MdmX RING domain heterodimer have shown that residues 448–453 are part of a β-strand (β1) involved in mediating RING-RING dimer formation ( Fig. 7 ) ( 27 , 33 ). Although Ile-450 (MdmX) and Val-451 (Mdm2) are primarily involved in the formation of the dimer interface, Asn-448 (MdmX) and Cys-449 (Mdm2) do not appear to directly contribute to dimer formation but are buried at the interface facing each other. Thus, although the structure of MdmX RING domain homodimers has not yet been reported, it seems conceivable that the presence of two asparagines at this position of the interface affects the overall conformation of the MdmX RING domain homodimer and/or its thermodynamic stability (note that dimerization of Mdm2 is required for E3 ligase activity ( 14 , 27 )). We attempted to test the latter possibility by studying the stability of the various RING domain dimers by surface plasmon resonance but were not successful (mainly because bacterially expressed RING domain of Mdm2 tends to aggregate). It should be noted that substitution of Cys-449 of Mdm2 by serine was previously reported to result in an Mdm2 mutant that retains some E3 activity in vitro but is incapable of acting as an E3 ligase within cells ( 37 ). Thus, even a slight deviation ( i.e. exchange of a thiol group by a hydroxyl group) at the respective position significantly affects E3 ligase activity. Nonetheless, to understand the mechanism by which the residue at position 448 (of MdmX) affects the E3 ligase activity of Mdm2 and MdmX, respectively, the structure of MdmX RING domain homodimers and, maybe more importantly, structures of the various RING domain dimers (Mdm2, Mdm2-MdmX, and MdmX) in complex with UbcH5b will have to be solved.

Amino acid residues 466–481 of Mdm2 but not the respective region of MdmX (residues 465–480) contain a nucleolar localization signal (residues 466–474) that was previously shown to be functional when p14ARF is bound to the central region of Mdm2 or when an extended central region of Mdm2 (residues 222–437) is deleted ( 34 , 35 ). Based on the data obtained with p53 mutants ( Fig. 4 ) and by immunofluorescence analysis ( supplemental Fig. 3 ), it seems likely that nucleolar localization of neither p53 nor Mdm2 is essential for p53 degradation. Moreover, the results obtained with the MdmX RING domain mutants CH8 and CH22 ( Fig. 5 ) indicate that residues 466–481 of Mdm2 serve an additional function. Structural comparison of the MDM2 RING domain with other RING domains in complex with their cognate E2s ( 27 , 33 ) implicated residues 468, 469, and 472–480 of Mdm2 to be important for UbcH5b binding ( Fig. 7 ). This assumption (amino acids 466–481 are part of the interaction surface of the RING domain of Mdm2 for UbcH5b) provides a reasonable explanation for the observation that in vitro CH22 has a more significant effect on UbcH5b activity than CH8 ( Fig. 5 D ). Thus, a tempting but purely speculative model is that Mdm2 nucleolar localization (which is likely mediated by interaction of the nucleolar localization signal with a yet unknown protein ( 34 , 35 , 39 )) and functional interaction of Mdm2 with UbcH5b are mutually exclusive events. In this model, binding of p14ARF to the central region of Mdm2 would disfavor the interaction of Mdm2 with UbcH5b and induce localization of Mdm2 to nucleoli and, as previously shown, p53 stabilization ( 23 ). On the other hand, interaction of the central region of Mdm2 with p53 ( 40 ) or with factors that stimulate Mdm2-mediated ubiquitination of p53, including CBP/p300 ( 41 , 42 ), may positively affect Mdm2-UbcH5b interaction.

The above scenario (interaction of the RING domain with UbcH5b is modulated by cellular factors) may also provide an explanation for the observation that the central region of Mdm2, which contains the acidic domain ( Fig. 2 A ), is required for p53 ubiquitination and degradation in cells but not in vitro . For example, it is conceivable that the respective region of MdmX interacts with negative regulatory factors but not with positive ones, and these potential inhibitory factors are not present in the in vitro system. Another possibility is provided by the finding that the central region of Mdm2 interacts with the Mdm2 RING domain ( 43 ). Thus, rather than interacting with additional proteins, interaction of the RING domain with the central region may positively affect Mdm2-UbcH5b interaction. In analogy to the results obtained for the region spanning amino acids 466–481 of Mmd2, this stimulatory effect may be required for efficient p53 degradation within cells but not for in vitro ubiquitination. To prove this possibility, experiments in defined in vitro systems will have to be performed.

Recent data indicate that RING domain E3 ligases do not only function as adaptor proteins between substrate proteins and E2s but in addition can act as allosteric activators of E2s ( 36 , 44 ). Most significantly with respect to this study, detailed analysis of the interaction of UbcH5b with APC2/APC11 and the isolated RING domain of CNOT4 revealed that binding of an isolated RING domain induces slight but significant conformational changes in the cognate E2 ( 36 ) with isoleucine at position 88 as one of the residues affected. Importantly, substitution of Ile-88 by alanine resulted in a UbcH5b mutant that can still bind to RING domains and form thioester complexes with ubiquitin with an efficiency similar to UbcH5b but can no longer be allosterically activated ( 36 ). In agreement with the notion that the Mdm2 RING domain also acts as an allosteric activator of UbcH5b, UbcH5bI88A was only poorly activated by Mdm2 ( Fig. 6 ) and only inefficiently supports Mdm2-facilitated ubiquitination of p53 in vitro (data not shown). However, the RING domain of Mdm2 has been reported to form dimeric and higher order multimeric structures ( 25 , 27 , 33 , 45 ). An alternative but not mutually exclusive possibility therefore is that by forming multimeric structures, the RING domain of Mdm2 brings two or more UbcH5b molecules into close proximity thereby facilitating UbcH5b-mediated formation of (free) ubiquitin chains and ubiquitination of Mdm2 or Mdm2-associated proteins ( e.g. p53).

Studies with transgenic mice indicate that Mdm2 and MdmX act synergistically but also have independent nonredundant functions in the control of p53 ( 5 – 10 ). In a simplified view, Mdm2 may counteract p53 activity mainly by targeting p53 for degradation, whereas MdmX may form tighter complexes with p53 than Mdm2 thereby interfering with p53 activity in a stoichiometric manner ( 16 , 25 ). In combination with the notion that Mdm2-MdmX heteromeric complexes may possess both properties, this may allow a cell to finely tune Mdm2/MdmX-mediated regulation of p53. In this context, it will be interesting to determine/characterize the phenotype of transgenic mice that, instead of Mdm2 and MdmX, express an Mdm2-MdmX chimeric protein such as AD-CH22 that potentially possesses the properties of both Mdm2 and MdmX.
