---
pmid: '10037815'
title: Combining SSH and cDNA microarrays for rapid identification of differentially
  expressed genes.
authors:
- Yang GP
- Ross DT
- Kuang WW
- Brown PO
- Weigel RJ
journal: Nucleic Acids Res
year: '1999'
full_text_available: false
full_text_extraction_method: html_abstract_only
pmcid: PMC148347
doi: 10.1093/nar/27.6.1517
---

# Combining SSH and cDNA microarrays for rapid identification of differentially expressed genes.
**Authors:** Yang GP, Ross DT, Kuang WW, Brown PO, Weigel RJ
**Journal:** Nucleic Acids Res (1999)
**DOI:** [10.1093/nar/27.6.1517](https://doi.org/10.1093/nar/27.6.1517)
**PMC:** [PMC148347](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC148347/)

## Abstract

1. Nucleic Acids Res. 1999 Mar 15;27(6):1517-23. doi: 10.1093/nar/27.6.1517.

Combining SSH and cDNA microarrays for rapid identification of differentially 
expressed genes.

Yang GP(1), Ross DT, Kuang WW, Brown PO, Weigel RJ.

Author information:
(1)Department of Surgery and Department of Biochemistry, Stanford University, 
Stanford, CA 94305-5414, USA.

Comparing patterns of gene expression in cell lines and tissues has important 
applications in a variety of biological systems. In this study we have examined 
whether the emerging technology of cDNA microarrays will allow a high throughput 
analysis of expression of cDNA clones generated by suppression subtractive 
hybridization (SSH). A set of cDNA clones including 332 SSH inserts amplified by 
PCR was arrayed using robotic printing. The cDNA arrays were hybridized with 
fluorescent labeled probes prepared from RNA from ER-positive (MCF7 and T47D) 
and ER-negative (MDA-MB-231 and HBL-100) breast cancer cell lines. Ten clones 
were identified that were over-expressed by at least a factor of five in the 
ER-positive cell lines. Northern blot analysis confirmed over-expression of 
these 10 cDNAs. Sequence analysis identified four of these clones as cytokeratin 
19, GATA-3, CD24 and glutathione-S-transferase mu-3. Of the remaining six cDNA 
clones, four clones matched EST sequences from two different genes and two 
clones were novel sequences. Flow cytometry and immunofluorescence confirmed 
that CD24 protein was over-expressed in the ER-positive cell lines. We conclude 
that SSH and microarray technology can be successfully applied to identify 
differentially expressed genes. This approach allowed the identification of 
differentially expressed genes without the need to obtain previously cloned 
cDNAs.

DOI: 10.1093/nar/27.6.1517
PMCID: PMC148347
PMID: 10037815 [Indexed for MEDLINE]

## Full Text

Abstract

Comparing patterns of gene expression in cell lines and tissues has important applications in a variety of biological systems. In this study we have examined whether the emerging technology of cDNA microarrays will allow a high throughput analysis of expression of cDNA clones generated by suppression subtractive hybridization (SSH). A set of cDNA clones including 332 SSH inserts amplified by PCR was arrayed using robotic printing. The cDNA arrays were hybridized with fluorescent labeled probes prepared from RNA from ER-positive (MCF7 and T47D) and ER-negative (MDA-MB-231 and HBL-100) breast cancer cell lines. Ten clones were identified that were over-expressed by at least a factor of five in the ER-positive cell lines. Northern blot analysis confirmed over-expression of these 10 cDNAs. Sequence analysis identified four of these clones as cytokeratin 19, GATA-3, CD24 and glutathione-S-transferase mu-3. Of the remaining six cDNA clones, four clones matched EST sequences from two different genes and two clones were novel sequences. Flow cytometry and immunofluorescence confirmed that CD24 protein was over-expressed in the ER-positive cell lines. We conclude that SSH and microarray technology can be successfully applied to identify differentially expressed genes. This approach allowed the identification of differentially expressed genes without the need to obtain previously cloned cDNAs.
