---
pmid: '24051492'
title: p53 regulates Period2 expression and the circadian clock.
authors:
- Miki T
- Matsumoto T
- Zhao Z
- Lee CC
journal: Nat Commun
year: '2013'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3798035
doi: 10.1038/ncomms3444
---

# p53 regulates Period2 expression and the circadian clock.
**Authors:** Miki T, Matsumoto T, Zhao Z, Lee CC
**Journal:** Nat Commun (2013)
**DOI:** [10.1038/ncomms3444](https://doi.org/10.1038/ncomms3444)
**PMC:** [PMC3798035](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3798035/)

## Abstract

1. Nat Commun. 2013;4:2444. doi: 10.1038/ncomms3444.

p53 regulates Period2 expression and the circadian clock.

Miki T(1), Matsumoto T, Zhao Z, Lee CC.

Author information:
(1)1] Department of Biochemistry and Molecular Biology, Medical School, 
University of Texas Health Science Center-Houston, Houston, Texas 77030, USA 
[2].

The mechanistic interconnectivity between circadian regulation and the genotoxic 
stress response remains poorly understood. Here we show that the expression of 
Period 2 (Per2), a circadian regulator, is directly regulated by p53 binding to 
a response element in the Per2 promoter. This p53 response element is 
evolutionarily conserved and overlaps with the E-Box element critical for 
BMAL1/CLOCK binding and its transcriptional activation of Per2 expression. Our 
studies reveal that p53 blocks BMAL1/CLOCK binding to the Per2 promoter, leading 
to repression of Per2 expression. In the suprachiasmatic nucleus (SCN), p53 
expression and its binding to the Per2 promoter are under circadian control. 
Per2 expression in the SCN is altered by p53 deficiency or stabilization of p53 
by Nutlin-3. Behaviourally, p53⁻/⁻ mice have a shorter period length that lacks 
stability, and they exhibit impaired photo-entrainment to a light pulse under a 
free-running state. Our studies demonstrate that p53 modulates mouse circadian 
behaviour.

DOI: 10.1038/ncomms3444
PMCID: PMC3798035
PMID: 24051492 [Indexed for MEDLINE]

Conflict of interest statement: Conflict of interest statement: The authors 
declare no competing financial interests.

## Full Text

IntroductionIt is recognized that circadian function and the genotoxic stress response are interconnected. This is thought to be an evolutionary adaption of the organism to sunlight irradiation on earth. The transcription factor p53 is known as the “guardian of the genome”1,2. The majority of all human tumors have mutations in the TP53 gene which encodes the p53 protein3. The roles of stress-induced p53 in cellular responses are well studied. For example, cell cycle arrest and the DNA damage response are among the biological processes that are regulated by stress-induced p534. Circadian rhythms are the daily oscillations of biological processes driven by endogenous molecular clocks5. The known core clock genes include Clock, Npas2, Bmal1, Per1, Per2, Cry1, Cry2, CK1ε, CK1δ, Rev-Erbα, and RORα. Typically, the CLOCK/BMAL1 heterodimer binds to the E-box in the promoter region of their target genes, leading to activation of the expression of these genes, including Per1, Per2, Cry1 and Cry2. The protein products of these core clock genes regulate each other’s expression in feedback loops. Recently, we showed that the promyelocytic leukemia (PML) protein regulates the circadian clock by interacting with PER2 and enhancing BMAL1/CLOCK mediated transcription6. PML is essential for formation of PML nuclear bodies (NB’s)7. It is thought that PML-NBs recruit specific proteins and regulate their activity by modulating their post-translational modification. To date, a growing list of PML interacting proteins has been reported8. Among them are both positive and negative regulators involved in the circadian clock mechanism including PER2, BMAL1, SIRT1, CBP, WDR5 and SIN3A6,9–12. These findings led us to screen other PML associated proteins for a possible role in modulating BMAL1/CLOCK mediated Per2 transcription. Among the PML associated proteins examined was p53, which is known to localize with the PML nuclear body. Studies have shown that acetylation and deacetylation of p53 by CBP and SIRT1, respectively, are PML-NB dependent13–15.Here, we show that p53 modulates the circadian behavior of mice by modulating Per2 expression, acting as a direct competitor of BMAL1/CLOCK binding to the Per2 promoter. This mechanistic interaction may represent a key regulatory link between the p53-mediated cellular stress response and the circadian clock-regulated cellular pathways.

Introduction

It is recognized that circadian function and the genotoxic stress response are interconnected. This is thought to be an evolutionary adaption of the organism to sunlight irradiation on earth. The transcription factor p53 is known as the “guardian of the genome”1,2. The majority of all human tumors have mutations in the TP53 gene which encodes the p53 protein3. The roles of stress-induced p53 in cellular responses are well studied. For example, cell cycle arrest and the DNA damage response are among the biological processes that are regulated by stress-induced p534. Circadian rhythms are the daily oscillations of biological processes driven by endogenous molecular clocks5. The known core clock genes include Clock, Npas2, Bmal1, Per1, Per2, Cry1, Cry2, CK1ε, CK1δ, Rev-Erbα, and RORα. Typically, the CLOCK/BMAL1 heterodimer binds to the E-box in the promoter region of their target genes, leading to activation of the expression of these genes, including Per1, Per2, Cry1 and Cry2. The protein products of these core clock genes regulate each other’s expression in feedback loops. Recently, we showed that the promyelocytic leukemia (PML) protein regulates the circadian clock by interacting with PER2 and enhancing BMAL1/CLOCK mediated transcription6. PML is essential for formation of PML nuclear bodies (NB’s)7. It is thought that PML-NBs recruit specific proteins and regulate their activity by modulating their post-translational modification. To date, a growing list of PML interacting proteins has been reported8. Among them are both positive and negative regulators involved in the circadian clock mechanism including PER2, BMAL1, SIRT1, CBP, WDR5 and SIN3A6,9–12. These findings led us to screen other PML associated proteins for a possible role in modulating BMAL1/CLOCK mediated Per2 transcription. Among the PML associated proteins examined was p53, which is known to localize with the PML nuclear body. Studies have shown that acetylation and deacetylation of p53 by CBP and SIRT1, respectively, are PML-NB dependent13–15.

Here, we show that p53 modulates the circadian behavior of mice by modulating Per2 expression, acting as a direct competitor of BMAL1/CLOCK binding to the Per2 promoter. This mechanistic interaction may represent a key regulatory link between the p53-mediated cellular stress response and the circadian clock-regulated cellular pathways.

Resultsp53 represses BMAL1/CLOCK-mediated transcription of Per2A number of PML-associated proteins were screened for their potential ability to modulate BMAL1/CLOCK mediated Per2 promoter transcription. Expression constructs for p53, WRN, RB, and RARα were individually examined using a Per2-Luc reporter assay for their ability to up or down regulate BMAL1/CLOCK mediated transcription. Among these expression constructs, we observed that the one encoding p53 strongly repressed BMAL1/CLOCK mediated transcription of the Per2 promoter (Figure 1a). Therefore, we asked whether this apparent repression of Per2 transcription by p53 was mediated through BMAL1/CLOCK. High levels of p53 can induce apoptosis of tumor cell lines and this could result in an apparent non-specific transcriptional repression16. To rule out this possibility, we compared wildtype mouse embryonic fibroblasts (MEF) with HeLa cells using a Luc-reporter assay driven by either the Bmal1 or Per2 promoter. In both wildtype MEF and HeLa cells, the expression of p53 did not repress RORγ mediated transcription of the Bmal1 promoter but repressed Per2 promoter transcription mediated by BMAL1/CLOCK (Figure 1b and Supplementary Figure S1).The endogenous p53 level is stabilized and enhanced by Nutlin-3, a small molecule inhibitor of MDM217. MDM2 is the major negative regulator of p53 and facilitates p53 degradation18. BMAL1/CLOCK-mediated transcription of the Per2 promoter was repressed in wildtype, but not in p53−/− MEF cells when treated with Nutlin-3 (Figure 1c). Transfection of a p53 expression construct into p53−/− MEF cells restored the repression of BMAL1/CLOCK-mediated transcription of the Per2 promoter in the presence of Nutlin-3. Western analysis with anti-p53 and anti-PER2 antibodies showed a time-dependent increase in the levels of endogenous p53, accompanied by a reciprocal decrease in endogenous PER2 in wildtype MEF cells following Nutlin-3 treatment (Figure 1d). No decrease in PER2 levels by Nutlin-3 treatment was observed in p53−/− MEF cells (Figure 1e). This inverse relationship between p53 and PER2 levels was further examined in a Nutlin-3 titration experiment. After a 16 h treatment, endogenous p53 levels correlated with increases in Nutlin-3 concentration, and were inversely correlated with PER2 levels in wildtype MEF cells (Figure 1f). No change in PER2 levels was observed when p53−/− MEF cells were treated with Nutlin-3 at a similar concentration (Figure 1g). Genotoxic stress such as γ-irradiation is known to rapidly stabilize p53 protein in some cells, including MEF and human osteosarcoma U2OS cells19. Western analysis of human U2OS cells and wildtype MEF treated with γ-irradiation displayed a rapid rise in p53 and a reciprocal decline in PER2 levels (Figure 1h, Supplementary Figure S2). This decline in PER2 level was not observed in p53−/− MEF cells that were similarly treated with γ-irradiation (Figure 1i). Together, these studies show that the cellular level of PER2 is inversely correlated with that of p53.The Per2 promoter contains a conserved p53 response elementThe inverse correlation between p53 and PER2 suggests that p53 is repressing BMAL1/CLOCK mediated transcription of the Per2 promoter. To map the putative response element for p53, BMAL1/CLOCK mediated transcription, reporter assays were carried out using serial deletion constructs of the mouse Per2 promoter at −2811 to −105 bp from the transcription start site (Figure 2a). The repression by p53 of BMAL1/CLOCK mediated transcription of the Per2 promoter was intact in all of these deletion constructs suggesting that the putative p53 response element is within the −105 bp region of the Per2 promoter. Using the NBLAST program, comparison of this 105 bp mouse sequence with Per2 promoter sequences of human, dog and rat from Genebank revealed a highly conserved 48 bp sequence (Figure 2b). Sequence analysis with the TFBIND program revealed a putative p53 response element. The p53 response element is a motif of two tandem decamers separated by 0–13 nucleotides. The consensus sequences can be depicted as 5'-RRXCXXGXYX-N(0–13)-XRXCXXGXYY-3', where R= A/G, Y= C/T and X= A/G/C/T 20. This putative p53 response element is upstream to the transcription start site of the Per2 promoter, and its proximal decamer overlaps with the critical E-box sequence for BMAL1/CLOCK binding. Mutation of nucleotides adjacent to the E-box severely dampened BMAL1/CLOCK mediated transcription of the Per2 promoter. These observations are consistent with previous findings that BMAL1/CLOCK mediated transcription of the Per2 promoter requires a segment about 20 bases from the transcription start site21. Interestingly, mutation of nucleotides in the distal decamer of the putative p53 response element did not significantly affect BMAL1/CLOCK mediated transcription while the strong repression by p53 was relieved (Figure 2b, 2c).To further characterize the region upstream of the Per2 promoter initiation site, sense and antisense oligonucleotides corresponding to the mouse Per2 promoter sequence from +7 to −45 bp were synthesized, annealed and cloned into a luciferase reporter vector to generate −45bpPer2-Luc (Figure 2b). Transcriptional activation by BMAL1/CLOCK and repression by p53 of expression from the −45bpPer2-Luc and −105bpPer2-Luc reporter constructs were comparable, indicating that the −45bpPer2 DNA promoter sequence contained both the p53 response element and the E-box of BMAL1/CLOCK (Figure 2d).Using an electrophoretic mobility shift assay (EMSA) we observed that recombinant p53 protein binds to −45bpPer2 DNA (Figure 2e). The binding of p53 to biotin-labeled −45bpPer2 DNA was competitively reduced by unlabeled −45bpPer2 DNA. Some antibodies to p53 are known to induce a super-shift of the p53/DNA complex. We tested two commercial p53 antibodies and they both generated the expected super-shifts (Figure 2e, Supplementary Figure S3). Further, the super-shift was competitively reduced by unlabeled −45bpPer2 DNA. However, when the unlabeled −45bpPer2 DNA contained a mutated sequence in the distal decamer, the super-shift binding was not reduced (Figure 2f).The Per2 genomic locus has five described E-box elements (E1-E5), and the one most critical for BMAL1/CLOCK binding is E2 located in the promoter region22. Chromatin immuno-precipitation (ChIP) assays with anti-p53 antibodies were performed using genomic DNA that had been fragmented to 0.5 Kb on average. ChIP-qPCR analysis revealed that p53 pull-down of E2 was significantly enhanced in wildtype when cells were treated with Nutlin-3 (Figure 2g). In contrast, p53 pull-down of E5, about 3 Kb away from E2, was not significant (Supplementary Figure S4). Next, we determined whether BMAL1/CLOCK binding to the E2 region of the Per2 promoter was disrupted by p53. Transfection of p53−/− MEF with expression constructs for FLAG-tagged Clock, Bmal1 in the presence or absence of a p53 expression construct was carried out. ChIP-qPCR analysis showed that in the absence of p53, pull down of FLAG-CLOCK by the anti-FLAG antibody was significantly higher (Supplementary Figure S5).We then investigated whether p53 binding to the E2 region of the Per2 promoter also occurs in the SCN in vivo. SCN from wildtype mouse brains were dissected at six zeitgeber time (ZT) points over 24 h for ChIP analysis. ChIP-qPCR with anti-p53 antibodies showed a peak level of endogenous p53 binding to the Per2 promoter in the SCN between ZT4-ZT8 and a nadir at ZT16-ZT20 (Figure 2h). These studies demonstrate that p53 is expressed in the SCN and its interaction with the Per2 promoter is under temporal control in vivo. Next, the level of p53 or BMAL1/CLOCK binding to the E2 region of the Per2 promoter was examined using wild type and p53−/− mouse thymus obtained at ZT8 and ZT20. Thymus was chosen since it expresses detectable levels of p53 under normal physiological conditions. ChIP-qPCR analysis revealed that p53 binding to the E2 region was about 2.5X higher at ZT8 than ZT20 (Figure 2i). BMAL1 binding to the E2 region in p53−/− MEF was significantly elevated compared with wildtype (Figure 2j).To investigate whether p53 directly competes with BMAL1/CLOCK for binding to −45bpPer2 DNA, nuclear extracts from WT, p53−/− MEF and Bmal1−/− fibroblasts were prepared for EMSA. Unlike nuclear extracts from WT and p53−/− MEF, those from Bmal1−/− fibroblasts did not display a prominent protein/DNA complex on EMSA with −45bpPer2 DNA indicating that this complex is BMAL1 dependent (Figure 2k). Further, transfection of expression plasmids for Bmal1 and Flag-Clock into Bmal1−/− fibroblasts restored the BMAL1/CLOCK:-45bpPer2 protein/DNA complex in nuclear extracts. Treatment with either anti-BMAL1 or anti-FLAG antibodies significantly reduced this protein/DNA complex. Consistent with results from ChIP-qPCR analysis, nuclear extracts from p53−/− MEF and thymus showed an increased level of the BMAL1/CLOCK:-45bpPer2 protein/DNA complex when compared to wildtype samples (Figure 2k, 2l). Adding recombinant p53 to the p53−/− nuclear extract reduced the level of the BMAL1/CLOCK:-45bpPer2 DNA complex (Figure 2m). This observation shows that p53 competes with BMAL1/CLOCK for binding to the DNA probe. Collectively, these studies show that the Per2 promoter contains a functional p53 response element that overlaps with the E-box for BMAL1/CLOCK binding that is critical for transcription of the Per2 gene.p53 levels regulate Per2 transcription in the SCNTo gain insight into the relevance of the p53 response element in the Per2 promoter to clock function, p53 levels in the SCN, the master clock structure in mammals, were investigated under temporal conditions. Previous studies have shown that in peripheral tissues, the p53 protein level is under temporal control23–25. Immunofluorescence analysis with anti-p53 antibodies showed that the level of p53 in the SCN from wildtype mice peaks at ZT4-ZT8 and reaches a nadir at ZT16-ZT20 (Figure 3a, 3b). There was no p53 staining in the SCN of p53−/− mice. This observation is consistent with results from the ChIP assay that showed peak levels of p53 binding to the Per2 promoter occurred at about ZT4-ZT8.Next, we examined whether the loss of p53 elevates Per2 expression in the SCN as predicted by our in vitro studies. In situ hybridization studies showed that Per2 transcript levels in the SCN of p53−/− mice were significantly elevated compared to wildtype mice at most sampling times (Figure 3c, 3d). Interestingly, the expression of Per2 transcripts in p53−/− mice remained under temporal control and was apparently phase advanced, reaching peak levels at about ZT8 and a nadir at about ZT20. Consistent with results from in situ analysis, immunofluorescence studies with anti-PER2 antibodies showed that the level of PER2 was significantly elevated at ZT0 and ZT12 in the SCN of p53−/− mice compared to wildtype mice (Figure 3e).Next, we examined whether increasing p53 protein levels by Nutlin-3 treatment would have the converse effect of repressing Per2 transcription in the SCN. First, we investigated whether Nutlin-3 could stabilize p53 levels in the SCN. Wildtype mice were administered Nutlin-3 or vehicle (DMSO) at ZT9 and then sacrificed at ZT14. We selected ZT14 for these studies since endogenous SCN Per2 and p53 protein levels are close to their peak and nadir, respectively at this time point. Immunofluorescence analysis with anti-p53 antibodies showed increased p53 protein in the SCN after treatment with Nutlin-3 but not vehicle (Figure 3f, 3g). In situ hybridization with an antisense Per2 probe showed that endogenous Per2 expression in the SCN of mice treated at ZT14 was significantly dampened after treatment with Nutlin-3 compared with vehicle (Figure 3f, 3g). Thus, in the SCN, increased p53 levels lead to repression of Per2 expression, consistent with the finding of elevated Per2 expression in p53−/− mice.p53 modulates circadian behavior of miceNext, we examined whether p53 regulates circadian rhythm behavior of mice. The circadian behavior of inbred male p53−/− and wildtype (C57BL/6) mice was analyzed by monitoring their wheel running activity. Both p53−/− and wildtype mice entrained to an LD cycle and were rhythmic throughout our studies. The p53−/− mouse period lengths were 22.8 ± 0.1 h (mean ± SEM, n=12) and were significantly shorter than wildtype period lengths of 23.3 ± 0.1 h (mean, ± SEM, n=10) (Figure 4a, 4b). In addition, the p53−/− mice had a more widely distributed period that exhibited sudden and unpredictable changes during free-running conditions (Figure 4a, 4c). To assess this phenotype further, the same p53−/− and wildtype mice were re-entrained to an LD cycle followed by a second release into free-running conditions. For wildtype mice, the changes in period measured from the second release into DD were all within 0.06 ± 0.01 h (mean ± SEM, n=8) of their first measurements. In contrast, p53−/− mice had periods that differed significantly from the first measurement (Figure 4c, 4d). Both lengthening and shortening of periods were observed. Comparing the longest and shortest period length of an individual mouse measured by linear regression sampled over at least 10 days revealed an average difference of 0.43 ± 0.1 h (mean ±SEM, n=7) for p53−/− mice.Phase shift behavior in response to a light pulse of a specific circadian time (CT) is an intrinsic clock control behavior26. Mice in the free-running state were given a 15 min light pulse at either CT14 or CT22 to measure their phase delay or phase advance response, respectively. The level of phase delay in response to a CT14 light pulse averaged −1.4 ± 0.1 h (mean ± SEM n=3) in wildtype, whereas p53−/− mice displayed a significantly enhanced phase delay response that averaged −2.3 ± 0.1 h (mean ± SEM n=4) (Figure 4e, 4f). When the light pulse was given at CT22, wildtype mice displayed a phase advance response that averaged +0.8 h± 0.3 h (mean ± SEM n=3). Surprisingly, the p53−/− mice displayed no phase advance but instead exhibited a phase delay response that averaged −0.9 h ± 0.1 h (mean ± SEM n=5) (Figure 4g, 4h). Thus, the circadian period of p53−/− compared with wildtype mice is shorter, unstable and its phase shift response to a light cue displays an enhanced phase delay response during free running conditions.Nutlin-3 phase shifts the circadian behavior of miceNutlin-3 is currently in clinical trials as an anti-cancer drug27. Preclinical studies showed that Nutlin-3 has apparent low toxicity and can be given daily to mice17. Our molecular studies raised the possibility that Nutlin-3 can alter Per2 expression via stabilization of p53 and thus reset the circadian clock. To investigate this possibility, we examined whether Nutlin-3 or vehicle (DMSO) can phase shift circadian rhythm in wildtype mice during free-running DD conditions. CT9 and CT21 were chosen due to the approximate peak and nadir levels of endogenous p53 in the SCN, respectively. Both vehicle and Nutlin-3 treated mice displayed a temporary reduction of locomotor activity although the effect of Nutlin-3 persisted longer in some animals. However, no phase shift response was observed in vehicle treated wildtype or Nutlin-3 treated p53−/− mice whether treated at CT9 or CT21 (Figure 5a, 5b and Supplementary Figure S6). In contrast, a phase advance response was observed when wildtype mice were treated with Nutlin-3 at CT9 or CT21. The effect of Nutlin-3 was also examined using a different protocol for phase shift response in the presence of a lighting cue. In this protocol, delivery of vehicle or Nutlin-3 was carried out on the last day of the LD cycle entrainment followed by free-running conditions28. This protocol showed that vehicle treated mice did not display phase shift behavior while Nutlin-3 given at ZT9 induced a robust phase advance response (Figure 5c, 5d, 5e). In contrast, p53−/− mice treated with vehicle or Nutlin-3 displayed no phase shift response. These results show that the clock phase advance induced by Nutlin-3 treatment is mediated through stabilization of p53 and is consistent with light pulse studies of p53−/− mice showing enhanced phase delay response.

p53 represses BMAL1/CLOCK-mediated transcription of Per2A number of PML-associated proteins were screened for their potential ability to modulate BMAL1/CLOCK mediated Per2 promoter transcription. Expression constructs for p53, WRN, RB, and RARα were individually examined using a Per2-Luc reporter assay for their ability to up or down regulate BMAL1/CLOCK mediated transcription. Among these expression constructs, we observed that the one encoding p53 strongly repressed BMAL1/CLOCK mediated transcription of the Per2 promoter (Figure 1a). Therefore, we asked whether this apparent repression of Per2 transcription by p53 was mediated through BMAL1/CLOCK. High levels of p53 can induce apoptosis of tumor cell lines and this could result in an apparent non-specific transcriptional repression16. To rule out this possibility, we compared wildtype mouse embryonic fibroblasts (MEF) with HeLa cells using a Luc-reporter assay driven by either the Bmal1 or Per2 promoter. In both wildtype MEF and HeLa cells, the expression of p53 did not repress RORγ mediated transcription of the Bmal1 promoter but repressed Per2 promoter transcription mediated by BMAL1/CLOCK (Figure 1b and Supplementary Figure S1).The endogenous p53 level is stabilized and enhanced by Nutlin-3, a small molecule inhibitor of MDM217. MDM2 is the major negative regulator of p53 and facilitates p53 degradation18. BMAL1/CLOCK-mediated transcription of the Per2 promoter was repressed in wildtype, but not in p53−/− MEF cells when treated with Nutlin-3 (Figure 1c). Transfection of a p53 expression construct into p53−/− MEF cells restored the repression of BMAL1/CLOCK-mediated transcription of the Per2 promoter in the presence of Nutlin-3. Western analysis with anti-p53 and anti-PER2 antibodies showed a time-dependent increase in the levels of endogenous p53, accompanied by a reciprocal decrease in endogenous PER2 in wildtype MEF cells following Nutlin-3 treatment (Figure 1d). No decrease in PER2 levels by Nutlin-3 treatment was observed in p53−/− MEF cells (Figure 1e). This inverse relationship between p53 and PER2 levels was further examined in a Nutlin-3 titration experiment. After a 16 h treatment, endogenous p53 levels correlated with increases in Nutlin-3 concentration, and were inversely correlated with PER2 levels in wildtype MEF cells (Figure 1f). No change in PER2 levels was observed when p53−/− MEF cells were treated with Nutlin-3 at a similar concentration (Figure 1g). Genotoxic stress such as γ-irradiation is known to rapidly stabilize p53 protein in some cells, including MEF and human osteosarcoma U2OS cells19. Western analysis of human U2OS cells and wildtype MEF treated with γ-irradiation displayed a rapid rise in p53 and a reciprocal decline in PER2 levels (Figure 1h, Supplementary Figure S2). This decline in PER2 level was not observed in p53−/− MEF cells that were similarly treated with γ-irradiation (Figure 1i). Together, these studies show that the cellular level of PER2 is inversely correlated with that of p53.

p53 represses BMAL1/CLOCK-mediated transcription of Per2

A number of PML-associated proteins were screened for their potential ability to modulate BMAL1/CLOCK mediated Per2 promoter transcription. Expression constructs for p53, WRN, RB, and RARα were individually examined using a Per2-Luc reporter assay for their ability to up or down regulate BMAL1/CLOCK mediated transcription. Among these expression constructs, we observed that the one encoding p53 strongly repressed BMAL1/CLOCK mediated transcription of the Per2 promoter (Figure 1a). Therefore, we asked whether this apparent repression of Per2 transcription by p53 was mediated through BMAL1/CLOCK. High levels of p53 can induce apoptosis of tumor cell lines and this could result in an apparent non-specific transcriptional repression16. To rule out this possibility, we compared wildtype mouse embryonic fibroblasts (MEF) with HeLa cells using a Luc-reporter assay driven by either the Bmal1 or Per2 promoter. In both wildtype MEF and HeLa cells, the expression of p53 did not repress RORγ mediated transcription of the Bmal1 promoter but repressed Per2 promoter transcription mediated by BMAL1/CLOCK (Figure 1b and Supplementary Figure S1).

The endogenous p53 level is stabilized and enhanced by Nutlin-3, a small molecule inhibitor of MDM217. MDM2 is the major negative regulator of p53 and facilitates p53 degradation18. BMAL1/CLOCK-mediated transcription of the Per2 promoter was repressed in wildtype, but not in p53−/− MEF cells when treated with Nutlin-3 (Figure 1c). Transfection of a p53 expression construct into p53−/− MEF cells restored the repression of BMAL1/CLOCK-mediated transcription of the Per2 promoter in the presence of Nutlin-3. Western analysis with anti-p53 and anti-PER2 antibodies showed a time-dependent increase in the levels of endogenous p53, accompanied by a reciprocal decrease in endogenous PER2 in wildtype MEF cells following Nutlin-3 treatment (Figure 1d). No decrease in PER2 levels by Nutlin-3 treatment was observed in p53−/− MEF cells (Figure 1e). This inverse relationship between p53 and PER2 levels was further examined in a Nutlin-3 titration experiment. After a 16 h treatment, endogenous p53 levels correlated with increases in Nutlin-3 concentration, and were inversely correlated with PER2 levels in wildtype MEF cells (Figure 1f). No change in PER2 levels was observed when p53−/− MEF cells were treated with Nutlin-3 at a similar concentration (Figure 1g). Genotoxic stress such as γ-irradiation is known to rapidly stabilize p53 protein in some cells, including MEF and human osteosarcoma U2OS cells19. Western analysis of human U2OS cells and wildtype MEF treated with γ-irradiation displayed a rapid rise in p53 and a reciprocal decline in PER2 levels (Figure 1h, Supplementary Figure S2). This decline in PER2 level was not observed in p53−/− MEF cells that were similarly treated with γ-irradiation (Figure 1i). Together, these studies show that the cellular level of PER2 is inversely correlated with that of p53.

The Per2 promoter contains a conserved p53 response elementThe inverse correlation between p53 and PER2 suggests that p53 is repressing BMAL1/CLOCK mediated transcription of the Per2 promoter. To map the putative response element for p53, BMAL1/CLOCK mediated transcription, reporter assays were carried out using serial deletion constructs of the mouse Per2 promoter at −2811 to −105 bp from the transcription start site (Figure 2a). The repression by p53 of BMAL1/CLOCK mediated transcription of the Per2 promoter was intact in all of these deletion constructs suggesting that the putative p53 response element is within the −105 bp region of the Per2 promoter. Using the NBLAST program, comparison of this 105 bp mouse sequence with Per2 promoter sequences of human, dog and rat from Genebank revealed a highly conserved 48 bp sequence (Figure 2b). Sequence analysis with the TFBIND program revealed a putative p53 response element. The p53 response element is a motif of two tandem decamers separated by 0–13 nucleotides. The consensus sequences can be depicted as 5'-RRXCXXGXYX-N(0–13)-XRXCXXGXYY-3', where R= A/G, Y= C/T and X= A/G/C/T 20. This putative p53 response element is upstream to the transcription start site of the Per2 promoter, and its proximal decamer overlaps with the critical E-box sequence for BMAL1/CLOCK binding. Mutation of nucleotides adjacent to the E-box severely dampened BMAL1/CLOCK mediated transcription of the Per2 promoter. These observations are consistent with previous findings that BMAL1/CLOCK mediated transcription of the Per2 promoter requires a segment about 20 bases from the transcription start site21. Interestingly, mutation of nucleotides in the distal decamer of the putative p53 response element did not significantly affect BMAL1/CLOCK mediated transcription while the strong repression by p53 was relieved (Figure 2b, 2c).To further characterize the region upstream of the Per2 promoter initiation site, sense and antisense oligonucleotides corresponding to the mouse Per2 promoter sequence from +7 to −45 bp were synthesized, annealed and cloned into a luciferase reporter vector to generate −45bpPer2-Luc (Figure 2b). Transcriptional activation by BMAL1/CLOCK and repression by p53 of expression from the −45bpPer2-Luc and −105bpPer2-Luc reporter constructs were comparable, indicating that the −45bpPer2 DNA promoter sequence contained both the p53 response element and the E-box of BMAL1/CLOCK (Figure 2d).Using an electrophoretic mobility shift assay (EMSA) we observed that recombinant p53 protein binds to −45bpPer2 DNA (Figure 2e). The binding of p53 to biotin-labeled −45bpPer2 DNA was competitively reduced by unlabeled −45bpPer2 DNA. Some antibodies to p53 are known to induce a super-shift of the p53/DNA complex. We tested two commercial p53 antibodies and they both generated the expected super-shifts (Figure 2e, Supplementary Figure S3). Further, the super-shift was competitively reduced by unlabeled −45bpPer2 DNA. However, when the unlabeled −45bpPer2 DNA contained a mutated sequence in the distal decamer, the super-shift binding was not reduced (Figure 2f).The Per2 genomic locus has five described E-box elements (E1-E5), and the one most critical for BMAL1/CLOCK binding is E2 located in the promoter region22. Chromatin immuno-precipitation (ChIP) assays with anti-p53 antibodies were performed using genomic DNA that had been fragmented to 0.5 Kb on average. ChIP-qPCR analysis revealed that p53 pull-down of E2 was significantly enhanced in wildtype when cells were treated with Nutlin-3 (Figure 2g). In contrast, p53 pull-down of E5, about 3 Kb away from E2, was not significant (Supplementary Figure S4). Next, we determined whether BMAL1/CLOCK binding to the E2 region of the Per2 promoter was disrupted by p53. Transfection of p53−/− MEF with expression constructs for FLAG-tagged Clock, Bmal1 in the presence or absence of a p53 expression construct was carried out. ChIP-qPCR analysis showed that in the absence of p53, pull down of FLAG-CLOCK by the anti-FLAG antibody was significantly higher (Supplementary Figure S5).We then investigated whether p53 binding to the E2 region of the Per2 promoter also occurs in the SCN in vivo. SCN from wildtype mouse brains were dissected at six zeitgeber time (ZT) points over 24 h for ChIP analysis. ChIP-qPCR with anti-p53 antibodies showed a peak level of endogenous p53 binding to the Per2 promoter in the SCN between ZT4-ZT8 and a nadir at ZT16-ZT20 (Figure 2h). These studies demonstrate that p53 is expressed in the SCN and its interaction with the Per2 promoter is under temporal control in vivo. Next, the level of p53 or BMAL1/CLOCK binding to the E2 region of the Per2 promoter was examined using wild type and p53−/− mouse thymus obtained at ZT8 and ZT20. Thymus was chosen since it expresses detectable levels of p53 under normal physiological conditions. ChIP-qPCR analysis revealed that p53 binding to the E2 region was about 2.5X higher at ZT8 than ZT20 (Figure 2i). BMAL1 binding to the E2 region in p53−/− MEF was significantly elevated compared with wildtype (Figure 2j).To investigate whether p53 directly competes with BMAL1/CLOCK for binding to −45bpPer2 DNA, nuclear extracts from WT, p53−/− MEF and Bmal1−/− fibroblasts were prepared for EMSA. Unlike nuclear extracts from WT and p53−/− MEF, those from Bmal1−/− fibroblasts did not display a prominent protein/DNA complex on EMSA with −45bpPer2 DNA indicating that this complex is BMAL1 dependent (Figure 2k). Further, transfection of expression plasmids for Bmal1 and Flag-Clock into Bmal1−/− fibroblasts restored the BMAL1/CLOCK:-45bpPer2 protein/DNA complex in nuclear extracts. Treatment with either anti-BMAL1 or anti-FLAG antibodies significantly reduced this protein/DNA complex. Consistent with results from ChIP-qPCR analysis, nuclear extracts from p53−/− MEF and thymus showed an increased level of the BMAL1/CLOCK:-45bpPer2 protein/DNA complex when compared to wildtype samples (Figure 2k, 2l). Adding recombinant p53 to the p53−/− nuclear extract reduced the level of the BMAL1/CLOCK:-45bpPer2 DNA complex (Figure 2m). This observation shows that p53 competes with BMAL1/CLOCK for binding to the DNA probe. Collectively, these studies show that the Per2 promoter contains a functional p53 response element that overlaps with the E-box for BMAL1/CLOCK binding that is critical for transcription of the Per2 gene.

The Per2 promoter contains a conserved p53 response element

The inverse correlation between p53 and PER2 suggests that p53 is repressing BMAL1/CLOCK mediated transcription of the Per2 promoter. To map the putative response element for p53, BMAL1/CLOCK mediated transcription, reporter assays were carried out using serial deletion constructs of the mouse Per2 promoter at −2811 to −105 bp from the transcription start site (Figure 2a). The repression by p53 of BMAL1/CLOCK mediated transcription of the Per2 promoter was intact in all of these deletion constructs suggesting that the putative p53 response element is within the −105 bp region of the Per2 promoter. Using the NBLAST program, comparison of this 105 bp mouse sequence with Per2 promoter sequences of human, dog and rat from Genebank revealed a highly conserved 48 bp sequence (Figure 2b). Sequence analysis with the TFBIND program revealed a putative p53 response element. The p53 response element is a motif of two tandem decamers separated by 0–13 nucleotides. The consensus sequences can be depicted as 5'-RRXCXXGXYX-N(0–13)-XRXCXXGXYY-3', where R= A/G, Y= C/T and X= A/G/C/T 20. This putative p53 response element is upstream to the transcription start site of the Per2 promoter, and its proximal decamer overlaps with the critical E-box sequence for BMAL1/CLOCK binding. Mutation of nucleotides adjacent to the E-box severely dampened BMAL1/CLOCK mediated transcription of the Per2 promoter. These observations are consistent with previous findings that BMAL1/CLOCK mediated transcription of the Per2 promoter requires a segment about 20 bases from the transcription start site21. Interestingly, mutation of nucleotides in the distal decamer of the putative p53 response element did not significantly affect BMAL1/CLOCK mediated transcription while the strong repression by p53 was relieved (Figure 2b, 2c).

To further characterize the region upstream of the Per2 promoter initiation site, sense and antisense oligonucleotides corresponding to the mouse Per2 promoter sequence from +7 to −45 bp were synthesized, annealed and cloned into a luciferase reporter vector to generate −45bpPer2-Luc (Figure 2b). Transcriptional activation by BMAL1/CLOCK and repression by p53 of expression from the −45bpPer2-Luc and −105bpPer2-Luc reporter constructs were comparable, indicating that the −45bpPer2 DNA promoter sequence contained both the p53 response element and the E-box of BMAL1/CLOCK (Figure 2d).

Using an electrophoretic mobility shift assay (EMSA) we observed that recombinant p53 protein binds to −45bpPer2 DNA (Figure 2e). The binding of p53 to biotin-labeled −45bpPer2 DNA was competitively reduced by unlabeled −45bpPer2 DNA. Some antibodies to p53 are known to induce a super-shift of the p53/DNA complex. We tested two commercial p53 antibodies and they both generated the expected super-shifts (Figure 2e, Supplementary Figure S3). Further, the super-shift was competitively reduced by unlabeled −45bpPer2 DNA. However, when the unlabeled −45bpPer2 DNA contained a mutated sequence in the distal decamer, the super-shift binding was not reduced (Figure 2f).

The Per2 genomic locus has five described E-box elements (E1-E5), and the one most critical for BMAL1/CLOCK binding is E2 located in the promoter region22. Chromatin immuno-precipitation (ChIP) assays with anti-p53 antibodies were performed using genomic DNA that had been fragmented to 0.5 Kb on average. ChIP-qPCR analysis revealed that p53 pull-down of E2 was significantly enhanced in wildtype when cells were treated with Nutlin-3 (Figure 2g). In contrast, p53 pull-down of E5, about 3 Kb away from E2, was not significant (Supplementary Figure S4). Next, we determined whether BMAL1/CLOCK binding to the E2 region of the Per2 promoter was disrupted by p53. Transfection of p53−/− MEF with expression constructs for FLAG-tagged Clock, Bmal1 in the presence or absence of a p53 expression construct was carried out. ChIP-qPCR analysis showed that in the absence of p53, pull down of FLAG-CLOCK by the anti-FLAG antibody was significantly higher (Supplementary Figure S5).

We then investigated whether p53 binding to the E2 region of the Per2 promoter also occurs in the SCN in vivo. SCN from wildtype mouse brains were dissected at six zeitgeber time (ZT) points over 24 h for ChIP analysis. ChIP-qPCR with anti-p53 antibodies showed a peak level of endogenous p53 binding to the Per2 promoter in the SCN between ZT4-ZT8 and a nadir at ZT16-ZT20 (Figure 2h). These studies demonstrate that p53 is expressed in the SCN and its interaction with the Per2 promoter is under temporal control in vivo. Next, the level of p53 or BMAL1/CLOCK binding to the E2 region of the Per2 promoter was examined using wild type and p53−/− mouse thymus obtained at ZT8 and ZT20. Thymus was chosen since it expresses detectable levels of p53 under normal physiological conditions. ChIP-qPCR analysis revealed that p53 binding to the E2 region was about 2.5X higher at ZT8 than ZT20 (Figure 2i). BMAL1 binding to the E2 region in p53−/− MEF was significantly elevated compared with wildtype (Figure 2j).

To investigate whether p53 directly competes with BMAL1/CLOCK for binding to −45bpPer2 DNA, nuclear extracts from WT, p53−/− MEF and Bmal1−/− fibroblasts were prepared for EMSA. Unlike nuclear extracts from WT and p53−/− MEF, those from Bmal1−/− fibroblasts did not display a prominent protein/DNA complex on EMSA with −45bpPer2 DNA indicating that this complex is BMAL1 dependent (Figure 2k). Further, transfection of expression plasmids for Bmal1 and Flag-Clock into Bmal1−/− fibroblasts restored the BMAL1/CLOCK:-45bpPer2 protein/DNA complex in nuclear extracts. Treatment with either anti-BMAL1 or anti-FLAG antibodies significantly reduced this protein/DNA complex. Consistent with results from ChIP-qPCR analysis, nuclear extracts from p53−/− MEF and thymus showed an increased level of the BMAL1/CLOCK:-45bpPer2 protein/DNA complex when compared to wildtype samples (Figure 2k, 2l). Adding recombinant p53 to the p53−/− nuclear extract reduced the level of the BMAL1/CLOCK:-45bpPer2 DNA complex (Figure 2m). This observation shows that p53 competes with BMAL1/CLOCK for binding to the DNA probe. Collectively, these studies show that the Per2 promoter contains a functional p53 response element that overlaps with the E-box for BMAL1/CLOCK binding that is critical for transcription of the Per2 gene.

p53 levels regulate Per2 transcription in the SCNTo gain insight into the relevance of the p53 response element in the Per2 promoter to clock function, p53 levels in the SCN, the master clock structure in mammals, were investigated under temporal conditions. Previous studies have shown that in peripheral tissues, the p53 protein level is under temporal control23–25. Immunofluorescence analysis with anti-p53 antibodies showed that the level of p53 in the SCN from wildtype mice peaks at ZT4-ZT8 and reaches a nadir at ZT16-ZT20 (Figure 3a, 3b). There was no p53 staining in the SCN of p53−/− mice. This observation is consistent with results from the ChIP assay that showed peak levels of p53 binding to the Per2 promoter occurred at about ZT4-ZT8.Next, we examined whether the loss of p53 elevates Per2 expression in the SCN as predicted by our in vitro studies. In situ hybridization studies showed that Per2 transcript levels in the SCN of p53−/− mice were significantly elevated compared to wildtype mice at most sampling times (Figure 3c, 3d). Interestingly, the expression of Per2 transcripts in p53−/− mice remained under temporal control and was apparently phase advanced, reaching peak levels at about ZT8 and a nadir at about ZT20. Consistent with results from in situ analysis, immunofluorescence studies with anti-PER2 antibodies showed that the level of PER2 was significantly elevated at ZT0 and ZT12 in the SCN of p53−/− mice compared to wildtype mice (Figure 3e).Next, we examined whether increasing p53 protein levels by Nutlin-3 treatment would have the converse effect of repressing Per2 transcription in the SCN. First, we investigated whether Nutlin-3 could stabilize p53 levels in the SCN. Wildtype mice were administered Nutlin-3 or vehicle (DMSO) at ZT9 and then sacrificed at ZT14. We selected ZT14 for these studies since endogenous SCN Per2 and p53 protein levels are close to their peak and nadir, respectively at this time point. Immunofluorescence analysis with anti-p53 antibodies showed increased p53 protein in the SCN after treatment with Nutlin-3 but not vehicle (Figure 3f, 3g). In situ hybridization with an antisense Per2 probe showed that endogenous Per2 expression in the SCN of mice treated at ZT14 was significantly dampened after treatment with Nutlin-3 compared with vehicle (Figure 3f, 3g). Thus, in the SCN, increased p53 levels lead to repression of Per2 expression, consistent with the finding of elevated Per2 expression in p53−/− mice.

p53 levels regulate Per2 transcription in the SCN

To gain insight into the relevance of the p53 response element in the Per2 promoter to clock function, p53 levels in the SCN, the master clock structure in mammals, were investigated under temporal conditions. Previous studies have shown that in peripheral tissues, the p53 protein level is under temporal control23–25. Immunofluorescence analysis with anti-p53 antibodies showed that the level of p53 in the SCN from wildtype mice peaks at ZT4-ZT8 and reaches a nadir at ZT16-ZT20 (Figure 3a, 3b). There was no p53 staining in the SCN of p53−/− mice. This observation is consistent with results from the ChIP assay that showed peak levels of p53 binding to the Per2 promoter occurred at about ZT4-ZT8.

Next, we examined whether the loss of p53 elevates Per2 expression in the SCN as predicted by our in vitro studies. In situ hybridization studies showed that Per2 transcript levels in the SCN of p53−/− mice were significantly elevated compared to wildtype mice at most sampling times (Figure 3c, 3d). Interestingly, the expression of Per2 transcripts in p53−/− mice remained under temporal control and was apparently phase advanced, reaching peak levels at about ZT8 and a nadir at about ZT20. Consistent with results from in situ analysis, immunofluorescence studies with anti-PER2 antibodies showed that the level of PER2 was significantly elevated at ZT0 and ZT12 in the SCN of p53−/− mice compared to wildtype mice (Figure 3e).

Next, we examined whether increasing p53 protein levels by Nutlin-3 treatment would have the converse effect of repressing Per2 transcription in the SCN. First, we investigated whether Nutlin-3 could stabilize p53 levels in the SCN. Wildtype mice were administered Nutlin-3 or vehicle (DMSO) at ZT9 and then sacrificed at ZT14. We selected ZT14 for these studies since endogenous SCN Per2 and p53 protein levels are close to their peak and nadir, respectively at this time point. Immunofluorescence analysis with anti-p53 antibodies showed increased p53 protein in the SCN after treatment with Nutlin-3 but not vehicle (Figure 3f, 3g). In situ hybridization with an antisense Per2 probe showed that endogenous Per2 expression in the SCN of mice treated at ZT14 was significantly dampened after treatment with Nutlin-3 compared with vehicle (Figure 3f, 3g). Thus, in the SCN, increased p53 levels lead to repression of Per2 expression, consistent with the finding of elevated Per2 expression in p53−/− mice.

p53 modulates circadian behavior of miceNext, we examined whether p53 regulates circadian rhythm behavior of mice. The circadian behavior of inbred male p53−/− and wildtype (C57BL/6) mice was analyzed by monitoring their wheel running activity. Both p53−/− and wildtype mice entrained to an LD cycle and were rhythmic throughout our studies. The p53−/− mouse period lengths were 22.8 ± 0.1 h (mean ± SEM, n=12) and were significantly shorter than wildtype period lengths of 23.3 ± 0.1 h (mean, ± SEM, n=10) (Figure 4a, 4b). In addition, the p53−/− mice had a more widely distributed period that exhibited sudden and unpredictable changes during free-running conditions (Figure 4a, 4c). To assess this phenotype further, the same p53−/− and wildtype mice were re-entrained to an LD cycle followed by a second release into free-running conditions. For wildtype mice, the changes in period measured from the second release into DD were all within 0.06 ± 0.01 h (mean ± SEM, n=8) of their first measurements. In contrast, p53−/− mice had periods that differed significantly from the first measurement (Figure 4c, 4d). Both lengthening and shortening of periods were observed. Comparing the longest and shortest period length of an individual mouse measured by linear regression sampled over at least 10 days revealed an average difference of 0.43 ± 0.1 h (mean ±SEM, n=7) for p53−/− mice.Phase shift behavior in response to a light pulse of a specific circadian time (CT) is an intrinsic clock control behavior26. Mice in the free-running state were given a 15 min light pulse at either CT14 or CT22 to measure their phase delay or phase advance response, respectively. The level of phase delay in response to a CT14 light pulse averaged −1.4 ± 0.1 h (mean ± SEM n=3) in wildtype, whereas p53−/− mice displayed a significantly enhanced phase delay response that averaged −2.3 ± 0.1 h (mean ± SEM n=4) (Figure 4e, 4f). When the light pulse was given at CT22, wildtype mice displayed a phase advance response that averaged +0.8 h± 0.3 h (mean ± SEM n=3). Surprisingly, the p53−/− mice displayed no phase advance but instead exhibited a phase delay response that averaged −0.9 h ± 0.1 h (mean ± SEM n=5) (Figure 4g, 4h). Thus, the circadian period of p53−/− compared with wildtype mice is shorter, unstable and its phase shift response to a light cue displays an enhanced phase delay response during free running conditions.

p53 modulates circadian behavior of mice

Next, we examined whether p53 regulates circadian rhythm behavior of mice. The circadian behavior of inbred male p53−/− and wildtype (C57BL/6) mice was analyzed by monitoring their wheel running activity. Both p53−/− and wildtype mice entrained to an LD cycle and were rhythmic throughout our studies. The p53−/− mouse period lengths were 22.8 ± 0.1 h (mean ± SEM, n=12) and were significantly shorter than wildtype period lengths of 23.3 ± 0.1 h (mean, ± SEM, n=10) (Figure 4a, 4b). In addition, the p53−/− mice had a more widely distributed period that exhibited sudden and unpredictable changes during free-running conditions (Figure 4a, 4c). To assess this phenotype further, the same p53−/− and wildtype mice were re-entrained to an LD cycle followed by a second release into free-running conditions. For wildtype mice, the changes in period measured from the second release into DD were all within 0.06 ± 0.01 h (mean ± SEM, n=8) of their first measurements. In contrast, p53−/− mice had periods that differed significantly from the first measurement (Figure 4c, 4d). Both lengthening and shortening of periods were observed. Comparing the longest and shortest period length of an individual mouse measured by linear regression sampled over at least 10 days revealed an average difference of 0.43 ± 0.1 h (mean ±SEM, n=7) for p53−/− mice.

Phase shift behavior in response to a light pulse of a specific circadian time (CT) is an intrinsic clock control behavior26. Mice in the free-running state were given a 15 min light pulse at either CT14 or CT22 to measure their phase delay or phase advance response, respectively. The level of phase delay in response to a CT14 light pulse averaged −1.4 ± 0.1 h (mean ± SEM n=3) in wildtype, whereas p53−/− mice displayed a significantly enhanced phase delay response that averaged −2.3 ± 0.1 h (mean ± SEM n=4) (Figure 4e, 4f). When the light pulse was given at CT22, wildtype mice displayed a phase advance response that averaged +0.8 h± 0.3 h (mean ± SEM n=3). Surprisingly, the p53−/− mice displayed no phase advance but instead exhibited a phase delay response that averaged −0.9 h ± 0.1 h (mean ± SEM n=5) (Figure 4g, 4h). Thus, the circadian period of p53−/− compared with wildtype mice is shorter, unstable and its phase shift response to a light cue displays an enhanced phase delay response during free running conditions.

Nutlin-3 phase shifts the circadian behavior of miceNutlin-3 is currently in clinical trials as an anti-cancer drug27. Preclinical studies showed that Nutlin-3 has apparent low toxicity and can be given daily to mice17. Our molecular studies raised the possibility that Nutlin-3 can alter Per2 expression via stabilization of p53 and thus reset the circadian clock. To investigate this possibility, we examined whether Nutlin-3 or vehicle (DMSO) can phase shift circadian rhythm in wildtype mice during free-running DD conditions. CT9 and CT21 were chosen due to the approximate peak and nadir levels of endogenous p53 in the SCN, respectively. Both vehicle and Nutlin-3 treated mice displayed a temporary reduction of locomotor activity although the effect of Nutlin-3 persisted longer in some animals. However, no phase shift response was observed in vehicle treated wildtype or Nutlin-3 treated p53−/− mice whether treated at CT9 or CT21 (Figure 5a, 5b and Supplementary Figure S6). In contrast, a phase advance response was observed when wildtype mice were treated with Nutlin-3 at CT9 or CT21. The effect of Nutlin-3 was also examined using a different protocol for phase shift response in the presence of a lighting cue. In this protocol, delivery of vehicle or Nutlin-3 was carried out on the last day of the LD cycle entrainment followed by free-running conditions28. This protocol showed that vehicle treated mice did not display phase shift behavior while Nutlin-3 given at ZT9 induced a robust phase advance response (Figure 5c, 5d, 5e). In contrast, p53−/− mice treated with vehicle or Nutlin-3 displayed no phase shift response. These results show that the clock phase advance induced by Nutlin-3 treatment is mediated through stabilization of p53 and is consistent with light pulse studies of p53−/− mice showing enhanced phase delay response.

Nutlin-3 phase shifts the circadian behavior of mice

Nutlin-3 is currently in clinical trials as an anti-cancer drug27. Preclinical studies showed that Nutlin-3 has apparent low toxicity and can be given daily to mice17. Our molecular studies raised the possibility that Nutlin-3 can alter Per2 expression via stabilization of p53 and thus reset the circadian clock. To investigate this possibility, we examined whether Nutlin-3 or vehicle (DMSO) can phase shift circadian rhythm in wildtype mice during free-running DD conditions. CT9 and CT21 were chosen due to the approximate peak and nadir levels of endogenous p53 in the SCN, respectively. Both vehicle and Nutlin-3 treated mice displayed a temporary reduction of locomotor activity although the effect of Nutlin-3 persisted longer in some animals. However, no phase shift response was observed in vehicle treated wildtype or Nutlin-3 treated p53−/− mice whether treated at CT9 or CT21 (Figure 5a, 5b and Supplementary Figure S6). In contrast, a phase advance response was observed when wildtype mice were treated with Nutlin-3 at CT9 or CT21. The effect of Nutlin-3 was also examined using a different protocol for phase shift response in the presence of a lighting cue. In this protocol, delivery of vehicle or Nutlin-3 was carried out on the last day of the LD cycle entrainment followed by free-running conditions28. This protocol showed that vehicle treated mice did not display phase shift behavior while Nutlin-3 given at ZT9 induced a robust phase advance response (Figure 5c, 5d, 5e). In contrast, p53−/− mice treated with vehicle or Nutlin-3 displayed no phase shift response. These results show that the clock phase advance induced by Nutlin-3 treatment is mediated through stabilization of p53 and is consistent with light pulse studies of p53−/− mice showing enhanced phase delay response.

DiscussionThe circadian clock mechanism governs the rhythmic activities that allow a living organism to anticipate and respond to opportunities and challenges presented in a daily diurnal rhythmic fashion. For this coordinated control to happen at all levels, from physiological processes to molecular signaling pathways, there must be a pervasive and effective mechanistic connection between the clock network and other cellular networks. It is now recognized that the clock mechanism or specific components of the clock are involved in responses to cellular stress including DNA damage response, checkpoint arrest and apoptosis29. p53 regulated genes are involved in diverse biological pathways including the cell cycle, DNA damage response, apoptosis and glucose metabolism4. However, the link between p53 controlled pathways such as the cell cycle and the circadian clock regulators is thought to be indirect or “opportunistic”30.Our present study has shown that p53 directly binds to the Per2 promoter and represses its transcriptional activation by BMAL1/CLOCK. We mapped a p53 response element with two characteristic decamers to an evolutionarily conserved region within −45 bp of the Per2 promoter transcription start site. This p53 response element overlaps with the major E-box element necessary for BMAL1/CLOCK mediated transcription. Both EMSA and reporter assay with −45bpPer2DNA demonstrate p53 binding and repression of BMAL1/CLOCK mediated transcription. Independently, by ChIP-qPCR analysis, we showed that CLOCK/BMAL1 binding to the Per2 promoter is inhibited by the expression of p53. Four modes of p53 transcriptional repression are now recognized20. One of these modes is the overlap of the p53 response element with the binding site of another transcription factor, allowing p53 to compete with that transcription factor for DNA binding (Figure 6). Consistent with this mode of transcriptional repression, EMSA results show that the formation of a BMAL1/CLOCK:-45bpPer2 DNA complex was significantly reduced by addition of recombinant p53.Our studies show that the physiological p53 level oscillates in the SCN of wildtype mice with peak expression around ZT8. In the SCN of p53−/− mice, Per2 expression was significantly elevated but remained under temporal control. These observations suggest that p53 regulates the amplitude of Per2 transcription. Indeed, the p53 response element of the Per2 promoter is within a region previously delineated to be important for the generation of amplitude21. The p53−/− mice display abnormal circadian behavior, exhibiting a shorter period length with reduced precision and altered phase shift response to a light pulse. Interestingly, transgenic mice that overexpress Per2 also display a shorter period length than wildtype mice31. Period length is also controlled by PER2 phosphorylation state since it regulates PER2 turnover32. Our current findings show that when the levels of p53 are elevated by Nutlin-3 treatment, γ-irradiation or by expression construct transfection in cultured cells, Per2 expression is significantly dampened. In the in vivo study, Nutlin-3 injection into mice produced elevated p53 protein and repressed Per2 expression in the SCN, resulting in phase advancement of the circadian clock. These findings suggest that both physiological and elevated levels of p53 regulate Per2 expression. The importance of physiological levels of p53 for other biological processes has been recently recognized. For example, physiological levels of p53 regulate CD44 expression and epithelial proliferation33. In addition, under non-stress conditions, p53 regulates adult neural stem cell proliferation34. Under stress conditions, stabilized p53 represses expression of Per2 that in turn could modulate the expression of other clock genes to help reset the clock. BMAL1/CLOCK mediated Per1 and Cry1 expressions also appear to be repressed by p53 (Supplementary Figure S7), though it is unclear if these genes are direct targets of p53 inhibition or if this is secondary to Per2 repression by p53. It was previously shown that loss of PER2 leads to highly dampened Per1 and Cry1 expression in vivo35.This study shows that p53−/− mice displayed an increased phase delay and an absence of phase advance response to a light pulse at CT14 and CT22, respectively. Mice deficient in PER2 function exhibited enhanced phase advance and a weak phase delay response indicating that PER2 plays a major role in phase shift control36–38. Per2 expression is differentially induced by light over the span of the phase response curve. A light pulse given during early, but not late subjective night, induced Per2 expression in the SCN36,37. These observations suggest that relative to the phase response curve, increased induction of PER2 is associated with phase delay while a lack of induction of PER2 is associated with the phase advance response. This correlation of rapid reduction of PER2 levels with phase shift advance is supported by cell culture studies showing that DNA damage induced by methyl methane sulfonate (MMS) or UV-irradiation caused clock phase advancement as a result of a rapid decrease in PER2 levels39. Genotoxic agents such as UV or γ-irradiation rapidly induce p5319,24,40. Our studies show that γ-irradiation caused rapid p53 stabilization that was associated with decreased PER2 levels. This decline in PER2 was not observed in p53−/− MEF treated with γ-irradiation, indicating that it is a p53 dependent effect. Consistent with these findings, mice treated with γ-irradiation also displayed phase advancement of their circadian clock41. Conversely, it has been observed that known casein kinase 1 (CK1) inhibitors when given during free-running conditions induce a phase delay response in mice42. This phase shift effect of CK1 inhibitors is thought to be caused by increased stabilization of PERIOD proteins particularly PER242,43. Our observation that Nutlin-3 induced p53 mediated repression of Per2 expression and phase advancement of the clock is consistent with a phase advance response resulting from a reduced induction of PER2 levels by a light pulse at ZT22.Previously, it had been observed that mice heterozygous for constitutively active p53 displayed an enhanced aging phenotype that included enhanced loss of soft tissues and development of spinal kyphosis44. Further, overactive p53 activities caused by the expression of the p53 isoform ΔN-p53 (p44) also resulted in a similar premature aging phenotype45,46. Interestingly, similar premature aging phenotypes, including progressive loss of soft tissues and spinal kyphosis, were observed in Per2 deficient mice47,48. These premature aging phenotypes in hyperactive p53 and Per2−/− mice are supportive of our findings that p53 suppresses Per2 expression.Current evidence suggests that there is an inter-regulatory loop between p53, PML and Per2. PML is involved in the regulation a p53 acetylation and function14,15. However, PML is a p53 target49. Both p53 and PML are under circadian control6,25. Our studies showed PML and p53 can directly regulate the circadian clock via Per2. Clearly, the mechanistic detail of their inter-regulation is complex and remains to be fully deciphered.In summary, our earlier study suggested that functional PER2 is important for p53-mediated stress signals to reach the circadian clock network. Here we show that p53 acts as a transcription factor that regulates the circadian clock by direct control of Per2 expression.

The circadian clock mechanism governs the rhythmic activities that allow a living organism to anticipate and respond to opportunities and challenges presented in a daily diurnal rhythmic fashion. For this coordinated control to happen at all levels, from physiological processes to molecular signaling pathways, there must be a pervasive and effective mechanistic connection between the clock network and other cellular networks. It is now recognized that the clock mechanism or specific components of the clock are involved in responses to cellular stress including DNA damage response, checkpoint arrest and apoptosis29. p53 regulated genes are involved in diverse biological pathways including the cell cycle, DNA damage response, apoptosis and glucose metabolism4. However, the link between p53 controlled pathways such as the cell cycle and the circadian clock regulators is thought to be indirect or “opportunistic”30.

Our present study has shown that p53 directly binds to the Per2 promoter and represses its transcriptional activation by BMAL1/CLOCK. We mapped a p53 response element with two characteristic decamers to an evolutionarily conserved region within −45 bp of the Per2 promoter transcription start site. This p53 response element overlaps with the major E-box element necessary for BMAL1/CLOCK mediated transcription. Both EMSA and reporter assay with −45bpPer2DNA demonstrate p53 binding and repression of BMAL1/CLOCK mediated transcription. Independently, by ChIP-qPCR analysis, we showed that CLOCK/BMAL1 binding to the Per2 promoter is inhibited by the expression of p53. Four modes of p53 transcriptional repression are now recognized20. One of these modes is the overlap of the p53 response element with the binding site of another transcription factor, allowing p53 to compete with that transcription factor for DNA binding (Figure 6). Consistent with this mode of transcriptional repression, EMSA results show that the formation of a BMAL1/CLOCK:-45bpPer2 DNA complex was significantly reduced by addition of recombinant p53.

Our studies show that the physiological p53 level oscillates in the SCN of wildtype mice with peak expression around ZT8. In the SCN of p53−/− mice, Per2 expression was significantly elevated but remained under temporal control. These observations suggest that p53 regulates the amplitude of Per2 transcription. Indeed, the p53 response element of the Per2 promoter is within a region previously delineated to be important for the generation of amplitude21. The p53−/− mice display abnormal circadian behavior, exhibiting a shorter period length with reduced precision and altered phase shift response to a light pulse. Interestingly, transgenic mice that overexpress Per2 also display a shorter period length than wildtype mice31. Period length is also controlled by PER2 phosphorylation state since it regulates PER2 turnover32. Our current findings show that when the levels of p53 are elevated by Nutlin-3 treatment, γ-irradiation or by expression construct transfection in cultured cells, Per2 expression is significantly dampened. In the in vivo study, Nutlin-3 injection into mice produced elevated p53 protein and repressed Per2 expression in the SCN, resulting in phase advancement of the circadian clock. These findings suggest that both physiological and elevated levels of p53 regulate Per2 expression. The importance of physiological levels of p53 for other biological processes has been recently recognized. For example, physiological levels of p53 regulate CD44 expression and epithelial proliferation33. In addition, under non-stress conditions, p53 regulates adult neural stem cell proliferation34. Under stress conditions, stabilized p53 represses expression of Per2 that in turn could modulate the expression of other clock genes to help reset the clock. BMAL1/CLOCK mediated Per1 and Cry1 expressions also appear to be repressed by p53 (Supplementary Figure S7), though it is unclear if these genes are direct targets of p53 inhibition or if this is secondary to Per2 repression by p53. It was previously shown that loss of PER2 leads to highly dampened Per1 and Cry1 expression in vivo35.

This study shows that p53−/− mice displayed an increased phase delay and an absence of phase advance response to a light pulse at CT14 and CT22, respectively. Mice deficient in PER2 function exhibited enhanced phase advance and a weak phase delay response indicating that PER2 plays a major role in phase shift control36–38. Per2 expression is differentially induced by light over the span of the phase response curve. A light pulse given during early, but not late subjective night, induced Per2 expression in the SCN36,37. These observations suggest that relative to the phase response curve, increased induction of PER2 is associated with phase delay while a lack of induction of PER2 is associated with the phase advance response. This correlation of rapid reduction of PER2 levels with phase shift advance is supported by cell culture studies showing that DNA damage induced by methyl methane sulfonate (MMS) or UV-irradiation caused clock phase advancement as a result of a rapid decrease in PER2 levels39. Genotoxic agents such as UV or γ-irradiation rapidly induce p5319,24,40. Our studies show that γ-irradiation caused rapid p53 stabilization that was associated with decreased PER2 levels. This decline in PER2 was not observed in p53−/− MEF treated with γ-irradiation, indicating that it is a p53 dependent effect. Consistent with these findings, mice treated with γ-irradiation also displayed phase advancement of their circadian clock41. Conversely, it has been observed that known casein kinase 1 (CK1) inhibitors when given during free-running conditions induce a phase delay response in mice42. This phase shift effect of CK1 inhibitors is thought to be caused by increased stabilization of PERIOD proteins particularly PER242,43. Our observation that Nutlin-3 induced p53 mediated repression of Per2 expression and phase advancement of the clock is consistent with a phase advance response resulting from a reduced induction of PER2 levels by a light pulse at ZT22.

Previously, it had been observed that mice heterozygous for constitutively active p53 displayed an enhanced aging phenotype that included enhanced loss of soft tissues and development of spinal kyphosis44. Further, overactive p53 activities caused by the expression of the p53 isoform ΔN-p53 (p44) also resulted in a similar premature aging phenotype45,46. Interestingly, similar premature aging phenotypes, including progressive loss of soft tissues and spinal kyphosis, were observed in Per2 deficient mice47,48. These premature aging phenotypes in hyperactive p53 and Per2−/− mice are supportive of our findings that p53 suppresses Per2 expression.

Current evidence suggests that there is an inter-regulatory loop between p53, PML and Per2. PML is involved in the regulation a p53 acetylation and function14,15. However, PML is a p53 target49. Both p53 and PML are under circadian control6,25. Our studies showed PML and p53 can directly regulate the circadian clock via Per2. Clearly, the mechanistic detail of their inter-regulation is complex and remains to be fully deciphered.

In summary, our earlier study suggested that functional PER2 is important for p53-mediated stress signals to reach the circadian clock network. Here we show that p53 acts as a transcription factor that regulates the circadian clock by direct control of Per2 expression.

MethodsAnimalsp53 −/− (C57BL/6) mice were obtained from The Jackson Laboratory (#002101). Animals were housed in a standard animal maintenance facility under a 12 h light : 12 h dark cycle. All animal studies were carried under institutional approved animal protocols (HSC-AWC-09–139).Cells and ReagentsWT MEF cells and Trp53−/− MEF cells were obtained from Dr. C. Takahashi (Kanazawa University, Japan) and maintained in DMEM/10 % FCS. Nutlin-3 was purchased from Cayman Chemical (#10009816). The mouse p53 expression vector was purchased from ORIGENE. A mouse anti-α-tubulin monoclonal antibody was purchased from COVANCE (B-5-1–2). The mPer2 antibody was raised as described previously35. Anti-p53 antibodies are from Santa Cruz (FL-393, #sc-6243, DO-1, #sc-126) and from Calbiochem (PAb421, #OP03). The anti-BMAL1 antibody is from Abcam (ab3350). The anti-CLOCK antibody is from Thermo Scientific (PA1–520). The anti-Lamin A antibody is from BD Transduction Laboratories (#612162). mPer2-promoter Luc reporter constructs including −2811, −1982, −661, −386, −105 were kind gifts from Dr. T. Takumi21. The −105 bp Per2-luc reporter construct containing five point mutations was generated by 2-step site directed mutagenesis using QuikChange Site-Directed Mutagenesis Kit (#200519, Stratagene). An intermediate mutant was created by introducing 2 mutations to the p53 binding motif in the Per2 promoter using an oligonucleotide with the sequence 5’-GCGCGCGCAGGGGAGGACTCAGCGCGCGC-3’. Then, an additional 3 mutations were introduced to the same motif using an oligonucleotide with a sequence of 5’-CCAATGGCGCGCGCAGTTGAGGACACAGCGCGCGCGG-3’.Luciferase reporter assayLuciferase reporter assays were performed as previously described50. Briefly, MEF cells were seeded into 12-well plates and transfected with the Per2 luciferase reporter construct or the Bmal1 luciferase reporter construct using the Lipofectamine 2000 (Invitrogen). Co-transfections were performed with the respective expression constructs for Clock, Bmal1, Rorα and p53 Cells were harvested 24 h post-transfection, and luciferase assay was performed with an assay kit obtained from Promega using a TD-20/201 luminometer (Turner Designs).ImmunoblottingCells were lysed in ice-cold lysis buffer containing 50 mM Tris-HCl, pH7.4, 0.1% SDS, 0.25 mM deoxycholate, 150 mM NaCl, 2 mM EGTA, 0.1 mM Na3VO4, 10 mM NaF, 1 mM PMSF, and complete Protease Inhibitor (Roche). The cell lysates were resolved by SDS-PAGE and used for immunoblotting as described previously51. Briefly, the 1st antibody incubation for immunoblotting was performed at 4°C overnight in the blocking buffer (5% milk, 1% goat serum, TBS-T). The antibodies dilution we used were anti-mPer2 (x150,000), anti-p53 (x10,000), anti-alpha-tubulin (x10,000) and anti-Lamin A (x2,500).Electric mobility shift assayElectric mobility shift assay (EMSA) was carried out using a Light-Shift Chemiluminescent EMSA kit (#20148, PIERCE) as described previously52. Briefly, the following double stranded 5’-biotinylated or unlabeled DNA’s were synthesized (SIGMA Aldrich)., Per2 consensus sequence DNA: 5’-CAGGGGCGGGCTCAGCGCGCGCGGTCACGTTTTCCACTATGTGACAGCGGAGG-3’, Per2 mutant promoter DNA: 5’-CAGTTGAGGACACAGCGCGCGCGGTCACGTTTTCCACTATGTGACAGCGGAGG-3’. Per2 consensus −55bp oligo: 5’-ATGGCGCGCGCAGGGGCGGGCTCAGCGCGCGCGGTCACGTTTTCCACTATGTGACAGCGGAGG-3’, Per2 mutant consensus −55bp oligo: 5’- ATGGCGCGCGCAGTTGAGGACACAGCGCGCGCGGTCACGTTTTCCACTATGTGACAGCGGAGG-3’. Recombinant mouse p53 was purchased from Active Motif (#31103) and 2 ng of recombinant p53 was used per reaction. Binding assays were performed in a buffer containing 20 mM HEPES (pH7.6), 30 mM KCl, 1 mM EDTA, 10 mM (NH4)2SO4, 1 mM DTT, 0.2% Tween 20, 1 µg poly [d(A–T)], 0.1 µg poly L- lysine. Anti-p53 antibodies (pAb421, Calbiochem or DO-1, Santa Cruz) were used for supershift assay. For BMAL1/CLOCK binding assay, nuclear extracts were prepared as described previously53. Briefly, cells were washed once with Buffer A containing 10 mM HEPES (pH 7.3), 1.5 mM MgCl2, 10 mM KCl, and 0.5 mM DTT, and resuspended in Buffer A. Cells were homogenized, centrifuged (200rpm, 10min), and nuclei were washed again with Buffer A. Cell pellets were resuspended in Buffer C containing 20 mM HEPES (pH 7.3), 20 % glycerol, 0.42 M NaCl, 1.5 mM MgCl2, 0.2 mM EDTA, 0.5 mM DTT, 0.5 mM phenylmethylsulfonyl fluoride and protease inhibitor cocktail (Roche). Lysates were incubated for 30 min with constant rotation at 4 °C and centrifuged (8000 rpm, 10 min). Supernatants were used as nuclear extracts. Binding assays were performed in a buffer containing 50 mM Tris-HCl (pH 8.0), 100 mM KCl, 1 mM EDTA, 6 mM DTT and 1 µg poly [d(A–T)]54. The anti-BMAL1 antibody (N-20, 2 µg, Santa Cruz) and anti-FLAG M2 antibody (5 µg, SIGMA) were used to detect the specific BMAL1/CLOCK band. The anti-BMAL1 antibody was previously validated by other investigators55. Protein binding reactions were performed in 10 µl scale.Chromatin immunoprecipitation assayChromatin immunoprecipitation (ChIP) assay was performed as described previously56. Briefly, cells or tissues were fixed with 1% formalin for 10 min. Crosslinking was stopped by adding 125 mM glycine. Cells were washed and homogenized using a homogenizer. Homogenized cells were collected by brief centrifugation and lysed with swelling buffer containing 5 mM PIPES (pH8.0), 85 mM KCl, 1% NP40, and protease inhibitor cocktail (# 11697498001, Roche). Nuclear fractions were collected by centrifugation and lysed with nuclear lysis buffer containing 50 mM Tris-HCl (pH 8.0), 10 mM EDTA, 1% SDS, and protease inhibitor cocktail. Sonication was performed to fragment DNA to about 500 bp. After centrifugation, lysates were precleared with protein A agarose (Invitrogen). Pre-cleared samples were diluted with the same volume of IP dilution buffer containing 16.7 mM Tris-HCl (pH 8.0), 0.01% SDS, 1.1% Triton X-100, 1.2 mM EDTA, 167 mM NaCl, and protease inhibitor cocktail. Then, p53 and BMAL1 were immunoprecipitated with anti-p53 antibody (2 µg, sc-6243) or anti-BMAL1 antibody (2 µg, ab3350, abcam) and collected on protein A agarose. After serial washing with dialysis buffer containing 50 mM Tris-HCl (pH8.0) 2 mM EDTA, and 0.2% sarkosyl and IP wash buffer containing 100 mM Tris-HCl (pH9.0), 500 mM LiCl, 1% NP40, 1% Deoxycholate, and protease inhibitor cocktail. p53-DNA complexes were eluted by elution buffer containing 50 mM NaHCO3 and 1% SDS. Eluted samples were mixed with 0.2 M NaCl and incubated at 67 °C overnight, then RNase A treated, and purified. PCR and quantitative real-time PCR were performed using Per2-Ebox2 forward primers: 5’-CAGGTTCCGCCCCGCCAGTAT-3’, reverse primer:5’-GTCGCCCTCCGCTGTCACATAG-3’ and Per2-Ebox5 forward primer: 5’-CTCTGTAGGGTGGAGCGGCGA-3’, reverse primer: 5’-ATCCCCACTGCTCCTTCGCAC-3’. ChIP samples obtained from SCN and thymus were analyzed by real-time PCR (ABI PRISM 7700 system, Applied Biosystems) using Maxima SYBR Green qPCR Master Mix (#0252, Fermentas). Real-time PCR data were analyzed as previously reported57. IgG control antibody was used as a negative control. ΔCt was obtained by Ct experiment sample–., Ct negative control and fold differences were calculated by [2(−ΔCt)].Immunofluorescence staining8–10 week old WT and Trp53−/− mouse brains were collected at the indicated time point, fixed with 4% PFA, and embedded in paraffin. Then 8 µm sections were obtained and stained with anti-p53 antibody (#sc-6243, x50, SantaCruz), or anti-mPER2 antibody (x4,000) and visualized by secondary anti-rabbit IgG Alexa Fluor 555 antibody (#A21429, x1000, Invitrogen).In situ hybridization8–10 week old WT and Trp53−/− mouse brains were collected at the indicated time point, fixed with 4% PFA, and embedded in paraffin. Then 8 µm sections were obtained and in situ hybridization was performed as described previously58 with some modifications. Briefly, paraffin sections were treated with 1 µg/ml protease K in PBS-T (0.1% Tween 20) at 37 °C for 15 min, washed, fixed with 4% performic acid at room temperature for 15 min, washed, and then incubated with digoxigenin-labeled cRNA probes in buffer containing 50% formamide, 1% SDS, 5 x SSC, 50 µg/ml tRNA, and 20 mg/ml heparin at 55°C overnight. The probe for Per2 was described previously59. Hybridized sections were washed with Buffer A containing 50% formamide, 1% SDS, and 5 x SSC twice at 55 °C for 30 min each and then washed with Buffer B containing 50% formamide and 2 x SSC three times at 55 °C for 30 min each. After washing with TBS-T (pH7.6, 0.1% Tween 20) three times, sections were blocked with 0.5% Blocking Reagent (#11096176001, Roche) for 1 h and incubated with anti-alkaline phosphatase-conjugated anti-DIG antibody (#11093274910, x1000, Roche) at 4 °C overnight. Sections were washed three times with TBS-T and immersed in NTMT buffer containing 100 mM Tris-HCl (pH 9.5), 100 mM NaCl, 50 mM MgCl2, 0.1% Tween 20, and 2 mM levamisol. Sections were then developed with NBT/BCIP reagents.Locomoter activityWheel-running activity was monitored as described60. Male wild type and Trp53 −/− mice aged about 4 weeks were used. Nutlin-3 (200mg/kg) or vehicle (DMSO) was injected i.p. at the indicated time point.

Animalsp53 −/− (C57BL/6) mice were obtained from The Jackson Laboratory (#002101). Animals were housed in a standard animal maintenance facility under a 12 h light : 12 h dark cycle. All animal studies were carried under institutional approved animal protocols (HSC-AWC-09–139).

p53 −/− (C57BL/6) mice were obtained from The Jackson Laboratory (#002101). Animals were housed in a standard animal maintenance facility under a 12 h light : 12 h dark cycle. All animal studies were carried under institutional approved animal protocols (HSC-AWC-09–139).

Cells and ReagentsWT MEF cells and Trp53−/− MEF cells were obtained from Dr. C. Takahashi (Kanazawa University, Japan) and maintained in DMEM/10 % FCS. Nutlin-3 was purchased from Cayman Chemical (#10009816). The mouse p53 expression vector was purchased from ORIGENE. A mouse anti-α-tubulin monoclonal antibody was purchased from COVANCE (B-5-1–2). The mPer2 antibody was raised as described previously35. Anti-p53 antibodies are from Santa Cruz (FL-393, #sc-6243, DO-1, #sc-126) and from Calbiochem (PAb421, #OP03). The anti-BMAL1 antibody is from Abcam (ab3350). The anti-CLOCK antibody is from Thermo Scientific (PA1–520). The anti-Lamin A antibody is from BD Transduction Laboratories (#612162). mPer2-promoter Luc reporter constructs including −2811, −1982, −661, −386, −105 were kind gifts from Dr. T. Takumi21. The −105 bp Per2-luc reporter construct containing five point mutations was generated by 2-step site directed mutagenesis using QuikChange Site-Directed Mutagenesis Kit (#200519, Stratagene). An intermediate mutant was created by introducing 2 mutations to the p53 binding motif in the Per2 promoter using an oligonucleotide with the sequence 5’-GCGCGCGCAGGGGAGGACTCAGCGCGCGC-3’. Then, an additional 3 mutations were introduced to the same motif using an oligonucleotide with a sequence of 5’-CCAATGGCGCGCGCAGTTGAGGACACAGCGCGCGCGG-3’.

Cells and Reagents

WT MEF cells and Trp53−/− MEF cells were obtained from Dr. C. Takahashi (Kanazawa University, Japan) and maintained in DMEM/10 % FCS. Nutlin-3 was purchased from Cayman Chemical (#10009816). The mouse p53 expression vector was purchased from ORIGENE. A mouse anti-α-tubulin monoclonal antibody was purchased from COVANCE (B-5-1–2). The mPer2 antibody was raised as described previously35. Anti-p53 antibodies are from Santa Cruz (FL-393, #sc-6243, DO-1, #sc-126) and from Calbiochem (PAb421, #OP03). The anti-BMAL1 antibody is from Abcam (ab3350). The anti-CLOCK antibody is from Thermo Scientific (PA1–520). The anti-Lamin A antibody is from BD Transduction Laboratories (#612162). mPer2-promoter Luc reporter constructs including −2811, −1982, −661, −386, −105 were kind gifts from Dr. T. Takumi21. The −105 bp Per2-luc reporter construct containing five point mutations was generated by 2-step site directed mutagenesis using QuikChange Site-Directed Mutagenesis Kit (#200519, Stratagene). An intermediate mutant was created by introducing 2 mutations to the p53 binding motif in the Per2 promoter using an oligonucleotide with the sequence 5’-GCGCGCGCAGGGGAGGACTCAGCGCGCGC-3’. Then, an additional 3 mutations were introduced to the same motif using an oligonucleotide with a sequence of 5’-CCAATGGCGCGCGCAGTTGAGGACACAGCGCGCGCGG-3’.

Luciferase reporter assayLuciferase reporter assays were performed as previously described50. Briefly, MEF cells were seeded into 12-well plates and transfected with the Per2 luciferase reporter construct or the Bmal1 luciferase reporter construct using the Lipofectamine 2000 (Invitrogen). Co-transfections were performed with the respective expression constructs for Clock, Bmal1, Rorα and p53 Cells were harvested 24 h post-transfection, and luciferase assay was performed with an assay kit obtained from Promega using a TD-20/201 luminometer (Turner Designs).

Luciferase reporter assay

Luciferase reporter assays were performed as previously described50. Briefly, MEF cells were seeded into 12-well plates and transfected with the Per2 luciferase reporter construct or the Bmal1 luciferase reporter construct using the Lipofectamine 2000 (Invitrogen). Co-transfections were performed with the respective expression constructs for Clock, Bmal1, Rorα and p53 Cells were harvested 24 h post-transfection, and luciferase assay was performed with an assay kit obtained from Promega using a TD-20/201 luminometer (Turner Designs).

ImmunoblottingCells were lysed in ice-cold lysis buffer containing 50 mM Tris-HCl, pH7.4, 0.1% SDS, 0.25 mM deoxycholate, 150 mM NaCl, 2 mM EGTA, 0.1 mM Na3VO4, 10 mM NaF, 1 mM PMSF, and complete Protease Inhibitor (Roche). The cell lysates were resolved by SDS-PAGE and used for immunoblotting as described previously51. Briefly, the 1st antibody incubation for immunoblotting was performed at 4°C overnight in the blocking buffer (5% milk, 1% goat serum, TBS-T). The antibodies dilution we used were anti-mPer2 (x150,000), anti-p53 (x10,000), anti-alpha-tubulin (x10,000) and anti-Lamin A (x2,500).

Immunoblotting

Cells were lysed in ice-cold lysis buffer containing 50 mM Tris-HCl, pH7.4, 0.1% SDS, 0.25 mM deoxycholate, 150 mM NaCl, 2 mM EGTA, 0.1 mM Na3VO4, 10 mM NaF, 1 mM PMSF, and complete Protease Inhibitor (Roche). The cell lysates were resolved by SDS-PAGE and used for immunoblotting as described previously51. Briefly, the 1st antibody incubation for immunoblotting was performed at 4°C overnight in the blocking buffer (5% milk, 1% goat serum, TBS-T). The antibodies dilution we used were anti-mPer2 (x150,000), anti-p53 (x10,000), anti-alpha-tubulin (x10,000) and anti-Lamin A (x2,500).

Electric mobility shift assayElectric mobility shift assay (EMSA) was carried out using a Light-Shift Chemiluminescent EMSA kit (#20148, PIERCE) as described previously52. Briefly, the following double stranded 5’-biotinylated or unlabeled DNA’s were synthesized (SIGMA Aldrich)., Per2 consensus sequence DNA: 5’-CAGGGGCGGGCTCAGCGCGCGCGGTCACGTTTTCCACTATGTGACAGCGGAGG-3’, Per2 mutant promoter DNA: 5’-CAGTTGAGGACACAGCGCGCGCGGTCACGTTTTCCACTATGTGACAGCGGAGG-3’. Per2 consensus −55bp oligo: 5’-ATGGCGCGCGCAGGGGCGGGCTCAGCGCGCGCGGTCACGTTTTCCACTATGTGACAGCGGAGG-3’, Per2 mutant consensus −55bp oligo: 5’- ATGGCGCGCGCAGTTGAGGACACAGCGCGCGCGGTCACGTTTTCCACTATGTGACAGCGGAGG-3’. Recombinant mouse p53 was purchased from Active Motif (#31103) and 2 ng of recombinant p53 was used per reaction. Binding assays were performed in a buffer containing 20 mM HEPES (pH7.6), 30 mM KCl, 1 mM EDTA, 10 mM (NH4)2SO4, 1 mM DTT, 0.2% Tween 20, 1 µg poly [d(A–T)], 0.1 µg poly L- lysine. Anti-p53 antibodies (pAb421, Calbiochem or DO-1, Santa Cruz) were used for supershift assay. For BMAL1/CLOCK binding assay, nuclear extracts were prepared as described previously53. Briefly, cells were washed once with Buffer A containing 10 mM HEPES (pH 7.3), 1.5 mM MgCl2, 10 mM KCl, and 0.5 mM DTT, and resuspended in Buffer A. Cells were homogenized, centrifuged (200rpm, 10min), and nuclei were washed again with Buffer A. Cell pellets were resuspended in Buffer C containing 20 mM HEPES (pH 7.3), 20 % glycerol, 0.42 M NaCl, 1.5 mM MgCl2, 0.2 mM EDTA, 0.5 mM DTT, 0.5 mM phenylmethylsulfonyl fluoride and protease inhibitor cocktail (Roche). Lysates were incubated for 30 min with constant rotation at 4 °C and centrifuged (8000 rpm, 10 min). Supernatants were used as nuclear extracts. Binding assays were performed in a buffer containing 50 mM Tris-HCl (pH 8.0), 100 mM KCl, 1 mM EDTA, 6 mM DTT and 1 µg poly [d(A–T)]54. The anti-BMAL1 antibody (N-20, 2 µg, Santa Cruz) and anti-FLAG M2 antibody (5 µg, SIGMA) were used to detect the specific BMAL1/CLOCK band. The anti-BMAL1 antibody was previously validated by other investigators55. Protein binding reactions were performed in 10 µl scale.

Electric mobility shift assay

Electric mobility shift assay (EMSA) was carried out using a Light-Shift Chemiluminescent EMSA kit (#20148, PIERCE) as described previously52. Briefly, the following double stranded 5’-biotinylated or unlabeled DNA’s were synthesized (SIGMA Aldrich)., Per2 consensus sequence DNA: 5’-CAGGGGCGGGCTCAGCGCGCGCGGTCACGTTTTCCACTATGTGACAGCGGAGG-3’, Per2 mutant promoter DNA: 5’-CAGTTGAGGACACAGCGCGCGCGGTCACGTTTTCCACTATGTGACAGCGGAGG-3’. Per2 consensus −55bp oligo: 5’-ATGGCGCGCGCAGGGGCGGGCTCAGCGCGCGCGGTCACGTTTTCCACTATGTGACAGCGGAGG-3’, Per2 mutant consensus −55bp oligo: 5’- ATGGCGCGCGCAGTTGAGGACACAGCGCGCGCGGTCACGTTTTCCACTATGTGACAGCGGAGG-3’. Recombinant mouse p53 was purchased from Active Motif (#31103) and 2 ng of recombinant p53 was used per reaction. Binding assays were performed in a buffer containing 20 mM HEPES (pH7.6), 30 mM KCl, 1 mM EDTA, 10 mM (NH4)2SO4, 1 mM DTT, 0.2% Tween 20, 1 µg poly [d(A–T)], 0.1 µg poly L- lysine. Anti-p53 antibodies (pAb421, Calbiochem or DO-1, Santa Cruz) were used for supershift assay. For BMAL1/CLOCK binding assay, nuclear extracts were prepared as described previously53. Briefly, cells were washed once with Buffer A containing 10 mM HEPES (pH 7.3), 1.5 mM MgCl2, 10 mM KCl, and 0.5 mM DTT, and resuspended in Buffer A. Cells were homogenized, centrifuged (200rpm, 10min), and nuclei were washed again with Buffer A. Cell pellets were resuspended in Buffer C containing 20 mM HEPES (pH 7.3), 20 % glycerol, 0.42 M NaCl, 1.5 mM MgCl2, 0.2 mM EDTA, 0.5 mM DTT, 0.5 mM phenylmethylsulfonyl fluoride and protease inhibitor cocktail (Roche). Lysates were incubated for 30 min with constant rotation at 4 °C and centrifuged (8000 rpm, 10 min). Supernatants were used as nuclear extracts. Binding assays were performed in a buffer containing 50 mM Tris-HCl (pH 8.0), 100 mM KCl, 1 mM EDTA, 6 mM DTT and 1 µg poly [d(A–T)]54. The anti-BMAL1 antibody (N-20, 2 µg, Santa Cruz) and anti-FLAG M2 antibody (5 µg, SIGMA) were used to detect the specific BMAL1/CLOCK band. The anti-BMAL1 antibody was previously validated by other investigators55. Protein binding reactions were performed in 10 µl scale.

Chromatin immunoprecipitation assayChromatin immunoprecipitation (ChIP) assay was performed as described previously56. Briefly, cells or tissues were fixed with 1% formalin for 10 min. Crosslinking was stopped by adding 125 mM glycine. Cells were washed and homogenized using a homogenizer. Homogenized cells were collected by brief centrifugation and lysed with swelling buffer containing 5 mM PIPES (pH8.0), 85 mM KCl, 1% NP40, and protease inhibitor cocktail (# 11697498001, Roche). Nuclear fractions were collected by centrifugation and lysed with nuclear lysis buffer containing 50 mM Tris-HCl (pH 8.0), 10 mM EDTA, 1% SDS, and protease inhibitor cocktail. Sonication was performed to fragment DNA to about 500 bp. After centrifugation, lysates were precleared with protein A agarose (Invitrogen). Pre-cleared samples were diluted with the same volume of IP dilution buffer containing 16.7 mM Tris-HCl (pH 8.0), 0.01% SDS, 1.1% Triton X-100, 1.2 mM EDTA, 167 mM NaCl, and protease inhibitor cocktail. Then, p53 and BMAL1 were immunoprecipitated with anti-p53 antibody (2 µg, sc-6243) or anti-BMAL1 antibody (2 µg, ab3350, abcam) and collected on protein A agarose. After serial washing with dialysis buffer containing 50 mM Tris-HCl (pH8.0) 2 mM EDTA, and 0.2% sarkosyl and IP wash buffer containing 100 mM Tris-HCl (pH9.0), 500 mM LiCl, 1% NP40, 1% Deoxycholate, and protease inhibitor cocktail. p53-DNA complexes were eluted by elution buffer containing 50 mM NaHCO3 and 1% SDS. Eluted samples were mixed with 0.2 M NaCl and incubated at 67 °C overnight, then RNase A treated, and purified. PCR and quantitative real-time PCR were performed using Per2-Ebox2 forward primers: 5’-CAGGTTCCGCCCCGCCAGTAT-3’, reverse primer:5’-GTCGCCCTCCGCTGTCACATAG-3’ and Per2-Ebox5 forward primer: 5’-CTCTGTAGGGTGGAGCGGCGA-3’, reverse primer: 5’-ATCCCCACTGCTCCTTCGCAC-3’. ChIP samples obtained from SCN and thymus were analyzed by real-time PCR (ABI PRISM 7700 system, Applied Biosystems) using Maxima SYBR Green qPCR Master Mix (#0252, Fermentas). Real-time PCR data were analyzed as previously reported57. IgG control antibody was used as a negative control. ΔCt was obtained by Ct experiment sample–., Ct negative control and fold differences were calculated by [2(−ΔCt)].

Chromatin immunoprecipitation assay

Chromatin immunoprecipitation (ChIP) assay was performed as described previously56. Briefly, cells or tissues were fixed with 1% formalin for 10 min. Crosslinking was stopped by adding 125 mM glycine. Cells were washed and homogenized using a homogenizer. Homogenized cells were collected by brief centrifugation and lysed with swelling buffer containing 5 mM PIPES (pH8.0), 85 mM KCl, 1% NP40, and protease inhibitor cocktail (# 11697498001, Roche). Nuclear fractions were collected by centrifugation and lysed with nuclear lysis buffer containing 50 mM Tris-HCl (pH 8.0), 10 mM EDTA, 1% SDS, and protease inhibitor cocktail. Sonication was performed to fragment DNA to about 500 bp. After centrifugation, lysates were precleared with protein A agarose (Invitrogen). Pre-cleared samples were diluted with the same volume of IP dilution buffer containing 16.7 mM Tris-HCl (pH 8.0), 0.01% SDS, 1.1% Triton X-100, 1.2 mM EDTA, 167 mM NaCl, and protease inhibitor cocktail. Then, p53 and BMAL1 were immunoprecipitated with anti-p53 antibody (2 µg, sc-6243) or anti-BMAL1 antibody (2 µg, ab3350, abcam) and collected on protein A agarose. After serial washing with dialysis buffer containing 50 mM Tris-HCl (pH8.0) 2 mM EDTA, and 0.2% sarkosyl and IP wash buffer containing 100 mM Tris-HCl (pH9.0), 500 mM LiCl, 1% NP40, 1% Deoxycholate, and protease inhibitor cocktail. p53-DNA complexes were eluted by elution buffer containing 50 mM NaHCO3 and 1% SDS. Eluted samples were mixed with 0.2 M NaCl and incubated at 67 °C overnight, then RNase A treated, and purified. PCR and quantitative real-time PCR were performed using Per2-Ebox2 forward primers: 5’-CAGGTTCCGCCCCGCCAGTAT-3’, reverse primer:5’-GTCGCCCTCCGCTGTCACATAG-3’ and Per2-Ebox5 forward primer: 5’-CTCTGTAGGGTGGAGCGGCGA-3’, reverse primer: 5’-ATCCCCACTGCTCCTTCGCAC-3’. ChIP samples obtained from SCN and thymus were analyzed by real-time PCR (ABI PRISM 7700 system, Applied Biosystems) using Maxima SYBR Green qPCR Master Mix (#0252, Fermentas). Real-time PCR data were analyzed as previously reported57. IgG control antibody was used as a negative control. ΔCt was obtained by Ct experiment sample–., Ct negative control and fold differences were calculated by [2(−ΔCt)].

Immunofluorescence staining8–10 week old WT and Trp53−/− mouse brains were collected at the indicated time point, fixed with 4% PFA, and embedded in paraffin. Then 8 µm sections were obtained and stained with anti-p53 antibody (#sc-6243, x50, SantaCruz), or anti-mPER2 antibody (x4,000) and visualized by secondary anti-rabbit IgG Alexa Fluor 555 antibody (#A21429, x1000, Invitrogen).

Immunofluorescence staining

8–10 week old WT and Trp53−/− mouse brains were collected at the indicated time point, fixed with 4% PFA, and embedded in paraffin. Then 8 µm sections were obtained and stained with anti-p53 antibody (#sc-6243, x50, SantaCruz), or anti-mPER2 antibody (x4,000) and visualized by secondary anti-rabbit IgG Alexa Fluor 555 antibody (#A21429, x1000, Invitrogen).

In situ hybridization8–10 week old WT and Trp53−/− mouse brains were collected at the indicated time point, fixed with 4% PFA, and embedded in paraffin. Then 8 µm sections were obtained and in situ hybridization was performed as described previously58 with some modifications. Briefly, paraffin sections were treated with 1 µg/ml protease K in PBS-T (0.1% Tween 20) at 37 °C for 15 min, washed, fixed with 4% performic acid at room temperature for 15 min, washed, and then incubated with digoxigenin-labeled cRNA probes in buffer containing 50% formamide, 1% SDS, 5 x SSC, 50 µg/ml tRNA, and 20 mg/ml heparin at 55°C overnight. The probe for Per2 was described previously59. Hybridized sections were washed with Buffer A containing 50% formamide, 1% SDS, and 5 x SSC twice at 55 °C for 30 min each and then washed with Buffer B containing 50% formamide and 2 x SSC three times at 55 °C for 30 min each. After washing with TBS-T (pH7.6, 0.1% Tween 20) three times, sections were blocked with 0.5% Blocking Reagent (#11096176001, Roche) for 1 h and incubated with anti-alkaline phosphatase-conjugated anti-DIG antibody (#11093274910, x1000, Roche) at 4 °C overnight. Sections were washed three times with TBS-T and immersed in NTMT buffer containing 100 mM Tris-HCl (pH 9.5), 100 mM NaCl, 50 mM MgCl2, 0.1% Tween 20, and 2 mM levamisol. Sections were then developed with NBT/BCIP reagents.

In situ hybridization

8–10 week old WT and Trp53−/− mouse brains were collected at the indicated time point, fixed with 4% PFA, and embedded in paraffin. Then 8 µm sections were obtained and in situ hybridization was performed as described previously58 with some modifications. Briefly, paraffin sections were treated with 1 µg/ml protease K in PBS-T (0.1% Tween 20) at 37 °C for 15 min, washed, fixed with 4% performic acid at room temperature for 15 min, washed, and then incubated with digoxigenin-labeled cRNA probes in buffer containing 50% formamide, 1% SDS, 5 x SSC, 50 µg/ml tRNA, and 20 mg/ml heparin at 55°C overnight. The probe for Per2 was described previously59. Hybridized sections were washed with Buffer A containing 50% formamide, 1% SDS, and 5 x SSC twice at 55 °C for 30 min each and then washed with Buffer B containing 50% formamide and 2 x SSC three times at 55 °C for 30 min each. After washing with TBS-T (pH7.6, 0.1% Tween 20) three times, sections were blocked with 0.5% Blocking Reagent (#11096176001, Roche) for 1 h and incubated with anti-alkaline phosphatase-conjugated anti-DIG antibody (#11093274910, x1000, Roche) at 4 °C overnight. Sections were washed three times with TBS-T and immersed in NTMT buffer containing 100 mM Tris-HCl (pH 9.5), 100 mM NaCl, 50 mM MgCl2, 0.1% Tween 20, and 2 mM levamisol. Sections were then developed with NBT/BCIP reagents.

Locomoter activityWheel-running activity was monitored as described60. Male wild type and Trp53 −/− mice aged about 4 weeks were used. Nutlin-3 (200mg/kg) or vehicle (DMSO) was injected i.p. at the indicated time point.

Locomoter activity

Wheel-running activity was monitored as described60. Male wild type and Trp53 −/− mice aged about 4 weeks were used. Nutlin-3 (200mg/kg) or vehicle (DMSO) was injected i.p. at the indicated time point.

Supplementary Material1

Supplementary Material
