---
pmid: '26642243'
title: Loss-of-function mutations in TNFAIP3 leading to A20 haploinsufficiency cause
  an early-onset autoinflammatory disease.
authors:
- Zhou Q
- Wang H
- Schwartz DM
- Stoffels M
- Park YH
- Zhang Y
- Yang D
- Demirkaya E
- Takeuchi M
- Tsai WL
- Lyons JJ
- Yu X
- Ouyang C
- Chen C
- Chin DT
- Zaal K
- Chandrasekharappa SC
- Hanson EP
- Yu Z
- Mullikin JC
- Hasni SA
- Wertz IE
- Ombrello AK
- Stone DL
- Hoffmann P
- Jones A
- Barham BK
- Leavis HL
- van Royen-Kerkof A
- Sibley C
- Batu ED
- Gül A
- Siegel RM
- Boehm M
- Milner JD
- Ozen S
- Gadina M
- Chae J
- Laxer RM
- Kastner DL
- Aksentijevich I
journal: Nat Genet
year: '2016'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4777523
doi: 10.1038/ng.3459
---

# Loss-of-function mutations in TNFAIP3 leading to A20 haploinsufficiency cause an early-onset autoinflammatory disease.
**Authors:** Zhou Q, Wang H, Schwartz DM, Stoffels M, Park YH, Zhang Y, Yang D, Demirkaya E, Takeuchi M, Tsai WL, Lyons JJ, Yu X, Ouyang C, Chen C, Chin DT, Zaal K, Chandrasekharappa SC, Hanson EP, Yu Z, Mullikin JC, Hasni SA, Wertz IE, Ombrello AK, Stone DL, Hoffmann P, Jones A, Barham BK, Leavis HL, van Royen-Kerkof A, Sibley C, Batu ED, Gül A, Siegel RM, Boehm M, Milner JD, Ozen S, Gadina M, Chae J, Laxer RM, Kastner DL, Aksentijevich I
**Journal:** Nat Genet (2016)
**DOI:** [10.1038/ng.3459](https://doi.org/10.1038/ng.3459)
**PMC:** [PMC4777523](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4777523/)

## Abstract

1. Nat Genet. 2016 Jan;48(1):67-73. doi: 10.1038/ng.3459. Epub 2015 Dec 7.

Loss-of-function mutations in TNFAIP3 leading to A20 haploinsufficiency cause an 
early-onset autoinflammatory disease.

Zhou Q(1), Wang H(1), Schwartz DM(2), Stoffels M(1), Park YH(1), Zhang Y(3), 
Yang D(4), Demirkaya E(5), Takeuchi M(1), Tsai WL(6), Lyons JJ(3), Yu X(3), 
Ouyang C(7), Chen C(1), Chin DT(1), Zaal K(8), Chandrasekharappa SC(9), Hanson 
EP(7), Yu Z(4), Mullikin JC(10), Hasni SA(11), Wertz IE(12), Ombrello AK(1), 
Stone DL(1), Hoffmann P(1), Jones A(1), Barham BK(1), Leavis HL(13), van 
Royen-Kerkof A(14), Sibley C(15), Batu ED(16), Gül A(17), Siegel RM(7), Boehm 
M(4), Milner JD(3), Ozen S(16), Gadina M(6), Chae J(1), Laxer RM(18), Kastner 
DL(1), Aksentijevich I(1).

Author information:
(1)Inflammatory Disease Section, National Human Genome Research Institute, 
Bethesda, Maryland, USA.
(2)Molecular Immunology and Inflammation Branch, National Institute of Arthritis 
and Musculoskeletal and Skin Diseases, Bethesda, Maryland, USA.
(3)Genetics and Pathogenesis of Allergy Section, National Institute of Allergy 
and Infectious Diseases, Laboratory of Allergic Diseases, Bethesda, Maryland, 
USA.
(4)Laboratory of Cardiovascular Regenerative Medicine, National Heart, Lung, and 
Blood Institute, Bethesda, Maryland, USA.
(5)FMF Arthritis Vasculitis and Orphan Disease Research Center (FAVOR), Gulhane 
Military Medical Academy, Ankara, Turkey.
(6)Translational Immunology Section, National Institute of Arthritis and 
Musculoskeletal and Skin Diseases, Bethesda, Maryland, USA.
(7)Autoimmunity Branch, National Institute of Arthritis and Musculoskeletal and 
Skin Diseases, Bethesda, Maryland, USA.
(8)Light Imaging Section, National Institute of Arthritis and Musculoskeletal 
and Skin Diseases, Bethesda, Maryland, USA.
(9)Cancer Genetics and Comparative Genomics Branch, National Human Genome 
Research Institute, Bethesda, Maryland, USA.
(10)National Institute of Health Intramural Sequencing Center, National Human 
Genome Research Institute, Bethesda, Maryland, USA.
(11)Systemic Autoimmune Branch, National Institute of Arthritis and 
Musculoskeletal and Skin Diseases, Bethesda, Maryland, USA.
(12)Department of Molecular Oncology, Genentech, Inc., San Francisco, 
California, USA.
(13)Department of Medical Microbiology, University Medical Center Utrecht, 
Utrecht, the Netherlands.
(14)Department of Pediatric Immunology, University Medical Center Utrecht, 
Utrecht, the Netherlands.
(15)Division of Arthritis and Rheumatic Diseases, Oregon Health and Science 
University, Portland, Oregon, USA.
(16)Department of Pediatric Rheumatology, Hacettepe University, Ankara, Turkey.
(17)Department of Internal Medicine, Istanbul University, Istanbul, Turkey.
(18)Division of Rheumatology, The Hospital for Sick Children, University of 
Toronto, Toronto, Ontario, Canada.

Systemic autoinflammatory diseases are driven by abnormal activation of innate 
immunity. Herein we describe a new disease caused by high-penetrance 
heterozygous germline mutations in TNFAIP3, which encodes the NF-κB regulatory 
protein A20, in six unrelated families with early-onset systemic inflammation. 
The disorder resembles Behçet's disease, which is typically considered a 
polygenic disorder with onset in early adulthood. A20 is a potent inhibitor of 
the NF-κB signaling pathway. Mutant, truncated A20 proteins are likely to act 
through haploinsufficiency because they do not exert a dominant-negative effect 
in overexpression experiments. Patient-derived cells show increased degradation 
of IκBα and nuclear translocation of the NF-κB p65 subunit together with 
increased expression of NF-κB-mediated proinflammatory cytokines. A20 restricts 
NF-κB signals via its deubiquitinase activity. In cells expressing mutant A20 
protein, there is defective removal of Lys63-linked ubiquitin from TRAF6, NEMO 
and RIP1 after stimulation with tumor necrosis factor (TNF). NF-κB-dependent 
proinflammatory cytokines are potential therapeutic targets for the patients 
with this disease.

DOI: 10.1038/ng.3459
PMCID: PMC4777523
PMID: 26642243 [Indexed for MEDLINE]

Conflict of interest statement: Competing financial interests The authors 
declare no competing financial interests

## Full Text

Monogenic autoinflammatory diseases are a heterogeneous group of diseases marked by unprovoked episodic or chronic inflammatory symptoms. Mutations in more than 20 causative genes have been reported, with many coding for proteins regulating signaling by pro-inflammatory cytokines such as interleukin-1, type 1 interferon and TNF 4-6. Therapies targeting pro-inflammatory cytokines, especially IL-1 and TNF, have been effective in many patients 7. However, a significant number of sporadic and familial cases remain genetically uncharacterized.

We studied a family of Caucasian ancestry (Family 1) with a dominant disorder manifesting early-onset systemic inflammation, arthralgia/arthritis, oral/genital ulcers and ocular inflammation (Fig. 1a, b, Supplementary Table 1). Whole exome sequencing (WES) on the affected mother and the two affected daughters identified 11 novel candidate variants. After Sanger sequencing of maternal unaffected family members, two novel variants in TNFAIP3 (tumor necrosis factor, alpha-induced protein 3; A20) and TNFRSF9 (tumor necrosis factor receptor superfamily, member 9) remained in consideration (Supplementary Table 2, Supplementary Fig. 1). Both genes function in the regulation of immune responses.

Independently, we performed WES in a family of Italian ancestry with an affected mother (P4) and daughter (P6). The family history was positive for a dominantly inherited inflammatory disease (Family 2, Fig. 1a). Patients P4, P6 and P5, who is the sister of P4 and aunt of P6, had in common oral/genital ulcers and polyarthritis. Patient P6 was initially diagnosed with an early-onset severe autoimmune condition resembling systemic lupus erythematosus, with CNS vasculitis and anterior uveitis (Fig. 1b, Supplementary Table 1). We identified 30 novel candidate variants that are shared between patients P4 and P6, including a novel variant in TNFAIP3. Sanger sequencing of patient P5 and the three unaffected family members confirmed that the novel variant p.Phe224Serfs*4 segregated with the inflammatory phenotype (Fig. 1a). Patient P6 did not have mutations in genes associated with early-onset autoimmune diseases8,9. We noted that Family 1 and Family 2 shared novel variants in a single candidate gene TNFAIP3 (Supplementary Fig. 2a).

Screening of an additional 150 patients with clinically similar disease identified three novel nonsense and frameshift variants in families of Turkish (Family 3), European -American (Family 4) and Dutch (Family 5) ancestry. In total, we identified 5 heterozygous truncating mutations in TNFAIP3 in five families (Fig. 1a, Supplementary Fig. 2b and Table 1). Targeted sequencing in 384 Turkish and 384 Japanese adult-onset Behçet’s cases from genome wide association studies identified one patient with a novel frameshift mutation in TNFAIP3, p.Pro268Leufs*19 (Supplementary Fig. 2c), which was subsequently identified in her two affected daughters. The identified mutations were considered pathogenic on the basis of proper family segregation and their absence from the Exome Aggregation Consortium (ExAC) database, dbSNP, and our exome database of more than 500 exomes. Previously, frameshift mutations in TNFAIP3 have been reported only in tumor tissues in the form of bi-allelic somatic mutations10. The pathogenic variants from our study were predicted to result in truncated proteins, suggesting haploinsufficiency or a dominant-negative effect of mutant TNFAIP3/A20 proteins.

TNFAIP3/A20 is a 790-residue protein that consists of an amino-terminal ovarian tumor domain (OTU) followed by 7 zinc finger domains (ZFs)11 (Fig. 1c). Five of the six disease-associated TNFAIP3 mutations are located in the OTU domain, which mediates the DUB activity, and they result in mutant truncated proteins of similar length. One disease-associated mutation, p.Thr604Argfs*93, resides in the ZnF4 domain, which recognizes K63-linked ubiquitin chains12,13 and is also essential for A20 E3 ligase activity12,13 and dimerization14. Expression of wild type A20 was reduced in patients’ PBMCs and fibroblasts, while mutant proteins were not detectable (Fig. 1d,e). We suspect that mutant proteins are not stable and undergo degradation.

We assessed the activity of each mutant A20 using an NFκB luciferase assay. Over-expressed mutant proteins failed to suppress TNF-induced NFκB activity in transfected human embryonic kidney (HEK) 293T cells (Fig. 2a) and a human T cell leukemia cell-line (Jurkat) (Supplementary Fig. 3). WT A20 suppresses NFkB activity, and co-transfection of a mutant truncated A20 construct with WT A20 did not reverse this outcome, suggesting that the mutant protein does not have a dominant-negative effect (Fig. 2a). IKK-mediated phosphorylation of IκBα resulting in IκB degradation, and subsequent nuclear translocation of the associated p50/65 NFκB heterodimer, is an essential step in the activation of the canonical NFκB pathway. To define the relationship of A20 mutants and NFκB induction in vivo, we studied activity of the canonical NFκB pathway in response to TNF stimulation. Stimulated patients’ cells showed increased phosphorylation of IKKα/β, increased phosphorylation of MAP kinases p38 and JNK, and increased degradation of IκBα compared to the controls (Fig. 2b, c and Supplementary Fig. 4a,b). Consistent with these findings, we observed increased nuclear translocation of p65 in patients’ PBMCs (Fig. 2d, and Supplementary Fig. 5a) and fibroblasts both at rest and after TNF stimulation (Fig. 2e, and Supplementary Fig. 5b,c). These data are strong evidence for enhanced signaling in the NFκB pathway.

The NFκB signaling cascade is under tight regulation by a number of post-translational modifications, including ubiquitination 15. A20 restricts cellular activation by cleaving K63-linked ubiquitin chains (Ub) from target substrates such as NEMO, RIP1, and TRAF612. Through its E3 ligase activity, A20 adds K48-linked Ub chains, targeting proteins for proteasome degradation 11,16. Upon TNF activation, TNFR forms a primary NF-kB activating signaling complex with TRADD, TRAF2 and RIP1. Consistent with reduced cellular levels of A20 in the cells of patients harboring TNFAIP3 truncation mutations, patient PBMCs displayed reduced recruitment of A20 to the TNFR complex compared to cells from healthy donors (Fig. 3a). To investigate whether the TNFAIP3/A20 mutants still retained their deubiquitinase function, WT and mutant A20 constructs were cotransfected into 293T cells with K63-linked ubiquitin and the A20 ubiquitination targets RIP1, NEMO/IKKγ and TRAF6 (Fig. 3b, c and d). Cells transfected with various disease-associated A20 mutants showed a marked defect in the deubiquitination of each of these target molecules, and cells co-transfected with wild-type and mutant A20 molecules, mimicking the situation in the patients, had intermediate levels of deubiquitination.

Consistent with the transfection data, TNFR1 signaling complexes from patients with A20 mutations accumulated sustained levels of K63-ubiquitinated NEMO and RIP1 and accumulated high-molecular weight (MW) Ub-aggregates (Fig. 3e and f, top panel, lane 5-8; Supplementary Fig. 6a,b, lane 5-8). In comparison, in healthy control PBMCs and fibroblasts, the expression of high MW Ub-aggregates gradually decreased over time following TNF stimulation (Fig. 3e, f, top panel, lane 1-4; Supplementary Fig. 6a,b, lane 1-4). Taken together, these results indicate that inefficient deubiquitination of A20 target proteins might explain a higher NFκB signaling activity in mutant cells.

Active NFκB subunits promote the transcription of genes encoding pro-inflammatory cytokines such as interleukin IL-1β, IL-6 and TNF, and facilitate the differentiation and activation of a variety of lymphocyte lineages. Multiple experiments showed evidence for increased expression of NFκB target genes in patients’ immune cells (Supplementary Fig. 7 and Supplementary Fig. 8). Levels of several proinflammatory cytokines were substantially increased in patients’ serum and in the supernatants of stimulated PBMCs relative to healthy controls (Fig. 4a, b). Intracellular cytokine staining also revealed increased polarization toward Th9 and Th17 CD4 T-effector cell lineages, which is consistent with the participation of IL-1β signaling in the differentiation of these lineages and may further contribute to tissue inflammation in A20-deficient patients (Fig. 4c). Finally, the frequency of CD14+ inflammatory monocytes was significantly higher in patients than in healthy controls (Fig. 4d).

Recent studies in murine macrophages lacking A20 suggest that A20 functions as a negative regulator of the Nlrp3 inflammasome independently of its role in NFκB regulation 1718. Consistent with these data, patients’ cells showed constitutive activation of the NLRP3 inflammasome (Fig. 5a), which resulted in activation of caspase-1 and increased secretion of active IL-1β and IL-18 (Fig. 4b). Patients’ PBMCs exhibited enhanced NLRP3 inflammasome-mediated caspase-1 activation and secreted IL-1β in response to LPS priming alone, and this effect was attenuated by use of a PLC inhibitor, an adenylate cyclase activator, or MCC950, all known NLRP3 Inflammasome inhibitors19,20 (Fig. 5b). Furthermore, initial experience in a patient with an agent targeting IL-1β has been positive (Fig. 5c).

In summary, this is the first report of a human disease, Haploinsufficiency of A20 (HA20), caused by high-penetrance loss-of-function germline mutations in TNFAIP3/A20. The findings of multiple nonsense and frameshift variants associated with impaired regulatory function of TNFAIP3/A20 provide strong evidence that these are pathogenic mutations. Although patients with HA20 and Behçet’s disease share similar symptoms, the genetics are distinct. Most sporadic late-onset Behçet’s cases are not caused by highly penetrant germline mutations in A20. A20-deficient mice (A20-/-) display persistent NFκB activation, spontaneous multi organ inflammation and early lethality 21. Aging heterozygous mice (A20+/-) develop autoantibodies resembling human autoimmune conditions 15. Targeted deletion of A20 in specific hematopoietic cells leads to phenotypes reminiscent of autoimmunity 22-24. Although five patients in our study developed autoantibodies, an overt autoimmune disease was only diagnosed in Patient 6. Notably, none of our patients have developed lymphomas or other malignancies.

TNFAIP3/A20 variants have been linked to many human diseases 15. Somatic inactivating mutations in TNFAIP3 have been described in B cell lymphomas, suggesting that TNFAIP3/A20 acts as a tumor suppressor25. Common low-penetrance coding and non-coding variants in TNFAIP3 have been associated with multiple autoimmune diseases 26-36 and susceptibility to allergy and asthma37. The current study, however, is the first to delineate the consequences of germline inactivating TNFAIP3 mutations in human biology.

Next generation sequencing will likely identify other rare monogenic inflammatory diseases, possibly in the same pathway, which might further inform genetic studies of common polygenic immune diseases38-40. There are also potential therapeutic implications from studying patients with monogenic diseases, such as HA20, which may apply to the more common inflammatory conditions such as Behçet’s disease.

Online MethodsHuman subjectsSeven patients were evaluated at the NIH Clinical Center, four were evaluated at University Medical Center Utrecht in Netherland and two at the Hacettepe University Children’s Hospital in Turkey. All patients enrolled in the study were evaluated under protocols approved by their respective institutional review boards. All the patients and family members provided written informed consent including consent to publish.Whole exome sequencingWhole exome sequencing and data analysis was performed as previously described41. Novel candidate variants were filtered by ExAC, 1000 genomes, dbSNP, NHLBI GO Exome Sequencing Project, Clinseq database, and an in-house database with over 500 exomes and selected based on autosomal dominant inheritance.Targeted sequencingTargeted sequencing of TNFAIP3 was performed on pooled DNA from 384 Turkish and 384 Japanese Behcet’s patients with adult-onset disease. Pooled DNA was prepared as previously described42. Ten amplicons covering the 9 exons of TNFAIP3 were generated using high fidelity polymerase AccuPrime™ Pfx SuperMix (Invitrogen). PCR products were pooled and prepared for library using Ovation® Ultralow Library Systems (NuGEN). Paired-end 300 bp reads were sequenced on a MiSeq instrument (Illumina). We called SNPs and INDELs, and calculated the frequencies from the pileup file by an in-house Perl script to identify SNPs and INDELs with frequency ≥1%.Sanger sequencingSanger sequencing was used to confirm mutations identified by exome sequencing, targeted sequencing and for the candidate gene screening as previously described41.Haplotype analysisSNPs within the TNFAIP3 gene and STR markers across the chromosome 6 were genotyped for all family members from family 1. Haplotypes were reconstructed by PHASE and MERLIN.Antibodies and expression plasmidsAntibodies specific for TNFAIP3 N-terminus (sc-166692), TRAF6 (sc-8409, sc-7221), Actin (sc-1615), Ubc13 (sc-376470, sc-292618), TNFR1 (sc-8436), RIP (sc-7881, sc-133102), NEMO (sc-56919, sc-8330), Ub (sc-271289), Caspase-1 (sc-515 and sc-622), GFP (sc-8334), HA-probe (sc-7392), c-MYC (sc-40) from Santa Cruz; TNFAIP3 N-terminal specific (#5630), K63-linkage Polyubiquitin (#5621), Phospho-IKKα/β (#2697), IKKα (#11930), IκBα (#4814, #9242), Phospho-IκBα (#2859), NFκB P65 (#8242), TNFR1 (#3736), RIP (#3493), Phospho-p38 MAPK (#4511), Phospho-p44/42 MAPK (Erk1/2) (#4370), Phospho-SAPK/JNK (#4668), p44/42 MAPK (Erk1/2) (#4695), SAPK/JNK (#9252), p38 MAPK (#8690), HRP-linked anti-rabbit IgG (#7074), HRP-linked anti-mouse IgG (#7076) from Cell Signaling; IL-1 β (AF-201-NA) from R&D Systems; NLRP3 (ALX-804-819-C100) from Enzo Life Sciences. pRK5-HA-Ubiquitin-K63 (Addgene plasmid # 17606) was a gift from Ted Dawson43. pEGFP-C1-TNFAIP3 (Addgene plasmid # 22141) was a gift from Yihong Ye 44. Myc-DDK-tagged-human RIPK1 (RC216024), Myc-DDK-tagged-human NFKBIA (RC200711), untagged human TRAF6 (sc109845) were from Origene. 3xFlag-TNFAIP3 (Ex-K6040-M120) was from Genecopoeia. pEF-NEMO was a gift from Dr. Chi Ma45. The GFP tagged or 3xFlag-TNFAIP3 mutant plasmids were constructed by PCR mutagenesis.Cell culturesHEK293T cells (human embryonic kidney cells expressing SV40 large T antigen, negative for mycoplasma, originally obtained from the American Type Culture Collection) and skin fibroblast cells derived from TNFAIP3-deficient patients or normal donors were grown in Dulbecco’s modified Eagle’s medium (Life Technologies) plus 10% fetal calf serum and 1x antibiotics (Life Technologies). Jurkat-T (human leukaemic T lymphoma cell line, negative for mycoplasma, originally obtained from the American Type Culture Collection) cells and human PBMCs were grown in RPMI1640 medium (Life Technologies), plus 10% fetal calf serum (Gemini Bio-Products) and antibiotics. For intracellular TNF staining, pan-T cells purified from PBMCs by negative selection (Miltenyi) were cultured in complete RPMI 1640 in the presence of plate-bound anti-CD3 and soluble anti-CD28 (both 1 μg/mL) for 5 days. For IL-9 staining, rhIL-4 (30 ng/mL), TGF-β (5 ng/mL), IL-1β (10 ng/ml), and IL-2 (10 U/ml), together with anti-IFNγ (10 μg/mL) were added to total PBMCs, and cells were cultured for 48 hours under the same conditions.NFκB-dependent reporter assay and in vivo deubiquitination assayPreformulated Cignal NFkB Pathway Reporter plasmids (Qiagen) (pGL4.32[luc2P/NF-κBRE/Hygro] luciferase reporter plus The pRLCMV-Renilla vector) were used to co-transfect in 293T cells with equal amount of expression plasmids for GFP-tag TNFAIP3 wild type or mutants using Polyjet transfection reagents (Signagen). After 24 hrs, cells were left untreated or stimulated with TNF (20 ng/ml). Luciferase activity (the firefly (luc) and renilla luciferase (ren) activity) was measured 5 hours afterwards by the dual-glo luciferase assay system (Promega). Results for NFkB activity are expressed as fold induction by normalizing firefly luciferase activity to Renilla luciferase activity.For deubiquitination assay, 293T cells were transfected with 1 μg plasmids expressing target protein (RIP1, NEMO, or TRAF6), together with 0.5 μg HA-K63Ub, 0.5 ug GFP-tagged TNFAIP3 wild type or mutants. Empty vectors were used to equalize the plasmid transfection amount. Cells were harvested 48 hrs after transfection. Cell lysates were prepared and immunoprecipitation was conducted as previously described46.NFκB reporter assay in Jurkat T cell line5×106 3T8 cells (Jurkat T-cell line) were transiently transfected with 20 μg of expression plasmids encoding either wildtype TNFAIP3-GFP or mutant TNFAIP3-GFP fusion genes by electroporation. Sixteen hours post transfection, 20 ng/ml of human TNF was added to transfected 3T8 cells. Cells were collected 16 hours after the addition of TNF and surface stained with anti-Thy1-PECy7 and live/dead-violet for FACS analysis. Samples were acquired with a FACSVerse (Becton Dickinson Biosciences) and data were analyzed with FlowJo. The mean fluorescence intensity (MFI) of Thy1, the indicator of NFκB activity, was gated on the GFP+ live singlet populations.PBMC preparation and stimulation, immunoprecipitation and immunoblottingHeparinated peripheral blood mononuclear cells (PBMCs) were separated by Ficoll (Ficoll-Paque PLUS; GE Healthcare) gradient centrifugation. Recombinant human TNF (Peprotech) was used to stimulate PBMC (10 ng/ml) or fibroblast cells (20 ng/ml) from 5 min to 1 hr. Whole cell lysis were prepared using ice-cold 1xcell lysis buffer (Cell Signaling) supplemented with complete protease inhibitors. Nuclear extraction kit (Cayman) was used to prepare cytosolic and nuclear fractions. Immunoprecipitation and immunoblotting were conducted as described previously4547 with specific antibodies. ImageJ was used to analyze the immunoblotting images.Inflammasome inhibitionPBMCs (2×106 cells/well) were plated in 12-well plates and then primed with 1 μg ml-1 LPS in RPMI 1640 (Invitrogen) containing 10% FBS for 3 hrs. For the inhibition of inflammasome activation, LPS primed-PBMCs were treated with U73122 (5 μM), U73343 (5 μM), NKH477 (100 or 250 μM), or MCC950 (with three different doses: 50, 100, and 500 nM) in RPMI 1640. After 1 hr of treatment, supernatants and cell lysates were collected for immunoblot analysis.ImmunocytochemistryFibroblasts grown overnight in 96-well plates or on poly-L-lysine coated coverslips and treated with TNF were fixed in 4% paraformaldehyde and stained for NFκB p65. Primary (Cell Signaling 8242) and secondary antibodies (A21206 Molecular Probes) were incubated overnight at 4 °C and for 30 minutes at room temperature, respectively. Coverslips were mounted with Vectashield containing DAPI (H-1200, Vector Laboratories), imaged with a Zeiss AxioImager D2 using a 20x objective. 96-well plates were imaged with an Incucyte microscope (Essen Bioscience) using a 20x objective and a SYTO 59 (Molecular Probes) nuclear stain. Nuclear patterns were converted to regions of interest in ImageJ and used to measure nuclear p65 intensities. Data were analyzed in GraphPad Prism 6 and shown as mean ±S.E.M.Immune cell cytokine production and serum cytokine detectionPBMCs (2×106 /ml) were stimulated with bacterial lipopolysaccharide (LPS) (Sigma), flagellin (Sigma), and Staphylococcal enterotoxin B (SEB) (Sigma) for 24 hour at 37°C. The concentrations of cytokines in the supernatants of stimulated and non-stimulated PBMCs and serum were determined using Bio-Plex Pro Human Cytokine 27-plex and 21-plex immunoassay kits (Bio-Rad Hercules). The Bio-plex pro human cytokine standard group I and group II were used as standards for the assays. The differences in the cytokine concentrations were statistically analyzed using the Mann-Whitney test, and plotted with GraphPad Prism software package.Primary human monocyte isolation, differentiation and stimulationMonocytes were purified from PBMCs by negative selection (Monocyte Isolation Kit II; Miltenyi Biotec). Monocytes were suspended in monocyte attachment medium (PromoCell) and seeded at a density of 150,000/cm2 for 2 hours. Monocytes were co-stained with APC-Cy7-conjugated anti-CD11b, PE-Cy7-conjugated CD11c, Fitc conjugated anti-CD14, APC-conjugated CD16 or relative isotype control and then were analyzed using a FACS Canton II (BD Biosciences).Primary human monocytes were differentiated into M1 macrophages using 20 ng/mL human Granulocyte-macrophage colony-stimulating factor (hGM-CSF, Sigma) or into M2 macrophages using 100 ng/mL human macrophage colony-stimulating factor (hM-CSF, Sigma) for 10 days. hGM-CSF differentiated M1 macrophages were stimulated with or without LPS (Sigma) 100 ng/mL for 6 hours.Real-time PCRRNA was isolated from M1 macrophages using the RNeasy kit (Qiagen) and isolated from unstimulated PBMC using TRIzol. Reverse transcription was performed using the Taqman kit (Applied Biosystems). Real-time PCR reactions were performed using the iQ Syber Green kit (Biorad) or TaqMan gene expression master mix and TaqMan gene expression assays probes. 18S RNA served as control. The reactions were run on an Applied Biosystems 7500 Real-Time PCR System (Life Technologies).Intracellular cytokine staining in T cellsIntracellular cytokine staining for IL-17A, TNF, IL-4, and IL-9 following either PMA and ionomycin stimulation (IL-17A, IL-4, and IL-9) or SEB stimulation (TNF) was performed as previously described.48 All events were collected on an LSRFortessa (BD Biosciences) and analyzed with FlowJo (Treestar).

Online Methods

Human subjectsSeven patients were evaluated at the NIH Clinical Center, four were evaluated at University Medical Center Utrecht in Netherland and two at the Hacettepe University Children’s Hospital in Turkey. All patients enrolled in the study were evaluated under protocols approved by their respective institutional review boards. All the patients and family members provided written informed consent including consent to publish.

Human subjects

Seven patients were evaluated at the NIH Clinical Center, four were evaluated at University Medical Center Utrecht in Netherland and two at the Hacettepe University Children’s Hospital in Turkey. All patients enrolled in the study were evaluated under protocols approved by their respective institutional review boards. All the patients and family members provided written informed consent including consent to publish.

Whole exome sequencingWhole exome sequencing and data analysis was performed as previously described41. Novel candidate variants were filtered by ExAC, 1000 genomes, dbSNP, NHLBI GO Exome Sequencing Project, Clinseq database, and an in-house database with over 500 exomes and selected based on autosomal dominant inheritance.

Whole exome sequencing

Whole exome sequencing and data analysis was performed as previously described41. Novel candidate variants were filtered by ExAC, 1000 genomes, dbSNP, NHLBI GO Exome Sequencing Project, Clinseq database, and an in-house database with over 500 exomes and selected based on autosomal dominant inheritance.

Targeted sequencingTargeted sequencing of TNFAIP3 was performed on pooled DNA from 384 Turkish and 384 Japanese Behcet’s patients with adult-onset disease. Pooled DNA was prepared as previously described42. Ten amplicons covering the 9 exons of TNFAIP3 were generated using high fidelity polymerase AccuPrime™ Pfx SuperMix (Invitrogen). PCR products were pooled and prepared for library using Ovation® Ultralow Library Systems (NuGEN). Paired-end 300 bp reads were sequenced on a MiSeq instrument (Illumina). We called SNPs and INDELs, and calculated the frequencies from the pileup file by an in-house Perl script to identify SNPs and INDELs with frequency ≥1%.

Targeted sequencing

Targeted sequencing of TNFAIP3 was performed on pooled DNA from 384 Turkish and 384 Japanese Behcet’s patients with adult-onset disease. Pooled DNA was prepared as previously described42. Ten amplicons covering the 9 exons of TNFAIP3 were generated using high fidelity polymerase AccuPrime™ Pfx SuperMix (Invitrogen). PCR products were pooled and prepared for library using Ovation® Ultralow Library Systems (NuGEN). Paired-end 300 bp reads were sequenced on a MiSeq instrument (Illumina). We called SNPs and INDELs, and calculated the frequencies from the pileup file by an in-house Perl script to identify SNPs and INDELs with frequency ≥1%.

Sanger sequencingSanger sequencing was used to confirm mutations identified by exome sequencing, targeted sequencing and for the candidate gene screening as previously described41.

Sanger sequencing

Sanger sequencing was used to confirm mutations identified by exome sequencing, targeted sequencing and for the candidate gene screening as previously described41.

Haplotype analysisSNPs within the TNFAIP3 gene and STR markers across the chromosome 6 were genotyped for all family members from family 1. Haplotypes were reconstructed by PHASE and MERLIN.

Haplotype analysis

SNPs within the TNFAIP3 gene and STR markers across the chromosome 6 were genotyped for all family members from family 1. Haplotypes were reconstructed by PHASE and MERLIN.

Antibodies and expression plasmidsAntibodies specific for TNFAIP3 N-terminus (sc-166692), TRAF6 (sc-8409, sc-7221), Actin (sc-1615), Ubc13 (sc-376470, sc-292618), TNFR1 (sc-8436), RIP (sc-7881, sc-133102), NEMO (sc-56919, sc-8330), Ub (sc-271289), Caspase-1 (sc-515 and sc-622), GFP (sc-8334), HA-probe (sc-7392), c-MYC (sc-40) from Santa Cruz; TNFAIP3 N-terminal specific (#5630), K63-linkage Polyubiquitin (#5621), Phospho-IKKα/β (#2697), IKKα (#11930), IκBα (#4814, #9242), Phospho-IκBα (#2859), NFκB P65 (#8242), TNFR1 (#3736), RIP (#3493), Phospho-p38 MAPK (#4511), Phospho-p44/42 MAPK (Erk1/2) (#4370), Phospho-SAPK/JNK (#4668), p44/42 MAPK (Erk1/2) (#4695), SAPK/JNK (#9252), p38 MAPK (#8690), HRP-linked anti-rabbit IgG (#7074), HRP-linked anti-mouse IgG (#7076) from Cell Signaling; IL-1 β (AF-201-NA) from R&D Systems; NLRP3 (ALX-804-819-C100) from Enzo Life Sciences. pRK5-HA-Ubiquitin-K63 (Addgene plasmid # 17606) was a gift from Ted Dawson43. pEGFP-C1-TNFAIP3 (Addgene plasmid # 22141) was a gift from Yihong Ye 44. Myc-DDK-tagged-human RIPK1 (RC216024), Myc-DDK-tagged-human NFKBIA (RC200711), untagged human TRAF6 (sc109845) were from Origene. 3xFlag-TNFAIP3 (Ex-K6040-M120) was from Genecopoeia. pEF-NEMO was a gift from Dr. Chi Ma45. The GFP tagged or 3xFlag-TNFAIP3 mutant plasmids were constructed by PCR mutagenesis.

Antibodies and expression plasmids

Antibodies specific for TNFAIP3 N-terminus (sc-166692), TRAF6 (sc-8409, sc-7221), Actin (sc-1615), Ubc13 (sc-376470, sc-292618), TNFR1 (sc-8436), RIP (sc-7881, sc-133102), NEMO (sc-56919, sc-8330), Ub (sc-271289), Caspase-1 (sc-515 and sc-622), GFP (sc-8334), HA-probe (sc-7392), c-MYC (sc-40) from Santa Cruz; TNFAIP3 N-terminal specific (#5630), K63-linkage Polyubiquitin (#5621), Phospho-IKKα/β (#2697), IKKα (#11930), IκBα (#4814, #9242), Phospho-IκBα (#2859), NFκB P65 (#8242), TNFR1 (#3736), RIP (#3493), Phospho-p38 MAPK (#4511), Phospho-p44/42 MAPK (Erk1/2) (#4370), Phospho-SAPK/JNK (#4668), p44/42 MAPK (Erk1/2) (#4695), SAPK/JNK (#9252), p38 MAPK (#8690), HRP-linked anti-rabbit IgG (#7074), HRP-linked anti-mouse IgG (#7076) from Cell Signaling; IL-1 β (AF-201-NA) from R&D Systems; NLRP3 (ALX-804-819-C100) from Enzo Life Sciences. pRK5-HA-Ubiquitin-K63 (Addgene plasmid # 17606) was a gift from Ted Dawson43. pEGFP-C1-TNFAIP3 (Addgene plasmid # 22141) was a gift from Yihong Ye 44. Myc-DDK-tagged-human RIPK1 (RC216024), Myc-DDK-tagged-human NFKBIA (RC200711), untagged human TRAF6 (sc109845) were from Origene. 3xFlag-TNFAIP3 (Ex-K6040-M120) was from Genecopoeia. pEF-NEMO was a gift from Dr. Chi Ma45. The GFP tagged or 3xFlag-TNFAIP3 mutant plasmids were constructed by PCR mutagenesis.

Cell culturesHEK293T cells (human embryonic kidney cells expressing SV40 large T antigen, negative for mycoplasma, originally obtained from the American Type Culture Collection) and skin fibroblast cells derived from TNFAIP3-deficient patients or normal donors were grown in Dulbecco’s modified Eagle’s medium (Life Technologies) plus 10% fetal calf serum and 1x antibiotics (Life Technologies). Jurkat-T (human leukaemic T lymphoma cell line, negative for mycoplasma, originally obtained from the American Type Culture Collection) cells and human PBMCs were grown in RPMI1640 medium (Life Technologies), plus 10% fetal calf serum (Gemini Bio-Products) and antibiotics. For intracellular TNF staining, pan-T cells purified from PBMCs by negative selection (Miltenyi) were cultured in complete RPMI 1640 in the presence of plate-bound anti-CD3 and soluble anti-CD28 (both 1 μg/mL) for 5 days. For IL-9 staining, rhIL-4 (30 ng/mL), TGF-β (5 ng/mL), IL-1β (10 ng/ml), and IL-2 (10 U/ml), together with anti-IFNγ (10 μg/mL) were added to total PBMCs, and cells were cultured for 48 hours under the same conditions.

Cell cultures

HEK293T cells (human embryonic kidney cells expressing SV40 large T antigen, negative for mycoplasma, originally obtained from the American Type Culture Collection) and skin fibroblast cells derived from TNFAIP3-deficient patients or normal donors were grown in Dulbecco’s modified Eagle’s medium (Life Technologies) plus 10% fetal calf serum and 1x antibiotics (Life Technologies). Jurkat-T (human leukaemic T lymphoma cell line, negative for mycoplasma, originally obtained from the American Type Culture Collection) cells and human PBMCs were grown in RPMI1640 medium (Life Technologies), plus 10% fetal calf serum (Gemini Bio-Products) and antibiotics. For intracellular TNF staining, pan-T cells purified from PBMCs by negative selection (Miltenyi) were cultured in complete RPMI 1640 in the presence of plate-bound anti-CD3 and soluble anti-CD28 (both 1 μg/mL) for 5 days. For IL-9 staining, rhIL-4 (30 ng/mL), TGF-β (5 ng/mL), IL-1β (10 ng/ml), and IL-2 (10 U/ml), together with anti-IFNγ (10 μg/mL) were added to total PBMCs, and cells were cultured for 48 hours under the same conditions.

NFκB-dependent reporter assay and in vivo deubiquitination assayPreformulated Cignal NFkB Pathway Reporter plasmids (Qiagen) (pGL4.32[luc2P/NF-κBRE/Hygro] luciferase reporter plus The pRLCMV-Renilla vector) were used to co-transfect in 293T cells with equal amount of expression plasmids for GFP-tag TNFAIP3 wild type or mutants using Polyjet transfection reagents (Signagen). After 24 hrs, cells were left untreated or stimulated with TNF (20 ng/ml). Luciferase activity (the firefly (luc) and renilla luciferase (ren) activity) was measured 5 hours afterwards by the dual-glo luciferase assay system (Promega). Results for NFkB activity are expressed as fold induction by normalizing firefly luciferase activity to Renilla luciferase activity.For deubiquitination assay, 293T cells were transfected with 1 μg plasmids expressing target protein (RIP1, NEMO, or TRAF6), together with 0.5 μg HA-K63Ub, 0.5 ug GFP-tagged TNFAIP3 wild type or mutants. Empty vectors were used to equalize the plasmid transfection amount. Cells were harvested 48 hrs after transfection. Cell lysates were prepared and immunoprecipitation was conducted as previously described46.

NFκB-dependent reporter assay and in vivo deubiquitination assay

Preformulated Cignal NFkB Pathway Reporter plasmids (Qiagen) (pGL4.32[luc2P/NF-κBRE/Hygro] luciferase reporter plus The pRLCMV-Renilla vector) were used to co-transfect in 293T cells with equal amount of expression plasmids for GFP-tag TNFAIP3 wild type or mutants using Polyjet transfection reagents (Signagen). After 24 hrs, cells were left untreated or stimulated with TNF (20 ng/ml). Luciferase activity (the firefly (luc) and renilla luciferase (ren) activity) was measured 5 hours afterwards by the dual-glo luciferase assay system (Promega). Results for NFkB activity are expressed as fold induction by normalizing firefly luciferase activity to Renilla luciferase activity.

For deubiquitination assay, 293T cells were transfected with 1 μg plasmids expressing target protein (RIP1, NEMO, or TRAF6), together with 0.5 μg HA-K63Ub, 0.5 ug GFP-tagged TNFAIP3 wild type or mutants. Empty vectors were used to equalize the plasmid transfection amount. Cells were harvested 48 hrs after transfection. Cell lysates were prepared and immunoprecipitation was conducted as previously described46.

NFκB reporter assay in Jurkat T cell line5×106 3T8 cells (Jurkat T-cell line) were transiently transfected with 20 μg of expression plasmids encoding either wildtype TNFAIP3-GFP or mutant TNFAIP3-GFP fusion genes by electroporation. Sixteen hours post transfection, 20 ng/ml of human TNF was added to transfected 3T8 cells. Cells were collected 16 hours after the addition of TNF and surface stained with anti-Thy1-PECy7 and live/dead-violet for FACS analysis. Samples were acquired with a FACSVerse (Becton Dickinson Biosciences) and data were analyzed with FlowJo. The mean fluorescence intensity (MFI) of Thy1, the indicator of NFκB activity, was gated on the GFP+ live singlet populations.

NFκB reporter assay in Jurkat T cell line

5×106 3T8 cells (Jurkat T-cell line) were transiently transfected with 20 μg of expression plasmids encoding either wildtype TNFAIP3-GFP or mutant TNFAIP3-GFP fusion genes by electroporation. Sixteen hours post transfection, 20 ng/ml of human TNF was added to transfected 3T8 cells. Cells were collected 16 hours after the addition of TNF and surface stained with anti-Thy1-PECy7 and live/dead-violet for FACS analysis. Samples were acquired with a FACSVerse (Becton Dickinson Biosciences) and data were analyzed with FlowJo. The mean fluorescence intensity (MFI) of Thy1, the indicator of NFκB activity, was gated on the GFP+ live singlet populations.

PBMC preparation and stimulation, immunoprecipitation and immunoblottingHeparinated peripheral blood mononuclear cells (PBMCs) were separated by Ficoll (Ficoll-Paque PLUS; GE Healthcare) gradient centrifugation. Recombinant human TNF (Peprotech) was used to stimulate PBMC (10 ng/ml) or fibroblast cells (20 ng/ml) from 5 min to 1 hr. Whole cell lysis were prepared using ice-cold 1xcell lysis buffer (Cell Signaling) supplemented with complete protease inhibitors. Nuclear extraction kit (Cayman) was used to prepare cytosolic and nuclear fractions. Immunoprecipitation and immunoblotting were conducted as described previously4547 with specific antibodies. ImageJ was used to analyze the immunoblotting images.

PBMC preparation and stimulation, immunoprecipitation and immunoblotting

Heparinated peripheral blood mononuclear cells (PBMCs) were separated by Ficoll (Ficoll-Paque PLUS; GE Healthcare) gradient centrifugation. Recombinant human TNF (Peprotech) was used to stimulate PBMC (10 ng/ml) or fibroblast cells (20 ng/ml) from 5 min to 1 hr. Whole cell lysis were prepared using ice-cold 1xcell lysis buffer (Cell Signaling) supplemented with complete protease inhibitors. Nuclear extraction kit (Cayman) was used to prepare cytosolic and nuclear fractions. Immunoprecipitation and immunoblotting were conducted as described previously4547 with specific antibodies. ImageJ was used to analyze the immunoblotting images.

Inflammasome inhibitionPBMCs (2×106 cells/well) were plated in 12-well plates and then primed with 1 μg ml-1 LPS in RPMI 1640 (Invitrogen) containing 10% FBS for 3 hrs. For the inhibition of inflammasome activation, LPS primed-PBMCs were treated with U73122 (5 μM), U73343 (5 μM), NKH477 (100 or 250 μM), or MCC950 (with three different doses: 50, 100, and 500 nM) in RPMI 1640. After 1 hr of treatment, supernatants and cell lysates were collected for immunoblot analysis.

Inflammasome inhibition

PBMCs (2×106 cells/well) were plated in 12-well plates and then primed with 1 μg ml-1 LPS in RPMI 1640 (Invitrogen) containing 10% FBS for 3 hrs. For the inhibition of inflammasome activation, LPS primed-PBMCs were treated with U73122 (5 μM), U73343 (5 μM), NKH477 (100 or 250 μM), or MCC950 (with three different doses: 50, 100, and 500 nM) in RPMI 1640. After 1 hr of treatment, supernatants and cell lysates were collected for immunoblot analysis.

ImmunocytochemistryFibroblasts grown overnight in 96-well plates or on poly-L-lysine coated coverslips and treated with TNF were fixed in 4% paraformaldehyde and stained for NFκB p65. Primary (Cell Signaling 8242) and secondary antibodies (A21206 Molecular Probes) were incubated overnight at 4 °C and for 30 minutes at room temperature, respectively. Coverslips were mounted with Vectashield containing DAPI (H-1200, Vector Laboratories), imaged with a Zeiss AxioImager D2 using a 20x objective. 96-well plates were imaged with an Incucyte microscope (Essen Bioscience) using a 20x objective and a SYTO 59 (Molecular Probes) nuclear stain. Nuclear patterns were converted to regions of interest in ImageJ and used to measure nuclear p65 intensities. Data were analyzed in GraphPad Prism 6 and shown as mean ±S.E.M.

Immunocytochemistry

Fibroblasts grown overnight in 96-well plates or on poly-L-lysine coated coverslips and treated with TNF were fixed in 4% paraformaldehyde and stained for NFκB p65. Primary (Cell Signaling 8242) and secondary antibodies (A21206 Molecular Probes) were incubated overnight at 4 °C and for 30 minutes at room temperature, respectively. Coverslips were mounted with Vectashield containing DAPI (H-1200, Vector Laboratories), imaged with a Zeiss AxioImager D2 using a 20x objective. 96-well plates were imaged with an Incucyte microscope (Essen Bioscience) using a 20x objective and a SYTO 59 (Molecular Probes) nuclear stain. Nuclear patterns were converted to regions of interest in ImageJ and used to measure nuclear p65 intensities. Data were analyzed in GraphPad Prism 6 and shown as mean ±S.E.M.

Immune cell cytokine production and serum cytokine detectionPBMCs (2×106 /ml) were stimulated with bacterial lipopolysaccharide (LPS) (Sigma), flagellin (Sigma), and Staphylococcal enterotoxin B (SEB) (Sigma) for 24 hour at 37°C. The concentrations of cytokines in the supernatants of stimulated and non-stimulated PBMCs and serum were determined using Bio-Plex Pro Human Cytokine 27-plex and 21-plex immunoassay kits (Bio-Rad Hercules). The Bio-plex pro human cytokine standard group I and group II were used as standards for the assays. The differences in the cytokine concentrations were statistically analyzed using the Mann-Whitney test, and plotted with GraphPad Prism software package.

Immune cell cytokine production and serum cytokine detection

PBMCs (2×106 /ml) were stimulated with bacterial lipopolysaccharide (LPS) (Sigma), flagellin (Sigma), and Staphylococcal enterotoxin B (SEB) (Sigma) for 24 hour at 37°C. The concentrations of cytokines in the supernatants of stimulated and non-stimulated PBMCs and serum were determined using Bio-Plex Pro Human Cytokine 27-plex and 21-plex immunoassay kits (Bio-Rad Hercules). The Bio-plex pro human cytokine standard group I and group II were used as standards for the assays. The differences in the cytokine concentrations were statistically analyzed using the Mann-Whitney test, and plotted with GraphPad Prism software package.

Primary human monocyte isolation, differentiation and stimulationMonocytes were purified from PBMCs by negative selection (Monocyte Isolation Kit II; Miltenyi Biotec). Monocytes were suspended in monocyte attachment medium (PromoCell) and seeded at a density of 150,000/cm2 for 2 hours. Monocytes were co-stained with APC-Cy7-conjugated anti-CD11b, PE-Cy7-conjugated CD11c, Fitc conjugated anti-CD14, APC-conjugated CD16 or relative isotype control and then were analyzed using a FACS Canton II (BD Biosciences).Primary human monocytes were differentiated into M1 macrophages using 20 ng/mL human Granulocyte-macrophage colony-stimulating factor (hGM-CSF, Sigma) or into M2 macrophages using 100 ng/mL human macrophage colony-stimulating factor (hM-CSF, Sigma) for 10 days. hGM-CSF differentiated M1 macrophages were stimulated with or without LPS (Sigma) 100 ng/mL for 6 hours.

Primary human monocyte isolation, differentiation and stimulation

Monocytes were purified from PBMCs by negative selection (Monocyte Isolation Kit II; Miltenyi Biotec). Monocytes were suspended in monocyte attachment medium (PromoCell) and seeded at a density of 150,000/cm2 for 2 hours. Monocytes were co-stained with APC-Cy7-conjugated anti-CD11b, PE-Cy7-conjugated CD11c, Fitc conjugated anti-CD14, APC-conjugated CD16 or relative isotype control and then were analyzed using a FACS Canton II (BD Biosciences).

Primary human monocytes were differentiated into M1 macrophages using 20 ng/mL human Granulocyte-macrophage colony-stimulating factor (hGM-CSF, Sigma) or into M2 macrophages using 100 ng/mL human macrophage colony-stimulating factor (hM-CSF, Sigma) for 10 days. hGM-CSF differentiated M1 macrophages were stimulated with or without LPS (Sigma) 100 ng/mL for 6 hours.

Real-time PCRRNA was isolated from M1 macrophages using the RNeasy kit (Qiagen) and isolated from unstimulated PBMC using TRIzol. Reverse transcription was performed using the Taqman kit (Applied Biosystems). Real-time PCR reactions were performed using the iQ Syber Green kit (Biorad) or TaqMan gene expression master mix and TaqMan gene expression assays probes. 18S RNA served as control. The reactions were run on an Applied Biosystems 7500 Real-Time PCR System (Life Technologies).

Real-time PCR

RNA was isolated from M1 macrophages using the RNeasy kit (Qiagen) and isolated from unstimulated PBMC using TRIzol. Reverse transcription was performed using the Taqman kit (Applied Biosystems). Real-time PCR reactions were performed using the iQ Syber Green kit (Biorad) or TaqMan gene expression master mix and TaqMan gene expression assays probes. 18S RNA served as control. The reactions were run on an Applied Biosystems 7500 Real-Time PCR System (Life Technologies).

Intracellular cytokine staining in T cellsIntracellular cytokine staining for IL-17A, TNF, IL-4, and IL-9 following either PMA and ionomycin stimulation (IL-17A, IL-4, and IL-9) or SEB stimulation (TNF) was performed as previously described.48 All events were collected on an LSRFortessa (BD Biosciences) and analyzed with FlowJo (Treestar).

Intracellular cytokine staining in T cells

Intracellular cytokine staining for IL-17A, TNF, IL-4, and IL-9 following either PMA and ionomycin stimulation (IL-17A, IL-4, and IL-9) or SEB stimulation (TNF) was performed as previously described.48 All events were collected on an LSRFortessa (BD Biosciences) and analyzed with FlowJo (Treestar).

Supplementary Material12

Supplementary Material
