---
pmid: '25118933'
title: The protease Omi regulates mitochondrial biogenesis through the GSK3β/PGC-1α
  pathway.
authors:
- Xu R
- Hu Q
- Ma Q
- Liu C
- Wang G
journal: Cell Death Dis
year: '2014'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4454303
doi: 10.1038/cddis.2014.328
---

# The protease Omi regulates mitochondrial biogenesis through the GSK3β/PGC-1α pathway.
**Authors:** Xu R, Hu Q, Ma Q, Liu C, Wang G
**Journal:** Cell Death Dis (2014)
**DOI:** [10.1038/cddis.2014.328](https://doi.org/10.1038/cddis.2014.328)
**PMC:** [PMC4454303](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4454303/)

## Abstract

1. Cell Death Dis. 2014 Aug 14;5(8):e1373. doi: 10.1038/cddis.2014.328.

The protease Omi regulates mitochondrial biogenesis through the GSK3β/PGC-1α 
pathway.

Xu R(1), Hu Q(2), Ma Q(2), Liu C(3), Wang G(4).

Author information:
(1)Laboratory of Molecular Neuropathology, Key Laboratory of Brain Function and 
Diseases and School of Life Sciences, University of Science and Technology of 
China, Chinese Academy of Sciences, Hefei, Anhui, China.
(2)Laboratory of Molecular Neuropathology, Jiangsu Key Laboratory of 
Translational Research and Therapy for Neuro-Psycho-Diseases and College of 
Pharmaceutical Sciences, Suzhou, Jiangsu, China.
(3)Institute of Neuroscience, Soochow University, Suzhou, Jiangsu, China.
(4)1] Laboratory of Molecular Neuropathology, Key Laboratory of Brain Function 
and Diseases and School of Life Sciences, University of Science and Technology 
of China, Chinese Academy of Sciences, Hefei, Anhui, China [2] Laboratory of 
Molecular Neuropathology, Jiangsu Key Laboratory of Translational Research and 
Therapy for Neuro-Psycho-Diseases and College of Pharmaceutical Sciences, 
Suzhou, Jiangsu, China.

Loss of the mitochondrial protease activity of Omi causes mitochondrial 
dysfunction, neurodegeneration with parkinsonian features and premature death in 
mnd2 (motor neuron degeneration 2) mice. However, the detailed mechanisms 
underlying this pathology remain largely unknown. Here, we report that Omi 
participates in the process of mitochondrial biogenesis, which has been linked 
to several neurodegenerative diseases. The mitochondrial biogenesis is deficit 
in mnd2 mice, evidenced by severe decreases of mitochondrial components, 
mitochondrial DNA and mitochondrial density. Omi cleaves glycogen synthase 
kinase 3β (GSK3β), a kinase promoting PPARγ coactivator-1α (PGC-1α) degradation, 
to regulate PGC-1α, a factor important for the mitochondrial biogenesis. In mnd2 
mice, GSK3β abundance is increased and PGC-1α abundance is decreased 
significantly. Inhibition of GSK3β by SB216763 or overexpression of PGC-1α can 
restore mitochondrial biogenesis in mnd2 mice or Omi-knockdown N2a cells. 
Furthermore, there is a significant improvement of the movement ability of mnd2 
mice after SB216763 treatment. Thus, our study identified Omi as a novel 
regulator of mitochondrial biogenesis, involving in Omi 
protease-deficient-induced neurodegeneration.

DOI: 10.1038/cddis.2014.328
PMCID: PMC4454303
PMID: 25118933 [Indexed for MEDLINE]

## Full Text

Mitochondria have a vital role in neuronal death and survival.1 As critical cellular organelles, mitochondria have highly dynamic properties, including mitochondrial fission, fusion, transport, biogenesis and degradation. The changes of those properties affect mitochondrial functions, leading to the occurrence of diseases.2, 3 Growing lines of evidence suggest that the mitochondrial dysfunction is involved in aging and neurodegenerative diseases, such as Alzheimer's disease (AD), Huntington's disease (HD) and Parkinson's disease (PD).4, 5 Similar to other neurodegenerative diseases, PD is a progressive neurological disorder, which is characterized by the development of cytoplasmic aggregates known as Lewy bodies and degeneration of dopaminergic (DA) neurons in the substantia nigra of midbrain and other brain regions.6 In PD, dysfunction of mitochondria has been documented to be associated with disease pathogenesis in PD brains and both genetic- and toxin-induced PD animal models. In PD brains, mutations in mitochondrial DNA (mtDNA) occur more frequently than those in age-matched control; and mutations in the nuclear-encoded mtDNA polymerase-γ gene, which impair mtDNA replication and result in multiple mtDNA deletions, cause PD-like symptoms.5 Meanwhile, several PD-associated gene products, including α-synuclein, parkin, DJ-1, PINK1 (PTEN-induced putative kinase 1), leucine-rich repeat kinase 2, ubiquitin carboxy-terminal hydrolase L1 and Omi, have been identified to be associated with PD, and lead to mitochondrial dysfunction with changes in mitochondrial morphology, biogenesis and mitophagy in vivo and in vitro.5, 7, 8, 9 Besides, mitochondrial toxins, such as MPTP (1-Methyl-4-phenyl-1,2,3,6-tetrahydropyridine) and rotenone that inhibit complex I of the mitochondrial respiratory chain, cause clinically parkinsonian phenotype.10, 11

The serine protease Omi (also known as HtrA2) belongs to the high-temperature requirement factor A (HtrA) family, and was originally identified as a mammalian homolog of the Escherichia coli heat-shock-induced serine protease HtrA/DegP and DegS.12 Omi is mainly localized in mitochondria, although a fraction of it is also found in nucleus.13 Omi is released from the mitochondria into the cytosol to mediate cell death by caspase-dependent or -independent pathways in response to apoptotic stimuli.14, 15 However, the notion that Omi is an apoptosis inducer in the central nervous system was challenged by studies of Omi-overexpressing or -deficient mice. Omi-overexpressing mice show normal development without any sign of apoptotic cell death.16 On the other hand, mnd2 (motor neuron degeneration 2) mice that harbor protease-deficient Omi S276C mutants, and Omi-knockout mice both suffer from progressive neurodegeneration, especially in striatum, and motor abnormalities similar to PD. Both mice fail to gain weight and die before postnatal day 40 due to neurodegeneration with progressive mitochondrial damage.17, 18, 19 Besides, mutations in the Omi gene have also been identified in PD patients.20, 21 Previous studies have shown that Omi has a vital role in the mitochondrial integrity, and the loss of protease activity leads to mitochondrial dysfunction, such as abnormal mitochondrial morphology and increased mtDNA mutation and deletions, increased susceptibility of mitochondrial membrane permeabilization, decreased mitochondrial membrane potential, and reduced mitochondrial density in mnd2 mice and Omi-knockout mice.17, 18, 22 Omi has been found to act downstream of PINK1, but parallel to parkin, in a mitochondrial stress sensing pathway to sense the different stresses, which may be defective in PD.23 These findings suggest that the primary function of Omi is involved in neuroprotection, especially in the maintenance of mitochondrial homeostasis.23, 24

In this article, we identified that Omi cleaves glycogen synthase kinase 3β (GSK3β) to regulate PPARγ coactivator-1α (PGC-1α) abundance and to ensure mitochondrial biogenesis.

ResultsOmi regulates mitochondrial componentsRecent findings show that mitochondrial density is likely to be reduced in tissues derived from Omi-knockout mice,19 and that the number of mitochondria, displaying reduced electron density and open cristae, is increased in Omi-deletion mutant fly.25 To determine whether mitochondrial components are altered when Omi protease activity is deficient, we examined the expressions of genes involved in mitochondrial respiratory chain function and oxidant metabolism, from the brains of 25-day-old mnd2 mice and age-matched wide-type mice using real-time quantitative PCR (RT-qPCR). The mRNA levels of the essential components of the respiration chain, including the nuclearly encoded mitochondrial ATP synthase 5 β subunit (ATP5B) and cytochorome c oxidase subunit IV (COX IV), the mitochondrially encoded cytochrome c oxidase subunit II (COX II) and cytochrome c (Cyt C), and the component of mitochondrial oxidant metabolism, adenine nucleotide translocase-1 (ANT1), were all decreased significantly in mnd2 mice compared with those in wide-type mice (Figure 1a). We next performed western blotting to analyze the abundance of COX IV and Cyt C at protein levels. Consistence with RT-qPCR analyses, the protein levels of COX IV and Cyt C were also decreased (Figure 1b). Meanwhile, in Omi-knockdown SH-SY5Y cells (Figure 1c), a human neuroblastoma cell line, and N2a cells (Supplementary Figure S1), a mouse neuroblastoma cell line, the abundance of COX IV and Cyt C was also reduced, similar to the alterations in mnd2 mice. Further, we detected the amount of mtDNA copy number in brains of mnd2 mice using RT-qPCR. The β-globin gene was used as the target template for the determination of nuclear DNA (nDNA); five different mitochondrial-derived genes, that is, mitochondrial NADH dehydrogenase 1 (mNADH1), mitochondrial cytochrome B, mitochondrial NADH dehydrogenase 4 (mNADH4), mitochondrial ATP synthase 6 (mATP6) and mitochondrial cytochrome oxidase subunit II (mCOX II) were used in the determination of mtDNA quantities. By calculating the ratio of the mitochondrial genes to the nDNA (mtDNA/nDNA), we found that mtDNA copy number was decreased dramatically in mnd2 mice compared with that in wide-type mice (Figure 1d). Moreover, to further identify whether mitochondrial number is certainly reduced in mnd2 mice, we performed transmission electron microscopic analyses to examine mitochondria in striatum from 25-day-old mnd2 mice, where neurodegeneration is most prominent. The mnd2 mice showed an obviously decreased number of mitochondria when compared with wide-type mice (Figure 1e). Quantitative morphometry of transmission electron micrographs confirmed that the mitochondrial density (mitochondrial number/cytoplasmic area) was significantly lower in mnd2 mice than that in wide-type mice (Figure 1f). Thus, our data suggest that Omi affects mitochondrial components and density.Omi regulates PGC-1α levelsAs mitochondrial components and density were decreased in mnd2 mice compared with those in wide-type mice, we wonder that whether mitochondrial biogenesis is decreased in mnd2 mice. It was reported that PGC-1α has a key role in the mitochondrial biogenesis.26, 27 We, therefore, examined the levels of PGC-1α in mnd2 mice. The abundance of PGC-1α was significantly decreased in mnd2 mouse brain and spinal cord (Figure 2a), suggesting that the loss of protease activity of Omi may be associated with the reduction of PGC-1α. We next transfected N2a cells with hemagglutinin (HA)-tagged full-length wild-type or S276C mutant Omi. The overexpression of wide-type Omi resulted in an increased abundance of PGC-1α, whereas the overexpression of S276C Omi did not change PGC-1α abundance (Figure 2b). Moreover, in Omi-knockdown SH-SY5Y cells (Figure 2c) and N2a cells (Supplementary Figure S1), the abundance of PGC-1α was also decreased, consistent with the data from mnd2 mice. Besides, immunoprecipitation assays showed that endogenous Omi did not interact with endogenous PGC-1α in the brain of mice (Supplementary Figure S2), suggesting no direct interaction between Omi and PGC-1α. We further examined two PGC-1α target genes, nuclear respiratory factor-1 (NRF-1) and mitochondrial transcription factor A (TFAM), which are important factors for mitochondrial biogenesis. We found that the levels of NRF-1 and TFAM mRNA were reduced in mnd2 mouse brain compared with those in wide-type mouse brain (Figure 2d). In addition, we found no alteration in the levels of PGC-1α mRNA in mnd2 mouse brain versus wide-type mouse brain (Figure 2d), suggesting that Omi may regulate PGC-1α abundance at protein levels, and that the regulation depends on Omi protease activity. The above data suggest that Omi regulates mitochondrial biogenesis.Omi represses GSK3β expressionAs we found that PGC-1α is decreased in mnd2 mice, we wonder whether some factors that regulate PGC-1α are changed in mnd2 mice. We therefore examined regulators that influence PCG-1α abundance or activity, including p38, GSK3β, p70s6k (ribosomal protein S6 kinase), AKT, pAMPK (AMP-activated protein kinase), Sirtuin-1 (SIRT-1) and Cdc 4 (Supplementary Figures S3A and B). Within them, GSK3β, which can phosphorylate PGC-1α and promote its ubiquitin-mediated degradation (Supplementary Figures S4A and B),28, 29 was upregulated significantly in mnd2 mouse brain and spinal cord compared with that in wide-type mice (Figure 3a). We next investigated whether Omi downregulates GSK3β in vitro. In N2a cells, the overexpression of wide-type full-length Omi, but not S276C mutant Omi, reduced the endogenous GSK3β abundance (Figure 3b), as well as enhanced green fluorescent protein (EGFP)-, Flag- and HA-tagged GSK3β (Supplementary Figures S5A, B and C), whereas knockdown of Omi increased the abundances of GSK3β in SH-SY5Y cells (Figure 3c) and N2a cells (Supplementary Figure S1). In addition, we examined the mRNA levels of GSK3β in brains of mnd2 mice and found that there was no difference between mnd2 mice and control mice (Figure 3d).GSK3β is the substrate of OmiAs the increase of GSK3β abundace in both mnd2 mouse brain and Omi-knockdown cells is associated with Omi protease activity and the mRNA levels of GSK3β is not changed in mnd2 mice, we wonder whether GSK3β is a substrate of Omi. First, we examine the subcellular localization of GSK3β and Omi in SH-SY5Y and N2a cells. Subcellular fractionation assay showed that GSK3β was localized in both cytosolic and mitochondrial fractions, and Omi was localized in mitochondrial fraction (Supplementary Figure S6), suggesting that GSK3β and Omi may interact in mitochondria physiologically. Next, we examined the interactions between Omi and GSK3β. Glutathione S-transferase (GST) pulldown assays showed that GST-GSK3β pulled His-tagged Omi (His-Omi) down, whereas GST alone did not (Figure 4a). Besides, GST-S276C mutant Omi pulled down endogenous GSK3β from brain lysates, whereas GST alone did not (Figure 4b). We further performed immunoprecipitation assays using N2a cells, in which HA-tagged full-length S276C mutant Omi and EGFP-GSK3β were co-transfected, to further confirm the interactions between Omi and GSK3β. As shown in Figure 4c, EGFP-GSK3β was coimmunoprecipitated with Omi when endogenous Omi or HA-Omi was precipitated with Omi-specific antibodies. These data indicate that Omi interacts with GSK3β directly in vitro. Moreover, endogenous Omi was coimmunoprecipitated with endogenous GSK3β after endogenous GSK3β was immunoprecipitated from mnd2 mouse brain lysates with a GSK3β-specific antibody (Figure 4d), further suggesting that Omi interacts with GSK3β under normal physiological conditions. Last, in vitro cleavage assay was performed to identify whether GSK3β is a substrate of Omi directly. The results showed that GST-GSK3β was cleaved by wide-type Omi but not by S276C Omi (Figure 4e), whereas GST alone was not cleaved by Omi (Supplementary Figure S7A). Meanwhile, the Omi substrate β-casein was cleaved by the wild-type Omi, but not S276C Omi, which demonstrated a protease activity of Omi (Supplementary Figure S7B). Taken together, these data indicate that GSK3β is a substrate of Omi.Regulation of PGC-1α by Omi is mediated by GSK3βWe already found that Omi decreases PGC-1α and cleaves GSK3β. As GSK3β is a kinase involved in PGC-1α degradation, we wonder whether GSK3β mediates the downregulation of PGC-1α by Omi. We therefore examined PGC-1α levels, in the presence or absence of a GSK3β specific inhibitor SB216763. In SH-SY5Y cells in which Omi was knocked down, a significant decrease in PGC-1α abundance was observed; however, the effect of Omi knockdown on PGC-1α abundance was blocked by SB216763 (Figure 5a). Similar results were observed in mnd2 mice after SB216763 treatment (Figure 5b). Besides, the decreased abundance of PGC-1α caused by Omi knockdown can also be blocked by the knockdown of GSK3β (Figure 5c). Moreover, immunoprecipitation assays showed that the interaction of PGC-1α and Cdc4, a component of SCF-Cdc4 E3 ligase that regulates PGC-1α for the ubiquitin-mediated degradation, was significantly increased in dimethylsulfoxide (DMSO)-treated mnd2 mice as compared with wild-type mice; however, the interaction of PGC-1α and Cdc4 was decreased in mnd2 mice after SB216763 treatment (Figure 5d), suggesting that Omi protease activity influences the interaction between PGC-1α and its associated SCF-Cdc4 E3 ligase complex in UPS pathway and that GSK3β affects the interaction, evidenced by the data obtained from the usage of GSK3β inhibitor. Thus, our data suggest that GSK3β mediates Omi-regulated PGC-1α abundance.Inhibition of GSK3β and overexpression of PGC-1α restore mitochondrial biogenesis influenced by OmiAs GSK3β is effective on PGC-1α abundance at the downstream of Omi, we wonder whether the inhibition of GSK3β is able to restore mitochondrial biogenesis in vivo and in vitro. We therefore treated mnd2 mice with GSK3β inhibitor SB216763 and detected the state of mitochondrial biogenesis. We found that the mRNA levels of ATP5B, ANT1, COX II, COX IV, CytC, NRF-1α and TFAM were significantly increased in mnd2 mice (Figure 6a). Consistence with RT-qPCR analyses, the protein levels of COX IV and Cyt C were also restored significantly (Figure 6b). Further, the inhibition of GSK3β significantly increased mtDNA in mnd2 mice (Figure 6c). Similar results were obtained in PGC-1α-overexpressing N2a cells, in which Omi was knocked down (Figure 6d). Moreover, transmission electron microscopic analyses showed that the mitochondria number was increased significantly in striatum in mnd2 mice after a treatment with SB216763 as compared with that in mnd2 mice without SB216763 treatment (Figure 6e). Quantitative morphometry of transmission electron micrographs confirmed that the mitochondrial density was significantly recovered in mnd2 mice treatment with SB216763 than that in mnd2 mice without SB216763 treatment (Figure 6f). Thus, our data suggest that inhibition of GSK3β and overexpression of PGC-1α significantly restore mitochondrial biogenesis that is influenced by Omi deficiency.Inhibition of GSK3β improves movement ability of mnd2 miceAs inhibition of GSK3β is able to restore mitochondrial biogenesis influenced by Omi in vivo and in vitro, we wonder whether there is a functional improvement in mnd2 mice after SB216763 treatment. We therefore performed the rotarod test, which is widely used to evaluate the overall motor deficits in rodent models of PD and others.30 We found that the performance of mnd2 mice was severely deteriorated compared with wide-type mice; however, the performance was significantly improved when the mnd2 mice was administrated with SB216763 (Figure 7).

Omi regulates mitochondrial componentsRecent findings show that mitochondrial density is likely to be reduced in tissues derived from Omi-knockout mice,19 and that the number of mitochondria, displaying reduced electron density and open cristae, is increased in Omi-deletion mutant fly.25 To determine whether mitochondrial components are altered when Omi protease activity is deficient, we examined the expressions of genes involved in mitochondrial respiratory chain function and oxidant metabolism, from the brains of 25-day-old mnd2 mice and age-matched wide-type mice using real-time quantitative PCR (RT-qPCR). The mRNA levels of the essential components of the respiration chain, including the nuclearly encoded mitochondrial ATP synthase 5 β subunit (ATP5B) and cytochorome c oxidase subunit IV (COX IV), the mitochondrially encoded cytochrome c oxidase subunit II (COX II) and cytochrome c (Cyt C), and the component of mitochondrial oxidant metabolism, adenine nucleotide translocase-1 (ANT1), were all decreased significantly in mnd2 mice compared with those in wide-type mice (Figure 1a). We next performed western blotting to analyze the abundance of COX IV and Cyt C at protein levels. Consistence with RT-qPCR analyses, the protein levels of COX IV and Cyt C were also decreased (Figure 1b). Meanwhile, in Omi-knockdown SH-SY5Y cells (Figure 1c), a human neuroblastoma cell line, and N2a cells (Supplementary Figure S1), a mouse neuroblastoma cell line, the abundance of COX IV and Cyt C was also reduced, similar to the alterations in mnd2 mice. Further, we detected the amount of mtDNA copy number in brains of mnd2 mice using RT-qPCR. The β-globin gene was used as the target template for the determination of nuclear DNA (nDNA); five different mitochondrial-derived genes, that is, mitochondrial NADH dehydrogenase 1 (mNADH1), mitochondrial cytochrome B, mitochondrial NADH dehydrogenase 4 (mNADH4), mitochondrial ATP synthase 6 (mATP6) and mitochondrial cytochrome oxidase subunit II (mCOX II) were used in the determination of mtDNA quantities. By calculating the ratio of the mitochondrial genes to the nDNA (mtDNA/nDNA), we found that mtDNA copy number was decreased dramatically in mnd2 mice compared with that in wide-type mice (Figure 1d). Moreover, to further identify whether mitochondrial number is certainly reduced in mnd2 mice, we performed transmission electron microscopic analyses to examine mitochondria in striatum from 25-day-old mnd2 mice, where neurodegeneration is most prominent. The mnd2 mice showed an obviously decreased number of mitochondria when compared with wide-type mice (Figure 1e). Quantitative morphometry of transmission electron micrographs confirmed that the mitochondrial density (mitochondrial number/cytoplasmic area) was significantly lower in mnd2 mice than that in wide-type mice (Figure 1f). Thus, our data suggest that Omi affects mitochondrial components and density.

Omi regulates mitochondrial components

Recent findings show that mitochondrial density is likely to be reduced in tissues derived from Omi-knockout mice,19 and that the number of mitochondria, displaying reduced electron density and open cristae, is increased in Omi-deletion mutant fly.25 To determine whether mitochondrial components are altered when Omi protease activity is deficient, we examined the expressions of genes involved in mitochondrial respiratory chain function and oxidant metabolism, from the brains of 25-day-old mnd2 mice and age-matched wide-type mice using real-time quantitative PCR (RT-qPCR). The mRNA levels of the essential components of the respiration chain, including the nuclearly encoded mitochondrial ATP synthase 5 β subunit (ATP5B) and cytochorome c oxidase subunit IV (COX IV), the mitochondrially encoded cytochrome c oxidase subunit II (COX II) and cytochrome c (Cyt C), and the component of mitochondrial oxidant metabolism, adenine nucleotide translocase-1 (ANT1), were all decreased significantly in mnd2 mice compared with those in wide-type mice (Figure 1a). We next performed western blotting to analyze the abundance of COX IV and Cyt C at protein levels. Consistence with RT-qPCR analyses, the protein levels of COX IV and Cyt C were also decreased (Figure 1b). Meanwhile, in Omi-knockdown SH-SY5Y cells (Figure 1c), a human neuroblastoma cell line, and N2a cells (Supplementary Figure S1), a mouse neuroblastoma cell line, the abundance of COX IV and Cyt C was also reduced, similar to the alterations in mnd2 mice. Further, we detected the amount of mtDNA copy number in brains of mnd2 mice using RT-qPCR. The β-globin gene was used as the target template for the determination of nuclear DNA (nDNA); five different mitochondrial-derived genes, that is, mitochondrial NADH dehydrogenase 1 (mNADH1), mitochondrial cytochrome B, mitochondrial NADH dehydrogenase 4 (mNADH4), mitochondrial ATP synthase 6 (mATP6) and mitochondrial cytochrome oxidase subunit II (mCOX II) were used in the determination of mtDNA quantities. By calculating the ratio of the mitochondrial genes to the nDNA (mtDNA/nDNA), we found that mtDNA copy number was decreased dramatically in mnd2 mice compared with that in wide-type mice (Figure 1d). Moreover, to further identify whether mitochondrial number is certainly reduced in mnd2 mice, we performed transmission electron microscopic analyses to examine mitochondria in striatum from 25-day-old mnd2 mice, where neurodegeneration is most prominent. The mnd2 mice showed an obviously decreased number of mitochondria when compared with wide-type mice (Figure 1e). Quantitative morphometry of transmission electron micrographs confirmed that the mitochondrial density (mitochondrial number/cytoplasmic area) was significantly lower in mnd2 mice than that in wide-type mice (Figure 1f). Thus, our data suggest that Omi affects mitochondrial components and density.

Omi regulates PGC-1α levelsAs mitochondrial components and density were decreased in mnd2 mice compared with those in wide-type mice, we wonder that whether mitochondrial biogenesis is decreased in mnd2 mice. It was reported that PGC-1α has a key role in the mitochondrial biogenesis.26, 27 We, therefore, examined the levels of PGC-1α in mnd2 mice. The abundance of PGC-1α was significantly decreased in mnd2 mouse brain and spinal cord (Figure 2a), suggesting that the loss of protease activity of Omi may be associated with the reduction of PGC-1α. We next transfected N2a cells with hemagglutinin (HA)-tagged full-length wild-type or S276C mutant Omi. The overexpression of wide-type Omi resulted in an increased abundance of PGC-1α, whereas the overexpression of S276C Omi did not change PGC-1α abundance (Figure 2b). Moreover, in Omi-knockdown SH-SY5Y cells (Figure 2c) and N2a cells (Supplementary Figure S1), the abundance of PGC-1α was also decreased, consistent with the data from mnd2 mice. Besides, immunoprecipitation assays showed that endogenous Omi did not interact with endogenous PGC-1α in the brain of mice (Supplementary Figure S2), suggesting no direct interaction between Omi and PGC-1α. We further examined two PGC-1α target genes, nuclear respiratory factor-1 (NRF-1) and mitochondrial transcription factor A (TFAM), which are important factors for mitochondrial biogenesis. We found that the levels of NRF-1 and TFAM mRNA were reduced in mnd2 mouse brain compared with those in wide-type mouse brain (Figure 2d). In addition, we found no alteration in the levels of PGC-1α mRNA in mnd2 mouse brain versus wide-type mouse brain (Figure 2d), suggesting that Omi may regulate PGC-1α abundance at protein levels, and that the regulation depends on Omi protease activity. The above data suggest that Omi regulates mitochondrial biogenesis.

Omi regulates PGC-1α levels

As mitochondrial components and density were decreased in mnd2 mice compared with those in wide-type mice, we wonder that whether mitochondrial biogenesis is decreased in mnd2 mice. It was reported that PGC-1α has a key role in the mitochondrial biogenesis.26, 27 We, therefore, examined the levels of PGC-1α in mnd2 mice. The abundance of PGC-1α was significantly decreased in mnd2 mouse brain and spinal cord (Figure 2a), suggesting that the loss of protease activity of Omi may be associated with the reduction of PGC-1α. We next transfected N2a cells with hemagglutinin (HA)-tagged full-length wild-type or S276C mutant Omi. The overexpression of wide-type Omi resulted in an increased abundance of PGC-1α, whereas the overexpression of S276C Omi did not change PGC-1α abundance (Figure 2b). Moreover, in Omi-knockdown SH-SY5Y cells (Figure 2c) and N2a cells (Supplementary Figure S1), the abundance of PGC-1α was also decreased, consistent with the data from mnd2 mice. Besides, immunoprecipitation assays showed that endogenous Omi did not interact with endogenous PGC-1α in the brain of mice (Supplementary Figure S2), suggesting no direct interaction between Omi and PGC-1α. We further examined two PGC-1α target genes, nuclear respiratory factor-1 (NRF-1) and mitochondrial transcription factor A (TFAM), which are important factors for mitochondrial biogenesis. We found that the levels of NRF-1 and TFAM mRNA were reduced in mnd2 mouse brain compared with those in wide-type mouse brain (Figure 2d). In addition, we found no alteration in the levels of PGC-1α mRNA in mnd2 mouse brain versus wide-type mouse brain (Figure 2d), suggesting that Omi may regulate PGC-1α abundance at protein levels, and that the regulation depends on Omi protease activity. The above data suggest that Omi regulates mitochondrial biogenesis.

Omi represses GSK3β expressionAs we found that PGC-1α is decreased in mnd2 mice, we wonder whether some factors that regulate PGC-1α are changed in mnd2 mice. We therefore examined regulators that influence PCG-1α abundance or activity, including p38, GSK3β, p70s6k (ribosomal protein S6 kinase), AKT, pAMPK (AMP-activated protein kinase), Sirtuin-1 (SIRT-1) and Cdc 4 (Supplementary Figures S3A and B). Within them, GSK3β, which can phosphorylate PGC-1α and promote its ubiquitin-mediated degradation (Supplementary Figures S4A and B),28, 29 was upregulated significantly in mnd2 mouse brain and spinal cord compared with that in wide-type mice (Figure 3a). We next investigated whether Omi downregulates GSK3β in vitro. In N2a cells, the overexpression of wide-type full-length Omi, but not S276C mutant Omi, reduced the endogenous GSK3β abundance (Figure 3b), as well as enhanced green fluorescent protein (EGFP)-, Flag- and HA-tagged GSK3β (Supplementary Figures S5A, B and C), whereas knockdown of Omi increased the abundances of GSK3β in SH-SY5Y cells (Figure 3c) and N2a cells (Supplementary Figure S1). In addition, we examined the mRNA levels of GSK3β in brains of mnd2 mice and found that there was no difference between mnd2 mice and control mice (Figure 3d).

Omi represses GSK3β expression

As we found that PGC-1α is decreased in mnd2 mice, we wonder whether some factors that regulate PGC-1α are changed in mnd2 mice. We therefore examined regulators that influence PCG-1α abundance or activity, including p38, GSK3β, p70s6k (ribosomal protein S6 kinase), AKT, pAMPK (AMP-activated protein kinase), Sirtuin-1 (SIRT-1) and Cdc 4 (Supplementary Figures S3A and B). Within them, GSK3β, which can phosphorylate PGC-1α and promote its ubiquitin-mediated degradation (Supplementary Figures S4A and B),28, 29 was upregulated significantly in mnd2 mouse brain and spinal cord compared with that in wide-type mice (Figure 3a). We next investigated whether Omi downregulates GSK3β in vitro. In N2a cells, the overexpression of wide-type full-length Omi, but not S276C mutant Omi, reduced the endogenous GSK3β abundance (Figure 3b), as well as enhanced green fluorescent protein (EGFP)-, Flag- and HA-tagged GSK3β (Supplementary Figures S5A, B and C), whereas knockdown of Omi increased the abundances of GSK3β in SH-SY5Y cells (Figure 3c) and N2a cells (Supplementary Figure S1). In addition, we examined the mRNA levels of GSK3β in brains of mnd2 mice and found that there was no difference between mnd2 mice and control mice (Figure 3d).

GSK3β is the substrate of OmiAs the increase of GSK3β abundace in both mnd2 mouse brain and Omi-knockdown cells is associated with Omi protease activity and the mRNA levels of GSK3β is not changed in mnd2 mice, we wonder whether GSK3β is a substrate of Omi. First, we examine the subcellular localization of GSK3β and Omi in SH-SY5Y and N2a cells. Subcellular fractionation assay showed that GSK3β was localized in both cytosolic and mitochondrial fractions, and Omi was localized in mitochondrial fraction (Supplementary Figure S6), suggesting that GSK3β and Omi may interact in mitochondria physiologically. Next, we examined the interactions between Omi and GSK3β. Glutathione S-transferase (GST) pulldown assays showed that GST-GSK3β pulled His-tagged Omi (His-Omi) down, whereas GST alone did not (Figure 4a). Besides, GST-S276C mutant Omi pulled down endogenous GSK3β from brain lysates, whereas GST alone did not (Figure 4b). We further performed immunoprecipitation assays using N2a cells, in which HA-tagged full-length S276C mutant Omi and EGFP-GSK3β were co-transfected, to further confirm the interactions between Omi and GSK3β. As shown in Figure 4c, EGFP-GSK3β was coimmunoprecipitated with Omi when endogenous Omi or HA-Omi was precipitated with Omi-specific antibodies. These data indicate that Omi interacts with GSK3β directly in vitro. Moreover, endogenous Omi was coimmunoprecipitated with endogenous GSK3β after endogenous GSK3β was immunoprecipitated from mnd2 mouse brain lysates with a GSK3β-specific antibody (Figure 4d), further suggesting that Omi interacts with GSK3β under normal physiological conditions. Last, in vitro cleavage assay was performed to identify whether GSK3β is a substrate of Omi directly. The results showed that GST-GSK3β was cleaved by wide-type Omi but not by S276C Omi (Figure 4e), whereas GST alone was not cleaved by Omi (Supplementary Figure S7A). Meanwhile, the Omi substrate β-casein was cleaved by the wild-type Omi, but not S276C Omi, which demonstrated a protease activity of Omi (Supplementary Figure S7B). Taken together, these data indicate that GSK3β is a substrate of Omi.

GSK3β is the substrate of Omi

As the increase of GSK3β abundace in both mnd2 mouse brain and Omi-knockdown cells is associated with Omi protease activity and the mRNA levels of GSK3β is not changed in mnd2 mice, we wonder whether GSK3β is a substrate of Omi. First, we examine the subcellular localization of GSK3β and Omi in SH-SY5Y and N2a cells. Subcellular fractionation assay showed that GSK3β was localized in both cytosolic and mitochondrial fractions, and Omi was localized in mitochondrial fraction (Supplementary Figure S6), suggesting that GSK3β and Omi may interact in mitochondria physiologically. Next, we examined the interactions between Omi and GSK3β. Glutathione S-transferase (GST) pulldown assays showed that GST-GSK3β pulled His-tagged Omi (His-Omi) down, whereas GST alone did not (Figure 4a). Besides, GST-S276C mutant Omi pulled down endogenous GSK3β from brain lysates, whereas GST alone did not (Figure 4b). We further performed immunoprecipitation assays using N2a cells, in which HA-tagged full-length S276C mutant Omi and EGFP-GSK3β were co-transfected, to further confirm the interactions between Omi and GSK3β. As shown in Figure 4c, EGFP-GSK3β was coimmunoprecipitated with Omi when endogenous Omi or HA-Omi was precipitated with Omi-specific antibodies. These data indicate that Omi interacts with GSK3β directly in vitro. Moreover, endogenous Omi was coimmunoprecipitated with endogenous GSK3β after endogenous GSK3β was immunoprecipitated from mnd2 mouse brain lysates with a GSK3β-specific antibody (Figure 4d), further suggesting that Omi interacts with GSK3β under normal physiological conditions. Last, in vitro cleavage assay was performed to identify whether GSK3β is a substrate of Omi directly. The results showed that GST-GSK3β was cleaved by wide-type Omi but not by S276C Omi (Figure 4e), whereas GST alone was not cleaved by Omi (Supplementary Figure S7A). Meanwhile, the Omi substrate β-casein was cleaved by the wild-type Omi, but not S276C Omi, which demonstrated a protease activity of Omi (Supplementary Figure S7B). Taken together, these data indicate that GSK3β is a substrate of Omi.

Regulation of PGC-1α by Omi is mediated by GSK3βWe already found that Omi decreases PGC-1α and cleaves GSK3β. As GSK3β is a kinase involved in PGC-1α degradation, we wonder whether GSK3β mediates the downregulation of PGC-1α by Omi. We therefore examined PGC-1α levels, in the presence or absence of a GSK3β specific inhibitor SB216763. In SH-SY5Y cells in which Omi was knocked down, a significant decrease in PGC-1α abundance was observed; however, the effect of Omi knockdown on PGC-1α abundance was blocked by SB216763 (Figure 5a). Similar results were observed in mnd2 mice after SB216763 treatment (Figure 5b). Besides, the decreased abundance of PGC-1α caused by Omi knockdown can also be blocked by the knockdown of GSK3β (Figure 5c). Moreover, immunoprecipitation assays showed that the interaction of PGC-1α and Cdc4, a component of SCF-Cdc4 E3 ligase that regulates PGC-1α for the ubiquitin-mediated degradation, was significantly increased in dimethylsulfoxide (DMSO)-treated mnd2 mice as compared with wild-type mice; however, the interaction of PGC-1α and Cdc4 was decreased in mnd2 mice after SB216763 treatment (Figure 5d), suggesting that Omi protease activity influences the interaction between PGC-1α and its associated SCF-Cdc4 E3 ligase complex in UPS pathway and that GSK3β affects the interaction, evidenced by the data obtained from the usage of GSK3β inhibitor. Thus, our data suggest that GSK3β mediates Omi-regulated PGC-1α abundance.

Regulation of PGC-1α by Omi is mediated by GSK3β

We already found that Omi decreases PGC-1α and cleaves GSK3β. As GSK3β is a kinase involved in PGC-1α degradation, we wonder whether GSK3β mediates the downregulation of PGC-1α by Omi. We therefore examined PGC-1α levels, in the presence or absence of a GSK3β specific inhibitor SB216763. In SH-SY5Y cells in which Omi was knocked down, a significant decrease in PGC-1α abundance was observed; however, the effect of Omi knockdown on PGC-1α abundance was blocked by SB216763 (Figure 5a). Similar results were observed in mnd2 mice after SB216763 treatment (Figure 5b). Besides, the decreased abundance of PGC-1α caused by Omi knockdown can also be blocked by the knockdown of GSK3β (Figure 5c). Moreover, immunoprecipitation assays showed that the interaction of PGC-1α and Cdc4, a component of SCF-Cdc4 E3 ligase that regulates PGC-1α for the ubiquitin-mediated degradation, was significantly increased in dimethylsulfoxide (DMSO)-treated mnd2 mice as compared with wild-type mice; however, the interaction of PGC-1α and Cdc4 was decreased in mnd2 mice after SB216763 treatment (Figure 5d), suggesting that Omi protease activity influences the interaction between PGC-1α and its associated SCF-Cdc4 E3 ligase complex in UPS pathway and that GSK3β affects the interaction, evidenced by the data obtained from the usage of GSK3β inhibitor. Thus, our data suggest that GSK3β mediates Omi-regulated PGC-1α abundance.

Inhibition of GSK3β and overexpression of PGC-1α restore mitochondrial biogenesis influenced by OmiAs GSK3β is effective on PGC-1α abundance at the downstream of Omi, we wonder whether the inhibition of GSK3β is able to restore mitochondrial biogenesis in vivo and in vitro. We therefore treated mnd2 mice with GSK3β inhibitor SB216763 and detected the state of mitochondrial biogenesis. We found that the mRNA levels of ATP5B, ANT1, COX II, COX IV, CytC, NRF-1α and TFAM were significantly increased in mnd2 mice (Figure 6a). Consistence with RT-qPCR analyses, the protein levels of COX IV and Cyt C were also restored significantly (Figure 6b). Further, the inhibition of GSK3β significantly increased mtDNA in mnd2 mice (Figure 6c). Similar results were obtained in PGC-1α-overexpressing N2a cells, in which Omi was knocked down (Figure 6d). Moreover, transmission electron microscopic analyses showed that the mitochondria number was increased significantly in striatum in mnd2 mice after a treatment with SB216763 as compared with that in mnd2 mice without SB216763 treatment (Figure 6e). Quantitative morphometry of transmission electron micrographs confirmed that the mitochondrial density was significantly recovered in mnd2 mice treatment with SB216763 than that in mnd2 mice without SB216763 treatment (Figure 6f). Thus, our data suggest that inhibition of GSK3β and overexpression of PGC-1α significantly restore mitochondrial biogenesis that is influenced by Omi deficiency.

Inhibition of GSK3β and overexpression of PGC-1α restore mitochondrial biogenesis influenced by Omi

As GSK3β is effective on PGC-1α abundance at the downstream of Omi, we wonder whether the inhibition of GSK3β is able to restore mitochondrial biogenesis in vivo and in vitro. We therefore treated mnd2 mice with GSK3β inhibitor SB216763 and detected the state of mitochondrial biogenesis. We found that the mRNA levels of ATP5B, ANT1, COX II, COX IV, CytC, NRF-1α and TFAM were significantly increased in mnd2 mice (Figure 6a). Consistence with RT-qPCR analyses, the protein levels of COX IV and Cyt C were also restored significantly (Figure 6b). Further, the inhibition of GSK3β significantly increased mtDNA in mnd2 mice (Figure 6c). Similar results were obtained in PGC-1α-overexpressing N2a cells, in which Omi was knocked down (Figure 6d). Moreover, transmission electron microscopic analyses showed that the mitochondria number was increased significantly in striatum in mnd2 mice after a treatment with SB216763 as compared with that in mnd2 mice without SB216763 treatment (Figure 6e). Quantitative morphometry of transmission electron micrographs confirmed that the mitochondrial density was significantly recovered in mnd2 mice treatment with SB216763 than that in mnd2 mice without SB216763 treatment (Figure 6f). Thus, our data suggest that inhibition of GSK3β and overexpression of PGC-1α significantly restore mitochondrial biogenesis that is influenced by Omi deficiency.

Inhibition of GSK3β improves movement ability of mnd2 miceAs inhibition of GSK3β is able to restore mitochondrial biogenesis influenced by Omi in vivo and in vitro, we wonder whether there is a functional improvement in mnd2 mice after SB216763 treatment. We therefore performed the rotarod test, which is widely used to evaluate the overall motor deficits in rodent models of PD and others.30 We found that the performance of mnd2 mice was severely deteriorated compared with wide-type mice; however, the performance was significantly improved when the mnd2 mice was administrated with SB216763 (Figure 7).

Inhibition of GSK3β improves movement ability of mnd2 mice

As inhibition of GSK3β is able to restore mitochondrial biogenesis influenced by Omi in vivo and in vitro, we wonder whether there is a functional improvement in mnd2 mice after SB216763 treatment. We therefore performed the rotarod test, which is widely used to evaluate the overall motor deficits in rodent models of PD and others.30 We found that the performance of mnd2 mice was severely deteriorated compared with wide-type mice; however, the performance was significantly improved when the mnd2 mice was administrated with SB216763 (Figure 7).

DiscussionIn this study, we demonstrated that Omi regulates mitochondrial biogenesis, which may involve in Omi protease-deficient-induced neurodegeneration. Omi protease-deficient mnd2 mice present decreases of mtDNA and mitochondrial density in brain, with a reduced amount of the essential components COX IV and Cyt C of the mitochondrial respiratory chain, as well as other components. The mitochondrial biogenesis factor PGC-1α is downregulated and the kinase GSK3β that promotes PGC-1α degradation is increased in mnd2 mice. Beisdes, GSK3β inhibitor SB216763 significantly improved the movement ability of mnd2 mice. Thus, our findings suggest that Omi protease activity is involved in the regulation of mitochondrial biogenesis and the deficiency of mitochondria biogenesis may be associated with Omi protease-deficient-induced neurodegeneration.Mitochondrial biogenesis is a complex process that requires the synthesis, import and incorporation of proteins, lipids and mtDNA to the existing mitochondrial reticulum, and it involves the coordinated expression of mitochondrial and nuclear genes.31 A growing body of evidence supports the hypothesis that the abnormality of mitochondrial biogenesis is tightly associated with neurodegenerative diseases, including PD.2, 32, 33 mtDNA depletion may occur in neurons in sporadic PD brains, which reflects an impairment of mitochondrial biogenesis.34 The Tfam-knockout mice show reduced mtDNA level and respiratory chain deficiency in the midbrain DA neurons, with adult onset of slowly progressive typical parkinsonian features, also supporting that the disturbance of mtDNA integrity or mitochondrial function may underlie neurodegenerative disorders.35 Normally, mitochondrial biogenesis and function are dynamically regulated to adapt to the energetic and metabolic demands in response to physiological signals.32 In many neurodegenerative diseases, brain mitochondrial function is impaired, and mitochondrial biogenesis is increased to compensate for functional declines. Neurodegeneration will occur in the case of insufficient mitochondrial biogenesis to balance the functional declines.In our study, we identified that Omi regulates mitochondrial biogenesis. We found that: first, the expression of the components of the mitochondrial respiration chain and oxidant metabolism is downregulated from transcriptional level; second, the amount of mtDNA is also decreased; third, the mitochondrial density is significantly reduced in mnd2 mouse brain; last, the abundance of PGC-1α is downregulated in the brain and spinal cord of mnd2 mice.PGC-1α, as the central regulator of mitochondrial biogenesis and function, integrates diverse physiological signals that impact mitochondrial biogenesis and coordinates multiple transcription factors, such as nuclear receptors such as NRF-1, NRF-2 and the orphan nuclear hormone receptors peroxisome proliferator–activated receptor α, PPARδ, estrogen-related receptors and thyroid hormone receptor, to induce broad sets of mitochondrial genes.26, 36 PGC-1α is highly expressed in tissues with high-energy demands and abundant mitochondria, such as brown adipose tissue, heart muscle, slow-twitch skeletal muscle and brain.36, 37 The association of PGC-1α with neurodegenerative disorders has been addressed in many studies: on one hand, the increased risk of the development of both PD and HD is associated with single nucleotide polymorphisms in the gene encoding PGC-1;37, 38 on the other hand, the abundance of PGC-1α was decreased in AD, HD and PD patients.37, 39, 40 Recently, several studies have suggested that an impairment of PGC-1α function is involved in PD pathogenesis. For example, a gene set enrichment analysis shows a coordinate downregulation of 425 PGC-1α regulated genes in PD patients.41 Furthermore, an activation of PGC-1α can protect DA neuron from the damage of mutant α-synuclein or the pesticide rotenone or MPTP in primary neuron models.42 In addition, PD-related gene product parkin regulates the levels of PGC-1α and enables PGC-1α-dependent gene expression.40Interestingly, PGC-1α-knockout mice display impaired mitochondrial function such as diminished mitochondrial number and respiratory capacity in muscle and brown fat, with movement disorder, including exaggerated startle responses, dystonic posturing, stimulus-induced myoclonus and frequent limb clasping, accompanied by neurodegeneration, particularly in the striatum, which show some similarities to the mnd2 mice.18, 19, 43, 44 As PGC-1α abundance is decreased in mnd2 mice, it is possible that the Omi protease-deficient-induced neurodegeneration may be associated with the decreased abundance of PGC-1α. Besides, it was reported that PGC-1α is a potent suppressor of reactive oxygen species (ROS) and induces the production of ROS-scavenging enzymes. Thus, the decreased expression of PGC-1α caused by Omi deficiency may also result in the insufficiency to defend ROS and neurodegeneration.42 On the other hand, as a protease, Omi involves many pathways and regulates many critical cellular processes, such as autophagy and neuroinflammation;45, 46 however, most functions regulated by PGC-1α are not associated with cell death directly. It is possible that besides PGC-1α, other factors and pathways may be involved in the pathogenesis of mnd2 mice, so that the phenotype by loss of Omi activity is much dramatic and the life span of PGC-1α-knockout mice is longer than mnd2 mice.GSK3β is an important serine/threonine kinase to regulate PGC-1α degradation.28, 29 GSK3β reduces PGC-1α levels by phosphorylating PGC-1α and subsequently stimulating PGC-1α degradation by the ubiquitin–proteasomal system. An inhibition of GSK3β promotes mitochondrial biogenesis during ischemia cerebral injury.47 In our study, we found that the interaction of PGC-1α and the component of E3 ubiquitin ligase Cdc4 was significantly increased in mnd2 mice and the interaction of PGC-1α and Cdc4 was decreased in mnd2 mice after SB216763 treatment. We also demonstrated that the regulation of PGC-1α by Omi is mediated by GSK3β. Omi cleaves GSK3β to reduce PGC-1α degradation and maintain mitochondrial biogenesis. Increases of GSK3β and decreases of PGC-1α abundance are presented in Omi knockdown SH-SY5Y cells and mnd2 mice, whereas a restoration of PGC-1α abundance and mitochondrial biogenesis occurs after a treatment with the GSK3β inhibitor SB216763.The function of GSK3β in PD has been documented in several studies. The polymorphisms in GSK3β, which affect its transcription and splicing, are associated with disease risk in PD.48 Besides, GSK3β activity is elevated in the striatum in postmortem tissue from PD patients and mouse models of PD,49, 50 and GSK3β levels are increased in peripheral blood lymphocytes in PD patients.50, 51 In addition, the abnormal activation of GSK3β is associated with several psychiatric disorders and AD, and the overexpression of GSK3β results in neurodegeneration in mice.52, 53 Thus, abnormal increases of GSK3β activity or abundance are able to induce PGC-1α degradation, subsequently leading to neurodegeneration. These studies suggest that it is essential for neurons to ensure GSK3β activity in a reasonable level for maintenance of cellular homeostasis. GSK3β has been reported to be located in cytosol, nucleus and mitochondria (Supplementary Figure S4),54, 55 and GSK3β activity is mainly regulated through phosphorylation by kinases, such as Akt, PKC and P70S6K, or interacted with other proteins, such as and DISC 1.56 In our study, we identified that the kinase GSK3β is a novel substrate of Omi, and Omi cleaves GSK3β to repress its levels. The abundance of GSK3β and the phosphorylation on tyrosine (Tyr) residues (Tyr 216) are upregulated in the brain and spinal cord of mnd2 mice, as well as in Omi-knockdown SH-SY5Y and N2a cells. Overexpression of wide-type Omi, but not of S276C Omi, substantially represses the total level of GSK3β. In addition, in vitro cleavage assays show that Omi cleaves GSK3β directly. Furthermore, GSK3β inhibitor SB216763 significantly improves the movement ability of mnd2 mice indicated by rotarod test. Thus, as a substrate of Omi, GSK3β that is the abnormally high expressed due to Omi protease deficiency may be associated with the pathogenesis in mnd2 mice.In summary, our study identified that GSK3β is a novel substrate of Omi. Loss of Omi protease activity results in an abnormal increase of GSK3β, leading to the degradation of PGC-1α, which causes an impairment of mitochondrial biogenesis and induces neurodegeneration.

In this study, we demonstrated that Omi regulates mitochondrial biogenesis, which may involve in Omi protease-deficient-induced neurodegeneration. Omi protease-deficient mnd2 mice present decreases of mtDNA and mitochondrial density in brain, with a reduced amount of the essential components COX IV and Cyt C of the mitochondrial respiratory chain, as well as other components. The mitochondrial biogenesis factor PGC-1α is downregulated and the kinase GSK3β that promotes PGC-1α degradation is increased in mnd2 mice. Beisdes, GSK3β inhibitor SB216763 significantly improved the movement ability of mnd2 mice. Thus, our findings suggest that Omi protease activity is involved in the regulation of mitochondrial biogenesis and the deficiency of mitochondria biogenesis may be associated with Omi protease-deficient-induced neurodegeneration.

Mitochondrial biogenesis is a complex process that requires the synthesis, import and incorporation of proteins, lipids and mtDNA to the existing mitochondrial reticulum, and it involves the coordinated expression of mitochondrial and nuclear genes.31 A growing body of evidence supports the hypothesis that the abnormality of mitochondrial biogenesis is tightly associated with neurodegenerative diseases, including PD.2, 32, 33 mtDNA depletion may occur in neurons in sporadic PD brains, which reflects an impairment of mitochondrial biogenesis.34 The Tfam-knockout mice show reduced mtDNA level and respiratory chain deficiency in the midbrain DA neurons, with adult onset of slowly progressive typical parkinsonian features, also supporting that the disturbance of mtDNA integrity or mitochondrial function may underlie neurodegenerative disorders.35 Normally, mitochondrial biogenesis and function are dynamically regulated to adapt to the energetic and metabolic demands in response to physiological signals.32 In many neurodegenerative diseases, brain mitochondrial function is impaired, and mitochondrial biogenesis is increased to compensate for functional declines. Neurodegeneration will occur in the case of insufficient mitochondrial biogenesis to balance the functional declines.

In our study, we identified that Omi regulates mitochondrial biogenesis. We found that: first, the expression of the components of the mitochondrial respiration chain and oxidant metabolism is downregulated from transcriptional level; second, the amount of mtDNA is also decreased; third, the mitochondrial density is significantly reduced in mnd2 mouse brain; last, the abundance of PGC-1α is downregulated in the brain and spinal cord of mnd2 mice.

PGC-1α, as the central regulator of mitochondrial biogenesis and function, integrates diverse physiological signals that impact mitochondrial biogenesis and coordinates multiple transcription factors, such as nuclear receptors such as NRF-1, NRF-2 and the orphan nuclear hormone receptors peroxisome proliferator–activated receptor α, PPARδ, estrogen-related receptors and thyroid hormone receptor, to induce broad sets of mitochondrial genes.26, 36 PGC-1α is highly expressed in tissues with high-energy demands and abundant mitochondria, such as brown adipose tissue, heart muscle, slow-twitch skeletal muscle and brain.36, 37 The association of PGC-1α with neurodegenerative disorders has been addressed in many studies: on one hand, the increased risk of the development of both PD and HD is associated with single nucleotide polymorphisms in the gene encoding PGC-1;37, 38 on the other hand, the abundance of PGC-1α was decreased in AD, HD and PD patients.37, 39, 40 Recently, several studies have suggested that an impairment of PGC-1α function is involved in PD pathogenesis. For example, a gene set enrichment analysis shows a coordinate downregulation of 425 PGC-1α regulated genes in PD patients.41 Furthermore, an activation of PGC-1α can protect DA neuron from the damage of mutant α-synuclein or the pesticide rotenone or MPTP in primary neuron models.42 In addition, PD-related gene product parkin regulates the levels of PGC-1α and enables PGC-1α-dependent gene expression.40

Interestingly, PGC-1α-knockout mice display impaired mitochondrial function such as diminished mitochondrial number and respiratory capacity in muscle and brown fat, with movement disorder, including exaggerated startle responses, dystonic posturing, stimulus-induced myoclonus and frequent limb clasping, accompanied by neurodegeneration, particularly in the striatum, which show some similarities to the mnd2 mice.18, 19, 43, 44 As PGC-1α abundance is decreased in mnd2 mice, it is possible that the Omi protease-deficient-induced neurodegeneration may be associated with the decreased abundance of PGC-1α. Besides, it was reported that PGC-1α is a potent suppressor of reactive oxygen species (ROS) and induces the production of ROS-scavenging enzymes. Thus, the decreased expression of PGC-1α caused by Omi deficiency may also result in the insufficiency to defend ROS and neurodegeneration.42 On the other hand, as a protease, Omi involves many pathways and regulates many critical cellular processes, such as autophagy and neuroinflammation;45, 46 however, most functions regulated by PGC-1α are not associated with cell death directly. It is possible that besides PGC-1α, other factors and pathways may be involved in the pathogenesis of mnd2 mice, so that the phenotype by loss of Omi activity is much dramatic and the life span of PGC-1α-knockout mice is longer than mnd2 mice.

GSK3β is an important serine/threonine kinase to regulate PGC-1α degradation.28, 29 GSK3β reduces PGC-1α levels by phosphorylating PGC-1α and subsequently stimulating PGC-1α degradation by the ubiquitin–proteasomal system. An inhibition of GSK3β promotes mitochondrial biogenesis during ischemia cerebral injury.47 In our study, we found that the interaction of PGC-1α and the component of E3 ubiquitin ligase Cdc4 was significantly increased in mnd2 mice and the interaction of PGC-1α and Cdc4 was decreased in mnd2 mice after SB216763 treatment. We also demonstrated that the regulation of PGC-1α by Omi is mediated by GSK3β. Omi cleaves GSK3β to reduce PGC-1α degradation and maintain mitochondrial biogenesis. Increases of GSK3β and decreases of PGC-1α abundance are presented in Omi knockdown SH-SY5Y cells and mnd2 mice, whereas a restoration of PGC-1α abundance and mitochondrial biogenesis occurs after a treatment with the GSK3β inhibitor SB216763.

The function of GSK3β in PD has been documented in several studies. The polymorphisms in GSK3β, which affect its transcription and splicing, are associated with disease risk in PD.48 Besides, GSK3β activity is elevated in the striatum in postmortem tissue from PD patients and mouse models of PD,49, 50 and GSK3β levels are increased in peripheral blood lymphocytes in PD patients.50, 51 In addition, the abnormal activation of GSK3β is associated with several psychiatric disorders and AD, and the overexpression of GSK3β results in neurodegeneration in mice.52, 53 Thus, abnormal increases of GSK3β activity or abundance are able to induce PGC-1α degradation, subsequently leading to neurodegeneration. These studies suggest that it is essential for neurons to ensure GSK3β activity in a reasonable level for maintenance of cellular homeostasis. GSK3β has been reported to be located in cytosol, nucleus and mitochondria (Supplementary Figure S4),54, 55 and GSK3β activity is mainly regulated through phosphorylation by kinases, such as Akt, PKC and P70S6K, or interacted with other proteins, such as and DISC 1.56 In our study, we identified that the kinase GSK3β is a novel substrate of Omi, and Omi cleaves GSK3β to repress its levels. The abundance of GSK3β and the phosphorylation on tyrosine (Tyr) residues (Tyr 216) are upregulated in the brain and spinal cord of mnd2 mice, as well as in Omi-knockdown SH-SY5Y and N2a cells. Overexpression of wide-type Omi, but not of S276C Omi, substantially represses the total level of GSK3β. In addition, in vitro cleavage assays show that Omi cleaves GSK3β directly. Furthermore, GSK3β inhibitor SB216763 significantly improves the movement ability of mnd2 mice indicated by rotarod test. Thus, as a substrate of Omi, GSK3β that is the abnormally high expressed due to Omi protease deficiency may be associated with the pathogenesis in mnd2 mice.

In summary, our study identified that GSK3β is a novel substrate of Omi. Loss of Omi protease activity results in an abnormal increase of GSK3β, leading to the degradation of PGC-1α, which causes an impairment of mitochondrial biogenesis and induces neurodegeneration.

Materials and MethodsAnimal experimentsThe mnd2 mice and age-matched wild-type mice were obtained from crossing paired heterozygotes (HtrA2mnd2, C57BL/6J) that were purchased from The Jackson Laboratory (Bar Harbor, ME, USA). Identifications of the genotypes of the offspring mice were performed according to the protocol provided by JAX mice (The Jackson Laboratory). Mice were maintained in a specific pathogen-free environment in compliance with national regulations. All animal experiments were approved by the animal welfare advisory committee of the University of Science and Technology of China. The mice were given an intraperitoneal injection of vehicle or SB216763 (2 mg/kg), suspended in PBS every 2 days for 15 or 20 days, from the time they were 10 days old.Cell culture, transfections and drug treatmentsSH-SY5Y cells, mouse neuroblastoma (N2a) cells were cultured in Dulbecco's modified Eagle's medium (DMEM; Gibco, Grand Island, NY, USA) F12 or DMEM containing 10% fetal bovine serum (Gibco) with penicillin (100 mg/ml) and streptomycin (100 mg/ml). Transfections were performed using Lipofectamine 2000 reagent (Invitrogen, La Jolla, CA, USA) according to the manufacturer's instructions. The cells were lysated and subjected to western blotting analysis 48 h after transfection. For the SB216763 treatment (Sigma-Aldrich, St. Louis, MO, USA), cells were treated with 5 mM SB216763, and after 24 h incubation, cell lysates were subjected to western blotting analysis.PlasmidsThe myc-CMV-PGC-1α was kindly provided by Dr Chang Liu (Nanjing Normal University, China). The Omi-related plasmids were previously described.45 Full-length human GSK3β complementary DNA (cDNA) was amplified by PCR from an adult human brain cDNA library with the primers 5′-AACTGCAGACCATGTCAGGGCGGCCCAGAA-3′ and 5′-TGCTCTAGAGGTGGAGTTGGAAGCTGATGC-3′ and inserted in-frame into the pGEX-5x-1 vector at the Bgl II/ECOR I sites or into the pEGFP-C2 vector at the BamH I/ECOR I sites. The C-terminal HA-tagged and Flag-tagged GSK3β was created by subcloning the PCR product amplified with primers 5′-AACTGCAGACCATGTCAGGGCGGCCCAGAA-3′ and 5′-TGCTCTAGAGGTGGAGTTGGAAGCTGATGC-3′ or 5′-TAAAGATCTGATGTCAGGGCGGCCCAGAAC-3′and 5′-TAAGTCGACGGTGGAGTTGGAAGCTGATGC-3′ into pKH3-HA vector or p3x-Flag-CMV vector at sites Pst I/Xba I or Bgl II/Sal I.RNA interferenceThe oligonucleotides were transfected with Oligofectamine (Invitrogen) according to the manufacturer's instructions. The oligonucleotide sequences targeting to Omi (si-Omi) and the negative control small interfering RNA (si-NC) were previously described,45 and the oligonucleotide sequences targeting to GSK3β are shown as follows: sense 1, 5′-CTGCATTTATCGTTAACCTAATTdTdT-3′ and antisense 1, 5′-AATTAGGTTAACGATAAATGCAGdTdT-3′ sense 2, 5′-CAGGGCAGTGATTCTGTTCAATTdTdT-3′ and antisense 2, 5′-AATTGAACAGAATCACACTGCCCTGdTdT-3′.Western blotting analysis and antibodiesTissue homogenate and cell extracts were lysed in the 1 × RIPA lysis buffer (25 mM tris-HCl, pH 7.6, 150 mM NaCl, 1% NP-40 and 1% sodium deoxycholate) in the presence of a protease inhibitor cocktail (Roche, Indianapolis, IN, USA). Nearly 20 μg of cell lysate was resolved by SDS–PAGE and transferred onto a PVDF membrane (Millipore, Bedford, MA, USA). Western blotting analysis was carried out with the following primary antibodies: anti-α-Tubulin (CP06, Calbiochem, San Diego, CA, USA), anti-AKT1 (sc-1618, Santa Cruz Biotechnology, Santa Cruz, CA, USA), anti-AMPKα (23A3, Cell Signaling, Beverly, MA, USA), anti-Cdc4 (ab171961, Abcam, Cambridge, MA, USA), anti-COX IV (S0553, Epitomics, Burlingame, CA, USA), anti-CytC (3895-1, Epitomics), anti-Flag (F3165, Sigma-Aldrich), anti-glyceraldehyde-3-phosphate dehydrogenase (GAPDH) (MAB374, CHEMICON, Temecula, CA, USA), anti-GFP (sc-9996, Santa Cruz Biotechnology), anti-GSK-3β (NBP1-47470, Novus, Littleton, CO, USA or sc-7291, Santa Cruz Biotechnology), anti-GST (sc-138, Santa Cruz Biotechnology), anti-p38 (sc-7972, Santa Cruz Biotechnology), anti-HA (sc-7392, Santa Cruz Biotechnology), anti-His (M30111, Abmart, Shanghai, China), anti-Omi (AF1458, R&D Systems, Minneapolis, MN, USA or prepared by our laboratory), anti-p70s6k (#3204-1, Epitomics), anti-PGC-1α (ST1202, Calbiochem, or sc-13067, Santa Cruz Biotechnology), anti-Phospho-AMPKα (Thr 172; 40H9, Cell Signaling), anti-Phospho-AKT1 (#2118-1, Epitomics), anti-Phospho-GSK3α/β (pY279/pY261) (#2309-1-1, Epitomics), anti-Phospho-GSK3β (Ser9; AG753, Biyuntian, Shanghai, China), anti-Phospho-p38 MAPK (pT180/pY182; 1229-1, Epitomics), anti-Phospho-p70s6k (#1494-1, Epitomics), anti-SIRT-1 (13161-AP, Proteintech, Chicago, IL, USA), anti-TOM70 (14528-1-AP, Proteintech), anti-Ub (sc-8017, Santa Cruz Biotechnology). The anti-mouse and anti-rabbit secondary antibodies coupled to horseradish peroxidase were from Amersham Pharmacia Biotech (Peapack, NJ, USA). The proteins were viewed with an ECL detection kit (Amersham Pharmacia Biotech, Arlington Heights, IL, USA).GST pulldown assaysFor in vitro pulldown assay, an aliquot containing 20 mg/μg of GST or GST- GSK3β that was expressed in E. coli strain JM109 was incubated with 30 ml of Glutathione Sepharose 4B (Amersham Biosciences, Tokyo, Japan) for 30 min at 4 °C. GST or GST-GSK-3β bound to Sepharose was incubated with 50 mg of His-Omi protein from the supernatants of E. coli crude extracts containing His-tagged proteins in 0.25 ml of HNTG buffer (20 mM Hepes-KOH, pH 7.5, 0.1% Triton X-100, 100 mM NaCl and 10% glycerol) for 1 h at 4 °C. For semi in vivo pulldown assay, an aliquot containing 20 μg of GST or GST-S276C mutant Omi was incubated with 30 ml of Glutathione Sepharose 4B for 30 min at 4 °C. GST or GST-Omi bound to Sepharose was incubated with mouse brain tissue homogenates in 0.25 ml of HNTG buffer for 1 h at 4 °C. After incubation, the beads were washed six times with 1 ml of HNTG buffer each time to remove unbound protein. Bound protein was eluted by boiling in SDS–PAGE sample buffer, and then subjected to western blotting analysis.ImmunoprecipitationHomogenates from mnd2 mouse brains and cells that were transfected with the indicated plasmids were lysed in TSPI buffer containing 50 mM Tris-HCl (pH 7.5), 150 mM sodium chloride, 1 mM EDTA and 1% NP-40 supplemented with a complete mini protease inhibitor cocktail (Roche).The lysate was incubated with anti-Omi antibodies or normal rabbit immunoglobulin G overnight at 4 °C. After incubation, protein G-Sepharose (Roche) was used for precipitations. The beads were washed with TSPI buffer six times and were eluted with SDS sample buffer for western blotting analysis.In vitro proteolytic cleavage assayGST, GST-GSK3β or β-casein (Sigma-Aldrich) was incubated with 3 μg of 6 × His-tagged mature Omi in protease cleavage buffer (50 mM tris-HCl, pH 7.6, 1 mM dithiothreitol) for 60 min at 37 °C. The reactions were stopped by the addition of 2 × SDS loading buffer and boiled for 10 min. The samples were subjected to western blotting analyses with an anti-GST or anti-GSK3β antibody.RNA isolation and real-time PCRTotal RNA was extracted from brain using Trizol (Invitrogen) according to the manufacturer's instructions. cDNA was synthesized using a TransScript First-Strand cDNA Synthesis Kit (Takara, Shiga, Japan). Real-time PCR was performed with SYBR Green Real-Time PCR Master Mix (Takara) reagents using a BioRad MiniOpticon System. Relative gene expressions were calculated by the 2−ΔΔCt method. Primers for RT-qPCR used here are shown as follows: PGC-1α forward 5′-AGCCGTGACCACTGACAACGAG-3′, reverse 5′-GCTGCATGGTTCTGAGTGCTAAG-3′ PGC-1β forward 5′-GGCAGGTTCAACCCCGA-3′, reverse 5′-CTTGCTAACATCACAGAGGATATCTTG-3′ GSK3β forward 5′-TAAGGAAGGAAAAGGTGAtTCGA-3′, reverse 5′-CCAAAAGCTGAAGGCTGCTG-3′ NRF-1 forward 5′-GTTGGTACAGGGGCAACAGT-3′, reverse 5′-TCGTCTGGATGGTCATTTCA-3′ TFAM forward 5′-CTGATGGGTATGGAGAAGGAGG-3′, reverse 5′-CCAACTTCAGCCATCTGCTCTTC-3′ UCP2 forward 5′-ACTTTCCCTCTGGATACCGC-3′, reverse 5′-ACGGAGGCAAAGCTCATCTG-3′ ANT1 forward 5′-GTCTCTGTCCAGGGCATCAT-3′, reverse 5′-ACGACGAACAGTGTCAAACG-3′ ATP5B forward 5′-GAGGGATTACCACCCATCCT-3′, reverse 5′-CATGATTCTGCCCAAGGTCT-3′ CYTC forward 5′-CCAAATCTCCACGGTCTGTTC-3′, reverse 5′-ATCAGGGTATCCTCTCCCCAG-3′ COX IV forward 5′-CGAGAGCTTCGCCGAGATGAAC-3′, reverse 5;-GCAGCTTCCAGCATGCCGAGG-3′ COX II forward 5′-ACGAAATCAACAACCCCGTA-3′, reverse 5′-GGCAGAACGACTCGGTTATC-3′ β-actin forward 5′-TGTCCACCTTCCAGCAGATGT-3′, reverse 5′-AGCTCAGTAACAGTCCGCCTAGA-3′.Analysis of mtDNATotal genomic DNA was extracted using a GenElute mammalian genomic DNA kit (Sigma-Aldrich) according to the manufacturer's instructions. mNADH1, mCYTB, mNADH4, mATP6 and mCOX II were used as mtDNA markers, and nuclear intron of β-globin was used as a nDNA marker. mTND1, mCYTB, mMTND4, mATP6 and mCOX II primers were designed from regions of mtDNA that are not found in nuclear-encoded mitochondrial pseudogenes, with the primers which were described elsewhere as following:57, 58, 59 mNADH1 forward 5′-TCTGCCAGCCTGACCCATAG-3′, reverse 5′-CCGGCTGCGTATTCTACGTT-3′ mCYTB forward 5′-TATTCCTTCATGTCGGACGA-3′, reverse 5′-AAATGCTGTGGCTATGACTG-3′ mNADH4 forward 5′-ATTATTATTACCCGATGAGGGAACC-3′, reverse 5′-ATTAAGATGAGGGCAATTAGCAGT-3′ mATP6 forward 5′-AATTACAGGCTTCCGACACAAAC-3′, reverse 5′-TGGAATTAGTGAAATTGGAGTTCCT-3′ mCOX2 forward 5′-GCCGACTAAATCAAGCAACA-3′, reverse 5′- CAATGGGCATAAAGCTATGG-3′ β-globin forward 5′-GAAGCGATTCTAGGGAGCAG-3′, reverse 5′-GGAGCAGCGATTCTGAGTAGA-3′. The quantification of abovementioned product was analyzed after being normalized to β-globin using one-way ANOVA.Subcellular Fractionation AssayThe cytosolic and mitochondrial fractions from SH-SY5Y and N2a cells were isolated according to the instructions of the Mitochondria Isolation kit for cultured Cells (Beyotime, China). The lysates of fractions were subjected to western blotting analyses. GAPDH was used as the cytosolic marker and TOM70 was used as the mitochondrial marker.Rotarod testMice were detected for their ability to run on a 3-cm diameter rotarod at speed of 10 revolutions per minute (r.p.m.) as previously described with modification.30 To reach a stable performance, mice were trained at speed of 5–10 r.p.m. for 2 days before the first-time test. The tests were performed using mice that were 16, 20, 25 and 30 days old, with a treatment using SB216763 or DMSO. The time that the mice spent on the rotarod before falling was recorded for each trial, and the trial was repeated thrice. To reduce stress and fatigue, there was at least 10 min of rest between trials, and a cut-off time point was set at 600 s.Electron microscopyFor electron microscopic analysis of the brain, the tissue was prepared as previously described.18
Mnd2 mice that were 25 days old and age-matched wild-type mice were transcardially perfused with 0.1 M phosphate buffer for 6–8 min, followed by 2% formaldehyde/4% glutaraldehyde fixative in 0.1 M phosphate buffer for 6–8 min. Dissected striatum were further fixed with the same fixative for 2–4 h at 4 °C and postfixed with 2% OsO4 in buffer and embedded in Araldite. Ultrathin sections of striatum were cut onto formvar-coated slot grids stained with uranyl acetate and lead citrate and viewed with a JEM-1230 electron microscope. To calculate mitochondrial density, six different fields at the magnification of 5000 for each animal were quantified as described previously.60Statistical analysisDensitometric analysis of western blots from three independent experiments was implemented with Adobe Photoshop 7.0 and the data were analyzed with Origin 6.0 (OriginLab, Northampton, MA, USA). Quantitative data are presented as means±S.E.M. Statistical significance was assessed via one-way ANOVA and the criterion of significance was set at P<0.05.

Materials and Methods

Animal experimentsThe mnd2 mice and age-matched wild-type mice were obtained from crossing paired heterozygotes (HtrA2mnd2, C57BL/6J) that were purchased from The Jackson Laboratory (Bar Harbor, ME, USA). Identifications of the genotypes of the offspring mice were performed according to the protocol provided by JAX mice (The Jackson Laboratory). Mice were maintained in a specific pathogen-free environment in compliance with national regulations. All animal experiments were approved by the animal welfare advisory committee of the University of Science and Technology of China. The mice were given an intraperitoneal injection of vehicle or SB216763 (2 mg/kg), suspended in PBS every 2 days for 15 or 20 days, from the time they were 10 days old.

Animal experiments

The mnd2 mice and age-matched wild-type mice were obtained from crossing paired heterozygotes (HtrA2mnd2, C57BL/6J) that were purchased from The Jackson Laboratory (Bar Harbor, ME, USA). Identifications of the genotypes of the offspring mice were performed according to the protocol provided by JAX mice (The Jackson Laboratory). Mice were maintained in a specific pathogen-free environment in compliance with national regulations. All animal experiments were approved by the animal welfare advisory committee of the University of Science and Technology of China. The mice were given an intraperitoneal injection of vehicle or SB216763 (2 mg/kg), suspended in PBS every 2 days for 15 or 20 days, from the time they were 10 days old.

Cell culture, transfections and drug treatmentsSH-SY5Y cells, mouse neuroblastoma (N2a) cells were cultured in Dulbecco's modified Eagle's medium (DMEM; Gibco, Grand Island, NY, USA) F12 or DMEM containing 10% fetal bovine serum (Gibco) with penicillin (100 mg/ml) and streptomycin (100 mg/ml). Transfections were performed using Lipofectamine 2000 reagent (Invitrogen, La Jolla, CA, USA) according to the manufacturer's instructions. The cells were lysated and subjected to western blotting analysis 48 h after transfection. For the SB216763 treatment (Sigma-Aldrich, St. Louis, MO, USA), cells were treated with 5 mM SB216763, and after 24 h incubation, cell lysates were subjected to western blotting analysis.

Cell culture, transfections and drug treatments

SH-SY5Y cells, mouse neuroblastoma (N2a) cells were cultured in Dulbecco's modified Eagle's medium (DMEM; Gibco, Grand Island, NY, USA) F12 or DMEM containing 10% fetal bovine serum (Gibco) with penicillin (100 mg/ml) and streptomycin (100 mg/ml). Transfections were performed using Lipofectamine 2000 reagent (Invitrogen, La Jolla, CA, USA) according to the manufacturer's instructions. The cells were lysated and subjected to western blotting analysis 48 h after transfection. For the SB216763 treatment (Sigma-Aldrich, St. Louis, MO, USA), cells were treated with 5 mM SB216763, and after 24 h incubation, cell lysates were subjected to western blotting analysis.

PlasmidsThe myc-CMV-PGC-1α was kindly provided by Dr Chang Liu (Nanjing Normal University, China). The Omi-related plasmids were previously described.45 Full-length human GSK3β complementary DNA (cDNA) was amplified by PCR from an adult human brain cDNA library with the primers 5′-AACTGCAGACCATGTCAGGGCGGCCCAGAA-3′ and 5′-TGCTCTAGAGGTGGAGTTGGAAGCTGATGC-3′ and inserted in-frame into the pGEX-5x-1 vector at the Bgl II/ECOR I sites or into the pEGFP-C2 vector at the BamH I/ECOR I sites. The C-terminal HA-tagged and Flag-tagged GSK3β was created by subcloning the PCR product amplified with primers 5′-AACTGCAGACCATGTCAGGGCGGCCCAGAA-3′ and 5′-TGCTCTAGAGGTGGAGTTGGAAGCTGATGC-3′ or 5′-TAAAGATCTGATGTCAGGGCGGCCCAGAAC-3′and 5′-TAAGTCGACGGTGGAGTTGGAAGCTGATGC-3′ into pKH3-HA vector or p3x-Flag-CMV vector at sites Pst I/Xba I or Bgl II/Sal I.

The myc-CMV-PGC-1α was kindly provided by Dr Chang Liu (Nanjing Normal University, China). The Omi-related plasmids were previously described.45 Full-length human GSK3β complementary DNA (cDNA) was amplified by PCR from an adult human brain cDNA library with the primers 5′-AACTGCAGACCATGTCAGGGCGGCCCAGAA-3′ and 5′-TGCTCTAGAGGTGGAGTTGGAAGCTGATGC-3′ and inserted in-frame into the pGEX-5x-1 vector at the Bgl II/ECOR I sites or into the pEGFP-C2 vector at the BamH I/ECOR I sites. The C-terminal HA-tagged and Flag-tagged GSK3β was created by subcloning the PCR product amplified with primers 5′-AACTGCAGACCATGTCAGGGCGGCCCAGAA-3′ and 5′-TGCTCTAGAGGTGGAGTTGGAAGCTGATGC-3′ or 5′-TAAAGATCTGATGTCAGGGCGGCCCAGAAC-3′and 5′-TAAGTCGACGGTGGAGTTGGAAGCTGATGC-3′ into pKH3-HA vector or p3x-Flag-CMV vector at sites Pst I/Xba I or Bgl II/Sal I.

RNA interferenceThe oligonucleotides were transfected with Oligofectamine (Invitrogen) according to the manufacturer's instructions. The oligonucleotide sequences targeting to Omi (si-Omi) and the negative control small interfering RNA (si-NC) were previously described,45 and the oligonucleotide sequences targeting to GSK3β are shown as follows: sense 1, 5′-CTGCATTTATCGTTAACCTAATTdTdT-3′ and antisense 1, 5′-AATTAGGTTAACGATAAATGCAGdTdT-3′ sense 2, 5′-CAGGGCAGTGATTCTGTTCAATTdTdT-3′ and antisense 2, 5′-AATTGAACAGAATCACACTGCCCTGdTdT-3′.

RNA interference

The oligonucleotides were transfected with Oligofectamine (Invitrogen) according to the manufacturer's instructions. The oligonucleotide sequences targeting to Omi (si-Omi) and the negative control small interfering RNA (si-NC) were previously described,45 and the oligonucleotide sequences targeting to GSK3β are shown as follows: sense 1, 5′-CTGCATTTATCGTTAACCTAATTdTdT-3′ and antisense 1, 5′-AATTAGGTTAACGATAAATGCAGdTdT-3′ sense 2, 5′-CAGGGCAGTGATTCTGTTCAATTdTdT-3′ and antisense 2, 5′-AATTGAACAGAATCACACTGCCCTGdTdT-3′.

Western blotting analysis and antibodiesTissue homogenate and cell extracts were lysed in the 1 × RIPA lysis buffer (25 mM tris-HCl, pH 7.6, 150 mM NaCl, 1% NP-40 and 1% sodium deoxycholate) in the presence of a protease inhibitor cocktail (Roche, Indianapolis, IN, USA). Nearly 20 μg of cell lysate was resolved by SDS–PAGE and transferred onto a PVDF membrane (Millipore, Bedford, MA, USA). Western blotting analysis was carried out with the following primary antibodies: anti-α-Tubulin (CP06, Calbiochem, San Diego, CA, USA), anti-AKT1 (sc-1618, Santa Cruz Biotechnology, Santa Cruz, CA, USA), anti-AMPKα (23A3, Cell Signaling, Beverly, MA, USA), anti-Cdc4 (ab171961, Abcam, Cambridge, MA, USA), anti-COX IV (S0553, Epitomics, Burlingame, CA, USA), anti-CytC (3895-1, Epitomics), anti-Flag (F3165, Sigma-Aldrich), anti-glyceraldehyde-3-phosphate dehydrogenase (GAPDH) (MAB374, CHEMICON, Temecula, CA, USA), anti-GFP (sc-9996, Santa Cruz Biotechnology), anti-GSK-3β (NBP1-47470, Novus, Littleton, CO, USA or sc-7291, Santa Cruz Biotechnology), anti-GST (sc-138, Santa Cruz Biotechnology), anti-p38 (sc-7972, Santa Cruz Biotechnology), anti-HA (sc-7392, Santa Cruz Biotechnology), anti-His (M30111, Abmart, Shanghai, China), anti-Omi (AF1458, R&D Systems, Minneapolis, MN, USA or prepared by our laboratory), anti-p70s6k (#3204-1, Epitomics), anti-PGC-1α (ST1202, Calbiochem, or sc-13067, Santa Cruz Biotechnology), anti-Phospho-AMPKα (Thr 172; 40H9, Cell Signaling), anti-Phospho-AKT1 (#2118-1, Epitomics), anti-Phospho-GSK3α/β (pY279/pY261) (#2309-1-1, Epitomics), anti-Phospho-GSK3β (Ser9; AG753, Biyuntian, Shanghai, China), anti-Phospho-p38 MAPK (pT180/pY182; 1229-1, Epitomics), anti-Phospho-p70s6k (#1494-1, Epitomics), anti-SIRT-1 (13161-AP, Proteintech, Chicago, IL, USA), anti-TOM70 (14528-1-AP, Proteintech), anti-Ub (sc-8017, Santa Cruz Biotechnology). The anti-mouse and anti-rabbit secondary antibodies coupled to horseradish peroxidase were from Amersham Pharmacia Biotech (Peapack, NJ, USA). The proteins were viewed with an ECL detection kit (Amersham Pharmacia Biotech, Arlington Heights, IL, USA).

Western blotting analysis and antibodies

Tissue homogenate and cell extracts were lysed in the 1 × RIPA lysis buffer (25 mM tris-HCl, pH 7.6, 150 mM NaCl, 1% NP-40 and 1% sodium deoxycholate) in the presence of a protease inhibitor cocktail (Roche, Indianapolis, IN, USA). Nearly 20 μg of cell lysate was resolved by SDS–PAGE and transferred onto a PVDF membrane (Millipore, Bedford, MA, USA). Western blotting analysis was carried out with the following primary antibodies: anti-α-Tubulin (CP06, Calbiochem, San Diego, CA, USA), anti-AKT1 (sc-1618, Santa Cruz Biotechnology, Santa Cruz, CA, USA), anti-AMPKα (23A3, Cell Signaling, Beverly, MA, USA), anti-Cdc4 (ab171961, Abcam, Cambridge, MA, USA), anti-COX IV (S0553, Epitomics, Burlingame, CA, USA), anti-CytC (3895-1, Epitomics), anti-Flag (F3165, Sigma-Aldrich), anti-glyceraldehyde-3-phosphate dehydrogenase (GAPDH) (MAB374, CHEMICON, Temecula, CA, USA), anti-GFP (sc-9996, Santa Cruz Biotechnology), anti-GSK-3β (NBP1-47470, Novus, Littleton, CO, USA or sc-7291, Santa Cruz Biotechnology), anti-GST (sc-138, Santa Cruz Biotechnology), anti-p38 (sc-7972, Santa Cruz Biotechnology), anti-HA (sc-7392, Santa Cruz Biotechnology), anti-His (M30111, Abmart, Shanghai, China), anti-Omi (AF1458, R&D Systems, Minneapolis, MN, USA or prepared by our laboratory), anti-p70s6k (#3204-1, Epitomics), anti-PGC-1α (ST1202, Calbiochem, or sc-13067, Santa Cruz Biotechnology), anti-Phospho-AMPKα (Thr 172; 40H9, Cell Signaling), anti-Phospho-AKT1 (#2118-1, Epitomics), anti-Phospho-GSK3α/β (pY279/pY261) (#2309-1-1, Epitomics), anti-Phospho-GSK3β (Ser9; AG753, Biyuntian, Shanghai, China), anti-Phospho-p38 MAPK (pT180/pY182; 1229-1, Epitomics), anti-Phospho-p70s6k (#1494-1, Epitomics), anti-SIRT-1 (13161-AP, Proteintech, Chicago, IL, USA), anti-TOM70 (14528-1-AP, Proteintech), anti-Ub (sc-8017, Santa Cruz Biotechnology). The anti-mouse and anti-rabbit secondary antibodies coupled to horseradish peroxidase were from Amersham Pharmacia Biotech (Peapack, NJ, USA). The proteins were viewed with an ECL detection kit (Amersham Pharmacia Biotech, Arlington Heights, IL, USA).

GST pulldown assaysFor in vitro pulldown assay, an aliquot containing 20 mg/μg of GST or GST- GSK3β that was expressed in E. coli strain JM109 was incubated with 30 ml of Glutathione Sepharose 4B (Amersham Biosciences, Tokyo, Japan) for 30 min at 4 °C. GST or GST-GSK-3β bound to Sepharose was incubated with 50 mg of His-Omi protein from the supernatants of E. coli crude extracts containing His-tagged proteins in 0.25 ml of HNTG buffer (20 mM Hepes-KOH, pH 7.5, 0.1% Triton X-100, 100 mM NaCl and 10% glycerol) for 1 h at 4 °C. For semi in vivo pulldown assay, an aliquot containing 20 μg of GST or GST-S276C mutant Omi was incubated with 30 ml of Glutathione Sepharose 4B for 30 min at 4 °C. GST or GST-Omi bound to Sepharose was incubated with mouse brain tissue homogenates in 0.25 ml of HNTG buffer for 1 h at 4 °C. After incubation, the beads were washed six times with 1 ml of HNTG buffer each time to remove unbound protein. Bound protein was eluted by boiling in SDS–PAGE sample buffer, and then subjected to western blotting analysis.

GST pulldown assays

For in vitro pulldown assay, an aliquot containing 20 mg/μg of GST or GST- GSK3β that was expressed in E. coli strain JM109 was incubated with 30 ml of Glutathione Sepharose 4B (Amersham Biosciences, Tokyo, Japan) for 30 min at 4 °C. GST or GST-GSK-3β bound to Sepharose was incubated with 50 mg of His-Omi protein from the supernatants of E. coli crude extracts containing His-tagged proteins in 0.25 ml of HNTG buffer (20 mM Hepes-KOH, pH 7.5, 0.1% Triton X-100, 100 mM NaCl and 10% glycerol) for 1 h at 4 °C. For semi in vivo pulldown assay, an aliquot containing 20 μg of GST or GST-S276C mutant Omi was incubated with 30 ml of Glutathione Sepharose 4B for 30 min at 4 °C. GST or GST-Omi bound to Sepharose was incubated with mouse brain tissue homogenates in 0.25 ml of HNTG buffer for 1 h at 4 °C. After incubation, the beads were washed six times with 1 ml of HNTG buffer each time to remove unbound protein. Bound protein was eluted by boiling in SDS–PAGE sample buffer, and then subjected to western blotting analysis.

ImmunoprecipitationHomogenates from mnd2 mouse brains and cells that were transfected with the indicated plasmids were lysed in TSPI buffer containing 50 mM Tris-HCl (pH 7.5), 150 mM sodium chloride, 1 mM EDTA and 1% NP-40 supplemented with a complete mini protease inhibitor cocktail (Roche).The lysate was incubated with anti-Omi antibodies or normal rabbit immunoglobulin G overnight at 4 °C. After incubation, protein G-Sepharose (Roche) was used for precipitations. The beads were washed with TSPI buffer six times and were eluted with SDS sample buffer for western blotting analysis.

Immunoprecipitation

Homogenates from mnd2 mouse brains and cells that were transfected with the indicated plasmids were lysed in TSPI buffer containing 50 mM Tris-HCl (pH 7.5), 150 mM sodium chloride, 1 mM EDTA and 1% NP-40 supplemented with a complete mini protease inhibitor cocktail (Roche).

The lysate was incubated with anti-Omi antibodies or normal rabbit immunoglobulin G overnight at 4 °C. After incubation, protein G-Sepharose (Roche) was used for precipitations. The beads were washed with TSPI buffer six times and were eluted with SDS sample buffer for western blotting analysis.

In vitro proteolytic cleavage assayGST, GST-GSK3β or β-casein (Sigma-Aldrich) was incubated with 3 μg of 6 × His-tagged mature Omi in protease cleavage buffer (50 mM tris-HCl, pH 7.6, 1 mM dithiothreitol) for 60 min at 37 °C. The reactions were stopped by the addition of 2 × SDS loading buffer and boiled for 10 min. The samples were subjected to western blotting analyses with an anti-GST or anti-GSK3β antibody.

In vitro proteolytic cleavage assay

GST, GST-GSK3β or β-casein (Sigma-Aldrich) was incubated with 3 μg of 6 × His-tagged mature Omi in protease cleavage buffer (50 mM tris-HCl, pH 7.6, 1 mM dithiothreitol) for 60 min at 37 °C. The reactions were stopped by the addition of 2 × SDS loading buffer and boiled for 10 min. The samples were subjected to western blotting analyses with an anti-GST or anti-GSK3β antibody.

RNA isolation and real-time PCRTotal RNA was extracted from brain using Trizol (Invitrogen) according to the manufacturer's instructions. cDNA was synthesized using a TransScript First-Strand cDNA Synthesis Kit (Takara, Shiga, Japan). Real-time PCR was performed with SYBR Green Real-Time PCR Master Mix (Takara) reagents using a BioRad MiniOpticon System. Relative gene expressions were calculated by the 2−ΔΔCt method. Primers for RT-qPCR used here are shown as follows: PGC-1α forward 5′-AGCCGTGACCACTGACAACGAG-3′, reverse 5′-GCTGCATGGTTCTGAGTGCTAAG-3′ PGC-1β forward 5′-GGCAGGTTCAACCCCGA-3′, reverse 5′-CTTGCTAACATCACAGAGGATATCTTG-3′ GSK3β forward 5′-TAAGGAAGGAAAAGGTGAtTCGA-3′, reverse 5′-CCAAAAGCTGAAGGCTGCTG-3′ NRF-1 forward 5′-GTTGGTACAGGGGCAACAGT-3′, reverse 5′-TCGTCTGGATGGTCATTTCA-3′ TFAM forward 5′-CTGATGGGTATGGAGAAGGAGG-3′, reverse 5′-CCAACTTCAGCCATCTGCTCTTC-3′ UCP2 forward 5′-ACTTTCCCTCTGGATACCGC-3′, reverse 5′-ACGGAGGCAAAGCTCATCTG-3′ ANT1 forward 5′-GTCTCTGTCCAGGGCATCAT-3′, reverse 5′-ACGACGAACAGTGTCAAACG-3′ ATP5B forward 5′-GAGGGATTACCACCCATCCT-3′, reverse 5′-CATGATTCTGCCCAAGGTCT-3′ CYTC forward 5′-CCAAATCTCCACGGTCTGTTC-3′, reverse 5′-ATCAGGGTATCCTCTCCCCAG-3′ COX IV forward 5′-CGAGAGCTTCGCCGAGATGAAC-3′, reverse 5;-GCAGCTTCCAGCATGCCGAGG-3′ COX II forward 5′-ACGAAATCAACAACCCCGTA-3′, reverse 5′-GGCAGAACGACTCGGTTATC-3′ β-actin forward 5′-TGTCCACCTTCCAGCAGATGT-3′, reverse 5′-AGCTCAGTAACAGTCCGCCTAGA-3′.

RNA isolation and real-time PCR

Total RNA was extracted from brain using Trizol (Invitrogen) according to the manufacturer's instructions. cDNA was synthesized using a TransScript First-Strand cDNA Synthesis Kit (Takara, Shiga, Japan). Real-time PCR was performed with SYBR Green Real-Time PCR Master Mix (Takara) reagents using a BioRad MiniOpticon System. Relative gene expressions were calculated by the 2−ΔΔCt method. Primers for RT-qPCR used here are shown as follows: PGC-1α forward 5′-AGCCGTGACCACTGACAACGAG-3′, reverse 5′-GCTGCATGGTTCTGAGTGCTAAG-3′ PGC-1β forward 5′-GGCAGGTTCAACCCCGA-3′, reverse 5′-CTTGCTAACATCACAGAGGATATCTTG-3′ GSK3β forward 5′-TAAGGAAGGAAAAGGTGAtTCGA-3′, reverse 5′-CCAAAAGCTGAAGGCTGCTG-3′ NRF-1 forward 5′-GTTGGTACAGGGGCAACAGT-3′, reverse 5′-TCGTCTGGATGGTCATTTCA-3′ TFAM forward 5′-CTGATGGGTATGGAGAAGGAGG-3′, reverse 5′-CCAACTTCAGCCATCTGCTCTTC-3′ UCP2 forward 5′-ACTTTCCCTCTGGATACCGC-3′, reverse 5′-ACGGAGGCAAAGCTCATCTG-3′ ANT1 forward 5′-GTCTCTGTCCAGGGCATCAT-3′, reverse 5′-ACGACGAACAGTGTCAAACG-3′ ATP5B forward 5′-GAGGGATTACCACCCATCCT-3′, reverse 5′-CATGATTCTGCCCAAGGTCT-3′ CYTC forward 5′-CCAAATCTCCACGGTCTGTTC-3′, reverse 5′-ATCAGGGTATCCTCTCCCCAG-3′ COX IV forward 5′-CGAGAGCTTCGCCGAGATGAAC-3′, reverse 5;-GCAGCTTCCAGCATGCCGAGG-3′ COX II forward 5′-ACGAAATCAACAACCCCGTA-3′, reverse 5′-GGCAGAACGACTCGGTTATC-3′ β-actin forward 5′-TGTCCACCTTCCAGCAGATGT-3′, reverse 5′-AGCTCAGTAACAGTCCGCCTAGA-3′.

Analysis of mtDNATotal genomic DNA was extracted using a GenElute mammalian genomic DNA kit (Sigma-Aldrich) according to the manufacturer's instructions. mNADH1, mCYTB, mNADH4, mATP6 and mCOX II were used as mtDNA markers, and nuclear intron of β-globin was used as a nDNA marker. mTND1, mCYTB, mMTND4, mATP6 and mCOX II primers were designed from regions of mtDNA that are not found in nuclear-encoded mitochondrial pseudogenes, with the primers which were described elsewhere as following:57, 58, 59 mNADH1 forward 5′-TCTGCCAGCCTGACCCATAG-3′, reverse 5′-CCGGCTGCGTATTCTACGTT-3′ mCYTB forward 5′-TATTCCTTCATGTCGGACGA-3′, reverse 5′-AAATGCTGTGGCTATGACTG-3′ mNADH4 forward 5′-ATTATTATTACCCGATGAGGGAACC-3′, reverse 5′-ATTAAGATGAGGGCAATTAGCAGT-3′ mATP6 forward 5′-AATTACAGGCTTCCGACACAAAC-3′, reverse 5′-TGGAATTAGTGAAATTGGAGTTCCT-3′ mCOX2 forward 5′-GCCGACTAAATCAAGCAACA-3′, reverse 5′- CAATGGGCATAAAGCTATGG-3′ β-globin forward 5′-GAAGCGATTCTAGGGAGCAG-3′, reverse 5′-GGAGCAGCGATTCTGAGTAGA-3′. The quantification of abovementioned product was analyzed after being normalized to β-globin using one-way ANOVA.

Analysis of mtDNA

Total genomic DNA was extracted using a GenElute mammalian genomic DNA kit (Sigma-Aldrich) according to the manufacturer's instructions. mNADH1, mCYTB, mNADH4, mATP6 and mCOX II were used as mtDNA markers, and nuclear intron of β-globin was used as a nDNA marker. mTND1, mCYTB, mMTND4, mATP6 and mCOX II primers were designed from regions of mtDNA that are not found in nuclear-encoded mitochondrial pseudogenes, with the primers which were described elsewhere as following:57, 58, 59 mNADH1 forward 5′-TCTGCCAGCCTGACCCATAG-3′, reverse 5′-CCGGCTGCGTATTCTACGTT-3′ mCYTB forward 5′-TATTCCTTCATGTCGGACGA-3′, reverse 5′-AAATGCTGTGGCTATGACTG-3′ mNADH4 forward 5′-ATTATTATTACCCGATGAGGGAACC-3′, reverse 5′-ATTAAGATGAGGGCAATTAGCAGT-3′ mATP6 forward 5′-AATTACAGGCTTCCGACACAAAC-3′, reverse 5′-TGGAATTAGTGAAATTGGAGTTCCT-3′ mCOX2 forward 5′-GCCGACTAAATCAAGCAACA-3′, reverse 5′- CAATGGGCATAAAGCTATGG-3′ β-globin forward 5′-GAAGCGATTCTAGGGAGCAG-3′, reverse 5′-GGAGCAGCGATTCTGAGTAGA-3′. The quantification of abovementioned product was analyzed after being normalized to β-globin using one-way ANOVA.

Subcellular Fractionation AssayThe cytosolic and mitochondrial fractions from SH-SY5Y and N2a cells were isolated according to the instructions of the Mitochondria Isolation kit for cultured Cells (Beyotime, China). The lysates of fractions were subjected to western blotting analyses. GAPDH was used as the cytosolic marker and TOM70 was used as the mitochondrial marker.

Subcellular Fractionation Assay

The cytosolic and mitochondrial fractions from SH-SY5Y and N2a cells were isolated according to the instructions of the Mitochondria Isolation kit for cultured Cells (Beyotime, China). The lysates of fractions were subjected to western blotting analyses. GAPDH was used as the cytosolic marker and TOM70 was used as the mitochondrial marker.

Rotarod testMice were detected for their ability to run on a 3-cm diameter rotarod at speed of 10 revolutions per minute (r.p.m.) as previously described with modification.30 To reach a stable performance, mice were trained at speed of 5–10 r.p.m. for 2 days before the first-time test. The tests were performed using mice that were 16, 20, 25 and 30 days old, with a treatment using SB216763 or DMSO. The time that the mice spent on the rotarod before falling was recorded for each trial, and the trial was repeated thrice. To reduce stress and fatigue, there was at least 10 min of rest between trials, and a cut-off time point was set at 600 s.

Rotarod test

Mice were detected for their ability to run on a 3-cm diameter rotarod at speed of 10 revolutions per minute (r.p.m.) as previously described with modification.30 To reach a stable performance, mice were trained at speed of 5–10 r.p.m. for 2 days before the first-time test. The tests were performed using mice that were 16, 20, 25 and 30 days old, with a treatment using SB216763 or DMSO. The time that the mice spent on the rotarod before falling was recorded for each trial, and the trial was repeated thrice. To reduce stress and fatigue, there was at least 10 min of rest between trials, and a cut-off time point was set at 600 s.

Electron microscopyFor electron microscopic analysis of the brain, the tissue was prepared as previously described.18
Mnd2 mice that were 25 days old and age-matched wild-type mice were transcardially perfused with 0.1 M phosphate buffer for 6–8 min, followed by 2% formaldehyde/4% glutaraldehyde fixative in 0.1 M phosphate buffer for 6–8 min. Dissected striatum were further fixed with the same fixative for 2–4 h at 4 °C and postfixed with 2% OsO4 in buffer and embedded in Araldite. Ultrathin sections of striatum were cut onto formvar-coated slot grids stained with uranyl acetate and lead citrate and viewed with a JEM-1230 electron microscope. To calculate mitochondrial density, six different fields at the magnification of 5000 for each animal were quantified as described previously.60

Electron microscopy

For electron microscopic analysis of the brain, the tissue was prepared as previously described.18
Mnd2 mice that were 25 days old and age-matched wild-type mice were transcardially perfused with 0.1 M phosphate buffer for 6–8 min, followed by 2% formaldehyde/4% glutaraldehyde fixative in 0.1 M phosphate buffer for 6–8 min. Dissected striatum were further fixed with the same fixative for 2–4 h at 4 °C and postfixed with 2% OsO4 in buffer and embedded in Araldite. Ultrathin sections of striatum were cut onto formvar-coated slot grids stained with uranyl acetate and lead citrate and viewed with a JEM-1230 electron microscope. To calculate mitochondrial density, six different fields at the magnification of 5000 for each animal were quantified as described previously.60

Statistical analysisDensitometric analysis of western blots from three independent experiments was implemented with Adobe Photoshop 7.0 and the data were analyzed with Origin 6.0 (OriginLab, Northampton, MA, USA). Quantitative data are presented as means±S.E.M. Statistical significance was assessed via one-way ANOVA and the criterion of significance was set at P<0.05.

Statistical analysis

Densitometric analysis of western blots from three independent experiments was implemented with Adobe Photoshop 7.0 and the data were analyzed with Origin 6.0 (OriginLab, Northampton, MA, USA). Quantitative data are presented as means±S.E.M. Statistical significance was assessed via one-way ANOVA and the criterion of significance was set at P<0.05.
