---
pmid: '24089205'
title: Autophagy promotes primary ciliogenesis by removing OFD1 from centriolar satellites.
authors:
- Tang Z
- Lin MG
- Stowe TR
- Chen S
- Zhu M
- Stearns T
- Franco B
- Zhong Q
journal: Nature
year: '2013'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4075283
doi: 10.1038/nature12606
---

# Autophagy promotes primary ciliogenesis by removing OFD1 from centriolar satellites.
**Authors:** Tang Z, Lin MG, Stowe TR, Chen S, Zhu M, Stearns T, Franco B, Zhong Q
**Journal:** Nature (2013)
**DOI:** [10.1038/nature12606](https://doi.org/10.1038/nature12606)
**PMC:** [PMC4075283](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4075283/)

## Abstract

1. Nature. 2013 Oct 10;502(7470):254-7. doi: 10.1038/nature12606. Epub 2013 Oct
2.

Autophagy promotes primary ciliogenesis by removing OFD1 from centriolar 
satellites.

Tang Z(1), Lin MG, Stowe TR, Chen S, Zhu M, Stearns T, Franco B, Zhong Q.

Author information:
(1)Center for Autophagy Research, Department of Internal Medicine, University of 
Texas Southwestern Medical Center, Dallas, Texas 75390, USA.

Comment in
    Nature. 2013 Oct 10;502(7470):180-1. doi: 10.1038/nature12693.
    Nat Rev Mol Cell Biol. 2013 Nov;14(11):687. doi: 10.1038/nrm3686.
    Dev Cell. 2013 Oct 28;27(2):126-127. doi: 10.1016/j.devcel.2013.10.013.

The primary cilium is a microtubule-based organelle that functions in sensory 
and signalling pathways. Defects in ciliogenesis can lead to a group of genetic 
syndromes known as ciliopathies. However, the regulatory mechanisms of primary 
ciliogenesis in normal and cancer cells are incompletely understood. Here we 
demonstrate that autophagic degradation of a ciliopathy protein, OFD1 
(oral-facial-digital syndrome 1), at centriolar satellites promotes primary 
cilium biogenesis. Autophagy is a catabolic pathway in which cytosol, damaged 
organelles and protein aggregates are engulfed in autophagosomes and delivered 
to lysosomes for destruction. We show that the population of OFD1 at the 
centriolar satellites is rapidly degraded by autophagy upon serum starvation. In 
autophagy-deficient Atg5 or Atg3 null mouse embryonic fibroblasts, OFD1 
accumulates at centriolar satellites, leading to fewer and shorter primary cilia 
and a defective recruitment of BBS4 (Bardet-Biedl syndrome 4) to cilia. These 
defects are fully rescued by OFD1 partial knockdown that reduces the population 
of OFD1 at centriolar satellites. More strikingly, OFD1 depletion at centriolar 
satellites promotes cilia formation in both cycling cells and transformed breast 
cancer MCF7 cells that normally do not form cilia. This work reveals that 
removal of OFD1 by autophagy at centriolar satellites represents a general 
mechanism to promote ciliogenesis in mammalian cells. These findings define a 
newly recognized role of autophagy in organelle biogenesis.

DOI: 10.1038/nature12606
PMCID: PMC4075283
PMID: 24089205 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing financial 
interests.

## Full Text

During autophagy, the membrane anchored LC3 (microtubule-associated protein 1 light chain 3) interacts with cargo and cargo-adaptor proteins, recruiting cargoes to the autophagosome for subsequent degradation upon fusion of the autophagosome with the lysosome5–11. We carried out a tandem-affinity purification using tagged LC3 as bait to search for its interacting proteins (Fig. 1a). In addition to known LC3-interacting proteins (MAP1B, FYCO1, p62, and Keap112), we identified a set of centriolar satellite proteins, including PCM1, OFD1 and CEP131, that had not previously been shown to associate with LC3. PCM1 was also pulled down by LC3 orthologs, GATE16 and GABARAP (Extended Data Fig. 1a). PCM1, OFD1 and LC3 co-immunoprecipitated with each other, suggesting that they are in the same complex (Fig. 1b–1d). PCM1 likely enhances the interaction between LC3 and OFD1, as the OFD1-LC3 interaction is compromised in PCM1-depleted cells (Fig. 1e, f).

Depletion of PCM1 by RNA interference had no significant effect on autophagy activity as determined by LC3 lipidation and p62 degradation (Extended Data Fig. 1b). We then examined if any of these centriolar satellite proteins is an autophagy substrate. Ofd1 protein levels were reduced by serum starvation and this reduction was compromised in autophagy-deficient Atg5−/− mouse embryonic fibroblasts (MEFs) compared to Atg5+/+ MEFs, whereas PCM1, IFT88 and BBS4 protein levels were not altered by serum starvation or in Atg5−/− MEFs (Fig. 1g). The messenger RNA levels of Ofd1 were not significantly changed upon serum starvation in Atg5+/+ and Atg5−/− MEFs (Extended Data Fig. 1c), suggesting that Ofd1 protein level reduction upon serum starvation is through protein degradation rather than transcriptional downregulation. Blocking autophagic flux by lysosomal inhibitors bafilomycin A1 (Baf) or chloroquine (CQ) resulted in increased Ofd1 accumulation upon serum starvation (Fig. 1h). Taken together, these data suggest that Ofd1 is degraded via the autophagy-lysosome pathway upon serum starvation.

OFD1 is the gene underlying the human disease oral-facial-digital syndrome type 1 (OFD1), an X-linked ciliopathy characterized by morphological abnormalities and renal cysts, as well as Joubert syndrome and Simpson-Golabi-Behmel syndrome type 213–18. OFD1 localizes to the distal ends of centrioles and is necessary for distal appendage formation, IFT88 recruitment, and primary cilium formation18,19. OFD1 also localizes to centriolar satellites, interacting with proteins associated with human ciliary disease, PCM1, Cep290, BBS420. However, the function of this OFD1 population remains unclear.

The centriolar satellite localization of OFD1 is determined by PCM1, since OFD1 was lost from satellites when PCM1 was depleted (Extended Data Fig. 2a, b). LC3 partially colocalized with PCM1 upon serum starvation in a majority of retinal pigment epithelial (RPE) cells, but rarely in unstressed cells (Extended Data Fig. 2c, d). LC3 also partially colocalized with endogenous OFD1 when lysosome activity is blocked by Baf or CQ treatment (Extended Data Fig. 3a, b). This colocalization was limited to the centriolar satellites, as LC3 did not colocalize with the centriole marker γ-tubulin (Extended Data Fig. 3c, d).

OFD1 was present at both the centrioles and the centriolar satellites in untreated RPE cells. Remarkably, the centriolar satellite pool of OFD1 was dramatically reduced upon serum starvation, while the population of OFD1 at the centrioles remained unchanged (Fig. 2a). This serum starvation- induced OFD1 degradation from centriolar satellites was blocked in Atg5+/+ MEFs treated with the lysosome inhibitor CQ (Extended Data Fig. 4a, b). Notably, PCM1 protein levels are not controlled by autophagy and the centriolar satellite distribution of PCM1 is not altered upon serum starvation (Fig. 1g, Extended Data Fig. 4c–e), suggesting that the autophagic degradation is specific to OFD1 at centriolar satellites rather than centriolar satellites as a whole. This notion is further supported by our observation that Ofd1 remained at centriolar satellites upon serum starvation in Atg5−/− MEFs but was lost from centriolar satellites in Atg5+/+ MEFs (Fig. 2b).

The loss of Ofd1 from centriolar satellites was significantly faster than the loss of Ofd1 from centrioles, and the rate of loss of Ofd1 in Atg5−/− MEFs was slower than that in Atg5+/+ MEFs (Extended Data Fig. 5a). The centriolar satellite pool of Ofd1 in Atg5+/+ cells was lost within six hours of serum starvation, whereas this pool of Ofd1 in Atg5−/− cells remained stable even after twenty-four hours of serum starvation (Extended Data Fig. 5b). These data confirm that Ofd1 at centriolar satellites has a faster turnover rate than Ofd1 at centrioles, and this serum starvation induced accelerated degradation is controlled by autophagy.

We next sought to understand how OFD1 regulation might affect ciliogenesis. We observed that the percentage of Atg5−/− cells that form a primary cilium, as compared to Atg5+/+ cells, was significantly reduced and the cilia that did form were shorter (Fig. 3a). This difference was not due to cell cycle regulation (Extended Data Fig. 6a). Lysosome inhibition also compromised primary ciliogenesis in Atg5+/+ MEFs (Extended Data Fig. 6b–d). The defective ciliogenesis phenotypes were not limited to Atg5−/− MEFs, as similar phenotypes were also observed in MEFs lacking another essential autophagy gene, Atg3 (Extended Data Fig. 6e–g).

We next investigated if the ciliary recruitment of BBS4, a critical event for ciliogenesis19,21, is affected in autophagy-deficient cells. We observed that BBS4 accumulated at centriolar satellites in Atg5−/− MEFs; more than 50% of cilia had detectable BBS4 in Atg5+/+ MEFs, whereas only about 10% of cilia were positive for BBS4 in Atg5−/− MEFs (Extended Data Fig. 7). These data indicate that BBS4 recruitment to cilia is also defective in autophagy deficient cells.

If Ofd1 accumulation at centriolar satellites is responsible for the ciliary defects in Atg5−/− MEFs, we would expect that depletion of Ofd1 by RNA interference might rescue these defects. In Atg5−/− MEFs stably expressing Ofd1 shRNA, with about 50% efficiency of overall depletion of Ofd1 (Extended Data Fig. 8a), Ofd1 remained at centrioles but was lost from centriolar satellites (Extended Data Fig. 8b–d). Primary cilium formation upon serum starvation was fully restored in these cells; nearly 70% of cells formed a cilium and the length of these cilia was comparable to those formed in Atg5+/+ MEFs (Fig. 3b). Hence, the centriolar satellite pool of Ofd1 indeed plays a key role in suppressing primary ciliogenesis in Atg5−/− MEFs. Strikingly, with Ofd1 knockdown, nearly 35% of the Atg5−/− cells formed cilia even without serum starvation (Fig. 3b), suggesting that Ofd1 degradation is likely required for serum starvation-induced primary ciliogenesis. This notion is supported by the observation that primary cilia formed efficiently even in the cycling Atg5+/+ MEFs depleted of Ofd1 without serum starvation (Extended Data Fig. 9a). The presence of cilia in MEFs was further confirmed by the positive staining of the cilium marker Arl13b (Extended Data Fig. 9b), which specifically decorates the ciliary membrane22.

Primary cilia are formed in normal breast epithelial cells but not detected in breast cancer cell lines such as MCF723. Restoration of primary cilia in human cancer cells might be beneficial to reduce malignancy, since cilia are required for proper functions of several signaling pathways (hedgehog etc.) that play crucial roles in many types of cancers24. We further tested if OFD1 reduction could restore primary ciliogenesis in MCF7 cells. We generated an OFD1 stable knockdown clone (C19) in MCF7 cells, which exhibited 44% ODF1 depletion efficiency (Fig. 4a). Similar to OFD1 knockdown MEFs, the centriolar satellite population of OFD1 in C19 cells was dramatically reduced, while the centriole pool remained stable (Extended Data Fig. 10a–c). Remarkably, primary cilia formed in about 20% C19 cells upon serum starvation, but were completely absent in parental MCF7 cells (Fig. 4b, c). The length of these cilia ranged from 2 μm to 6 μm (Fig. 4d). As expected for normal primary cilia, these cilia extended from basal bodies and were positive for Arl13b, acetylated-tubulin, and the intraflagellar transport protein IFT88 (Fig. 4e, Extended Data Fig. 10d–f). Finally these cilia could be directly visualized under the scanning electron microscope (Fig. 4f). Thus, by manipulating the expression of one protein, OFD1, it is possible to reverse the cilia-defective phenotype of a transformed breast cancer cell line. Primary ciliogenesis remains inefficient in C19 cells, suggesting that addition factors are required to reach full capacity. Nevertheless, these data confirm that OFD1 at the centriolar satellite functions as a crucial suppressor of primary ciliogenesis in human cancer cells.

We show here that autophagy is linked to primary ciliogenesis, by controlling the degradation of OFD1 at centriolar satellites. We propose that autophagy deficiency serves as a potential underlying mechanism of ciliopathies, and that autophagy modulation might provide a novel means of ciliopathy treatment. We demonstrate that OFD1 at centriolar satellites plays a crucial role in suppressing primary ciliogenesis, which is different from OFD1 at centrioles that has been shown to be essential for ciliogenesis. Removing OFD1 from centriolar satellites promotes ciliogenesis in autophagy-deficient cells, wild-type MEFs without serum starvation, and human breast cancer MCF7 cells that normally completely lack cilia, suggesting a general role of OFD1 in suppressing ciliogenesis. Primary ciliogenesis is defective in human breast and pancreatic cancer cells, but activated in the corresponding normal tissues/cells23,25. The contribution of primary cilium function to tumorigenesis is complex26,27, however our results suggest that dissecting the regulatory mechanisms of OFD1 will provide insight into these functions and potentially offer new therapeutic tools for treatment of ciliopathies and cancers.

Methods (online only)ReagentsRabbit anti-OFD1 antibody, EGFP-OFD1 and Flag-OFD1 were described before18,30,31. Another rabbit anti-OFD1 antibody was a gift from Jeremy Reiter at University of California at San Francisco. Rabbit anti-IFT88 antibody was a gift from Bradley Yoder at the University of Alabama at Birmingham. To generate BBS4-MycHAHis (pTS1686), BBS4 was PCR-amplified, verified by sequencing and cloned into a pEGFP-N1-derived plasmid (Clontech Laboratories, Inc.) in which GFP was replaced with a Myc-HA-His tag created as BBS4-MycHAHis32. Primary antibodies used for western blotting and immunostaining are; rabbit anti-PCM1 (Cell signaling), mouse anti-PCM1 (Sigma), mouse anti-β-tubulin (Sigma); rabbit anti-p62 (MBL); rabbit anti-LC3 (Sigma), mouse anti-tubulin (Sigma), mouse anti-acetylate tubulin (Sigma), mouse or rabbit anti-γ tubulin (Sigma), BBS4 (Santa Cruz), chloroquine (Sigma), bafilomycin A1 and rapamycin (LC Laboratories), and MG132 (Alexis).Cell culture and transfectionHEK293T, MCF7 and MEFs were cultured in DMEM with 10% FBS and 1% penicillin and streptomycin in a 5% CO2 incubator (Thermo). hTERT-RPE1 cells were cultured in DMEM/Hams F12 with 10% FBS and 1% penicillin at streptomycin in a 5% CO2 incubator (Thermo). Lipofectamine 2000 (Invitrogen) was used for mammalian cell transfection. Cells were incubated in OPTI medium (GIBCO 51985091) for one hour before transfection. 10 μg or 5 μg plasmid was used for each 150 mm or 100 mm tissue culture dish, respectively. U2OS cells stably expressing Myc-LC3 were described before28. Atg5+/+ and Atg5−/− MEFs were kind gifts from Noboru Mizushima at Tokyo Medical and Dental University33. Atg3+/+ and Atg3−/− MEFs were kind gifts from Masaaki Komatsu at Tokyo Metropolitan Institute of Medical Science34. PCM1 shRNA knockdown was carried out using a 19-mer shRNA targeting human PCM1 with the sequence: 5′-GTATCACATCTGAACTAAA-3′ (pTS2063). PCM1 shRNA oligos were designed using pSicoOligomaker 1.5, annealed and subcloned into the lentiviral vector, pSicoR-puro, which confers puromycin resistance. Stable depletion of PCM1 was carried out by infection of cells with shRNA-expressing lentiviral particles and selection with puromycin (3 μg/ml); positive clones were screened by western blotting. Stable depletion of OFD1 was carried out by infection of cells with shRNA-expressing lentiviral particles (sc-91245-V, from Santa Cruz) and selection with puromycin (1 μg/ml or 10 μg/ml) for MCF7 cells or MEFs, positive clones were screened by western blotting. The OFD1 Lentiviral Particles are a pool of 3 different shRNA plasmids, targeting the OFD1 sequence: 5′-GGAUGACUACAUC AUUAGAtt-3′, 5′-CUACUCAGGUUGCCGAUUUtt-3′, 5′-GAACGAAGAGAACU AGAAAtt-3′,Primary Cilia biogenesis assayEqual numbers (3×105 for serum rich medium and 4.5×105 for serum starved medium) of Atg5+/+ MEFS and Atg5−/− MEFs were seeded into 6-well dishes. After cells were attached on the coverslips for 8 hours, they were incubated in either fresh regular medium (10% FBS, 1% P/S DMEM) or serum starved medium (0.5% FBS, 1% P/S DMEM) for 24 hours. Cells were seeded at different numbers at beginning to ensure that these cells reach the same confluence one day after treatment. Cells were then fixed for immunofluorescence staining. For western blotting, cells cultured in 10 cm dishes were serum starved alone or in combination with 50 nM bafilomycin A1, 20 μM chloroquine or 1 μM MG132 treatment for 24 hours and lysed with TAP buffer.Immunofluorescence StainingCells were treated according to the protocol described above with or without serum starvation or compound treatment. Cells were fixed with cold methanol for five minutes at −20°C. Cells were then washed three times with PBS and then blocked with blocking buffer (2.5% BSA + 0.1% Triton X-100 in PBS) at room temperature for one hour. Cells were incubated with primary antibodies at 4°C overnight, washed five times with PBS buffer and then incubated with appropriate secondary antibodies conjugated to Alexa 488- Alexa 549 (Molecular Probes) for two hours at room temperature. DNA was stained with DAPI. Slides were examined using a laser scanning confocal microscope (Zeiss LSM 510 META UV/Vis).Autophagy AnalysisAutophagy was induced by serum starvation or rapamycin treatment, or blocked by lysosome inhibitors, chloroquine or bafilomycin A1. For rapamycin (LC Laboratories) treatment, cells were incubated with 500nM rapamycin in complete medium for 16 hours at 37°C. To block autophagy flux, 50 nM Bafilomycin A1 (LC Laboratories) or 20 μM chloroquine (Sigma) was added to complete medium or serum starvation medium and incubated for 24 h at 37°C. Autophagy activity was assessed using two approaches, LC3-II formation and p62 degradation. For LC3-II and p62 detection, cell lysates were prepared as described above and subjected to standard western blotting protocols using 1:10,000 dilution of the antibody against LC3 (Sigma) and 1:1,000 dilution of the antibody against p62 (MBL).Establishment of stable RNA interference cell linesHuman MCF7 cells or Atg5+/+ and Atg5−/− MEFs were seeded into 12-well dishes at 50% confluency before transfection. 24 hours after seeding, cells were incubated with complete medium containing 5 μg/ml polybrene (Santa Cruz), and infected overnight with lentiviral particles carrying short hairpin RNA OFD1 (Santa Cruz). 24 h later, cells were selected with puromycin (Invitrogen) (1 μg/ml for MCF 7 or 10 μg/ml for MEFs) until positive clones were identified. Single clones were picked and identified by western blotting or immunofluorescence analysis.Cell cycle analysis1×106 cells were centrifuged at 1500 rpm for 5 minutes, washed once with PBS, and pellets were resuspended in 1 ml PBS and fixed overnight in 75% ethanol at −20°C. Fixed cells were washed three times with 10 ml PBS, resuspended in 200–400 μl PBS with 10 μl RNAase (Qiagen) at 37°C for 30 minutes in the dark. Cells were subjected to FACS analysis using DAKO-Cytomation MoFlo High Speed Sorter at the Flow Cytometry Facility at UC Berkeley.Immunoprecipitation and immunoblotTransfected cells were lysed in TAP lysis buffer on ice for 30 minutes, and centrifuged at 13,000 g for 15 minutes. For immunoprecipitation, cell lysates were incubated with primary antibody overnight at 4°C, and then incubated with IgG beads for 2 hours at 4°C. The beads were washed 5 times with 1 ml of lysis buffer and boiled in SDS sample buffer. Co-immunoprecipitated proteins were resolved by SDS-PAGE, transferred to PVDF membranes (Bio-Rad), and detected with antigen-specific primary antibodies followed by HRP-conjugated secondary antibodies.Tandem Affinity Purification of LC3 ComplexesThis protocol is adapted from what was described before12,28,29. Stable cell lines expressing ZZ-Flag-LC3, ZZ-Flag-Gate16, or ZZ-Flag-GABARAP, upon doxycycline induction were treated with 10 ng/ml DOX for one day. The collected cell pellet was washed with chilled PBS three times and suspended in TAP lysis buffer. Resuspended cell pellets were incubated on ice for 30 minutes and then gently vortexed for one minute. Homogenates were centrifuged for 20 minutes at 10,000 g. The supernatants were transferred to fresh tubes containing 0.8 ml of packed IgG beads followed by gentle rotation overnight at 4°C. Bound protein was eluted using TEV protease cleavage and further purified by Anti-Flag M2 Affinity Gel (Sigma). Bound protein was eluted using Flag peptide and resolved using SDS-PAGE on a 4–12% gradient gel (Invitrogen) and visualized by silver staining (Invitrogen). Distinct bands were cut and subjected to mass spectrometry analysis.Quantitative RT–PCRTotal RNAs were extracted with Trizol reagent (Invitrogen), from which 1 μg was used for reverse transcription in a 20-μL reaction system with the RNA PCR (AMV) kit (Promega). Quantitative RT-PCR was performed with SYBR-Green PCR Mastermix (Applied Biosystems), and amplification was real-time monitored on a MicorAmp optical 96-Well Reaction Plate (Applied Biosystems). The level of transcript abundance relative to reference gene (termed ΔCT) was determined according to the function ΔCT = CT (test gene) − CT (reference gene). To compare untreated and treated expression levels, the function ΔΔCT was first determined using the equation ΔΔ CT = Δ CT (treatment) − Δ CT (control) where control represented mock-treated cells. The induction ratio of treatment/control was then calculated by the equation 2−ΔΔ CT. Gene expression levels were normalized with the Atg5+/+ untreated sample.

Methods (online only)

ReagentsRabbit anti-OFD1 antibody, EGFP-OFD1 and Flag-OFD1 were described before18,30,31. Another rabbit anti-OFD1 antibody was a gift from Jeremy Reiter at University of California at San Francisco. Rabbit anti-IFT88 antibody was a gift from Bradley Yoder at the University of Alabama at Birmingham. To generate BBS4-MycHAHis (pTS1686), BBS4 was PCR-amplified, verified by sequencing and cloned into a pEGFP-N1-derived plasmid (Clontech Laboratories, Inc.) in which GFP was replaced with a Myc-HA-His tag created as BBS4-MycHAHis32. Primary antibodies used for western blotting and immunostaining are; rabbit anti-PCM1 (Cell signaling), mouse anti-PCM1 (Sigma), mouse anti-β-tubulin (Sigma); rabbit anti-p62 (MBL); rabbit anti-LC3 (Sigma), mouse anti-tubulin (Sigma), mouse anti-acetylate tubulin (Sigma), mouse or rabbit anti-γ tubulin (Sigma), BBS4 (Santa Cruz), chloroquine (Sigma), bafilomycin A1 and rapamycin (LC Laboratories), and MG132 (Alexis).

Rabbit anti-OFD1 antibody, EGFP-OFD1 and Flag-OFD1 were described before18,30,31. Another rabbit anti-OFD1 antibody was a gift from Jeremy Reiter at University of California at San Francisco. Rabbit anti-IFT88 antibody was a gift from Bradley Yoder at the University of Alabama at Birmingham. To generate BBS4-MycHAHis (pTS1686), BBS4 was PCR-amplified, verified by sequencing and cloned into a pEGFP-N1-derived plasmid (Clontech Laboratories, Inc.) in which GFP was replaced with a Myc-HA-His tag created as BBS4-MycHAHis32. Primary antibodies used for western blotting and immunostaining are; rabbit anti-PCM1 (Cell signaling), mouse anti-PCM1 (Sigma), mouse anti-β-tubulin (Sigma); rabbit anti-p62 (MBL); rabbit anti-LC3 (Sigma), mouse anti-tubulin (Sigma), mouse anti-acetylate tubulin (Sigma), mouse or rabbit anti-γ tubulin (Sigma), BBS4 (Santa Cruz), chloroquine (Sigma), bafilomycin A1 and rapamycin (LC Laboratories), and MG132 (Alexis).

Cell culture and transfectionHEK293T, MCF7 and MEFs were cultured in DMEM with 10% FBS and 1% penicillin and streptomycin in a 5% CO2 incubator (Thermo). hTERT-RPE1 cells were cultured in DMEM/Hams F12 with 10% FBS and 1% penicillin at streptomycin in a 5% CO2 incubator (Thermo). Lipofectamine 2000 (Invitrogen) was used for mammalian cell transfection. Cells were incubated in OPTI medium (GIBCO 51985091) for one hour before transfection. 10 μg or 5 μg plasmid was used for each 150 mm or 100 mm tissue culture dish, respectively. U2OS cells stably expressing Myc-LC3 were described before28. Atg5+/+ and Atg5−/− MEFs were kind gifts from Noboru Mizushima at Tokyo Medical and Dental University33. Atg3+/+ and Atg3−/− MEFs were kind gifts from Masaaki Komatsu at Tokyo Metropolitan Institute of Medical Science34. PCM1 shRNA knockdown was carried out using a 19-mer shRNA targeting human PCM1 with the sequence: 5′-GTATCACATCTGAACTAAA-3′ (pTS2063). PCM1 shRNA oligos were designed using pSicoOligomaker 1.5, annealed and subcloned into the lentiviral vector, pSicoR-puro, which confers puromycin resistance. Stable depletion of PCM1 was carried out by infection of cells with shRNA-expressing lentiviral particles and selection with puromycin (3 μg/ml); positive clones were screened by western blotting. Stable depletion of OFD1 was carried out by infection of cells with shRNA-expressing lentiviral particles (sc-91245-V, from Santa Cruz) and selection with puromycin (1 μg/ml or 10 μg/ml) for MCF7 cells or MEFs, positive clones were screened by western blotting. The OFD1 Lentiviral Particles are a pool of 3 different shRNA plasmids, targeting the OFD1 sequence: 5′-GGAUGACUACAUC AUUAGAtt-3′, 5′-CUACUCAGGUUGCCGAUUUtt-3′, 5′-GAACGAAGAGAACU AGAAAtt-3′,

Cell culture and transfection

HEK293T, MCF7 and MEFs were cultured in DMEM with 10% FBS and 1% penicillin and streptomycin in a 5% CO2 incubator (Thermo). hTERT-RPE1 cells were cultured in DMEM/Hams F12 with 10% FBS and 1% penicillin at streptomycin in a 5% CO2 incubator (Thermo). Lipofectamine 2000 (Invitrogen) was used for mammalian cell transfection. Cells were incubated in OPTI medium (GIBCO 51985091) for one hour before transfection. 10 μg or 5 μg plasmid was used for each 150 mm or 100 mm tissue culture dish, respectively. U2OS cells stably expressing Myc-LC3 were described before28. Atg5+/+ and Atg5−/− MEFs were kind gifts from Noboru Mizushima at Tokyo Medical and Dental University33. Atg3+/+ and Atg3−/− MEFs were kind gifts from Masaaki Komatsu at Tokyo Metropolitan Institute of Medical Science34. PCM1 shRNA knockdown was carried out using a 19-mer shRNA targeting human PCM1 with the sequence: 5′-GTATCACATCTGAACTAAA-3′ (pTS2063). PCM1 shRNA oligos were designed using pSicoOligomaker 1.5, annealed and subcloned into the lentiviral vector, pSicoR-puro, which confers puromycin resistance. Stable depletion of PCM1 was carried out by infection of cells with shRNA-expressing lentiviral particles and selection with puromycin (3 μg/ml); positive clones were screened by western blotting. Stable depletion of OFD1 was carried out by infection of cells with shRNA-expressing lentiviral particles (sc-91245-V, from Santa Cruz) and selection with puromycin (1 μg/ml or 10 μg/ml) for MCF7 cells or MEFs, positive clones were screened by western blotting. The OFD1 Lentiviral Particles are a pool of 3 different shRNA plasmids, targeting the OFD1 sequence: 5′-GGAUGACUACAUC AUUAGAtt-3′, 5′-CUACUCAGGUUGCCGAUUUtt-3′, 5′-GAACGAAGAGAACU AGAAAtt-3′,

Primary Cilia biogenesis assayEqual numbers (3×105 for serum rich medium and 4.5×105 for serum starved medium) of Atg5+/+ MEFS and Atg5−/− MEFs were seeded into 6-well dishes. After cells were attached on the coverslips for 8 hours, they were incubated in either fresh regular medium (10% FBS, 1% P/S DMEM) or serum starved medium (0.5% FBS, 1% P/S DMEM) for 24 hours. Cells were seeded at different numbers at beginning to ensure that these cells reach the same confluence one day after treatment. Cells were then fixed for immunofluorescence staining. For western blotting, cells cultured in 10 cm dishes were serum starved alone or in combination with 50 nM bafilomycin A1, 20 μM chloroquine or 1 μM MG132 treatment for 24 hours and lysed with TAP buffer.

Primary Cilia biogenesis assay

Equal numbers (3×105 for serum rich medium and 4.5×105 for serum starved medium) of Atg5+/+ MEFS and Atg5−/− MEFs were seeded into 6-well dishes. After cells were attached on the coverslips for 8 hours, they were incubated in either fresh regular medium (10% FBS, 1% P/S DMEM) or serum starved medium (0.5% FBS, 1% P/S DMEM) for 24 hours. Cells were seeded at different numbers at beginning to ensure that these cells reach the same confluence one day after treatment. Cells were then fixed for immunofluorescence staining. For western blotting, cells cultured in 10 cm dishes were serum starved alone or in combination with 50 nM bafilomycin A1, 20 μM chloroquine or 1 μM MG132 treatment for 24 hours and lysed with TAP buffer.

Immunofluorescence StainingCells were treated according to the protocol described above with or without serum starvation or compound treatment. Cells were fixed with cold methanol for five minutes at −20°C. Cells were then washed three times with PBS and then blocked with blocking buffer (2.5% BSA + 0.1% Triton X-100 in PBS) at room temperature for one hour. Cells were incubated with primary antibodies at 4°C overnight, washed five times with PBS buffer and then incubated with appropriate secondary antibodies conjugated to Alexa 488- Alexa 549 (Molecular Probes) for two hours at room temperature. DNA was stained with DAPI. Slides were examined using a laser scanning confocal microscope (Zeiss LSM 510 META UV/Vis).

Immunofluorescence Staining

Cells were treated according to the protocol described above with or without serum starvation or compound treatment. Cells were fixed with cold methanol for five minutes at −20°C. Cells were then washed three times with PBS and then blocked with blocking buffer (2.5% BSA + 0.1% Triton X-100 in PBS) at room temperature for one hour. Cells were incubated with primary antibodies at 4°C overnight, washed five times with PBS buffer and then incubated with appropriate secondary antibodies conjugated to Alexa 488- Alexa 549 (Molecular Probes) for two hours at room temperature. DNA was stained with DAPI. Slides were examined using a laser scanning confocal microscope (Zeiss LSM 510 META UV/Vis).

Autophagy AnalysisAutophagy was induced by serum starvation or rapamycin treatment, or blocked by lysosome inhibitors, chloroquine or bafilomycin A1. For rapamycin (LC Laboratories) treatment, cells were incubated with 500nM rapamycin in complete medium for 16 hours at 37°C. To block autophagy flux, 50 nM Bafilomycin A1 (LC Laboratories) or 20 μM chloroquine (Sigma) was added to complete medium or serum starvation medium and incubated for 24 h at 37°C. Autophagy activity was assessed using two approaches, LC3-II formation and p62 degradation. For LC3-II and p62 detection, cell lysates were prepared as described above and subjected to standard western blotting protocols using 1:10,000 dilution of the antibody against LC3 (Sigma) and 1:1,000 dilution of the antibody against p62 (MBL).

Autophagy Analysis

Autophagy was induced by serum starvation or rapamycin treatment, or blocked by lysosome inhibitors, chloroquine or bafilomycin A1. For rapamycin (LC Laboratories) treatment, cells were incubated with 500nM rapamycin in complete medium for 16 hours at 37°C. To block autophagy flux, 50 nM Bafilomycin A1 (LC Laboratories) or 20 μM chloroquine (Sigma) was added to complete medium or serum starvation medium and incubated for 24 h at 37°C. Autophagy activity was assessed using two approaches, LC3-II formation and p62 degradation. For LC3-II and p62 detection, cell lysates were prepared as described above and subjected to standard western blotting protocols using 1:10,000 dilution of the antibody against LC3 (Sigma) and 1:1,000 dilution of the antibody against p62 (MBL).

Establishment of stable RNA interference cell linesHuman MCF7 cells or Atg5+/+ and Atg5−/− MEFs were seeded into 12-well dishes at 50% confluency before transfection. 24 hours after seeding, cells were incubated with complete medium containing 5 μg/ml polybrene (Santa Cruz), and infected overnight with lentiviral particles carrying short hairpin RNA OFD1 (Santa Cruz). 24 h later, cells were selected with puromycin (Invitrogen) (1 μg/ml for MCF 7 or 10 μg/ml for MEFs) until positive clones were identified. Single clones were picked and identified by western blotting or immunofluorescence analysis.

Establishment of stable RNA interference cell lines

Human MCF7 cells or Atg5+/+ and Atg5−/− MEFs were seeded into 12-well dishes at 50% confluency before transfection. 24 hours after seeding, cells were incubated with complete medium containing 5 μg/ml polybrene (Santa Cruz), and infected overnight with lentiviral particles carrying short hairpin RNA OFD1 (Santa Cruz). 24 h later, cells were selected with puromycin (Invitrogen) (1 μg/ml for MCF 7 or 10 μg/ml for MEFs) until positive clones were identified. Single clones were picked and identified by western blotting or immunofluorescence analysis.

Cell cycle analysis1×106 cells were centrifuged at 1500 rpm for 5 minutes, washed once with PBS, and pellets were resuspended in 1 ml PBS and fixed overnight in 75% ethanol at −20°C. Fixed cells were washed three times with 10 ml PBS, resuspended in 200–400 μl PBS with 10 μl RNAase (Qiagen) at 37°C for 30 minutes in the dark. Cells were subjected to FACS analysis using DAKO-Cytomation MoFlo High Speed Sorter at the Flow Cytometry Facility at UC Berkeley.

Cell cycle analysis

1×106 cells were centrifuged at 1500 rpm for 5 minutes, washed once with PBS, and pellets were resuspended in 1 ml PBS and fixed overnight in 75% ethanol at −20°C. Fixed cells were washed three times with 10 ml PBS, resuspended in 200–400 μl PBS with 10 μl RNAase (Qiagen) at 37°C for 30 minutes in the dark. Cells were subjected to FACS analysis using DAKO-Cytomation MoFlo High Speed Sorter at the Flow Cytometry Facility at UC Berkeley.

Immunoprecipitation and immunoblotTransfected cells were lysed in TAP lysis buffer on ice for 30 minutes, and centrifuged at 13,000 g for 15 minutes. For immunoprecipitation, cell lysates were incubated with primary antibody overnight at 4°C, and then incubated with IgG beads for 2 hours at 4°C. The beads were washed 5 times with 1 ml of lysis buffer and boiled in SDS sample buffer. Co-immunoprecipitated proteins were resolved by SDS-PAGE, transferred to PVDF membranes (Bio-Rad), and detected with antigen-specific primary antibodies followed by HRP-conjugated secondary antibodies.

Immunoprecipitation and immunoblot

Transfected cells were lysed in TAP lysis buffer on ice for 30 minutes, and centrifuged at 13,000 g for 15 minutes. For immunoprecipitation, cell lysates were incubated with primary antibody overnight at 4°C, and then incubated with IgG beads for 2 hours at 4°C. The beads were washed 5 times with 1 ml of lysis buffer and boiled in SDS sample buffer. Co-immunoprecipitated proteins were resolved by SDS-PAGE, transferred to PVDF membranes (Bio-Rad), and detected with antigen-specific primary antibodies followed by HRP-conjugated secondary antibodies.

Tandem Affinity Purification of LC3 ComplexesThis protocol is adapted from what was described before12,28,29. Stable cell lines expressing ZZ-Flag-LC3, ZZ-Flag-Gate16, or ZZ-Flag-GABARAP, upon doxycycline induction were treated with 10 ng/ml DOX for one day. The collected cell pellet was washed with chilled PBS three times and suspended in TAP lysis buffer. Resuspended cell pellets were incubated on ice for 30 minutes and then gently vortexed for one minute. Homogenates were centrifuged for 20 minutes at 10,000 g. The supernatants were transferred to fresh tubes containing 0.8 ml of packed IgG beads followed by gentle rotation overnight at 4°C. Bound protein was eluted using TEV protease cleavage and further purified by Anti-Flag M2 Affinity Gel (Sigma). Bound protein was eluted using Flag peptide and resolved using SDS-PAGE on a 4–12% gradient gel (Invitrogen) and visualized by silver staining (Invitrogen). Distinct bands were cut and subjected to mass spectrometry analysis.

Tandem Affinity Purification of LC3 Complexes

This protocol is adapted from what was described before12,28,29. Stable cell lines expressing ZZ-Flag-LC3, ZZ-Flag-Gate16, or ZZ-Flag-GABARAP, upon doxycycline induction were treated with 10 ng/ml DOX for one day. The collected cell pellet was washed with chilled PBS three times and suspended in TAP lysis buffer. Resuspended cell pellets were incubated on ice for 30 minutes and then gently vortexed for one minute. Homogenates were centrifuged for 20 minutes at 10,000 g. The supernatants were transferred to fresh tubes containing 0.8 ml of packed IgG beads followed by gentle rotation overnight at 4°C. Bound protein was eluted using TEV protease cleavage and further purified by Anti-Flag M2 Affinity Gel (Sigma). Bound protein was eluted using Flag peptide and resolved using SDS-PAGE on a 4–12% gradient gel (Invitrogen) and visualized by silver staining (Invitrogen). Distinct bands were cut and subjected to mass spectrometry analysis.

Quantitative RT–PCRTotal RNAs were extracted with Trizol reagent (Invitrogen), from which 1 μg was used for reverse transcription in a 20-μL reaction system with the RNA PCR (AMV) kit (Promega). Quantitative RT-PCR was performed with SYBR-Green PCR Mastermix (Applied Biosystems), and amplification was real-time monitored on a MicorAmp optical 96-Well Reaction Plate (Applied Biosystems). The level of transcript abundance relative to reference gene (termed ΔCT) was determined according to the function ΔCT = CT (test gene) − CT (reference gene). To compare untreated and treated expression levels, the function ΔΔCT was first determined using the equation ΔΔ CT = Δ CT (treatment) − Δ CT (control) where control represented mock-treated cells. The induction ratio of treatment/control was then calculated by the equation 2−ΔΔ CT. Gene expression levels were normalized with the Atg5+/+ untreated sample.

Quantitative RT–PCR

Total RNAs were extracted with Trizol reagent (Invitrogen), from which 1 μg was used for reverse transcription in a 20-μL reaction system with the RNA PCR (AMV) kit (Promega). Quantitative RT-PCR was performed with SYBR-Green PCR Mastermix (Applied Biosystems), and amplification was real-time monitored on a MicorAmp optical 96-Well Reaction Plate (Applied Biosystems). The level of transcript abundance relative to reference gene (termed ΔCT) was determined according to the function ΔCT = CT (test gene) − CT (reference gene). To compare untreated and treated expression levels, the function ΔΔCT was first determined using the equation ΔΔ CT = Δ CT (treatment) − Δ CT (control) where control represented mock-treated cells. The induction ratio of treatment/control was then calculated by the equation 2−ΔΔ CT. Gene expression levels were normalized with the Atg5+/+ untreated sample.
