---
pmid: '25686250'
title: Early steps in primary cilium assembly require EHD1/EHD3-dependent ciliary
  vesicle formation.
authors:
- Lu Q
- Insinna C
- Ott C
- Stauffer J
- Pintado PA
- Rahajeng J
- Baxa U
- Walia V
- Cuenca A
- Hwang YS
- Daar IO
- Lopes S
- Lippincott-Schwartz J
- Jackson PK
- Caplan S
- Westlake CJ
journal: Nat Cell Biol
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4344897
doi: 10.1038/ncb3109
---

# Early steps in primary cilium assembly require EHD1/EHD3-dependent ciliary vesicle formation.
**Authors:** Lu Q, Insinna C, Ott C, Stauffer J, Pintado PA, Rahajeng J, Baxa U, Walia V, Cuenca A, Hwang YS, Daar IO, Lopes S, Lippincott-Schwartz J, Jackson PK, Caplan S, Westlake CJ
**Journal:** Nat Cell Biol (2015)
**DOI:** [10.1038/ncb3109](https://doi.org/10.1038/ncb3109)
**PMC:** [PMC4344897](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4344897/)

## Abstract

1. Nat Cell Biol. 2015 Mar;17(3):228-240. doi: 10.1038/ncb3109. Epub 2015 Feb 16.

Early steps in primary cilium assembly require EHD1/EHD3-dependent ciliary 
vesicle formation.

Lu Q(#)(1), Insinna C(#)(1), Ott C(2), Stauffer J(1), Pintado PA(3), Rahajeng 
J(4), Baxa U(5), Walia V(1), Cuenca A(1), Hwang YS(1), Daar IO(1), Lopes S(5), 
Lippincott-Schwartz J(2), Jackson PK(6), Caplan S(4), Westlake CJ(1).

Author information:
(1)NCI-Frederick National Laboratory, Laboratory of Cellular and Developmental 
Signaling, Frederick, MD 21702, USA.
(2)Eunice Kennedy Shriver National Institute of Child Health and Human 
Development, NIH, Bethesda, MD.
(3)CEDOC, Faculdade de Ciências Médicas, Universidade Nova de Lisboa, 1169-056 
Lisboa, Portugal.
(4)Department of Biochemistry and Molecular Biology and the Fred and Pamela 
Buffett Cancer Center, University of Nebraska Medical Center, Omaha, Nebraska 
68198, USA.
(5)ATRF, NCI-Frederick National Laboratory, Frederick, MD.
(6)Baxter Laboratory for Stem Cell Biology, Department of Microbiology and 
Immunology, Stanford University School of Medicine, Stanford, CA 94305.
(#)Contributed equally

Erratum in
    Nat Cell Biol. 2015 Apr;17(4):531. doi: 10.1038/ncb3155.

Comment in
    Dev Cell. 2015 Mar 23;32(6):665-6. doi: 10.1016/j.devcel.2015.03.012.

Membrane association with mother centriole (M-centriole) distal appendages is 
critical for ciliogenesis initiation. How the Rab GTPase Rab11-Rab8 cascade 
functions in early ciliary membrane assembly is unknown. Here, we show that the 
membrane shaping proteins EHD1 and EHD3, in association with the Rab11-Rab8 
cascade, function in early ciliogenesis. EHD1 and EHD3 localize to preciliary 
membranes and the ciliary pocket. EHD-dependent membrane tubulation is essential 
for ciliary vesicle formation from smaller distal appendage vesicles (DAVs). 
Importantly, this step functions in M-centriole to basal body transformation and 
recruitment of transition zone proteins and IFT20. SNAP29, a SNARE membrane 
fusion regulator and EHD1-binding protein, is also required for DAV-mediated 
ciliary vesicle assembly. Interestingly, only after ciliary vesicle assembly is 
Rab8 activated for ciliary growth. Our studies uncover molecular mechanisms 
informing a previously uncharacterized ciliogenesis step, whereby EHD1 and EHD3 
reorganize the M-centriole and associated DAVs before coordinated ciliary 
membrane and axoneme growth.

DOI: 10.1038/ncb3109
PMCID: PMC4344897
PMID: 25686250 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONPrimary cilia play essential roles in signal transduction and defects in cilia formation or function cause ciliopathies1, 2. Cilia form at the distal end of the mother centriole (M-centriole) via recruitment of pre-ciliary membranes, intraflagellar transport (IFT) machinery and transition zone components to enable microtubule-based axonemal assembly. Association of pre-ciliary membranes with M-centriole distal appendages is needed for basal body formation and ciliogenesis progression3-11. The membrane trafficking regulator Rab small GTPases and in particular, the Rab11-Rab8 cascade are required for ciliary membrane formation during ciliogenesis12-17. In this cascade, Rabin8, the guanine nucleotide exchange factor for Rab8, binds to Rab11 and is delivered to the centrosome on vesicles to activate Rab8 to promote ciliary membrane assembly15.Over 50 years ago, Sergei Sorokin proposed a model whereby intracellular membranes organize at the distal end of the M-centriole prior to axoneme formation. A large ciliary vesicle (CV) assembles, reorganizes to form a sheath around the extending axoneme and later fuses with the plasma membrane10, 18. The requirement for the Rab11-Rab8 cascade in ciliogenesis provides a molecular explanation for these early ciliary assembly steps but is poorly understood. Furthermore, how ciliary membrane assembly is coordinated with other early ciliogenesis processes including establishment of the basal body, IFT recruitment, transition zone assembly and axoneme growth is largely unknown.The Eps15 homology domain (EHD)-family of proteins, composed of EHD1-4, is associated with Rab11 and Rab8 membranes and regulates endosomal membrane trafficking19. EHDs are characterized by an ATP-binding G-domain, a central coiled-coil domain, and a COOH-terminal EH domain, which interacts with asparagine-proline-phenylalanine (NPF) motif containing proteins20. EHD1 and EHD3 exhibit 87% amino acid identity, whereas EHD2 and EHD4 are <74% identical to EHD1. EHD1 and EHD3 regulate Rab11-endosome recycling compartment (ERC) trafficking and bind to Rab11-FIP2, a Rab11 effector21. Additionally, EHD1 and EHD3 bind to the Rab8 effector MICAL-L1 and affect membrane tubulo-vesicle formation and scission19, 22-25. EHD1 also associates with the membrane fusion regulator SNAP2926. Here, we investigated EHD protein involvement in ciliary membrane biogenesis. Using advanced microscopy imaging approaches, we dynamically observed the recruitment of proteins essential for early ciliogenesis processes. Moreover, depletion of EHD proteins shed light on a previously uncharacterized but required step in ciliogenesis. Our data suggest a model in which EHD1 and EHD3 coordinate critical steps at the onset of ciliogenesis.

INTRODUCTION

Primary cilia play essential roles in signal transduction and defects in cilia formation or function cause ciliopathies1, 2. Cilia form at the distal end of the mother centriole (M-centriole) via recruitment of pre-ciliary membranes, intraflagellar transport (IFT) machinery and transition zone components to enable microtubule-based axonemal assembly. Association of pre-ciliary membranes with M-centriole distal appendages is needed for basal body formation and ciliogenesis progression3-11. The membrane trafficking regulator Rab small GTPases and in particular, the Rab11-Rab8 cascade are required for ciliary membrane formation during ciliogenesis12-17. In this cascade, Rabin8, the guanine nucleotide exchange factor for Rab8, binds to Rab11 and is delivered to the centrosome on vesicles to activate Rab8 to promote ciliary membrane assembly15.

Over 50 years ago, Sergei Sorokin proposed a model whereby intracellular membranes organize at the distal end of the M-centriole prior to axoneme formation. A large ciliary vesicle (CV) assembles, reorganizes to form a sheath around the extending axoneme and later fuses with the plasma membrane10, 18. The requirement for the Rab11-Rab8 cascade in ciliogenesis provides a molecular explanation for these early ciliary assembly steps but is poorly understood. Furthermore, how ciliary membrane assembly is coordinated with other early ciliogenesis processes including establishment of the basal body, IFT recruitment, transition zone assembly and axoneme growth is largely unknown.

The Eps15 homology domain (EHD)-family of proteins, composed of EHD1-4, is associated with Rab11 and Rab8 membranes and regulates endosomal membrane trafficking19. EHDs are characterized by an ATP-binding G-domain, a central coiled-coil domain, and a COOH-terminal EH domain, which interacts with asparagine-proline-phenylalanine (NPF) motif containing proteins20. EHD1 and EHD3 exhibit 87% amino acid identity, whereas EHD2 and EHD4 are <74% identical to EHD1. EHD1 and EHD3 regulate Rab11-endosome recycling compartment (ERC) trafficking and bind to Rab11-FIP2, a Rab11 effector21. Additionally, EHD1 and EHD3 bind to the Rab8 effector MICAL-L1 and affect membrane tubulo-vesicle formation and scission19, 22-25. EHD1 also associates with the membrane fusion regulator SNAP2926. Here, we investigated EHD protein involvement in ciliary membrane biogenesis. Using advanced microscopy imaging approaches, we dynamically observed the recruitment of proteins essential for early ciliogenesis processes. Moreover, depletion of EHD proteins shed light on a previously uncharacterized but required step in ciliogenesis. Our data suggest a model in which EHD1 and EHD3 coordinate critical steps at the onset of ciliogenesis.

RESULTSEHD1 and EHD3 function in ciliogenesis and localize to the ciliary pocket membraneBecause EHD1 and EHD3 have been linked to both Rab11 and Rab8 membrane compartments19 we tested their role in ciliogenesis. SiRNA-mediated knockdown of EHD1, but not EHD2-4, impaired ciliation in hTERT-RPE (RPE) cells (Fig. 1a,b, Supplementary Fig. 1a-c). Importantly, an siRNA resistant form of GFP-EHD1 or GFP-EHD3 but not GFP, GFP-EHD2 or GFP-EHD4 rescued ciliation (Fig. 1c, Supplemental Fig. 1d-f) suggesting that EHD1 and EHD3 function in ciliogenesis. Interestingly, only GFP-EHD1 and GFP-EHD3 were detected at the proximal ciliary region (Fig. 1d, Supplementary Fig. 1d). Endogenous EHD1 was also detected in the proximal ciliary region in 30 ± 5% (± SD) of cells (n=162, pooled from 3 experiments) and did not completely overlap with ciliary Smo-tRFP or GFP-Rab8a (Supplementary Fig. 1g,h). Overexpressed GFP-EHD3 colocalizes with EHD1 at this region (Supplementary Fig. 1g). Immunoblotting of RPE cell lysates revealed that EHD1 levels were >5 times higher than EHD3 (Fig. 1e) indicating that EHD3 may be dispensable for RPE cell ciliogenesis. In IMCD3 cells, EHD1 and EHD3 had similar expression levels (Fig. 1e) and were both required for ciliogenesis (Fig. 1f,g). GFP-fused EHD1 and EHD3 were detected in the proximal ciliary region in 8 ± 2% (± SD, n=164, pooled from 2 experiments) and 8 ± 4 % (± SD, n= 152, pooled from 2 experiments) respectively (Supplementary Fig. 1i).We investigated EHD protein ciliary localization using correlative light and electron microscopy (CLEM), immuno-electron microscopy and super-resolution structured illumination microscopy (SIM) and demonstrated that EHD1 and EHD3 are primarily localized to the ciliary pocket membrane unlike Rab8 and Smoothened (Smo), largely present in the ciliary membrane (Fig. 1h,i,j, Supplementary Fig. 1j,k). Additionally, EHD1 localization did not overlap with Inversin (Supplementary Fig. 1l), a proximal intra-ciliary marker27. The localization of EHD1 and EHD3 in IMCD3 cells is consistent with lower levels of ciliary pockets reported in these cells compared to RPE cells, which could point to different ciliogenesis mechanisms28. Together our results indicate that EHD1 and EHD3 are important for ciliogenesis and localize to the ciliary pocket membrane.Requirements for Ehd1 and Ehd3 in ciliogenesis during development in zebrafishWe investigated the ciliogenesis function of EHD proteins in zebrafish embryos. Zebrafish have ehd homologs: ehd1a, ehd1b, ehd2a, ehd2b and ehd3. ehd1a and 1b (89% identity) are closely related to human EHD1 (85% and 87% respectively) and ehd3 is 93% identical to human EHD3. Combination of ehd1(a+b) MO gave synergistic effects while the triple knockdown with ehd3 MO had 70 ± 6% lethality (± SD, n>150, from 3 experiments). As was observed in mammalian cells, hEHD1 and hEHD3 rescued phenotypes observed in ehd3 and ehd1 morphants unlike hEHD4 (Fig. 2b,d,f, Supplementary Fig. 2a). Histological and electron microscopy (EM) analysis in ehd1 and ehd3 morphants revealed a failure to form photoreceptor cilia-outer segments (OS, Fig. 2b,c) suggesting that both proteins are important for photoreceptor ciliogenesis, similar to IMCD3 cell requirements. However, in other tissues, requirements for Ehd proteins varied. Ehd1, but not Ehd3, was required for kinocilia formation in otic vesicles (OV, Fig. 2d,e). Similarly, Ehd1 was essential for neuromast ciliation whereas ehd3 morphants showed a partial reduction in kinocilia (Fig. 2f,g, Supplementary Fig. 2b). Consistent with these results, Ehd1 and Ehd3 were expressed in photoreceptors and neuromasts whereas only Ehd1 but not Ehd3 was expressed in the OV during embryonic development (Fig. 2h, Supplementary Fig. 2c,d,e). In these tissues, Ehd1 and Ehd3 localized to puncta throughout the cell, with some protein detected at or close to the ciliary base. Together these results indicate that Ehd1 and Ehd3 have overlapping, albeit tissue specific, functions in ciliogenesis in zebrafish embryos.EHD1 localizes to pre-ciliary membranes but is dispensable for their centrosomal traffickingNext, we investigated EHD1 association with Rab11-dependent pre-ciliary vesicle transport of Rabin8. EHD1 co-localizes with GFP-Rabin8 vesicles following serum starvation, but not in the presence of serum (Fig. 3a). Moreover, in contrast to Rab11 requirements15, depletion of EHD1 did not affect GFP-Rabin8 centrosomal accumulation (Fig. 3b) suggesting that EHD1 is associated with pre-ciliary vesicles but is not required for their centrosomal trafficking. We performed live imaging on GFP-EHD1 and tRFP-Rab8a expressing RPE cells and determined that EHD1 and Rab8 associate with developing cilia (Fig. 3c, Supplementary Fig. 3, Movie 1). Remarkably, GFP-EHD1 was detected at a site in the cell, which later accumulated tRFP-Rab8a and extended ciliary structures suggesting that EHD1 accumulates at M-centriole-associated membranes before Rab8. Additionally, Smo-tRFP, a ciliary receptor marking early ciliary membrane structures6, localized to developing cilia about the same time as GFP-EHD1 (Fig. 3d, Supplementary Movie 2) and prior to GFP-Rab8 (Fig. 3e, Supplementary Movie 3) suggesting EHD1 and Smo occupy the same pre-ciliary vesicles. Similarly, the constitutive active SmoM2 mutant localized to the M-centriole following serum starvation and its trafficking was not affected by EHD1 or Rab8 depletion (Fig. 3f), indicating that SmoM2 M-centriole transport is independent of these proteins. Furthermore, SmoM2-GFP and EHD1 co-localize at the centrosome in the absence of Rab8 (Fig. 3g). Together these results indicate that EHD1 functions upstream of Rab8 in ciliogenesis.EHD1 is required for small distal appendage vesicles assembly into the CV, while Rab8 functions in CV extensionRPE cells use an intracellular ciliogenesis pathway with formation of the CV as a critical step5. To investigate EHD1 function in this pathway, we performed EM studies on serum starved EHD1-depleted cells (Fig 4a). Strikingly, the majority (68%) of the non-ciliated cells had small ~40-60nm diameter vesicles associated with the M-centriole distal appendages (Fig. 4a,b, Supplementary Fig 4a), that we called distal appendage vesicles (DAVs). CVs were detected in 16% of EHD1 knockdown cells. By comparison, DAVs were detected in 13% of siControl treated unstarved cells and 81% had no detectable membranes at the distal appendages suggesting that DAVs are the precursor membranes for the CV. In contrast to EHD1 depleted cells, 63% of Rab8 siRNA treated unciliated cells had CV structures while only 28% had DAVs (Fig. 4a,b, Supplementary Fig 4a). Combined with our live imaging studies, these results suggest that EHD1 and Rab8 regulate different steps in ciliogenesis, with EHD1 important for CV formation and Rab8 functioning in CV extension.SIM imaging has been used to resolve the structure of CEP164, a distal appendage protein forming a ring ~300nm in diameter at the distal end of the M-centriole29. We performed SIM on SmoM2-GFP expressing RPE cells to attempt to resolve pre-ciliary membrane assembly steps at the M-centriole distal end (Fig. 4c-e). In serum fed cells depleted of EHD1 SmoM2-GFP did not co-localize with the CEP164 ring, although SmoM2-GFP and Rab11 partially co-localized on pericentriolar vesicles (Fig. 4c). However, in serum starved EHD1 depleted cells, SmoM2-GFP vesicles were detected at the CEP164 ring and partially co-localized with Rab11 both at the distal appendages and in the pericentriolar region (Fig. 4d). From our EM studies, the SmoM2-GFP vesicles are likely DAVs. In contrast, in Rab8 depleted cells SIM showed a single large structure at the M-centriole distal end, which is likely a CV (Fig. 4e). Based on our previous work15 examining ciliogenesis initiation following serum withdrawal, we looked for DAVs similar to those observed in EHD1 depleted cells using SIM (Fig. 4 d,f). Following 3h starvation, we observed Smo-GFP at the pericentriolar region and on DAVs that co-localized with EHD1 supporting an association of EHD1 with pre-ciliary vesicles and DAVs (Fig. 4f).Because intracellular ciliogenesis has been observed in mouse photoreceptors30, we tested Ehd1 and Ehd3 ciliogenesis function in zebrafish photoreceptors. Consistent with our human cell data, in ehd1 and ehd3 double morphants DAVs were frequently observed on the M-centriole distal appendages whereas in rab8a morphants, the majority of non-ciliated M-centrioles had CVs (Fig. 4g,h, Supplementary Fig. 4b). Interestingly, Ehd proteins were detected near the centrioles prior to outer segment formation at 50hpf (Fig. 4i). Together our results in human RPE cells and zebrafish photoreceptors support a model wherein EHD1 is recruited on pre-ciliary vesicles to the M-centriole to function in CV assembly from DAVs, whereas Rab8 is associated with post-CV ciliogenesis processes.EHD1-dependent CV formation is required for recruitment of transition zone proteins and IFT20To study the relationship between DAV docking, CV assembly steps and other ciliogenesis processes we examined recruitment of transition zone (TZ) proteins and intraflagellar transport proteins (IFT). We discovered that EHD1 was required for the recruitment of TZ proteins CEP290, RPGRIP1L, TMEM67 and B9D2-GFP to the M-centriole distal region, whereas Rab8 appeared to be dispensable (Fig. 5a-d). We further confirmed that pre-ciliary membranes accumulated at the M-centriole before TZ protein recruitment by live cell imaging B9D2-GFP and Smo-tRFP (Fig. 5e), and another ciliary receptor 5-HT6-tRFP (Supplementary Fig. 5). B9D2-GFP also localized to the developing cilium prior to Rab8a (Fig. 5f). IFT20, a protein known to localize to the Golgi, centrosome and cilia, is thought to be important for CV formation6, 31. Similarly to TZ proteins, IFT20 accumulation at the M-centriole was strongly blocked in EHD1 but not Rab8 depleted cells (Fig. 5g). Furthermore, IFT20-GFP accumulated at the M-centriole after Smo-tRFP recruitment (Fig. 5h, Supplementary Movie 4), but before Rab8 localization (Fig. 5i, Supplementary Movie 5). IFT20 basal body enrichment coincided with B9D2 recruitment (Fig. 5j). Therefore, our studies suggest that EHD1-dependent CV assembly function is required prior to IFT20 and TZ proteins recruitment, while Rab8 targets to the CV following these steps.EHD1 is required for CP110 loss from the distal end of the M-centrioleBecause CP110 loss from the M-centriole is required for basal body formation and precedes axoneme growth32, we examined its relationship with EHD1-dependent CV assembly. Rab8 appears to be dispensable for CP110 loss with ~70% of cells having only daughter centriolar CP110. In contrast, in EHD1-depleted cells 80% of cells had CP110 localized to both centrioles (Fig. 6a,b). To further demonstrate requirements for DAV membranes in CP110 loss, during ciliogenesis, we investigated this process in live cells using pre-ciliary membrane markers. We observed 5-HT6-tRFP membranes accumulating at the M-centriole followed by the loss of CP110-GFP a few minutes later (Fig. 6c). Thus, these results indicate that CP110 loss from the M-centriole occurs after DAV docking to the M-centriole and prior to CV assembly.EHD1 membrane tubulation function is required for CV formationBecause EHD1-dependent CV assembly is critical for ciliogenesis initiation, we wanted to determine how EHD1 regulates DAV reorganization. We examined the ability of specific EHD1 loss-of-function mutations to rescue ciliation in RPE cells depleted of endogenous EHD1. EH domain mutants specifically affecting either NPF-substrate binding and membrane recruitment (W485A) or tubulo-vesicular (K483E) membrane functions failed to rescue compared to the wildtype siRNA resistant protein 33, 34 (Fig. 7a-c). Importantly, neither mutant protein accumulated significantly in the CP (Fig. 7d). Remarkably, using SIM we found that the GFP-K483E mutant localized to structures near the distal appendages when endogenous EHD1 was depleted following starvation (Fig. 7e). Furthermore, CP110 removal failed in K483E expressing cells (Fig. 7f). Together, these results indicate that the EHD1 K483 membrane tubulation function is critical for DAV assembly into the CV and M-centriole reorganizationEHD1 is required for the fusion of DAVs into the CVBased on our findings for EHD1 and EHD3 functioning in CV assembly from DAVs, we predicted that the SNARE membrane fusion machinery would play a critical role in early ciliogenesis. Because the SNARE protein SNAP29 directly interacts with EHD1 and EHD326, we tested its function in ciliogenesis. Consistent with the EHD-SNAP29 interaction, SNAP29 co-localized with EHD1 in the ciliary pocket and was required for ciliogenesis in RPE cells (Fig. 8a,b,c). Depletion of SNAP29 also did not affect GFP-Rabin8 pre-ciliary trafficking, nor SmoM2-GFP recruitment to the M-centriole (Fig. 8 d,e). As was observed with EHD1, SNAP29 co-localized with SmoM2-GFP on DAV-like membranes in 3h starved cells (Fig. 8f,g). Interestingly, SNAP29 failed to co-localize with SmoM2-GFP at the M-centriole distal end following EHD1 depletion (24 h starved) compared to 3h starved siControl treated cells, whereas SNAP29 was dispensable for EHD1 recruitment to early ciliary membranes (Fig. 8g). Given these findings and the requirement for EHD1 and EHD3 tubulation function in CV assembly, we theorized that EHD proteins recruit SNAP29 to DAVs where the SNARE functions in fusion of tubulated DAVs.

EHD1 and EHD3 function in ciliogenesis and localize to the ciliary pocket membraneBecause EHD1 and EHD3 have been linked to both Rab11 and Rab8 membrane compartments19 we tested their role in ciliogenesis. SiRNA-mediated knockdown of EHD1, but not EHD2-4, impaired ciliation in hTERT-RPE (RPE) cells (Fig. 1a,b, Supplementary Fig. 1a-c). Importantly, an siRNA resistant form of GFP-EHD1 or GFP-EHD3 but not GFP, GFP-EHD2 or GFP-EHD4 rescued ciliation (Fig. 1c, Supplemental Fig. 1d-f) suggesting that EHD1 and EHD3 function in ciliogenesis. Interestingly, only GFP-EHD1 and GFP-EHD3 were detected at the proximal ciliary region (Fig. 1d, Supplementary Fig. 1d). Endogenous EHD1 was also detected in the proximal ciliary region in 30 ± 5% (± SD) of cells (n=162, pooled from 3 experiments) and did not completely overlap with ciliary Smo-tRFP or GFP-Rab8a (Supplementary Fig. 1g,h). Overexpressed GFP-EHD3 colocalizes with EHD1 at this region (Supplementary Fig. 1g). Immunoblotting of RPE cell lysates revealed that EHD1 levels were >5 times higher than EHD3 (Fig. 1e) indicating that EHD3 may be dispensable for RPE cell ciliogenesis. In IMCD3 cells, EHD1 and EHD3 had similar expression levels (Fig. 1e) and were both required for ciliogenesis (Fig. 1f,g). GFP-fused EHD1 and EHD3 were detected in the proximal ciliary region in 8 ± 2% (± SD, n=164, pooled from 2 experiments) and 8 ± 4 % (± SD, n= 152, pooled from 2 experiments) respectively (Supplementary Fig. 1i).We investigated EHD protein ciliary localization using correlative light and electron microscopy (CLEM), immuno-electron microscopy and super-resolution structured illumination microscopy (SIM) and demonstrated that EHD1 and EHD3 are primarily localized to the ciliary pocket membrane unlike Rab8 and Smoothened (Smo), largely present in the ciliary membrane (Fig. 1h,i,j, Supplementary Fig. 1j,k). Additionally, EHD1 localization did not overlap with Inversin (Supplementary Fig. 1l), a proximal intra-ciliary marker27. The localization of EHD1 and EHD3 in IMCD3 cells is consistent with lower levels of ciliary pockets reported in these cells compared to RPE cells, which could point to different ciliogenesis mechanisms28. Together our results indicate that EHD1 and EHD3 are important for ciliogenesis and localize to the ciliary pocket membrane.

EHD1 and EHD3 function in ciliogenesis and localize to the ciliary pocket membrane

Because EHD1 and EHD3 have been linked to both Rab11 and Rab8 membrane compartments19 we tested their role in ciliogenesis. SiRNA-mediated knockdown of EHD1, but not EHD2-4, impaired ciliation in hTERT-RPE (RPE) cells (Fig. 1a,b, Supplementary Fig. 1a-c). Importantly, an siRNA resistant form of GFP-EHD1 or GFP-EHD3 but not GFP, GFP-EHD2 or GFP-EHD4 rescued ciliation (Fig. 1c, Supplemental Fig. 1d-f) suggesting that EHD1 and EHD3 function in ciliogenesis. Interestingly, only GFP-EHD1 and GFP-EHD3 were detected at the proximal ciliary region (Fig. 1d, Supplementary Fig. 1d). Endogenous EHD1 was also detected in the proximal ciliary region in 30 ± 5% (± SD) of cells (n=162, pooled from 3 experiments) and did not completely overlap with ciliary Smo-tRFP or GFP-Rab8a (Supplementary Fig. 1g,h). Overexpressed GFP-EHD3 colocalizes with EHD1 at this region (Supplementary Fig. 1g). Immunoblotting of RPE cell lysates revealed that EHD1 levels were >5 times higher than EHD3 (Fig. 1e) indicating that EHD3 may be dispensable for RPE cell ciliogenesis. In IMCD3 cells, EHD1 and EHD3 had similar expression levels (Fig. 1e) and were both required for ciliogenesis (Fig. 1f,g). GFP-fused EHD1 and EHD3 were detected in the proximal ciliary region in 8 ± 2% (± SD, n=164, pooled from 2 experiments) and 8 ± 4 % (± SD, n= 152, pooled from 2 experiments) respectively (Supplementary Fig. 1i).

We investigated EHD protein ciliary localization using correlative light and electron microscopy (CLEM), immuno-electron microscopy and super-resolution structured illumination microscopy (SIM) and demonstrated that EHD1 and EHD3 are primarily localized to the ciliary pocket membrane unlike Rab8 and Smoothened (Smo), largely present in the ciliary membrane (Fig. 1h,i,j, Supplementary Fig. 1j,k). Additionally, EHD1 localization did not overlap with Inversin (Supplementary Fig. 1l), a proximal intra-ciliary marker27. The localization of EHD1 and EHD3 in IMCD3 cells is consistent with lower levels of ciliary pockets reported in these cells compared to RPE cells, which could point to different ciliogenesis mechanisms28. Together our results indicate that EHD1 and EHD3 are important for ciliogenesis and localize to the ciliary pocket membrane.

Requirements for Ehd1 and Ehd3 in ciliogenesis during development in zebrafishWe investigated the ciliogenesis function of EHD proteins in zebrafish embryos. Zebrafish have ehd homologs: ehd1a, ehd1b, ehd2a, ehd2b and ehd3. ehd1a and 1b (89% identity) are closely related to human EHD1 (85% and 87% respectively) and ehd3 is 93% identical to human EHD3. Combination of ehd1(a+b) MO gave synergistic effects while the triple knockdown with ehd3 MO had 70 ± 6% lethality (± SD, n>150, from 3 experiments). As was observed in mammalian cells, hEHD1 and hEHD3 rescued phenotypes observed in ehd3 and ehd1 morphants unlike hEHD4 (Fig. 2b,d,f, Supplementary Fig. 2a). Histological and electron microscopy (EM) analysis in ehd1 and ehd3 morphants revealed a failure to form photoreceptor cilia-outer segments (OS, Fig. 2b,c) suggesting that both proteins are important for photoreceptor ciliogenesis, similar to IMCD3 cell requirements. However, in other tissues, requirements for Ehd proteins varied. Ehd1, but not Ehd3, was required for kinocilia formation in otic vesicles (OV, Fig. 2d,e). Similarly, Ehd1 was essential for neuromast ciliation whereas ehd3 morphants showed a partial reduction in kinocilia (Fig. 2f,g, Supplementary Fig. 2b). Consistent with these results, Ehd1 and Ehd3 were expressed in photoreceptors and neuromasts whereas only Ehd1 but not Ehd3 was expressed in the OV during embryonic development (Fig. 2h, Supplementary Fig. 2c,d,e). In these tissues, Ehd1 and Ehd3 localized to puncta throughout the cell, with some protein detected at or close to the ciliary base. Together these results indicate that Ehd1 and Ehd3 have overlapping, albeit tissue specific, functions in ciliogenesis in zebrafish embryos.

Requirements for Ehd1 and Ehd3 in ciliogenesis during development in zebrafish

We investigated the ciliogenesis function of EHD proteins in zebrafish embryos. Zebrafish have ehd homologs: ehd1a, ehd1b, ehd2a, ehd2b and ehd3. ehd1a and 1b (89% identity) are closely related to human EHD1 (85% and 87% respectively) and ehd3 is 93% identical to human EHD3. Combination of ehd1(a+b) MO gave synergistic effects while the triple knockdown with ehd3 MO had 70 ± 6% lethality (± SD, n>150, from 3 experiments). As was observed in mammalian cells, hEHD1 and hEHD3 rescued phenotypes observed in ehd3 and ehd1 morphants unlike hEHD4 (Fig. 2b,d,f, Supplementary Fig. 2a). Histological and electron microscopy (EM) analysis in ehd1 and ehd3 morphants revealed a failure to form photoreceptor cilia-outer segments (OS, Fig. 2b,c) suggesting that both proteins are important for photoreceptor ciliogenesis, similar to IMCD3 cell requirements. However, in other tissues, requirements for Ehd proteins varied. Ehd1, but not Ehd3, was required for kinocilia formation in otic vesicles (OV, Fig. 2d,e). Similarly, Ehd1 was essential for neuromast ciliation whereas ehd3 morphants showed a partial reduction in kinocilia (Fig. 2f,g, Supplementary Fig. 2b). Consistent with these results, Ehd1 and Ehd3 were expressed in photoreceptors and neuromasts whereas only Ehd1 but not Ehd3 was expressed in the OV during embryonic development (Fig. 2h, Supplementary Fig. 2c,d,e). In these tissues, Ehd1 and Ehd3 localized to puncta throughout the cell, with some protein detected at or close to the ciliary base. Together these results indicate that Ehd1 and Ehd3 have overlapping, albeit tissue specific, functions in ciliogenesis in zebrafish embryos.

EHD1 localizes to pre-ciliary membranes but is dispensable for their centrosomal traffickingNext, we investigated EHD1 association with Rab11-dependent pre-ciliary vesicle transport of Rabin8. EHD1 co-localizes with GFP-Rabin8 vesicles following serum starvation, but not in the presence of serum (Fig. 3a). Moreover, in contrast to Rab11 requirements15, depletion of EHD1 did not affect GFP-Rabin8 centrosomal accumulation (Fig. 3b) suggesting that EHD1 is associated with pre-ciliary vesicles but is not required for their centrosomal trafficking. We performed live imaging on GFP-EHD1 and tRFP-Rab8a expressing RPE cells and determined that EHD1 and Rab8 associate with developing cilia (Fig. 3c, Supplementary Fig. 3, Movie 1). Remarkably, GFP-EHD1 was detected at a site in the cell, which later accumulated tRFP-Rab8a and extended ciliary structures suggesting that EHD1 accumulates at M-centriole-associated membranes before Rab8. Additionally, Smo-tRFP, a ciliary receptor marking early ciliary membrane structures6, localized to developing cilia about the same time as GFP-EHD1 (Fig. 3d, Supplementary Movie 2) and prior to GFP-Rab8 (Fig. 3e, Supplementary Movie 3) suggesting EHD1 and Smo occupy the same pre-ciliary vesicles. Similarly, the constitutive active SmoM2 mutant localized to the M-centriole following serum starvation and its trafficking was not affected by EHD1 or Rab8 depletion (Fig. 3f), indicating that SmoM2 M-centriole transport is independent of these proteins. Furthermore, SmoM2-GFP and EHD1 co-localize at the centrosome in the absence of Rab8 (Fig. 3g). Together these results indicate that EHD1 functions upstream of Rab8 in ciliogenesis.

EHD1 localizes to pre-ciliary membranes but is dispensable for their centrosomal trafficking

Next, we investigated EHD1 association with Rab11-dependent pre-ciliary vesicle transport of Rabin8. EHD1 co-localizes with GFP-Rabin8 vesicles following serum starvation, but not in the presence of serum (Fig. 3a). Moreover, in contrast to Rab11 requirements15, depletion of EHD1 did not affect GFP-Rabin8 centrosomal accumulation (Fig. 3b) suggesting that EHD1 is associated with pre-ciliary vesicles but is not required for their centrosomal trafficking. We performed live imaging on GFP-EHD1 and tRFP-Rab8a expressing RPE cells and determined that EHD1 and Rab8 associate with developing cilia (Fig. 3c, Supplementary Fig. 3, Movie 1). Remarkably, GFP-EHD1 was detected at a site in the cell, which later accumulated tRFP-Rab8a and extended ciliary structures suggesting that EHD1 accumulates at M-centriole-associated membranes before Rab8. Additionally, Smo-tRFP, a ciliary receptor marking early ciliary membrane structures6, localized to developing cilia about the same time as GFP-EHD1 (Fig. 3d, Supplementary Movie 2) and prior to GFP-Rab8 (Fig. 3e, Supplementary Movie 3) suggesting EHD1 and Smo occupy the same pre-ciliary vesicles. Similarly, the constitutive active SmoM2 mutant localized to the M-centriole following serum starvation and its trafficking was not affected by EHD1 or Rab8 depletion (Fig. 3f), indicating that SmoM2 M-centriole transport is independent of these proteins. Furthermore, SmoM2-GFP and EHD1 co-localize at the centrosome in the absence of Rab8 (Fig. 3g). Together these results indicate that EHD1 functions upstream of Rab8 in ciliogenesis.

EHD1 is required for small distal appendage vesicles assembly into the CV, while Rab8 functions in CV extensionRPE cells use an intracellular ciliogenesis pathway with formation of the CV as a critical step5. To investigate EHD1 function in this pathway, we performed EM studies on serum starved EHD1-depleted cells (Fig 4a). Strikingly, the majority (68%) of the non-ciliated cells had small ~40-60nm diameter vesicles associated with the M-centriole distal appendages (Fig. 4a,b, Supplementary Fig 4a), that we called distal appendage vesicles (DAVs). CVs were detected in 16% of EHD1 knockdown cells. By comparison, DAVs were detected in 13% of siControl treated unstarved cells and 81% had no detectable membranes at the distal appendages suggesting that DAVs are the precursor membranes for the CV. In contrast to EHD1 depleted cells, 63% of Rab8 siRNA treated unciliated cells had CV structures while only 28% had DAVs (Fig. 4a,b, Supplementary Fig 4a). Combined with our live imaging studies, these results suggest that EHD1 and Rab8 regulate different steps in ciliogenesis, with EHD1 important for CV formation and Rab8 functioning in CV extension.SIM imaging has been used to resolve the structure of CEP164, a distal appendage protein forming a ring ~300nm in diameter at the distal end of the M-centriole29. We performed SIM on SmoM2-GFP expressing RPE cells to attempt to resolve pre-ciliary membrane assembly steps at the M-centriole distal end (Fig. 4c-e). In serum fed cells depleted of EHD1 SmoM2-GFP did not co-localize with the CEP164 ring, although SmoM2-GFP and Rab11 partially co-localized on pericentriolar vesicles (Fig. 4c). However, in serum starved EHD1 depleted cells, SmoM2-GFP vesicles were detected at the CEP164 ring and partially co-localized with Rab11 both at the distal appendages and in the pericentriolar region (Fig. 4d). From our EM studies, the SmoM2-GFP vesicles are likely DAVs. In contrast, in Rab8 depleted cells SIM showed a single large structure at the M-centriole distal end, which is likely a CV (Fig. 4e). Based on our previous work15 examining ciliogenesis initiation following serum withdrawal, we looked for DAVs similar to those observed in EHD1 depleted cells using SIM (Fig. 4 d,f). Following 3h starvation, we observed Smo-GFP at the pericentriolar region and on DAVs that co-localized with EHD1 supporting an association of EHD1 with pre-ciliary vesicles and DAVs (Fig. 4f).Because intracellular ciliogenesis has been observed in mouse photoreceptors30, we tested Ehd1 and Ehd3 ciliogenesis function in zebrafish photoreceptors. Consistent with our human cell data, in ehd1 and ehd3 double morphants DAVs were frequently observed on the M-centriole distal appendages whereas in rab8a morphants, the majority of non-ciliated M-centrioles had CVs (Fig. 4g,h, Supplementary Fig. 4b). Interestingly, Ehd proteins were detected near the centrioles prior to outer segment formation at 50hpf (Fig. 4i). Together our results in human RPE cells and zebrafish photoreceptors support a model wherein EHD1 is recruited on pre-ciliary vesicles to the M-centriole to function in CV assembly from DAVs, whereas Rab8 is associated with post-CV ciliogenesis processes.

EHD1 is required for small distal appendage vesicles assembly into the CV, while Rab8 functions in CV extension

RPE cells use an intracellular ciliogenesis pathway with formation of the CV as a critical step5. To investigate EHD1 function in this pathway, we performed EM studies on serum starved EHD1-depleted cells (Fig 4a). Strikingly, the majority (68%) of the non-ciliated cells had small ~40-60nm diameter vesicles associated with the M-centriole distal appendages (Fig. 4a,b, Supplementary Fig 4a), that we called distal appendage vesicles (DAVs). CVs were detected in 16% of EHD1 knockdown cells. By comparison, DAVs were detected in 13% of siControl treated unstarved cells and 81% had no detectable membranes at the distal appendages suggesting that DAVs are the precursor membranes for the CV. In contrast to EHD1 depleted cells, 63% of Rab8 siRNA treated unciliated cells had CV structures while only 28% had DAVs (Fig. 4a,b, Supplementary Fig 4a). Combined with our live imaging studies, these results suggest that EHD1 and Rab8 regulate different steps in ciliogenesis, with EHD1 important for CV formation and Rab8 functioning in CV extension.

SIM imaging has been used to resolve the structure of CEP164, a distal appendage protein forming a ring ~300nm in diameter at the distal end of the M-centriole29. We performed SIM on SmoM2-GFP expressing RPE cells to attempt to resolve pre-ciliary membrane assembly steps at the M-centriole distal end (Fig. 4c-e). In serum fed cells depleted of EHD1 SmoM2-GFP did not co-localize with the CEP164 ring, although SmoM2-GFP and Rab11 partially co-localized on pericentriolar vesicles (Fig. 4c). However, in serum starved EHD1 depleted cells, SmoM2-GFP vesicles were detected at the CEP164 ring and partially co-localized with Rab11 both at the distal appendages and in the pericentriolar region (Fig. 4d). From our EM studies, the SmoM2-GFP vesicles are likely DAVs. In contrast, in Rab8 depleted cells SIM showed a single large structure at the M-centriole distal end, which is likely a CV (Fig. 4e). Based on our previous work15 examining ciliogenesis initiation following serum withdrawal, we looked for DAVs similar to those observed in EHD1 depleted cells using SIM (Fig. 4 d,f). Following 3h starvation, we observed Smo-GFP at the pericentriolar region and on DAVs that co-localized with EHD1 supporting an association of EHD1 with pre-ciliary vesicles and DAVs (Fig. 4f).

Because intracellular ciliogenesis has been observed in mouse photoreceptors30, we tested Ehd1 and Ehd3 ciliogenesis function in zebrafish photoreceptors. Consistent with our human cell data, in ehd1 and ehd3 double morphants DAVs were frequently observed on the M-centriole distal appendages whereas in rab8a morphants, the majority of non-ciliated M-centrioles had CVs (Fig. 4g,h, Supplementary Fig. 4b). Interestingly, Ehd proteins were detected near the centrioles prior to outer segment formation at 50hpf (Fig. 4i). Together our results in human RPE cells and zebrafish photoreceptors support a model wherein EHD1 is recruited on pre-ciliary vesicles to the M-centriole to function in CV assembly from DAVs, whereas Rab8 is associated with post-CV ciliogenesis processes.

EHD1-dependent CV formation is required for recruitment of transition zone proteins and IFT20To study the relationship between DAV docking, CV assembly steps and other ciliogenesis processes we examined recruitment of transition zone (TZ) proteins and intraflagellar transport proteins (IFT). We discovered that EHD1 was required for the recruitment of TZ proteins CEP290, RPGRIP1L, TMEM67 and B9D2-GFP to the M-centriole distal region, whereas Rab8 appeared to be dispensable (Fig. 5a-d). We further confirmed that pre-ciliary membranes accumulated at the M-centriole before TZ protein recruitment by live cell imaging B9D2-GFP and Smo-tRFP (Fig. 5e), and another ciliary receptor 5-HT6-tRFP (Supplementary Fig. 5). B9D2-GFP also localized to the developing cilium prior to Rab8a (Fig. 5f). IFT20, a protein known to localize to the Golgi, centrosome and cilia, is thought to be important for CV formation6, 31. Similarly to TZ proteins, IFT20 accumulation at the M-centriole was strongly blocked in EHD1 but not Rab8 depleted cells (Fig. 5g). Furthermore, IFT20-GFP accumulated at the M-centriole after Smo-tRFP recruitment (Fig. 5h, Supplementary Movie 4), but before Rab8 localization (Fig. 5i, Supplementary Movie 5). IFT20 basal body enrichment coincided with B9D2 recruitment (Fig. 5j). Therefore, our studies suggest that EHD1-dependent CV assembly function is required prior to IFT20 and TZ proteins recruitment, while Rab8 targets to the CV following these steps.

EHD1-dependent CV formation is required for recruitment of transition zone proteins and IFT20

To study the relationship between DAV docking, CV assembly steps and other ciliogenesis processes we examined recruitment of transition zone (TZ) proteins and intraflagellar transport proteins (IFT). We discovered that EHD1 was required for the recruitment of TZ proteins CEP290, RPGRIP1L, TMEM67 and B9D2-GFP to the M-centriole distal region, whereas Rab8 appeared to be dispensable (Fig. 5a-d). We further confirmed that pre-ciliary membranes accumulated at the M-centriole before TZ protein recruitment by live cell imaging B9D2-GFP and Smo-tRFP (Fig. 5e), and another ciliary receptor 5-HT6-tRFP (Supplementary Fig. 5). B9D2-GFP also localized to the developing cilium prior to Rab8a (Fig. 5f). IFT20, a protein known to localize to the Golgi, centrosome and cilia, is thought to be important for CV formation6, 31. Similarly to TZ proteins, IFT20 accumulation at the M-centriole was strongly blocked in EHD1 but not Rab8 depleted cells (Fig. 5g). Furthermore, IFT20-GFP accumulated at the M-centriole after Smo-tRFP recruitment (Fig. 5h, Supplementary Movie 4), but before Rab8 localization (Fig. 5i, Supplementary Movie 5). IFT20 basal body enrichment coincided with B9D2 recruitment (Fig. 5j). Therefore, our studies suggest that EHD1-dependent CV assembly function is required prior to IFT20 and TZ proteins recruitment, while Rab8 targets to the CV following these steps.

EHD1 is required for CP110 loss from the distal end of the M-centrioleBecause CP110 loss from the M-centriole is required for basal body formation and precedes axoneme growth32, we examined its relationship with EHD1-dependent CV assembly. Rab8 appears to be dispensable for CP110 loss with ~70% of cells having only daughter centriolar CP110. In contrast, in EHD1-depleted cells 80% of cells had CP110 localized to both centrioles (Fig. 6a,b). To further demonstrate requirements for DAV membranes in CP110 loss, during ciliogenesis, we investigated this process in live cells using pre-ciliary membrane markers. We observed 5-HT6-tRFP membranes accumulating at the M-centriole followed by the loss of CP110-GFP a few minutes later (Fig. 6c). Thus, these results indicate that CP110 loss from the M-centriole occurs after DAV docking to the M-centriole and prior to CV assembly.

EHD1 is required for CP110 loss from the distal end of the M-centriole

Because CP110 loss from the M-centriole is required for basal body formation and precedes axoneme growth32, we examined its relationship with EHD1-dependent CV assembly. Rab8 appears to be dispensable for CP110 loss with ~70% of cells having only daughter centriolar CP110. In contrast, in EHD1-depleted cells 80% of cells had CP110 localized to both centrioles (Fig. 6a,b). To further demonstrate requirements for DAV membranes in CP110 loss, during ciliogenesis, we investigated this process in live cells using pre-ciliary membrane markers. We observed 5-HT6-tRFP membranes accumulating at the M-centriole followed by the loss of CP110-GFP a few minutes later (Fig. 6c). Thus, these results indicate that CP110 loss from the M-centriole occurs after DAV docking to the M-centriole and prior to CV assembly.

EHD1 membrane tubulation function is required for CV formationBecause EHD1-dependent CV assembly is critical for ciliogenesis initiation, we wanted to determine how EHD1 regulates DAV reorganization. We examined the ability of specific EHD1 loss-of-function mutations to rescue ciliation in RPE cells depleted of endogenous EHD1. EH domain mutants specifically affecting either NPF-substrate binding and membrane recruitment (W485A) or tubulo-vesicular (K483E) membrane functions failed to rescue compared to the wildtype siRNA resistant protein 33, 34 (Fig. 7a-c). Importantly, neither mutant protein accumulated significantly in the CP (Fig. 7d). Remarkably, using SIM we found that the GFP-K483E mutant localized to structures near the distal appendages when endogenous EHD1 was depleted following starvation (Fig. 7e). Furthermore, CP110 removal failed in K483E expressing cells (Fig. 7f). Together, these results indicate that the EHD1 K483 membrane tubulation function is critical for DAV assembly into the CV and M-centriole reorganization

EHD1 membrane tubulation function is required for CV formation

Because EHD1-dependent CV assembly is critical for ciliogenesis initiation, we wanted to determine how EHD1 regulates DAV reorganization. We examined the ability of specific EHD1 loss-of-function mutations to rescue ciliation in RPE cells depleted of endogenous EHD1. EH domain mutants specifically affecting either NPF-substrate binding and membrane recruitment (W485A) or tubulo-vesicular (K483E) membrane functions failed to rescue compared to the wildtype siRNA resistant protein 33, 34 (Fig. 7a-c). Importantly, neither mutant protein accumulated significantly in the CP (Fig. 7d). Remarkably, using SIM we found that the GFP-K483E mutant localized to structures near the distal appendages when endogenous EHD1 was depleted following starvation (Fig. 7e). Furthermore, CP110 removal failed in K483E expressing cells (Fig. 7f). Together, these results indicate that the EHD1 K483 membrane tubulation function is critical for DAV assembly into the CV and M-centriole reorganization

EHD1 is required for the fusion of DAVs into the CVBased on our findings for EHD1 and EHD3 functioning in CV assembly from DAVs, we predicted that the SNARE membrane fusion machinery would play a critical role in early ciliogenesis. Because the SNARE protein SNAP29 directly interacts with EHD1 and EHD326, we tested its function in ciliogenesis. Consistent with the EHD-SNAP29 interaction, SNAP29 co-localized with EHD1 in the ciliary pocket and was required for ciliogenesis in RPE cells (Fig. 8a,b,c). Depletion of SNAP29 also did not affect GFP-Rabin8 pre-ciliary trafficking, nor SmoM2-GFP recruitment to the M-centriole (Fig. 8 d,e). As was observed with EHD1, SNAP29 co-localized with SmoM2-GFP on DAV-like membranes in 3h starved cells (Fig. 8f,g). Interestingly, SNAP29 failed to co-localize with SmoM2-GFP at the M-centriole distal end following EHD1 depletion (24 h starved) compared to 3h starved siControl treated cells, whereas SNAP29 was dispensable for EHD1 recruitment to early ciliary membranes (Fig. 8g). Given these findings and the requirement for EHD1 and EHD3 tubulation function in CV assembly, we theorized that EHD proteins recruit SNAP29 to DAVs where the SNARE functions in fusion of tubulated DAVs.

EHD1 is required for the fusion of DAVs into the CV

Based on our findings for EHD1 and EHD3 functioning in CV assembly from DAVs, we predicted that the SNARE membrane fusion machinery would play a critical role in early ciliogenesis. Because the SNARE protein SNAP29 directly interacts with EHD1 and EHD326, we tested its function in ciliogenesis. Consistent with the EHD-SNAP29 interaction, SNAP29 co-localized with EHD1 in the ciliary pocket and was required for ciliogenesis in RPE cells (Fig. 8a,b,c). Depletion of SNAP29 also did not affect GFP-Rabin8 pre-ciliary trafficking, nor SmoM2-GFP recruitment to the M-centriole (Fig. 8 d,e). As was observed with EHD1, SNAP29 co-localized with SmoM2-GFP on DAV-like membranes in 3h starved cells (Fig. 8f,g). Interestingly, SNAP29 failed to co-localize with SmoM2-GFP at the M-centriole distal end following EHD1 depletion (24 h starved) compared to 3h starved siControl treated cells, whereas SNAP29 was dispensable for EHD1 recruitment to early ciliary membranes (Fig. 8g). Given these findings and the requirement for EHD1 and EHD3 tubulation function in CV assembly, we theorized that EHD proteins recruit SNAP29 to DAVs where the SNARE functions in fusion of tubulated DAVs.

DISCUSSIONIn the intracellular ciliogenesis pathway, the ciliary membrane is thought to arise from a CV that is reshaped into a membrane sheath around the developing axoneme. However, the molecular details governing this process remained unresolved. Based on Rab11-Rabin8 pre-ciliary vesicle transport observed following cues to ciliate15, we reasoned that the recruitment of smaller pre-ciliary vesicles to the M-centriole would be followed by fusion into a larger CV. Here, we describe a role for EHD1 and EHD3, regulators of Rab compartments, in early primary cilium assembly. We find that EHD1 and EHD3 are indispensable for CV formation from DAVs and orchestrate other ciliogenesis events, whereas Rab8 functions downstream in CV extension. In addition, we show that the EHD interacting protein SNAP29, a SNARE membrane fusion protein, regulates CV assembly.A role for EHD1 and EHD3 in CV formationEHD1−/− mice were reported to have ocular defects which could be associated with defective photoreceptor cilia formation35, as we have observed in zebrafish. These mice also had higher rates of embryonic lethality suggesting an essential role of EHD1. Unlike EHD1-null mice, EHD3 knockout mice had no discernable pathology36. Our studies in human cells and zebrafish suggest that EHD1 and EHD3 display functional overlap in cilia formation likely due to their ability to heterodimerize37. Importantly, EHD1 and EHD3 share the same binding partners through interaction via NPF motif-EH domain binding21,26,38. Our results are also consistent with prior studies showing that EHD proteins have tissue-specific and redundant roles39.The connection between EHD1 and EHD3 and the Rab11-Rab8 cascade is supported by our findings that these proteins co-localize with both Rab proteins in ciliary-associated membrane compartments. Although our data do not exclude the possibility that endocytic recycling dysfunction might affect ciliogenesis indirectly, EHD1 co-localized with Rab11, Rabin8, and Smo on pre-ciliary vesicles and DAVs, and with Rab8 on the growing ciliary membrane. EHD1 is presumably transported to the centrosome via pre-ciliary vesicles, likely originating from the ERC15, which subsequently associate with distal appendages and fuse to form the CV. In non-ciliating cells, an interaction between Rab11-ERC at the M-centriole was previously described40. EHD1 and EHD3 function in tubular membrane regulation of the ERC, although in HeLa cells EHD1 appears to function as a vesiculator of tubular recycling endosomes24. Our finding that GFP-EHD1 K483E fails to rescue ciliation and yet still localized to DAVs suggests that tubulo-vesicular functioning of EHD proteins is important for CV formation. Thus we hypothesize that EHD1- and EHD3-membrane remodeling function is responsible for bringing DAV membranes in close proximity to promote SNAP29-dependent fusion during CV assembly. Our discovery that SNAP29 localizes to the ciliary pocket and functions in ciliogenesis in RPE cells is also interesting as SNAP29 mutations are responsible for CEDNIK syndrome and SNAP29 knockout mice are embryonic lethal41, 42. CEDNIK patients do not appear to have classic ciliopathy, but this is something that cannot be ruled out at this time. In addition to SNAP29 other SNAREs would be expected to function in membrane fusion during the CV assembly stage. Given the observed importance of intracellular membrane assembly during ciliogenesis, investigation of SNARE protein function in CV assembly and ciliary progression merits further investigation.Pre-ciliary membrane reorganization and ciliogenesis initiationFailure to dock membranes to the distal appendages prevents CP110 loss from the M-centriole needed for basal body formation and ciliogenesis progression5,6,8,11,43. Our work further clarifies the molecular mechanism of this critical ciliogenesis initiation step by demonstrating that M-centriole modifications are initiated following docking of DAVs to the M-centriole. To our knowledge, EHD1 and EHD3 are the first direct membrane-associated proteins whose function affects M-centriole reorganization. How EHD1-dependent DAV assembly into the CV affects CP110 loss is not clear, but there are provocative possible mechanisms. CP110 loss from the M-centriole is associated with ubiquitination44. One possibility is that pre-ciliary membranes and/or EHD proteins recruit components of the ubiquitination machinery to the M-centriole. Alternatively, changes in the membrane accompanying DAV to CV formation could help establish the basal body.The orchestration of DAV reorganization by EHD proteins is also a prerequisite step for TZ proteins and IFT20 recruitment, which is consistent with other reports showing that without membrane association at M-centriole, TZ and IFT components fail to accumulate5,6,45,46. Likewise, Rab8 recruitment to the developing cilia appears to be different from IFT20 and TZ. Importantly, our work suggests that a mechanism may be present to control the timing of Rab8 localization to the CV. Could this be a checkpoint to ensure proper membrane association with the growing axoneme? Hence, only following IFT20 and TZ proteins recruitment does coordinated axoneme and ciliary membrane growth occur (Fig. 8h).EHD1 membrane shaping and the ciliary pocketA surprising finding was the localization of EHD1 and SNAP29 to the ciliary pocket membrane. The ciliary pocket is a site for TGF-β signaling47 and its proximity to the ciliary membrane suggests it is important for ciliary trafficking28. The ciliary pocket membrane resembles a tubular structure surrounding the proximal part of the cilium that much like a vesicle or tubule has a positive curvature membrane facing the cytosol, a preferential site for EHD proteins binding. In contrast, the intracellular face of the ciliary membrane has a negative curvature. Our work suggests that EHD1 and EHD3 are excluded from the ciliary membrane, possibly by the TZ established prior to CV reshaping and axoneme growth. We can speculate that TZ gating of EHD proteins is important for establishment of negative curvature after the CV stage to allow for close association with the axoneme and the developing ciliary membrane. In mature cilia, EHD proteins could also be important for ciliary pocket maintenance. Finally, whether EHD1 and EHD3 are important for signaling associated with the ciliary pocket or the cilium remains an open question.

In the intracellular ciliogenesis pathway, the ciliary membrane is thought to arise from a CV that is reshaped into a membrane sheath around the developing axoneme. However, the molecular details governing this process remained unresolved. Based on Rab11-Rabin8 pre-ciliary vesicle transport observed following cues to ciliate15, we reasoned that the recruitment of smaller pre-ciliary vesicles to the M-centriole would be followed by fusion into a larger CV. Here, we describe a role for EHD1 and EHD3, regulators of Rab compartments, in early primary cilium assembly. We find that EHD1 and EHD3 are indispensable for CV formation from DAVs and orchestrate other ciliogenesis events, whereas Rab8 functions downstream in CV extension. In addition, we show that the EHD interacting protein SNAP29, a SNARE membrane fusion protein, regulates CV assembly.

A role for EHD1 and EHD3 in CV formationEHD1−/− mice were reported to have ocular defects which could be associated with defective photoreceptor cilia formation35, as we have observed in zebrafish. These mice also had higher rates of embryonic lethality suggesting an essential role of EHD1. Unlike EHD1-null mice, EHD3 knockout mice had no discernable pathology36. Our studies in human cells and zebrafish suggest that EHD1 and EHD3 display functional overlap in cilia formation likely due to their ability to heterodimerize37. Importantly, EHD1 and EHD3 share the same binding partners through interaction via NPF motif-EH domain binding21,26,38. Our results are also consistent with prior studies showing that EHD proteins have tissue-specific and redundant roles39.The connection between EHD1 and EHD3 and the Rab11-Rab8 cascade is supported by our findings that these proteins co-localize with both Rab proteins in ciliary-associated membrane compartments. Although our data do not exclude the possibility that endocytic recycling dysfunction might affect ciliogenesis indirectly, EHD1 co-localized with Rab11, Rabin8, and Smo on pre-ciliary vesicles and DAVs, and with Rab8 on the growing ciliary membrane. EHD1 is presumably transported to the centrosome via pre-ciliary vesicles, likely originating from the ERC15, which subsequently associate with distal appendages and fuse to form the CV. In non-ciliating cells, an interaction between Rab11-ERC at the M-centriole was previously described40. EHD1 and EHD3 function in tubular membrane regulation of the ERC, although in HeLa cells EHD1 appears to function as a vesiculator of tubular recycling endosomes24. Our finding that GFP-EHD1 K483E fails to rescue ciliation and yet still localized to DAVs suggests that tubulo-vesicular functioning of EHD proteins is important for CV formation. Thus we hypothesize that EHD1- and EHD3-membrane remodeling function is responsible for bringing DAV membranes in close proximity to promote SNAP29-dependent fusion during CV assembly. Our discovery that SNAP29 localizes to the ciliary pocket and functions in ciliogenesis in RPE cells is also interesting as SNAP29 mutations are responsible for CEDNIK syndrome and SNAP29 knockout mice are embryonic lethal41, 42. CEDNIK patients do not appear to have classic ciliopathy, but this is something that cannot be ruled out at this time. In addition to SNAP29 other SNAREs would be expected to function in membrane fusion during the CV assembly stage. Given the observed importance of intracellular membrane assembly during ciliogenesis, investigation of SNARE protein function in CV assembly and ciliary progression merits further investigation.

A role for EHD1 and EHD3 in CV formation

EHD1−/− mice were reported to have ocular defects which could be associated with defective photoreceptor cilia formation35, as we have observed in zebrafish. These mice also had higher rates of embryonic lethality suggesting an essential role of EHD1. Unlike EHD1-null mice, EHD3 knockout mice had no discernable pathology36. Our studies in human cells and zebrafish suggest that EHD1 and EHD3 display functional overlap in cilia formation likely due to their ability to heterodimerize37. Importantly, EHD1 and EHD3 share the same binding partners through interaction via NPF motif-EH domain binding21,26,38. Our results are also consistent with prior studies showing that EHD proteins have tissue-specific and redundant roles39.

The connection between EHD1 and EHD3 and the Rab11-Rab8 cascade is supported by our findings that these proteins co-localize with both Rab proteins in ciliary-associated membrane compartments. Although our data do not exclude the possibility that endocytic recycling dysfunction might affect ciliogenesis indirectly, EHD1 co-localized with Rab11, Rabin8, and Smo on pre-ciliary vesicles and DAVs, and with Rab8 on the growing ciliary membrane. EHD1 is presumably transported to the centrosome via pre-ciliary vesicles, likely originating from the ERC15, which subsequently associate with distal appendages and fuse to form the CV. In non-ciliating cells, an interaction between Rab11-ERC at the M-centriole was previously described40. EHD1 and EHD3 function in tubular membrane regulation of the ERC, although in HeLa cells EHD1 appears to function as a vesiculator of tubular recycling endosomes24. Our finding that GFP-EHD1 K483E fails to rescue ciliation and yet still localized to DAVs suggests that tubulo-vesicular functioning of EHD proteins is important for CV formation. Thus we hypothesize that EHD1- and EHD3-membrane remodeling function is responsible for bringing DAV membranes in close proximity to promote SNAP29-dependent fusion during CV assembly. Our discovery that SNAP29 localizes to the ciliary pocket and functions in ciliogenesis in RPE cells is also interesting as SNAP29 mutations are responsible for CEDNIK syndrome and SNAP29 knockout mice are embryonic lethal41, 42. CEDNIK patients do not appear to have classic ciliopathy, but this is something that cannot be ruled out at this time. In addition to SNAP29 other SNAREs would be expected to function in membrane fusion during the CV assembly stage. Given the observed importance of intracellular membrane assembly during ciliogenesis, investigation of SNARE protein function in CV assembly and ciliary progression merits further investigation.

Pre-ciliary membrane reorganization and ciliogenesis initiationFailure to dock membranes to the distal appendages prevents CP110 loss from the M-centriole needed for basal body formation and ciliogenesis progression5,6,8,11,43. Our work further clarifies the molecular mechanism of this critical ciliogenesis initiation step by demonstrating that M-centriole modifications are initiated following docking of DAVs to the M-centriole. To our knowledge, EHD1 and EHD3 are the first direct membrane-associated proteins whose function affects M-centriole reorganization. How EHD1-dependent DAV assembly into the CV affects CP110 loss is not clear, but there are provocative possible mechanisms. CP110 loss from the M-centriole is associated with ubiquitination44. One possibility is that pre-ciliary membranes and/or EHD proteins recruit components of the ubiquitination machinery to the M-centriole. Alternatively, changes in the membrane accompanying DAV to CV formation could help establish the basal body.The orchestration of DAV reorganization by EHD proteins is also a prerequisite step for TZ proteins and IFT20 recruitment, which is consistent with other reports showing that without membrane association at M-centriole, TZ and IFT components fail to accumulate5,6,45,46. Likewise, Rab8 recruitment to the developing cilia appears to be different from IFT20 and TZ. Importantly, our work suggests that a mechanism may be present to control the timing of Rab8 localization to the CV. Could this be a checkpoint to ensure proper membrane association with the growing axoneme? Hence, only following IFT20 and TZ proteins recruitment does coordinated axoneme and ciliary membrane growth occur (Fig. 8h).

Pre-ciliary membrane reorganization and ciliogenesis initiation

Failure to dock membranes to the distal appendages prevents CP110 loss from the M-centriole needed for basal body formation and ciliogenesis progression5,6,8,11,43. Our work further clarifies the molecular mechanism of this critical ciliogenesis initiation step by demonstrating that M-centriole modifications are initiated following docking of DAVs to the M-centriole. To our knowledge, EHD1 and EHD3 are the first direct membrane-associated proteins whose function affects M-centriole reorganization. How EHD1-dependent DAV assembly into the CV affects CP110 loss is not clear, but there are provocative possible mechanisms. CP110 loss from the M-centriole is associated with ubiquitination44. One possibility is that pre-ciliary membranes and/or EHD proteins recruit components of the ubiquitination machinery to the M-centriole. Alternatively, changes in the membrane accompanying DAV to CV formation could help establish the basal body.

The orchestration of DAV reorganization by EHD proteins is also a prerequisite step for TZ proteins and IFT20 recruitment, which is consistent with other reports showing that without membrane association at M-centriole, TZ and IFT components fail to accumulate5,6,45,46. Likewise, Rab8 recruitment to the developing cilia appears to be different from IFT20 and TZ. Importantly, our work suggests that a mechanism may be present to control the timing of Rab8 localization to the CV. Could this be a checkpoint to ensure proper membrane association with the growing axoneme? Hence, only following IFT20 and TZ proteins recruitment does coordinated axoneme and ciliary membrane growth occur (Fig. 8h).

EHD1 membrane shaping and the ciliary pocketA surprising finding was the localization of EHD1 and SNAP29 to the ciliary pocket membrane. The ciliary pocket is a site for TGF-β signaling47 and its proximity to the ciliary membrane suggests it is important for ciliary trafficking28. The ciliary pocket membrane resembles a tubular structure surrounding the proximal part of the cilium that much like a vesicle or tubule has a positive curvature membrane facing the cytosol, a preferential site for EHD proteins binding. In contrast, the intracellular face of the ciliary membrane has a negative curvature. Our work suggests that EHD1 and EHD3 are excluded from the ciliary membrane, possibly by the TZ established prior to CV reshaping and axoneme growth. We can speculate that TZ gating of EHD proteins is important for establishment of negative curvature after the CV stage to allow for close association with the axoneme and the developing ciliary membrane. In mature cilia, EHD proteins could also be important for ciliary pocket maintenance. Finally, whether EHD1 and EHD3 are important for signaling associated with the ciliary pocket or the cilium remains an open question.

EHD1 membrane shaping and the ciliary pocket

A surprising finding was the localization of EHD1 and SNAP29 to the ciliary pocket membrane. The ciliary pocket is a site for TGF-β signaling47 and its proximity to the ciliary membrane suggests it is important for ciliary trafficking28. The ciliary pocket membrane resembles a tubular structure surrounding the proximal part of the cilium that much like a vesicle or tubule has a positive curvature membrane facing the cytosol, a preferential site for EHD proteins binding. In contrast, the intracellular face of the ciliary membrane has a negative curvature. Our work suggests that EHD1 and EHD3 are excluded from the ciliary membrane, possibly by the TZ established prior to CV reshaping and axoneme growth. We can speculate that TZ gating of EHD proteins is important for establishment of negative curvature after the CV stage to allow for close association with the axoneme and the developing ciliary membrane. In mature cilia, EHD proteins could also be important for ciliary pocket maintenance. Finally, whether EHD1 and EHD3 are important for signaling associated with the ciliary pocket or the cilium remains an open question.

METHODSAntibodies and ReagentsCommercial antibodies used were mouse anti-γ-tubulin (clone GTU-88, 1/5000, Sigma), anti-acetylated α-tubulin (Actub) (clone 6-11B-1, 1/10000, Sigma), anti-rhodopsin (clone 4D2, 1/100), anti-β–actin (clone AC-15, 1/30000, Sigma) and anti-β-tubulin (clone 2-28-33, 1/2000, Sigma), anti-EHD3 (1/1000, Abnova for IF and 1/100, Abcam, 1/500, Proteintech for WB), rabbit anti-pericentrin (1/5000, Novus Biologicals, 1/5000), anti-EHD2 (1/1000, Abcam), anti-EHD4 (1/1000, Abcam), anti-GFP (1/50, Abcam), anti-HA HRP (1:1000, Roche), anti-CEP290 (1/200, Bethyl), anti-RPGRIP1L (1/200, Proteintech), anti-IFT20 (1/200, Proteintech), anti-TMEM67 (1/200, Proteintech), anti-SNAP29 (1/1000 for WB Proteintech and 1/200 for IF from Dr. Andrew Peden) and goat anti-CEP164 (1/500, Santa Cruz). Rabbit anti-Rab11a (1/300) was a gift from Dr. Jim Goldenring and rabbit anti-CP110 (1/500) was a gift from Dr. Monica Diaz. Polyclonal rabbit anti-EHD1 (1/1000) has been previously described33. Fluorescent secondary antibodies and phalloidin (1/50) were from Life Technologies (Molecular probes, Eugene, OR). Nuclei were labeled with Hoechst (Molecular probes, Eugene, OR).Cell lines, Plasmids, and RNAiEHD1 (MGC 8143828; Invitrogen) and EHD3 (RC214168: Origene) cDNAs were purchased from Invitrogen. EHD4 (BC006287) was purchased from Open Biosystems. EHD1, -3, -4 was PCR amplified and cloned into pDONR221 plasmid using BP clonase (Invitrogen). pENTR-EHD2 (IOH13420) was purchased from Invitrogen. The siRNA resistant (Res) EHD1 and EHD1 K483E, W485A mutants were made by QuikChange mutagenesis kit (Agilent Technologies). EHD1-4, ResEHD1 constructs and CP110 (BC036654, purchased from DNASU) cDNAs were cloned into pGLAP1 destination vectors15 (LAP referring to the GFP-Tev-Stag49) using the Invitrogen Gateway cloning system. Mouse Snap29 (BC030066) constructs were purchased from DNASU and cloned into pGLAP1. EHD1-4 were also subcloned into pCS2+ vectors. EHD1, ResEHD1 and EHD1 mutants were Gateway cloned into pHUSH-LAP, alentivirus expression vector50 reengineered by PCR to have the gLAP1 vector CMV promoter, TetO2 and LAP-tag upstream of the Gateway forward acceptor site for in-frame Tet-inducible LAP-fusion construct expression.GFP-Rab8a, tRPF-Rab8a and tRPF-Rabin8 constructs were previously described15. Mouse IFT20 (NM_018854) was synthesized with flanking Gateway compatible attB sites (DNA2.0). Human Smo (BC009989) and the Smo-W535L (SmoM2) cDNAs51, B9D2 (BC004157, purchased from DNASU), and mIFT20 were cloned into pDONR221 and subcloned into pGLAP752. Smo, 5HT-615 and mIFT20 were cloned into pDEST-tRFP7, a pGLAP7 vector reengineered to replace the LAP tag with Tag-RFP (tRFP, Evrogen). The mouse DsRed-Inversin construct was a gift from Dr Tim Stearns.GFP-Rab8 and GFP-Rabin8 RPE cell lines have been previously described15. FuGENE 6 (Promega, Madison,WI) was used for plasmid DNA transfections into cells. RPE-FRT cell lines stably expressing SmoWT, SmoM2 and B9D2 were made using the FLP-IN system (Invitrogen). GFP-EHD1, siRNA resistant GFP-EHD1, GFP-EHD1-K483E and GFP-EHD1-W485A RPE cell lines were made using the pHUSH-LAP lentivirus expression system. Lentiviral infections were performed as described53. RPE cells infected with lentivirus were selected with 4ug/ml of puromycin. siRNA duplexes were purchased from Dharmacon (Table Supplementary 1). Rab11a and Rab11b siRNA have been described15. Cells were transfected with siRNA duplexes (50 nM) using RNAiMAX (Invitrogen).Immunofluorescence and time-lapse microscopyFor RPE and IMCD3 ciliation quantification, cells were serum-starved for 24h, unless otherwise stated, fixed and processed for immuno-staining and indirect immunofluorescence as described15. Briefly, cells were fixed in 4% paraformaldehyde for 10 min or cold methanol for 5 min and blocked with 1% BSA in PBS 0.1% TX100 or BSA solution alone, respectively for 10 min followed by incubation with primary antibody in blocking solution for 1h at RT or overnight at 4°C. Fluorophore conjugated secondary antibodies were incubated for 1h at RT. Pre-ciliary membranes, cilia, transition zone proteins, IFT20 and CP110 were imaged in > six fields or as a montage of 4-6 images from 2 or more areas using a 40× 1.4 NA or 63× 1.3 NA objective, unless otherwise indicated. Imaging was performed using a Zeiss Axio Scan.Z1 inverted epifluorescence microscope equipped with a Coolsnap HQ2 camera. Images were analyzed using the Slidebook software. Immuno-cytochemistry on cryosections of 3dpf zebrafish retinae was performed as previously described54. For whole-mount zebrafish cilia studies, embryos were fixed in Dent's fixative overnight at 4°C and processed for immuno-staining as described previously55. Briefly, immunostainings were performed using blocking solution (PBS with 0.1% TX100 and 5% goat serum) for 30 min at room temperature (RT) followed by overnight incubation in blocking solution with anti-acetylated α-tubulin antibody and phalloidin for wholemounts or anti-Ehd1, anti-Ehd3 and anti-rhodopsin or anti-γ-tubulin 2h at RT. Fluorophore-conjugated antibodies were incubated in blocking solution for 2h at RT for wholemounts and 30 min at RT for cyrosections. Note that in our zebrafish ciliation assays, morphants were selected for small eye defects whereas embryos from rescue experiments were chosen randomly. Organs with absent or strongly reduced cilia numbers were counted and this data was used to calculate the percentage of normal versus total number of organs observed. Imaging was performed using a 40× 1.4 NA or 63× 1.6 NA oil objective and a Marianas spinning disk confocal (SDC) microscope (3I, Denver)Time-lapse imaging of RFP- and GFP-fusions was performed using the Marianas SDC. The environmental chamber containing the XYZ automatic stage (ASI) was set at 37°C and 5% CO2. For SDC time-lapse experiments, image acquisition of z-stacks (10×00B5;m z-stack with 500nm step size) was performed every 5-20 minutes using an Evolve 512 EMCCD camera and 40× 1.4 NA (325 nm/pixel) or 63× 1.3 NA (206 nm/pixel) oil objective and Slidebook software. For all zebrafish studies, image acquisition (35 μm z-stacks with 1 μm step size) was performed using a 40×1.4 NA or 63× 1.3 NA oil objective lens. Processing of images and fluorescence intensity measurements were carried out using either ImageJ or Slidebook software. GFP-Rabin8 vesicle centrosome localization was detected by time-lapse epifluorescence microscopy and analyzed as described15 using a HQ2 digital camera (Photometrics).Structural illumination microscopy (SIM)Cells were grown on #1.5 cover glasses (Zeiss). After immunofluorescence processing, cover glasses were mounted onto glass slides with Vectashield (Vector) and sealed with nail polish. 5 phases and 3 rotations of 3D SIM images were captured using a Zeiss or Nikon N-SIM microscopes. Channels were aligned using parameters obtained from calibration measurements with 100 nm TetraSpeck beads. Zeiss SIM images were taken with a 63× 1.4 NA oil objective and a PCO edge sCMOS camera. Nikon N-SIM images were taken with an Apo TIRF 100X/1.49 oil objective and an EMCCD camera (Andor DU-897). The pixel size of both cameras is 16 μm. The raw data pixel size is 80 nm with the Zeiss SIM using a 63X objective and 60 nm with Nikon SIM using 100X objective. Raw images captured by the Nikon SIM microscope in Fig. 1i, 1j, 4f, 7e, 8a and 8f are 14 bit and raw images captured by the Zeiss SIM microscope in Fig. 4c-e, Supplementary Fig. 1k,l are 16 bit. Laser power and exposure time were optimized to use a large portion of the camera's dynamic range while minimizing bleaching. To avoid reconstruction artifacts we have done the following: 1) during the sample preparation antibody dilutions, coverslip thickness and mounting media conditions were optimized to give the best signal to noise ratio, 2) samples were checked by wide-field or confocal microscopy prior to SIM imaging to check integrity of sample, 3) SIM images were acquired using the largest possible portion of the dynamic range provided by the camera, 4) images were reconstructed using the Zeiss Zen software and Nikon NIS-Elements software, with Wiener filter settings to reduce image noise. The estimated resolutions after reconstruction were: Zeiss channel 488: ~110 nm lateral and 350-400 nm axial; Zeiss channel 561: ~130 nm lateral and 350-400 nm axial; Zeiss channel 640: ~200 nm lateral and 350-400 nm axial; Nikon channel 488: ~130 nm lateral and ~400 nm axial; Nikon channel 561: ~140 nm lateral and ~400 nm axial; Nikon channel 640: ~160 nm lateral and ~400 nm axial. Histograms of reconstructed images were adjusted in the Nikon NIS-Elements or Zeiss Zen software and tiffs cropped in ImageJ. Intensity profile plots were created in the NIS-elements or the Zen software by drawing lines across the structure on the reconstructed images (non-histogram adjusted) and by exporting the values into Excel and GraphPad Prism 6 for Macintosh OS.Transmission Electron Microscopy and HistologyElectron microscopy processing in RPE cells and zebrafish embryos were carried out as previously described54, 56. Briefly, cells or zebrafish embryos were fixed in a solution of 2% glutaraldehyde with or without 4% paraformaldehyde in 0.1 M sodium cacodylate buffer followed by post-fixation with 1% Osmium tetroxide and 1% Uranyl acetate. After dehydration in graded ethanol, cells were embedded in EMbed 812 (Electron Microscopy Sciences) and 80 nm sections were cut using a Leica EM UC7 microtome. Electron micrographs were acquired using a transmission electron microscope (Hitachi 7650 TEM). For photoreceptor cilia counting from histological analysis, semi-thin sections (250nm) cut from samples embedded for TEM analysis were stained with toluidine blue. Photoreceptor outer segments with intact inner segments were counted (>100). For immuno-EM, RPE cell line expressing GFP-EHD1 were fixed with 4% paraformaldehyde and 0.25% glutaraldehyde in 0.1 M sodium cacodylate buffer. After dehydration in graded ethanol, cells were embedded in LR White resin. Ultra-thin sections were immuno-stained with an anti-GFP antibody (1/100) for 2h at RT in blocking solution (AURION blocking solution Electron Microscopy sciences) and a secondary antibody conjugated with 10 nm gold particles for 1h RT. For correlative light and electron microscopy (CLEM), cells were plated on gridded-glass bottom dishes (MatTek). Fluorescence and DIC images of fixed cells were taken with a Zeiss fluorescence microscope. The position of cells was recorded using grid numbers on cover glasses. After imaging, cells were processed for electron microscopy. Serial ultra-thin sections of the recorded region were collected on formvar coated slot grids and low magnification TEM images were matched with fluorescence images to identify cells of interest.Morpholino knockdownFish care and husbandry were performed in compliance with NIH guideline for animal care. Zebrafish used in this study were TAB-5 and Tg(arl13b-GFP) line (a gift from Dr. Zhaoxia Sun). Knockdowns for each gene were performed using the following translation blocking morpholinos obtained from GENETOOLS, Philomath, OR: ehd1a CTGAACATGGTGGACGTTACACGAC, ehd1b ATCTTTGTTAGACCAACTGAACATT and ehd3 CATCGGTACCCAACCAGCTGAACAT.The splice-blocking morpholino oligonucleotide rab8- SP1 was previously described57 Injections were done using a microinjector PLI-90 (Harvard Apparatus, Cambridge, MA). For rescue experiments, the full-length sequences of human EHD1, EHD3 and EHD4 were cloned into PCS2+ vectors and mRNAs were transcribed using the mMESSAGE mMACHINE kit (Ambion) according to manufacturer's instructions. Embryos were co-injected with 250 μM of morpholinos and 300 pg/nl of capped mRNAs at the one-cell stage.Expression analysisTaqman gene expression probes for EHD1-4 were purchased from Applied Biosystems and qRT-PCR was performed as previously described15.Western blots on cell lysates were performed as previously described15. Briefly, RPE cells were homogenized in lysis buffer (20 mM Tris pH8, 137 mM NaCl, 10% glycerol, 1% TX100) containing protease inhibitor cocktail (PIC, Roche) and zebrafish embryos were lysed in RIPA buffer with PIC. Lysates were centrifuged for 10min at 13000rpm. Sample buffer was added to the supernatants and samples were boiled.Standard whole mount in situ hybridizations were performed as previously described58 at 48hpf and 72 hpf with both ehd1a+b and ehd3 probes. Due to the high sequence similarity between these genes, we designed probes in the 3'UTR regions to maximize the specificity. However, the ehd1 probe was predicted to recognize both ehd1a and ehd1b transcripts. Primers: ehd1 (F) primer: 5’GTTTTAACCCAAGCGCCACC 3’, ehd1 (R) primer: 5’ATCTGGAGGAATTGCGCAGC 3’ (amplified product: 394pb, specific to ehd1a+b 3’UTR), ehd3 (F) primer: 5’ ATGCTTCACTAGGAGGAATGG 3’. ehd3 (R) primer: 5’GTTGACTGTTGGGTTTTACGG 3’ (amplified product: 411bp. Specific to ehd3 3’UTR).Statistics and ReproducibilityStatistical analyses were performed as specified with GraphPad Prism 6 for Macintosh OS. All data are presented as mean ± S.E.M or S.D as specified in the figure legends, and two group comparisons were done with an unpaired two-tailed Student's t-test. A value of P<0.05 was considered as statistically significant. All P values were indicated on graphs in figures as followed: * P<0.05, **P<0.001. ***P<0.0001. Exact P values can be found in the Statistics Source Data Sheet (Supplementary Table 2).Experiments were reproducibly performed as follows: one experiment (Fig 1h, Supplementary Fig 2d), two independent experiments (Fig 1j, 2h, 3a, 4i, 5e, 5f, 7e, Supplementary Fig 1i, 1l, 2a, 2e, 5), 3 independent experiments (Fig 1a, 1d-f, 2a, 2c-e, 2g, 3f-g, 4c-g, 5d, 5g, 6a, 6c, 7b, 8a-b, 8f, Supplementary Figure 1d, 1g, 1k, 2c, 4b), two-four independent experiments (Fig 4a, Supplementary Fig 4a) and more than 3 independent experiments (Fig 1i, 3c-e, 5h-j, 7d, Supplementary Fig 1h, 1j, 3, Movie 1-5).

Antibodies and ReagentsCommercial antibodies used were mouse anti-γ-tubulin (clone GTU-88, 1/5000, Sigma), anti-acetylated α-tubulin (Actub) (clone 6-11B-1, 1/10000, Sigma), anti-rhodopsin (clone 4D2, 1/100), anti-β–actin (clone AC-15, 1/30000, Sigma) and anti-β-tubulin (clone 2-28-33, 1/2000, Sigma), anti-EHD3 (1/1000, Abnova for IF and 1/100, Abcam, 1/500, Proteintech for WB), rabbit anti-pericentrin (1/5000, Novus Biologicals, 1/5000), anti-EHD2 (1/1000, Abcam), anti-EHD4 (1/1000, Abcam), anti-GFP (1/50, Abcam), anti-HA HRP (1:1000, Roche), anti-CEP290 (1/200, Bethyl), anti-RPGRIP1L (1/200, Proteintech), anti-IFT20 (1/200, Proteintech), anti-TMEM67 (1/200, Proteintech), anti-SNAP29 (1/1000 for WB Proteintech and 1/200 for IF from Dr. Andrew Peden) and goat anti-CEP164 (1/500, Santa Cruz). Rabbit anti-Rab11a (1/300) was a gift from Dr. Jim Goldenring and rabbit anti-CP110 (1/500) was a gift from Dr. Monica Diaz. Polyclonal rabbit anti-EHD1 (1/1000) has been previously described33. Fluorescent secondary antibodies and phalloidin (1/50) were from Life Technologies (Molecular probes, Eugene, OR). Nuclei were labeled with Hoechst (Molecular probes, Eugene, OR).

Antibodies and Reagents

Commercial antibodies used were mouse anti-γ-tubulin (clone GTU-88, 1/5000, Sigma), anti-acetylated α-tubulin (Actub) (clone 6-11B-1, 1/10000, Sigma), anti-rhodopsin (clone 4D2, 1/100), anti-β–actin (clone AC-15, 1/30000, Sigma) and anti-β-tubulin (clone 2-28-33, 1/2000, Sigma), anti-EHD3 (1/1000, Abnova for IF and 1/100, Abcam, 1/500, Proteintech for WB), rabbit anti-pericentrin (1/5000, Novus Biologicals, 1/5000), anti-EHD2 (1/1000, Abcam), anti-EHD4 (1/1000, Abcam), anti-GFP (1/50, Abcam), anti-HA HRP (1:1000, Roche), anti-CEP290 (1/200, Bethyl), anti-RPGRIP1L (1/200, Proteintech), anti-IFT20 (1/200, Proteintech), anti-TMEM67 (1/200, Proteintech), anti-SNAP29 (1/1000 for WB Proteintech and 1/200 for IF from Dr. Andrew Peden) and goat anti-CEP164 (1/500, Santa Cruz). Rabbit anti-Rab11a (1/300) was a gift from Dr. Jim Goldenring and rabbit anti-CP110 (1/500) was a gift from Dr. Monica Diaz. Polyclonal rabbit anti-EHD1 (1/1000) has been previously described33. Fluorescent secondary antibodies and phalloidin (1/50) were from Life Technologies (Molecular probes, Eugene, OR). Nuclei were labeled with Hoechst (Molecular probes, Eugene, OR).

Cell lines, Plasmids, and RNAiEHD1 (MGC 8143828; Invitrogen) and EHD3 (RC214168: Origene) cDNAs were purchased from Invitrogen. EHD4 (BC006287) was purchased from Open Biosystems. EHD1, -3, -4 was PCR amplified and cloned into pDONR221 plasmid using BP clonase (Invitrogen). pENTR-EHD2 (IOH13420) was purchased from Invitrogen. The siRNA resistant (Res) EHD1 and EHD1 K483E, W485A mutants were made by QuikChange mutagenesis kit (Agilent Technologies). EHD1-4, ResEHD1 constructs and CP110 (BC036654, purchased from DNASU) cDNAs were cloned into pGLAP1 destination vectors15 (LAP referring to the GFP-Tev-Stag49) using the Invitrogen Gateway cloning system. Mouse Snap29 (BC030066) constructs were purchased from DNASU and cloned into pGLAP1. EHD1-4 were also subcloned into pCS2+ vectors. EHD1, ResEHD1 and EHD1 mutants were Gateway cloned into pHUSH-LAP, alentivirus expression vector50 reengineered by PCR to have the gLAP1 vector CMV promoter, TetO2 and LAP-tag upstream of the Gateway forward acceptor site for in-frame Tet-inducible LAP-fusion construct expression.GFP-Rab8a, tRPF-Rab8a and tRPF-Rabin8 constructs were previously described15. Mouse IFT20 (NM_018854) was synthesized with flanking Gateway compatible attB sites (DNA2.0). Human Smo (BC009989) and the Smo-W535L (SmoM2) cDNAs51, B9D2 (BC004157, purchased from DNASU), and mIFT20 were cloned into pDONR221 and subcloned into pGLAP752. Smo, 5HT-615 and mIFT20 were cloned into pDEST-tRFP7, a pGLAP7 vector reengineered to replace the LAP tag with Tag-RFP (tRFP, Evrogen). The mouse DsRed-Inversin construct was a gift from Dr Tim Stearns.GFP-Rab8 and GFP-Rabin8 RPE cell lines have been previously described15. FuGENE 6 (Promega, Madison,WI) was used for plasmid DNA transfections into cells. RPE-FRT cell lines stably expressing SmoWT, SmoM2 and B9D2 were made using the FLP-IN system (Invitrogen). GFP-EHD1, siRNA resistant GFP-EHD1, GFP-EHD1-K483E and GFP-EHD1-W485A RPE cell lines were made using the pHUSH-LAP lentivirus expression system. Lentiviral infections were performed as described53. RPE cells infected with lentivirus were selected with 4ug/ml of puromycin. siRNA duplexes were purchased from Dharmacon (Table Supplementary 1). Rab11a and Rab11b siRNA have been described15. Cells were transfected with siRNA duplexes (50 nM) using RNAiMAX (Invitrogen).

Cell lines, Plasmids, and RNAi

EHD1 (MGC 8143828; Invitrogen) and EHD3 (RC214168: Origene) cDNAs were purchased from Invitrogen. EHD4 (BC006287) was purchased from Open Biosystems. EHD1, -3, -4 was PCR amplified and cloned into pDONR221 plasmid using BP clonase (Invitrogen). pENTR-EHD2 (IOH13420) was purchased from Invitrogen. The siRNA resistant (Res) EHD1 and EHD1 K483E, W485A mutants were made by QuikChange mutagenesis kit (Agilent Technologies). EHD1-4, ResEHD1 constructs and CP110 (BC036654, purchased from DNASU) cDNAs were cloned into pGLAP1 destination vectors15 (LAP referring to the GFP-Tev-Stag49) using the Invitrogen Gateway cloning system. Mouse Snap29 (BC030066) constructs were purchased from DNASU and cloned into pGLAP1. EHD1-4 were also subcloned into pCS2+ vectors. EHD1, ResEHD1 and EHD1 mutants were Gateway cloned into pHUSH-LAP, alentivirus expression vector50 reengineered by PCR to have the gLAP1 vector CMV promoter, TetO2 and LAP-tag upstream of the Gateway forward acceptor site for in-frame Tet-inducible LAP-fusion construct expression.

GFP-Rab8a, tRPF-Rab8a and tRPF-Rabin8 constructs were previously described15. Mouse IFT20 (NM_018854) was synthesized with flanking Gateway compatible attB sites (DNA2.0). Human Smo (BC009989) and the Smo-W535L (SmoM2) cDNAs51, B9D2 (BC004157, purchased from DNASU), and mIFT20 were cloned into pDONR221 and subcloned into pGLAP752. Smo, 5HT-615 and mIFT20 were cloned into pDEST-tRFP7, a pGLAP7 vector reengineered to replace the LAP tag with Tag-RFP (tRFP, Evrogen). The mouse DsRed-Inversin construct was a gift from Dr Tim Stearns.

GFP-Rab8 and GFP-Rabin8 RPE cell lines have been previously described15. FuGENE 6 (Promega, Madison,WI) was used for plasmid DNA transfections into cells. RPE-FRT cell lines stably expressing SmoWT, SmoM2 and B9D2 were made using the FLP-IN system (Invitrogen). GFP-EHD1, siRNA resistant GFP-EHD1, GFP-EHD1-K483E and GFP-EHD1-W485A RPE cell lines were made using the pHUSH-LAP lentivirus expression system. Lentiviral infections were performed as described53. RPE cells infected with lentivirus were selected with 4ug/ml of puromycin. siRNA duplexes were purchased from Dharmacon (Table Supplementary 1). Rab11a and Rab11b siRNA have been described15. Cells were transfected with siRNA duplexes (50 nM) using RNAiMAX (Invitrogen).

Immunofluorescence and time-lapse microscopyFor RPE and IMCD3 ciliation quantification, cells were serum-starved for 24h, unless otherwise stated, fixed and processed for immuno-staining and indirect immunofluorescence as described15. Briefly, cells were fixed in 4% paraformaldehyde for 10 min or cold methanol for 5 min and blocked with 1% BSA in PBS 0.1% TX100 or BSA solution alone, respectively for 10 min followed by incubation with primary antibody in blocking solution for 1h at RT or overnight at 4°C. Fluorophore conjugated secondary antibodies were incubated for 1h at RT. Pre-ciliary membranes, cilia, transition zone proteins, IFT20 and CP110 were imaged in > six fields or as a montage of 4-6 images from 2 or more areas using a 40× 1.4 NA or 63× 1.3 NA objective, unless otherwise indicated. Imaging was performed using a Zeiss Axio Scan.Z1 inverted epifluorescence microscope equipped with a Coolsnap HQ2 camera. Images were analyzed using the Slidebook software. Immuno-cytochemistry on cryosections of 3dpf zebrafish retinae was performed as previously described54. For whole-mount zebrafish cilia studies, embryos were fixed in Dent's fixative overnight at 4°C and processed for immuno-staining as described previously55. Briefly, immunostainings were performed using blocking solution (PBS with 0.1% TX100 and 5% goat serum) for 30 min at room temperature (RT) followed by overnight incubation in blocking solution with anti-acetylated α-tubulin antibody and phalloidin for wholemounts or anti-Ehd1, anti-Ehd3 and anti-rhodopsin or anti-γ-tubulin 2h at RT. Fluorophore-conjugated antibodies were incubated in blocking solution for 2h at RT for wholemounts and 30 min at RT for cyrosections. Note that in our zebrafish ciliation assays, morphants were selected for small eye defects whereas embryos from rescue experiments were chosen randomly. Organs with absent or strongly reduced cilia numbers were counted and this data was used to calculate the percentage of normal versus total number of organs observed. Imaging was performed using a 40× 1.4 NA or 63× 1.6 NA oil objective and a Marianas spinning disk confocal (SDC) microscope (3I, Denver)Time-lapse imaging of RFP- and GFP-fusions was performed using the Marianas SDC. The environmental chamber containing the XYZ automatic stage (ASI) was set at 37°C and 5% CO2. For SDC time-lapse experiments, image acquisition of z-stacks (10×00B5;m z-stack with 500nm step size) was performed every 5-20 minutes using an Evolve 512 EMCCD camera and 40× 1.4 NA (325 nm/pixel) or 63× 1.3 NA (206 nm/pixel) oil objective and Slidebook software. For all zebrafish studies, image acquisition (35 μm z-stacks with 1 μm step size) was performed using a 40×1.4 NA or 63× 1.3 NA oil objective lens. Processing of images and fluorescence intensity measurements were carried out using either ImageJ or Slidebook software. GFP-Rabin8 vesicle centrosome localization was detected by time-lapse epifluorescence microscopy and analyzed as described15 using a HQ2 digital camera (Photometrics).

Immunofluorescence and time-lapse microscopy

For RPE and IMCD3 ciliation quantification, cells were serum-starved for 24h, unless otherwise stated, fixed and processed for immuno-staining and indirect immunofluorescence as described15. Briefly, cells were fixed in 4% paraformaldehyde for 10 min or cold methanol for 5 min and blocked with 1% BSA in PBS 0.1% TX100 or BSA solution alone, respectively for 10 min followed by incubation with primary antibody in blocking solution for 1h at RT or overnight at 4°C. Fluorophore conjugated secondary antibodies were incubated for 1h at RT. Pre-ciliary membranes, cilia, transition zone proteins, IFT20 and CP110 were imaged in > six fields or as a montage of 4-6 images from 2 or more areas using a 40× 1.4 NA or 63× 1.3 NA objective, unless otherwise indicated. Imaging was performed using a Zeiss Axio Scan.Z1 inverted epifluorescence microscope equipped with a Coolsnap HQ2 camera. Images were analyzed using the Slidebook software. Immuno-cytochemistry on cryosections of 3dpf zebrafish retinae was performed as previously described54. For whole-mount zebrafish cilia studies, embryos were fixed in Dent's fixative overnight at 4°C and processed for immuno-staining as described previously55. Briefly, immunostainings were performed using blocking solution (PBS with 0.1% TX100 and 5% goat serum) for 30 min at room temperature (RT) followed by overnight incubation in blocking solution with anti-acetylated α-tubulin antibody and phalloidin for wholemounts or anti-Ehd1, anti-Ehd3 and anti-rhodopsin or anti-γ-tubulin 2h at RT. Fluorophore-conjugated antibodies were incubated in blocking solution for 2h at RT for wholemounts and 30 min at RT for cyrosections. Note that in our zebrafish ciliation assays, morphants were selected for small eye defects whereas embryos from rescue experiments were chosen randomly. Organs with absent or strongly reduced cilia numbers were counted and this data was used to calculate the percentage of normal versus total number of organs observed. Imaging was performed using a 40× 1.4 NA or 63× 1.6 NA oil objective and a Marianas spinning disk confocal (SDC) microscope (3I, Denver)

Time-lapse imaging of RFP- and GFP-fusions was performed using the Marianas SDC. The environmental chamber containing the XYZ automatic stage (ASI) was set at 37°C and 5% CO2. For SDC time-lapse experiments, image acquisition of z-stacks (10×00B5;m z-stack with 500nm step size) was performed every 5-20 minutes using an Evolve 512 EMCCD camera and 40× 1.4 NA (325 nm/pixel) or 63× 1.3 NA (206 nm/pixel) oil objective and Slidebook software. For all zebrafish studies, image acquisition (35 μm z-stacks with 1 μm step size) was performed using a 40×1.4 NA or 63× 1.3 NA oil objective lens. Processing of images and fluorescence intensity measurements were carried out using either ImageJ or Slidebook software. GFP-Rabin8 vesicle centrosome localization was detected by time-lapse epifluorescence microscopy and analyzed as described15 using a HQ2 digital camera (Photometrics).

Structural illumination microscopy (SIM)Cells were grown on #1.5 cover glasses (Zeiss). After immunofluorescence processing, cover glasses were mounted onto glass slides with Vectashield (Vector) and sealed with nail polish. 5 phases and 3 rotations of 3D SIM images were captured using a Zeiss or Nikon N-SIM microscopes. Channels were aligned using parameters obtained from calibration measurements with 100 nm TetraSpeck beads. Zeiss SIM images were taken with a 63× 1.4 NA oil objective and a PCO edge sCMOS camera. Nikon N-SIM images were taken with an Apo TIRF 100X/1.49 oil objective and an EMCCD camera (Andor DU-897). The pixel size of both cameras is 16 μm. The raw data pixel size is 80 nm with the Zeiss SIM using a 63X objective and 60 nm with Nikon SIM using 100X objective. Raw images captured by the Nikon SIM microscope in Fig. 1i, 1j, 4f, 7e, 8a and 8f are 14 bit and raw images captured by the Zeiss SIM microscope in Fig. 4c-e, Supplementary Fig. 1k,l are 16 bit. Laser power and exposure time were optimized to use a large portion of the camera's dynamic range while minimizing bleaching. To avoid reconstruction artifacts we have done the following: 1) during the sample preparation antibody dilutions, coverslip thickness and mounting media conditions were optimized to give the best signal to noise ratio, 2) samples were checked by wide-field or confocal microscopy prior to SIM imaging to check integrity of sample, 3) SIM images were acquired using the largest possible portion of the dynamic range provided by the camera, 4) images were reconstructed using the Zeiss Zen software and Nikon NIS-Elements software, with Wiener filter settings to reduce image noise. The estimated resolutions after reconstruction were: Zeiss channel 488: ~110 nm lateral and 350-400 nm axial; Zeiss channel 561: ~130 nm lateral and 350-400 nm axial; Zeiss channel 640: ~200 nm lateral and 350-400 nm axial; Nikon channel 488: ~130 nm lateral and ~400 nm axial; Nikon channel 561: ~140 nm lateral and ~400 nm axial; Nikon channel 640: ~160 nm lateral and ~400 nm axial. Histograms of reconstructed images were adjusted in the Nikon NIS-Elements or Zeiss Zen software and tiffs cropped in ImageJ. Intensity profile plots were created in the NIS-elements or the Zen software by drawing lines across the structure on the reconstructed images (non-histogram adjusted) and by exporting the values into Excel and GraphPad Prism 6 for Macintosh OS.

Structural illumination microscopy (SIM)

Cells were grown on #1.5 cover glasses (Zeiss). After immunofluorescence processing, cover glasses were mounted onto glass slides with Vectashield (Vector) and sealed with nail polish. 5 phases and 3 rotations of 3D SIM images were captured using a Zeiss or Nikon N-SIM microscopes. Channels were aligned using parameters obtained from calibration measurements with 100 nm TetraSpeck beads. Zeiss SIM images were taken with a 63× 1.4 NA oil objective and a PCO edge sCMOS camera. Nikon N-SIM images were taken with an Apo TIRF 100X/1.49 oil objective and an EMCCD camera (Andor DU-897). The pixel size of both cameras is 16 μm. The raw data pixel size is 80 nm with the Zeiss SIM using a 63X objective and 60 nm with Nikon SIM using 100X objective. Raw images captured by the Nikon SIM microscope in Fig. 1i, 1j, 4f, 7e, 8a and 8f are 14 bit and raw images captured by the Zeiss SIM microscope in Fig. 4c-e, Supplementary Fig. 1k,l are 16 bit. Laser power and exposure time were optimized to use a large portion of the camera's dynamic range while minimizing bleaching. To avoid reconstruction artifacts we have done the following: 1) during the sample preparation antibody dilutions, coverslip thickness and mounting media conditions were optimized to give the best signal to noise ratio, 2) samples were checked by wide-field or confocal microscopy prior to SIM imaging to check integrity of sample, 3) SIM images were acquired using the largest possible portion of the dynamic range provided by the camera, 4) images were reconstructed using the Zeiss Zen software and Nikon NIS-Elements software, with Wiener filter settings to reduce image noise. The estimated resolutions after reconstruction were: Zeiss channel 488: ~110 nm lateral and 350-400 nm axial; Zeiss channel 561: ~130 nm lateral and 350-400 nm axial; Zeiss channel 640: ~200 nm lateral and 350-400 nm axial; Nikon channel 488: ~130 nm lateral and ~400 nm axial; Nikon channel 561: ~140 nm lateral and ~400 nm axial; Nikon channel 640: ~160 nm lateral and ~400 nm axial. Histograms of reconstructed images were adjusted in the Nikon NIS-Elements or Zeiss Zen software and tiffs cropped in ImageJ. Intensity profile plots were created in the NIS-elements or the Zen software by drawing lines across the structure on the reconstructed images (non-histogram adjusted) and by exporting the values into Excel and GraphPad Prism 6 for Macintosh OS.

Transmission Electron Microscopy and HistologyElectron microscopy processing in RPE cells and zebrafish embryos were carried out as previously described54, 56. Briefly, cells or zebrafish embryos were fixed in a solution of 2% glutaraldehyde with or without 4% paraformaldehyde in 0.1 M sodium cacodylate buffer followed by post-fixation with 1% Osmium tetroxide and 1% Uranyl acetate. After dehydration in graded ethanol, cells were embedded in EMbed 812 (Electron Microscopy Sciences) and 80 nm sections were cut using a Leica EM UC7 microtome. Electron micrographs were acquired using a transmission electron microscope (Hitachi 7650 TEM). For photoreceptor cilia counting from histological analysis, semi-thin sections (250nm) cut from samples embedded for TEM analysis were stained with toluidine blue. Photoreceptor outer segments with intact inner segments were counted (>100). For immuno-EM, RPE cell line expressing GFP-EHD1 were fixed with 4% paraformaldehyde and 0.25% glutaraldehyde in 0.1 M sodium cacodylate buffer. After dehydration in graded ethanol, cells were embedded in LR White resin. Ultra-thin sections were immuno-stained with an anti-GFP antibody (1/100) for 2h at RT in blocking solution (AURION blocking solution Electron Microscopy sciences) and a secondary antibody conjugated with 10 nm gold particles for 1h RT. For correlative light and electron microscopy (CLEM), cells were plated on gridded-glass bottom dishes (MatTek). Fluorescence and DIC images of fixed cells were taken with a Zeiss fluorescence microscope. The position of cells was recorded using grid numbers on cover glasses. After imaging, cells were processed for electron microscopy. Serial ultra-thin sections of the recorded region were collected on formvar coated slot grids and low magnification TEM images were matched with fluorescence images to identify cells of interest.

Transmission Electron Microscopy and Histology

Electron microscopy processing in RPE cells and zebrafish embryos were carried out as previously described54, 56. Briefly, cells or zebrafish embryos were fixed in a solution of 2% glutaraldehyde with or without 4% paraformaldehyde in 0.1 M sodium cacodylate buffer followed by post-fixation with 1% Osmium tetroxide and 1% Uranyl acetate. After dehydration in graded ethanol, cells were embedded in EMbed 812 (Electron Microscopy Sciences) and 80 nm sections were cut using a Leica EM UC7 microtome. Electron micrographs were acquired using a transmission electron microscope (Hitachi 7650 TEM). For photoreceptor cilia counting from histological analysis, semi-thin sections (250nm) cut from samples embedded for TEM analysis were stained with toluidine blue. Photoreceptor outer segments with intact inner segments were counted (>100). For immuno-EM, RPE cell line expressing GFP-EHD1 were fixed with 4% paraformaldehyde and 0.25% glutaraldehyde in 0.1 M sodium cacodylate buffer. After dehydration in graded ethanol, cells were embedded in LR White resin. Ultra-thin sections were immuno-stained with an anti-GFP antibody (1/100) for 2h at RT in blocking solution (AURION blocking solution Electron Microscopy sciences) and a secondary antibody conjugated with 10 nm gold particles for 1h RT. For correlative light and electron microscopy (CLEM), cells were plated on gridded-glass bottom dishes (MatTek). Fluorescence and DIC images of fixed cells were taken with a Zeiss fluorescence microscope. The position of cells was recorded using grid numbers on cover glasses. After imaging, cells were processed for electron microscopy. Serial ultra-thin sections of the recorded region were collected on formvar coated slot grids and low magnification TEM images were matched with fluorescence images to identify cells of interest.

Morpholino knockdownFish care and husbandry were performed in compliance with NIH guideline for animal care. Zebrafish used in this study were TAB-5 and Tg(arl13b-GFP) line (a gift from Dr. Zhaoxia Sun). Knockdowns for each gene were performed using the following translation blocking morpholinos obtained from GENETOOLS, Philomath, OR: ehd1a CTGAACATGGTGGACGTTACACGAC, ehd1b ATCTTTGTTAGACCAACTGAACATT and ehd3 CATCGGTACCCAACCAGCTGAACAT.The splice-blocking morpholino oligonucleotide rab8- SP1 was previously described57 Injections were done using a microinjector PLI-90 (Harvard Apparatus, Cambridge, MA). For rescue experiments, the full-length sequences of human EHD1, EHD3 and EHD4 were cloned into PCS2+ vectors and mRNAs were transcribed using the mMESSAGE mMACHINE kit (Ambion) according to manufacturer's instructions. Embryos were co-injected with 250 μM of morpholinos and 300 pg/nl of capped mRNAs at the one-cell stage.

Morpholino knockdown

Fish care and husbandry were performed in compliance with NIH guideline for animal care. Zebrafish used in this study were TAB-5 and Tg(arl13b-GFP) line (a gift from Dr. Zhaoxia Sun). Knockdowns for each gene were performed using the following translation blocking morpholinos obtained from GENETOOLS, Philomath, OR: ehd1a CTGAACATGGTGGACGTTACACGAC, ehd1b ATCTTTGTTAGACCAACTGAACATT and ehd3 CATCGGTACCCAACCAGCTGAACAT.

The splice-blocking morpholino oligonucleotide rab8- SP1 was previously described57 Injections were done using a microinjector PLI-90 (Harvard Apparatus, Cambridge, MA). For rescue experiments, the full-length sequences of human EHD1, EHD3 and EHD4 were cloned into PCS2+ vectors and mRNAs were transcribed using the mMESSAGE mMACHINE kit (Ambion) according to manufacturer's instructions. Embryos were co-injected with 250 μM of morpholinos and 300 pg/nl of capped mRNAs at the one-cell stage.

Expression analysisTaqman gene expression probes for EHD1-4 were purchased from Applied Biosystems and qRT-PCR was performed as previously described15.Western blots on cell lysates were performed as previously described15. Briefly, RPE cells were homogenized in lysis buffer (20 mM Tris pH8, 137 mM NaCl, 10% glycerol, 1% TX100) containing protease inhibitor cocktail (PIC, Roche) and zebrafish embryos were lysed in RIPA buffer with PIC. Lysates were centrifuged for 10min at 13000rpm. Sample buffer was added to the supernatants and samples were boiled.Standard whole mount in situ hybridizations were performed as previously described58 at 48hpf and 72 hpf with both ehd1a+b and ehd3 probes. Due to the high sequence similarity between these genes, we designed probes in the 3'UTR regions to maximize the specificity. However, the ehd1 probe was predicted to recognize both ehd1a and ehd1b transcripts. Primers: ehd1 (F) primer: 5’GTTTTAACCCAAGCGCCACC 3’, ehd1 (R) primer: 5’ATCTGGAGGAATTGCGCAGC 3’ (amplified product: 394pb, specific to ehd1a+b 3’UTR), ehd3 (F) primer: 5’ ATGCTTCACTAGGAGGAATGG 3’. ehd3 (R) primer: 5’GTTGACTGTTGGGTTTTACGG 3’ (amplified product: 411bp. Specific to ehd3 3’UTR).

Expression analysis

Taqman gene expression probes for EHD1-4 were purchased from Applied Biosystems and qRT-PCR was performed as previously described15.

Western blots on cell lysates were performed as previously described15. Briefly, RPE cells were homogenized in lysis buffer (20 mM Tris pH8, 137 mM NaCl, 10% glycerol, 1% TX100) containing protease inhibitor cocktail (PIC, Roche) and zebrafish embryos were lysed in RIPA buffer with PIC. Lysates were centrifuged for 10min at 13000rpm. Sample buffer was added to the supernatants and samples were boiled.

Standard whole mount in situ hybridizations were performed as previously described58 at 48hpf and 72 hpf with both ehd1a+b and ehd3 probes. Due to the high sequence similarity between these genes, we designed probes in the 3'UTR regions to maximize the specificity. However, the ehd1 probe was predicted to recognize both ehd1a and ehd1b transcripts. Primers: ehd1 (F) primer: 5’GTTTTAACCCAAGCGCCACC 3’, ehd1 (R) primer: 5’ATCTGGAGGAATTGCGCAGC 3’ (amplified product: 394pb, specific to ehd1a+b 3’UTR), ehd3 (F) primer: 5’ ATGCTTCACTAGGAGGAATGG 3’. ehd3 (R) primer: 5’GTTGACTGTTGGGTTTTACGG 3’ (amplified product: 411bp. Specific to ehd3 3’UTR).

Statistics and ReproducibilityStatistical analyses were performed as specified with GraphPad Prism 6 for Macintosh OS. All data are presented as mean ± S.E.M or S.D as specified in the figure legends, and two group comparisons were done with an unpaired two-tailed Student's t-test. A value of P<0.05 was considered as statistically significant. All P values were indicated on graphs in figures as followed: * P<0.05, **P<0.001. ***P<0.0001. Exact P values can be found in the Statistics Source Data Sheet (Supplementary Table 2).Experiments were reproducibly performed as follows: one experiment (Fig 1h, Supplementary Fig 2d), two independent experiments (Fig 1j, 2h, 3a, 4i, 5e, 5f, 7e, Supplementary Fig 1i, 1l, 2a, 2e, 5), 3 independent experiments (Fig 1a, 1d-f, 2a, 2c-e, 2g, 3f-g, 4c-g, 5d, 5g, 6a, 6c, 7b, 8a-b, 8f, Supplementary Figure 1d, 1g, 1k, 2c, 4b), two-four independent experiments (Fig 4a, Supplementary Fig 4a) and more than 3 independent experiments (Fig 1i, 3c-e, 5h-j, 7d, Supplementary Fig 1h, 1j, 3, Movie 1-5).

Statistics and Reproducibility

Statistical analyses were performed as specified with GraphPad Prism 6 for Macintosh OS. All data are presented as mean ± S.E.M or S.D as specified in the figure legends, and two group comparisons were done with an unpaired two-tailed Student's t-test. A value of P<0.05 was considered as statistically significant. All P values were indicated on graphs in figures as followed: * P<0.05, **P<0.001. ***P<0.0001. Exact P values can be found in the Statistics Source Data Sheet (Supplementary Table 2).

Experiments were reproducibly performed as follows: one experiment (Fig 1h, Supplementary Fig 2d), two independent experiments (Fig 1j, 2h, 3a, 4i, 5e, 5f, 7e, Supplementary Fig 1i, 1l, 2a, 2e, 5), 3 independent experiments (Fig 1a, 1d-f, 2a, 2c-e, 2g, 3f-g, 4c-g, 5d, 5g, 6a, 6c, 7b, 8a-b, 8f, Supplementary Figure 1d, 1g, 1k, 2c, 4b), two-four independent experiments (Fig 4a, Supplementary Fig 4a) and more than 3 independent experiments (Fig 1i, 3c-e, 5h-j, 7d, Supplementary Fig 1h, 1j, 3, Movie 1-5).

Supplementary Material1video1video2video3video4video5234567table 1table 2

Supplementary Material
