---
pmid: '22081014'
title: NSP-Cas protein structures reveal a promiscuous interaction module in cell
  signaling.
authors:
- Mace PD
- Wallez Y
- Dobaczewska MK
- Lee JJ
- Robinson H
- Pasquale EB
- Riedl SJ
journal: Nat Struct Mol Biol
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3230775
doi: 10.1038/nsmb.2152
---

# NSP-Cas protein structures reveal a promiscuous interaction module in cell signaling.
**Authors:** Mace PD, Wallez Y, Dobaczewska MK, Lee JJ, Robinson H, Pasquale EB, Riedl SJ
**Journal:** Nat Struct Mol Biol (2011)
**DOI:** [10.1038/nsmb.2152](https://doi.org/10.1038/nsmb.2152)
**PMC:** [PMC3230775](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3230775/)

## Abstract

1. Nat Struct Mol Biol. 2011 Nov 13;18(12):1381-7. doi: 10.1038/nsmb.2152.

NSP-Cas protein structures reveal a promiscuous interaction module in cell 
signaling.

Mace PD(1), Wallez Y, Dobaczewska MK, Lee JJ, Robinson H, Pasquale EB, Riedl SJ.

Author information:
(1)Program of Apoptosis and Cell Death Research, Cancer Center, Sanford-Burnham 
Medical Research Institute, La Jolla, California, USA.

Members of the novel SH2-containing protein (NSP) and Crk-associated substrate 
(Cas) protein families form multidomain signaling platforms that mediate cell 
migration and invasion through a collection of distinct signaling motifs. 
Members of each family interact via their respective C-terminal domains, but the 
mechanism of this association has remained enigmatic. Here we present the 
crystal structures of the C-terminal domain from the NSP protein BCAR3 and the 
complex of NSP3 with p130Cas. BCAR3 adopts the Cdc25-homology fold of Ras GTPase 
exchange factors, but it has a 'closed' conformation incapable of enzymatic 
activity. The structure of the NSP3-p130Cas complex reveals that this closed 
conformation is instrumental for interaction of NSP proteins with a focal 
adhesion-targeting domain present in Cas proteins. This enzyme-to-adaptor 
conversion enables high-affinity, yet promiscuous, interactions between NSP and 
Cas proteins and represents an unprecedented mechanistic paradigm linking 
cellular signaling networks.

DOI: 10.1038/nsmb.2152
PMCID: PMC3230775
PMID: 22081014 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONThe integration of signals triggered by diverse stimuli is essential for cell communication in higher organisms and is primarily achieved through the assembly of multiprotein signaling nodes. The Crk-associated substrate (Cas) and novel SH2-containing protein (NSP) families form one such class of signaling nodes, and correspondingly harbor multiple protein-protein interaction domains and motifs that mediate association with an abundance of cell signaling factors. This allows them to function as mediators that integrate signals emanating from cell adhesion and environmental stimuli to promote cell migration, adhesion and invasion. As such, disregulation of signaling through Cas and NSP proteins is relevant to a spectrum of disease processes, particularly tumor progression and cancer metastasis.The complexity of NSP-Cas functions is best exemplified by two prominent family members p130Cas and the NSP protein BCAR3, which are also known for their ability to confer antiestrogen resistance in breast cancer1,2. Firmly embedded in the Src-Crk signaling axis, p130Cas interacts with more than a dozen signaling factors besides Src family kinases and Crk, including FAK, PYK2, FRNK, RapGEF1, Aurora kinase A, PI3K, NMP4, NCK1 and SHIP2 (reviewed in ref 3). BCAR3 is associated with enhanced PI3K activity downstream of growth factor receptors and subsequent activation of Rac, Cdc42, AKT and PAK14. Besides this spectrum of signaling partners, p130Cas and BCAR3 also bind directly to each other5, thereby linking their respective signaling networks to form a platform that mediates migratory signaling. In addition to BCAR3, p130Cas is also capable of interacting with NSP family members NSP3 and NSP16,7. On the other hand, BCAR3 has been shown to interact with HEF1 (Cas-L) in addition to p130Cas8. Further combinations are created by association of the NSP family members NSP3 and NSP1 with the Cas proteins Cas-L and Efs, adding to the diversity of NSP–Cas signaling networks9–11. NSP and Cas family proteins are therefore capable of propagating class-specific but promiscuous combinatorial networks, which are each essential to allow their diverse functions in specialized cellular processes. As such the effect of aberrant NSP–Cas signaling is manifested in a range of disease processes. p130Cas and BCAR3 have long been known to exert a concerted effect in cancer12,13, while the NSP3 (SHEP1)–Cas-L signaling node has recently been found to be crucial for B-cell migration and maturation9,11. Additionally the NSP3–p130Cas module is reported to be essential in neuronal cells in for proper olfactory development, and when absent causes a phenotype reminiscent of the human developmental disorder Kallmann syndrome14. These examples underline the central importance of distinct NSP–Cas modules in cellular processes that require integration of adhesion and chemotactic stimuli.While it is known that the respective C-terminal domains of Cas and NSP proteins are fundamental to their interaction (Supplementary Fig. 1a)7, it has remained unclear at the molecular level how different family members can interact in this class-specific yet promiscuous manner. Moreover the predicted nature of the C-terminal Cas and NSP domains suggest an unprecedented interaction mechanism. The NSP-binding portion of Cas proteins has been proposed to assume a focal-adhesion targeting (FAT) domain-like fold, while the C-terminal domain of NSP proteins shares weak homology with Cdc25-homology domains (Cdc25-hd) found in GTPase exchange factors15,16. However, the molecular mechanism of a direct Cdc25-hd:FAT interaction is currently without precedent, and the question of how this influences the putative activities of each domain while mechanically linking NSP and Cas signaling remains outstanding.To shed light on the molecular features enabling the formation of Cas–NSP signaling modules, we have solved the crystal structures of the C-terminal region of unbound human BCAR3 and of the complex between human NSP3 and p130Cas. These structures in combination with our biochemical and biological analyses show that the Cdc25-homology domain present in NSP proteins has evolved as an adaptor domain based on an enzymatic fold used in RasGEFs. This unique NSP structure is specifically tailored to tightly bind the FAT domain of Cas proteins using a distinctive extended interface. Furthermore, the sophisticated and class-specific interaction patterns observed in the NSP3 and p130Cas interfaces are conserved among the members of both families. As a result, NSP and Cas proteins can promiscuously interact with each other to form various combinations of signaling modules that finely regulate cellular processes.

INTRODUCTION

The integration of signals triggered by diverse stimuli is essential for cell communication in higher organisms and is primarily achieved through the assembly of multiprotein signaling nodes. The Crk-associated substrate (Cas) and novel SH2-containing protein (NSP) families form one such class of signaling nodes, and correspondingly harbor multiple protein-protein interaction domains and motifs that mediate association with an abundance of cell signaling factors. This allows them to function as mediators that integrate signals emanating from cell adhesion and environmental stimuli to promote cell migration, adhesion and invasion. As such, disregulation of signaling through Cas and NSP proteins is relevant to a spectrum of disease processes, particularly tumor progression and cancer metastasis.

The complexity of NSP-Cas functions is best exemplified by two prominent family members p130Cas and the NSP protein BCAR3, which are also known for their ability to confer antiestrogen resistance in breast cancer1,2. Firmly embedded in the Src-Crk signaling axis, p130Cas interacts with more than a dozen signaling factors besides Src family kinases and Crk, including FAK, PYK2, FRNK, RapGEF1, Aurora kinase A, PI3K, NMP4, NCK1 and SHIP2 (reviewed in ref 3). BCAR3 is associated with enhanced PI3K activity downstream of growth factor receptors and subsequent activation of Rac, Cdc42, AKT and PAK14. Besides this spectrum of signaling partners, p130Cas and BCAR3 also bind directly to each other5, thereby linking their respective signaling networks to form a platform that mediates migratory signaling. In addition to BCAR3, p130Cas is also capable of interacting with NSP family members NSP3 and NSP16,7. On the other hand, BCAR3 has been shown to interact with HEF1 (Cas-L) in addition to p130Cas8. Further combinations are created by association of the NSP family members NSP3 and NSP1 with the Cas proteins Cas-L and Efs, adding to the diversity of NSP–Cas signaling networks9–11. NSP and Cas family proteins are therefore capable of propagating class-specific but promiscuous combinatorial networks, which are each essential to allow their diverse functions in specialized cellular processes. As such the effect of aberrant NSP–Cas signaling is manifested in a range of disease processes. p130Cas and BCAR3 have long been known to exert a concerted effect in cancer12,13, while the NSP3 (SHEP1)–Cas-L signaling node has recently been found to be crucial for B-cell migration and maturation9,11. Additionally the NSP3–p130Cas module is reported to be essential in neuronal cells in for proper olfactory development, and when absent causes a phenotype reminiscent of the human developmental disorder Kallmann syndrome14. These examples underline the central importance of distinct NSP–Cas modules in cellular processes that require integration of adhesion and chemotactic stimuli.

While it is known that the respective C-terminal domains of Cas and NSP proteins are fundamental to their interaction (Supplementary Fig. 1a)7, it has remained unclear at the molecular level how different family members can interact in this class-specific yet promiscuous manner. Moreover the predicted nature of the C-terminal Cas and NSP domains suggest an unprecedented interaction mechanism. The NSP-binding portion of Cas proteins has been proposed to assume a focal-adhesion targeting (FAT) domain-like fold, while the C-terminal domain of NSP proteins shares weak homology with Cdc25-homology domains (Cdc25-hd) found in GTPase exchange factors15,16. However, the molecular mechanism of a direct Cdc25-hd:FAT interaction is currently without precedent, and the question of how this influences the putative activities of each domain while mechanically linking NSP and Cas signaling remains outstanding.

To shed light on the molecular features enabling the formation of Cas–NSP signaling modules, we have solved the crystal structures of the C-terminal region of unbound human BCAR3 and of the complex between human NSP3 and p130Cas. These structures in combination with our biochemical and biological analyses show that the Cdc25-homology domain present in NSP proteins has evolved as an adaptor domain based on an enzymatic fold used in RasGEFs. This unique NSP structure is specifically tailored to tightly bind the FAT domain of Cas proteins using a distinctive extended interface. Furthermore, the sophisticated and class-specific interaction patterns observed in the NSP3 and p130Cas interfaces are conserved among the members of both families. As a result, NSP and Cas proteins can promiscuously interact with each other to form various combinations of signaling modules that finely regulate cellular processes.

RESULTSBCAR3 has an unconventional Cdc25-homology domainTo elucidate the enigmatic nature of NSP protein signaling, we sought to obtain structural and mechanistic insight into the C-terminal domain of the best characterized family member, BCAR3. Thorough in vitro investigation of this domain, or any other NSP C-terminal domain, had previously been prevented by the poor protein chemical properties of this region. After testing various constructs, we succeeded in generating a soluble BCAR3 C-terminal domain (spanning residues 502–825) with properties suitable for biochemical and structural investigation. Extensive screening of crystallization conditions resulted in well-diffracting BCAR3 crystals, which allowed us to solve the structure of BCAR3 at a resolution of 2.4 Å (Fig. 1a). The crystal structure contains four molecules of BCAR3 in the asymmetric unit, which overlay with excellent fidelity (RMSD range 0.24–0.45 Å). The BCAR3 structure reveals an unexpected Ras-type GDP exchange factor architecture that sheds light on the contradictory nature of NSP proteins reported in the literature. BCAR3 adopts the overall fold of the Cdc25-homology domain structures solved to date with RMSD values between the BCAR3 core and SOS, EPAC2 or RasGRF1 of 2.18 Å, 2.35 Å and 2.19 Å respectively17–19 (Supplementary Fig. 1b). However, intriguingly BCAR3 shows dramatic deviations in two distinct regions that result in a novel ”closed” arrangement of the Cdc25-homology domain (Fig. 1, Supplementary Fig. 2). In detail, helices αh and αi (nomenclature derived from SOS17) each bend approximately 60° in orthogonal directions relative to their known GEF counterparts (Fig. 1c,d). Helix αh is also markedly shorter in BCAR3 than in other Cdc25-homology domains, and hydrophobic interactions by Leu713, Val714 and Leu716 cause the helix to be buried between αb and αi. Helix αi is broken into two portions (αi and αi*, Supplementary Fig. 2), with αi folding towards the C-terminus of αf in the Cdc25-homology domain core achieving the closed BCAR3 conformation (Fig. 1). The second markedly different region is comprised of residues 565–592. This “NSP-specific region” adopts a distinct conformation that includes helices αNSPa and αNSPb and is well conserved in all NSP proteins.Comparison with complexes of catalytically competent Cdc25-homology domains bound to their target GTPases shows that BCAR3 assumes a conformation incompatible with classical GEF function because its GTPase binding site is completely occluded (Fig. 1e
and
Supplementary Fig. 3a). This occlusion is mainly achieved by the closed conformation of helix αi and its preceding linker comprised of BCAR3 residues 718–736. In active “open” Cdc25-homology domains, helices αh and αi form a hairpin structure (Fig. 1) that is crucial for activity because it displaces the switch 1 region of target GTPases17,20. In fact, previous studies show that even slight conformational adjustments in this hairpin have substantial effects on exchange activity19,21. Thus, the dramatic changes observed in BCAR3 imply a drastic functional redirection of crucial GEF catalytic elements.Closed BCAR3 lacks catalytic activity in vitroTo validate our structural observations and gain further insight into the functional properties of the BCAR3 C-terminal domain, we examined its ability to promote nucleotide exchange in vitro, which has previously been controversial in the literature5,16,22–24. Using BCAR3 502–825 allowed us to assess the ability of BCAR3 to induce GDP release from its putative target GTPase, Rap1A25. In contrast to positive controls with either EDTA or C3G (a bona fide GEF for Rap1A26), which effectively promote GDP release from Rap1A, no activity was observed for BCAR3 (Fig. 1f). Furthermore no interaction between the BCAR3 Cdc25-homology domain and Rap1A was detected in in vitro pulldowns or isothermal titration calorimetry experiments (data not shown). These results show a complete lack of catalytic activity for BCAR3, in accordance with the crystal structure, and reveal that BCAR3 represents a non-catalytic adaption of the Cdc25-homology domain fold.Structure of the NSP3–p130Cas complexTo further understand the function of the observed “closed” NSP conformation and ultimately understand the molecular basis of NSP–Cas signaling modules, we next sought to obtain the structure of an NSP–Cas interaction complex. To this end, we extended our investigations to the BCAR3 relative NSP3 (SHEP1) and the prototypical Cas family member p130Cas. We identified constructs encoding residues 382–703 of NSP3 and 645–870 of p130Cas, which allowed us to generate the individual protein domains as well as complexes of NSP3 or BCAR3 bound to p130Cas, all exhibiting excellent biochemical properties (Supplementary Fig. 3b,c). After extensive crystallization trials using both complexes, we were able to solve the crystal structure of NSP3 bound to p130Cas at a resolution of 2.5 Å (Fig. 2a). The asymmetric unit of the crystal contains two NSP3–p130Cas complexes (Supplementary Fig. 4), one of which is more clearly defined by electron density and is described hereafter. In line with our biochemical characterization (Supplementary Fig. 3b,c and 5a–c), the crystal structure shows that the two proteins form a 1:1 arrangement characterized by an extensive binding interface that buries a total of 1,192 Å2. This substantial interface is in good agreement with isothermal titration calorimetry results, which support a very tight (30 nM) interaction between p130Cas and the closely related NSP protein BCAR3 (Supplementary Fig. 5d).In the final model of the NSP3–p130Cas complex (Fig. 2a), NSP3 residues 386–698 are well defined by electron density with the exception of the loop between residues 599 and 612, which was not included in the model. The C-terminal domain of p130Cas is defined by electron density from residues 739 to 872 and adopts the four-helix bundle fold characteristic of FAT domains. The structure of NSP3 closely resembles a Cdc25-homology domain fold, but intriguingly shares the closed conformation observed for unbound BCAR3. A comparative backbone RMSD of 0.97 Å between p130Cas-bound NSP3 and the unbound BCAR3 underscores their extremely close structural similarity (Supplementary Fig. 6a)The closed NSP Cdc25-homology domain is instrumental for Cas bindingThe structure of the NSP3–p130Cas complex reveals that the closed conformation observed for the NSP3 C-terminal domain is key in enabling a Cdc25-hd–FAT domain interaction to form a previously undescribed signaling tether. This unique binding interface utilized by NSP3 and p130Cas can be divided into two main binding motifs (Fig. 3, Supplementary Fig. 7a), which coincide with the two regions that deviate from the canonical Cdc25-homology domain fold of NSP3 (and BCAR3) and are strictly dependent on the closed conformation of the domain. In the first motif, which we dub Site 1, a hydrophobic strip in helix αi of NSP3 interacts with a highly complementary surface on p130Cas. Site 2 is generated by a distinct region between αNSPa and αb of NSP3 and synergizes with Site 1 to generate a tight and unique signaling interaction.Specifically, in Site 1 Val616, Leu620 and Leu623 on helix αi packs into the hydrophobic groove between α2 and α3 of the p130Cas FAT domain (Fig. 4a, Supplementary Fig. 7a). These contacts are further augmented by charged interactions mediated by Glu617, Glu624 and Arg627 of NSP3. In site 2 helix αNSPb and subsequent residues Glu465 and Leu469-Pro470-His471 interact with a predominantly hydrophobic groove between α1 and α2 of the p130Cas FAT domain. Supplementing the two main interfaces is a peripheral association of residues 386–388 from the N-terminus of the NSP3 Cdc25-homology domain and Leu596 from helix αh of NSP3 (Fig. 3c), which contributes to a hydrophobic surface that links the Site 1 and Site 2 binding motifs.As discussed for BCAR3, the closed NSP3 conformation renders key elements of the Cdc25-homology domain incapable of participating in a canonical interaction with a Ras GTPase. Furthermore, the presence of Cas in the complex provides even greater steric hindrance to such a hypothetical GTPase interaction. To further confirm the validity of this structural observation, we performed in vitro nucleotide exchange assays with both unbound NSP3 and NSP3 in complex with the p130Cas FAT domain. In addition to Rap1A, we included Rap2A as a further putative target GTPase for NSP327. The results do not show any detectable in vitro exchange activity for either unbound or Cas-bound NSP3 (Fig. 2c and Supplementary Fig. 6b). This is in accordance with our structural observations depicting a closed conformation of the NSP3 Cdc25-homology domain as essential for Cas binding.Cas extends the paradigm of FAT domain interactionsBesides revealing a critical adaption in NSP proteins, the NSP3–p130Cas complex provides the first atomic resolution insight demonstrating that the C-terminal domains of Cas family proteins adopt a well-defined FAT domain type four-helix bundle. Extensive hydrophobic interactions in the core of the domain suggest a stable arrangement of these helices (Fig. 3a). Generally, FAT domains are best known to function as adaptor domains in focal adhesion signaling28, where they interact with binding partners by recruiting helical motifs to surface grooves between their constituent helices29–31. However, these interactions are characteristically weak29,32, whereas we observe a dissociation constant in the low nanomolar range for the binding of BCAR3 to the FAT domain of p130Cas and tight binding between NSP3 and p130Cas (Supplementary Fig. 5). While the tertiary structure of the C-terminal domain of p130Cas is nearly identical to that of other FAT domains (Fig. 3b), p130Cas achieves extremely tight binding to NSP3 by a novel adaptation of the FAT domain interaction mode (Fig. 3c) utilizing the bi-partite binding motifs described above. Firstly the NSP3–p130Cas interaction at Site 1 is accomplished through a binding groove between helices α2 and α3 of the p130Cas FAT domain. This interaction is reminiscent of the α2–α3 binding site used by FAK and Pyk2 to bind paxillin30,31, but p130Cas accommodates a distinctly longer helical stretch from NSP3 and buries a substantially greater surface area of 630 Å2 (Fig. 3c). The interaction at Site 2 on the other hand, has not been observed for other FAT-type four helical bundles. On p130Cas this interface is formed by helix α1 and the C-terminal portion of helix α2 of the FAT scaffold, and in concert with the extended Site 1 interface achieves the unusually tight binding of the p130Cas FAT domain to NSP proteins.Biochemistry and biology of NSP–Cas complexesTo further verify our structural findings, we performed extensive biochemical and biological mutational studies. We designed structure-based mutations to disrupt binding elements while conserving the integrity of the protein domains (Fig. 4a). All NSP3, BCAR3 and p130Cas mutant domains showed excellent expression and solubility suitable for binding assays. The NSP3 R627E Site 1 mutation or L469R Site 2 mutation reduced complex formation with the p130Cas FAT domain in vitro, indicating that they substantially weaken the tight interaction (Fig. 4b, Supplementary Fig. 7b). In addition, the NSP3 L623E Site 1 mutation completely abrogated the interaction even at the substantial concentrations used in our in vitro association experiments (Fig. 4b). In COS cells the full-length NSP3 L623E and L469R mutants also efficiently disrupted the complex with full-length p130Cas when the two proteins were co-expressed (Fig. 4c). BCAR3 residues Leu744 and Arg748 correspond to Leu623 and Arg627 in NSP3, and when mutated also effectively disrupted the association with p130Cas in COS cells confirming that different NSP proteins utilize a conserved p130Cas binding mode (Fig. 4d). Mutation of Arg743 in mouse BCAR3, corresponding to Arg748 in human BCAR3, has also been independently confirmed to disrupt p130Cas binding in recent studies16,33. However, our in vitro results suggest that mutating Leu623 in NSP3 more effectively abrogates binding and therefore represents a more potent probe for dissecting NSP3–Cas signaling modules. The potency of Leu623 is in good agreement with its location in the hydrophobic core of the Site 1 interaction, in close proximity to the hydrophobic pair Phe794 and Leu787 of p130Cas (Fig. 4a). In contrast, the salt bridge that Arg627 forms with Asp797 of p130Cas is more peripheral to the interface.We also examined interface mutations in p130Cas for their ability to counteract complex formation with NSP proteins. Based on structural insight, we chose Leu787, Phe794 and Asp797 as promising candidates for mutagenesis. Individual mutations of each of these residues (L787E, F794R and D797R) weakened the in vitro association of the p130Cas FAT domain with the NSP3 Cdc25-homology domain, while combinations of two mutations virtually abolished complex formation (Fig. 4e, Supplementary Fig. 7b). In addition, the single p130Cas L787E and F794R mutations markedly reduced association between full-length p130Cas and NSP3 co-expressed in COS cells, further confirming features of the NSP3–p130Cas complex in the cellular context (Fig. 4f).To examine the functional effects of targeted disruption of the NSP3–p130Cas module on cell migration, we tested the NSP3 L623E mutant in a Transwell migration assay. NSP3 has previously been shown to promote COS cell migration towards EGF as a chemoattractant in a Cas-dependent manner27. The specific interface mutant L623E abrogated the NSP3-dependent enhancement of COS cell migration towards epidermal growth factor (EGF). Cells expressing this mutant showed migratory ability towards EGF similar to that of cells transfected with empty vector control (Fig. 4g,h). This emphasizes the role of NSP–Cas module formation in linking cell motility and growth factor signaling, while at the same time demonstrating the potency of structural interface mutations for investigation of these signaling nodes.Promiscuous yet class-specific NSP–Cas interactionsThe structures of BCAR3, NSP3 and p130Cas provide not only the first insight into the C-terminal regions of NSP and Cas proteins and their interactions, but also a basis upon which to apply features depicted in the wider context of NSP and Cas family signaling. To this end, we created molecular models for the C-terminal domains of the remaining family members based on sequence profile alignments and the crystal structures solved in this work. Mapping of primary interface residues on these models shows that residues involved in NSP–Cas complex-formation are clearly conserved within each family (Fig. 5a). In line with this degree of conservation are residues that determine the core fold of NSP and Cas family C-terminal domains (Supplementary Fig. 8), whereas remaining surface features are considerably more divergent. This illustrates that all three NSP family proteins have pervasively repurposed an enzymatic fold as a recruitment module, while Cas family proteins share a C-terminal FAT domain that accommodates the extended and tight binding interface observed in the NSP3–p130Cas complex. This extended interface, built upon the Site 1 and Site 2 binding motifs, is elegantly specific to NSP–Cas interactions yet allows different combinations of high-affinity mechanical linkages that integrate heterogeneous signaling pathways between the two families of proteins.

BCAR3 has an unconventional Cdc25-homology domainTo elucidate the enigmatic nature of NSP protein signaling, we sought to obtain structural and mechanistic insight into the C-terminal domain of the best characterized family member, BCAR3. Thorough in vitro investigation of this domain, or any other NSP C-terminal domain, had previously been prevented by the poor protein chemical properties of this region. After testing various constructs, we succeeded in generating a soluble BCAR3 C-terminal domain (spanning residues 502–825) with properties suitable for biochemical and structural investigation. Extensive screening of crystallization conditions resulted in well-diffracting BCAR3 crystals, which allowed us to solve the structure of BCAR3 at a resolution of 2.4 Å (Fig. 1a). The crystal structure contains four molecules of BCAR3 in the asymmetric unit, which overlay with excellent fidelity (RMSD range 0.24–0.45 Å). The BCAR3 structure reveals an unexpected Ras-type GDP exchange factor architecture that sheds light on the contradictory nature of NSP proteins reported in the literature. BCAR3 adopts the overall fold of the Cdc25-homology domain structures solved to date with RMSD values between the BCAR3 core and SOS, EPAC2 or RasGRF1 of 2.18 Å, 2.35 Å and 2.19 Å respectively17–19 (Supplementary Fig. 1b). However, intriguingly BCAR3 shows dramatic deviations in two distinct regions that result in a novel ”closed” arrangement of the Cdc25-homology domain (Fig. 1, Supplementary Fig. 2). In detail, helices αh and αi (nomenclature derived from SOS17) each bend approximately 60° in orthogonal directions relative to their known GEF counterparts (Fig. 1c,d). Helix αh is also markedly shorter in BCAR3 than in other Cdc25-homology domains, and hydrophobic interactions by Leu713, Val714 and Leu716 cause the helix to be buried between αb and αi. Helix αi is broken into two portions (αi and αi*, Supplementary Fig. 2), with αi folding towards the C-terminus of αf in the Cdc25-homology domain core achieving the closed BCAR3 conformation (Fig. 1). The second markedly different region is comprised of residues 565–592. This “NSP-specific region” adopts a distinct conformation that includes helices αNSPa and αNSPb and is well conserved in all NSP proteins.Comparison with complexes of catalytically competent Cdc25-homology domains bound to their target GTPases shows that BCAR3 assumes a conformation incompatible with classical GEF function because its GTPase binding site is completely occluded (Fig. 1e
and
Supplementary Fig. 3a). This occlusion is mainly achieved by the closed conformation of helix αi and its preceding linker comprised of BCAR3 residues 718–736. In active “open” Cdc25-homology domains, helices αh and αi form a hairpin structure (Fig. 1) that is crucial for activity because it displaces the switch 1 region of target GTPases17,20. In fact, previous studies show that even slight conformational adjustments in this hairpin have substantial effects on exchange activity19,21. Thus, the dramatic changes observed in BCAR3 imply a drastic functional redirection of crucial GEF catalytic elements.

BCAR3 has an unconventional Cdc25-homology domain

To elucidate the enigmatic nature of NSP protein signaling, we sought to obtain structural and mechanistic insight into the C-terminal domain of the best characterized family member, BCAR3. Thorough in vitro investigation of this domain, or any other NSP C-terminal domain, had previously been prevented by the poor protein chemical properties of this region. After testing various constructs, we succeeded in generating a soluble BCAR3 C-terminal domain (spanning residues 502–825) with properties suitable for biochemical and structural investigation. Extensive screening of crystallization conditions resulted in well-diffracting BCAR3 crystals, which allowed us to solve the structure of BCAR3 at a resolution of 2.4 Å (Fig. 1a). The crystal structure contains four molecules of BCAR3 in the asymmetric unit, which overlay with excellent fidelity (RMSD range 0.24–0.45 Å). The BCAR3 structure reveals an unexpected Ras-type GDP exchange factor architecture that sheds light on the contradictory nature of NSP proteins reported in the literature. BCAR3 adopts the overall fold of the Cdc25-homology domain structures solved to date with RMSD values between the BCAR3 core and SOS, EPAC2 or RasGRF1 of 2.18 Å, 2.35 Å and 2.19 Å respectively17–19 (Supplementary Fig. 1b). However, intriguingly BCAR3 shows dramatic deviations in two distinct regions that result in a novel ”closed” arrangement of the Cdc25-homology domain (Fig. 1, Supplementary Fig. 2). In detail, helices αh and αi (nomenclature derived from SOS17) each bend approximately 60° in orthogonal directions relative to their known GEF counterparts (Fig. 1c,d). Helix αh is also markedly shorter in BCAR3 than in other Cdc25-homology domains, and hydrophobic interactions by Leu713, Val714 and Leu716 cause the helix to be buried between αb and αi. Helix αi is broken into two portions (αi and αi*, Supplementary Fig. 2), with αi folding towards the C-terminus of αf in the Cdc25-homology domain core achieving the closed BCAR3 conformation (Fig. 1). The second markedly different region is comprised of residues 565–592. This “NSP-specific region” adopts a distinct conformation that includes helices αNSPa and αNSPb and is well conserved in all NSP proteins.

Comparison with complexes of catalytically competent Cdc25-homology domains bound to their target GTPases shows that BCAR3 assumes a conformation incompatible with classical GEF function because its GTPase binding site is completely occluded (Fig. 1e
and
Supplementary Fig. 3a). This occlusion is mainly achieved by the closed conformation of helix αi and its preceding linker comprised of BCAR3 residues 718–736. In active “open” Cdc25-homology domains, helices αh and αi form a hairpin structure (Fig. 1) that is crucial for activity because it displaces the switch 1 region of target GTPases17,20. In fact, previous studies show that even slight conformational adjustments in this hairpin have substantial effects on exchange activity19,21. Thus, the dramatic changes observed in BCAR3 imply a drastic functional redirection of crucial GEF catalytic elements.

Closed BCAR3 lacks catalytic activity in vitroTo validate our structural observations and gain further insight into the functional properties of the BCAR3 C-terminal domain, we examined its ability to promote nucleotide exchange in vitro, which has previously been controversial in the literature5,16,22–24. Using BCAR3 502–825 allowed us to assess the ability of BCAR3 to induce GDP release from its putative target GTPase, Rap1A25. In contrast to positive controls with either EDTA or C3G (a bona fide GEF for Rap1A26), which effectively promote GDP release from Rap1A, no activity was observed for BCAR3 (Fig. 1f). Furthermore no interaction between the BCAR3 Cdc25-homology domain and Rap1A was detected in in vitro pulldowns or isothermal titration calorimetry experiments (data not shown). These results show a complete lack of catalytic activity for BCAR3, in accordance with the crystal structure, and reveal that BCAR3 represents a non-catalytic adaption of the Cdc25-homology domain fold.

Closed BCAR3 lacks catalytic activity in vitro

To validate our structural observations and gain further insight into the functional properties of the BCAR3 C-terminal domain, we examined its ability to promote nucleotide exchange in vitro, which has previously been controversial in the literature5,16,22–24. Using BCAR3 502–825 allowed us to assess the ability of BCAR3 to induce GDP release from its putative target GTPase, Rap1A25. In contrast to positive controls with either EDTA or C3G (a bona fide GEF for Rap1A26), which effectively promote GDP release from Rap1A, no activity was observed for BCAR3 (Fig. 1f). Furthermore no interaction between the BCAR3 Cdc25-homology domain and Rap1A was detected in in vitro pulldowns or isothermal titration calorimetry experiments (data not shown). These results show a complete lack of catalytic activity for BCAR3, in accordance with the crystal structure, and reveal that BCAR3 represents a non-catalytic adaption of the Cdc25-homology domain fold.

Structure of the NSP3–p130Cas complexTo further understand the function of the observed “closed” NSP conformation and ultimately understand the molecular basis of NSP–Cas signaling modules, we next sought to obtain the structure of an NSP–Cas interaction complex. To this end, we extended our investigations to the BCAR3 relative NSP3 (SHEP1) and the prototypical Cas family member p130Cas. We identified constructs encoding residues 382–703 of NSP3 and 645–870 of p130Cas, which allowed us to generate the individual protein domains as well as complexes of NSP3 or BCAR3 bound to p130Cas, all exhibiting excellent biochemical properties (Supplementary Fig. 3b,c). After extensive crystallization trials using both complexes, we were able to solve the crystal structure of NSP3 bound to p130Cas at a resolution of 2.5 Å (Fig. 2a). The asymmetric unit of the crystal contains two NSP3–p130Cas complexes (Supplementary Fig. 4), one of which is more clearly defined by electron density and is described hereafter. In line with our biochemical characterization (Supplementary Fig. 3b,c and 5a–c), the crystal structure shows that the two proteins form a 1:1 arrangement characterized by an extensive binding interface that buries a total of 1,192 Å2. This substantial interface is in good agreement with isothermal titration calorimetry results, which support a very tight (30 nM) interaction between p130Cas and the closely related NSP protein BCAR3 (Supplementary Fig. 5d).In the final model of the NSP3–p130Cas complex (Fig. 2a), NSP3 residues 386–698 are well defined by electron density with the exception of the loop between residues 599 and 612, which was not included in the model. The C-terminal domain of p130Cas is defined by electron density from residues 739 to 872 and adopts the four-helix bundle fold characteristic of FAT domains. The structure of NSP3 closely resembles a Cdc25-homology domain fold, but intriguingly shares the closed conformation observed for unbound BCAR3. A comparative backbone RMSD of 0.97 Å between p130Cas-bound NSP3 and the unbound BCAR3 underscores their extremely close structural similarity (Supplementary Fig. 6a)

Structure of the NSP3–p130Cas complex

To further understand the function of the observed “closed” NSP conformation and ultimately understand the molecular basis of NSP–Cas signaling modules, we next sought to obtain the structure of an NSP–Cas interaction complex. To this end, we extended our investigations to the BCAR3 relative NSP3 (SHEP1) and the prototypical Cas family member p130Cas. We identified constructs encoding residues 382–703 of NSP3 and 645–870 of p130Cas, which allowed us to generate the individual protein domains as well as complexes of NSP3 or BCAR3 bound to p130Cas, all exhibiting excellent biochemical properties (Supplementary Fig. 3b,c). After extensive crystallization trials using both complexes, we were able to solve the crystal structure of NSP3 bound to p130Cas at a resolution of 2.5 Å (Fig. 2a). The asymmetric unit of the crystal contains two NSP3–p130Cas complexes (Supplementary Fig. 4), one of which is more clearly defined by electron density and is described hereafter. In line with our biochemical characterization (Supplementary Fig. 3b,c and 5a–c), the crystal structure shows that the two proteins form a 1:1 arrangement characterized by an extensive binding interface that buries a total of 1,192 Å2. This substantial interface is in good agreement with isothermal titration calorimetry results, which support a very tight (30 nM) interaction between p130Cas and the closely related NSP protein BCAR3 (Supplementary Fig. 5d).

In the final model of the NSP3–p130Cas complex (Fig. 2a), NSP3 residues 386–698 are well defined by electron density with the exception of the loop between residues 599 and 612, which was not included in the model. The C-terminal domain of p130Cas is defined by electron density from residues 739 to 872 and adopts the four-helix bundle fold characteristic of FAT domains. The structure of NSP3 closely resembles a Cdc25-homology domain fold, but intriguingly shares the closed conformation observed for unbound BCAR3. A comparative backbone RMSD of 0.97 Å between p130Cas-bound NSP3 and the unbound BCAR3 underscores their extremely close structural similarity (Supplementary Fig. 6a)

The closed NSP Cdc25-homology domain is instrumental for Cas bindingThe structure of the NSP3–p130Cas complex reveals that the closed conformation observed for the NSP3 C-terminal domain is key in enabling a Cdc25-hd–FAT domain interaction to form a previously undescribed signaling tether. This unique binding interface utilized by NSP3 and p130Cas can be divided into two main binding motifs (Fig. 3, Supplementary Fig. 7a), which coincide with the two regions that deviate from the canonical Cdc25-homology domain fold of NSP3 (and BCAR3) and are strictly dependent on the closed conformation of the domain. In the first motif, which we dub Site 1, a hydrophobic strip in helix αi of NSP3 interacts with a highly complementary surface on p130Cas. Site 2 is generated by a distinct region between αNSPa and αb of NSP3 and synergizes with Site 1 to generate a tight and unique signaling interaction.Specifically, in Site 1 Val616, Leu620 and Leu623 on helix αi packs into the hydrophobic groove between α2 and α3 of the p130Cas FAT domain (Fig. 4a, Supplementary Fig. 7a). These contacts are further augmented by charged interactions mediated by Glu617, Glu624 and Arg627 of NSP3. In site 2 helix αNSPb and subsequent residues Glu465 and Leu469-Pro470-His471 interact with a predominantly hydrophobic groove between α1 and α2 of the p130Cas FAT domain. Supplementing the two main interfaces is a peripheral association of residues 386–388 from the N-terminus of the NSP3 Cdc25-homology domain and Leu596 from helix αh of NSP3 (Fig. 3c), which contributes to a hydrophobic surface that links the Site 1 and Site 2 binding motifs.As discussed for BCAR3, the closed NSP3 conformation renders key elements of the Cdc25-homology domain incapable of participating in a canonical interaction with a Ras GTPase. Furthermore, the presence of Cas in the complex provides even greater steric hindrance to such a hypothetical GTPase interaction. To further confirm the validity of this structural observation, we performed in vitro nucleotide exchange assays with both unbound NSP3 and NSP3 in complex with the p130Cas FAT domain. In addition to Rap1A, we included Rap2A as a further putative target GTPase for NSP327. The results do not show any detectable in vitro exchange activity for either unbound or Cas-bound NSP3 (Fig. 2c and Supplementary Fig. 6b). This is in accordance with our structural observations depicting a closed conformation of the NSP3 Cdc25-homology domain as essential for Cas binding.

The closed NSP Cdc25-homology domain is instrumental for Cas binding

The structure of the NSP3–p130Cas complex reveals that the closed conformation observed for the NSP3 C-terminal domain is key in enabling a Cdc25-hd–FAT domain interaction to form a previously undescribed signaling tether. This unique binding interface utilized by NSP3 and p130Cas can be divided into two main binding motifs (Fig. 3, Supplementary Fig. 7a), which coincide with the two regions that deviate from the canonical Cdc25-homology domain fold of NSP3 (and BCAR3) and are strictly dependent on the closed conformation of the domain. In the first motif, which we dub Site 1, a hydrophobic strip in helix αi of NSP3 interacts with a highly complementary surface on p130Cas. Site 2 is generated by a distinct region between αNSPa and αb of NSP3 and synergizes with Site 1 to generate a tight and unique signaling interaction.

Specifically, in Site 1 Val616, Leu620 and Leu623 on helix αi packs into the hydrophobic groove between α2 and α3 of the p130Cas FAT domain (Fig. 4a, Supplementary Fig. 7a). These contacts are further augmented by charged interactions mediated by Glu617, Glu624 and Arg627 of NSP3. In site 2 helix αNSPb and subsequent residues Glu465 and Leu469-Pro470-His471 interact with a predominantly hydrophobic groove between α1 and α2 of the p130Cas FAT domain. Supplementing the two main interfaces is a peripheral association of residues 386–388 from the N-terminus of the NSP3 Cdc25-homology domain and Leu596 from helix αh of NSP3 (Fig. 3c), which contributes to a hydrophobic surface that links the Site 1 and Site 2 binding motifs.

As discussed for BCAR3, the closed NSP3 conformation renders key elements of the Cdc25-homology domain incapable of participating in a canonical interaction with a Ras GTPase. Furthermore, the presence of Cas in the complex provides even greater steric hindrance to such a hypothetical GTPase interaction. To further confirm the validity of this structural observation, we performed in vitro nucleotide exchange assays with both unbound NSP3 and NSP3 in complex with the p130Cas FAT domain. In addition to Rap1A, we included Rap2A as a further putative target GTPase for NSP327. The results do not show any detectable in vitro exchange activity for either unbound or Cas-bound NSP3 (Fig. 2c and Supplementary Fig. 6b). This is in accordance with our structural observations depicting a closed conformation of the NSP3 Cdc25-homology domain as essential for Cas binding.

Cas extends the paradigm of FAT domain interactionsBesides revealing a critical adaption in NSP proteins, the NSP3–p130Cas complex provides the first atomic resolution insight demonstrating that the C-terminal domains of Cas family proteins adopt a well-defined FAT domain type four-helix bundle. Extensive hydrophobic interactions in the core of the domain suggest a stable arrangement of these helices (Fig. 3a). Generally, FAT domains are best known to function as adaptor domains in focal adhesion signaling28, where they interact with binding partners by recruiting helical motifs to surface grooves between their constituent helices29–31. However, these interactions are characteristically weak29,32, whereas we observe a dissociation constant in the low nanomolar range for the binding of BCAR3 to the FAT domain of p130Cas and tight binding between NSP3 and p130Cas (Supplementary Fig. 5). While the tertiary structure of the C-terminal domain of p130Cas is nearly identical to that of other FAT domains (Fig. 3b), p130Cas achieves extremely tight binding to NSP3 by a novel adaptation of the FAT domain interaction mode (Fig. 3c) utilizing the bi-partite binding motifs described above. Firstly the NSP3–p130Cas interaction at Site 1 is accomplished through a binding groove between helices α2 and α3 of the p130Cas FAT domain. This interaction is reminiscent of the α2–α3 binding site used by FAK and Pyk2 to bind paxillin30,31, but p130Cas accommodates a distinctly longer helical stretch from NSP3 and buries a substantially greater surface area of 630 Å2 (Fig. 3c). The interaction at Site 2 on the other hand, has not been observed for other FAT-type four helical bundles. On p130Cas this interface is formed by helix α1 and the C-terminal portion of helix α2 of the FAT scaffold, and in concert with the extended Site 1 interface achieves the unusually tight binding of the p130Cas FAT domain to NSP proteins.

Cas extends the paradigm of FAT domain interactions

Besides revealing a critical adaption in NSP proteins, the NSP3–p130Cas complex provides the first atomic resolution insight demonstrating that the C-terminal domains of Cas family proteins adopt a well-defined FAT domain type four-helix bundle. Extensive hydrophobic interactions in the core of the domain suggest a stable arrangement of these helices (Fig. 3a). Generally, FAT domains are best known to function as adaptor domains in focal adhesion signaling28, where they interact with binding partners by recruiting helical motifs to surface grooves between their constituent helices29–31. However, these interactions are characteristically weak29,32, whereas we observe a dissociation constant in the low nanomolar range for the binding of BCAR3 to the FAT domain of p130Cas and tight binding between NSP3 and p130Cas (Supplementary Fig. 5). While the tertiary structure of the C-terminal domain of p130Cas is nearly identical to that of other FAT domains (Fig. 3b), p130Cas achieves extremely tight binding to NSP3 by a novel adaptation of the FAT domain interaction mode (Fig. 3c) utilizing the bi-partite binding motifs described above. Firstly the NSP3–p130Cas interaction at Site 1 is accomplished through a binding groove between helices α2 and α3 of the p130Cas FAT domain. This interaction is reminiscent of the α2–α3 binding site used by FAK and Pyk2 to bind paxillin30,31, but p130Cas accommodates a distinctly longer helical stretch from NSP3 and buries a substantially greater surface area of 630 Å2 (Fig. 3c). The interaction at Site 2 on the other hand, has not been observed for other FAT-type four helical bundles. On p130Cas this interface is formed by helix α1 and the C-terminal portion of helix α2 of the FAT scaffold, and in concert with the extended Site 1 interface achieves the unusually tight binding of the p130Cas FAT domain to NSP proteins.

Biochemistry and biology of NSP–Cas complexesTo further verify our structural findings, we performed extensive biochemical and biological mutational studies. We designed structure-based mutations to disrupt binding elements while conserving the integrity of the protein domains (Fig. 4a). All NSP3, BCAR3 and p130Cas mutant domains showed excellent expression and solubility suitable for binding assays. The NSP3 R627E Site 1 mutation or L469R Site 2 mutation reduced complex formation with the p130Cas FAT domain in vitro, indicating that they substantially weaken the tight interaction (Fig. 4b, Supplementary Fig. 7b). In addition, the NSP3 L623E Site 1 mutation completely abrogated the interaction even at the substantial concentrations used in our in vitro association experiments (Fig. 4b). In COS cells the full-length NSP3 L623E and L469R mutants also efficiently disrupted the complex with full-length p130Cas when the two proteins were co-expressed (Fig. 4c). BCAR3 residues Leu744 and Arg748 correspond to Leu623 and Arg627 in NSP3, and when mutated also effectively disrupted the association with p130Cas in COS cells confirming that different NSP proteins utilize a conserved p130Cas binding mode (Fig. 4d). Mutation of Arg743 in mouse BCAR3, corresponding to Arg748 in human BCAR3, has also been independently confirmed to disrupt p130Cas binding in recent studies16,33. However, our in vitro results suggest that mutating Leu623 in NSP3 more effectively abrogates binding and therefore represents a more potent probe for dissecting NSP3–Cas signaling modules. The potency of Leu623 is in good agreement with its location in the hydrophobic core of the Site 1 interaction, in close proximity to the hydrophobic pair Phe794 and Leu787 of p130Cas (Fig. 4a). In contrast, the salt bridge that Arg627 forms with Asp797 of p130Cas is more peripheral to the interface.We also examined interface mutations in p130Cas for their ability to counteract complex formation with NSP proteins. Based on structural insight, we chose Leu787, Phe794 and Asp797 as promising candidates for mutagenesis. Individual mutations of each of these residues (L787E, F794R and D797R) weakened the in vitro association of the p130Cas FAT domain with the NSP3 Cdc25-homology domain, while combinations of two mutations virtually abolished complex formation (Fig. 4e, Supplementary Fig. 7b). In addition, the single p130Cas L787E and F794R mutations markedly reduced association between full-length p130Cas and NSP3 co-expressed in COS cells, further confirming features of the NSP3–p130Cas complex in the cellular context (Fig. 4f).To examine the functional effects of targeted disruption of the NSP3–p130Cas module on cell migration, we tested the NSP3 L623E mutant in a Transwell migration assay. NSP3 has previously been shown to promote COS cell migration towards EGF as a chemoattractant in a Cas-dependent manner27. The specific interface mutant L623E abrogated the NSP3-dependent enhancement of COS cell migration towards epidermal growth factor (EGF). Cells expressing this mutant showed migratory ability towards EGF similar to that of cells transfected with empty vector control (Fig. 4g,h). This emphasizes the role of NSP–Cas module formation in linking cell motility and growth factor signaling, while at the same time demonstrating the potency of structural interface mutations for investigation of these signaling nodes.

Biochemistry and biology of NSP–Cas complexes

To further verify our structural findings, we performed extensive biochemical and biological mutational studies. We designed structure-based mutations to disrupt binding elements while conserving the integrity of the protein domains (Fig. 4a). All NSP3, BCAR3 and p130Cas mutant domains showed excellent expression and solubility suitable for binding assays. The NSP3 R627E Site 1 mutation or L469R Site 2 mutation reduced complex formation with the p130Cas FAT domain in vitro, indicating that they substantially weaken the tight interaction (Fig. 4b, Supplementary Fig. 7b). In addition, the NSP3 L623E Site 1 mutation completely abrogated the interaction even at the substantial concentrations used in our in vitro association experiments (Fig. 4b). In COS cells the full-length NSP3 L623E and L469R mutants also efficiently disrupted the complex with full-length p130Cas when the two proteins were co-expressed (Fig. 4c). BCAR3 residues Leu744 and Arg748 correspond to Leu623 and Arg627 in NSP3, and when mutated also effectively disrupted the association with p130Cas in COS cells confirming that different NSP proteins utilize a conserved p130Cas binding mode (Fig. 4d). Mutation of Arg743 in mouse BCAR3, corresponding to Arg748 in human BCAR3, has also been independently confirmed to disrupt p130Cas binding in recent studies16,33. However, our in vitro results suggest that mutating Leu623 in NSP3 more effectively abrogates binding and therefore represents a more potent probe for dissecting NSP3–Cas signaling modules. The potency of Leu623 is in good agreement with its location in the hydrophobic core of the Site 1 interaction, in close proximity to the hydrophobic pair Phe794 and Leu787 of p130Cas (Fig. 4a). In contrast, the salt bridge that Arg627 forms with Asp797 of p130Cas is more peripheral to the interface.

We also examined interface mutations in p130Cas for their ability to counteract complex formation with NSP proteins. Based on structural insight, we chose Leu787, Phe794 and Asp797 as promising candidates for mutagenesis. Individual mutations of each of these residues (L787E, F794R and D797R) weakened the in vitro association of the p130Cas FAT domain with the NSP3 Cdc25-homology domain, while combinations of two mutations virtually abolished complex formation (Fig. 4e, Supplementary Fig. 7b). In addition, the single p130Cas L787E and F794R mutations markedly reduced association between full-length p130Cas and NSP3 co-expressed in COS cells, further confirming features of the NSP3–p130Cas complex in the cellular context (Fig. 4f).

To examine the functional effects of targeted disruption of the NSP3–p130Cas module on cell migration, we tested the NSP3 L623E mutant in a Transwell migration assay. NSP3 has previously been shown to promote COS cell migration towards EGF as a chemoattractant in a Cas-dependent manner27. The specific interface mutant L623E abrogated the NSP3-dependent enhancement of COS cell migration towards epidermal growth factor (EGF). Cells expressing this mutant showed migratory ability towards EGF similar to that of cells transfected with empty vector control (Fig. 4g,h). This emphasizes the role of NSP–Cas module formation in linking cell motility and growth factor signaling, while at the same time demonstrating the potency of structural interface mutations for investigation of these signaling nodes.

Promiscuous yet class-specific NSP–Cas interactionsThe structures of BCAR3, NSP3 and p130Cas provide not only the first insight into the C-terminal regions of NSP and Cas proteins and their interactions, but also a basis upon which to apply features depicted in the wider context of NSP and Cas family signaling. To this end, we created molecular models for the C-terminal domains of the remaining family members based on sequence profile alignments and the crystal structures solved in this work. Mapping of primary interface residues on these models shows that residues involved in NSP–Cas complex-formation are clearly conserved within each family (Fig. 5a). In line with this degree of conservation are residues that determine the core fold of NSP and Cas family C-terminal domains (Supplementary Fig. 8), whereas remaining surface features are considerably more divergent. This illustrates that all three NSP family proteins have pervasively repurposed an enzymatic fold as a recruitment module, while Cas family proteins share a C-terminal FAT domain that accommodates the extended and tight binding interface observed in the NSP3–p130Cas complex. This extended interface, built upon the Site 1 and Site 2 binding motifs, is elegantly specific to NSP–Cas interactions yet allows different combinations of high-affinity mechanical linkages that integrate heterogeneous signaling pathways between the two families of proteins.

Promiscuous yet class-specific NSP–Cas interactions

The structures of BCAR3, NSP3 and p130Cas provide not only the first insight into the C-terminal regions of NSP and Cas proteins and their interactions, but also a basis upon which to apply features depicted in the wider context of NSP and Cas family signaling. To this end, we created molecular models for the C-terminal domains of the remaining family members based on sequence profile alignments and the crystal structures solved in this work. Mapping of primary interface residues on these models shows that residues involved in NSP–Cas complex-formation are clearly conserved within each family (Fig. 5a). In line with this degree of conservation are residues that determine the core fold of NSP and Cas family C-terminal domains (Supplementary Fig. 8), whereas remaining surface features are considerably more divergent. This illustrates that all three NSP family proteins have pervasively repurposed an enzymatic fold as a recruitment module, while Cas family proteins share a C-terminal FAT domain that accommodates the extended and tight binding interface observed in the NSP3–p130Cas complex. This extended interface, built upon the Site 1 and Site 2 binding motifs, is elegantly specific to NSP–Cas interactions yet allows different combinations of high-affinity mechanical linkages that integrate heterogeneous signaling pathways between the two families of proteins.

DISCUSSIONNSP and Cas family proteins interact with each other to form multidomain signaling modules. Members from both families have been linked to pathological processes, such as breast cancer antiestrogen resistance2,34 or metastasis in melanoma and lung cancer35,36. Analysis of a NSP3 knockout mouse line also suggests an involvement in developmental disorders such as Kallmann syndrome14. Here we provide the first-in-class structures of the unbound BCAR3 Cdc25-homology domain and the complex of the NSP3 Cdc25-homology domain with the FAT domain of the scaffolding protein p130Cas, as well as comprehensive biochemical and cell-based analyses of these regions.The Cdc25-homology domain is typically found in exchange factors for Ras-type small GTPases, which activate their target GTPases by catalyzing nucleotide exchange. Since the GTP-GDP dependent cycling of Ras proteins is essential for many aspects of normal cellular physiology and plays a critical role in diseases such as cancer, Cdc25-homology domains have been the focus of intense investigation. Several elegant structural studies have revealed the molecular mechanisms underlying nucleotide exchange by Cdc25-homology domains17,19,20, including their regulation by additional interacting domains18,37 and other proteins including Ras itself38. In contrast to these catalytically active Cdc25-homology domains, the BCAR3 Cdc25-homology domain adopts an unprecedented closed conformation with nearly all functional elements completely remodeled, rendering it virtually incapable of performing canonical exchange factor function. Moreover, the structure of the NSP3–p130Cas complex shows that the same closed conformation is essential for a tight interaction with the FAT domain of p130Cas. Residues at the interface between NSP3 and p130Cas partially overlap with the interface suggested in a recent report utilizing small-angle X-ray scattering (SAXS) to study the BCAR3–p130Cas complex16. However, this SAXS model assumes the BCAR3 Cdc25-homology domain to adopt a classical GEF-type fold similar to SOS, rather than the closed form now observed in the crystal structures of both isolated BCAR3 and NSP3 bound to p130Cas. The complete rearrangement of the helical hairpin in NSP proteins markedly modifies the Cdc25-homology domain and in effect creates an entirely new binding surface. Consequently, NSP proteins employ an aberrant enzymatic domain as an adaptor to link two multidomain signaling proteins.Alignments of NSP proteins and Cas proteins reveal that both families (except the more distantly related CASS4) have preserved the key residues that are critical for their interaction (Fig. 5a and Supplementary Fig. 8). This analysis explains how NSP and Cas proteins are able to tightly associate in a promiscuous but class-specific manner11,12,39. As a consequence, different pairwise combinations of NSP and Cas proteins can exploit functional differences such as divergent phosphorylation sites characteristic of the particular NSP–Cas protein module3,6,10,27 (Fig. 5b). In such a manner the signaling outcome can be fine-tuned by the particular NSP–Cas signaling nodes present in a cell type. This class-specific promiscuity represents an effective structural mechanism to achieve sensitive regulation of cellular signaling networks by NSP–Cas modules.In summary, the novel closed Cdc25-homology domain conformation of NSP proteins represents a unique adaption enabling tight interaction with the C-terminal FAT domain of Cas proteins. Besides revealing a new structural paradigm used by the NSP class of adaptor complexes, this work also provides a new basis for investigating the role of NSP–Cas signaling modules in cell migration, invasiveness and other key physiological and pathological processes.

NSP and Cas family proteins interact with each other to form multidomain signaling modules. Members from both families have been linked to pathological processes, such as breast cancer antiestrogen resistance2,34 or metastasis in melanoma and lung cancer35,36. Analysis of a NSP3 knockout mouse line also suggests an involvement in developmental disorders such as Kallmann syndrome14. Here we provide the first-in-class structures of the unbound BCAR3 Cdc25-homology domain and the complex of the NSP3 Cdc25-homology domain with the FAT domain of the scaffolding protein p130Cas, as well as comprehensive biochemical and cell-based analyses of these regions.

The Cdc25-homology domain is typically found in exchange factors for Ras-type small GTPases, which activate their target GTPases by catalyzing nucleotide exchange. Since the GTP-GDP dependent cycling of Ras proteins is essential for many aspects of normal cellular physiology and plays a critical role in diseases such as cancer, Cdc25-homology domains have been the focus of intense investigation. Several elegant structural studies have revealed the molecular mechanisms underlying nucleotide exchange by Cdc25-homology domains17,19,20, including their regulation by additional interacting domains18,37 and other proteins including Ras itself38. In contrast to these catalytically active Cdc25-homology domains, the BCAR3 Cdc25-homology domain adopts an unprecedented closed conformation with nearly all functional elements completely remodeled, rendering it virtually incapable of performing canonical exchange factor function. Moreover, the structure of the NSP3–p130Cas complex shows that the same closed conformation is essential for a tight interaction with the FAT domain of p130Cas. Residues at the interface between NSP3 and p130Cas partially overlap with the interface suggested in a recent report utilizing small-angle X-ray scattering (SAXS) to study the BCAR3–p130Cas complex16. However, this SAXS model assumes the BCAR3 Cdc25-homology domain to adopt a classical GEF-type fold similar to SOS, rather than the closed form now observed in the crystal structures of both isolated BCAR3 and NSP3 bound to p130Cas. The complete rearrangement of the helical hairpin in NSP proteins markedly modifies the Cdc25-homology domain and in effect creates an entirely new binding surface. Consequently, NSP proteins employ an aberrant enzymatic domain as an adaptor to link two multidomain signaling proteins.

Alignments of NSP proteins and Cas proteins reveal that both families (except the more distantly related CASS4) have preserved the key residues that are critical for their interaction (Fig. 5a and Supplementary Fig. 8). This analysis explains how NSP and Cas proteins are able to tightly associate in a promiscuous but class-specific manner11,12,39. As a consequence, different pairwise combinations of NSP and Cas proteins can exploit functional differences such as divergent phosphorylation sites characteristic of the particular NSP–Cas protein module3,6,10,27 (Fig. 5b). In such a manner the signaling outcome can be fine-tuned by the particular NSP–Cas signaling nodes present in a cell type. This class-specific promiscuity represents an effective structural mechanism to achieve sensitive regulation of cellular signaling networks by NSP–Cas modules.

In summary, the novel closed Cdc25-homology domain conformation of NSP proteins represents a unique adaption enabling tight interaction with the C-terminal FAT domain of Cas proteins. Besides revealing a new structural paradigm used by the NSP class of adaptor complexes, this work also provides a new basis for investigating the role of NSP–Cas signaling modules in cell migration, invasiveness and other key physiological and pathological processes.

METHODSMethods and any associated references are available in the online version of the paper at http://www.nature.com/nsmb/.

Methods and any associated references are available in the online version of the paper at http://www.nature.com/nsmb/.

Supplementary Material12

Supplementary Material
