---
pmid: '32788727'
title: Structural basis of salicylic acid perception by Arabidopsis NPR proteins.
authors:
- Wang W
- Withers J
- Li H
- Zwack PJ
- Rusnac DV
- Shi H
- Liu L
- Yan S
- Hinds TR
- Guttman M
- Dong X
- Zheng N
journal: Nature
year: '2020'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC7554156
doi: 10.1038/s41586-020-2596-y
---

# Structural basis of salicylic acid perception by Arabidopsis NPR proteins.
**Authors:** Wang W, Withers J, Li H, Zwack PJ, Rusnac DV, Shi H, Liu L, Yan S, Hinds TR, Guttman M, Dong X, Zheng N
**Journal:** Nature (2020)
**DOI:** [10.1038/s41586-020-2596-y](https://doi.org/10.1038/s41586-020-2596-y)
**PMC:** [PMC7554156](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7554156/)

## Abstract

1. Nature. 2020 Oct;586(7828):311-316. doi: 10.1038/s41586-020-2596-y. Epub 2020 
Aug 12.

Structural basis of salicylic acid perception by Arabidopsis NPR proteins.

Wang W(#)(1)(2), Withers J(#)(3), Li H(#)(1)(4), Zwack PJ(3), Rusnac DV(1), Shi 
H(1), Liu L(3)(5), Yan S(3)(6), Hinds TR(1), Guttman M(7), Dong X(8), Zheng 
N(9).

Author information:
(1)Department of Pharmacology, Howard Hughes Medical Institute, University of 
Washington, Seattle, WA, USA.
(2)State Key Laboratory of Crop Stress Adaptation and Improvement, School of 
Life Sciences, Henan University, Kaifeng, China.
(3)Department of Biology, Howard Hughes Medical Institute, Duke University, 
Durham, NC, USA.
(4)Kronos Bio, Inc., Cambridge, MA, USA.
(5)Key Laboratory of Plant Development and Environment Adaptation Biology, 
Ministry of Education, School of Life Sciences, Shandong University, Qingdao, 
China.
(6)College of Life Science and Technology, Huazhong Agricultural University, 
Wuhan, China.
(7)Department of Medicinal Chemistry, University of Washington, Seattle, WA, 
USA.
(8)Department of Biology, Howard Hughes Medical Institute, Duke University, 
Durham, NC, USA. xdong@duke.edu.
(9)Department of Pharmacology, Howard Hughes Medical Institute, University of 
Washington, Seattle, WA, USA. nzheng@uw.edu.
(#)Contributed equally

Comment in
    Nat Plants. 2020 Oct;6(10):1197-1198. doi: 10.1038/s41477-020-0741-0.

Salicylic acid (SA) is a plant hormone that is critical for resistance to 
pathogens1-3. The NPR proteins have previously been identified as SA 
receptors4-10, although how they perceive SA and coordinate hormonal signalling 
remain unknown. Here we report the mapping of the SA-binding core of Arabidopsis 
thaliana NPR4 and its ligand-bound crystal structure. The SA-binding core domain 
of NPR4 refolded with SA adopts an α-helical fold that completely buries SA in 
its hydrophobic core. The lack of a ligand-entry pathway suggests that SA 
binding involves a major conformational remodelling of the SA-binding core of 
NPR4, which we validated using hydrogen-deuterium-exchange mass spectrometry 
analysis of the full-length protein and through SA-induced disruption of 
interactions between NPR1 and NPR4. We show that, despite the two proteins 
sharing nearly identical hormone-binding residues, NPR1 displays minimal 
SA-binding activity compared to NPR4. We further identify two surface residues 
of the SA-binding core, the mutation of which can alter the SA-binding ability 
of NPR4 and its interaction with NPR1. We also demonstrate that expressing a 
variant of NPR4 that is hypersensitive to SA could enhance SA-mediated basal 
immunity without compromising effector-triggered immunity, because the ability 
of this variant to re-associate with NPR1 at high levels of SA remains intact. 
By revealing the structural mechanisms of SA perception by NPR proteins, our 
work paves the way for future investigation of the specific roles of these 
proteins in SA signalling and their potential for engineering plant immunity.

DOI: 10.1038/s41586-020-2596-y
PMCID: PMC7554156
PMID: 32788727 [Indexed for MEDLINE]

Conflict of interest statement: Competing Interests. N.Z. is a cofounder of Coho 
Therapeutics and a Scientific Advisory Board member of Kymera Therapeutics. X. 
D. is a cofounder of Upstream Biotechnology, Inc., and a Scientific Advisory 
Board member of Inari Agriculture.

## Full Text

Salicylic acid (SA) is a pivotal defence hormone in plants that accumulates upon pathogen invasion to trigger systemic acquired resistance (SAR)1–3. Genetic studies of SA-insensitive mutants in Arabidopsis thaliana identified NONEXPRESSOR of PATHOGENESIS-RELATED GENES 1 (NPR1) as a transcriptional coactivator indispensable for expression of antimicrobial pathogenesis-related (PR) genes and broad-spectrum disease resistance4,5. In contrast to NPR1, NPR3 and NPR4 are negative regulators of plant defence, with the npr3 npr4 double mutants showing elevated PR gene expression and basal resisance6,7. Based on their high affinities towards SA, NPR3 and NPR4 have been established as SA receptors7,8. Interestingly, NPR1 has also been reported to bind SA, although the activity seems to vary in different studies8–10. NPR proteins share an N-terminal broad-complex, tram track, and bric-a-brac/poxvirus, zinc finger (BTB/POZ) domain, which is commonly found in the substrate receptor subunits of the CULLIN3-RBX1 ubiquitin ligase complexes (CRL3s)11. Consistent with their negative role in SA signalling, NPR3 and NPR4 act as CRL3 substrate receptors for NPR1 polyubiquitination and degradation7,12. Because their interactions with NPR1 are sensitive to SA, NPR3 and NPR4 regulate SA-mediated gene expression by controlling NPR1 stability7,13. A recent study has suggested that NPR3 and NPR4 might also function independently from NPR1, acting as transcriptional co-repressors, whose activities are blocked by SA8. A better understanding of how NPR proteins regulate plant immunity in response to SA calls for detailed analysis of their structure-function relationships.

Mapping of the SA-binding core domainBesides the N-terminal BTB domain, all NPR proteins share a central ankyrin repeat (ANK) domain and a C-terminal (CT) domain (Fig. 1a)4,14. Despite our extensive efforts, structure determination of the full-length NPR proteins has been hampered by the poor resolution of single crystal X-ray diffraction. To overcome this problem, we performed limited proteolytic digestion of different NPR4 constructs to map the minimal SA-binding core (SBC) responsible for the SA-sensitive digestion pattern. We identified amino acids 373 to 516 within the NPR4 CT domain as the SBC (Fig. 1a, b, Extended Data Fig. 1a–f, Supplementary Discussion). Using hydrogen-deuterium exchange mass spectrometry (HDX-MS) of the full-length protein, we confirmed that the SBC is the predominant region of NPR4 that has a SA-sensitive deuterium uptake profile (Fig. 1c, d, Extended Data Fig. 2, Supplementary Table). The critical role of SBC in sensing SA was further manifested by the striking enhancement of SA binding when the CT domain or SBC sequences of NPR4 was used to replace the responding regions of NPR3 (Fig. 1e, f). Although isolated NPR4-SBC was mostly insoluble when overexpressed in E. coli, we were able to purify the fragment under denaturing conditions and refold the polypeptide in the presence of SA (Extended Data Fig. 1g–i). We subsequently crystallized NPR4-SBC and determined its structure at 2.3 Å resolution (Extended Data Table 1).

Mapping of the SA-binding core domain

Besides the N-terminal BTB domain, all NPR proteins share a central ankyrin repeat (ANK) domain and a C-terminal (CT) domain (Fig. 1a)4,14. Despite our extensive efforts, structure determination of the full-length NPR proteins has been hampered by the poor resolution of single crystal X-ray diffraction. To overcome this problem, we performed limited proteolytic digestion of different NPR4 constructs to map the minimal SA-binding core (SBC) responsible for the SA-sensitive digestion pattern. We identified amino acids 373 to 516 within the NPR4 CT domain as the SBC (Fig. 1a, b, Extended Data Fig. 1a–f, Supplementary Discussion). Using hydrogen-deuterium exchange mass spectrometry (HDX-MS) of the full-length protein, we confirmed that the SBC is the predominant region of NPR4 that has a SA-sensitive deuterium uptake profile (Fig. 1c, d, Extended Data Fig. 2, Supplementary Table). The critical role of SBC in sensing SA was further manifested by the striking enhancement of SA binding when the CT domain or SBC sequences of NPR4 was used to replace the responding regions of NPR3 (Fig. 1e, f). Although isolated NPR4-SBC was mostly insoluble when overexpressed in E. coli, we were able to purify the fragment under denaturing conditions and refold the polypeptide in the presence of SA (Extended Data Fig. 1g–i). We subsequently crystallized NPR4-SBC and determined its structure at 2.3 Å resolution (Extended Data Table 1).

Crystal structure of SA-bound NPR4-SBCThe NPR4-SBC structure consists of five closely packed α-helices, with the C-terminal four-helix bundle (4HB)-like fold harbouring the SA-binding site (Fig. 2a). We named these four SA-contacting helices αSC1–αSC4. These four SC helices resemble two interlocked “V”-shaped nutcrackers with SA sequestered in between. Their assembly is further stabilized by the N-terminal α-helix (αN) on the side. Notably, the electron density is missing for the entire loop linking αSC2 and αSC3, which is presumably disordered in the crystal, a feature consistent with its rapid deuterium exchange in solution independent of SA (Figs. 1c, d, 2a, Extended Data Fig. 2b). This flexible loop includes the VDLNETP sequence (Extended Data Fig. 3a), which has been suggested as a putative ethylene-responsive element binding factor-associated amphipathic repression (EAR) motif that mediates the repressor function of NPR48.The SA-binding site is located at the tapered end of the NPR4-SBC 4HB and enclosed by residues from all four SC helices (Fig. 2a), which recognize the hormone from all angles. The bottom of the SA-binding pocket is formed by Glu430 of αSC2 and Cys499 of αSC4, which are supported by two opposing phenylalanine residues, Phe427 and Phe496, from αSC1 and αSC3, respectively (Fig. 2b, c, Extended Data Fig. 3b, c). With the carboxyl group of the Glu430 side chain pointing toward the solvent, these four residues construct a hydrophobic environment, accommodating the benzene moiety of SA. In the middle layer, the planar aromatic ring of the hormone is sandwiched by two small NPR4 residues, Ala423 of αSC1 and Gly492 of αSC3, while its edges are surrounded by four additional hydrophobic residues from the two short SC helices (Ala431 and Ala434 of αSC2 and Tyr500 and Leu503 of αSC4). By crossing each other right above the hormone, αSC1 and αSC3 seal the SA-binding pocket at the top with two face-to-face valine residues, Val420 and Val489 (Fig. 2a–c). They are joined on the side by Arg419 and Thr488, which introduce polar groups to the SA-binding site. As the hallmark of the SA-binding pocket, Arg419 neutralizes the carboxyl group of the hormone with a salt bridge and a hydrogen bond (Fig. 2b, Extended Data Fig. 3b, c). As a whole, the SA-binding pocket is characterized by its central location within the receptor SBC domain and its overall hydrophobicity, which are two properties shared by the high affinity SA-binding pocket of the methyl-SA esterase SABP215, and a strategically situated arginine residue. Interestingly, the SA-binding pocket completely buries the hormone inside an internal cavity at the tapered end of the 4HB-like fold, leaving no gap for the ligand to enter or escape (Fig. 2c, Extended Data Fig. 3d). This precise location of the SA-binding site is supported by its strongest protection by SA against deuterium exchange (Fig. 2d). To reconcile the ability of SA to access this site in the full-length protein as detected by HDX-MS (Fig. 1c, d), we postulate that the crystal structure has captured the closed SA-bound conformation of NPR4-SBC, which is stabilized by crystal packing in the absence of the rest of the protein.To validate the structure in the context of the full-length NPR4 protein, we purified a series of NPR4 mutants with the SA-contacting residues individually mutated. As expected, the majority of these mutants lost SA-binding activity (Fig. 2e). Importantly, neither NPR4 could tolerate a change at Arg419 (R419K, R419Q, and R419A) (Fig. 2e), nor NPR3 at Arg428 (Extended Data Fig. 3e), underscoring the critical role of the residue in binding SA. In fact, missense mutation of this arginine confers the SA-insensitive phenotype to the Arabidopsis npr4–4D mutant8. One of the few outliers of losing SA-binding activity, A434V, can be rationalized by its nearby empty space in the SA-binding pocket, whose size also potentially explains the affinity of the SA receptor toward the bulkier SA analogue, benzothiadiazole (BTH) (Extended Data Fig. 3d). Thr488 likely contributes to SA binding by van der Waals packing, despite its close proximity to the 2-hydroxyl group of SA (3.7 Å) (Fig. 2b). T488A showed a reduction in SA binding while T488V had little effect (Fig. 2e).

Crystal structure of SA-bound NPR4-SBC

The NPR4-SBC structure consists of five closely packed α-helices, with the C-terminal four-helix bundle (4HB)-like fold harbouring the SA-binding site (Fig. 2a). We named these four SA-contacting helices αSC1–αSC4. These four SC helices resemble two interlocked “V”-shaped nutcrackers with SA sequestered in between. Their assembly is further stabilized by the N-terminal α-helix (αN) on the side. Notably, the electron density is missing for the entire loop linking αSC2 and αSC3, which is presumably disordered in the crystal, a feature consistent with its rapid deuterium exchange in solution independent of SA (Figs. 1c, d, 2a, Extended Data Fig. 2b). This flexible loop includes the VDLNETP sequence (Extended Data Fig. 3a), which has been suggested as a putative ethylene-responsive element binding factor-associated amphipathic repression (EAR) motif that mediates the repressor function of NPR48.

The SA-binding site is located at the tapered end of the NPR4-SBC 4HB and enclosed by residues from all four SC helices (Fig. 2a), which recognize the hormone from all angles. The bottom of the SA-binding pocket is formed by Glu430 of αSC2 and Cys499 of αSC4, which are supported by two opposing phenylalanine residues, Phe427 and Phe496, from αSC1 and αSC3, respectively (Fig. 2b, c, Extended Data Fig. 3b, c). With the carboxyl group of the Glu430 side chain pointing toward the solvent, these four residues construct a hydrophobic environment, accommodating the benzene moiety of SA. In the middle layer, the planar aromatic ring of the hormone is sandwiched by two small NPR4 residues, Ala423 of αSC1 and Gly492 of αSC3, while its edges are surrounded by four additional hydrophobic residues from the two short SC helices (Ala431 and Ala434 of αSC2 and Tyr500 and Leu503 of αSC4). By crossing each other right above the hormone, αSC1 and αSC3 seal the SA-binding pocket at the top with two face-to-face valine residues, Val420 and Val489 (Fig. 2a–c). They are joined on the side by Arg419 and Thr488, which introduce polar groups to the SA-binding site. As the hallmark of the SA-binding pocket, Arg419 neutralizes the carboxyl group of the hormone with a salt bridge and a hydrogen bond (Fig. 2b, Extended Data Fig. 3b, c). As a whole, the SA-binding pocket is characterized by its central location within the receptor SBC domain and its overall hydrophobicity, which are two properties shared by the high affinity SA-binding pocket of the methyl-SA esterase SABP215, and a strategically situated arginine residue. Interestingly, the SA-binding pocket completely buries the hormone inside an internal cavity at the tapered end of the 4HB-like fold, leaving no gap for the ligand to enter or escape (Fig. 2c, Extended Data Fig. 3d). This precise location of the SA-binding site is supported by its strongest protection by SA against deuterium exchange (Fig. 2d). To reconcile the ability of SA to access this site in the full-length protein as detected by HDX-MS (Fig. 1c, d), we postulate that the crystal structure has captured the closed SA-bound conformation of NPR4-SBC, which is stabilized by crystal packing in the absence of the rest of the protein.

To validate the structure in the context of the full-length NPR4 protein, we purified a series of NPR4 mutants with the SA-contacting residues individually mutated. As expected, the majority of these mutants lost SA-binding activity (Fig. 2e). Importantly, neither NPR4 could tolerate a change at Arg419 (R419K, R419Q, and R419A) (Fig. 2e), nor NPR3 at Arg428 (Extended Data Fig. 3e), underscoring the critical role of the residue in binding SA. In fact, missense mutation of this arginine confers the SA-insensitive phenotype to the Arabidopsis npr4–4D mutant8. One of the few outliers of losing SA-binding activity, A434V, can be rationalized by its nearby empty space in the SA-binding pocket, whose size also potentially explains the affinity of the SA receptor toward the bulkier SA analogue, benzothiadiazole (BTH) (Extended Data Fig. 3d). Thr488 likely contributes to SA binding by van der Waals packing, despite its close proximity to the 2-hydroxyl group of SA (3.7 Å) (Fig. 2b). T488A showed a reduction in SA binding while T488V had little effect (Fig. 2e).

Disruption of NPR1-NPR4 interaction by SAThe lack of any ligand entry pathway in the NPR4-SBC-SA complex structure indicates that the apo form of the receptor must adopt a different conformation, in which its ligand-binding site is accessible to free SA. Such an obligated structural rearrangement of NPR4-SBC by SA is corroborated by the prominent SA-triggered changes of its deuterium exchange rate (Fig. 1d). To investigate the functional consequences of NPR4-SBC remodelling by SA, we first validated the NPR1-NPR4 binding in a GST pull-down assay and quantified the interaction in an AlphaScreen-based competition assay, which revealed a strong affinity between the two proteins (Fig. 2f, g). In a dose-dependent manner, SA, but not the inactive benzoic acid (BA), can block the NPR1-NPR4 interaction with a potency of ~390 nM (Fig. 2h). We subsequently prepared three representative NPR4 mutants, R419Q, T488V, and V489A, all of which interacted with NPR1 in the absence of SA (Fig. 2i). As expected, SA was able to weaken the interaction between NPR1 and npr4T488V, which retains an intact SA-binding activity (Fig. 2e, i). By contrast, the disruptive effect of SA was diminished in the R419Q and V489A mutants, which are defective in SA binding. These results support the notion that SA is able to regulate NPR4-NPR1 interaction by inducing conformational changes in NPR4-SBC.

Disruption of NPR1-NPR4 interaction by SA

The lack of any ligand entry pathway in the NPR4-SBC-SA complex structure indicates that the apo form of the receptor must adopt a different conformation, in which its ligand-binding site is accessible to free SA. Such an obligated structural rearrangement of NPR4-SBC by SA is corroborated by the prominent SA-triggered changes of its deuterium exchange rate (Fig. 1d). To investigate the functional consequences of NPR4-SBC remodelling by SA, we first validated the NPR1-NPR4 binding in a GST pull-down assay and quantified the interaction in an AlphaScreen-based competition assay, which revealed a strong affinity between the two proteins (Fig. 2f, g). In a dose-dependent manner, SA, but not the inactive benzoic acid (BA), can block the NPR1-NPR4 interaction with a potency of ~390 nM (Fig. 2h). We subsequently prepared three representative NPR4 mutants, R419Q, T488V, and V489A, all of which interacted with NPR1 in the absence of SA (Fig. 2i). As expected, SA was able to weaken the interaction between NPR1 and npr4T488V, which retains an intact SA-binding activity (Fig. 2e, i). By contrast, the disruptive effect of SA was diminished in the R419Q and V489A mutants, which are defective in SA binding. These results support the notion that SA is able to regulate NPR4-NPR1 interaction by inducing conformational changes in NPR4-SBC.

SA-binding activity of NPR1Arabidopsis NPR4 and NPR1 share 38.1% sequence identity in their SBC regions (Extended Data Fig. 4a). Based on sequence alignment, all 14 amino acids outlining the NPR4 SA-binding pocket are highly conserved between NPR4 and NPR1 orthologues and paralogues (Extended Data Fig. 3a). Notably, missense mutations of four of these residues have been identified in Arabidopsis npr1 alleles that are insensitive to BTH16. The SA-insensitive phenotype of the Arabidopsis nim1–4 mutant has also been attributed to a missense mutation of the NPR1 arginine residue (Arg432)5, which is equivalent to Arg419 of NPR4. In conjunction with our structure data, these lines of genetic and bioinformatic evidence suggest that NPR1 shares the ability and structural mechanism of SA recognition with NPR4.To re-evaluate SA binding by NPR1, we adopted a recently published procedure8,10 and established a dose-response curve for SA binding by MBP-NPR1 (Extended Data Fig. 4b, c). The SA binding signals in these experiments, nevertheless, were profoundly weaker than that of NPR4. Based on the experimental conditions, we estimated that less than 0.02% of the total MBP-NPR1 protein in the sample was competent for binding the hormone (Fig. 3a). By contrast, ~ 8% of NPR4 was occupied by SA at the same saturating concentration. Akin to NPR4, the isolated SBC fragment of NPR1 was insoluble when overexpressed in E. coli. Upon co-expression, we were able to co-purify soluble NPR1-SBC with NIMIN-2 (Fig. 3b), a previously identified NPR1-binding protein17,18, and detect an Arg432-dependent SA binding activity in the presence of NIMIN-2 (Fig. 3c). Thus, the SA-binding region of NPR1 can be mapped to its SBC, despite the absence of a cysteine residue (Cys529) that has been reported necessary for hormone binding9. Similar to the full-length protein, the predominant population of the NIMIN-2-bound NPR1-SBC fragment was dormant in the binding assay. Together, these results confirmed that NPR1 is equipped with an SBC module capable of sensing SA. However, NPR1 and NPR4 are categorically distinct from each other by the disparity of their SA-binding activities.

SA-binding activity of NPR1

Arabidopsis NPR4 and NPR1 share 38.1% sequence identity in their SBC regions (Extended Data Fig. 4a). Based on sequence alignment, all 14 amino acids outlining the NPR4 SA-binding pocket are highly conserved between NPR4 and NPR1 orthologues and paralogues (Extended Data Fig. 3a). Notably, missense mutations of four of these residues have been identified in Arabidopsis npr1 alleles that are insensitive to BTH16. The SA-insensitive phenotype of the Arabidopsis nim1–4 mutant has also been attributed to a missense mutation of the NPR1 arginine residue (Arg432)5, which is equivalent to Arg419 of NPR4. In conjunction with our structure data, these lines of genetic and bioinformatic evidence suggest that NPR1 shares the ability and structural mechanism of SA recognition with NPR4.

To re-evaluate SA binding by NPR1, we adopted a recently published procedure8,10 and established a dose-response curve for SA binding by MBP-NPR1 (Extended Data Fig. 4b, c). The SA binding signals in these experiments, nevertheless, were profoundly weaker than that of NPR4. Based on the experimental conditions, we estimated that less than 0.02% of the total MBP-NPR1 protein in the sample was competent for binding the hormone (Fig. 3a). By contrast, ~ 8% of NPR4 was occupied by SA at the same saturating concentration. Akin to NPR4, the isolated SBC fragment of NPR1 was insoluble when overexpressed in E. coli. Upon co-expression, we were able to co-purify soluble NPR1-SBC with NIMIN-2 (Fig. 3b), a previously identified NPR1-binding protein17,18, and detect an Arg432-dependent SA binding activity in the presence of NIMIN-2 (Fig. 3c). Thus, the SA-binding region of NPR1 can be mapped to its SBC, despite the absence of a cysteine residue (Cys529) that has been reported necessary for hormone binding9. Similar to the full-length protein, the predominant population of the NIMIN-2-bound NPR1-SBC fragment was dormant in the binding assay. Together, these results confirmed that NPR1 is equipped with an SBC module capable of sensing SA. However, NPR1 and NPR4 are categorically distinct from each other by the disparity of their SA-binding activities.

SBC surface residues affecting SA bindingTo map the sequence determinants of differential SA binding in NPR1 and NPR4, we performed domain swapping experiments, which indicated that the NPR4 CT or the structurally defined SBC domain is not the only region contributing to strong SA-binding activity (Fig. 3d, e and Extended Data Fig. 4d). Regions N-terminal to SBC, such as the ANK and BTB domains can effectively alter the ligand-binding activity of SBC in the context of the full-length NPR proteins, presumably through the SBC surface residues. Consistent with this idea, our HDX-MS analysis revealed small but detectable SA-induced structural changes in two overlapping ANK-domain peptides preceding the SBC α-N helix (Fig. 1d, Extended Data Fig. 2a). Supporting a role played by the SBC surface residues in affecting SA-binding, the NPR4 CT domain could confer a higher SA-binding activity to the more closely related NPR3 sequence (73.9%) in the chimeric protein (Fig. 1e). To identify such SBC surface residues, we performed phylogenetic analysis of angiosperm NPR CT sequences and found that Arabidopsis NPR1/NPR2-like and NPR3/NPR4-like CT sequences belong to two distinct clades (Extended Data Fig. 5). We selected six clade-specific surface residues for mutagenesis and SA binding analyses (Fig. 3f, g, Extended Data Fig. 3a).Among the three mutants with reduced SA binding, F426L, E469I and K505Q, we found that F426L had the strongest effect of reducing SA binding (Fig. 3g). As a solvent-exposed residue on the α-SC1 helix (Fig. 3f), F426 is most likely involved in an inter-domain interaction that impacts ligand binding. Similar to npr4R419Q (Fig. 2i), in vitro synthesized npr4F426L-HA maintained interaction with NPR1-FLAG in the presence of SA (Fig. 3h). By contrast, mutations of T459, T459A and T459G, in the middle of the disordered α-SC2-α-SC3 linker (Fig. 3f), increased SA binding to NPR4 by up to 50% (Fig. 3g). Unexpectedly, when T459G was combined with F426L, NPR4’s ability to bind SA was markedly enhanced by ~ 3-fold. Consistent with its augmented SA-binding activity, we found that the interaction between npr4F426L T459G and NPR1 was disrupted by 0.01 mM SA, which is 10 times lower than that required to interfere with wild-type NPR4 interaction with NPR1 (Fig. 3i). We speculate that these two mutations might have epistatically changed the conformational dynamic of NPR4, which could affect the transition between the apo- and SA-bound forms of the receptor. Our saturation binding analyses revealed that npr4F426L T459G (Kd = 17.2 ± 2.5 nM; h = 1.3) has a higher affinity and binds SA more efficiently than NPR4 (Kd = 49.9 ± 9.2 nM; h = 0.9) (Fig. 3j).

SBC surface residues affecting SA binding

To map the sequence determinants of differential SA binding in NPR1 and NPR4, we performed domain swapping experiments, which indicated that the NPR4 CT or the structurally defined SBC domain is not the only region contributing to strong SA-binding activity (Fig. 3d, e and Extended Data Fig. 4d). Regions N-terminal to SBC, such as the ANK and BTB domains can effectively alter the ligand-binding activity of SBC in the context of the full-length NPR proteins, presumably through the SBC surface residues. Consistent with this idea, our HDX-MS analysis revealed small but detectable SA-induced structural changes in two overlapping ANK-domain peptides preceding the SBC α-N helix (Fig. 1d, Extended Data Fig. 2a). Supporting a role played by the SBC surface residues in affecting SA-binding, the NPR4 CT domain could confer a higher SA-binding activity to the more closely related NPR3 sequence (73.9%) in the chimeric protein (Fig. 1e). To identify such SBC surface residues, we performed phylogenetic analysis of angiosperm NPR CT sequences and found that Arabidopsis NPR1/NPR2-like and NPR3/NPR4-like CT sequences belong to two distinct clades (Extended Data Fig. 5). We selected six clade-specific surface residues for mutagenesis and SA binding analyses (Fig. 3f, g, Extended Data Fig. 3a).

Among the three mutants with reduced SA binding, F426L, E469I and K505Q, we found that F426L had the strongest effect of reducing SA binding (Fig. 3g). As a solvent-exposed residue on the α-SC1 helix (Fig. 3f), F426 is most likely involved in an inter-domain interaction that impacts ligand binding. Similar to npr4R419Q (Fig. 2i), in vitro synthesized npr4F426L-HA maintained interaction with NPR1-FLAG in the presence of SA (Fig. 3h). By contrast, mutations of T459, T459A and T459G, in the middle of the disordered α-SC2-α-SC3 linker (Fig. 3f), increased SA binding to NPR4 by up to 50% (Fig. 3g). Unexpectedly, when T459G was combined with F426L, NPR4’s ability to bind SA was markedly enhanced by ~ 3-fold. Consistent with its augmented SA-binding activity, we found that the interaction between npr4F426L T459G and NPR1 was disrupted by 0.01 mM SA, which is 10 times lower than that required to interfere with wild-type NPR4 interaction with NPR1 (Fig. 3i). We speculate that these two mutations might have epistatically changed the conformational dynamic of NPR4, which could affect the transition between the apo- and SA-bound forms of the receptor. Our saturation binding analyses revealed that npr4F426L T459G (Kd = 17.2 ± 2.5 nM; h = 1.3) has a higher affinity and binds SA more efficiently than NPR4 (Kd = 49.9 ± 9.2 nM; h = 0.9) (Fig. 3j).

Characterization of a SA-hypersensitive mutantSince the functional significance of SA-contacting residues of SBC is validated by strong genetic data4,5,8,16, we sought to determine whether these NPR4-SBC surface residues can also impact SA-induced defence in planta. Because these residues are not conserved between the NPR1 and NPR4 clades (Extended Data Figs. 3a, 5), they are potential candidates for engineering SA receptors with variable activities. In protein degradation assays conducted using npr3 npr4 transgenic lines expressing similar amounts of wild-type and mutant NPR4, endogenous NPR1 was degraded slower in the presence of npr4F426L T459G, but faster with npr4F426L compared to the wild-type NPR4 protein (Fig. 4a, Extended Data Fig. 6a–d). These results are in full agreement with the opposite effects of these mutations on SA binding and interactions with NPR1 in vitro (Fig. 3g–i). We then examined the levels of SA-induced PR1 gene expression in these transgenic lines and found that NPR4-GFP could reduce PR1 levels in the npr3 npr4 double mutants to the wild type level, whereas two independent npr4F426L-GFP lines further diminished in PR1 induction (Fig. 4b, Extended Data Fig. 6e, f). By contrast, the npr4F426L T459G-GFP lines had a 4-fold increase in SA-induced PR1 gene expression compared to NPR4-GFP (Fig. 4b, Extended Data Fig. 6g, h). In the subsequent testing of SA-induced disease resistance, we observed that while the npr3 npr4 mutants were resistant to Pseudomonas syringae pv. maculicola ES4326 (Psm), expression of NPR4, npr4F426L, or npr4F426L T459G could all restore the susceptibility to this bacterial pathogen similar to Col-0 (Fig. 4c). Pre-treatment of plants with 0.1 mM SA produced a significant reduction in bacterial multiplication in the npr3 npr4 mutants, as well as in the wild-type NPR4 transgenic lines. However, this protective effect was lost in the npr4F426L plants, but markedly enhanced in the npr4F426L T459G plants as shown by the symptom development and bacterial titre in infected leaves (Fig. 4c, Extended Data Fig. 6i, j). These results clearly demonstrated the opposing biological impacts that the npr4F426L and npr4F426L T459G mutants had on SA-mediated immunity.Accumulation of NPR1 in npr3 npr4 enhances basal resistance to Psm, but compromises effector-triggered immunity (ETI) to Psm expressing the effector AvrRpt27. Interestingly, when infected with Psm/AvrRpt2, the npr4F426L T459G mutant showed effector-induced cell death (i.e., ion leakage) and inhibition of bacterial growth similar to NPR4 and the npr4F426L mutant (Fig. 4d, e), suggesting that it is possible to enhance SA-mediated resistance without compromising ETI through engineering of such residues. We rationalize that, despite its enhanced sensitivity to low levels of SA, npr4F426L T459G can re-associate with NPR1 at high concentrations of SA as the wild-type NPR4 through an unknown mechanism (Fig. 3i), to induce its degradation during ETI (Fig. 4f). In agreement, our in-planta protein degradation analysis demonstrates that NPR1 is destabilized in npr4F426L T459G and NPR4 plants when they are treated with 1 mM SA (Extended Data Fig. 6k). In addition to NPR1, npr4F426L T459G is most likely able to degrade JAZ proteins, which has been shown to be required for ETI19. Also interesting is the normal ETI observed for npr4F426L. This is in congruence with the CRL3 adapter model, which predicts that the mutant can constitutively remove NPR1’s inhibition on ETI.

Characterization of a SA-hypersensitive mutant

Since the functional significance of SA-contacting residues of SBC is validated by strong genetic data4,5,8,16, we sought to determine whether these NPR4-SBC surface residues can also impact SA-induced defence in planta. Because these residues are not conserved between the NPR1 and NPR4 clades (Extended Data Figs. 3a, 5), they are potential candidates for engineering SA receptors with variable activities. In protein degradation assays conducted using npr3 npr4 transgenic lines expressing similar amounts of wild-type and mutant NPR4, endogenous NPR1 was degraded slower in the presence of npr4F426L T459G, but faster with npr4F426L compared to the wild-type NPR4 protein (Fig. 4a, Extended Data Fig. 6a–d). These results are in full agreement with the opposite effects of these mutations on SA binding and interactions with NPR1 in vitro (Fig. 3g–i). We then examined the levels of SA-induced PR1 gene expression in these transgenic lines and found that NPR4-GFP could reduce PR1 levels in the npr3 npr4 double mutants to the wild type level, whereas two independent npr4F426L-GFP lines further diminished in PR1 induction (Fig. 4b, Extended Data Fig. 6e, f). By contrast, the npr4F426L T459G-GFP lines had a 4-fold increase in SA-induced PR1 gene expression compared to NPR4-GFP (Fig. 4b, Extended Data Fig. 6g, h). In the subsequent testing of SA-induced disease resistance, we observed that while the npr3 npr4 mutants were resistant to Pseudomonas syringae pv. maculicola ES4326 (Psm), expression of NPR4, npr4F426L, or npr4F426L T459G could all restore the susceptibility to this bacterial pathogen similar to Col-0 (Fig. 4c). Pre-treatment of plants with 0.1 mM SA produced a significant reduction in bacterial multiplication in the npr3 npr4 mutants, as well as in the wild-type NPR4 transgenic lines. However, this protective effect was lost in the npr4F426L plants, but markedly enhanced in the npr4F426L T459G plants as shown by the symptom development and bacterial titre in infected leaves (Fig. 4c, Extended Data Fig. 6i, j). These results clearly demonstrated the opposing biological impacts that the npr4F426L and npr4F426L T459G mutants had on SA-mediated immunity.

Accumulation of NPR1 in npr3 npr4 enhances basal resistance to Psm, but compromises effector-triggered immunity (ETI) to Psm expressing the effector AvrRpt27. Interestingly, when infected with Psm/AvrRpt2, the npr4F426L T459G mutant showed effector-induced cell death (i.e., ion leakage) and inhibition of bacterial growth similar to NPR4 and the npr4F426L mutant (Fig. 4d, e), suggesting that it is possible to enhance SA-mediated resistance without compromising ETI through engineering of such residues. We rationalize that, despite its enhanced sensitivity to low levels of SA, npr4F426L T459G can re-associate with NPR1 at high concentrations of SA as the wild-type NPR4 through an unknown mechanism (Fig. 3i), to induce its degradation during ETI (Fig. 4f). In agreement, our in-planta protein degradation analysis demonstrates that NPR1 is destabilized in npr4F426L T459G and NPR4 plants when they are treated with 1 mM SA (Extended Data Fig. 6k). In addition to NPR1, npr4F426L T459G is most likely able to degrade JAZ proteins, which has been shown to be required for ETI19. Also interesting is the normal ETI observed for npr4F426L. This is in congruence with the CRL3 adapter model, which predicts that the mutant can constitutively remove NPR1’s inhibition on ETI.

ConclusionThis study reveals the long-sought-after structural basis of SA recognition by NPR4 and provides the initial insights into the structure-function relationships of NPR proteins. Future studies will be needed to elucidate the intricate interplays between NPR proteins in SA signalling and to explore these SA receptors for engineering plant immunity.

This study reveals the long-sought-after structural basis of SA recognition by NPR4 and provides the initial insights into the structure-function relationships of NPR proteins. Future studies will be needed to elucidate the intricate interplays between NPR proteins in SA signalling and to explore these SA receptors for engineering plant immunity.

MethodsMolecular cloningThe coding sequences of Arabidopsis thaliana NPR1 (AT1G64280), NPR4 (AT4G19660) and NIMIN2 (AT3G25882) were amplified by polymerase chain reaction from an Arabidopsis cDNA library, and sub-cloned into the DH5α vector with different N-terminally fused tags and a tobacco etch virus (TEV)-cleavage site. The specific amino acid changes for the NPR1 and NPR4 point mutations were generated using the QuikChange II site-directed mutagenesis kit (Agilent). Glutathione-S-transferase (GST)-fused NPR4 coding sequence (CDS) was subcloned into the pFastBac vector and transformed to E. coli DH10Bac for making baculovirus for protein expression in insect cells. Protein domain swaps were generated by amplifying the desired regions of each CDS with primers designed to create overlapping sequences for each fragment. The DNA fragments were amplified in separate PCR reactions, processed with either a PCR clean-up kit or gel extraction kit (Bio Basic Inc.), and the desired fragments were fused by PCR using gene-specific forward and reverse primers containing attB1 and attB2 Gateway™ recombination sequences, respectively. All CDSs were recombined into pDONR207 or pDONR221 and subsequent expression vectors using the Gateway™ technology and sequenced to confirm accuracy.Protein purification and preparationThe GST-tagged NPR1, NPR4, and NPR4 fragment constructs were over-expressed in BL21 (DE3) strain. AtNPR4-GST protein for HDX experiments were expressed in insect cells according to the methods previously described20. The proteins were purified from the soluble cell lysate by glutathione affinity chromatography. After on-column tag cleavage by TEV protease at 4 °C for 16 hr or directly eluted from the affinity column, the proteins were further purified by anion exchange and size exclusion chromatography. The NPR4-SBC protein was expressed with an N-terminal 6 × His tag (His-NPR4-SBC) in BL21 (DE3) cells, first grown to OD600 nm of 0.9–1.0 at 37 °C and then induced by 0.5 mM isopropyl-β-D-thiogalactopyranoside (IPTG) at 25 °C overnight. Cells were collected and lysed in extraction buffer (50 mM Tris-HCl, pH 8.0, 500 mM NaCl, 1 mM TCEP) and pellets were harvested by centrifugation and resuspended in 50 mM Tris-HCl, pH 8.0, 500 mM NaCl, 8 M urea (denaturation buffer). Denatured His-NPR4-SBC was isolated with Ni-NTA resin and eluted using the denaturation buffer supplemented by 250 mM imidazole before being dialysed against a buffer containing 50 mM Tris-HCl, pH 8.0, 500 mM NaCl, 1 mM DTT, and 2 mM SA. His-NPR4-SBC was further purified by anion exchange and gel-filtration chromatography. The peak fractions containing His-NPR4-SBC were collected and concentrated to 15 mg ml−1. A six-amino-acid internal deletion mutant of His-NPR4-SBC, NPR4-SBC-Δ450–455, was purified following the same procedure. The His-NPR1-MBP was purified as previously described8. To copurify NPR1-SBC with NIMIN2-MBP, cells co-expressing the two proteins were collected and lysed in 50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 1 mM TCEP. The supernatant was loaded onto an amylose column, which was subsequently washed with the lysis buffer. The protein complexes were eluted using a buffer containing 50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 1 mM TCEP, and 10 mM maltose. Also see Supplementary Method.Crystallization, data collection, and structure determinationThe crystals of His-NPR4-SBC protein were grown at 16 °C by the hanging-drop vapor diffusion method with 1 μl protein samples containing 2 mM SA mixed with 1 μl reservoir solution containing 0.25 M potassium phosphate dibasic, 23% PEG 3350, pH 9.2. Large sized crystals were obtained and harvested after 1 week. 20% glycerol was included in the mother liquor as the cryoprotectant during crystal harvesting and data collection. To improve the resolution of His-NPR4-SBC crystals, an internal deletion mutant, NPR4-SBC-Δ450–455, was constructed to reduce the length of the disordered loop between αSC2 and αSC3. The heavy atom derivative crystals were obtained by soaking the native crystals in the presence of 0.1 mM cisplatin (cis Pt(NH3)2Cl2) for 6 days. All X-ray diffraction datasets were collected at the Advanced Light Source at Berkeley on beam lines 8.2.1 and 8.2.2. The single anomalous dispersion (SAD) dataset was collected near the platinum absorption edge (λ=1.072 Å). Reflection data were indexed, integrated and scaled with the HKL2000 package21. The SAD method was used to determine the initial phase using PHENIX22 with a 2.8 Å platinum derivative dataset. Initial structural models were built, refined using COOT23 and PHENIX. The final model was built and refined with a 2.28 Å native dataset. The final model has 99% of residues in the favoured region and 0% in outliner region of the Ramachandran plot. The asymmetric unit of the crystal contains two copies of NPR4-SBC, whose conformation might be stabilized by crystal packing.Trypsin digestion assayAtNPR4 constructs of various lengths or BSA protein was diluted to 2 mg ml−1 using reaction buffer containing 50 mM Tris, pH 8.0, 200 mM NaCl, 1 mM TCEP and incubated with or without 1 mM SA or 3-OH BA for 1 hr on ice. Trypsin (Promega, Madison, WI, USA) was added to the sample protein at a final concentration of 0.005 mg ml−1. The digestion reaction proceeded for different time lengths at 20 °C. After the digestion, the samples were analysed by 15 % SDS-PAGE and stained with Coomassie Brilliant Blue dye (CBB), then destained in water before imaging.Tritium-labelled SA binding assaysAll single concentration SA binding measurements for GST-NPR4, including all point mutations and chimeric proteins, were conducted as previously described using the 3H-SA concentrations stated in the figure legends7. For measuring the SA saturation binding curve of NPR4 and npr4F426L T459G, 2 μg of each protein were bound to 25 μl of magnetic glutathione beads (Pierce) and incubated for 1 hr at room temperature with shaking at 1000 rpm, in 200 μl of sodium citrate buffer pH 6.8 containing the indicated 3H-SA concentrations ranging from 10 to 800 nM. The samples were washed three times with 1 ml of binding buffer, resuspended in 100 μl of water, transferred to a scintillation vial and counted with 6 ml of Ultima Gold (Perkin Elmer). For measuring the SA saturation binding curve of the full-length NPR1 and the NPR1-SBC-NIMIN-2 complex, 3H-SA (American Radiolabeled Chemicals, Inc., 50 Ci/mmole) was mixed with various amounts of cold SA to obtain the required specific activity for each experiment. The binding assay was based on a previously reported procedure with a rapid centrifugation-based gel filtration method8,10. Specifically, G-25 QiaShredder mini columns were initially filled with 0.13 g of Sephadex G-25 (fine), which was allowed to swell over night with PBS buffer pH 7.4 containing 0.1% Tween20 in a final volume of about 650 μl. Columns were washed with 1.0 ml buffer by gravity and air pockets were removed prior to spinning at 1000 x g for 30 sec. The NPR proteins were diluted to between 0.2 to 10 μM. SA was serially diluted to 6.25 nM. The reaction volume was 200 μl and the samples were incubated on ice for 2 hr. After incubation, three 50 μl aliquots, each of which is equivalent to 7.7% column volume, were added to the top of the pre-equilibrated G-25 columns without touching the surface. Samples were then immediately centrifuged at 1000 x g for 2 min. The collected eluate was transferred to a scintillation vial and counted with 5 ml of Ultima Gold (Perkin Elmer). A protein-free SA-only sample was measured using the same procedure as a control for the background signal. For the comparison between NPR1-SBC- NIMIN2 and the NPR1-SBC-R432Q- NIMIN2 mutant, the SA-only background signal was not subtracted. To estimate the percentage of NPR proteins bound to SA in the 3H-SA binding assay, the concentration of 3H-SA stock solution was first calculated (20 μM) based on its specific activity (50 Ci/mmol) and radioactivity concentration (1 mCi/ml). By counting a small aliquot of the complete reaction solution with a specific amount of 3H-SA mixed with cold SA at a defined ratio, the number of moles of total SA per cpm was derived. For example, 10 μL of a complete reaction solution with 800 nM total SA prepared by a mixture of 3H-SA and cold SA had 331,686 cpm, which yielded 2.41 × 10−17 moles per cpm. The total number of moles of SA in the gel filtration flow-through fraction for each sample from the SA binding assay was then calculated by multiplying the total cpm by the number of moles of SA per cpm described above. The concentrations of the purified proteins were determined by the Bradford assay. The amount of the NPR proteins relative to the total proteins were quantified by gel densitometry analysis. The number of moles of the NPR proteins in each SA binding assay sample was then obtained based on the protein concentration and the reaction volume. The percentage of NPR proteins bound to SA was calculated by dividing the total number of moles of SA by the number of moles of the NPR proteins.In vitro pull-down assayThe pull-down reactions were conducted as previously described with the following modifications7. Each reaction was assembled in 4 ml of buffer with equal amounts (5–10 μg) of protein (GST-NPR4 or GST-npr4 point mutants) bound to glutathione agarose beads mixed with 50 μg of HIS-MBP-NPR1-StrepII in the presence or absence of sodium salicylate (Na-SA). The reactions were incubated over-night with end-over-end mixing. After incubation, the beads were collected by centrifugation at 700 x g for 1 min and washed 3 times with 0.5 ml of pull-down buffer with or without Na-SA at the appropriate concentration. The samples were resuspended in 50 μl of elution buffer (50 mM Tris-HCl, 200 mM NaCl, 5 mM DTT, 20 mM reduced glutathione, pH 8.0) and incubated at 22 °C with shaking at 900 rpm for 20 min. After centrifugation at 700 x g for 1 min, 50 μl of supernatant for each sample was added to 13 μl of NuPage 4x LDS sample buffer (Life Technologies) containing 200 mM DTT, which was subsequently heated at 95 °C for 10 min. The final samples were resolved by SDS-PAGE and visualized either by staining with the Colloidal Blue Staining Kit (Thermo Fisher Scientific) or by western blot using anti-GST-HRP (GE Healthcare) and anti-StrepII-HRP (Millipore) antibodies. Chemiluminescence was detected using the Super Signal West Pico chemiluminescent substrate (Thermo Fisher Scientific).HDX-MS assay10 μl of 0.2 mg ml−1
AtNPR4 full-length protein (incubated with or without 0.1 mM SA) were diluted into 85 μl of D2O with 5 μl of 20X PBS buffer (with or without 0.1 mM SA) and incubated at room temperature for either 3 sec, 1 min, 30 min, or 20 hr. At the desired time point, each sample was rapidly mixed with an equal volume of ice-cold 8 M urea with 0.2% formic acid and 0.1% trifluoroacetic acid (TFA) for a final pH of 2.5 in order to quench the exchange reaction. The samples were then immediately frozen in liquid nitrogen and stored at −80 °C until LC-MS analysis. A fully deuterated sample was prepared by incubating a denatured stock protein in 4 M guanidinium chloride at 85 °C for 10 min, diluting into deuterium just as with all other samples, incubating at 60 °C for 2 hr, and followed by the same quenching procedure. A “zero” time point to correct for in-exchange during digestion and sample handling was prepared by pre-mixing the 5 μl of 20X PBS, 85 μl of D2O and 100 μl of quench prior to adding the 10 μl of stock protein.Samples were thawed and injected onto a custom cold box that keeps the injection lines and columns at 0 °C. The protein was first passed over a custom packed 2.1 × 50 mm pepsin column at 200 μl min−1 for inline digestion. Peptides were then trapped on a Waters BEH C18 vanguard column (2.1 × 5 mm 1.7 μm 130 Å) and resolved over BEH C18 column (1 × 100 mm 1.7 μm 130 Å) using linear gradient of 5 to 35% B (A: 0.1% FA, 0.025% TFA, 5% ACN; B: ACN with 0.1% FA) over 10 min and analysed on a Waters Synapt G2-Si mass spectrometer. A series of washes over the trap and pepsin columns was used between injections to minimize carry-over. An identical LC protocol was used with the LC connected to a Thermo LTQ Orbitrap mass spectrometer to collect several rounds of DDA MS/MS of an undeuterated samples. Peptides were identified using Protein Prospector with a score threshold of 15.AlphaScreen Competition AssayTo monitor NPR1-NPR4 interaction and its disruption by SA, AlphaScreen assays were performed using EnSpire reader (PerkinElmer). MBP-tagged NPR1 was immobilized to Anti-MBP AlphaScreen acceptor beads. GST-tagged NPR4 was attached to anti-GST AlphaScreen donor beads. The donor and acceptor beads were brought into proximity by the interactions between the NPR1 and NPR4. Competition assays were performed in the presence of tag-free NPR4, SA or BA, all of which were titrated at various concentrations. The experiments were conducted with 10 nM GST-NPR4 and 10 nM MBP-NPR1 in the presence of 10 μg/mL donor and acceptor beads in a buffer of 50 mM Tris-HCl, pH 8.0, 200 mM NaCl, 1 mM TCEP, 0.1% Triton X-100. The experiments were performed in triplicates. IC50 values were calculated using non-linear curve fitting of the dose response curves generated with Prism 4 (GraphPad).In vitro translation and co-immunoprecipitationEpitope-tagged proteins were synthesized using a wheat germ-based translation system (BioSieg). Synthesized proteins were mixed and incubated with Pierce anti-HA magnetic beads (Thermo Fisher Scientific) overnight at 4 °C in the co-immunoprecipitation (co-IP) buffer (50 mM Tris pH 6.8, 100 mM NaCl, 0.1% nonidet P40, and Complete EDTA-free protease inhibitor cocktail (Roche) ± 0.1 mM Na-SA). Following immunoprecipitation (IP), beads were collected and washed three times with 1 ml of the pull-down buffer using a magnetic stand, and the samples were eluted by incubation at 95 °C for 10 min in the NuPage LDS sample buffer (Life Technologies™), resolved by SDS-PAGE, and visualized by western blot with anti-HA (BioLegend) or anti-FLAG (Sigma-Aldrich) antibodies and the Super Signal West Pico chemiluminescent substrate (Thermo Fisher Scientific).Plant materialThe Arabidopsis thaliana genotypes, WT, npr1–2 and the npr3–1 npr4–3 (npr3/4) double mutant used in this study were all in the Columbia-0 (Col-0) background. The CDS of wild-type NPR4 and all point mutants used in this study were recombined into pK7FWG2 to generate C terminal GFP fusions and transformed into Arabidopsis npr3 npr4 mutants by floral dip24. First generation transgenic lines were selected on MS media supplemented with 50 μg/ml kanamycin and screened by western blot for expression of the GFP-fusion proteins. T2 lines containing a single insertion were identified by segregation analysis of the antibiotic resistance in 100 seedlings, and T3 homozygous lines were confirmed by segregation analysis and western blots.Gene expression analysisTotal RNA was extracted from of plant tissue treated with water or Na-SA using TRIzol reagent (Thermo Fisher Scientific) following the manufacturer’s protocol. Total RNA samples were incubated with Turbo DNase (Thermo Fisher Scientific) to ensure removal of any residual DNA. The samples were quantified using a UV5 Nano spectrophotometer (Mettler Toledo) and 2 μg of each was used for cDNA synthesis with Superscript III reverse transcriptase (Thermo Fisher Scientific) following the manufacturer’s protocol. cDNA samples were diluted 1:8 in RNase-free water and 2 μl of each was used for quantitative PCR (qPCR) with Fast Start Universal Sybr Green master mix (Roche) using gene-specific primers. qPCR experiments were conducted and analysed using a RealPlex2 Ep-gradient Master Cycler (Eppendorf).Pathogen infectionPseudomonas syringae pv. maculicola (Psm) ES4326 or Psm carrying the effector protein AvrRpt2 were grown at 30 °C on plates containing the King’s B medium (KB) for 24 hr and resuspended in 10 mM MgCl2. 3.5-week-old plants grown in soil (Metro Mix 200, Grace-Sierra, Milpitas, CA) were used for infection assays. The SA protection assays were conducted as previously described25. However, the plants were sprayed with a sub-optimal SA concentration (0.1 mM) to demonstrate the enhanced sensitivity of plants expressing npr4F426L T459G. When indicated, bacterial growth from three experiments (8 biological replicates each) were combined using linear mixed effect model (lme4 R package) with experiment as random effects26. The ion leakage and bacterial growth experiments using Psm ES4326/AvrRpt2 were conducted as previously described19. All experiments were repeated at least three times with similar results.Protein degradation assaysCell-free degradation assays were performed using 12-day-old, liquid grown seedlings, treated with 0.1 mM Na-SA for 24 h to induce endogenous NPR1 accumulation. The assay was carried out as previously described27 with the following modifications: To monitor levels of the endogenous NPR1, protein extraction buffer was supplemented with 100 μg/ml cycloheximide to inhibit protein synthesis. 0.05% Tween-20 was also added to improve recovery of nuclear NPR1. Proteasome inhibitors (MG115 and MG132) were used at a final concentration of 417 μM. Incubations were carried out in the presence of 0.1 mM SA. The samples were resolved by SDS-PAGE and visualized by western blot with an anti-NPR1 antibody. Images were obtained using a ChemDoc XRS+ imaging system (Bio-Rad), lanes and bands were defined using Image Lab software (Bio-Rad), and the bands corresponding to NPR1 proteins were quantified based on the signal intensity relative to time 0. In planta degradation assays were performed as previously described27, except that endogenous NPR1 levels were monitored and seedlings were pre-treated with 0.1 mM SA or 1 mM. Western blotting and data analysis were performed as described for the cell-free degradation assays.Phylogenetic analysisFor alignment of the C-terminal domains of NPR orthologues, the sequences of representative proteins were obtained from the Pfam database and aligned using Clustal Omega with default settings (http://msb.embopress.org/content/7/1/539). A neighbour-joining tree was created using the Phylogeny.fr web application28 along with the iTOL software29.Statistics and ReproducibilityIn all statistical data, the centre values are the mean and the error bars represent standard deviation from the mean. No statistical methods were used to predetermine sample size. All experiments, except HDX-MS (once) and data associated with Extended Data Fig. 6a (twice), have been repeated three times or more with similar results.Reporting summaryFurther information on research design is available in the Nature Reporting Summary linked to this paper.

Molecular cloningThe coding sequences of Arabidopsis thaliana NPR1 (AT1G64280), NPR4 (AT4G19660) and NIMIN2 (AT3G25882) were amplified by polymerase chain reaction from an Arabidopsis cDNA library, and sub-cloned into the DH5α vector with different N-terminally fused tags and a tobacco etch virus (TEV)-cleavage site. The specific amino acid changes for the NPR1 and NPR4 point mutations were generated using the QuikChange II site-directed mutagenesis kit (Agilent). Glutathione-S-transferase (GST)-fused NPR4 coding sequence (CDS) was subcloned into the pFastBac vector and transformed to E. coli DH10Bac for making baculovirus for protein expression in insect cells. Protein domain swaps were generated by amplifying the desired regions of each CDS with primers designed to create overlapping sequences for each fragment. The DNA fragments were amplified in separate PCR reactions, processed with either a PCR clean-up kit or gel extraction kit (Bio Basic Inc.), and the desired fragments were fused by PCR using gene-specific forward and reverse primers containing attB1 and attB2 Gateway™ recombination sequences, respectively. All CDSs were recombined into pDONR207 or pDONR221 and subsequent expression vectors using the Gateway™ technology and sequenced to confirm accuracy.

Molecular cloning

The coding sequences of Arabidopsis thaliana NPR1 (AT1G64280), NPR4 (AT4G19660) and NIMIN2 (AT3G25882) were amplified by polymerase chain reaction from an Arabidopsis cDNA library, and sub-cloned into the DH5α vector with different N-terminally fused tags and a tobacco etch virus (TEV)-cleavage site. The specific amino acid changes for the NPR1 and NPR4 point mutations were generated using the QuikChange II site-directed mutagenesis kit (Agilent). Glutathione-S-transferase (GST)-fused NPR4 coding sequence (CDS) was subcloned into the pFastBac vector and transformed to E. coli DH10Bac for making baculovirus for protein expression in insect cells. Protein domain swaps were generated by amplifying the desired regions of each CDS with primers designed to create overlapping sequences for each fragment. The DNA fragments were amplified in separate PCR reactions, processed with either a PCR clean-up kit or gel extraction kit (Bio Basic Inc.), and the desired fragments were fused by PCR using gene-specific forward and reverse primers containing attB1 and attB2 Gateway™ recombination sequences, respectively. All CDSs were recombined into pDONR207 or pDONR221 and subsequent expression vectors using the Gateway™ technology and sequenced to confirm accuracy.

Protein purification and preparationThe GST-tagged NPR1, NPR4, and NPR4 fragment constructs were over-expressed in BL21 (DE3) strain. AtNPR4-GST protein for HDX experiments were expressed in insect cells according to the methods previously described20. The proteins were purified from the soluble cell lysate by glutathione affinity chromatography. After on-column tag cleavage by TEV protease at 4 °C for 16 hr or directly eluted from the affinity column, the proteins were further purified by anion exchange and size exclusion chromatography. The NPR4-SBC protein was expressed with an N-terminal 6 × His tag (His-NPR4-SBC) in BL21 (DE3) cells, first grown to OD600 nm of 0.9–1.0 at 37 °C and then induced by 0.5 mM isopropyl-β-D-thiogalactopyranoside (IPTG) at 25 °C overnight. Cells were collected and lysed in extraction buffer (50 mM Tris-HCl, pH 8.0, 500 mM NaCl, 1 mM TCEP) and pellets were harvested by centrifugation and resuspended in 50 mM Tris-HCl, pH 8.0, 500 mM NaCl, 8 M urea (denaturation buffer). Denatured His-NPR4-SBC was isolated with Ni-NTA resin and eluted using the denaturation buffer supplemented by 250 mM imidazole before being dialysed against a buffer containing 50 mM Tris-HCl, pH 8.0, 500 mM NaCl, 1 mM DTT, and 2 mM SA. His-NPR4-SBC was further purified by anion exchange and gel-filtration chromatography. The peak fractions containing His-NPR4-SBC were collected and concentrated to 15 mg ml−1. A six-amino-acid internal deletion mutant of His-NPR4-SBC, NPR4-SBC-Δ450–455, was purified following the same procedure. The His-NPR1-MBP was purified as previously described8. To copurify NPR1-SBC with NIMIN2-MBP, cells co-expressing the two proteins were collected and lysed in 50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 1 mM TCEP. The supernatant was loaded onto an amylose column, which was subsequently washed with the lysis buffer. The protein complexes were eluted using a buffer containing 50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 1 mM TCEP, and 10 mM maltose. Also see Supplementary Method.

Protein purification and preparation

The GST-tagged NPR1, NPR4, and NPR4 fragment constructs were over-expressed in BL21 (DE3) strain. AtNPR4-GST protein for HDX experiments were expressed in insect cells according to the methods previously described20. The proteins were purified from the soluble cell lysate by glutathione affinity chromatography. After on-column tag cleavage by TEV protease at 4 °C for 16 hr or directly eluted from the affinity column, the proteins were further purified by anion exchange and size exclusion chromatography. The NPR4-SBC protein was expressed with an N-terminal 6 × His tag (His-NPR4-SBC) in BL21 (DE3) cells, first grown to OD600 nm of 0.9–1.0 at 37 °C and then induced by 0.5 mM isopropyl-β-D-thiogalactopyranoside (IPTG) at 25 °C overnight. Cells were collected and lysed in extraction buffer (50 mM Tris-HCl, pH 8.0, 500 mM NaCl, 1 mM TCEP) and pellets were harvested by centrifugation and resuspended in 50 mM Tris-HCl, pH 8.0, 500 mM NaCl, 8 M urea (denaturation buffer). Denatured His-NPR4-SBC was isolated with Ni-NTA resin and eluted using the denaturation buffer supplemented by 250 mM imidazole before being dialysed against a buffer containing 50 mM Tris-HCl, pH 8.0, 500 mM NaCl, 1 mM DTT, and 2 mM SA. His-NPR4-SBC was further purified by anion exchange and gel-filtration chromatography. The peak fractions containing His-NPR4-SBC were collected and concentrated to 15 mg ml−1. A six-amino-acid internal deletion mutant of His-NPR4-SBC, NPR4-SBC-Δ450–455, was purified following the same procedure. The His-NPR1-MBP was purified as previously described8. To copurify NPR1-SBC with NIMIN2-MBP, cells co-expressing the two proteins were collected and lysed in 50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 1 mM TCEP. The supernatant was loaded onto an amylose column, which was subsequently washed with the lysis buffer. The protein complexes were eluted using a buffer containing 50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 1 mM TCEP, and 10 mM maltose. Also see Supplementary Method.

Crystallization, data collection, and structure determinationThe crystals of His-NPR4-SBC protein were grown at 16 °C by the hanging-drop vapor diffusion method with 1 μl protein samples containing 2 mM SA mixed with 1 μl reservoir solution containing 0.25 M potassium phosphate dibasic, 23% PEG 3350, pH 9.2. Large sized crystals were obtained and harvested after 1 week. 20% glycerol was included in the mother liquor as the cryoprotectant during crystal harvesting and data collection. To improve the resolution of His-NPR4-SBC crystals, an internal deletion mutant, NPR4-SBC-Δ450–455, was constructed to reduce the length of the disordered loop between αSC2 and αSC3. The heavy atom derivative crystals were obtained by soaking the native crystals in the presence of 0.1 mM cisplatin (cis Pt(NH3)2Cl2) for 6 days. All X-ray diffraction datasets were collected at the Advanced Light Source at Berkeley on beam lines 8.2.1 and 8.2.2. The single anomalous dispersion (SAD) dataset was collected near the platinum absorption edge (λ=1.072 Å). Reflection data were indexed, integrated and scaled with the HKL2000 package21. The SAD method was used to determine the initial phase using PHENIX22 with a 2.8 Å platinum derivative dataset. Initial structural models were built, refined using COOT23 and PHENIX. The final model was built and refined with a 2.28 Å native dataset. The final model has 99% of residues in the favoured region and 0% in outliner region of the Ramachandran plot. The asymmetric unit of the crystal contains two copies of NPR4-SBC, whose conformation might be stabilized by crystal packing.

Crystallization, data collection, and structure determination

The crystals of His-NPR4-SBC protein were grown at 16 °C by the hanging-drop vapor diffusion method with 1 μl protein samples containing 2 mM SA mixed with 1 μl reservoir solution containing 0.25 M potassium phosphate dibasic, 23% PEG 3350, pH 9.2. Large sized crystals were obtained and harvested after 1 week. 20% glycerol was included in the mother liquor as the cryoprotectant during crystal harvesting and data collection. To improve the resolution of His-NPR4-SBC crystals, an internal deletion mutant, NPR4-SBC-Δ450–455, was constructed to reduce the length of the disordered loop between αSC2 and αSC3. The heavy atom derivative crystals were obtained by soaking the native crystals in the presence of 0.1 mM cisplatin (cis Pt(NH3)2Cl2) for 6 days. All X-ray diffraction datasets were collected at the Advanced Light Source at Berkeley on beam lines 8.2.1 and 8.2.2. The single anomalous dispersion (SAD) dataset was collected near the platinum absorption edge (λ=1.072 Å). Reflection data were indexed, integrated and scaled with the HKL2000 package21. The SAD method was used to determine the initial phase using PHENIX22 with a 2.8 Å platinum derivative dataset. Initial structural models were built, refined using COOT23 and PHENIX. The final model was built and refined with a 2.28 Å native dataset. The final model has 99% of residues in the favoured region and 0% in outliner region of the Ramachandran plot. The asymmetric unit of the crystal contains two copies of NPR4-SBC, whose conformation might be stabilized by crystal packing.

Trypsin digestion assayAtNPR4 constructs of various lengths or BSA protein was diluted to 2 mg ml−1 using reaction buffer containing 50 mM Tris, pH 8.0, 200 mM NaCl, 1 mM TCEP and incubated with or without 1 mM SA or 3-OH BA for 1 hr on ice. Trypsin (Promega, Madison, WI, USA) was added to the sample protein at a final concentration of 0.005 mg ml−1. The digestion reaction proceeded for different time lengths at 20 °C. After the digestion, the samples were analysed by 15 % SDS-PAGE and stained with Coomassie Brilliant Blue dye (CBB), then destained in water before imaging.

Trypsin digestion assay

AtNPR4 constructs of various lengths or BSA protein was diluted to 2 mg ml−1 using reaction buffer containing 50 mM Tris, pH 8.0, 200 mM NaCl, 1 mM TCEP and incubated with or without 1 mM SA or 3-OH BA for 1 hr on ice. Trypsin (Promega, Madison, WI, USA) was added to the sample protein at a final concentration of 0.005 mg ml−1. The digestion reaction proceeded for different time lengths at 20 °C. After the digestion, the samples were analysed by 15 % SDS-PAGE and stained with Coomassie Brilliant Blue dye (CBB), then destained in water before imaging.

Tritium-labelled SA binding assaysAll single concentration SA binding measurements for GST-NPR4, including all point mutations and chimeric proteins, were conducted as previously described using the 3H-SA concentrations stated in the figure legends7. For measuring the SA saturation binding curve of NPR4 and npr4F426L T459G, 2 μg of each protein were bound to 25 μl of magnetic glutathione beads (Pierce) and incubated for 1 hr at room temperature with shaking at 1000 rpm, in 200 μl of sodium citrate buffer pH 6.8 containing the indicated 3H-SA concentrations ranging from 10 to 800 nM. The samples were washed three times with 1 ml of binding buffer, resuspended in 100 μl of water, transferred to a scintillation vial and counted with 6 ml of Ultima Gold (Perkin Elmer). For measuring the SA saturation binding curve of the full-length NPR1 and the NPR1-SBC-NIMIN-2 complex, 3H-SA (American Radiolabeled Chemicals, Inc., 50 Ci/mmole) was mixed with various amounts of cold SA to obtain the required specific activity for each experiment. The binding assay was based on a previously reported procedure with a rapid centrifugation-based gel filtration method8,10. Specifically, G-25 QiaShredder mini columns were initially filled with 0.13 g of Sephadex G-25 (fine), which was allowed to swell over night with PBS buffer pH 7.4 containing 0.1% Tween20 in a final volume of about 650 μl. Columns were washed with 1.0 ml buffer by gravity and air pockets were removed prior to spinning at 1000 x g for 30 sec. The NPR proteins were diluted to between 0.2 to 10 μM. SA was serially diluted to 6.25 nM. The reaction volume was 200 μl and the samples were incubated on ice for 2 hr. After incubation, three 50 μl aliquots, each of which is equivalent to 7.7% column volume, were added to the top of the pre-equilibrated G-25 columns without touching the surface. Samples were then immediately centrifuged at 1000 x g for 2 min. The collected eluate was transferred to a scintillation vial and counted with 5 ml of Ultima Gold (Perkin Elmer). A protein-free SA-only sample was measured using the same procedure as a control for the background signal. For the comparison between NPR1-SBC- NIMIN2 and the NPR1-SBC-R432Q- NIMIN2 mutant, the SA-only background signal was not subtracted. To estimate the percentage of NPR proteins bound to SA in the 3H-SA binding assay, the concentration of 3H-SA stock solution was first calculated (20 μM) based on its specific activity (50 Ci/mmol) and radioactivity concentration (1 mCi/ml). By counting a small aliquot of the complete reaction solution with a specific amount of 3H-SA mixed with cold SA at a defined ratio, the number of moles of total SA per cpm was derived. For example, 10 μL of a complete reaction solution with 800 nM total SA prepared by a mixture of 3H-SA and cold SA had 331,686 cpm, which yielded 2.41 × 10−17 moles per cpm. The total number of moles of SA in the gel filtration flow-through fraction for each sample from the SA binding assay was then calculated by multiplying the total cpm by the number of moles of SA per cpm described above. The concentrations of the purified proteins were determined by the Bradford assay. The amount of the NPR proteins relative to the total proteins were quantified by gel densitometry analysis. The number of moles of the NPR proteins in each SA binding assay sample was then obtained based on the protein concentration and the reaction volume. The percentage of NPR proteins bound to SA was calculated by dividing the total number of moles of SA by the number of moles of the NPR proteins.

Tritium-labelled SA binding assays

All single concentration SA binding measurements for GST-NPR4, including all point mutations and chimeric proteins, were conducted as previously described using the 3H-SA concentrations stated in the figure legends7. For measuring the SA saturation binding curve of NPR4 and npr4F426L T459G, 2 μg of each protein were bound to 25 μl of magnetic glutathione beads (Pierce) and incubated for 1 hr at room temperature with shaking at 1000 rpm, in 200 μl of sodium citrate buffer pH 6.8 containing the indicated 3H-SA concentrations ranging from 10 to 800 nM. The samples were washed three times with 1 ml of binding buffer, resuspended in 100 μl of water, transferred to a scintillation vial and counted with 6 ml of Ultima Gold (Perkin Elmer). For measuring the SA saturation binding curve of the full-length NPR1 and the NPR1-SBC-NIMIN-2 complex, 3H-SA (American Radiolabeled Chemicals, Inc., 50 Ci/mmole) was mixed with various amounts of cold SA to obtain the required specific activity for each experiment. The binding assay was based on a previously reported procedure with a rapid centrifugation-based gel filtration method8,10. Specifically, G-25 QiaShredder mini columns were initially filled with 0.13 g of Sephadex G-25 (fine), which was allowed to swell over night with PBS buffer pH 7.4 containing 0.1% Tween20 in a final volume of about 650 μl. Columns were washed with 1.0 ml buffer by gravity and air pockets were removed prior to spinning at 1000 x g for 30 sec. The NPR proteins were diluted to between 0.2 to 10 μM. SA was serially diluted to 6.25 nM. The reaction volume was 200 μl and the samples were incubated on ice for 2 hr. After incubation, three 50 μl aliquots, each of which is equivalent to 7.7% column volume, were added to the top of the pre-equilibrated G-25 columns without touching the surface. Samples were then immediately centrifuged at 1000 x g for 2 min. The collected eluate was transferred to a scintillation vial and counted with 5 ml of Ultima Gold (Perkin Elmer). A protein-free SA-only sample was measured using the same procedure as a control for the background signal. For the comparison between NPR1-SBC- NIMIN2 and the NPR1-SBC-R432Q- NIMIN2 mutant, the SA-only background signal was not subtracted. To estimate the percentage of NPR proteins bound to SA in the 3H-SA binding assay, the concentration of 3H-SA stock solution was first calculated (20 μM) based on its specific activity (50 Ci/mmol) and radioactivity concentration (1 mCi/ml). By counting a small aliquot of the complete reaction solution with a specific amount of 3H-SA mixed with cold SA at a defined ratio, the number of moles of total SA per cpm was derived. For example, 10 μL of a complete reaction solution with 800 nM total SA prepared by a mixture of 3H-SA and cold SA had 331,686 cpm, which yielded 2.41 × 10−17 moles per cpm. The total number of moles of SA in the gel filtration flow-through fraction for each sample from the SA binding assay was then calculated by multiplying the total cpm by the number of moles of SA per cpm described above. The concentrations of the purified proteins were determined by the Bradford assay. The amount of the NPR proteins relative to the total proteins were quantified by gel densitometry analysis. The number of moles of the NPR proteins in each SA binding assay sample was then obtained based on the protein concentration and the reaction volume. The percentage of NPR proteins bound to SA was calculated by dividing the total number of moles of SA by the number of moles of the NPR proteins.

In vitro pull-down assayThe pull-down reactions were conducted as previously described with the following modifications7. Each reaction was assembled in 4 ml of buffer with equal amounts (5–10 μg) of protein (GST-NPR4 or GST-npr4 point mutants) bound to glutathione agarose beads mixed with 50 μg of HIS-MBP-NPR1-StrepII in the presence or absence of sodium salicylate (Na-SA). The reactions were incubated over-night with end-over-end mixing. After incubation, the beads were collected by centrifugation at 700 x g for 1 min and washed 3 times with 0.5 ml of pull-down buffer with or without Na-SA at the appropriate concentration. The samples were resuspended in 50 μl of elution buffer (50 mM Tris-HCl, 200 mM NaCl, 5 mM DTT, 20 mM reduced glutathione, pH 8.0) and incubated at 22 °C with shaking at 900 rpm for 20 min. After centrifugation at 700 x g for 1 min, 50 μl of supernatant for each sample was added to 13 μl of NuPage 4x LDS sample buffer (Life Technologies) containing 200 mM DTT, which was subsequently heated at 95 °C for 10 min. The final samples were resolved by SDS-PAGE and visualized either by staining with the Colloidal Blue Staining Kit (Thermo Fisher Scientific) or by western blot using anti-GST-HRP (GE Healthcare) and anti-StrepII-HRP (Millipore) antibodies. Chemiluminescence was detected using the Super Signal West Pico chemiluminescent substrate (Thermo Fisher Scientific).

In vitro pull-down assay

The pull-down reactions were conducted as previously described with the following modifications7. Each reaction was assembled in 4 ml of buffer with equal amounts (5–10 μg) of protein (GST-NPR4 or GST-npr4 point mutants) bound to glutathione agarose beads mixed with 50 μg of HIS-MBP-NPR1-StrepII in the presence or absence of sodium salicylate (Na-SA). The reactions were incubated over-night with end-over-end mixing. After incubation, the beads were collected by centrifugation at 700 x g for 1 min and washed 3 times with 0.5 ml of pull-down buffer with or without Na-SA at the appropriate concentration. The samples were resuspended in 50 μl of elution buffer (50 mM Tris-HCl, 200 mM NaCl, 5 mM DTT, 20 mM reduced glutathione, pH 8.0) and incubated at 22 °C with shaking at 900 rpm for 20 min. After centrifugation at 700 x g for 1 min, 50 μl of supernatant for each sample was added to 13 μl of NuPage 4x LDS sample buffer (Life Technologies) containing 200 mM DTT, which was subsequently heated at 95 °C for 10 min. The final samples were resolved by SDS-PAGE and visualized either by staining with the Colloidal Blue Staining Kit (Thermo Fisher Scientific) or by western blot using anti-GST-HRP (GE Healthcare) and anti-StrepII-HRP (Millipore) antibodies. Chemiluminescence was detected using the Super Signal West Pico chemiluminescent substrate (Thermo Fisher Scientific).

HDX-MS assay10 μl of 0.2 mg ml−1
AtNPR4 full-length protein (incubated with or without 0.1 mM SA) were diluted into 85 μl of D2O with 5 μl of 20X PBS buffer (with or without 0.1 mM SA) and incubated at room temperature for either 3 sec, 1 min, 30 min, or 20 hr. At the desired time point, each sample was rapidly mixed with an equal volume of ice-cold 8 M urea with 0.2% formic acid and 0.1% trifluoroacetic acid (TFA) for a final pH of 2.5 in order to quench the exchange reaction. The samples were then immediately frozen in liquid nitrogen and stored at −80 °C until LC-MS analysis. A fully deuterated sample was prepared by incubating a denatured stock protein in 4 M guanidinium chloride at 85 °C for 10 min, diluting into deuterium just as with all other samples, incubating at 60 °C for 2 hr, and followed by the same quenching procedure. A “zero” time point to correct for in-exchange during digestion and sample handling was prepared by pre-mixing the 5 μl of 20X PBS, 85 μl of D2O and 100 μl of quench prior to adding the 10 μl of stock protein.Samples were thawed and injected onto a custom cold box that keeps the injection lines and columns at 0 °C. The protein was first passed over a custom packed 2.1 × 50 mm pepsin column at 200 μl min−1 for inline digestion. Peptides were then trapped on a Waters BEH C18 vanguard column (2.1 × 5 mm 1.7 μm 130 Å) and resolved over BEH C18 column (1 × 100 mm 1.7 μm 130 Å) using linear gradient of 5 to 35% B (A: 0.1% FA, 0.025% TFA, 5% ACN; B: ACN with 0.1% FA) over 10 min and analysed on a Waters Synapt G2-Si mass spectrometer. A series of washes over the trap and pepsin columns was used between injections to minimize carry-over. An identical LC protocol was used with the LC connected to a Thermo LTQ Orbitrap mass spectrometer to collect several rounds of DDA MS/MS of an undeuterated samples. Peptides were identified using Protein Prospector with a score threshold of 15.

HDX-MS assay

10 μl of 0.2 mg ml−1
AtNPR4 full-length protein (incubated with or without 0.1 mM SA) were diluted into 85 μl of D2O with 5 μl of 20X PBS buffer (with or without 0.1 mM SA) and incubated at room temperature for either 3 sec, 1 min, 30 min, or 20 hr. At the desired time point, each sample was rapidly mixed with an equal volume of ice-cold 8 M urea with 0.2% formic acid and 0.1% trifluoroacetic acid (TFA) for a final pH of 2.5 in order to quench the exchange reaction. The samples were then immediately frozen in liquid nitrogen and stored at −80 °C until LC-MS analysis. A fully deuterated sample was prepared by incubating a denatured stock protein in 4 M guanidinium chloride at 85 °C for 10 min, diluting into deuterium just as with all other samples, incubating at 60 °C for 2 hr, and followed by the same quenching procedure. A “zero” time point to correct for in-exchange during digestion and sample handling was prepared by pre-mixing the 5 μl of 20X PBS, 85 μl of D2O and 100 μl of quench prior to adding the 10 μl of stock protein.

Samples were thawed and injected onto a custom cold box that keeps the injection lines and columns at 0 °C. The protein was first passed over a custom packed 2.1 × 50 mm pepsin column at 200 μl min−1 for inline digestion. Peptides were then trapped on a Waters BEH C18 vanguard column (2.1 × 5 mm 1.7 μm 130 Å) and resolved over BEH C18 column (1 × 100 mm 1.7 μm 130 Å) using linear gradient of 5 to 35% B (A: 0.1% FA, 0.025% TFA, 5% ACN; B: ACN with 0.1% FA) over 10 min and analysed on a Waters Synapt G2-Si mass spectrometer. A series of washes over the trap and pepsin columns was used between injections to minimize carry-over. An identical LC protocol was used with the LC connected to a Thermo LTQ Orbitrap mass spectrometer to collect several rounds of DDA MS/MS of an undeuterated samples. Peptides were identified using Protein Prospector with a score threshold of 15.

AlphaScreen Competition AssayTo monitor NPR1-NPR4 interaction and its disruption by SA, AlphaScreen assays were performed using EnSpire reader (PerkinElmer). MBP-tagged NPR1 was immobilized to Anti-MBP AlphaScreen acceptor beads. GST-tagged NPR4 was attached to anti-GST AlphaScreen donor beads. The donor and acceptor beads were brought into proximity by the interactions between the NPR1 and NPR4. Competition assays were performed in the presence of tag-free NPR4, SA or BA, all of which were titrated at various concentrations. The experiments were conducted with 10 nM GST-NPR4 and 10 nM MBP-NPR1 in the presence of 10 μg/mL donor and acceptor beads in a buffer of 50 mM Tris-HCl, pH 8.0, 200 mM NaCl, 1 mM TCEP, 0.1% Triton X-100. The experiments were performed in triplicates. IC50 values were calculated using non-linear curve fitting of the dose response curves generated with Prism 4 (GraphPad).

AlphaScreen Competition Assay

To monitor NPR1-NPR4 interaction and its disruption by SA, AlphaScreen assays were performed using EnSpire reader (PerkinElmer). MBP-tagged NPR1 was immobilized to Anti-MBP AlphaScreen acceptor beads. GST-tagged NPR4 was attached to anti-GST AlphaScreen donor beads. The donor and acceptor beads were brought into proximity by the interactions between the NPR1 and NPR4. Competition assays were performed in the presence of tag-free NPR4, SA or BA, all of which were titrated at various concentrations. The experiments were conducted with 10 nM GST-NPR4 and 10 nM MBP-NPR1 in the presence of 10 μg/mL donor and acceptor beads in a buffer of 50 mM Tris-HCl, pH 8.0, 200 mM NaCl, 1 mM TCEP, 0.1% Triton X-100. The experiments were performed in triplicates. IC50 values were calculated using non-linear curve fitting of the dose response curves generated with Prism 4 (GraphPad).

In vitro translation and co-immunoprecipitationEpitope-tagged proteins were synthesized using a wheat germ-based translation system (BioSieg). Synthesized proteins were mixed and incubated with Pierce anti-HA magnetic beads (Thermo Fisher Scientific) overnight at 4 °C in the co-immunoprecipitation (co-IP) buffer (50 mM Tris pH 6.8, 100 mM NaCl, 0.1% nonidet P40, and Complete EDTA-free protease inhibitor cocktail (Roche) ± 0.1 mM Na-SA). Following immunoprecipitation (IP), beads were collected and washed three times with 1 ml of the pull-down buffer using a magnetic stand, and the samples were eluted by incubation at 95 °C for 10 min in the NuPage LDS sample buffer (Life Technologies™), resolved by SDS-PAGE, and visualized by western blot with anti-HA (BioLegend) or anti-FLAG (Sigma-Aldrich) antibodies and the Super Signal West Pico chemiluminescent substrate (Thermo Fisher Scientific).

In vitro translation and co-immunoprecipitation

Epitope-tagged proteins were synthesized using a wheat germ-based translation system (BioSieg). Synthesized proteins were mixed and incubated with Pierce anti-HA magnetic beads (Thermo Fisher Scientific) overnight at 4 °C in the co-immunoprecipitation (co-IP) buffer (50 mM Tris pH 6.8, 100 mM NaCl, 0.1% nonidet P40, and Complete EDTA-free protease inhibitor cocktail (Roche) ± 0.1 mM Na-SA). Following immunoprecipitation (IP), beads were collected and washed three times with 1 ml of the pull-down buffer using a magnetic stand, and the samples were eluted by incubation at 95 °C for 10 min in the NuPage LDS sample buffer (Life Technologies™), resolved by SDS-PAGE, and visualized by western blot with anti-HA (BioLegend) or anti-FLAG (Sigma-Aldrich) antibodies and the Super Signal West Pico chemiluminescent substrate (Thermo Fisher Scientific).

Plant materialThe Arabidopsis thaliana genotypes, WT, npr1–2 and the npr3–1 npr4–3 (npr3/4) double mutant used in this study were all in the Columbia-0 (Col-0) background. The CDS of wild-type NPR4 and all point mutants used in this study were recombined into pK7FWG2 to generate C terminal GFP fusions and transformed into Arabidopsis npr3 npr4 mutants by floral dip24. First generation transgenic lines were selected on MS media supplemented with 50 μg/ml kanamycin and screened by western blot for expression of the GFP-fusion proteins. T2 lines containing a single insertion were identified by segregation analysis of the antibiotic resistance in 100 seedlings, and T3 homozygous lines were confirmed by segregation analysis and western blots.

Plant material

The Arabidopsis thaliana genotypes, WT, npr1–2 and the npr3–1 npr4–3 (npr3/4) double mutant used in this study were all in the Columbia-0 (Col-0) background. The CDS of wild-type NPR4 and all point mutants used in this study were recombined into pK7FWG2 to generate C terminal GFP fusions and transformed into Arabidopsis npr3 npr4 mutants by floral dip24. First generation transgenic lines were selected on MS media supplemented with 50 μg/ml kanamycin and screened by western blot for expression of the GFP-fusion proteins. T2 lines containing a single insertion were identified by segregation analysis of the antibiotic resistance in 100 seedlings, and T3 homozygous lines were confirmed by segregation analysis and western blots.

Gene expression analysisTotal RNA was extracted from of plant tissue treated with water or Na-SA using TRIzol reagent (Thermo Fisher Scientific) following the manufacturer’s protocol. Total RNA samples were incubated with Turbo DNase (Thermo Fisher Scientific) to ensure removal of any residual DNA. The samples were quantified using a UV5 Nano spectrophotometer (Mettler Toledo) and 2 μg of each was used for cDNA synthesis with Superscript III reverse transcriptase (Thermo Fisher Scientific) following the manufacturer’s protocol. cDNA samples were diluted 1:8 in RNase-free water and 2 μl of each was used for quantitative PCR (qPCR) with Fast Start Universal Sybr Green master mix (Roche) using gene-specific primers. qPCR experiments were conducted and analysed using a RealPlex2 Ep-gradient Master Cycler (Eppendorf).

Gene expression analysis

Total RNA was extracted from of plant tissue treated with water or Na-SA using TRIzol reagent (Thermo Fisher Scientific) following the manufacturer’s protocol. Total RNA samples were incubated with Turbo DNase (Thermo Fisher Scientific) to ensure removal of any residual DNA. The samples were quantified using a UV5 Nano spectrophotometer (Mettler Toledo) and 2 μg of each was used for cDNA synthesis with Superscript III reverse transcriptase (Thermo Fisher Scientific) following the manufacturer’s protocol. cDNA samples were diluted 1:8 in RNase-free water and 2 μl of each was used for quantitative PCR (qPCR) with Fast Start Universal Sybr Green master mix (Roche) using gene-specific primers. qPCR experiments were conducted and analysed using a RealPlex2 Ep-gradient Master Cycler (Eppendorf).

Pathogen infectionPseudomonas syringae pv. maculicola (Psm) ES4326 or Psm carrying the effector protein AvrRpt2 were grown at 30 °C on plates containing the King’s B medium (KB) for 24 hr and resuspended in 10 mM MgCl2. 3.5-week-old plants grown in soil (Metro Mix 200, Grace-Sierra, Milpitas, CA) were used for infection assays. The SA protection assays were conducted as previously described25. However, the plants were sprayed with a sub-optimal SA concentration (0.1 mM) to demonstrate the enhanced sensitivity of plants expressing npr4F426L T459G. When indicated, bacterial growth from three experiments (8 biological replicates each) were combined using linear mixed effect model (lme4 R package) with experiment as random effects26. The ion leakage and bacterial growth experiments using Psm ES4326/AvrRpt2 were conducted as previously described19. All experiments were repeated at least three times with similar results.

Pathogen infection

Pseudomonas syringae pv. maculicola (Psm) ES4326 or Psm carrying the effector protein AvrRpt2 were grown at 30 °C on plates containing the King’s B medium (KB) for 24 hr and resuspended in 10 mM MgCl2. 3.5-week-old plants grown in soil (Metro Mix 200, Grace-Sierra, Milpitas, CA) were used for infection assays. The SA protection assays were conducted as previously described25. However, the plants were sprayed with a sub-optimal SA concentration (0.1 mM) to demonstrate the enhanced sensitivity of plants expressing npr4F426L T459G. When indicated, bacterial growth from three experiments (8 biological replicates each) were combined using linear mixed effect model (lme4 R package) with experiment as random effects26. The ion leakage and bacterial growth experiments using Psm ES4326/AvrRpt2 were conducted as previously described19. All experiments were repeated at least three times with similar results.

Protein degradation assaysCell-free degradation assays were performed using 12-day-old, liquid grown seedlings, treated with 0.1 mM Na-SA for 24 h to induce endogenous NPR1 accumulation. The assay was carried out as previously described27 with the following modifications: To monitor levels of the endogenous NPR1, protein extraction buffer was supplemented with 100 μg/ml cycloheximide to inhibit protein synthesis. 0.05% Tween-20 was also added to improve recovery of nuclear NPR1. Proteasome inhibitors (MG115 and MG132) were used at a final concentration of 417 μM. Incubations were carried out in the presence of 0.1 mM SA. The samples were resolved by SDS-PAGE and visualized by western blot with an anti-NPR1 antibody. Images were obtained using a ChemDoc XRS+ imaging system (Bio-Rad), lanes and bands were defined using Image Lab software (Bio-Rad), and the bands corresponding to NPR1 proteins were quantified based on the signal intensity relative to time 0. In planta degradation assays were performed as previously described27, except that endogenous NPR1 levels were monitored and seedlings were pre-treated with 0.1 mM SA or 1 mM. Western blotting and data analysis were performed as described for the cell-free degradation assays.

Protein degradation assays

Cell-free degradation assays were performed using 12-day-old, liquid grown seedlings, treated with 0.1 mM Na-SA for 24 h to induce endogenous NPR1 accumulation. The assay was carried out as previously described27 with the following modifications: To monitor levels of the endogenous NPR1, protein extraction buffer was supplemented with 100 μg/ml cycloheximide to inhibit protein synthesis. 0.05% Tween-20 was also added to improve recovery of nuclear NPR1. Proteasome inhibitors (MG115 and MG132) were used at a final concentration of 417 μM. Incubations were carried out in the presence of 0.1 mM SA. The samples were resolved by SDS-PAGE and visualized by western blot with an anti-NPR1 antibody. Images were obtained using a ChemDoc XRS+ imaging system (Bio-Rad), lanes and bands were defined using Image Lab software (Bio-Rad), and the bands corresponding to NPR1 proteins were quantified based on the signal intensity relative to time 0. In planta degradation assays were performed as previously described27, except that endogenous NPR1 levels were monitored and seedlings were pre-treated with 0.1 mM SA or 1 mM. Western blotting and data analysis were performed as described for the cell-free degradation assays.

Phylogenetic analysisFor alignment of the C-terminal domains of NPR orthologues, the sequences of representative proteins were obtained from the Pfam database and aligned using Clustal Omega with default settings (http://msb.embopress.org/content/7/1/539). A neighbour-joining tree was created using the Phylogeny.fr web application28 along with the iTOL software29.

Phylogenetic analysis

For alignment of the C-terminal domains of NPR orthologues, the sequences of representative proteins were obtained from the Pfam database and aligned using Clustal Omega with default settings (http://msb.embopress.org/content/7/1/539). A neighbour-joining tree was created using the Phylogeny.fr web application28 along with the iTOL software29.

Statistics and ReproducibilityIn all statistical data, the centre values are the mean and the error bars represent standard deviation from the mean. No statistical methods were used to predetermine sample size. All experiments, except HDX-MS (once) and data associated with Extended Data Fig. 6a (twice), have been repeated three times or more with similar results.

Statistics and Reproducibility

In all statistical data, the centre values are the mean and the error bars represent standard deviation from the mean. No statistical methods were used to predetermine sample size. All experiments, except HDX-MS (once) and data associated with Extended Data Fig. 6a (twice), have been repeated three times or more with similar results.

Reporting summaryFurther information on research design is available in the Nature Reporting Summary linked to this paper.

Reporting summary

Further information on research design is available in the Nature Reporting Summary linked to this paper.

Data AvailabilityUncropped gels and DNA sequencing results of all constructs are included in Supplementary Data. All source data for Fig. 1–4 and Extended Data Fig. 1–4 and 6 are provided. Structural coordinates and structural factors have been deposited in the Protein Data Bank under accession numbers 6WPG. All reagents are available from the corresponding authors upon request.

Data Availability

Uncropped gels and DNA sequencing results of all constructs are included in Supplementary Data. All source data for Fig. 1–4 and Extended Data Fig. 1–4 and 6 are provided. Structural coordinates and structural factors have been deposited in the Protein Data Bank under accession numbers 6WPG. All reagents are available from the corresponding authors upon request.

Code AvailabilityAll software used in this study is publicly available. These include HKL-2000 v720 package, GraphPad Prism 7.00 and 8, Phenix 1.14–3260, Protein Prospector V5.23.1, Microsoft Excel 2018, Clustal Omega, and R Studio V1.3.1225 with a script listed in Reporting Summary.

Code Availability

All software used in this study is publicly available. These include HKL-2000 v720 package, GraphPad Prism 7.00 and 8, Phenix 1.14–3260, Protein Prospector V5.23.1, Microsoft Excel 2018, Clustal Omega, and R Studio V1.3.1225 with a script listed in Reporting Summary.

Supplementary MaterialSuppInfoGuideSuppFile1SuppFile3SuppFile21594394_SourceDataFig11594394_SourceDataFig21594394_SourceDataFig41594394_SourceExtDataFig11594394_SourceExtDataFig21594394_SourceExtDataFig31594394_SourceExtDataFig41594394_SourceExtDataFig61594394_SourceDataFig3

Supplementary Material

Extended DataExtended Data Figure 1.Mapping and refolding of the NPR4-SBC.Related to Figs. 1 and 2. a, Domain arrangements of Arabidopsis thaliana NPR4 and different constructs used for mapping NPR4-SBC. b-f, Comparison of trypsin digestion profiles of truncated NPR4 proteins with or without 1 mM salicylic acid (SA) or 3-OH benzoic acid (BA). Negative controls of limited proteolytic digestion of NPR4 were conducted with bovine serum albumin (BSA) (b), SA-insensitive NPR4-R419Q mutant associated with npr4–4D (c), and NPR4–1-391 fragment (f). g, h, SA-dependent refolding of NPR4-SBC polypeptide affects its solubility. BA, benzoic acid, an inactive analogue of SA; Sup., supernatant; M, molecular weight marker. i, Superdex 75 size exclusion chromatography elution profile of the NPR4-SBC fragment refolded in the presence of SA. Excess SA was eluted after one column volume due to a weak interaction with the resin. The inset shows the final purified NPR4-SBC fragment analysed by SDS-PAGE with Coomassie staining. Experiments in (b-i) were repeated three times or more with similar results.Extended Data Figure 2.Deuterium exchange profiles of selected NPR4 peptides.Related to Fig. 2. a, Deuterium uptake plots of representative peptides of NPR4-SBC derived from samples with (red) or without (blue) the presence of 0.1 mM SA. The SA-insensitive deuterium uptake plots of a BTB domain N-terminal peptide are shown on the left as a representative SA-insensitive region. n=3 independent samples. Error bars representing standard deviation of the mean (centre value) are shown, but often too small to be seen. The peptide sequences, amino acid numbers, and the structural domain they belong to are indicated on top of the plots. b, The SA-insensitive deuterium uptake plots of three different peptides containing residues that belong to the proposed EAR motif (underlined). c, The SA-free HDX profile is mapped on the NPR4-SBC crystal structure for the 4 time points with a colour ramp scheme indicative of percentage of exchange. Regions coloured in grey were outside of the peptide coverage.Extended Data Figure 3.Sequence alignment of the SBC regions in NPR proteins from several plant species and details of the SA-binding pocket and activity.Related to Fig. 2. a, Structure-based sequence alignment of the SA binding core (SBC) regions of NPR4 and NPR1 orthologues. The secondary structure diagram of NPR4-SBC is shown above the sequences. Regions with no regular secondary structure are shown by lines, and α helices are represented by cylinders. The dashed lines indicate two disordered loops that are not resolved in the structure. Strictly conserved residues are coloured in blue. The rest of the residues are coloured with black (87.5%), brown (75%), or red (<75%) based on their degrees of conservation. The residues directly involved in SA binding are highlighted with asterisks. The putative EAR motif is labelled and indicated by a black bar. Six surface residues selected for mutagenesis analysis are labelled. At (Arabidopsis thaliana; NPR1-AT1G64280, NPR3-AT5G45110, and NPR4-AT4G19660); Os (Oriza sativa NH1-Os01g09800, NH2-Os01g56200, and NH3-Os03g46440); Nb (Nicotiana benthamiana NPR1- LOC107831756); Bn, (Brassica napa NPR1- LOC106389246). b, A close-up stereo view of the NPR4-SBC SA-binding pocket with the omit map electron density shown together with the residues in the stick model. SA is coloured in yellow and red and situated in the centre. Three selected SA-contacting residues in close proximity to the SA carboxyl group are indicated. c, Ligplot of the hydrophobic and polar interactions between SA and NPR4-SBC residues. d, A semi-transparent view of the SA-binding pocket with the SA analogue benzothiadiazole (BTH) (magenta-blue-red sticks) modelled onto SA (yellow-red sticks) situated in the centre and indicated by arrows. The view is related to the NPR4-SBC internal cavity shown in Fig. 2c by 180° vertical rotation. Ala434 is shown in yellow stick and indicated as A434. The internal cavity and surrounding surfaces of NPR4-SBC are shown in green surface representation. e, SA binding by wild type (WT) and R428A mutant of NPR3 as determined with radio-labelled ligand binding assay with 100 nM 3H-SA. n = 6 independent samples. Error bars indicate standard deviation from the mean (centre value).Extended Data Figure 4.Sequence comparison of Arabidopsis NPRs and characterization of His-MBP-NPR1.Related to Fig. 3. a, Neighbour-joining tree of NPR C-terminal (CT) domains and pairwise comparisons of amino acid sequence identity within the CT and SA-binding core (SBC) regions. Bootstrap values are noted for the branching of each node. Numbers 1–6 correspond to the six Arabidopsis NPRs. NOTE: Despite featuring similar CT regions, NPR5 and NPR6 do not contain a regular SBC, reflected by the low sequence identity of their CT domains to that of NPR1–4. b, Size exclusion chromatography elution profile of His-MBP-AtNPR1, which was first purified by amylose affinity chromatography. The inset shows the final purified His-MBP-AtNPR1 protein analysed by SDS-PAGE with Coomassie staining. Experiments were repeated three times with similar results. c, Dose response curve of SA binding by NPR1. In the radio-labelled ligand binding assay, 5 μg of His-MBP-NPR1 protein was incubated with 3H-SA at different concentrations. Three replicates in a single experiment were used to calculate the kD of SA binding to NPR1. n=3 independent samples. Error bars represent the standard errors of the mean (centre value). cpm, counts per minute. c, Diagrams of NPR1 and NPR4 domain boundaries that are relevant to Fig. 3d.Extended Data Figure 5.NPR amino acid sequence homology in angiosperms.Related to Fig. 3. a, Neighbour-joining tree depicting the divergence of the C-terminal (CT) domains of Arabidopsis thaliana NPRs and Oryza sativa NH proteins (highlighted), as well as relationship with other NPR-like proteins in angiosperms. Black, out groups; blue, NPR1/NPR2 clade; and orange, NPR3/NPR4 clade. b, c, Amino acid sequence alignments of NPR CTs indicating the amino acid conservation (black shade) at the position (arrow) of NPR4 residues R419 and F426 (b), as well as T459 and the putative EAR motif (c). The degree of conservation, alignment quality, and conservation strength are indicated by the histograms below the sequences.Extended Data Figure 6.NPR4 point mutant expression and their differential phenotypic effects.Related to Fig. 4. a, Western blot analysis of transgenic npr3 npr4 (npr3/4) seedlings expressing similar amounts of the NPR4-GFP variants 24 hr after treatment with 0.1 mM SA. An antibody against GFP (α-GFP) was used. *, non-specific band. Experiments were repeated two times with similar results. b, Western blot depicting cell-free protein degradation assays comparing the rate of endogenous NPR1 degradation in protein extracts from (a) and quantifications of the data are shown in Fig. 4a. Arrows, endogenous NPR1; MG115/132, proteasome inhibitors. The ratios listed below each sample indicate NPR1 levels compared to 0 min for the degradation assay or 30 min for samples containing MG115/132. An antibody against NPR1 (α-NPR1) was used. Experiments were repeated three times with similar results. c, d, in planta protein degradation assays comparing the rate of endogenous NPR1 degradation in seedlings pre-treated with 0.1 mM SA for 24 hr. NPR1 was detected using an α-NPR1 antibody (c) and the relative band intensities were quantified (d). n=3 independent biological samples. Error bars indicate standard deviation from the mean (centre values). e, Western blot analysis of transgenic npr3/4 seedlings expressing NPR4-GFP or npr4F426L-GFP after a 24-hr treatment with 0.1 mM SA. L1, 2, 6, 7, independent transgenic lines; TPE, total protein extract; CBB, Coomassie brilliant blue. An antibody against GFP (α-GFP) was used. *, non-specific band. f, Fold change of PR1 expression in seedlings from (e) 24 hr after 0.1 mM SA treatments. The data were normalized to UBQ5 expression, error bars indicate standard deviation from the mean (n = 3). Statistical significance was determined by 1-way ANOVA on log-transformed data followed by Tukey’s multiple comparison correction, letters indicate statistical significance, p < 0.05. g, Western blot analysis of mature leaves from transgenic npr3/4 plants expressing NPR4-GFP, npr4F426L-GFP or npr4F426L T459G-GFP after a 6-hr treatment with 0.5 mM SA spray. L8, 11, independent transgenic lines. An antibody against GFP (α-GFP) was used. *, non-specific band. h, Fold change of PR1 expression in leaves from (g) 6 hr after mock or 0.5 mM SA spray. The data were normalized to UBQ5 expression. n = 5 biologically independent samples. Error bars indicate standard deviation from the mean (centre values). Statistical significance was determined by 1-way ANOVA on log-transformed data followed by Tukey’s multiple comparison correction, letters indicate statistical significance, p < 0.05. i, j, SA protection against Psm ES4326 infection. Images of disease symptom development (i) and bacterial growth in infected leaves (j) were recorded 3 days after inoculation at OD600nm = 0.001. Light grey bars, mock; dark grey bars, 0.1 mM SA. Colony forming units (cfu) were determined for three experiments and combined using linear mixed effect model (lme4) with experiment as random effects. n = three experiments each with eight biological repeats / genotype and treatment. Error bars indicate standard deviation from the mean (centre value). Statistical significance was determined by 2-way ANOVA on log-transformed data. ns p = 0.6, * p = 0.03; ** p = 0.008; *** p = 0.0004. Experiments in (i) were repeated three times with similar results. k, Relative band intensities were quantified after in planta protein degradation assays comparing the rate of endogenous NPR1 degradation in seedlings pre-treated with 1 mM SA for 24 hr as in (c, d). n = 5 biological independent samples. Error bars indicate standard error from the mean (centre values).Extended Data Table 1.Crystallography data collection and refinement statistics.NativePt-NPR4-SBC-derivativeData collectionSpace groupP3121P3121Cell dimensions a, b, c (Å)88.293, 88.293, 138.00388.219, 88.219, 138.034 α, β, γ (°)90, 90, 12090, 90, 120PeakWavelength (Å)1.072Resolution (Å)2.282.80Rmerge0.064 (0.771)0.105 (0.673)I / σI23.3 (2.8)15.2 (2.6)Completeness (%)99.31 (96.44)99.8 (99.7)Redundancy10.6 (8.4)6.4 (6.2)RefinementResolution (Å)2.28No. reflections28714 (2733)Rwork / Rfree0.205/0.224No. atoms Protein1855 Ligand/ion20/0 Water76B-factors79.68 Protein79.85 Ligand/ion55.16 Water82.16R.m.s deviations Bond lengths (Å)0.009 Bond angles (°)0.878*This table describes the data collection, phasing and refinement statistics of His-AtNPR4-SBC crystals.*Values in parentheses are for highest-resolution shell.

Extended Data

Mapping and refolding of the NPR4-SBC.

Related to Figs. 1 and 2. a, Domain arrangements of Arabidopsis thaliana NPR4 and different constructs used for mapping NPR4-SBC. b-f, Comparison of trypsin digestion profiles of truncated NPR4 proteins with or without 1 mM salicylic acid (SA) or 3-OH benzoic acid (BA). Negative controls of limited proteolytic digestion of NPR4 were conducted with bovine serum albumin (BSA) (b), SA-insensitive NPR4-R419Q mutant associated with npr4–4D (c), and NPR4–1-391 fragment (f). g, h, SA-dependent refolding of NPR4-SBC polypeptide affects its solubility. BA, benzoic acid, an inactive analogue of SA; Sup., supernatant; M, molecular weight marker. i, Superdex 75 size exclusion chromatography elution profile of the NPR4-SBC fragment refolded in the presence of SA. Excess SA was eluted after one column volume due to a weak interaction with the resin. The inset shows the final purified NPR4-SBC fragment analysed by SDS-PAGE with Coomassie staining. Experiments in (b-i) were repeated three times or more with similar results.

Deuterium exchange profiles of selected NPR4 peptides.

Related to Fig. 2. a, Deuterium uptake plots of representative peptides of NPR4-SBC derived from samples with (red) or without (blue) the presence of 0.1 mM SA. The SA-insensitive deuterium uptake plots of a BTB domain N-terminal peptide are shown on the left as a representative SA-insensitive region. n=3 independent samples. Error bars representing standard deviation of the mean (centre value) are shown, but often too small to be seen. The peptide sequences, amino acid numbers, and the structural domain they belong to are indicated on top of the plots. b, The SA-insensitive deuterium uptake plots of three different peptides containing residues that belong to the proposed EAR motif (underlined). c, The SA-free HDX profile is mapped on the NPR4-SBC crystal structure for the 4 time points with a colour ramp scheme indicative of percentage of exchange. Regions coloured in grey were outside of the peptide coverage.

Sequence alignment of the SBC regions in NPR proteins from several plant species and details of the SA-binding pocket and activity.

Related to Fig. 2. a, Structure-based sequence alignment of the SA binding core (SBC) regions of NPR4 and NPR1 orthologues. The secondary structure diagram of NPR4-SBC is shown above the sequences. Regions with no regular secondary structure are shown by lines, and α helices are represented by cylinders. The dashed lines indicate two disordered loops that are not resolved in the structure. Strictly conserved residues are coloured in blue. The rest of the residues are coloured with black (87.5%), brown (75%), or red (<75%) based on their degrees of conservation. The residues directly involved in SA binding are highlighted with asterisks. The putative EAR motif is labelled and indicated by a black bar. Six surface residues selected for mutagenesis analysis are labelled. At (Arabidopsis thaliana; NPR1-AT1G64280, NPR3-AT5G45110, and NPR4-AT4G19660); Os (Oriza sativa NH1-Os01g09800, NH2-Os01g56200, and NH3-Os03g46440); Nb (Nicotiana benthamiana NPR1- LOC107831756); Bn, (Brassica napa NPR1- LOC106389246). b, A close-up stereo view of the NPR4-SBC SA-binding pocket with the omit map electron density shown together with the residues in the stick model. SA is coloured in yellow and red and situated in the centre. Three selected SA-contacting residues in close proximity to the SA carboxyl group are indicated. c, Ligplot of the hydrophobic and polar interactions between SA and NPR4-SBC residues. d, A semi-transparent view of the SA-binding pocket with the SA analogue benzothiadiazole (BTH) (magenta-blue-red sticks) modelled onto SA (yellow-red sticks) situated in the centre and indicated by arrows. The view is related to the NPR4-SBC internal cavity shown in Fig. 2c by 180° vertical rotation. Ala434 is shown in yellow stick and indicated as A434. The internal cavity and surrounding surfaces of NPR4-SBC are shown in green surface representation. e, SA binding by wild type (WT) and R428A mutant of NPR3 as determined with radio-labelled ligand binding assay with 100 nM 3H-SA. n = 6 independent samples. Error bars indicate standard deviation from the mean (centre value).

Sequence comparison of Arabidopsis NPRs and characterization of His-MBP-NPR1.

Related to Fig. 3. a, Neighbour-joining tree of NPR C-terminal (CT) domains and pairwise comparisons of amino acid sequence identity within the CT and SA-binding core (SBC) regions. Bootstrap values are noted for the branching of each node. Numbers 1–6 correspond to the six Arabidopsis NPRs. NOTE: Despite featuring similar CT regions, NPR5 and NPR6 do not contain a regular SBC, reflected by the low sequence identity of their CT domains to that of NPR1–4. b, Size exclusion chromatography elution profile of His-MBP-AtNPR1, which was first purified by amylose affinity chromatography. The inset shows the final purified His-MBP-AtNPR1 protein analysed by SDS-PAGE with Coomassie staining. Experiments were repeated three times with similar results. c, Dose response curve of SA binding by NPR1. In the radio-labelled ligand binding assay, 5 μg of His-MBP-NPR1 protein was incubated with 3H-SA at different concentrations. Three replicates in a single experiment were used to calculate the kD of SA binding to NPR1. n=3 independent samples. Error bars represent the standard errors of the mean (centre value). cpm, counts per minute. c, Diagrams of NPR1 and NPR4 domain boundaries that are relevant to Fig. 3d.

NPR amino acid sequence homology in angiosperms.

Related to Fig. 3. a, Neighbour-joining tree depicting the divergence of the C-terminal (CT) domains of Arabidopsis thaliana NPRs and Oryza sativa NH proteins (highlighted), as well as relationship with other NPR-like proteins in angiosperms. Black, out groups; blue, NPR1/NPR2 clade; and orange, NPR3/NPR4 clade. b, c, Amino acid sequence alignments of NPR CTs indicating the amino acid conservation (black shade) at the position (arrow) of NPR4 residues R419 and F426 (b), as well as T459 and the putative EAR motif (c). The degree of conservation, alignment quality, and conservation strength are indicated by the histograms below the sequences.

NPR4 point mutant expression and their differential phenotypic effects.

Related to Fig. 4. a, Western blot analysis of transgenic npr3 npr4 (npr3/4) seedlings expressing similar amounts of the NPR4-GFP variants 24 hr after treatment with 0.1 mM SA. An antibody against GFP (α-GFP) was used. *, non-specific band. Experiments were repeated two times with similar results. b, Western blot depicting cell-free protein degradation assays comparing the rate of endogenous NPR1 degradation in protein extracts from (a) and quantifications of the data are shown in Fig. 4a. Arrows, endogenous NPR1; MG115/132, proteasome inhibitors. The ratios listed below each sample indicate NPR1 levels compared to 0 min for the degradation assay or 30 min for samples containing MG115/132. An antibody against NPR1 (α-NPR1) was used. Experiments were repeated three times with similar results. c, d, in planta protein degradation assays comparing the rate of endogenous NPR1 degradation in seedlings pre-treated with 0.1 mM SA for 24 hr. NPR1 was detected using an α-NPR1 antibody (c) and the relative band intensities were quantified (d). n=3 independent biological samples. Error bars indicate standard deviation from the mean (centre values). e, Western blot analysis of transgenic npr3/4 seedlings expressing NPR4-GFP or npr4F426L-GFP after a 24-hr treatment with 0.1 mM SA. L1, 2, 6, 7, independent transgenic lines; TPE, total protein extract; CBB, Coomassie brilliant blue. An antibody against GFP (α-GFP) was used. *, non-specific band. f, Fold change of PR1 expression in seedlings from (e) 24 hr after 0.1 mM SA treatments. The data were normalized to UBQ5 expression, error bars indicate standard deviation from the mean (n = 3). Statistical significance was determined by 1-way ANOVA on log-transformed data followed by Tukey’s multiple comparison correction, letters indicate statistical significance, p < 0.05. g, Western blot analysis of mature leaves from transgenic npr3/4 plants expressing NPR4-GFP, npr4F426L-GFP or npr4F426L T459G-GFP after a 6-hr treatment with 0.5 mM SA spray. L8, 11, independent transgenic lines. An antibody against GFP (α-GFP) was used. *, non-specific band. h, Fold change of PR1 expression in leaves from (g) 6 hr after mock or 0.5 mM SA spray. The data were normalized to UBQ5 expression. n = 5 biologically independent samples. Error bars indicate standard deviation from the mean (centre values). Statistical significance was determined by 1-way ANOVA on log-transformed data followed by Tukey’s multiple comparison correction, letters indicate statistical significance, p < 0.05. i, j, SA protection against Psm ES4326 infection. Images of disease symptom development (i) and bacterial growth in infected leaves (j) were recorded 3 days after inoculation at OD600nm = 0.001. Light grey bars, mock; dark grey bars, 0.1 mM SA. Colony forming units (cfu) were determined for three experiments and combined using linear mixed effect model (lme4) with experiment as random effects. n = three experiments each with eight biological repeats / genotype and treatment. Error bars indicate standard deviation from the mean (centre value). Statistical significance was determined by 2-way ANOVA on log-transformed data. ns p = 0.6, * p = 0.03; ** p = 0.008; *** p = 0.0004. Experiments in (i) were repeated three times with similar results. k, Relative band intensities were quantified after in planta protein degradation assays comparing the rate of endogenous NPR1 degradation in seedlings pre-treated with 1 mM SA for 24 hr as in (c, d). n = 5 biological independent samples. Error bars indicate standard error from the mean (centre values).

Crystallography data collection and refinement statistics.

This table describes the data collection, phasing and refinement statistics of His-AtNPR4-SBC crystals.

Values in parentheses are for highest-resolution shell.
