---
pmid: '23870121'
title: SET1 and p300 act synergistically, through coupled histone modifications, in
  transcriptional activation by p53.
authors:
- Tang Z
- Chen WY
- Shimada M
- Nguyen UT
- Kim J
- Sun XJ
- Sengoku T
- McGinty RK
- Fernandez JP
- Muir TW
- Roeder RG
journal: Cell
year: '2013'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4023349
doi: 10.1016/j.cell.2013.06.027
---

# SET1 and p300 act synergistically, through coupled histone modifications, in transcriptional activation by p53.
**Authors:** Tang Z, Chen WY, Shimada M, Nguyen UT, Kim J, Sun XJ, Sengoku T, McGinty RK, Fernandez JP, Muir TW, Roeder RG
**Journal:** Cell (2013)
**DOI:** [10.1016/j.cell.2013.06.027](https://doi.org/10.1016/j.cell.2013.06.027)
**PMC:** [PMC4023349](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4023349/)

## Abstract

1. Cell. 2013 Jul 18;154(2):297-310. doi: 10.1016/j.cell.2013.06.027.

SET1 and p300 act synergistically, through coupled histone modifications, in 
transcriptional activation by p53.

Tang Z(1), Chen WY, Shimada M, Nguyen UT, Kim J, Sun XJ, Sengoku T, McGinty RK, 
Fernandez JP, Muir TW, Roeder RG.

Author information:
(1)Laboratory of Biochemistry and Molecular Biology, The Rockefeller University, 
New York, NY 10065, USA.

Erratum in
    Cell. 2023 May 11;186(10):2280. doi: 10.1016/j.cell.2023.04.013.

The H3K4me3 mark in chromatin is closely correlated with actively transcribed 
genes, although the mechanisms involved in its generation and function are not 
fully understood. In vitro studies with recombinant chromatin and purified human 
factors demonstrate a robust SET1 complex (SET1C)-mediated H3K4 trimethylation 
that is dependent upon p53- and p300-mediated H3 acetylation, a corresponding 
SET1C-mediated enhancement of p53- and p300-dependent transcription that 
reflects a primary effect of SET1C through H3K4 trimethylation, and direct 
SET1C-p53 and SET1C-p300 interactions indicative of a targeted recruitment 
mechanism. Complementary cell-based assays demonstrate a DNA-damage-induced 
p53-SET1C interaction, a corresponding enrichment of SET1C and H3K4me3 on a p53 
target gene (p21/WAF1), and a corresponding codependency of H3K4 trimethylation 
and transcription upon p300 and SET1C. These results establish a mechanism in 
which SET1C and p300 act cooperatively, through direct interactions and coupled 
histone modifications, to facilitate the function of p53.

Copyright © 2013 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.cell.2013.06.027
PMCID: PMC4023349
PMID: 23870121 [Indexed for MEDLINE]

## Full Text

INTRODUCTION

The transcription networks that underlie many physiological processes are controlled by sequence-specific DNA-binding transcription factors. These factors act in part through the recruitment of diverse cofactors that generate specific histone modification signatures, some involving histone crosstalk, associated with gene activation or repression events ( Suganuma and Workman, 2011 ; Young, 2011 ). Modified histones in turn may alter transcription through direct effects on chromatin structure or through recruitment of other effectors ( Patel and Wang, 2013 ). Of special interest here, two modifications closely correlated with gene activation are H3 acetylation and H3K4 methylation ( Kouzarides, 2007 ; Li et al., 2007 ).

In human cells, H3K4 methylation is catalyzed by the SET1/MLL family (SET1A, SET1B, MLL1, MLL2, MLL3, and MLL4) of histone methyltransferases (HMTs). Like the prototype yeast Set1 ( Shilatifard, 2012 ), the human SET1/MLL proteins exist in multiprotein complexes that possess both unique subunits and common subunits (WDR5, RbBP5, ASH2L, and DPY30) related to those in yeast COMPASS/Set1C ( Vermeulen and Timmers, 2010 ). Biochemical studies have shown that human SET1/MLL complexes possess variable intrinsic abilities to catalyze mono-, di-, and trimethylation of H3K4, although, intriguingly, these complexes appear to have largely nonredundant roles ( Eissenberg and Shilatifard, 2010 ). Notably, however, SET1C appears responsible for the majority of H3K4 trimethylation in mammalian cells ( Shilatifard, 2012 ).

Genome-wide studies have shown that H3K4me3 is localized mainly to promoter regions, whereas H3K4me1 is enriched at enhancers ( Heintzman et al., 2007 ; Zhou et al., 2011 ), raising the general question of whether these events are mediated by distinct enzymes or by position-specific activities of a common enzyme. Thus, distinct recruitment and/or enzymatic regulatory mechanisms for SET1/MLL complexes might be critical for the position-specific distribution of the distinct methylation states. A variety of alternative (nonmutually exclusive) mechanisms for recruitment (or stabilization) of SET1/MLL complexes have been described (reviewed in Ruthenburg et al., 2007 ; Schuettengruber et al., 2011 ; Smith and Shilatifard, 2010 ), with recruitment by sequence-specific DNA-binding transcription factors ( Goo et al., 2003 ; Vermeulen and Timmers, 2010 ) being of special interest here. Position-specific H3K4 methylation states might also be regulated by other factors, including other (local) histone modifications. In this regard, early studies in yeast reported that H2B ubiquitylation, which is generally coupled to transcription, is essential for H3K4 trimethylation, but not H3K4 monomethylation, by COMPASS/Set1C ( Shilatifard, 2012 ), whereas more recent studies have shown that ubiquitylated H2B directly stimulates H3K4 trimethylation of chromatin templates by the human and yeast SET1 complexes ( Kim et al., 2009 , 2013 ).

As a transcription factor, the tumor suppressor p53 is tightly regulated by complex mechanisms, which include various covalent modifications, in response to distinct cellular stresses ( Kruse and Gu, 2009 ; Vousden and Prives, 2009 ). Although p53 is well studied with respect to its regulation, the mechanisms by which it differentially activates target genes through interactions with various cofactors, in particular those that effect epigenetic changes in chromatin, are less well understood. However, in relation to the H3K4 methylation mechanisms of interest here, previous studies have demonstrated DNA-damage-induced accumulation of H3K4me3, as well as other histone modifications, on p53 target genes ( An et al., 2004 ; Lee et al., 2009 ).

To further investigate the factors and mechanisms involved in p53-dependent H3K4 methylation, we have employed cell-free systems reconstituted with purified factors and recombinant chromatin templates in conjunction with cell-based assays of a p53-dependent DNA-damage response. Our results establish a p53-p300/H3ac-SET1C interaction network that leads to synergistic SET1C and p300 functions in H3K4 methylation and transcription activation and, overall, insights into underlying mechanisms.

DISCUSSION

A variety of genetic and biochemical studies have provided important examples of crosstalk between different histone modifications ( Lee et al., 2010 ; Suganuma and Workman, 2011 ). Here, in relation to p53-dependent transcription, we have established strong physical and functional linkages between p300 and SET1C and their corresponding histone modifications. These results, discussed below, have revealed mechanistic insights into the regulation of H3K4 methylation that are likely of broad significance.
