---
pmid: '21798944'
title: Evidence for network evolution in an Arabidopsis interactome map.
authors:
- Arabidopsis Interactome Mapping Consortium
journal: Science
year: '2011'
full_text_available: false
full_text_extraction_method: html_abstract_only
pmcid: PMC3170756
doi: 10.1126/science.1203877
---

# Evidence for network evolution in an Arabidopsis interactome map.
**Authors:** Arabidopsis Interactome Mapping Consortium
**Journal:** Science (2011)
**DOI:** [10.1126/science.1203877](https://doi.org/10.1126/science.1203877)
**PMC:** [PMC3170756](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3170756/)

## Abstract

1. Science. 2011 Jul 29;333(6042):601-7. doi: 10.1126/science.1203877.

Evidence for network evolution in an Arabidopsis interactome map.

Arabidopsis Interactome Mapping Consortium(1).

Collaborators: Braun P, Carvunis AR, Charloteaux B, Dreze M, Ecker JR, Hill DE, 
Roth FP, Vidal M, Galli M, Balumuri P, Bautista V, Chesnut JD, Kim RC, de los 
Reyes C, Gilles P, Kim CJ, Matrubutham U, Mirchandani J, Olivares E, Patnaik S, 
Quan R, Ramaswamy G, Shinn P, Swamilingiah GM, Wu S, Ecker JR, Dreze M, Byrdsong 
D, Dricot A, Duarte M, Gebreab F, Gutierrez BJ, MacWilliams A, Monachello D, 
Mukhtar MS, Poulin MM, Reichert P, Romero V, Tam S, Waaijers S, Weiner EM, Vidal 
M, Hill DE, Braun P, Galli M, Carvunis AR, Cusick ME, Dreze M, Romero V, Roth 
FP, Tasan M, Yazaki J, Braun P, Ecker JR, Carvunis AR, Ahn YY, Barab√°si AL, 
Charloteaux B, Chen H, Cusick ME, Dangl JL, Dreze M, Ecker JR, Fan C, Gai L, 
Galli M, Ghoshal G, Hao T, Hill DE, Lurin C, Milenkovic T, Moore J, Mukhtar MS, 
Pevzner SJ, Przulj N, Rabello S, Rietman EA, Rolland T, Roth FP, Santhanam B, 
Schmitz RJ, Spooner W, Stein J, Tasan M, Vandenhaute J, Ware D, Braun P, Vidal 
M.

Author information:
(1)Center for Cancer Systems Biology (CCSB) and Department of Cancer Biology, 
Dana-Farber Cancer Institute, Boston, MA 02215, USA.

Comment in
    Science. 2011 Jul 29;333(6042):532-3. doi: 10.1126/science.1209753.
    Nat Rev Genet. 2011 Aug 02;12(9):586. doi: 10.1038/nrg3059.
    Nat Rev Genet. 2011 Aug 09;12(9):586. doi: 10.1038/nrg3058.

Plants have unique features that evolved in response to their environments and 
ecosystems. A full account of the complex cellular networks that underlie 
plant-specific functions is still missing. We describe a proteome-wide binary 
protein-protein interaction map for the interactome network of the plant 
Arabidopsis thaliana containing about 6200 highly reliable interactions between 
about 2700 proteins. A global organization of plant biological processes emerges 
from community analyses of the resulting network, together with large numbers of 
novel hypothetical functional links between proteins and pathways. We observe a 
dynamic rewiring of interactions following gene duplication events, providing 
evidence for a model of evolution acting upon interactome networks. This and 
future plant interactome maps should facilitate systems approaches to better 
understand plant biology and improve crops.

DOI: 10.1126/science.1203877
PMCID: PMC3170756
PMID: 21798944 [Indexed for MEDLINE]

## Full Text

Abstract

Plants have unique features that evolved in response to their environments and ecosystems. A full account of the complex cellular networks that underlie plant-specific functions is still missing. We describe a proteome-wide binary protein-protein interaction map for the interactome network of the plant Arabidopsis thaliana containing ~6,200 highly reliable interactions between ~2,700 proteins. A global organization of plant biological processes emerges from community analyses of the resulting network, together with large numbers of novel hypothetical functional links between proteins and pathways. We observe a dynamic rewiring of interactions following gene duplication events, providing evidence for a model of evolution acting upon interactome networks. This and future plant interactome maps should facilitate systems approaches to better understand plant biology and improve crops.

Conclusion

Our empirically determined high-quality protein-protein interaction map for a plant interactome network should not only hasten the functional characterization of unknown proteins, including those with potential biotechnological utility, but also enable systems level investigations of genotype-to-phenotype relationships in the plant kingdom. One example is how AI-1 illuminates mechanisms and strategies by which plants cope with pathogenic challenges (Mukhtar et al. , co-submitted).

The paradigms established here are compatible with models in which the interactome network constrains and shapes sequence evolution. Studying sequence variation, conservation, mutation, and evolution rate has shed light on how natural selection drives evolution. Explorations of interaction variation will similarly broaden the understanding of network evolution whether in the context of duplication or trans-kingdom comparative interactomics.
