---
pmid: '23152499'
title: Hepatitis C virus NS5B and host cyclophilin A share a common binding site on
  NS5A.
authors:
- Rosnoblet C
- Fritzinger B
- Legrand D
- Launay H
- Wieruszeski JM
- Lippens G
- Hanoulle X
journal: J Biol Chem
year: '2012'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC3531740
doi: 10.1074/jbc.M112.392209
---

# Hepatitis C virus NS5B and host cyclophilin A share a common binding site on NS5A.
**Authors:** Rosnoblet C, Fritzinger B, Legrand D, Launay H, Wieruszeski JM, Lippens G, Hanoulle X
**Journal:** J Biol Chem (2012)
**DOI:** [10.1074/jbc.M112.392209](https://doi.org/10.1074/jbc.M112.392209)
**PMC:** [PMC3531740](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531740/)

## Abstract

1. J Biol Chem. 2012 Dec 28;287(53):44249-60. doi: 10.1074/jbc.M112.392209. Epub 
2012 Nov 14.

Hepatitis C virus NS5B and host cyclophilin A share a common binding site on 
NS5A.

Rosnoblet C(1), Fritzinger B, Legrand D, Launay H, Wieruszeski JM, Lippens G, 
Hanoulle X.

Author information:
(1)Université Lille Nord de France, F-59000 Lille, France.

Nonstructural protein 5B (NS5B) is essential for hepatitis C virus (HCV) 
replication as it carries the viral RNA-dependent RNA polymerase enzymatic 
activity. HCV replication occurs in a membrane-associated multiprotein complex 
in which HCV NS5A and host cyclophilin A (CypA) have been shown to be present 
together with the viral polymerase. We used NMR spectroscopy to perform a per 
residue level characterization of the molecular interactions between the 
unfolded domains 2 and 3 of NS5A (NS5A-D2 and NS5A-D3), CypA, and NS5B(Δ21). We 
show that three regions of NS5A-D2 (residues 250-262 (region A), 274-287 (region 
B), and 306-333 (region C)) interact with NS5B(Δ21), whereas NS5A-D3 does not. 
We show that both NS5B(Δ21) and CypA share a common binding site on NS5A that 
contains residues Pro-306 to Glu-323. No direct molecular interaction has been 
detected by NMR spectroscopy between HCV NS5B(Δ21) and host CypA. We show that 
cyclosporine A added to a sample containing NS5B(Δ21), NS5A-D2, and CypA 
specifically inhibits the interaction between CypA and NS5A-D2 without altering 
the one between NS5A-D2 and NS5B(Δ21). A high quality heteronuclear NMR spectrum 
of HCV NS5B(Δ21) has been obtained and was used to characterize the binding site 
on the polymerase of NS5A-D2. Moreover these data highlight the potential of 
using NMR of NS5B(Δ21) as a powerful tool to characterize in solution the 
interactions of the HCV polymerase with all kinds of molecules (proteins, 
inhibitors, RNA). This work brings new insights into the comprehension of the 
molecular interplay between NS5B, NS5A, and CypA, three essentials proteins for 
HCV replication.

DOI: 10.1074/jbc.M112.392209
PMCID: PMC3531740
PMID: 23152499 [Indexed for MEDLINE]

## Full Text

Abstract

Nonstructural protein 5B (NS5B) is essential for hepatitis C virus (HCV) replication as it carries the viral RNA-dependent RNA polymerase enzymatic activity. HCV replication occurs in a membrane-associated multiprotein complex in which HCV NS5A and host cyclophilin A (CypA) have been shown to be present together with the viral polymerase. We used NMR spectroscopy to perform a per residue level characterization of the molecular interactions between the unfolded domains 2 and 3 of NS5A (NS5A-D2 and NS5A-D3), CypA, and NS5B Δ21 . We show that three regions of NS5A-D2 (residues 250–262 (region A), 274–287 (region B), and 306–333 (region C)) interact with NS5B Δ21 , whereas NS5A-D3 does not. We show that both NS5B Δ21 and CypA share a common binding site on NS5A that contains residues Pro-306 to Glu-323. No direct molecular interaction has been detected by NMR spectroscopy between HCV NS5B Δ21 and host CypA. We show that cyclosporine A added to a sample containing NS5B Δ21 , NS5A-D2, and CypA specifically inhibits the interaction between CypA and NS5A-D2 without altering the one between NS5A-D2 and NS5B Δ21 . A high quality heteronuclear NMR spectrum of HCV NS5B Δ21 has been obtained and was used to characterize the binding site on the polymerase of NS5A-D2. Moreover these data highlight the potential of using NMR of NS5B Δ21 as a powerful tool to characterize in solution the interactions of the HCV polymerase with all kinds of molecules (proteins, inhibitors, RNA). This work brings new insights into the comprehension of the molecular interplay between NS5B, NS5A, and CypA, three essentials proteins for HCV replication.

Introduction

Hepatitis C virus (HCV) 5 is a small RNA virus from the Flaviviridae family. Since its discovery in the late 80s HCV has infected an estimated 130–200 million people worldwide ( 1 ). HCV chronic infection is one of the main factors leading to liver cirrhosis and hepatocellular carcinoma and thus represents a serious public health challenge for almost all countries. HCV has been classified into six major genotypes (1–6) according to its RNA genomic sequence. Among them, genotype 1 is the most prevalent all over the world except for Africa where genotype 4 represents nearly half of the infections ( 1 ). The standard of care to treat HCV infection consists of a combination of ribavirin and pegylated-interferon α ( 2 ). This therapeutic strategy suffers both from a low efficacy in the case of genotype 1 infection and from non-negligible side effects for treated patients. The development of more efficient and safer new anti-HCV molecules is thus highly expected ( 3 , 4 ). HCV is an obligatory intracellular parasite and requires host cell factors in addition to its own components to achieve its complete life cycle ( 5 ). Consequently, anti-HCV drugs may be either direct-acting agents or host-targeting agents. Direct-acting agents have the advantage to be highly specific but may elicit resistance mutations selection, thereby limiting their long-time efficacy. The second category of molecules is thought to have a higher genetic barrier for resistance mutations but may interfere with host cellular processes. In both cases a better molecular characterization of the virus-virus and virus-host interactions mechanisms potentially will lead to more efficient anti-HCV drugs.

The HCV (+)RNA genome encodes for a unique polyprotein precursor (∼3000 amino acids) that after processing by viral and host proteases results in 10 different proteins divided into structural (Core, E1, E2) and non-structural (p7, NS2, NS3, NS4A, NS4B, NS5A, and NS5B) proteins ( 5 , 6 ). NS3 to NS5B constitute the minimal set of viral proteins that assures the viral RNA replication; all are anchored into the endoplasmic reticulum host membrane to form the replication complex. NS3, which carries both RNA helicase and serine protease activities, is actually one of the main targets for direct-acting agent development, with two compounds (telaprevir and boceprevir) that were last year Food and Drug Administration-approved ( 4 ). In addition to NS3, NS5B constitutes another promising target for direct-acting agent development ( 2 , 7 ).

NS5B is the central protein (65 kDa) for the HCV replication process as it carries the RNA-dependent RNA polymerase enzymatic activity ( 8 – 10 ). The last 21 residues at the C terminus of NS5B are required for endoplasmic reticulum membrane anchoring ( 11 , 12 ) and in vivo HCV replication ( 13 ), but they are not absolutely required for its enzymatic activity in vitro ( 14 , 15 ). NS5B exhibits a similar “right-hand” shape as other RNA-dependent RNA polymerases, with thumb, palm, and finger subdomains ( 16 – 18 ). NS5B inhibitors are divided into nucleoside inhibitors (NIs), which bind into the active site and are terminator substrates of the RNA polymerase activity, and non-nucleoside inhibitors (NNIs) that bind into one of the allosteric sites of the polymerase. Numerous crystal structures of NS5B in complex with NNIs have led to the identification of at least four major NNI binding sites (for review, see Ref. 2 ). However, the conformational changes that follow NNI binding ( 19 – 24 ) can interfere with proper crystal packing, thereby rendering crystallographic results difficult to interpret.

In vivo , HCV RNA replication occurs in a replication complex that contains endoplasmic reticulum-derived membrane(s), NS3 to NS5B viral proteins ( 25 ), and viral RNA as well as several host factors such as hVAP-33 ( 26 ), VAP-B ( 27 ), and cyclophilin A (CypA) ( 28 ). The latter is a major host factor required for HCV replication ( 29 ). It belongs to the cyclophilin (Cyp) family of proteins that bind cyclosporin A (CsA) and exhibit peptidyl-prolyl cis-trans isomerase activity ( 30 ). Cyps are able to catalyze the cis-trans isomerization of a peptide bond preceding a proline residue. Cyps have been identified as the main target of the early observed anti-HCV activity of CsA ( 31 ) or, more recently, of its non-immunosupressive derivatives ( 32 – 34 ). One of them, Debio-025 (alisporivir), is currently evaluated in a phase III clinical trial. Watashi et al. ( 35 ) initially proposed that cyclophilin B would be the target of CsA and proposed a mechanism whereby CypB would be a positive regulator of NS5B by modulating its RNA binding activity. Later, several members of the cyclophilins have been proposed to be involved into the HCV replication ( 29 , 31 ), and NS5B interactions with CypB ( 35 – 39 ) and CypA ( 28 , 29 , 36 , 40 ) have been reported. However, more recent evidence points to CypA as the major key player both for HCV replication and CsA sensitivity ( 29 , 41 ). The molecular mechanisms that confer to cyclophilin inhibitors an anti-HCV activity remain to be identified. In vitro selection for CsA-resistance mutations pointed out three viral non-structural proteins: NS2, NS5A, and NS5B as potential cyclophilin partners ( 37 , 41 , 42 ). Moreover, mutations located in the second half of NS5A confer the highest level of resistance to cyclophilin inhibitors ( 37 , 43 ).

NS5A is a 49-kDa phosphoprotein with RNA binding properties ( 44 ) that is required for HCV RNA replication and viral particle production ( 45 – 47 ) but for which up to now no enzymatic activity has been identified. The protein is involved in several steps of the HCV life cycle and interacts with almost all other non-structural proteins and with numerous host factors ( 48 , 49 ). NS5A is anchored at the cytoplasmic side of the endoplasmic reticulum membrane via an N-terminal helix ( 50 ) and comprises 3 cytosolic domains (D1, D2, and D3). NS5A-D1 carries most of the RNA binding properties of the protein. It has been shown that NS5A-D1 may adopt two different hetero-dimeric structures, at least in the context of a crystal ( 51 , 52 ). NS5A-D2 (250–342), which is essential for RNA replication ( 47 ), and NS5A-D3 (356–447), which is involved in the production and assembly of viral particles ( 45 ), have been shown to be natively unfolded ( 53 – 57 ). In previous studies we have shown that both NS5A-D2 ( 53 ) and NS5A-D3 ( 57 ) establish direct interactions with CypA in vitro and that these domains are substrates for the peptidyl-prolyl cis-trans isomerase activity of CypA. Using NMR spectroscopy, we have mapped onto NS5A-D2 the CypA interactions sites and found a correlation with CsA-resistance mutations or even mutations that impair RNA replication ( 53 ).

HCV NS5B and NS5A have been shown to interact together both in vivo and in vitro ( 58 – 60 ). Conflicting results have been reported about the functional consequences of this protein-protein interaction. Indeed, depending on the experimental setups used it has been shown that NS5A may either stimulate ( 59 , 61 ) or inhibit ( 59 , 62 , 63 ) the NS5B RNA-dependent RNA polymerase activity. At the molecular level, a strategy based on how NS5B mutations affect the NS5B-NS5A pulldown efficacy identified four discontinuous regions of NS5B (139–145, 149–155, 365–371, and 382–388) as essential for NS5A binding ( 60 ). Using truncated versions of NS5A, two regions of this protein (105–162 and 277–334) have been shown to be independently essential for the interaction with NS5B ( 59 ). Nevertheless, the absence of a structure of the complex leaves open many questions about the functional consequences of this interaction.

HCV NS5B, NS5A, and host CypA are three essential components of the viral replication complex and as such form three attractive targets for anti-HCV drugs. However, conflicting results have been reported as for the role of CypA for the proper association of these three proteins into the replication complex ( 28 , 40 ). Here we report NMR spectroscopy analyses at a per residue level of the molecular interactions between NS5A domains 2 and 3, NS5B, and CypA. We show that both HCV NS5B and host CypA share a common binding site on NS5A and use NMR of the NS5B enzyme to get a first localization of the NS5A-binding site on NS5B.

DISCUSSION

The replication of HCV together with the polyprotein processing constitute main targets for development of anti-HCV drugs. As the replication process requires the formation of a membrane-associated multiprotein complex, its impairment can be done by hitting several individual targets, which can be viral or even host factors. The drugs that are currently in development and that act through the inhibition of HCV replication target either the HCV RNA-dependent RNA polymerase NS5B, NS5A, or CypA, an essential host factor. These three proteins are essential components of the replicase and have all been reported to interact two by two. Here we characterized these molecular interactions at a per residue level by NMR spectroscopy ( Fig. 6 ).

Through the analysis of the chemical shift perturbations in the NS5A-D2 NMR spectrum that were induced after the addition of unlabeled NS5B Δ21 , we identified three regions in NS5A-D2 that are involved in the interaction with the polymerase (see Fig. 1 , A and C ). These NS5B-binding sites correlate with the regions in the unfolded domain 2 of NS5A that exhibit some residual secondary structure propensity (see Fig. 1 , B and C ). NS5A-D2 thereby joins the growing list of unfolded proteins that interact with relevant biological partners through their residual structural elements ( 73 , 74 ). Using the same NMR strategy, we showed that the unfolded domain 3 of NS5A does not contribute to the binding of NS5B (see Figs. 3 A and 6 ). Our results thereby complete the ones from Shirota et al. ( 59 ) who had performed a rough identification of the NS5B-binding site on NS5A using internal deletion mutants. They identified in genotype 1b two discontinuous regions of NS5A, residues 105–162 and 277–334, that are both required for proper interaction with NS5B. NS5A residues 277–334 in genotype 1b correspond to residues 273–330 in the genotype 2a we used in this study. As it has been later shown that region 105–162 belongs to the folded domain 1 of NS5A, its deletion may abolish the overall three-dimensional structure of this domain and render the previous result difficult to interpret. The second described region 273–330 (genotype 2a numbering) encompasses two different binding regions, 274–287 and 306–333, that we identified at a per residue level in this work (see Fig. 1 C ). Shirota et al. ( 59 ) have reported that the regions 105–162 and 273–330 (genotype 2a numbering), located into domain 1 and domain 2 of NS5A, respectively, are independently essential for the binding of NS5B. We do not confirm this conclusion, as we showed that the isolated domain 2 of NS5A (residues 248–341) is sufficient to interact with the polymerase. However, our NMR assay does not allow distinguishing whether regions A to C of NS5A-D2 interact at the same time with the same NS5B molecule or if individual regions A, B, and C interact at distinct moments with different NS5B molecules. Nevertheless, the Scatchard plot of the SPR data is consistent with a 1:1 Langmuir model (see supplemental Fig. 3 ). Considering the conservation level of the NS5A-D2 primary sequence among all HCV genotypes, the NS5B-interacting regions A, B, and C correspond to regions with less variability and even contain some strictly conserved residues (Ala-257, Asn-258, Asp-278, Pro-310, Trp-312, Ala-313, Pro-315, Pro-319, Trp-325 and Tyr-330) (see supplemental Fig. 11 and Ref. 53 ). This indicates that the three regions we identified in NS5A-D2 are functionally important for the virus; one possibility is that these regions are required for the proper interaction of NS5A-D2 with NS5B and hence for an efficient replication mechanism.

The NS5A-D2 region (273–330, genotype 2a numbering), identified by Shirota et al. ( 59 ) to be required for NS5B binding, was further characterized in two subregions. Alanine substitutions in residues 273–305 (genotype 2a numbering) partially impaired the level of replication, whereas residues 306–330 are essential as their substitutions led to absence of RNA replication ( 58 ). However, the absence of replication when NS5A-D2 residues 306–330 were alanine-substituted may not be directly related to an impaired NS5A-NS5B interaction but may also be linked to the functional interaction with host CypA. Indeed we showed here that NS5A-D2 residues G-304–Glu-323 and G337–Pro-341 constitute CypA-binding sites (see Fig. 1 D ). This result is identical with what we initially described ( 53 ) and is coherent with the tandem cyclophilin-binding site of NS5A that has recently been reported by Grisé et al. ( 69 ). We show here that HCV NS5B and host CypA share a common binding site on NS5A that contains residues Pro-306 to Glu-323 (see Fig. 1 , C and D ). Caution in the interpretation of replication assays with mutations and/or deletions in this particular NS5A region is thus required, as the observed effect may arise either from a modification of the NS5A-NS5B interaction and/or of the NS5A-CypA interaction ( Fig. 6 ). We did not detect any direct physical interaction between CypA and NS5B ( Fig. 6 ), in contradiction with several previous reports ( 28 , 29 , 36 , 40 ). The absence of chemical shift perturbations in the CypA spectrum upon the addition of NS5B (see Fig. 3 B ) was confirmed by the reverse experiment whereby the NMR spectrum of NS5B did not appreciably change upon the addition of CypA (data not shown). Although this apparent discrepancy may arise from the different constructs or genotypes that were used, a more plausible hypothesis is that the described CypA-NS5B interaction was indirect and mediated by a third interacting partner. Indeed, to our knowledge no studies have been done in vitro with only the two purified proteins but rather with cell lysate pulldown experiments ( 29 , 40 ), crude replication complex preparations pulldown experiments ( 28 ), or in in vitro translation reactions ( 36 ) that at least contain RNAs. Without interaction between NS5B and CypA, the interplay between HCV NS5B, NS5A, and host CypA, three essential viral replicase components, may be driven by the common binding site on NS5A-D2 that we identified. When we mixed NS5A-D2 with both NS5B Δ21 and CypA in the same tube, we observed in the NS5A-D2 NMR spectrum the sum of the spectral perturbations (see Fig. 1 E ) that were individually observed by the addition of either NS5B Δ21 (see Fig. 1 C ) or CypA (see Fig. 1 D ). Our NMR results are in favor of a model where NS5A-D2 would interact with similar affinities either with CypA or with NS5B ( Fig. 6 ). We previously estimated by NMR spectroscopy the affinity corresponding to the CypA-NS5A-D2 interaction at 64 μ m ( 53 ), and in this work, using SPR, we measured a K D of ∼18 μ m (see supplemental Fig. 4 A ). The fact that these two values were determined by different methods in solution or at surface may explain the relative variation. For the NS5B-NS5A-D2 interaction, using SPR we measured a K D of ∼21 μ m (see Fig. 2 ). Thus the interaction strengths between NS5A-D2 and either CypA or NS5B are indeed quite similar. Moreover, we show that CsA only disrupts the molecular interaction between CypA and NS5A-D2 without interfering with the NS5A-D2-NS5B interaction ( Fig. 6 ). Grisé et al. ( 69 ) recently described that they have preliminary data indicating the same findings. Chatterji et al. ( 75 ) have shown that CsA treatment has no influence on the association of NS5A and NS5B with the replication complex but rather depletes it from CypA. Their findings and our data are in favor of a model where the association of NS5A and NS5B into the replication complex would be mediated by direct interaction between these two viral proteins and that CypA incorporation in this functional complex would be done via its interaction with domain 2 of NS5A. This model is strengthened by our observation that CypA and NS5B have different cis-trans isomer specificities for NS5A-D2 binding in region B (see supplemental Figs. 6 and 7 ).

In this study we report the first high quality heteronuclear NMR spectrum of HCV NS5B (see Fig. 4 and supplemental Fig. 8 ) and show that this spectrum can be used to monitor molecular interactions directly onto the polymerase (see Fig. 5 and supplemental Fig. 9 ). Indeed we detected shifts and/or broadenings for a limited number of peaks in the NMR spectrum of a double-labeled 2 H, 15 N-NS5B Δ21 in the presence of unlabeled NS5A-D2 (see Fig. 5 , A and B , and supplemental Fig. 9 B ). These effects, which correspond to fast exchange, are in agreement with the K D of ∼21 μ m that we measured by SPR. The limited number of peaks that were affected indicates that NS5B does not undergo a global conformational change upon NS5A-D2 binding. This possibility has often been proposed to explain the effect, positive or negative, of NS5A-D2 on the RNA-dependent RNA polymerase enzymatic activity of NS5B ( 61 , 62 ). Currently, we have only started to assign the NMR peaks of NS5B Δ21 , but it is already clear that even a partial assignment will require intensive biochemistry and NMR spectroscopy efforts. Nevertheless, we used an indirect strategy and showed that the NS5A-D2-binding site on NS5B partially overlaps with the binding site of a TCA inhibitor (see Fig. 5 and supplemental Fig. 9 ), which is known to bind into the thumb site 2 of the polymerase ( 2 , 20 , 65 ) (see supplemental Fig. 10 ). NMR hence should allow the direct mapping of the residues involved into various molecular interactions without using deletion or resistance mutants that may interfere with the global structure of the polymerase. Direct solution NMR spectroscopy on NS5B Δ21 should also allow the study of compounds that induce relatively large conformational changes upon binding and that may interfere with crystal packing ( 19 , 20 , 23 ). Finally, the NMR spectrum of NS5B Δ21 that we obtained may be used to characterized low affinity (over the micromolar range) interactions mechanisms as is the case with NS5A-D2.

In conclusion, we report here the first direct high resolution characterization of the molecular interactions between HCV NS5A, NS5B, and host CypA. We show that NS5B and CypA share a binding site on NS5A-D2 and that CsA only disrupts the NS5A-D2-CypA interaction. Further studies will be required to make a link between the interactions we observed in vitro and their potential effects on the RNA-dependent RNA polymerase activity of the HCV polymerase. The NMR spectrum of HCV NS5B Δ21 furthermore constitutes a powerful tool to investigate its potential interactions with various interaction partners such as other proteins, small inhibitors, or even RNAs.
