---
pmid: '29128334'
title: A Map of Human Mitochondrial Protein Interactions Linked to Neurodegeneration
  Reveals New Mechanisms of Redox Homeostasis and NF-κB Signaling.
authors:
- Malty RH
- Aoki H
- Kumar A
- Phanse S
- Amin S
- Zhang Q
- Minic Z
- Goebels F
- Musso G
- Wu Z
- Abou-Tok H
- Meyer M
- Deineko V
- Kassir S
- Sidhu V
- Jessulat M
- Scott NE
- Xiong X
- Vlasblom J
- Prasad B
- Foster LJ
- Alberio T
- Garavaglia B
- Yu H
- Bader GD
- Nakamura K
- Parkinson J
- Babu M
journal: Cell Syst
year: '2017'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC5746455
doi: 10.1016/j.cels.2017.10.010
---

# A Map of Human Mitochondrial Protein Interactions Linked to Neurodegeneration Reveals New Mechanisms of Redox Homeostasis and NF-κB Signaling.
**Authors:** Malty RH, Aoki H, Kumar A, Phanse S, Amin S, Zhang Q, Minic Z, Goebels F, Musso G, Wu Z, Abou-Tok H, Meyer M, Deineko V, Kassir S, Sidhu V, Jessulat M, Scott NE, Xiong X, Vlasblom J, Prasad B, Foster LJ, Alberio T, Garavaglia B, Yu H, Bader GD, Nakamura K, Parkinson J, Babu M
**Journal:** Cell Syst (2017)
**DOI:** [10.1016/j.cels.2017.10.010](https://doi.org/10.1016/j.cels.2017.10.010)
**PMC:** [PMC5746455](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5746455/)

## Abstract

1. Cell Syst. 2017 Dec 27;5(6):564-577.e12. doi: 10.1016/j.cels.2017.10.010. Epub
 2017 Nov 8.

A Map of Human Mitochondrial Protein Interactions Linked to Neurodegeneration 
Reveals New Mechanisms of Redox Homeostasis and NF-κB Signaling.

Malty RH(1), Aoki H(1), Kumar A(2), Phanse S(1), Amin S(1), Zhang Q(1), Minic 
Z(1), Goebels F(3), Musso G(4), Wu Z(1), Abou-Tok H(1), Meyer M(5), Deineko 
V(1), Kassir S(1), Sidhu V(1), Jessulat M(1), Scott NE(6), Xiong X(7), Vlasblom 
J(1), Prasad B(8), Foster LJ(6), Alberio T(9), Garavaglia B(10), Yu H(5), Bader 
GD(3), Nakamura K(11), Parkinson J(7), Babu M(12).

Author information:
(1)Department of Biochemistry, University of Regina, Regina, SK S4S 0A2, Canada.
(2)Department of Computer Science, University of Regina, Regina, SK S4S 0A2, 
Canada.
(3)The Donnelly Centre, University of Toronto, Toronto, ON M5S 3E1, Canada.
(4)Department of Medicine, Harvard Medical School and Cardiovascular Division, 
Brigham and Women's Hospital, Boston, MA 02115, USA.
(5)Department of Biological Statistics and Computational Biology, Cornell 
University, Ithaca, NY 14853, USA.
(6)Department of Biochemistry and Molecular Biology, University of British 
Columbia, Vancouver, BC V6T 1Z3, Canada.
(7)Hospital for Sick Children, 21-9830 PGCRL, 686 Bay Street, Toronto, ON M5G 
0A4, Canada.
(8)Department of Medicine, Regina Qu'Appelle Health Region, Regina, SK S4P 0W5, 
Canada.
(9)Department of Science and High Technology, Center of Neuroscience, University 
of Insubria, Via Alberto da Giussano 12, Busto Arsizio I-21052, Italy.
(10)Molecular Neurogenetics Unit, IRCCS Foundation C. Besta Neurological 
Institute, via L. Temolo, 4, 20126 Milan, Italy.
(11)Gladstone Institute of Neurological Disease, San Francisco, CA 94158, USA.
(12)Department of Biochemistry, University of Regina, Regina, SK S4S 0A2, 
Canada. Electronic address: mohan.babu@uregina.ca.

Comment in
    Cell Syst. 2017 Dec 27;5(6):544-546. doi: 10.1016/j.cels.2017.12.009.

Mitochondrial protein (MP) dysfunction has been linked to neurodegenerative 
disorders (NDs); however, the discovery of the molecular mechanisms underlying 
NDs has been impeded by the limited characterization of interactions governing 
MP function. Here, using mass spectrometry (MS)-based analysis of 210 
affinity-purified mitochondrial (mt) fractions isolated from 27 epitope-tagged 
human ND-linked MPs in HEK293 cells, we report a high-confidence MP network 
including 1,964 interactions among 772 proteins (>90% previously unreported). 
Nearly three-fourths of these interactions were confirmed in mouse brain and 
multiple human differentiated neuronal cell lines by primary antibody 
immunoprecipitation and MS, with many linked to NDs and autism. We show that the 
SOD1-PRDX5 interaction, critical for mt redox homeostasis, can be perturbed by 
amyotrophic lateral sclerosis-linked SOD1 allelic variants and establish a 
functional role for ND-linked factors coupled with IκBɛ in NF-κB activation. Our 
results identify mechanisms for ND-linked MPs and expand the human mt 
interaction landscape.

Copyright © 2017 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.cels.2017.10.010
PMCID: PMC5746455
PMID: 29128334 [Indexed for MEDLINE]

## Full Text

INTRODUCTION

The importance of Mitochondria (mt) is underscored by a number of human diseases associated with mt dysfunction ( Nunnari and Suomalainen, 2012 ), including neurodegenerative disorders (NDs) such as Parkinson’s disease (PD) and Amyotrophic lateral sclerosis (ALS). Large-scale proteomic analyses have advanced our knowledge of the composition of the mt proteome in mouse and human ( Calvo et al., 2016 ; Pagliarini et al., 2008 ), allowing identification of mt disease gene candidates, and gene-disease associations for MPs ( Floyd et al., 2016 ; Malty et al., 2015 ). Active mechanisms coordinate the assembly of an estimated 1,500 nuclear- and 13 mt-encoded proteins into macromolecular complexes ( Malty et al., 2015 ). Yet, little is known about the biochemical organization of human MP complexes, and how human MPs interact within mt, between mt, and with proteins outside of the mt to contribute to the pathophysiology of mt diseases. These gaps impede the development of effective therapeutics for human mt disorders.

Since most cellular processes are mediated by macromolecular assemblies, the systematic identification of mt protein-protein interactions (mtPPIs) and the subunit composition of complexes can provide insights into protein function and decipher the molecular basis of disease. As a result, characterization of disease-linked PPI networks is being exploited as a means of therapeutic target identification and to develop translational clinical applications ( Barabasi et al., 2011 ). While such human PPIs can be detected by several experimental means, affinity purification coupled with mass spectrometry (AP/MS) is well-suited for large-scale isolation and characterization of protein complexes in many evolutionarily diverse organisms, including humans [reviewed in ( Huttlin et al., 2017 )].

Despite the large and near-comprehensive human interaction networks generated to date ( Havugimana et al., 2012 ; Hein et al., 2015 ; Huttlin et al., 2017 ), very few MPs have been used as baits in previous studies, with interactions largely reported from human whole cell, nuclear, or cytosolic extracts. As a result, mt interactions have been under-sampled. Recent effort to map a condition-specific human mtPPI network in mammalian cells has been successful in assigning functions for some previously uncharacterized MPs ( Floyd et al., 2016 ), but inherent technical challenges associated with organelle purification have limited the coverage of the human mtPPI network and cell-type specific interactions in mammalian cells, particularly among ND-linked proteins. Here, to overcome these problems we report an AP/MS framework, with experimental parameters optimized to systematically tag and affinity-purify the mt fraction isolated from 27 well-annotated ND-linked MPs expressed in human embryonic kidney 293 (HEK293) cells, identifying their interaction partners by precision MS.

The assembled PPI network encompasses putative hetero- and multi-meric MP complexes, including several not previously reported. Since the tagged MPs are linked with NDs, we compared the PPIs of these proteins in HEK293 cells to those identified from differentiated SH-SY5Y neuronal cells, and identified prey proteins associated with autism spectrum disorders (ASD). Finally, we show the biological significance of selected connections between peroxiredoxin 5 (PRDX5) and ALS-linked CuZn-superoxide dismutase (SOD1), as well as between nuclear factor-κB (NF-κB) inhibitor ε (IκBε) and ND-linked proteins (DLD or PINK1, PARKIN) in mouse N9 microglial cells and human PD patient ( PINK1 or PARKIN mutations) fibroblasts. These interactions have direct implications for our understanding of cellular damage to mt, and the role of ND-related proteins in this process.

DISCUSSION

Despite being one of the most heavily studied cellular organelles, human MPs are still underrepresented in proteomic studies of physical interactions, impeding a systems-level understanding of mt biology and the development of effective therapeutics to treat diverse disorders of mt origin. The systematic proteomic analysis presented in this study offers a practical strategy to survey PPIs for all nuclear-encoded MPs in human cells. Due to the complexity of NDs, no cellular model can truly recapitulate ND states. However, we used a multi-cell line approach to generate mt PPI network, to provide us with as many ND-relevant interactions as possible. Rigorous verification of these interactions using several complementary experimental approaches, including an AP/MS screen consisting of multiple biological replicates using the same bait MP, and verification in distinct neuronal cell lineages and mouse brain, allowed us to confidently expand the composition of known complexes, providing opportunities for functional inference.

Among the PPIs observed, our network revealed previously overlooked associations in mt redox homeostasis and NF-κB. These links were substantiated by additional biochemical and genetic experiments, providing molecular mechanistic evidence of, among others, a SOD1-PRDX5 interaction critical for mt ROS production and detoxification. Our investigation of the IκBε-PINK1/PARKIN association in N9 cells has also yielded substantial findings, notably: (1) during mt depolarization, IεBε phosphorylation and ubiquitination are reliant on PINK1 and PARKIN, respectively; (2) the edgetic mutations in PINK1 / PARKIN PD patient fibroblasts perturb the IκBε binding, phosphorylation, and ubiquitination, linking genotype to phenotype relationships; (3) PINK1-PARKIN are capable of activating the NF-κB via IκBε in a manner that has implications for PD; and (4) contrary to known kinetin application to inhibit apoptosis of human neurons in a PINK1-dependent manner ( Hertz et al., 2013 ), CCCP-induced depolarization and kinetin-treated N9 cells that show a significant increase in NF-κB activation via PINK1 suggest that enhancing NF-κB activation in microglial cells by kinetin may contribute to neuronal inflammation and neurodegeneration. Similarly, the proteolytic effects of DLD on IκBε, detected in N9 cells, suggests a proteasomal degradation-independent mechanism of NF-κB activation upon mt stress. Overall, these insights into the biology of IκBε with PINK1-PARKIN or DLD in NF-kB deepen our understanding into neurological inflammation, with potential applications to PD pathogenesis.

Through a broader network driven approach, we were able to show a testable disease relevant outcome by highlighting an extensive subnetwork of ASD-linked components interacting with MPs. This includes providing insights on MPs regulated by the targets of key epigenetic regulators, MeCP2 and FMRP. While in-depth experimentation is required to solidify our understanding of mtPPIs relevant to MeCP2/FMRP-mediated ASD pathogenesis, our study clearly identifies a role of the MP network in ASD. While the network-based proteomics approach we used for ND-linked MPs illustrates the utility in capturing both stable and transient interactions to dissect mt (or disease) biology, experimental limitations exist with AP/MS-based protein interaction detection. For example, affinity-tagging can potentially interfere with protein localization and interactions, even modest overexpression of some MP baits can be toxic to human cells, detergent solubilization may disrupt hydrophobic interactions, and model cell lines used may not recapitulate all relevant processes in human neurons ( Babu et al., 2012 ; Hosp et al., 2015 ). To limit these confounding effects, we avoided artifacts from overexpression by considering only PPIs from the tagged bait MPs that were captured consistently when the same set of endogenously expressed MPs were purified with protein-specific Abs in HEK293 and/or SH-SY5Y cells. Also, we found that PPIs from biological replicates that surpass NSAF scoring thresholds provided PPIs with a reduced overall false positive rate.

Analysis of the MP network suggests that this AP/MS workflow can conceptually be applied globally for all human (or disease-linked) MPs as a means to examine the specificity and stability of MP complexes. Exploration of how disease-associated variants affect interactions among these complex members could identify avenues for therapeutic design for the vast assembly of mt diseases. By providing access to the PPI data via public repository (PRIDE), we have presented a resource for hypothesis generation to spur further exploration of the unique biology of mt at a systems level.

CoreMethod and Other Bioinformatic Analyses

To identify multiprotein complex assemblies, we partitioned the neuronal-relevant PPIs into distinct sub-networks using the CoreMethod algorithm ( Leung et al., 2009 ), encompassing a core and an attachment. A core subunit is the central member of a cluster, while an attachment is the subunit that interacts with at least half of its complex subunit memberships. While an attachment can be a member of many multiprotein complexes, a subunit of a cluster is designated as core component if they are: highly connected with other proteins, unique within a cluster, and bound by the attachment proteins to form a complex.

The algorithm considers two proteins, p1 and p2 , having degrees (i.e. number of PPIs) d1 and d2 , respectively, as core members of a complex if the interaction i ( i = 0 or 1) between these two proteins and the number of common neighbors between them c is assessed. The probability ( p -value) of proteins p1 and p2 is then computed, having ≥ i interactions and ≥ c common neighbors between them under the null hypothesis that degrees d1 and d2 are randomly assigned in the PPI network following a uniform distribution. The steps used in the calculation of p -value includes probabilities of p1 and p2 with: (i) i interactions between them and (ii) exactly c common neighbors. The product of probabilities obtained from (i) and (ii) were combined to derive a final p -value, where a significant small p-value suggest p1 and p2 as likely core proteins of the same complex. Using this strategy, the predicted complexes are ranked based on the p -value, and only clusters with P ≤ 0.05 were considered as significant and reported in Table S4 .

To identify ASD-relevant subnetwork of bait MPs and their interactions, we mapped proteins from the FLAG MP network to various literature sources ( Table S5 ). These included the SFARI (Simons Foundation Autism Research Initiative) Human Gene Module; AutDB, a gene reference resource for autism; and exome sequencing data of ASD patients from the Simons Simplex collection (2,377 families including 1,786 quads and trios, for a total of 8,917 exomes). Using these datasets, we found 46 interacting proteins with disease-associated variants in ASD probands or siblings ( Table S5 ). Nearly a dozen of these ASD-linked binding partners regulated by FMRP and/or MeCP2 ( Figure 4E ) were retrieved through literature curation.

All bait-prey pairs in the FLAG MP network were analyzed for enrichment of annotations in GO biological processes or PFAM domains using Gene Set Enrichment Analysis. RNA expression data of the human tissues from brain regions and spinal cord ( Figure S2C ), 206 samples in total ( GSM1117145 - GSM1117930 ; GSE45878 ), were downloaded from the NCBI Gene Expression Omnibus (GEO). Expression values were used to compute the Pearson’s correlation between all proteins as well as for the interacting proteins from SH-SY5Y neuronal cells for each brain region. In the case of RNA-seq expression profiles from the BrainSpan data ( Parikshak et al., 2013 ), expression value (FPKM) for each ASD-linked interacting protein, as well as for the corresponding bait MP, were compiled from the cortical development stage listed in Table S5 . However, since the data for each stage encompassed expression values from multiple regions of the brain, we computed the average expression value for each protein. Expression values were also averaged when multiple brain developmental stages (e.g. 8–10, 17–21, and 24–26 weeks after conception) were considered.
