---
pmid: '19060904'
title: An empirical framework for binary interactome mapping.
authors:
- Venkatesan K
- Rual JF
- Vazquez A
- Stelzl U
- Lemmens I
- Hirozane-Kishikawa T
- Hao T
- Zenkner M
- Xin X
- Goh KI
- Yildirim MA
- Simonis N
- Heinzmann K
- Gebreab F
- Sahalie JM
- Cevik S
- Simon C
- de Smet AS
- Dann E
- Smolyar A
- Vinayagam A
- Yu H
- Szeto D
- Borick H
- Dricot A
- Klitgord N
- Murray RR
- Lin C
- Lalowski M
- Timm J
- Rau K
- Boone C
- Braun P
- Cusick ME
- Roth FP
- Hill DE
- Tavernier J
- Wanker EE
- Barabási AL
- Vidal M
journal: Nat Methods
year: '2009'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2872561
doi: 10.1038/nmeth.1280
---

# An empirical framework for binary interactome mapping.
**Authors:** Venkatesan K, Rual JF, Vazquez A, Stelzl U, Lemmens I, Hirozane-Kishikawa T, Hao T, Zenkner M, Xin X, Goh KI, Yildirim MA, Simonis N, Heinzmann K, Gebreab F, Sahalie JM, Cevik S, Simon C, de Smet AS, Dann E, Smolyar A, Vinayagam A, Yu H, Szeto D, Borick H, Dricot A, Klitgord N, Murray RR, Lin C, Lalowski M, Timm J, Rau K, Boone C, Braun P, Cusick ME, Roth FP, Hill DE, Tavernier J, Wanker EE, Barabási AL, Vidal M
**Journal:** Nat Methods (2009)
**DOI:** [10.1038/nmeth.1280](https://doi.org/10.1038/nmeth.1280)
**PMC:** [PMC2872561](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2872561/)

## Abstract

1. Nat Methods. 2009 Jan;6(1):83-90. doi: 10.1038/nmeth.1280. Epub 2008 Dec 7.

An empirical framework for binary interactome mapping.

Venkatesan K(1), Rual JF, Vazquez A, Stelzl U, Lemmens I, Hirozane-Kishikawa T, 
Hao T, Zenkner M, Xin X, Goh KI, Yildirim MA, Simonis N, Heinzmann K, Gebreab F, 
Sahalie JM, Cevik S, Simon C, de Smet AS, Dann E, Smolyar A, Vinayagam A, Yu H, 
Szeto D, Borick H, Dricot A, Klitgord N, Murray RR, Lin C, Lalowski M, Timm J, 
Rau K, Boone C, Braun P, Cusick ME, Roth FP, Hill DE, Tavernier J, Wanker EE, 
Barabási AL, Vidal M.

Author information:
(1)Center for Cancer Systems Biology and Department of Cancer Biology, 
Dana-Farber Cancer Institute, 1 Jimmy Fund Way, Boston, MA 02115, USA.

Several attempts have been made to systematically map protein-protein 
interaction, or 'interactome', networks. However, it remains difficult to assess 
the quality and coverage of existing data sets. Here we describe a framework 
that uses an empirically-based approach to rigorously dissect quality parameters 
of currently available human interactome maps. Our results indicate that 
high-throughput yeast two-hybrid (HT-Y2H) interactions for human proteins are 
more precise than literature-curated interactions supported by a single 
publication, suggesting that HT-Y2H is suitable to map a significant portion of 
the human interactome. We estimate that the human interactome contains 
approximately 130,000 binary interactions, most of which remain to be mapped. 
Similar to estimates of DNA sequence data quality and genome size early in the 
Human Genome Project, estimates of protein interaction data quality and 
interactome size are crucial to establish the magnitude of the task of 
comprehensive human interactome mapping and to elucidate a path toward this 
goal.

DOI: 10.1038/nmeth.1280
PMCID: PMC2872561
PMID: 19060904 [Indexed for MEDLINE]

## Full Text

The protein-protein interactome of an organism is the network formed by all protein-protein interactions that can occur in a range of physiologically relevant protein concentrations. Mapping protein-protein interactions is crucial, albeit not sufficient, for unraveling the dynamic aspects of cellular networks, including when, where, and for what purpose protein interactions do occur in vivo1. Currently available human protein-protein interactome maps have been derived using (i) high-throughput yeast two-hybrid (HT-Y2H)2,3, (ii) HT co-affinity purification followed by mass spectrometry4, (iii) curation of published low-throughput experiments5–10, or (iv) computational predictions11,12. Despite a few attempts2,3,13,14, it remains difficult to accurately estimate the quality of these interactome maps and how far away we are from a complete map of the human interactome.

Previous attempts to assess the quality of interactome maps for human13,14,18 or other species13,15,18–23 relied on measuring (i) the extent to which interacting proteins share other biological attributes, e.g., co-expression, or (ii) the extent to which different maps of the same interactome share common interactions. Both approaches suffer several inherent limitations. Methods that evaluate the quality of interactions with respect to mRNA co-expression22,23 are systematically biased against true biological interactions between proteins whose mRNAs are not necessarily correlated or are even anti-correlated in expression. Since available annotations for protein function and localization are far from comprehensive, lack of evidence for co-localization of a given pair of proteins does not imply that the interaction observed between these proteins is an artifact. Methods based on measuring the extent of overlap between two interactome maps13,20,21 require that the corresponding datasets be derived from identical or similar assays. Existing analyses have not always fulfilled this requirement13. Most existing methods for quality assessment do not distinguish between the multiple sources of false negatives and false positives associated with any interactome mapping strategy. For instance, those interactions missed by a single screen of an assay but identifiable after multiple screens must be distinguished from the interactions that would never be identified by that assay even after a saturating number of screens.

Here we developed a framework to estimate various quality parameters associated with currently used protein-protein interaction assays, namely screening completeness, assay sensitivity, sampling sensitivity and precision. We generated empirical data to rigorously dissect these quality parameters, without relying on correlation with other biological attributes. Combining these parameters provides an estimate of the size of the human binary biophysical interactome and projects a path towards the completion of its mapping.

RESULTSAn interaction mapping frameworkTo accurately assess the quality of a given interactome map, we need to consider every possible source of false negatives (true interactions missing) and false positives (spurious pairs reported) associated with the assay used to generate the map. Our framework considers four parameters to estimate quality: “screening completeness”, “assay sensitivity”, “sampling sensitivity” and “precision” (Fig. 1).“Screening completeness” is the fraction of the total possible space of open reading frame (ORF) pairs that is tested to generate a given interactome map (Fig. 1a). Since currently available ORF resources3,24 only allow proteome-wide investigations of one protein isoform per gene, we ignore isoforms encoded by alternatively spliced transcripts here. For example, if we assume that the human genome consists of 22,500 protein-coding genes (N = 22,500 × 22,500/2 protein pairs), then the screening completeness of CCSB-HI12, a proteome-scale HT-Y2H effort that tested n = 7,000 × 7,000/2 human protein pairs, is n/N, or ~10%.“Assay sensitivity” is the fraction of all biophysical interactions that can possibly be identified by an assay performed under a specific set of experimental conditions (Fig. 1b). For example, a given HT-Y2H assay may be unable to detect interactions involving specific types of membrane proteins or requiring post-translational modifications that do not occur in yeast cells.“Sampling sensitivity” is the fraction of all identifiable interactions that are found in a single trial of an assay performed under a specific set of experimental conditions (Fig. 1c). When testing tens if not hundreds of millions of protein pairs in any space of pair-wise combinations, it might be necessary to sample that space multiple times to report all identifiable interactions.Lastly, “precision” is the fraction of observed pairs in an interactome dataset that are true positives (Fig. 1d). False positive pairs reflect technical artifacts that erroneously score positive in a given assay performed under a specific set of experimental conditions. We distinguish between two types of artifactual pairs, “stochastic false positives”, which are observed in only one or a few trials of an assay and “systematic false positives”, which are observed in many or all trials.Estimation of assay sensitivityEstimation of the assay parameters described above requires reference sets of positive and negative interacting pairs. To compile a positive reference set of high-confidence human binary protein-protein interactions we started with interactions curated from the literature (“literature curated” or LC interactions) and from these, we chose 188 pairs present in our human ORFeome v1.1 clone collection24 that are supported by the greatest number of publications and curated by the highest number of databases. Systematic recuration of all publications thought to support these 188 protein pairs25 verified 107 direct binary interactions that involve human proteins and that are supported by multiple publications. Ninety-two of these interactions involve full-length proteins and constituted our Homo sapiens Positive Reference Set version 1 or “hsPRS-v1” (Fig. 2a and Supplementary Table 1 online). Proteins involved in the 92 hsPRS-v1 interactions exhibit broad cellular localization (Fig. 2b), suggesting that they are representative of the entire human proteome. It is impossible to generate a set of negative interacting pairs with absolute confidence. So we compiled a surrogate Random Reference Set (“hsRRS-v1”) of 188 protein pairs chosen randomly from the space of all ORFeome v1.1 pairs after excluding known interactions (Fig. 2c).PRS and RRS pairs can be used to experimentally calibrate conditions of an assay to achieve an optimal trade-off between the fraction of PRS and RRS pairs reporting positive26. We measured the fraction of hsPRS-v1/hsRRS-v1 pairs scoring positive across a range of experimental and scoring conditions of a stringent version of the Y2H system (Y2H-CCSB)2 and the mammalian protein-protein interaction trap assay (MAPPIT)27 (Supplementary Table 2 online and Fig. 2d,e).The results observed with the hsPRS-v1 and hsRRS-v1 pairs for Y2H-CCSB confirm that the specific experimental conditions used in generating our first human interactome map, CCSB-HI12, reflected good assay design. We also derived suitable experimental conditions for the MAPPIT assay. Under these experimental conditions we estimated the assay sensitivity of Y2H-CCSB and MAPPIT to be 17% and 21% respectively (Fig. 2f and Supplementary Table 3 online). Using a larger, more recently updated set of ~1,500 LC interactions that are supported by multiple publications we estimated an assay sensitivity of 20% for Y2H-CCSB, consistent with our hsPRS-v1-based estimate. Y2H-CCSB and MAPPIT recovered partially overlapping sets of hsPRS-v1 interactions. Of the hsPRS-v1 pairs 29% (27/92) were reported by at least one assay, and of these, 26% (7/27) were detected by both assays (Figs. 2f,g). That 74% (20/27) of positive hits are specific to a single assay reflects the complementarities between the two assays.We estimated the false positive rate (rate of scoring hsRRS-v1 pairs positive) of Y2H-CCSB and MAPPIT to be < 0.5% and 2% respectively (Fig. 2f and Supplementary Table 4 online). The results of testing hsRRS-v1 pairs by Y2H-CCSB do not permit a direct and reasonable estimate of the false discovery rate associated with the CCSB-HI1 dataset. The millions of pairs tested by Y2H-CCSB in the HT screen leading to the generation of CCSB-HI1 consist mostly of non-interacting pairs, so the number of non-interacting pairs tested in the HT-Y2H screen is orders of magnitude higher than the size of hsRRS-v1. Consequently, small changes in the hsRRS-v1-based estimate of the false positive rate of Y2H-CCSB can have a large effect on the resulting estimate of the false discovery rate of CCSB-HI1. Rather than using the Y2H-CCSB experiments on the hsRRS-v1 pairs, we instead used two alternate and independent approaches to estimate the false discovery rate of our Y2H-CCSB assay: (i) retesting Y2H-CCSB interactions in MAPPIT and (ii) modeling repeated screens of Y2H-CCSB (see below).Precision of existing human interactome datasetsWe estimated the precision of the two existing HT-Y2H human interactome datasets, CCSB-HI12 and MDC-HI13, and a low-throughput LC human interactome dataset2 by measuring the extent to which a subset of 188 positive pairs chosen randomly from each dataset (Supplementary Table 1) retested in MAPPIT. To do so, we first benchmarked the performance of a dataset in MAPPIT experiments against (i) the false positive rate of MAPPIT and (ii) the false negative rate of MAPPIT. We estimated these benchmarks by evaluating the fraction of hsRRS-v1 and hsPRS-v1 pairs reporting positive by MAPPIT, respectively. The results with the hsRRS-v1 pairs provided an estimate of MAPPIT’s false positive rate that is sufficiently resolved for estimating false discovery rates of the various interactome datasets, since the size of the hsRRS-v1 is similar to the size of each of the three different interactome datasets tested. Relative to the proportion of hsPRS-v1 and hsRRS-v1 pairs scoring positive (21% and 2%, respectively), the fractions of pairs that scored positive in the three datasets were: LC, 8%; MDC-HI1, 10%; and CCSB-HI1, 27% (Fig. 3a and Supplementary Table 4).This analysis needs to be adjusted for potential dataset biases. First, we minimized the effect due to differences between the sequences of the clones originally used to report the interactions and sequences of the full-length clones used here. We considered only pairs for which the proteins originally used were described as full-length (FL) or, whenever identifiable, pairs for which the isoforms originally used were the same (“Same”) as the ones used here. Second, since the CCSB-HI1 and MDC-HI1 datasets were each described in a single publication, we compared them to the subset of LC interactions also supported by a single publication (“Single”), which represents most currently available literature-curated interaction information25. Including interactions supported by multiple publications in the LC dataset would be circular since our hsPRS-v1 benchmark was derived from LC interactions supported by multiple publications. Lastly, to account for the moderate bias of MAPPIT in detecting Y2H-supported (“Y2H”) interactions, we considered the subset of hsPRS-v1 and LC interaction pairs supported by at least one Y2H experiment in the corresponding curated publications (Supplementary Data 1 and Supplementary Table 5 online). Based on these consolidated datasets, 34% of Y2H-supported hsPRS-v1 pairs (PRS-Y2H) and 2% of hsRRS-v1 pairs scored positive. Relative to this, the fractions of pairs that scored positive in the three subsets of protein pairs were: LC (Single, FL, Y2H), 10%; MDC-HI1 (Same, FL), 31%; and CCSB-HI1 (Same, FL), 27% (Fig. 3b). Thus, the two HT-Y2H datasets performed comparably to the PRS-Y2H pairs in MAPPIT, while the literature-curated interactions supported by a single publication performed poorly. Given the fraction of PRS-Y2H pairs and hsRRS-v1 pairs scoring positive by MAPPIT, the precision of each of the three datasets can be computed as: LC (Single, FL, Y2H), 25%; MDC-HI1 (Same, FL), 83%; and CCSB-HI1 (Same, FL), 79% (Fig. 3c and Supplementary Table 3).Sampling sensitivity and stochastic false discovery rateTo estimate sampling sensitivity and the number of screens required to achieve saturation, we repeated four Y2H-CCSB screens (“repeat screens”) in a defined search space of 1,822 DB-Xs (or “baits”; representing 1,744 unique genes) against 1,796 AD-Ys (or “preys”; representing 1,752 unique genes), representing approximately three million pair-wise combinations (Supplementary Table 6 online). We developed a probabilistic model that considered the search space of three million protein pairs to be a mixture of true biophysical interactions and non-interacting pairs. Using a Bayesian approach, our model estimated (i) the fraction of all identifiable true biophysical interactions found in one, two, or a saturating number of screens and (ii) the fraction of non-interacting pairs erroneously detected in a screen. In short, our approach estimated distributions of values of the above parameters that fit the experimental results observed in the repeat screens.Out of the three million pair-wise combinations tested, the four Y2H-CCSB repeat screens together reported 240 protein-protein interactions (Supplementary Tables 7 and 8 online). Of these interactions 49% appeared in multiple screens. The total number of new interactions identified after successive screens showed an increasing trend, indicating that more interactions would be found with additional screens (Fig. 3d). Based on our model, we estimated that the sampling sensitivity per screen is 45% and that after a saturating number of screens, Y2H-CCSB can identify 71 interactions per million pairs tested (Fig. 3e). Approximately six screens are needed to reach 90% saturation. Importantly, the number of single hits (interactions found in only one out of several screens) decreases while the contribution of multiple hits dominates after multiple screens. Adjusting for these repeat screens being done in only one Y2H configuration (bait-prey vs. prey-bait), we estimated that upon testing both configurations, the sampling sensitivity per screen is 53% and that after a saturating number of screens, Y2H-CCSB can identify 118 interactions per million pairs tested (Supplementary Table 3).Our model estimated that approximately eight out of every million non-interacting pairs tested falsely report positive in Y2H-CCSB. Consequently, our model estimated a stochastic false discovery rate of 12%, meaning that 12% of the interactions reported in a single Y2H-CCSB screen correspond to stochastic false positives. Since the MAPPIT experiments (Fig. 3c) evaluate the union of systematic and stochastic false positives in a given dataset and estimated an overall false discovery rate of 21% for CCSB-HI1, we deduce a systematic false discovery rate of 14% (Supplementary Methods online).The MAPPIT experiments show that existing human HT-Y2H maps have high precision. However, the fraction of CCSB-HI1 and MDC-HI1 interactions common to both maps is small although statistically significant, only 6% and 2%, respectively (P = 10−18, Supplementary Data 2 and Supplementary Fig. 1 online). Our results indicate that low sampling sensitivity and differences in assay sensitivity likely account for the small overlap.Estimation of the size of the human interactomeWe estimated four important parameters associated with the quality of human binary interactome maps (Fig. 1). For the Y2H-CCSB assay evaluated here, we computed the screening completeness of the repeat screens as ~1%; the hsPRS-v1 experiment estimated an assay-sensitivity of ~17% (Fig. 2f); the model of the repeat screens estimated a sampling sensitivity of ~53% (Fig. 3e); and the MAPPIT experiment estimated a precision of ~79% (Fig. 3c). We also estimated the variation of these estimates associated with sampling (Supplementary Table 3). Integrating these parameters, we predict that the entire human interactome, excluding splice variant complexity, contains approximately 74,000–200,000 binary biophysical interactions (Table 1).Interacting protein pairs and shared functional annotationA statistically significant fraction (P < 10−3) but not all of the interacting protein pairs in CCSB-HI1 and MDC-HI1 showed correlation for shared functional annotations compared to random expectation (Fig. 4). Given the high technical quality of these datasets demonstrated here, interacting pairs that do not correlate with known functional annotations could be: (i) promising candidates for biological discovery, particularly true biological interactions that involve proteins currently lacking adequate functional annotations; or (ii) true biophysical interactions that do not occur physiologically. We call this second class “pseudo-interactions” by analogy to pseudo-genes. Pseudo-interactions could correspond to ancient biological interactions that have evolved to lose physiological relevance and provide interesting insights into the evolution of the interactome.

An interaction mapping frameworkTo accurately assess the quality of a given interactome map, we need to consider every possible source of false negatives (true interactions missing) and false positives (spurious pairs reported) associated with the assay used to generate the map. Our framework considers four parameters to estimate quality: “screening completeness”, “assay sensitivity”, “sampling sensitivity” and “precision” (Fig. 1).“Screening completeness” is the fraction of the total possible space of open reading frame (ORF) pairs that is tested to generate a given interactome map (Fig. 1a). Since currently available ORF resources3,24 only allow proteome-wide investigations of one protein isoform per gene, we ignore isoforms encoded by alternatively spliced transcripts here. For example, if we assume that the human genome consists of 22,500 protein-coding genes (N = 22,500 × 22,500/2 protein pairs), then the screening completeness of CCSB-HI12, a proteome-scale HT-Y2H effort that tested n = 7,000 × 7,000/2 human protein pairs, is n/N, or ~10%.“Assay sensitivity” is the fraction of all biophysical interactions that can possibly be identified by an assay performed under a specific set of experimental conditions (Fig. 1b). For example, a given HT-Y2H assay may be unable to detect interactions involving specific types of membrane proteins or requiring post-translational modifications that do not occur in yeast cells.“Sampling sensitivity” is the fraction of all identifiable interactions that are found in a single trial of an assay performed under a specific set of experimental conditions (Fig. 1c). When testing tens if not hundreds of millions of protein pairs in any space of pair-wise combinations, it might be necessary to sample that space multiple times to report all identifiable interactions.Lastly, “precision” is the fraction of observed pairs in an interactome dataset that are true positives (Fig. 1d). False positive pairs reflect technical artifacts that erroneously score positive in a given assay performed under a specific set of experimental conditions. We distinguish between two types of artifactual pairs, “stochastic false positives”, which are observed in only one or a few trials of an assay and “systematic false positives”, which are observed in many or all trials.

An interaction mapping framework

To accurately assess the quality of a given interactome map, we need to consider every possible source of false negatives (true interactions missing) and false positives (spurious pairs reported) associated with the assay used to generate the map. Our framework considers four parameters to estimate quality: “screening completeness”, “assay sensitivity”, “sampling sensitivity” and “precision” (Fig. 1).

“Screening completeness” is the fraction of the total possible space of open reading frame (ORF) pairs that is tested to generate a given interactome map (Fig. 1a). Since currently available ORF resources3,24 only allow proteome-wide investigations of one protein isoform per gene, we ignore isoforms encoded by alternatively spliced transcripts here. For example, if we assume that the human genome consists of 22,500 protein-coding genes (N = 22,500 × 22,500/2 protein pairs), then the screening completeness of CCSB-HI12, a proteome-scale HT-Y2H effort that tested n = 7,000 × 7,000/2 human protein pairs, is n/N, or ~10%.

“Assay sensitivity” is the fraction of all biophysical interactions that can possibly be identified by an assay performed under a specific set of experimental conditions (Fig. 1b). For example, a given HT-Y2H assay may be unable to detect interactions involving specific types of membrane proteins or requiring post-translational modifications that do not occur in yeast cells.

“Sampling sensitivity” is the fraction of all identifiable interactions that are found in a single trial of an assay performed under a specific set of experimental conditions (Fig. 1c). When testing tens if not hundreds of millions of protein pairs in any space of pair-wise combinations, it might be necessary to sample that space multiple times to report all identifiable interactions.

Lastly, “precision” is the fraction of observed pairs in an interactome dataset that are true positives (Fig. 1d). False positive pairs reflect technical artifacts that erroneously score positive in a given assay performed under a specific set of experimental conditions. We distinguish between two types of artifactual pairs, “stochastic false positives”, which are observed in only one or a few trials of an assay and “systematic false positives”, which are observed in many or all trials.

Estimation of assay sensitivityEstimation of the assay parameters described above requires reference sets of positive and negative interacting pairs. To compile a positive reference set of high-confidence human binary protein-protein interactions we started with interactions curated from the literature (“literature curated” or LC interactions) and from these, we chose 188 pairs present in our human ORFeome v1.1 clone collection24 that are supported by the greatest number of publications and curated by the highest number of databases. Systematic recuration of all publications thought to support these 188 protein pairs25 verified 107 direct binary interactions that involve human proteins and that are supported by multiple publications. Ninety-two of these interactions involve full-length proteins and constituted our Homo sapiens Positive Reference Set version 1 or “hsPRS-v1” (Fig. 2a and Supplementary Table 1 online). Proteins involved in the 92 hsPRS-v1 interactions exhibit broad cellular localization (Fig. 2b), suggesting that they are representative of the entire human proteome. It is impossible to generate a set of negative interacting pairs with absolute confidence. So we compiled a surrogate Random Reference Set (“hsRRS-v1”) of 188 protein pairs chosen randomly from the space of all ORFeome v1.1 pairs after excluding known interactions (Fig. 2c).PRS and RRS pairs can be used to experimentally calibrate conditions of an assay to achieve an optimal trade-off between the fraction of PRS and RRS pairs reporting positive26. We measured the fraction of hsPRS-v1/hsRRS-v1 pairs scoring positive across a range of experimental and scoring conditions of a stringent version of the Y2H system (Y2H-CCSB)2 and the mammalian protein-protein interaction trap assay (MAPPIT)27 (Supplementary Table 2 online and Fig. 2d,e).The results observed with the hsPRS-v1 and hsRRS-v1 pairs for Y2H-CCSB confirm that the specific experimental conditions used in generating our first human interactome map, CCSB-HI12, reflected good assay design. We also derived suitable experimental conditions for the MAPPIT assay. Under these experimental conditions we estimated the assay sensitivity of Y2H-CCSB and MAPPIT to be 17% and 21% respectively (Fig. 2f and Supplementary Table 3 online). Using a larger, more recently updated set of ~1,500 LC interactions that are supported by multiple publications we estimated an assay sensitivity of 20% for Y2H-CCSB, consistent with our hsPRS-v1-based estimate. Y2H-CCSB and MAPPIT recovered partially overlapping sets of hsPRS-v1 interactions. Of the hsPRS-v1 pairs 29% (27/92) were reported by at least one assay, and of these, 26% (7/27) were detected by both assays (Figs. 2f,g). That 74% (20/27) of positive hits are specific to a single assay reflects the complementarities between the two assays.We estimated the false positive rate (rate of scoring hsRRS-v1 pairs positive) of Y2H-CCSB and MAPPIT to be < 0.5% and 2% respectively (Fig. 2f and Supplementary Table 4 online). The results of testing hsRRS-v1 pairs by Y2H-CCSB do not permit a direct and reasonable estimate of the false discovery rate associated with the CCSB-HI1 dataset. The millions of pairs tested by Y2H-CCSB in the HT screen leading to the generation of CCSB-HI1 consist mostly of non-interacting pairs, so the number of non-interacting pairs tested in the HT-Y2H screen is orders of magnitude higher than the size of hsRRS-v1. Consequently, small changes in the hsRRS-v1-based estimate of the false positive rate of Y2H-CCSB can have a large effect on the resulting estimate of the false discovery rate of CCSB-HI1. Rather than using the Y2H-CCSB experiments on the hsRRS-v1 pairs, we instead used two alternate and independent approaches to estimate the false discovery rate of our Y2H-CCSB assay: (i) retesting Y2H-CCSB interactions in MAPPIT and (ii) modeling repeated screens of Y2H-CCSB (see below).

Estimation of assay sensitivity

Estimation of the assay parameters described above requires reference sets of positive and negative interacting pairs. To compile a positive reference set of high-confidence human binary protein-protein interactions we started with interactions curated from the literature (“literature curated” or LC interactions) and from these, we chose 188 pairs present in our human ORFeome v1.1 clone collection24 that are supported by the greatest number of publications and curated by the highest number of databases. Systematic recuration of all publications thought to support these 188 protein pairs25 verified 107 direct binary interactions that involve human proteins and that are supported by multiple publications. Ninety-two of these interactions involve full-length proteins and constituted our Homo sapiens Positive Reference Set version 1 or “hsPRS-v1” (Fig. 2a and Supplementary Table 1 online). Proteins involved in the 92 hsPRS-v1 interactions exhibit broad cellular localization (Fig. 2b), suggesting that they are representative of the entire human proteome. It is impossible to generate a set of negative interacting pairs with absolute confidence. So we compiled a surrogate Random Reference Set (“hsRRS-v1”) of 188 protein pairs chosen randomly from the space of all ORFeome v1.1 pairs after excluding known interactions (Fig. 2c).

PRS and RRS pairs can be used to experimentally calibrate conditions of an assay to achieve an optimal trade-off between the fraction of PRS and RRS pairs reporting positive26. We measured the fraction of hsPRS-v1/hsRRS-v1 pairs scoring positive across a range of experimental and scoring conditions of a stringent version of the Y2H system (Y2H-CCSB)2 and the mammalian protein-protein interaction trap assay (MAPPIT)27 (Supplementary Table 2 online and Fig. 2d,e).

The results observed with the hsPRS-v1 and hsRRS-v1 pairs for Y2H-CCSB confirm that the specific experimental conditions used in generating our first human interactome map, CCSB-HI12, reflected good assay design. We also derived suitable experimental conditions for the MAPPIT assay. Under these experimental conditions we estimated the assay sensitivity of Y2H-CCSB and MAPPIT to be 17% and 21% respectively (Fig. 2f and Supplementary Table 3 online). Using a larger, more recently updated set of ~1,500 LC interactions that are supported by multiple publications we estimated an assay sensitivity of 20% for Y2H-CCSB, consistent with our hsPRS-v1-based estimate. Y2H-CCSB and MAPPIT recovered partially overlapping sets of hsPRS-v1 interactions. Of the hsPRS-v1 pairs 29% (27/92) were reported by at least one assay, and of these, 26% (7/27) were detected by both assays (Figs. 2f,g). That 74% (20/27) of positive hits are specific to a single assay reflects the complementarities between the two assays.

We estimated the false positive rate (rate of scoring hsRRS-v1 pairs positive) of Y2H-CCSB and MAPPIT to be < 0.5% and 2% respectively (Fig. 2f and Supplementary Table 4 online). The results of testing hsRRS-v1 pairs by Y2H-CCSB do not permit a direct and reasonable estimate of the false discovery rate associated with the CCSB-HI1 dataset. The millions of pairs tested by Y2H-CCSB in the HT screen leading to the generation of CCSB-HI1 consist mostly of non-interacting pairs, so the number of non-interacting pairs tested in the HT-Y2H screen is orders of magnitude higher than the size of hsRRS-v1. Consequently, small changes in the hsRRS-v1-based estimate of the false positive rate of Y2H-CCSB can have a large effect on the resulting estimate of the false discovery rate of CCSB-HI1. Rather than using the Y2H-CCSB experiments on the hsRRS-v1 pairs, we instead used two alternate and independent approaches to estimate the false discovery rate of our Y2H-CCSB assay: (i) retesting Y2H-CCSB interactions in MAPPIT and (ii) modeling repeated screens of Y2H-CCSB (see below).

Precision of existing human interactome datasetsWe estimated the precision of the two existing HT-Y2H human interactome datasets, CCSB-HI12 and MDC-HI13, and a low-throughput LC human interactome dataset2 by measuring the extent to which a subset of 188 positive pairs chosen randomly from each dataset (Supplementary Table 1) retested in MAPPIT. To do so, we first benchmarked the performance of a dataset in MAPPIT experiments against (i) the false positive rate of MAPPIT and (ii) the false negative rate of MAPPIT. We estimated these benchmarks by evaluating the fraction of hsRRS-v1 and hsPRS-v1 pairs reporting positive by MAPPIT, respectively. The results with the hsRRS-v1 pairs provided an estimate of MAPPIT’s false positive rate that is sufficiently resolved for estimating false discovery rates of the various interactome datasets, since the size of the hsRRS-v1 is similar to the size of each of the three different interactome datasets tested. Relative to the proportion of hsPRS-v1 and hsRRS-v1 pairs scoring positive (21% and 2%, respectively), the fractions of pairs that scored positive in the three datasets were: LC, 8%; MDC-HI1, 10%; and CCSB-HI1, 27% (Fig. 3a and Supplementary Table 4).This analysis needs to be adjusted for potential dataset biases. First, we minimized the effect due to differences between the sequences of the clones originally used to report the interactions and sequences of the full-length clones used here. We considered only pairs for which the proteins originally used were described as full-length (FL) or, whenever identifiable, pairs for which the isoforms originally used were the same (“Same”) as the ones used here. Second, since the CCSB-HI1 and MDC-HI1 datasets were each described in a single publication, we compared them to the subset of LC interactions also supported by a single publication (“Single”), which represents most currently available literature-curated interaction information25. Including interactions supported by multiple publications in the LC dataset would be circular since our hsPRS-v1 benchmark was derived from LC interactions supported by multiple publications. Lastly, to account for the moderate bias of MAPPIT in detecting Y2H-supported (“Y2H”) interactions, we considered the subset of hsPRS-v1 and LC interaction pairs supported by at least one Y2H experiment in the corresponding curated publications (Supplementary Data 1 and Supplementary Table 5 online). Based on these consolidated datasets, 34% of Y2H-supported hsPRS-v1 pairs (PRS-Y2H) and 2% of hsRRS-v1 pairs scored positive. Relative to this, the fractions of pairs that scored positive in the three subsets of protein pairs were: LC (Single, FL, Y2H), 10%; MDC-HI1 (Same, FL), 31%; and CCSB-HI1 (Same, FL), 27% (Fig. 3b). Thus, the two HT-Y2H datasets performed comparably to the PRS-Y2H pairs in MAPPIT, while the literature-curated interactions supported by a single publication performed poorly. Given the fraction of PRS-Y2H pairs and hsRRS-v1 pairs scoring positive by MAPPIT, the precision of each of the three datasets can be computed as: LC (Single, FL, Y2H), 25%; MDC-HI1 (Same, FL), 83%; and CCSB-HI1 (Same, FL), 79% (Fig. 3c and Supplementary Table 3).

Precision of existing human interactome datasets

We estimated the precision of the two existing HT-Y2H human interactome datasets, CCSB-HI12 and MDC-HI13, and a low-throughput LC human interactome dataset2 by measuring the extent to which a subset of 188 positive pairs chosen randomly from each dataset (Supplementary Table 1) retested in MAPPIT. To do so, we first benchmarked the performance of a dataset in MAPPIT experiments against (i) the false positive rate of MAPPIT and (ii) the false negative rate of MAPPIT. We estimated these benchmarks by evaluating the fraction of hsRRS-v1 and hsPRS-v1 pairs reporting positive by MAPPIT, respectively. The results with the hsRRS-v1 pairs provided an estimate of MAPPIT’s false positive rate that is sufficiently resolved for estimating false discovery rates of the various interactome datasets, since the size of the hsRRS-v1 is similar to the size of each of the three different interactome datasets tested. Relative to the proportion of hsPRS-v1 and hsRRS-v1 pairs scoring positive (21% and 2%, respectively), the fractions of pairs that scored positive in the three datasets were: LC, 8%; MDC-HI1, 10%; and CCSB-HI1, 27% (Fig. 3a and Supplementary Table 4).

This analysis needs to be adjusted for potential dataset biases. First, we minimized the effect due to differences between the sequences of the clones originally used to report the interactions and sequences of the full-length clones used here. We considered only pairs for which the proteins originally used were described as full-length (FL) or, whenever identifiable, pairs for which the isoforms originally used were the same (“Same”) as the ones used here. Second, since the CCSB-HI1 and MDC-HI1 datasets were each described in a single publication, we compared them to the subset of LC interactions also supported by a single publication (“Single”), which represents most currently available literature-curated interaction information25. Including interactions supported by multiple publications in the LC dataset would be circular since our hsPRS-v1 benchmark was derived from LC interactions supported by multiple publications. Lastly, to account for the moderate bias of MAPPIT in detecting Y2H-supported (“Y2H”) interactions, we considered the subset of hsPRS-v1 and LC interaction pairs supported by at least one Y2H experiment in the corresponding curated publications (Supplementary Data 1 and Supplementary Table 5 online). Based on these consolidated datasets, 34% of Y2H-supported hsPRS-v1 pairs (PRS-Y2H) and 2% of hsRRS-v1 pairs scored positive. Relative to this, the fractions of pairs that scored positive in the three subsets of protein pairs were: LC (Single, FL, Y2H), 10%; MDC-HI1 (Same, FL), 31%; and CCSB-HI1 (Same, FL), 27% (Fig. 3b). Thus, the two HT-Y2H datasets performed comparably to the PRS-Y2H pairs in MAPPIT, while the literature-curated interactions supported by a single publication performed poorly. Given the fraction of PRS-Y2H pairs and hsRRS-v1 pairs scoring positive by MAPPIT, the precision of each of the three datasets can be computed as: LC (Single, FL, Y2H), 25%; MDC-HI1 (Same, FL), 83%; and CCSB-HI1 (Same, FL), 79% (Fig. 3c and Supplementary Table 3).

Sampling sensitivity and stochastic false discovery rateTo estimate sampling sensitivity and the number of screens required to achieve saturation, we repeated four Y2H-CCSB screens (“repeat screens”) in a defined search space of 1,822 DB-Xs (or “baits”; representing 1,744 unique genes) against 1,796 AD-Ys (or “preys”; representing 1,752 unique genes), representing approximately three million pair-wise combinations (Supplementary Table 6 online). We developed a probabilistic model that considered the search space of three million protein pairs to be a mixture of true biophysical interactions and non-interacting pairs. Using a Bayesian approach, our model estimated (i) the fraction of all identifiable true biophysical interactions found in one, two, or a saturating number of screens and (ii) the fraction of non-interacting pairs erroneously detected in a screen. In short, our approach estimated distributions of values of the above parameters that fit the experimental results observed in the repeat screens.Out of the three million pair-wise combinations tested, the four Y2H-CCSB repeat screens together reported 240 protein-protein interactions (Supplementary Tables 7 and 8 online). Of these interactions 49% appeared in multiple screens. The total number of new interactions identified after successive screens showed an increasing trend, indicating that more interactions would be found with additional screens (Fig. 3d). Based on our model, we estimated that the sampling sensitivity per screen is 45% and that after a saturating number of screens, Y2H-CCSB can identify 71 interactions per million pairs tested (Fig. 3e). Approximately six screens are needed to reach 90% saturation. Importantly, the number of single hits (interactions found in only one out of several screens) decreases while the contribution of multiple hits dominates after multiple screens. Adjusting for these repeat screens being done in only one Y2H configuration (bait-prey vs. prey-bait), we estimated that upon testing both configurations, the sampling sensitivity per screen is 53% and that after a saturating number of screens, Y2H-CCSB can identify 118 interactions per million pairs tested (Supplementary Table 3).Our model estimated that approximately eight out of every million non-interacting pairs tested falsely report positive in Y2H-CCSB. Consequently, our model estimated a stochastic false discovery rate of 12%, meaning that 12% of the interactions reported in a single Y2H-CCSB screen correspond to stochastic false positives. Since the MAPPIT experiments (Fig. 3c) evaluate the union of systematic and stochastic false positives in a given dataset and estimated an overall false discovery rate of 21% for CCSB-HI1, we deduce a systematic false discovery rate of 14% (Supplementary Methods online).The MAPPIT experiments show that existing human HT-Y2H maps have high precision. However, the fraction of CCSB-HI1 and MDC-HI1 interactions common to both maps is small although statistically significant, only 6% and 2%, respectively (P = 10−18, Supplementary Data 2 and Supplementary Fig. 1 online). Our results indicate that low sampling sensitivity and differences in assay sensitivity likely account for the small overlap.

Sampling sensitivity and stochastic false discovery rate

To estimate sampling sensitivity and the number of screens required to achieve saturation, we repeated four Y2H-CCSB screens (“repeat screens”) in a defined search space of 1,822 DB-Xs (or “baits”; representing 1,744 unique genes) against 1,796 AD-Ys (or “preys”; representing 1,752 unique genes), representing approximately three million pair-wise combinations (Supplementary Table 6 online). We developed a probabilistic model that considered the search space of three million protein pairs to be a mixture of true biophysical interactions and non-interacting pairs. Using a Bayesian approach, our model estimated (i) the fraction of all identifiable true biophysical interactions found in one, two, or a saturating number of screens and (ii) the fraction of non-interacting pairs erroneously detected in a screen. In short, our approach estimated distributions of values of the above parameters that fit the experimental results observed in the repeat screens.

Out of the three million pair-wise combinations tested, the four Y2H-CCSB repeat screens together reported 240 protein-protein interactions (Supplementary Tables 7 and 8 online). Of these interactions 49% appeared in multiple screens. The total number of new interactions identified after successive screens showed an increasing trend, indicating that more interactions would be found with additional screens (Fig. 3d). Based on our model, we estimated that the sampling sensitivity per screen is 45% and that after a saturating number of screens, Y2H-CCSB can identify 71 interactions per million pairs tested (Fig. 3e). Approximately six screens are needed to reach 90% saturation. Importantly, the number of single hits (interactions found in only one out of several screens) decreases while the contribution of multiple hits dominates after multiple screens. Adjusting for these repeat screens being done in only one Y2H configuration (bait-prey vs. prey-bait), we estimated that upon testing both configurations, the sampling sensitivity per screen is 53% and that after a saturating number of screens, Y2H-CCSB can identify 118 interactions per million pairs tested (Supplementary Table 3).

Our model estimated that approximately eight out of every million non-interacting pairs tested falsely report positive in Y2H-CCSB. Consequently, our model estimated a stochastic false discovery rate of 12%, meaning that 12% of the interactions reported in a single Y2H-CCSB screen correspond to stochastic false positives. Since the MAPPIT experiments (Fig. 3c) evaluate the union of systematic and stochastic false positives in a given dataset and estimated an overall false discovery rate of 21% for CCSB-HI1, we deduce a systematic false discovery rate of 14% (Supplementary Methods online).

The MAPPIT experiments show that existing human HT-Y2H maps have high precision. However, the fraction of CCSB-HI1 and MDC-HI1 interactions common to both maps is small although statistically significant, only 6% and 2%, respectively (P = 10−18, Supplementary Data 2 and Supplementary Fig. 1 online). Our results indicate that low sampling sensitivity and differences in assay sensitivity likely account for the small overlap.

Estimation of the size of the human interactomeWe estimated four important parameters associated with the quality of human binary interactome maps (Fig. 1). For the Y2H-CCSB assay evaluated here, we computed the screening completeness of the repeat screens as ~1%; the hsPRS-v1 experiment estimated an assay-sensitivity of ~17% (Fig. 2f); the model of the repeat screens estimated a sampling sensitivity of ~53% (Fig. 3e); and the MAPPIT experiment estimated a precision of ~79% (Fig. 3c). We also estimated the variation of these estimates associated with sampling (Supplementary Table 3). Integrating these parameters, we predict that the entire human interactome, excluding splice variant complexity, contains approximately 74,000–200,000 binary biophysical interactions (Table 1).

Estimation of the size of the human interactome

We estimated four important parameters associated with the quality of human binary interactome maps (Fig. 1). For the Y2H-CCSB assay evaluated here, we computed the screening completeness of the repeat screens as ~1%; the hsPRS-v1 experiment estimated an assay-sensitivity of ~17% (Fig. 2f); the model of the repeat screens estimated a sampling sensitivity of ~53% (Fig. 3e); and the MAPPIT experiment estimated a precision of ~79% (Fig. 3c). We also estimated the variation of these estimates associated with sampling (Supplementary Table 3). Integrating these parameters, we predict that the entire human interactome, excluding splice variant complexity, contains approximately 74,000–200,000 binary biophysical interactions (Table 1).

Interacting protein pairs and shared functional annotationA statistically significant fraction (P < 10−3) but not all of the interacting protein pairs in CCSB-HI1 and MDC-HI1 showed correlation for shared functional annotations compared to random expectation (Fig. 4). Given the high technical quality of these datasets demonstrated here, interacting pairs that do not correlate with known functional annotations could be: (i) promising candidates for biological discovery, particularly true biological interactions that involve proteins currently lacking adequate functional annotations; or (ii) true biophysical interactions that do not occur physiologically. We call this second class “pseudo-interactions” by analogy to pseudo-genes. Pseudo-interactions could correspond to ancient biological interactions that have evolved to lose physiological relevance and provide interesting insights into the evolution of the interactome.

Interacting protein pairs and shared functional annotation

A statistically significant fraction (P < 10−3) but not all of the interacting protein pairs in CCSB-HI1 and MDC-HI1 showed correlation for shared functional annotations compared to random expectation (Fig. 4). Given the high technical quality of these datasets demonstrated here, interacting pairs that do not correlate with known functional annotations could be: (i) promising candidates for biological discovery, particularly true biological interactions that involve proteins currently lacking adequate functional annotations; or (ii) true biophysical interactions that do not occur physiologically. We call this second class “pseudo-interactions” by analogy to pseudo-genes. Pseudo-interactions could correspond to ancient biological interactions that have evolved to lose physiological relevance and provide interesting insights into the evolution of the interactome.

DISCUSSIONSeveral previous studies have estimated the precision of existing maps or the size of interactomes13–15,18,20–23,28. Our empirical framework addresses limitations of these studies (detailed discussion in Supplementary Data 3 online). Methods that rely on correlation with other biological attributes to estimate precision of interactome maps (i) often use as benchmark, LC interaction datasets, which are sociologically biased, (ii) assume that knowledge of biological attributes such as Gene Ontology functional annotation is complete and unbiased, and (iii) are inherently constrained by pre-existing paradigms regarding the expectation for interacting protein pairs to share biological attributes. Approaches based on analyzing the extent of overlap between interactome maps13,20,21 suffer specific limitations in their implementations such as comparing maps that were not derived using the same assay, or using LC datasets as a reference set, which may not be appropriate given a potentially higher false positive rate than previously anticipated (Fig. 3c)25. Earlier studies also failed to consider one or more of the parameters that influence interactome map quality, i.e., completeness, systematic false discovery rate, stochastic false discovery rate, assay sensitivity and sampling sensitivity, which could in turn significantly affect estimates of interactome size. All these limitations together may have led to overestimated false discovery rates for HT-Y2H human interactome maps.Our framework overcomes these limitations by (i) considering every possible source of false negatives and false positives, (ii) using a high-quality reference set requiring interactions to be supported by multiple publications and to pass additional recuration, (iii) assessing false discovery rates directly using information from independent protein-protein interaction assays and (iv) comparing overlaps between four homogeneously-derived repeat screens to assess the sampling sensitivity and stochastic false discovery rate of Y2H-CCSB. Close attention to these parameters will be vital to design the strategy, e.g., number of screens and types of assays to use, for future interactome mapping projects.The hsPRS-v1 and hsRRS-v1 provide hundreds of experimentally testable clone pairs of positive and random reference sets for binary protein-protein interactions. Previous assays typically relied on testing one or a few positive control pairs and a few or no random control protein pairs. Though our reference sets are a first version and will be improved, they mark a substantial effort towards the standardized calibration of binary interaction mapping assays, an objective that has not been previously achieved systematically.Although LC datasets are commonly perceived to be of better quality than datasets generated with HT technologies15–17, the results of our MAPPIT experiments indicate that stringent implementations of HT-Y2H assays produce interaction datasets with technical quality at least as good if not superior to low-throughput LC interactions (Fig. 3c). These results substantiate previous computational analyses of human29 and yeast30 interactome maps. Large-scale curation of the primary literature is challenging and may have higher error rates than previously anticipated25. HT interactome mapping strategies have several advantages over low-throughput strategies: (i) since defined search spaces are used, information about positives (pairs observed to interact) as well as negatives (pairs not observed to interact) is available; (ii) experiments are standardized therefore well-controlled, comparable and scalable; (iii) cost-efficient strategies can be developed; and (iv) HT strategies are less sociologically biased than low-throughput experiments.Implementation of our framework can be improved in various ways. The statistical power of the analyses can be increased by testing more PRS interactions, by repeatedly screening larger search spaces, or by using additional independent assays for measuring precision. Our current implementation does not consider multiple splice isoforms per gene, so we most likely underestimated the interactome size. Additional modifications to the framework will be required to thoroughly analyze non-binary co-complex membership maps such as those generated by HT co-affinity purification followed by mass spectrometry4. While more refined estimates can be made with future enhancements, the central concepts and overall approach are in place for design and comprehensive evaluation of any interactome mapping assay. Recently, our group has developed an interaction toolkit consisting of four independent assays to evaluate the quality of any protein interaction dataset26. Ongoing technological advancements related to assay automation and cost reduction will enable testing of expanded versions of the PRS and thousands (rather than hundreds) of Y2H, LC and other interactions using these assays.Similar to estimates of the number of protein-encoding genes in the human genome, ~14,000–300,000 in the early 1990s31, empirical sizing of the interactome is critical to establish the complexity of the network and to estimate how far we are from a complete human interactome map. Assuming one splice isoform per gene, we predict that the size of the human interactome is ~130,000 interactions. This confirms two previous estimates of human interactome size, which ranged from 150,000–370,000 interactions2,13. Out of ~23,000 currently reported human interactions (a combination of ~17,000 LC interactions and ~6,000 HT-Y2H interactions), our measurements indicate that ~10,000 (~42%) are true binary physical interactions (Supplementary Data 4 online). Thus, the fraction of interactions identified so far represents ~8% of the full interactome.With 22,500 protein-coding genes, nearly 250 million protein pairs need to be tested individually, clearly requiring unbiased, systematic and cost-effective HT approaches. Interactome mapping is gradual: six screens are necessary to reach 90% saturation with Y2H-CCSB. No single assay offers complete assay sensitivity. The fraction of protein-protein interactions detectable by the specific version of HT-Y2H used here (“Y2H-CCSB”) is ~17%. Combining different versions of the Y2H system and using increased expression of both hybrid proteins can increase this proportion to ~40% (data not shown and Braun et al.26). Still, comprehensive mapping of the interactome will require the development of additional HT versions of MAPPIT and other assays26.The potential impact on biology of a complete and reliable biophysical protein interaction map cannot be overestimated. Our results offer a quantitative roadmap in this direction, uncovering both the magnitude of the task ahead as well as the potential roadblocks.

Several previous studies have estimated the precision of existing maps or the size of interactomes13–15,18,20–23,28. Our empirical framework addresses limitations of these studies (detailed discussion in Supplementary Data 3 online). Methods that rely on correlation with other biological attributes to estimate precision of interactome maps (i) often use as benchmark, LC interaction datasets, which are sociologically biased, (ii) assume that knowledge of biological attributes such as Gene Ontology functional annotation is complete and unbiased, and (iii) are inherently constrained by pre-existing paradigms regarding the expectation for interacting protein pairs to share biological attributes. Approaches based on analyzing the extent of overlap between interactome maps13,20,21 suffer specific limitations in their implementations such as comparing maps that were not derived using the same assay, or using LC datasets as a reference set, which may not be appropriate given a potentially higher false positive rate than previously anticipated (Fig. 3c)25. Earlier studies also failed to consider one or more of the parameters that influence interactome map quality, i.e., completeness, systematic false discovery rate, stochastic false discovery rate, assay sensitivity and sampling sensitivity, which could in turn significantly affect estimates of interactome size. All these limitations together may have led to overestimated false discovery rates for HT-Y2H human interactome maps.

Our framework overcomes these limitations by (i) considering every possible source of false negatives and false positives, (ii) using a high-quality reference set requiring interactions to be supported by multiple publications and to pass additional recuration, (iii) assessing false discovery rates directly using information from independent protein-protein interaction assays and (iv) comparing overlaps between four homogeneously-derived repeat screens to assess the sampling sensitivity and stochastic false discovery rate of Y2H-CCSB. Close attention to these parameters will be vital to design the strategy, e.g., number of screens and types of assays to use, for future interactome mapping projects.

The hsPRS-v1 and hsRRS-v1 provide hundreds of experimentally testable clone pairs of positive and random reference sets for binary protein-protein interactions. Previous assays typically relied on testing one or a few positive control pairs and a few or no random control protein pairs. Though our reference sets are a first version and will be improved, they mark a substantial effort towards the standardized calibration of binary interaction mapping assays, an objective that has not been previously achieved systematically.

Although LC datasets are commonly perceived to be of better quality than datasets generated with HT technologies15–17, the results of our MAPPIT experiments indicate that stringent implementations of HT-Y2H assays produce interaction datasets with technical quality at least as good if not superior to low-throughput LC interactions (Fig. 3c). These results substantiate previous computational analyses of human29 and yeast30 interactome maps. Large-scale curation of the primary literature is challenging and may have higher error rates than previously anticipated25. HT interactome mapping strategies have several advantages over low-throughput strategies: (i) since defined search spaces are used, information about positives (pairs observed to interact) as well as negatives (pairs not observed to interact) is available; (ii) experiments are standardized therefore well-controlled, comparable and scalable; (iii) cost-efficient strategies can be developed; and (iv) HT strategies are less sociologically biased than low-throughput experiments.

Implementation of our framework can be improved in various ways. The statistical power of the analyses can be increased by testing more PRS interactions, by repeatedly screening larger search spaces, or by using additional independent assays for measuring precision. Our current implementation does not consider multiple splice isoforms per gene, so we most likely underestimated the interactome size. Additional modifications to the framework will be required to thoroughly analyze non-binary co-complex membership maps such as those generated by HT co-affinity purification followed by mass spectrometry4. While more refined estimates can be made with future enhancements, the central concepts and overall approach are in place for design and comprehensive evaluation of any interactome mapping assay. Recently, our group has developed an interaction toolkit consisting of four independent assays to evaluate the quality of any protein interaction dataset26. Ongoing technological advancements related to assay automation and cost reduction will enable testing of expanded versions of the PRS and thousands (rather than hundreds) of Y2H, LC and other interactions using these assays.

Similar to estimates of the number of protein-encoding genes in the human genome, ~14,000–300,000 in the early 1990s31, empirical sizing of the interactome is critical to establish the complexity of the network and to estimate how far we are from a complete human interactome map. Assuming one splice isoform per gene, we predict that the size of the human interactome is ~130,000 interactions. This confirms two previous estimates of human interactome size, which ranged from 150,000–370,000 interactions2,13. Out of ~23,000 currently reported human interactions (a combination of ~17,000 LC interactions and ~6,000 HT-Y2H interactions), our measurements indicate that ~10,000 (~42%) are true binary physical interactions (Supplementary Data 4 online). Thus, the fraction of interactions identified so far represents ~8% of the full interactome.

With 22,500 protein-coding genes, nearly 250 million protein pairs need to be tested individually, clearly requiring unbiased, systematic and cost-effective HT approaches. Interactome mapping is gradual: six screens are necessary to reach 90% saturation with Y2H-CCSB. No single assay offers complete assay sensitivity. The fraction of protein-protein interactions detectable by the specific version of HT-Y2H used here (“Y2H-CCSB”) is ~17%. Combining different versions of the Y2H system and using increased expression of both hybrid proteins can increase this proportion to ~40% (data not shown and Braun et al.26). Still, comprehensive mapping of the interactome will require the development of additional HT versions of MAPPIT and other assays26.

The potential impact on biology of a complete and reliable biophysical protein interaction map cannot be overestimated. Our results offer a quantitative roadmap in this direction, uncovering both the magnitude of the task ahead as well as the potential roadblocks.

METHODSThe Y2H-CCSB experiments were performed as described2 with minor changes. MAPPIT experiments were performed essentially as described32. Mathematical modeling of the repeat screens was performed using a Bayesian approach. All parameters observed from either the experimental data or from the mixture model were used as inputs into a Monte Carlo simulation to calculate the corresponding magnitudes of corresponding numbers reported in the text. Detailed descriptions of all datasets and methods can be found in Supplementary Methods online.

The Y2H-CCSB experiments were performed as described2 with minor changes. MAPPIT experiments were performed essentially as described32. Mathematical modeling of the repeat screens was performed using a Bayesian approach. All parameters observed from either the experimental data or from the mixture model were used as inputs into a Monte Carlo simulation to calculate the corresponding magnitudes of corresponding numbers reported in the text. Detailed descriptions of all datasets and methods can be found in Supplementary Methods online.

Supplementary Material1Supplementary Figure 1. Overlap and screening completeness of existing human binary interactome mapsSupplementary Table 2. Comparison of the features of the two different technologies, Y2H-CCSB and MAPPITSupplementary Table 3. Estimate of various parameters using Monte Carlo simulations based on experimental data and the mixture model of repeat screensSupplementary Table 5. Calculation of conditional dependence between Y2H-CCSB and MAPPITSupplementary Data 1. Consideration of the correlation between pairs scoring positive in Y2H and in MAPPIT assays in the analysis of MAPPIT experimentsSupplementary Data 2. Examination of screening completeness of search spaces and overlap between CCSB-HI1 and MDC-HI1 datasetsSupplementary Data 3. Limitations of previous approaches for estimating data quality and interactome sizeSupplementary Data 4. Current status of available human interactome mapsSupplementary Table 1. List of interactions in various datasets used in pair wise test experiments using MAPPIT and Y2H-CCSB assaysSupplementary Table 4. Scores for the Y2H-CCSB and MAPPIT experiments on the hsPRS-v1 and hsRRS-v1 to compute assay sensitivity and background positive rate and scores on subsets of the LC, MDC-HI1 and CCSB-HI1 interaction datasetsSupplementary Table 6. Identity of the ORFs making up the Y2H-CCSB repeat screensSupplementary Table 7. Interactions found in the Y2H-CCSB repeat screensSupplementary Table 8. Interactions found in the Y2H-CCSB repeat screens reported according to MIMIX specificationsSupplementary Table 9. Identity of the ORFs making up the MDC-HI1 search space (Space II)

Supplementary Material

Supplementary Figure 1. Overlap and screening completeness of existing human binary interactome maps

Supplementary Table 2. Comparison of the features of the two different technologies, Y2H-CCSB and MAPPIT

Supplementary Table 3. Estimate of various parameters using Monte Carlo simulations based on experimental data and the mixture model of repeat screens

Supplementary Table 5. Calculation of conditional dependence between Y2H-CCSB and MAPPIT

Supplementary Data 1. Consideration of the correlation between pairs scoring positive in Y2H and in MAPPIT assays in the analysis of MAPPIT experiments

Supplementary Data 2. Examination of screening completeness of search spaces and overlap between CCSB-HI1 and MDC-HI1 datasets

Supplementary Data 3. Limitations of previous approaches for estimating data quality and interactome size

Supplementary Data 4. Current status of available human interactome maps

Supplementary Table 1. List of interactions in various datasets used in pair wise test experiments using MAPPIT and Y2H-CCSB assays

Supplementary Table 4. Scores for the Y2H-CCSB and MAPPIT experiments on the hsPRS-v1 and hsRRS-v1 to compute assay sensitivity and background positive rate and scores on subsets of the LC, MDC-HI1 and CCSB-HI1 interaction datasets

Supplementary Table 6. Identity of the ORFs making up the Y2H-CCSB repeat screens

Supplementary Table 7. Interactions found in the Y2H-CCSB repeat screens

Supplementary Table 8. Interactions found in the Y2H-CCSB repeat screens reported according to MIMIX specifications

Supplementary Table 9. Identity of the ORFs making up the MDC-HI1 search space (Space II)
