---
pmid: '21566658'
title: A novel RING finger protein, Znf179, modulates cell cycle exit and neuronal
  differentiation of P19 embryonal carcinoma cells.
authors:
- Pao PC
- Huang NK
- Liu YW
- Yeh SH
- Lin ST
- Hsieh CP
- Huang AM
- Huang HS
- Tseng JT
- Chang WC
- Lee YC
journal: Cell Death Differ
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3190115
doi: 10.1038/cdd.2011.52
---

# A novel RING finger protein, Znf179, modulates cell cycle exit and neuronal differentiation of P19 embryonal carcinoma cells.
**Authors:** Pao PC, Huang NK, Liu YW, Yeh SH, Lin ST, Hsieh CP, Huang AM, Huang HS, Tseng JT, Chang WC, Lee YC
**Journal:** Cell Death Differ (2011)
**DOI:** [10.1038/cdd.2011.52](https://doi.org/10.1038/cdd.2011.52)
**PMC:** [PMC3190115](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3190115/)

## Abstract

1. Cell Death Differ. 2011 Nov;18(11):1791-804. doi: 10.1038/cdd.2011.52. Epub
2011  May 13.

A novel RING finger protein, Znf179, modulates cell cycle exit and neuronal 
differentiation of P19 embryonal carcinoma cells.

Pao PC(1), Huang NK, Liu YW, Yeh SH, Lin ST, Hsieh CP, Huang AM, Huang HS, Tseng 
JT, Chang WC, Lee YC.

Author information:
(1)Institute of Bioinformatics and Biosignal Transduction, College of Bioscience 
and Biotechnology, National Cheng Kung University, Tainan, Taiwan.

Znf179 is a member of the RING finger protein family. During embryogenesis, 
Znf179 is expressed in a restricted manner in the brain, suggesting a potential 
role in nervous system development. In this report, we show that the expression 
of Znf179 is upregulated during P19 cell neuronal differentiation. Inhibition of 
Znf179 expression by RNA interference significantly attenuated neuronal 
differentiation of P19 cells and a primary culture of cerebellar granule cells. 
Using a microarray approach and subsequent functional annotation analysis, we 
identified differentially expressed genes in Znf179-knockdown cells and found 
that several genes are involved in development, cellular growth, and cell cycle 
control. Flow cytometric analyses revealed that the population of G0/G1 cells 
decreased in Znf179-knockdown cells. In agreement with the flow cytometric data, 
the number of BrdU-incorporated cells significantly increased in 
Znf179-knockdown cells. Moreover, in Znf179-knockdown cells, p35, a 
neuronal-specific Cdk5 activator that is known to activate Cdk5 and may affect 
the cell cycle, and p27, a cell cycle inhibitor, also decreased. Collectively, 
these results show that induction of the Znf179 gene may be associated with p35 
expression and p27 protein accumulation, which lead to cell cycle arrest in the 
G0/G1 phase, and is critical for neuronal differentiation of P19 cells.

DOI: 10.1038/cdd.2011.52
PMCID: PMC3190115
PMID: 21566658 [Indexed for MEDLINE]

## Full Text

Neurogenesis is a tightly regulated process that is coordinated with cell cycle regulation and cell fate determination.1, 2 To initiate terminal differentiation, neural progenitors must exit the cell cycle, suggesting the existence of crosstalk between the signal pathways that control these two mechanisms. In eukaryotes, cell cycle progression is positively regulated by complexes of cyclin-dependent kinases (CDKs) and associated cyclins and is negatively regulated by cyclin-dependent kinase inhibitors (CKIs). Mammalian CKIs are divided into two families: the INK4 family, including p15, p16, p18, and p19; and the Cip/Kip family, including p21, p27, and p57.3 Among CKIs, p27 is of particular importance for neurogenesis. For instance, p27 is expressed in the developing brain and was implicated in promoting cell cycle arrest of neural progenitors.4, 5 In addition to animal experiments, the role of p27 in neuronal differentiation was also observed in several cell models such as P19 embryonal carcinoma cells.6 p27 binds to both cyclin D-Cdk4,6 and cyclin E-Cdk2 complexes to inhibit kinase activity, which leads to the arrest of the cell cycle in the G1 phase.7, 8 Recent studies revealed that p27 is involved in cell cycle withdrawal and also neuronal migration.9 In one study, Cdk5 and its activator, p35, were found to phosphorylate and stabilize p27. Unlike other CDKs that have central roles in cell cycle progression, Cdk5 is activated in post-mitotic neurons by non-cyclin proteins such as p35 and is known to be important in nervous system development.10

Znf179, a gene with unknown function, belongs to the RING finger protein family, which is characterized by a zinc-binding domain that serves as a protein-binding interface. RING finger proteins are implicated in the function of diverse cellular processes such as gene transcription, cell growth, cellular differentiation, and apoptosis, as well as in abnormal cellular behaviors such as oncogenesis.11 Recent studies indicated that many RING finger proteins act as E3 ubiquitin ligases and are associated with a ubiquitin proteasome pathway.12 RING finger proteins such as Mdm2, Mib, and G2E3 perform essential roles in embryonic and postnatal development.13, 14, 15 The expression of the Znf179 gene is restricted to the brain and is regulated during brain development.16 In the previous studies, Inoue et al.17 also reported that Znf179 is expressed in the retinoic acid (RA)-induced neuronal differentiated P19 cells. These findings suggest that Znf179 may have a role in brain development and neuronal differentiation. However, there is no further study that has confirmed this hypothesis. Moreover, microarray analyses revealed that Znf179 is significantly downregulated in neurodegenerative diseases such as Huntington's disease (HD) and amyotrophic lateral sclerosis, implying that Znf179 might be associated with neurodegenerative diseases.18, 19

In this study, we observed dramatic increases in Znf179 messenger (m)RNA and protein during P19 cell neuronal differentiation. Suppressing Znf179 induction impaired P19 cell neuronal differentiation. We further explored the underlying mechanism and found that Znf179 knockdown resulted in decreased expressions of p35 and p27 and interfered with cell cycle withdrawal and neuronal differentiation in P19 cells. To the best of our knowledge, this is the first study to provide evidence that Znf179 functions in neuronal differentiation.

ResultsZnf179 protein expression is restricted to the brain and particularly to neuronsAlthough the Znf179 gene was cloned >10 years ago, its function has not yet been elucidated.17 In this study, we generated polyclonal antibodies against Znf179 and used those antibodies to examine Znf179 protein expressions in various tissues. Antibody validation was performed, and the result is shown in Supplementary Figure S1. Among the examined tissues, a high level of Znf179 protein was observed in the brain (Figure 1a). We further compared expressions of the Znf179 protein in different brain regions and found that Znf179 was equally expressed in all selected regions, including the cerebral cortex, striatum, hippocampus, and cerebellum (Figure 1b), suggesting that Znf179 may have a general role in the brain. Using in situ hybridization, we also detected transcripts of Znf179 in the developing brain. As shown in Figure 2, Znf179 mRNA was detected in many brain areas, including cerebral cortex, hippocampus, and cerebellum. These data are consistent with a previous study16 and with the result of Allen Brain Atlas gene expression database (Image Series: 393241). Moreover, we found that the expression of Znf179 increased gradually during cerebellum development, suggesting a potential role of Znf179 in nervous system development. To further elucidate the cellular identity of Znf179-expressing cells, primary cultures of mouse cerebellar cells were immunostained with Znf179 and microtubule-associated protein 2 (MAP2) (a neuronal marker) or glial fibrillary acidic protein (GFAP) (a glial marker). As shown in Figure 1c, Znf179 was highly expressed in MAP2-positive cells and slightly expressed in GFAP-positive cells. Our result is consistent with the previous study by Orimo et al.16, which showed that Znf179 was detected in both neuron and glial cells.Znf179 expression is upregulated during P19 cell neuronal differentiationTo study the function of the Znf179 gene, P19 embryonal carcinoma cells, a pluripotent cell line that can be induced by RA to differentiate into neurons, were used as the model.20 Using MAP2 immunofluorescent staining, we examined the neuronal differentiation of P19 cells. As shown in Figure 3a, monolayer-cultured P19 cells were undifferentiated and did not express the neuronal marker, MAP2. In contrast, neuronally differentiated P19 cells showed strong positive staining with the anti-MAP2 antibody (Figure 3b). During RA-induced P19 cell neuronal differentiation, the quantitative RT-PCR analysis showed that Znf179 messenger (m)RNA was dramatically increased after 4 days of aggregation in the presence of RA (Figure 3c). After plating, the expression of Znf179 further increased and was maintained at a high level. We also found that P19 cell aggregation in the absence of RA did not induce the expression of Znf179, indicating that RA was required for Znf179 induction during P19 cell neuronal differentiation. We detected the protein expression of Znf179 and found that the pattern of Znf179 protein expression was similar to that of Znf179 mRNA expression (Figure 3d). In the western blot analysis, the neuronal markers, growth-associated protein 43 (GAP-43) and βIII-tubulin, were also detected and highly expressed in differentiated P19 cells, indicating that the cells had differentiated into neurons. Consistent with the findings reported by Inoue,17 our results revealed that Znf179 is elevated during the differentiation of P19 cells into neurons and provided the precise time course of Znf179 gene expression. The results show that Znf179 may be required for neuronal differentiation.Inhibition of Znf179 elevation by RNA interference leads to attenuated neuronal differentiation of P19 and cerebellar granule cellsGiven that Znf179 expression was dramatically induced during RA-induced P19 cell neuronal differentiation, we speculated that Znf179 may have a role in neuronal differentiation. In order to elucidate this possibility, we used lentiviral small hairpin (sh)RNA to knockdown Znf179 and examined the effect of Znf179 knockdown on P19 cell neuronal differentiation. Infection efficiency was analyzed by flow cytometry. More than 90% of the cells were infected (Supplementary Figure S2). As shown in Figure 4a, two Znf179 shRNA viruses (sh_2 and sh_3), which target different regions of the Znf179 transcript, significantly reduced expression of the Znf179 protein after 4 days of aggregation in the presence of RA. In contrast, the negative control, the lentiviral shRNA that targets the luciferase gene, had no effect on Znf179 expression. After plating, we examined the neuronal differentiation of P19 cells by immunofluorescent staining of the neuronal marker, MAP2. As shown in Figure 4b, high percentage of MAP2-positive cells were shown in control (non-infected and control virus-infected) cultures, whereas the number of MAP2-positive cells was significantly reduced in Znf179 shRNA virus-infected cells. Quantification of results showed that both the total number and the percentage of MAP2-positive cells were markedly reduced (Figures 4c and d). MAP2 protein expression was also detected by a western blot analysis, and similar results were observed (Figure 4e). In addition to MAP2-positive cells, the total cell number was also reduced in Znf179 shRNA virus-infected cells (Figure 4c). An MTT assay was performed to evaluate the number of attached cells after 4 h and 6 days of plating, which may reflect the proliferation or survival rate. As shown in Supplementary Figure S3, in control (non-infected and control virus-infected) cultures, the number of attached cells did not change between 4 h and 6 days of plating, indicating that the attached cells did not undergo proliferation. This is because proliferative, undifferentiated cells died in the serum-free media after plating. In contrast, after 6 days of plating, the amount of cells significantly decreased compared with that with 4 h of plating in Znf179 shRNA virus-infected cells. These results revealed that inhibition of Znf179 elevation could attenuate RA-induced P19 cell neuronal differentiation, which may have resulted from poor survival of cells during neuronal differentiation. To define whether early steps of the differentiation process were equally affected in culture, we detected the early neuronal marker, βIII-tubulin. As shown in Figure 4a, two Znf179 shRNA viruses (sh_2 and sh_3) reduced the expression of βIII-tubulin after 4 days of aggregation in the presence of RA, indicating that the cell fate was affected in the early stages of differentiation. We further detected the expression of neurogenin2 and neuroD1. Our results showed that the protein and mRNA expressions of neurogenin2 and neuroD1 did not change by knockdown of Znf179 (data not shown), indicating that the functional mechanisms of Znf179 on neuronal differentiation are different from neurogenein2 or neuroD1, or Znf179 functions downstream to neurogenein2 or neuroD1.In addition to P19 cells, we also examined the effect of Znf179 knockdown on primary cultured cerebellar granule cells. In a similar manner to P19 cells, knockdown of Znf179 expression led to attenuated cerebellar granule neuronal differentiation (Figures 5a and b). Immunofluorescent staining of the neuronal marker, MAP2, revealed that both the total number and the percentage of MAP2-positive cells were reduced in Znf179 shRNA virus-infected cerebellar granule cells (Figures 5c and d). As shown in Figure 5c, the total cell number also decreased in Znf179 shRNA virus-infected cells, indicating that cell viability was affected during the process of neuronal differentiation. To examine whether Znf179 knockdown also had an effect on glial differentiation, the glial marker, GFAP, was used for immunofluorescent staining. As shown in Figure 6, knockdown of Znf179 also led to attenuation of glial differentiation in primary cerebellar cultures. These results showed that Znf179 has a functional role in both neuronal and glial differentiation. The effect was seen in the cell line model, and also in the primary culture system.Identification and annotation analysis of differentially expressed genes with Znf179 knockdownTo dissect the neuronal differentiation mechanisms of Znf179, we used a microarray to compare the global gene expression profiles between Znf179-knockdown and control cells, both of which were aggregated for 4 days in the presence of RA. Microarray results showed that compared with control virus-infected cells, 52 genes were found to be upregulated and 110 genes were found to be downregulated by ≥1.5-fold in both Znf179 shRNA virus (sh_2 and sh_3)-infected cells (Supplementary Table S2). Two upregulated and four downregulated genes were randomly chosen, and their expressions were measured by a quantitative RT-PCR analysis. The results are consistent with the microarray data (Table 1). We used the 162 (52 upregulated and 110 downregulated) genes to perform a functional annotation analysis using the ingenuity pathway analysis (IPA). As shown in Figure 7a, the five highest-ranking processes in the category of ‘physiological system development and functions' were associated with development, including organ development, embryonic development, tissue development, and nervous system development. This result further supports the role of Znf179 in neuronal development. In addition, cell growth, proliferation, and the cell cycle were identified to be the most significant molecular and cellular functions (Figure 7b). It is known that withdrawal from the cell cycle is a prerequisite for terminal differentiation. Thus, we were interested in determining whether Znf179 affects cell cycle progression, subsequently resulting in neuronal differentiation.Effect of Znf179 knockdown on the cell cycle profile and cell cycle regulatory proteinsTo examine the role of Znf179 in the cell cycle during neuronal differentiation, P19 cells were infected with negative control and Znf179 shRNA viruses. After 4 days of aggregation in the presence of RA, flow cytometry was performed to investigate whether Znf179 knockdown had affected cell cycle withdrawal in P19 cells. As expected, G0/G1-phase cells in RA-treated P19 cells had increased compared with non-treated cells (Figures 8a and b), in agreement with RA-induced cell cycle arrest and neuronal differentiation in P19 cells. However, Znf179 knockdown was able to abolish the increase in G0/G1-phase cells caused by RA treatment, indicating a failure to establish G0/G1 arrest. BrdU incorporation and phospho-histone H3 expression were used to assess cell proliferation. The results were similar to those in flow cytometric experiments. As shown in Figures 8c and d and 9c, both the percentage of BrdU-positive cells and the expression of phospho-histone H3 increased in Znf179-knockdown cells. These results show that Znf179 has a role in inhibiting cell cycle progression, particularly in blocking cells at the G0/G1-phase, and promotes neuronal differentiation. Among the differentially expressed genes with Znf179 knockdown, cyclin D3 (Ccdn3) and p35 (Cdk5r1) are closely linked to cell cycle regulation. Ccdn3 is known to activate Cdk4 and Cdk6, and thereby stimulate G1 progression.21 On the other hand, p35 is known to activate Cdk5 to phosphorylate and stabilize p27, thereby promoting cell cycle arrest.9 Our microarray results showed that Ccdn3 was upregulated and p35 was downregulated in Znf179-knockdown cells (Supplementary Table 2). This result is consistent with the flow cytometric data showing that Znf179 knockdown reduced the G0/G1 fraction (Figures 8a and b).We further examined the expression patterns of Ccdn3 and p35 mRNAs, and found that the level of Ccdn3 did not significantly change during P19 cell neuronal differentiation (Figure 9a). In contrast, p35 expression, similar to Znf179 expression, dramatically increased during P19 cell neuronal differentiation (Figure 9a). This suggests that p35 has a more important role than Ccdn3 in P19 cell neuronal differentiation. To investigate whether the induction of p35 is associated with p27 accumulation, we detected the protein expressions of p35, Cdk5, p27, and Ser10-phosphorylated p27 during P19 cell neuronal differentiation. During the process, as shown in Figure 9b, while the protein level of Cdk5 remained essentially the same, protein levels of p35, p27, and Ser10-phosphorylated p27 dramatically increased. In addition to P19 cells, we also detected protein expressions of Znf179, p35, p27, and Ser10-phosphorylated p27 during cerebellar granule cell neuronal differentiation and found similar results (Supplementary Figure S4). Our data showed strong correlations of temporal expression patterns among Znf179, p35, and p27 during P19 cell neuronal differentiation. These all support the hypothesis that Znf179 regulates p35 and p27 expressions and results in cell cycle arrest during the neuronal differentiation of P19 cells. To further investigate this possibility, we examined the protein expressions of these genes in Znf179-knockdown P19 cells. As shown in Figure 9c, Znf179 knockdown decreased expressions of the p35, p27, and Ser10 phosphorylated p27 proteins. We further analyzed mRNA expression by a quantitative RT-PCR. Similar to the microarray data, the quantitative RT-PCR analysis revealed a significant decrease in p35 mRNA in Znf179-knockdown P19 cells (Figure 9d). In contrast, p27 mRNA did not change (Figure 9d). These results reveal that Znf179 is required for proper neuronal differentiation and suggest p35 and p27 as possible mediators.Effect of p35 knockdown is similar but not identical to that of Znf179 knockdownTo examine the possibility that p35 mediates Znf179's function in neuronal differentiation, we used lentiviral shRNA to knock down p35 and examined the effect of p35 knockdown on P19 cell neuronal differentiation. As shown in Figure 10a, a p35 shRNA virus (sh_4) significantly reduced expression of the p35 protein after 4 days of aggregation in the presence of RA. In contrast, the negative control, the lentiviral shRNA that targets the luciferase gene, had no effect on p35 expression. Similar to Znf179, knockdown of p35 increased BrdU incorporation (Figures 10b and c) and attenuated neuronal differentiation (Figures 10d–f). However, the protein expression of p27 did not change (Figure 10a). We also examined whether p35 re-expression was sufficient to rescue the differentiation inhibition when Znf179 was silenced. Our results showed that the effect of Znf179 knockdown on P19 cell neuronal differentiation was partially rescued by p35 re-expression (Figure 11 and Supplementary Figure S5). These results show that although p35 is required for P19 cell neuronal differentiation, the effect of p35 was not associated with p27 accumulation, and other mechanisms may be also involved in Znf179's function.

Znf179 protein expression is restricted to the brain and particularly to neuronsAlthough the Znf179 gene was cloned >10 years ago, its function has not yet been elucidated.17 In this study, we generated polyclonal antibodies against Znf179 and used those antibodies to examine Znf179 protein expressions in various tissues. Antibody validation was performed, and the result is shown in Supplementary Figure S1. Among the examined tissues, a high level of Znf179 protein was observed in the brain (Figure 1a). We further compared expressions of the Znf179 protein in different brain regions and found that Znf179 was equally expressed in all selected regions, including the cerebral cortex, striatum, hippocampus, and cerebellum (Figure 1b), suggesting that Znf179 may have a general role in the brain. Using in situ hybridization, we also detected transcripts of Znf179 in the developing brain. As shown in Figure 2, Znf179 mRNA was detected in many brain areas, including cerebral cortex, hippocampus, and cerebellum. These data are consistent with a previous study16 and with the result of Allen Brain Atlas gene expression database (Image Series: 393241). Moreover, we found that the expression of Znf179 increased gradually during cerebellum development, suggesting a potential role of Znf179 in nervous system development. To further elucidate the cellular identity of Znf179-expressing cells, primary cultures of mouse cerebellar cells were immunostained with Znf179 and microtubule-associated protein 2 (MAP2) (a neuronal marker) or glial fibrillary acidic protein (GFAP) (a glial marker). As shown in Figure 1c, Znf179 was highly expressed in MAP2-positive cells and slightly expressed in GFAP-positive cells. Our result is consistent with the previous study by Orimo et al.16, which showed that Znf179 was detected in both neuron and glial cells.

Znf179 protein expression is restricted to the brain and particularly to neurons

Although the Znf179 gene was cloned >10 years ago, its function has not yet been elucidated.17 In this study, we generated polyclonal antibodies against Znf179 and used those antibodies to examine Znf179 protein expressions in various tissues. Antibody validation was performed, and the result is shown in Supplementary Figure S1. Among the examined tissues, a high level of Znf179 protein was observed in the brain (Figure 1a). We further compared expressions of the Znf179 protein in different brain regions and found that Znf179 was equally expressed in all selected regions, including the cerebral cortex, striatum, hippocampus, and cerebellum (Figure 1b), suggesting that Znf179 may have a general role in the brain. Using in situ hybridization, we also detected transcripts of Znf179 in the developing brain. As shown in Figure 2, Znf179 mRNA was detected in many brain areas, including cerebral cortex, hippocampus, and cerebellum. These data are consistent with a previous study16 and with the result of Allen Brain Atlas gene expression database (Image Series: 393241). Moreover, we found that the expression of Znf179 increased gradually during cerebellum development, suggesting a potential role of Znf179 in nervous system development. To further elucidate the cellular identity of Znf179-expressing cells, primary cultures of mouse cerebellar cells were immunostained with Znf179 and microtubule-associated protein 2 (MAP2) (a neuronal marker) or glial fibrillary acidic protein (GFAP) (a glial marker). As shown in Figure 1c, Znf179 was highly expressed in MAP2-positive cells and slightly expressed in GFAP-positive cells. Our result is consistent with the previous study by Orimo et al.16, which showed that Znf179 was detected in both neuron and glial cells.

Znf179 expression is upregulated during P19 cell neuronal differentiationTo study the function of the Znf179 gene, P19 embryonal carcinoma cells, a pluripotent cell line that can be induced by RA to differentiate into neurons, were used as the model.20 Using MAP2 immunofluorescent staining, we examined the neuronal differentiation of P19 cells. As shown in Figure 3a, monolayer-cultured P19 cells were undifferentiated and did not express the neuronal marker, MAP2. In contrast, neuronally differentiated P19 cells showed strong positive staining with the anti-MAP2 antibody (Figure 3b). During RA-induced P19 cell neuronal differentiation, the quantitative RT-PCR analysis showed that Znf179 messenger (m)RNA was dramatically increased after 4 days of aggregation in the presence of RA (Figure 3c). After plating, the expression of Znf179 further increased and was maintained at a high level. We also found that P19 cell aggregation in the absence of RA did not induce the expression of Znf179, indicating that RA was required for Znf179 induction during P19 cell neuronal differentiation. We detected the protein expression of Znf179 and found that the pattern of Znf179 protein expression was similar to that of Znf179 mRNA expression (Figure 3d). In the western blot analysis, the neuronal markers, growth-associated protein 43 (GAP-43) and βIII-tubulin, were also detected and highly expressed in differentiated P19 cells, indicating that the cells had differentiated into neurons. Consistent with the findings reported by Inoue,17 our results revealed that Znf179 is elevated during the differentiation of P19 cells into neurons and provided the precise time course of Znf179 gene expression. The results show that Znf179 may be required for neuronal differentiation.

Znf179 expression is upregulated during P19 cell neuronal differentiation

To study the function of the Znf179 gene, P19 embryonal carcinoma cells, a pluripotent cell line that can be induced by RA to differentiate into neurons, were used as the model.20 Using MAP2 immunofluorescent staining, we examined the neuronal differentiation of P19 cells. As shown in Figure 3a, monolayer-cultured P19 cells were undifferentiated and did not express the neuronal marker, MAP2. In contrast, neuronally differentiated P19 cells showed strong positive staining with the anti-MAP2 antibody (Figure 3b). During RA-induced P19 cell neuronal differentiation, the quantitative RT-PCR analysis showed that Znf179 messenger (m)RNA was dramatically increased after 4 days of aggregation in the presence of RA (Figure 3c). After plating, the expression of Znf179 further increased and was maintained at a high level. We also found that P19 cell aggregation in the absence of RA did not induce the expression of Znf179, indicating that RA was required for Znf179 induction during P19 cell neuronal differentiation. We detected the protein expression of Znf179 and found that the pattern of Znf179 protein expression was similar to that of Znf179 mRNA expression (Figure 3d). In the western blot analysis, the neuronal markers, growth-associated protein 43 (GAP-43) and βIII-tubulin, were also detected and highly expressed in differentiated P19 cells, indicating that the cells had differentiated into neurons. Consistent with the findings reported by Inoue,17 our results revealed that Znf179 is elevated during the differentiation of P19 cells into neurons and provided the precise time course of Znf179 gene expression. The results show that Znf179 may be required for neuronal differentiation.

Inhibition of Znf179 elevation by RNA interference leads to attenuated neuronal differentiation of P19 and cerebellar granule cellsGiven that Znf179 expression was dramatically induced during RA-induced P19 cell neuronal differentiation, we speculated that Znf179 may have a role in neuronal differentiation. In order to elucidate this possibility, we used lentiviral small hairpin (sh)RNA to knockdown Znf179 and examined the effect of Znf179 knockdown on P19 cell neuronal differentiation. Infection efficiency was analyzed by flow cytometry. More than 90% of the cells were infected (Supplementary Figure S2). As shown in Figure 4a, two Znf179 shRNA viruses (sh_2 and sh_3), which target different regions of the Znf179 transcript, significantly reduced expression of the Znf179 protein after 4 days of aggregation in the presence of RA. In contrast, the negative control, the lentiviral shRNA that targets the luciferase gene, had no effect on Znf179 expression. After plating, we examined the neuronal differentiation of P19 cells by immunofluorescent staining of the neuronal marker, MAP2. As shown in Figure 4b, high percentage of MAP2-positive cells were shown in control (non-infected and control virus-infected) cultures, whereas the number of MAP2-positive cells was significantly reduced in Znf179 shRNA virus-infected cells. Quantification of results showed that both the total number and the percentage of MAP2-positive cells were markedly reduced (Figures 4c and d). MAP2 protein expression was also detected by a western blot analysis, and similar results were observed (Figure 4e). In addition to MAP2-positive cells, the total cell number was also reduced in Znf179 shRNA virus-infected cells (Figure 4c). An MTT assay was performed to evaluate the number of attached cells after 4 h and 6 days of plating, which may reflect the proliferation or survival rate. As shown in Supplementary Figure S3, in control (non-infected and control virus-infected) cultures, the number of attached cells did not change between 4 h and 6 days of plating, indicating that the attached cells did not undergo proliferation. This is because proliferative, undifferentiated cells died in the serum-free media after plating. In contrast, after 6 days of plating, the amount of cells significantly decreased compared with that with 4 h of plating in Znf179 shRNA virus-infected cells. These results revealed that inhibition of Znf179 elevation could attenuate RA-induced P19 cell neuronal differentiation, which may have resulted from poor survival of cells during neuronal differentiation. To define whether early steps of the differentiation process were equally affected in culture, we detected the early neuronal marker, βIII-tubulin. As shown in Figure 4a, two Znf179 shRNA viruses (sh_2 and sh_3) reduced the expression of βIII-tubulin after 4 days of aggregation in the presence of RA, indicating that the cell fate was affected in the early stages of differentiation. We further detected the expression of neurogenin2 and neuroD1. Our results showed that the protein and mRNA expressions of neurogenin2 and neuroD1 did not change by knockdown of Znf179 (data not shown), indicating that the functional mechanisms of Znf179 on neuronal differentiation are different from neurogenein2 or neuroD1, or Znf179 functions downstream to neurogenein2 or neuroD1.In addition to P19 cells, we also examined the effect of Znf179 knockdown on primary cultured cerebellar granule cells. In a similar manner to P19 cells, knockdown of Znf179 expression led to attenuated cerebellar granule neuronal differentiation (Figures 5a and b). Immunofluorescent staining of the neuronal marker, MAP2, revealed that both the total number and the percentage of MAP2-positive cells were reduced in Znf179 shRNA virus-infected cerebellar granule cells (Figures 5c and d). As shown in Figure 5c, the total cell number also decreased in Znf179 shRNA virus-infected cells, indicating that cell viability was affected during the process of neuronal differentiation. To examine whether Znf179 knockdown also had an effect on glial differentiation, the glial marker, GFAP, was used for immunofluorescent staining. As shown in Figure 6, knockdown of Znf179 also led to attenuation of glial differentiation in primary cerebellar cultures. These results showed that Znf179 has a functional role in both neuronal and glial differentiation. The effect was seen in the cell line model, and also in the primary culture system.

Inhibition of Znf179 elevation by RNA interference leads to attenuated neuronal differentiation of P19 and cerebellar granule cells

Given that Znf179 expression was dramatically induced during RA-induced P19 cell neuronal differentiation, we speculated that Znf179 may have a role in neuronal differentiation. In order to elucidate this possibility, we used lentiviral small hairpin (sh)RNA to knockdown Znf179 and examined the effect of Znf179 knockdown on P19 cell neuronal differentiation. Infection efficiency was analyzed by flow cytometry. More than 90% of the cells were infected (Supplementary Figure S2). As shown in Figure 4a, two Znf179 shRNA viruses (sh_2 and sh_3), which target different regions of the Znf179 transcript, significantly reduced expression of the Znf179 protein after 4 days of aggregation in the presence of RA. In contrast, the negative control, the lentiviral shRNA that targets the luciferase gene, had no effect on Znf179 expression. After plating, we examined the neuronal differentiation of P19 cells by immunofluorescent staining of the neuronal marker, MAP2. As shown in Figure 4b, high percentage of MAP2-positive cells were shown in control (non-infected and control virus-infected) cultures, whereas the number of MAP2-positive cells was significantly reduced in Znf179 shRNA virus-infected cells. Quantification of results showed that both the total number and the percentage of MAP2-positive cells were markedly reduced (Figures 4c and d). MAP2 protein expression was also detected by a western blot analysis, and similar results were observed (Figure 4e). In addition to MAP2-positive cells, the total cell number was also reduced in Znf179 shRNA virus-infected cells (Figure 4c). An MTT assay was performed to evaluate the number of attached cells after 4 h and 6 days of plating, which may reflect the proliferation or survival rate. As shown in Supplementary Figure S3, in control (non-infected and control virus-infected) cultures, the number of attached cells did not change between 4 h and 6 days of plating, indicating that the attached cells did not undergo proliferation. This is because proliferative, undifferentiated cells died in the serum-free media after plating. In contrast, after 6 days of plating, the amount of cells significantly decreased compared with that with 4 h of plating in Znf179 shRNA virus-infected cells. These results revealed that inhibition of Znf179 elevation could attenuate RA-induced P19 cell neuronal differentiation, which may have resulted from poor survival of cells during neuronal differentiation. To define whether early steps of the differentiation process were equally affected in culture, we detected the early neuronal marker, βIII-tubulin. As shown in Figure 4a, two Znf179 shRNA viruses (sh_2 and sh_3) reduced the expression of βIII-tubulin after 4 days of aggregation in the presence of RA, indicating that the cell fate was affected in the early stages of differentiation. We further detected the expression of neurogenin2 and neuroD1. Our results showed that the protein and mRNA expressions of neurogenin2 and neuroD1 did not change by knockdown of Znf179 (data not shown), indicating that the functional mechanisms of Znf179 on neuronal differentiation are different from neurogenein2 or neuroD1, or Znf179 functions downstream to neurogenein2 or neuroD1.

In addition to P19 cells, we also examined the effect of Znf179 knockdown on primary cultured cerebellar granule cells. In a similar manner to P19 cells, knockdown of Znf179 expression led to attenuated cerebellar granule neuronal differentiation (Figures 5a and b). Immunofluorescent staining of the neuronal marker, MAP2, revealed that both the total number and the percentage of MAP2-positive cells were reduced in Znf179 shRNA virus-infected cerebellar granule cells (Figures 5c and d). As shown in Figure 5c, the total cell number also decreased in Znf179 shRNA virus-infected cells, indicating that cell viability was affected during the process of neuronal differentiation. To examine whether Znf179 knockdown also had an effect on glial differentiation, the glial marker, GFAP, was used for immunofluorescent staining. As shown in Figure 6, knockdown of Znf179 also led to attenuation of glial differentiation in primary cerebellar cultures. These results showed that Znf179 has a functional role in both neuronal and glial differentiation. The effect was seen in the cell line model, and also in the primary culture system.

Identification and annotation analysis of differentially expressed genes with Znf179 knockdownTo dissect the neuronal differentiation mechanisms of Znf179, we used a microarray to compare the global gene expression profiles between Znf179-knockdown and control cells, both of which were aggregated for 4 days in the presence of RA. Microarray results showed that compared with control virus-infected cells, 52 genes were found to be upregulated and 110 genes were found to be downregulated by ≥1.5-fold in both Znf179 shRNA virus (sh_2 and sh_3)-infected cells (Supplementary Table S2). Two upregulated and four downregulated genes were randomly chosen, and their expressions were measured by a quantitative RT-PCR analysis. The results are consistent with the microarray data (Table 1). We used the 162 (52 upregulated and 110 downregulated) genes to perform a functional annotation analysis using the ingenuity pathway analysis (IPA). As shown in Figure 7a, the five highest-ranking processes in the category of ‘physiological system development and functions' were associated with development, including organ development, embryonic development, tissue development, and nervous system development. This result further supports the role of Znf179 in neuronal development. In addition, cell growth, proliferation, and the cell cycle were identified to be the most significant molecular and cellular functions (Figure 7b). It is known that withdrawal from the cell cycle is a prerequisite for terminal differentiation. Thus, we were interested in determining whether Znf179 affects cell cycle progression, subsequently resulting in neuronal differentiation.

Identification and annotation analysis of differentially expressed genes with Znf179 knockdown

To dissect the neuronal differentiation mechanisms of Znf179, we used a microarray to compare the global gene expression profiles between Znf179-knockdown and control cells, both of which were aggregated for 4 days in the presence of RA. Microarray results showed that compared with control virus-infected cells, 52 genes were found to be upregulated and 110 genes were found to be downregulated by ≥1.5-fold in both Znf179 shRNA virus (sh_2 and sh_3)-infected cells (Supplementary Table S2). Two upregulated and four downregulated genes were randomly chosen, and their expressions were measured by a quantitative RT-PCR analysis. The results are consistent with the microarray data (Table 1). We used the 162 (52 upregulated and 110 downregulated) genes to perform a functional annotation analysis using the ingenuity pathway analysis (IPA). As shown in Figure 7a, the five highest-ranking processes in the category of ‘physiological system development and functions' were associated with development, including organ development, embryonic development, tissue development, and nervous system development. This result further supports the role of Znf179 in neuronal development. In addition, cell growth, proliferation, and the cell cycle were identified to be the most significant molecular and cellular functions (Figure 7b). It is known that withdrawal from the cell cycle is a prerequisite for terminal differentiation. Thus, we were interested in determining whether Znf179 affects cell cycle progression, subsequently resulting in neuronal differentiation.

Effect of Znf179 knockdown on the cell cycle profile and cell cycle regulatory proteinsTo examine the role of Znf179 in the cell cycle during neuronal differentiation, P19 cells were infected with negative control and Znf179 shRNA viruses. After 4 days of aggregation in the presence of RA, flow cytometry was performed to investigate whether Znf179 knockdown had affected cell cycle withdrawal in P19 cells. As expected, G0/G1-phase cells in RA-treated P19 cells had increased compared with non-treated cells (Figures 8a and b), in agreement with RA-induced cell cycle arrest and neuronal differentiation in P19 cells. However, Znf179 knockdown was able to abolish the increase in G0/G1-phase cells caused by RA treatment, indicating a failure to establish G0/G1 arrest. BrdU incorporation and phospho-histone H3 expression were used to assess cell proliferation. The results were similar to those in flow cytometric experiments. As shown in Figures 8c and d and 9c, both the percentage of BrdU-positive cells and the expression of phospho-histone H3 increased in Znf179-knockdown cells. These results show that Znf179 has a role in inhibiting cell cycle progression, particularly in blocking cells at the G0/G1-phase, and promotes neuronal differentiation. Among the differentially expressed genes with Znf179 knockdown, cyclin D3 (Ccdn3) and p35 (Cdk5r1) are closely linked to cell cycle regulation. Ccdn3 is known to activate Cdk4 and Cdk6, and thereby stimulate G1 progression.21 On the other hand, p35 is known to activate Cdk5 to phosphorylate and stabilize p27, thereby promoting cell cycle arrest.9 Our microarray results showed that Ccdn3 was upregulated and p35 was downregulated in Znf179-knockdown cells (Supplementary Table 2). This result is consistent with the flow cytometric data showing that Znf179 knockdown reduced the G0/G1 fraction (Figures 8a and b).We further examined the expression patterns of Ccdn3 and p35 mRNAs, and found that the level of Ccdn3 did not significantly change during P19 cell neuronal differentiation (Figure 9a). In contrast, p35 expression, similar to Znf179 expression, dramatically increased during P19 cell neuronal differentiation (Figure 9a). This suggests that p35 has a more important role than Ccdn3 in P19 cell neuronal differentiation. To investigate whether the induction of p35 is associated with p27 accumulation, we detected the protein expressions of p35, Cdk5, p27, and Ser10-phosphorylated p27 during P19 cell neuronal differentiation. During the process, as shown in Figure 9b, while the protein level of Cdk5 remained essentially the same, protein levels of p35, p27, and Ser10-phosphorylated p27 dramatically increased. In addition to P19 cells, we also detected protein expressions of Znf179, p35, p27, and Ser10-phosphorylated p27 during cerebellar granule cell neuronal differentiation and found similar results (Supplementary Figure S4). Our data showed strong correlations of temporal expression patterns among Znf179, p35, and p27 during P19 cell neuronal differentiation. These all support the hypothesis that Znf179 regulates p35 and p27 expressions and results in cell cycle arrest during the neuronal differentiation of P19 cells. To further investigate this possibility, we examined the protein expressions of these genes in Znf179-knockdown P19 cells. As shown in Figure 9c, Znf179 knockdown decreased expressions of the p35, p27, and Ser10 phosphorylated p27 proteins. We further analyzed mRNA expression by a quantitative RT-PCR. Similar to the microarray data, the quantitative RT-PCR analysis revealed a significant decrease in p35 mRNA in Znf179-knockdown P19 cells (Figure 9d). In contrast, p27 mRNA did not change (Figure 9d). These results reveal that Znf179 is required for proper neuronal differentiation and suggest p35 and p27 as possible mediators.

Effect of Znf179 knockdown on the cell cycle profile and cell cycle regulatory proteins

To examine the role of Znf179 in the cell cycle during neuronal differentiation, P19 cells were infected with negative control and Znf179 shRNA viruses. After 4 days of aggregation in the presence of RA, flow cytometry was performed to investigate whether Znf179 knockdown had affected cell cycle withdrawal in P19 cells. As expected, G0/G1-phase cells in RA-treated P19 cells had increased compared with non-treated cells (Figures 8a and b), in agreement with RA-induced cell cycle arrest and neuronal differentiation in P19 cells. However, Znf179 knockdown was able to abolish the increase in G0/G1-phase cells caused by RA treatment, indicating a failure to establish G0/G1 arrest. BrdU incorporation and phospho-histone H3 expression were used to assess cell proliferation. The results were similar to those in flow cytometric experiments. As shown in Figures 8c and d and 9c, both the percentage of BrdU-positive cells and the expression of phospho-histone H3 increased in Znf179-knockdown cells. These results show that Znf179 has a role in inhibiting cell cycle progression, particularly in blocking cells at the G0/G1-phase, and promotes neuronal differentiation. Among the differentially expressed genes with Znf179 knockdown, cyclin D3 (Ccdn3) and p35 (Cdk5r1) are closely linked to cell cycle regulation. Ccdn3 is known to activate Cdk4 and Cdk6, and thereby stimulate G1 progression.21 On the other hand, p35 is known to activate Cdk5 to phosphorylate and stabilize p27, thereby promoting cell cycle arrest.9 Our microarray results showed that Ccdn3 was upregulated and p35 was downregulated in Znf179-knockdown cells (Supplementary Table 2). This result is consistent with the flow cytometric data showing that Znf179 knockdown reduced the G0/G1 fraction (Figures 8a and b).

We further examined the expression patterns of Ccdn3 and p35 mRNAs, and found that the level of Ccdn3 did not significantly change during P19 cell neuronal differentiation (Figure 9a). In contrast, p35 expression, similar to Znf179 expression, dramatically increased during P19 cell neuronal differentiation (Figure 9a). This suggests that p35 has a more important role than Ccdn3 in P19 cell neuronal differentiation. To investigate whether the induction of p35 is associated with p27 accumulation, we detected the protein expressions of p35, Cdk5, p27, and Ser10-phosphorylated p27 during P19 cell neuronal differentiation. During the process, as shown in Figure 9b, while the protein level of Cdk5 remained essentially the same, protein levels of p35, p27, and Ser10-phosphorylated p27 dramatically increased. In addition to P19 cells, we also detected protein expressions of Znf179, p35, p27, and Ser10-phosphorylated p27 during cerebellar granule cell neuronal differentiation and found similar results (Supplementary Figure S4). Our data showed strong correlations of temporal expression patterns among Znf179, p35, and p27 during P19 cell neuronal differentiation. These all support the hypothesis that Znf179 regulates p35 and p27 expressions and results in cell cycle arrest during the neuronal differentiation of P19 cells. To further investigate this possibility, we examined the protein expressions of these genes in Znf179-knockdown P19 cells. As shown in Figure 9c, Znf179 knockdown decreased expressions of the p35, p27, and Ser10 phosphorylated p27 proteins. We further analyzed mRNA expression by a quantitative RT-PCR. Similar to the microarray data, the quantitative RT-PCR analysis revealed a significant decrease in p35 mRNA in Znf179-knockdown P19 cells (Figure 9d). In contrast, p27 mRNA did not change (Figure 9d). These results reveal that Znf179 is required for proper neuronal differentiation and suggest p35 and p27 as possible mediators.

Effect of p35 knockdown is similar but not identical to that of Znf179 knockdownTo examine the possibility that p35 mediates Znf179's function in neuronal differentiation, we used lentiviral shRNA to knock down p35 and examined the effect of p35 knockdown on P19 cell neuronal differentiation. As shown in Figure 10a, a p35 shRNA virus (sh_4) significantly reduced expression of the p35 protein after 4 days of aggregation in the presence of RA. In contrast, the negative control, the lentiviral shRNA that targets the luciferase gene, had no effect on p35 expression. Similar to Znf179, knockdown of p35 increased BrdU incorporation (Figures 10b and c) and attenuated neuronal differentiation (Figures 10d–f). However, the protein expression of p27 did not change (Figure 10a). We also examined whether p35 re-expression was sufficient to rescue the differentiation inhibition when Znf179 was silenced. Our results showed that the effect of Znf179 knockdown on P19 cell neuronal differentiation was partially rescued by p35 re-expression (Figure 11 and Supplementary Figure S5). These results show that although p35 is required for P19 cell neuronal differentiation, the effect of p35 was not associated with p27 accumulation, and other mechanisms may be also involved in Znf179's function.

Effect of p35 knockdown is similar but not identical to that of Znf179 knockdown

To examine the possibility that p35 mediates Znf179's function in neuronal differentiation, we used lentiviral shRNA to knock down p35 and examined the effect of p35 knockdown on P19 cell neuronal differentiation. As shown in Figure 10a, a p35 shRNA virus (sh_4) significantly reduced expression of the p35 protein after 4 days of aggregation in the presence of RA. In contrast, the negative control, the lentiviral shRNA that targets the luciferase gene, had no effect on p35 expression. Similar to Znf179, knockdown of p35 increased BrdU incorporation (Figures 10b and c) and attenuated neuronal differentiation (Figures 10d–f). However, the protein expression of p27 did not change (Figure 10a). We also examined whether p35 re-expression was sufficient to rescue the differentiation inhibition when Znf179 was silenced. Our results showed that the effect of Znf179 knockdown on P19 cell neuronal differentiation was partially rescued by p35 re-expression (Figure 11 and Supplementary Figure S5). These results show that although p35 is required for P19 cell neuronal differentiation, the effect of p35 was not associated with p27 accumulation, and other mechanisms may be also involved in Znf179's function.

DiscussionNeuronal differentiation is essential for the formation of the mammalian nervous system. It is a complex event that is coordinated with cell cycle regulation and cell fate determination. The process of neuronal differentiation is regulated by various intrinsic factors and extrinsic signals and is still under investigation. In this study, we demonstrate that Znf179 is a novel cell-intrinsic factor that is critical for neuronal differentiation in both P19 cells and the primary culture of cerebellar granule cells. Our results show that Znf179 functions upstream of cell cycle regulatory genes to promote cell cycle exit and neuronal differentiation.In this study, we generated polyclonal antibodies against Znf179 and used these antibodies to examine Znf179 protein expressions in various tissues (Figure 1). Flag- and HA-tagged mouse Znf179 proteins were expressed in 293T cells and detected by western blotting with anti-Flag, anti-HA, or anti-Znf179 antibodies (Supplementary Figure S1). The immunoblotting experiments showed a band of around 72 kDa in size corresponding to the Flag- or HA-tagged Znf179 protein, while the size of endogenous mouse Znf179 protein in brain tissue and P19 cells was smaller (Figures 1, 3, 4, and 9). Two reasons may explain this difference. First, the overexpressed mouse Znf179 proteins contain an additional HA- or Flag-tag, which result in a molecular-weight increase. A second possible reason for the difference in size of the endogenous and overexpressed mouse Znf179 proteins may be different transcripts that are generated in mouse brain and Znf179 expression construct-transfected cells. The mouse Znf179 gene contains 15 exons (Entrez gene ID: 22671). Compared with the human and rat Znf179 genes, there is an additional exon (exon 3), which contains 69 nucleotides in the coding sequence (Supplementary Figure S6A) of the mouse Znf179 gene. In fact, a Znf179 transcript lacking exon 3 was also reported in a mouse cerebellum complementary (c)DNA library (GenBank acc. no. AK131651). Our result also revealed there are two alternative Znf179 transcripts (with or without exon 3) in the mouse cortex but not in the rat cortex or human brain (Supplementary Figure S6B). Therefore, it is possible that the endogenous Znf179 protein we detected in the mouse brain and P19 cell lysate was translated from the transcript lacking exon 3, resulting in a smaller-sized protein than that of the Flag- or HA-tagged Znf179 proteins, which were translated from the transcript with exon 3. Moreover, as the shRNAs against Znf179 reduce the expression of two bands in primary cells (Figure 5a), we cannot rule out the possibility of the presence of different Znf179 isoforms in primary cultured cerebellar granule cells.Znf179 is a RING finger protein with a characteristic C3HC4 motif located in the N-terminus. The expression of the Znf179 gene is restricted to the brain and is regulated during brain development.16 However, its function has not yet been elucidated. The human Znf179 gene is located on chromosome 17p11.2, which is one of the most recombination-prone regions of the genome. In fact, the locus of the human Znf179 gene is present in the Smith–Magenis syndrome (SMS) common deletion region.22 Therefore, Znf179 was considered to be one of the candidate genes for SMS.16 SMS is a complex neuropediatric–neurobehavioral syndrome. Patients are characterized by moderate mental retardation and multiple congenital anomalies, including mild craniofacial and skeletal abnormalities, behavioral abnormalities, sleep disturbances, a short stature, and cardiac and renal malformations.23 Although RAI1, another gene located at 17p11.2, is regarded as the primary gene responsible for most features of SMS, other genes in the 17p11.2 region still contribute to the variability and severity of the phenotype of SMS.24 In this study, we observed that the Znf179 gene was markedly induced during P19 cell neuronal differentiation (Figure 3). We further showed that Znf179 knockdown significantly attenuated neuronal differentiation of P19 cells and the primary culture of cerebellar granule cells, suggesting that Znf179 participates in neuronal differentiation and may have a role in SMS.Using a microarray approach and subsequent functional annotation analysis, we identified differentially expressed genes in Znf179-knockdown cells and found that several genes were involved in cell proliferation and cell cycle control (Figure 7b). Furthermore, a flow cytometric study revealed that Znf179 knockdown abolished the increase in G0/G1-phase cells caused by RA treatment, suggesting that Znf179 might inhibit cell cycle progression, resulting in neuronal differentiation. Among differentially expressed cell cycle regulatory genes, p35 was downregulated in Znf179-knockdown cells, and the expression pattern of p35 during P19 cell neuronal differentiation was similar to that of Znf179 (Figures 9a and b). p35 is a neuronal-specific Cdk5 activator. Expression of p35 and activation of Cdk5 in post-mitotic neurons are known to be important for nervous system development.10 We therefore speculated that p35 may function downstream of Znf179 to regulate neuronal differentiation. p35 is transcriptionally regulated by various pathways and transcription factors such as the ERK pathway and heat-shock factor 2, which are also important for neuronal differentiation.25, 26, 27 The mechanism by which Znf179 regulates p35 expression is not clear and needs further investigation. A previous study using a microarray analysis showed that Znf179 was significantly downregulated in the brain of HD transgenic mice.18 Interestingly, the expression of p35 also decreased in that animal model. Furthermore, p35/Cdk5 was found to suppress the formation of mutant Huntingtin aggregates, a major pathological feature of HD, in primary cultured neurons.28 Taken together, these observations support our speculation that p35 may function downstream of Znf17, and Znf179 may be associated with HD.p27, a CKI of the Cip/Kip family, which is involved in arresting cell cycle progression, was found to be indispensable for neuronal differentiation of embryonal carcinoma cells and to have a key role in cortical development.6, 29 In that work, Sasaki et al.6 found that the expression of p27 increased with P19 cell aggregation in the presence of RA, and inhibition of p27 expression by antisense p27 oligonucleotides resulted in blockade of neuronal differentiation. In addition to regulating the cell cycle, p27 was further reported to promote neuronal differentiation of cortical progenitors by stabilizing the proneural protein, neurogenin 2.29 These all suggest that p27 is a possible mediator of Znf179's effect on P19 cell neuronal differentiation. p27 was shown to be phosphorylated by p35/Cdk5, which resulted in increasing its protein stability.9 Moreover, the phosphorylation of p27 also promotes its translocation to the cytoplasm, contributing to unrelated cell cycle functions such as cell migration.9, 30 In this study, we observed that in agreement with the decrease in p35 expression, the protein level of p27 and phosphorylated p27 were diminished in Znf179-knockdown cells (Figure 9c). However, the p27 and phosphorylated p27 protein expression did not change after knockdown of p35 (Figure 10a), suggesting that the effect of Znf179 on p27 protein expression is not mediated by the p35/Cdk5 pathway. In addition to p35/cdk5, the S-phase kinase-associated protein (Skp)2 was found to be required for the ubiquitin-mediated degradation of p27, and destruction of Skp2 is essential for proper terminal differentiation of neuronal precursors.31, 32 In LAN-5 neuroblastoma cells, RA treatment was also found to reduce the rate of ubiquitin/proteasome-mediated degradation of the p27 protein and induce neuronal differentiation.33 The possible mechanisms involved in p27 protein regulation by Znf179 will be further investigated.In our result, inhibition of Znf179 elevation leads to decrease the number of total and MAP2-positive cells (Figures 4c and 5c). The result revealed that inhibition of Znf179 elevation can attenuate RA-induced P19 cell neuronal differentiation, which may have resulted from poor survival of cells during neuronal differentiation. The possible mechanism may be activation of E2F1, which is known to induce cell death during P19 and cerebellar granule cell differentiation.34, 35 In our results, Znf179 knockdown led to decreases in p35 and p27 expressions. p27 is a CKI. The decrease in p27 expression may have resulted in E2F1 activation. Another recent study also revealed that p35/Cdk5 binds to E2F1 and inhibits E2F1 activity.36 We further analyzed microarray data and found that many E2F1-regulated genes, such as Ccdn3, calregulin, cellular RA-binding protein II, Mcm3, and Rad54l, were upregulated by Znf179 knockdown (Supplementary Table S2). These all imply that E2F1 activation may be the cause of poor survival of Znf179-knockdown cells during neuronal differentiation.It is known that withdrawal from the cell cycle is a prerequisite for terminal differentiation. Recent studies suggested the existence of crosstalk between the signal pathways that control these two mechanisms.1, 2 In this study, we observed that Znf179 regulated cell cycle exit and was critical for neuronal differentiation. Further studies need to examine whether Znf179 also participates in cell fate determination. However, the functional annotation analysis revealed that several genes closely associated with nervous system development and function were differentially expressed in Znf179-knockdown cells (Figure 7a). For instance, the expressions of GAP-43, MAPT, neurexin 2 (NRXN2), NRXN3, glycoprotein M6A, and so on, which are all neuronal proteins and involved in neuronal differentiation, maturation, and neuronal cell functions, decreased upon knockdown of Znf179 (Supplementary Table S2).37, 38, 39 These results show that Znf179 may also regulate neuronal cell fate determination.In conclusion, we demonstrate that Znf179, a brain-specific RING finger protein, is induced during P19 cell neuronal differentiation and is predominantly expressed in neurons. We also reveal the cellular function of Znf179 in neuronal differentiation for the first time and further demonstrate that Znf179 possibly promotes neuronal differentiation through inhibiting cell cycle progression. Finally, p35 expression and p27 protein accumulation, which are important for neuronal differentiation, are associated with the expression of Znf179, suggesting the possible molecular mechanisms by which Znf179 exerts its function.

Neuronal differentiation is essential for the formation of the mammalian nervous system. It is a complex event that is coordinated with cell cycle regulation and cell fate determination. The process of neuronal differentiation is regulated by various intrinsic factors and extrinsic signals and is still under investigation. In this study, we demonstrate that Znf179 is a novel cell-intrinsic factor that is critical for neuronal differentiation in both P19 cells and the primary culture of cerebellar granule cells. Our results show that Znf179 functions upstream of cell cycle regulatory genes to promote cell cycle exit and neuronal differentiation.

In this study, we generated polyclonal antibodies against Znf179 and used these antibodies to examine Znf179 protein expressions in various tissues (Figure 1). Flag- and HA-tagged mouse Znf179 proteins were expressed in 293T cells and detected by western blotting with anti-Flag, anti-HA, or anti-Znf179 antibodies (Supplementary Figure S1). The immunoblotting experiments showed a band of around 72 kDa in size corresponding to the Flag- or HA-tagged Znf179 protein, while the size of endogenous mouse Znf179 protein in brain tissue and P19 cells was smaller (Figures 1, 3, 4, and 9). Two reasons may explain this difference. First, the overexpressed mouse Znf179 proteins contain an additional HA- or Flag-tag, which result in a molecular-weight increase. A second possible reason for the difference in size of the endogenous and overexpressed mouse Znf179 proteins may be different transcripts that are generated in mouse brain and Znf179 expression construct-transfected cells. The mouse Znf179 gene contains 15 exons (Entrez gene ID: 22671). Compared with the human and rat Znf179 genes, there is an additional exon (exon 3), which contains 69 nucleotides in the coding sequence (Supplementary Figure S6A) of the mouse Znf179 gene. In fact, a Znf179 transcript lacking exon 3 was also reported in a mouse cerebellum complementary (c)DNA library (GenBank acc. no. AK131651). Our result also revealed there are two alternative Znf179 transcripts (with or without exon 3) in the mouse cortex but not in the rat cortex or human brain (Supplementary Figure S6B). Therefore, it is possible that the endogenous Znf179 protein we detected in the mouse brain and P19 cell lysate was translated from the transcript lacking exon 3, resulting in a smaller-sized protein than that of the Flag- or HA-tagged Znf179 proteins, which were translated from the transcript with exon 3. Moreover, as the shRNAs against Znf179 reduce the expression of two bands in primary cells (Figure 5a), we cannot rule out the possibility of the presence of different Znf179 isoforms in primary cultured cerebellar granule cells.

Znf179 is a RING finger protein with a characteristic C3HC4 motif located in the N-terminus. The expression of the Znf179 gene is restricted to the brain and is regulated during brain development.16 However, its function has not yet been elucidated. The human Znf179 gene is located on chromosome 17p11.2, which is one of the most recombination-prone regions of the genome. In fact, the locus of the human Znf179 gene is present in the Smith–Magenis syndrome (SMS) common deletion region.22 Therefore, Znf179 was considered to be one of the candidate genes for SMS.16 SMS is a complex neuropediatric–neurobehavioral syndrome. Patients are characterized by moderate mental retardation and multiple congenital anomalies, including mild craniofacial and skeletal abnormalities, behavioral abnormalities, sleep disturbances, a short stature, and cardiac and renal malformations.23 Although RAI1, another gene located at 17p11.2, is regarded as the primary gene responsible for most features of SMS, other genes in the 17p11.2 region still contribute to the variability and severity of the phenotype of SMS.24 In this study, we observed that the Znf179 gene was markedly induced during P19 cell neuronal differentiation (Figure 3). We further showed that Znf179 knockdown significantly attenuated neuronal differentiation of P19 cells and the primary culture of cerebellar granule cells, suggesting that Znf179 participates in neuronal differentiation and may have a role in SMS.

Using a microarray approach and subsequent functional annotation analysis, we identified differentially expressed genes in Znf179-knockdown cells and found that several genes were involved in cell proliferation and cell cycle control (Figure 7b). Furthermore, a flow cytometric study revealed that Znf179 knockdown abolished the increase in G0/G1-phase cells caused by RA treatment, suggesting that Znf179 might inhibit cell cycle progression, resulting in neuronal differentiation. Among differentially expressed cell cycle regulatory genes, p35 was downregulated in Znf179-knockdown cells, and the expression pattern of p35 during P19 cell neuronal differentiation was similar to that of Znf179 (Figures 9a and b). p35 is a neuronal-specific Cdk5 activator. Expression of p35 and activation of Cdk5 in post-mitotic neurons are known to be important for nervous system development.10 We therefore speculated that p35 may function downstream of Znf179 to regulate neuronal differentiation. p35 is transcriptionally regulated by various pathways and transcription factors such as the ERK pathway and heat-shock factor 2, which are also important for neuronal differentiation.25, 26, 27 The mechanism by which Znf179 regulates p35 expression is not clear and needs further investigation. A previous study using a microarray analysis showed that Znf179 was significantly downregulated in the brain of HD transgenic mice.18 Interestingly, the expression of p35 also decreased in that animal model. Furthermore, p35/Cdk5 was found to suppress the formation of mutant Huntingtin aggregates, a major pathological feature of HD, in primary cultured neurons.28 Taken together, these observations support our speculation that p35 may function downstream of Znf17, and Znf179 may be associated with HD.

p27, a CKI of the Cip/Kip family, which is involved in arresting cell cycle progression, was found to be indispensable for neuronal differentiation of embryonal carcinoma cells and to have a key role in cortical development.6, 29 In that work, Sasaki et al.6 found that the expression of p27 increased with P19 cell aggregation in the presence of RA, and inhibition of p27 expression by antisense p27 oligonucleotides resulted in blockade of neuronal differentiation. In addition to regulating the cell cycle, p27 was further reported to promote neuronal differentiation of cortical progenitors by stabilizing the proneural protein, neurogenin 2.29 These all suggest that p27 is a possible mediator of Znf179's effect on P19 cell neuronal differentiation. p27 was shown to be phosphorylated by p35/Cdk5, which resulted in increasing its protein stability.9 Moreover, the phosphorylation of p27 also promotes its translocation to the cytoplasm, contributing to unrelated cell cycle functions such as cell migration.9, 30 In this study, we observed that in agreement with the decrease in p35 expression, the protein level of p27 and phosphorylated p27 were diminished in Znf179-knockdown cells (Figure 9c). However, the p27 and phosphorylated p27 protein expression did not change after knockdown of p35 (Figure 10a), suggesting that the effect of Znf179 on p27 protein expression is not mediated by the p35/Cdk5 pathway. In addition to p35/cdk5, the S-phase kinase-associated protein (Skp)2 was found to be required for the ubiquitin-mediated degradation of p27, and destruction of Skp2 is essential for proper terminal differentiation of neuronal precursors.31, 32 In LAN-5 neuroblastoma cells, RA treatment was also found to reduce the rate of ubiquitin/proteasome-mediated degradation of the p27 protein and induce neuronal differentiation.33 The possible mechanisms involved in p27 protein regulation by Znf179 will be further investigated.

In our result, inhibition of Znf179 elevation leads to decrease the number of total and MAP2-positive cells (Figures 4c and 5c). The result revealed that inhibition of Znf179 elevation can attenuate RA-induced P19 cell neuronal differentiation, which may have resulted from poor survival of cells during neuronal differentiation. The possible mechanism may be activation of E2F1, which is known to induce cell death during P19 and cerebellar granule cell differentiation.34, 35 In our results, Znf179 knockdown led to decreases in p35 and p27 expressions. p27 is a CKI. The decrease in p27 expression may have resulted in E2F1 activation. Another recent study also revealed that p35/Cdk5 binds to E2F1 and inhibits E2F1 activity.36 We further analyzed microarray data and found that many E2F1-regulated genes, such as Ccdn3, calregulin, cellular RA-binding protein II, Mcm3, and Rad54l, were upregulated by Znf179 knockdown (Supplementary Table S2). These all imply that E2F1 activation may be the cause of poor survival of Znf179-knockdown cells during neuronal differentiation.

It is known that withdrawal from the cell cycle is a prerequisite for terminal differentiation. Recent studies suggested the existence of crosstalk between the signal pathways that control these two mechanisms.1, 2 In this study, we observed that Znf179 regulated cell cycle exit and was critical for neuronal differentiation. Further studies need to examine whether Znf179 also participates in cell fate determination. However, the functional annotation analysis revealed that several genes closely associated with nervous system development and function were differentially expressed in Znf179-knockdown cells (Figure 7a). For instance, the expressions of GAP-43, MAPT, neurexin 2 (NRXN2), NRXN3, glycoprotein M6A, and so on, which are all neuronal proteins and involved in neuronal differentiation, maturation, and neuronal cell functions, decreased upon knockdown of Znf179 (Supplementary Table S2).37, 38, 39 These results show that Znf179 may also regulate neuronal cell fate determination.

In conclusion, we demonstrate that Znf179, a brain-specific RING finger protein, is induced during P19 cell neuronal differentiation and is predominantly expressed in neurons. We also reveal the cellular function of Znf179 in neuronal differentiation for the first time and further demonstrate that Znf179 possibly promotes neuronal differentiation through inhibiting cell cycle progression. Finally, p35 expression and p27 protein accumulation, which are important for neuronal differentiation, are associated with the expression of Znf179, suggesting the possible molecular mechanisms by which Znf179 exerts its function.

Subjects and MethodsAnimalsMale C57BL/6 J (B6) mice (8 weeks old) used in this study were treated in accordance with guidelines of the University Committee on the Care and Use of Experimental Animals of National Cheng Kung University (Tainan, Taiwan). They were housed in an air-conditioned vivarium with free access to food and water and a 12/12-h light/dark cycle.AntibodiesThe following antibodies were used for the western blot and immunofluorescence analyses. Antibodies against MAP2, and β-actin were purchased from Sigma-Aldrich (St. Louis, MO, USA); those against Cdk5, p35, p27, phospho-Ser 10 p27, and phospho-Ser 10 histone H3 were purchased from Santa Cruz Biotechnology (Santa Cruz, CA, USA); that against GAP-43 was purchased from AbD Serotec (Raleigh, NC, USA); and those against βIII-tubulin and GFAP were purchased from Promega (Madison, WI, USA) and Zymed Laboratories (San Francisco, CA, USA), respectively. The polyclonal Znf179 antibodies were generated against a synthetic peptide (CEKEEDERVQGDREPLLQEE) corresponding to C terminal amino acids 634∼654 of mouse Znf179.In situ hybridizationThe brains of P3, P7, and P14 mice were fixed by transcardial perfusion with 4% paraformaldehyde in 0.l M phosphate buffer, followed by immersion 2 days at 4 °C. The specimens were then dehydrated in 30% sucrose, embedded in OCT, and cryosectioned into 10-μm sections. The sections were then mounted onto poly--lysine-coated glass slides and treated with proteinase K (0.5 μg/ml; Sigma-Aldrich) and 0.25% acetic anhydride (Sigma-Aldrich). After acetylation, the sections were dehydrated by a graded series of ethanol and then stored at −80 °C. To generate Znf179 probes for hybridization, a 821-bp Znf179 cDNA fragment was generated by PCR using IMAGE clone 4506141 (GeneBank entry BC037118) as template with the forward primer 5′-GAGCCGCTGTCAAGGGTA-3′ and the reverse primer 5′-CCCAGCTCAGCCAGTGTT-3′. The cDNA fragment was subsequently inserted into the pGEM-T Easy vector. The radiolabeled (35S) sense and antisense cRNA probes were synthesized using an in vitro transcription kit from Promega, and were then purified with Sephadex G-50 column (Amersham Biosciences, Piscataway, NJ, USA). Dehydrated sections were hybridized with probes (104 c.p.m./μl) at 60 °C for 16 h. After hybridization, the sections were washed, dehydrated, and exposed to X-ray film.Primary cultures of mouse cerebellar granule cellsCerebellar granule neurons were prepared from 7-day-old B6 mice according to a procedure modified from Courtney et al.40 Briefly, the cerebellum was mechanically dissociated and filtered through a 70-μm nylon mesh filter (Small Parts, Miami Lakes, FL, USA). Cells were plated onto a plastic culture plate that was pre-coated with poly--lysine (30 μg/ml; Sigma-Aldrich) and maintained in Dulbecco's modified Eagle's medium Glutamax-I (Invitrogen, Carlsbad, CA, USA) medium containing 10% fetal bovine serum (FBS; Invitrogen), 20 mM KCl, and 100 U/ml penicillin and streptomycin (Invitrogen), and completed with N2 and B27 supplements (Invitrogen). The growth of non-neuronal cells was inhibited by the addition of the antimitotics, uridine, and fluorodeoxyuridine (10 μM; Sigma-Aldrich), for 18 h after seeding. Cells were kept at 37 °C in a humidified atmosphere of 5% CO2 and 95% air.ImmunofluorescenceCells were fixed for 15 min with 4% formaldehyde in phosphate-buffered saline (PBS) and then permeabilized with cold acetone. Antibodies were then incubated with fixed cells for 4 h at room temperature. Cells were washed three times with PBS followed by incubation with a secondary antibody for 1 h at room temperature. Nuclei were revealed by ProLong Gold antifade reagent with DAPI (Invitrogen). Coverslips were inverted, mounted on slides, and sealed with nail polish. Pictures were taken using fluorescence microscopy.P19 cell neuronal differentiationMouse embryonal carcinoma P19 cells were used in this study, and RA-induced P19 cell neuronal differentiation was performed with a slight modification of procedures described by Lee et al.41 Briefly, P19 cells were maintained in alpha minimum essential medium (α-MEM; Invitrogen) supplemented with 7.5% bovine serum (BS; Invitrogen) and 2.5% FBS at 37 °C in a humidified atmosphere (5% CO2 and 95% air). To induce differentiation, cells were allowed to aggregate in bacterial-grade Petri dishes at a seeding density of 1 × 105 cells/ml in the presence of 1 μM all-trans-RA (Sigma-Aldrich). After 4 days of aggregation, cells were dissociated into single cells by trypsin-EDTA, and were plated in a poly--lysine-coated tissue culture dish at a density of 1 × 105 cells/cm2 in Neurobasal-A medium (Invitrogen) with a 1 × B27 supplement. Cells were allowed to attach for 24 h, and then were exposed to 10 μM Ara-C 24 h to inhibit proliferation of non-neuronal cells.Reverse transcription (RT) and quantitative (q) real-time PCR assaysTotal RNA was extracted using the Trizol reagent (Invitrogen) following the manufacturer's recommendations. RT was performed with 1.5 μg of total RNA using M-MLV reverse transcriptase (Invitrogen). A real-time qPCR was performed using the SYBR advantage qPCR premix (Invitrogen). The PCRs were then performed using the following conditions for 40 cycles: 95 °C for 15 s, 60 °C for 15 s, and 72 °C for 20 s. Sequences of the primers used are listed in Supplementary Table 1. Real-time fluorescence monitoring and a melting-curve analysis were performed with LightCycler according to the manufacturer's recommendations (Roche Molecular Diagnostics, Lewes, East Sussex, UK). Negative controls containing no cDNA template were included in each experiment. A melting curve was created at the end of the PCR cycle to confirm that a single product had been amplified. Data were analyzed by LightCycler software vers. 3.5 (Roche Molecular Diagnostics) to determine the threshold cycle (Cp) above the background for each reaction. The relative transcript amount of the target gene, calculated using standard curves of serial cDNA dilutions, was normalized to that of β-actin of the same cDNA.Western blot analysisCells were rinsed with ice-cold PBS and lysed in radioimmunoprecipitation assay buffer (150 mM NaCl, 1% NP40, 0.5% Na-deoxycholate, 0.1% sodium dodecylsulfate, and 50 mM Tris-HCl; pH 8.0) with the addition of a protease inhibitor cocktail (Roche Molecular Diagnostics). Cell debris was removed by centrifugation at 12 000 r.p.m. for 15 min at 4 °C, and the supernatant was utilized for a western blot analysis. Protein concentrations were determined using the Bio-Rad dye-binding method (Bio-Rad Laboratories, Hercules, CA, USA) with bovine serum albumin as the standard. Equal amounts of sample were separated by 10% polyacrylamide gel electrophoresis. The resolved proteins were then electroblotted onto Immobilon polyvinylidene difluoride membranes (Millipore, Bedford, MA, USA). Membranes were blocked with 5% nonfat milk and then sequentially incubated with the primary and secondary antibodies. After washing, the blots were processed for visualization using an enhanced chemiluminescence system (Pierce, Rockford, IL, USA). Blots were then exposed to X-ray film to obtain the fluorographic images.Lentiviral vector production and infectionReplication-deficient lentiviruses were generated by transient transfection of shRNAs, the packaging vector, pCMV-ΔR8.91 (containing the gag, pol, and rev genes), and the envelope plasmid, pMD.G (VSV-G-expressing plasmid), in 293T cells using the Trans IT-LT1 reagent (Mirus Bio, Madison, WI, USA). The shRNAs included Znf179 shRNAs (TRCN0000040979 and TRCN0000040980), which targeted the mouse Znf179 transcript sequences, 5′-GCCGGAAGATCTGTAAGCAAA-3′ (sh_2) and 5′-GATACCGATCTGGGCTATCTA-3′ (sh_3), p35 shRNA (TRCN0000009518), which targeted the mouse p35 transcript sequence, 5′-GACCCACACTATTTCACACAA-3′ (sh_4), and the luciferase control (pLKO.1-shLuc) plasmid constructs were used. To monitor and track the virus titer, the puromycin resistance gene on pLKO.1-based lentiviral vectors was replaced with EGFP. To generate p35 expression vectors, p35 and EGFP-p35 cDNA fragments were inserted into the BamHI and KpnI sites of the sh_2 and shLuc constructs. Beginning at 36 h post-transfection, the viral supernatant was collected every 24 h for 4 days. The pooled supernatant was cleared by low-speed centrifugation followed by filtering through 0.45-μm filters. Viral particles were concentrated by ultracentrifugation at 110 500 g for 2 h. Pellets were resuspended in PBS at 1/500 of the original volume and kept at −70 °C. The titer of viral stocks was determined by flow cytometric analysis of EGFP expression 48 h after infection. In general, viral titers were about 3∼5 × 107 TU/ml. P19 cells were infected twice with the virus (MOI=4) before 2 days and at the days of cell aggregation in 8 μg/ml polybrene.Cell viability assayCell viability was determined by adding MTT (Sigma-Aldrich) to the cell cultures at a final concentration of 0.5 mg/ml. After a 1-h incubation at 37 °C, the medium was discarded, and the dark crystals formed were dissolved in DMSO. The absorbency at 550 nm was measured on a micro-enzyme-linked immunosorbent (ELISA) reader.DNA microarray and IPATotal RNA was extracted using the Trizol reagent (Invitrogen) following the manufacturer's recommendations, and RNA integrity was assessed using the Experion RNA StdSens Analysis Kit (Bio-Rad Laboratories). Microarray experiments were performed using MouseWG-6 v2.0 BeadChips, and over 48 000 known genes, gene candidates, and splice variants were analyzed (Illumina, San Diego, CA, USA) according to the instructions provided. The cDNA and cRNA were prepared from 500 ng of total RNA using Ambion's Illumina RNA Amplification kit (Ambion, Austin, TX, USA) following their instructions. The cRNA was hybridized to Illumina's MouseWG-6 v2.0 BeadChips, which were scanned on an Illumina BeadStation 500 × . Illumina Beadstudio software was used to assess fluorescent hybridization signals. Before proceeding with the differential gene expression analysis, quantile normalization algorithms, present in the Illumina data analysis software, BeadStudio,42 were applied to the data. To interpret the biological significance of the candidate genes, the IPA software was used (Ingenuity, Redwood, CA, USA). Specifically, the functional analysis tool identified the physiological system development and functions and molecular and cellular functions that were most significant to the data set. A right-tailed Fisher's exact test was used to calculate a P-value, which indicated the probability that each biological function assigned to the data set was due to chance alone.Cell cycle analysisCells were trypsinized, collected by centrifugation, resuspended, and washed in PBS before fixing in 75% ethanol for 1 h at room temperature. Fixed cells were incubated with 20 μg/ml propidium iodide, 10 mg/ml RNase A, and 0.1% Triton X-100 for 30 min before analysis. The cell cycle was analyzed using a BD FACS Calibur flow cytometer (BD Biosciences, San Jose, CA, USA), and the data were analyzed using WinMDI software (Joseph Trotter, Scripps Research Insitute, La Jolla, CA, USA).BrdU incorporationP19 cells, which were aggregated in the presence of 1 μM RA for 4 days, were dissociated and plated in α-MEM (Invitrogen) supplemented with 7.5% BS (Invitrogen) and 2.5% FBS (Invitrogen). BrdU (Sigma-Aldrich), at 40 μM, was added to the media and incubated for 4 h before fixation. An anti-BrdU antibody (Sigma-Aldrich) was used for immunofluorescence.

Subjects and Methods

AnimalsMale C57BL/6 J (B6) mice (8 weeks old) used in this study were treated in accordance with guidelines of the University Committee on the Care and Use of Experimental Animals of National Cheng Kung University (Tainan, Taiwan). They were housed in an air-conditioned vivarium with free access to food and water and a 12/12-h light/dark cycle.

Male C57BL/6 J (B6) mice (8 weeks old) used in this study were treated in accordance with guidelines of the University Committee on the Care and Use of Experimental Animals of National Cheng Kung University (Tainan, Taiwan). They were housed in an air-conditioned vivarium with free access to food and water and a 12/12-h light/dark cycle.

AntibodiesThe following antibodies were used for the western blot and immunofluorescence analyses. Antibodies against MAP2, and β-actin were purchased from Sigma-Aldrich (St. Louis, MO, USA); those against Cdk5, p35, p27, phospho-Ser 10 p27, and phospho-Ser 10 histone H3 were purchased from Santa Cruz Biotechnology (Santa Cruz, CA, USA); that against GAP-43 was purchased from AbD Serotec (Raleigh, NC, USA); and those against βIII-tubulin and GFAP were purchased from Promega (Madison, WI, USA) and Zymed Laboratories (San Francisco, CA, USA), respectively. The polyclonal Znf179 antibodies were generated against a synthetic peptide (CEKEEDERVQGDREPLLQEE) corresponding to C terminal amino acids 634∼654 of mouse Znf179.

The following antibodies were used for the western blot and immunofluorescence analyses. Antibodies against MAP2, and β-actin were purchased from Sigma-Aldrich (St. Louis, MO, USA); those against Cdk5, p35, p27, phospho-Ser 10 p27, and phospho-Ser 10 histone H3 were purchased from Santa Cruz Biotechnology (Santa Cruz, CA, USA); that against GAP-43 was purchased from AbD Serotec (Raleigh, NC, USA); and those against βIII-tubulin and GFAP were purchased from Promega (Madison, WI, USA) and Zymed Laboratories (San Francisco, CA, USA), respectively. The polyclonal Znf179 antibodies were generated against a synthetic peptide (CEKEEDERVQGDREPLLQEE) corresponding to C terminal amino acids 634∼654 of mouse Znf179.

In situ hybridizationThe brains of P3, P7, and P14 mice were fixed by transcardial perfusion with 4% paraformaldehyde in 0.l M phosphate buffer, followed by immersion 2 days at 4 °C. The specimens were then dehydrated in 30% sucrose, embedded in OCT, and cryosectioned into 10-μm sections. The sections were then mounted onto poly--lysine-coated glass slides and treated with proteinase K (0.5 μg/ml; Sigma-Aldrich) and 0.25% acetic anhydride (Sigma-Aldrich). After acetylation, the sections were dehydrated by a graded series of ethanol and then stored at −80 °C. To generate Znf179 probes for hybridization, a 821-bp Znf179 cDNA fragment was generated by PCR using IMAGE clone 4506141 (GeneBank entry BC037118) as template with the forward primer 5′-GAGCCGCTGTCAAGGGTA-3′ and the reverse primer 5′-CCCAGCTCAGCCAGTGTT-3′. The cDNA fragment was subsequently inserted into the pGEM-T Easy vector. The radiolabeled (35S) sense and antisense cRNA probes were synthesized using an in vitro transcription kit from Promega, and were then purified with Sephadex G-50 column (Amersham Biosciences, Piscataway, NJ, USA). Dehydrated sections were hybridized with probes (104 c.p.m./μl) at 60 °C for 16 h. After hybridization, the sections were washed, dehydrated, and exposed to X-ray film.

In situ hybridization

The brains of P3, P7, and P14 mice were fixed by transcardial perfusion with 4% paraformaldehyde in 0.l M phosphate buffer, followed by immersion 2 days at 4 °C. The specimens were then dehydrated in 30% sucrose, embedded in OCT, and cryosectioned into 10-μm sections. The sections were then mounted onto poly--lysine-coated glass slides and treated with proteinase K (0.5 μg/ml; Sigma-Aldrich) and 0.25% acetic anhydride (Sigma-Aldrich). After acetylation, the sections were dehydrated by a graded series of ethanol and then stored at −80 °C. To generate Znf179 probes for hybridization, a 821-bp Znf179 cDNA fragment was generated by PCR using IMAGE clone 4506141 (GeneBank entry BC037118) as template with the forward primer 5′-GAGCCGCTGTCAAGGGTA-3′ and the reverse primer 5′-CCCAGCTCAGCCAGTGTT-3′. The cDNA fragment was subsequently inserted into the pGEM-T Easy vector. The radiolabeled (35S) sense and antisense cRNA probes were synthesized using an in vitro transcription kit from Promega, and were then purified with Sephadex G-50 column (Amersham Biosciences, Piscataway, NJ, USA). Dehydrated sections were hybridized with probes (104 c.p.m./μl) at 60 °C for 16 h. After hybridization, the sections were washed, dehydrated, and exposed to X-ray film.

Primary cultures of mouse cerebellar granule cellsCerebellar granule neurons were prepared from 7-day-old B6 mice according to a procedure modified from Courtney et al.40 Briefly, the cerebellum was mechanically dissociated and filtered through a 70-μm nylon mesh filter (Small Parts, Miami Lakes, FL, USA). Cells were plated onto a plastic culture plate that was pre-coated with poly--lysine (30 μg/ml; Sigma-Aldrich) and maintained in Dulbecco's modified Eagle's medium Glutamax-I (Invitrogen, Carlsbad, CA, USA) medium containing 10% fetal bovine serum (FBS; Invitrogen), 20 mM KCl, and 100 U/ml penicillin and streptomycin (Invitrogen), and completed with N2 and B27 supplements (Invitrogen). The growth of non-neuronal cells was inhibited by the addition of the antimitotics, uridine, and fluorodeoxyuridine (10 μM; Sigma-Aldrich), for 18 h after seeding. Cells were kept at 37 °C in a humidified atmosphere of 5% CO2 and 95% air.

Primary cultures of mouse cerebellar granule cells

Cerebellar granule neurons were prepared from 7-day-old B6 mice according to a procedure modified from Courtney et al.40 Briefly, the cerebellum was mechanically dissociated and filtered through a 70-μm nylon mesh filter (Small Parts, Miami Lakes, FL, USA). Cells were plated onto a plastic culture plate that was pre-coated with poly--lysine (30 μg/ml; Sigma-Aldrich) and maintained in Dulbecco's modified Eagle's medium Glutamax-I (Invitrogen, Carlsbad, CA, USA) medium containing 10% fetal bovine serum (FBS; Invitrogen), 20 mM KCl, and 100 U/ml penicillin and streptomycin (Invitrogen), and completed with N2 and B27 supplements (Invitrogen). The growth of non-neuronal cells was inhibited by the addition of the antimitotics, uridine, and fluorodeoxyuridine (10 μM; Sigma-Aldrich), for 18 h after seeding. Cells were kept at 37 °C in a humidified atmosphere of 5% CO2 and 95% air.

ImmunofluorescenceCells were fixed for 15 min with 4% formaldehyde in phosphate-buffered saline (PBS) and then permeabilized with cold acetone. Antibodies were then incubated with fixed cells for 4 h at room temperature. Cells were washed three times with PBS followed by incubation with a secondary antibody for 1 h at room temperature. Nuclei were revealed by ProLong Gold antifade reagent with DAPI (Invitrogen). Coverslips were inverted, mounted on slides, and sealed with nail polish. Pictures were taken using fluorescence microscopy.

Immunofluorescence

Cells were fixed for 15 min with 4% formaldehyde in phosphate-buffered saline (PBS) and then permeabilized with cold acetone. Antibodies were then incubated with fixed cells for 4 h at room temperature. Cells were washed three times with PBS followed by incubation with a secondary antibody for 1 h at room temperature. Nuclei were revealed by ProLong Gold antifade reagent with DAPI (Invitrogen). Coverslips were inverted, mounted on slides, and sealed with nail polish. Pictures were taken using fluorescence microscopy.

P19 cell neuronal differentiationMouse embryonal carcinoma P19 cells were used in this study, and RA-induced P19 cell neuronal differentiation was performed with a slight modification of procedures described by Lee et al.41 Briefly, P19 cells were maintained in alpha minimum essential medium (α-MEM; Invitrogen) supplemented with 7.5% bovine serum (BS; Invitrogen) and 2.5% FBS at 37 °C in a humidified atmosphere (5% CO2 and 95% air). To induce differentiation, cells were allowed to aggregate in bacterial-grade Petri dishes at a seeding density of 1 × 105 cells/ml in the presence of 1 μM all-trans-RA (Sigma-Aldrich). After 4 days of aggregation, cells were dissociated into single cells by trypsin-EDTA, and were plated in a poly--lysine-coated tissue culture dish at a density of 1 × 105 cells/cm2 in Neurobasal-A medium (Invitrogen) with a 1 × B27 supplement. Cells were allowed to attach for 24 h, and then were exposed to 10 μM Ara-C 24 h to inhibit proliferation of non-neuronal cells.

P19 cell neuronal differentiation

Mouse embryonal carcinoma P19 cells were used in this study, and RA-induced P19 cell neuronal differentiation was performed with a slight modification of procedures described by Lee et al.41 Briefly, P19 cells were maintained in alpha minimum essential medium (α-MEM; Invitrogen) supplemented with 7.5% bovine serum (BS; Invitrogen) and 2.5% FBS at 37 °C in a humidified atmosphere (5% CO2 and 95% air). To induce differentiation, cells were allowed to aggregate in bacterial-grade Petri dishes at a seeding density of 1 × 105 cells/ml in the presence of 1 μM all-trans-RA (Sigma-Aldrich). After 4 days of aggregation, cells were dissociated into single cells by trypsin-EDTA, and were plated in a poly--lysine-coated tissue culture dish at a density of 1 × 105 cells/cm2 in Neurobasal-A medium (Invitrogen) with a 1 × B27 supplement. Cells were allowed to attach for 24 h, and then were exposed to 10 μM Ara-C 24 h to inhibit proliferation of non-neuronal cells.

Reverse transcription (RT) and quantitative (q) real-time PCR assaysTotal RNA was extracted using the Trizol reagent (Invitrogen) following the manufacturer's recommendations. RT was performed with 1.5 μg of total RNA using M-MLV reverse transcriptase (Invitrogen). A real-time qPCR was performed using the SYBR advantage qPCR premix (Invitrogen). The PCRs were then performed using the following conditions for 40 cycles: 95 °C for 15 s, 60 °C for 15 s, and 72 °C for 20 s. Sequences of the primers used are listed in Supplementary Table 1. Real-time fluorescence monitoring and a melting-curve analysis were performed with LightCycler according to the manufacturer's recommendations (Roche Molecular Diagnostics, Lewes, East Sussex, UK). Negative controls containing no cDNA template were included in each experiment. A melting curve was created at the end of the PCR cycle to confirm that a single product had been amplified. Data were analyzed by LightCycler software vers. 3.5 (Roche Molecular Diagnostics) to determine the threshold cycle (Cp) above the background for each reaction. The relative transcript amount of the target gene, calculated using standard curves of serial cDNA dilutions, was normalized to that of β-actin of the same cDNA.

Reverse transcription (RT) and quantitative (q) real-time PCR assays

Total RNA was extracted using the Trizol reagent (Invitrogen) following the manufacturer's recommendations. RT was performed with 1.5 μg of total RNA using M-MLV reverse transcriptase (Invitrogen). A real-time qPCR was performed using the SYBR advantage qPCR premix (Invitrogen). The PCRs were then performed using the following conditions for 40 cycles: 95 °C for 15 s, 60 °C for 15 s, and 72 °C for 20 s. Sequences of the primers used are listed in Supplementary Table 1. Real-time fluorescence monitoring and a melting-curve analysis were performed with LightCycler according to the manufacturer's recommendations (Roche Molecular Diagnostics, Lewes, East Sussex, UK). Negative controls containing no cDNA template were included in each experiment. A melting curve was created at the end of the PCR cycle to confirm that a single product had been amplified. Data were analyzed by LightCycler software vers. 3.5 (Roche Molecular Diagnostics) to determine the threshold cycle (Cp) above the background for each reaction. The relative transcript amount of the target gene, calculated using standard curves of serial cDNA dilutions, was normalized to that of β-actin of the same cDNA.

Western blot analysisCells were rinsed with ice-cold PBS and lysed in radioimmunoprecipitation assay buffer (150 mM NaCl, 1% NP40, 0.5% Na-deoxycholate, 0.1% sodium dodecylsulfate, and 50 mM Tris-HCl; pH 8.0) with the addition of a protease inhibitor cocktail (Roche Molecular Diagnostics). Cell debris was removed by centrifugation at 12 000 r.p.m. for 15 min at 4 °C, and the supernatant was utilized for a western blot analysis. Protein concentrations were determined using the Bio-Rad dye-binding method (Bio-Rad Laboratories, Hercules, CA, USA) with bovine serum albumin as the standard. Equal amounts of sample were separated by 10% polyacrylamide gel electrophoresis. The resolved proteins were then electroblotted onto Immobilon polyvinylidene difluoride membranes (Millipore, Bedford, MA, USA). Membranes were blocked with 5% nonfat milk and then sequentially incubated with the primary and secondary antibodies. After washing, the blots were processed for visualization using an enhanced chemiluminescence system (Pierce, Rockford, IL, USA). Blots were then exposed to X-ray film to obtain the fluorographic images.

Western blot analysis

Cells were rinsed with ice-cold PBS and lysed in radioimmunoprecipitation assay buffer (150 mM NaCl, 1% NP40, 0.5% Na-deoxycholate, 0.1% sodium dodecylsulfate, and 50 mM Tris-HCl; pH 8.0) with the addition of a protease inhibitor cocktail (Roche Molecular Diagnostics). Cell debris was removed by centrifugation at 12 000 r.p.m. for 15 min at 4 °C, and the supernatant was utilized for a western blot analysis. Protein concentrations were determined using the Bio-Rad dye-binding method (Bio-Rad Laboratories, Hercules, CA, USA) with bovine serum albumin as the standard. Equal amounts of sample were separated by 10% polyacrylamide gel electrophoresis. The resolved proteins were then electroblotted onto Immobilon polyvinylidene difluoride membranes (Millipore, Bedford, MA, USA). Membranes were blocked with 5% nonfat milk and then sequentially incubated with the primary and secondary antibodies. After washing, the blots were processed for visualization using an enhanced chemiluminescence system (Pierce, Rockford, IL, USA). Blots were then exposed to X-ray film to obtain the fluorographic images.

Lentiviral vector production and infectionReplication-deficient lentiviruses were generated by transient transfection of shRNAs, the packaging vector, pCMV-ΔR8.91 (containing the gag, pol, and rev genes), and the envelope plasmid, pMD.G (VSV-G-expressing plasmid), in 293T cells using the Trans IT-LT1 reagent (Mirus Bio, Madison, WI, USA). The shRNAs included Znf179 shRNAs (TRCN0000040979 and TRCN0000040980), which targeted the mouse Znf179 transcript sequences, 5′-GCCGGAAGATCTGTAAGCAAA-3′ (sh_2) and 5′-GATACCGATCTGGGCTATCTA-3′ (sh_3), p35 shRNA (TRCN0000009518), which targeted the mouse p35 transcript sequence, 5′-GACCCACACTATTTCACACAA-3′ (sh_4), and the luciferase control (pLKO.1-shLuc) plasmid constructs were used. To monitor and track the virus titer, the puromycin resistance gene on pLKO.1-based lentiviral vectors was replaced with EGFP. To generate p35 expression vectors, p35 and EGFP-p35 cDNA fragments were inserted into the BamHI and KpnI sites of the sh_2 and shLuc constructs. Beginning at 36 h post-transfection, the viral supernatant was collected every 24 h for 4 days. The pooled supernatant was cleared by low-speed centrifugation followed by filtering through 0.45-μm filters. Viral particles were concentrated by ultracentrifugation at 110 500 g for 2 h. Pellets were resuspended in PBS at 1/500 of the original volume and kept at −70 °C. The titer of viral stocks was determined by flow cytometric analysis of EGFP expression 48 h after infection. In general, viral titers were about 3∼5 × 107 TU/ml. P19 cells were infected twice with the virus (MOI=4) before 2 days and at the days of cell aggregation in 8 μg/ml polybrene.

Lentiviral vector production and infection

Replication-deficient lentiviruses were generated by transient transfection of shRNAs, the packaging vector, pCMV-ΔR8.91 (containing the gag, pol, and rev genes), and the envelope plasmid, pMD.G (VSV-G-expressing plasmid), in 293T cells using the Trans IT-LT1 reagent (Mirus Bio, Madison, WI, USA). The shRNAs included Znf179 shRNAs (TRCN0000040979 and TRCN0000040980), which targeted the mouse Znf179 transcript sequences, 5′-GCCGGAAGATCTGTAAGCAAA-3′ (sh_2) and 5′-GATACCGATCTGGGCTATCTA-3′ (sh_3), p35 shRNA (TRCN0000009518), which targeted the mouse p35 transcript sequence, 5′-GACCCACACTATTTCACACAA-3′ (sh_4), and the luciferase control (pLKO.1-shLuc) plasmid constructs were used. To monitor and track the virus titer, the puromycin resistance gene on pLKO.1-based lentiviral vectors was replaced with EGFP. To generate p35 expression vectors, p35 and EGFP-p35 cDNA fragments were inserted into the BamHI and KpnI sites of the sh_2 and shLuc constructs. Beginning at 36 h post-transfection, the viral supernatant was collected every 24 h for 4 days. The pooled supernatant was cleared by low-speed centrifugation followed by filtering through 0.45-μm filters. Viral particles were concentrated by ultracentrifugation at 110 500 g for 2 h. Pellets were resuspended in PBS at 1/500 of the original volume and kept at −70 °C. The titer of viral stocks was determined by flow cytometric analysis of EGFP expression 48 h after infection. In general, viral titers were about 3∼5 × 107 TU/ml. P19 cells were infected twice with the virus (MOI=4) before 2 days and at the days of cell aggregation in 8 μg/ml polybrene.

Cell viability assayCell viability was determined by adding MTT (Sigma-Aldrich) to the cell cultures at a final concentration of 0.5 mg/ml. After a 1-h incubation at 37 °C, the medium was discarded, and the dark crystals formed were dissolved in DMSO. The absorbency at 550 nm was measured on a micro-enzyme-linked immunosorbent (ELISA) reader.

Cell viability assay

Cell viability was determined by adding MTT (Sigma-Aldrich) to the cell cultures at a final concentration of 0.5 mg/ml. After a 1-h incubation at 37 °C, the medium was discarded, and the dark crystals formed were dissolved in DMSO. The absorbency at 550 nm was measured on a micro-enzyme-linked immunosorbent (ELISA) reader.

DNA microarray and IPATotal RNA was extracted using the Trizol reagent (Invitrogen) following the manufacturer's recommendations, and RNA integrity was assessed using the Experion RNA StdSens Analysis Kit (Bio-Rad Laboratories). Microarray experiments were performed using MouseWG-6 v2.0 BeadChips, and over 48 000 known genes, gene candidates, and splice variants were analyzed (Illumina, San Diego, CA, USA) according to the instructions provided. The cDNA and cRNA were prepared from 500 ng of total RNA using Ambion's Illumina RNA Amplification kit (Ambion, Austin, TX, USA) following their instructions. The cRNA was hybridized to Illumina's MouseWG-6 v2.0 BeadChips, which were scanned on an Illumina BeadStation 500 × . Illumina Beadstudio software was used to assess fluorescent hybridization signals. Before proceeding with the differential gene expression analysis, quantile normalization algorithms, present in the Illumina data analysis software, BeadStudio,42 were applied to the data. To interpret the biological significance of the candidate genes, the IPA software was used (Ingenuity, Redwood, CA, USA). Specifically, the functional analysis tool identified the physiological system development and functions and molecular and cellular functions that were most significant to the data set. A right-tailed Fisher's exact test was used to calculate a P-value, which indicated the probability that each biological function assigned to the data set was due to chance alone.

DNA microarray and IPA

Total RNA was extracted using the Trizol reagent (Invitrogen) following the manufacturer's recommendations, and RNA integrity was assessed using the Experion RNA StdSens Analysis Kit (Bio-Rad Laboratories). Microarray experiments were performed using MouseWG-6 v2.0 BeadChips, and over 48 000 known genes, gene candidates, and splice variants were analyzed (Illumina, San Diego, CA, USA) according to the instructions provided. The cDNA and cRNA were prepared from 500 ng of total RNA using Ambion's Illumina RNA Amplification kit (Ambion, Austin, TX, USA) following their instructions. The cRNA was hybridized to Illumina's MouseWG-6 v2.0 BeadChips, which were scanned on an Illumina BeadStation 500 × . Illumina Beadstudio software was used to assess fluorescent hybridization signals. Before proceeding with the differential gene expression analysis, quantile normalization algorithms, present in the Illumina data analysis software, BeadStudio,42 were applied to the data. To interpret the biological significance of the candidate genes, the IPA software was used (Ingenuity, Redwood, CA, USA). Specifically, the functional analysis tool identified the physiological system development and functions and molecular and cellular functions that were most significant to the data set. A right-tailed Fisher's exact test was used to calculate a P-value, which indicated the probability that each biological function assigned to the data set was due to chance alone.

Cell cycle analysisCells were trypsinized, collected by centrifugation, resuspended, and washed in PBS before fixing in 75% ethanol for 1 h at room temperature. Fixed cells were incubated with 20 μg/ml propidium iodide, 10 mg/ml RNase A, and 0.1% Triton X-100 for 30 min before analysis. The cell cycle was analyzed using a BD FACS Calibur flow cytometer (BD Biosciences, San Jose, CA, USA), and the data were analyzed using WinMDI software (Joseph Trotter, Scripps Research Insitute, La Jolla, CA, USA).

Cell cycle analysis

Cells were trypsinized, collected by centrifugation, resuspended, and washed in PBS before fixing in 75% ethanol for 1 h at room temperature. Fixed cells were incubated with 20 μg/ml propidium iodide, 10 mg/ml RNase A, and 0.1% Triton X-100 for 30 min before analysis. The cell cycle was analyzed using a BD FACS Calibur flow cytometer (BD Biosciences, San Jose, CA, USA), and the data were analyzed using WinMDI software (Joseph Trotter, Scripps Research Insitute, La Jolla, CA, USA).

BrdU incorporationP19 cells, which were aggregated in the presence of 1 μM RA for 4 days, were dissociated and plated in α-MEM (Invitrogen) supplemented with 7.5% BS (Invitrogen) and 2.5% FBS (Invitrogen). BrdU (Sigma-Aldrich), at 40 μM, was added to the media and incubated for 4 h before fixation. An anti-BrdU antibody (Sigma-Aldrich) was used for immunofluorescence.

BrdU incorporation

P19 cells, which were aggregated in the presence of 1 μM RA for 4 days, were dissociated and plated in α-MEM (Invitrogen) supplemented with 7.5% BS (Invitrogen) and 2.5% FBS (Invitrogen). BrdU (Sigma-Aldrich), at 40 μM, was added to the media and incubated for 4 h before fixation. An anti-BrdU antibody (Sigma-Aldrich) was used for immunofluorescence.
