---
pmid: '25959826'
title: Quantitative interaction proteomics of neurodegenerative disease proteins.
authors:
- Hosp F
- Vossfeldt H
- Heinig M
- Vasiljevic D
- Arumughan A
- Wyler E
- Genetic and Environmental Risk for Alzheimer’s Disease GERAD1 Consortium
- Landthaler M
- Hubner N
- Wanker EE
- Lannfelt L
- Ingelsson M
- Lalowski M
- Voigt A
- Selbach M
journal: Cell Rep
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC9014711
doi: 10.1016/j.celrep.2015.04.030
---

# Quantitative interaction proteomics of neurodegenerative disease proteins.
**Authors:** Hosp F, Vossfeldt H, Heinig M, Vasiljevic D, Arumughan A, Wyler E, Genetic and Environmental Risk for Alzheimer’s Disease GERAD1 Consortium, Landthaler M, Hubner N, Wanker EE, Lannfelt L, Ingelsson M, Lalowski M, Voigt A, Selbach M
**Journal:** Cell Rep (2015)
**DOI:** [10.1016/j.celrep.2015.04.030](https://doi.org/10.1016/j.celrep.2015.04.030)
**PMC:** [PMC9014711](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9014711/)

## Abstract

1. Cell Rep. 2015 May 19;11(7):1134-46. doi: 10.1016/j.celrep.2015.04.030. Epub 
2015 May 7.

Quantitative interaction proteomics of neurodegenerative disease proteins.

Hosp F(1), Vossfeldt H(2), Heinig M(3), Vasiljevic D(1), Arumughan A(1), Wyler 
E(1); Genetic and Environmental Risk for Alzheimer’s Disease GERAD1 Consortium; 
Landthaler M(1), Hubner N(1), Wanker EE(1), Lannfelt L(4), Ingelsson M(4), 
Lalowski M(5), Voigt A(2), Selbach M(6).

Collaborators: Harold D, Abraham R, Hollingworth P, Sims R, Gerrish A, Chapman 
J, Russo G, Hamshere M, Pahwa JS, Escott-Price V, Dowzell K, Williams A, Jones 
N, Thomas C, Stretton A, Morgan A, Lovestone S, Powell J, Proitsi P, Lupton MK, 
Brayne C, Rubinsztein DC, Gill M, Lawlor B, Lynch A, Morgan K, Brown K, Passmore 
P, Craig D, McGuinness B, Todd S, Johnston J, Holmes C, Mann D, Smith AD, Love 
S, Kehoe PG, Hardy J, Mead S, Fox N, Rossor M, Collinge J, Maier W, Jessen F, 
Heun R, Schürmann B, Ramirez A, Becker T, Herold C, Lacour A, Drichel D, van den 
Bussche H, Heuser I, Kornhuber J, Wiltfang J, Dichgans M, Frölich L, Hampel H, 
Hüll M, Rujescu D, Goate A, Kauwe JS, Cruchaga C, Nowotny P, Morris JC, Mayo K, 
Livingston G, Bass NJ, Gurling H, McQuillin A, Gwilliam R, Deloukas P, 
Al-Chalabi A, Shaw CE, Singleton AB, Guerreiro R, Mühleisen TW, Nöthen MM, 
Moebus S, Jöckel KH, Klopp N, Wichmann HE, Carrasquillo MM, Pankratz VS, Younkin 
SG, Holmans P, O'Donovan M, J Owen M, Williams J.

Author information:
(1)Max Delbrück Center for Molecular Medicine, Robert-Rössle-Straße 10, 13092 
Berlin, Germany.
(2)University Medical Center, RWTH Aachen, Pauwelsstraße 30, 52074 Aachen, 
Germany.
(3)Max Delbrück Center for Molecular Medicine, Robert-Rössle-Straße 10, 13092 
Berlin, Germany; Max Planck Institute for Molecular Genetics, Ihnestraße 63-73, 
14195 Berlin, Germany.
(4)Uppsala University, Department of Public Health and Geriatrics, Uppsala 
University Hospital, Box, 609, 751 25 Uppsala, Sweden.
(5)Biomedicum Helsinki, Meilahti Clinical Proteomics Core Facility and 
Folkhälsan Research Center, University of Helsinki, PO Box 63, Haartmaninkatu 8, 
Helsinki 00014, Finland.
(6)Max Delbrück Center for Molecular Medicine, Robert-Rössle-Straße 10, 13092 
Berlin, Germany. Electronic address: matthias.selbach@mdc-berlin.de.

Several proteins have been linked to neurodegenerative disorders (NDDs), but 
their molecular function is not completely understood. Here, we used 
quantitative interaction proteomics to identify binding partners of Amyloid beta 
precursor protein (APP) and Presenilin-1 (PSEN1) for Alzheimer's disease (AD), 
Huntingtin (HTT) for Huntington's disease, Parkin (PARK2) for Parkinson's 
disease, and Ataxin-1 (ATXN1) for spinocerebellar ataxia type 1. Our network 
reveals common signatures of protein degradation and misfolding and 
recapitulates known biology. Toxicity modifier screens and comparison to 
genome-wide association studies show that interaction partners are significantly 
linked to disease phenotypes in vivo. Direct comparison of wild-type proteins 
and disease-associated variants identified binders involved in pathogenesis, 
highlighting the value of differential interactome mapping. Finally, we show 
that the mitochondrial protein LRPPRC interacts preferentially with an 
early-onset AD variant of APP. This interaction appears to induce mitochondrial 
dysfunction, which is an early phenotype of AD.

Copyright © 2015 The Authors. Published by Elsevier Inc. All rights reserved.

DOI: 10.1016/j.celrep.2015.04.030
PMCID: PMC9014711
PMID: 25959826 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONThe functional characterization of disease-associated proteins is a major challenge in the post-genomic era. Since proteins typically exert their function by binding other proteins, systematic mapping of protein-protein interactions (PPIs) can help with the understanding of protein function. The yeast two-hybrid (Y2H) system generated large-scale human PPI networks (Rual et al., 2005; Stelzl et al., 2005). Similarly, affinity purification and mass spectrometry (AP-MS) has been employed to create interactome maps for model organisms and humans (Ewing et al., 2007; Gingras et al., 2007; Guruharsha et al., 2011; Malovannaya et al., 2011). Although such interaction networks are still incomplete, they are beginning to provide valuable functional insights for human diseases (Barabási et al., 2011; Ideker and Sharan, 2008; Vidal et al., 2011).Since mutations might alter PPIs, comparing the interactions of wild-type proteins and their disease-linked variants might be particularly informative (Schuster-Böckler and Bateman, 2008; Zhong et al., 2009). This information cannot be extracted from most available interaction datasets. The Y2H assay is at best semiquantitative and unable to detect small alterations in the affinity of PPIs (Estojak et al., 1995; Zhong et al., 2009). Similarly, most AP-MS studies only used semiquantitative methods like spectral counting or were performed entirely without quantification. Spectral counting yields unreliable data, especially when the interaction partner is of low abundance and/or the number of samples is small (Gingras and Raught, 2012; Rinner et al., 2007).To overcome these problems, we sought to investigate PPIs of disease-associated proteins in a quantitative manner. Quantitative affinity purification and mass spectrometry (qAP-MS) can accurately distinguish between specific interaction partners and non-specific contaminants and quantify changes in PPIs upon perturbation (Paul et al., 2011; Vermeulen et al., 2008; Wepf et al., 2009). We reasoned that qAP-MS should be particularly well suited to identify interaction partners of disease-associated proteins. Therefore, we employed this technology to assess interaction partners of proteins involved in neurodegenerative diseases (NDDs). We focused on five well-known disease proteins involved in Alzheimer’s disease (AD), Huntington’s disease (HD), Parkinson’s disease (PD), and spinocerebellar ataxia type 1 (SCA1). These four NDDs are characterized by the accumulation of protein aggregates in the brain and the progressive loss of neurons (Ross and Poirier, 2004). HD and SCA1 are caused by expansion of CAG repeats in the huntingtin and ataxin-1 genes, respectively, which gives rise to proteins with abnormally long polyglutamine (polyQ) tracts (Chung et al., 1993; Rubinsztein et al., 1996). Although many risk factors contribute to prevalent cases of AD and PD, rare familial forms have been identified that are caused by mutations in few genes like APP, PSEN1/PSEN2, or PARK2 (Lin and Farrer, 2014; Tanzi and Bertram, 2005).Our quantitative proteomics data provide a comparative protein interaction network across several NDDs, revealing both common modulators of several distinct disorders and disease-specific pathogenic properties. We validated our interaction data by co-immunoprecipitation (coIP) experiments and established links to disease phenotypes in animal models and humans. Our screen identified leucine-rich pentatricopeptide repeat motif containing protein (LRPPRC) as a preferential interactor of Amyloid beta precursor protein (APP) carrying the so-called Swedish mutation (APPsw), which causes early-onset AD. LRPPRC is a key regulator of mitochondrial gene expression. Thus, the interaction might contribute to mitochondrial dysfunction in AD.

INTRODUCTION

The functional characterization of disease-associated proteins is a major challenge in the post-genomic era. Since proteins typically exert their function by binding other proteins, systematic mapping of protein-protein interactions (PPIs) can help with the understanding of protein function. The yeast two-hybrid (Y2H) system generated large-scale human PPI networks (Rual et al., 2005; Stelzl et al., 2005). Similarly, affinity purification and mass spectrometry (AP-MS) has been employed to create interactome maps for model organisms and humans (Ewing et al., 2007; Gingras et al., 2007; Guruharsha et al., 2011; Malovannaya et al., 2011). Although such interaction networks are still incomplete, they are beginning to provide valuable functional insights for human diseases (Barabási et al., 2011; Ideker and Sharan, 2008; Vidal et al., 2011).

Since mutations might alter PPIs, comparing the interactions of wild-type proteins and their disease-linked variants might be particularly informative (Schuster-Böckler and Bateman, 2008; Zhong et al., 2009). This information cannot be extracted from most available interaction datasets. The Y2H assay is at best semiquantitative and unable to detect small alterations in the affinity of PPIs (Estojak et al., 1995; Zhong et al., 2009). Similarly, most AP-MS studies only used semiquantitative methods like spectral counting or were performed entirely without quantification. Spectral counting yields unreliable data, especially when the interaction partner is of low abundance and/or the number of samples is small (Gingras and Raught, 2012; Rinner et al., 2007).

To overcome these problems, we sought to investigate PPIs of disease-associated proteins in a quantitative manner. Quantitative affinity purification and mass spectrometry (qAP-MS) can accurately distinguish between specific interaction partners and non-specific contaminants and quantify changes in PPIs upon perturbation (Paul et al., 2011; Vermeulen et al., 2008; Wepf et al., 2009). We reasoned that qAP-MS should be particularly well suited to identify interaction partners of disease-associated proteins. Therefore, we employed this technology to assess interaction partners of proteins involved in neurodegenerative diseases (NDDs). We focused on five well-known disease proteins involved in Alzheimer’s disease (AD), Huntington’s disease (HD), Parkinson’s disease (PD), and spinocerebellar ataxia type 1 (SCA1). These four NDDs are characterized by the accumulation of protein aggregates in the brain and the progressive loss of neurons (Ross and Poirier, 2004). HD and SCA1 are caused by expansion of CAG repeats in the huntingtin and ataxin-1 genes, respectively, which gives rise to proteins with abnormally long polyglutamine (polyQ) tracts (Chung et al., 1993; Rubinsztein et al., 1996). Although many risk factors contribute to prevalent cases of AD and PD, rare familial forms have been identified that are caused by mutations in few genes like APP, PSEN1/PSEN2, or PARK2 (Lin and Farrer, 2014; Tanzi and Bertram, 2005).

Our quantitative proteomics data provide a comparative protein interaction network across several NDDs, revealing both common modulators of several distinct disorders and disease-specific pathogenic properties. We validated our interaction data by co-immunoprecipitation (coIP) experiments and established links to disease phenotypes in animal models and humans. Our screen identified leucine-rich pentatricopeptide repeat motif containing protein (LRPPRC) as a preferential interactor of Amyloid beta precursor protein (APP) carrying the so-called Swedish mutation (APPsw), which causes early-onset AD. LRPPRC is a key regulator of mitochondrial gene expression. Thus, the interaction might contribute to mitochondrial dysfunction in AD.

RESULTSQuantitative Interaction Screen for NDD ProteinsA fundamental challenge in biochemical screens for PPIs is the trade-off between sensitivity and specificity. To address this problem, we employed qAP-MS, in which the abundance of proteins co-purifying with the bait is compared with a control pull-down (Figure 1A). As an example, we show results from experiments with Ataxin-1 (ATXN1). We first investigated proteins specifically co-purifying with wild-type ATXN1 (normal polyQ track length, ATXN1-Q30) relative to the empty vector control in HEK293T cells. We identified 731 proteins, most of which had a heavy to light (H/L) stable isotope labeling by amino acids in cell culture (SILAC) ratio around 1 in both the forward experiment and the corresponding label swap (Figure 1B). These proteins were therefore considered unspecific contaminants (Figure S1A). ATXN1-Q30 itself had a high H/L ratio in the forward experiment and an inverse ratio in the reverse experiment, indicating efficient enrichment of the bait protein. In addition, several 14–3-3 proteins specifically co-purified with ATXN1-Q30 in both experiments (Figure S1B). 14–3-3 proteins are known interaction partners of wild-type and mutant ATXN1 with a critical role in ATXN1-mediated neurodegeneration in vivo (Chen et al., 2003). 14–3-3 proteins bind to ATXN1 phosphorylated on serine residue 776 by AKT (Chen et al., 2003; Emamian et al., 2003), and our mass spectrometry data showed that serine 776 is indeed phosphorylated (Figure S2). Another identified interaction partner was the transcriptional repressor Capicua protein homolog (CIC). ATXN1 is known to interact with CIC in the mouse brain and to alter its repressor activity (Lam et al., 2006). Collectively, these results indicate that our approach can identify functionally relevant in vivo interaction partners, including phosphorylation-dependent ones. Since AP-MS can enrich entire complexes, not all of these interaction partners are necessarily direct binders of the bait protein.Overall, we identified ~50 potential interaction partners of wild-type ATXN1. Several of these proteins are involved in splicing (MBNL1, SF1, U2AF1, and U2AF2) and transcriptional regulation (CIC, HDAC3, TBL1XR1, and MED27), consistent with the nuclear function of ATXN1 (Lam et al., 2006). Other proteins are involved in the ubiquitin proteasome system (PSMA4, PSMB1, SKP1, and UBC), consistent with previous data (Choi et al., 2007). Performing the same experiment with polyQ-extended ATXN1, we identified a similar number of potential interaction partners (Figures 1C and S1C). Of note, ~80% of the proteins were shared between the wild-type and disease-associated forms (Table S1). This result suggests that (1) polyQ expansion does not dramatically change the interaction partners of ATXN1 (see below) and (2) the reproducibility of the data is high.Interaction Network of NDD ProteinsIn total, we carried out qAP-MS screens for five different disease proteins involved in four NDDs (AD, HD, PD, and SCA1). For each protein, we used both the wild-type form and disease-associated variants, resulting in 12 different bait proteins (Figure S3A). Altogether, we performed 72 pull-downs of tagged bait proteins with corresponding empty vector controls. Every bait protein was tested in at least four biological replicates (i.e., at least two forward and two reverse experiments). Overall, the reproducibility between replicates was high (mean correlation coefficient R = 0.83; Figure S3B). To further increase robustness, we averaged over all forward and reverse experiments for each bait protein. Proteins were considered specific interaction partners when they (1) were enriched at least 2-fold on average and (2) showed inverted SILAC ratios in the reverse experiments (see the Supplemental Experimental Procedures for details). At this stage, we did not distinguish between wild-type forms and disease-associated variants but instead combined the data at the level of the disease protein (Table S1). In total, 373 proteins passed our cutoff criteria and were classified as interactors. Ten proteins were detected as shared interaction partners in at least three of the four diseases investigated (Figure 1D) and are enriched in the Gene Ontology (GO) term proteolysis (p < 3.5 × 10−4, Benjamini-Hochberg correction). This result is consistent with the idea that the ubiquitin-proteasome system and molecular chaperones are modulators of neurodegeneration in several NDDs (Williams and Paulson, 2008). Interestingly, however, most of the interactions were unique for a single disease. This was even the case for HD and SCA1, although both diseases are caused by proteins with elongated polyQ tracts. Hence, our data highlight considerable differences in disease pathogenesis and/or the native cellular function of the respective disease proteins.Our NDD network contains several known interactions of biological relevance. For example, we observed binding of Presenilin-1 (PSEN1) to the N-oligosaccharyl transferase complex (OSC) (Lee et al., 2010) and the interaction of VCP with both Huntingtin (HTT) and Parkin (PARK2) (Hirabayashi et al., 2001; Imai et al., 2003). We also found that enriched GO terms of interaction partners are largely consistent with the respective bait protein in terms of localization and function (Figure 1E). Thus, our unbiased interaction network recapitulates results from multiple studies, strongly supporting the functional relevance of our data. On the other hand, we failed to identify several well-described interactions such as binding of APP and PSEN1 to the gamma-secretase complex. This may be due to the transient nature of these interactions or other factors. For example, it has been reported previously that the mature gamma-secretase complex can only be purified when pull-downs are combined with lectin-based affinity chromatography (Kimberly et al., 2002; Yang et al., 2002).Since most of our interactions are novel, we sought to validate our data by independent experimental approaches. First, we tested if western blotting could validate the interactions. In 18 out of 18 tested cases (selection based on antibody availability), this confirmed our results (Figure S4A). Given that tagging, overexpression, and/or using HEK293T cells could also lead to false-positive interactions, we also performed coIP experiments with the endogenous proteins in neuroblastoma cells. In nine out of nine tested cases, these experiments validated the qAP-MS results (Figure S4B). Collectively, these results indicate that our qAP-MS screen detects PPIs with high specificity.Interaction Partners of ATXN1 Are Relevant for Neurodegeneration In VivoNext, we sought to investigate the link between identified PPIs and NDD phenotypes in vivo. To this end, we employed a well-established Drosophila melanogaster model for NDD to test the relevance of identified ATXN1 interaction partners in vivo (Fernandez-Funez et al., 2000). As expected, flies expressing human ATXN1-Q82 in photoreceptor cells showed the characteristic, so-called rough-eye phenotype (REP) indicative of neurodegeneration (Figures 2A–2C). We then asked if proteins, which we identified as ATXN1-Q82 interaction partners, have an impact on the REP by knocking them down in a tissue-specific manner with small hairpin RNAs (shRNAs) (Dietzl et al., 2007). Of the 49 identified interaction partners of ATXN1-Q82, 24 had direct fly orthologs, and 14 of those were available as non-sterile shRNA strains. Silencing two of these genes affected eye morphology alone, and the strains were therefore excluded. The remaining 12 strains were crossed with ATXN1-Q82 transgenic flies to generate F1 flies that express ATXN1-Q82 and the respective shRNA under the same eye-specific promoter. Strains were classified according to the impact of the shRNA on the REP (from 0 = no impact to +++ = lethal). Intriguingly, 9 of the 12 lines exhibited a strong enhancement of the REP (++ or +++, Figures 2F and 2G; Table S2; Figure S5), including necrotic spots indicative of neurodegeneration from subjacent photoreceptor neurons (purple coloring, Figures 2F and 2G). In some cases, no viable offspring was obtained, suggesting a strong enhancement of the neurodegeneration phenotype (Figure 2G). The high overlap between ATXN1 interaction partners and genetic modifiers of ATXN1-mediated neurodegeneration strongly suggests that our data are functionally relevant for NDD phenotypes. The 75% hit rate is considerably higher than the typical hit rate in unbiased genetic screens (1%–4%) (Fernandez-Funez et al., 2000; Kaltenbach et al., 2007), which makes the overlap highly significant (p < 3.4 × 10−10, hypergeometric test with an assumed hit rate of 5%). We also tested all 12 shRNA in another Drosophila model expressing human tau protein (GMR_Tau[R406W]), which exhibit a similar REP (Chatterjee et al., 2009). None of the 12 tested shRNA lines showed an apparent REP change in this model. Thus, the observed REP-modifying effects are specific for the ATXN1-Q82 disease model (Table S2). Collectively, these experiments show that our interaction partners are specifically relevant for ATXN1-Q82-mediated neurodegeneration in vivo.Interaction Partners of APP and PSEN1 Are Linked to Human Disease PhenotypesTo investigate whether our identified interaction partners are also linked to disease phenotypes in humans, we compared our interaction data for APP and PSEN1 with a genome-wide association study (GWAS) for AD (Li et al., 2008). We used gene set enrichment analysis of SNP data (Heinig et al., 2010; Holden et al., 2008) to test if genes encoding APP and PSEN1 interaction partners are also found as candidates in the AD GWAS data (Figure 3A; see Experimental Procedures for details). We found that SNPs associated with interaction partners of wild-type and mutant APP and mutant PSEN1 had significantly lower p values than the global distribution of SNPs (Figures 3B and 3C; Table S3; one-sided Wilcoxon-Mann-Whitney test). Thus, the interactome of these proteins is significantly associated with disease, even though no single interaction partner alone showed genome-wide significance in this GWAS. To validate these findings, we went on and replicated this analysis with an independent GWAS dataset based on a larger cohort (Harold et al., 2009). We were able to replicate our findings for wild-type and mutant APP (Figures 3B and 3C; Table S3). In summary, the comparison with GWAS data suggests that our interaction maps for APP and PSEN1 are functionally relevant for AD. This corroborates the view that PPI data can help linking disease genes and phenotypes (Lundby et al., 2014; Rossin et al., 2011).Preferential Interactions of Wild-Type Proteins and Disease-Associated VariantsNext, we asked if our approach could also identify differential protein binding behavior between the wild-type form of a protein and its disease-associated variant in the same pull-down experiment. To this end, we directly compared interaction partners of wild-type proteins and their disease-associated variants by transfecting HEK293T cells with either the myc-tagged wild-type bait proteins or their disease-associated variants (Figures 4A and S3A). Proteins were considered preferential binders when they were enriched at least 1.4-fold on average and showed an inverted ratio in the crossover experiment (Figure S6; Experimental Procedures). While most proteins had abundance ratios close to one, we found that several of our previously identified interactors indeed showed a preferential binding behavior for one of the two offered bait protein variants (Figure 4B). In total, we identified 125 proteins with preferential binding behavior with our qAP-MS approach (Table S1), including several already known differential interactors. For instance, we observed preferential binding of the Capicua homolog CIC toward the ATXN1-Q30 wild-type when compared to the ATXN1-Q82 mutant variant, as previously reported (Lim et al., 2008). The N-Cor/SMRT complex member HDAC3 showed the same behavior, consistent with the finding that ATXN1 loss of function contributes to transcriptional dysregulation (Crespo-Barreto et al., 2010). We also observed that two members (DDOST and RPN1) of the OST bound preferentially to wild-type PSEN1 compared to PSEN1-A431E. This interaction has recently been shown to mediate glycosylation of the v-ATPase required for lysosomal targeting, proteolysis, and autophagy, although it was not known before that it is affected by the disease-associated mutation (Lee et al., 2010).The APP Swedish Mutant K670N/M671L Binds LRPPRC and Impairs Mitochondrial FunctionTo further validate our data, we selected one differential binding partner for follow-up experiments. We focused on the interaction between APP and LRPPRC. LRPPRC is a conserved nuclear gene that encodes an RNA-binding protein that is imported into mitochondria (Sterky et al., 2010). In the mitochondrial matrix, LRPPRC post-transcriptionally regulates the expression of mitochondrial genes (Gohil et al., 2010; Ruzzenente et al., 2012; Sasarman et al., 2010). LRPPRC is therefore an essential regulator of the mitochondrial respiratory chain. Intriguingly, mutations in LRPPRC cause Leigh syndrome of the French-Canadian type (LSFC), a NDD characterized by complex IV deficiency (Mootha et al., 2003). Thus, LRPPRC is itself an NDD-associated gene.Our original screen identified LRPPRC as an interaction partner of both wild-type APP and APP-K670N/M671L (Table S1). In the direct comparison, LRPPRC co-purified preferentially with the disease-associated form (Figure 5A; Table S1). The K670N/M671L variant of APP is the so-called Swedish mutation (APPsw), which causes excessive production of Aβ and leads to early-onset AD with highly elevated levels of oxidative stress (Marques et al., 2003). Given that it is well established that mitochondrial function is impaired in AD (Querfurth and LaFerla, 2010), we hypothesized that the APP/LRPPRC interaction might contribute to mitochondrial dysfunction.First, we used co-immunoprecipitation to confirm that the amount of LRPPRC that co-purified with APP was slightly higher for the Swedish variant (Figure 5B). Conversely, when we immunoprecipitated LRPPRC, we detected more APPsw than wild-type APP in the precipitate. Thus, the preferential binding observed by qAP-MS could be validated by reciprocal coIPs. We also found that endogenous (i.e., wild-type) APP and LRPPRC interact in neuroblastoma cells (Figure S4B). Second, in order to assess the differential binding behavior with an independent method, we used the proximity ligation assay (PLA). PLA detects endogenous protein complexes in situ using antibody-linked oligonucleotides (Söderberg et al., 2006). We found that endogenous APP and LRPPRC co-localized in HEK293T cells (Figure 6C, upper panels). Moreover, transfecting APPsw, but not wild-type APP, significantly increased the signal compared to the empty vector control (Figures 5C, lower panels, and 5D). We also found that a small interfering RNA (siRNA) against LRPPRC significantly reduced the signal, which validates the specificity of the assay. PLA experiments also showed co-localization of endogenous APP and LRPPRC in neuroblastoma cells (Figure S7). Next, to investigate the relationship between LRPPRC and APP in the human brain, we performed immunohistochemistry in postmortem cortical brain slices of a patient with early-onset AD caused by APPsw and age-matched controls (Figure 5E). We found that APP and LRPPRC are co-expressed in cortical neurons of healthy controls with an overall similar staining pattern, although these data cannot be used to infer co-localization. Importantly, no LRPPRC staining was observed in the Aβ-positive amyloid plaques in AD patients. Thus, the protein does not appear to interact with Aβ in the extracellular aggregates in affected brain regions. In summary, these data confirm the specific interaction between LRPPRC and APP and the preferential interaction with the Swedish mutant. The observation that LRPPRC is not a component of amyloid plaques in AD patients suggests that the interaction occurs with intracellular APP and not with extracellular Aβ.Mature LRPPRC is considered to be exclusively located in the mitochondrial matrix (Sterky et al., 2010). While APP is mainly thought to be associated with the plasma membrane and the ER, it is also recruited to mitochondria via a cryptic targeting sequence (Devi and Anandatheerthavarada, 2010; Devi et al., 2006; Keil et al., 2004; Lin and Beal, 2006; Park et al., 2006). Mitochondrial APP is not completely imported into the organelle but remains associated with mitochondrial translocases such as TOMM40. Consistently, we also identified TOMM40 and other mitochondrial membrane proteins as APP interactors (Figure 1E; Table S1). The topology of APP in mitochondria is such that its N terminus is located in the matrix while the C terminus faces the cytoplasm (Anandatheerthavarada et al., 2003; Devi et al., 2006). Given the topology of mitochondria-associated APP, the interaction with LRPPRC should occur via the matrix-exposed N terminus. To investigate this possibility, we tested the interaction of LRPPRC with an N-terminal truncation (Figure 5F). LRPPRC co-purified with full-length APP (APP770), but not with APPΔN. These findings suggest that LRPPRC binds to the mitochondrial matrix-exposed N terminus of APP.Given that LRPPRC is a central regulator of mitochondrial gene expression, we asked how APPsw affects the proteome. We transiently transfected differentially SILAC-labeled cells with APP or APPsw for 48 hr. Quantitative proteomics revealed that APPsw markedly (~3-fold) downregulated the cellular levels of LRPPRC itself and its binding partner, SLIRP (Figure 6A). It is remarkable that these proteins were among the most strongly regulated ones in our unbiased experiment. Additionally, several members of the cytochrome c oxidase or the NADH dehydrogenase complexes were downregulated (Figure 6A). Thus, APPsw mediates downregulation of LRPPRC and its downstream targets. We speculated that the reduced expression of respiratory chain components may be due to the downregulation of LRPPRC. Therefore, we tested whether overexpressing LRPPRC together with APPsw might rescue its impact on mitochondria. Indeed, co-expression of LRPPRC together with APPsw resulted in almost-APPwt expression levels of LRPPRC/SLIRP and reduced the impact on the downstream targets (Figure 6B). To further characterize the relationship of APP, LRPPRC, and expression of respiratory chain components, we analyzed mRNA levels by qRT-PCR (Figure 6C). These data showed that APPsw significantly downregulates mRNA levels of LRPPRC and the respiratory chain member COX1, a primary downstream target of LRPPRC. In summary, these data show that APPsw downregulates expression of LRPPRC and respiratory chain genes, and that the impact of APPsw on respiratory chain proteins can partially be rescued by LRPPRC overexpression.Finally, in order to test whether these results are relevant for mitochondrial function, we transiently transfected cells with either APP wild-type or APPsw constructs and co-transfected either LRPPRC or a siRNA directed against LRPPRC. We measured aconitase activity as an enzymatic readout for oxidative stress and overall mitochondrial function (Gardner, 2002). Co-expression of LRPPRC increased the aconitase activity in all conditions, while LRPPRC knockdown considerably reduced the enzymatic activity (Figure 6D). This is consistent with the essential role of LRPPRC for mitochondrial function (Ruzzenente et al., 2012). Cells expressing APPsw showed a significant (p < 0.005) decrease in aconitase activity when compared to APP wild-type-expressing cells. Again, LRPPRC co-expression partially restored aconitase activity to almost wild-type levels. These findings suggest that LRPPRC levels might play an important role in APPsw-induced stress phenotypes.

Quantitative Interaction Screen for NDD ProteinsA fundamental challenge in biochemical screens for PPIs is the trade-off between sensitivity and specificity. To address this problem, we employed qAP-MS, in which the abundance of proteins co-purifying with the bait is compared with a control pull-down (Figure 1A). As an example, we show results from experiments with Ataxin-1 (ATXN1). We first investigated proteins specifically co-purifying with wild-type ATXN1 (normal polyQ track length, ATXN1-Q30) relative to the empty vector control in HEK293T cells. We identified 731 proteins, most of which had a heavy to light (H/L) stable isotope labeling by amino acids in cell culture (SILAC) ratio around 1 in both the forward experiment and the corresponding label swap (Figure 1B). These proteins were therefore considered unspecific contaminants (Figure S1A). ATXN1-Q30 itself had a high H/L ratio in the forward experiment and an inverse ratio in the reverse experiment, indicating efficient enrichment of the bait protein. In addition, several 14–3-3 proteins specifically co-purified with ATXN1-Q30 in both experiments (Figure S1B). 14–3-3 proteins are known interaction partners of wild-type and mutant ATXN1 with a critical role in ATXN1-mediated neurodegeneration in vivo (Chen et al., 2003). 14–3-3 proteins bind to ATXN1 phosphorylated on serine residue 776 by AKT (Chen et al., 2003; Emamian et al., 2003), and our mass spectrometry data showed that serine 776 is indeed phosphorylated (Figure S2). Another identified interaction partner was the transcriptional repressor Capicua protein homolog (CIC). ATXN1 is known to interact with CIC in the mouse brain and to alter its repressor activity (Lam et al., 2006). Collectively, these results indicate that our approach can identify functionally relevant in vivo interaction partners, including phosphorylation-dependent ones. Since AP-MS can enrich entire complexes, not all of these interaction partners are necessarily direct binders of the bait protein.Overall, we identified ~50 potential interaction partners of wild-type ATXN1. Several of these proteins are involved in splicing (MBNL1, SF1, U2AF1, and U2AF2) and transcriptional regulation (CIC, HDAC3, TBL1XR1, and MED27), consistent with the nuclear function of ATXN1 (Lam et al., 2006). Other proteins are involved in the ubiquitin proteasome system (PSMA4, PSMB1, SKP1, and UBC), consistent with previous data (Choi et al., 2007). Performing the same experiment with polyQ-extended ATXN1, we identified a similar number of potential interaction partners (Figures 1C and S1C). Of note, ~80% of the proteins were shared between the wild-type and disease-associated forms (Table S1). This result suggests that (1) polyQ expansion does not dramatically change the interaction partners of ATXN1 (see below) and (2) the reproducibility of the data is high.

Quantitative Interaction Screen for NDD Proteins

A fundamental challenge in biochemical screens for PPIs is the trade-off between sensitivity and specificity. To address this problem, we employed qAP-MS, in which the abundance of proteins co-purifying with the bait is compared with a control pull-down (Figure 1A). As an example, we show results from experiments with Ataxin-1 (ATXN1). We first investigated proteins specifically co-purifying with wild-type ATXN1 (normal polyQ track length, ATXN1-Q30) relative to the empty vector control in HEK293T cells. We identified 731 proteins, most of which had a heavy to light (H/L) stable isotope labeling by amino acids in cell culture (SILAC) ratio around 1 in both the forward experiment and the corresponding label swap (Figure 1B). These proteins were therefore considered unspecific contaminants (Figure S1A). ATXN1-Q30 itself had a high H/L ratio in the forward experiment and an inverse ratio in the reverse experiment, indicating efficient enrichment of the bait protein. In addition, several 14–3-3 proteins specifically co-purified with ATXN1-Q30 in both experiments (Figure S1B). 14–3-3 proteins are known interaction partners of wild-type and mutant ATXN1 with a critical role in ATXN1-mediated neurodegeneration in vivo (Chen et al., 2003). 14–3-3 proteins bind to ATXN1 phosphorylated on serine residue 776 by AKT (Chen et al., 2003; Emamian et al., 2003), and our mass spectrometry data showed that serine 776 is indeed phosphorylated (Figure S2). Another identified interaction partner was the transcriptional repressor Capicua protein homolog (CIC). ATXN1 is known to interact with CIC in the mouse brain and to alter its repressor activity (Lam et al., 2006). Collectively, these results indicate that our approach can identify functionally relevant in vivo interaction partners, including phosphorylation-dependent ones. Since AP-MS can enrich entire complexes, not all of these interaction partners are necessarily direct binders of the bait protein.

Overall, we identified ~50 potential interaction partners of wild-type ATXN1. Several of these proteins are involved in splicing (MBNL1, SF1, U2AF1, and U2AF2) and transcriptional regulation (CIC, HDAC3, TBL1XR1, and MED27), consistent with the nuclear function of ATXN1 (Lam et al., 2006). Other proteins are involved in the ubiquitin proteasome system (PSMA4, PSMB1, SKP1, and UBC), consistent with previous data (Choi et al., 2007). Performing the same experiment with polyQ-extended ATXN1, we identified a similar number of potential interaction partners (Figures 1C and S1C). Of note, ~80% of the proteins were shared between the wild-type and disease-associated forms (Table S1). This result suggests that (1) polyQ expansion does not dramatically change the interaction partners of ATXN1 (see below) and (2) the reproducibility of the data is high.

Interaction Network of NDD ProteinsIn total, we carried out qAP-MS screens for five different disease proteins involved in four NDDs (AD, HD, PD, and SCA1). For each protein, we used both the wild-type form and disease-associated variants, resulting in 12 different bait proteins (Figure S3A). Altogether, we performed 72 pull-downs of tagged bait proteins with corresponding empty vector controls. Every bait protein was tested in at least four biological replicates (i.e., at least two forward and two reverse experiments). Overall, the reproducibility between replicates was high (mean correlation coefficient R = 0.83; Figure S3B). To further increase robustness, we averaged over all forward and reverse experiments for each bait protein. Proteins were considered specific interaction partners when they (1) were enriched at least 2-fold on average and (2) showed inverted SILAC ratios in the reverse experiments (see the Supplemental Experimental Procedures for details). At this stage, we did not distinguish between wild-type forms and disease-associated variants but instead combined the data at the level of the disease protein (Table S1). In total, 373 proteins passed our cutoff criteria and were classified as interactors. Ten proteins were detected as shared interaction partners in at least three of the four diseases investigated (Figure 1D) and are enriched in the Gene Ontology (GO) term proteolysis (p < 3.5 × 10−4, Benjamini-Hochberg correction). This result is consistent with the idea that the ubiquitin-proteasome system and molecular chaperones are modulators of neurodegeneration in several NDDs (Williams and Paulson, 2008). Interestingly, however, most of the interactions were unique for a single disease. This was even the case for HD and SCA1, although both diseases are caused by proteins with elongated polyQ tracts. Hence, our data highlight considerable differences in disease pathogenesis and/or the native cellular function of the respective disease proteins.Our NDD network contains several known interactions of biological relevance. For example, we observed binding of Presenilin-1 (PSEN1) to the N-oligosaccharyl transferase complex (OSC) (Lee et al., 2010) and the interaction of VCP with both Huntingtin (HTT) and Parkin (PARK2) (Hirabayashi et al., 2001; Imai et al., 2003). We also found that enriched GO terms of interaction partners are largely consistent with the respective bait protein in terms of localization and function (Figure 1E). Thus, our unbiased interaction network recapitulates results from multiple studies, strongly supporting the functional relevance of our data. On the other hand, we failed to identify several well-described interactions such as binding of APP and PSEN1 to the gamma-secretase complex. This may be due to the transient nature of these interactions or other factors. For example, it has been reported previously that the mature gamma-secretase complex can only be purified when pull-downs are combined with lectin-based affinity chromatography (Kimberly et al., 2002; Yang et al., 2002).Since most of our interactions are novel, we sought to validate our data by independent experimental approaches. First, we tested if western blotting could validate the interactions. In 18 out of 18 tested cases (selection based on antibody availability), this confirmed our results (Figure S4A). Given that tagging, overexpression, and/or using HEK293T cells could also lead to false-positive interactions, we also performed coIP experiments with the endogenous proteins in neuroblastoma cells. In nine out of nine tested cases, these experiments validated the qAP-MS results (Figure S4B). Collectively, these results indicate that our qAP-MS screen detects PPIs with high specificity.

Interaction Network of NDD Proteins

In total, we carried out qAP-MS screens for five different disease proteins involved in four NDDs (AD, HD, PD, and SCA1). For each protein, we used both the wild-type form and disease-associated variants, resulting in 12 different bait proteins (Figure S3A). Altogether, we performed 72 pull-downs of tagged bait proteins with corresponding empty vector controls. Every bait protein was tested in at least four biological replicates (i.e., at least two forward and two reverse experiments). Overall, the reproducibility between replicates was high (mean correlation coefficient R = 0.83; Figure S3B). To further increase robustness, we averaged over all forward and reverse experiments for each bait protein. Proteins were considered specific interaction partners when they (1) were enriched at least 2-fold on average and (2) showed inverted SILAC ratios in the reverse experiments (see the Supplemental Experimental Procedures for details). At this stage, we did not distinguish between wild-type forms and disease-associated variants but instead combined the data at the level of the disease protein (Table S1). In total, 373 proteins passed our cutoff criteria and were classified as interactors. Ten proteins were detected as shared interaction partners in at least three of the four diseases investigated (Figure 1D) and are enriched in the Gene Ontology (GO) term proteolysis (p < 3.5 × 10−4, Benjamini-Hochberg correction). This result is consistent with the idea that the ubiquitin-proteasome system and molecular chaperones are modulators of neurodegeneration in several NDDs (Williams and Paulson, 2008). Interestingly, however, most of the interactions were unique for a single disease. This was even the case for HD and SCA1, although both diseases are caused by proteins with elongated polyQ tracts. Hence, our data highlight considerable differences in disease pathogenesis and/or the native cellular function of the respective disease proteins.

Our NDD network contains several known interactions of biological relevance. For example, we observed binding of Presenilin-1 (PSEN1) to the N-oligosaccharyl transferase complex (OSC) (Lee et al., 2010) and the interaction of VCP with both Huntingtin (HTT) and Parkin (PARK2) (Hirabayashi et al., 2001; Imai et al., 2003). We also found that enriched GO terms of interaction partners are largely consistent with the respective bait protein in terms of localization and function (Figure 1E). Thus, our unbiased interaction network recapitulates results from multiple studies, strongly supporting the functional relevance of our data. On the other hand, we failed to identify several well-described interactions such as binding of APP and PSEN1 to the gamma-secretase complex. This may be due to the transient nature of these interactions or other factors. For example, it has been reported previously that the mature gamma-secretase complex can only be purified when pull-downs are combined with lectin-based affinity chromatography (Kimberly et al., 2002; Yang et al., 2002).

Since most of our interactions are novel, we sought to validate our data by independent experimental approaches. First, we tested if western blotting could validate the interactions. In 18 out of 18 tested cases (selection based on antibody availability), this confirmed our results (Figure S4A). Given that tagging, overexpression, and/or using HEK293T cells could also lead to false-positive interactions, we also performed coIP experiments with the endogenous proteins in neuroblastoma cells. In nine out of nine tested cases, these experiments validated the qAP-MS results (Figure S4B). Collectively, these results indicate that our qAP-MS screen detects PPIs with high specificity.

Interaction Partners of ATXN1 Are Relevant for Neurodegeneration In VivoNext, we sought to investigate the link between identified PPIs and NDD phenotypes in vivo. To this end, we employed a well-established Drosophila melanogaster model for NDD to test the relevance of identified ATXN1 interaction partners in vivo (Fernandez-Funez et al., 2000). As expected, flies expressing human ATXN1-Q82 in photoreceptor cells showed the characteristic, so-called rough-eye phenotype (REP) indicative of neurodegeneration (Figures 2A–2C). We then asked if proteins, which we identified as ATXN1-Q82 interaction partners, have an impact on the REP by knocking them down in a tissue-specific manner with small hairpin RNAs (shRNAs) (Dietzl et al., 2007). Of the 49 identified interaction partners of ATXN1-Q82, 24 had direct fly orthologs, and 14 of those were available as non-sterile shRNA strains. Silencing two of these genes affected eye morphology alone, and the strains were therefore excluded. The remaining 12 strains were crossed with ATXN1-Q82 transgenic flies to generate F1 flies that express ATXN1-Q82 and the respective shRNA under the same eye-specific promoter. Strains were classified according to the impact of the shRNA on the REP (from 0 = no impact to +++ = lethal). Intriguingly, 9 of the 12 lines exhibited a strong enhancement of the REP (++ or +++, Figures 2F and 2G; Table S2; Figure S5), including necrotic spots indicative of neurodegeneration from subjacent photoreceptor neurons (purple coloring, Figures 2F and 2G). In some cases, no viable offspring was obtained, suggesting a strong enhancement of the neurodegeneration phenotype (Figure 2G). The high overlap between ATXN1 interaction partners and genetic modifiers of ATXN1-mediated neurodegeneration strongly suggests that our data are functionally relevant for NDD phenotypes. The 75% hit rate is considerably higher than the typical hit rate in unbiased genetic screens (1%–4%) (Fernandez-Funez et al., 2000; Kaltenbach et al., 2007), which makes the overlap highly significant (p < 3.4 × 10−10, hypergeometric test with an assumed hit rate of 5%). We also tested all 12 shRNA in another Drosophila model expressing human tau protein (GMR_Tau[R406W]), which exhibit a similar REP (Chatterjee et al., 2009). None of the 12 tested shRNA lines showed an apparent REP change in this model. Thus, the observed REP-modifying effects are specific for the ATXN1-Q82 disease model (Table S2). Collectively, these experiments show that our interaction partners are specifically relevant for ATXN1-Q82-mediated neurodegeneration in vivo.

Interaction Partners of ATXN1 Are Relevant for Neurodegeneration In Vivo

Next, we sought to investigate the link between identified PPIs and NDD phenotypes in vivo. To this end, we employed a well-established Drosophila melanogaster model for NDD to test the relevance of identified ATXN1 interaction partners in vivo (Fernandez-Funez et al., 2000). As expected, flies expressing human ATXN1-Q82 in photoreceptor cells showed the characteristic, so-called rough-eye phenotype (REP) indicative of neurodegeneration (Figures 2A–2C). We then asked if proteins, which we identified as ATXN1-Q82 interaction partners, have an impact on the REP by knocking them down in a tissue-specific manner with small hairpin RNAs (shRNAs) (Dietzl et al., 2007). Of the 49 identified interaction partners of ATXN1-Q82, 24 had direct fly orthologs, and 14 of those were available as non-sterile shRNA strains. Silencing two of these genes affected eye morphology alone, and the strains were therefore excluded. The remaining 12 strains were crossed with ATXN1-Q82 transgenic flies to generate F1 flies that express ATXN1-Q82 and the respective shRNA under the same eye-specific promoter. Strains were classified according to the impact of the shRNA on the REP (from 0 = no impact to +++ = lethal). Intriguingly, 9 of the 12 lines exhibited a strong enhancement of the REP (++ or +++, Figures 2F and 2G; Table S2; Figure S5), including necrotic spots indicative of neurodegeneration from subjacent photoreceptor neurons (purple coloring, Figures 2F and 2G). In some cases, no viable offspring was obtained, suggesting a strong enhancement of the neurodegeneration phenotype (Figure 2G). The high overlap between ATXN1 interaction partners and genetic modifiers of ATXN1-mediated neurodegeneration strongly suggests that our data are functionally relevant for NDD phenotypes. The 75% hit rate is considerably higher than the typical hit rate in unbiased genetic screens (1%–4%) (Fernandez-Funez et al., 2000; Kaltenbach et al., 2007), which makes the overlap highly significant (p < 3.4 × 10−10, hypergeometric test with an assumed hit rate of 5%). We also tested all 12 shRNA in another Drosophila model expressing human tau protein (GMR_Tau[R406W]), which exhibit a similar REP (Chatterjee et al., 2009). None of the 12 tested shRNA lines showed an apparent REP change in this model. Thus, the observed REP-modifying effects are specific for the ATXN1-Q82 disease model (Table S2). Collectively, these experiments show that our interaction partners are specifically relevant for ATXN1-Q82-mediated neurodegeneration in vivo.

Interaction Partners of APP and PSEN1 Are Linked to Human Disease PhenotypesTo investigate whether our identified interaction partners are also linked to disease phenotypes in humans, we compared our interaction data for APP and PSEN1 with a genome-wide association study (GWAS) for AD (Li et al., 2008). We used gene set enrichment analysis of SNP data (Heinig et al., 2010; Holden et al., 2008) to test if genes encoding APP and PSEN1 interaction partners are also found as candidates in the AD GWAS data (Figure 3A; see Experimental Procedures for details). We found that SNPs associated with interaction partners of wild-type and mutant APP and mutant PSEN1 had significantly lower p values than the global distribution of SNPs (Figures 3B and 3C; Table S3; one-sided Wilcoxon-Mann-Whitney test). Thus, the interactome of these proteins is significantly associated with disease, even though no single interaction partner alone showed genome-wide significance in this GWAS. To validate these findings, we went on and replicated this analysis with an independent GWAS dataset based on a larger cohort (Harold et al., 2009). We were able to replicate our findings for wild-type and mutant APP (Figures 3B and 3C; Table S3). In summary, the comparison with GWAS data suggests that our interaction maps for APP and PSEN1 are functionally relevant for AD. This corroborates the view that PPI data can help linking disease genes and phenotypes (Lundby et al., 2014; Rossin et al., 2011).

Interaction Partners of APP and PSEN1 Are Linked to Human Disease Phenotypes

To investigate whether our identified interaction partners are also linked to disease phenotypes in humans, we compared our interaction data for APP and PSEN1 with a genome-wide association study (GWAS) for AD (Li et al., 2008). We used gene set enrichment analysis of SNP data (Heinig et al., 2010; Holden et al., 2008) to test if genes encoding APP and PSEN1 interaction partners are also found as candidates in the AD GWAS data (Figure 3A; see Experimental Procedures for details). We found that SNPs associated with interaction partners of wild-type and mutant APP and mutant PSEN1 had significantly lower p values than the global distribution of SNPs (Figures 3B and 3C; Table S3; one-sided Wilcoxon-Mann-Whitney test). Thus, the interactome of these proteins is significantly associated with disease, even though no single interaction partner alone showed genome-wide significance in this GWAS. To validate these findings, we went on and replicated this analysis with an independent GWAS dataset based on a larger cohort (Harold et al., 2009). We were able to replicate our findings for wild-type and mutant APP (Figures 3B and 3C; Table S3). In summary, the comparison with GWAS data suggests that our interaction maps for APP and PSEN1 are functionally relevant for AD. This corroborates the view that PPI data can help linking disease genes and phenotypes (Lundby et al., 2014; Rossin et al., 2011).

Preferential Interactions of Wild-Type Proteins and Disease-Associated VariantsNext, we asked if our approach could also identify differential protein binding behavior between the wild-type form of a protein and its disease-associated variant in the same pull-down experiment. To this end, we directly compared interaction partners of wild-type proteins and their disease-associated variants by transfecting HEK293T cells with either the myc-tagged wild-type bait proteins or their disease-associated variants (Figures 4A and S3A). Proteins were considered preferential binders when they were enriched at least 1.4-fold on average and showed an inverted ratio in the crossover experiment (Figure S6; Experimental Procedures). While most proteins had abundance ratios close to one, we found that several of our previously identified interactors indeed showed a preferential binding behavior for one of the two offered bait protein variants (Figure 4B). In total, we identified 125 proteins with preferential binding behavior with our qAP-MS approach (Table S1), including several already known differential interactors. For instance, we observed preferential binding of the Capicua homolog CIC toward the ATXN1-Q30 wild-type when compared to the ATXN1-Q82 mutant variant, as previously reported (Lim et al., 2008). The N-Cor/SMRT complex member HDAC3 showed the same behavior, consistent with the finding that ATXN1 loss of function contributes to transcriptional dysregulation (Crespo-Barreto et al., 2010). We also observed that two members (DDOST and RPN1) of the OST bound preferentially to wild-type PSEN1 compared to PSEN1-A431E. This interaction has recently been shown to mediate glycosylation of the v-ATPase required for lysosomal targeting, proteolysis, and autophagy, although it was not known before that it is affected by the disease-associated mutation (Lee et al., 2010).

Preferential Interactions of Wild-Type Proteins and Disease-Associated Variants

Next, we asked if our approach could also identify differential protein binding behavior between the wild-type form of a protein and its disease-associated variant in the same pull-down experiment. To this end, we directly compared interaction partners of wild-type proteins and their disease-associated variants by transfecting HEK293T cells with either the myc-tagged wild-type bait proteins or their disease-associated variants (Figures 4A and S3A). Proteins were considered preferential binders when they were enriched at least 1.4-fold on average and showed an inverted ratio in the crossover experiment (Figure S6; Experimental Procedures). While most proteins had abundance ratios close to one, we found that several of our previously identified interactors indeed showed a preferential binding behavior for one of the two offered bait protein variants (Figure 4B). In total, we identified 125 proteins with preferential binding behavior with our qAP-MS approach (Table S1), including several already known differential interactors. For instance, we observed preferential binding of the Capicua homolog CIC toward the ATXN1-Q30 wild-type when compared to the ATXN1-Q82 mutant variant, as previously reported (Lim et al., 2008). The N-Cor/SMRT complex member HDAC3 showed the same behavior, consistent with the finding that ATXN1 loss of function contributes to transcriptional dysregulation (Crespo-Barreto et al., 2010). We also observed that two members (DDOST and RPN1) of the OST bound preferentially to wild-type PSEN1 compared to PSEN1-A431E. This interaction has recently been shown to mediate glycosylation of the v-ATPase required for lysosomal targeting, proteolysis, and autophagy, although it was not known before that it is affected by the disease-associated mutation (Lee et al., 2010).

The APP Swedish Mutant K670N/M671L Binds LRPPRC and Impairs Mitochondrial FunctionTo further validate our data, we selected one differential binding partner for follow-up experiments. We focused on the interaction between APP and LRPPRC. LRPPRC is a conserved nuclear gene that encodes an RNA-binding protein that is imported into mitochondria (Sterky et al., 2010). In the mitochondrial matrix, LRPPRC post-transcriptionally regulates the expression of mitochondrial genes (Gohil et al., 2010; Ruzzenente et al., 2012; Sasarman et al., 2010). LRPPRC is therefore an essential regulator of the mitochondrial respiratory chain. Intriguingly, mutations in LRPPRC cause Leigh syndrome of the French-Canadian type (LSFC), a NDD characterized by complex IV deficiency (Mootha et al., 2003). Thus, LRPPRC is itself an NDD-associated gene.Our original screen identified LRPPRC as an interaction partner of both wild-type APP and APP-K670N/M671L (Table S1). In the direct comparison, LRPPRC co-purified preferentially with the disease-associated form (Figure 5A; Table S1). The K670N/M671L variant of APP is the so-called Swedish mutation (APPsw), which causes excessive production of Aβ and leads to early-onset AD with highly elevated levels of oxidative stress (Marques et al., 2003). Given that it is well established that mitochondrial function is impaired in AD (Querfurth and LaFerla, 2010), we hypothesized that the APP/LRPPRC interaction might contribute to mitochondrial dysfunction.First, we used co-immunoprecipitation to confirm that the amount of LRPPRC that co-purified with APP was slightly higher for the Swedish variant (Figure 5B). Conversely, when we immunoprecipitated LRPPRC, we detected more APPsw than wild-type APP in the precipitate. Thus, the preferential binding observed by qAP-MS could be validated by reciprocal coIPs. We also found that endogenous (i.e., wild-type) APP and LRPPRC interact in neuroblastoma cells (Figure S4B). Second, in order to assess the differential binding behavior with an independent method, we used the proximity ligation assay (PLA). PLA detects endogenous protein complexes in situ using antibody-linked oligonucleotides (Söderberg et al., 2006). We found that endogenous APP and LRPPRC co-localized in HEK293T cells (Figure 6C, upper panels). Moreover, transfecting APPsw, but not wild-type APP, significantly increased the signal compared to the empty vector control (Figures 5C, lower panels, and 5D). We also found that a small interfering RNA (siRNA) against LRPPRC significantly reduced the signal, which validates the specificity of the assay. PLA experiments also showed co-localization of endogenous APP and LRPPRC in neuroblastoma cells (Figure S7). Next, to investigate the relationship between LRPPRC and APP in the human brain, we performed immunohistochemistry in postmortem cortical brain slices of a patient with early-onset AD caused by APPsw and age-matched controls (Figure 5E). We found that APP and LRPPRC are co-expressed in cortical neurons of healthy controls with an overall similar staining pattern, although these data cannot be used to infer co-localization. Importantly, no LRPPRC staining was observed in the Aβ-positive amyloid plaques in AD patients. Thus, the protein does not appear to interact with Aβ in the extracellular aggregates in affected brain regions. In summary, these data confirm the specific interaction between LRPPRC and APP and the preferential interaction with the Swedish mutant. The observation that LRPPRC is not a component of amyloid plaques in AD patients suggests that the interaction occurs with intracellular APP and not with extracellular Aβ.Mature LRPPRC is considered to be exclusively located in the mitochondrial matrix (Sterky et al., 2010). While APP is mainly thought to be associated with the plasma membrane and the ER, it is also recruited to mitochondria via a cryptic targeting sequence (Devi and Anandatheerthavarada, 2010; Devi et al., 2006; Keil et al., 2004; Lin and Beal, 2006; Park et al., 2006). Mitochondrial APP is not completely imported into the organelle but remains associated with mitochondrial translocases such as TOMM40. Consistently, we also identified TOMM40 and other mitochondrial membrane proteins as APP interactors (Figure 1E; Table S1). The topology of APP in mitochondria is such that its N terminus is located in the matrix while the C terminus faces the cytoplasm (Anandatheerthavarada et al., 2003; Devi et al., 2006). Given the topology of mitochondria-associated APP, the interaction with LRPPRC should occur via the matrix-exposed N terminus. To investigate this possibility, we tested the interaction of LRPPRC with an N-terminal truncation (Figure 5F). LRPPRC co-purified with full-length APP (APP770), but not with APPΔN. These findings suggest that LRPPRC binds to the mitochondrial matrix-exposed N terminus of APP.Given that LRPPRC is a central regulator of mitochondrial gene expression, we asked how APPsw affects the proteome. We transiently transfected differentially SILAC-labeled cells with APP or APPsw for 48 hr. Quantitative proteomics revealed that APPsw markedly (~3-fold) downregulated the cellular levels of LRPPRC itself and its binding partner, SLIRP (Figure 6A). It is remarkable that these proteins were among the most strongly regulated ones in our unbiased experiment. Additionally, several members of the cytochrome c oxidase or the NADH dehydrogenase complexes were downregulated (Figure 6A). Thus, APPsw mediates downregulation of LRPPRC and its downstream targets. We speculated that the reduced expression of respiratory chain components may be due to the downregulation of LRPPRC. Therefore, we tested whether overexpressing LRPPRC together with APPsw might rescue its impact on mitochondria. Indeed, co-expression of LRPPRC together with APPsw resulted in almost-APPwt expression levels of LRPPRC/SLIRP and reduced the impact on the downstream targets (Figure 6B). To further characterize the relationship of APP, LRPPRC, and expression of respiratory chain components, we analyzed mRNA levels by qRT-PCR (Figure 6C). These data showed that APPsw significantly downregulates mRNA levels of LRPPRC and the respiratory chain member COX1, a primary downstream target of LRPPRC. In summary, these data show that APPsw downregulates expression of LRPPRC and respiratory chain genes, and that the impact of APPsw on respiratory chain proteins can partially be rescued by LRPPRC overexpression.Finally, in order to test whether these results are relevant for mitochondrial function, we transiently transfected cells with either APP wild-type or APPsw constructs and co-transfected either LRPPRC or a siRNA directed against LRPPRC. We measured aconitase activity as an enzymatic readout for oxidative stress and overall mitochondrial function (Gardner, 2002). Co-expression of LRPPRC increased the aconitase activity in all conditions, while LRPPRC knockdown considerably reduced the enzymatic activity (Figure 6D). This is consistent with the essential role of LRPPRC for mitochondrial function (Ruzzenente et al., 2012). Cells expressing APPsw showed a significant (p < 0.005) decrease in aconitase activity when compared to APP wild-type-expressing cells. Again, LRPPRC co-expression partially restored aconitase activity to almost wild-type levels. These findings suggest that LRPPRC levels might play an important role in APPsw-induced stress phenotypes.

The APP Swedish Mutant K670N/M671L Binds LRPPRC and Impairs Mitochondrial Function

To further validate our data, we selected one differential binding partner for follow-up experiments. We focused on the interaction between APP and LRPPRC. LRPPRC is a conserved nuclear gene that encodes an RNA-binding protein that is imported into mitochondria (Sterky et al., 2010). In the mitochondrial matrix, LRPPRC post-transcriptionally regulates the expression of mitochondrial genes (Gohil et al., 2010; Ruzzenente et al., 2012; Sasarman et al., 2010). LRPPRC is therefore an essential regulator of the mitochondrial respiratory chain. Intriguingly, mutations in LRPPRC cause Leigh syndrome of the French-Canadian type (LSFC), a NDD characterized by complex IV deficiency (Mootha et al., 2003). Thus, LRPPRC is itself an NDD-associated gene.

Our original screen identified LRPPRC as an interaction partner of both wild-type APP and APP-K670N/M671L (Table S1). In the direct comparison, LRPPRC co-purified preferentially with the disease-associated form (Figure 5A; Table S1). The K670N/M671L variant of APP is the so-called Swedish mutation (APPsw), which causes excessive production of Aβ and leads to early-onset AD with highly elevated levels of oxidative stress (Marques et al., 2003). Given that it is well established that mitochondrial function is impaired in AD (Querfurth and LaFerla, 2010), we hypothesized that the APP/LRPPRC interaction might contribute to mitochondrial dysfunction.

First, we used co-immunoprecipitation to confirm that the amount of LRPPRC that co-purified with APP was slightly higher for the Swedish variant (Figure 5B). Conversely, when we immunoprecipitated LRPPRC, we detected more APPsw than wild-type APP in the precipitate. Thus, the preferential binding observed by qAP-MS could be validated by reciprocal coIPs. We also found that endogenous (i.e., wild-type) APP and LRPPRC interact in neuroblastoma cells (Figure S4B). Second, in order to assess the differential binding behavior with an independent method, we used the proximity ligation assay (PLA). PLA detects endogenous protein complexes in situ using antibody-linked oligonucleotides (Söderberg et al., 2006). We found that endogenous APP and LRPPRC co-localized in HEK293T cells (Figure 6C, upper panels). Moreover, transfecting APPsw, but not wild-type APP, significantly increased the signal compared to the empty vector control (Figures 5C, lower panels, and 5D). We also found that a small interfering RNA (siRNA) against LRPPRC significantly reduced the signal, which validates the specificity of the assay. PLA experiments also showed co-localization of endogenous APP and LRPPRC in neuroblastoma cells (Figure S7). Next, to investigate the relationship between LRPPRC and APP in the human brain, we performed immunohistochemistry in postmortem cortical brain slices of a patient with early-onset AD caused by APPsw and age-matched controls (Figure 5E). We found that APP and LRPPRC are co-expressed in cortical neurons of healthy controls with an overall similar staining pattern, although these data cannot be used to infer co-localization. Importantly, no LRPPRC staining was observed in the Aβ-positive amyloid plaques in AD patients. Thus, the protein does not appear to interact with Aβ in the extracellular aggregates in affected brain regions. In summary, these data confirm the specific interaction between LRPPRC and APP and the preferential interaction with the Swedish mutant. The observation that LRPPRC is not a component of amyloid plaques in AD patients suggests that the interaction occurs with intracellular APP and not with extracellular Aβ.

Mature LRPPRC is considered to be exclusively located in the mitochondrial matrix (Sterky et al., 2010). While APP is mainly thought to be associated with the plasma membrane and the ER, it is also recruited to mitochondria via a cryptic targeting sequence (Devi and Anandatheerthavarada, 2010; Devi et al., 2006; Keil et al., 2004; Lin and Beal, 2006; Park et al., 2006). Mitochondrial APP is not completely imported into the organelle but remains associated with mitochondrial translocases such as TOMM40. Consistently, we also identified TOMM40 and other mitochondrial membrane proteins as APP interactors (Figure 1E; Table S1). The topology of APP in mitochondria is such that its N terminus is located in the matrix while the C terminus faces the cytoplasm (Anandatheerthavarada et al., 2003; Devi et al., 2006). Given the topology of mitochondria-associated APP, the interaction with LRPPRC should occur via the matrix-exposed N terminus. To investigate this possibility, we tested the interaction of LRPPRC with an N-terminal truncation (Figure 5F). LRPPRC co-purified with full-length APP (APP770), but not with APPΔN. These findings suggest that LRPPRC binds to the mitochondrial matrix-exposed N terminus of APP.

Given that LRPPRC is a central regulator of mitochondrial gene expression, we asked how APPsw affects the proteome. We transiently transfected differentially SILAC-labeled cells with APP or APPsw for 48 hr. Quantitative proteomics revealed that APPsw markedly (~3-fold) downregulated the cellular levels of LRPPRC itself and its binding partner, SLIRP (Figure 6A). It is remarkable that these proteins were among the most strongly regulated ones in our unbiased experiment. Additionally, several members of the cytochrome c oxidase or the NADH dehydrogenase complexes were downregulated (Figure 6A). Thus, APPsw mediates downregulation of LRPPRC and its downstream targets. We speculated that the reduced expression of respiratory chain components may be due to the downregulation of LRPPRC. Therefore, we tested whether overexpressing LRPPRC together with APPsw might rescue its impact on mitochondria. Indeed, co-expression of LRPPRC together with APPsw resulted in almost-APPwt expression levels of LRPPRC/SLIRP and reduced the impact on the downstream targets (Figure 6B). To further characterize the relationship of APP, LRPPRC, and expression of respiratory chain components, we analyzed mRNA levels by qRT-PCR (Figure 6C). These data showed that APPsw significantly downregulates mRNA levels of LRPPRC and the respiratory chain member COX1, a primary downstream target of LRPPRC. In summary, these data show that APPsw downregulates expression of LRPPRC and respiratory chain genes, and that the impact of APPsw on respiratory chain proteins can partially be rescued by LRPPRC overexpression.

Finally, in order to test whether these results are relevant for mitochondrial function, we transiently transfected cells with either APP wild-type or APPsw constructs and co-transfected either LRPPRC or a siRNA directed against LRPPRC. We measured aconitase activity as an enzymatic readout for oxidative stress and overall mitochondrial function (Gardner, 2002). Co-expression of LRPPRC increased the aconitase activity in all conditions, while LRPPRC knockdown considerably reduced the enzymatic activity (Figure 6D). This is consistent with the essential role of LRPPRC for mitochondrial function (Ruzzenente et al., 2012). Cells expressing APPsw showed a significant (p < 0.005) decrease in aconitase activity when compared to APP wild-type-expressing cells. Again, LRPPRC co-expression partially restored aconitase activity to almost wild-type levels. These findings suggest that LRPPRC levels might play an important role in APPsw-induced stress phenotypes.

DISCUSSIONThe premise of network medicine is that health and disease depends on complex interactions between the different molecular components of human cells (Barabási et al., 2011). Here, we used qAP-MS to obtain the first quantitative PPI network for several protein variants associated with different NDDs. Several observations indicate that our data are of high quality and important for pathogenesis. First, we recovered several known interactions that have been shown to be relevant for NDDs. Second, we confirmed 18 out of 18 novel interactions by co-immunoprecipitation in transiently transfected HEK cells and nine out of nine endogenous interactions in neuroblastoma cells. Third, we find that ~75% of the identified interaction partners for ATXN1 are functionally relevant for neurodegeneration in a fly model, similar to previous observations for HTT (Kaltenbach et al., 2007). Fourth, we observe that interaction partners of APP and PSEN1 are significantly enriched in AD-associated SNPs. Collectively, these findings indicate that our network of mostly novel interactions is a rich and valuable resource for the community and may help us to better understand the molecular mechanisms of NDDs.Although the high validation rate shows the power of our screen, it is also important to keep the limitations in mind. Most importantly, we analyzed interactions of overexpressed proteins in human cell lines. Although this setup is arguably closer to the in vivo situation than the Y2H system, it cannot recapitulate all relevant processes in human neurons. Moreover, interactions might be lost during affinity purification or could be impaired by the tag. We also cannot rule out that overexpression of a disease-related protein might induce upregulation of specific proteins, which might therefore be misidentified as specific interaction partners.It has been argued that interaction maps are most informative when they reflect differences between states such as health and disease (Ideker and Sharan, 2008). While the Y2H system and standard AP-MS experiments can, in principle, identify interaction partners of mutant proteins, the non-quantitative nature of these methods makes it difficult to directly assess which interactions are affected by the mutation. We used SILAC-based quantification to directly compare the interactions of wild-type proteins and their disease-associated variants for the first time across several different NDDs. We found that for most proteins, disease-associated mutations affected only a small subset of specific PPIs. We also observed that this subset contains several proteins with a known role in pathogenesis. This suggests that differential binding partners may indeed be particularly informative for disease mechanisms.Mitochondrial dysfunction is one of the earliest markers of AD (Querfurth and LaFerla, 2010). While the role of Aβ in this process is well established, our finding that the mitochondrial matrix protein LRPPRC interacts with APP provides evidence that mitochondria-associated APP might also play a role. It has been shown that full-length APP associates with mitochondrial membranes in an N (in mitochondria) to C (out cytoplasm) orientation (Devi et al., 2006). We find that an N-terminal fragment of APP is required for the interaction with LRPPRC, suggesting that the interaction occurs via the matrix-exposed N terminus. It is not yet clear how the Swedish mutation in the C terminus can affect the interaction of the N terminus with LRPPRC. One possibility is that the mutation enhances mitochondrial import and thus the amount of APP accessible for the interaction. LRPPRC is a central regulator of mitochondrial gene expression required for the coordinated translation of mitochondrial mRNAs (Ruzzenente et al., 2012). It is intriguing that mutations in LRPPRC cause neurodegeneration in LSFC (Mootha et al., 2003). We therefore speculate that binding of APPsw to LRPPRC impairs mitochondrial gene expression (Figure 7). The Swedish mutation enhances both Aβ production and binding to LRPPRC. Thus, it is not straightforward to separate both effects from each other. Our observation that APPsw-induced mitochondrial stress is partially rescued by overexpressing LRPPRC indicates that both factors play a role. Since LRPPRC also associates with endogenous (i.e., wild-type) APP, the interaction may also be relevant for sporadic AD cases and/or for the normal cellular function of APP.Genetic linkage analysis has identified many genes related to inherited diseases, and the recent advance of sequencing technologies dramatically accelerated this process. Compared to the pace of disease-gene discovery, their functional characterization is lagging behind. This study shows that quantitative interaction proteomics of disease proteins captures many aspects relevant for pathogenesis. Systematic mapping of changes in PPIs might thus help bridging the gap between identification of disease-associated variants and disease phenotypes. The approach is generic and can thus be applied to other diseases. Eventually, differential interaction proteomics might open new avenues toward individualized therapy.

The premise of network medicine is that health and disease depends on complex interactions between the different molecular components of human cells (Barabási et al., 2011). Here, we used qAP-MS to obtain the first quantitative PPI network for several protein variants associated with different NDDs. Several observations indicate that our data are of high quality and important for pathogenesis. First, we recovered several known interactions that have been shown to be relevant for NDDs. Second, we confirmed 18 out of 18 novel interactions by co-immunoprecipitation in transiently transfected HEK cells and nine out of nine endogenous interactions in neuroblastoma cells. Third, we find that ~75% of the identified interaction partners for ATXN1 are functionally relevant for neurodegeneration in a fly model, similar to previous observations for HTT (Kaltenbach et al., 2007). Fourth, we observe that interaction partners of APP and PSEN1 are significantly enriched in AD-associated SNPs. Collectively, these findings indicate that our network of mostly novel interactions is a rich and valuable resource for the community and may help us to better understand the molecular mechanisms of NDDs.

Although the high validation rate shows the power of our screen, it is also important to keep the limitations in mind. Most importantly, we analyzed interactions of overexpressed proteins in human cell lines. Although this setup is arguably closer to the in vivo situation than the Y2H system, it cannot recapitulate all relevant processes in human neurons. Moreover, interactions might be lost during affinity purification or could be impaired by the tag. We also cannot rule out that overexpression of a disease-related protein might induce upregulation of specific proteins, which might therefore be misidentified as specific interaction partners.

It has been argued that interaction maps are most informative when they reflect differences between states such as health and disease (Ideker and Sharan, 2008). While the Y2H system and standard AP-MS experiments can, in principle, identify interaction partners of mutant proteins, the non-quantitative nature of these methods makes it difficult to directly assess which interactions are affected by the mutation. We used SILAC-based quantification to directly compare the interactions of wild-type proteins and their disease-associated variants for the first time across several different NDDs. We found that for most proteins, disease-associated mutations affected only a small subset of specific PPIs. We also observed that this subset contains several proteins with a known role in pathogenesis. This suggests that differential binding partners may indeed be particularly informative for disease mechanisms.

Mitochondrial dysfunction is one of the earliest markers of AD (Querfurth and LaFerla, 2010). While the role of Aβ in this process is well established, our finding that the mitochondrial matrix protein LRPPRC interacts with APP provides evidence that mitochondria-associated APP might also play a role. It has been shown that full-length APP associates with mitochondrial membranes in an N (in mitochondria) to C (out cytoplasm) orientation (Devi et al., 2006). We find that an N-terminal fragment of APP is required for the interaction with LRPPRC, suggesting that the interaction occurs via the matrix-exposed N terminus. It is not yet clear how the Swedish mutation in the C terminus can affect the interaction of the N terminus with LRPPRC. One possibility is that the mutation enhances mitochondrial import and thus the amount of APP accessible for the interaction. LRPPRC is a central regulator of mitochondrial gene expression required for the coordinated translation of mitochondrial mRNAs (Ruzzenente et al., 2012). It is intriguing that mutations in LRPPRC cause neurodegeneration in LSFC (Mootha et al., 2003). We therefore speculate that binding of APPsw to LRPPRC impairs mitochondrial gene expression (Figure 7). The Swedish mutation enhances both Aβ production and binding to LRPPRC. Thus, it is not straightforward to separate both effects from each other. Our observation that APPsw-induced mitochondrial stress is partially rescued by overexpressing LRPPRC indicates that both factors play a role. Since LRPPRC also associates with endogenous (i.e., wild-type) APP, the interaction may also be relevant for sporadic AD cases and/or for the normal cellular function of APP.

Genetic linkage analysis has identified many genes related to inherited diseases, and the recent advance of sequencing technologies dramatically accelerated this process. Compared to the pace of disease-gene discovery, their functional characterization is lagging behind. This study shows that quantitative interaction proteomics of disease proteins captures many aspects relevant for pathogenesis. Systematic mapping of changes in PPIs might thus help bridging the gap between identification of disease-associated variants and disease phenotypes. The approach is generic and can thus be applied to other diseases. Eventually, differential interaction proteomics might open new avenues toward individualized therapy.

EXPERIMENTAL PROCEDURESCell CultureHuman HEK293T cells were cultured and stable-isotope labeled under standard conditions (see the Supplemental Experimental Procedures). Transient transfections of 15 μg myc-tagged expression vectors were performed with 30 μg polyethylenimine transfection agent per 2 × 107 cells. Cells were lysed in RIPA buffer supplemented with complete protease inhibitor (Roche) and 1% Benzonase (Merck), and cell debris was cleared by centrifugation at 13,000 × g for 10 min.Pull-Down ExperimentsImmunoprecipitations of myc-tagged bait proteins were performed using the μMACS c-myc isolation Kit (Miltenyi Biotec) according to the manufacturer’s instructions. Proteins bound to the beads were eluted with 100 mM glycine (pH 2.5) and ethanol-precipitated overnight at 4°C. Precipitated proteins were processed essentially as described previously (Paul et al., 2011). Co-immunoprecipitations of endogenous proteins were performed by crosslinking specific anti-bait antibodies to NHS-activated Sepharose as described (Paul et al., 2011). Co-precipitates were eluted by boiling in SDS loading buffer and subsequently analyzed by western blotting.Drosophila Models and Genetic ScreenFlies were raised and maintained at 25°C on standard food. The screening stock was generated by recombination of GMR-GAL4 driver and UAS-ATXN1-Q82 insertion (GMR > ATXN1-Q82). Screening itself was essentially conducted as described previously (Vossfeldt et al., 2012).Reporter AssaysFor mitochondrial activity tests, mitochondria were isolated from 104–105 HEK293T cells using the mitochondria isolation kit (Miltenyi Biotec) according to the manufacturer’s instructions. Mitochondrial activity was then assessed by determining the mitochondrial aconitase activity (Aconitase Activity Assay Kit, Merck) according to the manufacturer’s instructions. Values were normalized to the mitochondrial protein concentration.Liquid Chromatography Tandem Mass Spectrometry AnalysisPull-down samples were measured on LTQ-Orbitrap or LTQ-Orbitrap XL mass spectrometers and whole-proteome experiments on LTQ-Orbitrap Velos machines. Raw mass spectrometric data were analyzed using the MaxQuant software environment (Cox and Mann, 2008). See the Supplemental Experimental Procedures for details.GWAS Enrichment AnalysisCohort data were acquired from two large GWASs with 753/736 cases/controls (GenADA; Li et al., 2008) and 3,941/7,848 cases/controls (GERAD; Harold et al., 2009). Genomic coordinates of all genes were obtained from BioMart (Ensembl release 54). SNP coordinates (NCBI36 assembly) and association summary statistics of the GenADA study were obtained from dbGAP (accession number phs000219v1). The same data structure for the GERAD study was kindly provided by the authors. For each bait protein SNP, sets were defined that comprise all SNPs that are located within a distance of 100 kb of any gene whose protein interacts with the corresponding bait protein. A one-sided Wilcoxon-Mann-Whitney test was applied in order to detect differences of the means of the distribution of GWAS p values within a SNP set compared to the global distribution of p values (Heinig et al., 2010). Combined p values were computed using Fisher’s method.

EXPERIMENTAL PROCEDURES

Cell CultureHuman HEK293T cells were cultured and stable-isotope labeled under standard conditions (see the Supplemental Experimental Procedures). Transient transfections of 15 μg myc-tagged expression vectors were performed with 30 μg polyethylenimine transfection agent per 2 × 107 cells. Cells were lysed in RIPA buffer supplemented with complete protease inhibitor (Roche) and 1% Benzonase (Merck), and cell debris was cleared by centrifugation at 13,000 × g for 10 min.

Cell Culture

Human HEK293T cells were cultured and stable-isotope labeled under standard conditions (see the Supplemental Experimental Procedures). Transient transfections of 15 μg myc-tagged expression vectors were performed with 30 μg polyethylenimine transfection agent per 2 × 107 cells. Cells were lysed in RIPA buffer supplemented with complete protease inhibitor (Roche) and 1% Benzonase (Merck), and cell debris was cleared by centrifugation at 13,000 × g for 10 min.

Pull-Down ExperimentsImmunoprecipitations of myc-tagged bait proteins were performed using the μMACS c-myc isolation Kit (Miltenyi Biotec) according to the manufacturer’s instructions. Proteins bound to the beads were eluted with 100 mM glycine (pH 2.5) and ethanol-precipitated overnight at 4°C. Precipitated proteins were processed essentially as described previously (Paul et al., 2011). Co-immunoprecipitations of endogenous proteins were performed by crosslinking specific anti-bait antibodies to NHS-activated Sepharose as described (Paul et al., 2011). Co-precipitates were eluted by boiling in SDS loading buffer and subsequently analyzed by western blotting.

Pull-Down Experiments

Immunoprecipitations of myc-tagged bait proteins were performed using the μMACS c-myc isolation Kit (Miltenyi Biotec) according to the manufacturer’s instructions. Proteins bound to the beads were eluted with 100 mM glycine (pH 2.5) and ethanol-precipitated overnight at 4°C. Precipitated proteins were processed essentially as described previously (Paul et al., 2011). Co-immunoprecipitations of endogenous proteins were performed by crosslinking specific anti-bait antibodies to NHS-activated Sepharose as described (Paul et al., 2011). Co-precipitates were eluted by boiling in SDS loading buffer and subsequently analyzed by western blotting.

Drosophila Models and Genetic ScreenFlies were raised and maintained at 25°C on standard food. The screening stock was generated by recombination of GMR-GAL4 driver and UAS-ATXN1-Q82 insertion (GMR > ATXN1-Q82). Screening itself was essentially conducted as described previously (Vossfeldt et al., 2012).

Drosophila Models and Genetic Screen

Flies were raised and maintained at 25°C on standard food. The screening stock was generated by recombination of GMR-GAL4 driver and UAS-ATXN1-Q82 insertion (GMR > ATXN1-Q82). Screening itself was essentially conducted as described previously (Vossfeldt et al., 2012).

Reporter AssaysFor mitochondrial activity tests, mitochondria were isolated from 104–105 HEK293T cells using the mitochondria isolation kit (Miltenyi Biotec) according to the manufacturer’s instructions. Mitochondrial activity was then assessed by determining the mitochondrial aconitase activity (Aconitase Activity Assay Kit, Merck) according to the manufacturer’s instructions. Values were normalized to the mitochondrial protein concentration.

Reporter Assays

For mitochondrial activity tests, mitochondria were isolated from 104–105 HEK293T cells using the mitochondria isolation kit (Miltenyi Biotec) according to the manufacturer’s instructions. Mitochondrial activity was then assessed by determining the mitochondrial aconitase activity (Aconitase Activity Assay Kit, Merck) according to the manufacturer’s instructions. Values were normalized to the mitochondrial protein concentration.

Liquid Chromatography Tandem Mass Spectrometry AnalysisPull-down samples were measured on LTQ-Orbitrap or LTQ-Orbitrap XL mass spectrometers and whole-proteome experiments on LTQ-Orbitrap Velos machines. Raw mass spectrometric data were analyzed using the MaxQuant software environment (Cox and Mann, 2008). See the Supplemental Experimental Procedures for details.

Liquid Chromatography Tandem Mass Spectrometry Analysis

Pull-down samples were measured on LTQ-Orbitrap or LTQ-Orbitrap XL mass spectrometers and whole-proteome experiments on LTQ-Orbitrap Velos machines. Raw mass spectrometric data were analyzed using the MaxQuant software environment (Cox and Mann, 2008). See the Supplemental Experimental Procedures for details.

GWAS Enrichment AnalysisCohort data were acquired from two large GWASs with 753/736 cases/controls (GenADA; Li et al., 2008) and 3,941/7,848 cases/controls (GERAD; Harold et al., 2009). Genomic coordinates of all genes were obtained from BioMart (Ensembl release 54). SNP coordinates (NCBI36 assembly) and association summary statistics of the GenADA study were obtained from dbGAP (accession number phs000219v1). The same data structure for the GERAD study was kindly provided by the authors. For each bait protein SNP, sets were defined that comprise all SNPs that are located within a distance of 100 kb of any gene whose protein interacts with the corresponding bait protein. A one-sided Wilcoxon-Mann-Whitney test was applied in order to detect differences of the means of the distribution of GWAS p values within a SNP set compared to the global distribution of p values (Heinig et al., 2010). Combined p values were computed using Fisher’s method.

GWAS Enrichment Analysis

Cohort data were acquired from two large GWASs with 753/736 cases/controls (GenADA; Li et al., 2008) and 3,941/7,848 cases/controls (GERAD; Harold et al., 2009). Genomic coordinates of all genes were obtained from BioMart (Ensembl release 54). SNP coordinates (NCBI36 assembly) and association summary statistics of the GenADA study were obtained from dbGAP (accession number phs000219v1). The same data structure for the GERAD study was kindly provided by the authors. For each bait protein SNP, sets were defined that comprise all SNPs that are located within a distance of 100 kb of any gene whose protein interacts with the corresponding bait protein. A one-sided Wilcoxon-Mann-Whitney test was applied in order to detect differences of the means of the distribution of GWAS p values within a SNP set compared to the global distribution of p values (Heinig et al., 2010). Combined p values were computed using Fisher’s method.

Supplementary MaterialSupplementary Files

Supplementary Material
