---
pmid: '22699612'
title: NPR3 and NPR4 are receptors for the immune signal salicylic acid in plants.
authors:
- Fu ZQ
- Yan S
- Saleh A
- Wang W
- Ruble J
- Oka N
- Mohan R
- Spoel SH
- Tada Y
- Zheng N
- Dong X
journal: Nature
year: '2012'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3376392
doi: 10.1038/nature11162
---

# NPR3 and NPR4 are receptors for the immune signal salicylic acid in plants.
**Authors:** Fu ZQ, Yan S, Saleh A, Wang W, Ruble J, Oka N, Mohan R, Spoel SH, Tada Y, Zheng N, Dong X
**Journal:** Nature (2012)
**DOI:** [10.1038/nature11162](https://doi.org/10.1038/nature11162)
**PMC:** [PMC3376392](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3376392/)

## Abstract

1. Nature. 2012 May 16;486(7402):228-32. doi: 10.1038/nature11162.

NPR3 and NPR4 are receptors for the immune signal salicylic acid in plants.

Fu ZQ(1), Yan S, Saleh A, Wang W, Ruble J, Oka N, Mohan R, Spoel SH, Tada Y, 
Zheng N, Dong X.

Author information:
(1)Howard Hughes Medical Institute-Gordon and Betty Moore Foundation, Department 
of Biology, PO Box 90338, Duke University, Durham, North Carolina 27708, USA.

Comment in
    Nature. 2012 Jun 13;486(7402):198-9. doi: 10.1038/486198a.
    Sci Signal. 2013 Jun 11;6(279):jc3. doi: 10.1126/scisignal.2003944.

Salicylic acid (SA) is a plant immune signal produced after pathogen challenge 
to induce systemic acquired resistance. It is the only major plant hormone for 
which the receptor has not been firmly identified. Systemic acquired resistance 
in Arabidopsis requires the transcription cofactor nonexpresser of PR genes 1 
(NPR1), the degradation of which acts as a molecular switch. Here we show that 
the NPR1 paralogues NPR3 and NPR4 are SA receptors that bind SA with different 
affinities. NPR3 and NPR4 function as adaptors of the Cullin 3 ubiquitin E3 
ligase to mediate NPR1 degradation in an SA-regulated manner. Accordingly, the 
Arabidopsis npr3 npr4 double mutant accumulates higher levels of NPR1, and is 
insensitive to induction of systemic acquired resistance. Moreover, this mutant 
is defective in pathogen effector-triggered programmed cell death and immunity. 
Our study reveals the mechanism of SA perception in determining cell death and 
survival in response to pathogen challenge.

DOI: 10.1038/nature11162
PMCID: PMC3376392
PMID: 22699612 [Indexed for MEDLINE]

## Full Text

Upon pathogen challenge, host cells have to make a life-and-death decision to fend off infection. Recognition of a pathogen effector by a host resistance (R) protein can lead to effector-triggered immunity (ETI), characterized by rapid programmed cell death (PCD) known as the hypersensitive response (HR)1. The clearly defined boundary of the HR indicates the presence of a mechanism that controls cell death and survival. Despite intense studies of plant mutants defective in controlling the spread of PCD2, the regulatory mechanism still remains a mystery.

Localized PCD can induce systemic acquired resistance (SAR) through the production of the immune signal, salicylic acid (SA)3. SA triggers global transcriptional reprogramming and resistance to a broad-spectrum of pathogens. The receptor for SA has been sought after for many years, mainly through biochemical purification of SA-binding proteins4-6. However, genetic data for these SA-binding proteins, which include a catalase, a chloroplast carbonic anhydrase, and a methyl SA esterase, suggest that none of them functions as a bona fide SA receptor. In contrast, genetic studies of SA-insensitive mutants have strongly suggested that NPR1, which contains a BTB (bric à brac, tramtrack, broad-complex) domain, an ankryin repeat domain and a nuclear localization sequence, is a potential SA receptor7. However, the NPR1 protein does not have significant SA binding activity under different test conditions (Supplementary Fig. 2).

Instead of direct binding, SA has been shown to control the nuclear translocation of NPR1 through cellular redox changes8. In the absence of pathogen challenge, NPR1 is retained in the cytoplasm as an oligomer through redox-sensitive intermolecular disulphide bonds. Upon induction, these disulphide bonds are reduced, releasing NPR1 monomers into the nucleus, where NPR1 serves as a cofactor for transcription factors, such as TGAs, to induce defence-related genes. In the absence of a functional NPR1 protein, SA-induced transcriptional reprogramming is almost completely blocked.

The presence of a BTB domain in NPR1 suggests that like other BTB domain-containing proteins, it may interact with Cullin 3 (CUL3) E3 ligase and mediate substrate degradation9. However, our research led to the surprising finding that the NPR1 protein itself is degraded by the proteasome. While NPR1 is degraded in the nucleus of resting cells to dampen basal expression of defence genes, it is phosphorylated upon immune activation at an IκB-like phosphodegron motif, ubiquitinylated by a CUL3 E3 ligase, and degraded to sustain maximum levels of target gene expression probably through accelerated recycling of the transcription initiation complex10. Blocking NPR1 degradation by mutating the IκB-like phophodegron in NPR1 or the two CUL3 genes (cul3a cul3b) in Arabidopsis led to elevated basal resistance, but insensitivity to SAR induction. Therefore, nuclear accumulation of NPR1 is needed for basal defence gene expression and resistance, while its subsequent turnover is required for establishing SAR.

NPR3 and NPR4 are Cullin 3 adaptors mediating NPR1 degradationIn a search for the adaptor proteins of the CUL3 E3 ligase that specifically target NPR1 for degradation, we considered its paralogues, NPR3 and NPR4, as possible candidates, because both contain the BTB-domain as well as an additional protein-protein interaction domain (ankyrin-repeat)(Supplementary Fig. 3), which are typical for CUL3 substrate adaptors9. More importantly, despite their sequence similarities to NPR1, the npr3 npr4 double mutant has the opposite phenotype of npr1 in that it exhibits enhanced disease resistance11, a phenotype reminiscent of the cul3a cul3b mutant10.To test our hypothesis that NPR3 and NPR4 are CUL3 adaptors for NPR1 degradation, we examined the accumulation of NPR1 protein in wild type (WT), npr3, npr4, and npr3 npr4 plants. NPR1 protein levels were higher in the npr4 and the npr3 npr4 mutants than in WT in the absence of exogenous SA, and increased faster in the npr3, npr4, and npr3 npr4 mutants compared to WT in response to SA treatment (Fig. 1a). The effects of npr3 and npr4 on NPR1 were likely post-transcriptional as NPR1 transcripts were not increased in these mutants (Supplementary Fig. 4). To further prove our hypothesis, we performed in vitro degradation experiments using purified recombinant GST-tagged NPR1 protein. We found that after 15 minutes of incubation, the recombinant NPR1 protein was degraded in the WT plant extract, but not in npr3 npr4 (Fig. 1b). Addition of purified His-MBP-tagged recombinant NPR3 and NPR4 proteins to the extract complemented the mutant phenotype, supporting a role of NPR3 and NPR4 in mediating NPR1 degradation. This degradation is likely through the proteasome, as application of the proteasome inhibitor MG115 stabilized the protein (Fig. 1b). To further demonstrate that NPR3 and NPR4 serve as adaptors for the CUL3 E3 ligase, we first performed pull-down experiment using in vitro translated NPR3-HA and NPR4-HA. We found that CUL3A could pull down NPR3 and NPR4, with NPR4 showing a stronger interaction (Fig. 1c). Then we performed co-immunoprecipitation (co-IP) assay using transgenic plants constitutively expressing NPR1-GFP in npr1 and npr1 npr3 npr4 mutants. We found that the amount of the endogenous CUL3 protein pulled down by NPR1-GFP was significantly reduced in the npr1 npr3 npr4 triple mutant compared to the npr1 single mutant (Fig. 1d), indicating that the NPR1-GFP interaction with CUL3 requires NPR3 and NPR4. These results further support our hypothesis that NPR4 and NPR3 are CUL3 adaptors for the degradation of NPR1 before and after SA induction, respectively (Fig. 1a).

NPR3 and NPR4 are Cullin 3 adaptors mediating NPR1 degradation

In a search for the adaptor proteins of the CUL3 E3 ligase that specifically target NPR1 for degradation, we considered its paralogues, NPR3 and NPR4, as possible candidates, because both contain the BTB-domain as well as an additional protein-protein interaction domain (ankyrin-repeat)(Supplementary Fig. 3), which are typical for CUL3 substrate adaptors9. More importantly, despite their sequence similarities to NPR1, the npr3 npr4 double mutant has the opposite phenotype of npr1 in that it exhibits enhanced disease resistance11, a phenotype reminiscent of the cul3a cul3b mutant10.

To test our hypothesis that NPR3 and NPR4 are CUL3 adaptors for NPR1 degradation, we examined the accumulation of NPR1 protein in wild type (WT), npr3, npr4, and npr3 npr4 plants. NPR1 protein levels were higher in the npr4 and the npr3 npr4 mutants than in WT in the absence of exogenous SA, and increased faster in the npr3, npr4, and npr3 npr4 mutants compared to WT in response to SA treatment (Fig. 1a). The effects of npr3 and npr4 on NPR1 were likely post-transcriptional as NPR1 transcripts were not increased in these mutants (Supplementary Fig. 4). To further prove our hypothesis, we performed in vitro degradation experiments using purified recombinant GST-tagged NPR1 protein. We found that after 15 minutes of incubation, the recombinant NPR1 protein was degraded in the WT plant extract, but not in npr3 npr4 (Fig. 1b). Addition of purified His-MBP-tagged recombinant NPR3 and NPR4 proteins to the extract complemented the mutant phenotype, supporting a role of NPR3 and NPR4 in mediating NPR1 degradation. This degradation is likely through the proteasome, as application of the proteasome inhibitor MG115 stabilized the protein (Fig. 1b). To further demonstrate that NPR3 and NPR4 serve as adaptors for the CUL3 E3 ligase, we first performed pull-down experiment using in vitro translated NPR3-HA and NPR4-HA. We found that CUL3A could pull down NPR3 and NPR4, with NPR4 showing a stronger interaction (Fig. 1c). Then we performed co-immunoprecipitation (co-IP) assay using transgenic plants constitutively expressing NPR1-GFP in npr1 and npr1 npr3 npr4 mutants. We found that the amount of the endogenous CUL3 protein pulled down by NPR1-GFP was significantly reduced in the npr1 npr3 npr4 triple mutant compared to the npr1 single mutant (Fig. 1d), indicating that the NPR1-GFP interaction with CUL3 requires NPR3 and NPR4. These results further support our hypothesis that NPR4 and NPR3 are CUL3 adaptors for the degradation of NPR1 before and after SA induction, respectively (Fig. 1a).

SA affects NPR1-NPR3 and NPR1-NPR4 interactionsProteasome-mediated protein degradation plays a crucial role in regulating plant hormone receptors12. In some of these cases, the hormones serve as a molecular glue to facilitate formation of the receptor complex13,14, which includes the substrate adaptor for the E3 ligase and the corresponding substrate. Our data show that proteasome-mediated degradation of NPR1 is also involved in SA signalling10, although a different E3 ligase (CUL3, instead of Cullin 1) is used.To test the possibility that SA is part of the NPR1-NPR3/4 complex, we performed yeast two-hybrid (Y2H) assay. Using NPR3 as bait and NPR1 as prey, little growth was observed on plates without SA (Fig. 2a). However, yeast growth was observed on plates supplemented with 100 μM SA or with the functional analogue of SA, INA (2, 6-dichloroisonicotinic acid)15, but not with 4-HBA (4-hydroxybenzoic acid)6, which cannot induce SAR. Interestingly, while SA promoted NPR1-NPR3 interaction, it disrupted the interaction between NPR1 and NPR4 (Fig. 2b). Moreover, NPR3 and NPR4 could form both homodimers and heterodimers with each other in the presence of SA and INA, but not 4-HBA (Fig. 2c). This suggests that NPR3 and NPR4 not only control NPR1 stability, but also self-regulate. Because NPR1 did not form homodimers with or without SA and interacted with NPR2 independent of SA (Supplementary Fig. 5), we focused on the regulatory roles of NPR3 and NPR4.To further validate the Y2H data, we performed in vitro pull-down assay. As shown in Fig. 2d, using the GST-NPR3 protein, we were only able to pull down His-MBP-NPR1 in the presence of SA. In contrast, the GST-NPR4 could pull down His-MBP-NPR1 only in the absence of SA, indicating that the NPR1-NPR4 interaction was disrupted by SA.

SA affects NPR1-NPR3 and NPR1-NPR4 interactions

Proteasome-mediated protein degradation plays a crucial role in regulating plant hormone receptors12. In some of these cases, the hormones serve as a molecular glue to facilitate formation of the receptor complex13,14, which includes the substrate adaptor for the E3 ligase and the corresponding substrate. Our data show that proteasome-mediated degradation of NPR1 is also involved in SA signalling10, although a different E3 ligase (CUL3, instead of Cullin 1) is used.

To test the possibility that SA is part of the NPR1-NPR3/4 complex, we performed yeast two-hybrid (Y2H) assay. Using NPR3 as bait and NPR1 as prey, little growth was observed on plates without SA (Fig. 2a). However, yeast growth was observed on plates supplemented with 100 μM SA or with the functional analogue of SA, INA (2, 6-dichloroisonicotinic acid)15, but not with 4-HBA (4-hydroxybenzoic acid)6, which cannot induce SAR. Interestingly, while SA promoted NPR1-NPR3 interaction, it disrupted the interaction between NPR1 and NPR4 (Fig. 2b). Moreover, NPR3 and NPR4 could form both homodimers and heterodimers with each other in the presence of SA and INA, but not 4-HBA (Fig. 2c). This suggests that NPR3 and NPR4 not only control NPR1 stability, but also self-regulate. Because NPR1 did not form homodimers with or without SA and interacted with NPR2 independent of SA (Supplementary Fig. 5), we focused on the regulatory roles of NPR3 and NPR4.

To further validate the Y2H data, we performed in vitro pull-down assay. As shown in Fig. 2d, using the GST-NPR3 protein, we were only able to pull down His-MBP-NPR1 in the presence of SA. In contrast, the GST-NPR4 could pull down His-MBP-NPR1 only in the absence of SA, indicating that the NPR1-NPR4 interaction was disrupted by SA.

NPR3 and NPR4 bind SA with different affinitiesBoth the Y2H and the in vitro pull-down results strongly suggest that SA directly binds to NPR3 and NPR4 to control their interactions with NPR1. To prove that NPR3 and NPR4 are SA receptors, we measured their SA binding activities using [3H]-SA. We found that both GST-tagged NPR3 and NPR4 recombinant proteins bound [3H]-SA (Fig. 3a, 3b, Supplementary Fig. 2a). Next, we assessed if active or inactive SA analogues could compete with [3H]-SA to bind to GST-NPR3 and GST-NPR4. CSA (chlorosalicylic acid; an active SAR inducer)6 and INA reduced the binding of [3H]-SA to GST-NPR3 and GST-NPR4, whereas 4-HBA had little effect (Fig. 3a and 3b). To assess the binding affinity of NPR3 and NPR4, we performed saturation binding experiments. While NPR4 exhibited a classical saturation curve (Fig. 3c), NPR3 binding could not be saturated even with 1000 nM [3H]-SA, indicating that NPR3 has a lower affinity than NPR4. Accordingly, the binding of SA to NPR3 was slower than NPR4 (Supplementary Fig. 6). Next, we analysed the saturation binding data with GraphPad Prism using different models and found that the model One site-Specific binding with Hill Slope is significantly better than the other models, which indicates that there are multiple binding sites or fractions in NPR3 and NPR4. The Kd value for NPR4 was 46.2±2.35 nM with a Hill coefficient (h) of 0.830±0.0314. To check the cooperativity of different binding sites, we carried out dissociation experiments by addition of 1 mM non-radioactive labelled SA (Cold SA) or by infinite dilution. The dissociation curves (Fig. 3d) indicate that NPR4 has multiple SA binding sites, and the lack of overlap between the two curves suggests negative cooperativity between these binding sites (the first binding reduces the affinity for subsequent binding). The Kd value for NPR3 (981 nM, Supplementary Fig. 7) was significantly higher than 100 nM, which made saturation binding an inappropriate way to calculate the Kd. Therefore, we performed homologous competitive binding assay (Fig. 3e). The IC50 value was calculated to be 1811 nM (Log IC50=3.26±0.0901) with a Hill coefficient of 0.554±0.0612. Through these analyses, we demonstrated that NPR3 and NPR4 bind SA specifically and with different affinities.To further examine the receptor complex, we performed gel filtration analysis on the purified recombinant NPR4 protein, the receptor with the higher affinity to SA. Because the recombinant NPR4 protein spontaneously oligomerized in vitro in the absence of a reducing agent, our analysis focused on samples pretreated with 100 mM DTT followed by dialysis against 5 mM DTT. We discovered that NPR4 was present in an estimated tetrameric form, which was competent in binding to SA (Fig. 3f). Interestingly, SA binding did not change the gel filtration elution profile of the protein. Further experiments are required to investigate how SA affects the receptor complexes to make them either more accessible (i.e., NPR3 binding to NPR1) or less accessible (i.e., NPR4 binding to NPR1) for substrate binding.

NPR3 and NPR4 bind SA with different affinities

Both the Y2H and the in vitro pull-down results strongly suggest that SA directly binds to NPR3 and NPR4 to control their interactions with NPR1. To prove that NPR3 and NPR4 are SA receptors, we measured their SA binding activities using [3H]-SA. We found that both GST-tagged NPR3 and NPR4 recombinant proteins bound [3H]-SA (Fig. 3a, 3b, Supplementary Fig. 2a). Next, we assessed if active or inactive SA analogues could compete with [3H]-SA to bind to GST-NPR3 and GST-NPR4. CSA (chlorosalicylic acid; an active SAR inducer)6 and INA reduced the binding of [3H]-SA to GST-NPR3 and GST-NPR4, whereas 4-HBA had little effect (Fig. 3a and 3b). To assess the binding affinity of NPR3 and NPR4, we performed saturation binding experiments. While NPR4 exhibited a classical saturation curve (Fig. 3c), NPR3 binding could not be saturated even with 1000 nM [3H]-SA, indicating that NPR3 has a lower affinity than NPR4. Accordingly, the binding of SA to NPR3 was slower than NPR4 (Supplementary Fig. 6). Next, we analysed the saturation binding data with GraphPad Prism using different models and found that the model One site-Specific binding with Hill Slope is significantly better than the other models, which indicates that there are multiple binding sites or fractions in NPR3 and NPR4. The Kd value for NPR4 was 46.2±2.35 nM with a Hill coefficient (h) of 0.830±0.0314. To check the cooperativity of different binding sites, we carried out dissociation experiments by addition of 1 mM non-radioactive labelled SA (Cold SA) or by infinite dilution. The dissociation curves (Fig. 3d) indicate that NPR4 has multiple SA binding sites, and the lack of overlap between the two curves suggests negative cooperativity between these binding sites (the first binding reduces the affinity for subsequent binding). The Kd value for NPR3 (981 nM, Supplementary Fig. 7) was significantly higher than 100 nM, which made saturation binding an inappropriate way to calculate the Kd. Therefore, we performed homologous competitive binding assay (Fig. 3e). The IC50 value was calculated to be 1811 nM (Log IC50=3.26±0.0901) with a Hill coefficient of 0.554±0.0612. Through these analyses, we demonstrated that NPR3 and NPR4 bind SA specifically and with different affinities.

To further examine the receptor complex, we performed gel filtration analysis on the purified recombinant NPR4 protein, the receptor with the higher affinity to SA. Because the recombinant NPR4 protein spontaneously oligomerized in vitro in the absence of a reducing agent, our analysis focused on samples pretreated with 100 mM DTT followed by dialysis against 5 mM DTT. We discovered that NPR4 was present in an estimated tetrameric form, which was competent in binding to SA (Fig. 3f). Interestingly, SA binding did not change the gel filtration elution profile of the protein. Further experiments are required to investigate how SA affects the receptor complexes to make them either more accessible (i.e., NPR3 binding to NPR1) or less accessible (i.e., NPR4 binding to NPR1) for substrate binding.

The npr3 npr4 double mutant is defective in SAR and ETITo understand the biological significance of NPR3/4-mediated degradation of NPR1, a positive regulator of SAR, we first performed SAR tests in the npr3, npr4 and npr3 npr4 mutants using Pseudomonas syringae pv. maculicola ES4326 (Psm ES4326). Consistent with a previous report11, there was a significant reduction in Psm ES4326 growth in the npr3 npr4 double mutant without SAR induction (Fig. 4a). However, even after SAR induction by local inoculation of avirulent Psm ES4326/avrRpt2, no further reduction in growth of virulent Psm ES4326 in systemic tissue was observed in the npr3 npr4 double mutant. To a lesser degree, SAR was also defective in the npr3 single mutant. Thus, stabilization of NPR1 protein in the npr3, and npr3 npr4 mutants rendered these plants insensitive to SAR induction. This phenotype is similar to that observed in the cul3a cul3b double mutant10, validating the role of NPR3 and NPR4 in CUL3-mediated degradation of NPR1 and SAR.Based on our knowledge that SAR and ETI are two distinct defence strategies, with the former promoting cell survival and the latter triggering PCD, we then tested the npr mutants for ETI using Pseudomonas strains expressing different effectors. Surprisingly, we found that the npr3 npr4 mutant failed to undergo PCD (Fig. 4b, Supplementary Fig. 8a) as quantified by ion leakage (Fig. 4c), and was compromised in resistance triggered by the effectors (Fig. 4d). The same phenotypes were observed in different mutant alleles of npr3, npr4 and npr3 npr4 (Supplementary Fig. 8b). The ETI deficiency observed in npr3 npr4 is likely caused by the elevated accumulation of NPR1, because this phenotype was suppressed in the npr1 npr3 npr4 triple mutant.To observe NPR1 turnover in response to pathogen challenge in situ, we inoculated Psm ES4326/avrRpt2 in the 35S:npr1C82A-GFP transgenic plant, in which NPR1 is constitutively localized in the nucleus (Fig. 4e)16. Eleven hours after inoculation, some cells showed increased chlorophyll leakage (red, Fig. 4f) with overlapping accumulation of phenolic compounds (larger green spots, Fig. 4e) indicative of PCD (yellow, Fig. 4g), while other cells were still intact. The npr1C82A-GFP fluorescence was markedly reduced inside the inoculated region (Fig. 4g, 4h). Strikingly, the npr1C82A-GFP fluorescence level was the highest in the cells surrounding the HR lesion (Fig. 4h), consistent with the genetic data suggesting that NPR1 is an inhibitor of PCD during ETI.

The npr3 npr4 double mutant is defective in SAR and ETI

To understand the biological significance of NPR3/4-mediated degradation of NPR1, a positive regulator of SAR, we first performed SAR tests in the npr3, npr4 and npr3 npr4 mutants using Pseudomonas syringae pv. maculicola ES4326 (Psm ES4326). Consistent with a previous report11, there was a significant reduction in Psm ES4326 growth in the npr3 npr4 double mutant without SAR induction (Fig. 4a). However, even after SAR induction by local inoculation of avirulent Psm ES4326/avrRpt2, no further reduction in growth of virulent Psm ES4326 in systemic tissue was observed in the npr3 npr4 double mutant. To a lesser degree, SAR was also defective in the npr3 single mutant. Thus, stabilization of NPR1 protein in the npr3, and npr3 npr4 mutants rendered these plants insensitive to SAR induction. This phenotype is similar to that observed in the cul3a cul3b double mutant10, validating the role of NPR3 and NPR4 in CUL3-mediated degradation of NPR1 and SAR.

Based on our knowledge that SAR and ETI are two distinct defence strategies, with the former promoting cell survival and the latter triggering PCD, we then tested the npr mutants for ETI using Pseudomonas strains expressing different effectors. Surprisingly, we found that the npr3 npr4 mutant failed to undergo PCD (Fig. 4b, Supplementary Fig. 8a) as quantified by ion leakage (Fig. 4c), and was compromised in resistance triggered by the effectors (Fig. 4d). The same phenotypes were observed in different mutant alleles of npr3, npr4 and npr3 npr4 (Supplementary Fig. 8b). The ETI deficiency observed in npr3 npr4 is likely caused by the elevated accumulation of NPR1, because this phenotype was suppressed in the npr1 npr3 npr4 triple mutant.

To observe NPR1 turnover in response to pathogen challenge in situ, we inoculated Psm ES4326/avrRpt2 in the 35S:npr1C82A-GFP transgenic plant, in which NPR1 is constitutively localized in the nucleus (Fig. 4e)16. Eleven hours after inoculation, some cells showed increased chlorophyll leakage (red, Fig. 4f) with overlapping accumulation of phenolic compounds (larger green spots, Fig. 4e) indicative of PCD (yellow, Fig. 4g), while other cells were still intact. The npr1C82A-GFP fluorescence was markedly reduced inside the inoculated region (Fig. 4g, 4h). Strikingly, the npr1C82A-GFP fluorescence level was the highest in the cells surrounding the HR lesion (Fig. 4h), consistent with the genetic data suggesting that NPR1 is an inhibitor of PCD during ETI.

DiscussionThrough this study, we identified the NPR1 paralogues, NPR3 and NPR4, as receptors for the immune signal SA. These receptors have different binding affinities to SA (Fig. 3), suggesting that they may be differentially responsive to spatiotemporal changes in cellular SA concentrations. SA controls accessibility of the CUL3 ligase adaptors, NPR3 and NPR4, to their substrate NPR1 (Fig. 2), thereby regulating NPR1 stability and activity (Figs. 1 and 4).Based on our findings, we present a working model for the regulation of NPR1 by NPR3 and NPR4 in response to different levels of SA (Supplementary Fig. 1). In the absence of pathogen challenge, NPR4 constantly removes most of the NPR1 protein by CUL3NPR4-mediated degradation. This degradation is important to prevent spurious activation of resistance. However, basal SA is required to disrupt some of the NPR1-NPR4 interactions in order to maintain the basal level of NPR1. This is critical because SA-deficient plants, eds517, ics118 and the NahG transgenic line expressing an SA-degrading enzyme19, are impaired in maintaining NPR1 homeostasis (Supplementary Fig. 9) resulting in enhanced disease susceptibility (Supplementary Fig. 1a, b). Upon challenge by pathogens that trigger ETI, SA levels increase both locally and systemically20 to form a concentration gradient from the infection site21,22. Previous research has shown that high levels of SA facilitate PCD23,24, and the spread of PCD may be controlled by activities of proteins such as LSD1 (a zinc-finger protein) and Atrboh D (an NADPH oxidase). Based on the fact that the npr3 npr4 double mutant can no longer undergo PCD in response to pathogen effectors, we propose that NPR1, which over-accumulates in the npr3 npr4 mutant, can act as a negative regulator of PCD. Our finding is in line with a previous report suggesting that NPR1 suppresses HR25. In support of this function of NPR1, the NPR1-GFP signal is the lowest inside the developing HR (Fig. 4e-h) due to CUL3NPR3-mediated degradation of NPR1 (Supplementary Fig. 1c). In neighbouring cells, the lower level of SA limits NPR1-NPR3 interaction, enabling NPR1 to accumulate in the margin of the HR to restrict the spread of PCD and establish SAR (Supplementary Fig. 1d).

Through this study, we identified the NPR1 paralogues, NPR3 and NPR4, as receptors for the immune signal SA. These receptors have different binding affinities to SA (Fig. 3), suggesting that they may be differentially responsive to spatiotemporal changes in cellular SA concentrations. SA controls accessibility of the CUL3 ligase adaptors, NPR3 and NPR4, to their substrate NPR1 (Fig. 2), thereby regulating NPR1 stability and activity (Figs. 1 and 4).

Based on our findings, we present a working model for the regulation of NPR1 by NPR3 and NPR4 in response to different levels of SA (Supplementary Fig. 1). In the absence of pathogen challenge, NPR4 constantly removes most of the NPR1 protein by CUL3NPR4-mediated degradation. This degradation is important to prevent spurious activation of resistance. However, basal SA is required to disrupt some of the NPR1-NPR4 interactions in order to maintain the basal level of NPR1. This is critical because SA-deficient plants, eds517, ics118 and the NahG transgenic line expressing an SA-degrading enzyme19, are impaired in maintaining NPR1 homeostasis (Supplementary Fig. 9) resulting in enhanced disease susceptibility (Supplementary Fig. 1a, b). Upon challenge by pathogens that trigger ETI, SA levels increase both locally and systemically20 to form a concentration gradient from the infection site21,22. Previous research has shown that high levels of SA facilitate PCD23,24, and the spread of PCD may be controlled by activities of proteins such as LSD1 (a zinc-finger protein) and Atrboh D (an NADPH oxidase). Based on the fact that the npr3 npr4 double mutant can no longer undergo PCD in response to pathogen effectors, we propose that NPR1, which over-accumulates in the npr3 npr4 mutant, can act as a negative regulator of PCD. Our finding is in line with a previous report suggesting that NPR1 suppresses HR25. In support of this function of NPR1, the NPR1-GFP signal is the lowest inside the developing HR (Fig. 4e-h) due to CUL3NPR3-mediated degradation of NPR1 (Supplementary Fig. 1c). In neighbouring cells, the lower level of SA limits NPR1-NPR3 interaction, enabling NPR1 to accumulate in the margin of the HR to restrict the spread of PCD and establish SAR (Supplementary Fig. 1d).

METHODSArabidopsis thaliana mutants and transgenic linesArabidopsis thaliana mutants (in ecotype Col-0) npr3-1, npr4-3, npr3-1 npr4-3, npr1-1 npr3-1 npr4-3, npr3-2 (SALK_043055), npr4-2 (SALK_098460), and npr3-2 npr4-2 were provided by Dr. Yuelin Zhang11. 35S:NPR1-GFP was introduced into the npr1-2 npr3-1 npr4-3 background by crossing the 35S:NPR1-GFP transgenic plants (in npr1-2) with the npr3-1 npr4-3 plants. Homozygous plants were selected by genotyping.Molecular cloning of NPRsThe coding regions of NPR1, 2, 3 and 4 were amplified with PrimeSTAR HS DNA polymerase (Takara) using specific primers containing Gateway attB sites (Supplementary Table 1) and then cloned into the pDONR207 entry vector using the BP Clonase (Invitrogen) to create the NPR entry clones. After verification by sequencing, each of the clones was mobilized using the LR Clonase (Invitrogen) into the Gateway destination vectors pDEST-GBKT7 and pDEST-GADT7 for yeast transformation28, the protein expression vectors pDEST15 (Invitrogen) and pDEST-HisMBP (Addgene plasmid 11085)29 for making the GST and His-MBP fusions, respectively.Detection of the NPR1 proteinFour-week-old plants were sprayed with 0.5 mM SA and harvested at different time points. Total protein was extracted in a buffer containing 50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 5 mM EDTA, 0.1% Triton X-100, 0.2% Nonidet P-40, and inhibitors: 50 μg/ml TPCK, 50 μg/ml TLCK, 0.6 mM PMSF, 40 μM MG11530. The homogenates were centrifuged twice at 12000 rpm for 15 min each. Protein was denatured in the SDS sample buffer containing 100 mM DTT at 75 °C for 10 min and western blotted using an antibody against NPR116.Quantitative real-time PCRTotal RNA was extracted from 4-week-old control and SA-treated plants at the indicated time points using TRIzol Reagent (Invitrogen). Genomic DNA was eliminated by treatment of the RNA with 2 units of TURBO DNA-free (Ambion). cDNA was synthesized using the Superscript III Reverse Transcription kit (Invitrogen) and analyzed by quantitative real-time PCR using the FastStart Universal SYBR Green Master (Rox) kit (Roche) with gene-specific primers for NPR1 and ubiquitin 5 (Supplementary Table 1).In vitro degradation assayNPR1 degradation assay was performed as described10. Leaves from WT or the npr3-1 npr4-3 double mutant plants were ground in liquid nitrogen and resuspended in the proteolysis buffer containing 25 mM Tris-HCl, pH 7.5, 10 mM MgCl2, 10 mM NaCl, 10 mM ATP, and 5 mM DTT. After centrifugation, the supernatants were mixed with the GST-NPR1 protein purified from E. coli and incubated at room temperature for 15 min. The reactions were stopped by adding the SDS sample buffer containing 100 mM DTT and incubated at 75 °C for 10 min. The level of the GST-NPR1 protein was analysed by western blotting using an anti-GST antibody (GE Health).In vitro pull-down assayThe coding sequence of Cullin 3A was cloned into the GST vector pGEX4T-2 (GE Health) for expression in E. coli BL21(DE3). The coding sequences of NPR3 and 4 were cloned into pCMX-PL2 for in vitro translation using TNT Quick Coupled Transcription/Translation System (Promega) to produce the HA-tagged NPR3 and 4 proteins. The purified GST-Cullin 3A protein was bound to the glutathione agarose beads, incubated with the HA-tagged NPR3 or NPR4 protein, washed three times with the EB buffer (50 mM Tris-HCl pH 7.2, 100 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1% DMSO, 20 mM DTT, and 0.1% NP40). The HA-tagged NPR3 or NPR4 protein bound to GST-Cullin 3A protein was detected by western blotting using an anti-HA antibody (GenScript). Recombinant His-MBP-tagged NPR1 and GST-tagged NPR3 and NPR4 proteins were produced in E. coli. The recombinant His-MBP-tagged NPR1 was purified using the Ni-NTA resin (QIAGEN). GST-tagged NPR3 and NPR4 proteins were purified using glutathione beads and retained on the beads to pull down purified His-MBP-NPR1 protein with or without 100 μM SA in a buffer containing 50 mM Tris-HCl, pH 6.8, 100 mM NaCl and 0.1% NP40. After washing three times, bound His-MBP-NPR1 was eluted by heating the glutathione beads at 95 °C for 10 min in the SDS buffer with 100 mM DTT and detected by western blotting using an anti-His antibody (GenScript). Equal loadings were verified by staining the membrane with Ponceau S.Co-immunoprecipitation assayThree-week-old plant sample was harvested and ground in liquid nitrogen. Protein was extracted in the extraction buffer (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 5 mM EDTA, 0.1% Triton X-100, 0.2% Nonidet P-40, and inhibitors: 50 μg/ml TPCK, 50 μg/ml TLCK, 0.6 mM PMSF, 40 μM MG115). The xtracts were then pre-cleared with 50 μl of Dynabeads Protein G (Invitrogen). After 1 μl of anti-GFP antibody (Abcam) was added to the extracts and incubated for 2 hr, 50 μl of magnetic Dynabeads were added to the samples and incubated for another hour with gentle rocking. The magnetic Dynabeads were then washed three times using the protein extraction buffer and bound proteins were eluted by heating the magnetic beads in the SDS sample buffer containing 100 mM DTT at 95 °C for 10 min. The NPR1-GFP and Cullin 3 proteins were detected by western blotting using an anti-GFP antibody (Clontech) and an anti-Cullin 3A antibody26, respectively10.Yeast two-hybrid assayThe Saccharomyces cerevisiae yeast strains AH109 and Y187 were transformed with pGADT7-NPR1, 2, 3, 4 and pGBKT7-NPR1, 2, 3, 4, respectively according to the Clontech yeast transformation protocol. Yeast strains were grown on SD-Trp-Leu plates, and then fresh single colonies were grown for one day in the SD-Trp-Leu liquid media. Interactions between bait and prey were detected on the selective media: SD-Trp-Leu-His (control), SD-Trp-Leu-His with 100 μM sodium salicylate (SA), 100 μM 2,6-dichloroisonicotinic acid (INA), or 100 μM 4-hydroxybenzoic acid (4-HBA). All of the SD −Trp-Leu-His selective media also contained 3 mM 3-aminotriazole.SA-binding assayThe SA-binding assay was performed as described6 with modifications. The GST-NPR3 and the GST-NPR4 proteins were expressed in E. coli C41 and purified using Pierce Glutathione Magnetic Beads (Thermo). The protein-bound beads were incubated in 100 μl buffer containing 30 mM sodium citrate, pH 6.3, 1 mM EDTA and [3H]-SA (American Radiolabeled Chemicals, specific activity 30 Ci/mmol). The beads were washed twice, resuspended in 100 μl H2O, mixed with 6 ml Ultima Gold™ Cocktails (PerkinElmer) and counted using the LC6000SC liquid scintillation counter (Beckman Instruments). The non-specific binding was determined in the presence of 1 mM unlabelled SA. The data were analysed using GraphPad Prism 5.Gel filtration analyses of NPR4NPR4 was overexpressed as a GST-fusion protein in insect cells and purified by glutathione affinity chromatography in the presence of 100 mM DTT. After TEV cleavage, NPR4 was further purified by anion exchange chromatography and dialyzed against a buffer containing 20 mM Tris-HCl pH 8.0, 200 mM NaCl, 5 mM DTT. Upon concentration, 0.5 – 1 mg NPR4 was analysed in the same buffer without and with 10 μM SA as indicated by size exclusion chromatography on a Supderdex 200 gel filtration column. Co-elution of SA and NPR4 was monitored by [3H]-SA, which was pre-mixed with the NPR4 protein before injection.Pathogen infectionTo test for the hypersensitive response, the avirulent pathogen Psm ES4326 carrying avrRpt2 (OD600 = 0.02) or avrRpm1 (OD600 = 0.1) and Pst carrying avrRps4 (OD600 = 0.1) or avrRpt2 (OD600 = 0.02) were infiltrated into 3 to 4-week-old leaves. The cell death was recorded 2-3 days after the infiltration. Ion leakage was recorded over time as described27. To test for SAR, two lower leaves of 3-week-old plants were pressure-infiltrated with 10 mM MgCl2 or avirulent bacterial pathogen Psm ES4326 carrying avrRpt2 (OD600 = 0.02). Three days later, virulent bacterial pathogen Psm ES4326 (OD600 = 0.001) was infiltrated into two upper leaves. Disease symptoms were monitored and bacterial growth was analysed 3 days after the inoculation10.Imaging of NPR1-GFP in infection siteThe 35S:npr1C82A-GFP plants were inoculated with Psm ES4326 carrying avrRpt2 (OD600 = 0.02) and incubated for 11 hours. Leaf tissues were mounted in 10% glycerol and viewed with a BIOREVO (Keyence, Osaka, Japan) BZ-9000 fluorescence microscope. GFP signal is monitored with an excitation wavelength of 472.5 nm and a bandpass 502.5 to 537.5 nm emission filter. Red chlorophylls were viewed with an excitation wavelength of 540 nm and a bandpass 573 to 613 nm emission filter. To obtain wide-field view (2 × 5 pictures), image stitching was performed by BZ-II Image Analysis Application. Same experiments were carried out 8 times.

Arabidopsis thaliana mutants and transgenic linesArabidopsis thaliana mutants (in ecotype Col-0) npr3-1, npr4-3, npr3-1 npr4-3, npr1-1 npr3-1 npr4-3, npr3-2 (SALK_043055), npr4-2 (SALK_098460), and npr3-2 npr4-2 were provided by Dr. Yuelin Zhang11. 35S:NPR1-GFP was introduced into the npr1-2 npr3-1 npr4-3 background by crossing the 35S:NPR1-GFP transgenic plants (in npr1-2) with the npr3-1 npr4-3 plants. Homozygous plants were selected by genotyping.

Arabidopsis thaliana mutants and transgenic lines

Arabidopsis thaliana mutants (in ecotype Col-0) npr3-1, npr4-3, npr3-1 npr4-3, npr1-1 npr3-1 npr4-3, npr3-2 (SALK_043055), npr4-2 (SALK_098460), and npr3-2 npr4-2 were provided by Dr. Yuelin Zhang11. 35S:NPR1-GFP was introduced into the npr1-2 npr3-1 npr4-3 background by crossing the 35S:NPR1-GFP transgenic plants (in npr1-2) with the npr3-1 npr4-3 plants. Homozygous plants were selected by genotyping.

Molecular cloning of NPRsThe coding regions of NPR1, 2, 3 and 4 were amplified with PrimeSTAR HS DNA polymerase (Takara) using specific primers containing Gateway attB sites (Supplementary Table 1) and then cloned into the pDONR207 entry vector using the BP Clonase (Invitrogen) to create the NPR entry clones. After verification by sequencing, each of the clones was mobilized using the LR Clonase (Invitrogen) into the Gateway destination vectors pDEST-GBKT7 and pDEST-GADT7 for yeast transformation28, the protein expression vectors pDEST15 (Invitrogen) and pDEST-HisMBP (Addgene plasmid 11085)29 for making the GST and His-MBP fusions, respectively.

Molecular cloning of NPRs

The coding regions of NPR1, 2, 3 and 4 were amplified with PrimeSTAR HS DNA polymerase (Takara) using specific primers containing Gateway attB sites (Supplementary Table 1) and then cloned into the pDONR207 entry vector using the BP Clonase (Invitrogen) to create the NPR entry clones. After verification by sequencing, each of the clones was mobilized using the LR Clonase (Invitrogen) into the Gateway destination vectors pDEST-GBKT7 and pDEST-GADT7 for yeast transformation28, the protein expression vectors pDEST15 (Invitrogen) and pDEST-HisMBP (Addgene plasmid 11085)29 for making the GST and His-MBP fusions, respectively.

Detection of the NPR1 proteinFour-week-old plants were sprayed with 0.5 mM SA and harvested at different time points. Total protein was extracted in a buffer containing 50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 5 mM EDTA, 0.1% Triton X-100, 0.2% Nonidet P-40, and inhibitors: 50 μg/ml TPCK, 50 μg/ml TLCK, 0.6 mM PMSF, 40 μM MG11530. The homogenates were centrifuged twice at 12000 rpm for 15 min each. Protein was denatured in the SDS sample buffer containing 100 mM DTT at 75 °C for 10 min and western blotted using an antibody against NPR116.

Detection of the NPR1 protein

Four-week-old plants were sprayed with 0.5 mM SA and harvested at different time points. Total protein was extracted in a buffer containing 50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 5 mM EDTA, 0.1% Triton X-100, 0.2% Nonidet P-40, and inhibitors: 50 μg/ml TPCK, 50 μg/ml TLCK, 0.6 mM PMSF, 40 μM MG11530. The homogenates were centrifuged twice at 12000 rpm for 15 min each. Protein was denatured in the SDS sample buffer containing 100 mM DTT at 75 °C for 10 min and western blotted using an antibody against NPR116.

Quantitative real-time PCRTotal RNA was extracted from 4-week-old control and SA-treated plants at the indicated time points using TRIzol Reagent (Invitrogen). Genomic DNA was eliminated by treatment of the RNA with 2 units of TURBO DNA-free (Ambion). cDNA was synthesized using the Superscript III Reverse Transcription kit (Invitrogen) and analyzed by quantitative real-time PCR using the FastStart Universal SYBR Green Master (Rox) kit (Roche) with gene-specific primers for NPR1 and ubiquitin 5 (Supplementary Table 1).

Quantitative real-time PCR

Total RNA was extracted from 4-week-old control and SA-treated plants at the indicated time points using TRIzol Reagent (Invitrogen). Genomic DNA was eliminated by treatment of the RNA with 2 units of TURBO DNA-free (Ambion). cDNA was synthesized using the Superscript III Reverse Transcription kit (Invitrogen) and analyzed by quantitative real-time PCR using the FastStart Universal SYBR Green Master (Rox) kit (Roche) with gene-specific primers for NPR1 and ubiquitin 5 (Supplementary Table 1).

In vitro degradation assayNPR1 degradation assay was performed as described10. Leaves from WT or the npr3-1 npr4-3 double mutant plants were ground in liquid nitrogen and resuspended in the proteolysis buffer containing 25 mM Tris-HCl, pH 7.5, 10 mM MgCl2, 10 mM NaCl, 10 mM ATP, and 5 mM DTT. After centrifugation, the supernatants were mixed with the GST-NPR1 protein purified from E. coli and incubated at room temperature for 15 min. The reactions were stopped by adding the SDS sample buffer containing 100 mM DTT and incubated at 75 °C for 10 min. The level of the GST-NPR1 protein was analysed by western blotting using an anti-GST antibody (GE Health).

In vitro degradation assay

NPR1 degradation assay was performed as described10. Leaves from WT or the npr3-1 npr4-3 double mutant plants were ground in liquid nitrogen and resuspended in the proteolysis buffer containing 25 mM Tris-HCl, pH 7.5, 10 mM MgCl2, 10 mM NaCl, 10 mM ATP, and 5 mM DTT. After centrifugation, the supernatants were mixed with the GST-NPR1 protein purified from E. coli and incubated at room temperature for 15 min. The reactions were stopped by adding the SDS sample buffer containing 100 mM DTT and incubated at 75 °C for 10 min. The level of the GST-NPR1 protein was analysed by western blotting using an anti-GST antibody (GE Health).

In vitro pull-down assayThe coding sequence of Cullin 3A was cloned into the GST vector pGEX4T-2 (GE Health) for expression in E. coli BL21(DE3). The coding sequences of NPR3 and 4 were cloned into pCMX-PL2 for in vitro translation using TNT Quick Coupled Transcription/Translation System (Promega) to produce the HA-tagged NPR3 and 4 proteins. The purified GST-Cullin 3A protein was bound to the glutathione agarose beads, incubated with the HA-tagged NPR3 or NPR4 protein, washed three times with the EB buffer (50 mM Tris-HCl pH 7.2, 100 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1% DMSO, 20 mM DTT, and 0.1% NP40). The HA-tagged NPR3 or NPR4 protein bound to GST-Cullin 3A protein was detected by western blotting using an anti-HA antibody (GenScript). Recombinant His-MBP-tagged NPR1 and GST-tagged NPR3 and NPR4 proteins were produced in E. coli. The recombinant His-MBP-tagged NPR1 was purified using the Ni-NTA resin (QIAGEN). GST-tagged NPR3 and NPR4 proteins were purified using glutathione beads and retained on the beads to pull down purified His-MBP-NPR1 protein with or without 100 μM SA in a buffer containing 50 mM Tris-HCl, pH 6.8, 100 mM NaCl and 0.1% NP40. After washing three times, bound His-MBP-NPR1 was eluted by heating the glutathione beads at 95 °C for 10 min in the SDS buffer with 100 mM DTT and detected by western blotting using an anti-His antibody (GenScript). Equal loadings were verified by staining the membrane with Ponceau S.

In vitro pull-down assay

The coding sequence of Cullin 3A was cloned into the GST vector pGEX4T-2 (GE Health) for expression in E. coli BL21(DE3). The coding sequences of NPR3 and 4 were cloned into pCMX-PL2 for in vitro translation using TNT Quick Coupled Transcription/Translation System (Promega) to produce the HA-tagged NPR3 and 4 proteins. The purified GST-Cullin 3A protein was bound to the glutathione agarose beads, incubated with the HA-tagged NPR3 or NPR4 protein, washed three times with the EB buffer (50 mM Tris-HCl pH 7.2, 100 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1% DMSO, 20 mM DTT, and 0.1% NP40). The HA-tagged NPR3 or NPR4 protein bound to GST-Cullin 3A protein was detected by western blotting using an anti-HA antibody (GenScript). Recombinant His-MBP-tagged NPR1 and GST-tagged NPR3 and NPR4 proteins were produced in E. coli. The recombinant His-MBP-tagged NPR1 was purified using the Ni-NTA resin (QIAGEN). GST-tagged NPR3 and NPR4 proteins were purified using glutathione beads and retained on the beads to pull down purified His-MBP-NPR1 protein with or without 100 μM SA in a buffer containing 50 mM Tris-HCl, pH 6.8, 100 mM NaCl and 0.1% NP40. After washing three times, bound His-MBP-NPR1 was eluted by heating the glutathione beads at 95 °C for 10 min in the SDS buffer with 100 mM DTT and detected by western blotting using an anti-His antibody (GenScript). Equal loadings were verified by staining the membrane with Ponceau S.

Co-immunoprecipitation assayThree-week-old plant sample was harvested and ground in liquid nitrogen. Protein was extracted in the extraction buffer (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 5 mM EDTA, 0.1% Triton X-100, 0.2% Nonidet P-40, and inhibitors: 50 μg/ml TPCK, 50 μg/ml TLCK, 0.6 mM PMSF, 40 μM MG115). The xtracts were then pre-cleared with 50 μl of Dynabeads Protein G (Invitrogen). After 1 μl of anti-GFP antibody (Abcam) was added to the extracts and incubated for 2 hr, 50 μl of magnetic Dynabeads were added to the samples and incubated for another hour with gentle rocking. The magnetic Dynabeads were then washed three times using the protein extraction buffer and bound proteins were eluted by heating the magnetic beads in the SDS sample buffer containing 100 mM DTT at 95 °C for 10 min. The NPR1-GFP and Cullin 3 proteins were detected by western blotting using an anti-GFP antibody (Clontech) and an anti-Cullin 3A antibody26, respectively10.

Co-immunoprecipitation assay

Three-week-old plant sample was harvested and ground in liquid nitrogen. Protein was extracted in the extraction buffer (50 mM Tris-HCl, pH 7.5, 150 mM NaCl, 5 mM EDTA, 0.1% Triton X-100, 0.2% Nonidet P-40, and inhibitors: 50 μg/ml TPCK, 50 μg/ml TLCK, 0.6 mM PMSF, 40 μM MG115). The xtracts were then pre-cleared with 50 μl of Dynabeads Protein G (Invitrogen). After 1 μl of anti-GFP antibody (Abcam) was added to the extracts and incubated for 2 hr, 50 μl of magnetic Dynabeads were added to the samples and incubated for another hour with gentle rocking. The magnetic Dynabeads were then washed three times using the protein extraction buffer and bound proteins were eluted by heating the magnetic beads in the SDS sample buffer containing 100 mM DTT at 95 °C for 10 min. The NPR1-GFP and Cullin 3 proteins were detected by western blotting using an anti-GFP antibody (Clontech) and an anti-Cullin 3A antibody26, respectively10.

Yeast two-hybrid assayThe Saccharomyces cerevisiae yeast strains AH109 and Y187 were transformed with pGADT7-NPR1, 2, 3, 4 and pGBKT7-NPR1, 2, 3, 4, respectively according to the Clontech yeast transformation protocol. Yeast strains were grown on SD-Trp-Leu plates, and then fresh single colonies were grown for one day in the SD-Trp-Leu liquid media. Interactions between bait and prey were detected on the selective media: SD-Trp-Leu-His (control), SD-Trp-Leu-His with 100 μM sodium salicylate (SA), 100 μM 2,6-dichloroisonicotinic acid (INA), or 100 μM 4-hydroxybenzoic acid (4-HBA). All of the SD −Trp-Leu-His selective media also contained 3 mM 3-aminotriazole.

Yeast two-hybrid assay

The Saccharomyces cerevisiae yeast strains AH109 and Y187 were transformed with pGADT7-NPR1, 2, 3, 4 and pGBKT7-NPR1, 2, 3, 4, respectively according to the Clontech yeast transformation protocol. Yeast strains were grown on SD-Trp-Leu plates, and then fresh single colonies were grown for one day in the SD-Trp-Leu liquid media. Interactions between bait and prey were detected on the selective media: SD-Trp-Leu-His (control), SD-Trp-Leu-His with 100 μM sodium salicylate (SA), 100 μM 2,6-dichloroisonicotinic acid (INA), or 100 μM 4-hydroxybenzoic acid (4-HBA). All of the SD −Trp-Leu-His selective media also contained 3 mM 3-aminotriazole.

SA-binding assayThe SA-binding assay was performed as described6 with modifications. The GST-NPR3 and the GST-NPR4 proteins were expressed in E. coli C41 and purified using Pierce Glutathione Magnetic Beads (Thermo). The protein-bound beads were incubated in 100 μl buffer containing 30 mM sodium citrate, pH 6.3, 1 mM EDTA and [3H]-SA (American Radiolabeled Chemicals, specific activity 30 Ci/mmol). The beads were washed twice, resuspended in 100 μl H2O, mixed with 6 ml Ultima Gold™ Cocktails (PerkinElmer) and counted using the LC6000SC liquid scintillation counter (Beckman Instruments). The non-specific binding was determined in the presence of 1 mM unlabelled SA. The data were analysed using GraphPad Prism 5.

SA-binding assay

The SA-binding assay was performed as described6 with modifications. The GST-NPR3 and the GST-NPR4 proteins were expressed in E. coli C41 and purified using Pierce Glutathione Magnetic Beads (Thermo). The protein-bound beads were incubated in 100 μl buffer containing 30 mM sodium citrate, pH 6.3, 1 mM EDTA and [3H]-SA (American Radiolabeled Chemicals, specific activity 30 Ci/mmol). The beads were washed twice, resuspended in 100 μl H2O, mixed with 6 ml Ultima Gold™ Cocktails (PerkinElmer) and counted using the LC6000SC liquid scintillation counter (Beckman Instruments). The non-specific binding was determined in the presence of 1 mM unlabelled SA. The data were analysed using GraphPad Prism 5.

Gel filtration analyses of NPR4NPR4 was overexpressed as a GST-fusion protein in insect cells and purified by glutathione affinity chromatography in the presence of 100 mM DTT. After TEV cleavage, NPR4 was further purified by anion exchange chromatography and dialyzed against a buffer containing 20 mM Tris-HCl pH 8.0, 200 mM NaCl, 5 mM DTT. Upon concentration, 0.5 – 1 mg NPR4 was analysed in the same buffer without and with 10 μM SA as indicated by size exclusion chromatography on a Supderdex 200 gel filtration column. Co-elution of SA and NPR4 was monitored by [3H]-SA, which was pre-mixed with the NPR4 protein before injection.

Gel filtration analyses of NPR4

NPR4 was overexpressed as a GST-fusion protein in insect cells and purified by glutathione affinity chromatography in the presence of 100 mM DTT. After TEV cleavage, NPR4 was further purified by anion exchange chromatography and dialyzed against a buffer containing 20 mM Tris-HCl pH 8.0, 200 mM NaCl, 5 mM DTT. Upon concentration, 0.5 – 1 mg NPR4 was analysed in the same buffer without and with 10 μM SA as indicated by size exclusion chromatography on a Supderdex 200 gel filtration column. Co-elution of SA and NPR4 was monitored by [3H]-SA, which was pre-mixed with the NPR4 protein before injection.

Pathogen infectionTo test for the hypersensitive response, the avirulent pathogen Psm ES4326 carrying avrRpt2 (OD600 = 0.02) or avrRpm1 (OD600 = 0.1) and Pst carrying avrRps4 (OD600 = 0.1) or avrRpt2 (OD600 = 0.02) were infiltrated into 3 to 4-week-old leaves. The cell death was recorded 2-3 days after the infiltration. Ion leakage was recorded over time as described27. To test for SAR, two lower leaves of 3-week-old plants were pressure-infiltrated with 10 mM MgCl2 or avirulent bacterial pathogen Psm ES4326 carrying avrRpt2 (OD600 = 0.02). Three days later, virulent bacterial pathogen Psm ES4326 (OD600 = 0.001) was infiltrated into two upper leaves. Disease symptoms were monitored and bacterial growth was analysed 3 days after the inoculation10.

Pathogen infection

To test for the hypersensitive response, the avirulent pathogen Psm ES4326 carrying avrRpt2 (OD600 = 0.02) or avrRpm1 (OD600 = 0.1) and Pst carrying avrRps4 (OD600 = 0.1) or avrRpt2 (OD600 = 0.02) were infiltrated into 3 to 4-week-old leaves. The cell death was recorded 2-3 days after the infiltration. Ion leakage was recorded over time as described27. To test for SAR, two lower leaves of 3-week-old plants were pressure-infiltrated with 10 mM MgCl2 or avirulent bacterial pathogen Psm ES4326 carrying avrRpt2 (OD600 = 0.02). Three days later, virulent bacterial pathogen Psm ES4326 (OD600 = 0.001) was infiltrated into two upper leaves. Disease symptoms were monitored and bacterial growth was analysed 3 days after the inoculation10.

Imaging of NPR1-GFP in infection siteThe 35S:npr1C82A-GFP plants were inoculated with Psm ES4326 carrying avrRpt2 (OD600 = 0.02) and incubated for 11 hours. Leaf tissues were mounted in 10% glycerol and viewed with a BIOREVO (Keyence, Osaka, Japan) BZ-9000 fluorescence microscope. GFP signal is monitored with an excitation wavelength of 472.5 nm and a bandpass 502.5 to 537.5 nm emission filter. Red chlorophylls were viewed with an excitation wavelength of 540 nm and a bandpass 573 to 613 nm emission filter. To obtain wide-field view (2 × 5 pictures), image stitching was performed by BZ-II Image Analysis Application. Same experiments were carried out 8 times.

Imaging of NPR1-GFP in infection site

The 35S:npr1C82A-GFP plants were inoculated with Psm ES4326 carrying avrRpt2 (OD600 = 0.02) and incubated for 11 hours. Leaf tissues were mounted in 10% glycerol and viewed with a BIOREVO (Keyence, Osaka, Japan) BZ-9000 fluorescence microscope. GFP signal is monitored with an excitation wavelength of 472.5 nm and a bandpass 502.5 to 537.5 nm emission filter. Red chlorophylls were viewed with an excitation wavelength of 540 nm and a bandpass 573 to 613 nm emission filter. To obtain wide-field view (2 × 5 pictures), image stitching was performed by BZ-II Image Analysis Application. Same experiments were carried out 8 times.

Supplementary Material12

Supplementary Material
