---
pmid: '14562106'
title: Global analysis of protein expression in yeast.
authors:
- Ghaemmaghami S
- Huh WK
- Bower K
- Howson RW
- Belle A
- Dephoure N
- O'Shea EK
- Weissman JS
journal: Nature
year: '2003'
full_text_available: false
doi: 10.1038/nature02046
---

# Global analysis of protein expression in yeast.
**Authors:** Ghaemmaghami S, Huh WK, Bower K, Howson RW, Belle A, Dephoure N, O'Shea EK, Weissman JS
**Journal:** Nature (2003)
**DOI:** [10.1038/nature02046](https://doi.org/10.1038/nature02046)

## Abstract

1. Nature. 2003 Oct 16;425(6959):737-41. doi: 10.1038/nature02046.

Global analysis of protein expression in yeast.

Ghaemmaghami S(1), Huh WK, Bower K, Howson RW, Belle A, Dephoure N, O'Shea EK, 
Weissman JS.

Author information:
(1)Howard Hughes Medical Institute, University of California-San Francisco, San 
Francisco, California 94143-2240, USA.

Comment in
    Nature. 2003 Oct 16;425(6959):671-2. doi: 10.1038/425671a.

The availability of complete genomic sequences and technologies that allow 
comprehensive analysis of global expression profiles of messenger RNA have 
greatly expanded our ability to monitor the internal state of a cell. Yet 
biological systems ultimately need to be explained in terms of the activity, 
regulation and modification of proteins--and the ubiquitous occurrence of 
post-transcriptional regulation makes mRNA an imperfect proxy for such 
information. To facilitate global protein analyses, we have created a 
Saccharomyces cerevisiae fusion library where each open reading frame is tagged 
with a high-affinity epitope and expressed from its natural chromosomal 
location. Through immunodetection of the common tag, we obtain a census of 
proteins expressed during log-phase growth and measurements of their absolute 
levels. We find that about 80% of the proteome is expressed during normal growth 
conditions, and, using additional sequence information, we systematically 
identify misannotated genes. The abundance of proteins ranges from fewer than 50 
to more than 10(6) molecules per cell. Many of these molecules, including 
essential proteins and most transcription factors, are present at levels that 
are not readily detectable by other proteomic techniques nor predictable by mRNA 
levels or codon bias measurements.

DOI: 10.1038/nature02046
PMID: 14562106 [Indexed for MEDLINE]
