---
pmid: '18724357'
title: STING is an endoplasmic reticulum adaptor that facilitates innate immune signalling.
authors:
- Ishikawa H
- Barber GN
journal: Nature
year: '2008'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2804933
doi: 10.1038/nature07317
---

# STING is an endoplasmic reticulum adaptor that facilitates innate immune signalling.
**Authors:** Ishikawa H, Barber GN
**Journal:** Nature (2008)
**DOI:** [10.1038/nature07317](https://doi.org/10.1038/nature07317)
**PMC:** [PMC2804933](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2804933/)

## Abstract

1. Nature. 2008 Oct 2;455(7213):674-8. doi: 10.1038/nature07317. Epub 2008 Aug
24.

STING is an endoplasmic reticulum adaptor that facilitates innate immune 
signalling.

Ishikawa H(1), Barber GN.

Author information:
(1)Department of Medicine and Sylvester Comprehensive Cancer Center, University 
of Miami School of Medicine, Miami, Florida 33136, USA.

Erratum in
    Nature. 2008 Nov 13;456(7219):274.

The cellular innate immune system is essential for recognizing pathogen 
infection and for establishing effective host defence. But critical molecular 
determinants responsible for facilitating an appropriate immune 
response-following infection with DNA and RNA viruses, for example-remain to be 
identified. Here we report the identification, following expression cloning, of 
a molecule (STING; stimulator of interferon genes) that appears essential for 
effective innate immune signalling processes. It comprises five putative 
transmembrane regions, predominantly resides in the endoplasmic reticulum and is 
able to activate both NF-kappaB and IRF3 transcription pathways to induce 
expression of type I interferon (IFN-alpha and IFN-beta ) and exert a potent 
anti-viral state following expression. In contrast, loss of STING rendered 
murine embryonic fibroblasts extremely susceptible to negative-stranded virus 
infection, including vesicular stomatitis virus. Further, STING ablation 
abrogated the ability of intracellular B-form DNA, as well as members of the 
herpesvirus family, to induce IFN-beta, but did not significantly affect the 
Toll-like receptor (TLR) pathway. Yeast two-hybrid and co-immunoprecipitation 
studies indicated that STING interacts with RIG-I and with SSR2 (also known as 
TRAPbeta), which is a member of the translocon-associated protein (TRAP) complex 
required for protein translocation across the endoplasmic reticulum membrane 
following translation. Ablation by RNA interference of both TRAPbeta and 
translocon adaptor SEC61beta was subsequently found to inhibit STING's ability 
to stimulate expression of IFN-beta. Thus, as well as identifying a regulator of 
innate immune signalling, our results imply a potential role for the translocon 
in innate signalling pathways activated by select viruses as well as 
intracellular DNA.

DOI: 10.1038/nature07317
PMCID: PMC2804933
PMID: 18724357 [Indexed for MEDLINE]

## Full Text

Cellular host defense responses to pathogen invasion principally involves the detection of pathogen associated molecular patterns (PAMPs) such as viral nucleic acid or bacterial cell wall components including lipopolysaccharide or flagellar proteins that results in the induction of anti-pathogen genes[3–9]. For example, viral RNA can be detected by membrane bound Toll-like receptors (TLR’s) present in the endoplasmic reticulum (ER) and/or endosomes (e.g. TLR 3 and 7/8) or by TLR-independent intracellular DExD/H box RNA helicases referred to as retinoic acid inducible gene 1 (RIG-I) or melanoma differentiation associated antigen 5 (MDA5, also referred to as IFIH1 and helicard)[3–10]. Pathogen DNA can be recognized by TLR 9 present in plasmacytoid dendritic cells, although it is now apparent that important TLR-independent pathways also exist to recognize DNA in alternate tissue, the mechanisms of action of which remain to be determined [3–10]. These events culminate in the activation of downstream signaling events, leading to the transcription of NF-κB and IRF3/7-dependent genes, including type I IFN.

To further determine the mechanisms of innate immune signaling, we employed an expression screening system in which approximately 5,500 human and 9,000 murine full length cDNA’s were individually transfected into 293T cells harboring a luciferase gene under control of the IFNβ promoter (IFNβ-Luc). Five genes whose overexpression lead to the significant induction of IFNβ-Luc was found to be IPS-1 (also referred to as VISA/CARDIF/MAVS) (Supplemental Fig. 1)[11–14]. However, we also isolated a previously uncharacterized molecule (gi:38093659/NP_938023/2610307O08RIK) which we referred to as STING (for STimulator of INterferon Genes) that harbored 5 predicted TM motifs (in humans) and existed as a 379 amino acid protein in human cells and 378 amino acids in murine cells (Fig. 1a and Supplemental Fig 1). A putative signal cleavage motif was found to exist at position 1–36 and a leucine rich region was apparent between amino acids 21–139 (Fig. 1a). The predicted molecular weight of human STING (hSTING) was 42,192 Da, which approximately corresponded to its observed molecular weight in human 293 cells following immunoblot analysis using a rabbit antiserum raised to a STING peptide (Fig. 1b). RNAi studies confirmed that the observed 42kDa band was indeed STING (Fig. 1b). STING was found to be ubiquitously expressed in a variety of tissues as determined by northern analysis and was found to predominantly reside in the ER region of the cell as determined by confocal microscopy and fractionation analysis (Fig. 1c–e).

Overexpression of STING in 293T cells was subsequently confirmed to robustly induce the expression of the IFN promoter (IFNβ-Luc) up to 400-fold, but not a control TK promoter driving luciferase (pRL-TK), interferon regulatory factor 3 (IRF3) responsive promoters (PRD-III-I-Luc) up to 1000-fold, an NF-κB responsive promoter (NF-κB-Luc) 12-fold and interferon-inducible promoters (interferon sensitive response element- ISRE-Luc) up to 800-fold (Fig. 2a–d). STING did not activate control promoters driving luciferase reporters such as those derived from the Rb, p53 or E2F genes (Supplemental Fig. 2). Increased dimerization of IRF3 was also observed in STING expressing 293T cells confirming that STING regulates the induction of type I IFN at upstream of IRF3 activation, dimerization and translocation (Supplemental Fig. 2)[15]. Endogenous IFNβ mRNA and IFNβ protein was induced significantly in MEFs transiently transfected with STING (Fig. 2e,f). DNA microarray analysis of STING expression in 293T cells further emphasized STING’s ability to induce primary innate immune response genes (Fig. 2g). Accordingly, MEFs cells expressing STING or IPS-1 were significantly resistant to VSV infection (Fig. 2h,i). Subsequent analysis indicated that STING function was ablated in the absence of the IκB kinase family member TBK-1, confirming that STING’s activity involved activation of IRF3 and was indeed upstream of this kinase (Fig. 2j)[15]. Finally, we observed that STING did not exert robust activity in the absence of FADD, which has also been shown to be important for efficient innate immune signaling processes (Supplemental Fig. 2)[16]. We also established that while STING, HA tagged at the carboxyl region, retained activity, this activity was lost when STING was tagged at the amino terminus or carboxyl terminus with GFP (data not shown). Subsequent analysis indicated that the amino terminus region of STING containing the 5 putative TM regions (amino acids 1–230) or just the carboxyl region of STING (amino acids 173–379) did not exhibit significant ability, alone, to induce the IFNβ-Luc promoter (Fig. 2k,l). Thus, full length intact STING is required for efficient function. However, we further observed that the carboxyl region of STING exerted a dominant-negative inhibitory effect and could impede the ability of full-length STING to stimulate IFNβ-Luc (Fig. 2m). Collectively, this data indicates that expression of STING activates the innate immune response including type I IFN leading to the induction of an antiviral state.

To further analyze STING function, we utilized an RNAi approach to ablate STING in a number of cell-types. Our data indicated that knockdown of STING in HEK 293 cells, modestly reduced the ability of the negative-stranded rhabdovirus VSV-GFP to induce IFNβ, presumably since these viruses are only weak activators of IFNβ (Supplemental Fig. 3). However, such cells were rendered extremely susceptible to virus infection and replication. To confirm a requirement for STING in the regulation of type I IFN induction and in host defense, we generated STING deficient (STING −/−) mice by targeted homologous recombination in ES cells (Supplemental Fig 4). STING −/− animals were born at the Mendelian ratio and developed and bred normally. Accordingly, MEFs from wild type and STING −/− animals were infected with VSV-GFP at varying MOI’s (0.01–1) for up to 36 hours post infection. This study confirmed that more progeny virus was produced in MEFs lacking STING compared to controls (2 logs; 24–36 hours) (Fig. 3a–d). This data was verified using VSV expressing a luciferase reporter gene and VSV-ΔM, which exhibits a defect in the viral matrix protein normally responsible for inhibiting cellular mRNA export from the nucleus[17](Supplemental Fig. 4 and Fig. 3e). Reconstitution of STING to STING −/− MEF’s rescued the susceptibility to VSV infection (Supplemental. Fig. 4). Similar analysis also indicated that loss of STING also reduced the ability of SeV to induce IFNβ (Fig 3f). In contrast, we did not observe a strong requirement for STING to mediate the ability of transfected poly I:C to induce IFNβ induction, which is largely governed by the intracellular RIG-I homologue MDA5[18] (Supplemental Fig. 5). We also observed that the positive-stranded encephalomyocarditis virus (EMCV), a member of the picornavirus family, did not effectively induce IFNβ nor replicate differently regardless of the presence of STING (Supplemental Fig. 5). Thus, we conclude that STING may play a more predominant role in facilitating RIG-I mediated innate signaling rather than MDA5. Interestingly, we did notice a significant defect (>5-fold) in the ability of transfected B-form DNA (poly dA-dT) to induce IFNβ in MEFs lacking STING compared to controls (Fig 3g,h). More strikingly, the non CpG containing interferon stimulatory DNA (ISD) was completely unable to induce IFNβ in STING −/− MEFs, as was the DNA virus herpes simplex virus 1 and bacteria Listeria monocytogenes (Fig. 3i). TLR9 is considered to govern CpG DNA-mediated induction of IFNβ, but is not active in MEFs[19, 20]. Thus, it is plausible that STING may function in TLR9-independent, DNA-mediated induction of type I IFN. This effect was similarly observed in murine STING lacking bone marrow derived macrophages (BMDM) or bone marrow derived dendritic cells (GM-DC) cultured using granulocyte-monocyte colony stimulating factor (GM-CSF) [Supplemental Fig. 6]. However, no significant difference was observed in the ability of exogenous poly I:C or lipopolysaccharide (LPS) to induce IFNβ, when comparing STING −/− BMDM or GM-DC’s to controls, events which depend on TLR 3 and 4 respectively. While, loss of STING rendered MEFs highly susceptible to VSV-GFP, less susceptibility was observed following VSV-GFP infection of STING −/−GM-DC’s or BMDM’s, indicating that STING may be more important in facilitating negative-stranded virus-mediated innate signaling in fibroblasts. Collectively, our data would indicate that loss of STING leads to a defect in RIG-I mediated type I IFN induction but does not affect the TLR pathway. In addition, we report that STING functions in the pathway utilized by intracellular B-form DNA to induce IFNβ.

To further examine the mechanisms of STINGs function in innate signaling, we attempted to determine if STING interacted with RIG-I and/or MDA5, which are putative innate immune signaling receptors for negative or positive stranded viral RNA respectively[3, 4, 10]. Co-immunoprecipitation experiments in 293T cells principally indicated that FLAG-tagged RIG-I but not MDA5 could associate with HA-tagged STING in co-transfection experiments (Fig 4a). The binding of RIG-I to STING was augmented upon infection of 293T cells with SeV (Fig 4a). We confirmed in normal human umbilical vein endothelial cells (HUVECs) that endogenous RIG-I could associate directly, or indirectly as a complex, with endogenous STING (Fig 4b). This data would be in agreement with previous data indicating that STING seems to preferentially modulate the RIG-I, rather than the MDA5 regulated pathway (Fig. 3d and Supplemental Fig. 5). We also determined that the CARD domains of RIG-I (amino acids 1-284) were preferentially able to associate with HA-STING in transfected 293T cells (Fig. 4c). RIG-I was subsequently shown to colocalize with STING in the co-transfected 293T cells (Fig. 4d). It was similarly observed that the carboxyl region of STING could inhibit the function of RIG-I, again indicating that this region of STING can exhibit a dominant-inhibitory effect (Fig 4e). We also noted association of the RIG-I downstream adaptor IPS-1 with STING although similar to the situation with RIG-I, it is not yet clear whether IPS-1 directly interacts with STING or exists as a complex with RIG-I/STING (Supplemental Fig 7). Accordingly, we observed that STING was able to induce the induction of an IFNβ driven luciferase construct in MEFs lacking RIG-I or IPS-1, likely confirming that that STING functions downstream of these latter molecules (Supplemental Fig 8). The ability of ΔRIG-I or IPS-1 to induce IFNβ appeared diminished in STING −/− MEFs (Fig. 4f). However, there was also a marked reduction in the induction of IFNβ by all transfected plasmids (including vector alone) in the absence of STING compared to control MEFs, since endogenous DNA innate signaling pathways are likely defective (Fig. 3g–i). This data indicates that STING may be an important downstream adaptor molecule that facilitates RIG-I and perhaps IPS-1 function.

To shed further insight into the molecular mechanisms of STING action, we screened an IFN-induced, human fibroblast yeast two-hybrid cDNA library using STING (amino acids 173–379) as a bait and repeatedly isolated Ssr2/TRAPβ, a member of the TRAP complex comprising four subunits (α-Δ) that facilitates translocation of proteins into the ER following translation (Fig. 4g)[1, 2]. The TRAP complex is known to associate with the translocon, comprising three subunits, Sec61 α, β and γ. Given this information, we confirmed that TRAPβ can indeed associate with endogenous STING in HEK 293 cells following co-immunoprecipitation experiments (Fig. 4h). Using this approach, we confirmed that TRAPβ also co-immunoprecipitated with endogenous Sec61β (Fig. 4h). We next verified that STING could also associate not only with TRAPβ but also Sec61β likely as a complex (Fig 4i). STING was also observed to colocalize with TRAPβ in the ER region of the cell (Fig. 4j). Indeed, loss of TRAPβ or SEC61β reduced STING’s, ability to induce an IFNβ promoter driving luciferase (Fig 4k; Supplemental Fig. 9). Taken together, this data would indicate that the STING may be involved in translocon function and that the translocon may be able to influence the induction of type I IFN.

Thus, STING is predominantly an ER resident protein that may link RIG-I and DNA-mediated intracellular innate signaling to the translocon. Speculatively, RIG-I may detect translating viral RNA’s at the intersection of ribosome/ER translocon association and/or with mitochondrial associated ER membranes (MAM’s) and require STING to exert effective function[2]. Alternatively, STING may participate in mediating ER stress response pathways an event that remains to be verified. While it is not clear how signaling from the translocon to IRF-3/NF-kB occurs, it has recently been established that translocon physically associates with the exocyst - the octomeric Sec6-Sec8 complex that also associates with the ER and tethers secretory vesicles to membranes, and facilitates protein synthesis and secretion[21, 22]. Recently, the exocyst complex was found to recruit and activate TBK1 and play a role in type I IFN-β induction[23]. Our preliminary analysis indicates that STING also co-immunprecipitates with TBK1 and that RNAi ablation of Sec5 also rendered cells defective in the STING function (Fig. 4l). Thus, STING may facilitate the detection of intracellular viral RNA species as well as B-form DNA indicating convergence of these intracellular PAMP recognition pathways.

Methods SummaryPlasmid constructsHuman STING (hSTING), murine STING (mSTING), hSTING-N (amino acids 1–230) and hSTING-C (amino acids 173–379) sequences were amplified by PCR and were cloned into pcDNA3 plasmids to generate C-terminally HA-tagged expression constructs. Expression plasmids encoding Flag-tagged RIG-I, ΔRIG-I (amino acids 1–284), IPS-1 and TBK-1 were described previously[24]. GFP-tagged RIG-I and TBK-1 were generated by cloning into pAcGFP1-C1 (Clontech, CA). Other plasmids were obtained as follows: p110-Luc and PRD-III-I-Luc (T. Maniatis), IFNβ-Luc and ISRE-Luc (J. Hiscott), NFκB-Luc (Stratagene, CA), ER-dsRED and Golgi-dsRED (Clontech, CA).AntibodiesRabbit polyclonal antibody against a synthetic peptide corresponding to residues 324–340 of human STING was obtained from EvoQuest Custom Antibody Services (Invitrogen, CA). Other antibodies were obtained from following sources: SEC61β (Upstate, NY), β-Actin, HA, FLAG (Sigma, MO), COX IV, Calreticulin, IRF3 and RIG-I (Abcam, MA).Confocal microscopyER-DsRED and Golgi-DsRED (Clontech, CA) were used for ER and Golgi marker, respectively. For mitochondria staining, living cells were incubated with 300 nM of Mito Tracker Red (Invitrogen, CA) for 45 min at 37°C.RNA interferenceChemically synthesized 21-nucleotide siRNA duplexes were obtained from Dharmacon, Inc (Lafayette, CO). RNA oligonucleotides used for human and murine STING were as follows: hSTING, GCAUCAAGGAUCGGGUUU; mSTING, CCAACAGCGUCUACGAGA.Generation of the STING knockout miceThe linearized targeting vector was electroporated into E14.1 ES cells originated from 129SvEv strain, followed by the selection in G418. One positive clone was injected into C57BL/6J blastocysts and STING −/− mice generated on a C57BL/6J background.Yeast two-hybrid analysesTo screen for interacting partners of STING a Yeast two Hybrid approach was utilized, adopting a novel IFN-induced, telomerase immortalized human fibroblast cDNA library, generated by our laboratory. hSTING-C (amino acids 173–379) sequence was amplified by PCR and was cloned into pGBK-T7 (Clontech) to generate a bait plasmid pGBK-T7-hSTING-C. 100μg of the IFN-induced library’s plasmid DNA was used to perform a library-scale transformation of yeast strain Y187.StatisticsStudents t-test was used to analyze data.

Methods Summary

Plasmid constructsHuman STING (hSTING), murine STING (mSTING), hSTING-N (amino acids 1–230) and hSTING-C (amino acids 173–379) sequences were amplified by PCR and were cloned into pcDNA3 plasmids to generate C-terminally HA-tagged expression constructs. Expression plasmids encoding Flag-tagged RIG-I, ΔRIG-I (amino acids 1–284), IPS-1 and TBK-1 were described previously[24]. GFP-tagged RIG-I and TBK-1 were generated by cloning into pAcGFP1-C1 (Clontech, CA). Other plasmids were obtained as follows: p110-Luc and PRD-III-I-Luc (T. Maniatis), IFNβ-Luc and ISRE-Luc (J. Hiscott), NFκB-Luc (Stratagene, CA), ER-dsRED and Golgi-dsRED (Clontech, CA).

Plasmid constructs

Human STING (hSTING), murine STING (mSTING), hSTING-N (amino acids 1–230) and hSTING-C (amino acids 173–379) sequences were amplified by PCR and were cloned into pcDNA3 plasmids to generate C-terminally HA-tagged expression constructs. Expression plasmids encoding Flag-tagged RIG-I, ΔRIG-I (amino acids 1–284), IPS-1 and TBK-1 were described previously[24]. GFP-tagged RIG-I and TBK-1 were generated by cloning into pAcGFP1-C1 (Clontech, CA). Other plasmids were obtained as follows: p110-Luc and PRD-III-I-Luc (T. Maniatis), IFNβ-Luc and ISRE-Luc (J. Hiscott), NFκB-Luc (Stratagene, CA), ER-dsRED and Golgi-dsRED (Clontech, CA).

AntibodiesRabbit polyclonal antibody against a synthetic peptide corresponding to residues 324–340 of human STING was obtained from EvoQuest Custom Antibody Services (Invitrogen, CA). Other antibodies were obtained from following sources: SEC61β (Upstate, NY), β-Actin, HA, FLAG (Sigma, MO), COX IV, Calreticulin, IRF3 and RIG-I (Abcam, MA).

Rabbit polyclonal antibody against a synthetic peptide corresponding to residues 324–340 of human STING was obtained from EvoQuest Custom Antibody Services (Invitrogen, CA). Other antibodies were obtained from following sources: SEC61β (Upstate, NY), β-Actin, HA, FLAG (Sigma, MO), COX IV, Calreticulin, IRF3 and RIG-I (Abcam, MA).

Confocal microscopyER-DsRED and Golgi-DsRED (Clontech, CA) were used for ER and Golgi marker, respectively. For mitochondria staining, living cells were incubated with 300 nM of Mito Tracker Red (Invitrogen, CA) for 45 min at 37°C.

Confocal microscopy

ER-DsRED and Golgi-DsRED (Clontech, CA) were used for ER and Golgi marker, respectively. For mitochondria staining, living cells were incubated with 300 nM of Mito Tracker Red (Invitrogen, CA) for 45 min at 37°C.

RNA interferenceChemically synthesized 21-nucleotide siRNA duplexes were obtained from Dharmacon, Inc (Lafayette, CO). RNA oligonucleotides used for human and murine STING were as follows: hSTING, GCAUCAAGGAUCGGGUUU; mSTING, CCAACAGCGUCUACGAGA.

RNA interference

Chemically synthesized 21-nucleotide siRNA duplexes were obtained from Dharmacon, Inc (Lafayette, CO). RNA oligonucleotides used for human and murine STING were as follows: hSTING, GCAUCAAGGAUCGGGUUU; mSTING, CCAACAGCGUCUACGAGA.

Generation of the STING knockout miceThe linearized targeting vector was electroporated into E14.1 ES cells originated from 129SvEv strain, followed by the selection in G418. One positive clone was injected into C57BL/6J blastocysts and STING −/− mice generated on a C57BL/6J background.

Generation of the STING knockout mice

The linearized targeting vector was electroporated into E14.1 ES cells originated from 129SvEv strain, followed by the selection in G418. One positive clone was injected into C57BL/6J blastocysts and STING −/− mice generated on a C57BL/6J background.

Yeast two-hybrid analysesTo screen for interacting partners of STING a Yeast two Hybrid approach was utilized, adopting a novel IFN-induced, telomerase immortalized human fibroblast cDNA library, generated by our laboratory. hSTING-C (amino acids 173–379) sequence was amplified by PCR and was cloned into pGBK-T7 (Clontech) to generate a bait plasmid pGBK-T7-hSTING-C. 100μg of the IFN-induced library’s plasmid DNA was used to perform a library-scale transformation of yeast strain Y187.

Yeast two-hybrid analyses

To screen for interacting partners of STING a Yeast two Hybrid approach was utilized, adopting a novel IFN-induced, telomerase immortalized human fibroblast cDNA library, generated by our laboratory. hSTING-C (amino acids 173–379) sequence was amplified by PCR and was cloned into pGBK-T7 (Clontech) to generate a bait plasmid pGBK-T7-hSTING-C. 100μg of the IFN-induced library’s plasmid DNA was used to perform a library-scale transformation of yeast strain Y187.

StatisticsStudents t-test was used to analyze data.

Students t-test was used to analyze data.

MethodsCells, viruses and reagents293T and HEK293 cells were obtained from the ATCC and were maintained in DMEM medium supplemented with 10% FBS. Human Umbilical Vein Endothelial Cells (HUVEC) were obtained from Lonza (Basel, Switzerland), and were maintained in endothelial growth medium (EGM-2) (Lonza). FADD+/+, FADD−/−, TBK-1+/+ and TBK-1−/− MEFs were provided by W.-C. Yeh[16]. VSV-GFP was used in infections and titered as described previously[16]. VSV-ΔM was constructed as described in Faria et al[17]. Murine IFN-β ELISA Kit was obtained from PBL (Piscataway, NJ). EMCV was purchased from ATCC. Synthetic ds B-DNA (poly dA-dT) and polyI:C were purchased from GE healthcare. Interferon stimulatory DNA (ISD) and reverse sequence of ISD were described previously[20]. For stimulation of cells, -DNA or polyI:C were mixed with Lipofectamine 2000 (Invitrogen, CA) at a ratio of 1:1 (vol/wt), and then added to cells at a final concentration of 3 μg/ml.Reporter analysis293T cells or MEFs seeded on 24-well plates were transiently transfected with 50 ng of the luciferase reporter plasmid together with a total of 250 ng of various expression plasmids or empty control plasmids. As an internal control, 10 ng of pRL-TK was transfected simultaneously. Then, 36 hours later, the luciferase activity in the total cell lysates were measuredNorthern blotHuman multiple tissue RNA blots (Clontech, CA) were hybridized with a 32 P-labelled full-length human STING probe.Real time PCRTotal RNA was isolated from cells using the RNeasy RNA extraction kit (Qiagen, Valencia, CA) and cDNA synthesis was performed using 1μg of total RNA (Roche, Indianapolis, IN). Fluorescence real time PCR analysis was performed using a LightCycler 2.0 instrument (Roche Molecular Biochemicals, Indianapolis, IN) and TaqMan Gene Expression Assays (Applied Biosystems, CA). Relative amounts of mRNA were normalized to the 18S ribosomal RNA levels in each sample.DNA Microarray analysisTotal RNA was extracted from 293T cells transfected with STING expressing vectors. Preparation of cDNA and microarray analysis was performed at the W.M. Keck Foundation Biotechnology Research Laboratory DNA microarray facility at Yale University (New Haven, CT). The Human Genome U133 Plus 2.0 Array (Affymetrix, CA) was used. Data analysis was performed with GeneSpring software (Silicon Genetics, CA)[16].Mitochondria and ER fraction isolationMitochondria and ER membranes were purified on discontinuous sucrose gradients as previously described[25]. Briefly, HUVEC cells in MTE buffer (0.27M mannitol, 10 mM Tris-HCl, 0.1mM EDTA, pH 7.4) were lysed by sonication. Lysed cells were centrifuged at 700 × g for 10 min to remove Nuclei and cellular debris. Mitochondria were obtained by centrifugation at 15,000 × g for 10 min, and post mitochondrial supernatant was used for purification of ER fractions. Mitochondria pellet was resuspended in MTE buffer, and was layered on discontinuous sucrose gradients consisting of 1.0 M and 1.7M sucrose and banded by centrifugation at 40,000 × g for 22 min. Mitochondria fraction was collected and pelleted by centrifugation at 15,000 × g for 10 min. Purified mitochondria were resuspended in PBS and used for western blot analysis. To isolate ER fractions, postmitochondrial supernatant described above was layered on discontinuous sucrose gradients consisting of 1.3M, 1.5 M and 2.0 M sucrose, and banded by centrifugation at 100,000 × g for 70 min. The ER fraction at the interface between the supernatant and the 1.3 M sucrose was collected, and pelleted by centrifugation at 100,000 × g for 45 min. The ER membranes were resuspended in PBS and were used for western blot analysis.RNA interference293T cells were plated on 24-well plates at 5 × 104 cells per well and transfected with 10 pmol of siRNA duplex per well using Lipofectamine RNAiMAX (Invitrogen, CA). MEFs were transfected by using an Amaxa nucleofector apparatus (program A-023) and Amaxa MEF nucleofector kit 1 according to the manufacturer’s recommendations. At 72 hours after transfection, cells were used for further experiments.Coimmunoprecipitation, native PAGE and immunoblot analysisCells were seeded on 100-mm dishes at 1 × 106 cells per dish. Cells were transfected with a total of 10 μg of empty plasmid or various expression plasmids using Lipofectamine 2000. At 36 hours after transfection, cells were lysed in M-PER buffer (PIERCE, IL) or 1% digitonin (Calbiochem, CA) buffer (20 mM Tris-HCl, 150 mM NaCl, 1% digitonin) containing protease inhibitors (Roche, IN). Lysates were incubated with HA affinity matrix (COVANCE, WI) or FLAG affinity beads (SIGMA) overnight. The beads were washed three times by TBS containing 0.05% Tween-20 and immunoprecipitates were eluted with non-reducing sample buffer by boiling for 5 min. For endogenous STING immunoprecipitation, HUVEC were lysed in 1% digitonin lysis buffer. Cleared supernatants were incubated with 10μg of anti-STING antibody, followed by incubation with immobilized protein G (PIERCE). The beads were washed four times by 1% digitonin lysis buffer and immunoprecipitates were eluted with SDS sample buffer by boiling for 5 min. Native PAGE and Immunoblotting was carried out as previously described[24].Generation of the STING knockout miceThe linearized targeting vector was electroporated into E14.1 ES cells originated from 129SvEv strain, followed by the selection in G418. Targeted clones were screened by PCR. From 90 clones, 1 positive clone was identified. This ES clone was subjected to the generation of chimera mice by injection using C57BL/6J blastocysts as the host. The resulting male chimeras were further mated with C57BL/6J female mice for germline transmission. The heterozygous mice (F1 mice) were interbred to obtain wild-type, heterozygous and homozygous littermates (F2). The genotypes of the mice were determined by PCR. Animals were generated at the University of Miami School of Medicine Transgenic Core Facility. Mice were allowed to freely access to food and water and housed at an ambient temperature of 23°C and at a 12 h light/dark cycle. Animal care and handling was performed as per IACUC guidelines.IFN induced library constructionTo screen for interacting partners of STING a Yeast two Hybrid approach was adopted using a novel IFN-induced, telomerase immortalized human fibroblast cDNA library. To construct this library, cells were treated overnight with 1000 units each of human IFN–α and –β to induce interferon dependent genes. Poly(A)+ RNA was extracted and cDNA synthesis was carried out using ‘BD Powerscript™’ RT (Clontech). The cDNA was then amplified by PCR, digested with SfiI and ligated to the yeast prey vector pGADT7-RecAB. The ligated mixture was transformed into E. coli strain DH10B. The number of independent clones in the unamplified library was estimated to be 3.2 × 106, with an average size of 1.52 kb and inserts ranging in size from 0.8 to 3.0 kb. 15 independent colonies were isolated and DNA extracted to check for size by PCR.

Cells, viruses and reagents293T and HEK293 cells were obtained from the ATCC and were maintained in DMEM medium supplemented with 10% FBS. Human Umbilical Vein Endothelial Cells (HUVEC) were obtained from Lonza (Basel, Switzerland), and were maintained in endothelial growth medium (EGM-2) (Lonza). FADD+/+, FADD−/−, TBK-1+/+ and TBK-1−/− MEFs were provided by W.-C. Yeh[16]. VSV-GFP was used in infections and titered as described previously[16]. VSV-ΔM was constructed as described in Faria et al[17]. Murine IFN-β ELISA Kit was obtained from PBL (Piscataway, NJ). EMCV was purchased from ATCC. Synthetic ds B-DNA (poly dA-dT) and polyI:C were purchased from GE healthcare. Interferon stimulatory DNA (ISD) and reverse sequence of ISD were described previously[20]. For stimulation of cells, -DNA or polyI:C were mixed with Lipofectamine 2000 (Invitrogen, CA) at a ratio of 1:1 (vol/wt), and then added to cells at a final concentration of 3 μg/ml.

Cells, viruses and reagents

293T and HEK293 cells were obtained from the ATCC and were maintained in DMEM medium supplemented with 10% FBS. Human Umbilical Vein Endothelial Cells (HUVEC) were obtained from Lonza (Basel, Switzerland), and were maintained in endothelial growth medium (EGM-2) (Lonza). FADD+/+, FADD−/−, TBK-1+/+ and TBK-1−/− MEFs were provided by W.-C. Yeh[16]. VSV-GFP was used in infections and titered as described previously[16]. VSV-ΔM was constructed as described in Faria et al[17]. Murine IFN-β ELISA Kit was obtained from PBL (Piscataway, NJ). EMCV was purchased from ATCC. Synthetic ds B-DNA (poly dA-dT) and polyI:C were purchased from GE healthcare. Interferon stimulatory DNA (ISD) and reverse sequence of ISD were described previously[20]. For stimulation of cells, -DNA or polyI:C were mixed with Lipofectamine 2000 (Invitrogen, CA) at a ratio of 1:1 (vol/wt), and then added to cells at a final concentration of 3 μg/ml.

Reporter analysis293T cells or MEFs seeded on 24-well plates were transiently transfected with 50 ng of the luciferase reporter plasmid together with a total of 250 ng of various expression plasmids or empty control plasmids. As an internal control, 10 ng of pRL-TK was transfected simultaneously. Then, 36 hours later, the luciferase activity in the total cell lysates were measured

Reporter analysis

293T cells or MEFs seeded on 24-well plates were transiently transfected with 50 ng of the luciferase reporter plasmid together with a total of 250 ng of various expression plasmids or empty control plasmids. As an internal control, 10 ng of pRL-TK was transfected simultaneously. Then, 36 hours later, the luciferase activity in the total cell lysates were measured

Northern blotHuman multiple tissue RNA blots (Clontech, CA) were hybridized with a 32 P-labelled full-length human STING probe.

Northern blot

Human multiple tissue RNA blots (Clontech, CA) were hybridized with a 32 P-labelled full-length human STING probe.

Real time PCRTotal RNA was isolated from cells using the RNeasy RNA extraction kit (Qiagen, Valencia, CA) and cDNA synthesis was performed using 1μg of total RNA (Roche, Indianapolis, IN). Fluorescence real time PCR analysis was performed using a LightCycler 2.0 instrument (Roche Molecular Biochemicals, Indianapolis, IN) and TaqMan Gene Expression Assays (Applied Biosystems, CA). Relative amounts of mRNA were normalized to the 18S ribosomal RNA levels in each sample.

Real time PCR

Total RNA was isolated from cells using the RNeasy RNA extraction kit (Qiagen, Valencia, CA) and cDNA synthesis was performed using 1μg of total RNA (Roche, Indianapolis, IN). Fluorescence real time PCR analysis was performed using a LightCycler 2.0 instrument (Roche Molecular Biochemicals, Indianapolis, IN) and TaqMan Gene Expression Assays (Applied Biosystems, CA). Relative amounts of mRNA were normalized to the 18S ribosomal RNA levels in each sample.

DNA Microarray analysisTotal RNA was extracted from 293T cells transfected with STING expressing vectors. Preparation of cDNA and microarray analysis was performed at the W.M. Keck Foundation Biotechnology Research Laboratory DNA microarray facility at Yale University (New Haven, CT). The Human Genome U133 Plus 2.0 Array (Affymetrix, CA) was used. Data analysis was performed with GeneSpring software (Silicon Genetics, CA)[16].

DNA Microarray analysis

Total RNA was extracted from 293T cells transfected with STING expressing vectors. Preparation of cDNA and microarray analysis was performed at the W.M. Keck Foundation Biotechnology Research Laboratory DNA microarray facility at Yale University (New Haven, CT). The Human Genome U133 Plus 2.0 Array (Affymetrix, CA) was used. Data analysis was performed with GeneSpring software (Silicon Genetics, CA)[16].

Mitochondria and ER fraction isolationMitochondria and ER membranes were purified on discontinuous sucrose gradients as previously described[25]. Briefly, HUVEC cells in MTE buffer (0.27M mannitol, 10 mM Tris-HCl, 0.1mM EDTA, pH 7.4) were lysed by sonication. Lysed cells were centrifuged at 700 × g for 10 min to remove Nuclei and cellular debris. Mitochondria were obtained by centrifugation at 15,000 × g for 10 min, and post mitochondrial supernatant was used for purification of ER fractions. Mitochondria pellet was resuspended in MTE buffer, and was layered on discontinuous sucrose gradients consisting of 1.0 M and 1.7M sucrose and banded by centrifugation at 40,000 × g for 22 min. Mitochondria fraction was collected and pelleted by centrifugation at 15,000 × g for 10 min. Purified mitochondria were resuspended in PBS and used for western blot analysis. To isolate ER fractions, postmitochondrial supernatant described above was layered on discontinuous sucrose gradients consisting of 1.3M, 1.5 M and 2.0 M sucrose, and banded by centrifugation at 100,000 × g for 70 min. The ER fraction at the interface between the supernatant and the 1.3 M sucrose was collected, and pelleted by centrifugation at 100,000 × g for 45 min. The ER membranes were resuspended in PBS and were used for western blot analysis.

Mitochondria and ER fraction isolation

Mitochondria and ER membranes were purified on discontinuous sucrose gradients as previously described[25]. Briefly, HUVEC cells in MTE buffer (0.27M mannitol, 10 mM Tris-HCl, 0.1mM EDTA, pH 7.4) were lysed by sonication. Lysed cells were centrifuged at 700 × g for 10 min to remove Nuclei and cellular debris. Mitochondria were obtained by centrifugation at 15,000 × g for 10 min, and post mitochondrial supernatant was used for purification of ER fractions. Mitochondria pellet was resuspended in MTE buffer, and was layered on discontinuous sucrose gradients consisting of 1.0 M and 1.7M sucrose and banded by centrifugation at 40,000 × g for 22 min. Mitochondria fraction was collected and pelleted by centrifugation at 15,000 × g for 10 min. Purified mitochondria were resuspended in PBS and used for western blot analysis. To isolate ER fractions, postmitochondrial supernatant described above was layered on discontinuous sucrose gradients consisting of 1.3M, 1.5 M and 2.0 M sucrose, and banded by centrifugation at 100,000 × g for 70 min. The ER fraction at the interface between the supernatant and the 1.3 M sucrose was collected, and pelleted by centrifugation at 100,000 × g for 45 min. The ER membranes were resuspended in PBS and were used for western blot analysis.

RNA interference293T cells were plated on 24-well plates at 5 × 104 cells per well and transfected with 10 pmol of siRNA duplex per well using Lipofectamine RNAiMAX (Invitrogen, CA). MEFs were transfected by using an Amaxa nucleofector apparatus (program A-023) and Amaxa MEF nucleofector kit 1 according to the manufacturer’s recommendations. At 72 hours after transfection, cells were used for further experiments.

RNA interference

293T cells were plated on 24-well plates at 5 × 104 cells per well and transfected with 10 pmol of siRNA duplex per well using Lipofectamine RNAiMAX (Invitrogen, CA). MEFs were transfected by using an Amaxa nucleofector apparatus (program A-023) and Amaxa MEF nucleofector kit 1 according to the manufacturer’s recommendations. At 72 hours after transfection, cells were used for further experiments.

Coimmunoprecipitation, native PAGE and immunoblot analysisCells were seeded on 100-mm dishes at 1 × 106 cells per dish. Cells were transfected with a total of 10 μg of empty plasmid or various expression plasmids using Lipofectamine 2000. At 36 hours after transfection, cells were lysed in M-PER buffer (PIERCE, IL) or 1% digitonin (Calbiochem, CA) buffer (20 mM Tris-HCl, 150 mM NaCl, 1% digitonin) containing protease inhibitors (Roche, IN). Lysates were incubated with HA affinity matrix (COVANCE, WI) or FLAG affinity beads (SIGMA) overnight. The beads were washed three times by TBS containing 0.05% Tween-20 and immunoprecipitates were eluted with non-reducing sample buffer by boiling for 5 min. For endogenous STING immunoprecipitation, HUVEC were lysed in 1% digitonin lysis buffer. Cleared supernatants were incubated with 10μg of anti-STING antibody, followed by incubation with immobilized protein G (PIERCE). The beads were washed four times by 1% digitonin lysis buffer and immunoprecipitates were eluted with SDS sample buffer by boiling for 5 min. Native PAGE and Immunoblotting was carried out as previously described[24].

Coimmunoprecipitation, native PAGE and immunoblot analysis

Cells were seeded on 100-mm dishes at 1 × 106 cells per dish. Cells were transfected with a total of 10 μg of empty plasmid or various expression plasmids using Lipofectamine 2000. At 36 hours after transfection, cells were lysed in M-PER buffer (PIERCE, IL) or 1% digitonin (Calbiochem, CA) buffer (20 mM Tris-HCl, 150 mM NaCl, 1% digitonin) containing protease inhibitors (Roche, IN). Lysates were incubated with HA affinity matrix (COVANCE, WI) or FLAG affinity beads (SIGMA) overnight. The beads were washed three times by TBS containing 0.05% Tween-20 and immunoprecipitates were eluted with non-reducing sample buffer by boiling for 5 min. For endogenous STING immunoprecipitation, HUVEC were lysed in 1% digitonin lysis buffer. Cleared supernatants were incubated with 10μg of anti-STING antibody, followed by incubation with immobilized protein G (PIERCE). The beads were washed four times by 1% digitonin lysis buffer and immunoprecipitates were eluted with SDS sample buffer by boiling for 5 min. Native PAGE and Immunoblotting was carried out as previously described[24].

Generation of the STING knockout miceThe linearized targeting vector was electroporated into E14.1 ES cells originated from 129SvEv strain, followed by the selection in G418. Targeted clones were screened by PCR. From 90 clones, 1 positive clone was identified. This ES clone was subjected to the generation of chimera mice by injection using C57BL/6J blastocysts as the host. The resulting male chimeras were further mated with C57BL/6J female mice for germline transmission. The heterozygous mice (F1 mice) were interbred to obtain wild-type, heterozygous and homozygous littermates (F2). The genotypes of the mice were determined by PCR. Animals were generated at the University of Miami School of Medicine Transgenic Core Facility. Mice were allowed to freely access to food and water and housed at an ambient temperature of 23°C and at a 12 h light/dark cycle. Animal care and handling was performed as per IACUC guidelines.

Generation of the STING knockout mice

The linearized targeting vector was electroporated into E14.1 ES cells originated from 129SvEv strain, followed by the selection in G418. Targeted clones were screened by PCR. From 90 clones, 1 positive clone was identified. This ES clone was subjected to the generation of chimera mice by injection using C57BL/6J blastocysts as the host. The resulting male chimeras were further mated with C57BL/6J female mice for germline transmission. The heterozygous mice (F1 mice) were interbred to obtain wild-type, heterozygous and homozygous littermates (F2). The genotypes of the mice were determined by PCR. Animals were generated at the University of Miami School of Medicine Transgenic Core Facility. Mice were allowed to freely access to food and water and housed at an ambient temperature of 23°C and at a 12 h light/dark cycle. Animal care and handling was performed as per IACUC guidelines.

IFN induced library constructionTo screen for interacting partners of STING a Yeast two Hybrid approach was adopted using a novel IFN-induced, telomerase immortalized human fibroblast cDNA library. To construct this library, cells were treated overnight with 1000 units each of human IFN–α and –β to induce interferon dependent genes. Poly(A)+ RNA was extracted and cDNA synthesis was carried out using ‘BD Powerscript™’ RT (Clontech). The cDNA was then amplified by PCR, digested with SfiI and ligated to the yeast prey vector pGADT7-RecAB. The ligated mixture was transformed into E. coli strain DH10B. The number of independent clones in the unamplified library was estimated to be 3.2 × 106, with an average size of 1.52 kb and inserts ranging in size from 0.8 to 3.0 kb. 15 independent colonies were isolated and DNA extracted to check for size by PCR.

IFN induced library construction

To screen for interacting partners of STING a Yeast two Hybrid approach was adopted using a novel IFN-induced, telomerase immortalized human fibroblast cDNA library. To construct this library, cells were treated overnight with 1000 units each of human IFN–α and –β to induce interferon dependent genes. Poly(A)+ RNA was extracted and cDNA synthesis was carried out using ‘BD Powerscript™’ RT (Clontech). The cDNA was then amplified by PCR, digested with SfiI and ligated to the yeast prey vector pGADT7-RecAB. The ligated mixture was transformed into E. coli strain DH10B. The number of independent clones in the unamplified library was estimated to be 3.2 × 106, with an average size of 1.52 kb and inserts ranging in size from 0.8 to 3.0 kb. 15 independent colonies were isolated and DNA extracted to check for size by PCR.
