---
pmid: '33147444'
title: Genome-wide CRISPR Screens Reveal Host Factors Critical for SARS-CoV-2 Infection.
authors:
- Wei J
- Alfajaro MM
- DeWeirdt PC
- Hanna RE
- Lu-Culligan WJ
- Cai WL
- Strine MS
- Zhang SM
- Graziano VR
- Schmitz CO
- Chen JS
- Mankowski MC
- Filler RB
- Ravindra NG
- Gasque V
- de Miguel FJ
- Patil A
- Chen H
- Oguntuyo KY
- Abriola L
- Surovtseva YV
- Orchard RC
- Lee B
- Lindenbach BD
- Politi K
- van Dijk D
- Kadoch C
- Simon MD
- Yan Q
- Doench JG
- Wilen CB
journal: Cell
year: '2021'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC7574718
doi: 10.1016/j.cell.2020.10.028
---

# Genome-wide CRISPR Screens Reveal Host Factors Critical for SARS-CoV-2 Infection.
**Authors:** Wei J, Alfajaro MM, DeWeirdt PC, Hanna RE, Lu-Culligan WJ, Cai WL, Strine MS, Zhang SM, Graziano VR, Schmitz CO, Chen JS, Mankowski MC, Filler RB, Ravindra NG, Gasque V, de Miguel FJ, Patil A, Chen H, Oguntuyo KY, Abriola L, Surovtseva YV, Orchard RC, Lee B, Lindenbach BD, Politi K, van Dijk D, Kadoch C, Simon MD, Yan Q, Doench JG, Wilen CB
**Journal:** Cell (2021)
**DOI:** [10.1016/j.cell.2020.10.028](https://doi.org/10.1016/j.cell.2020.10.028)
**PMC:** [PMC7574718](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7574718/)

## Abstract

1. Cell. 2021 Jan 7;184(1):76-91.e13. doi: 10.1016/j.cell.2020.10.028. Epub 2020 
Oct 20.

Genome-wide CRISPR Screens Reveal Host Factors Critical for SARS-CoV-2 
Infection.

Wei J(1), Alfajaro MM(1), DeWeirdt PC(2), Hanna RE(2), Lu-Culligan WJ(3), Cai 
WL(4), Strine MS(1), Zhang SM(4), Graziano VR(1), Schmitz CO(1), Chen JS(1), 
Mankowski MC(1), Filler RB(1), Ravindra NG(5), Gasque V(5), de Miguel FJ(6), 
Patil A(7), Chen H(4), Oguntuyo KY(8), Abriola L(9), Surovtseva YV(9), Orchard 
RC(10), Lee B(8), Lindenbach BD(11), Politi K(12), van Dijk D(5), Kadoch C(7), 
Simon MD(13), Yan Q(6), Doench JG(14), Wilen CB(15).

Author information:
(1)Department of Laboratory Medicine, Yale School of Medicine, New Haven, CT 
06520, USA; Department of Immunobiology, Yale School of Medicine, New Haven, CT 
06520, USA.
(2)Genetic Perturbation Platform, Broad Institute of MIT and Harvard, Cambridge, 
MA 02142, USA.
(3)Department of Molecular Biophysics and Biochemistry, Yale School of Medicine, 
New Haven, CT 06520, USA; Department of Cell Biology, Yale University, New 
Haven, CT 06520, USA; Chemical Biology Institute, Yale University, West Haven, 
CT 06516, USA.
(4)Department of Pathology, Yale School of Medicine, New Haven, CT 06520, USA.
(5)Cardiovascular Research Center, Section of Cardiovascular Medicine, 
Department of Internal Medicine, Yale School of Medicine, New Haven, CT 06520, 
USA; Department of Computer Science, Yale University, New Haven, CT 06520, USA.
(6)Department of Pathology, Yale School of Medicine, New Haven, CT 06520, USA; 
Yale Cancer Center, Yale School of Medicine, New Haven, CT 06520, USA.
(7)Department of Pediatric Oncology, Dana-Farber Cancer Institute and Harvard 
Medical School, Boston, MA 02215, USA; Broad Institute of MIT and Harvard, 
Cambridge, MA 02142, USA.
(8)Department of Microbiology, Icahn School of Medicine at Mount Sinai, New 
York, NY 10029, USA.
(9)Yale Center for Molecular Discovery, Yale University, West Haven, CT 06516, 
USA.
(10)Department of Immunology, University of Texas Southwestern Medical Center, 
Dallas, TX 75390, USA.
(11)Department of Microbial Pathogenesis, Yale School of Medicine, New Haven, CT 
06520, USA.
(12)Department of Pathology, Yale School of Medicine, New Haven, CT 06520, USA; 
Yale Cancer Center, Yale School of Medicine, New Haven, CT 06520, USA; 
Department of Medicine, Yale School of Medicine, New Haven, CT 06520, USA.
(13)Department of Molecular Biophysics and Biochemistry, Yale School of 
Medicine, New Haven, CT 06520, USA; Chemical Biology Institute, Yale University, 
West Haven, CT 06516, USA.
(14)Genetic Perturbation Platform, Broad Institute of MIT and Harvard, 
Cambridge, MA 02142, USA. Electronic address: jdoench@broadinstitute.org.
(15)Department of Laboratory Medicine, Yale School of Medicine, New Haven, CT 
06520, USA; Department of Immunobiology, Yale School of Medicine, New Haven, CT 
06520, USA; Yale Cancer Center, Yale School of Medicine, New Haven, CT 06520, 
USA. Electronic address: craig.wilen@yale.edu.

Comment in
    Nat Rev Genet. 2021 Jan;22(1):1. doi: 10.1038/s41576-020-00310-y.
    Cell. 2021 Jan 7;184(1):15-17. doi: 10.1016/j.cell.2020.12.003.

Update of
    bioRxiv. 2020 Jun 17:2020.06.16.155101. doi: 10.1101/2020.06.16.155101.

Identification of host genes essential for severe acute respiratory syndrome 
coronavirus 2 (SARS-CoV-2) infection may reveal novel therapeutic targets and 
inform our understanding of coronavirus disease 2019 (COVID-19) pathogenesis. 
Here we performed genome-wide CRISPR screens in Vero-E6 cells with SARS-CoV-2, 
Middle East respiratory syndrome CoV (MERS-CoV), bat CoV HKU5 expressing the 
SARS-CoV-1 spike, and vesicular stomatitis virus (VSV) expressing the SARS-CoV-2 
spike. We identified known SARS-CoV-2 host factors, including the receptor ACE2 
and protease Cathepsin L. We additionally discovered pro-viral genes and 
pathways, including HMGB1 and the SWI/SNF chromatin remodeling complex, that are 
SARS lineage and pan-coronavirus specific, respectively. We show that HMGB1 
regulates ACE2 expression and is critical for entry of SARS-CoV-2, SARS-CoV-1, 
and NL63. We also show that small-molecule antagonists of identified gene 
products inhibited SARS-CoV-2 infection in monkey and human cells, demonstrating 
the conserved role of these genetic hits across species. This identifies 
potential therapeutic targets for SARS-CoV-2 and reveals SARS lineage-specific 
and pan-CoV host factors that regulate susceptibility to highly pathogenic CoVs.

Copyright © 2020 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.cell.2020.10.028
PMCID: PMC7574718
PMID: 33147444 [Indexed for MEDLINE]

Conflict of interest statement: Declaration of Interests Yale University 
(C.B.W.) has a patent pending related to this work entitled “Compounds and 
Compositions for Treating, Ameliorating, and/or Preventing SARS-CoV-2 Infection 
and/or Complications Thereof.” Yale University has committed to rapidly 
executable non-exclusive royalty-free licenses to intellectual property rights 
for the purpose of making and distributing products to prevent, diagnose, and 
treat COVID-19 infection during the pandemic and for a short period thereafter. 
J.G.D. consults for Foghorn Therapeutics, Maze Therapeutics, Merck, Agios, and 
Pfizer. J.G.D. consults for and has equity in Tango Therapeutics. C.K. is the 
Scientific Founder, Board of Directors member, Scientific Advisory Board member, 
shareholder, and consultant for Foghorn Therapeutics, Inc. (Cambridge, MA).

## Full Text

IntroductionSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative agent of coronavirus disease 2019 (COVID-19), is the greatest public health threat in a century. More than 45,000,000 people have been infected, with more than 1,100,000 deaths globally (Dong et al., 2020). Novel therapeutic agents and vaccines are desperately needed. CoVs are enveloped, positive-sense RNA viruses with genomes of approximately 30 kb that have a broad host range among birds and mammals and are typically transmitted via the respiratory route (Li, 2016; Cui et al., 2019). There are four circulating seasonal CoVs in humans (NL63, OC43, 229E, and HKU1) and three highly pathogenic zoonotic CoVs (SARS-CoV-1, SARS-CoV-2, and Middle East respiratory syndrome CoV [MERS-CoV]), none of which have effective antiviral drugs or vaccines (Guan et al., 2003; Drosten, Kellam and Memish, 2014; Tang et al., 2015; Gordon et al., 2020).Viral entry, the first stage of the CoV life cycle, is mediated by the viral spike protein. The receptor binding domain of the spike binds to a specific cell surface receptor, a major determinant of host range and cell tropism (Letko et al., 2020; Shang et al., 2020). SARS-CoV-1, SARS-CoV-2, and NL63 use angiotensin-converting enzyme 2 (ACE2), whereas MERS-CoV uses dipeptidyl peptidase-4 (DPP4) as a receptor (Li et al., 2003; Hofmann et al., 2005; Raj et al., 2013; Letko et al., 2020) The CoV spike protein requires two proteolytic processing steps prior to entry. The first cleavage event can occur in the producer cell, the extracellular environment, or the endosome and can be mediated by several proteases, including Furin and TMPRSS2 (Hoffmann et al., 2020a; Walls et al., 2020; Yan et al., 2020; Zang et al., 2020). The second proteolytic event, which exposes the viral fusion peptide, can occur at the target cell plasma membrane by TMPRSS2 or in the endosome by Cathepsin L (Hoffmann et al., 2020b; Ou et al., 2020). Upon viral membrane fusion, the viral RNA is released into the cytoplasm, where it is translated and establishes viral replication and transcription complexes before assembling and budding (Snijder et al., 2006; Stertz et al., 2007; Knoops et al., 2008). The host genes that mediate these processes remain largely elusive.Identification of host factors essential for infection is critical to inform mechanisms of CoV pathogenesis, reveal variation in host susceptibility, and identify novel host-directed therapies that may have efficacy against current and future pandemic CoVs. To reveal host genes required for SARS-CoV-2 infection and cell death, we performed genome-wide CRISPR screens with SARS-CoV-2, SARS-CoV-1, and MERS-CoV in a Chlorocebus sabaeus (African green monkey or vervet) cell line, Vero-E6. Vero-E6 cells have several distinct advantages for SARS lineage and MERS-CoV genetic screening. First, Vero-E6 cells and African green monkeys are susceptible to the SARS lineage (SARS-CoV-1 and SARS-CoV-2) and MERS-CoV, enabling direct comparisons of all three highly pathogenic CoVs (Clay et al., 2012; Matsuyama et al., 2020; Ogando et al., 2020; Totura et al., 2020; Woolsey et al., 2020). Second, Vero-E6 cells endogenously express ACE2 and DPP4, which enables discovery of novel regulators of receptor expression. This may not be possible with a transgenic cell line that overexpresses ACE2 or DPP4 under an exogenous promoter (Heaton et al., 2020). Importantly, cigarette smoking upregulates ACE2 expression and is a risk factor for severe COVID-19, highlighting the importance of uncovering the determinants of ACE2 regulation (Smith et al., 2020). Third, unlike other CoV-susceptible cell lines (e.g., Huh7.5), SARS lineage and MERS-CoV infection of Vero-E6 cells is more cytopathic, which enables screening at a later stage of the viral life cycle than what is possible based on screening for expression of a virus-encoded protein or reporter. Notably, Vero-E6 cells do not express type I interferon. Although this may preclude identification of some genes, it also provides a reductionist system that may reduce potential bias toward interferon-stimulated genes (Desmyter et al., 1968; Emeny and Morgan, 1979; Chew et al., 2009).Our screens identified the protease Cathepsin L for the SARS lineage and MERS-CoV and the viral receptors ACE2 and DPP4 for SARS-lineage viruses and MERS-CoV, respectively. We identified genes that are SARS-CoV-2 specific, MERS-CoV specific, and pan-CoV specific and determined that the majority of genes that regulate SARS-CoV-2 infection acted at the level of viral entry. We performed pooled and individual validation of the top CRISPR gene hits. Specifically, we identified HMGB1 and the SWI/SNF chromatin remodeling complex as pro-viral. We found that HMGB1 is critical for ACE2 expression and viral entry of SARS-CoV-1, SARS-CoV-2, and NL63, all of which use ACE2 as a receptor. In contrast, several SWI/SNF complex members were critical for viral entry of the SARS lineage and MERS-CoV but not influenza A virus (IAV) or encephalomyocarditis virus (EMCV), demonstrating specificity for CoVs rather than a broadly acting anti-viral phenotype. We demonstrated that small-molecule antagonists of pro-viral gene products inhibit SARS-CoV-2 infection of Vero-E6 and human cells in vitro. These hits represent novel therapeutic targets for SARS-CoV-2 and enhance our understanding of COVID-19 pathogenesis.

Introduction

Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative agent of coronavirus disease 2019 (COVID-19), is the greatest public health threat in a century. More than 45,000,000 people have been infected, with more than 1,100,000 deaths globally (Dong et al., 2020). Novel therapeutic agents and vaccines are desperately needed. CoVs are enveloped, positive-sense RNA viruses with genomes of approximately 30 kb that have a broad host range among birds and mammals and are typically transmitted via the respiratory route (Li, 2016; Cui et al., 2019). There are four circulating seasonal CoVs in humans (NL63, OC43, 229E, and HKU1) and three highly pathogenic zoonotic CoVs (SARS-CoV-1, SARS-CoV-2, and Middle East respiratory syndrome CoV [MERS-CoV]), none of which have effective antiviral drugs or vaccines (Guan et al., 2003; Drosten, Kellam and Memish, 2014; Tang et al., 2015; Gordon et al., 2020).

Viral entry, the first stage of the CoV life cycle, is mediated by the viral spike protein. The receptor binding domain of the spike binds to a specific cell surface receptor, a major determinant of host range and cell tropism (Letko et al., 2020; Shang et al., 2020). SARS-CoV-1, SARS-CoV-2, and NL63 use angiotensin-converting enzyme 2 (ACE2), whereas MERS-CoV uses dipeptidyl peptidase-4 (DPP4) as a receptor (Li et al., 2003; Hofmann et al., 2005; Raj et al., 2013; Letko et al., 2020) The CoV spike protein requires two proteolytic processing steps prior to entry. The first cleavage event can occur in the producer cell, the extracellular environment, or the endosome and can be mediated by several proteases, including Furin and TMPRSS2 (Hoffmann et al., 2020a; Walls et al., 2020; Yan et al., 2020; Zang et al., 2020). The second proteolytic event, which exposes the viral fusion peptide, can occur at the target cell plasma membrane by TMPRSS2 or in the endosome by Cathepsin L (Hoffmann et al., 2020b; Ou et al., 2020). Upon viral membrane fusion, the viral RNA is released into the cytoplasm, where it is translated and establishes viral replication and transcription complexes before assembling and budding (Snijder et al., 2006; Stertz et al., 2007; Knoops et al., 2008). The host genes that mediate these processes remain largely elusive.

Identification of host factors essential for infection is critical to inform mechanisms of CoV pathogenesis, reveal variation in host susceptibility, and identify novel host-directed therapies that may have efficacy against current and future pandemic CoVs. To reveal host genes required for SARS-CoV-2 infection and cell death, we performed genome-wide CRISPR screens with SARS-CoV-2, SARS-CoV-1, and MERS-CoV in a Chlorocebus sabaeus (African green monkey or vervet) cell line, Vero-E6. Vero-E6 cells have several distinct advantages for SARS lineage and MERS-CoV genetic screening. First, Vero-E6 cells and African green monkeys are susceptible to the SARS lineage (SARS-CoV-1 and SARS-CoV-2) and MERS-CoV, enabling direct comparisons of all three highly pathogenic CoVs (Clay et al., 2012; Matsuyama et al., 2020; Ogando et al., 2020; Totura et al., 2020; Woolsey et al., 2020). Second, Vero-E6 cells endogenously express ACE2 and DPP4, which enables discovery of novel regulators of receptor expression. This may not be possible with a transgenic cell line that overexpresses ACE2 or DPP4 under an exogenous promoter (Heaton et al., 2020). Importantly, cigarette smoking upregulates ACE2 expression and is a risk factor for severe COVID-19, highlighting the importance of uncovering the determinants of ACE2 regulation (Smith et al., 2020). Third, unlike other CoV-susceptible cell lines (e.g., Huh7.5), SARS lineage and MERS-CoV infection of Vero-E6 cells is more cytopathic, which enables screening at a later stage of the viral life cycle than what is possible based on screening for expression of a virus-encoded protein or reporter. Notably, Vero-E6 cells do not express type I interferon. Although this may preclude identification of some genes, it also provides a reductionist system that may reduce potential bias toward interferon-stimulated genes (Desmyter et al., 1968; Emeny and Morgan, 1979; Chew et al., 2009).

Our screens identified the protease Cathepsin L for the SARS lineage and MERS-CoV and the viral receptors ACE2 and DPP4 for SARS-lineage viruses and MERS-CoV, respectively. We identified genes that are SARS-CoV-2 specific, MERS-CoV specific, and pan-CoV specific and determined that the majority of genes that regulate SARS-CoV-2 infection acted at the level of viral entry. We performed pooled and individual validation of the top CRISPR gene hits. Specifically, we identified HMGB1 and the SWI/SNF chromatin remodeling complex as pro-viral. We found that HMGB1 is critical for ACE2 expression and viral entry of SARS-CoV-1, SARS-CoV-2, and NL63, all of which use ACE2 as a receptor. In contrast, several SWI/SNF complex members were critical for viral entry of the SARS lineage and MERS-CoV but not influenza A virus (IAV) or encephalomyocarditis virus (EMCV), demonstrating specificity for CoVs rather than a broadly acting anti-viral phenotype. We demonstrated that small-molecule antagonists of pro-viral gene products inhibit SARS-CoV-2 infection of Vero-E6 and human cells in vitro. These hits represent novel therapeutic targets for SARS-CoV-2 and enhance our understanding of COVID-19 pathogenesis.

ResultsCRISPR Screens of Highly Pathogenic CoVs Reveal Host Genes Essential for InfectionDiscovery of host genes and pathways that mediate pathogenesis of pandemic CoVs is a critical resource that may promote our understanding of how these viruses cause disease, why there is variable host susceptibility, and the origins of host species range and may reveal host-directed therapeutic targets against known and unknown CoVs of pandemic potential. To identify host factors essential for cell survival in response to pandemic human CoVs, including SARS-CoV-1, SARS-CoV-2, and MERS-CoV, we used Vero-E6 cells, a model cell line for isolating viruses that was selected based on its susceptibility and cytopathic effects in response to all three pandemic CoVs (Matsuyama et al., 2020; Ogando et al., 2020; Woolsey et al., 2020). We performed genome-wide CRISPR screens with SARS-CoV-2 (isolate USA-WA1/2020), MERS-CoV (EMC/2012), a tissue culture-adapted MERS-CoV (T1015N), and a recombinant bat CoV (HKU5) containing the spike protein of SARS-CoV-1 (HKU5-SARS-CoV-1-S) (Scobey et al., 2013; Agnihothram et al., 2014). HKU5-SARS-CoV-1-S was used as a surrogate for SARS-CoV-1, which is a select agent. Further, to test whether genes implicated in SARS-CoV-2 infection act at the level of viral entry, we performed a genome-wide screen with replication-competent vesicular stomatitis virus (VSV) expressing the SARS-CoV-2 spike protein (rcVSV-SARS-CoV-2-S). rcVSV-SARS-CoV-2-S is deficient for the broadly fusogenic VSV glycoprotein (VSV-G); thus, viral entry is entirely mediated by the SARS-CoV-2 spike protein (Case et al., 2020; Dieterle et al., 2020)We used a C. sabaeus genome-wide pooled CRISPR library composed of 83,963 targeting single guide RNAs (sgRNAs) with an average of four sgRNAs per gene and 1,000 non-targeting control sgRNAs. We initially performed two independent SARS-CoV-2 genome-wide screens with Vero-E6 lines expressing two different Cas9 nuclease constructs (Cas9-v1 and Cas9-v2); Cas9-v2 has an additional nuclear localization sequence to increase activity. We transduced both Vero-Cas9 cell lines with the C. sabaeus sgRNA library and challenged cells with SARS-CoV-2 (Figure 1
A). To generate a robust dataset, we performed independent screens at different cell densities, fetal bovine serum (FBS) concentrations, and multiplicities of infection (MOI). Z scores for all SARS-CoV-2 screens are shown in Table S1. Subsequent MERS-CoV wild type (WT), MERS-CoV T1015N, HKU5-SARS-CoV-1-S, and rcVSV-SARS-CoV-2-S screens were performed in duplicate under a single cell density, FBS concentration, and MOI. Genomic DNA was harvested from surviving cells 7–9 days post-infection (dpi), and guide abundance was determined by PCR and massively parallel sequencing. Technical performance of the screens is described in STAR Methods and Figure S1
.Figure 1Genome-wide CRISPR Screens Identify Genes Critical for CoV-Induced Cell Death(A) Schematic of the pooled screen. Vero-E6-Cas9 cells transduced with the genome-wide C. sabaeus library received mock treatment or were challenged with SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, MERS-CoV, or MERS-CoV T1015N. Surviving cells from each virus infection were isolated, and the sgRNA sequences were amplified by PCR and sequenced.(B) Volcano plot showing top genes conferring resistance and sensitivity to SARS-CoV-2. The gene-level Z score and −log10 (FDR) were calculated using the mean of the five Cas9-v2 conditions. Non-targeting control sgRNAs were grouped randomly into sets of 4 to serve as “dummy” genes and are shown in green.(C) Heatmaps of the top gene hits for SARS-CoV-2 resistance (20) and sensitization (20), ranked by mean Z score. The top 5 hits for MERS-CoV are also included and indicated by an asterisk. ARID1A was a top resistance gene for SARS-CoV-2 and MERS-CoV.(D–F) Correlation between gene enrichment in SARS-CoV-2 and rcVSV-SARS-CoV-2-S (D), HKU5-SARS-CoV-1-S (E), and MERS-CoV (F) screens. R, pearson correlation.(G) Venn diagram of the top 100 pro-viral genes from SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, and MERS-CoV screens.Figure S1Quality Control Metrics for CRISPR Screen, Related to STAR Methods(A) Correlation matrix depicting the Pearson correlation between the guide-level log-fold change values relative to the plasmid DNA. Cells were cultured in DMEM with 2% FBS (D2), 5% FBS (D5), 10% FBS (D10), plated at 2.5 × 106 or 5.0 × 106 cells per T150 flask and infected at a MOI 0.1 (hi) or MOI 0.01 (lo). (B) Receiver-operator characteristic (ROC) curve for the recovery of guides targeting essential genes in the mock-treated condition of the Cas9-v1 and Cas9-v2 screens. True positives are n = 1,528 essential genes (n = 6,178 guides); true negative genes are n = 622 non-essential genes (n = 2,504 guides). We mapped essential and non-essential genes, which were derived for human cell lines, to the African green monkey genome simply by matching gene symbols. AUC = area under curve. (C) Correlation between gene enrichment in Cas9-v1 and Cas9-v2 screens. Pearson correlation is reported. (D-E) GFP-based Cas9 activity assay in Vero-E6 cells stably expressing either Cas9-v1 (D) or Cas9-v2 (E). The pXPR_047 construct expresses GFP and an sgRNA targeting GFP; therefore, cells without Cas9 activity will express GFP, whereas cells with high Cas9 activity will knock out GFP and resemble parental cells. (F) Approach to calculate residuals from log-fold change data, using ACE2 and the 5% FBS, 5 × 106 cells/flask, MOI 0.1 condition as an example. A natural cubic spline with four degrees of freedom is shown in blue, and a residual for each sgRNA is calculated to be the vertical distance from the fit spline.The SARS-CoV-2 screen identified numerous genes that confer resistance (pro-viral) or sensitization (anti-viral) when targeted by sgRNAs, with a minimum false discovery rate (FDR) of 0.03 for non-targeting controls (Figures 1B and 1C), demonstrating high technical quality. The strongest resistance hit was the viral receptor ACE2 (mean Z score = 4.9, descending rank = 1; Figures 1B, 1C, S2
A, and S2B). CTSL, which encodes the Cathepsin L protease, was also positively selected under all conditions (mean Z score = 3.0, descending rank = 18; Figures 1B, 1C, and S2B). We did not observe enrichment of TMPRSS2 (mean Z score = 0.9, descending rank = 2,726) or the proteases TMPRSS4 or FURIN (mean Z scores = 1.1, 0.4; descending ranks = 1,657, 7,180, respectively), which have also been implicated in SARS-CoV-2 entry (Hoffmann et al., 2020b; Shang et al., 2020; Zang et al., 2020), suggesting that these proteases are not essential in a cell-intrinsic manner for SARS-CoV-2-induced cell death as performed here and/or are functionally redundant.Figure S2A Genome-wide CRISPR Screen Identifies Genes Critical for SARS-CoV-2-Induced Cell Death, Related to Figure 1(A) Performance of individual sgRNAs targeting ACE2, SMARCA4, CTSL, and TMPRSS2. The mean residual across the five Cas9-v2 conditions is plotted for the full library (top) and for the 4 guide RNAs targeting each gene. (B)Heatmaps of the top 25 gene hits for resistance and sensitivity, ranked by mean z-score in the Cas9-v2 conditions. Genes that are included in one of the gene sets labeled in (Figure 2A) are colored accordingly. Condition A: Cas9-v2 D5 (DMEM+5%FBS) 2.5 × 106 cells/flask MOI 0.1; B: Cas9-v2 D5 5 × 106 cells/flask MO 0.1; C: Cas9-v2 D2 (DMEM+2%FBS) 5 × 106 cells/flask MOI 0.1; D: Cas9-v2 D10 (DMEM+10%FBS) 5 × 106 cells/flask MOI 0.1; E: Cas9-v2 D5 2.5 × 106 cells/flask MOI 0.01. (C) Nodes represent significantly enriched gene sets. The size of each gene set is proportional to its mean absolute z-score. Gene sets are colored by the direction in which they score. Edges represent significant overlap between gene sets. The transparency of each edge is proportional to the fraction of genes shared by two gene sets. Gene sets were clustered using the infomap algorithm and the most central set by PageRank is labeled for each cluster. The Fruchterman–Reingold algorithm was used to lay out the network.Comparison of SARS-CoV-2 with rcVSV-SARS-CoV-2-S revealed substantial concordance between the viruses, suggesting that concordant genes act at the level of viral entry (Figures 1C and 1D). SARS-CoV-2 and HKU5-SARS-CoV-1-S screens also yielded a similar overlap, consistent with similar mechanisms of entry as mediated by the SARS-CoV-1 and SARS-CoV-2 spike proteins (Figure 1E). Next we assessed the relationship between MERS-CoV and SARS-CoV-2 screens. The MERS-CoV receptor DPP4 was the top resistance hit in the MERS-CoV screen, whereas ACE2 was not enriched (Figure 1F). The SARS-CoV-2 resistance genes ARID1A, DYRK1A, KDM6A, and CTSL were also highly enriched in the MERS-CoV screen (Figure 1F). Pairwise correlations of all genome-wide screens are shown in Figure S3
, and Z scores for all genes are shown in Table S2. Next we compared the top 100 resistance genes across the genome-wide screens with SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, and MERS-CoV (WT) viruses. Five genes (ARID1A, KDM6A, JMJD6, SMARCC1, and CTSL) scored in each of the four virus screens. That these genes were enriched in the SARS-CoV-2, HKU5-SARS-CoV-1-S, and MERS-CoV (WT) screens suggest that they are potentially pan-coronaviral. Because they were also enriched in the rcVSV-SARS-CoV-2-S screen, they act at the level of CoV entry (Figure 1G). This suggests that these genes promote lineage-independent entry of pandemic CoVs. 14 genes scored in the three SARS-lineage viruses but not MERS-CoV, including ACE2, HMGB1, SMARCA4, DYRK1A, and DPF2, suggesting that these genes mediate entry of SARS-lineage viruses (Figure 1G). The MERS receptor DPP4 along with AXIN1 and TMEM41B were identified as MERS-CoV-specific pro-viral genes, whereas SMAD3 and SMAD4 were only enriched in the SARS-CoV-2 screens. Among sensitization genes, BPTF, which encodes the scaffold for the Nucleosome Remodeling Factor (NURF) chromatin remodeling complex, was broadly depleted for all four viruses. Similarly, the interferon-stimulated gene LY6E, which was recently identified as a pan-coronaviral entry inhibitor, was identified as an anti-viral gene for SARS-lineage and MERS-CoV screens (Pfaender et al., 2020). This is despite Vero-E6 cells being deficient in type I interferon. Overall, these results show that a survival assay in Vero-E6 cells is able to distinguish host factors common to and specific for a variety of pathogenic CoVs.Figure S3Comparison of All Viruses from Genome-wide CRISPR Screens, Related to Figure 1Comparison of gene enrichment of (A) SARS-CoV-2 relative to MERS-CoV T1015N, (B) rcVSV-SARS-CoV-2-S relative to HKU5-SARS-CoV-1-S, (C) rcVSV-SARS-CoV-2-S relative to MERS-CoV WT, (D) rcVSV-SARS-CoV-2-S relative to MERS-CoV T1015N. (E) HKU5-SARS-CoV-1-S relative to MERS-CoV WT, (F) HKU5-SARS-CoV-1-S relative to MERS-CoV T1015N, and (G) MERS-CoV WT relative to MERS-CoV T1015N. Pearson correlation is reported.Although we validated all genome-wide screens, we focused on SARS-CoV-2, given the immediacy of the current pandemic. To systematically identify hit gene sets enriched in the SARS-CoV-2 screen, we used the STRING-db enrichment detection tool and identified 623 significant gene sets (Figure S2C) from 10 sources (e.g., Kyoto Encyclopedia of Genes and Genomes [KEGG], Gene Ontology [GO] process) (Szklarczyk et al., 2019). The top gene sets that scored in the positive direction (pro-viral), negative direction (anti-viral), or both directions and their respective genes are shown in Figure 2
. SMARCA4 (BRG1), the catalytic subunit of the SWI/SNF remodeling complex (Centore et al., 2020), scored after ACE2 as the second-strongest SARS-CoV-2 resistance hit (mean Z score = 4.9; descending rank = 2; Figures 1B–1D), with several other members, including ARID1A, SMARCE1, SMARCB1, and SMARCC1, showing enrichment (mean Z scores = 3.6, 2.8, 2.4, 2.3; descending ranks = 9, 20, 47, 59, respectively; Figure 2B). The SWI/SNF complex is an ATP-dependent nucleosome remodeling complex that regulates chromatin accessibility and gene expression (Zhou et al., 2016; Clapier et al., 2017). Interestingly, although the SWI/SNF complex genes ARID1A, SMARCB1, and SMARCC1 were enriched in SARS-lineage and MERS-CoV screens, SMARCA4 was only enriched in the SARS-lineage screens. We also identified several other histone-modifying enzymes as key regulators of SARS-CoV-2-induced cell death (Figures 1C and S2B). Other pro-viral genes include the histone demethylase KDM6A (mean Z score = 4.1, descending rank = 4), the histone methyltransferase KMT2D (mean Z score = 2.6, descending rank = 35), as well as the lysyl hydroxylase JMJD6 (mean Z score = 3.7, descending rank = 6) (Figures 1C and S2B; Webby et al., 2009; Hu et al., 2013; Unoki et al., 2013; Chakraborty et al., 2019). In contrast, sgRNAs targeting HIRA, CABIN1, and ASF1A were negatively selected, revealing an anti-viral function. These genes encode three of the four proteins in the HIRA/UBN1/CABIN1/ASF1a (HUCA) histone H3.3 chaperone complex (mean Z scores = −5.7, −5.4, −3.0; ascending ranks = 1, 2, 64, respectively), suggesting an anti-viral role for deposition of the histone variant H3.3 (Rai et al., 2011).Figure 2Performance of Genes in the Top Gene Sets(A) The top three gene sets that score in the positive direction (resistance) and top gene set that scores in the negative direction (sensitization) or both, filtered for gene sets with at least five genes, and that are most central to a given module (Figure S2C) and then ranked by mean absolute Z score. The number of genes in each set is indicated in parentheses.(B) For each gene in the “SWI/SNF complex” gene set from STRING, the Z score in each virus screen is shown.(C–F) Similarly, the genes in the gene sets (C) “RUNX3 regulates CDKN1A transcription” from Reactome, (D) “Cystatin, and endolysosome lumen” from STRING, (E) “Viral translation” from GO, and (F) “NURF complex” from GO.We additionally observed enrichment in the “RUNX3 regulates CDKN1A transcription” gene set from Reactome (Figures 2A and 2C). This is driven by enrichment of sgRNAs targeting the signal transducers SMAD3 and SMAD4 (mean Z scores = 2.8, 3.1; descending ranks = 21, 15, respectively). The “cystatin and endolysosome lumen” gene set, which includes CTSL, was also enriched (Figure 2D). Not surprisingly, we observed the enrichment of “viral translation” in positive and negative directions (Figure 2E). The NURF complex was the top gene set enriched in the negative direction (Figure 2F).Pooled and Arrayed Validation Confirms Genome-wide CRISPR Screen Hits and Reveals Virus SpecificityTo validate the genome-wide screens, we generated a custom CRISPR subpool containing 10 sgRNAs for each of the top 250 and bottom 250 genes from an earlier analysis of the SARS-CoV-2 screen along with 500 non-targeting control sgRNAs and sgRNAs targeting other genes of interest, such as the MERS-CoV receptor DPP4. We introduced this sgRNA library into Vero-Cas9-v2 cells and challenged this pool with SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, MERS-CoV, or MERS-CoV T1015N. We included the orthomyxovirus IAV (A/WSN/1933), the picornavirus EMCV, and VSV (Indiana) as control viruses, which all cause cytopathic effects in Vero-E6 cells. All eight viruses were screened in duplicate except for IAV. The secondary screen validated the top pro-viral and anti-viral genes from the primary genome-wide SARS-CoV-2 screen (Figure 3
A; Table S3). Clustering the correlations between the log2 fold changes of each condition revealed that viruses containing a SARS-lineage spike grouped together, as did MERS-CoV WT and T1015N, whereas IAV and EMCV were outliers, as expected (Figure 3B). A principal-component analysis of the secondary screens revealed clusters of gene hits in an unbiased manner (Figure 3C). Focusing on the top resistance hits for each virus screen, we saw SARS lineage-specific (e.g., ACE2 and PHIP), MERS lineage-specific (e.g., DPP4), and pan-CoV-specific genes (e.g., CTSL, ARID1A, PCBD1, and KMT2D) (Figure S4
A). Consistent with the genome-wide CRISPR screens, the key genes encoding members of the SWI/SNF complex are pro-viral for CoVs but not IAV or EMCV (Figure S4B). We observed that SMAD3 and SMAD4 were modestly enriched in the SARS-CoV-2 and rcVSV-SARS-CoV-2-S subpool screens (Figure S4C). In addition, the top sensitization genes involved in the HUCA histone H3.3 chaperone complex were specific to the SARS-lineage spike protein (Figure S4D). The secondary screens for rcVSV-SARS-CoV-2-S and HKU5-SARS-CoV-1-S showed strong agreement with the SARS-CoV-2 secondary screen with correlation coefficients of 0.90 and 0.89, respectively (Figures 3D and 3E). The MERS-CoV secondary screen correlated with the SARS-CoV-2 screen to a lesser extent than rcVSV-SARS-CoV-2-S and HKU5-SARS-CoV-1-S (r = 0.43; Figure 3F). Minimal correlation was observed between IAV or EMCV and SARS-CoV-2, with correlations of 0.23 and 0.042, respectively (Figures 3G and 3H). No cells survived infection with VSV, precluding analysis. This demonstrates the virus specificity of the identified host genes and provides insights into the stage of the virus life cycle mediated by critical genes.Figure 3CRISPR Subpool Screens Validate Primary Genome-wide Screens and Demonstrate the Specificity of Hits for CoVsA CRISPR subpool was generated with 10 sgRNAs per gene for each of the top 250 and bottom 250 genes from the SARS-CoV-2 genome-wide screen along with non-targeting controls and other genes of interest, including DPP4.(A) Correlation between gene enrichment in primary genome-wide and secondary subpool SARS-CoV-2 subscreens. Pearson correlation is reported.(B) Correlation matrix depicting the Pearson correlation between the guide-level log-fold change values relative to the plasmid DNA for the 13 subpool screens with the indicated viruses. All viruses were screened in duplicate (#1 and #2), except IAV-WSN. VSV was also screened, but no cells survived infection.(C) Principal-component analysis (PCA) plot of all viruses reveals clustering and overlap of gene hits among SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, and MERS-CoV WT and the T1015N cluster. IAV/WSN/1933 (IAV-WSN) and encephalomyocarditis virus (EMCV) are outliers among the CoV screens.(D–H) Comparison of gene enrichment in SARS-CoV-2 relative to rcVSV-SARS-CoV-2-S (D), HKU5-SARS-CoV-1-S (E), MERS-CoV (F), IAV-WSN (G), and EMCV (H). Pearson correlation is reported.(I) We generated a CRISPR subpool targeting 32 genes (inclusive of control genes) in the human lung cancer cell line Calu-3. Gene enrichment from the primary SARS-CoV-2 screen correlates with results from Calu-3 cells. Pearson correlation is reported.Figure S4Comparison of Secondary CRISPR Subpool Screens, Related to Figure 3(A) Heatmap depicting genes with a z-score > 10 in any of the secondary subpool screens. (B) Heatmap showing genes involved in the SWI/SNF chromatin remodeling complex. (C) Heatmap showing genes in the “Runx3 regulates CDKN1A transcription” pathway. (D) Heatmap showing genes in the HUCA histone H3.3 chaperone complex.To begin to understand the generalizability of these hits to human cells, we screened a small CRISPR subpool targeting 32 genes in Calu-3 cells, a lung adenocarcinoma line that natively expresses ACE2 (Table S4). This screen revealed significant overlap in pro-viral (including ACE2, DYRK1A, SMARCA4, KDM6A, JMJD6, SMARCE1, and SIAH1) and anti-viral genes (including HIRA, PIAS2, and SMARCA5) between Vero-E6 and Calu-3 cells (Figure 3I), suggesting the conserved role of these genetic hits across species. Notably, TMPRSS2, but not CTSL, was enriched in the Calu-3 subpool screen, consistent with TMPRSS2 expression in Calu-3 cells but not Vero-E6 cells (Böttcher-Friebertshäuser et al., 2011).We selected 25 genes for further validation of the SARS-CoV-2 screen in an arrayed rather than pooled format, consisting of 18 resistance and 7 sensitization genes (Figure 4
A). We transduced Vero-Cas9-v2 cells with one of 42 individual sgRNAs (1–3 sgRNAs per gene). We then challenged each of the 42 cell lines with SARS-CoV-2 and assessed cell viability. Cells receiving sgRNAs targeting pro-viral genes exhibited greater viability than those with non-targeting control sgRNAs. We validated the knockout efficiency of several genes (including ACE2, SMARCA4, KDM6A, and SMAD3) by western blot, and the protein abundance correlated with the degree of protection (Figures 4B–4D). Cells receiving sgRNAs targeting anti-viral genes in the primary screen exhibited increased susceptibility to cell death relative to controls, confirming the efficiency and reproducibility of the screen (Figures 4B and 4D). The robust concordance between the primary screens and the several subsequent validation approaches indicates that these screens will be a useful resource for further investigation of CoV host-pathogen interactions.Figure 4Arrayed Validation of 18 Resistance and 7 Sensitization Hit Genes(A) Performance in the pooled screen of sgRNAs targeting the 25 genes selected for further validation. The mean residual across the five Cas9-v2 conditions is plotted for the full library (top) and for the 3–4 sgRNAs targeting each gene. Genes that scored as resistance hits are shown in red; genes that scored as sensitization hits are shown in blue. The dashed line indicates a residual of 0.(B) 42 unique sgRNAs targeting 25 genes were introduced into Vero-E6-Cas9-v2 cells. SARS-CoV-2 was added at MOI 0.2, and cell viability was measured at 3 dpi.(C) Western blot for ACE2, SMARCA4, KDM6A, and SMAD3 expression in control and the respective gene-disrupted Vero-E6 cells.(D) Z scores from the genome-wide CRISPR screen correlate with cell viability of individually disrupted genes. Genes with multiple sgRNAs from (B) are averaged to generate one point per geneData were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM. ns, not statistically significant; ∗p < 0.05; ∗∗p < 0.01; ∗∗∗p < 0.001.The CRISPR Screen Reveals Potential Host-Directed Therapeutic TargetsNext, we asked whether the genes and pathways revealed by the screen could be targeted with small molecules. We selected antagonists described previously to inhibit these gene products and investigated their effects on SARS-CoV-2-induced cell death. In addition, we utilized a replication-competent infectious clone of SARS-CoV-2 expressing the fluorescent reporter mNeonGreen (icSARS-CoV-2-mNG) to quantify the influence of these molecules on viral replication (Xie et al., 2020). We observed dose-dependent inhibition of SARS-CoV-2-induced cell death and virus replication with calpain inhibitor III, whose targets include Cathepsin L (Figures 5A, 5D, and 5E; Simmons et al., 2005; Hoffmann et al., 2019). Given the pro-viral role of SMARCA4 and SMAD3, we tested whether existing small-molecule antagonists of these pathways have antiviral activity against SARS-CoV-2. Specifically, treatment with PFI-3, which targets the bromodomains of the SWI/SNF proteins SMARCA4 and SMARCA2 (Fedorov et al., 2015; Xie et al., 2020), conferred protection from virus-induced cell death (Figure 5B) and reduced the frequency of viral infection, as measured by expression of mNeonGreen (Figures 5D and 5E). We also assessed the small molecule SIS3, which targets the pro-viral gene SMAD3 identified in the screen (Jinnin et al., 2006). SIS3 exhibited dose-dependent protection from virus-induced cell death and also inhibited SARS-CoV-2 fluorescent reporter expression (Figures 5C–5E). We next performed SARS-CoV-2 growth curves to investigate the effects of PFI-3 and SIS3 in Vero-E6 cells, Calu-3 cells, and the human liver cell line Huh7.5. We observed an ∼1-log reduction in PFI-3-treated cells and ∼2-log reduction in SIS3-treated cells (Figures 5F–5H). This provides pharmacological validation in Vero-E6 and human cells in addition to genetic evidence that these pathways are critical for SARS-CoV-2 infection.Figure 5Small Molecules Protect Cells from SARS-CoV-2-Induced Cell Death(A–C) Vero-E6 cells were pretreated with the indicated concentrations of the Cathepsin L inhibitor Calpain inhibitor III (A), the SMARCA4 inhibitor PFI-3 (B), or the SMAD3 inhibitor SIS3 (C) for 48 h and then infected with SARS-CoV-2 at a MOI of 0.2. Cell viability was measured at 3 dpi and compared with mock-infected controls. Red, infected; blue, mock-infected.(D and E) Vero-E6 cells were pretreated with 10 μM Calpain inhibitor III, PFI-3, or SIS3 for 48 h and then infected with icSARS-CoV-2 mNG at a MOI of 1. Infected cell frequencies were measured by mNeonGreen expression at 2 dpi. Scale bars, 300 μm.(F–H) Vero-E6 (F), Huh7.5 (G), and Calu-3 (H) cells were pretreated with 10 μM SIS3 and 40 μM PFI-3 for 48 h and then infected with SARS-CoV-2 at a MOI of 0.1. Virus production, as measured by plaque-forming units (PFU) per milliliter, was determined by plaque assay. LOD, limit of detection.Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM. ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.001.HMGB1 Regulates ACE2 Expression and Is Essential for Viral Entry of SARS-CoV-1, SARS-CoV-2, and NL63The SARS-CoV-2 screen revealed a putative pro-viral role of the gene LOC103214541, which is annotated as “HMGB1-like” in the C. sabaeus genome (mean Z score = 3.6, descending rank = 10; Figures 1C and 6A). HMGB1 is a nuclear protein that binds DNA but translocates to the cytoplasm under conditions of stress and can be secreted extracellularly, where it functions as an alarmin (Andersson et al., 2018). HMGB1-like was also identified as pro-viral in the rcVSV-SARS-CoV-2-S and HKU5-SARS-CoV-1-S but not the MERS-CoV screens (Figures 1C, 1G, and 3C–3E). To validate the role of HMGB1 in SARS-CoV-2 infection, we introduced two independent sgRNAs targeting HMGB1 into each of three independent cell lines: Vero-E6, Huh7.5, and Calu-3. We observed depletion of HMGB1 protein as measured by western blot (Figure 6B). HMGB1 disruption protected cells from SARS-CoV-2-induced cell death, and the degree of protection correlated with HMGB1 protein abundance (Figure 6C). We then performed SARS-CoV-2 growth curves on control and HMGB1-disrupted cells and observed an ∼2-log reduction in SARS-CoV-2 replication 24 and 48 h post-infection (Figure 6D). Next we investigated whether HMGB1 acts cell intrinsically or as an alarmin to regulate SARS-CoV-2 infection. SARS-CoV-2 infection increased HMGB1 protein levels in the nucleus and cytoplasm (Figure S5
A) and culture medium (Figures S5B and S5C). Recombinant HMGB1 protein had no effect on SARS-CoV-2 infection when added extracellularly to WT or HMGB1 KO cells, as measured by cell viability, icSARS-CoV-2-mNG infection, and SARS-CoV-2 pseudovirus infection (Figures S5D–S5F), demonstrating that HMGB1 acts cell intrinsically rather than as an alarmin or chemokine to regulate SARS-CoV-2 infection.Figure 6HMGB1 Is a Novel Regulator of ACE2(A) Performance of individual guide RNAs targeting LOC103214541 (HMGB1-like). The mean residual across the five Cas9-v2 conditions is plotted for the full library (top) and for the 3 guide RNAs targeting that gene.(B) Western blot for HMGB1 expression in control and HMGB1-disrupted Vero-E6, Huh7.5, and Calu-3 cells.(C) Control and HMGB1-disrupted Vero-E6, Huh7.5, and Calu-3 cells were infected with SARS-CoV-2 at a MOI of 0.2. Cell viability relative to an uninfected control was measured 3 dpi (Vero-E6 and Calu-3) or 4 dpi (Huh7.5) with CellTiter Glo.(D) Vero-E6 cells were infected with SARS-CoV-2 at a MOI of 0.1. Virus production was measured by plaque assay.(E) Correlation between CRISPR screen Z score and gene expression in control and HMGB1-disrupted Vero-E6 cells reveals downregulation of ACE2 in HMGB1-disrupted cells.(F) qPCR and western blot were performed in HMGB1 knockout and complemented Vero-E6 cells.(G) Genome tracks of RNA sequencing (RNA-seq), ChIP-seq for H3K27ac, and ATAC-seq at the ACE2 locus in control and HMGB1-disrupted Vero-E6 cells. The p values for ChIP-seq and ATAC-seq are for the genomic region indicated by the black bar below the tracks.(H) HMGB1 knockout and complemented Vero-E6 cells were infected with VSV pseudoparticles (VSVpp): VSVpp-SARS-CoV-1-S, VSVpp-SARS-CoV-2-S, VSVpp-NL63-S, VSVpp-MERS-S, and VSVpp-VSV-G. Luciferase relative to a VSVpp-VSV-G control was measured 1 dpi.Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM. ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.001.Figure S5HMGB1 Acts Cell-Intrinsically to Regulate Susceptibility to SARS-CoV-2 Infection, Related to Figure 6(A) Vero-E6 cells were mock-treated or infected with SARS-CoV-2 at a MOI of 1 for 24 hours before cell fractionation was performed. (B-C) Infection resulted in release of HMGB1 protein in the supernatant which was quantified by ELISA from Vero-E6 (B) and Huh7.5 (C) cells infected with SARS-CoV-2 for the indicated times. (D) Vero-E6 cells were pre-treated with the indicated concentration of recombinant HMGB1 (rHMGB1) for 24 hours and then infected with SARS-CoV-2 at a MOI of 0.2. Cell viability was measured at 3 dpi and compared to mock infected controls. (E) Vero-E6 cells were pre-treated with rHMGB1 for 24 hours and then infected with icSARS-CoV-2 mNG at a MOI of 1. Infected cell frequencies were measured by mNeonGreen expression at 1 dpi. (F) Vero-E6 cells were pre-treated with the indicated concentration of rHMGB1 for 24 hours and then infected with VSVpp-SARS-CoV-2-S and VSVpp-VSVG pseudovirus. Luciferase relative to VSVG control was measured at 1 dpi. Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM ns, not statistically significant; ∗p < 0.05; ∗∗p < 0.01; ∗∗∗p < 0.001.Because HMGB1 is a DNA binding protein that regulates chromatin, we hypothesized that HMGB1 controls a pro-viral gene expression program. When we compared differentially expressed genes between control and HMGB1 disrupted cells with the gene-level Z scores from the genome-wide CRISPR screen, we found that, among pro-viral genes, only HMGB1, ACE2, and CTSL gene expression was significantly downregulated in HMGB1-disrupted cells (Figures 6E and S6
A). Interestingly, few other differentially expressed genes were enriched in the positive or negative direction in the CRISPR screen. We confirmed that ACE2 transcripts were reduced significantly in HMGB1 knockout cells compared with WT Vero-E6 cells by qRT-PCR, as were ACE2 protein levels by western blot (Figure 6F). Gene Ontology enrichment analysis indicated that 16 gene sets were significantly enriched in differentially expression genes; however, these gene sets were not enriched in the SARS-CoV-2 CRISPR screen (Figure S6B).Figure S6Effects of HMGB1 Loss on Chromatin States across the Vero-E6 Genome, Related to Figure 6(A) Volcano plot for RNA sequencing of control and HMGB1 disrupted cells. The x axis shows log2 fold-change and the y axis shows −log10 of the adjusted P value (adj. P) as calculated by DESeq2. (B) Top gene sets, which significantly enriched in the upregulated, downregulated or both of differentially expression genes (fold change > 1.5 and p < 0.05) from GO. (C-D)Volcano plots for ATAC-seq (C) and H3K27ac ChIP-seq (D) of control and HMGB1 disrupted cells. Fold change and adjusted p value for each called peak was calculated by DESeq2. (E) Correlation between changes in overlapping ATAC-seq and H3K27ac ChIP-seq peaks upon HMGB1 disruption. Dashed lines represent p = 0.01. Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM ns, not statistically significant; ∗p < 0.05; ∗∗p < 0.01; ∗∗∗p < 0.001.We next analyzed the effects of HMGB1 disruption on chromatin states across the genome by chromatin immunoprecipitation sequencing (ChIP-seq) and assay for transposase-accessible chromatin sequencing (ATAC-seq; Figures S6C and S6D). Upon HMGB1 disruption, changes in chromatin accessibility by ATAC-seq were positively correlated with changes in acetylation of lysine 27 on histone H3 protein subunit (H3K27ac) ChIP-seq, a marker of active enhancers (Figure S6E), as expected. ChIP-seq revealed a significant reduction in H3K27ac level (p = 0.01) at a peak immediately downstream of the ACE2 transcription start site in HMGB1-disrupted cells compared with control cells (Figures 6G and S6D). In addition, we observed a trend toward reduced chromatin accessibility at the overlapping ATAC-seq peak at the ACE2 locus (p = 0.057) in HMGB1-disrupted cells compared with control cells (Figures 6G and S6C). HMGB1 is necessary for ACE2 expression as well as viral entry of SARS-CoV-1, SARS-CoV-2, and NL63, but not MERS-CoV, which mirrors receptor utilization (Figures 6F and 6H). These findings demonstrate that HMGB1 is a novel regulator of ACE2 expression that affects susceptibility to SARS-CoV-2.

CRISPR Screens of Highly Pathogenic CoVs Reveal Host Genes Essential for InfectionDiscovery of host genes and pathways that mediate pathogenesis of pandemic CoVs is a critical resource that may promote our understanding of how these viruses cause disease, why there is variable host susceptibility, and the origins of host species range and may reveal host-directed therapeutic targets against known and unknown CoVs of pandemic potential. To identify host factors essential for cell survival in response to pandemic human CoVs, including SARS-CoV-1, SARS-CoV-2, and MERS-CoV, we used Vero-E6 cells, a model cell line for isolating viruses that was selected based on its susceptibility and cytopathic effects in response to all three pandemic CoVs (Matsuyama et al., 2020; Ogando et al., 2020; Woolsey et al., 2020). We performed genome-wide CRISPR screens with SARS-CoV-2 (isolate USA-WA1/2020), MERS-CoV (EMC/2012), a tissue culture-adapted MERS-CoV (T1015N), and a recombinant bat CoV (HKU5) containing the spike protein of SARS-CoV-1 (HKU5-SARS-CoV-1-S) (Scobey et al., 2013; Agnihothram et al., 2014). HKU5-SARS-CoV-1-S was used as a surrogate for SARS-CoV-1, which is a select agent. Further, to test whether genes implicated in SARS-CoV-2 infection act at the level of viral entry, we performed a genome-wide screen with replication-competent vesicular stomatitis virus (VSV) expressing the SARS-CoV-2 spike protein (rcVSV-SARS-CoV-2-S). rcVSV-SARS-CoV-2-S is deficient for the broadly fusogenic VSV glycoprotein (VSV-G); thus, viral entry is entirely mediated by the SARS-CoV-2 spike protein (Case et al., 2020; Dieterle et al., 2020)We used a C. sabaeus genome-wide pooled CRISPR library composed of 83,963 targeting single guide RNAs (sgRNAs) with an average of four sgRNAs per gene and 1,000 non-targeting control sgRNAs. We initially performed two independent SARS-CoV-2 genome-wide screens with Vero-E6 lines expressing two different Cas9 nuclease constructs (Cas9-v1 and Cas9-v2); Cas9-v2 has an additional nuclear localization sequence to increase activity. We transduced both Vero-Cas9 cell lines with the C. sabaeus sgRNA library and challenged cells with SARS-CoV-2 (Figure 1
A). To generate a robust dataset, we performed independent screens at different cell densities, fetal bovine serum (FBS) concentrations, and multiplicities of infection (MOI). Z scores for all SARS-CoV-2 screens are shown in Table S1. Subsequent MERS-CoV wild type (WT), MERS-CoV T1015N, HKU5-SARS-CoV-1-S, and rcVSV-SARS-CoV-2-S screens were performed in duplicate under a single cell density, FBS concentration, and MOI. Genomic DNA was harvested from surviving cells 7–9 days post-infection (dpi), and guide abundance was determined by PCR and massively parallel sequencing. Technical performance of the screens is described in STAR Methods and Figure S1
.Figure 1Genome-wide CRISPR Screens Identify Genes Critical for CoV-Induced Cell Death(A) Schematic of the pooled screen. Vero-E6-Cas9 cells transduced with the genome-wide C. sabaeus library received mock treatment or were challenged with SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, MERS-CoV, or MERS-CoV T1015N. Surviving cells from each virus infection were isolated, and the sgRNA sequences were amplified by PCR and sequenced.(B) Volcano plot showing top genes conferring resistance and sensitivity to SARS-CoV-2. The gene-level Z score and −log10 (FDR) were calculated using the mean of the five Cas9-v2 conditions. Non-targeting control sgRNAs were grouped randomly into sets of 4 to serve as “dummy” genes and are shown in green.(C) Heatmaps of the top gene hits for SARS-CoV-2 resistance (20) and sensitization (20), ranked by mean Z score. The top 5 hits for MERS-CoV are also included and indicated by an asterisk. ARID1A was a top resistance gene for SARS-CoV-2 and MERS-CoV.(D–F) Correlation between gene enrichment in SARS-CoV-2 and rcVSV-SARS-CoV-2-S (D), HKU5-SARS-CoV-1-S (E), and MERS-CoV (F) screens. R, pearson correlation.(G) Venn diagram of the top 100 pro-viral genes from SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, and MERS-CoV screens.Figure S1Quality Control Metrics for CRISPR Screen, Related to STAR Methods(A) Correlation matrix depicting the Pearson correlation between the guide-level log-fold change values relative to the plasmid DNA. Cells were cultured in DMEM with 2% FBS (D2), 5% FBS (D5), 10% FBS (D10), plated at 2.5 × 106 or 5.0 × 106 cells per T150 flask and infected at a MOI 0.1 (hi) or MOI 0.01 (lo). (B) Receiver-operator characteristic (ROC) curve for the recovery of guides targeting essential genes in the mock-treated condition of the Cas9-v1 and Cas9-v2 screens. True positives are n = 1,528 essential genes (n = 6,178 guides); true negative genes are n = 622 non-essential genes (n = 2,504 guides). We mapped essential and non-essential genes, which were derived for human cell lines, to the African green monkey genome simply by matching gene symbols. AUC = area under curve. (C) Correlation between gene enrichment in Cas9-v1 and Cas9-v2 screens. Pearson correlation is reported. (D-E) GFP-based Cas9 activity assay in Vero-E6 cells stably expressing either Cas9-v1 (D) or Cas9-v2 (E). The pXPR_047 construct expresses GFP and an sgRNA targeting GFP; therefore, cells without Cas9 activity will express GFP, whereas cells with high Cas9 activity will knock out GFP and resemble parental cells. (F) Approach to calculate residuals from log-fold change data, using ACE2 and the 5% FBS, 5 × 106 cells/flask, MOI 0.1 condition as an example. A natural cubic spline with four degrees of freedom is shown in blue, and a residual for each sgRNA is calculated to be the vertical distance from the fit spline.The SARS-CoV-2 screen identified numerous genes that confer resistance (pro-viral) or sensitization (anti-viral) when targeted by sgRNAs, with a minimum false discovery rate (FDR) of 0.03 for non-targeting controls (Figures 1B and 1C), demonstrating high technical quality. The strongest resistance hit was the viral receptor ACE2 (mean Z score = 4.9, descending rank = 1; Figures 1B, 1C, S2
A, and S2B). CTSL, which encodes the Cathepsin L protease, was also positively selected under all conditions (mean Z score = 3.0, descending rank = 18; Figures 1B, 1C, and S2B). We did not observe enrichment of TMPRSS2 (mean Z score = 0.9, descending rank = 2,726) or the proteases TMPRSS4 or FURIN (mean Z scores = 1.1, 0.4; descending ranks = 1,657, 7,180, respectively), which have also been implicated in SARS-CoV-2 entry (Hoffmann et al., 2020b; Shang et al., 2020; Zang et al., 2020), suggesting that these proteases are not essential in a cell-intrinsic manner for SARS-CoV-2-induced cell death as performed here and/or are functionally redundant.Figure S2A Genome-wide CRISPR Screen Identifies Genes Critical for SARS-CoV-2-Induced Cell Death, Related to Figure 1(A) Performance of individual sgRNAs targeting ACE2, SMARCA4, CTSL, and TMPRSS2. The mean residual across the five Cas9-v2 conditions is plotted for the full library (top) and for the 4 guide RNAs targeting each gene. (B)Heatmaps of the top 25 gene hits for resistance and sensitivity, ranked by mean z-score in the Cas9-v2 conditions. Genes that are included in one of the gene sets labeled in (Figure 2A) are colored accordingly. Condition A: Cas9-v2 D5 (DMEM+5%FBS) 2.5 × 106 cells/flask MOI 0.1; B: Cas9-v2 D5 5 × 106 cells/flask MO 0.1; C: Cas9-v2 D2 (DMEM+2%FBS) 5 × 106 cells/flask MOI 0.1; D: Cas9-v2 D10 (DMEM+10%FBS) 5 × 106 cells/flask MOI 0.1; E: Cas9-v2 D5 2.5 × 106 cells/flask MOI 0.01. (C) Nodes represent significantly enriched gene sets. The size of each gene set is proportional to its mean absolute z-score. Gene sets are colored by the direction in which they score. Edges represent significant overlap between gene sets. The transparency of each edge is proportional to the fraction of genes shared by two gene sets. Gene sets were clustered using the infomap algorithm and the most central set by PageRank is labeled for each cluster. The Fruchterman–Reingold algorithm was used to lay out the network.Comparison of SARS-CoV-2 with rcVSV-SARS-CoV-2-S revealed substantial concordance between the viruses, suggesting that concordant genes act at the level of viral entry (Figures 1C and 1D). SARS-CoV-2 and HKU5-SARS-CoV-1-S screens also yielded a similar overlap, consistent with similar mechanisms of entry as mediated by the SARS-CoV-1 and SARS-CoV-2 spike proteins (Figure 1E). Next we assessed the relationship between MERS-CoV and SARS-CoV-2 screens. The MERS-CoV receptor DPP4 was the top resistance hit in the MERS-CoV screen, whereas ACE2 was not enriched (Figure 1F). The SARS-CoV-2 resistance genes ARID1A, DYRK1A, KDM6A, and CTSL were also highly enriched in the MERS-CoV screen (Figure 1F). Pairwise correlations of all genome-wide screens are shown in Figure S3
, and Z scores for all genes are shown in Table S2. Next we compared the top 100 resistance genes across the genome-wide screens with SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, and MERS-CoV (WT) viruses. Five genes (ARID1A, KDM6A, JMJD6, SMARCC1, and CTSL) scored in each of the four virus screens. That these genes were enriched in the SARS-CoV-2, HKU5-SARS-CoV-1-S, and MERS-CoV (WT) screens suggest that they are potentially pan-coronaviral. Because they were also enriched in the rcVSV-SARS-CoV-2-S screen, they act at the level of CoV entry (Figure 1G). This suggests that these genes promote lineage-independent entry of pandemic CoVs. 14 genes scored in the three SARS-lineage viruses but not MERS-CoV, including ACE2, HMGB1, SMARCA4, DYRK1A, and DPF2, suggesting that these genes mediate entry of SARS-lineage viruses (Figure 1G). The MERS receptor DPP4 along with AXIN1 and TMEM41B were identified as MERS-CoV-specific pro-viral genes, whereas SMAD3 and SMAD4 were only enriched in the SARS-CoV-2 screens. Among sensitization genes, BPTF, which encodes the scaffold for the Nucleosome Remodeling Factor (NURF) chromatin remodeling complex, was broadly depleted for all four viruses. Similarly, the interferon-stimulated gene LY6E, which was recently identified as a pan-coronaviral entry inhibitor, was identified as an anti-viral gene for SARS-lineage and MERS-CoV screens (Pfaender et al., 2020). This is despite Vero-E6 cells being deficient in type I interferon. Overall, these results show that a survival assay in Vero-E6 cells is able to distinguish host factors common to and specific for a variety of pathogenic CoVs.Figure S3Comparison of All Viruses from Genome-wide CRISPR Screens, Related to Figure 1Comparison of gene enrichment of (A) SARS-CoV-2 relative to MERS-CoV T1015N, (B) rcVSV-SARS-CoV-2-S relative to HKU5-SARS-CoV-1-S, (C) rcVSV-SARS-CoV-2-S relative to MERS-CoV WT, (D) rcVSV-SARS-CoV-2-S relative to MERS-CoV T1015N. (E) HKU5-SARS-CoV-1-S relative to MERS-CoV WT, (F) HKU5-SARS-CoV-1-S relative to MERS-CoV T1015N, and (G) MERS-CoV WT relative to MERS-CoV T1015N. Pearson correlation is reported.Although we validated all genome-wide screens, we focused on SARS-CoV-2, given the immediacy of the current pandemic. To systematically identify hit gene sets enriched in the SARS-CoV-2 screen, we used the STRING-db enrichment detection tool and identified 623 significant gene sets (Figure S2C) from 10 sources (e.g., Kyoto Encyclopedia of Genes and Genomes [KEGG], Gene Ontology [GO] process) (Szklarczyk et al., 2019). The top gene sets that scored in the positive direction (pro-viral), negative direction (anti-viral), or both directions and their respective genes are shown in Figure 2
. SMARCA4 (BRG1), the catalytic subunit of the SWI/SNF remodeling complex (Centore et al., 2020), scored after ACE2 as the second-strongest SARS-CoV-2 resistance hit (mean Z score = 4.9; descending rank = 2; Figures 1B–1D), with several other members, including ARID1A, SMARCE1, SMARCB1, and SMARCC1, showing enrichment (mean Z scores = 3.6, 2.8, 2.4, 2.3; descending ranks = 9, 20, 47, 59, respectively; Figure 2B). The SWI/SNF complex is an ATP-dependent nucleosome remodeling complex that regulates chromatin accessibility and gene expression (Zhou et al., 2016; Clapier et al., 2017). Interestingly, although the SWI/SNF complex genes ARID1A, SMARCB1, and SMARCC1 were enriched in SARS-lineage and MERS-CoV screens, SMARCA4 was only enriched in the SARS-lineage screens. We also identified several other histone-modifying enzymes as key regulators of SARS-CoV-2-induced cell death (Figures 1C and S2B). Other pro-viral genes include the histone demethylase KDM6A (mean Z score = 4.1, descending rank = 4), the histone methyltransferase KMT2D (mean Z score = 2.6, descending rank = 35), as well as the lysyl hydroxylase JMJD6 (mean Z score = 3.7, descending rank = 6) (Figures 1C and S2B; Webby et al., 2009; Hu et al., 2013; Unoki et al., 2013; Chakraborty et al., 2019). In contrast, sgRNAs targeting HIRA, CABIN1, and ASF1A were negatively selected, revealing an anti-viral function. These genes encode three of the four proteins in the HIRA/UBN1/CABIN1/ASF1a (HUCA) histone H3.3 chaperone complex (mean Z scores = −5.7, −5.4, −3.0; ascending ranks = 1, 2, 64, respectively), suggesting an anti-viral role for deposition of the histone variant H3.3 (Rai et al., 2011).Figure 2Performance of Genes in the Top Gene Sets(A) The top three gene sets that score in the positive direction (resistance) and top gene set that scores in the negative direction (sensitization) or both, filtered for gene sets with at least five genes, and that are most central to a given module (Figure S2C) and then ranked by mean absolute Z score. The number of genes in each set is indicated in parentheses.(B) For each gene in the “SWI/SNF complex” gene set from STRING, the Z score in each virus screen is shown.(C–F) Similarly, the genes in the gene sets (C) “RUNX3 regulates CDKN1A transcription” from Reactome, (D) “Cystatin, and endolysosome lumen” from STRING, (E) “Viral translation” from GO, and (F) “NURF complex” from GO.We additionally observed enrichment in the “RUNX3 regulates CDKN1A transcription” gene set from Reactome (Figures 2A and 2C). This is driven by enrichment of sgRNAs targeting the signal transducers SMAD3 and SMAD4 (mean Z scores = 2.8, 3.1; descending ranks = 21, 15, respectively). The “cystatin and endolysosome lumen” gene set, which includes CTSL, was also enriched (Figure 2D). Not surprisingly, we observed the enrichment of “viral translation” in positive and negative directions (Figure 2E). The NURF complex was the top gene set enriched in the negative direction (Figure 2F).

CRISPR Screens of Highly Pathogenic CoVs Reveal Host Genes Essential for Infection

Discovery of host genes and pathways that mediate pathogenesis of pandemic CoVs is a critical resource that may promote our understanding of how these viruses cause disease, why there is variable host susceptibility, and the origins of host species range and may reveal host-directed therapeutic targets against known and unknown CoVs of pandemic potential. To identify host factors essential for cell survival in response to pandemic human CoVs, including SARS-CoV-1, SARS-CoV-2, and MERS-CoV, we used Vero-E6 cells, a model cell line for isolating viruses that was selected based on its susceptibility and cytopathic effects in response to all three pandemic CoVs (Matsuyama et al., 2020; Ogando et al., 2020; Woolsey et al., 2020). We performed genome-wide CRISPR screens with SARS-CoV-2 (isolate USA-WA1/2020), MERS-CoV (EMC/2012), a tissue culture-adapted MERS-CoV (T1015N), and a recombinant bat CoV (HKU5) containing the spike protein of SARS-CoV-1 (HKU5-SARS-CoV-1-S) (Scobey et al., 2013; Agnihothram et al., 2014). HKU5-SARS-CoV-1-S was used as a surrogate for SARS-CoV-1, which is a select agent. Further, to test whether genes implicated in SARS-CoV-2 infection act at the level of viral entry, we performed a genome-wide screen with replication-competent vesicular stomatitis virus (VSV) expressing the SARS-CoV-2 spike protein (rcVSV-SARS-CoV-2-S). rcVSV-SARS-CoV-2-S is deficient for the broadly fusogenic VSV glycoprotein (VSV-G); thus, viral entry is entirely mediated by the SARS-CoV-2 spike protein (Case et al., 2020; Dieterle et al., 2020)

We used a C. sabaeus genome-wide pooled CRISPR library composed of 83,963 targeting single guide RNAs (sgRNAs) with an average of four sgRNAs per gene and 1,000 non-targeting control sgRNAs. We initially performed two independent SARS-CoV-2 genome-wide screens with Vero-E6 lines expressing two different Cas9 nuclease constructs (Cas9-v1 and Cas9-v2); Cas9-v2 has an additional nuclear localization sequence to increase activity. We transduced both Vero-Cas9 cell lines with the C. sabaeus sgRNA library and challenged cells with SARS-CoV-2 (Figure 1
A). To generate a robust dataset, we performed independent screens at different cell densities, fetal bovine serum (FBS) concentrations, and multiplicities of infection (MOI). Z scores for all SARS-CoV-2 screens are shown in Table S1. Subsequent MERS-CoV wild type (WT), MERS-CoV T1015N, HKU5-SARS-CoV-1-S, and rcVSV-SARS-CoV-2-S screens were performed in duplicate under a single cell density, FBS concentration, and MOI. Genomic DNA was harvested from surviving cells 7–9 days post-infection (dpi), and guide abundance was determined by PCR and massively parallel sequencing. Technical performance of the screens is described in STAR Methods and Figure S1
.Figure 1Genome-wide CRISPR Screens Identify Genes Critical for CoV-Induced Cell Death(A) Schematic of the pooled screen. Vero-E6-Cas9 cells transduced with the genome-wide C. sabaeus library received mock treatment or were challenged with SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, MERS-CoV, or MERS-CoV T1015N. Surviving cells from each virus infection were isolated, and the sgRNA sequences were amplified by PCR and sequenced.(B) Volcano plot showing top genes conferring resistance and sensitivity to SARS-CoV-2. The gene-level Z score and −log10 (FDR) were calculated using the mean of the five Cas9-v2 conditions. Non-targeting control sgRNAs were grouped randomly into sets of 4 to serve as “dummy” genes and are shown in green.(C) Heatmaps of the top gene hits for SARS-CoV-2 resistance (20) and sensitization (20), ranked by mean Z score. The top 5 hits for MERS-CoV are also included and indicated by an asterisk. ARID1A was a top resistance gene for SARS-CoV-2 and MERS-CoV.(D–F) Correlation between gene enrichment in SARS-CoV-2 and rcVSV-SARS-CoV-2-S (D), HKU5-SARS-CoV-1-S (E), and MERS-CoV (F) screens. R, pearson correlation.(G) Venn diagram of the top 100 pro-viral genes from SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, and MERS-CoV screens.Figure S1Quality Control Metrics for CRISPR Screen, Related to STAR Methods(A) Correlation matrix depicting the Pearson correlation between the guide-level log-fold change values relative to the plasmid DNA. Cells were cultured in DMEM with 2% FBS (D2), 5% FBS (D5), 10% FBS (D10), plated at 2.5 × 106 or 5.0 × 106 cells per T150 flask and infected at a MOI 0.1 (hi) or MOI 0.01 (lo). (B) Receiver-operator characteristic (ROC) curve for the recovery of guides targeting essential genes in the mock-treated condition of the Cas9-v1 and Cas9-v2 screens. True positives are n = 1,528 essential genes (n = 6,178 guides); true negative genes are n = 622 non-essential genes (n = 2,504 guides). We mapped essential and non-essential genes, which were derived for human cell lines, to the African green monkey genome simply by matching gene symbols. AUC = area under curve. (C) Correlation between gene enrichment in Cas9-v1 and Cas9-v2 screens. Pearson correlation is reported. (D-E) GFP-based Cas9 activity assay in Vero-E6 cells stably expressing either Cas9-v1 (D) or Cas9-v2 (E). The pXPR_047 construct expresses GFP and an sgRNA targeting GFP; therefore, cells without Cas9 activity will express GFP, whereas cells with high Cas9 activity will knock out GFP and resemble parental cells. (F) Approach to calculate residuals from log-fold change data, using ACE2 and the 5% FBS, 5 × 106 cells/flask, MOI 0.1 condition as an example. A natural cubic spline with four degrees of freedom is shown in blue, and a residual for each sgRNA is calculated to be the vertical distance from the fit spline.

Genome-wide CRISPR Screens Identify Genes Critical for CoV-Induced Cell Death

(A) Schematic of the pooled screen. Vero-E6-Cas9 cells transduced with the genome-wide C. sabaeus library received mock treatment or were challenged with SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, MERS-CoV, or MERS-CoV T1015N. Surviving cells from each virus infection were isolated, and the sgRNA sequences were amplified by PCR and sequenced.

(B) Volcano plot showing top genes conferring resistance and sensitivity to SARS-CoV-2. The gene-level Z score and −log10 (FDR) were calculated using the mean of the five Cas9-v2 conditions. Non-targeting control sgRNAs were grouped randomly into sets of 4 to serve as “dummy” genes and are shown in green.

(C) Heatmaps of the top gene hits for SARS-CoV-2 resistance (20) and sensitization (20), ranked by mean Z score. The top 5 hits for MERS-CoV are also included and indicated by an asterisk. ARID1A was a top resistance gene for SARS-CoV-2 and MERS-CoV.

(D–F) Correlation between gene enrichment in SARS-CoV-2 and rcVSV-SARS-CoV-2-S (D), HKU5-SARS-CoV-1-S (E), and MERS-CoV (F) screens. R, pearson correlation.

(G) Venn diagram of the top 100 pro-viral genes from SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, and MERS-CoV screens.

Quality Control Metrics for CRISPR Screen, Related to STAR Methods

(A) Correlation matrix depicting the Pearson correlation between the guide-level log-fold change values relative to the plasmid DNA. Cells were cultured in DMEM with 2% FBS (D2), 5% FBS (D5), 10% FBS (D10), plated at 2.5 × 106 or 5.0 × 106 cells per T150 flask and infected at a MOI 0.1 (hi) or MOI 0.01 (lo). (B) Receiver-operator characteristic (ROC) curve for the recovery of guides targeting essential genes in the mock-treated condition of the Cas9-v1 and Cas9-v2 screens. True positives are n = 1,528 essential genes (n = 6,178 guides); true negative genes are n = 622 non-essential genes (n = 2,504 guides). We mapped essential and non-essential genes, which were derived for human cell lines, to the African green monkey genome simply by matching gene symbols. AUC = area under curve. (C) Correlation between gene enrichment in Cas9-v1 and Cas9-v2 screens. Pearson correlation is reported. (D-E) GFP-based Cas9 activity assay in Vero-E6 cells stably expressing either Cas9-v1 (D) or Cas9-v2 (E). The pXPR_047 construct expresses GFP and an sgRNA targeting GFP; therefore, cells without Cas9 activity will express GFP, whereas cells with high Cas9 activity will knock out GFP and resemble parental cells. (F) Approach to calculate residuals from log-fold change data, using ACE2 and the 5% FBS, 5 × 106 cells/flask, MOI 0.1 condition as an example. A natural cubic spline with four degrees of freedom is shown in blue, and a residual for each sgRNA is calculated to be the vertical distance from the fit spline.

The SARS-CoV-2 screen identified numerous genes that confer resistance (pro-viral) or sensitization (anti-viral) when targeted by sgRNAs, with a minimum false discovery rate (FDR) of 0.03 for non-targeting controls (Figures 1B and 1C), demonstrating high technical quality. The strongest resistance hit was the viral receptor ACE2 (mean Z score = 4.9, descending rank = 1; Figures 1B, 1C, S2
A, and S2B). CTSL, which encodes the Cathepsin L protease, was also positively selected under all conditions (mean Z score = 3.0, descending rank = 18; Figures 1B, 1C, and S2B). We did not observe enrichment of TMPRSS2 (mean Z score = 0.9, descending rank = 2,726) or the proteases TMPRSS4 or FURIN (mean Z scores = 1.1, 0.4; descending ranks = 1,657, 7,180, respectively), which have also been implicated in SARS-CoV-2 entry (Hoffmann et al., 2020b; Shang et al., 2020; Zang et al., 2020), suggesting that these proteases are not essential in a cell-intrinsic manner for SARS-CoV-2-induced cell death as performed here and/or are functionally redundant.Figure S2A Genome-wide CRISPR Screen Identifies Genes Critical for SARS-CoV-2-Induced Cell Death, Related to Figure 1(A) Performance of individual sgRNAs targeting ACE2, SMARCA4, CTSL, and TMPRSS2. The mean residual across the five Cas9-v2 conditions is plotted for the full library (top) and for the 4 guide RNAs targeting each gene. (B)Heatmaps of the top 25 gene hits for resistance and sensitivity, ranked by mean z-score in the Cas9-v2 conditions. Genes that are included in one of the gene sets labeled in (Figure 2A) are colored accordingly. Condition A: Cas9-v2 D5 (DMEM+5%FBS) 2.5 × 106 cells/flask MOI 0.1; B: Cas9-v2 D5 5 × 106 cells/flask MO 0.1; C: Cas9-v2 D2 (DMEM+2%FBS) 5 × 106 cells/flask MOI 0.1; D: Cas9-v2 D10 (DMEM+10%FBS) 5 × 106 cells/flask MOI 0.1; E: Cas9-v2 D5 2.5 × 106 cells/flask MOI 0.01. (C) Nodes represent significantly enriched gene sets. The size of each gene set is proportional to its mean absolute z-score. Gene sets are colored by the direction in which they score. Edges represent significant overlap between gene sets. The transparency of each edge is proportional to the fraction of genes shared by two gene sets. Gene sets were clustered using the infomap algorithm and the most central set by PageRank is labeled for each cluster. The Fruchterman–Reingold algorithm was used to lay out the network.

A Genome-wide CRISPR Screen Identifies Genes Critical for SARS-CoV-2-Induced Cell Death, Related to Figure 1

(A) Performance of individual sgRNAs targeting ACE2, SMARCA4, CTSL, and TMPRSS2. The mean residual across the five Cas9-v2 conditions is plotted for the full library (top) and for the 4 guide RNAs targeting each gene. (B)Heatmaps of the top 25 gene hits for resistance and sensitivity, ranked by mean z-score in the Cas9-v2 conditions. Genes that are included in one of the gene sets labeled in (Figure 2A) are colored accordingly. Condition A: Cas9-v2 D5 (DMEM+5%FBS) 2.5 × 106 cells/flask MOI 0.1; B: Cas9-v2 D5 5 × 106 cells/flask MO 0.1; C: Cas9-v2 D2 (DMEM+2%FBS) 5 × 106 cells/flask MOI 0.1; D: Cas9-v2 D10 (DMEM+10%FBS) 5 × 106 cells/flask MOI 0.1; E: Cas9-v2 D5 2.5 × 106 cells/flask MOI 0.01. (C) Nodes represent significantly enriched gene sets. The size of each gene set is proportional to its mean absolute z-score. Gene sets are colored by the direction in which they score. Edges represent significant overlap between gene sets. The transparency of each edge is proportional to the fraction of genes shared by two gene sets. Gene sets were clustered using the infomap algorithm and the most central set by PageRank is labeled for each cluster. The Fruchterman–Reingold algorithm was used to lay out the network.

Comparison of SARS-CoV-2 with rcVSV-SARS-CoV-2-S revealed substantial concordance between the viruses, suggesting that concordant genes act at the level of viral entry (Figures 1C and 1D). SARS-CoV-2 and HKU5-SARS-CoV-1-S screens also yielded a similar overlap, consistent with similar mechanisms of entry as mediated by the SARS-CoV-1 and SARS-CoV-2 spike proteins (Figure 1E). Next we assessed the relationship between MERS-CoV and SARS-CoV-2 screens. The MERS-CoV receptor DPP4 was the top resistance hit in the MERS-CoV screen, whereas ACE2 was not enriched (Figure 1F). The SARS-CoV-2 resistance genes ARID1A, DYRK1A, KDM6A, and CTSL were also highly enriched in the MERS-CoV screen (Figure 1F). Pairwise correlations of all genome-wide screens are shown in Figure S3
, and Z scores for all genes are shown in Table S2. Next we compared the top 100 resistance genes across the genome-wide screens with SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, and MERS-CoV (WT) viruses. Five genes (ARID1A, KDM6A, JMJD6, SMARCC1, and CTSL) scored in each of the four virus screens. That these genes were enriched in the SARS-CoV-2, HKU5-SARS-CoV-1-S, and MERS-CoV (WT) screens suggest that they are potentially pan-coronaviral. Because they were also enriched in the rcVSV-SARS-CoV-2-S screen, they act at the level of CoV entry (Figure 1G). This suggests that these genes promote lineage-independent entry of pandemic CoVs. 14 genes scored in the three SARS-lineage viruses but not MERS-CoV, including ACE2, HMGB1, SMARCA4, DYRK1A, and DPF2, suggesting that these genes mediate entry of SARS-lineage viruses (Figure 1G). The MERS receptor DPP4 along with AXIN1 and TMEM41B were identified as MERS-CoV-specific pro-viral genes, whereas SMAD3 and SMAD4 were only enriched in the SARS-CoV-2 screens. Among sensitization genes, BPTF, which encodes the scaffold for the Nucleosome Remodeling Factor (NURF) chromatin remodeling complex, was broadly depleted for all four viruses. Similarly, the interferon-stimulated gene LY6E, which was recently identified as a pan-coronaviral entry inhibitor, was identified as an anti-viral gene for SARS-lineage and MERS-CoV screens (Pfaender et al., 2020). This is despite Vero-E6 cells being deficient in type I interferon. Overall, these results show that a survival assay in Vero-E6 cells is able to distinguish host factors common to and specific for a variety of pathogenic CoVs.Figure S3Comparison of All Viruses from Genome-wide CRISPR Screens, Related to Figure 1Comparison of gene enrichment of (A) SARS-CoV-2 relative to MERS-CoV T1015N, (B) rcVSV-SARS-CoV-2-S relative to HKU5-SARS-CoV-1-S, (C) rcVSV-SARS-CoV-2-S relative to MERS-CoV WT, (D) rcVSV-SARS-CoV-2-S relative to MERS-CoV T1015N. (E) HKU5-SARS-CoV-1-S relative to MERS-CoV WT, (F) HKU5-SARS-CoV-1-S relative to MERS-CoV T1015N, and (G) MERS-CoV WT relative to MERS-CoV T1015N. Pearson correlation is reported.

Comparison of All Viruses from Genome-wide CRISPR Screens, Related to Figure 1

Comparison of gene enrichment of (A) SARS-CoV-2 relative to MERS-CoV T1015N, (B) rcVSV-SARS-CoV-2-S relative to HKU5-SARS-CoV-1-S, (C) rcVSV-SARS-CoV-2-S relative to MERS-CoV WT, (D) rcVSV-SARS-CoV-2-S relative to MERS-CoV T1015N. (E) HKU5-SARS-CoV-1-S relative to MERS-CoV WT, (F) HKU5-SARS-CoV-1-S relative to MERS-CoV T1015N, and (G) MERS-CoV WT relative to MERS-CoV T1015N. Pearson correlation is reported.

Although we validated all genome-wide screens, we focused on SARS-CoV-2, given the immediacy of the current pandemic. To systematically identify hit gene sets enriched in the SARS-CoV-2 screen, we used the STRING-db enrichment detection tool and identified 623 significant gene sets (Figure S2C) from 10 sources (e.g., Kyoto Encyclopedia of Genes and Genomes [KEGG], Gene Ontology [GO] process) (Szklarczyk et al., 2019). The top gene sets that scored in the positive direction (pro-viral), negative direction (anti-viral), or both directions and their respective genes are shown in Figure 2
. SMARCA4 (BRG1), the catalytic subunit of the SWI/SNF remodeling complex (Centore et al., 2020), scored after ACE2 as the second-strongest SARS-CoV-2 resistance hit (mean Z score = 4.9; descending rank = 2; Figures 1B–1D), with several other members, including ARID1A, SMARCE1, SMARCB1, and SMARCC1, showing enrichment (mean Z scores = 3.6, 2.8, 2.4, 2.3; descending ranks = 9, 20, 47, 59, respectively; Figure 2B). The SWI/SNF complex is an ATP-dependent nucleosome remodeling complex that regulates chromatin accessibility and gene expression (Zhou et al., 2016; Clapier et al., 2017). Interestingly, although the SWI/SNF complex genes ARID1A, SMARCB1, and SMARCC1 were enriched in SARS-lineage and MERS-CoV screens, SMARCA4 was only enriched in the SARS-lineage screens. We also identified several other histone-modifying enzymes as key regulators of SARS-CoV-2-induced cell death (Figures 1C and S2B). Other pro-viral genes include the histone demethylase KDM6A (mean Z score = 4.1, descending rank = 4), the histone methyltransferase KMT2D (mean Z score = 2.6, descending rank = 35), as well as the lysyl hydroxylase JMJD6 (mean Z score = 3.7, descending rank = 6) (Figures 1C and S2B; Webby et al., 2009; Hu et al., 2013; Unoki et al., 2013; Chakraborty et al., 2019). In contrast, sgRNAs targeting HIRA, CABIN1, and ASF1A were negatively selected, revealing an anti-viral function. These genes encode three of the four proteins in the HIRA/UBN1/CABIN1/ASF1a (HUCA) histone H3.3 chaperone complex (mean Z scores = −5.7, −5.4, −3.0; ascending ranks = 1, 2, 64, respectively), suggesting an anti-viral role for deposition of the histone variant H3.3 (Rai et al., 2011).Figure 2Performance of Genes in the Top Gene Sets(A) The top three gene sets that score in the positive direction (resistance) and top gene set that scores in the negative direction (sensitization) or both, filtered for gene sets with at least five genes, and that are most central to a given module (Figure S2C) and then ranked by mean absolute Z score. The number of genes in each set is indicated in parentheses.(B) For each gene in the “SWI/SNF complex” gene set from STRING, the Z score in each virus screen is shown.(C–F) Similarly, the genes in the gene sets (C) “RUNX3 regulates CDKN1A transcription” from Reactome, (D) “Cystatin, and endolysosome lumen” from STRING, (E) “Viral translation” from GO, and (F) “NURF complex” from GO.

Performance of Genes in the Top Gene Sets

(A) The top three gene sets that score in the positive direction (resistance) and top gene set that scores in the negative direction (sensitization) or both, filtered for gene sets with at least five genes, and that are most central to a given module (Figure S2C) and then ranked by mean absolute Z score. The number of genes in each set is indicated in parentheses.

(B) For each gene in the “SWI/SNF complex” gene set from STRING, the Z score in each virus screen is shown.

(C–F) Similarly, the genes in the gene sets (C) “RUNX3 regulates CDKN1A transcription” from Reactome, (D) “Cystatin, and endolysosome lumen” from STRING, (E) “Viral translation” from GO, and (F) “NURF complex” from GO.

We additionally observed enrichment in the “RUNX3 regulates CDKN1A transcription” gene set from Reactome (Figures 2A and 2C). This is driven by enrichment of sgRNAs targeting the signal transducers SMAD3 and SMAD4 (mean Z scores = 2.8, 3.1; descending ranks = 21, 15, respectively). The “cystatin and endolysosome lumen” gene set, which includes CTSL, was also enriched (Figure 2D). Not surprisingly, we observed the enrichment of “viral translation” in positive and negative directions (Figure 2E). The NURF complex was the top gene set enriched in the negative direction (Figure 2F).

Pooled and Arrayed Validation Confirms Genome-wide CRISPR Screen Hits and Reveals Virus SpecificityTo validate the genome-wide screens, we generated a custom CRISPR subpool containing 10 sgRNAs for each of the top 250 and bottom 250 genes from an earlier analysis of the SARS-CoV-2 screen along with 500 non-targeting control sgRNAs and sgRNAs targeting other genes of interest, such as the MERS-CoV receptor DPP4. We introduced this sgRNA library into Vero-Cas9-v2 cells and challenged this pool with SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, MERS-CoV, or MERS-CoV T1015N. We included the orthomyxovirus IAV (A/WSN/1933), the picornavirus EMCV, and VSV (Indiana) as control viruses, which all cause cytopathic effects in Vero-E6 cells. All eight viruses were screened in duplicate except for IAV. The secondary screen validated the top pro-viral and anti-viral genes from the primary genome-wide SARS-CoV-2 screen (Figure 3
A; Table S3). Clustering the correlations between the log2 fold changes of each condition revealed that viruses containing a SARS-lineage spike grouped together, as did MERS-CoV WT and T1015N, whereas IAV and EMCV were outliers, as expected (Figure 3B). A principal-component analysis of the secondary screens revealed clusters of gene hits in an unbiased manner (Figure 3C). Focusing on the top resistance hits for each virus screen, we saw SARS lineage-specific (e.g., ACE2 and PHIP), MERS lineage-specific (e.g., DPP4), and pan-CoV-specific genes (e.g., CTSL, ARID1A, PCBD1, and KMT2D) (Figure S4
A). Consistent with the genome-wide CRISPR screens, the key genes encoding members of the SWI/SNF complex are pro-viral for CoVs but not IAV or EMCV (Figure S4B). We observed that SMAD3 and SMAD4 were modestly enriched in the SARS-CoV-2 and rcVSV-SARS-CoV-2-S subpool screens (Figure S4C). In addition, the top sensitization genes involved in the HUCA histone H3.3 chaperone complex were specific to the SARS-lineage spike protein (Figure S4D). The secondary screens for rcVSV-SARS-CoV-2-S and HKU5-SARS-CoV-1-S showed strong agreement with the SARS-CoV-2 secondary screen with correlation coefficients of 0.90 and 0.89, respectively (Figures 3D and 3E). The MERS-CoV secondary screen correlated with the SARS-CoV-2 screen to a lesser extent than rcVSV-SARS-CoV-2-S and HKU5-SARS-CoV-1-S (r = 0.43; Figure 3F). Minimal correlation was observed between IAV or EMCV and SARS-CoV-2, with correlations of 0.23 and 0.042, respectively (Figures 3G and 3H). No cells survived infection with VSV, precluding analysis. This demonstrates the virus specificity of the identified host genes and provides insights into the stage of the virus life cycle mediated by critical genes.Figure 3CRISPR Subpool Screens Validate Primary Genome-wide Screens and Demonstrate the Specificity of Hits for CoVsA CRISPR subpool was generated with 10 sgRNAs per gene for each of the top 250 and bottom 250 genes from the SARS-CoV-2 genome-wide screen along with non-targeting controls and other genes of interest, including DPP4.(A) Correlation between gene enrichment in primary genome-wide and secondary subpool SARS-CoV-2 subscreens. Pearson correlation is reported.(B) Correlation matrix depicting the Pearson correlation between the guide-level log-fold change values relative to the plasmid DNA for the 13 subpool screens with the indicated viruses. All viruses were screened in duplicate (#1 and #2), except IAV-WSN. VSV was also screened, but no cells survived infection.(C) Principal-component analysis (PCA) plot of all viruses reveals clustering and overlap of gene hits among SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, and MERS-CoV WT and the T1015N cluster. IAV/WSN/1933 (IAV-WSN) and encephalomyocarditis virus (EMCV) are outliers among the CoV screens.(D–H) Comparison of gene enrichment in SARS-CoV-2 relative to rcVSV-SARS-CoV-2-S (D), HKU5-SARS-CoV-1-S (E), MERS-CoV (F), IAV-WSN (G), and EMCV (H). Pearson correlation is reported.(I) We generated a CRISPR subpool targeting 32 genes (inclusive of control genes) in the human lung cancer cell line Calu-3. Gene enrichment from the primary SARS-CoV-2 screen correlates with results from Calu-3 cells. Pearson correlation is reported.Figure S4Comparison of Secondary CRISPR Subpool Screens, Related to Figure 3(A) Heatmap depicting genes with a z-score > 10 in any of the secondary subpool screens. (B) Heatmap showing genes involved in the SWI/SNF chromatin remodeling complex. (C) Heatmap showing genes in the “Runx3 regulates CDKN1A transcription” pathway. (D) Heatmap showing genes in the HUCA histone H3.3 chaperone complex.To begin to understand the generalizability of these hits to human cells, we screened a small CRISPR subpool targeting 32 genes in Calu-3 cells, a lung adenocarcinoma line that natively expresses ACE2 (Table S4). This screen revealed significant overlap in pro-viral (including ACE2, DYRK1A, SMARCA4, KDM6A, JMJD6, SMARCE1, and SIAH1) and anti-viral genes (including HIRA, PIAS2, and SMARCA5) between Vero-E6 and Calu-3 cells (Figure 3I), suggesting the conserved role of these genetic hits across species. Notably, TMPRSS2, but not CTSL, was enriched in the Calu-3 subpool screen, consistent with TMPRSS2 expression in Calu-3 cells but not Vero-E6 cells (Böttcher-Friebertshäuser et al., 2011).We selected 25 genes for further validation of the SARS-CoV-2 screen in an arrayed rather than pooled format, consisting of 18 resistance and 7 sensitization genes (Figure 4
A). We transduced Vero-Cas9-v2 cells with one of 42 individual sgRNAs (1–3 sgRNAs per gene). We then challenged each of the 42 cell lines with SARS-CoV-2 and assessed cell viability. Cells receiving sgRNAs targeting pro-viral genes exhibited greater viability than those with non-targeting control sgRNAs. We validated the knockout efficiency of several genes (including ACE2, SMARCA4, KDM6A, and SMAD3) by western blot, and the protein abundance correlated with the degree of protection (Figures 4B–4D). Cells receiving sgRNAs targeting anti-viral genes in the primary screen exhibited increased susceptibility to cell death relative to controls, confirming the efficiency and reproducibility of the screen (Figures 4B and 4D). The robust concordance between the primary screens and the several subsequent validation approaches indicates that these screens will be a useful resource for further investigation of CoV host-pathogen interactions.Figure 4Arrayed Validation of 18 Resistance and 7 Sensitization Hit Genes(A) Performance in the pooled screen of sgRNAs targeting the 25 genes selected for further validation. The mean residual across the five Cas9-v2 conditions is plotted for the full library (top) and for the 3–4 sgRNAs targeting each gene. Genes that scored as resistance hits are shown in red; genes that scored as sensitization hits are shown in blue. The dashed line indicates a residual of 0.(B) 42 unique sgRNAs targeting 25 genes were introduced into Vero-E6-Cas9-v2 cells. SARS-CoV-2 was added at MOI 0.2, and cell viability was measured at 3 dpi.(C) Western blot for ACE2, SMARCA4, KDM6A, and SMAD3 expression in control and the respective gene-disrupted Vero-E6 cells.(D) Z scores from the genome-wide CRISPR screen correlate with cell viability of individually disrupted genes. Genes with multiple sgRNAs from (B) are averaged to generate one point per geneData were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM. ns, not statistically significant; ∗p < 0.05; ∗∗p < 0.01; ∗∗∗p < 0.001.

Pooled and Arrayed Validation Confirms Genome-wide CRISPR Screen Hits and Reveals Virus Specificity

To validate the genome-wide screens, we generated a custom CRISPR subpool containing 10 sgRNAs for each of the top 250 and bottom 250 genes from an earlier analysis of the SARS-CoV-2 screen along with 500 non-targeting control sgRNAs and sgRNAs targeting other genes of interest, such as the MERS-CoV receptor DPP4. We introduced this sgRNA library into Vero-Cas9-v2 cells and challenged this pool with SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, MERS-CoV, or MERS-CoV T1015N. We included the orthomyxovirus IAV (A/WSN/1933), the picornavirus EMCV, and VSV (Indiana) as control viruses, which all cause cytopathic effects in Vero-E6 cells. All eight viruses were screened in duplicate except for IAV. The secondary screen validated the top pro-viral and anti-viral genes from the primary genome-wide SARS-CoV-2 screen (Figure 3
A; Table S3). Clustering the correlations between the log2 fold changes of each condition revealed that viruses containing a SARS-lineage spike grouped together, as did MERS-CoV WT and T1015N, whereas IAV and EMCV were outliers, as expected (Figure 3B). A principal-component analysis of the secondary screens revealed clusters of gene hits in an unbiased manner (Figure 3C). Focusing on the top resistance hits for each virus screen, we saw SARS lineage-specific (e.g., ACE2 and PHIP), MERS lineage-specific (e.g., DPP4), and pan-CoV-specific genes (e.g., CTSL, ARID1A, PCBD1, and KMT2D) (Figure S4
A). Consistent with the genome-wide CRISPR screens, the key genes encoding members of the SWI/SNF complex are pro-viral for CoVs but not IAV or EMCV (Figure S4B). We observed that SMAD3 and SMAD4 were modestly enriched in the SARS-CoV-2 and rcVSV-SARS-CoV-2-S subpool screens (Figure S4C). In addition, the top sensitization genes involved in the HUCA histone H3.3 chaperone complex were specific to the SARS-lineage spike protein (Figure S4D). The secondary screens for rcVSV-SARS-CoV-2-S and HKU5-SARS-CoV-1-S showed strong agreement with the SARS-CoV-2 secondary screen with correlation coefficients of 0.90 and 0.89, respectively (Figures 3D and 3E). The MERS-CoV secondary screen correlated with the SARS-CoV-2 screen to a lesser extent than rcVSV-SARS-CoV-2-S and HKU5-SARS-CoV-1-S (r = 0.43; Figure 3F). Minimal correlation was observed between IAV or EMCV and SARS-CoV-2, with correlations of 0.23 and 0.042, respectively (Figures 3G and 3H). No cells survived infection with VSV, precluding analysis. This demonstrates the virus specificity of the identified host genes and provides insights into the stage of the virus life cycle mediated by critical genes.Figure 3CRISPR Subpool Screens Validate Primary Genome-wide Screens and Demonstrate the Specificity of Hits for CoVsA CRISPR subpool was generated with 10 sgRNAs per gene for each of the top 250 and bottom 250 genes from the SARS-CoV-2 genome-wide screen along with non-targeting controls and other genes of interest, including DPP4.(A) Correlation between gene enrichment in primary genome-wide and secondary subpool SARS-CoV-2 subscreens. Pearson correlation is reported.(B) Correlation matrix depicting the Pearson correlation between the guide-level log-fold change values relative to the plasmid DNA for the 13 subpool screens with the indicated viruses. All viruses were screened in duplicate (#1 and #2), except IAV-WSN. VSV was also screened, but no cells survived infection.(C) Principal-component analysis (PCA) plot of all viruses reveals clustering and overlap of gene hits among SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, and MERS-CoV WT and the T1015N cluster. IAV/WSN/1933 (IAV-WSN) and encephalomyocarditis virus (EMCV) are outliers among the CoV screens.(D–H) Comparison of gene enrichment in SARS-CoV-2 relative to rcVSV-SARS-CoV-2-S (D), HKU5-SARS-CoV-1-S (E), MERS-CoV (F), IAV-WSN (G), and EMCV (H). Pearson correlation is reported.(I) We generated a CRISPR subpool targeting 32 genes (inclusive of control genes) in the human lung cancer cell line Calu-3. Gene enrichment from the primary SARS-CoV-2 screen correlates with results from Calu-3 cells. Pearson correlation is reported.Figure S4Comparison of Secondary CRISPR Subpool Screens, Related to Figure 3(A) Heatmap depicting genes with a z-score > 10 in any of the secondary subpool screens. (B) Heatmap showing genes involved in the SWI/SNF chromatin remodeling complex. (C) Heatmap showing genes in the “Runx3 regulates CDKN1A transcription” pathway. (D) Heatmap showing genes in the HUCA histone H3.3 chaperone complex.

CRISPR Subpool Screens Validate Primary Genome-wide Screens and Demonstrate the Specificity of Hits for CoVs

A CRISPR subpool was generated with 10 sgRNAs per gene for each of the top 250 and bottom 250 genes from the SARS-CoV-2 genome-wide screen along with non-targeting controls and other genes of interest, including DPP4.

(A) Correlation between gene enrichment in primary genome-wide and secondary subpool SARS-CoV-2 subscreens. Pearson correlation is reported.

(B) Correlation matrix depicting the Pearson correlation between the guide-level log-fold change values relative to the plasmid DNA for the 13 subpool screens with the indicated viruses. All viruses were screened in duplicate (#1 and #2), except IAV-WSN. VSV was also screened, but no cells survived infection.

(C) Principal-component analysis (PCA) plot of all viruses reveals clustering and overlap of gene hits among SARS-CoV-2, rcVSV-SARS-CoV-2-S, HKU5-SARS-CoV-1-S, and MERS-CoV WT and the T1015N cluster. IAV/WSN/1933 (IAV-WSN) and encephalomyocarditis virus (EMCV) are outliers among the CoV screens.

(D–H) Comparison of gene enrichment in SARS-CoV-2 relative to rcVSV-SARS-CoV-2-S (D), HKU5-SARS-CoV-1-S (E), MERS-CoV (F), IAV-WSN (G), and EMCV (H). Pearson correlation is reported.

(I) We generated a CRISPR subpool targeting 32 genes (inclusive of control genes) in the human lung cancer cell line Calu-3. Gene enrichment from the primary SARS-CoV-2 screen correlates with results from Calu-3 cells. Pearson correlation is reported.

Comparison of Secondary CRISPR Subpool Screens, Related to Figure 3

(A) Heatmap depicting genes with a z-score > 10 in any of the secondary subpool screens. (B) Heatmap showing genes involved in the SWI/SNF chromatin remodeling complex. (C) Heatmap showing genes in the “Runx3 regulates CDKN1A transcription” pathway. (D) Heatmap showing genes in the HUCA histone H3.3 chaperone complex.

To begin to understand the generalizability of these hits to human cells, we screened a small CRISPR subpool targeting 32 genes in Calu-3 cells, a lung adenocarcinoma line that natively expresses ACE2 (Table S4). This screen revealed significant overlap in pro-viral (including ACE2, DYRK1A, SMARCA4, KDM6A, JMJD6, SMARCE1, and SIAH1) and anti-viral genes (including HIRA, PIAS2, and SMARCA5) between Vero-E6 and Calu-3 cells (Figure 3I), suggesting the conserved role of these genetic hits across species. Notably, TMPRSS2, but not CTSL, was enriched in the Calu-3 subpool screen, consistent with TMPRSS2 expression in Calu-3 cells but not Vero-E6 cells (Böttcher-Friebertshäuser et al., 2011).

We selected 25 genes for further validation of the SARS-CoV-2 screen in an arrayed rather than pooled format, consisting of 18 resistance and 7 sensitization genes (Figure 4
A). We transduced Vero-Cas9-v2 cells with one of 42 individual sgRNAs (1–3 sgRNAs per gene). We then challenged each of the 42 cell lines with SARS-CoV-2 and assessed cell viability. Cells receiving sgRNAs targeting pro-viral genes exhibited greater viability than those with non-targeting control sgRNAs. We validated the knockout efficiency of several genes (including ACE2, SMARCA4, KDM6A, and SMAD3) by western blot, and the protein abundance correlated with the degree of protection (Figures 4B–4D). Cells receiving sgRNAs targeting anti-viral genes in the primary screen exhibited increased susceptibility to cell death relative to controls, confirming the efficiency and reproducibility of the screen (Figures 4B and 4D). The robust concordance between the primary screens and the several subsequent validation approaches indicates that these screens will be a useful resource for further investigation of CoV host-pathogen interactions.Figure 4Arrayed Validation of 18 Resistance and 7 Sensitization Hit Genes(A) Performance in the pooled screen of sgRNAs targeting the 25 genes selected for further validation. The mean residual across the five Cas9-v2 conditions is plotted for the full library (top) and for the 3–4 sgRNAs targeting each gene. Genes that scored as resistance hits are shown in red; genes that scored as sensitization hits are shown in blue. The dashed line indicates a residual of 0.(B) 42 unique sgRNAs targeting 25 genes were introduced into Vero-E6-Cas9-v2 cells. SARS-CoV-2 was added at MOI 0.2, and cell viability was measured at 3 dpi.(C) Western blot for ACE2, SMARCA4, KDM6A, and SMAD3 expression in control and the respective gene-disrupted Vero-E6 cells.(D) Z scores from the genome-wide CRISPR screen correlate with cell viability of individually disrupted genes. Genes with multiple sgRNAs from (B) are averaged to generate one point per geneData were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM. ns, not statistically significant; ∗p < 0.05; ∗∗p < 0.01; ∗∗∗p < 0.001.

Arrayed Validation of 18 Resistance and 7 Sensitization Hit Genes

(A) Performance in the pooled screen of sgRNAs targeting the 25 genes selected for further validation. The mean residual across the five Cas9-v2 conditions is plotted for the full library (top) and for the 3–4 sgRNAs targeting each gene. Genes that scored as resistance hits are shown in red; genes that scored as sensitization hits are shown in blue. The dashed line indicates a residual of 0.

(B) 42 unique sgRNAs targeting 25 genes were introduced into Vero-E6-Cas9-v2 cells. SARS-CoV-2 was added at MOI 0.2, and cell viability was measured at 3 dpi.

(C) Western blot for ACE2, SMARCA4, KDM6A, and SMAD3 expression in control and the respective gene-disrupted Vero-E6 cells.

(D) Z scores from the genome-wide CRISPR screen correlate with cell viability of individually disrupted genes. Genes with multiple sgRNAs from (B) are averaged to generate one point per gene

Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM. ns, not statistically significant; ∗p < 0.05; ∗∗p < 0.01; ∗∗∗p < 0.001.

The CRISPR Screen Reveals Potential Host-Directed Therapeutic TargetsNext, we asked whether the genes and pathways revealed by the screen could be targeted with small molecules. We selected antagonists described previously to inhibit these gene products and investigated their effects on SARS-CoV-2-induced cell death. In addition, we utilized a replication-competent infectious clone of SARS-CoV-2 expressing the fluorescent reporter mNeonGreen (icSARS-CoV-2-mNG) to quantify the influence of these molecules on viral replication (Xie et al., 2020). We observed dose-dependent inhibition of SARS-CoV-2-induced cell death and virus replication with calpain inhibitor III, whose targets include Cathepsin L (Figures 5A, 5D, and 5E; Simmons et al., 2005; Hoffmann et al., 2019). Given the pro-viral role of SMARCA4 and SMAD3, we tested whether existing small-molecule antagonists of these pathways have antiviral activity against SARS-CoV-2. Specifically, treatment with PFI-3, which targets the bromodomains of the SWI/SNF proteins SMARCA4 and SMARCA2 (Fedorov et al., 2015; Xie et al., 2020), conferred protection from virus-induced cell death (Figure 5B) and reduced the frequency of viral infection, as measured by expression of mNeonGreen (Figures 5D and 5E). We also assessed the small molecule SIS3, which targets the pro-viral gene SMAD3 identified in the screen (Jinnin et al., 2006). SIS3 exhibited dose-dependent protection from virus-induced cell death and also inhibited SARS-CoV-2 fluorescent reporter expression (Figures 5C–5E). We next performed SARS-CoV-2 growth curves to investigate the effects of PFI-3 and SIS3 in Vero-E6 cells, Calu-3 cells, and the human liver cell line Huh7.5. We observed an ∼1-log reduction in PFI-3-treated cells and ∼2-log reduction in SIS3-treated cells (Figures 5F–5H). This provides pharmacological validation in Vero-E6 and human cells in addition to genetic evidence that these pathways are critical for SARS-CoV-2 infection.Figure 5Small Molecules Protect Cells from SARS-CoV-2-Induced Cell Death(A–C) Vero-E6 cells were pretreated with the indicated concentrations of the Cathepsin L inhibitor Calpain inhibitor III (A), the SMARCA4 inhibitor PFI-3 (B), or the SMAD3 inhibitor SIS3 (C) for 48 h and then infected with SARS-CoV-2 at a MOI of 0.2. Cell viability was measured at 3 dpi and compared with mock-infected controls. Red, infected; blue, mock-infected.(D and E) Vero-E6 cells were pretreated with 10 μM Calpain inhibitor III, PFI-3, or SIS3 for 48 h and then infected with icSARS-CoV-2 mNG at a MOI of 1. Infected cell frequencies were measured by mNeonGreen expression at 2 dpi. Scale bars, 300 μm.(F–H) Vero-E6 (F), Huh7.5 (G), and Calu-3 (H) cells were pretreated with 10 μM SIS3 and 40 μM PFI-3 for 48 h and then infected with SARS-CoV-2 at a MOI of 0.1. Virus production, as measured by plaque-forming units (PFU) per milliliter, was determined by plaque assay. LOD, limit of detection.Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM. ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.001.

The CRISPR Screen Reveals Potential Host-Directed Therapeutic Targets

Next, we asked whether the genes and pathways revealed by the screen could be targeted with small molecules. We selected antagonists described previously to inhibit these gene products and investigated their effects on SARS-CoV-2-induced cell death. In addition, we utilized a replication-competent infectious clone of SARS-CoV-2 expressing the fluorescent reporter mNeonGreen (icSARS-CoV-2-mNG) to quantify the influence of these molecules on viral replication (Xie et al., 2020). We observed dose-dependent inhibition of SARS-CoV-2-induced cell death and virus replication with calpain inhibitor III, whose targets include Cathepsin L (Figures 5A, 5D, and 5E; Simmons et al., 2005; Hoffmann et al., 2019). Given the pro-viral role of SMARCA4 and SMAD3, we tested whether existing small-molecule antagonists of these pathways have antiviral activity against SARS-CoV-2. Specifically, treatment with PFI-3, which targets the bromodomains of the SWI/SNF proteins SMARCA4 and SMARCA2 (Fedorov et al., 2015; Xie et al., 2020), conferred protection from virus-induced cell death (Figure 5B) and reduced the frequency of viral infection, as measured by expression of mNeonGreen (Figures 5D and 5E). We also assessed the small molecule SIS3, which targets the pro-viral gene SMAD3 identified in the screen (Jinnin et al., 2006). SIS3 exhibited dose-dependent protection from virus-induced cell death and also inhibited SARS-CoV-2 fluorescent reporter expression (Figures 5C–5E). We next performed SARS-CoV-2 growth curves to investigate the effects of PFI-3 and SIS3 in Vero-E6 cells, Calu-3 cells, and the human liver cell line Huh7.5. We observed an ∼1-log reduction in PFI-3-treated cells and ∼2-log reduction in SIS3-treated cells (Figures 5F–5H). This provides pharmacological validation in Vero-E6 and human cells in addition to genetic evidence that these pathways are critical for SARS-CoV-2 infection.Figure 5Small Molecules Protect Cells from SARS-CoV-2-Induced Cell Death(A–C) Vero-E6 cells were pretreated with the indicated concentrations of the Cathepsin L inhibitor Calpain inhibitor III (A), the SMARCA4 inhibitor PFI-3 (B), or the SMAD3 inhibitor SIS3 (C) for 48 h and then infected with SARS-CoV-2 at a MOI of 0.2. Cell viability was measured at 3 dpi and compared with mock-infected controls. Red, infected; blue, mock-infected.(D and E) Vero-E6 cells were pretreated with 10 μM Calpain inhibitor III, PFI-3, or SIS3 for 48 h and then infected with icSARS-CoV-2 mNG at a MOI of 1. Infected cell frequencies were measured by mNeonGreen expression at 2 dpi. Scale bars, 300 μm.(F–H) Vero-E6 (F), Huh7.5 (G), and Calu-3 (H) cells were pretreated with 10 μM SIS3 and 40 μM PFI-3 for 48 h and then infected with SARS-CoV-2 at a MOI of 0.1. Virus production, as measured by plaque-forming units (PFU) per milliliter, was determined by plaque assay. LOD, limit of detection.Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM. ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.001.

Small Molecules Protect Cells from SARS-CoV-2-Induced Cell Death

(A–C) Vero-E6 cells were pretreated with the indicated concentrations of the Cathepsin L inhibitor Calpain inhibitor III (A), the SMARCA4 inhibitor PFI-3 (B), or the SMAD3 inhibitor SIS3 (C) for 48 h and then infected with SARS-CoV-2 at a MOI of 0.2. Cell viability was measured at 3 dpi and compared with mock-infected controls. Red, infected; blue, mock-infected.

(D and E) Vero-E6 cells were pretreated with 10 μM Calpain inhibitor III, PFI-3, or SIS3 for 48 h and then infected with icSARS-CoV-2 mNG at a MOI of 1. Infected cell frequencies were measured by mNeonGreen expression at 2 dpi. Scale bars, 300 μm.

(F–H) Vero-E6 (F), Huh7.5 (G), and Calu-3 (H) cells were pretreated with 10 μM SIS3 and 40 μM PFI-3 for 48 h and then infected with SARS-CoV-2 at a MOI of 0.1. Virus production, as measured by plaque-forming units (PFU) per milliliter, was determined by plaque assay. LOD, limit of detection.

Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM. ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.001.

HMGB1 Regulates ACE2 Expression and Is Essential for Viral Entry of SARS-CoV-1, SARS-CoV-2, and NL63The SARS-CoV-2 screen revealed a putative pro-viral role of the gene LOC103214541, which is annotated as “HMGB1-like” in the C. sabaeus genome (mean Z score = 3.6, descending rank = 10; Figures 1C and 6A). HMGB1 is a nuclear protein that binds DNA but translocates to the cytoplasm under conditions of stress and can be secreted extracellularly, where it functions as an alarmin (Andersson et al., 2018). HMGB1-like was also identified as pro-viral in the rcVSV-SARS-CoV-2-S and HKU5-SARS-CoV-1-S but not the MERS-CoV screens (Figures 1C, 1G, and 3C–3E). To validate the role of HMGB1 in SARS-CoV-2 infection, we introduced two independent sgRNAs targeting HMGB1 into each of three independent cell lines: Vero-E6, Huh7.5, and Calu-3. We observed depletion of HMGB1 protein as measured by western blot (Figure 6B). HMGB1 disruption protected cells from SARS-CoV-2-induced cell death, and the degree of protection correlated with HMGB1 protein abundance (Figure 6C). We then performed SARS-CoV-2 growth curves on control and HMGB1-disrupted cells and observed an ∼2-log reduction in SARS-CoV-2 replication 24 and 48 h post-infection (Figure 6D). Next we investigated whether HMGB1 acts cell intrinsically or as an alarmin to regulate SARS-CoV-2 infection. SARS-CoV-2 infection increased HMGB1 protein levels in the nucleus and cytoplasm (Figure S5
A) and culture medium (Figures S5B and S5C). Recombinant HMGB1 protein had no effect on SARS-CoV-2 infection when added extracellularly to WT or HMGB1 KO cells, as measured by cell viability, icSARS-CoV-2-mNG infection, and SARS-CoV-2 pseudovirus infection (Figures S5D–S5F), demonstrating that HMGB1 acts cell intrinsically rather than as an alarmin or chemokine to regulate SARS-CoV-2 infection.Figure 6HMGB1 Is a Novel Regulator of ACE2(A) Performance of individual guide RNAs targeting LOC103214541 (HMGB1-like). The mean residual across the five Cas9-v2 conditions is plotted for the full library (top) and for the 3 guide RNAs targeting that gene.(B) Western blot for HMGB1 expression in control and HMGB1-disrupted Vero-E6, Huh7.5, and Calu-3 cells.(C) Control and HMGB1-disrupted Vero-E6, Huh7.5, and Calu-3 cells were infected with SARS-CoV-2 at a MOI of 0.2. Cell viability relative to an uninfected control was measured 3 dpi (Vero-E6 and Calu-3) or 4 dpi (Huh7.5) with CellTiter Glo.(D) Vero-E6 cells were infected with SARS-CoV-2 at a MOI of 0.1. Virus production was measured by plaque assay.(E) Correlation between CRISPR screen Z score and gene expression in control and HMGB1-disrupted Vero-E6 cells reveals downregulation of ACE2 in HMGB1-disrupted cells.(F) qPCR and western blot were performed in HMGB1 knockout and complemented Vero-E6 cells.(G) Genome tracks of RNA sequencing (RNA-seq), ChIP-seq for H3K27ac, and ATAC-seq at the ACE2 locus in control and HMGB1-disrupted Vero-E6 cells. The p values for ChIP-seq and ATAC-seq are for the genomic region indicated by the black bar below the tracks.(H) HMGB1 knockout and complemented Vero-E6 cells were infected with VSV pseudoparticles (VSVpp): VSVpp-SARS-CoV-1-S, VSVpp-SARS-CoV-2-S, VSVpp-NL63-S, VSVpp-MERS-S, and VSVpp-VSV-G. Luciferase relative to a VSVpp-VSV-G control was measured 1 dpi.Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM. ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.001.Figure S5HMGB1 Acts Cell-Intrinsically to Regulate Susceptibility to SARS-CoV-2 Infection, Related to Figure 6(A) Vero-E6 cells were mock-treated or infected with SARS-CoV-2 at a MOI of 1 for 24 hours before cell fractionation was performed. (B-C) Infection resulted in release of HMGB1 protein in the supernatant which was quantified by ELISA from Vero-E6 (B) and Huh7.5 (C) cells infected with SARS-CoV-2 for the indicated times. (D) Vero-E6 cells were pre-treated with the indicated concentration of recombinant HMGB1 (rHMGB1) for 24 hours and then infected with SARS-CoV-2 at a MOI of 0.2. Cell viability was measured at 3 dpi and compared to mock infected controls. (E) Vero-E6 cells were pre-treated with rHMGB1 for 24 hours and then infected with icSARS-CoV-2 mNG at a MOI of 1. Infected cell frequencies were measured by mNeonGreen expression at 1 dpi. (F) Vero-E6 cells were pre-treated with the indicated concentration of rHMGB1 for 24 hours and then infected with VSVpp-SARS-CoV-2-S and VSVpp-VSVG pseudovirus. Luciferase relative to VSVG control was measured at 1 dpi. Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM ns, not statistically significant; ∗p < 0.05; ∗∗p < 0.01; ∗∗∗p < 0.001.Because HMGB1 is a DNA binding protein that regulates chromatin, we hypothesized that HMGB1 controls a pro-viral gene expression program. When we compared differentially expressed genes between control and HMGB1 disrupted cells with the gene-level Z scores from the genome-wide CRISPR screen, we found that, among pro-viral genes, only HMGB1, ACE2, and CTSL gene expression was significantly downregulated in HMGB1-disrupted cells (Figures 6E and S6
A). Interestingly, few other differentially expressed genes were enriched in the positive or negative direction in the CRISPR screen. We confirmed that ACE2 transcripts were reduced significantly in HMGB1 knockout cells compared with WT Vero-E6 cells by qRT-PCR, as were ACE2 protein levels by western blot (Figure 6F). Gene Ontology enrichment analysis indicated that 16 gene sets were significantly enriched in differentially expression genes; however, these gene sets were not enriched in the SARS-CoV-2 CRISPR screen (Figure S6B).Figure S6Effects of HMGB1 Loss on Chromatin States across the Vero-E6 Genome, Related to Figure 6(A) Volcano plot for RNA sequencing of control and HMGB1 disrupted cells. The x axis shows log2 fold-change and the y axis shows −log10 of the adjusted P value (adj. P) as calculated by DESeq2. (B) Top gene sets, which significantly enriched in the upregulated, downregulated or both of differentially expression genes (fold change > 1.5 and p < 0.05) from GO. (C-D)Volcano plots for ATAC-seq (C) and H3K27ac ChIP-seq (D) of control and HMGB1 disrupted cells. Fold change and adjusted p value for each called peak was calculated by DESeq2. (E) Correlation between changes in overlapping ATAC-seq and H3K27ac ChIP-seq peaks upon HMGB1 disruption. Dashed lines represent p = 0.01. Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM ns, not statistically significant; ∗p < 0.05; ∗∗p < 0.01; ∗∗∗p < 0.001.We next analyzed the effects of HMGB1 disruption on chromatin states across the genome by chromatin immunoprecipitation sequencing (ChIP-seq) and assay for transposase-accessible chromatin sequencing (ATAC-seq; Figures S6C and S6D). Upon HMGB1 disruption, changes in chromatin accessibility by ATAC-seq were positively correlated with changes in acetylation of lysine 27 on histone H3 protein subunit (H3K27ac) ChIP-seq, a marker of active enhancers (Figure S6E), as expected. ChIP-seq revealed a significant reduction in H3K27ac level (p = 0.01) at a peak immediately downstream of the ACE2 transcription start site in HMGB1-disrupted cells compared with control cells (Figures 6G and S6D). In addition, we observed a trend toward reduced chromatin accessibility at the overlapping ATAC-seq peak at the ACE2 locus (p = 0.057) in HMGB1-disrupted cells compared with control cells (Figures 6G and S6C). HMGB1 is necessary for ACE2 expression as well as viral entry of SARS-CoV-1, SARS-CoV-2, and NL63, but not MERS-CoV, which mirrors receptor utilization (Figures 6F and 6H). These findings demonstrate that HMGB1 is a novel regulator of ACE2 expression that affects susceptibility to SARS-CoV-2.

HMGB1 Regulates ACE2 Expression and Is Essential for Viral Entry of SARS-CoV-1, SARS-CoV-2, and NL63

The SARS-CoV-2 screen revealed a putative pro-viral role of the gene LOC103214541, which is annotated as “HMGB1-like” in the C. sabaeus genome (mean Z score = 3.6, descending rank = 10; Figures 1C and 6A). HMGB1 is a nuclear protein that binds DNA but translocates to the cytoplasm under conditions of stress and can be secreted extracellularly, where it functions as an alarmin (Andersson et al., 2018). HMGB1-like was also identified as pro-viral in the rcVSV-SARS-CoV-2-S and HKU5-SARS-CoV-1-S but not the MERS-CoV screens (Figures 1C, 1G, and 3C–3E). To validate the role of HMGB1 in SARS-CoV-2 infection, we introduced two independent sgRNAs targeting HMGB1 into each of three independent cell lines: Vero-E6, Huh7.5, and Calu-3. We observed depletion of HMGB1 protein as measured by western blot (Figure 6B). HMGB1 disruption protected cells from SARS-CoV-2-induced cell death, and the degree of protection correlated with HMGB1 protein abundance (Figure 6C). We then performed SARS-CoV-2 growth curves on control and HMGB1-disrupted cells and observed an ∼2-log reduction in SARS-CoV-2 replication 24 and 48 h post-infection (Figure 6D). Next we investigated whether HMGB1 acts cell intrinsically or as an alarmin to regulate SARS-CoV-2 infection. SARS-CoV-2 infection increased HMGB1 protein levels in the nucleus and cytoplasm (Figure S5
A) and culture medium (Figures S5B and S5C). Recombinant HMGB1 protein had no effect on SARS-CoV-2 infection when added extracellularly to WT or HMGB1 KO cells, as measured by cell viability, icSARS-CoV-2-mNG infection, and SARS-CoV-2 pseudovirus infection (Figures S5D–S5F), demonstrating that HMGB1 acts cell intrinsically rather than as an alarmin or chemokine to regulate SARS-CoV-2 infection.Figure 6HMGB1 Is a Novel Regulator of ACE2(A) Performance of individual guide RNAs targeting LOC103214541 (HMGB1-like). The mean residual across the five Cas9-v2 conditions is plotted for the full library (top) and for the 3 guide RNAs targeting that gene.(B) Western blot for HMGB1 expression in control and HMGB1-disrupted Vero-E6, Huh7.5, and Calu-3 cells.(C) Control and HMGB1-disrupted Vero-E6, Huh7.5, and Calu-3 cells were infected with SARS-CoV-2 at a MOI of 0.2. Cell viability relative to an uninfected control was measured 3 dpi (Vero-E6 and Calu-3) or 4 dpi (Huh7.5) with CellTiter Glo.(D) Vero-E6 cells were infected with SARS-CoV-2 at a MOI of 0.1. Virus production was measured by plaque assay.(E) Correlation between CRISPR screen Z score and gene expression in control and HMGB1-disrupted Vero-E6 cells reveals downregulation of ACE2 in HMGB1-disrupted cells.(F) qPCR and western blot were performed in HMGB1 knockout and complemented Vero-E6 cells.(G) Genome tracks of RNA sequencing (RNA-seq), ChIP-seq for H3K27ac, and ATAC-seq at the ACE2 locus in control and HMGB1-disrupted Vero-E6 cells. The p values for ChIP-seq and ATAC-seq are for the genomic region indicated by the black bar below the tracks.(H) HMGB1 knockout and complemented Vero-E6 cells were infected with VSV pseudoparticles (VSVpp): VSVpp-SARS-CoV-1-S, VSVpp-SARS-CoV-2-S, VSVpp-NL63-S, VSVpp-MERS-S, and VSVpp-VSV-G. Luciferase relative to a VSVpp-VSV-G control was measured 1 dpi.Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM. ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.001.Figure S5HMGB1 Acts Cell-Intrinsically to Regulate Susceptibility to SARS-CoV-2 Infection, Related to Figure 6(A) Vero-E6 cells were mock-treated or infected with SARS-CoV-2 at a MOI of 1 for 24 hours before cell fractionation was performed. (B-C) Infection resulted in release of HMGB1 protein in the supernatant which was quantified by ELISA from Vero-E6 (B) and Huh7.5 (C) cells infected with SARS-CoV-2 for the indicated times. (D) Vero-E6 cells were pre-treated with the indicated concentration of recombinant HMGB1 (rHMGB1) for 24 hours and then infected with SARS-CoV-2 at a MOI of 0.2. Cell viability was measured at 3 dpi and compared to mock infected controls. (E) Vero-E6 cells were pre-treated with rHMGB1 for 24 hours and then infected with icSARS-CoV-2 mNG at a MOI of 1. Infected cell frequencies were measured by mNeonGreen expression at 1 dpi. (F) Vero-E6 cells were pre-treated with the indicated concentration of rHMGB1 for 24 hours and then infected with VSVpp-SARS-CoV-2-S and VSVpp-VSVG pseudovirus. Luciferase relative to VSVG control was measured at 1 dpi. Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM ns, not statistically significant; ∗p < 0.05; ∗∗p < 0.01; ∗∗∗p < 0.001.

HMGB1 Is a Novel Regulator of ACE2

(A) Performance of individual guide RNAs targeting LOC103214541 (HMGB1-like). The mean residual across the five Cas9-v2 conditions is plotted for the full library (top) and for the 3 guide RNAs targeting that gene.

(B) Western blot for HMGB1 expression in control and HMGB1-disrupted Vero-E6, Huh7.5, and Calu-3 cells.

(C) Control and HMGB1-disrupted Vero-E6, Huh7.5, and Calu-3 cells were infected with SARS-CoV-2 at a MOI of 0.2. Cell viability relative to an uninfected control was measured 3 dpi (Vero-E6 and Calu-3) or 4 dpi (Huh7.5) with CellTiter Glo.

(D) Vero-E6 cells were infected with SARS-CoV-2 at a MOI of 0.1. Virus production was measured by plaque assay.

(E) Correlation between CRISPR screen Z score and gene expression in control and HMGB1-disrupted Vero-E6 cells reveals downregulation of ACE2 in HMGB1-disrupted cells.

(F) qPCR and western blot were performed in HMGB1 knockout and complemented Vero-E6 cells.

(G) Genome tracks of RNA sequencing (RNA-seq), ChIP-seq for H3K27ac, and ATAC-seq at the ACE2 locus in control and HMGB1-disrupted Vero-E6 cells. The p values for ChIP-seq and ATAC-seq are for the genomic region indicated by the black bar below the tracks.

(H) HMGB1 knockout and complemented Vero-E6 cells were infected with VSV pseudoparticles (VSVpp): VSVpp-SARS-CoV-1-S, VSVpp-SARS-CoV-2-S, VSVpp-NL63-S, VSVpp-MERS-S, and VSVpp-VSV-G. Luciferase relative to a VSVpp-VSV-G control was measured 1 dpi.

Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM. ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.001.

HMGB1 Acts Cell-Intrinsically to Regulate Susceptibility to SARS-CoV-2 Infection, Related to Figure 6

(A) Vero-E6 cells were mock-treated or infected with SARS-CoV-2 at a MOI of 1 for 24 hours before cell fractionation was performed. (B-C) Infection resulted in release of HMGB1 protein in the supernatant which was quantified by ELISA from Vero-E6 (B) and Huh7.5 (C) cells infected with SARS-CoV-2 for the indicated times. (D) Vero-E6 cells were pre-treated with the indicated concentration of recombinant HMGB1 (rHMGB1) for 24 hours and then infected with SARS-CoV-2 at a MOI of 0.2. Cell viability was measured at 3 dpi and compared to mock infected controls. (E) Vero-E6 cells were pre-treated with rHMGB1 for 24 hours and then infected with icSARS-CoV-2 mNG at a MOI of 1. Infected cell frequencies were measured by mNeonGreen expression at 1 dpi. (F) Vero-E6 cells were pre-treated with the indicated concentration of rHMGB1 for 24 hours and then infected with VSVpp-SARS-CoV-2-S and VSVpp-VSVG pseudovirus. Luciferase relative to VSVG control was measured at 1 dpi. Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM ns, not statistically significant; ∗p < 0.05; ∗∗p < 0.01; ∗∗∗p < 0.001.

Because HMGB1 is a DNA binding protein that regulates chromatin, we hypothesized that HMGB1 controls a pro-viral gene expression program. When we compared differentially expressed genes between control and HMGB1 disrupted cells with the gene-level Z scores from the genome-wide CRISPR screen, we found that, among pro-viral genes, only HMGB1, ACE2, and CTSL gene expression was significantly downregulated in HMGB1-disrupted cells (Figures 6E and S6
A). Interestingly, few other differentially expressed genes were enriched in the positive or negative direction in the CRISPR screen. We confirmed that ACE2 transcripts were reduced significantly in HMGB1 knockout cells compared with WT Vero-E6 cells by qRT-PCR, as were ACE2 protein levels by western blot (Figure 6F). Gene Ontology enrichment analysis indicated that 16 gene sets were significantly enriched in differentially expression genes; however, these gene sets were not enriched in the SARS-CoV-2 CRISPR screen (Figure S6B).Figure S6Effects of HMGB1 Loss on Chromatin States across the Vero-E6 Genome, Related to Figure 6(A) Volcano plot for RNA sequencing of control and HMGB1 disrupted cells. The x axis shows log2 fold-change and the y axis shows −log10 of the adjusted P value (adj. P) as calculated by DESeq2. (B) Top gene sets, which significantly enriched in the upregulated, downregulated or both of differentially expression genes (fold change > 1.5 and p < 0.05) from GO. (C-D)Volcano plots for ATAC-seq (C) and H3K27ac ChIP-seq (D) of control and HMGB1 disrupted cells. Fold change and adjusted p value for each called peak was calculated by DESeq2. (E) Correlation between changes in overlapping ATAC-seq and H3K27ac ChIP-seq peaks upon HMGB1 disruption. Dashed lines represent p = 0.01. Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM ns, not statistically significant; ∗p < 0.05; ∗∗p < 0.01; ∗∗∗p < 0.001.

Effects of HMGB1 Loss on Chromatin States across the Vero-E6 Genome, Related to Figure 6

(A) Volcano plot for RNA sequencing of control and HMGB1 disrupted cells. The x axis shows log2 fold-change and the y axis shows −log10 of the adjusted P value (adj. P) as calculated by DESeq2. (B) Top gene sets, which significantly enriched in the upregulated, downregulated or both of differentially expression genes (fold change > 1.5 and p < 0.05) from GO. (C-D)Volcano plots for ATAC-seq (C) and H3K27ac ChIP-seq (D) of control and HMGB1 disrupted cells. Fold change and adjusted p value for each called peak was calculated by DESeq2. (E) Correlation between changes in overlapping ATAC-seq and H3K27ac ChIP-seq peaks upon HMGB1 disruption. Dashed lines represent p = 0.01. Data were analyzed by one-way ANOVA with Tukey’s multiple comparison test. Shown are means ± SEM ns, not statistically significant; ∗p < 0.05; ∗∗p < 0.01; ∗∗∗p < 0.001.

We next analyzed the effects of HMGB1 disruption on chromatin states across the genome by chromatin immunoprecipitation sequencing (ChIP-seq) and assay for transposase-accessible chromatin sequencing (ATAC-seq; Figures S6C and S6D). Upon HMGB1 disruption, changes in chromatin accessibility by ATAC-seq were positively correlated with changes in acetylation of lysine 27 on histone H3 protein subunit (H3K27ac) ChIP-seq, a marker of active enhancers (Figure S6E), as expected. ChIP-seq revealed a significant reduction in H3K27ac level (p = 0.01) at a peak immediately downstream of the ACE2 transcription start site in HMGB1-disrupted cells compared with control cells (Figures 6G and S6D). In addition, we observed a trend toward reduced chromatin accessibility at the overlapping ATAC-seq peak at the ACE2 locus (p = 0.057) in HMGB1-disrupted cells compared with control cells (Figures 6G and S6C). HMGB1 is necessary for ACE2 expression as well as viral entry of SARS-CoV-1, SARS-CoV-2, and NL63, but not MERS-CoV, which mirrors receptor utilization (Figures 6F and 6H). These findings demonstrate that HMGB1 is a novel regulator of ACE2 expression that affects susceptibility to SARS-CoV-2.

DiscussionWe performed the first genome-wide screens for host genes that affect infection with the pandemic CoVs SARS-CoV-2 and MERS-CoV as well as the recombinant bat CoV HKU5-SARS-CoV-1-S. Identification of the viral receptors ACE2 and DPP4 and the protease CTSL demonstrates the technical quality of the screens, providing confidence in the additional genes that regulate SARS-CoV-2 infection (Hoffmann et al., 2019, 2020b; Ou et al., 2020; Shang et al., 2020). We discovered genes involved in diverse biological processes, including chromatin remodeling, histone modification, cellular signaling, and RNA regulation. We validated key genes in a pooled and arrayed format, including pro-viral and anti-viral genes, and identified small-molecule antagonists that confer protection against SARS-CoV-2-induced cell death and infection in Vero-E6 cells, human hepatocytes, and human lung cells.Our screen identified many genes with functional roles in chromatin regulation and histone modification, which highlights the potential importance of epigenetic regulation of pathogenic CoV infection. Epigenetic processes have been implicated previously in regulating antigen presentation and interferon-stimulated gene induction after MERS-CoV and SARS-CoV-1 infection (Menachery et al., 2014, 2018; Schäfer and Baric, 2017); however, given that Vero-E6 cells are type I interferon deficient, distinct mechanism(s) may be at play. Interestingly, the majority of pro-viral and anti-viral genes identified function at the level of viral entry, as determined by the high degree of concordance between SARS-CoV-2 and rcVSV-SARS-CoV-2-S screens. We identified ACE2-dependent and -independent mechanisms regulating CoV entry.Specifically, we identify a novel epigenetic role of HMGB1 in regulating ACE2 expression and, thus, susceptibility to SARS-CoV-1, SARS-CoV-2, and NL63. We demonstrate this in Vero-E6 cells and two human IFN-sufficient cell lines. HMGB1 is a pleiotropic protein that binds nucleosomes regulating chromatin in the nucleus, acts as a sentinel of non-self nucleic acids, transports genetic material, and functions as a secreted alarmin in response to virus infection (Menachery et al., 2014; Andersson et al., 2020; Simpson et al., 2020). Interestingly, anti-HMGB1 therapies can reduce respiratory syncytial virus replication and IAV-induced lung pathology in animal models (Manti et al., 2018; Hatayama et al., 2019), whereas in adenovirus infection, the viral protein VII binds HMGB1 and inhibits its proinflammatory functions (Avgousti et al., 2016). Notably, we find that HMGB1 regulates ACE2 expression in a cell-intrinsic manner and not via its function as a cytokine or alarmin, suggesting a distinct mechanism of HMGB1 in SARS-CoV-2 infection.Genes encoding members of the SWI/SNF chromatin remodeling complex were identified as pro-viral for SARS-CoV-2, MERS-CoV, and HKU5-SARS-CoV-1-S, suggesting that this complex is broadly important for pathogenic CoVs. These genes were also identified as pro-viral for rcVSV-SARS-CoV-2-S, suggesting a role of SWI/SNF complexes in promoting CoV entry. The SWI/SNF complex is comprised of a catalytic ATPase subunit, SMARCA2 or SMARCA4, and a larger, non-catalytic protein scaffold core that is bridged to the ATPase via ARID1A (Mashtalir et al., 2018). SWI/SNF complexes lack intrinsic DNA sequence specificity; thus, their targeting specificity is conferred by DNA-binding proteins that bind and recruit them to genomic target sites, where they then slide and eject nucleosomes regulating chromatin accessibility and gene expression. We speculate that the pro-viral role of SWI/SNF complexes may be opposed by the histone H3.3 (HUCA chaperone) complex, which has been shown to co-target with SWI/SNF complexes on chromatin (Pchelintsev et al., 2013).The pro-viral and anti-viral genes identified here have important implications for our understanding of COVID-19 pathogenesis, therapeutic agents, and vaccine design. First, SARS-CoV-2 can cause diverse phenotypes ranging from asymptomatic infection to severe respiratory failure and death (Wang et al., 2020; Zhu et al., 2020). The basis of this variation among people and between species is unclear. The genes and pathways identified here may explain this variation because disease susceptibility may positively correlate with expression of resistance genes and negatively correlate with sensitization genes on the cellular, tissue, and organismal level. For example, cigarette smoking increases ACE2 expression and exacerbates COVID-19 pathogenesis (Smith et al., 2020). The regulatory network underlying this is unknown, but it is intriguing to speculate that the chromatin- and histone-modifying genes identified here contribute to expression of a heterogeneous pro-viral gene expression program that potentially regulates ACE2 and other viral interaction genes. In addition, despite Vero-E6 cells being deficient in interferon, we identified the interferon-stimulated gene LY6E as an anti-viral entry factor for the SARS lineage and MERS-CoV. This suggests basal expression of LY6E in the absence of type I interferon and is consistent with a recent study identifying Ly6E as a pan-CoV anti-viral molecule (Pfaender et al., 2020) and reveals the utility and applicability of Vero-E6 cells as a model system to reveal host-pathogen interactions of pathogenic CoVs.The genetic screen revealed novel therapeutic targets for SARS-CoV-2 infection. As a proof of principle, we tested several small-molecule inhibitors and identified three molecules that inhibit SARS-CoV-2 replication and virus-induced cell death. Therapeutic targeting of these genes and pathways, including the SWI/SNF complex and SMAD3/SMAD4, may prove to be clinically useful. Additionally, we identified and validated anti-viral genes, including regulators of the histone variant H3.3 (CABIN1, HIRA, and ASF1A) (Rai et al., 2011; Ray-Gallet et al., 2018). Although these genes potentially provide protection from SARS-CoV-2, they may also prove to be fruitful in generating knockout cell lines with increased susceptibility to diverse human CoVs, which may facilitate CoV vaccine production (Gao et al., 2020).SARS-CoV-1, MERS-CoV, and SARS-CoV-2 reveal the pandemic potential and dangers of emerging CoVs, for which there are no US Food and Drug Administration (FDA)-approved therapeutics or vaccines (Guan et al., 2003; Drosten et al., 2014; Gordon et al., 2020). To our knowledge, this study represents the first genome-wide genetic screen performed with any human CoVs. Ultimately, our findings may be broadly applicable to other human and emerging CoVs, which may facilitate development of host-directed therapies against existing and future pandemic CoVs.Limitations of StudyThis study identified host genes that are essential for CoV infection in Vero-E6 cells, and we demonstrate that HMGB1 regulates ACE2 transcription. However, the mechanism underlying how the majority of these genes regulate infection remains to be determined. Although Vero cells are a model cell line for pathogenic CoV, they likely do not fully recapitulate all aspects of infection in primary cells, such as human airway epithelial cells; nor does this system fully recapitulate the complex cellular milieu in a human patient. We demonstrate that the Vero screen results predict results in a transformed human lung cell line, but the correlation is imperfect, suggesting a degree of cell-type-specific regulation of SARS-CoV-2 infection that requires future investigation. Finally, although the host genes required for SARS-CoV-2 infection here represent potential host-directed therapeutic targets for COVID-19, additional studies are required to further develop host-directed drugs against CoVs.

We performed the first genome-wide screens for host genes that affect infection with the pandemic CoVs SARS-CoV-2 and MERS-CoV as well as the recombinant bat CoV HKU5-SARS-CoV-1-S. Identification of the viral receptors ACE2 and DPP4 and the protease CTSL demonstrates the technical quality of the screens, providing confidence in the additional genes that regulate SARS-CoV-2 infection (Hoffmann et al., 2019, 2020b; Ou et al., 2020; Shang et al., 2020). We discovered genes involved in diverse biological processes, including chromatin remodeling, histone modification, cellular signaling, and RNA regulation. We validated key genes in a pooled and arrayed format, including pro-viral and anti-viral genes, and identified small-molecule antagonists that confer protection against SARS-CoV-2-induced cell death and infection in Vero-E6 cells, human hepatocytes, and human lung cells.

Our screen identified many genes with functional roles in chromatin regulation and histone modification, which highlights the potential importance of epigenetic regulation of pathogenic CoV infection. Epigenetic processes have been implicated previously in regulating antigen presentation and interferon-stimulated gene induction after MERS-CoV and SARS-CoV-1 infection (Menachery et al., 2014, 2018; Schäfer and Baric, 2017); however, given that Vero-E6 cells are type I interferon deficient, distinct mechanism(s) may be at play. Interestingly, the majority of pro-viral and anti-viral genes identified function at the level of viral entry, as determined by the high degree of concordance between SARS-CoV-2 and rcVSV-SARS-CoV-2-S screens. We identified ACE2-dependent and -independent mechanisms regulating CoV entry.

Specifically, we identify a novel epigenetic role of HMGB1 in regulating ACE2 expression and, thus, susceptibility to SARS-CoV-1, SARS-CoV-2, and NL63. We demonstrate this in Vero-E6 cells and two human IFN-sufficient cell lines. HMGB1 is a pleiotropic protein that binds nucleosomes regulating chromatin in the nucleus, acts as a sentinel of non-self nucleic acids, transports genetic material, and functions as a secreted alarmin in response to virus infection (Menachery et al., 2014; Andersson et al., 2020; Simpson et al., 2020). Interestingly, anti-HMGB1 therapies can reduce respiratory syncytial virus replication and IAV-induced lung pathology in animal models (Manti et al., 2018; Hatayama et al., 2019), whereas in adenovirus infection, the viral protein VII binds HMGB1 and inhibits its proinflammatory functions (Avgousti et al., 2016). Notably, we find that HMGB1 regulates ACE2 expression in a cell-intrinsic manner and not via its function as a cytokine or alarmin, suggesting a distinct mechanism of HMGB1 in SARS-CoV-2 infection.

Genes encoding members of the SWI/SNF chromatin remodeling complex were identified as pro-viral for SARS-CoV-2, MERS-CoV, and HKU5-SARS-CoV-1-S, suggesting that this complex is broadly important for pathogenic CoVs. These genes were also identified as pro-viral for rcVSV-SARS-CoV-2-S, suggesting a role of SWI/SNF complexes in promoting CoV entry. The SWI/SNF complex is comprised of a catalytic ATPase subunit, SMARCA2 or SMARCA4, and a larger, non-catalytic protein scaffold core that is bridged to the ATPase via ARID1A (Mashtalir et al., 2018). SWI/SNF complexes lack intrinsic DNA sequence specificity; thus, their targeting specificity is conferred by DNA-binding proteins that bind and recruit them to genomic target sites, where they then slide and eject nucleosomes regulating chromatin accessibility and gene expression. We speculate that the pro-viral role of SWI/SNF complexes may be opposed by the histone H3.3 (HUCA chaperone) complex, which has been shown to co-target with SWI/SNF complexes on chromatin (Pchelintsev et al., 2013).

The pro-viral and anti-viral genes identified here have important implications for our understanding of COVID-19 pathogenesis, therapeutic agents, and vaccine design. First, SARS-CoV-2 can cause diverse phenotypes ranging from asymptomatic infection to severe respiratory failure and death (Wang et al., 2020; Zhu et al., 2020). The basis of this variation among people and between species is unclear. The genes and pathways identified here may explain this variation because disease susceptibility may positively correlate with expression of resistance genes and negatively correlate with sensitization genes on the cellular, tissue, and organismal level. For example, cigarette smoking increases ACE2 expression and exacerbates COVID-19 pathogenesis (Smith et al., 2020). The regulatory network underlying this is unknown, but it is intriguing to speculate that the chromatin- and histone-modifying genes identified here contribute to expression of a heterogeneous pro-viral gene expression program that potentially regulates ACE2 and other viral interaction genes. In addition, despite Vero-E6 cells being deficient in interferon, we identified the interferon-stimulated gene LY6E as an anti-viral entry factor for the SARS lineage and MERS-CoV. This suggests basal expression of LY6E in the absence of type I interferon and is consistent with a recent study identifying Ly6E as a pan-CoV anti-viral molecule (Pfaender et al., 2020) and reveals the utility and applicability of Vero-E6 cells as a model system to reveal host-pathogen interactions of pathogenic CoVs.

The genetic screen revealed novel therapeutic targets for SARS-CoV-2 infection. As a proof of principle, we tested several small-molecule inhibitors and identified three molecules that inhibit SARS-CoV-2 replication and virus-induced cell death. Therapeutic targeting of these genes and pathways, including the SWI/SNF complex and SMAD3/SMAD4, may prove to be clinically useful. Additionally, we identified and validated anti-viral genes, including regulators of the histone variant H3.3 (CABIN1, HIRA, and ASF1A) (Rai et al., 2011; Ray-Gallet et al., 2018). Although these genes potentially provide protection from SARS-CoV-2, they may also prove to be fruitful in generating knockout cell lines with increased susceptibility to diverse human CoVs, which may facilitate CoV vaccine production (Gao et al., 2020).

SARS-CoV-1, MERS-CoV, and SARS-CoV-2 reveal the pandemic potential and dangers of emerging CoVs, for which there are no US Food and Drug Administration (FDA)-approved therapeutics or vaccines (Guan et al., 2003; Drosten et al., 2014; Gordon et al., 2020). To our knowledge, this study represents the first genome-wide genetic screen performed with any human CoVs. Ultimately, our findings may be broadly applicable to other human and emerging CoVs, which may facilitate development of host-directed therapies against existing and future pandemic CoVs.

Limitations of StudyThis study identified host genes that are essential for CoV infection in Vero-E6 cells, and we demonstrate that HMGB1 regulates ACE2 transcription. However, the mechanism underlying how the majority of these genes regulate infection remains to be determined. Although Vero cells are a model cell line for pathogenic CoV, they likely do not fully recapitulate all aspects of infection in primary cells, such as human airway epithelial cells; nor does this system fully recapitulate the complex cellular milieu in a human patient. We demonstrate that the Vero screen results predict results in a transformed human lung cell line, but the correlation is imperfect, suggesting a degree of cell-type-specific regulation of SARS-CoV-2 infection that requires future investigation. Finally, although the host genes required for SARS-CoV-2 infection here represent potential host-directed therapeutic targets for COVID-19, additional studies are required to further develop host-directed drugs against CoVs.

Limitations of Study

This study identified host genes that are essential for CoV infection in Vero-E6 cells, and we demonstrate that HMGB1 regulates ACE2 transcription. However, the mechanism underlying how the majority of these genes regulate infection remains to be determined. Although Vero cells are a model cell line for pathogenic CoV, they likely do not fully recapitulate all aspects of infection in primary cells, such as human airway epithelial cells; nor does this system fully recapitulate the complex cellular milieu in a human patient. We demonstrate that the Vero screen results predict results in a transformed human lung cell line, but the correlation is imperfect, suggesting a degree of cell-type-specific regulation of SARS-CoV-2 infection that requires future investigation. Finally, although the host genes required for SARS-CoV-2 infection here represent potential host-directed therapeutic targets for COVID-19, additional studies are required to further develop host-directed drugs against CoVs.

STAR★MethodsKey Resources Table
REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesAnti-ACE2 antibodyProSciCat#3217; RRID: AB_712925Anti-β-actin antibodyBioLegendCat#622102; RRID: AB_315946Anti-LMNB1 antibodyBioLegendCat#869802; RRID: AB_2820181Anti-GAPDH antibodyBioLegendCat#607902; RRID: AB_2734503Anti-HMGB1 antibodyAbcamCat#ab18256; RRID: AB_444360Anti-SMARCA4 antibodyAbcamCat#ab70558; RRID: AB_1209535Anti-H3K27ac antibodyAbcamCat#ab4729; RRID: AB_2118291Anti-SMAD3 antibodyCell Signaling TechnologyCat#9513; RRID: AB_2286450Anti-KDM6A antibodyCell Signaling TechnologyCat#33510; RRID: AB_2721244Goat anti-rabbit IgG/HRPJackson ImmunoResearchCat#111-035-003; RRID: AB_2337913Goat anti-mouse IgG/HRPJackson ImmunoResearchCat#115-035-003; RRID: AB_10015289Anti-VSV-G antibodyKerafastCat#EB0010; RRID: AB_2811223Bacterial and Virus StrainsCP0070 African Green Monkey (AGM) CRISPR KO libraryBroad InstituteCat#CP0070CP1564 CRISPR KO libraryBroad InstituteCat#CP1564CP1560 CRISPR KO libraryBroad InstituteCat#CP1560SARS-CoV-2, isolate US-WA1/2020BEI ResourcesCat#NR-52281rcVSV-SARS-CoV-2-SCase et al., 2020N/ASARS-CoV-1, HKU5BEI ResourcesCat#NR-48814MERS-CoV WTBEI ResourcesCat#NR-48813MERS-CoV T1015NBEI ResourcesCat#NR-48811EMCVBEI ResourcesCat#NR-19846Influenza A virus/WSN/1933 (IAV)A. Boon (Wustl)N/AicSARS-CoV-2 mNGXie et al., 2020N/AChemicals, Peptides, and Recombinant ProteinsDMEM mediaThermo Fisher ScientificCat#C11995500BTEMEM mediaATCCCat#MT10009CVFetal bovine serum (FBS)VWRCat# VW97068-085Puromycin DihydrochlorideGIBCOCat#A1113803Blasticidin S HClGIBCOCat#A1113903Lipofectamine 2000InvitrogenCat#11668019FormaldehydeJ.B BakerCat#14650-250Calpain inhibitor IIICaymanCat#14283PFI-3CaymanCat#15267SIS3CaymanCat#15495Recombinant human HMGB1 proteinBioLegendCat#557804Critical Commercial AssaysCellTiter-Glo® Luminescent Cell Viability AssayPromegaCat#G7570Direct-zol RNA Miniprep Plus kitZymo ResearchCat#R2072Renilla Luciferase Assay SystemPromegaCat#E2820Nuclear/Cytosol Fractionation KitBioVisionCat#K266-25Human HMGB1/HMG-1 ELISA KitNovus BiologicalsCat#NBP2-62766Deposited DataATAC-seq dataThis paperGSE154784RNA-seq dataThis paperGSE154784ChIP-seq dataThis paperGSE154761CRISPR dataThis paperhttps://doi.org/10.17632/pbcrc9c7zs.1sgRNA sequencesThis paperhttps://doi.org/10.17632/pbcrc9c7zs.1Experimental Models: Cell LinesHEK293T cellsATCCCat#CRL-3216Vero-E6 cellsATCCCat#CRL-1586Huh7.5 cellsATCCCat#CVCL-7927Calu-3 cellsATCCCat#HTB-55OligonucleotidessgRNAs target sequences, see Table S6This paperN/AACE2 qPCR primer forward: GGGATCAGAGATCGGAAGAAGA; reverse: AAGGAGGTCTGAACATCATCAGTGThis paperN/AActin qPCR primer forward GAGCACAGAGCCTCGCCTTT; reverse: ATCATCATCCATGGTGAGCTGGThis paperN/ARecombinant DNALenti-Cas9-Blast (v1)Sanjana et al., 2014Addgene Cat#52962pLX_311-Cas9 (v2)Doench et al., 2014Addgene Cat #96924pXPR_047Doench et al., 2014Addgene Cat #107645LentiCRISPRv2Sanjana et al., 2014Addgene Cat #52961Lenti-Guide-PuroSanjana et al., 2014Addgene Cat #52963pXPR_05Sanson et al., 2018Addgene Cat#96925psPAX2Didier TronoAddgene Cat#12260VSVGStewart et al., 2003Addgene Cat#8454pLenti6/V5-DEST-HMGB1Scott et al., 2011Addgene Cat#31208Software and AlgorithmsBowtie2 v2.2.9Langmead and Salzberg, 2012N/ACutadaptMartin, 2011N/ADESeq2 v1.32Love et al., 2014N/Adeeptools v3.1.3Ramírez et al., 2016N/AFlowjo 10.6.2FLOWJOhttps://www.flowjo.comGraphpad Prism 8Graphpad softwarehttps://www.graphpad.com/scientific-software/prism/MACS2Zhang et al., 2008N/APoolQ version 3.2.9Broad Institutehttps://portals.broadinstitute.org/gpp/public/software/poolq/Picard Tools v2.9.0Broad Institutehttp://broadinstitute.github.io/picard/STAR aligner v2.7.3aDobin et al., 2013N/ASAMTools v1.9Li et al., 2009N/ATrimmomatic v0.39Bolger et al., 2014N/ACRISPR screen analysisThis paperhttps://github.com/PeterDeWeirdt/coronavirus_screen_analysis
Resource AvailabilityLead ContactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Craig Wilen (craig.wilen@yale.edu).Materials AvailabilityAll requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact author. Materials will be made publicly available either through publicly available repositories or via the authors upon execution of a Material Transfer Agreement.Data and Code AvailabilityThe published article includes all data generated or analyzed during the study except as described below. The RNA-seq (accession GSE154784), ATAC-Seq (accession GSE154784) and ChIP-Seq data (accession GSE154761) are available at NCBI GEO. The analysis code generated during the study are available at: https://github.com/PeterDeWeirdt/coronavirus_screen_analysis.Experimental Model and Subject DetailsCellsHEK293T, Vero-E6 and Huh7.5 cells were cultured in Dulbecco’s Modified Eagle Medium (DMEM) with 10% heat-inactivated fetal bovine serum (FBS), and 1% Penicillin/Streptomycin unless otherwise indicated. Calu-3 cells were cultured in Eagle’s minimal essential medium (EMEM) with 10% FBS and 1% Penicillin/Streptomycin. For Vero-E6 and Huh7.5 cells, 5 μg/ml of puromycin (GIBCO) and 5 μg/ml blasticidin (GIBCO), were added as appropriate. For Calu-3 cells, 1 μg/ml of puromycin was added as appropriate.Viral stocksTo generate viral stocks, Huh7.5 (for SARS-CoV-2) or Vero-E6 (for HKU5-SARS-CoV-1-S and MERS-CoVs) were inoculated with HKU5-SARS-CoV-1-S (BEI Resources #NR-48814), SARS-CoV-2 isolate USA-WA1/2020 (BEI Resources #NR-52281), MERS-CoV and MERS-CoV T1015N (BEI Resources #NR-48813 and #NR-48811) at a MOI of approximately 0.01 for three days to generate a P1 stock. The P1 stock was then used to inoculate Vero-E6 cells for three days at approximately 50% cytopathic effects. Supernatant was harvested and clarified by centrifugation (450 g x 5 min) and filtered through a 0.45-micron filter, and then aliquoted for storage at −80°C. Virus titer was determined by plaque assay using Vero-E6 cells. To generate icSARS-CoV-2-mNG stocks, lyophilized icSARS-CoV-2-mNG was resuspended in 0.5 mL of deionized water and then 50 μl of virus was diluted in 5 mL media (Xie et al., 2020). icSARS-CoV-2-mNG was provided by the World Reference Center for Emerging Viruses and Arboviruses (Galveston, TX). This was then added to 107 Vero-E6 cells in a T175 flask. At 3 dpi, the supernatant was collected and clarified by centrifugation (450 gx 5 min), filtered through a 0.45-micron filter, and aliquoted for storage at −80°C. All work with infectious virus was performed in a Biosafety Level 3 laboratory and approved by the Yale University Biosafety Committee.Method DetailsCoronavirus plaque assaysVero-E6 cells were seeded at 7.5 × 105 cells/well in 6-well plates or 4 × 105 cells/well in 12-well plates. The following day, the media was removed and replaced with 100 μl of 10-fold serial dilutions of virus. Plates were incubated at 37°C for 1 hour with gentle rocking. Subsequently, overlay media (DMEM, 2% FBS, 0.6% Avicel RC-581) was added to each well. At 2 dpi for SARS-CoV-2 and 3 dpi for other coronaviruses, plates were fixed with 10% formaldehyde for 30 min, stained with crystal violet solution (0.5% crystal violet in 20% ethanol) for 30 min, and then rinsed with deionized water to visualize plaques.Genome-wide CRISPR screensVero-E6 cells (ATCC) were transduced with lenti-Cas9 (Cas9-v1, Addgene 52962) or pLX_311-Cas9 (Cas9-v2, Addgene 96924) and selected with blasticidin (5 μg/ml) for 10 days. Cas9 activity was assessed by transducing parental Vero-E6 or Vero-E6-Cas9 cells with pXPR_047 (Addgene 107645), which expresses eGFP and an sgRNA targeting eGFP (Doench et al., 2014). Susceptibility to SARS-CoV-2 infection remained similar between parental cells and Cas9 expressing cells. Cells were transduced for 24 hours, selected for five days with puromycin, and the frequency of eGFP expression was assessed by flow cytometry on a Cytoflex S (Beckman). The African green monkey (AGM) genome-wide CRISPR knockout library (CP0070), which contains four unique sgRNA per gene in pXPR_050 (Addgene 96925) was designed according to the same general principles as the ‘Brunello’ human genome-wide library (Sanson et al., 2018), was delivered by lentiviral transduction of 2 × 108 Vero-E6-Cas9 at ∼0.3 MOI. This equates to 6 × 107 transduced cells, which is sufficient for the integration of each sgRNA into ∼750 unique cells. Two days post-transduction, puromycin was added to the media and transduced cells were selected for seven days.For SARS-CoV-2 screens, two infection conditions were set up for the screening with Cas9-v1: (1) 10% FBS, 5 × 106 cells, MOI 0.1 (2) 10% FBS, 5 × 106 cells, MOI 0.01. Five infection conditions were set up for the screening with Cas9-v2: (1) 10% FBS, 5 × 106 cells, MOI 0.1; (2) 5% FBS, 5 × 106 cells, MOI 0.1; (3) 5% FBS, 2.5 × 106 cells, MOI 0.1; (4) 5% FBS, 2.5 × 106 cells, MOI 0.01; (5) 2% FBS, 5 × 106 cells, MOI 0.1. For each condition, a total of 4 × 107cells were seeded in T175 flasks at the indicated cell concentrations. For the Cas9-v1 screen, the mock sample was plated in 10% FBS at 5 × 106 cells in each of eight T175 flasks. For the Cas9-v2 screen, the mock sample was plated identically to condition (2) above in 5% FBS at 5 × 106 cells in each of eight T175 flasks. Cells were infected with SARS-CoV-2 at the indicated MOI. One condition (5% FBS, 2.5 × 106 cells per T175 flask, MOI 0.1) was used for HKU5-SARS-CoV-1-S, rcVSV-SARS-CoV-2-S, MERS-CoV WT (EMC/2012) and MERS-CoV T1015N screens. Mock infected cells were harvested 48 hours after seeding and served as a reference for sgRNA enrichment analysis. At 4 dpi, 80% of the media was exchanged for fresh media. At 7-9 dpi, cell lysates were harvested in DNA/RNA shield (Zymo Research) and genomic DNA (gDNA) of surviving cells was isolated using a gDNA cleanup kit according to manufacturer instructions (Zymo Research, D4065). For Illumina sequencing and screening analysis, PCR was performed on gDNA to construct Illumina sequencing libraries, with each well containing 10 μg gDNA (Doench et al., 2016; Orchard et al., 2016). For PCR amplification, gDNA was divided into 100 μL reactions such that each well had at most 10 μg of gDNA. Per 96 well plate, a master mix consisted of 144 μL of 50x Titanium Taq DNA Polymerase (Takara), 960 μL of 10x Titanium Taq buffer, 768 μL of dNTP (stock at 2.5mM) provided with the enzyme, 48 μL of P5 stagger primer mix (stock at 100 μM concentration), 480 μL of DMSO, and 1.44 mL water. Each well consisted of 50 μL gDNA plus water, 40 μL PCR master mix, and 10 μL of a uniquely barcoded P7 primer (stock at 5 μM concentration).PCR cycling conditions: an initial 1 min at 95 °C; followed by 30 s at 94 °C, 30 s at 53 °C, 30 s at 72 °C, for 28 cycles; and a final 10 min extension at 72 °C. PCR primers were synthesized at Integrated DNA Technologies (IDT). PCR products were purified with Agencourt AMPure XP SPRI beads according to manufacturer’s instructions (Beckman Coulter, A63880). Samples were sequenced on a HiSeq2500 High Output flowcell (Illumina). Reads were counted by alignment to a reference file of all possible guide RNAs present in the library. The read was then assigned to a condition (e.g., a well on the PCR plate) on the basis of the 8 nt index included in the P7 primer. The lentiviral plasmid DNA pool was also sequenced as a reference. sgRNA sequences for the genome-wide CRISPR library are in Table S5.To assess technical performance, we calculated the log-fold change of each guide relative to the original lentivirus plasmid pool and observed strong correlation between different cell culture conditions, with the greatest distinction between the two different Cas9 constructs (Pearson’s r > 0.61 among Cas9-v2 conditions and r > 0.46 among Cas9-v1 conditions; Figure S1A). We compared the depletion of sgRNAs targeting essential genes versus sgRNAs targeting non-essential genes in the mock-infected conditions with each Cas9 construct and observed superior performance with the Cas9-v2 construct (AUC = 0.82 versus 0.70 for Cas9-v1; Figure S1B), although the top positively and negatively selected hits remained concordant between the two Cas9 constructs (Pearson’s r = 0.24; Figure S1C) (Hart et al., 2014, 2015). The enhanced Cas9 activity of Cas9-v2 was confirmed with a GFP reporter assay (Figures S1D and S1E). We therefore proceeded with the data from the Cas9-v2 screens and calculated a guide-level residual (representing a log2 fold change) between mock-infected and SARS-CoV-2 infected cells (Figure S1F). A positive residual indicates a gene is pro-viral and confers resistance to virus-induced cell death, while a negative residual indicates a gene is anti-viral and sensitizes a cell to virus-induced cell death. A z-score for enrichment or depletion was determined for each condition based on the distribution of residuals for all sgRNAs. We then averaged z-scores across all five Cas9-v2 screen conditions, combined p values using Fisher’s method, and calculated a false discovery rate (FDR) using the Benjamini-Hochberg procedure to identify hit genes.Secondary CRISPR subpool screenA custom secondary CRISPR knockout subpool library (CP1564) was designed with 6208 unique sgRNAs including 10 sgRNAs for each of the top 250 and bottom 250 genes from the genome-wide SARS-CoV-2 screen. 500 non-targeting control sgRNAs and other genes of interest including DPP4 were also included. The sgRNAs were cloned into pXPR_050 (Addgene 96925). The sgRNAs were delivered by lentiviral transduction of 4 × 107 Vero-E6-Cas9-v2 at ∼0.2 MOI. This equates to 8 × 106 transduced cells, which is sufficient for the integration of each sgRNA into ∼1000 unique cells. Two days post-transduction, puromycin was added to the media and transduced cells were selected for seven days. Eight viruses were used for the secondary CRISPR screen, including HKU5-SARS-CoV-1-S, SARS-CoV-2, rcVSV-SARS-CoV-2-S, MERS-CoV, MERS-CoV T1015N, IAV-WSN, EMCV, and VSV (Indiana). All the viruses were screened in duplicate except IAV-WSN. 3 × 106 transduced Vero-E6 cells were plated in 5% FBS in T150 flasks. Mock infected cells were harvested 48 hours after seeding and served as a reference for sgRNA enrichment analysis. At 4 dpi, 80% of the media was exchanged for fresh media. At 7 dpi, cell lysates were harvested in DNA/RNA shield buffer and gDNA of surviving cells was isolated for sequencing.Tertiary CRISPR screen in Calu-3 cellsTertiary CRISPR knockout library (CP1560), which contains 148 sgRNAs by targeting 32 human genes with 4 sgRNAs per gene and 20 non-targeting control sgRNAs in lentiCRISPRv2 (Addgene 52961), was delivered by lentiviral transduction of 2 × 106 Calu-3 cells at ∼0.2 MOI. Two days post-transduction, puromycin was added to the media and transduced cells were selected for ten days. 5 × 105 Calu-3 cells were plated in 5% FBS media in 6-well plates and infected with SARS-CoV-2 at 0.1 MOI. At 4 dpi, 80% of the media was exchanged for fresh media. At 7 dpi, genomic DNA of surviving cells was isolated for sequencing.Screen analysisGuide sequences were extracted from the sequencing reads with PoolQ version 3.2.9 (Broad Institute; https://portals.broadinstitute.org/gpp/public/software/poolq), using a “CACCG” search prefix, and a counts matrix was generated. Read counts were log-normalized within each condition using the following formula:log−normalizedreadpermillionforguide=log2((#ofreadsforguide/totalreadsinconditionx1e6)+1)Prior to analysis, any sgRNAs with an outlier abundance in the plasmid DNA pool (defined as a log-normalized read count > 3 standard deviations from the mean) or that had > 5 predicted off-target sites with a CFD score = 1 (“Match Bin I”) were filtered out. This removed 755 sgRNAs; the remaining 84,208 sgRNAs were used for all analyses. We then calculated log-fold changes (LFCs) by subtracting the log-normalized plasmid DNA. For each condition, we fit a natural cubic spline with 4 degrees of freedom, using the mock infected LFCs as the independent variable and the relevant condition’s LFCs as the dependent variable. We used the residual from this fit spline to represent the deviation from the expected LFC for each guide. To combine these residuals at the gene level, we calculated a z-score for each condition, z = (x-μ)/(σ/n), where x is the mean residual for a gene, μ is the mean residual of all sgRNAs, σ is the standard deviation of all sgRNAs and n is the number of sgRNAs for a given gene. We used the normal distribution function to calculate p values from the z-scores. To combine p values across multiple conditions we used Fisher’s method. Finally, to calculate the false discovery rate for each gene we used the Benjamini-Hochberg procedure. We used the same pDNA and off-target filters for the secondary and tertiary libraries. To analyze these libraries, we z-scored LFCs using intergenic control sgRNAs.Gene set enrichment and network analysisWe used the STRING enrichment detection tool to identify significantly enriched gene sets, using African green monkey gene symbols, but testing for enrichment across human gene sets. We analyzed sets from all available sources provided by that tool, including sets of clusters of protein-protein interactors in STRING, and excluding the PubMed gene sets. We then generated a network of enriched gene sets by drawing edges between sets with a significant overlap between genes. We evaluated the significance of overlap using Fisher’s exact test. We clustered the network using a weighted graph, treating the fraction of genes that overlap between any sets as edge weights and proteins as nodes, weighted by the absolute value of the z-score. Then, we used the infomap algorithm (Rosvall and Bergstrom, 2008), to cluster the network. We evaluated the centrality of each node to a given cluster using the PageRank algorithm with a damping factor of 0.5 and employing the same edge and node weights (for personalized PageRank) as we did with clustering (Brin and Page, 1998).Arrayed secondary assessment of CRISPR screen hits by cell viabilityHMGB1 sgRNAs were cloned into lentiCRISPRv2 (Addgene 52961) which also encodes the Cas9 gene (Sanjana et al., 2014). sgRNAs for all other genes were cloned into lentiGuide-Puro or a variant thereof, pXPR_050 (Addgene 52963, 96925, respectively). Individual sgRNAs target sequences are in Table S6. Vero-E6-Cas9-v2 cells were individually transduced with lentiviruses expressing one to three unique sgRNA per gene and then selected with puromycin for 7 days. After selection, 1.25 × 103 cells were seeded in each well of a 384-well black walled clear bottom plate in 20 μl of DMEM + 5% FBS. The following day, 5 μl of SARS-CoV-2 was added for a final MOI of 0.2. Cells were incubated for three days before assessing cellular viability by CellTiter Glo (Promega). For each cell line, viability was determined in SARS-CoV-2 infected relative to mock infected cells. Five replicates per condition were performed in each of three independent experiments.SARS-CoV-2 fluorescent reporter virus assayVero-E6 cells were plated at 2.5 × 103 cells per well in a 384-well plate and then the following day, icSARS-CoV-2-mNG was added at a MOI of 1.0 (Xie et al., 2020). Infected cell frequencies as measured by mNeonGreen expression were assessed at 2 dpi by high content imaging (Cytation 5, BioTek) configured with bright field and GFP cubes. Total cell numbers were quantified by Gen5 software of brightfield images. Object analysis was used to determine the number of mNeonGreen positive cells. The percentage of infection was calculated as the ratio between the number of mNeonGreen+ cells and the total number of cells in brightfield. Data are normalized to the average of DMSO treated cells.Identification of anti-viral drugs targeting CRISPR gene hitsCalpain Inhibitor III (#14283), SIS3 (#15495), and PFI-3 (#15267) were purchased from Cayman Chemical. Drugs were resuspended at a stock concentration of 40 mM in DMSO and then two-fold serial dilutions were performed in DMSO. 20 nL of 1000X drug stock were spotted into each well of a 384-well plate using Labcyte ECHO acoustic dispenser at the Yale Center for Molecular Discovery. 1.25 × 103 Vero-E6 cells were plated per well in 20 μl of phenol-red free DMEM containing 5% FBS. Two days later, 5,000 PFU (MOI ∼1) icSARS-CoV-2-mNG in 5 μl media was added. Cells were incubated at 37°C and 5% CO2 for two days. Infected cell frequencies were quantified by mNeonGreen at 2 dpi (Cytation 5, BioTek). In parallel, 1000 PFU (MOI∼0.2) SARS-CoV-2 in 5 μl media was added to replicate plates and cell viability was quantified by CellTiter Glo at 3 dpi. Vero-E6, Huh7.5 and Calu-3 cells were pretreated with 10 μM SIS3 and 40 μM PFI-3 for 48 hours and then infected with SARS-CoV-2 at a MOI of 0.1. Viral production was determined by plaque assay. Cytotoxicity was not observed in these cell lines during the time and concentration of drug used.RNA-seqTotal cellular RNA was extracted using Direct-zol RNA MiniPrep Kit and submitted to the Yale Center for Genome Analysis for library preparation. RNA-seq libraries were sequenced on an Illumina NovaSeq 6000 instrument with the goal of at least 25 × 106 reads per sample. Reads were aligned to reference genome chlSab2, NCBI annotation release 100, using STAR aligner v2.7.3a (Dobin et al., 2013) with parameters–winAnchorMultimapNmax 200–outFilterMultimapNmax 100–quantMode GeneCounts. Differential expression was obtained using the R package DESeq2 v1.32 (Dobin et al., 2013; Love et al., 2014). Bigwig files were generated using deeptools v3.1.3 (Ramírez et al., 2016) with parameter–normalizeUsing RPKM.ChIP-seqAll ChIP samples were prepared in duplicate. Approximately 20 × 106 Vero-E6 cells were used per immunoprecipitation. Cells were washed twice with PBS and crosslinked with 1% formaldehyde (Pierce) for 10 min at room temperature. Crosslinking was quenched with 125 mM (final) glycine for 5 min and washed 2 × with PBS. Cells were lysed in 4 ml lysis buffer (50 mM HEPES pH 7.9, 140 mM NaCl, 1 mM EDTA, 10% glycerol, 0.5% NP-40, 0.25% Triton X-100), 1 × Protease inhibitor (Roche) for 10 min on ice. Lysed cells were centrifuged at 4,000 rpm for 5 min at 4 °C and washed 2 × with 4 ml cold wash buffer (10 mM Tris-Cl pH 7.5, 200 mM NaCl, 1 mM EDTA pH 8.0, 0.5 mM EGTA pH 8.0), 1 × protease inhibitors. The pellet was resuspended in 1 ml shearing buffer (0.1% SDS, 1 mM EDTA, 10 mM Tris pH 7.5), 1 × protease inhibitor and sheared on Covaris S220 (140 W, 5% duty factor, 200 bursts per cycle, 4 °C) for 12 min (determined by time course optimization experiment). Extract was diluted with Triton X-100 (1% final) and NaCl (150 mM final) and cleared by centrifugation at 21,000g, 4 °C for 10 min. For input, 10% of material was set aside. Cleared extract was supplemented with 2 μg of antibody and incubated overnight at 4 °C. Immuno-precipitated chromatin was captured on 30 μl of Dynabeads protein G (Invitrogen) at 4 °C for 1.5 h. Beads were washed twice with low-salt buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM HEPES–potassium hydroxide pH 7.5, 150 mM NaCl), 2 × with high-salt buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM HEPES–potassium hydroxide pH 7.5, 500 mM NaCl), 1 × LiCl Buffer (100 mM Tris-HCl pH 7.5, 0.5 M LiCl, 1% NP-40, 1% sodium deoxycholate) and 1 × TE buffer (10 mM Tris-HCl pH 8.0, 0.1 mM EDTA). Enriched DNA was eluted in 100 μl of proteinase K buffer (20mM HEPES pH 7.5, 1 mM EDTA, 0.5% SDS) supplemented with 40 μg of proteinase K (Ambion) for 30 min at 50 °C. Formaldehyde crosslinks were reversed by adding NaCl (150 mM final) and 0.25 μg DNase-free RNase (Roche), followed by incubation overnight at 65 °C. DNA was isolated using QIAquick PCR purification kit (QIAGEN) and submitted to the Yale Center for Genome Analysis for library preparation. ChIP-seq libraries were sequenced on a Illumina NovaSeq 6000 instrument as 101 nt long paired-end reads, with the goal of at least 20 × 106 reads per IP. Reads were trimmed of adaptor sequences using Cutadapt (Martin, 2011) and aligned to the reference genome chlSab2 using Bowtie2 (Langmead and Salzberg, 2012). Alignments were filtered using SAMtools (Li et al., 2009), and peak calls and enrichment tracks were created using MACS2 (Zhang et al., 2008). Differential analysis at called peaks was performed using DESeq2 (Love et al., 2014). Peaks were assigned to the nearest transcription start site within 100kb for integration with RNA-seq data and overlaps of ChIP-seq and ATAC-seq peaks were determined using bedtools.ATAC-seqATAC-seq libraries were generated following the omni-ATAC protocol as described (Corces et al., 2017). Two biological repeats were generated per each sample. 50,000 viable cells were resuspended in 50 μl cold ATAC-Resuspension Buffer (RSB) (10mM Tris-HCl, pH7.4, 10mM NaCl, and 3mM MgCl2) containing 0.1% NP40, 0.1% Tween-20, and 0.01% digitonin and incubated on ice for 3 minutes. 1ml of cold ATAC-RSB containing 0.1% Tween-20 was then added to wash out the lysis. Samples were centrifuged at 500 RCF for 10 minutes at 4°C. The nuclei pellets were resuspended in 50 μl transposase reaction mix (25 μl 2x TD buffer, 2.5 μl transposase, 16.5 μl PBS, 0.5 μl 1% digitonin, 0.5 μl 10% Tween-20, and 5 μl H2O) and incubated at 37°C for 30 minutes in a thermomixer with 1000 RPM mixing. Reactions were cleaned up with Zymo DNA Clean and Concentrator-5 kit; Transposed DNA sample was eluted in a 20 μl elution buffer. Transposed samples were pre-amplified for 5 cycles using NEBNext 2x Master Mix in 50 μl reaction mix (2.5 μl of 25 μM primer Ad1, 2.5 μl of 25 μM primer Ad2, 25 μl of 2x Master Mix and 20 μl transposed elution) at cycling conditions as following: 72°C, 5minutes; 98°C, 30 s; then 5 cycles of (98°C, 10 s; 63°C, 30 s; 72°C, 1 minute). 15 μl of qPCR amplification reaction (5 μl of pre-amplified sample; 0.5 μl of 25 μM primer Ad1, 0.5 μl of 25 μM primer Ad2, 5 μl of 2x NEBNext Master Mix, 0.24 μl of 25x SYBR Green in DMSO, and 3.76 μl of H2O) was carried out at cycling conditions as following: 98°C, 30 s; then 20 cycles of (98°C, 10 s; 63°C, 30 s; 72°C, 1 minute. The required number of additional cycles for each sample was determined as described (Buenrostro et al., 2015). After the final amplification, PCR reaction was purified using a Zymo DNA Clean and Concentrator-5 kit and eluted in 20 μl H2O. To remove primer dimers and larger than 1,000 bp fragments, double-sided bead purification was proceeded with Ampure XP beads. 0.5x volume AMPure XP beads were added to each reaction and incubated at room temperature for 10 minutes and then separated in a magnetic rack for 5 minutes. Supernatant was transferred to a new tube and incubated with 1.3x original volume AMPure XP beads at room temperature for 10 minutes and then separated in a magnetic rack for 5 minutes. Supernatant was discarded. Beads were washed twice with 200 μl 80% ethanol (freshly made) and air-dried to ensure all ethanol was removed. Final ATAC-seq libraries were eluted in 20 μl nuclease-free H2O from the beads. ATAC-seq libraries were sequenced on an Illumina NovaSeq S4 instrument as 101 nt long paired-end reads, with the goal of at least 50 × 106 reads per replicate. Reads were trimmed of Nextera adaptor sequences using Trimmomatic v0.39 (Bolger et al., 2014) and aligned to chlSab2 using Bowtie2 v2.2.9 (Langmead and Salzberg, 2012) with parameter -X2000. Duplicates were marked using Picard Tools v2.9.0 (Broad Institute. version 2.9.0. “Picard Tools.” Broad Institute, GitHub repository. http://broadinstitute.github.io/picard/). Duplicated, unpaired, and mitochondrial reads were removed using SAMTools v1.9 (Li et al., 2009). Reads were shifted +4 bp and −5 bp for forward and reverse strands, respectively. Peaks were called using MACS2 v2.2.6 (Zhang et al., 2008) with parameters–nomodel–keep-dup all -s 1–shift −75–extsize 150. Reads that fell inside peaks were counted using featureCounts v1.6.2 (Liao et al., 2014) and differential accessibility analysis was performed using DESeq2 v1.32 (Love et al., 2014). Bigwig files were generated using deeptools v3.1.3 (Ramírez et al., 2016) with parameter–normalizeUsing RPKM.RT-qPCRTotal RNA was isolated from cells using Direct-zol RNA MiniPrep Plus kit, reverse transcribed, and subjected to real-time PCR analysis to measure mRNA levels of tested genes. Data shown are the relative abundance of the indicated mRNA normalized to that of actin. Gene-specific primer sequences were as follows ACE2: GGGATCAGAGATCGGAAGAAGA (forward) and AAGGAGGTCTGAACATCATCAGTG (reverse); Actin: GAGCACAGAGCCTCGCCTTT (forward) and ATCATCATCCATGGTGAGCTGG (reverse).Nuclear/Cytosol FractionationVero-E6 cells were mock-infected or infected with SARS-CoV-2 at a MOI of 1.0 for 24 hours before cellular fractionation was performed using a Nuclear/Cytosol fractionation kit (BioVision Cat#K266-25) according to manufacturer instructions. In brief, 1 × 107 cells were collected by centrifugation at 600 x g for 5 min at 4°C. Add 0.2 mL Cytosol Extraction Buffer A (CEB-A) to fully resuspend the cell pellet. After 10 min incubation on ice, add 11 μl of ice-cold CEB-B and incubate on ice for 1min. Centrifuge at 16000 x g for 5 min, the supernatant was collected as cytoplasmic fraction. The pellet was resuspended in 100 μl of ice-cold Nuclear Extraction Buffer (NEB) vertex every 10 min for a total of 40 min. The samples were centrifuge at 16000 x g for 10 min, and the supernatant was collected as the nuclear fraction.Western blotCells were collected and lysed in Nonidet P-40 lysis buffer (20 mM Tris-HCl [pH 7.4], 150 mM NaCl, 1 mM EDTA, 1% Nonidet P-40, 10 mg/ml aprotinin, 10 mg/ml leupeptin, and 1 mM PMSF). The cell lysates or cellular fractions were fractionated on SDS-PAGE and transferred to a PVDF membrane. Immunoblotting analyses were performed with the indicated antibodies and visualized with horseradish peroxidase-coupled goat anti-mouse/rabbit IgG using a chemiluminescence detection system (BioRad ChemiDoc MP).Generation of HMGB1 knockout and complemented cellsVero-E6 cells were individually transduced with lentiviruses expressing two guide RNAs targeting HMGB1 (Table S6) and then selected with puromycin for 7 days. Single cells were then sorted by flow cytometry and HMGB1 knockout was confirmed by western blot. HMGB1 KO clones were complemented by lentiviral transduction of pLenti6/V5-DEST vector containing human HMGB1 with a C-terminal V5. Two days post transduction, blasticidin was added and cells were selected for five days. The expression of HMGB1 in complemented cells was detected by western blot.Pseudovirus productionVSV-based pseudovirus particles (VSVpp) were produced in 293T cells. Cells were transfected with pCAGGS or pcDNA3.1 vector expressing the CoV spike glycoprotein and then inoculated with a replication-deficient VSV virus that contains expression cassettes for Renilla luciferase instead of the vesicular stomatitis virus G (VSV-G) open reading frame (Avanzato, 2019). After an incubation period of 1 h at 37°C, the inoculum was removed and cells were washed with PBS before media supplemented with anti-VSV-G clone IE9F9 was added in order to neutralize residual input virus (no antibody was added to cells expressing VSV-G) (Lefrancois and Lyles, 1982). Pseudotyped particles were harvested 24 hours post inoculation, clarified from cellular debris by centrifugation and stored at −80°C before use.Plasmids encoding codon-optimized form of SARS-CoV-1-S glycoprotein, MERS-CoV SΔCT and NL63 SΔCT glycoproteins lacking cytoplasmic tail were previously described (Huang et al., 2006; Letko et al., 2020). Vector pCAGGS containing the SARS-CoV-2, Wuhan-Hu-1 Spike Glycoprotein Gene, NR-52310, was produced under HHSN272201400008C and obtained through BEI Resources, NIAID, NIH.Pseudovirus entry assay1x104 Vero-E6 cells were seeded in 100 μl total volume in each well of a black-walled clear bottom 96-well plate. The following day spike expressing VSV pseudovirus was added at 1:10 final concentration volume/volume and incubated for one day. Cells were lysed with Renilla Luciferase Assay System (Promega) according to manufacturer instructions. Luciferase activity was measured using a microplate reader (BioTek Synergy or Cytation 5).Quantification and Statistical AnalysisStatistical significance was determined as p < 0.05 using GraphPad Prism 8 unless otherwise indicated. Experiments were analyzed by unpaired two-tailed t tests, Mann-Whitney test, or ANOVA, as indicated. Schematic were created with biorender.com.

STAR★Methods

Key Resources Table
REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesAnti-ACE2 antibodyProSciCat#3217; RRID: AB_712925Anti-β-actin antibodyBioLegendCat#622102; RRID: AB_315946Anti-LMNB1 antibodyBioLegendCat#869802; RRID: AB_2820181Anti-GAPDH antibodyBioLegendCat#607902; RRID: AB_2734503Anti-HMGB1 antibodyAbcamCat#ab18256; RRID: AB_444360Anti-SMARCA4 antibodyAbcamCat#ab70558; RRID: AB_1209535Anti-H3K27ac antibodyAbcamCat#ab4729; RRID: AB_2118291Anti-SMAD3 antibodyCell Signaling TechnologyCat#9513; RRID: AB_2286450Anti-KDM6A antibodyCell Signaling TechnologyCat#33510; RRID: AB_2721244Goat anti-rabbit IgG/HRPJackson ImmunoResearchCat#111-035-003; RRID: AB_2337913Goat anti-mouse IgG/HRPJackson ImmunoResearchCat#115-035-003; RRID: AB_10015289Anti-VSV-G antibodyKerafastCat#EB0010; RRID: AB_2811223Bacterial and Virus StrainsCP0070 African Green Monkey (AGM) CRISPR KO libraryBroad InstituteCat#CP0070CP1564 CRISPR KO libraryBroad InstituteCat#CP1564CP1560 CRISPR KO libraryBroad InstituteCat#CP1560SARS-CoV-2, isolate US-WA1/2020BEI ResourcesCat#NR-52281rcVSV-SARS-CoV-2-SCase et al., 2020N/ASARS-CoV-1, HKU5BEI ResourcesCat#NR-48814MERS-CoV WTBEI ResourcesCat#NR-48813MERS-CoV T1015NBEI ResourcesCat#NR-48811EMCVBEI ResourcesCat#NR-19846Influenza A virus/WSN/1933 (IAV)A. Boon (Wustl)N/AicSARS-CoV-2 mNGXie et al., 2020N/AChemicals, Peptides, and Recombinant ProteinsDMEM mediaThermo Fisher ScientificCat#C11995500BTEMEM mediaATCCCat#MT10009CVFetal bovine serum (FBS)VWRCat# VW97068-085Puromycin DihydrochlorideGIBCOCat#A1113803Blasticidin S HClGIBCOCat#A1113903Lipofectamine 2000InvitrogenCat#11668019FormaldehydeJ.B BakerCat#14650-250Calpain inhibitor IIICaymanCat#14283PFI-3CaymanCat#15267SIS3CaymanCat#15495Recombinant human HMGB1 proteinBioLegendCat#557804Critical Commercial AssaysCellTiter-Glo® Luminescent Cell Viability AssayPromegaCat#G7570Direct-zol RNA Miniprep Plus kitZymo ResearchCat#R2072Renilla Luciferase Assay SystemPromegaCat#E2820Nuclear/Cytosol Fractionation KitBioVisionCat#K266-25Human HMGB1/HMG-1 ELISA KitNovus BiologicalsCat#NBP2-62766Deposited DataATAC-seq dataThis paperGSE154784RNA-seq dataThis paperGSE154784ChIP-seq dataThis paperGSE154761CRISPR dataThis paperhttps://doi.org/10.17632/pbcrc9c7zs.1sgRNA sequencesThis paperhttps://doi.org/10.17632/pbcrc9c7zs.1Experimental Models: Cell LinesHEK293T cellsATCCCat#CRL-3216Vero-E6 cellsATCCCat#CRL-1586Huh7.5 cellsATCCCat#CVCL-7927Calu-3 cellsATCCCat#HTB-55OligonucleotidessgRNAs target sequences, see Table S6This paperN/AACE2 qPCR primer forward: GGGATCAGAGATCGGAAGAAGA; reverse: AAGGAGGTCTGAACATCATCAGTGThis paperN/AActin qPCR primer forward GAGCACAGAGCCTCGCCTTT; reverse: ATCATCATCCATGGTGAGCTGGThis paperN/ARecombinant DNALenti-Cas9-Blast (v1)Sanjana et al., 2014Addgene Cat#52962pLX_311-Cas9 (v2)Doench et al., 2014Addgene Cat #96924pXPR_047Doench et al., 2014Addgene Cat #107645LentiCRISPRv2Sanjana et al., 2014Addgene Cat #52961Lenti-Guide-PuroSanjana et al., 2014Addgene Cat #52963pXPR_05Sanson et al., 2018Addgene Cat#96925psPAX2Didier TronoAddgene Cat#12260VSVGStewart et al., 2003Addgene Cat#8454pLenti6/V5-DEST-HMGB1Scott et al., 2011Addgene Cat#31208Software and AlgorithmsBowtie2 v2.2.9Langmead and Salzberg, 2012N/ACutadaptMartin, 2011N/ADESeq2 v1.32Love et al., 2014N/Adeeptools v3.1.3Ramírez et al., 2016N/AFlowjo 10.6.2FLOWJOhttps://www.flowjo.comGraphpad Prism 8Graphpad softwarehttps://www.graphpad.com/scientific-software/prism/MACS2Zhang et al., 2008N/APoolQ version 3.2.9Broad Institutehttps://portals.broadinstitute.org/gpp/public/software/poolq/Picard Tools v2.9.0Broad Institutehttp://broadinstitute.github.io/picard/STAR aligner v2.7.3aDobin et al., 2013N/ASAMTools v1.9Li et al., 2009N/ATrimmomatic v0.39Bolger et al., 2014N/ACRISPR screen analysisThis paperhttps://github.com/PeterDeWeirdt/coronavirus_screen_analysis

Key Resources Table

REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesAnti-ACE2 antibodyProSciCat#3217; RRID: AB_712925Anti-β-actin antibodyBioLegendCat#622102; RRID: AB_315946Anti-LMNB1 antibodyBioLegendCat#869802; RRID: AB_2820181Anti-GAPDH antibodyBioLegendCat#607902; RRID: AB_2734503Anti-HMGB1 antibodyAbcamCat#ab18256; RRID: AB_444360Anti-SMARCA4 antibodyAbcamCat#ab70558; RRID: AB_1209535Anti-H3K27ac antibodyAbcamCat#ab4729; RRID: AB_2118291Anti-SMAD3 antibodyCell Signaling TechnologyCat#9513; RRID: AB_2286450Anti-KDM6A antibodyCell Signaling TechnologyCat#33510; RRID: AB_2721244Goat anti-rabbit IgG/HRPJackson ImmunoResearchCat#111-035-003; RRID: AB_2337913Goat anti-mouse IgG/HRPJackson ImmunoResearchCat#115-035-003; RRID: AB_10015289Anti-VSV-G antibodyKerafastCat#EB0010; RRID: AB_2811223Bacterial and Virus StrainsCP0070 African Green Monkey (AGM) CRISPR KO libraryBroad InstituteCat#CP0070CP1564 CRISPR KO libraryBroad InstituteCat#CP1564CP1560 CRISPR KO libraryBroad InstituteCat#CP1560SARS-CoV-2, isolate US-WA1/2020BEI ResourcesCat#NR-52281rcVSV-SARS-CoV-2-SCase et al., 2020N/ASARS-CoV-1, HKU5BEI ResourcesCat#NR-48814MERS-CoV WTBEI ResourcesCat#NR-48813MERS-CoV T1015NBEI ResourcesCat#NR-48811EMCVBEI ResourcesCat#NR-19846Influenza A virus/WSN/1933 (IAV)A. Boon (Wustl)N/AicSARS-CoV-2 mNGXie et al., 2020N/AChemicals, Peptides, and Recombinant ProteinsDMEM mediaThermo Fisher ScientificCat#C11995500BTEMEM mediaATCCCat#MT10009CVFetal bovine serum (FBS)VWRCat# VW97068-085Puromycin DihydrochlorideGIBCOCat#A1113803Blasticidin S HClGIBCOCat#A1113903Lipofectamine 2000InvitrogenCat#11668019FormaldehydeJ.B BakerCat#14650-250Calpain inhibitor IIICaymanCat#14283PFI-3CaymanCat#15267SIS3CaymanCat#15495Recombinant human HMGB1 proteinBioLegendCat#557804Critical Commercial AssaysCellTiter-Glo® Luminescent Cell Viability AssayPromegaCat#G7570Direct-zol RNA Miniprep Plus kitZymo ResearchCat#R2072Renilla Luciferase Assay SystemPromegaCat#E2820Nuclear/Cytosol Fractionation KitBioVisionCat#K266-25Human HMGB1/HMG-1 ELISA KitNovus BiologicalsCat#NBP2-62766Deposited DataATAC-seq dataThis paperGSE154784RNA-seq dataThis paperGSE154784ChIP-seq dataThis paperGSE154761CRISPR dataThis paperhttps://doi.org/10.17632/pbcrc9c7zs.1sgRNA sequencesThis paperhttps://doi.org/10.17632/pbcrc9c7zs.1Experimental Models: Cell LinesHEK293T cellsATCCCat#CRL-3216Vero-E6 cellsATCCCat#CRL-1586Huh7.5 cellsATCCCat#CVCL-7927Calu-3 cellsATCCCat#HTB-55OligonucleotidessgRNAs target sequences, see Table S6This paperN/AACE2 qPCR primer forward: GGGATCAGAGATCGGAAGAAGA; reverse: AAGGAGGTCTGAACATCATCAGTGThis paperN/AActin qPCR primer forward GAGCACAGAGCCTCGCCTTT; reverse: ATCATCATCCATGGTGAGCTGGThis paperN/ARecombinant DNALenti-Cas9-Blast (v1)Sanjana et al., 2014Addgene Cat#52962pLX_311-Cas9 (v2)Doench et al., 2014Addgene Cat #96924pXPR_047Doench et al., 2014Addgene Cat #107645LentiCRISPRv2Sanjana et al., 2014Addgene Cat #52961Lenti-Guide-PuroSanjana et al., 2014Addgene Cat #52963pXPR_05Sanson et al., 2018Addgene Cat#96925psPAX2Didier TronoAddgene Cat#12260VSVGStewart et al., 2003Addgene Cat#8454pLenti6/V5-DEST-HMGB1Scott et al., 2011Addgene Cat#31208Software and AlgorithmsBowtie2 v2.2.9Langmead and Salzberg, 2012N/ACutadaptMartin, 2011N/ADESeq2 v1.32Love et al., 2014N/Adeeptools v3.1.3Ramírez et al., 2016N/AFlowjo 10.6.2FLOWJOhttps://www.flowjo.comGraphpad Prism 8Graphpad softwarehttps://www.graphpad.com/scientific-software/prism/MACS2Zhang et al., 2008N/APoolQ version 3.2.9Broad Institutehttps://portals.broadinstitute.org/gpp/public/software/poolq/Picard Tools v2.9.0Broad Institutehttp://broadinstitute.github.io/picard/STAR aligner v2.7.3aDobin et al., 2013N/ASAMTools v1.9Li et al., 2009N/ATrimmomatic v0.39Bolger et al., 2014N/ACRISPR screen analysisThis paperhttps://github.com/PeterDeWeirdt/coronavirus_screen_analysis

Resource AvailabilityLead ContactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Craig Wilen (craig.wilen@yale.edu).Materials AvailabilityAll requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact author. Materials will be made publicly available either through publicly available repositories or via the authors upon execution of a Material Transfer Agreement.Data and Code AvailabilityThe published article includes all data generated or analyzed during the study except as described below. The RNA-seq (accession GSE154784), ATAC-Seq (accession GSE154784) and ChIP-Seq data (accession GSE154761) are available at NCBI GEO. The analysis code generated during the study are available at: https://github.com/PeterDeWeirdt/coronavirus_screen_analysis.

Resource Availability

Lead ContactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Craig Wilen (craig.wilen@yale.edu).

Lead Contact

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Craig Wilen (craig.wilen@yale.edu).

Materials AvailabilityAll requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact author. Materials will be made publicly available either through publicly available repositories or via the authors upon execution of a Material Transfer Agreement.

Materials Availability

All requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact author. Materials will be made publicly available either through publicly available repositories or via the authors upon execution of a Material Transfer Agreement.

Data and Code AvailabilityThe published article includes all data generated or analyzed during the study except as described below. The RNA-seq (accession GSE154784), ATAC-Seq (accession GSE154784) and ChIP-Seq data (accession GSE154761) are available at NCBI GEO. The analysis code generated during the study are available at: https://github.com/PeterDeWeirdt/coronavirus_screen_analysis.

Data and Code Availability

The published article includes all data generated or analyzed during the study except as described below. The RNA-seq (accession GSE154784), ATAC-Seq (accession GSE154784) and ChIP-Seq data (accession GSE154761) are available at NCBI GEO. The analysis code generated during the study are available at: https://github.com/PeterDeWeirdt/coronavirus_screen_analysis.

Experimental Model and Subject DetailsCellsHEK293T, Vero-E6 and Huh7.5 cells were cultured in Dulbecco’s Modified Eagle Medium (DMEM) with 10% heat-inactivated fetal bovine serum (FBS), and 1% Penicillin/Streptomycin unless otherwise indicated. Calu-3 cells were cultured in Eagle’s minimal essential medium (EMEM) with 10% FBS and 1% Penicillin/Streptomycin. For Vero-E6 and Huh7.5 cells, 5 μg/ml of puromycin (GIBCO) and 5 μg/ml blasticidin (GIBCO), were added as appropriate. For Calu-3 cells, 1 μg/ml of puromycin was added as appropriate.Viral stocksTo generate viral stocks, Huh7.5 (for SARS-CoV-2) or Vero-E6 (for HKU5-SARS-CoV-1-S and MERS-CoVs) were inoculated with HKU5-SARS-CoV-1-S (BEI Resources #NR-48814), SARS-CoV-2 isolate USA-WA1/2020 (BEI Resources #NR-52281), MERS-CoV and MERS-CoV T1015N (BEI Resources #NR-48813 and #NR-48811) at a MOI of approximately 0.01 for three days to generate a P1 stock. The P1 stock was then used to inoculate Vero-E6 cells for three days at approximately 50% cytopathic effects. Supernatant was harvested and clarified by centrifugation (450 g x 5 min) and filtered through a 0.45-micron filter, and then aliquoted for storage at −80°C. Virus titer was determined by plaque assay using Vero-E6 cells. To generate icSARS-CoV-2-mNG stocks, lyophilized icSARS-CoV-2-mNG was resuspended in 0.5 mL of deionized water and then 50 μl of virus was diluted in 5 mL media (Xie et al., 2020). icSARS-CoV-2-mNG was provided by the World Reference Center for Emerging Viruses and Arboviruses (Galveston, TX). This was then added to 107 Vero-E6 cells in a T175 flask. At 3 dpi, the supernatant was collected and clarified by centrifugation (450 gx 5 min), filtered through a 0.45-micron filter, and aliquoted for storage at −80°C. All work with infectious virus was performed in a Biosafety Level 3 laboratory and approved by the Yale University Biosafety Committee.

Experimental Model and Subject Details

CellsHEK293T, Vero-E6 and Huh7.5 cells were cultured in Dulbecco’s Modified Eagle Medium (DMEM) with 10% heat-inactivated fetal bovine serum (FBS), and 1% Penicillin/Streptomycin unless otherwise indicated. Calu-3 cells were cultured in Eagle’s minimal essential medium (EMEM) with 10% FBS and 1% Penicillin/Streptomycin. For Vero-E6 and Huh7.5 cells, 5 μg/ml of puromycin (GIBCO) and 5 μg/ml blasticidin (GIBCO), were added as appropriate. For Calu-3 cells, 1 μg/ml of puromycin was added as appropriate.

HEK293T, Vero-E6 and Huh7.5 cells were cultured in Dulbecco’s Modified Eagle Medium (DMEM) with 10% heat-inactivated fetal bovine serum (FBS), and 1% Penicillin/Streptomycin unless otherwise indicated. Calu-3 cells were cultured in Eagle’s minimal essential medium (EMEM) with 10% FBS and 1% Penicillin/Streptomycin. For Vero-E6 and Huh7.5 cells, 5 μg/ml of puromycin (GIBCO) and 5 μg/ml blasticidin (GIBCO), were added as appropriate. For Calu-3 cells, 1 μg/ml of puromycin was added as appropriate.

Viral stocksTo generate viral stocks, Huh7.5 (for SARS-CoV-2) or Vero-E6 (for HKU5-SARS-CoV-1-S and MERS-CoVs) were inoculated with HKU5-SARS-CoV-1-S (BEI Resources #NR-48814), SARS-CoV-2 isolate USA-WA1/2020 (BEI Resources #NR-52281), MERS-CoV and MERS-CoV T1015N (BEI Resources #NR-48813 and #NR-48811) at a MOI of approximately 0.01 for three days to generate a P1 stock. The P1 stock was then used to inoculate Vero-E6 cells for three days at approximately 50% cytopathic effects. Supernatant was harvested and clarified by centrifugation (450 g x 5 min) and filtered through a 0.45-micron filter, and then aliquoted for storage at −80°C. Virus titer was determined by plaque assay using Vero-E6 cells. To generate icSARS-CoV-2-mNG stocks, lyophilized icSARS-CoV-2-mNG was resuspended in 0.5 mL of deionized water and then 50 μl of virus was diluted in 5 mL media (Xie et al., 2020). icSARS-CoV-2-mNG was provided by the World Reference Center for Emerging Viruses and Arboviruses (Galveston, TX). This was then added to 107 Vero-E6 cells in a T175 flask. At 3 dpi, the supernatant was collected and clarified by centrifugation (450 gx 5 min), filtered through a 0.45-micron filter, and aliquoted for storage at −80°C. All work with infectious virus was performed in a Biosafety Level 3 laboratory and approved by the Yale University Biosafety Committee.

Viral stocks

To generate viral stocks, Huh7.5 (for SARS-CoV-2) or Vero-E6 (for HKU5-SARS-CoV-1-S and MERS-CoVs) were inoculated with HKU5-SARS-CoV-1-S (BEI Resources #NR-48814), SARS-CoV-2 isolate USA-WA1/2020 (BEI Resources #NR-52281), MERS-CoV and MERS-CoV T1015N (BEI Resources #NR-48813 and #NR-48811) at a MOI of approximately 0.01 for three days to generate a P1 stock. The P1 stock was then used to inoculate Vero-E6 cells for three days at approximately 50% cytopathic effects. Supernatant was harvested and clarified by centrifugation (450 g x 5 min) and filtered through a 0.45-micron filter, and then aliquoted for storage at −80°C. Virus titer was determined by plaque assay using Vero-E6 cells. To generate icSARS-CoV-2-mNG stocks, lyophilized icSARS-CoV-2-mNG was resuspended in 0.5 mL of deionized water and then 50 μl of virus was diluted in 5 mL media (Xie et al., 2020). icSARS-CoV-2-mNG was provided by the World Reference Center for Emerging Viruses and Arboviruses (Galveston, TX). This was then added to 107 Vero-E6 cells in a T175 flask. At 3 dpi, the supernatant was collected and clarified by centrifugation (450 gx 5 min), filtered through a 0.45-micron filter, and aliquoted for storage at −80°C. All work with infectious virus was performed in a Biosafety Level 3 laboratory and approved by the Yale University Biosafety Committee.

Method DetailsCoronavirus plaque assaysVero-E6 cells were seeded at 7.5 × 105 cells/well in 6-well plates or 4 × 105 cells/well in 12-well plates. The following day, the media was removed and replaced with 100 μl of 10-fold serial dilutions of virus. Plates were incubated at 37°C for 1 hour with gentle rocking. Subsequently, overlay media (DMEM, 2% FBS, 0.6% Avicel RC-581) was added to each well. At 2 dpi for SARS-CoV-2 and 3 dpi for other coronaviruses, plates were fixed with 10% formaldehyde for 30 min, stained with crystal violet solution (0.5% crystal violet in 20% ethanol) for 30 min, and then rinsed with deionized water to visualize plaques.Genome-wide CRISPR screensVero-E6 cells (ATCC) were transduced with lenti-Cas9 (Cas9-v1, Addgene 52962) or pLX_311-Cas9 (Cas9-v2, Addgene 96924) and selected with blasticidin (5 μg/ml) for 10 days. Cas9 activity was assessed by transducing parental Vero-E6 or Vero-E6-Cas9 cells with pXPR_047 (Addgene 107645), which expresses eGFP and an sgRNA targeting eGFP (Doench et al., 2014). Susceptibility to SARS-CoV-2 infection remained similar between parental cells and Cas9 expressing cells. Cells were transduced for 24 hours, selected for five days with puromycin, and the frequency of eGFP expression was assessed by flow cytometry on a Cytoflex S (Beckman). The African green monkey (AGM) genome-wide CRISPR knockout library (CP0070), which contains four unique sgRNA per gene in pXPR_050 (Addgene 96925) was designed according to the same general principles as the ‘Brunello’ human genome-wide library (Sanson et al., 2018), was delivered by lentiviral transduction of 2 × 108 Vero-E6-Cas9 at ∼0.3 MOI. This equates to 6 × 107 transduced cells, which is sufficient for the integration of each sgRNA into ∼750 unique cells. Two days post-transduction, puromycin was added to the media and transduced cells were selected for seven days.For SARS-CoV-2 screens, two infection conditions were set up for the screening with Cas9-v1: (1) 10% FBS, 5 × 106 cells, MOI 0.1 (2) 10% FBS, 5 × 106 cells, MOI 0.01. Five infection conditions were set up for the screening with Cas9-v2: (1) 10% FBS, 5 × 106 cells, MOI 0.1; (2) 5% FBS, 5 × 106 cells, MOI 0.1; (3) 5% FBS, 2.5 × 106 cells, MOI 0.1; (4) 5% FBS, 2.5 × 106 cells, MOI 0.01; (5) 2% FBS, 5 × 106 cells, MOI 0.1. For each condition, a total of 4 × 107cells were seeded in T175 flasks at the indicated cell concentrations. For the Cas9-v1 screen, the mock sample was plated in 10% FBS at 5 × 106 cells in each of eight T175 flasks. For the Cas9-v2 screen, the mock sample was plated identically to condition (2) above in 5% FBS at 5 × 106 cells in each of eight T175 flasks. Cells were infected with SARS-CoV-2 at the indicated MOI. One condition (5% FBS, 2.5 × 106 cells per T175 flask, MOI 0.1) was used for HKU5-SARS-CoV-1-S, rcVSV-SARS-CoV-2-S, MERS-CoV WT (EMC/2012) and MERS-CoV T1015N screens. Mock infected cells were harvested 48 hours after seeding and served as a reference for sgRNA enrichment analysis. At 4 dpi, 80% of the media was exchanged for fresh media. At 7-9 dpi, cell lysates were harvested in DNA/RNA shield (Zymo Research) and genomic DNA (gDNA) of surviving cells was isolated using a gDNA cleanup kit according to manufacturer instructions (Zymo Research, D4065). For Illumina sequencing and screening analysis, PCR was performed on gDNA to construct Illumina sequencing libraries, with each well containing 10 μg gDNA (Doench et al., 2016; Orchard et al., 2016). For PCR amplification, gDNA was divided into 100 μL reactions such that each well had at most 10 μg of gDNA. Per 96 well plate, a master mix consisted of 144 μL of 50x Titanium Taq DNA Polymerase (Takara), 960 μL of 10x Titanium Taq buffer, 768 μL of dNTP (stock at 2.5mM) provided with the enzyme, 48 μL of P5 stagger primer mix (stock at 100 μM concentration), 480 μL of DMSO, and 1.44 mL water. Each well consisted of 50 μL gDNA plus water, 40 μL PCR master mix, and 10 μL of a uniquely barcoded P7 primer (stock at 5 μM concentration).PCR cycling conditions: an initial 1 min at 95 °C; followed by 30 s at 94 °C, 30 s at 53 °C, 30 s at 72 °C, for 28 cycles; and a final 10 min extension at 72 °C. PCR primers were synthesized at Integrated DNA Technologies (IDT). PCR products were purified with Agencourt AMPure XP SPRI beads according to manufacturer’s instructions (Beckman Coulter, A63880). Samples were sequenced on a HiSeq2500 High Output flowcell (Illumina). Reads were counted by alignment to a reference file of all possible guide RNAs present in the library. The read was then assigned to a condition (e.g., a well on the PCR plate) on the basis of the 8 nt index included in the P7 primer. The lentiviral plasmid DNA pool was also sequenced as a reference. sgRNA sequences for the genome-wide CRISPR library are in Table S5.To assess technical performance, we calculated the log-fold change of each guide relative to the original lentivirus plasmid pool and observed strong correlation between different cell culture conditions, with the greatest distinction between the two different Cas9 constructs (Pearson’s r > 0.61 among Cas9-v2 conditions and r > 0.46 among Cas9-v1 conditions; Figure S1A). We compared the depletion of sgRNAs targeting essential genes versus sgRNAs targeting non-essential genes in the mock-infected conditions with each Cas9 construct and observed superior performance with the Cas9-v2 construct (AUC = 0.82 versus 0.70 for Cas9-v1; Figure S1B), although the top positively and negatively selected hits remained concordant between the two Cas9 constructs (Pearson’s r = 0.24; Figure S1C) (Hart et al., 2014, 2015). The enhanced Cas9 activity of Cas9-v2 was confirmed with a GFP reporter assay (Figures S1D and S1E). We therefore proceeded with the data from the Cas9-v2 screens and calculated a guide-level residual (representing a log2 fold change) between mock-infected and SARS-CoV-2 infected cells (Figure S1F). A positive residual indicates a gene is pro-viral and confers resistance to virus-induced cell death, while a negative residual indicates a gene is anti-viral and sensitizes a cell to virus-induced cell death. A z-score for enrichment or depletion was determined for each condition based on the distribution of residuals for all sgRNAs. We then averaged z-scores across all five Cas9-v2 screen conditions, combined p values using Fisher’s method, and calculated a false discovery rate (FDR) using the Benjamini-Hochberg procedure to identify hit genes.Secondary CRISPR subpool screenA custom secondary CRISPR knockout subpool library (CP1564) was designed with 6208 unique sgRNAs including 10 sgRNAs for each of the top 250 and bottom 250 genes from the genome-wide SARS-CoV-2 screen. 500 non-targeting control sgRNAs and other genes of interest including DPP4 were also included. The sgRNAs were cloned into pXPR_050 (Addgene 96925). The sgRNAs were delivered by lentiviral transduction of 4 × 107 Vero-E6-Cas9-v2 at ∼0.2 MOI. This equates to 8 × 106 transduced cells, which is sufficient for the integration of each sgRNA into ∼1000 unique cells. Two days post-transduction, puromycin was added to the media and transduced cells were selected for seven days. Eight viruses were used for the secondary CRISPR screen, including HKU5-SARS-CoV-1-S, SARS-CoV-2, rcVSV-SARS-CoV-2-S, MERS-CoV, MERS-CoV T1015N, IAV-WSN, EMCV, and VSV (Indiana). All the viruses were screened in duplicate except IAV-WSN. 3 × 106 transduced Vero-E6 cells were plated in 5% FBS in T150 flasks. Mock infected cells were harvested 48 hours after seeding and served as a reference for sgRNA enrichment analysis. At 4 dpi, 80% of the media was exchanged for fresh media. At 7 dpi, cell lysates were harvested in DNA/RNA shield buffer and gDNA of surviving cells was isolated for sequencing.Tertiary CRISPR screen in Calu-3 cellsTertiary CRISPR knockout library (CP1560), which contains 148 sgRNAs by targeting 32 human genes with 4 sgRNAs per gene and 20 non-targeting control sgRNAs in lentiCRISPRv2 (Addgene 52961), was delivered by lentiviral transduction of 2 × 106 Calu-3 cells at ∼0.2 MOI. Two days post-transduction, puromycin was added to the media and transduced cells were selected for ten days. 5 × 105 Calu-3 cells were plated in 5% FBS media in 6-well plates and infected with SARS-CoV-2 at 0.1 MOI. At 4 dpi, 80% of the media was exchanged for fresh media. At 7 dpi, genomic DNA of surviving cells was isolated for sequencing.Screen analysisGuide sequences were extracted from the sequencing reads with PoolQ version 3.2.9 (Broad Institute; https://portals.broadinstitute.org/gpp/public/software/poolq), using a “CACCG” search prefix, and a counts matrix was generated. Read counts were log-normalized within each condition using the following formula:log−normalizedreadpermillionforguide=log2((#ofreadsforguide/totalreadsinconditionx1e6)+1)Prior to analysis, any sgRNAs with an outlier abundance in the plasmid DNA pool (defined as a log-normalized read count > 3 standard deviations from the mean) or that had > 5 predicted off-target sites with a CFD score = 1 (“Match Bin I”) were filtered out. This removed 755 sgRNAs; the remaining 84,208 sgRNAs were used for all analyses. We then calculated log-fold changes (LFCs) by subtracting the log-normalized plasmid DNA. For each condition, we fit a natural cubic spline with 4 degrees of freedom, using the mock infected LFCs as the independent variable and the relevant condition’s LFCs as the dependent variable. We used the residual from this fit spline to represent the deviation from the expected LFC for each guide. To combine these residuals at the gene level, we calculated a z-score for each condition, z = (x-μ)/(σ/n), where x is the mean residual for a gene, μ is the mean residual of all sgRNAs, σ is the standard deviation of all sgRNAs and n is the number of sgRNAs for a given gene. We used the normal distribution function to calculate p values from the z-scores. To combine p values across multiple conditions we used Fisher’s method. Finally, to calculate the false discovery rate for each gene we used the Benjamini-Hochberg procedure. We used the same pDNA and off-target filters for the secondary and tertiary libraries. To analyze these libraries, we z-scored LFCs using intergenic control sgRNAs.Gene set enrichment and network analysisWe used the STRING enrichment detection tool to identify significantly enriched gene sets, using African green monkey gene symbols, but testing for enrichment across human gene sets. We analyzed sets from all available sources provided by that tool, including sets of clusters of protein-protein interactors in STRING, and excluding the PubMed gene sets. We then generated a network of enriched gene sets by drawing edges between sets with a significant overlap between genes. We evaluated the significance of overlap using Fisher’s exact test. We clustered the network using a weighted graph, treating the fraction of genes that overlap between any sets as edge weights and proteins as nodes, weighted by the absolute value of the z-score. Then, we used the infomap algorithm (Rosvall and Bergstrom, 2008), to cluster the network. We evaluated the centrality of each node to a given cluster using the PageRank algorithm with a damping factor of 0.5 and employing the same edge and node weights (for personalized PageRank) as we did with clustering (Brin and Page, 1998).Arrayed secondary assessment of CRISPR screen hits by cell viabilityHMGB1 sgRNAs were cloned into lentiCRISPRv2 (Addgene 52961) which also encodes the Cas9 gene (Sanjana et al., 2014). sgRNAs for all other genes were cloned into lentiGuide-Puro or a variant thereof, pXPR_050 (Addgene 52963, 96925, respectively). Individual sgRNAs target sequences are in Table S6. Vero-E6-Cas9-v2 cells were individually transduced with lentiviruses expressing one to three unique sgRNA per gene and then selected with puromycin for 7 days. After selection, 1.25 × 103 cells were seeded in each well of a 384-well black walled clear bottom plate in 20 μl of DMEM + 5% FBS. The following day, 5 μl of SARS-CoV-2 was added for a final MOI of 0.2. Cells were incubated for three days before assessing cellular viability by CellTiter Glo (Promega). For each cell line, viability was determined in SARS-CoV-2 infected relative to mock infected cells. Five replicates per condition were performed in each of three independent experiments.SARS-CoV-2 fluorescent reporter virus assayVero-E6 cells were plated at 2.5 × 103 cells per well in a 384-well plate and then the following day, icSARS-CoV-2-mNG was added at a MOI of 1.0 (Xie et al., 2020). Infected cell frequencies as measured by mNeonGreen expression were assessed at 2 dpi by high content imaging (Cytation 5, BioTek) configured with bright field and GFP cubes. Total cell numbers were quantified by Gen5 software of brightfield images. Object analysis was used to determine the number of mNeonGreen positive cells. The percentage of infection was calculated as the ratio between the number of mNeonGreen+ cells and the total number of cells in brightfield. Data are normalized to the average of DMSO treated cells.Identification of anti-viral drugs targeting CRISPR gene hitsCalpain Inhibitor III (#14283), SIS3 (#15495), and PFI-3 (#15267) were purchased from Cayman Chemical. Drugs were resuspended at a stock concentration of 40 mM in DMSO and then two-fold serial dilutions were performed in DMSO. 20 nL of 1000X drug stock were spotted into each well of a 384-well plate using Labcyte ECHO acoustic dispenser at the Yale Center for Molecular Discovery. 1.25 × 103 Vero-E6 cells were plated per well in 20 μl of phenol-red free DMEM containing 5% FBS. Two days later, 5,000 PFU (MOI ∼1) icSARS-CoV-2-mNG in 5 μl media was added. Cells were incubated at 37°C and 5% CO2 for two days. Infected cell frequencies were quantified by mNeonGreen at 2 dpi (Cytation 5, BioTek). In parallel, 1000 PFU (MOI∼0.2) SARS-CoV-2 in 5 μl media was added to replicate plates and cell viability was quantified by CellTiter Glo at 3 dpi. Vero-E6, Huh7.5 and Calu-3 cells were pretreated with 10 μM SIS3 and 40 μM PFI-3 for 48 hours and then infected with SARS-CoV-2 at a MOI of 0.1. Viral production was determined by plaque assay. Cytotoxicity was not observed in these cell lines during the time and concentration of drug used.RNA-seqTotal cellular RNA was extracted using Direct-zol RNA MiniPrep Kit and submitted to the Yale Center for Genome Analysis for library preparation. RNA-seq libraries were sequenced on an Illumina NovaSeq 6000 instrument with the goal of at least 25 × 106 reads per sample. Reads were aligned to reference genome chlSab2, NCBI annotation release 100, using STAR aligner v2.7.3a (Dobin et al., 2013) with parameters–winAnchorMultimapNmax 200–outFilterMultimapNmax 100–quantMode GeneCounts. Differential expression was obtained using the R package DESeq2 v1.32 (Dobin et al., 2013; Love et al., 2014). Bigwig files were generated using deeptools v3.1.3 (Ramírez et al., 2016) with parameter–normalizeUsing RPKM.ChIP-seqAll ChIP samples were prepared in duplicate. Approximately 20 × 106 Vero-E6 cells were used per immunoprecipitation. Cells were washed twice with PBS and crosslinked with 1% formaldehyde (Pierce) for 10 min at room temperature. Crosslinking was quenched with 125 mM (final) glycine for 5 min and washed 2 × with PBS. Cells were lysed in 4 ml lysis buffer (50 mM HEPES pH 7.9, 140 mM NaCl, 1 mM EDTA, 10% glycerol, 0.5% NP-40, 0.25% Triton X-100), 1 × Protease inhibitor (Roche) for 10 min on ice. Lysed cells were centrifuged at 4,000 rpm for 5 min at 4 °C and washed 2 × with 4 ml cold wash buffer (10 mM Tris-Cl pH 7.5, 200 mM NaCl, 1 mM EDTA pH 8.0, 0.5 mM EGTA pH 8.0), 1 × protease inhibitors. The pellet was resuspended in 1 ml shearing buffer (0.1% SDS, 1 mM EDTA, 10 mM Tris pH 7.5), 1 × protease inhibitor and sheared on Covaris S220 (140 W, 5% duty factor, 200 bursts per cycle, 4 °C) for 12 min (determined by time course optimization experiment). Extract was diluted with Triton X-100 (1% final) and NaCl (150 mM final) and cleared by centrifugation at 21,000g, 4 °C for 10 min. For input, 10% of material was set aside. Cleared extract was supplemented with 2 μg of antibody and incubated overnight at 4 °C. Immuno-precipitated chromatin was captured on 30 μl of Dynabeads protein G (Invitrogen) at 4 °C for 1.5 h. Beads were washed twice with low-salt buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM HEPES–potassium hydroxide pH 7.5, 150 mM NaCl), 2 × with high-salt buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM HEPES–potassium hydroxide pH 7.5, 500 mM NaCl), 1 × LiCl Buffer (100 mM Tris-HCl pH 7.5, 0.5 M LiCl, 1% NP-40, 1% sodium deoxycholate) and 1 × TE buffer (10 mM Tris-HCl pH 8.0, 0.1 mM EDTA). Enriched DNA was eluted in 100 μl of proteinase K buffer (20mM HEPES pH 7.5, 1 mM EDTA, 0.5% SDS) supplemented with 40 μg of proteinase K (Ambion) for 30 min at 50 °C. Formaldehyde crosslinks were reversed by adding NaCl (150 mM final) and 0.25 μg DNase-free RNase (Roche), followed by incubation overnight at 65 °C. DNA was isolated using QIAquick PCR purification kit (QIAGEN) and submitted to the Yale Center for Genome Analysis for library preparation. ChIP-seq libraries were sequenced on a Illumina NovaSeq 6000 instrument as 101 nt long paired-end reads, with the goal of at least 20 × 106 reads per IP. Reads were trimmed of adaptor sequences using Cutadapt (Martin, 2011) and aligned to the reference genome chlSab2 using Bowtie2 (Langmead and Salzberg, 2012). Alignments were filtered using SAMtools (Li et al., 2009), and peak calls and enrichment tracks were created using MACS2 (Zhang et al., 2008). Differential analysis at called peaks was performed using DESeq2 (Love et al., 2014). Peaks were assigned to the nearest transcription start site within 100kb for integration with RNA-seq data and overlaps of ChIP-seq and ATAC-seq peaks were determined using bedtools.ATAC-seqATAC-seq libraries were generated following the omni-ATAC protocol as described (Corces et al., 2017). Two biological repeats were generated per each sample. 50,000 viable cells were resuspended in 50 μl cold ATAC-Resuspension Buffer (RSB) (10mM Tris-HCl, pH7.4, 10mM NaCl, and 3mM MgCl2) containing 0.1% NP40, 0.1% Tween-20, and 0.01% digitonin and incubated on ice for 3 minutes. 1ml of cold ATAC-RSB containing 0.1% Tween-20 was then added to wash out the lysis. Samples were centrifuged at 500 RCF for 10 minutes at 4°C. The nuclei pellets were resuspended in 50 μl transposase reaction mix (25 μl 2x TD buffer, 2.5 μl transposase, 16.5 μl PBS, 0.5 μl 1% digitonin, 0.5 μl 10% Tween-20, and 5 μl H2O) and incubated at 37°C for 30 minutes in a thermomixer with 1000 RPM mixing. Reactions were cleaned up with Zymo DNA Clean and Concentrator-5 kit; Transposed DNA sample was eluted in a 20 μl elution buffer. Transposed samples were pre-amplified for 5 cycles using NEBNext 2x Master Mix in 50 μl reaction mix (2.5 μl of 25 μM primer Ad1, 2.5 μl of 25 μM primer Ad2, 25 μl of 2x Master Mix and 20 μl transposed elution) at cycling conditions as following: 72°C, 5minutes; 98°C, 30 s; then 5 cycles of (98°C, 10 s; 63°C, 30 s; 72°C, 1 minute). 15 μl of qPCR amplification reaction (5 μl of pre-amplified sample; 0.5 μl of 25 μM primer Ad1, 0.5 μl of 25 μM primer Ad2, 5 μl of 2x NEBNext Master Mix, 0.24 μl of 25x SYBR Green in DMSO, and 3.76 μl of H2O) was carried out at cycling conditions as following: 98°C, 30 s; then 20 cycles of (98°C, 10 s; 63°C, 30 s; 72°C, 1 minute. The required number of additional cycles for each sample was determined as described (Buenrostro et al., 2015). After the final amplification, PCR reaction was purified using a Zymo DNA Clean and Concentrator-5 kit and eluted in 20 μl H2O. To remove primer dimers and larger than 1,000 bp fragments, double-sided bead purification was proceeded with Ampure XP beads. 0.5x volume AMPure XP beads were added to each reaction and incubated at room temperature for 10 minutes and then separated in a magnetic rack for 5 minutes. Supernatant was transferred to a new tube and incubated with 1.3x original volume AMPure XP beads at room temperature for 10 minutes and then separated in a magnetic rack for 5 minutes. Supernatant was discarded. Beads were washed twice with 200 μl 80% ethanol (freshly made) and air-dried to ensure all ethanol was removed. Final ATAC-seq libraries were eluted in 20 μl nuclease-free H2O from the beads. ATAC-seq libraries were sequenced on an Illumina NovaSeq S4 instrument as 101 nt long paired-end reads, with the goal of at least 50 × 106 reads per replicate. Reads were trimmed of Nextera adaptor sequences using Trimmomatic v0.39 (Bolger et al., 2014) and aligned to chlSab2 using Bowtie2 v2.2.9 (Langmead and Salzberg, 2012) with parameter -X2000. Duplicates were marked using Picard Tools v2.9.0 (Broad Institute. version 2.9.0. “Picard Tools.” Broad Institute, GitHub repository. http://broadinstitute.github.io/picard/). Duplicated, unpaired, and mitochondrial reads were removed using SAMTools v1.9 (Li et al., 2009). Reads were shifted +4 bp and −5 bp for forward and reverse strands, respectively. Peaks were called using MACS2 v2.2.6 (Zhang et al., 2008) with parameters–nomodel–keep-dup all -s 1–shift −75–extsize 150. Reads that fell inside peaks were counted using featureCounts v1.6.2 (Liao et al., 2014) and differential accessibility analysis was performed using DESeq2 v1.32 (Love et al., 2014). Bigwig files were generated using deeptools v3.1.3 (Ramírez et al., 2016) with parameter–normalizeUsing RPKM.RT-qPCRTotal RNA was isolated from cells using Direct-zol RNA MiniPrep Plus kit, reverse transcribed, and subjected to real-time PCR analysis to measure mRNA levels of tested genes. Data shown are the relative abundance of the indicated mRNA normalized to that of actin. Gene-specific primer sequences were as follows ACE2: GGGATCAGAGATCGGAAGAAGA (forward) and AAGGAGGTCTGAACATCATCAGTG (reverse); Actin: GAGCACAGAGCCTCGCCTTT (forward) and ATCATCATCCATGGTGAGCTGG (reverse).Nuclear/Cytosol FractionationVero-E6 cells were mock-infected or infected with SARS-CoV-2 at a MOI of 1.0 for 24 hours before cellular fractionation was performed using a Nuclear/Cytosol fractionation kit (BioVision Cat#K266-25) according to manufacturer instructions. In brief, 1 × 107 cells were collected by centrifugation at 600 x g for 5 min at 4°C. Add 0.2 mL Cytosol Extraction Buffer A (CEB-A) to fully resuspend the cell pellet. After 10 min incubation on ice, add 11 μl of ice-cold CEB-B and incubate on ice for 1min. Centrifuge at 16000 x g for 5 min, the supernatant was collected as cytoplasmic fraction. The pellet was resuspended in 100 μl of ice-cold Nuclear Extraction Buffer (NEB) vertex every 10 min for a total of 40 min. The samples were centrifuge at 16000 x g for 10 min, and the supernatant was collected as the nuclear fraction.Western blotCells were collected and lysed in Nonidet P-40 lysis buffer (20 mM Tris-HCl [pH 7.4], 150 mM NaCl, 1 mM EDTA, 1% Nonidet P-40, 10 mg/ml aprotinin, 10 mg/ml leupeptin, and 1 mM PMSF). The cell lysates or cellular fractions were fractionated on SDS-PAGE and transferred to a PVDF membrane. Immunoblotting analyses were performed with the indicated antibodies and visualized with horseradish peroxidase-coupled goat anti-mouse/rabbit IgG using a chemiluminescence detection system (BioRad ChemiDoc MP).Generation of HMGB1 knockout and complemented cellsVero-E6 cells were individually transduced with lentiviruses expressing two guide RNAs targeting HMGB1 (Table S6) and then selected with puromycin for 7 days. Single cells were then sorted by flow cytometry and HMGB1 knockout was confirmed by western blot. HMGB1 KO clones were complemented by lentiviral transduction of pLenti6/V5-DEST vector containing human HMGB1 with a C-terminal V5. Two days post transduction, blasticidin was added and cells were selected for five days. The expression of HMGB1 in complemented cells was detected by western blot.Pseudovirus productionVSV-based pseudovirus particles (VSVpp) were produced in 293T cells. Cells were transfected with pCAGGS or pcDNA3.1 vector expressing the CoV spike glycoprotein and then inoculated with a replication-deficient VSV virus that contains expression cassettes for Renilla luciferase instead of the vesicular stomatitis virus G (VSV-G) open reading frame (Avanzato, 2019). After an incubation period of 1 h at 37°C, the inoculum was removed and cells were washed with PBS before media supplemented with anti-VSV-G clone IE9F9 was added in order to neutralize residual input virus (no antibody was added to cells expressing VSV-G) (Lefrancois and Lyles, 1982). Pseudotyped particles were harvested 24 hours post inoculation, clarified from cellular debris by centrifugation and stored at −80°C before use.Plasmids encoding codon-optimized form of SARS-CoV-1-S glycoprotein, MERS-CoV SΔCT and NL63 SΔCT glycoproteins lacking cytoplasmic tail were previously described (Huang et al., 2006; Letko et al., 2020). Vector pCAGGS containing the SARS-CoV-2, Wuhan-Hu-1 Spike Glycoprotein Gene, NR-52310, was produced under HHSN272201400008C and obtained through BEI Resources, NIAID, NIH.Pseudovirus entry assay1x104 Vero-E6 cells were seeded in 100 μl total volume in each well of a black-walled clear bottom 96-well plate. The following day spike expressing VSV pseudovirus was added at 1:10 final concentration volume/volume and incubated for one day. Cells were lysed with Renilla Luciferase Assay System (Promega) according to manufacturer instructions. Luciferase activity was measured using a microplate reader (BioTek Synergy or Cytation 5).

Method Details

Coronavirus plaque assaysVero-E6 cells were seeded at 7.5 × 105 cells/well in 6-well plates or 4 × 105 cells/well in 12-well plates. The following day, the media was removed and replaced with 100 μl of 10-fold serial dilutions of virus. Plates were incubated at 37°C for 1 hour with gentle rocking. Subsequently, overlay media (DMEM, 2% FBS, 0.6% Avicel RC-581) was added to each well. At 2 dpi for SARS-CoV-2 and 3 dpi for other coronaviruses, plates were fixed with 10% formaldehyde for 30 min, stained with crystal violet solution (0.5% crystal violet in 20% ethanol) for 30 min, and then rinsed with deionized water to visualize plaques.

Coronavirus plaque assays

Vero-E6 cells were seeded at 7.5 × 105 cells/well in 6-well plates or 4 × 105 cells/well in 12-well plates. The following day, the media was removed and replaced with 100 μl of 10-fold serial dilutions of virus. Plates were incubated at 37°C for 1 hour with gentle rocking. Subsequently, overlay media (DMEM, 2% FBS, 0.6% Avicel RC-581) was added to each well. At 2 dpi for SARS-CoV-2 and 3 dpi for other coronaviruses, plates were fixed with 10% formaldehyde for 30 min, stained with crystal violet solution (0.5% crystal violet in 20% ethanol) for 30 min, and then rinsed with deionized water to visualize plaques.

Genome-wide CRISPR screensVero-E6 cells (ATCC) were transduced with lenti-Cas9 (Cas9-v1, Addgene 52962) or pLX_311-Cas9 (Cas9-v2, Addgene 96924) and selected with blasticidin (5 μg/ml) for 10 days. Cas9 activity was assessed by transducing parental Vero-E6 or Vero-E6-Cas9 cells with pXPR_047 (Addgene 107645), which expresses eGFP and an sgRNA targeting eGFP (Doench et al., 2014). Susceptibility to SARS-CoV-2 infection remained similar between parental cells and Cas9 expressing cells. Cells were transduced for 24 hours, selected for five days with puromycin, and the frequency of eGFP expression was assessed by flow cytometry on a Cytoflex S (Beckman). The African green monkey (AGM) genome-wide CRISPR knockout library (CP0070), which contains four unique sgRNA per gene in pXPR_050 (Addgene 96925) was designed according to the same general principles as the ‘Brunello’ human genome-wide library (Sanson et al., 2018), was delivered by lentiviral transduction of 2 × 108 Vero-E6-Cas9 at ∼0.3 MOI. This equates to 6 × 107 transduced cells, which is sufficient for the integration of each sgRNA into ∼750 unique cells. Two days post-transduction, puromycin was added to the media and transduced cells were selected for seven days.For SARS-CoV-2 screens, two infection conditions were set up for the screening with Cas9-v1: (1) 10% FBS, 5 × 106 cells, MOI 0.1 (2) 10% FBS, 5 × 106 cells, MOI 0.01. Five infection conditions were set up for the screening with Cas9-v2: (1) 10% FBS, 5 × 106 cells, MOI 0.1; (2) 5% FBS, 5 × 106 cells, MOI 0.1; (3) 5% FBS, 2.5 × 106 cells, MOI 0.1; (4) 5% FBS, 2.5 × 106 cells, MOI 0.01; (5) 2% FBS, 5 × 106 cells, MOI 0.1. For each condition, a total of 4 × 107cells were seeded in T175 flasks at the indicated cell concentrations. For the Cas9-v1 screen, the mock sample was plated in 10% FBS at 5 × 106 cells in each of eight T175 flasks. For the Cas9-v2 screen, the mock sample was plated identically to condition (2) above in 5% FBS at 5 × 106 cells in each of eight T175 flasks. Cells were infected with SARS-CoV-2 at the indicated MOI. One condition (5% FBS, 2.5 × 106 cells per T175 flask, MOI 0.1) was used for HKU5-SARS-CoV-1-S, rcVSV-SARS-CoV-2-S, MERS-CoV WT (EMC/2012) and MERS-CoV T1015N screens. Mock infected cells were harvested 48 hours after seeding and served as a reference for sgRNA enrichment analysis. At 4 dpi, 80% of the media was exchanged for fresh media. At 7-9 dpi, cell lysates were harvested in DNA/RNA shield (Zymo Research) and genomic DNA (gDNA) of surviving cells was isolated using a gDNA cleanup kit according to manufacturer instructions (Zymo Research, D4065). For Illumina sequencing and screening analysis, PCR was performed on gDNA to construct Illumina sequencing libraries, with each well containing 10 μg gDNA (Doench et al., 2016; Orchard et al., 2016). For PCR amplification, gDNA was divided into 100 μL reactions such that each well had at most 10 μg of gDNA. Per 96 well plate, a master mix consisted of 144 μL of 50x Titanium Taq DNA Polymerase (Takara), 960 μL of 10x Titanium Taq buffer, 768 μL of dNTP (stock at 2.5mM) provided with the enzyme, 48 μL of P5 stagger primer mix (stock at 100 μM concentration), 480 μL of DMSO, and 1.44 mL water. Each well consisted of 50 μL gDNA plus water, 40 μL PCR master mix, and 10 μL of a uniquely barcoded P7 primer (stock at 5 μM concentration).PCR cycling conditions: an initial 1 min at 95 °C; followed by 30 s at 94 °C, 30 s at 53 °C, 30 s at 72 °C, for 28 cycles; and a final 10 min extension at 72 °C. PCR primers were synthesized at Integrated DNA Technologies (IDT). PCR products were purified with Agencourt AMPure XP SPRI beads according to manufacturer’s instructions (Beckman Coulter, A63880). Samples were sequenced on a HiSeq2500 High Output flowcell (Illumina). Reads were counted by alignment to a reference file of all possible guide RNAs present in the library. The read was then assigned to a condition (e.g., a well on the PCR plate) on the basis of the 8 nt index included in the P7 primer. The lentiviral plasmid DNA pool was also sequenced as a reference. sgRNA sequences for the genome-wide CRISPR library are in Table S5.To assess technical performance, we calculated the log-fold change of each guide relative to the original lentivirus plasmid pool and observed strong correlation between different cell culture conditions, with the greatest distinction between the two different Cas9 constructs (Pearson’s r > 0.61 among Cas9-v2 conditions and r > 0.46 among Cas9-v1 conditions; Figure S1A). We compared the depletion of sgRNAs targeting essential genes versus sgRNAs targeting non-essential genes in the mock-infected conditions with each Cas9 construct and observed superior performance with the Cas9-v2 construct (AUC = 0.82 versus 0.70 for Cas9-v1; Figure S1B), although the top positively and negatively selected hits remained concordant between the two Cas9 constructs (Pearson’s r = 0.24; Figure S1C) (Hart et al., 2014, 2015). The enhanced Cas9 activity of Cas9-v2 was confirmed with a GFP reporter assay (Figures S1D and S1E). We therefore proceeded with the data from the Cas9-v2 screens and calculated a guide-level residual (representing a log2 fold change) between mock-infected and SARS-CoV-2 infected cells (Figure S1F). A positive residual indicates a gene is pro-viral and confers resistance to virus-induced cell death, while a negative residual indicates a gene is anti-viral and sensitizes a cell to virus-induced cell death. A z-score for enrichment or depletion was determined for each condition based on the distribution of residuals for all sgRNAs. We then averaged z-scores across all five Cas9-v2 screen conditions, combined p values using Fisher’s method, and calculated a false discovery rate (FDR) using the Benjamini-Hochberg procedure to identify hit genes.

Genome-wide CRISPR screens

Vero-E6 cells (ATCC) were transduced with lenti-Cas9 (Cas9-v1, Addgene 52962) or pLX_311-Cas9 (Cas9-v2, Addgene 96924) and selected with blasticidin (5 μg/ml) for 10 days. Cas9 activity was assessed by transducing parental Vero-E6 or Vero-E6-Cas9 cells with pXPR_047 (Addgene 107645), which expresses eGFP and an sgRNA targeting eGFP (Doench et al., 2014). Susceptibility to SARS-CoV-2 infection remained similar between parental cells and Cas9 expressing cells. Cells were transduced for 24 hours, selected for five days with puromycin, and the frequency of eGFP expression was assessed by flow cytometry on a Cytoflex S (Beckman). The African green monkey (AGM) genome-wide CRISPR knockout library (CP0070), which contains four unique sgRNA per gene in pXPR_050 (Addgene 96925) was designed according to the same general principles as the ‘Brunello’ human genome-wide library (Sanson et al., 2018), was delivered by lentiviral transduction of 2 × 108 Vero-E6-Cas9 at ∼0.3 MOI. This equates to 6 × 107 transduced cells, which is sufficient for the integration of each sgRNA into ∼750 unique cells. Two days post-transduction, puromycin was added to the media and transduced cells were selected for seven days.

For SARS-CoV-2 screens, two infection conditions were set up for the screening with Cas9-v1: (1) 10% FBS, 5 × 106 cells, MOI 0.1 (2) 10% FBS, 5 × 106 cells, MOI 0.01. Five infection conditions were set up for the screening with Cas9-v2: (1) 10% FBS, 5 × 106 cells, MOI 0.1; (2) 5% FBS, 5 × 106 cells, MOI 0.1; (3) 5% FBS, 2.5 × 106 cells, MOI 0.1; (4) 5% FBS, 2.5 × 106 cells, MOI 0.01; (5) 2% FBS, 5 × 106 cells, MOI 0.1. For each condition, a total of 4 × 107cells were seeded in T175 flasks at the indicated cell concentrations. For the Cas9-v1 screen, the mock sample was plated in 10% FBS at 5 × 106 cells in each of eight T175 flasks. For the Cas9-v2 screen, the mock sample was plated identically to condition (2) above in 5% FBS at 5 × 106 cells in each of eight T175 flasks. Cells were infected with SARS-CoV-2 at the indicated MOI. One condition (5% FBS, 2.5 × 106 cells per T175 flask, MOI 0.1) was used for HKU5-SARS-CoV-1-S, rcVSV-SARS-CoV-2-S, MERS-CoV WT (EMC/2012) and MERS-CoV T1015N screens. Mock infected cells were harvested 48 hours after seeding and served as a reference for sgRNA enrichment analysis. At 4 dpi, 80% of the media was exchanged for fresh media. At 7-9 dpi, cell lysates were harvested in DNA/RNA shield (Zymo Research) and genomic DNA (gDNA) of surviving cells was isolated using a gDNA cleanup kit according to manufacturer instructions (Zymo Research, D4065). For Illumina sequencing and screening analysis, PCR was performed on gDNA to construct Illumina sequencing libraries, with each well containing 10 μg gDNA (Doench et al., 2016; Orchard et al., 2016). For PCR amplification, gDNA was divided into 100 μL reactions such that each well had at most 10 μg of gDNA. Per 96 well plate, a master mix consisted of 144 μL of 50x Titanium Taq DNA Polymerase (Takara), 960 μL of 10x Titanium Taq buffer, 768 μL of dNTP (stock at 2.5mM) provided with the enzyme, 48 μL of P5 stagger primer mix (stock at 100 μM concentration), 480 μL of DMSO, and 1.44 mL water. Each well consisted of 50 μL gDNA plus water, 40 μL PCR master mix, and 10 μL of a uniquely barcoded P7 primer (stock at 5 μM concentration).

PCR cycling conditions: an initial 1 min at 95 °C; followed by 30 s at 94 °C, 30 s at 53 °C, 30 s at 72 °C, for 28 cycles; and a final 10 min extension at 72 °C. PCR primers were synthesized at Integrated DNA Technologies (IDT). PCR products were purified with Agencourt AMPure XP SPRI beads according to manufacturer’s instructions (Beckman Coulter, A63880). Samples were sequenced on a HiSeq2500 High Output flowcell (Illumina). Reads were counted by alignment to a reference file of all possible guide RNAs present in the library. The read was then assigned to a condition (e.g., a well on the PCR plate) on the basis of the 8 nt index included in the P7 primer. The lentiviral plasmid DNA pool was also sequenced as a reference. sgRNA sequences for the genome-wide CRISPR library are in Table S5.

To assess technical performance, we calculated the log-fold change of each guide relative to the original lentivirus plasmid pool and observed strong correlation between different cell culture conditions, with the greatest distinction between the two different Cas9 constructs (Pearson’s r > 0.61 among Cas9-v2 conditions and r > 0.46 among Cas9-v1 conditions; Figure S1A). We compared the depletion of sgRNAs targeting essential genes versus sgRNAs targeting non-essential genes in the mock-infected conditions with each Cas9 construct and observed superior performance with the Cas9-v2 construct (AUC = 0.82 versus 0.70 for Cas9-v1; Figure S1B), although the top positively and negatively selected hits remained concordant between the two Cas9 constructs (Pearson’s r = 0.24; Figure S1C) (Hart et al., 2014, 2015). The enhanced Cas9 activity of Cas9-v2 was confirmed with a GFP reporter assay (Figures S1D and S1E). We therefore proceeded with the data from the Cas9-v2 screens and calculated a guide-level residual (representing a log2 fold change) between mock-infected and SARS-CoV-2 infected cells (Figure S1F). A positive residual indicates a gene is pro-viral and confers resistance to virus-induced cell death, while a negative residual indicates a gene is anti-viral and sensitizes a cell to virus-induced cell death. A z-score for enrichment or depletion was determined for each condition based on the distribution of residuals for all sgRNAs. We then averaged z-scores across all five Cas9-v2 screen conditions, combined p values using Fisher’s method, and calculated a false discovery rate (FDR) using the Benjamini-Hochberg procedure to identify hit genes.

Secondary CRISPR subpool screenA custom secondary CRISPR knockout subpool library (CP1564) was designed with 6208 unique sgRNAs including 10 sgRNAs for each of the top 250 and bottom 250 genes from the genome-wide SARS-CoV-2 screen. 500 non-targeting control sgRNAs and other genes of interest including DPP4 were also included. The sgRNAs were cloned into pXPR_050 (Addgene 96925). The sgRNAs were delivered by lentiviral transduction of 4 × 107 Vero-E6-Cas9-v2 at ∼0.2 MOI. This equates to 8 × 106 transduced cells, which is sufficient for the integration of each sgRNA into ∼1000 unique cells. Two days post-transduction, puromycin was added to the media and transduced cells were selected for seven days. Eight viruses were used for the secondary CRISPR screen, including HKU5-SARS-CoV-1-S, SARS-CoV-2, rcVSV-SARS-CoV-2-S, MERS-CoV, MERS-CoV T1015N, IAV-WSN, EMCV, and VSV (Indiana). All the viruses were screened in duplicate except IAV-WSN. 3 × 106 transduced Vero-E6 cells were plated in 5% FBS in T150 flasks. Mock infected cells were harvested 48 hours after seeding and served as a reference for sgRNA enrichment analysis. At 4 dpi, 80% of the media was exchanged for fresh media. At 7 dpi, cell lysates were harvested in DNA/RNA shield buffer and gDNA of surviving cells was isolated for sequencing.

Secondary CRISPR subpool screen

A custom secondary CRISPR knockout subpool library (CP1564) was designed with 6208 unique sgRNAs including 10 sgRNAs for each of the top 250 and bottom 250 genes from the genome-wide SARS-CoV-2 screen. 500 non-targeting control sgRNAs and other genes of interest including DPP4 were also included. The sgRNAs were cloned into pXPR_050 (Addgene 96925). The sgRNAs were delivered by lentiviral transduction of 4 × 107 Vero-E6-Cas9-v2 at ∼0.2 MOI. This equates to 8 × 106 transduced cells, which is sufficient for the integration of each sgRNA into ∼1000 unique cells. Two days post-transduction, puromycin was added to the media and transduced cells were selected for seven days. Eight viruses were used for the secondary CRISPR screen, including HKU5-SARS-CoV-1-S, SARS-CoV-2, rcVSV-SARS-CoV-2-S, MERS-CoV, MERS-CoV T1015N, IAV-WSN, EMCV, and VSV (Indiana). All the viruses were screened in duplicate except IAV-WSN. 3 × 106 transduced Vero-E6 cells were plated in 5% FBS in T150 flasks. Mock infected cells were harvested 48 hours after seeding and served as a reference for sgRNA enrichment analysis. At 4 dpi, 80% of the media was exchanged for fresh media. At 7 dpi, cell lysates were harvested in DNA/RNA shield buffer and gDNA of surviving cells was isolated for sequencing.

Tertiary CRISPR screen in Calu-3 cellsTertiary CRISPR knockout library (CP1560), which contains 148 sgRNAs by targeting 32 human genes with 4 sgRNAs per gene and 20 non-targeting control sgRNAs in lentiCRISPRv2 (Addgene 52961), was delivered by lentiviral transduction of 2 × 106 Calu-3 cells at ∼0.2 MOI. Two days post-transduction, puromycin was added to the media and transduced cells were selected for ten days. 5 × 105 Calu-3 cells were plated in 5% FBS media in 6-well plates and infected with SARS-CoV-2 at 0.1 MOI. At 4 dpi, 80% of the media was exchanged for fresh media. At 7 dpi, genomic DNA of surviving cells was isolated for sequencing.

Tertiary CRISPR screen in Calu-3 cells

Tertiary CRISPR knockout library (CP1560), which contains 148 sgRNAs by targeting 32 human genes with 4 sgRNAs per gene and 20 non-targeting control sgRNAs in lentiCRISPRv2 (Addgene 52961), was delivered by lentiviral transduction of 2 × 106 Calu-3 cells at ∼0.2 MOI. Two days post-transduction, puromycin was added to the media and transduced cells were selected for ten days. 5 × 105 Calu-3 cells were plated in 5% FBS media in 6-well plates and infected with SARS-CoV-2 at 0.1 MOI. At 4 dpi, 80% of the media was exchanged for fresh media. At 7 dpi, genomic DNA of surviving cells was isolated for sequencing.

Screen analysisGuide sequences were extracted from the sequencing reads with PoolQ version 3.2.9 (Broad Institute; https://portals.broadinstitute.org/gpp/public/software/poolq), using a “CACCG” search prefix, and a counts matrix was generated. Read counts were log-normalized within each condition using the following formula:log−normalizedreadpermillionforguide=log2((#ofreadsforguide/totalreadsinconditionx1e6)+1)Prior to analysis, any sgRNAs with an outlier abundance in the plasmid DNA pool (defined as a log-normalized read count > 3 standard deviations from the mean) or that had > 5 predicted off-target sites with a CFD score = 1 (“Match Bin I”) were filtered out. This removed 755 sgRNAs; the remaining 84,208 sgRNAs were used for all analyses. We then calculated log-fold changes (LFCs) by subtracting the log-normalized plasmid DNA. For each condition, we fit a natural cubic spline with 4 degrees of freedom, using the mock infected LFCs as the independent variable and the relevant condition’s LFCs as the dependent variable. We used the residual from this fit spline to represent the deviation from the expected LFC for each guide. To combine these residuals at the gene level, we calculated a z-score for each condition, z = (x-μ)/(σ/n), where x is the mean residual for a gene, μ is the mean residual of all sgRNAs, σ is the standard deviation of all sgRNAs and n is the number of sgRNAs for a given gene. We used the normal distribution function to calculate p values from the z-scores. To combine p values across multiple conditions we used Fisher’s method. Finally, to calculate the false discovery rate for each gene we used the Benjamini-Hochberg procedure. We used the same pDNA and off-target filters for the secondary and tertiary libraries. To analyze these libraries, we z-scored LFCs using intergenic control sgRNAs.

Screen analysis

Guide sequences were extracted from the sequencing reads with PoolQ version 3.2.9 (Broad Institute; https://portals.broadinstitute.org/gpp/public/software/poolq), using a “CACCG” search prefix, and a counts matrix was generated. Read counts were log-normalized within each condition using the following formula:log−normalizedreadpermillionforguide=log2((#ofreadsforguide/totalreadsinconditionx1e6)+1)Prior to analysis, any sgRNAs with an outlier abundance in the plasmid DNA pool (defined as a log-normalized read count > 3 standard deviations from the mean) or that had > 5 predicted off-target sites with a CFD score = 1 (“Match Bin I”) were filtered out. This removed 755 sgRNAs; the remaining 84,208 sgRNAs were used for all analyses. We then calculated log-fold changes (LFCs) by subtracting the log-normalized plasmid DNA. For each condition, we fit a natural cubic spline with 4 degrees of freedom, using the mock infected LFCs as the independent variable and the relevant condition’s LFCs as the dependent variable. We used the residual from this fit spline to represent the deviation from the expected LFC for each guide. To combine these residuals at the gene level, we calculated a z-score for each condition, z = (x-μ)/(σ/n), where x is the mean residual for a gene, μ is the mean residual of all sgRNAs, σ is the standard deviation of all sgRNAs and n is the number of sgRNAs for a given gene. We used the normal distribution function to calculate p values from the z-scores. To combine p values across multiple conditions we used Fisher’s method. Finally, to calculate the false discovery rate for each gene we used the Benjamini-Hochberg procedure. We used the same pDNA and off-target filters for the secondary and tertiary libraries. To analyze these libraries, we z-scored LFCs using intergenic control sgRNAs.

Gene set enrichment and network analysisWe used the STRING enrichment detection tool to identify significantly enriched gene sets, using African green monkey gene symbols, but testing for enrichment across human gene sets. We analyzed sets from all available sources provided by that tool, including sets of clusters of protein-protein interactors in STRING, and excluding the PubMed gene sets. We then generated a network of enriched gene sets by drawing edges between sets with a significant overlap between genes. We evaluated the significance of overlap using Fisher’s exact test. We clustered the network using a weighted graph, treating the fraction of genes that overlap between any sets as edge weights and proteins as nodes, weighted by the absolute value of the z-score. Then, we used the infomap algorithm (Rosvall and Bergstrom, 2008), to cluster the network. We evaluated the centrality of each node to a given cluster using the PageRank algorithm with a damping factor of 0.5 and employing the same edge and node weights (for personalized PageRank) as we did with clustering (Brin and Page, 1998).

Gene set enrichment and network analysis

We used the STRING enrichment detection tool to identify significantly enriched gene sets, using African green monkey gene symbols, but testing for enrichment across human gene sets. We analyzed sets from all available sources provided by that tool, including sets of clusters of protein-protein interactors in STRING, and excluding the PubMed gene sets. We then generated a network of enriched gene sets by drawing edges between sets with a significant overlap between genes. We evaluated the significance of overlap using Fisher’s exact test. We clustered the network using a weighted graph, treating the fraction of genes that overlap between any sets as edge weights and proteins as nodes, weighted by the absolute value of the z-score. Then, we used the infomap algorithm (Rosvall and Bergstrom, 2008), to cluster the network. We evaluated the centrality of each node to a given cluster using the PageRank algorithm with a damping factor of 0.5 and employing the same edge and node weights (for personalized PageRank) as we did with clustering (Brin and Page, 1998).

Arrayed secondary assessment of CRISPR screen hits by cell viabilityHMGB1 sgRNAs were cloned into lentiCRISPRv2 (Addgene 52961) which also encodes the Cas9 gene (Sanjana et al., 2014). sgRNAs for all other genes were cloned into lentiGuide-Puro or a variant thereof, pXPR_050 (Addgene 52963, 96925, respectively). Individual sgRNAs target sequences are in Table S6. Vero-E6-Cas9-v2 cells were individually transduced with lentiviruses expressing one to three unique sgRNA per gene and then selected with puromycin for 7 days. After selection, 1.25 × 103 cells were seeded in each well of a 384-well black walled clear bottom plate in 20 μl of DMEM + 5% FBS. The following day, 5 μl of SARS-CoV-2 was added for a final MOI of 0.2. Cells were incubated for three days before assessing cellular viability by CellTiter Glo (Promega). For each cell line, viability was determined in SARS-CoV-2 infected relative to mock infected cells. Five replicates per condition were performed in each of three independent experiments.

Arrayed secondary assessment of CRISPR screen hits by cell viability

HMGB1 sgRNAs were cloned into lentiCRISPRv2 (Addgene 52961) which also encodes the Cas9 gene (Sanjana et al., 2014). sgRNAs for all other genes were cloned into lentiGuide-Puro or a variant thereof, pXPR_050 (Addgene 52963, 96925, respectively). Individual sgRNAs target sequences are in Table S6. Vero-E6-Cas9-v2 cells were individually transduced with lentiviruses expressing one to three unique sgRNA per gene and then selected with puromycin for 7 days. After selection, 1.25 × 103 cells were seeded in each well of a 384-well black walled clear bottom plate in 20 μl of DMEM + 5% FBS. The following day, 5 μl of SARS-CoV-2 was added for a final MOI of 0.2. Cells were incubated for three days before assessing cellular viability by CellTiter Glo (Promega). For each cell line, viability was determined in SARS-CoV-2 infected relative to mock infected cells. Five replicates per condition were performed in each of three independent experiments.

SARS-CoV-2 fluorescent reporter virus assayVero-E6 cells were plated at 2.5 × 103 cells per well in a 384-well plate and then the following day, icSARS-CoV-2-mNG was added at a MOI of 1.0 (Xie et al., 2020). Infected cell frequencies as measured by mNeonGreen expression were assessed at 2 dpi by high content imaging (Cytation 5, BioTek) configured with bright field and GFP cubes. Total cell numbers were quantified by Gen5 software of brightfield images. Object analysis was used to determine the number of mNeonGreen positive cells. The percentage of infection was calculated as the ratio between the number of mNeonGreen+ cells and the total number of cells in brightfield. Data are normalized to the average of DMSO treated cells.

SARS-CoV-2 fluorescent reporter virus assay

Vero-E6 cells were plated at 2.5 × 103 cells per well in a 384-well plate and then the following day, icSARS-CoV-2-mNG was added at a MOI of 1.0 (Xie et al., 2020). Infected cell frequencies as measured by mNeonGreen expression were assessed at 2 dpi by high content imaging (Cytation 5, BioTek) configured with bright field and GFP cubes. Total cell numbers were quantified by Gen5 software of brightfield images. Object analysis was used to determine the number of mNeonGreen positive cells. The percentage of infection was calculated as the ratio between the number of mNeonGreen+ cells and the total number of cells in brightfield. Data are normalized to the average of DMSO treated cells.

Identification of anti-viral drugs targeting CRISPR gene hitsCalpain Inhibitor III (#14283), SIS3 (#15495), and PFI-3 (#15267) were purchased from Cayman Chemical. Drugs were resuspended at a stock concentration of 40 mM in DMSO and then two-fold serial dilutions were performed in DMSO. 20 nL of 1000X drug stock were spotted into each well of a 384-well plate using Labcyte ECHO acoustic dispenser at the Yale Center for Molecular Discovery. 1.25 × 103 Vero-E6 cells were plated per well in 20 μl of phenol-red free DMEM containing 5% FBS. Two days later, 5,000 PFU (MOI ∼1) icSARS-CoV-2-mNG in 5 μl media was added. Cells were incubated at 37°C and 5% CO2 for two days. Infected cell frequencies were quantified by mNeonGreen at 2 dpi (Cytation 5, BioTek). In parallel, 1000 PFU (MOI∼0.2) SARS-CoV-2 in 5 μl media was added to replicate plates and cell viability was quantified by CellTiter Glo at 3 dpi. Vero-E6, Huh7.5 and Calu-3 cells were pretreated with 10 μM SIS3 and 40 μM PFI-3 for 48 hours and then infected with SARS-CoV-2 at a MOI of 0.1. Viral production was determined by plaque assay. Cytotoxicity was not observed in these cell lines during the time and concentration of drug used.

Identification of anti-viral drugs targeting CRISPR gene hits

Calpain Inhibitor III (#14283), SIS3 (#15495), and PFI-3 (#15267) were purchased from Cayman Chemical. Drugs were resuspended at a stock concentration of 40 mM in DMSO and then two-fold serial dilutions were performed in DMSO. 20 nL of 1000X drug stock were spotted into each well of a 384-well plate using Labcyte ECHO acoustic dispenser at the Yale Center for Molecular Discovery. 1.25 × 103 Vero-E6 cells were plated per well in 20 μl of phenol-red free DMEM containing 5% FBS. Two days later, 5,000 PFU (MOI ∼1) icSARS-CoV-2-mNG in 5 μl media was added. Cells were incubated at 37°C and 5% CO2 for two days. Infected cell frequencies were quantified by mNeonGreen at 2 dpi (Cytation 5, BioTek). In parallel, 1000 PFU (MOI∼0.2) SARS-CoV-2 in 5 μl media was added to replicate plates and cell viability was quantified by CellTiter Glo at 3 dpi. Vero-E6, Huh7.5 and Calu-3 cells were pretreated with 10 μM SIS3 and 40 μM PFI-3 for 48 hours and then infected with SARS-CoV-2 at a MOI of 0.1. Viral production was determined by plaque assay. Cytotoxicity was not observed in these cell lines during the time and concentration of drug used.

RNA-seqTotal cellular RNA was extracted using Direct-zol RNA MiniPrep Kit and submitted to the Yale Center for Genome Analysis for library preparation. RNA-seq libraries were sequenced on an Illumina NovaSeq 6000 instrument with the goal of at least 25 × 106 reads per sample. Reads were aligned to reference genome chlSab2, NCBI annotation release 100, using STAR aligner v2.7.3a (Dobin et al., 2013) with parameters–winAnchorMultimapNmax 200–outFilterMultimapNmax 100–quantMode GeneCounts. Differential expression was obtained using the R package DESeq2 v1.32 (Dobin et al., 2013; Love et al., 2014). Bigwig files were generated using deeptools v3.1.3 (Ramírez et al., 2016) with parameter–normalizeUsing RPKM.

Total cellular RNA was extracted using Direct-zol RNA MiniPrep Kit and submitted to the Yale Center for Genome Analysis for library preparation. RNA-seq libraries were sequenced on an Illumina NovaSeq 6000 instrument with the goal of at least 25 × 106 reads per sample. Reads were aligned to reference genome chlSab2, NCBI annotation release 100, using STAR aligner v2.7.3a (Dobin et al., 2013) with parameters–winAnchorMultimapNmax 200–outFilterMultimapNmax 100–quantMode GeneCounts. Differential expression was obtained using the R package DESeq2 v1.32 (Dobin et al., 2013; Love et al., 2014). Bigwig files were generated using deeptools v3.1.3 (Ramírez et al., 2016) with parameter–normalizeUsing RPKM.

ChIP-seqAll ChIP samples were prepared in duplicate. Approximately 20 × 106 Vero-E6 cells were used per immunoprecipitation. Cells were washed twice with PBS and crosslinked with 1% formaldehyde (Pierce) for 10 min at room temperature. Crosslinking was quenched with 125 mM (final) glycine for 5 min and washed 2 × with PBS. Cells were lysed in 4 ml lysis buffer (50 mM HEPES pH 7.9, 140 mM NaCl, 1 mM EDTA, 10% glycerol, 0.5% NP-40, 0.25% Triton X-100), 1 × Protease inhibitor (Roche) for 10 min on ice. Lysed cells were centrifuged at 4,000 rpm for 5 min at 4 °C and washed 2 × with 4 ml cold wash buffer (10 mM Tris-Cl pH 7.5, 200 mM NaCl, 1 mM EDTA pH 8.0, 0.5 mM EGTA pH 8.0), 1 × protease inhibitors. The pellet was resuspended in 1 ml shearing buffer (0.1% SDS, 1 mM EDTA, 10 mM Tris pH 7.5), 1 × protease inhibitor and sheared on Covaris S220 (140 W, 5% duty factor, 200 bursts per cycle, 4 °C) for 12 min (determined by time course optimization experiment). Extract was diluted with Triton X-100 (1% final) and NaCl (150 mM final) and cleared by centrifugation at 21,000g, 4 °C for 10 min. For input, 10% of material was set aside. Cleared extract was supplemented with 2 μg of antibody and incubated overnight at 4 °C. Immuno-precipitated chromatin was captured on 30 μl of Dynabeads protein G (Invitrogen) at 4 °C for 1.5 h. Beads were washed twice with low-salt buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM HEPES–potassium hydroxide pH 7.5, 150 mM NaCl), 2 × with high-salt buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM HEPES–potassium hydroxide pH 7.5, 500 mM NaCl), 1 × LiCl Buffer (100 mM Tris-HCl pH 7.5, 0.5 M LiCl, 1% NP-40, 1% sodium deoxycholate) and 1 × TE buffer (10 mM Tris-HCl pH 8.0, 0.1 mM EDTA). Enriched DNA was eluted in 100 μl of proteinase K buffer (20mM HEPES pH 7.5, 1 mM EDTA, 0.5% SDS) supplemented with 40 μg of proteinase K (Ambion) for 30 min at 50 °C. Formaldehyde crosslinks were reversed by adding NaCl (150 mM final) and 0.25 μg DNase-free RNase (Roche), followed by incubation overnight at 65 °C. DNA was isolated using QIAquick PCR purification kit (QIAGEN) and submitted to the Yale Center for Genome Analysis for library preparation. ChIP-seq libraries were sequenced on a Illumina NovaSeq 6000 instrument as 101 nt long paired-end reads, with the goal of at least 20 × 106 reads per IP. Reads were trimmed of adaptor sequences using Cutadapt (Martin, 2011) and aligned to the reference genome chlSab2 using Bowtie2 (Langmead and Salzberg, 2012). Alignments were filtered using SAMtools (Li et al., 2009), and peak calls and enrichment tracks were created using MACS2 (Zhang et al., 2008). Differential analysis at called peaks was performed using DESeq2 (Love et al., 2014). Peaks were assigned to the nearest transcription start site within 100kb for integration with RNA-seq data and overlaps of ChIP-seq and ATAC-seq peaks were determined using bedtools.

All ChIP samples were prepared in duplicate. Approximately 20 × 106 Vero-E6 cells were used per immunoprecipitation. Cells were washed twice with PBS and crosslinked with 1% formaldehyde (Pierce) for 10 min at room temperature. Crosslinking was quenched with 125 mM (final) glycine for 5 min and washed 2 × with PBS. Cells were lysed in 4 ml lysis buffer (50 mM HEPES pH 7.9, 140 mM NaCl, 1 mM EDTA, 10% glycerol, 0.5% NP-40, 0.25% Triton X-100), 1 × Protease inhibitor (Roche) for 10 min on ice. Lysed cells were centrifuged at 4,000 rpm for 5 min at 4 °C and washed 2 × with 4 ml cold wash buffer (10 mM Tris-Cl pH 7.5, 200 mM NaCl, 1 mM EDTA pH 8.0, 0.5 mM EGTA pH 8.0), 1 × protease inhibitors. The pellet was resuspended in 1 ml shearing buffer (0.1% SDS, 1 mM EDTA, 10 mM Tris pH 7.5), 1 × protease inhibitor and sheared on Covaris S220 (140 W, 5% duty factor, 200 bursts per cycle, 4 °C) for 12 min (determined by time course optimization experiment). Extract was diluted with Triton X-100 (1% final) and NaCl (150 mM final) and cleared by centrifugation at 21,000g, 4 °C for 10 min. For input, 10% of material was set aside. Cleared extract was supplemented with 2 μg of antibody and incubated overnight at 4 °C. Immuno-precipitated chromatin was captured on 30 μl of Dynabeads protein G (Invitrogen) at 4 °C for 1.5 h. Beads were washed twice with low-salt buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM HEPES–potassium hydroxide pH 7.5, 150 mM NaCl), 2 × with high-salt buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM HEPES–potassium hydroxide pH 7.5, 500 mM NaCl), 1 × LiCl Buffer (100 mM Tris-HCl pH 7.5, 0.5 M LiCl, 1% NP-40, 1% sodium deoxycholate) and 1 × TE buffer (10 mM Tris-HCl pH 8.0, 0.1 mM EDTA). Enriched DNA was eluted in 100 μl of proteinase K buffer (20mM HEPES pH 7.5, 1 mM EDTA, 0.5% SDS) supplemented with 40 μg of proteinase K (Ambion) for 30 min at 50 °C. Formaldehyde crosslinks were reversed by adding NaCl (150 mM final) and 0.25 μg DNase-free RNase (Roche), followed by incubation overnight at 65 °C. DNA was isolated using QIAquick PCR purification kit (QIAGEN) and submitted to the Yale Center for Genome Analysis for library preparation. ChIP-seq libraries were sequenced on a Illumina NovaSeq 6000 instrument as 101 nt long paired-end reads, with the goal of at least 20 × 106 reads per IP. Reads were trimmed of adaptor sequences using Cutadapt (Martin, 2011) and aligned to the reference genome chlSab2 using Bowtie2 (Langmead and Salzberg, 2012). Alignments were filtered using SAMtools (Li et al., 2009), and peak calls and enrichment tracks were created using MACS2 (Zhang et al., 2008). Differential analysis at called peaks was performed using DESeq2 (Love et al., 2014). Peaks were assigned to the nearest transcription start site within 100kb for integration with RNA-seq data and overlaps of ChIP-seq and ATAC-seq peaks were determined using bedtools.

ATAC-seqATAC-seq libraries were generated following the omni-ATAC protocol as described (Corces et al., 2017). Two biological repeats were generated per each sample. 50,000 viable cells were resuspended in 50 μl cold ATAC-Resuspension Buffer (RSB) (10mM Tris-HCl, pH7.4, 10mM NaCl, and 3mM MgCl2) containing 0.1% NP40, 0.1% Tween-20, and 0.01% digitonin and incubated on ice for 3 minutes. 1ml of cold ATAC-RSB containing 0.1% Tween-20 was then added to wash out the lysis. Samples were centrifuged at 500 RCF for 10 minutes at 4°C. The nuclei pellets were resuspended in 50 μl transposase reaction mix (25 μl 2x TD buffer, 2.5 μl transposase, 16.5 μl PBS, 0.5 μl 1% digitonin, 0.5 μl 10% Tween-20, and 5 μl H2O) and incubated at 37°C for 30 minutes in a thermomixer with 1000 RPM mixing. Reactions were cleaned up with Zymo DNA Clean and Concentrator-5 kit; Transposed DNA sample was eluted in a 20 μl elution buffer. Transposed samples were pre-amplified for 5 cycles using NEBNext 2x Master Mix in 50 μl reaction mix (2.5 μl of 25 μM primer Ad1, 2.5 μl of 25 μM primer Ad2, 25 μl of 2x Master Mix and 20 μl transposed elution) at cycling conditions as following: 72°C, 5minutes; 98°C, 30 s; then 5 cycles of (98°C, 10 s; 63°C, 30 s; 72°C, 1 minute). 15 μl of qPCR amplification reaction (5 μl of pre-amplified sample; 0.5 μl of 25 μM primer Ad1, 0.5 μl of 25 μM primer Ad2, 5 μl of 2x NEBNext Master Mix, 0.24 μl of 25x SYBR Green in DMSO, and 3.76 μl of H2O) was carried out at cycling conditions as following: 98°C, 30 s; then 20 cycles of (98°C, 10 s; 63°C, 30 s; 72°C, 1 minute. The required number of additional cycles for each sample was determined as described (Buenrostro et al., 2015). After the final amplification, PCR reaction was purified using a Zymo DNA Clean and Concentrator-5 kit and eluted in 20 μl H2O. To remove primer dimers and larger than 1,000 bp fragments, double-sided bead purification was proceeded with Ampure XP beads. 0.5x volume AMPure XP beads were added to each reaction and incubated at room temperature for 10 minutes and then separated in a magnetic rack for 5 minutes. Supernatant was transferred to a new tube and incubated with 1.3x original volume AMPure XP beads at room temperature for 10 minutes and then separated in a magnetic rack for 5 minutes. Supernatant was discarded. Beads were washed twice with 200 μl 80% ethanol (freshly made) and air-dried to ensure all ethanol was removed. Final ATAC-seq libraries were eluted in 20 μl nuclease-free H2O from the beads. ATAC-seq libraries were sequenced on an Illumina NovaSeq S4 instrument as 101 nt long paired-end reads, with the goal of at least 50 × 106 reads per replicate. Reads were trimmed of Nextera adaptor sequences using Trimmomatic v0.39 (Bolger et al., 2014) and aligned to chlSab2 using Bowtie2 v2.2.9 (Langmead and Salzberg, 2012) with parameter -X2000. Duplicates were marked using Picard Tools v2.9.0 (Broad Institute. version 2.9.0. “Picard Tools.” Broad Institute, GitHub repository. http://broadinstitute.github.io/picard/). Duplicated, unpaired, and mitochondrial reads were removed using SAMTools v1.9 (Li et al., 2009). Reads were shifted +4 bp and −5 bp for forward and reverse strands, respectively. Peaks were called using MACS2 v2.2.6 (Zhang et al., 2008) with parameters–nomodel–keep-dup all -s 1–shift −75–extsize 150. Reads that fell inside peaks were counted using featureCounts v1.6.2 (Liao et al., 2014) and differential accessibility analysis was performed using DESeq2 v1.32 (Love et al., 2014). Bigwig files were generated using deeptools v3.1.3 (Ramírez et al., 2016) with parameter–normalizeUsing RPKM.

ATAC-seq libraries were generated following the omni-ATAC protocol as described (Corces et al., 2017). Two biological repeats were generated per each sample. 50,000 viable cells were resuspended in 50 μl cold ATAC-Resuspension Buffer (RSB) (10mM Tris-HCl, pH7.4, 10mM NaCl, and 3mM MgCl2) containing 0.1% NP40, 0.1% Tween-20, and 0.01% digitonin and incubated on ice for 3 minutes. 1ml of cold ATAC-RSB containing 0.1% Tween-20 was then added to wash out the lysis. Samples were centrifuged at 500 RCF for 10 minutes at 4°C. The nuclei pellets were resuspended in 50 μl transposase reaction mix (25 μl 2x TD buffer, 2.5 μl transposase, 16.5 μl PBS, 0.5 μl 1% digitonin, 0.5 μl 10% Tween-20, and 5 μl H2O) and incubated at 37°C for 30 minutes in a thermomixer with 1000 RPM mixing. Reactions were cleaned up with Zymo DNA Clean and Concentrator-5 kit; Transposed DNA sample was eluted in a 20 μl elution buffer. Transposed samples were pre-amplified for 5 cycles using NEBNext 2x Master Mix in 50 μl reaction mix (2.5 μl of 25 μM primer Ad1, 2.5 μl of 25 μM primer Ad2, 25 μl of 2x Master Mix and 20 μl transposed elution) at cycling conditions as following: 72°C, 5minutes; 98°C, 30 s; then 5 cycles of (98°C, 10 s; 63°C, 30 s; 72°C, 1 minute). 15 μl of qPCR amplification reaction (5 μl of pre-amplified sample; 0.5 μl of 25 μM primer Ad1, 0.5 μl of 25 μM primer Ad2, 5 μl of 2x NEBNext Master Mix, 0.24 μl of 25x SYBR Green in DMSO, and 3.76 μl of H2O) was carried out at cycling conditions as following: 98°C, 30 s; then 20 cycles of (98°C, 10 s; 63°C, 30 s; 72°C, 1 minute. The required number of additional cycles for each sample was determined as described (Buenrostro et al., 2015). After the final amplification, PCR reaction was purified using a Zymo DNA Clean and Concentrator-5 kit and eluted in 20 μl H2O. To remove primer dimers and larger than 1,000 bp fragments, double-sided bead purification was proceeded with Ampure XP beads. 0.5x volume AMPure XP beads were added to each reaction and incubated at room temperature for 10 minutes and then separated in a magnetic rack for 5 minutes. Supernatant was transferred to a new tube and incubated with 1.3x original volume AMPure XP beads at room temperature for 10 minutes and then separated in a magnetic rack for 5 minutes. Supernatant was discarded. Beads were washed twice with 200 μl 80% ethanol (freshly made) and air-dried to ensure all ethanol was removed. Final ATAC-seq libraries were eluted in 20 μl nuclease-free H2O from the beads. ATAC-seq libraries were sequenced on an Illumina NovaSeq S4 instrument as 101 nt long paired-end reads, with the goal of at least 50 × 106 reads per replicate. Reads were trimmed of Nextera adaptor sequences using Trimmomatic v0.39 (Bolger et al., 2014) and aligned to chlSab2 using Bowtie2 v2.2.9 (Langmead and Salzberg, 2012) with parameter -X2000. Duplicates were marked using Picard Tools v2.9.0 (Broad Institute. version 2.9.0. “Picard Tools.” Broad Institute, GitHub repository. http://broadinstitute.github.io/picard/). Duplicated, unpaired, and mitochondrial reads were removed using SAMTools v1.9 (Li et al., 2009). Reads were shifted +4 bp and −5 bp for forward and reverse strands, respectively. Peaks were called using MACS2 v2.2.6 (Zhang et al., 2008) with parameters–nomodel–keep-dup all -s 1–shift −75–extsize 150. Reads that fell inside peaks were counted using featureCounts v1.6.2 (Liao et al., 2014) and differential accessibility analysis was performed using DESeq2 v1.32 (Love et al., 2014). Bigwig files were generated using deeptools v3.1.3 (Ramírez et al., 2016) with parameter–normalizeUsing RPKM.

RT-qPCRTotal RNA was isolated from cells using Direct-zol RNA MiniPrep Plus kit, reverse transcribed, and subjected to real-time PCR analysis to measure mRNA levels of tested genes. Data shown are the relative abundance of the indicated mRNA normalized to that of actin. Gene-specific primer sequences were as follows ACE2: GGGATCAGAGATCGGAAGAAGA (forward) and AAGGAGGTCTGAACATCATCAGTG (reverse); Actin: GAGCACAGAGCCTCGCCTTT (forward) and ATCATCATCCATGGTGAGCTGG (reverse).

Total RNA was isolated from cells using Direct-zol RNA MiniPrep Plus kit, reverse transcribed, and subjected to real-time PCR analysis to measure mRNA levels of tested genes. Data shown are the relative abundance of the indicated mRNA normalized to that of actin. Gene-specific primer sequences were as follows ACE2: GGGATCAGAGATCGGAAGAAGA (forward) and AAGGAGGTCTGAACATCATCAGTG (reverse); Actin: GAGCACAGAGCCTCGCCTTT (forward) and ATCATCATCCATGGTGAGCTGG (reverse).

Nuclear/Cytosol FractionationVero-E6 cells were mock-infected or infected with SARS-CoV-2 at a MOI of 1.0 for 24 hours before cellular fractionation was performed using a Nuclear/Cytosol fractionation kit (BioVision Cat#K266-25) according to manufacturer instructions. In brief, 1 × 107 cells were collected by centrifugation at 600 x g for 5 min at 4°C. Add 0.2 mL Cytosol Extraction Buffer A (CEB-A) to fully resuspend the cell pellet. After 10 min incubation on ice, add 11 μl of ice-cold CEB-B and incubate on ice for 1min. Centrifuge at 16000 x g for 5 min, the supernatant was collected as cytoplasmic fraction. The pellet was resuspended in 100 μl of ice-cold Nuclear Extraction Buffer (NEB) vertex every 10 min for a total of 40 min. The samples were centrifuge at 16000 x g for 10 min, and the supernatant was collected as the nuclear fraction.

Nuclear/Cytosol Fractionation

Vero-E6 cells were mock-infected or infected with SARS-CoV-2 at a MOI of 1.0 for 24 hours before cellular fractionation was performed using a Nuclear/Cytosol fractionation kit (BioVision Cat#K266-25) according to manufacturer instructions. In brief, 1 × 107 cells were collected by centrifugation at 600 x g for 5 min at 4°C. Add 0.2 mL Cytosol Extraction Buffer A (CEB-A) to fully resuspend the cell pellet. After 10 min incubation on ice, add 11 μl of ice-cold CEB-B and incubate on ice for 1min. Centrifuge at 16000 x g for 5 min, the supernatant was collected as cytoplasmic fraction. The pellet was resuspended in 100 μl of ice-cold Nuclear Extraction Buffer (NEB) vertex every 10 min for a total of 40 min. The samples were centrifuge at 16000 x g for 10 min, and the supernatant was collected as the nuclear fraction.

Western blotCells were collected and lysed in Nonidet P-40 lysis buffer (20 mM Tris-HCl [pH 7.4], 150 mM NaCl, 1 mM EDTA, 1% Nonidet P-40, 10 mg/ml aprotinin, 10 mg/ml leupeptin, and 1 mM PMSF). The cell lysates or cellular fractions were fractionated on SDS-PAGE and transferred to a PVDF membrane. Immunoblotting analyses were performed with the indicated antibodies and visualized with horseradish peroxidase-coupled goat anti-mouse/rabbit IgG using a chemiluminescence detection system (BioRad ChemiDoc MP).

Western blot

Cells were collected and lysed in Nonidet P-40 lysis buffer (20 mM Tris-HCl [pH 7.4], 150 mM NaCl, 1 mM EDTA, 1% Nonidet P-40, 10 mg/ml aprotinin, 10 mg/ml leupeptin, and 1 mM PMSF). The cell lysates or cellular fractions were fractionated on SDS-PAGE and transferred to a PVDF membrane. Immunoblotting analyses were performed with the indicated antibodies and visualized with horseradish peroxidase-coupled goat anti-mouse/rabbit IgG using a chemiluminescence detection system (BioRad ChemiDoc MP).

Generation of HMGB1 knockout and complemented cellsVero-E6 cells were individually transduced with lentiviruses expressing two guide RNAs targeting HMGB1 (Table S6) and then selected with puromycin for 7 days. Single cells were then sorted by flow cytometry and HMGB1 knockout was confirmed by western blot. HMGB1 KO clones were complemented by lentiviral transduction of pLenti6/V5-DEST vector containing human HMGB1 with a C-terminal V5. Two days post transduction, blasticidin was added and cells were selected for five days. The expression of HMGB1 in complemented cells was detected by western blot.

Generation of HMGB1 knockout and complemented cells

Vero-E6 cells were individually transduced with lentiviruses expressing two guide RNAs targeting HMGB1 (Table S6) and then selected with puromycin for 7 days. Single cells were then sorted by flow cytometry and HMGB1 knockout was confirmed by western blot. HMGB1 KO clones were complemented by lentiviral transduction of pLenti6/V5-DEST vector containing human HMGB1 with a C-terminal V5. Two days post transduction, blasticidin was added and cells were selected for five days. The expression of HMGB1 in complemented cells was detected by western blot.

Pseudovirus productionVSV-based pseudovirus particles (VSVpp) were produced in 293T cells. Cells were transfected with pCAGGS or pcDNA3.1 vector expressing the CoV spike glycoprotein and then inoculated with a replication-deficient VSV virus that contains expression cassettes for Renilla luciferase instead of the vesicular stomatitis virus G (VSV-G) open reading frame (Avanzato, 2019). After an incubation period of 1 h at 37°C, the inoculum was removed and cells were washed with PBS before media supplemented with anti-VSV-G clone IE9F9 was added in order to neutralize residual input virus (no antibody was added to cells expressing VSV-G) (Lefrancois and Lyles, 1982). Pseudotyped particles were harvested 24 hours post inoculation, clarified from cellular debris by centrifugation and stored at −80°C before use.Plasmids encoding codon-optimized form of SARS-CoV-1-S glycoprotein, MERS-CoV SΔCT and NL63 SΔCT glycoproteins lacking cytoplasmic tail were previously described (Huang et al., 2006; Letko et al., 2020). Vector pCAGGS containing the SARS-CoV-2, Wuhan-Hu-1 Spike Glycoprotein Gene, NR-52310, was produced under HHSN272201400008C and obtained through BEI Resources, NIAID, NIH.

Pseudovirus production

VSV-based pseudovirus particles (VSVpp) were produced in 293T cells. Cells were transfected with pCAGGS or pcDNA3.1 vector expressing the CoV spike glycoprotein and then inoculated with a replication-deficient VSV virus that contains expression cassettes for Renilla luciferase instead of the vesicular stomatitis virus G (VSV-G) open reading frame (Avanzato, 2019). After an incubation period of 1 h at 37°C, the inoculum was removed and cells were washed with PBS before media supplemented with anti-VSV-G clone IE9F9 was added in order to neutralize residual input virus (no antibody was added to cells expressing VSV-G) (Lefrancois and Lyles, 1982). Pseudotyped particles were harvested 24 hours post inoculation, clarified from cellular debris by centrifugation and stored at −80°C before use.

Plasmids encoding codon-optimized form of SARS-CoV-1-S glycoprotein, MERS-CoV SΔCT and NL63 SΔCT glycoproteins lacking cytoplasmic tail were previously described (Huang et al., 2006; Letko et al., 2020). Vector pCAGGS containing the SARS-CoV-2, Wuhan-Hu-1 Spike Glycoprotein Gene, NR-52310, was produced under HHSN272201400008C and obtained through BEI Resources, NIAID, NIH.

Pseudovirus entry assay1x104 Vero-E6 cells were seeded in 100 μl total volume in each well of a black-walled clear bottom 96-well plate. The following day spike expressing VSV pseudovirus was added at 1:10 final concentration volume/volume and incubated for one day. Cells were lysed with Renilla Luciferase Assay System (Promega) according to manufacturer instructions. Luciferase activity was measured using a microplate reader (BioTek Synergy or Cytation 5).

Pseudovirus entry assay

1x104 Vero-E6 cells were seeded in 100 μl total volume in each well of a black-walled clear bottom 96-well plate. The following day spike expressing VSV pseudovirus was added at 1:10 final concentration volume/volume and incubated for one day. Cells were lysed with Renilla Luciferase Assay System (Promega) according to manufacturer instructions. Luciferase activity was measured using a microplate reader (BioTek Synergy or Cytation 5).

Quantification and Statistical AnalysisStatistical significance was determined as p < 0.05 using GraphPad Prism 8 unless otherwise indicated. Experiments were analyzed by unpaired two-tailed t tests, Mann-Whitney test, or ANOVA, as indicated. Schematic were created with biorender.com.

Quantification and Statistical Analysis

Statistical significance was determined as p < 0.05 using GraphPad Prism 8 unless otherwise indicated. Experiments were analyzed by unpaired two-tailed t tests, Mann-Whitney test, or ANOVA, as indicated. Schematic were created with biorender.com.
