---
pmid: '35101990'
title: The cardiac-enriched microprotein mitolamban regulates mitochondrial respiratory
  complex assembly and function in mice.
authors:
- Makarewich CA
- Munir AZ
- Bezprozvannaya S
- Gibson AM
- Young Kim S
- Martin-Sandoval MS
- Mathews TP
- Szweda LI
- Bassel-Duby R
- Olson EN
journal: Proc Natl Acad Sci U S A
year: '2022'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC8833175
doi: 10.1073/pnas.2120476119
---

# The cardiac-enriched microprotein mitolamban regulates mitochondrial respiratory complex assembly and function in mice.
**Authors:** Makarewich CA, Munir AZ, Bezprozvannaya S, Gibson AM, Young Kim S, Martin-Sandoval MS, Mathews TP, Szweda LI, Bassel-Duby R, Olson EN
**Journal:** Proc Natl Acad Sci U S A (2022)
**DOI:** [10.1073/pnas.2120476119](https://doi.org/10.1073/pnas.2120476119)
**PMC:** [PMC8833175](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8833175/)

## Abstract

1. Proc Natl Acad Sci U S A. 2022 Feb 8;119(6):e2120476119. doi: 
10.1073/pnas.2120476119.

The cardiac-enriched microprotein mitolamban regulates mitochondrial respiratory 
complex assembly and function in mice.

Makarewich CA(1)(2), Munir AZ(3), Bezprozvannaya S(3), Gibson AM(2), Young Kim 
S(4), Martin-Sandoval MS(5), Mathews TP(5)(6), Szweda LI(4), Bassel-Duby R(3), 
Olson EN(7).

Author information:
(1)Department of Pediatrics, University of Cincinnati College of Medicine, 
Cincinnati, OH 45229; cat.makarewich@cchmc.org eric.olson@utsouthwestern.edu.
(2)The Heart Institute, Division of Molecular Cardiovascular Biology, Cincinnati 
Children's Hospital Medical Center, Cincinnati, OH 45229.
(3)Department of Molecular Biology, Hamon Center for Regenerative Science and 
Medicine, University of Texas Southwestern Medical Center, Dallas, TX 75390.
(4)Division of Cardiology, Department of Internal Medicine, University of Texas 
Southwestern Medical Center, Dallas, TX 75390.
(5)Children's Research Institute, University of Texas Southwestern Medical 
Center, Dallas, TX 75390.
(6)Department of Pediatrics, University of Texas Southwestern Medical Center, 
Dallas, TX 75390.
(7)Department of Molecular Biology, Hamon Center for Regenerative Science and 
Medicine, University of Texas Southwestern Medical Center, Dallas, TX 75390; 
cat.makarewich@cchmc.org eric.olson@utsouthwestern.edu.

Emerging evidence indicates that a subset of RNA molecules annotated as 
noncoding contain short open reading frames that code for small functional 
proteins called microproteins, which have largely been overlooked due to their 
small size. To search for cardiac-expressed microproteins, we used a comparative 
genomics approach and identified mitolamban (Mtlbn) as a highly conserved 
47-amino acid transmembrane protein that is abundantly expressed in the heart. 
Mtlbn localizes specifically to the inner mitochondrial membrane where it 
interacts with subunits of complex III of the electron transport chain and with 
mitochondrial respiratory supercomplexes. Genetic deletion of Mtlbn in mice 
altered complex III assembly dynamics and reduced complex III activity. Unbiased 
metabolomic analysis of heart tissue from Mtlbn knockout mice further revealed 
an altered metabolite profile consistent with deficiencies in complex III 
activity. Cardiac-specific Mtlbn overexpression in transgenic (TG) mice induced 
cardiomyopathy with histological, biochemical, and ultrastructural pathologic 
features that contributed to premature death. Metabolomic analysis and 
biochemical studies indicated that hearts from Mtlbn TG mice exhibited increased 
oxidative stress and mitochondrial dysfunction. These findings reveal Mtlbn as a 
cardiac-expressed inner mitochondrial membrane microprotein that contributes to 
mitochondrial electron transport chain activity through direct association with 
complex III and the regulation of its assembly and function.

Copyright © 2022 the Author(s). Published by PNAS.

DOI: 10.1073/pnas.2120476119
PMCID: PMC8833175
PMID: 35101990 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing interest.

## Full Text

Recent advances in computational biology and transcriptomics have revealed that a much larger portion of the genome is transcribed than was previously recognized, resulting in the production of a diverse population of RNA transcripts with both protein-coding and noncoding potential (1). Ongoing efforts to understand the biological roles of presumed noncoding transcripts have revealed that a growing number contain hidden short open reading frames (sORFs) that are translated to generate small functional proteins called microproteins, or micropeptides (2, 3). Microproteins play important roles in fundamental biological processes, including development (4, 5), DNA repair (6), calcium homeostasis (7–11), metabolism (12–14), myoblast fusion (15–17), stress signaling (18), and cell death (19, 20). Given their small size and distinct structures, microproteins may have evolved as singular protein domains designed to fine tune complex biological systems, and as such, may be uniquely suited to serve as potential therapeutic targets.

To identify cardiac-expressed microproteins, we analyzed annotated noncoding RNAs that are highly expressed in the heart using a comparative genomics approach to look for regions of sequence conservation and protein coding potential. Using this method, we identified an evolutionarily conserved sORF in the murine noncoding RNA 1810058I24Rik (human C7orf73) corresponding to a 47-amino acid protein, which we named mitolamban (Mtlbn), in light of another well-known cardiac microprotein, phospholamban. Interestingly, a recent publication identified this “noncoding” RNA as one of the most highly up-regulated transcripts in the heart in response to an experimental mouse model of cardiac pressure overload (thoracic aortic constriction [TAC]), and mapped it as a nodal hub in a disease module enriched for oxidative phosphorylation genes (21). Analysis of the Mtlbn sequence indicated it contained a predicted transmembrane domain, which is a common feature of many recently identified muscle-enriched microproteins (7, 8, 10, 13–18) that may enhance their stability and protect them from the rapid degradation that they may be otherwise susceptible to as small molecules (22). Transmembrane microproteins have been shown to act as important regulators of much larger membrane protein complexes such as the sarcoendoplasmic reticulum calcium ATPase (SERCA) (7–10, 23), the sodium-potassium (Na+, K+)-ATPase (24), the mitochondrial trifunctional protein (MTP) (13), and mitochondrial respiratory chain and oxidative phosphorylation components (14, 25). For example, phospholamban and related molecules play pivotal roles in the control of calcium cycling and contractility in cardiac and skeletal muscle cells (7–10, 23). Mtlbn is homologous to the zebrafish Stmp1 gene, which is predicted to encode a 46-amino acid microprotein that is ubiquitously expressed and enriched in adult zebrafish brain, and computational analysis of the predicted protein sequence indicates characteristics of a mitochondrial respiratory complex protein (26).

In the present study, we identify Mtlbn as a cardiac-enriched microprotein that localizes specifically to the inner mitochondrial membrane where it associates directly with complex III of the electron transport chain (ETC) and contributes to both its assembly and function. Hearts from Mtlbn loss-of-function mice are characterized by impaired complex III activity and exhibit metabolic alterations that are consistent with defects in complex III and reductions in oxidative phosphorylation capacity. These studies highlight a unique role for Mtlbn in the heart and underscore the importance of elucidating the functions of previously uncharacterized microproteins and expanding our understanding of the cellular proteome.

ResultsMtlbn Is a Heart and Skeletal Muscle–Enriched Microprotein.To search for evolutionarily conserved sORFs that code for cardiac-expressed microproteins, we used a comparative genomics approach to analyze annotated noncoding transcripts that are highly expressed in the heart. Bioinformatic analysis was performed using PhyloCSF, a multispecies nucleotide alignment tool with statistical comparison of phylogenetic codon models to score the genome for protein-coding potential (27). Using this method, we identified a conserved 141-base pair sORF, which codes for a putative 47-amino acid protein, within the highly expressed transcript, annotated as 1810058I24Rik in the mouse genome and C7orf73 in the human genome (Fig. 1 A–C). The murine Mtlbn gene is encoded by three exons (Fig. 1A) and the Mtlbn sORF spans all three of these exons and generates a high PhyloCSF score (Fig. 1B), corresponding to a conserved microprotein containing a predicted N-terminal transmembrane domain and a highly charged C terminus (Fig. 1C). Mass spectrometry of mouse heart extracts confirmed the presence of the putative Mtlbn microprotein and identified two distinct peptide fragments corresponding to unique regions of its sequence (Fig. 1 D and E), indicating it is stably expressed. qRT-PCR analysis of a mouse tissue cDNA library revealed Mtlbn is highly abundant and widely expressed with enrichment in heart and skeletal muscle (Fig. 1F). Consistent with a previous report indicating the up-regulation of 1810058I24Rik in hearts of mice in a heart failure model induced by TAC (21), we also observed a significant up-regulation of the transcript in response to pressure overload (Fig. 1G), indicating it may be involved in the cardiac response to stress.Fig. 1.Mtlbn is a conserved heart and skeletal muscle–enriched microprotein. (A) Schematic of the murine Mtlbn (1810058I24Rik) genomic locus. Mtlbn is located on chromosome 6 and consists of three exons (E1 to E3). The 141-base pair (bp) Mtlbn ORF is highlighted in red. (B) The PhyloCSF score for Mtlbn plotted using the Integrated Genomics Viewer (IGV). The region highlighted in red indicates a positive PhyloCSF score and corresponds with the Mtlbn ORF (negative scoring regions are blue). The three potential reading frames for the Mtlbn transcript are indicated in the positive strand as +1, +2, and +3. (C) Amino acid sequence alignment of Mtlbn shows high conservation across multiple species. The colors of the residues indicate their properties (pink, positive charge; blue, negative charge; red, hydrophobic; and green, polar). The symbols below the alignment represent the biochemical similarity of aligned amino acids (asterisks, identical conservation; colons, high similarity; periods, somewhat similar). The predicted transmembrane domain of Mtlbn is indicated above the sequence. (D and E) MS/MS spectrum of two unique peptide fragments corresponding to distinct regions of the Mtlbn protein identified by mass spectrometry of mouse heart tissue. (F) Detection of Mtlbn RNA by qRT-PCR in an adult mouse tissue cDNA library. Rpl4 was used for normalization, and expression is shown relative to heart for n = 4 technical replicates. Data are mean ± SD. Sk, skeletal; Sm, smooth. (G) Quantification of Mtlbn RNA in mouse heart tissue 8 weeks after sham (control) or TAC analyzed by qRT-PCR. Data are normalized to Rpl4 and shown relative to sham for n = 6 mice. Statistical analysis was performed using a two-tailed Student’s t test.Mtlbn Is an Inner Mitochondrial Membrane Microprotein.To determine the subcellular localization of Mtlbn, we cloned its full-length coding sequence in frame with a C-terminal human influenza hemagglutinin (HA) epitope tag and expressed the Mtlbn–HA fusion protein in C2C12 myoblasts for immunofluorescence labeling. We selected the HA epitope tag (1XHA) for these studies because the HA amino acid sequence is the least disruptive to the isoelectric point and hydrophobicity score of Mtlbn compared to other common tags, including 1XMyc, 1XFLAG, and 3XFLAG (calculated using the Expasy Swiss Bioinformatics Resource Portal) (28, 29). Confocal imaging revealed a mitochondrial distribution pattern for Mtlbn–HA and clear colocalization with the mitochondrial protein TOMM20 (translocase of outer mitochondrial membrane 20) (Fig. 2A). This staining pattern was distinct from the sarcoplasmic reticulum (Fig. 2B), which is another organelle rich in transmembrane microprotein function, indicating that Mtlbn specifically targets to mitochondria. Although Mtlbn does not encode a canonical mitochondrial localization signal, there is evidence that the transmembrane domains of some small mitochondrial membrane proteins serve as mitochondrial targeting and membrane anchoring regions (25). To test whether this is the case for Mtlbn, we cloned the first 22 amino acids of Mtlbn in frame with a C-terminal HA epitope tag (Mtlbn1-22–HA) and expressed this construct in C2C12 myoblasts. Immunofluorescence labeling and confocal microscopy revealed clear colocalization of Mtlbn1-22–HA with TOMM20 (Fig. 2C), indicating that the N-terminal transmembrane domain of Mtlbn is sufficient to target the protein to the mitochondria.Fig. 2.Mtlbn localizes to the inner mitochondrial membrane and interacts with subunits of mitochondrial respiratory chain complex III. (A–C) Immunofluorescence of C2C12 myoblasts expressing full-length Mtlbn (A and B) or a truncated version (Mtlbn1-22) (C) fused in frame to a C-terminal HA tag. HA (Mtlbn) immunolabeling is indicated in green in all images, while the mitochondrial protein TOMM20 (A and C) or the sarcoplasmic reticulum protein SERCA2 (B) are shown in red. Merged panels also include a nuclear DAPI stain (blue). (Scale bar, 5 μm.) (D) Intact mitochondria were isolated from HEK293 cells expressing Mtlbn–HA and subjected to proteinase K (PK) proteolysis at the indicated concentrations. Western blot analysis was performed using antibodies for markers of the OMM (TOMM20), IMM (ATPB), and Mtlbn (HA). (E) The PK proteolysis pattern of mitochondria isolated from HEK293 cells expressing Mtlbn–HA in the presence (+) and absence (−) of detergent (Triton). Intact mitochondria (whole mito) were treated with osmotic shock to isolate mitoplasts containing only the IMM and matrix. Western blots were performed using the same antibodies detailed in D. (F) Silver stain of lysates and HA elutions from HA IPs performed on HA–GFP or Mtlbn–HA expressing HEK293 cells. Arrows indicate the predicted migration of HA–GFP and Mtlbn–HA. (G) Panther gene ontology (GO) enrichment analysis of the proteins pulled down in association with Mtlbn–HA (enriched over HA–GFP control). The top 10 GO categories are depicted. (H) Western blots of lysates (input) and HA-eluted proteins from HA IPs on heart extracts from WT or α-MHC Mtlbn–HA TG mice. UQCRC1, UQCRC2, and UQCR10 are components of complex III. HADHA is a subunit of the mitochondrial trifunctional protein. (I) Western blot analysis of cardiac mitochondria isolated from WT or Mtlbn TG mice separated by BN-PAGE. Antibodies for the following respiratory complexes (CI–CV) were used: CI (NDUFB8), CII (SDHA), CIII (UQCRC1), CIV (COXIV), and CV (ATP5a). Red arrows highlight the identical migration pattern of Mtlbn–HA and complex III.To assess whether Mtlbn localizes to the outer or inner mitochondrial membrane (OMM or IMM), we performed two independent mitochondrial fractionation techniques using purified mitochondria from HEK293 cells expressing Mtlbn–HA. First, purified mitochondria were subjected to progressive proteolysis with serial dilutions of proteinase K (PK) and the products were analyzed by Western blot (Fig. 2D). The OMM protein TOMM20 was sensitive to proteolysis at lower PK concentrations than the IMM protein ATP-synthase F1 subunit beta (ATPB), which was protected at low PK levels. The PK proteolysis pattern of Mtlbn was identical to that of ATPB, indicating that it is likely an IMM protein with its C terminus in the matrix (Fig. 2D). To confirm this finding, we applied a second method using purified mitochondria from Mtlbn–HA-expressing HEK293 cells and subjected them to osmotic shock to create mitoplast fractions, which consist of only the IMM and matrix. Mitoplasts were exposed to PK proteolysis in the presence and absence of Triton detergent, and Mtlbn exhibited a PK proteolysis resistance pattern like that of the IMM protein ATPB and distinct from TOMM20 (Fig. 2E), thus substantiating the IMM localization of Mtlbn.Mtlbn Interacts with Components of Mitochondrial Respiratory Chain Complex III.Many transmembrane microproteins act as important regulatory effectors that modify the activity of large membrane proteins and protein complexes (22). To identify putative protein-interaction partners for Mtlbn, we expressed a Mtlbn–HA fusion protein (or a HA–GFP control protein) in HEK293 cells and performed HA immunoprecipitations (IPs). Analysis of HA-eluted proteins by silver stain revealed a unique banding pattern in Mtlbn samples compared to GFP samples (Fig. 2F). HA-eluted samples were analyzed by liquid chromatography-tandem mass spectrometry (LC-MS/MS) and peptide mass fingerprinting revealed several high-confidence interacting partners for Mtlbn (Dataset S1). The four most-enriched proteins in the Mtlbn IP sample were all components of complex III of the mitochondrial respiratory chain (UQCRC1, UQCRC2, UQCR10, and UQCRFS1P1) and Panther gene ontology enrichment analysis of the eluted proteins indicated a strong association with mitochondrial complex III activity (electron transfer from ubiquinol to cytochrome c) (Fig. 2G). Complex III, also known as the cytochrome bc1 complex or coenzyme Q-cytochrome c reductase, is part of the ETC and plays a critical role in mitochondrial oxidative phosphorylation and ATP generation. Complex III is a multisubunit transmembrane protein complex that catalyzes the reduction of cytochrome c by oxidation of coenzyme Q and the translocation of protons across the IMM to the intermembrane space. Consistent with these findings, a previously published proteomics analysis of bovine heart mitochondria identified a partial protein sequence for a proteolipid protein PL-5283, which shares sequence homology to Mtlbn, as a protein that copurifies with subunit IX of complex III (30).To validate the interaction of Mtlbn with complex III and analyze its role in vivo, we generated transgenic (TG) mice with cardiac-specific overexpression of the Mtlbn–HA fusion protein using the alpha-myosin heavy chain (α-MHC) promoter (31). Proper localization of the Mtlbn–HA fusion protein to cardiomyocyte mitochondria was confirmed by immunofluorescence (SI Appendix, Fig. S1). Subsequently, heart tissues were isolated from wild-type (WT) or Mtlbn TG mice and HA IPs were performed on cardiac extracts. HA-eluted proteins were analyzed by Western blot and a clear association between Mtlbn and UQCRC1, UQCRC2, and UQCR10 was confirmed (Fig. 2H). To validate the specificity of our IPs, we looked to see whether other highly abundant mitochondrial proteins from unrelated metabolic processes were also pulled down and saw no evidence of nonspecific interactions. As an example, HADHA, which was previously identified as a strong interaction partner of an unrelated inner mitochondrial membrane microprotein called MOXI (13), was not pulled down with Mtlbn (Fig. 2H).Mtlbn Contributes to Mitochondrial Supercomplex Organization and Complex III Activity In Vivo.The individual complexes of the ETC are dynamically regulated and form supercomplexes (SCs), or respirasomes, which are thought to increase the efficiency of respiration (32, 33). The organization of SCs can be visualized by blue native polyacrylamide gel electrophoresis (BN-PAGE), which resolves protein complexes by molecular weight while retaining their native structure (34). To assess whether Mtlbn associates with mitochondrial SCs, we performed BN-PAGE on mitochondria isolated from hearts of WT or TG mice and visualized the migration pattern of Mtlbn and the five respiratory complexes by Western blot analysis (Fig. 2I). Using an antibody for HA to detect Mtlbn, we observed that Mtlbn migrated in a pattern identical to that of complex III, substantiating its interaction with complex III and indicating it is a component of complex III containing SCs (Fig. 2I).To analyze the physiological role of Mtlbn in vivo, we generated a loss-of-function mouse model using the CRISPR-associated protein 9 (Cas9) system. Heterozygous Mtlbn+/− mice yielded knockout (KO) (Mtlbn−/−) offspring at expected Mendelian ratios and KO mice did not exhibit an overt phenotype, displaying normal heart function and cardiac dimensions (SI Appendix, Fig. S2). To assess whether loss of Mtlbn altered mitochondrial SC organization, we performed BN-PAGE on mitochondria isolated from hearts of WT and Mtlbn KO mice followed by Western blot analysis to visualize the migration patterns of complexes I–V (Fig. 3 A–E). Interestingly, loss of Mtlbn did not change total complex III expression (SI Appendix, Fig. S3), but resulted in the disruption of complex III assembly into SCs as evidenced by the appearance of two unique intermediate species in Mtlbn KO extracts (Fig. 3C, red arrows). These species were not visible in other ETC complex Western blots (Fig. 3 A–E), indicating that Mtlbn uniquely resides within complex III where it alters composition when deleted. In support of this, Coomassie blue stained BN-PAGE gels also did not reveal significant changes in prominent SC populations in Mtlbn KO heart mitochondria extracts (Fig. 3F), indicating that loss of Mtlbn results in specific complex III changes. We were unable to visualize the unique intermediate complex III species present in Mtlbn KO mouse heart samples by Coomassie stain (Fig. 3F), likely due to the low sensitivity and specificity of this staining method.Fig. 3.Mtlbn knockout mice display altered supercomplex organization and reduced complex III activity. (A–E) Isolated mitochondria from hearts of Mtlbn WT and KO mice were separated by BN-PAGE, and Western blot analysis was performed using antibodies for complexes I–V (CI–CV) to visualize SC migration using the following antibodies: (A) CI (NDUFB8); (B) CII (SDHA); (C) CIII (UQCRC1); (D) CIV (COXIV); and (E) CV (ATP5a). Red arrows in C indicate the unique complex III intermediates observed in Mtlbn KO hearts. For each genotype, n = 3 mice were analyzed. (F) Coomassie blue staining of BN-PAGE gels analyzing heart mitochondrial extracts from Mtlbn WT and KO mice (n = 3 mice/genotype). (G and H) Mitochondrial complex III activity (G) and total NADH oxidase activity (H) were measured in respirating mitochondria isolated from hearts of 1-mo-old WT and Mtlbn KO mice using spectrophotometric assays. Data are expressed as mean ± SD for n = 4 mice per group. (I–M) LC-MS/MS metabolomic analysis of heart tissue from 1-mo-old Mtlbn WT and KO mice. Metabolite levels for α-ketoglutarate (I), pyruvate (J), lactate (K), acetyl-CoA (L), and pentose phosphate (M) are expressed as mean ± SD relative to WT for n = 4 mice per group. (N) Mitochondrial respiratory complex activity measured via uncoupler stimulated respiration (4 μM carbonyl cyanide 4-(trifluoromethoxy)phenylhydrazone [FCCP]) in the presence of pyruvate (10 mM) and malate (2 mM). Mitochondria from hearts of WT and Mtlbn KO mice were stimulated with the indicated compounds and real-time oxygen consumption rates (OCRs) were quantified. Compounds and substrates were used at the following concentrations: 2 μM rotenone, 10 mM succinate, 4 μM antimycin A, 100 μM TMPD, and 10 mM ascorbate. Graphs represent mean ± SD for n = 7 WT and n = 6 Mtlbn KO mice. Statistics: For G–N, statistical comparisons between groups were evaluated by two-tailed Student’s t test.To directly measure complex III activity in the hearts of WT and Mtlbn KO mice, we isolated respirating mitochondria from cardiac tissue and determined the rate of antimycin A–sensitive reduction of cytochrome c using a spectrophotometric assay and observed a significant reduction in complex III activity in KO cardiac mitochondria (Fig. 3G). To test whether this was a specific defect in complex III activity, we also measured total NADH oxidase activity in the same isolated cardiac mitochondrial preps and saw no change in KO mice compared to WT animals (Fig. 3H). We further assessed cardiac metabolic changes in Mtlbn WT and KO mice, by performing unbiased metabolomic analysis in isolated heart tissue (Datasets S2 and S3). Consistent with reduced complex III activity observed in Mtlbn KO mice, we observed an increase in α-ketoglutarate in hearts of Mtlbn KO mice (Fig. 3I), which relies on NAD+ for clearance. Disruptions in ETC function would prevent the regeneration of NAD+, resulting in the accumulation of α-ketoglutarate. Additionally, we observed an increase in the glycolytic endpoints of pyruvate (Fig. 3J), lactate (Fig. 3K), and acetyl-CoA (Fig. 3L) in hearts from Mtlbn KO mice and an increase in pentose phosphate (Fig. 3M), which could also be related to an accumulation in glycolytic intermediates. To further interrogate cardiac mitochondrial function in Mtlbn KO mice, we performed a respiratory complex assay in isolated mitochondria from hearts of Mtlbn WT and KO mice via an uncoupler-stimulated respiration protocol (Fig. 3N). We observed that loss of Mtlbn resulted in reduced rates of pyruvate-driven respiration and a reduction in succinate oxidation (Fig. 3N). However, this respiratory defect was not observed when mitochondria were treated with ascorbate/N,N,N′,N′-tetramethyl-p-phenylenediamine (TMPD), which bypasses complex III and feeds electrons directly to complex IV, indicating that the defect in Mtlbn KO mitochondria lies within complex III. Taken together, these data document the deficiencies in complex III activity in hearts of Mtlbn KO mice.Cardiac-Specific Overexpression of Mtlbn Induces Mitochondrial Dysfunction and Oxidative Stress.While Mtlbn null mice failed to display overt cardiac pathology, cardiac-specific overexpression in TG mice (SI Appendix, Fig. S1) induced cardiomyopathy leading to premature death (Fig. 4A). Serial echocardiographic assessment of left ventricular function and geometry indicated that Mtlbn TG mice exhibited early signs of cardiomyopathy by weaning age (1 mo), which progressed to severe heart failure by 6 mo of age (Fig. 4B and SI Appendix, Fig. S4 A–D). Pathological cardiac remodeling was also readily observed by histological analysis of hearts from Mtlbn TG mice at both 2- and 6-mo of age (SI Appendix, Fig. S4E), and it was common to observe large left atrial thrombi (LAT) in 6-mo-old Mtlbn TG mice (SI Appendix, Fig. S4E), which is often a complication secondary to severe congestive heart failure and likely contributed to the premature death observed in TG mice. By 2 mo of age, before LATs were observed, TG mice had significant increases in both heart weight to tibia length measurements (SI Appendix, Fig. S4F) and lung weight to tibia length measurements (SI Appendix, Fig. S4G) without an observed change in liver weight (SI Appendix, Fig. S4H). Molecular characterization of Mtlbn TG hearts indicated a signature of pathological hypertrophy with robust activation of the fetal gene program evidenced by significant increases in transcript levels of Nppa (natriuretic peptide A), Nppb (natriuretic peptide B), and Acta1 (skeletal muscle actin) and a switch from normal adult cardiac myosin (Myh6, α-MHC) to fetal cardiac myosin (Myh7, β-MHC) (Fig. 4C). Masson’s trichrome staining of heart tissue sections from 2-mo-old Mtlbn TG mice indicated a low level of diffuse interstitial fibrosis throughout the heart that was not present in WT sections (SI Appendix, Fig. S4I). Additionally, using transmission electron microscopy (EM), we assessed the ultrastructure of heart muscle from WT and TG mice and observed striking mitochondrial abnormalities in TG mice that were present throughout the micrographs, including significant alterations in IMM cristae morphology and mitochondrial disorganization (Fig. 4D). Collectively, our phenotypic analysis of Mtlbn TG mice indicates that cardiac-specific overexpression of Mtlbn induces severe cardiomyopathy and culminates in premature death, which is likely caused by mitochondrial abnormalities that drive energetic defects.Fig. 4.Cardiac-specific overexpression of Mtlbn induces mitochondrial dysfunction and increased oxidative stress. (A) Kaplan–Meier survival curves of Mtlbn WT and TG mice. Statistical analysis was performed using a log-rank (Mantle–Cox) test. (B) Echocardiography analysis of left ventricular function (fractional shortening) in WT and Mtlbn TG mice at the indicated ages. Measurements are expressed as mean ± SD for n = 8 mice per group. (C) RNA expression was assessed in hearts of 2-mo-old mice by qRT-PCR. Data are expressed as mean ± SD for n = 10 mice per group. Nppa, natriuretic peptide A; Nppb, natriuretic peptide B; Acta1, skeletal muscle actin; Myh6, α-myosin heavy chain; and Myh7, β-myosin heavy chain. (D) Transmission electron micrographs of heart tissue from WT and Mtlbn TG mice. Red arrows have been placed throughout the TG image to indicate mitochondria with disrupted structures and regions of pronounced mitochondrial disorganization. (Scale bar, 1 μm.) (E and F) Complex III activity (E) and NADH oxidase activity (F) were measured in respirating mitochondria isolated from hearts of 1-mo-old WT and Mtlbn TG mice using spectrophotometric assays. Data are expressed as mean ± SD for n = 4 mice per group. (G) Quantification of Western blot analysis of cardiac extracts from WT and Mtlbn TG mice (Westerns shown in SI Appendix, Fig. S5B). Protein expression was normalized to voltage-dependent anion channel (VDAC), and data are expressed as relative to WT. Mean expression ± SD for n = 5 mice per genotype is shown. (H) Quantitative analysis of reduced glutathione (GSH) and oxidized glutathione (GSSG) by LC-MS/MS. Data are expressed as a ratio (GSH/GSSG) for n = 8 mice per genotype (mean ± SD). (I and J) LC-MS/MS metabolomic analysis of heart tissue from 1-mo-old Mtlbn WT and TG mice. Metabolite levels for cystathionine (I) and serine phosphate (J) are expressed as mean ± SD relative to WT for n = 4 mice per group. Statistics: For B, C, and E–J, statistical comparisons between groups were evaluated by two-tailed Student’s t test.To test whether the cardiomyopathy observed in Mtlbn TG mice correlated with altered complex III activity, we measured rates of antimycin A–sensitive reduction of cytochrome c in isolated heart mitochondria from WT and TG mice (Fig. 4E), however no significant changes in complex III activity were detected. Interestingly, we observed a reduction in total NADH oxidase activity in isolated cardiac mitochondria from TG mice compared to WT animals (Fig. 4F). BN-PAGE analysis of heart mitochondria from WT and TG mice showed no detectable changes in SC formation or stability in TG mice, indicating that Mtlbn overexpression did not alter complex III assembly (SI Appendix, Fig. S5A). To gain further insight into the reduced NADH oxidase activity observed in TG mice, we analyzed cardiac extracts from WT and TG mice by standard sodium dodecyl sulfate (SDS)-PAGE and Western blot analysis and detected significant reductions in protein levels of complex I (NDUFB8), complex IV (COXIV), and complex V (ATP5a) (Fig. 4G and SI Appendix, Fig. S5B). Furthermore, we observed an increase in glutathione peroxidase 4 (Gpx4), which is a potent reactive oxygen species (ROS) scavenger protein (Fig. 4G and SI Appendix, Fig. S5B). LC-MS/MS analysis of heart tissue from WT and TG mice showed that TG mice had a reduced ratio of glutathione (GSH) to oxidized glutathione (GSSG) (Fig. 4H), suggesting that TG hearts experienced a high level of oxidative stress, thereby consuming GSH to maintain redox homeostasis. Dramatic metabolomic changes were also observed in hearts of Mtlbn TG mice, including increases in cystathionine (Fig. 4I) and serine phosphate (Fig. 4J), which are both metabolites involved in the transsulfuration pathway. Consistent with these findings, a recent study on the hearts of “Deletor” mice, which are a model for adult-onset mitochondrial myopathy with multiple mitochondrial DNA deletions (35), found that transsulfuration metabolites increased in response to global mitochondrial dysfunction and raised intracellular glutathione to buffer oxidative stress (36). These data indicate that Mtlbn is a critical regulator of mitochondrial ETC activity in the heart and that Mtlbn overexpression results in mitochondrial dysfunction and alterations in redox homeostasis.

Mtlbn Is a Heart and Skeletal Muscle–Enriched Microprotein.To search for evolutionarily conserved sORFs that code for cardiac-expressed microproteins, we used a comparative genomics approach to analyze annotated noncoding transcripts that are highly expressed in the heart. Bioinformatic analysis was performed using PhyloCSF, a multispecies nucleotide alignment tool with statistical comparison of phylogenetic codon models to score the genome for protein-coding potential (27). Using this method, we identified a conserved 141-base pair sORF, which codes for a putative 47-amino acid protein, within the highly expressed transcript, annotated as 1810058I24Rik in the mouse genome and C7orf73 in the human genome (Fig. 1 A–C). The murine Mtlbn gene is encoded by three exons (Fig. 1A) and the Mtlbn sORF spans all three of these exons and generates a high PhyloCSF score (Fig. 1B), corresponding to a conserved microprotein containing a predicted N-terminal transmembrane domain and a highly charged C terminus (Fig. 1C). Mass spectrometry of mouse heart extracts confirmed the presence of the putative Mtlbn microprotein and identified two distinct peptide fragments corresponding to unique regions of its sequence (Fig. 1 D and E), indicating it is stably expressed. qRT-PCR analysis of a mouse tissue cDNA library revealed Mtlbn is highly abundant and widely expressed with enrichment in heart and skeletal muscle (Fig. 1F). Consistent with a previous report indicating the up-regulation of 1810058I24Rik in hearts of mice in a heart failure model induced by TAC (21), we also observed a significant up-regulation of the transcript in response to pressure overload (Fig. 1G), indicating it may be involved in the cardiac response to stress.Fig. 1.Mtlbn is a conserved heart and skeletal muscle–enriched microprotein. (A) Schematic of the murine Mtlbn (1810058I24Rik) genomic locus. Mtlbn is located on chromosome 6 and consists of three exons (E1 to E3). The 141-base pair (bp) Mtlbn ORF is highlighted in red. (B) The PhyloCSF score for Mtlbn plotted using the Integrated Genomics Viewer (IGV). The region highlighted in red indicates a positive PhyloCSF score and corresponds with the Mtlbn ORF (negative scoring regions are blue). The three potential reading frames for the Mtlbn transcript are indicated in the positive strand as +1, +2, and +3. (C) Amino acid sequence alignment of Mtlbn shows high conservation across multiple species. The colors of the residues indicate their properties (pink, positive charge; blue, negative charge; red, hydrophobic; and green, polar). The symbols below the alignment represent the biochemical similarity of aligned amino acids (asterisks, identical conservation; colons, high similarity; periods, somewhat similar). The predicted transmembrane domain of Mtlbn is indicated above the sequence. (D and E) MS/MS spectrum of two unique peptide fragments corresponding to distinct regions of the Mtlbn protein identified by mass spectrometry of mouse heart tissue. (F) Detection of Mtlbn RNA by qRT-PCR in an adult mouse tissue cDNA library. Rpl4 was used for normalization, and expression is shown relative to heart for n = 4 technical replicates. Data are mean ± SD. Sk, skeletal; Sm, smooth. (G) Quantification of Mtlbn RNA in mouse heart tissue 8 weeks after sham (control) or TAC analyzed by qRT-PCR. Data are normalized to Rpl4 and shown relative to sham for n = 6 mice. Statistical analysis was performed using a two-tailed Student’s t test.

Mtlbn Is a Heart and Skeletal Muscle–Enriched Microprotein.

To search for evolutionarily conserved sORFs that code for cardiac-expressed microproteins, we used a comparative genomics approach to analyze annotated noncoding transcripts that are highly expressed in the heart. Bioinformatic analysis was performed using PhyloCSF, a multispecies nucleotide alignment tool with statistical comparison of phylogenetic codon models to score the genome for protein-coding potential (27). Using this method, we identified a conserved 141-base pair sORF, which codes for a putative 47-amino acid protein, within the highly expressed transcript, annotated as 1810058I24Rik in the mouse genome and C7orf73 in the human genome (Fig. 1 A–C). The murine Mtlbn gene is encoded by three exons (Fig. 1A) and the Mtlbn sORF spans all three of these exons and generates a high PhyloCSF score (Fig. 1B), corresponding to a conserved microprotein containing a predicted N-terminal transmembrane domain and a highly charged C terminus (Fig. 1C). Mass spectrometry of mouse heart extracts confirmed the presence of the putative Mtlbn microprotein and identified two distinct peptide fragments corresponding to unique regions of its sequence (Fig. 1 D and E), indicating it is stably expressed. qRT-PCR analysis of a mouse tissue cDNA library revealed Mtlbn is highly abundant and widely expressed with enrichment in heart and skeletal muscle (Fig. 1F). Consistent with a previous report indicating the up-regulation of 1810058I24Rik in hearts of mice in a heart failure model induced by TAC (21), we also observed a significant up-regulation of the transcript in response to pressure overload (Fig. 1G), indicating it may be involved in the cardiac response to stress.

Mtlbn is a conserved heart and skeletal muscle–enriched microprotein. (A) Schematic of the murine Mtlbn (1810058I24Rik) genomic locus. Mtlbn is located on chromosome 6 and consists of three exons (E1 to E3). The 141-base pair (bp) Mtlbn ORF is highlighted in red. (B) The PhyloCSF score for Mtlbn plotted using the Integrated Genomics Viewer (IGV). The region highlighted in red indicates a positive PhyloCSF score and corresponds with the Mtlbn ORF (negative scoring regions are blue). The three potential reading frames for the Mtlbn transcript are indicated in the positive strand as +1, +2, and +3. (C) Amino acid sequence alignment of Mtlbn shows high conservation across multiple species. The colors of the residues indicate their properties (pink, positive charge; blue, negative charge; red, hydrophobic; and green, polar). The symbols below the alignment represent the biochemical similarity of aligned amino acids (asterisks, identical conservation; colons, high similarity; periods, somewhat similar). The predicted transmembrane domain of Mtlbn is indicated above the sequence. (D and E) MS/MS spectrum of two unique peptide fragments corresponding to distinct regions of the Mtlbn protein identified by mass spectrometry of mouse heart tissue. (F) Detection of Mtlbn RNA by qRT-PCR in an adult mouse tissue cDNA library. Rpl4 was used for normalization, and expression is shown relative to heart for n = 4 technical replicates. Data are mean ± SD. Sk, skeletal; Sm, smooth. (G) Quantification of Mtlbn RNA in mouse heart tissue 8 weeks after sham (control) or TAC analyzed by qRT-PCR. Data are normalized to Rpl4 and shown relative to sham for n = 6 mice. Statistical analysis was performed using a two-tailed Student’s t test.

Mtlbn Is an Inner Mitochondrial Membrane Microprotein.To determine the subcellular localization of Mtlbn, we cloned its full-length coding sequence in frame with a C-terminal human influenza hemagglutinin (HA) epitope tag and expressed the Mtlbn–HA fusion protein in C2C12 myoblasts for immunofluorescence labeling. We selected the HA epitope tag (1XHA) for these studies because the HA amino acid sequence is the least disruptive to the isoelectric point and hydrophobicity score of Mtlbn compared to other common tags, including 1XMyc, 1XFLAG, and 3XFLAG (calculated using the Expasy Swiss Bioinformatics Resource Portal) (28, 29). Confocal imaging revealed a mitochondrial distribution pattern for Mtlbn–HA and clear colocalization with the mitochondrial protein TOMM20 (translocase of outer mitochondrial membrane 20) (Fig. 2A). This staining pattern was distinct from the sarcoplasmic reticulum (Fig. 2B), which is another organelle rich in transmembrane microprotein function, indicating that Mtlbn specifically targets to mitochondria. Although Mtlbn does not encode a canonical mitochondrial localization signal, there is evidence that the transmembrane domains of some small mitochondrial membrane proteins serve as mitochondrial targeting and membrane anchoring regions (25). To test whether this is the case for Mtlbn, we cloned the first 22 amino acids of Mtlbn in frame with a C-terminal HA epitope tag (Mtlbn1-22–HA) and expressed this construct in C2C12 myoblasts. Immunofluorescence labeling and confocal microscopy revealed clear colocalization of Mtlbn1-22–HA with TOMM20 (Fig. 2C), indicating that the N-terminal transmembrane domain of Mtlbn is sufficient to target the protein to the mitochondria.Fig. 2.Mtlbn localizes to the inner mitochondrial membrane and interacts with subunits of mitochondrial respiratory chain complex III. (A–C) Immunofluorescence of C2C12 myoblasts expressing full-length Mtlbn (A and B) or a truncated version (Mtlbn1-22) (C) fused in frame to a C-terminal HA tag. HA (Mtlbn) immunolabeling is indicated in green in all images, while the mitochondrial protein TOMM20 (A and C) or the sarcoplasmic reticulum protein SERCA2 (B) are shown in red. Merged panels also include a nuclear DAPI stain (blue). (Scale bar, 5 μm.) (D) Intact mitochondria were isolated from HEK293 cells expressing Mtlbn–HA and subjected to proteinase K (PK) proteolysis at the indicated concentrations. Western blot analysis was performed using antibodies for markers of the OMM (TOMM20), IMM (ATPB), and Mtlbn (HA). (E) The PK proteolysis pattern of mitochondria isolated from HEK293 cells expressing Mtlbn–HA in the presence (+) and absence (−) of detergent (Triton). Intact mitochondria (whole mito) were treated with osmotic shock to isolate mitoplasts containing only the IMM and matrix. Western blots were performed using the same antibodies detailed in D. (F) Silver stain of lysates and HA elutions from HA IPs performed on HA–GFP or Mtlbn–HA expressing HEK293 cells. Arrows indicate the predicted migration of HA–GFP and Mtlbn–HA. (G) Panther gene ontology (GO) enrichment analysis of the proteins pulled down in association with Mtlbn–HA (enriched over HA–GFP control). The top 10 GO categories are depicted. (H) Western blots of lysates (input) and HA-eluted proteins from HA IPs on heart extracts from WT or α-MHC Mtlbn–HA TG mice. UQCRC1, UQCRC2, and UQCR10 are components of complex III. HADHA is a subunit of the mitochondrial trifunctional protein. (I) Western blot analysis of cardiac mitochondria isolated from WT or Mtlbn TG mice separated by BN-PAGE. Antibodies for the following respiratory complexes (CI–CV) were used: CI (NDUFB8), CII (SDHA), CIII (UQCRC1), CIV (COXIV), and CV (ATP5a). Red arrows highlight the identical migration pattern of Mtlbn–HA and complex III.To assess whether Mtlbn localizes to the outer or inner mitochondrial membrane (OMM or IMM), we performed two independent mitochondrial fractionation techniques using purified mitochondria from HEK293 cells expressing Mtlbn–HA. First, purified mitochondria were subjected to progressive proteolysis with serial dilutions of proteinase K (PK) and the products were analyzed by Western blot (Fig. 2D). The OMM protein TOMM20 was sensitive to proteolysis at lower PK concentrations than the IMM protein ATP-synthase F1 subunit beta (ATPB), which was protected at low PK levels. The PK proteolysis pattern of Mtlbn was identical to that of ATPB, indicating that it is likely an IMM protein with its C terminus in the matrix (Fig. 2D). To confirm this finding, we applied a second method using purified mitochondria from Mtlbn–HA-expressing HEK293 cells and subjected them to osmotic shock to create mitoplast fractions, which consist of only the IMM and matrix. Mitoplasts were exposed to PK proteolysis in the presence and absence of Triton detergent, and Mtlbn exhibited a PK proteolysis resistance pattern like that of the IMM protein ATPB and distinct from TOMM20 (Fig. 2E), thus substantiating the IMM localization of Mtlbn.

Mtlbn Is an Inner Mitochondrial Membrane Microprotein.

To determine the subcellular localization of Mtlbn, we cloned its full-length coding sequence in frame with a C-terminal human influenza hemagglutinin (HA) epitope tag and expressed the Mtlbn–HA fusion protein in C2C12 myoblasts for immunofluorescence labeling. We selected the HA epitope tag (1XHA) for these studies because the HA amino acid sequence is the least disruptive to the isoelectric point and hydrophobicity score of Mtlbn compared to other common tags, including 1XMyc, 1XFLAG, and 3XFLAG (calculated using the Expasy Swiss Bioinformatics Resource Portal) (28, 29). Confocal imaging revealed a mitochondrial distribution pattern for Mtlbn–HA and clear colocalization with the mitochondrial protein TOMM20 (translocase of outer mitochondrial membrane 20) (Fig. 2A). This staining pattern was distinct from the sarcoplasmic reticulum (Fig. 2B), which is another organelle rich in transmembrane microprotein function, indicating that Mtlbn specifically targets to mitochondria. Although Mtlbn does not encode a canonical mitochondrial localization signal, there is evidence that the transmembrane domains of some small mitochondrial membrane proteins serve as mitochondrial targeting and membrane anchoring regions (25). To test whether this is the case for Mtlbn, we cloned the first 22 amino acids of Mtlbn in frame with a C-terminal HA epitope tag (Mtlbn1-22–HA) and expressed this construct in C2C12 myoblasts. Immunofluorescence labeling and confocal microscopy revealed clear colocalization of Mtlbn1-22–HA with TOMM20 (Fig. 2C), indicating that the N-terminal transmembrane domain of Mtlbn is sufficient to target the protein to the mitochondria.

Mtlbn localizes to the inner mitochondrial membrane and interacts with subunits of mitochondrial respiratory chain complex III. (A–C) Immunofluorescence of C2C12 myoblasts expressing full-length Mtlbn (A and B) or a truncated version (Mtlbn1-22) (C) fused in frame to a C-terminal HA tag. HA (Mtlbn) immunolabeling is indicated in green in all images, while the mitochondrial protein TOMM20 (A and C) or the sarcoplasmic reticulum protein SERCA2 (B) are shown in red. Merged panels also include a nuclear DAPI stain (blue). (Scale bar, 5 μm.) (D) Intact mitochondria were isolated from HEK293 cells expressing Mtlbn–HA and subjected to proteinase K (PK) proteolysis at the indicated concentrations. Western blot analysis was performed using antibodies for markers of the OMM (TOMM20), IMM (ATPB), and Mtlbn (HA). (E) The PK proteolysis pattern of mitochondria isolated from HEK293 cells expressing Mtlbn–HA in the presence (+) and absence (−) of detergent (Triton). Intact mitochondria (whole mito) were treated with osmotic shock to isolate mitoplasts containing only the IMM and matrix. Western blots were performed using the same antibodies detailed in D. (F) Silver stain of lysates and HA elutions from HA IPs performed on HA–GFP or Mtlbn–HA expressing HEK293 cells. Arrows indicate the predicted migration of HA–GFP and Mtlbn–HA. (G) Panther gene ontology (GO) enrichment analysis of the proteins pulled down in association with Mtlbn–HA (enriched over HA–GFP control). The top 10 GO categories are depicted. (H) Western blots of lysates (input) and HA-eluted proteins from HA IPs on heart extracts from WT or α-MHC Mtlbn–HA TG mice. UQCRC1, UQCRC2, and UQCR10 are components of complex III. HADHA is a subunit of the mitochondrial trifunctional protein. (I) Western blot analysis of cardiac mitochondria isolated from WT or Mtlbn TG mice separated by BN-PAGE. Antibodies for the following respiratory complexes (CI–CV) were used: CI (NDUFB8), CII (SDHA), CIII (UQCRC1), CIV (COXIV), and CV (ATP5a). Red arrows highlight the identical migration pattern of Mtlbn–HA and complex III.

To assess whether Mtlbn localizes to the outer or inner mitochondrial membrane (OMM or IMM), we performed two independent mitochondrial fractionation techniques using purified mitochondria from HEK293 cells expressing Mtlbn–HA. First, purified mitochondria were subjected to progressive proteolysis with serial dilutions of proteinase K (PK) and the products were analyzed by Western blot (Fig. 2D). The OMM protein TOMM20 was sensitive to proteolysis at lower PK concentrations than the IMM protein ATP-synthase F1 subunit beta (ATPB), which was protected at low PK levels. The PK proteolysis pattern of Mtlbn was identical to that of ATPB, indicating that it is likely an IMM protein with its C terminus in the matrix (Fig. 2D). To confirm this finding, we applied a second method using purified mitochondria from Mtlbn–HA-expressing HEK293 cells and subjected them to osmotic shock to create mitoplast fractions, which consist of only the IMM and matrix. Mitoplasts were exposed to PK proteolysis in the presence and absence of Triton detergent, and Mtlbn exhibited a PK proteolysis resistance pattern like that of the IMM protein ATPB and distinct from TOMM20 (Fig. 2E), thus substantiating the IMM localization of Mtlbn.

Mtlbn Interacts with Components of Mitochondrial Respiratory Chain Complex III.Many transmembrane microproteins act as important regulatory effectors that modify the activity of large membrane proteins and protein complexes (22). To identify putative protein-interaction partners for Mtlbn, we expressed a Mtlbn–HA fusion protein (or a HA–GFP control protein) in HEK293 cells and performed HA immunoprecipitations (IPs). Analysis of HA-eluted proteins by silver stain revealed a unique banding pattern in Mtlbn samples compared to GFP samples (Fig. 2F). HA-eluted samples were analyzed by liquid chromatography-tandem mass spectrometry (LC-MS/MS) and peptide mass fingerprinting revealed several high-confidence interacting partners for Mtlbn (Dataset S1). The four most-enriched proteins in the Mtlbn IP sample were all components of complex III of the mitochondrial respiratory chain (UQCRC1, UQCRC2, UQCR10, and UQCRFS1P1) and Panther gene ontology enrichment analysis of the eluted proteins indicated a strong association with mitochondrial complex III activity (electron transfer from ubiquinol to cytochrome c) (Fig. 2G). Complex III, also known as the cytochrome bc1 complex or coenzyme Q-cytochrome c reductase, is part of the ETC and plays a critical role in mitochondrial oxidative phosphorylation and ATP generation. Complex III is a multisubunit transmembrane protein complex that catalyzes the reduction of cytochrome c by oxidation of coenzyme Q and the translocation of protons across the IMM to the intermembrane space. Consistent with these findings, a previously published proteomics analysis of bovine heart mitochondria identified a partial protein sequence for a proteolipid protein PL-5283, which shares sequence homology to Mtlbn, as a protein that copurifies with subunit IX of complex III (30).To validate the interaction of Mtlbn with complex III and analyze its role in vivo, we generated transgenic (TG) mice with cardiac-specific overexpression of the Mtlbn–HA fusion protein using the alpha-myosin heavy chain (α-MHC) promoter (31). Proper localization of the Mtlbn–HA fusion protein to cardiomyocyte mitochondria was confirmed by immunofluorescence (SI Appendix, Fig. S1). Subsequently, heart tissues were isolated from wild-type (WT) or Mtlbn TG mice and HA IPs were performed on cardiac extracts. HA-eluted proteins were analyzed by Western blot and a clear association between Mtlbn and UQCRC1, UQCRC2, and UQCR10 was confirmed (Fig. 2H). To validate the specificity of our IPs, we looked to see whether other highly abundant mitochondrial proteins from unrelated metabolic processes were also pulled down and saw no evidence of nonspecific interactions. As an example, HADHA, which was previously identified as a strong interaction partner of an unrelated inner mitochondrial membrane microprotein called MOXI (13), was not pulled down with Mtlbn (Fig. 2H).

Mtlbn Interacts with Components of Mitochondrial Respiratory Chain Complex III.

Many transmembrane microproteins act as important regulatory effectors that modify the activity of large membrane proteins and protein complexes (22). To identify putative protein-interaction partners for Mtlbn, we expressed a Mtlbn–HA fusion protein (or a HA–GFP control protein) in HEK293 cells and performed HA immunoprecipitations (IPs). Analysis of HA-eluted proteins by silver stain revealed a unique banding pattern in Mtlbn samples compared to GFP samples (Fig. 2F). HA-eluted samples were analyzed by liquid chromatography-tandem mass spectrometry (LC-MS/MS) and peptide mass fingerprinting revealed several high-confidence interacting partners for Mtlbn (Dataset S1). The four most-enriched proteins in the Mtlbn IP sample were all components of complex III of the mitochondrial respiratory chain (UQCRC1, UQCRC2, UQCR10, and UQCRFS1P1) and Panther gene ontology enrichment analysis of the eluted proteins indicated a strong association with mitochondrial complex III activity (electron transfer from ubiquinol to cytochrome c) (Fig. 2G). Complex III, also known as the cytochrome bc1 complex or coenzyme Q-cytochrome c reductase, is part of the ETC and plays a critical role in mitochondrial oxidative phosphorylation and ATP generation. Complex III is a multisubunit transmembrane protein complex that catalyzes the reduction of cytochrome c by oxidation of coenzyme Q and the translocation of protons across the IMM to the intermembrane space. Consistent with these findings, a previously published proteomics analysis of bovine heart mitochondria identified a partial protein sequence for a proteolipid protein PL-5283, which shares sequence homology to Mtlbn, as a protein that copurifies with subunit IX of complex III (30).

To validate the interaction of Mtlbn with complex III and analyze its role in vivo, we generated transgenic (TG) mice with cardiac-specific overexpression of the Mtlbn–HA fusion protein using the alpha-myosin heavy chain (α-MHC) promoter (31). Proper localization of the Mtlbn–HA fusion protein to cardiomyocyte mitochondria was confirmed by immunofluorescence (SI Appendix, Fig. S1). Subsequently, heart tissues were isolated from wild-type (WT) or Mtlbn TG mice and HA IPs were performed on cardiac extracts. HA-eluted proteins were analyzed by Western blot and a clear association between Mtlbn and UQCRC1, UQCRC2, and UQCR10 was confirmed (Fig. 2H). To validate the specificity of our IPs, we looked to see whether other highly abundant mitochondrial proteins from unrelated metabolic processes were also pulled down and saw no evidence of nonspecific interactions. As an example, HADHA, which was previously identified as a strong interaction partner of an unrelated inner mitochondrial membrane microprotein called MOXI (13), was not pulled down with Mtlbn (Fig. 2H).

Mtlbn Contributes to Mitochondrial Supercomplex Organization and Complex III Activity In Vivo.The individual complexes of the ETC are dynamically regulated and form supercomplexes (SCs), or respirasomes, which are thought to increase the efficiency of respiration (32, 33). The organization of SCs can be visualized by blue native polyacrylamide gel electrophoresis (BN-PAGE), which resolves protein complexes by molecular weight while retaining their native structure (34). To assess whether Mtlbn associates with mitochondrial SCs, we performed BN-PAGE on mitochondria isolated from hearts of WT or TG mice and visualized the migration pattern of Mtlbn and the five respiratory complexes by Western blot analysis (Fig. 2I). Using an antibody for HA to detect Mtlbn, we observed that Mtlbn migrated in a pattern identical to that of complex III, substantiating its interaction with complex III and indicating it is a component of complex III containing SCs (Fig. 2I).To analyze the physiological role of Mtlbn in vivo, we generated a loss-of-function mouse model using the CRISPR-associated protein 9 (Cas9) system. Heterozygous Mtlbn+/− mice yielded knockout (KO) (Mtlbn−/−) offspring at expected Mendelian ratios and KO mice did not exhibit an overt phenotype, displaying normal heart function and cardiac dimensions (SI Appendix, Fig. S2). To assess whether loss of Mtlbn altered mitochondrial SC organization, we performed BN-PAGE on mitochondria isolated from hearts of WT and Mtlbn KO mice followed by Western blot analysis to visualize the migration patterns of complexes I–V (Fig. 3 A–E). Interestingly, loss of Mtlbn did not change total complex III expression (SI Appendix, Fig. S3), but resulted in the disruption of complex III assembly into SCs as evidenced by the appearance of two unique intermediate species in Mtlbn KO extracts (Fig. 3C, red arrows). These species were not visible in other ETC complex Western blots (Fig. 3 A–E), indicating that Mtlbn uniquely resides within complex III where it alters composition when deleted. In support of this, Coomassie blue stained BN-PAGE gels also did not reveal significant changes in prominent SC populations in Mtlbn KO heart mitochondria extracts (Fig. 3F), indicating that loss of Mtlbn results in specific complex III changes. We were unable to visualize the unique intermediate complex III species present in Mtlbn KO mouse heart samples by Coomassie stain (Fig. 3F), likely due to the low sensitivity and specificity of this staining method.Fig. 3.Mtlbn knockout mice display altered supercomplex organization and reduced complex III activity. (A–E) Isolated mitochondria from hearts of Mtlbn WT and KO mice were separated by BN-PAGE, and Western blot analysis was performed using antibodies for complexes I–V (CI–CV) to visualize SC migration using the following antibodies: (A) CI (NDUFB8); (B) CII (SDHA); (C) CIII (UQCRC1); (D) CIV (COXIV); and (E) CV (ATP5a). Red arrows in C indicate the unique complex III intermediates observed in Mtlbn KO hearts. For each genotype, n = 3 mice were analyzed. (F) Coomassie blue staining of BN-PAGE gels analyzing heart mitochondrial extracts from Mtlbn WT and KO mice (n = 3 mice/genotype). (G and H) Mitochondrial complex III activity (G) and total NADH oxidase activity (H) were measured in respirating mitochondria isolated from hearts of 1-mo-old WT and Mtlbn KO mice using spectrophotometric assays. Data are expressed as mean ± SD for n = 4 mice per group. (I–M) LC-MS/MS metabolomic analysis of heart tissue from 1-mo-old Mtlbn WT and KO mice. Metabolite levels for α-ketoglutarate (I), pyruvate (J), lactate (K), acetyl-CoA (L), and pentose phosphate (M) are expressed as mean ± SD relative to WT for n = 4 mice per group. (N) Mitochondrial respiratory complex activity measured via uncoupler stimulated respiration (4 μM carbonyl cyanide 4-(trifluoromethoxy)phenylhydrazone [FCCP]) in the presence of pyruvate (10 mM) and malate (2 mM). Mitochondria from hearts of WT and Mtlbn KO mice were stimulated with the indicated compounds and real-time oxygen consumption rates (OCRs) were quantified. Compounds and substrates were used at the following concentrations: 2 μM rotenone, 10 mM succinate, 4 μM antimycin A, 100 μM TMPD, and 10 mM ascorbate. Graphs represent mean ± SD for n = 7 WT and n = 6 Mtlbn KO mice. Statistics: For G–N, statistical comparisons between groups were evaluated by two-tailed Student’s t test.To directly measure complex III activity in the hearts of WT and Mtlbn KO mice, we isolated respirating mitochondria from cardiac tissue and determined the rate of antimycin A–sensitive reduction of cytochrome c using a spectrophotometric assay and observed a significant reduction in complex III activity in KO cardiac mitochondria (Fig. 3G). To test whether this was a specific defect in complex III activity, we also measured total NADH oxidase activity in the same isolated cardiac mitochondrial preps and saw no change in KO mice compared to WT animals (Fig. 3H). We further assessed cardiac metabolic changes in Mtlbn WT and KO mice, by performing unbiased metabolomic analysis in isolated heart tissue (Datasets S2 and S3). Consistent with reduced complex III activity observed in Mtlbn KO mice, we observed an increase in α-ketoglutarate in hearts of Mtlbn KO mice (Fig. 3I), which relies on NAD+ for clearance. Disruptions in ETC function would prevent the regeneration of NAD+, resulting in the accumulation of α-ketoglutarate. Additionally, we observed an increase in the glycolytic endpoints of pyruvate (Fig. 3J), lactate (Fig. 3K), and acetyl-CoA (Fig. 3L) in hearts from Mtlbn KO mice and an increase in pentose phosphate (Fig. 3M), which could also be related to an accumulation in glycolytic intermediates. To further interrogate cardiac mitochondrial function in Mtlbn KO mice, we performed a respiratory complex assay in isolated mitochondria from hearts of Mtlbn WT and KO mice via an uncoupler-stimulated respiration protocol (Fig. 3N). We observed that loss of Mtlbn resulted in reduced rates of pyruvate-driven respiration and a reduction in succinate oxidation (Fig. 3N). However, this respiratory defect was not observed when mitochondria were treated with ascorbate/N,N,N′,N′-tetramethyl-p-phenylenediamine (TMPD), which bypasses complex III and feeds electrons directly to complex IV, indicating that the defect in Mtlbn KO mitochondria lies within complex III. Taken together, these data document the deficiencies in complex III activity in hearts of Mtlbn KO mice.

Mtlbn Contributes to Mitochondrial Supercomplex Organization and Complex III Activity In Vivo.

The individual complexes of the ETC are dynamically regulated and form supercomplexes (SCs), or respirasomes, which are thought to increase the efficiency of respiration (32, 33). The organization of SCs can be visualized by blue native polyacrylamide gel electrophoresis (BN-PAGE), which resolves protein complexes by molecular weight while retaining their native structure (34). To assess whether Mtlbn associates with mitochondrial SCs, we performed BN-PAGE on mitochondria isolated from hearts of WT or TG mice and visualized the migration pattern of Mtlbn and the five respiratory complexes by Western blot analysis (Fig. 2I). Using an antibody for HA to detect Mtlbn, we observed that Mtlbn migrated in a pattern identical to that of complex III, substantiating its interaction with complex III and indicating it is a component of complex III containing SCs (Fig. 2I).

To analyze the physiological role of Mtlbn in vivo, we generated a loss-of-function mouse model using the CRISPR-associated protein 9 (Cas9) system. Heterozygous Mtlbn+/− mice yielded knockout (KO) (Mtlbn−/−) offspring at expected Mendelian ratios and KO mice did not exhibit an overt phenotype, displaying normal heart function and cardiac dimensions (SI Appendix, Fig. S2). To assess whether loss of Mtlbn altered mitochondrial SC organization, we performed BN-PAGE on mitochondria isolated from hearts of WT and Mtlbn KO mice followed by Western blot analysis to visualize the migration patterns of complexes I–V (Fig. 3 A–E). Interestingly, loss of Mtlbn did not change total complex III expression (SI Appendix, Fig. S3), but resulted in the disruption of complex III assembly into SCs as evidenced by the appearance of two unique intermediate species in Mtlbn KO extracts (Fig. 3C, red arrows). These species were not visible in other ETC complex Western blots (Fig. 3 A–E), indicating that Mtlbn uniquely resides within complex III where it alters composition when deleted. In support of this, Coomassie blue stained BN-PAGE gels also did not reveal significant changes in prominent SC populations in Mtlbn KO heart mitochondria extracts (Fig. 3F), indicating that loss of Mtlbn results in specific complex III changes. We were unable to visualize the unique intermediate complex III species present in Mtlbn KO mouse heart samples by Coomassie stain (Fig. 3F), likely due to the low sensitivity and specificity of this staining method.

Mtlbn knockout mice display altered supercomplex organization and reduced complex III activity. (A–E) Isolated mitochondria from hearts of Mtlbn WT and KO mice were separated by BN-PAGE, and Western blot analysis was performed using antibodies for complexes I–V (CI–CV) to visualize SC migration using the following antibodies: (A) CI (NDUFB8); (B) CII (SDHA); (C) CIII (UQCRC1); (D) CIV (COXIV); and (E) CV (ATP5a). Red arrows in C indicate the unique complex III intermediates observed in Mtlbn KO hearts. For each genotype, n = 3 mice were analyzed. (F) Coomassie blue staining of BN-PAGE gels analyzing heart mitochondrial extracts from Mtlbn WT and KO mice (n = 3 mice/genotype). (G and H) Mitochondrial complex III activity (G) and total NADH oxidase activity (H) were measured in respirating mitochondria isolated from hearts of 1-mo-old WT and Mtlbn KO mice using spectrophotometric assays. Data are expressed as mean ± SD for n = 4 mice per group. (I–M) LC-MS/MS metabolomic analysis of heart tissue from 1-mo-old Mtlbn WT and KO mice. Metabolite levels for α-ketoglutarate (I), pyruvate (J), lactate (K), acetyl-CoA (L), and pentose phosphate (M) are expressed as mean ± SD relative to WT for n = 4 mice per group. (N) Mitochondrial respiratory complex activity measured via uncoupler stimulated respiration (4 μM carbonyl cyanide 4-(trifluoromethoxy)phenylhydrazone [FCCP]) in the presence of pyruvate (10 mM) and malate (2 mM). Mitochondria from hearts of WT and Mtlbn KO mice were stimulated with the indicated compounds and real-time oxygen consumption rates (OCRs) were quantified. Compounds and substrates were used at the following concentrations: 2 μM rotenone, 10 mM succinate, 4 μM antimycin A, 100 μM TMPD, and 10 mM ascorbate. Graphs represent mean ± SD for n = 7 WT and n = 6 Mtlbn KO mice. Statistics: For G–N, statistical comparisons between groups were evaluated by two-tailed Student’s t test.

To directly measure complex III activity in the hearts of WT and Mtlbn KO mice, we isolated respirating mitochondria from cardiac tissue and determined the rate of antimycin A–sensitive reduction of cytochrome c using a spectrophotometric assay and observed a significant reduction in complex III activity in KO cardiac mitochondria (Fig. 3G). To test whether this was a specific defect in complex III activity, we also measured total NADH oxidase activity in the same isolated cardiac mitochondrial preps and saw no change in KO mice compared to WT animals (Fig. 3H). We further assessed cardiac metabolic changes in Mtlbn WT and KO mice, by performing unbiased metabolomic analysis in isolated heart tissue (Datasets S2 and S3). Consistent with reduced complex III activity observed in Mtlbn KO mice, we observed an increase in α-ketoglutarate in hearts of Mtlbn KO mice (Fig. 3I), which relies on NAD+ for clearance. Disruptions in ETC function would prevent the regeneration of NAD+, resulting in the accumulation of α-ketoglutarate. Additionally, we observed an increase in the glycolytic endpoints of pyruvate (Fig. 3J), lactate (Fig. 3K), and acetyl-CoA (Fig. 3L) in hearts from Mtlbn KO mice and an increase in pentose phosphate (Fig. 3M), which could also be related to an accumulation in glycolytic intermediates. To further interrogate cardiac mitochondrial function in Mtlbn KO mice, we performed a respiratory complex assay in isolated mitochondria from hearts of Mtlbn WT and KO mice via an uncoupler-stimulated respiration protocol (Fig. 3N). We observed that loss of Mtlbn resulted in reduced rates of pyruvate-driven respiration and a reduction in succinate oxidation (Fig. 3N). However, this respiratory defect was not observed when mitochondria were treated with ascorbate/N,N,N′,N′-tetramethyl-p-phenylenediamine (TMPD), which bypasses complex III and feeds electrons directly to complex IV, indicating that the defect in Mtlbn KO mitochondria lies within complex III. Taken together, these data document the deficiencies in complex III activity in hearts of Mtlbn KO mice.

Cardiac-Specific Overexpression of Mtlbn Induces Mitochondrial Dysfunction and Oxidative Stress.While Mtlbn null mice failed to display overt cardiac pathology, cardiac-specific overexpression in TG mice (SI Appendix, Fig. S1) induced cardiomyopathy leading to premature death (Fig. 4A). Serial echocardiographic assessment of left ventricular function and geometry indicated that Mtlbn TG mice exhibited early signs of cardiomyopathy by weaning age (1 mo), which progressed to severe heart failure by 6 mo of age (Fig. 4B and SI Appendix, Fig. S4 A–D). Pathological cardiac remodeling was also readily observed by histological analysis of hearts from Mtlbn TG mice at both 2- and 6-mo of age (SI Appendix, Fig. S4E), and it was common to observe large left atrial thrombi (LAT) in 6-mo-old Mtlbn TG mice (SI Appendix, Fig. S4E), which is often a complication secondary to severe congestive heart failure and likely contributed to the premature death observed in TG mice. By 2 mo of age, before LATs were observed, TG mice had significant increases in both heart weight to tibia length measurements (SI Appendix, Fig. S4F) and lung weight to tibia length measurements (SI Appendix, Fig. S4G) without an observed change in liver weight (SI Appendix, Fig. S4H). Molecular characterization of Mtlbn TG hearts indicated a signature of pathological hypertrophy with robust activation of the fetal gene program evidenced by significant increases in transcript levels of Nppa (natriuretic peptide A), Nppb (natriuretic peptide B), and Acta1 (skeletal muscle actin) and a switch from normal adult cardiac myosin (Myh6, α-MHC) to fetal cardiac myosin (Myh7, β-MHC) (Fig. 4C). Masson’s trichrome staining of heart tissue sections from 2-mo-old Mtlbn TG mice indicated a low level of diffuse interstitial fibrosis throughout the heart that was not present in WT sections (SI Appendix, Fig. S4I). Additionally, using transmission electron microscopy (EM), we assessed the ultrastructure of heart muscle from WT and TG mice and observed striking mitochondrial abnormalities in TG mice that were present throughout the micrographs, including significant alterations in IMM cristae morphology and mitochondrial disorganization (Fig. 4D). Collectively, our phenotypic analysis of Mtlbn TG mice indicates that cardiac-specific overexpression of Mtlbn induces severe cardiomyopathy and culminates in premature death, which is likely caused by mitochondrial abnormalities that drive energetic defects.Fig. 4.Cardiac-specific overexpression of Mtlbn induces mitochondrial dysfunction and increased oxidative stress. (A) Kaplan–Meier survival curves of Mtlbn WT and TG mice. Statistical analysis was performed using a log-rank (Mantle–Cox) test. (B) Echocardiography analysis of left ventricular function (fractional shortening) in WT and Mtlbn TG mice at the indicated ages. Measurements are expressed as mean ± SD for n = 8 mice per group. (C) RNA expression was assessed in hearts of 2-mo-old mice by qRT-PCR. Data are expressed as mean ± SD for n = 10 mice per group. Nppa, natriuretic peptide A; Nppb, natriuretic peptide B; Acta1, skeletal muscle actin; Myh6, α-myosin heavy chain; and Myh7, β-myosin heavy chain. (D) Transmission electron micrographs of heart tissue from WT and Mtlbn TG mice. Red arrows have been placed throughout the TG image to indicate mitochondria with disrupted structures and regions of pronounced mitochondrial disorganization. (Scale bar, 1 μm.) (E and F) Complex III activity (E) and NADH oxidase activity (F) were measured in respirating mitochondria isolated from hearts of 1-mo-old WT and Mtlbn TG mice using spectrophotometric assays. Data are expressed as mean ± SD for n = 4 mice per group. (G) Quantification of Western blot analysis of cardiac extracts from WT and Mtlbn TG mice (Westerns shown in SI Appendix, Fig. S5B). Protein expression was normalized to voltage-dependent anion channel (VDAC), and data are expressed as relative to WT. Mean expression ± SD for n = 5 mice per genotype is shown. (H) Quantitative analysis of reduced glutathione (GSH) and oxidized glutathione (GSSG) by LC-MS/MS. Data are expressed as a ratio (GSH/GSSG) for n = 8 mice per genotype (mean ± SD). (I and J) LC-MS/MS metabolomic analysis of heart tissue from 1-mo-old Mtlbn WT and TG mice. Metabolite levels for cystathionine (I) and serine phosphate (J) are expressed as mean ± SD relative to WT for n = 4 mice per group. Statistics: For B, C, and E–J, statistical comparisons between groups were evaluated by two-tailed Student’s t test.To test whether the cardiomyopathy observed in Mtlbn TG mice correlated with altered complex III activity, we measured rates of antimycin A–sensitive reduction of cytochrome c in isolated heart mitochondria from WT and TG mice (Fig. 4E), however no significant changes in complex III activity were detected. Interestingly, we observed a reduction in total NADH oxidase activity in isolated cardiac mitochondria from TG mice compared to WT animals (Fig. 4F). BN-PAGE analysis of heart mitochondria from WT and TG mice showed no detectable changes in SC formation or stability in TG mice, indicating that Mtlbn overexpression did not alter complex III assembly (SI Appendix, Fig. S5A). To gain further insight into the reduced NADH oxidase activity observed in TG mice, we analyzed cardiac extracts from WT and TG mice by standard sodium dodecyl sulfate (SDS)-PAGE and Western blot analysis and detected significant reductions in protein levels of complex I (NDUFB8), complex IV (COXIV), and complex V (ATP5a) (Fig. 4G and SI Appendix, Fig. S5B). Furthermore, we observed an increase in glutathione peroxidase 4 (Gpx4), which is a potent reactive oxygen species (ROS) scavenger protein (Fig. 4G and SI Appendix, Fig. S5B). LC-MS/MS analysis of heart tissue from WT and TG mice showed that TG mice had a reduced ratio of glutathione (GSH) to oxidized glutathione (GSSG) (Fig. 4H), suggesting that TG hearts experienced a high level of oxidative stress, thereby consuming GSH to maintain redox homeostasis. Dramatic metabolomic changes were also observed in hearts of Mtlbn TG mice, including increases in cystathionine (Fig. 4I) and serine phosphate (Fig. 4J), which are both metabolites involved in the transsulfuration pathway. Consistent with these findings, a recent study on the hearts of “Deletor” mice, which are a model for adult-onset mitochondrial myopathy with multiple mitochondrial DNA deletions (35), found that transsulfuration metabolites increased in response to global mitochondrial dysfunction and raised intracellular glutathione to buffer oxidative stress (36). These data indicate that Mtlbn is a critical regulator of mitochondrial ETC activity in the heart and that Mtlbn overexpression results in mitochondrial dysfunction and alterations in redox homeostasis.

Cardiac-Specific Overexpression of Mtlbn Induces Mitochondrial Dysfunction and Oxidative Stress.

While Mtlbn null mice failed to display overt cardiac pathology, cardiac-specific overexpression in TG mice (SI Appendix, Fig. S1) induced cardiomyopathy leading to premature death (Fig. 4A). Serial echocardiographic assessment of left ventricular function and geometry indicated that Mtlbn TG mice exhibited early signs of cardiomyopathy by weaning age (1 mo), which progressed to severe heart failure by 6 mo of age (Fig. 4B and SI Appendix, Fig. S4 A–D). Pathological cardiac remodeling was also readily observed by histological analysis of hearts from Mtlbn TG mice at both 2- and 6-mo of age (SI Appendix, Fig. S4E), and it was common to observe large left atrial thrombi (LAT) in 6-mo-old Mtlbn TG mice (SI Appendix, Fig. S4E), which is often a complication secondary to severe congestive heart failure and likely contributed to the premature death observed in TG mice. By 2 mo of age, before LATs were observed, TG mice had significant increases in both heart weight to tibia length measurements (SI Appendix, Fig. S4F) and lung weight to tibia length measurements (SI Appendix, Fig. S4G) without an observed change in liver weight (SI Appendix, Fig. S4H). Molecular characterization of Mtlbn TG hearts indicated a signature of pathological hypertrophy with robust activation of the fetal gene program evidenced by significant increases in transcript levels of Nppa (natriuretic peptide A), Nppb (natriuretic peptide B), and Acta1 (skeletal muscle actin) and a switch from normal adult cardiac myosin (Myh6, α-MHC) to fetal cardiac myosin (Myh7, β-MHC) (Fig. 4C). Masson’s trichrome staining of heart tissue sections from 2-mo-old Mtlbn TG mice indicated a low level of diffuse interstitial fibrosis throughout the heart that was not present in WT sections (SI Appendix, Fig. S4I). Additionally, using transmission electron microscopy (EM), we assessed the ultrastructure of heart muscle from WT and TG mice and observed striking mitochondrial abnormalities in TG mice that were present throughout the micrographs, including significant alterations in IMM cristae morphology and mitochondrial disorganization (Fig. 4D). Collectively, our phenotypic analysis of Mtlbn TG mice indicates that cardiac-specific overexpression of Mtlbn induces severe cardiomyopathy and culminates in premature death, which is likely caused by mitochondrial abnormalities that drive energetic defects.

Cardiac-specific overexpression of Mtlbn induces mitochondrial dysfunction and increased oxidative stress. (A) Kaplan–Meier survival curves of Mtlbn WT and TG mice. Statistical analysis was performed using a log-rank (Mantle–Cox) test. (B) Echocardiography analysis of left ventricular function (fractional shortening) in WT and Mtlbn TG mice at the indicated ages. Measurements are expressed as mean ± SD for n = 8 mice per group. (C) RNA expression was assessed in hearts of 2-mo-old mice by qRT-PCR. Data are expressed as mean ± SD for n = 10 mice per group. Nppa, natriuretic peptide A; Nppb, natriuretic peptide B; Acta1, skeletal muscle actin; Myh6, α-myosin heavy chain; and Myh7, β-myosin heavy chain. (D) Transmission electron micrographs of heart tissue from WT and Mtlbn TG mice. Red arrows have been placed throughout the TG image to indicate mitochondria with disrupted structures and regions of pronounced mitochondrial disorganization. (Scale bar, 1 μm.) (E and F) Complex III activity (E) and NADH oxidase activity (F) were measured in respirating mitochondria isolated from hearts of 1-mo-old WT and Mtlbn TG mice using spectrophotometric assays. Data are expressed as mean ± SD for n = 4 mice per group. (G) Quantification of Western blot analysis of cardiac extracts from WT and Mtlbn TG mice (Westerns shown in SI Appendix, Fig. S5B). Protein expression was normalized to voltage-dependent anion channel (VDAC), and data are expressed as relative to WT. Mean expression ± SD for n = 5 mice per genotype is shown. (H) Quantitative analysis of reduced glutathione (GSH) and oxidized glutathione (GSSG) by LC-MS/MS. Data are expressed as a ratio (GSH/GSSG) for n = 8 mice per genotype (mean ± SD). (I and J) LC-MS/MS metabolomic analysis of heart tissue from 1-mo-old Mtlbn WT and TG mice. Metabolite levels for cystathionine (I) and serine phosphate (J) are expressed as mean ± SD relative to WT for n = 4 mice per group. Statistics: For B, C, and E–J, statistical comparisons between groups were evaluated by two-tailed Student’s t test.

To test whether the cardiomyopathy observed in Mtlbn TG mice correlated with altered complex III activity, we measured rates of antimycin A–sensitive reduction of cytochrome c in isolated heart mitochondria from WT and TG mice (Fig. 4E), however no significant changes in complex III activity were detected. Interestingly, we observed a reduction in total NADH oxidase activity in isolated cardiac mitochondria from TG mice compared to WT animals (Fig. 4F). BN-PAGE analysis of heart mitochondria from WT and TG mice showed no detectable changes in SC formation or stability in TG mice, indicating that Mtlbn overexpression did not alter complex III assembly (SI Appendix, Fig. S5A). To gain further insight into the reduced NADH oxidase activity observed in TG mice, we analyzed cardiac extracts from WT and TG mice by standard sodium dodecyl sulfate (SDS)-PAGE and Western blot analysis and detected significant reductions in protein levels of complex I (NDUFB8), complex IV (COXIV), and complex V (ATP5a) (Fig. 4G and SI Appendix, Fig. S5B). Furthermore, we observed an increase in glutathione peroxidase 4 (Gpx4), which is a potent reactive oxygen species (ROS) scavenger protein (Fig. 4G and SI Appendix, Fig. S5B). LC-MS/MS analysis of heart tissue from WT and TG mice showed that TG mice had a reduced ratio of glutathione (GSH) to oxidized glutathione (GSSG) (Fig. 4H), suggesting that TG hearts experienced a high level of oxidative stress, thereby consuming GSH to maintain redox homeostasis. Dramatic metabolomic changes were also observed in hearts of Mtlbn TG mice, including increases in cystathionine (Fig. 4I) and serine phosphate (Fig. 4J), which are both metabolites involved in the transsulfuration pathway. Consistent with these findings, a recent study on the hearts of “Deletor” mice, which are a model for adult-onset mitochondrial myopathy with multiple mitochondrial DNA deletions (35), found that transsulfuration metabolites increased in response to global mitochondrial dysfunction and raised intracellular glutathione to buffer oxidative stress (36). These data indicate that Mtlbn is a critical regulator of mitochondrial ETC activity in the heart and that Mtlbn overexpression results in mitochondrial dysfunction and alterations in redox homeostasis.

DiscussionThe growing number of recently identified microproteins derived from previously unannotated sORFs has increased the complexity of the proteome and expanded our understanding of many intricate biological processes. In this study, we characterized Mtlbn as an inner mitochondrial membrane microprotein that is expressed at high levels in the heart and regulates ETC activity through respiratory complex III. In the mitochondrial respiratory chain, electrons pass from complex I and II to complex III via the hydrophobic electron carrier ubiquinol, and then from complex III to complex IV via cytochrome c. Respiratory complex III functions as a dimer with each 240-kDa monomer containing 11 known subunits. Within the IMM, dimeric complex III can associate with complex I and/or complex IV to form SCs, which are thought to increase the efficiency of respiration (32). Efficient electron transfer along the respiratory chain reduces electron leak, which can generate mitochondrial ROS (33, 37). In addition to the central catalytic subunits of the ETC, the complexes have also been shown to associate with short transmembrane subunits, which are not directly involved in electron/proton transfer, but are believed to regulate the proper assembly, stabilization, or conformational movement of respiratory complexes (38). Our data indicate that Mtlbn acts as such a cofactor for complex III and is required for its proper function and assembly.Overall, the processes that regulate SC formation and how SCs influence mitochondrial function are not well understood; however, the recent discovery of several microproteins that are directly involved in this process, including BRAWNIN (C12orf73) (39), UQCC3 (C11orf83) (25), and Mitoregulin (LINC00116) (14), highlights how mitochondrial microproteins may be important missing pieces to this puzzle. Both BRAWNIN and UQCC3 map to complex III (25, 39), with BRAWNIN expression being controlled by the energy-sensing AMP-activated protein kinase (AMPK) pathway and contributing to complex III-containing SC assembly and stability (39). Interestingly, Mtlbn expression is increased in response to pressure-overload–induced heart failure in the mouse heart (Fig. 1G), and published proteomics datasets indicate that both Uqcr10 and Uqcrc2, which we identified as binding partners of Mtlbn in IP/MS studies (Fig. 2 and Dataset S1), are similarly increased in response to TAC (40, 41). The mechanisms that directly regulate Mtlbn expression are currently unknown and will be a focus of future studies. The discovery of Mtlbn as a component of complex III SCs may add an important level of detail to how this intricate metabolic regulatory process is coordinated.Typical to mitochondrial syndromes, human patients with complex III deficiencies and defects have highly variable phenotypes with a wide range of clinical presentations, including mild muscle weakness, exercise intolerance, and cardiomyopathy (42). It is thus not surprising that despite the reduced complex III activity and altered metabolomic profile observed in Mtlbn null mice, the animals do not develop a significant cardiac phenotype at baseline. Future studies will directly test the role of Mtlbn in contributing to mitochondrial function in response to cardiac stress conditions such as TAC and ischemia/reperfusion with the anticipation that Mtlbn knockout mice will have a more exaggerated disease phenotype. Conversely, it is interesting that Mtlbn TG mice exhibit a robust cardiac phenotype at baseline despite having a seemingly normal distribution of mitochondrial SCs and intact complex III activity. While the cardiomyopathy observed in Mtlbn TG mice can be generally explained by an alteration in mitochondrial redox state, the initial insult that drives this phenotype has not yet been determined. One potential contributing factor could be increased ROS production by complex III in Mtlbn transgenic mice, which could drive an oxidative stress phenotype and negatively impact NADH oxidase activity (43); however, this has not yet been conclusively shown to be a primary driver of disease in Mtlbn TG mice.While this work was in progress, an independent group used a series of in vitro studies to identify the Mtlbn microprotein as a mitochondrial protein (Mm47) that is required for activation of the Nlrp3 inflammasome, which is a critical component of the innate immune system (44). While the mechanism of action for Mtlbn was not described in this study, and a knockout mouse model was not analyzed, it is interesting to speculate that the cellular phenotype observed may be due to altered complex III assembly and supercomplex dynamics, which could prevent coordinated activation of the inflammasome in response to stress (45–47). In future studies, we will explore this directly using our Mtlbn gene–deleted mice.Mammalian mitochondria are thought to contain ∼1,500 different proteins (48, 49), many of which are assembled into distinct complexes in the IMM and OMM. Interestingly, a large fraction of annotated proteins that are <100 amino acids in length (i.e., microproteins) are mitochondrial proteins and include important respiratory chain complex components and their associated assembly factors (49–52). A recent study analyzing the translational landscape of the human heart employed both experimental and computational analyses to search for putative cardiac-expressed microproteins and found that many of these potential proteins may be associated with mitochondria (53). Their findings indicated strong microprotein coregulation with ETC subunits, including multiple small accessory proteins, whose expression patterns are altered in heart failure (53). Numerous human mutations in mitochondrial assembly factors have been shown to contribute to respiratory chain dysfunction and disease (54), and given that there are currently many clinical cases of mitochondrial deficiency with no known genetic cause, it is possible that a subset of these may be due to mutations in microprotein assembly factors that have yet to be discovered. Mitochondrial defects are also a common cause of pediatric cardiomyopathy (55, 56), and cardiac dysfunction is frequent in children with mitochondrial disease and is associated with increased mortality (57). This area of investigation is still poorly understood, and the genetic risk factors associated with these diseases have yet to be fully elucidated (58), suggesting that defects in newly identified or currently unknown microproteins may be contributing factors.In summary, we have shown that the cardiac-enriched microprotein Mtlbn localizes to the inner mitochondrial membrane where it interacts with respiratory complex III and contributes to its assembly and activity. Given that the processes that regulate SC formation and how SCs influence overall mitochondrial function are not well understood, the discovery of Mtlbn as a component of complex III SCs adds detail to this highly regulated metabolic process. Notably, our work also highlights the importance of further analyzing the noncoding RNA landscape for sORFs that code for putative functional microproteins, which likely play important regulatory roles in mitochondrial processes and contribute to cellular homeostasis.

The growing number of recently identified microproteins derived from previously unannotated sORFs has increased the complexity of the proteome and expanded our understanding of many intricate biological processes. In this study, we characterized Mtlbn as an inner mitochondrial membrane microprotein that is expressed at high levels in the heart and regulates ETC activity through respiratory complex III. In the mitochondrial respiratory chain, electrons pass from complex I and II to complex III via the hydrophobic electron carrier ubiquinol, and then from complex III to complex IV via cytochrome c. Respiratory complex III functions as a dimer with each 240-kDa monomer containing 11 known subunits. Within the IMM, dimeric complex III can associate with complex I and/or complex IV to form SCs, which are thought to increase the efficiency of respiration (32). Efficient electron transfer along the respiratory chain reduces electron leak, which can generate mitochondrial ROS (33, 37). In addition to the central catalytic subunits of the ETC, the complexes have also been shown to associate with short transmembrane subunits, which are not directly involved in electron/proton transfer, but are believed to regulate the proper assembly, stabilization, or conformational movement of respiratory complexes (38). Our data indicate that Mtlbn acts as such a cofactor for complex III and is required for its proper function and assembly.

Overall, the processes that regulate SC formation and how SCs influence mitochondrial function are not well understood; however, the recent discovery of several microproteins that are directly involved in this process, including BRAWNIN (C12orf73) (39), UQCC3 (C11orf83) (25), and Mitoregulin (LINC00116) (14), highlights how mitochondrial microproteins may be important missing pieces to this puzzle. Both BRAWNIN and UQCC3 map to complex III (25, 39), with BRAWNIN expression being controlled by the energy-sensing AMP-activated protein kinase (AMPK) pathway and contributing to complex III-containing SC assembly and stability (39). Interestingly, Mtlbn expression is increased in response to pressure-overload–induced heart failure in the mouse heart (Fig. 1G), and published proteomics datasets indicate that both Uqcr10 and Uqcrc2, which we identified as binding partners of Mtlbn in IP/MS studies (Fig. 2 and Dataset S1), are similarly increased in response to TAC (40, 41). The mechanisms that directly regulate Mtlbn expression are currently unknown and will be a focus of future studies. The discovery of Mtlbn as a component of complex III SCs may add an important level of detail to how this intricate metabolic regulatory process is coordinated.

Typical to mitochondrial syndromes, human patients with complex III deficiencies and defects have highly variable phenotypes with a wide range of clinical presentations, including mild muscle weakness, exercise intolerance, and cardiomyopathy (42). It is thus not surprising that despite the reduced complex III activity and altered metabolomic profile observed in Mtlbn null mice, the animals do not develop a significant cardiac phenotype at baseline. Future studies will directly test the role of Mtlbn in contributing to mitochondrial function in response to cardiac stress conditions such as TAC and ischemia/reperfusion with the anticipation that Mtlbn knockout mice will have a more exaggerated disease phenotype. Conversely, it is interesting that Mtlbn TG mice exhibit a robust cardiac phenotype at baseline despite having a seemingly normal distribution of mitochondrial SCs and intact complex III activity. While the cardiomyopathy observed in Mtlbn TG mice can be generally explained by an alteration in mitochondrial redox state, the initial insult that drives this phenotype has not yet been determined. One potential contributing factor could be increased ROS production by complex III in Mtlbn transgenic mice, which could drive an oxidative stress phenotype and negatively impact NADH oxidase activity (43); however, this has not yet been conclusively shown to be a primary driver of disease in Mtlbn TG mice.

While this work was in progress, an independent group used a series of in vitro studies to identify the Mtlbn microprotein as a mitochondrial protein (Mm47) that is required for activation of the Nlrp3 inflammasome, which is a critical component of the innate immune system (44). While the mechanism of action for Mtlbn was not described in this study, and a knockout mouse model was not analyzed, it is interesting to speculate that the cellular phenotype observed may be due to altered complex III assembly and supercomplex dynamics, which could prevent coordinated activation of the inflammasome in response to stress (45–47). In future studies, we will explore this directly using our Mtlbn gene–deleted mice.

Mammalian mitochondria are thought to contain ∼1,500 different proteins (48, 49), many of which are assembled into distinct complexes in the IMM and OMM. Interestingly, a large fraction of annotated proteins that are <100 amino acids in length (i.e., microproteins) are mitochondrial proteins and include important respiratory chain complex components and their associated assembly factors (49–52). A recent study analyzing the translational landscape of the human heart employed both experimental and computational analyses to search for putative cardiac-expressed microproteins and found that many of these potential proteins may be associated with mitochondria (53). Their findings indicated strong microprotein coregulation with ETC subunits, including multiple small accessory proteins, whose expression patterns are altered in heart failure (53). Numerous human mutations in mitochondrial assembly factors have been shown to contribute to respiratory chain dysfunction and disease (54), and given that there are currently many clinical cases of mitochondrial deficiency with no known genetic cause, it is possible that a subset of these may be due to mutations in microprotein assembly factors that have yet to be discovered. Mitochondrial defects are also a common cause of pediatric cardiomyopathy (55, 56), and cardiac dysfunction is frequent in children with mitochondrial disease and is associated with increased mortality (57). This area of investigation is still poorly understood, and the genetic risk factors associated with these diseases have yet to be fully elucidated (58), suggesting that defects in newly identified or currently unknown microproteins may be contributing factors.

In summary, we have shown that the cardiac-enriched microprotein Mtlbn localizes to the inner mitochondrial membrane where it interacts with respiratory complex III and contributes to its assembly and activity. Given that the processes that regulate SC formation and how SCs influence overall mitochondrial function are not well understood, the discovery of Mtlbn as a component of complex III SCs adds detail to this highly regulated metabolic process. Notably, our work also highlights the importance of further analyzing the noncoding RNA landscape for sORFs that code for putative functional microproteins, which likely play important regulatory roles in mitochondrial processes and contribute to cellular homeostasis.

Materials and MethodsA full description of experimental materials and methods is provided in SI Appendix, Supplementary Materials and Methods.Mitochondrial Isolation and Subfractionation.Cultured cells or isolated mouse heart tissue were homogenized with a Teflon Potter dounce in mitochondrial isolation buffer (75 mM sucrose, 225 mM mannitol, 5 mM Hepes, and 1 mM ethylene glycol-bis(2-aminoethylether)-N,N,N',N'-tetraacetic acid (EGTA), pH 7.4) as previously described (13). As detailed in SI Appendix, a series of centrifugation steps was performed and the resulting mitochondrial pellets were resuspended in assay buffer (125 mM KCl, 20 mM Hepes, 2 mM MgCl2, 2 mM KH2PO4, and 0.04 mM EGTA, pH 7.2). For proteinase K proteolysis experiments, mitochondrial samples were incubated with a range of PK concentrations from 12 ng/mL to 1 mg/mL at 37 °C and then analyzed by gel electrophoresis and Western blotting. For subfractionation experiments, mitochondria were subjected to rounds of swelling/shrinking using hypotonic (10 mM KH2PO4, pH 7.4) and hypertonic (1.8 M sucrose, 10 mM MgCl2, pH 7.4) buffers and then treated with proteinase K at a final concentration of 10 μg/mL and Triton X-100 at a final concentration of 1%.Generation of Mtlbn Knockout and Transgenic Mice.Animal work described in this manuscript has been approved and conducted under the oversight of the Institutional Animal Care and Use Committees at University of Texas Southwestern and Cincinnati Children’s Hospital Medical Center. All mouse lines were generated on a pure C57BL/6 background. Knockout mice were generated using the CRISPR-Cas9 system as previously described (10) targeting exon 2 of Mtlbn. We selected a founder with a 20-base pair deletion that disrupts the Mtlbn ORF and results in a premature stop codon for further analysis. Transgenic mice were derived by pronuclear injection of mouse embryos using the Mtlbn–HA coding sequence driven by the αMHC promoter.BN-PAGE.BN-PAGE was performed as previously described in detail (34). Fifty microgram of mitochondrial protein was prepared in 4 g/g digitonin/Coomassie G-250 brilliant blue protein ratio (G-250 was one-fourth the detergent concentration). Samples were run on NativePAGE 3 to 12% Bis-Tris Protein Gels (Novex) using Novex cathode buffers and transferred to polyvinylidene fluoride (PVDF) membranes and processed for standard Western analysis (detailed in SI Appendix). For Coomassie blue staining, NativePAGE gels were fixed with 50% methanol/10% acetic acid and stained directly with EZBlue Gel Staining Reagent.Mitochondrial Complex III and NADH Oxidase Activity Measurements.Isolated cardiac mitochondria were incubated in 25 mM MOPS (pH 7.4) with 30 mM KCl, 0.5 mM ethylenediaminetetraacetic acid (EDTA), and 0.05% Tween-20 and assayed at room temperature using a cuvette spectrophotometer in kinetic mode (Agilent) as previously described (59, 60). Complex III activity was measured as the rate of antimycin A–sensitive reduction of cytochrome c upon the addition of 5 μM MgCl2, 2 mM KCN, 50 μM cytochrome C, 60 μM decycloubiquinol (freshly reduced), and ±4 μM antimycin A (550 nm, ε = 18,500 M−1cm−1). NADH oxidase activity (coupled activity of complexes I, III, and IV) was measured as the rate of NADH oxidation upon addition of 0.1 mM NADH (340 nm, ε = 6,200 M−1cm−1).LC-MS/MS Metabolomic Analysis.Metabolomic analysis was performed as previously described in detail (61, 62). Mouse heart tissues were homogenized in 80% acetonitrile solution and metabolites were measured with a Thermo Scientific QExactive HF-X hybrid quadrupole orbitrap high-resolution mass spectrometer (HRMS) coupled to a Vanquish ultra-high performance liquid chromatography (UHPLC). For quantitation of GSH and GSSG, mouse heart tissues were homogenized in 80% methanol with 0.1% formic acid to inhibit spontaneous GSH oxidation and analyzed by LC/MS. To calculate GSH and GSSG amounts, sample values were matched to a standard curve with known quantities of GSH and GSSG (63).

Materials and Methods

A full description of experimental materials and methods is provided in SI Appendix, Supplementary Materials and Methods.

Mitochondrial Isolation and Subfractionation.Cultured cells or isolated mouse heart tissue were homogenized with a Teflon Potter dounce in mitochondrial isolation buffer (75 mM sucrose, 225 mM mannitol, 5 mM Hepes, and 1 mM ethylene glycol-bis(2-aminoethylether)-N,N,N',N'-tetraacetic acid (EGTA), pH 7.4) as previously described (13). As detailed in SI Appendix, a series of centrifugation steps was performed and the resulting mitochondrial pellets were resuspended in assay buffer (125 mM KCl, 20 mM Hepes, 2 mM MgCl2, 2 mM KH2PO4, and 0.04 mM EGTA, pH 7.2). For proteinase K proteolysis experiments, mitochondrial samples were incubated with a range of PK concentrations from 12 ng/mL to 1 mg/mL at 37 °C and then analyzed by gel electrophoresis and Western blotting. For subfractionation experiments, mitochondria were subjected to rounds of swelling/shrinking using hypotonic (10 mM KH2PO4, pH 7.4) and hypertonic (1.8 M sucrose, 10 mM MgCl2, pH 7.4) buffers and then treated with proteinase K at a final concentration of 10 μg/mL and Triton X-100 at a final concentration of 1%.

Mitochondrial Isolation and Subfractionation.

Cultured cells or isolated mouse heart tissue were homogenized with a Teflon Potter dounce in mitochondrial isolation buffer (75 mM sucrose, 225 mM mannitol, 5 mM Hepes, and 1 mM ethylene glycol-bis(2-aminoethylether)-N,N,N',N'-tetraacetic acid (EGTA), pH 7.4) as previously described (13). As detailed in SI Appendix, a series of centrifugation steps was performed and the resulting mitochondrial pellets were resuspended in assay buffer (125 mM KCl, 20 mM Hepes, 2 mM MgCl2, 2 mM KH2PO4, and 0.04 mM EGTA, pH 7.2). For proteinase K proteolysis experiments, mitochondrial samples were incubated with a range of PK concentrations from 12 ng/mL to 1 mg/mL at 37 °C and then analyzed by gel electrophoresis and Western blotting. For subfractionation experiments, mitochondria were subjected to rounds of swelling/shrinking using hypotonic (10 mM KH2PO4, pH 7.4) and hypertonic (1.8 M sucrose, 10 mM MgCl2, pH 7.4) buffers and then treated with proteinase K at a final concentration of 10 μg/mL and Triton X-100 at a final concentration of 1%.

Generation of Mtlbn Knockout and Transgenic Mice.Animal work described in this manuscript has been approved and conducted under the oversight of the Institutional Animal Care and Use Committees at University of Texas Southwestern and Cincinnati Children’s Hospital Medical Center. All mouse lines were generated on a pure C57BL/6 background. Knockout mice were generated using the CRISPR-Cas9 system as previously described (10) targeting exon 2 of Mtlbn. We selected a founder with a 20-base pair deletion that disrupts the Mtlbn ORF and results in a premature stop codon for further analysis. Transgenic mice were derived by pronuclear injection of mouse embryos using the Mtlbn–HA coding sequence driven by the αMHC promoter.

Generation of Mtlbn Knockout and Transgenic Mice.

Animal work described in this manuscript has been approved and conducted under the oversight of the Institutional Animal Care and Use Committees at University of Texas Southwestern and Cincinnati Children’s Hospital Medical Center. All mouse lines were generated on a pure C57BL/6 background. Knockout mice were generated using the CRISPR-Cas9 system as previously described (10) targeting exon 2 of Mtlbn. We selected a founder with a 20-base pair deletion that disrupts the Mtlbn ORF and results in a premature stop codon for further analysis. Transgenic mice were derived by pronuclear injection of mouse embryos using the Mtlbn–HA coding sequence driven by the αMHC promoter.

BN-PAGE.BN-PAGE was performed as previously described in detail (34). Fifty microgram of mitochondrial protein was prepared in 4 g/g digitonin/Coomassie G-250 brilliant blue protein ratio (G-250 was one-fourth the detergent concentration). Samples were run on NativePAGE 3 to 12% Bis-Tris Protein Gels (Novex) using Novex cathode buffers and transferred to polyvinylidene fluoride (PVDF) membranes and processed for standard Western analysis (detailed in SI Appendix). For Coomassie blue staining, NativePAGE gels were fixed with 50% methanol/10% acetic acid and stained directly with EZBlue Gel Staining Reagent.

BN-PAGE was performed as previously described in detail (34). Fifty microgram of mitochondrial protein was prepared in 4 g/g digitonin/Coomassie G-250 brilliant blue protein ratio (G-250 was one-fourth the detergent concentration). Samples were run on NativePAGE 3 to 12% Bis-Tris Protein Gels (Novex) using Novex cathode buffers and transferred to polyvinylidene fluoride (PVDF) membranes and processed for standard Western analysis (detailed in SI Appendix). For Coomassie blue staining, NativePAGE gels were fixed with 50% methanol/10% acetic acid and stained directly with EZBlue Gel Staining Reagent.

Mitochondrial Complex III and NADH Oxidase Activity Measurements.Isolated cardiac mitochondria were incubated in 25 mM MOPS (pH 7.4) with 30 mM KCl, 0.5 mM ethylenediaminetetraacetic acid (EDTA), and 0.05% Tween-20 and assayed at room temperature using a cuvette spectrophotometer in kinetic mode (Agilent) as previously described (59, 60). Complex III activity was measured as the rate of antimycin A–sensitive reduction of cytochrome c upon the addition of 5 μM MgCl2, 2 mM KCN, 50 μM cytochrome C, 60 μM decycloubiquinol (freshly reduced), and ±4 μM antimycin A (550 nm, ε = 18,500 M−1cm−1). NADH oxidase activity (coupled activity of complexes I, III, and IV) was measured as the rate of NADH oxidation upon addition of 0.1 mM NADH (340 nm, ε = 6,200 M−1cm−1).

Mitochondrial Complex III and NADH Oxidase Activity Measurements.

Isolated cardiac mitochondria were incubated in 25 mM MOPS (pH 7.4) with 30 mM KCl, 0.5 mM ethylenediaminetetraacetic acid (EDTA), and 0.05% Tween-20 and assayed at room temperature using a cuvette spectrophotometer in kinetic mode (Agilent) as previously described (59, 60). Complex III activity was measured as the rate of antimycin A–sensitive reduction of cytochrome c upon the addition of 5 μM MgCl2, 2 mM KCN, 50 μM cytochrome C, 60 μM decycloubiquinol (freshly reduced), and ±4 μM antimycin A (550 nm, ε = 18,500 M−1cm−1). NADH oxidase activity (coupled activity of complexes I, III, and IV) was measured as the rate of NADH oxidation upon addition of 0.1 mM NADH (340 nm, ε = 6,200 M−1cm−1).

LC-MS/MS Metabolomic Analysis.Metabolomic analysis was performed as previously described in detail (61, 62). Mouse heart tissues were homogenized in 80% acetonitrile solution and metabolites were measured with a Thermo Scientific QExactive HF-X hybrid quadrupole orbitrap high-resolution mass spectrometer (HRMS) coupled to a Vanquish ultra-high performance liquid chromatography (UHPLC). For quantitation of GSH and GSSG, mouse heart tissues were homogenized in 80% methanol with 0.1% formic acid to inhibit spontaneous GSH oxidation and analyzed by LC/MS. To calculate GSH and GSSG amounts, sample values were matched to a standard curve with known quantities of GSH and GSSG (63).

LC-MS/MS Metabolomic Analysis.

Metabolomic analysis was performed as previously described in detail (61, 62). Mouse heart tissues were homogenized in 80% acetonitrile solution and metabolites were measured with a Thermo Scientific QExactive HF-X hybrid quadrupole orbitrap high-resolution mass spectrometer (HRMS) coupled to a Vanquish ultra-high performance liquid chromatography (UHPLC). For quantitation of GSH and GSSG, mouse heart tissues were homogenized in 80% methanol with 0.1% formic acid to inhibit spontaneous GSH oxidation and analyzed by LC/MS. To calculate GSH and GSSG amounts, sample values were matched to a standard curve with known quantities of GSH and GSSG (63).

Supplementary MaterialSupplementary FileSupplementary FileSupplementary File

Supplementary Material
