---
pmid: '11927608'
title: The Maguk protein, Pals1, functions as an adapter, linking mammalian homologues
  of Crumbs and Discs Lost.
authors:
- Roh MH
- Makarova O
- Liu CJ
- Shin K
- Lee S
- Laurinec S
- Goyal M
- Wiggins R
- Margolis B
journal: J Cell Biol
year: '2002'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2173254
doi: 10.1083/jcb.200109010
---

# The Maguk protein, Pals1, functions as an adapter, linking mammalian homologues of Crumbs and Discs Lost.
**Authors:** Roh MH, Makarova O, Liu CJ, Shin K, Lee S, Laurinec S, Goyal M, Wiggins R, Margolis B
**Journal:** J Cell Biol (2002)
**DOI:** [10.1083/jcb.200109010](https://doi.org/10.1083/jcb.200109010)
**PMC:** [PMC2173254](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2173254/)

## Abstract

1. J Cell Biol. 2002 Apr 1;157(1):161-72. doi: 10.1083/jcb.200109010. Epub 2002
Apr  1.

The Maguk protein, Pals1, functions as an adapter, linking mammalian homologues 
of Crumbs and Discs Lost.

Roh MH(1), Makarova O, Liu CJ, Shin K, Lee S, Laurinec S, Goyal M, Wiggins R, 
Margolis B.

Author information:
(1)Department of Biological Chemistry, University of Michigan Medical School, 
Ann Arbor, MI 48109, USA.

Membrane-associated guanylate kinase (Maguk) proteins are scaffold proteins that 
contain PSD-95-Discs Large-zona occludens-1 (PDZ), Src homology 3, and guanylate 
kinase domains. A subset of Maguk proteins, such as mLin-2 and protein 
associated with Lin-7 (Pals)1, also contain two L27 domains: an L27C domain that 
binds mLin-7 and an L27N domain of unknown function. Here, we demonstrate that 
the L27N domain targets Pals1 to tight junctions by binding to a PDZ domain 
protein, Pals1-associated tight junction (PATJ) protein, via a unique Maguk 
recruitment domain. PATJ is a homologue of Drosophila Discs Lost, a protein that 
is crucial for epithelial polarity and that exists in a complex with the apical 
polarity determinant, Crumbs. PATJ and a human Crumbs homologue, CRB1, 
colocalize with Pals1 to tight junctions, and CRB1 interacts with PATJ albeit 
indirectly via binding the Pals1 PDZ domain. In agreement, we find that a 
Drosophila homologue of Pals1 participates in identical interactions with 
Drosophila Crumbs and Discs Lost. This Drosophila Pals1 homologue has been 
demonstrated recently to represent Stardust, a crucial polarity gene in 
Drosophila. Thus, our data identifies a new multiprotein complex that appears to 
be evolutionarily conserved and likely plays an important role in protein 
targeting and cell polarity.

DOI: 10.1083/jcb.200109010
PMCID: PMC2173254
PMID: 11927608 [Indexed for MEDLINE]

## Full Text

IntroductionThe asymmetric localization of macromolecules and organelles represents the central feature of cell polarity. Establishment and maintenance of polarity relies on polarized vesicle trafficking and precise targeting of proteins to discrete membrane subdomains in response to extracellular and spatial cues. In turn, the composition of these membrane subdomains is determined in part by scaffolding proteins that contain distinct protein domains. For example, many studies have demonstrated an important role for proteins containing the PSD-95–Discs Large–zona occludens (ZO)*-1 (PDZ) domain during cell polarization (Sheng and Sala, 2001). One of the major groups of PDZ proteins is the membrane-associated guanylate kinase (Maguk) proteins. These proteins usually contain a noncatalytic guanylate kinase (GUK) domain in combination with PDZ and Src homology (SH)3 domains. Many also contain a 4.1B or hook domain that allow these Maguk proteins to bind members of the protein 4.1 superfamily, such as protein 4.1, ezrin, radixin, and moesin (Lue et al., 1994; Cohen et al., 1998). Maguk proteins have been extensively studied for their role in targeting proteins in polarized cells, such as neurons and epithelia (Fanning and Anderson, 1999). One such targeting system utilizes the Lin-2, Lin-7, and Lin-10 proteins first identified in Caenorhabditis elegans (Hoskins et al., 1996; Kaech et al., 1998). In worm vulval precursor epithelia, mutations in any of these proteins result in the mislocalization of the growth factor receptor, Let23, and in vulval agenesis (Kaech et al., 1998).Mammalian homologues of these proteins have been described and are referred to as mLin-2–CASK, mLin-7–Veli–Mals, and mLin-10–X11α–Mint-1 (Hata et al., 1996; Butz et al., 1998; Cohen et al., 1998; Okamoto and Sudhof, 1998; Borg et al., 1999; Irie et al., 1999; Jo et al., 1999). In mammalian brain, mLin-7 is bound to mLin-2, which is in turn bound to mLin-10 (Butz et al., 1998; Kaech et al., 1998; Borg et al., 1999). This complex likely plays a role in targeting N-methyl-d-aspartic acid receptors in mammalian neurons (Jo et al., 1999). Recent work has suggested that mLin-10 also binds a kinesin, Kif17, explaining how complexes containing mLin-10–mLin-7–mLin-2 may target proteins in neurons (Setou et al., 2000).Studies have also examined the mLin-2 and mLin-7 proteins in epithelial cells. In these cells, mLin-2 is bound to mLin-7, but there appears to be no Lin-10 homologue in this complex (Borg et al., 1998). Work has shown that mLin-7 plays a role in the retention of the betaine–GABA transporter-1 at the basolateral surface and binds β-catenin in renal epithelial cells (Perego et al., 1999, 2000). We have also shown that mLin-7 and its binding partners play a role in endocytic trafficking (Straight et al., 2001). Further data has shown that mLin-2 is not the only partner for mLin-7 in cells. Several Maguk proteins have been identified as mLin-7 binding partners, including novel proteins that we call proteins associated with Lin-7 (Pals; Kamberov et al., 2000). Additional Maguk binding partners for mLin-7 have also been reported (Butz et al., 1998; Tseng et al., 2001). Our group has extensively studied the interactions of mLin-7 with these different Maguk proteins and identified a domain, the L27 domain, that mediates these interactions (Doerks et al., 2000). Lin-7 contains a single L27 domain, whereas this domain is duplicated in the mLin-7 binding Maguk proteins. We term these two domains L27N and L27C, formerly referred to as L27A and L27B, respectively (Doerks et al., 2000). The COOH-terminal L27 domain (L27C), which lies between the NH2-terminal L27 domain (L27N) and the PDZ domain, binds the L27 domain of mLin-7. The function and binding partners of the L27N domains remain unclear. In this paper, we describe an important function for the L27N domain in the targeting of the Pals1 protein and the assembly of an evolutionarily conserved complex at tight junctions.

Introduction

The asymmetric localization of macromolecules and organelles represents the central feature of cell polarity. Establishment and maintenance of polarity relies on polarized vesicle trafficking and precise targeting of proteins to discrete membrane subdomains in response to extracellular and spatial cues. In turn, the composition of these membrane subdomains is determined in part by scaffolding proteins that contain distinct protein domains. For example, many studies have demonstrated an important role for proteins containing the PSD-95–Discs Large–zona occludens (ZO)*-1 (PDZ) domain during cell polarization (Sheng and Sala, 2001). One of the major groups of PDZ proteins is the membrane-associated guanylate kinase (Maguk) proteins. These proteins usually contain a noncatalytic guanylate kinase (GUK) domain in combination with PDZ and Src homology (SH)3 domains. Many also contain a 4.1B or hook domain that allow these Maguk proteins to bind members of the protein 4.1 superfamily, such as protein 4.1, ezrin, radixin, and moesin (Lue et al., 1994; Cohen et al., 1998). Maguk proteins have been extensively studied for their role in targeting proteins in polarized cells, such as neurons and epithelia (Fanning and Anderson, 1999). One such targeting system utilizes the Lin-2, Lin-7, and Lin-10 proteins first identified in Caenorhabditis elegans (Hoskins et al., 1996; Kaech et al., 1998). In worm vulval precursor epithelia, mutations in any of these proteins result in the mislocalization of the growth factor receptor, Let23, and in vulval agenesis (Kaech et al., 1998).

Mammalian homologues of these proteins have been described and are referred to as mLin-2–CASK, mLin-7–Veli–Mals, and mLin-10–X11α–Mint-1 (Hata et al., 1996; Butz et al., 1998; Cohen et al., 1998; Okamoto and Sudhof, 1998; Borg et al., 1999; Irie et al., 1999; Jo et al., 1999). In mammalian brain, mLin-7 is bound to mLin-2, which is in turn bound to mLin-10 (Butz et al., 1998; Kaech et al., 1998; Borg et al., 1999). This complex likely plays a role in targeting N-methyl-d-aspartic acid receptors in mammalian neurons (Jo et al., 1999). Recent work has suggested that mLin-10 also binds a kinesin, Kif17, explaining how complexes containing mLin-10–mLin-7–mLin-2 may target proteins in neurons (Setou et al., 2000).

Studies have also examined the mLin-2 and mLin-7 proteins in epithelial cells. In these cells, mLin-2 is bound to mLin-7, but there appears to be no Lin-10 homologue in this complex (Borg et al., 1998). Work has shown that mLin-7 plays a role in the retention of the betaine–GABA transporter-1 at the basolateral surface and binds β-catenin in renal epithelial cells (Perego et al., 1999, 2000). We have also shown that mLin-7 and its binding partners play a role in endocytic trafficking (Straight et al., 2001). Further data has shown that mLin-2 is not the only partner for mLin-7 in cells. Several Maguk proteins have been identified as mLin-7 binding partners, including novel proteins that we call proteins associated with Lin-7 (Pals; Kamberov et al., 2000). Additional Maguk binding partners for mLin-7 have also been reported (Butz et al., 1998; Tseng et al., 2001). Our group has extensively studied the interactions of mLin-7 with these different Maguk proteins and identified a domain, the L27 domain, that mediates these interactions (Doerks et al., 2000). Lin-7 contains a single L27 domain, whereas this domain is duplicated in the mLin-7 binding Maguk proteins. We term these two domains L27N and L27C, formerly referred to as L27A and L27B, respectively (Doerks et al., 2000). The COOH-terminal L27 domain (L27C), which lies between the NH2-terminal L27 domain (L27N) and the PDZ domain, binds the L27 domain of mLin-7. The function and binding partners of the L27N domains remain unclear. In this paper, we describe an important function for the L27N domain in the targeting of the Pals1 protein and the assembly of an evolutionarily conserved complex at tight junctions.

ResultsThe L27N domain targets Pals1 to the tight junctionIn MDCK cells, mLin-7 has several partners, one of which is the Pals1 protein (Kamberov et al., 2000). In addition to containing an SH3, PDZ, and GUK domain, Pals1 also bears two L27 domains (Fig. 1 A). The extreme NH2-terminal 125 amino acid region bears no significant similarity to other proteins, and we refer to it as the unknown 1 (U1) domain. Our previous studies had shown that mLin-2 localizes to the lateral surface of MDCK cells where it overlaps with E-cadherin (Straight et al., 2001). In contrast, we find that Pals1 colocalizes with ZO-1 to tight junctions in both MDCK cells (Fig. 1 B) and in rat kidney (Fig. 1 C).Figure 1.
Targeting of Pals1 in MDCK cells. (A) Pals1 has multiple protein–protein interaction domains including two Lin-2/Lin-7 (L27N and L27C) domains, a PDZ domain, SH3 domain, protein 4.1 binding (4.1B) domain, and GUK domain. (B) MDCK cells were immunostained as described in Materials and methods. The square panels represent digital photomicrographs (x-y dimension of the z-series), and the rectangular panels show the x-z dimension (z-section). (C) Rat kidney sections were immunostained as described in Materials and methods.We next sought to determine the domains of Pals1 important for the targeting of this protein to cell contacts (Fig. 2 A). Pals1 harboring deletions of the SH3, 4.1B, or GUK domain all targeted to sites of cell–cell adhesion but with decreased efficiency (Fig. 2 B). In fact, when all of these domains were deleted the resulting Pals1 (1–346) protein still targeted to sites of cell adhesion. Deletion of the U1 and both L27 domains resulted in the mistargeting of Pals1 (207–end) away from sites of cell contacts (Fig. 2 B). In contrast, we found that the U1 and L27N domains were sufficient for targeting Pals1 to tight junctions (Fig. 2 D).Figure 2.
Localization efficiency of EYFP-tagged Pals1 deletion mutants. (A) Schematic representation of the different Pals1-EYFP constructs. Numbers overlying the dotted lines refer to the amino acid positions flanking the internal domain deletions. (B) Localization of wild-type Pals1 and Pals1 deletion mutants in MDCK cells. Cells expressing the various constructs shown in A were fixed and directly visualized. (C) Lysates derived from HEK293 cells expressing the wild-type Pals1-EYFP and the various mutant constructs were resolved by SDS-PAGE. The chimeric protein in which the L27N domain of Pals2 was replaced by the Pals1 L27N domain is denoted as PALS2_PALS1L27N (Fig. 3). The Pals1-EYFP protein harboring a double point mutation in the L27N domain is denoted as PALS1L150GV154G (Fig. 3). (D) Immunostaining of MDCK cells expressing Myc-Pals1 (1–182), which contain only the U1 and L27N domains.To further understand the role of this NH2 terminal region, including the U1 and L27 domains, we made a series of deletions and mutations in this region (Fig. 3 A). Deleting the U1 domain had no effect on the targeting of Pals1 to tight junctions (Fig. 3 B). Similarly, deleting the L27C domain had no effect on Pals1 targeting, indicating that this targeting is not dependent on the interaction between Pals1 and mLin-7. In contrast, mutating or deleting the L27N domain redirected Pals1 from tight junctions to the apical surface (Fig. 3, B and C). To further investigate the role of the Pals1 L27N domain in protein targeting, a chimera was generated whereby this domain was used to replace the L27N domain of Pals2 (Fig. 3 D). Pals2 is a Maguk protein like Pals1 (Fig. 3 D) but is missing the U1 domain extension (Kamberov et al., 2000). In the conserved region that extends from the L27N to the GUK domain, Pals2 is ∼35% identical to Pals1. Whereas Pals2 colocalizes with E-cadherin below tight junctions, this chimeric protein (Pals2_Pals1L27N) targets to tight junctions (Fig. 3 E). When we removed the Pals1 L27N domain from the chimera and reintroduced the Pals2 L27N domain (Pals2ΔPals2L27N), the resulting protein returned to the lateral surface. This confirmed the role of the Pals1 L27N domain in the targeting of proteins to the tight junction.Figure 3.
The L27N domain of Pals1 targets proteins to tight junctions. (A) Schematic diagram of the deletion and point mutant versions of Pals1. Asterisks denote the double point mutation in the L27N domain (L150G;V154G). (B) Examination of the distribution of Pals1 mutants displayed in A via fluorescence microscopy. (C) Z-section images of MDCK cells expressing wild-type Pals1 or L27N domain mutant Pals1 proteins (green). Red staining represents either ZO-1 or E-cadherin as indicated. (D) Schematic diagram of the wild-type Pals2-EYFP protein and the chimeric protein where the L27N domain of Pals2 is replaced by the Pals1 L27N domain (Pals2_Pals1L27N). We also generated a construct (unpublished data) in which we removed the Pals1 L27N domain from Pals2_Pals1L27N chimera and replaced it with the original Pals2 L27N domain (Pals2_Pals2L27N). (E) Z-section images of MDCK cells expressing Pals1, Pals2, Pals2_Pals1L27N, and Pals2_Pals2L27N proteins. Pals proteins are shown in green. Red staining represents either ZO-1 or E-cadherin.The Pals1 L27N domain is a protein–protein interaction domainPrevious work had shown that the L27C domain of Pals1 is necessary for Pals1 to bind mLin-7 (Kamberov et al., 2000). However, our data indicate that the L27N domains of all the Pals family members do not bind mLin-7 and do not appear to be involved in homo- or heterodimerization with other L27 domains (Lee et al., 2002). Accordingly, we purified and radiolabeled a glutathione S-transferase (GST) fusion protein of the Pals1 L27N domain and performed Far Western blotting to look for a possible binding partner that might mediate the targeting of Pals1 to tight junctions. The Far Western probe bound to a protein of ∼200 kD in tissues and cell lines including rat brain, mouse brain, MDCK, and 293 cells (Fig. 4 A). Anti–mLin-7 and anti-Myc antibodies could coimmunoprecipitate the 200-kD protein in MDCK and 293 cells expressing Myc-Pals1 (Fig. 4 B), suggesting that this 200-kD protein is in a complex with Pals1 and mLin-7.Figure 4.
The Pals1 L27N domain binds a protein of ∼200 kD. (A) Lysates prepared from rat brain, mouse brain, MDCK cells, and 293 cells were resolved by SDS-PAGE. Far Western blotting with 32P-labeled Pals1 L27N probe was performed. (B) Coimmunoprecipitation experiments using antibodies against Myc or mLin-7 and lysates derived from MDCK and 293 cells expressing Myc-Pals1 were performed. Precipitated proteins, resolved by SDS-PAGE, were probed with 32P-labeled Pals1 L27N domain (top) and immunoblotted with anti-Myc antibody (bottom). The arrowhead points to the IgG heavy chain. (C) The indicated EYFP- and Myc-tagged Pals1 proteins expressed in 293 cells were immunoprecipitated, and bound proteins were resolved by SDS-PAGE. Immunoblotting with anti-EYFP and anti-Myc antibodies confirmed the expression of the Pals constructs (top). Far Western blotting was performed with 32P-labeled Pals1 L27N probe (bottom). Pals1DM-EYFP refers to EYFP-Pals1 (L150G;V154G), whereas P2P1-EYFP refers to the Pals2_Pals1L27N chimeric protein.Next, we examined the interaction of the 200-kD protein with the different EYFP-Pals1 mutants described previously. Far Western blotting confirmed that the 200-kD protein coimmunoprecipitated with EYFP-Pals1 and EYFP-Pals1(1–182) expressed in 293 cells (Fig. 4 C). Pals1 missing or containing a double point mutation in the L27N domain could not interact with this protein. Likewise, Pals2 did not interact with this 200-kD protein. However, a chimera of Pals2 containing the L27N domain of Pals1 did interact with the 200-kD protein. As expected, we showed that the 200-kD protein coimmunoprecipitated with Myc-Pals1 but not with the Myc-Pals1 protein missing the L27N domain (Fig. 4 C). The targeting of Pals1 to tight junctions correlated exactly with its ability to bind to this 200-kD protein.Identification of a novel 200-kD protein interacting with the L27N domain of Pals1To identify the 200-kD protein, we performed large scale coimmunoprecipitations using lysates derived from MDCK cell lines expressing Myc-Pals1 and Myc-Pals1 missing the U1 and L27N domains. The precipitated proteins were resolved electrophoretically and visualized by silver staining (Fig. 5 A) and colloidal Coomassie staining. We could identify a ∼200-kD protein that was present in the Myc-Pals1 immunoprecipitates but absent from the Myc-Pals1Δ(U1 + L27N) immunoprecipitates. We excised this band from the Coomassie-stained gel and determined its identity by MS–MS sequence analysis. The peptide sequences obtained matched those found in a PDZ protein identified previously as a human homologue of the Drosophila protein INAD (hINADL) Table I and Philipp and Flockerzi, 1997). Additional peptide sequences corresponding to PDZ domains not found in hINADL (PDZ domain protein 3′ variant 4 [Table I]) were also identified. Using the peptide sequences and EST database, we cloned the cDNA encoding the full-length Pals1 binding partner into mammalian expression vectors.Figure 5.
Purification and identification of the 200 kD binding partner of the Pals1 L27N domain. (A) Large scale Myc-Pals1 and Myc-Pals1Δ(U1 + L27N) coimmunoprecipitations were performed. 5% of the purified proteins were resolved by SDS-PAGE and visualized by silver staining. The arrowhead points to the band representing the 200 kD Pals1 binding partner. (B) Structural alignment between MUPP1, PATJ, and Discs Lost (DLT). Percentage amino acid sequence identity are displayed between the respective MRE domains and PDZ domains (circles and rectangles, respectively). (C) Sequence alignment of the conserved MRE domains in PATJ, MUPP1, C. elegans Discs Large-1 (CeDLG1), SAP97, and Discs Lost (DLT). Identical residues seen in at least three of the five MRE domains are in bold. The sequence data for PATJ have been submitted to GenBank (sequence data available from GenBank/EMBL/DDBJ under accession no. AF397170.Table I.
Peptide sequences obtained from the ∼200 kD binding partner for the L27N domain of Pals1
Peptide sequences obtainedIdentitySequence data available fromGenBank/EMBL/DDBJ under accession no.DTPLNAIVIHEVYEEGAAARhINADL
AJ224747
EPNVSLGISIVGGQTVIKhINADL
AJ224747
GLGFSILDYQDPLDPTRhINADL
AJ224747
LQVLQVLDRhINADL
AJ224747
RLFDDEASVDEPRhINADL
AJ224747
NAGQVVHLTLVRhINADL
AJ224747
VPDSPENELKhINADL
AJ224747
LQTGDHILKhINADL
AJ224747
TIVPGGLADRDGRhINADL
AJ224747
DGQSLGIRhINADL
AJ224747
SQNLGKVDIFVKhINADL
AJ224747
IGDELLEINNQILYGRhINADL
AJ224747
QVLEDSPAGKhINADL
AJ224747
EAVSFLKhINADL
AJ224747
GLGLSIVGKhINADL
AJ224747
LIQGDQILSVNGEDMRPDZ domain protein 3′ variant 4
AB044807
NGSGVFISDIVKPDZ domain protein 3′ variant 4
AB044807(
We found that this protein consists of 10 PDZ domains and that hINADL represents an incomplete version of this protein (Fig. 5 B). The full-length protein consists of PDZ domains found in both hINADL and PDZ domain protein 3′ variant 4. Pals1-associated tight junction (PATJ) protein is the paralogue of multiple PDZ protein (MUPP)1 and is the ortholog of Drosophila Discs Lost (Ullmer et al., 1998; Bhat et al., 1999; Mancini et al., 2000). The PDZ domains of PATJ, MUPP1, and Discs Lost are highly conserved (Fig. 5 B). Because the closest ortholog of hINADL in Drosophila is not INAD, we have decided to rename this protein PATJ protein in agreement with its ability to recruit Pals1 to tight junctions. As predicted, when an EYFP–PATJ fusion protein was expressed in MDCK cells its localization was coincident with Pals1 and ZO-1 at tight junctions (Fig. 6 A) and superior to E-cadherin (Fig. 6 B).Figure 6.
PATJ localizes to tight junctions in MDCK cells. MDCK cells expressing full-length EYFP-PATJ were costained with either anti-Pals1 and anti–ZO-1 antibodies (A) or antiuvomorulin antibody (B) and examined by confocal microscopy.A novel evolutionarily conserved domain in PATJ targets Pals1 to tight junctionsWhen expressed in 293 cells, full-length PATJ was observed to be an ∼190-kD protein that coimmunoprecipitated with Pals1 but not Pals1 missing its L27N domain (Fig. 7 A). We next sought to determine the region in PATJ that was responsible for binding the Pals1 L27N domain. GST fused to the first 238, 109, and 68 residues of PATJ could bind full-length Pals1 (Fig. 7 B). These fusion proteins all contain the ∼50 amino acid domain conserved in PATJ, MUPP1, and Discs Lost (Fig. 5 C). We also found that the NH2-terminal 45 residues of PATJ could interact with Pals1 albeit weakly. None of the GST–PATJ fusion proteins could bind the Pals1ΔL27N mutant protein. A GST–mLin-7 fusion protein was able to bind both Pals1 and Pals1ΔL27N, since both contain an intact L27C domain (positive control). Thus, we now report an evolutionarily conserved region contained within the NH2 terminus of PATJ as a novel protein–protein interaction domain. In light of its ability to bind and recruit the Maguk protein Pals1 to tight junctions, we have decided to name it the Maguk recruitment (MRE) domain. Our preliminary data suggest that although the MUPP1 MRE domain also binds to the Pals1 L27N domain, the Discs Lost MRE domain fails to interact with Pals1 (unpublished data).Figure 7.
PATJ interacts with the L27N domain of Pals1. (A) Full-length HA epitope-tagged PATJ was coexpressed in 293 cells with either Myc-Pals1 or Myc-Pals1 missing the U1 and L27N domains. Anti-Myc immunoprecipitates from both lysates were resolved by SDS-PAGE. Immunoblotting reveals that HA-PATJ coprecipitates only with full-length Myc-Pals1. (B) GST-PATJ pulldown assays were performed using 293 lysates containing overexpressed Myc-Pals1 or Myc-Pals1ΔL27N. Numbers in parentheses denote the amino acids of PATJ fused to GST. (C) Immunoprecipitations using preimmune serum or anti–mLin-7 antibodies, and lysates from the indicated cell lines were performed. Precipitated proteins were resolved by SDS-PAGE and probed with 32P-labeled GST-PATJ (1–238). (D) MDCK cell pools expressing Myc-PATJ (1–238) were examined by confocal microscopy. Cells were costained with anti-Pals1 and anti-Myc antibodies.To determine if additional binding partners for the PATJ MRE domain existed, a radiolabeled GST–PATJ (1–238) probe was used in Far Western blot overlay experiments. In 293, MDCK, HeLa, and HT29 cells, two proteins at ∼71 kD were recognized by the probe (Fig. 7 C). These bands were also observed in mLin-7 coimmunoprecipitates. Immunoblotting with antibodies raised against the COOH terminus of Pals1 recognizes two bands at ∼71 kD (Fig. 8 D), suggesting that Pals1 is the predominant partner for the PATJ MRE domain. A third band representing a smaller unidentified protein was also observed in HeLa cells.Figure 8.
Full-length CRB1 targets to tight junctions via the interaction between its COOH terminus and Pals1. (A) CRB1 was identified in the critical chromosomal region linked to a severe form of autosomal recessive Retinitis Pigmentosa, RP12. The published domain structure of CRB1 and the protein fragment encoded by the human EST BF912191 are schematically shown. The full-length Myc-CRB1 construct used in this study is also shown. The open rectangle and filled circle represent the signal peptide and the Myc tag, respectively. Rectangles containing horizontal and vertical lines correspond to the EGF-like and C-type lectin domains, respectively. The laminin A G-like domains and the transmembrane region are represented by filled pentagons and the filled rectangle, respectively. (B) Comparison of the transmembrane segments and cytoplasmic tails of Drosophila Crumbs (CRB) and full-length CRB1. Identical residues are in bold. Underlined residues were fused to GST for GST pulldown experiments. (C) MDCK cells stably expressing Myc-CRB1 were immunostained with anti-Myc antibody and antibodies directed against either ZO-1 or Pals1. (D) Immunoprecipitations with preimmune serum (Control IP), anti-PATJ serum, and anti-Myc ascites were performed on lysates derived from the cells in C. Endogenous PATJ and Pals1 were detected by immunoblotting with the respective antisera. CRB1 was detected by blotting with anti-Myc antibody. (E) GST was fused to the COOH-terminal residues of Drosophila Crumbs (GST-CRB) or human CRB1 (GST-CRB1) as indicated in B. 293 lysates containing indicated proteins were incubated with GST-CRB and GST-CRB1 beads. Adsorbed proteins were resolved by SDS-PAGE and detected by immunoblotting with anti-HA and anti-Myc antibodies.To confirm that the interaction between the PATJ MRE domain and the Pals1 L27N domain was a physiologically relevant interaction, we expressed the first 238 residues of PATJ in MDCK cells. PATJ (1–238) consists of the MRE and first PDZ domains of PATJ. When expressed in MDCK cells, Myc-PATJ (1–238) exhibits a diffuse localization pattern (Fig. 7 D). In cells expressing this dominant negative PATJ protein, endogenous Pals1 was absent from tight junctions. However, in the surrounding cells Pals1 was present at tight junctions. Although adherens and tight junctions were not affected by the expression of this dominant negative protein, (as determined by E-cadherin and ZO-1 immunostaining [unpublished data]), these results confirm the role of PATJ in targeting Pals1 to tight junctions.Human Crumbs exists in a ternary complex with Pals1 and PATJ at tight junctionsIn Drosophila, Discs Lost has been reported to bind the COOH terminus of Crumbs. A recent study has linked a human Crumbs homologue, CRB1, to a visual disorder, Retinitis Pigmentosa. Drosophila Crumbs (CRB) contains a signal peptide and a transmembrane segment; however, CRB1 was reported to lack the transmembrane segment (den Hollander et al., 1999). Consequently, the authors proposed that CRB1 was a secreted protein. However, they raised the possibility that their CRB1 cDNA sequence may have been incomplete at the 3′ end. To resolve this issue, we searched the human EST database for ESTs encoding the last EGF-like domain of CRB1. We identified an EST obtained from a dendritic oligodendroglioma (sequence data available from GenBank/EMBL/DDBJ under accession no. BG912191), which encoded for the last three extracellular EGF-like domains, a single C-type lectin domain, a transmembrane segment, and a 37 amino acid tail that was 62% identical to that of CRB (Fig. 8, A and B). Therefore, it seemed likely that a longer CRB1 cDNA existed and that it encoded for an integral membrane protein. Based on this notion, we amplified this full-length cDNA by PCR and cloned it into mammalian expression vectors. When full-length Myc-CRB1 (Fig. 8 A) was expressed in MDCK cells, it was observed to colocalize with Pals1 and ZO-1 at tight junctions (Fig. 8 C). We could also demonstrate that the ability of CRB1 to target to tight junctions correlated with its ability to exist in a ternary complex with endogenous PATJ and Pals1 (Fig. 8 D). We find that CRB1 may not be only expressed in neuronal tissues, since we can detect CRB1 DNA fragments using template cDNA derived from epithelial tissues such as kidney and colon (Fig. S1 available at http://www.jcb.org/cgi/content/full/jcb.200109010/DC1). It is also important to note that there are several other isoforms of Crumbs identified in the human genome that have similar transmembrane segments and COOH termini as CRB1 (Tepass et al., 2001).To determine if the CRB and CRB1 COOH termini could bind PATJ and/or Pals1, we performed GST pulldown assays using GST fused to the tails of CRB and CRB1 (GST-CRB and GST-CRB1, respectively). Both were able to coprecipitate Myc-Pals1 and hemagglutinin (HA)-PATJ expressed in 293 cells (Fig. 8 E). When GST-CRB and GST-CRB1 were incubated with HA-PATJ and Myc-Pals1 missing its PDZ domain, a significantly lower amount of HA-PATJ was precipitated. Based on this result, the possibility still remains that GST-CRB and GST-CRB1 interact directly with PATJ. Alternatively, this result could be explained by the presence of endogenous Pals1 in the 293 lysate mediating the indirect interaction between GST-CRB and GST-CRB1 and HA-PATJ. To distinguish between these two possibilities, the GST fusion proteins were incubated with Myc-Pals1 and HA-PATJ missing its MRE domain. HA-PATJΔMRE should not be able to bind either Myc-Pals1 or endogenous Pals1 in the 293 lysate. Both fusion proteins were able to bind Myc-Pals1 but not HA-PATJΔMRE (Fig. 8 E). This suggests that in mammalian epithelia Pals1 serves as an adaptor protein, mediating the indirect interaction between CRB1 and PATJ.The CRB1–Pals1–PATJ complex is evolutionarily conserved in invertebratesNext, we searched the Drosophila genome for an ortholog of Pals1. We were able to identify a putative Maguk protein, CG1617, that was highly related to Pals1. To determine if Drosophila Crumbs could bind the PDZ domain of CG1617, we performed GST-CRB pulldown assays in the presence of CG1617 fragments (Fig. 9 A). GST-CRB was able to bind CG1617 in a PDZ-dependent fashion; in contrast, GST-CRB failed to bind full-length Discs Lost (Fig. 9 B).Figure 9.
The
Drosophila
Pals1 homologue binds to both Crumbs and Discs Lost via its PDZ and L27 domains, respectively. (A) Schematic diagrams of the Drosophila Pals1 homologue, CG1617, and CG1617 fragments used in B and C are shown. All of these proteins were Myc-epitope tagged. (B) GST-CRB pulldowns were performed using 293 cell lysates containing the indicated expressed proteins. (C) 293 lysates containing the denoted Myc-tagged CG1617 proteins were incubated with either GST or GST fused to the NH2-terminal 135 residues of Discs Lost (DLT), which encompasses the MRE domain. In B and C, bound proteins were resolved by SDS-PAGE and visualized by anti-Myc immunoblot.Discs Lost contains an NH2-terminal MRE domain similar to PATJ (Fig. 5 C). Likewise, CG1617 and Pals1 contain L27 domains. Based on our observation that the MRE domain of PATJ binds to one of the two L27 domains in Pals1, we wondered if the Discs Lost MRE domain functioned similarly by binding to the single L27 domain of CG1617. In Fig. 9 C, we demonstrate that the first 135 resides of Discs Lost, which encompasses the MRE domain, bind to CG1617. In contrast, these residues in Discs Lost failed to bind to a CG1617 protein fragment lacking the L27 domain. These in vitro data suggest that in mammalian and Drosophila epithelia a Maguk protein (Pals1 and CG1617, respectively) acts as an adaptor protein, allowing for the assembly of a multiprotein complex containing PATJ/Discs Lost and Crumbs (Fig. 10).Figure 10.
Model depicting the similarities between the mammalian Crumbs–Pals1–PATJ complex and the
Drosophila
 Crumbs–Stardust–Discs Lost complex. In mammalian epithelia, Crumbs interacts with the Pals1 PDZ domain via its COOH terminus. In turn, the L27N domain of Pals1 binds to the NH2-terminal MRE domain of the multi-PDZ protein, PATJ. In this study, Crumbs, Pals1, and PATJ were all shown to localize to tight junctions. In Drosophila epithelial cells, Crumbs interacts with the Pals1 homologue, CG1617, which encodes a fragment of the Stardust gene. Stardust contains a single L27 domain, which interacts with the MRE domain of the PDZ protein, Discs Lost (DLT). Pals1 and its Drosophila homologue are important adaptors that allow Crumbs to interact with PATJ/Disc Lost.

The L27N domain targets Pals1 to the tight junctionIn MDCK cells, mLin-7 has several partners, one of which is the Pals1 protein (Kamberov et al., 2000). In addition to containing an SH3, PDZ, and GUK domain, Pals1 also bears two L27 domains (Fig. 1 A). The extreme NH2-terminal 125 amino acid region bears no significant similarity to other proteins, and we refer to it as the unknown 1 (U1) domain. Our previous studies had shown that mLin-2 localizes to the lateral surface of MDCK cells where it overlaps with E-cadherin (Straight et al., 2001). In contrast, we find that Pals1 colocalizes with ZO-1 to tight junctions in both MDCK cells (Fig. 1 B) and in rat kidney (Fig. 1 C).Figure 1.
Targeting of Pals1 in MDCK cells. (A) Pals1 has multiple protein–protein interaction domains including two Lin-2/Lin-7 (L27N and L27C) domains, a PDZ domain, SH3 domain, protein 4.1 binding (4.1B) domain, and GUK domain. (B) MDCK cells were immunostained as described in Materials and methods. The square panels represent digital photomicrographs (x-y dimension of the z-series), and the rectangular panels show the x-z dimension (z-section). (C) Rat kidney sections were immunostained as described in Materials and methods.We next sought to determine the domains of Pals1 important for the targeting of this protein to cell contacts (Fig. 2 A). Pals1 harboring deletions of the SH3, 4.1B, or GUK domain all targeted to sites of cell–cell adhesion but with decreased efficiency (Fig. 2 B). In fact, when all of these domains were deleted the resulting Pals1 (1–346) protein still targeted to sites of cell adhesion. Deletion of the U1 and both L27 domains resulted in the mistargeting of Pals1 (207–end) away from sites of cell contacts (Fig. 2 B). In contrast, we found that the U1 and L27N domains were sufficient for targeting Pals1 to tight junctions (Fig. 2 D).Figure 2.
Localization efficiency of EYFP-tagged Pals1 deletion mutants. (A) Schematic representation of the different Pals1-EYFP constructs. Numbers overlying the dotted lines refer to the amino acid positions flanking the internal domain deletions. (B) Localization of wild-type Pals1 and Pals1 deletion mutants in MDCK cells. Cells expressing the various constructs shown in A were fixed and directly visualized. (C) Lysates derived from HEK293 cells expressing the wild-type Pals1-EYFP and the various mutant constructs were resolved by SDS-PAGE. The chimeric protein in which the L27N domain of Pals2 was replaced by the Pals1 L27N domain is denoted as PALS2_PALS1L27N (Fig. 3). The Pals1-EYFP protein harboring a double point mutation in the L27N domain is denoted as PALS1L150GV154G (Fig. 3). (D) Immunostaining of MDCK cells expressing Myc-Pals1 (1–182), which contain only the U1 and L27N domains.To further understand the role of this NH2 terminal region, including the U1 and L27 domains, we made a series of deletions and mutations in this region (Fig. 3 A). Deleting the U1 domain had no effect on the targeting of Pals1 to tight junctions (Fig. 3 B). Similarly, deleting the L27C domain had no effect on Pals1 targeting, indicating that this targeting is not dependent on the interaction between Pals1 and mLin-7. In contrast, mutating or deleting the L27N domain redirected Pals1 from tight junctions to the apical surface (Fig. 3, B and C). To further investigate the role of the Pals1 L27N domain in protein targeting, a chimera was generated whereby this domain was used to replace the L27N domain of Pals2 (Fig. 3 D). Pals2 is a Maguk protein like Pals1 (Fig. 3 D) but is missing the U1 domain extension (Kamberov et al., 2000). In the conserved region that extends from the L27N to the GUK domain, Pals2 is ∼35% identical to Pals1. Whereas Pals2 colocalizes with E-cadherin below tight junctions, this chimeric protein (Pals2_Pals1L27N) targets to tight junctions (Fig. 3 E). When we removed the Pals1 L27N domain from the chimera and reintroduced the Pals2 L27N domain (Pals2ΔPals2L27N), the resulting protein returned to the lateral surface. This confirmed the role of the Pals1 L27N domain in the targeting of proteins to the tight junction.Figure 3.
The L27N domain of Pals1 targets proteins to tight junctions. (A) Schematic diagram of the deletion and point mutant versions of Pals1. Asterisks denote the double point mutation in the L27N domain (L150G;V154G). (B) Examination of the distribution of Pals1 mutants displayed in A via fluorescence microscopy. (C) Z-section images of MDCK cells expressing wild-type Pals1 or L27N domain mutant Pals1 proteins (green). Red staining represents either ZO-1 or E-cadherin as indicated. (D) Schematic diagram of the wild-type Pals2-EYFP protein and the chimeric protein where the L27N domain of Pals2 is replaced by the Pals1 L27N domain (Pals2_Pals1L27N). We also generated a construct (unpublished data) in which we removed the Pals1 L27N domain from Pals2_Pals1L27N chimera and replaced it with the original Pals2 L27N domain (Pals2_Pals2L27N). (E) Z-section images of MDCK cells expressing Pals1, Pals2, Pals2_Pals1L27N, and Pals2_Pals2L27N proteins. Pals proteins are shown in green. Red staining represents either ZO-1 or E-cadherin.

The L27N domain targets Pals1 to the tight junction

In MDCK cells, mLin-7 has several partners, one of which is the Pals1 protein (Kamberov et al., 2000). In addition to containing an SH3, PDZ, and GUK domain, Pals1 also bears two L27 domains (Fig. 1 A). The extreme NH2-terminal 125 amino acid region bears no significant similarity to other proteins, and we refer to it as the unknown 1 (U1) domain. Our previous studies had shown that mLin-2 localizes to the lateral surface of MDCK cells where it overlaps with E-cadherin (Straight et al., 2001). In contrast, we find that Pals1 colocalizes with ZO-1 to tight junctions in both MDCK cells (Fig. 1 B) and in rat kidney (Fig. 1 C).

Targeting of Pals1 in MDCK cells. (A) Pals1 has multiple protein–protein interaction domains including two Lin-2/Lin-7 (L27N and L27C) domains, a PDZ domain, SH3 domain, protein 4.1 binding (4.1B) domain, and GUK domain. (B) MDCK cells were immunostained as described in Materials and methods. The square panels represent digital photomicrographs (x-y dimension of the z-series), and the rectangular panels show the x-z dimension (z-section). (C) Rat kidney sections were immunostained as described in Materials and methods.

We next sought to determine the domains of Pals1 important for the targeting of this protein to cell contacts (Fig. 2 A). Pals1 harboring deletions of the SH3, 4.1B, or GUK domain all targeted to sites of cell–cell adhesion but with decreased efficiency (Fig. 2 B). In fact, when all of these domains were deleted the resulting Pals1 (1–346) protein still targeted to sites of cell adhesion. Deletion of the U1 and both L27 domains resulted in the mistargeting of Pals1 (207–end) away from sites of cell contacts (Fig. 2 B). In contrast, we found that the U1 and L27N domains were sufficient for targeting Pals1 to tight junctions (Fig. 2 D).

Localization efficiency of EYFP-tagged Pals1 deletion mutants. (A) Schematic representation of the different Pals1-EYFP constructs. Numbers overlying the dotted lines refer to the amino acid positions flanking the internal domain deletions. (B) Localization of wild-type Pals1 and Pals1 deletion mutants in MDCK cells. Cells expressing the various constructs shown in A were fixed and directly visualized. (C) Lysates derived from HEK293 cells expressing the wild-type Pals1-EYFP and the various mutant constructs were resolved by SDS-PAGE. The chimeric protein in which the L27N domain of Pals2 was replaced by the Pals1 L27N domain is denoted as PALS2_PALS1L27N (Fig. 3). The Pals1-EYFP protein harboring a double point mutation in the L27N domain is denoted as PALS1L150GV154G (Fig. 3). (D) Immunostaining of MDCK cells expressing Myc-Pals1 (1–182), which contain only the U1 and L27N domains.

To further understand the role of this NH2 terminal region, including the U1 and L27 domains, we made a series of deletions and mutations in this region (Fig. 3 A). Deleting the U1 domain had no effect on the targeting of Pals1 to tight junctions (Fig. 3 B). Similarly, deleting the L27C domain had no effect on Pals1 targeting, indicating that this targeting is not dependent on the interaction between Pals1 and mLin-7. In contrast, mutating or deleting the L27N domain redirected Pals1 from tight junctions to the apical surface (Fig. 3, B and C). To further investigate the role of the Pals1 L27N domain in protein targeting, a chimera was generated whereby this domain was used to replace the L27N domain of Pals2 (Fig. 3 D). Pals2 is a Maguk protein like Pals1 (Fig. 3 D) but is missing the U1 domain extension (Kamberov et al., 2000). In the conserved region that extends from the L27N to the GUK domain, Pals2 is ∼35% identical to Pals1. Whereas Pals2 colocalizes with E-cadherin below tight junctions, this chimeric protein (Pals2_Pals1L27N) targets to tight junctions (Fig. 3 E). When we removed the Pals1 L27N domain from the chimera and reintroduced the Pals2 L27N domain (Pals2ΔPals2L27N), the resulting protein returned to the lateral surface. This confirmed the role of the Pals1 L27N domain in the targeting of proteins to the tight junction.

The L27N domain of Pals1 targets proteins to tight junctions. (A) Schematic diagram of the deletion and point mutant versions of Pals1. Asterisks denote the double point mutation in the L27N domain (L150G;V154G). (B) Examination of the distribution of Pals1 mutants displayed in A via fluorescence microscopy. (C) Z-section images of MDCK cells expressing wild-type Pals1 or L27N domain mutant Pals1 proteins (green). Red staining represents either ZO-1 or E-cadherin as indicated. (D) Schematic diagram of the wild-type Pals2-EYFP protein and the chimeric protein where the L27N domain of Pals2 is replaced by the Pals1 L27N domain (Pals2_Pals1L27N). We also generated a construct (unpublished data) in which we removed the Pals1 L27N domain from Pals2_Pals1L27N chimera and replaced it with the original Pals2 L27N domain (Pals2_Pals2L27N). (E) Z-section images of MDCK cells expressing Pals1, Pals2, Pals2_Pals1L27N, and Pals2_Pals2L27N proteins. Pals proteins are shown in green. Red staining represents either ZO-1 or E-cadherin.

The Pals1 L27N domain is a protein–protein interaction domainPrevious work had shown that the L27C domain of Pals1 is necessary for Pals1 to bind mLin-7 (Kamberov et al., 2000). However, our data indicate that the L27N domains of all the Pals family members do not bind mLin-7 and do not appear to be involved in homo- or heterodimerization with other L27 domains (Lee et al., 2002). Accordingly, we purified and radiolabeled a glutathione S-transferase (GST) fusion protein of the Pals1 L27N domain and performed Far Western blotting to look for a possible binding partner that might mediate the targeting of Pals1 to tight junctions. The Far Western probe bound to a protein of ∼200 kD in tissues and cell lines including rat brain, mouse brain, MDCK, and 293 cells (Fig. 4 A). Anti–mLin-7 and anti-Myc antibodies could coimmunoprecipitate the 200-kD protein in MDCK and 293 cells expressing Myc-Pals1 (Fig. 4 B), suggesting that this 200-kD protein is in a complex with Pals1 and mLin-7.Figure 4.
The Pals1 L27N domain binds a protein of ∼200 kD. (A) Lysates prepared from rat brain, mouse brain, MDCK cells, and 293 cells were resolved by SDS-PAGE. Far Western blotting with 32P-labeled Pals1 L27N probe was performed. (B) Coimmunoprecipitation experiments using antibodies against Myc or mLin-7 and lysates derived from MDCK and 293 cells expressing Myc-Pals1 were performed. Precipitated proteins, resolved by SDS-PAGE, were probed with 32P-labeled Pals1 L27N domain (top) and immunoblotted with anti-Myc antibody (bottom). The arrowhead points to the IgG heavy chain. (C) The indicated EYFP- and Myc-tagged Pals1 proteins expressed in 293 cells were immunoprecipitated, and bound proteins were resolved by SDS-PAGE. Immunoblotting with anti-EYFP and anti-Myc antibodies confirmed the expression of the Pals constructs (top). Far Western blotting was performed with 32P-labeled Pals1 L27N probe (bottom). Pals1DM-EYFP refers to EYFP-Pals1 (L150G;V154G), whereas P2P1-EYFP refers to the Pals2_Pals1L27N chimeric protein.Next, we examined the interaction of the 200-kD protein with the different EYFP-Pals1 mutants described previously. Far Western blotting confirmed that the 200-kD protein coimmunoprecipitated with EYFP-Pals1 and EYFP-Pals1(1–182) expressed in 293 cells (Fig. 4 C). Pals1 missing or containing a double point mutation in the L27N domain could not interact with this protein. Likewise, Pals2 did not interact with this 200-kD protein. However, a chimera of Pals2 containing the L27N domain of Pals1 did interact with the 200-kD protein. As expected, we showed that the 200-kD protein coimmunoprecipitated with Myc-Pals1 but not with the Myc-Pals1 protein missing the L27N domain (Fig. 4 C). The targeting of Pals1 to tight junctions correlated exactly with its ability to bind to this 200-kD protein.

The Pals1 L27N domain is a protein–protein interaction domain

Previous work had shown that the L27C domain of Pals1 is necessary for Pals1 to bind mLin-7 (Kamberov et al., 2000). However, our data indicate that the L27N domains of all the Pals family members do not bind mLin-7 and do not appear to be involved in homo- or heterodimerization with other L27 domains (Lee et al., 2002). Accordingly, we purified and radiolabeled a glutathione S-transferase (GST) fusion protein of the Pals1 L27N domain and performed Far Western blotting to look for a possible binding partner that might mediate the targeting of Pals1 to tight junctions. The Far Western probe bound to a protein of ∼200 kD in tissues and cell lines including rat brain, mouse brain, MDCK, and 293 cells (Fig. 4 A). Anti–mLin-7 and anti-Myc antibodies could coimmunoprecipitate the 200-kD protein in MDCK and 293 cells expressing Myc-Pals1 (Fig. 4 B), suggesting that this 200-kD protein is in a complex with Pals1 and mLin-7.

The Pals1 L27N domain binds a protein of ∼200 kD. (A) Lysates prepared from rat brain, mouse brain, MDCK cells, and 293 cells were resolved by SDS-PAGE. Far Western blotting with 32P-labeled Pals1 L27N probe was performed. (B) Coimmunoprecipitation experiments using antibodies against Myc or mLin-7 and lysates derived from MDCK and 293 cells expressing Myc-Pals1 were performed. Precipitated proteins, resolved by SDS-PAGE, were probed with 32P-labeled Pals1 L27N domain (top) and immunoblotted with anti-Myc antibody (bottom). The arrowhead points to the IgG heavy chain. (C) The indicated EYFP- and Myc-tagged Pals1 proteins expressed in 293 cells were immunoprecipitated, and bound proteins were resolved by SDS-PAGE. Immunoblotting with anti-EYFP and anti-Myc antibodies confirmed the expression of the Pals constructs (top). Far Western blotting was performed with 32P-labeled Pals1 L27N probe (bottom). Pals1DM-EYFP refers to EYFP-Pals1 (L150G;V154G), whereas P2P1-EYFP refers to the Pals2_Pals1L27N chimeric protein.

Next, we examined the interaction of the 200-kD protein with the different EYFP-Pals1 mutants described previously. Far Western blotting confirmed that the 200-kD protein coimmunoprecipitated with EYFP-Pals1 and EYFP-Pals1(1–182) expressed in 293 cells (Fig. 4 C). Pals1 missing or containing a double point mutation in the L27N domain could not interact with this protein. Likewise, Pals2 did not interact with this 200-kD protein. However, a chimera of Pals2 containing the L27N domain of Pals1 did interact with the 200-kD protein. As expected, we showed that the 200-kD protein coimmunoprecipitated with Myc-Pals1 but not with the Myc-Pals1 protein missing the L27N domain (Fig. 4 C). The targeting of Pals1 to tight junctions correlated exactly with its ability to bind to this 200-kD protein.

Identification of a novel 200-kD protein interacting with the L27N domain of Pals1To identify the 200-kD protein, we performed large scale coimmunoprecipitations using lysates derived from MDCK cell lines expressing Myc-Pals1 and Myc-Pals1 missing the U1 and L27N domains. The precipitated proteins were resolved electrophoretically and visualized by silver staining (Fig. 5 A) and colloidal Coomassie staining. We could identify a ∼200-kD protein that was present in the Myc-Pals1 immunoprecipitates but absent from the Myc-Pals1Δ(U1 + L27N) immunoprecipitates. We excised this band from the Coomassie-stained gel and determined its identity by MS–MS sequence analysis. The peptide sequences obtained matched those found in a PDZ protein identified previously as a human homologue of the Drosophila protein INAD (hINADL) Table I and Philipp and Flockerzi, 1997). Additional peptide sequences corresponding to PDZ domains not found in hINADL (PDZ domain protein 3′ variant 4 [Table I]) were also identified. Using the peptide sequences and EST database, we cloned the cDNA encoding the full-length Pals1 binding partner into mammalian expression vectors.Figure 5.
Purification and identification of the 200 kD binding partner of the Pals1 L27N domain. (A) Large scale Myc-Pals1 and Myc-Pals1Δ(U1 + L27N) coimmunoprecipitations were performed. 5% of the purified proteins were resolved by SDS-PAGE and visualized by silver staining. The arrowhead points to the band representing the 200 kD Pals1 binding partner. (B) Structural alignment between MUPP1, PATJ, and Discs Lost (DLT). Percentage amino acid sequence identity are displayed between the respective MRE domains and PDZ domains (circles and rectangles, respectively). (C) Sequence alignment of the conserved MRE domains in PATJ, MUPP1, C. elegans Discs Large-1 (CeDLG1), SAP97, and Discs Lost (DLT). Identical residues seen in at least three of the five MRE domains are in bold. The sequence data for PATJ have been submitted to GenBank (sequence data available from GenBank/EMBL/DDBJ under accession no. AF397170.Table I.
Peptide sequences obtained from the ∼200 kD binding partner for the L27N domain of Pals1
Peptide sequences obtainedIdentitySequence data available fromGenBank/EMBL/DDBJ under accession no.DTPLNAIVIHEVYEEGAAARhINADL
AJ224747
EPNVSLGISIVGGQTVIKhINADL
AJ224747
GLGFSILDYQDPLDPTRhINADL
AJ224747
LQVLQVLDRhINADL
AJ224747
RLFDDEASVDEPRhINADL
AJ224747
NAGQVVHLTLVRhINADL
AJ224747
VPDSPENELKhINADL
AJ224747
LQTGDHILKhINADL
AJ224747
TIVPGGLADRDGRhINADL
AJ224747
DGQSLGIRhINADL
AJ224747
SQNLGKVDIFVKhINADL
AJ224747
IGDELLEINNQILYGRhINADL
AJ224747
QVLEDSPAGKhINADL
AJ224747
EAVSFLKhINADL
AJ224747
GLGLSIVGKhINADL
AJ224747
LIQGDQILSVNGEDMRPDZ domain protein 3′ variant 4
AB044807
NGSGVFISDIVKPDZ domain protein 3′ variant 4
AB044807(
We found that this protein consists of 10 PDZ domains and that hINADL represents an incomplete version of this protein (Fig. 5 B). The full-length protein consists of PDZ domains found in both hINADL and PDZ domain protein 3′ variant 4. Pals1-associated tight junction (PATJ) protein is the paralogue of multiple PDZ protein (MUPP)1 and is the ortholog of Drosophila Discs Lost (Ullmer et al., 1998; Bhat et al., 1999; Mancini et al., 2000). The PDZ domains of PATJ, MUPP1, and Discs Lost are highly conserved (Fig. 5 B). Because the closest ortholog of hINADL in Drosophila is not INAD, we have decided to rename this protein PATJ protein in agreement with its ability to recruit Pals1 to tight junctions. As predicted, when an EYFP–PATJ fusion protein was expressed in MDCK cells its localization was coincident with Pals1 and ZO-1 at tight junctions (Fig. 6 A) and superior to E-cadherin (Fig. 6 B).Figure 6.
PATJ localizes to tight junctions in MDCK cells. MDCK cells expressing full-length EYFP-PATJ were costained with either anti-Pals1 and anti–ZO-1 antibodies (A) or antiuvomorulin antibody (B) and examined by confocal microscopy.

Identification of a novel 200-kD protein interacting with the L27N domain of Pals1

To identify the 200-kD protein, we performed large scale coimmunoprecipitations using lysates derived from MDCK cell lines expressing Myc-Pals1 and Myc-Pals1 missing the U1 and L27N domains. The precipitated proteins were resolved electrophoretically and visualized by silver staining (Fig. 5 A) and colloidal Coomassie staining. We could identify a ∼200-kD protein that was present in the Myc-Pals1 immunoprecipitates but absent from the Myc-Pals1Δ(U1 + L27N) immunoprecipitates. We excised this band from the Coomassie-stained gel and determined its identity by MS–MS sequence analysis. The peptide sequences obtained matched those found in a PDZ protein identified previously as a human homologue of the Drosophila protein INAD (hINADL) Table I and Philipp and Flockerzi, 1997). Additional peptide sequences corresponding to PDZ domains not found in hINADL (PDZ domain protein 3′ variant 4 [Table I]) were also identified. Using the peptide sequences and EST database, we cloned the cDNA encoding the full-length Pals1 binding partner into mammalian expression vectors.

Purification and identification of the 200 kD binding partner of the Pals1 L27N domain. (A) Large scale Myc-Pals1 and Myc-Pals1Δ(U1 + L27N) coimmunoprecipitations were performed. 5% of the purified proteins were resolved by SDS-PAGE and visualized by silver staining. The arrowhead points to the band representing the 200 kD Pals1 binding partner. (B) Structural alignment between MUPP1, PATJ, and Discs Lost (DLT). Percentage amino acid sequence identity are displayed between the respective MRE domains and PDZ domains (circles and rectangles, respectively). (C) Sequence alignment of the conserved MRE domains in PATJ, MUPP1, C. elegans Discs Large-1 (CeDLG1), SAP97, and Discs Lost (DLT). Identical residues seen in at least three of the five MRE domains are in bold. The sequence data for PATJ have been submitted to GenBank (sequence data available from GenBank/EMBL/DDBJ under accession no. AF397170.

Peptide sequences obtained from the ∼200 kD binding partner for the L27N domain of Pals1

We found that this protein consists of 10 PDZ domains and that hINADL represents an incomplete version of this protein (Fig. 5 B). The full-length protein consists of PDZ domains found in both hINADL and PDZ domain protein 3′ variant 4. Pals1-associated tight junction (PATJ) protein is the paralogue of multiple PDZ protein (MUPP)1 and is the ortholog of Drosophila Discs Lost (Ullmer et al., 1998; Bhat et al., 1999; Mancini et al., 2000). The PDZ domains of PATJ, MUPP1, and Discs Lost are highly conserved (Fig. 5 B). Because the closest ortholog of hINADL in Drosophila is not INAD, we have decided to rename this protein PATJ protein in agreement with its ability to recruit Pals1 to tight junctions. As predicted, when an EYFP–PATJ fusion protein was expressed in MDCK cells its localization was coincident with Pals1 and ZO-1 at tight junctions (Fig. 6 A) and superior to E-cadherin (Fig. 6 B).

PATJ localizes to tight junctions in MDCK cells. MDCK cells expressing full-length EYFP-PATJ were costained with either anti-Pals1 and anti–ZO-1 antibodies (A) or antiuvomorulin antibody (B) and examined by confocal microscopy.

A novel evolutionarily conserved domain in PATJ targets Pals1 to tight junctionsWhen expressed in 293 cells, full-length PATJ was observed to be an ∼190-kD protein that coimmunoprecipitated with Pals1 but not Pals1 missing its L27N domain (Fig. 7 A). We next sought to determine the region in PATJ that was responsible for binding the Pals1 L27N domain. GST fused to the first 238, 109, and 68 residues of PATJ could bind full-length Pals1 (Fig. 7 B). These fusion proteins all contain the ∼50 amino acid domain conserved in PATJ, MUPP1, and Discs Lost (Fig. 5 C). We also found that the NH2-terminal 45 residues of PATJ could interact with Pals1 albeit weakly. None of the GST–PATJ fusion proteins could bind the Pals1ΔL27N mutant protein. A GST–mLin-7 fusion protein was able to bind both Pals1 and Pals1ΔL27N, since both contain an intact L27C domain (positive control). Thus, we now report an evolutionarily conserved region contained within the NH2 terminus of PATJ as a novel protein–protein interaction domain. In light of its ability to bind and recruit the Maguk protein Pals1 to tight junctions, we have decided to name it the Maguk recruitment (MRE) domain. Our preliminary data suggest that although the MUPP1 MRE domain also binds to the Pals1 L27N domain, the Discs Lost MRE domain fails to interact with Pals1 (unpublished data).Figure 7.
PATJ interacts with the L27N domain of Pals1. (A) Full-length HA epitope-tagged PATJ was coexpressed in 293 cells with either Myc-Pals1 or Myc-Pals1 missing the U1 and L27N domains. Anti-Myc immunoprecipitates from both lysates were resolved by SDS-PAGE. Immunoblotting reveals that HA-PATJ coprecipitates only with full-length Myc-Pals1. (B) GST-PATJ pulldown assays were performed using 293 lysates containing overexpressed Myc-Pals1 or Myc-Pals1ΔL27N. Numbers in parentheses denote the amino acids of PATJ fused to GST. (C) Immunoprecipitations using preimmune serum or anti–mLin-7 antibodies, and lysates from the indicated cell lines were performed. Precipitated proteins were resolved by SDS-PAGE and probed with 32P-labeled GST-PATJ (1–238). (D) MDCK cell pools expressing Myc-PATJ (1–238) were examined by confocal microscopy. Cells were costained with anti-Pals1 and anti-Myc antibodies.To determine if additional binding partners for the PATJ MRE domain existed, a radiolabeled GST–PATJ (1–238) probe was used in Far Western blot overlay experiments. In 293, MDCK, HeLa, and HT29 cells, two proteins at ∼71 kD were recognized by the probe (Fig. 7 C). These bands were also observed in mLin-7 coimmunoprecipitates. Immunoblotting with antibodies raised against the COOH terminus of Pals1 recognizes two bands at ∼71 kD (Fig. 8 D), suggesting that Pals1 is the predominant partner for the PATJ MRE domain. A third band representing a smaller unidentified protein was also observed in HeLa cells.Figure 8.
Full-length CRB1 targets to tight junctions via the interaction between its COOH terminus and Pals1. (A) CRB1 was identified in the critical chromosomal region linked to a severe form of autosomal recessive Retinitis Pigmentosa, RP12. The published domain structure of CRB1 and the protein fragment encoded by the human EST BF912191 are schematically shown. The full-length Myc-CRB1 construct used in this study is also shown. The open rectangle and filled circle represent the signal peptide and the Myc tag, respectively. Rectangles containing horizontal and vertical lines correspond to the EGF-like and C-type lectin domains, respectively. The laminin A G-like domains and the transmembrane region are represented by filled pentagons and the filled rectangle, respectively. (B) Comparison of the transmembrane segments and cytoplasmic tails of Drosophila Crumbs (CRB) and full-length CRB1. Identical residues are in bold. Underlined residues were fused to GST for GST pulldown experiments. (C) MDCK cells stably expressing Myc-CRB1 were immunostained with anti-Myc antibody and antibodies directed against either ZO-1 or Pals1. (D) Immunoprecipitations with preimmune serum (Control IP), anti-PATJ serum, and anti-Myc ascites were performed on lysates derived from the cells in C. Endogenous PATJ and Pals1 were detected by immunoblotting with the respective antisera. CRB1 was detected by blotting with anti-Myc antibody. (E) GST was fused to the COOH-terminal residues of Drosophila Crumbs (GST-CRB) or human CRB1 (GST-CRB1) as indicated in B. 293 lysates containing indicated proteins were incubated with GST-CRB and GST-CRB1 beads. Adsorbed proteins were resolved by SDS-PAGE and detected by immunoblotting with anti-HA and anti-Myc antibodies.To confirm that the interaction between the PATJ MRE domain and the Pals1 L27N domain was a physiologically relevant interaction, we expressed the first 238 residues of PATJ in MDCK cells. PATJ (1–238) consists of the MRE and first PDZ domains of PATJ. When expressed in MDCK cells, Myc-PATJ (1–238) exhibits a diffuse localization pattern (Fig. 7 D). In cells expressing this dominant negative PATJ protein, endogenous Pals1 was absent from tight junctions. However, in the surrounding cells Pals1 was present at tight junctions. Although adherens and tight junctions were not affected by the expression of this dominant negative protein, (as determined by E-cadherin and ZO-1 immunostaining [unpublished data]), these results confirm the role of PATJ in targeting Pals1 to tight junctions.

A novel evolutionarily conserved domain in PATJ targets Pals1 to tight junctions

When expressed in 293 cells, full-length PATJ was observed to be an ∼190-kD protein that coimmunoprecipitated with Pals1 but not Pals1 missing its L27N domain (Fig. 7 A). We next sought to determine the region in PATJ that was responsible for binding the Pals1 L27N domain. GST fused to the first 238, 109, and 68 residues of PATJ could bind full-length Pals1 (Fig. 7 B). These fusion proteins all contain the ∼50 amino acid domain conserved in PATJ, MUPP1, and Discs Lost (Fig. 5 C). We also found that the NH2-terminal 45 residues of PATJ could interact with Pals1 albeit weakly. None of the GST–PATJ fusion proteins could bind the Pals1ΔL27N mutant protein. A GST–mLin-7 fusion protein was able to bind both Pals1 and Pals1ΔL27N, since both contain an intact L27C domain (positive control). Thus, we now report an evolutionarily conserved region contained within the NH2 terminus of PATJ as a novel protein–protein interaction domain. In light of its ability to bind and recruit the Maguk protein Pals1 to tight junctions, we have decided to name it the Maguk recruitment (MRE) domain. Our preliminary data suggest that although the MUPP1 MRE domain also binds to the Pals1 L27N domain, the Discs Lost MRE domain fails to interact with Pals1 (unpublished data).

PATJ interacts with the L27N domain of Pals1. (A) Full-length HA epitope-tagged PATJ was coexpressed in 293 cells with either Myc-Pals1 or Myc-Pals1 missing the U1 and L27N domains. Anti-Myc immunoprecipitates from both lysates were resolved by SDS-PAGE. Immunoblotting reveals that HA-PATJ coprecipitates only with full-length Myc-Pals1. (B) GST-PATJ pulldown assays were performed using 293 lysates containing overexpressed Myc-Pals1 or Myc-Pals1ΔL27N. Numbers in parentheses denote the amino acids of PATJ fused to GST. (C) Immunoprecipitations using preimmune serum or anti–mLin-7 antibodies, and lysates from the indicated cell lines were performed. Precipitated proteins were resolved by SDS-PAGE and probed with 32P-labeled GST-PATJ (1–238). (D) MDCK cell pools expressing Myc-PATJ (1–238) were examined by confocal microscopy. Cells were costained with anti-Pals1 and anti-Myc antibodies.

To determine if additional binding partners for the PATJ MRE domain existed, a radiolabeled GST–PATJ (1–238) probe was used in Far Western blot overlay experiments. In 293, MDCK, HeLa, and HT29 cells, two proteins at ∼71 kD were recognized by the probe (Fig. 7 C). These bands were also observed in mLin-7 coimmunoprecipitates. Immunoblotting with antibodies raised against the COOH terminus of Pals1 recognizes two bands at ∼71 kD (Fig. 8 D), suggesting that Pals1 is the predominant partner for the PATJ MRE domain. A third band representing a smaller unidentified protein was also observed in HeLa cells.

Full-length CRB1 targets to tight junctions via the interaction between its COOH terminus and Pals1. (A) CRB1 was identified in the critical chromosomal region linked to a severe form of autosomal recessive Retinitis Pigmentosa, RP12. The published domain structure of CRB1 and the protein fragment encoded by the human EST BF912191 are schematically shown. The full-length Myc-CRB1 construct used in this study is also shown. The open rectangle and filled circle represent the signal peptide and the Myc tag, respectively. Rectangles containing horizontal and vertical lines correspond to the EGF-like and C-type lectin domains, respectively. The laminin A G-like domains and the transmembrane region are represented by filled pentagons and the filled rectangle, respectively. (B) Comparison of the transmembrane segments and cytoplasmic tails of Drosophila Crumbs (CRB) and full-length CRB1. Identical residues are in bold. Underlined residues were fused to GST for GST pulldown experiments. (C) MDCK cells stably expressing Myc-CRB1 were immunostained with anti-Myc antibody and antibodies directed against either ZO-1 or Pals1. (D) Immunoprecipitations with preimmune serum (Control IP), anti-PATJ serum, and anti-Myc ascites were performed on lysates derived from the cells in C. Endogenous PATJ and Pals1 were detected by immunoblotting with the respective antisera. CRB1 was detected by blotting with anti-Myc antibody. (E) GST was fused to the COOH-terminal residues of Drosophila Crumbs (GST-CRB) or human CRB1 (GST-CRB1) as indicated in B. 293 lysates containing indicated proteins were incubated with GST-CRB and GST-CRB1 beads. Adsorbed proteins were resolved by SDS-PAGE and detected by immunoblotting with anti-HA and anti-Myc antibodies.

To confirm that the interaction between the PATJ MRE domain and the Pals1 L27N domain was a physiologically relevant interaction, we expressed the first 238 residues of PATJ in MDCK cells. PATJ (1–238) consists of the MRE and first PDZ domains of PATJ. When expressed in MDCK cells, Myc-PATJ (1–238) exhibits a diffuse localization pattern (Fig. 7 D). In cells expressing this dominant negative PATJ protein, endogenous Pals1 was absent from tight junctions. However, in the surrounding cells Pals1 was present at tight junctions. Although adherens and tight junctions were not affected by the expression of this dominant negative protein, (as determined by E-cadherin and ZO-1 immunostaining [unpublished data]), these results confirm the role of PATJ in targeting Pals1 to tight junctions.

Human Crumbs exists in a ternary complex with Pals1 and PATJ at tight junctionsIn Drosophila, Discs Lost has been reported to bind the COOH terminus of Crumbs. A recent study has linked a human Crumbs homologue, CRB1, to a visual disorder, Retinitis Pigmentosa. Drosophila Crumbs (CRB) contains a signal peptide and a transmembrane segment; however, CRB1 was reported to lack the transmembrane segment (den Hollander et al., 1999). Consequently, the authors proposed that CRB1 was a secreted protein. However, they raised the possibility that their CRB1 cDNA sequence may have been incomplete at the 3′ end. To resolve this issue, we searched the human EST database for ESTs encoding the last EGF-like domain of CRB1. We identified an EST obtained from a dendritic oligodendroglioma (sequence data available from GenBank/EMBL/DDBJ under accession no. BG912191), which encoded for the last three extracellular EGF-like domains, a single C-type lectin domain, a transmembrane segment, and a 37 amino acid tail that was 62% identical to that of CRB (Fig. 8, A and B). Therefore, it seemed likely that a longer CRB1 cDNA existed and that it encoded for an integral membrane protein. Based on this notion, we amplified this full-length cDNA by PCR and cloned it into mammalian expression vectors. When full-length Myc-CRB1 (Fig. 8 A) was expressed in MDCK cells, it was observed to colocalize with Pals1 and ZO-1 at tight junctions (Fig. 8 C). We could also demonstrate that the ability of CRB1 to target to tight junctions correlated with its ability to exist in a ternary complex with endogenous PATJ and Pals1 (Fig. 8 D). We find that CRB1 may not be only expressed in neuronal tissues, since we can detect CRB1 DNA fragments using template cDNA derived from epithelial tissues such as kidney and colon (Fig. S1 available at http://www.jcb.org/cgi/content/full/jcb.200109010/DC1). It is also important to note that there are several other isoforms of Crumbs identified in the human genome that have similar transmembrane segments and COOH termini as CRB1 (Tepass et al., 2001).To determine if the CRB and CRB1 COOH termini could bind PATJ and/or Pals1, we performed GST pulldown assays using GST fused to the tails of CRB and CRB1 (GST-CRB and GST-CRB1, respectively). Both were able to coprecipitate Myc-Pals1 and hemagglutinin (HA)-PATJ expressed in 293 cells (Fig. 8 E). When GST-CRB and GST-CRB1 were incubated with HA-PATJ and Myc-Pals1 missing its PDZ domain, a significantly lower amount of HA-PATJ was precipitated. Based on this result, the possibility still remains that GST-CRB and GST-CRB1 interact directly with PATJ. Alternatively, this result could be explained by the presence of endogenous Pals1 in the 293 lysate mediating the indirect interaction between GST-CRB and GST-CRB1 and HA-PATJ. To distinguish between these two possibilities, the GST fusion proteins were incubated with Myc-Pals1 and HA-PATJ missing its MRE domain. HA-PATJΔMRE should not be able to bind either Myc-Pals1 or endogenous Pals1 in the 293 lysate. Both fusion proteins were able to bind Myc-Pals1 but not HA-PATJΔMRE (Fig. 8 E). This suggests that in mammalian epithelia Pals1 serves as an adaptor protein, mediating the indirect interaction between CRB1 and PATJ.

Human Crumbs exists in a ternary complex with Pals1 and PATJ at tight junctions

In Drosophila, Discs Lost has been reported to bind the COOH terminus of Crumbs. A recent study has linked a human Crumbs homologue, CRB1, to a visual disorder, Retinitis Pigmentosa. Drosophila Crumbs (CRB) contains a signal peptide and a transmembrane segment; however, CRB1 was reported to lack the transmembrane segment (den Hollander et al., 1999). Consequently, the authors proposed that CRB1 was a secreted protein. However, they raised the possibility that their CRB1 cDNA sequence may have been incomplete at the 3′ end. To resolve this issue, we searched the human EST database for ESTs encoding the last EGF-like domain of CRB1. We identified an EST obtained from a dendritic oligodendroglioma (sequence data available from GenBank/EMBL/DDBJ under accession no. BG912191), which encoded for the last three extracellular EGF-like domains, a single C-type lectin domain, a transmembrane segment, and a 37 amino acid tail that was 62% identical to that of CRB (Fig. 8, A and B). Therefore, it seemed likely that a longer CRB1 cDNA existed and that it encoded for an integral membrane protein. Based on this notion, we amplified this full-length cDNA by PCR and cloned it into mammalian expression vectors. When full-length Myc-CRB1 (Fig. 8 A) was expressed in MDCK cells, it was observed to colocalize with Pals1 and ZO-1 at tight junctions (Fig. 8 C). We could also demonstrate that the ability of CRB1 to target to tight junctions correlated with its ability to exist in a ternary complex with endogenous PATJ and Pals1 (Fig. 8 D). We find that CRB1 may not be only expressed in neuronal tissues, since we can detect CRB1 DNA fragments using template cDNA derived from epithelial tissues such as kidney and colon (Fig. S1 available at http://www.jcb.org/cgi/content/full/jcb.200109010/DC1). It is also important to note that there are several other isoforms of Crumbs identified in the human genome that have similar transmembrane segments and COOH termini as CRB1 (Tepass et al., 2001).

To determine if the CRB and CRB1 COOH termini could bind PATJ and/or Pals1, we performed GST pulldown assays using GST fused to the tails of CRB and CRB1 (GST-CRB and GST-CRB1, respectively). Both were able to coprecipitate Myc-Pals1 and hemagglutinin (HA)-PATJ expressed in 293 cells (Fig. 8 E). When GST-CRB and GST-CRB1 were incubated with HA-PATJ and Myc-Pals1 missing its PDZ domain, a significantly lower amount of HA-PATJ was precipitated. Based on this result, the possibility still remains that GST-CRB and GST-CRB1 interact directly with PATJ. Alternatively, this result could be explained by the presence of endogenous Pals1 in the 293 lysate mediating the indirect interaction between GST-CRB and GST-CRB1 and HA-PATJ. To distinguish between these two possibilities, the GST fusion proteins were incubated with Myc-Pals1 and HA-PATJ missing its MRE domain. HA-PATJΔMRE should not be able to bind either Myc-Pals1 or endogenous Pals1 in the 293 lysate. Both fusion proteins were able to bind Myc-Pals1 but not HA-PATJΔMRE (Fig. 8 E). This suggests that in mammalian epithelia Pals1 serves as an adaptor protein, mediating the indirect interaction between CRB1 and PATJ.

The CRB1–Pals1–PATJ complex is evolutionarily conserved in invertebratesNext, we searched the Drosophila genome for an ortholog of Pals1. We were able to identify a putative Maguk protein, CG1617, that was highly related to Pals1. To determine if Drosophila Crumbs could bind the PDZ domain of CG1617, we performed GST-CRB pulldown assays in the presence of CG1617 fragments (Fig. 9 A). GST-CRB was able to bind CG1617 in a PDZ-dependent fashion; in contrast, GST-CRB failed to bind full-length Discs Lost (Fig. 9 B).Figure 9.
The
Drosophila
Pals1 homologue binds to both Crumbs and Discs Lost via its PDZ and L27 domains, respectively. (A) Schematic diagrams of the Drosophila Pals1 homologue, CG1617, and CG1617 fragments used in B and C are shown. All of these proteins were Myc-epitope tagged. (B) GST-CRB pulldowns were performed using 293 cell lysates containing the indicated expressed proteins. (C) 293 lysates containing the denoted Myc-tagged CG1617 proteins were incubated with either GST or GST fused to the NH2-terminal 135 residues of Discs Lost (DLT), which encompasses the MRE domain. In B and C, bound proteins were resolved by SDS-PAGE and visualized by anti-Myc immunoblot.Discs Lost contains an NH2-terminal MRE domain similar to PATJ (Fig. 5 C). Likewise, CG1617 and Pals1 contain L27 domains. Based on our observation that the MRE domain of PATJ binds to one of the two L27 domains in Pals1, we wondered if the Discs Lost MRE domain functioned similarly by binding to the single L27 domain of CG1617. In Fig. 9 C, we demonstrate that the first 135 resides of Discs Lost, which encompasses the MRE domain, bind to CG1617. In contrast, these residues in Discs Lost failed to bind to a CG1617 protein fragment lacking the L27 domain. These in vitro data suggest that in mammalian and Drosophila epithelia a Maguk protein (Pals1 and CG1617, respectively) acts as an adaptor protein, allowing for the assembly of a multiprotein complex containing PATJ/Discs Lost and Crumbs (Fig. 10).Figure 10.
Model depicting the similarities between the mammalian Crumbs–Pals1–PATJ complex and the
Drosophila
 Crumbs–Stardust–Discs Lost complex. In mammalian epithelia, Crumbs interacts with the Pals1 PDZ domain via its COOH terminus. In turn, the L27N domain of Pals1 binds to the NH2-terminal MRE domain of the multi-PDZ protein, PATJ. In this study, Crumbs, Pals1, and PATJ were all shown to localize to tight junctions. In Drosophila epithelial cells, Crumbs interacts with the Pals1 homologue, CG1617, which encodes a fragment of the Stardust gene. Stardust contains a single L27 domain, which interacts with the MRE domain of the PDZ protein, Discs Lost (DLT). Pals1 and its Drosophila homologue are important adaptors that allow Crumbs to interact with PATJ/Disc Lost.

The CRB1–Pals1–PATJ complex is evolutionarily conserved in invertebrates

Next, we searched the Drosophila genome for an ortholog of Pals1. We were able to identify a putative Maguk protein, CG1617, that was highly related to Pals1. To determine if Drosophila Crumbs could bind the PDZ domain of CG1617, we performed GST-CRB pulldown assays in the presence of CG1617 fragments (Fig. 9 A). GST-CRB was able to bind CG1617 in a PDZ-dependent fashion; in contrast, GST-CRB failed to bind full-length Discs Lost (Fig. 9 B).

The
Drosophila
Pals1 homologue binds to both Crumbs and Discs Lost via its PDZ and L27 domains, respectively. (A) Schematic diagrams of the Drosophila Pals1 homologue, CG1617, and CG1617 fragments used in B and C are shown. All of these proteins were Myc-epitope tagged. (B) GST-CRB pulldowns were performed using 293 cell lysates containing the indicated expressed proteins. (C) 293 lysates containing the denoted Myc-tagged CG1617 proteins were incubated with either GST or GST fused to the NH2-terminal 135 residues of Discs Lost (DLT), which encompasses the MRE domain. In B and C, bound proteins were resolved by SDS-PAGE and visualized by anti-Myc immunoblot.

Discs Lost contains an NH2-terminal MRE domain similar to PATJ (Fig. 5 C). Likewise, CG1617 and Pals1 contain L27 domains. Based on our observation that the MRE domain of PATJ binds to one of the two L27 domains in Pals1, we wondered if the Discs Lost MRE domain functioned similarly by binding to the single L27 domain of CG1617. In Fig. 9 C, we demonstrate that the first 135 resides of Discs Lost, which encompasses the MRE domain, bind to CG1617. In contrast, these residues in Discs Lost failed to bind to a CG1617 protein fragment lacking the L27 domain. These in vitro data suggest that in mammalian and Drosophila epithelia a Maguk protein (Pals1 and CG1617, respectively) acts as an adaptor protein, allowing for the assembly of a multiprotein complex containing PATJ/Discs Lost and Crumbs (Fig. 10).

Model depicting the similarities between the mammalian Crumbs–Pals1–PATJ complex and the
Drosophila
 Crumbs–Stardust–Discs Lost complex. In mammalian epithelia, Crumbs interacts with the Pals1 PDZ domain via its COOH terminus. In turn, the L27N domain of Pals1 binds to the NH2-terminal MRE domain of the multi-PDZ protein, PATJ. In this study, Crumbs, Pals1, and PATJ were all shown to localize to tight junctions. In Drosophila epithelial cells, Crumbs interacts with the Pals1 homologue, CG1617, which encodes a fragment of the Stardust gene. Stardust contains a single L27 domain, which interacts with the MRE domain of the PDZ protein, Discs Lost (DLT). Pals1 and its Drosophila homologue are important adaptors that allow Crumbs to interact with PATJ/Disc Lost.

DiscussionHere, we describe the role of the L27N domain of the Maguk protein, Pals1, in tight junction targeting. We have identified the binding partner of the Pals1 L27N domain and have named it PATJ. PATJ contains 10 PDZ domains but bears a unique NH2-terminal MRE domain that binds to the L27N domain of Pals1. MRE domains are present in other PDZ proteins such as SAP97 and MUPP1. We have found that this region in SAP97 and MUPP1 can bind to the L27N domain of mLin-2–Cask and Pals1, respectively (Lee et al., 2002; unpublished data). Thus, the interaction of L27 domains with MRE domains appears to be a set of interactions used by several Maguk and PDZ domain proteins.The Drosophila PATJ ortholog, Discs Lost, has been reported to play a crucial role in cell polarity (Bhat et al., 1999). Epithelial cells lacking Discs Lost mislocalize basolateral proteins to the apical surface, and absence of Discs Lost leads to loss of imaginal discs and larval lethality. Discs Lost exists in a complex with the apical polarity determinant, Crumbs. Crumbs is also crucial for cell polarity, since loss of Crumbs results in the inability to form zonula adherens from spot adherens junctions during embryogenesis (Knust et al., 1993). Overexpression of Crumbs or just its transmembrane and intracellular COOH-terminal segments results in the expansion of the apical surface, disruption of the zonula adherens, and induction of multilayered epithelia (Wodarz et al., 1995). Thus, Crumbs and its associated proteins may have an important role in the demarcation of the apical surface and the localization of the adherens junction. Since PATJ is related to Discs Lost, we wondered if PATJ was in a complex with CRB1. It has been suggested that the first PDZ domain of Discs Lost interacts with the extreme COOH terminus of Drosophila Crumbs (Bhat et al., 1999). Based on this, we predicted that the second PDZ domain of PATJ (Fig. 5 B) would interact directly with the CRB1 tail. Surprisingly, we found that the CRB1 tail did not interact directly with any of the 10 PDZ domains of PATJ but rather with the single PDZ domain of Pals1. Thus, we find that CRB1 interacts with the Pals1 PDZ domain, whereas in turn Pals1 interacts with PATJ via its L27N domain (Fig. 10). Similarly we could not find a direct interaction between the Drosophila Crumbs tail and Discs Lost. Instead, the Crumbs tail was observed to bind the Drosophila Pals1 homologue, CG1617 (Fig. 9 B).Interestingly, CG1617 maps to the exact region on Drosophila chromosome X as Stardust, a gene known to genetically interact with Crumbs (Tepass and Knust, 1993; Grawe et al., 1996; Muller and Wieschaus, 1996). Deletion of either Stardust or Crumbs results in very similar phenotypes, including polarity defects and failure to form a proper zonula adherens. The Stardust gene product has been recently reported to be a Maguk protein (Bachmann et al., 2001; Hong et al., 2001). The core Maguk region of Stardust, including the L27 domain, PDZ, SH3, and GUK, is encoded by CG1617. These reports also find that Drosophila Crumbs can bind Stardust similar to our data demonstrating the association between Pals1–CG1617 and Crumbs. In Stardust mutants, both Discs Lost and Crumbs are mislocalized, indicating that these proteins likely cooperate to function in polarity determination. However, in these studies it was not clear how this genetic interaction might occur at the biochemical level. Previous reports collectively suggest that Discs Lost and Stardust might compete for binding to the Crumbs tail (Bhat et al., 1999; Klebes and Knust, 2000). However, our data indicate that like Pals1, Stardust acts to nucleate a heterotrimeric complex containing Discs Lost and Crumbs (Fig. 10).This notion fits well with the previous observations that Crumbs stability at the apical membrane is dependent on both Stardust and Discs Lost (Tepass and Knust, 1993; Hong et al., 2001). In turn, the recruitment of Discs Lost to the subapical complex (SAC; also known as the marginal zone) is dependent on Crumbs and Stardust (Bhat et al., 1999; Bachmann et al., 2001; Hong et al., 2001; Tepass et al., 2001). In the initial cellularization of blastoderm epithelia in Drosophila, Crumbs and Stardust are found at the apical surface, whereas Discs Lost sits at the lateral furrow where intercellular junctions are beginning to develop. As junctions mature, Discs Lost is recruited to the SAC where it colocalizes with Crumbs and Stardust. This process appears to be crucial for proper conversion of spot adherens junctions into zonula adherens. In the Stardust mutant, Discs Lost starts in the correct region on the lateral furrow but fails to be recruited to the SAC (Bachmann et al., 2001). Based on our biochemical data, it could be proposed that the interactions of Stardust and Discs Lost via their respective L27 and MRE domains allow the proteins to interact at the SAC.In this study, PATJ, Pals1, and CRB1 were shown to localize to tight junctions. Some of our confocal microscopy data suggest that Pals1 and PATJ may colocalize to the upper aspect of tight junctions (unpublished data). In agreement, PATJ and its paralogue, MUPP1, have been shown to localize to the upper aspect of the tight junction by immunoelectron microscopy (Hamazaki et al., 2002; A. LeBivic, personal communication). In Drosophila, Discs Lost, Stardust, and Crumbs localize to the SAC, which lies just superior to the adherens junction (Klebes and Knust, 2000; Tanentzapf et al., 2000; Bachmann et al., 2001; Hong et al., 2001). In both situations, the proteins sit at the surface where the lateral membrane joins to the apical surface. This close conservation of localization suggests conserved function of this complex in epithelial function. Crumbs may serve as an apical polarity determinant by tethering the apical cytoskeleton to the tight junction via Pals1 and PATJ in mammalian cells. Maguk proteins, like Pals1, mLin-2–Cask, and Stardust, have lysine-rich regions between the GUK and SH3 domains where actin binding members of the protein 4.1 superfamily might bind. This may allow Maguk proteins to control the cortical actin cytoskeleton and epithelial structure. This would be similar to the role that another Maguk, p55, plays in erythrocytes where it couples a transmembrane protein, glycophorin C, to the actin cytoskeleton via protein 4.1 (Marfatia et al., 1995; Nunomura et al., 2000). This complex is important for the proper maintenance of the cortical cytoskeleton in and the biconcave shape of red cells.Another mechanism by which the Pals1 complex might control cell polarity is by the targeting of vesicles to the apical surface. Previously, we have demonstrated that the mLin-7–containing complexes can control vesicle trafficking during endocytosis (Straight et al., 2001), and this has been suggested for other mammalian PDZ domain proteins (Cao et al., 1999). It has also been proposed that the Drosophila PDZ domain proteins, Scribble and Discs Large, might control vesicle trafficking via their interaction with the putative SNARE binding protein lethal giant larvae (Bilder et al., 2000). Similarly, the PATJ–Pals1–Crumbs complex might contribute to polarity by targeting vesicles to the tight junction and/or the apical surface. Thus, it seems likely that these PDZ proteins serve multiple roles, including the demarcation of surface boundaries necessary for cellular polarity and the proper targeting of other proteins in both vertebrates and invertebrates.Although the expression of PATJ (1–238) resulted in the absence of Pals1 from tight junctions, this dominant negative PATJ protein did not have a deleterious effect on the overall polarity of MDCK cells grown on filters. There are many possible explanations for this lack of effect. First, there may be redundant polarizing signals in MDCK cells that may negate the need for this complex in the polarity of cells grown on filters. Second, it may be possible that in mammalian cells this complex is necessary for the targeting of only a subset of polarity proteins rather than having global effects on cell polarization. It is interesting to note that in C. elegans, two Crumbs genes are known to exist; however, no obvious polarity defects were detected when antisense approaches were used to reduce expression of both genes (Bossinger et al., 2001). Similarly, it has been found in Drosophila that although Crumbs is necessary for normal polarity early in development, it may not be required at later developmental stages (Tepass et al., 2001). Further studies will be necessary to examine the exact role(s) of the Pals1 complex in mammalian cells. However, the high degree of evolutionary conservation in regard to the localization and interactions between these proteins suggests an important role of this complex in epithelial cell biology.

Here, we describe the role of the L27N domain of the Maguk protein, Pals1, in tight junction targeting. We have identified the binding partner of the Pals1 L27N domain and have named it PATJ. PATJ contains 10 PDZ domains but bears a unique NH2-terminal MRE domain that binds to the L27N domain of Pals1. MRE domains are present in other PDZ proteins such as SAP97 and MUPP1. We have found that this region in SAP97 and MUPP1 can bind to the L27N domain of mLin-2–Cask and Pals1, respectively (Lee et al., 2002; unpublished data). Thus, the interaction of L27 domains with MRE domains appears to be a set of interactions used by several Maguk and PDZ domain proteins.

The Drosophila PATJ ortholog, Discs Lost, has been reported to play a crucial role in cell polarity (Bhat et al., 1999). Epithelial cells lacking Discs Lost mislocalize basolateral proteins to the apical surface, and absence of Discs Lost leads to loss of imaginal discs and larval lethality. Discs Lost exists in a complex with the apical polarity determinant, Crumbs. Crumbs is also crucial for cell polarity, since loss of Crumbs results in the inability to form zonula adherens from spot adherens junctions during embryogenesis (Knust et al., 1993). Overexpression of Crumbs or just its transmembrane and intracellular COOH-terminal segments results in the expansion of the apical surface, disruption of the zonula adherens, and induction of multilayered epithelia (Wodarz et al., 1995). Thus, Crumbs and its associated proteins may have an important role in the demarcation of the apical surface and the localization of the adherens junction. Since PATJ is related to Discs Lost, we wondered if PATJ was in a complex with CRB1. It has been suggested that the first PDZ domain of Discs Lost interacts with the extreme COOH terminus of Drosophila Crumbs (Bhat et al., 1999). Based on this, we predicted that the second PDZ domain of PATJ (Fig. 5 B) would interact directly with the CRB1 tail. Surprisingly, we found that the CRB1 tail did not interact directly with any of the 10 PDZ domains of PATJ but rather with the single PDZ domain of Pals1. Thus, we find that CRB1 interacts with the Pals1 PDZ domain, whereas in turn Pals1 interacts with PATJ via its L27N domain (Fig. 10). Similarly we could not find a direct interaction between the Drosophila Crumbs tail and Discs Lost. Instead, the Crumbs tail was observed to bind the Drosophila Pals1 homologue, CG1617 (Fig. 9 B).

Interestingly, CG1617 maps to the exact region on Drosophila chromosome X as Stardust, a gene known to genetically interact with Crumbs (Tepass and Knust, 1993; Grawe et al., 1996; Muller and Wieschaus, 1996). Deletion of either Stardust or Crumbs results in very similar phenotypes, including polarity defects and failure to form a proper zonula adherens. The Stardust gene product has been recently reported to be a Maguk protein (Bachmann et al., 2001; Hong et al., 2001). The core Maguk region of Stardust, including the L27 domain, PDZ, SH3, and GUK, is encoded by CG1617. These reports also find that Drosophila Crumbs can bind Stardust similar to our data demonstrating the association between Pals1–CG1617 and Crumbs. In Stardust mutants, both Discs Lost and Crumbs are mislocalized, indicating that these proteins likely cooperate to function in polarity determination. However, in these studies it was not clear how this genetic interaction might occur at the biochemical level. Previous reports collectively suggest that Discs Lost and Stardust might compete for binding to the Crumbs tail (Bhat et al., 1999; Klebes and Knust, 2000). However, our data indicate that like Pals1, Stardust acts to nucleate a heterotrimeric complex containing Discs Lost and Crumbs (Fig. 10).

This notion fits well with the previous observations that Crumbs stability at the apical membrane is dependent on both Stardust and Discs Lost (Tepass and Knust, 1993; Hong et al., 2001). In turn, the recruitment of Discs Lost to the subapical complex (SAC; also known as the marginal zone) is dependent on Crumbs and Stardust (Bhat et al., 1999; Bachmann et al., 2001; Hong et al., 2001; Tepass et al., 2001). In the initial cellularization of blastoderm epithelia in Drosophila, Crumbs and Stardust are found at the apical surface, whereas Discs Lost sits at the lateral furrow where intercellular junctions are beginning to develop. As junctions mature, Discs Lost is recruited to the SAC where it colocalizes with Crumbs and Stardust. This process appears to be crucial for proper conversion of spot adherens junctions into zonula adherens. In the Stardust mutant, Discs Lost starts in the correct region on the lateral furrow but fails to be recruited to the SAC (Bachmann et al., 2001). Based on our biochemical data, it could be proposed that the interactions of Stardust and Discs Lost via their respective L27 and MRE domains allow the proteins to interact at the SAC.

In this study, PATJ, Pals1, and CRB1 were shown to localize to tight junctions. Some of our confocal microscopy data suggest that Pals1 and PATJ may colocalize to the upper aspect of tight junctions (unpublished data). In agreement, PATJ and its paralogue, MUPP1, have been shown to localize to the upper aspect of the tight junction by immunoelectron microscopy (Hamazaki et al., 2002; A. LeBivic, personal communication). In Drosophila, Discs Lost, Stardust, and Crumbs localize to the SAC, which lies just superior to the adherens junction (Klebes and Knust, 2000; Tanentzapf et al., 2000; Bachmann et al., 2001; Hong et al., 2001). In both situations, the proteins sit at the surface where the lateral membrane joins to the apical surface. This close conservation of localization suggests conserved function of this complex in epithelial function. Crumbs may serve as an apical polarity determinant by tethering the apical cytoskeleton to the tight junction via Pals1 and PATJ in mammalian cells. Maguk proteins, like Pals1, mLin-2–Cask, and Stardust, have lysine-rich regions between the GUK and SH3 domains where actin binding members of the protein 4.1 superfamily might bind. This may allow Maguk proteins to control the cortical actin cytoskeleton and epithelial structure. This would be similar to the role that another Maguk, p55, plays in erythrocytes where it couples a transmembrane protein, glycophorin C, to the actin cytoskeleton via protein 4.1 (Marfatia et al., 1995; Nunomura et al., 2000). This complex is important for the proper maintenance of the cortical cytoskeleton in and the biconcave shape of red cells.

Another mechanism by which the Pals1 complex might control cell polarity is by the targeting of vesicles to the apical surface. Previously, we have demonstrated that the mLin-7–containing complexes can control vesicle trafficking during endocytosis (Straight et al., 2001), and this has been suggested for other mammalian PDZ domain proteins (Cao et al., 1999). It has also been proposed that the Drosophila PDZ domain proteins, Scribble and Discs Large, might control vesicle trafficking via their interaction with the putative SNARE binding protein lethal giant larvae (Bilder et al., 2000). Similarly, the PATJ–Pals1–Crumbs complex might contribute to polarity by targeting vesicles to the tight junction and/or the apical surface. Thus, it seems likely that these PDZ proteins serve multiple roles, including the demarcation of surface boundaries necessary for cellular polarity and the proper targeting of other proteins in both vertebrates and invertebrates.

Although the expression of PATJ (1–238) resulted in the absence of Pals1 from tight junctions, this dominant negative PATJ protein did not have a deleterious effect on the overall polarity of MDCK cells grown on filters. There are many possible explanations for this lack of effect. First, there may be redundant polarizing signals in MDCK cells that may negate the need for this complex in the polarity of cells grown on filters. Second, it may be possible that in mammalian cells this complex is necessary for the targeting of only a subset of polarity proteins rather than having global effects on cell polarization. It is interesting to note that in C. elegans, two Crumbs genes are known to exist; however, no obvious polarity defects were detected when antisense approaches were used to reduce expression of both genes (Bossinger et al., 2001). Similarly, it has been found in Drosophila that although Crumbs is necessary for normal polarity early in development, it may not be required at later developmental stages (Tepass et al., 2001). Further studies will be necessary to examine the exact role(s) of the Pals1 complex in mammalian cells. However, the high degree of evolutionary conservation in regard to the localization and interactions between these proteins suggests an important role of this complex in epithelial cell biology.

Materials and methodsDNA constructsFull-length Pals1 and Pals2 constructs have been described elsewhere (Kamberov et al., 2000). Truncation mutations in Pals1 were generated by PCR and cloned into pRK5Myc and pEYFP-N1 (CLONTECH Laboratories, Inc.). Internal domain deletions and point mutations were generated as described (Makarova et al., 2000). The DNA encoding the Pals1 L27N domain (amino acids 116–185) was amplified by PCR and inserted into the pGSTag vector. This construct allowed for expression of the GST-Pals1L27N fusion protein in BL21 bacterial cells.To synthesize the Pals2_Pals1L27N chimeric construct, deletion mutagenesis was initially performed on the pEYFP-N1-Pals2 plasmid with a single primer, 5′-CGACCGAAATCGAAATGCAGGGTTTAAACGATTCAAAGTTAGAAGCTGTG-3′, as described previously (Makarova et al., 2000). This replaces the DNA encoding the L27N domain of Pals2 with a PmeI site. The DNA encoding the L27N domain of Pals1 was amplified by PCR and inserted into the mutated pEYFP-N1-Pals2 construct. For control experiments, the Pals1 L27N domain was removed from the chimeric construct via the Pme1 sites, and the Pals2 L27N domain was reinserted.Human kidney Quickclone cDNA (CLONTECH Laboratories, Inc.) was used to amplify the cDNA encoding PATJ. The sense primer, 5′-TTAATTCTCGAGCGATGCCTGAAAATCCTGCTAC-3′, corresponding to the NH2 terminus of hINADL, and the antisense primer 5′-TTAATTGGTACCAGGAACTGGGAAACATTAG-3′, corresponding to the 3′ untranslated region of the PDZ domain protein 3′ variant 4, was used in the PCR reaction. The 5.8-kb PCR product was cloned into pcDNA3.1(−)/HA (Straight et al., 2001). Sequencing of the plasmid insert revealed a 5.4-kb ORF. This construct encodes an ∼190-kD protein with an NH2-terminal triple HA epitope tag. The 5.4-kb ORF was subcloned into pEYFP-C1. PCR products encoding PATJ (1–45), PATJ (1–68), PATJ (1–109), and PATJ (1–238) were cloned into pGSTag. The DNA encoding PATJ (1–238) was also subcloned into pRK5myc. The pGSTag-mLin-7 (1–92) construct has been described elsewhere (Borg et al., 1999).Human retina Marathon Ready cDNA (CLONTECH Laboratories, Inc.) was used to amplify the full-length CRB1. The CRB1 PCR product was cloned into the pGEM-T vector (Promega). The sequence encoding Crumbs lacking the signal peptide was amplified by PCR using the following primers: 5′-TATAATAAGCTTGAGCAGAAGCTGATCAGCGAGGAGGACCTGTTTTGCAATAAAAACAACACCAGGTGCC-3′ (sense) and 5′-TAATTAGGATCCCATGCTCCTAAATCAGTCTCTCCATTGCAGG-3′ (antisense). The underlined sequence in the sense primer encodes for the Myc epitope tag (EQKLISEEDL). The resulting PCR product was cloned into pSecTag2B. This vector already contains an immunoglobulin κ-chain signal peptide sequence, allowing for the expression of an NH2-terminal Myc-tagged Crumbs complete with transmembrane segment and the cytoplasmic 37 amino acid tail. Drosophila embryo Quickclone cDNA (CLONTECH Laboratories, Inc.) was used to PCR amplify the cDNA encoding Discs Lost and the PDZ domain containing fragments of CG1617. The PCR products were cloned into pRK5myc.All constructs were verified by automated sequencing at the University of Michigan DNA Sequencing Core Facility.Cell culture and transfectionHuman embryonic kidney 293 (HEK293) and MDCK cells were grown in DME (Life Technologies, Inc.) containing 100 U/ml penicillin, 100 μg/ml streptomycin sulfate, 2 mM l-glutamine (Life Technologies, Inc.), and 10% FBS. HEK293 and MDCK cells were transfected using the Fugene 6 transfection reagent (Roche Molecular Biochemicals). 2 d after transfection, HEK293 cells were harvested in lysis buffer, and MDCK cells were selected under 600 μg/ml G418 (Life Technologies, Inc.) or 200 μg/ml zeocin (Invitrogen).AntibodiesThe anti-Myc monoclonal antibody (clone 9E10) was used for immunoprecipitation, immunostaining, and immunoblotting. The anti-HA monoclonal antibodies 3F10 and 12CA5 (Roche Molecular Biochemicals) were used for immunoprecipitation and immunoblot, respectively. Mouse monoclonal and rabbit polyclonal anti-EYFP antibodies (CLONTECH Laboratories, Inc.) were used for immunoblotting and immunoprecipitations, respectively. Mouse monoclonal and rabbit polyclonal anti–ZO-1 antibodies and rat antiuvomorulin/E-cadherin monoclonal antibodies were purchased from Zymed Laboratories and Sigma-Aldrich, respectively. Fluorochrome-conjugated secondary antibodies were purchased from Jackson ImmunoResearch Laboratories and Molecular Probes. For generation of anti-Pals1 antibodies, rabbits were immunized with a Pals1 peptide (NH2-LRLINKLDTEPQWVPSTWLR-COOH) coupled to keyhole limpet hemocyanin. Anti-Pals1 antibodies were purified using an Affigel 15 column (Bio-Rad Laboratories, Inc.) previously coupled to the peptide. To obtain anti-PATJ antisera, rabbits were injected with GST-PATJ(1–238).Immunostaining of MDCK cells and confocal microscopyMDCK cells, grown at confluence on Transwell membrane filters, were washed with PBS, fixed with 4% formaldehyde/PBS for 20 min, and permeabilized with either 0.1% Triton X-100/PBS or 1% SDS/PBS for 3 min. After blocking with goat serum, filters were incubated with primary antibodies diluted in 2% goat serum/PBS (mouse anti-Myc at 1:1,000, mouse anti–ZO-1 at 1:2,000, rabbit anti–ZO-1 at 1:500, rabbit anti-Pals1 at 1:1,000, and rat anti–E-cadherin at 1:1,600) in a humidified chamber overnight at 30°C. After five washes with 2% goat serum/PBS, filters were incubated with secondary antibodies coupled to Alexa Fluor 488, 594, and/or 647 (diluted at 1:1,000 in 2% goat serum/PBS) for 2 h at 30°C. Filters were washed with 2% goat serum/PBS and mounted on slides with ProLong antifade reagent (Molecular Probes, Inc.). Confocal laser-scanning microscopy was performed at the Microscopy and Image-analysis Laboratory at the University of Michigan using a ZEISS LSM510 Axiovert 100M inverted confocal microscope.Immunostaining of rat kidneyCryostat sections (7 μm) of 2% paraformaldehyde perfused fixed frozen rat kidney were treated with Retrieve-all 1 (Signet Labs) for 2 h at 90°C, cooled down to room temperature, and washed with PBS. Sections were permeabilized with 1% SDS/PBS, rinsed with PBS, and blocked using PBS supplemented with 10% goat serum and 10% rat serum. Subsequently, the sections were incubated with anti-Pals1 and mouse anti–ZO-1 antibodies. Sections were then washed twice with PBS and incubated with Cy3- and FITC-conjugated secondary antibodies. The sections were again washed with PBS, mounted, and photographed with a Nikon 807427 fluorescence microscope (Mager Scientific, Inc.) using the Metamorph imaging program (Universal Imaging Corp.).Immunopurification of Pals1-associated proteinsImmunoblotting, immunoprecipitation, GST pulldown, and Far Western blotting experiments were performed as described previously (Stein et al., 1994; Kamberov et al., 2000). For the large scale purification of the Pals1 L27N binding partner, an anti-Myc immunoaffinity column was initially prepared. Essentially, anti-Myc antibody was bound to protein A–Sepharose beads. The beads were washed three times with cross-linking buffer (0.2 M sodium borate, pH 9.0). Antibodies were then cross-linked to the beads in cross-linking buffer supplemented with 20 mM dimethylpimelimidate.MDCK stable cell lines expressing either Myc-Pals1 or Myc-Pals1Δ(U1 + L27N) were each grown to confluency on 150 15-cm tissue culture dishes. Cells were harvested in 65 ml lysis buffer (50 mM Hepes, pH 7.5, 10% glycerol, 150 mM NaCl, 1% Triton X-100, 1.5 mM MgCl2, and 1 mM EGTA) supplemented with 1 mM PMSF, 10 μg/ml aprotinin, and 10 μg/ml leupeptin. The lysates were cleared by centrifugation at 16,000 g for 30 min at 4°C.Cell lysates were preadsorbed with 1.0 ml protein A–Sepharose beads and subsequently incubated with 0.5 ml anti-Myc beads overnight at 4°C. The beads were transferred to a column, washed rapidly with 8 ml of ice-cold lysis buffer, and subsequently with 10 ml of 10 mM Tris, pH 7.4. Bound proteins were eluted with elution buffer (100 mM glycine, pH 2.5, and 10% ethylene glycol), dialyzed against 1× PBS, and concentrated using a Centricon-10 apparatus (Amicon). Proteins were resolved on 3–8% Tris-Acetate Novex gradient gels and visualized by silver staining or colloidal Coomassie blue staining (Invitrogen). The band corresponding to the Pals1 binding partner was excised, washed twice in 50% acetonitrile, and submitted to the Harvard Microchemistry Facility for MS–MS sequence analysis.Online supplemental materialsFig. S1, showing PCR analysis of CRB1 expression in different human tissues, is available online at http://www.jcb.org/cgi/content/full/jcb.200109010/DC1. Portions of the human CRB1 cDNA were PCR amplified from different tissues using Marathon Ready Tissue cDNA pools from CLONTECH Laboratories, Inc. The results suggest that isoforms of CRB1 could be expressed outside the nervous system and retina.

Materials and methods

DNA constructsFull-length Pals1 and Pals2 constructs have been described elsewhere (Kamberov et al., 2000). Truncation mutations in Pals1 were generated by PCR and cloned into pRK5Myc and pEYFP-N1 (CLONTECH Laboratories, Inc.). Internal domain deletions and point mutations were generated as described (Makarova et al., 2000). The DNA encoding the Pals1 L27N domain (amino acids 116–185) was amplified by PCR and inserted into the pGSTag vector. This construct allowed for expression of the GST-Pals1L27N fusion protein in BL21 bacterial cells.To synthesize the Pals2_Pals1L27N chimeric construct, deletion mutagenesis was initially performed on the pEYFP-N1-Pals2 plasmid with a single primer, 5′-CGACCGAAATCGAAATGCAGGGTTTAAACGATTCAAAGTTAGAAGCTGTG-3′, as described previously (Makarova et al., 2000). This replaces the DNA encoding the L27N domain of Pals2 with a PmeI site. The DNA encoding the L27N domain of Pals1 was amplified by PCR and inserted into the mutated pEYFP-N1-Pals2 construct. For control experiments, the Pals1 L27N domain was removed from the chimeric construct via the Pme1 sites, and the Pals2 L27N domain was reinserted.Human kidney Quickclone cDNA (CLONTECH Laboratories, Inc.) was used to amplify the cDNA encoding PATJ. The sense primer, 5′-TTAATTCTCGAGCGATGCCTGAAAATCCTGCTAC-3′, corresponding to the NH2 terminus of hINADL, and the antisense primer 5′-TTAATTGGTACCAGGAACTGGGAAACATTAG-3′, corresponding to the 3′ untranslated region of the PDZ domain protein 3′ variant 4, was used in the PCR reaction. The 5.8-kb PCR product was cloned into pcDNA3.1(−)/HA (Straight et al., 2001). Sequencing of the plasmid insert revealed a 5.4-kb ORF. This construct encodes an ∼190-kD protein with an NH2-terminal triple HA epitope tag. The 5.4-kb ORF was subcloned into pEYFP-C1. PCR products encoding PATJ (1–45), PATJ (1–68), PATJ (1–109), and PATJ (1–238) were cloned into pGSTag. The DNA encoding PATJ (1–238) was also subcloned into pRK5myc. The pGSTag-mLin-7 (1–92) construct has been described elsewhere (Borg et al., 1999).Human retina Marathon Ready cDNA (CLONTECH Laboratories, Inc.) was used to amplify the full-length CRB1. The CRB1 PCR product was cloned into the pGEM-T vector (Promega). The sequence encoding Crumbs lacking the signal peptide was amplified by PCR using the following primers: 5′-TATAATAAGCTTGAGCAGAAGCTGATCAGCGAGGAGGACCTGTTTTGCAATAAAAACAACACCAGGTGCC-3′ (sense) and 5′-TAATTAGGATCCCATGCTCCTAAATCAGTCTCTCCATTGCAGG-3′ (antisense). The underlined sequence in the sense primer encodes for the Myc epitope tag (EQKLISEEDL). The resulting PCR product was cloned into pSecTag2B. This vector already contains an immunoglobulin κ-chain signal peptide sequence, allowing for the expression of an NH2-terminal Myc-tagged Crumbs complete with transmembrane segment and the cytoplasmic 37 amino acid tail. Drosophila embryo Quickclone cDNA (CLONTECH Laboratories, Inc.) was used to PCR amplify the cDNA encoding Discs Lost and the PDZ domain containing fragments of CG1617. The PCR products were cloned into pRK5myc.All constructs were verified by automated sequencing at the University of Michigan DNA Sequencing Core Facility.

DNA constructs

Full-length Pals1 and Pals2 constructs have been described elsewhere (Kamberov et al., 2000). Truncation mutations in Pals1 were generated by PCR and cloned into pRK5Myc and pEYFP-N1 (CLONTECH Laboratories, Inc.). Internal domain deletions and point mutations were generated as described (Makarova et al., 2000). The DNA encoding the Pals1 L27N domain (amino acids 116–185) was amplified by PCR and inserted into the pGSTag vector. This construct allowed for expression of the GST-Pals1L27N fusion protein in BL21 bacterial cells.

To synthesize the Pals2_Pals1L27N chimeric construct, deletion mutagenesis was initially performed on the pEYFP-N1-Pals2 plasmid with a single primer, 5′-CGACCGAAATCGAAATGCAGGGTTTAAACGATTCAAAGTTAGAAGCTGTG-3′, as described previously (Makarova et al., 2000). This replaces the DNA encoding the L27N domain of Pals2 with a PmeI site. The DNA encoding the L27N domain of Pals1 was amplified by PCR and inserted into the mutated pEYFP-N1-Pals2 construct. For control experiments, the Pals1 L27N domain was removed from the chimeric construct via the Pme1 sites, and the Pals2 L27N domain was reinserted.

Human kidney Quickclone cDNA (CLONTECH Laboratories, Inc.) was used to amplify the cDNA encoding PATJ. The sense primer, 5′-TTAATTCTCGAGCGATGCCTGAAAATCCTGCTAC-3′, corresponding to the NH2 terminus of hINADL, and the antisense primer 5′-TTAATTGGTACCAGGAACTGGGAAACATTAG-3′, corresponding to the 3′ untranslated region of the PDZ domain protein 3′ variant 4, was used in the PCR reaction. The 5.8-kb PCR product was cloned into pcDNA3.1(−)/HA (Straight et al., 2001). Sequencing of the plasmid insert revealed a 5.4-kb ORF. This construct encodes an ∼190-kD protein with an NH2-terminal triple HA epitope tag. The 5.4-kb ORF was subcloned into pEYFP-C1. PCR products encoding PATJ (1–45), PATJ (1–68), PATJ (1–109), and PATJ (1–238) were cloned into pGSTag. The DNA encoding PATJ (1–238) was also subcloned into pRK5myc. The pGSTag-mLin-7 (1–92) construct has been described elsewhere (Borg et al., 1999).

Human retina Marathon Ready cDNA (CLONTECH Laboratories, Inc.) was used to amplify the full-length CRB1. The CRB1 PCR product was cloned into the pGEM-T vector (Promega). The sequence encoding Crumbs lacking the signal peptide was amplified by PCR using the following primers: 5′-TATAATAAGCTTGAGCAGAAGCTGATCAGCGAGGAGGACCTGTTTTGCAATAAAAACAACACCAGGTGCC-3′ (sense) and 5′-TAATTAGGATCCCATGCTCCTAAATCAGTCTCTCCATTGCAGG-3′ (antisense). The underlined sequence in the sense primer encodes for the Myc epitope tag (EQKLISEEDL). The resulting PCR product was cloned into pSecTag2B. This vector already contains an immunoglobulin κ-chain signal peptide sequence, allowing for the expression of an NH2-terminal Myc-tagged Crumbs complete with transmembrane segment and the cytoplasmic 37 amino acid tail. Drosophila embryo Quickclone cDNA (CLONTECH Laboratories, Inc.) was used to PCR amplify the cDNA encoding Discs Lost and the PDZ domain containing fragments of CG1617. The PCR products were cloned into pRK5myc.

All constructs were verified by automated sequencing at the University of Michigan DNA Sequencing Core Facility.

Cell culture and transfectionHuman embryonic kidney 293 (HEK293) and MDCK cells were grown in DME (Life Technologies, Inc.) containing 100 U/ml penicillin, 100 μg/ml streptomycin sulfate, 2 mM l-glutamine (Life Technologies, Inc.), and 10% FBS. HEK293 and MDCK cells were transfected using the Fugene 6 transfection reagent (Roche Molecular Biochemicals). 2 d after transfection, HEK293 cells were harvested in lysis buffer, and MDCK cells were selected under 600 μg/ml G418 (Life Technologies, Inc.) or 200 μg/ml zeocin (Invitrogen).

Cell culture and transfection

Human embryonic kidney 293 (HEK293) and MDCK cells were grown in DME (Life Technologies, Inc.) containing 100 U/ml penicillin, 100 μg/ml streptomycin sulfate, 2 mM l-glutamine (Life Technologies, Inc.), and 10% FBS. HEK293 and MDCK cells were transfected using the Fugene 6 transfection reagent (Roche Molecular Biochemicals). 2 d after transfection, HEK293 cells were harvested in lysis buffer, and MDCK cells were selected under 600 μg/ml G418 (Life Technologies, Inc.) or 200 μg/ml zeocin (Invitrogen).

AntibodiesThe anti-Myc monoclonal antibody (clone 9E10) was used for immunoprecipitation, immunostaining, and immunoblotting. The anti-HA monoclonal antibodies 3F10 and 12CA5 (Roche Molecular Biochemicals) were used for immunoprecipitation and immunoblot, respectively. Mouse monoclonal and rabbit polyclonal anti-EYFP antibodies (CLONTECH Laboratories, Inc.) were used for immunoblotting and immunoprecipitations, respectively. Mouse monoclonal and rabbit polyclonal anti–ZO-1 antibodies and rat antiuvomorulin/E-cadherin monoclonal antibodies were purchased from Zymed Laboratories and Sigma-Aldrich, respectively. Fluorochrome-conjugated secondary antibodies were purchased from Jackson ImmunoResearch Laboratories and Molecular Probes. For generation of anti-Pals1 antibodies, rabbits were immunized with a Pals1 peptide (NH2-LRLINKLDTEPQWVPSTWLR-COOH) coupled to keyhole limpet hemocyanin. Anti-Pals1 antibodies were purified using an Affigel 15 column (Bio-Rad Laboratories, Inc.) previously coupled to the peptide. To obtain anti-PATJ antisera, rabbits were injected with GST-PATJ(1–238).

The anti-Myc monoclonal antibody (clone 9E10) was used for immunoprecipitation, immunostaining, and immunoblotting. The anti-HA monoclonal antibodies 3F10 and 12CA5 (Roche Molecular Biochemicals) were used for immunoprecipitation and immunoblot, respectively. Mouse monoclonal and rabbit polyclonal anti-EYFP antibodies (CLONTECH Laboratories, Inc.) were used for immunoblotting and immunoprecipitations, respectively. Mouse monoclonal and rabbit polyclonal anti–ZO-1 antibodies and rat antiuvomorulin/E-cadherin monoclonal antibodies were purchased from Zymed Laboratories and Sigma-Aldrich, respectively. Fluorochrome-conjugated secondary antibodies were purchased from Jackson ImmunoResearch Laboratories and Molecular Probes. For generation of anti-Pals1 antibodies, rabbits were immunized with a Pals1 peptide (NH2-LRLINKLDTEPQWVPSTWLR-COOH) coupled to keyhole limpet hemocyanin. Anti-Pals1 antibodies were purified using an Affigel 15 column (Bio-Rad Laboratories, Inc.) previously coupled to the peptide. To obtain anti-PATJ antisera, rabbits were injected with GST-PATJ(1–238).

Immunostaining of MDCK cells and confocal microscopyMDCK cells, grown at confluence on Transwell membrane filters, were washed with PBS, fixed with 4% formaldehyde/PBS for 20 min, and permeabilized with either 0.1% Triton X-100/PBS or 1% SDS/PBS for 3 min. After blocking with goat serum, filters were incubated with primary antibodies diluted in 2% goat serum/PBS (mouse anti-Myc at 1:1,000, mouse anti–ZO-1 at 1:2,000, rabbit anti–ZO-1 at 1:500, rabbit anti-Pals1 at 1:1,000, and rat anti–E-cadherin at 1:1,600) in a humidified chamber overnight at 30°C. After five washes with 2% goat serum/PBS, filters were incubated with secondary antibodies coupled to Alexa Fluor 488, 594, and/or 647 (diluted at 1:1,000 in 2% goat serum/PBS) for 2 h at 30°C. Filters were washed with 2% goat serum/PBS and mounted on slides with ProLong antifade reagent (Molecular Probes, Inc.). Confocal laser-scanning microscopy was performed at the Microscopy and Image-analysis Laboratory at the University of Michigan using a ZEISS LSM510 Axiovert 100M inverted confocal microscope.

Immunostaining of MDCK cells and confocal microscopy

MDCK cells, grown at confluence on Transwell membrane filters, were washed with PBS, fixed with 4% formaldehyde/PBS for 20 min, and permeabilized with either 0.1% Triton X-100/PBS or 1% SDS/PBS for 3 min. After blocking with goat serum, filters were incubated with primary antibodies diluted in 2% goat serum/PBS (mouse anti-Myc at 1:1,000, mouse anti–ZO-1 at 1:2,000, rabbit anti–ZO-1 at 1:500, rabbit anti-Pals1 at 1:1,000, and rat anti–E-cadherin at 1:1,600) in a humidified chamber overnight at 30°C. After five washes with 2% goat serum/PBS, filters were incubated with secondary antibodies coupled to Alexa Fluor 488, 594, and/or 647 (diluted at 1:1,000 in 2% goat serum/PBS) for 2 h at 30°C. Filters were washed with 2% goat serum/PBS and mounted on slides with ProLong antifade reagent (Molecular Probes, Inc.). Confocal laser-scanning microscopy was performed at the Microscopy and Image-analysis Laboratory at the University of Michigan using a ZEISS LSM510 Axiovert 100M inverted confocal microscope.

Immunostaining of rat kidneyCryostat sections (7 μm) of 2% paraformaldehyde perfused fixed frozen rat kidney were treated with Retrieve-all 1 (Signet Labs) for 2 h at 90°C, cooled down to room temperature, and washed with PBS. Sections were permeabilized with 1% SDS/PBS, rinsed with PBS, and blocked using PBS supplemented with 10% goat serum and 10% rat serum. Subsequently, the sections were incubated with anti-Pals1 and mouse anti–ZO-1 antibodies. Sections were then washed twice with PBS and incubated with Cy3- and FITC-conjugated secondary antibodies. The sections were again washed with PBS, mounted, and photographed with a Nikon 807427 fluorescence microscope (Mager Scientific, Inc.) using the Metamorph imaging program (Universal Imaging Corp.).

Immunostaining of rat kidney

Cryostat sections (7 μm) of 2% paraformaldehyde perfused fixed frozen rat kidney were treated with Retrieve-all 1 (Signet Labs) for 2 h at 90°C, cooled down to room temperature, and washed with PBS. Sections were permeabilized with 1% SDS/PBS, rinsed with PBS, and blocked using PBS supplemented with 10% goat serum and 10% rat serum. Subsequently, the sections were incubated with anti-Pals1 and mouse anti–ZO-1 antibodies. Sections were then washed twice with PBS and incubated with Cy3- and FITC-conjugated secondary antibodies. The sections were again washed with PBS, mounted, and photographed with a Nikon 807427 fluorescence microscope (Mager Scientific, Inc.) using the Metamorph imaging program (Universal Imaging Corp.).

Immunopurification of Pals1-associated proteinsImmunoblotting, immunoprecipitation, GST pulldown, and Far Western blotting experiments were performed as described previously (Stein et al., 1994; Kamberov et al., 2000). For the large scale purification of the Pals1 L27N binding partner, an anti-Myc immunoaffinity column was initially prepared. Essentially, anti-Myc antibody was bound to protein A–Sepharose beads. The beads were washed three times with cross-linking buffer (0.2 M sodium borate, pH 9.0). Antibodies were then cross-linked to the beads in cross-linking buffer supplemented with 20 mM dimethylpimelimidate.MDCK stable cell lines expressing either Myc-Pals1 or Myc-Pals1Δ(U1 + L27N) were each grown to confluency on 150 15-cm tissue culture dishes. Cells were harvested in 65 ml lysis buffer (50 mM Hepes, pH 7.5, 10% glycerol, 150 mM NaCl, 1% Triton X-100, 1.5 mM MgCl2, and 1 mM EGTA) supplemented with 1 mM PMSF, 10 μg/ml aprotinin, and 10 μg/ml leupeptin. The lysates were cleared by centrifugation at 16,000 g for 30 min at 4°C.Cell lysates were preadsorbed with 1.0 ml protein A–Sepharose beads and subsequently incubated with 0.5 ml anti-Myc beads overnight at 4°C. The beads were transferred to a column, washed rapidly with 8 ml of ice-cold lysis buffer, and subsequently with 10 ml of 10 mM Tris, pH 7.4. Bound proteins were eluted with elution buffer (100 mM glycine, pH 2.5, and 10% ethylene glycol), dialyzed against 1× PBS, and concentrated using a Centricon-10 apparatus (Amicon). Proteins were resolved on 3–8% Tris-Acetate Novex gradient gels and visualized by silver staining or colloidal Coomassie blue staining (Invitrogen). The band corresponding to the Pals1 binding partner was excised, washed twice in 50% acetonitrile, and submitted to the Harvard Microchemistry Facility for MS–MS sequence analysis.

Immunopurification of Pals1-associated proteins

Immunoblotting, immunoprecipitation, GST pulldown, and Far Western blotting experiments were performed as described previously (Stein et al., 1994; Kamberov et al., 2000). For the large scale purification of the Pals1 L27N binding partner, an anti-Myc immunoaffinity column was initially prepared. Essentially, anti-Myc antibody was bound to protein A–Sepharose beads. The beads were washed three times with cross-linking buffer (0.2 M sodium borate, pH 9.0). Antibodies were then cross-linked to the beads in cross-linking buffer supplemented with 20 mM dimethylpimelimidate.

MDCK stable cell lines expressing either Myc-Pals1 or Myc-Pals1Δ(U1 + L27N) were each grown to confluency on 150 15-cm tissue culture dishes. Cells were harvested in 65 ml lysis buffer (50 mM Hepes, pH 7.5, 10% glycerol, 150 mM NaCl, 1% Triton X-100, 1.5 mM MgCl2, and 1 mM EGTA) supplemented with 1 mM PMSF, 10 μg/ml aprotinin, and 10 μg/ml leupeptin. The lysates were cleared by centrifugation at 16,000 g for 30 min at 4°C.

Cell lysates were preadsorbed with 1.0 ml protein A–Sepharose beads and subsequently incubated with 0.5 ml anti-Myc beads overnight at 4°C. The beads were transferred to a column, washed rapidly with 8 ml of ice-cold lysis buffer, and subsequently with 10 ml of 10 mM Tris, pH 7.4. Bound proteins were eluted with elution buffer (100 mM glycine, pH 2.5, and 10% ethylene glycol), dialyzed against 1× PBS, and concentrated using a Centricon-10 apparatus (Amicon). Proteins were resolved on 3–8% Tris-Acetate Novex gradient gels and visualized by silver staining or colloidal Coomassie blue staining (Invitrogen). The band corresponding to the Pals1 binding partner was excised, washed twice in 50% acetonitrile, and submitted to the Harvard Microchemistry Facility for MS–MS sequence analysis.

Online supplemental materialsFig. S1, showing PCR analysis of CRB1 expression in different human tissues, is available online at http://www.jcb.org/cgi/content/full/jcb.200109010/DC1. Portions of the human CRB1 cDNA were PCR amplified from different tissues using Marathon Ready Tissue cDNA pools from CLONTECH Laboratories, Inc. The results suggest that isoforms of CRB1 could be expressed outside the nervous system and retina.

Online supplemental materials

Fig. S1, showing PCR analysis of CRB1 expression in different human tissues, is available online at http://www.jcb.org/cgi/content/full/jcb.200109010/DC1. Portions of the human CRB1 cDNA were PCR amplified from different tissues using Marathon Ready Tissue cDNA pools from CLONTECH Laboratories, Inc. The results suggest that isoforms of CRB1 could be expressed outside the nervous system and retina.

Supplemental Material[Supplemental Material Index]

Supplemental Material

[Supplemental Material Index]
