---
pmid: '11777942'
title: Exportin-5, a novel karyopherin, mediates nuclear export of double-stranded
  RNA binding proteins.
authors:
- Brownawell AM
- Macara IG
journal: J Cell Biol
year: '2002'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2173575
doi: 10.1083/jcb.200110082
---

# Exportin-5, a novel karyopherin, mediates nuclear export of double-stranded RNA binding proteins.
**Authors:** Brownawell AM, Macara IG
**Journal:** J Cell Biol (2002)
**DOI:** [10.1083/jcb.200110082](https://doi.org/10.1083/jcb.200110082)
**PMC:** [PMC2173575](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2173575/)

## Abstract

1. J Cell Biol. 2002 Jan 7;156(1):53-64. doi: 10.1083/jcb.200110082. Epub 2002
Jan  3.

Exportin-5, a novel karyopherin, mediates nuclear export of double-stranded RNA 
binding proteins.

Brownawell AM(1), Macara IG.

Author information:
(1)Center for Cell Signaling, University of Virginia, Charlottesville, VA 22908, 
USA. amb7c@virginia.edu

We have identified a novel human karyopherin (Kap) beta family member that is 
related to human Crm1 and the Saccharomyces cerevisiae protein, Msn5p/Kap142p. 
Like other known transport receptors, this Kap binds specifically to RanGTP, 
interacts with nucleoporins, and shuttles between the nuclear and cytoplasmic 
compartments. We report that interleukin enhancer binding factor (ILF)3, a 
double-stranded RNA binding protein, associates with this Kap in a 
RanGTP-dependent manner and that its double-stranded RNA binding domain (dsRBD) 
is the limiting sequence required for this interaction. Importantly, the Kap 
interacts with dsRBDs found in several other proteins and binding is blocked by 
double-stranded RNA. We find that the dsRBD of ILF3 functions as a novel nuclear 
export sequence (NES) in intact cells, and its ability to serve as an NES is 
dependent on the expression of the Kap. In digitonin-permeabilized cells, the 
Kap but not Crm1 stimulated nuclear export of ILF3. Based on the ability of this 
Kap to mediate the export of dsRNA binding proteins, we named the protein 
exportin-5. We propose that exportin-5 is not an RNA export factor but instead 
participates in the regulated translocation of dsRBD proteins to the cytoplasm 
where they interact with target mRNAs.

DOI: 10.1083/jcb.200110082
PMCID: PMC2173575
PMID: 11777942 [Indexed for MEDLINE]

## Full Text

IntroductionMacromolecular traffic between the cytoplasmic and nuclear compartments of eukaryotic cells moves through nuclear pore complexes (NPCs)* (Stoffler et al., 1999). Assembled from proteins called nucleoporins, NPCs span the nuclear envelope and possess an aqueous channel with distinct cytoplasmic and nuclear faces. Small molecules and proteins (<50 kD) can diffuse through the pores. However, the translocation of larger molecules requires facilitated mechanisms that utilize a family of transport receptors called karyopherins (Kaps) (Gorlich and Kutay, 1999).All Kapβ family members identified to date possess three characteristics: (a) they bind directly and specifically to the RanGTPase in its GTP-bound conformation; (b) they bind directly to nucleoporins to allow movement between the cytoplasmic and nuclear compartments; and (c) they recognize and associate with their cargo in a Ran-regulated manner (for reviews see Wozniak et al., 1998; Gorlich and Kutay, 1999). The factors that alter the nucleotide-bound state of Ran are distributed asymmetrically within the cell: the RanGTPase-activating protein (RanGAP) is cytoplasmic (Bischoff et al., 1995), whereas the Ran exchange factor (RCC1) is nuclear (Bischoff and Ponstingl, 1991). Therefore, it is predicted that most of the Ran present in the nucleus is GTP bound, whereas cytoplasmic Ran is predicted to be GDP bound. The directionality of transport depends on this RanGTP gradient. Unloading of cargo from an import receptor in the nucleus is triggered by the binding of RanGTP to the receptor (Izaurralde et al., 1997). Conversely, the loading of cargo onto an export receptor is facilitated by RanGTP (Richards et al., 1997; Askjaer et al., 1998). Release of export cargo into the cytoplasm occurs in response to the hydrolysis of RanGTP to RanGDP, stimulated by RanGAP and other required cofactors (Kehlenbach et al., 1999).There are 14 structurally related Kapβ proteins in Saccharomyces cerevisiae (Kaffman and O'Shea, 1999). To date, ten Kapβ family members have been shown to act as import receptors, three as export receptors, and one as both an import and export receptor. However, in mammalian cells there are ∼22 Kapβ family members, and the function of many is unknown (Gorlich and Kutay, 1999). To date, only five mammalian Kapβ family members have been shown to act as export receptors: exportin-t, CAS, exportin-4, importin 13, and Crm1. Exportin-t is an export receptor for tRNA (Arts et al., 1998; Kutay et al., 1998). CAS is an exportin for the adaptor protein importin-α (Kutay et al., 1997). Exportin-4 mediates nuclear export of hypusine-modified eukaryotic translation initiation factor–5A (Lipowsky et al., 2000) and importin 13 exports eukaryotic translation initiation factor–1A (Mingot et al., 2001). It is widely accepted that Crm1 is the general receptor for nuclear protein export. Crm1 mediates the export of many proteins that contain leucine-rich nuclear export sequences (NES) (Fornerod et al., 1997; Stade et al., 1997). The study of Crm1-dependent export pathways has been facilitated by the discovery that leptomycin B (LMB) specifically inactivates Crm1 (Kudo et al., 1999). Other export receptors that are not members of the Kapβ family have also been identified. Mex67p mediates the export of mRNAs from nuclei in S. cerevisiae (Segref et al., 1997), and TAP/NXF family members may function similarly in metazoans (Gruter et al., 1998). Additionally, calreticulin facilitates the nuclear export of both protein kinase inhibitor (PKI) and glucocorticoid receptor in an LMB-insensitive manner (Holaska et al., 2001).While characterizing a novel Kap, we identified interleukin enhancer binding factor (ILF)3 as a transport substrate. ILF3 (also called NFAR-2) is a nuclear protein that possesses a bipartite nuclear localization signal (NLS), an RGG box RNA-binding motif, and two double-stranded RNA binding domains (dsRBDs) (Marcoulatos et al., 1998; Saunders et al., 2001). A dsRBD is an ∼70 residue motif that confers binding to double-stranded RNA (dsRNA) but not single-stranded RNA (ssRNA) or double-stranded DNA (Dreyfuss et al., 1993). DsRNA binding proteins play essential roles in mRNA localization during development (Bashirullah et al., 1998) (see Discussion).ILF3 is an ortholog of the Xenopus protein CBTF122 (Brzostowski et al., 2000). CBTF122 has the same domain structure as ILF3, and the two proteins share ∼80% identity. CBTF122 is a subunit of the CCAATT box transcription factor complex (CBTF) that regulates GATA-2 gene expression during early development (Orford et al., 1998). CBTF activity is controlled by changes in the subcellular distribution of CBTF122 (Brewer et al., 1995; Orford et al., 1998). CBTF122 is nuclear in oocytes, becomes cytoplasmic during meiotic maturation, and is then reimported into the nucleus at the midblastula transition. RNA binding is necessary for the cytoplasmic localization of CBTF122 before the midblastula transition, since the artificial destruction of RNA leads to premature nuclear accumulation of CBTF122 (Brzostowski et al., 2000).In this article, we identify and characterize exportin-5 as a novel Kapβ family member that mediates the export of dsRNA binding proteins via their RNA binding domains. Importantly, dsRNA binding to these proteins inhibits exportin-5 binding. Therefore, we propose that exportin-5 does not function as an RNA export factor but instead controls the nuclear export of dsRBD proteins. In addition, target RNA binding to dsRBD proteins may facilitate cargo unloading in the cytoplasm.

Introduction

Macromolecular traffic between the cytoplasmic and nuclear compartments of eukaryotic cells moves through nuclear pore complexes (NPCs)* (Stoffler et al., 1999). Assembled from proteins called nucleoporins, NPCs span the nuclear envelope and possess an aqueous channel with distinct cytoplasmic and nuclear faces. Small molecules and proteins (<50 kD) can diffuse through the pores. However, the translocation of larger molecules requires facilitated mechanisms that utilize a family of transport receptors called karyopherins (Kaps) (Gorlich and Kutay, 1999).

All Kapβ family members identified to date possess three characteristics: (a) they bind directly and specifically to the RanGTPase in its GTP-bound conformation; (b) they bind directly to nucleoporins to allow movement between the cytoplasmic and nuclear compartments; and (c) they recognize and associate with their cargo in a Ran-regulated manner (for reviews see Wozniak et al., 1998; Gorlich and Kutay, 1999). The factors that alter the nucleotide-bound state of Ran are distributed asymmetrically within the cell: the RanGTPase-activating protein (RanGAP) is cytoplasmic (Bischoff et al., 1995), whereas the Ran exchange factor (RCC1) is nuclear (Bischoff and Ponstingl, 1991). Therefore, it is predicted that most of the Ran present in the nucleus is GTP bound, whereas cytoplasmic Ran is predicted to be GDP bound. The directionality of transport depends on this RanGTP gradient. Unloading of cargo from an import receptor in the nucleus is triggered by the binding of RanGTP to the receptor (Izaurralde et al., 1997). Conversely, the loading of cargo onto an export receptor is facilitated by RanGTP (Richards et al., 1997; Askjaer et al., 1998). Release of export cargo into the cytoplasm occurs in response to the hydrolysis of RanGTP to RanGDP, stimulated by RanGAP and other required cofactors (Kehlenbach et al., 1999).

There are 14 structurally related Kapβ proteins in Saccharomyces cerevisiae (Kaffman and O'Shea, 1999). To date, ten Kapβ family members have been shown to act as import receptors, three as export receptors, and one as both an import and export receptor. However, in mammalian cells there are ∼22 Kapβ family members, and the function of many is unknown (Gorlich and Kutay, 1999). To date, only five mammalian Kapβ family members have been shown to act as export receptors: exportin-t, CAS, exportin-4, importin 13, and Crm1. Exportin-t is an export receptor for tRNA (Arts et al., 1998; Kutay et al., 1998). CAS is an exportin for the adaptor protein importin-α (Kutay et al., 1997). Exportin-4 mediates nuclear export of hypusine-modified eukaryotic translation initiation factor–5A (Lipowsky et al., 2000) and importin 13 exports eukaryotic translation initiation factor–1A (Mingot et al., 2001). It is widely accepted that Crm1 is the general receptor for nuclear protein export. Crm1 mediates the export of many proteins that contain leucine-rich nuclear export sequences (NES) (Fornerod et al., 1997; Stade et al., 1997). The study of Crm1-dependent export pathways has been facilitated by the discovery that leptomycin B (LMB) specifically inactivates Crm1 (Kudo et al., 1999). Other export receptors that are not members of the Kapβ family have also been identified. Mex67p mediates the export of mRNAs from nuclei in S. cerevisiae (Segref et al., 1997), and TAP/NXF family members may function similarly in metazoans (Gruter et al., 1998). Additionally, calreticulin facilitates the nuclear export of both protein kinase inhibitor (PKI) and glucocorticoid receptor in an LMB-insensitive manner (Holaska et al., 2001).

While characterizing a novel Kap, we identified interleukin enhancer binding factor (ILF)3 as a transport substrate. ILF3 (also called NFAR-2) is a nuclear protein that possesses a bipartite nuclear localization signal (NLS), an RGG box RNA-binding motif, and two double-stranded RNA binding domains (dsRBDs) (Marcoulatos et al., 1998; Saunders et al., 2001). A dsRBD is an ∼70 residue motif that confers binding to double-stranded RNA (dsRNA) but not single-stranded RNA (ssRNA) or double-stranded DNA (Dreyfuss et al., 1993). DsRNA binding proteins play essential roles in mRNA localization during development (Bashirullah et al., 1998) (see Discussion).

ILF3 is an ortholog of the Xenopus protein CBTF122 (Brzostowski et al., 2000). CBTF122 has the same domain structure as ILF3, and the two proteins share ∼80% identity. CBTF122 is a subunit of the CCAATT box transcription factor complex (CBTF) that regulates GATA-2 gene expression during early development (Orford et al., 1998). CBTF activity is controlled by changes in the subcellular distribution of CBTF122 (Brewer et al., 1995; Orford et al., 1998). CBTF122 is nuclear in oocytes, becomes cytoplasmic during meiotic maturation, and is then reimported into the nucleus at the midblastula transition. RNA binding is necessary for the cytoplasmic localization of CBTF122 before the midblastula transition, since the artificial destruction of RNA leads to premature nuclear accumulation of CBTF122 (Brzostowski et al., 2000).

In this article, we identify and characterize exportin-5 as a novel Kapβ family member that mediates the export of dsRNA binding proteins via their RNA binding domains. Importantly, dsRNA binding to these proteins inhibits exportin-5 binding. Therefore, we propose that exportin-5 does not function as an RNA export factor but instead controls the nuclear export of dsRBD proteins. In addition, target RNA binding to dsRBD proteins may facilitate cargo unloading in the cytoplasm.

ResultsMolecular cloning of exportin-5Although five mammalian Kapβs have been characterized as nuclear export receptors, four mediate the export of extremely specific cargo with only one cargo identified for each receptor. Crm1 is the only Kapβ that has been shown to function as a general export receptor. To identify additional export receptors, we searched databases with the Crm1 sequence and identified a human clone, KIAA1291, that contained an ORF of 3,291 bp but lacked an initiation codon. To obtain the complete ORF, the sequence information from this clone was used to generate primers for 5′ rapid amplification of cDNA ends (RACE). An ∼1,000 bp product was amplified from human brain cDNA, which corresponded to the expected 5′ end of the gene. The complete ORF was found to be 3,615 bp, encoding a 1,205 residue protein (∼136 kD) that is 25% identical and 49% similar to human Crm1 (Fig. 1 A). The Met designated as the initiation codon is in a good Kozak sequence context and matches the expected NH2 terminus by comparison with a murine EST sequence (sequence data available from GenBank/EMBL/DDBJ under accession no. BAB26904). This gene product was named exportin-5 because it exports protein cargo from the nucleus (see below) and was the fifth mammalian Kapβ family member initially characterized as an export receptor.Figure 1.
Exportin-5 is a Kapβ family member. (A) Amino acid sequences of exportin-5, Crm1, and the S. cerevisiae Kapβ, Msn5p, were aligned using CLUSTAL. Identical residues are shown in black; closely related residues are shown in gray. (B) Unrooted phylogenetic tree of exportin-5 and the 14 S. cerevisiae Kapβs generated using the ProtPars program in the Phylip package. A total of 1,000 boot-strapped replica datasets were run from an alignment of the S. cerevisiae Kap and exportin-5 sequences that was generated using Clustal W. Each dataset was analyzed 13 times with the sequence input order randomized each time. The tree represents the consensus of all trees produced. Values on the branches refer to the proportion of times a given branch was recorded among all trees. Values >90% indicate strong support, values 90–60% indicate moderate to weak support, and values <60% indicate no support for a given branching pattern. (C) Northern blotting of mRNAs from human tissues was performed with a probe generated using the exportin-5 cDNA as a template. The same blot was hybridized with a β-actin probe as a loading control. The exportin-5 sequence data are available from GenBank/EMBL/DDBJ under accession no. AF298880.Phylogenetic analysis revealed that exportin-5 is more closely related to Msn5p than to other members of the S. cerevisiae Kapβ family, although it is not clear that they are orthologous (Fig. 1 B). Overall, exportin-5 is ∼20% identical and ∼40% similar to Msn5p. However, it appears that exportin-5 also has sequence similarity to Los1p, Kap121p, Kap122p, Kap123p, and Xpo1p based on the inability of the analysis to produce a strongly supported branching pattern of these proteins with respect to exportin-5 (Fig. 1 B). A Northern blot was performed on mRNAs from multiple human tissues using an exportin-5–specific probe (Fig. 1 C). Two mRNAs of ∼3.6 and 5 kb were detected in all tissues examined. The length of the smaller mRNA corresponds to the size of the cDNA sequence. The length of the larger mRNA is consistent with the original KIAA 1291 clone that contained a 1.5 kb 3′ noncoding region before the polyadenylation signal. Subsequent to the identification and cloning of the gene, we characterized the exportin-5 protein.Exportin-5 binds the RanGTPase and nucleoporinsAll Kapβ family members identified to date bind directly to both RanGTP and nucleoporins. Initially, we performed a yeast dihybrid mating assay to determine whether exportin-5 could interact with Ran. Full-length exportin-5 interacts with Ran, but a construct lacking the first 108 amino acids of exportin-5 (ΔN exportin-5) is unable to interact productively with Ran (Fig. 2 A). These data are consistent with the observation that the NH2-terminal regions of Kapβs are required for RanGTP binding (Wozniak et al., 1998). In addition, the growth of exportin-5 on L−W−H− is more robust than that of importin-11, a known Ran binding protein (Fig. 2 A) (Plafker and Macara, 2000). Together, these data suggested that exportin-5 could interact with Ran.Figure 2.
Exportin-5 interacts with the RanGTPase and nucleoporins. (A) HF7c (MATa) yeast that express the indicated bait proteins as GAL4 DBD fusions were mated with the W303 (MATα) strain expressing the VP16 transactivation domain either alone (VP16) or as a fusion with Ran (VP16-Ran). Diploid yeast were selected on Leu−Trp− plates and replica plated onto Leu−Trp−His− ±3-aminotriazole (3AT) as indicated. (B) Exportin-5–His6 was incubated with either GST-RanGDP or GST-RanGTP bound to GSH beads. Bound proteins were separated by SDS-PAGE and detected by Coomassie staining. (C) The apparent dissociation constant of the RanGTP–exportin-5 complex was estimated using a RanGAP protection assay. Ran[γ-32P]GTP was preincubated for 15 min with the indicated concentrations of exportin-5–His6. The GTPase reaction was started by addition of RanGAP. After 3 min, the proteins were filter bound and washed. Hydrolysis of GTP was determined by the amount of Ran[γ-32P]GTP bound. Assays were in duplicate and are representative of two independent experiments. Error is expressed as the differences from the means. (D) HA–importin β and HA–exportin-5 were immunoprecipitated from HEK293 cell lysates to which RanQ69L was added as indicated. Untransfected HEK293 cells were used as a control for nonspecific nucleoporin binding (Mock IP). Lysate was prepared by solubilizing HEK293 cells with boiling sample buffer. Coimmunoprecipitated nucleoporins were detected with mAb RL1 (Snow et al., 1987). RL1 efficiently detected RanBP2, Can/Nup214, and Nup153 but not other nucleoporins in HEK293 lysate. HA–importin β and HA–exportin-5 were detected using mAb 12CA5. A protein not characterized previously as reacting with the RL1 mAb bound exportin-5 in the absence of RanQ69L and is denoted with an asterisk. (E) Heterotypic cell fusions were performed between BHK21 cells, transiently transfected with pKmyc–exportin-5, and a stably transfected HeLa cell line expressing GFP–streptavidin–NLS (GSN2). After fusion with polyethylene glycol, the cells were fixed and stained for myc-tagged proteins with 9E10 and a Texas red–conjugated secondary antibody. Nuclei were visualized using DAPI. The arrows denote the GSN2 cells in the fusion. Bar, 10 μm.To test for a direct interaction between exportin-5 and Ran, we expressed and purified glutathione S-transferase (GST)–tagged Ran and exportin-5 from Escherichia coli. Exportin-5 bound specifically to GST-RanGTP but was unable to bind GST-RanGDP (Fig. 2 B). To determine the affinity of recombinant exportin-5 for RanGTP, we used a RanGAP protection assay. This assay is based on the observation that complex formation between Kapβ family members and RanGTP inhibits hydrolysis by RanGAP. Therefore, we tested the ability of a fixed concentration of RanGAP to stimulate hydrolysis of Ran[γ-32P]GTP over a range of exportin-5 concentrations (Fig. 2 C). Exportin-5 could prevent hydrolysis of Ran-GTP completely with an apparent dissociation constant of ∼50 nM. This value is intermediate to that reported for importin β (0.5 nM) and Crm1 (>1,000 nM) using this same assay (Kutay et al., 2000). Together, these data demonstrate that exportin-5 binds directly and specifically to RanGTP.To transport cargo, Kaps must bind nuclear pore proteins and shuttle between the nucleus and cytoplasm. In addition, Ran may regulate the binding of transport receptors to nucleoporins. Thus, we wanted to verify exportin-5 binding to nucleoporins and test whether this binding is RanGTP sensitive. HA-tagged importin β and exportin-5 were immunoprecipitated from HEK293 cell lysates in the presence or absence of RanQ69L, a Ran mutant locked in the GTP-bound form. Both Can/Nup214 and Nup153 were coimmunoprecipitated with exportin-5 in a RanQ69L-independent manner (Fig. 2 D, exp-5 IP). As a positive control, we confirmed previous findings that importin β binds Nup153 only in the absence of RanGTP and binds RanBP2 both in the absence and presence of RanQ69L (Fig. 2 D, impβ IP) (Kehlenbach et al., 1999). As a negative control, we incubated untransfected cell lysate with 12CA5 and protein A–Sepharose beads. No nonspecific interaction of the beads with RanBP2, Can/Nup214, or Nup153 could be detected, indicating that the association of nucleoporins was mediated by the immunoprecipitated importin β and exportin-5 (Fig. 2 D, mock IP). These results show that exportin-5 can associate with nucleoporins. However, RanGTP-regulated binding may require additional cofactors or be mediated by nucleoporins not detected by RL1.Like most Kapβs, exportin-5 is predominantly nuclear at steady-state when expressed in mammalian cells. To determine whether exportin-5 shuttles between the nucleus and the cytoplasm, we used heterokaryon fusion assays. To construct heterokaryons, we fused BHK21 cells expressing exportin-5 with GSN2 cells. These latter cells stably express a green fluorescent protein (GFP)–streptavidin–NLS fusion protein that is constitutively nuclear and does not shuttle, thereby acting as a marker for these nuclei (Black et al., 1999). Shuttling of exportin-5 would lead to its equilibration into the nuclei of fused GSN2 cells. On the other hand, if exportin-5 is permanently nuclear it would not be detected in the nuclei of fused GSN2 cells. Fig. 2 E shows a heterokaryon with 5 nuclei: three exportin-5 nuclei and two GSN2 nuclei. Comparison of the left and center panels reveals that the GSN2 nuclei contained exportin-5, indicating that it had been exported from the BHK21 nuclei and imported into the GSN2 nuclei. Thus, although exportin-5 is predominantly nuclear at steady-state, it is continuously traversing the nuclear envelope. This property is consistent with its potential function as a transport receptor in mammalian cells.Exportin-5 binds ILF3 in a RanGTP-dependent mannerTo determine the function of this novel Kapβ, we needed to identify a cargo protein. To this end, we performed a yeast dihybrid screen of a murine 10-d embryo library, using full-length exportin-5 as bait. This screen identified six positive clones, three representing the nuclear pore-associated protein, Npap60 (Fan et al., 1997), and three representing the murine homologue of the human ILF (Fig. 3 A). All three ILF3 clones included amino acids 533–640, a region that contains the second dsRBD plus 38 additional COOH-terminal residues. Murine and human ILF3 are 98% identical within this region. Exportin-5 interacted with ILF3(533–640) but not with two other transport receptors, importin β and importin-11, in a dihybrid mating assay (Fig. 3 B). These results suggest a specific interaction between exportin-5 and ILF3.Figure 3.
Exportin-5 interacts with ILF3 in a RanGTP-dependent manner. (A) Schematic representation of human ILF3. Important elements found in this protein include the following: a putative bipartite NLS (striped box), two dsRBDs (shaded boxes), and an RGG RNA-binding motif (dotted box). The fragment that bound exportin-5 in the dihybrid screen is underlined. (B) Conjugation assays were performed as in the legend to Fig. 2 using VP16-ILF3(533–640) as the prey. Diploid yeast were replica plated onto selective medium (L−W−H−). (C) Equal amounts of GST or GST-ILF3(533–640) were immobilized on GSH beads and incubated with 200 nM recombinant exportin-5–His6 ±3 μM RanQ69L. An aliquot of the supernatant (1/10 the total) was analyzed to ensure the beads were incubated with equal concentrations of exportin-5–His6. Bound and unbound fractions were analyzed by SDS-PAGE and immunoblotting with anti-His6, anti-Ran, and anti-GST antibodies. All lanes were exposed equally to film. (D) HEK293 cells were transfected with full-length human pKH3-ILF3. Cells were lysed after 48 h. HA-tagged proteins were immunoprecipitated with 12CA5, immobilized on protein A–Sepharose, and then incubated with 200 nM exportin-5–His6 ±3 μM RanQ69L. HEK293 cells transfected with empty vector were used as a negative control. Bound proteins were analyzed by SDS-PAGE and immunoblotting with anti-His6 and anti-HA antibodies.The prevailing model for nucleocytoplasmic protein transport asserts that the binding of cargo to an import receptor is mutually exclusive to the binding of RanGTP (Gorlich and Kutay, 1999). Conversely, RanGTP stimulates binding of export receptors to their cargo. Thus, to obtain insight into whether exportin-5 behaves as an import or export receptor for ILF3 we tested whether RanQ69L promoted or disrupted binding of ILF3 to exportin-5. Recombinant exportin-5 bound directly to recombinant GST-ILF3(533–640) in the presence but not in the absence of RanQ69L (Fig. 3 C).To verify that full-length ILF3 binds exportin-5, ILF3 was obtained by PCR from human brain cDNA. Full-length ILF3 could not be expressed undegraded in E. coli (unpublished data). Therefore, ILF3 was expressed with an HA tag in mammalian cells, immunoprecipitated with 12CA5 mAb, and incubated with recombinant exportin-5 in the absence or presence of RanQ69L. Exportin-5 bound full-length ILF3 in the presence of RanQ69L but not 12CA5-bound protein A beads incubated with lysate from cells transfected with empty vector (Fig. 3 D). These results demonstrate that the interaction between exportin-5 and ILF3 is RanGTP dependent and suggest that exportin-5 may act as a nuclear export receptor for ILF3.Exportin-5 binds ILF3 specifically and in an LMB-insensitive mannerTo evaluate the specificity of the ILF3–exportin-5 interaction, we tested whether other Kapβs are able to bind directly to ILF3. Importantly, neither Crm1 nor importin β were able to bind GST-ILF3(533–640) in either the absence or presence of RanQ69L (Fig. 4 A). These results suggest that exportin-5 interacts specifically with ILF3.Figure 4.
Exportin-5 binds ILF3 specifically and in an LMB- insensitive manner. (A) Equal amounts of GST-ILF3(533–640) were immobilized on GSH beads and incubated with 200 nM exportin-5–His6, 500 nM Crm1-His6, or 200 nM importin β–His6 ±3 μM RanQ69L as indicated. Proteins were analyzed and detected as in the legend to Fig. 3. (B) Equal amounts of GST-PKI(36–50) or GST-ILF3(533–640) were immobilized on GSH beads. The beads were incubated with 200 nM Crm1-His6 or exportin-5–His6 ±3 μM RanQ69L. Where indicated, Crm1-His6 and exportin-5–His6 were incubated with 500 nM LMB at room temperature for 15 min before their addition to the binding assay. Proteins were analyzed and detected as described in the legend to Fig. 3. Exportin-5 is unable to bind GST-PKI(36–50) (unpublished data).The best characterized nuclear export pathway uses a leucine-rich NES, which is bound by the export receptor Crm1 in the presence of RanGTP (Askjaer et al., 1998). Binding of Crm1 to RanGTP and NES is inhibited by LMB (Kudo et al., 1999). Therefore, we tested whether LMB affected the ILF3–exportin-5 interaction. LMB had no detectable effect on the amount of exportin-5 bound to GST-ILF3(533–640) in the presence of RanQ69L (Fig. 4 B). In contrast, untreated Crm1 bound GST-PKI(36–50) in the presence of RanQ69L, but pretreatment with LMB abolished this interaction (Fig. 4 B). These data demonstrate that the exportin-5–ILF3 interaction is LMB insensitive.Exportin-5 binds dsRBD2 of ILF3 and the dsRBDs of other proteinsTo refine the region of ILF3 required for exportin-5 binding, we tested the ability of exportin-5 to bind ILF3 dsRBD2 (524–592). Exportin-5 bound dsRBD2 in a RanGTP-dependent manner (Fig. 5 A). Since the exportin-5 recognition motif appears to be contained within the ILF3 dsRBD, we cloned dsRBDs from other proteins to test binding to exportin-5. These included dsRBD2 from spermatid perinuclear binding protein, Spnr (92% identical/93% similar to ILF3 dsRBD2), dsRBD 3 from staufen (42%/63%), and dsRBD1 from protein kinase R (PKR) (30/58%). All bound exportin-5 in a RanGTP-dependent manner, although the binding of exportin-5 to dsRBD1 of PKR appeared less robust than binding to the other domains. These results suggest that the exportin-5 recognition sequence is conserved between different dsRBDs.Figure 5.
Exportin-5 binds the dsRBDs of multiple proteins, but binding is blocked by dsRNA. (A) Equal amounts of GST-ILF3 dsRBD2, GST-Spnr dsRBD2, GST-staufen dsRBD3, or GST-PKR dsRBD1 were immobilized on GSH beads. Beads were incubated with 200 nM exportin-5–His6 ±3 μM RanQ69L. Proteins were analyzed and detected as described in the legend to Fig. 3. (B) Amino acid sequences of ILF3 dsRBD2, Spnr dsRBD2, staufen dsRBD3, and PKR dsRBD1 were aligned using CLUSTAL. Residues conserved in all four dsRBDs are in black. Closely related residues found in at least three of the four sequences are in gray. Regions of secondary structure are denoted as boxes above the sequences. Asterisks denote conserved residues in the three regions that make essential contacts with the RNA. Residues essential for the correct packing of the α-helices against the β sheet are marked with diamonds. (C) GST-ILF3(404–592) and GST-ILF3(404–592)mut (1 μg each) were separated by SDS-PAGE and stained using Coomassie blue. GST-ILF3(404–592)mut is a construct that is deficient in dsRNA binding. A duplicate gel was transferred to nitrocellulose. The immobilized proteins were incubated with an α-32P–labeled dsRNA probe. After washing to remove excess probe, the filter was exposed to film for ∼18 h before developing. (D) GST-ILF3(404–592) was immobilized on GSH-agarose beads and either ssRNA or dsRNA was added. After RNA binding, the beads were incubated with 100 nM exportin-5 ±3 μM RanQ69L. Proteins were analyzed and detected as described in the legend to Fig. 3.Structural analysis of multiple dsRBDs has shown an α1-β1-β2-β3-α2 topology in which the NH2- and COOH-terminal α helices pack against one face of the three-stranded antiparallel β sheet (Ryter and Schultz, 1998). Several residues make essential contacts with the RNA (Fig. 5 B, asterisks). In addition, other residues are essential for correct packing of the α-helices against the β sheet (Fig. 5 B, ♦). Mutation of any of the above residues results in the loss of RNA binding. Importantly, deletion of either α1 or α2 from ILF3 results in the loss of exportin-5 binding (unpublished data), strongly suggesting that the exportin-5 recognition motif is the dsRBD itself.To determine the requirements for dsRNA binding, we assayed the ability of several ILF3 dsRBD constructs to interact with a 32P-labeled dsRNA probe. We were unable to detect dsRNA binding to the isolated dsRBD2 fragment (unpublished data). Therefore, we made a new construct, GST-ILF3(404–592) that encompasses both dsRBD1 and dsRBD2. We also made a mutant, GST-ILF3(404–592) mut, in which F432 and F559 are mutated to alanine. Because these residues pack the α-helices against the β sheet, it is believed that these mutations cause a loss of dsRBD tertiary structure (Ryter and Schultz, 1998). As such, GST-ILF3 (404–592) but not the mutant construct bound the 32P-labeled dsRNA probe (Fig. 5 C).Based on our data that the dsRBD is the limiting domain for exportin-5 binding, dsRNA and exportin-5 binding to ILF3 may involve overlapping residues. Therefore, bound RNA could either promote or prevent the interaction of ILF3 with exportin-5. We used GST-ILF3(404–592) to test whether dsRNA affected the binding of exportin-5. DsRNA but not ssRNA inhibited exportin-5 binding (Fig. 5 D). These data demonstrate that ILF3 cannot bind simultaneously to dsRNA and exportin-5 and suggest that exportin-5 cannot export RNA-bound ILF3.Nuclear export of ILF3 is dependent on exportin-5 in intact cellsTo verify the presence of exportin-5 mRNA in common cell lines, we performed reverse transcriptase (RT)-PCR from cytoplasmic RNAs isolated from BHK21 and HeLa cells. Exportin-5 RNA was readily detectable in HeLa cells (Fig. 6 A). In contrast, no exportin-5 RNA could be detected in BHK21 cells, although the control transcript, glyceraldehyde-3-phosphate dehydrogenase, could be amplified from the same RNA (Fig. 6 A). These data suggest that the exportin-5 gene product is expressed in HeLa cells but not in BHK21 cells. Therefore, BHK21 cells were used as an exportin-5 “null” cell line to study the nuclear export of ILF3 in intact cells.Figure 6.
Nuclear export of ILF3 is dependent on exportin-5 in intact cells. (A) Results of coupled reverse transcription and PCRs on BHK21 (B) and HeLa (H) cell RNA. All reactions contained 1 μg of RNA and either exportin-5 or glyceraldehyde-3-phosphate dehydrogenase gene-specific primers. A fraction of each reaction (1/20) was visualized on an ethidium bromide-stained agarose gel. GGIBF3(533–640) (18 μM) was injected into the nuclei of (B) HeLa cells or (C) BHK21 cells using fluorescent dextran (TRITC-dextran, 1 mg/ml) as an injection marker. Where indicated, GGIBF3 was coinjected with either 2 μM exportin-5–His6 or 2 μM Crm1-His6. All injected cells were incubated 30 min before fixation and visualization of the injected import substrate. Bars, 10 μm.Amino acids 533–640 of ILF3 were expressed as a COOH-terminal fusion protein to GST-GFP in E. coli (GGILF [533–640]). This protein (∼72 kD, monomer) forms dimers that are well above the diffusion limit of the nuclear pore. When injected into HeLa nuclei, GGILF3 (533–640) was exported rapidly to the cytoplasm (Fig. 6 B). In contrast, injection of GGILF3(533–640) into nuclei of BHK21 cells resulted in retention of the protein in the nucleus (Fig. 6 C). However, coinjection of exportin-5 into BHK21 nuclei restored rapid nuclear export of GGILF3 (533–640). Export was not observed when Crm1 was coinjected into nuclei with GGILF3(533–640) instead of exportin-5 (Fig. 6 C), strongly suggesting that the signal sequence contained in ILF3 is specific for exportin-5. These data demonstrate that exportin-5 functions as a nuclear export receptor for ILF3 and that no other protein within the BHK21 cell can mediate ILF3 export.Exportin-5 mediates export of ILF3 in digitonin-permeabilized cellsTo test directly whether exportin-5 mediates nuclear export of ILF3, we used a digitonin permeabilized cell export assay. ILF3 was expressed in HeLa cells as a GFP fusion protein together with a red fluorescent protein (RFP) fusion of the Ran exchange factor, RCC1. RCC1 is a constitutively nuclear protein and, therefore, served as a marker for transfected cells. On addition of exportin-5 to permeabilized cells, GFP-ILF3 was exported rapidly from the nucleus, whereas no export was observed upon addition of Crm1 or buffer alone (Fig. 7 A). Export of GFP-ILF3 by exportin-5 was inhibited completely by the addition of WGA, a protein that binds tightly to nuclear pores and inhibits receptor-mediated nucleocytoplasmic transport pathways. LMB had no effect on the export activity of exportin-5 (Fig. 7 A). Additionally, exportin-5 was unable to promote nuclear export of PKI, a protein whose export is facilitated by both Crm1 and calreticulin (Jim Holaska, personal communication). Quantitation of the mean nuclear fluorescence demonstrated that the addition of exportin-5 reduced the nuclear fluorescence of GFP-ILF3 by ∼70% over buffer alone (Fig. 7 B). Together these data demonstrate that exportin-5 is a nuclear export receptor for ILF3.Figure 7.
Exportin-5 mediates export of ILF3 in digitonin- permeabilized cells. (A) HeLa cells were cotransfected with pEGFP-ILF3 and pKRed-RCC1. pKRed-RCC1 was used as a nuclear marker for transfected cells. The cells were permeabilized with digitonin, and export assays were performed in buffer alone or with buffer plus exportin-5–His6 (100 μg/ml), exportin-5–His6 (100 μg/ml) pretreated with LMB (1 μM), exportin-5–His6 (100 μg/ml) plus WGA (200 μg/ml), or Crm1-His6 (100 μg/ml). (B) Mean nuclear fluorescence values for GFP-ILF3 were obtained using Openlab (Improvision). All values were corrected for background fluorescence levels and normalized to the mean nuclear fluorescence levels of permeabilized buffer-treated cells. Each data point represents the mean nuclear fluorescence obtained from 30 randomly chosen cells. Error is expressed as ±1 SD from the mean. Bar, 10 μm.

Molecular cloning of exportin-5Although five mammalian Kapβs have been characterized as nuclear export receptors, four mediate the export of extremely specific cargo with only one cargo identified for each receptor. Crm1 is the only Kapβ that has been shown to function as a general export receptor. To identify additional export receptors, we searched databases with the Crm1 sequence and identified a human clone, KIAA1291, that contained an ORF of 3,291 bp but lacked an initiation codon. To obtain the complete ORF, the sequence information from this clone was used to generate primers for 5′ rapid amplification of cDNA ends (RACE). An ∼1,000 bp product was amplified from human brain cDNA, which corresponded to the expected 5′ end of the gene. The complete ORF was found to be 3,615 bp, encoding a 1,205 residue protein (∼136 kD) that is 25% identical and 49% similar to human Crm1 (Fig. 1 A). The Met designated as the initiation codon is in a good Kozak sequence context and matches the expected NH2 terminus by comparison with a murine EST sequence (sequence data available from GenBank/EMBL/DDBJ under accession no. BAB26904). This gene product was named exportin-5 because it exports protein cargo from the nucleus (see below) and was the fifth mammalian Kapβ family member initially characterized as an export receptor.Figure 1.
Exportin-5 is a Kapβ family member. (A) Amino acid sequences of exportin-5, Crm1, and the S. cerevisiae Kapβ, Msn5p, were aligned using CLUSTAL. Identical residues are shown in black; closely related residues are shown in gray. (B) Unrooted phylogenetic tree of exportin-5 and the 14 S. cerevisiae Kapβs generated using the ProtPars program in the Phylip package. A total of 1,000 boot-strapped replica datasets were run from an alignment of the S. cerevisiae Kap and exportin-5 sequences that was generated using Clustal W. Each dataset was analyzed 13 times with the sequence input order randomized each time. The tree represents the consensus of all trees produced. Values on the branches refer to the proportion of times a given branch was recorded among all trees. Values >90% indicate strong support, values 90–60% indicate moderate to weak support, and values <60% indicate no support for a given branching pattern. (C) Northern blotting of mRNAs from human tissues was performed with a probe generated using the exportin-5 cDNA as a template. The same blot was hybridized with a β-actin probe as a loading control. The exportin-5 sequence data are available from GenBank/EMBL/DDBJ under accession no. AF298880.Phylogenetic analysis revealed that exportin-5 is more closely related to Msn5p than to other members of the S. cerevisiae Kapβ family, although it is not clear that they are orthologous (Fig. 1 B). Overall, exportin-5 is ∼20% identical and ∼40% similar to Msn5p. However, it appears that exportin-5 also has sequence similarity to Los1p, Kap121p, Kap122p, Kap123p, and Xpo1p based on the inability of the analysis to produce a strongly supported branching pattern of these proteins with respect to exportin-5 (Fig. 1 B). A Northern blot was performed on mRNAs from multiple human tissues using an exportin-5–specific probe (Fig. 1 C). Two mRNAs of ∼3.6 and 5 kb were detected in all tissues examined. The length of the smaller mRNA corresponds to the size of the cDNA sequence. The length of the larger mRNA is consistent with the original KIAA 1291 clone that contained a 1.5 kb 3′ noncoding region before the polyadenylation signal. Subsequent to the identification and cloning of the gene, we characterized the exportin-5 protein.

Molecular cloning of exportin-5

Although five mammalian Kapβs have been characterized as nuclear export receptors, four mediate the export of extremely specific cargo with only one cargo identified for each receptor. Crm1 is the only Kapβ that has been shown to function as a general export receptor. To identify additional export receptors, we searched databases with the Crm1 sequence and identified a human clone, KIAA1291, that contained an ORF of 3,291 bp but lacked an initiation codon. To obtain the complete ORF, the sequence information from this clone was used to generate primers for 5′ rapid amplification of cDNA ends (RACE). An ∼1,000 bp product was amplified from human brain cDNA, which corresponded to the expected 5′ end of the gene. The complete ORF was found to be 3,615 bp, encoding a 1,205 residue protein (∼136 kD) that is 25% identical and 49% similar to human Crm1 (Fig. 1 A). The Met designated as the initiation codon is in a good Kozak sequence context and matches the expected NH2 terminus by comparison with a murine EST sequence (sequence data available from GenBank/EMBL/DDBJ under accession no. BAB26904). This gene product was named exportin-5 because it exports protein cargo from the nucleus (see below) and was the fifth mammalian Kapβ family member initially characterized as an export receptor.

Exportin-5 is a Kapβ family member. (A) Amino acid sequences of exportin-5, Crm1, and the S. cerevisiae Kapβ, Msn5p, were aligned using CLUSTAL. Identical residues are shown in black; closely related residues are shown in gray. (B) Unrooted phylogenetic tree of exportin-5 and the 14 S. cerevisiae Kapβs generated using the ProtPars program in the Phylip package. A total of 1,000 boot-strapped replica datasets were run from an alignment of the S. cerevisiae Kap and exportin-5 sequences that was generated using Clustal W. Each dataset was analyzed 13 times with the sequence input order randomized each time. The tree represents the consensus of all trees produced. Values on the branches refer to the proportion of times a given branch was recorded among all trees. Values >90% indicate strong support, values 90–60% indicate moderate to weak support, and values <60% indicate no support for a given branching pattern. (C) Northern blotting of mRNAs from human tissues was performed with a probe generated using the exportin-5 cDNA as a template. The same blot was hybridized with a β-actin probe as a loading control. The exportin-5 sequence data are available from GenBank/EMBL/DDBJ under accession no. AF298880.

Phylogenetic analysis revealed that exportin-5 is more closely related to Msn5p than to other members of the S. cerevisiae Kapβ family, although it is not clear that they are orthologous (Fig. 1 B). Overall, exportin-5 is ∼20% identical and ∼40% similar to Msn5p. However, it appears that exportin-5 also has sequence similarity to Los1p, Kap121p, Kap122p, Kap123p, and Xpo1p based on the inability of the analysis to produce a strongly supported branching pattern of these proteins with respect to exportin-5 (Fig. 1 B). A Northern blot was performed on mRNAs from multiple human tissues using an exportin-5–specific probe (Fig. 1 C). Two mRNAs of ∼3.6 and 5 kb were detected in all tissues examined. The length of the smaller mRNA corresponds to the size of the cDNA sequence. The length of the larger mRNA is consistent with the original KIAA 1291 clone that contained a 1.5 kb 3′ noncoding region before the polyadenylation signal. Subsequent to the identification and cloning of the gene, we characterized the exportin-5 protein.

Exportin-5 binds the RanGTPase and nucleoporinsAll Kapβ family members identified to date bind directly to both RanGTP and nucleoporins. Initially, we performed a yeast dihybrid mating assay to determine whether exportin-5 could interact with Ran. Full-length exportin-5 interacts with Ran, but a construct lacking the first 108 amino acids of exportin-5 (ΔN exportin-5) is unable to interact productively with Ran (Fig. 2 A). These data are consistent with the observation that the NH2-terminal regions of Kapβs are required for RanGTP binding (Wozniak et al., 1998). In addition, the growth of exportin-5 on L−W−H− is more robust than that of importin-11, a known Ran binding protein (Fig. 2 A) (Plafker and Macara, 2000). Together, these data suggested that exportin-5 could interact with Ran.Figure 2.
Exportin-5 interacts with the RanGTPase and nucleoporins. (A) HF7c (MATa) yeast that express the indicated bait proteins as GAL4 DBD fusions were mated with the W303 (MATα) strain expressing the VP16 transactivation domain either alone (VP16) or as a fusion with Ran (VP16-Ran). Diploid yeast were selected on Leu−Trp− plates and replica plated onto Leu−Trp−His− ±3-aminotriazole (3AT) as indicated. (B) Exportin-5–His6 was incubated with either GST-RanGDP or GST-RanGTP bound to GSH beads. Bound proteins were separated by SDS-PAGE and detected by Coomassie staining. (C) The apparent dissociation constant of the RanGTP–exportin-5 complex was estimated using a RanGAP protection assay. Ran[γ-32P]GTP was preincubated for 15 min with the indicated concentrations of exportin-5–His6. The GTPase reaction was started by addition of RanGAP. After 3 min, the proteins were filter bound and washed. Hydrolysis of GTP was determined by the amount of Ran[γ-32P]GTP bound. Assays were in duplicate and are representative of two independent experiments. Error is expressed as the differences from the means. (D) HA–importin β and HA–exportin-5 were immunoprecipitated from HEK293 cell lysates to which RanQ69L was added as indicated. Untransfected HEK293 cells were used as a control for nonspecific nucleoporin binding (Mock IP). Lysate was prepared by solubilizing HEK293 cells with boiling sample buffer. Coimmunoprecipitated nucleoporins were detected with mAb RL1 (Snow et al., 1987). RL1 efficiently detected RanBP2, Can/Nup214, and Nup153 but not other nucleoporins in HEK293 lysate. HA–importin β and HA–exportin-5 were detected using mAb 12CA5. A protein not characterized previously as reacting with the RL1 mAb bound exportin-5 in the absence of RanQ69L and is denoted with an asterisk. (E) Heterotypic cell fusions were performed between BHK21 cells, transiently transfected with pKmyc–exportin-5, and a stably transfected HeLa cell line expressing GFP–streptavidin–NLS (GSN2). After fusion with polyethylene glycol, the cells were fixed and stained for myc-tagged proteins with 9E10 and a Texas red–conjugated secondary antibody. Nuclei were visualized using DAPI. The arrows denote the GSN2 cells in the fusion. Bar, 10 μm.To test for a direct interaction between exportin-5 and Ran, we expressed and purified glutathione S-transferase (GST)–tagged Ran and exportin-5 from Escherichia coli. Exportin-5 bound specifically to GST-RanGTP but was unable to bind GST-RanGDP (Fig. 2 B). To determine the affinity of recombinant exportin-5 for RanGTP, we used a RanGAP protection assay. This assay is based on the observation that complex formation between Kapβ family members and RanGTP inhibits hydrolysis by RanGAP. Therefore, we tested the ability of a fixed concentration of RanGAP to stimulate hydrolysis of Ran[γ-32P]GTP over a range of exportin-5 concentrations (Fig. 2 C). Exportin-5 could prevent hydrolysis of Ran-GTP completely with an apparent dissociation constant of ∼50 nM. This value is intermediate to that reported for importin β (0.5 nM) and Crm1 (>1,000 nM) using this same assay (Kutay et al., 2000). Together, these data demonstrate that exportin-5 binds directly and specifically to RanGTP.To transport cargo, Kaps must bind nuclear pore proteins and shuttle between the nucleus and cytoplasm. In addition, Ran may regulate the binding of transport receptors to nucleoporins. Thus, we wanted to verify exportin-5 binding to nucleoporins and test whether this binding is RanGTP sensitive. HA-tagged importin β and exportin-5 were immunoprecipitated from HEK293 cell lysates in the presence or absence of RanQ69L, a Ran mutant locked in the GTP-bound form. Both Can/Nup214 and Nup153 were coimmunoprecipitated with exportin-5 in a RanQ69L-independent manner (Fig. 2 D, exp-5 IP). As a positive control, we confirmed previous findings that importin β binds Nup153 only in the absence of RanGTP and binds RanBP2 both in the absence and presence of RanQ69L (Fig. 2 D, impβ IP) (Kehlenbach et al., 1999). As a negative control, we incubated untransfected cell lysate with 12CA5 and protein A–Sepharose beads. No nonspecific interaction of the beads with RanBP2, Can/Nup214, or Nup153 could be detected, indicating that the association of nucleoporins was mediated by the immunoprecipitated importin β and exportin-5 (Fig. 2 D, mock IP). These results show that exportin-5 can associate with nucleoporins. However, RanGTP-regulated binding may require additional cofactors or be mediated by nucleoporins not detected by RL1.Like most Kapβs, exportin-5 is predominantly nuclear at steady-state when expressed in mammalian cells. To determine whether exportin-5 shuttles between the nucleus and the cytoplasm, we used heterokaryon fusion assays. To construct heterokaryons, we fused BHK21 cells expressing exportin-5 with GSN2 cells. These latter cells stably express a green fluorescent protein (GFP)–streptavidin–NLS fusion protein that is constitutively nuclear and does not shuttle, thereby acting as a marker for these nuclei (Black et al., 1999). Shuttling of exportin-5 would lead to its equilibration into the nuclei of fused GSN2 cells. On the other hand, if exportin-5 is permanently nuclear it would not be detected in the nuclei of fused GSN2 cells. Fig. 2 E shows a heterokaryon with 5 nuclei: three exportin-5 nuclei and two GSN2 nuclei. Comparison of the left and center panels reveals that the GSN2 nuclei contained exportin-5, indicating that it had been exported from the BHK21 nuclei and imported into the GSN2 nuclei. Thus, although exportin-5 is predominantly nuclear at steady-state, it is continuously traversing the nuclear envelope. This property is consistent with its potential function as a transport receptor in mammalian cells.

Exportin-5 binds the RanGTPase and nucleoporins

All Kapβ family members identified to date bind directly to both RanGTP and nucleoporins. Initially, we performed a yeast dihybrid mating assay to determine whether exportin-5 could interact with Ran. Full-length exportin-5 interacts with Ran, but a construct lacking the first 108 amino acids of exportin-5 (ΔN exportin-5) is unable to interact productively with Ran (Fig. 2 A). These data are consistent with the observation that the NH2-terminal regions of Kapβs are required for RanGTP binding (Wozniak et al., 1998). In addition, the growth of exportin-5 on L−W−H− is more robust than that of importin-11, a known Ran binding protein (Fig. 2 A) (Plafker and Macara, 2000). Together, these data suggested that exportin-5 could interact with Ran.

Exportin-5 interacts with the RanGTPase and nucleoporins. (A) HF7c (MATa) yeast that express the indicated bait proteins as GAL4 DBD fusions were mated with the W303 (MATα) strain expressing the VP16 transactivation domain either alone (VP16) or as a fusion with Ran (VP16-Ran). Diploid yeast were selected on Leu−Trp− plates and replica plated onto Leu−Trp−His− ±3-aminotriazole (3AT) as indicated. (B) Exportin-5–His6 was incubated with either GST-RanGDP or GST-RanGTP bound to GSH beads. Bound proteins were separated by SDS-PAGE and detected by Coomassie staining. (C) The apparent dissociation constant of the RanGTP–exportin-5 complex was estimated using a RanGAP protection assay. Ran[γ-32P]GTP was preincubated for 15 min with the indicated concentrations of exportin-5–His6. The GTPase reaction was started by addition of RanGAP. After 3 min, the proteins were filter bound and washed. Hydrolysis of GTP was determined by the amount of Ran[γ-32P]GTP bound. Assays were in duplicate and are representative of two independent experiments. Error is expressed as the differences from the means. (D) HA–importin β and HA–exportin-5 were immunoprecipitated from HEK293 cell lysates to which RanQ69L was added as indicated. Untransfected HEK293 cells were used as a control for nonspecific nucleoporin binding (Mock IP). Lysate was prepared by solubilizing HEK293 cells with boiling sample buffer. Coimmunoprecipitated nucleoporins were detected with mAb RL1 (Snow et al., 1987). RL1 efficiently detected RanBP2, Can/Nup214, and Nup153 but not other nucleoporins in HEK293 lysate. HA–importin β and HA–exportin-5 were detected using mAb 12CA5. A protein not characterized previously as reacting with the RL1 mAb bound exportin-5 in the absence of RanQ69L and is denoted with an asterisk. (E) Heterotypic cell fusions were performed between BHK21 cells, transiently transfected with pKmyc–exportin-5, and a stably transfected HeLa cell line expressing GFP–streptavidin–NLS (GSN2). After fusion with polyethylene glycol, the cells were fixed and stained for myc-tagged proteins with 9E10 and a Texas red–conjugated secondary antibody. Nuclei were visualized using DAPI. The arrows denote the GSN2 cells in the fusion. Bar, 10 μm.

To test for a direct interaction between exportin-5 and Ran, we expressed and purified glutathione S-transferase (GST)–tagged Ran and exportin-5 from Escherichia coli. Exportin-5 bound specifically to GST-RanGTP but was unable to bind GST-RanGDP (Fig. 2 B). To determine the affinity of recombinant exportin-5 for RanGTP, we used a RanGAP protection assay. This assay is based on the observation that complex formation between Kapβ family members and RanGTP inhibits hydrolysis by RanGAP. Therefore, we tested the ability of a fixed concentration of RanGAP to stimulate hydrolysis of Ran[γ-32P]GTP over a range of exportin-5 concentrations (Fig. 2 C). Exportin-5 could prevent hydrolysis of Ran-GTP completely with an apparent dissociation constant of ∼50 nM. This value is intermediate to that reported for importin β (0.5 nM) and Crm1 (>1,000 nM) using this same assay (Kutay et al., 2000). Together, these data demonstrate that exportin-5 binds directly and specifically to RanGTP.

To transport cargo, Kaps must bind nuclear pore proteins and shuttle between the nucleus and cytoplasm. In addition, Ran may regulate the binding of transport receptors to nucleoporins. Thus, we wanted to verify exportin-5 binding to nucleoporins and test whether this binding is RanGTP sensitive. HA-tagged importin β and exportin-5 were immunoprecipitated from HEK293 cell lysates in the presence or absence of RanQ69L, a Ran mutant locked in the GTP-bound form. Both Can/Nup214 and Nup153 were coimmunoprecipitated with exportin-5 in a RanQ69L-independent manner (Fig. 2 D, exp-5 IP). As a positive control, we confirmed previous findings that importin β binds Nup153 only in the absence of RanGTP and binds RanBP2 both in the absence and presence of RanQ69L (Fig. 2 D, impβ IP) (Kehlenbach et al., 1999). As a negative control, we incubated untransfected cell lysate with 12CA5 and protein A–Sepharose beads. No nonspecific interaction of the beads with RanBP2, Can/Nup214, or Nup153 could be detected, indicating that the association of nucleoporins was mediated by the immunoprecipitated importin β and exportin-5 (Fig. 2 D, mock IP). These results show that exportin-5 can associate with nucleoporins. However, RanGTP-regulated binding may require additional cofactors or be mediated by nucleoporins not detected by RL1.

Like most Kapβs, exportin-5 is predominantly nuclear at steady-state when expressed in mammalian cells. To determine whether exportin-5 shuttles between the nucleus and the cytoplasm, we used heterokaryon fusion assays. To construct heterokaryons, we fused BHK21 cells expressing exportin-5 with GSN2 cells. These latter cells stably express a green fluorescent protein (GFP)–streptavidin–NLS fusion protein that is constitutively nuclear and does not shuttle, thereby acting as a marker for these nuclei (Black et al., 1999). Shuttling of exportin-5 would lead to its equilibration into the nuclei of fused GSN2 cells. On the other hand, if exportin-5 is permanently nuclear it would not be detected in the nuclei of fused GSN2 cells. Fig. 2 E shows a heterokaryon with 5 nuclei: three exportin-5 nuclei and two GSN2 nuclei. Comparison of the left and center panels reveals that the GSN2 nuclei contained exportin-5, indicating that it had been exported from the BHK21 nuclei and imported into the GSN2 nuclei. Thus, although exportin-5 is predominantly nuclear at steady-state, it is continuously traversing the nuclear envelope. This property is consistent with its potential function as a transport receptor in mammalian cells.

Exportin-5 binds ILF3 in a RanGTP-dependent mannerTo determine the function of this novel Kapβ, we needed to identify a cargo protein. To this end, we performed a yeast dihybrid screen of a murine 10-d embryo library, using full-length exportin-5 as bait. This screen identified six positive clones, three representing the nuclear pore-associated protein, Npap60 (Fan et al., 1997), and three representing the murine homologue of the human ILF (Fig. 3 A). All three ILF3 clones included amino acids 533–640, a region that contains the second dsRBD plus 38 additional COOH-terminal residues. Murine and human ILF3 are 98% identical within this region. Exportin-5 interacted with ILF3(533–640) but not with two other transport receptors, importin β and importin-11, in a dihybrid mating assay (Fig. 3 B). These results suggest a specific interaction between exportin-5 and ILF3.Figure 3.
Exportin-5 interacts with ILF3 in a RanGTP-dependent manner. (A) Schematic representation of human ILF3. Important elements found in this protein include the following: a putative bipartite NLS (striped box), two dsRBDs (shaded boxes), and an RGG RNA-binding motif (dotted box). The fragment that bound exportin-5 in the dihybrid screen is underlined. (B) Conjugation assays were performed as in the legend to Fig. 2 using VP16-ILF3(533–640) as the prey. Diploid yeast were replica plated onto selective medium (L−W−H−). (C) Equal amounts of GST or GST-ILF3(533–640) were immobilized on GSH beads and incubated with 200 nM recombinant exportin-5–His6 ±3 μM RanQ69L. An aliquot of the supernatant (1/10 the total) was analyzed to ensure the beads were incubated with equal concentrations of exportin-5–His6. Bound and unbound fractions were analyzed by SDS-PAGE and immunoblotting with anti-His6, anti-Ran, and anti-GST antibodies. All lanes were exposed equally to film. (D) HEK293 cells were transfected with full-length human pKH3-ILF3. Cells were lysed after 48 h. HA-tagged proteins were immunoprecipitated with 12CA5, immobilized on protein A–Sepharose, and then incubated with 200 nM exportin-5–His6 ±3 μM RanQ69L. HEK293 cells transfected with empty vector were used as a negative control. Bound proteins were analyzed by SDS-PAGE and immunoblotting with anti-His6 and anti-HA antibodies.The prevailing model for nucleocytoplasmic protein transport asserts that the binding of cargo to an import receptor is mutually exclusive to the binding of RanGTP (Gorlich and Kutay, 1999). Conversely, RanGTP stimulates binding of export receptors to their cargo. Thus, to obtain insight into whether exportin-5 behaves as an import or export receptor for ILF3 we tested whether RanQ69L promoted or disrupted binding of ILF3 to exportin-5. Recombinant exportin-5 bound directly to recombinant GST-ILF3(533–640) in the presence but not in the absence of RanQ69L (Fig. 3 C).To verify that full-length ILF3 binds exportin-5, ILF3 was obtained by PCR from human brain cDNA. Full-length ILF3 could not be expressed undegraded in E. coli (unpublished data). Therefore, ILF3 was expressed with an HA tag in mammalian cells, immunoprecipitated with 12CA5 mAb, and incubated with recombinant exportin-5 in the absence or presence of RanQ69L. Exportin-5 bound full-length ILF3 in the presence of RanQ69L but not 12CA5-bound protein A beads incubated with lysate from cells transfected with empty vector (Fig. 3 D). These results demonstrate that the interaction between exportin-5 and ILF3 is RanGTP dependent and suggest that exportin-5 may act as a nuclear export receptor for ILF3.

Exportin-5 binds ILF3 in a RanGTP-dependent manner

To determine the function of this novel Kapβ, we needed to identify a cargo protein. To this end, we performed a yeast dihybrid screen of a murine 10-d embryo library, using full-length exportin-5 as bait. This screen identified six positive clones, three representing the nuclear pore-associated protein, Npap60 (Fan et al., 1997), and three representing the murine homologue of the human ILF (Fig. 3 A). All three ILF3 clones included amino acids 533–640, a region that contains the second dsRBD plus 38 additional COOH-terminal residues. Murine and human ILF3 are 98% identical within this region. Exportin-5 interacted with ILF3(533–640) but not with two other transport receptors, importin β and importin-11, in a dihybrid mating assay (Fig. 3 B). These results suggest a specific interaction between exportin-5 and ILF3.

Exportin-5 interacts with ILF3 in a RanGTP-dependent manner. (A) Schematic representation of human ILF3. Important elements found in this protein include the following: a putative bipartite NLS (striped box), two dsRBDs (shaded boxes), and an RGG RNA-binding motif (dotted box). The fragment that bound exportin-5 in the dihybrid screen is underlined. (B) Conjugation assays were performed as in the legend to Fig. 2 using VP16-ILF3(533–640) as the prey. Diploid yeast were replica plated onto selective medium (L−W−H−). (C) Equal amounts of GST or GST-ILF3(533–640) were immobilized on GSH beads and incubated with 200 nM recombinant exportin-5–His6 ±3 μM RanQ69L. An aliquot of the supernatant (1/10 the total) was analyzed to ensure the beads were incubated with equal concentrations of exportin-5–His6. Bound and unbound fractions were analyzed by SDS-PAGE and immunoblotting with anti-His6, anti-Ran, and anti-GST antibodies. All lanes were exposed equally to film. (D) HEK293 cells were transfected with full-length human pKH3-ILF3. Cells were lysed after 48 h. HA-tagged proteins were immunoprecipitated with 12CA5, immobilized on protein A–Sepharose, and then incubated with 200 nM exportin-5–His6 ±3 μM RanQ69L. HEK293 cells transfected with empty vector were used as a negative control. Bound proteins were analyzed by SDS-PAGE and immunoblotting with anti-His6 and anti-HA antibodies.

The prevailing model for nucleocytoplasmic protein transport asserts that the binding of cargo to an import receptor is mutually exclusive to the binding of RanGTP (Gorlich and Kutay, 1999). Conversely, RanGTP stimulates binding of export receptors to their cargo. Thus, to obtain insight into whether exportin-5 behaves as an import or export receptor for ILF3 we tested whether RanQ69L promoted or disrupted binding of ILF3 to exportin-5. Recombinant exportin-5 bound directly to recombinant GST-ILF3(533–640) in the presence but not in the absence of RanQ69L (Fig. 3 C).

To verify that full-length ILF3 binds exportin-5, ILF3 was obtained by PCR from human brain cDNA. Full-length ILF3 could not be expressed undegraded in E. coli (unpublished data). Therefore, ILF3 was expressed with an HA tag in mammalian cells, immunoprecipitated with 12CA5 mAb, and incubated with recombinant exportin-5 in the absence or presence of RanQ69L. Exportin-5 bound full-length ILF3 in the presence of RanQ69L but not 12CA5-bound protein A beads incubated with lysate from cells transfected with empty vector (Fig. 3 D). These results demonstrate that the interaction between exportin-5 and ILF3 is RanGTP dependent and suggest that exportin-5 may act as a nuclear export receptor for ILF3.

Exportin-5 binds ILF3 specifically and in an LMB-insensitive mannerTo evaluate the specificity of the ILF3–exportin-5 interaction, we tested whether other Kapβs are able to bind directly to ILF3. Importantly, neither Crm1 nor importin β were able to bind GST-ILF3(533–640) in either the absence or presence of RanQ69L (Fig. 4 A). These results suggest that exportin-5 interacts specifically with ILF3.Figure 4.
Exportin-5 binds ILF3 specifically and in an LMB- insensitive manner. (A) Equal amounts of GST-ILF3(533–640) were immobilized on GSH beads and incubated with 200 nM exportin-5–His6, 500 nM Crm1-His6, or 200 nM importin β–His6 ±3 μM RanQ69L as indicated. Proteins were analyzed and detected as in the legend to Fig. 3. (B) Equal amounts of GST-PKI(36–50) or GST-ILF3(533–640) were immobilized on GSH beads. The beads were incubated with 200 nM Crm1-His6 or exportin-5–His6 ±3 μM RanQ69L. Where indicated, Crm1-His6 and exportin-5–His6 were incubated with 500 nM LMB at room temperature for 15 min before their addition to the binding assay. Proteins were analyzed and detected as described in the legend to Fig. 3. Exportin-5 is unable to bind GST-PKI(36–50) (unpublished data).The best characterized nuclear export pathway uses a leucine-rich NES, which is bound by the export receptor Crm1 in the presence of RanGTP (Askjaer et al., 1998). Binding of Crm1 to RanGTP and NES is inhibited by LMB (Kudo et al., 1999). Therefore, we tested whether LMB affected the ILF3–exportin-5 interaction. LMB had no detectable effect on the amount of exportin-5 bound to GST-ILF3(533–640) in the presence of RanQ69L (Fig. 4 B). In contrast, untreated Crm1 bound GST-PKI(36–50) in the presence of RanQ69L, but pretreatment with LMB abolished this interaction (Fig. 4 B). These data demonstrate that the exportin-5–ILF3 interaction is LMB insensitive.

Exportin-5 binds ILF3 specifically and in an LMB-insensitive manner

To evaluate the specificity of the ILF3–exportin-5 interaction, we tested whether other Kapβs are able to bind directly to ILF3. Importantly, neither Crm1 nor importin β were able to bind GST-ILF3(533–640) in either the absence or presence of RanQ69L (Fig. 4 A). These results suggest that exportin-5 interacts specifically with ILF3.

Exportin-5 binds ILF3 specifically and in an LMB- insensitive manner. (A) Equal amounts of GST-ILF3(533–640) were immobilized on GSH beads and incubated with 200 nM exportin-5–His6, 500 nM Crm1-His6, or 200 nM importin β–His6 ±3 μM RanQ69L as indicated. Proteins were analyzed and detected as in the legend to Fig. 3. (B) Equal amounts of GST-PKI(36–50) or GST-ILF3(533–640) were immobilized on GSH beads. The beads were incubated with 200 nM Crm1-His6 or exportin-5–His6 ±3 μM RanQ69L. Where indicated, Crm1-His6 and exportin-5–His6 were incubated with 500 nM LMB at room temperature for 15 min before their addition to the binding assay. Proteins were analyzed and detected as described in the legend to Fig. 3. Exportin-5 is unable to bind GST-PKI(36–50) (unpublished data).

The best characterized nuclear export pathway uses a leucine-rich NES, which is bound by the export receptor Crm1 in the presence of RanGTP (Askjaer et al., 1998). Binding of Crm1 to RanGTP and NES is inhibited by LMB (Kudo et al., 1999). Therefore, we tested whether LMB affected the ILF3–exportin-5 interaction. LMB had no detectable effect on the amount of exportin-5 bound to GST-ILF3(533–640) in the presence of RanQ69L (Fig. 4 B). In contrast, untreated Crm1 bound GST-PKI(36–50) in the presence of RanQ69L, but pretreatment with LMB abolished this interaction (Fig. 4 B). These data demonstrate that the exportin-5–ILF3 interaction is LMB insensitive.

Exportin-5 binds dsRBD2 of ILF3 and the dsRBDs of other proteinsTo refine the region of ILF3 required for exportin-5 binding, we tested the ability of exportin-5 to bind ILF3 dsRBD2 (524–592). Exportin-5 bound dsRBD2 in a RanGTP-dependent manner (Fig. 5 A). Since the exportin-5 recognition motif appears to be contained within the ILF3 dsRBD, we cloned dsRBDs from other proteins to test binding to exportin-5. These included dsRBD2 from spermatid perinuclear binding protein, Spnr (92% identical/93% similar to ILF3 dsRBD2), dsRBD 3 from staufen (42%/63%), and dsRBD1 from protein kinase R (PKR) (30/58%). All bound exportin-5 in a RanGTP-dependent manner, although the binding of exportin-5 to dsRBD1 of PKR appeared less robust than binding to the other domains. These results suggest that the exportin-5 recognition sequence is conserved between different dsRBDs.Figure 5.
Exportin-5 binds the dsRBDs of multiple proteins, but binding is blocked by dsRNA. (A) Equal amounts of GST-ILF3 dsRBD2, GST-Spnr dsRBD2, GST-staufen dsRBD3, or GST-PKR dsRBD1 were immobilized on GSH beads. Beads were incubated with 200 nM exportin-5–His6 ±3 μM RanQ69L. Proteins were analyzed and detected as described in the legend to Fig. 3. (B) Amino acid sequences of ILF3 dsRBD2, Spnr dsRBD2, staufen dsRBD3, and PKR dsRBD1 were aligned using CLUSTAL. Residues conserved in all four dsRBDs are in black. Closely related residues found in at least three of the four sequences are in gray. Regions of secondary structure are denoted as boxes above the sequences. Asterisks denote conserved residues in the three regions that make essential contacts with the RNA. Residues essential for the correct packing of the α-helices against the β sheet are marked with diamonds. (C) GST-ILF3(404–592) and GST-ILF3(404–592)mut (1 μg each) were separated by SDS-PAGE and stained using Coomassie blue. GST-ILF3(404–592)mut is a construct that is deficient in dsRNA binding. A duplicate gel was transferred to nitrocellulose. The immobilized proteins were incubated with an α-32P–labeled dsRNA probe. After washing to remove excess probe, the filter was exposed to film for ∼18 h before developing. (D) GST-ILF3(404–592) was immobilized on GSH-agarose beads and either ssRNA or dsRNA was added. After RNA binding, the beads were incubated with 100 nM exportin-5 ±3 μM RanQ69L. Proteins were analyzed and detected as described in the legend to Fig. 3.Structural analysis of multiple dsRBDs has shown an α1-β1-β2-β3-α2 topology in which the NH2- and COOH-terminal α helices pack against one face of the three-stranded antiparallel β sheet (Ryter and Schultz, 1998). Several residues make essential contacts with the RNA (Fig. 5 B, asterisks). In addition, other residues are essential for correct packing of the α-helices against the β sheet (Fig. 5 B, ♦). Mutation of any of the above residues results in the loss of RNA binding. Importantly, deletion of either α1 or α2 from ILF3 results in the loss of exportin-5 binding (unpublished data), strongly suggesting that the exportin-5 recognition motif is the dsRBD itself.To determine the requirements for dsRNA binding, we assayed the ability of several ILF3 dsRBD constructs to interact with a 32P-labeled dsRNA probe. We were unable to detect dsRNA binding to the isolated dsRBD2 fragment (unpublished data). Therefore, we made a new construct, GST-ILF3(404–592) that encompasses both dsRBD1 and dsRBD2. We also made a mutant, GST-ILF3(404–592) mut, in which F432 and F559 are mutated to alanine. Because these residues pack the α-helices against the β sheet, it is believed that these mutations cause a loss of dsRBD tertiary structure (Ryter and Schultz, 1998). As such, GST-ILF3 (404–592) but not the mutant construct bound the 32P-labeled dsRNA probe (Fig. 5 C).Based on our data that the dsRBD is the limiting domain for exportin-5 binding, dsRNA and exportin-5 binding to ILF3 may involve overlapping residues. Therefore, bound RNA could either promote or prevent the interaction of ILF3 with exportin-5. We used GST-ILF3(404–592) to test whether dsRNA affected the binding of exportin-5. DsRNA but not ssRNA inhibited exportin-5 binding (Fig. 5 D). These data demonstrate that ILF3 cannot bind simultaneously to dsRNA and exportin-5 and suggest that exportin-5 cannot export RNA-bound ILF3.

Exportin-5 binds dsRBD2 of ILF3 and the dsRBDs of other proteins

To refine the region of ILF3 required for exportin-5 binding, we tested the ability of exportin-5 to bind ILF3 dsRBD2 (524–592). Exportin-5 bound dsRBD2 in a RanGTP-dependent manner (Fig. 5 A). Since the exportin-5 recognition motif appears to be contained within the ILF3 dsRBD, we cloned dsRBDs from other proteins to test binding to exportin-5. These included dsRBD2 from spermatid perinuclear binding protein, Spnr (92% identical/93% similar to ILF3 dsRBD2), dsRBD 3 from staufen (42%/63%), and dsRBD1 from protein kinase R (PKR) (30/58%). All bound exportin-5 in a RanGTP-dependent manner, although the binding of exportin-5 to dsRBD1 of PKR appeared less robust than binding to the other domains. These results suggest that the exportin-5 recognition sequence is conserved between different dsRBDs.

Exportin-5 binds the dsRBDs of multiple proteins, but binding is blocked by dsRNA. (A) Equal amounts of GST-ILF3 dsRBD2, GST-Spnr dsRBD2, GST-staufen dsRBD3, or GST-PKR dsRBD1 were immobilized on GSH beads. Beads were incubated with 200 nM exportin-5–His6 ±3 μM RanQ69L. Proteins were analyzed and detected as described in the legend to Fig. 3. (B) Amino acid sequences of ILF3 dsRBD2, Spnr dsRBD2, staufen dsRBD3, and PKR dsRBD1 were aligned using CLUSTAL. Residues conserved in all four dsRBDs are in black. Closely related residues found in at least three of the four sequences are in gray. Regions of secondary structure are denoted as boxes above the sequences. Asterisks denote conserved residues in the three regions that make essential contacts with the RNA. Residues essential for the correct packing of the α-helices against the β sheet are marked with diamonds. (C) GST-ILF3(404–592) and GST-ILF3(404–592)mut (1 μg each) were separated by SDS-PAGE and stained using Coomassie blue. GST-ILF3(404–592)mut is a construct that is deficient in dsRNA binding. A duplicate gel was transferred to nitrocellulose. The immobilized proteins were incubated with an α-32P–labeled dsRNA probe. After washing to remove excess probe, the filter was exposed to film for ∼18 h before developing. (D) GST-ILF3(404–592) was immobilized on GSH-agarose beads and either ssRNA or dsRNA was added. After RNA binding, the beads were incubated with 100 nM exportin-5 ±3 μM RanQ69L. Proteins were analyzed and detected as described in the legend to Fig. 3.

Structural analysis of multiple dsRBDs has shown an α1-β1-β2-β3-α2 topology in which the NH2- and COOH-terminal α helices pack against one face of the three-stranded antiparallel β sheet (Ryter and Schultz, 1998). Several residues make essential contacts with the RNA (Fig. 5 B, asterisks). In addition, other residues are essential for correct packing of the α-helices against the β sheet (Fig. 5 B, ♦). Mutation of any of the above residues results in the loss of RNA binding. Importantly, deletion of either α1 or α2 from ILF3 results in the loss of exportin-5 binding (unpublished data), strongly suggesting that the exportin-5 recognition motif is the dsRBD itself.

To determine the requirements for dsRNA binding, we assayed the ability of several ILF3 dsRBD constructs to interact with a 32P-labeled dsRNA probe. We were unable to detect dsRNA binding to the isolated dsRBD2 fragment (unpublished data). Therefore, we made a new construct, GST-ILF3(404–592) that encompasses both dsRBD1 and dsRBD2. We also made a mutant, GST-ILF3(404–592) mut, in which F432 and F559 are mutated to alanine. Because these residues pack the α-helices against the β sheet, it is believed that these mutations cause a loss of dsRBD tertiary structure (Ryter and Schultz, 1998). As such, GST-ILF3 (404–592) but not the mutant construct bound the 32P-labeled dsRNA probe (Fig. 5 C).

Based on our data that the dsRBD is the limiting domain for exportin-5 binding, dsRNA and exportin-5 binding to ILF3 may involve overlapping residues. Therefore, bound RNA could either promote or prevent the interaction of ILF3 with exportin-5. We used GST-ILF3(404–592) to test whether dsRNA affected the binding of exportin-5. DsRNA but not ssRNA inhibited exportin-5 binding (Fig. 5 D). These data demonstrate that ILF3 cannot bind simultaneously to dsRNA and exportin-5 and suggest that exportin-5 cannot export RNA-bound ILF3.

Nuclear export of ILF3 is dependent on exportin-5 in intact cellsTo verify the presence of exportin-5 mRNA in common cell lines, we performed reverse transcriptase (RT)-PCR from cytoplasmic RNAs isolated from BHK21 and HeLa cells. Exportin-5 RNA was readily detectable in HeLa cells (Fig. 6 A). In contrast, no exportin-5 RNA could be detected in BHK21 cells, although the control transcript, glyceraldehyde-3-phosphate dehydrogenase, could be amplified from the same RNA (Fig. 6 A). These data suggest that the exportin-5 gene product is expressed in HeLa cells but not in BHK21 cells. Therefore, BHK21 cells were used as an exportin-5 “null” cell line to study the nuclear export of ILF3 in intact cells.Figure 6.
Nuclear export of ILF3 is dependent on exportin-5 in intact cells. (A) Results of coupled reverse transcription and PCRs on BHK21 (B) and HeLa (H) cell RNA. All reactions contained 1 μg of RNA and either exportin-5 or glyceraldehyde-3-phosphate dehydrogenase gene-specific primers. A fraction of each reaction (1/20) was visualized on an ethidium bromide-stained agarose gel. GGIBF3(533–640) (18 μM) was injected into the nuclei of (B) HeLa cells or (C) BHK21 cells using fluorescent dextran (TRITC-dextran, 1 mg/ml) as an injection marker. Where indicated, GGIBF3 was coinjected with either 2 μM exportin-5–His6 or 2 μM Crm1-His6. All injected cells were incubated 30 min before fixation and visualization of the injected import substrate. Bars, 10 μm.Amino acids 533–640 of ILF3 were expressed as a COOH-terminal fusion protein to GST-GFP in E. coli (GGILF [533–640]). This protein (∼72 kD, monomer) forms dimers that are well above the diffusion limit of the nuclear pore. When injected into HeLa nuclei, GGILF3 (533–640) was exported rapidly to the cytoplasm (Fig. 6 B). In contrast, injection of GGILF3(533–640) into nuclei of BHK21 cells resulted in retention of the protein in the nucleus (Fig. 6 C). However, coinjection of exportin-5 into BHK21 nuclei restored rapid nuclear export of GGILF3 (533–640). Export was not observed when Crm1 was coinjected into nuclei with GGILF3(533–640) instead of exportin-5 (Fig. 6 C), strongly suggesting that the signal sequence contained in ILF3 is specific for exportin-5. These data demonstrate that exportin-5 functions as a nuclear export receptor for ILF3 and that no other protein within the BHK21 cell can mediate ILF3 export.

Nuclear export of ILF3 is dependent on exportin-5 in intact cells

To verify the presence of exportin-5 mRNA in common cell lines, we performed reverse transcriptase (RT)-PCR from cytoplasmic RNAs isolated from BHK21 and HeLa cells. Exportin-5 RNA was readily detectable in HeLa cells (Fig. 6 A). In contrast, no exportin-5 RNA could be detected in BHK21 cells, although the control transcript, glyceraldehyde-3-phosphate dehydrogenase, could be amplified from the same RNA (Fig. 6 A). These data suggest that the exportin-5 gene product is expressed in HeLa cells but not in BHK21 cells. Therefore, BHK21 cells were used as an exportin-5 “null” cell line to study the nuclear export of ILF3 in intact cells.

Nuclear export of ILF3 is dependent on exportin-5 in intact cells. (A) Results of coupled reverse transcription and PCRs on BHK21 (B) and HeLa (H) cell RNA. All reactions contained 1 μg of RNA and either exportin-5 or glyceraldehyde-3-phosphate dehydrogenase gene-specific primers. A fraction of each reaction (1/20) was visualized on an ethidium bromide-stained agarose gel. GGIBF3(533–640) (18 μM) was injected into the nuclei of (B) HeLa cells or (C) BHK21 cells using fluorescent dextran (TRITC-dextran, 1 mg/ml) as an injection marker. Where indicated, GGIBF3 was coinjected with either 2 μM exportin-5–His6 or 2 μM Crm1-His6. All injected cells were incubated 30 min before fixation and visualization of the injected import substrate. Bars, 10 μm.

Amino acids 533–640 of ILF3 were expressed as a COOH-terminal fusion protein to GST-GFP in E. coli (GGILF [533–640]). This protein (∼72 kD, monomer) forms dimers that are well above the diffusion limit of the nuclear pore. When injected into HeLa nuclei, GGILF3 (533–640) was exported rapidly to the cytoplasm (Fig. 6 B). In contrast, injection of GGILF3(533–640) into nuclei of BHK21 cells resulted in retention of the protein in the nucleus (Fig. 6 C). However, coinjection of exportin-5 into BHK21 nuclei restored rapid nuclear export of GGILF3 (533–640). Export was not observed when Crm1 was coinjected into nuclei with GGILF3(533–640) instead of exportin-5 (Fig. 6 C), strongly suggesting that the signal sequence contained in ILF3 is specific for exportin-5. These data demonstrate that exportin-5 functions as a nuclear export receptor for ILF3 and that no other protein within the BHK21 cell can mediate ILF3 export.

Exportin-5 mediates export of ILF3 in digitonin-permeabilized cellsTo test directly whether exportin-5 mediates nuclear export of ILF3, we used a digitonin permeabilized cell export assay. ILF3 was expressed in HeLa cells as a GFP fusion protein together with a red fluorescent protein (RFP) fusion of the Ran exchange factor, RCC1. RCC1 is a constitutively nuclear protein and, therefore, served as a marker for transfected cells. On addition of exportin-5 to permeabilized cells, GFP-ILF3 was exported rapidly from the nucleus, whereas no export was observed upon addition of Crm1 or buffer alone (Fig. 7 A). Export of GFP-ILF3 by exportin-5 was inhibited completely by the addition of WGA, a protein that binds tightly to nuclear pores and inhibits receptor-mediated nucleocytoplasmic transport pathways. LMB had no effect on the export activity of exportin-5 (Fig. 7 A). Additionally, exportin-5 was unable to promote nuclear export of PKI, a protein whose export is facilitated by both Crm1 and calreticulin (Jim Holaska, personal communication). Quantitation of the mean nuclear fluorescence demonstrated that the addition of exportin-5 reduced the nuclear fluorescence of GFP-ILF3 by ∼70% over buffer alone (Fig. 7 B). Together these data demonstrate that exportin-5 is a nuclear export receptor for ILF3.Figure 7.
Exportin-5 mediates export of ILF3 in digitonin- permeabilized cells. (A) HeLa cells were cotransfected with pEGFP-ILF3 and pKRed-RCC1. pKRed-RCC1 was used as a nuclear marker for transfected cells. The cells were permeabilized with digitonin, and export assays were performed in buffer alone or with buffer plus exportin-5–His6 (100 μg/ml), exportin-5–His6 (100 μg/ml) pretreated with LMB (1 μM), exportin-5–His6 (100 μg/ml) plus WGA (200 μg/ml), or Crm1-His6 (100 μg/ml). (B) Mean nuclear fluorescence values for GFP-ILF3 were obtained using Openlab (Improvision). All values were corrected for background fluorescence levels and normalized to the mean nuclear fluorescence levels of permeabilized buffer-treated cells. Each data point represents the mean nuclear fluorescence obtained from 30 randomly chosen cells. Error is expressed as ±1 SD from the mean. Bar, 10 μm.

Exportin-5 mediates export of ILF3 in digitonin-permeabilized cells

To test directly whether exportin-5 mediates nuclear export of ILF3, we used a digitonin permeabilized cell export assay. ILF3 was expressed in HeLa cells as a GFP fusion protein together with a red fluorescent protein (RFP) fusion of the Ran exchange factor, RCC1. RCC1 is a constitutively nuclear protein and, therefore, served as a marker for transfected cells. On addition of exportin-5 to permeabilized cells, GFP-ILF3 was exported rapidly from the nucleus, whereas no export was observed upon addition of Crm1 or buffer alone (Fig. 7 A). Export of GFP-ILF3 by exportin-5 was inhibited completely by the addition of WGA, a protein that binds tightly to nuclear pores and inhibits receptor-mediated nucleocytoplasmic transport pathways. LMB had no effect on the export activity of exportin-5 (Fig. 7 A). Additionally, exportin-5 was unable to promote nuclear export of PKI, a protein whose export is facilitated by both Crm1 and calreticulin (Jim Holaska, personal communication). Quantitation of the mean nuclear fluorescence demonstrated that the addition of exportin-5 reduced the nuclear fluorescence of GFP-ILF3 by ∼70% over buffer alone (Fig. 7 B). Together these data demonstrate that exportin-5 is a nuclear export receptor for ILF3.

Exportin-5 mediates export of ILF3 in digitonin- permeabilized cells. (A) HeLa cells were cotransfected with pEGFP-ILF3 and pKRed-RCC1. pKRed-RCC1 was used as a nuclear marker for transfected cells. The cells were permeabilized with digitonin, and export assays were performed in buffer alone or with buffer plus exportin-5–His6 (100 μg/ml), exportin-5–His6 (100 μg/ml) pretreated with LMB (1 μM), exportin-5–His6 (100 μg/ml) plus WGA (200 μg/ml), or Crm1-His6 (100 μg/ml). (B) Mean nuclear fluorescence values for GFP-ILF3 were obtained using Openlab (Improvision). All values were corrected for background fluorescence levels and normalized to the mean nuclear fluorescence levels of permeabilized buffer-treated cells. Each data point represents the mean nuclear fluorescence obtained from 30 randomly chosen cells. Error is expressed as ±1 SD from the mean. Bar, 10 μm.

DiscussionWe have identified exportin-5, a novel member of the Kapβ family, as a nuclear export receptor for the double-stranded RNA binding protein (dsRBP), ILF3. Importantly, the association of RNA with ILF3 inhibits exportin-5 binding. Exportin-5 also interacts with dsRBDs from several other RNA binding proteins, including staufen, Spnr, and PKR, in a RanGTP-dependent manner. These results argue that exportin-5 is a general export receptor for proteins bearing dsRBDs, and we propose the following model (Fig. 8): dsRBP associate with exportin-5 and RanGTP in the nucleus. These complexes translocate through the NPC to the cytoplasm. RanGTP hydrolysis coupled with RNA binding promotes release and retention of the cargoes in the cytoplasm. Since cytoplasmic injection of RNase has been shown to result in nuclear accumulation of CBTF122 (Brzostowski et al., 2000), we speculate that RNA binding to dsRBDs prevents import by blocking the interaction of adjacent bipartite NLSs with their import receptors. Therefore, the destruction of bound RNA transcripts would permit reimport into the nucleus.Figure 8.
Exportin-5 is a general export receptor for dsRBP. Exportin-5 and RanGTP associate with dsRBP in the nucleus. This export complex translocates through the NPC to the cytoplasm. RanGTP hydrolysis and binding to 3′ untranslated region elements on the target mRNA promotes release and retention of the dsRBP in the cytoplasm. RNA binding to the dsRBDs may inhibit import by preventing the interaction of an adjacent bipartite NLS with its nuclear import receptor (Imp). Release of the mRNA transcript due to RNA degradation would then permit import of the dsRBP. The shaded region of the dsRBP represents the dsRBD. A putative bipartite NLS is adjacent to the dsRBDs (see Fig. 3 A).DsRNA binding proteins are essential during development but may not be required in differentiated cells. Nonetheless, we were surprised by the absence of detectable exportin-5 transcripts in BHK21 cells. It is conceivable that the sequence of hamster exportin-5 is sufficiently different from human that the PCR could not efficiently amplify the transcripts. However, BHK21 cells are also unable to support the export of ILF3 dsRBD2. Conversely, HeLa cells, which express exportin-5 transcripts, are competent for ILF3 export. Therefore, BHK21 cells behave empirically as an exportin-5–null cell line, and we exploited this phenotype to demonstrate that the coinjection of recombinant exportin-5 can rescue ILF3 export. These data combined with the results of the in vitro export assays argue strongly that exportin-5 is the major, and perhaps only, transport receptor for the export of dsRBD-containing proteins in mammalian cells.Binding of exportin-5 to ILF3 dsRBD2 is sensitive to deletions that remove either the α1 or α2 helices. Therefore, the exportin-5 binding domain overlaps with the dsRBD. This ∼69 residue NES is considerably longer than the leucine-rich NESs recognized by Crm1 but is of comparable length to the ∼41 residues of importin α that are required to bind importin β (IBB) (Gorlich et al., 1996) and the ∼42 amino acids of the ribosomal protein rpL23a that are required to bind multiple Kapβs (BIB) (Jakel and Gorlich, 1998). A large interaction domain may provide numerous contact residues and result in enhanced transport receptor binding. Moreover, complex NESs may have multiple functions that allow for the regulation of receptor–cargo interactions. Since RNA competes with exportin-5 for binding to dsRBDs, this is a compelling argument for dsRBD-containing proteins. One consequence of this competition is that cytoplasmic RNAs may facilitate cargo release from the exportin, although exportin-5 may also prevent the premature association of cargoes with dsRNAs in the nucleus.DsRBD-containing proteins are trans-acting factors involved in mRNA localization, translation, and stability. In the Drosophila oocyte, staufen is a cytoplasmic protein required to localize maternal bicoid mRNAs to the anterior pole and maternal oskar mRNAs to the posterior pole of the oocyte. This asymmetric localization of mRNAs is essential for anterior-posterior axis formation in the developing embryo (St Johnston et al., 1991). Spnr interacts with and is involved in the translational activation of protamine-1 mRNA (Schumacher et al., 1995). Viral dsRNAs, such as those from adenovirus or HIV, activate PKR (Clemens and Elia, 1997). Activated PKR phosphorylates initiation factor 2α, which inhibits protein synthesis.For all mRNAs studied to date, elements necessary for localization are found in the 3′ untranslated region (Bashirullah et al., 1998). It is generally accepted that localized RNAs form large ribonucleoprotein complexes that anchor these RNAs to cytoskeletal elements and promote their transport to distinct subcellular compartments (Bashirullah et al., 1998). Accordingly, staufen and Spnr colocalize with cytoplasmic microtubule networks in developing Drosophila oocytes and mouse spermatids, respectively (Schumacher et al., 1995; Bashirullah et al., 1998). However, these proteins also have bipartite NLSs directly adjacent to their dsRBDs, suggesting that they may have both cytoplasmic and nuclear activities that are regulated by RNA binding. The balance between these activities may be controlled by access to both exportin-5 and import receptors. In the future, it will be essential to test our model by first identifying the import mechanism for dsRBD proteins and then determining the developmental consequences of disrupting either the import or export of ILF3/CBTF122.

We have identified exportin-5, a novel member of the Kapβ family, as a nuclear export receptor for the double-stranded RNA binding protein (dsRBP), ILF3. Importantly, the association of RNA with ILF3 inhibits exportin-5 binding. Exportin-5 also interacts with dsRBDs from several other RNA binding proteins, including staufen, Spnr, and PKR, in a RanGTP-dependent manner. These results argue that exportin-5 is a general export receptor for proteins bearing dsRBDs, and we propose the following model (Fig. 8): dsRBP associate with exportin-5 and RanGTP in the nucleus. These complexes translocate through the NPC to the cytoplasm. RanGTP hydrolysis coupled with RNA binding promotes release and retention of the cargoes in the cytoplasm. Since cytoplasmic injection of RNase has been shown to result in nuclear accumulation of CBTF122 (Brzostowski et al., 2000), we speculate that RNA binding to dsRBDs prevents import by blocking the interaction of adjacent bipartite NLSs with their import receptors. Therefore, the destruction of bound RNA transcripts would permit reimport into the nucleus.

Exportin-5 is a general export receptor for dsRBP. Exportin-5 and RanGTP associate with dsRBP in the nucleus. This export complex translocates through the NPC to the cytoplasm. RanGTP hydrolysis and binding to 3′ untranslated region elements on the target mRNA promotes release and retention of the dsRBP in the cytoplasm. RNA binding to the dsRBDs may inhibit import by preventing the interaction of an adjacent bipartite NLS with its nuclear import receptor (Imp). Release of the mRNA transcript due to RNA degradation would then permit import of the dsRBP. The shaded region of the dsRBP represents the dsRBD. A putative bipartite NLS is adjacent to the dsRBDs (see Fig. 3 A).

DsRNA binding proteins are essential during development but may not be required in differentiated cells. Nonetheless, we were surprised by the absence of detectable exportin-5 transcripts in BHK21 cells. It is conceivable that the sequence of hamster exportin-5 is sufficiently different from human that the PCR could not efficiently amplify the transcripts. However, BHK21 cells are also unable to support the export of ILF3 dsRBD2. Conversely, HeLa cells, which express exportin-5 transcripts, are competent for ILF3 export. Therefore, BHK21 cells behave empirically as an exportin-5–null cell line, and we exploited this phenotype to demonstrate that the coinjection of recombinant exportin-5 can rescue ILF3 export. These data combined with the results of the in vitro export assays argue strongly that exportin-5 is the major, and perhaps only, transport receptor for the export of dsRBD-containing proteins in mammalian cells.

Binding of exportin-5 to ILF3 dsRBD2 is sensitive to deletions that remove either the α1 or α2 helices. Therefore, the exportin-5 binding domain overlaps with the dsRBD. This ∼69 residue NES is considerably longer than the leucine-rich NESs recognized by Crm1 but is of comparable length to the ∼41 residues of importin α that are required to bind importin β (IBB) (Gorlich et al., 1996) and the ∼42 amino acids of the ribosomal protein rpL23a that are required to bind multiple Kapβs (BIB) (Jakel and Gorlich, 1998). A large interaction domain may provide numerous contact residues and result in enhanced transport receptor binding. Moreover, complex NESs may have multiple functions that allow for the regulation of receptor–cargo interactions. Since RNA competes with exportin-5 for binding to dsRBDs, this is a compelling argument for dsRBD-containing proteins. One consequence of this competition is that cytoplasmic RNAs may facilitate cargo release from the exportin, although exportin-5 may also prevent the premature association of cargoes with dsRNAs in the nucleus.

DsRBD-containing proteins are trans-acting factors involved in mRNA localization, translation, and stability. In the Drosophila oocyte, staufen is a cytoplasmic protein required to localize maternal bicoid mRNAs to the anterior pole and maternal oskar mRNAs to the posterior pole of the oocyte. This asymmetric localization of mRNAs is essential for anterior-posterior axis formation in the developing embryo (St Johnston et al., 1991). Spnr interacts with and is involved in the translational activation of protamine-1 mRNA (Schumacher et al., 1995). Viral dsRNAs, such as those from adenovirus or HIV, activate PKR (Clemens and Elia, 1997). Activated PKR phosphorylates initiation factor 2α, which inhibits protein synthesis.

For all mRNAs studied to date, elements necessary for localization are found in the 3′ untranslated region (Bashirullah et al., 1998). It is generally accepted that localized RNAs form large ribonucleoprotein complexes that anchor these RNAs to cytoskeletal elements and promote their transport to distinct subcellular compartments (Bashirullah et al., 1998). Accordingly, staufen and Spnr colocalize with cytoplasmic microtubule networks in developing Drosophila oocytes and mouse spermatids, respectively (Schumacher et al., 1995; Bashirullah et al., 1998). However, these proteins also have bipartite NLSs directly adjacent to their dsRBDs, suggesting that they may have both cytoplasmic and nuclear activities that are regulated by RNA binding. The balance between these activities may be controlled by access to both exportin-5 and import receptors. In the future, it will be essential to test our model by first identifying the import mechanism for dsRBD proteins and then determining the developmental consequences of disrupting either the import or export of ILF3/CBTF122.

Materials and methodsCloning and recombinant protein expressionKIAA clone 1291 was obtained from Dr. Takahiro Nagase (Kazusa DNA Research Institute, Chiba, Japan). The clone had a two base pair deletion at nucleotides 2,265 and 2,266. To reinsert these bases, we performed site-directed mutagenesis using Quikchange (Stratagene). 5′ RACE was performed to obtain the complete ORF of exportin-5. We used human brain Matchmaker cDNA (CLONTECH Laboratories, Inc.) as template, a sense primer (AP1; CLONTECH Laboratories, Inc.) to the Marathon cDNA adaptor, and two nested primers complementary to nucleotides 996–1,019 and 962–987 of the exportin-5 cDNA sequence. The ∼1,050-bp product was cloned into pCR2.1-TOPO (Invitrogen) and sequenced.The full-length exportin-5 cDNA was assembled using a unique NsiI site at position 891 of exportin-5. The RACE product was excised from PCR2.1-TOPO XhoI/NsiI and ligated into KIAA1291 (in pBluescript II SK+) digested with XhoI/NsiI. The entire exportin-5 cDNA was sequenced and then moved into pKmyc, pKH3, pGBT10, and pQE60 (QIAGEN). pKmyc and pKH3 are mammalian vectors that express fusion proteins bearing an NH2-terminal myc and a triple HA tag, respectively. pGBT10 is a yeast vector that expresses proteins bearing the GAL4 DNA binding domain (DBD). pQE60 is a bacterial vector that expresses proteins bearing a COOH-terminal His6 tag. The protein fragment encoded by KIAA1291 is referred to as ΔN exportin-5.GST-ILF3(533–640) (GST-ILF3[533–640]) and GST-GFP-ILF3(533–640) (GGILF3[533–640]) were generated by PCR amplification of the murine dihybrid clone and ligation into pGEX-2T (Amersham Pharmacia Biotech) and pGEX-GFP. Full-length ILF3, Spnr, and staufen were generated by PCR amplification using human brain cDNA (CLONTECH Laboratories, Inc.) as the template. ILF3 was cloned into pKH3 and pEGFP-C2 (CLONTECH Laboratories, Inc.). GST-ILF3 dsRBD2(524–592), GST-Spnr dsRBD2(505–574), GST-staufen dsRBD3 (99–167), and GST-ILF3(404–592) were generated by PCR amplification off the human cDNA followed by ligation into pGEX-2T. Site-directed mutagenesis of ILF3 F432 and F559 to alanine to construct GST-ILF3(404–592)mut was performed using Quikchange (Stratagene). Murine PKR dsRBD (2–77) was excised from VP16 and inserted into pGEX-4T.The pKRFP-RCC1 vector expressing RCC1 with an NH2-terminal RFP tag was provided by Michael Nemergut (University of Virginia). PGBT10-imp 11 was provided by Scott Plafker (University of Virginia). pQE60-Crm1 was a gift from Iain Mattaj (European Molecular Biology Laboratory, Heidelberg, Germany) (Arts et al., 1998), and pQE32-RanQ69L was a gift from Dirk Görlich (University of Heidelberg, Heidelberg, Germany) (Izaurralde et al., 1997). pGEX-PKI(36–50) that expresses the PKI NES was a gift from Ben Black (University of Virginia).Exportin-5-His6 and all GST fusion proteins were expressed in E. coli TG1, first by growth at 37°C in LB supplemented with 2% (vol/vol) ethanol followed by induction with 400 μM IPTG at room temperature overnight. Exportin-5-His6 was purified on NTA-Ni2+ beads (QIAGEN). All GST fusion proteins were purified on glutathione–Sepharose (GSH) beads; Amersham Pharmacia Biotech). Purification of wild-type Ran, RanQ69L, Crm1, importin β, and RanGAP has been described previously (Lounsbury et al., 1996; Plafker and Macara, 2000; Brownawell et al., 2001).Northern analysisA human poly(A)+ RNA tissue Northern blot (CLONTECH Laboratories, Inc.) was hybridized according to the manufacturer's instructions. [α-32P]dCTP-labeled exportin-5 and human β-actin DNA probes were generated with the Random Primer DNA labeling kit (GIBCO BRL).Ran binding assaysGST-Ran was loaded with GTP or GDP as described (Lounsbury et al., 1994). After loading, GST-RanGDP was treated with RanGAP for 10 min at 30°C to remove any residual GTP-bound Ran. GST-RanGTP and GST-RanGDP were bound to GSH beads, washed with binding buffer (20 mM Hepes-KOH, pH 7.3, 150 mM potassium acetate, 2 mM magnesium acetate, 0.1% Tween-20, 14 mM β-mercaptoethanol [βME], and 0.5% ovalbumin), and incubated with 200 nM exportin-5-His6 for 1 h at 4°C. After washing, the beads were resuspended in laemmli sample buffer and analyzed by SDS-PAGE and Coomassie staining. RanGAP protection assays were performed in duplicate as described (Lounsbury et al., 1996).Nucleoporin binding assaysHEK293 cells were transfected with pKH3-imp β or pKH3-exp5. The cells were washed with ice-cold PBS and lysed with solubilization buffer (SB: 50 mM Tris, pH 8.0, 300 mM sodium chloride, 5 mM EDTA, 5 mM EGTA, 15 mM magnesium chloride, 1% NP-40, 20 μg aprotinin/ml, 10 μg leupeptin/ml, 1 mM PMSF, and 4 mM βME). Lysed cells were scraped off the plates, and the supernatant was cleared by centrifugation (14,000 rpms; 5 min). To each cell lysate 2 μg of 12CA5 mAb was added ±2 μM RanQ69L. These samples were incubated for 1 h at 4°C. Protein A beads were then added and incubated for 1 h at 4°C. Immunoprecipitates were washed four times with SB, resuspended in Laemmli sample buffer, separated by SDS-PAGE, and analyzed by Western blotting. Nucleoporins were detected using RL1 mAb (1:500; Affinity BioReagents, Inc.) and HRP-conjugated goat anti–mouse IgG (1:20,000; Pierce Chemical Co.). HA3-impβ and HA3-exp-5 were detected using HRP-conjugated 12CA5 (1:5,000).Cell culture, transfections, and processingBHK21, human embryonic kidney (HEK293), HeLa, and GSN2 cells were grown in DME (supplemented with 5% [vol/vol] FCS) and cultured in a humidified 37°C/5% CO2 incubator. All cells for fusion and binding assays were transfected using the calcium phosphate method. At 24 h after transfection, the medium was replaced and the cells were incubated for an additional 24 h. Transfected HEK293 cells were harvested for immunoprecipitation at 48 h after transfection.Cells to be analyzed by fluorescence microscopy were fixed with 4% (wt/vol) paraformaldehyde, 2% (wt/vol) sucrose in PBS, permeabilized with −20°C methanol, and blocked in 10% (wt/vol) BSA/PBS at room temperature. BHK21 cells expressing myc-tagged exportin-5 in the fusion assays were incubated with 9E10 mAb (2 μg/ml) and Texas red–conjugated goat anti–mouse IgG (1:1,500; Jackson ImmunoResearch Laboratories). DNA was stained with DAPI (10 ng/ml), and then the coverslips were mounted on glass slides using GelMount (Biomeda). Images were captured using a 60× water immersion objective lens on a Nikon inverted microscope with a Hamamatsu CCD camera. All immunofluorescence data were obtained and processed using Openlab (Improvision) and Adobe Photoshop® software.Heterokaryon fusion assaysBHK21 cells were transfected with pKmyc–exportin-5. An acceptor cell line, GSN2, was a gift from Bryce Paschal (University of Virginia) (Black et al., 1999). BHK21 and GSN2 cells were coplated onto poly-l-lysine–coated coverslips overnight. Cells were treated with 50 μM cycloheximide for 30 min. The plasma membranes were then fused for 2 min with 50% (wt/vol) polyethylene glycol (8,000) prewarmed to 37°C. Fused cells were washed and incubated at 37°C for 1 h in the presence of cycloheximide. Cells were then fixed and processed as described above.Yeast dihybrid and conjugation assaysFull-length exportin-5 was used as bait for a yeast dihybrid screen of a random-primed murine 10-d embryo library. Mating assays were performed as described (Plafker and Macara, 2000). Exportin-5, ΔN exportin-5, importin-11, and importin β were all expressed in the S. cerevisiae HF7c (MATa) strain as COOH-terminal fusions to the DBD of the GAL4 protein. Ran and ILF3(533–640) were expressed in the W303 (MATα) strain as COOH-terminal fusions to the transactivation domain of VP16.Exportin-5 binding assaysAll GST-ILF3 and GST-dsRBD constructs were bound to GSH beads and incubated with 200 nM recombinant exportin-5–His6, 500 nM Crm1-His6, or 200 nM importin β–His6 ±3 μM RanQ69L for 1 h at 4°C in binding buffer. GST-PKI(36–50) was incubated with 200 nM Crm1-His6 ±3 μM RanQ69L. Where indicated, 200 nM exportin-5–His6 or Crm1-His6 was incubated with 500 nM LMB for 15 min at room temperature before addition to the binding assays. After washing, the beads were resuspended in Laemmli sample buffer, separated by SDS-PAGE, and analyzed by Western blotting. Exportin-5–His6, Crm1-His6, and importin β-His6 were detected with anti-His6 mAb (1:2,000; BabCo). GST fusions were detected with anti-GST mAb (1:20,000; Sigma-Aldrich). Ran was detected using anti-Ran mAb (1:2,000; Transduction Labs). All were followed by HRP-conjugated goat anti–mouse IgG (1:20,000; Jackson ImmunoResearch Laboratories). Binding of HA-ILF3 to exportin-5 was performed as described previously (Brownawell et al., 2001).RNA binding assayspBluescript II KS (Stratagene) was linearized with SacI or Kpn1 and used as template for T3 and T7 polymerase transcription reactions (Ampliscribe RNA synthesis kit; Epicentre). The resulting ∼100-bp RNA products corresponded to the plus and minus ends of the polylinker. Following RNA synthesis, the template was degraded using DNase, and unincorporated nucleotides were removed by gel filtration. Opposing RNA strands were annealed by boiling for 3 min then cooling to room temperature. Where indicated, the T3 strand was internally labeled with [α-32P]ATP using the Riboscribe T3 RNA synthesis kit (Epicentre).For the α-32P–labeled dsRNA binding assay, 1 μg of each protein was separated by SDS-PAGE, transferred to nitrocellulose, incubated overnight at 4°C in RNA buffer (50 mM Tris, pH 7.5, 50 mM sodium chloride, 1 mM EDTA, 1 mM DTT, and 0.25 mM βME) supplemented with 5% nonfat dry milk, and transferred to 10 ml RNA buffer supplemented with 5 μg/ml polydI-dC (Sigma-Aldrich) to compete nonspecific interactions. Approximately 250,000 cpm of α-32P–labeled dsRNA probe was added for 1 h at room temperature. After washing with RNA buffer, the nitrocellulose was exposed to film for ∼18 h before developing. A duplicate gel was Coomassie stained.For the dsRNA/exportin-5 competition assay, nonradiolabeled RNA was synthesized: one-half of the product was annealed (dsRNA), whereas the other half was not and was used as a ssRNA control. The amount of synthesized RNA was quantified using the absorbance at 260 nm. Approximately 3 μg GST-ILF3(404–592) was bound to GSH-agarose beads to which ∼6 μg of either ssRNA or dsRNA was added and incubated for 1 h at room temperature. Exportin-5 was added to 100 nM ±3 μM RanQ69L for 1 h at room temperature. All samples were washed, resuspended in Laemmli sample buffer, separated by SDS-PAGE, and analyzed by Western blotting as described above.RT-PCR analysisRNAs were isolated from the cytoplasm of ∼107 BHK21 or HeLa cells using the RNeasy kit from QIAGEN. RT-PCR was performed using the One Step RT-PCR kit (QIAGEN). The exportin-5 primers were designed to anneal to two different exons to differentiate between amplification off of mRNA and any contaminating genomic DNA.MicroinjectionsMicroinjections were performed using BHK21 and HeLa cells as described (Plafker and Macara, 2000). GGILF3(533–640) was injected at 18 μM. Crm1-His6 and exportin-5-His6 were injected at 2 μM. TRITC-labeled dextran (1 mg/ml; Sigma-Aldrich) was included as an injection site marker. After injections, samples were incubated for 30 min at 37°C before fixation.In vitro nuclear export assaysHeLa cells were plated on poly-l-lysine–coated coverslips and cotransfected with pEGFP-C2-ILF3 and pKRFP-RCC1 using Effectene (QIAGEN). At ∼20 h after transfection, the cells were permeabilized for 5 min at 4°C with 0.005% digitonin in transport buffer (TB: 20 mM Hepes-KOH, pH 7.4, 110 mM potassium acetate, 2 mM magnesium acetate, and 0.5 mM EGTA). The cells were washed with TB. All export cocktails contained an energy regenerating system (20 mM phosphocreatine, 1 mM GTP, 1 mM ATP, and 50 μg/ml creatine phosphokinase) and wild-type Ran (1.5 μM) in TB. Exportin-5–His6 or Crm1-His6 was added to a final concentration of 100 μg/ml, and where indicated WGA was added to 200 μg/ml. As indicated, LMB (Sigma-Aldrich) was incubated (at 1 μM) with the soluble factors for 15 min at room temperature before performing the export reactions. Export was performed for 20 min at 30°C. After export, the cells were fixed, permeabilized, and processed as above. Images for all export assays were obtained using identical camera settings.

Materials and methods

Cloning and recombinant protein expressionKIAA clone 1291 was obtained from Dr. Takahiro Nagase (Kazusa DNA Research Institute, Chiba, Japan). The clone had a two base pair deletion at nucleotides 2,265 and 2,266. To reinsert these bases, we performed site-directed mutagenesis using Quikchange (Stratagene). 5′ RACE was performed to obtain the complete ORF of exportin-5. We used human brain Matchmaker cDNA (CLONTECH Laboratories, Inc.) as template, a sense primer (AP1; CLONTECH Laboratories, Inc.) to the Marathon cDNA adaptor, and two nested primers complementary to nucleotides 996–1,019 and 962–987 of the exportin-5 cDNA sequence. The ∼1,050-bp product was cloned into pCR2.1-TOPO (Invitrogen) and sequenced.The full-length exportin-5 cDNA was assembled using a unique NsiI site at position 891 of exportin-5. The RACE product was excised from PCR2.1-TOPO XhoI/NsiI and ligated into KIAA1291 (in pBluescript II SK+) digested with XhoI/NsiI. The entire exportin-5 cDNA was sequenced and then moved into pKmyc, pKH3, pGBT10, and pQE60 (QIAGEN). pKmyc and pKH3 are mammalian vectors that express fusion proteins bearing an NH2-terminal myc and a triple HA tag, respectively. pGBT10 is a yeast vector that expresses proteins bearing the GAL4 DNA binding domain (DBD). pQE60 is a bacterial vector that expresses proteins bearing a COOH-terminal His6 tag. The protein fragment encoded by KIAA1291 is referred to as ΔN exportin-5.GST-ILF3(533–640) (GST-ILF3[533–640]) and GST-GFP-ILF3(533–640) (GGILF3[533–640]) were generated by PCR amplification of the murine dihybrid clone and ligation into pGEX-2T (Amersham Pharmacia Biotech) and pGEX-GFP. Full-length ILF3, Spnr, and staufen were generated by PCR amplification using human brain cDNA (CLONTECH Laboratories, Inc.) as the template. ILF3 was cloned into pKH3 and pEGFP-C2 (CLONTECH Laboratories, Inc.). GST-ILF3 dsRBD2(524–592), GST-Spnr dsRBD2(505–574), GST-staufen dsRBD3 (99–167), and GST-ILF3(404–592) were generated by PCR amplification off the human cDNA followed by ligation into pGEX-2T. Site-directed mutagenesis of ILF3 F432 and F559 to alanine to construct GST-ILF3(404–592)mut was performed using Quikchange (Stratagene). Murine PKR dsRBD (2–77) was excised from VP16 and inserted into pGEX-4T.The pKRFP-RCC1 vector expressing RCC1 with an NH2-terminal RFP tag was provided by Michael Nemergut (University of Virginia). PGBT10-imp 11 was provided by Scott Plafker (University of Virginia). pQE60-Crm1 was a gift from Iain Mattaj (European Molecular Biology Laboratory, Heidelberg, Germany) (Arts et al., 1998), and pQE32-RanQ69L was a gift from Dirk Görlich (University of Heidelberg, Heidelberg, Germany) (Izaurralde et al., 1997). pGEX-PKI(36–50) that expresses the PKI NES was a gift from Ben Black (University of Virginia).Exportin-5-His6 and all GST fusion proteins were expressed in E. coli TG1, first by growth at 37°C in LB supplemented with 2% (vol/vol) ethanol followed by induction with 400 μM IPTG at room temperature overnight. Exportin-5-His6 was purified on NTA-Ni2+ beads (QIAGEN). All GST fusion proteins were purified on glutathione–Sepharose (GSH) beads; Amersham Pharmacia Biotech). Purification of wild-type Ran, RanQ69L, Crm1, importin β, and RanGAP has been described previously (Lounsbury et al., 1996; Plafker and Macara, 2000; Brownawell et al., 2001).

Cloning and recombinant protein expression

KIAA clone 1291 was obtained from Dr. Takahiro Nagase (Kazusa DNA Research Institute, Chiba, Japan). The clone had a two base pair deletion at nucleotides 2,265 and 2,266. To reinsert these bases, we performed site-directed mutagenesis using Quikchange (Stratagene). 5′ RACE was performed to obtain the complete ORF of exportin-5. We used human brain Matchmaker cDNA (CLONTECH Laboratories, Inc.) as template, a sense primer (AP1; CLONTECH Laboratories, Inc.) to the Marathon cDNA adaptor, and two nested primers complementary to nucleotides 996–1,019 and 962–987 of the exportin-5 cDNA sequence. The ∼1,050-bp product was cloned into pCR2.1-TOPO (Invitrogen) and sequenced.

The full-length exportin-5 cDNA was assembled using a unique NsiI site at position 891 of exportin-5. The RACE product was excised from PCR2.1-TOPO XhoI/NsiI and ligated into KIAA1291 (in pBluescript II SK+) digested with XhoI/NsiI. The entire exportin-5 cDNA was sequenced and then moved into pKmyc, pKH3, pGBT10, and pQE60 (QIAGEN). pKmyc and pKH3 are mammalian vectors that express fusion proteins bearing an NH2-terminal myc and a triple HA tag, respectively. pGBT10 is a yeast vector that expresses proteins bearing the GAL4 DNA binding domain (DBD). pQE60 is a bacterial vector that expresses proteins bearing a COOH-terminal His6 tag. The protein fragment encoded by KIAA1291 is referred to as ΔN exportin-5.

GST-ILF3(533–640) (GST-ILF3[533–640]) and GST-GFP-ILF3(533–640) (GGILF3[533–640]) were generated by PCR amplification of the murine dihybrid clone and ligation into pGEX-2T (Amersham Pharmacia Biotech) and pGEX-GFP. Full-length ILF3, Spnr, and staufen were generated by PCR amplification using human brain cDNA (CLONTECH Laboratories, Inc.) as the template. ILF3 was cloned into pKH3 and pEGFP-C2 (CLONTECH Laboratories, Inc.). GST-ILF3 dsRBD2(524–592), GST-Spnr dsRBD2(505–574), GST-staufen dsRBD3 (99–167), and GST-ILF3(404–592) were generated by PCR amplification off the human cDNA followed by ligation into pGEX-2T. Site-directed mutagenesis of ILF3 F432 and F559 to alanine to construct GST-ILF3(404–592)mut was performed using Quikchange (Stratagene). Murine PKR dsRBD (2–77) was excised from VP16 and inserted into pGEX-4T.

The pKRFP-RCC1 vector expressing RCC1 with an NH2-terminal RFP tag was provided by Michael Nemergut (University of Virginia). PGBT10-imp 11 was provided by Scott Plafker (University of Virginia). pQE60-Crm1 was a gift from Iain Mattaj (European Molecular Biology Laboratory, Heidelberg, Germany) (Arts et al., 1998), and pQE32-RanQ69L was a gift from Dirk Görlich (University of Heidelberg, Heidelberg, Germany) (Izaurralde et al., 1997). pGEX-PKI(36–50) that expresses the PKI NES was a gift from Ben Black (University of Virginia).

Exportin-5-His6 and all GST fusion proteins were expressed in E. coli TG1, first by growth at 37°C in LB supplemented with 2% (vol/vol) ethanol followed by induction with 400 μM IPTG at room temperature overnight. Exportin-5-His6 was purified on NTA-Ni2+ beads (QIAGEN). All GST fusion proteins were purified on glutathione–Sepharose (GSH) beads; Amersham Pharmacia Biotech). Purification of wild-type Ran, RanQ69L, Crm1, importin β, and RanGAP has been described previously (Lounsbury et al., 1996; Plafker and Macara, 2000; Brownawell et al., 2001).

Northern analysisA human poly(A)+ RNA tissue Northern blot (CLONTECH Laboratories, Inc.) was hybridized according to the manufacturer's instructions. [α-32P]dCTP-labeled exportin-5 and human β-actin DNA probes were generated with the Random Primer DNA labeling kit (GIBCO BRL).

Northern analysis

A human poly(A)+ RNA tissue Northern blot (CLONTECH Laboratories, Inc.) was hybridized according to the manufacturer's instructions. [α-32P]dCTP-labeled exportin-5 and human β-actin DNA probes were generated with the Random Primer DNA labeling kit (GIBCO BRL).

Ran binding assaysGST-Ran was loaded with GTP or GDP as described (Lounsbury et al., 1994). After loading, GST-RanGDP was treated with RanGAP for 10 min at 30°C to remove any residual GTP-bound Ran. GST-RanGTP and GST-RanGDP were bound to GSH beads, washed with binding buffer (20 mM Hepes-KOH, pH 7.3, 150 mM potassium acetate, 2 mM magnesium acetate, 0.1% Tween-20, 14 mM β-mercaptoethanol [βME], and 0.5% ovalbumin), and incubated with 200 nM exportin-5-His6 for 1 h at 4°C. After washing, the beads were resuspended in laemmli sample buffer and analyzed by SDS-PAGE and Coomassie staining. RanGAP protection assays were performed in duplicate as described (Lounsbury et al., 1996).

Ran binding assays

GST-Ran was loaded with GTP or GDP as described (Lounsbury et al., 1994). After loading, GST-RanGDP was treated with RanGAP for 10 min at 30°C to remove any residual GTP-bound Ran. GST-RanGTP and GST-RanGDP were bound to GSH beads, washed with binding buffer (20 mM Hepes-KOH, pH 7.3, 150 mM potassium acetate, 2 mM magnesium acetate, 0.1% Tween-20, 14 mM β-mercaptoethanol [βME], and 0.5% ovalbumin), and incubated with 200 nM exportin-5-His6 for 1 h at 4°C. After washing, the beads were resuspended in laemmli sample buffer and analyzed by SDS-PAGE and Coomassie staining. RanGAP protection assays were performed in duplicate as described (Lounsbury et al., 1996).

Nucleoporin binding assaysHEK293 cells were transfected with pKH3-imp β or pKH3-exp5. The cells were washed with ice-cold PBS and lysed with solubilization buffer (SB: 50 mM Tris, pH 8.0, 300 mM sodium chloride, 5 mM EDTA, 5 mM EGTA, 15 mM magnesium chloride, 1% NP-40, 20 μg aprotinin/ml, 10 μg leupeptin/ml, 1 mM PMSF, and 4 mM βME). Lysed cells were scraped off the plates, and the supernatant was cleared by centrifugation (14,000 rpms; 5 min). To each cell lysate 2 μg of 12CA5 mAb was added ±2 μM RanQ69L. These samples were incubated for 1 h at 4°C. Protein A beads were then added and incubated for 1 h at 4°C. Immunoprecipitates were washed four times with SB, resuspended in Laemmli sample buffer, separated by SDS-PAGE, and analyzed by Western blotting. Nucleoporins were detected using RL1 mAb (1:500; Affinity BioReagents, Inc.) and HRP-conjugated goat anti–mouse IgG (1:20,000; Pierce Chemical Co.). HA3-impβ and HA3-exp-5 were detected using HRP-conjugated 12CA5 (1:5,000).

Nucleoporin binding assays

HEK293 cells were transfected with pKH3-imp β or pKH3-exp5. The cells were washed with ice-cold PBS and lysed with solubilization buffer (SB: 50 mM Tris, pH 8.0, 300 mM sodium chloride, 5 mM EDTA, 5 mM EGTA, 15 mM magnesium chloride, 1% NP-40, 20 μg aprotinin/ml, 10 μg leupeptin/ml, 1 mM PMSF, and 4 mM βME). Lysed cells were scraped off the plates, and the supernatant was cleared by centrifugation (14,000 rpms; 5 min). To each cell lysate 2 μg of 12CA5 mAb was added ±2 μM RanQ69L. These samples were incubated for 1 h at 4°C. Protein A beads were then added and incubated for 1 h at 4°C. Immunoprecipitates were washed four times with SB, resuspended in Laemmli sample buffer, separated by SDS-PAGE, and analyzed by Western blotting. Nucleoporins were detected using RL1 mAb (1:500; Affinity BioReagents, Inc.) and HRP-conjugated goat anti–mouse IgG (1:20,000; Pierce Chemical Co.). HA3-impβ and HA3-exp-5 were detected using HRP-conjugated 12CA5 (1:5,000).

Cell culture, transfections, and processingBHK21, human embryonic kidney (HEK293), HeLa, and GSN2 cells were grown in DME (supplemented with 5% [vol/vol] FCS) and cultured in a humidified 37°C/5% CO2 incubator. All cells for fusion and binding assays were transfected using the calcium phosphate method. At 24 h after transfection, the medium was replaced and the cells were incubated for an additional 24 h. Transfected HEK293 cells were harvested for immunoprecipitation at 48 h after transfection.Cells to be analyzed by fluorescence microscopy were fixed with 4% (wt/vol) paraformaldehyde, 2% (wt/vol) sucrose in PBS, permeabilized with −20°C methanol, and blocked in 10% (wt/vol) BSA/PBS at room temperature. BHK21 cells expressing myc-tagged exportin-5 in the fusion assays were incubated with 9E10 mAb (2 μg/ml) and Texas red–conjugated goat anti–mouse IgG (1:1,500; Jackson ImmunoResearch Laboratories). DNA was stained with DAPI (10 ng/ml), and then the coverslips were mounted on glass slides using GelMount (Biomeda). Images were captured using a 60× water immersion objective lens on a Nikon inverted microscope with a Hamamatsu CCD camera. All immunofluorescence data were obtained and processed using Openlab (Improvision) and Adobe Photoshop® software.

Cell culture, transfections, and processing

BHK21, human embryonic kidney (HEK293), HeLa, and GSN2 cells were grown in DME (supplemented with 5% [vol/vol] FCS) and cultured in a humidified 37°C/5% CO2 incubator. All cells for fusion and binding assays were transfected using the calcium phosphate method. At 24 h after transfection, the medium was replaced and the cells were incubated for an additional 24 h. Transfected HEK293 cells were harvested for immunoprecipitation at 48 h after transfection.

Cells to be analyzed by fluorescence microscopy were fixed with 4% (wt/vol) paraformaldehyde, 2% (wt/vol) sucrose in PBS, permeabilized with −20°C methanol, and blocked in 10% (wt/vol) BSA/PBS at room temperature. BHK21 cells expressing myc-tagged exportin-5 in the fusion assays were incubated with 9E10 mAb (2 μg/ml) and Texas red–conjugated goat anti–mouse IgG (1:1,500; Jackson ImmunoResearch Laboratories). DNA was stained with DAPI (10 ng/ml), and then the coverslips were mounted on glass slides using GelMount (Biomeda). Images were captured using a 60× water immersion objective lens on a Nikon inverted microscope with a Hamamatsu CCD camera. All immunofluorescence data were obtained and processed using Openlab (Improvision) and Adobe Photoshop® software.

Heterokaryon fusion assaysBHK21 cells were transfected with pKmyc–exportin-5. An acceptor cell line, GSN2, was a gift from Bryce Paschal (University of Virginia) (Black et al., 1999). BHK21 and GSN2 cells were coplated onto poly-l-lysine–coated coverslips overnight. Cells were treated with 50 μM cycloheximide for 30 min. The plasma membranes were then fused for 2 min with 50% (wt/vol) polyethylene glycol (8,000) prewarmed to 37°C. Fused cells were washed and incubated at 37°C for 1 h in the presence of cycloheximide. Cells were then fixed and processed as described above.

Heterokaryon fusion assays

BHK21 cells were transfected with pKmyc–exportin-5. An acceptor cell line, GSN2, was a gift from Bryce Paschal (University of Virginia) (Black et al., 1999). BHK21 and GSN2 cells were coplated onto poly-l-lysine–coated coverslips overnight. Cells were treated with 50 μM cycloheximide for 30 min. The plasma membranes were then fused for 2 min with 50% (wt/vol) polyethylene glycol (8,000) prewarmed to 37°C. Fused cells were washed and incubated at 37°C for 1 h in the presence of cycloheximide. Cells were then fixed and processed as described above.

Yeast dihybrid and conjugation assaysFull-length exportin-5 was used as bait for a yeast dihybrid screen of a random-primed murine 10-d embryo library. Mating assays were performed as described (Plafker and Macara, 2000). Exportin-5, ΔN exportin-5, importin-11, and importin β were all expressed in the S. cerevisiae HF7c (MATa) strain as COOH-terminal fusions to the DBD of the GAL4 protein. Ran and ILF3(533–640) were expressed in the W303 (MATα) strain as COOH-terminal fusions to the transactivation domain of VP16.

Yeast dihybrid and conjugation assays

Full-length exportin-5 was used as bait for a yeast dihybrid screen of a random-primed murine 10-d embryo library. Mating assays were performed as described (Plafker and Macara, 2000). Exportin-5, ΔN exportin-5, importin-11, and importin β were all expressed in the S. cerevisiae HF7c (MATa) strain as COOH-terminal fusions to the DBD of the GAL4 protein. Ran and ILF3(533–640) were expressed in the W303 (MATα) strain as COOH-terminal fusions to the transactivation domain of VP16.

Exportin-5 binding assaysAll GST-ILF3 and GST-dsRBD constructs were bound to GSH beads and incubated with 200 nM recombinant exportin-5–His6, 500 nM Crm1-His6, or 200 nM importin β–His6 ±3 μM RanQ69L for 1 h at 4°C in binding buffer. GST-PKI(36–50) was incubated with 200 nM Crm1-His6 ±3 μM RanQ69L. Where indicated, 200 nM exportin-5–His6 or Crm1-His6 was incubated with 500 nM LMB for 15 min at room temperature before addition to the binding assays. After washing, the beads were resuspended in Laemmli sample buffer, separated by SDS-PAGE, and analyzed by Western blotting. Exportin-5–His6, Crm1-His6, and importin β-His6 were detected with anti-His6 mAb (1:2,000; BabCo). GST fusions were detected with anti-GST mAb (1:20,000; Sigma-Aldrich). Ran was detected using anti-Ran mAb (1:2,000; Transduction Labs). All were followed by HRP-conjugated goat anti–mouse IgG (1:20,000; Jackson ImmunoResearch Laboratories). Binding of HA-ILF3 to exportin-5 was performed as described previously (Brownawell et al., 2001).

Exportin-5 binding assays

All GST-ILF3 and GST-dsRBD constructs were bound to GSH beads and incubated with 200 nM recombinant exportin-5–His6, 500 nM Crm1-His6, or 200 nM importin β–His6 ±3 μM RanQ69L for 1 h at 4°C in binding buffer. GST-PKI(36–50) was incubated with 200 nM Crm1-His6 ±3 μM RanQ69L. Where indicated, 200 nM exportin-5–His6 or Crm1-His6 was incubated with 500 nM LMB for 15 min at room temperature before addition to the binding assays. After washing, the beads were resuspended in Laemmli sample buffer, separated by SDS-PAGE, and analyzed by Western blotting. Exportin-5–His6, Crm1-His6, and importin β-His6 were detected with anti-His6 mAb (1:2,000; BabCo). GST fusions were detected with anti-GST mAb (1:20,000; Sigma-Aldrich). Ran was detected using anti-Ran mAb (1:2,000; Transduction Labs). All were followed by HRP-conjugated goat anti–mouse IgG (1:20,000; Jackson ImmunoResearch Laboratories). Binding of HA-ILF3 to exportin-5 was performed as described previously (Brownawell et al., 2001).

RNA binding assayspBluescript II KS (Stratagene) was linearized with SacI or Kpn1 and used as template for T3 and T7 polymerase transcription reactions (Ampliscribe RNA synthesis kit; Epicentre). The resulting ∼100-bp RNA products corresponded to the plus and minus ends of the polylinker. Following RNA synthesis, the template was degraded using DNase, and unincorporated nucleotides were removed by gel filtration. Opposing RNA strands were annealed by boiling for 3 min then cooling to room temperature. Where indicated, the T3 strand was internally labeled with [α-32P]ATP using the Riboscribe T3 RNA synthesis kit (Epicentre).For the α-32P–labeled dsRNA binding assay, 1 μg of each protein was separated by SDS-PAGE, transferred to nitrocellulose, incubated overnight at 4°C in RNA buffer (50 mM Tris, pH 7.5, 50 mM sodium chloride, 1 mM EDTA, 1 mM DTT, and 0.25 mM βME) supplemented with 5% nonfat dry milk, and transferred to 10 ml RNA buffer supplemented with 5 μg/ml polydI-dC (Sigma-Aldrich) to compete nonspecific interactions. Approximately 250,000 cpm of α-32P–labeled dsRNA probe was added for 1 h at room temperature. After washing with RNA buffer, the nitrocellulose was exposed to film for ∼18 h before developing. A duplicate gel was Coomassie stained.For the dsRNA/exportin-5 competition assay, nonradiolabeled RNA was synthesized: one-half of the product was annealed (dsRNA), whereas the other half was not and was used as a ssRNA control. The amount of synthesized RNA was quantified using the absorbance at 260 nm. Approximately 3 μg GST-ILF3(404–592) was bound to GSH-agarose beads to which ∼6 μg of either ssRNA or dsRNA was added and incubated for 1 h at room temperature. Exportin-5 was added to 100 nM ±3 μM RanQ69L for 1 h at room temperature. All samples were washed, resuspended in Laemmli sample buffer, separated by SDS-PAGE, and analyzed by Western blotting as described above.

RNA binding assays

pBluescript II KS (Stratagene) was linearized with SacI or Kpn1 and used as template for T3 and T7 polymerase transcription reactions (Ampliscribe RNA synthesis kit; Epicentre). The resulting ∼100-bp RNA products corresponded to the plus and minus ends of the polylinker. Following RNA synthesis, the template was degraded using DNase, and unincorporated nucleotides were removed by gel filtration. Opposing RNA strands were annealed by boiling for 3 min then cooling to room temperature. Where indicated, the T3 strand was internally labeled with [α-32P]ATP using the Riboscribe T3 RNA synthesis kit (Epicentre).

For the α-32P–labeled dsRNA binding assay, 1 μg of each protein was separated by SDS-PAGE, transferred to nitrocellulose, incubated overnight at 4°C in RNA buffer (50 mM Tris, pH 7.5, 50 mM sodium chloride, 1 mM EDTA, 1 mM DTT, and 0.25 mM βME) supplemented with 5% nonfat dry milk, and transferred to 10 ml RNA buffer supplemented with 5 μg/ml polydI-dC (Sigma-Aldrich) to compete nonspecific interactions. Approximately 250,000 cpm of α-32P–labeled dsRNA probe was added for 1 h at room temperature. After washing with RNA buffer, the nitrocellulose was exposed to film for ∼18 h before developing. A duplicate gel was Coomassie stained.

For the dsRNA/exportin-5 competition assay, nonradiolabeled RNA was synthesized: one-half of the product was annealed (dsRNA), whereas the other half was not and was used as a ssRNA control. The amount of synthesized RNA was quantified using the absorbance at 260 nm. Approximately 3 μg GST-ILF3(404–592) was bound to GSH-agarose beads to which ∼6 μg of either ssRNA or dsRNA was added and incubated for 1 h at room temperature. Exportin-5 was added to 100 nM ±3 μM RanQ69L for 1 h at room temperature. All samples were washed, resuspended in Laemmli sample buffer, separated by SDS-PAGE, and analyzed by Western blotting as described above.

RT-PCR analysisRNAs were isolated from the cytoplasm of ∼107 BHK21 or HeLa cells using the RNeasy kit from QIAGEN. RT-PCR was performed using the One Step RT-PCR kit (QIAGEN). The exportin-5 primers were designed to anneal to two different exons to differentiate between amplification off of mRNA and any contaminating genomic DNA.

RT-PCR analysis

RNAs were isolated from the cytoplasm of ∼107 BHK21 or HeLa cells using the RNeasy kit from QIAGEN. RT-PCR was performed using the One Step RT-PCR kit (QIAGEN). The exportin-5 primers were designed to anneal to two different exons to differentiate between amplification off of mRNA and any contaminating genomic DNA.

MicroinjectionsMicroinjections were performed using BHK21 and HeLa cells as described (Plafker and Macara, 2000). GGILF3(533–640) was injected at 18 μM. Crm1-His6 and exportin-5-His6 were injected at 2 μM. TRITC-labeled dextran (1 mg/ml; Sigma-Aldrich) was included as an injection site marker. After injections, samples were incubated for 30 min at 37°C before fixation.

Microinjections

Microinjections were performed using BHK21 and HeLa cells as described (Plafker and Macara, 2000). GGILF3(533–640) was injected at 18 μM. Crm1-His6 and exportin-5-His6 were injected at 2 μM. TRITC-labeled dextran (1 mg/ml; Sigma-Aldrich) was included as an injection site marker. After injections, samples were incubated for 30 min at 37°C before fixation.

In vitro nuclear export assaysHeLa cells were plated on poly-l-lysine–coated coverslips and cotransfected with pEGFP-C2-ILF3 and pKRFP-RCC1 using Effectene (QIAGEN). At ∼20 h after transfection, the cells were permeabilized for 5 min at 4°C with 0.005% digitonin in transport buffer (TB: 20 mM Hepes-KOH, pH 7.4, 110 mM potassium acetate, 2 mM magnesium acetate, and 0.5 mM EGTA). The cells were washed with TB. All export cocktails contained an energy regenerating system (20 mM phosphocreatine, 1 mM GTP, 1 mM ATP, and 50 μg/ml creatine phosphokinase) and wild-type Ran (1.5 μM) in TB. Exportin-5–His6 or Crm1-His6 was added to a final concentration of 100 μg/ml, and where indicated WGA was added to 200 μg/ml. As indicated, LMB (Sigma-Aldrich) was incubated (at 1 μM) with the soluble factors for 15 min at room temperature before performing the export reactions. Export was performed for 20 min at 30°C. After export, the cells were fixed, permeabilized, and processed as above. Images for all export assays were obtained using identical camera settings.

In vitro nuclear export assays

HeLa cells were plated on poly-l-lysine–coated coverslips and cotransfected with pEGFP-C2-ILF3 and pKRFP-RCC1 using Effectene (QIAGEN). At ∼20 h after transfection, the cells were permeabilized for 5 min at 4°C with 0.005% digitonin in transport buffer (TB: 20 mM Hepes-KOH, pH 7.4, 110 mM potassium acetate, 2 mM magnesium acetate, and 0.5 mM EGTA). The cells were washed with TB. All export cocktails contained an energy regenerating system (20 mM phosphocreatine, 1 mM GTP, 1 mM ATP, and 50 μg/ml creatine phosphokinase) and wild-type Ran (1.5 μM) in TB. Exportin-5–His6 or Crm1-His6 was added to a final concentration of 100 μg/ml, and where indicated WGA was added to 200 μg/ml. As indicated, LMB (Sigma-Aldrich) was incubated (at 1 μM) with the soluble factors for 15 min at room temperature before performing the export reactions. Export was performed for 20 min at 30°C. After export, the cells were fixed, permeabilized, and processed as above. Images for all export assays were obtained using identical camera settings.
