---
pmid: '30337683'
title: Cryo-EM structure of human SRCAP complex.
authors:
- Feng Y
- Tian Y
- Wu Z
- Xu Y
journal: Cell Res
year: '2018'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC6218446
doi: 10.1038/s41422-018-0102-y
---

# Cryo-EM structure of human SRCAP complex.
**Authors:** Feng Y, Tian Y, Wu Z, Xu Y
**Journal:** Cell Res (2018)
**DOI:** [10.1038/s41422-018-0102-y](https://doi.org/10.1038/s41422-018-0102-y)
**PMC:** [PMC6218446](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6218446/)

## Abstract

1. Cell Res. 2018 Nov;28(11):1121-1123. doi: 10.1038/s41422-018-0102-y. Epub 2018
 Oct 18.

Cryo-EM structure of human SRCAP complex.

Feng Y(1), Tian Y(1), Wu Z(1), Xu Y(2)(3)(4)(5).

Author information:
(1)Fudan University Shanghai Cancer Center, Key Laboratory of Medical 
Epigenetics and Metabolism, Institute of Biomedical Sciences, Shanghai Medical 
College of Fudan University, Shanghai, 200032, China.
(2)Fudan University Shanghai Cancer Center, Key Laboratory of Medical 
Epigenetics and Metabolism, Institute of Biomedical Sciences, Shanghai Medical 
College of Fudan University, Shanghai, 200032, China. xuyh@fudan.edu.cn.
(3)Key Laboratory of Molecular Medicine, Ministry of Education, Department of 
Systems Biology for Medicine, School of Basic Medical Sciences, Shanghai Medical 
College of Fudan University, Shanghai, 200032, China. xuyh@fudan.edu.cn.
(4)State Key Laboratory of Genetic Engineering, Collaborative Innovation Center 
of Genetics and Development, School of Life Sciences, Fudan University, 
Shanghai, 200433, China. xuyh@fudan.edu.cn.
(5)CAS Center for Excellence in Molecular Cell Science, Chinese Academy of 
Sciences, Shanghai, 200031, China. xuyh@fudan.edu.cn.

DOI: 10.1038/s41422-018-0102-y
PMCID: PMC6218446
PMID: 30337683 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing interests.

## Full Text

Received 2018 Aug 25; Revised 2018 Sep 25; Accepted 2018 Sep 27; Issue date 2018 Nov.

Eukaryotic genome packs into highly condensed chromatin, in which genomic DNA wraps around histone octamer formed by evolutionarily conserved histone proteins H2A, H2B, H3, and H4. The histone variant H2A.Z primarily exists within nucleosomes that flank gene promoters, centromeres, and chromatin boundaries. The H2A.Z-containing nucleosomes undergo rapid turnover, which is important for the regulation of gene expression, DNA repair, and chromosome stability 1 . The deposition of H2A.Z into genome is uniquely dependent on the SWI/SNF (switch/sucrose non-fermentable)-related Swr1 complex (referred to as Swr1-C hereafter) in yeast and SNF2-Related CBP Activator Protein (SRCAP) complex (referred to as SRCAP-C hereafter) in human. Swr1-C/SRCAP-C incorporates H2A.Z/H2B dimer into canonical nucleosomes coupled with the eviction of H2A/H2B dimer in a stepwise exchange manner 2 – 4 .

SRCAP complex is named for its scaffold protein, SRCAP, which is a member of ATP-dependent INO80 chromatin remodeler family that belongs to SNF2 family of ATPases 2 . The representative SNF2 chromatin remodeler family members include INO80, SWI/SNF, ISWI (imitation SWI) and CHD (chromodomain helicase DNA-binding). These chromatin-remodeling complexes play critical roles in transcriptional regulation through carrying out histone exchange, sliding, and spacing of nucleosomes in an ATP-dependent manner.

Structure of human SRCAP complex. a Structural organization and color-coded domain structure of human SRCAP complex components. The colors used for RUVBL1/2 hexamer are shown in a hexagon (right) with similar relative position in the structure. The same color scheme is used in all structure figures. The SRCAP protein serves as a scaffold. The three modules are indicated in boxes with grey background. N module, H2A.Z/H2B, and part of C module are invisible in the EM map and are indicated in dashed boxes. b Ribbon representations of SRCAP core complex in two distinct views. The insert of SRCAP is shown in mesh and indicated. The two parallel α-helices (right panel) are from the insert of SRCAP protein. c Structural comparison of SRCAP complex (left panel) and INO80-nucleosome complex (right panel) 9 in a similar view. The structures are shown in ribbon representations. The predicted binding position of nucleosome in SRCAP-C is indicated. The histone octamer and nucleosomal DNA in INO80-nucleosome structure are colored in grey and yellow, respectively. The inserts of SRCAP and INO80 are shown in surface representations for clarity. d Isolated inserts of SRCAP and INO80 shown in a similar view with surrounding RUVBL1-RUVBL2 parts shown in ribbon representation. e Isosurface representations of cryo-EM reconstruction of SRCAP complex with the EM density at 8.5 Å resolution shown in low threshold. f Electrostatic potential surfaces of ARP6 are shown in the context of core complex structure. The surface colored in blue reveals potential DNA-binding surface

Although the functions of Swr1-C/SRCAP-C have been extensively studied, the structural studies have been limited by low-resolution EM structures of yeast Swr1 complex 5 , 7 , 8 . Recently, cryo-EM structure of human INO80 complex bound to mononucleosome was reported 9 . However, the mechanisms for SRCAP-C assembly and histone exchange remain largely unknown.

We here proposed mechanism for SRCAP complex-mediated histone exchange. The nucleosome is primarily recognized by SRCAP-C on two binding surfaces. The ATPase domain of SRCAP protein binds DNA for translocation. The nucleosome is further stabilized by ARP6 and possibly other SRCAP-C components. The ATPase domain serves as a motor to push DNA to the ARP6, leading to a partial unwraping of nucleosomal DNA and loss of contacts between H2A/H2B dimers within the histone octamer, and thus facilitating histone exchange. YL1 may specifically bind to H2A.Z/H2B dimer and be involved in the histone exchange process.

Intriguingly, INO80 complex possesses histone exchange activity and nucleosome sliding activity, but Swr1-C/SRCAP-C has no sliding activity. A possible explanation is that ARP6 and some other SRCAP-C-specific components stably bind nucleosome and prevent translocation of DNA with respective to histone octamer. Thus, although the ATPase domain can translocate nucleosomal DNA, the relative position of DNA and histone octamer is fixed to avoid nucleosome sliding. The complex-specific components, ARP5-IES6 in INO80 and ARP6-YL1 in SRCAP, may contribute to this functional difference.

In summary, our study reveals the structure of SRCAP complex. We proposed a model for nucleosome recognition and histone exchange mediated by SRCAP complex. The structure provides a framework for further investigating SRCAP complex assembly and regulatory mechanism of histone exchange.

The electron density maps and corresponding atomic coordinates have been deposited in the Protein Data Bank ( http://www.rcsb.org/pdb ) with code: 6IGM, and EMDB ( http://www.ebi.ac.uk/pdbe/emdb/ ) with codes: EMD-9668 (core complex) and EMD-9669 (full complex).

We thank Center for Biological Imaging of Institute of Biophysics of Chinese Academy of Sciences and National Center for Protein Science Shanghai for the support on cryo-EM data collection and data analyses. We thank staff members at Biomedical Core Facility, Fudan University for their help with mass spectrometry analyses. This work was supported by the Ministry of Science and Technology of China (2016YFA0500700), the National Natural Science Foundation of China (31830107, U1432242, 31425008), the National Program for support of Top-Notch Young Professionals (Y.X.), and the Strategic Priority Research Program of the Chinese Academy of Sciences (XDB08000000).

Y.F., Y.T. and Y.X. designed the experiments. Y.F. purified the proteins. Y.T. prepared the cryo-EM sample, collected the data and determined the structure. Z.W. helped with structure determination. Y.F., Y.T. and Y.X. analyzed the data and wrote the manuscript. Y.X. supervised the project.

Supplementary information accompanies this paper at 10.1038/s41422-018-0102-y.

Articles from Cell Research are provided here courtesy of Nature Publishing Group
