---
pmid: '28241135'
title: An atlas of human long non-coding RNAs with accurate 5' ends.
authors:
- Hon CC
- Ramilowski JA
- Harshbarger J
- Bertin N
- Rackham OJ
- Gough J
- Denisenko E
- Schmeier S
- Poulsen TM
- Severin J
- Lizio M
- Kawaji H
- Kasukawa T
- Itoh M
- Burroughs AM
- Noma S
- Djebali S
- Alam T
- Medvedeva YA
- Testa AC
- Lipovich L
- Yip CW
- Abugessaisa I
- Mendez M
- Hasegawa A
- Tang D
- Lassmann T
- Heutink P
- Babina M
- Wells CA
- Kojima S
- Nakamura Y
- Suzuki H
- Daub CO
- de Hoon MJ
- Arner E
- Hayashizaki Y
- Carninci P
- Forrest AR
journal: Nature
year: '2017'
full_text_available: false
pmcid: PMC6857182
doi: 10.1038/nature21374
---

# An atlas of human long non-coding RNAs with accurate 5' ends.
**Authors:** Hon CC, Ramilowski JA, Harshbarger J, Bertin N, Rackham OJ, Gough J, Denisenko E, Schmeier S, Poulsen TM, Severin J, Lizio M, Kawaji H, Kasukawa T, Itoh M, Burroughs AM, Noma S, Djebali S, Alam T, Medvedeva YA, Testa AC, Lipovich L, Yip CW, Abugessaisa I, Mendez M, Hasegawa A, Tang D, Lassmann T, Heutink P, Babina M, Wells CA, Kojima S, Nakamura Y, Suzuki H, Daub CO, de Hoon MJ, Arner E, Hayashizaki Y, Carninci P, Forrest AR
**Journal:** Nature (2017)
**DOI:** [10.1038/nature21374](https://doi.org/10.1038/nature21374)
**PMC:** [PMC6857182](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6857182/)

## Abstract

1. Nature. 2017 Mar 9;543(7644):199-204. doi: 10.1038/nature21374. Epub 2017 Mar
1.

An atlas of human long non-coding RNAs with accurate 5' ends.

Hon CC(1), Ramilowski JA(1)(2), Harshbarger J(1)(2), Bertin N(2)(3), Rackham 
OJ(4)(5), Gough J(4), Denisenko E(6), Schmeier S(6), Poulsen TM(7), Severin 
J(1)(2), Lizio M(1)(2), Kawaji H(1)(2)(8), Kasukawa T(1), Itoh M(1)(2)(8), 
Burroughs AM(1)(2)(9), Noma S(1)(2), Djebali S(10)(11), Alam T(12), Medvedeva 
YA(13)(14), Testa AC(15), Lipovich L(16)(17), Yip CW(1), Abugessaisa I(1), 
Mendez M(1)(2), Hasegawa A(1)(2), Tang D(1)(2)(18), Lassmann T(1)(2)(18), 
Heutink P(1)(19), Babina M(20), Wells CA(21)(22), Kojima S(23), Nakamura 
Y(24)(25), Suzuki H(1)(2), Daub CO(1)(2)(26), de Hoon MJ(1)(2), Arner E(1)(2), 
Hayashizaki Y(2)(8), Carninci P(1)(2), Forrest AR(1)(2)(15).

Author information:
(1)RIKEN Center for Life Science Technologies (Division of Genomic 
Technologies), 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, 230-0045 Japan.
(2)RIKEN Omics Science Center (OSC), 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 
230-0045, Japan.
(3)Cancer Science Institute of Singapore, National University of Singapore, 
Centre for Translational Medicine, 14 Medical Drive, #12-01, Singapore 117599, 
Singapore.
(4)University of Bristol, Department of Computer Science, Life Sciences 
building, 24 Tyndall Avenue, Bristol BS8 1TQ, UK.
(5)Program in Cardiovascular and Metabolic Disorders, Duke-NUS Medical School, 8 
College Road, 169857 Singapore.
(6)Institute of Natural and Mathematical Sciences, Massey University Auckland, 
Albany 0632, New Zealand.
(7)Biotechnology Research Institute for Drug Discovery (BRD), National Institute 
of Advanced Industrial Science and Technology (AIST), Tsukuba Central 2, 1-1-1 
Umezono, Tsukuba, Ibaraki, 305-8568, Japan.
(8)RIKEN Preventive Medicine and Diagnosis Innovation Program, 2-1 Hirosawa, 
Wako, Saitama, 351-0198, Japan.
(9)National Center for Biotechnology Information, National Library of Medicine, 
National Institutes of Health, Bethesda, Maryland 20894, USA.
(10)Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and 
Technology, Dr. Aiguader 88, Barcelona 08003, Spain.
(11)Universitat Pompeu Fabra (UPF), Barcelona Biomedical Research Park (PRBB), 
Dr Aiguader 88, Barcelona 08003, Spain.
(12)Computational Bioscience Research Center; Computer, Electrical and 
Mathematical Sciences and Engineering Division, King Abdullah University of 
Science and Technology (KAUST), Thuwal 23955-6900, Saudi Arabia.
(13)Institute of Bioengineering, Research Center of Biotechnology RAS, Moscow 
119071, Russia.
(14)Vavilov Institute of General Genetic, RAS, Moscow 119991, Russia.
(15)Harry Perkins Institute of Medical Research, QEII Medical Centre and Centre 
for Medical Research, the University of Western Australia, Nedlands 6009, 
Western Australia, Australia.
(16)Center for Molecular Medicine and Genetics, Wayne State University, Detroit, 
Michigan 48201, USA.
(17)Department of Neurology, School of Medicine, Wayne State University, 
Detroit, Michigan 48201, USA.
(18)Telethon Kids Institute, The University of Western Australia, 100 Roberts 
Road, Subiaco, Subiaco, 6008, Western Australia, Australia.
(19)German Center for Neurodegenerative Diseases (DZNE), D-72076 Tübingen, 
Germany.
(20)Department of Dermatology and Allergy, Charité Universitätsmedizin Berlin, 
10117 Berlin, Germany.
(21)Australian Institute for Bioengineering and Nanotechnology, The University 
of Queensland, Brisbane 4072, Australia.
(22)Faculty of Medicine, Department of Anatomy and Neuroscience, The University 
of Melbourne, 3010, Australia.
(23)RIKEN CLST (Division of Bio-Function Dynamics Imaging), Wako, Saitama 
351-0198, Japan.
(24)Cell Engineering Division, RIKEN BioResource Center, Tsukuba, Ibaraki 
305-0074, Japan.
(25)Faculty of Medicine, University of Tsukuba, Tsukuba, Ibaraki 305-8577, 
Japan.
(26)Department of Biosciences and Nutrition, Karolinska Institutet, 141 83 
Huddinge, Sweden.

Long non-coding RNAs (lncRNAs) are largely heterogeneous and functionally 
uncharacterized. Here, using FANTOM5 cap analysis of gene expression (CAGE) 
data, we integrate multiple transcript collections to generate a comprehensive 
atlas of 27,919 human lncRNA genes with high-confidence 5' ends and expression 
profiles across 1,829 samples from the major human primary cell types and 
tissues. Genomic and epigenomic classification of these lncRNAs reveals that 
most intergenic lncRNAs originate from enhancers rather than from promoters. 
Incorporating genetic and expression data, we show that lncRNAs overlapping 
trait-associated single nucleotide polymorphisms are specifically expressed in 
cell types relevant to the traits, implicating these lncRNAs in multiple 
diseases. We further demonstrate that lncRNAs overlapping expression 
quantitative trait loci (eQTL)-associated single nucleotide polymorphisms of 
messenger RNAs are co-expressed with the corresponding messenger RNAs, 
suggesting their potential roles in transcriptional regulation. Combining these 
findings with conservation data, we identify 19,175 potentially functional 
lncRNAs in the human genome.

DOI: 10.1038/nature21374
PMCID: PMC6857182
PMID: 28241135 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing financial 
interests. Readers are welcome to comment on the online version of the paper.
