---
pmid: '23629966'
title: Deacetylation of p53 induces autophagy by suppressing Bmf expression.
authors:
- Contreras AU
- Mebratu Y
- Delgado M
- Montano G
- Hu CA
- Ryter SW
- Choi AM
- Lin Y
- Xiang J
- Chand H
- Tesfaigzi Y
journal: J Cell Biol
year: '2013'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3639396
doi: 10.1083/jcb.201205064
---

# Deacetylation of p53 induces autophagy by suppressing Bmf expression.
**Authors:** Contreras AU, Mebratu Y, Delgado M, Montano G, Hu CA, Ryter SW, Choi AM, Lin Y, Xiang J, Chand H, Tesfaigzi Y
**Journal:** J Cell Biol (2013)
**DOI:** [10.1083/jcb.201205064](https://doi.org/10.1083/jcb.201205064)
**PMC:** [PMC3639396](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3639396/)

## Abstract

1. J Cell Biol. 2013 Apr 29;201(3):427-37. doi: 10.1083/jcb.201205064.

Deacetylation of p53 induces autophagy by suppressing Bmf expression.

Contreras AU(1), Mebratu Y, Delgado M, Montano G, Hu CA, Ryter SW, Choi AM, Lin 
Y, Xiang J, Chand H, Tesfaigzi Y.

Author information:
(1)Chronic Obstructive Pulmonary Disease Program, Lovelace Respiratory Research 
Institute, Albuquerque, NM 87108, USA.

Comment in
    Cell Cycle. 2013 Nov 15;12(22):3453-4. doi: 10.4161/cc.26696.

Interferon γ (IFN-γ)-induced cell death is mediated by the BH3-only domain 
protein, Bik, in a p53-independent manner. However, the effect of IFN-γ on p53 
and how this affects autophagy have not been reported. The present study 
demonstrates that IFN-γ down-regulated expression of the BH3 domain-only 
protein, Bmf, in human and mouse airway epithelial cells in a p53-dependent 
manner. p53 also suppressed Bmf expression in response to other cell 
death-stimulating agents, including ultraviolet radiation and histone 
deacetylase inhibitors. IFN-γ did not affect Bmf messenger RNA half-life but 
increased nuclear p53 levels and the interaction of p53 with the Bmf promoter. 
IFN-γ-induced interaction of HDAC1 and p53 resulted in the deacetylation of p53 
and suppression of Bmf expression independent of p53's proline-rich domain. 
Suppression of Bmf facilitated IFN-γ-induced autophagy by reducing the 
interaction of Beclin-1 and Bcl-2. Furthermore, autophagy was prominent in 
cultured bmf(-/-) but not in bmf(+/+) cells. Collectively, these observations 
show that deacetylation of p53 suppresses Bmf expression and facilitates 
autophagy.

DOI: 10.1083/jcb.201205064
PMCID: PMC3639396
PMID: 23629966 [Indexed for MEDLINE]

## Full Text

IntroductionAs a pleiotropic cytokine, IFN-γ mediates many of its antiviral and anticancer properties directly by activating STAT1 (Hu and Ivashkiv, 2009). The anticancer property of IFN-γ stems from its ability to increase the susceptibility of many cell types, including melanoma and colorectal cells, to undergo apoptosis in response to cytotoxic chemotherapies (Borden et al., 2007). IFN-γ induces apoptosis in not only carcinoma (Ossina et al., 1997; Ruiz-Ruiz et al., 2000) but also primary cells (Trautmann et al., 2000; Tesfaigzi et al., 2002a) by activating various pathways in a cell type–specific manner. In asthma, IFN-γ reduces epithelial cell hyperplasia (Shi et al., 2002) by inducing expression of the BH3-only protein, Bik (Mebratu et al., 2008), and by translocating Bax to the ER (Stout et al., 2007). We and others have shown that IFN-γ–induced cell death is p53 independent (Deiss et al., 1995; Ossina et al., 1997; Mebratu et al., 2008). However, the effect of IFN-γ on p53 and the resulting cellular conditions have not been reported.The Bcl-2 family of proteins is characterized by the Bcl-2 homology (BH) domains. The prosurvival proteins, Bcl-2, Bcl-xL, and Mcl-1, have four BH domains (BH1–4). The first group of proapoptotic proteins, Bax, Bak, and Bok, is characterized by three BH domains (BH1–3), whereas the second group contains only the BH3 domain and, therefore, is designated as the BH3-only group of proteins (Strasser, 2005). One of the BH3-only proteins, Bmf, was first reported to cause cell death upon loss of cell attachment (anoikis) by being released from dynein light chain and inhibiting the function of the prosurvival Bcl-2 (Puthalakath et al., 2001). Although Bmf was dispensable for anoikis in certain cell types (Labi et al., 2008), it appears to play an essential role for anoikis in others (Hausmann et al., 2011). In addition, anoikis in human endothelial cells seem to involve Bmf (Schmelzle et al., 2007), whereas those isolated from mouse (Labi et al., 2008) do not. Because we have observed that airway epithelial cells detach from the basement membrane during IFN-γ–induced resolution of airway epithelial cells (Tesfaigzi, 2006), we investigated the effect of IFN-γ on Bmf expression in airway epithelial cells. Bmf mRNA is transcribed from three start sites, but only two isoforms, BmfCUG and BmfS, have been detected in murine thymus, whereas the third isoform, BmfL, has not been detected so far in any tissue (Grespi et al., 2010). In humans, BmfL is likely not relevant because of a frame shift in the start site (Grespi et al., 2010).Histone deacetylase (HDAC) inhibitors (HDACis) induce Bmf expression in a broad range of cancer cells by hyperacetylating histone tails (H3 and H4) at the Bmf promoter and facilitating transcription (Zhang et al., 2006a). Loss of Bmf protein renders lymphocytes resistant to glucocorticoid- or HDACi-induced cell death (Labi et al., 2008). HDACs were initially known to target histones; however, it is now clear that many other nonhistone proteins are substrates for various HDACs. The first nonhistone protein known to be regulated by acetylation and deacetylation was p53 (Gu and Roeder, 1997; Tang et al., 2006). Acetylation of human p53 at lysine 382 or murine p53 at lysine 379 and acetylation in general have been shown to be important for p53 stability, sequence-specific DNA-binding activities, and recruitment of transcriptional activators (Itahana et al., 2009). In the present study, we found that IFN-γ causes deacetylation and nuclear accumulation of p53 to promote its interaction with the Bmf promoter and suppress Bmf expression and thereby facilitate autophagy.

Introduction

As a pleiotropic cytokine, IFN-γ mediates many of its antiviral and anticancer properties directly by activating STAT1 (Hu and Ivashkiv, 2009). The anticancer property of IFN-γ stems from its ability to increase the susceptibility of many cell types, including melanoma and colorectal cells, to undergo apoptosis in response to cytotoxic chemotherapies (Borden et al., 2007). IFN-γ induces apoptosis in not only carcinoma (Ossina et al., 1997; Ruiz-Ruiz et al., 2000) but also primary cells (Trautmann et al., 2000; Tesfaigzi et al., 2002a) by activating various pathways in a cell type–specific manner. In asthma, IFN-γ reduces epithelial cell hyperplasia (Shi et al., 2002) by inducing expression of the BH3-only protein, Bik (Mebratu et al., 2008), and by translocating Bax to the ER (Stout et al., 2007). We and others have shown that IFN-γ–induced cell death is p53 independent (Deiss et al., 1995; Ossina et al., 1997; Mebratu et al., 2008). However, the effect of IFN-γ on p53 and the resulting cellular conditions have not been reported.

The Bcl-2 family of proteins is characterized by the Bcl-2 homology (BH) domains. The prosurvival proteins, Bcl-2, Bcl-xL, and Mcl-1, have four BH domains (BH1–4). The first group of proapoptotic proteins, Bax, Bak, and Bok, is characterized by three BH domains (BH1–3), whereas the second group contains only the BH3 domain and, therefore, is designated as the BH3-only group of proteins (Strasser, 2005). One of the BH3-only proteins, Bmf, was first reported to cause cell death upon loss of cell attachment (anoikis) by being released from dynein light chain and inhibiting the function of the prosurvival Bcl-2 (Puthalakath et al., 2001). Although Bmf was dispensable for anoikis in certain cell types (Labi et al., 2008), it appears to play an essential role for anoikis in others (Hausmann et al., 2011). In addition, anoikis in human endothelial cells seem to involve Bmf (Schmelzle et al., 2007), whereas those isolated from mouse (Labi et al., 2008) do not. Because we have observed that airway epithelial cells detach from the basement membrane during IFN-γ–induced resolution of airway epithelial cells (Tesfaigzi, 2006), we investigated the effect of IFN-γ on Bmf expression in airway epithelial cells. Bmf mRNA is transcribed from three start sites, but only two isoforms, BmfCUG and BmfS, have been detected in murine thymus, whereas the third isoform, BmfL, has not been detected so far in any tissue (Grespi et al., 2010). In humans, BmfL is likely not relevant because of a frame shift in the start site (Grespi et al., 2010).

Histone deacetylase (HDAC) inhibitors (HDACis) induce Bmf expression in a broad range of cancer cells by hyperacetylating histone tails (H3 and H4) at the Bmf promoter and facilitating transcription (Zhang et al., 2006a). Loss of Bmf protein renders lymphocytes resistant to glucocorticoid- or HDACi-induced cell death (Labi et al., 2008). HDACs were initially known to target histones; however, it is now clear that many other nonhistone proteins are substrates for various HDACs. The first nonhistone protein known to be regulated by acetylation and deacetylation was p53 (Gu and Roeder, 1997; Tang et al., 2006). Acetylation of human p53 at lysine 382 or murine p53 at lysine 379 and acetylation in general have been shown to be important for p53 stability, sequence-specific DNA-binding activities, and recruitment of transcriptional activators (Itahana et al., 2009). In the present study, we found that IFN-γ causes deacetylation and nuclear accumulation of p53 to promote its interaction with the Bmf promoter and suppress Bmf expression and thereby facilitate autophagy.

ResultsOur previous studies demonstrate that physiologically relevant levels of IFN-γ induce cell death in proliferating primary human airway epithelial cells (HAECs), murine airway epithelial cells (MAECs), and in AALEB cells, a cell line derived from HAECs (Tesfaigzi et al., 2002b; Stout et al., 2007) by STAT1-dependent Bik expression (Mebratu et al., 2008). Although screening for the effect of IFN-γ on the expression of BH3-only proteins, we found that IFN-γ reduced Bmf mRNA levels in AALEB cells by fivefold (Fig. 1 A), and this reduction was replicated in both HAECs (Fig. 1 B) and MAECs (Fig. 1 C). Treatment of bik−/− (Fig. 1 D) or STAT1−/− (Fig. 1 E) with IFN-γ also reduced Bmf mRNA levels, suggesting that Bmf down-regulation was not mediated by STAT1 or Bik. IFN-γ reduced all three isoforms of Bmf mRNAs, BmfL, BmfCUG, and BmfS (Fig. 1 F). Similar to a previous study for the thymus (Labi et al., 2008), we also detected two Bmf proteins in the murine lung tissue representing BmfCUG (25 kD) and BmfS (20 kD), which were absent in tissues from bmf−/− mice (Fig. 1 G); however, in contrast to the thymus, BmfCUG was the predominantly expressed isoform in mouse lung tissue (Fig. 1, G and I). Consistent with what was observed for the Bmf mRNA isoforms, IFN-γ also reduced protein levels for both BmfCUG and BmfS (Fig. 1 H). Because we did not expect a proapoptotic protein to be reduced during IFN-γ–induced cell death, we generated adenoviral expression vectors to investigate whether one of the Bmf isoforms may inhibit cell death in airway epithelial cells. We introduced a Kozak consensus sequence upstream of the BmfCUG transcriptional start site to allow expression. Adenoviral expression of BmfCUG and BmfS showed that BmfCUG appeared to be processed to BmfS, and the expressed proteins were of the same size as the isoforms induced in MAECs by the HDACi trichostatin A (TSA). However, both BmfCUG and BmfS killed ∼50% of cells within 18 h after infection compared with adenovirus (Ad)-GFP–infected controls (Fig. 1 I), demonstrating that expression of both isoforms is proapoptotic.Figure 1.IFN-γ down-regulates expression of proapoptotic Bmf isoforms. (A–E) Bmf mRNA levels in AALEB cells (A), HAECs (B), wild-type MAECs (C), Bik−/− MAECs (D), or STAT1−/− MAECs (E) 48 h after treatment with 50 ng/ml IFN-γ as quantified by qRT-PCR. The relative standard curve method was used for analysis of unknown samples, and data are presented as fold change after averaging the ΔCt values for the nontreated (NT) samples. (F) BmfS, BmfCUG, and BmfL mRNA levels in IFN-γ–treated or untreated MAECs as analyzed by RT-PCR with GAPDH levels as controls. (G) Western blot of protein lysates extracted from homogenized lung or thymus tissue from bmf−/− and bmf+/+ mice and analyzed for Bmf expression. (H) Bmf protein levels in protein lysates prepared from IFN-γ–treated or nontreated MAECs. (I) Bmf protein levels in protein lysates from MAECs treated with 300 nM TSA, infected with 100 MOI of Ad-BmfS, Ad-BmfCUG, or nontreated controls, and the percentage of viable cells. Cells were harvested and quantified using trypan blue exclusion. Data presented are means ± SEM for three independent experiments. *, P < 0.05.To investigate this paradox that the proapoptotic Bmf was down-regulated by the cell death–inducing IFN-γ, we explored the Bmf mRNA levels in epithelial cancer cell lines, as previous studies had described Bmf expression in various cancers (Puthalakath et al., 2001; Schmelzle et al., 2007). We noticed that the p53-deficient cell lines, SOAS-2 and Calu-6 cells, showed significantly higher Bmf mRNA levels compared with the p53-sufficient cells, A549 and AALEB cells (Fig. 2 A). The role of p53 in affecting Bmf expression was validated by expressing p53 in Calu-6 cells using an adenoviral overexpression system (Fig. 2 B). Interestingly, p53 expression alone was not sufficient, but additional treatment with 10 mJ UV radiation, which is known to activate p53 (Zhang and Xiong, 2001), was necessary for reducing Bmf levels, whereas the same treatment significantly increased Bmf mRNA levels in cells infected with adenoviral GFP as a control (Fig. 2 C). Identical results were obtained when p53 was expressed in Calu-6 cells using a lentiviral expression system after treatment with UV radiation (unpublished data). Exposure of MAECs to 10 mJ UV radiation increased p53 levels (Fig. 2 C) and reduced Bmf mRNA levels to 50% of nontreated controls in both MAECs (Fig. 2 C) and AALEB cells (Fig. 2 D) 6 h after treatment.Figure 2.p53 mediates IFN-γ–induced Bmf suppression. (A) Bmf expression levels in p53-sufficient and -deficient cell lines evaluated by qRT-PCR. Bmf mRNA levels relative to p53-sufficient A549 are shown for AALEB cells and the p53-deficient SAOS-2 and Calu-6 cells. Data presented are representative of four independent experiments. (B) Western blot analysis of p53 in Calu-6 cells infected with an adenoviral expression vector for p53 and the relative Bmf mRNA levels quantified by qRT-PCR in these cell lines when exposed to 10 mJ UV radiation compared with the respective nontreated controls. (C) p53 protein levels in MAEC 6 h after exposure to 10 mJ UV radiation and Bmf mRNA levels quantified by qRT-PCR in MAECs exposed to10 mJ UV and nontreated (NT) controls. (D) Bmf mRNA levels in AALEB cells 6, 12, and 24 h after exposure to 10 mJ UV radiation compared with nontreated controls. Data presented are means ± SEM for three independent experiments. *, P < 0.05.These findings suggested that suppression of Bmf expression by IFN-γ may be mediated by p53. Therefore, we investigated the effect of IFN-γ on primary MAECs from p53−/− and p53+/+ mice and found that Bmf mRNA levels were significantly up-regulated by IFN-γ in p53−/− MAECs (Fig. 3 A), and BmfCUG and BmfS proteins were detected in p53−/− MAECs treated with IFN-γ (Fig. 3 B). Similarly, suppression of p53 levels in AALEB cells using shRNA resulted in IFN-γ failing to suppress Bmf mRNA levels (unpublished data). Having established that p53 mediates IFN-γ–induced suppression of Bmf expression, we investigated the possibility that p53 may be affecting Bmf mRNA stability. In general, 3′UTRs have conserved sequences that regulate mRNA stability (Stoecklin and Anderson, 2007), and Bmf mRNA has an unusually long 4-kb 3′UTR. AALEB cells treated with and without IFN-γ were harvested at 0, 0.5, 1, 2, and 4 h after blocking transcription with DRB (5,6-dichloro-1-β-d-ribobenzimidazole). On average, the half-life of Bmf mRNA in nontreated p53-sufficient A549 and AALEB cells was 1.94 and 2.43 h and in the p53-deficient Calu-6 and SAOS-2 cells, was 2.36 and 1.73 h, respectively. Bmf mRNA half-life in IFN-γ–treated p53-sufficient and p53-deficient cells was 1.5 and 1.8 h, respectively. The similarity of the mRNA half-lives among cells with and without p53 and IFN-γ treatment suggested that p53 does not affect Bmf mRNA stability.Figure 3.p53 suppresses IFN-γ– and HDACi-induced Bmf expression. (A) Bmf mRNA in IFN-γ–treated and nontreated (NT) p53−/− MAECs. (B) Bmf protein levels in protein lysates prepared from p53−/− MAECs treated with nothing or IFN-γ for 48 h. (C) Bmf mRNA levels in AALEB cells treated with 5 mM sodium butyrate, 300 nM TSA, and 5 µM MS-275 for 18 h compared with nontreated controls. (D) Bmf mRNA expression in p53−/− and p53+/+ MAECs treated with 300 nM TSA for 18 h. Error bars indicate ±SEM (n = 3 independent experiments). *, P < 0.05; statistically significant difference from controls.Inhibition of HDACs (HDACis) with sodium butyrate, TSA, or MS-275 significantly increased Bmf mRNA in AALEB cells (Fig. 3 C) and BmfCUG and BmfS protein levels in MAECs (Fig. 1 I). Although Bmf mRNA levels were increased by TSA in both p53−/− and p53+/+ MAECs, Bmf mRNA expression was twofold higher in p53−/− compared with p53+/+ MAECs (Fig. 3 D), suggesting that the inhibitory effect of p53 was still present in cells treated with HDACis.A previous study has established that overexpression of HDAC1 suppresses Bmf expression by inhibiting the promoter activity (Zhang et al., 2006a). Therefore, we next evaluated whether IFN-γ affects the interaction of p53 with the Bmf promoter. The Bmf promoter region at −97 is the region where histone hyperacetylation occurs in cancer cell lines in response to HDACi treatment to increase Bmf transcription (Zhang et al., 2006a). We found by chromatin immunoprecipitation (ChIP) that region −97, but not −560, of the Bmf promoter interacted with HDAC1 in p53−/− but not in p53+/+ HCT116 cells, and IFN-γ had no effect on this interaction (Fig. 4 A). In AALEB cells, p53 interaction with the Bmf promoter was absent in nontreated controls but was present at region −97 and not −560 when cells were treated with IFN-γ (Fig. 4 B). These findings suggest that HDAC1 requires the absence of p53 for its interaction with the Bmf promoter and that IFN-γ enhances the interaction of p53 with the Bmf promoter to suppress transcription.Figure 4.p53 is enriched in the nucleus and interacts with the Bmf promoter. (A) ChIP assays on p53+/+ or p53−/− HCT116 cells using a polyclonal antibody to HDAC1 or rabbit IgG1 as a control. DNA identification was performed using PCR with primers specific for Bmf promoter regions at −97 or −560 regions. (B) ChIP assay on AALEB cells nontreated (NT) or treated with IFN-γ for 48 h using a monoclonal antibody to p53 or mouse IgG1 as a control using primers specific for Bmf promoter regions at −97 or −560 regions. Densitometry of PCR products from six independent experiments for DNA pulled down with p53 normalized to IgG1 and IFN-γ–treated cells normalized to the nontreated values. IN, input. (C) Increased p53 protein levels in the nuclear fractions of AALEB cells 48 h after treatment with IFN-γ compared with nontreated controls. Nuclear and cytosolic extracts were analyzed for p53, lamin, and actin. Panel is representative of three independent experiments. (D) Representative photomicrographs of nontreated and IFN-γ–treated AALEB cells immunostained with the anti-p53 antibody and quantification of p53-positive nuclei (n = 4/group). Bar, 15 µm. (E) Bmf mRNA levels in MAECs isolated from p53ΔP, p53AXXA, and p53wt littermates 48 h after IFN-γ treatment compared with nontreated MAECs (n = 3 independent experiments). WT, wild type. Error bars indicate ±SEM. *, P < 0.05.Because the nuclear or cytosolic localization defines p53 function (Lee and Gu, 2010), we assessed p53 localization in IFN-γ–treated cells and found that compared with nontreated controls, IFN-γ increased nuclear p53 levels, whereas cytosolic p53 levels remained low and unchanged (Fig. 4 C). Immunofluorescence also demonstrated that the percentage of cells with nuclear p53 was significantly increased by IFN-γ treatment (Fig. 4 D).In humans, the proline-rich domain (PRD) of p53 is defined by residues 58–98, which contains 15 prolines and five repeats of the amino acid motif PXXP (in which P designates proline and X designates any amino acid). The histone acetyltransferase p300 binds to PXXP-containing peptides derived from the proline repeat domain, and the PXXP motif in p53 is required for p53 acetylation by the transcription coactivator p300 (Dornan et al., 2003). Therefore, we investigated whether the PRD region plays a role in the IFN-γ–induced Bmf suppression. In mice, this PRD consists of two PXXP motifs, and we obtained mice with a deletion of the PRD (p53ΔP) or lacking the four critical proline residues at loci 79, 82, 84, and 87 that make up the tandem PXXP sites (p53AXXA; Toledo et al., 2006, 2007). MAECs isolated from p53ΔP, p53AXXA, and wild-type littermates showed a reduction in Bmf levels when treated with IFN-γ similar to that observed in wild-type MAECs (Fig. 4 E).p53 interacts with HDAC1 as part of a deacetylation complex (Luo et al., 2000, 2004), and our experiments showed that HDAC1 interacts with the Bmf promoter, and IFN-γ drives the interaction of p53 with the Bmf promoter. Therefore, we investigated the interaction of p53 and HDAC1 in IFN-γ–treated cells using immunoprecipitation assays and found that IFN-γ increased HDAC1 levels in pull-down products using anti-p53 antibodies (Fig. 5 A), demonstrating that IFN-γ enhanced the p53–HDAC1 interaction. Therefore, we assessed the acetylation state of p53 after IFN-γ treatment and found significantly reduced acetylated p53 levels in IFN-γ–treated compared with nontreated cells (Fig. 5 A). An antibody specific to Lys382 was selected for detection because this is the site mostly acetylated within p53 (Gu and Roeder, 1997). Deacetylation of p53 by IFN-γ was also observed in IFN-γ–treated MAECs (Fig. 5 B).Figure 5.IFN-γ causes HDAC1–p53 interaction and deacetylates p53. (A) HDAC1 interacts with p53. Nuclear lysates were prepared from AALEB cells after treatment with IFN-γ for 48 h and were immunoprecipitated (IP) using a p53-specific monoclonal antibody. The nuclear lysates (input) and immunoprecipitates were resolved by SDS-PAGE and analyzed using Western blotting with antibodies to HDAC1, total p53, and acetyl-p53. Panel is representative of three independent experiments. (B) Nuclear extracts prepared from IFN-γ–treated and nontreated MAECs and probed for total p53, acetyl-p53, and lamin. (C) Bmf mRNA levels in IFN-γ–, TSA-, or IFN-γ + TSA–treated AALEB cells compared with nontreated (NT) controls. (D) Representative photomicrographs IFN-γ–, TSA-, or IFN-γ/TSA-treated or nontreated AALEB cells immunostained for p53 (shown in red) and acetyl-p53 (shown in green). Nuclei were stained with DAPI (shown in blue), and merged images are shown in the right-most column. Bars, 10 µm. (E) Quantification of the percentage of cells positive for p53 and acetyl-p53 (Ac-p53) in wild-type (WT) and AXXA MEFs either nontreated or treated with IFN-γ, TSA, or IFN-γ/TSA. Error bars show group means ± SEM (n = 4/group). *, P < 0.05; **, P < 0.01; ***, P < 0.001.To determine whether prevention of p53 deacetylation would abrogate the down-regulation of Bmf, we treated airway epithelial cells with IFN-γ for 24 h followed by TSA in the presence or absence of IFN-γ for an additional 24 h. Bmf was up-regulated in the TSA + IFN-γ–treated compared with IFN-γ–only treated control samples, suggesting that TSA prevented the HDAC1 and p53 interaction (Fig. 5 C), further supporting the idea that acetylation of p53 affects Bmf expression. Increased accumulation of nuclear p53 and enhanced deacetylation of p53 in IFN-γ–treated cells was verified by double staining of nontreated, IFN-γ–, TSA-, and TSA + IFN-γ–treated AALEB cells (Fig. 5 D). These findings were further validated in immunostained wild-type and AXXA mutant mouse embryonic fibroblasts (MEFs), and quantification showed that the percentage of p53-positive cells was increased, whereas the percentage of acetyl-p53–positive cells was decreased in IFN-γ–treated compared with nontreated controls (Fig. 5 E). TSA increased the acetylated p53 in control cells and in IFN-γ–stimulated cells in both wild-type and AXXA mutant MEFs.IFN-γ–induced deacetylation of p53 (Fig. 6 A) and suppression of Bmf mRNA (Fig. 6 B) occurs in MAECs as early as 0.5 h after treatment, suggesting that it is a direct effect rather than by other mediators that result from IFN-γ treatment. This deacetylation of p53 was accompanied with the down-regulation of Bmf mRNA. Furthermore, IFN-γ suppressed Bmf in p53-sufficient wild-type colon epithelial cells (HCT116 p53+/+) but increased Bmf mRNA levels in HCT116 p53−/− cells within 0.5 h after treatment (Fig. 6 C).Figure 6.IFN-γ deacetylates p53 but does not affect IFN-γ–induced cell death. (A) Nuclear extracts from MAECs 0.5 h after IFN-γ treatment or nontreated (NT) controls that were probed for total p53, acetyl-p53, and lamin. (B) Bmf mRNA levels in p53+/+ HCT116 cells 0.5 h after IFN-γ treatment or nontreated controls. *, P < 0.05. (C) Bmf mRNA in p53−/− and p53+/+ MAECs treated with IFN-γ for 0.5 h (n = 3 independent experiments). (D) Quantification of viable p53−/− and p53+/+ MAECs treated with IFN-γ for 48 h as assessed by trypan blue exclusion. (E) Quantification of viable AALEB cells transfected with an empty vector plasmid (shRNA control [CTR]) or one expressing p53 shRNA were treated with IFN-γ for 48 h. (F) Bcl-xL and Bcl-2 protein levels in protein lysates prepared from p53−/− and p53+/+ MAECs treated with nothing or IFN-γ for 48 h. Error bars indicate ±SEM.Although IFN-γ increased the levels of the proapoptotic BmfCUG and BmfS in p53−/− cells, cell death and the viability of p53−/− and p53+/+ MAECs remained unchanged (Fig. 6 D). These results were confirmed in IFN-γ–treated AALEB cells, in which p53 was suppressed using shRNA targeting p53 (Fig. 6 E). Because Bcl-2 blocks the proapoptotic function of Bmf (Puthalakath et al., 2001), we tested whether IFN-γ increases expression of antiapoptotic proteins, Bcl-2 and Bcl-xL, and found that IFN-γ increased Bcl-2 and Bcl-xL protein levels in p53−/−, but not in p53+/+, MAECs (Fig. 6 F). Although these experiments suggest that increased expression of these antiapoptotic proteins may suppress apoptosis in IFN-γ–treated p53-deficient MAECs when BmfCUG and BmfS are increased, the role of IFN-γ suppressing the proapoptotic Bmf was still not clear.IFN-γ induces autophagy in HeLa cells (Inbal et al., 2002) and in gastric epithelial cells (Tu et al., 2011). We had also observed that IFN-γ treatment increased both Beclin-1 and conversion of LC3-I to LC3-II in AALEB cells and wild-type MAECs (Fig. 7, A and B). Because the proapoptotic Bmf isoforms were suppressed by IFN-γ, we reasoned that Bmf may have a role in suppressing autophagy and that IFN-γ–induced down-regulation of Bmf may facilitate autophagy. Infection of AALEB cells with Ad-BmfCUG to reconstitute the suppression by IFN-γ or with Ad-GFP as a control showed that increased Bmf expression reduced the levels of IFN-γ–induced Beclin-1 and the processing of LC3-I to LC3-II (Fig. 7 C). Although lung or thymus tissues from bmf−/− showed no evidence of increased Beclin or LC3-II compared with those from bmf-+/+ mice (Fig. 7 D), cultured but nontreated bmf−/− MAECs and MEFs showed increased levels of Beclin-1 and accumulation of LC3-II compared with bmf-+/+ MAECs (Fig. 7 E). The formation of autophagosomes in cultured MAECs and MEFs was also indicated by transfection of pmCherry-LC3, displaying significantly increased punctation in bmf−/− compared with bmf-+/+ MAECs and MEFs (Fig. 7 F). In addition, transmission electron micrographs revealed significantly more autophagic vacuoles per unit area in bmf−/− compared with bmf-+/+ MAECs (Fig. 7 G). Some cells displayed mitochondria surrounded by membranous material that may be evidence for a premitophagic process. Although the number of mitochondria between bmf−/− and bmf-+/+ MAECs showed no difference, there appeared to be morphological differences with more damaged mitochondria being present in bmf−/− compared with bmf-+/+ MAECs. To determine the physiological relevance of reduced Bmf expression causing autophagy, we exposed bmf-+/+ and bmf−/− MEFs to autophagy-promoting conditions and found that starvation (Fig. 7 H) and treatment with pp242 (Fig. 7 I), a mammalian target of rapamycin (mTOR) inhibitor, caused enhanced cell death in bmf−/− compared with bmf-+/+ MEFs. These findings suggest that increased baseline autophagy in bmf−/− MEFs reduces the protective role of autophagy in these cells. Suppression of Bmf in p53−/− HCT116 cells using short hairpin Bmf (shBmf) increased autophagy (Fig. 7 J), and IFN-γ–induced autophagy was increased in p53+/+ but reduced in p53−/− HCT116 cells (Fig. 7 K), suggesting that IFN-γ–induced autophagy is p53 dependent. To investigate the mechanism by which Bmf may be regulating autophagy, we assessed whether Beclin-1–Bcl-2 interaction is affected by the presence or absence of Bmf. Although Bcl-2 levels are higher in bmf-+/+ compared with bmf−/− MEFs the difference is more evident after immunoprecipitation with Beclin-1 antibodies. Beclin-1–Bcl-2 interaction was also reduced when bmf-+/+ MEFs were treated with IFN-γ consistent with the observation that IFN-γ reduces Bmf levels and thereby enhances autophagy (Fig. 7 L). These findings suggest that the Beclin-1–Bcl-2 interaction is stabilized by the presence of Bmf, and loss or even reduced levels of Bmf releases Beclin-1 to cause autophagy. Bmf was not detected in the input but only in the immunoprecipitate from nontreated bmf-+/+ MEFs because it was concentrated by the immunoprecipitation process. Together, these findings suggest that IFN-γ, by deacetylating p53, suppresses Bmf expression to facilitate autophagy.Figure 7.IFN-γ suppresses Bmf expression to induce autophagy. (A and B) IFN-γ increases protein levels of Beclin-1 and LC3B in AALEB cells (A) and MAECs (B) compared with nontreated (NT) controls as detected by Western blot analysis. (C) AALEB cells treated with IFN-γ for 48 h and infected with 50 MOI of Ad-BmfCUG or Ad-GFP. Bmf expression inhibits IFN-γ–induced Beclin-1 and LC3B protein levels. (D) Western blot of lung and thymus tissues from bmf+/+ (wild type [WT]) and bmf−/− (knockout [KO]) mice probed for Beclin-1 and LC3B proteins. (E) Western blot analysis of protein extracts from bmf+/+ (wild type) and bmf−/− (knockout) MAECs and MEFs probed for Beclin-1, LC3B, and β-actin. (F) Representative micrographs of bmf+/+ and bmf−/− MAECs and MEFs expressing mCherry-LC3B and cells with punctuate LC3B were quantified from >50 bmf+/+ and bmf−/− MAECs. (G) Representative electron micrographs of bmf+/+ and bmf−/− MAECs that were cultured in 6-well dishes. Thin sections of cells were analyzed by scanning electron microscopy. The arrowhead denotes an autophagic vesicle, and the arrow denotes a mitochondrion surrounded by a double membrane. Quantification of autophagic vesicles per 100 µm2 in >30 each bmf+/+ and bmf−/− MAECs. (H) Quantification of viable bmf+/+ and bmf−/− MEFs maintained in starvation media for 4 or 24 h relative to cell grown in regular media (n = 3 independent experiments). (I) Quantification of viable bmf+/+ and bmf−/− MEFs 24 h after treatment with the mTOR inhibitor, pp242, at 2.5 µM relative to nontreated control cells (n = 3 independent experiments). (J) Knockdown of Bmf mRNA using shBmf in p53−/− HCT116 cells reduced Bmf mRNA levels. Western blot analysis of shRNA control (shCtr)– and shBmf-transfected p53−/− HCT116 cells probed with Beclin-1, LC3B, and β-actin antibodies. The Western blot is representative of three experiments using three different shBmf constructs. (K) Western blot of IFN-γ–treated p53+/+ and p53−/− HCT116 cells probed with Beclin-1, LC3B, and β-actin antibodies. (L) Immunoprecipitation of protein extracts from IFN-γ–treated and nontreated bmf+/+ and nontreated bmf−/− MEFs using anti–Beclin-1 and Western blot analysis of input and immunoprecipitates (IP) with Bcl-2, Beclin-1, Bmf, and β-actin antibodies. Results are representative of four independent immunoprecipitations. Error bars indicate ±SEM. *, P < 0.05; **, P < 0.01; ***, P < 0.001; ****, P < 0.0001.

Our previous studies demonstrate that physiologically relevant levels of IFN-γ induce cell death in proliferating primary human airway epithelial cells (HAECs), murine airway epithelial cells (MAECs), and in AALEB cells, a cell line derived from HAECs (Tesfaigzi et al., 2002b; Stout et al., 2007) by STAT1-dependent Bik expression (Mebratu et al., 2008). Although screening for the effect of IFN-γ on the expression of BH3-only proteins, we found that IFN-γ reduced Bmf mRNA levels in AALEB cells by fivefold (Fig. 1 A), and this reduction was replicated in both HAECs (Fig. 1 B) and MAECs (Fig. 1 C). Treatment of bik−/− (Fig. 1 D) or STAT1−/− (Fig. 1 E) with IFN-γ also reduced Bmf mRNA levels, suggesting that Bmf down-regulation was not mediated by STAT1 or Bik. IFN-γ reduced all three isoforms of Bmf mRNAs, BmfL, BmfCUG, and BmfS (Fig. 1 F). Similar to a previous study for the thymus (Labi et al., 2008), we also detected two Bmf proteins in the murine lung tissue representing BmfCUG (25 kD) and BmfS (20 kD), which were absent in tissues from bmf−/− mice (Fig. 1 G); however, in contrast to the thymus, BmfCUG was the predominantly expressed isoform in mouse lung tissue (Fig. 1, G and I). Consistent with what was observed for the Bmf mRNA isoforms, IFN-γ also reduced protein levels for both BmfCUG and BmfS (Fig. 1 H). Because we did not expect a proapoptotic protein to be reduced during IFN-γ–induced cell death, we generated adenoviral expression vectors to investigate whether one of the Bmf isoforms may inhibit cell death in airway epithelial cells. We introduced a Kozak consensus sequence upstream of the BmfCUG transcriptional start site to allow expression. Adenoviral expression of BmfCUG and BmfS showed that BmfCUG appeared to be processed to BmfS, and the expressed proteins were of the same size as the isoforms induced in MAECs by the HDACi trichostatin A (TSA). However, both BmfCUG and BmfS killed ∼50% of cells within 18 h after infection compared with adenovirus (Ad)-GFP–infected controls (Fig. 1 I), demonstrating that expression of both isoforms is proapoptotic.

IFN-γ down-regulates expression of proapoptotic Bmf isoforms. (A–E) Bmf mRNA levels in AALEB cells (A), HAECs (B), wild-type MAECs (C), Bik−/− MAECs (D), or STAT1−/− MAECs (E) 48 h after treatment with 50 ng/ml IFN-γ as quantified by qRT-PCR. The relative standard curve method was used for analysis of unknown samples, and data are presented as fold change after averaging the ΔCt values for the nontreated (NT) samples. (F) BmfS, BmfCUG, and BmfL mRNA levels in IFN-γ–treated or untreated MAECs as analyzed by RT-PCR with GAPDH levels as controls. (G) Western blot of protein lysates extracted from homogenized lung or thymus tissue from bmf−/− and bmf+/+ mice and analyzed for Bmf expression. (H) Bmf protein levels in protein lysates prepared from IFN-γ–treated or nontreated MAECs. (I) Bmf protein levels in protein lysates from MAECs treated with 300 nM TSA, infected with 100 MOI of Ad-BmfS, Ad-BmfCUG, or nontreated controls, and the percentage of viable cells. Cells were harvested and quantified using trypan blue exclusion. Data presented are means ± SEM for three independent experiments. *, P < 0.05.

To investigate this paradox that the proapoptotic Bmf was down-regulated by the cell death–inducing IFN-γ, we explored the Bmf mRNA levels in epithelial cancer cell lines, as previous studies had described Bmf expression in various cancers (Puthalakath et al., 2001; Schmelzle et al., 2007). We noticed that the p53-deficient cell lines, SOAS-2 and Calu-6 cells, showed significantly higher Bmf mRNA levels compared with the p53-sufficient cells, A549 and AALEB cells (Fig. 2 A). The role of p53 in affecting Bmf expression was validated by expressing p53 in Calu-6 cells using an adenoviral overexpression system (Fig. 2 B). Interestingly, p53 expression alone was not sufficient, but additional treatment with 10 mJ UV radiation, which is known to activate p53 (Zhang and Xiong, 2001), was necessary for reducing Bmf levels, whereas the same treatment significantly increased Bmf mRNA levels in cells infected with adenoviral GFP as a control (Fig. 2 C). Identical results were obtained when p53 was expressed in Calu-6 cells using a lentiviral expression system after treatment with UV radiation (unpublished data). Exposure of MAECs to 10 mJ UV radiation increased p53 levels (Fig. 2 C) and reduced Bmf mRNA levels to 50% of nontreated controls in both MAECs (Fig. 2 C) and AALEB cells (Fig. 2 D) 6 h after treatment.

p53 mediates IFN-γ–induced Bmf suppression. (A) Bmf expression levels in p53-sufficient and -deficient cell lines evaluated by qRT-PCR. Bmf mRNA levels relative to p53-sufficient A549 are shown for AALEB cells and the p53-deficient SAOS-2 and Calu-6 cells. Data presented are representative of four independent experiments. (B) Western blot analysis of p53 in Calu-6 cells infected with an adenoviral expression vector for p53 and the relative Bmf mRNA levels quantified by qRT-PCR in these cell lines when exposed to 10 mJ UV radiation compared with the respective nontreated controls. (C) p53 protein levels in MAEC 6 h after exposure to 10 mJ UV radiation and Bmf mRNA levels quantified by qRT-PCR in MAECs exposed to10 mJ UV and nontreated (NT) controls. (D) Bmf mRNA levels in AALEB cells 6, 12, and 24 h after exposure to 10 mJ UV radiation compared with nontreated controls. Data presented are means ± SEM for three independent experiments. *, P < 0.05.

These findings suggested that suppression of Bmf expression by IFN-γ may be mediated by p53. Therefore, we investigated the effect of IFN-γ on primary MAECs from p53−/− and p53+/+ mice and found that Bmf mRNA levels were significantly up-regulated by IFN-γ in p53−/− MAECs (Fig. 3 A), and BmfCUG and BmfS proteins were detected in p53−/− MAECs treated with IFN-γ (Fig. 3 B). Similarly, suppression of p53 levels in AALEB cells using shRNA resulted in IFN-γ failing to suppress Bmf mRNA levels (unpublished data). Having established that p53 mediates IFN-γ–induced suppression of Bmf expression, we investigated the possibility that p53 may be affecting Bmf mRNA stability. In general, 3′UTRs have conserved sequences that regulate mRNA stability (Stoecklin and Anderson, 2007), and Bmf mRNA has an unusually long 4-kb 3′UTR. AALEB cells treated with and without IFN-γ were harvested at 0, 0.5, 1, 2, and 4 h after blocking transcription with DRB (5,6-dichloro-1-β-d-ribobenzimidazole). On average, the half-life of Bmf mRNA in nontreated p53-sufficient A549 and AALEB cells was 1.94 and 2.43 h and in the p53-deficient Calu-6 and SAOS-2 cells, was 2.36 and 1.73 h, respectively. Bmf mRNA half-life in IFN-γ–treated p53-sufficient and p53-deficient cells was 1.5 and 1.8 h, respectively. The similarity of the mRNA half-lives among cells with and without p53 and IFN-γ treatment suggested that p53 does not affect Bmf mRNA stability.

p53 suppresses IFN-γ– and HDACi-induced Bmf expression. (A) Bmf mRNA in IFN-γ–treated and nontreated (NT) p53−/− MAECs. (B) Bmf protein levels in protein lysates prepared from p53−/− MAECs treated with nothing or IFN-γ for 48 h. (C) Bmf mRNA levels in AALEB cells treated with 5 mM sodium butyrate, 300 nM TSA, and 5 µM MS-275 for 18 h compared with nontreated controls. (D) Bmf mRNA expression in p53−/− and p53+/+ MAECs treated with 300 nM TSA for 18 h. Error bars indicate ±SEM (n = 3 independent experiments). *, P < 0.05; statistically significant difference from controls.

Inhibition of HDACs (HDACis) with sodium butyrate, TSA, or MS-275 significantly increased Bmf mRNA in AALEB cells (Fig. 3 C) and BmfCUG and BmfS protein levels in MAECs (Fig. 1 I). Although Bmf mRNA levels were increased by TSA in both p53−/− and p53+/+ MAECs, Bmf mRNA expression was twofold higher in p53−/− compared with p53+/+ MAECs (Fig. 3 D), suggesting that the inhibitory effect of p53 was still present in cells treated with HDACis.

A previous study has established that overexpression of HDAC1 suppresses Bmf expression by inhibiting the promoter activity (Zhang et al., 2006a). Therefore, we next evaluated whether IFN-γ affects the interaction of p53 with the Bmf promoter. The Bmf promoter region at −97 is the region where histone hyperacetylation occurs in cancer cell lines in response to HDACi treatment to increase Bmf transcription (Zhang et al., 2006a). We found by chromatin immunoprecipitation (ChIP) that region −97, but not −560, of the Bmf promoter interacted with HDAC1 in p53−/− but not in p53+/+ HCT116 cells, and IFN-γ had no effect on this interaction (Fig. 4 A). In AALEB cells, p53 interaction with the Bmf promoter was absent in nontreated controls but was present at region −97 and not −560 when cells were treated with IFN-γ (Fig. 4 B). These findings suggest that HDAC1 requires the absence of p53 for its interaction with the Bmf promoter and that IFN-γ enhances the interaction of p53 with the Bmf promoter to suppress transcription.

p53 is enriched in the nucleus and interacts with the Bmf promoter. (A) ChIP assays on p53+/+ or p53−/− HCT116 cells using a polyclonal antibody to HDAC1 or rabbit IgG1 as a control. DNA identification was performed using PCR with primers specific for Bmf promoter regions at −97 or −560 regions. (B) ChIP assay on AALEB cells nontreated (NT) or treated with IFN-γ for 48 h using a monoclonal antibody to p53 or mouse IgG1 as a control using primers specific for Bmf promoter regions at −97 or −560 regions. Densitometry of PCR products from six independent experiments for DNA pulled down with p53 normalized to IgG1 and IFN-γ–treated cells normalized to the nontreated values. IN, input. (C) Increased p53 protein levels in the nuclear fractions of AALEB cells 48 h after treatment with IFN-γ compared with nontreated controls. Nuclear and cytosolic extracts were analyzed for p53, lamin, and actin. Panel is representative of three independent experiments. (D) Representative photomicrographs of nontreated and IFN-γ–treated AALEB cells immunostained with the anti-p53 antibody and quantification of p53-positive nuclei (n = 4/group). Bar, 15 µm. (E) Bmf mRNA levels in MAECs isolated from p53ΔP, p53AXXA, and p53wt littermates 48 h after IFN-γ treatment compared with nontreated MAECs (n = 3 independent experiments). WT, wild type. Error bars indicate ±SEM. *, P < 0.05.

Because the nuclear or cytosolic localization defines p53 function (Lee and Gu, 2010), we assessed p53 localization in IFN-γ–treated cells and found that compared with nontreated controls, IFN-γ increased nuclear p53 levels, whereas cytosolic p53 levels remained low and unchanged (Fig. 4 C). Immunofluorescence also demonstrated that the percentage of cells with nuclear p53 was significantly increased by IFN-γ treatment (Fig. 4 D).

In humans, the proline-rich domain (PRD) of p53 is defined by residues 58–98, which contains 15 prolines and five repeats of the amino acid motif PXXP (in which P designates proline and X designates any amino acid). The histone acetyltransferase p300 binds to PXXP-containing peptides derived from the proline repeat domain, and the PXXP motif in p53 is required for p53 acetylation by the transcription coactivator p300 (Dornan et al., 2003). Therefore, we investigated whether the PRD region plays a role in the IFN-γ–induced Bmf suppression. In mice, this PRD consists of two PXXP motifs, and we obtained mice with a deletion of the PRD (p53ΔP) or lacking the four critical proline residues at loci 79, 82, 84, and 87 that make up the tandem PXXP sites (p53AXXA; Toledo et al., 2006, 2007). MAECs isolated from p53ΔP, p53AXXA, and wild-type littermates showed a reduction in Bmf levels when treated with IFN-γ similar to that observed in wild-type MAECs (Fig. 4 E).

p53 interacts with HDAC1 as part of a deacetylation complex (Luo et al., 2000, 2004), and our experiments showed that HDAC1 interacts with the Bmf promoter, and IFN-γ drives the interaction of p53 with the Bmf promoter. Therefore, we investigated the interaction of p53 and HDAC1 in IFN-γ–treated cells using immunoprecipitation assays and found that IFN-γ increased HDAC1 levels in pull-down products using anti-p53 antibodies (Fig. 5 A), demonstrating that IFN-γ enhanced the p53–HDAC1 interaction. Therefore, we assessed the acetylation state of p53 after IFN-γ treatment and found significantly reduced acetylated p53 levels in IFN-γ–treated compared with nontreated cells (Fig. 5 A). An antibody specific to Lys382 was selected for detection because this is the site mostly acetylated within p53 (Gu and Roeder, 1997). Deacetylation of p53 by IFN-γ was also observed in IFN-γ–treated MAECs (Fig. 5 B).

IFN-γ causes HDAC1–p53 interaction and deacetylates p53. (A) HDAC1 interacts with p53. Nuclear lysates were prepared from AALEB cells after treatment with IFN-γ for 48 h and were immunoprecipitated (IP) using a p53-specific monoclonal antibody. The nuclear lysates (input) and immunoprecipitates were resolved by SDS-PAGE and analyzed using Western blotting with antibodies to HDAC1, total p53, and acetyl-p53. Panel is representative of three independent experiments. (B) Nuclear extracts prepared from IFN-γ–treated and nontreated MAECs and probed for total p53, acetyl-p53, and lamin. (C) Bmf mRNA levels in IFN-γ–, TSA-, or IFN-γ + TSA–treated AALEB cells compared with nontreated (NT) controls. (D) Representative photomicrographs IFN-γ–, TSA-, or IFN-γ/TSA-treated or nontreated AALEB cells immunostained for p53 (shown in red) and acetyl-p53 (shown in green). Nuclei were stained with DAPI (shown in blue), and merged images are shown in the right-most column. Bars, 10 µm. (E) Quantification of the percentage of cells positive for p53 and acetyl-p53 (Ac-p53) in wild-type (WT) and AXXA MEFs either nontreated or treated with IFN-γ, TSA, or IFN-γ/TSA. Error bars show group means ± SEM (n = 4/group). *, P < 0.05; **, P < 0.01; ***, P < 0.001.

To determine whether prevention of p53 deacetylation would abrogate the down-regulation of Bmf, we treated airway epithelial cells with IFN-γ for 24 h followed by TSA in the presence or absence of IFN-γ for an additional 24 h. Bmf was up-regulated in the TSA + IFN-γ–treated compared with IFN-γ–only treated control samples, suggesting that TSA prevented the HDAC1 and p53 interaction (Fig. 5 C), further supporting the idea that acetylation of p53 affects Bmf expression. Increased accumulation of nuclear p53 and enhanced deacetylation of p53 in IFN-γ–treated cells was verified by double staining of nontreated, IFN-γ–, TSA-, and TSA + IFN-γ–treated AALEB cells (Fig. 5 D). These findings were further validated in immunostained wild-type and AXXA mutant mouse embryonic fibroblasts (MEFs), and quantification showed that the percentage of p53-positive cells was increased, whereas the percentage of acetyl-p53–positive cells was decreased in IFN-γ–treated compared with nontreated controls (Fig. 5 E). TSA increased the acetylated p53 in control cells and in IFN-γ–stimulated cells in both wild-type and AXXA mutant MEFs.

IFN-γ–induced deacetylation of p53 (Fig. 6 A) and suppression of Bmf mRNA (Fig. 6 B) occurs in MAECs as early as 0.5 h after treatment, suggesting that it is a direct effect rather than by other mediators that result from IFN-γ treatment. This deacetylation of p53 was accompanied with the down-regulation of Bmf mRNA. Furthermore, IFN-γ suppressed Bmf in p53-sufficient wild-type colon epithelial cells (HCT116 p53+/+) but increased Bmf mRNA levels in HCT116 p53−/− cells within 0.5 h after treatment (Fig. 6 C).

IFN-γ deacetylates p53 but does not affect IFN-γ–induced cell death. (A) Nuclear extracts from MAECs 0.5 h after IFN-γ treatment or nontreated (NT) controls that were probed for total p53, acetyl-p53, and lamin. (B) Bmf mRNA levels in p53+/+ HCT116 cells 0.5 h after IFN-γ treatment or nontreated controls. *, P < 0.05. (C) Bmf mRNA in p53−/− and p53+/+ MAECs treated with IFN-γ for 0.5 h (n = 3 independent experiments). (D) Quantification of viable p53−/− and p53+/+ MAECs treated with IFN-γ for 48 h as assessed by trypan blue exclusion. (E) Quantification of viable AALEB cells transfected with an empty vector plasmid (shRNA control [CTR]) or one expressing p53 shRNA were treated with IFN-γ for 48 h. (F) Bcl-xL and Bcl-2 protein levels in protein lysates prepared from p53−/− and p53+/+ MAECs treated with nothing or IFN-γ for 48 h. Error bars indicate ±SEM.

Although IFN-γ increased the levels of the proapoptotic BmfCUG and BmfS in p53−/− cells, cell death and the viability of p53−/− and p53+/+ MAECs remained unchanged (Fig. 6 D). These results were confirmed in IFN-γ–treated AALEB cells, in which p53 was suppressed using shRNA targeting p53 (Fig. 6 E). Because Bcl-2 blocks the proapoptotic function of Bmf (Puthalakath et al., 2001), we tested whether IFN-γ increases expression of antiapoptotic proteins, Bcl-2 and Bcl-xL, and found that IFN-γ increased Bcl-2 and Bcl-xL protein levels in p53−/−, but not in p53+/+, MAECs (Fig. 6 F). Although these experiments suggest that increased expression of these antiapoptotic proteins may suppress apoptosis in IFN-γ–treated p53-deficient MAECs when BmfCUG and BmfS are increased, the role of IFN-γ suppressing the proapoptotic Bmf was still not clear.

IFN-γ induces autophagy in HeLa cells (Inbal et al., 2002) and in gastric epithelial cells (Tu et al., 2011). We had also observed that IFN-γ treatment increased both Beclin-1 and conversion of LC3-I to LC3-II in AALEB cells and wild-type MAECs (Fig. 7, A and B). Because the proapoptotic Bmf isoforms were suppressed by IFN-γ, we reasoned that Bmf may have a role in suppressing autophagy and that IFN-γ–induced down-regulation of Bmf may facilitate autophagy. Infection of AALEB cells with Ad-BmfCUG to reconstitute the suppression by IFN-γ or with Ad-GFP as a control showed that increased Bmf expression reduced the levels of IFN-γ–induced Beclin-1 and the processing of LC3-I to LC3-II (Fig. 7 C). Although lung or thymus tissues from bmf−/− showed no evidence of increased Beclin or LC3-II compared with those from bmf-+/+ mice (Fig. 7 D), cultured but nontreated bmf−/− MAECs and MEFs showed increased levels of Beclin-1 and accumulation of LC3-II compared with bmf-+/+ MAECs (Fig. 7 E). The formation of autophagosomes in cultured MAECs and MEFs was also indicated by transfection of pmCherry-LC3, displaying significantly increased punctation in bmf−/− compared with bmf-+/+ MAECs and MEFs (Fig. 7 F). In addition, transmission electron micrographs revealed significantly more autophagic vacuoles per unit area in bmf−/− compared with bmf-+/+ MAECs (Fig. 7 G). Some cells displayed mitochondria surrounded by membranous material that may be evidence for a premitophagic process. Although the number of mitochondria between bmf−/− and bmf-+/+ MAECs showed no difference, there appeared to be morphological differences with more damaged mitochondria being present in bmf−/− compared with bmf-+/+ MAECs. To determine the physiological relevance of reduced Bmf expression causing autophagy, we exposed bmf-+/+ and bmf−/− MEFs to autophagy-promoting conditions and found that starvation (Fig. 7 H) and treatment with pp242 (Fig. 7 I), a mammalian target of rapamycin (mTOR) inhibitor, caused enhanced cell death in bmf−/− compared with bmf-+/+ MEFs. These findings suggest that increased baseline autophagy in bmf−/− MEFs reduces the protective role of autophagy in these cells. Suppression of Bmf in p53−/− HCT116 cells using short hairpin Bmf (shBmf) increased autophagy (Fig. 7 J), and IFN-γ–induced autophagy was increased in p53+/+ but reduced in p53−/− HCT116 cells (Fig. 7 K), suggesting that IFN-γ–induced autophagy is p53 dependent. To investigate the mechanism by which Bmf may be regulating autophagy, we assessed whether Beclin-1–Bcl-2 interaction is affected by the presence or absence of Bmf. Although Bcl-2 levels are higher in bmf-+/+ compared with bmf−/− MEFs the difference is more evident after immunoprecipitation with Beclin-1 antibodies. Beclin-1–Bcl-2 interaction was also reduced when bmf-+/+ MEFs were treated with IFN-γ consistent with the observation that IFN-γ reduces Bmf levels and thereby enhances autophagy (Fig. 7 L). These findings suggest that the Beclin-1–Bcl-2 interaction is stabilized by the presence of Bmf, and loss or even reduced levels of Bmf releases Beclin-1 to cause autophagy. Bmf was not detected in the input but only in the immunoprecipitate from nontreated bmf-+/+ MEFs because it was concentrated by the immunoprecipitation process. Together, these findings suggest that IFN-γ, by deacetylating p53, suppresses Bmf expression to facilitate autophagy.

IFN-γ suppresses Bmf expression to induce autophagy. (A and B) IFN-γ increases protein levels of Beclin-1 and LC3B in AALEB cells (A) and MAECs (B) compared with nontreated (NT) controls as detected by Western blot analysis. (C) AALEB cells treated with IFN-γ for 48 h and infected with 50 MOI of Ad-BmfCUG or Ad-GFP. Bmf expression inhibits IFN-γ–induced Beclin-1 and LC3B protein levels. (D) Western blot of lung and thymus tissues from bmf+/+ (wild type [WT]) and bmf−/− (knockout [KO]) mice probed for Beclin-1 and LC3B proteins. (E) Western blot analysis of protein extracts from bmf+/+ (wild type) and bmf−/− (knockout) MAECs and MEFs probed for Beclin-1, LC3B, and β-actin. (F) Representative micrographs of bmf+/+ and bmf−/− MAECs and MEFs expressing mCherry-LC3B and cells with punctuate LC3B were quantified from >50 bmf+/+ and bmf−/− MAECs. (G) Representative electron micrographs of bmf+/+ and bmf−/− MAECs that were cultured in 6-well dishes. Thin sections of cells were analyzed by scanning electron microscopy. The arrowhead denotes an autophagic vesicle, and the arrow denotes a mitochondrion surrounded by a double membrane. Quantification of autophagic vesicles per 100 µm2 in >30 each bmf+/+ and bmf−/− MAECs. (H) Quantification of viable bmf+/+ and bmf−/− MEFs maintained in starvation media for 4 or 24 h relative to cell grown in regular media (n = 3 independent experiments). (I) Quantification of viable bmf+/+ and bmf−/− MEFs 24 h after treatment with the mTOR inhibitor, pp242, at 2.5 µM relative to nontreated control cells (n = 3 independent experiments). (J) Knockdown of Bmf mRNA using shBmf in p53−/− HCT116 cells reduced Bmf mRNA levels. Western blot analysis of shRNA control (shCtr)– and shBmf-transfected p53−/− HCT116 cells probed with Beclin-1, LC3B, and β-actin antibodies. The Western blot is representative of three experiments using three different shBmf constructs. (K) Western blot of IFN-γ–treated p53+/+ and p53−/− HCT116 cells probed with Beclin-1, LC3B, and β-actin antibodies. (L) Immunoprecipitation of protein extracts from IFN-γ–treated and nontreated bmf+/+ and nontreated bmf−/− MEFs using anti–Beclin-1 and Western blot analysis of input and immunoprecipitates (IP) with Bcl-2, Beclin-1, Bmf, and β-actin antibodies. Results are representative of four independent immunoprecipitations. Error bars indicate ±SEM. *, P < 0.05; **, P < 0.01; ***, P < 0.001; ****, P < 0.0001.

DiscussionThe present experiments show that IFN-γ induces autophagy by suppressing Bmf expression through HDAC1-mediated deacetylation of p53. A novel finding of this study is that IFN-γ caused deacetylation of human p53 at Lys382 and murine p53 at Lys379. We were compelled to investigate the role of IFN-γ in affecting p53 acetylation because HDAC1 interacts with the Bmf promoter to suppress its activity, and IFN-γ modified the interaction of p53 with the Bmf promoter but had no effect on Bmf mRNA half-life. These findings suggest that p53 and HDAC1 affect promoter activity by affecting chromatin remodeling. Our finding is consistent with a previous study that deacetylation of p53 is mediated by an HDAC1-containing complex (Murphy et al., 1999). IFN-γ–induced deacetylation of p53 and suppression of Bmf mRNA occur as early as 0.5 h after IFN-γ treatment, suggesting that these events are a direct effect of IFN-γ. However, the IFN-γ–induced signaling pathways that lead to HDAC1–p53 interaction and whether other proteins are involved in deacetylating p53 remain to be clarified. SIRT1 (Sirtuin 1), a mammalian NAD+-dependent HDAC, also deacetylates p53 and may be involved in this process (Liu et al., 2011).The fact that IFN-γ significantly decreased the number of AALEB cells that are immunopositive for acetylated p53 suggests that acetylated p53 present in nontreated cells may have a role for the regular proliferation of HAECs and MAECs. Loss of acetylation completely abolishes p53-dependent growth arrest and apoptosis (Tang et al., 2008) by abrogating the sequence-specific transcriptional activity of p53 (Gu and Roeder, 1997; Luo et al., 2000). These studies are consistent with our findings that IFN-γ–induced cell death utilizes a pathway that is independent of p53. The observation that p53+/+ and p53−/− MAECs are equally susceptible to IFN-γ–induced cell death is supported by our previous study demonstrating that IFN-γ–induced Bik, the central mediator of IFN-γ–induced cell death, causes cell death in a p53-independent manner (Mebratu et al., 2008). Similarly, other studies in various cell types, including breast carcinoma and neuroblastoma cell lines (Porta et al., 2005), human colon adenocarcinoma cells (Ossina et al., 1997), and gastric cancer cells (Gao et al., 2010), have shown that IFN-γ–induced cell death is p53 independent. However, IFN-γ may require p53 in certain cell types, as p53 accumulation is associated with IFN-γ sensitizing hepatocytes to apoptosis induced by genotoxic stress in mice that overexpress IFN-γ in the liver (Lüth et al., 2011).IFN-γ–induced p53 deacetylation was accompanied with nuclear accumulation of p53. Therefore, it is possible that nuclear export mechanisms for p53 involve acetylation. Although the PXXP motif in p53 is required for p53 acetylation by the transcription coactivator p300 (Dornan et al., 2003), our findings show that this motif appears to be dispensable for the deacetylation. p53 is acetylated on the eight lysine residues in the DNA-binding and C-terminal regions, and such acetylation prevents interaction with Mdm2 and leads to stabilization and increase in the p53 protein (Tang et al., 2008). The classical importin-α–β pathway is responsible for import of only nonubiquitinated p53 into the nucleus during the early stages of stress response, such as DNA damage (Marchenko et al., 2010). Future studies will investigate whether lysine residues other than Lys382 are deacetylated by IFN-γ and whether this modification minimizes the interaction of p53 with nuclear export proteins, such as MDM2-mediated ubiquitination (Boyd et al., 2000), which may result in increased nuclear p53 levels.Although the importance of p53 acetylation on transcriptional activation has been studied, the functional significance of deacetylated p53 remains unknown. Our findings suggest that p53 blocks the interaction of HDAC1 with the Bmf promoter and that deacetylated p53 is likely to be part of a repressive complex responsible for the down-regulation of Bmf. Many other genes have been reported to be suppressed by p53, including DNA topoisomerase II, cyclin B, Cdc2 (Yun et al., 1999), MMP-1 and -13 (Sun et al., 2000), presenilin-1 (Roperch et al., 1998), myc, and Map-4 (Kidokoro et al., 2008). In general, transcriptional activation requires p53 to bind a consensus sequence (Wei et al., 2006); however, the repression mechanism by p53 is not well studied. For example, p53 directly binds to the Mad1L1 promoter, but no p53 consensus site was found (Chun and Jin, 2003). Similarly, we found that p53 interacts with the Bmf promoter in IFN-γ–treated cells, although the bmf upstream region lacks a consensus p53 response element. Although DNA binding is required for p53 to suppress cdc2 (Yun et al., 1999) or cdc20 (Banerjee et al., 2009), p53 can also suppress by interfering with transactivating factors (St Clair et al., 2004). Future studies will elucidate whether similar mechanisms are involved for p53 to suppress Bmf expression.HDACis consistently increased Bmf expression in primary HAECs and MAECs, and p53 suppressed the extent of Bmf induction. The fact that p53 dampens HDACi-induced Bmf expression supports the idea that deacetylated p53 interacts with the Bmf promoter to suppress its activity. Similar to our findings, the HDACis induce acetylation of histones H3 and H4 at the Bmf promoter region in various human cancer cell lines, and ectopic expression of HDAC1 reduces Bmf expression (Zhang et al., 2006a). The previous studies also showed that overexpression of histone acetyltransferase p300 mimics the effects of the HDACis, suggesting that Bmf expression in cancer cells is primarily regulated by histone hyperacetylation (Zhang et al., 2006b), and this approach was the basis to promote HDACis for cancer therapy. However, the present study shows that HDACis also induce Bmf in primary airway epithelial cells, suggesting that the use of these compounds for cancer therapy may be associated with side effects and should be approached with caution.Although the role IFN-γ plays in cell death has been established, the role of IFN-γ in inducing autophagy in airway epithelial cells and the mechanisms involved have not been previously reported. IFN-γ increases Beclin-1 expression and the conversion of LC3I to LC3II, suggesting the formation of autophagosomes. We find that IFN-γ–induced suppression of Bmf plays a critical role in allowing autophagy to proceed because restoring Bmf expression suppressed the induction of Beclin-1 and the processing of LC3I to LC3II. However, in p53−/− cells in which IFN-γ induces Bmf expression, autophagy was suppressed. The observation that there is no evidence for autophagy in the lung or thymus tissues from bmf−/− mice, but autophagy is enhanced in cultured bmf−/− MAECs and MEFs, suggests that Bmf is crucial in suppressing autophagy in cells when there is minimal stress, such as present in culture conditions. The fact that increased autophagy in bmf−/− MEFs reduces the protective role of autophagy in these cells supports the overall idea that IFN-γ, by reducing Bmf expression, sensitizes cells to death. Grespi et al. (2010) used MEFs from wild-type and knockout mice that were immortalized with SV40 and tested the effect of serum deprivation or inhibition of the phosphatidylinositol-3-kinase/AKT/mTOR network over time, and death was reduced by Bmf deficiency in these cells. It is possible that starvation media cause autophagy that is different from FCS withdrawal or mTOR inhibition, or the response may be different in primary MEFs. Although IFN-γ–induced autophagy in melanoma cells (Yan et al., 2011) and in gastric cells (Tu et al., 2011) has recently been reported, the mechanisms by which this autophagy is mediated are unknown. Our experiments suggest that IFN-γ, by reducing the interaction of Beclin-1 with Bcl-2, enhances autophagy and that Bmf stabilizes the Beclin-1–Bcl-2 interaction because loss of Bmf diminished the interaction of Beclin-1 and Bcl-2. To our knowledge, this is the first study to show that Bmf is a potent inhibitor of autophagy. A previous study showed that Ras-induced expression of Noxa and Beclin-1 promotes autophagic cell death by Noxa displacing Bcl-2 from Beclin-1 (Elgendy et al., 2011). In contrast, Bim inhibits autophagy by recruiting Beclin-1 to microtubules (Luo et al., 2012). Together, these findings add a new dimension to the role of the BH3-only protein family in regulating autophagy. p53 plays a dual role as a positive and a negative regulator of autophagy (Maiuri et al., 2010). Nuclear p53 is reported to encourage autophagy through transcriptional control of specific genes, whereas cytoplasmic p53 has an inhibitory effect (Tasdemir et al., 2008). Our findings suggest that suppression of Bmf by nuclear p53 may be one of the mechanisms by which autophagy is enhanced by nuclear p53.

The present experiments show that IFN-γ induces autophagy by suppressing Bmf expression through HDAC1-mediated deacetylation of p53. A novel finding of this study is that IFN-γ caused deacetylation of human p53 at Lys382 and murine p53 at Lys379. We were compelled to investigate the role of IFN-γ in affecting p53 acetylation because HDAC1 interacts with the Bmf promoter to suppress its activity, and IFN-γ modified the interaction of p53 with the Bmf promoter but had no effect on Bmf mRNA half-life. These findings suggest that p53 and HDAC1 affect promoter activity by affecting chromatin remodeling. Our finding is consistent with a previous study that deacetylation of p53 is mediated by an HDAC1-containing complex (Murphy et al., 1999). IFN-γ–induced deacetylation of p53 and suppression of Bmf mRNA occur as early as 0.5 h after IFN-γ treatment, suggesting that these events are a direct effect of IFN-γ. However, the IFN-γ–induced signaling pathways that lead to HDAC1–p53 interaction and whether other proteins are involved in deacetylating p53 remain to be clarified. SIRT1 (Sirtuin 1), a mammalian NAD+-dependent HDAC, also deacetylates p53 and may be involved in this process (Liu et al., 2011).

The fact that IFN-γ significantly decreased the number of AALEB cells that are immunopositive for acetylated p53 suggests that acetylated p53 present in nontreated cells may have a role for the regular proliferation of HAECs and MAECs. Loss of acetylation completely abolishes p53-dependent growth arrest and apoptosis (Tang et al., 2008) by abrogating the sequence-specific transcriptional activity of p53 (Gu and Roeder, 1997; Luo et al., 2000). These studies are consistent with our findings that IFN-γ–induced cell death utilizes a pathway that is independent of p53. The observation that p53+/+ and p53−/− MAECs are equally susceptible to IFN-γ–induced cell death is supported by our previous study demonstrating that IFN-γ–induced Bik, the central mediator of IFN-γ–induced cell death, causes cell death in a p53-independent manner (Mebratu et al., 2008). Similarly, other studies in various cell types, including breast carcinoma and neuroblastoma cell lines (Porta et al., 2005), human colon adenocarcinoma cells (Ossina et al., 1997), and gastric cancer cells (Gao et al., 2010), have shown that IFN-γ–induced cell death is p53 independent. However, IFN-γ may require p53 in certain cell types, as p53 accumulation is associated with IFN-γ sensitizing hepatocytes to apoptosis induced by genotoxic stress in mice that overexpress IFN-γ in the liver (Lüth et al., 2011).

IFN-γ–induced p53 deacetylation was accompanied with nuclear accumulation of p53. Therefore, it is possible that nuclear export mechanisms for p53 involve acetylation. Although the PXXP motif in p53 is required for p53 acetylation by the transcription coactivator p300 (Dornan et al., 2003), our findings show that this motif appears to be dispensable for the deacetylation. p53 is acetylated on the eight lysine residues in the DNA-binding and C-terminal regions, and such acetylation prevents interaction with Mdm2 and leads to stabilization and increase in the p53 protein (Tang et al., 2008). The classical importin-α–β pathway is responsible for import of only nonubiquitinated p53 into the nucleus during the early stages of stress response, such as DNA damage (Marchenko et al., 2010). Future studies will investigate whether lysine residues other than Lys382 are deacetylated by IFN-γ and whether this modification minimizes the interaction of p53 with nuclear export proteins, such as MDM2-mediated ubiquitination (Boyd et al., 2000), which may result in increased nuclear p53 levels.

Although the importance of p53 acetylation on transcriptional activation has been studied, the functional significance of deacetylated p53 remains unknown. Our findings suggest that p53 blocks the interaction of HDAC1 with the Bmf promoter and that deacetylated p53 is likely to be part of a repressive complex responsible for the down-regulation of Bmf. Many other genes have been reported to be suppressed by p53, including DNA topoisomerase II, cyclin B, Cdc2 (Yun et al., 1999), MMP-1 and -13 (Sun et al., 2000), presenilin-1 (Roperch et al., 1998), myc, and Map-4 (Kidokoro et al., 2008). In general, transcriptional activation requires p53 to bind a consensus sequence (Wei et al., 2006); however, the repression mechanism by p53 is not well studied. For example, p53 directly binds to the Mad1L1 promoter, but no p53 consensus site was found (Chun and Jin, 2003). Similarly, we found that p53 interacts with the Bmf promoter in IFN-γ–treated cells, although the bmf upstream region lacks a consensus p53 response element. Although DNA binding is required for p53 to suppress cdc2 (Yun et al., 1999) or cdc20 (Banerjee et al., 2009), p53 can also suppress by interfering with transactivating factors (St Clair et al., 2004). Future studies will elucidate whether similar mechanisms are involved for p53 to suppress Bmf expression.

HDACis consistently increased Bmf expression in primary HAECs and MAECs, and p53 suppressed the extent of Bmf induction. The fact that p53 dampens HDACi-induced Bmf expression supports the idea that deacetylated p53 interacts with the Bmf promoter to suppress its activity. Similar to our findings, the HDACis induce acetylation of histones H3 and H4 at the Bmf promoter region in various human cancer cell lines, and ectopic expression of HDAC1 reduces Bmf expression (Zhang et al., 2006a). The previous studies also showed that overexpression of histone acetyltransferase p300 mimics the effects of the HDACis, suggesting that Bmf expression in cancer cells is primarily regulated by histone hyperacetylation (Zhang et al., 2006b), and this approach was the basis to promote HDACis for cancer therapy. However, the present study shows that HDACis also induce Bmf in primary airway epithelial cells, suggesting that the use of these compounds for cancer therapy may be associated with side effects and should be approached with caution.

Although the role IFN-γ plays in cell death has been established, the role of IFN-γ in inducing autophagy in airway epithelial cells and the mechanisms involved have not been previously reported. IFN-γ increases Beclin-1 expression and the conversion of LC3I to LC3II, suggesting the formation of autophagosomes. We find that IFN-γ–induced suppression of Bmf plays a critical role in allowing autophagy to proceed because restoring Bmf expression suppressed the induction of Beclin-1 and the processing of LC3I to LC3II. However, in p53−/− cells in which IFN-γ induces Bmf expression, autophagy was suppressed. The observation that there is no evidence for autophagy in the lung or thymus tissues from bmf−/− mice, but autophagy is enhanced in cultured bmf−/− MAECs and MEFs, suggests that Bmf is crucial in suppressing autophagy in cells when there is minimal stress, such as present in culture conditions. The fact that increased autophagy in bmf−/− MEFs reduces the protective role of autophagy in these cells supports the overall idea that IFN-γ, by reducing Bmf expression, sensitizes cells to death. Grespi et al. (2010) used MEFs from wild-type and knockout mice that were immortalized with SV40 and tested the effect of serum deprivation or inhibition of the phosphatidylinositol-3-kinase/AKT/mTOR network over time, and death was reduced by Bmf deficiency in these cells. It is possible that starvation media cause autophagy that is different from FCS withdrawal or mTOR inhibition, or the response may be different in primary MEFs. Although IFN-γ–induced autophagy in melanoma cells (Yan et al., 2011) and in gastric cells (Tu et al., 2011) has recently been reported, the mechanisms by which this autophagy is mediated are unknown. Our experiments suggest that IFN-γ, by reducing the interaction of Beclin-1 with Bcl-2, enhances autophagy and that Bmf stabilizes the Beclin-1–Bcl-2 interaction because loss of Bmf diminished the interaction of Beclin-1 and Bcl-2. To our knowledge, this is the first study to show that Bmf is a potent inhibitor of autophagy. A previous study showed that Ras-induced expression of Noxa and Beclin-1 promotes autophagic cell death by Noxa displacing Bcl-2 from Beclin-1 (Elgendy et al., 2011). In contrast, Bim inhibits autophagy by recruiting Beclin-1 to microtubules (Luo et al., 2012). Together, these findings add a new dimension to the role of the BH3-only protein family in regulating autophagy. p53 plays a dual role as a positive and a negative regulator of autophagy (Maiuri et al., 2010). Nuclear p53 is reported to encourage autophagy through transcriptional control of specific genes, whereas cytoplasmic p53 has an inhibitory effect (Tasdemir et al., 2008). Our findings suggest that suppression of Bmf by nuclear p53 may be one of the mechanisms by which autophagy is enhanced by nuclear p53.

Materials and methodsAnimalsPathogen-free STAT1−/−, p53+/− mice were purchased from The Jackson Laboratory, and bik−/− and bmf−/− mice on the C57BL/6 background were previously described (Coultas et al., 2004; Labi et al., 2008) and were made available by A. Strasser (Walter and Eliza Hall Institute, Parkville, Australia). Mice with modified PRD p53ΔP and p53AXXA (Toledo et al., 2007), on the 129J background, were obtained from G.M. Wahl (The Salk Institute for Biological Studies, La Jolla, CA). These mice along with the wild-type littermates were bred at the Lovelace Respiratory Research Institute under specific pathogen-free conditions and genotyped as described previously (Labi et al., 2008). STAT1−/−, p53+/− mice were genotyped using protocols provided by The Jackson Laboratory. All animal experiments were approved by the Institutional Animal Care and Use Committee and at the Lovelace Respiratory Research Institute, a facility approved by the Association for the Assessment and Accreditation for Laboratory Animal Care International.Cell cultureThe preparation of MAECs was performed as described previously (You et al., 2002). In brief, tracheas were cut open lengthwise and incubated in pronase solution (DMEM, 1.4 mg/ml pronase, and 0.1 mg/ml DNase) overnight at 4°C. Enzymatic activity was stopped with 10% FBS (Invitrogen), and cells were collected by gently rocking tracheas in DMEM/Ham’s H12 media (Invitrogen) followed by centrifugation at 400 g for 10 min at 4°C. Cells were incubated in 5 ml of declumping solution (DMEM and 2 mM EDTA) and plated on collagen-coated plates. The immortalized HAECs, AALEB cells (Stout et al., 2007), and HAECs (Takara Bio Inc.) were maintained in bronchial epithelial growth medium (Lonza). The p53-deficient SAOS-2 and Calu-6 and the p53-sufficient small airway carcinoma-derived A549 cell lines were maintained in bronchial epithelial growth medium with penicillin/streptomycin, FBS, and l-glutamine. HCT116 p53−/− and p53+/+ cells (a gift from B. Vogelstein, Johns Hopkins University, Baltimore, MD) were maintained in McCoy’s medium supplemented with FBS. MEFs from bmf+/+ and bmf−/− mice were cultured in DMEM supplemented with 10% FBS and were used for experiments at passages 3–15. We used Earle’s Balanced Salt Solution (Sigma-Aldrich) as a starvation medium. Cells were seeded on 6-well tissue-culture plates and cultured to 60–70% confluency before irradiation with 10 mJ UV light using a UV cross-linker (Stratalinker 1800; Agilent Technologies).Quantitative RT-PCR (qRT-PCR)Extraction of RNA from cell pellets was performed using the RNeasy kit (QIAGEN), and concentration was determined using a spectrophotometer (NanoDrop 1000; Thermo Fisher Scientific). The primer/probe sets for Bmf, CDKN1B, and 18s were obtained from Applied Biosystems. Target mRNAs were amplified by quantitative real-time PCR in 20-µl reactions on the real-time PCR system (PRISM 7900HT; Applied Biosystems) using the One-Step RT-PCR Master Mix (TaqMan; Applied Biosystems). Relative quantities from duplicate amplifications were calculated by normalizing averaged threshold cycle (Ct) values to CDKN1B and/or 18s to obtain ΔCt, and the relative standard curve method was used for determining the fold change as described previously (Schwalm et al., 2008). For qRT-PCR, primers specific for mouse GAPDH, 5′-AGGCCGGTGCTGAGTATGTC-3′ and 5′-TGCCTGCTTCACCACCTTCT-3′, were used for normalizing RNA levels.Immunoprecipitation and Western blot analysisTotal protein lysates or cytosolic and nuclear fractions were prepared (Mebratu et al., 2008), and protein was analyzed by Western blotting as previously described (Stout et al., 2007). In brief, cells were lysed in NP-40 to obtain the cytosolic fraction, and nuclear fractions were prepared extracting the nuclear pellet with a hypertonic extraction buffer (50 mM Hepes, pH 7.8, 50 mM KCl, and 300 mM NaCl) in the presence of protease and phosphatase inhibitors. For immunoprecipitation using the cross-link immunoprecipitation kit (Thermo Fisher Scientific), cells were rinsed twice with cold PBS, scraped into cold PBS plus protease inhibitors, and analyzed per the manufacturer’s instructions. We used the rat anti-Bmf monoclonal antibody, a gift from A. Strasser and A. Villunger (Innsbruck Medical University, Innsbruck, Austria) at 2 µg/ml, and the rabbit anti-p53 polyclonal antibody (FL-393; Santa Cruz Biotechnology, Inc.), acetyl-p53 (Lys382; Cell Signaling Technology), rabbit antilamin polyclonal (#2032; Cell Signaling Technology), or rabbit anti-HDAC1 polyclonal (EMD Millipore) were used at a 1:1,000 dilution. We tested various Bcl-2 antibodies, including those obtained from BD (catalog nos. 554279 and 554087) and Santa Cruz Biotechnology, Inc. (sc-492, sc-7382, and sc-578), and found that sc-7382 and sc-578 showed a single 26-kD band representing Bcl-2. Therefore, the rat anti–Bcl-2 monoclonal sc-578 was used for immunoprecipitation and Western blot experiments. The Bcl-2 and the rabbit anti–Bcl-xL polyclonal (sc-634; Santa Cruz Biotechnology, Inc.) antibodies were used at 1:300 and 1:200 dilutions, respectively. For immunoprecipitation experiments, rabbit anti–Beclin-1 (Abcam) was used, but for Western blotting, rabbit anti–Beclin-1 and rabbit anti-LC3B (Cell Signaling Technology) antibodies were both used at 1:1,000. The mouse actin polyclonal antibody was used at 1:5,000 (Santa Cruz Biotechnology, Inc.), and goat anti–rat, goat anti–rabbit secondary, or rabbit anti–mouse antibodies were used for visualizing proteins with chemiluminescence (PerkinElmer) using the image reader (LAS-4000; Fujifilm).Immunofluorescence analysisFor immunofluorescence, AALEB cells grown on eight-chamber slides (Lab-Tek II; Thermo Fisher Scientific) were fixed using 3% paraformaldehyde with 3% sucrose in PBS and permeabilized using 0.2% Triton X-100 with 0.2% saponin in a blocking solution containing 3% IgG-free BSA, 1% gelatin, and 2% normal donkey serum. Cells were probed with anti-p53 (#sc126; clone DO1; Santa Cruz Biotechnology, Inc.), anti–acetyl-p53 (Lys382; #2525; Cell Signaling Technology), or isotype controls at a 1: 500 dilution. MEFs were probed with anti-p53 (#sc6243; clone FL393; Santa Cruz Biotechnology, Inc.) or antiacetyl-p53 (Lys379; #2570; Cell Signaling Technology). The immunolabeled cells were detected using F(ab)2 fragments of respective secondary antibodies conjugated to Dylight-549 or -649 (Jackson ImmunoResearch Laboratories, Inc.) at a 1:500 dilution and mounted with Fluoromount-G (SouthernBiotech) containing DAPI for nuclear staining. Fluorescently labeled cells were analyzed as described previously (Stout et al., 2007) using an imaging system (Axioplan 2; Carl Zeiss) equipped with a charge-coupled device camera (ORCA-ER; Hamamatsu Photonics) coupled with a wavelength switch (Lambda DG-4; Sutter Instrument) and acquisition software (SlideBook 5; Intelligent Imaging Innovations, Inc.). Quantification of p53-positive or acetyl-p53–positive cells was conducted by counting ≥400 cells/well by a person unaware of the treatment groups.Adenoviral and retroviral expression vectorsAdenoviral expression vectors for p53 and BmfCUG and BmfS were developed by cloning the respective cDNAs into the shuttle vector. RNA was isolated from MAECs, and 1.5 µg was converted to cDNA using the cDNA synthesis kit (SuperScript; Invitrogen). RT-PCR was performed using the following primers: BmfS, 5′-AAAATGGAGCCACCTCAGTGTGTG-3′ and 5′-TCACCAGGGCCCCACCCCTTC-3′; BmfCUG, 5′-AAACTGGCGCAGAGCCCTGGCATCA-3′ and 5′-TCACCAGGGCCCCACCCCTTC-3′; and BmfL, 5′-ATGCCCGGAGCGGGCGTATTT-3′ and 5′-TCACCAGGGCCCCACCCCTTC-3′. Shuttle cytomegalovirus (CMV)_p53, shuttle CMV_BmfCUG, and shuttle CMV_BmfS were packaged into virus-producing cells, and Ad particles were harvested by collecting media at the time when cells were rounding up as described previously (Xiang et al., 1996).The retroviral silencing vector encoding for p53 shRNA and the control vector were purchased from OriGene. Retroviral vectors expressing the respective shRNA or control constructs were packaged in Phoenix by transfecting using FuGENE (Promega), and the harvested virus was used to infect AALEB cells as previously described (Mebratu et al., 2008).Bmf mRNA half-lifeCells were treated with the RNA polymerase inhibitor DRB (Sigma-Aldrich) at a final concentration of 50 ng/ml for SAOS-2, Calu-3, A549, and Calu-6 and at 100 ng/ml for AALEBs to stop RNA polymerase activity. The relative mRNA abundance was calculated using the ΔΔCt method, and mRNA half-life was calculated using the Greenberg formula by averaging the calculated slope from mRNA levels over time (Greenberg, 1972).ChIPAfter treating with 50 ng/ml IFN-γ for 48 h, cells were fixed with 1% formaldehyde, and the reaction was quenched using 1.25 M glycine and scraped into cold PBS containing protease inhibitors. ChIP was performed using the Magna ChIP G kit (MAGNA0002; EMD Millipore) as described by the manufacturer. Sonicated nuclear fractions were incubated with mouse IgG1 monoclonal antibody to p53 (sc-98 Pab1801; Santa Cruz Biotechnology, Inc.), rabbit polyclonal antibody to HDAC1 (06–720; EMD Millipore), rabbit polyclonal antibody to acetyl-H4 (06–598; EMD Millipore), rabbit polyclonal antibody to acetyl-H3 (06-599B; EMD Millipore), mouse IgG1, or rabbit as a control (CBL600; EMD Millipore). DNA identification was confirmed with PCR using the following primers specific for the Bmf promoter: region −560, 5′-ACCTAAGGGCTCCCCTGGA-3′ and 5′-GCAGGTCGGAAGAAAACTGCAGC-3′, and region −97, 5′-TTGGCGCTTCACTCGCCATT-3′ and 5′-ATCCCGCAAACAGCTGAT-3′.Transmission electron microscopyCells used for electron microscopy analysis were fixed in 2.5% glutaraldehyde and postfixed with 1% osmium tetroxide before being dehydrated in ethanol. Tissues were infiltrated with propylene oxide and embedded in Agar 100 resin for preparation of ultrathin sections. After staining with uranyl acetate and lead citrate, sections were examined using a transmission electron microscope (JEM 1210; JEOL) at 80 or 60 kV onto electron microscope film (ESTAR Thick Base; Kodak). Electron micrographs (n = 30) for each sample were quantified for mitochondria, and autophagosome number and relative area were calculated using ImageJ software (National Institutes of Health).Statistical analysisFold changes or scanned density values were averaged and compared for significance between groups using the Student’s t test. Data were analyzed using Prism statistical analysis software (GraphPad Software), and P < 0.05 was considered statistically significant.

Materials and methods

AnimalsPathogen-free STAT1−/−, p53+/− mice were purchased from The Jackson Laboratory, and bik−/− and bmf−/− mice on the C57BL/6 background were previously described (Coultas et al., 2004; Labi et al., 2008) and were made available by A. Strasser (Walter and Eliza Hall Institute, Parkville, Australia). Mice with modified PRD p53ΔP and p53AXXA (Toledo et al., 2007), on the 129J background, were obtained from G.M. Wahl (The Salk Institute for Biological Studies, La Jolla, CA). These mice along with the wild-type littermates were bred at the Lovelace Respiratory Research Institute under specific pathogen-free conditions and genotyped as described previously (Labi et al., 2008). STAT1−/−, p53+/− mice were genotyped using protocols provided by The Jackson Laboratory. All animal experiments were approved by the Institutional Animal Care and Use Committee and at the Lovelace Respiratory Research Institute, a facility approved by the Association for the Assessment and Accreditation for Laboratory Animal Care International.

Pathogen-free STAT1−/−, p53+/− mice were purchased from The Jackson Laboratory, and bik−/− and bmf−/− mice on the C57BL/6 background were previously described (Coultas et al., 2004; Labi et al., 2008) and were made available by A. Strasser (Walter and Eliza Hall Institute, Parkville, Australia). Mice with modified PRD p53ΔP and p53AXXA (Toledo et al., 2007), on the 129J background, were obtained from G.M. Wahl (The Salk Institute for Biological Studies, La Jolla, CA). These mice along with the wild-type littermates were bred at the Lovelace Respiratory Research Institute under specific pathogen-free conditions and genotyped as described previously (Labi et al., 2008). STAT1−/−, p53+/− mice were genotyped using protocols provided by The Jackson Laboratory. All animal experiments were approved by the Institutional Animal Care and Use Committee and at the Lovelace Respiratory Research Institute, a facility approved by the Association for the Assessment and Accreditation for Laboratory Animal Care International.

Cell cultureThe preparation of MAECs was performed as described previously (You et al., 2002). In brief, tracheas were cut open lengthwise and incubated in pronase solution (DMEM, 1.4 mg/ml pronase, and 0.1 mg/ml DNase) overnight at 4°C. Enzymatic activity was stopped with 10% FBS (Invitrogen), and cells were collected by gently rocking tracheas in DMEM/Ham’s H12 media (Invitrogen) followed by centrifugation at 400 g for 10 min at 4°C. Cells were incubated in 5 ml of declumping solution (DMEM and 2 mM EDTA) and plated on collagen-coated plates. The immortalized HAECs, AALEB cells (Stout et al., 2007), and HAECs (Takara Bio Inc.) were maintained in bronchial epithelial growth medium (Lonza). The p53-deficient SAOS-2 and Calu-6 and the p53-sufficient small airway carcinoma-derived A549 cell lines were maintained in bronchial epithelial growth medium with penicillin/streptomycin, FBS, and l-glutamine. HCT116 p53−/− and p53+/+ cells (a gift from B. Vogelstein, Johns Hopkins University, Baltimore, MD) were maintained in McCoy’s medium supplemented with FBS. MEFs from bmf+/+ and bmf−/− mice were cultured in DMEM supplemented with 10% FBS and were used for experiments at passages 3–15. We used Earle’s Balanced Salt Solution (Sigma-Aldrich) as a starvation medium. Cells were seeded on 6-well tissue-culture plates and cultured to 60–70% confluency before irradiation with 10 mJ UV light using a UV cross-linker (Stratalinker 1800; Agilent Technologies).

Cell culture

The preparation of MAECs was performed as described previously (You et al., 2002). In brief, tracheas were cut open lengthwise and incubated in pronase solution (DMEM, 1.4 mg/ml pronase, and 0.1 mg/ml DNase) overnight at 4°C. Enzymatic activity was stopped with 10% FBS (Invitrogen), and cells were collected by gently rocking tracheas in DMEM/Ham’s H12 media (Invitrogen) followed by centrifugation at 400 g for 10 min at 4°C. Cells were incubated in 5 ml of declumping solution (DMEM and 2 mM EDTA) and plated on collagen-coated plates. The immortalized HAECs, AALEB cells (Stout et al., 2007), and HAECs (Takara Bio Inc.) were maintained in bronchial epithelial growth medium (Lonza). The p53-deficient SAOS-2 and Calu-6 and the p53-sufficient small airway carcinoma-derived A549 cell lines were maintained in bronchial epithelial growth medium with penicillin/streptomycin, FBS, and l-glutamine. HCT116 p53−/− and p53+/+ cells (a gift from B. Vogelstein, Johns Hopkins University, Baltimore, MD) were maintained in McCoy’s medium supplemented with FBS. MEFs from bmf+/+ and bmf−/− mice were cultured in DMEM supplemented with 10% FBS and were used for experiments at passages 3–15. We used Earle’s Balanced Salt Solution (Sigma-Aldrich) as a starvation medium. Cells were seeded on 6-well tissue-culture plates and cultured to 60–70% confluency before irradiation with 10 mJ UV light using a UV cross-linker (Stratalinker 1800; Agilent Technologies).

Quantitative RT-PCR (qRT-PCR)Extraction of RNA from cell pellets was performed using the RNeasy kit (QIAGEN), and concentration was determined using a spectrophotometer (NanoDrop 1000; Thermo Fisher Scientific). The primer/probe sets for Bmf, CDKN1B, and 18s were obtained from Applied Biosystems. Target mRNAs were amplified by quantitative real-time PCR in 20-µl reactions on the real-time PCR system (PRISM 7900HT; Applied Biosystems) using the One-Step RT-PCR Master Mix (TaqMan; Applied Biosystems). Relative quantities from duplicate amplifications were calculated by normalizing averaged threshold cycle (Ct) values to CDKN1B and/or 18s to obtain ΔCt, and the relative standard curve method was used for determining the fold change as described previously (Schwalm et al., 2008). For qRT-PCR, primers specific for mouse GAPDH, 5′-AGGCCGGTGCTGAGTATGTC-3′ and 5′-TGCCTGCTTCACCACCTTCT-3′, were used for normalizing RNA levels.

Quantitative RT-PCR (qRT-PCR)

Extraction of RNA from cell pellets was performed using the RNeasy kit (QIAGEN), and concentration was determined using a spectrophotometer (NanoDrop 1000; Thermo Fisher Scientific). The primer/probe sets for Bmf, CDKN1B, and 18s were obtained from Applied Biosystems. Target mRNAs were amplified by quantitative real-time PCR in 20-µl reactions on the real-time PCR system (PRISM 7900HT; Applied Biosystems) using the One-Step RT-PCR Master Mix (TaqMan; Applied Biosystems). Relative quantities from duplicate amplifications were calculated by normalizing averaged threshold cycle (Ct) values to CDKN1B and/or 18s to obtain ΔCt, and the relative standard curve method was used for determining the fold change as described previously (Schwalm et al., 2008). For qRT-PCR, primers specific for mouse GAPDH, 5′-AGGCCGGTGCTGAGTATGTC-3′ and 5′-TGCCTGCTTCACCACCTTCT-3′, were used for normalizing RNA levels.

Immunoprecipitation and Western blot analysisTotal protein lysates or cytosolic and nuclear fractions were prepared (Mebratu et al., 2008), and protein was analyzed by Western blotting as previously described (Stout et al., 2007). In brief, cells were lysed in NP-40 to obtain the cytosolic fraction, and nuclear fractions were prepared extracting the nuclear pellet with a hypertonic extraction buffer (50 mM Hepes, pH 7.8, 50 mM KCl, and 300 mM NaCl) in the presence of protease and phosphatase inhibitors. For immunoprecipitation using the cross-link immunoprecipitation kit (Thermo Fisher Scientific), cells were rinsed twice with cold PBS, scraped into cold PBS plus protease inhibitors, and analyzed per the manufacturer’s instructions. We used the rat anti-Bmf monoclonal antibody, a gift from A. Strasser and A. Villunger (Innsbruck Medical University, Innsbruck, Austria) at 2 µg/ml, and the rabbit anti-p53 polyclonal antibody (FL-393; Santa Cruz Biotechnology, Inc.), acetyl-p53 (Lys382; Cell Signaling Technology), rabbit antilamin polyclonal (#2032; Cell Signaling Technology), or rabbit anti-HDAC1 polyclonal (EMD Millipore) were used at a 1:1,000 dilution. We tested various Bcl-2 antibodies, including those obtained from BD (catalog nos. 554279 and 554087) and Santa Cruz Biotechnology, Inc. (sc-492, sc-7382, and sc-578), and found that sc-7382 and sc-578 showed a single 26-kD band representing Bcl-2. Therefore, the rat anti–Bcl-2 monoclonal sc-578 was used for immunoprecipitation and Western blot experiments. The Bcl-2 and the rabbit anti–Bcl-xL polyclonal (sc-634; Santa Cruz Biotechnology, Inc.) antibodies were used at 1:300 and 1:200 dilutions, respectively. For immunoprecipitation experiments, rabbit anti–Beclin-1 (Abcam) was used, but for Western blotting, rabbit anti–Beclin-1 and rabbit anti-LC3B (Cell Signaling Technology) antibodies were both used at 1:1,000. The mouse actin polyclonal antibody was used at 1:5,000 (Santa Cruz Biotechnology, Inc.), and goat anti–rat, goat anti–rabbit secondary, or rabbit anti–mouse antibodies were used for visualizing proteins with chemiluminescence (PerkinElmer) using the image reader (LAS-4000; Fujifilm).

Immunoprecipitation and Western blot analysis

Total protein lysates or cytosolic and nuclear fractions were prepared (Mebratu et al., 2008), and protein was analyzed by Western blotting as previously described (Stout et al., 2007). In brief, cells were lysed in NP-40 to obtain the cytosolic fraction, and nuclear fractions were prepared extracting the nuclear pellet with a hypertonic extraction buffer (50 mM Hepes, pH 7.8, 50 mM KCl, and 300 mM NaCl) in the presence of protease and phosphatase inhibitors. For immunoprecipitation using the cross-link immunoprecipitation kit (Thermo Fisher Scientific), cells were rinsed twice with cold PBS, scraped into cold PBS plus protease inhibitors, and analyzed per the manufacturer’s instructions. We used the rat anti-Bmf monoclonal antibody, a gift from A. Strasser and A. Villunger (Innsbruck Medical University, Innsbruck, Austria) at 2 µg/ml, and the rabbit anti-p53 polyclonal antibody (FL-393; Santa Cruz Biotechnology, Inc.), acetyl-p53 (Lys382; Cell Signaling Technology), rabbit antilamin polyclonal (#2032; Cell Signaling Technology), or rabbit anti-HDAC1 polyclonal (EMD Millipore) were used at a 1:1,000 dilution. We tested various Bcl-2 antibodies, including those obtained from BD (catalog nos. 554279 and 554087) and Santa Cruz Biotechnology, Inc. (sc-492, sc-7382, and sc-578), and found that sc-7382 and sc-578 showed a single 26-kD band representing Bcl-2. Therefore, the rat anti–Bcl-2 monoclonal sc-578 was used for immunoprecipitation and Western blot experiments. The Bcl-2 and the rabbit anti–Bcl-xL polyclonal (sc-634; Santa Cruz Biotechnology, Inc.) antibodies were used at 1:300 and 1:200 dilutions, respectively. For immunoprecipitation experiments, rabbit anti–Beclin-1 (Abcam) was used, but for Western blotting, rabbit anti–Beclin-1 and rabbit anti-LC3B (Cell Signaling Technology) antibodies were both used at 1:1,000. The mouse actin polyclonal antibody was used at 1:5,000 (Santa Cruz Biotechnology, Inc.), and goat anti–rat, goat anti–rabbit secondary, or rabbit anti–mouse antibodies were used for visualizing proteins with chemiluminescence (PerkinElmer) using the image reader (LAS-4000; Fujifilm).

Immunofluorescence analysisFor immunofluorescence, AALEB cells grown on eight-chamber slides (Lab-Tek II; Thermo Fisher Scientific) were fixed using 3% paraformaldehyde with 3% sucrose in PBS and permeabilized using 0.2% Triton X-100 with 0.2% saponin in a blocking solution containing 3% IgG-free BSA, 1% gelatin, and 2% normal donkey serum. Cells were probed with anti-p53 (#sc126; clone DO1; Santa Cruz Biotechnology, Inc.), anti–acetyl-p53 (Lys382; #2525; Cell Signaling Technology), or isotype controls at a 1: 500 dilution. MEFs were probed with anti-p53 (#sc6243; clone FL393; Santa Cruz Biotechnology, Inc.) or antiacetyl-p53 (Lys379; #2570; Cell Signaling Technology). The immunolabeled cells were detected using F(ab)2 fragments of respective secondary antibodies conjugated to Dylight-549 or -649 (Jackson ImmunoResearch Laboratories, Inc.) at a 1:500 dilution and mounted with Fluoromount-G (SouthernBiotech) containing DAPI for nuclear staining. Fluorescently labeled cells were analyzed as described previously (Stout et al., 2007) using an imaging system (Axioplan 2; Carl Zeiss) equipped with a charge-coupled device camera (ORCA-ER; Hamamatsu Photonics) coupled with a wavelength switch (Lambda DG-4; Sutter Instrument) and acquisition software (SlideBook 5; Intelligent Imaging Innovations, Inc.). Quantification of p53-positive or acetyl-p53–positive cells was conducted by counting ≥400 cells/well by a person unaware of the treatment groups.

Immunofluorescence analysis

For immunofluorescence, AALEB cells grown on eight-chamber slides (Lab-Tek II; Thermo Fisher Scientific) were fixed using 3% paraformaldehyde with 3% sucrose in PBS and permeabilized using 0.2% Triton X-100 with 0.2% saponin in a blocking solution containing 3% IgG-free BSA, 1% gelatin, and 2% normal donkey serum. Cells were probed with anti-p53 (#sc126; clone DO1; Santa Cruz Biotechnology, Inc.), anti–acetyl-p53 (Lys382; #2525; Cell Signaling Technology), or isotype controls at a 1: 500 dilution. MEFs were probed with anti-p53 (#sc6243; clone FL393; Santa Cruz Biotechnology, Inc.) or antiacetyl-p53 (Lys379; #2570; Cell Signaling Technology). The immunolabeled cells were detected using F(ab)2 fragments of respective secondary antibodies conjugated to Dylight-549 or -649 (Jackson ImmunoResearch Laboratories, Inc.) at a 1:500 dilution and mounted with Fluoromount-G (SouthernBiotech) containing DAPI for nuclear staining. Fluorescently labeled cells were analyzed as described previously (Stout et al., 2007) using an imaging system (Axioplan 2; Carl Zeiss) equipped with a charge-coupled device camera (ORCA-ER; Hamamatsu Photonics) coupled with a wavelength switch (Lambda DG-4; Sutter Instrument) and acquisition software (SlideBook 5; Intelligent Imaging Innovations, Inc.). Quantification of p53-positive or acetyl-p53–positive cells was conducted by counting ≥400 cells/well by a person unaware of the treatment groups.

Adenoviral and retroviral expression vectorsAdenoviral expression vectors for p53 and BmfCUG and BmfS were developed by cloning the respective cDNAs into the shuttle vector. RNA was isolated from MAECs, and 1.5 µg was converted to cDNA using the cDNA synthesis kit (SuperScript; Invitrogen). RT-PCR was performed using the following primers: BmfS, 5′-AAAATGGAGCCACCTCAGTGTGTG-3′ and 5′-TCACCAGGGCCCCACCCCTTC-3′; BmfCUG, 5′-AAACTGGCGCAGAGCCCTGGCATCA-3′ and 5′-TCACCAGGGCCCCACCCCTTC-3′; and BmfL, 5′-ATGCCCGGAGCGGGCGTATTT-3′ and 5′-TCACCAGGGCCCCACCCCTTC-3′. Shuttle cytomegalovirus (CMV)_p53, shuttle CMV_BmfCUG, and shuttle CMV_BmfS were packaged into virus-producing cells, and Ad particles were harvested by collecting media at the time when cells were rounding up as described previously (Xiang et al., 1996).The retroviral silencing vector encoding for p53 shRNA and the control vector were purchased from OriGene. Retroviral vectors expressing the respective shRNA or control constructs were packaged in Phoenix by transfecting using FuGENE (Promega), and the harvested virus was used to infect AALEB cells as previously described (Mebratu et al., 2008).

Adenoviral and retroviral expression vectors

Adenoviral expression vectors for p53 and BmfCUG and BmfS were developed by cloning the respective cDNAs into the shuttle vector. RNA was isolated from MAECs, and 1.5 µg was converted to cDNA using the cDNA synthesis kit (SuperScript; Invitrogen). RT-PCR was performed using the following primers: BmfS, 5′-AAAATGGAGCCACCTCAGTGTGTG-3′ and 5′-TCACCAGGGCCCCACCCCTTC-3′; BmfCUG, 5′-AAACTGGCGCAGAGCCCTGGCATCA-3′ and 5′-TCACCAGGGCCCCACCCCTTC-3′; and BmfL, 5′-ATGCCCGGAGCGGGCGTATTT-3′ and 5′-TCACCAGGGCCCCACCCCTTC-3′. Shuttle cytomegalovirus (CMV)_p53, shuttle CMV_BmfCUG, and shuttle CMV_BmfS were packaged into virus-producing cells, and Ad particles were harvested by collecting media at the time when cells were rounding up as described previously (Xiang et al., 1996).

The retroviral silencing vector encoding for p53 shRNA and the control vector were purchased from OriGene. Retroviral vectors expressing the respective shRNA or control constructs were packaged in Phoenix by transfecting using FuGENE (Promega), and the harvested virus was used to infect AALEB cells as previously described (Mebratu et al., 2008).

Bmf mRNA half-lifeCells were treated with the RNA polymerase inhibitor DRB (Sigma-Aldrich) at a final concentration of 50 ng/ml for SAOS-2, Calu-3, A549, and Calu-6 and at 100 ng/ml for AALEBs to stop RNA polymerase activity. The relative mRNA abundance was calculated using the ΔΔCt method, and mRNA half-life was calculated using the Greenberg formula by averaging the calculated slope from mRNA levels over time (Greenberg, 1972).

Bmf mRNA half-life

Cells were treated with the RNA polymerase inhibitor DRB (Sigma-Aldrich) at a final concentration of 50 ng/ml for SAOS-2, Calu-3, A549, and Calu-6 and at 100 ng/ml for AALEBs to stop RNA polymerase activity. The relative mRNA abundance was calculated using the ΔΔCt method, and mRNA half-life was calculated using the Greenberg formula by averaging the calculated slope from mRNA levels over time (Greenberg, 1972).

ChIPAfter treating with 50 ng/ml IFN-γ for 48 h, cells were fixed with 1% formaldehyde, and the reaction was quenched using 1.25 M glycine and scraped into cold PBS containing protease inhibitors. ChIP was performed using the Magna ChIP G kit (MAGNA0002; EMD Millipore) as described by the manufacturer. Sonicated nuclear fractions were incubated with mouse IgG1 monoclonal antibody to p53 (sc-98 Pab1801; Santa Cruz Biotechnology, Inc.), rabbit polyclonal antibody to HDAC1 (06–720; EMD Millipore), rabbit polyclonal antibody to acetyl-H4 (06–598; EMD Millipore), rabbit polyclonal antibody to acetyl-H3 (06-599B; EMD Millipore), mouse IgG1, or rabbit as a control (CBL600; EMD Millipore). DNA identification was confirmed with PCR using the following primers specific for the Bmf promoter: region −560, 5′-ACCTAAGGGCTCCCCTGGA-3′ and 5′-GCAGGTCGGAAGAAAACTGCAGC-3′, and region −97, 5′-TTGGCGCTTCACTCGCCATT-3′ and 5′-ATCCCGCAAACAGCTGAT-3′.

After treating with 50 ng/ml IFN-γ for 48 h, cells were fixed with 1% formaldehyde, and the reaction was quenched using 1.25 M glycine and scraped into cold PBS containing protease inhibitors. ChIP was performed using the Magna ChIP G kit (MAGNA0002; EMD Millipore) as described by the manufacturer. Sonicated nuclear fractions were incubated with mouse IgG1 monoclonal antibody to p53 (sc-98 Pab1801; Santa Cruz Biotechnology, Inc.), rabbit polyclonal antibody to HDAC1 (06–720; EMD Millipore), rabbit polyclonal antibody to acetyl-H4 (06–598; EMD Millipore), rabbit polyclonal antibody to acetyl-H3 (06-599B; EMD Millipore), mouse IgG1, or rabbit as a control (CBL600; EMD Millipore). DNA identification was confirmed with PCR using the following primers specific for the Bmf promoter: region −560, 5′-ACCTAAGGGCTCCCCTGGA-3′ and 5′-GCAGGTCGGAAGAAAACTGCAGC-3′, and region −97, 5′-TTGGCGCTTCACTCGCCATT-3′ and 5′-ATCCCGCAAACAGCTGAT-3′.

Transmission electron microscopyCells used for electron microscopy analysis were fixed in 2.5% glutaraldehyde and postfixed with 1% osmium tetroxide before being dehydrated in ethanol. Tissues were infiltrated with propylene oxide and embedded in Agar 100 resin for preparation of ultrathin sections. After staining with uranyl acetate and lead citrate, sections were examined using a transmission electron microscope (JEM 1210; JEOL) at 80 or 60 kV onto electron microscope film (ESTAR Thick Base; Kodak). Electron micrographs (n = 30) for each sample were quantified for mitochondria, and autophagosome number and relative area were calculated using ImageJ software (National Institutes of Health).

Transmission electron microscopy

Cells used for electron microscopy analysis were fixed in 2.5% glutaraldehyde and postfixed with 1% osmium tetroxide before being dehydrated in ethanol. Tissues were infiltrated with propylene oxide and embedded in Agar 100 resin for preparation of ultrathin sections. After staining with uranyl acetate and lead citrate, sections were examined using a transmission electron microscope (JEM 1210; JEOL) at 80 or 60 kV onto electron microscope film (ESTAR Thick Base; Kodak). Electron micrographs (n = 30) for each sample were quantified for mitochondria, and autophagosome number and relative area were calculated using ImageJ software (National Institutes of Health).

Statistical analysisFold changes or scanned density values were averaged and compared for significance between groups using the Student’s t test. Data were analyzed using Prism statistical analysis software (GraphPad Software), and P < 0.05 was considered statistically significant.

Statistical analysis

Fold changes or scanned density values were averaged and compared for significance between groups using the Student’s t test. Data were analyzed using Prism statistical analysis software (GraphPad Software), and P < 0.05 was considered statistically significant.
