---
pmid: '20559380'
title: Diphthamide biosynthesis requires an organic radical generated by an iron-sulphur
  enzyme.
authors:
- Zhang Y
- Zhu X
- Torelli AT
- Lee M
- Dzikovski B
- Koralewski RM
- Wang E
- Freed J
- Krebs C
- Ealick SE
- Lin H
journal: Nature
year: '2010'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3006227
doi: 10.1038/nature09138
---

# Diphthamide biosynthesis requires an organic radical generated by an iron-sulphur enzyme.
**Authors:** Zhang Y, Zhu X, Torelli AT, Lee M, Dzikovski B, Koralewski RM, Wang E, Freed J, Krebs C, Ealick SE, Lin H
**Journal:** Nature (2010)
**DOI:** [10.1038/nature09138](https://doi.org/10.1038/nature09138)
**PMC:** [PMC3006227](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3006227/)

## Abstract

1. Nature. 2010 Jun 17;465(7300):891-6. doi: 10.1038/nature09138.

Diphthamide biosynthesis requires an organic radical generated by an 
iron-sulphur enzyme.

Zhang Y(1), Zhu X, Torelli AT, Lee M, Dzikovski B, Koralewski RM, Wang E, Freed 
J, Krebs C, Ealick SE, Lin H.

Author information:
(1)Department of Chemistry and Chemical Biology, Cornell University, Ithaca, New 
York 14853, USA.

Erratum in
    Nature. 2011 May 26;473(7348):544.

Comment in
    Nature. 2010 Jun 17;465(7300):877-8. doi: 10.1038/465877a.

Archaeal and eukaryotic translation elongation factor 2 contain a unique 
post-translationally modified histidine residue called diphthamide, which is the 
target of diphtheria toxin. The biosynthesis of diphthamide was proposed to 
involve three steps, with the first being the formation of a C-C bond between 
the histidine residue and the 3-amino-3-carboxypropyl group of 
S-adenosyl-l-methionine (SAM). However, further details of the biosynthesis 
remain unknown. Here we present structural and biochemical evidence showing that 
the first step of diphthamide biosynthesis in the archaeon Pyrococcus horikoshii 
uses a novel iron-sulphur-cluster enzyme, Dph2. Dph2 is a homodimer and each of 
its monomers can bind a [4Fe-4S] cluster. Biochemical data suggest that unlike 
the enzymes in the radical SAM superfamily, Dph2 does not form the canonical 
5'-deoxyadenosyl radical. Instead, it breaks the C(gamma,Met)-S bond of SAM and 
generates a 3-amino-3-carboxypropyl radical. Our results suggest that P. 
horikoshii Dph2 represents a previously unknown, SAM-dependent, 
[4Fe-4S]-containing enzyme that catalyses unprecedented chemistry.

DOI: 10.1038/nature09138
PMCID: PMC3006227
PMID: 20559380 [Indexed for MEDLINE]

## Full Text

Corynebacterium diphtheriae is a pathogenic bacterium that causes the infectious disease diphtheria in humans.1 This bacterium kills host cells by secreting a protein factor, diphtheria toxin2, which catalyzes the ADP-ribosylation of a posttranslationally modified histidine residue (Figure 1) in eukaryotic translation elongation factor 2 (eEF2).3 Because this posttranslational modification is the target of diphtheria toxin, it was named “diphthamide”. eEF2 is a GTPase required for the translocation step of ribosomal protein synthesis.4 The diphthamide modification is conserved in all eukaryotes and archaea and is important for ribosomal protein synthesis.4, 5 Although diphthamide was identified more than three decades ago, its biosynthesis has remained an enigma.6 Five genes required for diphthamide biosynthesis were identified in eukaryotes, DPH1, DPH2, DPH3, DPH4, and DPH5,3, 7–13 and a biosynthetic pathway has been proposed (Figure 1).

The first step of diphthamide biosynthesis is the transfer of the 3-amino-3-carboxypropyl (ACP) group from S-adenosylmethionine (SAM) to the C-2 position of the imidazole ring of the target histidine residue in eEF2 and is catalyzed by DPH1–4 in eukaryotes. This step is followed by a trimethylation, catalyzed by DPH5, and an amidation, catalyzed by an unidentified enzyme. The first step is particularly interesting for several reasons. First, SAM is generally a methyl donor, but in the first step the ACP group is transferred from SAM. Second, protein posttranslational modifications that involve C-C bond formation are rare6 and in diphthamide biosynthesis the C-C bond formation involves the poorly nucleophilic C-2 of the imidazole ring. Third, in eukaryotes, this reaction requires four proteins, DPH1–4, raising questions about the function of each protein.

DPH1 and DPH2 share about 20% sequence identity, but are not similar to any other protein with known function. Iterative BLAST searches14 starting with Saccharomyces cerevisiae DPH1 or DPH2 generate both proteins from other eukaryotic species. In contrast, BLAST searches identify only one protein, Dph2, in archaeal species. Archaeal Dph2s are more similar to eukaryotic DPH1 than to DPH2. Eukaryotic DPH3 and DPH4 have no orthologs in archaea based on BLAST searches. To better understand diphthamide biosynthesis, we initially attempted to reconstitute the first step using Pyrococcus horikoshii Dph2 (PhDph2) and translation elongation factor 2 (PhEF2) under aerobic conditions without success. The X-ray crystal structure of PhDph2 revealed an intriguing constellation of three conserved cysteine residues -- each from a different structural domain -- suggestive of an iron-sulfur cluster. Subsequently, PhDph2 activity was reconstituted in the presence of dithionite under anaerobic conditions. A crystal structure of reconstituted PhDph2 along with UV-Vis, EPR, and Mössbauer spectroscopies confirmed the presence of a [4Fe-4S] cluster. Detailed biochemical characterization suggests that the PhDph2-catalyzed reaction involves a 3-amino-3-carboxypropyl radical intermediate. The data suggest that PhDph2 is a novel SAM-dependent [4Fe-4S]-containing enzyme15 that catalyzes unprecedented chemistry.

PhDph2 is aerobically inactivePhDph2 and PhEF2 were expressed in Escherichia coli and purified under aerobic conditions. No activity was observed when using these proteins to reconstitute the first step of diphthamide biosynthesis. One explanation for the lack of activity is that the reaction requires an oxygen-sensitive cofactor and another is that additional proteins or small molecules are required. In the latter case the additional proteins might be orthologs of eukaryotic DPH3 and DPH4; however, attempts to reconstitute activity under similar conditions using yeast DPH1–4 and eEF2 were also unsuccessful (data not shown).

PhDph2 is aerobically inactive

PhDph2 and PhEF2 were expressed in Escherichia coli and purified under aerobic conditions. No activity was observed when using these proteins to reconstitute the first step of diphthamide biosynthesis. One explanation for the lack of activity is that the reaction requires an oxygen-sensitive cofactor and another is that additional proteins or small molecules are required. In the latter case the additional proteins might be orthologs of eukaryotic DPH3 and DPH4; however, attempts to reconstitute activity under similar conditions using yeast DPH1–4 and eEF2 were also unsuccessful (data not shown).

Crystal structure of PhDph2To provide structural insight into the catalytic mechanism of PhDph2, we determined its X-ray crystal structure at 2.3 Å resolution using selenomethionine (SeMet) single anomalous diffraction (SAD) phasing. The structure showed that PhDph2 is a homodimer (Figure 2). Each PhDph2 monomer consists of three domains with all three domains sharing the same overall fold. The basic domain fold is a four-stranded parallel β-sheet with three flanking α-helices (or two α-helices and one 310 helix in the case of domain 2) (Supplementary Figure 1). The two β-sheets in domain 1 and 2 each contain an additional β-strand that is antiparallel to the rest of the β-sheet. Domains 2 and 3 have two additional α-helices. Domain 1 of one monomer and domain 3 of the adjacent monomer form the dimer interface, creating an extended nine-stranded β-sheet. The domain folds and their arrangement resemble the structure of quinolinate synthase16; however, the orientations of the domains with respect to each other are different in the two enzymes (Supplementary Figure 2). Three conserved cysteine residues (Cys59, Cys163 and Cys287), each coming from a different structural domain, are clustered together in the center of the PhDph2 monomers. All three cysteine residues are conserved in eukaryotic DPH1s. The first and third cysteine residues are conserved in eukaryotic DPH2s (Supplementary Figure 3).

Crystal structure of PhDph2

To provide structural insight into the catalytic mechanism of PhDph2, we determined its X-ray crystal structure at 2.3 Å resolution using selenomethionine (SeMet) single anomalous diffraction (SAD) phasing. The structure showed that PhDph2 is a homodimer (Figure 2). Each PhDph2 monomer consists of three domains with all three domains sharing the same overall fold. The basic domain fold is a four-stranded parallel β-sheet with three flanking α-helices (or two α-helices and one 310 helix in the case of domain 2) (Supplementary Figure 1). The two β-sheets in domain 1 and 2 each contain an additional β-strand that is antiparallel to the rest of the β-sheet. Domains 2 and 3 have two additional α-helices. Domain 1 of one monomer and domain 3 of the adjacent monomer form the dimer interface, creating an extended nine-stranded β-sheet. The domain folds and their arrangement resemble the structure of quinolinate synthase16; however, the orientations of the domains with respect to each other are different in the two enzymes (Supplementary Figure 2). Three conserved cysteine residues (Cys59, Cys163 and Cys287), each coming from a different structural domain, are clustered together in the center of the PhDph2 monomers. All three cysteine residues are conserved in eukaryotic DPH1s. The first and third cysteine residues are conserved in eukaryotic DPH2s (Supplementary Figure 3).

Reconstitution of PhDph2 activityThe clustering of the three cysteine residues in the crystal structure and the requirement for SAM raised the possibility that PhDph2 utilizes a [4Fe-4S] cluster.15 Radical SAM enzymes harbor a [4Fe-4S] cluster coordinated by three cysteines in a CX3CX2C motif17, although variations of this motif have been reported,18,24 to generate a 5′-deoxyadenosyl radical. Since [4Fe-4S] clusters are typically oxygen-sensitive, PhDph2 was purified and assayed anaerobically. Using 14C-SAM, we showed that PhEF2 can be labeled in the presence of PhDph2 (Figure 3a, lane 6), but not in the absence of PhDph2 (lane 3) or dithionite (lane 7). When His600 of PhEF2, the site of the diphthamide modification, was changed to alanine, no reaction occurred in the presence of PhDph2 (lane 5). MALDI-MS of the PhEF2 protein confirmed that an ACP group was added after the reaction (Figure 3b). These results suggest the possibility that PhDph2 is a SAM-dependent Fe-S enzyme and demonstrate that no other enzyme is required for the first step of diphthamide biosynthesis in vitro.

Reconstitution of PhDph2 activity

The clustering of the three cysteine residues in the crystal structure and the requirement for SAM raised the possibility that PhDph2 utilizes a [4Fe-4S] cluster.15 Radical SAM enzymes harbor a [4Fe-4S] cluster coordinated by three cysteines in a CX3CX2C motif17, although variations of this motif have been reported,18,24 to generate a 5′-deoxyadenosyl radical. Since [4Fe-4S] clusters are typically oxygen-sensitive, PhDph2 was purified and assayed anaerobically. Using 14C-SAM, we showed that PhEF2 can be labeled in the presence of PhDph2 (Figure 3a, lane 6), but not in the absence of PhDph2 (lane 3) or dithionite (lane 7). When His600 of PhEF2, the site of the diphthamide modification, was changed to alanine, no reaction occurred in the presence of PhDph2 (lane 5). MALDI-MS of the PhEF2 protein confirmed that an ACP group was added after the reaction (Figure 3b). These results suggest the possibility that PhDph2 is a SAM-dependent Fe-S enzyme and demonstrate that no other enzyme is required for the first step of diphthamide biosynthesis in vitro.

Characterization of the [4Fe-4S] clusterThe anaerobically purified PhDph2 contains 1.3 ± 0.2 and 1.9 ± 0.2 equivalents of iron and sulfur per polypeptide, respectively, and displays a broad absorption band at ~400 nm, which disappears upon reduction by 0.5 mM dithionite (Figure 4a). The 400 nm absorption is typical of a [4Fe-4S]2+ cluster. Quantification based on the 400 nm absorption suggests the presence of ~0.3 [4Fe-4S]2+ per PhDph2. EPR spectra of dithionite-reduced PhDph2 are shown in Figure 4b. The g-values (2.03, 1.92, and 1.86) and the temperature-dependence are typical of a [4Fe-4S]+ cluster.19–22 Quantification of the EPR spectrum suggests the presence of ~0.3 [4Fe-4S]+ per PhDph2.The 57Fe-enriched anaerobically-isolated PhDph2 contains 2.0 ± 0.2 and 2.1 ± 0.2 equivalents of iron and sulfur per polypeptide, respectively. The 4.2-K/53-mT Mössbauer spectrum (Figure 4c) is dominated by a quadrupole doublet with parameters (isomer shift (δ) of 0.43 mm/s and quadrupole splitting parameter (ΔEQ) of 1.13 mm/s) typical of [4Fe-4S]2+ clusters (solid line in Figure 4c, 73% of total intensity). The weak absorption peak labeled (a) suggests the presence of a small amount (~10%) of high-spin Fe(II), which is presumably nonspecifically bound to the protein. The shoulder labeled (b) belongs to a quadrupole doublet (~15% intensity), the left line of which contributes to the prominent peak at −0.2 mm/s. Although the nature of the Fe species that gives rise to this absorption is not known, similar spectral features were observed for a sample of P. horikoshii quinolinate synthase,23 which is structurally similar to PhDph2 and also harbors a [4Fe-4S] cluster. Thus, all the spectroscopic data indicate that PhDph2 contains a [4Fe-4S] cluster.Brown crystals of the anaerobically purified PhDph2 were obtained that belong to the same space group as the inactive PhDph2. A crystal structure determined to 2.1 Å resolution showed clear electron density for a [4Fe-4S] cluster (Figure 4d and Supplementary Figure 4). Refinement of the PhDph2 structure with a [4Fe-4S] cluster included gave final R and Rfree values of 20.4% and 25.2%, respectively (Supplementary Table 1).

Characterization of the [4Fe-4S] cluster

The anaerobically purified PhDph2 contains 1.3 ± 0.2 and 1.9 ± 0.2 equivalents of iron and sulfur per polypeptide, respectively, and displays a broad absorption band at ~400 nm, which disappears upon reduction by 0.5 mM dithionite (Figure 4a). The 400 nm absorption is typical of a [4Fe-4S]2+ cluster. Quantification based on the 400 nm absorption suggests the presence of ~0.3 [4Fe-4S]2+ per PhDph2. EPR spectra of dithionite-reduced PhDph2 are shown in Figure 4b. The g-values (2.03, 1.92, and 1.86) and the temperature-dependence are typical of a [4Fe-4S]+ cluster.19–22 Quantification of the EPR spectrum suggests the presence of ~0.3 [4Fe-4S]+ per PhDph2.

The 57Fe-enriched anaerobically-isolated PhDph2 contains 2.0 ± 0.2 and 2.1 ± 0.2 equivalents of iron and sulfur per polypeptide, respectively. The 4.2-K/53-mT Mössbauer spectrum (Figure 4c) is dominated by a quadrupole doublet with parameters (isomer shift (δ) of 0.43 mm/s and quadrupole splitting parameter (ΔEQ) of 1.13 mm/s) typical of [4Fe-4S]2+ clusters (solid line in Figure 4c, 73% of total intensity). The weak absorption peak labeled (a) suggests the presence of a small amount (~10%) of high-spin Fe(II), which is presumably nonspecifically bound to the protein. The shoulder labeled (b) belongs to a quadrupole doublet (~15% intensity), the left line of which contributes to the prominent peak at −0.2 mm/s. Although the nature of the Fe species that gives rise to this absorption is not known, similar spectral features were observed for a sample of P. horikoshii quinolinate synthase,23 which is structurally similar to PhDph2 and also harbors a [4Fe-4S] cluster. Thus, all the spectroscopic data indicate that PhDph2 contains a [4Fe-4S] cluster.

Brown crystals of the anaerobically purified PhDph2 were obtained that belong to the same space group as the inactive PhDph2. A crystal structure determined to 2.1 Å resolution showed clear electron density for a [4Fe-4S] cluster (Figure 4d and Supplementary Figure 4). Refinement of the PhDph2 structure with a [4Fe-4S] cluster included gave final R and Rfree values of 20.4% and 25.2%, respectively (Supplementary Table 1).

Reaction mechanismTo explore the PhDph2 reaction mechanism, HPLC was used to analyze the reaction products. In the reaction that contained SAM, PhDph2, PhEF2 and dithionite, most SAM molecules were converted to 5′-deoxy-5′-methylthioadenosine (MTA, Figure 5a). In control reactions without PhDph2 or dithionite, only low levels of MTA were observed and most SAM molecules were left intact. This is consistent with the activity assay results shown in Figure 3. Cleavage of the C5′-S bond of SAM did not occur because the formation of 5′-deoxyadenosine (the most likely product of the adenosyl moiety) was not observed. Collectively, the results suggest that PhDph2 catalyzes the cleavage of the Cγ,Met-S bond of SAM only in the presence of reductant, transfers the ACP group to PhEF2, and releases the remaining MTA.Two different mechanisms can be proposed for the PhDph2-catalyzed cleavage of the Cγ,Met-S bond of SAM. One is that the [4Fe-4S]+ cluster provides one electron to reductively cleave the Cγ,Met-S bond of SAM, forming MTA, an ACP radical, and the oxidized [4Fe-4S]2+ cluster (Supplementary Figure 5a). Alternatively, the [4Fe-4S] cluster in PhDph2 binds SAM and orients it correctly for nucleophilic attack by the C2 of the imidazole ring (Supplementary Figure 5b), leading to the formation of products. Further evidence to differentiate these two possibilities was provided by the identification of the product derived from the ACP group in the reaction without PhEF2. In the absence of PhEF2, PhDph2 can still cleave the Cγ,Met-S bond of SAM, generating MTA (Figure 5a). The fate of the ACP group was interrogated by 1H-NMR (Figure 5b). In the reaction containing PhDph2, SAM, and dithionite, several new peaks were observed, which were not observed in control samples without dithionite or PhDph2 (Figure 5b). These peaks were assigned to two products: 2-aminobutyric acid (ABA) and homocysteine sulfinic acid (HSA). The NMR spectra of authentic samples of ABA and HSA confirmed these assignments (Figure 5b). In Supplementary Figure 6, these NMR spectra were compared to those of homoserine, homoserine lactone, and SAM, ruling out the possibility that PhDph2 catalyzes the formation of homoserine or homoserine lactone via a nucleophilic mechanism.To further validate these results, the reaction mixtures were purified by TLC, dansylated, and subsequently analyzed by LCMS. The structures and molecular weights of the dansylated compounds are shown in Supplementary Figure 7. In the control reaction without PhDPh2, the formation of dansylated homoserine (m/z 337, MH+, retention time 18.35 min) was observed (Figure 5c and Supplementary Figure 8), which is consistent with the NMR results (Supplementary Figure 6). In the reaction with PhDph2, SAM, and dithionite, the formation of dansylated homoserine was suppressed compared with the control. Instead, dansylated ABA (m/z 337, MH+, 23.60 min) and HSA (m/z 401, MH+, 16.65 min) were observed (Figure 5c and Supplementary Figure 8). Dansylated homoserine lactone and ABA have the same retention time, but can be differentiated by their m/z-values (337 and 335 for ABA and homoserine lactone, respectively, Supplementary Figure 9). During the TLC purification and dansylation reaction, HSA was partially oxidized to homocysteine sulfonic acid, as evidenced by the ion with m/z 417 (MH+, Figure 5c and Supplementary Figure 8). Overall, the results from the LCMS analysis and NMR analysis demonstrate that PhDph2 catalyzes the formation of ABA and HAS in the absence of PhEF2. The formation of ABA and HSA can be best explained by the generation of an ACP radical followed by hydrogen extraction to give ABA or quenching by dithionite to give HSA (Figure 6).

Reaction mechanism

To explore the PhDph2 reaction mechanism, HPLC was used to analyze the reaction products. In the reaction that contained SAM, PhDph2, PhEF2 and dithionite, most SAM molecules were converted to 5′-deoxy-5′-methylthioadenosine (MTA, Figure 5a). In control reactions without PhDph2 or dithionite, only low levels of MTA were observed and most SAM molecules were left intact. This is consistent with the activity assay results shown in Figure 3. Cleavage of the C5′-S bond of SAM did not occur because the formation of 5′-deoxyadenosine (the most likely product of the adenosyl moiety) was not observed. Collectively, the results suggest that PhDph2 catalyzes the cleavage of the Cγ,Met-S bond of SAM only in the presence of reductant, transfers the ACP group to PhEF2, and releases the remaining MTA.

Two different mechanisms can be proposed for the PhDph2-catalyzed cleavage of the Cγ,Met-S bond of SAM. One is that the [4Fe-4S]+ cluster provides one electron to reductively cleave the Cγ,Met-S bond of SAM, forming MTA, an ACP radical, and the oxidized [4Fe-4S]2+ cluster (Supplementary Figure 5a). Alternatively, the [4Fe-4S] cluster in PhDph2 binds SAM and orients it correctly for nucleophilic attack by the C2 of the imidazole ring (Supplementary Figure 5b), leading to the formation of products. Further evidence to differentiate these two possibilities was provided by the identification of the product derived from the ACP group in the reaction without PhEF2. In the absence of PhEF2, PhDph2 can still cleave the Cγ,Met-S bond of SAM, generating MTA (Figure 5a). The fate of the ACP group was interrogated by 1H-NMR (Figure 5b). In the reaction containing PhDph2, SAM, and dithionite, several new peaks were observed, which were not observed in control samples without dithionite or PhDph2 (Figure 5b). These peaks were assigned to two products: 2-aminobutyric acid (ABA) and homocysteine sulfinic acid (HSA). The NMR spectra of authentic samples of ABA and HSA confirmed these assignments (Figure 5b). In Supplementary Figure 6, these NMR spectra were compared to those of homoserine, homoserine lactone, and SAM, ruling out the possibility that PhDph2 catalyzes the formation of homoserine or homoserine lactone via a nucleophilic mechanism.

To further validate these results, the reaction mixtures were purified by TLC, dansylated, and subsequently analyzed by LCMS. The structures and molecular weights of the dansylated compounds are shown in Supplementary Figure 7. In the control reaction without PhDPh2, the formation of dansylated homoserine (m/z 337, MH+, retention time 18.35 min) was observed (Figure 5c and Supplementary Figure 8), which is consistent with the NMR results (Supplementary Figure 6). In the reaction with PhDph2, SAM, and dithionite, the formation of dansylated homoserine was suppressed compared with the control. Instead, dansylated ABA (m/z 337, MH+, 23.60 min) and HSA (m/z 401, MH+, 16.65 min) were observed (Figure 5c and Supplementary Figure 8). Dansylated homoserine lactone and ABA have the same retention time, but can be differentiated by their m/z-values (337 and 335 for ABA and homoserine lactone, respectively, Supplementary Figure 9). During the TLC purification and dansylation reaction, HSA was partially oxidized to homocysteine sulfonic acid, as evidenced by the ion with m/z 417 (MH+, Figure 5c and Supplementary Figure 8). Overall, the results from the LCMS analysis and NMR analysis demonstrate that PhDph2 catalyzes the formation of ABA and HAS in the absence of PhEF2. The formation of ABA and HSA can be best explained by the generation of an ACP radical followed by hydrogen extraction to give ABA or quenching by dithionite to give HSA (Figure 6).

DiscussionThe biochemical, structural and spectroscopic data presented here establish that PhDph2 is a novel [4Fe-4S] cluster enzyme. PhDph2 cleaves the Cγ,Met-S bond of SAM to MTA and transfers the ACP group to His600 of PhEF2. This reaction is strictly dependent on the presence of reductant. In the absence of the natural substrate, PhEF2, the ACP moiety is trapped either as ABA or as HSA, which suggests the intermediacy of an ACP radical. The reductive cleavage of SAM to a thioether and an alkyl radical by a reduced [4Fe-4S]+ cluster is the hallmark feature of the superfamily of radical SAM enzymes.15 However, there are two crucial differences between the radical SAM enzymes and PhDph2. First, the radical SAM enzymes exclusively cleave the C5′-S bond to generate methionine and a 5′-deoxyadenosyl radical, which is used for a variety of downstream C-H cleavage reactions. Second, the radical SAM superfamily is characterized by a conserved CX3CX2C motif17 (or CX2CX4C in ThiC18, or CX5CX2C in HmdA24) that binds the [4Fe-4S] cluster. This motif is not present in PhDph2. Instead the three conserved cysteine residues are located in separate structural domains and are separated by more than 100 residues in the sequence. Consequently, the three-dimensional structure of PhDph2 is distinct from the structures of the known radical SAM enzymes BioB,25 HemN,26 LAM,27 MoaA,28 PFL-AE,29 and ThiC18, which all have β-barrel or modified β-barrel folds. PhDph2 is structurally similar to quinolinate synthase,16 which is also composed of three structurally homologous domains in a triangular arrangement. The triangular arrangement of domains in PhDph2 positions the three conserved cysteine residues in the central cavity to bind the [4Fe-4S] cluster. In quinolinate synthase, the three conserved cysteine residues required to bind the cluster are also widely separated in the amino acid sequence and located in different domains.30,22 However, quinolinate synthase is not SAM-dependent and its proposed role is in the dehydration of the penultimate precursor of quinolinate.22 In addition, the IspH enzyme in the non-mevalonate pathway for isoprenoid biosynthesis also uses a similar triangular arrangement to bind a [3Fe-4S] cluster.31It is likely that the different reaction outcome, i.e. cleavage of the C5′-S bond in the radical SAM enzymes vs. cleavage of the Cγ,Met-S bond in PhDph2, is controlled by different orientations of SAM relative to the [4Fe-4S] cluster. In the radical SAM enzymes, the amino and carboxyl groups of SAM coordinate to the non-cysteine-ligated Fe site of the [4Fe-4S] cluster.32 Future structural and spectroscopic studies are required to investigate how SAM is bound at the active site of PhDph2.Our data demonstrated that PhDph2 is the only gene product required to catalyze the first step of diphthamide biosynthesis in vitro. In contrast, biosynthesis of diphthamide in eukaryotes requires four gene products, DPH1–4. Studies on PhDph2 provide important insight into the functions of eukaryotic DPH1–4. The crystal structure shows that PhDph2 is a homodimer. Eukaryotic DPH1 and DPH2 are both homologous to each other and to archaeal Dph2. In addition, DPH1 and DPH2 in eukaryotes form a heterodimer.3, 33–35 Therefore it is possible that the eukaryotic DPH1–DPH2 heterodimer is structurally homologous to the PhDph2 homodimer. The three cysteine residues required to bind the cluster are conserved in DPH1 and two of the cysteine residues are conserved in DPH2. Thus the heterodimer of DPH1–DPH2 should at least bind one [4Fe-4S] cluster and may be sufficient to catalyze the first step in vitro. DPH2, which only has two of the conserved Cys residues, could either have a different catalytic function than DPH1 or could be regulatory. In vivo, DPH3 and DPH4 are also required for diphthamide biosynthesis.3 These gene products may be required to keep the [4Fe-4S] cluster in a reduced state. This hypothesis is supported by the observation that DPH3 can bind iron and is redox active.36 Alternatively DPH3 and DPH4 may be required for proper assembly of the [4Fe-4S] clusters. The Fe-S cluster assembly pathways in bacteria and mitochondria of eukaryotes are known to involve J domain-containing co-chaperone proteins, such as bacterial HscB and yeast JAC137, 38, that are similar to DPH4. Confirmation of these functional assignments awaits detailed biochemical and structural studies.

The biochemical, structural and spectroscopic data presented here establish that PhDph2 is a novel [4Fe-4S] cluster enzyme. PhDph2 cleaves the Cγ,Met-S bond of SAM to MTA and transfers the ACP group to His600 of PhEF2. This reaction is strictly dependent on the presence of reductant. In the absence of the natural substrate, PhEF2, the ACP moiety is trapped either as ABA or as HSA, which suggests the intermediacy of an ACP radical. The reductive cleavage of SAM to a thioether and an alkyl radical by a reduced [4Fe-4S]+ cluster is the hallmark feature of the superfamily of radical SAM enzymes.15 However, there are two crucial differences between the radical SAM enzymes and PhDph2. First, the radical SAM enzymes exclusively cleave the C5′-S bond to generate methionine and a 5′-deoxyadenosyl radical, which is used for a variety of downstream C-H cleavage reactions. Second, the radical SAM superfamily is characterized by a conserved CX3CX2C motif17 (or CX2CX4C in ThiC18, or CX5CX2C in HmdA24) that binds the [4Fe-4S] cluster. This motif is not present in PhDph2. Instead the three conserved cysteine residues are located in separate structural domains and are separated by more than 100 residues in the sequence. Consequently, the three-dimensional structure of PhDph2 is distinct from the structures of the known radical SAM enzymes BioB,25 HemN,26 LAM,27 MoaA,28 PFL-AE,29 and ThiC18, which all have β-barrel or modified β-barrel folds. PhDph2 is structurally similar to quinolinate synthase,16 which is also composed of three structurally homologous domains in a triangular arrangement. The triangular arrangement of domains in PhDph2 positions the three conserved cysteine residues in the central cavity to bind the [4Fe-4S] cluster. In quinolinate synthase, the three conserved cysteine residues required to bind the cluster are also widely separated in the amino acid sequence and located in different domains.30,22 However, quinolinate synthase is not SAM-dependent and its proposed role is in the dehydration of the penultimate precursor of quinolinate.22 In addition, the IspH enzyme in the non-mevalonate pathway for isoprenoid biosynthesis also uses a similar triangular arrangement to bind a [3Fe-4S] cluster.31

It is likely that the different reaction outcome, i.e. cleavage of the C5′-S bond in the radical SAM enzymes vs. cleavage of the Cγ,Met-S bond in PhDph2, is controlled by different orientations of SAM relative to the [4Fe-4S] cluster. In the radical SAM enzymes, the amino and carboxyl groups of SAM coordinate to the non-cysteine-ligated Fe site of the [4Fe-4S] cluster.32 Future structural and spectroscopic studies are required to investigate how SAM is bound at the active site of PhDph2.

Our data demonstrated that PhDph2 is the only gene product required to catalyze the first step of diphthamide biosynthesis in vitro. In contrast, biosynthesis of diphthamide in eukaryotes requires four gene products, DPH1–4. Studies on PhDph2 provide important insight into the functions of eukaryotic DPH1–4. The crystal structure shows that PhDph2 is a homodimer. Eukaryotic DPH1 and DPH2 are both homologous to each other and to archaeal Dph2. In addition, DPH1 and DPH2 in eukaryotes form a heterodimer.3, 33–35 Therefore it is possible that the eukaryotic DPH1–DPH2 heterodimer is structurally homologous to the PhDph2 homodimer. The three cysteine residues required to bind the cluster are conserved in DPH1 and two of the cysteine residues are conserved in DPH2. Thus the heterodimer of DPH1–DPH2 should at least bind one [4Fe-4S] cluster and may be sufficient to catalyze the first step in vitro. DPH2, which only has two of the conserved Cys residues, could either have a different catalytic function than DPH1 or could be regulatory. In vivo, DPH3 and DPH4 are also required for diphthamide biosynthesis.3 These gene products may be required to keep the [4Fe-4S] cluster in a reduced state. This hypothesis is supported by the observation that DPH3 can bind iron and is redox active.36 Alternatively DPH3 and DPH4 may be required for proper assembly of the [4Fe-4S] clusters. The Fe-S cluster assembly pathways in bacteria and mitochondria of eukaryotes are known to involve J domain-containing co-chaperone proteins, such as bacterial HscB and yeast JAC137, 38, that are similar to DPH4. Confirmation of these functional assignments awaits detailed biochemical and structural studies.

METHODS SUMMARYCrystallization, data collection, and structure determinationSeMet PhDph2 and anaerobically reconstituted PhDph2 were crystallized using the hanging drop vapor diffusion method. The X-ray data were collected at the NE-CAT beamlines at the Advanced Photon Source (APS). The structures of iron-free PhDph2 and reconstituted PhDph2 were determined by SeMet SAD phasing and the Fourier synthesis, respectively.Reconstitution of activityReconstituted PhDph2 was prepared by growing cells in LB media supplemented with FeCl3, Fe(NH4)2(SO4)2, and L-Cys, and purified using Ni-NTA affinity chromatography anaerobically. The reaction was monitored using carboxyl-14C SAM.Analysis of reaction productsThe formation of MTA was detected by HPLC. Modification of PhEF2 His600 was confirmed by MALDI-MS after trypsin digestion. The products derived from the ACP group were detected by NMR directly and by LCMS after dansylation.EPR and Mössbauer spectroscopyESR spectra were recorded on a Bruker EMX spectrometer at a frequency of 9.24 GHz under standard conditions. Mössbauer spectra were recorded on a spectrometer from WEB research (Edina, MN) operating in the constant acceleration mode in transmission geometry.Full Methods and any associated references are available in the online version of the paper at www.nature.com/nature.

METHODS SUMMARY

Crystallization, data collection, and structure determinationSeMet PhDph2 and anaerobically reconstituted PhDph2 were crystallized using the hanging drop vapor diffusion method. The X-ray data were collected at the NE-CAT beamlines at the Advanced Photon Source (APS). The structures of iron-free PhDph2 and reconstituted PhDph2 were determined by SeMet SAD phasing and the Fourier synthesis, respectively.

Crystallization, data collection, and structure determination

SeMet PhDph2 and anaerobically reconstituted PhDph2 were crystallized using the hanging drop vapor diffusion method. The X-ray data were collected at the NE-CAT beamlines at the Advanced Photon Source (APS). The structures of iron-free PhDph2 and reconstituted PhDph2 were determined by SeMet SAD phasing and the Fourier synthesis, respectively.

Reconstitution of activityReconstituted PhDph2 was prepared by growing cells in LB media supplemented with FeCl3, Fe(NH4)2(SO4)2, and L-Cys, and purified using Ni-NTA affinity chromatography anaerobically. The reaction was monitored using carboxyl-14C SAM.

Reconstitution of activity

Reconstituted PhDph2 was prepared by growing cells in LB media supplemented with FeCl3, Fe(NH4)2(SO4)2, and L-Cys, and purified using Ni-NTA affinity chromatography anaerobically. The reaction was monitored using carboxyl-14C SAM.

Analysis of reaction productsThe formation of MTA was detected by HPLC. Modification of PhEF2 His600 was confirmed by MALDI-MS after trypsin digestion. The products derived from the ACP group were detected by NMR directly and by LCMS after dansylation.

Analysis of reaction products

The formation of MTA was detected by HPLC. Modification of PhEF2 His600 was confirmed by MALDI-MS after trypsin digestion. The products derived from the ACP group were detected by NMR directly and by LCMS after dansylation.

EPR and Mössbauer spectroscopyESR spectra were recorded on a Bruker EMX spectrometer at a frequency of 9.24 GHz under standard conditions. Mössbauer spectra were recorded on a spectrometer from WEB research (Edina, MN) operating in the constant acceleration mode in transmission geometry.Full Methods and any associated references are available in the online version of the paper at www.nature.com/nature.

EPR and Mössbauer spectroscopy

ESR spectra were recorded on a Bruker EMX spectrometer at a frequency of 9.24 GHz under standard conditions. Mössbauer spectra were recorded on a spectrometer from WEB research (Edina, MN) operating in the constant acceleration mode in transmission geometry.

Full Methods and any associated references are available in the online version of the paper at www.nature.com/nature.

MethodsCloning, expression and purification of PhDph2 under anaerobic conditionsThe gene encoding P. horikoshii Dph2 was amplified by PCR from Pyrococcus horikoshii genomic DNA and inserted into pENTR™/TEV/D-TOPO® entry vector (Invitrogen), followed by recombination with pDESTF1 destination vector to create expression clones with an N-terminal His6 tag. The plasmids were transformed into the E. coli expression strain BL21(DE3) with pRARE. The cells were grown in LB media with 100 μg/ml ampicillin at 37 °C and were supplemented with FeCl3, Fe(NH4)2(SO4)2 and L-cysteine to final concentrations of 50 μM, 50 μM and 400 μM, respectively, when the absorbance of the cell culture reached an OD600 of 0.8. The cells were induced at an OD600 of 0.8 – 1.0 with 0.1 mM isopropyl-β-D-thiogalactopyranoside (IPTG), at which point the culture flasks were sealed to limit the amount of oxygen in the system. The induced cells were incubated in a shaker (New Brunswick Scientific Excella E25) at 37 °C and 200 rpm for 3 h before being transferred to the 4 °C cold room, where they were kept overnight without agitation. Cells were harvested the second day by centrifugation at 6,371 g (Beckman Coulter Avanti J-E), 4 °C for 10 min. Purification of PhDph2 was performed in an anaerobic chamber (Coy Laboratory Products) except for the centrifugation step. Cell pellets (from 2 l LB culture) were re-suspended in 30 mL lysis buffer (500 mM NaCl, 10 mM MgCl2, 5 mM imidazole, 1 mM DTT and 20 mM Tris-HCl at pH 7.4). Cells were lysed by incubating with 0.3% (w/v) lysozyme (Fisher) at 26 °C for 1 h, followed by freezing in liquid nitrogen and thawing at 26 °C once. Cell debris was removed by centrifugation at 48,384 g (Beckman Coulter Avanti J-E) for 30 min. The supernatant was incubated for 1 h with 1.2 ml Ni-NTA resin (Invitrogen) pre-equilibrated with the lysis buffer. The resin after incubation was loaded onto a polypropylene column and washed with 20 ml lysis buffer. PhDph2 was eluted from the column with elution buffers (100 mM or 150 mM imidazole in the lysis buffer, 3 ml each). The brown-colored elution fractions were buffer exchanged to 150 mM NaCl, 1 mM DTT, and 200 mM Tris-HCl at pH 7.4 using a Bio-Rad 10–DG desalting column. The protein was further purified by heating at 95 °C for 10 min and centrifugation at 48,384 g to remove the precipitate. Purified PhDph2 was concentrated using Amicon Ultra-4 centrifugal filter devices (Millipore).Expression and purification of SeMet substituted PhDph2PhDph2.pDESTF1 was transformed into the methionine-auxotrophic E. coli strain B834(DE3) pRARE that was obtained by transforming pRARE plasmid into B834(DE3). Cells were grown in M9 minimal medium supplemented with all amino acids (0.04 mg/mL) except L-methionine, 50 mg/l L-SeMet, 1× MEM vitamin solution, 0.4% (w/v) glucose, 2 mM MgSO4, 25 mg/ml FeSO4 and 0.1 mM CaCl2. The SeMet substituted PhDph2 was overexpressed and purified similarly as described above except aerobically and no additional iron and cysteine were added to the media.Expression and purification of 57Fe-labeled PhDph2 for Mössbauer spectroscopyE. coli BL21pRARE cells transformed with PhDph2.pDESTF1 were grown in M9 minimal medium supplemented with 0.2% (w/v) glucose, 2 mM MgSO4 and 0.1 mM CaCl2 at 37 °C. The 57Fe stock solution was prepared by dissolving 57Fe powder (Isoflex USA) in HCl to final concentrations of 1 M iron and 2.5 M chloride. The 57Fe stock solution and L-Cys were added to M9 media to final concentrations of 100 μM and 400 μM, respectively, before induction. The cells were induced at an OD600 of 0.8 with 100 μM IPTG and incubated at 20 °C for an additional 20 h. 57Fe labeled PhDph2 was anaerobically purified by following the same procedure used for the native protein purification. The final protein concentration, determined by Bradford protein assay (Bio-Rad), was 30 mg/ml (~800 μM). Iron was quantified by using the commercial Quantichrom iron assay kit (DIFE-250, Bioassay systems).Cloning, expression and purification of PhEF2Cloning of PhEF2 followed the same protocol as that of PhDph2. The plasmid was transformed into the E. coli expression strain, BL21 (DE3) with a pRARE plasmid. The cells were grown in LB media at 37 °C and induced at an OD600 of 1.0 with 0.1 mM IPTG. Cells were harvested after 3 h of induction by centrifugation at 6,000 rpm for 10 min. PhEF2 was purified through Ni-NTA affinity chromatography following the same protocol for PhDph2. The protein was further purified by heating at 95 °C for 10 min and subsequent FPLC purification using a Superdex 200 gel filtration column and a Q6 anion exchange column (Bio-Rad).Anaerobic reconstitution of PhDph2 activity and mass characterization of PhEF2The reaction components, 12 μM PhEF2, 24 μM PhDph2, and 10 mM dithionite were added to 150 mM NaCl, 1 mM DTT, and 200 mM Tris-HCl at pH 7.4 to a final volume of 15 μl in the anaerobic chamber under strictly anaerobic conditions. The reaction vials were sealed before taking out of the anaerobic chamber. 14C-SAM (2μL, final concentration of 267 μM) was injected into each reaction vial to start the reaction. The reaction mixtures were vortexed briefly to mix and incubated at 65 °C for 40 min. The reaction was stopped by adding protein loading dye to the reaction mixture and subsequent heating at 100 °C for 10 min, followed by 12% SDS-PAGE electrophoresis. The dried gel was exposed to a PhosphorImaging screen (GE Healthcare, Piscataway, NJ) and the radioactivity was detected using a STORM860 phosphorimager (GE Healthcare, Piscataway, NJ).Enzymatic reactions with normal SAM followed the same procedure, except that normal SAM was introduced in the anaerobic chamber. The PhEF2 band from the Coomassie blue-stained SDS-PAGE gel was cut out and digested by trypsin. Digestion products were extracted and cleaned using a Millipore Ziptip C4, then characterized by MALDI-MS at the Proteomics and Mass Spectrometry Facility of Cornell University.Analysis of reaction products with HPLCUnder anaerobic conditions, reactions were set up that contained 30 μM PhEF2, 30 μM PhDph2, 10 mM dithionite, 31 μM SAM, 150 mM NaCl, 1 mM DTT, and 200 mM Tris-HCl at pH 7.4 in a final volume of 64 μl. The mixture was incubated at 65 °C for 5 min, and then frozen at −20 °C. The reaction mixture was ice-thawed and TFA was added to a final concentration of 5%, followed by centrifugation to separate the precipitated proteins and the supernatant. The precipitated proteins were re-dissolved and PhEF2 was checked by MALDI-MS as described above to make sure the reaction had occurred. The supernatant was analyzed by HPLC (Shimadzu) on a C18 column (HαSprite) monitored at 260 nm absorbance, using a linear gradient from 0 to 40% buffer B in 20 min at a flow rate of 0.3 ml min−1 (buffer A: 50 mM ammonium acetate, pH 5.4; buffer B, 50% (v/v) methanol/water).1H NMR of reaction mixtureA complete reaction (260 μM PhDph2, 10 mM dithionite and 1000 μM SAM, in 1 ml of 100 mM phosphate buffer with 150 mM sodium chloride, pH=7.4) and control (without PhDph2 or without dithionite) were set up anaerobically. After incubation at 65 °C for 40 min, PhDPh2 was removed using a Millipore YM-10 Microcon filter unit. The flow-through was lyophilized overnight to dryness and then dissolved in 300 μL D2O for NMR. Shigemi D2O matched NMR tube was used. 1H NMR spectra were obtained on an INOVA 400 spectrometer. Compared with controls, four new peaks were identified on 1H NMR: a. 0.95 ppm (t, 1H, J = 7.6 Hz), b. 1.88 ppm (m, 1H), c. 2.12 ppm (m, 2H) and d. 2.43 ppm (t, 1H, J = 7.5 Hz). H-H DQCOSY 2D NMR spectrum (data not shown) showed that peak a is coupled to b, peak c is coupled to d and another peak (3.78 ppm) that is hidden under the huge signal from buffer. NMR data were analyzed by MestReNova (version 6.0.1).Dansylation reaction to detect the MS of the reaction products by LC-MSNMR samples were desalted and purified by TLC silica gel 60F254 (EMD Chemicals Inc.) with developing solvent (n-butanol : acetic acid : water = 2:1:1). The desired product bands (Rf 0.15 –0.65, below the Rf value of 5′-deoxy-5′-methylthioadenosine) were cut off the TLC plates and the products were washed off the silica gel by water and lyophilized overnight to dryness. The lyophilized products were dissolved with 50 mM sodium bicarbonate to a final concentration approximately 5 times of that of the NMR reaction. The solution was adjusted to pH 9–10 by 12% NaOH. Dansylation reactions were initiated by adding 0.5 volume of 50 mM dansyl chloride in acetonitrile to the solution and the reactions were carried out at room temperature in the dark for 30 min. Dansylated products were separated and analyzed by LC-MS with a linear gradient 0–80% solvent B in 33 min, at a flow rate of 0.8 ml/min. LC-MS experiments were carried out on a SHIMADZU LCMS-QP8000α with C18 column (250 × 4.6 mm, 10 μm, Grace Davison Discovery Sciences Headquarters) monitoring at 254 and 335 nm with positive mode for mass detection. Solvents for LC-MS were water with 0.1% formic acid (solvent A) and acetonitrile with 0.1% formic acid (solvent B).Sample preparation for Mössbauer spectroscopy and EPRAnaerobically purified 57Fe-labeled PhDph2 was dialyzed into a buffer containing 200 mM Tris-HCl (pH 7.4), 150 mM NaCl, 1 mM DTT and 10% glycerol, and concentrated to 25 – 30 mg/mL. The sample was frozen in liquid nitrogen in the anaerobic chamber for Mössbauer spectroscopy.For EPR measurements, PhDph2 (700 μM, with 15% glycerol) with and without 16 mM dithionite were incubated for 30 min before loading into an EPR quartz tube in the glovebox.Crystallization and structure determination of iron-free PhDph2Aerobically purified iron-free PhDph2 proteins were dialyzed to 10 mM sodium acetate at pH 4.6 and concentrated to 12 mg/ml for the crystallization experiments. The native protein was subjected to a series of sparse matrix screens (Hampton Research, Emerald Biostructures) using the hanging drop vapor-diffusion method at 18 °C in order to determine initial crystallization conditions. Best crystals for both SeMet substituted and native PhDph2 were obtained from 6 – 8% PEG 4000, 0.1 M ammonium acetate, 0.2 M KCl, 2% ethylene glycol, and 0.05 M sodium citrate at pH 5.1 – 5.3. These crystals belong to the space group P212121 with typical unit cell dimensions of a = 58.5 Å, b = 82.0, and c = 160.0 Å. Each asymmetric unit contains two monomers, corresponding to a solvent content of 50.3% and Matthews coefficient of 2.47 Å3/Da.The PhDph2 SeMet crystals were briefly transferred into a solution containing 6% glycerol, 16% ethylene glycol, 10% PEG 4000, 0.2 M ammonium acetate, 0.2 M KCl, and 0.1 M sodium citrate at pH 5.3 for cryoprotection. The crystals were allowed to soak in the cryo-solution for 30 – 45 s before plunging them into liquid nitrogen. In an attempt to reconstitute the iron sulfur clusters in crystals, native crystals were soaked in 10% PEG 4000, 100 mM citrate pH 5.3, 200 mM ammonium acetate, 200 mM KCl, 10% ethylene glycol, 8 mM SAM, 4 mM Fe(NH4)2(SO4)2, 4 mM NaS, and 40 mM DTT for 1 h prior to the same cryoprotection and freezing procedure described above.Data sets were collected at the Advanced Photon Source beamlines 24-ID-C and 24-ID-E using ADSC Quantum 315 CCD detectors. For the single wavelength SeMet data set, the energy was selected to maximize Δf” of the incorporated selenium (12661.5 eV, 0.97922 Å). Data sets were integrated and scaled using HKL200039. Data processing statistics are summarized in Supplementary Table 1.Eight selenium atom positions were determined using HKL2MAP40. These sites were used for SAD phasing using MLPHARE41 at 2.5 Å resolution. Initial phases were further improved through density modification, twofold noncrystallographic symmetry averaging, and phase extension for the 2.3 Å resolution native data using RESOLVE.42, 43 The resulting map was readily interpretable and an initial model was built using the interactive graphics program Coot44. The model refinement was carried out through alternating cycles of manually rebuilding using Coot, restrained refinement and water picking using Refmac545 and Phenix46. Structure refinement statistics are summarized in Supplementary Table 1.Crystallization and structure determination of reconstituted PhDph2Reconstituted PhDph2 protein was dialyzed to 100 mM NaCl, 1 mM DTT, and 10 mM sodium acetate at pH 4.6, concentrated to 20 mg/ml, and crystallized anaerobically at 26 °C in the glove box using the hanging drop vapor diffusion method. Anoxic sparse matrix screening solutions (Hampton Research, Emerald Biostructures) were used for initial crystallization screens. The optimized crystallization condition is as follows: drops were set up with 1.3 μl protein and an equal volume of 25 – 30% PEG 400, 0.2 M lithium sulfate and 0.1 M MES at pH 6.5, and were equilibrated against a reservoir solution of 0.6 M lithium chloride. Crystals appeared in a week and belonged to the same space group as that of the iron-free structure (P212121 with averaged unit cell dimensions of a = 55.7 Å, b = 80.5 Å and c = 162.1 Å). Prior to the data collection experiment, crystals were cryoprotected with 2.5 – 5% ethylene glycol, 25 – 30% PEG 400, 0.2 M lithium sulfate, and 0.1 M MES at pH 6.5, then plunged directly into liquid nitrogen in the glove box. A total of 200° of data were recorded at an energy of 12662.0 eV on an ADSC Quantum 315 CCD detector. The data were integrated and scaled to 2.1 Å resolution using HKL200039. The previously solved structure of PhDph2 lacking the iron sulfur cluster was used to generate phases by Fourier synthesis. A difference Fourier map was calculated and averaged for the two monomers to improve the electron density, and the resulting map was used to model the [4Fe-4S] cluster. A 2.8 Å resolution anomalous difference Fourier map calculated from a data set collected at 7150 eV (1.73405 Å) was also used as a reference for positioning the [4Fe-4S] cluster (not shown). The structure was refined using CNS47. X-ray experiment and structure refinement statistics are summarized in Supplementary Table 1.UV-Vis spectroscopySamples of PhDph2 (50 μM), with and without dithionite, were prepared anaerobically in 150 mM NaCl and 200 mM Tris-HCl at pH 7.4. The sample treated with dithionite was allowed to incubate for 30 min after adding the reducing agent at a final concentration of 0.5 mM. The samples were sealed in a Quartz cell (100 μl each) before taking out from the anaerobic chamber. UV-Vis spectra were obtained on a Cary 50 Bio UV-Visible spectrophotometer (Varian), scanning from 200 nm to 800 nm. The baseline was corrected with the buffer used to prepare the samples.EPR spectroscopyESR spectra were recorded at ACERT on a Bruker EMX spectrometer at a frequency of 9.24 GHz under standard conditions in 4 mm ID quartz tubes. The tubes were filled with PhDph2 solutions in an oxygen-free atmosphere and sealed under vacuum at 77 K. ESR measurements at 5–50 K were carried out using a liquid helium cryostat, ESR-10 (Oxford Instruments Ltd, England). The spectrometer settings were: modulation frequency 100 kHz, modulation amplitude 8 G, microwave power 0.63 mW.Mössbauer spectroscopyMössbauer spectra were recorded on a spectrometer from WEB research (Edina, MN) operating in the constant acceleration mode in transmission geometry. Spectra were recorded with the temperature of the sample maintained at 4.2 K using a SVT-400 cryostat from Janis (Wilmington, MA) in an externally applied magnetic field of 53 mT oriented parallel to the γ-beam. The quoted isomer shifts are relative to the centroid of the spectrum of a foil of α-Fe metal at room temperature. Data analysis was performed using the program WMOSS from WEB research.

Cloning, expression and purification of PhDph2 under anaerobic conditionsThe gene encoding P. horikoshii Dph2 was amplified by PCR from Pyrococcus horikoshii genomic DNA and inserted into pENTR™/TEV/D-TOPO® entry vector (Invitrogen), followed by recombination with pDESTF1 destination vector to create expression clones with an N-terminal His6 tag. The plasmids were transformed into the E. coli expression strain BL21(DE3) with pRARE. The cells were grown in LB media with 100 μg/ml ampicillin at 37 °C and were supplemented with FeCl3, Fe(NH4)2(SO4)2 and L-cysteine to final concentrations of 50 μM, 50 μM and 400 μM, respectively, when the absorbance of the cell culture reached an OD600 of 0.8. The cells were induced at an OD600 of 0.8 – 1.0 with 0.1 mM isopropyl-β-D-thiogalactopyranoside (IPTG), at which point the culture flasks were sealed to limit the amount of oxygen in the system. The induced cells were incubated in a shaker (New Brunswick Scientific Excella E25) at 37 °C and 200 rpm for 3 h before being transferred to the 4 °C cold room, where they were kept overnight without agitation. Cells were harvested the second day by centrifugation at 6,371 g (Beckman Coulter Avanti J-E), 4 °C for 10 min. Purification of PhDph2 was performed in an anaerobic chamber (Coy Laboratory Products) except for the centrifugation step. Cell pellets (from 2 l LB culture) were re-suspended in 30 mL lysis buffer (500 mM NaCl, 10 mM MgCl2, 5 mM imidazole, 1 mM DTT and 20 mM Tris-HCl at pH 7.4). Cells were lysed by incubating with 0.3% (w/v) lysozyme (Fisher) at 26 °C for 1 h, followed by freezing in liquid nitrogen and thawing at 26 °C once. Cell debris was removed by centrifugation at 48,384 g (Beckman Coulter Avanti J-E) for 30 min. The supernatant was incubated for 1 h with 1.2 ml Ni-NTA resin (Invitrogen) pre-equilibrated with the lysis buffer. The resin after incubation was loaded onto a polypropylene column and washed with 20 ml lysis buffer. PhDph2 was eluted from the column with elution buffers (100 mM or 150 mM imidazole in the lysis buffer, 3 ml each). The brown-colored elution fractions were buffer exchanged to 150 mM NaCl, 1 mM DTT, and 200 mM Tris-HCl at pH 7.4 using a Bio-Rad 10–DG desalting column. The protein was further purified by heating at 95 °C for 10 min and centrifugation at 48,384 g to remove the precipitate. Purified PhDph2 was concentrated using Amicon Ultra-4 centrifugal filter devices (Millipore).

Cloning, expression and purification of PhDph2 under anaerobic conditions

The gene encoding P. horikoshii Dph2 was amplified by PCR from Pyrococcus horikoshii genomic DNA and inserted into pENTR™/TEV/D-TOPO® entry vector (Invitrogen), followed by recombination with pDESTF1 destination vector to create expression clones with an N-terminal His6 tag. The plasmids were transformed into the E. coli expression strain BL21(DE3) with pRARE. The cells were grown in LB media with 100 μg/ml ampicillin at 37 °C and were supplemented with FeCl3, Fe(NH4)2(SO4)2 and L-cysteine to final concentrations of 50 μM, 50 μM and 400 μM, respectively, when the absorbance of the cell culture reached an OD600 of 0.8. The cells were induced at an OD600 of 0.8 – 1.0 with 0.1 mM isopropyl-β-D-thiogalactopyranoside (IPTG), at which point the culture flasks were sealed to limit the amount of oxygen in the system. The induced cells were incubated in a shaker (New Brunswick Scientific Excella E25) at 37 °C and 200 rpm for 3 h before being transferred to the 4 °C cold room, where they were kept overnight without agitation. Cells were harvested the second day by centrifugation at 6,371 g (Beckman Coulter Avanti J-E), 4 °C for 10 min. Purification of PhDph2 was performed in an anaerobic chamber (Coy Laboratory Products) except for the centrifugation step. Cell pellets (from 2 l LB culture) were re-suspended in 30 mL lysis buffer (500 mM NaCl, 10 mM MgCl2, 5 mM imidazole, 1 mM DTT and 20 mM Tris-HCl at pH 7.4). Cells were lysed by incubating with 0.3% (w/v) lysozyme (Fisher) at 26 °C for 1 h, followed by freezing in liquid nitrogen and thawing at 26 °C once. Cell debris was removed by centrifugation at 48,384 g (Beckman Coulter Avanti J-E) for 30 min. The supernatant was incubated for 1 h with 1.2 ml Ni-NTA resin (Invitrogen) pre-equilibrated with the lysis buffer. The resin after incubation was loaded onto a polypropylene column and washed with 20 ml lysis buffer. PhDph2 was eluted from the column with elution buffers (100 mM or 150 mM imidazole in the lysis buffer, 3 ml each). The brown-colored elution fractions were buffer exchanged to 150 mM NaCl, 1 mM DTT, and 200 mM Tris-HCl at pH 7.4 using a Bio-Rad 10–DG desalting column. The protein was further purified by heating at 95 °C for 10 min and centrifugation at 48,384 g to remove the precipitate. Purified PhDph2 was concentrated using Amicon Ultra-4 centrifugal filter devices (Millipore).

Expression and purification of SeMet substituted PhDph2PhDph2.pDESTF1 was transformed into the methionine-auxotrophic E. coli strain B834(DE3) pRARE that was obtained by transforming pRARE plasmid into B834(DE3). Cells were grown in M9 minimal medium supplemented with all amino acids (0.04 mg/mL) except L-methionine, 50 mg/l L-SeMet, 1× MEM vitamin solution, 0.4% (w/v) glucose, 2 mM MgSO4, 25 mg/ml FeSO4 and 0.1 mM CaCl2. The SeMet substituted PhDph2 was overexpressed and purified similarly as described above except aerobically and no additional iron and cysteine were added to the media.

Expression and purification of SeMet substituted PhDph2

PhDph2.pDESTF1 was transformed into the methionine-auxotrophic E. coli strain B834(DE3) pRARE that was obtained by transforming pRARE plasmid into B834(DE3). Cells were grown in M9 minimal medium supplemented with all amino acids (0.04 mg/mL) except L-methionine, 50 mg/l L-SeMet, 1× MEM vitamin solution, 0.4% (w/v) glucose, 2 mM MgSO4, 25 mg/ml FeSO4 and 0.1 mM CaCl2. The SeMet substituted PhDph2 was overexpressed and purified similarly as described above except aerobically and no additional iron and cysteine were added to the media.

Expression and purification of 57Fe-labeled PhDph2 for Mössbauer spectroscopyE. coli BL21pRARE cells transformed with PhDph2.pDESTF1 were grown in M9 minimal medium supplemented with 0.2% (w/v) glucose, 2 mM MgSO4 and 0.1 mM CaCl2 at 37 °C. The 57Fe stock solution was prepared by dissolving 57Fe powder (Isoflex USA) in HCl to final concentrations of 1 M iron and 2.5 M chloride. The 57Fe stock solution and L-Cys were added to M9 media to final concentrations of 100 μM and 400 μM, respectively, before induction. The cells were induced at an OD600 of 0.8 with 100 μM IPTG and incubated at 20 °C for an additional 20 h. 57Fe labeled PhDph2 was anaerobically purified by following the same procedure used for the native protein purification. The final protein concentration, determined by Bradford protein assay (Bio-Rad), was 30 mg/ml (~800 μM). Iron was quantified by using the commercial Quantichrom iron assay kit (DIFE-250, Bioassay systems).

Expression and purification of 57Fe-labeled PhDph2 for Mössbauer spectroscopy

E. coli BL21pRARE cells transformed with PhDph2.pDESTF1 were grown in M9 minimal medium supplemented with 0.2% (w/v) glucose, 2 mM MgSO4 and 0.1 mM CaCl2 at 37 °C. The 57Fe stock solution was prepared by dissolving 57Fe powder (Isoflex USA) in HCl to final concentrations of 1 M iron and 2.5 M chloride. The 57Fe stock solution and L-Cys were added to M9 media to final concentrations of 100 μM and 400 μM, respectively, before induction. The cells were induced at an OD600 of 0.8 with 100 μM IPTG and incubated at 20 °C for an additional 20 h. 57Fe labeled PhDph2 was anaerobically purified by following the same procedure used for the native protein purification. The final protein concentration, determined by Bradford protein assay (Bio-Rad), was 30 mg/ml (~800 μM). Iron was quantified by using the commercial Quantichrom iron assay kit (DIFE-250, Bioassay systems).

Cloning, expression and purification of PhEF2Cloning of PhEF2 followed the same protocol as that of PhDph2. The plasmid was transformed into the E. coli expression strain, BL21 (DE3) with a pRARE plasmid. The cells were grown in LB media at 37 °C and induced at an OD600 of 1.0 with 0.1 mM IPTG. Cells were harvested after 3 h of induction by centrifugation at 6,000 rpm for 10 min. PhEF2 was purified through Ni-NTA affinity chromatography following the same protocol for PhDph2. The protein was further purified by heating at 95 °C for 10 min and subsequent FPLC purification using a Superdex 200 gel filtration column and a Q6 anion exchange column (Bio-Rad).

Cloning, expression and purification of PhEF2

Cloning of PhEF2 followed the same protocol as that of PhDph2. The plasmid was transformed into the E. coli expression strain, BL21 (DE3) with a pRARE plasmid. The cells were grown in LB media at 37 °C and induced at an OD600 of 1.0 with 0.1 mM IPTG. Cells were harvested after 3 h of induction by centrifugation at 6,000 rpm for 10 min. PhEF2 was purified through Ni-NTA affinity chromatography following the same protocol for PhDph2. The protein was further purified by heating at 95 °C for 10 min and subsequent FPLC purification using a Superdex 200 gel filtration column and a Q6 anion exchange column (Bio-Rad).

Anaerobic reconstitution of PhDph2 activity and mass characterization of PhEF2The reaction components, 12 μM PhEF2, 24 μM PhDph2, and 10 mM dithionite were added to 150 mM NaCl, 1 mM DTT, and 200 mM Tris-HCl at pH 7.4 to a final volume of 15 μl in the anaerobic chamber under strictly anaerobic conditions. The reaction vials were sealed before taking out of the anaerobic chamber. 14C-SAM (2μL, final concentration of 267 μM) was injected into each reaction vial to start the reaction. The reaction mixtures were vortexed briefly to mix and incubated at 65 °C for 40 min. The reaction was stopped by adding protein loading dye to the reaction mixture and subsequent heating at 100 °C for 10 min, followed by 12% SDS-PAGE electrophoresis. The dried gel was exposed to a PhosphorImaging screen (GE Healthcare, Piscataway, NJ) and the radioactivity was detected using a STORM860 phosphorimager (GE Healthcare, Piscataway, NJ).Enzymatic reactions with normal SAM followed the same procedure, except that normal SAM was introduced in the anaerobic chamber. The PhEF2 band from the Coomassie blue-stained SDS-PAGE gel was cut out and digested by trypsin. Digestion products were extracted and cleaned using a Millipore Ziptip C4, then characterized by MALDI-MS at the Proteomics and Mass Spectrometry Facility of Cornell University.

Anaerobic reconstitution of PhDph2 activity and mass characterization of PhEF2

The reaction components, 12 μM PhEF2, 24 μM PhDph2, and 10 mM dithionite were added to 150 mM NaCl, 1 mM DTT, and 200 mM Tris-HCl at pH 7.4 to a final volume of 15 μl in the anaerobic chamber under strictly anaerobic conditions. The reaction vials were sealed before taking out of the anaerobic chamber. 14C-SAM (2μL, final concentration of 267 μM) was injected into each reaction vial to start the reaction. The reaction mixtures were vortexed briefly to mix and incubated at 65 °C for 40 min. The reaction was stopped by adding protein loading dye to the reaction mixture and subsequent heating at 100 °C for 10 min, followed by 12% SDS-PAGE electrophoresis. The dried gel was exposed to a PhosphorImaging screen (GE Healthcare, Piscataway, NJ) and the radioactivity was detected using a STORM860 phosphorimager (GE Healthcare, Piscataway, NJ).

Enzymatic reactions with normal SAM followed the same procedure, except that normal SAM was introduced in the anaerobic chamber. The PhEF2 band from the Coomassie blue-stained SDS-PAGE gel was cut out and digested by trypsin. Digestion products were extracted and cleaned using a Millipore Ziptip C4, then characterized by MALDI-MS at the Proteomics and Mass Spectrometry Facility of Cornell University.

Analysis of reaction products with HPLCUnder anaerobic conditions, reactions were set up that contained 30 μM PhEF2, 30 μM PhDph2, 10 mM dithionite, 31 μM SAM, 150 mM NaCl, 1 mM DTT, and 200 mM Tris-HCl at pH 7.4 in a final volume of 64 μl. The mixture was incubated at 65 °C for 5 min, and then frozen at −20 °C. The reaction mixture was ice-thawed and TFA was added to a final concentration of 5%, followed by centrifugation to separate the precipitated proteins and the supernatant. The precipitated proteins were re-dissolved and PhEF2 was checked by MALDI-MS as described above to make sure the reaction had occurred. The supernatant was analyzed by HPLC (Shimadzu) on a C18 column (HαSprite) monitored at 260 nm absorbance, using a linear gradient from 0 to 40% buffer B in 20 min at a flow rate of 0.3 ml min−1 (buffer A: 50 mM ammonium acetate, pH 5.4; buffer B, 50% (v/v) methanol/water).

Analysis of reaction products with HPLC

Under anaerobic conditions, reactions were set up that contained 30 μM PhEF2, 30 μM PhDph2, 10 mM dithionite, 31 μM SAM, 150 mM NaCl, 1 mM DTT, and 200 mM Tris-HCl at pH 7.4 in a final volume of 64 μl. The mixture was incubated at 65 °C for 5 min, and then frozen at −20 °C. The reaction mixture was ice-thawed and TFA was added to a final concentration of 5%, followed by centrifugation to separate the precipitated proteins and the supernatant. The precipitated proteins were re-dissolved and PhEF2 was checked by MALDI-MS as described above to make sure the reaction had occurred. The supernatant was analyzed by HPLC (Shimadzu) on a C18 column (HαSprite) monitored at 260 nm absorbance, using a linear gradient from 0 to 40% buffer B in 20 min at a flow rate of 0.3 ml min−1 (buffer A: 50 mM ammonium acetate, pH 5.4; buffer B, 50% (v/v) methanol/water).

1H NMR of reaction mixtureA complete reaction (260 μM PhDph2, 10 mM dithionite and 1000 μM SAM, in 1 ml of 100 mM phosphate buffer with 150 mM sodium chloride, pH=7.4) and control (without PhDph2 or without dithionite) were set up anaerobically. After incubation at 65 °C for 40 min, PhDPh2 was removed using a Millipore YM-10 Microcon filter unit. The flow-through was lyophilized overnight to dryness and then dissolved in 300 μL D2O for NMR. Shigemi D2O matched NMR tube was used. 1H NMR spectra were obtained on an INOVA 400 spectrometer. Compared with controls, four new peaks were identified on 1H NMR: a. 0.95 ppm (t, 1H, J = 7.6 Hz), b. 1.88 ppm (m, 1H), c. 2.12 ppm (m, 2H) and d. 2.43 ppm (t, 1H, J = 7.5 Hz). H-H DQCOSY 2D NMR spectrum (data not shown) showed that peak a is coupled to b, peak c is coupled to d and another peak (3.78 ppm) that is hidden under the huge signal from buffer. NMR data were analyzed by MestReNova (version 6.0.1).

1H NMR of reaction mixture

A complete reaction (260 μM PhDph2, 10 mM dithionite and 1000 μM SAM, in 1 ml of 100 mM phosphate buffer with 150 mM sodium chloride, pH=7.4) and control (without PhDph2 or without dithionite) were set up anaerobically. After incubation at 65 °C for 40 min, PhDPh2 was removed using a Millipore YM-10 Microcon filter unit. The flow-through was lyophilized overnight to dryness and then dissolved in 300 μL D2O for NMR. Shigemi D2O matched NMR tube was used. 1H NMR spectra were obtained on an INOVA 400 spectrometer. Compared with controls, four new peaks were identified on 1H NMR: a. 0.95 ppm (t, 1H, J = 7.6 Hz), b. 1.88 ppm (m, 1H), c. 2.12 ppm (m, 2H) and d. 2.43 ppm (t, 1H, J = 7.5 Hz). H-H DQCOSY 2D NMR spectrum (data not shown) showed that peak a is coupled to b, peak c is coupled to d and another peak (3.78 ppm) that is hidden under the huge signal from buffer. NMR data were analyzed by MestReNova (version 6.0.1).

Dansylation reaction to detect the MS of the reaction products by LC-MSNMR samples were desalted and purified by TLC silica gel 60F254 (EMD Chemicals Inc.) with developing solvent (n-butanol : acetic acid : water = 2:1:1). The desired product bands (Rf 0.15 –0.65, below the Rf value of 5′-deoxy-5′-methylthioadenosine) were cut off the TLC plates and the products were washed off the silica gel by water and lyophilized overnight to dryness. The lyophilized products were dissolved with 50 mM sodium bicarbonate to a final concentration approximately 5 times of that of the NMR reaction. The solution was adjusted to pH 9–10 by 12% NaOH. Dansylation reactions were initiated by adding 0.5 volume of 50 mM dansyl chloride in acetonitrile to the solution and the reactions were carried out at room temperature in the dark for 30 min. Dansylated products were separated and analyzed by LC-MS with a linear gradient 0–80% solvent B in 33 min, at a flow rate of 0.8 ml/min. LC-MS experiments were carried out on a SHIMADZU LCMS-QP8000α with C18 column (250 × 4.6 mm, 10 μm, Grace Davison Discovery Sciences Headquarters) monitoring at 254 and 335 nm with positive mode for mass detection. Solvents for LC-MS were water with 0.1% formic acid (solvent A) and acetonitrile with 0.1% formic acid (solvent B).

Dansylation reaction to detect the MS of the reaction products by LC-MS

NMR samples were desalted and purified by TLC silica gel 60F254 (EMD Chemicals Inc.) with developing solvent (n-butanol : acetic acid : water = 2:1:1). The desired product bands (Rf 0.15 –0.65, below the Rf value of 5′-deoxy-5′-methylthioadenosine) were cut off the TLC plates and the products were washed off the silica gel by water and lyophilized overnight to dryness. The lyophilized products were dissolved with 50 mM sodium bicarbonate to a final concentration approximately 5 times of that of the NMR reaction. The solution was adjusted to pH 9–10 by 12% NaOH. Dansylation reactions were initiated by adding 0.5 volume of 50 mM dansyl chloride in acetonitrile to the solution and the reactions were carried out at room temperature in the dark for 30 min. Dansylated products were separated and analyzed by LC-MS with a linear gradient 0–80% solvent B in 33 min, at a flow rate of 0.8 ml/min. LC-MS experiments were carried out on a SHIMADZU LCMS-QP8000α with C18 column (250 × 4.6 mm, 10 μm, Grace Davison Discovery Sciences Headquarters) monitoring at 254 and 335 nm with positive mode for mass detection. Solvents for LC-MS were water with 0.1% formic acid (solvent A) and acetonitrile with 0.1% formic acid (solvent B).

Sample preparation for Mössbauer spectroscopy and EPRAnaerobically purified 57Fe-labeled PhDph2 was dialyzed into a buffer containing 200 mM Tris-HCl (pH 7.4), 150 mM NaCl, 1 mM DTT and 10% glycerol, and concentrated to 25 – 30 mg/mL. The sample was frozen in liquid nitrogen in the anaerobic chamber for Mössbauer spectroscopy.For EPR measurements, PhDph2 (700 μM, with 15% glycerol) with and without 16 mM dithionite were incubated for 30 min before loading into an EPR quartz tube in the glovebox.

Sample preparation for Mössbauer spectroscopy and EPR

Anaerobically purified 57Fe-labeled PhDph2 was dialyzed into a buffer containing 200 mM Tris-HCl (pH 7.4), 150 mM NaCl, 1 mM DTT and 10% glycerol, and concentrated to 25 – 30 mg/mL. The sample was frozen in liquid nitrogen in the anaerobic chamber for Mössbauer spectroscopy.

For EPR measurements, PhDph2 (700 μM, with 15% glycerol) with and without 16 mM dithionite were incubated for 30 min before loading into an EPR quartz tube in the glovebox.

Crystallization and structure determination of iron-free PhDph2Aerobically purified iron-free PhDph2 proteins were dialyzed to 10 mM sodium acetate at pH 4.6 and concentrated to 12 mg/ml for the crystallization experiments. The native protein was subjected to a series of sparse matrix screens (Hampton Research, Emerald Biostructures) using the hanging drop vapor-diffusion method at 18 °C in order to determine initial crystallization conditions. Best crystals for both SeMet substituted and native PhDph2 were obtained from 6 – 8% PEG 4000, 0.1 M ammonium acetate, 0.2 M KCl, 2% ethylene glycol, and 0.05 M sodium citrate at pH 5.1 – 5.3. These crystals belong to the space group P212121 with typical unit cell dimensions of a = 58.5 Å, b = 82.0, and c = 160.0 Å. Each asymmetric unit contains two monomers, corresponding to a solvent content of 50.3% and Matthews coefficient of 2.47 Å3/Da.The PhDph2 SeMet crystals were briefly transferred into a solution containing 6% glycerol, 16% ethylene glycol, 10% PEG 4000, 0.2 M ammonium acetate, 0.2 M KCl, and 0.1 M sodium citrate at pH 5.3 for cryoprotection. The crystals were allowed to soak in the cryo-solution for 30 – 45 s before plunging them into liquid nitrogen. In an attempt to reconstitute the iron sulfur clusters in crystals, native crystals were soaked in 10% PEG 4000, 100 mM citrate pH 5.3, 200 mM ammonium acetate, 200 mM KCl, 10% ethylene glycol, 8 mM SAM, 4 mM Fe(NH4)2(SO4)2, 4 mM NaS, and 40 mM DTT for 1 h prior to the same cryoprotection and freezing procedure described above.Data sets were collected at the Advanced Photon Source beamlines 24-ID-C and 24-ID-E using ADSC Quantum 315 CCD detectors. For the single wavelength SeMet data set, the energy was selected to maximize Δf” of the incorporated selenium (12661.5 eV, 0.97922 Å). Data sets were integrated and scaled using HKL200039. Data processing statistics are summarized in Supplementary Table 1.Eight selenium atom positions were determined using HKL2MAP40. These sites were used for SAD phasing using MLPHARE41 at 2.5 Å resolution. Initial phases were further improved through density modification, twofold noncrystallographic symmetry averaging, and phase extension for the 2.3 Å resolution native data using RESOLVE.42, 43 The resulting map was readily interpretable and an initial model was built using the interactive graphics program Coot44. The model refinement was carried out through alternating cycles of manually rebuilding using Coot, restrained refinement and water picking using Refmac545 and Phenix46. Structure refinement statistics are summarized in Supplementary Table 1.

Crystallization and structure determination of iron-free PhDph2

Aerobically purified iron-free PhDph2 proteins were dialyzed to 10 mM sodium acetate at pH 4.6 and concentrated to 12 mg/ml for the crystallization experiments. The native protein was subjected to a series of sparse matrix screens (Hampton Research, Emerald Biostructures) using the hanging drop vapor-diffusion method at 18 °C in order to determine initial crystallization conditions. Best crystals for both SeMet substituted and native PhDph2 were obtained from 6 – 8% PEG 4000, 0.1 M ammonium acetate, 0.2 M KCl, 2% ethylene glycol, and 0.05 M sodium citrate at pH 5.1 – 5.3. These crystals belong to the space group P212121 with typical unit cell dimensions of a = 58.5 Å, b = 82.0, and c = 160.0 Å. Each asymmetric unit contains two monomers, corresponding to a solvent content of 50.3% and Matthews coefficient of 2.47 Å3/Da.

The PhDph2 SeMet crystals were briefly transferred into a solution containing 6% glycerol, 16% ethylene glycol, 10% PEG 4000, 0.2 M ammonium acetate, 0.2 M KCl, and 0.1 M sodium citrate at pH 5.3 for cryoprotection. The crystals were allowed to soak in the cryo-solution for 30 – 45 s before plunging them into liquid nitrogen. In an attempt to reconstitute the iron sulfur clusters in crystals, native crystals were soaked in 10% PEG 4000, 100 mM citrate pH 5.3, 200 mM ammonium acetate, 200 mM KCl, 10% ethylene glycol, 8 mM SAM, 4 mM Fe(NH4)2(SO4)2, 4 mM NaS, and 40 mM DTT for 1 h prior to the same cryoprotection and freezing procedure described above.

Data sets were collected at the Advanced Photon Source beamlines 24-ID-C and 24-ID-E using ADSC Quantum 315 CCD detectors. For the single wavelength SeMet data set, the energy was selected to maximize Δf” of the incorporated selenium (12661.5 eV, 0.97922 Å). Data sets were integrated and scaled using HKL200039. Data processing statistics are summarized in Supplementary Table 1.

Eight selenium atom positions were determined using HKL2MAP40. These sites were used for SAD phasing using MLPHARE41 at 2.5 Å resolution. Initial phases were further improved through density modification, twofold noncrystallographic symmetry averaging, and phase extension for the 2.3 Å resolution native data using RESOLVE.42, 43 The resulting map was readily interpretable and an initial model was built using the interactive graphics program Coot44. The model refinement was carried out through alternating cycles of manually rebuilding using Coot, restrained refinement and water picking using Refmac545 and Phenix46. Structure refinement statistics are summarized in Supplementary Table 1.

Crystallization and structure determination of reconstituted PhDph2Reconstituted PhDph2 protein was dialyzed to 100 mM NaCl, 1 mM DTT, and 10 mM sodium acetate at pH 4.6, concentrated to 20 mg/ml, and crystallized anaerobically at 26 °C in the glove box using the hanging drop vapor diffusion method. Anoxic sparse matrix screening solutions (Hampton Research, Emerald Biostructures) were used for initial crystallization screens. The optimized crystallization condition is as follows: drops were set up with 1.3 μl protein and an equal volume of 25 – 30% PEG 400, 0.2 M lithium sulfate and 0.1 M MES at pH 6.5, and were equilibrated against a reservoir solution of 0.6 M lithium chloride. Crystals appeared in a week and belonged to the same space group as that of the iron-free structure (P212121 with averaged unit cell dimensions of a = 55.7 Å, b = 80.5 Å and c = 162.1 Å). Prior to the data collection experiment, crystals were cryoprotected with 2.5 – 5% ethylene glycol, 25 – 30% PEG 400, 0.2 M lithium sulfate, and 0.1 M MES at pH 6.5, then plunged directly into liquid nitrogen in the glove box. A total of 200° of data were recorded at an energy of 12662.0 eV on an ADSC Quantum 315 CCD detector. The data were integrated and scaled to 2.1 Å resolution using HKL200039. The previously solved structure of PhDph2 lacking the iron sulfur cluster was used to generate phases by Fourier synthesis. A difference Fourier map was calculated and averaged for the two monomers to improve the electron density, and the resulting map was used to model the [4Fe-4S] cluster. A 2.8 Å resolution anomalous difference Fourier map calculated from a data set collected at 7150 eV (1.73405 Å) was also used as a reference for positioning the [4Fe-4S] cluster (not shown). The structure was refined using CNS47. X-ray experiment and structure refinement statistics are summarized in Supplementary Table 1.

Crystallization and structure determination of reconstituted PhDph2

Reconstituted PhDph2 protein was dialyzed to 100 mM NaCl, 1 mM DTT, and 10 mM sodium acetate at pH 4.6, concentrated to 20 mg/ml, and crystallized anaerobically at 26 °C in the glove box using the hanging drop vapor diffusion method. Anoxic sparse matrix screening solutions (Hampton Research, Emerald Biostructures) were used for initial crystallization screens. The optimized crystallization condition is as follows: drops were set up with 1.3 μl protein and an equal volume of 25 – 30% PEG 400, 0.2 M lithium sulfate and 0.1 M MES at pH 6.5, and were equilibrated against a reservoir solution of 0.6 M lithium chloride. Crystals appeared in a week and belonged to the same space group as that of the iron-free structure (P212121 with averaged unit cell dimensions of a = 55.7 Å, b = 80.5 Å and c = 162.1 Å). Prior to the data collection experiment, crystals were cryoprotected with 2.5 – 5% ethylene glycol, 25 – 30% PEG 400, 0.2 M lithium sulfate, and 0.1 M MES at pH 6.5, then plunged directly into liquid nitrogen in the glove box. A total of 200° of data were recorded at an energy of 12662.0 eV on an ADSC Quantum 315 CCD detector. The data were integrated and scaled to 2.1 Å resolution using HKL200039. The previously solved structure of PhDph2 lacking the iron sulfur cluster was used to generate phases by Fourier synthesis. A difference Fourier map was calculated and averaged for the two monomers to improve the electron density, and the resulting map was used to model the [4Fe-4S] cluster. A 2.8 Å resolution anomalous difference Fourier map calculated from a data set collected at 7150 eV (1.73405 Å) was also used as a reference for positioning the [4Fe-4S] cluster (not shown). The structure was refined using CNS47. X-ray experiment and structure refinement statistics are summarized in Supplementary Table 1.

UV-Vis spectroscopySamples of PhDph2 (50 μM), with and without dithionite, were prepared anaerobically in 150 mM NaCl and 200 mM Tris-HCl at pH 7.4. The sample treated with dithionite was allowed to incubate for 30 min after adding the reducing agent at a final concentration of 0.5 mM. The samples were sealed in a Quartz cell (100 μl each) before taking out from the anaerobic chamber. UV-Vis spectra were obtained on a Cary 50 Bio UV-Visible spectrophotometer (Varian), scanning from 200 nm to 800 nm. The baseline was corrected with the buffer used to prepare the samples.

UV-Vis spectroscopy

Samples of PhDph2 (50 μM), with and without dithionite, were prepared anaerobically in 150 mM NaCl and 200 mM Tris-HCl at pH 7.4. The sample treated with dithionite was allowed to incubate for 30 min after adding the reducing agent at a final concentration of 0.5 mM. The samples were sealed in a Quartz cell (100 μl each) before taking out from the anaerobic chamber. UV-Vis spectra were obtained on a Cary 50 Bio UV-Visible spectrophotometer (Varian), scanning from 200 nm to 800 nm. The baseline was corrected with the buffer used to prepare the samples.

EPR spectroscopyESR spectra were recorded at ACERT on a Bruker EMX spectrometer at a frequency of 9.24 GHz under standard conditions in 4 mm ID quartz tubes. The tubes were filled with PhDph2 solutions in an oxygen-free atmosphere and sealed under vacuum at 77 K. ESR measurements at 5–50 K were carried out using a liquid helium cryostat, ESR-10 (Oxford Instruments Ltd, England). The spectrometer settings were: modulation frequency 100 kHz, modulation amplitude 8 G, microwave power 0.63 mW.

EPR spectroscopy

ESR spectra were recorded at ACERT on a Bruker EMX spectrometer at a frequency of 9.24 GHz under standard conditions in 4 mm ID quartz tubes. The tubes were filled with PhDph2 solutions in an oxygen-free atmosphere and sealed under vacuum at 77 K. ESR measurements at 5–50 K were carried out using a liquid helium cryostat, ESR-10 (Oxford Instruments Ltd, England). The spectrometer settings were: modulation frequency 100 kHz, modulation amplitude 8 G, microwave power 0.63 mW.

Mössbauer spectroscopyMössbauer spectra were recorded on a spectrometer from WEB research (Edina, MN) operating in the constant acceleration mode in transmission geometry. Spectra were recorded with the temperature of the sample maintained at 4.2 K using a SVT-400 cryostat from Janis (Wilmington, MA) in an externally applied magnetic field of 53 mT oriented parallel to the γ-beam. The quoted isomer shifts are relative to the centroid of the spectrum of a foil of α-Fe metal at room temperature. Data analysis was performed using the program WMOSS from WEB research.

Mössbauer spectroscopy

Mössbauer spectra were recorded on a spectrometer from WEB research (Edina, MN) operating in the constant acceleration mode in transmission geometry. Spectra were recorded with the temperature of the sample maintained at 4.2 K using a SVT-400 cryostat from Janis (Wilmington, MA) in an externally applied magnetic field of 53 mT oriented parallel to the γ-beam. The quoted isomer shifts are relative to the centroid of the spectrum of a foil of α-Fe metal at room temperature. Data analysis was performed using the program WMOSS from WEB research.

Supplementary Material1

Supplementary Material
