---
pmid: '15888726'
title: ZAC, LIT1 (KCNQ1OT1) and p57KIP2 (CDKN1C) are in an imprinted gene network
  that may play a role in Beckwith-Wiedemann syndrome.
authors:
- Arima T
- Kamikihara T
- Hayashida T
- Kato K
- Inoue T
- Shirayoshi Y
- Oshimura M
- Soejima H
- Mukai T
- Wake N
journal: Nucleic Acids Res
year: '2005'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC1097765
doi: 10.1093/nar/gki555
---

# ZAC, LIT1 (KCNQ1OT1) and p57KIP2 (CDKN1C) are in an imprinted gene network that may play a role in Beckwith-Wiedemann syndrome.
**Authors:** Arima T, Kamikihara T, Hayashida T, Kato K, Inoue T, Shirayoshi Y, Oshimura M, Soejima H, Mukai T, Wake N
**Journal:** Nucleic Acids Res (2005)
**DOI:** [10.1093/nar/gki555](https://doi.org/10.1093/nar/gki555)
**PMC:** [PMC1097765](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1097765/)

## Abstract

1. Nucleic Acids Res. 2005 May 11;33(8):2650-60. doi: 10.1093/nar/gki555. Print 
2005.

ZAC, LIT1 (KCNQ1OT1) and p57KIP2 (CDKN1C) are in an imprinted gene network that 
may play a role in Beckwith-Wiedemann syndrome.

Arima T(1), Kamikihara T, Hayashida T, Kato K, Inoue T, Shirayoshi Y, Oshimura 
M, Soejima H, Mukai T, Wake N.

Author information:
(1)Department of Reproductive Physiology and Endocrinology, Medical Institute of 
Bioregulation, Kyusyu University 4546, Tsurumihara, Beppu, Oita 874-0838, Japan.

Loss of genomic imprinting is involved in a number of developmental 
abnormalities and cancers. ZAC is an imprinted gene expressed from the paternal 
allele of chromosome 6q24 within a region known to harbor a tumor suppressor 
gene for several types of neoplasia. p57(KIP2) (CDKN1C) is a maternally 
expressed gene located on chromosome 11p15.5 which encodes a cyclin-dependent 
kinase inhibitor that may also act as a tumor suppressor gene. Mutations in ZAC 
and p57KIP2 have been implicated in transient neonatal diabetes mellitus (TNDB) 
and Beckwith-Wiedemann syndrome, respectively. Patients with these diseases 
share many characteristics. Here we show that mouse Zac1 and p57Kip2 have a 
strikingly similar expression pattern. ZAC, a sequence-specific DNA-binding 
protein, binds within the CpG island of LIT1 (KCNQ1OT1), a paternally expressed, 
anti-sense RNA thought to negatively regulate p57(KIP2) in cis. ZAC induces LIT1 
transcription in a methylation-dependent manner. Our data suggest that ZAC may 
regulate p57(KIP2) through LIT1, forming part of a novel signaling pathway 
regulating cell growth. Mutations in ZAC may, therefore, contribute to 
Beckwith-Wiedemann syndrome. Furthermore, we find changes in DNA methylation at 
the LIT1 putative imprinting control region in two patients with TNDB.

DOI: 10.1093/nar/gki555
PMCID: PMC1097765
PMID: 15888726 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONGenomic imprinting is a gamete-specific modification that results in the differential expression of the two parental alleles in somatic cells (1,2). Mutations that affect the epigenetic status of imprinted loci are involved in a number of human diseases, developmental abnormalities and malignant tumors indicating a general role for imprinted genes in mammalian development (3). Several imprinted genes, including INS2, p57KIP2 (CDKN1C), GNAS, RASGRF1, MASH2 and ZAC play a role in regulating the cell cycle (4).ZAC was originally identified, along with p53, in a functional screen for factors that induce expression of the pituitary adenylate cyclase activating polypeptide (PACAP) type I receptor gene (5). ZAC encodes a zinc finger protein and is expressed only from the paternal allele with the maternal allele silent and methylated (6–8). Mutations in ZAC are thought to play a role in transient neonatal diabetes mellitus (TNDM). The gene maps to 6q24–25, a region implicated in the origin of several cancers (9–12). Abdollahi et al. (13) reported a high incidence of allelic loss at this chromosomal region in ovarian cancers. In addition, they independently cloned as Lot1 (Lost on transformation), the rat ortholog of ZAC, from rat ovarian surface epithelial cells that spontaneously transform in vitro.ZAC exhibits a tumor suppressor activity characterized by induction of apoptosis and G1 arrest. This proceeds independently of known cell cycle control proteins, such as pRB, p21, p27 and p16 (5). Nevertheless, ZAC shares a number of similar functions to p53. Both proteins regulate the cell cycle, apoptosis and nuclear receptors. They interact physically and functionally with CBP and p300 that serve as integrators of multiple signaling pathways (14,15). Both p53 and ZAC are sequence-specific DNA-binding proteins and can act as transcriptional co-factors for other transcriptional activator proteins (16,17). Recently, ZAC was shown to specifically enhance the activity of p53-responsive promoters in cells expressing wild-type p53 (18). It is, therefore, possible that ZAC may act as a transcription factor with a similar mode of action to p53.Three types of mutations have been shown to result in TNDM; paternal uniparental isodisomy of chromosome 6, paternally inherited duplication of 6q24 and a methylation defect at a CpG island overlapping exon 1 of ZAC/HYMAI (19). We showed that this CpG island is differentially methylated (20). The region is unmethylated in sperm but methylated in oocytes, a difference that persists between parental alleles throughout pre- and post implantation development suggestive of an imprint control region (ICR). There is a region within this putative ICR that exhibits a high degree of homology between mouse and human that acts as a strong transcriptional repressor when methylated. Loss of methylation at 8 CpG sites within this region was seen in five of six TNDM patients studied with a normal karyotype. We proposed that the DMR adjacent to ZAC may regulate expression of imprinted genes within the domain, and that epigenetic or genetic mutations of this region result in TNDM by affecting expression of ZAC in the pancreas and/or the pituitary (20). This is supported by the recent findings that mice overexpressing human ZAC from a transgene show impaired development of the endocrine pancreas and impaired β-cell function (21). ZAC may mediate its actions through transcriptional regulation of the PACAP type I receptor gene which is a potent insulin secretagogue and an important mediator of autocrine control of insulin secretion in the pancreatic islet (20).During our studies on mouse Zac1 (20), we noted a similarity in the expression pattern of Zac1 with a second imprinted gene, p57Kip2 (Cdkn1c), hereafter referred to as p57Kip2. p57KIP2 is a maternally expressed gene that encodes a cyclin-dependent kinase inhibitor. The protein inhibits all G1 cyclin/Cdk complexes and negatively regulates cell proliferation, acting downstream of p53. Like ZAC, p57KIP2 may also act as a tumor suppressor through its role in regulating the cell cycle (22–24). p57KIP2 is located on human chromosome 11p15.5, a region implicated in both sporadic cancers and Beckwith–Wiedemann syndrome (BWS), a familial cancer syndrome, making it a candidate tumor suppressor gene (23). Several types of childhood tumor display a specific loss of the maternal 11p15 allele that suggests the involvement of an imprinted gene in this region (25).BWS patients generally present with three major features: exomphalos, macroglossia and giantism. Other features include hypoglycemia, organomegaly, hemihypertrophy, genitourinary abnormalities, cleft palate and a susceptibility to embryonal tumors. In turn, TNDM is associated with intra-uterine growth failure, dehydration, hyperglycemia and failure to thrive (26,27). There have been a number of reported associations with TNDM including umbilical hernia (7%) and macroglossia (23%) (28). These two anomalies are also found in patients with BWS, which typically also involves hyperinsulinemia. The role for ZAC is a predicted gain of function with biallelic expression of ZAC in TNDM patients (since UPD of the region also results in TNDM). Loss of function of p57KIP2 in BWS is implicated in patients with point mutations within the coding sequence of the gene (29–32).Several imprinted genes lie within the 11p15.5 region. LIT1 (KCNQ1OT1), hereafter referred to as LIT1, is a paternally expressed, antisense RNA located within the KCNQ1 gene that may regulate imprinting of the 11p15.5 domain (25). Differential methylation of the LIT1 CpG island is conserved between human and mouse (25). This differential methylation is acquired in the germ line (33) making it an excellent candidate for an ICR. Frequent loss of maternal methylation at the LIT1 CpG island has been observed in BWS patients (40–50%) (34). A targeted deletion of LIT1 leads to loss of silencing of p57Kip2 in both mice and in human cell lines (33,35). Finally, the region between the Kcnq1 and p57Kip2 shows both physical and mechanistic similarities to the imprinted domain containing Igf2r and its antisense control transcript, Air (36).We, therefore, investigated whether the similarity in phenotypes between BWS and TNDM is a reflection of a regulatory association between ZAC and p57KIP2.

INTRODUCTION

Genomic imprinting is a gamete-specific modification that results in the differential expression of the two parental alleles in somatic cells (1,2). Mutations that affect the epigenetic status of imprinted loci are involved in a number of human diseases, developmental abnormalities and malignant tumors indicating a general role for imprinted genes in mammalian development (3). Several imprinted genes, including INS2, p57KIP2 (CDKN1C), GNAS, RASGRF1, MASH2 and ZAC play a role in regulating the cell cycle (4).

ZAC was originally identified, along with p53, in a functional screen for factors that induce expression of the pituitary adenylate cyclase activating polypeptide (PACAP) type I receptor gene (5). ZAC encodes a zinc finger protein and is expressed only from the paternal allele with the maternal allele silent and methylated (6–8). Mutations in ZAC are thought to play a role in transient neonatal diabetes mellitus (TNDM). The gene maps to 6q24–25, a region implicated in the origin of several cancers (9–12). Abdollahi et al. (13) reported a high incidence of allelic loss at this chromosomal region in ovarian cancers. In addition, they independently cloned as Lot1 (Lost on transformation), the rat ortholog of ZAC, from rat ovarian surface epithelial cells that spontaneously transform in vitro.

ZAC exhibits a tumor suppressor activity characterized by induction of apoptosis and G1 arrest. This proceeds independently of known cell cycle control proteins, such as pRB, p21, p27 and p16 (5). Nevertheless, ZAC shares a number of similar functions to p53. Both proteins regulate the cell cycle, apoptosis and nuclear receptors. They interact physically and functionally with CBP and p300 that serve as integrators of multiple signaling pathways (14,15). Both p53 and ZAC are sequence-specific DNA-binding proteins and can act as transcriptional co-factors for other transcriptional activator proteins (16,17). Recently, ZAC was shown to specifically enhance the activity of p53-responsive promoters in cells expressing wild-type p53 (18). It is, therefore, possible that ZAC may act as a transcription factor with a similar mode of action to p53.

Three types of mutations have been shown to result in TNDM; paternal uniparental isodisomy of chromosome 6, paternally inherited duplication of 6q24 and a methylation defect at a CpG island overlapping exon 1 of ZAC/HYMAI (19). We showed that this CpG island is differentially methylated (20). The region is unmethylated in sperm but methylated in oocytes, a difference that persists between parental alleles throughout pre- and post implantation development suggestive of an imprint control region (ICR). There is a region within this putative ICR that exhibits a high degree of homology between mouse and human that acts as a strong transcriptional repressor when methylated. Loss of methylation at 8 CpG sites within this region was seen in five of six TNDM patients studied with a normal karyotype. We proposed that the DMR adjacent to ZAC may regulate expression of imprinted genes within the domain, and that epigenetic or genetic mutations of this region result in TNDM by affecting expression of ZAC in the pancreas and/or the pituitary (20). This is supported by the recent findings that mice overexpressing human ZAC from a transgene show impaired development of the endocrine pancreas and impaired β-cell function (21). ZAC may mediate its actions through transcriptional regulation of the PACAP type I receptor gene which is a potent insulin secretagogue and an important mediator of autocrine control of insulin secretion in the pancreatic islet (20).

During our studies on mouse Zac1 (20), we noted a similarity in the expression pattern of Zac1 with a second imprinted gene, p57Kip2 (Cdkn1c), hereafter referred to as p57Kip2. p57KIP2 is a maternally expressed gene that encodes a cyclin-dependent kinase inhibitor. The protein inhibits all G1 cyclin/Cdk complexes and negatively regulates cell proliferation, acting downstream of p53. Like ZAC, p57KIP2 may also act as a tumor suppressor through its role in regulating the cell cycle (22–24). p57KIP2 is located on human chromosome 11p15.5, a region implicated in both sporadic cancers and Beckwith–Wiedemann syndrome (BWS), a familial cancer syndrome, making it a candidate tumor suppressor gene (23). Several types of childhood tumor display a specific loss of the maternal 11p15 allele that suggests the involvement of an imprinted gene in this region (25).

BWS patients generally present with three major features: exomphalos, macroglossia and giantism. Other features include hypoglycemia, organomegaly, hemihypertrophy, genitourinary abnormalities, cleft palate and a susceptibility to embryonal tumors. In turn, TNDM is associated with intra-uterine growth failure, dehydration, hyperglycemia and failure to thrive (26,27). There have been a number of reported associations with TNDM including umbilical hernia (7%) and macroglossia (23%) (28). These two anomalies are also found in patients with BWS, which typically also involves hyperinsulinemia. The role for ZAC is a predicted gain of function with biallelic expression of ZAC in TNDM patients (since UPD of the region also results in TNDM). Loss of function of p57KIP2 in BWS is implicated in patients with point mutations within the coding sequence of the gene (29–32).

Several imprinted genes lie within the 11p15.5 region. LIT1 (KCNQ1OT1), hereafter referred to as LIT1, is a paternally expressed, antisense RNA located within the KCNQ1 gene that may regulate imprinting of the 11p15.5 domain (25). Differential methylation of the LIT1 CpG island is conserved between human and mouse (25). This differential methylation is acquired in the germ line (33) making it an excellent candidate for an ICR. Frequent loss of maternal methylation at the LIT1 CpG island has been observed in BWS patients (40–50%) (34). A targeted deletion of LIT1 leads to loss of silencing of p57Kip2 in both mice and in human cell lines (33,35). Finally, the region between the Kcnq1 and p57Kip2 shows both physical and mechanistic similarities to the imprinted domain containing Igf2r and its antisense control transcript, Air (36).

We, therefore, investigated whether the similarity in phenotypes between BWS and TNDM is a reflection of a regulatory association between ZAC and p57KIP2.

MATERIALS AND METHODSCell linesHuman ovarian cancer (HOC) cell lines (PA-1 and SKOV-3) were used in this study. The source of these cells is as described previously (37). They were grown in either DMEM or RPMI1640 supplemented with 10% fetal bovine serum.Bisulfite PCR methylation assayGenomic DNAs from primary leucocytes from 17 TNDM patients, 8 BWS patients and 2 normal individuals were prepared as described previously (20). DNA (0.3 μg) was digested with EcoRI. Bisulfite treatment was carried using the EZ DNA methylation kit (Zymo research) according to the manufacturer's instructions. PCR was carried out using the following primers. Primes sequences for the PCR: For ZAC: BS2F (ZAC), 5′-GTTTTTTATGTGTGATTGGGTTTTGGYGG-3′ and BS2R (ZAC), 5′-AAAACRCTAAAACCCCTAACRAAAAC-3′. For LIT1: BS2F (LIT1), 5′-TAGGATTTTGTTGAGGAGTTTTTTGG-3′ and BS2R (LIT1), 5′-CCACCTCACACCCAACCAATACCTCACATAC-3′. PCR condition was as follows: denaturation at 95°C for 30 s, annealing at 57°C for 30 s and extension at 72°C for 30 s for 30 cycles. PCR products were digested with the appropriate restriction enzyme (for ZAC: BssHII and RsaI; for LIT1: AciI and MaeIII) and electrophoresed on 2.5% agarose gels. PCR products were subcloned into the TA-TOPO cloning vector (Invitrogen) and sequenced.Southern blottingThe methylation status of the LIT1 CpG island was assessed by Southern blotting. Genomic DNAs were digested with BamHI alone or with the methylation sensitive enzyme NotI and electrophoresed on a 0.8% agarose gel, transferred onto Hybond-N+ filter and fixed by UV cross-linking. The filter was hybridized with the LIT1 CpG island 1.5 kb probe as described previously (25). Hybridization was carried out at 65°C overnight in Church-Gilbert buffer. Filters were washed with 0.1× SSC, 0.1% SDS at 65°C.For the transient transfection assay, we examined the methylation status of the transfected plasmids by Southern blotting. Genomic DNAs from the transfected cultured cells (unmethylated and methylated plasmids) were prepared and digested with BamHI and XbaI with or without the methylation sensitive enzyme, HpaII. The plasmids were used as the hybridization probes. Blotting and washing were described as above.In situ hybridization analysisMouse cDNA clones for Zac1 and p57Kip2 were used to prepare sense and antisense RNA by in vitro transcription using the DIG RNA labeling kit (Boehringer Mannheim). Sagittal and transverse sections of 8 μm from mouse embryos and placentas at E12.5 were used for in situ hybridization, essentially as described previously (38). Sections were counter stained with 4% eosin.Band shift assayOligonucleotides were synthesized for the human p57KIP2 CpG island including one putative ZAC binding site (accession number AC005950 and position of sequence 89690–89770) and for the LIT1 CpG island including three putative ZAC binding sites (accession number U90095 and position of sequence 67890–67810). Where stated, the oligonucleotides were methylated with SssI methylase (New England Biolabs). Nuclear extracts were prepared from 293T cells transfected with the pAc5.1/V5-His, an expression vector (Invitrogen), according to the method of Schreiber et al. (39). For a band shift assay, the unmethylated or methylated double-strand DNA oligonucleotides (0.2 ng) were end-labeled with [γ-32P]ATP and were incubated on ice for 30 min with the nuclear extract in 10 μl binding buffer [20 mM HEPES (pH 7.4), 1 mM EDTA, 3 mM MgCl2, 60 mM KCl, 10 mM 2-mercaptoethanol, 4% glycerol, 0.1% Triton X-100 and 1 μg of poly(dI–dC)]. Excess (molar excess of 20-, 100- or 200-fold) unlabeled competitor oligonucleotides of the LIT1 and p57KIP2 CpG islands, either unmethylated or methylated, were added to the reaction mixture. The DNA–protein complexes were then electrophoresed on 5% acrylamide gels.The super shift assay was performed by adding 0.2–1.0 μg ZAC antibody (Sigma) after 20 min of preincubation of nuclear extract with labeled unmethylated oligonucleotides of the human LIT1 CpG island.Methylation-dependent transcriptional repression assayTo test the effect of DNA methylation on the human LIT1 CpG island we used a transient transfection assay in HeLa cells. A fragment containing the LIT1 CpG island was generated by PCR using primers (F and R as shown in Figure 3) bearing specific restriction sites at their 5′ and 3′ ends. The primer sequences were as follows: F, 5′-ACTTGAAGGAAAGCAGGCAGGCAGGCAGGAT-3′; R, 5′-CCAACTGGAAGTTTGAGTGGAGTCCTGTTG-3′. PCR products were digested, gel-purified and subcloned into the pGL3-Promoter Firefly Luciferase reporter vector (Promega). Fragments LIT-W (869 bp NotI–HpaI), LIT-X (411 bp NotI–SacI), LIT-Y (458 bp SacI–HpaI) and LIT-Z (387 bp HpaI–R) were generated using the restriction sites present within the LIT1 CpG island. Plasmids were prepared using a midi prep kit (Qiagen). In vitro DNA methylation was performed by incubation with CpG methylase. DNA constructs (2 μg) were transfected into HeLa cells, cultured for 22 h, lyzed and luciferase reading assayed. Firefly luciferase values were normalized against a co-transfected Renilla luciferase reporter, as described in the DLR assay protocol (Promega). Each construct was tested in triplicate in each experiment and the experiment was repeated three times.Transactivation luciferase assayThe fragment used in the transcriptional activation assay was cloned into PicaGene Enhancer Vector 2 (NIPPON GENE). This fragment corresponds to LIT-X (411 bp) from within the LIT1 CpG island. It was used unmethylated or in vitro methylated (LIT-X me). The human ZAC cDNA fragment was cloned into the pSG5 expression vector (Promega). DNA constructs were transfected into cultured cells with the reporter genes using Lipofectamine (Invitrogen). Samples were taken after 48 h, lyzed and luciferase readings taken. Firefly luciferase values were normalized against the control luciferase reporter vector.

MATERIALS AND METHODS

Cell linesHuman ovarian cancer (HOC) cell lines (PA-1 and SKOV-3) were used in this study. The source of these cells is as described previously (37). They were grown in either DMEM or RPMI1640 supplemented with 10% fetal bovine serum.

Human ovarian cancer (HOC) cell lines (PA-1 and SKOV-3) were used in this study. The source of these cells is as described previously (37). They were grown in either DMEM or RPMI1640 supplemented with 10% fetal bovine serum.

Bisulfite PCR methylation assayGenomic DNAs from primary leucocytes from 17 TNDM patients, 8 BWS patients and 2 normal individuals were prepared as described previously (20). DNA (0.3 μg) was digested with EcoRI. Bisulfite treatment was carried using the EZ DNA methylation kit (Zymo research) according to the manufacturer's instructions. PCR was carried out using the following primers. Primes sequences for the PCR: For ZAC: BS2F (ZAC), 5′-GTTTTTTATGTGTGATTGGGTTTTGGYGG-3′ and BS2R (ZAC), 5′-AAAACRCTAAAACCCCTAACRAAAAC-3′. For LIT1: BS2F (LIT1), 5′-TAGGATTTTGTTGAGGAGTTTTTTGG-3′ and BS2R (LIT1), 5′-CCACCTCACACCCAACCAATACCTCACATAC-3′. PCR condition was as follows: denaturation at 95°C for 30 s, annealing at 57°C for 30 s and extension at 72°C for 30 s for 30 cycles. PCR products were digested with the appropriate restriction enzyme (for ZAC: BssHII and RsaI; for LIT1: AciI and MaeIII) and electrophoresed on 2.5% agarose gels. PCR products were subcloned into the TA-TOPO cloning vector (Invitrogen) and sequenced.

Bisulfite PCR methylation assay

Genomic DNAs from primary leucocytes from 17 TNDM patients, 8 BWS patients and 2 normal individuals were prepared as described previously (20). DNA (0.3 μg) was digested with EcoRI. Bisulfite treatment was carried using the EZ DNA methylation kit (Zymo research) according to the manufacturer's instructions. PCR was carried out using the following primers. Primes sequences for the PCR: For ZAC: BS2F (ZAC), 5′-GTTTTTTATGTGTGATTGGGTTTTGGYGG-3′ and BS2R (ZAC), 5′-AAAACRCTAAAACCCCTAACRAAAAC-3′. For LIT1: BS2F (LIT1), 5′-TAGGATTTTGTTGAGGAGTTTTTTGG-3′ and BS2R (LIT1), 5′-CCACCTCACACCCAACCAATACCTCACATAC-3′. PCR condition was as follows: denaturation at 95°C for 30 s, annealing at 57°C for 30 s and extension at 72°C for 30 s for 30 cycles. PCR products were digested with the appropriate restriction enzyme (for ZAC: BssHII and RsaI; for LIT1: AciI and MaeIII) and electrophoresed on 2.5% agarose gels. PCR products were subcloned into the TA-TOPO cloning vector (Invitrogen) and sequenced.

Southern blottingThe methylation status of the LIT1 CpG island was assessed by Southern blotting. Genomic DNAs were digested with BamHI alone or with the methylation sensitive enzyme NotI and electrophoresed on a 0.8% agarose gel, transferred onto Hybond-N+ filter and fixed by UV cross-linking. The filter was hybridized with the LIT1 CpG island 1.5 kb probe as described previously (25). Hybridization was carried out at 65°C overnight in Church-Gilbert buffer. Filters were washed with 0.1× SSC, 0.1% SDS at 65°C.For the transient transfection assay, we examined the methylation status of the transfected plasmids by Southern blotting. Genomic DNAs from the transfected cultured cells (unmethylated and methylated plasmids) were prepared and digested with BamHI and XbaI with or without the methylation sensitive enzyme, HpaII. The plasmids were used as the hybridization probes. Blotting and washing were described as above.

Southern blotting

The methylation status of the LIT1 CpG island was assessed by Southern blotting. Genomic DNAs were digested with BamHI alone or with the methylation sensitive enzyme NotI and electrophoresed on a 0.8% agarose gel, transferred onto Hybond-N+ filter and fixed by UV cross-linking. The filter was hybridized with the LIT1 CpG island 1.5 kb probe as described previously (25). Hybridization was carried out at 65°C overnight in Church-Gilbert buffer. Filters were washed with 0.1× SSC, 0.1% SDS at 65°C.

For the transient transfection assay, we examined the methylation status of the transfected plasmids by Southern blotting. Genomic DNAs from the transfected cultured cells (unmethylated and methylated plasmids) were prepared and digested with BamHI and XbaI with or without the methylation sensitive enzyme, HpaII. The plasmids were used as the hybridization probes. Blotting and washing were described as above.

In situ hybridization analysisMouse cDNA clones for Zac1 and p57Kip2 were used to prepare sense and antisense RNA by in vitro transcription using the DIG RNA labeling kit (Boehringer Mannheim). Sagittal and transverse sections of 8 μm from mouse embryos and placentas at E12.5 were used for in situ hybridization, essentially as described previously (38). Sections were counter stained with 4% eosin.

In situ hybridization analysis

Mouse cDNA clones for Zac1 and p57Kip2 were used to prepare sense and antisense RNA by in vitro transcription using the DIG RNA labeling kit (Boehringer Mannheim). Sagittal and transverse sections of 8 μm from mouse embryos and placentas at E12.5 were used for in situ hybridization, essentially as described previously (38). Sections were counter stained with 4% eosin.

Band shift assayOligonucleotides were synthesized for the human p57KIP2 CpG island including one putative ZAC binding site (accession number AC005950 and position of sequence 89690–89770) and for the LIT1 CpG island including three putative ZAC binding sites (accession number U90095 and position of sequence 67890–67810). Where stated, the oligonucleotides were methylated with SssI methylase (New England Biolabs). Nuclear extracts were prepared from 293T cells transfected with the pAc5.1/V5-His, an expression vector (Invitrogen), according to the method of Schreiber et al. (39). For a band shift assay, the unmethylated or methylated double-strand DNA oligonucleotides (0.2 ng) were end-labeled with [γ-32P]ATP and were incubated on ice for 30 min with the nuclear extract in 10 μl binding buffer [20 mM HEPES (pH 7.4), 1 mM EDTA, 3 mM MgCl2, 60 mM KCl, 10 mM 2-mercaptoethanol, 4% glycerol, 0.1% Triton X-100 and 1 μg of poly(dI–dC)]. Excess (molar excess of 20-, 100- or 200-fold) unlabeled competitor oligonucleotides of the LIT1 and p57KIP2 CpG islands, either unmethylated or methylated, were added to the reaction mixture. The DNA–protein complexes were then electrophoresed on 5% acrylamide gels.The super shift assay was performed by adding 0.2–1.0 μg ZAC antibody (Sigma) after 20 min of preincubation of nuclear extract with labeled unmethylated oligonucleotides of the human LIT1 CpG island.

Band shift assay

Oligonucleotides were synthesized for the human p57KIP2 CpG island including one putative ZAC binding site (accession number AC005950 and position of sequence 89690–89770) and for the LIT1 CpG island including three putative ZAC binding sites (accession number U90095 and position of sequence 67890–67810). Where stated, the oligonucleotides were methylated with SssI methylase (New England Biolabs). Nuclear extracts were prepared from 293T cells transfected with the pAc5.1/V5-His, an expression vector (Invitrogen), according to the method of Schreiber et al. (39). For a band shift assay, the unmethylated or methylated double-strand DNA oligonucleotides (0.2 ng) were end-labeled with [γ-32P]ATP and were incubated on ice for 30 min with the nuclear extract in 10 μl binding buffer [20 mM HEPES (pH 7.4), 1 mM EDTA, 3 mM MgCl2, 60 mM KCl, 10 mM 2-mercaptoethanol, 4% glycerol, 0.1% Triton X-100 and 1 μg of poly(dI–dC)]. Excess (molar excess of 20-, 100- or 200-fold) unlabeled competitor oligonucleotides of the LIT1 and p57KIP2 CpG islands, either unmethylated or methylated, were added to the reaction mixture. The DNA–protein complexes were then electrophoresed on 5% acrylamide gels.

The super shift assay was performed by adding 0.2–1.0 μg ZAC antibody (Sigma) after 20 min of preincubation of nuclear extract with labeled unmethylated oligonucleotides of the human LIT1 CpG island.

Methylation-dependent transcriptional repression assayTo test the effect of DNA methylation on the human LIT1 CpG island we used a transient transfection assay in HeLa cells. A fragment containing the LIT1 CpG island was generated by PCR using primers (F and R as shown in Figure 3) bearing specific restriction sites at their 5′ and 3′ ends. The primer sequences were as follows: F, 5′-ACTTGAAGGAAAGCAGGCAGGCAGGCAGGAT-3′; R, 5′-CCAACTGGAAGTTTGAGTGGAGTCCTGTTG-3′. PCR products were digested, gel-purified and subcloned into the pGL3-Promoter Firefly Luciferase reporter vector (Promega). Fragments LIT-W (869 bp NotI–HpaI), LIT-X (411 bp NotI–SacI), LIT-Y (458 bp SacI–HpaI) and LIT-Z (387 bp HpaI–R) were generated using the restriction sites present within the LIT1 CpG island. Plasmids were prepared using a midi prep kit (Qiagen). In vitro DNA methylation was performed by incubation with CpG methylase. DNA constructs (2 μg) were transfected into HeLa cells, cultured for 22 h, lyzed and luciferase reading assayed. Firefly luciferase values were normalized against a co-transfected Renilla luciferase reporter, as described in the DLR assay protocol (Promega). Each construct was tested in triplicate in each experiment and the experiment was repeated three times.

Methylation-dependent transcriptional repression assay

To test the effect of DNA methylation on the human LIT1 CpG island we used a transient transfection assay in HeLa cells. A fragment containing the LIT1 CpG island was generated by PCR using primers (F and R as shown in Figure 3) bearing specific restriction sites at their 5′ and 3′ ends. The primer sequences were as follows: F, 5′-ACTTGAAGGAAAGCAGGCAGGCAGGCAGGAT-3′; R, 5′-CCAACTGGAAGTTTGAGTGGAGTCCTGTTG-3′. PCR products were digested, gel-purified and subcloned into the pGL3-Promoter Firefly Luciferase reporter vector (Promega). Fragments LIT-W (869 bp NotI–HpaI), LIT-X (411 bp NotI–SacI), LIT-Y (458 bp SacI–HpaI) and LIT-Z (387 bp HpaI–R) were generated using the restriction sites present within the LIT1 CpG island. Plasmids were prepared using a midi prep kit (Qiagen). In vitro DNA methylation was performed by incubation with CpG methylase. DNA constructs (2 μg) were transfected into HeLa cells, cultured for 22 h, lyzed and luciferase reading assayed. Firefly luciferase values were normalized against a co-transfected Renilla luciferase reporter, as described in the DLR assay protocol (Promega). Each construct was tested in triplicate in each experiment and the experiment was repeated three times.

Transactivation luciferase assayThe fragment used in the transcriptional activation assay was cloned into PicaGene Enhancer Vector 2 (NIPPON GENE). This fragment corresponds to LIT-X (411 bp) from within the LIT1 CpG island. It was used unmethylated or in vitro methylated (LIT-X me). The human ZAC cDNA fragment was cloned into the pSG5 expression vector (Promega). DNA constructs were transfected into cultured cells with the reporter genes using Lipofectamine (Invitrogen). Samples were taken after 48 h, lyzed and luciferase readings taken. Firefly luciferase values were normalized against the control luciferase reporter vector.

Transactivation luciferase assay

The fragment used in the transcriptional activation assay was cloned into PicaGene Enhancer Vector 2 (NIPPON GENE). This fragment corresponds to LIT-X (411 bp) from within the LIT1 CpG island. It was used unmethylated or in vitro methylated (LIT-X me). The human ZAC cDNA fragment was cloned into the pSG5 expression vector (Promega). DNA constructs were transfected into cultured cells with the reporter genes using Lipofectamine (Invitrogen). Samples were taken after 48 h, lyzed and luciferase readings taken. Firefly luciferase values were normalized against the control luciferase reporter vector.

RESULTSComparison of the expression pattern of the mouse Zac1 and p57Kip2 genesZac1 and p57Kip2 appear to be expressed in many of the same tissues (7,20,40,41). Given the precedent for co-ordinated regulation of imprinted genes (4), we decided to investigate this more rigorously. We first compared the expression patterns of Zac1 and p57Kip2 in adjacent sagittal sections of E12.5 (embryonic day 12.5) mouse embryos. The genes showed a strikingly similar expression pattern in many tissues. There was strong expression in the lung, tongue, sclerotome, telencephalon and the labyrinth layer of the placenta (Figure 1A, B and D). Both genes were also expressed in Rathke's pouch (Figure 1D), the lumen of cardiac ventricle and bronchus of the lung (Figure 1E) and the epithelium of the adult ovary (Figure 1F). This is the developmental origin of ovarian cancers. We detected some differences in the pattern of expression of the two genes. p57Kip2 was detected in the neural epithelium (Figure 1G), the equatorial region of the lens (Figure 1H) and retina (Figure 1I), the nasal epithelium (Figure 1J) and mesonephrum (Figure 1K) where expression of Zac1 was absent.The ZAC protein binds within the LIT1 CpG islandThe striking similarity in expression between mouse Zac1 and p57Kip2 raises the possibility of an interaction between two genes. The presence of a zinc finger motif within the ZAC protein suggests a DNA-binding capability. The consensus binding sequence for the ZAC protein has been identified as GGGGCCTC (17). We found that the CpG island of the human p57KIP2 gene contains two ZAC binding sites (Figure 2A and B). Given the proposed role for the LIT1 transcript in regulating the expression of p57KIP2, we also looked for ZAC binding sites within the LIT1 CpG island and found eight potential binding sites (Figure 2A and B). We used an electromobility gel shift assay to ask whether the human ZAC protein could bind either of these regions.Cell extracts containing the ZAC protein were incubated with the LIT1 and p57KIP2 CpG island oligonucleotides containing the putative ZAC binding sites. The probes were either unmethylated (M−) or methylated (M+). A band shift analysis was performed (Figure 2C). The ZAC protein bound preferentially to unmethylated, a portion of the LIT1 CpG island probe that contained the putative ZAC binding sites. No detectable binding was observed to the p57KIP2 CpG island probe (data not shown). The binding was specific as only the unlabeled, unmethylated LIT1 CpG island probe competed out the binding and not the p57KIP2 CpG island probe or the methylated LIT1 CpG island probe. Furthermore, specificity of this DNA–protein complex was confirmed in a super shift assay using an anti-ZAC antibody (Figure 2D). This suggests that ZAC protein may bind to the LIT1 CpG island in vivo and perhaps indirectly regulate p57KIP2 expression.Methylation-dependent transcriptional repression assay of the LIT1 CpG islandThere is evidence to suggest that the LIT1 transcript may regulate imprinting of the 11p15.5 domain. This differentially methylated LIT1 CpG island is conserved between human and mouse (25). The differential methylation is acquired in the germ line (33) making it an excellent candidate for an ICR.To determine whether the LIT1 differentially methylated region has the capacity to function as a regulatory element, we tested it in a transient transfection assay in HeLa cells in which a reporter gene (firefly luciferase) was expressed from an SV40 promoter. Unmethylated or methylated fragments from the human LIT1 CpG island were assayed for their effect on transcription of the reporter (Figure 3A). The maintenance of their methylation status after transfection was confirmed by Southern blotting (Figure 3B). Among the fragments we tested, the LIT-X (411 bp NotI–SacI), which contains 57 CpG sites, was shown to act as a strong transcriptional silencer, but only when methylated (Figure 3A). Significantly, the cluster of the ZAC binding sites is present within this region. In contrast, methylation of the LIT-Y fragment, which contains 55 CpG sites, caused negligible repression of the reporter gene (Figure 3A). Interestingly, the LIT-X fragment is contained within a larger fragment (LIT-W) which has a lower repression activity. This may be due to the inclusion of the LIT1 promoter within the larger LIT-W fragment (M. Oshimura, unpublished data). This promoter may harbor both transcriptional activation and repression properties in this assay. This indicated that LIT-X within the ICR can function as a cis-regulatory element and capable of strong transcriptional silencing in this assay.Although a large CpG island is present in the 10th intron of KvLQT1 in both human and mice, its primary sequence is poorly conserved. However, in mice the region containing the DNase I hypersensitive sites within the Lit1 CpG island, which are upstream of the transcription initiation site, is critical for promoter activity (42,43). The transcriptional initiation site of human LIT1 gene has been determined in an RNase protection assay and by primer extension analysis by M. Oshimura et al. (personal communication). The transcription initiation site is at 67499 nt (U90095) and lies within the LIT1 CpG island. Also present are two putative CCAAT boxes and a Sp1-binding site but no TATA-box. There is also an AP-1 site downstream of this region. The distal region including F-SacI fragment was previously reported to act as a silencer (44). In mouse, a fragment containing both the hypersensitive sites and the transcriptional start site displayed the strong silencing activity (42). According to the model proposed by Constancia et al. (45) repeat sequences may play a role in the methylation of CpG islands. It is interesting that this distal region also contains two direct repeats.Cell transfection assayWe established an assay to examine whether ZAC binding activates transcription from the LIT1 promoter. We used two ovarian cancer cell lines (PA1 and SKOV) in which we had shown that endogenous ZAC expression was absent (46). These lines were transfected with a luciferase reporter vector with the LIT-X fragment. We also transiently transfected a ZAC expression construct into these cells and then measured luciferase activity. ZAC behaved as a transcriptional activator (Figure 4) consistent with our binding assay results. Furthermore, when we assayed whether the ZAC protein would activate the methylated LIT1 reporter in the SKOV ovarian cancer cell line, we did not observe any activation of the reporter suggesting that ZAC acts in a signaling pathway only on the unmethylated, paternal LIT1 allele in vivo.Epigenetic changes at the ZAC ICR in BWS and the LIT1 ICR in TNDM patientsThe similarity in phenotypes between BWS and TNDM may reflect a regulatory association between ZAC and p57KIP2. Our data suggest a possible pathway where over expression of ZAC, possibly by loss of methylation of the ZAC CpG island, may hyperactivate the LIT1 gene and silence p57KIP2. We therefore examined the imprinting status of ZAC in BWS patients and LIT1 in TNDM patients.We examined the methylation status of the ZAC and LIT1 DMRs in both types of patients. We performed the restriction enzyme digest and sequencing after PCR of bisulfite-modified genomic DNA (47). After bisulfite treatment of genomic DNA from 8 BWS and 17 TNDM patients, PCR products were digested with AciI and MseIII for the LIT1 CpG island and RsaI and BssHII for the ZAC CpG island (Figure 5A). In this assay, a methylated cytosine will maintain the enzyme recognition site and will be cut, whereas the unmethylated cytocine is converted to thymidine destroying the recognition site. In addition, sodium bisulfite-treated DNA was PCR amplified, subcloned, sequenced and the methylation status was determined at 22 CpG sites in the ZAC locus and at 24 CpG sites within the LIT1 locus. All 8 BWS patients showed normal methylation patterns in ZAC (Figure 5B) but 2 TNDM patients showed loss of methylation in the LIT1 CpG locus (Figure 5B, data shown for AciI). One of these patients exhibited UPD of chromosome 6 (UPD6). The other patient had a normal karyotype and loss of methylation on the maternal ZAC DMR region. Sequencing revealed that all 24 LIT1 CpG sites were predominantly unmethylated in both patients (Figure 5C). The second patient was characterized with TNDM complicated with umbilical hernia and macroglossia, features commonly seen in BWS patients. Finally, we confirmed loss of methylation at the LIT1 locus in these two patients by a second technique, Southern blotting. (Figure 5D). Both patients showed loss of differential methylation of the LIT1 locus in association with presumed biallelic expression of ZAC.

Comparison of the expression pattern of the mouse Zac1 and p57Kip2 genesZac1 and p57Kip2 appear to be expressed in many of the same tissues (7,20,40,41). Given the precedent for co-ordinated regulation of imprinted genes (4), we decided to investigate this more rigorously. We first compared the expression patterns of Zac1 and p57Kip2 in adjacent sagittal sections of E12.5 (embryonic day 12.5) mouse embryos. The genes showed a strikingly similar expression pattern in many tissues. There was strong expression in the lung, tongue, sclerotome, telencephalon and the labyrinth layer of the placenta (Figure 1A, B and D). Both genes were also expressed in Rathke's pouch (Figure 1D), the lumen of cardiac ventricle and bronchus of the lung (Figure 1E) and the epithelium of the adult ovary (Figure 1F). This is the developmental origin of ovarian cancers. We detected some differences in the pattern of expression of the two genes. p57Kip2 was detected in the neural epithelium (Figure 1G), the equatorial region of the lens (Figure 1H) and retina (Figure 1I), the nasal epithelium (Figure 1J) and mesonephrum (Figure 1K) where expression of Zac1 was absent.

Comparison of the expression pattern of the mouse Zac1 and p57Kip2 genes

Zac1 and p57Kip2 appear to be expressed in many of the same tissues (7,20,40,41). Given the precedent for co-ordinated regulation of imprinted genes (4), we decided to investigate this more rigorously. We first compared the expression patterns of Zac1 and p57Kip2 in adjacent sagittal sections of E12.5 (embryonic day 12.5) mouse embryos. The genes showed a strikingly similar expression pattern in many tissues. There was strong expression in the lung, tongue, sclerotome, telencephalon and the labyrinth layer of the placenta (Figure 1A, B and D). Both genes were also expressed in Rathke's pouch (Figure 1D), the lumen of cardiac ventricle and bronchus of the lung (Figure 1E) and the epithelium of the adult ovary (Figure 1F). This is the developmental origin of ovarian cancers. We detected some differences in the pattern of expression of the two genes. p57Kip2 was detected in the neural epithelium (Figure 1G), the equatorial region of the lens (Figure 1H) and retina (Figure 1I), the nasal epithelium (Figure 1J) and mesonephrum (Figure 1K) where expression of Zac1 was absent.

The ZAC protein binds within the LIT1 CpG islandThe striking similarity in expression between mouse Zac1 and p57Kip2 raises the possibility of an interaction between two genes. The presence of a zinc finger motif within the ZAC protein suggests a DNA-binding capability. The consensus binding sequence for the ZAC protein has been identified as GGGGCCTC (17). We found that the CpG island of the human p57KIP2 gene contains two ZAC binding sites (Figure 2A and B). Given the proposed role for the LIT1 transcript in regulating the expression of p57KIP2, we also looked for ZAC binding sites within the LIT1 CpG island and found eight potential binding sites (Figure 2A and B). We used an electromobility gel shift assay to ask whether the human ZAC protein could bind either of these regions.Cell extracts containing the ZAC protein were incubated with the LIT1 and p57KIP2 CpG island oligonucleotides containing the putative ZAC binding sites. The probes were either unmethylated (M−) or methylated (M+). A band shift analysis was performed (Figure 2C). The ZAC protein bound preferentially to unmethylated, a portion of the LIT1 CpG island probe that contained the putative ZAC binding sites. No detectable binding was observed to the p57KIP2 CpG island probe (data not shown). The binding was specific as only the unlabeled, unmethylated LIT1 CpG island probe competed out the binding and not the p57KIP2 CpG island probe or the methylated LIT1 CpG island probe. Furthermore, specificity of this DNA–protein complex was confirmed in a super shift assay using an anti-ZAC antibody (Figure 2D). This suggests that ZAC protein may bind to the LIT1 CpG island in vivo and perhaps indirectly regulate p57KIP2 expression.

The ZAC protein binds within the LIT1 CpG island

The striking similarity in expression between mouse Zac1 and p57Kip2 raises the possibility of an interaction between two genes. The presence of a zinc finger motif within the ZAC protein suggests a DNA-binding capability. The consensus binding sequence for the ZAC protein has been identified as GGGGCCTC (17). We found that the CpG island of the human p57KIP2 gene contains two ZAC binding sites (Figure 2A and B). Given the proposed role for the LIT1 transcript in regulating the expression of p57KIP2, we also looked for ZAC binding sites within the LIT1 CpG island and found eight potential binding sites (Figure 2A and B). We used an electromobility gel shift assay to ask whether the human ZAC protein could bind either of these regions.

Cell extracts containing the ZAC protein were incubated with the LIT1 and p57KIP2 CpG island oligonucleotides containing the putative ZAC binding sites. The probes were either unmethylated (M−) or methylated (M+). A band shift analysis was performed (Figure 2C). The ZAC protein bound preferentially to unmethylated, a portion of the LIT1 CpG island probe that contained the putative ZAC binding sites. No detectable binding was observed to the p57KIP2 CpG island probe (data not shown). The binding was specific as only the unlabeled, unmethylated LIT1 CpG island probe competed out the binding and not the p57KIP2 CpG island probe or the methylated LIT1 CpG island probe. Furthermore, specificity of this DNA–protein complex was confirmed in a super shift assay using an anti-ZAC antibody (Figure 2D). This suggests that ZAC protein may bind to the LIT1 CpG island in vivo and perhaps indirectly regulate p57KIP2 expression.

Methylation-dependent transcriptional repression assay of the LIT1 CpG islandThere is evidence to suggest that the LIT1 transcript may regulate imprinting of the 11p15.5 domain. This differentially methylated LIT1 CpG island is conserved between human and mouse (25). The differential methylation is acquired in the germ line (33) making it an excellent candidate for an ICR.To determine whether the LIT1 differentially methylated region has the capacity to function as a regulatory element, we tested it in a transient transfection assay in HeLa cells in which a reporter gene (firefly luciferase) was expressed from an SV40 promoter. Unmethylated or methylated fragments from the human LIT1 CpG island were assayed for their effect on transcription of the reporter (Figure 3A). The maintenance of their methylation status after transfection was confirmed by Southern blotting (Figure 3B). Among the fragments we tested, the LIT-X (411 bp NotI–SacI), which contains 57 CpG sites, was shown to act as a strong transcriptional silencer, but only when methylated (Figure 3A). Significantly, the cluster of the ZAC binding sites is present within this region. In contrast, methylation of the LIT-Y fragment, which contains 55 CpG sites, caused negligible repression of the reporter gene (Figure 3A). Interestingly, the LIT-X fragment is contained within a larger fragment (LIT-W) which has a lower repression activity. This may be due to the inclusion of the LIT1 promoter within the larger LIT-W fragment (M. Oshimura, unpublished data). This promoter may harbor both transcriptional activation and repression properties in this assay. This indicated that LIT-X within the ICR can function as a cis-regulatory element and capable of strong transcriptional silencing in this assay.Although a large CpG island is present in the 10th intron of KvLQT1 in both human and mice, its primary sequence is poorly conserved. However, in mice the region containing the DNase I hypersensitive sites within the Lit1 CpG island, which are upstream of the transcription initiation site, is critical for promoter activity (42,43). The transcriptional initiation site of human LIT1 gene has been determined in an RNase protection assay and by primer extension analysis by M. Oshimura et al. (personal communication). The transcription initiation site is at 67499 nt (U90095) and lies within the LIT1 CpG island. Also present are two putative CCAAT boxes and a Sp1-binding site but no TATA-box. There is also an AP-1 site downstream of this region. The distal region including F-SacI fragment was previously reported to act as a silencer (44). In mouse, a fragment containing both the hypersensitive sites and the transcriptional start site displayed the strong silencing activity (42). According to the model proposed by Constancia et al. (45) repeat sequences may play a role in the methylation of CpG islands. It is interesting that this distal region also contains two direct repeats.

Methylation-dependent transcriptional repression assay of the LIT1 CpG island

There is evidence to suggest that the LIT1 transcript may regulate imprinting of the 11p15.5 domain. This differentially methylated LIT1 CpG island is conserved between human and mouse (25). The differential methylation is acquired in the germ line (33) making it an excellent candidate for an ICR.

To determine whether the LIT1 differentially methylated region has the capacity to function as a regulatory element, we tested it in a transient transfection assay in HeLa cells in which a reporter gene (firefly luciferase) was expressed from an SV40 promoter. Unmethylated or methylated fragments from the human LIT1 CpG island were assayed for their effect on transcription of the reporter (Figure 3A). The maintenance of their methylation status after transfection was confirmed by Southern blotting (Figure 3B). Among the fragments we tested, the LIT-X (411 bp NotI–SacI), which contains 57 CpG sites, was shown to act as a strong transcriptional silencer, but only when methylated (Figure 3A). Significantly, the cluster of the ZAC binding sites is present within this region. In contrast, methylation of the LIT-Y fragment, which contains 55 CpG sites, caused negligible repression of the reporter gene (Figure 3A). Interestingly, the LIT-X fragment is contained within a larger fragment (LIT-W) which has a lower repression activity. This may be due to the inclusion of the LIT1 promoter within the larger LIT-W fragment (M. Oshimura, unpublished data). This promoter may harbor both transcriptional activation and repression properties in this assay. This indicated that LIT-X within the ICR can function as a cis-regulatory element and capable of strong transcriptional silencing in this assay.

Although a large CpG island is present in the 10th intron of KvLQT1 in both human and mice, its primary sequence is poorly conserved. However, in mice the region containing the DNase I hypersensitive sites within the Lit1 CpG island, which are upstream of the transcription initiation site, is critical for promoter activity (42,43). The transcriptional initiation site of human LIT1 gene has been determined in an RNase protection assay and by primer extension analysis by M. Oshimura et al. (personal communication). The transcription initiation site is at 67499 nt (U90095) and lies within the LIT1 CpG island. Also present are two putative CCAAT boxes and a Sp1-binding site but no TATA-box. There is also an AP-1 site downstream of this region. The distal region including F-SacI fragment was previously reported to act as a silencer (44). In mouse, a fragment containing both the hypersensitive sites and the transcriptional start site displayed the strong silencing activity (42). According to the model proposed by Constancia et al. (45) repeat sequences may play a role in the methylation of CpG islands. It is interesting that this distal region also contains two direct repeats.

Cell transfection assayWe established an assay to examine whether ZAC binding activates transcription from the LIT1 promoter. We used two ovarian cancer cell lines (PA1 and SKOV) in which we had shown that endogenous ZAC expression was absent (46). These lines were transfected with a luciferase reporter vector with the LIT-X fragment. We also transiently transfected a ZAC expression construct into these cells and then measured luciferase activity. ZAC behaved as a transcriptional activator (Figure 4) consistent with our binding assay results. Furthermore, when we assayed whether the ZAC protein would activate the methylated LIT1 reporter in the SKOV ovarian cancer cell line, we did not observe any activation of the reporter suggesting that ZAC acts in a signaling pathway only on the unmethylated, paternal LIT1 allele in vivo.

Cell transfection assay

We established an assay to examine whether ZAC binding activates transcription from the LIT1 promoter. We used two ovarian cancer cell lines (PA1 and SKOV) in which we had shown that endogenous ZAC expression was absent (46). These lines were transfected with a luciferase reporter vector with the LIT-X fragment. We also transiently transfected a ZAC expression construct into these cells and then measured luciferase activity. ZAC behaved as a transcriptional activator (Figure 4) consistent with our binding assay results. Furthermore, when we assayed whether the ZAC protein would activate the methylated LIT1 reporter in the SKOV ovarian cancer cell line, we did not observe any activation of the reporter suggesting that ZAC acts in a signaling pathway only on the unmethylated, paternal LIT1 allele in vivo.

Epigenetic changes at the ZAC ICR in BWS and the LIT1 ICR in TNDM patientsThe similarity in phenotypes between BWS and TNDM may reflect a regulatory association between ZAC and p57KIP2. Our data suggest a possible pathway where over expression of ZAC, possibly by loss of methylation of the ZAC CpG island, may hyperactivate the LIT1 gene and silence p57KIP2. We therefore examined the imprinting status of ZAC in BWS patients and LIT1 in TNDM patients.We examined the methylation status of the ZAC and LIT1 DMRs in both types of patients. We performed the restriction enzyme digest and sequencing after PCR of bisulfite-modified genomic DNA (47). After bisulfite treatment of genomic DNA from 8 BWS and 17 TNDM patients, PCR products were digested with AciI and MseIII for the LIT1 CpG island and RsaI and BssHII for the ZAC CpG island (Figure 5A). In this assay, a methylated cytosine will maintain the enzyme recognition site and will be cut, whereas the unmethylated cytocine is converted to thymidine destroying the recognition site. In addition, sodium bisulfite-treated DNA was PCR amplified, subcloned, sequenced and the methylation status was determined at 22 CpG sites in the ZAC locus and at 24 CpG sites within the LIT1 locus. All 8 BWS patients showed normal methylation patterns in ZAC (Figure 5B) but 2 TNDM patients showed loss of methylation in the LIT1 CpG locus (Figure 5B, data shown for AciI). One of these patients exhibited UPD of chromosome 6 (UPD6). The other patient had a normal karyotype and loss of methylation on the maternal ZAC DMR region. Sequencing revealed that all 24 LIT1 CpG sites were predominantly unmethylated in both patients (Figure 5C). The second patient was characterized with TNDM complicated with umbilical hernia and macroglossia, features commonly seen in BWS patients. Finally, we confirmed loss of methylation at the LIT1 locus in these two patients by a second technique, Southern blotting. (Figure 5D). Both patients showed loss of differential methylation of the LIT1 locus in association with presumed biallelic expression of ZAC.

Epigenetic changes at the ZAC ICR in BWS and the LIT1 ICR in TNDM patients

The similarity in phenotypes between BWS and TNDM may reflect a regulatory association between ZAC and p57KIP2. Our data suggest a possible pathway where over expression of ZAC, possibly by loss of methylation of the ZAC CpG island, may hyperactivate the LIT1 gene and silence p57KIP2. We therefore examined the imprinting status of ZAC in BWS patients and LIT1 in TNDM patients.

We examined the methylation status of the ZAC and LIT1 DMRs in both types of patients. We performed the restriction enzyme digest and sequencing after PCR of bisulfite-modified genomic DNA (47). After bisulfite treatment of genomic DNA from 8 BWS and 17 TNDM patients, PCR products were digested with AciI and MseIII for the LIT1 CpG island and RsaI and BssHII for the ZAC CpG island (Figure 5A). In this assay, a methylated cytosine will maintain the enzyme recognition site and will be cut, whereas the unmethylated cytocine is converted to thymidine destroying the recognition site. In addition, sodium bisulfite-treated DNA was PCR amplified, subcloned, sequenced and the methylation status was determined at 22 CpG sites in the ZAC locus and at 24 CpG sites within the LIT1 locus. All 8 BWS patients showed normal methylation patterns in ZAC (Figure 5B) but 2 TNDM patients showed loss of methylation in the LIT1 CpG locus (Figure 5B, data shown for AciI). One of these patients exhibited UPD of chromosome 6 (UPD6). The other patient had a normal karyotype and loss of methylation on the maternal ZAC DMR region. Sequencing revealed that all 24 LIT1 CpG sites were predominantly unmethylated in both patients (Figure 5C). The second patient was characterized with TNDM complicated with umbilical hernia and macroglossia, features commonly seen in BWS patients. Finally, we confirmed loss of methylation at the LIT1 locus in these two patients by a second technique, Southern blotting. (Figure 5D). Both patients showed loss of differential methylation of the LIT1 locus in association with presumed biallelic expression of ZAC.

DISCUSSIONWhile investigating the function of ZAC, an imprinted tumor suppressor gene, we found that this gene has a strikingly similar expression pattern to a second imprinted gene, p57Kip2. In humans, ZAC is a candidate in TNDM while p57KIP2 has been shown to play a role in BWS. In addition to the similarity in expression, we noticed that some of the features seen in BWS, such as tumors, intrauterine growth abnormalities, umbilical hernias, macroglossia and defects in insulin are also found in TNDM patients metabolism.We were unable to demonstrate a direct interaction between the ZAC transcription factor and the p57KIP2 gene. However, we found that ZAC bound to, and activated expression of a third imprinted gene, LIT1. LIT1 has been shown to act an ICR that negatively regulates the p57KIP2 gene in cis. Though few BWS patients have genetic mutations in the p57KIP2 gene itself, nearly 60% showed loss of methylation of the LIT1 CpG island. By showing that ZAC directly binds to the LIT1 CpG island in vitro and activates transcription, we have identified a possible role for ZAC in the regulation of p57KIP2.We have shown that some patients with TNDM (2 out of 17) have defects in methylation of LIT1 CpG island, supporting a link between these genes. Furthermore, ZAC has similarities with p53, a known tumor suppressor gene. p53 regulates p21, a member of the same family of cyclin-dependent kinase inhibitors as p57KIP2. Since our data suggest that ZAC may regulate p57KIP2, the two genes may form part of a signaling pathway for regulating cell cycle progression. p73, a homologue of p53, was suggested as an imprinted gene (48) and more recently has been shown to directly regulate p57KIP2 (49). Further work is required to link these genes but our initial conclusion is that we have potentially identified a novel imprinted pathway similar to the Igf2/Igf2r pathway.Finally, our finding that ZAC may indirectly regulate p57KIP2 indicates a potential role for ZAC in BWS. While we did not detect changes in DNA methylation at the ZAC CpG island in BWS patients, we were only able to survey a small number of patients. We did find changes in LIT1 DNA methylation in two TNDM patients suggesting a more global defect in imprinted DNA methylation in these patients.

While investigating the function of ZAC, an imprinted tumor suppressor gene, we found that this gene has a strikingly similar expression pattern to a second imprinted gene, p57Kip2. In humans, ZAC is a candidate in TNDM while p57KIP2 has been shown to play a role in BWS. In addition to the similarity in expression, we noticed that some of the features seen in BWS, such as tumors, intrauterine growth abnormalities, umbilical hernias, macroglossia and defects in insulin are also found in TNDM patients metabolism.

We were unable to demonstrate a direct interaction between the ZAC transcription factor and the p57KIP2 gene. However, we found that ZAC bound to, and activated expression of a third imprinted gene, LIT1. LIT1 has been shown to act an ICR that negatively regulates the p57KIP2 gene in cis. Though few BWS patients have genetic mutations in the p57KIP2 gene itself, nearly 60% showed loss of methylation of the LIT1 CpG island. By showing that ZAC directly binds to the LIT1 CpG island in vitro and activates transcription, we have identified a possible role for ZAC in the regulation of p57KIP2.

We have shown that some patients with TNDM (2 out of 17) have defects in methylation of LIT1 CpG island, supporting a link between these genes. Furthermore, ZAC has similarities with p53, a known tumor suppressor gene. p53 regulates p21, a member of the same family of cyclin-dependent kinase inhibitors as p57KIP2. Since our data suggest that ZAC may regulate p57KIP2, the two genes may form part of a signaling pathway for regulating cell cycle progression. p73, a homologue of p53, was suggested as an imprinted gene (48) and more recently has been shown to directly regulate p57KIP2 (49). Further work is required to link these genes but our initial conclusion is that we have potentially identified a novel imprinted pathway similar to the Igf2/Igf2r pathway.

Finally, our finding that ZAC may indirectly regulate p57KIP2 indicates a potential role for ZAC in BWS. While we did not detect changes in DNA methylation at the ZAC CpG island in BWS patients, we were only able to survey a small number of patients. We did find changes in LIT1 DNA methylation in two TNDM patients suggesting a more global defect in imprinted DNA methylation in these patients.
