---
pmid: '31354687'
title: Transcriptional Response of Streptomyces coelicolor to Rapid Chromosome Relaxation
  or Long-Term Supercoiling Imbalance.
authors:
- Szafran MJ
- Gongerowska M
- Małecki T
- Elliot M
- Jakimowicz D
journal: Front Microbiol
year: '2019'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6637917
doi: 10.3389/fmicb.2019.01605
---

# Transcriptional Response of Streptomyces coelicolor to Rapid Chromosome Relaxation or Long-Term Supercoiling Imbalance.
**Authors:** Szafran MJ, Gongerowska M, Małecki T, Elliot M, Jakimowicz D
**Journal:** Front Microbiol (2019)
**DOI:** [10.3389/fmicb.2019.01605](https://doi.org/10.3389/fmicb.2019.01605)
**PMC:** [PMC6637917](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6637917/)

## Abstract

1. Front Microbiol. 2019 Jul 11;10:1605. doi: 10.3389/fmicb.2019.01605.
eCollection  2019.

Transcriptional Response of Streptomyces coelicolor to Rapid Chromosome 
Relaxation or Long-Term Supercoiling Imbalance.

Szafran MJ(1), Gongerowska M(1), Małecki T(1), Elliot M(2), Jakimowicz D(1).

Author information:
(1)Laboratory of Molecular Microbiology, Faculty of Biotechnology, University of 
Wrocław, Wrocław, Poland.
(2)Department of Biology, M.G. DeGroote Institute for Infectious Disease 
Research, McMaster University, Hamilton, ON, Canada.

Negative DNA supercoiling allows chromosome condensation and facilitates DNA 
unwinding, which is required for the occurrence of DNA transaction processes, 
i.e., DNA replication, transcription and recombination. In bacteria, changes in 
chromosome supercoiling impact global gene expression; however, the limited 
studies on the global transcriptional response have focused mostly on pathogenic 
species and have reported various fractions of affected genes. Furthermore, the 
transcriptional response to long-term supercoiling imbalance is still poorly 
understood. Here, we address the transcriptional response to both 
novobiocin-induced rapid chromosome relaxation or long-term topological 
imbalance, both increased and decreased supercoiling, in environmental 
antibiotic-producing bacteria belonging to the Streptomyces genus. During the 
Streptomyces complex developmental cycle, multiple copies of GC-rich linear 
chromosomes present in hyphal cells undergo profound topological changes, from 
being loosely condensed in vegetative hyphae, to being highly compacted in 
spores. Moreover, changes in chromosomal supercoiling have been suggested to be 
associated with the control of antibiotic production and environmental stress 
response. Remarkably, in S. coelicolor, a model Streptomyces species, 
topoisomerase I (TopA) is solely responsible for the removal of negative DNA 
supercoils. Using a S. coelicolor strain in which topA transcription is under 
the control of an inducible promoter, we identified genes involved in the 
transcriptional response to long-term supercoiling imbalance. The affected genes 
are preferentially organized in several clusters, and a 
supercoiling-hypersensitive cluster (SHC) was found to be located in the core of 
the S. coelicolor chromosome. The transcripts affected by long-term topological 
imbalance encompassed genes encoding nucleoid-associated proteins, DNA repair 
proteins and transcriptional regulators, including multiple developmental 
regulators. Moreover, using a gyrase inhibitor, we identified those genes that 
were directly affected by novobiocin, and found this was correlated with 
increased AT content in their promoter regions. In contrast to the genes 
affected by long-term supercoiling changes, among the novobiocin-sensitive 
genes, a significant fraction encoded for proteins associated with membrane 
transport or secondary metabolite synthesis. Collectively, our results show that 
long-term supercoiling imbalance globally regulates gene transcription and has 
the potential to impact development, secondary metabolism and DNA repair, 
amongst others.

DOI: 10.3389/fmicb.2019.01605
PMCID: PMC6637917
PMID: 31354687

## Full Text

IntroductionA bacterial chromosome is highly constrained within the cell, yet remains accessible for DNA replication, segregation and transcription. In bacteria, these processes are not separated in space and time, and their co-occurrence significantly impacts chromosome architecture. Chromosome topology is sustained by the coordinated action of nucleoid-associated proteins (NAPs), condensins (SMC, MukB), and DNA topoisomerases (Dame, 2005; Dillon and Dorman, 2010; Rovinskiy et al., 2012; Bjorkegren and Baranello, 2018; Racko et al., 2018). Whereas NAPs are responsible for local DNA bending, wrapping and bridging, which organize the bacterial chromosome into topologically independent domains, condensins, and topoisomerases induce global compaction and control overall chromosome negative supercoiling (Lindow et al., 2002; Postow et al., 2004). Moreover, topoisomerases are critical for the removal of excessive DNA supercoiling generated by DNA replication and transcription (Baaklini et al., 2008). All known bacterial genomes encode at least two essential topoisomerases – topoisomerase I (TopA and TopoI) and gyrase (GyrAB). Whereas TopA removes negative DNA supercoils, gyrase generates negative supercoils, thus, the opposite activities of TopA and GyrAB are essential for maintaining topological balance (topological homeostasis) (Champoux, 2001). Disturbances in the topological balance are reflected in the changes in transcriptional activity of the so-called supercoiling-sensitive genes (SSGs) (Peter et al., 2004). Decreased level of DNA supercoiling, resulting from gyrase inhibition, affects the transcription of approximately 7% of genes in Escherichia coli and up to 37% of genes in Haemophilus influenzae (Gmuender et al., 2001; Peter et al., 2004). Similarly, increased supercoiling, generated by the inhibition of TopA activity, leads to changes in the transcription of 10% of all genes in Streptococcus pneumoniae (Ferrandiz et al., 2016). Although the influence of global DNA supercoiling on overall gene transcription in bacteria is well established, the scale of transcriptional changes differs between bacteria, and much is still not understood about these processes.The genes most efficiently induced by the shift in DNA supercoiling are those that are critical for restoring topological balance, and encompass genes encoding topoisomerases (Tse-Dinh, 1985; Ahmed et al., 2015; Szafran et al., 2016), NAPs, and condensins (Schneider et al., 2000; Ghosh et al., 2014; Guha et al., 2018). However, SSGs also include genes involved in the regulation of many other cellular processes, i.e., DNA replication, cell division, and metabolic pathways, as well as DNA repair or stress adaptation (Peter et al., 2004). Interestingly, even after the apparent restoration of supercoiling homeostasis, the transcription of many genes has been reported to remain affected (Ferrándiz et al., 2014). Since changes in chromosome supercoiling are induced by environmental stresses, including osmotic and oxidative stress, host invasion and temperature down- or up-shift, transcriptional regulation by chromosome topology may be an important bacterial strategy for rapid adaptation to stress conditions (Horiuchi et al., 1984; Camacho-Carranza et al., 1995; Tse-Dinh et al., 1997; Weinstein-Fischer et al., 2000; Cheung et al., 2003; Cameron et al., 2011).Given that soil bacteria are exposed to a wide range of environmental stressors, they have evolved a multitude of strategies to facilitate their survival in harsh conditions. A classic example is provided by the soil-dwelling Streptomyces, which are Gram-positive, mycelial bacteria that possess linear, GC-rich chromosomes (Bentley et al., 2002). Streptomyces environmental niche adaptations include a complex regulatory network that employs a high number of transcriptional regulators, including more than 60 sigma factors and approximately 60–70 two component systems (TCSs). Moreover, Streptomyces possess a high number of DNA protection proteins and enzymes involved in the posttranslational modifications of proteins, i.e., proteases, acetyltransferases, and kinases (de Crecy-Lagard et al., 1999; Hutchings et al., 2004; Facey et al., 2009). Finally, Streptomyces produce an assortment of secondary metabolites with varied biological activities (Jeong et al., 2016) and exhibit a complex life cycle that involves the formation of stress-resistant spores. Vegetatively growing Streptomyces develop branched multicellular hyphae composed of elongated cells containing multiple loosely condensed chromosomes (Kim et al., 2000; McCormick and Flardh, 2012). During sporulation, the multigenomic unbranched aerial hyphae are converted into chains of unigenomic spores. This transition involves the condensation and segregation of tens of chromosomes, which in turn is coordinated with multiple synchronized cell divisions (Jakimowicz et al., 2005; Kois et al., 2009; Jakimowicz and van Wezel, 2012; Szafran et al., 2013). The different steps of Streptomyces differentiation are tightly regulated by a cascade of regulatory proteins (encoded by the bld and whi genes) that controls the expression of those genes involved in cell division and spore maturation (Flardh and Buttner, 2009; McCormick and Flardh, 2012; Bush et al., 2013).Streptomyces sporulation involves prominent changes in chromosome organization and requires the activity of proteins that exert chromosome rearrangements, including topoisomerase TopA (Szafran et al., 2013). In contrast to other bacterial species, TopA is the only type I topoisomerase in Streptomyces. As in other actinobacteria, the enzyme differs remarkably from its bacterial homologs in having an unusually high processivity, which is provided by a long stretch of alanine/lysine repeats in its C-terminal domain (Bhaduri et al., 1998; Szafran et al., 2014; Strzalka et al., 2017). In S. coelicolor and S. venezuelae, two model species broadly used in the studies of Streptomyces biology, TopA depletion increased DNA supercoiling and led to growth retardation as well as impaired spore production (Szafran et al., 2013; Donczew et al., 2016). Thus, in Streptomyces, the removal of the topological tensions by the activity of the highly processive and indispensable TopA is required for efficient growth and sporulation.In Streptomyces, as in other bacteria (e.g., E. coli, Mycobacterium smegmatis, and S. pneumoniae), novobiocin-mediated gyrase inhibition leads to the rapid loss of chromosome supercoiling (Peter et al., 2004; Ferrandiz et al., 2010; Szafran et al., 2016; Guha et al., 2018). In most bacteria, topological balance is subsequently restored by simultaneous upregulation of gyrase genes, and downregulation of topA transcription (Tse-Dinh, 1985; Unniraman and Nagaraja, 1999; Ferrandiz et al., 2010), while uniquely in Streptomyces, chromosome relaxation appears to affect only the gyrBA operon but does not influence topA transcription (Szafran et al., 2016). Conversely, increased negative chromosome supercoiling caused by depletion of TopA leads to the induction of topA expression, but, interestingly, does not influence gyrBA transcription which seems also unique for Streptomyces. Changes in topoisomerase levels or superhelical density have also been observed as a result of Streptomyces exposure to increased temperature or osmotic stress (Aldridge et al., 2013; Szafran et al., 2016). Thus, Streptomyces provide an interesting model for studies of topological homeostasis and its role in stress responses, given their soil habitat and unique system of chromosome supercoiling maintenance.Here, we analyzed the transcriptional response of S. coelicolor to changes in global chromosome supercoiling by inhibiting DNA gyrase activity. We identified those genes that were immediately impacted by the rapid loss of DNA supercoiling. Additionally, using a strain in which TopA levels could be modified we defined the effect of long-term topological imbalance on global gene expression.

Introduction

A bacterial chromosome is highly constrained within the cell, yet remains accessible for DNA replication, segregation and transcription. In bacteria, these processes are not separated in space and time, and their co-occurrence significantly impacts chromosome architecture. Chromosome topology is sustained by the coordinated action of nucleoid-associated proteins (NAPs), condensins (SMC, MukB), and DNA topoisomerases (Dame, 2005; Dillon and Dorman, 2010; Rovinskiy et al., 2012; Bjorkegren and Baranello, 2018; Racko et al., 2018). Whereas NAPs are responsible for local DNA bending, wrapping and bridging, which organize the bacterial chromosome into topologically independent domains, condensins, and topoisomerases induce global compaction and control overall chromosome negative supercoiling (Lindow et al., 2002; Postow et al., 2004). Moreover, topoisomerases are critical for the removal of excessive DNA supercoiling generated by DNA replication and transcription (Baaklini et al., 2008). All known bacterial genomes encode at least two essential topoisomerases – topoisomerase I (TopA and TopoI) and gyrase (GyrAB). Whereas TopA removes negative DNA supercoils, gyrase generates negative supercoils, thus, the opposite activities of TopA and GyrAB are essential for maintaining topological balance (topological homeostasis) (Champoux, 2001). Disturbances in the topological balance are reflected in the changes in transcriptional activity of the so-called supercoiling-sensitive genes (SSGs) (Peter et al., 2004). Decreased level of DNA supercoiling, resulting from gyrase inhibition, affects the transcription of approximately 7% of genes in Escherichia coli and up to 37% of genes in Haemophilus influenzae (Gmuender et al., 2001; Peter et al., 2004). Similarly, increased supercoiling, generated by the inhibition of TopA activity, leads to changes in the transcription of 10% of all genes in Streptococcus pneumoniae (Ferrandiz et al., 2016). Although the influence of global DNA supercoiling on overall gene transcription in bacteria is well established, the scale of transcriptional changes differs between bacteria, and much is still not understood about these processes.

The genes most efficiently induced by the shift in DNA supercoiling are those that are critical for restoring topological balance, and encompass genes encoding topoisomerases (Tse-Dinh, 1985; Ahmed et al., 2015; Szafran et al., 2016), NAPs, and condensins (Schneider et al., 2000; Ghosh et al., 2014; Guha et al., 2018). However, SSGs also include genes involved in the regulation of many other cellular processes, i.e., DNA replication, cell division, and metabolic pathways, as well as DNA repair or stress adaptation (Peter et al., 2004). Interestingly, even after the apparent restoration of supercoiling homeostasis, the transcription of many genes has been reported to remain affected (Ferrándiz et al., 2014). Since changes in chromosome supercoiling are induced by environmental stresses, including osmotic and oxidative stress, host invasion and temperature down- or up-shift, transcriptional regulation by chromosome topology may be an important bacterial strategy for rapid adaptation to stress conditions (Horiuchi et al., 1984; Camacho-Carranza et al., 1995; Tse-Dinh et al., 1997; Weinstein-Fischer et al., 2000; Cheung et al., 2003; Cameron et al., 2011).

Given that soil bacteria are exposed to a wide range of environmental stressors, they have evolved a multitude of strategies to facilitate their survival in harsh conditions. A classic example is provided by the soil-dwelling Streptomyces, which are Gram-positive, mycelial bacteria that possess linear, GC-rich chromosomes (Bentley et al., 2002). Streptomyces environmental niche adaptations include a complex regulatory network that employs a high number of transcriptional regulators, including more than 60 sigma factors and approximately 60–70 two component systems (TCSs). Moreover, Streptomyces possess a high number of DNA protection proteins and enzymes involved in the posttranslational modifications of proteins, i.e., proteases, acetyltransferases, and kinases (de Crecy-Lagard et al., 1999; Hutchings et al., 2004; Facey et al., 2009). Finally, Streptomyces produce an assortment of secondary metabolites with varied biological activities (Jeong et al., 2016) and exhibit a complex life cycle that involves the formation of stress-resistant spores. Vegetatively growing Streptomyces develop branched multicellular hyphae composed of elongated cells containing multiple loosely condensed chromosomes (Kim et al., 2000; McCormick and Flardh, 2012). During sporulation, the multigenomic unbranched aerial hyphae are converted into chains of unigenomic spores. This transition involves the condensation and segregation of tens of chromosomes, which in turn is coordinated with multiple synchronized cell divisions (Jakimowicz et al., 2005; Kois et al., 2009; Jakimowicz and van Wezel, 2012; Szafran et al., 2013). The different steps of Streptomyces differentiation are tightly regulated by a cascade of regulatory proteins (encoded by the bld and whi genes) that controls the expression of those genes involved in cell division and spore maturation (Flardh and Buttner, 2009; McCormick and Flardh, 2012; Bush et al., 2013).

Streptomyces sporulation involves prominent changes in chromosome organization and requires the activity of proteins that exert chromosome rearrangements, including topoisomerase TopA (Szafran et al., 2013). In contrast to other bacterial species, TopA is the only type I topoisomerase in Streptomyces. As in other actinobacteria, the enzyme differs remarkably from its bacterial homologs in having an unusually high processivity, which is provided by a long stretch of alanine/lysine repeats in its C-terminal domain (Bhaduri et al., 1998; Szafran et al., 2014; Strzalka et al., 2017). In S. coelicolor and S. venezuelae, two model species broadly used in the studies of Streptomyces biology, TopA depletion increased DNA supercoiling and led to growth retardation as well as impaired spore production (Szafran et al., 2013; Donczew et al., 2016). Thus, in Streptomyces, the removal of the topological tensions by the activity of the highly processive and indispensable TopA is required for efficient growth and sporulation.

In Streptomyces, as in other bacteria (e.g., E. coli, Mycobacterium smegmatis, and S. pneumoniae), novobiocin-mediated gyrase inhibition leads to the rapid loss of chromosome supercoiling (Peter et al., 2004; Ferrandiz et al., 2010; Szafran et al., 2016; Guha et al., 2018). In most bacteria, topological balance is subsequently restored by simultaneous upregulation of gyrase genes, and downregulation of topA transcription (Tse-Dinh, 1985; Unniraman and Nagaraja, 1999; Ferrandiz et al., 2010), while uniquely in Streptomyces, chromosome relaxation appears to affect only the gyrBA operon but does not influence topA transcription (Szafran et al., 2016). Conversely, increased negative chromosome supercoiling caused by depletion of TopA leads to the induction of topA expression, but, interestingly, does not influence gyrBA transcription which seems also unique for Streptomyces. Changes in topoisomerase levels or superhelical density have also been observed as a result of Streptomyces exposure to increased temperature or osmotic stress (Aldridge et al., 2013; Szafran et al., 2016). Thus, Streptomyces provide an interesting model for studies of topological homeostasis and its role in stress responses, given their soil habitat and unique system of chromosome supercoiling maintenance.

Here, we analyzed the transcriptional response of S. coelicolor to changes in global chromosome supercoiling by inhibiting DNA gyrase activity. We identified those genes that were immediately impacted by the rapid loss of DNA supercoiling. Additionally, using a strain in which TopA levels could be modified we defined the effect of long-term topological imbalance on global gene expression.

Materials and MethodsBacterial StrainsThe S. coelicolor strains used in this study are listed in Table 1.Table 1Strains used in this study.StrainRelevant genotypeSource or referencesM145SCP1- SCP2-Bentley et al., 2002PS04M145 ΔtopA::scar attBΦC31::pIJ6902topASzafran et al., 2013PS08M145 pIJ6902This studyMS10M145 pWHM3HygSzafran et al., 2016MS11PS04 pWHM3HygSzafran et al., 2016To construct the PS08 strain wild type S. coelicolor (M145) was transformed with the integrative pIJ6902 plasmid (Huang et al., 2005) according to the procedures described by Kieser et al. (2000) and subsequently selected for apramycin resistance.Western Blot AnalysisTo determine TopA protein levels, crude cell extracts were prepared from 18-h cultures in 79 medium (Prauser and Falta, 1968). Cell lysate proteins (5 μg in total) were separated in a 10% denaturing polyacrylamide gel before being transferred to a nitrocellulose membrane. The TopA protein was subsequently detected using primary rabbit polyclonal anti-TopA antibodies and mouse secondary anti-rabbit IgG antibodies conjugated with alkaline phosphatase (Sigma-Aldrich), according to the procedure described previously (Szafran et al., 2013). The relative protein level (RPL) was calculated using Fiji Software as the TopA band intensity compared to the wild type strain.Reporter Plasmid IsolationThe pWHM3Hyg plasmid, which served as a probe of the DNA supercoiling state in vivo, was isolated, according to the procedure described previously (Szafran et al., 2016), from S. coelicolor liquid cultures cultivated for 18 h in YEME/TSB or in 79 medium supplemented with 50 μg/ml hygromycin B (Kieser et al., 2000) at 30°C. To inhibit gyrase activity and induce rapid chromosome relaxation, novobiocin was added to the S. coelicolor culture to a final concentration of 10 μg/ml, which strongly inhibited S. coelicolor growth (Supplementary Figure S1), 10 min before mycelia were collected. The isolated plasmid DNA was resolved in an 0.8% agarose gel. To visualize particular topoisomers, the gel was stained with ethidium bromide. The topoisomer distribution was analyzed using ImageJ software.RNA Isolation and RNA-Seq AnalysesTo follow the effects of altered DNA topology on cellular transcription, RNA was isolated from S. coelicolor mycelia (2–4 biological replicates) cultivated in 30 mL YEME/TSB liquid medium for 18 h. A single time point was used for all strains, since growth curve analyses indicated that in YEME/TSB medium, the growth of the PS04 strain cultured in the absence of inducer (TopA-depleted strain) during the initial stage of culture was only somewhat slower than wild type, and the transition phase (the growth arrest at late exponential phase) occurred at the same time point (although this depended on the culture scale, Supplementary Figure S2). Mycelia were collected by centrifugation, frozen and stored at -70°C for subsequent RNA isolation. RNA was isolated using the procedure described previously by Moody et al. (2013), digested with TURBO DNase I (Invitrogen, United States) and checked for chromosomal DNA contamination using PCR. Ribosomal RNA was removed using Ribo-Zero RNA Removal Kit (Illumina). The construction of strand-specific cDNA libraries, with an average fragment size of 250 bp, and sequencing using the MiSeq kit (Illumina) were performed by the Farncombe Metagenomics Facility at McMaster University (Canada). Paired-end 76-bp reads were subsequently mapped against the S. coelicolor chromosome using Rockhopper software (McClure et al., 2013), which enabled the successful alignment of 1.0–1.5 × 106 reads per sample (Supplementary Figure S3). Raw data are available at ArrayExpress (EMBL-EBI); accession number E-MTAB-8071 as well as at Gene Expression Omnibus (NCBI); accession number GSE132702.For data visualization, Integrated Genome Viewer (IGV) software was used. The analysis of differentially regulated genes was based on the data generated by Rockhopper software. To calculate the fold change in gene transcription, normalized gene expression (normalization by the upper quartile of gene expression) of the wild type strain under control conditions was divided by normalized gene expression under particular experimental conditions. Subsequently, the log2 value of the fold change was calculated. To minimize the frequency of false-positives, transcripts with a log2 value between -1.5 and 1.5 were eliminated from subsequent analyses, with the exception of NAPs and topoisomerases encoding genes, for which the threshold was based only on the p-value significance. The gene distribution analysis was performed by mapping the positions of transcription start points of identified genes or the first gene in a regulated operon determined on the basis of the RNA-Seq data, to the S. coelicolor chromosome. To identify gene clusters of interest, we determined the percent of affected genes within a 250 kbp fragment of the S. coelicolor chromosome, subsequently moving the calculation window by 125 kbp in every step. Those regions in which the percentage of affected genes was higher than 5% were classified as supercoiling-sensitive clusters.Reverse-Transcription and Quantitative PCR (RT-qPCR)For RT-qPCR analyses, S. coelicolor strains: M145 (wild type), PS04 (with modified TopA level – topA under tipA promoter cloned in pIJ6902 integrative plasmid) and PS08 (wild type with integrated empty pIJ6902 plasmid) were cultivated in liquid 79 medium for 24 h in the presence and absence of inducer. Using the reporter plasmid assay, the DNA supercoiling level in cultures in 79 medium was found to be similar to supercoiling in cultures grown in YEME/TSB medium (Supplementary Figure S4). RNA from S. coelicolor mycelia was isolated using the GeneJET RNA isolation kit (Thermo Fisher Scientific, United States) according to the manufacturer’s procedure (except that the concentration of lysozyme in the suspension buffer was increased to 10 mg/ml). The isolated RNA was digested with TURBO DNase I (Invitrogen) to remove traces of chromosomal DNA and then purified and concentrated using the GeneJET RNA Cleanup kit (Thermo Fisher Scientific, United States). A total of 500 ng of RNA was used for cDNA synthesis using the Maxima First Strand cDNA synthesis kit (Thermo Fisher Scientific, United States) in a final volume of 20 μl. The original manufacturer protocol was modified for GC-rich S. coelicolor transcripts by increasing the temperature for first-strand synthesis to 65°C and extending the synthesis time up to 30 min. Subsequently, the resulting cDNA was diluted to 100 μL and directly used for quantitative PCRs performed with PowerUp SYBR Green Master Mix (Applied Biosystems, United States). The relative level of a particular transcript was quantified using the comparative ΔΔCt method and the hrdB gene as the endogenous control (StepOne Plus real-time PCR system, Applied Biosystems, United States). The optimized oligonucleotides used in this study were synthetized by Sigma-Aldrich (United States), and are listed in the supplementary data (see the Supplementary Table S1 “oligonucleotides”).Analysis of the Promoter AT ContentTo estimate the AT content in the promoter regions of novobiocin-induced genes, 66 DNA sequences (960 bp in length) encompassing 750 bp upstream of the translation start codons and 210 bp downstream (960 bp in total) were analyzed. As the control experiments, 66 randomly chosen 750 bp promoter regions with 210 bp intragenic sequence or 66 random S. coelicolor genomic sequences (each 960 bp in length) were used. Subsequently, we calculated the AT percentage of these sequences in 100-bp windows, with a 1-bp step. The AT content for each window was subsequently plotted against the first nucleotide position of the particular window. The differences in mean AT content calculated for specific nucleotides within the promoter region (in position 340 bp) or the GC-enriched region (in position 660 bp) were tested using the Shapiro–Wilk model and t-test.

Materials and Methods

Bacterial StrainsThe S. coelicolor strains used in this study are listed in Table 1.Table 1Strains used in this study.StrainRelevant genotypeSource or referencesM145SCP1- SCP2-Bentley et al., 2002PS04M145 ΔtopA::scar attBΦC31::pIJ6902topASzafran et al., 2013PS08M145 pIJ6902This studyMS10M145 pWHM3HygSzafran et al., 2016MS11PS04 pWHM3HygSzafran et al., 2016To construct the PS08 strain wild type S. coelicolor (M145) was transformed with the integrative pIJ6902 plasmid (Huang et al., 2005) according to the procedures described by Kieser et al. (2000) and subsequently selected for apramycin resistance.

Bacterial Strains

The S. coelicolor strains used in this study are listed in Table 1.

Strains used in this study.

To construct the PS08 strain wild type S. coelicolor (M145) was transformed with the integrative pIJ6902 plasmid (Huang et al., 2005) according to the procedures described by Kieser et al. (2000) and subsequently selected for apramycin resistance.

Western Blot AnalysisTo determine TopA protein levels, crude cell extracts were prepared from 18-h cultures in 79 medium (Prauser and Falta, 1968). Cell lysate proteins (5 μg in total) were separated in a 10% denaturing polyacrylamide gel before being transferred to a nitrocellulose membrane. The TopA protein was subsequently detected using primary rabbit polyclonal anti-TopA antibodies and mouse secondary anti-rabbit IgG antibodies conjugated with alkaline phosphatase (Sigma-Aldrich), according to the procedure described previously (Szafran et al., 2013). The relative protein level (RPL) was calculated using Fiji Software as the TopA band intensity compared to the wild type strain.

Western Blot Analysis

To determine TopA protein levels, crude cell extracts were prepared from 18-h cultures in 79 medium (Prauser and Falta, 1968). Cell lysate proteins (5 μg in total) were separated in a 10% denaturing polyacrylamide gel before being transferred to a nitrocellulose membrane. The TopA protein was subsequently detected using primary rabbit polyclonal anti-TopA antibodies and mouse secondary anti-rabbit IgG antibodies conjugated with alkaline phosphatase (Sigma-Aldrich), according to the procedure described previously (Szafran et al., 2013). The relative protein level (RPL) was calculated using Fiji Software as the TopA band intensity compared to the wild type strain.

Reporter Plasmid IsolationThe pWHM3Hyg plasmid, which served as a probe of the DNA supercoiling state in vivo, was isolated, according to the procedure described previously (Szafran et al., 2016), from S. coelicolor liquid cultures cultivated for 18 h in YEME/TSB or in 79 medium supplemented with 50 μg/ml hygromycin B (Kieser et al., 2000) at 30°C. To inhibit gyrase activity and induce rapid chromosome relaxation, novobiocin was added to the S. coelicolor culture to a final concentration of 10 μg/ml, which strongly inhibited S. coelicolor growth (Supplementary Figure S1), 10 min before mycelia were collected. The isolated plasmid DNA was resolved in an 0.8% agarose gel. To visualize particular topoisomers, the gel was stained with ethidium bromide. The topoisomer distribution was analyzed using ImageJ software.

Reporter Plasmid Isolation

The pWHM3Hyg plasmid, which served as a probe of the DNA supercoiling state in vivo, was isolated, according to the procedure described previously (Szafran et al., 2016), from S. coelicolor liquid cultures cultivated for 18 h in YEME/TSB or in 79 medium supplemented with 50 μg/ml hygromycin B (Kieser et al., 2000) at 30°C. To inhibit gyrase activity and induce rapid chromosome relaxation, novobiocin was added to the S. coelicolor culture to a final concentration of 10 μg/ml, which strongly inhibited S. coelicolor growth (Supplementary Figure S1), 10 min before mycelia were collected. The isolated plasmid DNA was resolved in an 0.8% agarose gel. To visualize particular topoisomers, the gel was stained with ethidium bromide. The topoisomer distribution was analyzed using ImageJ software.

RNA Isolation and RNA-Seq AnalysesTo follow the effects of altered DNA topology on cellular transcription, RNA was isolated from S. coelicolor mycelia (2–4 biological replicates) cultivated in 30 mL YEME/TSB liquid medium for 18 h. A single time point was used for all strains, since growth curve analyses indicated that in YEME/TSB medium, the growth of the PS04 strain cultured in the absence of inducer (TopA-depleted strain) during the initial stage of culture was only somewhat slower than wild type, and the transition phase (the growth arrest at late exponential phase) occurred at the same time point (although this depended on the culture scale, Supplementary Figure S2). Mycelia were collected by centrifugation, frozen and stored at -70°C for subsequent RNA isolation. RNA was isolated using the procedure described previously by Moody et al. (2013), digested with TURBO DNase I (Invitrogen, United States) and checked for chromosomal DNA contamination using PCR. Ribosomal RNA was removed using Ribo-Zero RNA Removal Kit (Illumina). The construction of strand-specific cDNA libraries, with an average fragment size of 250 bp, and sequencing using the MiSeq kit (Illumina) were performed by the Farncombe Metagenomics Facility at McMaster University (Canada). Paired-end 76-bp reads were subsequently mapped against the S. coelicolor chromosome using Rockhopper software (McClure et al., 2013), which enabled the successful alignment of 1.0–1.5 × 106 reads per sample (Supplementary Figure S3). Raw data are available at ArrayExpress (EMBL-EBI); accession number E-MTAB-8071 as well as at Gene Expression Omnibus (NCBI); accession number GSE132702.For data visualization, Integrated Genome Viewer (IGV) software was used. The analysis of differentially regulated genes was based on the data generated by Rockhopper software. To calculate the fold change in gene transcription, normalized gene expression (normalization by the upper quartile of gene expression) of the wild type strain under control conditions was divided by normalized gene expression under particular experimental conditions. Subsequently, the log2 value of the fold change was calculated. To minimize the frequency of false-positives, transcripts with a log2 value between -1.5 and 1.5 were eliminated from subsequent analyses, with the exception of NAPs and topoisomerases encoding genes, for which the threshold was based only on the p-value significance. The gene distribution analysis was performed by mapping the positions of transcription start points of identified genes or the first gene in a regulated operon determined on the basis of the RNA-Seq data, to the S. coelicolor chromosome. To identify gene clusters of interest, we determined the percent of affected genes within a 250 kbp fragment of the S. coelicolor chromosome, subsequently moving the calculation window by 125 kbp in every step. Those regions in which the percentage of affected genes was higher than 5% were classified as supercoiling-sensitive clusters.

RNA Isolation and RNA-Seq Analyses

To follow the effects of altered DNA topology on cellular transcription, RNA was isolated from S. coelicolor mycelia (2–4 biological replicates) cultivated in 30 mL YEME/TSB liquid medium for 18 h. A single time point was used for all strains, since growth curve analyses indicated that in YEME/TSB medium, the growth of the PS04 strain cultured in the absence of inducer (TopA-depleted strain) during the initial stage of culture was only somewhat slower than wild type, and the transition phase (the growth arrest at late exponential phase) occurred at the same time point (although this depended on the culture scale, Supplementary Figure S2). Mycelia were collected by centrifugation, frozen and stored at -70°C for subsequent RNA isolation. RNA was isolated using the procedure described previously by Moody et al. (2013), digested with TURBO DNase I (Invitrogen, United States) and checked for chromosomal DNA contamination using PCR. Ribosomal RNA was removed using Ribo-Zero RNA Removal Kit (Illumina). The construction of strand-specific cDNA libraries, with an average fragment size of 250 bp, and sequencing using the MiSeq kit (Illumina) were performed by the Farncombe Metagenomics Facility at McMaster University (Canada). Paired-end 76-bp reads were subsequently mapped against the S. coelicolor chromosome using Rockhopper software (McClure et al., 2013), which enabled the successful alignment of 1.0–1.5 × 106 reads per sample (Supplementary Figure S3). Raw data are available at ArrayExpress (EMBL-EBI); accession number E-MTAB-8071 as well as at Gene Expression Omnibus (NCBI); accession number GSE132702.

For data visualization, Integrated Genome Viewer (IGV) software was used. The analysis of differentially regulated genes was based on the data generated by Rockhopper software. To calculate the fold change in gene transcription, normalized gene expression (normalization by the upper quartile of gene expression) of the wild type strain under control conditions was divided by normalized gene expression under particular experimental conditions. Subsequently, the log2 value of the fold change was calculated. To minimize the frequency of false-positives, transcripts with a log2 value between -1.5 and 1.5 were eliminated from subsequent analyses, with the exception of NAPs and topoisomerases encoding genes, for which the threshold was based only on the p-value significance. The gene distribution analysis was performed by mapping the positions of transcription start points of identified genes or the first gene in a regulated operon determined on the basis of the RNA-Seq data, to the S. coelicolor chromosome. To identify gene clusters of interest, we determined the percent of affected genes within a 250 kbp fragment of the S. coelicolor chromosome, subsequently moving the calculation window by 125 kbp in every step. Those regions in which the percentage of affected genes was higher than 5% were classified as supercoiling-sensitive clusters.

Reverse-Transcription and Quantitative PCR (RT-qPCR)For RT-qPCR analyses, S. coelicolor strains: M145 (wild type), PS04 (with modified TopA level – topA under tipA promoter cloned in pIJ6902 integrative plasmid) and PS08 (wild type with integrated empty pIJ6902 plasmid) were cultivated in liquid 79 medium for 24 h in the presence and absence of inducer. Using the reporter plasmid assay, the DNA supercoiling level in cultures in 79 medium was found to be similar to supercoiling in cultures grown in YEME/TSB medium (Supplementary Figure S4). RNA from S. coelicolor mycelia was isolated using the GeneJET RNA isolation kit (Thermo Fisher Scientific, United States) according to the manufacturer’s procedure (except that the concentration of lysozyme in the suspension buffer was increased to 10 mg/ml). The isolated RNA was digested with TURBO DNase I (Invitrogen) to remove traces of chromosomal DNA and then purified and concentrated using the GeneJET RNA Cleanup kit (Thermo Fisher Scientific, United States). A total of 500 ng of RNA was used for cDNA synthesis using the Maxima First Strand cDNA synthesis kit (Thermo Fisher Scientific, United States) in a final volume of 20 μl. The original manufacturer protocol was modified for GC-rich S. coelicolor transcripts by increasing the temperature for first-strand synthesis to 65°C and extending the synthesis time up to 30 min. Subsequently, the resulting cDNA was diluted to 100 μL and directly used for quantitative PCRs performed with PowerUp SYBR Green Master Mix (Applied Biosystems, United States). The relative level of a particular transcript was quantified using the comparative ΔΔCt method and the hrdB gene as the endogenous control (StepOne Plus real-time PCR system, Applied Biosystems, United States). The optimized oligonucleotides used in this study were synthetized by Sigma-Aldrich (United States), and are listed in the supplementary data (see the Supplementary Table S1 “oligonucleotides”).

Reverse-Transcription and Quantitative PCR (RT-qPCR)

For RT-qPCR analyses, S. coelicolor strains: M145 (wild type), PS04 (with modified TopA level – topA under tipA promoter cloned in pIJ6902 integrative plasmid) and PS08 (wild type with integrated empty pIJ6902 plasmid) were cultivated in liquid 79 medium for 24 h in the presence and absence of inducer. Using the reporter plasmid assay, the DNA supercoiling level in cultures in 79 medium was found to be similar to supercoiling in cultures grown in YEME/TSB medium (Supplementary Figure S4). RNA from S. coelicolor mycelia was isolated using the GeneJET RNA isolation kit (Thermo Fisher Scientific, United States) according to the manufacturer’s procedure (except that the concentration of lysozyme in the suspension buffer was increased to 10 mg/ml). The isolated RNA was digested with TURBO DNase I (Invitrogen) to remove traces of chromosomal DNA and then purified and concentrated using the GeneJET RNA Cleanup kit (Thermo Fisher Scientific, United States). A total of 500 ng of RNA was used for cDNA synthesis using the Maxima First Strand cDNA synthesis kit (Thermo Fisher Scientific, United States) in a final volume of 20 μl. The original manufacturer protocol was modified for GC-rich S. coelicolor transcripts by increasing the temperature for first-strand synthesis to 65°C and extending the synthesis time up to 30 min. Subsequently, the resulting cDNA was diluted to 100 μL and directly used for quantitative PCRs performed with PowerUp SYBR Green Master Mix (Applied Biosystems, United States). The relative level of a particular transcript was quantified using the comparative ΔΔCt method and the hrdB gene as the endogenous control (StepOne Plus real-time PCR system, Applied Biosystems, United States). The optimized oligonucleotides used in this study were synthetized by Sigma-Aldrich (United States), and are listed in the supplementary data (see the Supplementary Table S1 “oligonucleotides”).

Analysis of the Promoter AT ContentTo estimate the AT content in the promoter regions of novobiocin-induced genes, 66 DNA sequences (960 bp in length) encompassing 750 bp upstream of the translation start codons and 210 bp downstream (960 bp in total) were analyzed. As the control experiments, 66 randomly chosen 750 bp promoter regions with 210 bp intragenic sequence or 66 random S. coelicolor genomic sequences (each 960 bp in length) were used. Subsequently, we calculated the AT percentage of these sequences in 100-bp windows, with a 1-bp step. The AT content for each window was subsequently plotted against the first nucleotide position of the particular window. The differences in mean AT content calculated for specific nucleotides within the promoter region (in position 340 bp) or the GC-enriched region (in position 660 bp) were tested using the Shapiro–Wilk model and t-test.

Analysis of the Promoter AT Content

To estimate the AT content in the promoter regions of novobiocin-induced genes, 66 DNA sequences (960 bp in length) encompassing 750 bp upstream of the translation start codons and 210 bp downstream (960 bp in total) were analyzed. As the control experiments, 66 randomly chosen 750 bp promoter regions with 210 bp intragenic sequence or 66 random S. coelicolor genomic sequences (each 960 bp in length) were used. Subsequently, we calculated the AT percentage of these sequences in 100-bp windows, with a 1-bp step. The AT content for each window was subsequently plotted against the first nucleotide position of the particular window. The differences in mean AT content calculated for specific nucleotides within the promoter region (in position 340 bp) or the GC-enriched region (in position 660 bp) were tested using the Shapiro–Wilk model and t-test.

Results and DiscussionRapid Loss of Chromosome Supercoiling Positively Regulates S. coelicolor Gene TranscriptionIn previous work, we found that exposing S. coelicolor to novobiocin, initially led to the rapid relaxation of DNA, followed by gradual restoration of topological homeostasis (Szafran et al., 2016), as had been seen in other bacteria (Ferrandiz et al., 2010). The return to topological balance appeared to stem from changes in the expression of topoisomerase encoding genes, which in S. coelicolor corresponded to a strong induction of gyrAB but only slight inhibition of topA (Szafran et al., 2016); this is different from the simultaneous induction of gyrAB and inhibition of topA transcription observed in other bacteria (Unniraman and Nagaraja, 1999; Ahmed et al., 2015). Here, we sought to identify other genes in S. coelicolor whose transcription was also directly affected by the rapid chromosome relaxation. To this end, we treated S. coelicolor liquid cultures with 10 μg/ml novobiocin, which strongly inhibited S. coelicolor growth (Supplementary Figure S1). We analyzed the changes in global transcript abundance after 10 min of novobiocin exposure, and compared these with an untreated control strain. Based on previous work (Gmuender et al., 2001; Szafran et al., 2016), 10 min of novobiocin exposure was presumed to be sufficient to affect promoters directly sensitive to gyrase inhibition, triggering the primary transcriptional response, but was deemed insufficient time to induce a secondary transcriptional response (which depends on any primary-induced transcriptional regulators).In analyzing our RNA-Seq data, we found that the rapid DNA relaxation caused by novobiocin (detected using a reporter plasmid) (Figure 1A and Supplementary Figure S5) led to a 2.71-fold induction of the gyrBA operon transcription (Figure 1B), confirming that the primary transcriptional response could be induced by 10 min of gyrase inhibition. We next set out to identify genes exhibiting distinct changes in transcription in comparison to the untreated control (at least 2.83-fold), and found 121 genes that were sensitive to the rapid loss of chromosome supercoiling. These genes constitute 1.5% of the S. coelicolor open reading frames (ORFs) (see Supplementary Table S2 “novobiocin” for the complete list of the identified genes). Mapping of the transcription start sites (TSSs) of the identified genes to the S. coelicolor chromosome revealed that the novobiocin-sensitive genes were unevenly distributed along the S. coelicolor chromosome (Figure 1C and Supplementary Figure S6A). Surprisingly, in contrast to E. coli, in which only one third of genes affected by gyrase inhibition were induced and the others were repressed (Peter et al., 2004), in S. coelicolor, most of the novobiocin-sensitive genes (117 of 121) were upregulated by chromosome relaxation, with only 4 genes (sco0498, sco1377, sco3726, and sco3918) being strongly downregulated (log2-fold >1.5).FIGURE 1Identification of novobiocin-sensitive genes in S. coelicolor. (A) Analysis of reporter plasmid supercoiling density. The reporter plasmid was isolated from untreated wild type strain (upper panel) and the same strain exposed to novobiocin (10 μg/mL) for 10 min (bottom panel). The most abundant topoisomers identified under each condition are marked with arrows. (B) RNA-Seq based gyrB expression normalized by the upper quartile, in the untreated wild type strain (gray) and the novobiocin-treated strain [as described for panel (A)] (black). (C) The chromosomal localization of the novobiocin-sensitive genes. Differentially upregulated (green) and downregulated (red) transcripts in the S. coelicolor culture following novobiocin exposure (as in panel A) were identified using log2-fold changes >1.5; p-value <0.05 thresholds. (D) Analysis of the average AT content in the promoter regions of novobiocin-sensitive genes (blue) in comparison to the random promoters (red) and random chromosomal sequences (gray). The transcriptional start site (TSS), promoter region and GC-enriched region identified upstream of novobiocin-sensitive promoters are marked. (E) The box plot comparison of AT content within the promoter region (upper panel) and GC-enriched region (bottom panel) of novobiocin-sensitive (blue) and randomly selected (red) promoters. The asterisks indicate the calculated p-value <0.05 (∗) or <0.01 (∗∗).We speculate that the gene upregulation after novobiocin exposure rather than their transcriptional repression may be due to the high GC content (72%) of the S. coelicolor genome (Bentley et al., 2002). In E. coli, promoter regions of relaxation-induced genes are notably enriched in AT base pairs, whereas relaxation-repressed promoters have increased GC content (Peter et al., 2004). In S. coelicolor, the increase in GC content in the promoter regions to levels above the average would be expected to result in increased DNA stability, which in turn would inhibit promoter unwinding. Conversely, more AT-rich promoter regions would be expected to favor promoter upregulation. To test this hypothesis, we compared the AT content of 66 novobiocin-induced promoters (the identified 121 genes were recognized as 66 transcriptional units) with the AT content of randomly chosen promoter regions. As predicted, we found that novobiocin-induced genes showed approximately 2–3% higher AT content compared with the randomly selected promoters and 8–9% higher in comparison to the average of the S. coelicolor genome (28%). Notably, the AT-rich region was significantly expanded in the novobiocin-sensitive genes relative to the random promoters (Figure 1D). Unexpectedly, AT-content analysis revealed a GC-rich region located approximately 100–200 bp upstream of the novobiocin-induced genes (Figure 1D), and not the random promoters. The difference in the mean AT-content values between the novobiocin-sensitive and random promoters, calculated for position 340 bp (GC-enriched region) and 660 bp (promoter region) were statistically relevant (p-values of 0.006 and 0.017, respectively; Figure 1E). Interestingly, similar analysis performed in E. coli did not indicate the existence of a GC-rich fragment preceding the relaxation-induced genes (Peter et al., 2004), although the significance of the GC-enriched region upstream of the Streptomyces novobiocin-inducible promoters is currently unclear.Among the 121 novobiocin-sensitive genes, 15 encoded putative regulatory proteins, including the anti-sigma factor RsfA (Homerova et al., 2000), the developmental regulator WblC (Fowler-Goldsworthy et al., 2011), and the secondary metabolism regulator NsdB (Zhang et al., 2007). Interestingly, a high fraction of the novobiocin-sensitive genes (∼30%) encode proteins involved in either antibiotic production (10 genes, including the actinorhodin activator sco5085/actII-orf4 and genes within the coelibactin biosynthetic cluster sco7681-sco7688) or putative membrane transporters (at least 25 genes) (Table 2). This suggests that rapid chromosome relaxation functions as the stress signal that triggers the transcription of genes whose products are involved in stress adaptation (i.e., those encoding regulatory proteins) and/or interspecies competition. Considering that decreased gyrase activity and rapid chromosome relaxation may result from exposure to different cell membrane destabilizing agents that affect transmembrane potential and ATP synthesis, or other biologically active molecules produced by competing bacterial species (Prakash et al., 2009), it is conceivable that the intrinsic response to such agents should encompass the induction of transport proteins that should restore homeostasis to the intracellular environment and/or function to eliminate toxic compounds.Table 2Selected novobiocin-induced genes encoding transporters or transcriptional regulators (based on RNA-Seq data; log2-fold >1.5; p < 0.05).GeneLocusNormalized expression in the wild type∗Log2-fold changeDescriptionTransporterssco0623571.86Membrane transportersco2310191.86Integral membrane efflux proteinsco2763111.79ABC transporter ATP-binding proteinsco2896/sco2895/sco28942/7/64.09/2.51/2.17Possible ABC transportersco3206581.81Transmembrane efflux proteinsco333191.64Transmembrane transportersco345591.92ABC transportersco4031172.25Integral membrane transporterragAsco4075241.52ABC transporter ATP-binding proteinsco5451361.54ABC transportersco5957302.65Transportersco672053.17ABC transporterRegulatorsssgGsco292452.32Putative regulatorsco296454.60LysR family transcriptional regulatorsco4032143.58MarR regulatory proteinrsfAsco4677111.67Regulatory proteinwblCsco51901641.91DNA-binding proteinnsdBsco72521181.53Regulatory proteinAntibiotic synthesisactII-4sco5085151.68Actinorhodin cluster activator proteinsco722191.64Polyketide synthasesco7681–sco76886–171.58–2.00Coelibactin synthesis genesOther genesrpmGsco0570681.5850S ribosomal protein L33tuf3sco1321104.60Elongation factor Tusco2171141.79Putative secreted proteaseohrAsco298621.88Hypothetical proteinsco3425241.5030S ribosomal protein S18sco4316112.37ATP/GTP-binding proteinsco4449213.23Hydrolaselonsco5285102.56ATP-dependent protease∗Normalization by the upper quartile of gene expression.In summary, a significant fraction of novobiocin-sensitive genes encompassed those encoding membrane transporters, antibiotic synthesis and regulatory proteins. The novobiocin-sensitive genes were non-uniformly distributed along the chromosome, and most of them were upregulated. The upregulation was reflected by the increased AT-content of their promoter regions and the presence of an upstream GC-rich DNA stretch.Long-Term Exposure to Topological Imbalance Globally Affects Gene TranscriptionHaving established that approximately 120 genes were directly impacted by gyrase inhibition, we were next interested in assessing the response of S. coelicolor to the long-term effects of altered chromosome supercoiling. To date, there is little understood about the long-term effects of altered supercoiling, since most studied bacteria were able to quickly restore topological homeostasis. Moreover, the effects of increased negative supercoiling on global gene transcription in bacteria remain relatively unexplored, in part due to the limited availability of selective TopA inhibitors (Garcia et al., 2011; Cheng et al., 2013; Godbole et al., 2014; Szafran et al., 2018).To investigate long terms effects of supercoiling imbalance, we took advantage of a S. coelicolor strain (PS04) that had been engineered such that chromosome supercoiling could be modulated by tuning TopA levels using a thiostrepton-inducible topA gene (Szafran et al., 2013). In this PS04 strain, TopA depletion (no induction) dramatically increased the overall negative DNA supercoiling (Figure 2A and Supplementary Figures S5A,B), but TopA levels could not be restored, thus impeding the restoration of supercoiling balance. Moreover, inefficient repression of the S. coelicolor gyrase operon eliminated another possible pathway to restoring chromosomal topological balance in the TopA-depleted strain. On the other hand induction of topA expression in the PS04 strain led only to a slight (approximately 22%) increase of TopA protein levels which also corresponded to a modest shift in global DNA supercoiling levels if compared to the wild type strain (Figure 2A and Supplementary Figure S5). This observation is in agreement with our previous data showing that by increasing the concentration of topA inducer, only a 25% increase in TopA levels could be achieved (Szafran et al., 2013). This may be partially explained by induction of the gyrBA operon, which could compensate for high levels of TopA, and restore the supercoiling balance (Szafran et al., 2016) or by posttranscriptional regulation mechanisms of topA gene expression.FIGURE 2The transcriptional response to long-term supercoiling imbalance. (A) The supercoiling density analysis of the reporter plasmid isolated from the wild type strain (upper panel), the topA-upregulated strain (middle panel), and the topA-downregulated strain (lower panel). The most abundant topoisomers under each condition are marked with arrows. (B) Differentially regulated genes identified using RNA-Seq following topA-downregulation (left) and topA-upregulation (right). The up- (black) or downregulated (gray) genes were grouped according to the magnitude of their transcriptional changes compared to the wild type strain (log2-fold change value). (C) Comparison of the number of genes specifically affected by topA-upregulation and topA-downregulation, and those common to both conditions.First, we compared the number of S. coelicolor genes that were strongly affected by topA-down- or upregulation (cultivation in the absence or presence of 1 μg/ml thiostrepton). We identified 552 genes whose transcription was changed by topA-downregulation and 214 genes altered by the induction of topA transcription (at least 2.83-fold change; log2-fold >1.5), which corresponded to 7.0 and 2.7% of the S. coelicolor predicted coding sequences, respectively (Figure 2B; see Supplementary Tables S3, S4 “topA-downregulation,” “topA-upregulation” for the complete list of affected genes). Notably, most of the identified ORFs were positively regulated (Figure 2B), and the positive effect on gene transcription was particularly pronounced following topA-upregulation. These observations corroborate the notion that chromosome relaxation preferentially promoted the induction of gene transcription in GC-rich bacteria. Surprisingly, a significant number of genes were sensitive to both changes in supercoiling conditions (Figure 2C). Nevertheless, a substantial fraction of genes responded specifically to an increase of negative supercoiling resulting from TopA depletion (408 of 552, more than 70%), whereas topA-upregulation regulated specifically only 32% (70 of 214) of identified genes (Figure 2C). The observed difference in the number of the supercoiling sensitive genes (SSGs), as well as their specificity for particular supercoiling conditions, may be presumably explained by the scale of topological imbalance, which is more pronounced under the former than under the latter conditions (Figure 2A and Supplementary Figure S5). However, our results also show that, even though the topological balance was only slightly distorted following topA induction, the transcription of a relatively high number of genes was still affected, suggesting the presence of a mechanism(s) that maintains a modified level of gene transcription. Markedly, similar observations were made recently for S. pneumoniae transcriptional activity of genes under TopA inhibition conditions (Ferrandiz et al., 2016), where the expression of many genes was specifically altered. Interestingly, the inhibition of TopA in S. pneumoniae (which was followed by restoring of supercoiling imbalance) affected only 2% of the genome, while long term TopA depletion in S. coelicolor had a more pronounced effect on the gene transcription. Interestingly, the genes affected by long-term supercoiling changes only partially overlapped with those that were novobiocin-sensitive (12% of novobiocin-affected genes, Supplementary Table S5), suggesting that long-term rearrangement of bacterial chromosome topology provokes a global change in gene transcription and includes primary and secondary response pathways.Overall, we observed that a persistent supercoiling imbalance induced by a constitutive change of TopA expression, affected the transcription of approximately 3–7% of S. coelicolor genes. A significant subfraction of these were sensitive to any supercoiling imbalance, suggesting shared transcriptional response pathway(s). Our results suggest that the response to topological imbalance depends on accessible pathways of supercoiling restoration, being the most pronounced when topoisomerase gene expression cannot not be readjusted.Long-Term Supercoiling Imbalance Impacts the Expression of Discrete Gene ClustersRemarkably, supercoiling sensitive genes whose transcription was affected by long-term topological imbalance were non-uniformly distributed along the S. coelicolor chromosome, similar to genes induced in the novobiocin-treated cells whose distribution also was non-uniform. SSGs induced by topA-downregulation or topA-upregulation showed characteristic grouping, suggesting they were organized into supercoiling-sensitive clusters, which were in turn separated by supercoiling insensitive regions (Figure 3A). To quantify these observations, we calculated the percentage of genes affected by different supercoiling conditions within 250 kbp chromosome regions along the whole chromosome. We found that gene clustering was strongly detectable under topA-downregulation conditions, with 6–7 chromosomal regions containing more than 5% of genes affected negatively or positively in each cluster (Figure 3B). Notably, those clusters containing the positively and negatively regulated genes partially overlapped, further supporting the existence of DNA domains that are particularly sensitive to changes in DNA supercoiling. Interestingly, gene clustering was also detectable, although less pronounced, under topA-upregulation conditions, where the distribution of the supercoiling-sensitive regions along the chromosome was comparable to that observed in the TopA-depleted strain (Figure 3B and Supplementary Figure S6B). Analogous clustering of SSGs has been described previously for E. coli and S. pneumoniae (Peter et al., 2004; Ferrandiz et al., 2016). Collectively, these observations suggest that the organization of SSGs into topologically separated domains is a feature conserved among bacteria, presumably ensuring the coordinated expression of the associated genes. Remarkably, the loci of the genes affected by topological imbalance showed increased density in the central part of the chromosome, however, the initiation of the chromosomal replication region (oriC) was found to be outside the identified SSG clusters, and the oriC proximal genes, which predominantly encode proteins involved in DNA replication, were insensitive to the changes in chromosome supercoiling (Figure 3). Surprisingly, the genes and genes clusters that were affected by long-term supercoiling modification did not overlap with those induced by novobiocin (Supplementary Figures S6A,B). Presumably during long-term DNA relaxation, the first response genes are no longer activated and the affected genes include those of the pathways responsible for constitutive maintenance of cell homeostasis under such conditions.FIGURE 3Identification of long-term supercoiling imbalance-sensitive clusters. (A) The chromosomal positions of differentially upregulated (green) and downregulated (red) transcripts identified in the topA-downregulated strain (upper panel) and topA-upregulated strain (lower panel). The positions of the oriC region and the supercoiling-hypersensitive cluster (SHC) are marked in gray. (B) Identification of supercoiling-sensitive clusters. The percent of genes affected by topA downregulation (upper panel) or topA upregulation (lower panel) was calculated in a 250-kbp window and the mid of the window was plotted versus chromosome position. The regions with the percentage of upregulated (green) or downregulated (red) genes higher than 5% (supercoiling-sensitive clusters) are marked at the bottom of the plot.Interestingly, among several identified gene clusters impacted by supercoiling imbalance, one cluster was particularly enriched in SSGs, which constituted ∼20% of all genes positioned within the 250 kbp region. Closer examination of this region revealed that 26 out of 34 genes within 30 kbp (between sco4667 and sco4700) were supercoiling sensitive. We therefore termed this region the supercoiling-hypersensitive cluster (SHC). Most of the SHC genes encoded products of unassigned function and these genes themselves were poorly transcribed under standard culture conditions. However, they were strongly upregulated under both topA-downregulation and topA-upregulation conditions (Table 3). We confirmed these observations using RT-qPCR for the first (sco4667) and one before last (sco4699) genes of the SHC region and excluded the effect of growth medium or genetic modifications in the topA depleted PS04 strain (the integration of pIJ6902 plasmid, that contains topA gene under tipA promoter), by measuring the relative transcript level in PS04 and control strain (wild type with empty pIJ602 plasmid) cultures grown in liquid 79 medium. The transcription of both genes appeared to be highly dependent on the level of TopA: significantly increased when TopA was depleted, decreased when TopA was at the wild type level, and increased slightly again at topA induction (Supplementary Figure S6C). These results were further supported by the observation that SHC genes upregulation was correlated with relative degree of topological imbalance. However, since we did not observe increased expression of the genes in the SHC region following a 10 min exposure to novobiocin (Supplementary Figure S6D), we speculate that their activation requires not only a supercoiling imbalance, but also the activity of other factors.Table 3Transcription of genes in the SHC region at constitutive topA-down- or upregulation (based on RNA-Seq data).GeneLocusNormalized expression in the wild type∗Log2-fold change topA-downregulationLog2-fold change topA-upregulationDescriptionsco4667129.758.93Two-component system sensor kinasesco4668677.516.59Two-component system response regulatorsco4669356.065.39Hypothetical proteinsco467000.000.00Serine proteasesco4671212.312.08LysR family transcriptional regulatorsco4672551.611.73Hypothetical proteinsco4673563.202.35DeoR family transcriptional regulatorsco4674262.002.08Hypothetical proteinsco4675771.762.12Hypothetical proteinsco467631.220.41Hypothetical proteinrsfAsco4677111.240.79Regulatory proteinsco46781971.991.59Hypothetical proteinsco46795301.981.63Hypothetical proteinsco46801161.371.56DNA-binding proteinsco46811370.691.29Short chain dehydrogenasesco46821670.651.37TautomerasegdhAsco46832122.062.36Glutamate dehydrogenasescoF3sco468430874.083.00Cold shock proteinsco4685163.432.67DEAD/DEATH box helicasesco4686442.531.83Hypothetical proteinsco46876742.381.79Hypothetical proteinsco46881372.031.08Hypothetical proteinsco4689345-0.262.03Hypothetical proteinsco46901480.412.39Hypothetical proteinsco4691101.582.43Hypothetical proteinsco46921422.562.18Hypothetical proteinsco4693622.862.03Hypothetical proteinsco4694124.202.67Hypothetical proteinsco469504.462.32Hypothetical proteinsco469684.623.22Hypothetical proteinsco4697234.483.08Hypothetical proteinsco4698342.842.67IS1652 transposasesco469985.783.74Rhs proteinsco4700136.044.01Hypothetical protein∗Normalization by the upper quartile of gene expression.The SHC genes include those encoding a putative two-component system (sco4667 and sco4668) genes organized in a single operon and additional two putative transcriptional regulator-encoding genes positioned upstream (sco4671 and sco4673). The cluster also encompasses genes whose products may contribute to DNA/RNA transactions, including: putative DNA helicase (sco4685), transposase (sco4698), and a homolog of E. coli Rhs protein (rearrangement hotspot protein), which putatively contains the C-terminal toxin domain and is involved in bacterial intercellular competition (Koskiniemi et al., 2013). When we compared the genes in this region to other Streptomyces species, we found the synteny was not strong, perhaps suggesting that some of these genes may have been acquired through horizontal transfer (Hindra et al., 2014). However, the AT content of the SHC region was not significantly different from the rest of S. coelicolor chromosome (Supplementary Figure S7). On the other hand, it has been shown that mobile elements tend to be sensitive to changes in chromosome topology (Lodge and Berg, 1990). Indeed, among 552 genes affected by TopA depletion conditions, we found 16 genes (2.9% of SSGs) encoded putative transposable elements (which constitute only 0.7% of the S. coelicolor genome; see the complete list in the Supplementary Tables S2–S4). Why these mobile elements, and more broadly, the genes in the SHC, are specifically subject to topological control, remains an interesting question to address.In summary, as in other bacteria, in S. coelicolor chromosome, there are supercoiling sensitive domains that could be detected. They may constitute the regions of chromosomes particularly prone to changes of supercoiling and/or may be regulated by supercoiling sensitive transcriptional regulators.Long-Term Topological Imbalance Affects the Transcription of Genes Encoding Topology-Controlling ProteinsSince topological homeostasis could not be restored in the TopA-depleted S. coelicolor PS04 strain, either by induction of the topA promoter or by gyrBA silencing (Szafran et al., 2016), we predicted that the resulting increase in DNA supercoiling may promote changes in the level of NAPs to compensate for the changes in chromosome topology. According to previous reports (Bradshaw et al., 2013) some NAPs such as HupA are present at high levels during wild type S. coelicolor vegetative growth. Based on RNA-Seq data we confirmed that under standard growth conditions, many NAP genes, including hupA, ssbA, lsr2, sihf, and hupS were highly transcribed (Figure 4A), while other sporulation and/or stress responsive genes like dpsB, dpsC, and ssbB, were poorly transcribed. Next, we assessed the impact of the chromosome supercoiling changes on the expression of these genes.FIGURE 4The topological imbalance-induced transcriptional response of the genes encoding chromosome organizing proteins. (A) RNA-Seq based expression of the genes encoding DNA-organizing proteins (gray) and topoisomerases (black) in the wild type S. coelicolor strain normalized by the upper quartile. The error bars correspond to standard deviations calculated for four independent biological replicates. (B) RNA-Seq analysis of the relative transcriptional response of genes encoding DNA-organizing proteins under topA-downregulation conditions. The expression level of particular genes in the wild type strain was estimated as 1. (C) The relative transcription of selected genes in the TopA-depleted and control wild type strains calculated using RT-qPCR analysis. (D) RT-qPCR comparison of non-uniform topA gene transcription under different concentrations of topA inducer using oligonucleotide pairs specific for particular transcript regions. The transcript enrichment of particular regions was normalized by the expression of the hrdB gene which served as the endogenous control and compared to the wild type strain, in which transcription was estimated as 1. The amplified regions of the topA gene (1 – white; 2 – gray, and 3 – black) are shown above.Following TopA-depletion (conditions of increased negative supercoiling), we observed transcript levels of DNA-organizing proteins to be statistically different (p < 0.05) (Figure 4B). Of the NAPs affected by changes in chromosome topology, hupA and hupS were amongst the most impacted. Notably, however, the transcriptional effects were in opposite directions: hupA transcript levels increased to 1.44 relative to wild type conditions, while hupS transcript levels decreased to 0.76 compared with wild type (Figure 4B). To assess whether these effects were growth medium-specific, and to confirm the supercoiling dependence of these two HU-encoding genes, we grew the wild type and the PS04 strains in the 79 medium, and using RT-qPCR compared their expression under a range of topA induction conditions. As before, we observed hupA upregulation and hupS downregulation following TopA depletion if when compared to the wild type levels (Figure 4C). Previous work has suggested that HupS functions primarily during S. coelicolor sporulation (Salerno et al., 2009); however, both HU homologs, HupS and HupA, were shown to be present during S. coelicolor vegetative growth, although their levels are unequal (Bradshaw et al., 2013) (our transcriptional analysis showed that hupA transcript levels are over 14 times higher than the hupS transcripts). Our observations suggest that both S. coelicolor HU homologs contribute to the adjustment to altered supercoiling conditions. In E. coli two HU homologs, HupA and HupB form homo- and heterodimers at different growth stages, with the heterodimer having a crucial role when the culture reaches stationary phase (Claret and Rouviere-Yaniv, 1997). Furthermore, DNA supercoiling influences hupB transcription in mycobacteria, and its product affects TopA activity (Ghosh et al., 2014; Guha et al., 2018), suggesting that cooperation interactions between the HU proteins and TopA may be conserved in bacteria.In addition to the effects on hupA and hupS, increased chromosome supercoiling also profoundly affected the transcription (based on RNA-Seq data) of dpsA (decreased to 0.53 of wild type levels), smc (increased to 1.53 of wild type levels) (Figure 4B) and two genes (parC and parE) encoding topoisomerase IV subunits (increased to 1.86 and 1.67 of wild type levels, respectively). Using RT-qPCR, we confirmed upregulation of parC, parE, and smc genes. Similar observations have been made in E. coli, where increases in negative supercoiling result in the induction of mukBEF expression (Peter et al., 2004), where these genes encode homologs of SMC. Additionally, it is conceivable, that upregulating the expression of parC and parE (topoisomerase IV) may partially complement the effects of TopA depletion. However, the changes in dpsA transcription were not confirmed using RT-qPCR in cultures grown in different medium than used for our RNA-Seq experiment (Figure 4B,C). Since we did not observe any difference in DNA supercoiling (based on reporter plasmid assays, Supplementary Figure S4) in cultures growing in 79 or YEME/TSB medium, a possible explanation for this observation could be that dpsA transcription is affected by DNA supercoiling but only under osmotic stress conditions (in the YEME/TSB medium used in RNA-Seq experiment but not in 79 medium used for the RT-qPCR experiments) (Facey et al., 2009). Whether and how the changes in expression of genes encoding DNA organizing proteins influence chromosome topology have yet to be determined.Next, we assessed the effect of elevated topA expression (15–25× higher transcript levels compared with levels in the wild type strain). As mentioned, these increased topA transcript levels were not correlated with increased TopA protein whose levels were elevated only 22% in comparison with the wild type strain (Szafran et al., 2016; Figure 1 and Supplementary Figure S5A), thus only slightly affecting DNA supercoiling. Based on that observation, we speculated that TopA may be regulated posttranscriptionally and/or posttranslationally (e.g., may be subject to transcript processing or specific protein degradation). Indeed, while topA transcript levels were uniformly distributed along the 2858 bp topA gene in the wild type and the topA-downregulation strain), topA induction considerably increased the fraction of reads mapping to the beginning (region 1) but not to middle (region 2) or end (region 3) of the topA gene (Supplementary Figure S8). The non-uniform topA transcript levels were confirmed using RT-qPCR with specific oligonucleotides complementary to distinct fragments of the topA gene (Figure 4D). Interestingly, in the closely related M. smegmatis, gyrase binds within the topA promoter region, preventing its transcription (Ahmed et al., 2015), thus, similar mechanism of topA regulation in Streptomyces cannot be excluded. On the other hand, the possibility that topA transcripts are also subject to posttranscriptional processing and/or mRNA degradation could also explain the non-uniform detection of the particular topA transcript regions.In summary, the increased level of negative DNA supercoiling induces the coordinated modification of topoisomerases and NAPs level, presumably in an attempt to counteract the topological imbalance. Among them, the HupA protein may play a crucial role in maintaining nucleoid architecture under TopA depletion; with potential contributions from topoisomerase IV (ParCE) and downregulation of HupS – the other HU homolog present in Streptomyces. Intriguingly, topA induction had little effect on global DNA supercoiling, as Streptomyces appear to have mechanisms in place to ensure TopA levels are maintained below a certain threshold, through possible autoregulatory control.Alternating topA Transcript Levels Stimulates the Expression of DNA Damage Response GenesChromosome regulation stemming from gyrase inhibition with novobiocin had an immediate impact on 1.5% of the genes in the S. coelicolor chromosome. Conversely, the long-term severe alteration of topA levels during S. coelicolor growth resulted in more prominent changes in global genes transcription affecting up to 7% of the S. coelicolor genome. Interestingly, among the genes affected by severe depletion or modest increase in TopA levels we identified genes whose transcription was modified under both conditions.Among the genes sensitive to any alteration of TopA levels, 14 regulatory protein-encoding genes were identified (Table 4), including the four regulators in the SHC region [sco4667 (sensor kinase), sco4668 (regulatory protein), and putative transcriptional regulators (sco4671 and sco4673)]. Outside of this region was sco5803, which encodes the LexA repressor that controls the DNA damage response (highly induced by topA-up- and downregulation). In E. coli, the LexA-regulon encompasses at least 31 genes, including recA and lexA (the latter being negatively autoregulated), uvrA, ftsK, polB, dinF, and dnaE2, alongside others involved in the DNA damage response (Fernandez De Henestrosa et al., 2000). We found that the transcription of recA, recX, dnaE2, dinP, or uvrA genes, which presumably belong to the LexA-regulon in S. coelicolor, based on the predicted binding consensus sequence (Novichkov et al., 2013), were similarly affected by changes in chromosome supercoiling. This implies that the entire LexA regulon was impacted (Tables 5, 6), although the RT-qPCR analysis indicated that in 79 medium, this induction was less pronounced (Supplementary Figure S9).Table 4The regulators encoding genes affected by both constitutive topA-downregulation and topA-upregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).GeneLocusNormalized expression in the wild type∗Log2-fold change topA-downregulationLog2-fold change topA-upregulationDescriptionRegulatorssco3435141.721.58Transcriptional regulatorsco42142429-6.55-6.60AbaA-like regulatory proteinxlnRsco4215820-2.05-2.20GntR family transcriptional regulatorsco4671212.312.08LysR family transcriptional regulatorsco4673563.202.35DeoR family transcriptional regulatorsco56562804-1.80-2.15Transcriptional regulatorlexAsco58032402.041.62LexA repressornsdBsco7252118-1.67-2.80Regulatory proteinTCSssco4667129.758.93Two-component system sensor kinasesco4668677.516.59Two-component system response regulatorProteasesLonsco5285101.891.85ATP-dependent proteaseGNATssco1624132.512.74Acetyltransferasesco2379122.141.87AcetyltransferasedesCsco2784285-2.32-1.83Acetyltransferase∗Normalization by the upper quartile of gene expression.Table 5DNA processing and DNA repair genes impacted by constitutive topA-downregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).GeneLocusNormalized expression in the wild type∗Log2-fold changeDescriptiondnaE2sco173955.22DNA polymerase III subunit alphadinPsco173864.84DNA polymerase IVsco1767291.92DNA hydrolasesco1827721.51DNA polymerase III subunit epsilonuvrAsco1958691.89Exconuclease ABC subunit Asco1969671.88DNA-methyltransferasesco2863181.98Helicasesco3434272.01DNA polymerase Isco4685163.43DEAD/DEAH box helicasexseAsco5056691.51Exodeoxyribonuclease VII large subunitsco5183121.70ATP-dependent DNA helicasesco5184111.54ATP-dependent DNA helicaserecXsco57701363.15Recombination regulator RecXrecAsco57694262.76Recombinase AlexAsco58032402.04LexA repressorsco592091.53DEAD/DEAH box helicasesco671923.86UvrA-like ABC transportersco752272.32DNA ligase∗Normalization by the upper quartile of gene expression.Table 6DNA processing and DNA repair genes impacted by constitutive topA-upregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).GeneLocusNormalized expression in the wild type∗Log2-fold changeDescriptiondinPsco173865.04DNA polymerase IVdnaE2sco173954.96DNA polymerase III subunit alphasco17671221.81DNA hydrolaseuvrAsco1958691.98Excinuclease ABC subunit Asco3434272.29DNA polymerase Isco379821-2.81Chromosome condensation proteinsco4685162.67DEAD/DEAH box helicasesco5761201.54ATP-dependent DNA helicaserecAsco57694262.34Recombinase ArecXsco57701362.81Recombination regulator RecXlexAsco58032401.62LexA repressorsco752272.05DNA ligase∗Normalization by the upper quartile of gene expression.The identification of putative LexA-dependent genes among the genes upregulated when topA transcription was altered may suggest that these conditions trigger the LexA-dependent DNA repair pathway. However, the mechanism of LexA regulon induction by altered chromosome supercoiling remains speculative. We assume that in Streptomyces, as in other bacteria, LexA activity is controlled by RecA, which in the presence of ssDNA and in an ATP-dependent manner stimulates the self-cleavage of LexA, resulting in derepression of the LexA regulon. It is conceivable that changes in chromosome topology alter LexA binding to DNA, or that the formation of single-stranded DNA patches upon depleting TopA (Parsa et al., 2012) promote RecA-dependent cleavage of LexA and de-repression of LexA-regulated genes. However, the latter possibility could not account for the release of LexA repression during chromosome relaxation and further studies are required to understand the mechanism underlying LexA regulation in response to supercoiling changes. Since DNA damage is frequently accompanied by changes in local chromosome architecture, it is possible that changes in chromosome supercoiling may be sensed as a sign of DNA damage, leading to the induction of DNA repair genes. Interestingly, in Corynebacterium glutamicum, an actinobacterial relative of Streptomyces, induction of the DNA repair pathway was associated with the inhibition of cell division, which could explain the inhibition of sporulation seen for the S. coelicolor TopA-depleted strain (Ogino et al., 2008).Since we did not identify LexA-regulon members among novobiocin-sensitive genes, we speculate that the DNA repair response may be a result of long-term exposure to topological imbalance. Thus, topological imbalance, if cannot be compensated by response pathways, could serve as a marker of DNA damage.Developmental Transcriptional Regulators Are Sensitive to Specific Supercoiling ConditionsAlthough topA downregulation and topA-upregulation induced a common transcriptional response, a significant fraction of the affected genes responded specifically to particular supercoiling conditions (Figure 2C). We identified 12 SSGs-encoding regulatory proteins that were affected specifically by topA-upregulation (Table 7). These numbers are similar to those induced by novobiocin treatment, but surprisingly, the sets of genes identified in both experiments did not overlap (Supplementary Table S5). Amongst the many uncharacterized genes that were sensitive to topA-upregulation, developmental regulators such as sigF (upregulated) and nsdA (downregulated) were identified; however, these changes in expression were not associated with any obvious phenotype for the topA up-regulated strain.Table 7Regulatory protein-encoding genes impacted by constitutive topA-upregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).GeneLocusNormalized expression in the wild type∗Log2-fold changeDescriptionRegulatorssco014033.84MerR family transcriptional regulatorsco2865154-2.27Regulatory proteinsigFsco403542.70RNA polymerase sigma factorsco410262.42MerR family transcriptional regulatorsco5025227.71Transcriptional regulatorsco5437412.65MerR family transcriptional regulatornsdAsco558267-1.67Regulatorsco596228.10Transcriptional regulatorsco652034-1.63RNA polymerase sigma factorsco753064.61Regulatory proteinsco769829.13MerR family transcriptional regulatorTCSscutSsco5863116-1.57Two-component sensor kinase∗Normalization by the upper quartile of gene expression.The increased chromosome supercoiling resulting from TopA depletion specifically influenced a substantial number of genes encoding regulatory proteins. Among 35 of the TopA depletion-sensitive genes, we identified 18 putative transcriptional regulators, including seven genes encoding TCSs (kinases and/or their putative phosphorylation targets), six genes encoding putative acetyltransferases (GNATs family) and four genes encoding subunits of regulatory Clp proteases (Table 8), alongside the sporulation-specific sigma factor encoding whiG.Table 8Regulatory genes affected by topA-downregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).GeneLocusNormalized expression in the wild type∗Log2-fold changeDescriptionRegulatorssco013256-2.49Transcriptional regulatorsco02041059-1.88LuxR family transcriptional regulatorsco1104561.74TetR family transcriptional regulatorsco173662.84MarR family transcriptional regulatorsco2223541.72TetR family transcriptional regulatorsco4059221.61Transcriptional regulatorsco437591.58MarR family regulatory proteinsco5323521.57Transcriptional regulator AsnCsco541881.81Transcriptional regulatorwhiGsco562164.40RNA polymerase sigma factor WhiGsco5840112.24Transcriptional regulatorsco6299351.57TetR family transcriptional regulatorsco6664142.03Transcriptional regulatorsco7014221.75LacI family transcriptional regulatorsco70421141.60MarR family transcriptional regulatorsco7270151.62Transcriptional regulatorsco7639451.84MarR family regulatory proteinsco7727100-1.56MarR family regulatory proteinTCSsohkAsco1596311.77Two-component sensor kinaseabrA2sco174571.89Two-component system response regulatorsco314462.42Two-component system response regulatorsco3818179-1.96Two-component system response transcriptional regulatorsco436282.29Two component system sensor kinasesco5455151.87Two-component system response regulatorsco7649151.65Two-component system sensor kinaseProteasesclpP2sco26182584-1.77ATP-dependent Clp protease proteolytic subunitclpP1sco26193091-1.61ATP-dependent Clp protease proteolytic subunitclpAsco640829-1.69Clp protease ATP-binding subunitclpP4sco728081.91ATP-dependent Clp protease proteolytic subunit 2GNATssco1563461.55Acetyltransferasesco222733-2.24Acetyltransferasesco2876142.00Acetyltransferasesco405093-2.15Acetyltransferasesco522882.09Acetyltransferasesco744714.70Acetyltransferase∗Normalization by the upper quartile of gene expression.The gene encoding the sigma factor WhiG was one of the most significantly upregulated regulatory genes under TopA depletion based on RNA-Seq results (over 21-fold induction; Table 8 and Figure 5A). RT-qPCR confirmed that whiG transcription was stimulated by increased DNA supercoiling and decreased to wild type levels upon induction of normal topA transcription (Figure 5B); although the calculated fold-change was lower, probably due to detection of a small amount of whiG transcript in the wild type strain growing in 79 medium. WhiG is typically expressed at low levels during vegetative growth and is involved in the regulatory cascade that governs Streptomyces sporulation (Kelemen et al., 1996; Elliot et al., 2001). Since WhiG directs the expression of the whiI and whiH genes, but does not affect the transcription of other whi-family regulators, such as whiA and whiB (Mendez and Chater, 1987; Chater et al., 1989; Ryding et al., 1998), we analyzed the transcriptional response of these genes to increased DNA supercoiling. As expected, the transcription of both whiI and whiH was elevated following TopA depletion (although they were not identified in the initial screening due to below-threshold q-values, however, p < 0.001 was still statistically relevant), suggesting the induction of a WhiG-dependent regulatory cascade by changes in chromosome supercoiling (Figure 5A). In S. coelicolor whiG was shown to belong to the BldD-regulon, which also encompasses bldN, bldM, and whiB (Elliot et al., 2001; den Hengst et al., 2010). In fact, in S. venezuelae, the whiG gene was shown to be directly repressed by BldD, the master regulator that binds to DNA upon association with c-di-GMP (Tschowri et al., 2014). This prompted us to test whether increased DNA supercoiling influenced the transcription of other BldD-dependent genes. Indeed, upon TopA depletion, we observed high upregulation of three BldD targets: bldM, bldN, and whiG (Figure 5A). The mechanism of supercoiling-dependent upregulation of the BldD-regulon may, similar to that proposed for LexA, result from decreased BldD DNA-binding affinity during higher DNA supercoiling. Thus, alleviating BldD binding during times of increased DNA supercoiling could lead to increased target gene transcription.FIGURE 5Topoisomerase I depletion-induced changes in the transcription of genes involved in sporulation regulatory cascades. (A) RNA-Seq based expression of selected genes encoding for whi and bld regulatory cascades genes normalized by the upper quartile in the wild type strain (black) and the TopA-depleted strain (red). The genes belonging to the BldD regulon are marked with arrows. The statistically significant changes in transcript levels were marked with asterisks (∗p-value <0.05; ∗∗∗p-value <0.001). p-Values were calculated based on RNA-Seq data analysis for four independent biological replicates for the wild type and topA-downregulated strains. (B) RT-qPCR analysis of whiG transcript levels at different concentrations of the topA gene inducer. Wild type levels of whiG transcription were set at 1, and standard deviation for wild type transcription is marked with a gray bar. The inset shows RT-qPCR analysis of whiG transcript levels in the TopA-depleted and the wild type strains, as well as the control strain that contained the pIJ6902 plasmid without the topA gene. The whiG transcript enrichment was normalized by the expression of the hrdB gene which served as the endogenous control and compared to the wild type strain in which transcription was estimated as 1.Remarkably, even though elevated supercoiling induced key sporulation genes including whiG, their induction did not lead to sporulation; in fact, the TopA-depleted strain fails to form spores (Szafran et al., 2013). This may indicate either a lack of additional regulators needed for differentiation, or the activity of other signaling pathways that prevent sporulation (e.g., the DNA damage response, as described for RecA/LexA above). The observed induction of sporulation cascades does, however, suggest that DNA topology may function as a global regulator that triggers sporulation cascades either in response to environmental stress or physiological conditions that induce increased DNA supercoiling.

Results and Discussion

Rapid Loss of Chromosome Supercoiling Positively Regulates S. coelicolor Gene TranscriptionIn previous work, we found that exposing S. coelicolor to novobiocin, initially led to the rapid relaxation of DNA, followed by gradual restoration of topological homeostasis (Szafran et al., 2016), as had been seen in other bacteria (Ferrandiz et al., 2010). The return to topological balance appeared to stem from changes in the expression of topoisomerase encoding genes, which in S. coelicolor corresponded to a strong induction of gyrAB but only slight inhibition of topA (Szafran et al., 2016); this is different from the simultaneous induction of gyrAB and inhibition of topA transcription observed in other bacteria (Unniraman and Nagaraja, 1999; Ahmed et al., 2015). Here, we sought to identify other genes in S. coelicolor whose transcription was also directly affected by the rapid chromosome relaxation. To this end, we treated S. coelicolor liquid cultures with 10 μg/ml novobiocin, which strongly inhibited S. coelicolor growth (Supplementary Figure S1). We analyzed the changes in global transcript abundance after 10 min of novobiocin exposure, and compared these with an untreated control strain. Based on previous work (Gmuender et al., 2001; Szafran et al., 2016), 10 min of novobiocin exposure was presumed to be sufficient to affect promoters directly sensitive to gyrase inhibition, triggering the primary transcriptional response, but was deemed insufficient time to induce a secondary transcriptional response (which depends on any primary-induced transcriptional regulators).In analyzing our RNA-Seq data, we found that the rapid DNA relaxation caused by novobiocin (detected using a reporter plasmid) (Figure 1A and Supplementary Figure S5) led to a 2.71-fold induction of the gyrBA operon transcription (Figure 1B), confirming that the primary transcriptional response could be induced by 10 min of gyrase inhibition. We next set out to identify genes exhibiting distinct changes in transcription in comparison to the untreated control (at least 2.83-fold), and found 121 genes that were sensitive to the rapid loss of chromosome supercoiling. These genes constitute 1.5% of the S. coelicolor open reading frames (ORFs) (see Supplementary Table S2 “novobiocin” for the complete list of the identified genes). Mapping of the transcription start sites (TSSs) of the identified genes to the S. coelicolor chromosome revealed that the novobiocin-sensitive genes were unevenly distributed along the S. coelicolor chromosome (Figure 1C and Supplementary Figure S6A). Surprisingly, in contrast to E. coli, in which only one third of genes affected by gyrase inhibition were induced and the others were repressed (Peter et al., 2004), in S. coelicolor, most of the novobiocin-sensitive genes (117 of 121) were upregulated by chromosome relaxation, with only 4 genes (sco0498, sco1377, sco3726, and sco3918) being strongly downregulated (log2-fold >1.5).FIGURE 1Identification of novobiocin-sensitive genes in S. coelicolor. (A) Analysis of reporter plasmid supercoiling density. The reporter plasmid was isolated from untreated wild type strain (upper panel) and the same strain exposed to novobiocin (10 μg/mL) for 10 min (bottom panel). The most abundant topoisomers identified under each condition are marked with arrows. (B) RNA-Seq based gyrB expression normalized by the upper quartile, in the untreated wild type strain (gray) and the novobiocin-treated strain [as described for panel (A)] (black). (C) The chromosomal localization of the novobiocin-sensitive genes. Differentially upregulated (green) and downregulated (red) transcripts in the S. coelicolor culture following novobiocin exposure (as in panel A) were identified using log2-fold changes >1.5; p-value <0.05 thresholds. (D) Analysis of the average AT content in the promoter regions of novobiocin-sensitive genes (blue) in comparison to the random promoters (red) and random chromosomal sequences (gray). The transcriptional start site (TSS), promoter region and GC-enriched region identified upstream of novobiocin-sensitive promoters are marked. (E) The box plot comparison of AT content within the promoter region (upper panel) and GC-enriched region (bottom panel) of novobiocin-sensitive (blue) and randomly selected (red) promoters. The asterisks indicate the calculated p-value <0.05 (∗) or <0.01 (∗∗).We speculate that the gene upregulation after novobiocin exposure rather than their transcriptional repression may be due to the high GC content (72%) of the S. coelicolor genome (Bentley et al., 2002). In E. coli, promoter regions of relaxation-induced genes are notably enriched in AT base pairs, whereas relaxation-repressed promoters have increased GC content (Peter et al., 2004). In S. coelicolor, the increase in GC content in the promoter regions to levels above the average would be expected to result in increased DNA stability, which in turn would inhibit promoter unwinding. Conversely, more AT-rich promoter regions would be expected to favor promoter upregulation. To test this hypothesis, we compared the AT content of 66 novobiocin-induced promoters (the identified 121 genes were recognized as 66 transcriptional units) with the AT content of randomly chosen promoter regions. As predicted, we found that novobiocin-induced genes showed approximately 2–3% higher AT content compared with the randomly selected promoters and 8–9% higher in comparison to the average of the S. coelicolor genome (28%). Notably, the AT-rich region was significantly expanded in the novobiocin-sensitive genes relative to the random promoters (Figure 1D). Unexpectedly, AT-content analysis revealed a GC-rich region located approximately 100–200 bp upstream of the novobiocin-induced genes (Figure 1D), and not the random promoters. The difference in the mean AT-content values between the novobiocin-sensitive and random promoters, calculated for position 340 bp (GC-enriched region) and 660 bp (promoter region) were statistically relevant (p-values of 0.006 and 0.017, respectively; Figure 1E). Interestingly, similar analysis performed in E. coli did not indicate the existence of a GC-rich fragment preceding the relaxation-induced genes (Peter et al., 2004), although the significance of the GC-enriched region upstream of the Streptomyces novobiocin-inducible promoters is currently unclear.Among the 121 novobiocin-sensitive genes, 15 encoded putative regulatory proteins, including the anti-sigma factor RsfA (Homerova et al., 2000), the developmental regulator WblC (Fowler-Goldsworthy et al., 2011), and the secondary metabolism regulator NsdB (Zhang et al., 2007). Interestingly, a high fraction of the novobiocin-sensitive genes (∼30%) encode proteins involved in either antibiotic production (10 genes, including the actinorhodin activator sco5085/actII-orf4 and genes within the coelibactin biosynthetic cluster sco7681-sco7688) or putative membrane transporters (at least 25 genes) (Table 2). This suggests that rapid chromosome relaxation functions as the stress signal that triggers the transcription of genes whose products are involved in stress adaptation (i.e., those encoding regulatory proteins) and/or interspecies competition. Considering that decreased gyrase activity and rapid chromosome relaxation may result from exposure to different cell membrane destabilizing agents that affect transmembrane potential and ATP synthesis, or other biologically active molecules produced by competing bacterial species (Prakash et al., 2009), it is conceivable that the intrinsic response to such agents should encompass the induction of transport proteins that should restore homeostasis to the intracellular environment and/or function to eliminate toxic compounds.Table 2Selected novobiocin-induced genes encoding transporters or transcriptional regulators (based on RNA-Seq data; log2-fold >1.5; p < 0.05).GeneLocusNormalized expression in the wild type∗Log2-fold changeDescriptionTransporterssco0623571.86Membrane transportersco2310191.86Integral membrane efflux proteinsco2763111.79ABC transporter ATP-binding proteinsco2896/sco2895/sco28942/7/64.09/2.51/2.17Possible ABC transportersco3206581.81Transmembrane efflux proteinsco333191.64Transmembrane transportersco345591.92ABC transportersco4031172.25Integral membrane transporterragAsco4075241.52ABC transporter ATP-binding proteinsco5451361.54ABC transportersco5957302.65Transportersco672053.17ABC transporterRegulatorsssgGsco292452.32Putative regulatorsco296454.60LysR family transcriptional regulatorsco4032143.58MarR regulatory proteinrsfAsco4677111.67Regulatory proteinwblCsco51901641.91DNA-binding proteinnsdBsco72521181.53Regulatory proteinAntibiotic synthesisactII-4sco5085151.68Actinorhodin cluster activator proteinsco722191.64Polyketide synthasesco7681–sco76886–171.58–2.00Coelibactin synthesis genesOther genesrpmGsco0570681.5850S ribosomal protein L33tuf3sco1321104.60Elongation factor Tusco2171141.79Putative secreted proteaseohrAsco298621.88Hypothetical proteinsco3425241.5030S ribosomal protein S18sco4316112.37ATP/GTP-binding proteinsco4449213.23Hydrolaselonsco5285102.56ATP-dependent protease∗Normalization by the upper quartile of gene expression.In summary, a significant fraction of novobiocin-sensitive genes encompassed those encoding membrane transporters, antibiotic synthesis and regulatory proteins. The novobiocin-sensitive genes were non-uniformly distributed along the chromosome, and most of them were upregulated. The upregulation was reflected by the increased AT-content of their promoter regions and the presence of an upstream GC-rich DNA stretch.

Rapid Loss of Chromosome Supercoiling Positively Regulates S. coelicolor Gene Transcription

In previous work, we found that exposing S. coelicolor to novobiocin, initially led to the rapid relaxation of DNA, followed by gradual restoration of topological homeostasis (Szafran et al., 2016), as had been seen in other bacteria (Ferrandiz et al., 2010). The return to topological balance appeared to stem from changes in the expression of topoisomerase encoding genes, which in S. coelicolor corresponded to a strong induction of gyrAB but only slight inhibition of topA (Szafran et al., 2016); this is different from the simultaneous induction of gyrAB and inhibition of topA transcription observed in other bacteria (Unniraman and Nagaraja, 1999; Ahmed et al., 2015). Here, we sought to identify other genes in S. coelicolor whose transcription was also directly affected by the rapid chromosome relaxation. To this end, we treated S. coelicolor liquid cultures with 10 μg/ml novobiocin, which strongly inhibited S. coelicolor growth (Supplementary Figure S1). We analyzed the changes in global transcript abundance after 10 min of novobiocin exposure, and compared these with an untreated control strain. Based on previous work (Gmuender et al., 2001; Szafran et al., 2016), 10 min of novobiocin exposure was presumed to be sufficient to affect promoters directly sensitive to gyrase inhibition, triggering the primary transcriptional response, but was deemed insufficient time to induce a secondary transcriptional response (which depends on any primary-induced transcriptional regulators).

In analyzing our RNA-Seq data, we found that the rapid DNA relaxation caused by novobiocin (detected using a reporter plasmid) (Figure 1A and Supplementary Figure S5) led to a 2.71-fold induction of the gyrBA operon transcription (Figure 1B), confirming that the primary transcriptional response could be induced by 10 min of gyrase inhibition. We next set out to identify genes exhibiting distinct changes in transcription in comparison to the untreated control (at least 2.83-fold), and found 121 genes that were sensitive to the rapid loss of chromosome supercoiling. These genes constitute 1.5% of the S. coelicolor open reading frames (ORFs) (see Supplementary Table S2 “novobiocin” for the complete list of the identified genes). Mapping of the transcription start sites (TSSs) of the identified genes to the S. coelicolor chromosome revealed that the novobiocin-sensitive genes were unevenly distributed along the S. coelicolor chromosome (Figure 1C and Supplementary Figure S6A). Surprisingly, in contrast to E. coli, in which only one third of genes affected by gyrase inhibition were induced and the others were repressed (Peter et al., 2004), in S. coelicolor, most of the novobiocin-sensitive genes (117 of 121) were upregulated by chromosome relaxation, with only 4 genes (sco0498, sco1377, sco3726, and sco3918) being strongly downregulated (log2-fold >1.5).

Identification of novobiocin-sensitive genes in S. coelicolor. (A) Analysis of reporter plasmid supercoiling density. The reporter plasmid was isolated from untreated wild type strain (upper panel) and the same strain exposed to novobiocin (10 μg/mL) for 10 min (bottom panel). The most abundant topoisomers identified under each condition are marked with arrows. (B) RNA-Seq based gyrB expression normalized by the upper quartile, in the untreated wild type strain (gray) and the novobiocin-treated strain [as described for panel (A)] (black). (C) The chromosomal localization of the novobiocin-sensitive genes. Differentially upregulated (green) and downregulated (red) transcripts in the S. coelicolor culture following novobiocin exposure (as in panel A) were identified using log2-fold changes >1.5; p-value <0.05 thresholds. (D) Analysis of the average AT content in the promoter regions of novobiocin-sensitive genes (blue) in comparison to the random promoters (red) and random chromosomal sequences (gray). The transcriptional start site (TSS), promoter region and GC-enriched region identified upstream of novobiocin-sensitive promoters are marked. (E) The box plot comparison of AT content within the promoter region (upper panel) and GC-enriched region (bottom panel) of novobiocin-sensitive (blue) and randomly selected (red) promoters. The asterisks indicate the calculated p-value <0.05 (∗) or <0.01 (∗∗).

We speculate that the gene upregulation after novobiocin exposure rather than their transcriptional repression may be due to the high GC content (72%) of the S. coelicolor genome (Bentley et al., 2002). In E. coli, promoter regions of relaxation-induced genes are notably enriched in AT base pairs, whereas relaxation-repressed promoters have increased GC content (Peter et al., 2004). In S. coelicolor, the increase in GC content in the promoter regions to levels above the average would be expected to result in increased DNA stability, which in turn would inhibit promoter unwinding. Conversely, more AT-rich promoter regions would be expected to favor promoter upregulation. To test this hypothesis, we compared the AT content of 66 novobiocin-induced promoters (the identified 121 genes were recognized as 66 transcriptional units) with the AT content of randomly chosen promoter regions. As predicted, we found that novobiocin-induced genes showed approximately 2–3% higher AT content compared with the randomly selected promoters and 8–9% higher in comparison to the average of the S. coelicolor genome (28%). Notably, the AT-rich region was significantly expanded in the novobiocin-sensitive genes relative to the random promoters (Figure 1D). Unexpectedly, AT-content analysis revealed a GC-rich region located approximately 100–200 bp upstream of the novobiocin-induced genes (Figure 1D), and not the random promoters. The difference in the mean AT-content values between the novobiocin-sensitive and random promoters, calculated for position 340 bp (GC-enriched region) and 660 bp (promoter region) were statistically relevant (p-values of 0.006 and 0.017, respectively; Figure 1E). Interestingly, similar analysis performed in E. coli did not indicate the existence of a GC-rich fragment preceding the relaxation-induced genes (Peter et al., 2004), although the significance of the GC-enriched region upstream of the Streptomyces novobiocin-inducible promoters is currently unclear.

Among the 121 novobiocin-sensitive genes, 15 encoded putative regulatory proteins, including the anti-sigma factor RsfA (Homerova et al., 2000), the developmental regulator WblC (Fowler-Goldsworthy et al., 2011), and the secondary metabolism regulator NsdB (Zhang et al., 2007). Interestingly, a high fraction of the novobiocin-sensitive genes (∼30%) encode proteins involved in either antibiotic production (10 genes, including the actinorhodin activator sco5085/actII-orf4 and genes within the coelibactin biosynthetic cluster sco7681-sco7688) or putative membrane transporters (at least 25 genes) (Table 2). This suggests that rapid chromosome relaxation functions as the stress signal that triggers the transcription of genes whose products are involved in stress adaptation (i.e., those encoding regulatory proteins) and/or interspecies competition. Considering that decreased gyrase activity and rapid chromosome relaxation may result from exposure to different cell membrane destabilizing agents that affect transmembrane potential and ATP synthesis, or other biologically active molecules produced by competing bacterial species (Prakash et al., 2009), it is conceivable that the intrinsic response to such agents should encompass the induction of transport proteins that should restore homeostasis to the intracellular environment and/or function to eliminate toxic compounds.

Selected novobiocin-induced genes encoding transporters or transcriptional regulators (based on RNA-Seq data; log2-fold >1.5; p < 0.05).

In summary, a significant fraction of novobiocin-sensitive genes encompassed those encoding membrane transporters, antibiotic synthesis and regulatory proteins. The novobiocin-sensitive genes were non-uniformly distributed along the chromosome, and most of them were upregulated. The upregulation was reflected by the increased AT-content of their promoter regions and the presence of an upstream GC-rich DNA stretch.

Long-Term Exposure to Topological Imbalance Globally Affects Gene TranscriptionHaving established that approximately 120 genes were directly impacted by gyrase inhibition, we were next interested in assessing the response of S. coelicolor to the long-term effects of altered chromosome supercoiling. To date, there is little understood about the long-term effects of altered supercoiling, since most studied bacteria were able to quickly restore topological homeostasis. Moreover, the effects of increased negative supercoiling on global gene transcription in bacteria remain relatively unexplored, in part due to the limited availability of selective TopA inhibitors (Garcia et al., 2011; Cheng et al., 2013; Godbole et al., 2014; Szafran et al., 2018).To investigate long terms effects of supercoiling imbalance, we took advantage of a S. coelicolor strain (PS04) that had been engineered such that chromosome supercoiling could be modulated by tuning TopA levels using a thiostrepton-inducible topA gene (Szafran et al., 2013). In this PS04 strain, TopA depletion (no induction) dramatically increased the overall negative DNA supercoiling (Figure 2A and Supplementary Figures S5A,B), but TopA levels could not be restored, thus impeding the restoration of supercoiling balance. Moreover, inefficient repression of the S. coelicolor gyrase operon eliminated another possible pathway to restoring chromosomal topological balance in the TopA-depleted strain. On the other hand induction of topA expression in the PS04 strain led only to a slight (approximately 22%) increase of TopA protein levels which also corresponded to a modest shift in global DNA supercoiling levels if compared to the wild type strain (Figure 2A and Supplementary Figure S5). This observation is in agreement with our previous data showing that by increasing the concentration of topA inducer, only a 25% increase in TopA levels could be achieved (Szafran et al., 2013). This may be partially explained by induction of the gyrBA operon, which could compensate for high levels of TopA, and restore the supercoiling balance (Szafran et al., 2016) or by posttranscriptional regulation mechanisms of topA gene expression.FIGURE 2The transcriptional response to long-term supercoiling imbalance. (A) The supercoiling density analysis of the reporter plasmid isolated from the wild type strain (upper panel), the topA-upregulated strain (middle panel), and the topA-downregulated strain (lower panel). The most abundant topoisomers under each condition are marked with arrows. (B) Differentially regulated genes identified using RNA-Seq following topA-downregulation (left) and topA-upregulation (right). The up- (black) or downregulated (gray) genes were grouped according to the magnitude of their transcriptional changes compared to the wild type strain (log2-fold change value). (C) Comparison of the number of genes specifically affected by topA-upregulation and topA-downregulation, and those common to both conditions.First, we compared the number of S. coelicolor genes that were strongly affected by topA-down- or upregulation (cultivation in the absence or presence of 1 μg/ml thiostrepton). We identified 552 genes whose transcription was changed by topA-downregulation and 214 genes altered by the induction of topA transcription (at least 2.83-fold change; log2-fold >1.5), which corresponded to 7.0 and 2.7% of the S. coelicolor predicted coding sequences, respectively (Figure 2B; see Supplementary Tables S3, S4 “topA-downregulation,” “topA-upregulation” for the complete list of affected genes). Notably, most of the identified ORFs were positively regulated (Figure 2B), and the positive effect on gene transcription was particularly pronounced following topA-upregulation. These observations corroborate the notion that chromosome relaxation preferentially promoted the induction of gene transcription in GC-rich bacteria. Surprisingly, a significant number of genes were sensitive to both changes in supercoiling conditions (Figure 2C). Nevertheless, a substantial fraction of genes responded specifically to an increase of negative supercoiling resulting from TopA depletion (408 of 552, more than 70%), whereas topA-upregulation regulated specifically only 32% (70 of 214) of identified genes (Figure 2C). The observed difference in the number of the supercoiling sensitive genes (SSGs), as well as their specificity for particular supercoiling conditions, may be presumably explained by the scale of topological imbalance, which is more pronounced under the former than under the latter conditions (Figure 2A and Supplementary Figure S5). However, our results also show that, even though the topological balance was only slightly distorted following topA induction, the transcription of a relatively high number of genes was still affected, suggesting the presence of a mechanism(s) that maintains a modified level of gene transcription. Markedly, similar observations were made recently for S. pneumoniae transcriptional activity of genes under TopA inhibition conditions (Ferrandiz et al., 2016), where the expression of many genes was specifically altered. Interestingly, the inhibition of TopA in S. pneumoniae (which was followed by restoring of supercoiling imbalance) affected only 2% of the genome, while long term TopA depletion in S. coelicolor had a more pronounced effect on the gene transcription. Interestingly, the genes affected by long-term supercoiling changes only partially overlapped with those that were novobiocin-sensitive (12% of novobiocin-affected genes, Supplementary Table S5), suggesting that long-term rearrangement of bacterial chromosome topology provokes a global change in gene transcription and includes primary and secondary response pathways.Overall, we observed that a persistent supercoiling imbalance induced by a constitutive change of TopA expression, affected the transcription of approximately 3–7% of S. coelicolor genes. A significant subfraction of these were sensitive to any supercoiling imbalance, suggesting shared transcriptional response pathway(s). Our results suggest that the response to topological imbalance depends on accessible pathways of supercoiling restoration, being the most pronounced when topoisomerase gene expression cannot not be readjusted.

Long-Term Exposure to Topological Imbalance Globally Affects Gene Transcription

Having established that approximately 120 genes were directly impacted by gyrase inhibition, we were next interested in assessing the response of S. coelicolor to the long-term effects of altered chromosome supercoiling. To date, there is little understood about the long-term effects of altered supercoiling, since most studied bacteria were able to quickly restore topological homeostasis. Moreover, the effects of increased negative supercoiling on global gene transcription in bacteria remain relatively unexplored, in part due to the limited availability of selective TopA inhibitors (Garcia et al., 2011; Cheng et al., 2013; Godbole et al., 2014; Szafran et al., 2018).

To investigate long terms effects of supercoiling imbalance, we took advantage of a S. coelicolor strain (PS04) that had been engineered such that chromosome supercoiling could be modulated by tuning TopA levels using a thiostrepton-inducible topA gene (Szafran et al., 2013). In this PS04 strain, TopA depletion (no induction) dramatically increased the overall negative DNA supercoiling (Figure 2A and Supplementary Figures S5A,B), but TopA levels could not be restored, thus impeding the restoration of supercoiling balance. Moreover, inefficient repression of the S. coelicolor gyrase operon eliminated another possible pathway to restoring chromosomal topological balance in the TopA-depleted strain. On the other hand induction of topA expression in the PS04 strain led only to a slight (approximately 22%) increase of TopA protein levels which also corresponded to a modest shift in global DNA supercoiling levels if compared to the wild type strain (Figure 2A and Supplementary Figure S5). This observation is in agreement with our previous data showing that by increasing the concentration of topA inducer, only a 25% increase in TopA levels could be achieved (Szafran et al., 2013). This may be partially explained by induction of the gyrBA operon, which could compensate for high levels of TopA, and restore the supercoiling balance (Szafran et al., 2016) or by posttranscriptional regulation mechanisms of topA gene expression.

The transcriptional response to long-term supercoiling imbalance. (A) The supercoiling density analysis of the reporter plasmid isolated from the wild type strain (upper panel), the topA-upregulated strain (middle panel), and the topA-downregulated strain (lower panel). The most abundant topoisomers under each condition are marked with arrows. (B) Differentially regulated genes identified using RNA-Seq following topA-downregulation (left) and topA-upregulation (right). The up- (black) or downregulated (gray) genes were grouped according to the magnitude of their transcriptional changes compared to the wild type strain (log2-fold change value). (C) Comparison of the number of genes specifically affected by topA-upregulation and topA-downregulation, and those common to both conditions.

First, we compared the number of S. coelicolor genes that were strongly affected by topA-down- or upregulation (cultivation in the absence or presence of 1 μg/ml thiostrepton). We identified 552 genes whose transcription was changed by topA-downregulation and 214 genes altered by the induction of topA transcription (at least 2.83-fold change; log2-fold >1.5), which corresponded to 7.0 and 2.7% of the S. coelicolor predicted coding sequences, respectively (Figure 2B; see Supplementary Tables S3, S4 “topA-downregulation,” “topA-upregulation” for the complete list of affected genes). Notably, most of the identified ORFs were positively regulated (Figure 2B), and the positive effect on gene transcription was particularly pronounced following topA-upregulation. These observations corroborate the notion that chromosome relaxation preferentially promoted the induction of gene transcription in GC-rich bacteria. Surprisingly, a significant number of genes were sensitive to both changes in supercoiling conditions (Figure 2C). Nevertheless, a substantial fraction of genes responded specifically to an increase of negative supercoiling resulting from TopA depletion (408 of 552, more than 70%), whereas topA-upregulation regulated specifically only 32% (70 of 214) of identified genes (Figure 2C). The observed difference in the number of the supercoiling sensitive genes (SSGs), as well as their specificity for particular supercoiling conditions, may be presumably explained by the scale of topological imbalance, which is more pronounced under the former than under the latter conditions (Figure 2A and Supplementary Figure S5). However, our results also show that, even though the topological balance was only slightly distorted following topA induction, the transcription of a relatively high number of genes was still affected, suggesting the presence of a mechanism(s) that maintains a modified level of gene transcription. Markedly, similar observations were made recently for S. pneumoniae transcriptional activity of genes under TopA inhibition conditions (Ferrandiz et al., 2016), where the expression of many genes was specifically altered. Interestingly, the inhibition of TopA in S. pneumoniae (which was followed by restoring of supercoiling imbalance) affected only 2% of the genome, while long term TopA depletion in S. coelicolor had a more pronounced effect on the gene transcription. Interestingly, the genes affected by long-term supercoiling changes only partially overlapped with those that were novobiocin-sensitive (12% of novobiocin-affected genes, Supplementary Table S5), suggesting that long-term rearrangement of bacterial chromosome topology provokes a global change in gene transcription and includes primary and secondary response pathways.

Overall, we observed that a persistent supercoiling imbalance induced by a constitutive change of TopA expression, affected the transcription of approximately 3–7% of S. coelicolor genes. A significant subfraction of these were sensitive to any supercoiling imbalance, suggesting shared transcriptional response pathway(s). Our results suggest that the response to topological imbalance depends on accessible pathways of supercoiling restoration, being the most pronounced when topoisomerase gene expression cannot not be readjusted.

Long-Term Supercoiling Imbalance Impacts the Expression of Discrete Gene ClustersRemarkably, supercoiling sensitive genes whose transcription was affected by long-term topological imbalance were non-uniformly distributed along the S. coelicolor chromosome, similar to genes induced in the novobiocin-treated cells whose distribution also was non-uniform. SSGs induced by topA-downregulation or topA-upregulation showed characteristic grouping, suggesting they were organized into supercoiling-sensitive clusters, which were in turn separated by supercoiling insensitive regions (Figure 3A). To quantify these observations, we calculated the percentage of genes affected by different supercoiling conditions within 250 kbp chromosome regions along the whole chromosome. We found that gene clustering was strongly detectable under topA-downregulation conditions, with 6–7 chromosomal regions containing more than 5% of genes affected negatively or positively in each cluster (Figure 3B). Notably, those clusters containing the positively and negatively regulated genes partially overlapped, further supporting the existence of DNA domains that are particularly sensitive to changes in DNA supercoiling. Interestingly, gene clustering was also detectable, although less pronounced, under topA-upregulation conditions, where the distribution of the supercoiling-sensitive regions along the chromosome was comparable to that observed in the TopA-depleted strain (Figure 3B and Supplementary Figure S6B). Analogous clustering of SSGs has been described previously for E. coli and S. pneumoniae (Peter et al., 2004; Ferrandiz et al., 2016). Collectively, these observations suggest that the organization of SSGs into topologically separated domains is a feature conserved among bacteria, presumably ensuring the coordinated expression of the associated genes. Remarkably, the loci of the genes affected by topological imbalance showed increased density in the central part of the chromosome, however, the initiation of the chromosomal replication region (oriC) was found to be outside the identified SSG clusters, and the oriC proximal genes, which predominantly encode proteins involved in DNA replication, were insensitive to the changes in chromosome supercoiling (Figure 3). Surprisingly, the genes and genes clusters that were affected by long-term supercoiling modification did not overlap with those induced by novobiocin (Supplementary Figures S6A,B). Presumably during long-term DNA relaxation, the first response genes are no longer activated and the affected genes include those of the pathways responsible for constitutive maintenance of cell homeostasis under such conditions.FIGURE 3Identification of long-term supercoiling imbalance-sensitive clusters. (A) The chromosomal positions of differentially upregulated (green) and downregulated (red) transcripts identified in the topA-downregulated strain (upper panel) and topA-upregulated strain (lower panel). The positions of the oriC region and the supercoiling-hypersensitive cluster (SHC) are marked in gray. (B) Identification of supercoiling-sensitive clusters. The percent of genes affected by topA downregulation (upper panel) or topA upregulation (lower panel) was calculated in a 250-kbp window and the mid of the window was plotted versus chromosome position. The regions with the percentage of upregulated (green) or downregulated (red) genes higher than 5% (supercoiling-sensitive clusters) are marked at the bottom of the plot.Interestingly, among several identified gene clusters impacted by supercoiling imbalance, one cluster was particularly enriched in SSGs, which constituted ∼20% of all genes positioned within the 250 kbp region. Closer examination of this region revealed that 26 out of 34 genes within 30 kbp (between sco4667 and sco4700) were supercoiling sensitive. We therefore termed this region the supercoiling-hypersensitive cluster (SHC). Most of the SHC genes encoded products of unassigned function and these genes themselves were poorly transcribed under standard culture conditions. However, they were strongly upregulated under both topA-downregulation and topA-upregulation conditions (Table 3). We confirmed these observations using RT-qPCR for the first (sco4667) and one before last (sco4699) genes of the SHC region and excluded the effect of growth medium or genetic modifications in the topA depleted PS04 strain (the integration of pIJ6902 plasmid, that contains topA gene under tipA promoter), by measuring the relative transcript level in PS04 and control strain (wild type with empty pIJ602 plasmid) cultures grown in liquid 79 medium. The transcription of both genes appeared to be highly dependent on the level of TopA: significantly increased when TopA was depleted, decreased when TopA was at the wild type level, and increased slightly again at topA induction (Supplementary Figure S6C). These results were further supported by the observation that SHC genes upregulation was correlated with relative degree of topological imbalance. However, since we did not observe increased expression of the genes in the SHC region following a 10 min exposure to novobiocin (Supplementary Figure S6D), we speculate that their activation requires not only a supercoiling imbalance, but also the activity of other factors.Table 3Transcription of genes in the SHC region at constitutive topA-down- or upregulation (based on RNA-Seq data).GeneLocusNormalized expression in the wild type∗Log2-fold change topA-downregulationLog2-fold change topA-upregulationDescriptionsco4667129.758.93Two-component system sensor kinasesco4668677.516.59Two-component system response regulatorsco4669356.065.39Hypothetical proteinsco467000.000.00Serine proteasesco4671212.312.08LysR family transcriptional regulatorsco4672551.611.73Hypothetical proteinsco4673563.202.35DeoR family transcriptional regulatorsco4674262.002.08Hypothetical proteinsco4675771.762.12Hypothetical proteinsco467631.220.41Hypothetical proteinrsfAsco4677111.240.79Regulatory proteinsco46781971.991.59Hypothetical proteinsco46795301.981.63Hypothetical proteinsco46801161.371.56DNA-binding proteinsco46811370.691.29Short chain dehydrogenasesco46821670.651.37TautomerasegdhAsco46832122.062.36Glutamate dehydrogenasescoF3sco468430874.083.00Cold shock proteinsco4685163.432.67DEAD/DEATH box helicasesco4686442.531.83Hypothetical proteinsco46876742.381.79Hypothetical proteinsco46881372.031.08Hypothetical proteinsco4689345-0.262.03Hypothetical proteinsco46901480.412.39Hypothetical proteinsco4691101.582.43Hypothetical proteinsco46921422.562.18Hypothetical proteinsco4693622.862.03Hypothetical proteinsco4694124.202.67Hypothetical proteinsco469504.462.32Hypothetical proteinsco469684.623.22Hypothetical proteinsco4697234.483.08Hypothetical proteinsco4698342.842.67IS1652 transposasesco469985.783.74Rhs proteinsco4700136.044.01Hypothetical protein∗Normalization by the upper quartile of gene expression.The SHC genes include those encoding a putative two-component system (sco4667 and sco4668) genes organized in a single operon and additional two putative transcriptional regulator-encoding genes positioned upstream (sco4671 and sco4673). The cluster also encompasses genes whose products may contribute to DNA/RNA transactions, including: putative DNA helicase (sco4685), transposase (sco4698), and a homolog of E. coli Rhs protein (rearrangement hotspot protein), which putatively contains the C-terminal toxin domain and is involved in bacterial intercellular competition (Koskiniemi et al., 2013). When we compared the genes in this region to other Streptomyces species, we found the synteny was not strong, perhaps suggesting that some of these genes may have been acquired through horizontal transfer (Hindra et al., 2014). However, the AT content of the SHC region was not significantly different from the rest of S. coelicolor chromosome (Supplementary Figure S7). On the other hand, it has been shown that mobile elements tend to be sensitive to changes in chromosome topology (Lodge and Berg, 1990). Indeed, among 552 genes affected by TopA depletion conditions, we found 16 genes (2.9% of SSGs) encoded putative transposable elements (which constitute only 0.7% of the S. coelicolor genome; see the complete list in the Supplementary Tables S2–S4). Why these mobile elements, and more broadly, the genes in the SHC, are specifically subject to topological control, remains an interesting question to address.In summary, as in other bacteria, in S. coelicolor chromosome, there are supercoiling sensitive domains that could be detected. They may constitute the regions of chromosomes particularly prone to changes of supercoiling and/or may be regulated by supercoiling sensitive transcriptional regulators.

Long-Term Supercoiling Imbalance Impacts the Expression of Discrete Gene Clusters

Remarkably, supercoiling sensitive genes whose transcription was affected by long-term topological imbalance were non-uniformly distributed along the S. coelicolor chromosome, similar to genes induced in the novobiocin-treated cells whose distribution also was non-uniform. SSGs induced by topA-downregulation or topA-upregulation showed characteristic grouping, suggesting they were organized into supercoiling-sensitive clusters, which were in turn separated by supercoiling insensitive regions (Figure 3A). To quantify these observations, we calculated the percentage of genes affected by different supercoiling conditions within 250 kbp chromosome regions along the whole chromosome. We found that gene clustering was strongly detectable under topA-downregulation conditions, with 6–7 chromosomal regions containing more than 5% of genes affected negatively or positively in each cluster (Figure 3B). Notably, those clusters containing the positively and negatively regulated genes partially overlapped, further supporting the existence of DNA domains that are particularly sensitive to changes in DNA supercoiling. Interestingly, gene clustering was also detectable, although less pronounced, under topA-upregulation conditions, where the distribution of the supercoiling-sensitive regions along the chromosome was comparable to that observed in the TopA-depleted strain (Figure 3B and Supplementary Figure S6B). Analogous clustering of SSGs has been described previously for E. coli and S. pneumoniae (Peter et al., 2004; Ferrandiz et al., 2016). Collectively, these observations suggest that the organization of SSGs into topologically separated domains is a feature conserved among bacteria, presumably ensuring the coordinated expression of the associated genes. Remarkably, the loci of the genes affected by topological imbalance showed increased density in the central part of the chromosome, however, the initiation of the chromosomal replication region (oriC) was found to be outside the identified SSG clusters, and the oriC proximal genes, which predominantly encode proteins involved in DNA replication, were insensitive to the changes in chromosome supercoiling (Figure 3). Surprisingly, the genes and genes clusters that were affected by long-term supercoiling modification did not overlap with those induced by novobiocin (Supplementary Figures S6A,B). Presumably during long-term DNA relaxation, the first response genes are no longer activated and the affected genes include those of the pathways responsible for constitutive maintenance of cell homeostasis under such conditions.

Identification of long-term supercoiling imbalance-sensitive clusters. (A) The chromosomal positions of differentially upregulated (green) and downregulated (red) transcripts identified in the topA-downregulated strain (upper panel) and topA-upregulated strain (lower panel). The positions of the oriC region and the supercoiling-hypersensitive cluster (SHC) are marked in gray. (B) Identification of supercoiling-sensitive clusters. The percent of genes affected by topA downregulation (upper panel) or topA upregulation (lower panel) was calculated in a 250-kbp window and the mid of the window was plotted versus chromosome position. The regions with the percentage of upregulated (green) or downregulated (red) genes higher than 5% (supercoiling-sensitive clusters) are marked at the bottom of the plot.

Interestingly, among several identified gene clusters impacted by supercoiling imbalance, one cluster was particularly enriched in SSGs, which constituted ∼20% of all genes positioned within the 250 kbp region. Closer examination of this region revealed that 26 out of 34 genes within 30 kbp (between sco4667 and sco4700) were supercoiling sensitive. We therefore termed this region the supercoiling-hypersensitive cluster (SHC). Most of the SHC genes encoded products of unassigned function and these genes themselves were poorly transcribed under standard culture conditions. However, they were strongly upregulated under both topA-downregulation and topA-upregulation conditions (Table 3). We confirmed these observations using RT-qPCR for the first (sco4667) and one before last (sco4699) genes of the SHC region and excluded the effect of growth medium or genetic modifications in the topA depleted PS04 strain (the integration of pIJ6902 plasmid, that contains topA gene under tipA promoter), by measuring the relative transcript level in PS04 and control strain (wild type with empty pIJ602 plasmid) cultures grown in liquid 79 medium. The transcription of both genes appeared to be highly dependent on the level of TopA: significantly increased when TopA was depleted, decreased when TopA was at the wild type level, and increased slightly again at topA induction (Supplementary Figure S6C). These results were further supported by the observation that SHC genes upregulation was correlated with relative degree of topological imbalance. However, since we did not observe increased expression of the genes in the SHC region following a 10 min exposure to novobiocin (Supplementary Figure S6D), we speculate that their activation requires not only a supercoiling imbalance, but also the activity of other factors.

Transcription of genes in the SHC region at constitutive topA-down- or upregulation (based on RNA-Seq data).

The SHC genes include those encoding a putative two-component system (sco4667 and sco4668) genes organized in a single operon and additional two putative transcriptional regulator-encoding genes positioned upstream (sco4671 and sco4673). The cluster also encompasses genes whose products may contribute to DNA/RNA transactions, including: putative DNA helicase (sco4685), transposase (sco4698), and a homolog of E. coli Rhs protein (rearrangement hotspot protein), which putatively contains the C-terminal toxin domain and is involved in bacterial intercellular competition (Koskiniemi et al., 2013). When we compared the genes in this region to other Streptomyces species, we found the synteny was not strong, perhaps suggesting that some of these genes may have been acquired through horizontal transfer (Hindra et al., 2014). However, the AT content of the SHC region was not significantly different from the rest of S. coelicolor chromosome (Supplementary Figure S7). On the other hand, it has been shown that mobile elements tend to be sensitive to changes in chromosome topology (Lodge and Berg, 1990). Indeed, among 552 genes affected by TopA depletion conditions, we found 16 genes (2.9% of SSGs) encoded putative transposable elements (which constitute only 0.7% of the S. coelicolor genome; see the complete list in the Supplementary Tables S2–S4). Why these mobile elements, and more broadly, the genes in the SHC, are specifically subject to topological control, remains an interesting question to address.

In summary, as in other bacteria, in S. coelicolor chromosome, there are supercoiling sensitive domains that could be detected. They may constitute the regions of chromosomes particularly prone to changes of supercoiling and/or may be regulated by supercoiling sensitive transcriptional regulators.

Long-Term Topological Imbalance Affects the Transcription of Genes Encoding Topology-Controlling ProteinsSince topological homeostasis could not be restored in the TopA-depleted S. coelicolor PS04 strain, either by induction of the topA promoter or by gyrBA silencing (Szafran et al., 2016), we predicted that the resulting increase in DNA supercoiling may promote changes in the level of NAPs to compensate for the changes in chromosome topology. According to previous reports (Bradshaw et al., 2013) some NAPs such as HupA are present at high levels during wild type S. coelicolor vegetative growth. Based on RNA-Seq data we confirmed that under standard growth conditions, many NAP genes, including hupA, ssbA, lsr2, sihf, and hupS were highly transcribed (Figure 4A), while other sporulation and/or stress responsive genes like dpsB, dpsC, and ssbB, were poorly transcribed. Next, we assessed the impact of the chromosome supercoiling changes on the expression of these genes.FIGURE 4The topological imbalance-induced transcriptional response of the genes encoding chromosome organizing proteins. (A) RNA-Seq based expression of the genes encoding DNA-organizing proteins (gray) and topoisomerases (black) in the wild type S. coelicolor strain normalized by the upper quartile. The error bars correspond to standard deviations calculated for four independent biological replicates. (B) RNA-Seq analysis of the relative transcriptional response of genes encoding DNA-organizing proteins under topA-downregulation conditions. The expression level of particular genes in the wild type strain was estimated as 1. (C) The relative transcription of selected genes in the TopA-depleted and control wild type strains calculated using RT-qPCR analysis. (D) RT-qPCR comparison of non-uniform topA gene transcription under different concentrations of topA inducer using oligonucleotide pairs specific for particular transcript regions. The transcript enrichment of particular regions was normalized by the expression of the hrdB gene which served as the endogenous control and compared to the wild type strain, in which transcription was estimated as 1. The amplified regions of the topA gene (1 – white; 2 – gray, and 3 – black) are shown above.Following TopA-depletion (conditions of increased negative supercoiling), we observed transcript levels of DNA-organizing proteins to be statistically different (p < 0.05) (Figure 4B). Of the NAPs affected by changes in chromosome topology, hupA and hupS were amongst the most impacted. Notably, however, the transcriptional effects were in opposite directions: hupA transcript levels increased to 1.44 relative to wild type conditions, while hupS transcript levels decreased to 0.76 compared with wild type (Figure 4B). To assess whether these effects were growth medium-specific, and to confirm the supercoiling dependence of these two HU-encoding genes, we grew the wild type and the PS04 strains in the 79 medium, and using RT-qPCR compared their expression under a range of topA induction conditions. As before, we observed hupA upregulation and hupS downregulation following TopA depletion if when compared to the wild type levels (Figure 4C). Previous work has suggested that HupS functions primarily during S. coelicolor sporulation (Salerno et al., 2009); however, both HU homologs, HupS and HupA, were shown to be present during S. coelicolor vegetative growth, although their levels are unequal (Bradshaw et al., 2013) (our transcriptional analysis showed that hupA transcript levels are over 14 times higher than the hupS transcripts). Our observations suggest that both S. coelicolor HU homologs contribute to the adjustment to altered supercoiling conditions. In E. coli two HU homologs, HupA and HupB form homo- and heterodimers at different growth stages, with the heterodimer having a crucial role when the culture reaches stationary phase (Claret and Rouviere-Yaniv, 1997). Furthermore, DNA supercoiling influences hupB transcription in mycobacteria, and its product affects TopA activity (Ghosh et al., 2014; Guha et al., 2018), suggesting that cooperation interactions between the HU proteins and TopA may be conserved in bacteria.In addition to the effects on hupA and hupS, increased chromosome supercoiling also profoundly affected the transcription (based on RNA-Seq data) of dpsA (decreased to 0.53 of wild type levels), smc (increased to 1.53 of wild type levels) (Figure 4B) and two genes (parC and parE) encoding topoisomerase IV subunits (increased to 1.86 and 1.67 of wild type levels, respectively). Using RT-qPCR, we confirmed upregulation of parC, parE, and smc genes. Similar observations have been made in E. coli, where increases in negative supercoiling result in the induction of mukBEF expression (Peter et al., 2004), where these genes encode homologs of SMC. Additionally, it is conceivable, that upregulating the expression of parC and parE (topoisomerase IV) may partially complement the effects of TopA depletion. However, the changes in dpsA transcription were not confirmed using RT-qPCR in cultures grown in different medium than used for our RNA-Seq experiment (Figure 4B,C). Since we did not observe any difference in DNA supercoiling (based on reporter plasmid assays, Supplementary Figure S4) in cultures growing in 79 or YEME/TSB medium, a possible explanation for this observation could be that dpsA transcription is affected by DNA supercoiling but only under osmotic stress conditions (in the YEME/TSB medium used in RNA-Seq experiment but not in 79 medium used for the RT-qPCR experiments) (Facey et al., 2009). Whether and how the changes in expression of genes encoding DNA organizing proteins influence chromosome topology have yet to be determined.Next, we assessed the effect of elevated topA expression (15–25× higher transcript levels compared with levels in the wild type strain). As mentioned, these increased topA transcript levels were not correlated with increased TopA protein whose levels were elevated only 22% in comparison with the wild type strain (Szafran et al., 2016; Figure 1 and Supplementary Figure S5A), thus only slightly affecting DNA supercoiling. Based on that observation, we speculated that TopA may be regulated posttranscriptionally and/or posttranslationally (e.g., may be subject to transcript processing or specific protein degradation). Indeed, while topA transcript levels were uniformly distributed along the 2858 bp topA gene in the wild type and the topA-downregulation strain), topA induction considerably increased the fraction of reads mapping to the beginning (region 1) but not to middle (region 2) or end (region 3) of the topA gene (Supplementary Figure S8). The non-uniform topA transcript levels were confirmed using RT-qPCR with specific oligonucleotides complementary to distinct fragments of the topA gene (Figure 4D). Interestingly, in the closely related M. smegmatis, gyrase binds within the topA promoter region, preventing its transcription (Ahmed et al., 2015), thus, similar mechanism of topA regulation in Streptomyces cannot be excluded. On the other hand, the possibility that topA transcripts are also subject to posttranscriptional processing and/or mRNA degradation could also explain the non-uniform detection of the particular topA transcript regions.In summary, the increased level of negative DNA supercoiling induces the coordinated modification of topoisomerases and NAPs level, presumably in an attempt to counteract the topological imbalance. Among them, the HupA protein may play a crucial role in maintaining nucleoid architecture under TopA depletion; with potential contributions from topoisomerase IV (ParCE) and downregulation of HupS – the other HU homolog present in Streptomyces. Intriguingly, topA induction had little effect on global DNA supercoiling, as Streptomyces appear to have mechanisms in place to ensure TopA levels are maintained below a certain threshold, through possible autoregulatory control.

Long-Term Topological Imbalance Affects the Transcription of Genes Encoding Topology-Controlling Proteins

Since topological homeostasis could not be restored in the TopA-depleted S. coelicolor PS04 strain, either by induction of the topA promoter or by gyrBA silencing (Szafran et al., 2016), we predicted that the resulting increase in DNA supercoiling may promote changes in the level of NAPs to compensate for the changes in chromosome topology. According to previous reports (Bradshaw et al., 2013) some NAPs such as HupA are present at high levels during wild type S. coelicolor vegetative growth. Based on RNA-Seq data we confirmed that under standard growth conditions, many NAP genes, including hupA, ssbA, lsr2, sihf, and hupS were highly transcribed (Figure 4A), while other sporulation and/or stress responsive genes like dpsB, dpsC, and ssbB, were poorly transcribed. Next, we assessed the impact of the chromosome supercoiling changes on the expression of these genes.

The topological imbalance-induced transcriptional response of the genes encoding chromosome organizing proteins. (A) RNA-Seq based expression of the genes encoding DNA-organizing proteins (gray) and topoisomerases (black) in the wild type S. coelicolor strain normalized by the upper quartile. The error bars correspond to standard deviations calculated for four independent biological replicates. (B) RNA-Seq analysis of the relative transcriptional response of genes encoding DNA-organizing proteins under topA-downregulation conditions. The expression level of particular genes in the wild type strain was estimated as 1. (C) The relative transcription of selected genes in the TopA-depleted and control wild type strains calculated using RT-qPCR analysis. (D) RT-qPCR comparison of non-uniform topA gene transcription under different concentrations of topA inducer using oligonucleotide pairs specific for particular transcript regions. The transcript enrichment of particular regions was normalized by the expression of the hrdB gene which served as the endogenous control and compared to the wild type strain, in which transcription was estimated as 1. The amplified regions of the topA gene (1 – white; 2 – gray, and 3 – black) are shown above.

Following TopA-depletion (conditions of increased negative supercoiling), we observed transcript levels of DNA-organizing proteins to be statistically different (p < 0.05) (Figure 4B). Of the NAPs affected by changes in chromosome topology, hupA and hupS were amongst the most impacted. Notably, however, the transcriptional effects were in opposite directions: hupA transcript levels increased to 1.44 relative to wild type conditions, while hupS transcript levels decreased to 0.76 compared with wild type (Figure 4B). To assess whether these effects were growth medium-specific, and to confirm the supercoiling dependence of these two HU-encoding genes, we grew the wild type and the PS04 strains in the 79 medium, and using RT-qPCR compared their expression under a range of topA induction conditions. As before, we observed hupA upregulation and hupS downregulation following TopA depletion if when compared to the wild type levels (Figure 4C). Previous work has suggested that HupS functions primarily during S. coelicolor sporulation (Salerno et al., 2009); however, both HU homologs, HupS and HupA, were shown to be present during S. coelicolor vegetative growth, although their levels are unequal (Bradshaw et al., 2013) (our transcriptional analysis showed that hupA transcript levels are over 14 times higher than the hupS transcripts). Our observations suggest that both S. coelicolor HU homologs contribute to the adjustment to altered supercoiling conditions. In E. coli two HU homologs, HupA and HupB form homo- and heterodimers at different growth stages, with the heterodimer having a crucial role when the culture reaches stationary phase (Claret and Rouviere-Yaniv, 1997). Furthermore, DNA supercoiling influences hupB transcription in mycobacteria, and its product affects TopA activity (Ghosh et al., 2014; Guha et al., 2018), suggesting that cooperation interactions between the HU proteins and TopA may be conserved in bacteria.

In addition to the effects on hupA and hupS, increased chromosome supercoiling also profoundly affected the transcription (based on RNA-Seq data) of dpsA (decreased to 0.53 of wild type levels), smc (increased to 1.53 of wild type levels) (Figure 4B) and two genes (parC and parE) encoding topoisomerase IV subunits (increased to 1.86 and 1.67 of wild type levels, respectively). Using RT-qPCR, we confirmed upregulation of parC, parE, and smc genes. Similar observations have been made in E. coli, where increases in negative supercoiling result in the induction of mukBEF expression (Peter et al., 2004), where these genes encode homologs of SMC. Additionally, it is conceivable, that upregulating the expression of parC and parE (topoisomerase IV) may partially complement the effects of TopA depletion. However, the changes in dpsA transcription were not confirmed using RT-qPCR in cultures grown in different medium than used for our RNA-Seq experiment (Figure 4B,C). Since we did not observe any difference in DNA supercoiling (based on reporter plasmid assays, Supplementary Figure S4) in cultures growing in 79 or YEME/TSB medium, a possible explanation for this observation could be that dpsA transcription is affected by DNA supercoiling but only under osmotic stress conditions (in the YEME/TSB medium used in RNA-Seq experiment but not in 79 medium used for the RT-qPCR experiments) (Facey et al., 2009). Whether and how the changes in expression of genes encoding DNA organizing proteins influence chromosome topology have yet to be determined.

Next, we assessed the effect of elevated topA expression (15–25× higher transcript levels compared with levels in the wild type strain). As mentioned, these increased topA transcript levels were not correlated with increased TopA protein whose levels were elevated only 22% in comparison with the wild type strain (Szafran et al., 2016; Figure 1 and Supplementary Figure S5A), thus only slightly affecting DNA supercoiling. Based on that observation, we speculated that TopA may be regulated posttranscriptionally and/or posttranslationally (e.g., may be subject to transcript processing or specific protein degradation). Indeed, while topA transcript levels were uniformly distributed along the 2858 bp topA gene in the wild type and the topA-downregulation strain), topA induction considerably increased the fraction of reads mapping to the beginning (region 1) but not to middle (region 2) or end (region 3) of the topA gene (Supplementary Figure S8). The non-uniform topA transcript levels were confirmed using RT-qPCR with specific oligonucleotides complementary to distinct fragments of the topA gene (Figure 4D). Interestingly, in the closely related M. smegmatis, gyrase binds within the topA promoter region, preventing its transcription (Ahmed et al., 2015), thus, similar mechanism of topA regulation in Streptomyces cannot be excluded. On the other hand, the possibility that topA transcripts are also subject to posttranscriptional processing and/or mRNA degradation could also explain the non-uniform detection of the particular topA transcript regions.

In summary, the increased level of negative DNA supercoiling induces the coordinated modification of topoisomerases and NAPs level, presumably in an attempt to counteract the topological imbalance. Among them, the HupA protein may play a crucial role in maintaining nucleoid architecture under TopA depletion; with potential contributions from topoisomerase IV (ParCE) and downregulation of HupS – the other HU homolog present in Streptomyces. Intriguingly, topA induction had little effect on global DNA supercoiling, as Streptomyces appear to have mechanisms in place to ensure TopA levels are maintained below a certain threshold, through possible autoregulatory control.

Alternating topA Transcript Levels Stimulates the Expression of DNA Damage Response GenesChromosome regulation stemming from gyrase inhibition with novobiocin had an immediate impact on 1.5% of the genes in the S. coelicolor chromosome. Conversely, the long-term severe alteration of topA levels during S. coelicolor growth resulted in more prominent changes in global genes transcription affecting up to 7% of the S. coelicolor genome. Interestingly, among the genes affected by severe depletion or modest increase in TopA levels we identified genes whose transcription was modified under both conditions.Among the genes sensitive to any alteration of TopA levels, 14 regulatory protein-encoding genes were identified (Table 4), including the four regulators in the SHC region [sco4667 (sensor kinase), sco4668 (regulatory protein), and putative transcriptional regulators (sco4671 and sco4673)]. Outside of this region was sco5803, which encodes the LexA repressor that controls the DNA damage response (highly induced by topA-up- and downregulation). In E. coli, the LexA-regulon encompasses at least 31 genes, including recA and lexA (the latter being negatively autoregulated), uvrA, ftsK, polB, dinF, and dnaE2, alongside others involved in the DNA damage response (Fernandez De Henestrosa et al., 2000). We found that the transcription of recA, recX, dnaE2, dinP, or uvrA genes, which presumably belong to the LexA-regulon in S. coelicolor, based on the predicted binding consensus sequence (Novichkov et al., 2013), were similarly affected by changes in chromosome supercoiling. This implies that the entire LexA regulon was impacted (Tables 5, 6), although the RT-qPCR analysis indicated that in 79 medium, this induction was less pronounced (Supplementary Figure S9).Table 4The regulators encoding genes affected by both constitutive topA-downregulation and topA-upregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).GeneLocusNormalized expression in the wild type∗Log2-fold change topA-downregulationLog2-fold change topA-upregulationDescriptionRegulatorssco3435141.721.58Transcriptional regulatorsco42142429-6.55-6.60AbaA-like regulatory proteinxlnRsco4215820-2.05-2.20GntR family transcriptional regulatorsco4671212.312.08LysR family transcriptional regulatorsco4673563.202.35DeoR family transcriptional regulatorsco56562804-1.80-2.15Transcriptional regulatorlexAsco58032402.041.62LexA repressornsdBsco7252118-1.67-2.80Regulatory proteinTCSssco4667129.758.93Two-component system sensor kinasesco4668677.516.59Two-component system response regulatorProteasesLonsco5285101.891.85ATP-dependent proteaseGNATssco1624132.512.74Acetyltransferasesco2379122.141.87AcetyltransferasedesCsco2784285-2.32-1.83Acetyltransferase∗Normalization by the upper quartile of gene expression.Table 5DNA processing and DNA repair genes impacted by constitutive topA-downregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).GeneLocusNormalized expression in the wild type∗Log2-fold changeDescriptiondnaE2sco173955.22DNA polymerase III subunit alphadinPsco173864.84DNA polymerase IVsco1767291.92DNA hydrolasesco1827721.51DNA polymerase III subunit epsilonuvrAsco1958691.89Exconuclease ABC subunit Asco1969671.88DNA-methyltransferasesco2863181.98Helicasesco3434272.01DNA polymerase Isco4685163.43DEAD/DEAH box helicasexseAsco5056691.51Exodeoxyribonuclease VII large subunitsco5183121.70ATP-dependent DNA helicasesco5184111.54ATP-dependent DNA helicaserecXsco57701363.15Recombination regulator RecXrecAsco57694262.76Recombinase AlexAsco58032402.04LexA repressorsco592091.53DEAD/DEAH box helicasesco671923.86UvrA-like ABC transportersco752272.32DNA ligase∗Normalization by the upper quartile of gene expression.Table 6DNA processing and DNA repair genes impacted by constitutive topA-upregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).GeneLocusNormalized expression in the wild type∗Log2-fold changeDescriptiondinPsco173865.04DNA polymerase IVdnaE2sco173954.96DNA polymerase III subunit alphasco17671221.81DNA hydrolaseuvrAsco1958691.98Excinuclease ABC subunit Asco3434272.29DNA polymerase Isco379821-2.81Chromosome condensation proteinsco4685162.67DEAD/DEAH box helicasesco5761201.54ATP-dependent DNA helicaserecAsco57694262.34Recombinase ArecXsco57701362.81Recombination regulator RecXlexAsco58032401.62LexA repressorsco752272.05DNA ligase∗Normalization by the upper quartile of gene expression.The identification of putative LexA-dependent genes among the genes upregulated when topA transcription was altered may suggest that these conditions trigger the LexA-dependent DNA repair pathway. However, the mechanism of LexA regulon induction by altered chromosome supercoiling remains speculative. We assume that in Streptomyces, as in other bacteria, LexA activity is controlled by RecA, which in the presence of ssDNA and in an ATP-dependent manner stimulates the self-cleavage of LexA, resulting in derepression of the LexA regulon. It is conceivable that changes in chromosome topology alter LexA binding to DNA, or that the formation of single-stranded DNA patches upon depleting TopA (Parsa et al., 2012) promote RecA-dependent cleavage of LexA and de-repression of LexA-regulated genes. However, the latter possibility could not account for the release of LexA repression during chromosome relaxation and further studies are required to understand the mechanism underlying LexA regulation in response to supercoiling changes. Since DNA damage is frequently accompanied by changes in local chromosome architecture, it is possible that changes in chromosome supercoiling may be sensed as a sign of DNA damage, leading to the induction of DNA repair genes. Interestingly, in Corynebacterium glutamicum, an actinobacterial relative of Streptomyces, induction of the DNA repair pathway was associated with the inhibition of cell division, which could explain the inhibition of sporulation seen for the S. coelicolor TopA-depleted strain (Ogino et al., 2008).Since we did not identify LexA-regulon members among novobiocin-sensitive genes, we speculate that the DNA repair response may be a result of long-term exposure to topological imbalance. Thus, topological imbalance, if cannot be compensated by response pathways, could serve as a marker of DNA damage.

Alternating topA Transcript Levels Stimulates the Expression of DNA Damage Response Genes

Chromosome regulation stemming from gyrase inhibition with novobiocin had an immediate impact on 1.5% of the genes in the S. coelicolor chromosome. Conversely, the long-term severe alteration of topA levels during S. coelicolor growth resulted in more prominent changes in global genes transcription affecting up to 7% of the S. coelicolor genome. Interestingly, among the genes affected by severe depletion or modest increase in TopA levels we identified genes whose transcription was modified under both conditions.

Among the genes sensitive to any alteration of TopA levels, 14 regulatory protein-encoding genes were identified (Table 4), including the four regulators in the SHC region [sco4667 (sensor kinase), sco4668 (regulatory protein), and putative transcriptional regulators (sco4671 and sco4673)]. Outside of this region was sco5803, which encodes the LexA repressor that controls the DNA damage response (highly induced by topA-up- and downregulation). In E. coli, the LexA-regulon encompasses at least 31 genes, including recA and lexA (the latter being negatively autoregulated), uvrA, ftsK, polB, dinF, and dnaE2, alongside others involved in the DNA damage response (Fernandez De Henestrosa et al., 2000). We found that the transcription of recA, recX, dnaE2, dinP, or uvrA genes, which presumably belong to the LexA-regulon in S. coelicolor, based on the predicted binding consensus sequence (Novichkov et al., 2013), were similarly affected by changes in chromosome supercoiling. This implies that the entire LexA regulon was impacted (Tables 5, 6), although the RT-qPCR analysis indicated that in 79 medium, this induction was less pronounced (Supplementary Figure S9).

The regulators encoding genes affected by both constitutive topA-downregulation and topA-upregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).

DNA processing and DNA repair genes impacted by constitutive topA-downregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).

DNA processing and DNA repair genes impacted by constitutive topA-upregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).

The identification of putative LexA-dependent genes among the genes upregulated when topA transcription was altered may suggest that these conditions trigger the LexA-dependent DNA repair pathway. However, the mechanism of LexA regulon induction by altered chromosome supercoiling remains speculative. We assume that in Streptomyces, as in other bacteria, LexA activity is controlled by RecA, which in the presence of ssDNA and in an ATP-dependent manner stimulates the self-cleavage of LexA, resulting in derepression of the LexA regulon. It is conceivable that changes in chromosome topology alter LexA binding to DNA, or that the formation of single-stranded DNA patches upon depleting TopA (Parsa et al., 2012) promote RecA-dependent cleavage of LexA and de-repression of LexA-regulated genes. However, the latter possibility could not account for the release of LexA repression during chromosome relaxation and further studies are required to understand the mechanism underlying LexA regulation in response to supercoiling changes. Since DNA damage is frequently accompanied by changes in local chromosome architecture, it is possible that changes in chromosome supercoiling may be sensed as a sign of DNA damage, leading to the induction of DNA repair genes. Interestingly, in Corynebacterium glutamicum, an actinobacterial relative of Streptomyces, induction of the DNA repair pathway was associated with the inhibition of cell division, which could explain the inhibition of sporulation seen for the S. coelicolor TopA-depleted strain (Ogino et al., 2008).

Since we did not identify LexA-regulon members among novobiocin-sensitive genes, we speculate that the DNA repair response may be a result of long-term exposure to topological imbalance. Thus, topological imbalance, if cannot be compensated by response pathways, could serve as a marker of DNA damage.

Developmental Transcriptional Regulators Are Sensitive to Specific Supercoiling ConditionsAlthough topA downregulation and topA-upregulation induced a common transcriptional response, a significant fraction of the affected genes responded specifically to particular supercoiling conditions (Figure 2C). We identified 12 SSGs-encoding regulatory proteins that were affected specifically by topA-upregulation (Table 7). These numbers are similar to those induced by novobiocin treatment, but surprisingly, the sets of genes identified in both experiments did not overlap (Supplementary Table S5). Amongst the many uncharacterized genes that were sensitive to topA-upregulation, developmental regulators such as sigF (upregulated) and nsdA (downregulated) were identified; however, these changes in expression were not associated with any obvious phenotype for the topA up-regulated strain.Table 7Regulatory protein-encoding genes impacted by constitutive topA-upregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).GeneLocusNormalized expression in the wild type∗Log2-fold changeDescriptionRegulatorssco014033.84MerR family transcriptional regulatorsco2865154-2.27Regulatory proteinsigFsco403542.70RNA polymerase sigma factorsco410262.42MerR family transcriptional regulatorsco5025227.71Transcriptional regulatorsco5437412.65MerR family transcriptional regulatornsdAsco558267-1.67Regulatorsco596228.10Transcriptional regulatorsco652034-1.63RNA polymerase sigma factorsco753064.61Regulatory proteinsco769829.13MerR family transcriptional regulatorTCSscutSsco5863116-1.57Two-component sensor kinase∗Normalization by the upper quartile of gene expression.The increased chromosome supercoiling resulting from TopA depletion specifically influenced a substantial number of genes encoding regulatory proteins. Among 35 of the TopA depletion-sensitive genes, we identified 18 putative transcriptional regulators, including seven genes encoding TCSs (kinases and/or their putative phosphorylation targets), six genes encoding putative acetyltransferases (GNATs family) and four genes encoding subunits of regulatory Clp proteases (Table 8), alongside the sporulation-specific sigma factor encoding whiG.Table 8Regulatory genes affected by topA-downregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).GeneLocusNormalized expression in the wild type∗Log2-fold changeDescriptionRegulatorssco013256-2.49Transcriptional regulatorsco02041059-1.88LuxR family transcriptional regulatorsco1104561.74TetR family transcriptional regulatorsco173662.84MarR family transcriptional regulatorsco2223541.72TetR family transcriptional regulatorsco4059221.61Transcriptional regulatorsco437591.58MarR family regulatory proteinsco5323521.57Transcriptional regulator AsnCsco541881.81Transcriptional regulatorwhiGsco562164.40RNA polymerase sigma factor WhiGsco5840112.24Transcriptional regulatorsco6299351.57TetR family transcriptional regulatorsco6664142.03Transcriptional regulatorsco7014221.75LacI family transcriptional regulatorsco70421141.60MarR family transcriptional regulatorsco7270151.62Transcriptional regulatorsco7639451.84MarR family regulatory proteinsco7727100-1.56MarR family regulatory proteinTCSsohkAsco1596311.77Two-component sensor kinaseabrA2sco174571.89Two-component system response regulatorsco314462.42Two-component system response regulatorsco3818179-1.96Two-component system response transcriptional regulatorsco436282.29Two component system sensor kinasesco5455151.87Two-component system response regulatorsco7649151.65Two-component system sensor kinaseProteasesclpP2sco26182584-1.77ATP-dependent Clp protease proteolytic subunitclpP1sco26193091-1.61ATP-dependent Clp protease proteolytic subunitclpAsco640829-1.69Clp protease ATP-binding subunitclpP4sco728081.91ATP-dependent Clp protease proteolytic subunit 2GNATssco1563461.55Acetyltransferasesco222733-2.24Acetyltransferasesco2876142.00Acetyltransferasesco405093-2.15Acetyltransferasesco522882.09Acetyltransferasesco744714.70Acetyltransferase∗Normalization by the upper quartile of gene expression.The gene encoding the sigma factor WhiG was one of the most significantly upregulated regulatory genes under TopA depletion based on RNA-Seq results (over 21-fold induction; Table 8 and Figure 5A). RT-qPCR confirmed that whiG transcription was stimulated by increased DNA supercoiling and decreased to wild type levels upon induction of normal topA transcription (Figure 5B); although the calculated fold-change was lower, probably due to detection of a small amount of whiG transcript in the wild type strain growing in 79 medium. WhiG is typically expressed at low levels during vegetative growth and is involved in the regulatory cascade that governs Streptomyces sporulation (Kelemen et al., 1996; Elliot et al., 2001). Since WhiG directs the expression of the whiI and whiH genes, but does not affect the transcription of other whi-family regulators, such as whiA and whiB (Mendez and Chater, 1987; Chater et al., 1989; Ryding et al., 1998), we analyzed the transcriptional response of these genes to increased DNA supercoiling. As expected, the transcription of both whiI and whiH was elevated following TopA depletion (although they were not identified in the initial screening due to below-threshold q-values, however, p < 0.001 was still statistically relevant), suggesting the induction of a WhiG-dependent regulatory cascade by changes in chromosome supercoiling (Figure 5A). In S. coelicolor whiG was shown to belong to the BldD-regulon, which also encompasses bldN, bldM, and whiB (Elliot et al., 2001; den Hengst et al., 2010). In fact, in S. venezuelae, the whiG gene was shown to be directly repressed by BldD, the master regulator that binds to DNA upon association with c-di-GMP (Tschowri et al., 2014). This prompted us to test whether increased DNA supercoiling influenced the transcription of other BldD-dependent genes. Indeed, upon TopA depletion, we observed high upregulation of three BldD targets: bldM, bldN, and whiG (Figure 5A). The mechanism of supercoiling-dependent upregulation of the BldD-regulon may, similar to that proposed for LexA, result from decreased BldD DNA-binding affinity during higher DNA supercoiling. Thus, alleviating BldD binding during times of increased DNA supercoiling could lead to increased target gene transcription.FIGURE 5Topoisomerase I depletion-induced changes in the transcription of genes involved in sporulation regulatory cascades. (A) RNA-Seq based expression of selected genes encoding for whi and bld regulatory cascades genes normalized by the upper quartile in the wild type strain (black) and the TopA-depleted strain (red). The genes belonging to the BldD regulon are marked with arrows. The statistically significant changes in transcript levels were marked with asterisks (∗p-value <0.05; ∗∗∗p-value <0.001). p-Values were calculated based on RNA-Seq data analysis for four independent biological replicates for the wild type and topA-downregulated strains. (B) RT-qPCR analysis of whiG transcript levels at different concentrations of the topA gene inducer. Wild type levels of whiG transcription were set at 1, and standard deviation for wild type transcription is marked with a gray bar. The inset shows RT-qPCR analysis of whiG transcript levels in the TopA-depleted and the wild type strains, as well as the control strain that contained the pIJ6902 plasmid without the topA gene. The whiG transcript enrichment was normalized by the expression of the hrdB gene which served as the endogenous control and compared to the wild type strain in which transcription was estimated as 1.Remarkably, even though elevated supercoiling induced key sporulation genes including whiG, their induction did not lead to sporulation; in fact, the TopA-depleted strain fails to form spores (Szafran et al., 2013). This may indicate either a lack of additional regulators needed for differentiation, or the activity of other signaling pathways that prevent sporulation (e.g., the DNA damage response, as described for RecA/LexA above). The observed induction of sporulation cascades does, however, suggest that DNA topology may function as a global regulator that triggers sporulation cascades either in response to environmental stress or physiological conditions that induce increased DNA supercoiling.

Developmental Transcriptional Regulators Are Sensitive to Specific Supercoiling Conditions

Although topA downregulation and topA-upregulation induced a common transcriptional response, a significant fraction of the affected genes responded specifically to particular supercoiling conditions (Figure 2C). We identified 12 SSGs-encoding regulatory proteins that were affected specifically by topA-upregulation (Table 7). These numbers are similar to those induced by novobiocin treatment, but surprisingly, the sets of genes identified in both experiments did not overlap (Supplementary Table S5). Amongst the many uncharacterized genes that were sensitive to topA-upregulation, developmental regulators such as sigF (upregulated) and nsdA (downregulated) were identified; however, these changes in expression were not associated with any obvious phenotype for the topA up-regulated strain.

Regulatory protein-encoding genes impacted by constitutive topA-upregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).

The increased chromosome supercoiling resulting from TopA depletion specifically influenced a substantial number of genes encoding regulatory proteins. Among 35 of the TopA depletion-sensitive genes, we identified 18 putative transcriptional regulators, including seven genes encoding TCSs (kinases and/or their putative phosphorylation targets), six genes encoding putative acetyltransferases (GNATs family) and four genes encoding subunits of regulatory Clp proteases (Table 8), alongside the sporulation-specific sigma factor encoding whiG.

Regulatory genes affected by topA-downregulation (based on RNA-Seq data; log2-fold >1.5; p < 0.05).

The gene encoding the sigma factor WhiG was one of the most significantly upregulated regulatory genes under TopA depletion based on RNA-Seq results (over 21-fold induction; Table 8 and Figure 5A). RT-qPCR confirmed that whiG transcription was stimulated by increased DNA supercoiling and decreased to wild type levels upon induction of normal topA transcription (Figure 5B); although the calculated fold-change was lower, probably due to detection of a small amount of whiG transcript in the wild type strain growing in 79 medium. WhiG is typically expressed at low levels during vegetative growth and is involved in the regulatory cascade that governs Streptomyces sporulation (Kelemen et al., 1996; Elliot et al., 2001). Since WhiG directs the expression of the whiI and whiH genes, but does not affect the transcription of other whi-family regulators, such as whiA and whiB (Mendez and Chater, 1987; Chater et al., 1989; Ryding et al., 1998), we analyzed the transcriptional response of these genes to increased DNA supercoiling. As expected, the transcription of both whiI and whiH was elevated following TopA depletion (although they were not identified in the initial screening due to below-threshold q-values, however, p < 0.001 was still statistically relevant), suggesting the induction of a WhiG-dependent regulatory cascade by changes in chromosome supercoiling (Figure 5A). In S. coelicolor whiG was shown to belong to the BldD-regulon, which also encompasses bldN, bldM, and whiB (Elliot et al., 2001; den Hengst et al., 2010). In fact, in S. venezuelae, the whiG gene was shown to be directly repressed by BldD, the master regulator that binds to DNA upon association with c-di-GMP (Tschowri et al., 2014). This prompted us to test whether increased DNA supercoiling influenced the transcription of other BldD-dependent genes. Indeed, upon TopA depletion, we observed high upregulation of three BldD targets: bldM, bldN, and whiG (Figure 5A). The mechanism of supercoiling-dependent upregulation of the BldD-regulon may, similar to that proposed for LexA, result from decreased BldD DNA-binding affinity during higher DNA supercoiling. Thus, alleviating BldD binding during times of increased DNA supercoiling could lead to increased target gene transcription.

Topoisomerase I depletion-induced changes in the transcription of genes involved in sporulation regulatory cascades. (A) RNA-Seq based expression of selected genes encoding for whi and bld regulatory cascades genes normalized by the upper quartile in the wild type strain (black) and the TopA-depleted strain (red). The genes belonging to the BldD regulon are marked with arrows. The statistically significant changes in transcript levels were marked with asterisks (∗p-value <0.05; ∗∗∗p-value <0.001). p-Values were calculated based on RNA-Seq data analysis for four independent biological replicates for the wild type and topA-downregulated strains. (B) RT-qPCR analysis of whiG transcript levels at different concentrations of the topA gene inducer. Wild type levels of whiG transcription were set at 1, and standard deviation for wild type transcription is marked with a gray bar. The inset shows RT-qPCR analysis of whiG transcript levels in the TopA-depleted and the wild type strains, as well as the control strain that contained the pIJ6902 plasmid without the topA gene. The whiG transcript enrichment was normalized by the expression of the hrdB gene which served as the endogenous control and compared to the wild type strain in which transcription was estimated as 1.

Remarkably, even though elevated supercoiling induced key sporulation genes including whiG, their induction did not lead to sporulation; in fact, the TopA-depleted strain fails to form spores (Szafran et al., 2013). This may indicate either a lack of additional regulators needed for differentiation, or the activity of other signaling pathways that prevent sporulation (e.g., the DNA damage response, as described for RecA/LexA above). The observed induction of sporulation cascades does, however, suggest that DNA topology may function as a global regulator that triggers sporulation cascades either in response to environmental stress or physiological conditions that induce increased DNA supercoiling.

SummaryTopological imbalance has profound effects on gene transcription in all bacteria; however, different organisms evoke distinct responses (Gmuender et al., 2001; Peter et al., 2004; Ferrandiz et al., 2010). We showed here that in S. coelicolor supercoiling-sensitive genes are organized in discrete clusters, a feature that seems to be a conserved strategy for topological regulation among many bacteria. Supercoiling imbalance triggers the activation of genes involved in stress responses, including DNA repair pathway, transmembrane transporters and chaperonins, as well as genes involved in the production of secondary metabolites. Both increased and decreased DNA supercoiling appear to have been detected as signals of DNA damage, inducing DNA repair genes and oxidative stress response genes. The long-term response to supercoiling imbalance is based on the activation of the set of primary and downstream regulatory proteins. Sporulation is triggered under stress conditions in Streptomyces, and accordingly, we found that several differentiation-regulating genes are affected by DNA supercoiling. In general, topology-governed regulons are expected to rely on the supercoiling-dependent binding of various regulatory proteins to DNA.

Topological imbalance has profound effects on gene transcription in all bacteria; however, different organisms evoke distinct responses (Gmuender et al., 2001; Peter et al., 2004; Ferrandiz et al., 2010). We showed here that in S. coelicolor supercoiling-sensitive genes are organized in discrete clusters, a feature that seems to be a conserved strategy for topological regulation among many bacteria. Supercoiling imbalance triggers the activation of genes involved in stress responses, including DNA repair pathway, transmembrane transporters and chaperonins, as well as genes involved in the production of secondary metabolites. Both increased and decreased DNA supercoiling appear to have been detected as signals of DNA damage, inducing DNA repair genes and oxidative stress response genes. The long-term response to supercoiling imbalance is based on the activation of the set of primary and downstream regulatory proteins. Sporulation is triggered under stress conditions in Streptomyces, and accordingly, we found that several differentiation-regulating genes are affected by DNA supercoiling. In general, topology-governed regulons are expected to rely on the supercoiling-dependent binding of various regulatory proteins to DNA.

Data AvailabilityRaw data are available at ArrayExpress (EMBL-EBI); accession number E-MTAB-8071 as well as at Gene Expression Omnibus (NCBI); accession number GSE132702.

Data Availability

Raw data are available at ArrayExpress (EMBL-EBI); accession number E-MTAB-8071 as well as at Gene Expression Omnibus (NCBI); accession number GSE132702.

Conflict of Interest StatementThe authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

Conflict of Interest Statement

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.
