---
pmid: '37252361'
title: Semisynthetic LC3 Probes for Autophagy Pathways Reveal a Noncanonical LC3 Interacting
  Region Motif Crucial for the Enzymatic Activity of Human ATG3.
authors:
- Farnung J
- Muhar M
- Liang JR
- Tolmachova KA
- Benoit RM
- Corn JE
- Bode JW
journal: ACS Cent Sci
year: '2023'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC10214526
doi: 10.1021/acscentsci.3c00009
---

# Semisynthetic LC3 Probes for Autophagy Pathways Reveal a Noncanonical LC3 Interacting Region Motif Crucial for the Enzymatic Activity of Human ATG3.
**Authors:** Farnung J, Muhar M, Liang JR, Tolmachova KA, Benoit RM, Corn JE, Bode JW
**Journal:** ACS Cent Sci (2023)
**DOI:** [10.1021/acscentsci.3c00009](https://doi.org/10.1021/acscentsci.3c00009)
**PMC:** [PMC10214526](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10214526/)

## Abstract

1. ACS Cent Sci. 2023 Apr 27;9(5):1025-1034. doi: 10.1021/acscentsci.3c00009. 
eCollection 2023 May 24.

Semisynthetic LC3 Probes for Autophagy Pathways Reveal a Noncanonical LC3 
Interacting Region Motif Crucial for the Enzymatic Activity of Human ATG3.

Farnung J(1), Muhar M(2), Liang JR(2), Tolmachova KA(1), Benoit RM(3), Corn 
JE(2), Bode JW(1).

Author information:
(1)Laboratory for Organic Chemistry, Department of Chemistry and Applied 
Biosciences ETH Zürich, CH-8093 Zürich, Switzerland.
(2)Institute of Molecular Health Sciences, Department of Biology ETH Zürich, 
CH-8093 Zürich, Switzerland.
(3)Laboratory of Nanoscale Biology, Division of Biology and Chemistry, Paul 
Scherrer Institute, 5232 Villigen PSI, Switzerland.

Macroautophagy is one of two major degradation systems in eukaryotic cells. 
Regulation and control of autophagy are often achieved through the presence of 
short peptide sequences called LC3 interacting regions (LIR) in 
autophagy-involved proteins. Using a combination of new protein-derived 
activity-based probes prepared from recombinant LC3 proteins, along with protein 
modeling and X-ray crystallography of the ATG3-LIR peptide complex, we 
identified a noncanonical LIR motif in the human E2 enzyme responsible for LC3 
lipidation, ATG3. The LIR motif is present in the flexible region of ATG3 and 
adopts an uncommon β-sheet structure binding to the backside of LC3. We show 
that the β-sheet conformation is crucial for its interaction with LC3 and used 
this insight to design synthetic macrocyclic peptide-binders to ATG3. 
CRISPR-enabled in cellulo studies provide evidence that LIRATG3 is required for 
LC3 lipidation and ATG3∼LC3 thioester formation. Removal of LIRATG3 negatively 
impacts the rate of thioester transfer from ATG7 to ATG3.

© 2023 The Authors. Published by American Chemical Society.

DOI: 10.1021/acscentsci.3c00009
PMCID: PMC10214526
PMID: 37252361

Conflict of interest statement: The authors declare no competing financial 
interest.

## Full Text

IntroductionMacroautophagy (herein referred to as
autophagy) is a major catabolic
process in eukaryotic cells,1 responsible
for the bulk degradation of various cellular components such as proteins,2 cellular compartments,3 and pathogens.4 Akin to the ubiquitin
proteasome system, two families of small protein modifiers are crucial
regulatory elements of autophagy. Proteins of the LC3 or GABARAP families
are conjugated via their C-terminal glycine residue to phosphatidylethanolamine-containing
lipids.5 This process is catalyzed by an
intricate enzymatic cascade in which ATG7 functions as an activating
E1 enzyme using ATP to form a thioester with LC3/GABARAP and transferring
them to an E2 enzyme, ATG3, via a trans-thioesterification reaction.
ATG3 performs the lipidation of LC3/GABARAP in conjunction with an
E3-like enzyme complex of isopeptide-linked ATG5-ATG12. Membrane tethering
of LC3/GABARAP is crucial for membrane expansion of autophagosomes
to engulf the autophagic cargo and for eventual lysosome fusion.6−9 However, the mechanistic details of LC3/GABARAP-lipidation by ATG3
remain enigmatic.A conserved feature of the autophagy pathway
is the recurring presence
of short peptide motifs called LC3 interacting regions (LIR) in proteins
associated with autophagy.10 Proteins containing
LIR motifs are recruited to LC3/GABARAP through hydrophobic interactions.
The core sequence of LIR motifs, ΦxxΨ, is characterized
by the presence of an aromatic residue (Φ)—Trp, Phe,
and Tyr—followed by two variable positions and an aliphatic,
hydrophobic amino acid (Ψ), generally Ile, Leu, or Val.11 The LIR motif adopts an extended conformation
forming an intermolecular β-sheet with β2 of LC3/GABARAP.
A variety of LIR motifs have been identified in selective autophagy
receptors, which employ them to recruit cargos to expanding autophagosomes.2 In addition, these motifs can also be found in
proteins involved in the attachment of LC3/GABARAP to membranes such
as ATG4,12 a protease required for processing
of proLC3/proGABARAP and delipidation of LC3/GABARAP.Few chemical
probes for autophagy have been developed.13−15 The majority
are inhibitors of autophagy proteins functioning upstream
of the lipidation cascade, such as wortmannin, a PI3K inhibitor that
abrogates localization of lipidation enzymes to expanding autophagosomes.16,17 Despite the similarity to the ubiquitin proteasome pathway, few
chemical probes exist for LC3/GABARAP lipidation.18,19 Hemelaar et al. reported the synthesis of LC3/GABARAP activity-based
probes by direct aminolysis and employed these probes to identify
ATG4 as the processing protease of proLC3/proGABARAP.20 However, access to these probes by direct aminolysis is
generally hampered by harsh reaction conditions and inefficient conversion.Herein, we report the facile preparation of GABARAP and LC3A activity-based
probes (ABPs) using a recently established hydrazide acylation protocol.21,22 Access to these ABPs was critical for the identification of an unknown
noncanonical LIR motif embedded in an unusual β-sheet conformation
in human ATG3. Further investigation of this motif with macrocyclic
peptide binders, X-ray crystallography, and CRISPR-enabled in cellulo studies revealed that this LIR motif is crucial
for the enzymatic function of ATG3.

Introduction

Macroautophagy (herein referred to as
autophagy) is a major catabolic
process in eukaryotic cells,1 responsible
for the bulk degradation of various cellular components such as proteins,2 cellular compartments,3 and pathogens.4 Akin to the ubiquitin
proteasome system, two families of small protein modifiers are crucial
regulatory elements of autophagy. Proteins of the LC3 or GABARAP families
are conjugated via their C-terminal glycine residue to phosphatidylethanolamine-containing
lipids.5 This process is catalyzed by an
intricate enzymatic cascade in which ATG7 functions as an activating
E1 enzyme using ATP to form a thioester with LC3/GABARAP and transferring
them to an E2 enzyme, ATG3, via a trans-thioesterification reaction.
ATG3 performs the lipidation of LC3/GABARAP in conjunction with an
E3-like enzyme complex of isopeptide-linked ATG5-ATG12. Membrane tethering
of LC3/GABARAP is crucial for membrane expansion of autophagosomes
to engulf the autophagic cargo and for eventual lysosome fusion.6−9 However, the mechanistic details of LC3/GABARAP-lipidation by ATG3
remain enigmatic.

A conserved feature of the autophagy pathway
is the recurring presence
of short peptide motifs called LC3 interacting regions (LIR) in proteins
associated with autophagy.10 Proteins containing
LIR motifs are recruited to LC3/GABARAP through hydrophobic interactions.
The core sequence of LIR motifs, ΦxxΨ, is characterized
by the presence of an aromatic residue (Φ)—Trp, Phe,
and Tyr—followed by two variable positions and an aliphatic,
hydrophobic amino acid (Ψ), generally Ile, Leu, or Val.11 The LIR motif adopts an extended conformation
forming an intermolecular β-sheet with β2 of LC3/GABARAP.
A variety of LIR motifs have been identified in selective autophagy
receptors, which employ them to recruit cargos to expanding autophagosomes.2 In addition, these motifs can also be found in
proteins involved in the attachment of LC3/GABARAP to membranes such
as ATG4,12 a protease required for processing
of proLC3/proGABARAP and delipidation of LC3/GABARAP.

Few chemical
probes for autophagy have been developed.13−15 The majority
are inhibitors of autophagy proteins functioning upstream
of the lipidation cascade, such as wortmannin, a PI3K inhibitor that
abrogates localization of lipidation enzymes to expanding autophagosomes.16,17 Despite the similarity to the ubiquitin proteasome pathway, few
chemical probes exist for LC3/GABARAP lipidation.18,19 Hemelaar et al. reported the synthesis of LC3/GABARAP activity-based
probes by direct aminolysis and employed these probes to identify
ATG4 as the processing protease of proLC3/proGABARAP.20 However, access to these probes by direct aminolysis is
generally hampered by harsh reaction conditions and inefficient conversion.

Herein, we report the facile preparation of GABARAP and LC3A activity-based
probes (ABPs) using a recently established hydrazide acylation protocol.21,22 Access to these ABPs was critical for the identification of an unknown
noncanonical LIR motif embedded in an unusual β-sheet conformation
in human ATG3. Further investigation of this motif with macrocyclic
peptide binders, X-ray crystallography, and CRISPR-enabled in cellulo studies revealed that this LIR motif is crucial
for the enzymatic function of ATG3.

Results and DiscussionPreparation of GABARAP and LC3A Activity-Based ProbesWe recently reported access to Ubl proteins bearing thiol-reactive
electrophiles at their C-terminus by chemoselective acylation of recombinant
protein-hydrazides with carboxylic acid anhydrides.22 The lack of ABPs for autophagy inspired us to prepare GABARAP
ABPs using this protocol. GABARAP was expressed as an Mxe GyrA intein fusion to C-terminal glycine deletion (ΔG116)
to conserve the atomic register at the C-terminus after the introduction
of electrophilic groups. The expressed fusion proteins were cleaved
with hydrazine and chemoselectively acylated with symmetric anhydrides
at pH 3.0. The resulting acylhydrazides mimic the native glycine and
place the electrophile close to the reactive site of the native C-terminus
(Figure S1). In analogy to probes that showed
excellent activity in the UFM1-pathway, we selected α-chloroacetyl
probe 1 and methyl fumarate probe 2 derived
from GABARAP(ΔG116)-NHNH2 for further studies into
autophagy pathways (Figure 1a).21Figure 1Modification of ATG3
with GABARAP ABPs depends on LIRATG3. a. Characterization
of GABARAP activity-based probes, 1 and 2, obtained by hydrazide-acylation. Deconvoluted
mass-spectrum (ESI) of GABARAP(ΔG116)–NHNH α-chloroacetyl 1. Expected mass 13707 Da. Deconvoluted mass-spectrum (ESI)
of GABARAP(ΔG116)–NHNH methyl-fumarate 2. Expected mass 13743 Da. b. Reaction of 2 or Ub(ΔGG)–NHNH methyl fumarate probe (15 μM)
with recombinant E2s ATG3 and Ube2K (15 μM). The reaction was
allowed to proceed for 1 h and was analyzed by SDS-PAGE and Coomassie
Blue staining. c. ColabFold-predicted structure of the
protein-complex of ATG3 and GABARAP. Side-chain atoms are shown for
the predicted LIR motif in ATG3 (90–112). d. Reaction
of 2 (15 μM) with ATG3 variants C1, C0, and ΔLIR
(15 μM). ATG3 C1 contains only active-site cysteine C264. ATG3
C0 contains no cysteines. ATG3 ΔLIR lacks amino acids 95–111.
Reaction was allowed to proceed for 1 h and was analyzed by SDS-PAGE
and Coomassie Blue staining. Full-gel images for b and d are available in the Supplementary Information.To assess the reactivity of the probes, we incubated 1 and 2 (15 μM) with recombinantly expressed
ATG3
(15 μM) and its catalytically inactive variant C264A (Figure 1b, Figure S2). Probe 2 reacted with ATG3 efficiently
leading to the formation of multiple ATG3–GABARAP bands. Mutation
of active-site cysteine 264 to alanine caused the loss of one ATG3–GABARAP
band but did not abrogate additional bands for the complex, indicating
unspecific modification of ATG3, likely due to the presence of multiple
cysteine residues in flexible regions of ATG3. Probe 2 nonetheless reacted specifically with its cognate E2 ATG3, as ubiquitin
E2 Ube2K did not react with 2 and ATG3 wt did not react
with Ub(ΔGG) methyl fumarate probe. We were intrigued by the
cross-linking efficiency of 2 with ATG3 and sought to
investigate the origin of this efficiency by generating a C1 variant
of ATG3 that only contains the catalytic cysteine 264 and a corresponding
C0 variant containing no cysteines. Probe 2 showed excellent
reactivity with ATG3 C1, and as expected, no reaction was observed
with ATG3 C0 (Figure 1d, lanes 1–2). ATG3 C1 maintains its cross-linking efficiency
compared to ATG3 wt indicating that cross-linking does not arise simply
from the presence of numerous cysteine residues in ATG3 but rather
from enhanced affinity between ATG3 and GABARAP. In general, the affinity
of E2s for their cognate Ubls are quite low, as the thioester transfer
from the E1 to the E2 enzyme is facilitated by additional interactions
from the UFD domain of the
E1 enzyme.23 The specificity and high reactivity
of our probes with human ATG3 indicated that ATG3 may contain additional
binding elements, resulting in a tighter interaction than generally
observed for E2s and their Ubl cognates.24Human ATG3 Contains a Noncanonical LIR MotifPrevious
reports established that yeast ATG3 contains an ATG8 interaction motif
required for interaction with ATG8, the yeast analogue of LC3 and
GABARAP.25 However, this motif is not conserved
in human ATG3 and therefore did not explain the enhanced interaction
of human ATG3 with GABARAP (Figure S3).
As there was no structural information available on human ATG3 that
could explain the higher affinity of ATG3 for GABARAP, we turned to
computational methods. Artificial intelligence driven modeling such
as AlphaFold has shown great promise in accessing structural data
on proteins for which no structural data is available.26 Recent improvements have enabled researchers
to model protein–protein interactions,27 and we employed an open-source modeling tool, ColabFold, to model
the protein–protein interaction between ATG3 and GABARAP (Figure 1c, Figures S4–S5).28 Intriguingly,
ColabFold modeled a complex of ATG3 and GABARAP that resembled the
canonical closed-conformation observed in E2-Ub thioester complexes.29,30 Additionally, a section of the flexible region of ATG3, L94-Y111,
folded into a short β-sheet that was bound to a groove formed
by helices α2 and α3. The interaction site on GABARAP
was equivalent to the canonical binding region of LIR motifs. Inspection
of the β-sheet sequence shows a sequence motif, W107VDT110, reminiscent of core LIR motifs but containing
threonine instead of the canonical Ile, Leu, or Val residues. The
binding mode of the WDVT motif was similar to binding modes observed
for previously investigated LIR motifs; W107 binds to hydrophobic
pocket (HP) 1 and T110 to HP2. Therefore, the WVDT motif likely represents
a noncanonical LIR motif previously undiscovered in human ATG3, in
which T110 binds to GABARAP instead of the canonical aliphatic amino
acids. The β-sheet forms further interactions with GABARAP outside
of the core LIR motif. Several ionic interactions between Asp and
Glu residues in the peptide and Lys/Arg residues surrounding the binding
site seem to further stabilize the interaction.Sequence alignment
of ATG3 protein sequences from different organisms shows that the
LIR motif described herein is conserved across species and kingdoms
(Figures S6, S7). Intriguingly, several
species contain two LIR motifs, an LIR motif analogous to the motif
found in S. cerevisiae and a second LIR motif analogous
to the LIR motif found in human ATG3 described herein. In contrast,
animals and plants only encode for one LIR motif corresponding to
the human LIR motif.31S. cerevisiae and the closely related S. pastorianus were the
only analyzed species not containing an analogue of the human LIR
motif. Alignment of the predicted LIR motif shows strict conservation
of tryptophan and preference for threonine. Surprisingly, the second
conserved residue, occupied by threonine in humans, is almost exclusively
populated by residues not considered canonical for LIR motifs (Figure S8).LIRATG3 was predicted
to be embedded in a β-sheet,
a conformation rarely found in reported LIR motifs. Most LIR motifs
are embedded in an extended conformation that forms an intermolecular
β-sheet with β2 of LC3/GABARAP. The presence of LIR-motifs
within β-sheets has only recently been reported for the pathogenic
protein RavZ,32 FNIP,33 and TP53INP234 (vide infra).
However, several other noncanonical LIR motifs have been reported.
For example, UBA5 binds GABARAP using two aliphatic residues and an
additional aromatic amino acid outside of the core LIR sequence,35 and NDP52 exclusively binds to LC3C via its
cLIR motif consisting of only aliphatic residues.36 Nevertheless, even these noncanonical motifs bind in an
extended linear fashion to LC3/GABARAP.We hypothesized that
the putative noncanonical LIR motif in ATG3
was responsible for its high affinity interaction with GABARAP, resulting
in the efficient reaction of our probes with ATG3. We sought to test
this with a cross-linking assay and chose ATG3 variants C1 and C0
(vide supra) to facilitate analysis. We expressed a variant of ATG3
C1 lacking the LIR motif (Δ95–111), ΔLIR. Incubation
of 2 with ATG3 ΔLIR showed no complex formation
by SDS-PAGE analysis (Figure 1d). Removal of the LIR motif by deletion of residues 95–111
had the same effect as removal of the catalytic cysteine C264. The
same LIR-dependence was observed for cross-linking with wild-type
ATG3, which retained all its cysteine residues (Figure S9). This observation was independent of the probe
used; reaction of probe 1 and cross-linking with ATG3
were also strictly dependent on LIRATG3. These findings
support the ColabFold model that ATG3 contains an additional binding
element for GABARAP through its LIR motif.GABARAP is representative
of one of two protein families conjugated
by ATG3. The LC3-family is also tethered to membranes by ATG3. To
test the general involvement of LIRATG3 in ATG3 activity
beyond GABARAP we also prepared LC3A probes 3 and 4. We allowed these probes to react with ATG3 C1, C0, and
ΔLIR. As observed for GABARAP, LC3A shows efficient reaction
with ATG3 C1 but no reaction with either C0 or ΔLIR variants
(Figure S10). These results indicate that
LIRATG3 has a general role in ATG3 binding to LC3/GABARAP.LIRATG3 Motif Forms a β-SheetTo exclude
an effect of LIR deletion on protein activity we performed a competition
experiment with chemically prepared LIR (L94-H112) peptide 5. Probe 2 (15 μM) was incubated with ATG3 C1 in
the presence of increasing amounts of 5. The peptide
blocked ATG3 modification in a concentration-dependent manner with
an IC50 of 104 μM, indicating that the LIR motif
is involved in binding GABARAP (Figure 2a,b,d). We also postulated that cyclization of the
LIR peptide would recapitulate its β-sheet conformation and
lead to tighter binding than the linear peptide due to preorganization.37 This approach has recently been shown to be
successful in the de novo design of GABARAP-specific
peptide binders.38 Cyclic peptide 6 was prepared by solid-phase peptide synthesis followed by
cyclization using selective cysteine alkylation (Figure S11).39,40 Peptide 6 inhibited
the reaction of 2 with ATG3 C1 with an IC50 of 24 μM (Figure 2c,d). This is 5-fold lower than the IC50 observed
for the linear peptide 5, suggesting that the β-sheet
conformation is indeed crucial for the interaction of the ATG3 LIR
motif with GABARAP. CD spectroscopic analysis of the peptides showed
that linear peptide 5 is not structured in solution.
Upon cyclization 6 remains mostly unstructured with some
transition to an organized structure (Figure S12). Additionally, we confirmed the interaction of LIRATG3 peptides with GABARAP and LC3A by fluorescence polarization (Figure 2e–g). LIRATG3-derived peptides bind to both LC3A and GABARAP. As indicated
by the competition experiments, cyclic peptides bound markedly tighter
to GABARAP and LC3A than the linear peptides. Tighter binding upon
peptide cyclization strongly suggests that it is the β-sheet
conformation, observed in our prediction, that binds to GABARAP/LC3A
because conformational restriction of the peptide precludes binding
to other regions of GABARAP in an extended fashion. A truncated peptide,
containing amino acids G103-Y111 including the core LIR motif, bound
6-fold weaker than the linear peptide. The severely diminished affinity
observed for the truncated peptide indicates that the solvent-exposed
upper half of the β-sheet (104–95) contributes significantly
to the binding of LIRATG3.Figure 2Characterization of LIRATG3. a. Reaction
scheme outlining competition assay used for b, c, and d. b. Coomassie Blue-stained
SDS-PAGE analysis for reaction of 2 with ATG3 C1 in the
presence of varying concentrations of linear LIRATG3 peptide 5 as outlined in a. c. Coomassie
Blue-stained SDS-PAGE analysis for reaction of 2 with
ATG3 C1 in the presence of varying concentrations of cyclized LIRATG3 peptide 6 as outlined in a. d. Quantification of competition assays shown in b and c. The amount of GABARAP–ATG3 C1 complex
was quantified by gel-densitometry and normalized to the reaction
without peptide for each gel. IC50 was estimated by nonlinear
regression. n = 2,3 independent experiments with
similar results. Data are presented as average values ± s.d. e. Fluorescence polarization binding-data of fluorescein-modified
LIRATG3 peptide binding to GABARAP. Either linear, cyclized,
or truncated (103–111) LIRATG3 peptide was used.
The measurement was performed in triplicates and data shown as average
values ± s.d. KD was estimated using
nonlinear regression. f. Fluorescence polarization data
of fluorescein-modified LIRATG3 peptide binding to LC3A.
Either linear, cyclized, or truncated (103–111) LIRATG3 peptide was used. KD was estimated using
nonlinear regression. The measurement was performed in triplicate
and data shown as average values ± s.d. g. Table
summarizing KD values obtained in e,f. Full-gel images for b and c are available in the Supplementary Information.Cocrystal Structure of GABARAP and LIRATG3To further corroborate our findings, we cocrystallized GABARAP with
an LIRATG3 peptide (Y90–H112). We solved the cocrystal
structure at a resolution of 2.6 Å and could resolve amino acids
E95–T110 of the LIRATG3 peptide (Table S1, Figure 3a). GABARAP adopted its previously described closed conformation
and showed no structural rearrangements.41 The structure of the LIR peptide and also its interactions with
GABARAP were consistent with respect to the ColabFold prediction with
a backbone-atom root-mean-square deviation of 0.61 Å (Figure 3b). ATG3 W107 binds
to HP1 via hydrophobic interactions in a deep pocket formed by GABARAP
P30, K48, and F104. The binding mode is identical to that observed
for other LIR motifs (Figure 3c). ATG3 V108 is buried by hydrophobic interactions with GABARAP
K46 and Y49 and the upper strand of the LIR β-sheet (E95, I97).
These interactions block any solvent contact of V108 and likely strengthen
its hydrophobic packing. ATG3 D109 forms a surface-exposed salt bridge
with R28. ATG3 T110 binds to HP2 akin to the more conserved Ile, Leu,
and Val residues commonly observed in LIR motifs. The shallow pocket
is formed by GABARAP Y49, V51, P52, L55, and L63. Thr110 likely mimics
the canonical LIR residues, Ile, Leu, and Val, by rotation of its
methyl group to form hydrophobic contacts with HP2, enabling its alcohol
moiety to participate in H-bonding interactions with Pro52. To form
the observed β-sheet conformation, residues N-terminal of W107
form a hairpin turn consisting of glycine and aspartic acid residues,
mainly binding via H-bonding and electrostatic interactions to GABARAP.
ATG3 D104 binds to H9 via a hydrogen bond. Aspartic and glutamic acid
residues E99, E100, D101, and D102 form salt bridges with various
basic residues, K20, K24, and K48 (Figure 3d). Following residue E99, a short β-sheet
is formed for which backbone atoms could be resolved until E95. Overall,
the interaction between the ATG3 LIR motif and GABARAP buries 660
Å2.42Figure 3Cocrystal structure of
GABARAP and LIRATG3. a. Cocrystal structure
of GABARAP (gray) and LIRATG3 (orange).
Side-chains in contact with LIRATG3 are shown. For clarity,
W107 is colored in red. One copy of the GABARAP-LIRATG3 complex was chosen from the asymmetric unit. Other copies in the
asymmetric unit show similar structures of the complex. b. Zoom-in and overlap of LIRATG3 (orange) with structure
of LIRATG3 predicted by ColabFold (dark blue); backbone-atom
RMSD is 0.61 Å. c. Zoom-in of core LIRATG3 motif binding to hydrophobic pockets HP1 and HP2. H-bonding and
ionic interactions are indicated by dashed lines. d.
Zoom-in of hairpin turn and the interaction of LIRATG3 with
basic residues in GABARAP. Residues E95–D104 have been omitted
for clarity. H-bonding and ionic interactions are indicated by dashed
lines.Based on the predicted ATG3-GABARAP model and our
cocrystal structure,
we identified residues in the LIR motif involved in the ATG3-GABARAP
interaction. To probe the contribution of LIRATG3 residues
to GABARAP binding we performed an Ala-screen using our previously
established cross-linking assay as the readout (Figure 4). Cross-linking efficiency of probe 2 with ATG3 or its mutants is dependent on the affinity of
the LIR motif. We expressed ATG3 alanine mutants of the identified
residues and reacted them with probe 2. As expected for
LIR motifs, mutation of W107 almost completely abrogated the reaction
of 2 with ATG3. Similarly, V108A significantly reduced
the cross-linking efficiency. This is in agreement with observations
for other LIR motifs showing that the first variable position contributes
significantly to the binding of LIR motifs.43 Intriguingly, mutation of D109 or T110 had almost no effect on reaction
efficiency. Additionally, the majority of the other screened residues
have at least some impact on the conjugation with GABARAP. E95 and
I97 severely diminish cross-linking supporting our hypothesis that
these residues are important for β-sheet formation and burying
V108. The hairpin turn binds mainly via electrostatic interactions
to GABARAP. Increasing the salt concentration disrupts this interaction
as shown by reduced fluorescence polarization of LIRATG3 peptide upon binding to GABARAP and cross-linking with ATG3 (Figure S13). These data highlight how binding
of LIRATG3 to GABARAP/LC3A is driven by a combination of
hydrophobic and ionic interactions.Figure 4Dissection of interactions between GABARAP
and LIRATG3 by alanine scanning. GABARAP probe 2 (15 μM)
was reacted with ATG3 C1 alanine variants (15 μM) for 30 min.
Coomassie Blue stained SDS-PAGE gel showing modification of ATG3 C1
variants with GABARAP methyl fumarate. The indicated ATG3 residues
were mutated to alanine. Cross-linking efficiency was estimated by
gel densitometry and normalized to the reaction with ATG3 C1. Data
are presented as average values ± s.d. n = 2
independent experiments. Full-gel images are available in the Supplementary Information.LIR Motif Affects LC3 Lipidation and Thioester Transfer In CelluloLIR motifs are crucial to the function
of various proteins involved in autophagy including selective autophagy
receptors as well as enzymes involved in LC3 lipidation such as ATG744 or ATG4.12 The LIR
motif in yeast ATG3 was shown to affect Atg8 lipidation in
vitro and also affects the cytoplasm-to-vacuole pathway.25,45To gauge the role of the LIR motif in human ATG3, we generated
a homozygous HEK293T ATG3 knockout (KO) cell line using CRISPR-Cas9.
The knockout was validated by immunoblotting against ATG3 and LC3A/B
showing defective LC3 lipidation (Figure 5a). Upon starvation, rescue with FLAG-ATG3
wild-type led to LC3 lipidation as evidenced by the presence of the
LC3–II band. However, expression of ATG3 C264A or ATG3 ΔLIR
did not induce LC3 lipidation (Figure 5b, Figure S14). This finding
indicates that the LIR motif in human ATG3 is a prerequisite for LC3
lipidation by ATG3 and contrasts with S. cerevisiae, in which the LIR motif had no effect on starvation induced Atg8
lipidation.Figure 5LIRATG3 effects LC3 lipidation and thioester transfer. a. Generation of homozygous knockouts of HEK293T cells using
CRISPR-Cas9 gene editing. Gene editing efficiency was validated by
immunoblotting against ATG3 and LC3A/B. Analysis for two knockout
clones is shown. b. Rescue with FLAG-ATG3 variants in
HEK293T ATG3 – /– cells and autophagy induced by starvation
for 1 h in the presence of chloroquine (40 μM). Lipidation was
assessed by immunoblotting against LC3A/B. As a positive control wild-type
HEK293T cells were starved and analyzed in parallel to ATG3 –
/– cells. c. Coimmunoprecipitation of FLAG-ATG3
variants (wt, C264A, ΔLIR, and W107A). Interaction of ATG3 with
ATG5–ATG12 and ATG16L1 was assessed by immunoblotting. d. Pulse-chase assay of GABARAP transfer from ATG7 to ATG3.
ATG7 was charged with fluorescein-labeled GABARAP. The reaction was
quenched by addition of EDTA, and transfer initiated by addition of
ATG3 variants (wt, C264A, ΔLIR). The reaction was monitored
for the indicated time-points. Thioester-transfer was assessed by
SDS-PAGE and in-gel fluorescence. e. Quantification of
in-gel fluorescence of d. f. Structural
models showing the potential role of LIRATG3 in thioester
transfer from ATG7 to ATG3. ATG3–ATG7 complex was modeled and
overlaid with the previously reported structure (PDB: 4GHL). The Atg7-Atg8
model44 (PDB: 3VH3) was overlaid with structures. For the
post-transfer model, the ATG3–GABARAP model from Figure 1c was used and aligned to ATG3
of 4GHL. g. Detection of ATG3∼LC3A/B thioester in cellulo. FLAG-ATG3 variants were transiently expressed
in HCT116 cells in which endogenous ATG3 was knocked down by CRISPR
inhibition (CRISPRi). Autophagy was induced by starvation in the presence
of chloroquine (40 μM) for 1 h. Cell lysates were prepared
either in the absence or in the presence of β-mercaptoethanol
(β-ME). Samples containing reducing agent were boiled prior
to SDS-PAGE analysis. The presence of a thioester intermediate was
analyzed by anti-FLAG immunoblotting. ATG3∼LC3A/B thioester
was only observed for ATG3 wt. Data are presented as average values
± s.d. and normalized to gel density of ATG3 wt 5 min. n = 3 independent experiments. Full-gel images for a–e are available in the Supplementary Information.How Does LIRATG3 Influence ATG3’s Activity?We investigated the interaction of ATG3 with components of its
enzymatic cascade by coimmunoprecipitation. We transiently expressed
FLAG-ATG3 variants in HCT116 ATG3 knock-down cells generated using
CRISPRi (Figure S15) and assessed their
interaction with E3 complex subunits by anti-FLAG Co-IP.46 ATG3 wt, C264A, ΔLIR, and W107A precipitated
the ATG5-ATG12 complex and ATG16L1 with similar efficiency (Figure 5c). Therefore, LIRATG3 does not affect lipidation through defective binding to
the E3 enzyme complex. Likewise, we performed pull-down of recombinantly
expressed ATG3 variants with FLAG-ATG7 immobilized on anti-FLAG resin.
No difference in pull-down efficiency was observed (Figure S16). This indicates that LIRATG3 or the
lack thereof does not impact binding of ATG3 to ATG7 via its ATG7
interacting region (RIA7).47In order
to assess the influence of LIRATG3 on thioester transfer
from ATG7 to ATG3 we performed a pulse-chase assay. Fluorescein-labeled
GABARAP was charged onto ATG7 and thioester transfer initiated by
addition of ATG3 variants. We observed a significant reduction in
transfer rate between ATG3 wt and ΔLIR (Figure 5d,e). This observation contrasts with S. cerevisiae Atg3 in which the LIR motif had no effect
on Atg8 transfer. In order to understand the origin of this rate defect
we modeled ATG3 onto the E1 enzyme ATG7 using ColabFold. We correctly
predicted RIA7 of ATG3 and its interaction with ATG7.47 The predicted structure is in good agreement with a previously
reported structure of the ATG7–ATG3 complex.48 Overlaying the predicted structure with a reported structure
of Atg8 bound to Atg7 places LIRATG3 in proximity (20–30
Å) of the LIR binding region of LC3/Atg8. Based on this proximity
we speculate that the LIR motif aids ATG3 in abstracting LC3 from
ATG7 by facilitating thioester exchange (Figure 5f). LC3 can undergo a 180° rotation
around its C-terminus to adopt the closed conformation observed in
our ColabFold prediction (Figure 1c).To confirm these in vitro results in cellulo, we expressed FLAG-ATG3 variants
in HEK293T cells to analyze the
presence of the ATG3∼LC3A/B thioester complex formation. Cell
lysis under reducing or nonreducing conditions showed the presence
of a thiol-sensitive band for wild-type ATG3. This band corresponds
to the ATG3∼LC3 thioester complex, as no thioester was detected
for ATG3 C264A (Figure 5g). With ATG3 ΔLIR no higher MW thiol-sensitive band was detected
either. This observation suggests that LIRATG3 influences
ATG3∼LC3A/B complex beyond a defect in its formation potentially
due to complex stability.

Results and Discussion

Preparation of GABARAP and LC3A Activity-Based ProbesWe recently reported access to Ubl proteins bearing thiol-reactive
electrophiles at their C-terminus by chemoselective acylation of recombinant
protein-hydrazides with carboxylic acid anhydrides.22 The lack of ABPs for autophagy inspired us to prepare GABARAP
ABPs using this protocol. GABARAP was expressed as an Mxe GyrA intein fusion to C-terminal glycine deletion (ΔG116)
to conserve the atomic register at the C-terminus after the introduction
of electrophilic groups. The expressed fusion proteins were cleaved
with hydrazine and chemoselectively acylated with symmetric anhydrides
at pH 3.0. The resulting acylhydrazides mimic the native glycine and
place the electrophile close to the reactive site of the native C-terminus
(Figure S1). In analogy to probes that showed
excellent activity in the UFM1-pathway, we selected α-chloroacetyl
probe 1 and methyl fumarate probe 2 derived
from GABARAP(ΔG116)-NHNH2 for further studies into
autophagy pathways (Figure 1a).21Figure 1Modification of ATG3
with GABARAP ABPs depends on LIRATG3. a. Characterization
of GABARAP activity-based probes, 1 and 2, obtained by hydrazide-acylation. Deconvoluted
mass-spectrum (ESI) of GABARAP(ΔG116)–NHNH α-chloroacetyl 1. Expected mass 13707 Da. Deconvoluted mass-spectrum (ESI)
of GABARAP(ΔG116)–NHNH methyl-fumarate 2. Expected mass 13743 Da. b. Reaction of 2 or Ub(ΔGG)–NHNH methyl fumarate probe (15 μM)
with recombinant E2s ATG3 and Ube2K (15 μM). The reaction was
allowed to proceed for 1 h and was analyzed by SDS-PAGE and Coomassie
Blue staining. c. ColabFold-predicted structure of the
protein-complex of ATG3 and GABARAP. Side-chain atoms are shown for
the predicted LIR motif in ATG3 (90–112). d. Reaction
of 2 (15 μM) with ATG3 variants C1, C0, and ΔLIR
(15 μM). ATG3 C1 contains only active-site cysteine C264. ATG3
C0 contains no cysteines. ATG3 ΔLIR lacks amino acids 95–111.
Reaction was allowed to proceed for 1 h and was analyzed by SDS-PAGE
and Coomassie Blue staining. Full-gel images for b and d are available in the Supplementary Information.To assess the reactivity of the probes, we incubated 1 and 2 (15 μM) with recombinantly expressed
ATG3
(15 μM) and its catalytically inactive variant C264A (Figure 1b, Figure S2). Probe 2 reacted with ATG3 efficiently
leading to the formation of multiple ATG3–GABARAP bands. Mutation
of active-site cysteine 264 to alanine caused the loss of one ATG3–GABARAP
band but did not abrogate additional bands for the complex, indicating
unspecific modification of ATG3, likely due to the presence of multiple
cysteine residues in flexible regions of ATG3. Probe 2 nonetheless reacted specifically with its cognate E2 ATG3, as ubiquitin
E2 Ube2K did not react with 2 and ATG3 wt did not react
with Ub(ΔGG) methyl fumarate probe. We were intrigued by the
cross-linking efficiency of 2 with ATG3 and sought to
investigate the origin of this efficiency by generating a C1 variant
of ATG3 that only contains the catalytic cysteine 264 and a corresponding
C0 variant containing no cysteines. Probe 2 showed excellent
reactivity with ATG3 C1, and as expected, no reaction was observed
with ATG3 C0 (Figure 1d, lanes 1–2). ATG3 C1 maintains its cross-linking efficiency
compared to ATG3 wt indicating that cross-linking does not arise simply
from the presence of numerous cysteine residues in ATG3 but rather
from enhanced affinity between ATG3 and GABARAP. In general, the affinity
of E2s for their cognate Ubls are quite low, as the thioester transfer
from the E1 to the E2 enzyme is facilitated by additional interactions
from the UFD domain of the
E1 enzyme.23 The specificity and high reactivity
of our probes with human ATG3 indicated that ATG3 may contain additional
binding elements, resulting in a tighter interaction than generally
observed for E2s and their Ubl cognates.24

Preparation of GABARAP and LC3A Activity-Based Probes

We recently reported access to Ubl proteins bearing thiol-reactive
electrophiles at their C-terminus by chemoselective acylation of recombinant
protein-hydrazides with carboxylic acid anhydrides.22 The lack of ABPs for autophagy inspired us to prepare GABARAP
ABPs using this protocol. GABARAP was expressed as an Mxe GyrA intein fusion to C-terminal glycine deletion (ΔG116)
to conserve the atomic register at the C-terminus after the introduction
of electrophilic groups. The expressed fusion proteins were cleaved
with hydrazine and chemoselectively acylated with symmetric anhydrides
at pH 3.0. The resulting acylhydrazides mimic the native glycine and
place the electrophile close to the reactive site of the native C-terminus
(Figure S1). In analogy to probes that showed
excellent activity in the UFM1-pathway, we selected α-chloroacetyl
probe 1 and methyl fumarate probe 2 derived
from GABARAP(ΔG116)-NHNH2 for further studies into
autophagy pathways (Figure 1a).21

Modification of ATG3
with GABARAP ABPs depends on LIRATG3. a. Characterization
of GABARAP activity-based probes, 1 and 2, obtained by hydrazide-acylation. Deconvoluted
mass-spectrum (ESI) of GABARAP(ΔG116)–NHNH α-chloroacetyl 1. Expected mass 13707 Da. Deconvoluted mass-spectrum (ESI)
of GABARAP(ΔG116)–NHNH methyl-fumarate 2. Expected mass 13743 Da. b. Reaction of 2 or Ub(ΔGG)–NHNH methyl fumarate probe (15 μM)
with recombinant E2s ATG3 and Ube2K (15 μM). The reaction was
allowed to proceed for 1 h and was analyzed by SDS-PAGE and Coomassie
Blue staining. c. ColabFold-predicted structure of the
protein-complex of ATG3 and GABARAP. Side-chain atoms are shown for
the predicted LIR motif in ATG3 (90–112). d. Reaction
of 2 (15 μM) with ATG3 variants C1, C0, and ΔLIR
(15 μM). ATG3 C1 contains only active-site cysteine C264. ATG3
C0 contains no cysteines. ATG3 ΔLIR lacks amino acids 95–111.
Reaction was allowed to proceed for 1 h and was analyzed by SDS-PAGE
and Coomassie Blue staining. Full-gel images for b and d are available in the Supplementary Information.

To assess the reactivity of the probes, we incubated 1 and 2 (15 μM) with recombinantly expressed
ATG3
(15 μM) and its catalytically inactive variant C264A (Figure 1b, Figure S2). Probe 2 reacted with ATG3 efficiently
leading to the formation of multiple ATG3–GABARAP bands. Mutation
of active-site cysteine 264 to alanine caused the loss of one ATG3–GABARAP
band but did not abrogate additional bands for the complex, indicating
unspecific modification of ATG3, likely due to the presence of multiple
cysteine residues in flexible regions of ATG3. Probe 2 nonetheless reacted specifically with its cognate E2 ATG3, as ubiquitin
E2 Ube2K did not react with 2 and ATG3 wt did not react
with Ub(ΔGG) methyl fumarate probe. We were intrigued by the
cross-linking efficiency of 2 with ATG3 and sought to
investigate the origin of this efficiency by generating a C1 variant
of ATG3 that only contains the catalytic cysteine 264 and a corresponding
C0 variant containing no cysteines. Probe 2 showed excellent
reactivity with ATG3 C1, and as expected, no reaction was observed
with ATG3 C0 (Figure 1d, lanes 1–2). ATG3 C1 maintains its cross-linking efficiency
compared to ATG3 wt indicating that cross-linking does not arise simply
from the presence of numerous cysteine residues in ATG3 but rather
from enhanced affinity between ATG3 and GABARAP. In general, the affinity
of E2s for their cognate Ubls are quite low, as the thioester transfer
from the E1 to the E2 enzyme is facilitated by additional interactions
from the UFD domain of the
E1 enzyme.23 The specificity and high reactivity
of our probes with human ATG3 indicated that ATG3 may contain additional
binding elements, resulting in a tighter interaction than generally
observed for E2s and their Ubl cognates.24

Human ATG3 Contains a Noncanonical LIR MotifPrevious
reports established that yeast ATG3 contains an ATG8 interaction motif
required for interaction with ATG8, the yeast analogue of LC3 and
GABARAP.25 However, this motif is not conserved
in human ATG3 and therefore did not explain the enhanced interaction
of human ATG3 with GABARAP (Figure S3).
As there was no structural information available on human ATG3 that
could explain the higher affinity of ATG3 for GABARAP, we turned to
computational methods. Artificial intelligence driven modeling such
as AlphaFold has shown great promise in accessing structural data
on proteins for which no structural data is available.26 Recent improvements have enabled researchers
to model protein–protein interactions,27 and we employed an open-source modeling tool, ColabFold, to model
the protein–protein interaction between ATG3 and GABARAP (Figure 1c, Figures S4–S5).28 Intriguingly,
ColabFold modeled a complex of ATG3 and GABARAP that resembled the
canonical closed-conformation observed in E2-Ub thioester complexes.29,30 Additionally, a section of the flexible region of ATG3, L94-Y111,
folded into a short β-sheet that was bound to a groove formed
by helices α2 and α3. The interaction site on GABARAP
was equivalent to the canonical binding region of LIR motifs. Inspection
of the β-sheet sequence shows a sequence motif, W107VDT110, reminiscent of core LIR motifs but containing
threonine instead of the canonical Ile, Leu, or Val residues. The
binding mode of the WDVT motif was similar to binding modes observed
for previously investigated LIR motifs; W107 binds to hydrophobic
pocket (HP) 1 and T110 to HP2. Therefore, the WVDT motif likely represents
a noncanonical LIR motif previously undiscovered in human ATG3, in
which T110 binds to GABARAP instead of the canonical aliphatic amino
acids. The β-sheet forms further interactions with GABARAP outside
of the core LIR motif. Several ionic interactions between Asp and
Glu residues in the peptide and Lys/Arg residues surrounding the binding
site seem to further stabilize the interaction.Sequence alignment
of ATG3 protein sequences from different organisms shows that the
LIR motif described herein is conserved across species and kingdoms
(Figures S6, S7). Intriguingly, several
species contain two LIR motifs, an LIR motif analogous to the motif
found in S. cerevisiae and a second LIR motif analogous
to the LIR motif found in human ATG3 described herein. In contrast,
animals and plants only encode for one LIR motif corresponding to
the human LIR motif.31S. cerevisiae and the closely related S. pastorianus were the
only analyzed species not containing an analogue of the human LIR
motif. Alignment of the predicted LIR motif shows strict conservation
of tryptophan and preference for threonine. Surprisingly, the second
conserved residue, occupied by threonine in humans, is almost exclusively
populated by residues not considered canonical for LIR motifs (Figure S8).LIRATG3 was predicted
to be embedded in a β-sheet,
a conformation rarely found in reported LIR motifs. Most LIR motifs
are embedded in an extended conformation that forms an intermolecular
β-sheet with β2 of LC3/GABARAP. The presence of LIR-motifs
within β-sheets has only recently been reported for the pathogenic
protein RavZ,32 FNIP,33 and TP53INP234 (vide infra).
However, several other noncanonical LIR motifs have been reported.
For example, UBA5 binds GABARAP using two aliphatic residues and an
additional aromatic amino acid outside of the core LIR sequence,35 and NDP52 exclusively binds to LC3C via its
cLIR motif consisting of only aliphatic residues.36 Nevertheless, even these noncanonical motifs bind in an
extended linear fashion to LC3/GABARAP.We hypothesized that
the putative noncanonical LIR motif in ATG3
was responsible for its high affinity interaction with GABARAP, resulting
in the efficient reaction of our probes with ATG3. We sought to test
this with a cross-linking assay and chose ATG3 variants C1 and C0
(vide supra) to facilitate analysis. We expressed a variant of ATG3
C1 lacking the LIR motif (Δ95–111), ΔLIR. Incubation
of 2 with ATG3 ΔLIR showed no complex formation
by SDS-PAGE analysis (Figure 1d). Removal of the LIR motif by deletion of residues 95–111
had the same effect as removal of the catalytic cysteine C264. The
same LIR-dependence was observed for cross-linking with wild-type
ATG3, which retained all its cysteine residues (Figure S9). This observation was independent of the probe
used; reaction of probe 1 and cross-linking with ATG3
were also strictly dependent on LIRATG3. These findings
support the ColabFold model that ATG3 contains an additional binding
element for GABARAP through its LIR motif.GABARAP is representative
of one of two protein families conjugated
by ATG3. The LC3-family is also tethered to membranes by ATG3. To
test the general involvement of LIRATG3 in ATG3 activity
beyond GABARAP we also prepared LC3A probes 3 and 4. We allowed these probes to react with ATG3 C1, C0, and
ΔLIR. As observed for GABARAP, LC3A shows efficient reaction
with ATG3 C1 but no reaction with either C0 or ΔLIR variants
(Figure S10). These results indicate that
LIRATG3 has a general role in ATG3 binding to LC3/GABARAP.

Human ATG3 Contains a Noncanonical LIR Motif

Previous
reports established that yeast ATG3 contains an ATG8 interaction motif
required for interaction with ATG8, the yeast analogue of LC3 and
GABARAP.25 However, this motif is not conserved
in human ATG3 and therefore did not explain the enhanced interaction
of human ATG3 with GABARAP (Figure S3).
As there was no structural information available on human ATG3 that
could explain the higher affinity of ATG3 for GABARAP, we turned to
computational methods. Artificial intelligence driven modeling such
as AlphaFold has shown great promise in accessing structural data
on proteins for which no structural data is available.26 Recent improvements have enabled researchers
to model protein–protein interactions,27 and we employed an open-source modeling tool, ColabFold, to model
the protein–protein interaction between ATG3 and GABARAP (Figure 1c, Figures S4–S5).28 Intriguingly,
ColabFold modeled a complex of ATG3 and GABARAP that resembled the
canonical closed-conformation observed in E2-Ub thioester complexes.29,30 Additionally, a section of the flexible region of ATG3, L94-Y111,
folded into a short β-sheet that was bound to a groove formed
by helices α2 and α3. The interaction site on GABARAP
was equivalent to the canonical binding region of LIR motifs. Inspection
of the β-sheet sequence shows a sequence motif, W107VDT110, reminiscent of core LIR motifs but containing
threonine instead of the canonical Ile, Leu, or Val residues. The
binding mode of the WDVT motif was similar to binding modes observed
for previously investigated LIR motifs; W107 binds to hydrophobic
pocket (HP) 1 and T110 to HP2. Therefore, the WVDT motif likely represents
a noncanonical LIR motif previously undiscovered in human ATG3, in
which T110 binds to GABARAP instead of the canonical aliphatic amino
acids. The β-sheet forms further interactions with GABARAP outside
of the core LIR motif. Several ionic interactions between Asp and
Glu residues in the peptide and Lys/Arg residues surrounding the binding
site seem to further stabilize the interaction.

Sequence alignment
of ATG3 protein sequences from different organisms shows that the
LIR motif described herein is conserved across species and kingdoms
(Figures S6, S7). Intriguingly, several
species contain two LIR motifs, an LIR motif analogous to the motif
found in S. cerevisiae and a second LIR motif analogous
to the LIR motif found in human ATG3 described herein. In contrast,
animals and plants only encode for one LIR motif corresponding to
the human LIR motif.31S. cerevisiae and the closely related S. pastorianus were the
only analyzed species not containing an analogue of the human LIR
motif. Alignment of the predicted LIR motif shows strict conservation
of tryptophan and preference for threonine. Surprisingly, the second
conserved residue, occupied by threonine in humans, is almost exclusively
populated by residues not considered canonical for LIR motifs (Figure S8).

LIRATG3 was predicted
to be embedded in a β-sheet,
a conformation rarely found in reported LIR motifs. Most LIR motifs
are embedded in an extended conformation that forms an intermolecular
β-sheet with β2 of LC3/GABARAP. The presence of LIR-motifs
within β-sheets has only recently been reported for the pathogenic
protein RavZ,32 FNIP,33 and TP53INP234 (vide infra).
However, several other noncanonical LIR motifs have been reported.
For example, UBA5 binds GABARAP using two aliphatic residues and an
additional aromatic amino acid outside of the core LIR sequence,35 and NDP52 exclusively binds to LC3C via its
cLIR motif consisting of only aliphatic residues.36 Nevertheless, even these noncanonical motifs bind in an
extended linear fashion to LC3/GABARAP.

We hypothesized that
the putative noncanonical LIR motif in ATG3
was responsible for its high affinity interaction with GABARAP, resulting
in the efficient reaction of our probes with ATG3. We sought to test
this with a cross-linking assay and chose ATG3 variants C1 and C0
(vide supra) to facilitate analysis. We expressed a variant of ATG3
C1 lacking the LIR motif (Δ95–111), ΔLIR. Incubation
of 2 with ATG3 ΔLIR showed no complex formation
by SDS-PAGE analysis (Figure 1d). Removal of the LIR motif by deletion of residues 95–111
had the same effect as removal of the catalytic cysteine C264. The
same LIR-dependence was observed for cross-linking with wild-type
ATG3, which retained all its cysteine residues (Figure S9). This observation was independent of the probe
used; reaction of probe 1 and cross-linking with ATG3
were also strictly dependent on LIRATG3. These findings
support the ColabFold model that ATG3 contains an additional binding
element for GABARAP through its LIR motif.

GABARAP is representative
of one of two protein families conjugated
by ATG3. The LC3-family is also tethered to membranes by ATG3. To
test the general involvement of LIRATG3 in ATG3 activity
beyond GABARAP we also prepared LC3A probes 3 and 4. We allowed these probes to react with ATG3 C1, C0, and
ΔLIR. As observed for GABARAP, LC3A shows efficient reaction
with ATG3 C1 but no reaction with either C0 or ΔLIR variants
(Figure S10). These results indicate that
LIRATG3 has a general role in ATG3 binding to LC3/GABARAP.

LIRATG3 Motif Forms a β-SheetTo exclude
an effect of LIR deletion on protein activity we performed a competition
experiment with chemically prepared LIR (L94-H112) peptide 5. Probe 2 (15 μM) was incubated with ATG3 C1 in
the presence of increasing amounts of 5. The peptide
blocked ATG3 modification in a concentration-dependent manner with
an IC50 of 104 μM, indicating that the LIR motif
is involved in binding GABARAP (Figure 2a,b,d). We also postulated that cyclization of the
LIR peptide would recapitulate its β-sheet conformation and
lead to tighter binding than the linear peptide due to preorganization.37 This approach has recently been shown to be
successful in the de novo design of GABARAP-specific
peptide binders.38 Cyclic peptide 6 was prepared by solid-phase peptide synthesis followed by
cyclization using selective cysteine alkylation (Figure S11).39,40 Peptide 6 inhibited
the reaction of 2 with ATG3 C1 with an IC50 of 24 μM (Figure 2c,d). This is 5-fold lower than the IC50 observed
for the linear peptide 5, suggesting that the β-sheet
conformation is indeed crucial for the interaction of the ATG3 LIR
motif with GABARAP. CD spectroscopic analysis of the peptides showed
that linear peptide 5 is not structured in solution.
Upon cyclization 6 remains mostly unstructured with some
transition to an organized structure (Figure S12). Additionally, we confirmed the interaction of LIRATG3 peptides with GABARAP and LC3A by fluorescence polarization (Figure 2e–g). LIRATG3-derived peptides bind to both LC3A and GABARAP. As indicated
by the competition experiments, cyclic peptides bound markedly tighter
to GABARAP and LC3A than the linear peptides. Tighter binding upon
peptide cyclization strongly suggests that it is the β-sheet
conformation, observed in our prediction, that binds to GABARAP/LC3A
because conformational restriction of the peptide precludes binding
to other regions of GABARAP in an extended fashion. A truncated peptide,
containing amino acids G103-Y111 including the core LIR motif, bound
6-fold weaker than the linear peptide. The severely diminished affinity
observed for the truncated peptide indicates that the solvent-exposed
upper half of the β-sheet (104–95) contributes significantly
to the binding of LIRATG3.Figure 2Characterization of LIRATG3. a. Reaction
scheme outlining competition assay used for b, c, and d. b. Coomassie Blue-stained
SDS-PAGE analysis for reaction of 2 with ATG3 C1 in the
presence of varying concentrations of linear LIRATG3 peptide 5 as outlined in a. c. Coomassie
Blue-stained SDS-PAGE analysis for reaction of 2 with
ATG3 C1 in the presence of varying concentrations of cyclized LIRATG3 peptide 6 as outlined in a. d. Quantification of competition assays shown in b and c. The amount of GABARAP–ATG3 C1 complex
was quantified by gel-densitometry and normalized to the reaction
without peptide for each gel. IC50 was estimated by nonlinear
regression. n = 2,3 independent experiments with
similar results. Data are presented as average values ± s.d. e. Fluorescence polarization binding-data of fluorescein-modified
LIRATG3 peptide binding to GABARAP. Either linear, cyclized,
or truncated (103–111) LIRATG3 peptide was used.
The measurement was performed in triplicates and data shown as average
values ± s.d. KD was estimated using
nonlinear regression. f. Fluorescence polarization data
of fluorescein-modified LIRATG3 peptide binding to LC3A.
Either linear, cyclized, or truncated (103–111) LIRATG3 peptide was used. KD was estimated using
nonlinear regression. The measurement was performed in triplicate
and data shown as average values ± s.d. g. Table
summarizing KD values obtained in e,f. Full-gel images for b and c are available in the Supplementary Information.

LIRATG3 Motif Forms a β-Sheet

To exclude
an effect of LIR deletion on protein activity we performed a competition
experiment with chemically prepared LIR (L94-H112) peptide 5. Probe 2 (15 μM) was incubated with ATG3 C1 in
the presence of increasing amounts of 5. The peptide
blocked ATG3 modification in a concentration-dependent manner with
an IC50 of 104 μM, indicating that the LIR motif
is involved in binding GABARAP (Figure 2a,b,d). We also postulated that cyclization of the
LIR peptide would recapitulate its β-sheet conformation and
lead to tighter binding than the linear peptide due to preorganization.37 This approach has recently been shown to be
successful in the de novo design of GABARAP-specific
peptide binders.38 Cyclic peptide 6 was prepared by solid-phase peptide synthesis followed by
cyclization using selective cysteine alkylation (Figure S11).39,40 Peptide 6 inhibited
the reaction of 2 with ATG3 C1 with an IC50 of 24 μM (Figure 2c,d). This is 5-fold lower than the IC50 observed
for the linear peptide 5, suggesting that the β-sheet
conformation is indeed crucial for the interaction of the ATG3 LIR
motif with GABARAP. CD spectroscopic analysis of the peptides showed
that linear peptide 5 is not structured in solution.
Upon cyclization 6 remains mostly unstructured with some
transition to an organized structure (Figure S12). Additionally, we confirmed the interaction of LIRATG3 peptides with GABARAP and LC3A by fluorescence polarization (Figure 2e–g). LIRATG3-derived peptides bind to both LC3A and GABARAP. As indicated
by the competition experiments, cyclic peptides bound markedly tighter
to GABARAP and LC3A than the linear peptides. Tighter binding upon
peptide cyclization strongly suggests that it is the β-sheet
conformation, observed in our prediction, that binds to GABARAP/LC3A
because conformational restriction of the peptide precludes binding
to other regions of GABARAP in an extended fashion. A truncated peptide,
containing amino acids G103-Y111 including the core LIR motif, bound
6-fold weaker than the linear peptide. The severely diminished affinity
observed for the truncated peptide indicates that the solvent-exposed
upper half of the β-sheet (104–95) contributes significantly
to the binding of LIRATG3.

Characterization of LIRATG3. a. Reaction
scheme outlining competition assay used for b, c, and d. b. Coomassie Blue-stained
SDS-PAGE analysis for reaction of 2 with ATG3 C1 in the
presence of varying concentrations of linear LIRATG3 peptide 5 as outlined in a. c. Coomassie
Blue-stained SDS-PAGE analysis for reaction of 2 with
ATG3 C1 in the presence of varying concentrations of cyclized LIRATG3 peptide 6 as outlined in a. d. Quantification of competition assays shown in b and c. The amount of GABARAP–ATG3 C1 complex
was quantified by gel-densitometry and normalized to the reaction
without peptide for each gel. IC50 was estimated by nonlinear
regression. n = 2,3 independent experiments with
similar results. Data are presented as average values ± s.d. e. Fluorescence polarization binding-data of fluorescein-modified
LIRATG3 peptide binding to GABARAP. Either linear, cyclized,
or truncated (103–111) LIRATG3 peptide was used.
The measurement was performed in triplicates and data shown as average
values ± s.d. KD was estimated using
nonlinear regression. f. Fluorescence polarization data
of fluorescein-modified LIRATG3 peptide binding to LC3A.
Either linear, cyclized, or truncated (103–111) LIRATG3 peptide was used. KD was estimated using
nonlinear regression. The measurement was performed in triplicate
and data shown as average values ± s.d. g. Table
summarizing KD values obtained in e,f. Full-gel images for b and c are available in the Supplementary Information.

Cocrystal Structure of GABARAP and LIRATG3To further corroborate our findings, we cocrystallized GABARAP with
an LIRATG3 peptide (Y90–H112). We solved the cocrystal
structure at a resolution of 2.6 Å and could resolve amino acids
E95–T110 of the LIRATG3 peptide (Table S1, Figure 3a). GABARAP adopted its previously described closed conformation
and showed no structural rearrangements.41 The structure of the LIR peptide and also its interactions with
GABARAP were consistent with respect to the ColabFold prediction with
a backbone-atom root-mean-square deviation of 0.61 Å (Figure 3b). ATG3 W107 binds
to HP1 via hydrophobic interactions in a deep pocket formed by GABARAP
P30, K48, and F104. The binding mode is identical to that observed
for other LIR motifs (Figure 3c). ATG3 V108 is buried by hydrophobic interactions with GABARAP
K46 and Y49 and the upper strand of the LIR β-sheet (E95, I97).
These interactions block any solvent contact of V108 and likely strengthen
its hydrophobic packing. ATG3 D109 forms a surface-exposed salt bridge
with R28. ATG3 T110 binds to HP2 akin to the more conserved Ile, Leu,
and Val residues commonly observed in LIR motifs. The shallow pocket
is formed by GABARAP Y49, V51, P52, L55, and L63. Thr110 likely mimics
the canonical LIR residues, Ile, Leu, and Val, by rotation of its
methyl group to form hydrophobic contacts with HP2, enabling its alcohol
moiety to participate in H-bonding interactions with Pro52. To form
the observed β-sheet conformation, residues N-terminal of W107
form a hairpin turn consisting of glycine and aspartic acid residues,
mainly binding via H-bonding and electrostatic interactions to GABARAP.
ATG3 D104 binds to H9 via a hydrogen bond. Aspartic and glutamic acid
residues E99, E100, D101, and D102 form salt bridges with various
basic residues, K20, K24, and K48 (Figure 3d). Following residue E99, a short β-sheet
is formed for which backbone atoms could be resolved until E95. Overall,
the interaction between the ATG3 LIR motif and GABARAP buries 660
Å2.42Figure 3Cocrystal structure of
GABARAP and LIRATG3. a. Cocrystal structure
of GABARAP (gray) and LIRATG3 (orange).
Side-chains in contact with LIRATG3 are shown. For clarity,
W107 is colored in red. One copy of the GABARAP-LIRATG3 complex was chosen from the asymmetric unit. Other copies in the
asymmetric unit show similar structures of the complex. b. Zoom-in and overlap of LIRATG3 (orange) with structure
of LIRATG3 predicted by ColabFold (dark blue); backbone-atom
RMSD is 0.61 Å. c. Zoom-in of core LIRATG3 motif binding to hydrophobic pockets HP1 and HP2. H-bonding and
ionic interactions are indicated by dashed lines. d.
Zoom-in of hairpin turn and the interaction of LIRATG3 with
basic residues in GABARAP. Residues E95–D104 have been omitted
for clarity. H-bonding and ionic interactions are indicated by dashed
lines.Based on the predicted ATG3-GABARAP model and our
cocrystal structure,
we identified residues in the LIR motif involved in the ATG3-GABARAP
interaction. To probe the contribution of LIRATG3 residues
to GABARAP binding we performed an Ala-screen using our previously
established cross-linking assay as the readout (Figure 4). Cross-linking efficiency of probe 2 with ATG3 or its mutants is dependent on the affinity of
the LIR motif. We expressed ATG3 alanine mutants of the identified
residues and reacted them with probe 2. As expected for
LIR motifs, mutation of W107 almost completely abrogated the reaction
of 2 with ATG3. Similarly, V108A significantly reduced
the cross-linking efficiency. This is in agreement with observations
for other LIR motifs showing that the first variable position contributes
significantly to the binding of LIR motifs.43 Intriguingly, mutation of D109 or T110 had almost no effect on reaction
efficiency. Additionally, the majority of the other screened residues
have at least some impact on the conjugation with GABARAP. E95 and
I97 severely diminish cross-linking supporting our hypothesis that
these residues are important for β-sheet formation and burying
V108. The hairpin turn binds mainly via electrostatic interactions
to GABARAP. Increasing the salt concentration disrupts this interaction
as shown by reduced fluorescence polarization of LIRATG3 peptide upon binding to GABARAP and cross-linking with ATG3 (Figure S13). These data highlight how binding
of LIRATG3 to GABARAP/LC3A is driven by a combination of
hydrophobic and ionic interactions.Figure 4Dissection of interactions between GABARAP
and LIRATG3 by alanine scanning. GABARAP probe 2 (15 μM)
was reacted with ATG3 C1 alanine variants (15 μM) for 30 min.
Coomassie Blue stained SDS-PAGE gel showing modification of ATG3 C1
variants with GABARAP methyl fumarate. The indicated ATG3 residues
were mutated to alanine. Cross-linking efficiency was estimated by
gel densitometry and normalized to the reaction with ATG3 C1. Data
are presented as average values ± s.d. n = 2
independent experiments. Full-gel images are available in the Supplementary Information.

Cocrystal Structure of GABARAP and LIRATG3

To further corroborate our findings, we cocrystallized GABARAP with
an LIRATG3 peptide (Y90–H112). We solved the cocrystal
structure at a resolution of 2.6 Å and could resolve amino acids
E95–T110 of the LIRATG3 peptide (Table S1, Figure 3a). GABARAP adopted its previously described closed conformation
and showed no structural rearrangements.41 The structure of the LIR peptide and also its interactions with
GABARAP were consistent with respect to the ColabFold prediction with
a backbone-atom root-mean-square deviation of 0.61 Å (Figure 3b). ATG3 W107 binds
to HP1 via hydrophobic interactions in a deep pocket formed by GABARAP
P30, K48, and F104. The binding mode is identical to that observed
for other LIR motifs (Figure 3c). ATG3 V108 is buried by hydrophobic interactions with GABARAP
K46 and Y49 and the upper strand of the LIR β-sheet (E95, I97).
These interactions block any solvent contact of V108 and likely strengthen
its hydrophobic packing. ATG3 D109 forms a surface-exposed salt bridge
with R28. ATG3 T110 binds to HP2 akin to the more conserved Ile, Leu,
and Val residues commonly observed in LIR motifs. The shallow pocket
is formed by GABARAP Y49, V51, P52, L55, and L63. Thr110 likely mimics
the canonical LIR residues, Ile, Leu, and Val, by rotation of its
methyl group to form hydrophobic contacts with HP2, enabling its alcohol
moiety to participate in H-bonding interactions with Pro52. To form
the observed β-sheet conformation, residues N-terminal of W107
form a hairpin turn consisting of glycine and aspartic acid residues,
mainly binding via H-bonding and electrostatic interactions to GABARAP.
ATG3 D104 binds to H9 via a hydrogen bond. Aspartic and glutamic acid
residues E99, E100, D101, and D102 form salt bridges with various
basic residues, K20, K24, and K48 (Figure 3d). Following residue E99, a short β-sheet
is formed for which backbone atoms could be resolved until E95. Overall,
the interaction between the ATG3 LIR motif and GABARAP buries 660
Å2.42

Cocrystal structure of
GABARAP and LIRATG3. a. Cocrystal structure
of GABARAP (gray) and LIRATG3 (orange).
Side-chains in contact with LIRATG3 are shown. For clarity,
W107 is colored in red. One copy of the GABARAP-LIRATG3 complex was chosen from the asymmetric unit. Other copies in the
asymmetric unit show similar structures of the complex. b. Zoom-in and overlap of LIRATG3 (orange) with structure
of LIRATG3 predicted by ColabFold (dark blue); backbone-atom
RMSD is 0.61 Å. c. Zoom-in of core LIRATG3 motif binding to hydrophobic pockets HP1 and HP2. H-bonding and
ionic interactions are indicated by dashed lines. d.
Zoom-in of hairpin turn and the interaction of LIRATG3 with
basic residues in GABARAP. Residues E95–D104 have been omitted
for clarity. H-bonding and ionic interactions are indicated by dashed
lines.

Based on the predicted ATG3-GABARAP model and our
cocrystal structure,
we identified residues in the LIR motif involved in the ATG3-GABARAP
interaction. To probe the contribution of LIRATG3 residues
to GABARAP binding we performed an Ala-screen using our previously
established cross-linking assay as the readout (Figure 4). Cross-linking efficiency of probe 2 with ATG3 or its mutants is dependent on the affinity of
the LIR motif. We expressed ATG3 alanine mutants of the identified
residues and reacted them with probe 2. As expected for
LIR motifs, mutation of W107 almost completely abrogated the reaction
of 2 with ATG3. Similarly, V108A significantly reduced
the cross-linking efficiency. This is in agreement with observations
for other LIR motifs showing that the first variable position contributes
significantly to the binding of LIR motifs.43 Intriguingly, mutation of D109 or T110 had almost no effect on reaction
efficiency. Additionally, the majority of the other screened residues
have at least some impact on the conjugation with GABARAP. E95 and
I97 severely diminish cross-linking supporting our hypothesis that
these residues are important for β-sheet formation and burying
V108. The hairpin turn binds mainly via electrostatic interactions
to GABARAP. Increasing the salt concentration disrupts this interaction
as shown by reduced fluorescence polarization of LIRATG3 peptide upon binding to GABARAP and cross-linking with ATG3 (Figure S13). These data highlight how binding
of LIRATG3 to GABARAP/LC3A is driven by a combination of
hydrophobic and ionic interactions.

Dissection of interactions between GABARAP
and LIRATG3 by alanine scanning. GABARAP probe 2 (15 μM)
was reacted with ATG3 C1 alanine variants (15 μM) for 30 min.
Coomassie Blue stained SDS-PAGE gel showing modification of ATG3 C1
variants with GABARAP methyl fumarate. The indicated ATG3 residues
were mutated to alanine. Cross-linking efficiency was estimated by
gel densitometry and normalized to the reaction with ATG3 C1. Data
are presented as average values ± s.d. n = 2
independent experiments. Full-gel images are available in the Supplementary Information.

LIR Motif Affects LC3 Lipidation and Thioester Transfer In CelluloLIR motifs are crucial to the function
of various proteins involved in autophagy including selective autophagy
receptors as well as enzymes involved in LC3 lipidation such as ATG744 or ATG4.12 The LIR
motif in yeast ATG3 was shown to affect Atg8 lipidation in
vitro and also affects the cytoplasm-to-vacuole pathway.25,45To gauge the role of the LIR motif in human ATG3, we generated
a homozygous HEK293T ATG3 knockout (KO) cell line using CRISPR-Cas9.
The knockout was validated by immunoblotting against ATG3 and LC3A/B
showing defective LC3 lipidation (Figure 5a). Upon starvation, rescue with FLAG-ATG3
wild-type led to LC3 lipidation as evidenced by the presence of the
LC3–II band. However, expression of ATG3 C264A or ATG3 ΔLIR
did not induce LC3 lipidation (Figure 5b, Figure S14). This finding
indicates that the LIR motif in human ATG3 is a prerequisite for LC3
lipidation by ATG3 and contrasts with S. cerevisiae, in which the LIR motif had no effect on starvation induced Atg8
lipidation.Figure 5LIRATG3 effects LC3 lipidation and thioester transfer. a. Generation of homozygous knockouts of HEK293T cells using
CRISPR-Cas9 gene editing. Gene editing efficiency was validated by
immunoblotting against ATG3 and LC3A/B. Analysis for two knockout
clones is shown. b. Rescue with FLAG-ATG3 variants in
HEK293T ATG3 – /– cells and autophagy induced by starvation
for 1 h in the presence of chloroquine (40 μM). Lipidation was
assessed by immunoblotting against LC3A/B. As a positive control wild-type
HEK293T cells were starved and analyzed in parallel to ATG3 –
/– cells. c. Coimmunoprecipitation of FLAG-ATG3
variants (wt, C264A, ΔLIR, and W107A). Interaction of ATG3 with
ATG5–ATG12 and ATG16L1 was assessed by immunoblotting. d. Pulse-chase assay of GABARAP transfer from ATG7 to ATG3.
ATG7 was charged with fluorescein-labeled GABARAP. The reaction was
quenched by addition of EDTA, and transfer initiated by addition of
ATG3 variants (wt, C264A, ΔLIR). The reaction was monitored
for the indicated time-points. Thioester-transfer was assessed by
SDS-PAGE and in-gel fluorescence. e. Quantification of
in-gel fluorescence of d. f. Structural
models showing the potential role of LIRATG3 in thioester
transfer from ATG7 to ATG3. ATG3–ATG7 complex was modeled and
overlaid with the previously reported structure (PDB: 4GHL). The Atg7-Atg8
model44 (PDB: 3VH3) was overlaid with structures. For the
post-transfer model, the ATG3–GABARAP model from Figure 1c was used and aligned to ATG3
of 4GHL. g. Detection of ATG3∼LC3A/B thioester in cellulo. FLAG-ATG3 variants were transiently expressed
in HCT116 cells in which endogenous ATG3 was knocked down by CRISPR
inhibition (CRISPRi). Autophagy was induced by starvation in the presence
of chloroquine (40 μM) for 1 h. Cell lysates were prepared
either in the absence or in the presence of β-mercaptoethanol
(β-ME). Samples containing reducing agent were boiled prior
to SDS-PAGE analysis. The presence of a thioester intermediate was
analyzed by anti-FLAG immunoblotting. ATG3∼LC3A/B thioester
was only observed for ATG3 wt. Data are presented as average values
± s.d. and normalized to gel density of ATG3 wt 5 min. n = 3 independent experiments. Full-gel images for a–e are available in the Supplementary Information.

LIR Motif Affects LC3 Lipidation and Thioester Transfer In Cellulo

LIR motifs are crucial to the function
of various proteins involved in autophagy including selective autophagy
receptors as well as enzymes involved in LC3 lipidation such as ATG744 or ATG4.12 The LIR
motif in yeast ATG3 was shown to affect Atg8 lipidation in
vitro and also affects the cytoplasm-to-vacuole pathway.25,45

To gauge the role of the LIR motif in human ATG3, we generated
a homozygous HEK293T ATG3 knockout (KO) cell line using CRISPR-Cas9.
The knockout was validated by immunoblotting against ATG3 and LC3A/B
showing defective LC3 lipidation (Figure 5a). Upon starvation, rescue with FLAG-ATG3
wild-type led to LC3 lipidation as evidenced by the presence of the
LC3–II band. However, expression of ATG3 C264A or ATG3 ΔLIR
did not induce LC3 lipidation (Figure 5b, Figure S14). This finding
indicates that the LIR motif in human ATG3 is a prerequisite for LC3
lipidation by ATG3 and contrasts with S. cerevisiae, in which the LIR motif had no effect on starvation induced Atg8
lipidation.

LIRATG3 effects LC3 lipidation and thioester transfer. a. Generation of homozygous knockouts of HEK293T cells using
CRISPR-Cas9 gene editing. Gene editing efficiency was validated by
immunoblotting against ATG3 and LC3A/B. Analysis for two knockout
clones is shown. b. Rescue with FLAG-ATG3 variants in
HEK293T ATG3 – /– cells and autophagy induced by starvation
for 1 h in the presence of chloroquine (40 μM). Lipidation was
assessed by immunoblotting against LC3A/B. As a positive control wild-type
HEK293T cells were starved and analyzed in parallel to ATG3 –
/– cells. c. Coimmunoprecipitation of FLAG-ATG3
variants (wt, C264A, ΔLIR, and W107A). Interaction of ATG3 with
ATG5–ATG12 and ATG16L1 was assessed by immunoblotting. d. Pulse-chase assay of GABARAP transfer from ATG7 to ATG3.
ATG7 was charged with fluorescein-labeled GABARAP. The reaction was
quenched by addition of EDTA, and transfer initiated by addition of
ATG3 variants (wt, C264A, ΔLIR). The reaction was monitored
for the indicated time-points. Thioester-transfer was assessed by
SDS-PAGE and in-gel fluorescence. e. Quantification of
in-gel fluorescence of d. f. Structural
models showing the potential role of LIRATG3 in thioester
transfer from ATG7 to ATG3. ATG3–ATG7 complex was modeled and
overlaid with the previously reported structure (PDB: 4GHL). The Atg7-Atg8
model44 (PDB: 3VH3) was overlaid with structures. For the
post-transfer model, the ATG3–GABARAP model from Figure 1c was used and aligned to ATG3
of 4GHL. g. Detection of ATG3∼LC3A/B thioester in cellulo. FLAG-ATG3 variants were transiently expressed
in HCT116 cells in which endogenous ATG3 was knocked down by CRISPR
inhibition (CRISPRi). Autophagy was induced by starvation in the presence
of chloroquine (40 μM) for 1 h. Cell lysates were prepared
either in the absence or in the presence of β-mercaptoethanol
(β-ME). Samples containing reducing agent were boiled prior
to SDS-PAGE analysis. The presence of a thioester intermediate was
analyzed by anti-FLAG immunoblotting. ATG3∼LC3A/B thioester
was only observed for ATG3 wt. Data are presented as average values
± s.d. and normalized to gel density of ATG3 wt 5 min. n = 3 independent experiments. Full-gel images for a–e are available in the Supplementary Information.

How Does LIRATG3 Influence ATG3’s Activity?We investigated the interaction of ATG3 with components of its
enzymatic cascade by coimmunoprecipitation. We transiently expressed
FLAG-ATG3 variants in HCT116 ATG3 knock-down cells generated using
CRISPRi (Figure S15) and assessed their
interaction with E3 complex subunits by anti-FLAG Co-IP.46 ATG3 wt, C264A, ΔLIR, and W107A precipitated
the ATG5-ATG12 complex and ATG16L1 with similar efficiency (Figure 5c). Therefore, LIRATG3 does not affect lipidation through defective binding to
the E3 enzyme complex. Likewise, we performed pull-down of recombinantly
expressed ATG3 variants with FLAG-ATG7 immobilized on anti-FLAG resin.
No difference in pull-down efficiency was observed (Figure S16). This indicates that LIRATG3 or the
lack thereof does not impact binding of ATG3 to ATG7 via its ATG7
interacting region (RIA7).47In order
to assess the influence of LIRATG3 on thioester transfer
from ATG7 to ATG3 we performed a pulse-chase assay. Fluorescein-labeled
GABARAP was charged onto ATG7 and thioester transfer initiated by
addition of ATG3 variants. We observed a significant reduction in
transfer rate between ATG3 wt and ΔLIR (Figure 5d,e). This observation contrasts with S. cerevisiae Atg3 in which the LIR motif had no effect
on Atg8 transfer. In order to understand the origin of this rate defect
we modeled ATG3 onto the E1 enzyme ATG7 using ColabFold. We correctly
predicted RIA7 of ATG3 and its interaction with ATG7.47 The predicted structure is in good agreement with a previously
reported structure of the ATG7–ATG3 complex.48 Overlaying the predicted structure with a reported structure
of Atg8 bound to Atg7 places LIRATG3 in proximity (20–30
Å) of the LIR binding region of LC3/Atg8. Based on this proximity
we speculate that the LIR motif aids ATG3 in abstracting LC3 from
ATG7 by facilitating thioester exchange (Figure 5f). LC3 can undergo a 180° rotation
around its C-terminus to adopt the closed conformation observed in
our ColabFold prediction (Figure 1c).To confirm these in vitro results in cellulo, we expressed FLAG-ATG3 variants
in HEK293T cells to analyze the
presence of the ATG3∼LC3A/B thioester complex formation. Cell
lysis under reducing or nonreducing conditions showed the presence
of a thiol-sensitive band for wild-type ATG3. This band corresponds
to the ATG3∼LC3 thioester complex, as no thioester was detected
for ATG3 C264A (Figure 5g). With ATG3 ΔLIR no higher MW thiol-sensitive band was detected
either. This observation suggests that LIRATG3 influences
ATG3∼LC3A/B complex beyond a defect in its formation potentially
due to complex stability.

How Does LIRATG3 Influence ATG3’s Activity?

We investigated the interaction of ATG3 with components of its
enzymatic cascade by coimmunoprecipitation. We transiently expressed
FLAG-ATG3 variants in HCT116 ATG3 knock-down cells generated using
CRISPRi (Figure S15) and assessed their
interaction with E3 complex subunits by anti-FLAG Co-IP.46 ATG3 wt, C264A, ΔLIR, and W107A precipitated
the ATG5-ATG12 complex and ATG16L1 with similar efficiency (Figure 5c). Therefore, LIRATG3 does not affect lipidation through defective binding to
the E3 enzyme complex. Likewise, we performed pull-down of recombinantly
expressed ATG3 variants with FLAG-ATG7 immobilized on anti-FLAG resin.
No difference in pull-down efficiency was observed (Figure S16). This indicates that LIRATG3 or the
lack thereof does not impact binding of ATG3 to ATG7 via its ATG7
interacting region (RIA7).47

In order
to assess the influence of LIRATG3 on thioester transfer
from ATG7 to ATG3 we performed a pulse-chase assay. Fluorescein-labeled
GABARAP was charged onto ATG7 and thioester transfer initiated by
addition of ATG3 variants. We observed a significant reduction in
transfer rate between ATG3 wt and ΔLIR (Figure 5d,e). This observation contrasts with S. cerevisiae Atg3 in which the LIR motif had no effect
on Atg8 transfer. In order to understand the origin of this rate defect
we modeled ATG3 onto the E1 enzyme ATG7 using ColabFold. We correctly
predicted RIA7 of ATG3 and its interaction with ATG7.47 The predicted structure is in good agreement with a previously
reported structure of the ATG7–ATG3 complex.48 Overlaying the predicted structure with a reported structure
of Atg8 bound to Atg7 places LIRATG3 in proximity (20–30
Å) of the LIR binding region of LC3/Atg8. Based on this proximity
we speculate that the LIR motif aids ATG3 in abstracting LC3 from
ATG7 by facilitating thioester exchange (Figure 5f). LC3 can undergo a 180° rotation
around its C-terminus to adopt the closed conformation observed in
our ColabFold prediction (Figure 1c).

To confirm these in vitro results in cellulo, we expressed FLAG-ATG3 variants
in HEK293T cells to analyze the
presence of the ATG3∼LC3A/B thioester complex formation. Cell
lysis under reducing or nonreducing conditions showed the presence
of a thiol-sensitive band for wild-type ATG3. This band corresponds
to the ATG3∼LC3 thioester complex, as no thioester was detected
for ATG3 C264A (Figure 5g). With ATG3 ΔLIR no higher MW thiol-sensitive band was detected
either. This observation suggests that LIRATG3 influences
ATG3∼LC3A/B complex beyond a defect in its formation potentially
due to complex stability.

ConclusionsUsing a combination of chemical, biochemical,
computational, and
structural methods we have identified a previously unknown LIR motif
in human ATG3. The LIR motif is distinct from a previously reported
LIR motif in S. cerevisiae Atg3 (LIRSC), which occurs in a completely different part of its homologue.
Sequence alignment shows that the yeast LIR motif was lost and replaced
by the LIR motif described herein. These findings suggest an evolutionary
advantage of the human LIR (LIRHS) motif over LIRSC. The conservation of LIRHS among other species including
most fungi suggests that S. cerevisiae is a poor
model to derive general conclusions on the enzymatic function of ATG3
across different species.The core LIR motif, WVDT, is noncanonical
and embedded in an unusual
β-sheet conformation. Amino acids beyond the core LIR motif
and the β-sheet conformation are involved in ATG3 binding to
LC3/GABARAP. This uncommon β-sheet conformation for LIR motifs
has only been reported for a few cases, including the FNIP tumor suppressor,33 the Legionella effector protein
RavZ.32 A cursory search of reported LIR
structures suggests that ALFY likely contains a β-sheet embedded
LIR motif.49 We provide conclusive evidence
that the bent β-sheet conformation for LIRATG3 is
required for efficient binding to LC3/GABARAP and is not an artifact
of cocrystallizing LIR peptide and LC3/GABARAP. This finding suggests
that bent LIR conformations may be more prevalent than currently appreciated.
It is likely that a more widespread identification of this conformation
has been precluded to this date by investigating LIR motifs in structural
studies that were truncated in their N-terminal region and could not
form the β-sheet conformation. This information will be useful
in guiding bioinformatics methods to identify previously unidentified
LIR motifs and extending the realm of LIR structures.In addition,
we show that LIRATG3 is required for LC3
lipidation in cellulo and effects efficient thioester
transfer from ATG7 to ATG3. While we observed no requirement of LIRATG3 for interaction with either ATG5–ATG12, ATG16L1,
or ATG7, the LIR motif influences the catalytic activity of the complex.
ATG3’s flexible region contains not only the LIR motif but
also two additional amino acid stretches required for ATG7 interaction,
RIA7, and ATG12, RIA12. RIA7 and RIA12 partially overlap and are therefore
mutually exclusive in their binding. It is probable that LIRATG3 facilitates or influences binding of either region to their respective
binding partner and therefore drives the lipidation reaction forward.
It is also plausible that LIRATG3 binds to and blocks the
LIR-binding region of LC3/GABARAP from binding to the plethora of
proteins bearing LIR motifs during its transfer from ATG7 to ATG3
to its substrate lipid. Interaction of LC3/GABARAP with one of these
effector proteins during its transfer would likely negatively impact
the efficiency of the lipidation reaction and stall efficient autophagosome
expansion.

Conclusions

Using a combination of chemical, biochemical,
computational, and
structural methods we have identified a previously unknown LIR motif
in human ATG3. The LIR motif is distinct from a previously reported
LIR motif in S. cerevisiae Atg3 (LIRSC), which occurs in a completely different part of its homologue.
Sequence alignment shows that the yeast LIR motif was lost and replaced
by the LIR motif described herein. These findings suggest an evolutionary
advantage of the human LIR (LIRHS) motif over LIRSC. The conservation of LIRHS among other species including
most fungi suggests that S. cerevisiae is a poor
model to derive general conclusions on the enzymatic function of ATG3
across different species.

The core LIR motif, WVDT, is noncanonical
and embedded in an unusual
β-sheet conformation. Amino acids beyond the core LIR motif
and the β-sheet conformation are involved in ATG3 binding to
LC3/GABARAP. This uncommon β-sheet conformation for LIR motifs
has only been reported for a few cases, including the FNIP tumor suppressor,33 the Legionella effector protein
RavZ.32 A cursory search of reported LIR
structures suggests that ALFY likely contains a β-sheet embedded
LIR motif.49 We provide conclusive evidence
that the bent β-sheet conformation for LIRATG3 is
required for efficient binding to LC3/GABARAP and is not an artifact
of cocrystallizing LIR peptide and LC3/GABARAP. This finding suggests
that bent LIR conformations may be more prevalent than currently appreciated.
It is likely that a more widespread identification of this conformation
has been precluded to this date by investigating LIR motifs in structural
studies that were truncated in their N-terminal region and could not
form the β-sheet conformation. This information will be useful
in guiding bioinformatics methods to identify previously unidentified
LIR motifs and extending the realm of LIR structures.

In addition,
we show that LIRATG3 is required for LC3
lipidation in cellulo and effects efficient thioester
transfer from ATG7 to ATG3. While we observed no requirement of LIRATG3 for interaction with either ATG5–ATG12, ATG16L1,
or ATG7, the LIR motif influences the catalytic activity of the complex.
ATG3’s flexible region contains not only the LIR motif but
also two additional amino acid stretches required for ATG7 interaction,
RIA7, and ATG12, RIA12. RIA7 and RIA12 partially overlap and are therefore
mutually exclusive in their binding. It is probable that LIRATG3 facilitates or influences binding of either region to their respective
binding partner and therefore drives the lipidation reaction forward.
It is also plausible that LIRATG3 binds to and blocks the
LIR-binding region of LC3/GABARAP from binding to the plethora of
proteins bearing LIR motifs during its transfer from ATG7 to ATG3
to its substrate lipid. Interaction of LC3/GABARAP with one of these
effector proteins during its transfer would likely negatively impact
the efficiency of the lipidation reaction and stall efficient autophagosome
expansion.
