---
pmid: '23942337'
title: Structural basis for the modulation of the neuronal voltage-gated sodium channel
  NaV1.6 by calmodulin.
authors:
- Reddy Chichili VP
- Xiao Y
- Seetharaman J
- Cummins TR
- Sivaraman J
journal: Sci Rep
year: '2013'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3743062
doi: 10.1038/srep02435
---

# Structural basis for the modulation of the neuronal voltage-gated sodium channel NaV1.6 by calmodulin.
**Authors:** Reddy Chichili VP, Xiao Y, Seetharaman J, Cummins TR, Sivaraman J
**Journal:** Sci Rep (2013)
**DOI:** [10.1038/srep02435](https://doi.org/10.1038/srep02435)
**PMC:** [PMC3743062](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3743062/)

## Abstract

1. Sci Rep. 2013;3:2435. doi: 10.1038/srep02435.

Structural basis for the modulation of the neuronal voltage-gated sodium channel 
NaV1.6 by calmodulin.

Reddy Chichili VP(1), Xiao Y, Seetharaman J, Cummins TR, Sivaraman J.

Author information:
(1)Department of Biological Sciences, National University of Singapore, 
Singapore, 117543.

The neuronal-voltage gated sodium channel (VGSC), Na(V)1.6, plays an important 
role in propagating action potentials along myelinated axons. Calmodulin (CaM) 
is known to modulate the inactivation kinetics of Na(V)1.6 by interacting with 
its IQ motif. Here we report the crystal structure of apo-CaM:Na(V)1.6IQ motif, 
along with functional studies. The IQ motif of Na(V)1.6 adopts an α-helical 
conformation in its interaction with the C-lobe of CaM. CaM uses different 
residues to interact with Na(V)1.6IQ motif depending on the presence or absence 
of Ca²⁺. Three residues from Na(V)1.6, Arg1902, Tyr1904 and Arg1905 were 
identified as the key common interacting residues in both the presence and 
absence of Ca²⁺. Substitution of Arg1902 and Tyr1904 with alanine showed a 
reduced rate of Na(V)1.6 inactivation in electrophysiological experiments in 
vivo. Compared with other CaM:Na(V) complexes, our results reveal a different 
mode of interaction for CaM:Na(V)1.6 and provides structural insight into the 
isoform-specific modulation of VGSCs.

DOI: 10.1038/srep02435
PMCID: PMC3743062
PMID: 23942337 [Indexed for MEDLINE]

## Full Text

Voltage-gated ion channels play a crucial role in the spread of action potentials by producing electrical signals in neurons and other excitable cells1. Voltage-gated sodium channels (VGSCs) are especially responsible for excitability in neuronal, neuroendocrine, skeletal muscle and cardiac cells. VGSCs are transmembrane proteins of approximately 260 kDa and consist of a central, pore-forming functional α subunit (VGSCα) and one or more auxiliary β subunits (VGSCβ)2. VGSCs are known to exist in three basic states: deactivated (closed), activated (open) and inactivated (closed). In the absence of external stimuli, VSGCs are found in the deactivated state and become opened (activated) upon receiving signals, which leads to the propagation of the action potential. Once activated, the VGSCs rapidly return to an inactivated state, during which another signal cannot be evoked3. Thus, activation and inactivation kinetics of VGSCs play a key role in normal nerve function and muscle contraction.

Among all the VGSCs, NaV1.1, NaV1.2 and NaV1.6 are predominantly found in the central nervous system (CNS). NaV1.6 channels are particularly crucial channels that exist at certain axonal sites, such as the nodes of Ranvier and axon initial segments (ISs), where action potentials are generated45. Proper function of NaV1.6 is important since its loss or disruption in mice leads to symptoms of motor dysfunction, dystonia, paralysis, abnormalities in retinal pathways and even juvenile lethality67. A recent study on the differences between NaV1.1, NaV1.2 and NaV1.6 has shown that NaV1.6 channels are more suited for repetitive or high-frequency firing8. Repetitive firing is required when the postsynaptic and presynaptic regions of the neuron are separated by a few millimeters. In a study designed to investigate the role of NaV1.6 in repetitive firing in Purkinje cells, it was shown that mice lacking NaV1.6 channels exhibit loss of spontaneous firing, diminished repetitive firing abilities and cell degeneration9. Moreover, Herzog et al. reported that NaV1.6 exhibits faster recovery from inactivation when compared with other VGSC isoforms, which helps in achieving repetitive firing10. Thus, NaV1.6 plays an important role in propagating action potentials along the length of myelinated axons.

Calmodulin (CaM) is a ubiquitous protein that binds to many target proteins and regulates a number of cellular functions. CaM is known to interact with several of its binding partners via an isoleucine-glutamine (IQ) motif11. The general core region of the IQ motif is typically [I/L/V]QXXXRGXXX[R/K] and interactions mediated by IQ motifs can be either calcium-dependent or -independent11. The carboxyl terminus of all VGSC isoforms (NaV 1.1–1.9) possesses an IQ motif that is conserved to various degrees and recognized by CaM. Indeed, several reports have shown that CaM is able to bind to and modulate the activation and steady-state inactivation of various VGSCs via this IQ motif in an isoform-dependent manner1213141516171819.

Mutations in the core region of NaV1.6 IQ motif have been shown to cause reduced binding with CaM as well as reduced peak sodium current in the absence of Ca2+
13. Changes in the intracellular Ca2+ concentration are known to alter the inactivation kinetics of NaV1.6 currents in a CaM-dependent mechanism13. Ca2+/CaM has been shown to delay NaV1.6 channel inactivation by 50% when compared with apo-CaM (Ca2+ free) induced rate of inactivation. Ca2+/CaM-dependent slowing of NaV1.6 inactivation kinetics could prolong action potential duration to enhance neurotransmitter release at nerve endings and thus regulate synaptic plasticity13. In order to determine how NaV1.6 is regulated by CaM, it is necessary to understand the interaction between these two proteins.

In this study, we report the structural basis for the interaction of NaV1.6IQ motif with CaM in the presence and absence of Ca2+. The crystal structure of the apo-CaM-NaV1.6 IQ motif complex revealed a different mode of interaction between CaM and NaV1.6 IQ motif as compared with other CaM-NaV IQ motif complexes. We also show that CaM employs different modes of interaction to bind the NaV1.6 IQ motif in the presence and absence of Ca2+, with enhanced affinity observed in the presence of Ca2+. The crystal structure also revealed the importance of several key residues in mediating the interaction, and structure-based mutational studies were used to validate these interactions that were dependent on the presence and absence of Ca2+. A comparison of the electrophysiological properties between these mutants and wild-type NaV1.6 further validated the role of these interacting residues in modulating the inactivation kinetics of NaV1.6.

ResultsSequence analysisSequence alignment of the IQ motifs of various isoforms of VGSCs reveals significant sequence similarity (Figure 1). The IQ motif of NaV1.6 possesses Leu at position 1 of the IQ motif, whereas this position contains an Ile for the IQ motifs of the other isoforms. The NaV1.6 IQ motif possesses Gly at position 7, whereas Arg is conserved at this position in many of the VGSC isoforms. This particular difference between IQ motifs has been known to represent two different functional classes of IQ domains20. About 50% of IQ domains are known to have a Gly residue at position 7 instead of bulky amino acid and this substitution may provide some specificity for CaM interactions21. To further understand the NaV1.6 interaction with CaM, the NaV1.6 IQ motif peptide was selected based on the analysis of structurally known CaM:IQ motif complexes22. This led us to select 1891–1914 aa of NaV1.6 from Mus musculus as an appropriate NaV1.6 IQ motif peptide for the biophysical interaction studies using ITC experiments.Isothermal Titration Calorimetry (ITC)Interactions between CaM and the NaV1.6 IQ motif peptide were studied in the presence and absence of Ca2+ (Table 1). The NaV1.6 IQ motif peptide bound to CaM in a 1:1 ratio in the presence and absence of Ca2+ (Figure 2). The negative Gibbs free energy change for CaM-NaV1.6 IQ interactions, in the presence and absence of Ca2+ indicated that all the interactions were thermodynamically favorable (Table 1). Moreover, the binding affinity for CaM with NaV1.6 IQ motif peptide was enhanced in the presence of Ca2+. It is possible that the mode of interaction between the IQ motif and CaM varies depending on the presence (Ca2+ bound) or absence (apo) of Ca2+
23. To better understand the structural basis for the difference in these interactions, we determined the crystal structure of CaM-NaV1.6 IQ motif peptide complex.Structure of apo-CaM-(Gly)5-NaV1.6 IQ motif peptide complexBrief attempts to co-crystallize NaV1.6 IQ motif peptide with CaM did not yield complex crystals. This led us to adopt the linker method, which has been described earlier22242526. Initially, residues 1891–1914 of NaV1.6 were linked to the C-terminus of CaM using a (Gly)5 linker. However, the crystals obtained were not of good quality. To optimize the fused construct, we instead used residues 1893–1914 of NaV1.6 and obtained much better diffraction-quality crystals. The initial phases were obtained using SelMet-labeled crystals. The final model was refined with a native data set up to 1.95Å resolution with an R value of 0.201 (Rfree = 0.236) (Table 2).Our findings show that CaM exists in an extended conformation, with no electron density observed in any of the divalent metal ion binding sites. The helices of EF-hand motifs are parallel to each other, indicating the absence of Ca2+. The electron density for residues 1893–1913 of NaV1.6 IQ peptide is also well defined (Figure 3). Figure 4 shows that the bound NaV1.6 IQ peptide assumes an α-helical structure and adopts an almost perpendicular orientation to the central helix of CaM, making several contacts with the C-lobe of CaM.The C-lobe of CaM appears to be in a “semi-open” conformation, which would allow the IQ motif to interact. The N-lobe of CaM is not interacting with the IQ motif, remaining in a “closed” conformation. The interaction between CaM and NaV1.6 IQ peptide is mainly electrostatic in nature. We identified the key residues of the NaV1.6 IQ peptide involved in this interaction as Gln1901, Arg1902, Tyr1904, Arg1905, His1907 and Arg1911 (Figure 4). Leu1900 (at position 1) of the NaV1.6 IQ motif peptide is buried in the hydrophobic cluster formed by Ile85, Phe89, Phe92, Leu105 and Met109 of CaM, with a buried area of 110 Å2. The side-chain of Gln 1901 (position 2) has established hydrogen bonds with the main chain atoms of Met110, Leu113 and Glu115 of CaM, and is buried in the pocket formed by these interacting residues from CaM. Tyr1904 at position 5 is buried in the hydrophobic cluster formed by Leu116, Met125, Phe141 and Met146 from CaM, with a buried area of 139 Å2. The hydroxyl group of Tyr1904 at position 5 forms a hydrogen bonding contact with Glu128 of CaM. Similarly, the side-chains of Arg1902, Arg1905, His1907 and Arg1911 from NaV1.6 interact with the side-chains of Glu115, Glu121, Glu124 and Glu128 from CaM via hydrogen bonds. Altogether, NaV1.6 IQ motif was shown to make14 hydrogen bond contacts (<3.2 Å) with the C-lobe of CaM (Supplementary Table 1).Comparison with other CaM-NaV IQ motif complexesThe NMR structures of apo-CaM C-lobe-NaV1.2 IQ motif complex and apo-CaM-NaV1.5 IQ motif complex have been reported2327. The structure of the CaM-NaV1.6 IQ motif peptide complex, particularly the C-lobe, was superimposed with the equivalent region of NaV1.2 and NaV1.5 IQ motif peptide complexes, with an rmsd of 1.3 Å and 0.76 Å for 59 and 58 Cα atoms, respectively. The interactions between the apo-C-lobe CaM and NaV1.2 IQ motif complex were reported to be mainly hydrophobic in nature23.The one-to-one comparison of NaV1.2 and NaV1.6 IQ motif complex structures in the absence of Ca2+ is outlined in Figure 5A–C. It can be seen that the Arg residues at positions 3 and 6 of NaV1.6 IQ motif make hydrogen bonding contacts with Glu115 and Glu121 of CaM, respectively (Figure 5A), whereas the same Arg residues of NaV1.2 IQ motif do not interact with CaM (Figure 5B). Moreover, the orientation of the Tyr residues at position 5 of NaV1.6 IQ motif and NaV1.2 IQ motif are different; this is thought to allow the hydroxyl group of the Tyr residue in NaV1.6 IQ motif to make hydrogen bonding contacts with Glu128 of CaM (Figure 5A); however, the same is not observed for the NaV1.2 IQ motif (Figure 5B). The His residue at position 8 of NaV1.6 IQ motif makes a hydrogen bonding contact with Glu128 of CaM, but this residue is replaced by a Tyr residue in the NaV1.2 IQ motif, which is buried in the hydrophobic pocket formed by various hydrophobic residues of C-lobe of CaM; this causes the sidechain of Glu128 of CaM to be oriented away from the Tyr at position 8 of NaV1.2 (Figure 5B).The NMR structure of apo-CaM and NaV1.5 IQ motif revealed an interaction mainly driven by hydrophobic forces and, to some extent, electrostatic interactions. Arg at position 3 of NaV1.5 and NaV1.6 IQ motif are involved in hydrogen bonding contact with CaM, but with different residues (Figure 5C). The Tyr at position 5 of NaV1.6 IQ motif is replaced by Phe in the NaV1.5 IQ motif, and has no hydrogen bonding contact with CaM (Figure 5C).Structure-based mutational studiesBased on the crystal structure of CaM-NaV1.6 IQ motif complex, we selected key interacting residues of CaM (Glu115, Glu121, Glu124, and Glu128) and NaV1.6 IQ motif (Gln1901, Arg1902, Tyr1904, and Arg1905) and mutated them to alanine using site-directed mutagenesis. We then compared the binding of these unlinked CaM and NaV1.6 IQ motifs using ITC to determine the importance of these key residues in mediating binding. We found that mutating any single residue either on CaM or NaV1.6 IQ motif peptide abolished the binding in the absence of Ca2+(Table 1 and Supplementary Figure 1). Similar losses of interaction between proteins and peptides have been previously reported following mutation of key interacting residues28293031.Next, we sought to examine the role of apo-CaM interacting residues of NaV1.6 IQ motif in recognizing Ca2+-bound CaM using ITC. We found that single mutations in the NaV1.6 IQ motif peptide led to reduced binding with Ca2+/CaM. Of the four NaV1.6 mutants, Y1904A showed the most significant reduction in binding affinity (Table 1), followed by R1902A (Table 1 and Supplementary Figure 1). Since the conformation of CaM might be different in the presence of Ca2+, it is likely that different residues would be exposed on the surface of CaM to interact with IQ motifs32. We therefore made brief attempts to crystallize Ca2+/CaM-NaV1.6 IQ motif peptide complex. However, we did not obtain any crystals. We then assessed the role of Ca2+ in regulating the interaction between mutated CaM and WT NaV1.6 IQ motif in the presence of Ca2+. These mutants of CaM did not show any reduction in binding affinity in the presence of Ca2+ (Table 1), suggesting that the NaV1.6 IQ motif is recognized by different residues of CaM in the presence and absence of Ca2+.Reduced rate of inactivation by R1902A and Y1904A of NaV1.6Finally, we sought to compare the electrophysiological properties of mutating two key residues (R1902A and Y1904A) of NaV1.6 channels with WT NaV1.6 channels. Both WT and mutant channels were expressed in ND7/23 cells and studied under conditions of minimal (essentially zero) intracellular calcium. The two mutations significantly slowed the rate of the inactivation at potentials ranging from 0 to +40 mV (Figure 6A). At +20 mV, R1902A and Y1904A slowed the rate of fast inactivation of Nav1.6 channels by 31% and 48%, respectively. The slower rate of fast inactivation is not caused by the shift in the voltage dependence of channel gating. As can be seen in Supplementary Figure 2, neither R1902A nor Y1904A altered the steady-state activation of Nav1.6 channels. The two mutations also did not significantly alter the steady-state inactivation of NaV1.6 channels (Supplementary Figure 2). Consistent with our previous finding that the IQ motif is an important determinant of current amplitude13, the R1902A and Y1904A mutations decreased current density of NaV1.6 channels by 42.4% and 73.8%, respectively (Figure 6B).

Sequence analysisSequence alignment of the IQ motifs of various isoforms of VGSCs reveals significant sequence similarity (Figure 1). The IQ motif of NaV1.6 possesses Leu at position 1 of the IQ motif, whereas this position contains an Ile for the IQ motifs of the other isoforms. The NaV1.6 IQ motif possesses Gly at position 7, whereas Arg is conserved at this position in many of the VGSC isoforms. This particular difference between IQ motifs has been known to represent two different functional classes of IQ domains20. About 50% of IQ domains are known to have a Gly residue at position 7 instead of bulky amino acid and this substitution may provide some specificity for CaM interactions21. To further understand the NaV1.6 interaction with CaM, the NaV1.6 IQ motif peptide was selected based on the analysis of structurally known CaM:IQ motif complexes22. This led us to select 1891–1914 aa of NaV1.6 from Mus musculus as an appropriate NaV1.6 IQ motif peptide for the biophysical interaction studies using ITC experiments.

Sequence analysis

Sequence alignment of the IQ motifs of various isoforms of VGSCs reveals significant sequence similarity (Figure 1). The IQ motif of NaV1.6 possesses Leu at position 1 of the IQ motif, whereas this position contains an Ile for the IQ motifs of the other isoforms. The NaV1.6 IQ motif possesses Gly at position 7, whereas Arg is conserved at this position in many of the VGSC isoforms. This particular difference between IQ motifs has been known to represent two different functional classes of IQ domains20. About 50% of IQ domains are known to have a Gly residue at position 7 instead of bulky amino acid and this substitution may provide some specificity for CaM interactions21. To further understand the NaV1.6 interaction with CaM, the NaV1.6 IQ motif peptide was selected based on the analysis of structurally known CaM:IQ motif complexes22. This led us to select 1891–1914 aa of NaV1.6 from Mus musculus as an appropriate NaV1.6 IQ motif peptide for the biophysical interaction studies using ITC experiments.

Isothermal Titration Calorimetry (ITC)Interactions between CaM and the NaV1.6 IQ motif peptide were studied in the presence and absence of Ca2+ (Table 1). The NaV1.6 IQ motif peptide bound to CaM in a 1:1 ratio in the presence and absence of Ca2+ (Figure 2). The negative Gibbs free energy change for CaM-NaV1.6 IQ interactions, in the presence and absence of Ca2+ indicated that all the interactions were thermodynamically favorable (Table 1). Moreover, the binding affinity for CaM with NaV1.6 IQ motif peptide was enhanced in the presence of Ca2+. It is possible that the mode of interaction between the IQ motif and CaM varies depending on the presence (Ca2+ bound) or absence (apo) of Ca2+
23. To better understand the structural basis for the difference in these interactions, we determined the crystal structure of CaM-NaV1.6 IQ motif peptide complex.

Isothermal Titration Calorimetry (ITC)

Interactions between CaM and the NaV1.6 IQ motif peptide were studied in the presence and absence of Ca2+ (Table 1). The NaV1.6 IQ motif peptide bound to CaM in a 1:1 ratio in the presence and absence of Ca2+ (Figure 2). The negative Gibbs free energy change for CaM-NaV1.6 IQ interactions, in the presence and absence of Ca2+ indicated that all the interactions were thermodynamically favorable (Table 1). Moreover, the binding affinity for CaM with NaV1.6 IQ motif peptide was enhanced in the presence of Ca2+. It is possible that the mode of interaction between the IQ motif and CaM varies depending on the presence (Ca2+ bound) or absence (apo) of Ca2+
23. To better understand the structural basis for the difference in these interactions, we determined the crystal structure of CaM-NaV1.6 IQ motif peptide complex.

Structure of apo-CaM-(Gly)5-NaV1.6 IQ motif peptide complexBrief attempts to co-crystallize NaV1.6 IQ motif peptide with CaM did not yield complex crystals. This led us to adopt the linker method, which has been described earlier22242526. Initially, residues 1891–1914 of NaV1.6 were linked to the C-terminus of CaM using a (Gly)5 linker. However, the crystals obtained were not of good quality. To optimize the fused construct, we instead used residues 1893–1914 of NaV1.6 and obtained much better diffraction-quality crystals. The initial phases were obtained using SelMet-labeled crystals. The final model was refined with a native data set up to 1.95Å resolution with an R value of 0.201 (Rfree = 0.236) (Table 2).Our findings show that CaM exists in an extended conformation, with no electron density observed in any of the divalent metal ion binding sites. The helices of EF-hand motifs are parallel to each other, indicating the absence of Ca2+. The electron density for residues 1893–1913 of NaV1.6 IQ peptide is also well defined (Figure 3). Figure 4 shows that the bound NaV1.6 IQ peptide assumes an α-helical structure and adopts an almost perpendicular orientation to the central helix of CaM, making several contacts with the C-lobe of CaM.The C-lobe of CaM appears to be in a “semi-open” conformation, which would allow the IQ motif to interact. The N-lobe of CaM is not interacting with the IQ motif, remaining in a “closed” conformation. The interaction between CaM and NaV1.6 IQ peptide is mainly electrostatic in nature. We identified the key residues of the NaV1.6 IQ peptide involved in this interaction as Gln1901, Arg1902, Tyr1904, Arg1905, His1907 and Arg1911 (Figure 4). Leu1900 (at position 1) of the NaV1.6 IQ motif peptide is buried in the hydrophobic cluster formed by Ile85, Phe89, Phe92, Leu105 and Met109 of CaM, with a buried area of 110 Å2. The side-chain of Gln 1901 (position 2) has established hydrogen bonds with the main chain atoms of Met110, Leu113 and Glu115 of CaM, and is buried in the pocket formed by these interacting residues from CaM. Tyr1904 at position 5 is buried in the hydrophobic cluster formed by Leu116, Met125, Phe141 and Met146 from CaM, with a buried area of 139 Å2. The hydroxyl group of Tyr1904 at position 5 forms a hydrogen bonding contact with Glu128 of CaM. Similarly, the side-chains of Arg1902, Arg1905, His1907 and Arg1911 from NaV1.6 interact with the side-chains of Glu115, Glu121, Glu124 and Glu128 from CaM via hydrogen bonds. Altogether, NaV1.6 IQ motif was shown to make14 hydrogen bond contacts (<3.2 Å) with the C-lobe of CaM (Supplementary Table 1).

Structure of apo-CaM-(Gly)5-NaV1.6 IQ motif peptide complex

Brief attempts to co-crystallize NaV1.6 IQ motif peptide with CaM did not yield complex crystals. This led us to adopt the linker method, which has been described earlier22242526. Initially, residues 1891–1914 of NaV1.6 were linked to the C-terminus of CaM using a (Gly)5 linker. However, the crystals obtained were not of good quality. To optimize the fused construct, we instead used residues 1893–1914 of NaV1.6 and obtained much better diffraction-quality crystals. The initial phases were obtained using SelMet-labeled crystals. The final model was refined with a native data set up to 1.95Å resolution with an R value of 0.201 (Rfree = 0.236) (Table 2).

Our findings show that CaM exists in an extended conformation, with no electron density observed in any of the divalent metal ion binding sites. The helices of EF-hand motifs are parallel to each other, indicating the absence of Ca2+. The electron density for residues 1893–1913 of NaV1.6 IQ peptide is also well defined (Figure 3). Figure 4 shows that the bound NaV1.6 IQ peptide assumes an α-helical structure and adopts an almost perpendicular orientation to the central helix of CaM, making several contacts with the C-lobe of CaM.

The C-lobe of CaM appears to be in a “semi-open” conformation, which would allow the IQ motif to interact. The N-lobe of CaM is not interacting with the IQ motif, remaining in a “closed” conformation. The interaction between CaM and NaV1.6 IQ peptide is mainly electrostatic in nature. We identified the key residues of the NaV1.6 IQ peptide involved in this interaction as Gln1901, Arg1902, Tyr1904, Arg1905, His1907 and Arg1911 (Figure 4). Leu1900 (at position 1) of the NaV1.6 IQ motif peptide is buried in the hydrophobic cluster formed by Ile85, Phe89, Phe92, Leu105 and Met109 of CaM, with a buried area of 110 Å2. The side-chain of Gln 1901 (position 2) has established hydrogen bonds with the main chain atoms of Met110, Leu113 and Glu115 of CaM, and is buried in the pocket formed by these interacting residues from CaM. Tyr1904 at position 5 is buried in the hydrophobic cluster formed by Leu116, Met125, Phe141 and Met146 from CaM, with a buried area of 139 Å2. The hydroxyl group of Tyr1904 at position 5 forms a hydrogen bonding contact with Glu128 of CaM. Similarly, the side-chains of Arg1902, Arg1905, His1907 and Arg1911 from NaV1.6 interact with the side-chains of Glu115, Glu121, Glu124 and Glu128 from CaM via hydrogen bonds. Altogether, NaV1.6 IQ motif was shown to make14 hydrogen bond contacts (<3.2 Å) with the C-lobe of CaM (Supplementary Table 1).

Comparison with other CaM-NaV IQ motif complexesThe NMR structures of apo-CaM C-lobe-NaV1.2 IQ motif complex and apo-CaM-NaV1.5 IQ motif complex have been reported2327. The structure of the CaM-NaV1.6 IQ motif peptide complex, particularly the C-lobe, was superimposed with the equivalent region of NaV1.2 and NaV1.5 IQ motif peptide complexes, with an rmsd of 1.3 Å and 0.76 Å for 59 and 58 Cα atoms, respectively. The interactions between the apo-C-lobe CaM and NaV1.2 IQ motif complex were reported to be mainly hydrophobic in nature23.The one-to-one comparison of NaV1.2 and NaV1.6 IQ motif complex structures in the absence of Ca2+ is outlined in Figure 5A–C. It can be seen that the Arg residues at positions 3 and 6 of NaV1.6 IQ motif make hydrogen bonding contacts with Glu115 and Glu121 of CaM, respectively (Figure 5A), whereas the same Arg residues of NaV1.2 IQ motif do not interact with CaM (Figure 5B). Moreover, the orientation of the Tyr residues at position 5 of NaV1.6 IQ motif and NaV1.2 IQ motif are different; this is thought to allow the hydroxyl group of the Tyr residue in NaV1.6 IQ motif to make hydrogen bonding contacts with Glu128 of CaM (Figure 5A); however, the same is not observed for the NaV1.2 IQ motif (Figure 5B). The His residue at position 8 of NaV1.6 IQ motif makes a hydrogen bonding contact with Glu128 of CaM, but this residue is replaced by a Tyr residue in the NaV1.2 IQ motif, which is buried in the hydrophobic pocket formed by various hydrophobic residues of C-lobe of CaM; this causes the sidechain of Glu128 of CaM to be oriented away from the Tyr at position 8 of NaV1.2 (Figure 5B).The NMR structure of apo-CaM and NaV1.5 IQ motif revealed an interaction mainly driven by hydrophobic forces and, to some extent, electrostatic interactions. Arg at position 3 of NaV1.5 and NaV1.6 IQ motif are involved in hydrogen bonding contact with CaM, but with different residues (Figure 5C). The Tyr at position 5 of NaV1.6 IQ motif is replaced by Phe in the NaV1.5 IQ motif, and has no hydrogen bonding contact with CaM (Figure 5C).

Comparison with other CaM-NaV IQ motif complexes

The NMR structures of apo-CaM C-lobe-NaV1.2 IQ motif complex and apo-CaM-NaV1.5 IQ motif complex have been reported2327. The structure of the CaM-NaV1.6 IQ motif peptide complex, particularly the C-lobe, was superimposed with the equivalent region of NaV1.2 and NaV1.5 IQ motif peptide complexes, with an rmsd of 1.3 Å and 0.76 Å for 59 and 58 Cα atoms, respectively. The interactions between the apo-C-lobe CaM and NaV1.2 IQ motif complex were reported to be mainly hydrophobic in nature23.

The one-to-one comparison of NaV1.2 and NaV1.6 IQ motif complex structures in the absence of Ca2+ is outlined in Figure 5A–C. It can be seen that the Arg residues at positions 3 and 6 of NaV1.6 IQ motif make hydrogen bonding contacts with Glu115 and Glu121 of CaM, respectively (Figure 5A), whereas the same Arg residues of NaV1.2 IQ motif do not interact with CaM (Figure 5B). Moreover, the orientation of the Tyr residues at position 5 of NaV1.6 IQ motif and NaV1.2 IQ motif are different; this is thought to allow the hydroxyl group of the Tyr residue in NaV1.6 IQ motif to make hydrogen bonding contacts with Glu128 of CaM (Figure 5A); however, the same is not observed for the NaV1.2 IQ motif (Figure 5B). The His residue at position 8 of NaV1.6 IQ motif makes a hydrogen bonding contact with Glu128 of CaM, but this residue is replaced by a Tyr residue in the NaV1.2 IQ motif, which is buried in the hydrophobic pocket formed by various hydrophobic residues of C-lobe of CaM; this causes the sidechain of Glu128 of CaM to be oriented away from the Tyr at position 8 of NaV1.2 (Figure 5B).

The NMR structure of apo-CaM and NaV1.5 IQ motif revealed an interaction mainly driven by hydrophobic forces and, to some extent, electrostatic interactions. Arg at position 3 of NaV1.5 and NaV1.6 IQ motif are involved in hydrogen bonding contact with CaM, but with different residues (Figure 5C). The Tyr at position 5 of NaV1.6 IQ motif is replaced by Phe in the NaV1.5 IQ motif, and has no hydrogen bonding contact with CaM (Figure 5C).

Structure-based mutational studiesBased on the crystal structure of CaM-NaV1.6 IQ motif complex, we selected key interacting residues of CaM (Glu115, Glu121, Glu124, and Glu128) and NaV1.6 IQ motif (Gln1901, Arg1902, Tyr1904, and Arg1905) and mutated them to alanine using site-directed mutagenesis. We then compared the binding of these unlinked CaM and NaV1.6 IQ motifs using ITC to determine the importance of these key residues in mediating binding. We found that mutating any single residue either on CaM or NaV1.6 IQ motif peptide abolished the binding in the absence of Ca2+(Table 1 and Supplementary Figure 1). Similar losses of interaction between proteins and peptides have been previously reported following mutation of key interacting residues28293031.Next, we sought to examine the role of apo-CaM interacting residues of NaV1.6 IQ motif in recognizing Ca2+-bound CaM using ITC. We found that single mutations in the NaV1.6 IQ motif peptide led to reduced binding with Ca2+/CaM. Of the four NaV1.6 mutants, Y1904A showed the most significant reduction in binding affinity (Table 1), followed by R1902A (Table 1 and Supplementary Figure 1). Since the conformation of CaM might be different in the presence of Ca2+, it is likely that different residues would be exposed on the surface of CaM to interact with IQ motifs32. We therefore made brief attempts to crystallize Ca2+/CaM-NaV1.6 IQ motif peptide complex. However, we did not obtain any crystals. We then assessed the role of Ca2+ in regulating the interaction between mutated CaM and WT NaV1.6 IQ motif in the presence of Ca2+. These mutants of CaM did not show any reduction in binding affinity in the presence of Ca2+ (Table 1), suggesting that the NaV1.6 IQ motif is recognized by different residues of CaM in the presence and absence of Ca2+.

Structure-based mutational studies

Based on the crystal structure of CaM-NaV1.6 IQ motif complex, we selected key interacting residues of CaM (Glu115, Glu121, Glu124, and Glu128) and NaV1.6 IQ motif (Gln1901, Arg1902, Tyr1904, and Arg1905) and mutated them to alanine using site-directed mutagenesis. We then compared the binding of these unlinked CaM and NaV1.6 IQ motifs using ITC to determine the importance of these key residues in mediating binding. We found that mutating any single residue either on CaM or NaV1.6 IQ motif peptide abolished the binding in the absence of Ca2+(Table 1 and Supplementary Figure 1). Similar losses of interaction between proteins and peptides have been previously reported following mutation of key interacting residues28293031.

Next, we sought to examine the role of apo-CaM interacting residues of NaV1.6 IQ motif in recognizing Ca2+-bound CaM using ITC. We found that single mutations in the NaV1.6 IQ motif peptide led to reduced binding with Ca2+/CaM. Of the four NaV1.6 mutants, Y1904A showed the most significant reduction in binding affinity (Table 1), followed by R1902A (Table 1 and Supplementary Figure 1). Since the conformation of CaM might be different in the presence of Ca2+, it is likely that different residues would be exposed on the surface of CaM to interact with IQ motifs32. We therefore made brief attempts to crystallize Ca2+/CaM-NaV1.6 IQ motif peptide complex. However, we did not obtain any crystals. We then assessed the role of Ca2+ in regulating the interaction between mutated CaM and WT NaV1.6 IQ motif in the presence of Ca2+. These mutants of CaM did not show any reduction in binding affinity in the presence of Ca2+ (Table 1), suggesting that the NaV1.6 IQ motif is recognized by different residues of CaM in the presence and absence of Ca2+.

Reduced rate of inactivation by R1902A and Y1904A of NaV1.6Finally, we sought to compare the electrophysiological properties of mutating two key residues (R1902A and Y1904A) of NaV1.6 channels with WT NaV1.6 channels. Both WT and mutant channels were expressed in ND7/23 cells and studied under conditions of minimal (essentially zero) intracellular calcium. The two mutations significantly slowed the rate of the inactivation at potentials ranging from 0 to +40 mV (Figure 6A). At +20 mV, R1902A and Y1904A slowed the rate of fast inactivation of Nav1.6 channels by 31% and 48%, respectively. The slower rate of fast inactivation is not caused by the shift in the voltage dependence of channel gating. As can be seen in Supplementary Figure 2, neither R1902A nor Y1904A altered the steady-state activation of Nav1.6 channels. The two mutations also did not significantly alter the steady-state inactivation of NaV1.6 channels (Supplementary Figure 2). Consistent with our previous finding that the IQ motif is an important determinant of current amplitude13, the R1902A and Y1904A mutations decreased current density of NaV1.6 channels by 42.4% and 73.8%, respectively (Figure 6B).

Reduced rate of inactivation by R1902A and Y1904A of NaV1.6

Finally, we sought to compare the electrophysiological properties of mutating two key residues (R1902A and Y1904A) of NaV1.6 channels with WT NaV1.6 channels. Both WT and mutant channels were expressed in ND7/23 cells and studied under conditions of minimal (essentially zero) intracellular calcium. The two mutations significantly slowed the rate of the inactivation at potentials ranging from 0 to +40 mV (Figure 6A). At +20 mV, R1902A and Y1904A slowed the rate of fast inactivation of Nav1.6 channels by 31% and 48%, respectively. The slower rate of fast inactivation is not caused by the shift in the voltage dependence of channel gating. As can be seen in Supplementary Figure 2, neither R1902A nor Y1904A altered the steady-state activation of Nav1.6 channels. The two mutations also did not significantly alter the steady-state inactivation of NaV1.6 channels (Supplementary Figure 2). Consistent with our previous finding that the IQ motif is an important determinant of current amplitude13, the R1902A and Y1904A mutations decreased current density of NaV1.6 channels by 42.4% and 73.8%, respectively (Figure 6B).

DiscussionThe carboxy termini of VGSCs possess a CaM-binding IQ motif that is involved in the regulation of its inactivation kinetics (Supplementary Figure 3). Moreover, CaM is known to modulate the function of VGSCs in an isoform-dependent manner1213141516171819. Disruption of CaM-mediated VGSC regulation through mutations in the IQ motif results in abnormalities linked to life-threatening idiopathic ventricular arrhythmias in cardiac muscle and various other disorders17333435. The aim of the present study was to understand the interactions between CaM and the IQ motif of NaV1.6, a VGSC involved in the propagation of action potentials along myelinated axons in the central nervous system.Ca2+ plays a crucial role in CaM mediated regulation of VGSCs. It is known that Ca2+/CaM mediates slow inactivation and apo-CaM mediates fast inactivation observed in the electrophysiological studies of several VGSCs interacting via their IQ motifs, NaV1.4, NaV1.5 and NaV1.613151718. We have observed that the IQ motif of NaV1.6 has a higher affinity towards Ca2+/CaM than apo-CaM. However, it is known that the IQ motif of cardiac muscle VGSC, NaV1.5, shows a higher affinity towards apo-CaM than Ca2+/CaM15 (Table 3). The isoform-specific differences in terms of binding affinity and how this affects Ca2+/CaM and apo-CaM-mediated inactivation may depend on the cellular localization of the respective VGSCs.Although NaV1.2 and NaV1.6 are expressed in the CNS, they are known to exhibit different electrophysiological properties, including the amount of Na+ current, as well as the onset of and recovery from inactivation3637. NaV1.2 and NaV1.6 are necessary to propagate action potentials along the length of the unmyelinated and myelinated axons in the CNS, respectively. NaV1.2 and NaV1.6 are known to undergo fast inactivation via its C-terminus in the absence of Ca2+
1338. However, there are a few differences between the two isoforms that have been highlighted. For instance, NaV1.6 is more suitable for repetitive firing than NaV1.28. One study showed that the high affinity (nM Kd) between CaM and NaV1.2 IQ motif would promote constitutive binding of CaM to NaV1.2 IQ motif, independent of intracellular Ca2+ levels23 (Table 3). In the present study, we observed moderate affinity (μM Kd) between CaM and the NaV1.6 IQ motif in the presence and absence of Ca2+. Thus, this difference in binding affinity exhibited by CaM towards NaV1.2 and NaV1.6 IQ motifs could dictate the functional properties of the respective VGSCs.The crystal structure of the apo-CaM-NaV1.6 IQ motif peptide complex showed that the interaction is mediated by the C-lobe of CaM. CaM-NaV1.6 IQ motif complex has been obtained by linking the IQ motif peptide to the C-terminus of CaM using a (Gly)5 linker. In the complex crystal structure, instead of having intramolecular interactions between CaM and NaV1.6 IQ peptide, we have observed intermolecular interactions such that the IQ peptide fused with one CaM molecule is interacting with the nearby symmetry related (adjacent) CaM molecule. Similar intermolecular interactions have been previously reported222439. In the core “IQ” region of NaV1.6, mutations of LQ to LE and EE are known to result in reduced binding with CaM. In particular, the EE mutant slows NaV1.6 inactivation by 48% by affecting its interaction with apo-CaM13. In our report, the crystal structure of apo-CaM-NaV1.6 IQ motif complex revealed that Leu1900 at position 1 is buried in the hydrophobic cluster of CaM residues, and thus replacing it with a negatively charged residue such as Glu may affect the interaction. Indeed, substitution of Gln1901 with Glu affected apo-CaM mediated fast inactivation, but did not result in a complete loss of this affect13. Despite this, we observed a complete loss of interaction between NaV1.6 IQ motif and apo-CaM in our ITC experiments when Gln1901 was mutated to Ala.Our structure-guided mutational studies demonstrated that Arg1902, Tyr1904 and Arg1905 of the NaV1.6 core IQ motif region are not only important for its interaction with apo-CaM but also for that with Ca2+/CaM. In particular, the Y1904A substitution abolishes the interaction between NaV1.6 IQ motif and apo-CaM and, reduces affinity to a large extent towards Ca2+/CaM. R1902A also decreased the interaction with both forms of CaM. Importantly, channels containing these mutations showed reduced rates of inactivation and current density as compared with their wild-type counterpart (in the absence of Ca2+), further confirming the importance of these residues. We also observed that the residues of CaM involved in its interaction with NaV1.6 IQ motif are different in the presence and absence of Ca2+. Although E115A, E121A, E124A and E128A mutants of CaM showed no reduction in binding with NaV1.6 IQ motif in the presence of Ca2+, the E115A mutant of Ca2+/CaM showed reduced enthalpy and increased entropy as compared with the wild-type interaction. This enthalpy-entropy compensation suggests a loss of key interaction in the bound state and is compensated by increased disorder4041. Thus, Glu115 of CaM could be involved in the interaction with NaV1.6 IQ motif, whereas the other residues (Glu121, Glu124 and Glu128 of CaM) are not involved in the interaction in the presence of Ca2+.Although the structures of apo-CaM in complex with NaV1.2 and NaV1.5 IQ motif were similar to that observed with NaV1.6 IQ motif, the sidechain interactions of the IQ motifs were different for each case. Moreover, in silico alanine-scanning mutagenesis of apo-CaM-NaV1.5 IQ motif complex revealed that none of the residues from CaM provide the greatest contribution for the interaction, possibly due to the large number of CaM residues involved in creating the binding interface27. The mutational studies based on apo-CaM-NaV1.6 IQ motif complex revealed that substituting a single interacting residue of CaM with Ala resulted in complete loss of interaction with NaV1.6 IQ motif. This could probably explain why the interactions of apo-CaM-NaV1.2 IQ motif and apo-CaM-NaV1.5 IQ motif exhibit lower dissociation constants (Kd) than that of apo-CaM-NaV1.6 IQ motif (Table 3). The binding of CaM to the IQ motif of NaV1.8 VGSC is involved in recovery from inactivation42. However, the mutants R1902A and Y1904A of NaV1.6 did not show any significant difference in the recovery from inactivation (Supplementary Figure 2). It is possible that the inactivation and recovery from inactivation properties of NaV1.6 and NaV1.8 are quite different. However, our data does not rule out the possibility that CaM might modulate recovery from slow inactivation for NaV1.6 channels.In summary, our studies reveal that the NaV1.6 IQ motif has a relatively lower affinity towards CaM as compared with NaV1.2 IQ motif, which provides a basis for the differences in the properties exhibited by these two neuronal VGSCs. Moreover, NaV1.5 and NaV1.6 IQ motifs show differences in affinity towards Ca2+/CaM or apo-CaM, suggesting that the affinity of the NaV IQ motif and CaM is different between cardiac and neuronal VGSCs. In the apo-CaM-NaV1.6 IQ motif complex structure, we show that the C-lobe of CaM is in a semi-open conformation and its interactions with NaV1.6 IQ motif are mediated by hydrogen bonds and hydrophobic contacts. Furthermore, ITC experiments identified Tyr1904 as a key residue for the interaction of NaV1.6 IQ motif with both apo- and Ca2+/CaM. The electrophysiological experiments confirmed the role of these key interacting residues in modulating the inactivation kinetics of NaV1.6 channel. In addition, NaV1.6 IQ motif adopts a different mechanism of interaction with apo-CaM compared to other known CaM-NaV complex structures. Overall, these findings provide additional molecular insight into the isoform-specific regulation of VGSCs by CaM.

The carboxy termini of VGSCs possess a CaM-binding IQ motif that is involved in the regulation of its inactivation kinetics (Supplementary Figure 3). Moreover, CaM is known to modulate the function of VGSCs in an isoform-dependent manner1213141516171819. Disruption of CaM-mediated VGSC regulation through mutations in the IQ motif results in abnormalities linked to life-threatening idiopathic ventricular arrhythmias in cardiac muscle and various other disorders17333435. The aim of the present study was to understand the interactions between CaM and the IQ motif of NaV1.6, a VGSC involved in the propagation of action potentials along myelinated axons in the central nervous system.

Ca2+ plays a crucial role in CaM mediated regulation of VGSCs. It is known that Ca2+/CaM mediates slow inactivation and apo-CaM mediates fast inactivation observed in the electrophysiological studies of several VGSCs interacting via their IQ motifs, NaV1.4, NaV1.5 and NaV1.613151718. We have observed that the IQ motif of NaV1.6 has a higher affinity towards Ca2+/CaM than apo-CaM. However, it is known that the IQ motif of cardiac muscle VGSC, NaV1.5, shows a higher affinity towards apo-CaM than Ca2+/CaM15 (Table 3). The isoform-specific differences in terms of binding affinity and how this affects Ca2+/CaM and apo-CaM-mediated inactivation may depend on the cellular localization of the respective VGSCs.

Although NaV1.2 and NaV1.6 are expressed in the CNS, they are known to exhibit different electrophysiological properties, including the amount of Na+ current, as well as the onset of and recovery from inactivation3637. NaV1.2 and NaV1.6 are necessary to propagate action potentials along the length of the unmyelinated and myelinated axons in the CNS, respectively. NaV1.2 and NaV1.6 are known to undergo fast inactivation via its C-terminus in the absence of Ca2+
1338. However, there are a few differences between the two isoforms that have been highlighted. For instance, NaV1.6 is more suitable for repetitive firing than NaV1.28. One study showed that the high affinity (nM Kd) between CaM and NaV1.2 IQ motif would promote constitutive binding of CaM to NaV1.2 IQ motif, independent of intracellular Ca2+ levels23 (Table 3). In the present study, we observed moderate affinity (μM Kd) between CaM and the NaV1.6 IQ motif in the presence and absence of Ca2+. Thus, this difference in binding affinity exhibited by CaM towards NaV1.2 and NaV1.6 IQ motifs could dictate the functional properties of the respective VGSCs.

The crystal structure of the apo-CaM-NaV1.6 IQ motif peptide complex showed that the interaction is mediated by the C-lobe of CaM. CaM-NaV1.6 IQ motif complex has been obtained by linking the IQ motif peptide to the C-terminus of CaM using a (Gly)5 linker. In the complex crystal structure, instead of having intramolecular interactions between CaM and NaV1.6 IQ peptide, we have observed intermolecular interactions such that the IQ peptide fused with one CaM molecule is interacting with the nearby symmetry related (adjacent) CaM molecule. Similar intermolecular interactions have been previously reported222439. In the core “IQ” region of NaV1.6, mutations of LQ to LE and EE are known to result in reduced binding with CaM. In particular, the EE mutant slows NaV1.6 inactivation by 48% by affecting its interaction with apo-CaM13. In our report, the crystal structure of apo-CaM-NaV1.6 IQ motif complex revealed that Leu1900 at position 1 is buried in the hydrophobic cluster of CaM residues, and thus replacing it with a negatively charged residue such as Glu may affect the interaction. Indeed, substitution of Gln1901 with Glu affected apo-CaM mediated fast inactivation, but did not result in a complete loss of this affect13. Despite this, we observed a complete loss of interaction between NaV1.6 IQ motif and apo-CaM in our ITC experiments when Gln1901 was mutated to Ala.

Our structure-guided mutational studies demonstrated that Arg1902, Tyr1904 and Arg1905 of the NaV1.6 core IQ motif region are not only important for its interaction with apo-CaM but also for that with Ca2+/CaM. In particular, the Y1904A substitution abolishes the interaction between NaV1.6 IQ motif and apo-CaM and, reduces affinity to a large extent towards Ca2+/CaM. R1902A also decreased the interaction with both forms of CaM. Importantly, channels containing these mutations showed reduced rates of inactivation and current density as compared with their wild-type counterpart (in the absence of Ca2+), further confirming the importance of these residues. We also observed that the residues of CaM involved in its interaction with NaV1.6 IQ motif are different in the presence and absence of Ca2+. Although E115A, E121A, E124A and E128A mutants of CaM showed no reduction in binding with NaV1.6 IQ motif in the presence of Ca2+, the E115A mutant of Ca2+/CaM showed reduced enthalpy and increased entropy as compared with the wild-type interaction. This enthalpy-entropy compensation suggests a loss of key interaction in the bound state and is compensated by increased disorder4041. Thus, Glu115 of CaM could be involved in the interaction with NaV1.6 IQ motif, whereas the other residues (Glu121, Glu124 and Glu128 of CaM) are not involved in the interaction in the presence of Ca2+.

Although the structures of apo-CaM in complex with NaV1.2 and NaV1.5 IQ motif were similar to that observed with NaV1.6 IQ motif, the sidechain interactions of the IQ motifs were different for each case. Moreover, in silico alanine-scanning mutagenesis of apo-CaM-NaV1.5 IQ motif complex revealed that none of the residues from CaM provide the greatest contribution for the interaction, possibly due to the large number of CaM residues involved in creating the binding interface27. The mutational studies based on apo-CaM-NaV1.6 IQ motif complex revealed that substituting a single interacting residue of CaM with Ala resulted in complete loss of interaction with NaV1.6 IQ motif. This could probably explain why the interactions of apo-CaM-NaV1.2 IQ motif and apo-CaM-NaV1.5 IQ motif exhibit lower dissociation constants (Kd) than that of apo-CaM-NaV1.6 IQ motif (Table 3). The binding of CaM to the IQ motif of NaV1.8 VGSC is involved in recovery from inactivation42. However, the mutants R1902A and Y1904A of NaV1.6 did not show any significant difference in the recovery from inactivation (Supplementary Figure 2). It is possible that the inactivation and recovery from inactivation properties of NaV1.6 and NaV1.8 are quite different. However, our data does not rule out the possibility that CaM might modulate recovery from slow inactivation for NaV1.6 channels.

In summary, our studies reveal that the NaV1.6 IQ motif has a relatively lower affinity towards CaM as compared with NaV1.2 IQ motif, which provides a basis for the differences in the properties exhibited by these two neuronal VGSCs. Moreover, NaV1.5 and NaV1.6 IQ motifs show differences in affinity towards Ca2+/CaM or apo-CaM, suggesting that the affinity of the NaV IQ motif and CaM is different between cardiac and neuronal VGSCs. In the apo-CaM-NaV1.6 IQ motif complex structure, we show that the C-lobe of CaM is in a semi-open conformation and its interactions with NaV1.6 IQ motif are mediated by hydrogen bonds and hydrophobic contacts. Furthermore, ITC experiments identified Tyr1904 as a key residue for the interaction of NaV1.6 IQ motif with both apo- and Ca2+/CaM. The electrophysiological experiments confirmed the role of these key interacting residues in modulating the inactivation kinetics of NaV1.6 channel. In addition, NaV1.6 IQ motif adopts a different mechanism of interaction with apo-CaM compared to other known CaM-NaV complex structures. Overall, these findings provide additional molecular insight into the isoform-specific regulation of VGSCs by CaM.

MethodsCloning, expression and purification of CaM constructsCaM was cloned into MCS1 cloning site (between BamHI and SalI restriction sites) of the pETDuet-1 (Novagen, Madison, WI) expression vector. NaV1.6IQ motif (1893–1914) was linked to the C-terminus of CaM via a 5-glycine flexible linker (CaM-(Gly)5-NaV1.6 IQ) using a three-step fusion PCR procedure, as described by Ye et al43. The final PCR product was digested with NdeI and XhoI restriction enzymes (New England Biolabs, Ips wich, MA) along with the pGS21a vector (GeneScript, Piscataway, NJ). Predigested CaM-(Gly)5-NaV1.6 IQ gene and pGS21a vector were ligated, transformed into chemically competent E. coli DH5α cells and screened for positive colonies.For protein expression, recombinant plasmids were transformed into E. coli BL21 (DE3) competent cells and cultured in 1 L LB media (supplemented with 100 μg/mL ampicillin) at 37°C until the OD600 reached between 0.6–0.8 AU. Protein expression was induced with 0.15 mM IPTG for 16 h at 16°C. Cell pellets were resuspended in 50 ml of lysis buffer (50 mM TrisHCl pH 7.4, 200 mM NaCl, 5% glycerol, 5 mM imidazole, 10 mM β-mercaptoethanol and 1 ml of protease inhibitor cocktail (Sigma-Aldrich, St. Louis, MO)). The cell suspension was sonicated and then centrifuged at 39,000 xg for 30 min. The supernatant was mixed with 5 ml of Ni-NTA resin (Qiagen, Valencia, CA) pre-equilibrated with lysis buffer for 1 hr. Resin was washed 3 times with lysis buffer and the bound proteins were eluted using 10 ml of lysis buffer supplemented with 500 mM imidazole. Eluted proteins were passed on to HiLoad 16/60 Superdex™ 75 prep grade (GE Healthcare, Buckinghamshire, UK) and eluted in one of two solutions comprising 50 mM TrisHCl pH 7.4 with 1 mM CaCl2 (solution 1) or 50 mM TrisHCl pH 7.4, 1 mM EGTA (solution 2). In order to facilitate structure determination by heavy atom phasing, the Seleno-L-methionine labeled proteins were produced using M9 media, and proteins were purified by adopting a similar protocol, as mentioned above. All protein purification steps were carried out at 4°C.Isothermal titration calorimetryBinding affinities between NaV1.6 IQ motifs (1891–1914) and Ca2+/CaM or apo-CaM were studied using isothermal titration calorimetry (ITC). All peptides used in this study were purchased from GL Biochem Ltd (Shanghai, China). All proteins/peptides were dialyzed with buffer consisting of 50 mM Tris HCl (pH 7.4), 100 mM NaCl and 1 mM CaCl2 for Ca2+/CaM interaction studies, and 50 mM Tris HCl (pH 7.4), 100 mM NaCl and 1 mM EGTA for apo-CaM interaction studies. ITC experiments were performed using VP-ITC calorimeter (Microcal, LLC, Northampton, MA) at 25°C with 0.3 ml of 120–150 μM of NaV1.6IQ motif peptides in the syringe and 1.4 ml of 10 μM of Ca2+/CaM or apo-CaM in the sample cell. All the samples were thoroughly degassed and centrifuged to remove any precipitates. Volumes of 10 μl per injection were used for all experiments and consecutive injections were separated by 4 min to allow the peak to return to baseline. ITC data were analyzed with a single-site fitting model using Origin 7.0 software (OriginLab Corp., Northampton, MA).Crystallization and structure determinationCrystallization screening for apo-CaM-(Gly)5-NaV1.6 IQ complexes were performed with a concentration of 18 mg/ml using the Hanging drop vapor diffusion method at room temperature. The initially identified condition from Hampton Research (Aliso Viejo, CA) was further optimized and the best crystals for apo-CaM-(Gly)5-NaV1.6 IQ were obtained from a condition consisting of 0.1 M Bis-Tris propane pH 7.0 and 56% Tacsimate. Where necessary, crystals were cryo-protected with 70% Tacsimate and flash-cooled in N2 cold stream at 100 K.The apo-CaM-(Gly)5-NaV1.6 IQ crystallized in the P21 space group and had four molecules in the asymmetric unit. Complete SAD (Single wavelength Anomalous Dispersion) data sets were collected at X6A beam line, NSLS, Brookhaven National Laboratory using a Quantum 4-CCD detector (Area Detector Systems Corp Poway, CA). All data sets were collected at 100 K. Data sets were indexed, integrated and scaled using HKL200044. Heavy atom (Se) location, phasing and density modification was performed using Phenix Autosol45. Model building was carried out using Phenix Autobuild45. Where required, the protein model was manually built using the program COOT46 and refinement was performed using Phenix Refine45. When the R-factors were less than 30%, well-ordered water molecules were added. The model has good stereochemistry, with all the residues within the allowed region of Ramachandran plot, as analyzed by PROCHECK47. All structure-related figures reported in this paper were generated using PyMol48.Site directed mutagenesisSite-directed mutagenesis on CaM gene was achieved via inverse PCR technique49 using the KapaHiFi DNA polymerase Kit (KAPA Biosystems, MA). Linear gene products were gel extracted and transformed into E. coli DH5αcells. Positive plasmids were transformed into E. coli BL21 (DE3) competent cells for protein expression, as described earlier.Plasmid and construction of Nav1.6 mutantsThe cDNA encoding a TTX-resistant mouse Nav1.6 construct was subcloned into a modified pcDNA3.1 vector, as described previously13. All mutations were introduced into the TTX-resistant mNav1.6 cDNA construct using the QuikChange II XL site-directed mutagenesis kit (Agilent Technologies, Inc., Santa Clara, CA) according to the manufacturer's instruction. The constructs were sequenced to confirm that the appropriate mutations were made.Electrophysiology experimentsWT and mutant mNav1.6 channels were transiently transfected into ND7/23 cells using the standard calcium phosphate precipitation method, as previously described13. Whole-cell patch-clamp recordings were performed at room temperature (approximately 21°C) using an EPC-10 amplifier (HEKA, Lambrecht, Germany). Fire-polished electrodes were fabricated from 1.7-mm capillary glass (VWR, West Chester, PA) using a P-97 puller (Sutter, Novato, CA). The standard pipette solution contained 140 mM CsF, 1 mM EGTA, 10 mM NaCl, and 10 mM HEPES, pH 7.3. The standard bathing solution was 140 mM NaCl, 3 mM KCl, 1 mM MgCl2, 1 mM CaCl2, and 10 mM HEPES, pH 7.3. After filling with pipette solution, the access resistance of the electrode pipette ranged from 1.2 to 1.6 MΩ. The liquid junction potential for these solutions was <8 mV; data were not corrected to account for this offset. Voltage errors were minimized using 70–90% series resistance compensation, and the capacitance artifact was canceled using the computer-controlled circuitry of the patch clamp amplifier. Linear leak subtraction, based on resistance estimates from four to five hyperpolarizing pulses applied before the depolarizing test potential, was used for all voltage clamp recordings. Membrane currents were usually filtered at 5 kHz and sampled at 20 kHz. All cells were held at −100 mV for 5 min before the measurement of sodium currents. In order to avoid contamination from endogenous ND7/23 sodium currents, the currents of the TTX-resistant constructs were recorded in the presence of 500 nM TTX.Protein data bank accession codeCoordinates and structure factors of apo-CaM-(Gly)5-NaV1.6 IQ have been deposited with RCSB Protein Data Bank with code 3WFN.

Cloning, expression and purification of CaM constructsCaM was cloned into MCS1 cloning site (between BamHI and SalI restriction sites) of the pETDuet-1 (Novagen, Madison, WI) expression vector. NaV1.6IQ motif (1893–1914) was linked to the C-terminus of CaM via a 5-glycine flexible linker (CaM-(Gly)5-NaV1.6 IQ) using a three-step fusion PCR procedure, as described by Ye et al43. The final PCR product was digested with NdeI and XhoI restriction enzymes (New England Biolabs, Ips wich, MA) along with the pGS21a vector (GeneScript, Piscataway, NJ). Predigested CaM-(Gly)5-NaV1.6 IQ gene and pGS21a vector were ligated, transformed into chemically competent E. coli DH5α cells and screened for positive colonies.For protein expression, recombinant plasmids were transformed into E. coli BL21 (DE3) competent cells and cultured in 1 L LB media (supplemented with 100 μg/mL ampicillin) at 37°C until the OD600 reached between 0.6–0.8 AU. Protein expression was induced with 0.15 mM IPTG for 16 h at 16°C. Cell pellets were resuspended in 50 ml of lysis buffer (50 mM TrisHCl pH 7.4, 200 mM NaCl, 5% glycerol, 5 mM imidazole, 10 mM β-mercaptoethanol and 1 ml of protease inhibitor cocktail (Sigma-Aldrich, St. Louis, MO)). The cell suspension was sonicated and then centrifuged at 39,000 xg for 30 min. The supernatant was mixed with 5 ml of Ni-NTA resin (Qiagen, Valencia, CA) pre-equilibrated with lysis buffer for 1 hr. Resin was washed 3 times with lysis buffer and the bound proteins were eluted using 10 ml of lysis buffer supplemented with 500 mM imidazole. Eluted proteins were passed on to HiLoad 16/60 Superdex™ 75 prep grade (GE Healthcare, Buckinghamshire, UK) and eluted in one of two solutions comprising 50 mM TrisHCl pH 7.4 with 1 mM CaCl2 (solution 1) or 50 mM TrisHCl pH 7.4, 1 mM EGTA (solution 2). In order to facilitate structure determination by heavy atom phasing, the Seleno-L-methionine labeled proteins were produced using M9 media, and proteins were purified by adopting a similar protocol, as mentioned above. All protein purification steps were carried out at 4°C.

Cloning, expression and purification of CaM constructs

CaM was cloned into MCS1 cloning site (between BamHI and SalI restriction sites) of the pETDuet-1 (Novagen, Madison, WI) expression vector. NaV1.6IQ motif (1893–1914) was linked to the C-terminus of CaM via a 5-glycine flexible linker (CaM-(Gly)5-NaV1.6 IQ) using a three-step fusion PCR procedure, as described by Ye et al43. The final PCR product was digested with NdeI and XhoI restriction enzymes (New England Biolabs, Ips wich, MA) along with the pGS21a vector (GeneScript, Piscataway, NJ). Predigested CaM-(Gly)5-NaV1.6 IQ gene and pGS21a vector were ligated, transformed into chemically competent E. coli DH5α cells and screened for positive colonies.

For protein expression, recombinant plasmids were transformed into E. coli BL21 (DE3) competent cells and cultured in 1 L LB media (supplemented with 100 μg/mL ampicillin) at 37°C until the OD600 reached between 0.6–0.8 AU. Protein expression was induced with 0.15 mM IPTG for 16 h at 16°C. Cell pellets were resuspended in 50 ml of lysis buffer (50 mM TrisHCl pH 7.4, 200 mM NaCl, 5% glycerol, 5 mM imidazole, 10 mM β-mercaptoethanol and 1 ml of protease inhibitor cocktail (Sigma-Aldrich, St. Louis, MO)). The cell suspension was sonicated and then centrifuged at 39,000 xg for 30 min. The supernatant was mixed with 5 ml of Ni-NTA resin (Qiagen, Valencia, CA) pre-equilibrated with lysis buffer for 1 hr. Resin was washed 3 times with lysis buffer and the bound proteins were eluted using 10 ml of lysis buffer supplemented with 500 mM imidazole. Eluted proteins were passed on to HiLoad 16/60 Superdex™ 75 prep grade (GE Healthcare, Buckinghamshire, UK) and eluted in one of two solutions comprising 50 mM TrisHCl pH 7.4 with 1 mM CaCl2 (solution 1) or 50 mM TrisHCl pH 7.4, 1 mM EGTA (solution 2). In order to facilitate structure determination by heavy atom phasing, the Seleno-L-methionine labeled proteins were produced using M9 media, and proteins were purified by adopting a similar protocol, as mentioned above. All protein purification steps were carried out at 4°C.

Isothermal titration calorimetryBinding affinities between NaV1.6 IQ motifs (1891–1914) and Ca2+/CaM or apo-CaM were studied using isothermal titration calorimetry (ITC). All peptides used in this study were purchased from GL Biochem Ltd (Shanghai, China). All proteins/peptides were dialyzed with buffer consisting of 50 mM Tris HCl (pH 7.4), 100 mM NaCl and 1 mM CaCl2 for Ca2+/CaM interaction studies, and 50 mM Tris HCl (pH 7.4), 100 mM NaCl and 1 mM EGTA for apo-CaM interaction studies. ITC experiments were performed using VP-ITC calorimeter (Microcal, LLC, Northampton, MA) at 25°C with 0.3 ml of 120–150 μM of NaV1.6IQ motif peptides in the syringe and 1.4 ml of 10 μM of Ca2+/CaM or apo-CaM in the sample cell. All the samples were thoroughly degassed and centrifuged to remove any precipitates. Volumes of 10 μl per injection were used for all experiments and consecutive injections were separated by 4 min to allow the peak to return to baseline. ITC data were analyzed with a single-site fitting model using Origin 7.0 software (OriginLab Corp., Northampton, MA).

Isothermal titration calorimetry

Binding affinities between NaV1.6 IQ motifs (1891–1914) and Ca2+/CaM or apo-CaM were studied using isothermal titration calorimetry (ITC). All peptides used in this study were purchased from GL Biochem Ltd (Shanghai, China). All proteins/peptides were dialyzed with buffer consisting of 50 mM Tris HCl (pH 7.4), 100 mM NaCl and 1 mM CaCl2 for Ca2+/CaM interaction studies, and 50 mM Tris HCl (pH 7.4), 100 mM NaCl and 1 mM EGTA for apo-CaM interaction studies. ITC experiments were performed using VP-ITC calorimeter (Microcal, LLC, Northampton, MA) at 25°C with 0.3 ml of 120–150 μM of NaV1.6IQ motif peptides in the syringe and 1.4 ml of 10 μM of Ca2+/CaM or apo-CaM in the sample cell. All the samples were thoroughly degassed and centrifuged to remove any precipitates. Volumes of 10 μl per injection were used for all experiments and consecutive injections were separated by 4 min to allow the peak to return to baseline. ITC data were analyzed with a single-site fitting model using Origin 7.0 software (OriginLab Corp., Northampton, MA).

Crystallization and structure determinationCrystallization screening for apo-CaM-(Gly)5-NaV1.6 IQ complexes were performed with a concentration of 18 mg/ml using the Hanging drop vapor diffusion method at room temperature. The initially identified condition from Hampton Research (Aliso Viejo, CA) was further optimized and the best crystals for apo-CaM-(Gly)5-NaV1.6 IQ were obtained from a condition consisting of 0.1 M Bis-Tris propane pH 7.0 and 56% Tacsimate. Where necessary, crystals were cryo-protected with 70% Tacsimate and flash-cooled in N2 cold stream at 100 K.The apo-CaM-(Gly)5-NaV1.6 IQ crystallized in the P21 space group and had four molecules in the asymmetric unit. Complete SAD (Single wavelength Anomalous Dispersion) data sets were collected at X6A beam line, NSLS, Brookhaven National Laboratory using a Quantum 4-CCD detector (Area Detector Systems Corp Poway, CA). All data sets were collected at 100 K. Data sets were indexed, integrated and scaled using HKL200044. Heavy atom (Se) location, phasing and density modification was performed using Phenix Autosol45. Model building was carried out using Phenix Autobuild45. Where required, the protein model was manually built using the program COOT46 and refinement was performed using Phenix Refine45. When the R-factors were less than 30%, well-ordered water molecules were added. The model has good stereochemistry, with all the residues within the allowed region of Ramachandran plot, as analyzed by PROCHECK47. All structure-related figures reported in this paper were generated using PyMol48.

Crystallization and structure determination

Crystallization screening for apo-CaM-(Gly)5-NaV1.6 IQ complexes were performed with a concentration of 18 mg/ml using the Hanging drop vapor diffusion method at room temperature. The initially identified condition from Hampton Research (Aliso Viejo, CA) was further optimized and the best crystals for apo-CaM-(Gly)5-NaV1.6 IQ were obtained from a condition consisting of 0.1 M Bis-Tris propane pH 7.0 and 56% Tacsimate. Where necessary, crystals were cryo-protected with 70% Tacsimate and flash-cooled in N2 cold stream at 100 K.

The apo-CaM-(Gly)5-NaV1.6 IQ crystallized in the P21 space group and had four molecules in the asymmetric unit. Complete SAD (Single wavelength Anomalous Dispersion) data sets were collected at X6A beam line, NSLS, Brookhaven National Laboratory using a Quantum 4-CCD detector (Area Detector Systems Corp Poway, CA). All data sets were collected at 100 K. Data sets were indexed, integrated and scaled using HKL200044. Heavy atom (Se) location, phasing and density modification was performed using Phenix Autosol45. Model building was carried out using Phenix Autobuild45. Where required, the protein model was manually built using the program COOT46 and refinement was performed using Phenix Refine45. When the R-factors were less than 30%, well-ordered water molecules were added. The model has good stereochemistry, with all the residues within the allowed region of Ramachandran plot, as analyzed by PROCHECK47. All structure-related figures reported in this paper were generated using PyMol48.

Site directed mutagenesisSite-directed mutagenesis on CaM gene was achieved via inverse PCR technique49 using the KapaHiFi DNA polymerase Kit (KAPA Biosystems, MA). Linear gene products were gel extracted and transformed into E. coli DH5αcells. Positive plasmids were transformed into E. coli BL21 (DE3) competent cells for protein expression, as described earlier.

Site directed mutagenesis

Site-directed mutagenesis on CaM gene was achieved via inverse PCR technique49 using the KapaHiFi DNA polymerase Kit (KAPA Biosystems, MA). Linear gene products were gel extracted and transformed into E. coli DH5αcells. Positive plasmids were transformed into E. coli BL21 (DE3) competent cells for protein expression, as described earlier.

Plasmid and construction of Nav1.6 mutantsThe cDNA encoding a TTX-resistant mouse Nav1.6 construct was subcloned into a modified pcDNA3.1 vector, as described previously13. All mutations were introduced into the TTX-resistant mNav1.6 cDNA construct using the QuikChange II XL site-directed mutagenesis kit (Agilent Technologies, Inc., Santa Clara, CA) according to the manufacturer's instruction. The constructs were sequenced to confirm that the appropriate mutations were made.

Plasmid and construction of Nav1.6 mutants

The cDNA encoding a TTX-resistant mouse Nav1.6 construct was subcloned into a modified pcDNA3.1 vector, as described previously13. All mutations were introduced into the TTX-resistant mNav1.6 cDNA construct using the QuikChange II XL site-directed mutagenesis kit (Agilent Technologies, Inc., Santa Clara, CA) according to the manufacturer's instruction. The constructs were sequenced to confirm that the appropriate mutations were made.

Electrophysiology experimentsWT and mutant mNav1.6 channels were transiently transfected into ND7/23 cells using the standard calcium phosphate precipitation method, as previously described13. Whole-cell patch-clamp recordings were performed at room temperature (approximately 21°C) using an EPC-10 amplifier (HEKA, Lambrecht, Germany). Fire-polished electrodes were fabricated from 1.7-mm capillary glass (VWR, West Chester, PA) using a P-97 puller (Sutter, Novato, CA). The standard pipette solution contained 140 mM CsF, 1 mM EGTA, 10 mM NaCl, and 10 mM HEPES, pH 7.3. The standard bathing solution was 140 mM NaCl, 3 mM KCl, 1 mM MgCl2, 1 mM CaCl2, and 10 mM HEPES, pH 7.3. After filling with pipette solution, the access resistance of the electrode pipette ranged from 1.2 to 1.6 MΩ. The liquid junction potential for these solutions was <8 mV; data were not corrected to account for this offset. Voltage errors were minimized using 70–90% series resistance compensation, and the capacitance artifact was canceled using the computer-controlled circuitry of the patch clamp amplifier. Linear leak subtraction, based on resistance estimates from four to five hyperpolarizing pulses applied before the depolarizing test potential, was used for all voltage clamp recordings. Membrane currents were usually filtered at 5 kHz and sampled at 20 kHz. All cells were held at −100 mV for 5 min before the measurement of sodium currents. In order to avoid contamination from endogenous ND7/23 sodium currents, the currents of the TTX-resistant constructs were recorded in the presence of 500 nM TTX.

Electrophysiology experiments

WT and mutant mNav1.6 channels were transiently transfected into ND7/23 cells using the standard calcium phosphate precipitation method, as previously described13. Whole-cell patch-clamp recordings were performed at room temperature (approximately 21°C) using an EPC-10 amplifier (HEKA, Lambrecht, Germany). Fire-polished electrodes were fabricated from 1.7-mm capillary glass (VWR, West Chester, PA) using a P-97 puller (Sutter, Novato, CA). The standard pipette solution contained 140 mM CsF, 1 mM EGTA, 10 mM NaCl, and 10 mM HEPES, pH 7.3. The standard bathing solution was 140 mM NaCl, 3 mM KCl, 1 mM MgCl2, 1 mM CaCl2, and 10 mM HEPES, pH 7.3. After filling with pipette solution, the access resistance of the electrode pipette ranged from 1.2 to 1.6 MΩ. The liquid junction potential for these solutions was <8 mV; data were not corrected to account for this offset. Voltage errors were minimized using 70–90% series resistance compensation, and the capacitance artifact was canceled using the computer-controlled circuitry of the patch clamp amplifier. Linear leak subtraction, based on resistance estimates from four to five hyperpolarizing pulses applied before the depolarizing test potential, was used for all voltage clamp recordings. Membrane currents were usually filtered at 5 kHz and sampled at 20 kHz. All cells were held at −100 mV for 5 min before the measurement of sodium currents. In order to avoid contamination from endogenous ND7/23 sodium currents, the currents of the TTX-resistant constructs were recorded in the presence of 500 nM TTX.

Protein data bank accession codeCoordinates and structure factors of apo-CaM-(Gly)5-NaV1.6 IQ have been deposited with RCSB Protein Data Bank with code 3WFN.

Protein data bank accession code

Coordinates and structure factors of apo-CaM-(Gly)5-NaV1.6 IQ have been deposited with RCSB Protein Data Bank with code 3WFN.

J.S. conceived of the study. V.P.R.C. carried out crystallographic, mutational and I.T.C. experiments. V.P.R.C. and J.Se. analyzed the X-ray data. Y.X. and T.R.C. performed and analyzed electrophysiology experiments. V.P.R.C. and J.S. wrote the manuscript.

Supplementary MaterialSupplementary InformationSupplementary information

Supplementary Material

Supplementary Information

Supplementary information
