---
pmid: '30662352'
title: COPI-Mediated Nuclear Translocation of EGFRvIII Promotes STAT3 Phosphorylation
  and PKM2 Nuclear Localization.
authors:
- Zhang M
- Sun H
- Deng Y
- Su M
- Wei S
- Wang P
- Yu L
- Liu J
- Guo J
- Wang X
- Han X
- He Q
- Shen L
journal: Int J Biol Sci
year: '2019'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6329918
doi: 10.7150/ijbs.28679
---

# COPI-Mediated Nuclear Translocation of EGFRvIII Promotes STAT3 Phosphorylation and PKM2 Nuclear Localization.
**Authors:** Zhang M, Sun H, Deng Y, Su M, Wei S, Wang P, Yu L, Liu J, Guo J, Wang X, Han X, He Q, Shen L
**Journal:** Int J Biol Sci (2019)
**DOI:** [10.7150/ijbs.28679](https://doi.org/10.7150/ijbs.28679)
**PMC:** [PMC6329918](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6329918/)

## Abstract

1. Int J Biol Sci. 2019 Jan 1;15(1):114-126. doi: 10.7150/ijbs.28679. eCollection
 2019.

COPI-Mediated Nuclear Translocation of EGFRvIII Promotes STAT3 Phosphorylation 
and PKM2 Nuclear Localization.

Zhang M(1), Sun H(2), Deng Y(3), Su M(1), Wei S(1), Wang P(3), Yu L(4), Liu 
J(4), Guo J(4), Wang X(5), Han X(5), He Q(6), Shen L(1).

Author information:
(1)Department of Cell Biology, Stem Cell Research Center, School of Basic 
Medical Sciences, Peking University Health Science Center, Beijing, People's 
Republic of China.
(2)Neuroscience Research Institute, School of Basic Medical Sciences, Peking 
University Health Science Center, Beijing, People's Republic of China.
(3)Beijing Cellonis Biotechnologies Co. Ltd, Zhongguancun Bio-Medicine Park, 
Beijing, People's Republic of China.
(4)Beijing DongFang YaMei Gene Science and Technology Research Institute, 
Beijing, People's Republic of China.
(5)Research Center of Artificial Organ and Tissue Engineering, Second Department 
of Hepatobiliary Surgery, ZhuJiang Hospital, Southern Medical University, 
Guangzhou, Guangdong Province, People's Republic of China.
(6)Medical and Health Analytical Center, Peking University Health Science 
Center, People's Republic of China.

As a non-ligand-dependent activation protein, EGFRvIII is the most common mutant 
of EGFR, and its existence or especially its nuclear translocation in tumors can 
exacerbate the malignancy. Compared with the nuclear translocation of EGFR, 
which has been studied extensively, the specific mechanism by which EGFRvIII 
undergoes nuclear translocation has not yet been reported. Here, we found that 
EGFRvIII eventually reached the nucleus with the involvement of the Golgi and 
endoplasmic reticulum (ER) in glioma cells. In this process, syntaxin-6 was 
responsible for the identification and transport of EGFRvIII on Golgi. We also 
demonstrated that COPI mediated the reverse transport of EGFRvIII from the Golgi 
to ER, which process was also important for EGFRvIII's nuclear accumulation. 
EGFRvIII's nuclear translocation can significantly promote STAT3 phosphorylation 
and PKM2 nuclear localization. Finally, we showed that EGFRvIII's nuclear 
translocation obviously induced the growth of gliomas in an intracranial 
xenotransplantation experiment. These data suggested that searching methods that 
inhibit EGFRvIII entry into the nucleus will be effective glioma treatments.

DOI: 10.7150/ijbs.28679
PMCID: PMC6329918
PMID: 30662352 [Indexed for MEDLINE]

Conflict of interest statement: Competing Interests: The authors have declared 
that no competing interest exists.

## Full Text

IntroductionEGFR belongs to the family of receptor tyrosine kinases (RKTs) and can activate downstream signaling cascades in a ligand-dependent manner 1, 2. Signal pathway disorders caused by the overexpression, amplification and mutation of EGFR are the root cause of many malignant tumors, occurring 50%-70% in glioblastoma multiforme (GBM) cases 3-5.In EGFR-induced glioma patients, the most common mutation in EGFR is EGFRvIII, which lacks an extracellular domain. Deletion of exons 2-7 in EGFRvIII at the genetic level results in the deletion of extracellular amino acids 6-273 5-7. The presence of EGFRvIII is an important marker of more highly malignant gliomas and a worse prognosis in patients, accounting for 25%-33% of all gliomas 8-10.Based on existing studies, there are two main causes of EGFRvIII-aggravated gliomas: ligand-independent activation of EGFRvIII can continuously activate signaling pathways that are related to proliferation, migration, and invasion of tumor cells11-13; and besides, as a cell membrane localization protein, EGFRvIII, which undergoes nuclear localization, exposing its nuclear localization signal (NLS) during endocytosis and then occurring transposition into the nucleus with the aid of some protein molecules 14, 15, where it regulates downstream signals by functioning as a transcription factor. During the nuclear translocation process of EGFRvIII, the most commonly studied cascade is EGFRvIII-STAT3-mediated pro-inflammatory signaling pathway, which is the pathogenesis of many tumors. 3, 15-17.Thus, the specific process by which EGFRvIII undergoes nuclear translocation must be examined to determine the pathogenesis of glioma and identify targets for its treatment. Unfortunately, there are few reports on the nuclear translocation process of EGFRvIII, but existing studies on the nuclear translocation of EGFR can provide us with a reference 18-20.Recent studies have shown that many RTK family members exert their unique functions after being transported to different organelles along the secretory pathway 21-23. After endocytosis, EGFR may be transferred to the Golgi apparatus and ER, known as retrograde trafficking process, and then localizes in the nucleus 18, 19.As an EGFR mutant, EGFRvIII is likely to undergo a similar process. In addition, retrograde transport between the Golgi and ER is mediated by COPI, which is an important step in nuclear translocation. COPI, consisting of the small GTPase ADP- ribosylation factor (ARF) and seven subunits (α, β, β', γ, δ, ε, and ζ), is coat protein complex I 24. After GDP-GTP exchange, ARF recruits a coatomer complex that is preassembled by the seven subunits to form COPI that has transport function 25, 26. Thus, the activation of ARF plays an important role in COPI-mediated protein transport; the inactivation of ARF will cause the disassembly of COPI, suppressing reverse transport. Brefeldin A (BFA), used in this study, is an inhibitor that can inactivate ARF 27. In this paper, the involvement of the Golgi and ER apparatuses, especially COPI, was studied in the retrograde trafficking of EGFRvIII, and the process by which EGFRvIII undergoes nuclear translocation was determined.In the alternative splicing of PKM pre-messenger RNA, exon 10 is retained in PKM2, whereas exon 9 is excluded from PKM1, which two share the same pre-messenger RNA 28-30. PKM2 is an isoenzyme of glycolytic pyruvate kinase, which is highly expressed in many tumors and participates in excess glucose metabolism and lactate production 31-33. Previous studies have shown that EGFR can promote the nuclear translocation of PKM2 34, 35, which can further enhance the Warburg effect and possibility of tumorigenesis. Therefore, based on these findings, we investigated the regulation function of EGFRvIII in PKM2 nuclear translocation and prompted us to determine the major role of EGFRvIII nuclear translocation in promoting the nuclear localization of PKM2. This is an overlooked but important discovery, because it has implications for the future treatment of tumors, especially GBM.

Introduction

EGFR belongs to the family of receptor tyrosine kinases (RKTs) and can activate downstream signaling cascades in a ligand-dependent manner 1, 2. Signal pathway disorders caused by the overexpression, amplification and mutation of EGFR are the root cause of many malignant tumors, occurring 50%-70% in glioblastoma multiforme (GBM) cases 3-5.

In EGFR-induced glioma patients, the most common mutation in EGFR is EGFRvIII, which lacks an extracellular domain. Deletion of exons 2-7 in EGFRvIII at the genetic level results in the deletion of extracellular amino acids 6-273 5-7. The presence of EGFRvIII is an important marker of more highly malignant gliomas and a worse prognosis in patients, accounting for 25%-33% of all gliomas 8-10.

Based on existing studies, there are two main causes of EGFRvIII-aggravated gliomas: ligand-independent activation of EGFRvIII can continuously activate signaling pathways that are related to proliferation, migration, and invasion of tumor cells11-13; and besides, as a cell membrane localization protein, EGFRvIII, which undergoes nuclear localization, exposing its nuclear localization signal (NLS) during endocytosis and then occurring transposition into the nucleus with the aid of some protein molecules 14, 15, where it regulates downstream signals by functioning as a transcription factor. During the nuclear translocation process of EGFRvIII, the most commonly studied cascade is EGFRvIII-STAT3-mediated pro-inflammatory signaling pathway, which is the pathogenesis of many tumors. 3, 15-17.

Thus, the specific process by which EGFRvIII undergoes nuclear translocation must be examined to determine the pathogenesis of glioma and identify targets for its treatment. Unfortunately, there are few reports on the nuclear translocation process of EGFRvIII, but existing studies on the nuclear translocation of EGFR can provide us with a reference 18-20.

Recent studies have shown that many RTK family members exert their unique functions after being transported to different organelles along the secretory pathway 21-23. After endocytosis, EGFR may be transferred to the Golgi apparatus and ER, known as retrograde trafficking process, and then localizes in the nucleus 18, 19.

As an EGFR mutant, EGFRvIII is likely to undergo a similar process. In addition, retrograde transport between the Golgi and ER is mediated by COPI, which is an important step in nuclear translocation. COPI, consisting of the small GTPase ADP- ribosylation factor (ARF) and seven subunits (α, β, β', γ, δ, ε, and ζ), is coat protein complex I 24. After GDP-GTP exchange, ARF recruits a coatomer complex that is preassembled by the seven subunits to form COPI that has transport function 25, 26. Thus, the activation of ARF plays an important role in COPI-mediated protein transport; the inactivation of ARF will cause the disassembly of COPI, suppressing reverse transport. Brefeldin A (BFA), used in this study, is an inhibitor that can inactivate ARF 27. In this paper, the involvement of the Golgi and ER apparatuses, especially COPI, was studied in the retrograde trafficking of EGFRvIII, and the process by which EGFRvIII undergoes nuclear translocation was determined.

In the alternative splicing of PKM pre-messenger RNA, exon 10 is retained in PKM2, whereas exon 9 is excluded from PKM1, which two share the same pre-messenger RNA 28-30. PKM2 is an isoenzyme of glycolytic pyruvate kinase, which is highly expressed in many tumors and participates in excess glucose metabolism and lactate production 31-33. Previous studies have shown that EGFR can promote the nuclear translocation of PKM2 34, 35, which can further enhance the Warburg effect and possibility of tumorigenesis. Therefore, based on these findings, we investigated the regulation function of EGFRvIII in PKM2 nuclear translocation and prompted us to determine the major role of EGFRvIII nuclear translocation in promoting the nuclear localization of PKM2. This is an overlooked but important discovery, because it has implications for the future treatment of tumors, especially GBM.

Materials and MethodsTumor samplesPrimary tumor samples were obtained in accordance with research ethics board approval from the Department of Pathology, San Bo Brain Hospital, Capital Medical University, Haidian District.Cell lines, reagents and processingF98 related cells and C6 rat glioma cell were purchased from the American Type Culture Collection (ATCC) and cultured in Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% fetal bovine serum (HyClone, USA) and 5% CO2 at 37°C. U251 human glioma cell was purchased from Institute of Basic Medical of Science, Chinese Academy of Medical Sciences and cultured in Eagle's Minimal Essential Medium (EMEM) supplemented with 10% fetal bovine serum and 5% CO2 at 37°C. Starvation treatment mentioned in the article refers to serum-free starvation culture for 24h and then cells were used in the subsequent experiment. Brefeldin A (BFA, 20350-15-6, ApexBio) was used in culturing cells for 30min firstly, then replaced with serum-free medium to continue culture for 24h.Plasmids ConstructionThe full-length open reading frame of EGFRvIII was amplified from pEGFRP-N1-EGFRvIII plasmid (a gift from Dr. Janusz Rak, Cancer and Angiogenesis Laboratory, MUHCRI-Montreal Children's Hospital Research Institute, Canada) and we directly cloned EGFRvIII into pHBLV-CMVIE-IRES-puro lentiviral plasmid vector, which product is abbreviated as pHBLV-EGFRvIII. On the basis of pHBLV-EGFRvIII, we knocked out the nucleic acid sequence of EGFRvIII expressing 645RRRHIVRKR653 peptide to obtain EGFRvIII-NLS-del with the loss of nuclear localization function, which product is named pHBLV-EGFRvIII-NLS-del. The cDNA of PKM2 come from U251 cell line. We first made a full RNA extraction, and then we got the open reading frame of PKM2 through reverse transcription and PCR technique. PKM2 was cloned into the PITA lentivirus plasmid vector and was added a Flag tag on the N-terminal domain, which was named PITA-Flag-PKM2. On the basis of PITA-Flag-PKM2, we mutated the Arg399/400 of PKM2 into alanine, so that PKM2 lost its nuclear transposition capability and which is named PITA-Flag-R399/400A. All of the above recombinant constructs were verified by DNA sequencing. The shsyntaxin-6-1 (5'-AACCATCAGCATAGTTGAAGCAAATCCTA-3') and shsyntaxin-6-2 (5'-GAAATCGACTGGACCACCAACGAGCTGAG-3') were inserted into lentivirus small hairpin RNA vector pSilencer2.0-u6.Glioma cells were transfected with lentivirus plasmids according to previously described methods 36. Followed by selection with puromycin for one week or G418 for ten days. The following are the screening concentrations of the plasmids and the cell lines: pHBLV (puromycin: U251, 0.5ug/ml; C6, 5ug/ml), PITA (G418: C6, 600ug/ml).Immunofluorescence analysisThe glioma cells were sequentially fixed with 4% polyformaldehyde for 15 minutes and permeabilized with 0.3%Triton-X100 for 10 minutes. Then Cells were blocked with 10% bovine serum albumin at 37°C for 1h. Finally, we incubated cells with primary antibodies overnight at 4°C, and Alex Fluor dye-conjugated secondary antibodies (Fluor®-488 or -594 goat anti-rabbit or anti-mouse IgG (ZSGB-BIO, China)) and Hoechst 33342 together for 1h at 37°C. The primary antibodies include EGFRvIII murine monoclonal antibody (#HTA0001, Beijing Cellonis Biotechnologies Co. Ltd, China), EGFR (D38B1) XP Rabbit mAb (#4267, CST), syntaxin-6 (C34B2) Rabbit mAb (#2869, CST), Anti-GM130 antibody (ab3156, abcam), Calregulin antibody (sc-11398, Santa Cruz), PKM2 (D78A4) XP Rabbit mAb (#4053P, CST). Images were captured under a TCSSP8 Confocal Microscope (Leica, Buffalo Grove, USA). The co-localization coefficient of merged images was processed with Image-Pro Plus6.0 software.Immunoblotting and immunoprecipitationRIPA lysates (P0013B, Beyotime, China) and IP lysates (P0013, Beyotime, China) are used for the extraction of protein in these two experiments respectively. The extraction procedure is based on the manufacturer's instructions. Some of the antibodies used have been described in the immunofluorescence analysis, and the rest of the antibodies include GAPDH antibody (TA-08, ZSGB-BIO, China), Lamin B1 antibody (PA532474, Pierce), Histone H3 (D1H2) XPR Rabbit mAb (#4499, CST), p-STAT3 antibody (#9145, CST), STAT3 antibody (#12640, CST), p-ERK1/2 antibody (sc-7383, Santa Cruz), ERK1/2 antibody (sc-514302, Santa Cruz), p-PKM2 ser37 (12822-1, signalway antibody). We used image J software to analyze the gray levels of the immunoblotting results.Subcellular fractionationNuclei and cytosol were isolated using a nuclear extract kit (#P0028) from a Chinese biology company Beyotime. The extraction procedure is based on the manufacturer's instruction, whose isolation process has no much difference compared with general separation steps. The extracted protein was used for immunoblotting analysis.Xenograft tumor modelsFor the intracranial xenotransplantation tumor model, 1×105(10μl) cells in PBS were inoculated into the brain of female BALB/c nude mice. All animal experiments were performed in accordance with the Guidelines of Peking University Health Science Center Animal Care and Use Committee.ImmunohistochemistryImmunohistochemistry analysis involves clinical primary glioma samples and intracranial tumorigenic tissue of nude mice. Glioma tissue samples were routinely fixed in 10% formalin and embedded in paraffin. 5μm-thick tissue sections were deparaffinized in xylene and rehydrated over a graded ethanol series. For antigen retrieval, rehydrated sections were treated in 10mM citrate buffer at pH 6.0 for 20 min in a steamer. Incubating with 3% H2O2 for 10 minutes at room temperature and goat serum for 20 minutes at 37°C in turn. Then sections were immune-stained with primary antibodies against EGFRvIII or p-STAT3. The catalogs have been described in the immunofluorescence analysis. Immunohistochemistry was performed on the Dako Autostainer Plus automated slide-processing system using the Ultravision LP Large Volume Detection System HRP Polymer (ZSGB-BIO, China) for detection of antibody binding according to the manufacturer's protocol. Slides were counterstained with hematoxylin (ZSGB-BIO, China) for several seconds.Bioluminescence imaging with IVISMice were intraperitoneally injected with 100μl of 7.5 mg/ml D-luciferin (Xenogen), and after 10 minutes they were anaesthetized with isoflurane inhalation. And then Bioluminescence imaging with a CCD camera (IVIS, Xenogen) was used for living imaging with automatic exposure time. Flurorescence emission was normalized to photons per second per centimeter squared per steradian (p/s/cm2/sr) and the final imaging results of three weeks were unified to the same scale size.Statistical analysisAll data are expressed as the mean ± SD from at least three independent experiments. The Student's two-tailed t-test with significance set at p<0.05 was used to determine statistical significance for each assay. One-way ANOVA was performed to compare more than two groups. Error bars indicate standard deviation. The software of GraphPad Prism 5 was used for statistical analysis and related mapping results.

Materials and Methods

Tumor samplesPrimary tumor samples were obtained in accordance with research ethics board approval from the Department of Pathology, San Bo Brain Hospital, Capital Medical University, Haidian District.

Tumor samples

Primary tumor samples were obtained in accordance with research ethics board approval from the Department of Pathology, San Bo Brain Hospital, Capital Medical University, Haidian District.

Cell lines, reagents and processingF98 related cells and C6 rat glioma cell were purchased from the American Type Culture Collection (ATCC) and cultured in Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% fetal bovine serum (HyClone, USA) and 5% CO2 at 37°C. U251 human glioma cell was purchased from Institute of Basic Medical of Science, Chinese Academy of Medical Sciences and cultured in Eagle's Minimal Essential Medium (EMEM) supplemented with 10% fetal bovine serum and 5% CO2 at 37°C. Starvation treatment mentioned in the article refers to serum-free starvation culture for 24h and then cells were used in the subsequent experiment. Brefeldin A (BFA, 20350-15-6, ApexBio) was used in culturing cells for 30min firstly, then replaced with serum-free medium to continue culture for 24h.

Cell lines, reagents and processing

F98 related cells and C6 rat glioma cell were purchased from the American Type Culture Collection (ATCC) and cultured in Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% fetal bovine serum (HyClone, USA) and 5% CO2 at 37°C. U251 human glioma cell was purchased from Institute of Basic Medical of Science, Chinese Academy of Medical Sciences and cultured in Eagle's Minimal Essential Medium (EMEM) supplemented with 10% fetal bovine serum and 5% CO2 at 37°C. Starvation treatment mentioned in the article refers to serum-free starvation culture for 24h and then cells were used in the subsequent experiment. Brefeldin A (BFA, 20350-15-6, ApexBio) was used in culturing cells for 30min firstly, then replaced with serum-free medium to continue culture for 24h.

Plasmids ConstructionThe full-length open reading frame of EGFRvIII was amplified from pEGFRP-N1-EGFRvIII plasmid (a gift from Dr. Janusz Rak, Cancer and Angiogenesis Laboratory, MUHCRI-Montreal Children's Hospital Research Institute, Canada) and we directly cloned EGFRvIII into pHBLV-CMVIE-IRES-puro lentiviral plasmid vector, which product is abbreviated as pHBLV-EGFRvIII. On the basis of pHBLV-EGFRvIII, we knocked out the nucleic acid sequence of EGFRvIII expressing 645RRRHIVRKR653 peptide to obtain EGFRvIII-NLS-del with the loss of nuclear localization function, which product is named pHBLV-EGFRvIII-NLS-del. The cDNA of PKM2 come from U251 cell line. We first made a full RNA extraction, and then we got the open reading frame of PKM2 through reverse transcription and PCR technique. PKM2 was cloned into the PITA lentivirus plasmid vector and was added a Flag tag on the N-terminal domain, which was named PITA-Flag-PKM2. On the basis of PITA-Flag-PKM2, we mutated the Arg399/400 of PKM2 into alanine, so that PKM2 lost its nuclear transposition capability and which is named PITA-Flag-R399/400A. All of the above recombinant constructs were verified by DNA sequencing. The shsyntaxin-6-1 (5'-AACCATCAGCATAGTTGAAGCAAATCCTA-3') and shsyntaxin-6-2 (5'-GAAATCGACTGGACCACCAACGAGCTGAG-3') were inserted into lentivirus small hairpin RNA vector pSilencer2.0-u6.Glioma cells were transfected with lentivirus plasmids according to previously described methods 36. Followed by selection with puromycin for one week or G418 for ten days. The following are the screening concentrations of the plasmids and the cell lines: pHBLV (puromycin: U251, 0.5ug/ml; C6, 5ug/ml), PITA (G418: C6, 600ug/ml).

Plasmids Construction

The full-length open reading frame of EGFRvIII was amplified from pEGFRP-N1-EGFRvIII plasmid (a gift from Dr. Janusz Rak, Cancer and Angiogenesis Laboratory, MUHCRI-Montreal Children's Hospital Research Institute, Canada) and we directly cloned EGFRvIII into pHBLV-CMVIE-IRES-puro lentiviral plasmid vector, which product is abbreviated as pHBLV-EGFRvIII. On the basis of pHBLV-EGFRvIII, we knocked out the nucleic acid sequence of EGFRvIII expressing 645RRRHIVRKR653 peptide to obtain EGFRvIII-NLS-del with the loss of nuclear localization function, which product is named pHBLV-EGFRvIII-NLS-del. The cDNA of PKM2 come from U251 cell line. We first made a full RNA extraction, and then we got the open reading frame of PKM2 through reverse transcription and PCR technique. PKM2 was cloned into the PITA lentivirus plasmid vector and was added a Flag tag on the N-terminal domain, which was named PITA-Flag-PKM2. On the basis of PITA-Flag-PKM2, we mutated the Arg399/400 of PKM2 into alanine, so that PKM2 lost its nuclear transposition capability and which is named PITA-Flag-R399/400A. All of the above recombinant constructs were verified by DNA sequencing. The shsyntaxin-6-1 (5'-AACCATCAGCATAGTTGAAGCAAATCCTA-3') and shsyntaxin-6-2 (5'-GAAATCGACTGGACCACCAACGAGCTGAG-3') were inserted into lentivirus small hairpin RNA vector pSilencer2.0-u6.

Glioma cells were transfected with lentivirus plasmids according to previously described methods 36. Followed by selection with puromycin for one week or G418 for ten days. The following are the screening concentrations of the plasmids and the cell lines: pHBLV (puromycin: U251, 0.5ug/ml; C6, 5ug/ml), PITA (G418: C6, 600ug/ml).

Immunofluorescence analysisThe glioma cells were sequentially fixed with 4% polyformaldehyde for 15 minutes and permeabilized with 0.3%Triton-X100 for 10 minutes. Then Cells were blocked with 10% bovine serum albumin at 37°C for 1h. Finally, we incubated cells with primary antibodies overnight at 4°C, and Alex Fluor dye-conjugated secondary antibodies (Fluor®-488 or -594 goat anti-rabbit or anti-mouse IgG (ZSGB-BIO, China)) and Hoechst 33342 together for 1h at 37°C. The primary antibodies include EGFRvIII murine monoclonal antibody (#HTA0001, Beijing Cellonis Biotechnologies Co. Ltd, China), EGFR (D38B1) XP Rabbit mAb (#4267, CST), syntaxin-6 (C34B2) Rabbit mAb (#2869, CST), Anti-GM130 antibody (ab3156, abcam), Calregulin antibody (sc-11398, Santa Cruz), PKM2 (D78A4) XP Rabbit mAb (#4053P, CST). Images were captured under a TCSSP8 Confocal Microscope (Leica, Buffalo Grove, USA). The co-localization coefficient of merged images was processed with Image-Pro Plus6.0 software.

Immunofluorescence analysis

The glioma cells were sequentially fixed with 4% polyformaldehyde for 15 minutes and permeabilized with 0.3%Triton-X100 for 10 minutes. Then Cells were blocked with 10% bovine serum albumin at 37°C for 1h. Finally, we incubated cells with primary antibodies overnight at 4°C, and Alex Fluor dye-conjugated secondary antibodies (Fluor®-488 or -594 goat anti-rabbit or anti-mouse IgG (ZSGB-BIO, China)) and Hoechst 33342 together for 1h at 37°C. The primary antibodies include EGFRvIII murine monoclonal antibody (#HTA0001, Beijing Cellonis Biotechnologies Co. Ltd, China), EGFR (D38B1) XP Rabbit mAb (#4267, CST), syntaxin-6 (C34B2) Rabbit mAb (#2869, CST), Anti-GM130 antibody (ab3156, abcam), Calregulin antibody (sc-11398, Santa Cruz), PKM2 (D78A4) XP Rabbit mAb (#4053P, CST). Images were captured under a TCSSP8 Confocal Microscope (Leica, Buffalo Grove, USA). The co-localization coefficient of merged images was processed with Image-Pro Plus6.0 software.

Immunoblotting and immunoprecipitationRIPA lysates (P0013B, Beyotime, China) and IP lysates (P0013, Beyotime, China) are used for the extraction of protein in these two experiments respectively. The extraction procedure is based on the manufacturer's instructions. Some of the antibodies used have been described in the immunofluorescence analysis, and the rest of the antibodies include GAPDH antibody (TA-08, ZSGB-BIO, China), Lamin B1 antibody (PA532474, Pierce), Histone H3 (D1H2) XPR Rabbit mAb (#4499, CST), p-STAT3 antibody (#9145, CST), STAT3 antibody (#12640, CST), p-ERK1/2 antibody (sc-7383, Santa Cruz), ERK1/2 antibody (sc-514302, Santa Cruz), p-PKM2 ser37 (12822-1, signalway antibody). We used image J software to analyze the gray levels of the immunoblotting results.

Immunoblotting and immunoprecipitation

RIPA lysates (P0013B, Beyotime, China) and IP lysates (P0013, Beyotime, China) are used for the extraction of protein in these two experiments respectively. The extraction procedure is based on the manufacturer's instructions. Some of the antibodies used have been described in the immunofluorescence analysis, and the rest of the antibodies include GAPDH antibody (TA-08, ZSGB-BIO, China), Lamin B1 antibody (PA532474, Pierce), Histone H3 (D1H2) XPR Rabbit mAb (#4499, CST), p-STAT3 antibody (#9145, CST), STAT3 antibody (#12640, CST), p-ERK1/2 antibody (sc-7383, Santa Cruz), ERK1/2 antibody (sc-514302, Santa Cruz), p-PKM2 ser37 (12822-1, signalway antibody). We used image J software to analyze the gray levels of the immunoblotting results.

Subcellular fractionationNuclei and cytosol were isolated using a nuclear extract kit (#P0028) from a Chinese biology company Beyotime. The extraction procedure is based on the manufacturer's instruction, whose isolation process has no much difference compared with general separation steps. The extracted protein was used for immunoblotting analysis.

Subcellular fractionation

Nuclei and cytosol were isolated using a nuclear extract kit (#P0028) from a Chinese biology company Beyotime. The extraction procedure is based on the manufacturer's instruction, whose isolation process has no much difference compared with general separation steps. The extracted protein was used for immunoblotting analysis.

Xenograft tumor modelsFor the intracranial xenotransplantation tumor model, 1×105(10μl) cells in PBS were inoculated into the brain of female BALB/c nude mice. All animal experiments were performed in accordance with the Guidelines of Peking University Health Science Center Animal Care and Use Committee.

Xenograft tumor models

For the intracranial xenotransplantation tumor model, 1×105(10μl) cells in PBS were inoculated into the brain of female BALB/c nude mice. All animal experiments were performed in accordance with the Guidelines of Peking University Health Science Center Animal Care and Use Committee.

ImmunohistochemistryImmunohistochemistry analysis involves clinical primary glioma samples and intracranial tumorigenic tissue of nude mice. Glioma tissue samples were routinely fixed in 10% formalin and embedded in paraffin. 5μm-thick tissue sections were deparaffinized in xylene and rehydrated over a graded ethanol series. For antigen retrieval, rehydrated sections were treated in 10mM citrate buffer at pH 6.0 for 20 min in a steamer. Incubating with 3% H2O2 for 10 minutes at room temperature and goat serum for 20 minutes at 37°C in turn. Then sections were immune-stained with primary antibodies against EGFRvIII or p-STAT3. The catalogs have been described in the immunofluorescence analysis. Immunohistochemistry was performed on the Dako Autostainer Plus automated slide-processing system using the Ultravision LP Large Volume Detection System HRP Polymer (ZSGB-BIO, China) for detection of antibody binding according to the manufacturer's protocol. Slides were counterstained with hematoxylin (ZSGB-BIO, China) for several seconds.

Immunohistochemistry

Immunohistochemistry analysis involves clinical primary glioma samples and intracranial tumorigenic tissue of nude mice. Glioma tissue samples were routinely fixed in 10% formalin and embedded in paraffin. 5μm-thick tissue sections were deparaffinized in xylene and rehydrated over a graded ethanol series. For antigen retrieval, rehydrated sections were treated in 10mM citrate buffer at pH 6.0 for 20 min in a steamer. Incubating with 3% H2O2 for 10 minutes at room temperature and goat serum for 20 minutes at 37°C in turn. Then sections were immune-stained with primary antibodies against EGFRvIII or p-STAT3. The catalogs have been described in the immunofluorescence analysis. Immunohistochemistry was performed on the Dako Autostainer Plus automated slide-processing system using the Ultravision LP Large Volume Detection System HRP Polymer (ZSGB-BIO, China) for detection of antibody binding according to the manufacturer's protocol. Slides were counterstained with hematoxylin (ZSGB-BIO, China) for several seconds.

Bioluminescence imaging with IVISMice were intraperitoneally injected with 100μl of 7.5 mg/ml D-luciferin (Xenogen), and after 10 minutes they were anaesthetized with isoflurane inhalation. And then Bioluminescence imaging with a CCD camera (IVIS, Xenogen) was used for living imaging with automatic exposure time. Flurorescence emission was normalized to photons per second per centimeter squared per steradian (p/s/cm2/sr) and the final imaging results of three weeks were unified to the same scale size.

Bioluminescence imaging with IVIS

Mice were intraperitoneally injected with 100μl of 7.5 mg/ml D-luciferin (Xenogen), and after 10 minutes they were anaesthetized with isoflurane inhalation. And then Bioluminescence imaging with a CCD camera (IVIS, Xenogen) was used for living imaging with automatic exposure time. Flurorescence emission was normalized to photons per second per centimeter squared per steradian (p/s/cm2/sr) and the final imaging results of three weeks were unified to the same scale size.

Statistical analysisAll data are expressed as the mean ± SD from at least three independent experiments. The Student's two-tailed t-test with significance set at p<0.05 was used to determine statistical significance for each assay. One-way ANOVA was performed to compare more than two groups. Error bars indicate standard deviation. The software of GraphPad Prism 5 was used for statistical analysis and related mapping results.

Statistical analysis

All data are expressed as the mean ± SD from at least three independent experiments. The Student's two-tailed t-test with significance set at p<0.05 was used to determine statistical significance for each assay. One-way ANOVA was performed to compare more than two groups. Error bars indicate standard deviation. The software of GraphPad Prism 5 was used for statistical analysis and related mapping results.

ResultsEGFRvIII nuclear localization is evident in primary human glioblastoma and glioma cellsTo verify that EGFRvIII undergoes nuclear translocation, we stained for EGFRvIII in a series of human primary glioblastoma tissues. EGFRvIII experienced nuclear translocation in nearly every tissue sample; we present the typical nuclear translocation results in three different glioma patients' samples (Fig. 1A). According to previous results, EGFRvIII nuclear localization will lead to downstream gene transcription, cell proliferation, drug inhibition, and other effects, contributing to a poor prognosis in patients with glioma. However, the mechanism by which EGFRvIII is transported into the nucleus has not been determined, although EGFR's nuclear translocation has been reported in many studies. The aim of this study was to identify the pathway of EGFRvIII nuclear transport and examine how EGFRvIII nuclear translocation affects downstream signaling molecules to find new therapeutic interventions for the treatment of glioma.Prior to our research, we constructed a series of cell lines based on the U251 human glioma cell line and C6 rat glioma cell line. Originally, there was no EGFRvIII expression in the U251 and C6 cells. And thus, we overexpressed EGFRvIII and EGFRvIII-NLS-del, respectively, in these lines by lentiviral infection. EGFRvIII-NLS-del has a partial deletion in the nuclear localization signal of EGFRvIII, whose nuclear localization sequence is distributed throughout the juxta-membrane domain (Fig. 1B).By western blot, using EGFRvIII and EGFR antibodies, we identified successfully constructed cell lines, which we named U251EGFRvIII, U251EGFRvIII-NLS-del, C6EGFRvIII, and C6EGFRvIII-NLS-del; The F98, F98EGFR, and F98npEGFRvIII cell lines were purchased from ATCC as references (Fig. 1C). EGFRvIII antibody can only recognize EGFRvIII and its mutant form, EGFRvIII-NLS-del, whereas EGFR antibody can recognize EGFR, EGFRvIII, and EGFRvIII-NLS-del. The immunofluorescence staining results (Fig. 1D), performed only with EGFRvIII antibody, were consistent with the western blot findings and again suggested we have successfully constructed the target cell lines.As a constitutively active protein, the nuclear translocation of EGFRvIII does not require ligands stimulation. According to previous studies, EGFRvIII's nuclear translocation is increased by serum-free starvation (referred to as “starvation” in all figures) for 12-24 hours 17, 37. This model is also based on the fact that glioma cells are sometimes subjected to nutrient deprivation in vivo.Compared with normal culture conditions, the amount of nuclear EGFRvIII in U251EGFRvIII cells rose after serum-free starvation for 24h in western-blot result, whereas nuclear EGFRvIII-NLS-del was not detected in U251EGFRvIII-NLS-del, indicating that EGFRvIII-NLS-del, with its partially deleted nuclear localization signal, did not undergo nuclear translocation (Fig. 1E). We also performed an immunoblot analysis of nuclear EGFRvIII by using histone H3 as a nuclear marker in U251EGFRvIII, and obtained a consistent EGFRvIII nuclear translocation result, indicating that EGFRvIII can indeed enter the nuclear (Fig. S1-A). Figure 1F shows the immunofluorescence results of U251EGFRvIII and U251EGFRvIII-NLS-del cells, demonstrating that nuclear translocation occurs in U251EGFRvIII but rarely in U251EGFRvIII-NLS-del cells after serum-free starvation for 24h. To be more rigorous, we performed sequential confocal imaging of EGFRvIII through the sections of the nucleus in U251EGFRvIII and U251EGFRvIII-NLS-del (Fig. S1-B), and results showed that EGFRvIII can enter the nucleus instead of EGFRvIII-NLS-del. Figure 1G and 1H, respectively, showed the western blot and immunofluorescence results about EGFRvIII's nuclear transloncation in C6EGFRvIII and C6EGFRvIII-NLS-del cells, also revealing contrasting nuclear translocation results of EGFRvIII and EGFRvIII-NLS-del in corresponding cell lines. In C6EGFRvIII and C6EGFRvIII-NLS-del, we also performed immunoblot analysis of nuclear EGFRvIII by using histone H3 as a nuclear marker (Fig. S2-A) and sequential confocal imaging of EGFRvIII through the sections of the nucleus (Fig. S2-B), which all have consistent experimental verification results about EGFRvIII's nuclear translocation.These results suggest that EGFRvIII has a nuclear transposition in the clinical tumor tissues and glioma cell lines, and the nuclear transposition was blocked when we knocked out its nuclear location sequence.Golgi, especially syntaxin-6, participates in EGFRvIII's reverse transposition to the nucleusTo examine whether the Golgi is involved in EGFRvIII's retrograde trafficking, we first asked whether serum-free starvation can stimulate EGFRvIII's localization on the Golgi. We performed double staining for EGFRvIII and syntaxin-6 (STX6), which is a marker of trans Golgi and is involved in protein transport, and then performed imaging with confocal microscopy, by which we observed that EGFRvIII localized on the Golgi apparatus and after serum-free stimulation for 24h, their co-localization signals were enhanced (Fig. 2A). Rigorously, we carried out a statistical analysis of syntaxin-6's positive area with and without serum-free starvation and results showed that serum-free starvation didn't expand the positive region of syntaxin-6 in U251EGFRvIII or C6EGFRvIII (Fig. S3). Besides, we performed reciprocal IP-western blot and observed that serum-free starvation improved the co-localization of EGFRvIII and syntaxin-6 in U251EGFRvIII (Fig. 2B). We did not detect any increase in the amount of their co-localization in U251EGFRvIII-NLS-del, further suggesting that syntaxin-6 is likely to be involved in the nuclear translocation of EGFRvIII. The same co-localization results were validated between EGFRvIII and GM130, a Golgi endosomal marker, by confocal fluorescence microscopy (Fig. 2C).We then studied the function of syntaxin-6 in EGFRvIII's nuclear translocation. To generate positive or negative contrast results, we produced two shRNA constructs targeting syntaxin-6. Figure 2D shows that both compounds have good knockdown efficacy against syntaxin-6.As indicated in Figure 2E, compared with the control group, the amount of nuclear EGFRvIII decreased when syntaxin-6 was down-regulated by shRNA in U251EGFRvIII cells, showing that syntaxin-6 positively regulates the reverse transport of EGFRvIII to the nucleus. Similarly, the immunofluorescence results in Figure 2F also demonstrated that syntaxin-6 is closely related to the nuclear transport of EGFRvIII, which differed significantly between the control and shRNA groups in the subsequent statistical analysis.These results suggest that Golgi, in particular syntaxin-6, has a positive effect on EGFRvIII's reverse transposition to the nucleus.COPI-mediated retrograde trafficking process from Golgi to ER is associated with the nuclear translocation of EGFRvIIICOPI is a pivotal retrograde transporting complex that transfers proteins from cis Golgi to the endoplasmic reticulum. We then examined whether COPI can transfer EGFRvIII from the Golgi apparatus to ER organelles. By double immunofluorescence staining of EGFRvIII with the ER marker calregulin, the localization of EGFRvIII to the ER declined after the addition of BFA compared with serum-free starvation and the DMSO control group (Fig. 3A). BFA is an assembly inhibitor that blocks the transport function of COPI. Moreover, we compared syntaxin-6 control groups with its shRNA group by immunofluorescence imaging and observed that serum-free starvation provoked the co-localization of EGFRvIII and calregulin, while their co-location signal decreased when the expression of syntaxin-6 was down-regulated by shRNA (Fig. 3B). These data indicate that COPI participates in the transport of EGFRvIII from the Golgi apparatus to ER.To validate whether COPI is involved in the nuclear translocation of EGFRvIII, we performed immunoblotting analysis, in which BFA was administered to inhibit the activation of ARF1-guanine exchange factors, resulting in the disassembly of the COPI coat on the Golgi; DMSO was used as a control (Fig. 3C). Results showed that compared with the serum-free starvation or DMSO control group, BFA reduced the amount of EGFRvIII in the nucleus. The immunofluorescence results in Figure 3D provided further support that pretreatment with BFA in U251EGFRvIII decreased the cell number of EGFRvIII's nuclear translocation, which suggested that intact COPI is required for EGFRvIII's nuclear translocation.These results strongly indicate that COPI mediates the retrograde trafficking of EGFRvIII from the Golgi to ER, and meanwhile COPI participates in the reverse nuclear transport of EGFRvIII.EGFRvIII's nuclear positioning can increase the phosphorylation level of STAT3As seen in the immunoblotting results in Figure 4A, EGFRvIII, especially under serum-free starvation culture condition, increased the levels of p-STAT3, which is the active form of STAT3. This result suggests that EGFRvIII's nuclear translocation may be closely related to the activation of STAT3. Subsequently, we confirmed this association in the following experiment. In U251EGFRvIII cells, serum-free starvation up-regulated p-STAT3, and meanwhile, compared with the DMSO control group, BFA inhibited the phosphorylation level of STAT3 (Fig. 4B). Existing researches showed that phosphorylated STAT3 often translocate to the nucleus to play its transcriptional regulation function 38-40. Thus, we performed nuclear and non-nuclear fractionation experiments and detected the level of nuclear p-STAT3 with western blot experiment. EGFRvIII promoted nuclear distribution of p-STAT3 in C6EGFRvIII cells versus in C6 cells, especially under serum-free starvation culture, whereas this increase was not observed in C6EGFRvIII-NLS-del cells (Fig. 4C).Next, we separately pretreated U251, U251EGFRvIII, and U251EGFRvIII-NLS-del cells in normal culture or under serum-free starvation culture and found that the latter culture condition significantly up-regulated p-STAT3 in U251EGFRvIII but not in U251 or U251EGFRvIII-NLS-del cells (Fig. 4D). We obtained similar outcomes in C6, C6EGFRvIII, and C6EGFRvIII-NLS-del cells (Fig. 4E).In summary, EGFRvIII's nuclear translocation can enhance the phosphorylation level and nuclear distribution of STAT3, which has a great help for STAT3 to play its transcriptional regulatory function in the nucleus.Nuclear translocation of EGFRvIII enhances nuclear localization of PKM2In U251EGFRvIII cells, serum-free starvation improved the cell number of PKM2's nuclear localization compared with normal culture, whereas no such increase occurred in U251EGFRvIII-NLS-del cells (Fig. 5A). We then compared nuclear PKM2 levels by immunoblotting analysis in U251, U251EGFRvIII, and U251EGFRvIII-NLS-del cells. The amount of nuclear PKM2 in U251EGFRvIII cells was significantly higher than in U251 and U251EGFRvIII-NLS-del cells in the same serum-free starvation environment, which environment can increase the nuclear translocation of EGFRvIII (Fig. 5B). Figure 5C and 5D show additional immunofluorescence and western blot results in C6-associated cell lines, confirming the same changes in PKM2's nuclear translocation. Thus, we can say that the EGFRvIII with the complete nuclear transposition function rather than the EGFRvIII-NLS-del can further promote the nuclear translocation of PKM2.In U251EGFRvIII, we performed further validation with EGFRvIII's nuclear translocation inhibitor BFA. Results showed that, when EGFRVIII's nuclear translocation was inhibited by BFA, the amount of nuclear PKM2 also decreased compared with the serum-free starvation group and DMSO control group (Fig.5E).To further confirm the role of EGFRvIII's nuclear translocation in regulating PKM2 entry into the nucleus, we constructed two cell lines for subsequent experiments. We respectively overexpressed Flag-R399/400A in C6EGFRvIII and C6EGFRvIII-NLS-del cells via lentiviral infection; R399/400A is a mutant of PKM2 in which Arg399/400 is changed to alanine, causing a failure in PKM2's nuclear translocation. Figure 5F indicated that, in the same serum-free starvation environment, EGFRvIII enhanced the nuclear translocation of Flag-R399/400A more effectively than EGFRvIII-NLS-del.In conclusion, EGFRVIII's nuclear translocation can increase the localization of PKM2 in the nucleus, while nuclear PKM2 is an important prerequisite for its many regulatory functions.EGFRvIII's nuclear translocation can promote the progression of glioblastoma in vivoNext, we determined the effects of EGFRvIII's nuclear translocation on the development of GBM.C6, C6EGFRvIII, and C6EGFRvIII-NLS-del cells that stably overexpress luciferase were injected separately into the brains of nude mice, and these three kinds of GBM animals were imaged through a luciferase reporter assay device on days 7, 14, and 21 after heterologous transplantation. The luminescence signals in the C6EGFRvIII group were higher than the C6 and C6EGFRvIII-NLS-del groups in the three imaging results, and meanwhile the signal intensity in C6EGFRvIII-NLS-del cells exceeded that in C6 cells (Fig. 6A); the statistical results describe the trend in signal intensity in the three groups (Fig. 6B). In addition, we sampled the above three kinds of tumorigenic brain tissues and analyzed them by immune-histochemical staining. Figure 6C showed the p-STAT3 level in three tissues, and the highest expression of p-STAT3 occurred in EGFRvIII- overexpressing tissues.Compared with C6 and C6EGFRvIII-NLS-del groups, EGFRvIII, having intact nuclear transposition ability, in C6EGFRvIII accelerated disease progression significantly. These results are consistent with previous studies that EGFRvIII, as a constitutively active protein, particularly with regard to its nuclear translocation, provokes the occurrence and progression of GBM 3, 41, 42.

EGFRvIII nuclear localization is evident in primary human glioblastoma and glioma cellsTo verify that EGFRvIII undergoes nuclear translocation, we stained for EGFRvIII in a series of human primary glioblastoma tissues. EGFRvIII experienced nuclear translocation in nearly every tissue sample; we present the typical nuclear translocation results in three different glioma patients' samples (Fig. 1A). According to previous results, EGFRvIII nuclear localization will lead to downstream gene transcription, cell proliferation, drug inhibition, and other effects, contributing to a poor prognosis in patients with glioma. However, the mechanism by which EGFRvIII is transported into the nucleus has not been determined, although EGFR's nuclear translocation has been reported in many studies. The aim of this study was to identify the pathway of EGFRvIII nuclear transport and examine how EGFRvIII nuclear translocation affects downstream signaling molecules to find new therapeutic interventions for the treatment of glioma.Prior to our research, we constructed a series of cell lines based on the U251 human glioma cell line and C6 rat glioma cell line. Originally, there was no EGFRvIII expression in the U251 and C6 cells. And thus, we overexpressed EGFRvIII and EGFRvIII-NLS-del, respectively, in these lines by lentiviral infection. EGFRvIII-NLS-del has a partial deletion in the nuclear localization signal of EGFRvIII, whose nuclear localization sequence is distributed throughout the juxta-membrane domain (Fig. 1B).By western blot, using EGFRvIII and EGFR antibodies, we identified successfully constructed cell lines, which we named U251EGFRvIII, U251EGFRvIII-NLS-del, C6EGFRvIII, and C6EGFRvIII-NLS-del; The F98, F98EGFR, and F98npEGFRvIII cell lines were purchased from ATCC as references (Fig. 1C). EGFRvIII antibody can only recognize EGFRvIII and its mutant form, EGFRvIII-NLS-del, whereas EGFR antibody can recognize EGFR, EGFRvIII, and EGFRvIII-NLS-del. The immunofluorescence staining results (Fig. 1D), performed only with EGFRvIII antibody, were consistent with the western blot findings and again suggested we have successfully constructed the target cell lines.As a constitutively active protein, the nuclear translocation of EGFRvIII does not require ligands stimulation. According to previous studies, EGFRvIII's nuclear translocation is increased by serum-free starvation (referred to as “starvation” in all figures) for 12-24 hours 17, 37. This model is also based on the fact that glioma cells are sometimes subjected to nutrient deprivation in vivo.Compared with normal culture conditions, the amount of nuclear EGFRvIII in U251EGFRvIII cells rose after serum-free starvation for 24h in western-blot result, whereas nuclear EGFRvIII-NLS-del was not detected in U251EGFRvIII-NLS-del, indicating that EGFRvIII-NLS-del, with its partially deleted nuclear localization signal, did not undergo nuclear translocation (Fig. 1E). We also performed an immunoblot analysis of nuclear EGFRvIII by using histone H3 as a nuclear marker in U251EGFRvIII, and obtained a consistent EGFRvIII nuclear translocation result, indicating that EGFRvIII can indeed enter the nuclear (Fig. S1-A). Figure 1F shows the immunofluorescence results of U251EGFRvIII and U251EGFRvIII-NLS-del cells, demonstrating that nuclear translocation occurs in U251EGFRvIII but rarely in U251EGFRvIII-NLS-del cells after serum-free starvation for 24h. To be more rigorous, we performed sequential confocal imaging of EGFRvIII through the sections of the nucleus in U251EGFRvIII and U251EGFRvIII-NLS-del (Fig. S1-B), and results showed that EGFRvIII can enter the nucleus instead of EGFRvIII-NLS-del. Figure 1G and 1H, respectively, showed the western blot and immunofluorescence results about EGFRvIII's nuclear transloncation in C6EGFRvIII and C6EGFRvIII-NLS-del cells, also revealing contrasting nuclear translocation results of EGFRvIII and EGFRvIII-NLS-del in corresponding cell lines. In C6EGFRvIII and C6EGFRvIII-NLS-del, we also performed immunoblot analysis of nuclear EGFRvIII by using histone H3 as a nuclear marker (Fig. S2-A) and sequential confocal imaging of EGFRvIII through the sections of the nucleus (Fig. S2-B), which all have consistent experimental verification results about EGFRvIII's nuclear translocation.These results suggest that EGFRvIII has a nuclear transposition in the clinical tumor tissues and glioma cell lines, and the nuclear transposition was blocked when we knocked out its nuclear location sequence.

EGFRvIII nuclear localization is evident in primary human glioblastoma and glioma cells

To verify that EGFRvIII undergoes nuclear translocation, we stained for EGFRvIII in a series of human primary glioblastoma tissues. EGFRvIII experienced nuclear translocation in nearly every tissue sample; we present the typical nuclear translocation results in three different glioma patients' samples (Fig. 1A). According to previous results, EGFRvIII nuclear localization will lead to downstream gene transcription, cell proliferation, drug inhibition, and other effects, contributing to a poor prognosis in patients with glioma. However, the mechanism by which EGFRvIII is transported into the nucleus has not been determined, although EGFR's nuclear translocation has been reported in many studies. The aim of this study was to identify the pathway of EGFRvIII nuclear transport and examine how EGFRvIII nuclear translocation affects downstream signaling molecules to find new therapeutic interventions for the treatment of glioma.

Prior to our research, we constructed a series of cell lines based on the U251 human glioma cell line and C6 rat glioma cell line. Originally, there was no EGFRvIII expression in the U251 and C6 cells. And thus, we overexpressed EGFRvIII and EGFRvIII-NLS-del, respectively, in these lines by lentiviral infection. EGFRvIII-NLS-del has a partial deletion in the nuclear localization signal of EGFRvIII, whose nuclear localization sequence is distributed throughout the juxta-membrane domain (Fig. 1B).

By western blot, using EGFRvIII and EGFR antibodies, we identified successfully constructed cell lines, which we named U251EGFRvIII, U251EGFRvIII-NLS-del, C6EGFRvIII, and C6EGFRvIII-NLS-del; The F98, F98EGFR, and F98npEGFRvIII cell lines were purchased from ATCC as references (Fig. 1C). EGFRvIII antibody can only recognize EGFRvIII and its mutant form, EGFRvIII-NLS-del, whereas EGFR antibody can recognize EGFR, EGFRvIII, and EGFRvIII-NLS-del. The immunofluorescence staining results (Fig. 1D), performed only with EGFRvIII antibody, were consistent with the western blot findings and again suggested we have successfully constructed the target cell lines.

As a constitutively active protein, the nuclear translocation of EGFRvIII does not require ligands stimulation. According to previous studies, EGFRvIII's nuclear translocation is increased by serum-free starvation (referred to as “starvation” in all figures) for 12-24 hours 17, 37. This model is also based on the fact that glioma cells are sometimes subjected to nutrient deprivation in vivo.

Compared with normal culture conditions, the amount of nuclear EGFRvIII in U251EGFRvIII cells rose after serum-free starvation for 24h in western-blot result, whereas nuclear EGFRvIII-NLS-del was not detected in U251EGFRvIII-NLS-del, indicating that EGFRvIII-NLS-del, with its partially deleted nuclear localization signal, did not undergo nuclear translocation (Fig. 1E). We also performed an immunoblot analysis of nuclear EGFRvIII by using histone H3 as a nuclear marker in U251EGFRvIII, and obtained a consistent EGFRvIII nuclear translocation result, indicating that EGFRvIII can indeed enter the nuclear (Fig. S1-A). Figure 1F shows the immunofluorescence results of U251EGFRvIII and U251EGFRvIII-NLS-del cells, demonstrating that nuclear translocation occurs in U251EGFRvIII but rarely in U251EGFRvIII-NLS-del cells after serum-free starvation for 24h. To be more rigorous, we performed sequential confocal imaging of EGFRvIII through the sections of the nucleus in U251EGFRvIII and U251EGFRvIII-NLS-del (Fig. S1-B), and results showed that EGFRvIII can enter the nucleus instead of EGFRvIII-NLS-del. Figure 1G and 1H, respectively, showed the western blot and immunofluorescence results about EGFRvIII's nuclear transloncation in C6EGFRvIII and C6EGFRvIII-NLS-del cells, also revealing contrasting nuclear translocation results of EGFRvIII and EGFRvIII-NLS-del in corresponding cell lines. In C6EGFRvIII and C6EGFRvIII-NLS-del, we also performed immunoblot analysis of nuclear EGFRvIII by using histone H3 as a nuclear marker (Fig. S2-A) and sequential confocal imaging of EGFRvIII through the sections of the nucleus (Fig. S2-B), which all have consistent experimental verification results about EGFRvIII's nuclear translocation.

These results suggest that EGFRvIII has a nuclear transposition in the clinical tumor tissues and glioma cell lines, and the nuclear transposition was blocked when we knocked out its nuclear location sequence.

Golgi, especially syntaxin-6, participates in EGFRvIII's reverse transposition to the nucleusTo examine whether the Golgi is involved in EGFRvIII's retrograde trafficking, we first asked whether serum-free starvation can stimulate EGFRvIII's localization on the Golgi. We performed double staining for EGFRvIII and syntaxin-6 (STX6), which is a marker of trans Golgi and is involved in protein transport, and then performed imaging with confocal microscopy, by which we observed that EGFRvIII localized on the Golgi apparatus and after serum-free stimulation for 24h, their co-localization signals were enhanced (Fig. 2A). Rigorously, we carried out a statistical analysis of syntaxin-6's positive area with and without serum-free starvation and results showed that serum-free starvation didn't expand the positive region of syntaxin-6 in U251EGFRvIII or C6EGFRvIII (Fig. S3). Besides, we performed reciprocal IP-western blot and observed that serum-free starvation improved the co-localization of EGFRvIII and syntaxin-6 in U251EGFRvIII (Fig. 2B). We did not detect any increase in the amount of their co-localization in U251EGFRvIII-NLS-del, further suggesting that syntaxin-6 is likely to be involved in the nuclear translocation of EGFRvIII. The same co-localization results were validated between EGFRvIII and GM130, a Golgi endosomal marker, by confocal fluorescence microscopy (Fig. 2C).We then studied the function of syntaxin-6 in EGFRvIII's nuclear translocation. To generate positive or negative contrast results, we produced two shRNA constructs targeting syntaxin-6. Figure 2D shows that both compounds have good knockdown efficacy against syntaxin-6.As indicated in Figure 2E, compared with the control group, the amount of nuclear EGFRvIII decreased when syntaxin-6 was down-regulated by shRNA in U251EGFRvIII cells, showing that syntaxin-6 positively regulates the reverse transport of EGFRvIII to the nucleus. Similarly, the immunofluorescence results in Figure 2F also demonstrated that syntaxin-6 is closely related to the nuclear transport of EGFRvIII, which differed significantly between the control and shRNA groups in the subsequent statistical analysis.These results suggest that Golgi, in particular syntaxin-6, has a positive effect on EGFRvIII's reverse transposition to the nucleus.

Golgi, especially syntaxin-6, participates in EGFRvIII's reverse transposition to the nucleus

To examine whether the Golgi is involved in EGFRvIII's retrograde trafficking, we first asked whether serum-free starvation can stimulate EGFRvIII's localization on the Golgi. We performed double staining for EGFRvIII and syntaxin-6 (STX6), which is a marker of trans Golgi and is involved in protein transport, and then performed imaging with confocal microscopy, by which we observed that EGFRvIII localized on the Golgi apparatus and after serum-free stimulation for 24h, their co-localization signals were enhanced (Fig. 2A). Rigorously, we carried out a statistical analysis of syntaxin-6's positive area with and without serum-free starvation and results showed that serum-free starvation didn't expand the positive region of syntaxin-6 in U251EGFRvIII or C6EGFRvIII (Fig. S3). Besides, we performed reciprocal IP-western blot and observed that serum-free starvation improved the co-localization of EGFRvIII and syntaxin-6 in U251EGFRvIII (Fig. 2B). We did not detect any increase in the amount of their co-localization in U251EGFRvIII-NLS-del, further suggesting that syntaxin-6 is likely to be involved in the nuclear translocation of EGFRvIII. The same co-localization results were validated between EGFRvIII and GM130, a Golgi endosomal marker, by confocal fluorescence microscopy (Fig. 2C).

We then studied the function of syntaxin-6 in EGFRvIII's nuclear translocation. To generate positive or negative contrast results, we produced two shRNA constructs targeting syntaxin-6. Figure 2D shows that both compounds have good knockdown efficacy against syntaxin-6.

As indicated in Figure 2E, compared with the control group, the amount of nuclear EGFRvIII decreased when syntaxin-6 was down-regulated by shRNA in U251EGFRvIII cells, showing that syntaxin-6 positively regulates the reverse transport of EGFRvIII to the nucleus. Similarly, the immunofluorescence results in Figure 2F also demonstrated that syntaxin-6 is closely related to the nuclear transport of EGFRvIII, which differed significantly between the control and shRNA groups in the subsequent statistical analysis.

These results suggest that Golgi, in particular syntaxin-6, has a positive effect on EGFRvIII's reverse transposition to the nucleus.

COPI-mediated retrograde trafficking process from Golgi to ER is associated with the nuclear translocation of EGFRvIIICOPI is a pivotal retrograde transporting complex that transfers proteins from cis Golgi to the endoplasmic reticulum. We then examined whether COPI can transfer EGFRvIII from the Golgi apparatus to ER organelles. By double immunofluorescence staining of EGFRvIII with the ER marker calregulin, the localization of EGFRvIII to the ER declined after the addition of BFA compared with serum-free starvation and the DMSO control group (Fig. 3A). BFA is an assembly inhibitor that blocks the transport function of COPI. Moreover, we compared syntaxin-6 control groups with its shRNA group by immunofluorescence imaging and observed that serum-free starvation provoked the co-localization of EGFRvIII and calregulin, while their co-location signal decreased when the expression of syntaxin-6 was down-regulated by shRNA (Fig. 3B). These data indicate that COPI participates in the transport of EGFRvIII from the Golgi apparatus to ER.To validate whether COPI is involved in the nuclear translocation of EGFRvIII, we performed immunoblotting analysis, in which BFA was administered to inhibit the activation of ARF1-guanine exchange factors, resulting in the disassembly of the COPI coat on the Golgi; DMSO was used as a control (Fig. 3C). Results showed that compared with the serum-free starvation or DMSO control group, BFA reduced the amount of EGFRvIII in the nucleus. The immunofluorescence results in Figure 3D provided further support that pretreatment with BFA in U251EGFRvIII decreased the cell number of EGFRvIII's nuclear translocation, which suggested that intact COPI is required for EGFRvIII's nuclear translocation.These results strongly indicate that COPI mediates the retrograde trafficking of EGFRvIII from the Golgi to ER, and meanwhile COPI participates in the reverse nuclear transport of EGFRvIII.

COPI-mediated retrograde trafficking process from Golgi to ER is associated with the nuclear translocation of EGFRvIII

COPI is a pivotal retrograde transporting complex that transfers proteins from cis Golgi to the endoplasmic reticulum. We then examined whether COPI can transfer EGFRvIII from the Golgi apparatus to ER organelles. By double immunofluorescence staining of EGFRvIII with the ER marker calregulin, the localization of EGFRvIII to the ER declined after the addition of BFA compared with serum-free starvation and the DMSO control group (Fig. 3A). BFA is an assembly inhibitor that blocks the transport function of COPI. Moreover, we compared syntaxin-6 control groups with its shRNA group by immunofluorescence imaging and observed that serum-free starvation provoked the co-localization of EGFRvIII and calregulin, while their co-location signal decreased when the expression of syntaxin-6 was down-regulated by shRNA (Fig. 3B). These data indicate that COPI participates in the transport of EGFRvIII from the Golgi apparatus to ER.

To validate whether COPI is involved in the nuclear translocation of EGFRvIII, we performed immunoblotting analysis, in which BFA was administered to inhibit the activation of ARF1-guanine exchange factors, resulting in the disassembly of the COPI coat on the Golgi; DMSO was used as a control (Fig. 3C). Results showed that compared with the serum-free starvation or DMSO control group, BFA reduced the amount of EGFRvIII in the nucleus. The immunofluorescence results in Figure 3D provided further support that pretreatment with BFA in U251EGFRvIII decreased the cell number of EGFRvIII's nuclear translocation, which suggested that intact COPI is required for EGFRvIII's nuclear translocation.

These results strongly indicate that COPI mediates the retrograde trafficking of EGFRvIII from the Golgi to ER, and meanwhile COPI participates in the reverse nuclear transport of EGFRvIII.

EGFRvIII's nuclear positioning can increase the phosphorylation level of STAT3As seen in the immunoblotting results in Figure 4A, EGFRvIII, especially under serum-free starvation culture condition, increased the levels of p-STAT3, which is the active form of STAT3. This result suggests that EGFRvIII's nuclear translocation may be closely related to the activation of STAT3. Subsequently, we confirmed this association in the following experiment. In U251EGFRvIII cells, serum-free starvation up-regulated p-STAT3, and meanwhile, compared with the DMSO control group, BFA inhibited the phosphorylation level of STAT3 (Fig. 4B). Existing researches showed that phosphorylated STAT3 often translocate to the nucleus to play its transcriptional regulation function 38-40. Thus, we performed nuclear and non-nuclear fractionation experiments and detected the level of nuclear p-STAT3 with western blot experiment. EGFRvIII promoted nuclear distribution of p-STAT3 in C6EGFRvIII cells versus in C6 cells, especially under serum-free starvation culture, whereas this increase was not observed in C6EGFRvIII-NLS-del cells (Fig. 4C).Next, we separately pretreated U251, U251EGFRvIII, and U251EGFRvIII-NLS-del cells in normal culture or under serum-free starvation culture and found that the latter culture condition significantly up-regulated p-STAT3 in U251EGFRvIII but not in U251 or U251EGFRvIII-NLS-del cells (Fig. 4D). We obtained similar outcomes in C6, C6EGFRvIII, and C6EGFRvIII-NLS-del cells (Fig. 4E).In summary, EGFRvIII's nuclear translocation can enhance the phosphorylation level and nuclear distribution of STAT3, which has a great help for STAT3 to play its transcriptional regulatory function in the nucleus.

EGFRvIII's nuclear positioning can increase the phosphorylation level of STAT3

As seen in the immunoblotting results in Figure 4A, EGFRvIII, especially under serum-free starvation culture condition, increased the levels of p-STAT3, which is the active form of STAT3. This result suggests that EGFRvIII's nuclear translocation may be closely related to the activation of STAT3. Subsequently, we confirmed this association in the following experiment. In U251EGFRvIII cells, serum-free starvation up-regulated p-STAT3, and meanwhile, compared with the DMSO control group, BFA inhibited the phosphorylation level of STAT3 (Fig. 4B). Existing researches showed that phosphorylated STAT3 often translocate to the nucleus to play its transcriptional regulation function 38-40. Thus, we performed nuclear and non-nuclear fractionation experiments and detected the level of nuclear p-STAT3 with western blot experiment. EGFRvIII promoted nuclear distribution of p-STAT3 in C6EGFRvIII cells versus in C6 cells, especially under serum-free starvation culture, whereas this increase was not observed in C6EGFRvIII-NLS-del cells (Fig. 4C).

Next, we separately pretreated U251, U251EGFRvIII, and U251EGFRvIII-NLS-del cells in normal culture or under serum-free starvation culture and found that the latter culture condition significantly up-regulated p-STAT3 in U251EGFRvIII but not in U251 or U251EGFRvIII-NLS-del cells (Fig. 4D). We obtained similar outcomes in C6, C6EGFRvIII, and C6EGFRvIII-NLS-del cells (Fig. 4E).

In summary, EGFRvIII's nuclear translocation can enhance the phosphorylation level and nuclear distribution of STAT3, which has a great help for STAT3 to play its transcriptional regulatory function in the nucleus.

Nuclear translocation of EGFRvIII enhances nuclear localization of PKM2In U251EGFRvIII cells, serum-free starvation improved the cell number of PKM2's nuclear localization compared with normal culture, whereas no such increase occurred in U251EGFRvIII-NLS-del cells (Fig. 5A). We then compared nuclear PKM2 levels by immunoblotting analysis in U251, U251EGFRvIII, and U251EGFRvIII-NLS-del cells. The amount of nuclear PKM2 in U251EGFRvIII cells was significantly higher than in U251 and U251EGFRvIII-NLS-del cells in the same serum-free starvation environment, which environment can increase the nuclear translocation of EGFRvIII (Fig. 5B). Figure 5C and 5D show additional immunofluorescence and western blot results in C6-associated cell lines, confirming the same changes in PKM2's nuclear translocation. Thus, we can say that the EGFRvIII with the complete nuclear transposition function rather than the EGFRvIII-NLS-del can further promote the nuclear translocation of PKM2.In U251EGFRvIII, we performed further validation with EGFRvIII's nuclear translocation inhibitor BFA. Results showed that, when EGFRVIII's nuclear translocation was inhibited by BFA, the amount of nuclear PKM2 also decreased compared with the serum-free starvation group and DMSO control group (Fig.5E).To further confirm the role of EGFRvIII's nuclear translocation in regulating PKM2 entry into the nucleus, we constructed two cell lines for subsequent experiments. We respectively overexpressed Flag-R399/400A in C6EGFRvIII and C6EGFRvIII-NLS-del cells via lentiviral infection; R399/400A is a mutant of PKM2 in which Arg399/400 is changed to alanine, causing a failure in PKM2's nuclear translocation. Figure 5F indicated that, in the same serum-free starvation environment, EGFRvIII enhanced the nuclear translocation of Flag-R399/400A more effectively than EGFRvIII-NLS-del.In conclusion, EGFRVIII's nuclear translocation can increase the localization of PKM2 in the nucleus, while nuclear PKM2 is an important prerequisite for its many regulatory functions.

Nuclear translocation of EGFRvIII enhances nuclear localization of PKM2

In U251EGFRvIII cells, serum-free starvation improved the cell number of PKM2's nuclear localization compared with normal culture, whereas no such increase occurred in U251EGFRvIII-NLS-del cells (Fig. 5A). We then compared nuclear PKM2 levels by immunoblotting analysis in U251, U251EGFRvIII, and U251EGFRvIII-NLS-del cells. The amount of nuclear PKM2 in U251EGFRvIII cells was significantly higher than in U251 and U251EGFRvIII-NLS-del cells in the same serum-free starvation environment, which environment can increase the nuclear translocation of EGFRvIII (Fig. 5B). Figure 5C and 5D show additional immunofluorescence and western blot results in C6-associated cell lines, confirming the same changes in PKM2's nuclear translocation. Thus, we can say that the EGFRvIII with the complete nuclear transposition function rather than the EGFRvIII-NLS-del can further promote the nuclear translocation of PKM2.

In U251EGFRvIII, we performed further validation with EGFRvIII's nuclear translocation inhibitor BFA. Results showed that, when EGFRVIII's nuclear translocation was inhibited by BFA, the amount of nuclear PKM2 also decreased compared with the serum-free starvation group and DMSO control group (Fig.5E).

To further confirm the role of EGFRvIII's nuclear translocation in regulating PKM2 entry into the nucleus, we constructed two cell lines for subsequent experiments. We respectively overexpressed Flag-R399/400A in C6EGFRvIII and C6EGFRvIII-NLS-del cells via lentiviral infection; R399/400A is a mutant of PKM2 in which Arg399/400 is changed to alanine, causing a failure in PKM2's nuclear translocation. Figure 5F indicated that, in the same serum-free starvation environment, EGFRvIII enhanced the nuclear translocation of Flag-R399/400A more effectively than EGFRvIII-NLS-del.

In conclusion, EGFRVIII's nuclear translocation can increase the localization of PKM2 in the nucleus, while nuclear PKM2 is an important prerequisite for its many regulatory functions.

EGFRvIII's nuclear translocation can promote the progression of glioblastoma in vivoNext, we determined the effects of EGFRvIII's nuclear translocation on the development of GBM.C6, C6EGFRvIII, and C6EGFRvIII-NLS-del cells that stably overexpress luciferase were injected separately into the brains of nude mice, and these three kinds of GBM animals were imaged through a luciferase reporter assay device on days 7, 14, and 21 after heterologous transplantation. The luminescence signals in the C6EGFRvIII group were higher than the C6 and C6EGFRvIII-NLS-del groups in the three imaging results, and meanwhile the signal intensity in C6EGFRvIII-NLS-del cells exceeded that in C6 cells (Fig. 6A); the statistical results describe the trend in signal intensity in the three groups (Fig. 6B). In addition, we sampled the above three kinds of tumorigenic brain tissues and analyzed them by immune-histochemical staining. Figure 6C showed the p-STAT3 level in three tissues, and the highest expression of p-STAT3 occurred in EGFRvIII- overexpressing tissues.Compared with C6 and C6EGFRvIII-NLS-del groups, EGFRvIII, having intact nuclear transposition ability, in C6EGFRvIII accelerated disease progression significantly. These results are consistent with previous studies that EGFRvIII, as a constitutively active protein, particularly with regard to its nuclear translocation, provokes the occurrence and progression of GBM 3, 41, 42.

EGFRvIII's nuclear translocation can promote the progression of glioblastoma in vivo

Next, we determined the effects of EGFRvIII's nuclear translocation on the development of GBM.

C6, C6EGFRvIII, and C6EGFRvIII-NLS-del cells that stably overexpress luciferase were injected separately into the brains of nude mice, and these three kinds of GBM animals were imaged through a luciferase reporter assay device on days 7, 14, and 21 after heterologous transplantation. The luminescence signals in the C6EGFRvIII group were higher than the C6 and C6EGFRvIII-NLS-del groups in the three imaging results, and meanwhile the signal intensity in C6EGFRvIII-NLS-del cells exceeded that in C6 cells (Fig. 6A); the statistical results describe the trend in signal intensity in the three groups (Fig. 6B). In addition, we sampled the above three kinds of tumorigenic brain tissues and analyzed them by immune-histochemical staining. Figure 6C showed the p-STAT3 level in three tissues, and the highest expression of p-STAT3 occurred in EGFRvIII- overexpressing tissues.

Compared with C6 and C6EGFRvIII-NLS-del groups, EGFRvIII, having intact nuclear transposition ability, in C6EGFRvIII accelerated disease progression significantly. These results are consistent with previous studies that EGFRvIII, as a constitutively active protein, particularly with regard to its nuclear translocation, provokes the occurrence and progression of GBM 3, 41, 42.

DiscussionEGFRvIII, the most common mutant of EGFR, coexists with EGFR in glioma cells and does not exist alone. In the absence of ligands, EGFRvIII also can activate downstream cascade signals by forming a heterogeneous or homologous polymer, promoting the proliferation, migration, and infiltration of glioma cells. As nuclear localization proteins, EGFR/ EGFRvIII can activate STAT3 and STAT5 through its tyrosine kinase function during nuclear translocation and can act as transcription factors in the nucleus, interacting with RHA (RNA helicase A), E2F, and STAT3/5. These pathways are often associated with inflammatory reactions, tumorigenesis or cancer progression. Although the nuclear translocation pathway of EGFR has been reported extensively, there is no detailed study on the mechanism of EGFRvIII's nuclear translocation.Our research revealed complicated reverse nuclear trafficking of EGFRvIII, which involves the Golgi and ER and the participation of syntaxin-6 and COPI. Based on our results and previous studies, we generated a model of EGFRvIII's reverse transposition from endocytosis to nuclear localization (Fig.7).This process includes several key nodes: (1) as a SNARE protein and located primarily on the trans Golgi, syntaxin-6 is responsible for the identification, presentation, and transportation of EGFRvIII toward the cytoplasm; (2) COPI can identify EGFRvIII, shuttle it back to the cis Golgi, and transfer it from the cis Golgi to ER; (3) inhibition of sytaxin6 or COPI disassembly weakens the localization of EGFRvIII in the nucleus, which demonstrates their importance in EGFRvIII's reverse nuclear trafficking; and (4) the nuclear translocation of EGFRvIII facilitates the phosphorylation of STAT3, which often has a role in cancer progression, and when the nuclear translocation of EGFRvIII is inhibited, the levels of p-STAT3 decrease. In conclusion, our study reveals a mechanism by which EGFRvIII undergoes reverse nuclear translocation and syntaxin-6 and COPI implicate in this process.Overexpression of PKM2 and its nuclear translocation are important markers of Warburg effect and tumorigenesis. Although the promotion of PKM2's nuclear translocation by EGFR has been elucidated 34, 35, 43, we found the relationship between EGFRvIII's nuclear transposition and PKM2's nuclear localization, that is, the former having a significant promoting effect on the latter.Our nude mouse experiments demonstrated that EGFRvIII's nuclear translocation has a prominent effect on malignant progression of glioma. Numerous literatures have shown that the level of p-STAT3 or PKM2's nuclear transposition has a positive effect on tumors' growth. This paper shows that EGFRvIII's nuclear transposition is the key upstream mechanism for the change of the above two. Therefore, EGFRvIII and its nuclear transposition may be an important target for the treatment of glioma.In conclusion, EGFRvIII's nuclear transposition is a key point in the progression of glioma, and the revelation of its nuclear translocation process may help us to find new therapeutic strategies.

EGFRvIII, the most common mutant of EGFR, coexists with EGFR in glioma cells and does not exist alone. In the absence of ligands, EGFRvIII also can activate downstream cascade signals by forming a heterogeneous or homologous polymer, promoting the proliferation, migration, and infiltration of glioma cells. As nuclear localization proteins, EGFR/ EGFRvIII can activate STAT3 and STAT5 through its tyrosine kinase function during nuclear translocation and can act as transcription factors in the nucleus, interacting with RHA (RNA helicase A), E2F, and STAT3/5. These pathways are often associated with inflammatory reactions, tumorigenesis or cancer progression. Although the nuclear translocation pathway of EGFR has been reported extensively, there is no detailed study on the mechanism of EGFRvIII's nuclear translocation.

Our research revealed complicated reverse nuclear trafficking of EGFRvIII, which involves the Golgi and ER and the participation of syntaxin-6 and COPI. Based on our results and previous studies, we generated a model of EGFRvIII's reverse transposition from endocytosis to nuclear localization (Fig.7).

This process includes several key nodes: (1) as a SNARE protein and located primarily on the trans Golgi, syntaxin-6 is responsible for the identification, presentation, and transportation of EGFRvIII toward the cytoplasm; (2) COPI can identify EGFRvIII, shuttle it back to the cis Golgi, and transfer it from the cis Golgi to ER; (3) inhibition of sytaxin6 or COPI disassembly weakens the localization of EGFRvIII in the nucleus, which demonstrates their importance in EGFRvIII's reverse nuclear trafficking; and (4) the nuclear translocation of EGFRvIII facilitates the phosphorylation of STAT3, which often has a role in cancer progression, and when the nuclear translocation of EGFRvIII is inhibited, the levels of p-STAT3 decrease. In conclusion, our study reveals a mechanism by which EGFRvIII undergoes reverse nuclear translocation and syntaxin-6 and COPI implicate in this process.

Overexpression of PKM2 and its nuclear translocation are important markers of Warburg effect and tumorigenesis. Although the promotion of PKM2's nuclear translocation by EGFR has been elucidated 34, 35, 43, we found the relationship between EGFRvIII's nuclear transposition and PKM2's nuclear localization, that is, the former having a significant promoting effect on the latter.

Our nude mouse experiments demonstrated that EGFRvIII's nuclear translocation has a prominent effect on malignant progression of glioma. Numerous literatures have shown that the level of p-STAT3 or PKM2's nuclear transposition has a positive effect on tumors' growth. This paper shows that EGFRvIII's nuclear transposition is the key upstream mechanism for the change of the above two. Therefore, EGFRvIII and its nuclear transposition may be an important target for the treatment of glioma.

In conclusion, EGFRvIII's nuclear transposition is a key point in the progression of glioma, and the revelation of its nuclear translocation process may help us to find new therapeutic strategies.

Supplementary MaterialSupplementary figures.Click here for additional data file.

Supplementary Material

Supplementary figures.

Click here for additional data file.
