---
pmid: '25403480'
title: 'Glucose tolerance is associated with differential expression of microRNAs
  in skeletal muscle: results from studies of twins with and without type 2 diabetes.'
authors:
- Bork-Jensen J
- Scheele C
- Christophersen DV
- Nilsson E
- Friedrichsen M
- Fernandez-Twinn DS
- Grunnet LG
- Litman T
- Holmstrøm K
- Vind B
- Højlund K
- Beck-Nielsen H
- Wojtaszewski J
- Ozanne SE
- Pedersen BK
- Poulsen P
- Vaag A
journal: Diabetologia
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4287682
doi: 10.1007/s00125-014-3434-2
---

# Glucose tolerance is associated with differential expression of microRNAs in skeletal muscle: results from studies of twins with and without type 2 diabetes.
**Authors:** Bork-Jensen J, Scheele C, Christophersen DV, Nilsson E, Friedrichsen M, Fernandez-Twinn DS, Grunnet LG, Litman T, Holmstrøm K, Vind B, Højlund K, Beck-Nielsen H, Wojtaszewski J, Ozanne SE, Pedersen BK, Poulsen P, Vaag A
**Journal:** Diabetologia (2015)
**DOI:** [10.1007/s00125-014-3434-2](https://doi.org/10.1007/s00125-014-3434-2)
**PMC:** [PMC4287682](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4287682/)

## Abstract

1. Diabetologia. 2015 Feb;58(2):363-73. doi: 10.1007/s00125-014-3434-2. Epub 2014
 Nov 19.

Glucose tolerance is associated with differential expression of microRNAs in 
skeletal muscle: results from studies of twins with and without type 2 diabetes.

Bork-Jensen J(1), Scheele C, Christophersen DV, Nilsson E, Friedrichsen M, 
Fernandez-Twinn DS, Grunnet LG, Litman T, Holmstrøm K, Vind B, Højlund K, 
Beck-Nielsen H, Wojtaszewski J, Ozanne SE, Pedersen BK, Poulsen P, Vaag A.

Author information:
(1)Steno Diabetes Center, Gentofte, Denmark, jette.bork.jensen@sund.ku.dk.

AIMS/HYPOTHESIS: We aimed to identify microRNAs (miRNAs) associated with type 2 
diabetes and risk of developing the disease in skeletal muscle biopsies from 
phenotypically well-characterised twins.
METHODS: We measured muscle miRNA levels in monozygotic (MZ) twins discordant 
for type 2 diabetes using arrays. Further investigations of selected miRNAs 
included target prediction, pathway analysis, silencing in cells and association 
analyses in a separate cohort of 164 non-diabetic MZ and dizygotic twins. The 
effects of elevated glucose and insulin levels on miRNA expression were 
examined, and the effect of low birthweight (LBW) was studied in rats.
RESULTS: We identified 20 miRNAs that were downregulated in MZ twins with 
diabetes compared with their non-diabetic co-twins. Differences for members of 
the miR-15 family (miR-15b and miR-16) were the most statistically significant, 
and these miRNAs were predicted to influence insulin signalling. Indeed, miR-15b 
and miR-16 levels were associated with levels of key insulin signalling 
proteins, miR-15b was associated with the insulin receptor in non-diabetic twins 
and knockdown of miR-15b/miR-16 in myocytes changed the levels of insulin 
signalling proteins. LBW in twins and undernutrition during pregnancy in rats 
were, in contrast to overt type 2 diabetes, associated with increased expression 
of miR-15b and/or miR-16. Elevated glucose and insulin suppressed miR-16 
expression in vitro.
CONCLUSIONS: Type 2 diabetes is associated with non-genetic downregulation of 
several miRNAs in skeletal muscle including miR-15b and miR-16, potentially 
targeting insulin signalling. The paradoxical findings in twins with overt 
diabetes and twins at increased risk of the disease underscore the complexity of 
the regulation of muscle insulin signalling in glucose homeostasis.

DOI: 10.1007/s00125-014-3434-2
PMCID: PMC4287682
PMID: 25403480 [Indexed for MEDLINE]

## Full Text

IntroductionType 2 diabetes is caused by multiple organ defects, with inadequate insulin secretion and insulin resistance being the most important. Skeletal muscle insulin resistance is of specific interest as muscle is the major site for glucose disposal [1]. The insulin signalling cascade is initiated by the binding of insulin to its receptor, which activates the key cellular processes of glucose metabolism, including glucose transport across the membrane and storage of glucose as muscle glycogen. Although the genetic influence on muscle insulin resistance cannot be disputed [2], non-genetic factors, including obesity, physical inactivity and low birthweight (LBW) [3–6], are important in controlling muscle insulin resistance [7–9].Emerging evidence points towards microRNAs (miRNAs) as major players in the regulation of protein levels. These miRNAs are short single-stranded RNA molecules that regulate their target mRNAs mainly by translational repression and/or degradation [10]. We recently demonstrated that miR-483-3p, involved in adipose tissue development and lipid storage, is upregulated in adipose tissue from individuals born with LBW as well as in young adult rats exposed to a sub-optimal intrauterine environment [11]. Another recent study indicated that several miRNAs were differentially expressed in skeletal muscle from patients with type 2 diabetes [12], and Zhu et al recently demonstrated that overexpression of the miRNA let-7 results in insulin resistance and impaired glucose tolerance in rats, probably through repression of proteins in the insulin–phosphoinositide 3-kinase–mechanistic target of rapamycin (insulin–PI3K–mTOR) pathway [13]. Thus, miRNAs could therefore be important regulators of insulin signalling.We have previously provided extensive proof-of-concept evidence for the powerful approach of using genetically identical monozygotic (MZ) twins discordant for type 2 diabetes to identify non-genetic metabolic and molecular defects in type 2 diabetes [4, 14]. When unravelling the complex underlying molecular mechanisms involved in the development of type 2 diabetes, it is important to distinguish between the primary mechanisms relevant to the development of type 2 diabetes and the secondary molecular alterations occurring as a consequence of the metabolic derangements including ‘glucotoxicity’ in hyperglycaemic patients with overt type 2 diabetes.In the present study, we first aimed to identify altered miRNA levels of non-genetic origin in MZ twins discordant for type 2 diabetes. The top candidates identified were subsequently investigated in a larger group of phenotypically well-characterised twins without diabetes to uncover their role in skeletal muscle insulin action. Our comprehensive approach also included computational target prediction and in vitro target validation using antagomirs, while the effects of elevated glucose and insulin levels on miRNA expression level were studied in vitro. Finally, we studied miRNA expression in a rat model of LBW.

Introduction

Type 2 diabetes is caused by multiple organ defects, with inadequate insulin secretion and insulin resistance being the most important. Skeletal muscle insulin resistance is of specific interest as muscle is the major site for glucose disposal [1]. The insulin signalling cascade is initiated by the binding of insulin to its receptor, which activates the key cellular processes of glucose metabolism, including glucose transport across the membrane and storage of glucose as muscle glycogen. Although the genetic influence on muscle insulin resistance cannot be disputed [2], non-genetic factors, including obesity, physical inactivity and low birthweight (LBW) [3–6], are important in controlling muscle insulin resistance [7–9].

Emerging evidence points towards microRNAs (miRNAs) as major players in the regulation of protein levels. These miRNAs are short single-stranded RNA molecules that regulate their target mRNAs mainly by translational repression and/or degradation [10]. We recently demonstrated that miR-483-3p, involved in adipose tissue development and lipid storage, is upregulated in adipose tissue from individuals born with LBW as well as in young adult rats exposed to a sub-optimal intrauterine environment [11]. Another recent study indicated that several miRNAs were differentially expressed in skeletal muscle from patients with type 2 diabetes [12], and Zhu et al recently demonstrated that overexpression of the miRNA let-7 results in insulin resistance and impaired glucose tolerance in rats, probably through repression of proteins in the insulin–phosphoinositide 3-kinase–mechanistic target of rapamycin (insulin–PI3K–mTOR) pathway [13]. Thus, miRNAs could therefore be important regulators of insulin signalling.

We have previously provided extensive proof-of-concept evidence for the powerful approach of using genetically identical monozygotic (MZ) twins discordant for type 2 diabetes to identify non-genetic metabolic and molecular defects in type 2 diabetes [4, 14]. When unravelling the complex underlying molecular mechanisms involved in the development of type 2 diabetes, it is important to distinguish between the primary mechanisms relevant to the development of type 2 diabetes and the secondary molecular alterations occurring as a consequence of the metabolic derangements including ‘glucotoxicity’ in hyperglycaemic patients with overt type 2 diabetes.

In the present study, we first aimed to identify altered miRNA levels of non-genetic origin in MZ twins discordant for type 2 diabetes. The top candidates identified were subsequently investigated in a larger group of phenotypically well-characterised twins without diabetes to uncover their role in skeletal muscle insulin action. Our comprehensive approach also included computational target prediction and in vitro target validation using antagomirs, while the effects of elevated glucose and insulin levels on miRNA expression level were studied in vitro. Finally, we studied miRNA expression in a rat model of LBW.

MethodsParticipantsThe MZ discordant twin population comprised 22 white MZ twins (11 pairs), aged 53–78 years, discordant for type 2 diabetes who were recruited through the Danish Twin Registry, University of Southern Denmark. Five pairs were from a previous cohort [15–17] where discordance for type 2 diabetes was recognised based on an OGTT. The additional six twin pairs were recruited based on their discordance for type 2 diabetes. The recruitment and clinical examinations of all 11 pairs has been described previously [18]. The basic clinical and anthropometric measures are presented in Table 1. Based on a 2 h OGTT, nine of the non-diabetic co-twins had impaired glucose tolerance (IGT) and two had normal glucose tolerance (NGT).Table 1Clinical and metabolic characteristics of the study populationsCharacteristicDiscordant monozygotic twin studyNon-diabetic twin studyDiabetic twinsNon-diabetic co-twins
p value
n (female/male)11 (6/5)11 (6/5)164 (76/88)
n (NGT/IGT/T2D)11 (0/0/11)11 (2/9/0)—Zygosity (MZ/DZ)——90/74Age (years)68 ± 868 ± 842.9 ± 17.1Birthweight (g)——2,671 ± 468BMI (kg/m2)32.5 ± 6.730.4 ± 6.50.0425.3 ± 4Fasting glucose (mmol/l)10.6 ± 2.16.3 ± 0.5<0.0015.4 ± 0.7Glu120 (mmol/l)19.5 ± 3.59.0 ± 1.7<0.00016.5 ± 1.7GIR (mg m−2 min−1)a
153.8 ± 63.2288.1 ± 175.40.05—HbA1c (%)b
7.7 ± 1.55.9 ± 0.5<0.003—HbA1c (mmol/mol)61 ± 16.441 ± 5.5—Total fat (%)——24.7 ± 9
R
d (mg kgFFM
−1 min−1)——10.9 ± 3.4
\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \overset{\cdot }{V}{\mathrm{O}}_{2 \max } $$\end{document}V⋅O2max (ml kg−2 min−1)——33.9 ± 10.1Data are represented as mean ± SD
a
n = 9 in each group; b
n = 10 in each groupFFM, fat-free mass; Glu120, 120 min glucose values from an OGTTThe non-diabetic twin population was a subgroup of 164 MZ and same-sex dizygotic (DZ) twins (the number of individuals with available tissue material) from a previously described population [7] of 196 young and old twins (aged 25–32 and 58–66 years). Participant characteristics are displayed in Table 1.All study participants gave informed consent and the study was carried out in accordance with the Declaration of Helsinki as revised in 2008.Clinical examinationIn the MZ discordant twin population, insulin sensitivity was measured by a 2 h euglycaemic–hyperinsulinaemic clamp (40 mU m−2 min−1) and expressed as the mean glucose infusion rate (GIR) during the final 30 min steady-state period of the clamp. In the non-diabetic twin population, 3-3H-glucose was infused for the calculation of the peripheral glucose disposal rate (Rd), and body composition was determined by dual-energy x-ray absorptiometry (DXA) scanning. A 2 h 75 g OGTT was performed in both populations and muscle biopsies (vastus lateralis) were excised under local anaesthesia (lidocaine), during basal fasting conditions using a Bergström needle with suction applied. The biopsies were frozen immediately in liquid nitrogen and stored at −80°C for later analysis. In the non-diabetic twin population, physical fitness was estimated as maximal oxygen consumption rate (\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \overset{\cdot }{V}{\mathrm{O}}_{2 \max } $$\end{document}V⋅O2max) calculated from the maximal load on ergometer bicycle [19]. More details can be found in previous publications of this cohort [7, 15–18].LNA arraysFor each of the discordant twins (n = 22), 1 μg total RNA was isolated from a muscle biopsy (described in electronic supplementary material [ESM] methods) and each sample was run on a miRCURY locked nucleic acid (LNA) array (Exiqon, Vedbaek, Denmark) representing version 9.2 of the Sanger miRBase as described in the ESM Methods. Annotation was updated to miRBase version 14, and the data were normalised using normexp [20] and quantile normalisation [21] in R (version 2.9.1). The dataset has been deposited in the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus [22], accession number GSE38835.Familiar relationships and target analysisThe familiar relationships between miRNAs were based on miRBase version 17. TargetScan 4.2 [23–25] was used to predict mRNA targets. The lists of targets were imported into Ingenuity pathway analysis (IPA) version 8.0 (IPA, QIAGEN, Redwood City, www.qiagen.com/ingenuity) for canonical pathway analysis or the Database for Annotation, Visualization and Integrated Discovery (DAVID) 6.2. for a Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis [26]. The IPA results were corrected for multiple testing using the Benjamini–Hochberg method.Statistical methodsDifferential expression levels in the discordant twins were assessed using a paired design matrix with linear models for microarray data (Limma) [27, 28] in R. Spot quality weights [29] were used and p values were adjusted with Benjamini–Hochberg’s false-discovery rate (FDR) [30]. Limma was used to analyse 170 miRNAs in a published online dataset [12] for differences between those with NGT, IGT and type 2 diabetes. Pearson intra-pair correlation coefficients were calculated using 2n, as previously recommended [31]. The SAS ‘proc mixed’ procedure (SAS Institute, Cary, NC, USA) was used to perform multiple regression analysis with adjustments for zygosity and twin-pair status. If not otherwise stated, additional adjustment for age, sex, \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \overset{\cdot }{V}{\mathrm{O}}_{2 \max } $$\end{document}V⋅O2max, birthweight and total body fat percentage was performed. GraphPad Prism (GraphPad Software, La Jolla, CA, USA) was used to perform ANOVA.Human myocytesAntagomiR experimentProliferating myoblasts from five healthy donors (three men and two women) aged 59 ± 7 years were transfected using Lipofectamine 2000 (Invitrogen, Carlsbad, CA, USA) with 10 nmol/l antagomiRs (Exiqon) targeting miR-15b/miR-16 or a non-targeting antagomiR (Exiqon) for 48 h. Simultaneous knockdown of miR-15b and miR-16 was performed to hinder a possible rescue effect that could occur when inhibiting them separately. Cells were harvested after 5–10 min incubation with 0, 1, 10 or 100 nmol/l insulin (Actrapid, Novo Nordisk, Bagsværd, Denmark) in Trizol (Invitrogen) for RNA isolation or in lysis buffer (Cell Signaling Technology, Beverly, MA, USA) for protein isolation. See the ESM Methods for further details.Short-term insulin experimentSee the ESM Methods for the description of this experiment.Rat protocolAdult females were fed ad libitum with either a control diet (20% protein) or an isoenergetic low-protein (LP) (8% protein) diet (Arie Blok, Woerden, the Netherlands) during gestation and lactation as described previously [32]. At 3 months of age, 11 control males and 11 males from LP litters, each of the 22 rats being from an independent litter, were killed and vastus lateralis muscle biopsies were excised and stored for later use. All animal procedures were approved by the local animal ethical review committee and were carried out under compliance with the UK Animal (Scientific Procedures) Act 1986. The investigation conforms to the Guide for the Care and Use of Laboratory Animals published by the US National Institutes of Health (NIH Publication no. 85–23, revised 1996). See the ESM Methods for further details.L6 cellsL6 myoblasts were obtained from the American Type Culture Collection (CRL-1458, ATCC) and cultured in DMEM (Sigma-Aldrich, Poole, UK) with 10% fetal bovine serum, penicillin (100 units/ml), streptomycin (100 mg/ml) and glutamine (0.2 mmol/l), as previously described [33]. Semi-confluent cultures were treated with four different combinations of insulin (Actrapid; Novo Nordisk Insulin, Bagsværd, Denmark) and glucose levels, 0 nmol/l insulin and 5.6 mmol/l glucose, 100 nmol/l insulin and 5.6 mmol/l glucose, 0 nmol/l insulin and 25 mmol/l glucose or 100 nmol/l insulin and 25 mmol/l glucose, every 24 h for a total of 96 h until harvesting.Gene expressionRNA isolationThis is described in the RNA isolation section of the ESM Methods. ABI-PRISM 7900 Sequence Detection system ([ABI] Applied Biosystems, Foster City, CA, USA) was used for both miRNA and mRNA detection. Expression was quantified using TaqMan probes for miRNA and primers from ABI or the Roche Universal probe library for mRNA. Details are described in the ESM Methods.Protein levelsMuscle cell culturesLevels of phosphoinositide 3-kinase R1 regulatory subunit 1 α (PIK3R1), Akt and p-Akt were measured using western blotting (Invitrogen). Insulin receptor (INSR) levels were measured using Meso Scale Discovery analysis (Meso Scale Discovery, Rockville, MD, USA).Muscle biopsiesThe protein levels of INSR, IRS-1 and PIK3R1 were measured using western blotting as described in [2, 34]. Details of these methods are described in the ESM Methods.

ParticipantsThe MZ discordant twin population comprised 22 white MZ twins (11 pairs), aged 53–78 years, discordant for type 2 diabetes who were recruited through the Danish Twin Registry, University of Southern Denmark. Five pairs were from a previous cohort [15–17] where discordance for type 2 diabetes was recognised based on an OGTT. The additional six twin pairs were recruited based on their discordance for type 2 diabetes. The recruitment and clinical examinations of all 11 pairs has been described previously [18]. The basic clinical and anthropometric measures are presented in Table 1. Based on a 2 h OGTT, nine of the non-diabetic co-twins had impaired glucose tolerance (IGT) and two had normal glucose tolerance (NGT).Table 1Clinical and metabolic characteristics of the study populationsCharacteristicDiscordant monozygotic twin studyNon-diabetic twin studyDiabetic twinsNon-diabetic co-twins
p value
n (female/male)11 (6/5)11 (6/5)164 (76/88)
n (NGT/IGT/T2D)11 (0/0/11)11 (2/9/0)—Zygosity (MZ/DZ)——90/74Age (years)68 ± 868 ± 842.9 ± 17.1Birthweight (g)——2,671 ± 468BMI (kg/m2)32.5 ± 6.730.4 ± 6.50.0425.3 ± 4Fasting glucose (mmol/l)10.6 ± 2.16.3 ± 0.5<0.0015.4 ± 0.7Glu120 (mmol/l)19.5 ± 3.59.0 ± 1.7<0.00016.5 ± 1.7GIR (mg m−2 min−1)a
153.8 ± 63.2288.1 ± 175.40.05—HbA1c (%)b
7.7 ± 1.55.9 ± 0.5<0.003—HbA1c (mmol/mol)61 ± 16.441 ± 5.5—Total fat (%)——24.7 ± 9
R
d (mg kgFFM
−1 min−1)——10.9 ± 3.4
\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \overset{\cdot }{V}{\mathrm{O}}_{2 \max } $$\end{document}V⋅O2max (ml kg−2 min−1)——33.9 ± 10.1Data are represented as mean ± SD
a
n = 9 in each group; b
n = 10 in each groupFFM, fat-free mass; Glu120, 120 min glucose values from an OGTTThe non-diabetic twin population was a subgroup of 164 MZ and same-sex dizygotic (DZ) twins (the number of individuals with available tissue material) from a previously described population [7] of 196 young and old twins (aged 25–32 and 58–66 years). Participant characteristics are displayed in Table 1.All study participants gave informed consent and the study was carried out in accordance with the Declaration of Helsinki as revised in 2008.

Participants

The MZ discordant twin population comprised 22 white MZ twins (11 pairs), aged 53–78 years, discordant for type 2 diabetes who were recruited through the Danish Twin Registry, University of Southern Denmark. Five pairs were from a previous cohort [15–17] where discordance for type 2 diabetes was recognised based on an OGTT. The additional six twin pairs were recruited based on their discordance for type 2 diabetes. The recruitment and clinical examinations of all 11 pairs has been described previously [18]. The basic clinical and anthropometric measures are presented in Table 1. Based on a 2 h OGTT, nine of the non-diabetic co-twins had impaired glucose tolerance (IGT) and two had normal glucose tolerance (NGT).Table 1Clinical and metabolic characteristics of the study populationsCharacteristicDiscordant monozygotic twin studyNon-diabetic twin studyDiabetic twinsNon-diabetic co-twins
p value
n (female/male)11 (6/5)11 (6/5)164 (76/88)
n (NGT/IGT/T2D)11 (0/0/11)11 (2/9/0)—Zygosity (MZ/DZ)——90/74Age (years)68 ± 868 ± 842.9 ± 17.1Birthweight (g)——2,671 ± 468BMI (kg/m2)32.5 ± 6.730.4 ± 6.50.0425.3 ± 4Fasting glucose (mmol/l)10.6 ± 2.16.3 ± 0.5<0.0015.4 ± 0.7Glu120 (mmol/l)19.5 ± 3.59.0 ± 1.7<0.00016.5 ± 1.7GIR (mg m−2 min−1)a
153.8 ± 63.2288.1 ± 175.40.05—HbA1c (%)b
7.7 ± 1.55.9 ± 0.5<0.003—HbA1c (mmol/mol)61 ± 16.441 ± 5.5—Total fat (%)——24.7 ± 9
R
d (mg kgFFM
−1 min−1)——10.9 ± 3.4
\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \overset{\cdot }{V}{\mathrm{O}}_{2 \max } $$\end{document}V⋅O2max (ml kg−2 min−1)——33.9 ± 10.1Data are represented as mean ± SD
a
n = 9 in each group; b
n = 10 in each groupFFM, fat-free mass; Glu120, 120 min glucose values from an OGTT

Clinical and metabolic characteristics of the study populations

Data are represented as mean ± SD

a
n = 9 in each group; b
n = 10 in each group

FFM, fat-free mass; Glu120, 120 min glucose values from an OGTT

The non-diabetic twin population was a subgroup of 164 MZ and same-sex dizygotic (DZ) twins (the number of individuals with available tissue material) from a previously described population [7] of 196 young and old twins (aged 25–32 and 58–66 years). Participant characteristics are displayed in Table 1.

All study participants gave informed consent and the study was carried out in accordance with the Declaration of Helsinki as revised in 2008.

Clinical examinationIn the MZ discordant twin population, insulin sensitivity was measured by a 2 h euglycaemic–hyperinsulinaemic clamp (40 mU m−2 min−1) and expressed as the mean glucose infusion rate (GIR) during the final 30 min steady-state period of the clamp. In the non-diabetic twin population, 3-3H-glucose was infused for the calculation of the peripheral glucose disposal rate (Rd), and body composition was determined by dual-energy x-ray absorptiometry (DXA) scanning. A 2 h 75 g OGTT was performed in both populations and muscle biopsies (vastus lateralis) were excised under local anaesthesia (lidocaine), during basal fasting conditions using a Bergström needle with suction applied. The biopsies were frozen immediately in liquid nitrogen and stored at −80°C for later analysis. In the non-diabetic twin population, physical fitness was estimated as maximal oxygen consumption rate (\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \overset{\cdot }{V}{\mathrm{O}}_{2 \max } $$\end{document}V⋅O2max) calculated from the maximal load on ergometer bicycle [19]. More details can be found in previous publications of this cohort [7, 15–18].

Clinical examination

In the MZ discordant twin population, insulin sensitivity was measured by a 2 h euglycaemic–hyperinsulinaemic clamp (40 mU m−2 min−1) and expressed as the mean glucose infusion rate (GIR) during the final 30 min steady-state period of the clamp. In the non-diabetic twin population, 3-3H-glucose was infused for the calculation of the peripheral glucose disposal rate (Rd), and body composition was determined by dual-energy x-ray absorptiometry (DXA) scanning. A 2 h 75 g OGTT was performed in both populations and muscle biopsies (vastus lateralis) were excised under local anaesthesia (lidocaine), during basal fasting conditions using a Bergström needle with suction applied. The biopsies were frozen immediately in liquid nitrogen and stored at −80°C for later analysis. In the non-diabetic twin population, physical fitness was estimated as maximal oxygen consumption rate (\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \overset{\cdot }{V}{\mathrm{O}}_{2 \max } $$\end{document}V⋅O2max) calculated from the maximal load on ergometer bicycle [19]. More details can be found in previous publications of this cohort [7, 15–18].

LNA arraysFor each of the discordant twins (n = 22), 1 μg total RNA was isolated from a muscle biopsy (described in electronic supplementary material [ESM] methods) and each sample was run on a miRCURY locked nucleic acid (LNA) array (Exiqon, Vedbaek, Denmark) representing version 9.2 of the Sanger miRBase as described in the ESM Methods. Annotation was updated to miRBase version 14, and the data were normalised using normexp [20] and quantile normalisation [21] in R (version 2.9.1). The dataset has been deposited in the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus [22], accession number GSE38835.

For each of the discordant twins (n = 22), 1 μg total RNA was isolated from a muscle biopsy (described in electronic supplementary material [ESM] methods) and each sample was run on a miRCURY locked nucleic acid (LNA) array (Exiqon, Vedbaek, Denmark) representing version 9.2 of the Sanger miRBase as described in the ESM Methods. Annotation was updated to miRBase version 14, and the data were normalised using normexp [20] and quantile normalisation [21] in R (version 2.9.1). The dataset has been deposited in the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus [22], accession number GSE38835.

Familiar relationships and target analysisThe familiar relationships between miRNAs were based on miRBase version 17. TargetScan 4.2 [23–25] was used to predict mRNA targets. The lists of targets were imported into Ingenuity pathway analysis (IPA) version 8.0 (IPA, QIAGEN, Redwood City, www.qiagen.com/ingenuity) for canonical pathway analysis or the Database for Annotation, Visualization and Integrated Discovery (DAVID) 6.2. for a Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis [26]. The IPA results were corrected for multiple testing using the Benjamini–Hochberg method.

Familiar relationships and target analysis

The familiar relationships between miRNAs were based on miRBase version 17. TargetScan 4.2 [23–25] was used to predict mRNA targets. The lists of targets were imported into Ingenuity pathway analysis (IPA) version 8.0 (IPA, QIAGEN, Redwood City, www.qiagen.com/ingenuity) for canonical pathway analysis or the Database for Annotation, Visualization and Integrated Discovery (DAVID) 6.2. for a Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis [26]. The IPA results were corrected for multiple testing using the Benjamini–Hochberg method.

Statistical methodsDifferential expression levels in the discordant twins were assessed using a paired design matrix with linear models for microarray data (Limma) [27, 28] in R. Spot quality weights [29] were used and p values were adjusted with Benjamini–Hochberg’s false-discovery rate (FDR) [30]. Limma was used to analyse 170 miRNAs in a published online dataset [12] for differences between those with NGT, IGT and type 2 diabetes. Pearson intra-pair correlation coefficients were calculated using 2n, as previously recommended [31]. The SAS ‘proc mixed’ procedure (SAS Institute, Cary, NC, USA) was used to perform multiple regression analysis with adjustments for zygosity and twin-pair status. If not otherwise stated, additional adjustment for age, sex, \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \overset{\cdot }{V}{\mathrm{O}}_{2 \max } $$\end{document}V⋅O2max, birthweight and total body fat percentage was performed. GraphPad Prism (GraphPad Software, La Jolla, CA, USA) was used to perform ANOVA.

Statistical methods

Differential expression levels in the discordant twins were assessed using a paired design matrix with linear models for microarray data (Limma) [27, 28] in R. Spot quality weights [29] were used and p values were adjusted with Benjamini–Hochberg’s false-discovery rate (FDR) [30]. Limma was used to analyse 170 miRNAs in a published online dataset [12] for differences between those with NGT, IGT and type 2 diabetes. Pearson intra-pair correlation coefficients were calculated using 2n, as previously recommended [31]. The SAS ‘proc mixed’ procedure (SAS Institute, Cary, NC, USA) was used to perform multiple regression analysis with adjustments for zygosity and twin-pair status. If not otherwise stated, additional adjustment for age, sex, \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \overset{\cdot }{V}{\mathrm{O}}_{2 \max } $$\end{document}V⋅O2max, birthweight and total body fat percentage was performed. GraphPad Prism (GraphPad Software, La Jolla, CA, USA) was used to perform ANOVA.

Human myocytesAntagomiR experimentProliferating myoblasts from five healthy donors (three men and two women) aged 59 ± 7 years were transfected using Lipofectamine 2000 (Invitrogen, Carlsbad, CA, USA) with 10 nmol/l antagomiRs (Exiqon) targeting miR-15b/miR-16 or a non-targeting antagomiR (Exiqon) for 48 h. Simultaneous knockdown of miR-15b and miR-16 was performed to hinder a possible rescue effect that could occur when inhibiting them separately. Cells were harvested after 5–10 min incubation with 0, 1, 10 or 100 nmol/l insulin (Actrapid, Novo Nordisk, Bagsværd, Denmark) in Trizol (Invitrogen) for RNA isolation or in lysis buffer (Cell Signaling Technology, Beverly, MA, USA) for protein isolation. See the ESM Methods for further details.Short-term insulin experimentSee the ESM Methods for the description of this experiment.

Human myocytes

AntagomiR experimentProliferating myoblasts from five healthy donors (three men and two women) aged 59 ± 7 years were transfected using Lipofectamine 2000 (Invitrogen, Carlsbad, CA, USA) with 10 nmol/l antagomiRs (Exiqon) targeting miR-15b/miR-16 or a non-targeting antagomiR (Exiqon) for 48 h. Simultaneous knockdown of miR-15b and miR-16 was performed to hinder a possible rescue effect that could occur when inhibiting them separately. Cells were harvested after 5–10 min incubation with 0, 1, 10 or 100 nmol/l insulin (Actrapid, Novo Nordisk, Bagsværd, Denmark) in Trizol (Invitrogen) for RNA isolation or in lysis buffer (Cell Signaling Technology, Beverly, MA, USA) for protein isolation. See the ESM Methods for further details.

AntagomiR experiment

Proliferating myoblasts from five healthy donors (three men and two women) aged 59 ± 7 years were transfected using Lipofectamine 2000 (Invitrogen, Carlsbad, CA, USA) with 10 nmol/l antagomiRs (Exiqon) targeting miR-15b/miR-16 or a non-targeting antagomiR (Exiqon) for 48 h. Simultaneous knockdown of miR-15b and miR-16 was performed to hinder a possible rescue effect that could occur when inhibiting them separately. Cells were harvested after 5–10 min incubation with 0, 1, 10 or 100 nmol/l insulin (Actrapid, Novo Nordisk, Bagsværd, Denmark) in Trizol (Invitrogen) for RNA isolation or in lysis buffer (Cell Signaling Technology, Beverly, MA, USA) for protein isolation. See the ESM Methods for further details.

Short-term insulin experimentSee the ESM Methods for the description of this experiment.

Short-term insulin experiment

See the ESM Methods for the description of this experiment.

Rat protocolAdult females were fed ad libitum with either a control diet (20% protein) or an isoenergetic low-protein (LP) (8% protein) diet (Arie Blok, Woerden, the Netherlands) during gestation and lactation as described previously [32]. At 3 months of age, 11 control males and 11 males from LP litters, each of the 22 rats being from an independent litter, were killed and vastus lateralis muscle biopsies were excised and stored for later use. All animal procedures were approved by the local animal ethical review committee and were carried out under compliance with the UK Animal (Scientific Procedures) Act 1986. The investigation conforms to the Guide for the Care and Use of Laboratory Animals published by the US National Institutes of Health (NIH Publication no. 85–23, revised 1996). See the ESM Methods for further details.

Rat protocol

Adult females were fed ad libitum with either a control diet (20% protein) or an isoenergetic low-protein (LP) (8% protein) diet (Arie Blok, Woerden, the Netherlands) during gestation and lactation as described previously [32]. At 3 months of age, 11 control males and 11 males from LP litters, each of the 22 rats being from an independent litter, were killed and vastus lateralis muscle biopsies were excised and stored for later use. All animal procedures were approved by the local animal ethical review committee and were carried out under compliance with the UK Animal (Scientific Procedures) Act 1986. The investigation conforms to the Guide for the Care and Use of Laboratory Animals published by the US National Institutes of Health (NIH Publication no. 85–23, revised 1996). See the ESM Methods for further details.

L6 cellsL6 myoblasts were obtained from the American Type Culture Collection (CRL-1458, ATCC) and cultured in DMEM (Sigma-Aldrich, Poole, UK) with 10% fetal bovine serum, penicillin (100 units/ml), streptomycin (100 mg/ml) and glutamine (0.2 mmol/l), as previously described [33]. Semi-confluent cultures were treated with four different combinations of insulin (Actrapid; Novo Nordisk Insulin, Bagsværd, Denmark) and glucose levels, 0 nmol/l insulin and 5.6 mmol/l glucose, 100 nmol/l insulin and 5.6 mmol/l glucose, 0 nmol/l insulin and 25 mmol/l glucose or 100 nmol/l insulin and 25 mmol/l glucose, every 24 h for a total of 96 h until harvesting.

L6 myoblasts were obtained from the American Type Culture Collection (CRL-1458, ATCC) and cultured in DMEM (Sigma-Aldrich, Poole, UK) with 10% fetal bovine serum, penicillin (100 units/ml), streptomycin (100 mg/ml) and glutamine (0.2 mmol/l), as previously described [33]. Semi-confluent cultures were treated with four different combinations of insulin (Actrapid; Novo Nordisk Insulin, Bagsværd, Denmark) and glucose levels, 0 nmol/l insulin and 5.6 mmol/l glucose, 100 nmol/l insulin and 5.6 mmol/l glucose, 0 nmol/l insulin and 25 mmol/l glucose or 100 nmol/l insulin and 25 mmol/l glucose, every 24 h for a total of 96 h until harvesting.

Gene expressionRNA isolationThis is described in the RNA isolation section of the ESM Methods. ABI-PRISM 7900 Sequence Detection system ([ABI] Applied Biosystems, Foster City, CA, USA) was used for both miRNA and mRNA detection. Expression was quantified using TaqMan probes for miRNA and primers from ABI or the Roche Universal probe library for mRNA. Details are described in the ESM Methods.

Gene expression

RNA isolationThis is described in the RNA isolation section of the ESM Methods. ABI-PRISM 7900 Sequence Detection system ([ABI] Applied Biosystems, Foster City, CA, USA) was used for both miRNA and mRNA detection. Expression was quantified using TaqMan probes for miRNA and primers from ABI or the Roche Universal probe library for mRNA. Details are described in the ESM Methods.

RNA isolation

This is described in the RNA isolation section of the ESM Methods. ABI-PRISM 7900 Sequence Detection system ([ABI] Applied Biosystems, Foster City, CA, USA) was used for both miRNA and mRNA detection. Expression was quantified using TaqMan probes for miRNA and primers from ABI or the Roche Universal probe library for mRNA. Details are described in the ESM Methods.

Protein levelsMuscle cell culturesLevels of phosphoinositide 3-kinase R1 regulatory subunit 1 α (PIK3R1), Akt and p-Akt were measured using western blotting (Invitrogen). Insulin receptor (INSR) levels were measured using Meso Scale Discovery analysis (Meso Scale Discovery, Rockville, MD, USA).Muscle biopsiesThe protein levels of INSR, IRS-1 and PIK3R1 were measured using western blotting as described in [2, 34]. Details of these methods are described in the ESM Methods.

Protein levels

Muscle cell culturesLevels of phosphoinositide 3-kinase R1 regulatory subunit 1 α (PIK3R1), Akt and p-Akt were measured using western blotting (Invitrogen). Insulin receptor (INSR) levels were measured using Meso Scale Discovery analysis (Meso Scale Discovery, Rockville, MD, USA).

Muscle cell cultures

Levels of phosphoinositide 3-kinase R1 regulatory subunit 1 α (PIK3R1), Akt and p-Akt were measured using western blotting (Invitrogen). Insulin receptor (INSR) levels were measured using Meso Scale Discovery analysis (Meso Scale Discovery, Rockville, MD, USA).

Muscle biopsiesThe protein levels of INSR, IRS-1 and PIK3R1 were measured using western blotting as described in [2, 34]. Details of these methods are described in the ESM Methods.

Muscle biopsies

The protein levels of INSR, IRS-1 and PIK3R1 were measured using western blotting as described in [2, 34]. Details of these methods are described in the ESM Methods.

ResultsDifferentially expressed miRNAs in MZ twins discordant for type 2 diabetesTo identify miRNAs related to type 2 diabetes in a non-genetic manner, we studied 571 human miRNAs in skeletal muscle from 11 pairs of MZ twins discordant for type 2 diabetes. The clinical and metabolic characteristics of these twins are summarised in Table 1. Twenty miRNAs were identified as significantly downregulated after correction for multiple comparisons (pFDR < 0.1) in twins with type 2 diabetes compared with their corresponding non-diabetic co-twins (Table 2). No miRNAs were significantly upregulated in the type 2 diabetes twins. No difference in the total miRNA expression levels were found between the diabetic twins and their non-diabetic co-twins when the medians of the log2 transformed raw expression values were compared using a paired t test (p = 0.25). Box and density plots are shown in ESM Figs 1 and 2, respectively. Intra-pair correlations between miRNA expression and HbA1c showed that ten of the 20 identified miRNAs, including miR-15b, were negatively correlated with HbA1c levels. Of these miR-15b, miR-451 and miR-363 remained significant after FDR correction for multiple comparisons (pFDR < 0.1) (Table 2). Similar intra-pair correlations showed that miR-15b alone was significantly negatively correlated with the 2 h post-OGTT glucose values. This result was not significant after FDR correction (pFDR = 0.2) (Table 2).Table 2Differentially expressed miRNAs in skeletal muscle biopsies from genetically identical twins with type 2 diabetes when compared with their non-diabetic co-twinsmiRNAFamilyCFC
p

p
(FDR)

r
Glu120

p-r
Glu120

\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ {r}_{{\mathrm{HbA}}_{1\mathrm{c}}} $$\end{document}rHbA1c

p-
\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ {r}_{{\mathrm{HbA}}_{1\mathrm{c}}} $$\end{document}rHbA1c
hsa-miR-15bmiR-15a−2.02.4 × 10−8
8.0 × 10−6
−0.640.04−0.770.01hsa-miR-16miR-15a−1.75.7 × 10−6
9.4 × 10−4
−0.410.21−0.630.05hsa-miR-17miR-17b−1.77.3 × 10−5
0.007−0.530.1−0.660.04hsa-miR-451−1.89.0 × 10−5
0.007−0.550.08−0.760.01hsa-miR-106bmiR-17c−1.52.2 × 10−4
0.01−0.570.07−0.640.05hsa-miR-103miR-103−1.55.3 × 10−4
0.03−0.40.1−0.670.03hsa-miR-425−1.66.2 × 10−4
0.03−0.520.22−0.520.12hsa-miR-106amiR-17d−1.78.9 × 10−4
0.04−0.520.1−0.670.03hsa-miR-10b−1.51.0 × 10−3
0.040.10.77−0.10.77hsa-miR-25miR-25c−1.71.3 × 10−3
0.04−0.40.11−0.640.05hsa-miR-93miR-17c−1.51.5 × 10−3
0.04−0.510.23−0.550.1hsa-miR-363d−1.71.6 × 10−3
0.04−0.430.19−0.760.01hsa-miR-107miR-103−1.61.8 × 10−3
0.05−0.560.07−0.570.08hsa-miR-191−1.41.9 × 10−3
0.05−0.560.07−0.560.09hsa-miR-30e*−1.62.2 × 10−3
0.05−0.20.56−0.10.78hsa-let-7i−1.43.2 × 10−3
0.07−0.250.46−0.370.3hsa-miR-20amiR-17b−1.73.4 × 10−3
0.07−0.410.2−0.520.12hsa-miR-223−1.64.1 × 10−3
0.08−0.550.08−0.650.04hsa-miR-92amiR-25bd−1.44.9 × 10−3
0.09−0.540.04−0.590.08hsa-miR-20bmiR-17d−1.55.6 × 10−3
0.09−0.390.21−0.610.06The miR family names from miRBase demonstrate the relatedness between the miRNAsLetters in ‘C’ show which miRNAs are from the same transcriptional clusterFC, fold change; p
(FDR), p values adjusted with Benjamini–Hochberg’s false-discovery rate; r
Glu120, intra-pair correlation coefficient between miRNA expression and 120 min OGTT glucose values; r
HbA1c, intra-pair correlation coefficient between miRNA expression and HbA1c values; p-r
HbA1c and p-r
Glu120, the p values for the intra-pair correlation analysesThe diabetes-dysregulated miR-15 family is involved in insulin signallingThe 20 identified miRNAs belong to 11 different miRNA families (ESM Table 1); miRNAs within the same family are predicted to target the same mRNAs. As shown in Table 2, the top-ranking miRNAs according to statistical significance were miR-15b and miR-16. These both belong to the miR-15 family. The human miR-15 family consists of miR-15a, miR-15b, miR-16 and miR-195 (Fig. 1a), in addition miR-497 shares the seed sequence nucleotides 2–8. In humans, miR-15b and miR-16 originate from the same transcriptional cluster on chromosome 3 in the intron of SMC4, which encodes structural maintenance of chromosomes 4. In addition, miR-16 is transcribed from chromosome 13, where it is situated in the non-protein-coding region DLEU2 (deleted in lymphocytic leukaemia 2) clustered together with miR-15a. TargetScan predicts 968 conserved mRNA targets for the miR-15 family, and analysing these targets using IPA showed a total of 20 pathways that contained significantly higher numbers of the predicted miR-15 family targets than expected by chance (pFDR < 0.05) (Table 3). Among these, the insulin signalling pathway had the most significant over-representation of miR-15 family targets (pFDR = 0.004) (Fig. 1b and Table 3). This pathway analysis finding was confirmed using DAVID software, which was also used to perform additional pathway analyses on the predicted mRNA targets for each of the ten other miRNA families (ESM Table 1).Fig. 1The miR-15 family and its targets in the INSR signalling pathway. (a) Alignment of the nucleotide sequences of the human miR-15 family members using CLUSTAL 2.0.12. Asterisks indicate conserved residues. (b) The INSR signalling pathway. Grey nodes indicate predicted targets of the miR-15 family (tuberous sclerosis 1 [TSC1], Raf-1 proto-oncogene, serine/threonine kinase [RAF1], serum/glucocorticoid regulated kinase 1 [SGK1], thyroid hormone receptor interactor 10 [TRIP10], v-crk avian sarcoma virus CT10 oncogene homolog-like [CRKL], PIK3R1, protein kinase, cAMP-dependent, regulatory, type II, α [PRKAR2A], lipase, hormone-sensitive [LIPE], protein phosphatase 1, regulatory [inhibitor] subunit 11 [PPP1R11], eukaryotic translation initiation factor 2B, subunit 2 β, 39 kDa [eIF2B2], eukaryotic translation initiation factor 4E [eIF4E], growth factor receptor-bound protein 10 [GRB10], forkhead box O1 [FOXO1], IRS1, son of sevenless homolog 1 [Drosophila] [SOS1], eukaryotic translation initiation factor 2B, subunit 5 ε, 82 kDa [EIF2B5], Akt3, IRS2, INSR and mitogen-activated protein kinase kinase 1 [MAP2K1]. Additional information on the nodes in the pathway can be found in ESM Table 2. ER, endoplasmic reticulumTable 3Pathway analysis of the miR-15 family targetsIngenuity canonical pathways
p
(FDR)
INSR signalling0.0039AMPK signalling0.0045PPARα/RXRα activation0.0045TGF-β signalling0.0068Wnt/β-catenin signalling0.0126IGF-1 signalling0.0126Molecular mechanisms of cancer0.0126mTOR signalling0.0126EIF2 signalling0.0126PI3K/Akt signalling0.0126BMP signalling pathway0.015514-3-3-mediated signalling0.0158PTEN signalling0.0245Protein ubiquitination pathway0.0339Cell cycle regulation by BTG family proteins0.0339Axonal guidance signalling0.0339Ceramide signalling0.0339Mitotic roles of polo-like kinase0.0355Myc-mediated apoptosis signalling0.0417Cell cycle: G1/S checkpoint regulation0.0417IPA of the miR-15 family target genes predicted using TargetScan. Only pathways with p
(FDR) <0.05 are shownAMPK, protein kinase, AMP-activated, alpha 1 catalytic subunit; BMP, bone morphogenetic protein; BTG, B-cell translocation gene; EIF2, eukaryotic translation initiation factor 2; p
(FDR), Benjamini–Hochberg’s multiple comparison adjusted p values; PPARα/RXRα, peroxisome proliferator-activated receptor α/retinoid X receptor α; PTEN, phosphatase and tensin homologue; WNT, wingless-type MMTV integration site familymiR-15b and miR-16 regulate protein levels of insulin signalling moleculesThree prominent players of the insulin signalling pathway (INSR, IRS1 and PIK3R1) were among the predicted miR-15b/-16 targets. To address the role of miR-15b and miR-16 in the control of insulin signalling proteins, we measured the protein and mRNA expression levels of these three predicted targets in muscle biopsies from the non-diabetic twin population.We found miR-15b levels negatively associated with both INSR and IRS-1 but not with PIK3R1 protein levels when adjusted for confounding factors (Table 4). For miR-16, no significant associations with the levels of these proteins were observed. IRS1 mRNA levels was negatively associated with miR-15b, but not with miR-16. No significant associations between miR-15b or miR-16 and the mRNA levels of INSR and PIK3R1 were found (Table 4). Adjustments for fasting glucose levels and 2 h glucose values did not change any of the results.Table 4Impact of miR-15b and miR-16 on specific molecular targets in the 164 non-diabetic twinsTargetEffectCI 95%Change
p
INSR protein miR-15b0.91(0.86, 0.97)9%↓0.004 miR-160.93(0.86, 1.00)7%↓0.06
INSR mRNA miR-15b0.94(0.87, 1.02)0.1 miR-161.00(0.92, 1.09)1.0IRS-1 protein miR-15b0.82(0.75, 0.88)18%↓<0.0001 miR-160.95(0.87, 1.03)0.2
IRS-1 mRNA miR-15b0.92(0.86, 0.99)8%↓0.03 miR-160.95(0.88, 1.02)0.2PIK3R1 protein miR-15b0.97(0.93, 1.02)0.2 miR-161.04(1.00, 1.08)4%↑0.07
PIK3R1 mRNA miR-15b0.92(0.85, 1.01)8%↓0.07 miR-160.98(0.90, 1.07)0.6Multivariate analysis of the effects of miR-15b or miR-16 on target mRNA and protein expression with adjustment for age, sex, total fat percentage, \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max, birthweight, twin-pair and zygosity statusEffect, the effect on the response variable expressed as the change by 1 SD increase in either miR-15b or miR-16To study the potential regulation of INSR and PIK3R1 protein and mRNA levels by miR-15b and miR-16, we transfected antisense inhibitors specific to these miRNAs into human primary myocytes, with insulin added 5–10 min prior to harvest. Using ANOVA models we found no general effect of insulin on protein levels and no interaction effects. We observed a general increase in the total INSR protein content following miR-15b/miR-16 inhibition (p = 0.01) and a general downregulation of PIK3R1 protein levels in response to miR-15b/16 inhibition (p = 0.0002) (ESM Fig. 3). The result on PIK3R1 was in line with the borderline positive association found between PIK3R1 protein levels and miR-16 in the non-diabetic twins. As miR-15b and miR-16 target multiple insulin signalling molecules (Fig. 1b), it is relevant to study the collective effect of their inhibition; to this end, we measured the levels of p-Akt/Akt after miR-15b/miR-16 inhibition (ESM Fig. 4). We did not observe any effect of miR-15b and miR-16 inhibition on p-Akt/Akt levels. However, this result was considered inconclusive due to large variations in the p-Akt measurements.Effects of high glucose and insulin levels on miRNA expressions in cell cultureElevated glucose and insulin levels defining a diabetogenic environment might regulate miR-15b and miR-16. Short-term incubation of cultured human myocytes with 10 nmol/l insulin for 5, 10, 30, 60 or 120 min did not affect miR-15b or miR-16 levels (data not shown). We also tested the effect of high glucose and high insulin levels in L6 cells (a rat myoblast cell line) using a higher insulin concentration (100 nmol/l) with or without high glucose (high 25 mmol/l, low 5.6 mmol/l) for a prolonged time period of 96 h. Although the effects of high glucose (p = 0.2) and high insulin levels (p = 0.2) on miR-15b expression did not reach statistical significance in these cells (Fig. 2a), we found that high insulin (p = 0.01) and high glucose (p = 0.003) levels reduced the miR-16 levels (Fig. 2b).Fig. 2The effect of high insulin and high glucose on miR-15b and miR-16 expression levels in L6 cells. Levels of miR-15b (a) and miR-16 (b) were measured in L6 cells after 96 h of stimulation with either low glucose and no insulin, high glucose no insulin, low glucose with high insulin or high glucose with high insulin. Data are mean ± SD, n = 3. p values from two-way ANOVA: †
p = 0.02 for interaction effect, ‡‡
p = 0.01 for insulin effect, §§§
p = 0.003 for glucose effect. AU, arbitrary expression unit; LG, low glucose (5.6 mmol/l); HG, high glucose (25 mmol/l)Implications for intrauterine-dependent regulation of the miR-15 familyTo further explore the potential reasons for dysregulation of the miR-15 family in type 2 diabetes, we measured the expression levels of miR-15b and miR-16 in the non-diabetic twins. In this cohort, we found no associations between miR-15b and miR-16 levels on one side and insulin action (Rd clamp) or the 2 h post-OGTT glucose values on the other (data not shown). Interestingly, multiple regression analysis showed a negative association between birthweight and both miR-15b and miR-16 expression levels (Table 5). Total body fat percentage was positively associated with miR-16 but not miR-15b expression. Age, sex or \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \overset{\cdot }{V}{\mathrm{O}}_{2 \max } $$\end{document}V⋅O2max values were not associated with miR-15b or miR-16 expression levels (Table 5). To study further the effect of a poor intrauterine environment on the expression of miR-15b and miR-16, we studied muscle tissue from 3-month-old male rat offspring from dams fed either an LP or a control diet [35, 36]. The LP offspring had lower body weights both at day 3 after birth (p = 0.004) and at 3 months of age (p < 0.0001). Elevated miR-15b levels were found in the muscle of LP offspring (p < 0.001) (Fig. 3a), whereas the levels of miR-16 were similar in the two groups (Fig. 3b).Table 5Factors associated with the expression of either miR-15b or miR-16 in skeletal muscle obtained from the non-diabetic twinsEffect95% CIChange
p
miR-15b Age0.97(0.72, 1.28)0.8 Sex0.81(0.62, 1.05)0.1 
\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max
0.99(0.88, 1.11)0.9 Birthweight0.87(0.79, 0.96)13%↓0.005 Total fat percentage1.11(0.93, 1.34)0.3miR-16 Age1.28(0.99, 1.64)0.06 Sex1.06(0.83, 1.35)0.6 
\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max
1.08(0.97, 1.19)0.2 Birthweight0.87(0.80, 0.95)13%↓0.002 Total fat percentage1.27(1.08, 1.48)27%↑0.003The effects of age, sex, \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max, birthweight, and total body fat percentage on the expression of either miR-15b or miR-16, investigated by multiple regression analyses adjusted for twin-pair and zygosity statusEffect, the effect on the response variable expressed as the change by 1 SD increase in either miR-15b or miR-16Fig. 3The effect of maternal protein restriction on skeletal muscle expression levels of miR-15b and miR-16 in rats. miR-15b (a) and miR-16 (b) expression levels were measured in vastus lateralis muscle biopsies from 3-month-old male offspring of dams fed with low protein (n = 11) and control dams (n = 11). Data are mean ± SD. ***p < 0.001 for the effect of LP from two-sided unpaired t tests. AU, arbitrary expression unit; LP, low protein

Differentially expressed miRNAs in MZ twins discordant for type 2 diabetesTo identify miRNAs related to type 2 diabetes in a non-genetic manner, we studied 571 human miRNAs in skeletal muscle from 11 pairs of MZ twins discordant for type 2 diabetes. The clinical and metabolic characteristics of these twins are summarised in Table 1. Twenty miRNAs were identified as significantly downregulated after correction for multiple comparisons (pFDR < 0.1) in twins with type 2 diabetes compared with their corresponding non-diabetic co-twins (Table 2). No miRNAs were significantly upregulated in the type 2 diabetes twins. No difference in the total miRNA expression levels were found between the diabetic twins and their non-diabetic co-twins when the medians of the log2 transformed raw expression values were compared using a paired t test (p = 0.25). Box and density plots are shown in ESM Figs 1 and 2, respectively. Intra-pair correlations between miRNA expression and HbA1c showed that ten of the 20 identified miRNAs, including miR-15b, were negatively correlated with HbA1c levels. Of these miR-15b, miR-451 and miR-363 remained significant after FDR correction for multiple comparisons (pFDR < 0.1) (Table 2). Similar intra-pair correlations showed that miR-15b alone was significantly negatively correlated with the 2 h post-OGTT glucose values. This result was not significant after FDR correction (pFDR = 0.2) (Table 2).Table 2Differentially expressed miRNAs in skeletal muscle biopsies from genetically identical twins with type 2 diabetes when compared with their non-diabetic co-twinsmiRNAFamilyCFC
p

p
(FDR)

r
Glu120

p-r
Glu120

\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ {r}_{{\mathrm{HbA}}_{1\mathrm{c}}} $$\end{document}rHbA1c

p-
\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ {r}_{{\mathrm{HbA}}_{1\mathrm{c}}} $$\end{document}rHbA1c
hsa-miR-15bmiR-15a−2.02.4 × 10−8
8.0 × 10−6
−0.640.04−0.770.01hsa-miR-16miR-15a−1.75.7 × 10−6
9.4 × 10−4
−0.410.21−0.630.05hsa-miR-17miR-17b−1.77.3 × 10−5
0.007−0.530.1−0.660.04hsa-miR-451−1.89.0 × 10−5
0.007−0.550.08−0.760.01hsa-miR-106bmiR-17c−1.52.2 × 10−4
0.01−0.570.07−0.640.05hsa-miR-103miR-103−1.55.3 × 10−4
0.03−0.40.1−0.670.03hsa-miR-425−1.66.2 × 10−4
0.03−0.520.22−0.520.12hsa-miR-106amiR-17d−1.78.9 × 10−4
0.04−0.520.1−0.670.03hsa-miR-10b−1.51.0 × 10−3
0.040.10.77−0.10.77hsa-miR-25miR-25c−1.71.3 × 10−3
0.04−0.40.11−0.640.05hsa-miR-93miR-17c−1.51.5 × 10−3
0.04−0.510.23−0.550.1hsa-miR-363d−1.71.6 × 10−3
0.04−0.430.19−0.760.01hsa-miR-107miR-103−1.61.8 × 10−3
0.05−0.560.07−0.570.08hsa-miR-191−1.41.9 × 10−3
0.05−0.560.07−0.560.09hsa-miR-30e*−1.62.2 × 10−3
0.05−0.20.56−0.10.78hsa-let-7i−1.43.2 × 10−3
0.07−0.250.46−0.370.3hsa-miR-20amiR-17b−1.73.4 × 10−3
0.07−0.410.2−0.520.12hsa-miR-223−1.64.1 × 10−3
0.08−0.550.08−0.650.04hsa-miR-92amiR-25bd−1.44.9 × 10−3
0.09−0.540.04−0.590.08hsa-miR-20bmiR-17d−1.55.6 × 10−3
0.09−0.390.21−0.610.06The miR family names from miRBase demonstrate the relatedness between the miRNAsLetters in ‘C’ show which miRNAs are from the same transcriptional clusterFC, fold change; p
(FDR), p values adjusted with Benjamini–Hochberg’s false-discovery rate; r
Glu120, intra-pair correlation coefficient between miRNA expression and 120 min OGTT glucose values; r
HbA1c, intra-pair correlation coefficient between miRNA expression and HbA1c values; p-r
HbA1c and p-r
Glu120, the p values for the intra-pair correlation analyses

Differentially expressed miRNAs in MZ twins discordant for type 2 diabetes

To identify miRNAs related to type 2 diabetes in a non-genetic manner, we studied 571 human miRNAs in skeletal muscle from 11 pairs of MZ twins discordant for type 2 diabetes. The clinical and metabolic characteristics of these twins are summarised in Table 1. Twenty miRNAs were identified as significantly downregulated after correction for multiple comparisons (pFDR < 0.1) in twins with type 2 diabetes compared with their corresponding non-diabetic co-twins (Table 2). No miRNAs were significantly upregulated in the type 2 diabetes twins. No difference in the total miRNA expression levels were found between the diabetic twins and their non-diabetic co-twins when the medians of the log2 transformed raw expression values were compared using a paired t test (p = 0.25). Box and density plots are shown in ESM Figs 1 and 2, respectively. Intra-pair correlations between miRNA expression and HbA1c showed that ten of the 20 identified miRNAs, including miR-15b, were negatively correlated with HbA1c levels. Of these miR-15b, miR-451 and miR-363 remained significant after FDR correction for multiple comparisons (pFDR < 0.1) (Table 2). Similar intra-pair correlations showed that miR-15b alone was significantly negatively correlated with the 2 h post-OGTT glucose values. This result was not significant after FDR correction (pFDR = 0.2) (Table 2).Table 2Differentially expressed miRNAs in skeletal muscle biopsies from genetically identical twins with type 2 diabetes when compared with their non-diabetic co-twinsmiRNAFamilyCFC
p

p
(FDR)

r
Glu120

p-r
Glu120

\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ {r}_{{\mathrm{HbA}}_{1\mathrm{c}}} $$\end{document}rHbA1c

p-
\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ {r}_{{\mathrm{HbA}}_{1\mathrm{c}}} $$\end{document}rHbA1c
hsa-miR-15bmiR-15a−2.02.4 × 10−8
8.0 × 10−6
−0.640.04−0.770.01hsa-miR-16miR-15a−1.75.7 × 10−6
9.4 × 10−4
−0.410.21−0.630.05hsa-miR-17miR-17b−1.77.3 × 10−5
0.007−0.530.1−0.660.04hsa-miR-451−1.89.0 × 10−5
0.007−0.550.08−0.760.01hsa-miR-106bmiR-17c−1.52.2 × 10−4
0.01−0.570.07−0.640.05hsa-miR-103miR-103−1.55.3 × 10−4
0.03−0.40.1−0.670.03hsa-miR-425−1.66.2 × 10−4
0.03−0.520.22−0.520.12hsa-miR-106amiR-17d−1.78.9 × 10−4
0.04−0.520.1−0.670.03hsa-miR-10b−1.51.0 × 10−3
0.040.10.77−0.10.77hsa-miR-25miR-25c−1.71.3 × 10−3
0.04−0.40.11−0.640.05hsa-miR-93miR-17c−1.51.5 × 10−3
0.04−0.510.23−0.550.1hsa-miR-363d−1.71.6 × 10−3
0.04−0.430.19−0.760.01hsa-miR-107miR-103−1.61.8 × 10−3
0.05−0.560.07−0.570.08hsa-miR-191−1.41.9 × 10−3
0.05−0.560.07−0.560.09hsa-miR-30e*−1.62.2 × 10−3
0.05−0.20.56−0.10.78hsa-let-7i−1.43.2 × 10−3
0.07−0.250.46−0.370.3hsa-miR-20amiR-17b−1.73.4 × 10−3
0.07−0.410.2−0.520.12hsa-miR-223−1.64.1 × 10−3
0.08−0.550.08−0.650.04hsa-miR-92amiR-25bd−1.44.9 × 10−3
0.09−0.540.04−0.590.08hsa-miR-20bmiR-17d−1.55.6 × 10−3
0.09−0.390.21−0.610.06The miR family names from miRBase demonstrate the relatedness between the miRNAsLetters in ‘C’ show which miRNAs are from the same transcriptional clusterFC, fold change; p
(FDR), p values adjusted with Benjamini–Hochberg’s false-discovery rate; r
Glu120, intra-pair correlation coefficient between miRNA expression and 120 min OGTT glucose values; r
HbA1c, intra-pair correlation coefficient between miRNA expression and HbA1c values; p-r
HbA1c and p-r
Glu120, the p values for the intra-pair correlation analyses

Differentially expressed miRNAs in skeletal muscle biopsies from genetically identical twins with type 2 diabetes when compared with their non-diabetic co-twins

The miR family names from miRBase demonstrate the relatedness between the miRNAs

Letters in ‘C’ show which miRNAs are from the same transcriptional cluster

FC, fold change; p
(FDR), p values adjusted with Benjamini–Hochberg’s false-discovery rate; r
Glu120, intra-pair correlation coefficient between miRNA expression and 120 min OGTT glucose values; r
HbA1c, intra-pair correlation coefficient between miRNA expression and HbA1c values; p-r
HbA1c and p-r
Glu120, the p values for the intra-pair correlation analyses

The diabetes-dysregulated miR-15 family is involved in insulin signallingThe 20 identified miRNAs belong to 11 different miRNA families (ESM Table 1); miRNAs within the same family are predicted to target the same mRNAs. As shown in Table 2, the top-ranking miRNAs according to statistical significance were miR-15b and miR-16. These both belong to the miR-15 family. The human miR-15 family consists of miR-15a, miR-15b, miR-16 and miR-195 (Fig. 1a), in addition miR-497 shares the seed sequence nucleotides 2–8. In humans, miR-15b and miR-16 originate from the same transcriptional cluster on chromosome 3 in the intron of SMC4, which encodes structural maintenance of chromosomes 4. In addition, miR-16 is transcribed from chromosome 13, where it is situated in the non-protein-coding region DLEU2 (deleted in lymphocytic leukaemia 2) clustered together with miR-15a. TargetScan predicts 968 conserved mRNA targets for the miR-15 family, and analysing these targets using IPA showed a total of 20 pathways that contained significantly higher numbers of the predicted miR-15 family targets than expected by chance (pFDR < 0.05) (Table 3). Among these, the insulin signalling pathway had the most significant over-representation of miR-15 family targets (pFDR = 0.004) (Fig. 1b and Table 3). This pathway analysis finding was confirmed using DAVID software, which was also used to perform additional pathway analyses on the predicted mRNA targets for each of the ten other miRNA families (ESM Table 1).Fig. 1The miR-15 family and its targets in the INSR signalling pathway. (a) Alignment of the nucleotide sequences of the human miR-15 family members using CLUSTAL 2.0.12. Asterisks indicate conserved residues. (b) The INSR signalling pathway. Grey nodes indicate predicted targets of the miR-15 family (tuberous sclerosis 1 [TSC1], Raf-1 proto-oncogene, serine/threonine kinase [RAF1], serum/glucocorticoid regulated kinase 1 [SGK1], thyroid hormone receptor interactor 10 [TRIP10], v-crk avian sarcoma virus CT10 oncogene homolog-like [CRKL], PIK3R1, protein kinase, cAMP-dependent, regulatory, type II, α [PRKAR2A], lipase, hormone-sensitive [LIPE], protein phosphatase 1, regulatory [inhibitor] subunit 11 [PPP1R11], eukaryotic translation initiation factor 2B, subunit 2 β, 39 kDa [eIF2B2], eukaryotic translation initiation factor 4E [eIF4E], growth factor receptor-bound protein 10 [GRB10], forkhead box O1 [FOXO1], IRS1, son of sevenless homolog 1 [Drosophila] [SOS1], eukaryotic translation initiation factor 2B, subunit 5 ε, 82 kDa [EIF2B5], Akt3, IRS2, INSR and mitogen-activated protein kinase kinase 1 [MAP2K1]. Additional information on the nodes in the pathway can be found in ESM Table 2. ER, endoplasmic reticulumTable 3Pathway analysis of the miR-15 family targetsIngenuity canonical pathways
p
(FDR)
INSR signalling0.0039AMPK signalling0.0045PPARα/RXRα activation0.0045TGF-β signalling0.0068Wnt/β-catenin signalling0.0126IGF-1 signalling0.0126Molecular mechanisms of cancer0.0126mTOR signalling0.0126EIF2 signalling0.0126PI3K/Akt signalling0.0126BMP signalling pathway0.015514-3-3-mediated signalling0.0158PTEN signalling0.0245Protein ubiquitination pathway0.0339Cell cycle regulation by BTG family proteins0.0339Axonal guidance signalling0.0339Ceramide signalling0.0339Mitotic roles of polo-like kinase0.0355Myc-mediated apoptosis signalling0.0417Cell cycle: G1/S checkpoint regulation0.0417IPA of the miR-15 family target genes predicted using TargetScan. Only pathways with p
(FDR) <0.05 are shownAMPK, protein kinase, AMP-activated, alpha 1 catalytic subunit; BMP, bone morphogenetic protein; BTG, B-cell translocation gene; EIF2, eukaryotic translation initiation factor 2; p
(FDR), Benjamini–Hochberg’s multiple comparison adjusted p values; PPARα/RXRα, peroxisome proliferator-activated receptor α/retinoid X receptor α; PTEN, phosphatase and tensin homologue; WNT, wingless-type MMTV integration site family

The diabetes-dysregulated miR-15 family is involved in insulin signalling

The 20 identified miRNAs belong to 11 different miRNA families (ESM Table 1); miRNAs within the same family are predicted to target the same mRNAs. As shown in Table 2, the top-ranking miRNAs according to statistical significance were miR-15b and miR-16. These both belong to the miR-15 family. The human miR-15 family consists of miR-15a, miR-15b, miR-16 and miR-195 (Fig. 1a), in addition miR-497 shares the seed sequence nucleotides 2–8. In humans, miR-15b and miR-16 originate from the same transcriptional cluster on chromosome 3 in the intron of SMC4, which encodes structural maintenance of chromosomes 4. In addition, miR-16 is transcribed from chromosome 13, where it is situated in the non-protein-coding region DLEU2 (deleted in lymphocytic leukaemia 2) clustered together with miR-15a. TargetScan predicts 968 conserved mRNA targets for the miR-15 family, and analysing these targets using IPA showed a total of 20 pathways that contained significantly higher numbers of the predicted miR-15 family targets than expected by chance (pFDR < 0.05) (Table 3). Among these, the insulin signalling pathway had the most significant over-representation of miR-15 family targets (pFDR = 0.004) (Fig. 1b and Table 3). This pathway analysis finding was confirmed using DAVID software, which was also used to perform additional pathway analyses on the predicted mRNA targets for each of the ten other miRNA families (ESM Table 1).Fig. 1The miR-15 family and its targets in the INSR signalling pathway. (a) Alignment of the nucleotide sequences of the human miR-15 family members using CLUSTAL 2.0.12. Asterisks indicate conserved residues. (b) The INSR signalling pathway. Grey nodes indicate predicted targets of the miR-15 family (tuberous sclerosis 1 [TSC1], Raf-1 proto-oncogene, serine/threonine kinase [RAF1], serum/glucocorticoid regulated kinase 1 [SGK1], thyroid hormone receptor interactor 10 [TRIP10], v-crk avian sarcoma virus CT10 oncogene homolog-like [CRKL], PIK3R1, protein kinase, cAMP-dependent, regulatory, type II, α [PRKAR2A], lipase, hormone-sensitive [LIPE], protein phosphatase 1, regulatory [inhibitor] subunit 11 [PPP1R11], eukaryotic translation initiation factor 2B, subunit 2 β, 39 kDa [eIF2B2], eukaryotic translation initiation factor 4E [eIF4E], growth factor receptor-bound protein 10 [GRB10], forkhead box O1 [FOXO1], IRS1, son of sevenless homolog 1 [Drosophila] [SOS1], eukaryotic translation initiation factor 2B, subunit 5 ε, 82 kDa [EIF2B5], Akt3, IRS2, INSR and mitogen-activated protein kinase kinase 1 [MAP2K1]. Additional information on the nodes in the pathway can be found in ESM Table 2. ER, endoplasmic reticulumTable 3Pathway analysis of the miR-15 family targetsIngenuity canonical pathways
p
(FDR)
INSR signalling0.0039AMPK signalling0.0045PPARα/RXRα activation0.0045TGF-β signalling0.0068Wnt/β-catenin signalling0.0126IGF-1 signalling0.0126Molecular mechanisms of cancer0.0126mTOR signalling0.0126EIF2 signalling0.0126PI3K/Akt signalling0.0126BMP signalling pathway0.015514-3-3-mediated signalling0.0158PTEN signalling0.0245Protein ubiquitination pathway0.0339Cell cycle regulation by BTG family proteins0.0339Axonal guidance signalling0.0339Ceramide signalling0.0339Mitotic roles of polo-like kinase0.0355Myc-mediated apoptosis signalling0.0417Cell cycle: G1/S checkpoint regulation0.0417IPA of the miR-15 family target genes predicted using TargetScan. Only pathways with p
(FDR) <0.05 are shownAMPK, protein kinase, AMP-activated, alpha 1 catalytic subunit; BMP, bone morphogenetic protein; BTG, B-cell translocation gene; EIF2, eukaryotic translation initiation factor 2; p
(FDR), Benjamini–Hochberg’s multiple comparison adjusted p values; PPARα/RXRα, peroxisome proliferator-activated receptor α/retinoid X receptor α; PTEN, phosphatase and tensin homologue; WNT, wingless-type MMTV integration site family

The miR-15 family and its targets in the INSR signalling pathway. (a) Alignment of the nucleotide sequences of the human miR-15 family members using CLUSTAL 2.0.12. Asterisks indicate conserved residues. (b) The INSR signalling pathway. Grey nodes indicate predicted targets of the miR-15 family (tuberous sclerosis 1 [TSC1], Raf-1 proto-oncogene, serine/threonine kinase [RAF1], serum/glucocorticoid regulated kinase 1 [SGK1], thyroid hormone receptor interactor 10 [TRIP10], v-crk avian sarcoma virus CT10 oncogene homolog-like [CRKL], PIK3R1, protein kinase, cAMP-dependent, regulatory, type II, α [PRKAR2A], lipase, hormone-sensitive [LIPE], protein phosphatase 1, regulatory [inhibitor] subunit 11 [PPP1R11], eukaryotic translation initiation factor 2B, subunit 2 β, 39 kDa [eIF2B2], eukaryotic translation initiation factor 4E [eIF4E], growth factor receptor-bound protein 10 [GRB10], forkhead box O1 [FOXO1], IRS1, son of sevenless homolog 1 [Drosophila] [SOS1], eukaryotic translation initiation factor 2B, subunit 5 ε, 82 kDa [EIF2B5], Akt3, IRS2, INSR and mitogen-activated protein kinase kinase 1 [MAP2K1]. Additional information on the nodes in the pathway can be found in ESM Table 2. ER, endoplasmic reticulum

Pathway analysis of the miR-15 family targets

IPA of the miR-15 family target genes predicted using TargetScan. Only pathways with p
(FDR) <0.05 are shown

AMPK, protein kinase, AMP-activated, alpha 1 catalytic subunit; BMP, bone morphogenetic protein; BTG, B-cell translocation gene; EIF2, eukaryotic translation initiation factor 2; p
(FDR), Benjamini–Hochberg’s multiple comparison adjusted p values; PPARα/RXRα, peroxisome proliferator-activated receptor α/retinoid X receptor α; PTEN, phosphatase and tensin homologue; WNT, wingless-type MMTV integration site family

miR-15b and miR-16 regulate protein levels of insulin signalling moleculesThree prominent players of the insulin signalling pathway (INSR, IRS1 and PIK3R1) were among the predicted miR-15b/-16 targets. To address the role of miR-15b and miR-16 in the control of insulin signalling proteins, we measured the protein and mRNA expression levels of these three predicted targets in muscle biopsies from the non-diabetic twin population.We found miR-15b levels negatively associated with both INSR and IRS-1 but not with PIK3R1 protein levels when adjusted for confounding factors (Table 4). For miR-16, no significant associations with the levels of these proteins were observed. IRS1 mRNA levels was negatively associated with miR-15b, but not with miR-16. No significant associations between miR-15b or miR-16 and the mRNA levels of INSR and PIK3R1 were found (Table 4). Adjustments for fasting glucose levels and 2 h glucose values did not change any of the results.Table 4Impact of miR-15b and miR-16 on specific molecular targets in the 164 non-diabetic twinsTargetEffectCI 95%Change
p
INSR protein miR-15b0.91(0.86, 0.97)9%↓0.004 miR-160.93(0.86, 1.00)7%↓0.06
INSR mRNA miR-15b0.94(0.87, 1.02)0.1 miR-161.00(0.92, 1.09)1.0IRS-1 protein miR-15b0.82(0.75, 0.88)18%↓<0.0001 miR-160.95(0.87, 1.03)0.2
IRS-1 mRNA miR-15b0.92(0.86, 0.99)8%↓0.03 miR-160.95(0.88, 1.02)0.2PIK3R1 protein miR-15b0.97(0.93, 1.02)0.2 miR-161.04(1.00, 1.08)4%↑0.07
PIK3R1 mRNA miR-15b0.92(0.85, 1.01)8%↓0.07 miR-160.98(0.90, 1.07)0.6Multivariate analysis of the effects of miR-15b or miR-16 on target mRNA and protein expression with adjustment for age, sex, total fat percentage, \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max, birthweight, twin-pair and zygosity statusEffect, the effect on the response variable expressed as the change by 1 SD increase in either miR-15b or miR-16To study the potential regulation of INSR and PIK3R1 protein and mRNA levels by miR-15b and miR-16, we transfected antisense inhibitors specific to these miRNAs into human primary myocytes, with insulin added 5–10 min prior to harvest. Using ANOVA models we found no general effect of insulin on protein levels and no interaction effects. We observed a general increase in the total INSR protein content following miR-15b/miR-16 inhibition (p = 0.01) and a general downregulation of PIK3R1 protein levels in response to miR-15b/16 inhibition (p = 0.0002) (ESM Fig. 3). The result on PIK3R1 was in line with the borderline positive association found between PIK3R1 protein levels and miR-16 in the non-diabetic twins. As miR-15b and miR-16 target multiple insulin signalling molecules (Fig. 1b), it is relevant to study the collective effect of their inhibition; to this end, we measured the levels of p-Akt/Akt after miR-15b/miR-16 inhibition (ESM Fig. 4). We did not observe any effect of miR-15b and miR-16 inhibition on p-Akt/Akt levels. However, this result was considered inconclusive due to large variations in the p-Akt measurements.

miR-15b and miR-16 regulate protein levels of insulin signalling molecules

Three prominent players of the insulin signalling pathway (INSR, IRS1 and PIK3R1) were among the predicted miR-15b/-16 targets. To address the role of miR-15b and miR-16 in the control of insulin signalling proteins, we measured the protein and mRNA expression levels of these three predicted targets in muscle biopsies from the non-diabetic twin population.

We found miR-15b levels negatively associated with both INSR and IRS-1 but not with PIK3R1 protein levels when adjusted for confounding factors (Table 4). For miR-16, no significant associations with the levels of these proteins were observed. IRS1 mRNA levels was negatively associated with miR-15b, but not with miR-16. No significant associations between miR-15b or miR-16 and the mRNA levels of INSR and PIK3R1 were found (Table 4). Adjustments for fasting glucose levels and 2 h glucose values did not change any of the results.Table 4Impact of miR-15b and miR-16 on specific molecular targets in the 164 non-diabetic twinsTargetEffectCI 95%Change
p
INSR protein miR-15b0.91(0.86, 0.97)9%↓0.004 miR-160.93(0.86, 1.00)7%↓0.06
INSR mRNA miR-15b0.94(0.87, 1.02)0.1 miR-161.00(0.92, 1.09)1.0IRS-1 protein miR-15b0.82(0.75, 0.88)18%↓<0.0001 miR-160.95(0.87, 1.03)0.2
IRS-1 mRNA miR-15b0.92(0.86, 0.99)8%↓0.03 miR-160.95(0.88, 1.02)0.2PIK3R1 protein miR-15b0.97(0.93, 1.02)0.2 miR-161.04(1.00, 1.08)4%↑0.07
PIK3R1 mRNA miR-15b0.92(0.85, 1.01)8%↓0.07 miR-160.98(0.90, 1.07)0.6Multivariate analysis of the effects of miR-15b or miR-16 on target mRNA and protein expression with adjustment for age, sex, total fat percentage, \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max, birthweight, twin-pair and zygosity statusEffect, the effect on the response variable expressed as the change by 1 SD increase in either miR-15b or miR-16

Impact of miR-15b and miR-16 on specific molecular targets in the 164 non-diabetic twins

Multivariate analysis of the effects of miR-15b or miR-16 on target mRNA and protein expression with adjustment for age, sex, total fat percentage, \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max, birthweight, twin-pair and zygosity status

Effect, the effect on the response variable expressed as the change by 1 SD increase in either miR-15b or miR-16

To study the potential regulation of INSR and PIK3R1 protein and mRNA levels by miR-15b and miR-16, we transfected antisense inhibitors specific to these miRNAs into human primary myocytes, with insulin added 5–10 min prior to harvest. Using ANOVA models we found no general effect of insulin on protein levels and no interaction effects. We observed a general increase in the total INSR protein content following miR-15b/miR-16 inhibition (p = 0.01) and a general downregulation of PIK3R1 protein levels in response to miR-15b/16 inhibition (p = 0.0002) (ESM Fig. 3). The result on PIK3R1 was in line with the borderline positive association found between PIK3R1 protein levels and miR-16 in the non-diabetic twins. As miR-15b and miR-16 target multiple insulin signalling molecules (Fig. 1b), it is relevant to study the collective effect of their inhibition; to this end, we measured the levels of p-Akt/Akt after miR-15b/miR-16 inhibition (ESM Fig. 4). We did not observe any effect of miR-15b and miR-16 inhibition on p-Akt/Akt levels. However, this result was considered inconclusive due to large variations in the p-Akt measurements.

Effects of high glucose and insulin levels on miRNA expressions in cell cultureElevated glucose and insulin levels defining a diabetogenic environment might regulate miR-15b and miR-16. Short-term incubation of cultured human myocytes with 10 nmol/l insulin for 5, 10, 30, 60 or 120 min did not affect miR-15b or miR-16 levels (data not shown). We also tested the effect of high glucose and high insulin levels in L6 cells (a rat myoblast cell line) using a higher insulin concentration (100 nmol/l) with or without high glucose (high 25 mmol/l, low 5.6 mmol/l) for a prolonged time period of 96 h. Although the effects of high glucose (p = 0.2) and high insulin levels (p = 0.2) on miR-15b expression did not reach statistical significance in these cells (Fig. 2a), we found that high insulin (p = 0.01) and high glucose (p = 0.003) levels reduced the miR-16 levels (Fig. 2b).Fig. 2The effect of high insulin and high glucose on miR-15b and miR-16 expression levels in L6 cells. Levels of miR-15b (a) and miR-16 (b) were measured in L6 cells after 96 h of stimulation with either low glucose and no insulin, high glucose no insulin, low glucose with high insulin or high glucose with high insulin. Data are mean ± SD, n = 3. p values from two-way ANOVA: †
p = 0.02 for interaction effect, ‡‡
p = 0.01 for insulin effect, §§§
p = 0.003 for glucose effect. AU, arbitrary expression unit; LG, low glucose (5.6 mmol/l); HG, high glucose (25 mmol/l)

Effects of high glucose and insulin levels on miRNA expressions in cell culture

Elevated glucose and insulin levels defining a diabetogenic environment might regulate miR-15b and miR-16. Short-term incubation of cultured human myocytes with 10 nmol/l insulin for 5, 10, 30, 60 or 120 min did not affect miR-15b or miR-16 levels (data not shown). We also tested the effect of high glucose and high insulin levels in L6 cells (a rat myoblast cell line) using a higher insulin concentration (100 nmol/l) with or without high glucose (high 25 mmol/l, low 5.6 mmol/l) for a prolonged time period of 96 h. Although the effects of high glucose (p = 0.2) and high insulin levels (p = 0.2) on miR-15b expression did not reach statistical significance in these cells (Fig. 2a), we found that high insulin (p = 0.01) and high glucose (p = 0.003) levels reduced the miR-16 levels (Fig. 2b).Fig. 2The effect of high insulin and high glucose on miR-15b and miR-16 expression levels in L6 cells. Levels of miR-15b (a) and miR-16 (b) were measured in L6 cells after 96 h of stimulation with either low glucose and no insulin, high glucose no insulin, low glucose with high insulin or high glucose with high insulin. Data are mean ± SD, n = 3. p values from two-way ANOVA: †
p = 0.02 for interaction effect, ‡‡
p = 0.01 for insulin effect, §§§
p = 0.003 for glucose effect. AU, arbitrary expression unit; LG, low glucose (5.6 mmol/l); HG, high glucose (25 mmol/l)

The effect of high insulin and high glucose on miR-15b and miR-16 expression levels in L6 cells. Levels of miR-15b (a) and miR-16 (b) were measured in L6 cells after 96 h of stimulation with either low glucose and no insulin, high glucose no insulin, low glucose with high insulin or high glucose with high insulin. Data are mean ± SD, n = 3. p values from two-way ANOVA: †
p = 0.02 for interaction effect, ‡‡
p = 0.01 for insulin effect, §§§
p = 0.003 for glucose effect. AU, arbitrary expression unit; LG, low glucose (5.6 mmol/l); HG, high glucose (25 mmol/l)

Implications for intrauterine-dependent regulation of the miR-15 familyTo further explore the potential reasons for dysregulation of the miR-15 family in type 2 diabetes, we measured the expression levels of miR-15b and miR-16 in the non-diabetic twins. In this cohort, we found no associations between miR-15b and miR-16 levels on one side and insulin action (Rd clamp) or the 2 h post-OGTT glucose values on the other (data not shown). Interestingly, multiple regression analysis showed a negative association between birthweight and both miR-15b and miR-16 expression levels (Table 5). Total body fat percentage was positively associated with miR-16 but not miR-15b expression. Age, sex or \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \overset{\cdot }{V}{\mathrm{O}}_{2 \max } $$\end{document}V⋅O2max values were not associated with miR-15b or miR-16 expression levels (Table 5). To study further the effect of a poor intrauterine environment on the expression of miR-15b and miR-16, we studied muscle tissue from 3-month-old male rat offspring from dams fed either an LP or a control diet [35, 36]. The LP offspring had lower body weights both at day 3 after birth (p = 0.004) and at 3 months of age (p < 0.0001). Elevated miR-15b levels were found in the muscle of LP offspring (p < 0.001) (Fig. 3a), whereas the levels of miR-16 were similar in the two groups (Fig. 3b).Table 5Factors associated with the expression of either miR-15b or miR-16 in skeletal muscle obtained from the non-diabetic twinsEffect95% CIChange
p
miR-15b Age0.97(0.72, 1.28)0.8 Sex0.81(0.62, 1.05)0.1 
\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max
0.99(0.88, 1.11)0.9 Birthweight0.87(0.79, 0.96)13%↓0.005 Total fat percentage1.11(0.93, 1.34)0.3miR-16 Age1.28(0.99, 1.64)0.06 Sex1.06(0.83, 1.35)0.6 
\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max
1.08(0.97, 1.19)0.2 Birthweight0.87(0.80, 0.95)13%↓0.002 Total fat percentage1.27(1.08, 1.48)27%↑0.003The effects of age, sex, \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max, birthweight, and total body fat percentage on the expression of either miR-15b or miR-16, investigated by multiple regression analyses adjusted for twin-pair and zygosity statusEffect, the effect on the response variable expressed as the change by 1 SD increase in either miR-15b or miR-16Fig. 3The effect of maternal protein restriction on skeletal muscle expression levels of miR-15b and miR-16 in rats. miR-15b (a) and miR-16 (b) expression levels were measured in vastus lateralis muscle biopsies from 3-month-old male offspring of dams fed with low protein (n = 11) and control dams (n = 11). Data are mean ± SD. ***p < 0.001 for the effect of LP from two-sided unpaired t tests. AU, arbitrary expression unit; LP, low protein

Implications for intrauterine-dependent regulation of the miR-15 family

To further explore the potential reasons for dysregulation of the miR-15 family in type 2 diabetes, we measured the expression levels of miR-15b and miR-16 in the non-diabetic twins. In this cohort, we found no associations between miR-15b and miR-16 levels on one side and insulin action (Rd clamp) or the 2 h post-OGTT glucose values on the other (data not shown). Interestingly, multiple regression analysis showed a negative association between birthweight and both miR-15b and miR-16 expression levels (Table 5). Total body fat percentage was positively associated with miR-16 but not miR-15b expression. Age, sex or \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \overset{\cdot }{V}{\mathrm{O}}_{2 \max } $$\end{document}V⋅O2max values were not associated with miR-15b or miR-16 expression levels (Table 5). To study further the effect of a poor intrauterine environment on the expression of miR-15b and miR-16, we studied muscle tissue from 3-month-old male rat offspring from dams fed either an LP or a control diet [35, 36]. The LP offspring had lower body weights both at day 3 after birth (p = 0.004) and at 3 months of age (p < 0.0001). Elevated miR-15b levels were found in the muscle of LP offspring (p < 0.001) (Fig. 3a), whereas the levels of miR-16 were similar in the two groups (Fig. 3b).Table 5Factors associated with the expression of either miR-15b or miR-16 in skeletal muscle obtained from the non-diabetic twinsEffect95% CIChange
p
miR-15b Age0.97(0.72, 1.28)0.8 Sex0.81(0.62, 1.05)0.1 
\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max
0.99(0.88, 1.11)0.9 Birthweight0.87(0.79, 0.96)13%↓0.005 Total fat percentage1.11(0.93, 1.34)0.3miR-16 Age1.28(0.99, 1.64)0.06 Sex1.06(0.83, 1.35)0.6 
\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max
1.08(0.97, 1.19)0.2 Birthweight0.87(0.80, 0.95)13%↓0.002 Total fat percentage1.27(1.08, 1.48)27%↑0.003The effects of age, sex, \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max, birthweight, and total body fat percentage on the expression of either miR-15b or miR-16, investigated by multiple regression analyses adjusted for twin-pair and zygosity statusEffect, the effect on the response variable expressed as the change by 1 SD increase in either miR-15b or miR-16Fig. 3The effect of maternal protein restriction on skeletal muscle expression levels of miR-15b and miR-16 in rats. miR-15b (a) and miR-16 (b) expression levels were measured in vastus lateralis muscle biopsies from 3-month-old male offspring of dams fed with low protein (n = 11) and control dams (n = 11). Data are mean ± SD. ***p < 0.001 for the effect of LP from two-sided unpaired t tests. AU, arbitrary expression unit; LP, low protein

Factors associated with the expression of either miR-15b or miR-16 in skeletal muscle obtained from the non-diabetic twins

The effects of age, sex, \documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \dot{V}{\mathrm{O}}_{2 \max } $$\end{document}V˙O2max, birthweight, and total body fat percentage on the expression of either miR-15b or miR-16, investigated by multiple regression analyses adjusted for twin-pair and zygosity status

Effect, the effect on the response variable expressed as the change by 1 SD increase in either miR-15b or miR-16

The effect of maternal protein restriction on skeletal muscle expression levels of miR-15b and miR-16 in rats. miR-15b (a) and miR-16 (b) expression levels were measured in vastus lateralis muscle biopsies from 3-month-old male offspring of dams fed with low protein (n = 11) and control dams (n = 11). Data are mean ± SD. ***p < 0.001 for the effect of LP from two-sided unpaired t tests. AU, arbitrary expression unit; LP, low protein

DiscussionIn a unique cohort of 11 MZ twin pairs discordant for type 2 diabetes, we identified 20 miRNAs that were differentially expressed in the skeletal muscle of twins with type 2 diabetes compared with their non-diabetic identical twins. In silico analyses identified the two most significantly downregulated miRNAs in twins with diabetes, miR-15b and miR-16, as possible regulators of important metabolic functions including insulin signalling. The predicted insulin signalling targets of miR-15b and miR-16 included INSR, IRS-1 and PIK3R1, for which we demonstrated a likely biological role in vivo (for INSR and IRS-1) and in vitro (for INSR and p85). We found a significant positive association between LBW and both miR-15b and miR-16 in non-diabetic twins and elevated miR-15b levels in a rat model of LBW. Finally, we showed a downregulation of miR-16 by high glucose and insulin levels in cell cultures.Interestingly, all of the 20 differentially expressed miRNAs in the MZ twins with overt type 2 diabetes were downregulated compared with their genetically identical twins, most of whom had IGT. In support of this we found eight of the 20 miRNAs, including miR-15b and miR-16, to be downregulated in singletons with overt type 2 diabetes when we compared their levels to those in singletons with IGT using an online dataset [12]. Differences between these studies may be explained by the selective identification of non-genetic differences in discordant MZ twins.Diabetes is defined by the presence of hyperglycaemia, and we therefore tested the correlation between average glycaemic levels (HbA1c) and the expression levels of each of the 20 miRNAs. Although only three of these miRNAs, including miR-15b, showed a significant negative correlation with HbA1c level after correction for multiple testing, we found the same trend for all the 20 miRNAs, suggesting that hyperglycaemia may be involved in the coordinated downregulation of miRNAs in twins with overt diabetes. Whether this downregulation represents, for example, secondary potentially deleterious changes due to hyperglycaemia, or plays a role in the compensation for insulin resistance via hyperglycaemia-induced glucose uptake [37] remains to be investigated.A previous study in pancreatic beta cells showed upregulated miR-15a levels after hyperglycaemic exposure for 1 h, with subsequent downregulation after 3 days of exposure [38]. We found that long-term exposure to elevated insulin and glucose levels was associated with a downregulation of miR-16 but not miR-15b in L6 cells, which suggests differential regulation. Acute insulin stimulation for up to 120 min in human myocytes using a concentration of 10 nmol/l did not show any effect, supporting previous in vivo clamp results in humans [39].Based on the presence of complementary binding sites for the miR-15 family in the 3′UTR of mRNAs, we predicted insulin signalling proteins including INSR, IRS-1 and PIK3R1 to represent important targets for miR-15b and miR-16. Indeed, our experiments in human myocytes showed that the levels of INSR and PIK3R1 protein were altered following inhibition of miR-15b/miR-16. INSR was upregulated, indicating that miR-15b/miR-16 represses INSR translation, whereas PIK3R1 was downregulated, indicating enhanced translation. Both miRNA-mediated down- [10] and upregulation [40, 41] of translation have previously been described. Increased PIK3R1 levels were previously reported to be associated with insulin resistance [42, 43], thus elevated miR-15b and miR-16 levels could contribute to insulin resistance through downregulation of INSR and a parallel upregulation of PIK3R1. The biological relevance of our results was demonstrated in the non-diabetic twins, with a negative association between levels of miR-15b and INSR, a borderline significant positive association between miR-16 and PIK3R1 and a highly significant negative association between miR-15b and IRS-1 protein levels. 3′UTR luciferase studies could be instrumental in providing more insights into the extent to which miR-15b and miR-16 bind to the 3′UTR regions of their predicted targets, and to determine the extent to which these miRNAs—alone or in concert with other miRNAs—may influence the expression of one or more insulin signalling proteins. Given the complex and unknown interactions of miR-15b and miR-16 with many other miRNAs, including the 20 miRNA we found downregulated in type 2 diabetes, such experiments will need to be addressed in future papers.There are multiple routes via which the insulin signal travels [44, 45], and the predicted targets of miR-15b and miR-16 suggest several that could be interesting. Here, we tested downstream signalling through p-Akt/Akt but reached no clear conclusion; thus, further studies will be needed to determine the specific route of action for these miRNAs as well as to determine their role in the other pathways containing their targets, several of which are known or suspected to influence glucose metabolism. We did not find any significant correlations between levels of miR-15b and miR-16 and in vivo peripheral insulin resistance in the non-diabetic twins. This may suggest that the absolute muscle expression level of these miRNAs plays only a minor role in the control of in vivo whole-body insulin action or that the potential effects need unmasking by adverse ‘diabetogenic’ lifestyle conditions.In our large cohort of non-diabetic twins, we observed a negative association between birthweight and miR-15b and miR-16 levels, which was further supported by elevated miR-15b levels in an LBW rat model. The lack of significantly elevated miR-16 levels in the rat model could reflect interspecies differences in the regulation of the two clusters from which the miR-15 family are transcribed. The upregulation of miR-15b and miR-16 in twins with LBW, and the opposing downregulation in the MZ twins with diabetes, is to some extent paradoxical as LBW is associated with increased risk of developing type 2 diabetes. It suggests differential regulation as well as a potential impact of miR-15b and miR-16 at different stages in the life-long development of type 2 diabetes. This is supported by the finding of upregulated miR-15b and miR-16 levels in individuals with IGT when compared with both individuals with NGT and those with type 2 diabetes in an independent online dataset [12]. In addition, we previously reported a similar differential regulation of insulin signalling genes in patients with overt type 2 diabetes compared with their first-degree relatives which, similar to individuals with IGT and individuals born with LBW, have an increased risk of developing T2D [46]. Whether an adverse intrauterine environment directly alters miR-15b and miR-16 levels and the extent to which these miRNAs play a role in the development of insulin resistance in individuals with an increased risk of developing type 2 diabetes or whether they perhaps work to counteract the enhanced activity in the insulin signalling pathway that we have previously observed in insulin-resistant individuals [47] remains to be disclosed by future studies.In conclusion, we identified 20 downregulated miRNAs including miR-15b and miR-16 in twins with overt type 2 diabetes compared with their genetically identical non-diabetic co-twins. In a larger cohort of non-diabetic twins, LBW was somewhat paradoxically associated with upregulation of miR-15b and miR-16 levels. Our results indicate that miR-15b and miR-16 may regulate key insulin signalling protein levels. Additional studies are needed to understand not only the role of the miR-15 family in the development of insulin resistance and type 2 diabetes, but also the distinct or interactive roles of the additional 18 downregulated miRNAs in type 2 diabetes.

In a unique cohort of 11 MZ twin pairs discordant for type 2 diabetes, we identified 20 miRNAs that were differentially expressed in the skeletal muscle of twins with type 2 diabetes compared with their non-diabetic identical twins. In silico analyses identified the two most significantly downregulated miRNAs in twins with diabetes, miR-15b and miR-16, as possible regulators of important metabolic functions including insulin signalling. The predicted insulin signalling targets of miR-15b and miR-16 included INSR, IRS-1 and PIK3R1, for which we demonstrated a likely biological role in vivo (for INSR and IRS-1) and in vitro (for INSR and p85). We found a significant positive association between LBW and both miR-15b and miR-16 in non-diabetic twins and elevated miR-15b levels in a rat model of LBW. Finally, we showed a downregulation of miR-16 by high glucose and insulin levels in cell cultures.

Interestingly, all of the 20 differentially expressed miRNAs in the MZ twins with overt type 2 diabetes were downregulated compared with their genetically identical twins, most of whom had IGT. In support of this we found eight of the 20 miRNAs, including miR-15b and miR-16, to be downregulated in singletons with overt type 2 diabetes when we compared their levels to those in singletons with IGT using an online dataset [12]. Differences between these studies may be explained by the selective identification of non-genetic differences in discordant MZ twins.

Diabetes is defined by the presence of hyperglycaemia, and we therefore tested the correlation between average glycaemic levels (HbA1c) and the expression levels of each of the 20 miRNAs. Although only three of these miRNAs, including miR-15b, showed a significant negative correlation with HbA1c level after correction for multiple testing, we found the same trend for all the 20 miRNAs, suggesting that hyperglycaemia may be involved in the coordinated downregulation of miRNAs in twins with overt diabetes. Whether this downregulation represents, for example, secondary potentially deleterious changes due to hyperglycaemia, or plays a role in the compensation for insulin resistance via hyperglycaemia-induced glucose uptake [37] remains to be investigated.

A previous study in pancreatic beta cells showed upregulated miR-15a levels after hyperglycaemic exposure for 1 h, with subsequent downregulation after 3 days of exposure [38]. We found that long-term exposure to elevated insulin and glucose levels was associated with a downregulation of miR-16 but not miR-15b in L6 cells, which suggests differential regulation. Acute insulin stimulation for up to 120 min in human myocytes using a concentration of 10 nmol/l did not show any effect, supporting previous in vivo clamp results in humans [39].

Based on the presence of complementary binding sites for the miR-15 family in the 3′UTR of mRNAs, we predicted insulin signalling proteins including INSR, IRS-1 and PIK3R1 to represent important targets for miR-15b and miR-16. Indeed, our experiments in human myocytes showed that the levels of INSR and PIK3R1 protein were altered following inhibition of miR-15b/miR-16. INSR was upregulated, indicating that miR-15b/miR-16 represses INSR translation, whereas PIK3R1 was downregulated, indicating enhanced translation. Both miRNA-mediated down- [10] and upregulation [40, 41] of translation have previously been described. Increased PIK3R1 levels were previously reported to be associated with insulin resistance [42, 43], thus elevated miR-15b and miR-16 levels could contribute to insulin resistance through downregulation of INSR and a parallel upregulation of PIK3R1. The biological relevance of our results was demonstrated in the non-diabetic twins, with a negative association between levels of miR-15b and INSR, a borderline significant positive association between miR-16 and PIK3R1 and a highly significant negative association between miR-15b and IRS-1 protein levels. 3′UTR luciferase studies could be instrumental in providing more insights into the extent to which miR-15b and miR-16 bind to the 3′UTR regions of their predicted targets, and to determine the extent to which these miRNAs—alone or in concert with other miRNAs—may influence the expression of one or more insulin signalling proteins. Given the complex and unknown interactions of miR-15b and miR-16 with many other miRNAs, including the 20 miRNA we found downregulated in type 2 diabetes, such experiments will need to be addressed in future papers.

There are multiple routes via which the insulin signal travels [44, 45], and the predicted targets of miR-15b and miR-16 suggest several that could be interesting. Here, we tested downstream signalling through p-Akt/Akt but reached no clear conclusion; thus, further studies will be needed to determine the specific route of action for these miRNAs as well as to determine their role in the other pathways containing their targets, several of which are known or suspected to influence glucose metabolism. We did not find any significant correlations between levels of miR-15b and miR-16 and in vivo peripheral insulin resistance in the non-diabetic twins. This may suggest that the absolute muscle expression level of these miRNAs plays only a minor role in the control of in vivo whole-body insulin action or that the potential effects need unmasking by adverse ‘diabetogenic’ lifestyle conditions.

In our large cohort of non-diabetic twins, we observed a negative association between birthweight and miR-15b and miR-16 levels, which was further supported by elevated miR-15b levels in an LBW rat model. The lack of significantly elevated miR-16 levels in the rat model could reflect interspecies differences in the regulation of the two clusters from which the miR-15 family are transcribed. The upregulation of miR-15b and miR-16 in twins with LBW, and the opposing downregulation in the MZ twins with diabetes, is to some extent paradoxical as LBW is associated with increased risk of developing type 2 diabetes. It suggests differential regulation as well as a potential impact of miR-15b and miR-16 at different stages in the life-long development of type 2 diabetes. This is supported by the finding of upregulated miR-15b and miR-16 levels in individuals with IGT when compared with both individuals with NGT and those with type 2 diabetes in an independent online dataset [12]. In addition, we previously reported a similar differential regulation of insulin signalling genes in patients with overt type 2 diabetes compared with their first-degree relatives which, similar to individuals with IGT and individuals born with LBW, have an increased risk of developing T2D [46]. Whether an adverse intrauterine environment directly alters miR-15b and miR-16 levels and the extent to which these miRNAs play a role in the development of insulin resistance in individuals with an increased risk of developing type 2 diabetes or whether they perhaps work to counteract the enhanced activity in the insulin signalling pathway that we have previously observed in insulin-resistant individuals [47] remains to be disclosed by future studies.

In conclusion, we identified 20 downregulated miRNAs including miR-15b and miR-16 in twins with overt type 2 diabetes compared with their genetically identical non-diabetic co-twins. In a larger cohort of non-diabetic twins, LBW was somewhat paradoxically associated with upregulation of miR-15b and miR-16 levels. Our results indicate that miR-15b and miR-16 may regulate key insulin signalling protein levels. Additional studies are needed to understand not only the role of the miR-15 family in the development of insulin resistance and type 2 diabetes, but also the distinct or interactive roles of the additional 18 downregulated miRNAs in type 2 diabetes.

Electronic supplementary materialESM Methods(PDF 152 kb)ESM Fig. 1(PDF 145 kb)ESM Fig. 2(PDF 315 kb)ESM Fig. 3(PDF 153 kb)ESM Fig. 4(PDF 125 kb)ESM Table 1(PDF 190 kb)ESM Table 2(PDF 29.7 kb)

Electronic supplementary material

ESM Methods(PDF 152 kb)ESM Fig. 1(PDF 145 kb)ESM Fig. 2(PDF 315 kb)ESM Fig. 3(PDF 153 kb)ESM Fig. 4(PDF 125 kb)ESM Table 1(PDF 190 kb)ESM Table 2(PDF 29.7 kb)

(PDF 152 kb)

(PDF 145 kb)

(PDF 315 kb)

(PDF 153 kb)

(PDF 125 kb)

(PDF 190 kb)

(PDF 29.7 kb)
