---
pmid: '35916866'
title: De Novo ZMYND8 variants result in an autosomal dominant neurodevelopmental
  disorder with cardiac malformations.
authors:
- Dias KR
- Carlston CM
- Blok LER
- De Hayr L
- Nawaz U
- Evans CA
- Bayrak-Toydemir P
- Htun S
- Zhu Y
- Ma A
- Lynch SA
- Moorwood C
- Stals K
- Ellard S
- Bainbridge MN
- Friedman J
- Pappas JG
- Rabin R
- Nowak CB
- Douglas J
- Wilson TE
- Guillen Sacoto MJ
- Mullegama SV
- Palculict TB
- Kirk EP
- Pinner JR
- Edwards M
- Montanari F
- Graziano C
- Pippucci T
- Dingmann B
- Glass I
- Mefford HC
- Shimoji T
- Suzuki T
- Yamakawa K
- Streff H
- Schaaf CP
- Slavotinek AM
- Voineagu I
- Carey JC
- Buckley MF
- Schenck A
- Harvey RJ
- Roscioli T
journal: Genet Med
year: '2022'
full_text_available: false
doi: 10.1016/j.gim.2022.06.001
---

# De Novo ZMYND8 variants result in an autosomal dominant neurodevelopmental disorder with cardiac malformations.
**Authors:** Dias KR, Carlston CM, Blok LER, De Hayr L, Nawaz U, Evans CA, Bayrak-Toydemir P, Htun S, Zhu Y, Ma A, Lynch SA, Moorwood C, Stals K, Ellard S, Bainbridge MN, Friedman J, Pappas JG, Rabin R, Nowak CB, Douglas J, Wilson TE, Guillen Sacoto MJ, Mullegama SV, Palculict TB, Kirk EP, Pinner JR, Edwards M, Montanari F, Graziano C, Pippucci T, Dingmann B, Glass I, Mefford HC, Shimoji T, Suzuki T, Yamakawa K, Streff H, Schaaf CP, Slavotinek AM, Voineagu I, Carey JC, Buckley MF, Schenck A, Harvey RJ, Roscioli T
**Journal:** Genet Med (2022)
**DOI:** [10.1016/j.gim.2022.06.001](https://doi.org/10.1016/j.gim.2022.06.001)

## Abstract

1. Genet Med. 2022 Sep;24(9):1952-1966. doi: 10.1016/j.gim.2022.06.001. Epub 2022
 Aug 1.

De Novo ZMYND8 variants result in an autosomal dominant neurodevelopmental 
disorder with cardiac malformations.

Dias KR(1), Carlston CM(2), Blok LER(3), De Hayr L(4), Nawaz U(5), Evans CA(6), 
Bayrak-Toydemir P(7), Htun S(8), Zhu Y(9), Ma A(10), Lynch SA(11), Moorwood 
C(12), Stals K(12), Ellard S(12), Bainbridge MN(13), Friedman J(14), Pappas 
JG(15), Rabin R(15), Nowak CB(16), Douglas J(16), Wilson TE(17), Guillen Sacoto 
MJ(18), Mullegama SV(18), Palculict TB(18), Kirk EP(19), Pinner JR(20), Edwards 
M(21), Montanari F(22), Graziano C(22), Pippucci T(22), Dingmann B(23), Glass 
I(23), Mefford HC(24), Shimoji T(25), Suzuki T(26), Yamakawa K(26), Streff 
H(27), Schaaf CP(28), Slavotinek AM(8), Voineagu I(29), Carey JC(30), Buckley 
MF(9), Schenck A(3), Harvey RJ(4), Roscioli T(31).

Author information:
(1)Neuroscience Research Australia, Sydney, New South Wales, Australia; Prince 
of Wales Clinical School, Faculty of Medicine, University of New South Wales, 
Sydney, New South Wales, Australia.
(2)Division of Genetics and Genomics, Boston Children's Hospital, Harvard 
Medical School, Boston, MA.
(3)Department of Human Genetics, Donders Institute for Brain, Cognition and 
Behaviour, Radboud University Medical Center, Nijmegen, the Netherlands.
(4)School of Health and Behavioural Sciences, University of the Sunshine Coast, 
Maroochydore, Queensland, Australia; Sunshine Coast Health Institute, Birtinya, 
Queensland, Australia.
(5)Adelaide Medical School and Robinson Research Institute, The University of 
Adelaide, Adelaide, South Australia, Australia.
(6)Neuroscience Research Australia, Sydney, New South Wales, Australia; New 
South Wales Health Pathology, Randwick Genomics Laboratory, Prince of Wales 
Hospital, Sydney, New South Wales, Australia.
(7)Department of Pathology, School of Medicine, University of Utah, Salt Lake 
City, UT; Molecular Genomics, ARUP Laboratories, University of Utah School of 
Medicine, University of Utah, Salt Lake City, UT.
(8)Division of Medical Genetics, Department of Pediatrics, University of 
California San Francisco, San Francisco, CA.
(9)New South Wales Health Pathology, Randwick Genomics Laboratory, Prince of 
Wales Hospital, Sydney, New South Wales, Australia.
(10)Department of Clinical Genetics, Children's Hospital Westmead, Sydney 
Children's Hospitals Network, Sydney, New South Wales, Australia; Specialty of 
Genomic Medicine, Sydney Medical School, The University of Sydney, Sydney, New 
South Wales, Australia.
(11)Department of Clinical Genetics, Temple Street Children's University 
Hospital, Dublin, Ireland.
(12)Exeter Genomics Laboratory, Royal Devon and Exeter NHS Foundation Trust, 
Exeter, United Kingdom.
(13)Rady Children's Institute of Genomic Medicine, Rady Children's Hospital, San 
Diego, CA.
(14)Rady Children's Institute of Genomic Medicine, Rady Children's Hospital, San 
Diego, CA; Departments of Neurosciences and Pediatrics, University of California 
San Diego, San Diego, CA.
(15)Department of Pediatrics, Clinical Genetic Services, NYU Grossman School of 
Medicine, NYU Langone Health, New York, NY.
(16)The Feingold Center for Children, Boston Children's Hospital, Harvard 
Medical School, Boston, MA.
(17)Department of Medical & Molecular Genetics, Indiana University School of 
Medicine, Riley Hospital for Children at Indiana University Health, 
Indianapolis, IN.
(18)GeneDx, Gaithersburg, MD.
(19)New South Wales Health Pathology, Randwick Genomics Laboratory, Prince of 
Wales Hospital, Sydney, New South Wales, Australia; School of Women's and 
Children's Health, University of New South Wales, Sydney, New South Wales, 
Australia; Centre for Clinical Genetics, Sydney Children's Hospital, Sydney, New 
South Wales, Australia.
(20)School of Women's and Children's Health, University of New South Wales, 
Sydney, New South Wales, Australia; Centre for Clinical Genetics, Sydney 
Children's Hospital, Sydney, New South Wales, Australia.
(21)Hunter Genetics, Hunter New England Health, New Lambton, New South Wales, 
Australia; School of Medicine, Western Sydney University, Penrith, New South 
Wales, Australia.
(22)U.O. Genetica Medica, IRCCS Azienda Ospedaliero-Universitaria di Bologna, 
Bologna, Italy.
(23)Medical Genetics Department, Seattle Children's Hospital, Seattle, WA.
(24)Center for Pediatric Neurological Disease Research, St. Jude Research, St. 
Jude Children's Hospital, Memphis, TN.
(25)Department of Neurosurgery, Okinawa Pref. Nanbu Medical Center and 
Children's Medical Center, Okinawa, Japan; Okinawa Central Hospital, Okinawa, 
Japan.
(26)Department of Neurodevelopmental Disorder Genetics, Institute of Brain 
Sciences, Nagoya City University Graduate School of Medical Sciences, Nagoya, 
Japan.
(27)Department of Molecular and Human Genetics, Baylor College of Medicine, 
Houston, TX.
(28)Department of Molecular and Human Genetics, Baylor College of Medicine, 
Houston, TX; Institute of Human Genetics, Heidelberg University, Heidelberg 
University Hospital, Heidelberg, Germany.
(29)School of Biotechnology and Biomolecular Sciences, University of New South 
Wales, Sydney, New South Wales, Australia.
(30)Division of Medical Genetics, Department of Pediatrics, University of Utah 
Health, Salt Lake City, UT.
(31)Neuroscience Research Australia, Sydney, New South Wales, Australia; Prince 
of Wales Clinical School, Faculty of Medicine, University of New South Wales, 
Sydney, New South Wales, Australia; New South Wales Health Pathology, Randwick 
Genomics Laboratory, Prince of Wales Hospital, Sydney, New South Wales, 
Australia; Centre for Clinical Genetics, Sydney Children's Hospital, Sydney, New 
South Wales, Australia. Electronic address: tony.roscioli@health.nsw.gov.au.

PURPOSE: ZMYND8 encodes a multidomain protein that serves as a central 
interactive hub for coordinating critical roles in transcription regulation, 
chromatin remodeling, regulation of super-enhancers, DNA damage response and 
tumor suppression. We delineate a novel neurocognitive disorder caused by 
variants in the ZMYND8 gene.
METHODS: An international collaboration, exome sequencing, molecular modeling, 
yeast two-hybrid assays, analysis of available transcriptomic data and a 
knockdown Drosophila model were used to characterize the ZMYND8 variants.
RESULTS: ZMYND8 variants were identified in 11 unrelated individuals; 10 
occurred de novo and one suspected de novo; 2 were truncating, 9 were missense, 
of which one was recurrent. The disorder is characterized by intellectual 
disability with variable cardiovascular, ophthalmologic and minor skeletal 
anomalies. Missense variants in the PWWP domain of ZMYND8 abolish the 
interaction with Drebrin and missense variants in the MYND domain disrupt the 
interaction with GATAD2A. ZMYND8 is broadly expressed across cell types in all 
brain regions and shows highest expression in the early stages of brain 
development. Neuronal knockdown of the DrosophilaZMYND8 ortholog results in 
decreased habituation learning, consistent with a role in cognitive function.
CONCLUSION: We present genomic and functional evidence for disruption of ZMYND8 
as a novel etiology of syndromic intellectual disability.

Copyright Â© 2022 The Authors. Published by Elsevier Inc. All rights reserved.

DOI: 10.1016/j.gim.2022.06.001
PMID: 35916866 [Indexed for MEDLINE]

Conflict of interest statement: Conflict of Interest S.V.M., T.B.P., and 
M.J.G.S. are employees of GeneDx, Inc. All other authors declare no conflicts of 
interest.
