---
pmid: '26337392'
title: MDM1 is a microtubule-binding protein that negatively regulates centriole duplication.
authors:
- Van de Mark D
- Kong D
- Loncarek J
- Stearns T
journal: Mol Biol Cell
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4626064
doi: 10.1091/mbc.E15-04-0235
---

# MDM1 is a microtubule-binding protein that negatively regulates centriole duplication.
**Authors:** Van de Mark D, Kong D, Loncarek J, Stearns T
**Journal:** Mol Biol Cell (2015)
**DOI:** [10.1091/mbc.E15-04-0235](https://doi.org/10.1091/mbc.E15-04-0235)
**PMC:** [PMC4626064](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4626064/)

## Abstract

1. Mol Biol Cell. 2015 Nov 1;26(21):3788-802. doi: 10.1091/mbc.E15-04-0235. Epub 
2015 Sep 2.

MDM1 is a microtubule-binding protein that negatively regulates centriole 
duplication.

Van de Mark D(1), Kong D(2), Loncarek J(2), Stearns T(3).

Author information:
(1)Department of Biology, Stanford University, Stanford, CA 94305.
(2)Laboratory of Protein Dynamics and Signaling, Center for Cancer 
Research-Frederick, National Cancer Institute, National Institutes of Health, 
Frederick, MD 21702.
(3)Department of Biology, Stanford University, Stanford, CA 94305 Department of 
Genetics, Stanford University School of Medicine, Stanford University, Stanford, 
CA 94305 stearns@stanford.eu.

Mouse double-minute 1 (Mdm1) was originally identified as a gene amplified in 
transformed mouse cells and more recently as being highly up-regulated during 
differentiation of multiciliated epithelial cells, a specialized cell type 
having hundreds of centrioles and motile cilia. Here we show that the MDM1 
protein localizes to centrioles of dividing cells and differentiating 
multiciliated cells. 3D-SIM microscopy showed that MDM1 is closely associated 
with the centriole barrel, likely residing in the centriole lumen. 
Overexpression of MDM1 suppressed centriole duplication, whereas depletion of 
MDM1 resulted in an increase in granular material that likely represents early 
intermediates in centriole formation. We show that MDM1 binds microtubules in 
vivo and in vitro. We identified a repeat motif in MDM1 that is required for 
efficient microtubule binding and found that these repeats are also present in 
CCSAP, another microtubule-binding protein. We propose that MDM1 is a negative 
regulator of centriole duplication and that its function is mediated through 
microtubule binding.

© 2015 Van de Mark et al. This article is distributed by The American Society 
for Cell Biology under license from the author(s). Two months after publication 
it is available to the public under an Attribution–Noncommercial–Share Alike 3.0 
Unported Creative Commons License 
(http://creativecommons.org/licenses/by-nc-sa/3.0).

DOI: 10.1091/mbc.E15-04-0235
PMCID: PMC4626064
PMID: 26337392 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONCentrosomes are the main microtubule-organizing centers in animal cells and are important for cell division, cell motility, signaling, and intracellular trafficking. They are composed of two centrioles, which are associated with a matrix of pericentriolar material (PCM). The older of the two centrioles has specialized appendages at its distal end, allowing it to dock with the plasma membrane and nucleate the formation of a primary cilium in interphase. Mutations in genes important for the formation, structure, and function of centrosomes and cilia are associated with ciliopathies, a class of diseases marked by common phenotypes, including neurodevelopmental defects, polycystic kidneys, obesity, polydactyly, and retinal degeneration (Hildebrandt et al., 2011).In most cell types, centrioles duplicate once per cell cycle. Maintaining centriole number is important both for proper cell division in development (Marthiens et al., 2013; Bazzi and Anderson, 2014) and for prevention of aneuploidy and uncontrolled proliferation (Basto et al., 2008; Ganem et al., 2009; Silkworth et al., 2009; Godinho et al., 2014). We will use the convention of referring to the centrioles of a G1-phase cell as parental centrioles and newly formed centrioles as procentrioles. Procentrioles begin forming orthogonally to the parental centriole at the G1/S-phase transition, coincident with the onset of DNA replication, and elongate throughout S and G2 phases (Kuriyama and Borisy, 1981). Many proteins important for forming procentrioles have been identified, including the kinase PLK4, which is capable of initiating the formation of multiple daughter centrioles when overexpressed (Habedanck et al., 2005), and the PCM proteins CEP192 (Sonnen et al., 2013), CEP63 (Sir et al., 2011; Brown et al., 2013), and CEP152 (Cizmecioglu et al., 2010; Dzhindzhev et al., 2010; Hatch et al., 2010; Sonnen et al., 2013), which are involved in recruitment of PLK4 to the origin of duplication. Subsequent to recruitment of PLK4, the proteins SASS6 (Nakazawa et al., 2007), CEP135 (Kleylein-Sohn et al., 2007), STIL (Stevens et al., 2010), and CPAP (Kleylein-Sohn et al., 2007) initiate procentriole formation by assembling the cartwheel, a ninefold-symmetric structure that templates the formation of new microtubule barrels.Although a number of proteins and mechanisms that control initiation of centriole duplication have been identified, the mechanisms that complete the process of centriole duplication and prevent reduplication are less well understood. A well-studied example of a negative regulatory circuit is the autoregulation of the stability of PLK4 by trans-autophosphorylation, rendering it a substrate for SLIMB/b-TrCP and the Skp1-Cull-F-box (SCF) ubiquitin ligase complex (Cunha-Ferreira et al., 2009; Rogers et al., 2009; Holland et al., 2010). In addition, centriole length is negatively regulated in part by the distal end protein CP110, depletion of which leads to elongation of centrioles from the distal end (Schmidt et al., 2009; Franz et al., 2013). Other described negative regulators of centriole duplication include the origin recognition complex subunit ORC1, which prevents centriole reduplication through interactions with cyclins A and E (Hemerly et al., 2009), centriolar satellite protein CCDC14, which mediates centriolar recruitment of CEP63 (Firat-Karalar et al., 2014), nuclear paraspeckle and cytosolic protein RBM14, which is proposed to prevent interaction between CPAP and STIL (Shiratsuchi et al., 2015), and the CP110 interaction partners NEURL4 (Li et al., 2012), which is believed to regulate duplication by promoting ubiquitylation of CP110, and CEP76 (Tsang et al., 2009). A recent genetic screen in mammalian cells identified the ubiquitin ligase TRIM37 as a negative regulator of centriole formation, although targets of this protein are not known (Balestra et al., 2013). In contrast to the paradigm of positive regulation of centriole duplication, where proteins are hierarchically recruited to the origin of duplication and form observable structures, negative regulation of centriole duplication appears to affect duplication by a variety of mechanisms.We identified a previously uncharacterized protein, Mouse double minute 1 (MDM1), as being highly transcriptionally up-regulated during the formation of centrioles and cilia in multiciliated epithelial cells (MCCs) derived from cultures of mouse tracheal epithelial cells (MTECs; Hoh et al., 2012). Green fluorescent protein (GFP)–tagged MDM1 localized to the centrosome in cultured mouse cells. Mdm1 was originally identified in a transformed mouse 3T3 cell line as a gene present on small, amplified chromosome fragments termed double minutes (Cahilly-Snyder et al., 1987; Snyder et al., 1988). The well-studied oncogene Mdm2 was also identified in this way (Cahilly-Snyder et al., 1987) and is near Mdm1 in the mouse and human genomes; the two proteins share no homology despite their common name. In addition, an Mdm1 nonsense mutation was suggested to be the causal mutation underlying an age-related retinal degeneration phenotype of a wild mouse strain (the Arrd2 mouse; Chang et al., 2008).Here we report that MDM1 is a microtubule-binding protein that localizes to centrioles, MDM1 overexpression results in stabilization of microtubules and reduced centriole number, and depletion of MDM1 results in an increase in granular material that likely represents early intermediates in centriole formation. These results suggest that MDM1 is a negative regulator of centriole duplication and that its action might be mediated through direct contact with microtubules of the centriole barrel.

INTRODUCTION

Centrosomes are the main microtubule-organizing centers in animal cells and are important for cell division, cell motility, signaling, and intracellular trafficking. They are composed of two centrioles, which are associated with a matrix of pericentriolar material (PCM). The older of the two centrioles has specialized appendages at its distal end, allowing it to dock with the plasma membrane and nucleate the formation of a primary cilium in interphase. Mutations in genes important for the formation, structure, and function of centrosomes and cilia are associated with ciliopathies, a class of diseases marked by common phenotypes, including neurodevelopmental defects, polycystic kidneys, obesity, polydactyly, and retinal degeneration (Hildebrandt et al., 2011).

In most cell types, centrioles duplicate once per cell cycle. Maintaining centriole number is important both for proper cell division in development (Marthiens et al., 2013; Bazzi and Anderson, 2014) and for prevention of aneuploidy and uncontrolled proliferation (Basto et al., 2008; Ganem et al., 2009; Silkworth et al., 2009; Godinho et al., 2014). We will use the convention of referring to the centrioles of a G1-phase cell as parental centrioles and newly formed centrioles as procentrioles. Procentrioles begin forming orthogonally to the parental centriole at the G1/S-phase transition, coincident with the onset of DNA replication, and elongate throughout S and G2 phases (Kuriyama and Borisy, 1981). Many proteins important for forming procentrioles have been identified, including the kinase PLK4, which is capable of initiating the formation of multiple daughter centrioles when overexpressed (Habedanck et al., 2005), and the PCM proteins CEP192 (Sonnen et al., 2013), CEP63 (Sir et al., 2011; Brown et al., 2013), and CEP152 (Cizmecioglu et al., 2010; Dzhindzhev et al., 2010; Hatch et al., 2010; Sonnen et al., 2013), which are involved in recruitment of PLK4 to the origin of duplication. Subsequent to recruitment of PLK4, the proteins SASS6 (Nakazawa et al., 2007), CEP135 (Kleylein-Sohn et al., 2007), STIL (Stevens et al., 2010), and CPAP (Kleylein-Sohn et al., 2007) initiate procentriole formation by assembling the cartwheel, a ninefold-symmetric structure that templates the formation of new microtubule barrels.

Although a number of proteins and mechanisms that control initiation of centriole duplication have been identified, the mechanisms that complete the process of centriole duplication and prevent reduplication are less well understood. A well-studied example of a negative regulatory circuit is the autoregulation of the stability of PLK4 by trans-autophosphorylation, rendering it a substrate for SLIMB/b-TrCP and the Skp1-Cull-F-box (SCF) ubiquitin ligase complex (Cunha-Ferreira et al., 2009; Rogers et al., 2009; Holland et al., 2010). In addition, centriole length is negatively regulated in part by the distal end protein CP110, depletion of which leads to elongation of centrioles from the distal end (Schmidt et al., 2009; Franz et al., 2013). Other described negative regulators of centriole duplication include the origin recognition complex subunit ORC1, which prevents centriole reduplication through interactions with cyclins A and E (Hemerly et al., 2009), centriolar satellite protein CCDC14, which mediates centriolar recruitment of CEP63 (Firat-Karalar et al., 2014), nuclear paraspeckle and cytosolic protein RBM14, which is proposed to prevent interaction between CPAP and STIL (Shiratsuchi et al., 2015), and the CP110 interaction partners NEURL4 (Li et al., 2012), which is believed to regulate duplication by promoting ubiquitylation of CP110, and CEP76 (Tsang et al., 2009). A recent genetic screen in mammalian cells identified the ubiquitin ligase TRIM37 as a negative regulator of centriole formation, although targets of this protein are not known (Balestra et al., 2013). In contrast to the paradigm of positive regulation of centriole duplication, where proteins are hierarchically recruited to the origin of duplication and form observable structures, negative regulation of centriole duplication appears to affect duplication by a variety of mechanisms.

We identified a previously uncharacterized protein, Mouse double minute 1 (MDM1), as being highly transcriptionally up-regulated during the formation of centrioles and cilia in multiciliated epithelial cells (MCCs) derived from cultures of mouse tracheal epithelial cells (MTECs; Hoh et al., 2012). Green fluorescent protein (GFP)–tagged MDM1 localized to the centrosome in cultured mouse cells. Mdm1 was originally identified in a transformed mouse 3T3 cell line as a gene present on small, amplified chromosome fragments termed double minutes (Cahilly-Snyder et al., 1987; Snyder et al., 1988). The well-studied oncogene Mdm2 was also identified in this way (Cahilly-Snyder et al., 1987) and is near Mdm1 in the mouse and human genomes; the two proteins share no homology despite their common name. In addition, an Mdm1 nonsense mutation was suggested to be the causal mutation underlying an age-related retinal degeneration phenotype of a wild mouse strain (the Arrd2 mouse; Chang et al., 2008).

Here we report that MDM1 is a microtubule-binding protein that localizes to centrioles, MDM1 overexpression results in stabilization of microtubules and reduced centriole number, and depletion of MDM1 results in an increase in granular material that likely represents early intermediates in centriole formation. These results suggest that MDM1 is a negative regulator of centriole duplication and that its action might be mediated through direct contact with microtubules of the centriole barrel.

RESULTSMDM1 localizes to the lumen of parental centriolesWe previously identified MDM1 as a centrosome component based on greatly elevated expression during ciliogenesis in mouse MCCs and colocalization of MDM1-GFP with γ-tubulin at the centrosome in cultured cells (Hoh et al., 2012). To confirm this localization for the endogenous protein, we used two anti-MDM1 polyclonal antibodies that recognize the N- or C-terminus of the human MDM1 protein (Supplemental Figure S1). We were unable to visualize endogenous MDM1 by Western blot (Supplemental Figure S1, A and B), consistent with a previous report of low-level expression (Snyder et al., 1988). Both antibodies showed that MDM1 localized to centrosomes throughout the cell cycle in hTERT-immortalized human RPE-1 cells (henceforth, RPE-1), appearing as two foci per cell in G1 through G2 phases (Figure 1A) and four foci per cell beginning in prophase of mitosis (Figure 1C). The antibodies were specific for MDM1 in human cells, as shown by loss of centrosomal MDM1 signal upon MDM1 depletion (e.g., see later discussion of Figure 6, A and B, and Supplemental Figure S4, A and B). This suggests that MDM1 is associated with centrioles and only localizes to the procentrioles at the onset of mitosis, when centriole duplication is complete and centrosomes undergo several maturation events. In addition, MDM1 remained associated with centrioles of RPE-1 cells that were arrested in G0 phase by serum withdrawal and had formed a primary cilium (Figure 1B). We also examined whether MDM1 localizes to centrioles in MCCs derived from tracheal epithelium. MTEC cultures were fixed 4 d after induction of differentiation by shifting cells to air–liquid interface culture (ALI+4). We determined the localization of MDM1 with respect to CEP164, a marker of fully formed centrioles. Both CEP164 and MDM1 antibodies labeled foci in the apical domain of MCCs, consistent with localization to centrioles (Figure 1D).FIGURE 1:MDM1 localizes to centrioles. (A) Cycling RPE-1 cells were fixed and stained for MDM1 (green), CETN3 (red), and DNA (DAPI, blue). Right, magnified images showing the centrioles of a cell in G1 phase (two CETN3 foci); left, a cell in S/G2 (four CETN3 foci). (B) RPE-1 cells were serum starved, fixed, and stained for MDM1 (green), polyglutamylated tubulin (pg-tub, red), and DNA (blue). (C) Mitotic RPE-1 cells at indicated mitotic stages stained for MDM1 (green), CETN (red), and DNA (blue). Each inset corresponds to one spindle pole. All images in A–C were acquired by wide-field epifluorescence microscopy, and all insets represent 5× magnifications. Scale bars, 5 μm. (D) ALI+4 MTECs stained with antibodies against MDM1 (green) and CEP164 (red). Square images are maximum intensity projections of stacks acquired by spinning-disk confocal microscopy, and rectangles below them are xz-projections of a region within the boxed area in the merged maximum intensity projections.The localization of MDM1 at centrioles was further defined in RPE-1 cells using three-dimensional structured illumination microscopy (3D-SIM). MDM1 localization was determined in reference to markers of the centriole distal appendages (CEP164), distal lumen (CETN3), proximal end (C-NAP1), and PCM (CEP152) (Figure 2A). MDM1 localized to the mid-proximal region of centrioles between the distal markers CETN3 and CEP164 and the proximal-end marker C-NAP1. The MDM1 signal was associated with each parental centriole and frequently observed to be bilobed in appearance. Whereas distal appendage and PCM proteins form rings resolvable by 3D-SIM, MDM1 localized in the center of the CEP164 and CEP152 rings, suggesting that MDM1 is in the lumen of the centriole. Consistent with this interpretation, MDM1 localized in the center of the centriole, as visualized by labeling polyglutamylated tubulin, a marker for the modified tubulin in the centriole barrels (Figure 2B). These results suggest that MDM1 resides in the centriole lumen between centrin and C-NAP1 (Figure 2C). Furthermore, in cases in which the bilobed appearance was observed, all MDM1 signal colocalized with the polyglutamylated tubulin signal of the parent centrioles and not to SASS6 foci (Figure 2B). Thus, as with wide-field imaging, 3D-SIM imaging showed that MDM1 only associated with the two parental centrioles in postduplication interphase cells (Figure 2, A–C).FIGURE 2:MDM1 localizes to the centriole lumen. (A, B, D, E) Representative 3D-SIM micrographs. (A) RPE-1 cells were fixed and stained for MDM1 (green) and markers of the centriole distal lumen (CETN3), the PCM­­ (CEP152), the centriole distal appendages (CEP164), and the centriole proximal end (C-NAP1; all comarkers in red). (B) RPE-1 cells fixed and colabeled for MDM1 (green), polyglutamylated tubulin to mark the centriole barrels (pg-tub, red), and SASS6 to mark the procentriole cartwheels (blue). All images in A and B are maximum intensity projections. Scale bars, 1 μm. (C) Schematic depicting localization of MDM1 inside the centriole lumen. (D, E) 3D-SIM micrographs of MTEC cultures fixed at (D) ALI+6 or (E) ALI+20. (D) ALI+6 MTECs costained with antibodies against MDM1 (green) and CEP164 (red, marking distal appendages). Images are maximum intensity projections. Left, boxed regions magnified 2.7×. (E) ALI+20 MTECs fixed and stained with antibodies against MDM1 (green) and glu-tubulin (red) to visualize cilia. Images are maximum intensity projections. Left and right, boxed regions magnified 2.7×. Scale bars, 5 μm (D, E).Examination of ALI+6 MTEC cells by 3D-SIM revealed that MDM1 localized to centrioles in MCCs (Figure 2D). MTECs were labeled for MDM1 and CEP164 (Figure 2D). A distinct MDM1 focus was observed inside of each CEP164 ring, in agreement with the results in RPE-1 cells (Figure 2A). In late-stage ALI+20 MTECs, MDM1 staining was dim and diffuse in most ciliated cells (Figure 2E), only having the defined centriole labeling typical of early-stage cells in the few newly differentiating cells present at this stage. We conclude that MDM1 localizes to MCC centrioles during MCC differentiation but declines in abundance at centrioles in fully mature MCCs.MDM1 is a microtubule-binding proteinGiven the original identification of MDM1 on amplified genome fragments in cancer cells, we examined the effects of overexpression of MDM1. GFP-MDM1 expressed by transient transfection in RPE-1 cells localized to the centrosome as well as the nucleus (Supplemental Figure S2A), consistent with previous reports (Snyder et al., 1988; Hoh et al., 2012). In addition to these two expected localizations, GFP-MDM1 localized to cytoplasmic microtubules in highly expressing cells (Figure 3A). In some cells, GFP-MDM1 localized with filaments that resembled bundled microtubules. These filaments were associated with the centrosome and stained for α-tubulin and acetylated α-tubulin (Figure 3B and Supplemental Figure S2B) but did not stain for other markers of the centriole or ciliary axoneme, suggesting they are not hyperelongated centrioles or normal cilia (Figure 3B and Supplemental Figure S2B).FIGURE 3:MDM1 is a microtubule-binding protein. (A) RPE-1 cells transfected with GFP-MDM1 were fixed and stained for GFP (green), α-tubulin (red), and DNA (DAPI, blue). Image is a maximum intensity projection of stacks acquired by spinning-disk confocal microscopy. Scale bar, 10 μm. (B) RPE-1 cells expressing GFP-MDM1 were fixed and stained for GFP in green and (top) CETN3 or (bottom) α-tubulin in red. Images were taken by wide-field epifluorescence microscopy. Scale bar, 5 μm. (C) Domain structure of human MDM1 isoform 1 (Uniprot ID Q8TC05-1, NCBI NP_059136.2). (D) Phylogenetic tree showing conservation of MDM1. (E) Clustal Omega alignment showing the high degree of conservation of the MDM1 (S/T)EYxxx(F/Y) repeat sequences in the same species shown in the phylogenetic tree. (F) Schematic of MDM1 domains required for microtubule colocalization in RPE-1 cells. MDM1rm corresponds to a mutant version of MDM1 in which all four (S/T)EYxxxF repeat motifs have been mutated to AAAxxxA by site-directed mutagenesis. All other constructs are indicated by amino acid positions included within the construct. Colocalization was assessed by expressing GFP-fusion proteins in RPE-1 cells, treating with Taxol to bundle microtubules, staining for GFP and α-tubulin, and scoring the percentage of high-expressing transfected cells with GFP-microtubule colocalization in three trials, 100 cells/condition per trial (see Supplemental Figure S2, C and D, for data). Percentage of cells showing colocalization in all three trials indicated as follows: +++, >90%; ++, >70%; +, >5%, –, <5%. (G) Microtubule cosedimentation assay using purified GST-tagged recombinant protein and Taxol-stabilized microtubules. Indicated proteins were incubated with Taxol-stabilized microtubules or buffer as indicated, loaded onto a 40% glycerol cushion, and spun down (55,000 rpm [∼250,000 × g] for 10 min). Input (I), supernatant (S), and pellet (P) fractions were run out on an SDS–PAGE gel, and proteins were detected by Western blot using antibody against GST.Other microtubule-binding proteins, such as MAP2 and TAU, have repeated motifs that are directly involved in their microtubule binding (reviewed in Dehmelt and Halpain, 2005). We found that MDM1 lacks any known microtubule-binding motifs but does have a repeated sequence, present in four copies in the N-terminal half, consisting of the motif (S/T)EYxxxF (Figure 3C). If these repeats were important for MDM1 function, we would expect them to be conserved in evolution. MDM1 itself is present in multicellular organisms with ciliated cells, ranging from the placozoan Trichoplax adhaerens to human, although, of note, it is absent in Caenorhabditis elegans (Figure 3D). The repeats were found in all MDM1 orthologues examined (Figure 3E), with the protein in most species having at least two copies of the repeat, in some cases with a tyrosine (Y) residue in place of the phenylalanine (F). We also found two of the (S/T)EYxxx(F/Y) repeats in centriole, cilia, and spindle–associated protein (CCSAP), a microtubule-binding protein that localizes to centrioles and cilia (Backer et al., 2012).To test whe­­ther these repeats are important for microtubule binding, we constructed a mutant version of MDM1 that had the conserved residues in all four repeats replaced by alanine (4x (S/T)EYxxxF to AAAxxxA, hereafter called MDM1rm), as well as truncations with different numbers of the repeats. GFP fusions of these constructs were expressed in RPE-1 cells that had been treated with the microtubule-stabilizing drug Taxol before fixation, creating large bundles of microtubules to enhance visualization of colocalization (Schiff and Horwitz, 1980; Figure 3F and Supplemental Figure S2, C and D). GFP-MDM1 colocalized with microtubules in 97.0 ± 3.00% of cells. In contrast, the GFP-MDM1rm fusion was expressed at the same level as MDM1 (Supplemental Figure S2E) but colocalized with microtubules in only 8.67 ± 1.20% of cells and even in positive cells was less enriched on microtubules than in wild type (wt). Although MDM1rm did not associate with microtubules, it was able to localize to the centrosome (see later discussion of Figure 5C). Constructs that had at least two of the repeats localized to microtubules, whereas the single-repeat-containing construct and two C-terminal fragments that lack all repeats did not.To test whether the association of MDM1 with microtubules reflects direct binding, we purified recombinant glutathione S-transferase (GST)–tagged MDM1 and used in in vitro microtubule cosedimentation assays (Figure 3G). GST-tagged fusions were prepared for full- length MDM1, MDM1rm, MDM11-416 (contains all four repeats), and MDM1490-714 (lacks all four repeats). GST-MDM1, but not GST alone, pelleted with microtubules, indicating a direct interaction of MDM1 with microtubules. Consistent with the localization results, GST-MDM11–416 also pelleted with microtubules. Of interest, GST-MDM1rm also pelleted with microtubules, although about half of the protein remained in the supernatant, whereas the GST-MDM1490–713 protein remained primarily in the supernatant. Together the localization and binding results show that MDM1 is a microtubule-binding protein and that the conserved repeat elements of MDM1 are important for full binding activity.MDM1 stabilizes microtubulesGiven that MDM1 interacts directly with microtubules in vitro and is closely associated with the stable centriole microtubules, we considered the possibility that MDM1 binding stabilizes micro­tubules. To test this, we expressed GFP-MDM1 in RPE-1 cells, followed by treatment with the microtubule-depolymerizing drug nocodazole and/or incubation of the cells at 4ºC (Figure 4, A and B). Treatment with 5 μg/ml nocodazole, 4ºC incubation, or both treatments for 1–2 h was sufficient to depolymerize micro­tubules in nearly all control GFP-transfected cells (Figure 4B). In contrast, many GFP-MDM1–transfected cells retained some polymerized microtubules: 46.7 ± 5.84% of cells in nocodazole, 76.7 ± 5.86% of cells in the 4ºC incubation, and 74.0 ± 14.0% of cells in nocodazole plus 4ºC incubation. Consistent with the localization and binding results, GFP-MDM1rm had little effect on microtubule stability in any of the treatments (Figure 4, A and B). Thus MDM1 binds and stabilizes microtubules, and the repeat elements are important for both properties.FIGURE 4:MDM1 stabilizes microtubules. (A) RPE-1 cells expressing GFP, GFP-MDM1, or GFP-MDM1rm were incubated with either 5 μg/ml nocodazole or the equivalent volume of DMSO (vehicle control) for 1–2 h at either 37 or 4ºC. Cells were fixed immediately after and stained for GFP (green), α-tubulin (red), and DNA (DAPI, blue) to determine how many transfected cells had polymerized microtubules intact. Representative images of the nocodazole treatment at 37ºC. All images are maximum intensity projections of stacks acquired by spinning-disk confocal microscopy. Scale bar, 10 μm. (B) Quantitation of phenotypes observed in A across three trials, mean ± SEM (100 transfected cells/construct were counted for each condition in each trial).MDM1 overexpression suppresses centriole duplicationTwo other microtubule-binding proteins that localize to the centriole lumen, CPAP (Hsu et al., 2008) and CEP135 (Carvalho-Santos et al., 2012), are important for centriole duplication. We tested whether MDM1 might also be involved in centriole duplication. First, we overexpressed GFP-MDM1 in RPE-1 cells and counted centrioles by centrin staining (Figure 5A). At 24 h posttransfection, there was no significant difference in the number of centrioles per cell on comparing GFP-expressing and GFP-MDM1–expressing cells, with most cells having two centrioles (G1-phase cells) or four centrioles (S/G2/mitotic cells; p = 0.148, Fisher’s exact test; Supplemental Figure S3A). At 48 h posttransfection, however, GFP-MDM1–expressing cells had fewer centrioles (Figure 5A, p = 2.20 × 10−16, Fisher’s exact test), apparent as a decrease in the number of cells with four centrioles and an increase in those with zero or one centriole(s) per cell. A similar result was observed using γ-tubulin as the centrosome marker and quantifying the percentage of cells with fewer than two γ-tubulin foci (p = 0.0146 at 24 h posttransfection and p = 2.30 × 10−15 at 48 h posttransfection, Fisher’s exact test, Supplemental Figure S3, B and C).We next tested whether overexpression of MDM1 is capable of blocking centriole duplication in cells that reduplicate centrioles during S-phase arrest (Balczon et al., 1995) (Figure 5, B and C). GFP-MDM1 significantly suppressed reduplication compared with controls (p = 0.00403 compared with GFP-expressing control cells, two-tailed unpaired Student’s t test), whereas neither GFP nor GFP fused to the pericentrin centrosomal targeting domain (Gillingham and Munro, 2000) had an effect on reduplication. Of importance, GFP-MDM1rm did not significantly block centriole reduplication (Figure 5, B and C, p = 0.729 compared with GFP-expressing control cells, two-tailed unpaired Student’s t test), although it was expressed at the same level as GFP-MDM1 (Figure 5D). In agreement with this, we also found that GFP-MDM1rm was less effective at blocking normal centriole duplication in cycling RPE-1 cells than was GFP-MDM1 (Supplemental Figure S3D). These results demonstrate that the microtubule-binding and -stabilizing properties of MDM1 mediate the suppression of centriole duplication caused by MDM1 overexpression and suggest that the normal function of MDM1 is to negatively regulate centriole duplication.FIGURE 5:Transient MDM1 overexpression inhibits centriole duplication. (A) RPE-1 cells were transfected with GFP or GFP-MDM1, fixed 48 h later, and stained for GFP and CETN3. The number of CETN3 foci per transfected cell was scored for each condition (three trials, 200 cells/trial for each condition). Average percentage of transfected cells with indicated number of CETN3 foci ± SEM is shown. Statistical significance was assessed using a Fisher’s exact test in R (p = 2.20 × 10−16) with the summed data for all three trials. (B, C) U2OS cells were transfected with the indicated constructs and then arrested in S phase with the addition of 4 mM HU for 48 h, fixed, and stained for GFP and CETN3. The number of cells with more than four CETN3 foci (representing cells with reduplicated centrioles) in transfected cells was scored for each condition (three trials, 100 cells/trial for each condition). Average percentage of cells with more than four CETN3 foci ±SEM. The p values were computed using a two-tailed unpaired Student’s t test, comparing the means in cells overexpressing the indicated construct to the mean of cells overexpressing GFP alone (**p < 0.01). (C) Representative images from the U2OS reduplication assays for control conditions, GFP-MDM1 expression, and GFP-MDM1rm expression. Cells were stained with GFP (green) and CETN3 (red). Images were acquired by wide-field epifluorescence microscopy. Scale bar, 1 μm. (D) Lysates from U2OS cells transfected with the indicated constructs were run on an SDS–PAGE gel for Western blot to show relative expression levels of constructs used in the U2OS reduplication assay. Blots were probed with antibody against GFP and α-tubulin (loading control).MDM1 depletion results in stimulation of the centriole duplication pathwayIf MDM1 were a negative regulator of centriole duplication, we might expect depletion of MDM1 to result in stimulation of the centriole duplication pathway, as described for other putative negative regulators (Cunha-Ferreira et al., 2009; Hemerly et al., 2009; Rogers et al., 2009; Tsang et al., 2009; Balestra et al., 2013; Firat-Karalar et al., 2014). To test this, we depleted MDM1 from RPE-1 cells by lentivirus-mediated short hairpin RNA (shRNA) treatment. MDM1 depletion was confirmed by immunofluorescence (Figure 6, A and B, and Supplemental Figure S4, A and B). Depletion of MDM1 did not affect cell cycle progression as assessed by flow cytometry (Supplemental Figure S4, C and D). The effect of MDM1 depletion on centrosome number was assessed by staining for centriole and centrosome markers (Figure 6, C and D). There was no difference in the number of centrosomes per cell, as assayed by γ-tubulin staining (Supplemental Figure S5B), and no increase in the centrosomal abundance of the positive regulators PLK4, CEP63, CEP152, and SASS6 (Supplemental Figure S6). However, there was a statistically significant enrichment of cells with more than four centrin foci in depleted cells (4.38 ± 0.800% in control cells and 10.9 ± 1.46% in depleted cells; p = 0.00801, unpaired two-tailed Student’s t test; Figure 6, C and D); this phenotype was rescued by expression of an shRNA-resistant version of MDM1 (Supplemental Figure S5A). Similar results were observed with CPAP (Supplemental Figure S5, C and D, 2.67 ± 0.882% in controls vs. 8.00 ± 1.53% in depleted cells; p = 0.0390, unpaired two-tailed Student’s t test) but not with CP110 (Supplemental Figure S5E), suggesting that the foci formed in MDM1-depleted cells are not mature centrioles.FIGURE 6:MDM1 depletion stimulates the formation of supernumerary centrin foci. (A) RPE-1 cells transduced with empty vector carrying control lentivirus or MDM1 shRNA were fixed 6 d after start of transduction and stained for MDM1 (green; Abnova mouse anti-MDM1 antibody) and CEP164 (red) to mark the centrosome region. Images are maximum intensity projections of stacks acquired by spinning-disk confocal microscopy with identical gain, laser intensity, and exposure settings for both samples. Brightness and contrast of original images are unchanged. Scale, 2 μm. (B) Effectiveness of MDM1 depletion was measured by loss of fluorescence intensity at the centrosome using mouse anti-MDM1 (Abnova). Centrosome regions were defined by CEP164 labeling as in A. Boxplots depict centrosomal MDM1 intensities of all control and depleted cells measured (three trials, 100 cells/trial for each condition). For each trial, normalized mean MDM1 intensities were computed for control and depleted cells. The average normalized mean MDM1 intensity in depleted cells was 0.290 that of control cells (±0.0711, SEM; p = 6.56 × 10−5, unpaired two-tailed Student’s t test). (C) RPE-1 cells transduced with empty vector or MDM1 shRNA were fixed 6 d after start of transduction and stained for CETN3 (green) and γ-tubulin (red) to visualize centrioles and centrosomes, respectively. Scale, 10 μm. (D) The percentage of control and MDM1-depleted cells with more than four CETN3 foci 6 d after start of transduction was quantified (four trials, 200 cells/trial for each sample). Average percentage of cells with more than four CETN3 foci (±SEM) for each sample (control cells, 4.38 ± 0.800%; shRNA-transduced cells, 10.9 ± 1.46%; p = 0.00801, unpaired two-tailed Student’s t test).To determine the nature of the foci in MDM1-depleted cells, we performed correlative light and electron microscopy (CLEM). MDM1 was depleted in RPE-1 cells expressing centrin1-GFP, and cells with multiple centrin1-GFP foci were identified by live-cell fluorescence microscopy. The coverslips were then fixed, and the same cells were analyzed by transmission electron microscopy (TEM). These cells usually had four centrioles, indicating that they were in S/G2 (five of six cells with more than four centrin foci were observed to have procentrioles in EM micrographs). The centrin foci were visible as darkly staining aggregates lacking microtubule structure characteristic of bona fide centrioles (Figure 7, n = 6). These aggregates had an average diameter of 70 ± 20 nm (SD), similar in size to structures that have previously been observed as early indicators of de novo centriole formation (Prosser et al., 2009) and to the fibrous granules found preceding centriole amplification in multicilated epithelial cells (Sorokin, 1968; Steinman, 1968; Anderson and Brenner, 1971; Dirksen, 1971). Similar structures were reported in CLEM analysis of GFP-CPAP foci observed upon depletion of another negative regulator of centriole duplication, RBM14 (Shiratsuchi et al., 2015).FIGURE 7:Supernumerary centrin foci resemble granular material by CLEM. (A–C) CLEM of CETN1-GFP RPE-1 cells transduced with MDM1 shRNA lentivirus. (A) Maximum intensity projection of an MDM1-depleted CETN1-GFP RPE-1 cell with more than four centrin-GFP foci imaged by spinning-disk confocal microscopy. Scale, 5 μm. (B) TEM micrographs of serial sections of the same cell (which has four centrioles, and split centrosomes). Scale, 2 μm. (C) Magnified insets of the previous series that focus on each centrosome and surrounding granular material. Scale bar, 500 nm. Alike arrowheads point to CETN1-GFP material that roughly colocalizes with dark-staining granules in EM micrographs throughout all images.

MDM1 localizes to the lumen of parental centriolesWe previously identified MDM1 as a centrosome component based on greatly elevated expression during ciliogenesis in mouse MCCs and colocalization of MDM1-GFP with γ-tubulin at the centrosome in cultured cells (Hoh et al., 2012). To confirm this localization for the endogenous protein, we used two anti-MDM1 polyclonal antibodies that recognize the N- or C-terminus of the human MDM1 protein (Supplemental Figure S1). We were unable to visualize endogenous MDM1 by Western blot (Supplemental Figure S1, A and B), consistent with a previous report of low-level expression (Snyder et al., 1988). Both antibodies showed that MDM1 localized to centrosomes throughout the cell cycle in hTERT-immortalized human RPE-1 cells (henceforth, RPE-1), appearing as two foci per cell in G1 through G2 phases (Figure 1A) and four foci per cell beginning in prophase of mitosis (Figure 1C). The antibodies were specific for MDM1 in human cells, as shown by loss of centrosomal MDM1 signal upon MDM1 depletion (e.g., see later discussion of Figure 6, A and B, and Supplemental Figure S4, A and B). This suggests that MDM1 is associated with centrioles and only localizes to the procentrioles at the onset of mitosis, when centriole duplication is complete and centrosomes undergo several maturation events. In addition, MDM1 remained associated with centrioles of RPE-1 cells that were arrested in G0 phase by serum withdrawal and had formed a primary cilium (Figure 1B). We also examined whether MDM1 localizes to centrioles in MCCs derived from tracheal epithelium. MTEC cultures were fixed 4 d after induction of differentiation by shifting cells to air–liquid interface culture (ALI+4). We determined the localization of MDM1 with respect to CEP164, a marker of fully formed centrioles. Both CEP164 and MDM1 antibodies labeled foci in the apical domain of MCCs, consistent with localization to centrioles (Figure 1D).FIGURE 1:MDM1 localizes to centrioles. (A) Cycling RPE-1 cells were fixed and stained for MDM1 (green), CETN3 (red), and DNA (DAPI, blue). Right, magnified images showing the centrioles of a cell in G1 phase (two CETN3 foci); left, a cell in S/G2 (four CETN3 foci). (B) RPE-1 cells were serum starved, fixed, and stained for MDM1 (green), polyglutamylated tubulin (pg-tub, red), and DNA (blue). (C) Mitotic RPE-1 cells at indicated mitotic stages stained for MDM1 (green), CETN (red), and DNA (blue). Each inset corresponds to one spindle pole. All images in A–C were acquired by wide-field epifluorescence microscopy, and all insets represent 5× magnifications. Scale bars, 5 μm. (D) ALI+4 MTECs stained with antibodies against MDM1 (green) and CEP164 (red). Square images are maximum intensity projections of stacks acquired by spinning-disk confocal microscopy, and rectangles below them are xz-projections of a region within the boxed area in the merged maximum intensity projections.The localization of MDM1 at centrioles was further defined in RPE-1 cells using three-dimensional structured illumination microscopy (3D-SIM). MDM1 localization was determined in reference to markers of the centriole distal appendages (CEP164), distal lumen (CETN3), proximal end (C-NAP1), and PCM (CEP152) (Figure 2A). MDM1 localized to the mid-proximal region of centrioles between the distal markers CETN3 and CEP164 and the proximal-end marker C-NAP1. The MDM1 signal was associated with each parental centriole and frequently observed to be bilobed in appearance. Whereas distal appendage and PCM proteins form rings resolvable by 3D-SIM, MDM1 localized in the center of the CEP164 and CEP152 rings, suggesting that MDM1 is in the lumen of the centriole. Consistent with this interpretation, MDM1 localized in the center of the centriole, as visualized by labeling polyglutamylated tubulin, a marker for the modified tubulin in the centriole barrels (Figure 2B). These results suggest that MDM1 resides in the centriole lumen between centrin and C-NAP1 (Figure 2C). Furthermore, in cases in which the bilobed appearance was observed, all MDM1 signal colocalized with the polyglutamylated tubulin signal of the parent centrioles and not to SASS6 foci (Figure 2B). Thus, as with wide-field imaging, 3D-SIM imaging showed that MDM1 only associated with the two parental centrioles in postduplication interphase cells (Figure 2, A–C).FIGURE 2:MDM1 localizes to the centriole lumen. (A, B, D, E) Representative 3D-SIM micrographs. (A) RPE-1 cells were fixed and stained for MDM1 (green) and markers of the centriole distal lumen (CETN3), the PCM­­ (CEP152), the centriole distal appendages (CEP164), and the centriole proximal end (C-NAP1; all comarkers in red). (B) RPE-1 cells fixed and colabeled for MDM1 (green), polyglutamylated tubulin to mark the centriole barrels (pg-tub, red), and SASS6 to mark the procentriole cartwheels (blue). All images in A and B are maximum intensity projections. Scale bars, 1 μm. (C) Schematic depicting localization of MDM1 inside the centriole lumen. (D, E) 3D-SIM micrographs of MTEC cultures fixed at (D) ALI+6 or (E) ALI+20. (D) ALI+6 MTECs costained with antibodies against MDM1 (green) and CEP164 (red, marking distal appendages). Images are maximum intensity projections. Left, boxed regions magnified 2.7×. (E) ALI+20 MTECs fixed and stained with antibodies against MDM1 (green) and glu-tubulin (red) to visualize cilia. Images are maximum intensity projections. Left and right, boxed regions magnified 2.7×. Scale bars, 5 μm (D, E).Examination of ALI+6 MTEC cells by 3D-SIM revealed that MDM1 localized to centrioles in MCCs (Figure 2D). MTECs were labeled for MDM1 and CEP164 (Figure 2D). A distinct MDM1 focus was observed inside of each CEP164 ring, in agreement with the results in RPE-1 cells (Figure 2A). In late-stage ALI+20 MTECs, MDM1 staining was dim and diffuse in most ciliated cells (Figure 2E), only having the defined centriole labeling typical of early-stage cells in the few newly differentiating cells present at this stage. We conclude that MDM1 localizes to MCC centrioles during MCC differentiation but declines in abundance at centrioles in fully mature MCCs.

MDM1 localizes to the lumen of parental centrioles

We previously identified MDM1 as a centrosome component based on greatly elevated expression during ciliogenesis in mouse MCCs and colocalization of MDM1-GFP with γ-tubulin at the centrosome in cultured cells (Hoh et al., 2012). To confirm this localization for the endogenous protein, we used two anti-MDM1 polyclonal antibodies that recognize the N- or C-terminus of the human MDM1 protein (Supplemental Figure S1). We were unable to visualize endogenous MDM1 by Western blot (Supplemental Figure S1, A and B), consistent with a previous report of low-level expression (Snyder et al., 1988). Both antibodies showed that MDM1 localized to centrosomes throughout the cell cycle in hTERT-immortalized human RPE-1 cells (henceforth, RPE-1), appearing as two foci per cell in G1 through G2 phases (Figure 1A) and four foci per cell beginning in prophase of mitosis (Figure 1C). The antibodies were specific for MDM1 in human cells, as shown by loss of centrosomal MDM1 signal upon MDM1 depletion (e.g., see later discussion of Figure 6, A and B, and Supplemental Figure S4, A and B). This suggests that MDM1 is associated with centrioles and only localizes to the procentrioles at the onset of mitosis, when centriole duplication is complete and centrosomes undergo several maturation events. In addition, MDM1 remained associated with centrioles of RPE-1 cells that were arrested in G0 phase by serum withdrawal and had formed a primary cilium (Figure 1B). We also examined whether MDM1 localizes to centrioles in MCCs derived from tracheal epithelium. MTEC cultures were fixed 4 d after induction of differentiation by shifting cells to air–liquid interface culture (ALI+4). We determined the localization of MDM1 with respect to CEP164, a marker of fully formed centrioles. Both CEP164 and MDM1 antibodies labeled foci in the apical domain of MCCs, consistent with localization to centrioles (Figure 1D).

MDM1 localizes to centrioles. (A) Cycling RPE-1 cells were fixed and stained for MDM1 (green), CETN3 (red), and DNA (DAPI, blue). Right, magnified images showing the centrioles of a cell in G1 phase (two CETN3 foci); left, a cell in S/G2 (four CETN3 foci). (B) RPE-1 cells were serum starved, fixed, and stained for MDM1 (green), polyglutamylated tubulin (pg-tub, red), and DNA (blue). (C) Mitotic RPE-1 cells at indicated mitotic stages stained for MDM1 (green), CETN (red), and DNA (blue). Each inset corresponds to one spindle pole. All images in A–C were acquired by wide-field epifluorescence microscopy, and all insets represent 5× magnifications. Scale bars, 5 μm. (D) ALI+4 MTECs stained with antibodies against MDM1 (green) and CEP164 (red). Square images are maximum intensity projections of stacks acquired by spinning-disk confocal microscopy, and rectangles below them are xz-projections of a region within the boxed area in the merged maximum intensity projections.

The localization of MDM1 at centrioles was further defined in RPE-1 cells using three-dimensional structured illumination microscopy (3D-SIM). MDM1 localization was determined in reference to markers of the centriole distal appendages (CEP164), distal lumen (CETN3), proximal end (C-NAP1), and PCM (CEP152) (Figure 2A). MDM1 localized to the mid-proximal region of centrioles between the distal markers CETN3 and CEP164 and the proximal-end marker C-NAP1. The MDM1 signal was associated with each parental centriole and frequently observed to be bilobed in appearance. Whereas distal appendage and PCM proteins form rings resolvable by 3D-SIM, MDM1 localized in the center of the CEP164 and CEP152 rings, suggesting that MDM1 is in the lumen of the centriole. Consistent with this interpretation, MDM1 localized in the center of the centriole, as visualized by labeling polyglutamylated tubulin, a marker for the modified tubulin in the centriole barrels (Figure 2B). These results suggest that MDM1 resides in the centriole lumen between centrin and C-NAP1 (Figure 2C). Furthermore, in cases in which the bilobed appearance was observed, all MDM1 signal colocalized with the polyglutamylated tubulin signal of the parent centrioles and not to SASS6 foci (Figure 2B). Thus, as with wide-field imaging, 3D-SIM imaging showed that MDM1 only associated with the two parental centrioles in postduplication interphase cells (Figure 2, A–C).

MDM1 localizes to the centriole lumen. (A, B, D, E) Representative 3D-SIM micrographs. (A) RPE-1 cells were fixed and stained for MDM1 (green) and markers of the centriole distal lumen (CETN3), the PCM­­ (CEP152), the centriole distal appendages (CEP164), and the centriole proximal end (C-NAP1; all comarkers in red). (B) RPE-1 cells fixed and colabeled for MDM1 (green), polyglutamylated tubulin to mark the centriole barrels (pg-tub, red), and SASS6 to mark the procentriole cartwheels (blue). All images in A and B are maximum intensity projections. Scale bars, 1 μm. (C) Schematic depicting localization of MDM1 inside the centriole lumen. (D, E) 3D-SIM micrographs of MTEC cultures fixed at (D) ALI+6 or (E) ALI+20. (D) ALI+6 MTECs costained with antibodies against MDM1 (green) and CEP164 (red, marking distal appendages). Images are maximum intensity projections. Left, boxed regions magnified 2.7×. (E) ALI+20 MTECs fixed and stained with antibodies against MDM1 (green) and glu-tubulin (red) to visualize cilia. Images are maximum intensity projections. Left and right, boxed regions magnified 2.7×. Scale bars, 5 μm (D, E).

Examination of ALI+6 MTEC cells by 3D-SIM revealed that MDM1 localized to centrioles in MCCs (Figure 2D). MTECs were labeled for MDM1 and CEP164 (Figure 2D). A distinct MDM1 focus was observed inside of each CEP164 ring, in agreement with the results in RPE-1 cells (Figure 2A). In late-stage ALI+20 MTECs, MDM1 staining was dim and diffuse in most ciliated cells (Figure 2E), only having the defined centriole labeling typical of early-stage cells in the few newly differentiating cells present at this stage. We conclude that MDM1 localizes to MCC centrioles during MCC differentiation but declines in abundance at centrioles in fully mature MCCs.

MDM1 is a microtubule-binding proteinGiven the original identification of MDM1 on amplified genome fragments in cancer cells, we examined the effects of overexpression of MDM1. GFP-MDM1 expressed by transient transfection in RPE-1 cells localized to the centrosome as well as the nucleus (Supplemental Figure S2A), consistent with previous reports (Snyder et al., 1988; Hoh et al., 2012). In addition to these two expected localizations, GFP-MDM1 localized to cytoplasmic microtubules in highly expressing cells (Figure 3A). In some cells, GFP-MDM1 localized with filaments that resembled bundled microtubules. These filaments were associated with the centrosome and stained for α-tubulin and acetylated α-tubulin (Figure 3B and Supplemental Figure S2B) but did not stain for other markers of the centriole or ciliary axoneme, suggesting they are not hyperelongated centrioles or normal cilia (Figure 3B and Supplemental Figure S2B).FIGURE 3:MDM1 is a microtubule-binding protein. (A) RPE-1 cells transfected with GFP-MDM1 were fixed and stained for GFP (green), α-tubulin (red), and DNA (DAPI, blue). Image is a maximum intensity projection of stacks acquired by spinning-disk confocal microscopy. Scale bar, 10 μm. (B) RPE-1 cells expressing GFP-MDM1 were fixed and stained for GFP in green and (top) CETN3 or (bottom) α-tubulin in red. Images were taken by wide-field epifluorescence microscopy. Scale bar, 5 μm. (C) Domain structure of human MDM1 isoform 1 (Uniprot ID Q8TC05-1, NCBI NP_059136.2). (D) Phylogenetic tree showing conservation of MDM1. (E) Clustal Omega alignment showing the high degree of conservation of the MDM1 (S/T)EYxxx(F/Y) repeat sequences in the same species shown in the phylogenetic tree. (F) Schematic of MDM1 domains required for microtubule colocalization in RPE-1 cells. MDM1rm corresponds to a mutant version of MDM1 in which all four (S/T)EYxxxF repeat motifs have been mutated to AAAxxxA by site-directed mutagenesis. All other constructs are indicated by amino acid positions included within the construct. Colocalization was assessed by expressing GFP-fusion proteins in RPE-1 cells, treating with Taxol to bundle microtubules, staining for GFP and α-tubulin, and scoring the percentage of high-expressing transfected cells with GFP-microtubule colocalization in three trials, 100 cells/condition per trial (see Supplemental Figure S2, C and D, for data). Percentage of cells showing colocalization in all three trials indicated as follows: +++, >90%; ++, >70%; +, >5%, –, <5%. (G) Microtubule cosedimentation assay using purified GST-tagged recombinant protein and Taxol-stabilized microtubules. Indicated proteins were incubated with Taxol-stabilized microtubules or buffer as indicated, loaded onto a 40% glycerol cushion, and spun down (55,000 rpm [∼250,000 × g] for 10 min). Input (I), supernatant (S), and pellet (P) fractions were run out on an SDS–PAGE gel, and proteins were detected by Western blot using antibody against GST.Other microtubule-binding proteins, such as MAP2 and TAU, have repeated motifs that are directly involved in their microtubule binding (reviewed in Dehmelt and Halpain, 2005). We found that MDM1 lacks any known microtubule-binding motifs but does have a repeated sequence, present in four copies in the N-terminal half, consisting of the motif (S/T)EYxxxF (Figure 3C). If these repeats were important for MDM1 function, we would expect them to be conserved in evolution. MDM1 itself is present in multicellular organisms with ciliated cells, ranging from the placozoan Trichoplax adhaerens to human, although, of note, it is absent in Caenorhabditis elegans (Figure 3D). The repeats were found in all MDM1 orthologues examined (Figure 3E), with the protein in most species having at least two copies of the repeat, in some cases with a tyrosine (Y) residue in place of the phenylalanine (F). We also found two of the (S/T)EYxxx(F/Y) repeats in centriole, cilia, and spindle–associated protein (CCSAP), a microtubule-binding protein that localizes to centrioles and cilia (Backer et al., 2012).To test whe­­ther these repeats are important for microtubule binding, we constructed a mutant version of MDM1 that had the conserved residues in all four repeats replaced by alanine (4x (S/T)EYxxxF to AAAxxxA, hereafter called MDM1rm), as well as truncations with different numbers of the repeats. GFP fusions of these constructs were expressed in RPE-1 cells that had been treated with the microtubule-stabilizing drug Taxol before fixation, creating large bundles of microtubules to enhance visualization of colocalization (Schiff and Horwitz, 1980; Figure 3F and Supplemental Figure S2, C and D). GFP-MDM1 colocalized with microtubules in 97.0 ± 3.00% of cells. In contrast, the GFP-MDM1rm fusion was expressed at the same level as MDM1 (Supplemental Figure S2E) but colocalized with microtubules in only 8.67 ± 1.20% of cells and even in positive cells was less enriched on microtubules than in wild type (wt). Although MDM1rm did not associate with microtubules, it was able to localize to the centrosome (see later discussion of Figure 5C). Constructs that had at least two of the repeats localized to microtubules, whereas the single-repeat-containing construct and two C-terminal fragments that lack all repeats did not.To test whether the association of MDM1 with microtubules reflects direct binding, we purified recombinant glutathione S-transferase (GST)–tagged MDM1 and used in in vitro microtubule cosedimentation assays (Figure 3G). GST-tagged fusions were prepared for full- length MDM1, MDM1rm, MDM11-416 (contains all four repeats), and MDM1490-714 (lacks all four repeats). GST-MDM1, but not GST alone, pelleted with microtubules, indicating a direct interaction of MDM1 with microtubules. Consistent with the localization results, GST-MDM11–416 also pelleted with microtubules. Of interest, GST-MDM1rm also pelleted with microtubules, although about half of the protein remained in the supernatant, whereas the GST-MDM1490–713 protein remained primarily in the supernatant. Together the localization and binding results show that MDM1 is a microtubule-binding protein and that the conserved repeat elements of MDM1 are important for full binding activity.

MDM1 is a microtubule-binding protein

Given the original identification of MDM1 on amplified genome fragments in cancer cells, we examined the effects of overexpression of MDM1. GFP-MDM1 expressed by transient transfection in RPE-1 cells localized to the centrosome as well as the nucleus (Supplemental Figure S2A), consistent with previous reports (Snyder et al., 1988; Hoh et al., 2012). In addition to these two expected localizations, GFP-MDM1 localized to cytoplasmic microtubules in highly expressing cells (Figure 3A). In some cells, GFP-MDM1 localized with filaments that resembled bundled microtubules. These filaments were associated with the centrosome and stained for α-tubulin and acetylated α-tubulin (Figure 3B and Supplemental Figure S2B) but did not stain for other markers of the centriole or ciliary axoneme, suggesting they are not hyperelongated centrioles or normal cilia (Figure 3B and Supplemental Figure S2B).

MDM1 is a microtubule-binding protein. (A) RPE-1 cells transfected with GFP-MDM1 were fixed and stained for GFP (green), α-tubulin (red), and DNA (DAPI, blue). Image is a maximum intensity projection of stacks acquired by spinning-disk confocal microscopy. Scale bar, 10 μm. (B) RPE-1 cells expressing GFP-MDM1 were fixed and stained for GFP in green and (top) CETN3 or (bottom) α-tubulin in red. Images were taken by wide-field epifluorescence microscopy. Scale bar, 5 μm. (C) Domain structure of human MDM1 isoform 1 (Uniprot ID Q8TC05-1, NCBI NP_059136.2). (D) Phylogenetic tree showing conservation of MDM1. (E) Clustal Omega alignment showing the high degree of conservation of the MDM1 (S/T)EYxxx(F/Y) repeat sequences in the same species shown in the phylogenetic tree. (F) Schematic of MDM1 domains required for microtubule colocalization in RPE-1 cells. MDM1rm corresponds to a mutant version of MDM1 in which all four (S/T)EYxxxF repeat motifs have been mutated to AAAxxxA by site-directed mutagenesis. All other constructs are indicated by amino acid positions included within the construct. Colocalization was assessed by expressing GFP-fusion proteins in RPE-1 cells, treating with Taxol to bundle microtubules, staining for GFP and α-tubulin, and scoring the percentage of high-expressing transfected cells with GFP-microtubule colocalization in three trials, 100 cells/condition per trial (see Supplemental Figure S2, C and D, for data). Percentage of cells showing colocalization in all three trials indicated as follows: +++, >90%; ++, >70%; +, >5%, –, <5%. (G) Microtubule cosedimentation assay using purified GST-tagged recombinant protein and Taxol-stabilized microtubules. Indicated proteins were incubated with Taxol-stabilized microtubules or buffer as indicated, loaded onto a 40% glycerol cushion, and spun down (55,000 rpm [∼250,000 × g] for 10 min). Input (I), supernatant (S), and pellet (P) fractions were run out on an SDS–PAGE gel, and proteins were detected by Western blot using antibody against GST.

Other microtubule-binding proteins, such as MAP2 and TAU, have repeated motifs that are directly involved in their microtubule binding (reviewed in Dehmelt and Halpain, 2005). We found that MDM1 lacks any known microtubule-binding motifs but does have a repeated sequence, present in four copies in the N-terminal half, consisting of the motif (S/T)EYxxxF (Figure 3C). If these repeats were important for MDM1 function, we would expect them to be conserved in evolution. MDM1 itself is present in multicellular organisms with ciliated cells, ranging from the placozoan Trichoplax adhaerens to human, although, of note, it is absent in Caenorhabditis elegans (Figure 3D). The repeats were found in all MDM1 orthologues examined (Figure 3E), with the protein in most species having at least two copies of the repeat, in some cases with a tyrosine (Y) residue in place of the phenylalanine (F). We also found two of the (S/T)EYxxx(F/Y) repeats in centriole, cilia, and spindle–associated protein (CCSAP), a microtubule-binding protein that localizes to centrioles and cilia (Backer et al., 2012).

To test whe­­ther these repeats are important for microtubule binding, we constructed a mutant version of MDM1 that had the conserved residues in all four repeats replaced by alanine (4x (S/T)EYxxxF to AAAxxxA, hereafter called MDM1rm), as well as truncations with different numbers of the repeats. GFP fusions of these constructs were expressed in RPE-1 cells that had been treated with the microtubule-stabilizing drug Taxol before fixation, creating large bundles of microtubules to enhance visualization of colocalization (Schiff and Horwitz, 1980; Figure 3F and Supplemental Figure S2, C and D). GFP-MDM1 colocalized with microtubules in 97.0 ± 3.00% of cells. In contrast, the GFP-MDM1rm fusion was expressed at the same level as MDM1 (Supplemental Figure S2E) but colocalized with microtubules in only 8.67 ± 1.20% of cells and even in positive cells was less enriched on microtubules than in wild type (wt). Although MDM1rm did not associate with microtubules, it was able to localize to the centrosome (see later discussion of Figure 5C). Constructs that had at least two of the repeats localized to microtubules, whereas the single-repeat-containing construct and two C-terminal fragments that lack all repeats did not.

To test whether the association of MDM1 with microtubules reflects direct binding, we purified recombinant glutathione S-transferase (GST)–tagged MDM1 and used in in vitro microtubule cosedimentation assays (Figure 3G). GST-tagged fusions were prepared for full- length MDM1, MDM1rm, MDM11-416 (contains all four repeats), and MDM1490-714 (lacks all four repeats). GST-MDM1, but not GST alone, pelleted with microtubules, indicating a direct interaction of MDM1 with microtubules. Consistent with the localization results, GST-MDM11–416 also pelleted with microtubules. Of interest, GST-MDM1rm also pelleted with microtubules, although about half of the protein remained in the supernatant, whereas the GST-MDM1490–713 protein remained primarily in the supernatant. Together the localization and binding results show that MDM1 is a microtubule-binding protein and that the conserved repeat elements of MDM1 are important for full binding activity.

MDM1 stabilizes microtubulesGiven that MDM1 interacts directly with microtubules in vitro and is closely associated with the stable centriole microtubules, we considered the possibility that MDM1 binding stabilizes micro­tubules. To test this, we expressed GFP-MDM1 in RPE-1 cells, followed by treatment with the microtubule-depolymerizing drug nocodazole and/or incubation of the cells at 4ºC (Figure 4, A and B). Treatment with 5 μg/ml nocodazole, 4ºC incubation, or both treatments for 1–2 h was sufficient to depolymerize micro­tubules in nearly all control GFP-transfected cells (Figure 4B). In contrast, many GFP-MDM1–transfected cells retained some polymerized microtubules: 46.7 ± 5.84% of cells in nocodazole, 76.7 ± 5.86% of cells in the 4ºC incubation, and 74.0 ± 14.0% of cells in nocodazole plus 4ºC incubation. Consistent with the localization and binding results, GFP-MDM1rm had little effect on microtubule stability in any of the treatments (Figure 4, A and B). Thus MDM1 binds and stabilizes microtubules, and the repeat elements are important for both properties.FIGURE 4:MDM1 stabilizes microtubules. (A) RPE-1 cells expressing GFP, GFP-MDM1, or GFP-MDM1rm were incubated with either 5 μg/ml nocodazole or the equivalent volume of DMSO (vehicle control) for 1–2 h at either 37 or 4ºC. Cells were fixed immediately after and stained for GFP (green), α-tubulin (red), and DNA (DAPI, blue) to determine how many transfected cells had polymerized microtubules intact. Representative images of the nocodazole treatment at 37ºC. All images are maximum intensity projections of stacks acquired by spinning-disk confocal microscopy. Scale bar, 10 μm. (B) Quantitation of phenotypes observed in A across three trials, mean ± SEM (100 transfected cells/construct were counted for each condition in each trial).

MDM1 stabilizes microtubules

Given that MDM1 interacts directly with microtubules in vitro and is closely associated with the stable centriole microtubules, we considered the possibility that MDM1 binding stabilizes micro­tubules. To test this, we expressed GFP-MDM1 in RPE-1 cells, followed by treatment with the microtubule-depolymerizing drug nocodazole and/or incubation of the cells at 4ºC (Figure 4, A and B). Treatment with 5 μg/ml nocodazole, 4ºC incubation, or both treatments for 1–2 h was sufficient to depolymerize micro­tubules in nearly all control GFP-transfected cells (Figure 4B). In contrast, many GFP-MDM1–transfected cells retained some polymerized microtubules: 46.7 ± 5.84% of cells in nocodazole, 76.7 ± 5.86% of cells in the 4ºC incubation, and 74.0 ± 14.0% of cells in nocodazole plus 4ºC incubation. Consistent with the localization and binding results, GFP-MDM1rm had little effect on microtubule stability in any of the treatments (Figure 4, A and B). Thus MDM1 binds and stabilizes microtubules, and the repeat elements are important for both properties.

MDM1 stabilizes microtubules. (A) RPE-1 cells expressing GFP, GFP-MDM1, or GFP-MDM1rm were incubated with either 5 μg/ml nocodazole or the equivalent volume of DMSO (vehicle control) for 1–2 h at either 37 or 4ºC. Cells were fixed immediately after and stained for GFP (green), α-tubulin (red), and DNA (DAPI, blue) to determine how many transfected cells had polymerized microtubules intact. Representative images of the nocodazole treatment at 37ºC. All images are maximum intensity projections of stacks acquired by spinning-disk confocal microscopy. Scale bar, 10 μm. (B) Quantitation of phenotypes observed in A across three trials, mean ± SEM (100 transfected cells/construct were counted for each condition in each trial).

MDM1 overexpression suppresses centriole duplicationTwo other microtubule-binding proteins that localize to the centriole lumen, CPAP (Hsu et al., 2008) and CEP135 (Carvalho-Santos et al., 2012), are important for centriole duplication. We tested whether MDM1 might also be involved in centriole duplication. First, we overexpressed GFP-MDM1 in RPE-1 cells and counted centrioles by centrin staining (Figure 5A). At 24 h posttransfection, there was no significant difference in the number of centrioles per cell on comparing GFP-expressing and GFP-MDM1–expressing cells, with most cells having two centrioles (G1-phase cells) or four centrioles (S/G2/mitotic cells; p = 0.148, Fisher’s exact test; Supplemental Figure S3A). At 48 h posttransfection, however, GFP-MDM1–expressing cells had fewer centrioles (Figure 5A, p = 2.20 × 10−16, Fisher’s exact test), apparent as a decrease in the number of cells with four centrioles and an increase in those with zero or one centriole(s) per cell. A similar result was observed using γ-tubulin as the centrosome marker and quantifying the percentage of cells with fewer than two γ-tubulin foci (p = 0.0146 at 24 h posttransfection and p = 2.30 × 10−15 at 48 h posttransfection, Fisher’s exact test, Supplemental Figure S3, B and C).We next tested whether overexpression of MDM1 is capable of blocking centriole duplication in cells that reduplicate centrioles during S-phase arrest (Balczon et al., 1995) (Figure 5, B and C). GFP-MDM1 significantly suppressed reduplication compared with controls (p = 0.00403 compared with GFP-expressing control cells, two-tailed unpaired Student’s t test), whereas neither GFP nor GFP fused to the pericentrin centrosomal targeting domain (Gillingham and Munro, 2000) had an effect on reduplication. Of importance, GFP-MDM1rm did not significantly block centriole reduplication (Figure 5, B and C, p = 0.729 compared with GFP-expressing control cells, two-tailed unpaired Student’s t test), although it was expressed at the same level as GFP-MDM1 (Figure 5D). In agreement with this, we also found that GFP-MDM1rm was less effective at blocking normal centriole duplication in cycling RPE-1 cells than was GFP-MDM1 (Supplemental Figure S3D). These results demonstrate that the microtubule-binding and -stabilizing properties of MDM1 mediate the suppression of centriole duplication caused by MDM1 overexpression and suggest that the normal function of MDM1 is to negatively regulate centriole duplication.FIGURE 5:Transient MDM1 overexpression inhibits centriole duplication. (A) RPE-1 cells were transfected with GFP or GFP-MDM1, fixed 48 h later, and stained for GFP and CETN3. The number of CETN3 foci per transfected cell was scored for each condition (three trials, 200 cells/trial for each condition). Average percentage of transfected cells with indicated number of CETN3 foci ± SEM is shown. Statistical significance was assessed using a Fisher’s exact test in R (p = 2.20 × 10−16) with the summed data for all three trials. (B, C) U2OS cells were transfected with the indicated constructs and then arrested in S phase with the addition of 4 mM HU for 48 h, fixed, and stained for GFP and CETN3. The number of cells with more than four CETN3 foci (representing cells with reduplicated centrioles) in transfected cells was scored for each condition (three trials, 100 cells/trial for each condition). Average percentage of cells with more than four CETN3 foci ±SEM. The p values were computed using a two-tailed unpaired Student’s t test, comparing the means in cells overexpressing the indicated construct to the mean of cells overexpressing GFP alone (**p < 0.01). (C) Representative images from the U2OS reduplication assays for control conditions, GFP-MDM1 expression, and GFP-MDM1rm expression. Cells were stained with GFP (green) and CETN3 (red). Images were acquired by wide-field epifluorescence microscopy. Scale bar, 1 μm. (D) Lysates from U2OS cells transfected with the indicated constructs were run on an SDS–PAGE gel for Western blot to show relative expression levels of constructs used in the U2OS reduplication assay. Blots were probed with antibody against GFP and α-tubulin (loading control).

MDM1 overexpression suppresses centriole duplication

Two other microtubule-binding proteins that localize to the centriole lumen, CPAP (Hsu et al., 2008) and CEP135 (Carvalho-Santos et al., 2012), are important for centriole duplication. We tested whether MDM1 might also be involved in centriole duplication. First, we overexpressed GFP-MDM1 in RPE-1 cells and counted centrioles by centrin staining (Figure 5A). At 24 h posttransfection, there was no significant difference in the number of centrioles per cell on comparing GFP-expressing and GFP-MDM1–expressing cells, with most cells having two centrioles (G1-phase cells) or four centrioles (S/G2/mitotic cells; p = 0.148, Fisher’s exact test; Supplemental Figure S3A). At 48 h posttransfection, however, GFP-MDM1–expressing cells had fewer centrioles (Figure 5A, p = 2.20 × 10−16, Fisher’s exact test), apparent as a decrease in the number of cells with four centrioles and an increase in those with zero or one centriole(s) per cell. A similar result was observed using γ-tubulin as the centrosome marker and quantifying the percentage of cells with fewer than two γ-tubulin foci (p = 0.0146 at 24 h posttransfection and p = 2.30 × 10−15 at 48 h posttransfection, Fisher’s exact test, Supplemental Figure S3, B and C).

We next tested whether overexpression of MDM1 is capable of blocking centriole duplication in cells that reduplicate centrioles during S-phase arrest (Balczon et al., 1995) (Figure 5, B and C). GFP-MDM1 significantly suppressed reduplication compared with controls (p = 0.00403 compared with GFP-expressing control cells, two-tailed unpaired Student’s t test), whereas neither GFP nor GFP fused to the pericentrin centrosomal targeting domain (Gillingham and Munro, 2000) had an effect on reduplication. Of importance, GFP-MDM1rm did not significantly block centriole reduplication (Figure 5, B and C, p = 0.729 compared with GFP-expressing control cells, two-tailed unpaired Student’s t test), although it was expressed at the same level as GFP-MDM1 (Figure 5D). In agreement with this, we also found that GFP-MDM1rm was less effective at blocking normal centriole duplication in cycling RPE-1 cells than was GFP-MDM1 (Supplemental Figure S3D). These results demonstrate that the microtubule-binding and -stabilizing properties of MDM1 mediate the suppression of centriole duplication caused by MDM1 overexpression and suggest that the normal function of MDM1 is to negatively regulate centriole duplication.

Transient MDM1 overexpression inhibits centriole duplication. (A) RPE-1 cells were transfected with GFP or GFP-MDM1, fixed 48 h later, and stained for GFP and CETN3. The number of CETN3 foci per transfected cell was scored for each condition (three trials, 200 cells/trial for each condition). Average percentage of transfected cells with indicated number of CETN3 foci ± SEM is shown. Statistical significance was assessed using a Fisher’s exact test in R (p = 2.20 × 10−16) with the summed data for all three trials. (B, C) U2OS cells were transfected with the indicated constructs and then arrested in S phase with the addition of 4 mM HU for 48 h, fixed, and stained for GFP and CETN3. The number of cells with more than four CETN3 foci (representing cells with reduplicated centrioles) in transfected cells was scored for each condition (three trials, 100 cells/trial for each condition). Average percentage of cells with more than four CETN3 foci ±SEM. The p values were computed using a two-tailed unpaired Student’s t test, comparing the means in cells overexpressing the indicated construct to the mean of cells overexpressing GFP alone (**p < 0.01). (C) Representative images from the U2OS reduplication assays for control conditions, GFP-MDM1 expression, and GFP-MDM1rm expression. Cells were stained with GFP (green) and CETN3 (red). Images were acquired by wide-field epifluorescence microscopy. Scale bar, 1 μm. (D) Lysates from U2OS cells transfected with the indicated constructs were run on an SDS–PAGE gel for Western blot to show relative expression levels of constructs used in the U2OS reduplication assay. Blots were probed with antibody against GFP and α-tubulin (loading control).

MDM1 depletion results in stimulation of the centriole duplication pathwayIf MDM1 were a negative regulator of centriole duplication, we might expect depletion of MDM1 to result in stimulation of the centriole duplication pathway, as described for other putative negative regulators (Cunha-Ferreira et al., 2009; Hemerly et al., 2009; Rogers et al., 2009; Tsang et al., 2009; Balestra et al., 2013; Firat-Karalar et al., 2014). To test this, we depleted MDM1 from RPE-1 cells by lentivirus-mediated short hairpin RNA (shRNA) treatment. MDM1 depletion was confirmed by immunofluorescence (Figure 6, A and B, and Supplemental Figure S4, A and B). Depletion of MDM1 did not affect cell cycle progression as assessed by flow cytometry (Supplemental Figure S4, C and D). The effect of MDM1 depletion on centrosome number was assessed by staining for centriole and centrosome markers (Figure 6, C and D). There was no difference in the number of centrosomes per cell, as assayed by γ-tubulin staining (Supplemental Figure S5B), and no increase in the centrosomal abundance of the positive regulators PLK4, CEP63, CEP152, and SASS6 (Supplemental Figure S6). However, there was a statistically significant enrichment of cells with more than four centrin foci in depleted cells (4.38 ± 0.800% in control cells and 10.9 ± 1.46% in depleted cells; p = 0.00801, unpaired two-tailed Student’s t test; Figure 6, C and D); this phenotype was rescued by expression of an shRNA-resistant version of MDM1 (Supplemental Figure S5A). Similar results were observed with CPAP (Supplemental Figure S5, C and D, 2.67 ± 0.882% in controls vs. 8.00 ± 1.53% in depleted cells; p = 0.0390, unpaired two-tailed Student’s t test) but not with CP110 (Supplemental Figure S5E), suggesting that the foci formed in MDM1-depleted cells are not mature centrioles.FIGURE 6:MDM1 depletion stimulates the formation of supernumerary centrin foci. (A) RPE-1 cells transduced with empty vector carrying control lentivirus or MDM1 shRNA were fixed 6 d after start of transduction and stained for MDM1 (green; Abnova mouse anti-MDM1 antibody) and CEP164 (red) to mark the centrosome region. Images are maximum intensity projections of stacks acquired by spinning-disk confocal microscopy with identical gain, laser intensity, and exposure settings for both samples. Brightness and contrast of original images are unchanged. Scale, 2 μm. (B) Effectiveness of MDM1 depletion was measured by loss of fluorescence intensity at the centrosome using mouse anti-MDM1 (Abnova). Centrosome regions were defined by CEP164 labeling as in A. Boxplots depict centrosomal MDM1 intensities of all control and depleted cells measured (three trials, 100 cells/trial for each condition). For each trial, normalized mean MDM1 intensities were computed for control and depleted cells. The average normalized mean MDM1 intensity in depleted cells was 0.290 that of control cells (±0.0711, SEM; p = 6.56 × 10−5, unpaired two-tailed Student’s t test). (C) RPE-1 cells transduced with empty vector or MDM1 shRNA were fixed 6 d after start of transduction and stained for CETN3 (green) and γ-tubulin (red) to visualize centrioles and centrosomes, respectively. Scale, 10 μm. (D) The percentage of control and MDM1-depleted cells with more than four CETN3 foci 6 d after start of transduction was quantified (four trials, 200 cells/trial for each sample). Average percentage of cells with more than four CETN3 foci (±SEM) for each sample (control cells, 4.38 ± 0.800%; shRNA-transduced cells, 10.9 ± 1.46%; p = 0.00801, unpaired two-tailed Student’s t test).To determine the nature of the foci in MDM1-depleted cells, we performed correlative light and electron microscopy (CLEM). MDM1 was depleted in RPE-1 cells expressing centrin1-GFP, and cells with multiple centrin1-GFP foci were identified by live-cell fluorescence microscopy. The coverslips were then fixed, and the same cells were analyzed by transmission electron microscopy (TEM). These cells usually had four centrioles, indicating that they were in S/G2 (five of six cells with more than four centrin foci were observed to have procentrioles in EM micrographs). The centrin foci were visible as darkly staining aggregates lacking microtubule structure characteristic of bona fide centrioles (Figure 7, n = 6). These aggregates had an average diameter of 70 ± 20 nm (SD), similar in size to structures that have previously been observed as early indicators of de novo centriole formation (Prosser et al., 2009) and to the fibrous granules found preceding centriole amplification in multicilated epithelial cells (Sorokin, 1968; Steinman, 1968; Anderson and Brenner, 1971; Dirksen, 1971). Similar structures were reported in CLEM analysis of GFP-CPAP foci observed upon depletion of another negative regulator of centriole duplication, RBM14 (Shiratsuchi et al., 2015).FIGURE 7:Supernumerary centrin foci resemble granular material by CLEM. (A–C) CLEM of CETN1-GFP RPE-1 cells transduced with MDM1 shRNA lentivirus. (A) Maximum intensity projection of an MDM1-depleted CETN1-GFP RPE-1 cell with more than four centrin-GFP foci imaged by spinning-disk confocal microscopy. Scale, 5 μm. (B) TEM micrographs of serial sections of the same cell (which has four centrioles, and split centrosomes). Scale, 2 μm. (C) Magnified insets of the previous series that focus on each centrosome and surrounding granular material. Scale bar, 500 nm. Alike arrowheads point to CETN1-GFP material that roughly colocalizes with dark-staining granules in EM micrographs throughout all images.

MDM1 depletion results in stimulation of the centriole duplication pathway

If MDM1 were a negative regulator of centriole duplication, we might expect depletion of MDM1 to result in stimulation of the centriole duplication pathway, as described for other putative negative regulators (Cunha-Ferreira et al., 2009; Hemerly et al., 2009; Rogers et al., 2009; Tsang et al., 2009; Balestra et al., 2013; Firat-Karalar et al., 2014). To test this, we depleted MDM1 from RPE-1 cells by lentivirus-mediated short hairpin RNA (shRNA) treatment. MDM1 depletion was confirmed by immunofluorescence (Figure 6, A and B, and Supplemental Figure S4, A and B). Depletion of MDM1 did not affect cell cycle progression as assessed by flow cytometry (Supplemental Figure S4, C and D). The effect of MDM1 depletion on centrosome number was assessed by staining for centriole and centrosome markers (Figure 6, C and D). There was no difference in the number of centrosomes per cell, as assayed by γ-tubulin staining (Supplemental Figure S5B), and no increase in the centrosomal abundance of the positive regulators PLK4, CEP63, CEP152, and SASS6 (Supplemental Figure S6). However, there was a statistically significant enrichment of cells with more than four centrin foci in depleted cells (4.38 ± 0.800% in control cells and 10.9 ± 1.46% in depleted cells; p = 0.00801, unpaired two-tailed Student’s t test; Figure 6, C and D); this phenotype was rescued by expression of an shRNA-resistant version of MDM1 (Supplemental Figure S5A). Similar results were observed with CPAP (Supplemental Figure S5, C and D, 2.67 ± 0.882% in controls vs. 8.00 ± 1.53% in depleted cells; p = 0.0390, unpaired two-tailed Student’s t test) but not with CP110 (Supplemental Figure S5E), suggesting that the foci formed in MDM1-depleted cells are not mature centrioles.

MDM1 depletion stimulates the formation of supernumerary centrin foci. (A) RPE-1 cells transduced with empty vector carrying control lentivirus or MDM1 shRNA were fixed 6 d after start of transduction and stained for MDM1 (green; Abnova mouse anti-MDM1 antibody) and CEP164 (red) to mark the centrosome region. Images are maximum intensity projections of stacks acquired by spinning-disk confocal microscopy with identical gain, laser intensity, and exposure settings for both samples. Brightness and contrast of original images are unchanged. Scale, 2 μm. (B) Effectiveness of MDM1 depletion was measured by loss of fluorescence intensity at the centrosome using mouse anti-MDM1 (Abnova). Centrosome regions were defined by CEP164 labeling as in A. Boxplots depict centrosomal MDM1 intensities of all control and depleted cells measured (three trials, 100 cells/trial for each condition). For each trial, normalized mean MDM1 intensities were computed for control and depleted cells. The average normalized mean MDM1 intensity in depleted cells was 0.290 that of control cells (±0.0711, SEM; p = 6.56 × 10−5, unpaired two-tailed Student’s t test). (C) RPE-1 cells transduced with empty vector or MDM1 shRNA were fixed 6 d after start of transduction and stained for CETN3 (green) and γ-tubulin (red) to visualize centrioles and centrosomes, respectively. Scale, 10 μm. (D) The percentage of control and MDM1-depleted cells with more than four CETN3 foci 6 d after start of transduction was quantified (four trials, 200 cells/trial for each sample). Average percentage of cells with more than four CETN3 foci (±SEM) for each sample (control cells, 4.38 ± 0.800%; shRNA-transduced cells, 10.9 ± 1.46%; p = 0.00801, unpaired two-tailed Student’s t test).

To determine the nature of the foci in MDM1-depleted cells, we performed correlative light and electron microscopy (CLEM). MDM1 was depleted in RPE-1 cells expressing centrin1-GFP, and cells with multiple centrin1-GFP foci were identified by live-cell fluorescence microscopy. The coverslips were then fixed, and the same cells were analyzed by transmission electron microscopy (TEM). These cells usually had four centrioles, indicating that they were in S/G2 (five of six cells with more than four centrin foci were observed to have procentrioles in EM micrographs). The centrin foci were visible as darkly staining aggregates lacking microtubule structure characteristic of bona fide centrioles (Figure 7, n = 6). These aggregates had an average diameter of 70 ± 20 nm (SD), similar in size to structures that have previously been observed as early indicators of de novo centriole formation (Prosser et al., 2009) and to the fibrous granules found preceding centriole amplification in multicilated epithelial cells (Sorokin, 1968; Steinman, 1968; Anderson and Brenner, 1971; Dirksen, 1971). Similar structures were reported in CLEM analysis of GFP-CPAP foci observed upon depletion of another negative regulator of centriole duplication, RBM14 (Shiratsuchi et al., 2015).

Supernumerary centrin foci resemble granular material by CLEM. (A–C) CLEM of CETN1-GFP RPE-1 cells transduced with MDM1 shRNA lentivirus. (A) Maximum intensity projection of an MDM1-depleted CETN1-GFP RPE-1 cell with more than four centrin-GFP foci imaged by spinning-disk confocal microscopy. Scale, 5 μm. (B) TEM micrographs of serial sections of the same cell (which has four centrioles, and split centrosomes). Scale, 2 μm. (C) Magnified insets of the previous series that focus on each centrosome and surrounding granular material. Scale bar, 500 nm. Alike arrowheads point to CETN1-GFP material that roughly colocalizes with dark-staining granules in EM micrographs throughout all images.

DISCUSSIONWe previously identified Mdm1 as a gene that is transcriptionally up-regulated during the in vitro differentiation of MCCs of the mouse tracheal epithelium (Hoh et al., 2012). A mutation in Mdm1 was reported to be associated with retinal degeneration in mice, a phenotype that is common to ciliopathies (Chang et al., 2008). Here we report that MDM1 is component of centrioles and localizes close to the centriole microtubule barrel. We found that MDM1 associates with microtubules when overexpressed in cells and binds microtubules directly in vitro. Overexpression of MDM1 inhibits centriole formation, whereas depletion results in formation of structures resembling assembly intermediates. Together these data suggest that MDM1 negatively regulates centriole formation and that microtubule binding is likely to be an important part of that function.MDM1 normally localizes only to centrioles but, when overexpressed, also associates with cytoplasmic microtubules. How might a microtubule-binding protein be limited to only a subset of microtubules? The microtubules of the centriole barrel are modified by posttranslational modifications and have a unique triplet micro­tubule structure. We note that CCSAP, a microtubule-binding protein that has repeats similar to those of MDM1, preferentially associates with polyglutamylated microtubules (Backer et al., 2012). Similarly, MDM1 might have a higher affinity for these structurally unique microtubules of the centriole or might be directed there by interaction with other proteins that do. Based on the overexpression phenotype, it is likely that MDM1 contributes to the stability of the centriole microtubules. Although no obvious centriole barrel defects were observed in MDM1-depleted cells, a number of other centriole-associated proteins, including CEP120 (Lin et al., 2013), CEP135 (Carvalho-Santos et al., 2012), CCSAP (Backer et al., 2012), and CPAP (Hsu et al., 2008), are also microtubule-binding proteins, and thus there are likely to be redundant mechanisms contributing to barrel stability.We also identified repeats of a novel motif, (S/T)EYxxx(F/Y), which are highly conserved in MDM1 orthologues. We found that the centriole-associated microtubule-binding protein CCSAP (Backer et al., 2012) also has two copies of this motif, which are also conserved. This motif in MDM1 was important for both in vitro binding of MDM1 to microtubules and MDM1-mediated microtubule stabilization. We note that mutating the repeat sequences in MDM1 could have a nonspecific effect on protein structure, as is true for any mutant, although we consider it unlikely, given that the wt and repeat mutant forms of MDM1 behave similarly with respect to expression, solubility, and centrosome localization. It is likely that sequences outside of these repeats are also important for the microtubule associations observed, since the repeat mutant MDM1rm partially associated with microtubules both in vivo and in vitro. Most MDM1 orthologues contain at least two repeat motifs, and the microtubule association required two or more of the motifs (Figure 3F and Supplemental Figure S2, C and D). The repeats might be involved in directly binding microtubules, as observed for repeated motifs in other microtubule-binding proteins, such as TAU and MAP2 (Dehmelt and Halpain, 2005). The repeats also might be subject to posttranslational modification that modulates the interaction between MDM1 and microtubules.Our results suggest that MDM1 is a negative regulator of centriole duplication and that MDM1-microtubule binding is likely to be important for this function. MDM1 is recruited to procentrioles near the onset of mitosis, when procentrioles finish duplication and begin a maturation process required to become parental centrioles in the next cell cycle (Wang et al., 2011). MDM1 overexpression and depletion experiments both revealed phenotypes consistent with it functioning as a negative regulator of centriole duplication. Over­expression of MDM1 blocked both centriole duplication in cycling RPE-1 cells and centriole reduplication in S phase–arrested U2OS cells. Because reduplication is believed to derive from repeated rounds of centriole disengagement and centriole formation (Loncarek et al., 2010), this suggests that MDM1 functions in the canonical duplication pathway, thus affecting both. This is in contrast to the mechanism of ORC1, which was also shown to be a negative regulator of centriole duplication and specifically blocks reduplication and not canonical duplication (Hemerly et al., 2009). Of interest, expression of MDM1rm failed to block centriole reduplication (Figure 5B) and was defective at blocking normal centriole duplication (Supplemental Figure S3D), despite the fact that it localized to centrosomes (Figure 5C). This suggests that MDM1–microtubule binding is important for the inhibition of centriole duplication caused by MDM1 overexpression. A possible mechanism for the MDM1 inhibition of duplication is that overexpressed MDM1 binds to its sites at the proximal end of centrioles and physically blocks the recruitment of a protein or complex required for duplication. This would also be consistent with the seemingly paradoxical observation that MDM1 is highly up-regulated in MTECs, which are making hundreds of centrioles; centrioles in MTECs are made by a different pathway, initiated by deuterosomes rather than preexisting centrioles (Zhao et al., 2013), and MDM1 might block duplication from newly formed centrioles in these cells. A recent study suggested that SASS6 localizes to the proximal lumen of parental centrioles at the G1/S transition and that this localization is involved in templating cartwheel formation for the new procentriole (Fong et al., 2014). It is possible that excess MDM1 interferes with the reentry of SASS6 into the parental centriole proximal lumen, in turn blocking duplication.Depletion of MDM1 was associated with an increase in centrin foci that appeared as dense granules by EM. We also detected increased foci of CPAP, a centriole duplication protein, by immunofluorescence. Such foci are characteristic of early stages of centriole formation in the de novo centriole duplication pathway (Prosser et al., 2009), and similar centrin+/CPAP+ structures have been observed in cases in which other negative regulators of centriole duplication are depleted, such as is the case for RBM14 (Shiratsuchi et al., 2015) and NEURL4 (Li et al., 2012). We stress that we did not observe extra, bona fide centrioles in these experiments. The presence of existing centrioles inhibits the de novo formation pathway by an unknown mechanism (La Terra et al., 2005). Recruitment of MDM1 to procentrioles in late G2 could serve as a signal that duplication is complete and that further recruitment of centriole components should cease. In this model, the extra centrin foci might represent centriole material transported to the centrosomes of cells that have structurally complete centrioles but lack this “completion” signal. Another possibility is that procentrioles may have structural defects in the absence of MDM1 and that the centrin foci might be material recruited to repair procentrioles. Again, we did not observe obvious centriole structural defects in the cells with extra centrin foci observed by CLEM, suggesting that if this latter model is correct, any defects would likely be minor.Finally, we note that MDM1 was originally identified as part of an amplicon in mouse tumor cells that includes MDM2, an inhibitor of p53 that is otherwise unrelated to MDM1 (Cahilly-Snyder et al., 1987; Snyder et al., 1988). Clearly, overexpression of MDM2 provides benefits to a cancer cell by blocking p53-mediated events such as apoptosis, but it is possible that overexpression of MDM1 is also relevant to the phenotypes of cancer cells in which it is amplified. Our results show that MDM1 overexpression results both in centriole loss and microtubule stabilization, and either could be involved in the commonly observed cancer phenotypes of genome instability, altered cell motility, and invasiveness.

We previously identified Mdm1 as a gene that is transcriptionally up-regulated during the in vitro differentiation of MCCs of the mouse tracheal epithelium (Hoh et al., 2012). A mutation in Mdm1 was reported to be associated with retinal degeneration in mice, a phenotype that is common to ciliopathies (Chang et al., 2008). Here we report that MDM1 is component of centrioles and localizes close to the centriole microtubule barrel. We found that MDM1 associates with microtubules when overexpressed in cells and binds microtubules directly in vitro. Overexpression of MDM1 inhibits centriole formation, whereas depletion results in formation of structures resembling assembly intermediates. Together these data suggest that MDM1 negatively regulates centriole formation and that microtubule binding is likely to be an important part of that function.

MDM1 normally localizes only to centrioles but, when overexpressed, also associates with cytoplasmic microtubules. How might a microtubule-binding protein be limited to only a subset of microtubules? The microtubules of the centriole barrel are modified by posttranslational modifications and have a unique triplet micro­tubule structure. We note that CCSAP, a microtubule-binding protein that has repeats similar to those of MDM1, preferentially associates with polyglutamylated microtubules (Backer et al., 2012). Similarly, MDM1 might have a higher affinity for these structurally unique microtubules of the centriole or might be directed there by interaction with other proteins that do. Based on the overexpression phenotype, it is likely that MDM1 contributes to the stability of the centriole microtubules. Although no obvious centriole barrel defects were observed in MDM1-depleted cells, a number of other centriole-associated proteins, including CEP120 (Lin et al., 2013), CEP135 (Carvalho-Santos et al., 2012), CCSAP (Backer et al., 2012), and CPAP (Hsu et al., 2008), are also microtubule-binding proteins, and thus there are likely to be redundant mechanisms contributing to barrel stability.

We also identified repeats of a novel motif, (S/T)EYxxx(F/Y), which are highly conserved in MDM1 orthologues. We found that the centriole-associated microtubule-binding protein CCSAP (Backer et al., 2012) also has two copies of this motif, which are also conserved. This motif in MDM1 was important for both in vitro binding of MDM1 to microtubules and MDM1-mediated microtubule stabilization. We note that mutating the repeat sequences in MDM1 could have a nonspecific effect on protein structure, as is true for any mutant, although we consider it unlikely, given that the wt and repeat mutant forms of MDM1 behave similarly with respect to expression, solubility, and centrosome localization. It is likely that sequences outside of these repeats are also important for the microtubule associations observed, since the repeat mutant MDM1rm partially associated with microtubules both in vivo and in vitro. Most MDM1 orthologues contain at least two repeat motifs, and the microtubule association required two or more of the motifs (Figure 3F and Supplemental Figure S2, C and D). The repeats might be involved in directly binding microtubules, as observed for repeated motifs in other microtubule-binding proteins, such as TAU and MAP2 (Dehmelt and Halpain, 2005). The repeats also might be subject to posttranslational modification that modulates the interaction between MDM1 and microtubules.

Our results suggest that MDM1 is a negative regulator of centriole duplication and that MDM1-microtubule binding is likely to be important for this function. MDM1 is recruited to procentrioles near the onset of mitosis, when procentrioles finish duplication and begin a maturation process required to become parental centrioles in the next cell cycle (Wang et al., 2011). MDM1 overexpression and depletion experiments both revealed phenotypes consistent with it functioning as a negative regulator of centriole duplication. Over­expression of MDM1 blocked both centriole duplication in cycling RPE-1 cells and centriole reduplication in S phase–arrested U2OS cells. Because reduplication is believed to derive from repeated rounds of centriole disengagement and centriole formation (Loncarek et al., 2010), this suggests that MDM1 functions in the canonical duplication pathway, thus affecting both. This is in contrast to the mechanism of ORC1, which was also shown to be a negative regulator of centriole duplication and specifically blocks reduplication and not canonical duplication (Hemerly et al., 2009). Of interest, expression of MDM1rm failed to block centriole reduplication (Figure 5B) and was defective at blocking normal centriole duplication (Supplemental Figure S3D), despite the fact that it localized to centrosomes (Figure 5C). This suggests that MDM1–microtubule binding is important for the inhibition of centriole duplication caused by MDM1 overexpression. A possible mechanism for the MDM1 inhibition of duplication is that overexpressed MDM1 binds to its sites at the proximal end of centrioles and physically blocks the recruitment of a protein or complex required for duplication. This would also be consistent with the seemingly paradoxical observation that MDM1 is highly up-regulated in MTECs, which are making hundreds of centrioles; centrioles in MTECs are made by a different pathway, initiated by deuterosomes rather than preexisting centrioles (Zhao et al., 2013), and MDM1 might block duplication from newly formed centrioles in these cells. A recent study suggested that SASS6 localizes to the proximal lumen of parental centrioles at the G1/S transition and that this localization is involved in templating cartwheel formation for the new procentriole (Fong et al., 2014). It is possible that excess MDM1 interferes with the reentry of SASS6 into the parental centriole proximal lumen, in turn blocking duplication.

Depletion of MDM1 was associated with an increase in centrin foci that appeared as dense granules by EM. We also detected increased foci of CPAP, a centriole duplication protein, by immunofluorescence. Such foci are characteristic of early stages of centriole formation in the de novo centriole duplication pathway (Prosser et al., 2009), and similar centrin+/CPAP+ structures have been observed in cases in which other negative regulators of centriole duplication are depleted, such as is the case for RBM14 (Shiratsuchi et al., 2015) and NEURL4 (Li et al., 2012). We stress that we did not observe extra, bona fide centrioles in these experiments. The presence of existing centrioles inhibits the de novo formation pathway by an unknown mechanism (La Terra et al., 2005). Recruitment of MDM1 to procentrioles in late G2 could serve as a signal that duplication is complete and that further recruitment of centriole components should cease. In this model, the extra centrin foci might represent centriole material transported to the centrosomes of cells that have structurally complete centrioles but lack this “completion” signal. Another possibility is that procentrioles may have structural defects in the absence of MDM1 and that the centrin foci might be material recruited to repair procentrioles. Again, we did not observe obvious centriole structural defects in the cells with extra centrin foci observed by CLEM, suggesting that if this latter model is correct, any defects would likely be minor.

Finally, we note that MDM1 was originally identified as part of an amplicon in mouse tumor cells that includes MDM2, an inhibitor of p53 that is otherwise unrelated to MDM1 (Cahilly-Snyder et al., 1987; Snyder et al., 1988). Clearly, overexpression of MDM2 provides benefits to a cancer cell by blocking p53-mediated events such as apoptosis, but it is possible that overexpression of MDM1 is also relevant to the phenotypes of cancer cells in which it is amplified. Our results show that MDM1 overexpression results both in centriole loss and microtubule stabilization, and either could be involved in the commonly observed cancer phenotypes of genome instability, altered cell motility, and invasiveness.

MATERIALS AND METHODSPlasmidsA full-length cDNA clone of human MDM1 (GenBank accession no. BC028355.2) was obtained from Thermo Fisher Scientific (Waltham, MA). Full-length MDM1 and truncation constructs were generated by PCR amplification of the appropriate regions and cloned into pDONR221 using Gateway recombination technology (Invitrogen, Carlsbad, CA). The MDM1 (S/T)EYxxxF to AAAxxxA mutant (MDM1rm) was made by overlapping PCR using the following primers to mutate the indicated residues to code for alanine. Repeat 1 starting at CDS amino acid 9: forward primer 5′-cggtgcgcttcaaggggctgGCtgCaGCccagaggaacGCcctgtggaaaaagtcttattt-3′, reverse primer 5′-aaataagactttttccacagg­GCgttcctctggGCtGcaGCcagccccttgaagcgcaccg-3′; repeat 2 starting at CDS amino acid 189: forward primer 5′-catataatgccttga­gaaatGctgCaGCtcaaaggcagGCtgtttggaagacttctaaaga-3′, reverse primer 5′-tctttagaagtcttccaaacaGCctgcctttgaGCtGcagCatttctcaa­ggcattatatg-3′; repeat 3 starting at CDS amino acid 232: forward primer 5′-gtaactcagtcatccatgaaGctgCaGCcaaaagaaatGCcaaggg­tttatctccagtgaa-3′, reverse primer 5′-ttcactggagataaacccttgGCatttcttttgGCtGcagCttcatggatgactgagttac-3′; and repeat 4 starting at CDS amino acid 303: forward primer 5′-aaaggcttgggaaggtgaatGccgCaGCtagagcaaaaGCtctgagccc­agctcagtattt-3′, reverse primer 5′-aaatactgagctgggctcagaGCtttt­gctctaGCtGcggCattcaccttcccaagccttt-3′. Resulting PCR products were cloned into pDONR221 using Gateway recombination (Invitrogen). shRNA-resistant MDM1 (shRes MDM1) was generated by using overlapping PCR to make five synonymous base changes in the shRNA targeting region for human MDM1 shRNA. Primers used were as follows: forward primer 5′-gataatgggttggaCagGctGctAcgAaagaaagctggattgac-3′, reverse primer 5′-gtcaatccagctttcttTcgTagCagCctGtccaacccattatc-3′. Resulting PCR products were cloned into pDONR221 using Gateway recombination technology (Invitrogen). N-terminal GFP-tagged constructs were made by Gateway LR reactions between pDONR constructs and pDEST53. N-terminal GST- and maltose-binding protein (MBP)–tagged constructs for bacterial expression were made by Gateway LR reactions between pDONR constructs and pDEST15 (Invitrogen) and pKM596 (Addgene, Cambridge, MA; plasmid 8837; Fox et al., 2003), respectively.For the MDM1 shRNA construct, a 19-mer shRNA targeting human MDM1 was chosen (5′-GAUAGACUUCUGCGUAAGA-3′), and oligos against the target were designed using pSicoOligomaker 1.5. The corresponding DNA oligos were annealed and cloned into pSicoR-puro (Ventura et al., 2004) to create pTS3533 (human MDM1 shRNA). See Supplemental Table S1 for a list of all plasmids constructed for this study.AntibodiesAnti-MDM1 antibody was produced by immunizing rabbits with a purified, bacterially expressed MBP fusion protein of the N-terminal 255 amino acids of human MDM1 (Cocalico, Reamstown, PA). MDM1-specific antibodies were affinity purified from rabbit antisera on nitrocellulose blots with a GST fusion of the N-terminal 255 amino acids of human MDM1. The antibody was used at 1 μg/ml for immunofluorescence. Other antibodies used in this study for immunofluorescence include mouse anti-MDM1 (H00056890-B01; Abnova, Taipei, Taiwan) at 1:1000, rabbit anti-GFP (described previously; Hatch et al., 2010) at 1:5000, rat anti-GFP (GF090R; Nacalai USA, San Diego, CA) at 1:2000, mouse anti-CETN3 (Abnova) at 1:2000–1:4000, rabbit anti-Xenopus centrin (described previously; Hatch et al., 2010) at 1:5000, mouse anti–α-tubulin (DM1α; Sigma-Aldrich, St. Louis, MO) at 1:4000, mouse anti–polyglutamylated tubulin (GT335; Adipogen, San Diego, CA) at 1:500, mouse anti–acetylated α-tubulin (6-11B-1; Abcam, Cambridge, MA) at 1:15,000, rabbit anti–glu-tubulin (AB3201; Millipore, Billerica, MA) at 1:200, mouse anti–γ-tubulin (GTU-88; Sigma-Aldrich) at 1:5000, rabbit anti-CEP164 (described previously; Lee et al., 2014) at 1:100 (1 ng/ml), rabbit anti-CP110 (Chen et al., 2002) at 1:500, mouse anti-myc (9E10; Sigma-Aldrich) at 1:400, rabbit anti-CEP63 (Millipore) at 1:1000, rabbit anti-CEP152 (A302-480A, Bethyl Laboratories, Montgomery, TX) at 1:1000, rabbit anti-CPAP (Proteintech, Chicago, IL) at 1:1000, rabbit anti-Plx4 (described previously; Hatch et al., 2010) at 1:500 (1.5 μg/ml), and mouse anti-SASS6 (sc-81431; Santa Cruz Biotechnology, Santa Cruz, CA) at 1:200.For Western blots, the following antibodies were used: rabbit anti-MDM1 at 1:1000, mouse anti-MDM1 (H00056890-B01; Abnova) at 1:300, rabbit anti-GFP (described previously; Hatch et al., 2010) at 1:5000, goat anti-GFP (600-101-215; Rockland Immunochemicals, Limerick, PA) at 1:500, rabbit anti-p38 (C-20; Santa Cruz Biotechnology) at 1:5000, mouse anti–α-tubulin (DM1α; Sigma-Aldrich) at 1:10000, and rabbit anti-GST (affinity purified from sera described in Hatch et al. [2010]) at 1:2000.Cell culture and transfectionsHEK293T and U2OS cells were cultured in DMEM (Cellgro, Manassas, VA) supplemented with 10% fetal bovine serum (FBS; Atlanta Biologicals, Flowery Branch, GA). RPE-1 cells were cultured in DMEM/F-12 (Cellgro) supplemented with 10% FBS. For primary cilium formation experiments, cells were cultured in media supplemented with 0.5% FBS. Plasmid transfections were performed using Lipofectamine 2000 or Lipofectamine LTX according to the manufacturer’s protocol (Invitrogen). For the MDM1 colocalization with Taxol-stabilized microtubule assay, 24 h after the start of transfection, Taxol (Paclitaxel, 580555; Millipore) was added to the medium at a final concentration of 15 μM. RPE-1 cells were incubated in Taxol for 4 h before fixation. For nocodazole and cold resistance assays, RPE-1 cells were transfected and 14 h later incubated in 5 μg/ml nocodazole (N3000; US Biological, Salem, MA) or the equivalent volume of dimethyl sulfoxide (DMSO) at the indicated temperature for 1–2 h before fixation. For U2OS centriole reduplication assays, 4 mM hydroxyurea (HU; Sigma-Aldrich) was added to the medium when medium was changed 4–6 h after start of transfection. Cells were incubated in HU for 48 h before fixation.MTEC cell culture and mediaMTECs were cultured as described previously (You et al., 2002; Vladar and Stearns, 2007; Vladar and Brody, 2013). Briefly, trachea were freshly excised, opened longitudinally, exposing the lumen, and incubated for ∼18 h in 1.5 mg/ml Pronase E in F-12K nutrient mixture (Invitrogen) at 4ºC. The digested trachea samples were gently agitated to dislodge the epithelial cells, and cells were collected in F-12K supplemented with 10% FBS (Atlanta Biologicals). Cells were pelleted and incubated in 0.5 mg/ml DNase I for 5 min at 4ºC. Cells were pelleted at 400 × g for 10 min at 4ºC and then resuspended in DME/F-12 (Invitrogen) with 10% FBS and plated in a Primaria tissue culture dish (Corning, Tewskbury, MA) for 3 h and 30 min at 37ºC and 5% CO2 to allow contaminating fibroblasts to adhere to the plate. Nonadhered cells were collected, pelleted, and resuspended in MTEC Plus medium (You et al., 2002). Cells were seeded into Transwell-Clear permeable filter supports (Corning) at an approximate density of 105 cells/cm2. Cells were monitored every 2 d until they became confluent, after which ALI was created by adding MTEC serum-free medium (You et al., 2002) supplemented with 1 μM N-[N-(3,5-difluorophenacetyl)-l-alanyl]-S-phenylglycine t-butyl ester (DAPT) in the basal chamber below the filter supports. All chemicals were purchased from Sigma-Aldrich unless otherwise indicated. All media were supplemented with 100 U/ml penicillin, 100 mg/ml streptomycin, and 0.25 mg/ml Fungizone (all from Invitrogen). MTECs were fixed at indicated time points after establishing ALI.Lentivirus production and cell transductionRecombinant lentivirus expressing shRNA targeting human MDM1 (pTS3533), puromycin-resistance empty transfer vector (pSicoR PGK Puro), or GFP empty transfer vector (pSicoR CMV GFP) were made by cotransfection of HEK293T cells with the respective vectors and packaging and envelope vectors (pCMVΔR8.74 and pMD2.VSVG, respectively; Dull et al., 1998) using 1 μg/μl polyethylenimine, molecular weight 25 kDa (PEI; Polysciences, Warrington, PA). Medium was changed 6–8 h after transfection, and viral supernatant was harvested an additional 48 h later. GFP empty vector–expressing lentivirus was titered by transducing RPE-1 cells, and multiplicity of infection (MOI) was assumed to be similar for pSicoR PGK Puro–carrying lentivirus preparations. For transduction, 2 × 104 cells were plated in 12-well tissue culture plates containing DMEM/F-12 supplemented with 10% FBS and 1% penicillin/streptomycin. Cells were transduced with an approximate MOI of 1 on the following day. Medium was changed 12–16 h later, and cells were given 6–8 h to recover before being transduced for a second round. Medium was again changed 12–16 h later and replaced with fresh medium containing 4 μg/ml puromycin (Invivogen, San Diego, CA). After 2 d of puromycin selection, medium was changed to DMEM/F-12 supplemented with 10% FBS. For rescue experiments, cells were transfected 4 d after start of transduction. All cells were assayed 6 d after the start of transduction.Immunofluorescence staining and microscopyTissue culture cells were grown on poly-l-lysine–coated #1.5 glass coverslips (Electron Microscopy Sciences, Hatfield, PA) for wide-field epifluorescence and spinning-disk confocal microscopy and on high-tolerance #1.5 coverslips (Marienfeld, Lauda-Konigshofen, Germany) for 3D-SIM. Cells and MTECs were washed with phosphate-buffered saline (PBS) and fixed in −20°C methanol for 10 min. After fixation, cells were washed with PBS and extracted and blocked in PBS containing 3% bovine serum albumin (Sigma-Aldrich), 0.1% Triton X-100, and 0.02% sodium azide (PBS-BT). Coverslips were incubated with primary antibodies diluted in PBS-BT for 1 h at room temperature, washed in PBS-BT, and then incubated in Alexa Fluor dye–conjugated secondary antibodies (Invitrogen) diluted 1:1000 in PBS-BT (1:100 for far-red antibodies) at room temperature for 1 h. When applicable, appropriate isotype-specific secondary antibodies were used to distinguish different monoclonal mouse antibodies. After secondary staining, coverslips were washed in PBS-BT, and nuclei were stained by brief incubation in 4′,6-diamidino-2-phenylindole (DAPI; 1 μg/ml), followed by additional PBS-BT washes. Coverslips were mounted to glass slides using Mowiol (Polysciences) in glycerol containing 1,4,-diazobicycli-[2.2.2]-octane (Sigma-Aldrich) antifade, or SlowFade Gold (Invitrogen) for 3D-SIM.Wide-field epifluorescence images were acquired using an Axiovert 200M microscope (Carl Zeiss, Jena, Germany) with Plan-Neofluar 100×/1.3 numerical aperture (NA) and PlanApoChromat 63×/1.4 NA objectives and a cooled, charge-coupled device (CCD) camera (Orca ER; Hamamatsu Photonics, Hamamatsu, Japan). Confocal microscopy images were acquired as Z-stacks collected at 0.2-μm intervals across a 3- to 5-μm total range for tissue culture cells and 8–10 μm for MTECs on a Zeiss Axio Observer microscope (Carl Zeiss) with a confocal spinning-disk head (Yokogawa Electric Corporation, Tokyo, Japan), PlanApoChromat 63×/1.4 NA objective, and a Cascade II:512 electron-multiplying (EM) CCD camera (Photometrics, Tucson, AZ).The 3D-SIM micrographs were acquired using a DeltaVision OMX V4 BLAZE system (Applied Precision, Issaquah, WA). The microscope was equipped with a 100×/1.42 NA U-PLANAPO SIM oil immersion objective (Olympus, Tokyo, Japan), 405-, 488-, 568-, and 642-nm lasers, and three EMCCD cameras. The sequential imaging mode was used to acquire images. An electro-optical high-speed SI diffraction grating engine was used to generate SI patterns. Image stacks were composed of 15 images/plane (five phases and three angles), sectioning with a z-distance of 0.125 μm, and spanning a total distance of 4–8 μm for tissue culture cells and 8–10 μm for MTECs. The resulting images were computationally reconstructed, generating superresolution optical serial sections with twofold extended resolution in all three axes. Color channels were aligned computationally, using measurements taken with 0.1-μm multispectral fluorescent beads (Tetraspeck beads; Invitrogen) as a control. The SoftWoRx 3.7 imaging software package (Applied Precision) was used for SI reconstruction and image processing.All images were processed using Fiji (National Institutes of Health, Bethesda, MD) and/or Photoshop (Adobe, San Jose, CA).Quantitation of fluorescence intensity measurements and statistical analysesCentrosomal fluorescence intensities were quantified using maximum intensity projections of stacks acquired by spinning-disk confocal microscopy. Control and depleted cells were imaged using identical settings across all samples in an individual trial for a given antibody staining. Centrosomal regions were defined using either γ-tubulin or CEP164 as a comarker. Signal intensity was quantified using Fiji. Intensities were background corrected by measuring a region immediately adjacent to the centrosomal region and subtracting the background value from the centrosomal region value. The raw data are summarized in box-plots created in R (www.r-project.org/; Berkeley, CA) and RStudio (www.rstudio.com/about/; Boston, MA). Within each trial, the mean fluorescence intensities of the depleted cell populations were normalized to that of the control cells to adjust for experimental differences in raw data between trials.Statistical analysis was performed using either Student’s t test in Excel (Microsoft, Redmond, WA) or Fisher’s exact test in R and RStudio. Fisher’s exact test was used for comparing centriole/centrosome distributions in cases in which multiple categories were counted. Student’s t test was used for all other analyses. Error bars indicate SEM.Correlative light and electron microscopyCorrelative light and electron microscopy (CLEM) was performed as described previously (Kong et al., 2014), using centrin1-GFP RPE-1 cells depleted for MDM1. In brief, cells in Rose chambers were enclosed in an environmental chamber at 37°C and imaged on an inverted microscope (Eclipse Ti; Nikon, Tokyo, Japan) equipped with a spinning-disk confocal head (CSUX Spinning Disk; Yokogawa Electric Corporation, Tokyo, Japan) and a back-illuminated 16-μm-pixel EMCCD camera (DU897; Andor Technology, Belfast, United Kingdom). Both 100×/1.42 NA Plan Apochromat and 60×/1.45 NA Plan Apochromat total internal reflection fluorescence objective lenses were used in conjunction with a 1.5× magnifying lens. After live confocal imaging, chambers were perfused with 2.5% glutaraldehyde, and 200-nm-thick Z-sections were recorded through the entire cell volume to register the position of the centrin-GFP foci in the cells. Cell positions on coverslips were marked by a diamond scribe. Rose chambers were disassembled, and cells were washed in PBS for 30 min, followed by dehydration and staining with osmium tetroxide and uranyl acetate. Cells were embedded in Embed 812 resin. The same cell identified by light microscopy was then serially sectioned. The 80-nm-thick serial sections were transferred onto copper slot grids, stained with uranyl acetate and lead citrate, and imaged using a transmission electron microscope (H-7650; Hitachi, Tokyo, Japan). Image analysis and section alignment were performed in Photoshop and Fiji/ImageJ (National Institutes of Health). The diameter of darkly stained aggregates was measured from electron micrographs in Fiji.Microtubule cosedimentation assaysPurified bovine brain tubulin was diluted to a concentration of 2 mg/ml in BRB80 (80 mM 1,4-piperazinediethanesulfonic acid, pH 6.8, 1 mM MgCl2, 5 mM 1-palmitoyl-2-oleoyl-sn-glycero-3-phosphoethanol, 1 mM GTP, and 1 mM dithiothreitol), and precleared at 90,000 rpm (∼350,000 × g) in a TLA100 rotor for 5 min at 4ºC. Cleared tubulin was brought to a final concentration of 20 μM Taxol in a stepwise manner at 37ºC. Tubulin was incubated at room temperature overnight to polymerize. Aliquots, 0.188 nmol, of each purified GST-tagged recombinant protein were diluted in 250 final volume of BRB80 supplemented with 0.5 mg/ml bovine serum albumin and cleared at 90,000 rpm (∼350,000 × g) in a TLA100 rotor (Beckman, Indianapolis, IN) for 5 min at 4ºC. Half of each protein sample was brought to a final Taxol concentration of 20 μM and then mixed 1:1 (vol/vol) with the polymerized tubulin, and the other half was mixed 1:1 with BRB80. Volumes were increased to 480-μl total volume with BRB80 with or without Taxol for microtubule and buffer control samples, respectively. All samples were incubated 30 min at 30ºC. Eighty microliters of each sample was removed as input and diluted 1:1 with sample buffer. The remaining 400-μl volumes were loaded onto 400 μl of 40% glycerol BRB80 cushions and centrifuged at 55,000 rpm (∼250,000 × g) for 10 min in a TLS-55 rotor (Beckman). A 400-μl supernatant was pipetted off and diluted 1:1 in sample buffer. Pellets were resuspended in a final volume of 800 μl of sample buffer. Samples were boiled, and equivalent volumes were separated by SDS–PAGE for analysis.Cell extracts and Western blotsWhole-cell extracts for Western blotting were prepared by lysing cells in MECB buffer (50 mM Tris-HCl, pH 7.4, 100 mM NaCl, 2 mM MgCl2, 0.5% NP-40 substitute, 1 μg/ml each leupeptin, pepstatin, and chymostatin, and 1 mM phenylmethylsulfonyl fluoride). Protein extracts were boiled in sample buffer, resolved by SDS–PAGE, and transferred to nitrocellulose membrane (Bio-Rad, Hercules, CA). Membranes were blocked with 10% milk in Tris-buffered saline containing 0.5% Tween-20 (Sigma-Aldrich) and probed with primary antibodies overnight at 4ºC. Bound primary antibodies were detected using either secondary antibodies conjugated to IRDye 800CW or IRDye 680RD on an Odyssey CLx fluorescence imaging system according to manufacturer’s recommendations (Li-Cor, Lincoln, NE) or secondary antibodies conjugated to horseradish peroxidase (Jackson ImmunoResearch Laboratories, West Grove, PA, and Santa Cruz Biotechnology), developing the blots with SuperSignal West Pico Chemiluminescent Substrate (ThermoFisher Scientific), and exposing them to film.Flow cytometryControl and depleted RPE-1 cells were harvested and fixed in 70% ethanol at −20ºC overnight. Fixed cells were washed with PBS and stained with 40 μg/ml propidium iodide and 10 μg/ml RNaseA in PBS to stain DNA or in 10 μg/ml RNaseA in PBS as a negative control at 37ºC for 30 min. DNA content was assessed by analyzing cells using a FACSCalibur flow cytometer (BD Biosciences, San Jose, CA).Phylogenetic tree constructionUniprot IDs for sequences used to make the phylogenetic tree were as follows: A8JQV2 (Drosophila melanogaster), A7SBD8 (Nematostella vectensis), V4ASR9 (Lottia gigantea), R7UH63 (Capitella teleta), B3S729 (Trichoplax adhaerens), W4XHL8 (Strongylocentrotus purpuratus), Q5RHU7 (Danio rerio), F7BNF8 (Xenopus tropicalis), Q5ZMW6 (Gallus gallus), Q9D067 (Mus musculus), and Q8TC05 (Homo sapiens). The D. melanogaster MDM1 orthologue (CG17816) was identified by a reciprocal best-hits BLAST search starting with the L. gigantea MDM1 protein sequence. Sequences were aligned using Clustal Omega (www.ebi.ac.uk/Tools/msa/clustalo/;
Goujon et al., 2010; Sievers et al., 2011), and Interactive Tree of Life (itol.embl.de/export.cgi; Letunic and Bork, 2007, 2011) was used to draw the tree from the alignment.

MATERIALS AND METHODS

PlasmidsA full-length cDNA clone of human MDM1 (GenBank accession no. BC028355.2) was obtained from Thermo Fisher Scientific (Waltham, MA). Full-length MDM1 and truncation constructs were generated by PCR amplification of the appropriate regions and cloned into pDONR221 using Gateway recombination technology (Invitrogen, Carlsbad, CA). The MDM1 (S/T)EYxxxF to AAAxxxA mutant (MDM1rm) was made by overlapping PCR using the following primers to mutate the indicated residues to code for alanine. Repeat 1 starting at CDS amino acid 9: forward primer 5′-cggtgcgcttcaaggggctgGCtgCaGCccagaggaacGCcctgtggaaaaagtcttattt-3′, reverse primer 5′-aaataagactttttccacagg­GCgttcctctggGCtGcaGCcagccccttgaagcgcaccg-3′; repeat 2 starting at CDS amino acid 189: forward primer 5′-catataatgccttga­gaaatGctgCaGCtcaaaggcagGCtgtttggaagacttctaaaga-3′, reverse primer 5′-tctttagaagtcttccaaacaGCctgcctttgaGCtGcagCatttctcaa­ggcattatatg-3′; repeat 3 starting at CDS amino acid 232: forward primer 5′-gtaactcagtcatccatgaaGctgCaGCcaaaagaaatGCcaaggg­tttatctccagtgaa-3′, reverse primer 5′-ttcactggagataaacccttgGCatttcttttgGCtGcagCttcatggatgactgagttac-3′; and repeat 4 starting at CDS amino acid 303: forward primer 5′-aaaggcttgggaaggtgaatGccgCaGCtagagcaaaaGCtctgagccc­agctcagtattt-3′, reverse primer 5′-aaatactgagctgggctcagaGCtttt­gctctaGCtGcggCattcaccttcccaagccttt-3′. Resulting PCR products were cloned into pDONR221 using Gateway recombination (Invitrogen). shRNA-resistant MDM1 (shRes MDM1) was generated by using overlapping PCR to make five synonymous base changes in the shRNA targeting region for human MDM1 shRNA. Primers used were as follows: forward primer 5′-gataatgggttggaCagGctGctAcgAaagaaagctggattgac-3′, reverse primer 5′-gtcaatccagctttcttTcgTagCagCctGtccaacccattatc-3′. Resulting PCR products were cloned into pDONR221 using Gateway recombination technology (Invitrogen). N-terminal GFP-tagged constructs were made by Gateway LR reactions between pDONR constructs and pDEST53. N-terminal GST- and maltose-binding protein (MBP)–tagged constructs for bacterial expression were made by Gateway LR reactions between pDONR constructs and pDEST15 (Invitrogen) and pKM596 (Addgene, Cambridge, MA; plasmid 8837; Fox et al., 2003), respectively.For the MDM1 shRNA construct, a 19-mer shRNA targeting human MDM1 was chosen (5′-GAUAGACUUCUGCGUAAGA-3′), and oligos against the target were designed using pSicoOligomaker 1.5. The corresponding DNA oligos were annealed and cloned into pSicoR-puro (Ventura et al., 2004) to create pTS3533 (human MDM1 shRNA). See Supplemental Table S1 for a list of all plasmids constructed for this study.

A full-length cDNA clone of human MDM1 (GenBank accession no. BC028355.2) was obtained from Thermo Fisher Scientific (Waltham, MA). Full-length MDM1 and truncation constructs were generated by PCR amplification of the appropriate regions and cloned into pDONR221 using Gateway recombination technology (Invitrogen, Carlsbad, CA). The MDM1 (S/T)EYxxxF to AAAxxxA mutant (MDM1rm) was made by overlapping PCR using the following primers to mutate the indicated residues to code for alanine. Repeat 1 starting at CDS amino acid 9: forward primer 5′-cggtgcgcttcaaggggctgGCtgCaGCccagaggaacGCcctgtggaaaaagtcttattt-3′, reverse primer 5′-aaataagactttttccacagg­GCgttcctctggGCtGcaGCcagccccttgaagcgcaccg-3′; repeat 2 starting at CDS amino acid 189: forward primer 5′-catataatgccttga­gaaatGctgCaGCtcaaaggcagGCtgtttggaagacttctaaaga-3′, reverse primer 5′-tctttagaagtcttccaaacaGCctgcctttgaGCtGcagCatttctcaa­ggcattatatg-3′; repeat 3 starting at CDS amino acid 232: forward primer 5′-gtaactcagtcatccatgaaGctgCaGCcaaaagaaatGCcaaggg­tttatctccagtgaa-3′, reverse primer 5′-ttcactggagataaacccttgGCatttcttttgGCtGcagCttcatggatgactgagttac-3′; and repeat 4 starting at CDS amino acid 303: forward primer 5′-aaaggcttgggaaggtgaatGccgCaGCtagagcaaaaGCtctgagccc­agctcagtattt-3′, reverse primer 5′-aaatactgagctgggctcagaGCtttt­gctctaGCtGcggCattcaccttcccaagccttt-3′. Resulting PCR products were cloned into pDONR221 using Gateway recombination (Invitrogen). shRNA-resistant MDM1 (shRes MDM1) was generated by using overlapping PCR to make five synonymous base changes in the shRNA targeting region for human MDM1 shRNA. Primers used were as follows: forward primer 5′-gataatgggttggaCagGctGctAcgAaagaaagctggattgac-3′, reverse primer 5′-gtcaatccagctttcttTcgTagCagCctGtccaacccattatc-3′. Resulting PCR products were cloned into pDONR221 using Gateway recombination technology (Invitrogen). N-terminal GFP-tagged constructs were made by Gateway LR reactions between pDONR constructs and pDEST53. N-terminal GST- and maltose-binding protein (MBP)–tagged constructs for bacterial expression were made by Gateway LR reactions between pDONR constructs and pDEST15 (Invitrogen) and pKM596 (Addgene, Cambridge, MA; plasmid 8837; Fox et al., 2003), respectively.

For the MDM1 shRNA construct, a 19-mer shRNA targeting human MDM1 was chosen (5′-GAUAGACUUCUGCGUAAGA-3′), and oligos against the target were designed using pSicoOligomaker 1.5. The corresponding DNA oligos were annealed and cloned into pSicoR-puro (Ventura et al., 2004) to create pTS3533 (human MDM1 shRNA). See Supplemental Table S1 for a list of all plasmids constructed for this study.

AntibodiesAnti-MDM1 antibody was produced by immunizing rabbits with a purified, bacterially expressed MBP fusion protein of the N-terminal 255 amino acids of human MDM1 (Cocalico, Reamstown, PA). MDM1-specific antibodies were affinity purified from rabbit antisera on nitrocellulose blots with a GST fusion of the N-terminal 255 amino acids of human MDM1. The antibody was used at 1 μg/ml for immunofluorescence. Other antibodies used in this study for immunofluorescence include mouse anti-MDM1 (H00056890-B01; Abnova, Taipei, Taiwan) at 1:1000, rabbit anti-GFP (described previously; Hatch et al., 2010) at 1:5000, rat anti-GFP (GF090R; Nacalai USA, San Diego, CA) at 1:2000, mouse anti-CETN3 (Abnova) at 1:2000–1:4000, rabbit anti-Xenopus centrin (described previously; Hatch et al., 2010) at 1:5000, mouse anti–α-tubulin (DM1α; Sigma-Aldrich, St. Louis, MO) at 1:4000, mouse anti–polyglutamylated tubulin (GT335; Adipogen, San Diego, CA) at 1:500, mouse anti–acetylated α-tubulin (6-11B-1; Abcam, Cambridge, MA) at 1:15,000, rabbit anti–glu-tubulin (AB3201; Millipore, Billerica, MA) at 1:200, mouse anti–γ-tubulin (GTU-88; Sigma-Aldrich) at 1:5000, rabbit anti-CEP164 (described previously; Lee et al., 2014) at 1:100 (1 ng/ml), rabbit anti-CP110 (Chen et al., 2002) at 1:500, mouse anti-myc (9E10; Sigma-Aldrich) at 1:400, rabbit anti-CEP63 (Millipore) at 1:1000, rabbit anti-CEP152 (A302-480A, Bethyl Laboratories, Montgomery, TX) at 1:1000, rabbit anti-CPAP (Proteintech, Chicago, IL) at 1:1000, rabbit anti-Plx4 (described previously; Hatch et al., 2010) at 1:500 (1.5 μg/ml), and mouse anti-SASS6 (sc-81431; Santa Cruz Biotechnology, Santa Cruz, CA) at 1:200.For Western blots, the following antibodies were used: rabbit anti-MDM1 at 1:1000, mouse anti-MDM1 (H00056890-B01; Abnova) at 1:300, rabbit anti-GFP (described previously; Hatch et al., 2010) at 1:5000, goat anti-GFP (600-101-215; Rockland Immunochemicals, Limerick, PA) at 1:500, rabbit anti-p38 (C-20; Santa Cruz Biotechnology) at 1:5000, mouse anti–α-tubulin (DM1α; Sigma-Aldrich) at 1:10000, and rabbit anti-GST (affinity purified from sera described in Hatch et al. [2010]) at 1:2000.

Anti-MDM1 antibody was produced by immunizing rabbits with a purified, bacterially expressed MBP fusion protein of the N-terminal 255 amino acids of human MDM1 (Cocalico, Reamstown, PA). MDM1-specific antibodies were affinity purified from rabbit antisera on nitrocellulose blots with a GST fusion of the N-terminal 255 amino acids of human MDM1. The antibody was used at 1 μg/ml for immunofluorescence. Other antibodies used in this study for immunofluorescence include mouse anti-MDM1 (H00056890-B01; Abnova, Taipei, Taiwan) at 1:1000, rabbit anti-GFP (described previously; Hatch et al., 2010) at 1:5000, rat anti-GFP (GF090R; Nacalai USA, San Diego, CA) at 1:2000, mouse anti-CETN3 (Abnova) at 1:2000–1:4000, rabbit anti-Xenopus centrin (described previously; Hatch et al., 2010) at 1:5000, mouse anti–α-tubulin (DM1α; Sigma-Aldrich, St. Louis, MO) at 1:4000, mouse anti–polyglutamylated tubulin (GT335; Adipogen, San Diego, CA) at 1:500, mouse anti–acetylated α-tubulin (6-11B-1; Abcam, Cambridge, MA) at 1:15,000, rabbit anti–glu-tubulin (AB3201; Millipore, Billerica, MA) at 1:200, mouse anti–γ-tubulin (GTU-88; Sigma-Aldrich) at 1:5000, rabbit anti-CEP164 (described previously; Lee et al., 2014) at 1:100 (1 ng/ml), rabbit anti-CP110 (Chen et al., 2002) at 1:500, mouse anti-myc (9E10; Sigma-Aldrich) at 1:400, rabbit anti-CEP63 (Millipore) at 1:1000, rabbit anti-CEP152 (A302-480A, Bethyl Laboratories, Montgomery, TX) at 1:1000, rabbit anti-CPAP (Proteintech, Chicago, IL) at 1:1000, rabbit anti-Plx4 (described previously; Hatch et al., 2010) at 1:500 (1.5 μg/ml), and mouse anti-SASS6 (sc-81431; Santa Cruz Biotechnology, Santa Cruz, CA) at 1:200.

For Western blots, the following antibodies were used: rabbit anti-MDM1 at 1:1000, mouse anti-MDM1 (H00056890-B01; Abnova) at 1:300, rabbit anti-GFP (described previously; Hatch et al., 2010) at 1:5000, goat anti-GFP (600-101-215; Rockland Immunochemicals, Limerick, PA) at 1:500, rabbit anti-p38 (C-20; Santa Cruz Biotechnology) at 1:5000, mouse anti–α-tubulin (DM1α; Sigma-Aldrich) at 1:10000, and rabbit anti-GST (affinity purified from sera described in Hatch et al. [2010]) at 1:2000.

Cell culture and transfectionsHEK293T and U2OS cells were cultured in DMEM (Cellgro, Manassas, VA) supplemented with 10% fetal bovine serum (FBS; Atlanta Biologicals, Flowery Branch, GA). RPE-1 cells were cultured in DMEM/F-12 (Cellgro) supplemented with 10% FBS. For primary cilium formation experiments, cells were cultured in media supplemented with 0.5% FBS. Plasmid transfections were performed using Lipofectamine 2000 or Lipofectamine LTX according to the manufacturer’s protocol (Invitrogen). For the MDM1 colocalization with Taxol-stabilized microtubule assay, 24 h after the start of transfection, Taxol (Paclitaxel, 580555; Millipore) was added to the medium at a final concentration of 15 μM. RPE-1 cells were incubated in Taxol for 4 h before fixation. For nocodazole and cold resistance assays, RPE-1 cells were transfected and 14 h later incubated in 5 μg/ml nocodazole (N3000; US Biological, Salem, MA) or the equivalent volume of dimethyl sulfoxide (DMSO) at the indicated temperature for 1–2 h before fixation. For U2OS centriole reduplication assays, 4 mM hydroxyurea (HU; Sigma-Aldrich) was added to the medium when medium was changed 4–6 h after start of transfection. Cells were incubated in HU for 48 h before fixation.

Cell culture and transfections

HEK293T and U2OS cells were cultured in DMEM (Cellgro, Manassas, VA) supplemented with 10% fetal bovine serum (FBS; Atlanta Biologicals, Flowery Branch, GA). RPE-1 cells were cultured in DMEM/F-12 (Cellgro) supplemented with 10% FBS. For primary cilium formation experiments, cells were cultured in media supplemented with 0.5% FBS. Plasmid transfections were performed using Lipofectamine 2000 or Lipofectamine LTX according to the manufacturer’s protocol (Invitrogen). For the MDM1 colocalization with Taxol-stabilized microtubule assay, 24 h after the start of transfection, Taxol (Paclitaxel, 580555; Millipore) was added to the medium at a final concentration of 15 μM. RPE-1 cells were incubated in Taxol for 4 h before fixation. For nocodazole and cold resistance assays, RPE-1 cells were transfected and 14 h later incubated in 5 μg/ml nocodazole (N3000; US Biological, Salem, MA) or the equivalent volume of dimethyl sulfoxide (DMSO) at the indicated temperature for 1–2 h before fixation. For U2OS centriole reduplication assays, 4 mM hydroxyurea (HU; Sigma-Aldrich) was added to the medium when medium was changed 4–6 h after start of transfection. Cells were incubated in HU for 48 h before fixation.

MTEC cell culture and mediaMTECs were cultured as described previously (You et al., 2002; Vladar and Stearns, 2007; Vladar and Brody, 2013). Briefly, trachea were freshly excised, opened longitudinally, exposing the lumen, and incubated for ∼18 h in 1.5 mg/ml Pronase E in F-12K nutrient mixture (Invitrogen) at 4ºC. The digested trachea samples were gently agitated to dislodge the epithelial cells, and cells were collected in F-12K supplemented with 10% FBS (Atlanta Biologicals). Cells were pelleted and incubated in 0.5 mg/ml DNase I for 5 min at 4ºC. Cells were pelleted at 400 × g for 10 min at 4ºC and then resuspended in DME/F-12 (Invitrogen) with 10% FBS and plated in a Primaria tissue culture dish (Corning, Tewskbury, MA) for 3 h and 30 min at 37ºC and 5% CO2 to allow contaminating fibroblasts to adhere to the plate. Nonadhered cells were collected, pelleted, and resuspended in MTEC Plus medium (You et al., 2002). Cells were seeded into Transwell-Clear permeable filter supports (Corning) at an approximate density of 105 cells/cm2. Cells were monitored every 2 d until they became confluent, after which ALI was created by adding MTEC serum-free medium (You et al., 2002) supplemented with 1 μM N-[N-(3,5-difluorophenacetyl)-l-alanyl]-S-phenylglycine t-butyl ester (DAPT) in the basal chamber below the filter supports. All chemicals were purchased from Sigma-Aldrich unless otherwise indicated. All media were supplemented with 100 U/ml penicillin, 100 mg/ml streptomycin, and 0.25 mg/ml Fungizone (all from Invitrogen). MTECs were fixed at indicated time points after establishing ALI.

MTEC cell culture and media

MTECs were cultured as described previously (You et al., 2002; Vladar and Stearns, 2007; Vladar and Brody, 2013). Briefly, trachea were freshly excised, opened longitudinally, exposing the lumen, and incubated for ∼18 h in 1.5 mg/ml Pronase E in F-12K nutrient mixture (Invitrogen) at 4ºC. The digested trachea samples were gently agitated to dislodge the epithelial cells, and cells were collected in F-12K supplemented with 10% FBS (Atlanta Biologicals). Cells were pelleted and incubated in 0.5 mg/ml DNase I for 5 min at 4ºC. Cells were pelleted at 400 × g for 10 min at 4ºC and then resuspended in DME/F-12 (Invitrogen) with 10% FBS and plated in a Primaria tissue culture dish (Corning, Tewskbury, MA) for 3 h and 30 min at 37ºC and 5% CO2 to allow contaminating fibroblasts to adhere to the plate. Nonadhered cells were collected, pelleted, and resuspended in MTEC Plus medium (You et al., 2002). Cells were seeded into Transwell-Clear permeable filter supports (Corning) at an approximate density of 105 cells/cm2. Cells were monitored every 2 d until they became confluent, after which ALI was created by adding MTEC serum-free medium (You et al., 2002) supplemented with 1 μM N-[N-(3,5-difluorophenacetyl)-l-alanyl]-S-phenylglycine t-butyl ester (DAPT) in the basal chamber below the filter supports. All chemicals were purchased from Sigma-Aldrich unless otherwise indicated. All media were supplemented with 100 U/ml penicillin, 100 mg/ml streptomycin, and 0.25 mg/ml Fungizone (all from Invitrogen). MTECs were fixed at indicated time points after establishing ALI.

Lentivirus production and cell transductionRecombinant lentivirus expressing shRNA targeting human MDM1 (pTS3533), puromycin-resistance empty transfer vector (pSicoR PGK Puro), or GFP empty transfer vector (pSicoR CMV GFP) were made by cotransfection of HEK293T cells with the respective vectors and packaging and envelope vectors (pCMVΔR8.74 and pMD2.VSVG, respectively; Dull et al., 1998) using 1 μg/μl polyethylenimine, molecular weight 25 kDa (PEI; Polysciences, Warrington, PA). Medium was changed 6–8 h after transfection, and viral supernatant was harvested an additional 48 h later. GFP empty vector–expressing lentivirus was titered by transducing RPE-1 cells, and multiplicity of infection (MOI) was assumed to be similar for pSicoR PGK Puro–carrying lentivirus preparations. For transduction, 2 × 104 cells were plated in 12-well tissue culture plates containing DMEM/F-12 supplemented with 10% FBS and 1% penicillin/streptomycin. Cells were transduced with an approximate MOI of 1 on the following day. Medium was changed 12–16 h later, and cells were given 6–8 h to recover before being transduced for a second round. Medium was again changed 12–16 h later and replaced with fresh medium containing 4 μg/ml puromycin (Invivogen, San Diego, CA). After 2 d of puromycin selection, medium was changed to DMEM/F-12 supplemented with 10% FBS. For rescue experiments, cells were transfected 4 d after start of transduction. All cells were assayed 6 d after the start of transduction.

Lentivirus production and cell transduction

Recombinant lentivirus expressing shRNA targeting human MDM1 (pTS3533), puromycin-resistance empty transfer vector (pSicoR PGK Puro), or GFP empty transfer vector (pSicoR CMV GFP) were made by cotransfection of HEK293T cells with the respective vectors and packaging and envelope vectors (pCMVΔR8.74 and pMD2.VSVG, respectively; Dull et al., 1998) using 1 μg/μl polyethylenimine, molecular weight 25 kDa (PEI; Polysciences, Warrington, PA). Medium was changed 6–8 h after transfection, and viral supernatant was harvested an additional 48 h later. GFP empty vector–expressing lentivirus was titered by transducing RPE-1 cells, and multiplicity of infection (MOI) was assumed to be similar for pSicoR PGK Puro–carrying lentivirus preparations. For transduction, 2 × 104 cells were plated in 12-well tissue culture plates containing DMEM/F-12 supplemented with 10% FBS and 1% penicillin/streptomycin. Cells were transduced with an approximate MOI of 1 on the following day. Medium was changed 12–16 h later, and cells were given 6–8 h to recover before being transduced for a second round. Medium was again changed 12–16 h later and replaced with fresh medium containing 4 μg/ml puromycin (Invivogen, San Diego, CA). After 2 d of puromycin selection, medium was changed to DMEM/F-12 supplemented with 10% FBS. For rescue experiments, cells were transfected 4 d after start of transduction. All cells were assayed 6 d after the start of transduction.

Immunofluorescence staining and microscopyTissue culture cells were grown on poly-l-lysine–coated #1.5 glass coverslips (Electron Microscopy Sciences, Hatfield, PA) for wide-field epifluorescence and spinning-disk confocal microscopy and on high-tolerance #1.5 coverslips (Marienfeld, Lauda-Konigshofen, Germany) for 3D-SIM. Cells and MTECs were washed with phosphate-buffered saline (PBS) and fixed in −20°C methanol for 10 min. After fixation, cells were washed with PBS and extracted and blocked in PBS containing 3% bovine serum albumin (Sigma-Aldrich), 0.1% Triton X-100, and 0.02% sodium azide (PBS-BT). Coverslips were incubated with primary antibodies diluted in PBS-BT for 1 h at room temperature, washed in PBS-BT, and then incubated in Alexa Fluor dye–conjugated secondary antibodies (Invitrogen) diluted 1:1000 in PBS-BT (1:100 for far-red antibodies) at room temperature for 1 h. When applicable, appropriate isotype-specific secondary antibodies were used to distinguish different monoclonal mouse antibodies. After secondary staining, coverslips were washed in PBS-BT, and nuclei were stained by brief incubation in 4′,6-diamidino-2-phenylindole (DAPI; 1 μg/ml), followed by additional PBS-BT washes. Coverslips were mounted to glass slides using Mowiol (Polysciences) in glycerol containing 1,4,-diazobicycli-[2.2.2]-octane (Sigma-Aldrich) antifade, or SlowFade Gold (Invitrogen) for 3D-SIM.Wide-field epifluorescence images were acquired using an Axiovert 200M microscope (Carl Zeiss, Jena, Germany) with Plan-Neofluar 100×/1.3 numerical aperture (NA) and PlanApoChromat 63×/1.4 NA objectives and a cooled, charge-coupled device (CCD) camera (Orca ER; Hamamatsu Photonics, Hamamatsu, Japan). Confocal microscopy images were acquired as Z-stacks collected at 0.2-μm intervals across a 3- to 5-μm total range for tissue culture cells and 8–10 μm for MTECs on a Zeiss Axio Observer microscope (Carl Zeiss) with a confocal spinning-disk head (Yokogawa Electric Corporation, Tokyo, Japan), PlanApoChromat 63×/1.4 NA objective, and a Cascade II:512 electron-multiplying (EM) CCD camera (Photometrics, Tucson, AZ).The 3D-SIM micrographs were acquired using a DeltaVision OMX V4 BLAZE system (Applied Precision, Issaquah, WA). The microscope was equipped with a 100×/1.42 NA U-PLANAPO SIM oil immersion objective (Olympus, Tokyo, Japan), 405-, 488-, 568-, and 642-nm lasers, and three EMCCD cameras. The sequential imaging mode was used to acquire images. An electro-optical high-speed SI diffraction grating engine was used to generate SI patterns. Image stacks were composed of 15 images/plane (five phases and three angles), sectioning with a z-distance of 0.125 μm, and spanning a total distance of 4–8 μm for tissue culture cells and 8–10 μm for MTECs. The resulting images were computationally reconstructed, generating superresolution optical serial sections with twofold extended resolution in all three axes. Color channels were aligned computationally, using measurements taken with 0.1-μm multispectral fluorescent beads (Tetraspeck beads; Invitrogen) as a control. The SoftWoRx 3.7 imaging software package (Applied Precision) was used for SI reconstruction and image processing.All images were processed using Fiji (National Institutes of Health, Bethesda, MD) and/or Photoshop (Adobe, San Jose, CA).

Immunofluorescence staining and microscopy

Tissue culture cells were grown on poly-l-lysine–coated #1.5 glass coverslips (Electron Microscopy Sciences, Hatfield, PA) for wide-field epifluorescence and spinning-disk confocal microscopy and on high-tolerance #1.5 coverslips (Marienfeld, Lauda-Konigshofen, Germany) for 3D-SIM. Cells and MTECs were washed with phosphate-buffered saline (PBS) and fixed in −20°C methanol for 10 min. After fixation, cells were washed with PBS and extracted and blocked in PBS containing 3% bovine serum albumin (Sigma-Aldrich), 0.1% Triton X-100, and 0.02% sodium azide (PBS-BT). Coverslips were incubated with primary antibodies diluted in PBS-BT for 1 h at room temperature, washed in PBS-BT, and then incubated in Alexa Fluor dye–conjugated secondary antibodies (Invitrogen) diluted 1:1000 in PBS-BT (1:100 for far-red antibodies) at room temperature for 1 h. When applicable, appropriate isotype-specific secondary antibodies were used to distinguish different monoclonal mouse antibodies. After secondary staining, coverslips were washed in PBS-BT, and nuclei were stained by brief incubation in 4′,6-diamidino-2-phenylindole (DAPI; 1 μg/ml), followed by additional PBS-BT washes. Coverslips were mounted to glass slides using Mowiol (Polysciences) in glycerol containing 1,4,-diazobicycli-[2.2.2]-octane (Sigma-Aldrich) antifade, or SlowFade Gold (Invitrogen) for 3D-SIM.

Wide-field epifluorescence images were acquired using an Axiovert 200M microscope (Carl Zeiss, Jena, Germany) with Plan-Neofluar 100×/1.3 numerical aperture (NA) and PlanApoChromat 63×/1.4 NA objectives and a cooled, charge-coupled device (CCD) camera (Orca ER; Hamamatsu Photonics, Hamamatsu, Japan). Confocal microscopy images were acquired as Z-stacks collected at 0.2-μm intervals across a 3- to 5-μm total range for tissue culture cells and 8–10 μm for MTECs on a Zeiss Axio Observer microscope (Carl Zeiss) with a confocal spinning-disk head (Yokogawa Electric Corporation, Tokyo, Japan), PlanApoChromat 63×/1.4 NA objective, and a Cascade II:512 electron-multiplying (EM) CCD camera (Photometrics, Tucson, AZ).

The 3D-SIM micrographs were acquired using a DeltaVision OMX V4 BLAZE system (Applied Precision, Issaquah, WA). The microscope was equipped with a 100×/1.42 NA U-PLANAPO SIM oil immersion objective (Olympus, Tokyo, Japan), 405-, 488-, 568-, and 642-nm lasers, and three EMCCD cameras. The sequential imaging mode was used to acquire images. An electro-optical high-speed SI diffraction grating engine was used to generate SI patterns. Image stacks were composed of 15 images/plane (five phases and three angles), sectioning with a z-distance of 0.125 μm, and spanning a total distance of 4–8 μm for tissue culture cells and 8–10 μm for MTECs. The resulting images were computationally reconstructed, generating superresolution optical serial sections with twofold extended resolution in all three axes. Color channels were aligned computationally, using measurements taken with 0.1-μm multispectral fluorescent beads (Tetraspeck beads; Invitrogen) as a control. The SoftWoRx 3.7 imaging software package (Applied Precision) was used for SI reconstruction and image processing.

All images were processed using Fiji (National Institutes of Health, Bethesda, MD) and/or Photoshop (Adobe, San Jose, CA).

Quantitation of fluorescence intensity measurements and statistical analysesCentrosomal fluorescence intensities were quantified using maximum intensity projections of stacks acquired by spinning-disk confocal microscopy. Control and depleted cells were imaged using identical settings across all samples in an individual trial for a given antibody staining. Centrosomal regions were defined using either γ-tubulin or CEP164 as a comarker. Signal intensity was quantified using Fiji. Intensities were background corrected by measuring a region immediately adjacent to the centrosomal region and subtracting the background value from the centrosomal region value. The raw data are summarized in box-plots created in R (www.r-project.org/; Berkeley, CA) and RStudio (www.rstudio.com/about/; Boston, MA). Within each trial, the mean fluorescence intensities of the depleted cell populations were normalized to that of the control cells to adjust for experimental differences in raw data between trials.Statistical analysis was performed using either Student’s t test in Excel (Microsoft, Redmond, WA) or Fisher’s exact test in R and RStudio. Fisher’s exact test was used for comparing centriole/centrosome distributions in cases in which multiple categories were counted. Student’s t test was used for all other analyses. Error bars indicate SEM.

Quantitation of fluorescence intensity measurements and statistical analyses

Centrosomal fluorescence intensities were quantified using maximum intensity projections of stacks acquired by spinning-disk confocal microscopy. Control and depleted cells were imaged using identical settings across all samples in an individual trial for a given antibody staining. Centrosomal regions were defined using either γ-tubulin or CEP164 as a comarker. Signal intensity was quantified using Fiji. Intensities were background corrected by measuring a region immediately adjacent to the centrosomal region and subtracting the background value from the centrosomal region value. The raw data are summarized in box-plots created in R (www.r-project.org/; Berkeley, CA) and RStudio (www.rstudio.com/about/; Boston, MA). Within each trial, the mean fluorescence intensities of the depleted cell populations were normalized to that of the control cells to adjust for experimental differences in raw data between trials.

Statistical analysis was performed using either Student’s t test in Excel (Microsoft, Redmond, WA) or Fisher’s exact test in R and RStudio. Fisher’s exact test was used for comparing centriole/centrosome distributions in cases in which multiple categories were counted. Student’s t test was used for all other analyses. Error bars indicate SEM.

Correlative light and electron microscopyCorrelative light and electron microscopy (CLEM) was performed as described previously (Kong et al., 2014), using centrin1-GFP RPE-1 cells depleted for MDM1. In brief, cells in Rose chambers were enclosed in an environmental chamber at 37°C and imaged on an inverted microscope (Eclipse Ti; Nikon, Tokyo, Japan) equipped with a spinning-disk confocal head (CSUX Spinning Disk; Yokogawa Electric Corporation, Tokyo, Japan) and a back-illuminated 16-μm-pixel EMCCD camera (DU897; Andor Technology, Belfast, United Kingdom). Both 100×/1.42 NA Plan Apochromat and 60×/1.45 NA Plan Apochromat total internal reflection fluorescence objective lenses were used in conjunction with a 1.5× magnifying lens. After live confocal imaging, chambers were perfused with 2.5% glutaraldehyde, and 200-nm-thick Z-sections were recorded through the entire cell volume to register the position of the centrin-GFP foci in the cells. Cell positions on coverslips were marked by a diamond scribe. Rose chambers were disassembled, and cells were washed in PBS for 30 min, followed by dehydration and staining with osmium tetroxide and uranyl acetate. Cells were embedded in Embed 812 resin. The same cell identified by light microscopy was then serially sectioned. The 80-nm-thick serial sections were transferred onto copper slot grids, stained with uranyl acetate and lead citrate, and imaged using a transmission electron microscope (H-7650; Hitachi, Tokyo, Japan). Image analysis and section alignment were performed in Photoshop and Fiji/ImageJ (National Institutes of Health). The diameter of darkly stained aggregates was measured from electron micrographs in Fiji.

Correlative light and electron microscopy

Correlative light and electron microscopy (CLEM) was performed as described previously (Kong et al., 2014), using centrin1-GFP RPE-1 cells depleted for MDM1. In brief, cells in Rose chambers were enclosed in an environmental chamber at 37°C and imaged on an inverted microscope (Eclipse Ti; Nikon, Tokyo, Japan) equipped with a spinning-disk confocal head (CSUX Spinning Disk; Yokogawa Electric Corporation, Tokyo, Japan) and a back-illuminated 16-μm-pixel EMCCD camera (DU897; Andor Technology, Belfast, United Kingdom). Both 100×/1.42 NA Plan Apochromat and 60×/1.45 NA Plan Apochromat total internal reflection fluorescence objective lenses were used in conjunction with a 1.5× magnifying lens. After live confocal imaging, chambers were perfused with 2.5% glutaraldehyde, and 200-nm-thick Z-sections were recorded through the entire cell volume to register the position of the centrin-GFP foci in the cells. Cell positions on coverslips were marked by a diamond scribe. Rose chambers were disassembled, and cells were washed in PBS for 30 min, followed by dehydration and staining with osmium tetroxide and uranyl acetate. Cells were embedded in Embed 812 resin. The same cell identified by light microscopy was then serially sectioned. The 80-nm-thick serial sections were transferred onto copper slot grids, stained with uranyl acetate and lead citrate, and imaged using a transmission electron microscope (H-7650; Hitachi, Tokyo, Japan). Image analysis and section alignment were performed in Photoshop and Fiji/ImageJ (National Institutes of Health). The diameter of darkly stained aggregates was measured from electron micrographs in Fiji.

Microtubule cosedimentation assaysPurified bovine brain tubulin was diluted to a concentration of 2 mg/ml in BRB80 (80 mM 1,4-piperazinediethanesulfonic acid, pH 6.8, 1 mM MgCl2, 5 mM 1-palmitoyl-2-oleoyl-sn-glycero-3-phosphoethanol, 1 mM GTP, and 1 mM dithiothreitol), and precleared at 90,000 rpm (∼350,000 × g) in a TLA100 rotor for 5 min at 4ºC. Cleared tubulin was brought to a final concentration of 20 μM Taxol in a stepwise manner at 37ºC. Tubulin was incubated at room temperature overnight to polymerize. Aliquots, 0.188 nmol, of each purified GST-tagged recombinant protein were diluted in 250 final volume of BRB80 supplemented with 0.5 mg/ml bovine serum albumin and cleared at 90,000 rpm (∼350,000 × g) in a TLA100 rotor (Beckman, Indianapolis, IN) for 5 min at 4ºC. Half of each protein sample was brought to a final Taxol concentration of 20 μM and then mixed 1:1 (vol/vol) with the polymerized tubulin, and the other half was mixed 1:1 with BRB80. Volumes were increased to 480-μl total volume with BRB80 with or without Taxol for microtubule and buffer control samples, respectively. All samples were incubated 30 min at 30ºC. Eighty microliters of each sample was removed as input and diluted 1:1 with sample buffer. The remaining 400-μl volumes were loaded onto 400 μl of 40% glycerol BRB80 cushions and centrifuged at 55,000 rpm (∼250,000 × g) for 10 min in a TLS-55 rotor (Beckman). A 400-μl supernatant was pipetted off and diluted 1:1 in sample buffer. Pellets were resuspended in a final volume of 800 μl of sample buffer. Samples were boiled, and equivalent volumes were separated by SDS–PAGE for analysis.

Microtubule cosedimentation assays

Purified bovine brain tubulin was diluted to a concentration of 2 mg/ml in BRB80 (80 mM 1,4-piperazinediethanesulfonic acid, pH 6.8, 1 mM MgCl2, 5 mM 1-palmitoyl-2-oleoyl-sn-glycero-3-phosphoethanol, 1 mM GTP, and 1 mM dithiothreitol), and precleared at 90,000 rpm (∼350,000 × g) in a TLA100 rotor for 5 min at 4ºC. Cleared tubulin was brought to a final concentration of 20 μM Taxol in a stepwise manner at 37ºC. Tubulin was incubated at room temperature overnight to polymerize. Aliquots, 0.188 nmol, of each purified GST-tagged recombinant protein were diluted in 250 final volume of BRB80 supplemented with 0.5 mg/ml bovine serum albumin and cleared at 90,000 rpm (∼350,000 × g) in a TLA100 rotor (Beckman, Indianapolis, IN) for 5 min at 4ºC. Half of each protein sample was brought to a final Taxol concentration of 20 μM and then mixed 1:1 (vol/vol) with the polymerized tubulin, and the other half was mixed 1:1 with BRB80. Volumes were increased to 480-μl total volume with BRB80 with or without Taxol for microtubule and buffer control samples, respectively. All samples were incubated 30 min at 30ºC. Eighty microliters of each sample was removed as input and diluted 1:1 with sample buffer. The remaining 400-μl volumes were loaded onto 400 μl of 40% glycerol BRB80 cushions and centrifuged at 55,000 rpm (∼250,000 × g) for 10 min in a TLS-55 rotor (Beckman). A 400-μl supernatant was pipetted off and diluted 1:1 in sample buffer. Pellets were resuspended in a final volume of 800 μl of sample buffer. Samples were boiled, and equivalent volumes were separated by SDS–PAGE for analysis.

Cell extracts and Western blotsWhole-cell extracts for Western blotting were prepared by lysing cells in MECB buffer (50 mM Tris-HCl, pH 7.4, 100 mM NaCl, 2 mM MgCl2, 0.5% NP-40 substitute, 1 μg/ml each leupeptin, pepstatin, and chymostatin, and 1 mM phenylmethylsulfonyl fluoride). Protein extracts were boiled in sample buffer, resolved by SDS–PAGE, and transferred to nitrocellulose membrane (Bio-Rad, Hercules, CA). Membranes were blocked with 10% milk in Tris-buffered saline containing 0.5% Tween-20 (Sigma-Aldrich) and probed with primary antibodies overnight at 4ºC. Bound primary antibodies were detected using either secondary antibodies conjugated to IRDye 800CW or IRDye 680RD on an Odyssey CLx fluorescence imaging system according to manufacturer’s recommendations (Li-Cor, Lincoln, NE) or secondary antibodies conjugated to horseradish peroxidase (Jackson ImmunoResearch Laboratories, West Grove, PA, and Santa Cruz Biotechnology), developing the blots with SuperSignal West Pico Chemiluminescent Substrate (ThermoFisher Scientific), and exposing them to film.

Cell extracts and Western blots

Whole-cell extracts for Western blotting were prepared by lysing cells in MECB buffer (50 mM Tris-HCl, pH 7.4, 100 mM NaCl, 2 mM MgCl2, 0.5% NP-40 substitute, 1 μg/ml each leupeptin, pepstatin, and chymostatin, and 1 mM phenylmethylsulfonyl fluoride). Protein extracts were boiled in sample buffer, resolved by SDS–PAGE, and transferred to nitrocellulose membrane (Bio-Rad, Hercules, CA). Membranes were blocked with 10% milk in Tris-buffered saline containing 0.5% Tween-20 (Sigma-Aldrich) and probed with primary antibodies overnight at 4ºC. Bound primary antibodies were detected using either secondary antibodies conjugated to IRDye 800CW or IRDye 680RD on an Odyssey CLx fluorescence imaging system according to manufacturer’s recommendations (Li-Cor, Lincoln, NE) or secondary antibodies conjugated to horseradish peroxidase (Jackson ImmunoResearch Laboratories, West Grove, PA, and Santa Cruz Biotechnology), developing the blots with SuperSignal West Pico Chemiluminescent Substrate (ThermoFisher Scientific), and exposing them to film.

Flow cytometryControl and depleted RPE-1 cells were harvested and fixed in 70% ethanol at −20ºC overnight. Fixed cells were washed with PBS and stained with 40 μg/ml propidium iodide and 10 μg/ml RNaseA in PBS to stain DNA or in 10 μg/ml RNaseA in PBS as a negative control at 37ºC for 30 min. DNA content was assessed by analyzing cells using a FACSCalibur flow cytometer (BD Biosciences, San Jose, CA).

Flow cytometry

Control and depleted RPE-1 cells were harvested and fixed in 70% ethanol at −20ºC overnight. Fixed cells were washed with PBS and stained with 40 μg/ml propidium iodide and 10 μg/ml RNaseA in PBS to stain DNA or in 10 μg/ml RNaseA in PBS as a negative control at 37ºC for 30 min. DNA content was assessed by analyzing cells using a FACSCalibur flow cytometer (BD Biosciences, San Jose, CA).

Phylogenetic tree constructionUniprot IDs for sequences used to make the phylogenetic tree were as follows: A8JQV2 (Drosophila melanogaster), A7SBD8 (Nematostella vectensis), V4ASR9 (Lottia gigantea), R7UH63 (Capitella teleta), B3S729 (Trichoplax adhaerens), W4XHL8 (Strongylocentrotus purpuratus), Q5RHU7 (Danio rerio), F7BNF8 (Xenopus tropicalis), Q5ZMW6 (Gallus gallus), Q9D067 (Mus musculus), and Q8TC05 (Homo sapiens). The D. melanogaster MDM1 orthologue (CG17816) was identified by a reciprocal best-hits BLAST search starting with the L. gigantea MDM1 protein sequence. Sequences were aligned using Clustal Omega (www.ebi.ac.uk/Tools/msa/clustalo/;
Goujon et al., 2010; Sievers et al., 2011), and Interactive Tree of Life (itol.embl.de/export.cgi; Letunic and Bork, 2007, 2011) was used to draw the tree from the alignment.

Phylogenetic tree construction

Uniprot IDs for sequences used to make the phylogenetic tree were as follows: A8JQV2 (Drosophila melanogaster), A7SBD8 (Nematostella vectensis), V4ASR9 (Lottia gigantea), R7UH63 (Capitella teleta), B3S729 (Trichoplax adhaerens), W4XHL8 (Strongylocentrotus purpuratus), Q5RHU7 (Danio rerio), F7BNF8 (Xenopus tropicalis), Q5ZMW6 (Gallus gallus), Q9D067 (Mus musculus), and Q8TC05 (Homo sapiens). The D. melanogaster MDM1 orthologue (CG17816) was identified by a reciprocal best-hits BLAST search starting with the L. gigantea MDM1 protein sequence. Sequences were aligned using Clustal Omega (www.ebi.ac.uk/Tools/msa/clustalo/;
Goujon et al., 2010; Sievers et al., 2011), and Interactive Tree of Life (itol.embl.de/export.cgi; Letunic and Bork, 2007, 2011) was used to draw the tree from the alignment.

Supplementary MaterialSupplemental Materials

Supplementary Material

Supplemental Materials
