---
pmid: '12186849'
title: A survival pathway for Caenorhabditis elegans with a blocked unfolded protein
  response.
authors:
- Urano F
- Calfon M
- Yoneda T
- Yun C
- Kiraly M
- Clark SG
- Ron D
journal: J Cell Biol
year: '2002'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2174003
doi: 10.1083/jcb.200203086
---

# A survival pathway for Caenorhabditis elegans with a blocked unfolded protein response.
**Authors:** Urano F, Calfon M, Yoneda T, Yun C, Kiraly M, Clark SG, Ron D
**Journal:** J Cell Biol (2002)
**DOI:** [10.1083/jcb.200203086](https://doi.org/10.1083/jcb.200203086)
**PMC:** [PMC2174003](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2174003/)

## Abstract

1. J Cell Biol. 2002 Aug 19;158(4):639-46. doi: 10.1083/jcb.200203086. Epub 2002 
Aug 19.

A survival pathway for Caenorhabditis elegans with a blocked unfolded protein 
response.

Urano F(1), Calfon M, Yoneda T, Yun C, Kiraly M, Clark SG, Ron D.

Author information:
(1)Skirball Institute, New York University School of Medicine, 540 First Avenue, 
New York, NY 10016, USA.

The unfolded protein response (UPR) counteracts stress caused by unprocessed ER 
client proteins. A genome-wide survey showed impaired induction of many UPR 
target genes in xbp-1 mutant Caenorhabditis elegans that are unable to signal in 
the highly conserved IRE1-dependent UPR pathway. However a family of genes, abu 
(activated in blocked UPR), was induced to higher levels in ER-stressed xbp-1 
mutant animals than in ER-stressed wild-type animals. RNA-mediated interference 
(RNAi) inactivation of a representative abu family member, abu-1 (AC3.3), 
activated the ER stress marker hsp-4::gfp in otherwise normal animals and killed 
50% of ER-stressed ire-1 and xbp-1 mutant animals. Abu-1(RNAi) also enhanced the 
effect of inactivation of sel-1, an ER-associated protein degradation gene. The 
nine abu genes encode highly related type I transmembrane proteins whose lumenal 
domains have sequence similarity to a mammalian cell surface scavenger receptor 
of endothelial cells that binds chemically modified extracellular proteins and 
directs their lysosomal degradation. Our findings that ABU-1 is an intracellular 
protein located within the endomembrane system that is induced by ER stress in 
xbp-1 mutant animals suggest that ABU proteins may interact with abnormal ER 
client proteins and this function may be particularly important in animals with 
an impaired UPR.

DOI: 10.1083/jcb.200203086
PMCID: PMC2174003
PMID: 12186849 [Indexed for MEDLINE]

## Full Text

IntroductionThe ER participates in folding and posttranslational processing of proteins destined for secretion or membrane insertion. Several signaling pathways, collectively referred to as the unfolded protein response (UPR),* maintain an equilibrium between the capacity of the ER to process its client proteins and the physiological demand on the organelle (Kaufman, 1999; Mori, 2000; Patil and Walter, 2001). Inactivation of UPR signaling has deleterious consequences on cell survival and organ function (Bertolotti et al., 2001; Harding et al., 2000, 2001; Scheuner et al., 2001; Shen et al., 2001), and accumulation of malfolded proteins in the ER may play an important role in human diseases (Carrell and Lomas, 1997; Aridor and Balch, 1999).In yeast, ER stress activates the ER resident transmembrane protein kinase and endonuclease Ire1p, leading to the posttranscriptional processing of a downstream mRNA encoding the transcription factor Hac1p. Hac1p directly induces target genes of the UPR. These target genes encode proteins involved in processing, trafficking, and degradation of ER client proteins (for reviews see Mori, 2000; Patil and Walter, 2001). The UPR has diversified considerably in metazoans. In addition to a conserved signaling pathway of IRE1 homologues and a downstream HAC1-like transcription factor, XBP-1 (Yoshida et al., 2001; Calfon et al., 2002), two new pathways, not present in yeast, have evolved. Pancreatic-enriched ER kinase (PERK) phosphorylates eukaryotic translation initiation factor 2 to attenuate protein synthesis and activate specific gene expression during ER stress (for review see Ron and Harding, 2000), and the ER stress–activated transcription factor ATF6 directly activates UPR target genes (Haze et al., 1999; Ye et al., 2000; Yoshida et al., 2001).The coordination of these three pathways and their specific contribution to the metazoan ER stress response are unclear, though in Caenorhabditis elegans, the IRE1 and PERK pathways provide redundant protection against ER stress (Shen et al., 2001). We used cDNA microarrays to characterize the transcriptional response to ER stress in C. elegans, a simple metazoan that has counterparts of all three known components of the mammalian UPR (Shen et al., 2001; Calfon et al., 2002). We find that in C. elegans, the ire-1 and xbp-1 pathway has retained its essential role in upregulating expression of many UPR target genes that are similarly upregulated by the homologous pathway in yeast. We also discovered a novel family of highly related genes that protect against ER stress when the ire-1 and xbp-1 signaling pathway is defective.

Introduction

The ER participates in folding and posttranslational processing of proteins destined for secretion or membrane insertion. Several signaling pathways, collectively referred to as the unfolded protein response (UPR),* maintain an equilibrium between the capacity of the ER to process its client proteins and the physiological demand on the organelle (Kaufman, 1999; Mori, 2000; Patil and Walter, 2001). Inactivation of UPR signaling has deleterious consequences on cell survival and organ function (Bertolotti et al., 2001; Harding et al., 2000, 2001; Scheuner et al., 2001; Shen et al., 2001), and accumulation of malfolded proteins in the ER may play an important role in human diseases (Carrell and Lomas, 1997; Aridor and Balch, 1999).

In yeast, ER stress activates the ER resident transmembrane protein kinase and endonuclease Ire1p, leading to the posttranscriptional processing of a downstream mRNA encoding the transcription factor Hac1p. Hac1p directly induces target genes of the UPR. These target genes encode proteins involved in processing, trafficking, and degradation of ER client proteins (for reviews see Mori, 2000; Patil and Walter, 2001). The UPR has diversified considerably in metazoans. In addition to a conserved signaling pathway of IRE1 homologues and a downstream HAC1-like transcription factor, XBP-1 (Yoshida et al., 2001; Calfon et al., 2002), two new pathways, not present in yeast, have evolved. Pancreatic-enriched ER kinase (PERK) phosphorylates eukaryotic translation initiation factor 2 to attenuate protein synthesis and activate specific gene expression during ER stress (for review see Ron and Harding, 2000), and the ER stress–activated transcription factor ATF6 directly activates UPR target genes (Haze et al., 1999; Ye et al., 2000; Yoshida et al., 2001).

The coordination of these three pathways and their specific contribution to the metazoan ER stress response are unclear, though in Caenorhabditis elegans, the IRE1 and PERK pathways provide redundant protection against ER stress (Shen et al., 2001). We used cDNA microarrays to characterize the transcriptional response to ER stress in C. elegans, a simple metazoan that has counterparts of all three known components of the mammalian UPR (Shen et al., 2001; Calfon et al., 2002). We find that in C. elegans, the ire-1 and xbp-1 pathway has retained its essential role in upregulating expression of many UPR target genes that are similarly upregulated by the homologous pathway in yeast. We also discovered a novel family of highly related genes that protect against ER stress when the ire-1 and xbp-1 signaling pathway is defective.

ResultsWe compared the gene expression profile in unstressed and ER-stressed wild-type young adult C. elegans with the profile in an xbp-1 mutant strain that is defective in the ire-1 and xbp-1 signaling pathway (Calfon et al., 2002). Exposure of wild-type animals to the ER stress–inducing glycosylation inhibitor tunicamycin resulted in a 2.5-fold or greater statistically significant activation (P < 0.05) of 34 of the ∼18,000 genes on the array. Of these 34 genes, 26 were uninduced or induced to significantly lower levels (P < 0.05) in the xbp-1 mutant strain. All but 2 of the 26 xbp-1–regulated genes encode familiar proteins such as ER or cytosolic chaperones, proteins involved in vesicular transport or other aspects of posttranslational processing of ER client proteins (Table I). Therefore, in C. elegans, as in yeast, the UPR entails induction of genes involved in many different aspects of posttranslational protein processing in the secretory pathway. We confined our presentation only to genes whose induction withstood stringent statistical criteria that took into account both variation among replicate points in our experiments (so-called local variation) and historical information on variation between replicates in other experiments performed on the same array (so-called global variation). Among the genes whose induction passed these stringent statistical criteria, <1/4 were xbp-1 independent. These observations suggest that robustly induced UPR genes are significantly controlled by xbp-1, however we have not addressed the role of xbp-1 in the activation of genes with more variable levels of induction by ER stress.Table I.Genes whose activation by tunicamycin treatment is attenuated in xbp-1(zc12)III mutant animalsGeneInduction in N2Induction in xbp-1
ProteinF43E2.81.92 ± 0.140.44 ± 0.25HSP-4C15H9.61.32 ± 0.060.68 ± 0.05HSP-3T27E4.82.85 ± 0.300.18 ± 0.37HSP-16Y46H3A.E2.67 ± 0.340.49 ± 0.40HSP16-41Y46H3A.D2.55 ± 0.48−0.20 ± 0.17HSP16-2T27E4.22.48 ± 0.220.29 ± 0.41HSP-16T27E4.91.97 ± 0.08−0.57 ± 0.32HSP16-49C12C8.12.08 ± 0.150.54 ± 0.17Member of the heat shock HSP70 protein familyT14G8.31.35 ± 0.05−0.17 ± 0.13HSP70-like ATPaseC14B9.21.83 ± 0.160.19 ± 0.15Member of the protein disulfide isomerase protein familyC07A12.41.38 ± 0.100.42 ± 0.17PDI-2B0403.41.66 ± 0.120.46 ± 0.10Member of the thioredoxin protein familyC53B4.72.28 ± 0.44−0.09 ± 0.14Putative GDP-mannose 4,6-dehydrataseY41C4A.113.27 ± 0.21−0.40 ± 0.13Member of the WD repeat protein familyF45E4.13.05 ± 0.340.67 ± 0.15ARL-6F20D1.52.80 ± 0.510.68 ± 0.20ARL-7B0285.93.50 ± 0.30−0.28 ± 0.11Member of the choline/ethanolamine kinase protein familyK01D12.114.21 ± 0.310.85 ± 0.13Member of the GST protein familyC55B6.21.85 ± 0.170.29 ± 0.14Putative orthologue of human protein kinase inhibitor p58ZK662.43.10 ± 0.310.67 ± 0.02Negative regulator of the LET-23 receptor tyrosine kinase LIN-15F20D6.42.69 ± 0.100.04 ± 0.31SRP-7, serine protease inhibitorB0222.92.05 ± 0.201.00 ± 0.08Similar to xanthine dehydrogenaseY40B10A.G1.40 ± 0.130.27 ± 0.16Similar to O-methyltransferaseR07B1.101.34 ± 0.100.19 ± 0.09Member of the galectin familyF41B4.32.86 ± 0.450.06 ± 0.09UnknownK02D7.41.70 ± 0.050.38 ± 0.14UnknownShown is the mean ± SEM (n = 3) of the base2 log–fold increase in tunicamycin-treated animals over untreated animals for each genotype. The induction of lin-15 (ZK662.4) is likely to be an artifact of the hsp-4::gfp;lin-15 array present in both wild-type and mutant animals.19 genes were induced to higher levels in xbp-1 mutant animals than in wild-type animals (Table II). Nine of these encode highly similar, novel proteins with a hydrophobic NH2-terminal signal sequence, a potential transmembrane domain, and a short COOH-terminal cytoplasmic domain (Fig. 1, A, B, and C)
. We refer to members of this family as activated in blocked UPR (abu). Two other similarly regulated genes encode proteins that are more distantly related to the nine ABU family members (Table II). A representative family member, abu-1 (AC3.3), was chosen for further study based on its central location in the family tree (Fig. 1 A). Northern blot analysis confirmed the induction of abu-1 by tunicamycin treatment of xbp-1 mutant animals but not wild-type animals (Fig. 1 D).Table II.Genes whose activation by tunicamycin treatment is enhanced in xbp-1(zc12)III mutant animalsGenesInduction in xbp-1
Induction in N2ProteinChromosomeF19G12.71.90 ± 0.26−0.11 ± 0.11ABU (see below)XY5H2A.C1.89 ± 0.280.24 ± 0.28ABU (see below)VF31A3.11.52 ± 0.130.12 ± 0.27ABU (see below)XAC3.31.45 ± 0.29−0.16 ± 0.46ABU-1 (see below)VY105C5A.D1.42 ± 0.37−0.35 ± 0.27ABU (see below)IVC03A7.141.77 ± 0.29−0.39 ± 0.15ABU (see below)VC03A7.71.75 ± 0.28−0.39 ± 0.13ABU (see below)VR09F10.21.29 ± 0.40−0.39 ± 0.10ABU (see below)XC03A7.81.22 ± 0.49−0.67 ± 0.22ABU (see below)VT01D1.61.59 ± 0.30−0.07 ± 0.08ABU (see below)IIF35A5.31.44 ± 0.30−0.25 ± 0.12ABU (see below)XY38C1BA.G1.60 ± 0.390.02 ± 0.08CollagenF53B1.41.24 ± 0.13−0.18 ± 0.10Thymidine metabolismT06E4.81.81 ± 0.13−0.43 ± 0.08UnknownY73F8A.G1.66 ± 0.32−0.29 ± 0.16UnknownF41E6.111.65 ± 0.13−0.46 ± 0.14UnknownW08E12.G1.51 ± 0.36−0.43 ± 0.08UnknownW02A2.31.49 ± 0.31−0.41 ± 0.12UnknownD2096.61.47 ± 0.15−0.27 ± 0.30UnknownShown is the mean ± SEM (n = 3) of the base2 log–fold increase in hybridization signal in tunicamycin-treated animals over untreated animals for each genotype. The linkage group for the nine abu genes is also shown.Figure 1.
The C. elegans activated in blocked UPR (abu) gene family. (A) The amino acid sequence identity relationship of the ABU family members, C. elegans CED-1 (Zhou et al., 2001), and human scavenger receptor of endothelial cells (Adachi et al., 1997) is displayed as a radial tree (Page, 1998). (B) Amino acid sequence alignment of the nine ABU proteins using the Clustal W (1.81) program. (C) Predicted structure of ABU family members. (D) Northern blot analysis of total RNA from untreated and tunicamycin-treated wild-type and xbp-1(zc12)III mutant C. elegans. The blot was hybridized sequentially to labeled abu-1 and hsp-4 cDNA. The ethidium bromide–stained gel (bottom panel) reports on the integrity of the mRNA in all samples.ABU proteins have a distant similarity to a mammalian scavenger receptor of endothelial cells and to the C. elegans cell corpse engulfment protein CED-1, which are transmembrane cell surface proteins (Adachi et al., 1997; Zhou et al., 2001). To study the subcellular localization of ABU-1, the protein was tagged at its COOH terminus with GFP and expressed in the intestine of C. elegans from a transgene driven by the ges-1 promoter. ges-1::abu-1–gfp transgenic animals had a punctate pattern of green fluorescence. Confocal microscopy showed that the puncta corresponded to vesicular structures that were present throughout the large intestinal cell and tended to cluster near its apical surface (Fig. 2
A). No cell surface staining was noted. In contrast, ges-1::gfp fluorescence was observed throughout the cytoplasm and nucleus of the intestinal cell (Fig. 2 B) and nontransgenic animals had only background signal from autofluorescence (Fig. 2 C). To expand on these observations, we expressed the ABU-1–GFP fusion protein from its endogenous abu-1 promoter. The expression level of this transgene was much lower than that of ges-1::abu-1–gfp and was noted in the pharynx and not the intestine (also see Fig. 5
A). ABU-1–GFP expressed at these low levels was also associated with vesicular structures (Fig. 2, D and E), suggesting that this pattern of expression is not an artifact of unphysiologically high expression levels of the fusion proteins. However, given that we were not able to detect the endogenous ABU-1 protein, it remains possible that the apparent localization to intracellular vesicles is due to misfolding of the fusion proteins.Figure 2.
Confocal fluorescence micrographs of C. elegans expressing fluorescent fusion proteins. (A) ges-1::abu-1::gfp(zcEx7) expressing ABU-1 with GFP fused to its COOH terminus in the intestinal cell. (B) ges-1::gfp(zcEx6) expresses unmodified GFP in the intestinal cell. (C) Animals with no transgene. The arrows point to the vesicular structures positive for ABU-1–GFP and the arrowheads point to the background autofluorescence that is present in worms with no transgene. Shown are representative images obtained using the same settings. (D) The abu-1::gfp(zcEx8) transgene expresses an ABU-1–GFP fusion protein from the endogenous abu-1 promoter. The arrows point to the vesicular structures positive for ABU-1–GFP protein. (E) The abu-1::gfp(zcIs8)X reporter expresses GFP from the abu-1 promoter.Figure 5.
Inactivation of abu-1 causes ER stress and enhances the phenotype of C. elegans with defective ER-associated protein degradation. (A) Fluorescence micrographs of untreated (UT) or cadmium-treated (Cd2+) wild-type or xbp-1(zc12)III mutant C. elegans transgenic for an abu-1::gfp(zcIs8)X reporter. (B) Fluorescence micrographs of untreated (UT), abu-1(RNAi), or sel-1(RNAi) animals transgenic for the ER stress reporter gene hsp-4::gfp(zcIs4)V. (C) Lethality at 72 h of wild-type and xbp-1(zc12)III mutant C. elegans untreated (UT), abu-1(RNAi), sel-1(RNAi), or compound abu-1(RNAi); sel-1(RNAi). Shown are mean ± SEM of a typical experiment performed in duplicate on 30 animals of each genotype and reproduced four times. (D) Photomicrographs using Nomarski optics (X400) of the intestine of living young adult C. elegans of the following genotypes: (1) wild type, (2) abu-1(RNAi);sel-1(RNAi), (3) xbp-1(zc12)III, (4) xbp-1(zc12)III; abu-1(RNAi), (5) xbp-1(zc12)III; sel-1(RNAi), and (6) xbp-1(zc12)III;abu-1(RNAi);sel-1(RNAi). The arrow points to the dark large granules that accumulate in the cytoplasm of the compound mutant animals.The fluorescence pattern of ABU-1–GFP suggested that the protein was associated with the intracellular endomembrane system. To extend these observations, we studied the subcellular localization of C. elegans ABU-1 in mammalian cells, an experimental system in which we could make use of established markers for the various organelles and compartments. In transfected mammalian COS1 cells, metabolically-labeled ABU-1 tagged at its COOH terminus with the FLAG epitope remained associated with the cell pellet, from which it could be extracted only by detergent solubilization (Fig. 3
A). Deletion of the predicted transmembrane domain led to secretion of the protein into the culture media (Fig. 3 A). Immunostaining of the FLAG-tagged ABU-1–expressing COS1 cells with anti-FLAG antibodies showed a diffuse reticular pattern that colocalized with the ER marker ribophorin I (Fig. 3 B). These results suggest that ABU-1 encodes an integral membrane protein that is retained in the ER by its transmembrane domain. The secretion of transmembrane domain–deleted ABU-1 suggests that the lumenal portion of the protein folds properly in mammalian cells and that the full-length ABU-1 is unlikely to be retained in the ER as a consequence of malfolding in mammalian cells.Figure 3.
ABU-1 is retained in the ER of mammalian cells by its transmembrane domain. (A) Autoradiogram of metabolically-labeled wild-type ABU-1 (WT) or ABU-1 lacking the transmembrane domain (ΔTM) expressed in COS1 cells and immunoprecipitated immediately after the labeling period (lanes 1 and 3) or after 4 h of cold chase (lanes 2 and 4–6). The protein in lanes 1–4 was extracted by detergent from disrupted cells, whereas the protein in lanes 5 and 6 was recovered from the culture supernatant. (B) Immunocytochemical staining of FLAG-tagged ABU-1 expressed in COS1 cells using an anti-FLAG monoclonal antibody. Staining of the same cells with ribophorin I outlines the ER and the merged image reveals the colocalization of ABU-1 (anti-FLAG) and ribophorin 1.The protein-coding region of the nine ABU genes is >89% identical at the nucleotide level over a stretch of ∼100 residues in the 3′ end of the genes. Therefore, we considered it possible to simultaneously inactivate several members of this gene family by RNA-mediated interference (RNAi), using sequences derived from this conserved region. To study the impact of this genetic manipulation, we first identified a pharmacological treatment that would cause ER stress but could be tolerated by both wild-type and ire-1 or xbp-1 mutant animals. The heavy metal cadmium activated the hsp-4::gfp reporter (a marker of ER stress) in an ire-1– and xbp-1–dependent manner (Fig. 4
A), indicating that it causes ER stress. Both wild-type and xbp-1 mutant animals tolerated cadmium exposure for extended periods of time (Fig. 4 B). Feeding of double-stranded gfp RNA (gfp[RNAi]) had little impact on the viability of wild-type or mutant animals whether exposed to cadmium or not (<5% lethality). However, feeding of double-stranded abu-1 RNA markedly impaired survival of cadmium-treated xbp-1 and ire-1 mutant animals, but had minimal impact on survival of cadmium-treated wild-type animals or on survival of untreated xbp-1 and ire-1 mutant animals (Fig. 4 B). The inhibition of ABU-1 protein expression by abu-1(RNAi) was revealed by the observation that feeding double-stranded abu-1 RNA to ges-1::abu-1–gfp animals strongly inhibited GFP fluorescence (Fig. 4 C). These observations are consistent with a role for abu-1 (and possibly other ABU genes) in protecting animals with a defective UPR against ER stress.Figure 4.
Inactivation of abu-1 by RNAi reduces the viability of C. elegans with a blocked UPR. (A) Fluorescence micrographs of untreated (UT) or cadmium-treated (Cd2+) wild-type, xbp-1(zc12)III mutant, or ire-1(zc14)II mutant C. elegans transgenic for the ER stress reporter gene hsp-4::gfp(zcIs4)V. (B) Survival at 48 h of untreated (UT) or cadmium-treated (Cd2+) wild-type, xbp-1(zc12)III mutant, or ire-1(zc14)II mutant C. elegans at the L4 stage that had been subjected to abu-1(RNAi), gfp(RNAi), or no RNAi. Shown are mean ± SEM of a typical experiment performed in duplicate on 30 animals in each group and reproduced four times. (C) Fluorescence micrograph of untreated (UT) and abu-1(RNAi)–treated ges-1::abu-1:: gfp(zcEx7) animals.To determine the expression pattern of abu-1 in ER-stressed animals, we generated an abu-1::gfp transcriptional reporter C. elegans strain. abu-1::gfp was expressed constitutively in the pharynx and head region of wild-type and xbp-1 mutant adult animals. A low level of basal expression was observed in the gut. Intestinal expression was induced by ER stress, most notably in xbp-1 mutant animals (Fig. 5 A). The induction of abu-1::gfp in the intestine of cadmium- and tunicamycin-treated animals is consistent with observations that the intestine is active in protein secretion and is a major target for ER stress in C. elegans (Fig. 4 A) (Shen et al., 2001; Calfon et al., 2002). abu-1(RNAi) upregulated the ER stress reporter gene hsp-4::gfp in the intestine (Fig. 5 B), implying that abu-1 (and possibly other ABU genes) may also be active in the intestine under basal conditions and that interfering with ABU function causes ER stress.Genes active in degradation of malfolded ER proteins (ER-associated degradation [ERAD] genes) are not essential for yeast viability under normal growth conditions but are essential during ER stress or when the UPR is blocked. Furthermore, disruption of ERAD genes in yeast activates the UPR (Casagrande et al., 2000; Friedlander et al., 2000; Ng et al., 2000; Travers et al., 2000). We considered the possibility that abu genes might interact with the process of malfolded protein disposal from the C. elegans ER. First we confirmed that in C. elegans, like yeast, inactivation of ERAD genes causes ER stress. RNAi of sel-1, the C. elegans homologue of the yeast ERAD gene HRD3 (Hampton et al., 1996), resulted in marked upregulation of the ER stress indicator hsp-4::gfp (Fig. 5 B). The induction of hsp-4::gfp by sel-1(RNAi) and by RNAi of a homologue of another yeast ERAD gene, HRD1 (encoded in C. elegans by F55A11.3), were both xbp-1 dependent (unpublished data).Inactivation of sel-1 by RNAi had no impact on the viability of wild-type animals and only modestly reduced the viability of xbp-1 mutants. However, when combined with inactivation of abu-1, sel-1(RNAi) increased lethality of xbp-1 mutant animals (Fig. 5 C). Combined inactivation of sel-1 and abu-1 also affected the appearance of xbp-1 mutant animals. Viewed under Nomarski optics, the intestine of wild-type, xbp-1 mutant, abu-1(RNAi), or sel-1(RNAi) young adults had a similar, fine, granular appearance. Animals with compound xbp-1;abu-1(RNAi) or xbp-1;sel-1(RNAi) genotypes had more dark staining large intestinal granules. The xbp-1;abu-1(RNAi);sel-1(RNAi) animal had a marked increase in such granules (Fig. 5 D). These observations suggest that abu-1 (and possibly other abu genes) and sel-1 perform partially redundant functions in animals with a blocked UPR.

We compared the gene expression profile in unstressed and ER-stressed wild-type young adult C. elegans with the profile in an xbp-1 mutant strain that is defective in the ire-1 and xbp-1 signaling pathway (Calfon et al., 2002). Exposure of wild-type animals to the ER stress–inducing glycosylation inhibitor tunicamycin resulted in a 2.5-fold or greater statistically significant activation (P < 0.05) of 34 of the ∼18,000 genes on the array. Of these 34 genes, 26 were uninduced or induced to significantly lower levels (P < 0.05) in the xbp-1 mutant strain. All but 2 of the 26 xbp-1–regulated genes encode familiar proteins such as ER or cytosolic chaperones, proteins involved in vesicular transport or other aspects of posttranslational processing of ER client proteins (Table I). Therefore, in C. elegans, as in yeast, the UPR entails induction of genes involved in many different aspects of posttranslational protein processing in the secretory pathway. We confined our presentation only to genes whose induction withstood stringent statistical criteria that took into account both variation among replicate points in our experiments (so-called local variation) and historical information on variation between replicates in other experiments performed on the same array (so-called global variation). Among the genes whose induction passed these stringent statistical criteria, <1/4 were xbp-1 independent. These observations suggest that robustly induced UPR genes are significantly controlled by xbp-1, however we have not addressed the role of xbp-1 in the activation of genes with more variable levels of induction by ER stress.

Genes whose activation by tunicamycin treatment is attenuated in xbp-1(zc12)III mutant animals

Shown is the mean ± SEM (n = 3) of the base2 log–fold increase in tunicamycin-treated animals over untreated animals for each genotype. The induction of lin-15 (ZK662.4) is likely to be an artifact of the hsp-4::gfp;lin-15 array present in both wild-type and mutant animals.

19 genes were induced to higher levels in xbp-1 mutant animals than in wild-type animals (Table II). Nine of these encode highly similar, novel proteins with a hydrophobic NH2-terminal signal sequence, a potential transmembrane domain, and a short COOH-terminal cytoplasmic domain (Fig. 1, A, B, and C)
. We refer to members of this family as activated in blocked UPR (abu). Two other similarly regulated genes encode proteins that are more distantly related to the nine ABU family members (Table II). A representative family member, abu-1 (AC3.3), was chosen for further study based on its central location in the family tree (Fig. 1 A). Northern blot analysis confirmed the induction of abu-1 by tunicamycin treatment of xbp-1 mutant animals but not wild-type animals (Fig. 1 D).

Genes whose activation by tunicamycin treatment is enhanced in xbp-1(zc12)III mutant animals

Shown is the mean ± SEM (n = 3) of the base2 log–fold increase in hybridization signal in tunicamycin-treated animals over untreated animals for each genotype. The linkage group for the nine abu genes is also shown.

The C. elegans activated in blocked UPR (abu) gene family. (A) The amino acid sequence identity relationship of the ABU family members, C. elegans CED-1 (Zhou et al., 2001), and human scavenger receptor of endothelial cells (Adachi et al., 1997) is displayed as a radial tree (Page, 1998). (B) Amino acid sequence alignment of the nine ABU proteins using the Clustal W (1.81) program. (C) Predicted structure of ABU family members. (D) Northern blot analysis of total RNA from untreated and tunicamycin-treated wild-type and xbp-1(zc12)III mutant C. elegans. The blot was hybridized sequentially to labeled abu-1 and hsp-4 cDNA. The ethidium bromide–stained gel (bottom panel) reports on the integrity of the mRNA in all samples.

ABU proteins have a distant similarity to a mammalian scavenger receptor of endothelial cells and to the C. elegans cell corpse engulfment protein CED-1, which are transmembrane cell surface proteins (Adachi et al., 1997; Zhou et al., 2001). To study the subcellular localization of ABU-1, the protein was tagged at its COOH terminus with GFP and expressed in the intestine of C. elegans from a transgene driven by the ges-1 promoter. ges-1::abu-1–gfp transgenic animals had a punctate pattern of green fluorescence. Confocal microscopy showed that the puncta corresponded to vesicular structures that were present throughout the large intestinal cell and tended to cluster near its apical surface (Fig. 2
A). No cell surface staining was noted. In contrast, ges-1::gfp fluorescence was observed throughout the cytoplasm and nucleus of the intestinal cell (Fig. 2 B) and nontransgenic animals had only background signal from autofluorescence (Fig. 2 C). To expand on these observations, we expressed the ABU-1–GFP fusion protein from its endogenous abu-1 promoter. The expression level of this transgene was much lower than that of ges-1::abu-1–gfp and was noted in the pharynx and not the intestine (also see Fig. 5
A). ABU-1–GFP expressed at these low levels was also associated with vesicular structures (Fig. 2, D and E), suggesting that this pattern of expression is not an artifact of unphysiologically high expression levels of the fusion proteins. However, given that we were not able to detect the endogenous ABU-1 protein, it remains possible that the apparent localization to intracellular vesicles is due to misfolding of the fusion proteins.

Confocal fluorescence micrographs of C. elegans expressing fluorescent fusion proteins. (A) ges-1::abu-1::gfp(zcEx7) expressing ABU-1 with GFP fused to its COOH terminus in the intestinal cell. (B) ges-1::gfp(zcEx6) expresses unmodified GFP in the intestinal cell. (C) Animals with no transgene. The arrows point to the vesicular structures positive for ABU-1–GFP and the arrowheads point to the background autofluorescence that is present in worms with no transgene. Shown are representative images obtained using the same settings. (D) The abu-1::gfp(zcEx8) transgene expresses an ABU-1–GFP fusion protein from the endogenous abu-1 promoter. The arrows point to the vesicular structures positive for ABU-1–GFP protein. (E) The abu-1::gfp(zcIs8)X reporter expresses GFP from the abu-1 promoter.

Inactivation of abu-1 causes ER stress and enhances the phenotype of C. elegans with defective ER-associated protein degradation. (A) Fluorescence micrographs of untreated (UT) or cadmium-treated (Cd2+) wild-type or xbp-1(zc12)III mutant C. elegans transgenic for an abu-1::gfp(zcIs8)X reporter. (B) Fluorescence micrographs of untreated (UT), abu-1(RNAi), or sel-1(RNAi) animals transgenic for the ER stress reporter gene hsp-4::gfp(zcIs4)V. (C) Lethality at 72 h of wild-type and xbp-1(zc12)III mutant C. elegans untreated (UT), abu-1(RNAi), sel-1(RNAi), or compound abu-1(RNAi); sel-1(RNAi). Shown are mean ± SEM of a typical experiment performed in duplicate on 30 animals of each genotype and reproduced four times. (D) Photomicrographs using Nomarski optics (X400) of the intestine of living young adult C. elegans of the following genotypes: (1) wild type, (2) abu-1(RNAi);sel-1(RNAi), (3) xbp-1(zc12)III, (4) xbp-1(zc12)III; abu-1(RNAi), (5) xbp-1(zc12)III; sel-1(RNAi), and (6) xbp-1(zc12)III;abu-1(RNAi);sel-1(RNAi). The arrow points to the dark large granules that accumulate in the cytoplasm of the compound mutant animals.

The fluorescence pattern of ABU-1–GFP suggested that the protein was associated with the intracellular endomembrane system. To extend these observations, we studied the subcellular localization of C. elegans ABU-1 in mammalian cells, an experimental system in which we could make use of established markers for the various organelles and compartments. In transfected mammalian COS1 cells, metabolically-labeled ABU-1 tagged at its COOH terminus with the FLAG epitope remained associated with the cell pellet, from which it could be extracted only by detergent solubilization (Fig. 3
A). Deletion of the predicted transmembrane domain led to secretion of the protein into the culture media (Fig. 3 A). Immunostaining of the FLAG-tagged ABU-1–expressing COS1 cells with anti-FLAG antibodies showed a diffuse reticular pattern that colocalized with the ER marker ribophorin I (Fig. 3 B). These results suggest that ABU-1 encodes an integral membrane protein that is retained in the ER by its transmembrane domain. The secretion of transmembrane domain–deleted ABU-1 suggests that the lumenal portion of the protein folds properly in mammalian cells and that the full-length ABU-1 is unlikely to be retained in the ER as a consequence of malfolding in mammalian cells.

ABU-1 is retained in the ER of mammalian cells by its transmembrane domain. (A) Autoradiogram of metabolically-labeled wild-type ABU-1 (WT) or ABU-1 lacking the transmembrane domain (ΔTM) expressed in COS1 cells and immunoprecipitated immediately after the labeling period (lanes 1 and 3) or after 4 h of cold chase (lanes 2 and 4–6). The protein in lanes 1–4 was extracted by detergent from disrupted cells, whereas the protein in lanes 5 and 6 was recovered from the culture supernatant. (B) Immunocytochemical staining of FLAG-tagged ABU-1 expressed in COS1 cells using an anti-FLAG monoclonal antibody. Staining of the same cells with ribophorin I outlines the ER and the merged image reveals the colocalization of ABU-1 (anti-FLAG) and ribophorin 1.

The protein-coding region of the nine ABU genes is >89% identical at the nucleotide level over a stretch of ∼100 residues in the 3′ end of the genes. Therefore, we considered it possible to simultaneously inactivate several members of this gene family by RNA-mediated interference (RNAi), using sequences derived from this conserved region. To study the impact of this genetic manipulation, we first identified a pharmacological treatment that would cause ER stress but could be tolerated by both wild-type and ire-1 or xbp-1 mutant animals. The heavy metal cadmium activated the hsp-4::gfp reporter (a marker of ER stress) in an ire-1– and xbp-1–dependent manner (Fig. 4
A), indicating that it causes ER stress. Both wild-type and xbp-1 mutant animals tolerated cadmium exposure for extended periods of time (Fig. 4 B). Feeding of double-stranded gfp RNA (gfp[RNAi]) had little impact on the viability of wild-type or mutant animals whether exposed to cadmium or not (<5% lethality). However, feeding of double-stranded abu-1 RNA markedly impaired survival of cadmium-treated xbp-1 and ire-1 mutant animals, but had minimal impact on survival of cadmium-treated wild-type animals or on survival of untreated xbp-1 and ire-1 mutant animals (Fig. 4 B). The inhibition of ABU-1 protein expression by abu-1(RNAi) was revealed by the observation that feeding double-stranded abu-1 RNA to ges-1::abu-1–gfp animals strongly inhibited GFP fluorescence (Fig. 4 C). These observations are consistent with a role for abu-1 (and possibly other ABU genes) in protecting animals with a defective UPR against ER stress.

Inactivation of abu-1 by RNAi reduces the viability of C. elegans with a blocked UPR. (A) Fluorescence micrographs of untreated (UT) or cadmium-treated (Cd2+) wild-type, xbp-1(zc12)III mutant, or ire-1(zc14)II mutant C. elegans transgenic for the ER stress reporter gene hsp-4::gfp(zcIs4)V. (B) Survival at 48 h of untreated (UT) or cadmium-treated (Cd2+) wild-type, xbp-1(zc12)III mutant, or ire-1(zc14)II mutant C. elegans at the L4 stage that had been subjected to abu-1(RNAi), gfp(RNAi), or no RNAi. Shown are mean ± SEM of a typical experiment performed in duplicate on 30 animals in each group and reproduced four times. (C) Fluorescence micrograph of untreated (UT) and abu-1(RNAi)–treated ges-1::abu-1:: gfp(zcEx7) animals.

To determine the expression pattern of abu-1 in ER-stressed animals, we generated an abu-1::gfp transcriptional reporter C. elegans strain. abu-1::gfp was expressed constitutively in the pharynx and head region of wild-type and xbp-1 mutant adult animals. A low level of basal expression was observed in the gut. Intestinal expression was induced by ER stress, most notably in xbp-1 mutant animals (Fig. 5 A). The induction of abu-1::gfp in the intestine of cadmium- and tunicamycin-treated animals is consistent with observations that the intestine is active in protein secretion and is a major target for ER stress in C. elegans (Fig. 4 A) (Shen et al., 2001; Calfon et al., 2002). abu-1(RNAi) upregulated the ER stress reporter gene hsp-4::gfp in the intestine (Fig. 5 B), implying that abu-1 (and possibly other ABU genes) may also be active in the intestine under basal conditions and that interfering with ABU function causes ER stress.

Genes active in degradation of malfolded ER proteins (ER-associated degradation [ERAD] genes) are not essential for yeast viability under normal growth conditions but are essential during ER stress or when the UPR is blocked. Furthermore, disruption of ERAD genes in yeast activates the UPR (Casagrande et al., 2000; Friedlander et al., 2000; Ng et al., 2000; Travers et al., 2000). We considered the possibility that abu genes might interact with the process of malfolded protein disposal from the C. elegans ER. First we confirmed that in C. elegans, like yeast, inactivation of ERAD genes causes ER stress. RNAi of sel-1, the C. elegans homologue of the yeast ERAD gene HRD3 (Hampton et al., 1996), resulted in marked upregulation of the ER stress indicator hsp-4::gfp (Fig. 5 B). The induction of hsp-4::gfp by sel-1(RNAi) and by RNAi of a homologue of another yeast ERAD gene, HRD1 (encoded in C. elegans by F55A11.3), were both xbp-1 dependent (unpublished data).

Inactivation of sel-1 by RNAi had no impact on the viability of wild-type animals and only modestly reduced the viability of xbp-1 mutants. However, when combined with inactivation of abu-1, sel-1(RNAi) increased lethality of xbp-1 mutant animals (Fig. 5 C). Combined inactivation of sel-1 and abu-1 also affected the appearance of xbp-1 mutant animals. Viewed under Nomarski optics, the intestine of wild-type, xbp-1 mutant, abu-1(RNAi), or sel-1(RNAi) young adults had a similar, fine, granular appearance. Animals with compound xbp-1;abu-1(RNAi) or xbp-1;sel-1(RNAi) genotypes had more dark staining large intestinal granules. The xbp-1;abu-1(RNAi);sel-1(RNAi) animal had a marked increase in such granules (Fig. 5 D). These observations suggest that abu-1 (and possibly other abu genes) and sel-1 perform partially redundant functions in animals with a blocked UPR.

DiscussionOur search for genes activated in the C. elegans UPR revealed similarities to the yeast S. cerevisiae. In both species the UPR induces genes that function at many levels of the secretory pathway and encode proteins with diverse and unrelated biochemical activities. In both species, the signaling pathway initiated by IRE1 exerts nearly complete control over the induction of well-characterized components of ER client protein processing machinery, such as BiP (hsp-4), protein disulfide isomerases, and 4-prolyl hydroxylases. We confined our analysis to genes whose induction by tunicamycin passed stringent statistical criteria. The application of these stringent statistical criteria eliminated many genes that probably are true UPR targets and therefore the list presented in Table I is only representative of the most reliably induced UPR genes in C. elegans. Heterogeneity in developmental stage and variation in uptake of drug stand out as likely contributors to the lower power of the whole animal array performed here compared with yeast arrays (Casagrande et al., 2000; Travers et al., 2000), and as Fig. 1 D shows, the microarray hybridization procedure is also less sensitive than Northern blot analysis in detecting fold changes in expression of hsp-4. Despite these limitations, we can firmly conclude that most of the reliably induced UPR genes in C. elegans are xbp-1 dependent.Our genome-wide survey also identified 19 genes that were activated by ER stress in xbp-1 mutant animals but not in wild-type animals. Among these were nine genes predicted to encode a family of highly related ABU proteins. These abu genes are coregulated across different physiological and developmental perturbation; clustering together in a single, small, isolated and well-defined “expression mountain,” mount 29, in the recently established gene expression map of C. elegans (Kim et al., 2001). GFP driven by the promoter of a representative family member, abu-1::gfp, was expressed strongly in the pharynx and head region of transgenic animals from larval stages L3–L4 to young adult (Fig. 5 A). A similar expression pattern was also observed in animals transgenic for a transcriptional fusion of gfp to a different abu gene F19G12.7 (unpublished data). Furthermore, the basal expression pattern of the abu-1 transcriptional reporters correlated nicely with the in situ hybridization pattern of another abu family member, C03A7.7. Together, these observations suggest that the abu-1::gfp reporter reflects the activity of the endogenous gene. In older animals, we noted low levels of basal expression of abu-1::gfp in the intestine that was markedly upregulated by ER stress in xbp-1 mutant animals (Fig. 5 A). This last observation is consistent with the Northern blot analysis of abu-1 expression (Fig. 1 D). Although the significance of basal expression of abu genes in the pharynx is currently not understood, the intestinal expression correlates with the phenotype of animals in which abu-1 had been inactivated by RNAi. We do not know if abu-1 induction by ER stress in xbp-1 mutant animals is dependent on PERK (pek-1) or ATF6 (atf-6) signaling, as compound xbp-1;pek-1 and xbp-1;atf-6(RNAi) mutant animals arrest at early larval stages before we could reliably examine abu-1::gfp expression in the gut (Shen et al., 2001; unpublished data).Inactivation of abu-1 by RNAi led to marked induction of the ER stress indicator hsp-4::gfp in the intestine (Fig. 5 B). This induction was completely blocked by the xbp-1 mutation (unpublished data), indicating the development of ER stress in the gut of abu-1(RNAi) animals and implying that abu-1 plays a role in an ER function, which, when inactivated, causes ER stress. This ER stress appeared to be well tolerated, as the abu-1(RNAi) animals developed at a normal rate and survived exposure to Cd2+, a toxin that induced further ER stress. However, in xbp-1 mutant animals, abu-1(RNAi) had serious consequences, significantly reducing their viability after exposure to additional ER stress (Fig. 4 B). The abu genes have a very high level of sequence identity at the nucleotide level, and the double-stranded RNA used to inactivate abu-1 could also initiate RNA inactivation of other family members. It is therefore possible that the strong phenotype observed in the abu-1(RNAi) animals reflects the simultaneous inactivation of other abu family members. Collectively, these results point to a role for abu-1, and possibly other abu genes, in some aspect of ER function that is also active under basal conditions. Reduced levels of this abu-dependent activity can be compensated by a normal UPR. However, when the UPR is compromised, loss of this abu-dependent activity becomes limiting, killing the affected animals.In yeast, the ability to dispose of misfolded proteins in the secretory pathway (ERAD) plays an important role in adapting to high levels of ER stress, but is dispensable for viability under normal conditions (Casagrande et al., 2000; Friedlander et al., 2000; Ng et al., 2000; Travers et al., 2000). Given that abu-1(RNAi) is well tolerated under normal conditions, but not in animals experiencing high levels of ER stress, we considered the possibility that the abu genes may interact genetically with components of the C. elegans ERAD apparatus. An interaction was uncovered between sel-1, a C. elegans gene encoding an ER-localized component of the ERAD apparatus, and abu-1. Animals with diminished xbp-1, sel-1, and abu-1 activity had reduced viability and accumulated unusually large vesicles with dark content in their intestinal cells (Fig. 5 D). sel-1 and abu-1 are unlikely to function in the exact same pathway, as abu-1(RNAi) did not affect the phenotype of a hypomorphic allele of lin-12, whereas sel-1(RNAi) suppressed the egg-laying defect of lin-12(n676; n930)III (unpublished data). However, the observation that inactivating either gene leads to ER stress and the synthetic interactions between them suggest that sel-1 and abu-1 function in interacting cellular processes.ABU-1 is a transmembrane protein that likely functions within the endomembrane system (Figs. 2 and 3). The lumenal domain of some ABU proteins resembles the extracellular domains of a mammalian scavenger receptor and C. elegans CED-1. Both the scavenger receptors and CED-1 are believed to serve as cell surface receptors for chemically modified macromolecules, oxidized lipoproteins, and other altered plasma proteins in the case of the scavenger receptors (Van Berkel et al., 2000) and corpses of apoptotic cells in the case of CED-1 (Zhou et al., 2001). Endocytosis and lysosomal degradation follow binding of these abnormal ligands to their receptors. Scavenger receptors and CED-1 are therefore believed to play a role in recycling the building blocks of damaged macromolecules that are present outside the cell. It is possible therefore that the ABU proteins may be playing a similar role within the endomembrane system, perhaps by binding to altered ER client proteins and modulating their intracellular fate.

Our search for genes activated in the C. elegans UPR revealed similarities to the yeast S. cerevisiae. In both species the UPR induces genes that function at many levels of the secretory pathway and encode proteins with diverse and unrelated biochemical activities. In both species, the signaling pathway initiated by IRE1 exerts nearly complete control over the induction of well-characterized components of ER client protein processing machinery, such as BiP (hsp-4), protein disulfide isomerases, and 4-prolyl hydroxylases. We confined our analysis to genes whose induction by tunicamycin passed stringent statistical criteria. The application of these stringent statistical criteria eliminated many genes that probably are true UPR targets and therefore the list presented in Table I is only representative of the most reliably induced UPR genes in C. elegans. Heterogeneity in developmental stage and variation in uptake of drug stand out as likely contributors to the lower power of the whole animal array performed here compared with yeast arrays (Casagrande et al., 2000; Travers et al., 2000), and as Fig. 1 D shows, the microarray hybridization procedure is also less sensitive than Northern blot analysis in detecting fold changes in expression of hsp-4. Despite these limitations, we can firmly conclude that most of the reliably induced UPR genes in C. elegans are xbp-1 dependent.

Our genome-wide survey also identified 19 genes that were activated by ER stress in xbp-1 mutant animals but not in wild-type animals. Among these were nine genes predicted to encode a family of highly related ABU proteins. These abu genes are coregulated across different physiological and developmental perturbation; clustering together in a single, small, isolated and well-defined “expression mountain,” mount 29, in the recently established gene expression map of C. elegans (Kim et al., 2001). GFP driven by the promoter of a representative family member, abu-1::gfp, was expressed strongly in the pharynx and head region of transgenic animals from larval stages L3–L4 to young adult (Fig. 5 A). A similar expression pattern was also observed in animals transgenic for a transcriptional fusion of gfp to a different abu gene F19G12.7 (unpublished data). Furthermore, the basal expression pattern of the abu-1 transcriptional reporters correlated nicely with the in situ hybridization pattern of another abu family member, C03A7.7. Together, these observations suggest that the abu-1::gfp reporter reflects the activity of the endogenous gene. In older animals, we noted low levels of basal expression of abu-1::gfp in the intestine that was markedly upregulated by ER stress in xbp-1 mutant animals (Fig. 5 A). This last observation is consistent with the Northern blot analysis of abu-1 expression (Fig. 1 D). Although the significance of basal expression of abu genes in the pharynx is currently not understood, the intestinal expression correlates with the phenotype of animals in which abu-1 had been inactivated by RNAi. We do not know if abu-1 induction by ER stress in xbp-1 mutant animals is dependent on PERK (pek-1) or ATF6 (atf-6) signaling, as compound xbp-1;pek-1 and xbp-1;atf-6(RNAi) mutant animals arrest at early larval stages before we could reliably examine abu-1::gfp expression in the gut (Shen et al., 2001; unpublished data).

Inactivation of abu-1 by RNAi led to marked induction of the ER stress indicator hsp-4::gfp in the intestine (Fig. 5 B). This induction was completely blocked by the xbp-1 mutation (unpublished data), indicating the development of ER stress in the gut of abu-1(RNAi) animals and implying that abu-1 plays a role in an ER function, which, when inactivated, causes ER stress. This ER stress appeared to be well tolerated, as the abu-1(RNAi) animals developed at a normal rate and survived exposure to Cd2+, a toxin that induced further ER stress. However, in xbp-1 mutant animals, abu-1(RNAi) had serious consequences, significantly reducing their viability after exposure to additional ER stress (Fig. 4 B). The abu genes have a very high level of sequence identity at the nucleotide level, and the double-stranded RNA used to inactivate abu-1 could also initiate RNA inactivation of other family members. It is therefore possible that the strong phenotype observed in the abu-1(RNAi) animals reflects the simultaneous inactivation of other abu family members. Collectively, these results point to a role for abu-1, and possibly other abu genes, in some aspect of ER function that is also active under basal conditions. Reduced levels of this abu-dependent activity can be compensated by a normal UPR. However, when the UPR is compromised, loss of this abu-dependent activity becomes limiting, killing the affected animals.

In yeast, the ability to dispose of misfolded proteins in the secretory pathway (ERAD) plays an important role in adapting to high levels of ER stress, but is dispensable for viability under normal conditions (Casagrande et al., 2000; Friedlander et al., 2000; Ng et al., 2000; Travers et al., 2000). Given that abu-1(RNAi) is well tolerated under normal conditions, but not in animals experiencing high levels of ER stress, we considered the possibility that the abu genes may interact genetically with components of the C. elegans ERAD apparatus. An interaction was uncovered between sel-1, a C. elegans gene encoding an ER-localized component of the ERAD apparatus, and abu-1. Animals with diminished xbp-1, sel-1, and abu-1 activity had reduced viability and accumulated unusually large vesicles with dark content in their intestinal cells (Fig. 5 D). sel-1 and abu-1 are unlikely to function in the exact same pathway, as abu-1(RNAi) did not affect the phenotype of a hypomorphic allele of lin-12, whereas sel-1(RNAi) suppressed the egg-laying defect of lin-12(n676; n930)III (unpublished data). However, the observation that inactivating either gene leads to ER stress and the synthetic interactions between them suggest that sel-1 and abu-1 function in interacting cellular processes.

ABU-1 is a transmembrane protein that likely functions within the endomembrane system (Figs. 2 and 3). The lumenal domain of some ABU proteins resembles the extracellular domains of a mammalian scavenger receptor and C. elegans CED-1. Both the scavenger receptors and CED-1 are believed to serve as cell surface receptors for chemically modified macromolecules, oxidized lipoproteins, and other altered plasma proteins in the case of the scavenger receptors (Van Berkel et al., 2000) and corpses of apoptotic cells in the case of CED-1 (Zhou et al., 2001). Endocytosis and lysosomal degradation follow binding of these abnormal ligands to their receptors. Scavenger receptors and CED-1 are therefore believed to play a role in recycling the building blocks of damaged macromolecules that are present outside the cell. It is possible therefore that the ABU proteins may be playing a similar role within the endomembrane system, perhaps by binding to altered ER client proteins and modulating their intracellular fate.

Materials and methodsRNA isolation, cDNA synthesis, microarray hybridization, and data analysisWhole genome hybridization experiments were conducted as previously described (Reinke et al., 2000). In brief, RNA was harvested from untreated and tunicamycin-treated (5 μg/ml, 5 h) hsp-4::gfp(zcIs4)V (wild type) and hsp-4::gfp(zcIs4)V; xbp-1(zc12)III (mutant) animals. Total RNA was isolated from populations of stage-synchronized young adult hermaphrodite animals. Poly(A)+ RNA was purified from total RNA by oligo (dT) affinity chromatography. Labeled cDNA probe for DNA microarray hybridization was prepared from 10 μg of poly(A)+ RNA. cDNA from untreated and tunicamycin-treated animals was labeled with Cy5 and Cy3, respectively. Microarrays containing 17,871 C. elegans genes were used. Microarrays were hybridized for 20 h, scanned using an Axon scanner, and the expression level for each gene was determined using GenePix software. The base2 log of the ratio of Cy3/Cy5 signal for each gene was determined in each of three hybridization experiments with a pair of untreated and tunicamycin-treated mRNA samples with wild-type or xbp-1(zc12)III genotypes (for a total of six hybridization experiments). A t test was used to determine the probability that differences in treatment-induced levels of mRNA between the wild type and xbp-1 mutant were due to chance. A 95% confidence limit was applied. The data were analyzed twice, first using the standard deviation resulting from the experiments presented in this paper and then using the global standard deviation derived from the large set of combined experiments in the Stanford Microarray Database. Differences in levels of expression were reported only if they were significant using both measurements of standard deviation.Transgenic C. elegans
A 1.3-kb fragment of C. elegans genomic DNA immediately 5′ of the predicted initiation ATG codon of abu-1 (AC3.3) was amplified by PCR using the oligonucleotides AC3.1S (5′-GGCATTGTGGCACGCATTGAACTG-3′) and AC3Bam.2AS (5′-GATAGGATCCATTGTTAATATGCTTGAAGAGCTGC-3′) and ligated in frame with the GFP coding region in the plasmid of pPD95.75 (gift of Andy Fire, Carnegie Institute of Washington, Baltimore, MD). The abu-1::gfp(zcEx8) strain was created by coinjecting the ac3.3.pPD95.75 plasmid (25 μg/ml) with a lin-15 rescuing plasmid, pSK1 (25 μg/ml), into lin-15(n765ts) strain. The extrachromosomal array was integrated into the chromosome with ultraviolet/trimethylpsoraren treatment, yielding the abu-1::gfp(zcIs8)X reporter strain.To produce the ges-1::gfp(zcEx6) strain, the ges-1 promoter was amplified by PCR from C. elegans genomic DNA using the primers ges-1.1S (5′-CTCATACATCATTGTCAAGTGACG-3′) and ges-1.Pst.2S (5′-AGTACTGCAGAGACAAGGAATATCCGCATCT-3′) and ligated into the HindIII-PstI sites of pPD95.81 (gift of Andy Fire). The abu-1 coding region fragment encoding ABU-1 amino acids 7–425 was ligated in frame with GFP into PstI-BamHI sites of this plasmid to produce the ges-1::abu-1::gfp(zxEx7) strain. The promoter and coding region of abu-1 was ligated in frame with GFP to produce abu1::abu-1::gfp(ZcEx8) strain. The animals containing transmissible extrachromosomal arrays were analyzed using a 510 laser scanning confocal microscope (ZEISS). Digital images were processed with Adobe Photoshop® 5.0.RNAi feeding and activation of ER stress by pharmacological meansInterference with gene function by RNAi followed an established protocol (Timmons et al., 2001). In brief, double-stranded RNA was produced in HT115 strain of Escherichia coli transformed with pPD129 plasmids containing cDNA fragments of genes being studied (nucleotides 1032–1278 of abu-1 cDNA and nucleotides 478–1631 of sel-1 cDNA). IPTG (1mM) was added to the bacterial growth media to induce transcription of the double-stranded RNA, and L4-staged animals of defined genotype were added to plates individually and produced their brood. RNAi phenotypes were evaluated in these progeny. Where indicated, 30 L4-stage progeny from the RNAi plate or from a control plate were transferred to a new plate seeded with fresh E. coli where they were exposed to tunicamycin (5 μg/ml) for 5 h or CdCl2 (10 mM) for the indicated period. Animals were scored as dead when pharyngeal pumping had ceased and they failed to move after being tapped on the nose with a platinum wire.Cell culture, transfection, immunocytochemistry, and pulse-chase analysisCOS-1 cells were obtained from American Type Culture Collection. All cells were grown in DME containing 10% FBS (Cellgro®) at 5% CO2 and 37°C.ABU-1 was tagged with a Flag epitope at its COOH terminus by overlapping PCR using the primers AC3.Bam.7S (5′-GCGCGGATCCGCAATGCGCTTTATCGCAATTGCAG-3′), AC3.Flag.16AS (5′-ATCGTCGTCCTTGTAGTCCTTTCTCTTGCAACACTG-3′), and Flag3′U.Not (5′-CTAG-CGGCCGCTCACTTGTCATCGTCGTCCTTGTAGTC-3′). The full-length coding region of abu-1 with the COOH-terminal Flag tag was cloned as a BamHI-NotI fragment into pcDNA3 (Invitrogen). COS-1 cells were transfected with this plasmid and stained with anti-Flag antibody (Sigma-Aldrich). Rabbit anti–ribophorin I antibody (a gift from Dr. Gert Kreibich, New York University [NYU] School of Medicine) was used to visualize the ER staining pattern.The abu-1 coding region fragment encoding ABU-1 amino acids 22–425 was amplified by PCR using primers AC3.EcoRI.5S (5′-GGCAACAAGAATTCGCGATAAACG-3′) and AC3.XbaI.4AS (5′-GCGCTCTAGATGTCTACTTTCTCTTGCAAC-3′). The abu-1 coding region fragment coding ABU-1 amino acids 22–331 was amplified by PCR using AC3.EcoRI.5S and AC3.XbaI.6AS (5′-GGCATCTAGAGCTTGGCATGCAGATTGGC-3′), deleting the transmembrane domain and COOH terminus of ABU-1. These fragments were cloned into pcDNA3 containing a Flag epitope with signal peptides. COS-1 cells were transfected with wild-type ABU-1 or ABU-1 lacking the transmembrane domain with a Flag tag. Newly synthesized proteins were labeled in vivo with 35S-methionine and 35S-cysteine (500 μCi/ml; ICN Biochemicals). After removal of the labeling media, cells were washed and incubated in complete media for cold chase for the indicated times. Cells were washed and lysed with 1% Triton buffer (20 mM Hepes, pH 7.5, 150 mM NaCl, 1% Triton X-100, 10% glycerol, 1 mM EDTA, 1 mM PMSF, 4 mg/ml aprotinin, and 2 mg/ml pepstatin A). Samples were clarified by centrifugation at 14,000 g for 10 min and by preincubation for 1 h with 10 μl protein A–Sepharose. Soluble proteins from the cells and secreted proteins in the media were immunoprecipitated by anti-Flag antibody bound to 10 μl protein A–Sepharose and washed three times with RIPA buffer. Bound proteins were resolved by 10% SDS-PAGE under reducing conditions and revealed by autoradiography.

Materials and methods

RNA isolation, cDNA synthesis, microarray hybridization, and data analysisWhole genome hybridization experiments were conducted as previously described (Reinke et al., 2000). In brief, RNA was harvested from untreated and tunicamycin-treated (5 μg/ml, 5 h) hsp-4::gfp(zcIs4)V (wild type) and hsp-4::gfp(zcIs4)V; xbp-1(zc12)III (mutant) animals. Total RNA was isolated from populations of stage-synchronized young adult hermaphrodite animals. Poly(A)+ RNA was purified from total RNA by oligo (dT) affinity chromatography. Labeled cDNA probe for DNA microarray hybridization was prepared from 10 μg of poly(A)+ RNA. cDNA from untreated and tunicamycin-treated animals was labeled with Cy5 and Cy3, respectively. Microarrays containing 17,871 C. elegans genes were used. Microarrays were hybridized for 20 h, scanned using an Axon scanner, and the expression level for each gene was determined using GenePix software. The base2 log of the ratio of Cy3/Cy5 signal for each gene was determined in each of three hybridization experiments with a pair of untreated and tunicamycin-treated mRNA samples with wild-type or xbp-1(zc12)III genotypes (for a total of six hybridization experiments). A t test was used to determine the probability that differences in treatment-induced levels of mRNA between the wild type and xbp-1 mutant were due to chance. A 95% confidence limit was applied. The data were analyzed twice, first using the standard deviation resulting from the experiments presented in this paper and then using the global standard deviation derived from the large set of combined experiments in the Stanford Microarray Database. Differences in levels of expression were reported only if they were significant using both measurements of standard deviation.

RNA isolation, cDNA synthesis, microarray hybridization, and data analysis

Whole genome hybridization experiments were conducted as previously described (Reinke et al., 2000). In brief, RNA was harvested from untreated and tunicamycin-treated (5 μg/ml, 5 h) hsp-4::gfp(zcIs4)V (wild type) and hsp-4::gfp(zcIs4)V; xbp-1(zc12)III (mutant) animals. Total RNA was isolated from populations of stage-synchronized young adult hermaphrodite animals. Poly(A)+ RNA was purified from total RNA by oligo (dT) affinity chromatography. Labeled cDNA probe for DNA microarray hybridization was prepared from 10 μg of poly(A)+ RNA. cDNA from untreated and tunicamycin-treated animals was labeled with Cy5 and Cy3, respectively. Microarrays containing 17,871 C. elegans genes were used. Microarrays were hybridized for 20 h, scanned using an Axon scanner, and the expression level for each gene was determined using GenePix software. The base2 log of the ratio of Cy3/Cy5 signal for each gene was determined in each of three hybridization experiments with a pair of untreated and tunicamycin-treated mRNA samples with wild-type or xbp-1(zc12)III genotypes (for a total of six hybridization experiments). A t test was used to determine the probability that differences in treatment-induced levels of mRNA between the wild type and xbp-1 mutant were due to chance. A 95% confidence limit was applied. The data were analyzed twice, first using the standard deviation resulting from the experiments presented in this paper and then using the global standard deviation derived from the large set of combined experiments in the Stanford Microarray Database. Differences in levels of expression were reported only if they were significant using both measurements of standard deviation.

Transgenic C. elegans
A 1.3-kb fragment of C. elegans genomic DNA immediately 5′ of the predicted initiation ATG codon of abu-1 (AC3.3) was amplified by PCR using the oligonucleotides AC3.1S (5′-GGCATTGTGGCACGCATTGAACTG-3′) and AC3Bam.2AS (5′-GATAGGATCCATTGTTAATATGCTTGAAGAGCTGC-3′) and ligated in frame with the GFP coding region in the plasmid of pPD95.75 (gift of Andy Fire, Carnegie Institute of Washington, Baltimore, MD). The abu-1::gfp(zcEx8) strain was created by coinjecting the ac3.3.pPD95.75 plasmid (25 μg/ml) with a lin-15 rescuing plasmid, pSK1 (25 μg/ml), into lin-15(n765ts) strain. The extrachromosomal array was integrated into the chromosome with ultraviolet/trimethylpsoraren treatment, yielding the abu-1::gfp(zcIs8)X reporter strain.To produce the ges-1::gfp(zcEx6) strain, the ges-1 promoter was amplified by PCR from C. elegans genomic DNA using the primers ges-1.1S (5′-CTCATACATCATTGTCAAGTGACG-3′) and ges-1.Pst.2S (5′-AGTACTGCAGAGACAAGGAATATCCGCATCT-3′) and ligated into the HindIII-PstI sites of pPD95.81 (gift of Andy Fire). The abu-1 coding region fragment encoding ABU-1 amino acids 7–425 was ligated in frame with GFP into PstI-BamHI sites of this plasmid to produce the ges-1::abu-1::gfp(zxEx7) strain. The promoter and coding region of abu-1 was ligated in frame with GFP to produce abu1::abu-1::gfp(ZcEx8) strain. The animals containing transmissible extrachromosomal arrays were analyzed using a 510 laser scanning confocal microscope (ZEISS). Digital images were processed with Adobe Photoshop® 5.0.

Transgenic C. elegans

A 1.3-kb fragment of C. elegans genomic DNA immediately 5′ of the predicted initiation ATG codon of abu-1 (AC3.3) was amplified by PCR using the oligonucleotides AC3.1S (5′-GGCATTGTGGCACGCATTGAACTG-3′) and AC3Bam.2AS (5′-GATAGGATCCATTGTTAATATGCTTGAAGAGCTGC-3′) and ligated in frame with the GFP coding region in the plasmid of pPD95.75 (gift of Andy Fire, Carnegie Institute of Washington, Baltimore, MD). The abu-1::gfp(zcEx8) strain was created by coinjecting the ac3.3.pPD95.75 plasmid (25 μg/ml) with a lin-15 rescuing plasmid, pSK1 (25 μg/ml), into lin-15(n765ts) strain. The extrachromosomal array was integrated into the chromosome with ultraviolet/trimethylpsoraren treatment, yielding the abu-1::gfp(zcIs8)X reporter strain.

To produce the ges-1::gfp(zcEx6) strain, the ges-1 promoter was amplified by PCR from C. elegans genomic DNA using the primers ges-1.1S (5′-CTCATACATCATTGTCAAGTGACG-3′) and ges-1.Pst.2S (5′-AGTACTGCAGAGACAAGGAATATCCGCATCT-3′) and ligated into the HindIII-PstI sites of pPD95.81 (gift of Andy Fire). The abu-1 coding region fragment encoding ABU-1 amino acids 7–425 was ligated in frame with GFP into PstI-BamHI sites of this plasmid to produce the ges-1::abu-1::gfp(zxEx7) strain. The promoter and coding region of abu-1 was ligated in frame with GFP to produce abu1::abu-1::gfp(ZcEx8) strain. The animals containing transmissible extrachromosomal arrays were analyzed using a 510 laser scanning confocal microscope (ZEISS). Digital images were processed with Adobe Photoshop® 5.0.

RNAi feeding and activation of ER stress by pharmacological meansInterference with gene function by RNAi followed an established protocol (Timmons et al., 2001). In brief, double-stranded RNA was produced in HT115 strain of Escherichia coli transformed with pPD129 plasmids containing cDNA fragments of genes being studied (nucleotides 1032–1278 of abu-1 cDNA and nucleotides 478–1631 of sel-1 cDNA). IPTG (1mM) was added to the bacterial growth media to induce transcription of the double-stranded RNA, and L4-staged animals of defined genotype were added to plates individually and produced their brood. RNAi phenotypes were evaluated in these progeny. Where indicated, 30 L4-stage progeny from the RNAi plate or from a control plate were transferred to a new plate seeded with fresh E. coli where they were exposed to tunicamycin (5 μg/ml) for 5 h or CdCl2 (10 mM) for the indicated period. Animals were scored as dead when pharyngeal pumping had ceased and they failed to move after being tapped on the nose with a platinum wire.

RNAi feeding and activation of ER stress by pharmacological means

Interference with gene function by RNAi followed an established protocol (Timmons et al., 2001). In brief, double-stranded RNA was produced in HT115 strain of Escherichia coli transformed with pPD129 plasmids containing cDNA fragments of genes being studied (nucleotides 1032–1278 of abu-1 cDNA and nucleotides 478–1631 of sel-1 cDNA). IPTG (1mM) was added to the bacterial growth media to induce transcription of the double-stranded RNA, and L4-staged animals of defined genotype were added to plates individually and produced their brood. RNAi phenotypes were evaluated in these progeny. Where indicated, 30 L4-stage progeny from the RNAi plate or from a control plate were transferred to a new plate seeded with fresh E. coli where they were exposed to tunicamycin (5 μg/ml) for 5 h or CdCl2 (10 mM) for the indicated period. Animals were scored as dead when pharyngeal pumping had ceased and they failed to move after being tapped on the nose with a platinum wire.

Cell culture, transfection, immunocytochemistry, and pulse-chase analysisCOS-1 cells were obtained from American Type Culture Collection. All cells were grown in DME containing 10% FBS (Cellgro®) at 5% CO2 and 37°C.ABU-1 was tagged with a Flag epitope at its COOH terminus by overlapping PCR using the primers AC3.Bam.7S (5′-GCGCGGATCCGCAATGCGCTTTATCGCAATTGCAG-3′), AC3.Flag.16AS (5′-ATCGTCGTCCTTGTAGTCCTTTCTCTTGCAACACTG-3′), and Flag3′U.Not (5′-CTAG-CGGCCGCTCACTTGTCATCGTCGTCCTTGTAGTC-3′). The full-length coding region of abu-1 with the COOH-terminal Flag tag was cloned as a BamHI-NotI fragment into pcDNA3 (Invitrogen). COS-1 cells were transfected with this plasmid and stained with anti-Flag antibody (Sigma-Aldrich). Rabbit anti–ribophorin I antibody (a gift from Dr. Gert Kreibich, New York University [NYU] School of Medicine) was used to visualize the ER staining pattern.The abu-1 coding region fragment encoding ABU-1 amino acids 22–425 was amplified by PCR using primers AC3.EcoRI.5S (5′-GGCAACAAGAATTCGCGATAAACG-3′) and AC3.XbaI.4AS (5′-GCGCTCTAGATGTCTACTTTCTCTTGCAAC-3′). The abu-1 coding region fragment coding ABU-1 amino acids 22–331 was amplified by PCR using AC3.EcoRI.5S and AC3.XbaI.6AS (5′-GGCATCTAGAGCTTGGCATGCAGATTGGC-3′), deleting the transmembrane domain and COOH terminus of ABU-1. These fragments were cloned into pcDNA3 containing a Flag epitope with signal peptides. COS-1 cells were transfected with wild-type ABU-1 or ABU-1 lacking the transmembrane domain with a Flag tag. Newly synthesized proteins were labeled in vivo with 35S-methionine and 35S-cysteine (500 μCi/ml; ICN Biochemicals). After removal of the labeling media, cells were washed and incubated in complete media for cold chase for the indicated times. Cells were washed and lysed with 1% Triton buffer (20 mM Hepes, pH 7.5, 150 mM NaCl, 1% Triton X-100, 10% glycerol, 1 mM EDTA, 1 mM PMSF, 4 mg/ml aprotinin, and 2 mg/ml pepstatin A). Samples were clarified by centrifugation at 14,000 g for 10 min and by preincubation for 1 h with 10 μl protein A–Sepharose. Soluble proteins from the cells and secreted proteins in the media were immunoprecipitated by anti-Flag antibody bound to 10 μl protein A–Sepharose and washed three times with RIPA buffer. Bound proteins were resolved by 10% SDS-PAGE under reducing conditions and revealed by autoradiography.

Cell culture, transfection, immunocytochemistry, and pulse-chase analysis

COS-1 cells were obtained from American Type Culture Collection. All cells were grown in DME containing 10% FBS (Cellgro®) at 5% CO2 and 37°C.

ABU-1 was tagged with a Flag epitope at its COOH terminus by overlapping PCR using the primers AC3.Bam.7S (5′-GCGCGGATCCGCAATGCGCTTTATCGCAATTGCAG-3′), AC3.Flag.16AS (5′-ATCGTCGTCCTTGTAGTCCTTTCTCTTGCAACACTG-3′), and Flag3′U.Not (5′-CTAG-CGGCCGCTCACTTGTCATCGTCGTCCTTGTAGTC-3′). The full-length coding region of abu-1 with the COOH-terminal Flag tag was cloned as a BamHI-NotI fragment into pcDNA3 (Invitrogen). COS-1 cells were transfected with this plasmid and stained with anti-Flag antibody (Sigma-Aldrich). Rabbit anti–ribophorin I antibody (a gift from Dr. Gert Kreibich, New York University [NYU] School of Medicine) was used to visualize the ER staining pattern.

The abu-1 coding region fragment encoding ABU-1 amino acids 22–425 was amplified by PCR using primers AC3.EcoRI.5S (5′-GGCAACAAGAATTCGCGATAAACG-3′) and AC3.XbaI.4AS (5′-GCGCTCTAGATGTCTACTTTCTCTTGCAAC-3′). The abu-1 coding region fragment coding ABU-1 amino acids 22–331 was amplified by PCR using AC3.EcoRI.5S and AC3.XbaI.6AS (5′-GGCATCTAGAGCTTGGCATGCAGATTGGC-3′), deleting the transmembrane domain and COOH terminus of ABU-1. These fragments were cloned into pcDNA3 containing a Flag epitope with signal peptides. COS-1 cells were transfected with wild-type ABU-1 or ABU-1 lacking the transmembrane domain with a Flag tag. Newly synthesized proteins were labeled in vivo with 35S-methionine and 35S-cysteine (500 μCi/ml; ICN Biochemicals). After removal of the labeling media, cells were washed and incubated in complete media for cold chase for the indicated times. Cells were washed and lysed with 1% Triton buffer (20 mM Hepes, pH 7.5, 150 mM NaCl, 1% Triton X-100, 10% glycerol, 1 mM EDTA, 1 mM PMSF, 4 mg/ml aprotinin, and 2 mg/ml pepstatin A). Samples were clarified by centrifugation at 14,000 g for 10 min and by preincubation for 1 h with 10 μl protein A–Sepharose. Soluble proteins from the cells and secreted proteins in the media were immunoprecipitated by anti-Flag antibody bound to 10 μl protein A–Sepharose and washed three times with RIPA buffer. Bound proteins were resolved by 10% SDS-PAGE under reducing conditions and revealed by autoradiography.
