---
pmid: '28733458'
title: Mapping of Wnt-Frizzled interactions by multiplex CRISPR targeting of receptor
  gene families.
authors:
- Voloshanenko O
- Gmach P
- Winter J
- Kranz D
- Boutros M
journal: FASEB J
year: '2017'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC5636703
doi: 10.1096/fj.201700144R
---

# Mapping of Wnt-Frizzled interactions by multiplex CRISPR targeting of receptor gene families.
**Authors:** Voloshanenko O, Gmach P, Winter J, Kranz D, Boutros M
**Journal:** FASEB J (2017)
**DOI:** [10.1096/fj.201700144R](https://doi.org/10.1096/fj.201700144R)
**PMC:** [PMC5636703](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5636703/)

## Abstract

1. FASEB J. 2017 Nov;31(11):4832-4844. doi: 10.1096/fj.201700144R. Epub 2017 Jul 
21.

Mapping of Wnt-Frizzled interactions by multiplex CRISPR targeting of receptor 
gene families.

Voloshanenko O(1), Gmach P(2), Winter J(2), Kranz D(2), Boutros M(3).

Author information:
(1)Division of Signaling and Functional Genomics, German Cancer Research Center 
(DKFZ), Heidelberg, Germany; and Department of Cell and Molecular Biology, 
Medical Faculty Mannheim, Heidelberg University, Heidelberg, Germany 
o.voloshanenko@dkfz.de.
(2)Division of Signaling and Functional Genomics, German Cancer Research Center 
(DKFZ), Heidelberg, Germany; and Department of Cell and Molecular Biology, 
Medical Faculty Mannheim, Heidelberg University, Heidelberg, Germany.
(3)Division of Signaling and Functional Genomics, German Cancer Research Center 
(DKFZ), Heidelberg, Germany; and Department of Cell and Molecular Biology, 
Medical Faculty Mannheim, Heidelberg University, Heidelberg, Germany 
m.boutros@dkfz.de.

Signaling pathway modules are often encoded by several closely related 
paralogous genes that can have redundant roles and are therefore difficult to 
analyze by loss-of-function analysis. A typical example is the Wnt signaling 
pathway, which in mammals is mediated by 19 Wnt ligands that can bind to 10 
Frizzled (FZD) receptors. Although significant progress in understanding Wnt-FZD 
receptor interactions has been made in recent years, tools to generate 
systematic interaction maps have been largely lacking. Here we generated cell 
lines with multiplex mutant alleles of FZD1, FZD2, and FZD7 and demonstrate that 
these cells are unresponsive to canonical Wnt ligands. Subsequently, we 
performed genetic rescue experiments with combinations of FZDs and canonical 
Wnts to create a functional ligand-receptor interaction map. These experiments 
showed that whereas several Wnt ligands, such as Wnt3a, induce signaling through 
a broad spectrum of FZD receptors, others, such as Wnt8a, act through a 
restricted set of FZD genes. Together, our results map functional interactions 
of FZDs and 10 Wnt ligands and demonstrate how multiplex targeting by clustered 
regularly interspaced short palindromic repeat (CRISPR)/Cas9 can be used to 
systematically elucidate the functions of multigene families.-Voloshanenko, O., 
Gmach, P., Winter, J., Kranz, D., Boutros, M. Mapping of Wnt-Frizzled 
interactions by multiplex CRISPR targeting of receptor gene families.

© The Author(s).

DOI: 10.1096/fj.201700144R
PMCID: PMC5636703
PMID: 28733458 [Indexed for MEDLINE]

## Full Text

Wnt signaling is required for a broad range of developmental processes, including cell fate decisions, tissue patterning, and homeostasis in all metazoan animals (1, 2). Aberrant Wnt signaling in humans has been implicated in many diseases, including cancer (3, 4). Wnt ligands are lipidated and glycosylated proteins that can act in an autocrine or paracrine manner. In the secretory pathway, Wnt proteins are S-palmitoylated (5, 6) by the N-acyltransferase Porcupine (Porcn) (7, 8), which can be targeted by pharmacologic inhibitors (9, 10). Lipidated Wnt proteins then bind to the transmembrane protein Evenness interrupted/Wntless/GPR177 (Evi/Wls) (11, 12) and are transported to the plasma membrane and shuttled onto extracellular carriers such as exosomes and other lipid structures (13, 14). After ligands reach the target cell, they induce signaling responses by interacting with members of the Frizzled (FZD) receptor family (15, 16). In mammalian genomes, the Wnt and FZD protein families are encoded by 19 and 10 paralogous genes, respectively. As the first inhibitors targeting Wnt pathways in disease are entering clinical studies, understanding the landscape of Wnt-FZD interactions is an important issue.

Binding of canonical Wnt ligands, such as Wnt1, Wnt3/3a, Wnt7a/b, Wnt8a/b, and Wnt10a/b (17), to a complex of FZD receptors and to low-density lipoprotein receptor–related protein (LRP) 5/6 coreceptor leads to the accumulation of β-catenin. In the absence of canonical Wnt signaling, β-catenin binds to 2 scaffold proteins, adenomatous polyposis coli and axis inhibition protein 1, which are components of the destruction complex (3, 18). β-Catenin is phosphorylated by the serine/threonine kinases casein kinase 1 α and glycogen synthase kinase 3β, then interacts with the β-transducin repeat-containing protein of the SKP1-Cullin1-F-box E3 ubiquitin ligase complex, thus leading to the ubiquitylation and subsequent proteasomal degradation of cytoplasmic β-catenin (19). When β-catenin accumulates, it enters the nucleus and leads to the formation of complexes with T-cell factor (TCF)/lymphoid enhancer factor proteins and the activation of target genes.

Many Wnt-FZD pairs have in principle the ability to activate β-catenin-dependent signaling (20), but direct biochemical studies are often limited by the hydrophobic nature of lipid-modified Wnt proteins and the requirement of the lipid adducts for receptor binding (5, 21–26). Progress has been made in elucidating receptor interactions for selected Wnt ligands, such as Wnt1, Wnt3/3a, Wnt4, and Wnt5a/b (20, 27–31). However, a systematic map of human canonical Wnt-FZD functional interactions has not yet been defined.

In this study, we generated FZD-deficient human embryonic kidney (HEK293T) cell lines to perform selective Wnt-FZD rescue experiments. We designed highly effective single guide (sg) RNAs that target highly homologous regions of FZD receptors and that generate multiple simultaneous mutations in FZD family members. Frameshift mutations were confirmed using next-generation sequencing. We then used these mutant cells to conduct pairwise genetic rescue experiments that confirmed known and identified new Wnt-FZD interactions.

MATERIALS AND METHODSCell cultureHEK293T cells were cultured in DMEM (Thermo Fisher Scientific, Waltham, MA, USA) supplemented with 10% fetal bovine serum (Biochrom, Berlin, Germany) without antibiotics. HCT116 TCF4/Wnt-reporter cells were cultured in McCoy medium (Thermo Fisher Scientific), supplemented with 100 μg/ml of hygromycin and 10% fetal bovine serum (Biochrom). Cell lines were obtained from the American Type Culture Collection (Manassas, VA, USA) and authenticated by single nucleotide polymorphism–based authentication (Multiplexion, Heidelberg, Germany).sgRNA designMultitargeting sgRNAs for FZD genes were identified by aligning the gene sequences using ClustalOmega (32). For FZD1, FZD2, and FZD7, highly conserved regions were identified in cysteine-rich domains (CRDs) and in the seventh transmembrane region. For FZD1,2,4,5,7,8, a region at the start of the seventh transmembrane region was highly conserved among all 5 FZDs. The 18- to 20-nt almost identical target sequences were selected to precede a NGG protospacer-adjacent motif (PAM) sequence. For evaluating their predicted specificity of identified sgRNAs, the sgRNA design tool E-CRISP was used (33). All identified sgRNAs were specific for FZD genes and did not have predicted off-target binding sites. Oligonucleotides were purchased from Eurofins (Ebersberg, Germany), then annealed and cloned into the Bbs1-Bbs1 (BpiI, Fermentas; Thermo Fisher Scientific) sites downstream of the human U6 promoter in a px459 vector (48139; Addgene, Cambridge, MA, USA) as previously described (1). Notably, 5′-CACC-3′ and 5′-AAAC-3′ are adapters required for cloning of sgRNAs (Table 1) into the px459 vector backbone.TABLE 1.sgRNA sequencessgRNASequence, 5′–3′sgFZD1,2,7-IGGGCGATGTCCGTGCACAGCsgFZD1,2,7-IIIgATCTCCATCCCGCTGTGCAsgFZD1,2,4,5,7,8gATGGCCAGCTCCATCTGGTsgFZD1,2,7-IIgCGCGATGTCCGTGCACACCGeneration of FZD mutant cellsFZD1,2,7sgRNA-I, FZD1,2,7sgRNA-III, and FZD1,2,4,5,7,8sgRNA clones and pools of HEK293T or FZD1,2,7sgRNA-I, or FZD1,2,7sgRNA-III HCT116 cells were generated through methods similar to those previously described (1). The px459 vector expressing an sgRNA and Cas9 was transiently transfected. After 48 h, selection with 1 to 2 μg/ml puromycin for 48 to 72 h was performed. For experiments with cell pools of HEK293T, cells were grown for at least 7 d before a genome-editing experiment. All pools of HCT116 cells were used for the experiments within 3 to 5 d because the growth of HCT116 cells is dependent on canonical Wnt signaling. FZD1,2,7sgRNA-I, FZD1,2,7sgRNA-III, and FZD1,2,4,5,7,8sgRNA clonal cell lines were generated by serial dilutions. In brief, 10 cells per well of a 96-well plate were seeded in the first rows of a plate in 220 µl of medium. Then 20 µl of sample was moved to a new well with a new 200 µl of medium and mixed, and this process was repeated another 3 times. Rows that harbored grown cells in every third well were taken for further culturing and analysis, as previously described (1).Mutation analysis by indel-nested PCRTo detect mutations in clones of CRISPR (clustered regularly interspaced short palindromic repeat)/Cas9-generated cell lines, primer pairs 100 to 150 up- and downstream of sgRNA target regions were generated with Primer3 Web (34). Primer adapters were added for a second step of indel-nested PCR. Primer pairs were tested using wild-type (WT) DNA of HEK293T cells as templates. Only primers that amplified the corresponding FZD region were used in further analysis. Genomic DNA of FZDsgRNA clones was isolated using a DNeasy Blood and Tissue Kit (Qiagen, Germantown, MD, USA). Genomic regions of interest were amplified with 20 cycles with specific primer pairs (Table 2), which has forward adapter 5′-TCCCTACACGACGCTCTTCCGATCT-3′ in the case of forward primers and reverse adapter 3′-AGTTCAGACGTGTGCTCTTCCGATCT-5′ (reverse primers). Then PCR products were purified with a PCR Cleanup Kit (Machery-Nagel, Düren, Germany). PCR products were amplified using a 2-step indel-nested PCR with the oligos presented in Table
3 and prepared for MiSeq sequencing (Illumina, San Diego, CA, USA). For detection of mutations in amplified regions, we performed a multiple sequence alignment using ClustalOmega (https://www.ebi.ac.uk/Tools/msa/clustalo) (35).TABLE 2.Primers used for indel-nested PCR 1 for validation of mutations generated by sgRNAssgRNATarget geneForward primer, 5′–3′Reverse primer, 5′–3′sgFZD1,2,7-I/-IIIFZD1CCTCAGCAGCAACAGAGCGAACTTGAGCTCAGCGGAACFZD2GAGAAGGGCATCTCCATCCACCAGCGGATAGAACTGGTGFZD7CTTTCGTCCCTGGGCCTCAGCGGGTAGAACTGGTGCsgFZD1,2,4,5,7,8FZD1GGAAGACCGAGTGGTGTGTAACGGCCAGGTGAAAATACTGTFZD2GCTTCTCCGAGGACGGTTACCAGGATGGTGATGGTCTTGAFZD4AGCAGAACCTGTTCTCATCCATTCATCTGCATCCACCAGTCFZD5GCACAACCACATCCACTACGGTGCCGTGATGGACTTGACFZD7GCTTCTCGGACGATGGCTACATGGCCAGGATAGTGATGGTFZD8AGTACGAGGAGCTGGGCGCAACATGGTGCCGATGAAGAGTABLE 3.Library primers used for indel-nested PCR 2 in Illumina MiSeq sequencingNameSequence, 5′–3′F-Sq_Lib_D501AATGATACGGCGACCACCGAGATCTACAC-TATAGCCT-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D502AATGATACGGCGACCACCGAGATCTACAC-ATAGAGGC-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D503AATGATACGGCGACCACCGAGATCTACAC-CCTATCCT-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D504AATGATACGGCGACCACCGAGATCTACAC-GGCTCTGA-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D505AATGATACGGCGACCACCGAGATCTACAC-AGGCGAAG-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D506AATGATACGGCGACCACCGAGATCTACAC-TAATCTTA-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D507AATGATACGGCGACCACCGAGATCTACAC-CAGGACGT-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D508AATGATACGGCGACCACCGAGATCTACAC-GTACTGAC-ACACTCTTTCCCTACACGACGCTCTTCCGATCTR-Sq_Lib_D701CAAGCAGAAGACGGCATACGAGAT-CGAGTAAT-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D702CAAGCAGAAGACGGCATACGAGAT-TCTCCGGA-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D703CAAGCAGAAGACGGCATACGAGAT-AATGAGCG-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D704CAAGCAGAAGACGGCATACGAGAT-GGAATCTC-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D705CAAGCAGAAGACGGCATACGAGAT-TTCTGAAT-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D706CAAGCAGAAGACGGCATACGAGAT-ACGAATTC-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D707CAAGCAGAAGACGGCATACGAGAT-AGCTTCTG-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D708CAAGCAGAAGACGGCATACGAGAT-GCGCATTA-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D709CAAGCAGAAGACGGCATACGAGAT-CATAGCCG-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D710CAAGCAGAAGACGGCATACGAGAT-TTCGCGGA-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D711CAAGCAGAAGACGGCATACGAGAT-GCGCGAGA-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D712CAAGCAGAAGACGGCATACGAGAT-CTATCGCT-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCF-Sq_Lib are forward and R-Sq_Lib are reverse library primers. Unique index tags of primers are underlined.Luciferase assayLuciferase assays were performed in a 384-well format using white, flat-bottom polystyrene plates (Greiner, Mannheim, Germany). Cells were transfected with 20 ng of TCF4/Wnt firefly luciferase reporter, 10 ng of actin–Renilla luciferase reporter, and 20 ng of Wnts or control (pcDNA3)/β-catenin. The amount of DNA was normalized by the addition of a control plasmid (pcDNA3.1). Paracrine induction of Wnt signaling was achieved by treating cells with 100 ng/ml recombinant mouse Wnt3a (PeproTech, Hamburg, Germany) unless otherwise indicated. Wnt3a was added 16 h before the luciferase activity readout. Luminescence was measured with the Mithras LB940 plate reader (Berthold Technologies, Bad Wildbad, Germany). TCF4/Wnt-luciferase signal was normalized to the actin–Renilla luciferase reporter signal. Constructs used in this study are listed in Table 4.TABLE 4.Constructs used in this studySourceShort nameFull nameManufacturerCatalog no.Referencepx459pSpCas9(BB)-2A-Puro (PX459)Addgene481391TCF4/Wnt luciferase reporter6xKD; pGL4.26 6xTcf-Firefly luciferaseK. Demir, M.B. laboratory36Renilla reporterpAct-RL, Renilla luciferaseD. Nickles, M.B. laboratory37Wnt1pcDNA3 Wnt1M.B. laboratoryWnt3pcDNA3 Wnt3Addgene3590917Wnt3apcDNA3 Wnt3aAddgene3590817Wnt7apCMV-XL4 Wnt7aOrigeneSC117232Wnt7bpcDNA3 Wnt7bAddgene3591517Wnt8apcDNA3 Wnt8aAddgene3591617Wnt8bpcDNA3 Wnt8bAddgene3591717Wnt10apcDNA3 Wnt10aAddgene3592017Wnt10bpcDNA3 Wnt10bAddgene3592117β-CateninpcDNA3 β-cateninM.B. laboratoryFZD1pCMV-XL4 FZD1OrigeneSC117910FZD2pCMV-XL4 FZD2OrigeneSC127603FZD4pCMV6-XL6 FZD4OrigeneSC115479FZD5pcDNA3.2-FZD5-V5M.B. laboratory, recloned from pENTR223.1 FZD5, DKFZ–Vector and Clone Repository of GPCFFZD6pCMV6-XL4 FZD6OrigeneSC316995FZD7pCMV6-XL5 FZD7OrigeneSC122259FZD8pcDNA3.2-FZD8-V5M.B. laboratory, recloned from pENTR223.1 FZD8, DKFZ–Vector and Clone Repository of GPCFFZD9pCMV6-XL4OrigeneSC117912FZD10pCMV6-XL4OrigeneSC115678GPCF, Genomics and Proteomics Core Facility.Immunoblot analysisProtein extraction was performed using a Triton-containing lysis buffer: 20 mM Tris-HCl pH 7.4, 130 mM NaCl, 2 mM EDTA, 1% Triton X-100, complete protease inhibitors (Roche, Basel, Switzerland). Phosphatase inhibitor cocktails 2 (P5726; Sigma-Aldrich, St. Louis, MO, USA) and 3 (P0044; Sigma-Aldrich) were added to the lysis buffer according to the manufacturer’s instructions. NUPAGE 4–12% BT gels (Life Technologies, Carlsbad, CA, USA) and Protran Western blot nitrocellulose membrane (GE10600002; GE Healthcare, Waukesha, WI, USA) were used. Full scans of Western blots are shown in Supplemental Figs. 3 and 4. Antibodies used are listed in Table 5.TABLE 5.Antibodies used in this studyAntibodyReferenceDilutionCompanyHostActive β-catenin, non-P-β-catenin8E7/05-6651:1000MilliporeMouseβ-catenin6101541:3000BD BioscienceMouseWnt3a/Wnt3ab284721:1000AbcamRabbitβ-actinsc-477781:20,000Santa Cruz BiotechnologyMouseAnti-mouse IgG, H+L 115-035-0031:10,000Jackson/DianovaGoatAnti-rabbit IgG, H+L 111-035-0031:10,000Jackson/DianovaGoatRescue experimentsFZD and Wnt constructs used in this study are listed in Table 4. For luciferase assays, 5 ng of FZD plasmid or 5 ng of control plasmid (pcDNA3.1) per well was added. For rescue experiments of active β-catenin levels after expression of Wnt3, the FZD constructs were added at 100 ng per well in 6-well plates.Real-time quantitative PCRIn order to evaluate the gene expression pattern of each FZD receptor in HEK293T cells, we designed 2 to 4 primer pairs with corresponding probe using Roche Universal ProbeLibrary Assay Design Center. Primers were tested in several cell lines with different amount of templates. For cDNA preparation, RNA treated with Qiagen RNase-Free DNase was isolated with the Qiagen RNeasy Mini Kit. Then 1 to 2 µg of RNA was reverse transcribed with the help of RevertAid H Minus First Strand cDNA Synthesis Kit (Thermo Fisher Scientific). Subsequently the cDNA was diluted to 5 ng/µl and used for real-time quantitative PCR (qPCR) on the LightCycler 480 (Roche) using the universal probe library system (Roche) in a 384-well format. The ubiquitin C (UBC) gene was used as the reference gene for relative quantification. The amount of FZD1 was normalized to 1, and all other FZD RNA levels were shown relative to its expression. qPCR primers and probes are listed in Table
6.TABLE 6.TaqMan qPCR primers and probesGeneForwardReverseProbe No.FZD1CGGCAAGACCCTCAACTCCCTTGTTTGCTGTTGGTGAG69FZD2CGATGGTTCCATGTTCTTCTCGACCAGGTGAGGATCCAGAG63FZD3ACAGCAAAGTGAGCAGCTACCCTGTAACTGCAGGGCGTGTA75FZD4AAGGGTGATTCACTTTCTGACAAGGGGTCCGTCAGGTAAGTC6FZD5ATGAAGTGGGGCAACGAGGACTTGACGCTGGGGATG71FZD6GAAGCAAAAAGACATGCACAGATTCGACTTTCACTGATTGGATCT23FZD7GCCAGCTTGTGCCTAATAGAAAGCCGGGAGAAACTCACAG54FZD8GCCTGTTCACCGTGCTCTAGGTTGTGCTGCTCGTAGAAGA77FZD9CACTGTGCTCACCTTCTTGCGCCAGCGAGTAGACGTTGTA55FZD10ACTCGTATGCCCAACCTGAGCAGCCGTACTCCACCAG46Clustering and heat mapResults of the TCF4/Wnt firefly luciferase reporter in FZD1,2,7sgRNAI#1 cells upon reexpression of different FZDs were scaled to 1 for the maximum fold induction of Wnt/FZDs interaction. A heat map was generated out of the scaled data using the R software package “pheatmap” and clustering by Euclidian distance for both the FZDs and the Wnts (R Foundation for Statistical Computing, Vienna, Austria; http://www.r-project.org/). The script is available online (https://github.com/boutroslab/Supplemental-Material).

MATERIALS AND METHODS

Cell cultureHEK293T cells were cultured in DMEM (Thermo Fisher Scientific, Waltham, MA, USA) supplemented with 10% fetal bovine serum (Biochrom, Berlin, Germany) without antibiotics. HCT116 TCF4/Wnt-reporter cells were cultured in McCoy medium (Thermo Fisher Scientific), supplemented with 100 μg/ml of hygromycin and 10% fetal bovine serum (Biochrom). Cell lines were obtained from the American Type Culture Collection (Manassas, VA, USA) and authenticated by single nucleotide polymorphism–based authentication (Multiplexion, Heidelberg, Germany).

Cell culture

HEK293T cells were cultured in DMEM (Thermo Fisher Scientific, Waltham, MA, USA) supplemented with 10% fetal bovine serum (Biochrom, Berlin, Germany) without antibiotics. HCT116 TCF4/Wnt-reporter cells were cultured in McCoy medium (Thermo Fisher Scientific), supplemented with 100 μg/ml of hygromycin and 10% fetal bovine serum (Biochrom). Cell lines were obtained from the American Type Culture Collection (Manassas, VA, USA) and authenticated by single nucleotide polymorphism–based authentication (Multiplexion, Heidelberg, Germany).

sgRNA designMultitargeting sgRNAs for FZD genes were identified by aligning the gene sequences using ClustalOmega (32). For FZD1, FZD2, and FZD7, highly conserved regions were identified in cysteine-rich domains (CRDs) and in the seventh transmembrane region. For FZD1,2,4,5,7,8, a region at the start of the seventh transmembrane region was highly conserved among all 5 FZDs. The 18- to 20-nt almost identical target sequences were selected to precede a NGG protospacer-adjacent motif (PAM) sequence. For evaluating their predicted specificity of identified sgRNAs, the sgRNA design tool E-CRISP was used (33). All identified sgRNAs were specific for FZD genes and did not have predicted off-target binding sites. Oligonucleotides were purchased from Eurofins (Ebersberg, Germany), then annealed and cloned into the Bbs1-Bbs1 (BpiI, Fermentas; Thermo Fisher Scientific) sites downstream of the human U6 promoter in a px459 vector (48139; Addgene, Cambridge, MA, USA) as previously described (1). Notably, 5′-CACC-3′ and 5′-AAAC-3′ are adapters required for cloning of sgRNAs (Table 1) into the px459 vector backbone.TABLE 1.sgRNA sequencessgRNASequence, 5′–3′sgFZD1,2,7-IGGGCGATGTCCGTGCACAGCsgFZD1,2,7-IIIgATCTCCATCCCGCTGTGCAsgFZD1,2,4,5,7,8gATGGCCAGCTCCATCTGGTsgFZD1,2,7-IIgCGCGATGTCCGTGCACACC

sgRNA design

Multitargeting sgRNAs for FZD genes were identified by aligning the gene sequences using ClustalOmega (32). For FZD1, FZD2, and FZD7, highly conserved regions were identified in cysteine-rich domains (CRDs) and in the seventh transmembrane region. For FZD1,2,4,5,7,8, a region at the start of the seventh transmembrane region was highly conserved among all 5 FZDs. The 18- to 20-nt almost identical target sequences were selected to precede a NGG protospacer-adjacent motif (PAM) sequence. For evaluating their predicted specificity of identified sgRNAs, the sgRNA design tool E-CRISP was used (33). All identified sgRNAs were specific for FZD genes and did not have predicted off-target binding sites. Oligonucleotides were purchased from Eurofins (Ebersberg, Germany), then annealed and cloned into the Bbs1-Bbs1 (BpiI, Fermentas; Thermo Fisher Scientific) sites downstream of the human U6 promoter in a px459 vector (48139; Addgene, Cambridge, MA, USA) as previously described (1). Notably, 5′-CACC-3′ and 5′-AAAC-3′ are adapters required for cloning of sgRNAs (Table 1) into the px459 vector backbone.

sgRNA sequences

Generation of FZD mutant cellsFZD1,2,7sgRNA-I, FZD1,2,7sgRNA-III, and FZD1,2,4,5,7,8sgRNA clones and pools of HEK293T or FZD1,2,7sgRNA-I, or FZD1,2,7sgRNA-III HCT116 cells were generated through methods similar to those previously described (1). The px459 vector expressing an sgRNA and Cas9 was transiently transfected. After 48 h, selection with 1 to 2 μg/ml puromycin for 48 to 72 h was performed. For experiments with cell pools of HEK293T, cells were grown for at least 7 d before a genome-editing experiment. All pools of HCT116 cells were used for the experiments within 3 to 5 d because the growth of HCT116 cells is dependent on canonical Wnt signaling. FZD1,2,7sgRNA-I, FZD1,2,7sgRNA-III, and FZD1,2,4,5,7,8sgRNA clonal cell lines were generated by serial dilutions. In brief, 10 cells per well of a 96-well plate were seeded in the first rows of a plate in 220 µl of medium. Then 20 µl of sample was moved to a new well with a new 200 µl of medium and mixed, and this process was repeated another 3 times. Rows that harbored grown cells in every third well were taken for further culturing and analysis, as previously described (1).

Generation of FZD mutant cells

FZD1,2,7sgRNA-I, FZD1,2,7sgRNA-III, and FZD1,2,4,5,7,8sgRNA clones and pools of HEK293T or FZD1,2,7sgRNA-I, or FZD1,2,7sgRNA-III HCT116 cells were generated through methods similar to those previously described (1). The px459 vector expressing an sgRNA and Cas9 was transiently transfected. After 48 h, selection with 1 to 2 μg/ml puromycin for 48 to 72 h was performed. For experiments with cell pools of HEK293T, cells were grown for at least 7 d before a genome-editing experiment. All pools of HCT116 cells were used for the experiments within 3 to 5 d because the growth of HCT116 cells is dependent on canonical Wnt signaling. FZD1,2,7sgRNA-I, FZD1,2,7sgRNA-III, and FZD1,2,4,5,7,8sgRNA clonal cell lines were generated by serial dilutions. In brief, 10 cells per well of a 96-well plate were seeded in the first rows of a plate in 220 µl of medium. Then 20 µl of sample was moved to a new well with a new 200 µl of medium and mixed, and this process was repeated another 3 times. Rows that harbored grown cells in every third well were taken for further culturing and analysis, as previously described (1).

Mutation analysis by indel-nested PCRTo detect mutations in clones of CRISPR (clustered regularly interspaced short palindromic repeat)/Cas9-generated cell lines, primer pairs 100 to 150 up- and downstream of sgRNA target regions were generated with Primer3 Web (34). Primer adapters were added for a second step of indel-nested PCR. Primer pairs were tested using wild-type (WT) DNA of HEK293T cells as templates. Only primers that amplified the corresponding FZD region were used in further analysis. Genomic DNA of FZDsgRNA clones was isolated using a DNeasy Blood and Tissue Kit (Qiagen, Germantown, MD, USA). Genomic regions of interest were amplified with 20 cycles with specific primer pairs (Table 2), which has forward adapter 5′-TCCCTACACGACGCTCTTCCGATCT-3′ in the case of forward primers and reverse adapter 3′-AGTTCAGACGTGTGCTCTTCCGATCT-5′ (reverse primers). Then PCR products were purified with a PCR Cleanup Kit (Machery-Nagel, Düren, Germany). PCR products were amplified using a 2-step indel-nested PCR with the oligos presented in Table
3 and prepared for MiSeq sequencing (Illumina, San Diego, CA, USA). For detection of mutations in amplified regions, we performed a multiple sequence alignment using ClustalOmega (https://www.ebi.ac.uk/Tools/msa/clustalo) (35).TABLE 2.Primers used for indel-nested PCR 1 for validation of mutations generated by sgRNAssgRNATarget geneForward primer, 5′–3′Reverse primer, 5′–3′sgFZD1,2,7-I/-IIIFZD1CCTCAGCAGCAACAGAGCGAACTTGAGCTCAGCGGAACFZD2GAGAAGGGCATCTCCATCCACCAGCGGATAGAACTGGTGFZD7CTTTCGTCCCTGGGCCTCAGCGGGTAGAACTGGTGCsgFZD1,2,4,5,7,8FZD1GGAAGACCGAGTGGTGTGTAACGGCCAGGTGAAAATACTGTFZD2GCTTCTCCGAGGACGGTTACCAGGATGGTGATGGTCTTGAFZD4AGCAGAACCTGTTCTCATCCATTCATCTGCATCCACCAGTCFZD5GCACAACCACATCCACTACGGTGCCGTGATGGACTTGACFZD7GCTTCTCGGACGATGGCTACATGGCCAGGATAGTGATGGTFZD8AGTACGAGGAGCTGGGCGCAACATGGTGCCGATGAAGAGTABLE 3.Library primers used for indel-nested PCR 2 in Illumina MiSeq sequencingNameSequence, 5′–3′F-Sq_Lib_D501AATGATACGGCGACCACCGAGATCTACAC-TATAGCCT-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D502AATGATACGGCGACCACCGAGATCTACAC-ATAGAGGC-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D503AATGATACGGCGACCACCGAGATCTACAC-CCTATCCT-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D504AATGATACGGCGACCACCGAGATCTACAC-GGCTCTGA-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D505AATGATACGGCGACCACCGAGATCTACAC-AGGCGAAG-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D506AATGATACGGCGACCACCGAGATCTACAC-TAATCTTA-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D507AATGATACGGCGACCACCGAGATCTACAC-CAGGACGT-ACACTCTTTCCCTACACGACGCTCTTCCGATCTF-Sq_Lib_D508AATGATACGGCGACCACCGAGATCTACAC-GTACTGAC-ACACTCTTTCCCTACACGACGCTCTTCCGATCTR-Sq_Lib_D701CAAGCAGAAGACGGCATACGAGAT-CGAGTAAT-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D702CAAGCAGAAGACGGCATACGAGAT-TCTCCGGA-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D703CAAGCAGAAGACGGCATACGAGAT-AATGAGCG-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D704CAAGCAGAAGACGGCATACGAGAT-GGAATCTC-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D705CAAGCAGAAGACGGCATACGAGAT-TTCTGAAT-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D706CAAGCAGAAGACGGCATACGAGAT-ACGAATTC-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D707CAAGCAGAAGACGGCATACGAGAT-AGCTTCTG-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D708CAAGCAGAAGACGGCATACGAGAT-GCGCATTA-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D709CAAGCAGAAGACGGCATACGAGAT-CATAGCCG-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D710CAAGCAGAAGACGGCATACGAGAT-TTCGCGGA-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D711CAAGCAGAAGACGGCATACGAGAT-GCGCGAGA-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCR-Sq_Lib_D712CAAGCAGAAGACGGCATACGAGAT-CTATCGCT-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCF-Sq_Lib are forward and R-Sq_Lib are reverse library primers. Unique index tags of primers are underlined.

Mutation analysis by indel-nested PCR

To detect mutations in clones of CRISPR (clustered regularly interspaced short palindromic repeat)/Cas9-generated cell lines, primer pairs 100 to 150 up- and downstream of sgRNA target regions were generated with Primer3 Web (34). Primer adapters were added for a second step of indel-nested PCR. Primer pairs were tested using wild-type (WT) DNA of HEK293T cells as templates. Only primers that amplified the corresponding FZD region were used in further analysis. Genomic DNA of FZDsgRNA clones was isolated using a DNeasy Blood and Tissue Kit (Qiagen, Germantown, MD, USA). Genomic regions of interest were amplified with 20 cycles with specific primer pairs (Table 2), which has forward adapter 5′-TCCCTACACGACGCTCTTCCGATCT-3′ in the case of forward primers and reverse adapter 3′-AGTTCAGACGTGTGCTCTTCCGATCT-5′ (reverse primers). Then PCR products were purified with a PCR Cleanup Kit (Machery-Nagel, Düren, Germany). PCR products were amplified using a 2-step indel-nested PCR with the oligos presented in Table
3 and prepared for MiSeq sequencing (Illumina, San Diego, CA, USA). For detection of mutations in amplified regions, we performed a multiple sequence alignment using ClustalOmega (https://www.ebi.ac.uk/Tools/msa/clustalo) (35).

Primers used for indel-nested PCR 1 for validation of mutations generated by sgRNAs

Library primers used for indel-nested PCR 2 in Illumina MiSeq sequencing

F-Sq_Lib are forward and R-Sq_Lib are reverse library primers. Unique index tags of primers are underlined.

Luciferase assayLuciferase assays were performed in a 384-well format using white, flat-bottom polystyrene plates (Greiner, Mannheim, Germany). Cells were transfected with 20 ng of TCF4/Wnt firefly luciferase reporter, 10 ng of actin–Renilla luciferase reporter, and 20 ng of Wnts or control (pcDNA3)/β-catenin. The amount of DNA was normalized by the addition of a control plasmid (pcDNA3.1). Paracrine induction of Wnt signaling was achieved by treating cells with 100 ng/ml recombinant mouse Wnt3a (PeproTech, Hamburg, Germany) unless otherwise indicated. Wnt3a was added 16 h before the luciferase activity readout. Luminescence was measured with the Mithras LB940 plate reader (Berthold Technologies, Bad Wildbad, Germany). TCF4/Wnt-luciferase signal was normalized to the actin–Renilla luciferase reporter signal. Constructs used in this study are listed in Table 4.TABLE 4.Constructs used in this studySourceShort nameFull nameManufacturerCatalog no.Referencepx459pSpCas9(BB)-2A-Puro (PX459)Addgene481391TCF4/Wnt luciferase reporter6xKD; pGL4.26 6xTcf-Firefly luciferaseK. Demir, M.B. laboratory36Renilla reporterpAct-RL, Renilla luciferaseD. Nickles, M.B. laboratory37Wnt1pcDNA3 Wnt1M.B. laboratoryWnt3pcDNA3 Wnt3Addgene3590917Wnt3apcDNA3 Wnt3aAddgene3590817Wnt7apCMV-XL4 Wnt7aOrigeneSC117232Wnt7bpcDNA3 Wnt7bAddgene3591517Wnt8apcDNA3 Wnt8aAddgene3591617Wnt8bpcDNA3 Wnt8bAddgene3591717Wnt10apcDNA3 Wnt10aAddgene3592017Wnt10bpcDNA3 Wnt10bAddgene3592117β-CateninpcDNA3 β-cateninM.B. laboratoryFZD1pCMV-XL4 FZD1OrigeneSC117910FZD2pCMV-XL4 FZD2OrigeneSC127603FZD4pCMV6-XL6 FZD4OrigeneSC115479FZD5pcDNA3.2-FZD5-V5M.B. laboratory, recloned from pENTR223.1 FZD5, DKFZ–Vector and Clone Repository of GPCFFZD6pCMV6-XL4 FZD6OrigeneSC316995FZD7pCMV6-XL5 FZD7OrigeneSC122259FZD8pcDNA3.2-FZD8-V5M.B. laboratory, recloned from pENTR223.1 FZD8, DKFZ–Vector and Clone Repository of GPCFFZD9pCMV6-XL4OrigeneSC117912FZD10pCMV6-XL4OrigeneSC115678GPCF, Genomics and Proteomics Core Facility.

Luciferase assay

Luciferase assays were performed in a 384-well format using white, flat-bottom polystyrene plates (Greiner, Mannheim, Germany). Cells were transfected with 20 ng of TCF4/Wnt firefly luciferase reporter, 10 ng of actin–Renilla luciferase reporter, and 20 ng of Wnts or control (pcDNA3)/β-catenin. The amount of DNA was normalized by the addition of a control plasmid (pcDNA3.1). Paracrine induction of Wnt signaling was achieved by treating cells with 100 ng/ml recombinant mouse Wnt3a (PeproTech, Hamburg, Germany) unless otherwise indicated. Wnt3a was added 16 h before the luciferase activity readout. Luminescence was measured with the Mithras LB940 plate reader (Berthold Technologies, Bad Wildbad, Germany). TCF4/Wnt-luciferase signal was normalized to the actin–Renilla luciferase reporter signal. Constructs used in this study are listed in Table 4.

Constructs used in this study

GPCF, Genomics and Proteomics Core Facility.

Immunoblot analysisProtein extraction was performed using a Triton-containing lysis buffer: 20 mM Tris-HCl pH 7.4, 130 mM NaCl, 2 mM EDTA, 1% Triton X-100, complete protease inhibitors (Roche, Basel, Switzerland). Phosphatase inhibitor cocktails 2 (P5726; Sigma-Aldrich, St. Louis, MO, USA) and 3 (P0044; Sigma-Aldrich) were added to the lysis buffer according to the manufacturer’s instructions. NUPAGE 4–12% BT gels (Life Technologies, Carlsbad, CA, USA) and Protran Western blot nitrocellulose membrane (GE10600002; GE Healthcare, Waukesha, WI, USA) were used. Full scans of Western blots are shown in Supplemental Figs. 3 and 4. Antibodies used are listed in Table 5.TABLE 5.Antibodies used in this studyAntibodyReferenceDilutionCompanyHostActive β-catenin, non-P-β-catenin8E7/05-6651:1000MilliporeMouseβ-catenin6101541:3000BD BioscienceMouseWnt3a/Wnt3ab284721:1000AbcamRabbitβ-actinsc-477781:20,000Santa Cruz BiotechnologyMouseAnti-mouse IgG, H+L 115-035-0031:10,000Jackson/DianovaGoatAnti-rabbit IgG, H+L 111-035-0031:10,000Jackson/DianovaGoat

Immunoblot analysis

Protein extraction was performed using a Triton-containing lysis buffer: 20 mM Tris-HCl pH 7.4, 130 mM NaCl, 2 mM EDTA, 1% Triton X-100, complete protease inhibitors (Roche, Basel, Switzerland). Phosphatase inhibitor cocktails 2 (P5726; Sigma-Aldrich, St. Louis, MO, USA) and 3 (P0044; Sigma-Aldrich) were added to the lysis buffer according to the manufacturer’s instructions. NUPAGE 4–12% BT gels (Life Technologies, Carlsbad, CA, USA) and Protran Western blot nitrocellulose membrane (GE10600002; GE Healthcare, Waukesha, WI, USA) were used. Full scans of Western blots are shown in Supplemental Figs. 3 and 4. Antibodies used are listed in Table 5.

Antibodies used in this study

Rescue experimentsFZD and Wnt constructs used in this study are listed in Table 4. For luciferase assays, 5 ng of FZD plasmid or 5 ng of control plasmid (pcDNA3.1) per well was added. For rescue experiments of active β-catenin levels after expression of Wnt3, the FZD constructs were added at 100 ng per well in 6-well plates.

Rescue experiments

FZD and Wnt constructs used in this study are listed in Table 4. For luciferase assays, 5 ng of FZD plasmid or 5 ng of control plasmid (pcDNA3.1) per well was added. For rescue experiments of active β-catenin levels after expression of Wnt3, the FZD constructs were added at 100 ng per well in 6-well plates.

Real-time quantitative PCRIn order to evaluate the gene expression pattern of each FZD receptor in HEK293T cells, we designed 2 to 4 primer pairs with corresponding probe using Roche Universal ProbeLibrary Assay Design Center. Primers were tested in several cell lines with different amount of templates. For cDNA preparation, RNA treated with Qiagen RNase-Free DNase was isolated with the Qiagen RNeasy Mini Kit. Then 1 to 2 µg of RNA was reverse transcribed with the help of RevertAid H Minus First Strand cDNA Synthesis Kit (Thermo Fisher Scientific). Subsequently the cDNA was diluted to 5 ng/µl and used for real-time quantitative PCR (qPCR) on the LightCycler 480 (Roche) using the universal probe library system (Roche) in a 384-well format. The ubiquitin C (UBC) gene was used as the reference gene for relative quantification. The amount of FZD1 was normalized to 1, and all other FZD RNA levels were shown relative to its expression. qPCR primers and probes are listed in Table
6.TABLE 6.TaqMan qPCR primers and probesGeneForwardReverseProbe No.FZD1CGGCAAGACCCTCAACTCCCTTGTTTGCTGTTGGTGAG69FZD2CGATGGTTCCATGTTCTTCTCGACCAGGTGAGGATCCAGAG63FZD3ACAGCAAAGTGAGCAGCTACCCTGTAACTGCAGGGCGTGTA75FZD4AAGGGTGATTCACTTTCTGACAAGGGGTCCGTCAGGTAAGTC6FZD5ATGAAGTGGGGCAACGAGGACTTGACGCTGGGGATG71FZD6GAAGCAAAAAGACATGCACAGATTCGACTTTCACTGATTGGATCT23FZD7GCCAGCTTGTGCCTAATAGAAAGCCGGGAGAAACTCACAG54FZD8GCCTGTTCACCGTGCTCTAGGTTGTGCTGCTCGTAGAAGA77FZD9CACTGTGCTCACCTTCTTGCGCCAGCGAGTAGACGTTGTA55FZD10ACTCGTATGCCCAACCTGAGCAGCCGTACTCCACCAG46

Real-time quantitative PCR

In order to evaluate the gene expression pattern of each FZD receptor in HEK293T cells, we designed 2 to 4 primer pairs with corresponding probe using Roche Universal ProbeLibrary Assay Design Center. Primers were tested in several cell lines with different amount of templates. For cDNA preparation, RNA treated with Qiagen RNase-Free DNase was isolated with the Qiagen RNeasy Mini Kit. Then 1 to 2 µg of RNA was reverse transcribed with the help of RevertAid H Minus First Strand cDNA Synthesis Kit (Thermo Fisher Scientific). Subsequently the cDNA was diluted to 5 ng/µl and used for real-time quantitative PCR (qPCR) on the LightCycler 480 (Roche) using the universal probe library system (Roche) in a 384-well format. The ubiquitin C (UBC) gene was used as the reference gene for relative quantification. The amount of FZD1 was normalized to 1, and all other FZD RNA levels were shown relative to its expression. qPCR primers and probes are listed in Table
6.

TaqMan qPCR primers and probes

Clustering and heat mapResults of the TCF4/Wnt firefly luciferase reporter in FZD1,2,7sgRNAI#1 cells upon reexpression of different FZDs were scaled to 1 for the maximum fold induction of Wnt/FZDs interaction. A heat map was generated out of the scaled data using the R software package “pheatmap” and clustering by Euclidian distance for both the FZDs and the Wnts (R Foundation for Statistical Computing, Vienna, Austria; http://www.r-project.org/). The script is available online (https://github.com/boutroslab/Supplemental-Material).

Clustering and heat map

Results of the TCF4/Wnt firefly luciferase reporter in FZD1,2,7sgRNAI#1 cells upon reexpression of different FZDs were scaled to 1 for the maximum fold induction of Wnt/FZDs interaction. A heat map was generated out of the scaled data using the R software package “pheatmap” and clustering by Euclidian distance for both the FZDs and the Wnts (R Foundation for Statistical Computing, Vienna, Austria; http://www.r-project.org/). The script is available online (https://github.com/boutroslab/Supplemental-Material).

RESULTSGeneration of FZD-mutant cells by a single sgRNAMost human tissues and cell lines express multiple FZD receptors (38) that may act in a redundant manner. To create a genetic background for selective rescue experiments, we generated FZD loss-of-function HEK293T cell lines. HEK293T cells have been extensively used for the analysis of canonical Wnt signaling activity (36). HEK293T cells express high levels of FZD2 and FZD7, and to a lesser extent FZD1, FZD4, and FZD9 (Fig. 1A). Moreover, we detected mRNA expression of the sequence-divergent FZD3 and FZD6 receptors, which have been reported to be involved in the noncanonical, β-catenin-independent signaling branch (39–42).Figure 1.Multiple targeting of FZD homologous genes with single sgRNA is functional. A) RNA expression of FZDs in HEK293T cells. qPCR analysis of FZD receptors expression normalized to UBC levels in HEK293T cells. FZD levels were compared to FZD1 expression. Results are means ± sem of 3 independent experiments. B) Schematic representation of FZD proteins and sequences targeted by sgRNAs. SS indicates N-terminal signal sequence; CRD, extracellular cysteine-rich domain required for Wnt binding. C) Mutant DNA sequences induced by FZD1,2,7 sgRNA-I. Representative examples of FZD sequences for FZD1,2,7sgRNA-I#1 cells. PAM sequences are shown in red and inserted nucleotides in blue.To target several genes simultaneously, we identified highly homologous sequence regions in all FZD genes with the exception of the divergent FZD3 and FZD6. We identified suitable sequences in the CRDs of FZD1, FZD2, and FZD7, the most abundant receptors in HEK293T cells. Bioinformatic analysis was performed to select 2 sgRNA sequences to target those regions (33) (Fig. 1B and Supplemental Fig. 1A). We then generated FZD mutant cells by transient transfection of HEK293T cells with Cas9 and sgRNA-encoding plasmids. Single-cell clones were derived and the relevant genomic regions analyzed by next-generation sequencing. Sequencing reads for each sgRNA-targeted FZD gene were aligned to identify mutations (Fig. 1C).Both sgRNAs induced indel mutations around the target sites (Fig. 1C and Table 7). Next-generation sequencing revealed 3 mutant alleles for FZD1 and FZD2 and 2 alleles for the FZD7 gene for most clones of sgFZD1,2,7sgRNA-I. These findings are in line with results from previously performed cytogenetic analyses revealing a pseudo-triploid genotype of the HEK293T cell line and its derivatives (35, 43). From the clones analyzed, only a single cell clone contained an FZD1 WT allele after CRISPR/Cas9-mediated genome engineering, but several clones harbored in-frame deletions (Table 7). Sequencing cell clones derived from FZD1,2,7sgRNA-III revealed single-nucleotide deletions or insertions for all 3 FZDs (Table 7). Together, these experiments showed that paralogous genes with high sequence similarity can be efficiently targeted using a single sgRNA.TABLE 7.Multiple targeting of FZD homologous genes with single sgRNACloneNo.FZD1FZD2FZD7FZD1,2,7sgRNA-I1Deletion 19 ntDeletion 8 ntDeletion 2 ntDeletion 2 ntInsertion 1 ntInsertion 1 ntInsertion 1 ntInsertion 1 nt2Insertion 1 ntInsertion 1 ntDeletion 6 ntDeletion 10 ntDeletion 6 ntInsertion 1 ntInsertion 1 ntDeletion 10 nt3WTDeletion 1 ntDeletion 27 ntInsertion 1 ntInsertion 1 ntDeletion 27 ntInsertion 68 ntDeletion 9 nt + mutation5Deletion 36 ntDeletion 6 ntDeletion 2 ntDeletion 2 ntDeletion 5 ntDeletion 3 ntFZD1,2,7sgRNA-III1Deletion 19 ntDeletion 1 ntInsertion 1 ntDeletion 7 ntWT2Insertion 1 ntInsertion 1 ntInsertion 1 ntDeletion 1 nt3Deletion 2 ntInsertion 1 ntInsertion 1 ntInsertion 1 nt4WTInsertion 1 ntInsertion 1 nt5Insertion 1 ntInsertion 1 ntInsertion 1 nt6Insertion 1 ntInsertion 1 ntInsertion 1 ntInsertion 2 ntGenomic sequencing verifies deletions and insertions in FZD genes generated by single sgRNA in FZD1,2,7sgRNA-I and FZD1,2,7sgRNA-III HEK293T clones.Multiplex targeting of 5 FZD genesNext, we attempted to simultaneously disrupt up to 5 FZD genes by CRISPR/Cas9-mediated mutagenesis. By aligning FZD genes, we identified a highly conserved DNA sequence in the beginning of the transmembrane region. As shown in Fig. 2A, we designed a sgRNA for this target region, which is homologous in FZD1, FZD2, FZD4 (1 mismatch), FZD5, FZD7, and FZD8. Similar to the approach described above, we generated single-cell clones after expression of pan-FZD-sgRNA. Sequence analysis showed that the pan-FZD disruption was indeed successful (Fig. 2B and Table 8). We identified several indel mutations, with fewer mutations found in FZD4 (Table 8).Figure 2.Multiple targeting of up to 5 FZD homologous genes with single sgRNA. A) Schematic representation of FZD proteins; location and sequence targeted by FZD1,2,4,5,7,8 sgRNA. B) Mutant DNA sequences induced by FZD1,2,4,5,7,8 sgRNA. Representative examples of FZD sequences are shown for FZD1,2,4,5,7,8sgRNA#1 cells. PAM sequences are shown in red, inserted nucleotides in blue. Mismatch in targeting region of FZD4 compared to all FZDs is indicated in red. C) Wnt3 cannot induce β-catenin signaling in FZD1,2,4,5,7,8sgRNA cells. Indicated cells were transfected with Wnt3/3a or β-catenin plasmids together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. Results of 3–4 independent experiments are shown as means ± sem.TABLE 8.Multiple targeting of up to 5 FZD homologous genes with single sgRNA for FZD1,2,4,5,7,8sgRNANo.FZD1FZD2FZD4FZD5FZD7FZD81Deletion 15/insertion 3Insertion 16/deletion 3WTDeletion 9WTDeletion 8Deletion 12Insertion 13WTDeletion 4/insertion 1WTDeletion 1Insertion 1Deletion 102Deletion 18/insertion 2Insertion 38Deletion 4Deletion 6Deletion 6Deletion 10/insertion1Insertion 1Insertion 1Insertion 1Deletion 1Insertion 33Insertion 1Deletion 5Deletion 26Insertion 1WTDeletion 1Deletion 26WTDeletion 10Insertion 14Insertion 1Deletion 6Deletion 2Deletion 7Insertion 1Deletion 1Deletion 5Deletion 14Deletion 7Deletion 1WTDeletion 2WTWTDeletion 95Deletion 1Insertion 1WTDeletion 8WTDeletion 1Insertion 1Deletion 14WTDeletion 8Deletion 10Deletion 96Deletion 14Insertion 1Deletion 16WTWTInsertion 1Insertion 1Deletion 5WTInsertion 1Insertion 1Deletion 1Deletion 8Deletion 17WTWTWTWTWTWTDeletion 9Deletion 108Insertion 1Insertion 1WTWTWTDeletion 1Deletion 2Deletion 4Insertion 1Insertion 1Insertion 1Deletion 4Deletion 2WT9Insertion 1Deletion 1WTDeletion 7Deletion 3Insertion 1Deletion 9Deletion 24WTWTDeletion 1Deletion 31Deletion 22Genomic sequencing confirms deletions and insertions in FZD genes generated by single sgRNA in FZD1,2,4,5,7-8sgRNA HEK293T clones.Targeting more than one gene at once can introduce DNA breaks that might lead to large gene rearrangements or deletions. To test for this possibility in the generated cells, we performed PCRs in several clones of pan-FZD cell lines according to the scheme presented in Supplemental Fig. 1B. However, we did not detect any cross-amplification between primers for different FZDs.FZD1, FZD2, and FZD7 are required for Wnt3/3a-induced β-catenin signaling in HEK293T cellsNext, we investigated the functional outcomes of the generated clones. In order to functionally test the FZD1,2,4,5,7,8sgRNA cell clones, we ectopically expressed Wnt3 and Wnt3a (Fig. 2C). All clones, except clone 7, showed complete inhibition of TCF4/Wnt-reporter activity, whereas the reporter activity was not impaired by overexpression of β-catenin. In agreement with these results, only 2 mutated alleles of all 6 FZDs were detected in clone 7 (Table 8). These data suggest that one or multiple FZDs are required for canonical Wnt signaling in HEK293T cells.Next, we tested whether mutations of FZD1, FZD2 and FZD7 were sufficient to deplete Wnt3/3a-induced canonical Wnt signaling in HEK293T cells, as these three are the most abundantly expressed FZD receptors (Fig. 1A). As shown in Fig. 3A, B, in clones derived from both FZD1,2,7sgRNA-I and FZD1,2,7sgRNA-III canonical Wnt signaling could not be triggered by Wnt3 and Wnt3a, but was induced by β-catenin expression. Similar effects were observed for Wnt1 (Supplemental Fig. 1C). Furthermore, levels of active (nonphosphorylated) β-catenin were not elevated in FZD1,2,7sgRNA-I#1 cells after overexpression of Wnt3 (Fig. 3C).Figure 3.Wnt-induced canonical signaling is completely down-regulated in FZD1,2,7sgRNA HEK293T cells. A, B) FZD1,2,7sgRNA-I (A) and FZD1,2,7sgRNA-III (B) cells were transfected with Wnt3/3a or β-catenin plasmids together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. Fold induction normalized to control (ctrl) is shown. Results of 3–4 independent experiments are shown as means ± sem. C) Wnt3 does not change active β-catenin levels in FZD1,2,7sgRNA-I#1 HEK293T cells. Indicated cells were transfected with Wnt3 plasmid for 48 h. Representative experiment out of 3 independent experiments is shown.To confirm that the Wnt loss-of-function phenotype is dependent on mutant FZD1,2,7 alleles and to exclude off-target effects, we rescued the phenotype by ectopic expression of FZD receptors. As shown in Fig. 4A, B, expression of FZD2 or FZD7 each rescued Wnt3- and Wnt1-dependent induction of TCF4/Wnt reporter activity (Fig. 4A, B and Supplemental Fig. 2A). Similar results were obtained for the rescue of canonical β-catenin signaling by recombinant Wnt3a and in single-cell clones derived from additional sgRNAs (Fig. 4C and Supplemental Fig. 2B). We also showed that signaling at the level of active β-catenin was rescued by FZD2 after expression of Wnt3 in FZD1,2,7 mutant cells (Fig. 4D). Together, these data demonstrate that mutating FZD1, FZD2, and FZD7 is sufficient to generate a Wnt-signaling silent background in HEK293T cells.Figure 4.FZD2 and FZD7 can rescue Wnt-induced canonical signaling in FZD1,2,7sgRNA HEK293T cells. A, B) FZD1,2,7sgRNA-I HEK293T cells were transfected with Wnt3 (A) or Wnt1 (B) and FZD2 or FZD7 plasmids together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. C) FZD2 rescues canonical signaling in FZD1,2,7sgRNA-I HEK293T cells upon addition of recombinant Wnt3a. FZD1,2,7sgRNA-1 cells were transfected with FZD2 plasmid together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. Sixteen hours before luciferase read-out, cells were treated with indicated amounts of recombinant Wnt3a. A–C) Results of 3–4 independent experiments are shown as means ± sem. D) FZD2 rescues induction of active β-catenin levels upon Wnt3 expression. FZD1,2,7sgRNA-1 cells were transfected with Wnt3 together with FZD2 plasmids or control plasmid for 48 h. One of 3 independent experiments is shown.Deriving single-cell clones under some experimental conditions can be difficult and may lead to compensatory changes, such as when the loss of members of gene families results in growth defects. In these cases, a short-term, transient approach analyzing mutant pools instead of single clones may be desirable. To this end, we tested the feasibility of a pooled approach to detect the functional outcome of multiplex sgRNA experiments. HEK293T cells were transfected with sgFZD1,2,7-I, sgFZD1,2,7-II, or sgFZD1,2,4,5,7,8 in the presence of Cas9 and short-term selected with puromycin. Pooled knockouts were sufficient to reduce Wnt3/3a-induced canonical signaling by 90% (Supplemental Fig. 2C, D). These results demonstrate that it is possible to use multigene targeting sgRNAs without further single-cell selection and in short-term experiments.Because HCT116 colon cancer cells depend on Wnt secretion for growth, we next tested the requirement for FZD receptor in this context (44). We found that coexpression of sgFZD1,2,7-I or sgFZD1,2,7-II led to growth inhibition. Transfection of these HCT116 cells with a TCF4/Wnt reporter showed a reduced TCF4/Wnt reporter activity (Supplemental Fig. 2E). Thus, our results demonstrate that multigene sgRNAs can be also used in cell lines without further isolation of single-cell clones.Differential selectivity of Wnt ligands and FZD receptorsTo systematically map Wnt-FZD interactions, we next expressed 9 human FZDs (all FZDs except FZD3) and 10 human Wnt ligands in all possible combinations in WT HEK293T cells and in FZD mutant cells and evaluated which combinations led to the rescue of canonical TCF4/Wnt reporter activity (Fig. 5 and Supplemental Fig. 3A). In addition, we assessed the requirement of FZDs for canonical signaling in response to recombinant mouse Wnt3a (Supplemental Fig. 4A). In total, we measured 90 Wnt-FZD combinations. In the case of Wnt3, we confirmed the results of the TCF4/Wnt reporter assay by measuring active β-catenin levels after expression of different FZDs (Fig. 5B).Figure 5.Mapping of canonical Wnt-FZD interactions. A, C) FZD1,2,7sgRNA-I#1 cells were transfected with indicated Wnt and FZD plasmids together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. Results of 3–4 independent experiments are shown as means ± sem. B) Several FZDs rescue induction of active β-catenin levels upon Wnt3 expression in FZD1,2,7sgRNA-I#1 cells. Cells were transfected with Wnt3 and indicated FZDs plasmids for 48 h. Representative experiment out of 3 independent experiments is shown.These experiments revealed that in addition to FZD1, FZD2, and FZD7, all other human FZD receptors with the exception of FZD6 and FZD9 rescued canonical signaling in response to several Wnt ligands in FZD-knockout cells (Fig. 5 and Supplemental Fig. 3). FZD6, FZD9, and Wnt16 protein expression were tested by Western blot analysis to exclude the possibility of a nonfunctional plasmid (Supplemental Fig. 4B).To group Wnt and FZD proteins, we clustered WNT and FZD genes on the basis of their similarity in the signaling activity across all rescue experiments (Fig. 6A). Hierarchical clustering showed high similarity of FZD1, FZD2, and FZD7, whereas FZD5 and FZD8 as well as FZD4 and FZD10 acted differently (Fig. 6A). Interestingly, Wnt3/Wnt3a induced signaling via all FZDs, in contrast to other Wnts that were more selective. For example, Wnt9a signaled only via FZD4 and FZD10, while Wnt1, Wnt8a, Wnt7b, Wnt8b, and Wnt10a induced signaling via FZD5 and FZD8. We found that Wnt7a seems to specifically bind to FZD5, as assessed by TCF4/Wnt-reporter activity. In contrast, Wnt9b induced canonical signaling with FZD5 and FZD8, but also with FZD4 and FZD10 (Fig. 6B).Figure 6.Wnt-FZD interaction map shows diversity of induction of canonical β-catenin signaling between different pairs. A) Canonical Wnt-FZD interactions from Fig. 5 and Supplemental Fig. 3 represented as signaling heat map, showing clustering according to similarity of FZDs and Wnts in induction of canonical β-catenin signaling. Maximum response of Wnt-FZD pair was normalized to 1. B) Schematic representation of Wnts with FZDs and their link to canonical β-catenin signaling.In summary, our results suggest that not all canonical Wnts signal via every FZD receptor. Almost all FZD receptors rescued the impaired canonical signaling in the FZD-knockout cells. Despite functional overlaps among members of the receptor and ligand families, there were still several striking differences among Wnts as well as among FZDs.

Generation of FZD-mutant cells by a single sgRNAMost human tissues and cell lines express multiple FZD receptors (38) that may act in a redundant manner. To create a genetic background for selective rescue experiments, we generated FZD loss-of-function HEK293T cell lines. HEK293T cells have been extensively used for the analysis of canonical Wnt signaling activity (36). HEK293T cells express high levels of FZD2 and FZD7, and to a lesser extent FZD1, FZD4, and FZD9 (Fig. 1A). Moreover, we detected mRNA expression of the sequence-divergent FZD3 and FZD6 receptors, which have been reported to be involved in the noncanonical, β-catenin-independent signaling branch (39–42).Figure 1.Multiple targeting of FZD homologous genes with single sgRNA is functional. A) RNA expression of FZDs in HEK293T cells. qPCR analysis of FZD receptors expression normalized to UBC levels in HEK293T cells. FZD levels were compared to FZD1 expression. Results are means ± sem of 3 independent experiments. B) Schematic representation of FZD proteins and sequences targeted by sgRNAs. SS indicates N-terminal signal sequence; CRD, extracellular cysteine-rich domain required for Wnt binding. C) Mutant DNA sequences induced by FZD1,2,7 sgRNA-I. Representative examples of FZD sequences for FZD1,2,7sgRNA-I#1 cells. PAM sequences are shown in red and inserted nucleotides in blue.To target several genes simultaneously, we identified highly homologous sequence regions in all FZD genes with the exception of the divergent FZD3 and FZD6. We identified suitable sequences in the CRDs of FZD1, FZD2, and FZD7, the most abundant receptors in HEK293T cells. Bioinformatic analysis was performed to select 2 sgRNA sequences to target those regions (33) (Fig. 1B and Supplemental Fig. 1A). We then generated FZD mutant cells by transient transfection of HEK293T cells with Cas9 and sgRNA-encoding plasmids. Single-cell clones were derived and the relevant genomic regions analyzed by next-generation sequencing. Sequencing reads for each sgRNA-targeted FZD gene were aligned to identify mutations (Fig. 1C).Both sgRNAs induced indel mutations around the target sites (Fig. 1C and Table 7). Next-generation sequencing revealed 3 mutant alleles for FZD1 and FZD2 and 2 alleles for the FZD7 gene for most clones of sgFZD1,2,7sgRNA-I. These findings are in line with results from previously performed cytogenetic analyses revealing a pseudo-triploid genotype of the HEK293T cell line and its derivatives (35, 43). From the clones analyzed, only a single cell clone contained an FZD1 WT allele after CRISPR/Cas9-mediated genome engineering, but several clones harbored in-frame deletions (Table 7). Sequencing cell clones derived from FZD1,2,7sgRNA-III revealed single-nucleotide deletions or insertions for all 3 FZDs (Table 7). Together, these experiments showed that paralogous genes with high sequence similarity can be efficiently targeted using a single sgRNA.TABLE 7.Multiple targeting of FZD homologous genes with single sgRNACloneNo.FZD1FZD2FZD7FZD1,2,7sgRNA-I1Deletion 19 ntDeletion 8 ntDeletion 2 ntDeletion 2 ntInsertion 1 ntInsertion 1 ntInsertion 1 ntInsertion 1 nt2Insertion 1 ntInsertion 1 ntDeletion 6 ntDeletion 10 ntDeletion 6 ntInsertion 1 ntInsertion 1 ntDeletion 10 nt3WTDeletion 1 ntDeletion 27 ntInsertion 1 ntInsertion 1 ntDeletion 27 ntInsertion 68 ntDeletion 9 nt + mutation5Deletion 36 ntDeletion 6 ntDeletion 2 ntDeletion 2 ntDeletion 5 ntDeletion 3 ntFZD1,2,7sgRNA-III1Deletion 19 ntDeletion 1 ntInsertion 1 ntDeletion 7 ntWT2Insertion 1 ntInsertion 1 ntInsertion 1 ntDeletion 1 nt3Deletion 2 ntInsertion 1 ntInsertion 1 ntInsertion 1 nt4WTInsertion 1 ntInsertion 1 nt5Insertion 1 ntInsertion 1 ntInsertion 1 nt6Insertion 1 ntInsertion 1 ntInsertion 1 ntInsertion 2 ntGenomic sequencing verifies deletions and insertions in FZD genes generated by single sgRNA in FZD1,2,7sgRNA-I and FZD1,2,7sgRNA-III HEK293T clones.

Generation of FZD-mutant cells by a single sgRNA

Most human tissues and cell lines express multiple FZD receptors (38) that may act in a redundant manner. To create a genetic background for selective rescue experiments, we generated FZD loss-of-function HEK293T cell lines. HEK293T cells have been extensively used for the analysis of canonical Wnt signaling activity (36). HEK293T cells express high levels of FZD2 and FZD7, and to a lesser extent FZD1, FZD4, and FZD9 (Fig. 1A). Moreover, we detected mRNA expression of the sequence-divergent FZD3 and FZD6 receptors, which have been reported to be involved in the noncanonical, β-catenin-independent signaling branch (39–42).

Multiple targeting of FZD homologous genes with single sgRNA is functional. A) RNA expression of FZDs in HEK293T cells. qPCR analysis of FZD receptors expression normalized to UBC levels in HEK293T cells. FZD levels were compared to FZD1 expression. Results are means ± sem of 3 independent experiments. B) Schematic representation of FZD proteins and sequences targeted by sgRNAs. SS indicates N-terminal signal sequence; CRD, extracellular cysteine-rich domain required for Wnt binding. C) Mutant DNA sequences induced by FZD1,2,7 sgRNA-I. Representative examples of FZD sequences for FZD1,2,7sgRNA-I#1 cells. PAM sequences are shown in red and inserted nucleotides in blue.

To target several genes simultaneously, we identified highly homologous sequence regions in all FZD genes with the exception of the divergent FZD3 and FZD6. We identified suitable sequences in the CRDs of FZD1, FZD2, and FZD7, the most abundant receptors in HEK293T cells. Bioinformatic analysis was performed to select 2 sgRNA sequences to target those regions (33) (Fig. 1B and Supplemental Fig. 1A). We then generated FZD mutant cells by transient transfection of HEK293T cells with Cas9 and sgRNA-encoding plasmids. Single-cell clones were derived and the relevant genomic regions analyzed by next-generation sequencing. Sequencing reads for each sgRNA-targeted FZD gene were aligned to identify mutations (Fig. 1C).

Both sgRNAs induced indel mutations around the target sites (Fig. 1C and Table 7). Next-generation sequencing revealed 3 mutant alleles for FZD1 and FZD2 and 2 alleles for the FZD7 gene for most clones of sgFZD1,2,7sgRNA-I. These findings are in line with results from previously performed cytogenetic analyses revealing a pseudo-triploid genotype of the HEK293T cell line and its derivatives (35, 43). From the clones analyzed, only a single cell clone contained an FZD1 WT allele after CRISPR/Cas9-mediated genome engineering, but several clones harbored in-frame deletions (Table 7). Sequencing cell clones derived from FZD1,2,7sgRNA-III revealed single-nucleotide deletions or insertions for all 3 FZDs (Table 7). Together, these experiments showed that paralogous genes with high sequence similarity can be efficiently targeted using a single sgRNA.

Multiple targeting of FZD homologous genes with single sgRNA

Genomic sequencing verifies deletions and insertions in FZD genes generated by single sgRNA in FZD1,2,7sgRNA-I and FZD1,2,7sgRNA-III HEK293T clones.

Multiplex targeting of 5 FZD genesNext, we attempted to simultaneously disrupt up to 5 FZD genes by CRISPR/Cas9-mediated mutagenesis. By aligning FZD genes, we identified a highly conserved DNA sequence in the beginning of the transmembrane region. As shown in Fig. 2A, we designed a sgRNA for this target region, which is homologous in FZD1, FZD2, FZD4 (1 mismatch), FZD5, FZD7, and FZD8. Similar to the approach described above, we generated single-cell clones after expression of pan-FZD-sgRNA. Sequence analysis showed that the pan-FZD disruption was indeed successful (Fig. 2B and Table 8). We identified several indel mutations, with fewer mutations found in FZD4 (Table 8).Figure 2.Multiple targeting of up to 5 FZD homologous genes with single sgRNA. A) Schematic representation of FZD proteins; location and sequence targeted by FZD1,2,4,5,7,8 sgRNA. B) Mutant DNA sequences induced by FZD1,2,4,5,7,8 sgRNA. Representative examples of FZD sequences are shown for FZD1,2,4,5,7,8sgRNA#1 cells. PAM sequences are shown in red, inserted nucleotides in blue. Mismatch in targeting region of FZD4 compared to all FZDs is indicated in red. C) Wnt3 cannot induce β-catenin signaling in FZD1,2,4,5,7,8sgRNA cells. Indicated cells were transfected with Wnt3/3a or β-catenin plasmids together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. Results of 3–4 independent experiments are shown as means ± sem.TABLE 8.Multiple targeting of up to 5 FZD homologous genes with single sgRNA for FZD1,2,4,5,7,8sgRNANo.FZD1FZD2FZD4FZD5FZD7FZD81Deletion 15/insertion 3Insertion 16/deletion 3WTDeletion 9WTDeletion 8Deletion 12Insertion 13WTDeletion 4/insertion 1WTDeletion 1Insertion 1Deletion 102Deletion 18/insertion 2Insertion 38Deletion 4Deletion 6Deletion 6Deletion 10/insertion1Insertion 1Insertion 1Insertion 1Deletion 1Insertion 33Insertion 1Deletion 5Deletion 26Insertion 1WTDeletion 1Deletion 26WTDeletion 10Insertion 14Insertion 1Deletion 6Deletion 2Deletion 7Insertion 1Deletion 1Deletion 5Deletion 14Deletion 7Deletion 1WTDeletion 2WTWTDeletion 95Deletion 1Insertion 1WTDeletion 8WTDeletion 1Insertion 1Deletion 14WTDeletion 8Deletion 10Deletion 96Deletion 14Insertion 1Deletion 16WTWTInsertion 1Insertion 1Deletion 5WTInsertion 1Insertion 1Deletion 1Deletion 8Deletion 17WTWTWTWTWTWTDeletion 9Deletion 108Insertion 1Insertion 1WTWTWTDeletion 1Deletion 2Deletion 4Insertion 1Insertion 1Insertion 1Deletion 4Deletion 2WT9Insertion 1Deletion 1WTDeletion 7Deletion 3Insertion 1Deletion 9Deletion 24WTWTDeletion 1Deletion 31Deletion 22Genomic sequencing confirms deletions and insertions in FZD genes generated by single sgRNA in FZD1,2,4,5,7-8sgRNA HEK293T clones.Targeting more than one gene at once can introduce DNA breaks that might lead to large gene rearrangements or deletions. To test for this possibility in the generated cells, we performed PCRs in several clones of pan-FZD cell lines according to the scheme presented in Supplemental Fig. 1B. However, we did not detect any cross-amplification between primers for different FZDs.

Multiplex targeting of 5 FZD genes

Next, we attempted to simultaneously disrupt up to 5 FZD genes by CRISPR/Cas9-mediated mutagenesis. By aligning FZD genes, we identified a highly conserved DNA sequence in the beginning of the transmembrane region. As shown in Fig. 2A, we designed a sgRNA for this target region, which is homologous in FZD1, FZD2, FZD4 (1 mismatch), FZD5, FZD7, and FZD8. Similar to the approach described above, we generated single-cell clones after expression of pan-FZD-sgRNA. Sequence analysis showed that the pan-FZD disruption was indeed successful (Fig. 2B and Table 8). We identified several indel mutations, with fewer mutations found in FZD4 (Table 8).

Multiple targeting of up to 5 FZD homologous genes with single sgRNA. A) Schematic representation of FZD proteins; location and sequence targeted by FZD1,2,4,5,7,8 sgRNA. B) Mutant DNA sequences induced by FZD1,2,4,5,7,8 sgRNA. Representative examples of FZD sequences are shown for FZD1,2,4,5,7,8sgRNA#1 cells. PAM sequences are shown in red, inserted nucleotides in blue. Mismatch in targeting region of FZD4 compared to all FZDs is indicated in red. C) Wnt3 cannot induce β-catenin signaling in FZD1,2,4,5,7,8sgRNA cells. Indicated cells were transfected with Wnt3/3a or β-catenin plasmids together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. Results of 3–4 independent experiments are shown as means ± sem.

Multiple targeting of up to 5 FZD homologous genes with single sgRNA for FZD1,2,4,5,7,8sgRNA

Genomic sequencing confirms deletions and insertions in FZD genes generated by single sgRNA in FZD1,2,4,5,7-8sgRNA HEK293T clones.

Targeting more than one gene at once can introduce DNA breaks that might lead to large gene rearrangements or deletions. To test for this possibility in the generated cells, we performed PCRs in several clones of pan-FZD cell lines according to the scheme presented in Supplemental Fig. 1B. However, we did not detect any cross-amplification between primers for different FZDs.

FZD1, FZD2, and FZD7 are required for Wnt3/3a-induced β-catenin signaling in HEK293T cellsNext, we investigated the functional outcomes of the generated clones. In order to functionally test the FZD1,2,4,5,7,8sgRNA cell clones, we ectopically expressed Wnt3 and Wnt3a (Fig. 2C). All clones, except clone 7, showed complete inhibition of TCF4/Wnt-reporter activity, whereas the reporter activity was not impaired by overexpression of β-catenin. In agreement with these results, only 2 mutated alleles of all 6 FZDs were detected in clone 7 (Table 8). These data suggest that one or multiple FZDs are required for canonical Wnt signaling in HEK293T cells.Next, we tested whether mutations of FZD1, FZD2 and FZD7 were sufficient to deplete Wnt3/3a-induced canonical Wnt signaling in HEK293T cells, as these three are the most abundantly expressed FZD receptors (Fig. 1A). As shown in Fig. 3A, B, in clones derived from both FZD1,2,7sgRNA-I and FZD1,2,7sgRNA-III canonical Wnt signaling could not be triggered by Wnt3 and Wnt3a, but was induced by β-catenin expression. Similar effects were observed for Wnt1 (Supplemental Fig. 1C). Furthermore, levels of active (nonphosphorylated) β-catenin were not elevated in FZD1,2,7sgRNA-I#1 cells after overexpression of Wnt3 (Fig. 3C).Figure 3.Wnt-induced canonical signaling is completely down-regulated in FZD1,2,7sgRNA HEK293T cells. A, B) FZD1,2,7sgRNA-I (A) and FZD1,2,7sgRNA-III (B) cells were transfected with Wnt3/3a or β-catenin plasmids together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. Fold induction normalized to control (ctrl) is shown. Results of 3–4 independent experiments are shown as means ± sem. C) Wnt3 does not change active β-catenin levels in FZD1,2,7sgRNA-I#1 HEK293T cells. Indicated cells were transfected with Wnt3 plasmid for 48 h. Representative experiment out of 3 independent experiments is shown.To confirm that the Wnt loss-of-function phenotype is dependent on mutant FZD1,2,7 alleles and to exclude off-target effects, we rescued the phenotype by ectopic expression of FZD receptors. As shown in Fig. 4A, B, expression of FZD2 or FZD7 each rescued Wnt3- and Wnt1-dependent induction of TCF4/Wnt reporter activity (Fig. 4A, B and Supplemental Fig. 2A). Similar results were obtained for the rescue of canonical β-catenin signaling by recombinant Wnt3a and in single-cell clones derived from additional sgRNAs (Fig. 4C and Supplemental Fig. 2B). We also showed that signaling at the level of active β-catenin was rescued by FZD2 after expression of Wnt3 in FZD1,2,7 mutant cells (Fig. 4D). Together, these data demonstrate that mutating FZD1, FZD2, and FZD7 is sufficient to generate a Wnt-signaling silent background in HEK293T cells.Figure 4.FZD2 and FZD7 can rescue Wnt-induced canonical signaling in FZD1,2,7sgRNA HEK293T cells. A, B) FZD1,2,7sgRNA-I HEK293T cells were transfected with Wnt3 (A) or Wnt1 (B) and FZD2 or FZD7 plasmids together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. C) FZD2 rescues canonical signaling in FZD1,2,7sgRNA-I HEK293T cells upon addition of recombinant Wnt3a. FZD1,2,7sgRNA-1 cells were transfected with FZD2 plasmid together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. Sixteen hours before luciferase read-out, cells were treated with indicated amounts of recombinant Wnt3a. A–C) Results of 3–4 independent experiments are shown as means ± sem. D) FZD2 rescues induction of active β-catenin levels upon Wnt3 expression. FZD1,2,7sgRNA-1 cells were transfected with Wnt3 together with FZD2 plasmids or control plasmid for 48 h. One of 3 independent experiments is shown.Deriving single-cell clones under some experimental conditions can be difficult and may lead to compensatory changes, such as when the loss of members of gene families results in growth defects. In these cases, a short-term, transient approach analyzing mutant pools instead of single clones may be desirable. To this end, we tested the feasibility of a pooled approach to detect the functional outcome of multiplex sgRNA experiments. HEK293T cells were transfected with sgFZD1,2,7-I, sgFZD1,2,7-II, or sgFZD1,2,4,5,7,8 in the presence of Cas9 and short-term selected with puromycin. Pooled knockouts were sufficient to reduce Wnt3/3a-induced canonical signaling by 90% (Supplemental Fig. 2C, D). These results demonstrate that it is possible to use multigene targeting sgRNAs without further single-cell selection and in short-term experiments.Because HCT116 colon cancer cells depend on Wnt secretion for growth, we next tested the requirement for FZD receptor in this context (44). We found that coexpression of sgFZD1,2,7-I or sgFZD1,2,7-II led to growth inhibition. Transfection of these HCT116 cells with a TCF4/Wnt reporter showed a reduced TCF4/Wnt reporter activity (Supplemental Fig. 2E). Thus, our results demonstrate that multigene sgRNAs can be also used in cell lines without further isolation of single-cell clones.

FZD1, FZD2, and FZD7 are required for Wnt3/3a-induced β-catenin signaling in HEK293T cells

Next, we investigated the functional outcomes of the generated clones. In order to functionally test the FZD1,2,4,5,7,8sgRNA cell clones, we ectopically expressed Wnt3 and Wnt3a (Fig. 2C). All clones, except clone 7, showed complete inhibition of TCF4/Wnt-reporter activity, whereas the reporter activity was not impaired by overexpression of β-catenin. In agreement with these results, only 2 mutated alleles of all 6 FZDs were detected in clone 7 (Table 8). These data suggest that one or multiple FZDs are required for canonical Wnt signaling in HEK293T cells.

Next, we tested whether mutations of FZD1, FZD2 and FZD7 were sufficient to deplete Wnt3/3a-induced canonical Wnt signaling in HEK293T cells, as these three are the most abundantly expressed FZD receptors (Fig. 1A). As shown in Fig. 3A, B, in clones derived from both FZD1,2,7sgRNA-I and FZD1,2,7sgRNA-III canonical Wnt signaling could not be triggered by Wnt3 and Wnt3a, but was induced by β-catenin expression. Similar effects were observed for Wnt1 (Supplemental Fig. 1C). Furthermore, levels of active (nonphosphorylated) β-catenin were not elevated in FZD1,2,7sgRNA-I#1 cells after overexpression of Wnt3 (Fig. 3C).

Wnt-induced canonical signaling is completely down-regulated in FZD1,2,7sgRNA HEK293T cells. A, B) FZD1,2,7sgRNA-I (A) and FZD1,2,7sgRNA-III (B) cells were transfected with Wnt3/3a or β-catenin plasmids together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. Fold induction normalized to control (ctrl) is shown. Results of 3–4 independent experiments are shown as means ± sem. C) Wnt3 does not change active β-catenin levels in FZD1,2,7sgRNA-I#1 HEK293T cells. Indicated cells were transfected with Wnt3 plasmid for 48 h. Representative experiment out of 3 independent experiments is shown.

To confirm that the Wnt loss-of-function phenotype is dependent on mutant FZD1,2,7 alleles and to exclude off-target effects, we rescued the phenotype by ectopic expression of FZD receptors. As shown in Fig. 4A, B, expression of FZD2 or FZD7 each rescued Wnt3- and Wnt1-dependent induction of TCF4/Wnt reporter activity (Fig. 4A, B and Supplemental Fig. 2A). Similar results were obtained for the rescue of canonical β-catenin signaling by recombinant Wnt3a and in single-cell clones derived from additional sgRNAs (Fig. 4C and Supplemental Fig. 2B). We also showed that signaling at the level of active β-catenin was rescued by FZD2 after expression of Wnt3 in FZD1,2,7 mutant cells (Fig. 4D). Together, these data demonstrate that mutating FZD1, FZD2, and FZD7 is sufficient to generate a Wnt-signaling silent background in HEK293T cells.

FZD2 and FZD7 can rescue Wnt-induced canonical signaling in FZD1,2,7sgRNA HEK293T cells. A, B) FZD1,2,7sgRNA-I HEK293T cells were transfected with Wnt3 (A) or Wnt1 (B) and FZD2 or FZD7 plasmids together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. C) FZD2 rescues canonical signaling in FZD1,2,7sgRNA-I HEK293T cells upon addition of recombinant Wnt3a. FZD1,2,7sgRNA-1 cells were transfected with FZD2 plasmid together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. Sixteen hours before luciferase read-out, cells were treated with indicated amounts of recombinant Wnt3a. A–C) Results of 3–4 independent experiments are shown as means ± sem. D) FZD2 rescues induction of active β-catenin levels upon Wnt3 expression. FZD1,2,7sgRNA-1 cells were transfected with Wnt3 together with FZD2 plasmids or control plasmid for 48 h. One of 3 independent experiments is shown.

Deriving single-cell clones under some experimental conditions can be difficult and may lead to compensatory changes, such as when the loss of members of gene families results in growth defects. In these cases, a short-term, transient approach analyzing mutant pools instead of single clones may be desirable. To this end, we tested the feasibility of a pooled approach to detect the functional outcome of multiplex sgRNA experiments. HEK293T cells were transfected with sgFZD1,2,7-I, sgFZD1,2,7-II, or sgFZD1,2,4,5,7,8 in the presence of Cas9 and short-term selected with puromycin. Pooled knockouts were sufficient to reduce Wnt3/3a-induced canonical signaling by 90% (Supplemental Fig. 2C, D). These results demonstrate that it is possible to use multigene targeting sgRNAs without further single-cell selection and in short-term experiments.

Because HCT116 colon cancer cells depend on Wnt secretion for growth, we next tested the requirement for FZD receptor in this context (44). We found that coexpression of sgFZD1,2,7-I or sgFZD1,2,7-II led to growth inhibition. Transfection of these HCT116 cells with a TCF4/Wnt reporter showed a reduced TCF4/Wnt reporter activity (Supplemental Fig. 2E). Thus, our results demonstrate that multigene sgRNAs can be also used in cell lines without further isolation of single-cell clones.

Differential selectivity of Wnt ligands and FZD receptorsTo systematically map Wnt-FZD interactions, we next expressed 9 human FZDs (all FZDs except FZD3) and 10 human Wnt ligands in all possible combinations in WT HEK293T cells and in FZD mutant cells and evaluated which combinations led to the rescue of canonical TCF4/Wnt reporter activity (Fig. 5 and Supplemental Fig. 3A). In addition, we assessed the requirement of FZDs for canonical signaling in response to recombinant mouse Wnt3a (Supplemental Fig. 4A). In total, we measured 90 Wnt-FZD combinations. In the case of Wnt3, we confirmed the results of the TCF4/Wnt reporter assay by measuring active β-catenin levels after expression of different FZDs (Fig. 5B).Figure 5.Mapping of canonical Wnt-FZD interactions. A, C) FZD1,2,7sgRNA-I#1 cells were transfected with indicated Wnt and FZD plasmids together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. Results of 3–4 independent experiments are shown as means ± sem. B) Several FZDs rescue induction of active β-catenin levels upon Wnt3 expression in FZD1,2,7sgRNA-I#1 cells. Cells were transfected with Wnt3 and indicated FZDs plasmids for 48 h. Representative experiment out of 3 independent experiments is shown.These experiments revealed that in addition to FZD1, FZD2, and FZD7, all other human FZD receptors with the exception of FZD6 and FZD9 rescued canonical signaling in response to several Wnt ligands in FZD-knockout cells (Fig. 5 and Supplemental Fig. 3). FZD6, FZD9, and Wnt16 protein expression were tested by Western blot analysis to exclude the possibility of a nonfunctional plasmid (Supplemental Fig. 4B).To group Wnt and FZD proteins, we clustered WNT and FZD genes on the basis of their similarity in the signaling activity across all rescue experiments (Fig. 6A). Hierarchical clustering showed high similarity of FZD1, FZD2, and FZD7, whereas FZD5 and FZD8 as well as FZD4 and FZD10 acted differently (Fig. 6A). Interestingly, Wnt3/Wnt3a induced signaling via all FZDs, in contrast to other Wnts that were more selective. For example, Wnt9a signaled only via FZD4 and FZD10, while Wnt1, Wnt8a, Wnt7b, Wnt8b, and Wnt10a induced signaling via FZD5 and FZD8. We found that Wnt7a seems to specifically bind to FZD5, as assessed by TCF4/Wnt-reporter activity. In contrast, Wnt9b induced canonical signaling with FZD5 and FZD8, but also with FZD4 and FZD10 (Fig. 6B).Figure 6.Wnt-FZD interaction map shows diversity of induction of canonical β-catenin signaling between different pairs. A) Canonical Wnt-FZD interactions from Fig. 5 and Supplemental Fig. 3 represented as signaling heat map, showing clustering according to similarity of FZDs and Wnts in induction of canonical β-catenin signaling. Maximum response of Wnt-FZD pair was normalized to 1. B) Schematic representation of Wnts with FZDs and their link to canonical β-catenin signaling.In summary, our results suggest that not all canonical Wnts signal via every FZD receptor. Almost all FZD receptors rescued the impaired canonical signaling in the FZD-knockout cells. Despite functional overlaps among members of the receptor and ligand families, there were still several striking differences among Wnts as well as among FZDs.

Differential selectivity of Wnt ligands and FZD receptors

To systematically map Wnt-FZD interactions, we next expressed 9 human FZDs (all FZDs except FZD3) and 10 human Wnt ligands in all possible combinations in WT HEK293T cells and in FZD mutant cells and evaluated which combinations led to the rescue of canonical TCF4/Wnt reporter activity (Fig. 5 and Supplemental Fig. 3A). In addition, we assessed the requirement of FZDs for canonical signaling in response to recombinant mouse Wnt3a (Supplemental Fig. 4A). In total, we measured 90 Wnt-FZD combinations. In the case of Wnt3, we confirmed the results of the TCF4/Wnt reporter assay by measuring active β-catenin levels after expression of different FZDs (Fig. 5B).

Mapping of canonical Wnt-FZD interactions. A, C) FZD1,2,7sgRNA-I#1 cells were transfected with indicated Wnt and FZD plasmids together with TCF4/Wnt–firefly luciferase and actin–Renilla reporters. Results of 3–4 independent experiments are shown as means ± sem. B) Several FZDs rescue induction of active β-catenin levels upon Wnt3 expression in FZD1,2,7sgRNA-I#1 cells. Cells were transfected with Wnt3 and indicated FZDs plasmids for 48 h. Representative experiment out of 3 independent experiments is shown.

These experiments revealed that in addition to FZD1, FZD2, and FZD7, all other human FZD receptors with the exception of FZD6 and FZD9 rescued canonical signaling in response to several Wnt ligands in FZD-knockout cells (Fig. 5 and Supplemental Fig. 3). FZD6, FZD9, and Wnt16 protein expression were tested by Western blot analysis to exclude the possibility of a nonfunctional plasmid (Supplemental Fig. 4B).

To group Wnt and FZD proteins, we clustered WNT and FZD genes on the basis of their similarity in the signaling activity across all rescue experiments (Fig. 6A). Hierarchical clustering showed high similarity of FZD1, FZD2, and FZD7, whereas FZD5 and FZD8 as well as FZD4 and FZD10 acted differently (Fig. 6A). Interestingly, Wnt3/Wnt3a induced signaling via all FZDs, in contrast to other Wnts that were more selective. For example, Wnt9a signaled only via FZD4 and FZD10, while Wnt1, Wnt8a, Wnt7b, Wnt8b, and Wnt10a induced signaling via FZD5 and FZD8. We found that Wnt7a seems to specifically bind to FZD5, as assessed by TCF4/Wnt-reporter activity. In contrast, Wnt9b induced canonical signaling with FZD5 and FZD8, but also with FZD4 and FZD10 (Fig. 6B).

Wnt-FZD interaction map shows diversity of induction of canonical β-catenin signaling between different pairs. A) Canonical Wnt-FZD interactions from Fig. 5 and Supplemental Fig. 3 represented as signaling heat map, showing clustering according to similarity of FZDs and Wnts in induction of canonical β-catenin signaling. Maximum response of Wnt-FZD pair was normalized to 1. B) Schematic representation of Wnts with FZDs and their link to canonical β-catenin signaling.

In summary, our results suggest that not all canonical Wnts signal via every FZD receptor. Almost all FZD receptors rescued the impaired canonical signaling in the FZD-knockout cells. Despite functional overlaps among members of the receptor and ligand families, there were still several striking differences among Wnts as well as among FZDs.

DISCUSSIONGenetic redundancy affects a large percentage of genes within the human genome, thus posing challenges for loss-of-function analysis of many cellular pathways. Homologous genes can have similar functions and consequently may be able to compensate for each other. Many key components of important signaling pathways are represented by duplicated genes (45). One of the best examples is the Wnt/β-catenin signaling pathway, in which almost each level of the pathway involves several paralogous genes (1, 2). There are 10 FZD receptors, which are highly similar, and 19 Wnt ligands in the Wnt signaling pathway. Some Wnt-FZD interactions have been previously identified (20, 46), but many of them are still uncharacterized.To elucidate the functional outcomes of FZD and Wnt expression, a human cell line lacking the relevant FZD proteins is required. Using CRISPR/Cas9-mediated genome editing, it is now possible to disrupt genes in almost any given cell type (42, 47, 48). Our results indicate that at least 5 homologous genes can be successfully targeted by a single sgRNA. Currently homologous genes are either often excluded from library screens or they are not identified phenotypically in a screen as a result of genetic redundancy. Using a multiplex targeting approach will now enable screening experiments targeting larger, potentially redundant gene families. To overcome the problem of targeting several proteins at once, we identified 20- to 24-bp-long sequences of FZDs with few mismatches that could be used as a target sequence for sgRNAs. Notably, this approach poses certain challenges, as it might not be efficient in targeting multiple alleles. However, our results showed that a functional sgRNA was able to target more than 2 alleles. Another issue could arise from rearrangements (49–51) or deletions (52, 53) of chromosomes. While we cannot exclude the possibility that rearrangements might have occurred in some cells, we detected by sequencing FZD1,2,7sgRNA clones all 3 alleles for FZD1 and FZD2, and 2 for FZD7 alleles with different mutations, which corresponded to the number of alleles present in HEK293T cells (43, 54). For FZD1,2,4,5,7,8sgRNA clones 1 and 2, we performed rearrangement by PCR analysis as previously described (55), but we did not detect any signal corresponding to rearranged FZDs. We also performed rescue experiments showing that after reconstitution of FZDs, Wnt signaling was still induced. Concerning chromosomal rearrangements reported previously with CRISPR/Cas9 (49–51), these selected sites are known to be prone to such events—for example, Eml4-Alk, EWSR1-FLI1, and CD74-ROS1. As with other clonal experiments, it is important to test several clones and constructs and to perform rescue experiments to exclude off-target effects.The focus of the Wnt-FZD interaction map in this study has been on canonical β-catenin signaling, which complements a previous study by Dijksterhuis et al. (20) that concentrated on the canonical and noncanonical ligands Wnt3a, Wnt5a, Wnt5b, and Wnt4 encoded by the mouse genome. The identified Wnt3a-FZD interactions are consistent with previous studies (20, 46). In line with our results, a previous study identified FZD5 as the most effective receptor for canonical Wnt signaling (56). Importantly, by providing a broad analysis of 90 Wnts-FZDs pairs in the FZD loss-of-function background, we identified new interactions for Wnt9a and Wnt9b, and Wnt7b and Wnt7a. In contrast to the study by Yu et al. (56), which was performed with mouse Wnts-FZDs and Lrp5 overexpression in WT HEK293T cells, we generated knockout cell lines. As expected, our data showed that Wnt3 and Wnt3a interactions with FZDs were almost identical; this was not the case for Wnt9a and Wnt9b, or for Wnt7b and Wnt7a. For example, Wnt7a induces canonical Wnt signaling only via FZD5, whereas Wnt7b can do so via FZD5 and FZD8. This result is in contrast to the induction of noncanonical Wnt signaling by Wnt7a via FZD10 (57). In agreement with our results, it has been recently demonstrated that pancreatic cancer cells that mainly secrete Wnt7b are dependent on FZD5 (58). The focus of this study was the canonical Wnt signaling pathway. In future studies, it would be interesting to investigate the requirement of noncanonical Wnt ligands with different FZD receptors.Our map (Fig. 6A) indicates that the similarity in Wnt activity pattern among FZDs can be linked to the phylogenetic tree of these receptors (59, 60). Previously performed phylogenetic analyses based on protein sequence homology have revealed that FZDs can be grouped into 4 main clusters. FZD receptors within a cluster share higher identity—FZD1,2,7 (75%), FZD5,8 (70%), and FZD4,9,10 and FZD3,6 (50%)—than FZDs from different clusters (20–40%). FZDs of one cluster might be functionally related, assuming that each cluster derives from a common parental gene (40, 60). Indeed FZD1, FZD2, and FZD7 share up to 75% homology and specifically induce signaling after binding of Wnt3 and Wnt3a. FZD5 and FZD8 share ∼70% homology and are the most promiscuous receptors. These results indicate that, in agreement with a previous study (56), FZDs with high amino acid sequence similarity possess similar Wnt activity patterns.In summary, we show that multitargeting sgRNAs which simultaneously disrupt several FZDs can be applied to study Wnt-FZD interactions and their effects on canonical Wnt signaling. This approach can be extended in the future to analyze noncanonical Wnt signaling, as well as other signaling pathways relying on homologous genes. Our results on the one hand highlight the similarities between certain canonical Wnt ligands and receptor family members and on the other hand reveal unexpected dependencies among certain Wnts and FZDs.

Genetic redundancy affects a large percentage of genes within the human genome, thus posing challenges for loss-of-function analysis of many cellular pathways. Homologous genes can have similar functions and consequently may be able to compensate for each other. Many key components of important signaling pathways are represented by duplicated genes (45). One of the best examples is the Wnt/β-catenin signaling pathway, in which almost each level of the pathway involves several paralogous genes (1, 2). There are 10 FZD receptors, which are highly similar, and 19 Wnt ligands in the Wnt signaling pathway. Some Wnt-FZD interactions have been previously identified (20, 46), but many of them are still uncharacterized.

To elucidate the functional outcomes of FZD and Wnt expression, a human cell line lacking the relevant FZD proteins is required. Using CRISPR/Cas9-mediated genome editing, it is now possible to disrupt genes in almost any given cell type (42, 47, 48). Our results indicate that at least 5 homologous genes can be successfully targeted by a single sgRNA. Currently homologous genes are either often excluded from library screens or they are not identified phenotypically in a screen as a result of genetic redundancy. Using a multiplex targeting approach will now enable screening experiments targeting larger, potentially redundant gene families. To overcome the problem of targeting several proteins at once, we identified 20- to 24-bp-long sequences of FZDs with few mismatches that could be used as a target sequence for sgRNAs. Notably, this approach poses certain challenges, as it might not be efficient in targeting multiple alleles. However, our results showed that a functional sgRNA was able to target more than 2 alleles. Another issue could arise from rearrangements (49–51) or deletions (52, 53) of chromosomes. While we cannot exclude the possibility that rearrangements might have occurred in some cells, we detected by sequencing FZD1,2,7sgRNA clones all 3 alleles for FZD1 and FZD2, and 2 for FZD7 alleles with different mutations, which corresponded to the number of alleles present in HEK293T cells (43, 54). For FZD1,2,4,5,7,8sgRNA clones 1 and 2, we performed rearrangement by PCR analysis as previously described (55), but we did not detect any signal corresponding to rearranged FZDs. We also performed rescue experiments showing that after reconstitution of FZDs, Wnt signaling was still induced. Concerning chromosomal rearrangements reported previously with CRISPR/Cas9 (49–51), these selected sites are known to be prone to such events—for example, Eml4-Alk, EWSR1-FLI1, and CD74-ROS1. As with other clonal experiments, it is important to test several clones and constructs and to perform rescue experiments to exclude off-target effects.

The focus of the Wnt-FZD interaction map in this study has been on canonical β-catenin signaling, which complements a previous study by Dijksterhuis et al. (20) that concentrated on the canonical and noncanonical ligands Wnt3a, Wnt5a, Wnt5b, and Wnt4 encoded by the mouse genome. The identified Wnt3a-FZD interactions are consistent with previous studies (20, 46). In line with our results, a previous study identified FZD5 as the most effective receptor for canonical Wnt signaling (56). Importantly, by providing a broad analysis of 90 Wnts-FZDs pairs in the FZD loss-of-function background, we identified new interactions for Wnt9a and Wnt9b, and Wnt7b and Wnt7a. In contrast to the study by Yu et al. (56), which was performed with mouse Wnts-FZDs and Lrp5 overexpression in WT HEK293T cells, we generated knockout cell lines. As expected, our data showed that Wnt3 and Wnt3a interactions with FZDs were almost identical; this was not the case for Wnt9a and Wnt9b, or for Wnt7b and Wnt7a. For example, Wnt7a induces canonical Wnt signaling only via FZD5, whereas Wnt7b can do so via FZD5 and FZD8. This result is in contrast to the induction of noncanonical Wnt signaling by Wnt7a via FZD10 (57). In agreement with our results, it has been recently demonstrated that pancreatic cancer cells that mainly secrete Wnt7b are dependent on FZD5 (58). The focus of this study was the canonical Wnt signaling pathway. In future studies, it would be interesting to investigate the requirement of noncanonical Wnt ligands with different FZD receptors.

Our map (Fig. 6A) indicates that the similarity in Wnt activity pattern among FZDs can be linked to the phylogenetic tree of these receptors (59, 60). Previously performed phylogenetic analyses based on protein sequence homology have revealed that FZDs can be grouped into 4 main clusters. FZD receptors within a cluster share higher identity—FZD1,2,7 (75%), FZD5,8 (70%), and FZD4,9,10 and FZD3,6 (50%)—than FZDs from different clusters (20–40%). FZDs of one cluster might be functionally related, assuming that each cluster derives from a common parental gene (40, 60). Indeed FZD1, FZD2, and FZD7 share up to 75% homology and specifically induce signaling after binding of Wnt3 and Wnt3a. FZD5 and FZD8 share ∼70% homology and are the most promiscuous receptors. These results indicate that, in agreement with a previous study (56), FZDs with high amino acid sequence similarity possess similar Wnt activity patterns.

In summary, we show that multitargeting sgRNAs which simultaneously disrupt several FZDs can be applied to study Wnt-FZD interactions and their effects on canonical Wnt signaling. This approach can be extended in the future to analyze noncanonical Wnt signaling, as well as other signaling pathways relying on homologous genes. Our results on the one hand highlight the similarities between certain canonical Wnt ligands and receptor family members and on the other hand reveal unexpected dependencies among certain Wnts and FZDs.
