---
pmid: '26642240'
title: A missense mutation in TFRC, encoding transferrin receptor 1, causes combined
  immunodeficiency.
authors:
- Jabara HH
- Boyden SE
- Chou J
- Ramesh N
- Massaad MJ
- Benson H
- Bainter W
- Fraulino D
- Rahimov F
- Sieff C
- Liu ZJ
- Alshemmari SH
- Al-Ramadi BK
- Al-Dhekri H
- Arnaout R
- Abu-Shukair M
- Vatsayan A
- Silver E
- Ahuja S
- Davies EG
- Sola-Visner M
- Ohsumi TK
- Andrews NC
- Notarangelo LD
- Fleming MD
- Al-Herz W
- Kunkel LM
- Geha RS
journal: Nat Genet
year: '2016'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4696875
doi: 10.1038/ng.3465
---

# A missense mutation in TFRC, encoding transferrin receptor 1, causes combined immunodeficiency.
**Authors:** Jabara HH, Boyden SE, Chou J, Ramesh N, Massaad MJ, Benson H, Bainter W, Fraulino D, Rahimov F, Sieff C, Liu ZJ, Alshemmari SH, Al-Ramadi BK, Al-Dhekri H, Arnaout R, Abu-Shukair M, Vatsayan A, Silver E, Ahuja S, Davies EG, Sola-Visner M, Ohsumi TK, Andrews NC, Notarangelo LD, Fleming MD, Al-Herz W, Kunkel LM, Geha RS
**Journal:** Nat Genet (2016)
**DOI:** [10.1038/ng.3465](https://doi.org/10.1038/ng.3465)
**PMC:** [PMC4696875](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4696875/)

## Abstract

1. Nat Genet. 2016 Jan;48(1):74-8. doi: 10.1038/ng.3465. Epub 2015 Dec 7.

A missense mutation in TFRC, encoding transferrin receptor 1, causes combined 
immunodeficiency.

Jabara HH(1)(2), Boyden SE(3)(4)(5), Chou J(1)(2), Ramesh N(1)(2), Massaad 
MJ(1)(2), Benson H(1), Bainter W(1), Fraulino D(1), Rahimov F(3)(4), Sieff 
C(2)(6), Liu ZJ(2)(7), Alshemmari SH(8), Al-Ramadi BK(9), Al-Dhekri H(10), 
Arnaout R(10), Abu-Shukair M(11), Vatsayan A(12), Silver E(13), Ahuja S(12), 
Davies EG(14), Sola-Visner M(2)(7), Ohsumi TK(5)(15), Andrews NC(16)(17), 
Notarangelo LD(1)(2), Fleming MD(18), Al-Herz W(19), Kunkel LM(2)(3)(4)(5), Geha 
RS(1)(2).

Author information:
(1)Division of Immunology, Boston Children's Hospital, Boston, Massachusetts, 
USA.
(2)Department of Pediatrics, Harvard Medical School, Boston, Massachusetts, USA.
(3)Division of Genetics and Genomics, Boston Children's Hospital, Boston, 
Massachusetts, USA.
(4)Manton Center for Orphan Disease Research, Boston Children's Hospital, 
Boston, Massachusetts, USA.
(5)Department of Genetics, Harvard Medical School, Boston, Massachusetts, USA.
(6)Division of Hematology, Boston Children's Hospital, Boston, Massachusetts, 
USA.
(7)Division of Neonatology, Boston Children's Hospital, Boston, Massachusetts, 
USA.
(8)Department of Medicine, Faculty of Medicine, Kuwait University, Kuwait City, 
Kuwait.
(9)Department of Medical Microbiology and Immunology, College of Medicine and 
Health Sciences, United Arab Emirates University, Al-Ain, United Arab Emirates.
(10)King Faisal Specialist Hospital and Research Center, Riyadh, Saudi Arabia.
(11)Immunology, Allergy and Rheumatology Division, Queen Rania Hospital for 
Children, Amman, Jordan.
(12)Department of Pediatric Hematology/Oncology, University Hospitals Rainbow 
Babies and Children's Hospital, Case Western Reserve University, Cleveland, 
Ohio, USA.
(13)Division of Pulmonology and Sleep Medicine, University Hospitals Rainbow 
Babies and Children's Hospital, Case Western Reserve University, Cleveland, 
Ohio, USA.
(14)Centre for Immunodeficiency, Great Ormond Street Hospital and Institute of 
Child Health, London, UK.
(15)Department of Molecular Biology, Massachusetts General Hospital, Boston, 
Massachusetts, USA.
(16)Department of Pediatrics, Duke University School of Medicine, Durham, North 
Carolina, USA.
(17)Department of Pharmacology and Cancer Biology, Duke University School of 
Medicine, Durham, North Carolina, USA.
(18)Department of Pathology, Boston Children's Hospital and Harvard Medical 
School, Boston, Massachusetts, USA.
(19)Department of Pediatrics, Faculty of Medicine, Kuwait University, Kuwait 
City, Kuwait.

Comment in
    Nat Genet. 2016 Jan;48(1):10-1. doi: 10.1038/ng.3478.

Patients with a combined immunodeficiency characterized by normal numbers but 
impaired function of T and B cells had a homozygous p.Tyr20His substitution in 
transferrin receptor 1 (TfR1), encoded by TFRC. The substitution disrupts the 
TfR1 internalization motif, resulting in defective receptor endocytosis and 
markedly increased TfR1 expression on the cell surface. Iron citrate rescued the 
lymphocyte defects, and expression of wild-type but not mutant TfR1 rescued 
impaired transferrin uptake in patient-derived fibroblasts. Tfrc(Y20H/Y20H) mice 
recapitulated the immunological defects of patients. Despite the critical role 
of TfR1 in erythrocyte development and function, patients had only mild anemia 
and only slightly increased TfR1 expression in erythroid precursors. We show 
that STEAP3, a metalloreductase expressed in erythroblasts, associates with TfR1 
and partially rescues transferrin uptake in patient-derived fibroblasts, 
suggesting that STEAP3 may provide an accessory TfR1 endocytosis signal that 
spares patients from severe anemia. These findings demonstrate the importance of 
TfR1 in adaptive immunity.

DOI: 10.1038/ng.3465
PMCID: PMC4696875
PMID: 26642240 [Indexed for MEDLINE]

## Full Text

Combined immunodeficiencies (CID) are characterized by life-threatening infections due to genetic defects resulting in impaired T and B lymphocyte development or function1. The 41 documented monogenic causes of CID have identified pathways and molecules important for adaptive immunity, but many patients with CID remain without a genetic diagnosis1. We report the first human immunodeficiency caused by defective iron transport.

Fourteen Kuwaiti children in Family A (Supplementary Fig. 1) had severe childhood infections leading to the death of six individuals (Supplementary Table 1). Three patients (A1, A2, and A3) followed at our center had hypogammaglobulinemia, normal lymphocyte counts, intermittent neutropenia, and intermittent thrombocytopenia (Supplementary Tables 2 and 3). Hematologic parameters were normal except for borderline-low hemoglobin in two patients and low mean corpuscular volume (MCV) in all three (Supplementary Table 2). Data on six additional patients revealed severe hypogammaglobulinemia and mild anemia resistant to iron supplementation (data not shown). Eight patients received early matched sibling hematopoietic stem cell transplantation (HSCT), with resolution of clinical and laboratory abnormalities.

Patient 1 of Family B from western Saudi Arabia is a five-year old son of consanguineous parents, with early-onset chronic diarrhea and recurrent infections (Supplementary Table 1). He had agammaglobulinemia, normal lymphocyte counts, intermittent thrombocytopenia, mildly low hemoglobin, and low MCV (Supplementary Tables 2 and 3). He was treated with anti-CD20 antibody for presumed autoimmune thrombocytopenia, resulting in loss of circulating B cells without clinical improvement.

The numbers of circulating total (CD3+), helper (CD4+), and cytotoxic (CD8+) T cells, natural killer (CD3−CD16+/CD56+) cells, and B (CD19+) cells in the patients were normal or near normal. However, percentages of CD19+CD27+ memory B cells, important for antibody production, were significantly reduced (Supplementary Table 3). Proliferation of peripheral blood mononuclear cells (PBMCs) in response to the mitogen phytohemagglutinin (PHA), crosslinking of the T cell receptor (TCR) with anti-CD3 antibody, and phorbol 12-myristate 13-acetate and ionomycin (PMA+IO), which bypass the TCR, was significantly decreased in all four patients (Fig. 1a). T cell co-stimulation using anti-CD28 antibody or addition of IL-2 growth factor did not correct the defective TCR-driven proliferation, which was not associated with increased apoptosis (data not shown). These observations demonstrate a global defect in T cell proliferation.

Ligation of CD40 on B cells by CD40 ligand expressed on activated T cells in the presence of IL-4 causes proliferation-dependent immunoglobulin class-switch recombination from IgM to IgG and IgE, reflective of high-affinity, protective antibody production2. Proliferation and secretion of IgG and IgE in response to anti-CD40+IL-4 were significantly decreased in patients’ PBMCs (Fig. 1b). IgE switching requires expression of Iε-Cε germline transcripts, which are early products of class-switch recombination, and activation-induced cytidine deaminase (AICDA), which initiates deletional switch recombination followed by expression of mature Iμ-Cε transcripts3. The patients had normal expression of immature Iε-Cε germline transcripts and AICDA mRNA in their B cells, but undetectable mature Iμ-Cε transcripts (Fig. 1c and Fig. 3c). Collectively, these data demonstrate impaired T cell proliferation as well as defective B cell proliferation and class switching, which in combination constitute the mechanism underlying the susceptibility to severe infections characteristic of CID1.

Genome-wide linkage scans of Family A implicated a single locus at chromosome 3q28-29, but no pathogenic mutation was found within this linkage peak (Supplementary Text). Therefore, whole genome sequencing was performed on Patient A1, his unaffected father, and Patient A2. A missense mutation in TFRC (c.58T>C, NM_003234.2), which encodes transferrin receptor 1 (TfR1, also known as CD71), was the only rare nonsynonymous or splice site mutation homozygous in both patients and heterozygous in the obligate carrier father (Fig. 2a). TFRC is located 919 kb downstream of the distal boundary of the linkage peak, which can be explained by a recent de novo occurrence of the mutation and segregation of both mutant and non-mutant copies of the disease haplotype within the family (Supplementary Text). The c.58T>C mutation segregated perfectly with the phenotype in 34 available family members and was absent from multiple variant databases and 731 genotyped controls (Supplementary Table 4). The resulting p.Tyr20His (Y20H, NP_003225.2) substitution disrupts the TfR1 intracellular internalization motif4 (Fig. 2b), and the p.Y20 residue is perfectly conserved in 81 non-human vertebrate species surveyed (Supplementary Fig. 2).

Due to the similarities among patients from Families A and B, Sanger sequencing of the c.58T>C mutation was performed in Family B; it was homozygous in Patient B1 and heterozygous in his parents and his sister (Fig. 2a). Although the families were from different geographic regions and not known to be related, Patient B1 shares a homozygous haplotype with the five genotyped patients from Family A across a 3.3 Mb interval at chromosome 3q29-ter that includes TFRC, suggesting identical by descent inheritance of the mutation from an unknown common ancestor.

TfR1 is important for cellular iron uptake. Circulating apotransferrin binds two Fe3+ ions to form holotransferrin, which binds to TfR15. The TfR1-holotransferrin complex is internalized by receptor-mediated endocytosis, which requires an aromatic residue at the p.Y20 position mutated in the patients6. When intracellular iron is low, iron regulatory proteins bind to the 3’ untranslated region of TFRC mRNA, increasing its stability7. TFRC mRNA and TfR1 protein levels were higher in PBMCs from patients than in controls (Fig. 2c–d), consistent with this feedback mechanism. Surface TfR1 was minimally expressed on unstimulated control T and B cells, but was expressed on a large percentage of patients’ T and B cells, at levels 13- and 7-fold higher, respectively, than in controls (Fig. 2e). Likewise, surface TfR1 expression was 6-fold higher on patient than control fibroblasts (Supplementary Fig. 3a). Soluble TfR1, generated by cleavage of surface TfR1, was significantly elevated in the patients’ sera (Supplementary Table 2), consistent with increased surface TfR1 expression. TfR1 endocytosis was examined in Epstein-Barr virus-transformed B cells or activated T cells, both of which highly express surface TfR1. Crosslinking of TfR1 dramatically decreased its surface expression on control but not patient cells, demonstrating that the p.Tyr20His mutation impairs TfR1 internalization (Fig. 2f and Supplementary Fig. 3b). Furthermore, transduction of wild type TfR1, but not TfR1Y20H or GAPDH, corrected defective transferrin uptake by patient fibroblasts (Fig. 2g–h). HSCT corrected TfR1 expression on patient T and B cells and fully corrected in vitro T and B cell function (Supplementary Fig. 4).

TfR1-mediated iron endocytosis is essential for lymphocyte development and proliferation8,9. Lymphocytes can also internalize iron through non-transferrin-bound iron pathways10,11. Addition of iron citrate, which supersaturates transferrin so that excess free iron is internalized independently of TfR1, corrected the patients’ lymphocyte proliferation, IgE secretion, and Iμ-Cε transcript expression (Fig. 3). This observation demonstrates that insufficient iron uptake is the cause of the defective T and B cell activation underlying the patients’ CID. Moreover, the patients’ CID phenotype indicates that non-transferrin-bound iron pathways cannot compensate for impaired TfR1 internalization in vivo, thus demonstrating the essential role of TfR1 in host immunity.

To further support the pathogenicity of the p.Tyr20His mutation, we generated TfrcY20H/Y20H knock-in mice (Supplementary Fig. 5a–b). In contrast to Tfrc−/− null mice12, TfrcY20H/Y20H mutant mice were viable, indicating that the mutation is hypomorphic. TfrcY20H/Y20H mice had significantly decreased serum IgG, hemoglobin, and MCV compared to controls, but normal percentages of splenic T and B cells, naïve and memory T cells, NK cells, and iNKT cells (Supplementary Table 5). However, TfrcY20H/Y20H T cells proliferated poorly to anti-CD3 and PMA+IO, which was significantly improved by addition of iron citrate (Fig. 4a–b). TfrcY20H/Y20H B cells proliferated poorly to anti-CD40+IL-4 and LPS+IL-4 stimulation, with minimal IgG1 and IgE secretion (Fig. 4c). TfR1 surface expression on mutant T and B cells was significantly increased, reflecting impaired internalization (Fig. 4d–e).

The stringent requirement for murine TfR1 for hemoglobin synthesis in erythroblasts and erythrocyte development accounts for the embryonic lethality observed in Tfrc−/− mice12. Therefore, the mild anemia in the patients was unexpected. Surface expression of TfR1 on glycophorin A (CD235a)+ erythroid precursor cells (EPCs) from bone marrow was increased by only 2- to 2.5-fold in patients compared to controls (Fig. 5a and Supplementary Fig. 6a–b). Importantly, TfrcY20H/Y20H EPCs internalized comparable amounts of TfR1 compared to wild type (WT) EPCs (Fig. 5b), suggesting a compensatory mechanism for TfR1Y20H internalization specific to EPCs. STEAP proteins are conserved ferrireductases that co-localize with TfR113 and possess a cytoplasmic YXXF motif14 similar to the 20YTRF23 internalization motif of TfR115. The Y288 residue in this motif is critical for STEAP3 internalization14 and Steap3−/− mice exhibit increased surface TfR1 expression on erythroblasts and reticulocytes16,17, suggesting that STEAP3 is important for optimal TfR1 internalization in erythroid cells. STEAP3 mRNA was highly expressed in normal EPCs, whereas all four STEAP genes were poorly expressed in normal fibroblasts, T cells, and B cells (Fig. 5c). Wild type TfR1 and STEAP3 co-immunoprecipitated from lysates of co-transfected HEK293T cells, and both WT and the internalization-defective STEAP3Y288H mutant associated with the TfR1Y20H mutant (Fig. 5d–e). Transfection of WT STEAP3, but not STEAP3Y288H, partially rescued defective transferrin uptake in patients’ fibroblasts (Fig. 5f). These findings suggest that through interaction with TfR1, STEAP3 facilitates TfR1 endocytosis in EPCs, which may protect the patients from severe anemia.

Heretofore, the role of TfR1 in host defense had not been established, and TFRC mutations had not been implicated in human disease. We describe a new CID caused by a homozygous mutation in TFRC that hinders TfR1-mediated iron internalization, resulting in defective T and B cell proliferation as well as impaired class switching, which is necessary for antibody production. To our knowledge, this is the first human mutation implicating impaired iron transport in the pathogenesis of an immunodeficiency. We demonstrate that insufficient iron internalization constitutes the mechanism underlying this CID through in vitro correction of the patients’ lymphocyte defects with iron citrate, which is internalized independently of TfR1. Furthermore, the relative importance of transferrin-dependent and non-transferrin-bound iron to host immunity was previously unknown. Both the patients and the TfrcY20H/Y20H mouse model demonstrate the essential role of iron internalized via TfR1 in immune function. Our data also suggest a novel pathway for TfR1 endocytosis mediated by its interacting partner STEAP3, which may account for the mild effect of the mutation on erythropoiesis. Of clinical importance, we have taken advantage of elevated TfR1 surface expression on lymphocytes to diagnose two patients at birth, thereby facilitating early cure by HSCT.

Online methodsStudy participantsWe enrolled 34 members of a consanguineous Kuwaiti family (Family A) and 4 members of a consanguineous family from western Saudi Arabia (Family B) in the study. Genomic DNA was prepared from saliva, whole blood samples, or skin fibroblast cells. Informed consent was provided by adult donors or by the children’s parents or guardian on forms approved by local ethics committees and by the Institutional Review Board at Boston Children’s Hospital. DNA samples from 479 healthy donors of Middle Eastern ancestry and 252 subjects of European ancestry were used as controls. Protocols used in the human studies have been approved by the Committee on Clinical Investigations at Boston Children’s Hospital.Genetic analysisGenomic DNA from 32 subjects (5 affected and 27 unaffected) from Family A and the proband from Family B was genotyped at 909,622 single nucleotide polymorphisms (SNPs) on the Genome-Wide Human SNP 6.0 Array (Affymetrix). Genotype calling for linkage analysis in Family A was performed using Birdseed v2 with a stringent confidence threshold of 0.02. To enrich for the highest-performing and most informative SNPs, we then filtered the panel to include only 145,751 SNPs with an across-sample SNP Call Rate of 1.00, a Hardy-Weinberg equilibrium P value ≥ 10−3, and a minor allele frequency > 0.Due to the high sample numbers and multiple consanguinity loops, the full Family A pedigree was much larger than could be analyzed by any program that computes an exact LOD score. Genome-wide linkage scans were therefore conducted with MERLIN v1.1.218 using a series of overlapping subfamilies. A focused linkage analysis of 127 SNPs on chromosome 3q28-ter and using all available subjects was conducted with SimWalk2 v2.91, which can analyze pedigrees of arbitrary size but computes an approximate rather than exact LOD score19.Polymerase chain reaction (PCR) and Sanger DNA sequencing were performed according to standard protocols. Copy number analysis was performed in Genotyping Console software (Affymetrix). The mutation was genotyped in DNA samples from unrelated control subjects using a Custom TaqMan SNP Genotyping Assay (Applied Biosystems) on the 7900HT Sequence Detection System (Applied Biosystems). Quantitative reverse transcriptase (qRT)-PCR of positional candidate genes was performed using TaqMan Gene Expression Assays (Applied Biosystems) on cDNA synthesized with the High Capacity RNA-to-cDNA Master Mix (Applied Biosystems), on the BioMark 48.48 Dynamic Array platform (Fluidigm), and analyzed using qbasePLUS v1.5 (Biogazelle).Whole genome sequencing and preliminary analysis (initial mappings of reads, local de novo assembly, and variant calling and annotation) were performed commercially by Complete Genomics, Inc20. We filtered variants for those that were homozygous in two affected subjects, heterozygous in an obligate carrier parent, absent from dbSNP v131, and altered the amino acid sequence of an encoded protein or were in a consensus splice acceptor or splice donor site within 2 bp of an exon. The disease-causing variant has been submitted to ClinVar.Cell preparationHuman peripheral blood mononuclear cells (PBMCs) were isolated from heparinized blood obtained from patients and unaffected donors using Ficoll-Hypaque (GE Healthcare). Cells were suspended in RPMI-1640 containing 10% heat-inactivated fetal calf serum (FCS, Hyclone), 2 mM L-glutamine, 50 μg/ml streptomycin, and 100 U/ml penicillin (medium). Epstein-Barr virus (EBV)-transformed cell lines were established as previously described21. Fibroblast cell lines were established from skin biopsies following standard procedures and were used after at least 4–5 passages. All cell lines were prepared using mycoplasma-free reagents. T and B cell populations were purified from PBMCs by negative magnetic separation (Miltenyi Biotec). Glycophorin A+ red cell precursors and CD33+ myeloid cell progenitors were isolated from BM by positive magnetic sorting (Miltenyi Biotec).Cell culturesPBMCs (1 × 106 and 1.5 × 106 cells/ml for proliferation and immunoglobulin production respectively) were cultured with medium, phytohemagglutinin (PHA, 1 μg/ml, Sigma), anti-CD3 (100 ng/ml, OKT3, eBioscience), phorbol 12-myristate 13-acetate (PMA, 20 nM, Sigma) plus ionomycin (IO, 0.5 μM, Sigma), and anti-CD40 (5 μg/ml, #626.1) plus recombinant (r) IL-4 (5 ng/ml, R&D Systems) for 4 days and with tetanus toxoid (TT, National Institute for Biological Standards and Control) for 6 days. Ferric ammonium citrate (0.5 μg/ml, Sigma) was added where indicated. Proliferation was assayed by measuring [3H]thymidine incorporation added for the last 16 hours of culture. The proliferative index (PI) was calculated as follows: PI=Log[FInd/MFIall]/Log[2], where FInd represents the peak fluorescence intensity of the viable non-divided cells and MFIall represents the median fluorescence intensity of all viable T cells. The production of IgE and IgG was measured in culture supernatants at 14 days by enzyme-linked immunosorbent assay (ELISA) as described22. PMA+IO-mediated up-regulation of CD25, CD69, and CD40L, and CD40+IL-4-mediated up-regulation of CD23 and CD86 expression were analyzed by flow cytometry after 24 and 48 hours respectively. IL-2 in culture supernatants was measured after 48 hours with ELISA kits from eBioscience.Flow cytometryAnti-human monoclonal antibodies (mAbs) to the following molecules and the appropriate isotype controls were used for staining: CD3 (HIT3a), CD19 (HIB19), CD27 (M-T271), CD23 (M-L233), CD86 (2331, FUN-1), TfR1 (M-A712), CD235a (GA-R2, HIR2), CD41a (HIP8), and CD42b (HIP1) from BD Biosciences; CD25 (BC96) and CD69 (FN50) from eBioscience; and CD40L (40804) from R&D Systems. Standard flow cytometric methods were used for staining of cell-surface proteins. Data collected with a FACSCalibur or a LSRFortessa cell analyzer (BD Biosciences) were analyzed with FlowJo software (TreeStar).RT-PCR and qRT-PCRRNA was extracted from freshly isolated or cultured cells using TRIzol (Invitrogen) and was reverse transcribed by Superscript II (Invitrogen) according to the manufacturer’s instructions using gene-specific PCR primers (Supplementary Table 6). Quantitative RT-PCR for TfR1 and STEAPs mRNA expression was performed using primers and an ABI Prism 7300 Sequence Detection System from Applied Biosystems. Relative RNA expression normalized to GAPDH mRNA abundance was calculated by the relative standard curve method as outlined in the manufacturer’s technical bulletin (Applied Biosystems).Immunoblot analysisCell lysates from PBMCs were separated by SDS/PAGE gel and transferred to nitrocellulose membranes. Specific proteins were detected using anti-human TfR1 mAb (H68.4, Invitrogen) and blots were re-probed with anti-actin mAb (Chemicon) as a loading control.TfR1 internalizationEBV-transformed B cells from patients and controls were washed with cold PBS and incubated with mouse anti-human TfR1 antibody (10 μg/ml, M-A712, BD Biosciences) or with mouse IgG2a isotype control for 30 minutes on ice. Cells were then washed with cold PBS, resuspended in cold PBS, and an aliquot was stained using a FITC-conjugated rat anti-mouse IgG2a (RMG2a-62, BioLegend) to measure the baseline for TfR1 expression. The rest of the cells were placed at 37°C for 15 and 30 minutes. Internalization was stopped by incubating the cells on ice and washing with cold PBS followed by staining and flow cytometry analysis23. Percent TfR1 internalized was calculated as follows: (MFI at zero min − MFI at specified time point) × 100 / MFI at zero min.Transferrin uptake assayPatient and control fibroblasts were cultured on 12 mm coverslips in DMEM containing 15% FCS. Cells were washed with PBS and incubated at 37°C overnight in serum-free DMEM. Transferrin (TF) conjugated to Alexafluor 568 (25 μg/ml, Invitrogen) was added to TF-starved cells for 30 minutes at 37°C. Following incubation, cells were rinsed with ice-cold PBS and incubated in ice-cold acid wash buffer (0.025 M citric acid, 0.025 M sodium citrate, 0.15 M sucrose, and 200 μM deferoxamine mesylate, all from Sigma) for exactly 3 minutes. This was followed by extensive rinses with serum-free medium and PBS. Cells were fixed in 4% paraformaldehyde in PBS, rinsed with PBS, and mounted on coverslips using mounting medium (Prolong Gold with DAPI, Invitrogen). When dry, the coverslips were sealed and imaged using a Nikon Eclipse 800 microscope. Mean red fluorescence intensity of fifty cells from 8–10 different fields was quantified using Nikon NS2 v3.2 imaging software.Transduction of patient fibroblasts with human TFRCWild type human (h) TFRC and GAPDH cDNA open reading frames, cloned in the expression vector pLOC under the control of a CMV promoter, were purchased from Open Biosystems. The QuikChange site-directed mutagenesis kit (Stratagene) was used to generate TFRC p.Tyr20His mutant (Mut) in the same vector. The vector expresses the green fluorescent protein TurboGFP as a transfection marker independently through an internal ribosome entry site. The vector was packaged into lentiviral particles using pseudotyped vesicular stomatitis virus lentiviral packaging system (Cell Biolabs, Inc.). Patient and control fibroblasts were pre-treated with polybrene (0.5 μg/ml, Sigma) and infected with lentivirus at a multiplicity of infection of 10. Eighteen hours post-infection, the medium was replaced with fresh culture medium and the cells were monitored every two hours for GFP expression. The transferrin uptake assay was performed when peak GFP expression was observed (24–28 hours). Mean red fluorescence intensity of 50 GFP-positive cells from 8–10 different fields was quantified using Nikon NS2 v3.2 software.Nucleofection of Steap3 into patient fibroblastsFlag-tagged murine Steap3 (six transmembrane epithelial antigen of prostate) and Flag-tagged mutant Steap3Y288H were cloned in the expression vector pCMV-Tag3. One microgram of wild type or mutant Steap3-expressing plasmid was nucleofected into 0.75×106 fibroblasts with Amaxa nucleofector (Lonza) using the program U-023 (high efficiency for normal human dermal fibroblasts) and plated on six-well clusters. After 24 hours, cells were trypsinized and plated onto 12 mm coverslips. The transferrin uptake assay was performed after 24 hours as described above and the fixed cells were permeabilized with 0.2% saponin in PBS. After blocking with 4% BSA in PBS, the cells were incubated with mouse Flag mAb (M2, Sigma), rinsed with Tris-buffered saline (TBS), and incubated with goat anti-mouse antibody conjugated to Alexafluor 488 (Invitrogen). After incubation, coverslips were rinsed with TBS and mounted and sealed as described above in the transferrin uptake section. Mean red fluorescence intensity of 25 Flag-positive cells from 6–8 different fields was quantified using Nikon NS2 v3.2 software.Transient transfections and co-immunoprecipitationHEK 293T cells (ATCC) were co-transfected using Lipofectamine LTX (Invitrogen) with plasmids encoding Myc-DDK-tagged wild type human TFRC (Origene) or p.Tyr20His mutant TFRC (prepared using the QuikChange site-directed mutagenesis kit from Stratagene) with either Flag-tagged wild type murine Steap3, Flag-tagged mutant Steap3Y288H, or Flag-tagged PYK2. After 19–24 hours, cells were lysed with buffer containing 1% BRIJ 35 (Sigma). Immunoprecipitation was performed using c-Myc mAb (9E10, Millipore), followed by agarose protein G plus beads (Calbiochem). The complexes were separated by SDS-PAGE, transferred to nitrocellulose membranes, and immunoblotted with antibodies against c-Myc (9E10, Millipore) and Flag (M2, Sigma).Animal studiesAll experimental procedures performed on mice were approved by the Animal Care and Use Committee of Boston Children’s Hospital, Boston, MA, United States of America. All experiments used 7–12 week old, sex-matched TfrcY20H/Y20H and WT littermates.Generation of TfrcY20H/Y20H knock-in miceA gene-targeting construct was generated to introduce the p.Tyr20His mutation into exon 3 of mouse Tfrc (22). DNA fragments 3.5 kb and 3.3 kb in length were PCR-amplified from the Tfrc gene in C57BL/6 genomic DNA and cloned into a gene-targeting vector upstream and downstream, respectively, of a neomycin resistance gene under the phosphoglycerate kinase 1 promoter, used as a positive selectable marker. The diphtheria toxin A gene fragment, located outside the homology arms, was used as a negative selectable marker. The c.58T>C mutation was introduced into the construct by site-directed mutagenesis, and the mutant clone was sequenced to ensure the absence of off-target PCR-induced mutations. The construct was then linearized and electroporated into C57BL/6N embryonic stem (ES) cells, which were then selected for neomycin resistance using standard techniques24. ES cell clones that had successful integration of the targeting construct, as determined by Southern blot and PCR, were injected into C57BL/6 blastocysts, which were implanted into foster mothers. Offspring were genotyped by PCR and Sanger sequencing. Chimeric mice were backcrossed to generate germline TfrcY20H/+ heterozygous mice, which were then intercrossed to produce TfrcY20H/Y20H homozygous mutant mice. All mice were kept in a pathogen-free environment. Retro-orbital bleeds were performed to obtain peripheral blood for quantification of murine hematologic parameters (Hemavet 950FS) and serum immunoglobulins, which were quantified using standard ELISA techniques.Murine cell cultures and reagentsAnti-mouse monoclonal antibodies (mAbs) to the following molecules and the appropriate isotype controls were used for flow cytometry: CD4 (GK1.4), CD8 (53-6.7), CD62L (MEL-14) from BioLegend; CD44 (IM7), CD25 (PC61.5), FOXP3 (FJK-16S), NK1.1 (PK136), CD49b (DX5), Ter119 (TER-119) from eBioscience. Murine CD1d (α-GalCer analog) from the NIH Tetramer Facility-Emory University Vaccine Center, Atlanta, GA. Murine splenic T and B cells were purified by negative selection (Pan-T Cell Isolation Kit II and CD43 Kit respectively, Miltenyi Biotec, Inc.). Purified murine T cells were labeled with CellTrace Violet (Invitrogen) according to the manufacturer’s protocol, followed by stimulation with plate-bound anti-CD3ε (5 μg/ml, 145-2C1) mAb, soluble anti-CD28 (2 μg/ml, 37.51) mAb (both from eBioscience), murine rIL-2 (40 ng/ml, PeproTech), and PMA+IO for 96 hours prior to fluorescence-activated cell sorting (FACS) analysis. T cell proliferation was measured using the proliferative index (PI), calculated as follows: PI=Log[FInd/MFIall]/Log[2], where FInd represents the peak fluorescence intensity of the viable non-divided cells and MFIall represents the median fluorescence intensity of all viable T cells.Purified B cells were stimulated with murine rIL-4 (50 ng/ml, R&D systems) plus either anti-mouse CD40 (1 μg/ml, HM40-3, Pharmingen) or lipopolysaccharide (LPS) (10 μg/ml, Sigma). Proliferation was assayed at day 4 by measuring [3H]-thymidine incorporation, added for the last 16–20 hours of culture. On Day 7 supernatants were collected and assayed for IgE and IgG1 by ELISA. Standard flow cytometric methods were used for staining of cell-surface proteins, and data were collected and analyzed as shown for human cell staining. Fluorochrome-labeled mAbs to the following murine molecules and the appropriate controls were used for staining: TfR1 (R17217) from eBioscience, and CD3 (17A2) and B220 (RA3-6B2) from BioLegend. For TfR1 internalization, resting mutant T cells, purified wild type or mutant T cells stimulated with PMA+IO for 16–18 hours, or bone marrow harvested from murine femurs were used, following the same procedure performed on human EBV-transformed B cells. Erythroid precursors were identified using fluorochrome-labeled mAbs to CD44 (IM7) and Ter119 (Ter-119) from eBioscience. Purified rat anti-mouse TfR1 (R17217) was used for crosslinking and phycoerythrin-conjugated goat anti-rat IgG was used for secondary detection of surface TfR1 expression. Percent TfR1 internalized was calculated as described previously in the Methods. Internalized TFR1 in murine erythroblasts was calculated as follows: MFI at zero minutes – MFI after 30 minutes of TFR1 cross-linking.Statistical analysisStudent’s t-test was used to compare groups with equal variances and two-way ANOVA was used for comparing more than two groups. For animal experiments, the sample size was determined using a conservative estimate of a 50% decrease in lymphocyte proliferation, immunoglobulin secretion, TfR1 expression, and TfR1 internalization in TfrcY20H/Y20H cells compared to controls, with a standard deviation of 10%. To detect this difference with 95% power, we would need at least two mice of each genotype for each experiment. All experiments were performed a minimum of two times unless otherwise indicated.

Online methods

Study participantsWe enrolled 34 members of a consanguineous Kuwaiti family (Family A) and 4 members of a consanguineous family from western Saudi Arabia (Family B) in the study. Genomic DNA was prepared from saliva, whole blood samples, or skin fibroblast cells. Informed consent was provided by adult donors or by the children’s parents or guardian on forms approved by local ethics committees and by the Institutional Review Board at Boston Children’s Hospital. DNA samples from 479 healthy donors of Middle Eastern ancestry and 252 subjects of European ancestry were used as controls. Protocols used in the human studies have been approved by the Committee on Clinical Investigations at Boston Children’s Hospital.

Study participants

We enrolled 34 members of a consanguineous Kuwaiti family (Family A) and 4 members of a consanguineous family from western Saudi Arabia (Family B) in the study. Genomic DNA was prepared from saliva, whole blood samples, or skin fibroblast cells. Informed consent was provided by adult donors or by the children’s parents or guardian on forms approved by local ethics committees and by the Institutional Review Board at Boston Children’s Hospital. DNA samples from 479 healthy donors of Middle Eastern ancestry and 252 subjects of European ancestry were used as controls. Protocols used in the human studies have been approved by the Committee on Clinical Investigations at Boston Children’s Hospital.

Genetic analysisGenomic DNA from 32 subjects (5 affected and 27 unaffected) from Family A and the proband from Family B was genotyped at 909,622 single nucleotide polymorphisms (SNPs) on the Genome-Wide Human SNP 6.0 Array (Affymetrix). Genotype calling for linkage analysis in Family A was performed using Birdseed v2 with a stringent confidence threshold of 0.02. To enrich for the highest-performing and most informative SNPs, we then filtered the panel to include only 145,751 SNPs with an across-sample SNP Call Rate of 1.00, a Hardy-Weinberg equilibrium P value ≥ 10−3, and a minor allele frequency > 0.Due to the high sample numbers and multiple consanguinity loops, the full Family A pedigree was much larger than could be analyzed by any program that computes an exact LOD score. Genome-wide linkage scans were therefore conducted with MERLIN v1.1.218 using a series of overlapping subfamilies. A focused linkage analysis of 127 SNPs on chromosome 3q28-ter and using all available subjects was conducted with SimWalk2 v2.91, which can analyze pedigrees of arbitrary size but computes an approximate rather than exact LOD score19.Polymerase chain reaction (PCR) and Sanger DNA sequencing were performed according to standard protocols. Copy number analysis was performed in Genotyping Console software (Affymetrix). The mutation was genotyped in DNA samples from unrelated control subjects using a Custom TaqMan SNP Genotyping Assay (Applied Biosystems) on the 7900HT Sequence Detection System (Applied Biosystems). Quantitative reverse transcriptase (qRT)-PCR of positional candidate genes was performed using TaqMan Gene Expression Assays (Applied Biosystems) on cDNA synthesized with the High Capacity RNA-to-cDNA Master Mix (Applied Biosystems), on the BioMark 48.48 Dynamic Array platform (Fluidigm), and analyzed using qbasePLUS v1.5 (Biogazelle).Whole genome sequencing and preliminary analysis (initial mappings of reads, local de novo assembly, and variant calling and annotation) were performed commercially by Complete Genomics, Inc20. We filtered variants for those that were homozygous in two affected subjects, heterozygous in an obligate carrier parent, absent from dbSNP v131, and altered the amino acid sequence of an encoded protein or were in a consensus splice acceptor or splice donor site within 2 bp of an exon. The disease-causing variant has been submitted to ClinVar.

Genetic analysis

Genomic DNA from 32 subjects (5 affected and 27 unaffected) from Family A and the proband from Family B was genotyped at 909,622 single nucleotide polymorphisms (SNPs) on the Genome-Wide Human SNP 6.0 Array (Affymetrix). Genotype calling for linkage analysis in Family A was performed using Birdseed v2 with a stringent confidence threshold of 0.02. To enrich for the highest-performing and most informative SNPs, we then filtered the panel to include only 145,751 SNPs with an across-sample SNP Call Rate of 1.00, a Hardy-Weinberg equilibrium P value ≥ 10−3, and a minor allele frequency > 0.

Due to the high sample numbers and multiple consanguinity loops, the full Family A pedigree was much larger than could be analyzed by any program that computes an exact LOD score. Genome-wide linkage scans were therefore conducted with MERLIN v1.1.218 using a series of overlapping subfamilies. A focused linkage analysis of 127 SNPs on chromosome 3q28-ter and using all available subjects was conducted with SimWalk2 v2.91, which can analyze pedigrees of arbitrary size but computes an approximate rather than exact LOD score19.

Polymerase chain reaction (PCR) and Sanger DNA sequencing were performed according to standard protocols. Copy number analysis was performed in Genotyping Console software (Affymetrix). The mutation was genotyped in DNA samples from unrelated control subjects using a Custom TaqMan SNP Genotyping Assay (Applied Biosystems) on the 7900HT Sequence Detection System (Applied Biosystems). Quantitative reverse transcriptase (qRT)-PCR of positional candidate genes was performed using TaqMan Gene Expression Assays (Applied Biosystems) on cDNA synthesized with the High Capacity RNA-to-cDNA Master Mix (Applied Biosystems), on the BioMark 48.48 Dynamic Array platform (Fluidigm), and analyzed using qbasePLUS v1.5 (Biogazelle).

Whole genome sequencing and preliminary analysis (initial mappings of reads, local de novo assembly, and variant calling and annotation) were performed commercially by Complete Genomics, Inc20. We filtered variants for those that were homozygous in two affected subjects, heterozygous in an obligate carrier parent, absent from dbSNP v131, and altered the amino acid sequence of an encoded protein or were in a consensus splice acceptor or splice donor site within 2 bp of an exon. The disease-causing variant has been submitted to ClinVar.

Cell preparationHuman peripheral blood mononuclear cells (PBMCs) were isolated from heparinized blood obtained from patients and unaffected donors using Ficoll-Hypaque (GE Healthcare). Cells were suspended in RPMI-1640 containing 10% heat-inactivated fetal calf serum (FCS, Hyclone), 2 mM L-glutamine, 50 μg/ml streptomycin, and 100 U/ml penicillin (medium). Epstein-Barr virus (EBV)-transformed cell lines were established as previously described21. Fibroblast cell lines were established from skin biopsies following standard procedures and were used after at least 4–5 passages. All cell lines were prepared using mycoplasma-free reagents. T and B cell populations were purified from PBMCs by negative magnetic separation (Miltenyi Biotec). Glycophorin A+ red cell precursors and CD33+ myeloid cell progenitors were isolated from BM by positive magnetic sorting (Miltenyi Biotec).

Cell preparation

Human peripheral blood mononuclear cells (PBMCs) were isolated from heparinized blood obtained from patients and unaffected donors using Ficoll-Hypaque (GE Healthcare). Cells were suspended in RPMI-1640 containing 10% heat-inactivated fetal calf serum (FCS, Hyclone), 2 mM L-glutamine, 50 μg/ml streptomycin, and 100 U/ml penicillin (medium). Epstein-Barr virus (EBV)-transformed cell lines were established as previously described21. Fibroblast cell lines were established from skin biopsies following standard procedures and were used after at least 4–5 passages. All cell lines were prepared using mycoplasma-free reagents. T and B cell populations were purified from PBMCs by negative magnetic separation (Miltenyi Biotec). Glycophorin A+ red cell precursors and CD33+ myeloid cell progenitors were isolated from BM by positive magnetic sorting (Miltenyi Biotec).

Cell culturesPBMCs (1 × 106 and 1.5 × 106 cells/ml for proliferation and immunoglobulin production respectively) were cultured with medium, phytohemagglutinin (PHA, 1 μg/ml, Sigma), anti-CD3 (100 ng/ml, OKT3, eBioscience), phorbol 12-myristate 13-acetate (PMA, 20 nM, Sigma) plus ionomycin (IO, 0.5 μM, Sigma), and anti-CD40 (5 μg/ml, #626.1) plus recombinant (r) IL-4 (5 ng/ml, R&D Systems) for 4 days and with tetanus toxoid (TT, National Institute for Biological Standards and Control) for 6 days. Ferric ammonium citrate (0.5 μg/ml, Sigma) was added where indicated. Proliferation was assayed by measuring [3H]thymidine incorporation added for the last 16 hours of culture. The proliferative index (PI) was calculated as follows: PI=Log[FInd/MFIall]/Log[2], where FInd represents the peak fluorescence intensity of the viable non-divided cells and MFIall represents the median fluorescence intensity of all viable T cells. The production of IgE and IgG was measured in culture supernatants at 14 days by enzyme-linked immunosorbent assay (ELISA) as described22. PMA+IO-mediated up-regulation of CD25, CD69, and CD40L, and CD40+IL-4-mediated up-regulation of CD23 and CD86 expression were analyzed by flow cytometry after 24 and 48 hours respectively. IL-2 in culture supernatants was measured after 48 hours with ELISA kits from eBioscience.

Cell cultures

PBMCs (1 × 106 and 1.5 × 106 cells/ml for proliferation and immunoglobulin production respectively) were cultured with medium, phytohemagglutinin (PHA, 1 μg/ml, Sigma), anti-CD3 (100 ng/ml, OKT3, eBioscience), phorbol 12-myristate 13-acetate (PMA, 20 nM, Sigma) plus ionomycin (IO, 0.5 μM, Sigma), and anti-CD40 (5 μg/ml, #626.1) plus recombinant (r) IL-4 (5 ng/ml, R&D Systems) for 4 days and with tetanus toxoid (TT, National Institute for Biological Standards and Control) for 6 days. Ferric ammonium citrate (0.5 μg/ml, Sigma) was added where indicated. Proliferation was assayed by measuring [3H]thymidine incorporation added for the last 16 hours of culture. The proliferative index (PI) was calculated as follows: PI=Log[FInd/MFIall]/Log[2], where FInd represents the peak fluorescence intensity of the viable non-divided cells and MFIall represents the median fluorescence intensity of all viable T cells. The production of IgE and IgG was measured in culture supernatants at 14 days by enzyme-linked immunosorbent assay (ELISA) as described22. PMA+IO-mediated up-regulation of CD25, CD69, and CD40L, and CD40+IL-4-mediated up-regulation of CD23 and CD86 expression were analyzed by flow cytometry after 24 and 48 hours respectively. IL-2 in culture supernatants was measured after 48 hours with ELISA kits from eBioscience.

Flow cytometryAnti-human monoclonal antibodies (mAbs) to the following molecules and the appropriate isotype controls were used for staining: CD3 (HIT3a), CD19 (HIB19), CD27 (M-T271), CD23 (M-L233), CD86 (2331, FUN-1), TfR1 (M-A712), CD235a (GA-R2, HIR2), CD41a (HIP8), and CD42b (HIP1) from BD Biosciences; CD25 (BC96) and CD69 (FN50) from eBioscience; and CD40L (40804) from R&D Systems. Standard flow cytometric methods were used for staining of cell-surface proteins. Data collected with a FACSCalibur or a LSRFortessa cell analyzer (BD Biosciences) were analyzed with FlowJo software (TreeStar).

Flow cytometry

Anti-human monoclonal antibodies (mAbs) to the following molecules and the appropriate isotype controls were used for staining: CD3 (HIT3a), CD19 (HIB19), CD27 (M-T271), CD23 (M-L233), CD86 (2331, FUN-1), TfR1 (M-A712), CD235a (GA-R2, HIR2), CD41a (HIP8), and CD42b (HIP1) from BD Biosciences; CD25 (BC96) and CD69 (FN50) from eBioscience; and CD40L (40804) from R&D Systems. Standard flow cytometric methods were used for staining of cell-surface proteins. Data collected with a FACSCalibur or a LSRFortessa cell analyzer (BD Biosciences) were analyzed with FlowJo software (TreeStar).

RT-PCR and qRT-PCRRNA was extracted from freshly isolated or cultured cells using TRIzol (Invitrogen) and was reverse transcribed by Superscript II (Invitrogen) according to the manufacturer’s instructions using gene-specific PCR primers (Supplementary Table 6). Quantitative RT-PCR for TfR1 and STEAPs mRNA expression was performed using primers and an ABI Prism 7300 Sequence Detection System from Applied Biosystems. Relative RNA expression normalized to GAPDH mRNA abundance was calculated by the relative standard curve method as outlined in the manufacturer’s technical bulletin (Applied Biosystems).

RT-PCR and qRT-PCR

RNA was extracted from freshly isolated or cultured cells using TRIzol (Invitrogen) and was reverse transcribed by Superscript II (Invitrogen) according to the manufacturer’s instructions using gene-specific PCR primers (Supplementary Table 6). Quantitative RT-PCR for TfR1 and STEAPs mRNA expression was performed using primers and an ABI Prism 7300 Sequence Detection System from Applied Biosystems. Relative RNA expression normalized to GAPDH mRNA abundance was calculated by the relative standard curve method as outlined in the manufacturer’s technical bulletin (Applied Biosystems).

Immunoblot analysisCell lysates from PBMCs were separated by SDS/PAGE gel and transferred to nitrocellulose membranes. Specific proteins were detected using anti-human TfR1 mAb (H68.4, Invitrogen) and blots were re-probed with anti-actin mAb (Chemicon) as a loading control.

Immunoblot analysis

Cell lysates from PBMCs were separated by SDS/PAGE gel and transferred to nitrocellulose membranes. Specific proteins were detected using anti-human TfR1 mAb (H68.4, Invitrogen) and blots were re-probed with anti-actin mAb (Chemicon) as a loading control.

TfR1 internalizationEBV-transformed B cells from patients and controls were washed with cold PBS and incubated with mouse anti-human TfR1 antibody (10 μg/ml, M-A712, BD Biosciences) or with mouse IgG2a isotype control for 30 minutes on ice. Cells were then washed with cold PBS, resuspended in cold PBS, and an aliquot was stained using a FITC-conjugated rat anti-mouse IgG2a (RMG2a-62, BioLegend) to measure the baseline for TfR1 expression. The rest of the cells were placed at 37°C for 15 and 30 minutes. Internalization was stopped by incubating the cells on ice and washing with cold PBS followed by staining and flow cytometry analysis23. Percent TfR1 internalized was calculated as follows: (MFI at zero min − MFI at specified time point) × 100 / MFI at zero min.

TfR1 internalization

EBV-transformed B cells from patients and controls were washed with cold PBS and incubated with mouse anti-human TfR1 antibody (10 μg/ml, M-A712, BD Biosciences) or with mouse IgG2a isotype control for 30 minutes on ice. Cells were then washed with cold PBS, resuspended in cold PBS, and an aliquot was stained using a FITC-conjugated rat anti-mouse IgG2a (RMG2a-62, BioLegend) to measure the baseline for TfR1 expression. The rest of the cells were placed at 37°C for 15 and 30 minutes. Internalization was stopped by incubating the cells on ice and washing with cold PBS followed by staining and flow cytometry analysis23. Percent TfR1 internalized was calculated as follows: (MFI at zero min − MFI at specified time point) × 100 / MFI at zero min.

Transferrin uptake assayPatient and control fibroblasts were cultured on 12 mm coverslips in DMEM containing 15% FCS. Cells were washed with PBS and incubated at 37°C overnight in serum-free DMEM. Transferrin (TF) conjugated to Alexafluor 568 (25 μg/ml, Invitrogen) was added to TF-starved cells for 30 minutes at 37°C. Following incubation, cells were rinsed with ice-cold PBS and incubated in ice-cold acid wash buffer (0.025 M citric acid, 0.025 M sodium citrate, 0.15 M sucrose, and 200 μM deferoxamine mesylate, all from Sigma) for exactly 3 minutes. This was followed by extensive rinses with serum-free medium and PBS. Cells were fixed in 4% paraformaldehyde in PBS, rinsed with PBS, and mounted on coverslips using mounting medium (Prolong Gold with DAPI, Invitrogen). When dry, the coverslips were sealed and imaged using a Nikon Eclipse 800 microscope. Mean red fluorescence intensity of fifty cells from 8–10 different fields was quantified using Nikon NS2 v3.2 imaging software.

Transferrin uptake assay

Patient and control fibroblasts were cultured on 12 mm coverslips in DMEM containing 15% FCS. Cells were washed with PBS and incubated at 37°C overnight in serum-free DMEM. Transferrin (TF) conjugated to Alexafluor 568 (25 μg/ml, Invitrogen) was added to TF-starved cells for 30 minutes at 37°C. Following incubation, cells were rinsed with ice-cold PBS and incubated in ice-cold acid wash buffer (0.025 M citric acid, 0.025 M sodium citrate, 0.15 M sucrose, and 200 μM deferoxamine mesylate, all from Sigma) for exactly 3 minutes. This was followed by extensive rinses with serum-free medium and PBS. Cells were fixed in 4% paraformaldehyde in PBS, rinsed with PBS, and mounted on coverslips using mounting medium (Prolong Gold with DAPI, Invitrogen). When dry, the coverslips were sealed and imaged using a Nikon Eclipse 800 microscope. Mean red fluorescence intensity of fifty cells from 8–10 different fields was quantified using Nikon NS2 v3.2 imaging software.

Transduction of patient fibroblasts with human TFRCWild type human (h) TFRC and GAPDH cDNA open reading frames, cloned in the expression vector pLOC under the control of a CMV promoter, were purchased from Open Biosystems. The QuikChange site-directed mutagenesis kit (Stratagene) was used to generate TFRC p.Tyr20His mutant (Mut) in the same vector. The vector expresses the green fluorescent protein TurboGFP as a transfection marker independently through an internal ribosome entry site. The vector was packaged into lentiviral particles using pseudotyped vesicular stomatitis virus lentiviral packaging system (Cell Biolabs, Inc.). Patient and control fibroblasts were pre-treated with polybrene (0.5 μg/ml, Sigma) and infected with lentivirus at a multiplicity of infection of 10. Eighteen hours post-infection, the medium was replaced with fresh culture medium and the cells were monitored every two hours for GFP expression. The transferrin uptake assay was performed when peak GFP expression was observed (24–28 hours). Mean red fluorescence intensity of 50 GFP-positive cells from 8–10 different fields was quantified using Nikon NS2 v3.2 software.

Transduction of patient fibroblasts with human TFRC

Wild type human (h) TFRC and GAPDH cDNA open reading frames, cloned in the expression vector pLOC under the control of a CMV promoter, were purchased from Open Biosystems. The QuikChange site-directed mutagenesis kit (Stratagene) was used to generate TFRC p.Tyr20His mutant (Mut) in the same vector. The vector expresses the green fluorescent protein TurboGFP as a transfection marker independently through an internal ribosome entry site. The vector was packaged into lentiviral particles using pseudotyped vesicular stomatitis virus lentiviral packaging system (Cell Biolabs, Inc.). Patient and control fibroblasts were pre-treated with polybrene (0.5 μg/ml, Sigma) and infected with lentivirus at a multiplicity of infection of 10. Eighteen hours post-infection, the medium was replaced with fresh culture medium and the cells were monitored every two hours for GFP expression. The transferrin uptake assay was performed when peak GFP expression was observed (24–28 hours). Mean red fluorescence intensity of 50 GFP-positive cells from 8–10 different fields was quantified using Nikon NS2 v3.2 software.

Nucleofection of Steap3 into patient fibroblastsFlag-tagged murine Steap3 (six transmembrane epithelial antigen of prostate) and Flag-tagged mutant Steap3Y288H were cloned in the expression vector pCMV-Tag3. One microgram of wild type or mutant Steap3-expressing plasmid was nucleofected into 0.75×106 fibroblasts with Amaxa nucleofector (Lonza) using the program U-023 (high efficiency for normal human dermal fibroblasts) and plated on six-well clusters. After 24 hours, cells were trypsinized and plated onto 12 mm coverslips. The transferrin uptake assay was performed after 24 hours as described above and the fixed cells were permeabilized with 0.2% saponin in PBS. After blocking with 4% BSA in PBS, the cells were incubated with mouse Flag mAb (M2, Sigma), rinsed with Tris-buffered saline (TBS), and incubated with goat anti-mouse antibody conjugated to Alexafluor 488 (Invitrogen). After incubation, coverslips were rinsed with TBS and mounted and sealed as described above in the transferrin uptake section. Mean red fluorescence intensity of 25 Flag-positive cells from 6–8 different fields was quantified using Nikon NS2 v3.2 software.

Nucleofection of Steap3 into patient fibroblasts

Flag-tagged murine Steap3 (six transmembrane epithelial antigen of prostate) and Flag-tagged mutant Steap3Y288H were cloned in the expression vector pCMV-Tag3. One microgram of wild type or mutant Steap3-expressing plasmid was nucleofected into 0.75×106 fibroblasts with Amaxa nucleofector (Lonza) using the program U-023 (high efficiency for normal human dermal fibroblasts) and plated on six-well clusters. After 24 hours, cells were trypsinized and plated onto 12 mm coverslips. The transferrin uptake assay was performed after 24 hours as described above and the fixed cells were permeabilized with 0.2% saponin in PBS. After blocking with 4% BSA in PBS, the cells were incubated with mouse Flag mAb (M2, Sigma), rinsed with Tris-buffered saline (TBS), and incubated with goat anti-mouse antibody conjugated to Alexafluor 488 (Invitrogen). After incubation, coverslips were rinsed with TBS and mounted and sealed as described above in the transferrin uptake section. Mean red fluorescence intensity of 25 Flag-positive cells from 6–8 different fields was quantified using Nikon NS2 v3.2 software.

Transient transfections and co-immunoprecipitationHEK 293T cells (ATCC) were co-transfected using Lipofectamine LTX (Invitrogen) with plasmids encoding Myc-DDK-tagged wild type human TFRC (Origene) or p.Tyr20His mutant TFRC (prepared using the QuikChange site-directed mutagenesis kit from Stratagene) with either Flag-tagged wild type murine Steap3, Flag-tagged mutant Steap3Y288H, or Flag-tagged PYK2. After 19–24 hours, cells were lysed with buffer containing 1% BRIJ 35 (Sigma). Immunoprecipitation was performed using c-Myc mAb (9E10, Millipore), followed by agarose protein G plus beads (Calbiochem). The complexes were separated by SDS-PAGE, transferred to nitrocellulose membranes, and immunoblotted with antibodies against c-Myc (9E10, Millipore) and Flag (M2, Sigma).

Transient transfections and co-immunoprecipitation

HEK 293T cells (ATCC) were co-transfected using Lipofectamine LTX (Invitrogen) with plasmids encoding Myc-DDK-tagged wild type human TFRC (Origene) or p.Tyr20His mutant TFRC (prepared using the QuikChange site-directed mutagenesis kit from Stratagene) with either Flag-tagged wild type murine Steap3, Flag-tagged mutant Steap3Y288H, or Flag-tagged PYK2. After 19–24 hours, cells were lysed with buffer containing 1% BRIJ 35 (Sigma). Immunoprecipitation was performed using c-Myc mAb (9E10, Millipore), followed by agarose protein G plus beads (Calbiochem). The complexes were separated by SDS-PAGE, transferred to nitrocellulose membranes, and immunoblotted with antibodies against c-Myc (9E10, Millipore) and Flag (M2, Sigma).

Animal studiesAll experimental procedures performed on mice were approved by the Animal Care and Use Committee of Boston Children’s Hospital, Boston, MA, United States of America. All experiments used 7–12 week old, sex-matched TfrcY20H/Y20H and WT littermates.

Animal studies

All experimental procedures performed on mice were approved by the Animal Care and Use Committee of Boston Children’s Hospital, Boston, MA, United States of America. All experiments used 7–12 week old, sex-matched TfrcY20H/Y20H and WT littermates.

Generation of TfrcY20H/Y20H knock-in miceA gene-targeting construct was generated to introduce the p.Tyr20His mutation into exon 3 of mouse Tfrc (22). DNA fragments 3.5 kb and 3.3 kb in length were PCR-amplified from the Tfrc gene in C57BL/6 genomic DNA and cloned into a gene-targeting vector upstream and downstream, respectively, of a neomycin resistance gene under the phosphoglycerate kinase 1 promoter, used as a positive selectable marker. The diphtheria toxin A gene fragment, located outside the homology arms, was used as a negative selectable marker. The c.58T>C mutation was introduced into the construct by site-directed mutagenesis, and the mutant clone was sequenced to ensure the absence of off-target PCR-induced mutations. The construct was then linearized and electroporated into C57BL/6N embryonic stem (ES) cells, which were then selected for neomycin resistance using standard techniques24. ES cell clones that had successful integration of the targeting construct, as determined by Southern blot and PCR, were injected into C57BL/6 blastocysts, which were implanted into foster mothers. Offspring were genotyped by PCR and Sanger sequencing. Chimeric mice were backcrossed to generate germline TfrcY20H/+ heterozygous mice, which were then intercrossed to produce TfrcY20H/Y20H homozygous mutant mice. All mice were kept in a pathogen-free environment. Retro-orbital bleeds were performed to obtain peripheral blood for quantification of murine hematologic parameters (Hemavet 950FS) and serum immunoglobulins, which were quantified using standard ELISA techniques.

Generation of TfrcY20H/Y20H knock-in mice

A gene-targeting construct was generated to introduce the p.Tyr20His mutation into exon 3 of mouse Tfrc (22). DNA fragments 3.5 kb and 3.3 kb in length were PCR-amplified from the Tfrc gene in C57BL/6 genomic DNA and cloned into a gene-targeting vector upstream and downstream, respectively, of a neomycin resistance gene under the phosphoglycerate kinase 1 promoter, used as a positive selectable marker. The diphtheria toxin A gene fragment, located outside the homology arms, was used as a negative selectable marker. The c.58T>C mutation was introduced into the construct by site-directed mutagenesis, and the mutant clone was sequenced to ensure the absence of off-target PCR-induced mutations. The construct was then linearized and electroporated into C57BL/6N embryonic stem (ES) cells, which were then selected for neomycin resistance using standard techniques24. ES cell clones that had successful integration of the targeting construct, as determined by Southern blot and PCR, were injected into C57BL/6 blastocysts, which were implanted into foster mothers. Offspring were genotyped by PCR and Sanger sequencing. Chimeric mice were backcrossed to generate germline TfrcY20H/+ heterozygous mice, which were then intercrossed to produce TfrcY20H/Y20H homozygous mutant mice. All mice were kept in a pathogen-free environment. Retro-orbital bleeds were performed to obtain peripheral blood for quantification of murine hematologic parameters (Hemavet 950FS) and serum immunoglobulins, which were quantified using standard ELISA techniques.

Murine cell cultures and reagentsAnti-mouse monoclonal antibodies (mAbs) to the following molecules and the appropriate isotype controls were used for flow cytometry: CD4 (GK1.4), CD8 (53-6.7), CD62L (MEL-14) from BioLegend; CD44 (IM7), CD25 (PC61.5), FOXP3 (FJK-16S), NK1.1 (PK136), CD49b (DX5), Ter119 (TER-119) from eBioscience. Murine CD1d (α-GalCer analog) from the NIH Tetramer Facility-Emory University Vaccine Center, Atlanta, GA. Murine splenic T and B cells were purified by negative selection (Pan-T Cell Isolation Kit II and CD43 Kit respectively, Miltenyi Biotec, Inc.). Purified murine T cells were labeled with CellTrace Violet (Invitrogen) according to the manufacturer’s protocol, followed by stimulation with plate-bound anti-CD3ε (5 μg/ml, 145-2C1) mAb, soluble anti-CD28 (2 μg/ml, 37.51) mAb (both from eBioscience), murine rIL-2 (40 ng/ml, PeproTech), and PMA+IO for 96 hours prior to fluorescence-activated cell sorting (FACS) analysis. T cell proliferation was measured using the proliferative index (PI), calculated as follows: PI=Log[FInd/MFIall]/Log[2], where FInd represents the peak fluorescence intensity of the viable non-divided cells and MFIall represents the median fluorescence intensity of all viable T cells.Purified B cells were stimulated with murine rIL-4 (50 ng/ml, R&D systems) plus either anti-mouse CD40 (1 μg/ml, HM40-3, Pharmingen) or lipopolysaccharide (LPS) (10 μg/ml, Sigma). Proliferation was assayed at day 4 by measuring [3H]-thymidine incorporation, added for the last 16–20 hours of culture. On Day 7 supernatants were collected and assayed for IgE and IgG1 by ELISA. Standard flow cytometric methods were used for staining of cell-surface proteins, and data were collected and analyzed as shown for human cell staining. Fluorochrome-labeled mAbs to the following murine molecules and the appropriate controls were used for staining: TfR1 (R17217) from eBioscience, and CD3 (17A2) and B220 (RA3-6B2) from BioLegend. For TfR1 internalization, resting mutant T cells, purified wild type or mutant T cells stimulated with PMA+IO for 16–18 hours, or bone marrow harvested from murine femurs were used, following the same procedure performed on human EBV-transformed B cells. Erythroid precursors were identified using fluorochrome-labeled mAbs to CD44 (IM7) and Ter119 (Ter-119) from eBioscience. Purified rat anti-mouse TfR1 (R17217) was used for crosslinking and phycoerythrin-conjugated goat anti-rat IgG was used for secondary detection of surface TfR1 expression. Percent TfR1 internalized was calculated as described previously in the Methods. Internalized TFR1 in murine erythroblasts was calculated as follows: MFI at zero minutes – MFI after 30 minutes of TFR1 cross-linking.

Murine cell cultures and reagents

Anti-mouse monoclonal antibodies (mAbs) to the following molecules and the appropriate isotype controls were used for flow cytometry: CD4 (GK1.4), CD8 (53-6.7), CD62L (MEL-14) from BioLegend; CD44 (IM7), CD25 (PC61.5), FOXP3 (FJK-16S), NK1.1 (PK136), CD49b (DX5), Ter119 (TER-119) from eBioscience. Murine CD1d (α-GalCer analog) from the NIH Tetramer Facility-Emory University Vaccine Center, Atlanta, GA. Murine splenic T and B cells were purified by negative selection (Pan-T Cell Isolation Kit II and CD43 Kit respectively, Miltenyi Biotec, Inc.). Purified murine T cells were labeled with CellTrace Violet (Invitrogen) according to the manufacturer’s protocol, followed by stimulation with plate-bound anti-CD3ε (5 μg/ml, 145-2C1) mAb, soluble anti-CD28 (2 μg/ml, 37.51) mAb (both from eBioscience), murine rIL-2 (40 ng/ml, PeproTech), and PMA+IO for 96 hours prior to fluorescence-activated cell sorting (FACS) analysis. T cell proliferation was measured using the proliferative index (PI), calculated as follows: PI=Log[FInd/MFIall]/Log[2], where FInd represents the peak fluorescence intensity of the viable non-divided cells and MFIall represents the median fluorescence intensity of all viable T cells.

Purified B cells were stimulated with murine rIL-4 (50 ng/ml, R&D systems) plus either anti-mouse CD40 (1 μg/ml, HM40-3, Pharmingen) or lipopolysaccharide (LPS) (10 μg/ml, Sigma). Proliferation was assayed at day 4 by measuring [3H]-thymidine incorporation, added for the last 16–20 hours of culture. On Day 7 supernatants were collected and assayed for IgE and IgG1 by ELISA. Standard flow cytometric methods were used for staining of cell-surface proteins, and data were collected and analyzed as shown for human cell staining. Fluorochrome-labeled mAbs to the following murine molecules and the appropriate controls were used for staining: TfR1 (R17217) from eBioscience, and CD3 (17A2) and B220 (RA3-6B2) from BioLegend. For TfR1 internalization, resting mutant T cells, purified wild type or mutant T cells stimulated with PMA+IO for 16–18 hours, or bone marrow harvested from murine femurs were used, following the same procedure performed on human EBV-transformed B cells. Erythroid precursors were identified using fluorochrome-labeled mAbs to CD44 (IM7) and Ter119 (Ter-119) from eBioscience. Purified rat anti-mouse TfR1 (R17217) was used for crosslinking and phycoerythrin-conjugated goat anti-rat IgG was used for secondary detection of surface TfR1 expression. Percent TfR1 internalized was calculated as described previously in the Methods. Internalized TFR1 in murine erythroblasts was calculated as follows: MFI at zero minutes – MFI after 30 minutes of TFR1 cross-linking.

Statistical analysisStudent’s t-test was used to compare groups with equal variances and two-way ANOVA was used for comparing more than two groups. For animal experiments, the sample size was determined using a conservative estimate of a 50% decrease in lymphocyte proliferation, immunoglobulin secretion, TfR1 expression, and TfR1 internalization in TfrcY20H/Y20H cells compared to controls, with a standard deviation of 10%. To detect this difference with 95% power, we would need at least two mice of each genotype for each experiment. All experiments were performed a minimum of two times unless otherwise indicated.

Statistical analysis

Student’s t-test was used to compare groups with equal variances and two-way ANOVA was used for comparing more than two groups. For animal experiments, the sample size was determined using a conservative estimate of a 50% decrease in lymphocyte proliferation, immunoglobulin secretion, TfR1 expression, and TfR1 internalization in TfrcY20H/Y20H cells compared to controls, with a standard deviation of 10%. To detect this difference with 95% power, we would need at least two mice of each genotype for each experiment. All experiments were performed a minimum of two times unless otherwise indicated.

Supplementary Material12

Supplementary Material
