---
pmid: '12242698'
title: 'Highlight: BRCA1 and BRCA2 proteins in breast cancer.'
authors:
- Daniel DC
journal: Microsc Res Tech
year: '2002'
pmcid: PMC7012442
doi: 10.1002/jemt.10178
---

# Highlight: BRCA1 and BRCA2 proteins in breast cancer.
**Authors:** Daniel DC
**Journal:** Microsc Res Tech (2002)
**DOI:** [10.1002/jemt.10178](https://doi.org/10.1002/jemt.10178)
**PMC:** [PMC7012442](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7012442/)

## Abstract

1. Microsc Res Tech. 2002 Oct 1;59(1):68-83. doi: 10.1002/jemt.10178.

Highlight: BRCA1 and BRCA2 proteins in breast cancer.

Daniel DC(1).

Author information:
(1)Mount Sinai School of Medicine, Department of Pathology, New York, New York 
10029, USA. dianne.daniel@mssm.edu

Heterozygous carriers of loss-of-function germline mutations in the BRCA1 or 
BRCA2 breast cancer susceptibility genes have a predisposition to breast and 
ovarian cancer. Multiple functions have been ascribed to the products of these 
genes, linking them to pathways that inhibit progression to neoplasia. Various 
investigators have assigned roles for these tumor suppressor gene products in 
the cell functions of genome repair, transcription, and growth control. There is 
emerging evidence that BRCA1 may participate in ubiquitin E3 ligase activity. 
BRCA1 and BRCA2 have each been implicated in chromatin remodeling dynamics via 
protein partnering. Ubiquitin ligase and chromatin remodeling activities need 
not be mutually exclusive and both may function in DNA repair, transcriptional 
regulation, or cell cycle control. Here we highlight certain recent findings and 
currently unanswered questions regarding BRCA1 and BRCA2 in breast cancer.

Copyright 2002 Wiley-Liss, Inc.

DOI: 10.1002/jemt.10178
PMID: 12242698 [Indexed for MEDLINE]

## Full Text

Competing Interests: The authors have declared that no competing interests exist.

ObjectivesOncology has become more reliant on new testing methods and a greater use of electronic medical records, which provide a plethora of information available to physicians and researchers. However, to take advantage of vital clinical and research data for precision medicine, we must initially make an effort to create an infrastructure for the collection, storage, and utilization of this information with uniquely designed disease-specific registries that could support the collection of a large number of patients.

Oncology has become more reliant on new testing methods and a greater use of electronic medical records, which provide a plethora of information available to physicians and researchers. However, to take advantage of vital clinical and research data for precision medicine, we must initially make an effort to create an infrastructure for the collection, storage, and utilization of this information with uniquely designed disease-specific registries that could support the collection of a large number of patients.

Materials and methodsIn this study, we perform an in-depth analysis of a series of lung adenocarcinoma patients (n = 415) with genomic and clinical data in a recently created thoracic patient registry.

In this study, we perform an in-depth analysis of a series of lung adenocarcinoma patients (n = 415) with genomic and clinical data in a recently created thoracic patient registry.

ResultsOf the 415 patients with lung adenocarcinoma, 59% (n = 245) were female; the median age was 64 (range, 22–92) years with a median OS of 33.29 months (95% CI, 29.77–39.48). The most common actionable alterations were identified in EGFR (n = 177/415 [42.7%]), ALK (n = 28/377 [7.4%]), and BRAF V600E (n = 7/288 [2.4%]). There was also a discernible difference in survival for 222 patients, who had an actionable alteration, with a median OS of 39.8 months as compared to 193 wild-type patients with a median OS of 26.0 months (P<0.001). We identified an unprecedented number of actionable alterations [53.5% (222/415)], including distinct individual alteration rates, as compared with 15.0% and 22.3% in TCGA and GENIE respectively.

Of the 415 patients with lung adenocarcinoma, 59% (n = 245) were female; the median age was 64 (range, 22–92) years with a median OS of 33.29 months (95% CI, 29.77–39.48). The most common actionable alterations were identified in EGFR (n = 177/415 [42.7%]), ALK (n = 28/377 [7.4%]), and BRAF V600E (n = 7/288 [2.4%]). There was also a discernible difference in survival for 222 patients, who had an actionable alteration, with a median OS of 39.8 months as compared to 193 wild-type patients with a median OS of 26.0 months (P<0.001). We identified an unprecedented number of actionable alterations [53.5% (222/415)], including distinct individual alteration rates, as compared with 15.0% and 22.3% in TCGA and GENIE respectively.

ConclusionThe use of patient registries, focused genomic panels and the appropriate use of clinical guidelines in community and academic settings may influence cohort selection for clinical trials and improve survival outcomes.

The use of patient registries, focused genomic panels and the appropriate use of clinical guidelines in community and academic settings may influence cohort selection for clinical trials and improve survival outcomes.

All relevant study data are within the manuscript and its Supporting Information files. Any additional data is available through TCGA (https://www.cancer.gov/tcga) and AACR GENIE (https://www.aacr.org/professionals/research/aacr-project-genie/aacr-project-genie-data/).

IntroductionIn response to advances in genomic testing and the rapid integration of new drugs and publications, oncologists have been adapting the concept of precision medicine where evidence-based medicine guides treatment decisions for individuals [1]. However, more effort is required to translate these benefits into real-world durable survivals for patients. Therefore, in this pursuit, several organizations have implemented the utilization of guidelines and pathways to ensure that patients receive proper testing and are assigned to proper treatment options, which in theory should then translate into durable survival outcomes [2–6]. This spur towards personalized medicine is primarily driven by the advances in genomic testing, biomarker-driven therapy, and immunotherapy that have transformed the landscape of oncology care and have greatly improved outcomes for patients [7–12]. Next-generation sequencing (NGS) has been highly influential in its ability to identify genomic alterations that confer sensitivity to approved and investigational targeted therapies in patients suffering from a variety of advanced stage cancers. The application of molecular testing is transforming cancer into a diverse template of genomic alterations that drive oncogenesis [13].In view of the vast clinical data offered by NGS in non-small cell lung cancer (NSCLC), City of Hope (COH) has established a concise and efficient patient registry (Registry of Hope) for the collection of genomic alterations and outcomes-focused clinical data. Here we present the results of genomic testing performed as part of routine clinical care and correlative analysis of 415 lung adenocarcinoma (LUAD) patients within the Thoracic Oncology Registry (THOR). We hypothesized that the application of broad genomic testing provides not only a comprehensive overview of clinical heterogeneity in lung cancer but may also guide the future of oncology care as more and more precision medicine therapeutics emerge. In this study, we also evaluate the genomic profile of our COH cohort as it compares to national testing results found in GENIE/TCGA databases.

In response to advances in genomic testing and the rapid integration of new drugs and publications, oncologists have been adapting the concept of precision medicine where evidence-based medicine guides treatment decisions for individuals [1]. However, more effort is required to translate these benefits into real-world durable survivals for patients. Therefore, in this pursuit, several organizations have implemented the utilization of guidelines and pathways to ensure that patients receive proper testing and are assigned to proper treatment options, which in theory should then translate into durable survival outcomes [2–6]. This spur towards personalized medicine is primarily driven by the advances in genomic testing, biomarker-driven therapy, and immunotherapy that have transformed the landscape of oncology care and have greatly improved outcomes for patients [7–12]. Next-generation sequencing (NGS) has been highly influential in its ability to identify genomic alterations that confer sensitivity to approved and investigational targeted therapies in patients suffering from a variety of advanced stage cancers. The application of molecular testing is transforming cancer into a diverse template of genomic alterations that drive oncogenesis [13].

In view of the vast clinical data offered by NGS in non-small cell lung cancer (NSCLC), City of Hope (COH) has established a concise and efficient patient registry (Registry of Hope) for the collection of genomic alterations and outcomes-focused clinical data. Here we present the results of genomic testing performed as part of routine clinical care and correlative analysis of 415 lung adenocarcinoma (LUAD) patients within the Thoracic Oncology Registry (THOR). We hypothesized that the application of broad genomic testing provides not only a comprehensive overview of clinical heterogeneity in lung cancer but may also guide the future of oncology care as more and more precision medicine therapeutics emerge. In this study, we also evaluate the genomic profile of our COH cohort as it compares to national testing results found in GENIE/TCGA databases.

Materials and methodsPatientsPatients with advanced LUAD (n = 415), were enrolled in this analysis and evaluated at COH from 2008 to 2016, the data was collected between 2016 and 2018 through retrospective chart review on patients who had LUAD diagnosis and molecular testing performed at the discretion of their primary clinical provider. All 415 patients had metastatic disease, with 89 percent of patients who presented with metastatic disease at the time of initial diagnosis while others later developed metastatic disease. THOR was used to perform this study and data was collected into the registry over time on eligible patients. Different NGS platforms were used as described in S1 Table. Patients were categorized by race/ethnicity according to what they had reported to their oncologist and the data was pulled from the electronic medical record. The categories included African American, American Indian or Alaska Native, Asian, White, Native Hawaiian or Other Pacific Island, Other, and Unknown/Declined to answer. Confounding variables, such as socioeconomic status, were not adjusted but the researchers felt that it was important to understand the diverse demographic makeup of the lung cancer patients and how it relates to their mutational status. The study was approved by the City of Hope institutional review board and in accord with an assurance filed with and approved by the Department of Health and Human Services at COH. This study was approved by the Institutional Review Board at COH under IRB 18217 and was conducted according to the Declaration of Helsinki. Data was de-identified and analyzed anonymously.Genomic analysisPatient genomic alteration data was collected manually from clinical genomic tests alongside the clinical information. The molecular testing results in this study were all performed at around the time of metastatic diagnosis and did not include multiple time points. Clinical actionability of genomic alterations was assessed for each patient and included genomic alterations that had US Food and Drug Administration (FDA) approved targeted therapy in NSCLC. The actionable genomic alterations in this study were defined as EGFR exon 19 deletions and L858R mutation, ALK rearrangement, ROS1 rearrangement, NTRK fusions, and BRAF V600E alterations based on FDA-approved therapy available. MET exon 14 splice site/deletion was also included as actionable based on FDA accelerated path to approval [14, 15]. These alterations were chosen because they are largely exclusive of one another. Any patient who had at least one of these six gene alterations was considered to have an actionable alteration and anyone whose genomic testing results did not identify any of the six gene alterations was considered wild-type. When individual genes were evaluated for survival and various statistical analyses the patients with alterations were only compared with patients who were tested negative for that specific gene. Tile plot maps were generated using seaborn library for Python (version 2.7.14) [16].Comparison with TCGA and GENIEFor the alteration rate of the THOR data set, each gene that had an alteration was calculated as an individual patient divided by the number of patients who were tested for that gene. Genomic alteration data were retrieved from TCGA (study id = luad_tcga_pub) and GENIE (GENIE Cohort v4.1-public) [17, 18]. For each gene, only samples profiled in all molecular profiles including copy-number alteration, somatic mutations, and structural rearrangement were counted. Alteration rate for each gene was calculated by number of samples with at least one alteration divided by the number of profiled samples.Statistical analysisPatient characteristic parameters were evaluated using the χ2 test to test for association between characteristic values and age. OS was calculated from the date of diagnosis of metastatic disease to death or last follow-up visit. Patients who were thought to be alive at the end of the study were censored at the time of the last visit. Survival estimates for the studied patients were generated using the Kaplan-Meier method and the Cox proportional hazards models were used to estimate the hazard ratios. The analysis was performed using survival and survminer packages for R software (version 3.4.4) and SAS version 9.4 (SAS Institute, Cary, NC, United States). The assumption of proportional hazards was tested using goodness-of-fit tests, graphical methods and time-dependent variable methods. The extended Cox models using time-dependent variables were used to adjust the non-proportionality of variables. By adding interaction terms between time and the variables that violated the proportional hazards assumption, the models allow for the possible diverging survival curves over time.

PatientsPatients with advanced LUAD (n = 415), were enrolled in this analysis and evaluated at COH from 2008 to 2016, the data was collected between 2016 and 2018 through retrospective chart review on patients who had LUAD diagnosis and molecular testing performed at the discretion of their primary clinical provider. All 415 patients had metastatic disease, with 89 percent of patients who presented with metastatic disease at the time of initial diagnosis while others later developed metastatic disease. THOR was used to perform this study and data was collected into the registry over time on eligible patients. Different NGS platforms were used as described in S1 Table. Patients were categorized by race/ethnicity according to what they had reported to their oncologist and the data was pulled from the electronic medical record. The categories included African American, American Indian or Alaska Native, Asian, White, Native Hawaiian or Other Pacific Island, Other, and Unknown/Declined to answer. Confounding variables, such as socioeconomic status, were not adjusted but the researchers felt that it was important to understand the diverse demographic makeup of the lung cancer patients and how it relates to their mutational status. The study was approved by the City of Hope institutional review board and in accord with an assurance filed with and approved by the Department of Health and Human Services at COH. This study was approved by the Institutional Review Board at COH under IRB 18217 and was conducted according to the Declaration of Helsinki. Data was de-identified and analyzed anonymously.

Patients with advanced LUAD (n = 415), were enrolled in this analysis and evaluated at COH from 2008 to 2016, the data was collected between 2016 and 2018 through retrospective chart review on patients who had LUAD diagnosis and molecular testing performed at the discretion of their primary clinical provider. All 415 patients had metastatic disease, with 89 percent of patients who presented with metastatic disease at the time of initial diagnosis while others later developed metastatic disease. THOR was used to perform this study and data was collected into the registry over time on eligible patients. Different NGS platforms were used as described in S1 Table. Patients were categorized by race/ethnicity according to what they had reported to their oncologist and the data was pulled from the electronic medical record. The categories included African American, American Indian or Alaska Native, Asian, White, Native Hawaiian or Other Pacific Island, Other, and Unknown/Declined to answer. Confounding variables, such as socioeconomic status, were not adjusted but the researchers felt that it was important to understand the diverse demographic makeup of the lung cancer patients and how it relates to their mutational status. The study was approved by the City of Hope institutional review board and in accord with an assurance filed with and approved by the Department of Health and Human Services at COH. This study was approved by the Institutional Review Board at COH under IRB 18217 and was conducted according to the Declaration of Helsinki. Data was de-identified and analyzed anonymously.

Genomic analysisPatient genomic alteration data was collected manually from clinical genomic tests alongside the clinical information. The molecular testing results in this study were all performed at around the time of metastatic diagnosis and did not include multiple time points. Clinical actionability of genomic alterations was assessed for each patient and included genomic alterations that had US Food and Drug Administration (FDA) approved targeted therapy in NSCLC. The actionable genomic alterations in this study were defined as EGFR exon 19 deletions and L858R mutation, ALK rearrangement, ROS1 rearrangement, NTRK fusions, and BRAF V600E alterations based on FDA-approved therapy available. MET exon 14 splice site/deletion was also included as actionable based on FDA accelerated path to approval [14, 15]. These alterations were chosen because they are largely exclusive of one another. Any patient who had at least one of these six gene alterations was considered to have an actionable alteration and anyone whose genomic testing results did not identify any of the six gene alterations was considered wild-type. When individual genes were evaluated for survival and various statistical analyses the patients with alterations were only compared with patients who were tested negative for that specific gene. Tile plot maps were generated using seaborn library for Python (version 2.7.14) [16].

Patient genomic alteration data was collected manually from clinical genomic tests alongside the clinical information. The molecular testing results in this study were all performed at around the time of metastatic diagnosis and did not include multiple time points. Clinical actionability of genomic alterations was assessed for each patient and included genomic alterations that had US Food and Drug Administration (FDA) approved targeted therapy in NSCLC. The actionable genomic alterations in this study were defined as EGFR exon 19 deletions and L858R mutation, ALK rearrangement, ROS1 rearrangement, NTRK fusions, and BRAF V600E alterations based on FDA-approved therapy available. MET exon 14 splice site/deletion was also included as actionable based on FDA accelerated path to approval [14, 15]. These alterations were chosen because they are largely exclusive of one another. Any patient who had at least one of these six gene alterations was considered to have an actionable alteration and anyone whose genomic testing results did not identify any of the six gene alterations was considered wild-type. When individual genes were evaluated for survival and various statistical analyses the patients with alterations were only compared with patients who were tested negative for that specific gene. Tile plot maps were generated using seaborn library for Python (version 2.7.14) [16].

Comparison with TCGA and GENIEFor the alteration rate of the THOR data set, each gene that had an alteration was calculated as an individual patient divided by the number of patients who were tested for that gene. Genomic alteration data were retrieved from TCGA (study id = luad_tcga_pub) and GENIE (GENIE Cohort v4.1-public) [17, 18]. For each gene, only samples profiled in all molecular profiles including copy-number alteration, somatic mutations, and structural rearrangement were counted. Alteration rate for each gene was calculated by number of samples with at least one alteration divided by the number of profiled samples.

For the alteration rate of the THOR data set, each gene that had an alteration was calculated as an individual patient divided by the number of patients who were tested for that gene. Genomic alteration data were retrieved from TCGA (study id = luad_tcga_pub) and GENIE (GENIE Cohort v4.1-public) [17, 18]. For each gene, only samples profiled in all molecular profiles including copy-number alteration, somatic mutations, and structural rearrangement were counted. Alteration rate for each gene was calculated by number of samples with at least one alteration divided by the number of profiled samples.

Statistical analysisPatient characteristic parameters were evaluated using the χ2 test to test for association between characteristic values and age. OS was calculated from the date of diagnosis of metastatic disease to death or last follow-up visit. Patients who were thought to be alive at the end of the study were censored at the time of the last visit. Survival estimates for the studied patients were generated using the Kaplan-Meier method and the Cox proportional hazards models were used to estimate the hazard ratios. The analysis was performed using survival and survminer packages for R software (version 3.4.4) and SAS version 9.4 (SAS Institute, Cary, NC, United States). The assumption of proportional hazards was tested using goodness-of-fit tests, graphical methods and time-dependent variable methods. The extended Cox models using time-dependent variables were used to adjust the non-proportionality of variables. By adding interaction terms between time and the variables that violated the proportional hazards assumption, the models allow for the possible diverging survival curves over time.

Patient characteristic parameters were evaluated using the χ2 test to test for association between characteristic values and age. OS was calculated from the date of diagnosis of metastatic disease to death or last follow-up visit. Patients who were thought to be alive at the end of the study were censored at the time of the last visit. Survival estimates for the studied patients were generated using the Kaplan-Meier method and the Cox proportional hazards models were used to estimate the hazard ratios. The analysis was performed using survival and survminer packages for R software (version 3.4.4) and SAS version 9.4 (SAS Institute, Cary, NC, United States). The assumption of proportional hazards was tested using goodness-of-fit tests, graphical methods and time-dependent variable methods. The extended Cox models using time-dependent variables were used to adjust the non-proportionality of variables. By adding interaction terms between time and the variables that violated the proportional hazards assumption, the models allow for the possible diverging survival curves over time.

ResultsPatients415 eligible patients with LUAD and tumor genotyping results were identified in THOR at the time of this study. The majority of patients were Stage IV (n = 369/415 [89%]) at the time of initial diagnosis and 46/415 (11%) patients were stage I-III at diagnosis who eventually recurred as metastatic disease. There were 281 patients who underwent broad-based genomic testing (more than 30 genes) using NGS and 134 patients were tested for a few genes in a small panel (less than 10 genes). 254 (61%) patients were never-smokers (n = 212) or had a smoking history of fewer than 10 pack-years (light smokers; n = 42) and 161 (39%) patients were smokers with a history of 10–29 pack-years (medium smokers; n = 74) or > = 30 pack-years (heavy smokers; n = 87). The overall median age at diagnosis was 64 (22–92) and the median OS was 33.29 months (95% CI, 29.77–39.48), with the majority of patients being female (n = 245 [59%]). Detailed patient characteristics noted in Table 1.10.1371/journal.pone.0228188.t001Table 1Patient characteristics.CharacteristicTotal, No. (%)Age, No. (%)P ValueChi-square Statistic<70> = 70Patients415 (100)289 (100)126 (100)NANASex0.3410.906    Female245 (59)175 (61)70 (56)    Male170 (41)114 (39)56 (44)Race0.1279.956    African American10 (2)6 (2)4 (3)    American Indian or Alaska Native2 (0)1 (0)1 (1)    Asian136 (33)107 (37)29 (23)    White247 (60)161 (56)86 (68)    Native Hawaiian or Other Pacific Islander3 (1)3 (1)0 (0)    Other12 (3)8 (3)4 (3)    Unknown/Declined to Answer5 (1)3 (1)2 (2)Ethnicity0.1503.796    Hispanic or Latino40 (10)28 (10)12 (10)    Not Hispanic or Latino369 (89)259 (90)110 (87)    Unknown/Declined to Answer6 (1)2 (1)4 (3)Smoking Status0.0368.550    Never212 (51)159 (55)53 (42)    Light (<10 pack years)42 (10)31 (11)11 (9)    Medium (10–29 pack years)74 (18)44 (15)30 (24)    Heavy (> = 30 pack years)87 (21)55 (19)32 (25)Stage0.00413.595    I15 (4)7 (2)8 (6)    II14 (3)5 (2)9 (7)    III17 (4)10 (3)7 (6)    IV369 (89)267 (92)102 (81)EGFR (L858R/exon 19 deletion) *0.0593.559    Alteration177 (43)132 (46)45 (36)    Tested Negative238 (57)157 (54)81 (64)ALK (Rearrangement)*0.00110.349    Positive28 (7)27 (10)1 (1)    Tested Negative349 (93)235 (90)114 (99)ROS1 (Rearrangement)*0.2331.422    Positive3 (1)3 (2)0 (0)    Tested Negative254 (99)172 (98)82 (100)BRAF (V600E)*0.3190.995    Positive7 (2)6 (3)1 (1)    Tested Negative281 (98)194 (97)90 (99)MET (exon 14 splice-site/deletion)*0.0225.228    Positive7 (2)2 (1)5 (5)    Tested Negative280 (98)194 (99)86 (95)KRAS   <0.00112.410    Positive97 (28)53 (22)44 (40)    Tested Negative255 (72)189 (78)66 (60)TP53   0.3061.046    Positive140 (49)99 (52)41 (45)    Tested Negative143 (51)93 (48)50 (55)*Only patients who had genomic test results were counted for each gene. Total number of patients with ALK, ROS1, BRAF, MET, KRAS and TP53 tested were 377, 257, 288, 287, 352, and 283 respectively.Genomic alterationsThere were 323 different genes with evidence of genomic alteration in this group of patients. The most commonly occurring alterations in oncogenes were found in EGFR (n = 207/415 [50%]), KRAS (n = 97/352 [28%]), and ALK rearrangement (n = 28/377 [7%]), while the most commonly occurring tumor suppressor genes consisted of TP53 (n = 140/283 [49%]), LRP1B (n = 63/228 [28%]), and STK11 (n = 39/278 [14%]) (Fig 1A). The median number of genes altered in patients who underwent broad-based sequencing was 10. The most common actionable alterations were identified in EGFR L858R/exon 19 deletion (n = 177/415 [42.7%]), ALK rearrangement (n = 28/377 [7.4%]), ROS1 rearrangement (n = 3/257 [1.2%]), BRAF V600E (n = 7/288 [2.4%]), and MET exon 14 splice site/deletion (n = 7/287 [2.4%]). No NTRK fusion alterations were identified in our cohort. 86% (177/207) of patients with EGFR alterations had an actionable alteration while 14% (29/207) consisted of exon 18 mutations, exon 20 insertions and other EGFR alterations including amplifications or substitution variants of unknown significance. EGFR alterations were split into five subtypes with the majority of Asians (n = 51/99 [52%]) and whites (n = 46/95 [48%]) presenting with an exon 19 deletion. KRAS alterations predominated in whites (n = 77/97 [79%]) and patients with history of medium (n = 24/97 [25%]) or heavy smoking (n = 47/97 [48%]) (Table 2).10.1371/journal.pone.0228188.g001Fig 1Landscape overview of molecular profiling in 415 THOR lung adenocarcinoma patients.A) Tile plot of top 25 most prevalent genes and their demographic parameters sorted by genomic alteration rate and subtypes, including amplification, complex (more than one type of alteration), frameshift, indel (insertion/deletion), loss, missense, nonsense, rearrangement, and splice site. For each gene, the alteration rate was calculated by number of patients with alteration divided by the number of patients tested for this gene. B) Heatmap showing a comparison of the genomic alteration rates between THOR, TCGA, GENIE, and GENIE Metastatic for genes in 1A.10.1371/journal.pone.0228188.t002Table 2Patients characteristic table for EGFR and KRAS alteration subtypes.CharacteristicEGFR, No. (%)*KRAS, No. (%)Totalexon 18 mutationexon 19 deletionexon 20 insertionexon 21 mutationOthersTotalG12G13Q61OthersPatients207 (100)9 (4)106 (51)16 (8)72 (35)7 (3)97 (100)79 (81)7 (7)9 (9)2 (2)Sex    Female130 (63)4 (44)65 (61)12 (75)47 (65)3 (43)49 (51)39 (49)5 (71)4 (44)1 (50)    Male77 (37)5 (56)41 (39)4 (25)25 (35)4 (57)48 (49)40 (51)2 (29)5 (56)1 (50)Age    <70155 (75)5 (56)84 (79)15 (94)48 (67)6 (86)53 (55)45 (57)3 (43)4 (44)1 (50)    > = 7052 (25)4 (44)22 (21)1 (6)24 (33)1 (14)44 (45)34 (43)4 (57)5 (56)1 (50)Race    African American4 (2)0 (0)3 (3)0 (0)1 (1)0 (0)4 (4)3 (4)0 (0)1 (11)0 (0)    American Indian or Alaska Native0 (0)0 (0)0 (0)0 (0)0 (0)0 (0)2 (2)2 (3)0 (0)0 (0)0 (0)    Asian99 (48)6 (67)51 (48)6 (38)37 (51)2 (29)11 (11)9 (11)1 (14)0 (0)1 (50)    White95 (46)3 (33)46 (43)10 (63)32 (44)4 (57)77 (79)62 (78)6 (86)8 (89)1 (50)    Native Hawaiian or Other Pacific Islander0 (0)0 (0)0 (0)0 (0)0 (0)0 (0)1 (1)1 (1)0 (0)0 (0)0 (0)    Other8 (4)0 (0)6 (6)0 (0)1 (1)1 (14)1 (1)1 (1)0 (0)0 (0)0 (0)    Unknown/Declined to Answer1 (0)0 (0)0 (0)0 (0)1 (1)0 (0)1 (1)1 (1)0 (0)0 (0)0 (0)Ethnicity    Hispanic or Latino17 (8)0 (0)13 (12)1 (6)2 (3)1 (14)11 (11)10 (13)0 (0)1 (11)0 (0)    Not Hispanic or Latino188 (91)9 (100)92 (87)15 (94)69 (96)6 (86)84 (87)68 (86)6 (86)8 (89)2 (100)    Unknown/Declined to Answer2 (1)0 (0)1 (1)0 (0)1 (1)0 (0)2 (2)1 (1)1 (14)0 (0)0 (0)Smoking Status    Never137 (66)5 (56)72 (68)9 (56)50 (69)3 (43)17 (18)14 (18)1 (14)1 (11)1 (50)    Light (<10 pack years)23 (11)1 (11)11 (10)3 (19)7 (10)2 (29)9 (9)7 (9)1 (14)1 (11)0 (0)    Medium (10–29 pack years)39 (19)3 (33)21 (20)3 (19)11 (15)1 (14)24 (25)19 (24)1 (14)3 (33)1 (50)    Heavy (> = 30 pack years)8 (4)0 (0)2 (2)1 (6)4 (6)1 (14)47 (48)39 (49)4 (57)4 (44)0 (0)*Three patients carried two subtypes of EGFR alteration (e.g., exon 19 deletion and exon 18 mutation) and were counted two times.Comparison with TCGA and GENIEWhile the gene alteration rates of LUAD patients from TCGA/GENIE were comparable [17, 18], evident differences were observed between the patients of THOR and these two public consortiums (Fig 1B). Importantly, a large proportion of patients had actionable alterations [53.5% (222/415)] with the highest rate of actionable EGFR mutations at 42.7% (177/415), as compared with 9.1% and 17.1% actionable rate in TCGA/GENIE respectively. In particular, our dataset had a significantly increased prevalence of EGFR (49.9%), BRCA2 (14.1%), and chromatin modifying genes (ARID1B [18.8%], ARID1A [17.7%], MLL2 [13.6%] and MLL [13.2%]) as compared to the other datasets (Table 3). Analysis of subgroups of patients in GENIE that were diagnosed with or without metastasis showed similar gene alteration rates of these genes, indicating the difference we observed between THOR and TCGA/GENIE patients was not directly linked to metastasis (Table 3).10.1371/journal.pone.0228188.t003Table 3Genomic alteration rates of THOR referring to TCGA/GENIE/GENIE metastasis patients.GeneAlteredTestedTHOR Alteration Rate (n = 415)TCGA Alternation Rate (n = 507)GENIE Alternation Rate (n = 6529)GENIE Metastasis Alternation Rate (n = 2697)EGFR20741550%16%25%25%TP5314028350%52%47%55%LRP1B6322828%37%22%23%KRAS9735228%33%34%30%ARID1B3619119%7%5%6%MLL33519018%15%6%7%SPTA13519018%31%10%13%ARID1A4123218%7%8%9%FAT12919115%12%9%10%BRCA23323414%6%5%6%STK113927814%16%14%15%MLL23122814%8%9%10%MLL3022813%7%5%5%GPR1242922813%7%5%6%ATM3527613%10%8%9%KEAP12722812%20%15%17%MET3328712%7%5%7%NKX2-12622811%14%8%9%ALK4237711%7%5%6%SETD22522811%7%7%7%NFKBIA2422811%12%4%6%SMARCA42422811%10%9%11%APC2827410%6%5%6%SDHA1919110%14%4%6%NOTCH12727610%5%3%4%Genomic alteration data were collected for 415 THOR lung adenocarcinoma patients, 507 TCGA Lung Adenocarcinoma patients (study id = luad_tcga_pan_can_atlas_2018), 6529 GENIE Lung Adenocarcinoma patients (GENIE Cohort v5.0-public, cancer type detailed = lung adenocarcinoma), and 2697 GENIE metastasis lung adenocarcinoma patients (GENIE Cohort v5.0-public, cancer type detailed = lung adenocarcinoma, sample type = Metastasis).SurvivalThe median OS of all patients was 33.3 months (95% CI; 29.8–39.5) and female patients had a better median OS of 39.8 months (95% CI, 33.5–45.5 months; HR, 1.56; 95% CI, 1.21–2.00; Table 4) as compared to median OS of 27.4 months for male patients. The OS was also better for never/light smokers (<10 pack years) with a median OS of 39.5 months (95% CI, 32.9–45.5 months; HR, 1.42; 95% CI, 1.10–1.83; Table 4) as compared to medium/heavy smokers (> = 10 pack years) who had a median OS of 25.7 months (95% CI, 22.1–33.3 months). Two genes had violations of the proportional hazards assumption EGFR in actionable and KRAS in actionable, therefore, we added an interaction with time to account for changes in the HR. When comparing the median OS of KRAS positive patient (25.5 months; 95% CI, 17.6–32.9 months) with the median OS of actionable patient who had tested KRAS negative (41.1 months; 95% CI, 39.3–55.0 months), the difference was noticeable (Fig 2A), the HR changes from high risk of death in the KRAS positive patients initially to less risk over time (Table 4). Of the six actionable genes only ALK rearrangement patients had a difference in survival with a median OS of 82.6 months (95% CI, 82.6-NR; HR, 0.35; 95% CI, 0.17–0.68;) as compared to 26.6 months (95% CI, 22.1–33.1 months) for tested ALK rearrangement negative patients (Fig 2B). The genomic testing panel size had no discernible OS difference and broad-based sequencing had a median OS of 33.4 months (95% CI, 29.2–44.2 months) as compared to 33.5 months (95% CI, 27.5–41.2 months) for the small panel (Fig 2C). There was a discernible difference in survival for 222 patients, who had an actionable alteration (such as EGFR L858R/exon 19 deletion, ALK rearrangements, ROS1 rearrangements, BRAF V600E, NTRK fusions and MET exon 14 splice site/deletion), with a median OS of 39.8 months as compared to 193 patients who were wild-type with a median OS of 26.0 months (Fig 2D).10.1371/journal.pone.0228188.g002Fig 2Kaplan-Meier estimates of overall survival.A) Overall survival among KRAS altered and tested KRAS negative patients with actionable alteration. B) Overall survival among ALK rearrangement and tested ALK rearrangement negative patients. C) Overall survival among broad-based panel tested and small-panel tested patients. D) Overall survival among actionable altered and wild-type patients. P values comparing risk groups were calculated with the log-rank test.10.1371/journal.pone.0228188.t004Table 4Cox proportional hazard regression models for univariate survival.Risk FactorMedian Survival (95% CI), monthsHazard Ratio (95% CI)P ValueSex, Male vs Female27.4 (20.6–31.7) vs 39.8 (33.5–45.5)1.56 (1.21–2.00)<0.001Age, > = 70 vs <7029.0 (19.5–36.5) vs 36.3 (31.4–42.9)1.41 (1.08–1.85)0.012Smoking Status, Medium + Heavy vs Never + Light25.7 (22.1–33.3) vs 39.5 (32.9–45.5)1.42 (1.10–1.83)0.006EGFR (Actionable)ab    Positive vs NegativeEGFR x time39 (32.6–43.8) vs 26 (22.1–32.9)0.46 (0.31–0.68)1.02 (1.01–1.03)0.00010.0022ALK (Rearrangement)b    Positive vs Negative82.6 (82.6-NR) vs 26.6 (22.1–33.1)0.35 (0.17–0.68)0.002ROS1 (Rearrangement)b    Positive vs NegativeNR vs 26 (20.8–33.3)--BRAF (V600E)b    Positive vs Negative73.4 (11.6-NR) vs 28.0 (22.2–34.4)0.70 (0.26–1.91)0.487MET (exon 14 Splice-site/Deletion)b    Positive vs Negative17.2 (8.77-NR) vs 28.0 (22.2–34.4)0.59 (0.19–1.89)0.377KRAS in Actionablec    Positive vs NegativeKRAS x time25.5 (17.6–32.9) vs 41.4 (39.3–55.0)2.80 (1.70–4.61)0.98 (0.97–0.99)<0.00010.0065KRAS in Non-Actionabled    Positive vs Negative25.5 (17.6–32.9) vs 27.7 (22.2–42.9)1.06 (0.73–1.53)0.769TP53 in Actionablec    Positive vs Negative25.5 (20.6–34.9) vs 67.6 (39.5-NR)1.87 (1.12–3.12)0.017TP53 in Non-Actionabled    Positive vs Negative25.5 (20.6–34.9) vs 30.1 (21.6–54.0)1.13 (0.75–1.69)0.561a. EGFR All patients had EGFR tested.b. For each gene, patients carrying actionable alterations were compared with patients who were tested negative, with patients carrying actionable alterations from other genes excluded from its analysis.c. For KRAS and TP53, patients who were tested positive with no actionable alteration were compared with patients who were tested negative but had actionable gene alterations.d. For KRAS and TP53, patients who were tested positive were compared with patients who were tested negative and patients who had actionable gene alterations were excluded from both groups.

Patients415 eligible patients with LUAD and tumor genotyping results were identified in THOR at the time of this study. The majority of patients were Stage IV (n = 369/415 [89%]) at the time of initial diagnosis and 46/415 (11%) patients were stage I-III at diagnosis who eventually recurred as metastatic disease. There were 281 patients who underwent broad-based genomic testing (more than 30 genes) using NGS and 134 patients were tested for a few genes in a small panel (less than 10 genes). 254 (61%) patients were never-smokers (n = 212) or had a smoking history of fewer than 10 pack-years (light smokers; n = 42) and 161 (39%) patients were smokers with a history of 10–29 pack-years (medium smokers; n = 74) or > = 30 pack-years (heavy smokers; n = 87). The overall median age at diagnosis was 64 (22–92) and the median OS was 33.29 months (95% CI, 29.77–39.48), with the majority of patients being female (n = 245 [59%]). Detailed patient characteristics noted in Table 1.10.1371/journal.pone.0228188.t001Table 1Patient characteristics.CharacteristicTotal, No. (%)Age, No. (%)P ValueChi-square Statistic<70> = 70Patients415 (100)289 (100)126 (100)NANASex0.3410.906    Female245 (59)175 (61)70 (56)    Male170 (41)114 (39)56 (44)Race0.1279.956    African American10 (2)6 (2)4 (3)    American Indian or Alaska Native2 (0)1 (0)1 (1)    Asian136 (33)107 (37)29 (23)    White247 (60)161 (56)86 (68)    Native Hawaiian or Other Pacific Islander3 (1)3 (1)0 (0)    Other12 (3)8 (3)4 (3)    Unknown/Declined to Answer5 (1)3 (1)2 (2)Ethnicity0.1503.796    Hispanic or Latino40 (10)28 (10)12 (10)    Not Hispanic or Latino369 (89)259 (90)110 (87)    Unknown/Declined to Answer6 (1)2 (1)4 (3)Smoking Status0.0368.550    Never212 (51)159 (55)53 (42)    Light (<10 pack years)42 (10)31 (11)11 (9)    Medium (10–29 pack years)74 (18)44 (15)30 (24)    Heavy (> = 30 pack years)87 (21)55 (19)32 (25)Stage0.00413.595    I15 (4)7 (2)8 (6)    II14 (3)5 (2)9 (7)    III17 (4)10 (3)7 (6)    IV369 (89)267 (92)102 (81)EGFR (L858R/exon 19 deletion) *0.0593.559    Alteration177 (43)132 (46)45 (36)    Tested Negative238 (57)157 (54)81 (64)ALK (Rearrangement)*0.00110.349    Positive28 (7)27 (10)1 (1)    Tested Negative349 (93)235 (90)114 (99)ROS1 (Rearrangement)*0.2331.422    Positive3 (1)3 (2)0 (0)    Tested Negative254 (99)172 (98)82 (100)BRAF (V600E)*0.3190.995    Positive7 (2)6 (3)1 (1)    Tested Negative281 (98)194 (97)90 (99)MET (exon 14 splice-site/deletion)*0.0225.228    Positive7 (2)2 (1)5 (5)    Tested Negative280 (98)194 (99)86 (95)KRAS   <0.00112.410    Positive97 (28)53 (22)44 (40)    Tested Negative255 (72)189 (78)66 (60)TP53   0.3061.046    Positive140 (49)99 (52)41 (45)    Tested Negative143 (51)93 (48)50 (55)*Only patients who had genomic test results were counted for each gene. Total number of patients with ALK, ROS1, BRAF, MET, KRAS and TP53 tested were 377, 257, 288, 287, 352, and 283 respectively.

415 eligible patients with LUAD and tumor genotyping results were identified in THOR at the time of this study. The majority of patients were Stage IV (n = 369/415 [89%]) at the time of initial diagnosis and 46/415 (11%) patients were stage I-III at diagnosis who eventually recurred as metastatic disease. There were 281 patients who underwent broad-based genomic testing (more than 30 genes) using NGS and 134 patients were tested for a few genes in a small panel (less than 10 genes). 254 (61%) patients were never-smokers (n = 212) or had a smoking history of fewer than 10 pack-years (light smokers; n = 42) and 161 (39%) patients were smokers with a history of 10–29 pack-years (medium smokers; n = 74) or > = 30 pack-years (heavy smokers; n = 87). The overall median age at diagnosis was 64 (22–92) and the median OS was 33.29 months (95% CI, 29.77–39.48), with the majority of patients being female (n = 245 [59%]). Detailed patient characteristics noted in Table 1.

*Only patients who had genomic test results were counted for each gene. Total number of patients with ALK, ROS1, BRAF, MET, KRAS and TP53 tested were 377, 257, 288, 287, 352, and 283 respectively.

Genomic alterationsThere were 323 different genes with evidence of genomic alteration in this group of patients. The most commonly occurring alterations in oncogenes were found in EGFR (n = 207/415 [50%]), KRAS (n = 97/352 [28%]), and ALK rearrangement (n = 28/377 [7%]), while the most commonly occurring tumor suppressor genes consisted of TP53 (n = 140/283 [49%]), LRP1B (n = 63/228 [28%]), and STK11 (n = 39/278 [14%]) (Fig 1A). The median number of genes altered in patients who underwent broad-based sequencing was 10. The most common actionable alterations were identified in EGFR L858R/exon 19 deletion (n = 177/415 [42.7%]), ALK rearrangement (n = 28/377 [7.4%]), ROS1 rearrangement (n = 3/257 [1.2%]), BRAF V600E (n = 7/288 [2.4%]), and MET exon 14 splice site/deletion (n = 7/287 [2.4%]). No NTRK fusion alterations were identified in our cohort. 86% (177/207) of patients with EGFR alterations had an actionable alteration while 14% (29/207) consisted of exon 18 mutations, exon 20 insertions and other EGFR alterations including amplifications or substitution variants of unknown significance. EGFR alterations were split into five subtypes with the majority of Asians (n = 51/99 [52%]) and whites (n = 46/95 [48%]) presenting with an exon 19 deletion. KRAS alterations predominated in whites (n = 77/97 [79%]) and patients with history of medium (n = 24/97 [25%]) or heavy smoking (n = 47/97 [48%]) (Table 2).10.1371/journal.pone.0228188.g001Fig 1Landscape overview of molecular profiling in 415 THOR lung adenocarcinoma patients.A) Tile plot of top 25 most prevalent genes and their demographic parameters sorted by genomic alteration rate and subtypes, including amplification, complex (more than one type of alteration), frameshift, indel (insertion/deletion), loss, missense, nonsense, rearrangement, and splice site. For each gene, the alteration rate was calculated by number of patients with alteration divided by the number of patients tested for this gene. B) Heatmap showing a comparison of the genomic alteration rates between THOR, TCGA, GENIE, and GENIE Metastatic for genes in 1A.10.1371/journal.pone.0228188.t002Table 2Patients characteristic table for EGFR and KRAS alteration subtypes.CharacteristicEGFR, No. (%)*KRAS, No. (%)Totalexon 18 mutationexon 19 deletionexon 20 insertionexon 21 mutationOthersTotalG12G13Q61OthersPatients207 (100)9 (4)106 (51)16 (8)72 (35)7 (3)97 (100)79 (81)7 (7)9 (9)2 (2)Sex    Female130 (63)4 (44)65 (61)12 (75)47 (65)3 (43)49 (51)39 (49)5 (71)4 (44)1 (50)    Male77 (37)5 (56)41 (39)4 (25)25 (35)4 (57)48 (49)40 (51)2 (29)5 (56)1 (50)Age    <70155 (75)5 (56)84 (79)15 (94)48 (67)6 (86)53 (55)45 (57)3 (43)4 (44)1 (50)    > = 7052 (25)4 (44)22 (21)1 (6)24 (33)1 (14)44 (45)34 (43)4 (57)5 (56)1 (50)Race    African American4 (2)0 (0)3 (3)0 (0)1 (1)0 (0)4 (4)3 (4)0 (0)1 (11)0 (0)    American Indian or Alaska Native0 (0)0 (0)0 (0)0 (0)0 (0)0 (0)2 (2)2 (3)0 (0)0 (0)0 (0)    Asian99 (48)6 (67)51 (48)6 (38)37 (51)2 (29)11 (11)9 (11)1 (14)0 (0)1 (50)    White95 (46)3 (33)46 (43)10 (63)32 (44)4 (57)77 (79)62 (78)6 (86)8 (89)1 (50)    Native Hawaiian or Other Pacific Islander0 (0)0 (0)0 (0)0 (0)0 (0)0 (0)1 (1)1 (1)0 (0)0 (0)0 (0)    Other8 (4)0 (0)6 (6)0 (0)1 (1)1 (14)1 (1)1 (1)0 (0)0 (0)0 (0)    Unknown/Declined to Answer1 (0)0 (0)0 (0)0 (0)1 (1)0 (0)1 (1)1 (1)0 (0)0 (0)0 (0)Ethnicity    Hispanic or Latino17 (8)0 (0)13 (12)1 (6)2 (3)1 (14)11 (11)10 (13)0 (0)1 (11)0 (0)    Not Hispanic or Latino188 (91)9 (100)92 (87)15 (94)69 (96)6 (86)84 (87)68 (86)6 (86)8 (89)2 (100)    Unknown/Declined to Answer2 (1)0 (0)1 (1)0 (0)1 (1)0 (0)2 (2)1 (1)1 (14)0 (0)0 (0)Smoking Status    Never137 (66)5 (56)72 (68)9 (56)50 (69)3 (43)17 (18)14 (18)1 (14)1 (11)1 (50)    Light (<10 pack years)23 (11)1 (11)11 (10)3 (19)7 (10)2 (29)9 (9)7 (9)1 (14)1 (11)0 (0)    Medium (10–29 pack years)39 (19)3 (33)21 (20)3 (19)11 (15)1 (14)24 (25)19 (24)1 (14)3 (33)1 (50)    Heavy (> = 30 pack years)8 (4)0 (0)2 (2)1 (6)4 (6)1 (14)47 (48)39 (49)4 (57)4 (44)0 (0)*Three patients carried two subtypes of EGFR alteration (e.g., exon 19 deletion and exon 18 mutation) and were counted two times.

There were 323 different genes with evidence of genomic alteration in this group of patients. The most commonly occurring alterations in oncogenes were found in EGFR (n = 207/415 [50%]), KRAS (n = 97/352 [28%]), and ALK rearrangement (n = 28/377 [7%]), while the most commonly occurring tumor suppressor genes consisted of TP53 (n = 140/283 [49%]), LRP1B (n = 63/228 [28%]), and STK11 (n = 39/278 [14%]) (Fig 1A). The median number of genes altered in patients who underwent broad-based sequencing was 10. The most common actionable alterations were identified in EGFR L858R/exon 19 deletion (n = 177/415 [42.7%]), ALK rearrangement (n = 28/377 [7.4%]), ROS1 rearrangement (n = 3/257 [1.2%]), BRAF V600E (n = 7/288 [2.4%]), and MET exon 14 splice site/deletion (n = 7/287 [2.4%]). No NTRK fusion alterations were identified in our cohort. 86% (177/207) of patients with EGFR alterations had an actionable alteration while 14% (29/207) consisted of exon 18 mutations, exon 20 insertions and other EGFR alterations including amplifications or substitution variants of unknown significance. EGFR alterations were split into five subtypes with the majority of Asians (n = 51/99 [52%]) and whites (n = 46/95 [48%]) presenting with an exon 19 deletion. KRAS alterations predominated in whites (n = 77/97 [79%]) and patients with history of medium (n = 24/97 [25%]) or heavy smoking (n = 47/97 [48%]) (Table 2).

A) Tile plot of top 25 most prevalent genes and their demographic parameters sorted by genomic alteration rate and subtypes, including amplification, complex (more than one type of alteration), frameshift, indel (insertion/deletion), loss, missense, nonsense, rearrangement, and splice site. For each gene, the alteration rate was calculated by number of patients with alteration divided by the number of patients tested for this gene. B) Heatmap showing a comparison of the genomic alteration rates between THOR, TCGA, GENIE, and GENIE Metastatic for genes in 1A.

*Three patients carried two subtypes of EGFR alteration (e.g., exon 19 deletion and exon 18 mutation) and were counted two times.

Comparison with TCGA and GENIEWhile the gene alteration rates of LUAD patients from TCGA/GENIE were comparable [17, 18], evident differences were observed between the patients of THOR and these two public consortiums (Fig 1B). Importantly, a large proportion of patients had actionable alterations [53.5% (222/415)] with the highest rate of actionable EGFR mutations at 42.7% (177/415), as compared with 9.1% and 17.1% actionable rate in TCGA/GENIE respectively. In particular, our dataset had a significantly increased prevalence of EGFR (49.9%), BRCA2 (14.1%), and chromatin modifying genes (ARID1B [18.8%], ARID1A [17.7%], MLL2 [13.6%] and MLL [13.2%]) as compared to the other datasets (Table 3). Analysis of subgroups of patients in GENIE that were diagnosed with or without metastasis showed similar gene alteration rates of these genes, indicating the difference we observed between THOR and TCGA/GENIE patients was not directly linked to metastasis (Table 3).10.1371/journal.pone.0228188.t003Table 3Genomic alteration rates of THOR referring to TCGA/GENIE/GENIE metastasis patients.GeneAlteredTestedTHOR Alteration Rate (n = 415)TCGA Alternation Rate (n = 507)GENIE Alternation Rate (n = 6529)GENIE Metastasis Alternation Rate (n = 2697)EGFR20741550%16%25%25%TP5314028350%52%47%55%LRP1B6322828%37%22%23%KRAS9735228%33%34%30%ARID1B3619119%7%5%6%MLL33519018%15%6%7%SPTA13519018%31%10%13%ARID1A4123218%7%8%9%FAT12919115%12%9%10%BRCA23323414%6%5%6%STK113927814%16%14%15%MLL23122814%8%9%10%MLL3022813%7%5%5%GPR1242922813%7%5%6%ATM3527613%10%8%9%KEAP12722812%20%15%17%MET3328712%7%5%7%NKX2-12622811%14%8%9%ALK4237711%7%5%6%SETD22522811%7%7%7%NFKBIA2422811%12%4%6%SMARCA42422811%10%9%11%APC2827410%6%5%6%SDHA1919110%14%4%6%NOTCH12727610%5%3%4%Genomic alteration data were collected for 415 THOR lung adenocarcinoma patients, 507 TCGA Lung Adenocarcinoma patients (study id = luad_tcga_pan_can_atlas_2018), 6529 GENIE Lung Adenocarcinoma patients (GENIE Cohort v5.0-public, cancer type detailed = lung adenocarcinoma), and 2697 GENIE metastasis lung adenocarcinoma patients (GENIE Cohort v5.0-public, cancer type detailed = lung adenocarcinoma, sample type = Metastasis).

While the gene alteration rates of LUAD patients from TCGA/GENIE were comparable [17, 18], evident differences were observed between the patients of THOR and these two public consortiums (Fig 1B). Importantly, a large proportion of patients had actionable alterations [53.5% (222/415)] with the highest rate of actionable EGFR mutations at 42.7% (177/415), as compared with 9.1% and 17.1% actionable rate in TCGA/GENIE respectively. In particular, our dataset had a significantly increased prevalence of EGFR (49.9%), BRCA2 (14.1%), and chromatin modifying genes (ARID1B [18.8%], ARID1A [17.7%], MLL2 [13.6%] and MLL [13.2%]) as compared to the other datasets (Table 3). Analysis of subgroups of patients in GENIE that were diagnosed with or without metastasis showed similar gene alteration rates of these genes, indicating the difference we observed between THOR and TCGA/GENIE patients was not directly linked to metastasis (Table 3).

Genomic alteration data were collected for 415 THOR lung adenocarcinoma patients, 507 TCGA Lung Adenocarcinoma patients (study id = luad_tcga_pan_can_atlas_2018), 6529 GENIE Lung Adenocarcinoma patients (GENIE Cohort v5.0-public, cancer type detailed = lung adenocarcinoma), and 2697 GENIE metastasis lung adenocarcinoma patients (GENIE Cohort v5.0-public, cancer type detailed = lung adenocarcinoma, sample type = Metastasis).

SurvivalThe median OS of all patients was 33.3 months (95% CI; 29.8–39.5) and female patients had a better median OS of 39.8 months (95% CI, 33.5–45.5 months; HR, 1.56; 95% CI, 1.21–2.00; Table 4) as compared to median OS of 27.4 months for male patients. The OS was also better for never/light smokers (<10 pack years) with a median OS of 39.5 months (95% CI, 32.9–45.5 months; HR, 1.42; 95% CI, 1.10–1.83; Table 4) as compared to medium/heavy smokers (> = 10 pack years) who had a median OS of 25.7 months (95% CI, 22.1–33.3 months). Two genes had violations of the proportional hazards assumption EGFR in actionable and KRAS in actionable, therefore, we added an interaction with time to account for changes in the HR. When comparing the median OS of KRAS positive patient (25.5 months; 95% CI, 17.6–32.9 months) with the median OS of actionable patient who had tested KRAS negative (41.1 months; 95% CI, 39.3–55.0 months), the difference was noticeable (Fig 2A), the HR changes from high risk of death in the KRAS positive patients initially to less risk over time (Table 4). Of the six actionable genes only ALK rearrangement patients had a difference in survival with a median OS of 82.6 months (95% CI, 82.6-NR; HR, 0.35; 95% CI, 0.17–0.68;) as compared to 26.6 months (95% CI, 22.1–33.1 months) for tested ALK rearrangement negative patients (Fig 2B). The genomic testing panel size had no discernible OS difference and broad-based sequencing had a median OS of 33.4 months (95% CI, 29.2–44.2 months) as compared to 33.5 months (95% CI, 27.5–41.2 months) for the small panel (Fig 2C). There was a discernible difference in survival for 222 patients, who had an actionable alteration (such as EGFR L858R/exon 19 deletion, ALK rearrangements, ROS1 rearrangements, BRAF V600E, NTRK fusions and MET exon 14 splice site/deletion), with a median OS of 39.8 months as compared to 193 patients who were wild-type with a median OS of 26.0 months (Fig 2D).10.1371/journal.pone.0228188.g002Fig 2Kaplan-Meier estimates of overall survival.A) Overall survival among KRAS altered and tested KRAS negative patients with actionable alteration. B) Overall survival among ALK rearrangement and tested ALK rearrangement negative patients. C) Overall survival among broad-based panel tested and small-panel tested patients. D) Overall survival among actionable altered and wild-type patients. P values comparing risk groups were calculated with the log-rank test.10.1371/journal.pone.0228188.t004Table 4Cox proportional hazard regression models for univariate survival.Risk FactorMedian Survival (95% CI), monthsHazard Ratio (95% CI)P ValueSex, Male vs Female27.4 (20.6–31.7) vs 39.8 (33.5–45.5)1.56 (1.21–2.00)<0.001Age, > = 70 vs <7029.0 (19.5–36.5) vs 36.3 (31.4–42.9)1.41 (1.08–1.85)0.012Smoking Status, Medium + Heavy vs Never + Light25.7 (22.1–33.3) vs 39.5 (32.9–45.5)1.42 (1.10–1.83)0.006EGFR (Actionable)ab    Positive vs NegativeEGFR x time39 (32.6–43.8) vs 26 (22.1–32.9)0.46 (0.31–0.68)1.02 (1.01–1.03)0.00010.0022ALK (Rearrangement)b    Positive vs Negative82.6 (82.6-NR) vs 26.6 (22.1–33.1)0.35 (0.17–0.68)0.002ROS1 (Rearrangement)b    Positive vs NegativeNR vs 26 (20.8–33.3)--BRAF (V600E)b    Positive vs Negative73.4 (11.6-NR) vs 28.0 (22.2–34.4)0.70 (0.26–1.91)0.487MET (exon 14 Splice-site/Deletion)b    Positive vs Negative17.2 (8.77-NR) vs 28.0 (22.2–34.4)0.59 (0.19–1.89)0.377KRAS in Actionablec    Positive vs NegativeKRAS x time25.5 (17.6–32.9) vs 41.4 (39.3–55.0)2.80 (1.70–4.61)0.98 (0.97–0.99)<0.00010.0065KRAS in Non-Actionabled    Positive vs Negative25.5 (17.6–32.9) vs 27.7 (22.2–42.9)1.06 (0.73–1.53)0.769TP53 in Actionablec    Positive vs Negative25.5 (20.6–34.9) vs 67.6 (39.5-NR)1.87 (1.12–3.12)0.017TP53 in Non-Actionabled    Positive vs Negative25.5 (20.6–34.9) vs 30.1 (21.6–54.0)1.13 (0.75–1.69)0.561a. EGFR All patients had EGFR tested.b. For each gene, patients carrying actionable alterations were compared with patients who were tested negative, with patients carrying actionable alterations from other genes excluded from its analysis.c. For KRAS and TP53, patients who were tested positive with no actionable alteration were compared with patients who were tested negative but had actionable gene alterations.d. For KRAS and TP53, patients who were tested positive were compared with patients who were tested negative and patients who had actionable gene alterations were excluded from both groups.

The median OS of all patients was 33.3 months (95% CI; 29.8–39.5) and female patients had a better median OS of 39.8 months (95% CI, 33.5–45.5 months; HR, 1.56; 95% CI, 1.21–2.00; Table 4) as compared to median OS of 27.4 months for male patients. The OS was also better for never/light smokers (<10 pack years) with a median OS of 39.5 months (95% CI, 32.9–45.5 months; HR, 1.42; 95% CI, 1.10–1.83; Table 4) as compared to medium/heavy smokers (> = 10 pack years) who had a median OS of 25.7 months (95% CI, 22.1–33.3 months). Two genes had violations of the proportional hazards assumption EGFR in actionable and KRAS in actionable, therefore, we added an interaction with time to account for changes in the HR. When comparing the median OS of KRAS positive patient (25.5 months; 95% CI, 17.6–32.9 months) with the median OS of actionable patient who had tested KRAS negative (41.1 months; 95% CI, 39.3–55.0 months), the difference was noticeable (Fig 2A), the HR changes from high risk of death in the KRAS positive patients initially to less risk over time (Table 4). Of the six actionable genes only ALK rearrangement patients had a difference in survival with a median OS of 82.6 months (95% CI, 82.6-NR; HR, 0.35; 95% CI, 0.17–0.68;) as compared to 26.6 months (95% CI, 22.1–33.1 months) for tested ALK rearrangement negative patients (Fig 2B). The genomic testing panel size had no discernible OS difference and broad-based sequencing had a median OS of 33.4 months (95% CI, 29.2–44.2 months) as compared to 33.5 months (95% CI, 27.5–41.2 months) for the small panel (Fig 2C). There was a discernible difference in survival for 222 patients, who had an actionable alteration (such as EGFR L858R/exon 19 deletion, ALK rearrangements, ROS1 rearrangements, BRAF V600E, NTRK fusions and MET exon 14 splice site/deletion), with a median OS of 39.8 months as compared to 193 patients who were wild-type with a median OS of 26.0 months (Fig 2D).

A) Overall survival among KRAS altered and tested KRAS negative patients with actionable alteration. B) Overall survival among ALK rearrangement and tested ALK rearrangement negative patients. C) Overall survival among broad-based panel tested and small-panel tested patients. D) Overall survival among actionable altered and wild-type patients. P values comparing risk groups were calculated with the log-rank test.

a. EGFR All patients had EGFR tested.

b. For each gene, patients carrying actionable alterations were compared with patients who were tested negative, with patients carrying actionable alterations from other genes excluded from its analysis.

c. For KRAS and TP53, patients who were tested positive with no actionable alteration were compared with patients who were tested negative but had actionable gene alterations.

d. For KRAS and TP53, patients who were tested positive were compared with patients who were tested negative and patients who had actionable gene alterations were excluded from both groups.

DiscussionThe actualization of tyrosine kinase inhibitors, monoclonal antibodies, and immunotherapy drugs has pushed the treatment of lung cancer forward, however, the reality remains that lung cancer is the leading cause of cancer deaths [19]. Genomic testing has become more crucial in oncology care and recently there have been several studies launched that aim to match targeted therapy to patients based on their omic profile [20–27]. It is now a standard recommendation that patients with advanced NSCLC undergo routine molecular testing for identification of certain known genomic abnormalities, most notably ALK rearrangements, EGFR mutations, BRAF V600E, ROS1 rearrangements, and NTRK fusions [5]. More so several inhibitors for various markers, such as ERBB2, MET, and RET are quickly being implemented in standard clinical care and several others await FDA approval [28–32]. Thus, this study aims to highlight a unique single-site perspective into the clinical heterogeneity found in our cohort of patients that is largely composed of clinically relevant markers that will benefit from recent advances in precision medicine and a cohort that is distinctly different from other databases [17, 18]. To achieve durable survivals for patients it vital to implement appropriate testing at diagnosis and upon progression, and guided treatment plans (both guidelines and pathways) through consolidation of clinical data into patient registries [33].Precision genomicsIn a recent study that compared the use of broad-based genomic sequencing versus small-panel testing, it was shown that there was no significant difference in OS [34]. While our data recapitulates that broad-based sequencing panels were not independently associated with better median OS, our results also show that patients with actionable alterations had a discernibly improved median OS as compared to wild-type patients who did not have an actionable alteration. Most notably ALK rearrangement patients had an improved OS as compared to tested ALK rearrangement negative (median OS 82.6 vs 26.6 months), which is suspected due to appropriate molecular testing at presentation and apt selection of therapy based on results [35, 36]. While Presley et al. include ALK in their small routine panel it should be noted that in a study evaluating the methods of ALK rearrangement testing it was found that comprehensive genomic profiling by NGS was able to identify a number of patients who had previously tested negative using the standard ALK fluorescence in situ hybridization (FISH) assay [37]. While there were several studies that showed improved PFS with targeted therapy which did not translate into OS benefit [7–12], more recent results in lung cancer have shown there is a durable survival benefit for patients who are properly treated with appropriate inhibitors as opposed to patients who do not receive targeted therapy [30, 38–41]. 53.5% (222/415) of patients presented with actionable alterations in our cohort had specific targeted therapies approved by the FDA. While many question the utility to targeted therapy due to low actionable alteration rates [42–44], this study shows that actionable alterations rates may vary demographically and by location presenting a challenge for improving outcomes that may be addressed through the implementation of clinical pathway guidelines and continuing genomic education [45].Guided value-based medicineSeveral studies have been performed that have determined the cost-effectiveness of NGS panels over single gene testing and in one study there was an improvement in PFS without additional health care costs to the patient [46–48]. In several meta-analyses of prospective clinical trials for solid tumors and hematological malignancies, personalized treatment strategies showed superior outcomes to those in control arms [49–54]. Therefore, the question becomes not whether genomic testing should be performed with a specific panel but whether the patient is treated accordingly based on all information available to improve their individual outcomes [54]. For this purpose, several associations including the National Comprehensive Cancer Network (NCCN) have issued guidelines which strongly emphasize not only testing for EGFR, ALK, ROS1, BRAF, and PD-L1 alterations but more importantly advise to conduct broader molecular profiling to identify rare mutational drivers [5, 6]. In practice, it was reported that the majority of oncologists (60%) in North America did not utilize genomic alteration results in their treatment decision making [55, 56].To improve these statistics, the American Society of Clinical Oncology (ASCO) established in 2015 a task force on clinical pathways to improve treatment decision making through evidence-based clinical pathway guidelines [3]. Since then, ASCO revealed in a 2017 State of Cancer Care in America report a 42% increase from 2014 to 2016 of practices complying with a pathway program [57, 58]. Moreover, a recent study that evaluated 7 cancer programs, including COH (both academic and community), that employed clinical pathways demonstrated unprecedently high rates of molecular testing and concordant appropriate first-line treatment decision making based on actionable biomarkers [59]. Thusly, the end-goal for improvement of outcomes can only be achieved under the right circumstances where the appropriate testing is performed and the accurate treatment decisions are made.THOR in comparison with TCGA and GENIEWhile there is some overlap in the alteration rates between our cohort and the publicly available datasets (TCGA/GENIE), there are more differences in the comparable rates of occurrences. Our study identified a disproportionally high rate of EGFR alterations with 49.9% (207/415) and actionable alteration rate with 53.5%, which was notably higher than the TCGA/GENIE rates [17, 18]. However, these results are consistent with a large gene profiling study of NSCLC from the Icahn School of Medicine at Mount Sinai that was able to identify actionable alterations in 65% of the cases [60]. While it is estimated that the actionability rate for NSCLC is around 30%, the results of this single-site analysis warrant further consideration to understand the heterogeneity of lung cancer in different populations and how this may impact survival outcomes [17]. This may in part be due to referrals to COH for clinical care and potential clinical trials, but it cannot be discounted that the high rates of EGFR alterations may be due to a large number of Asian never-smokers at presentation. In fact, 72.8% (99/136) of Asians in our cohort presented with an EGFR alteration as compared with 50.0% EGFR incidence rate found in the GENIE dataset [17]. This large occurrence of EGFR in Asian never-smokers is consistent with another study that identified 75.3% of Asian never-smokers who harbored EGFR mutations [61].Unexpectedly a number of chromatin modulating genes were irregularly prevalent in our cohort as compared with TCGA/GENIE. ARID1A alterations were highly expressed in our cohort alongside ARID1B and the two are known to be mutually exclusive within individual SWItch/Sucrose Non-Fermentable (SWI/SNF) chromatin remodeling complexes [62]. However, ARID1A has been shown to be the strongest tumor suppressor gene amongst the ARID genes in various cancers [63–66] and ARID1A is known to have the highest alteration rate among the SWI/SNF subunit genes [67]. The dysfunction of these complexes is known to destabilize the lung cancer genome by affecting chromatin remodeling and also disrupt DNA repair [68–70]. Furthermore, the high prevalence of genes such as MLL, MLL2, and BRCA2 suggest that other chromatin remodeling motifs may be involved in tumorigenesis of these patients rather than an event specific to a particular tumor type [71, 72].

The actualization of tyrosine kinase inhibitors, monoclonal antibodies, and immunotherapy drugs has pushed the treatment of lung cancer forward, however, the reality remains that lung cancer is the leading cause of cancer deaths [19]. Genomic testing has become more crucial in oncology care and recently there have been several studies launched that aim to match targeted therapy to patients based on their omic profile [20–27]. It is now a standard recommendation that patients with advanced NSCLC undergo routine molecular testing for identification of certain known genomic abnormalities, most notably ALK rearrangements, EGFR mutations, BRAF V600E, ROS1 rearrangements, and NTRK fusions [5]. More so several inhibitors for various markers, such as ERBB2, MET, and RET are quickly being implemented in standard clinical care and several others await FDA approval [28–32]. Thus, this study aims to highlight a unique single-site perspective into the clinical heterogeneity found in our cohort of patients that is largely composed of clinically relevant markers that will benefit from recent advances in precision medicine and a cohort that is distinctly different from other databases [17, 18]. To achieve durable survivals for patients it vital to implement appropriate testing at diagnosis and upon progression, and guided treatment plans (both guidelines and pathways) through consolidation of clinical data into patient registries [33].

Precision genomicsIn a recent study that compared the use of broad-based genomic sequencing versus small-panel testing, it was shown that there was no significant difference in OS [34]. While our data recapitulates that broad-based sequencing panels were not independently associated with better median OS, our results also show that patients with actionable alterations had a discernibly improved median OS as compared to wild-type patients who did not have an actionable alteration. Most notably ALK rearrangement patients had an improved OS as compared to tested ALK rearrangement negative (median OS 82.6 vs 26.6 months), which is suspected due to appropriate molecular testing at presentation and apt selection of therapy based on results [35, 36]. While Presley et al. include ALK in their small routine panel it should be noted that in a study evaluating the methods of ALK rearrangement testing it was found that comprehensive genomic profiling by NGS was able to identify a number of patients who had previously tested negative using the standard ALK fluorescence in situ hybridization (FISH) assay [37]. While there were several studies that showed improved PFS with targeted therapy which did not translate into OS benefit [7–12], more recent results in lung cancer have shown there is a durable survival benefit for patients who are properly treated with appropriate inhibitors as opposed to patients who do not receive targeted therapy [30, 38–41]. 53.5% (222/415) of patients presented with actionable alterations in our cohort had specific targeted therapies approved by the FDA. While many question the utility to targeted therapy due to low actionable alteration rates [42–44], this study shows that actionable alterations rates may vary demographically and by location presenting a challenge for improving outcomes that may be addressed through the implementation of clinical pathway guidelines and continuing genomic education [45].

In a recent study that compared the use of broad-based genomic sequencing versus small-panel testing, it was shown that there was no significant difference in OS [34]. While our data recapitulates that broad-based sequencing panels were not independently associated with better median OS, our results also show that patients with actionable alterations had a discernibly improved median OS as compared to wild-type patients who did not have an actionable alteration. Most notably ALK rearrangement patients had an improved OS as compared to tested ALK rearrangement negative (median OS 82.6 vs 26.6 months), which is suspected due to appropriate molecular testing at presentation and apt selection of therapy based on results [35, 36]. While Presley et al. include ALK in their small routine panel it should be noted that in a study evaluating the methods of ALK rearrangement testing it was found that comprehensive genomic profiling by NGS was able to identify a number of patients who had previously tested negative using the standard ALK fluorescence in situ hybridization (FISH) assay [37]. While there were several studies that showed improved PFS with targeted therapy which did not translate into OS benefit [7–12], more recent results in lung cancer have shown there is a durable survival benefit for patients who are properly treated with appropriate inhibitors as opposed to patients who do not receive targeted therapy [30, 38–41]. 53.5% (222/415) of patients presented with actionable alterations in our cohort had specific targeted therapies approved by the FDA. While many question the utility to targeted therapy due to low actionable alteration rates [42–44], this study shows that actionable alterations rates may vary demographically and by location presenting a challenge for improving outcomes that may be addressed through the implementation of clinical pathway guidelines and continuing genomic education [45].

Guided value-based medicineSeveral studies have been performed that have determined the cost-effectiveness of NGS panels over single gene testing and in one study there was an improvement in PFS without additional health care costs to the patient [46–48]. In several meta-analyses of prospective clinical trials for solid tumors and hematological malignancies, personalized treatment strategies showed superior outcomes to those in control arms [49–54]. Therefore, the question becomes not whether genomic testing should be performed with a specific panel but whether the patient is treated accordingly based on all information available to improve their individual outcomes [54]. For this purpose, several associations including the National Comprehensive Cancer Network (NCCN) have issued guidelines which strongly emphasize not only testing for EGFR, ALK, ROS1, BRAF, and PD-L1 alterations but more importantly advise to conduct broader molecular profiling to identify rare mutational drivers [5, 6]. In practice, it was reported that the majority of oncologists (60%) in North America did not utilize genomic alteration results in their treatment decision making [55, 56].To improve these statistics, the American Society of Clinical Oncology (ASCO) established in 2015 a task force on clinical pathways to improve treatment decision making through evidence-based clinical pathway guidelines [3]. Since then, ASCO revealed in a 2017 State of Cancer Care in America report a 42% increase from 2014 to 2016 of practices complying with a pathway program [57, 58]. Moreover, a recent study that evaluated 7 cancer programs, including COH (both academic and community), that employed clinical pathways demonstrated unprecedently high rates of molecular testing and concordant appropriate first-line treatment decision making based on actionable biomarkers [59]. Thusly, the end-goal for improvement of outcomes can only be achieved under the right circumstances where the appropriate testing is performed and the accurate treatment decisions are made.

Several studies have been performed that have determined the cost-effectiveness of NGS panels over single gene testing and in one study there was an improvement in PFS without additional health care costs to the patient [46–48]. In several meta-analyses of prospective clinical trials for solid tumors and hematological malignancies, personalized treatment strategies showed superior outcomes to those in control arms [49–54]. Therefore, the question becomes not whether genomic testing should be performed with a specific panel but whether the patient is treated accordingly based on all information available to improve their individual outcomes [54]. For this purpose, several associations including the National Comprehensive Cancer Network (NCCN) have issued guidelines which strongly emphasize not only testing for EGFR, ALK, ROS1, BRAF, and PD-L1 alterations but more importantly advise to conduct broader molecular profiling to identify rare mutational drivers [5, 6]. In practice, it was reported that the majority of oncologists (60%) in North America did not utilize genomic alteration results in their treatment decision making [55, 56].

To improve these statistics, the American Society of Clinical Oncology (ASCO) established in 2015 a task force on clinical pathways to improve treatment decision making through evidence-based clinical pathway guidelines [3]. Since then, ASCO revealed in a 2017 State of Cancer Care in America report a 42% increase from 2014 to 2016 of practices complying with a pathway program [57, 58]. Moreover, a recent study that evaluated 7 cancer programs, including COH (both academic and community), that employed clinical pathways demonstrated unprecedently high rates of molecular testing and concordant appropriate first-line treatment decision making based on actionable biomarkers [59]. Thusly, the end-goal for improvement of outcomes can only be achieved under the right circumstances where the appropriate testing is performed and the accurate treatment decisions are made.

THOR in comparison with TCGA and GENIEWhile there is some overlap in the alteration rates between our cohort and the publicly available datasets (TCGA/GENIE), there are more differences in the comparable rates of occurrences. Our study identified a disproportionally high rate of EGFR alterations with 49.9% (207/415) and actionable alteration rate with 53.5%, which was notably higher than the TCGA/GENIE rates [17, 18]. However, these results are consistent with a large gene profiling study of NSCLC from the Icahn School of Medicine at Mount Sinai that was able to identify actionable alterations in 65% of the cases [60]. While it is estimated that the actionability rate for NSCLC is around 30%, the results of this single-site analysis warrant further consideration to understand the heterogeneity of lung cancer in different populations and how this may impact survival outcomes [17]. This may in part be due to referrals to COH for clinical care and potential clinical trials, but it cannot be discounted that the high rates of EGFR alterations may be due to a large number of Asian never-smokers at presentation. In fact, 72.8% (99/136) of Asians in our cohort presented with an EGFR alteration as compared with 50.0% EGFR incidence rate found in the GENIE dataset [17]. This large occurrence of EGFR in Asian never-smokers is consistent with another study that identified 75.3% of Asian never-smokers who harbored EGFR mutations [61].Unexpectedly a number of chromatin modulating genes were irregularly prevalent in our cohort as compared with TCGA/GENIE. ARID1A alterations were highly expressed in our cohort alongside ARID1B and the two are known to be mutually exclusive within individual SWItch/Sucrose Non-Fermentable (SWI/SNF) chromatin remodeling complexes [62]. However, ARID1A has been shown to be the strongest tumor suppressor gene amongst the ARID genes in various cancers [63–66] and ARID1A is known to have the highest alteration rate among the SWI/SNF subunit genes [67]. The dysfunction of these complexes is known to destabilize the lung cancer genome by affecting chromatin remodeling and also disrupt DNA repair [68–70]. Furthermore, the high prevalence of genes such as MLL, MLL2, and BRCA2 suggest that other chromatin remodeling motifs may be involved in tumorigenesis of these patients rather than an event specific to a particular tumor type [71, 72].

While there is some overlap in the alteration rates between our cohort and the publicly available datasets (TCGA/GENIE), there are more differences in the comparable rates of occurrences. Our study identified a disproportionally high rate of EGFR alterations with 49.9% (207/415) and actionable alteration rate with 53.5%, which was notably higher than the TCGA/GENIE rates [17, 18]. However, these results are consistent with a large gene profiling study of NSCLC from the Icahn School of Medicine at Mount Sinai that was able to identify actionable alterations in 65% of the cases [60]. While it is estimated that the actionability rate for NSCLC is around 30%, the results of this single-site analysis warrant further consideration to understand the heterogeneity of lung cancer in different populations and how this may impact survival outcomes [17]. This may in part be due to referrals to COH for clinical care and potential clinical trials, but it cannot be discounted that the high rates of EGFR alterations may be due to a large number of Asian never-smokers at presentation. In fact, 72.8% (99/136) of Asians in our cohort presented with an EGFR alteration as compared with 50.0% EGFR incidence rate found in the GENIE dataset [17]. This large occurrence of EGFR in Asian never-smokers is consistent with another study that identified 75.3% of Asian never-smokers who harbored EGFR mutations [61].

Unexpectedly a number of chromatin modulating genes were irregularly prevalent in our cohort as compared with TCGA/GENIE. ARID1A alterations were highly expressed in our cohort alongside ARID1B and the two are known to be mutually exclusive within individual SWItch/Sucrose Non-Fermentable (SWI/SNF) chromatin remodeling complexes [62]. However, ARID1A has been shown to be the strongest tumor suppressor gene amongst the ARID genes in various cancers [63–66] and ARID1A is known to have the highest alteration rate among the SWI/SNF subunit genes [67]. The dysfunction of these complexes is known to destabilize the lung cancer genome by affecting chromatin remodeling and also disrupt DNA repair [68–70]. Furthermore, the high prevalence of genes such as MLL, MLL2, and BRCA2 suggest that other chromatin remodeling motifs may be involved in tumorigenesis of these patients rather than an event specific to a particular tumor type [71, 72].

ConclusionThe limitation of current oncology practice is the speed of integration and adaptation to the rapidly advancing testing modalities and multiple therapeutic options that are garnering swift approval in various cancer types [73]. These efforts will require standardization of molecular testing modalities, cohesive guidelines, and pathways for precision medicine, and focused patient registries to enable cohort management and efficient clinical trial accrual. While our study was limited by the sample size and the demographics of our patients, it is still important to utilize these tools to guide physicians in determining which patient groups can benefit from clinical trials, tumor resistance or chemotherapy sensitivity. Review of the genomic data of our lung cancer patient cohorts in THOR showed significantly different incidences of actionable mutations as compared to the public datasets. Thus, instead of relying on generalized demographics and genomic results within the public datasets, the individual centers are required to perform their own assessments of their cohorts in order to have a proper analysis of their clinic population.

The limitation of current oncology practice is the speed of integration and adaptation to the rapidly advancing testing modalities and multiple therapeutic options that are garnering swift approval in various cancer types [73]. These efforts will require standardization of molecular testing modalities, cohesive guidelines, and pathways for precision medicine, and focused patient registries to enable cohort management and efficient clinical trial accrual. While our study was limited by the sample size and the demographics of our patients, it is still important to utilize these tools to guide physicians in determining which patient groups can benefit from clinical trials, tumor resistance or chemotherapy sensitivity. Review of the genomic data of our lung cancer patient cohorts in THOR showed significantly different incidences of actionable mutations as compared to the public datasets. Thus, instead of relying on generalized demographics and genomic results within the public datasets, the individual centers are required to perform their own assessments of their cohorts in order to have a proper analysis of their clinic population.

Supporting informationS1 TableNGS platforms of testing.(DOCX)Click here for additional data file.S2 TableActionable genomic alteration rates in TCGA and GENIE.(DOCX)Click here for additional data file.S3 TableMultivariate Cox Proportional Hazards Regression Models for Adjusted for Sex, Age, and Smoking Status.(DOCX)Click here for additional data file.S4 TableCox Proportional Hazards Regression Models for Actionable Genes Adjusted for Sex, Age and Smoking Status.(DOCX)Click here for additional data file.S1 FilePatient clinical and molecular marker data.For molecular markers, "0" means tested negative, and blank means not tested.(XLSX)Click here for additional data file.

(DOCX)

Click here for additional data file.

(DOCX)

Click here for additional data file.

(DOCX)

Click here for additional data file.

(DOCX)

Click here for additional data file.

For molecular markers, "0" means tested negative, and blank means not tested.

(XLSX)

Click here for additional data file.

We thank the clinical staff at City of Hope for their skill and dedication in helping the patients presented in this manuscript. The authors would like to acknowledge the American Association for Cancer Research and its support in the development of the AACR Project GENIE registry, as well as members of the consortium for their commitment to data sharing. Interpretations are the responsibility of study authors. The results published here are in whole or part based upon data generated by the TCGA Research Network: https://www.cancer.gov/tcga.

Non-small cell lung cancer

Institutional Review Board

Fluorescent in situ hybridization

Lung adenocarcinoma

Next-Generation Sequencing

tyrosine-kinase inhibitors

U.S. Food and Drug Administration

formalin-fixed paraffin-embedded

immunohistochemistry

polymerase chain reaction

progression-free survival

Thoracic Oncology Registry

overall survival

SWItch/Sucrose Non-Fermentable

National Comprehensive Cancer Network

American Society of Clinical Oncology

17 Oct 2019

PONE-D-19-20200

Precision Medicine and Actionable Alterations in Lung Cancer

PLOS ONE

Dear Dr. Ravi Salgia,

Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE’s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.

We would appreciate receiving your revised manuscript by 31st October 2019. When you are ready to submit your revision, log on to https://www.editorialmanager.com/pone/ and select the 'Submissions Needing Revision' folder to locate your manuscript file.

If you would like to make changes to your financial disclosure, please include your updated statement in your cover letter.

To enhance the reproducibility of your results, we recommend that if applicable you deposit your laboratory protocols in protocols.io, where a protocol can be assigned its own identifier (DOI) such that it can be cited independently in the future. For instructions see: http://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols

Please include the following items when submitting your revised manuscript:

A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). This letter should be uploaded as separate file and labeled 'Response to Reviewers'.A marked-up copy of your manuscript that highlights changes made to the original version. This file should be uploaded as separate file and labeled 'Revised Manuscript with Track Changes'.An unmarked version of your revised paper without tracked changes. This file should be uploaded as separate file and labeled 'Manuscript'.

A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). This letter should be uploaded as separate file and labeled 'Response to Reviewers'.

A marked-up copy of your manuscript that highlights changes made to the original version. This file should be uploaded as separate file and labeled 'Revised Manuscript with Track Changes'.

An unmarked version of your revised paper without tracked changes. This file should be uploaded as separate file and labeled 'Manuscript'.

Please note while forming your response, if your article is accepted, you may have the opportunity to make the peer review history publicly available. The record will include editor decision letters (with reviews) and your responses to reviewer comments. If eligible, we will contact you to opt in or out.

We look forward to receiving your revised manuscript.

Kind regards,

Ramon Andrade De Mello, MD, FACP, PhD

Academic Editor

PLOS ONE

Journal Requirements:

When submitting your revision, we need you to address these additional requirements.

1. Please ensure that your manuscript meets PLOS ONE's style requirements, including those for file naming. The PLOS ONE style templates can be found at

http://www.journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf and http://www.journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf

2.  Please note that according to our submission guidelines (http://journals.plos.org/plosone/s/submission-guidelines), outmoded terms and potentially stigmatizing labels should be changed to more current, acceptable terminology. For example: “Caucasian” should be changed to “white” or “of [Western] European descent” (as appropriate).

3. As your study involved humans categorized by race/ethnicity, please:

- Explicitly describe the methods of categorizing human populations

- Define categories in as much detail as the study protocol allows

- Justify the choices of definitions and categories, including for example whether any rules of human categorization were required by the funding agency

- Explain whether (and if so, how) confounding variables such as socioeconomic status, nutrition, environmental exposures, or similar factors were controlled for in the analysis

For more information, please see our author guidelines: https://journals.plos.org/plosone/s/submission-guidelines#loc-human-subjects-research

[Note: HTML markup is below. Please do not edit.]

Reviewers' comments:

Reviewer's Responses to Questions

Comments to the Author

1. Is the manuscript technically sound, and do the data support the conclusions?

The manuscript must describe a technically sound piece of scientific research with data that supports the conclusions. Experiments must have been conducted rigorously, with appropriate controls, replication, and sample sizes. The conclusions must be drawn appropriately based on the data presented.

Reviewer #1: Yes

Reviewer #2: Yes

**********

2. Has the statistical analysis been performed appropriately and rigorously?

Reviewer #1: Yes

Reviewer #2: Yes

**********

3. Have the authors made all data underlying the findings in their manuscript fully available?

The PLOS Data policy requires authors to make all data underlying the findings described in their manuscript fully available without restriction, with rare exception (please refer to the Data Availability Statement in the manuscript PDF file). The data should be provided as part of the manuscript or its supporting information, or deposited to a public repository. For example, in addition to summary statistics, the data points behind means, medians and variance measures should be available. If there are restrictions on publicly sharing data—e.g. participant privacy or use of data from a third party—those must be specified.

Reviewer #1: Yes

Reviewer #2: Yes

**********

4. Is the manuscript presented in an intelligible fashion and written in standard English?

PLOS ONE does not copyedit accepted manuscripts, so the language in submitted articles must be clear, correct, and unambiguous. Any typographical or grammatical errors should be corrected at revision, so please note any specific errors here.

Reviewer #1: Yes

Reviewer #2: Yes

**********

5. Review Comments to the Author

Please use the space provided to explain your answers to the questions above. You may also include additional comments for the author, including concerns about dual publication, research ethics, or publication ethics. (Please upload your review as an attachment if it exceeds 20,000 characters)

Reviewer #1: The authors nicely summarized the single institution experience in generating patient registry for comprehensive tumor genomic profiling in lung adenocarcinoma and its impact of patient survival. They found high EGFR mutation rate was likely driven by the enriched Asian population and clinical trials at the academic institution. Thus it is informative to the research community.

There are a few minor concerns:

1. The Title seems to be very general and does not reflect the study refers to single institutional experience.

2. The patients were diagnosed between 2008 and 2016, during which period the diagnosis and treatment of lung adenocarcinomas had evolved, it would be interesting to analyze if there were any differences between the lung cancer cases diagnosed in early and later years.

Reviewer #2: Motivated by the use of genomic tests for cancer treatment as well as the need for organizations to structure and use data to develop a personalized patient's treatment, the authors presents the use of genetic test of 415 patients with lung adenocarcinoma obtained by the Thoracic Oncology Registry. In general, the authors compared patients with genetic alteration with those without alteration (wild type) using statistical analyzes of association (chi-square test) and survival methods. The results show that the most frequent genetic alterations in oncogenes were in EGFR, KRAS and ALK, while the most commonly occurring tumor suppressor genes consisted of TP53, LRP1B and STK11. In addition, the authors compared the study's genomic alterations with other databases. Regarding the results, significant differences were observed in overall survival of patients with actionable change compared to wild type, justifying the use of patient registries and genomic data in cancer treatment. Although the statistical analysis is good, there are differences in the observed frequency for EGFG in table 1 and 2 (177 vs 207) and in table 2 the sum of observed frequency of exon 18 mutation, exon 19 deletion, exon 20 insertion, exon 21 mutation genes it is not equals to 207 (the presented sum is equal to 210). Furthermore, the authors proposed the use of cox proportional hazards models, but did not present or check the validity of proportionality assumption. What was the criteria for justifying a good adjust of the model to data? Also, besides the p-value, the authors should present the qui squared test statistic in table 1. It would be a good idea to check these inconsistencies.

**********

6. PLOS authors have the option to publish the peer review history of their article (what does this mean?). If published, this will include your full peer review and any attached files.

If you choose “no”, your identity will remain anonymous but your review may still be made public.

Do you want your identity to be public for this peer review? For information about this choice, including consent withdrawal, please see our Privacy Policy.

Reviewer #1: Yes: Tianhong Li

Reviewer #2: No

[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files to be viewed.]

While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, https://pacev2.apexcovantage.com/. PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email us at figures@plos.org. Please note that Supporting Information files do not need this step.

19 Nov 2019

Reviewer #1: The authors nicely summarized the single institution experience in generating patient registry for comprehensive tumor genomic profiling in lung adenocarcinoma and its impact of patient survival. They found high EGFR mutation rate was likely driven by the enriched Asian population and clinical trials at the academic institution. Thus it is informative to the research community.

There are a few minor concerns:

1. The Title seems to be very general and does not reflect the study refers to single institutional experience.

We thank the reviewer for their comment and have revised the title to “Precision Medicine and Actionable Alterations in Lung Cancer: A Single Institution Experience”.

2. The patients were diagnosed between 2008 and 2016, during which period the diagnosis and treatment of lung adenocarcinomas had evolved, it would be interesting to analyze if there were any differences between the lung cancer cases diagnosed in early and later years.

In our evaluation of patients who were diagnosed in the early years, we found that the majority of patients only had molecular testing on a small panel of genes, including EGFR, ALK, and KRAS. With the advent of NGS, patients in the later years had larger panels of genes and alterations that were identified, including ROS1 fusion, MET exon 14 alterations and others such as TP53 and STK11. However, as shown in our results, the difference in panels did not show a significant difference in overall survival. Nevertheless, there was a difference in survival between patients with actionable alterations versus those without actionable alterations. As mentioned in our discussion, more and more genes and alterations are receiving FDA approval for targeted therapy and it will be important to continue to screen patients for all actionable alterations to improve their outcomes. Most recently several MET exon 14 deletion targeted therapies, such as tepotinib and capmatinib, have received FDA breakthrough therapy designation with significant improvements in patient outcomes.

Reviewer #2: Motivated by the use of genomic tests for cancer treatment as well as the need for organizations to structure and use data to develop a personalized patient's treatment, the authors presents the use of genetic test of 415 patients with lung adenocarcinoma obtained by the Thoracic Oncology Registry. In general, the authors compared patients with genetic alteration with those without alteration (wild type) using statistical analyzes of association (chi-square test) and survival methods. The results show that the most frequent genetic alterations in oncogenes were in EGFR, KRAS and ALK, while the most commonly occurring tumor suppressor genes consisted of TP53, LRP1B and STK11. In addition, the authors compared the study's genomic alterations with other databases. Regarding the results, significant differences were observed in overall survival of patients with actionable change compared to wild type, justifying the use of patient registries and genomic data in cancer treatment. Although the statistical analysis is good, there are differences in the observed frequency for EGFG in table 1 and 2 (177 vs 207) and in table 2 the sum of observed frequency of exon 18 mutation, exon 19 deletion, exon 20 insertion, exon 21 mutation genes it is not equals to 207 (the presented sum is equal to 210). Furthermore, the authors proposed the use of cox proportional hazards models, but did not present or check the validity of proportionality assumption. What was the criteria for justifying a good adjust of the model to data? Also, besides the p-value, the authors should present the qui squared test statistic in table 1. It would be a good idea to check these inconsistencies.

We thank the reviewer for their detailed comments and have addressed their concerns below.

Regarding the discrepancy in EGFR incidence between Table 1 and Table 2: In Table 1 the EGFR is labeled “EGFR (L858R/exon 19 deletion)” and therefore only includes EGFR patients who had those two alterations. The reason for this is because they are the only two types of EGFR alterations that are considered “actionable” by the FDA and eligible for EGFR TKIs. Meanwhile, Table 2 includes all types of EGFR alterations including exon 18 mutation, exon 20 insertion and others. Hence, why there is a discrepancy of 177 vs 207. The methods for testing the assumption of proportional hazards was provided in the methods of the manuscript. Furthermore, the Cox models were adjusted using time-dependent variables to adjust the non-proportionality of variables. The revised Cox models are presented in Table 4, Supplementary Table 3, and Supplementary Table 4. Table 1 was revised to include the chi-squared test statistics.

Submitted filename: Response to Reviewers.docx

Click here for additional data file.

10 Jan 2020

Precision medicine and actionable alterations in lung cancer: a single institution experience

PONE-D-19-20200R1

Dear Dr. Ravi Salgia,

We are pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it complies with all outstanding technical requirements.

Within one week, you will receive an e-mail containing information on the amendments required prior to publication. When all required modifications have been addressed, you will receive a formal acceptance letter and your manuscript will proceed to our production department and be scheduled for publication.

Shortly after the formal acceptance letter is sent, an invoice for payment will follow. To ensure an efficient production and billing process, please log into Editorial Manager at https://www.editorialmanager.com/pone/, click the "Update My Information" link at the top of the page, and update your user information. If you have any billing related questions, please contact our Author Billing department directly at authorbilling@plos.org.

If your institution or institutions have a press office, please notify them about your upcoming paper to enable them to help maximize its impact. If they will be preparing press materials for this manuscript, you must inform our press team as soon as possible and no later than 48 hours after receiving the formal acceptance. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.

With kind regards,

Ramon Andrade De Mello, MD, PhD, FACP

Academic Editor

PLOS ONE

3 Feb 2020

PONE-D-19-20200R1

Precision medicine and actionable alterations in lung cancer: a single institution experience

Dear Dr. Salgia:

I am pleased to inform you that your manuscript has been deemed suitable for publication in PLOS ONE. Congratulations! Your manuscript is now with our production department.

If your institution or institutions have a press office, please notify them about your upcoming paper at this point, to enable them to help maximize its impact. If they will be preparing press materials for this manuscript, please inform our press team within the next 48 hours. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information please contact onepress@plos.org.

For any other questions or concerns, please email plosone@plos.org.

Thank you for submitting your work to PLOS ONE.

With kind regards,

PLOS ONE Editorial Office Staff

on behalf of

Professor Ramon Andrade De Mello

Academic Editor

PLOS ONE
