---
pmid: '12845604'
title: Genome-wide identification of fungal GPI proteins.
authors:
- De Groot PW
- Hellingwerf KJ
- Klis FM
journal: Yeast
year: '2003'
full_text_available: false
pmcid: PMC12305055
doi: 10.1002/yea.1007
---

# Genome-wide identification of fungal GPI proteins.
**Authors:** De Groot PW, Hellingwerf KJ, Klis FM
**Journal:** Yeast (2003)
**DOI:** [10.1002/yea.1007](https://doi.org/10.1002/yea.1007)
**PMC:** [PMC12305055](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12305055/)

## Abstract

1. Yeast. 2003 Jul 15;20(9):781-96. doi: 10.1002/yea.1007.

Genome-wide identification of fungal GPI proteins.

De Groot PW(1), Hellingwerf KJ, Klis FM.

Author information:
(1)Laboratory for Microbiology, Swammerdam Institute for Life Sciences, 
University of Amsterdam, Nieuwe Achtergracht 166, 1018 WV Amsterdam, The 
Netherlands. pgroot@science.uva.nl

Glycosylphosphatidylinositol-modified (GPI) proteins share structural features 
that allow their identification using a genomic approach. From the known S. 
cerevisiae and C. albicans GPI proteins, the following consensus sequence for 
the GPI attachment site and its downstream region was derived: 
[NSGDAC]-[GASVIETKDLF]-[GASV]-X(4,19)-[FILMVAGPSTCYWN](10)>, where > indicates 
the C-terminal end of the protein. This consensus sequence, which recognized 
known GPI proteins from various fungi, was used to screen the genomes of the 
yeasts S. cerevisiae, C. albicans, Sz. pombe and the filamentous fungus N. 
crassa for putative GPI proteins. The subsets of proteins so obtained were 
further screened for the presence of an N-terminal signal sequence for the 
secretion and absence of internal transmembrane domains. In this way, we 
identified 66 putative GPI proteins in S. cerevisiae. Some of these are known 
GPI proteins that were not identified by earlier genomic analyses, indicating 
that this selection procedure renders a more complete image of the S. cerevisiae 
GPI proteome. Using the same approach, 104 putative GPI proteins were identified 
in the human pathogen C. albicans. Among these were the proteins Gas/Phr, Ecm33, 
Crh and Plb, all members of GPI protein families that are also present in S. 
cerevisiae. In addition, several proteins and protein families with no 
significant homology to S. cerevisiae proteins were identified, including the 
cell wall-associated Als, Csa1/Rbt5, Hwp1/Rbt1 and Hyr1 protein families. In Sz. 
pombe, which has a low level of (galacto)mannan in the cell wall compared to C. 
albicans and S. cerevisiae, only 33 GPI candidates were identified and in N. 
crassa 97. BLAST searches revealed that about half of the putative GPI proteins 
that were identified in Sz. pombe and N. crassa are homologous to known or 
putative GPI proteins from other fungi. We conclude that our algorithm is 
selective and can also be used for GPI protein identification in other fungi.

Copyright 2003 John Wiley & Sons, Ltd.

DOI: 10.1002/yea.1007
PMID: 12845604 [Indexed for MEDLINE]
