---
pmid: '23623729'
title: Architecture of human translation initiation factor 3.
authors:
- Querol-Audi J
- Sun C
- Vogan JM
- Smith MD
- Gu Y
- Cate JH
- Nogales E
journal: Structure
year: '2013'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC3739965
doi: 10.1016/j.str.2013.04.002
---

# Architecture of human translation initiation factor 3.
**Authors:** Querol-Audi J, Sun C, Vogan JM, Smith MD, Gu Y, Cate JH, Nogales E
**Journal:** Structure (2013)
**DOI:** [10.1016/j.str.2013.04.002](https://doi.org/10.1016/j.str.2013.04.002)
**PMC:** [PMC3739965](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3739965/)

## Abstract

1. Structure. 2013 Jun 4;21(6):920-8. doi: 10.1016/j.str.2013.04.002. Epub 2013
Apr  25.

Architecture of human translation initiation factor 3.

Querol-Audi J(1), Sun C, Vogan JM, Smith MD, Gu Y, Cate JH, Nogales E.

Author information:
(1)Department of Molecular and Cell Biology, California Institute for 
Quantitative Biosciences, University of California-Berkeley, Berkeley, CA 94720, 
USA.

Eukaryotic translation initiation factor 3 (eIF3) plays a central role in 
protein synthesis by organizing the formation of the 43S preinitiation complex. 
Using genetic tag visualization by electron microscopy, we reveal the molecular 
organization of ten human eIF3 subunits, including an octameric core. The 
structure of eIF3 bears a close resemblance to that of the proteasome lid, with 
a conserved spatial organization of eight core subunits containing PCI and MPN 
domains that coordinate functional interactions in both complexes. We further 
show that eIF3 subunits a and c interact with initiation factors eIF1 and eIF1A, 
which control the stringency of start codon selection. Finally, we find that 
subunit j, which modulates messenger RNA interactions with the small ribosomal 
subunit, makes multiple independent interactions with the eIF3 octameric core. 
These results highlight the conserved architecture of eIF3 and how it scaffolds 
key factors that control translation initiation in higher eukaryotes, including 
humans.

Copyright © 2013 Elsevier Ltd. All rights reserved.

DOI: 10.1016/j.str.2013.04.002
PMCID: PMC3739965
PMID: 23623729 [Indexed for MEDLINE]

## Full Text

INTRODUCTION

Human translation initiation factor 3 (eIF3) plays a central role in assembling translation initiation complexes. Comprising 13 different subunits, named eIF3a to eIF3m ( Zhou et al., 2008 ) ( FIG. 1A ), eIF3 is the largest of the eukaryotic translation initiation factors (800 kDa molecular mass). The importance of eIF3 in cell homeostasis is reflected by the fact that misregulation of eIF3 subunit expression has been implicated in oncogenesis and the maintenance of the cancerous state ( Silvera et al., 2010 ). Interactions between eIF3 and other translation initiation factors control the binding of the eIF2/Met-tRNA i /GTP ternary complex to the small (40S) ribosomal subunit, positioning the mRNA on the 40S subunit, and modulate the stringency of start codon selection ( Hinnebusch and Lorsch, 2012 ). Despite its biological significance, our mechanistic understanding of preinitiation complex formation remains limited due to the scarcity of structural information. For example, little is known about the molecular organizaton of eIF3, although it interacts with most of the other initiation factors during translation initiation ( Hinnebusch and Lorsch, 2012 ). Our previous cryo-electron microscopy reconstruction of endogenously purified human eIF3 revealed a five-lobed structure with anthropomorphic features that binds to the platform of the 40S ribosomal subunit ( Siridechadilok et al., 2005 ). Remarkably, our 3D reconstruction of a bacterially reconstituted octameric core complex of eIF3 showed it to be structurally very similar to the native eIF3 ( Sun et al., 2011 ). Intriguingly, the eight subunits within the human eIF3 octameric core complex are compositionally related to two other functionally unrelated complexes: the COP9 signalosome and the 19S proteasome lid. All three complexes contain six subunits bearing PCI ( P roteasome, C OP9, e I F3) signature domains (in eIF3 subunits a, c, e, k, l and m), and two subunits bearing MPN ( M pr1– P ad1 N -terminal) signature domains (in eIF3 subunits f and h) ( Hofmann and Bucher, 1998 ) ( FIG. 1A ). The roles of the PCI and MPN domains are not well understood, but the 3-dimensional reconstruction of the proteasome lid recently reported indicates that the PCI domains serve as a central structural scaffold ( da Fonseca et al., 2012 ; Lander et al., 2012 ; Lasker et al., 2012 ).

The PCI/MPN core of eIF3 is also involved in binding to other translation factors ( Pick et al., 2009 ; Sun et al., 2011 ; Zhou et al., 2008 ), suggesting also a scaffolding role, similar to the proteasome lid subunits, which interact with the proteasome 20S particle ( da Fonseca et al., 2012 ; Lander et al., 2012 ). For example, the eIF3 PCI/MPN octamer binds to translation initiation factors eIF1, eIF1A, eIF2, as well as to the 40S ribosomal subunit ( Sun et al., 2011 ). Human eIF3 also interacts directly with the hepatitis C viral (HCV) 5′-untranslated region (5′-UTR) via an internal ribosome entry site (IRES) to promote the translation of viral proteins in a cap independent manner ( Fraser and Doudna, 2007 ; Sizova et al., 1998 ) We have recently shown that the eIF3 PCI/MPN octamer exhibits similar binding capacities to the HCV IRES ( Sun et al., 2011 ).

Here, we present the cryo-electron microscopy reconstruction of the reconstituted eIF3 PCI/MPN octamer. The E. coli expression system for the eIF3 octamer allowed us to use genetic tags as labels for EM localization of all eight subunits of the eIF3 PCI/MPN core that, together with the cryo-EM reconstruction, provides a complete architectural picture of the eIF3 core. When compared with the structure of the proteasome lid, the present cryo-EM reconstruction of the eIF3 PCI/MPN core reveals that they both share the same overall architecture and PCI domain organization. Thus, our studies provide new structural understanding on the conservation of the PCI/MPN core in eIF3, the proteasome lid and the COP9 signalosome. We further define the sites of interaction of the PCI/MPN core with two other non-core subunits within eIF3 and with translation initiation factors eIF1 and eIF1A, providing structural insight into eIF3 functions in regulating translation initiation.

DISCUSSION

The improved cryo-electron microscopy reconstruction of the eIF3 core particle presented here, together with genetic tagging, has allowed us to derive a model for the subunit architecture of human eIF3. Using the new reconstruction and tagging approach, we were able to locate the position of the six PCI-containing, two MPN-containing and two non-core subunits in eIF3. The spatial organization of the subunits in the eIF3 core complex is in agreement with mass-spectrometry results that showed that subunits a and c serve as bridges between subcomplexes or “modules” containing subunits h-f-m, which reside in the left leg, and k-l-e-d, which comprise the right leg and arm ( Zhou et al., 2008 ). The positions of the subunits are also consistent with limited proteolysis experiments that revealed the presence of the eIF3 core subunits through their pattern of protection in the presence of the HCV IRES ( Cai et al., 2010 ).

The overall shape and subunit organization of the eIF3 core complex remarkably resembles the 26S proteasome lid subcomplex, in particular when the lid complex is seen interacting with the rest of the proteasome. Previously reported comparisons between eIF3 and the proteasome lid complex based on models at about 40 Å resolution, invoked removing densities from the left and right arms of eIF3 to establish a one-to-one correspondence between its structural elements and those of the lid ( Enchev et al., 2010 ). Interestingly, upon lid binding to the holoenzyme, the Rpn5 N-terminus reorients, making this lid subunit adopt the appearance of an extended arm resembling that of subunit a in eIF3 ( Lander et al., 2012 ). Our present comparison shows, more clearly than ever before, that the eIF3 octameric core and the 26S proteasome lid share a strikingly common architecture. The comparison between the CSN complex ( Enchev et al., 2012 ) and the proteasome lid ( da Fonseca et al., 2012 ) also shows a common spatial organization where the PCI-containing proteins Csn1, Csn2, Csn3, Csn4, Csn7 and Csn8 are positioned in a very similar fashion when compared to the corresponding ones in either eIF3 or the proteasome lid. Csn6 is also located in a similar region of its MPN-containing counterparts. More interestingly, Csn5, the other MPN-containing protein, is protruding and occupies a peripheral location within the complex ( Enchev et al., 2012 ). Thus, in contrast with what we have previously described ( Sun et al., 2011 ) for eIF3, where both MPN subunits are required in order to obtain a stable core complex (indicating their scaffolding role), Csn5 is dispensable for assembly of the CSN complex.

We propose, based both on the pairwise identification of corresponding subunit domains by EM labeling of the proteasome lid ( Lander et al., 2012 ) and the core eIF3 (this work), and on the direct docking of PCI domains within our cryo-EM reconstruction of the octameric eIF3 core, that the C-terminal PCI domains of subunits a, c, e, l, m and k laterally interact to form a horseshoe shaped structure at the base of eIF3. This arrangement in both subcomplexes strongly indicates a common scaffolding role of the PCI domains in PCI-containing complexes. It is interesting to note that the MPN-subunits f and h form a bridge between the left and right legs, probably stabilizing the complex and explaining why we were not able to over-express and purify an eIF3 sub-complex containing only the six PCI-subunits. Interestingly, previous studies for both the CSN and the proteasome lid complexes ( da Fonseca et al., 2012 ; Enchev et al., 2012 ) propose a spatial arrangement of the PCI domains in which only the five PCI-containing subunits built up the horseshoe shaped region. Further biochemical and structural studies will be needed to determine whether this is the case for eIF3.

The densities attributed to subunits a, c and e in the left arm, head and right arm extensions, respectively, correspond well to the expected size of the a*, c* and e subunits based on docking of the homologous Rpn6 into the cryo-EM reconstruction. Notably, the a* and c* arms are the most mobile elements in the eIF3 core complex ( Figure 1 ), and correspond to the regions in closest proximity to the mRNA binding cleft of the 40S subunit in the context of the existing 40S/eIF3 model ( Siridechadilok et al., 2005 ). The position of subunit a in the left arm is also in agreement with experiments in yeast showing that the N-terminal domain of subunit a binds the small ribosomal protein Rps0A, located near the mRNA exit pore ( Ben-Shem et al., 2011 ; Kouba et al., 2012 ; Rabl et al., 2011 ; Szamecz et al., 2008 ). Interestingly, experiments also performed in yeast led the authors to propose that the C-terminal domain of subunit a would be positioned near the 40S mRNA entry channel ( Chiu et al., 2010 ). In light of our results, this C-terminal extension likely folds back from the mid-body of eIF3, where its PCI domain is located, to reach the mRNA entry channel. In order for subunits a and c to interact with eIF1 and eIF1A on the 40S platform, a structural reorientation of both subunits would be required ( Figure 6 ). This kind of rearrangement would not be surprising on the light of the huge subunit movements observed in the highly structurally related proteasome lid ( Lander et al., 2012 ).

Although subunit j in eIF3 is conserved in both yeast and humans, biochemical and genetic evidence suggest that the details of its functions in translation have likely diverged. For example, in yeast, both the N-terminal and C-terminal regions of eIF3j are required for its tight association with eIF3 ( Chiu et al., 2010 ), whereas human eIF3j employs multiple interactions to bind to the eIF3 octameric core ( Figure 5E ). Furthermore, the C-terminal domain of yeast eIF3j is not required for wild-type growth rates ( Elantak et al., 2010 ), whereas in humans, the C-terminal 16 amino acids of subunit j are cleaved off by caspase 3 during apoptosis ( Bushell et al., 2000 ), suggesting a biological significance for this region. Notably in humans, subunit j negatively regulates the binding of mRNA to the mRNA binding cleft in the 40S subunit by occupying the mRNA decoding site with its C-terminal tail ( Fraser et al., 2007 ; Fraser et al., 2004 ). Formation of the intact 43S preinitiation complex with mRNA bound does not fully displace eIF3j from the mRNA entry channel ( Fraser et al., 2007 ), possibly due to the interaction of the C-terminal region of subunit j with the c* region of eIF3 ( Figure 5F ) or interactions between other regions of eIF3 and the N-terminal region of subunit j. The interactions of subunit j with the octameric core may also play important roles in stabilizing eIF3 binding to the 40S subunit ( Fraser et al., 2004 ; Unbehaun et al., 2004 ). Notably, the interactions of subunit j with the core are much tighter than its interactions with subunit b ( Figure 5D ) ( ElAntak et al., 2007 ; Sun et al., 2011 ). Future experiments will be required to delineate the specific roles in humans of the relatively high affinity interactions between eIF3j and the octameric core, and the weaker interaction of eIF3j with eIF3b.

Translation initiation factors eIF1 and eIF1A play important roles in the stringency of start codon selection, in part mediated by their interactions with eIF3. Although the details of eIF1 interaction with the N-terminal domain of subunit c have been delineated in yeast ( Karaskova et al., 2012 ; Valasek et al., 2004 ), the additional interactions of eIF1 with the eIF3 octameric core, and the interaction of the N-terminal extension of eIF1A with eIF3 ( Olsen et al., 2003 ) were not fully understood. We now show that eIF1 has an additional interaction with subunit c* in the human octameric core, and also binds to a*. Furthermore, eIF1A binds to both the a* and c* subunits in the octameric core. Based on the position of the a* and c* in the left arm and head of eIF3, respectively, these subunits are well positioned to direct subunits eIF1 and eIF1A into the intersubunit space of the 40S subunit, where they function in modulating the conformation of the 43S preinitiation complex during mRNA scanning and start codon selection ( Algire et al., 2005 ; Karaskova et al., 2012 ; Maag et al., 2005 ; Martin-Marcos et al., 2011 ; Olsen et al., 2003 ; Passmore et al., 2007 ; Pestova and Kolupaeva, 2002 ; Valasek et al., 2004 ).

The structural and biochemical results presented here pave the way for future efforts to understand the multiple roles of eIF3 in the assembly of the 43S preinitiation complex and subsequent steps of translation initiation.
