---
pmid: '25065913'
title: ELOVL5 mutations cause spinocerebellar ataxia 38.
authors:
- Di Gregorio E
- Borroni B
- Giorgio E
- Lacerenza D
- Ferrero M
- Lo Buono N
- Ragusa N
- Mancini C
- Gaussen M
- Calcia A
- Mitro N
- Hoxha E
- Mura I
- Coviello DA
- Moon YA
- Tesson C
- Vaula G
- Couarch P
- Orsi L
- Duregon E
- Papotti MG
- Deleuze JF
- Imbert J
- Costanzi C
- Padovani A
- Giunti P
- Maillet-Vioud M
- Durr A
- Brice A
- Tempia F
- Funaro A
- Boccone L
- Caruso D
- Stevanin G
- Brusco A
journal: Am J Hum Genet
year: '2014'
full_text_available: false
pmcid: PMC4129408
doi: 10.1016/j.ajhg.2014.07.001
---

# ELOVL5 mutations cause spinocerebellar ataxia 38.
**Authors:** Di Gregorio E, Borroni B, Giorgio E, Lacerenza D, Ferrero M, Lo Buono N, Ragusa N, Mancini C, Gaussen M, Calcia A, Mitro N, Hoxha E, Mura I, Coviello DA, Moon YA, Tesson C, Vaula G, Couarch P, Orsi L, Duregon E, Papotti MG, Deleuze JF, Imbert J, Costanzi C, Padovani A, Giunti P, Maillet-Vioud M, Durr A, Brice A, Tempia F, Funaro A, Boccone L, Caruso D, Stevanin G, Brusco A
**Journal:** Am J Hum Genet (2014)
**DOI:** [10.1016/j.ajhg.2014.07.001](https://doi.org/10.1016/j.ajhg.2014.07.001)
**PMC:** [PMC4129408](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4129408/)

## Abstract

1. Am J Hum Genet. 2014 Aug 7;95(2):209-17. doi: 10.1016/j.ajhg.2014.07.001. Epub
 2014 Jul 24.

ELOVL5 mutations cause spinocerebellar ataxia 38.

Di Gregorio E(1), Borroni B(2), Giorgio E(3), Lacerenza D(3), Ferrero M(3), Lo 
Buono N(3), Ragusa N(3), Mancini C(3), Gaussen M(4), Calcia A(3), Mitro N(5), 
Hoxha E(6), Mura I(7), Coviello DA(7), Moon YA(8), Tesson C(4), Vaula G(9), 
Couarch P(10), Orsi L(9), Duregon E(11), Papotti MG(11), Deleuze JF(12), Imbert 
J(13), Costanzi C(2), Padovani A(2), Giunti P(14), Maillet-Vioud M(15), Durr 
A(16), Brice A(16), Tempia F(6), Funaro A(3), Boccone L(17), Caruso D(6), 
Stevanin G(18), Brusco A(19).

Author information:
(1)Department of Medical Sciences, University of Torino, 10126 Torino, Italy; 
Medical Genetics Unit, Azienda Ospedaliera Universitaria Città della Salute e 
della Scienza, 10126 Torino, Italy.
(2)Department of Neurology, University of Brescia, 25100 Brescia, Italy.
(3)Department of Medical Sciences, University of Torino, 10126 Torino, Italy.
(4)Institut National de la Santé et de la Recherche Médicale U1127, 75013 Paris, 
France; Centre National de la Recherche Scientifique UMR 7225, 75013 Paris, 
France; Sorbonne Universités, Université Pierre et Marie Curie (Paris 6) UMR_S 
1127, Institut du Cerveau et de la Moelle Épinière, 75013 Paris, France; 
Neurogenetics team, École Pratique des Hautes Études, HéSam Université, 75013 
Paris, France.
(5)Department of Pharmacological and Biomolecular Sciences, University of 
Milano, 20133 Milano, Italy.
(6)Neuroscience Institute Cavalieri Ottolenghi, University of Torino, 10043 
Orbassano, Italy.
(7)Laboratory of Human Genetics, Galliera Hospital, 16128 Genova, Italy.
(8)Department of Molecular Genetics, University of Texas Southwestern Medical 
Center, Dallas, TX 75390-9046, USA.
(9)Neurologic Division 1, Department of Neuroscience and Mental Health, Azienda 
Ospedaliera Universitaria Città della Salute e della Scienza, 10126 Torino, 
Italy.
(10)Institut National de la Santé et de la Recherche Médicale U1127, 75013 
Paris, France; Centre National de la Recherche Scientifique UMR 7225, 75013 
Paris, France; Sorbonne Universités, Université Pierre et Marie Curie (Paris 6) 
UMR_S 1127, Institut du Cerveau et de la Moelle Épinière, 75013 Paris, France.
(11)Department of Oncology, University of Torino at San Luigi Hospital, 10043 
Orbassano, Italy.
(12)Centre National de Génotypage, 91057 Evry, France.
(13)Transcriptomic and Genomic Marseille-Luminy platform, Technological Advances 
for Genomics and Clinics Laboratory, Institut National de la Santé et de la 
Recherche Médicale UMR_S 1090, Aix-Marseille University, 13009 Marseille, 
France.
(14)Department of Molecular Neuroscience, University College London Institute of 
Neurology, WC1 N3BG London, UK.
(15)Centre Hospitalier General de Montlucon, 03113 Montluçon, France.
(16)Institut National de la Santé et de la Recherche Médicale U1127, 75013 
Paris, France; Centre National de la Recherche Scientifique UMR 7225, 75013 
Paris, France; Sorbonne Universités, Université Pierre et Marie Curie (Paris 6) 
UMR_S 1127, Institut du Cerveau et de la Moelle Épinière, 75013 Paris, France; 
Fédération de Génétique, Pitié-Salpêtrière Hospital, Assistance Publique - 
Hôpitaux de Paris, 75013 Paris, France.
(17)Ospedale Regionale Microcitemie, Azienda Unità Sanitaria Locale 8, 09121 
Cagliari, Italy.
(18)Institut National de la Santé et de la Recherche Médicale U1127, 75013 
Paris, France; Centre National de la Recherche Scientifique UMR 7225, 75013 
Paris, France; Sorbonne Universités, Université Pierre et Marie Curie (Paris 6) 
UMR_S 1127, Institut du Cerveau et de la Moelle Épinière, 75013 Paris, France; 
Neurogenetics team, École Pratique des Hautes Études, HéSam Université, 75013 
Paris, France; Fédération de Génétique, Pitié-Salpêtrière Hospital, Assistance 
Publique - Hôpitaux de Paris, 75013 Paris, France.
(19)Department of Medical Sciences, University of Torino, 10126 Torino, Italy; 
Medical Genetics Unit, Azienda Ospedaliera Universitaria Città della Salute e 
della Scienza, 10126 Torino, Italy. Electronic address: alfredo.brusco@unito.it.

Spinocerebellar ataxias (SCAs) are a heterogeneous group of autosomal-dominant 
neurodegenerative disorders involving the cerebellum and 23 different genes. We 
mapped SCA38 to a 56 Mb region on chromosome 6p in a SCA-affected Italian family 
by whole-genome linkage analysis. Targeted resequencing identified a single 
missense mutation (c.689G>T [p.Gly230Val]) in ELOVL5. Mutation screening of 456 
independent SCA-affected individuals identified the same mutation in two further 
unrelated Italian families. Haplotyping showed that at least two of the three 
families shared a common ancestor. One further missense variant (c.214C>G 
[p.Leu72Val]) was found in a French family. Both missense changes affect 
conserved amino acids, are predicted to be damaging by multiple bioinformatics 
tools, and were not identified in ethnically matched controls or within variant 
databases. ELOVL5 encodes an elongase involved in the synthesis of 
polyunsaturated fatty acids of the ω3 and ω6 series. Arachidonic acid and 
docosahexaenoic acid, two final products of the enzyme, were reduced in the 
serum of affected individuals. Immunohistochemistry on control mice and human 
brain demonstrated high levels in Purkinje cells. In transfection experiments, 
subcellular localization of altered ELOVL5 showed a perinuclear distribution 
with a signal increase in the Golgi compartment, whereas the wild-type showed a 
widespread signal in the endoplasmic reticulum. SCA38 and SCA34 are examples of 
SCAs due to mutations in elongase-encoding genes, emphasizing the importance of 
fatty-acid metabolism in neurological diseases.

Copyright © 2014 The American Society of Human Genetics. Published by Elsevier 
Inc. All rights reserved.

DOI: 10.1016/j.ajhg.2014.07.001
PMCID: PMC4129408
PMID: 25065913 [Indexed for MEDLINE]
