---
pmid: '21186352'
title: Drosophila Piwi functions in Hsp90-mediated suppression of phenotypic variation.
authors:
- Gangaraju VK
- Yin H
- Weiner MM
- Wang J
- Huang XA
- Lin H
journal: Nat Genet
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3443399
doi: 10.1038/ng.743
---

# Drosophila Piwi functions in Hsp90-mediated suppression of phenotypic variation.
**Authors:** Gangaraju VK, Yin H, Weiner MM, Wang J, Huang XA, Lin H
**Journal:** Nat Genet (2011)
**DOI:** [10.1038/ng.743](https://doi.org/10.1038/ng.743)
**PMC:** [PMC3443399](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3443399/)

## Abstract

1. Nat Genet. 2011 Feb;43(2):153-8. doi: 10.1038/ng.743. Epub 2010 Dec 26.

Drosophila Piwi functions in Hsp90-mediated suppression of phenotypic variation.

Gangaraju VK(1), Yin H, Weiner MM, Wang J, Huang XA, Lin H.

Author information:
(1)Yale Stem Cell Center and Department of Cell Biology, Yale University School 
of Medicine, New Haven, Connecticut, USA.

Comment in
    Nat Genet. 2011 Feb;43(2):88-9. doi: 10.1038/ng0211-88.

Canalization, also known as developmental robustness, describes an organism's 
ability to produce the same phenotype despite genotypic variations and 
environmental influences. In Drosophila, Hsp90, the trithorax-group proteins and 
transposon silencing have been previously implicated in canalization. Despite 
this, the molecular mechanism underlying canalization remains elusive. Here 
using a Drosophila eye-outgrowth assay sensitized by the dominant Kr(irregular 
facets-1)(Kr(If-1)) allele, we show that the Piwi-interacting RNA (piRNA) 
pathway, but not the short interfering RNA or micro RNA pathway, is involved in 
canalization. Furthermore, we isolated a protein complex composed of Hsp90, Piwi 
and Hop, the Hsp70/Hsp90 organizing protein homolog, and we demonstrated the 
function of this complex in canalization. Our data indicate that Hsp90 and Hop 
regulate the piRNA pathway through Piwi to mediate canalization. Moreover, they 
point to epigenetic silencing of the expression of existing genetic variants and 
the suppression of transposon-induced new genetic variation as two major 
mechanisms underlying piRNA pathway-mediated canalization.

DOI: 10.1038/ng.743
PMCID: PMC3443399
PMID: 21186352 [Indexed for MEDLINE]

## Full Text

Canalization, also known as developmental robustness, describes an organism’s ability to produce the same phenotype despite genotypic variations and environmental influences1,2. In Drosophila, Hsp90, the Trithorax group proteins, and transposon silencing have been implicated in canalization3,4. Despite this, molecular mechanism underlying canalization remains elusive. Here, using an Drosophila eye-outgrowth assay sensitized by the dominant KrIrregular facets-1(KrIf-1), allele3, we show that the piRNA pathway, but not siRNA or miRNA pathways, is involved in canalization. Furthermore, we isolated a protein complex composed of Hsp90, Piwi, and the Hsp70/Hsp90 Organizing Protein Homolog (Hop), and demonstrated the function of this complex in canalization. Our data indicate that Hsp90 and Hop regulate the piRNA pathway via Piwi to mediate canalization. Moreover, they point to epigenetic silencing of the expression of existing genetic variants and the suppression of transposon-induced new genetic variation as two major mechanisms underlying piRNA pathway-mediated canalization.

In both plants and animals, Hsp90 buffers against morphological changes induced either by genetic or non-genetic mechanisms, thereby promoting the robustness of the developmental programs that have been subjected to natural selection5-8. However, under certain conditions, such as environmental stress, Hsp90 becomes overwhelmed, loosens its grip on canalization, and fails to repress the expression of genotype variants that have accumulated during evolution. The expressed phenotypes quickly become independent of Hsp90 deficiency, can be inherited in later generations, and could be subject to natural selection3,5. In addition to Hsp90, maternally inherited epigenetic machineries also prevent expression of genotypic variants3, indicating that faithful transmission of epigenetic marks across generations is also critical for canalization. Hence, examining the regulation of epigenetic inheritance should provide important insights into the molecular mechanisms underlying canalization. Piwi, a piRNA binding protein, is implicated in epigenetic regulation as both a maternal and a zygotic factor9-14. Hence, we reasoned that Piwi might mediate canalization through its epigenetic function.

To test our hypothesis, we used a dominant gain-of-function allele, KrIf-1, that ectopically expresses Krüppel, a zinc-finger transcription factor, in the ventral region of the developing eye imaginal disc15. This ectopic expression misregulates homeotic genes in the eye disc and generates eye outgrowths, which, however, are normally repressed and present only in less than 0.1% of KrIf-1 progeny3,15,16. The loss-of-function mutations of Hsp83 and the trithorax group of genes enhance the expression of this phenotype, implicating these factors in canalization3. We used this sensitized assay to examine if a reduction in maternal dose of Piwi also enhances the outgrowths. The genetic cross was setup as shown in Figure 1A. We observed that strong piwi alleles, piwi−1 and piwi−2, are dominant enhancers of the eye outgrowth phenotype induced by Krüppel ectopic expression (Figure 1B). When piwi−1/+ or piwi−2/+ female flies were crossed to KrIf-1/+ males, the outgrowth phenotype was observed in approximately 7% progeny (n=146). The reciprocal cross (KrIf-1/+ females x piwi−1/+ or piwi−2/+ males), however, produced no offspring with the outgrowth (n=122; data not shown), indicating that maternal Piwi mediates canalization in a dose-sensitive fashion.

If canalization is solely mediated by maternal Piwi, it should be independent of the genotype of the progeny. Instead we found that the expression of the outgrowth phenotype also depends on the presence of piwi mutation in the progeny, since only KrIf-1/ piwi−2, but not their KrIf-1/+ siblings, express the phenotype (Figure 1B). These data indicate that zygotic Piwi also plays a role in canalization and that both piwi−1 and piwi−2 produce the same phenotype as the loss-of-function alleles of Hsp83 and the trithorax group of genes3.

To further link the outgrowth phenotype to the ectopic expression of genes underlying the outgrowth, we examined the expression of wingless in eye imaginal discs of the progeny of +/+ and piwi−1/+ females. wingless is a target gene of maternal enhancers of KrIf-1-induced eye outgrowth3. When piwi−1/+ or piwi−2/+ female flies were crossed to KrIf-1/+ males, wingless became ectopically expressed in approximately 10% of the eye imaginal discs of the progeny (Supplementary Figure 1A). This indicates that the Piwi/piRNA pathway can affect nontransposon gene expression in a dose-sensitive manner to achieve canalization.

We next examined whether canalization is specific to piwi. Reduction in maternal dose of Aubergine, another Piwi subfamily protein involved in the piRNA pathway, resulted in ~16% of progeny with the eye outgrowth phenotype (Supplementary Figure 1B). However, reduction in maternal doses of Dicer-1 or Dicer-2, key proteins in the miRNA and siRNA pathways, respectively, did not result in any eye outgrowth phenotype (Supplementary Figure 1C). These findings indicate that canalization is mediated by the piRNA pathway, but not the miRNA or siRNA pathway.

It is argued that canalization is a general feature of complex gene networks and that perturbation in any node of these networks can lead to compromised canalization17. Hence, we examined whether Piwi and Hsp90 function in the same pathway or in parallel pathways that happen to produce similar phenotypes. We tested if over-expression of maternal Piwi suppresses the eye outgrowth phenotypes of Hsp83 induced by geldanamycin, a chemical that specifically inhibits Hsp90 and induces eye outgrowths in KrIf-1 flies3. To over-express maternal Piwi, we used a transgenic myc-piwi line (G38) wherein a fully functional myc-piwi gene was inserted into the second chromosome that contains endogenous piwi13,18, thereby increasing the piwi copy number to four. We generated KrIf-1/myc-piwi virgin females (which contain three copies of piwi), and crossed them to KrIf-1/+ males to generate KrIf-1/KrIf-1 flies. In KrIf-1/KrIf-1 flies from females containing two copies of piwi, inhibition of Hsp90 by geldanamycin increased ectopic outgrowth by ~10 fold (Figure 1C; increase from 1.2±1.79% to 10.8±2.3%). However, in KrIf-1/KrIf-1 flies from females containing three copies of piwi (KrIf-1/myc-piwi; ~50% increase in maternal Piwi), the ectopic outgrowth phenotype was rescued by ~52% (from 10.8±2.3% to 5.6±1.7%). These results indicate that piwi and Hsp83 genetically interact in achieving canalization. This interaction could reflect that Hsp83 and piwi act on different pathways with additive effect towards canalization. Alternatively, it could reflect that piwi and Hsp83 function in the same pathway, with piwi downstream of Hsp83 in regulating canalization (Figure 1D).

To explore molecular mechanism underlying the Piwi-mediated canalization, we fractionated cytoplasmic extracts of 0-12 hour (h) embryos using column chromatography (Figure 2A and Supplementary Figure 2). After the final column, Piwi migrated with an apparent molecular weight of ~150kDa (Figure 2C, upper panel). The peak fraction for Piwi (# 27) was resolved using gel electrophoresis. Co-migrating peptides were visualized using silver staining (Figure 2B), excised from the gel, and identified by mass spectrometry. In addition to Piwi that migrates at ~90 kDa, another abundant protein migrating at ~60kDa was identified as Hsp70/Hsp90 Organizing Protein Homolog (Hop, CG2720; Figure 2B). Western blotting of fractions from the Superdex 200 column showed that Piwi and Hop co-migrate during size exclusion chromatography (Figure 2C). The interaction was further confirmed by coimmunoprecipitation of Piwi with Hop from 0-12h embryonic extracts (Figure 2D). Hop contains three tetratricopeptide repeats (TPR1a, TPR2a and TPR2b) and a small DP repeat motif called DP219,20. TPR1 domain of Hop binds to Hsp70 and TPR2a domain binds to Hsp9021,22. Furthermore, in support of our genetic experiments implicating Piwi as a client of the highly selective chaperone Hsp90 (Figure 1C), we found that Piwi and Hop together coimmunoprecipitate with Hsp90 (Figure 2D and E). These results indicate that Piwi, Hop, and Hsp90 likely exist in the same complex.

To seek for a second line of evidence for the Piwi-Hop-Hsp90 complex, we tried immunoprecipitating Piwi, but noted that antibody against first 200 amino acids of Piwi, failed to immunoprecipitate both Hop and Hsp90 (Figure 2D, see Piwi IP lane). This may be due to either of the two possibilities: a) Hop and Hsp90 hold Piwi in a conformation where its N-terminus is not accessible; b) the Piwi antibody recognition site is the same as Hop or Hsp90 binding. This possibility is supported by the fact that this antibody cannot co-immunoprecipitate proteins such as Heterochromatin Protein 1a that binds to residues 28-32 of Piwi12. Hence, to provide an independent line of evidence for the Piwi-Hop-Hsp90 complex, we co-expressed His-SUMO-Piwi, Myc-Hop, and HA-Hsp90 in the rabbit reticulocyte lysate system and conducted serial immunoprecipitation on the lysate (Figure 2F). After the second immunoprecipitation, we found that Piwi exists in the same pool as both Hop and Hsp90. This verified that Piwi, Hop, and Hsp90 exist in the same complex. The above findings, together, led us to hypothesize that Hsp90, Hop, and Piwi function in the same complex in which Hop mediates interaction between Piwi and Hsp90.

To test this hypothesis, we reasoned that a reduction in maternal dose of Hop would also compromise canalization. When we crossed Hopk00616/+ virgin females with KrIf-1/KrIf-1males, we observed 5-6% of the male progeny with the eye outgrowth phenotype (Figure 3). This observation confirmed our hypothesis and demonstrated that Hop is also a dominant enhancer of the KrIf-1 phenotype. The male-only outgrowth might be because some of the mutations required for the outgrowth are X-linked recessive and need hemi- or homo-zygosity to be expressed. As in the case of Piwi and Hsp90, the reciprocal cross with the Hopk00616 inherited from the father did not produce any outgrowth phenotype (data not shown). Hence, maternal Hop plays a critical role in canalization, likely by mediating interaction between Hsp90 and Piwi.

We then further examined if the outgrowth phenotype generated in piwi−1 and Hopk00616 mutants can become independent of the piwi and Hop mutations and transmitted to the next generation (Figure 4A). Males with eye outgrowths were crossed with virgin Canton-S flies to separate piwi−1 and Hopk00616 mutations from KrIf-1. The resulting heterozygous KrIf-1/+ progeny did not have any eye outgrowth. However, when we intercrossed these flies among themselves, the resulting KrIf-1/KrIf-1flies had the outgrowths. When the KrIf-1/KrIf-1flies with the outgrowths were selected and intercrossed among themselves in subsequent crosses, the percentage of both male and female flies with the outgrowths increased in each successive generation (Figure 4B). Consistent with the link between wingless expression and eye outgrowth phenotype, we observed higher wingless expression in the heads of F8 flies with the outgrowth phenotype (Figure 4C). This indicates that phenotypic variants and their corresponding gene expression patterns, once induced by piwi and Hop mutations, can be ‘fixed’ in a population and then stably inherited in subsequent generations under selection.

As Hsp90 and Piwi are in the same complex, yet over-expression of Piwi can rescue the deficiency of Hsp90 in canalization, Hsp90 and Piwi must function in the same pathway, with Piwi down stream of Hsp90. We therefore further examined how Hsp90 may regulate Piwi function. We first examined whether Hsp90 regulates Piwi expression and/or stability by comparing the Piwi levels in wildtype flies with and without geldanamycin treatment (Figure 5A), and further confirm these results in Hsp8308445/Hsp8308445 mutants (Figure 5B). As expected, the known Hsp90 client proteins Akt and B-Raf become unstable after geldanamycin treatment (Figure 5A). However, the Piwi protein levels do not change either with geldanamycin treatment or in Hsp8308445/Hsp8308445 mutants (Figure 5A and B). This indicates that Hsp90 does not regulate the expression and/or stability of Piwi.

However, Hsp90 regulates the posttranslational modification of Piwi. In wild type conditions, two-dimensional gel electrophoresis reveals three isoforms of Piwi with pI~10 (Figure 5C, first panel, isoforms 2, 3, and 4). These isoforms are likely due to different levels of phosphorylation because they have very similar molecular weights but different pI values. Upon inhibition of Hsp90 by geldanamycin, we noticed the appearance of a new isoform that is less negatively charged (Figure 5C, second panel, isoform 1 labeled by white arrow head). This indicates that that Hsp90 mediates post-translational modification of Piwi. This was further confirmed by comparing Piwi isoforms in ovary lysates from Hsp8308445/TM3 and Hsp8308445/Hsp8308445 flies. In Hsp8308445/TM3 flies with a reduced level of Hsp90, we noted the four isoforms that we initially observed in geldanamycin-treated flies, Further reduction of Hsp90 levels in Hsp8308445/Hsp8308445 flies resulted in complete absence of isoform 3 (Figure 5C, third panel, black arrow) and an appearance of a new isoform that migrates between isoforms 2 and 3, closer to isoform 2 (Figure 5C, fourth panel, black arrow head).

To test whether the posttranslational modification is indeed phosphorylation, we treated Hsp8308445/TM3 ovary lysate with calf intestinal phosphatase (CIP) and then subjecting the lysate to 2D gel analysis. After CIP treatment, we noticed complete absence of isoforms 3 and 4 (Figure 5C, bottom panel, circled area) and reduced intensity of isoforms 1 and 2. This confirms that the four isoforms are indeed phosphorylated forms of Piwi.

To further verify the phosphorylation of Piwi and determine the type of phosphorylation, we performed immunoprecipitation using anti-phospho-serine, anti-phospho-threonine, and anti-phospho-tyrosine antibodies, followed by western blotting analysis of the immuno-precipitates with anti-Piwi antibody. Piwi was immunoprecipitated by both anti-phopho-serine and anti-phospho-tyrosine antibodies, but not by the anti-phospho-threonine antibody (Figure 5D). Consistent with this, anti-phospho-serine and anti-phospho tyrosine antibodies failed to immunoprecipitate Piwi when the lysate was treated with CIP prior to immunoprecipitation (Figure 5E). These results indicate that Piwi is phosphorylated on serine and tyrosine residues.

To investigate whether the phosphorylation of serine and tyrosine residues in Piwi is dependent on Hsp90, we conducted the phospho-Piwi immunoprecipitation in wildtype and Hsp83 mutant ovarian lysates. Both anti-phopho-serine and anti-phospho-tyrosine antibodies immunoprecipitated Piwi from wildtype but not from Hsp83 mutant ovarian lysates (Figure 5F). These results indicate that Hsp90 is required for the phosphorylation of Piwi. A salient feature of Hsp90-mediated chaperoning, unlike that of Hsp70, is that it predominantly binds to metastable states of proteins instead of hydrophobic stretches7. It will be interesting to see in the future how Hsp90 binding to Piwi results in its phosphorylation and what affect this may have on the function of Piwi and canalization.

Recently, Specchia et al. suggested that Hsp90 prevents phenotypic variation by suppressing transposon-induced mutagenesis via piRNAs4. Having demonstrated that Hsp90 forms a complex with Piwi and regulates its phosphorylation, we set out to test whether this hypothesis is true. It has been observed that deficiency in the Hsp90 activity reduces piRNA expression, activates transposition, compromises multiple aspects of DNA damage repair, and increases CAG repeat instability, which ultimately produce genotype variations4,23-26. Consistent with Specchia et al, 23, we observed an increase in the RNA levels of transposons upon geldanymycin treatment (Supplementary Figure 3), even though this treatment did not decrease the mRNA and protein levels of Piwi (Supplementary Figure 3B and C). Furthermore, this increase can be largely repressed by increasing the piwi copy number to four (Supplementary Figure 3A). These data further support that a mechanism through which Hsp90 achieves canalization is the suppression of new mutations via transposition and deficiency in DNA repair.

However, our experiments suggest that Hsp90, Hop and Piwi mediate canalization also through a non-genetic mechanism. First, we found that the eye outgrowth phenotype was observed only when either piwi or Hop mutations were in the mother. This is in contrast to the recent report that geldanaymycin treatment can de-repress transposons mainly in the male germline4. If eye outgrowths resulted from a genetic lesion, it should be independent of the parental source of contribution. Second, we have observed no increase in transposon RNA levels in the female germ line of piwi−1/+ (Supplementary Figure 4A), i.e., one copy of piwi is sufficient to silence transposons, which is consistent with a previous report that piwi is haplo-sufficient to prevent new transpositions in the progeny27. However, piwi is haplo-insufficient to suppress eye outgrowths (Figure 1) as well as position effect variegation (Supplementary Figure 4B). Thus, the eye outgrowth phenotype observed in Kr(If-1)/piwi1 is unlikely due to new genetic mutations caused by transposons. Third, in KrIf-1/KrIf-1 files eight generations (F8) after piwi and Hop mutations were outcrossed, new mutations from the F1 flies, if any, should have been fixed. However, among these F8 flies, those with the outgrowth phenotype had approximately 50-60% more Kr mRNA and at least twice as much wg mRNA in their heads as compared to their siblings without the phenotype (Figure 4C). These statistically significant differences in Kr and wg expression among the same population of flies are more difficult to be explained by a stable genetic change by transposons. Last but not least, Piwi has been broadly implicated in epigenetic regulation, from long range physical interaction among Polycomb group response elements to Heterochromatin Protein 1-mediated epigenetic silencing9-14. Hence, we conclude that eye outgrowth phenotypes we observed in this study are due to defects in epigenetic silencing of normally non-expressed genotypes, so-called cryptic genotypes, by maternal Piwi rather than new transposon insertions.

The mechanism of canalization has been a subject of great debate. Rutherford and Lindquist’s findings indicate that Hsp90 acts as a capacitor for phenotypic variation5; however, a complex gene network model generated by Bergman and Siegal predicts that a mutation in any one gene can result in expression of cryptic genotypes17. Yet another report argues that expression of cryptic genotypes is not caused by canalization and no particular mechanism is needed to prevent expression of the cryptic phenotypes 28. Our finding of piwi and Hop mutations as enhancers for expression of cryptic genotypes validates the existence of a piRNA-pathway dependent mechanism for preventing phenotypic variation. Piwi is a piRNA-binding protein that is required for silencing of transposons29 and epigenetic regulation13,30. Hence, post-translational regulation of Piwi by Hsp90 and Hop may allow Piwi both suppress the generation of new genotypes and epigenetically silence the expression of existing genetic variants (Figure 6). Both mechanisms can be ‘fixed’ and inherited in subsequent generations. Our study also shows that Piwi acts at two distinct phases of fly development in mediating phenotypic capacitance. First, maternal Piwi plays a direct role in canalization and/or suppresses transposon-induced mutagenesis during embryogenesis. This allows the inheritance of correct epigenetic and genetic codes from parental cells to daughter cells, thereby ensuring the robustness of the developmental programs. Subsequently, zygotic Piwi is required for maintaining the inherited developmental programs during subsequent stages of development. This Piwi function likely represents the piRNA pathway, since Aubergine, a Piwi homolog also involved in the piRNA pathway, has similar function. Furthermore, canalization appears to involve only the piRNA pathway, but not miRNA or siRNA pathway, since Dicer 1 and 2 deficiency does not lead to increased eye outgrowth.

Then, what could be the roles for Hsp90 and Hop in Piwi-mediated canalization? As an essential component in canalization, Hsp90 likely ensures proper function of its clients involved in canalization, such as Piwi, by mediating their proper post-translational modification, such as phosphorylation, that is required for their molecular activities. A salient feature of Hsp90-mediated chaperoning, unlike Hsp70, is that Hsp90 predominantly binds to metastable states of proteins instead of hydrophobic stretches7. The Hsp90-bound metastable state of Piwi may be a necessary step for its phosphorylation at proper sites, which may then be required to form active complexes with piRNAs and/or epigenetic factors to promote epigenetic and transposon silencing, leading to canalization.

METHODSFly stocks and maintenanceAll fly stocks were maintained at 25°C. piwi−1, piwi−2 alleles and myc-piwi transgene were generated by P-element insertions and are described elsewhere18,31. b1 KrIf-1 (#4194), w1118; aubQC42 cn1 bw1/CyO, P{ry+t7.2=sevRas1.V12}FK1 (# 4968), y1 w67c23; P{w+mC=lacW}Hopk00616/CyO (#10483), P{ry+t7.2=PZ}Hsp8308445 ry506/TM3, ryRK Sb1 Ser1 (#11797) were obtained from Bloomington stock center. The numbers in parentheses represent the stock IDs at the Bloomington stock center. yw eyFLP;FRT82B Dcr-1Q1147X/TM3 and yw eyFLP;FRT42D Dcr-2L811fsX/Cyo strains were gifts from Richard Carthew and are described elsewhere32.Pharmacological inhibition of Hsp83Geldanamycin (Sigma) was first dissolved in 50% ethyl alcohol to make a 1000X stock (3.56mM). To make Drosophila medium containing geldanamycin, a scoop of Formula 4-24 (Carolina biological supply), was mixed with 15ml of water containing 3.56μM geldanamycin. After mixing food and water in a vial, it was left to solidify for a minute before adding flies. Alternatively, geldanamycin was dissolved in DMSO to make a stock of 10mM from where fly medium containing geldanamycin was prepared as described above. For inhibiting Hsp90 in vivo we allowed female flies to feed on geldanamycin containing medium for either 2 days (Supplementary Figure 4A) or 4 days (Figure 5A) prior to dissecting ovaries and lysate preparation.Embryo collection and extract preparationw1118 flies were grown at 25°C and 70-80% relative humidity in population cages and embryos were collected on grape juice agar plates at 12-hour intervals. Embryos were carefully transferred using a soft paint brush into mini embryo collection cages (Genesee Scientific) and washed well with tap water to remove traces of yeast. The embryos were then dechorionated by immersing in 50% bleach (Chlorox) for 2 min and rinsed with embryo wash buffer [0.7% NaCl (wt/vol); 0.02% Triton X-100 (vol/vol)] and tap water. They were then dried on a paper towel and stored at −80°C until further use.All steps in extract preparation were performed at 4°C. Ten grams of frozen embryos were thawed on ice and carefully resuspended in 3 ml per gram of embryos of H (0.3) buffer (25 mM HEPES-NaOH, pH 7.8, 300 mM NaCl, 0.5 mM EGTA, 0.1 mM EDTA, 2 mM MgCl2, 0.02% NP-40 and 20% glycerol) where the number in the parenthesis represents the final NaCl concentration in the buffer (0.3 M). Just before use, the buffer was supplemented with 1 tablet of Complete Mini-EDTA free protease inhibitor cocktail (Roche) per 10ml of the buffer, 1mM phenylmethanesulfonyl fluoride (Fluka), 2mM 2-mercaptoethanol (Sigma) and 0.5mM sodium metabisulfite. The resuspended embryos were transferred 1ml at a time into a Wheaton homogenizer and were lysed using 10 strokes of loose pestle A and 10 strokes of tight pestle B. The homogenate was then centrifuged at 14,000 rpm for 30 min using SW 40 Ti rotor and Optima L-XP series preparative ultra centrifuge (Beckman) to pellet the nuclei and debris. Supernatant was carefully collected and again centrifuged at 100,000g for 4h to further clarify the extract. Supernatant (cytoplasmic fraction) was collected and stored at −80°C until further use.Biochemical purification and identification of cytoplasmic Piwi interacting peptidesAll chromatographic steps were performed using ÄKTA purifier (GE Healthcare) at 4°C. After each step, Piwi was detected by western blotting. Cytoplasmic extract was diluted with buffer H(0) to bring down the final salt concentration of NaCl to 100mM. The extract was then fractionated on a DEAE Sepharose (GE Healthcare) column over a linear gradient of 100-1000mM NaCl. Piwi was detected in the flow through of the column showing that Piwi does not bind to this resin. Flow through was fractionated on 10ml Bio-Rex70 (Bio-Rad) column over a linear gradient of 100-1000mM NaCl. Fractions containing Piwi were again pooled; final concentration was brought down to 100mM and fractionated on HiPrep 16/10 Heparin FF column (GE Healthcare) over a linear gradient of 100-1000mM NaCl and 7ml fractions were collected. Fractions containing Piwi were pooled and concentrated by fractionation on 1ml HiTrap Heparin FF column. Three 1ml fractions containing Piwi were pooled and 500μl aliquots were fractionated using Superdex 200 size exclusion chromatography. Fraction number 27 was first resolved using 4-20% gradient polyacrylamide gel electrophoresis and stained with coomassie blue staining. Bands detected were excised and then identified by mass spectrometry. In addition, liquid sample was also analyzed in order to identify peptides that were missed by coomassie blue staining. Mass spectrometry analysis was performed using services provided by ProtTech, Inc located in Norristown, PA, USA.Whole ovary lysate preparationFlies, either wild type or with desired genotypes as shown in individual figures, were yeast fed for 36 h prior to ovary dissections. Ovaries were first dissected into ice cold phosphate buffered saline (PBS) and transferred into eppendorf tubes. PBS was exchanged with ovary lysis buffer (20 mM HEPES pH 7.5, 100 mM KCl, 5 mM MgCl2, 0.1% SDS, 0.1% sodium deoxycholate, 1% Triton X-100, 1 mM dithiothreitol and 5% glycerol) containing 1 tablet of complete mini EDTA free protease inhibitor cocktail (Roche) per 10 ml of lysate and 1mM PMSF. Lysis was performed manually using pestles designed for eppendorf tubes. Lysate was then cleared by spinning at 14,000rpm for 30min. Cleared lysate was carefully separated from the pellet and floating fat and stored at −80°C until use.In vitro reconstitution of Piwi, Hop and Hsp90 complexN-terminal His-Sumo tagged Piwi was generated by cloning Piwi CDS into pETSUMO vector(Invitrogen). N-terminal Myc and HA tagged versions of Hop and Hsp90 CDS respectively were generated by PCR and cloned into pCR2.1 (Invitrogen) vector to create pHop and pHsp90 plasmids. Equimolar concentrations of pETSUMO-Piwi, pHop and pHsp90 were added to TnT T7 coupled rabbit reticulocyte lysate system (Promega) and were co-expressed following manufacturer’s protocol. Prior to Myc IP, co-expression lysate was diluted to 500μl with H(0.1) buffer. Myc-Hop was first immunoprecipitated using 50μl EZview Red Anti-cMyc Affinity gel (Sigma) suspension. After 3h incubation at 4°C, beads were washed thrice with 750μl of H(0.1) buffer. Myc-Hop was then eluted using 100μl of 100μg/ml c-Myc peptide (Sigma). The eluted sample was then diluted to 500μl using H(0.1) buffer and used as input for HA tag IP using EZ view Red Anti-HA affinity gel (Sigma). IP procedure was the same as for Myc IP. After the final wash, beads were resuspended in 50μl 2X SDS sample loading buffer (62.5 mM Tris HCl pH 6.8, 25% glycerol, 2% SDS, 5% β-mercaptoethanol and 0.01% bromophenol blue) and proceeded with standard western blotting protocol.Immunoprecipitation and western blottingFor Figures 2D and 2E, cytoplasmic extracts from 0-12h embryos were prepared in buffer H (0.1) buffer as described in extract preparation section. A total of 1mg of total protein in a volume of 1ml was used for each immunoprecipitation reaction. The lysates were pre-cleared using Protein A/G PLUS-Agarose (Santa Cruz) for 1h at 4°C. Pre-cleared lysates were incubated with appropriate antibody over night at 4°C with gentle agitation. Twenty μl of beads were then added to the lysate-antibody mixture and incubated further for 3h at 4°C. Beads were then washed thrice with 1ml of H(0.1) buffer and finally analyzed by western blotting.For IP using anti-phospho antibodies, 15 pairs of ovaries for each genotype were lysed in 50μl ovary lysis buffer. SDS was then added to a final concentration of 1% and the lysate was then boiled at 95°C for 10minutes to disrupt protein-protein interactions and also to inactivate any dephosphatases. The lysate was then centrifuged at 14000rpm for 5min and the supernatant was diluted 10X with H(0.1) buffer prior to immunoprecipitation.For calf intestinal phosphatase (CIP) treatment, NEB buffer 3 (NEB) was added to the ovary lysate to 1X final concentration. CIP (NEB) was then added to a final concentration of 1unit/μg of protein and incubated at 37°C for 60min prior to immunoprecipitation.2D gel electrophoresis2D gel electrophoresis was performed following manufacturer’s protocol provided for ReadyPrep™ 2-D Starter kit (Bio-Rad). For each experiment, 150μg of total ovary lysate was used. The samples were concentrated either by freeze drying or TCA precipitation and resuspended in 125μl of 2D rehydration buffer. The samples are then applied to pH 3-10 IPG strips provided in the kit (BioRad). After overnight rehydration step, the strips were electro focused using PROTEAN IEF Cell (BioRad). Following electro-focusing, second dimension gel electrophoresis was performed and Piwi was detected by western blotting.AntibodiesAnti Piwi, Aub and Ago3 antibodies, all mouse monoclonal, were a kind gift from Haruhiko Siomi and were used at a concentration of 1:10, 1:5000 and 1:5000 respectively. Anti Hop antibody (rabbit) was provided by Dr. Michael Chinkers and was used at a dilution of 1:1000. Hsp90 antibodies were obtained either from Santa Cruz biotechnologies or Cell Signaling and were used as recommended. Immunoprecipitation of Hsp90 as shown in figure 2E was performed using antibody obtained from Santa Cruz biotechnologies. Mouse monoclonal antibody for anti-phospho-serine was obtained from Sigma and was used at a dilution of 1:100 for IP. Anti-phospho-threonine and anti-phospho-serine antibodies were obtained from Cell Signaling and were used at dilutions of 1:50 and 1:100 respectively for IP.Total RNA extraction from ovaries and real time quantitative PCRTen pairs of ovaries were dissected into ice cold Buffer B from flies that have been fed on yeast for at least 36 h. For checking the effect of geldanamycin, flies were fed on geldanamycin containing medium for 2 days before dissecting the ovaries. Total RNA was then extracted using TRIzol™ (Invitrogen) and then cleaned up using RNeasy kit (Qiagen) following manufacturers’ protocols. RNA was finally eluted in 30μl nuclease free water and quantified by Nanodrop. One μg of eluted RNA was used to generate cDNA in a 20μl reaction using High Capacity cDNA Reverse Transcription kit (Applied Biosystems). Prepared cDNA was diluted 5X in nuclease free water, 1μl of which was used for quantitative PCR using primer sets mentioned in Supplementary Table 1. Quantitative PCR was performed using iQ SYBR green super mix and CFX96 real time PCR detection system (both from Bio-Rad).

Fly stocks and maintenanceAll fly stocks were maintained at 25°C. piwi−1, piwi−2 alleles and myc-piwi transgene were generated by P-element insertions and are described elsewhere18,31. b1 KrIf-1 (#4194), w1118; aubQC42 cn1 bw1/CyO, P{ry+t7.2=sevRas1.V12}FK1 (# 4968), y1 w67c23; P{w+mC=lacW}Hopk00616/CyO (#10483), P{ry+t7.2=PZ}Hsp8308445 ry506/TM3, ryRK Sb1 Ser1 (#11797) were obtained from Bloomington stock center. The numbers in parentheses represent the stock IDs at the Bloomington stock center. yw eyFLP;FRT82B Dcr-1Q1147X/TM3 and yw eyFLP;FRT42D Dcr-2L811fsX/Cyo strains were gifts from Richard Carthew and are described elsewhere32.

Fly stocks and maintenance

All fly stocks were maintained at 25°C. piwi−1, piwi−2 alleles and myc-piwi transgene were generated by P-element insertions and are described elsewhere18,31. b1 KrIf-1 (#4194), w1118; aubQC42 cn1 bw1/CyO, P{ry+t7.2=sevRas1.V12}FK1 (# 4968), y1 w67c23; P{w+mC=lacW}Hopk00616/CyO (#10483), P{ry+t7.2=PZ}Hsp8308445 ry506/TM3, ryRK Sb1 Ser1 (#11797) were obtained from Bloomington stock center. The numbers in parentheses represent the stock IDs at the Bloomington stock center. yw eyFLP;FRT82B Dcr-1Q1147X/TM3 and yw eyFLP;FRT42D Dcr-2L811fsX/Cyo strains were gifts from Richard Carthew and are described elsewhere32.

Pharmacological inhibition of Hsp83Geldanamycin (Sigma) was first dissolved in 50% ethyl alcohol to make a 1000X stock (3.56mM). To make Drosophila medium containing geldanamycin, a scoop of Formula 4-24 (Carolina biological supply), was mixed with 15ml of water containing 3.56μM geldanamycin. After mixing food and water in a vial, it was left to solidify for a minute before adding flies. Alternatively, geldanamycin was dissolved in DMSO to make a stock of 10mM from where fly medium containing geldanamycin was prepared as described above. For inhibiting Hsp90 in vivo we allowed female flies to feed on geldanamycin containing medium for either 2 days (Supplementary Figure 4A) or 4 days (Figure 5A) prior to dissecting ovaries and lysate preparation.

Pharmacological inhibition of Hsp83

Geldanamycin (Sigma) was first dissolved in 50% ethyl alcohol to make a 1000X stock (3.56mM). To make Drosophila medium containing geldanamycin, a scoop of Formula 4-24 (Carolina biological supply), was mixed with 15ml of water containing 3.56μM geldanamycin. After mixing food and water in a vial, it was left to solidify for a minute before adding flies. Alternatively, geldanamycin was dissolved in DMSO to make a stock of 10mM from where fly medium containing geldanamycin was prepared as described above. For inhibiting Hsp90 in vivo we allowed female flies to feed on geldanamycin containing medium for either 2 days (Supplementary Figure 4A) or 4 days (Figure 5A) prior to dissecting ovaries and lysate preparation.

Embryo collection and extract preparationw1118 flies were grown at 25°C and 70-80% relative humidity in population cages and embryos were collected on grape juice agar plates at 12-hour intervals. Embryos were carefully transferred using a soft paint brush into mini embryo collection cages (Genesee Scientific) and washed well with tap water to remove traces of yeast. The embryos were then dechorionated by immersing in 50% bleach (Chlorox) for 2 min and rinsed with embryo wash buffer [0.7% NaCl (wt/vol); 0.02% Triton X-100 (vol/vol)] and tap water. They were then dried on a paper towel and stored at −80°C until further use.All steps in extract preparation were performed at 4°C. Ten grams of frozen embryos were thawed on ice and carefully resuspended in 3 ml per gram of embryos of H (0.3) buffer (25 mM HEPES-NaOH, pH 7.8, 300 mM NaCl, 0.5 mM EGTA, 0.1 mM EDTA, 2 mM MgCl2, 0.02% NP-40 and 20% glycerol) where the number in the parenthesis represents the final NaCl concentration in the buffer (0.3 M). Just before use, the buffer was supplemented with 1 tablet of Complete Mini-EDTA free protease inhibitor cocktail (Roche) per 10ml of the buffer, 1mM phenylmethanesulfonyl fluoride (Fluka), 2mM 2-mercaptoethanol (Sigma) and 0.5mM sodium metabisulfite. The resuspended embryos were transferred 1ml at a time into a Wheaton homogenizer and were lysed using 10 strokes of loose pestle A and 10 strokes of tight pestle B. The homogenate was then centrifuged at 14,000 rpm for 30 min using SW 40 Ti rotor and Optima L-XP series preparative ultra centrifuge (Beckman) to pellet the nuclei and debris. Supernatant was carefully collected and again centrifuged at 100,000g for 4h to further clarify the extract. Supernatant (cytoplasmic fraction) was collected and stored at −80°C until further use.

Embryo collection and extract preparation

w1118 flies were grown at 25°C and 70-80% relative humidity in population cages and embryos were collected on grape juice agar plates at 12-hour intervals. Embryos were carefully transferred using a soft paint brush into mini embryo collection cages (Genesee Scientific) and washed well with tap water to remove traces of yeast. The embryos were then dechorionated by immersing in 50% bleach (Chlorox) for 2 min and rinsed with embryo wash buffer [0.7% NaCl (wt/vol); 0.02% Triton X-100 (vol/vol)] and tap water. They were then dried on a paper towel and stored at −80°C until further use.

All steps in extract preparation were performed at 4°C. Ten grams of frozen embryos were thawed on ice and carefully resuspended in 3 ml per gram of embryos of H (0.3) buffer (25 mM HEPES-NaOH, pH 7.8, 300 mM NaCl, 0.5 mM EGTA, 0.1 mM EDTA, 2 mM MgCl2, 0.02% NP-40 and 20% glycerol) where the number in the parenthesis represents the final NaCl concentration in the buffer (0.3 M). Just before use, the buffer was supplemented with 1 tablet of Complete Mini-EDTA free protease inhibitor cocktail (Roche) per 10ml of the buffer, 1mM phenylmethanesulfonyl fluoride (Fluka), 2mM 2-mercaptoethanol (Sigma) and 0.5mM sodium metabisulfite. The resuspended embryos were transferred 1ml at a time into a Wheaton homogenizer and were lysed using 10 strokes of loose pestle A and 10 strokes of tight pestle B. The homogenate was then centrifuged at 14,000 rpm for 30 min using SW 40 Ti rotor and Optima L-XP series preparative ultra centrifuge (Beckman) to pellet the nuclei and debris. Supernatant was carefully collected and again centrifuged at 100,000g for 4h to further clarify the extract. Supernatant (cytoplasmic fraction) was collected and stored at −80°C until further use.

Biochemical purification and identification of cytoplasmic Piwi interacting peptidesAll chromatographic steps were performed using ÄKTA purifier (GE Healthcare) at 4°C. After each step, Piwi was detected by western blotting. Cytoplasmic extract was diluted with buffer H(0) to bring down the final salt concentration of NaCl to 100mM. The extract was then fractionated on a DEAE Sepharose (GE Healthcare) column over a linear gradient of 100-1000mM NaCl. Piwi was detected in the flow through of the column showing that Piwi does not bind to this resin. Flow through was fractionated on 10ml Bio-Rex70 (Bio-Rad) column over a linear gradient of 100-1000mM NaCl. Fractions containing Piwi were again pooled; final concentration was brought down to 100mM and fractionated on HiPrep 16/10 Heparin FF column (GE Healthcare) over a linear gradient of 100-1000mM NaCl and 7ml fractions were collected. Fractions containing Piwi were pooled and concentrated by fractionation on 1ml HiTrap Heparin FF column. Three 1ml fractions containing Piwi were pooled and 500μl aliquots were fractionated using Superdex 200 size exclusion chromatography. Fraction number 27 was first resolved using 4-20% gradient polyacrylamide gel electrophoresis and stained with coomassie blue staining. Bands detected were excised and then identified by mass spectrometry. In addition, liquid sample was also analyzed in order to identify peptides that were missed by coomassie blue staining. Mass spectrometry analysis was performed using services provided by ProtTech, Inc located in Norristown, PA, USA.

Biochemical purification and identification of cytoplasmic Piwi interacting peptides

All chromatographic steps were performed using ÄKTA purifier (GE Healthcare) at 4°C. After each step, Piwi was detected by western blotting. Cytoplasmic extract was diluted with buffer H(0) to bring down the final salt concentration of NaCl to 100mM. The extract was then fractionated on a DEAE Sepharose (GE Healthcare) column over a linear gradient of 100-1000mM NaCl. Piwi was detected in the flow through of the column showing that Piwi does not bind to this resin. Flow through was fractionated on 10ml Bio-Rex70 (Bio-Rad) column over a linear gradient of 100-1000mM NaCl. Fractions containing Piwi were again pooled; final concentration was brought down to 100mM and fractionated on HiPrep 16/10 Heparin FF column (GE Healthcare) over a linear gradient of 100-1000mM NaCl and 7ml fractions were collected. Fractions containing Piwi were pooled and concentrated by fractionation on 1ml HiTrap Heparin FF column. Three 1ml fractions containing Piwi were pooled and 500μl aliquots were fractionated using Superdex 200 size exclusion chromatography. Fraction number 27 was first resolved using 4-20% gradient polyacrylamide gel electrophoresis and stained with coomassie blue staining. Bands detected were excised and then identified by mass spectrometry. In addition, liquid sample was also analyzed in order to identify peptides that were missed by coomassie blue staining. Mass spectrometry analysis was performed using services provided by ProtTech, Inc located in Norristown, PA, USA.

Whole ovary lysate preparationFlies, either wild type or with desired genotypes as shown in individual figures, were yeast fed for 36 h prior to ovary dissections. Ovaries were first dissected into ice cold phosphate buffered saline (PBS) and transferred into eppendorf tubes. PBS was exchanged with ovary lysis buffer (20 mM HEPES pH 7.5, 100 mM KCl, 5 mM MgCl2, 0.1% SDS, 0.1% sodium deoxycholate, 1% Triton X-100, 1 mM dithiothreitol and 5% glycerol) containing 1 tablet of complete mini EDTA free protease inhibitor cocktail (Roche) per 10 ml of lysate and 1mM PMSF. Lysis was performed manually using pestles designed for eppendorf tubes. Lysate was then cleared by spinning at 14,000rpm for 30min. Cleared lysate was carefully separated from the pellet and floating fat and stored at −80°C until use.

Whole ovary lysate preparation

Flies, either wild type or with desired genotypes as shown in individual figures, were yeast fed for 36 h prior to ovary dissections. Ovaries were first dissected into ice cold phosphate buffered saline (PBS) and transferred into eppendorf tubes. PBS was exchanged with ovary lysis buffer (20 mM HEPES pH 7.5, 100 mM KCl, 5 mM MgCl2, 0.1% SDS, 0.1% sodium deoxycholate, 1% Triton X-100, 1 mM dithiothreitol and 5% glycerol) containing 1 tablet of complete mini EDTA free protease inhibitor cocktail (Roche) per 10 ml of lysate and 1mM PMSF. Lysis was performed manually using pestles designed for eppendorf tubes. Lysate was then cleared by spinning at 14,000rpm for 30min. Cleared lysate was carefully separated from the pellet and floating fat and stored at −80°C until use.

In vitro reconstitution of Piwi, Hop and Hsp90 complexN-terminal His-Sumo tagged Piwi was generated by cloning Piwi CDS into pETSUMO vector(Invitrogen). N-terminal Myc and HA tagged versions of Hop and Hsp90 CDS respectively were generated by PCR and cloned into pCR2.1 (Invitrogen) vector to create pHop and pHsp90 plasmids. Equimolar concentrations of pETSUMO-Piwi, pHop and pHsp90 were added to TnT T7 coupled rabbit reticulocyte lysate system (Promega) and were co-expressed following manufacturer’s protocol. Prior to Myc IP, co-expression lysate was diluted to 500μl with H(0.1) buffer. Myc-Hop was first immunoprecipitated using 50μl EZview Red Anti-cMyc Affinity gel (Sigma) suspension. After 3h incubation at 4°C, beads were washed thrice with 750μl of H(0.1) buffer. Myc-Hop was then eluted using 100μl of 100μg/ml c-Myc peptide (Sigma). The eluted sample was then diluted to 500μl using H(0.1) buffer and used as input for HA tag IP using EZ view Red Anti-HA affinity gel (Sigma). IP procedure was the same as for Myc IP. After the final wash, beads were resuspended in 50μl 2X SDS sample loading buffer (62.5 mM Tris HCl pH 6.8, 25% glycerol, 2% SDS, 5% β-mercaptoethanol and 0.01% bromophenol blue) and proceeded with standard western blotting protocol.

In vitro reconstitution of Piwi, Hop and Hsp90 complex

N-terminal His-Sumo tagged Piwi was generated by cloning Piwi CDS into pETSUMO vector(Invitrogen). N-terminal Myc and HA tagged versions of Hop and Hsp90 CDS respectively were generated by PCR and cloned into pCR2.1 (Invitrogen) vector to create pHop and pHsp90 plasmids. Equimolar concentrations of pETSUMO-Piwi, pHop and pHsp90 were added to TnT T7 coupled rabbit reticulocyte lysate system (Promega) and were co-expressed following manufacturer’s protocol. Prior to Myc IP, co-expression lysate was diluted to 500μl with H(0.1) buffer. Myc-Hop was first immunoprecipitated using 50μl EZview Red Anti-cMyc Affinity gel (Sigma) suspension. After 3h incubation at 4°C, beads were washed thrice with 750μl of H(0.1) buffer. Myc-Hop was then eluted using 100μl of 100μg/ml c-Myc peptide (Sigma). The eluted sample was then diluted to 500μl using H(0.1) buffer and used as input for HA tag IP using EZ view Red Anti-HA affinity gel (Sigma). IP procedure was the same as for Myc IP. After the final wash, beads were resuspended in 50μl 2X SDS sample loading buffer (62.5 mM Tris HCl pH 6.8, 25% glycerol, 2% SDS, 5% β-mercaptoethanol and 0.01% bromophenol blue) and proceeded with standard western blotting protocol.

Immunoprecipitation and western blottingFor Figures 2D and 2E, cytoplasmic extracts from 0-12h embryos were prepared in buffer H (0.1) buffer as described in extract preparation section. A total of 1mg of total protein in a volume of 1ml was used for each immunoprecipitation reaction. The lysates were pre-cleared using Protein A/G PLUS-Agarose (Santa Cruz) for 1h at 4°C. Pre-cleared lysates were incubated with appropriate antibody over night at 4°C with gentle agitation. Twenty μl of beads were then added to the lysate-antibody mixture and incubated further for 3h at 4°C. Beads were then washed thrice with 1ml of H(0.1) buffer and finally analyzed by western blotting.For IP using anti-phospho antibodies, 15 pairs of ovaries for each genotype were lysed in 50μl ovary lysis buffer. SDS was then added to a final concentration of 1% and the lysate was then boiled at 95°C for 10minutes to disrupt protein-protein interactions and also to inactivate any dephosphatases. The lysate was then centrifuged at 14000rpm for 5min and the supernatant was diluted 10X with H(0.1) buffer prior to immunoprecipitation.For calf intestinal phosphatase (CIP) treatment, NEB buffer 3 (NEB) was added to the ovary lysate to 1X final concentration. CIP (NEB) was then added to a final concentration of 1unit/μg of protein and incubated at 37°C for 60min prior to immunoprecipitation.

Immunoprecipitation and western blotting

For Figures 2D and 2E, cytoplasmic extracts from 0-12h embryos were prepared in buffer H (0.1) buffer as described in extract preparation section. A total of 1mg of total protein in a volume of 1ml was used for each immunoprecipitation reaction. The lysates were pre-cleared using Protein A/G PLUS-Agarose (Santa Cruz) for 1h at 4°C. Pre-cleared lysates were incubated with appropriate antibody over night at 4°C with gentle agitation. Twenty μl of beads were then added to the lysate-antibody mixture and incubated further for 3h at 4°C. Beads were then washed thrice with 1ml of H(0.1) buffer and finally analyzed by western blotting.

For IP using anti-phospho antibodies, 15 pairs of ovaries for each genotype were lysed in 50μl ovary lysis buffer. SDS was then added to a final concentration of 1% and the lysate was then boiled at 95°C for 10minutes to disrupt protein-protein interactions and also to inactivate any dephosphatases. The lysate was then centrifuged at 14000rpm for 5min and the supernatant was diluted 10X with H(0.1) buffer prior to immunoprecipitation.

For calf intestinal phosphatase (CIP) treatment, NEB buffer 3 (NEB) was added to the ovary lysate to 1X final concentration. CIP (NEB) was then added to a final concentration of 1unit/μg of protein and incubated at 37°C for 60min prior to immunoprecipitation.

2D gel electrophoresis2D gel electrophoresis was performed following manufacturer’s protocol provided for ReadyPrep™ 2-D Starter kit (Bio-Rad). For each experiment, 150μg of total ovary lysate was used. The samples were concentrated either by freeze drying or TCA precipitation and resuspended in 125μl of 2D rehydration buffer. The samples are then applied to pH 3-10 IPG strips provided in the kit (BioRad). After overnight rehydration step, the strips were electro focused using PROTEAN IEF Cell (BioRad). Following electro-focusing, second dimension gel electrophoresis was performed and Piwi was detected by western blotting.

2D gel electrophoresis

2D gel electrophoresis was performed following manufacturer’s protocol provided for ReadyPrep™ 2-D Starter kit (Bio-Rad). For each experiment, 150μg of total ovary lysate was used. The samples were concentrated either by freeze drying or TCA precipitation and resuspended in 125μl of 2D rehydration buffer. The samples are then applied to pH 3-10 IPG strips provided in the kit (BioRad). After overnight rehydration step, the strips were electro focused using PROTEAN IEF Cell (BioRad). Following electro-focusing, second dimension gel electrophoresis was performed and Piwi was detected by western blotting.

AntibodiesAnti Piwi, Aub and Ago3 antibodies, all mouse monoclonal, were a kind gift from Haruhiko Siomi and were used at a concentration of 1:10, 1:5000 and 1:5000 respectively. Anti Hop antibody (rabbit) was provided by Dr. Michael Chinkers and was used at a dilution of 1:1000. Hsp90 antibodies were obtained either from Santa Cruz biotechnologies or Cell Signaling and were used as recommended. Immunoprecipitation of Hsp90 as shown in figure 2E was performed using antibody obtained from Santa Cruz biotechnologies. Mouse monoclonal antibody for anti-phospho-serine was obtained from Sigma and was used at a dilution of 1:100 for IP. Anti-phospho-threonine and anti-phospho-serine antibodies were obtained from Cell Signaling and were used at dilutions of 1:50 and 1:100 respectively for IP.

Anti Piwi, Aub and Ago3 antibodies, all mouse monoclonal, were a kind gift from Haruhiko Siomi and were used at a concentration of 1:10, 1:5000 and 1:5000 respectively. Anti Hop antibody (rabbit) was provided by Dr. Michael Chinkers and was used at a dilution of 1:1000. Hsp90 antibodies were obtained either from Santa Cruz biotechnologies or Cell Signaling and were used as recommended. Immunoprecipitation of Hsp90 as shown in figure 2E was performed using antibody obtained from Santa Cruz biotechnologies. Mouse monoclonal antibody for anti-phospho-serine was obtained from Sigma and was used at a dilution of 1:100 for IP. Anti-phospho-threonine and anti-phospho-serine antibodies were obtained from Cell Signaling and were used at dilutions of 1:50 and 1:100 respectively for IP.

Total RNA extraction from ovaries and real time quantitative PCRTen pairs of ovaries were dissected into ice cold Buffer B from flies that have been fed on yeast for at least 36 h. For checking the effect of geldanamycin, flies were fed on geldanamycin containing medium for 2 days before dissecting the ovaries. Total RNA was then extracted using TRIzol™ (Invitrogen) and then cleaned up using RNeasy kit (Qiagen) following manufacturers’ protocols. RNA was finally eluted in 30μl nuclease free water and quantified by Nanodrop. One μg of eluted RNA was used to generate cDNA in a 20μl reaction using High Capacity cDNA Reverse Transcription kit (Applied Biosystems). Prepared cDNA was diluted 5X in nuclease free water, 1μl of which was used for quantitative PCR using primer sets mentioned in Supplementary Table 1. Quantitative PCR was performed using iQ SYBR green super mix and CFX96 real time PCR detection system (both from Bio-Rad).

Total RNA extraction from ovaries and real time quantitative PCR

Ten pairs of ovaries were dissected into ice cold Buffer B from flies that have been fed on yeast for at least 36 h. For checking the effect of geldanamycin, flies were fed on geldanamycin containing medium for 2 days before dissecting the ovaries. Total RNA was then extracted using TRIzol™ (Invitrogen) and then cleaned up using RNeasy kit (Qiagen) following manufacturers’ protocols. RNA was finally eluted in 30μl nuclease free water and quantified by Nanodrop. One μg of eluted RNA was used to generate cDNA in a 20μl reaction using High Capacity cDNA Reverse Transcription kit (Applied Biosystems). Prepared cDNA was diluted 5X in nuclease free water, 1μl of which was used for quantitative PCR using primer sets mentioned in Supplementary Table 1. Quantitative PCR was performed using iQ SYBR green super mix and CFX96 real time PCR detection system (both from Bio-Rad).

Supplementary Material1

Supplementary Material
