---
pmid: '27501151'
title: Structural basis for the antifolding activity of a molecular chaperone.
authors:
- Huang C
- Rossi P
- Saio T
- Kalodimos CG
journal: Nature
year: '2016'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC5161705
doi: 10.1038/nature18965
---

# Structural basis for the antifolding activity of a molecular chaperone.
**Authors:** Huang C, Rossi P, Saio T, Kalodimos CG
**Journal:** Nature (2016)
**DOI:** [10.1038/nature18965](https://doi.org/10.1038/nature18965)
**PMC:** [PMC5161705](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5161705/)

## Abstract

1. Nature. 2016 Sep 8;537(7619):202-206. doi: 10.1038/nature18965. Epub 2016 Aug
8.

Structural basis for the antifolding activity of a molecular chaperone.

Huang C(1), Rossi P(1), Saio T(1), Kalodimos CG(1).

Author information:
(1)Department of Biochemistry, Molecular Biology &Biophysics, University of 
Minnesota, Minneapolis, Minnesota 55455, USA.

Molecular chaperones act on non-native proteins in the cell to prevent their 
aggregation, premature folding or misfolding. Different chaperones often exert 
distinct effects, such as acceleration or delay of folding, on client proteins 
via mechanisms that are poorly understood. Here we report the solution structure 
of SecB, a chaperone that exhibits strong antifolding activity, in complex with 
alkaline phosphatase and maltose-binding protein captured in their unfolded 
states. SecB uses long hydrophobic grooves that run around its disk-like shape 
to recognize and bind to multiple hydrophobic segments across the length of 
non-native proteins. The multivalent binding mode results in proteins wrapping 
around SecB. This unique complex architecture alters the kinetics of protein 
binding to SecB and confers strong antifolding activity on the chaperone. The 
data show how the different architectures of chaperones result in distinct 
binding modes with non-native proteins that ultimately define the activity of 
the chaperone.

DOI: 10.1038/nature18965
PMCID: PMC5161705
PMID: 27501151 [Indexed for MEDLINE]

## Full Text

Molecular chaperones rescue non-native proteins in the cell from aggregation and assist with their folding or unfolding to maintain a functional proteome1–4. Despite common features, different families of chaperones exhibit distinct activity and biological function5. Chaperones may exhibit foldase activity, whereby they accelerate folding of client proteins, or antifolding (holdase) activity, whereby they delay folding of client proteins, and the strength of the activity can vary significantly1,2. Molecular chaperones come in different sizes and a great variety of molecular shapes1–3. However, scarcity of structural data of chaperones in complex with non-native proteins has impeded an understanding of how different chaperones engage these proteins and how distinct chaperone architectures may alter activity.

SecB is a multitasking molecular chaperone in the cytosol that exhibits an unusually strong antifolding activity6. SecB is responsible for maintaining secretory proteins in an unfolded, secretion-competent state7–10, as well as for their targeted delivery to the SecA ATPase7,11. SecB also acts as a generalized chaperone in the cell12–17 and a secB null mutation results in severe protein aggregation15,18. Although extensively studied by biochemical and biophysical techniques6,10,19, the structural and mechanistic details of how SecB recognizes non-native proteins and how it exerts its antifolding activity are unknown. Recent advances in nuclear magnetic resonance (NMR) and isotope labeling approaches have enabled the characterization of large, dynamic protein complexes including molecular chaperones20–23. We have exploited these approaches to determine the solution structure of SecB in complex with client proteins captured in their unfolded state revealing a unique binding architecture among protein-protein complexes.

Recognition sites in SecB and client proteinsSecB exists as a tetramer organized as a dimer of dimers (Kd of tetramer-dimer equilibrium is ~20 nM24) with an overall rectangular disk-like shape25,26 (Fig. 1a). Each subunit consists of 155 residues (17.5 kDa) composed of a simple α/β fold. The 1H-15N– and 1H-13C–correlated NMR spectra of the 70 kDa Escherichia coli SecB labeled in methyl-bearing (Ala, Ile, Met, Leu, Thr and Val) and aromatic (Phe, Trp, and Tyr) residues are of high quality and near-complete assignment has been obtained (Methods and Extended Data Fig. 1a,b). We used MBP (396 amino acids) and PhoA (471 amino acids) as SecB protein substrates. NMR analysis (Extended Data Figs. 1,c,d and 2a) showed that there are five distinct SecB-recognition sites in PhoA (labeled a through e; Fig. 1b) and seven sites in MBP (labeled a through g; Fig. 1c), with all sites being enriched in hydrophobic and aromatic residues, as shown before13.To determine the client-binding sites in SecB we sought to identify the SecB residues that show intermolecular NOEs to short fragments of PhoA and MBP encompassing SecB-recognition sites. The SecB residues that interact with the substrates (Fig. 1d,e) collectively form long, continuous hydrophobic grooves that constitute the primary binding sites for non-native proteins (Fig. 1f). Most prominent is a shallow groove running along the surface formed by helices α1 and α2, the helix-connecting loop, the cross-over loop and strand β2 (Fig. 1a,d–f). This groove, referred to as the primary client-binding site (Fig. 1f), is ~60 Å long and exposes ~1,300 Å2 of hydrophobic surface, per SecB subunit. In addition, a smaller surface (~600 Å2) formed by residues emanating from helix α1 and strands β1 and β4 also interacts with the unfolded proteins (Fig. 1e). This small surface, the secondary client-binding site (Fig. 1f), features several bulky nonpolar amino acids. All four subunits combined, SecB exposes ~7,600 Å2 of hydrophobic surface that NMR has shown to interact with non-native proteins (Fig. 1f).

Recognition sites in SecB and client proteins

SecB exists as a tetramer organized as a dimer of dimers (Kd of tetramer-dimer equilibrium is ~20 nM24) with an overall rectangular disk-like shape25,26 (Fig. 1a). Each subunit consists of 155 residues (17.5 kDa) composed of a simple α/β fold. The 1H-15N– and 1H-13C–correlated NMR spectra of the 70 kDa Escherichia coli SecB labeled in methyl-bearing (Ala, Ile, Met, Leu, Thr and Val) and aromatic (Phe, Trp, and Tyr) residues are of high quality and near-complete assignment has been obtained (Methods and Extended Data Fig. 1a,b). We used MBP (396 amino acids) and PhoA (471 amino acids) as SecB protein substrates. NMR analysis (Extended Data Figs. 1,c,d and 2a) showed that there are five distinct SecB-recognition sites in PhoA (labeled a through e; Fig. 1b) and seven sites in MBP (labeled a through g; Fig. 1c), with all sites being enriched in hydrophobic and aromatic residues, as shown before13.

To determine the client-binding sites in SecB we sought to identify the SecB residues that show intermolecular NOEs to short fragments of PhoA and MBP encompassing SecB-recognition sites. The SecB residues that interact with the substrates (Fig. 1d,e) collectively form long, continuous hydrophobic grooves that constitute the primary binding sites for non-native proteins (Fig. 1f). Most prominent is a shallow groove running along the surface formed by helices α1 and α2, the helix-connecting loop, the cross-over loop and strand β2 (Fig. 1a,d–f). This groove, referred to as the primary client-binding site (Fig. 1f), is ~60 Å long and exposes ~1,300 Å2 of hydrophobic surface, per SecB subunit. In addition, a smaller surface (~600 Å2) formed by residues emanating from helix α1 and strands β1 and β4 also interacts with the unfolded proteins (Fig. 1e). This small surface, the secondary client-binding site (Fig. 1f), features several bulky nonpolar amino acids. All four subunits combined, SecB exposes ~7,600 Å2 of hydrophobic surface that NMR has shown to interact with non-native proteins (Fig. 1f).

SecB holds proteins in the unfolded state and disrupts their structureWe used NMR spectroscopy to monitor at the residue level the effect of SecB on the folding of PhoA and MBP. Urea-treated PhoA and MBP refold quickly to their native state upon removal of urea (Extended Data Fig. 2b,c). Notably, SecB prevents the folding of PhoA or MBP, with both proteins adopting an unfolded conformation when bound to SecB (Extended Data Fig. 2b,c). The NMR data indicate that SecB-bound PhoA and MBP lack a tertiary structure and the regions of the protein substrates in contact with SecB do not form any secondary structure.

SecB holds proteins in the unfolded state and disrupts their structure

We used NMR spectroscopy to monitor at the residue level the effect of SecB on the folding of PhoA and MBP. Urea-treated PhoA and MBP refold quickly to their native state upon removal of urea (Extended Data Fig. 2b,c). Notably, SecB prevents the folding of PhoA or MBP, with both proteins adopting an unfolded conformation when bound to SecB (Extended Data Fig. 2b,c). The NMR data indicate that SecB-bound PhoA and MBP lack a tertiary structure and the regions of the protein substrates in contact with SecB do not form any secondary structure.

Client proteins wrap around SecBTo understand how SecB retains bound proteins in the unfolded state, we sought to structurally characterize the complexes of SecB with PhoA and MBP under native conditions. Multi-angle light scattering (MALS), isothermal titration calorimetry (ITC) and NMR all demonstrate that SecB forms stoichiometric complexes with PhoA and MBP (Extended Data Fig. 3), as is the case with other large client proteins including OmpA17,27. The structure of the SecB–PhoA complex (~120 kDa) was determined by NMR as detailed in Methods (Extended Data Figs. 4,5 and Extended Data Table 1) and is shown in Fig. 2. The most remarkable feature is that PhoA wraps around SecB in an overall arrangement that maximizes the interacting surface between the client protein, which is held in an unfolded conformation, and the chaperone. All of the grooves, the primary client-binding sites in SecB, in the four subunits are occupied by specific PhoA sites (a, c, d, and e) while the short PhoA site b binds to the smaller, secondary binding site (Fig. 2). The simultaneous engagement of all PhoA sites by SecB results in a significant enhancement in the affinity of the unfolded protein for SecB (Extended Data Fig. 3b,c), although the binding synergy is not strong. This is probably due to the fact that the linkers tethering the SecB-recognition sites in PhoA are long and flexible (Fig. 1b and Extended Data Fig. 4a) thereby reducing the effective concentration of the sites and the measured avidity28.Analysis of the SecB–PhoA structure revealed how SecB recognizes PhoA and how it accommodates all five PhoA sites (Fig. 1b) within one SecB molecule (Fig. 3). Most of the PhoA site a residues (Thr5 through Ala21) are engaged in nonpolar contacts with the SecB residues in the groove, burying a total of ~2250 Å2 of surface (~1900 Å2 nonpolar and ~350 Å2 polar). Interestingly, helix α2 in SecB, which acts as a lid of the binding groove, swings outward, by ~50°, upon PhoA binding (Fig. 4a). Together with an outward displacement of the first two turns of the helix α1, the width of the hydrophobic groove increases significantly so as to accommodate the large nonpolar side chains of the client (Figs 3, 4a). Moreover, the rearrangement of several side chains lining the SecB groove allows some of the bulky PhoA residues (e.g. Leu8, Leu11, and Phe15) to bury their side chains into the groove. Although the vast majority of the contacts are hydrophobic, several of the polar groups in PhoA site a are poised to form hydrogen bonds with polar SecB residues lining the groove (Fig. 3). PhoA site a binds to SecB in an extended conformation, which maximizes the interacting surface. Of note, this region of PhoA forms an α helix when bound to a hydrophobic groove in the SecA ATPase29. Thus, SecB disfavors the formation of any regular secondary structure of the bound client.PhoA site c is the longest SecB-recognition site in PhoA consisting of ~50 residues (Fig. 1b). It binds to SecB in an extended conformation spanning a distance of ~100 Å (Fig. 2, 3). The first 33 residues (Phe93-Ala125) of PhoA site c bind exclusively within the groove of one subunit, whereas the remaining PhoAc residues (Ala126-Tyr138) extend across the surface at the tetramerization interface. The total surface buried by the binding of PhoA site c to SecB is ~5150 Å2 (~3500 Å2 nonpolar and ~1600 Å2 polar). PhoA site d encompasses a stretch of 30 residues (Ala271-Thr309) and binds to SecB in an extended conformation running along the entire groove spanning a distance of ~70 Å (Fig. 2, 3). The buried surface amounts to a total of ~4200 Å2, with ~2800 Å2 nonpolar and ~1400 Å2 polar. PhoA site e (residues Asn450-Lys471) binds to SecB in a very similar manner to PhoA site a. PhoA site e is one of the regions that retains significant α helical structure in the unfolded PhoA20. It binds to SecB, however, in an extended conformation further highlighting the tendency of SecB to disrupt any regular secondary structure.SecB can adjust the structure of the primary binding grooves to allow longer substrates to fit in the groove. For example, whereas ~25 residues of the PhoA site d fit in the groove in an extended conformation, more than 40 residues of the PhoA site c fit within the same space (Fig. 4b). When the SecB helix α2 swings outward upon client binding (Fig. 4a), the movement not only widens the binding groove but also exposes additional nonpolar and polar surface that is available for binding by the unfolded client.It should be noted that structure determination of isolated PhoA sites (PhoAa, PhoAc, PhoAd, and PhoAe, where the superscript denotes the corresponding site; Fig. 1b) in complex with SecB shows that multiple molecules of the individual sites can be accommodated within a SecB tetramer, owing to their relatively short length (Extended Data Figs. 3c, 5.NMR structure determination of MBP sites d and e in complex with SecB (SecB–MBPd and SecB–MBPe complexes; Extended Data 6 and Extended Data Table 1) showed that MBP binds to SecB in a very similar fashion to PhoA. Thus, non-native proteins share a similar binding mode for SecB. Analysis of the NMR spectra of labeled full-length MBP in complex with SecB demonstrated that all seven binding sites in MBP (Fig. 1c) are engaged by SecB in the SecB–MBP complex (Extended Data 7a). NMR-driven modeling of the SecB–MBP complex (Methods) shows that MBP, similarly to PhoA, wraps around SecB using the chaperone’s entire binding surface (Extended Data 7b). Interestingly, the gain in avidity for MBP binding to SecB (Kd ~0.05 M), when compared to the isolated sites, appears to be an order of magnitude stronger than in the case of PhoA (Extended Data 3c,f). The reasons for the higher avidity are likely the larger interacting interface in the complex with MBP (~130 PhoA residues compared to ~240 MBP residues interacting with SecB) and the fact that the SecB-recognition sites in MBP are tethered with linkers that are much shorter in length than in the case of PhoA (Fig. 1b,c). Thermodynamic analysis reveals a large and favorable enthalpy of binding for both SecB–MBP and SecB–PhoA complexes, but with the overall affinity being reduced by unfavorable entropy of binding (Extended Data 3b,e).Amino acid substitutions at the client-binding sites in SecB resulted in a substantial decrease in the affinity for unfolded proteins and a marked decrease of its antifolding activity (Extended Data Fig. 8a–c).

Client proteins wrap around SecB

To understand how SecB retains bound proteins in the unfolded state, we sought to structurally characterize the complexes of SecB with PhoA and MBP under native conditions. Multi-angle light scattering (MALS), isothermal titration calorimetry (ITC) and NMR all demonstrate that SecB forms stoichiometric complexes with PhoA and MBP (Extended Data Fig. 3), as is the case with other large client proteins including OmpA17,27. The structure of the SecB–PhoA complex (~120 kDa) was determined by NMR as detailed in Methods (Extended Data Figs. 4,5 and Extended Data Table 1) and is shown in Fig. 2. The most remarkable feature is that PhoA wraps around SecB in an overall arrangement that maximizes the interacting surface between the client protein, which is held in an unfolded conformation, and the chaperone. All of the grooves, the primary client-binding sites in SecB, in the four subunits are occupied by specific PhoA sites (a, c, d, and e) while the short PhoA site b binds to the smaller, secondary binding site (Fig. 2). The simultaneous engagement of all PhoA sites by SecB results in a significant enhancement in the affinity of the unfolded protein for SecB (Extended Data Fig. 3b,c), although the binding synergy is not strong. This is probably due to the fact that the linkers tethering the SecB-recognition sites in PhoA are long and flexible (Fig. 1b and Extended Data Fig. 4a) thereby reducing the effective concentration of the sites and the measured avidity28.

Analysis of the SecB–PhoA structure revealed how SecB recognizes PhoA and how it accommodates all five PhoA sites (Fig. 1b) within one SecB molecule (Fig. 3). Most of the PhoA site a residues (Thr5 through Ala21) are engaged in nonpolar contacts with the SecB residues in the groove, burying a total of ~2250 Å2 of surface (~1900 Å2 nonpolar and ~350 Å2 polar). Interestingly, helix α2 in SecB, which acts as a lid of the binding groove, swings outward, by ~50°, upon PhoA binding (Fig. 4a). Together with an outward displacement of the first two turns of the helix α1, the width of the hydrophobic groove increases significantly so as to accommodate the large nonpolar side chains of the client (Figs 3, 4a). Moreover, the rearrangement of several side chains lining the SecB groove allows some of the bulky PhoA residues (e.g. Leu8, Leu11, and Phe15) to bury their side chains into the groove. Although the vast majority of the contacts are hydrophobic, several of the polar groups in PhoA site a are poised to form hydrogen bonds with polar SecB residues lining the groove (Fig. 3). PhoA site a binds to SecB in an extended conformation, which maximizes the interacting surface. Of note, this region of PhoA forms an α helix when bound to a hydrophobic groove in the SecA ATPase29. Thus, SecB disfavors the formation of any regular secondary structure of the bound client.

PhoA site c is the longest SecB-recognition site in PhoA consisting of ~50 residues (Fig. 1b). It binds to SecB in an extended conformation spanning a distance of ~100 Å (Fig. 2, 3). The first 33 residues (Phe93-Ala125) of PhoA site c bind exclusively within the groove of one subunit, whereas the remaining PhoAc residues (Ala126-Tyr138) extend across the surface at the tetramerization interface. The total surface buried by the binding of PhoA site c to SecB is ~5150 Å2 (~3500 Å2 nonpolar and ~1600 Å2 polar). PhoA site d encompasses a stretch of 30 residues (Ala271-Thr309) and binds to SecB in an extended conformation running along the entire groove spanning a distance of ~70 Å (Fig. 2, 3). The buried surface amounts to a total of ~4200 Å2, with ~2800 Å2 nonpolar and ~1400 Å2 polar. PhoA site e (residues Asn450-Lys471) binds to SecB in a very similar manner to PhoA site a. PhoA site e is one of the regions that retains significant α helical structure in the unfolded PhoA20. It binds to SecB, however, in an extended conformation further highlighting the tendency of SecB to disrupt any regular secondary structure.

SecB can adjust the structure of the primary binding grooves to allow longer substrates to fit in the groove. For example, whereas ~25 residues of the PhoA site d fit in the groove in an extended conformation, more than 40 residues of the PhoA site c fit within the same space (Fig. 4b). When the SecB helix α2 swings outward upon client binding (Fig. 4a), the movement not only widens the binding groove but also exposes additional nonpolar and polar surface that is available for binding by the unfolded client.

It should be noted that structure determination of isolated PhoA sites (PhoAa, PhoAc, PhoAd, and PhoAe, where the superscript denotes the corresponding site; Fig. 1b) in complex with SecB shows that multiple molecules of the individual sites can be accommodated within a SecB tetramer, owing to their relatively short length (Extended Data Figs. 3c, 5.

NMR structure determination of MBP sites d and e in complex with SecB (SecB–MBPd and SecB–MBPe complexes; Extended Data 6 and Extended Data Table 1) showed that MBP binds to SecB in a very similar fashion to PhoA. Thus, non-native proteins share a similar binding mode for SecB. Analysis of the NMR spectra of labeled full-length MBP in complex with SecB demonstrated that all seven binding sites in MBP (Fig. 1c) are engaged by SecB in the SecB–MBP complex (Extended Data 7a). NMR-driven modeling of the SecB–MBP complex (Methods) shows that MBP, similarly to PhoA, wraps around SecB using the chaperone’s entire binding surface (Extended Data 7b). Interestingly, the gain in avidity for MBP binding to SecB (Kd ~0.05 M), when compared to the isolated sites, appears to be an order of magnitude stronger than in the case of PhoA (Extended Data 3c,f). The reasons for the higher avidity are likely the larger interacting interface in the complex with MBP (~130 PhoA residues compared to ~240 MBP residues interacting with SecB) and the fact that the SecB-recognition sites in MBP are tethered with linkers that are much shorter in length than in the case of PhoA (Fig. 1b,c). Thermodynamic analysis reveals a large and favorable enthalpy of binding for both SecB–MBP and SecB–PhoA complexes, but with the overall affinity being reduced by unfavorable entropy of binding (Extended Data 3b,e).

Amino acid substitutions at the client-binding sites in SecB resulted in a substantial decrease in the affinity for unfolded proteins and a marked decrease of its antifolding activity (Extended Data Fig. 8a–c).

Chaperone-client binding mode modulates kineticsSecB may prevent the folding of a protein altogether whereas other chaperones, such as TF, cannot typically do so (Fig. 5a). We used surface plasmon resonance (SPR) and bio-layer interferometry (BLI) to measure the kinetics of interaction between unfolded MBP and PhoA with the SecB and Trigger Factor (TF) chaperones (Fig. 5b and Extended Data Fig. 9a–f). Notably, MBP associates with SecB with a ~10-fold higher rate (kon ~106 M−1 s−1) than with TF (kon ~105 M−1 s−1) and dissociates from SecB with a ~5-fold slower rate (koff ~0.01 s−1) than from TF (koff ~0.05 s−1). Of note, SecB prevents folding8 of the cytosolic pre-form of MBP (Fig. 5a and Extended Data Fig. 2c), but it cannot prevent folding of the mature (periplasmic) form of MBP, which lacks the N-terminal signal sequence (Extended Data Fig. 9g–i). This is because of the much faster intrinsic folding rate (kf) of the mature MBP (0.02 s−1) compared to the kf of the pre-form of MBP (0.003 s−1) (Extended Data Fig. 9g,h). Interestingly, an MBP variant8 with much slower folding rate (kf ~0.0008 s−1) allows even TF to delay its folding (Extended Data Fig. 9j) highlighting the importance of the kinetics of client intrinsic folding and binding to the chaperone.

Chaperone-client binding mode modulates kinetics

SecB may prevent the folding of a protein altogether whereas other chaperones, such as TF, cannot typically do so (Fig. 5a). We used surface plasmon resonance (SPR) and bio-layer interferometry (BLI) to measure the kinetics of interaction between unfolded MBP and PhoA with the SecB and Trigger Factor (TF) chaperones (Fig. 5b and Extended Data Fig. 9a–f). Notably, MBP associates with SecB with a ~10-fold higher rate (kon ~106 M−1 s−1) than with TF (kon ~105 M−1 s−1) and dissociates from SecB with a ~5-fold slower rate (koff ~0.01 s−1) than from TF (koff ~0.05 s−1). Of note, SecB prevents folding8 of the cytosolic pre-form of MBP (Fig. 5a and Extended Data Fig. 2c), but it cannot prevent folding of the mature (periplasmic) form of MBP, which lacks the N-terminal signal sequence (Extended Data Fig. 9g–i). This is because of the much faster intrinsic folding rate (kf) of the mature MBP (0.02 s−1) compared to the kf of the pre-form of MBP (0.003 s−1) (Extended Data Fig. 9g,h). Interestingly, an MBP variant8 with much slower folding rate (kf ~0.0008 s−1) allows even TF to delay its folding (Extended Data Fig. 9j) highlighting the importance of the kinetics of client intrinsic folding and binding to the chaperone.

SecB rescues folded proteins that are prone to aggregation by transient interactionTo understand how SecB rescues cytosolic proteins6 and increases the yield of natively folded proteins (Extended Data Fig. 9h), we used the aggregation-prone MBPG32D/I33P (hereafter MBPmut) variant20 that has a high tendency to aggregate especially at temperatures higher than 30 °C. Interestingly, in the presence of SecB, NMR shows that MBPmut remains folded and soluble even at temperatures as high as 50 °C (Extended Data Fig. 8d,e). At such high temperatures NMR showed that SecB binds to and shields the transiently exposed unfolded state of MBPmut resulting in its protection from aggregation (Extended Data Fig. 10f,g). The aggregation-prone, transiently populated conformation of the otherwise folded MBPmut that is protected by SecB, is only partially unfolded and dissociates rapidly from SecB giving rise to an anti-aggregation effect as opposed to an antifolding effect.

SecB rescues folded proteins that are prone to aggregation by transient interaction

To understand how SecB rescues cytosolic proteins6 and increases the yield of natively folded proteins (Extended Data Fig. 9h), we used the aggregation-prone MBPG32D/I33P (hereafter MBPmut) variant20 that has a high tendency to aggregate especially at temperatures higher than 30 °C. Interestingly, in the presence of SecB, NMR shows that MBPmut remains folded and soluble even at temperatures as high as 50 °C (Extended Data Fig. 8d,e). At such high temperatures NMR showed that SecB binds to and shields the transiently exposed unfolded state of MBPmut resulting in its protection from aggregation (Extended Data Fig. 10f,g). The aggregation-prone, transiently populated conformation of the otherwise folded MBPmut that is protected by SecB, is only partially unfolded and dissociates rapidly from SecB giving rise to an anti-aggregation effect as opposed to an antifolding effect.

ConclusionsThe present data demonstrate how the unique binding mode of SecB for non-native proteins (Fig. 2 and Extended Data Fig. 7b) enables the chaperone to prevent folding of bound proteins (Fig. 5a). When compared to TF (Fig. 5c,d), a chaperone for which its structure in complex with full length PhoA is known20, the structural data explain how the overall architecture of the chaperone and the way it engages non-native proteins give rise to different chaperone activities (Fig. 5a). Although both SecB and TF prevent aggregation and misfolding, as most molecular chaperones do, SecB has a much stronger antifolding activity than TF. Each TF molecule can accommodate a stretch of maximum ~50 interacting residues of an unfolded polypeptide whereas SecB can accommodate as many as ~250 interacting residues (Fig. 5c,d). Because SecB recognizes and binds to multiple regions within an unfolded protein, long client proteins wrap around SecB so as to maximize the binding interface, thereby altering the binding kinetics. The overall binding architecture appears to be unique among known protein-protein complexes. More structural data on complexes of chaperones with proteins30 are needed to discover the full repertoire of binding architectures and how they influence chaperone activity.

Conclusions

The present data demonstrate how the unique binding mode of SecB for non-native proteins (Fig. 2 and Extended Data Fig. 7b) enables the chaperone to prevent folding of bound proteins (Fig. 5a). When compared to TF (Fig. 5c,d), a chaperone for which its structure in complex with full length PhoA is known20, the structural data explain how the overall architecture of the chaperone and the way it engages non-native proteins give rise to different chaperone activities (Fig. 5a). Although both SecB and TF prevent aggregation and misfolding, as most molecular chaperones do, SecB has a much stronger antifolding activity than TF. Each TF molecule can accommodate a stretch of maximum ~50 interacting residues of an unfolded polypeptide whereas SecB can accommodate as many as ~250 interacting residues (Fig. 5c,d). Because SecB recognizes and binds to multiple regions within an unfolded protein, long client proteins wrap around SecB so as to maximize the binding interface, thereby altering the binding kinetics. The overall binding architecture appears to be unique among known protein-protein complexes. More structural data on complexes of chaperones with proteins30 are needed to discover the full repertoire of binding architectures and how they influence chaperone activity.

METHODSExpression and Preparations of ProteinsThe E. coli SecB gene was cloned into the pET-16b vector (Novagen) containing a His6-tag and a tobacco etch virus (TEV) protease cleavage site at the N-terminus. Protein samples of E. coli PhoA were produced as described before20. All E. coli MBP constructs were cloned into the pET-16b vector containing a His6-tag and a TEV protease cleavage site at the N-terminus. The following MBP constructs were prepared in this study (residue numbers of the boundaries are in superscript): MBP1–396, mature MBP23–396, MBP29–99, MBP67–99, MBP97–164, MBP160–201, MBP198–265, MBP260–336, MBP331–396, and the MBP variants MBPG32D/I33P, MBPY283D and MBPV8G/Y283D (MBP mutants are numbered based on the amino acid sequence of the mature form of MBP). All constructs were transformed into BL21(DE3) cells. Isotopically unlabeled protein samples were produced in cells grown in Luria-Bertani (LB) medium at 37 °C in the presence of ampicillin (100 µg ml−1) to an OD600 ~0.8. Protein induction was induced by the addition of 0.2 mM isopropyl-β-D-1-thiogalactopyranoside (IPTG) and cells were allowed to grow for 16 hours at 18 °C. Cells were harvested at OD600 ~1.5 and resuspended in lysis buffer (50 mM Tris-HCl, 500 mM NaCl, pH 8 and 1 mM PMSF). Cells were disrupted by a high-pressure homogenizer and centrifuged at 50,000g. Proteins were purified using Ni Sepharose 6 Fast Flow resin (GE Healthcare), followed by tag removal by TEV protease at 4°C (incubation for 16 hours) and gel filtration using Superdex 75 16/60 or 200 16/60 columns (GE Healthcare). Protein concentration was determined spectrophotometrically at 280 nm using the corresponding extinction coefficient.MALS ExperimentsMALS was measured using DAWN HELEOS-II (Wyatt Technology Corporation) downstream of a Shimadzu liquid chromatography system connected to a Superdex 200 10/300 GL (GE Healthcare) gel filtration column. The running buffer for SecB–PhoA complexes was 20 mM KPi (pH 7.0), 100 mM KCl, 4 mM βME, and 0.5 mM EDTA, whereas for SecB–MBP complexes was 20 mM HEPES, pH 7, 150 mM KOAc and 0.05% NaN3. Protein samples at a concentration of 0.05 to 0.2 mM were used. The flow rate was set to 0.5 ml min−1 with an injection volume of 200 µl and the light scattering signal was collected at room temperature (~23 °C). The data were analyzed with ASTRA version 6.0.5 (Wyatt Technology Corporation).ITC ExperimentsITC was carried out using an iTC200 microcalorimeter (GE Healthcare) at temperatures ranging from 4 °C to 25 °C. All protein samples were extensively dialyzed against the ITC buffer containing 50 mM KPi (pH 7.0), 50 mM KCl, 0.05% NaN3 and 2 mM tris(2-carboxyethyl)phosphine (TCEP). All solutions were filtered using membrane filters (pore size, 0.45 µm) and thoroughly degassed for 20 min before the titrations. The 40-µl injection syringe was filled with ~0.05 to 1 mM of SecB solution and the 200-µl cell was filled with ~0.01 to 0.2 mM PhoA or MBP. To measure the binding affinity of MBP to SecB, the slowly folding MBPV8G/Y283D variant was used to measure the affinity of MBP for SecB. MBPV8G/Y283D was unfolded in 8 M urea, 20 mM HEPES, pH 7, 150 mM KOAc and 0.05% NaN3, and diluted 20 times to give a final concentration of 2.7 µM immediately before loading into the cell. The solution containing SecB was precisely adjusted to match the urea concentration. The titrations were carried out with a preliminary 0.2-µl injection, followed typically by 15 injections of 2.5 µl each with time intervals of 3 min. The solution was stirred at 1000 rpm. Data for the preliminary injection, which are affected by diffusion of the solution from and into the injection syringe during the initial equilibration period, were discarded. Binding isotherms were generated by plotting heats of reaction normalized by the modes of injectant versus the ratio of total injectant to total protein per injection. The data were fitted with Origin 7.0 (OriginLab Corporation).Protein Isotope Labeling for NMR StudiesIsotopically labeled samples for NMR studies were prepared by growing the cells in minimal (M9) medium. Cells were typically harvested at OD600 ~1.0. U-[2H,13C,15N]–labeled samples were prepared for the backbone assignment of SecB and large MBP fragments by supplementing the growing medium with 15NH4Cl (1 g liter−1) and 2H7,13C6-glucose (2 g liter−1) in 99.9% 2H2O (CIL and Isotec). The 1H-13C methyl labeled samples were prepared as described20,29,31. a-Ketobutyric acid (50 mg liter−1) and a-ketoisovaleric acid (85 mg liter−1) were added to the culture 1 hour before the addition of IPTG. Met-[13CH3]– and Ala-[13CH3]–labeled samples were produced by supplementing the medium with [13CH3]-Met (50 mg liter−1) and [2H2,13CH3]-Ala (50 mg liter−1). For Thr labeling, a Thr-auxotrophic cell strain was used, and the medium was supplemented with [2H2,13CH3]-Thr (25 mg liter−1). For Phe, Tyr, and Trp labeling, U-[1H,13C]–labeled amino acids were used. Alternate 13C-labeling of aromatic residues was performed as described32. All precursors and amino acids were added to the culture 1 hour before the addition of IPTG, except Ala, which was added 30 min before induction.NMR SpectroscopyNMR samples were typically prepared in 50 mM KPi (pH 7.0), 50 mM KCl, 0.05% NaN3, 5mM βME and 7% D2O. NMR experiments were recorded on Bruker 900, 850 and 700 MHz spectrometers. NMR spectra were typically recorded at 10 °C for the isolated PhoA and MBP fragments and at 35 °C for SecB and its complexes. Protein sample concentration ranged from 0.1 to 1.0 mM. All NMR spectra were processed using NMRPipe33 and analyzed using NMRView (http://www.onemoonscientific.com).NMR Assignment of SecBSecB tetramer packs as a dimer of dimers and gives rise to two pairs of magnetically equivalent subunits: A and D give one set of resonances and subunits B and C give another set of resonances (Extended Data Fig. 1a). Sequential backbone assignment of SecB was achieved by the use of standard triple-resonance NMR pulse sequences. Three-dimensional (3D) 1H-15N NOESY experiments were used to confirm and extend the backbone assignment within each subunit. Side-chain assignment for methyls and aromatic residues was accomplished using the following NMR experiments: 3D (1H)-13C heteronuclear multiple-quantum coherence (HMQC)-NOESY-1H-13C HMQC, 13C-edited NOESY-HSQC, 13C-edited HSQC-NOESY, 15N-edited NOESY-HSQC, 3D (1H)-13C HSQC-NOESY-1H-15N HSQC, and 3D (1H)-15N HSQC-NOESY-1H-13C HSQC.NMR assignment of PhoA and MBP in the unfolded stateWe previously described the assignment strategy for unfolded PhoA20. We followed a similar strategy to assign MBP in the unfolded state by making use of several MBP fragments that remain soluble and unfolded when isolated (Extended Data Fig. 1c): MBP29–99, MBP67–99, MBP97–164, MBP160–201, MBP198–265, MBP260–336 and MBP331–396. Isolated MBP fragments encompassing the first 23 N-terminal residues (signal sequence) were not stable and this region could only be assigned in complex with SecB. Overlay of the spectra of the MBP fragments with the spectra of full-length MBP in 4M urea indicated very good resonance correspondence. This is expected because all of the fragments as well as the MBP in 4 M urea are unfolded. Resonance assignment obtained for the various fragments was transferred to full length MBP in urea, and ambiguities were resolved by the use of 3D NMR spectra. It should be noted that although resonance dispersion in unliganded PhoA and MBP is poor, complex formation with SecB alleviates this problem (for the PhoA and MBP residues in the SecB-binding regions) with the spectra being of high resolution (Extended Data Fig. 4c).Structure Determination of SecB–PhoA and SecB–MBP ComplexesAssignment of the resonances in SecB–PhoA was accomplished by first assigning the complexes between SecB and the individual PhoA sites (SecB–PhoAa, SecB–PhoAc, SecB–PhoAd, SecB–PhoAe). We used U-12C,15N–labeled samples that contained specifically protonated methyl groups of Ala, Val, Leu, Met, Thr, and Ile (δ1) and protonated aromatic residues Phe, Tyr, and Trp in an otherwise deuterated background. The high sensitivity and resolution of the methyl region, combined with the high abundance of these nine amino acids in SecB (Extended Data Fig. 1a) and in the SecB-binding sites of PhoA and MBP, provided a large number of intermolecular NOEs for the SecB–PhoA and SecB–MBP complexes (Extended Data Table 1). Because PhoA in complex with SecB provided higher quality spectra than the spectra of MBP in complex with SecB, we determined first the structure of the SecB–PhoA complex (~120 kDa) by NMR. We initially characterized the structure of the each PhoA site (a through e) individually in complex with SecB (Extended Data Fig. 5). The structures of SecB–PhoAa, SecB–PhoAc, SecB–PhoAd, and SecB–PhoAe, were determined by NMR and presented in Extended Data Fig. 5. A large number of inter-molecular NOEs were collected for each one of the complexes (Extended Data Table 1). Because of the relatively short length of the polypeptides encompassing the individual PhoA sites, multiple PhoA molecules bound to SecB, as shown in Extended Data Fig. 5. We should also note that we detected the presence of a small number of inter-molecular NOEs that were suggestive of alternative conformations of the PhoA sites bound to SecB. However, the intensity of these sets of NOEs is much weaker indicating that the population of such alternative complexes is low. To solve the structure of the SecB–PhoA complex we sought to determine how each one of the PhoA sites binds to SecB in the context of the full length PhoA. To circumvent the signal overlap in this large complex, we used samples where the two proteins were isotopically labeled in different amino acids. For example, in one of these samples SecB was labeled in the methyls of Leu, Val and Met, whereas PhoA in the methyls of Ile amino acids. Because of the distinct chemical shifts of 1H and 13C resonances of the methyls and the isotope labeling scheme, it was possible to measure specific intermolecular NOEs between SecB and PhoA (Extended Data Fig. 4b). Several of these samples were used in order to determine as many inter-molecular NOEs as possible. As expected, the NOEs were compatible with the structure of each PhoA site in complex with SecB, with the crucial difference that only one PhoA molecule could be accommodated in SecB. Due to its short length, the isolated PhoA site b (PhoAb) binds to almost all of the exposed hydrophobic surface of SecB, as determined by NMR. In the SecB–PhoA complex with SecB, PhoA site b can only bind to the secondary binding site, as determined by NOEs. To further corroborate the structure of the SecB–PhoA complex we used PRE data (see below). The PRE-derived distances were fully compatible with the NOE data collected on SecB–PhoA. The structure of the SecB–PhoA complex was determined using the set of inter-molecular NOEs collected directly in the complex and further refined using the inter-molecular NOEs collected for the corresponding isolated PhoA sites in complex with SecB. It should be noted that because of the symmetry in SecB, the various PhoA sites may bind to any of the four SecB subunits. The final arrangement will be dictated by the length of the linkers tethering the SecB-recognition sites (as shown in Fig. 2), namely how far nearby recognition sites can bind from each other, and thus alternative routes of the polypeptide bound to SecB may be present. The only conceivable difference among the various conformations is the relative disposition of the PhoA sites. In all cases all of the SecB-recognition sites in PhoA are engaged by SecB in the complex and PhoA wraps around SecB. The NMR-driven structural model of the SecB–MBP complex (Extended Data Fig. 7b) was determined as follows: NMR analysis demonstrated that all seven recognition sites in MBP (labeled a through g) are bound to SecB in the SecB–MBP complex (Extended Data Fig. 7a). We have determined the high-resolution structure of MBPd and MBPe in complex with SecB (Extended Data Fig. 6). Because of their length and the short linker tethering the two sites, d and e sites most likely bind to the same side of SecB. MBP site f is the longest one, consisting of ~90 residues, and is thus entirely accommodated on the other side of SecB. With sites d, e and f occupying the primary binding sites, the other recognition sites (a, b, c and g) being much shorter can be accommodated within the secondary client-binding sites on SecB. The structure of MBP sites d and e in complex with SecB was determined using the experimental inter-molecular NOE data. The hydrophobic residues of the sites a, b, c, f, and g showing the strongest effect upon SecB binding, as determined by differential line broadening, were used to drive the docking of these sites to nonpolar residues on SecB. The modeled structure shows that the entire MBP sequence can be accommodated within one SecB molecule. The structures of SecB in complex with PhoA and MBP were calculated with CYANA 3.9734, using NOE peak lists from 3D (1H)-13C HMQC-NOESY–1H-13C HMQC, 3D (1H)-15N HSQC-NOESY–1H-13C HSQC, 13C–edited NOESY-HSQC, and 15N–edited NOESY-HSQC. The 13Cα, 13Cβ, 13C’, 15N, and NH chemical shifts served as input for the TALOS+ program35 to extract dihedral angles (φ and ψ). The side chains of SecB residues within or nearby the PhoA and MBP binding sites were set flexible and their conformation was determined using intermolecular NOEs collected for each one of the complexes. The SecB regions remote to the binding sites were set rigid using the crystal structure coordinates for E. coli SecB26. The 20 lowest-energy structures were refined by restrained molecular dynamics in explicit water with CNS36. The percentage of residues falling in favoured and disallowed regions, respectively, of the Ramachandran plot is as follows: 99.4% and 0.6% for SecB–PhoA; 99.4% and 0.6% for SecB–PhoAa; 99.3% and 0.7% for SecB–PhoAc; 99.2% and 0.8% for SecB–PhoAd; 99.3% and 0.7% for SecB–PhoAe; 99.4% and 0.6% for SecB–MBPd; and 99.4% and 0.6% for SecB–MBPe.PRE ExperimentsPRE experiments were used to confirm the position of each individual PhoA binding site in the SecB–PhoA complex. First, a “Cys-free” variant of PhoA was prepared by mutating the four naturally-occurring Cys residues in PhoA (Cys190, Cys200, Cys308 and Cys358) to Ser. We then introduced a Cys residue to either end of each SecB-binding site in PhoA and prepared a total of ten single-Cys mutants: T5C, T23C, K65C, M75C, G91C, G140C, Q274C, C308, N450C and C472. The protein purified from Ni-NTA column was quickly concentrated and loaded onto HiLoad® 16/60 Superdex® 200 gel filtration column (GE healthcare) using a buffer containing 50 mM KPi (pH 7.0), 150 mM NaCl, and 0.05% NaN3. Immediately after elution the purified single-Cys PhoA mutant was divided into two equal portions for parallel treatment with (1-oxyl- 2,2,5,5-tetramethyl-3-pyrroline-3-methyl)-ethanethiosulfonate (MTSL, Toronto Research Chemicals, Toronto) and a diamagnetic MTSL analog, in a 10-fold molar excess at 4 °C for 16–20 hrs. MTSL was prepared in a 50 mM concentrated stock in acetonitrile. Free MTSL was removed by extensive buffer exchange using Centricon® Centrifugal Filter with a MWCO of 10,000 (Millipore) at 4 °C. The MTSL-labeled PhoA protein samples were then concentrated and added into the 2H-methyl-13CH3-labeled SecB at a final molar ratio of PhoA: SecB = 1:1. 2D 1H,13C HMQC spectra were recorded at 28 °C. A sample of SecB in complex with PhoA cross-linked to a diamagnetic MTSL analog was used as a reference. Residues experienced significant NMR signal intensity reduction (>50% intensity loss) were identified as sites being within 20 Å of the paramagnetic center whereas residues experiencing more than 90% intensity loss were identified as sites being within 14 Å of the paramagnetic center.Protein Folding AssaysRefolding experiments of MBP were performed as described before37 with some modifications. Briefly, MBP was first denatured in 8 M urea, 100 mM HEPES, 20 mM KOAc, 5 mM Mg(OAc)2, pH 7.4, and 0.05% NaN3). Refolding was initiated by rapid dilution (20 times dilution) in the urea-free buffer and the refolding process of MBP in the absence and presence of SecB or TF was monitored by the change in the intrinsic Trp fluorescence. Fluorescence intensity was measured using either a spectrofluorometer (FluoroMax-4, Horiba) or a microplate reader (Infinite 200 PRO, Tecan). The excitation and emission wavelengths were set to 295 nm and 345 nm, respectively. For measurement using the FluoroMax-4 instrument, the MBP concentration in the 1-ml cuvette was 0.4 µM, whereas for the microplate reader experiments the concentration of MBP was 4 µM in the 30 µl-plate well. All fluorescence measurements were performed at 25°C. Data were fitted by the Prism 6 (GraphPad) software using the non-linear regression analysis equation38.Surface plasmon resonance (SPR)All SPR experiments were carried out on a Biacore T200 system (GE Healthcare) using a NTA-coated Sensor Chip NTA (GE Healthcare) at a flow rate of 50 µl min−1. The PhoA protein sample used for SPR experiments was genetically constructed with a His12-tag at the C-terminus and a flexible (Gly-Ser)5 linker repeat inserted in between to avoid steric hindrance. A single-cycle kinetic procedure was used to characterize the interaction of SecB and PhoA. The His-tagged PhoA was immobilized onto a NTA sensor chip, followed by washing with the running buffer containing 50 mM Phosphate, 50 mM KCl, pH 7, 0.05% NaN3, and 2 mM TCEP. The reducing agent (TCEP) ensured that PhoA is in the unfolded state20. SecB (analyte) at a range of concentrations (0.1 µM to 25.6 µM) were injected, and data for a period of 30 seconds of association and 60 seconds of dissociation were collected. MBP was prepared with a His10- tag at the N-terminus followed by a flexible nine-residue linker to avoid steric hindrance. Multiple-cycle kinetic analysis was performed for the SPR experiments of the binding between MBP and SecB where each sample concentration was run in a separate cycle, and the surface was regenerated between each cycle using NTA regeneration buffer. His-tagged MBP was denatured in 8 M urea and immobilized onto a NTA sensor chip. Urea was quickly washed away by running buffer containing 20 mM HEPES, pH 7.4, 150 mM KOAc, and 0.05% NaN3. SecB was injected at concentrations ranging from 2.5 nM to 1.6 µM. The association and dissociation time for data collection was set as 90 s and 120 s, respectively. After urea was removed MBP remained in the unfolded conformation for sufficient time to interact with SecB. This was confirmed by monitoring the refolding behavior of MBP using an Infinite 200 PRO microplate reader (Tecan) at the temperature range of the experiments. All SPR experiments were repeated three times and highly reproducible data were obtained. The sensorgrams obtained from the assay channel were subtracted by the buffer control, and data were fitted using the Biacore T200 evaluation software (version 1.0).Bio-Layer Interferometry (BLI)BLI experiments were performed using an Octet system (forteBIO) at room temperature (~23 °C). MBP was biotinylated using the biotination kit EZ-Link™ NHS-PEG4-Biotin (Thermo Fisher Scientific). Biotin label freshly dissolved in water was added to the protein solution to a final molar ratio of 1:1 in buffer containing 50 mM KPi, pH 7, 150 mM NaCl, 0.05% NaN3, and the solution was mixed at room temperature for 45 min. Unlabeled biotin label was removed by extensive buffer exchange using Centricon® Centrifugal Filter with a MWCO of 10,000 (Millipore) at 4 °C using a buffer containing 20 mM HEPES (pH 7), 150 mM KoAc, and 0.05% NaN3. Biotin-labeled MBP (200 nM) denatured in 8 M urea was immobilized onto the Streptavidin (SA) biosensor, and the biosensors were subsequently blocked with Biocytin in 8 M urea solution before a quick 30 s dip into the urea-free buffer. SecB or TF previously diluted was applied in a dose dependent manner to the biosensors immobilized with MBP. Bovine serum albumin (BSA) powder (Sigma-Aldrich) was added to a final concentration of 2% to avoid nonspecific interaction. Parallel experiments were performed for reference sensors with no MBP captured and the signals were subsequently subtracted during data analysis. The association and dissociation periods were set to 2 min and 5 min, respectively.

Expression and Preparations of ProteinsThe E. coli SecB gene was cloned into the pET-16b vector (Novagen) containing a His6-tag and a tobacco etch virus (TEV) protease cleavage site at the N-terminus. Protein samples of E. coli PhoA were produced as described before20. All E. coli MBP constructs were cloned into the pET-16b vector containing a His6-tag and a TEV protease cleavage site at the N-terminus. The following MBP constructs were prepared in this study (residue numbers of the boundaries are in superscript): MBP1–396, mature MBP23–396, MBP29–99, MBP67–99, MBP97–164, MBP160–201, MBP198–265, MBP260–336, MBP331–396, and the MBP variants MBPG32D/I33P, MBPY283D and MBPV8G/Y283D (MBP mutants are numbered based on the amino acid sequence of the mature form of MBP). All constructs were transformed into BL21(DE3) cells. Isotopically unlabeled protein samples were produced in cells grown in Luria-Bertani (LB) medium at 37 °C in the presence of ampicillin (100 µg ml−1) to an OD600 ~0.8. Protein induction was induced by the addition of 0.2 mM isopropyl-β-D-1-thiogalactopyranoside (IPTG) and cells were allowed to grow for 16 hours at 18 °C. Cells were harvested at OD600 ~1.5 and resuspended in lysis buffer (50 mM Tris-HCl, 500 mM NaCl, pH 8 and 1 mM PMSF). Cells were disrupted by a high-pressure homogenizer and centrifuged at 50,000g. Proteins were purified using Ni Sepharose 6 Fast Flow resin (GE Healthcare), followed by tag removal by TEV protease at 4°C (incubation for 16 hours) and gel filtration using Superdex 75 16/60 or 200 16/60 columns (GE Healthcare). Protein concentration was determined spectrophotometrically at 280 nm using the corresponding extinction coefficient.

Expression and Preparations of Proteins

The E. coli SecB gene was cloned into the pET-16b vector (Novagen) containing a His6-tag and a tobacco etch virus (TEV) protease cleavage site at the N-terminus. Protein samples of E. coli PhoA were produced as described before20. All E. coli MBP constructs were cloned into the pET-16b vector containing a His6-tag and a TEV protease cleavage site at the N-terminus. The following MBP constructs were prepared in this study (residue numbers of the boundaries are in superscript): MBP1–396, mature MBP23–396, MBP29–99, MBP67–99, MBP97–164, MBP160–201, MBP198–265, MBP260–336, MBP331–396, and the MBP variants MBPG32D/I33P, MBPY283D and MBPV8G/Y283D (MBP mutants are numbered based on the amino acid sequence of the mature form of MBP). All constructs were transformed into BL21(DE3) cells. Isotopically unlabeled protein samples were produced in cells grown in Luria-Bertani (LB) medium at 37 °C in the presence of ampicillin (100 µg ml−1) to an OD600 ~0.8. Protein induction was induced by the addition of 0.2 mM isopropyl-β-D-1-thiogalactopyranoside (IPTG) and cells were allowed to grow for 16 hours at 18 °C. Cells were harvested at OD600 ~1.5 and resuspended in lysis buffer (50 mM Tris-HCl, 500 mM NaCl, pH 8 and 1 mM PMSF). Cells were disrupted by a high-pressure homogenizer and centrifuged at 50,000g. Proteins were purified using Ni Sepharose 6 Fast Flow resin (GE Healthcare), followed by tag removal by TEV protease at 4°C (incubation for 16 hours) and gel filtration using Superdex 75 16/60 or 200 16/60 columns (GE Healthcare). Protein concentration was determined spectrophotometrically at 280 nm using the corresponding extinction coefficient.

MALS ExperimentsMALS was measured using DAWN HELEOS-II (Wyatt Technology Corporation) downstream of a Shimadzu liquid chromatography system connected to a Superdex 200 10/300 GL (GE Healthcare) gel filtration column. The running buffer for SecB–PhoA complexes was 20 mM KPi (pH 7.0), 100 mM KCl, 4 mM βME, and 0.5 mM EDTA, whereas for SecB–MBP complexes was 20 mM HEPES, pH 7, 150 mM KOAc and 0.05% NaN3. Protein samples at a concentration of 0.05 to 0.2 mM were used. The flow rate was set to 0.5 ml min−1 with an injection volume of 200 µl and the light scattering signal was collected at room temperature (~23 °C). The data were analyzed with ASTRA version 6.0.5 (Wyatt Technology Corporation).

MALS Experiments

MALS was measured using DAWN HELEOS-II (Wyatt Technology Corporation) downstream of a Shimadzu liquid chromatography system connected to a Superdex 200 10/300 GL (GE Healthcare) gel filtration column. The running buffer for SecB–PhoA complexes was 20 mM KPi (pH 7.0), 100 mM KCl, 4 mM βME, and 0.5 mM EDTA, whereas for SecB–MBP complexes was 20 mM HEPES, pH 7, 150 mM KOAc and 0.05% NaN3. Protein samples at a concentration of 0.05 to 0.2 mM were used. The flow rate was set to 0.5 ml min−1 with an injection volume of 200 µl and the light scattering signal was collected at room temperature (~23 °C). The data were analyzed with ASTRA version 6.0.5 (Wyatt Technology Corporation).

ITC ExperimentsITC was carried out using an iTC200 microcalorimeter (GE Healthcare) at temperatures ranging from 4 °C to 25 °C. All protein samples were extensively dialyzed against the ITC buffer containing 50 mM KPi (pH 7.0), 50 mM KCl, 0.05% NaN3 and 2 mM tris(2-carboxyethyl)phosphine (TCEP). All solutions were filtered using membrane filters (pore size, 0.45 µm) and thoroughly degassed for 20 min before the titrations. The 40-µl injection syringe was filled with ~0.05 to 1 mM of SecB solution and the 200-µl cell was filled with ~0.01 to 0.2 mM PhoA or MBP. To measure the binding affinity of MBP to SecB, the slowly folding MBPV8G/Y283D variant was used to measure the affinity of MBP for SecB. MBPV8G/Y283D was unfolded in 8 M urea, 20 mM HEPES, pH 7, 150 mM KOAc and 0.05% NaN3, and diluted 20 times to give a final concentration of 2.7 µM immediately before loading into the cell. The solution containing SecB was precisely adjusted to match the urea concentration. The titrations were carried out with a preliminary 0.2-µl injection, followed typically by 15 injections of 2.5 µl each with time intervals of 3 min. The solution was stirred at 1000 rpm. Data for the preliminary injection, which are affected by diffusion of the solution from and into the injection syringe during the initial equilibration period, were discarded. Binding isotherms were generated by plotting heats of reaction normalized by the modes of injectant versus the ratio of total injectant to total protein per injection. The data were fitted with Origin 7.0 (OriginLab Corporation).

ITC Experiments

ITC was carried out using an iTC200 microcalorimeter (GE Healthcare) at temperatures ranging from 4 °C to 25 °C. All protein samples were extensively dialyzed against the ITC buffer containing 50 mM KPi (pH 7.0), 50 mM KCl, 0.05% NaN3 and 2 mM tris(2-carboxyethyl)phosphine (TCEP). All solutions were filtered using membrane filters (pore size, 0.45 µm) and thoroughly degassed for 20 min before the titrations. The 40-µl injection syringe was filled with ~0.05 to 1 mM of SecB solution and the 200-µl cell was filled with ~0.01 to 0.2 mM PhoA or MBP. To measure the binding affinity of MBP to SecB, the slowly folding MBPV8G/Y283D variant was used to measure the affinity of MBP for SecB. MBPV8G/Y283D was unfolded in 8 M urea, 20 mM HEPES, pH 7, 150 mM KOAc and 0.05% NaN3, and diluted 20 times to give a final concentration of 2.7 µM immediately before loading into the cell. The solution containing SecB was precisely adjusted to match the urea concentration. The titrations were carried out with a preliminary 0.2-µl injection, followed typically by 15 injections of 2.5 µl each with time intervals of 3 min. The solution was stirred at 1000 rpm. Data for the preliminary injection, which are affected by diffusion of the solution from and into the injection syringe during the initial equilibration period, were discarded. Binding isotherms were generated by plotting heats of reaction normalized by the modes of injectant versus the ratio of total injectant to total protein per injection. The data were fitted with Origin 7.0 (OriginLab Corporation).

Protein Isotope Labeling for NMR StudiesIsotopically labeled samples for NMR studies were prepared by growing the cells in minimal (M9) medium. Cells were typically harvested at OD600 ~1.0. U-[2H,13C,15N]–labeled samples were prepared for the backbone assignment of SecB and large MBP fragments by supplementing the growing medium with 15NH4Cl (1 g liter−1) and 2H7,13C6-glucose (2 g liter−1) in 99.9% 2H2O (CIL and Isotec). The 1H-13C methyl labeled samples were prepared as described20,29,31. a-Ketobutyric acid (50 mg liter−1) and a-ketoisovaleric acid (85 mg liter−1) were added to the culture 1 hour before the addition of IPTG. Met-[13CH3]– and Ala-[13CH3]–labeled samples were produced by supplementing the medium with [13CH3]-Met (50 mg liter−1) and [2H2,13CH3]-Ala (50 mg liter−1). For Thr labeling, a Thr-auxotrophic cell strain was used, and the medium was supplemented with [2H2,13CH3]-Thr (25 mg liter−1). For Phe, Tyr, and Trp labeling, U-[1H,13C]–labeled amino acids were used. Alternate 13C-labeling of aromatic residues was performed as described32. All precursors and amino acids were added to the culture 1 hour before the addition of IPTG, except Ala, which was added 30 min before induction.

Protein Isotope Labeling for NMR Studies

Isotopically labeled samples for NMR studies were prepared by growing the cells in minimal (M9) medium. Cells were typically harvested at OD600 ~1.0. U-[2H,13C,15N]–labeled samples were prepared for the backbone assignment of SecB and large MBP fragments by supplementing the growing medium with 15NH4Cl (1 g liter−1) and 2H7,13C6-glucose (2 g liter−1) in 99.9% 2H2O (CIL and Isotec). The 1H-13C methyl labeled samples were prepared as described20,29,31. a-Ketobutyric acid (50 mg liter−1) and a-ketoisovaleric acid (85 mg liter−1) were added to the culture 1 hour before the addition of IPTG. Met-[13CH3]– and Ala-[13CH3]–labeled samples were produced by supplementing the medium with [13CH3]-Met (50 mg liter−1) and [2H2,13CH3]-Ala (50 mg liter−1). For Thr labeling, a Thr-auxotrophic cell strain was used, and the medium was supplemented with [2H2,13CH3]-Thr (25 mg liter−1). For Phe, Tyr, and Trp labeling, U-[1H,13C]–labeled amino acids were used. Alternate 13C-labeling of aromatic residues was performed as described32. All precursors and amino acids were added to the culture 1 hour before the addition of IPTG, except Ala, which was added 30 min before induction.

NMR SpectroscopyNMR samples were typically prepared in 50 mM KPi (pH 7.0), 50 mM KCl, 0.05% NaN3, 5mM βME and 7% D2O. NMR experiments were recorded on Bruker 900, 850 and 700 MHz spectrometers. NMR spectra were typically recorded at 10 °C for the isolated PhoA and MBP fragments and at 35 °C for SecB and its complexes. Protein sample concentration ranged from 0.1 to 1.0 mM. All NMR spectra were processed using NMRPipe33 and analyzed using NMRView (http://www.onemoonscientific.com).

NMR Spectroscopy

NMR samples were typically prepared in 50 mM KPi (pH 7.0), 50 mM KCl, 0.05% NaN3, 5mM βME and 7% D2O. NMR experiments were recorded on Bruker 900, 850 and 700 MHz spectrometers. NMR spectra were typically recorded at 10 °C for the isolated PhoA and MBP fragments and at 35 °C for SecB and its complexes. Protein sample concentration ranged from 0.1 to 1.0 mM. All NMR spectra were processed using NMRPipe33 and analyzed using NMRView (http://www.onemoonscientific.com).

NMR Assignment of SecBSecB tetramer packs as a dimer of dimers and gives rise to two pairs of magnetically equivalent subunits: A and D give one set of resonances and subunits B and C give another set of resonances (Extended Data Fig. 1a). Sequential backbone assignment of SecB was achieved by the use of standard triple-resonance NMR pulse sequences. Three-dimensional (3D) 1H-15N NOESY experiments were used to confirm and extend the backbone assignment within each subunit. Side-chain assignment for methyls and aromatic residues was accomplished using the following NMR experiments: 3D (1H)-13C heteronuclear multiple-quantum coherence (HMQC)-NOESY-1H-13C HMQC, 13C-edited NOESY-HSQC, 13C-edited HSQC-NOESY, 15N-edited NOESY-HSQC, 3D (1H)-13C HSQC-NOESY-1H-15N HSQC, and 3D (1H)-15N HSQC-NOESY-1H-13C HSQC.

NMR Assignment of SecB

SecB tetramer packs as a dimer of dimers and gives rise to two pairs of magnetically equivalent subunits: A and D give one set of resonances and subunits B and C give another set of resonances (Extended Data Fig. 1a). Sequential backbone assignment of SecB was achieved by the use of standard triple-resonance NMR pulse sequences. Three-dimensional (3D) 1H-15N NOESY experiments were used to confirm and extend the backbone assignment within each subunit. Side-chain assignment for methyls and aromatic residues was accomplished using the following NMR experiments: 3D (1H)-13C heteronuclear multiple-quantum coherence (HMQC)-NOESY-1H-13C HMQC, 13C-edited NOESY-HSQC, 13C-edited HSQC-NOESY, 15N-edited NOESY-HSQC, 3D (1H)-13C HSQC-NOESY-1H-15N HSQC, and 3D (1H)-15N HSQC-NOESY-1H-13C HSQC.

NMR assignment of PhoA and MBP in the unfolded stateWe previously described the assignment strategy for unfolded PhoA20. We followed a similar strategy to assign MBP in the unfolded state by making use of several MBP fragments that remain soluble and unfolded when isolated (Extended Data Fig. 1c): MBP29–99, MBP67–99, MBP97–164, MBP160–201, MBP198–265, MBP260–336 and MBP331–396. Isolated MBP fragments encompassing the first 23 N-terminal residues (signal sequence) were not stable and this region could only be assigned in complex with SecB. Overlay of the spectra of the MBP fragments with the spectra of full-length MBP in 4M urea indicated very good resonance correspondence. This is expected because all of the fragments as well as the MBP in 4 M urea are unfolded. Resonance assignment obtained for the various fragments was transferred to full length MBP in urea, and ambiguities were resolved by the use of 3D NMR spectra. It should be noted that although resonance dispersion in unliganded PhoA and MBP is poor, complex formation with SecB alleviates this problem (for the PhoA and MBP residues in the SecB-binding regions) with the spectra being of high resolution (Extended Data Fig. 4c).

NMR assignment of PhoA and MBP in the unfolded state

We previously described the assignment strategy for unfolded PhoA20. We followed a similar strategy to assign MBP in the unfolded state by making use of several MBP fragments that remain soluble and unfolded when isolated (Extended Data Fig. 1c): MBP29–99, MBP67–99, MBP97–164, MBP160–201, MBP198–265, MBP260–336 and MBP331–396. Isolated MBP fragments encompassing the first 23 N-terminal residues (signal sequence) were not stable and this region could only be assigned in complex with SecB. Overlay of the spectra of the MBP fragments with the spectra of full-length MBP in 4M urea indicated very good resonance correspondence. This is expected because all of the fragments as well as the MBP in 4 M urea are unfolded. Resonance assignment obtained for the various fragments was transferred to full length MBP in urea, and ambiguities were resolved by the use of 3D NMR spectra. It should be noted that although resonance dispersion in unliganded PhoA and MBP is poor, complex formation with SecB alleviates this problem (for the PhoA and MBP residues in the SecB-binding regions) with the spectra being of high resolution (Extended Data Fig. 4c).

Structure Determination of SecB–PhoA and SecB–MBP ComplexesAssignment of the resonances in SecB–PhoA was accomplished by first assigning the complexes between SecB and the individual PhoA sites (SecB–PhoAa, SecB–PhoAc, SecB–PhoAd, SecB–PhoAe). We used U-12C,15N–labeled samples that contained specifically protonated methyl groups of Ala, Val, Leu, Met, Thr, and Ile (δ1) and protonated aromatic residues Phe, Tyr, and Trp in an otherwise deuterated background. The high sensitivity and resolution of the methyl region, combined with the high abundance of these nine amino acids in SecB (Extended Data Fig. 1a) and in the SecB-binding sites of PhoA and MBP, provided a large number of intermolecular NOEs for the SecB–PhoA and SecB–MBP complexes (Extended Data Table 1). Because PhoA in complex with SecB provided higher quality spectra than the spectra of MBP in complex with SecB, we determined first the structure of the SecB–PhoA complex (~120 kDa) by NMR. We initially characterized the structure of the each PhoA site (a through e) individually in complex with SecB (Extended Data Fig. 5). The structures of SecB–PhoAa, SecB–PhoAc, SecB–PhoAd, and SecB–PhoAe, were determined by NMR and presented in Extended Data Fig. 5. A large number of inter-molecular NOEs were collected for each one of the complexes (Extended Data Table 1). Because of the relatively short length of the polypeptides encompassing the individual PhoA sites, multiple PhoA molecules bound to SecB, as shown in Extended Data Fig. 5. We should also note that we detected the presence of a small number of inter-molecular NOEs that were suggestive of alternative conformations of the PhoA sites bound to SecB. However, the intensity of these sets of NOEs is much weaker indicating that the population of such alternative complexes is low. To solve the structure of the SecB–PhoA complex we sought to determine how each one of the PhoA sites binds to SecB in the context of the full length PhoA. To circumvent the signal overlap in this large complex, we used samples where the two proteins were isotopically labeled in different amino acids. For example, in one of these samples SecB was labeled in the methyls of Leu, Val and Met, whereas PhoA in the methyls of Ile amino acids. Because of the distinct chemical shifts of 1H and 13C resonances of the methyls and the isotope labeling scheme, it was possible to measure specific intermolecular NOEs between SecB and PhoA (Extended Data Fig. 4b). Several of these samples were used in order to determine as many inter-molecular NOEs as possible. As expected, the NOEs were compatible with the structure of each PhoA site in complex with SecB, with the crucial difference that only one PhoA molecule could be accommodated in SecB. Due to its short length, the isolated PhoA site b (PhoAb) binds to almost all of the exposed hydrophobic surface of SecB, as determined by NMR. In the SecB–PhoA complex with SecB, PhoA site b can only bind to the secondary binding site, as determined by NOEs. To further corroborate the structure of the SecB–PhoA complex we used PRE data (see below). The PRE-derived distances were fully compatible with the NOE data collected on SecB–PhoA. The structure of the SecB–PhoA complex was determined using the set of inter-molecular NOEs collected directly in the complex and further refined using the inter-molecular NOEs collected for the corresponding isolated PhoA sites in complex with SecB. It should be noted that because of the symmetry in SecB, the various PhoA sites may bind to any of the four SecB subunits. The final arrangement will be dictated by the length of the linkers tethering the SecB-recognition sites (as shown in Fig. 2), namely how far nearby recognition sites can bind from each other, and thus alternative routes of the polypeptide bound to SecB may be present. The only conceivable difference among the various conformations is the relative disposition of the PhoA sites. In all cases all of the SecB-recognition sites in PhoA are engaged by SecB in the complex and PhoA wraps around SecB. The NMR-driven structural model of the SecB–MBP complex (Extended Data Fig. 7b) was determined as follows: NMR analysis demonstrated that all seven recognition sites in MBP (labeled a through g) are bound to SecB in the SecB–MBP complex (Extended Data Fig. 7a). We have determined the high-resolution structure of MBPd and MBPe in complex with SecB (Extended Data Fig. 6). Because of their length and the short linker tethering the two sites, d and e sites most likely bind to the same side of SecB. MBP site f is the longest one, consisting of ~90 residues, and is thus entirely accommodated on the other side of SecB. With sites d, e and f occupying the primary binding sites, the other recognition sites (a, b, c and g) being much shorter can be accommodated within the secondary client-binding sites on SecB. The structure of MBP sites d and e in complex with SecB was determined using the experimental inter-molecular NOE data. The hydrophobic residues of the sites a, b, c, f, and g showing the strongest effect upon SecB binding, as determined by differential line broadening, were used to drive the docking of these sites to nonpolar residues on SecB. The modeled structure shows that the entire MBP sequence can be accommodated within one SecB molecule. The structures of SecB in complex with PhoA and MBP were calculated with CYANA 3.9734, using NOE peak lists from 3D (1H)-13C HMQC-NOESY–1H-13C HMQC, 3D (1H)-15N HSQC-NOESY–1H-13C HSQC, 13C–edited NOESY-HSQC, and 15N–edited NOESY-HSQC. The 13Cα, 13Cβ, 13C’, 15N, and NH chemical shifts served as input for the TALOS+ program35 to extract dihedral angles (φ and ψ). The side chains of SecB residues within or nearby the PhoA and MBP binding sites were set flexible and their conformation was determined using intermolecular NOEs collected for each one of the complexes. The SecB regions remote to the binding sites were set rigid using the crystal structure coordinates for E. coli SecB26. The 20 lowest-energy structures were refined by restrained molecular dynamics in explicit water with CNS36. The percentage of residues falling in favoured and disallowed regions, respectively, of the Ramachandran plot is as follows: 99.4% and 0.6% for SecB–PhoA; 99.4% and 0.6% for SecB–PhoAa; 99.3% and 0.7% for SecB–PhoAc; 99.2% and 0.8% for SecB–PhoAd; 99.3% and 0.7% for SecB–PhoAe; 99.4% and 0.6% for SecB–MBPd; and 99.4% and 0.6% for SecB–MBPe.

Structure Determination of SecB–PhoA and SecB–MBP Complexes

Assignment of the resonances in SecB–PhoA was accomplished by first assigning the complexes between SecB and the individual PhoA sites (SecB–PhoAa, SecB–PhoAc, SecB–PhoAd, SecB–PhoAe). We used U-12C,15N–labeled samples that contained specifically protonated methyl groups of Ala, Val, Leu, Met, Thr, and Ile (δ1) and protonated aromatic residues Phe, Tyr, and Trp in an otherwise deuterated background. The high sensitivity and resolution of the methyl region, combined with the high abundance of these nine amino acids in SecB (Extended Data Fig. 1a) and in the SecB-binding sites of PhoA and MBP, provided a large number of intermolecular NOEs for the SecB–PhoA and SecB–MBP complexes (Extended Data Table 1). Because PhoA in complex with SecB provided higher quality spectra than the spectra of MBP in complex with SecB, we determined first the structure of the SecB–PhoA complex (~120 kDa) by NMR. We initially characterized the structure of the each PhoA site (a through e) individually in complex with SecB (Extended Data Fig. 5). The structures of SecB–PhoAa, SecB–PhoAc, SecB–PhoAd, and SecB–PhoAe, were determined by NMR and presented in Extended Data Fig. 5. A large number of inter-molecular NOEs were collected for each one of the complexes (Extended Data Table 1). Because of the relatively short length of the polypeptides encompassing the individual PhoA sites, multiple PhoA molecules bound to SecB, as shown in Extended Data Fig. 5. We should also note that we detected the presence of a small number of inter-molecular NOEs that were suggestive of alternative conformations of the PhoA sites bound to SecB. However, the intensity of these sets of NOEs is much weaker indicating that the population of such alternative complexes is low. To solve the structure of the SecB–PhoA complex we sought to determine how each one of the PhoA sites binds to SecB in the context of the full length PhoA. To circumvent the signal overlap in this large complex, we used samples where the two proteins were isotopically labeled in different amino acids. For example, in one of these samples SecB was labeled in the methyls of Leu, Val and Met, whereas PhoA in the methyls of Ile amino acids. Because of the distinct chemical shifts of 1H and 13C resonances of the methyls and the isotope labeling scheme, it was possible to measure specific intermolecular NOEs between SecB and PhoA (Extended Data Fig. 4b). Several of these samples were used in order to determine as many inter-molecular NOEs as possible. As expected, the NOEs were compatible with the structure of each PhoA site in complex with SecB, with the crucial difference that only one PhoA molecule could be accommodated in SecB. Due to its short length, the isolated PhoA site b (PhoAb) binds to almost all of the exposed hydrophobic surface of SecB, as determined by NMR. In the SecB–PhoA complex with SecB, PhoA site b can only bind to the secondary binding site, as determined by NOEs. To further corroborate the structure of the SecB–PhoA complex we used PRE data (see below). The PRE-derived distances were fully compatible with the NOE data collected on SecB–PhoA. The structure of the SecB–PhoA complex was determined using the set of inter-molecular NOEs collected directly in the complex and further refined using the inter-molecular NOEs collected for the corresponding isolated PhoA sites in complex with SecB. It should be noted that because of the symmetry in SecB, the various PhoA sites may bind to any of the four SecB subunits. The final arrangement will be dictated by the length of the linkers tethering the SecB-recognition sites (as shown in Fig. 2), namely how far nearby recognition sites can bind from each other, and thus alternative routes of the polypeptide bound to SecB may be present. The only conceivable difference among the various conformations is the relative disposition of the PhoA sites. In all cases all of the SecB-recognition sites in PhoA are engaged by SecB in the complex and PhoA wraps around SecB. The NMR-driven structural model of the SecB–MBP complex (Extended Data Fig. 7b) was determined as follows: NMR analysis demonstrated that all seven recognition sites in MBP (labeled a through g) are bound to SecB in the SecB–MBP complex (Extended Data Fig. 7a). We have determined the high-resolution structure of MBPd and MBPe in complex with SecB (Extended Data Fig. 6). Because of their length and the short linker tethering the two sites, d and e sites most likely bind to the same side of SecB. MBP site f is the longest one, consisting of ~90 residues, and is thus entirely accommodated on the other side of SecB. With sites d, e and f occupying the primary binding sites, the other recognition sites (a, b, c and g) being much shorter can be accommodated within the secondary client-binding sites on SecB. The structure of MBP sites d and e in complex with SecB was determined using the experimental inter-molecular NOE data. The hydrophobic residues of the sites a, b, c, f, and g showing the strongest effect upon SecB binding, as determined by differential line broadening, were used to drive the docking of these sites to nonpolar residues on SecB. The modeled structure shows that the entire MBP sequence can be accommodated within one SecB molecule. The structures of SecB in complex with PhoA and MBP were calculated with CYANA 3.9734, using NOE peak lists from 3D (1H)-13C HMQC-NOESY–1H-13C HMQC, 3D (1H)-15N HSQC-NOESY–1H-13C HSQC, 13C–edited NOESY-HSQC, and 15N–edited NOESY-HSQC. The 13Cα, 13Cβ, 13C’, 15N, and NH chemical shifts served as input for the TALOS+ program35 to extract dihedral angles (φ and ψ). The side chains of SecB residues within or nearby the PhoA and MBP binding sites were set flexible and their conformation was determined using intermolecular NOEs collected for each one of the complexes. The SecB regions remote to the binding sites were set rigid using the crystal structure coordinates for E. coli SecB26. The 20 lowest-energy structures were refined by restrained molecular dynamics in explicit water with CNS36. The percentage of residues falling in favoured and disallowed regions, respectively, of the Ramachandran plot is as follows: 99.4% and 0.6% for SecB–PhoA; 99.4% and 0.6% for SecB–PhoAa; 99.3% and 0.7% for SecB–PhoAc; 99.2% and 0.8% for SecB–PhoAd; 99.3% and 0.7% for SecB–PhoAe; 99.4% and 0.6% for SecB–MBPd; and 99.4% and 0.6% for SecB–MBPe.

PRE ExperimentsPRE experiments were used to confirm the position of each individual PhoA binding site in the SecB–PhoA complex. First, a “Cys-free” variant of PhoA was prepared by mutating the four naturally-occurring Cys residues in PhoA (Cys190, Cys200, Cys308 and Cys358) to Ser. We then introduced a Cys residue to either end of each SecB-binding site in PhoA and prepared a total of ten single-Cys mutants: T5C, T23C, K65C, M75C, G91C, G140C, Q274C, C308, N450C and C472. The protein purified from Ni-NTA column was quickly concentrated and loaded onto HiLoad® 16/60 Superdex® 200 gel filtration column (GE healthcare) using a buffer containing 50 mM KPi (pH 7.0), 150 mM NaCl, and 0.05% NaN3. Immediately after elution the purified single-Cys PhoA mutant was divided into two equal portions for parallel treatment with (1-oxyl- 2,2,5,5-tetramethyl-3-pyrroline-3-methyl)-ethanethiosulfonate (MTSL, Toronto Research Chemicals, Toronto) and a diamagnetic MTSL analog, in a 10-fold molar excess at 4 °C for 16–20 hrs. MTSL was prepared in a 50 mM concentrated stock in acetonitrile. Free MTSL was removed by extensive buffer exchange using Centricon® Centrifugal Filter with a MWCO of 10,000 (Millipore) at 4 °C. The MTSL-labeled PhoA protein samples were then concentrated and added into the 2H-methyl-13CH3-labeled SecB at a final molar ratio of PhoA: SecB = 1:1. 2D 1H,13C HMQC spectra were recorded at 28 °C. A sample of SecB in complex with PhoA cross-linked to a diamagnetic MTSL analog was used as a reference. Residues experienced significant NMR signal intensity reduction (>50% intensity loss) were identified as sites being within 20 Å of the paramagnetic center whereas residues experiencing more than 90% intensity loss were identified as sites being within 14 Å of the paramagnetic center.

PRE Experiments

PRE experiments were used to confirm the position of each individual PhoA binding site in the SecB–PhoA complex. First, a “Cys-free” variant of PhoA was prepared by mutating the four naturally-occurring Cys residues in PhoA (Cys190, Cys200, Cys308 and Cys358) to Ser. We then introduced a Cys residue to either end of each SecB-binding site in PhoA and prepared a total of ten single-Cys mutants: T5C, T23C, K65C, M75C, G91C, G140C, Q274C, C308, N450C and C472. The protein purified from Ni-NTA column was quickly concentrated and loaded onto HiLoad® 16/60 Superdex® 200 gel filtration column (GE healthcare) using a buffer containing 50 mM KPi (pH 7.0), 150 mM NaCl, and 0.05% NaN3. Immediately after elution the purified single-Cys PhoA mutant was divided into two equal portions for parallel treatment with (1-oxyl- 2,2,5,5-tetramethyl-3-pyrroline-3-methyl)-ethanethiosulfonate (MTSL, Toronto Research Chemicals, Toronto) and a diamagnetic MTSL analog, in a 10-fold molar excess at 4 °C for 16–20 hrs. MTSL was prepared in a 50 mM concentrated stock in acetonitrile. Free MTSL was removed by extensive buffer exchange using Centricon® Centrifugal Filter with a MWCO of 10,000 (Millipore) at 4 °C. The MTSL-labeled PhoA protein samples were then concentrated and added into the 2H-methyl-13CH3-labeled SecB at a final molar ratio of PhoA: SecB = 1:1. 2D 1H,13C HMQC spectra were recorded at 28 °C. A sample of SecB in complex with PhoA cross-linked to a diamagnetic MTSL analog was used as a reference. Residues experienced significant NMR signal intensity reduction (>50% intensity loss) were identified as sites being within 20 Å of the paramagnetic center whereas residues experiencing more than 90% intensity loss were identified as sites being within 14 Å of the paramagnetic center.

Protein Folding AssaysRefolding experiments of MBP were performed as described before37 with some modifications. Briefly, MBP was first denatured in 8 M urea, 100 mM HEPES, 20 mM KOAc, 5 mM Mg(OAc)2, pH 7.4, and 0.05% NaN3). Refolding was initiated by rapid dilution (20 times dilution) in the urea-free buffer and the refolding process of MBP in the absence and presence of SecB or TF was monitored by the change in the intrinsic Trp fluorescence. Fluorescence intensity was measured using either a spectrofluorometer (FluoroMax-4, Horiba) or a microplate reader (Infinite 200 PRO, Tecan). The excitation and emission wavelengths were set to 295 nm and 345 nm, respectively. For measurement using the FluoroMax-4 instrument, the MBP concentration in the 1-ml cuvette was 0.4 µM, whereas for the microplate reader experiments the concentration of MBP was 4 µM in the 30 µl-plate well. All fluorescence measurements were performed at 25°C. Data were fitted by the Prism 6 (GraphPad) software using the non-linear regression analysis equation38.

Protein Folding Assays

Refolding experiments of MBP were performed as described before37 with some modifications. Briefly, MBP was first denatured in 8 M urea, 100 mM HEPES, 20 mM KOAc, 5 mM Mg(OAc)2, pH 7.4, and 0.05% NaN3). Refolding was initiated by rapid dilution (20 times dilution) in the urea-free buffer and the refolding process of MBP in the absence and presence of SecB or TF was monitored by the change in the intrinsic Trp fluorescence. Fluorescence intensity was measured using either a spectrofluorometer (FluoroMax-4, Horiba) or a microplate reader (Infinite 200 PRO, Tecan). The excitation and emission wavelengths were set to 295 nm and 345 nm, respectively. For measurement using the FluoroMax-4 instrument, the MBP concentration in the 1-ml cuvette was 0.4 µM, whereas for the microplate reader experiments the concentration of MBP was 4 µM in the 30 µl-plate well. All fluorescence measurements were performed at 25°C. Data were fitted by the Prism 6 (GraphPad) software using the non-linear regression analysis equation38.

Surface plasmon resonance (SPR)All SPR experiments were carried out on a Biacore T200 system (GE Healthcare) using a NTA-coated Sensor Chip NTA (GE Healthcare) at a flow rate of 50 µl min−1. The PhoA protein sample used for SPR experiments was genetically constructed with a His12-tag at the C-terminus and a flexible (Gly-Ser)5 linker repeat inserted in between to avoid steric hindrance. A single-cycle kinetic procedure was used to characterize the interaction of SecB and PhoA. The His-tagged PhoA was immobilized onto a NTA sensor chip, followed by washing with the running buffer containing 50 mM Phosphate, 50 mM KCl, pH 7, 0.05% NaN3, and 2 mM TCEP. The reducing agent (TCEP) ensured that PhoA is in the unfolded state20. SecB (analyte) at a range of concentrations (0.1 µM to 25.6 µM) were injected, and data for a period of 30 seconds of association and 60 seconds of dissociation were collected. MBP was prepared with a His10- tag at the N-terminus followed by a flexible nine-residue linker to avoid steric hindrance. Multiple-cycle kinetic analysis was performed for the SPR experiments of the binding between MBP and SecB where each sample concentration was run in a separate cycle, and the surface was regenerated between each cycle using NTA regeneration buffer. His-tagged MBP was denatured in 8 M urea and immobilized onto a NTA sensor chip. Urea was quickly washed away by running buffer containing 20 mM HEPES, pH 7.4, 150 mM KOAc, and 0.05% NaN3. SecB was injected at concentrations ranging from 2.5 nM to 1.6 µM. The association and dissociation time for data collection was set as 90 s and 120 s, respectively. After urea was removed MBP remained in the unfolded conformation for sufficient time to interact with SecB. This was confirmed by monitoring the refolding behavior of MBP using an Infinite 200 PRO microplate reader (Tecan) at the temperature range of the experiments. All SPR experiments were repeated three times and highly reproducible data were obtained. The sensorgrams obtained from the assay channel were subtracted by the buffer control, and data were fitted using the Biacore T200 evaluation software (version 1.0).

Surface plasmon resonance (SPR)

All SPR experiments were carried out on a Biacore T200 system (GE Healthcare) using a NTA-coated Sensor Chip NTA (GE Healthcare) at a flow rate of 50 µl min−1. The PhoA protein sample used for SPR experiments was genetically constructed with a His12-tag at the C-terminus and a flexible (Gly-Ser)5 linker repeat inserted in between to avoid steric hindrance. A single-cycle kinetic procedure was used to characterize the interaction of SecB and PhoA. The His-tagged PhoA was immobilized onto a NTA sensor chip, followed by washing with the running buffer containing 50 mM Phosphate, 50 mM KCl, pH 7, 0.05% NaN3, and 2 mM TCEP. The reducing agent (TCEP) ensured that PhoA is in the unfolded state20. SecB (analyte) at a range of concentrations (0.1 µM to 25.6 µM) were injected, and data for a period of 30 seconds of association and 60 seconds of dissociation were collected. MBP was prepared with a His10- tag at the N-terminus followed by a flexible nine-residue linker to avoid steric hindrance. Multiple-cycle kinetic analysis was performed for the SPR experiments of the binding between MBP and SecB where each sample concentration was run in a separate cycle, and the surface was regenerated between each cycle using NTA regeneration buffer. His-tagged MBP was denatured in 8 M urea and immobilized onto a NTA sensor chip. Urea was quickly washed away by running buffer containing 20 mM HEPES, pH 7.4, 150 mM KOAc, and 0.05% NaN3. SecB was injected at concentrations ranging from 2.5 nM to 1.6 µM. The association and dissociation time for data collection was set as 90 s and 120 s, respectively. After urea was removed MBP remained in the unfolded conformation for sufficient time to interact with SecB. This was confirmed by monitoring the refolding behavior of MBP using an Infinite 200 PRO microplate reader (Tecan) at the temperature range of the experiments. All SPR experiments were repeated three times and highly reproducible data were obtained. The sensorgrams obtained from the assay channel were subtracted by the buffer control, and data were fitted using the Biacore T200 evaluation software (version 1.0).

Bio-Layer Interferometry (BLI)BLI experiments were performed using an Octet system (forteBIO) at room temperature (~23 °C). MBP was biotinylated using the biotination kit EZ-Link™ NHS-PEG4-Biotin (Thermo Fisher Scientific). Biotin label freshly dissolved in water was added to the protein solution to a final molar ratio of 1:1 in buffer containing 50 mM KPi, pH 7, 150 mM NaCl, 0.05% NaN3, and the solution was mixed at room temperature for 45 min. Unlabeled biotin label was removed by extensive buffer exchange using Centricon® Centrifugal Filter with a MWCO of 10,000 (Millipore) at 4 °C using a buffer containing 20 mM HEPES (pH 7), 150 mM KoAc, and 0.05% NaN3. Biotin-labeled MBP (200 nM) denatured in 8 M urea was immobilized onto the Streptavidin (SA) biosensor, and the biosensors were subsequently blocked with Biocytin in 8 M urea solution before a quick 30 s dip into the urea-free buffer. SecB or TF previously diluted was applied in a dose dependent manner to the biosensors immobilized with MBP. Bovine serum albumin (BSA) powder (Sigma-Aldrich) was added to a final concentration of 2% to avoid nonspecific interaction. Parallel experiments were performed for reference sensors with no MBP captured and the signals were subsequently subtracted during data analysis. The association and dissociation periods were set to 2 min and 5 min, respectively.

Bio-Layer Interferometry (BLI)

BLI experiments were performed using an Octet system (forteBIO) at room temperature (~23 °C). MBP was biotinylated using the biotination kit EZ-Link™ NHS-PEG4-Biotin (Thermo Fisher Scientific). Biotin label freshly dissolved in water was added to the protein solution to a final molar ratio of 1:1 in buffer containing 50 mM KPi, pH 7, 150 mM NaCl, 0.05% NaN3, and the solution was mixed at room temperature for 45 min. Unlabeled biotin label was removed by extensive buffer exchange using Centricon® Centrifugal Filter with a MWCO of 10,000 (Millipore) at 4 °C using a buffer containing 20 mM HEPES (pH 7), 150 mM KoAc, and 0.05% NaN3. Biotin-labeled MBP (200 nM) denatured in 8 M urea was immobilized onto the Streptavidin (SA) biosensor, and the biosensors were subsequently blocked with Biocytin in 8 M urea solution before a quick 30 s dip into the urea-free buffer. SecB or TF previously diluted was applied in a dose dependent manner to the biosensors immobilized with MBP. Bovine serum albumin (BSA) powder (Sigma-Aldrich) was added to a final concentration of 2% to avoid nonspecific interaction. Parallel experiments were performed for reference sensors with no MBP captured and the signals were subsequently subtracted during data analysis. The association and dissociation periods were set to 2 min and 5 min, respectively.

Extended DataExtended Data Figure 1NMR characterization of SecB and unfolded MBPa, SecB is enriched in hydrophobic amino acids, such as methyl-bearing (Ala, Ile, Leu, Met, Thr, and Val) and aromatic (Phe and Tyr). b, 1H-15N TROSY HSQC (left) and 1H-13C methyl HMQC (right) spectra of [U-2H; Ala-13CH3; Met-13CH3; Ile-δ1-13CH3; Leu,Val-13CH3/13CH3; Thr-13CH3]-labeled SecB. SecB packing gives rise to two pairs of spectroscopically equivalent subunits: one pair is formed by subunits A and D, and the other pair by subunits B and C. Select assignment is included in the methyl spectrum with the asterisk indicating the other pair. c, 1H-15N HSQC spectra of select MBP fragments spanning the entire sequence of MBP. d, Secondary structure propensity (SSP) values39,40 of unfolded MBP (extracted collectively from the fragments) plotted as a function of the amino acid sequence. A SSP score at a given residue of 1 or −1 reflects a fully formed α-helical or β-structure (extended), respectively, whereas a score of, for example, 0.5 indicates that 50% of the conformers in the native-state ensemble of the protein are helical at that position. The data show that several of the secondary structure elements in the folded MBP retain some transient secondary structure in the unfolded MBP fragments.Extended Data Figure 2NMR characterization of PhoA and MBP binding to SecBa, To determine the SecB-recognition sites within PhoA and MBP 15N labeled PhoA and MBP fragments were titrated with unlabeled SecB. Due to the labeling scheme and the size of SecB, the intensity of the PhoA and MBP residues that are bound by SecB decreases dramatically or disappears. Several titration points were recorded but here only the spectra for the SecB:PhoA and SecB:MBP 1:1 are shown for two select fragments. The 1H-15N HSQC spectra of PhoA or MBP are shown in the absence (blue) and presence (red) of SecB. b–c, PhoA (b) and MBP (c) refolding in the presence and absence of SecB monitored by 1H-15N HSQC spectra. Spectra of the “refolded” state were recorded after rapid dilution of urea-treated MBP/PhoA in native buffer. Spectra of the “unfolded” state were recorded in urea. MBP and PhoA refolded in the their native structure in the absence of SecB but were retained in the unfolded state in the presence of SecB.Extended Data Figure 3Energetics of SecB interaction with PhoA and MBPa, MALS of SecB–PhoA complex showing a stoichiometry of 1:1. b, ITC of SecB binding to PhoA and the energetics of binding. c, Kd values for complexes between select PhoA fragments encompassing the five (a through e) SecB-recognition sites and SecB. d, MALS of SecB–MBP complex showing a stoichiometry of 1:1. e, ITC of SecB binding to MBP and the energetics of binding. f, Kd values for complexes between select MBP fragments encompassing the seven (a through g) SecB-recognition sites and SecB. More than one of the smaller PhoA or MBP fragments (e.g. PhoAc, PhoAd-e, MBPc-d) can be accommodated within SecB. Of note is the large favorable enthalpy of binding for the interaction of MBP and PhoA with SecB reflecting the large interacting surface. However, a large but unfavorable entropy diminishes the overall binding.Extended Data Figure 4NMR characterization of the SecB–PhoA complexa, 1H-15N TROSY HSQC spectra of PhoA in the unfolded state (light blue) and in complex with SecB (grey). The unfolded state was induced by the addition of reducing agent20 or urea and assigned and characterized by NMR as shown before20. Select resonance assignment of SecB-recognition sites in PhoA is included (the color is per the color code for each SecB-recognition site within PhoA; see Fig. 1b). There is an excellent correspondence between the PhoA residues identified to bind to SecB using the various PhoA fragments (Extended Data Fig. 2a) and the residues of full-length PhoA that are bound to SecB in the SecB–PhoA complex. All five SecB-recognition sites in PhoA (a through e) are engaged by SecB in the SecB–PhoA complex. The PhoA regions that are not bound to SecB (they retain their intensity in the complex) are all in an unfolded conformation as suggested by their essentially identical chemical shifts to the unfolded PhoA. b, Select strips from 13C-edited NOESY experiments highlighting inter-molecular NOEs in the SecB–PhoA complex. Owing to severe resonance overlap in the 120 kD SecB–PhoA complex, in order to identify specific inter-molecular NOEs we prepared samples wherein the two protein partners are labeled in different methyl-bearing type of amino acids. In this example, SecB was labeled in Leu, Met and Val residues and PhoA in Ile residues. Thus, all NOEs detected between Leu/Val/Met and Ile methyls are inter-molecular. c, 1H-13C methyl HMQC spectra of SecB in complex with PhoA fragments carrying the individual PhoA sites: PhoAa (green), PhoAc (orange), PhoAd (magenta) and PhoAe (red). Both SecB and PhoA fragments are [U-2H; Ala-13CH3; Met-13CH3; Ile-δ1-13CH3; Leu,Val-13CH3/13CH3; Thr-13CH3]-labeled. d, Representative strips from 13C-edited NOESY-HSQC and HMQC-NOESY-HMQC NMR experiments. The NOE cross-peaks between SecB and residues of PhoA fragments are designated by a dashed-line red circle. e, Characteristic NOEs showing that the primary binding groove in SecA is enlarged by the displacement of helix α2 as shown in Figure 4a. For example, the NOE between SecB residues Ala95 and Phe137 is consistent with the closed conformation observed in apo SecB. This NOE is not present in the SecB–PhoA complex because the two SecB residues have moved apart as a result of the displacement of the helix α2.Extended Data Figure 5Strategy for the structure determination of the SecB–PhoA complexThe three main steps are briefly described here. More details can be found in Methods. The lowest-energy NMR structures of the SecB complexes with the individual PhoA sites a, c, d, and e are shown. The structural and NMR statistics for each structure are shown in Extended Data Table 1.Extended Data Figure 6Structures of SecB with MBP sitesa, Lowest-energy structure of SecB in complex with a MBP fragment encompassing site d (MBPd, residues 105–152). b, Lowest-energy structure of SecB in complex with a MBP fragment encompassing site e (MBPe, residues 165–210). SecB is shown as grey solvent-accessible surface (left) or as white cartoon (right). Expanded views (right) of the contacts between SecB and MBP. The SecB residues mediating contacts with MBP are shown as blue ball-and-stick. In both complexes an additional MBP molecule binds symmetrically to the opposite face of SecB but are not shown for clarity.Extended Data Figure 7NMR-driven model structure of SecB–MBP complexa, 1H-15N TROSY HSQC spectra of MBP fragments (grey), MBP fragments in complex with SecB (blue) and full-length MBP in complex with SecB (magenta). The Gly (left) and Trp Nε (right) regions are shown as examples because of the excellent dispersion and lack of severe resonance overlap. The various MBP fragments covering the entire MBP sequence (Extended Data Fig. 1c) are colored grey and if they are located within a SecB-recognition site it is denoted in the superscript. The MBP residues that do not interact with SecB retain their intensity. These are residues located in regions that are not SecB-recognition sites (Fig. 1c). When these spectra are compared with the spectra of full-length MBP in complex with SecB (in magenta) a very good resonance correspondence is observed. Thus, two important observations can be made: first, all seven SecB-recognition sites (a through g) in MBP are engaged by SecB in the SecB–MBP complex; and, second, the MBP regions that do not interact with SecB in the SecB–MBP complex remain in an unfolded state. The Trp spectra (right) provide direct evidence in support of these observations: all Trp residues, with the exception of Trp155, are located in SecB-recognitions sites and they all interact with SecB in the SecB–MBP complex. In contrast, Trp155 does not bind to SecB when the corresponding MBP fragment was used, and this also the case for MBP. b, Modeled structure of the SecB–MBP complex. SecB is shown as a solvent-exposed surface and MBP as a pink ribbon. The seven MBP sites recognized by SecB are shown as sidechain surface and colored per the color code in the graphic of the MBP sequence at the top. The structure of the complex was modeled as detailed in Methods. Briefly, as mentioned above, NMR analysis demonstrated that all seven recognition sites in MBP (labeled a through g) are bound to SecB in the SecB–MBP complex. We have determined the high-resolution structure of MBPd and MBPe in complex with SecB (Extended Data Fig. 6). Because of their length and the short linker tethering the two sites, d and e sites most likely bind to the same side of SecB. MBP site f is the longest one, consisting of ~90 residues, and is thus entirely accommodated on the other side of SecB. With sites d, e and f occupying the primary binding sites, the other recognition sites (a, b, c and g) being much shorter can be accommodated within the secondary client-binding sites on SecB. The structure of MBP sites d and e in complex with SecB was determined using the experimental inter-molecular NOE data. The hydrophobic residues of the sites a, b, c, f, and g showing the strongest effect upon SecB binding, as determined by differential line broadening, were used to drive the docking of these sites to nonpolar residues on SecB. The modeled structure shows that the entire MBP sequence can be accommodated within one SecB molecule.Extended Data Figure 8Antiaggregation activity of SecBa, A triple amino acid substitution in the SecB (V40A/L42A/L44A) client-binding site was prepared and is referred to as the triple mutant SecB (SecB™). ITC profile of the binding of PhoA to SecB™ to be compared with PhoA binding to wild type SecB (Extended Data Fig. 3b). The triple substitution causes a 40-fold reduction in the affinity of SecB for PhoA. b, Fluorescence-monitored MBP folding in the absence of SecB (blue), in the presence of wild-type SecB (green) and in the presence of SecB™ (red). The triple mutant diminishes significantly the antifolding activity of SecB. c, 1H-15N TROSY HSQC spectra of MBP refolded in the absence (blue) and presence of SecB™ (red). In contrast to wild-type SecB (Extended Data Fig. 2c), SecB™ cannot hold MBP in the unfolded state. d, 1H-13C methyl HMQC spectra of MBPmut (blue) and in the presence of SecB (red) recorded at 22 °C. The MBP mutant (MBPmut) carries two amino acid substitutions (G32D/I33P) that renders the protein prone to aggregation41, especially at temperatures above 30 °C. No NMR signal of MBPmut can be detected at temperatures above 30 °C and the protein precipitates in the NMR tube. At 22 °C, MBPmut is folded, as evidenced by the resonance dispersion in the NMR spectra, and does not interact with SecB. e, 1H-13C methyl HMQC spectrum of MBPmut in the presence of SecB recorded at 50 °C. MBPmut suffers heavy precipitation and aggregation at temperatures higher than 30 °C, but in the presence of SecB it is stable and folded even at temperatures as high as 50 °C. f, 1H-15N TROSY HSQC spectra of SecB (blue) and in the presence of MBPmut (orange) at 42 °C, indicating binding. Because of the elevated temperature, a significant unfolded population of MBPmut is present, which binds to SecB (see main text). g, Mapping of the sites (orange) used by SecB to interact with MBPmut, based on the chemical shift perturbation data from the spectra in (f).Extended Data Figure 9Kinetics of PhoA and MBP interaction with SecB and TF(a–c) SPR analysis of the interaction of SecB with PhoA (a) and MBP at 20 °C (b) and 30 °C (c). Single-cycle and multiple-cycle procedures were used for the SPR analysis of SecB with PhoA and MBP, respectively. (d–f) BLI analysis of the binding of MBP to SecB (d), SecB™ (e) and TF (f). His-tagged PhoA or MBP (for SPR) or biotinylated MBP (for BLI) experiments was immobilized on an NTA chip (SPR) or streptavidin biosensor (BLI) and interactions were examined at different SecB or TF concentrations as indicated. Binding is reported in response units (RU) for SPR and wavelength shift (nm) for BLI as a function of time. (g–h) Effect of SecB on the kinetics of MBP folding. (g) Fluorescence-monitored folding of MBP (pre form) and mature MBP (h) in the absence (blue), and presence of 1- (green) and 4-fold (purple) excess of SecB. SecB does not appreciably delay folding of mature MBP. In fact, SecB excess appears to increase the yield of soluble, folded mature MBP (purple). i, 1H-15N TROSY HSQC spectra of mature MBP refolded in the absence (blue) and presence of SecB (red). SecB cannot retain the mature MBP unfolded. j, Fluorescence-monitored folding of the slowly-folding MBPY283D variant in the absence (blue), and presence of 1- (green) and 5-fold (orange) TF. As elaborated in the main text, TF does not delay folding of pre-MBP (Fig. 5a). However, it does delay folding of an inherently slowly folding MBP mutant (MBPY283D) thus highlighting the importance of the intrinsic folding of the client protein and its association rate to the chaperone.Extended Data Table 1NMR and refinement statistics for the SecB complexes with PhoA and MBPSecB-PhoASecB-PhoAaSecB-PhoAcSecB-PhoAdSecB-hoAeSecB-MBPdSecB-MBPeNMR distance and dihedral constraintsDistance restraints  Total NOE1362163621511338104313201446  Inter-residue    Sequential (|i-j| = 1)343402435376151371362    Non-sequential (|i-j| > 1 )1019123417169628929491084  Inter-molecule171335254272522Total dihedral angle restraints11691004996101210121006976    phi583502498506506503488    psi586502498506506503488Structure statisticsViolations (mean and s.d.)  Distance constraints (Å)0.012±0.0470.015±0.0520.015±0.0520.016±0.0520.018±0.0550.013±0.0450.016±0.054  Dihedral angle constraints (°)0.42±1.40.26±0.850.26±0.850.28±0.890.23±0.790.027±0.850.31±0.96  Max. dihedral angle violation (°)26.89.38.79.99.79.59.9  Max. distance constraintviolation (Å)1.110.860.890.800.830.950.95Average pairwise r.m.s.d. (Å)  Heavy4.42.12.92.42.53.62.9  Backbone4.01.52.11.71.72.82.1Statistics for each structure were computed for the ensembles of 20 deposited structures. Ordered residue ranges [S() + S() > 1.8] : 10–141 (of SecB subunits A, B, C, and D), backbone (heavy atom) r.m.s.d. was ~1.0 (1.3) Å within the specified range for all complexes. Additionally, the r.m.s.d. within the PhoA fragments is reported for each structure. Average distance constraints violations were calculated with PdbStat42.

Extended Data

NMR characterization of SecB and unfolded MBP

a, SecB is enriched in hydrophobic amino acids, such as methyl-bearing (Ala, Ile, Leu, Met, Thr, and Val) and aromatic (Phe and Tyr). b, 1H-15N TROSY HSQC (left) and 1H-13C methyl HMQC (right) spectra of [U-2H; Ala-13CH3; Met-13CH3; Ile-δ1-13CH3; Leu,Val-13CH3/13CH3; Thr-13CH3]-labeled SecB. SecB packing gives rise to two pairs of spectroscopically equivalent subunits: one pair is formed by subunits A and D, and the other pair by subunits B and C. Select assignment is included in the methyl spectrum with the asterisk indicating the other pair. c, 1H-15N HSQC spectra of select MBP fragments spanning the entire sequence of MBP. d, Secondary structure propensity (SSP) values39,40 of unfolded MBP (extracted collectively from the fragments) plotted as a function of the amino acid sequence. A SSP score at a given residue of 1 or −1 reflects a fully formed α-helical or β-structure (extended), respectively, whereas a score of, for example, 0.5 indicates that 50% of the conformers in the native-state ensemble of the protein are helical at that position. The data show that several of the secondary structure elements in the folded MBP retain some transient secondary structure in the unfolded MBP fragments.

NMR characterization of PhoA and MBP binding to SecB

a, To determine the SecB-recognition sites within PhoA and MBP 15N labeled PhoA and MBP fragments were titrated with unlabeled SecB. Due to the labeling scheme and the size of SecB, the intensity of the PhoA and MBP residues that are bound by SecB decreases dramatically or disappears. Several titration points were recorded but here only the spectra for the SecB:PhoA and SecB:MBP 1:1 are shown for two select fragments. The 1H-15N HSQC spectra of PhoA or MBP are shown in the absence (blue) and presence (red) of SecB. b–c, PhoA (b) and MBP (c) refolding in the presence and absence of SecB monitored by 1H-15N HSQC spectra. Spectra of the “refolded” state were recorded after rapid dilution of urea-treated MBP/PhoA in native buffer. Spectra of the “unfolded” state were recorded in urea. MBP and PhoA refolded in the their native structure in the absence of SecB but were retained in the unfolded state in the presence of SecB.

Energetics of SecB interaction with PhoA and MBP

a, MALS of SecB–PhoA complex showing a stoichiometry of 1:1. b, ITC of SecB binding to PhoA and the energetics of binding. c, Kd values for complexes between select PhoA fragments encompassing the five (a through e) SecB-recognition sites and SecB. d, MALS of SecB–MBP complex showing a stoichiometry of 1:1. e, ITC of SecB binding to MBP and the energetics of binding. f, Kd values for complexes between select MBP fragments encompassing the seven (a through g) SecB-recognition sites and SecB. More than one of the smaller PhoA or MBP fragments (e.g. PhoAc, PhoAd-e, MBPc-d) can be accommodated within SecB. Of note is the large favorable enthalpy of binding for the interaction of MBP and PhoA with SecB reflecting the large interacting surface. However, a large but unfavorable entropy diminishes the overall binding.

NMR characterization of the SecB–PhoA complex

a, 1H-15N TROSY HSQC spectra of PhoA in the unfolded state (light blue) and in complex with SecB (grey). The unfolded state was induced by the addition of reducing agent20 or urea and assigned and characterized by NMR as shown before20. Select resonance assignment of SecB-recognition sites in PhoA is included (the color is per the color code for each SecB-recognition site within PhoA; see Fig. 1b). There is an excellent correspondence between the PhoA residues identified to bind to SecB using the various PhoA fragments (Extended Data Fig. 2a) and the residues of full-length PhoA that are bound to SecB in the SecB–PhoA complex. All five SecB-recognition sites in PhoA (a through e) are engaged by SecB in the SecB–PhoA complex. The PhoA regions that are not bound to SecB (they retain their intensity in the complex) are all in an unfolded conformation as suggested by their essentially identical chemical shifts to the unfolded PhoA. b, Select strips from 13C-edited NOESY experiments highlighting inter-molecular NOEs in the SecB–PhoA complex. Owing to severe resonance overlap in the 120 kD SecB–PhoA complex, in order to identify specific inter-molecular NOEs we prepared samples wherein the two protein partners are labeled in different methyl-bearing type of amino acids. In this example, SecB was labeled in Leu, Met and Val residues and PhoA in Ile residues. Thus, all NOEs detected between Leu/Val/Met and Ile methyls are inter-molecular. c, 1H-13C methyl HMQC spectra of SecB in complex with PhoA fragments carrying the individual PhoA sites: PhoAa (green), PhoAc (orange), PhoAd (magenta) and PhoAe (red). Both SecB and PhoA fragments are [U-2H; Ala-13CH3; Met-13CH3; Ile-δ1-13CH3; Leu,Val-13CH3/13CH3; Thr-13CH3]-labeled. d, Representative strips from 13C-edited NOESY-HSQC and HMQC-NOESY-HMQC NMR experiments. The NOE cross-peaks between SecB and residues of PhoA fragments are designated by a dashed-line red circle. e, Characteristic NOEs showing that the primary binding groove in SecA is enlarged by the displacement of helix α2 as shown in Figure 4a. For example, the NOE between SecB residues Ala95 and Phe137 is consistent with the closed conformation observed in apo SecB. This NOE is not present in the SecB–PhoA complex because the two SecB residues have moved apart as a result of the displacement of the helix α2.

Strategy for the structure determination of the SecB–PhoA complex

The three main steps are briefly described here. More details can be found in Methods. The lowest-energy NMR structures of the SecB complexes with the individual PhoA sites a, c, d, and e are shown. The structural and NMR statistics for each structure are shown in Extended Data Table 1.

Structures of SecB with MBP sites

a, Lowest-energy structure of SecB in complex with a MBP fragment encompassing site d (MBPd, residues 105–152). b, Lowest-energy structure of SecB in complex with a MBP fragment encompassing site e (MBPe, residues 165–210). SecB is shown as grey solvent-accessible surface (left) or as white cartoon (right). Expanded views (right) of the contacts between SecB and MBP. The SecB residues mediating contacts with MBP are shown as blue ball-and-stick. In both complexes an additional MBP molecule binds symmetrically to the opposite face of SecB but are not shown for clarity.

NMR-driven model structure of SecB–MBP complex

a, 1H-15N TROSY HSQC spectra of MBP fragments (grey), MBP fragments in complex with SecB (blue) and full-length MBP in complex with SecB (magenta). The Gly (left) and Trp Nε (right) regions are shown as examples because of the excellent dispersion and lack of severe resonance overlap. The various MBP fragments covering the entire MBP sequence (Extended Data Fig. 1c) are colored grey and if they are located within a SecB-recognition site it is denoted in the superscript. The MBP residues that do not interact with SecB retain their intensity. These are residues located in regions that are not SecB-recognition sites (Fig. 1c). When these spectra are compared with the spectra of full-length MBP in complex with SecB (in magenta) a very good resonance correspondence is observed. Thus, two important observations can be made: first, all seven SecB-recognition sites (a through g) in MBP are engaged by SecB in the SecB–MBP complex; and, second, the MBP regions that do not interact with SecB in the SecB–MBP complex remain in an unfolded state. The Trp spectra (right) provide direct evidence in support of these observations: all Trp residues, with the exception of Trp155, are located in SecB-recognitions sites and they all interact with SecB in the SecB–MBP complex. In contrast, Trp155 does not bind to SecB when the corresponding MBP fragment was used, and this also the case for MBP. b, Modeled structure of the SecB–MBP complex. SecB is shown as a solvent-exposed surface and MBP as a pink ribbon. The seven MBP sites recognized by SecB are shown as sidechain surface and colored per the color code in the graphic of the MBP sequence at the top. The structure of the complex was modeled as detailed in Methods. Briefly, as mentioned above, NMR analysis demonstrated that all seven recognition sites in MBP (labeled a through g) are bound to SecB in the SecB–MBP complex. We have determined the high-resolution structure of MBPd and MBPe in complex with SecB (Extended Data Fig. 6). Because of their length and the short linker tethering the two sites, d and e sites most likely bind to the same side of SecB. MBP site f is the longest one, consisting of ~90 residues, and is thus entirely accommodated on the other side of SecB. With sites d, e and f occupying the primary binding sites, the other recognition sites (a, b, c and g) being much shorter can be accommodated within the secondary client-binding sites on SecB. The structure of MBP sites d and e in complex with SecB was determined using the experimental inter-molecular NOE data. The hydrophobic residues of the sites a, b, c, f, and g showing the strongest effect upon SecB binding, as determined by differential line broadening, were used to drive the docking of these sites to nonpolar residues on SecB. The modeled structure shows that the entire MBP sequence can be accommodated within one SecB molecule.

Antiaggregation activity of SecB

a, A triple amino acid substitution in the SecB (V40A/L42A/L44A) client-binding site was prepared and is referred to as the triple mutant SecB (SecB™). ITC profile of the binding of PhoA to SecB™ to be compared with PhoA binding to wild type SecB (Extended Data Fig. 3b). The triple substitution causes a 40-fold reduction in the affinity of SecB for PhoA. b, Fluorescence-monitored MBP folding in the absence of SecB (blue), in the presence of wild-type SecB (green) and in the presence of SecB™ (red). The triple mutant diminishes significantly the antifolding activity of SecB. c, 1H-15N TROSY HSQC spectra of MBP refolded in the absence (blue) and presence of SecB™ (red). In contrast to wild-type SecB (Extended Data Fig. 2c), SecB™ cannot hold MBP in the unfolded state. d, 1H-13C methyl HMQC spectra of MBPmut (blue) and in the presence of SecB (red) recorded at 22 °C. The MBP mutant (MBPmut) carries two amino acid substitutions (G32D/I33P) that renders the protein prone to aggregation41, especially at temperatures above 30 °C. No NMR signal of MBPmut can be detected at temperatures above 30 °C and the protein precipitates in the NMR tube. At 22 °C, MBPmut is folded, as evidenced by the resonance dispersion in the NMR spectra, and does not interact with SecB. e, 1H-13C methyl HMQC spectrum of MBPmut in the presence of SecB recorded at 50 °C. MBPmut suffers heavy precipitation and aggregation at temperatures higher than 30 °C, but in the presence of SecB it is stable and folded even at temperatures as high as 50 °C. f, 1H-15N TROSY HSQC spectra of SecB (blue) and in the presence of MBPmut (orange) at 42 °C, indicating binding. Because of the elevated temperature, a significant unfolded population of MBPmut is present, which binds to SecB (see main text). g, Mapping of the sites (orange) used by SecB to interact with MBPmut, based on the chemical shift perturbation data from the spectra in (f).

Kinetics of PhoA and MBP interaction with SecB and TF

(a–c) SPR analysis of the interaction of SecB with PhoA (a) and MBP at 20 °C (b) and 30 °C (c). Single-cycle and multiple-cycle procedures were used for the SPR analysis of SecB with PhoA and MBP, respectively. (d–f) BLI analysis of the binding of MBP to SecB (d), SecB™ (e) and TF (f). His-tagged PhoA or MBP (for SPR) or biotinylated MBP (for BLI) experiments was immobilized on an NTA chip (SPR) or streptavidin biosensor (BLI) and interactions were examined at different SecB or TF concentrations as indicated. Binding is reported in response units (RU) for SPR and wavelength shift (nm) for BLI as a function of time. (g–h) Effect of SecB on the kinetics of MBP folding. (g) Fluorescence-monitored folding of MBP (pre form) and mature MBP (h) in the absence (blue), and presence of 1- (green) and 4-fold (purple) excess of SecB. SecB does not appreciably delay folding of mature MBP. In fact, SecB excess appears to increase the yield of soluble, folded mature MBP (purple). i, 1H-15N TROSY HSQC spectra of mature MBP refolded in the absence (blue) and presence of SecB (red). SecB cannot retain the mature MBP unfolded. j, Fluorescence-monitored folding of the slowly-folding MBPY283D variant in the absence (blue), and presence of 1- (green) and 5-fold (orange) TF. As elaborated in the main text, TF does not delay folding of pre-MBP (Fig. 5a). However, it does delay folding of an inherently slowly folding MBP mutant (MBPY283D) thus highlighting the importance of the intrinsic folding of the client protein and its association rate to the chaperone.

NMR and refinement statistics for the SecB complexes with PhoA and MBP

Statistics for each structure were computed for the ensembles of 20 deposited structures. Ordered residue ranges [S() + S() > 1.8] : 10–141 (of SecB subunits A, B, C, and D), backbone (heavy atom) r.m.s.d. was ~1.0 (1.3) Å within the specified range for all complexes. Additionally, the r.m.s.d. within the PhoA fragments is reported for each structure. Average distance constraints violations were calculated with PdbStat42.
