---
pmid: '33511665'
title: Poly(U)-specific endoribonuclease ENDOU promotes translation of human CHOP
  mRNA by releasing uORF element-mediated inhibition.
authors:
- Lee HC
- Fu CY
- Lin CY
- Hu JR
- Huang TY
- Lo KY
- Tsai HY
- Sheu JC
- Tsai HJ
journal: EMBO J
year: '2021'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC8167367
doi: 10.15252/embj.2019104123
---

# Poly(U)-specific endoribonuclease ENDOU promotes translation of human CHOP mRNA by releasing uORF element-mediated inhibition.
**Authors:** Lee HC, Fu CY, Lin CY, Hu JR, Huang TY, Lo KY, Tsai HY, Sheu JC, Tsai HJ
**Journal:** EMBO J (2021)
**DOI:** [10.15252/embj.2019104123](https://doi.org/10.15252/embj.2019104123)
**PMC:** [PMC8167367](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8167367/)

## Abstract

1. EMBO J. 2021 Jun 1;40(11):e104123. doi: 10.15252/embj.2019104123. Epub 2021
Jan  28.

Poly(U)-specific endoribonuclease ENDOU promotes translation of human CHOP mRNA 
by releasing uORF element-mediated inhibition.

Lee HC(1), Fu CY(1), Lin CY(1), Hu JR(2), Huang TY(2), Lo KY(3), Tsai HY(4), 
Sheu JC(5), Tsai HJ(1)(2)(6).

Author information:
(1)Institute of Biomedical Sciences, Mackay Medical College, New Taipei City, 
Taiwan.
(2)Institute of Molecular and Cellular Biology, National Taiwan University, 
Taipei, Taiwan.
(3)Department of Agricultural Chemistry, National Taiwan University, Taipei, 
Taiwan.
(4)Institute of Molecular Medicine, School of Medicine, National Taiwan 
University, Taipei, Taiwan.
(5)Liver Disease Prevention and Treatment Research Foundation, Taipei, Taiwan.
(6)Department of Life Science, Fu Jen Catholic University, New Taipei, Taiwan.

Upstream open reading frames (uORFs) are known to negatively affect translation 
of the downstream ORF. The regulatory proteins involved in relieving this 
inhibition are however poorly characterized. In response to cellular stress, 
eIF2α phosphorylation leads to an inhibition of global protein synthesis, while 
translation of specific factors such as CHOP is induced. We analyzed a 105-nt 
inhibitory uORF in the transcript of human CHOP (huORFchop ) and found that 
overexpression of the zebrafish or human ENDOU poly(U)-endoribonuclease (Endouc 
or ENDOU-1, respectively) increases CHOP mRNA translation also in the absence of 
stress. We also found that Endouc/ENDOU-1 binds and cleaves the huORFchop 
transcript at position 80G-81U, which induces CHOP translation independently of 
phosphorylated eIF2α. However, both ENDOU and phospho-eIF2α are nonetheless 
required for maximal translation of CHOP mRNA. Increased levels of ENDOU shift a 
huORFchop reporter as well as endogenous CHOP transcripts from the monosome to 
polysome fraction, indicating an increase in translation. Furthermore, we found 
that the uncapped truncated huORFchop -69-105-nt transcript contains an internal 
ribosome entry site (IRES), facilitating translation of the cleaved transcript. 
Therefore, we propose a model where ENDOU-mediated transcript cleavage 
positively regulates CHOP translation resulting in increased CHOP protein levels 
upon stress. Specifically, CHOP transcript cleavage changes the configuration of 
huORFchop thereby releasing its inhibition and allowing the stalled ribosomes to 
resume translation of the downstream ORF.

© 2021 The Authors. Published under the terms of the CC BY NC ND 4.0 license.

DOI: 10.15252/embj.2019104123
PMCID: PMC8167367
PMID: 33511665 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflict 
of interest.

## Full Text

IntroductionAlmost half of human, mouse, and zebrafish mRNAs contain at least one upstream open reading frame (uORF) located at the 5′ untranslated region (Iacono et al, 2005; Matsui et al, 2007; Calvo et al, 2009; Barbosa et al, 2013; Chew et al, 2013). While a few uORFs activate the downstream coding sequence (DCS) such as Cauliflower mosaic virus 35S RNA (Pooggin et al, 2001), most uORFs function as inhibitory elements to block DCS hereinafter termed as uORF‐guided translational inhibition (uORF‐TI) by different mechanisms (Kozak, 1987; Yaman et al, 2003; Vattem & Wek, 2004; Sachs & Geballe, 2006; Col et al, 2007; Zhou et al, 2008; Roberts et al, 2009; Chen et al, 2010; Palam et al, 2011). Such mechanisms include the nonsense‐mediated decay pathway (He et al, 2003; Mendell et al, 2004; Sachs & Geballe, 2006; Ramani et al, 2009), the amino acid sequence of uORF‐encoded peptide (von Arnim et al, 2014; Ebina et al, 2015), methylation of uORF2ATF4 (Zhou et al, 2018), stalled ribosome elongation at uORFs (Palam et al, 2011; Barbosa & Romão, 2014; Young et al, 2016; Young & Wek, 2016), and the stability of uORF secondary structure (Chew et al, 2016). Variable biological and structural properties of uORFs in each gene may affect the regulatory potential of uORFs (Barbosa et al, 2013). Consequently, only a limited number of uORFs have been functionally characterized such as the delayed translation reinitiation model for GCN4 and ATF4 (Young & Wek, 2016).The mRNA encoding CCAAT/enhancer‐binding protein homologous protein (CHOP) (Park et al, 1992; Barone et al, 1994) undergoes highly regulated translation through a single uORF element located at the 5′UTR of CHOP’s mRNA (Jousse et al, 2001). Previously, Chen et al (2010) proved that different stressors could induce a correspondingly different regulatory networks to control human uORFchop (huORFchop)‐driven CHOP mRNA translation. Palam et al (2011) reported that phosphorylation of translation initiation factor eIF2α at S51 facilitates ribosomal bypass of an inhibitory huORFchop to enhance CHOP mRNA translation. Young et al (2016) found that an RNA sequence encoding Ile‐Phe‐Ile (IFI) in huORFchop is a key sequence for CHOP translation. However, while the inhibition of translation, as mediated by huORFchop, has already been defined, it is still too early to conclusively declare that protein(s) other than canonical translation initiation factors might be directly involved in the translation of human CHOP mRNA during ER stress by suppressing its inhibitory uORF element.Therefore, we aimed to address this question by using a zebrafish transgenic line, huORFZ, which harbors an inhibitory uORF of huORFchop fused with GFP reporter serving as DCS and driven by cytomegalovirus promoter (Lee et al, 2011). The GFP signal is exclusively expressed in some specific subtype cells of the central nervous system within huORFZ embryos because they are highly responsive to heat‐shock and hypoxic stresses (Lee et al, 2011; Zeng et al, 2016). The GFP reporter could be translated in these specific subtype cells because the inhibitory huORFchop element was suppressed during stress. Therefore, we used laser microdissection to harvest GFP‐expressing (+) and non‐GFP‐expressing (–) same subtype cells in the brain to identify de novo protein(s) that might play a repressive role in huORFchop‐TI and, hence, facilitate the translation of DCS. Microarrays and expression pattern analysis were used to select zebrafish Endouc, a human orthologue ENDOU (Lee et al, 2017), because it was significantly upregulated in the GFP(+) of brain during the applied stresses.The eukaryotic ENDOU ribonuclease family consists of several enzymes that share amino acid homology with XendoU (Snijder et al, 2003; Renzi et al, 2006). This ENDOU family is broadly conserved from viruses to humans. For example, Nsp15, a viral orthologue of XendoU, plays an essential role in coronavirus replication (Bhardwaj et al, 2012) and host immune suppression (Ricagno et al, 2006). Xenopus XendoU is a uridylate‐specific, divalent cation‐dependent enzyme that produces molecules with 2′,3′‐cyclic phosphate ends—this is a unique characteristic of this particular class of RNases involved in generating U16 and U86 small nucleolar RNAs through the cleavage of pre‐mRNAs encoded within introns (Laneve et al, 2003; Gioia et al, 2005; Renzi et al, 2006). Human PP11 has been recently categorized as a member of the XendoU family. Despite its annotated function as a putative serine protease, PP11 is a placental‐specific endoribonuclease. Moreover, the dysregulated expression of PP11 in tumor tissues may be associated with carcinogenesis (Inaba et al, 1982). Human ENDOU, or PP11, is a mammalian orthologue of zebrafish Endouc, and human ENDOU mRNA contains spliced variants encoding three different polypeptides, ENDOU‐1 (NM_001172439.2), ENDOU‐2 (NM_006025.4), and ENDOU‐3 (NM_001172440.2) isoform (Lee et al, 2017).
Xenopus XendoU and human ENDOU play roles in cellular processes including the regulation of ER structure, RNA degradation, and cell survival (Poe et al, 2014; Schwarz & Blower., 2014). Moreover, C. elegans ENDU‐2 was reported to be involved in regulating cold intolerance and synaptic remodeling through caspase signaling suggesting that ENDOU may play a role in stress response (Ujisawa et al, 2018). However, it is unknown whether eukaryotic ENDOU family protein might also be involved in suppressing uORF‐TI to facilitate the translation of CHOP mRNA.In this study, we found that overexpression of zebrafish Endouc or human orthologue ENDOU isoform 1 (Endouc/ENDOU‐1) could induce the translation of exogenous huORFchop‐tag reporter and endogenous CHOP transcripts in HEK293T cells in vitro and zebrafish embryos in vivo without stress treatment, suggesting that Endouc/ENDOU‐1 could suppress huORFchop‐TI. We also demonstrated that such blockade driven by Endouc/ENDOU‐1 was dependent on its endoribonuclease activity. Endouc/ENDOU‐1 could directly bind the huORFchop transcript and cleave it at uridylate‐specific residue at 80G‐81U. Moreover, increased Endouc resulted in a shift of huORFchop‐tagged and CHOP transcripts from monosomes toward polysomes. Furthermore, we demonstrated that the uncapped truncated huORFchop‐69‐105‐nt transcript possessed an internal ribosome entry site (IRES) activity. The increase of this IRES activity was dependent on the increase of Encouc/ENDOU‐1. Therefore, we propose that Encouc/ENDOU‐1 might facilitate the otherwise stalled ribosomes to suppress the inhibitory configuration caused by a huORFchop cassette to result in the translation of the DCS of CHOP transcript during stress through its binding and ribonuclease activity on the huORFchop transcript.

Introduction

Almost half of human, mouse, and zebrafish mRNAs contain at least one upstream open reading frame (uORF) located at the 5′ untranslated region (Iacono et al, 2005; Matsui et al, 2007; Calvo et al, 2009; Barbosa et al, 2013; Chew et al, 2013). While a few uORFs activate the downstream coding sequence (DCS) such as Cauliflower mosaic virus 35S RNA (Pooggin et al, 2001), most uORFs function as inhibitory elements to block DCS hereinafter termed as uORF‐guided translational inhibition (uORF‐TI) by different mechanisms (Kozak, 1987; Yaman et al, 2003; Vattem & Wek, 2004; Sachs & Geballe, 2006; Col et al, 2007; Zhou et al, 2008; Roberts et al, 2009; Chen et al, 2010; Palam et al, 2011). Such mechanisms include the nonsense‐mediated decay pathway (He et al, 2003; Mendell et al, 2004; Sachs & Geballe, 2006; Ramani et al, 2009), the amino acid sequence of uORF‐encoded peptide (von Arnim et al, 2014; Ebina et al, 2015), methylation of uORF2ATF4 (Zhou et al, 2018), stalled ribosome elongation at uORFs (Palam et al, 2011; Barbosa & Romão, 2014; Young et al, 2016; Young & Wek, 2016), and the stability of uORF secondary structure (Chew et al, 2016). Variable biological and structural properties of uORFs in each gene may affect the regulatory potential of uORFs (Barbosa et al, 2013). Consequently, only a limited number of uORFs have been functionally characterized such as the delayed translation reinitiation model for GCN4 and ATF4 (Young & Wek, 2016).

The mRNA encoding CCAAT/enhancer‐binding protein homologous protein (CHOP) (Park et al, 1992; Barone et al, 1994) undergoes highly regulated translation through a single uORF element located at the 5′UTR of CHOP’s mRNA (Jousse et al, 2001). Previously, Chen et al (2010) proved that different stressors could induce a correspondingly different regulatory networks to control human uORFchop (huORFchop)‐driven CHOP mRNA translation. Palam et al (2011) reported that phosphorylation of translation initiation factor eIF2α at S51 facilitates ribosomal bypass of an inhibitory huORFchop to enhance CHOP mRNA translation. Young et al (2016) found that an RNA sequence encoding Ile‐Phe‐Ile (IFI) in huORFchop is a key sequence for CHOP translation. However, while the inhibition of translation, as mediated by huORFchop, has already been defined, it is still too early to conclusively declare that protein(s) other than canonical translation initiation factors might be directly involved in the translation of human CHOP mRNA during ER stress by suppressing its inhibitory uORF element.

Therefore, we aimed to address this question by using a zebrafish transgenic line, huORFZ, which harbors an inhibitory uORF of huORFchop fused with GFP reporter serving as DCS and driven by cytomegalovirus promoter (Lee et al, 2011). The GFP signal is exclusively expressed in some specific subtype cells of the central nervous system within huORFZ embryos because they are highly responsive to heat‐shock and hypoxic stresses (Lee et al, 2011; Zeng et al, 2016). The GFP reporter could be translated in these specific subtype cells because the inhibitory huORFchop element was suppressed during stress. Therefore, we used laser microdissection to harvest GFP‐expressing (+) and non‐GFP‐expressing (–) same subtype cells in the brain to identify de novo protein(s) that might play a repressive role in huORFchop‐TI and, hence, facilitate the translation of DCS. Microarrays and expression pattern analysis were used to select zebrafish Endouc, a human orthologue ENDOU (Lee et al, 2017), because it was significantly upregulated in the GFP(+) of brain during the applied stresses.

The eukaryotic ENDOU ribonuclease family consists of several enzymes that share amino acid homology with XendoU (Snijder et al, 2003; Renzi et al, 2006). This ENDOU family is broadly conserved from viruses to humans. For example, Nsp15, a viral orthologue of XendoU, plays an essential role in coronavirus replication (Bhardwaj et al, 2012) and host immune suppression (Ricagno et al, 2006). Xenopus XendoU is a uridylate‐specific, divalent cation‐dependent enzyme that produces molecules with 2′,3′‐cyclic phosphate ends—this is a unique characteristic of this particular class of RNases involved in generating U16 and U86 small nucleolar RNAs through the cleavage of pre‐mRNAs encoded within introns (Laneve et al, 2003; Gioia et al, 2005; Renzi et al, 2006). Human PP11 has been recently categorized as a member of the XendoU family. Despite its annotated function as a putative serine protease, PP11 is a placental‐specific endoribonuclease. Moreover, the dysregulated expression of PP11 in tumor tissues may be associated with carcinogenesis (Inaba et al, 1982). Human ENDOU, or PP11, is a mammalian orthologue of zebrafish Endouc, and human ENDOU mRNA contains spliced variants encoding three different polypeptides, ENDOU‐1 (NM_001172439.2), ENDOU‐2 (NM_006025.4), and ENDOU‐3 (NM_001172440.2) isoform (Lee et al, 2017).

Xenopus XendoU and human ENDOU play roles in cellular processes including the regulation of ER structure, RNA degradation, and cell survival (Poe et al, 2014; Schwarz & Blower., 2014). Moreover, C. elegans ENDU‐2 was reported to be involved in regulating cold intolerance and synaptic remodeling through caspase signaling suggesting that ENDOU may play a role in stress response (Ujisawa et al, 2018). However, it is unknown whether eukaryotic ENDOU family protein might also be involved in suppressing uORF‐TI to facilitate the translation of CHOP mRNA.

In this study, we found that overexpression of zebrafish Endouc or human orthologue ENDOU isoform 1 (Endouc/ENDOU‐1) could induce the translation of exogenous huORFchop‐tag reporter and endogenous CHOP transcripts in HEK293T cells in vitro and zebrafish embryos in vivo without stress treatment, suggesting that Endouc/ENDOU‐1 could suppress huORFchop‐TI. We also demonstrated that such blockade driven by Endouc/ENDOU‐1 was dependent on its endoribonuclease activity. Endouc/ENDOU‐1 could directly bind the huORFchop transcript and cleave it at uridylate‐specific residue at 80G‐81U. Moreover, increased Endouc resulted in a shift of huORFchop‐tagged and CHOP transcripts from monosomes toward polysomes. Furthermore, we demonstrated that the uncapped truncated huORFchop‐69‐105‐nt transcript possessed an internal ribosome entry site (IRES) activity. The increase of this IRES activity was dependent on the increase of Encouc/ENDOU‐1. Therefore, we propose that Encouc/ENDOU‐1 might facilitate the otherwise stalled ribosomes to suppress the inhibitory configuration caused by a huORFchop cassette to result in the translation of the DCS of CHOP transcript during stress through its binding and ribonuclease activity on the huORFchop transcript.

ResultsMicroarrays to screen for genes responsible for the translation of DCS during ER stressTo better understand the role of huORFchop‐TI in vivo, we first employed a zebrafish transgenic line huORFZ (Lee et al, 2011) which harbors huORFchop‐tagged reporter cDNA because GFP was exclusively apparent in stress‐treated embryos. The number of GFP‐expressing cells in the brain of huORFZ embryos at 72 h post‐fertilization (hpf) was positively correlated with the duration of heat‐shock treatment (Fig EV1A). Of the three different sections performed to isolate GFP(+) cells in the brain of stress‐treated huORFZ embryos (Fig EV1B), more GFP(+) cells appeared on one slide from section 2 (Fig EV1C). Upon examining cells with immunostaining with the glial and neuronal markers to identify GFP(+) cells around tectal and telencephalic ventricles, we found that heat‐shock‐induced GFP(+) cells were a subtype of glial cells but not neurons (Fig EV1D).Figure EV1Subtype‐specific neuronal cells in the brain of zebrafish embryos can respond to heat‐shock stress, resulting in GFP expression
Views of transgenic huORFZ embryos treated with heat‐shock stress for different exposure time under fluorescence microscopy. GFP was expressed in some subtype‐specific cells. The percentage shown on the right lower corner indicated the number of embryos those treated (n = 50) that presented similar GFP expression pattern.Schematic representations of different sections of the brain of zebrafish embryo. Lines of sections 1–3 indicated three sectional orientations.GFP‐expressing cells were observed under fluorescence microscopy through three dissections. Tectal ventricle (TeVe) and telencephalic ventricle (TVe) were observed in sections 1 and 2 of upper panels, respectively. Lower panels presented the higher magnification of area indicated by the box in the upper panels.Immunohistochemistry staining of cell types using antibodies conjugated with red fluorescence signal against glutamine synthase (GS) and HuCD. Nuclei were marked with DAPI in blue, while GFP‐expressing cells were presented in green. CeP: cerebellar plate; DT: dorsal thalamus; H: hypothalamus; Ha: habenula; OB: olfactory bulb; MO: medulla oblongata; P: pallium; PTd: dorsal part of posterior tuberculum; PTv: ventral part of posterior tuberculum; TeO: tectum opticum; T: midbrain tegmentum; and VT: ventral thalamus.
Most importantly, we next studied the genes involved in huORFchop‐TI, and a microarray was performed to compare the expression levels of genes between GFP(+)‐ and GFP(−)‐cells of the same subtype in the brain (Fig 1A). Based on Agilent GeneSpring software analysis, a total of 643 under‐represented transcripts and 1,014 over‐represented transcripts were identified in GFP(+) cells from heat‐shock stress when compared to GFP(−) cells. A total of 1,346 under‐represented transcripts and 2,210 over‐represented transcripts were identified in GFP(+) cells from hypoxic stress versus GFP(−) cells. Moreover, a total of 298 over‐represented transcripts and 104 under‐represented transcripts were identified for both stresses (Fig 1B and Appendix Table S1).Figure 1Method of searching for candidate genes identified as potentially involved in translational control mediated by huORFchop transcript through their upregulation in GFP‐expressing cells
Schematic diagram illustrating the separation of GFP‐expressing and non‐GFP‐expressing cells from stress‐treated zebrafish embryos of transgenic line huORFZ using LMD methods followed by microarrays.Graphical representation of two microarray expression analyses. A total of 298 over‐represented transcripts and 104 under‐represented transcripts were obtained from both microarrays.Upper panels: top view of EGFP expressed in zebrafish embryos derived from zebrafish transgenic line huORFZ under non‐stress and heat‐shock treatment. Middle panels: lateral view of endouc expression patterns in embryos under non‐stress condition and treated with heat‐shock. Lower panels: top view of endouc expression patterns.Quantitative RT–PCR determined the relative expression level of endouc mRNA in non‐stress‐ and heat‐shock‐treated embryos. Values representing the non‐stress‐ and heat‐shocked conditions were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).The endouc and ela3l (negative control) mRNAs were separately microinjected into zebrafish embryos from huORFZ at one‐cell stage. Upper panels: observation of GFP signal expressed in huORFZ embryos at 72 hpf. Lower panels: magnification of pictures corresponding to the upper panels.
Screening used the three steps: First, we focused on the highly upregulated genes from the microarray data. Typically, we would use an antisense morpholino‐knockdown strategy to determine highly downregulated putative genes. However, it was easier to confirm whether putative genes were involved in huORFchop‐TI simply through microinjection of mRNAs in non‐stressed zebrafish embryos. In total, 298 highly upregulated genes were picked for cloning. Of these, 183 candidate genes were successfully amplified by PCR and cloned into the pGEMTeasy vector for whole‐mount in situ hybridization (WISH) riboprobe synthesis; 107 out of the 183 candidates were successfully cloned into the pCS2 vector for in vitro mRNA synthesis. Each of these prepared mRNAs was then individually microinjected into zebrafish huORFZ embryos. Strikingly, most injected embryos were either lethal or devoid of GFP expression before 24 hpf with the notable exception of those introduced with mRNAs of endonuclease polyU‐specific C (endouc), eukaryotic translation initiation factor 3 subunit Hb (eif3hb), transcription elongation factor A (SII) (teca2), and peroxiredoxin 6 (prbx6). When these four putative genes were overexpressed in untreated huORFZ embryos, we observed that GFP reporter was detectable in all embryos (Appendix Fig S1A). Among these four candidates, only embryos injected with endouc mRNA could induce the GFP reporter specifically at the CNS tissues; the injection of ela3l mRNA (false positive control) did not (Fig 1E).Second, we performed WISH to examine whether these putative genes were upregulated in embryos after stress treatment. This step resulted in the discovery of two genes involved in translation including eif3hb and tcea2 as well as two genes that encoded enzymes including endouc and heme oxygenase 1a (hmox1); these had a significant upregulation in heat‐shock‐treated embryos compared to non‐stress control embryos (Appendix Fig S1B). We also found that some candidate genes were unchanged or slightly increased in the heat‐shock‐treated embryos compared to non‐stress control embryos such as chymotrypsinogen B1 (ctrb1), elastase 3 like (ela3l), serine protease 35 (prss35), death‐associated protein 1b (dap1b), peroxiredoxin 6 (prdx6), and glutamate‐cysteine ligase (gclc) (Appendix Fig S1C). We noticed that endouc transcript was significantly upregulated in brain compared to the control untreated embryos (Fig 1C). This upregulated expression pattern of endouc in the brain corresponded to the results of the GFP reporter, which was also significantly expressed in the brain of huORFZ embryos treated with heat‐shock (Fig 1C). The level of endouc transcript was markedly increased around the tectal ventricle and upper hindbrain after embryos were treated with heat‐shock stress (Fig 1C).Therefore, based on our stepwise elimination approach to determine the genes involved in huORFchop‐TI, zebrafish endouc (accession number si:dkey‐103i16.2)—which is conserved to the human orthologue ENDOU gene (Lee et al, 2017)—became the primary hit for further study for two reasons. First, the endouc transcript displayed a 5.39‐ and 2.85‐fold increase of expression in the heat‐shocked‐treated and hypoxia‐treated GFP(+) cells, respectively (Appendix Table S1). Second, real‐time qPCR result of endouc transcript was consistent with those obtained from the microarray data and WISH (Fig 1C and D). Therefore, we propose that Endouc might be a novel non‐ribosomal protein able to suppress huORFchop‐TI control thus facilitating the translation of DCS during ER stress.Transfection of Endouc/ENDOU‐1 alone is sufficient to drive the translation of DCS reporter even in the absence of stressWe quantified the increase of downstream luciferase (luc) reporter induced by overexpression of Endouc through in vivo study. Wild‐type (WT) zebrafish embryos were microinjected with a plasmid carrying huORFchop‐luc, a dual‐luc‐control vector (phRG‐TK), and an Endouc‐Flag‐overexpression vector. Under normal conditions, the zebrafish embryos injected with all three plasmids noted above exhibited 5.3‐fold higher luc activity than that of the control group, which was injected with plasmids puORFchop‐luc, pCS2 vector, and phRG‐TK (Fig 2A). The elevated luc activity that occurred in the Endouc‐injected embryos under normal conditions was consistent with that of the Endouc‐injected embryos under stress conditions such as the application of heat‐shock (Fig 2A).Figure 2Overexpression of zebrafish Endouc blocks huORFchop‐TI and translates downstream reporter gene in the absence of stress
AHistograms show the luc activity obtained from zebrafish embryos microinjected simultaneously with puORFchop‐luc, phRG‐TK, and each indicated plasmid followed by analysis of luc activity at 96 hpf. Embryos microinjected with pCS2 vector during normal condition (non‐stress; blank column) served as a control group, while the microinjected embryos treated with 40°C at 72 hpf for 1 h were the heat‐shocked group (solid column). The relative luc activity of microinjected embryos shown on the right was normalized by the amount of RNA in the injected embryos shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from the control group (this was normalized to 1). The luc activity mediated by the huORFchop transcript was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR. Values representing the non‐stress and stress condition were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).BHistograms present the luc activity obtained from HEK293T cells co‐transfected with puORFchop‐luc, phRG‐TK, and each indicated plasmid and then treated with either DMSO (control group) or thapsigargin (TH; stress group), followed by analysis of luc activity. Cells transfected with pCS2‐vector and kept at normal condition served as a control group. The relative luc activity of transfected cells shown on the right was normalized by the amount of mRNA in the transfected cells shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group (this was normalized to 1). The luc activity mediated by the huORFchop transcript was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR. Values representing the non‐stress and stress conditions were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).C–EWestern blot analysis of proteins, as indicated, in (C) zebrafish embryos, (D) HEK293T cells and (E) HeLa cells. GAPDH and α‐tubulin served as internal controls. Protein levels relative to each internal control are presented below each lane.

Source data are available online for this figure.
We next performed an in vitro study, in which the above plasmids were transfected into HEK293T cells. Under normal conditions, luc activity was increased ~2.6‐fold when the vector containing Endouc was transfected (Fig 2B). This was consistent with the results of Endouc‐overexpressed cells under stress conditions such as treatment with thapsigargin (TH)—an ER stress inducer (Fig 2B). The consistency of both in vitro and in vivo results led us to conclude that the overexpression of Endouc causes a dramatic increase in the DCS reporter expression suggesting that Endouc facilitates the translation of huORFchop–tag transcript.To investigate whether overexpression of Endouc can increase CHOP translation, we overexpressed Endouc at the one‐cell stage of zebrafish embryos treated with heat‐shock at 72 hpf. We then analyzed CHOP level at 96 hpf. The results demonstrated that overexpression of Endouc in zebrafish embryos increased the protein level of CHOP both in non‐stress and heat‐shock conditions (Fig 2C). Similar results were observed in the in vitro study where overexpression of Endouc in HEK293T and HeLa cells increased CHOP expression (Fig 2D and E).The translation of CHOP is tightly linked to the degree of p‐eIF2α during stress (Palam et al, 2011), and we asked if overexpression of Endouc would correlate with increased levels of p‐eIF2α. Western blot results showed that overexpression of Endouc in zebrafish embryos, as well as HEK293T and HeLa cells, did increase the level of p‐eIF2α (Fig 2C–E) suggesting that Endouc overexpression induces an increase of p‐eIF2α. These data demonstrated that Endouc functions as a positive regulator of CHOP translation during stress response.Like zebrafish Endouc, the human orthologue ENDOU‐1 can suppress the inhibitory huORFchop elementTransfection of any one of three plasmids containing the three isoforms of human ENDOU could greatly induce the translation of reporter gene resulting in increased luc activity up to ~4.8‐fold, which is higher than that achieved by the transfection of zebrafish endouc expression plasmid (Fig EV2A and B). Similar to TH‐treated cells, transfection of overexpressed human ENDOU‐1 in cells could effectively suppress the repressive effect mediated by huORFchop transcript, in turn, enhancing the translation of the reporter gene (Fig EV2A).Figure EV2Overexpression of three isoforms of human ENDOU gene enables embryos to translate downstream GFP reporter, even in the absence of stress
Histograms presented the luc activity obtained from an in vitro experiment in which HEK293T cells were co‐transfected simultaneously with puORFchop–luc, phRG‐TK, and each one of the plasmids, as indicated, for 24 h and treated with either DMSO (control group) or thapsigargin (TH; stress group) for 2 h, followed by analysis of luc activity. The luc activity value of cells transfected with pCS2 vector and kept at normal condition served as a control group normalized as 1. The luc activity mediated by huORFchop element was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc mRNA was measured by RT–qPCR. The luc activity of transfected cells shown on the right was normalized by the mRNA level shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group (this was normalized to 1). The values featuring the non‐stressed and stressed conditions were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (*P < 0.01; **P < 0.005, ***P < 0.001).Protein levels of zebrafish Endouc and human ENDOU isoforms used in the luc assay shown in panel (A) were determined. The α‐tubulin served as an internal control.Western blot analysis of the time course of expression levels of human ENDOU‐1, p‐eIF2α, total eIF2α, and CHOP proteins in HEK293T cells after TH treatment. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane.

Source data are available online for this figure. 

To further confirm the suppressive effect of ENDOU‐1 on huORFchop‐TI, we studied the time course expression of endogenous ENDOU‐1 in HEK293T cells treated with stress. As shown in Fig EV2C, a rapid increase in the expression of ENDOU‐1 was observed 1 h after stress treatment, reaching a high level after 2 h; this level was maintained for up to 8 h. CHOP, however, exhibited a slow response and reduced expression after 8 h. The p‐eIF2α protein showed early expression, but it then slowed after 4 h. Therefore, we suggest that human ENDOU‐1 is also a stress‐responsive gene with a suppressive effect on huORFchop‐TI.Endoribonuclease activity of Endouc/ENDOU‐1 is required to suppress the inhibitory huORFchop structureWe next sought to mechanistically explain the involvement of zebrafish Endouc in huORFchop‐TI. To accomplish this, a series of Endouc deletion constructs (Fig 3A) were co‐transfected with puORFchop–luc. A luc activity assay was then performed to analyze the effects of these constructs on regulating huORFchop‐TI both in vivo (zebrafish embryos) and in vitro (HEK293T cells).Figure 3Identification of Endouc functional domain involved in regulating huORFchop‐TI
Schematic representation of zebrafish Endouc protein and its internal deletion mutants.‡
‡
Histograms presenting the relative luc activity obtained from zebrafish embryos microinjected simultaneously with puORFchop‐luc, phRG‐TK, and each of the expression plasmids as indicated followed by analysis of luc activity at 96 hpf. Embryos microinjected with pCS2 vector during normal condition served as a control group (non‐stress; blank column), while the microinjected embryos treated with 40°C at 72 hpf for 1 h were the heat‐shocked group (solid column). The relative luc activity of microinjected embryos shown on the right was normalized by the amount of RNA in the injected embryos shown on the left. The relative luc activity is represented by the fold increase in Fluc/Rluc ratio over that obtained from control group. The luc activity mediated by the huORFchop element was measured by a dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.Histograms presented the luc activity obtained from HEK293T cells co‐transfected with puORFchop‐luc, phRG‐TK, and each one of the plasmids as indicated and treated with either DMSO (control group) or thapsigargin (TH; stress group) followed by analysis of luc activity. The luc activity of cells transfected with pCS2 vector and kept at normal condition served as a control group. The relative luc activity of transfected cells shown on the right was normalized by the amount of RNA in the transfected cells shown on the left. The relative luc activity is represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group normalized as 1. The luc activity mediated by the huORFchop transcript was measured by the dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.Zebrafish embryos at one‐cell stage were microinjected with plasmids as indicated, followed by extraction of embryonic proteins at the 96‐hpf stage. Protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants were detected using Western blot analysis. GAPDH served as an internal control.HEK293T cells were transfected with indicated plasmids followed by analysis of the protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants using Western blot. The α‐tubulin served as an internal control. Protein levels relative to each internal control are presented in each lane.

Source data are available online for this figure.
Compared with control, the luc activity was greatly increased in recipients with intact Endouc, Endouc△1–25, Endouc△26–136, and Endouc△299–310; it was slightly reduced in recipients with Endouc△137–298 (Fig 3B and C; Appendix Fig S2A and B). We noticed that transfection of Endouc△137–298 failed to suppress huORFchop‐mediated mRNA translation in recipients under both normal and stress conditions (Fig 3B). Western blot analysis also demonstrated that the protein levels of p‐eIF2α and CHOP were greatly reduced in cells expressing Endouc△137–298 both in vivo and in vitro (Fig 3D and E) suggesting that the key domain of Endouc for regulating huORFchop‐TI is located at residues 137–298. E243, H244, E249, H259, and K302 are necessary for human ENDOU‐2 endoribonuclease activity (Laneve et al, 2008), while H162 and K224 are necessary for Xenopus XendoU RNase activity (Schwarz & Blower, 2014). Interestingly, the key residues required to suppress huORFchop‐TI of Endouc are all within the 137–298 residues. On the basis of this evidence, we mutated zebrafish Endouc at H181 (EndoucH181A) and K242 (EndoucK242A) to alanine to generate two endoribonuclease‐deficient Endouc proteins (Fig 4A). Cells and embryos transfected with intact Endouc showed evidence of a huORFchop‐TI blockage. However, no luc activity was induced when embryos were transfected with both EndoucH181A and EndoucK242A (Fig 4B and C; Appendix Fig S2C and D). Furthermore, Western blot analysis demonstrated that overexpression of intact Endouc in recipients increased the protein levels of p‐eIF2α and CHOP at normal condition, while overexpression of both EndoucH181A and EndoucK242A, for which endoribonuclease activity was lost in recipients, did not (Fig 4D and E). These results were consistent with both TH‐treated cells and heat‐shock‐treated embryos. Therefore, we conclude that the endoribonuclease activity of zebrafish Endouc is critical to blocking huORFchop‐TI.Figure 4The positive effect of Endouc on p‐eIF2α and CHOP expressions is dependent on its endoribonuclease activity
Schematic representation of zebrafish Endouc protein and its internal deletion mutants.‡
Histograms presented the relative luc activity obtained from zebrafish embryos microinjected simultaneously with puORFchop‐luc, phRG‐TK, and each one of the expression plasmids as indicated followed by analysis of luc activity at 96 hpf. Embryos microinjected with pCS2 vector during normal condition (non‐stress; blank column) served as a control group, while the microinjected embryos treated with 40°C at 72 hpf for 1 h were the heat‐shocked group (solid column). The relative luc activity of microinjected embryos shown on the right was normalized by the amount of RNA in the injected embryos shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from control group. The luc activity mediated by the huORFchop transcript was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.Histograms presented the luc activity obtained from HEK293T cells co‐transfected with puORFchop‐luc, phRG‐TK, and each one of the plasmids as indicated; these were treated with either DMSO (control group) or thapsigargin (TH; stress group) followed by analysis of luc activity. The luc activity of cells transfected with pCS2 vector and kept at normal condition served as a control group. The relative luc activity of transfected cells shown on the right was normalized by the amount of RNA in the transfected cells shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group normalized as 1. The luc activity mediated by the huORFchop transcript was measured by the dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.Zebrafish embryos at one‐cell stage were microinjected with plasmids as indicated followed by extraction of embryonic proteins at the 96‐hpf stage. Protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants were detected using Western blot analysis. GAPDH served as an internal control.HEK293T cells were transfected with indicated plasmids followed by analysis of the protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants using Western blot. Here, α‐tubulin served as an internal control. Protein levels relative to each internal control were presented in each lane.

Source data are available online for this figure.
Next, we demonstrated that overexpression of ENDOU‐1 increased the expression levels of p‐eIF2α and CHOP in HEK293T and U373MG cells (Fig EV3A and B). However, when we mutated residue H285 of ENDOU‐1 (responsible for endoribonuclease activity), we found that overexpression of the ENDOU‐1H285A mutant affected neither p‐eIF2α nor CHOP levels (Fig EV3A and B) suggesting that the increase of p‐eIF2α and CHOP levels induced by ENDOU‐1 is dependent on the endoribonuclease activity of ENDOU‐1. We further confirmed that both Endouc/ENDOU‐1 were functionally conserved increasing p‐eIF2α and CHOP expression and playing a suppressive role in huORFchop‐TI through their endoribonuclease activity. The GFP(+)‐expressing cells were zebrafish glial‐like cells, while the U373MG cells were human glioblastoma–astrocytoma cells. Therefore, these results also strengthened our hypothesis that Endouc/ENDOU‐1 plays important roles in CHOP translation in glial‐like cells of zebrafish embryos.Figure EV3Increase of p‐eIF2α and CHOP expression levels induced by ENDOU‐1 is dependent on the endoribonuclease activity of Endouc
A, B(A) HEK293T and (B) U373MG cells were transfected with plasmid, as indicated, for 24 h, treated with either DMSO (control group) or thapsigargin (TH; stress group) for 2 h, and then submitted to detection of protein levels of CHOP, p‐eIF2α, total eIF2α, and ENDOU‐1 by Western blot analysis. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane as indicated.

Source data are available online for this figure. 

Increased CHOP protein level by overexpression of Endouc is specific but independent of ER stressTo further prove the specific roles of Endouc/ENDOU‐1 in facilitating CHOP mRNA translation, we first determined if CHOP induced by Endouc/ENDOU‐1 resulted from ER stress caused by Endouc/ENDOU‐1 overexpression. We examined the mRNA level of spliced xbp1 and protein level of ER stress markers in Endouc/ENDOU‐1‐expressing cells. Compared to TH‐treated cells, Endouc/ENDOU‐1‐overexpressed cells only displayed increased GADD34, p‐eIF2α, and CHOP, while spliced xbp1 mRNA and other stress factors were not increased (Fig EV4A, B and D). GADD34 was induced in Endouc‐ and ENDOU‐1‐overexpressing cells (Fig EV4D), which could have resulted from the increase of p‐eIF2α induced by overexpression of Endouc/ENDOU‐1 as supported by Novoa et al (2003) who reported that eIF2α phosphorylation promotes GADD34 expression. Most stress markers were unaffected suggesting the specific induction of p‐eIF2α and CHOP expression by Endouc/ENDOU‐1 again confirming that Endouc/ENDOU‐1 was functionally conserved as described above. Specifically, Endouc/ENDOU‐1 could increase eIF2α phosphorylation and suppress huORFchop‐mediated repression of mRNA translation to increase CHOP expression through their endoribonuclease activities.Figure EV4Increase of p‐eIF2α and CHOP induced by Endouc/ENDOU‐1 overexpression is specific, but independent of ER stress
A–CRT–PCR analysis: (A) Unspliced and spliced forms of xbp1 RNA were indicated. Cells treated with thapsigargin (TH) were used as positive control. GAPDH served as an internal control. The RT–qPCR was used to quantify the amount of (B) spliced xbp1 RNA and (C) CHOP mRNA. The amount of spliced xbp‐1 and CHOP mRNAs in cells transfected with pCS2 vector and kept at normal condition served as a control group, normalized as 1. The values were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).DWestern blot analysis of proteins, as indicated, expressed in HEK293T cells. Cells treated with TH were used as positive control. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane.

Source data are available online for this figure. 

The RT‐quantitative PCR (RT–qPCR) data indicate that the CHOP mRNA level did not increase in cells overexpressing Endouc/ENDOU‐1. Rather, CHOP mRNA was slightly reduced in these cells (Fig EV4C). In contrast, the amount of CHOP protein was increased in cells overexpressing Endouc/ENDOU‐1. Collectively, these data proved that Endouc/ENDOU‐1 positively regulates CHOP expression at the translational level.Endouc/ENDOU‐1 induces CHOP translationTo investigate the effect of Endouc/ENDOU‐1 expression and eIF2α phosphorylation on CHOP translation, we first employed HEK293T cells treated with ISRIB—an integrated stress response inhibitor that can reverse the effect of eIF2α phosphorylation thereby inhibiting eIF2 function. HEK293T cells were transfected with an expression vector containing either vector only or endouc cDNA for 24 h; this was then treated with either DMSO or 200 nM ISRIB followed by an analysis of the level of CHOP protein. Similar to the CHOP protein weakly present in the control, CHOP was only slightly detected in ISRIB‐treated cells in the absence of Endouc (lanes 1 and 2 of Fig 5A). However, CHOP was significantly detected in the Endouc‐expressing cells in the absence of ISRIB (lane 3 of Fig 5B). Interestingly, we found that the protein level of CHOP expressed in cells treated with Endouc plus ISRIB was lower than that in cells treated with Endouc plus p‐eIF2α (lanes 3 versus 4 in Fig 5B) suggesting that both Endouc/ENDOU‐1 expression and eIF2α phosphorylation impact CHOP translation.Figure 5Besides p‐eIF2α, Endouc/ENDOU‐1 is another key protein that causes an increase in the CHOP protein level
AHEK293T cells were transfected with plasmids as indicated for 24 h and treated with DMSO or 200 nM ISRIB for 2 h followed by analysis of the protein level of CHOP, p‐eIF2α, total eIF2α, and Endou‐Flag using Western blot. The α‐tubulin served as an internal control. Protein levels relative to each internal control are presented in each lane.BHEK293T cells were transfected with plasmids as indicated for 24 h and treated with DMSO, thapsigargin (TH), ISRIB, or TH plus ISRIB for 2 h followed by analysis of the protein level of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants using Western blot. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane.CMEF wild‐type (WT) and its eIF2αS51A mutant (S51A) cells were treated with 1 μg of TH for 4 h followed by Western blot analysis of p‐eIF2α, total eIF2α, and ENDOU. GAPDH served as an internal control. Protein levels relative to each internal control are presented in each lane.D, EMEF WT and mutant S51A cells were transfected with plasmids as indicated for 24 h and treated with either DMSO or TH for 4 h followed by analysis of the protein levels of CHOP, p‐eIF2α, total eIF2α, GADD34, Endouc‐Flag and ENDOU‐1‐Flag using Western blot. GAPDH served as an internal control. Protein levels relative to each internal control were presented at each lane.FHEK293T cells were transfected with negative control siRNA and ENDOU‐siRNA for 48 h. After treatment, protein lysates were prepared and subjected to Western blot analysis for ENDOU‐1 and CHOP detection using antibodies. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented in each lane.

Source data are available online for this figure.
Next, we studied the control mechanism between Endouc and p‐eIF2α on suppressing huORFchop‐TI under stress. HEK293T cells were transfected with vector containing Endouc and EndoucK242A mutant, and then treated with TH and/or 200 nM ISRIB followed by an analysis of the protein level of CHOP. Compared to cells treated with TH, the CHOP protein was reduced in cells treated with TH plus ISRIB (lanes 2 versus 3 in Fig 5B) suggesting that ISRIB treatment could reduce the CHOP level induced by TH. This result was consistent with that obtained in Endouc‐expressing cells (lanes 4 versus 5 in Fig 5B). The level of CHOP expressed in cells treated with Endouc/TH/ISRIB was higher than that of cells treated with vector/TH/ISRIB (lanes 3 versus 5 in Fig 5B). CHOP was reduced in the EndoucK242A mutant (lanes 4 versus 6 in Fig 5B) compared to Endouc‐expressing cells. When Endouc‐expressing cells and EndoucK242A‐expressing cells were treated with TH and ISRIB, CHOP was significantly reduced in the EndoucK242A‐expressing cells in comparison to the Endouc‐expressing cells (lanes 5 versus 7 in Fig 5B). Unlike the expression profile of CHOP protein, we noticed that p‐eIF2α was always reduced in ISRIB‐treated cells. This suggests that Endouc alone is sufficient to induce CHOP translation independent of the effect of eIF2α phosphorylation.Our next step was to strengthen the hypothesis that Endouc alone is sufficient to induce CHOP translation and does so independent of eIF2α phosphorylation. To accomplish this, we employed mouse embryonic fibroblast (MEF) WT and mutant S51A cells that cannot phosphorylate eIF2α due to an S51A knock‐in mutation at the eIF2α phosphorylation site (Scheuner et al, 2001). First, when MEF WT cells were treated with TH, both p‐eIF2α and ENDOU‐1 were induced compared to DMSO‐treated control cells (Fig 5C) indicating that Endouc/ENDOU‐1 induced by stress is conserved among fish, mice, and humans. Additionally, the pronounced upregulation of Endouc/ENDOU‐1 failed to materialize in MEF S51A mutant cells treated with TH (Fig 5C) suggesting that p‐eIF2α also facilitates Endouc/ENDOU‐1 upregulation and that increased p‐eIF2α is required for the full expression of Endouc/ENDOU‐1 to facilitate CHOP translation during ER stress.Second, we further confirmed if Endouc/ENDOU‐1 could induce CHOP translation in the absence of p‐eIF2α. To address this question, MEF WT and S51A mutant cells were individually transfected with a plasmid harboring cDNA encoded with either Flag‐tagged Endouc/ENDOU‐1 or Endouc/ENDOU‐1K242A/H285A mutant followed by analysis of the protein level of CHOP and p‐eIF2α. In MEF WT cells, the results showed that both CHOP and p‐eIF2α were significantly induced in Endouc/ENDOU‐1‐overexpressing cells compared to untreated control cells, while neither CHOP nor p‐eIF2α was induced by Endouc/ENDOU‐1K242A/H285A mutant (Fig 5D and E). Nevertheless, CHOP was still increased in MEF S51A mutant cells treated with TH, although the level of CHOP was decreased in S51A mutant cells compared to WT cells (Fig 5D and E), suggesting that Endouc/ENDOU‐1 could induce CHOP translation in the absence of p‐eIF2α through its endoribonuclease activity because the endoribonuclease‐deficient mutant Endouc/ENDOU‐1K242A/H285A could not induce CHOP translation. We also noticed that GADD34 could be induced by Endouc/ENDOU‐1 in MEF WT cells but not in MEF S51A mutant cells (Fig 5D and E), suggesting that GADD34 is not directly induced by Endouc/ENDOU‐1. Rather, the GADD34 induced by Endouc/ENDOU‐1 is dependent on p‐eIF2α.CHOP was increased in the TH‐treated MEF S51A mutant cells, and thus, we conclude that (i) endoribonuclease activity of Endouc/ENDOU‐1 is necessary for Endouc/ENDOU‐1 to induce CHOP and p‐eIF2α expressions; (ii) CHOP translation induced by Endouc/ENDOU‐1 during stress is independent of eIF2α phosphorylation unlike the p‐eIF2α‐dependent induction of GADD34; and (iii) CHOP reached a maximal level in the presence of both Endouc/ENDOU‐1 and p‐eIF2α during ER stress.Finally, we performed siRNA to knock down ENDOU‐1 in HEK293T cells. Versus cells transfected with siCTRL‐siRNA after incubation with TH, the CHOP in HEK293T cells transfected with ENDOU‐1‐siRNA was reduced significantly in both the absence and presence of TH (Fig 5F). We conclude that overexpression of Endouc/ENDOU‐1 results in an increase in the p‐eIF2α and CHOP levels—this is specific and independent of ER stress. Nevertheless, both Endouc and p‐eIF2α are required for CHOP mRNA translation to reach a maximal level.Endouc/ENDOU‐1 induces eIF2α phosphorylation through the PKR pathwayThe coronaviral EndoU family protein Nsp15 has been associated with PKR activity in host cells (Kindler et al, 2017). Since Endouc and ENDOU‐1 are members of the EndoU family of proteins, we next studied a possible role of RNA‐activated protein kinase PKR in the induction of eIF2α phosphorylation and translational upregulation of CHOP as a result of ENDOU‐1 expression. We analyzed the p‐PKR level in Endouc/ENDOU‐1‐expressing cells and found that the p‐PKR level was induced significantly in HEK293T, HeLa, and MEF WT cells treated with TH compared to DMSO‐treated control cells (Fig 6A; Appendix Fig S3A and B). Moreover, the p‐PKR level was induced in Endouc/ENDOU‐1‐overexpressing cells without stress treatment. Meanwhile, the endoribonuclease‐deficient EndoucK242A/ENDOU‐1H285A‐overexpressing cells did not induce p‐PKR suggesting that Endouc/ENDOU‐1 induces PKR phosphorylation through its endoribonuclease activity.Figure 6Endouc/ENDOU‐1 induces the phosphorylation of eIF2α through the PKR pathway
MEF wild‐type (WT) cells were transfected with plasmids as indicated for 24 h and treated with either DMSO or thapsigargin (TH) for 4 h followed by analysis of the protein level of p‐PKR, total PKR, Endouc‐Flag and ENDOU‐1‐Flag. GAPDH served as an internal control. Protein levels relative to each internal control were presented at each lane.MEF WT cells were transfected with plasmids as indicated for 24 h and treated with either DMSO or TH for 4 h followed by analysis of the protein levels of p‐eIF2α, total eIF2α, CHOP, Endouc‐Flag and ENDOU‐1‐Flag. GAPDH served as an internal control. Protein levels relative to each internal control were presented at each lane. The results obtained from the two trials are shown on the left and right.

Source data are available online for this figure.
We further employed PKRΔ6, which is a PKR dominant‐negative mutant that lacks six amino acids in its catalytic domain and, thus, fails to phosphorylate eIF2α (Koromilas et al, 1992; Romano et al, 1995; Tu et al, 2012). The level of p‐eIF2α was reduced when Endouc/ENDOU‐1 and PKRΔ6 were co‐expressed in cells (Fig 6B; Appendix Fig S3C and D). Compared to the control cells, even though p‐eIF2α was absent in cells co‐transfected with Endouc/ENDOU‐1 and PKRΔ6, CHOP protein was still present (Fig 6B; Appendix Fig S3C and D), albeit in minimally detectable level suggesting that inhibition of PKR pathway in Endouc‐expressing cells only blocks the induction of p‐eIF2α. Moreover, the cells transfected with mutant EndoucK242A/ENDOU‐1H285A failed to induce PKR, eIF2α phosphorylation, and CHOP protein (Fig 6B; Appendix Fig S3C and D). This line of evidence suggests that Endouc/ENDOU‐1 induces eIF2α phosphorylation through the PKR pathway.Taken together, it can be concluded that (i) the increase of eIF2α phosphorylation induced by Endouc is dependent on PKR phosphorylation; (ii) Endouc/ENDOU‐1 induces the PKR‐eIF2α pathway through its endoribonuclease activity; and (iii) expression between Endouc/ENDOU‐1 and p‐eIF2α exhibits feedback loop control.Small RNA fragments cleaved from huORFchop transcript under stress were detected in in vivo and in vitro systemsNorthern blot analysis was performed to further study whether the huORFchop transcript could be cleaved in vivo and in vitro under stress. An increase in small RNA fragments was observed in heat‐shock‐treated huORFZ embryos (Fig 7A), suggesting that the huORFchop‐gfp transcript could be cleaved in vivo under stress. In HEK293T cells, compared to DMSO‐treated control cells, endogenous CHOP mRNAs were increased, but small mRNAs of fewer than 50 nucleotides (nts) were significantly increased in TH‐treated cells. Meanwhile, the internal control gapdh mRNA did not change in either DMSO‐treated or TH‐treated cells (Fig 7B) suggesting that huORFchop transcripts could be cleaved under stress.Figure 7Endouc can bind and cleave the huORFchop transcript
ANorthern blot analysis to detect the endogenous huORFchop‐tagged gfp (uORF‐gfp) mRNA and its cleaved form in zebrafish embryos of transgenic line huORFZ treated with non‐stress condition (NS) or heat‐shocked (HS) stress. Zebrafish 18S rRNA served as an internal control. The RNA fragments cleaved from huORFchop transcript were detected.BNorthern blot analysis to detect endogenous CHOP mRNA in HEK293T cells treated with either DMSO (control group) or thapsigargin (TH). Human GAPDH RNA served as internal control. The RNA fragments cleaved from CHOP mRNA were detected.CThe predicted RNA sequences of Endouc recognition site on huORFchop transcript from 72 to 91 nts are illustrated. The predicted cleavage sites at uridylates (U) are indicated by arrowheads.D, EThe cleavage activity of Endouc was studied using materials presented at each lane. EndoucK242A: Endouc mutant; Extract: incubated in cell extracts; and Buf: reaction buffer. RNA probe without adding reaction buffer served as a negative control. The predicted cleavage sites (U) were mutated to generate mutant huORFchop transcript (unlabeled huORF‐mt) and labeled by biotin (biotin‐huORF‐mt). Unlabeled competitor RNA was added in reactions at five times higher than that of the biotinylated probe.FElectrophoretic mobility shifted assay of RNA. Biotin‐labeled huORFchop transcript was reacted with the increased amount of Endouc. The positions of free RNA (free probe) and RNA–protein complex (shifted band) were indicated on the right. The luc 105‐nt RNA and single‐stranded huORFchop 105‐nt DNA served as negative controls.
Endouc/ENDOU‐1 cleaves huORFchop RNA at uridylates within a specific structureIt was reported that human ENDOU‐1 could cut at uridylates of oligonucleotide GGAACGUAUCCUUUGGGAG derived from small nucleolar RNAs containing multiple natural cleavage sites for XendoU (Laneve et al, 2003, 2008). Interestingly, this XendoU‐specific cutting sequence corresponds to sequences ranging from 72 to 91 nts of huORFchop transcript. When we used the Rochester Predict secondary structure server, we found that the sequences from 72 to 91 nts of huORFchop transcript per se could plausibly form a stem‐loop structure similar to that described by Laneve et al (2008) (Fig 7C), and that it could be repeatedly presented if the full‐length sequence of huORFchop transcript were used to predict its secondary structure (Appendix Fig S4).ENDOU has a ribonuclease domain that can cleave RNAs at UU dinucleotides or a single U in vitro (Caffarelli et al, 1997; Laneve et al, 2003, 2008; Bhardwaj et al, 2006). Therefore, we performed an RNase activity assay to examine whether zebrafish Endouc could cleave huORFchop transcript. Three fragments were generated in a dose‐dependent manner when biotin was labeled at the 3′‐end of huORFchop transcript (Fig 7D). However, cleaved fragments could not be detected when uridylates were mutated to guanines within 69–92 nt of huORFchop transcript (Fig 7D). Competition experiments also showed that the WT huORFchop transcript, but not mutant RNA, could reduce the cleavage signal (Fig 7E). Furthermore, mutant EndoucK242A failed to digest huORFchop transcript (Fig 7D). This line of evidence suggests that the cleavage of Endouc on huORFchop transcript is specific to uridylates and that the amino acid residue at K242 is required for RNase activity of Endouc.Endouc binds directly with huORFchop transcriptTo confirm binding between Endouc and huORFchop transcript, we performed RNA EMSA using biotin‐labeled, full‐length huORFchop transcript and Endouc. Compared to luc mRNA (Appendix Fig S5A), the results confirmed that Endouc could bind specifically with the huORFchop transcript in a dose‐dependent manner (Fig 7F). However, the band shift shown on the EMSA gel was abolished by the increase of unlabeled huORFchop transcript (Appendix Fig S5B). Moreover, Endouc bound neither luc mRNA nor huORFchop DNA (Fig 7F), suggesting that the binding between huORFchop transcript and Endouc is specific.RNA EMSA demonstrated that Endouc could still bind with the mutated huORFchop transcript (Appendix Fig S5C) although binding efficiency was relatively weak relative to the normal huORFchop transcript. However, Endouc failed to cleave mutated huORFchop transcript (Fig 7E) in which uridylates within 72–91 nts were mutated. Thus, it is clear that binding in the presence of uridylate is permissive for the specific cleavage of huORFchop transcript by Endouc.Endouc facilitates stalled ribosomes to suppress the inhibitory huORFchop element and plays a positive role in the translation of CHOP mRNAWe next performed polysome profiling to directly examine the role of Endouc in regulating huORFchop‐TI. HEK293T cells were transiently transfected with Endouc‐ or EndoucK242A‐expression vector and treated with cycloheximide to arrest elongating ribosomes on mRNAs. Their lysates were then subjected to sucrose gradient ultracentrifugation, and sample fraction were pooled into three groups shown in Figure 8A: free ribosomal subunits (Fractions 1 and 2), monosomes (Fractions 3–6), and light and heavy polysomes (Fractions 7–12). Compared to control cells, overexpression of Endouc caused an inhibition of global translational initiation as documented by the loss of heavy polysomes; overexpression of EndoucK242A did not change the polysome profile (Fig 8A). Western blot analysis demonstrated that both Endouc and EndoucK242A were mostly found in the monosome and light polysome fractions, whereas a small percentage of Endouc was found in the heavy polysome fractions (Fig 8A). Although no difference was observed in the distribution at each polysome fraction between protein levels of Endouc and EndoucK242A, the overexpression of Endouc could reduce global translation, while overexpression of EndoucK242A did not change the polysome profile. Moreover, LC‐MS/MS analysis proved that the major proteins interacting with recombinant Endouc‐Flag were ribosomal proteins and proteins involved in translation (Appendix Table S2). This evidence suggested that blockade of huORFchop‐TI via Endouc is dependent on its endoribonuclease activity.Figure 8Endouc associates with monosomes and plays a key role in regulating CHOP post‐translation
AHEK293T cells were transfected with pCS2 vector (black), pCS2‐Endouc (red), and pCS2‐EndoucK242A (green) for 24 h and subjected to either DMSO (control) or thapsigargin (TH; stress) treatment for 1 h. The lysates of transfected cells were applied to polysome profile analysis (PPA). The sucrose gradient from top to bottom was shown from left to right, respectively. The sedimentation of 40S, 60S, 80S, and polysomes was indicated. Lysates from control, Endouc‐expressing, and EndoucK242A‐expressing cells were applied to PPA. The resulting fractions were TCA‐precipitated, analyzed by SDS–PAGE, and subjected to Western blot using antibodies against Flag and S6. Upper and lower rows of Flag- and S6 blots were Endouc‐ and EndoucK242A‐ expressing cells, respectively.B, CThe lysates from control (pCS2), Endouc‐expressing (Endouc), and EndoucK242A‐expressing (EndoucK242A) cells were applied to PPA. The resulting fractions from 1‐10 collected from sucrose gradient were subsequently subjected to RT–qPCR assay to quantify (B) exogenous huORFchop‐tagged luc (luc mRNA) and (C) endogenous CHOP transcripts (CHOP mRNA). The distribution of relative abundance of luc and CHOP transcripts contained in each fraction was determined.D, EHEK293T cells were co‐transfected with pCS2‐Endouc and phuORFchop‐luc for 24 h and then subjected to DMSO or TH treatment for 2 h. The lysates of transfected cells were then applied to PPA. Total RNAs were isolated from heavy polysome fractions followed by performing 5′RACE to characterize the nucleotide sequences of truncated (D) exogenous huORFchop‐luc and (E) endogenous huORFchop‐CHOP transcripts. The first sequence at N‐terminus of RNA samples from the truncated forms derived from exogenous huORFchop‐luc (n = 14) and endogenous huORFchop‐CHOP transcripts (n = 11) was determined; the percentages of positive colonies containing various 5′UTR lengths (truncated forms) of luc and CHOP transcripts among total colonies examined were calculated.

Source data are available online for this figure.
We next measured the levels of endogenous CHOP and exogenous huORFchop‐tagged luc transcripts among the sucrose fractions. In the control cells, the huORFchop‐tagged luc (Fig 8B) and CHOP mRNAs (Fig 8C) were largely associated with monosomes (Fractions 4 and 5). However, in Endouc‐overexpressing cells, the huORFchop‐tagged luc and CHOP mRNAs displayed a shift of 43 and 37% of transcripts from monosome toward light polysomes (Fractions 6–8), respectively. In contrast, similar to the control group, overexpression of endoribonuclease‐deficient mutant EndoucK242A did not trigger these mRNAs into the light polysome fraction (Fractions 6–8). The addition of Endouc caused a higher percentage of luc reporter and CHOP mRNAs to shift from the monosome toward the light polysome fraction to proceed with continuous translation (Appendix Fig S6A and B).The RNA fragment cleaved by Endouc at the 80G‐81U of huORFchop transcript is found in polysome fractionsNext, to isolate and characterize CHOP mRNA after nuclease activity, we performed an experiment in which exogenous huORFchop‐luc mRNA and endogenous huORFchop‐CHOP mRNA were isolated from polysome fractions and characterized them after nuclease activity. After the sequences of these isolated mRNAs were confirmed using 5′RACE, we focused on finding truncated sequences derived from the huORFchop transcript in the polysome fractions isolated from both exogenous huORFchop‐luc mRNA and endogenous huORFchop‐CHOP mRNA groups. Specifically, the first nucleotide of major truncated mRNAs was the T at the nucleotide position of 81 (81T) (Fig 8D and E) suggesting the following: First, exogenous huORFchop‐luc (42%, n = 14) and endogenous huORFchop‐CHOP mRNAs (55%, n = 11) were cleaved by Endouc, a uridylate‐specific endoribonuclease, at the nucleotide position between 80G and 81U (80G‐81U). Second, the non‐5′‐capped truncated forms of exogenous huORFchop‐luc transcript and endogenous huORFchop‐CHOP mRNA were undergoing translation.Interestingly, the identified Endouc cleavage site at 80G‐81U was located at the hairpin structure of huORFchop cassette, which is one of the predicted sites recognized and cleaved by Endouc (Fig 7C). Moreover, 80G‐81U sequences were located next to the Ile‐Phe‐Ile sequence (82A‐90A nts), which is an essential sequence that allows the huORFchop motif to act as an inhibitory element to repress the translation of CHOP mRNA at normal state (Young et al, 2016). Therefore, we demonstrated cleavage of both exogenous huORFchop‐luc and endogenous huORFchop‐CHOP transcripts by Endouc at 80G‐81U.The uncapped truncated transcript derived from huORFchop RNA did not decay immediately in cellsTo determine whether overexpression of Endouc could change the turnover of CHOP mRNA, we treated Endouc‐overexpressing MEF cells with TH followed by actinomycin D treatment. The half‐life of CHOP transcript from the control (no stress treatment) and TH‐treated cells was 2.61 and 2.4 h, respectively (Fig EV5A and B). This is consistent with the previous study reported by Rutkowski et al (2006) and Young et al (2016) who demonstrated that the half‐life of CHOP transcript is 2–3 h, while overexpression of Endouc reduced the half‐life of CHOP transcript to 2.18 h suggesting that overexpression of Endouc did not significantly impact the turnover of CHOP transcript.Figure EV5The truncated form of CHOP mRNA after cleavage does not decay immediately in cells
A, BMEF cells were transfected with plasmid pCS2 vector or pCS2‐Endouc‐Flag for 24 h, cultured either in DMSO or thapsigargin (TH) for 3 h, followed by actinomycin D (10 μg/ml) treatment for 0, 1, 2, or 3 h as indicated. RNAs were then harvested, and the expression level of CHOP mRNA was quantified by RT–qPCR and normalized to 18S rRNA. Data present as bar graph (A) and line graph (B) were calculated from three independent experiments and presented as mean ± SD (n = 3) Student's t‐test was used to determine significant differences between each group (*P < 0.05, **P < 0.005, ***P < 0.001).CMEF cells were transfected with plasmid pCS2‐Endouc‐Flag for 24 h, followed by actinomycin D treatment for 0, 1, 2, or 3 h, as indicated. RNAs were then harvested, and the expression levels of full‐length and cleaved CHOP mRNA were quantified by RT–qPCR and normalized to 18S rRNA. Data were calculated from three independent experiments and presented as mean ± SD (n = 3).DMEF cells were transfected with plasmid pCS2‐Endouc‐Flag for 24 h. Some cells were used to isolate total RNAs, and the expression levels of full‐length and cleaved CHOP mRNA were measured by RT–qPCR and normalized by 18S rRNA. Data were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005). Other cells were used to detect the protein levels of CHOP and Endouc‐Flag proteins by Western blot analysis. The GAPDH served as an internal control.

Source data are available online for this figure. 

We next determined whether this uncapped truncated transcript would decay immediately or remain in cells. To accomplish this, MEF cells were transfected with pCS2‐Endouc‐Flag followed by actinomycin D treatment; an adaptor at the 5′ end was added for cDNA synthesis. We then used primers that could specifically target the adaptor‐linked cleavage of huORFchop cDNA for our qPCR experiment (see Materials and Methods). Compared to the half‐life between capped full‐length huORFchop and uncapped truncated huORFchop transcripts, we found that the half‐life of full‐length huORFchop transcript was 3 h, while the half‐life of uncapped truncated huORFchop transcript was reduced to about 43 min (Fig EV5C), suggesting that the uncapped truncated huORFchop transcript could remain in cells for up to 90 min. Since some translated mRNAs have a half‐life shorter than 1 h (Sharova, et al, 2009) or even shorter than 30 min (Raghavan et al, 2002), the half‐life of huORFchop 81‐105‐nt transcript is reasonably sufficient for CHOP translation. We also demonstrated that more CHOP transcript correlated with more CHOP protein further suggesting that the truncated transcripts cleaved from huORFchop may contribute to the translation of CHOP protein (Fig EV5D).Truncated huORFchop‐69‐105‐nt transcript contains internal translation activity, but truncated huORFchop‐81‐105‐nt transcript does notLuc and CHOP proteins could be produced from exogenous huORFchop‐luc and endogenous huORFchop‐CHOP mRNAs after Endouc binding and cleavage. Therefore, we hypothesized that the 5′‐region of the truncated huORFchop transcript may have internal translational activity that enables the translation of DCS. The predicted RNA sequences of Endouc recognition, binding, and cleavage sites on the 105‐nt huORFchop RNA were located from 72 to 91 nts (Fig 7C). Thus, we first engineered a truncated form that only contains 69–105 nts RNA fragment (huORFchop−69‐105‐nt), and then employed a pRF/phpRF bicistronic system (Terenin et al, 2017). In the pRF bicistronic reporter system, Renilla luc (Rluc) protein is translated in a cap‐dependent manner from the first cistron through ribosomal scanning, while Firefly luc (Fluc) protein is translated from the second cistron only if a fragment inserted at the upstream region possesses a cap‐independent translation mechanism. Since the Fluc/Rluc ratio represents a cap‐independent translation efficiency, four transcripts were designed to obtain Fluc/Rluc ratios. Appendix Fig S7A indicates the following: (i) RF: No insertion between Rluc‐ and Fluc‐cistrons which served as mock control; (ii) RF‐huORF69‐105 (RF‐huORFchop‐69‐105‐nt; an experimental group); (iii) RF‐cmyc (the 5′UTR of c‐myc which served as positive control); and (iv) RF‐Non105 (a 105‐nt RNA encoding luc that served as negative control). Based on the value of Fluc/Rluc ratio, we determined the cap‐independent translation efficiency of each inserted fragment. Following in vitro synthesis, these four bicistronic mRNAs were transfected individually into HEK293T cells, and their luc activities were determined by a dual‐luc assay. The Fluc/Rluc ratio obtained from transfection of RNA transcribed from vector construct only with no insert (RF; mock group) was set as 1. Compared to the RF group, the Fluc/Rluc ratio from transfection of RF‐Non105 transcript (negative control) did not increase (Appendix Fig S7A) suggesting that the 105‐nt luc RNA did not contain internal translation activity. In contrast, the Fluc/Rluc ratio from transfection of RF‐cmyc transcript (positive control) was dramatically increased ~100‐fold compared to the RF group suggesting that the 5′UTR of c‐myc contained strong internal translation activity consistent with the results reported by Stoneley et al (1998). Interestingly, the Fluc/Rluc ratio from transfection of the RF‐huORF69‐105 transcript was increased 3.2‐fold over that of control RF mRNA (Appendix Fig S7A) suggesting that the RF‐huORF69‐105 RNA fragment also has internal translation activity.The phpRF bicistronic reporter vector system excludes the possibility of either ribosomal reinitiation or read‐through from the stop codon of the Rluc cistron to the start codon of the Fluc cistron. Thus, a stable palindromic sequence is inserted before the start codon of the first ORF of Rluc generating a stable mRNA hairpin (−55 Kcal/mol) that inhibits cap‐dependent translation (Yang & Wang, 2019). Consequently, cap‐dependent translation of the Rluc cistron is diminished, while cap‐independent internal translation driven by downstream Fluc cistron is not affected. As depicted in Appendix Fig S7, four transcripts were designed as hpRF (mock control), hpRF‐huORF69‐105 (an experimental group), hpRF‐cmyc (positive control), and hpRF‐Non105 (negative control). Since the amount of input mRNA was not significantly different (Appendix Fig S7A and C) and Rluc activity was no different among the four groups, the change in the Fluc/Rluc ratio was completely due to the change in the Fluc activity (Appendix Fig S7B and D).As shown in Appendix Fig S7B, the Fluc/Rluc ratio from cells transfected with bicistronic mRNA containing c‐myc (hpRF‐cmyc transcript) was significantly increased: up to ~1,500‐fold greater than the Fluc/Rluc ratio of hpRF mock control and hpRF‐Non105 negative control groups. This increase indicates that the 5′UTR of c‐myc contained pronounced internal translation activity. Interestingly, the Fluc/Rluc ratio of cells transfected with the hpRF‐huORF69‐105 transcript was increased 80.5‐fold over that of the hpRF mock control (Appendix Fig S7C). These findings strongly suggest that the huORFchop‐69‐105‐nt transcript has internal translation activity albeit relatively weaker than c‐myc.To further determine whether the truncated form of huORFchop‐81‐105‐nt derived from Endouc‐cleaved huORFchop also contains internal translation activity, we engineered the 81‐105‐nt segment of huORFchop where the cleavage site of Endouc was at 80G‐81U. We then co‐transfected endouc mRNA with bicistronic RNA fragment containing huORFchop‐69‐105‐nt and huORFchop‐81‐105‐nt into HEK293T cells. As shown in Appendix Fig S8, the Fluc/Rluc ratio in cells transfected with endouc mRNA was slightly increased from 3.4‐ to 5.5‐fold in RF‐huORF69‐105 mRNA (Appendix Fig S8A and B). The ratio was dramatically elevated from 43‐ to 226‐fold in hpRF‐huORF69‐105 mRNA compared to that of control hpRF mRNA (Appendix Fig S8C and D). Surprisingly, the internal translation activity driven by huORFchop‐81‐105‐nt was completely lost suggesting that the truncated Endouc‐cut huORFchop transcript from the T81 cleavage site to the end of the huORFchop transcript (i.e., huORFchop‐81‐105‐nt) does not contain internal translation activity.The internal translation activity possessed by huORFchop‐69‐105‐nt transcript is increased in the presence of EndoucTo investigate whether Endouc increases internal translation‐dependent translation of the huORFchop transcript, we co‐transfected endouc mRNA combined with bicistronic RNA containing huORFchop‐69‐105‐nt transcript into HEK293T cells whereas co‐transfection of gfp mRNA and huORFchop‐69‐105‐nt transcript served as a negative control. In the pRF bicistronic reporter system, overexpression of Endouc could increase the translation of DCS of huORFchop‐69‐105‐nt inserted in the intercistronic region (RF‐huORF69‐105 transcript) because the Fluc/Rluc ratio was raised from 3.9 to 5.3 (Fig 9A). Moreover, overexpression of Endouc could dramatically increase the translation of the DCS reporter within huORFchop‐69‐105‐nt transcript (hpRF‐huORF69‐105) in the phpRF bicistronic reporter system in which a hairpin structure inhibits the cap‐dependent translation (Rluc). This is because Fluc/Rluc ratio was elevated from 41‐ to 219‐fold compared to that of hpRF control mRNA (Fig 9C). Again, the Rluc activity showed no difference among groups (Fig 9B and D) indicating that the change of Fluc/Rluc ratio was completely dependent on the change of Fluc activity. In contrast, if the 80G‐81U site in the huORFchop‐69‐105‐nt transcript was mutated (RF‐huORF69‐105‐T81C/hpRF‐huORF69‐105‐T81C), then the internal translation activity was completely lost (Fig 9A and C). We conclude that (i) overexpressed Endouc increases the internal translation activity driven by huORFchop‐69‐105‐nt transcript and (ii) the 5′‐region of uncapped truncated huORFchop‐69‐105‐nt may also contribute to continuous translation as mediated by internal translation initiation activity.Figure 9The 5′ region of the cleaved huORFchop transcript contains internal translation activity
Schematic representation of RF bicistronic transcript is shown above the bar graphs. Rluc: Renilla
luciferase (blank box); Fluc: Firefly
luciferase (solid box). The luc activity of HEK293T cells transfected with bicistronic mRNA, as indicated, in the presence (+) or absence (−) of endouc‐Flag mRNA was determined. The relative luc activity of transfected cells shown on the right was normalized by the amount of the transfected bicistronic transcript shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from control group, normalized as 1. The luc activity was measured by dual‐luciferase assay, and its corresponding bicistronic mRNAs were measured by RT–qPCR. The values were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).The luc activity of Fluc (solid column) versus Rluc (blank column) in HEK293T cells transfected with bicistronic mRNAs shown in A was presented as an increase in fold over that obtained from control group of RF which was normalized as 1. These data suggested that the change of Fluc/Rluc ratio shown in A was entirely dependent on the change of Fluc activity. The data were averaged from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005).Schematic representation of hpRF bicistronic transcript. Rluc: Renilla
luciferase (blank box); Fluc: Firefly
luciferase (solid box). The luc activity of HEK293T cells transfected with bicistronic mRNA, as indicated, in the presence (+) or absence (−) of endouc‐Flag mRNA was determined. The relative luc activity of transfected cells shown on the right was normalized by the amount of the transfected bicistronic transcript shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from control group, normalized as 1. The luc activity was measured by dual‐luciferase assay, and its corresponding bicistronic mRNAs were measured by RT–qPCR. The values were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).The luc activity of Fluc (solid column) versus Rluc (blank column) in HEK293T cells transfected with bicistronic mRNAs shown in C was presented as an increase in fold over that obtained from control group of hpRF which was normalized as 1. These data suggested that the change of Fluc/Rluc ratio shown in C was entirely dependent on the change of Fluc activity. The data were averaged from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).
The huORFchop‐69‐105‐nt transcript possesses an IRES activityWe used the pRF/phpRF bicistronic system described above to demonstrate that the huORFchop‐69‐105‐nt transcript, but not the huORFchop‐81‐105‐nt transcript, contains internal translation activity. Next, we determined whether this huORFchop‐69‐105‐nt transcript possesses an IRES activity. To make this determination, we performed an in vitro system to transcribe polyA‐tailed Fluc mRNA fused after huORFchop‐69‐105‐nt transcript which was capped with a normal m7Gppp (a 5′‐capped form) or a non‐functional ApppG (a 5′ A‐capped form) at the 5′ end. However, the m7GpppG‐capped mRNAs are translated in both a cap‐dependent and a cap‐independent manner, while A‐capped mRNAs are only translated in a cap‐independent manner. Therefore, to make the distinction between cap‐dependence and cap‐independence, we compared the translation efficiency between m7Gppp‐ and A‐capped mRNAs. To do this, we next used A‐capped cmyc IRES transcripts fused at the upstream of a Fluc reporter to serve as a positive control. The A‐capped 105 nt of luc cDNA (Non 105) served as a negative control. After a two‐hr transfection of these transcripts in HEK293T cells, we found that the translation efficiency of A‐capped cmyc‐Fluc mRNA only reached approximately 15% of that driven by m7GpppG‐capped cmyc‐Fluc mRNA (Fig 10A). As expected, the reporter activity driven by A‐capped Fluc mRNA (Fig 10B) and A‐capped Non105 (Fig 10C) was very low (~1.5%). The A‐capped full‐length huORFchop‐1‐105‐nt reporter transcript (A‐capped huORF1‐105Fluc) exhibited only 1.5% of that driven by m7GpppG‐capped huORF1‐105Fluc (Fig 10D) suggesting that full‐length huORFchop‐1‐105‐nt exhibited no IRES activity. We also found that the huORFchop‐81‐105‐nt transcript (A‐capped huORF81‐105Fluc) showed no IRES activity (Fig 10F). Interestingly, the A‐capped huORF69‐105Fluc RNA fragment exhibited 4.3% reporter activity compared to m7GpppG‐capped huORF69‐105Fluc RNA (Fig 10E) suggesting that the uncapped huORFchop‐69‐105‐nt transcript does have IRES activity.Figure 10Comparison between cap‐dependent and IRES‐dependent translation of the huORFchop transcript containing reporter
A–FSchematic representation of capped (m7G‐capped) and A‐capped (non‐functional capped) monocistronic mRNAs with poly(A)‐tail‐encoded firefly luciferase (Fluc) and fused upstream with (A) c‐myc IRES, (B) no insertion, (C) non‐specific 105‐nt luc RNA (Non105), (D) huORFchop‐1‐105‐nt, (E) huORFchop‐69‐105‐nt, and (F) huORFchop‐81‐105‐nt as shown above each bar graph. The RNA transfection assay of each transcript equipped with m7G‐capped or A‐capped monocistronic mRNA in HEK293T cells was compared. To normalize the luc activities of capped and A‐capped Fluc transcripts, the m7G‐capped Renilla luc (Rluc) mRNA was used as an internal control. The luc activity of cells transfected with cap‐monocistronic mRNA in each graph was set as 100%.GThe luc activity driven by the indicated A‐capped monocistronic mRNA in the presence of Endouc or EndoucK242A in HEK293T cells. The m7G‐capped Rluc mRNA was used as an internal control to normalize the activities of A‐cap Fluc transcripts. The egfp mRNA was used as a negative control, and the luc activity of cells transfected with egfp mRNA and A‐cap Fluc mRNA containing full‐length huORFchop‐1‐105‐nt was set to 1. The data were averaged from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).
To further analyze whether the presence of Endouc impacts huORFchop‐69‐105‐nt IRES activity, we studied the luc activity of A‐capped huORF69‐105Fluc transcript in the presence of Endouc or EndoucK242A in HEK293T cells. The results showed that the luc activity of A‐capped monocistronic mRNAs containing full‐length huORFchop and huORFchop‐69‐105‐nt transcripts was induced in the presence of Endouc, but not in the presence of endoribonuclease‐deficient mutant EndoucK242A (Fig 10G) suggesting that huORFchop IRES activity induced by Endouc is mediated through its endoribonuclease activity. Interestingly, the A‐capped mRNA containing huORFchop‐81‐105‐nt failed to induce luc activity (Fig 10G) suggesting that the truncated uncapped huORFchop‐81‐105‐nt transcript generated from the cleavage of Endouc/‐ENDOU‐1 on the full‐length of huORFchop‐1‐105‐nt transcript at 80G‐81U does not possess IRES activity unlike the huORFchop‐69‐105‐nt transcript.To further analyze whether the presence of Endouc impacts the huORFchop‐69‐105‐nt IRES activity, we studied the luc activity of ApppG‐capped huORF69‐105 Luc transcript in the presence of Endouc or EndoucK242A in HEK293T cells. The results showed that the luc activity of ApppG‐capped monocistronic mRNAs containing full‐length huORFchop and huORFchop‐69‐105‐nt transcripts was induced in the presence of Endouc but not in the presence of endoribonuclease‐deficient mutant EndoucK242A (Fig 10G) suggesting that the huORFchop IRES activity induced by Endouc is mediated through its endoribonuclease activity. Interestingly, the ApppG‐capped mRNA containing huORFchop‐81‐105‐nt transcript failed to induce luc activity (Fig 10G). Additionally, in the absence of Endouc, only huORF69‐105, but not full‐length huORFchop, displayed weak IRES activity (Fig 10G). Strikingly, however, when Endouc was overexpressed in cells, both huORFchop‐69‐105‐nt and full‐length huORFchop transcripts displayed IRES activity (Fig 10G).Following the logic of this line of collective evidence, we suggest that an inhibitory element could be present in huORFchop‐1‐68‐nt causing the repression of IRES activity. In clear contrast, when Endouc is overexpressed in cells, the inhibitory structure of huORFchop transcript is disrupted via endoribonuclease, likely altering the inhibitory element presented in huORFchop‐1‐68‐nt but promoting the IRES activity driven by huORFchop‐69‐105‐nt to then facilitate the translation of CHOP DCS.

Microarrays to screen for genes responsible for the translation of DCS during ER stressTo better understand the role of huORFchop‐TI in vivo, we first employed a zebrafish transgenic line huORFZ (Lee et al, 2011) which harbors huORFchop‐tagged reporter cDNA because GFP was exclusively apparent in stress‐treated embryos. The number of GFP‐expressing cells in the brain of huORFZ embryos at 72 h post‐fertilization (hpf) was positively correlated with the duration of heat‐shock treatment (Fig EV1A). Of the three different sections performed to isolate GFP(+) cells in the brain of stress‐treated huORFZ embryos (Fig EV1B), more GFP(+) cells appeared on one slide from section 2 (Fig EV1C). Upon examining cells with immunostaining with the glial and neuronal markers to identify GFP(+) cells around tectal and telencephalic ventricles, we found that heat‐shock‐induced GFP(+) cells were a subtype of glial cells but not neurons (Fig EV1D).Figure EV1Subtype‐specific neuronal cells in the brain of zebrafish embryos can respond to heat‐shock stress, resulting in GFP expression
Views of transgenic huORFZ embryos treated with heat‐shock stress for different exposure time under fluorescence microscopy. GFP was expressed in some subtype‐specific cells. The percentage shown on the right lower corner indicated the number of embryos those treated (n = 50) that presented similar GFP expression pattern.Schematic representations of different sections of the brain of zebrafish embryo. Lines of sections 1–3 indicated three sectional orientations.GFP‐expressing cells were observed under fluorescence microscopy through three dissections. Tectal ventricle (TeVe) and telencephalic ventricle (TVe) were observed in sections 1 and 2 of upper panels, respectively. Lower panels presented the higher magnification of area indicated by the box in the upper panels.Immunohistochemistry staining of cell types using antibodies conjugated with red fluorescence signal against glutamine synthase (GS) and HuCD. Nuclei were marked with DAPI in blue, while GFP‐expressing cells were presented in green. CeP: cerebellar plate; DT: dorsal thalamus; H: hypothalamus; Ha: habenula; OB: olfactory bulb; MO: medulla oblongata; P: pallium; PTd: dorsal part of posterior tuberculum; PTv: ventral part of posterior tuberculum; TeO: tectum opticum; T: midbrain tegmentum; and VT: ventral thalamus.
Most importantly, we next studied the genes involved in huORFchop‐TI, and a microarray was performed to compare the expression levels of genes between GFP(+)‐ and GFP(−)‐cells of the same subtype in the brain (Fig 1A). Based on Agilent GeneSpring software analysis, a total of 643 under‐represented transcripts and 1,014 over‐represented transcripts were identified in GFP(+) cells from heat‐shock stress when compared to GFP(−) cells. A total of 1,346 under‐represented transcripts and 2,210 over‐represented transcripts were identified in GFP(+) cells from hypoxic stress versus GFP(−) cells. Moreover, a total of 298 over‐represented transcripts and 104 under‐represented transcripts were identified for both stresses (Fig 1B and Appendix Table S1).Figure 1Method of searching for candidate genes identified as potentially involved in translational control mediated by huORFchop transcript through their upregulation in GFP‐expressing cells
Schematic diagram illustrating the separation of GFP‐expressing and non‐GFP‐expressing cells from stress‐treated zebrafish embryos of transgenic line huORFZ using LMD methods followed by microarrays.Graphical representation of two microarray expression analyses. A total of 298 over‐represented transcripts and 104 under‐represented transcripts were obtained from both microarrays.Upper panels: top view of EGFP expressed in zebrafish embryos derived from zebrafish transgenic line huORFZ under non‐stress and heat‐shock treatment. Middle panels: lateral view of endouc expression patterns in embryos under non‐stress condition and treated with heat‐shock. Lower panels: top view of endouc expression patterns.Quantitative RT–PCR determined the relative expression level of endouc mRNA in non‐stress‐ and heat‐shock‐treated embryos. Values representing the non‐stress‐ and heat‐shocked conditions were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).The endouc and ela3l (negative control) mRNAs were separately microinjected into zebrafish embryos from huORFZ at one‐cell stage. Upper panels: observation of GFP signal expressed in huORFZ embryos at 72 hpf. Lower panels: magnification of pictures corresponding to the upper panels.
Screening used the three steps: First, we focused on the highly upregulated genes from the microarray data. Typically, we would use an antisense morpholino‐knockdown strategy to determine highly downregulated putative genes. However, it was easier to confirm whether putative genes were involved in huORFchop‐TI simply through microinjection of mRNAs in non‐stressed zebrafish embryos. In total, 298 highly upregulated genes were picked for cloning. Of these, 183 candidate genes were successfully amplified by PCR and cloned into the pGEMTeasy vector for whole‐mount in situ hybridization (WISH) riboprobe synthesis; 107 out of the 183 candidates were successfully cloned into the pCS2 vector for in vitro mRNA synthesis. Each of these prepared mRNAs was then individually microinjected into zebrafish huORFZ embryos. Strikingly, most injected embryos were either lethal or devoid of GFP expression before 24 hpf with the notable exception of those introduced with mRNAs of endonuclease polyU‐specific C (endouc), eukaryotic translation initiation factor 3 subunit Hb (eif3hb), transcription elongation factor A (SII) (teca2), and peroxiredoxin 6 (prbx6). When these four putative genes were overexpressed in untreated huORFZ embryos, we observed that GFP reporter was detectable in all embryos (Appendix Fig S1A). Among these four candidates, only embryos injected with endouc mRNA could induce the GFP reporter specifically at the CNS tissues; the injection of ela3l mRNA (false positive control) did not (Fig 1E).Second, we performed WISH to examine whether these putative genes were upregulated in embryos after stress treatment. This step resulted in the discovery of two genes involved in translation including eif3hb and tcea2 as well as two genes that encoded enzymes including endouc and heme oxygenase 1a (hmox1); these had a significant upregulation in heat‐shock‐treated embryos compared to non‐stress control embryos (Appendix Fig S1B). We also found that some candidate genes were unchanged or slightly increased in the heat‐shock‐treated embryos compared to non‐stress control embryos such as chymotrypsinogen B1 (ctrb1), elastase 3 like (ela3l), serine protease 35 (prss35), death‐associated protein 1b (dap1b), peroxiredoxin 6 (prdx6), and glutamate‐cysteine ligase (gclc) (Appendix Fig S1C). We noticed that endouc transcript was significantly upregulated in brain compared to the control untreated embryos (Fig 1C). This upregulated expression pattern of endouc in the brain corresponded to the results of the GFP reporter, which was also significantly expressed in the brain of huORFZ embryos treated with heat‐shock (Fig 1C). The level of endouc transcript was markedly increased around the tectal ventricle and upper hindbrain after embryos were treated with heat‐shock stress (Fig 1C).Therefore, based on our stepwise elimination approach to determine the genes involved in huORFchop‐TI, zebrafish endouc (accession number si:dkey‐103i16.2)—which is conserved to the human orthologue ENDOU gene (Lee et al, 2017)—became the primary hit for further study for two reasons. First, the endouc transcript displayed a 5.39‐ and 2.85‐fold increase of expression in the heat‐shocked‐treated and hypoxia‐treated GFP(+) cells, respectively (Appendix Table S1). Second, real‐time qPCR result of endouc transcript was consistent with those obtained from the microarray data and WISH (Fig 1C and D). Therefore, we propose that Endouc might be a novel non‐ribosomal protein able to suppress huORFchop‐TI control thus facilitating the translation of DCS during ER stress.

Microarrays to screen for genes responsible for the translation of DCS during ER stress

To better understand the role of huORFchop‐TI in vivo, we first employed a zebrafish transgenic line huORFZ (Lee et al, 2011) which harbors huORFchop‐tagged reporter cDNA because GFP was exclusively apparent in stress‐treated embryos. The number of GFP‐expressing cells in the brain of huORFZ embryos at 72 h post‐fertilization (hpf) was positively correlated with the duration of heat‐shock treatment (Fig EV1A). Of the three different sections performed to isolate GFP(+) cells in the brain of stress‐treated huORFZ embryos (Fig EV1B), more GFP(+) cells appeared on one slide from section 2 (Fig EV1C). Upon examining cells with immunostaining with the glial and neuronal markers to identify GFP(+) cells around tectal and telencephalic ventricles, we found that heat‐shock‐induced GFP(+) cells were a subtype of glial cells but not neurons (Fig EV1D).

Subtype‐specific neuronal cells in the brain of zebrafish embryos can respond to heat‐shock stress, resulting in GFP expression

Views of transgenic huORFZ embryos treated with heat‐shock stress for different exposure time under fluorescence microscopy. GFP was expressed in some subtype‐specific cells. The percentage shown on the right lower corner indicated the number of embryos those treated (n = 50) that presented similar GFP expression pattern.Schematic representations of different sections of the brain of zebrafish embryo. Lines of sections 1–3 indicated three sectional orientations.GFP‐expressing cells were observed under fluorescence microscopy through three dissections. Tectal ventricle (TeVe) and telencephalic ventricle (TVe) were observed in sections 1 and 2 of upper panels, respectively. Lower panels presented the higher magnification of area indicated by the box in the upper panels.Immunohistochemistry staining of cell types using antibodies conjugated with red fluorescence signal against glutamine synthase (GS) and HuCD. Nuclei were marked with DAPI in blue, while GFP‐expressing cells were presented in green. CeP: cerebellar plate; DT: dorsal thalamus; H: hypothalamus; Ha: habenula; OB: olfactory bulb; MO: medulla oblongata; P: pallium; PTd: dorsal part of posterior tuberculum; PTv: ventral part of posterior tuberculum; TeO: tectum opticum; T: midbrain tegmentum; and VT: ventral thalamus.

Views of transgenic huORFZ embryos treated with heat‐shock stress for different exposure time under fluorescence microscopy. GFP was expressed in some subtype‐specific cells. The percentage shown on the right lower corner indicated the number of embryos those treated (n = 50) that presented similar GFP expression pattern.

Schematic representations of different sections of the brain of zebrafish embryo. Lines of sections 1–3 indicated three sectional orientations.

GFP‐expressing cells were observed under fluorescence microscopy through three dissections. Tectal ventricle (TeVe) and telencephalic ventricle (TVe) were observed in sections 1 and 2 of upper panels, respectively. Lower panels presented the higher magnification of area indicated by the box in the upper panels.

Immunohistochemistry staining of cell types using antibodies conjugated with red fluorescence signal against glutamine synthase (GS) and HuCD. Nuclei were marked with DAPI in blue, while GFP‐expressing cells were presented in green. CeP: cerebellar plate; DT: dorsal thalamus; H: hypothalamus; Ha: habenula; OB: olfactory bulb; MO: medulla oblongata; P: pallium; PTd: dorsal part of posterior tuberculum; PTv: ventral part of posterior tuberculum; TeO: tectum opticum; T: midbrain tegmentum; and VT: ventral thalamus.

Most importantly, we next studied the genes involved in huORFchop‐TI, and a microarray was performed to compare the expression levels of genes between GFP(+)‐ and GFP(−)‐cells of the same subtype in the brain (Fig 1A). Based on Agilent GeneSpring software analysis, a total of 643 under‐represented transcripts and 1,014 over‐represented transcripts were identified in GFP(+) cells from heat‐shock stress when compared to GFP(−) cells. A total of 1,346 under‐represented transcripts and 2,210 over‐represented transcripts were identified in GFP(+) cells from hypoxic stress versus GFP(−) cells. Moreover, a total of 298 over‐represented transcripts and 104 under‐represented transcripts were identified for both stresses (Fig 1B and Appendix Table S1).

Method of searching for candidate genes identified as potentially involved in translational control mediated by huORFchop transcript through their upregulation in GFP‐expressing cells

Schematic diagram illustrating the separation of GFP‐expressing and non‐GFP‐expressing cells from stress‐treated zebrafish embryos of transgenic line huORFZ using LMD methods followed by microarrays.Graphical representation of two microarray expression analyses. A total of 298 over‐represented transcripts and 104 under‐represented transcripts were obtained from both microarrays.Upper panels: top view of EGFP expressed in zebrafish embryos derived from zebrafish transgenic line huORFZ under non‐stress and heat‐shock treatment. Middle panels: lateral view of endouc expression patterns in embryos under non‐stress condition and treated with heat‐shock. Lower panels: top view of endouc expression patterns.Quantitative RT–PCR determined the relative expression level of endouc mRNA in non‐stress‐ and heat‐shock‐treated embryos. Values representing the non‐stress‐ and heat‐shocked conditions were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).The endouc and ela3l (negative control) mRNAs were separately microinjected into zebrafish embryos from huORFZ at one‐cell stage. Upper panels: observation of GFP signal expressed in huORFZ embryos at 72 hpf. Lower panels: magnification of pictures corresponding to the upper panels.

Schematic diagram illustrating the separation of GFP‐expressing and non‐GFP‐expressing cells from stress‐treated zebrafish embryos of transgenic line huORFZ using LMD methods followed by microarrays.

Graphical representation of two microarray expression analyses. A total of 298 over‐represented transcripts and 104 under‐represented transcripts were obtained from both microarrays.

Upper panels: top view of EGFP expressed in zebrafish embryos derived from zebrafish transgenic line huORFZ under non‐stress and heat‐shock treatment. Middle panels: lateral view of endouc expression patterns in embryos under non‐stress condition and treated with heat‐shock. Lower panels: top view of endouc expression patterns.

Quantitative RT–PCR determined the relative expression level of endouc mRNA in non‐stress‐ and heat‐shock‐treated embryos. Values representing the non‐stress‐ and heat‐shocked conditions were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).

The endouc and ela3l (negative control) mRNAs were separately microinjected into zebrafish embryos from huORFZ at one‐cell stage. Upper panels: observation of GFP signal expressed in huORFZ embryos at 72 hpf. Lower panels: magnification of pictures corresponding to the upper panels.

Screening used the three steps: First, we focused on the highly upregulated genes from the microarray data. Typically, we would use an antisense morpholino‐knockdown strategy to determine highly downregulated putative genes. However, it was easier to confirm whether putative genes were involved in huORFchop‐TI simply through microinjection of mRNAs in non‐stressed zebrafish embryos. In total, 298 highly upregulated genes were picked for cloning. Of these, 183 candidate genes were successfully amplified by PCR and cloned into the pGEMTeasy vector for whole‐mount in situ hybridization (WISH) riboprobe synthesis; 107 out of the 183 candidates were successfully cloned into the pCS2 vector for in vitro mRNA synthesis. Each of these prepared mRNAs was then individually microinjected into zebrafish huORFZ embryos. Strikingly, most injected embryos were either lethal or devoid of GFP expression before 24 hpf with the notable exception of those introduced with mRNAs of endonuclease polyU‐specific C (endouc), eukaryotic translation initiation factor 3 subunit Hb (eif3hb), transcription elongation factor A (SII) (teca2), and peroxiredoxin 6 (prbx6). When these four putative genes were overexpressed in untreated huORFZ embryos, we observed that GFP reporter was detectable in all embryos (Appendix Fig S1A). Among these four candidates, only embryos injected with endouc mRNA could induce the GFP reporter specifically at the CNS tissues; the injection of ela3l mRNA (false positive control) did not (Fig 1E).

Second, we performed WISH to examine whether these putative genes were upregulated in embryos after stress treatment. This step resulted in the discovery of two genes involved in translation including eif3hb and tcea2 as well as two genes that encoded enzymes including endouc and heme oxygenase 1a (hmox1); these had a significant upregulation in heat‐shock‐treated embryos compared to non‐stress control embryos (Appendix Fig S1B). We also found that some candidate genes were unchanged or slightly increased in the heat‐shock‐treated embryos compared to non‐stress control embryos such as chymotrypsinogen B1 (ctrb1), elastase 3 like (ela3l), serine protease 35 (prss35), death‐associated protein 1b (dap1b), peroxiredoxin 6 (prdx6), and glutamate‐cysteine ligase (gclc) (Appendix Fig S1C). We noticed that endouc transcript was significantly upregulated in brain compared to the control untreated embryos (Fig 1C). This upregulated expression pattern of endouc in the brain corresponded to the results of the GFP reporter, which was also significantly expressed in the brain of huORFZ embryos treated with heat‐shock (Fig 1C). The level of endouc transcript was markedly increased around the tectal ventricle and upper hindbrain after embryos were treated with heat‐shock stress (Fig 1C).

Therefore, based on our stepwise elimination approach to determine the genes involved in huORFchop‐TI, zebrafish endouc (accession number si:dkey‐103i16.2)—which is conserved to the human orthologue ENDOU gene (Lee et al, 2017)—became the primary hit for further study for two reasons. First, the endouc transcript displayed a 5.39‐ and 2.85‐fold increase of expression in the heat‐shocked‐treated and hypoxia‐treated GFP(+) cells, respectively (Appendix Table S1). Second, real‐time qPCR result of endouc transcript was consistent with those obtained from the microarray data and WISH (Fig 1C and D). Therefore, we propose that Endouc might be a novel non‐ribosomal protein able to suppress huORFchop‐TI control thus facilitating the translation of DCS during ER stress.

Transfection of Endouc/ENDOU‐1 alone is sufficient to drive the translation of DCS reporter even in the absence of stressWe quantified the increase of downstream luciferase (luc) reporter induced by overexpression of Endouc through in vivo study. Wild‐type (WT) zebrafish embryos were microinjected with a plasmid carrying huORFchop‐luc, a dual‐luc‐control vector (phRG‐TK), and an Endouc‐Flag‐overexpression vector. Under normal conditions, the zebrafish embryos injected with all three plasmids noted above exhibited 5.3‐fold higher luc activity than that of the control group, which was injected with plasmids puORFchop‐luc, pCS2 vector, and phRG‐TK (Fig 2A). The elevated luc activity that occurred in the Endouc‐injected embryos under normal conditions was consistent with that of the Endouc‐injected embryos under stress conditions such as the application of heat‐shock (Fig 2A).Figure 2Overexpression of zebrafish Endouc blocks huORFchop‐TI and translates downstream reporter gene in the absence of stress
AHistograms show the luc activity obtained from zebrafish embryos microinjected simultaneously with puORFchop‐luc, phRG‐TK, and each indicated plasmid followed by analysis of luc activity at 96 hpf. Embryos microinjected with pCS2 vector during normal condition (non‐stress; blank column) served as a control group, while the microinjected embryos treated with 40°C at 72 hpf for 1 h were the heat‐shocked group (solid column). The relative luc activity of microinjected embryos shown on the right was normalized by the amount of RNA in the injected embryos shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from the control group (this was normalized to 1). The luc activity mediated by the huORFchop transcript was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR. Values representing the non‐stress and stress condition were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).BHistograms present the luc activity obtained from HEK293T cells co‐transfected with puORFchop‐luc, phRG‐TK, and each indicated plasmid and then treated with either DMSO (control group) or thapsigargin (TH; stress group), followed by analysis of luc activity. Cells transfected with pCS2‐vector and kept at normal condition served as a control group. The relative luc activity of transfected cells shown on the right was normalized by the amount of mRNA in the transfected cells shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group (this was normalized to 1). The luc activity mediated by the huORFchop transcript was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR. Values representing the non‐stress and stress conditions were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).C–EWestern blot analysis of proteins, as indicated, in (C) zebrafish embryos, (D) HEK293T cells and (E) HeLa cells. GAPDH and α‐tubulin served as internal controls. Protein levels relative to each internal control are presented below each lane.

Source data are available online for this figure.
We next performed an in vitro study, in which the above plasmids were transfected into HEK293T cells. Under normal conditions, luc activity was increased ~2.6‐fold when the vector containing Endouc was transfected (Fig 2B). This was consistent with the results of Endouc‐overexpressed cells under stress conditions such as treatment with thapsigargin (TH)—an ER stress inducer (Fig 2B). The consistency of both in vitro and in vivo results led us to conclude that the overexpression of Endouc causes a dramatic increase in the DCS reporter expression suggesting that Endouc facilitates the translation of huORFchop–tag transcript.To investigate whether overexpression of Endouc can increase CHOP translation, we overexpressed Endouc at the one‐cell stage of zebrafish embryos treated with heat‐shock at 72 hpf. We then analyzed CHOP level at 96 hpf. The results demonstrated that overexpression of Endouc in zebrafish embryos increased the protein level of CHOP both in non‐stress and heat‐shock conditions (Fig 2C). Similar results were observed in the in vitro study where overexpression of Endouc in HEK293T and HeLa cells increased CHOP expression (Fig 2D and E).The translation of CHOP is tightly linked to the degree of p‐eIF2α during stress (Palam et al, 2011), and we asked if overexpression of Endouc would correlate with increased levels of p‐eIF2α. Western blot results showed that overexpression of Endouc in zebrafish embryos, as well as HEK293T and HeLa cells, did increase the level of p‐eIF2α (Fig 2C–E) suggesting that Endouc overexpression induces an increase of p‐eIF2α. These data demonstrated that Endouc functions as a positive regulator of CHOP translation during stress response.

Transfection of Endouc/ENDOU‐1 alone is sufficient to drive the translation of DCS reporter even in the absence of stress

We quantified the increase of downstream luciferase (luc) reporter induced by overexpression of Endouc through in vivo study. Wild‐type (WT) zebrafish embryos were microinjected with a plasmid carrying huORFchop‐luc, a dual‐luc‐control vector (phRG‐TK), and an Endouc‐Flag‐overexpression vector. Under normal conditions, the zebrafish embryos injected with all three plasmids noted above exhibited 5.3‐fold higher luc activity than that of the control group, which was injected with plasmids puORFchop‐luc, pCS2 vector, and phRG‐TK (Fig 2A). The elevated luc activity that occurred in the Endouc‐injected embryos under normal conditions was consistent with that of the Endouc‐injected embryos under stress conditions such as the application of heat‐shock (Fig 2A).

Overexpression of zebrafish Endouc blocks huORFchop‐TI and translates downstream reporter gene in the absence of stress

AHistograms show the luc activity obtained from zebrafish embryos microinjected simultaneously with puORFchop‐luc, phRG‐TK, and each indicated plasmid followed by analysis of luc activity at 96 hpf. Embryos microinjected with pCS2 vector during normal condition (non‐stress; blank column) served as a control group, while the microinjected embryos treated with 40°C at 72 hpf for 1 h were the heat‐shocked group (solid column). The relative luc activity of microinjected embryos shown on the right was normalized by the amount of RNA in the injected embryos shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from the control group (this was normalized to 1). The luc activity mediated by the huORFchop transcript was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR. Values representing the non‐stress and stress condition were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).BHistograms present the luc activity obtained from HEK293T cells co‐transfected with puORFchop‐luc, phRG‐TK, and each indicated plasmid and then treated with either DMSO (control group) or thapsigargin (TH; stress group), followed by analysis of luc activity. Cells transfected with pCS2‐vector and kept at normal condition served as a control group. The relative luc activity of transfected cells shown on the right was normalized by the amount of mRNA in the transfected cells shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group (this was normalized to 1). The luc activity mediated by the huORFchop transcript was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR. Values representing the non‐stress and stress conditions were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).C–EWestern blot analysis of proteins, as indicated, in (C) zebrafish embryos, (D) HEK293T cells and (E) HeLa cells. GAPDH and α‐tubulin served as internal controls. Protein levels relative to each internal control are presented below each lane.

Histograms show the luc activity obtained from zebrafish embryos microinjected simultaneously with puORFchop‐luc, phRG‐TK, and each indicated plasmid followed by analysis of luc activity at 96 hpf. Embryos microinjected with pCS2 vector during normal condition (non‐stress; blank column) served as a control group, while the microinjected embryos treated with 40°C at 72 hpf for 1 h were the heat‐shocked group (solid column). The relative luc activity of microinjected embryos shown on the right was normalized by the amount of RNA in the injected embryos shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from the control group (this was normalized to 1). The luc activity mediated by the huORFchop transcript was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR. Values representing the non‐stress and stress condition were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).

Histograms present the luc activity obtained from HEK293T cells co‐transfected with puORFchop‐luc, phRG‐TK, and each indicated plasmid and then treated with either DMSO (control group) or thapsigargin (TH; stress group), followed by analysis of luc activity. Cells transfected with pCS2‐vector and kept at normal condition served as a control group. The relative luc activity of transfected cells shown on the right was normalized by the amount of mRNA in the transfected cells shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group (this was normalized to 1). The luc activity mediated by the huORFchop transcript was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR. Values representing the non‐stress and stress conditions were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).

Western blot analysis of proteins, as indicated, in (C) zebrafish embryos, (D) HEK293T cells and (E) HeLa cells. GAPDH and α‐tubulin served as internal controls. Protein levels relative to each internal control are presented below each lane.

Source data are available online for this figure.

We next performed an in vitro study, in which the above plasmids were transfected into HEK293T cells. Under normal conditions, luc activity was increased ~2.6‐fold when the vector containing Endouc was transfected (Fig 2B). This was consistent with the results of Endouc‐overexpressed cells under stress conditions such as treatment with thapsigargin (TH)—an ER stress inducer (Fig 2B). The consistency of both in vitro and in vivo results led us to conclude that the overexpression of Endouc causes a dramatic increase in the DCS reporter expression suggesting that Endouc facilitates the translation of huORFchop–tag transcript.

To investigate whether overexpression of Endouc can increase CHOP translation, we overexpressed Endouc at the one‐cell stage of zebrafish embryos treated with heat‐shock at 72 hpf. We then analyzed CHOP level at 96 hpf. The results demonstrated that overexpression of Endouc in zebrafish embryos increased the protein level of CHOP both in non‐stress and heat‐shock conditions (Fig 2C). Similar results were observed in the in vitro study where overexpression of Endouc in HEK293T and HeLa cells increased CHOP expression (Fig 2D and E).

The translation of CHOP is tightly linked to the degree of p‐eIF2α during stress (Palam et al, 2011), and we asked if overexpression of Endouc would correlate with increased levels of p‐eIF2α. Western blot results showed that overexpression of Endouc in zebrafish embryos, as well as HEK293T and HeLa cells, did increase the level of p‐eIF2α (Fig 2C–E) suggesting that Endouc overexpression induces an increase of p‐eIF2α. These data demonstrated that Endouc functions as a positive regulator of CHOP translation during stress response.

Like zebrafish Endouc, the human orthologue ENDOU‐1 can suppress the inhibitory huORFchop elementTransfection of any one of three plasmids containing the three isoforms of human ENDOU could greatly induce the translation of reporter gene resulting in increased luc activity up to ~4.8‐fold, which is higher than that achieved by the transfection of zebrafish endouc expression plasmid (Fig EV2A and B). Similar to TH‐treated cells, transfection of overexpressed human ENDOU‐1 in cells could effectively suppress the repressive effect mediated by huORFchop transcript, in turn, enhancing the translation of the reporter gene (Fig EV2A).Figure EV2Overexpression of three isoforms of human ENDOU gene enables embryos to translate downstream GFP reporter, even in the absence of stress
Histograms presented the luc activity obtained from an in vitro experiment in which HEK293T cells were co‐transfected simultaneously with puORFchop–luc, phRG‐TK, and each one of the plasmids, as indicated, for 24 h and treated with either DMSO (control group) or thapsigargin (TH; stress group) for 2 h, followed by analysis of luc activity. The luc activity value of cells transfected with pCS2 vector and kept at normal condition served as a control group normalized as 1. The luc activity mediated by huORFchop element was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc mRNA was measured by RT–qPCR. The luc activity of transfected cells shown on the right was normalized by the mRNA level shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group (this was normalized to 1). The values featuring the non‐stressed and stressed conditions were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (*P < 0.01; **P < 0.005, ***P < 0.001).Protein levels of zebrafish Endouc and human ENDOU isoforms used in the luc assay shown in panel (A) were determined. The α‐tubulin served as an internal control.Western blot analysis of the time course of expression levels of human ENDOU‐1, p‐eIF2α, total eIF2α, and CHOP proteins in HEK293T cells after TH treatment. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane.

Source data are available online for this figure. 

To further confirm the suppressive effect of ENDOU‐1 on huORFchop‐TI, we studied the time course expression of endogenous ENDOU‐1 in HEK293T cells treated with stress. As shown in Fig EV2C, a rapid increase in the expression of ENDOU‐1 was observed 1 h after stress treatment, reaching a high level after 2 h; this level was maintained for up to 8 h. CHOP, however, exhibited a slow response and reduced expression after 8 h. The p‐eIF2α protein showed early expression, but it then slowed after 4 h. Therefore, we suggest that human ENDOU‐1 is also a stress‐responsive gene with a suppressive effect on huORFchop‐TI.

Like zebrafish Endouc, the human orthologue ENDOU‐1 can suppress the inhibitory huORFchop element

Transfection of any one of three plasmids containing the three isoforms of human ENDOU could greatly induce the translation of reporter gene resulting in increased luc activity up to ~4.8‐fold, which is higher than that achieved by the transfection of zebrafish endouc expression plasmid (Fig EV2A and B). Similar to TH‐treated cells, transfection of overexpressed human ENDOU‐1 in cells could effectively suppress the repressive effect mediated by huORFchop transcript, in turn, enhancing the translation of the reporter gene (Fig EV2A).

Overexpression of three isoforms of human ENDOU gene enables embryos to translate downstream GFP reporter, even in the absence of stress

Histograms presented the luc activity obtained from an in vitro experiment in which HEK293T cells were co‐transfected simultaneously with puORFchop–luc, phRG‐TK, and each one of the plasmids, as indicated, for 24 h and treated with either DMSO (control group) or thapsigargin (TH; stress group) for 2 h, followed by analysis of luc activity. The luc activity value of cells transfected with pCS2 vector and kept at normal condition served as a control group normalized as 1. The luc activity mediated by huORFchop element was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc mRNA was measured by RT–qPCR. The luc activity of transfected cells shown on the right was normalized by the mRNA level shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group (this was normalized to 1). The values featuring the non‐stressed and stressed conditions were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (*P < 0.01; **P < 0.005, ***P < 0.001).Protein levels of zebrafish Endouc and human ENDOU isoforms used in the luc assay shown in panel (A) were determined. The α‐tubulin served as an internal control.Western blot analysis of the time course of expression levels of human ENDOU‐1, p‐eIF2α, total eIF2α, and CHOP proteins in HEK293T cells after TH treatment. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane.

Histograms presented the luc activity obtained from an in vitro experiment in which HEK293T cells were co‐transfected simultaneously with puORFchop–luc, phRG‐TK, and each one of the plasmids, as indicated, for 24 h and treated with either DMSO (control group) or thapsigargin (TH; stress group) for 2 h, followed by analysis of luc activity. The luc activity value of cells transfected with pCS2 vector and kept at normal condition served as a control group normalized as 1. The luc activity mediated by huORFchop element was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc mRNA was measured by RT–qPCR. The luc activity of transfected cells shown on the right was normalized by the mRNA level shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group (this was normalized to 1). The values featuring the non‐stressed and stressed conditions were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (*P < 0.01; **P < 0.005, ***P < 0.001).

Protein levels of zebrafish Endouc and human ENDOU isoforms used in the luc assay shown in panel (A) were determined. The α‐tubulin served as an internal control.

Western blot analysis of the time course of expression levels of human ENDOU‐1, p‐eIF2α, total eIF2α, and CHOP proteins in HEK293T cells after TH treatment. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane.

Source data are available online for this figure.

To further confirm the suppressive effect of ENDOU‐1 on huORFchop‐TI, we studied the time course expression of endogenous ENDOU‐1 in HEK293T cells treated with stress. As shown in Fig EV2C, a rapid increase in the expression of ENDOU‐1 was observed 1 h after stress treatment, reaching a high level after 2 h; this level was maintained for up to 8 h. CHOP, however, exhibited a slow response and reduced expression after 8 h. The p‐eIF2α protein showed early expression, but it then slowed after 4 h. Therefore, we suggest that human ENDOU‐1 is also a stress‐responsive gene with a suppressive effect on huORFchop‐TI.

Endoribonuclease activity of Endouc/ENDOU‐1 is required to suppress the inhibitory huORFchop structureWe next sought to mechanistically explain the involvement of zebrafish Endouc in huORFchop‐TI. To accomplish this, a series of Endouc deletion constructs (Fig 3A) were co‐transfected with puORFchop–luc. A luc activity assay was then performed to analyze the effects of these constructs on regulating huORFchop‐TI both in vivo (zebrafish embryos) and in vitro (HEK293T cells).Figure 3Identification of Endouc functional domain involved in regulating huORFchop‐TI
Schematic representation of zebrafish Endouc protein and its internal deletion mutants.‡
‡
Histograms presenting the relative luc activity obtained from zebrafish embryos microinjected simultaneously with puORFchop‐luc, phRG‐TK, and each of the expression plasmids as indicated followed by analysis of luc activity at 96 hpf. Embryos microinjected with pCS2 vector during normal condition served as a control group (non‐stress; blank column), while the microinjected embryos treated with 40°C at 72 hpf for 1 h were the heat‐shocked group (solid column). The relative luc activity of microinjected embryos shown on the right was normalized by the amount of RNA in the injected embryos shown on the left. The relative luc activity is represented by the fold increase in Fluc/Rluc ratio over that obtained from control group. The luc activity mediated by the huORFchop element was measured by a dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.Histograms presented the luc activity obtained from HEK293T cells co‐transfected with puORFchop‐luc, phRG‐TK, and each one of the plasmids as indicated and treated with either DMSO (control group) or thapsigargin (TH; stress group) followed by analysis of luc activity. The luc activity of cells transfected with pCS2 vector and kept at normal condition served as a control group. The relative luc activity of transfected cells shown on the right was normalized by the amount of RNA in the transfected cells shown on the left. The relative luc activity is represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group normalized as 1. The luc activity mediated by the huORFchop transcript was measured by the dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.Zebrafish embryos at one‐cell stage were microinjected with plasmids as indicated, followed by extraction of embryonic proteins at the 96‐hpf stage. Protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants were detected using Western blot analysis. GAPDH served as an internal control.HEK293T cells were transfected with indicated plasmids followed by analysis of the protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants using Western blot. The α‐tubulin served as an internal control. Protein levels relative to each internal control are presented in each lane.

Source data are available online for this figure.
Compared with control, the luc activity was greatly increased in recipients with intact Endouc, Endouc△1–25, Endouc△26–136, and Endouc△299–310; it was slightly reduced in recipients with Endouc△137–298 (Fig 3B and C; Appendix Fig S2A and B). We noticed that transfection of Endouc△137–298 failed to suppress huORFchop‐mediated mRNA translation in recipients under both normal and stress conditions (Fig 3B). Western blot analysis also demonstrated that the protein levels of p‐eIF2α and CHOP were greatly reduced in cells expressing Endouc△137–298 both in vivo and in vitro (Fig 3D and E) suggesting that the key domain of Endouc for regulating huORFchop‐TI is located at residues 137–298. E243, H244, E249, H259, and K302 are necessary for human ENDOU‐2 endoribonuclease activity (Laneve et al, 2008), while H162 and K224 are necessary for Xenopus XendoU RNase activity (Schwarz & Blower, 2014). Interestingly, the key residues required to suppress huORFchop‐TI of Endouc are all within the 137–298 residues. On the basis of this evidence, we mutated zebrafish Endouc at H181 (EndoucH181A) and K242 (EndoucK242A) to alanine to generate two endoribonuclease‐deficient Endouc proteins (Fig 4A). Cells and embryos transfected with intact Endouc showed evidence of a huORFchop‐TI blockage. However, no luc activity was induced when embryos were transfected with both EndoucH181A and EndoucK242A (Fig 4B and C; Appendix Fig S2C and D). Furthermore, Western blot analysis demonstrated that overexpression of intact Endouc in recipients increased the protein levels of p‐eIF2α and CHOP at normal condition, while overexpression of both EndoucH181A and EndoucK242A, for which endoribonuclease activity was lost in recipients, did not (Fig 4D and E). These results were consistent with both TH‐treated cells and heat‐shock‐treated embryos. Therefore, we conclude that the endoribonuclease activity of zebrafish Endouc is critical to blocking huORFchop‐TI.Figure 4The positive effect of Endouc on p‐eIF2α and CHOP expressions is dependent on its endoribonuclease activity
Schematic representation of zebrafish Endouc protein and its internal deletion mutants.‡
Histograms presented the relative luc activity obtained from zebrafish embryos microinjected simultaneously with puORFchop‐luc, phRG‐TK, and each one of the expression plasmids as indicated followed by analysis of luc activity at 96 hpf. Embryos microinjected with pCS2 vector during normal condition (non‐stress; blank column) served as a control group, while the microinjected embryos treated with 40°C at 72 hpf for 1 h were the heat‐shocked group (solid column). The relative luc activity of microinjected embryos shown on the right was normalized by the amount of RNA in the injected embryos shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from control group. The luc activity mediated by the huORFchop transcript was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.Histograms presented the luc activity obtained from HEK293T cells co‐transfected with puORFchop‐luc, phRG‐TK, and each one of the plasmids as indicated; these were treated with either DMSO (control group) or thapsigargin (TH; stress group) followed by analysis of luc activity. The luc activity of cells transfected with pCS2 vector and kept at normal condition served as a control group. The relative luc activity of transfected cells shown on the right was normalized by the amount of RNA in the transfected cells shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group normalized as 1. The luc activity mediated by the huORFchop transcript was measured by the dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.Zebrafish embryos at one‐cell stage were microinjected with plasmids as indicated followed by extraction of embryonic proteins at the 96‐hpf stage. Protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants were detected using Western blot analysis. GAPDH served as an internal control.HEK293T cells were transfected with indicated plasmids followed by analysis of the protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants using Western blot. Here, α‐tubulin served as an internal control. Protein levels relative to each internal control were presented in each lane.

Source data are available online for this figure.
Next, we demonstrated that overexpression of ENDOU‐1 increased the expression levels of p‐eIF2α and CHOP in HEK293T and U373MG cells (Fig EV3A and B). However, when we mutated residue H285 of ENDOU‐1 (responsible for endoribonuclease activity), we found that overexpression of the ENDOU‐1H285A mutant affected neither p‐eIF2α nor CHOP levels (Fig EV3A and B) suggesting that the increase of p‐eIF2α and CHOP levels induced by ENDOU‐1 is dependent on the endoribonuclease activity of ENDOU‐1. We further confirmed that both Endouc/ENDOU‐1 were functionally conserved increasing p‐eIF2α and CHOP expression and playing a suppressive role in huORFchop‐TI through their endoribonuclease activity. The GFP(+)‐expressing cells were zebrafish glial‐like cells, while the U373MG cells were human glioblastoma–astrocytoma cells. Therefore, these results also strengthened our hypothesis that Endouc/ENDOU‐1 plays important roles in CHOP translation in glial‐like cells of zebrafish embryos.Figure EV3Increase of p‐eIF2α and CHOP expression levels induced by ENDOU‐1 is dependent on the endoribonuclease activity of Endouc
A, B(A) HEK293T and (B) U373MG cells were transfected with plasmid, as indicated, for 24 h, treated with either DMSO (control group) or thapsigargin (TH; stress group) for 2 h, and then submitted to detection of protein levels of CHOP, p‐eIF2α, total eIF2α, and ENDOU‐1 by Western blot analysis. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane as indicated.

Source data are available online for this figure.

Endoribonuclease activity of Endouc/ENDOU‐1 is required to suppress the inhibitory huORFchop structure

We next sought to mechanistically explain the involvement of zebrafish Endouc in huORFchop‐TI. To accomplish this, a series of Endouc deletion constructs (Fig 3A) were co‐transfected with puORFchop–luc. A luc activity assay was then performed to analyze the effects of these constructs on regulating huORFchop‐TI both in vivo (zebrafish embryos) and in vitro (HEK293T cells).

Identification of Endouc functional domain involved in regulating huORFchop‐TI

Schematic representation of zebrafish Endouc protein and its internal deletion mutants.‡
‡
Histograms presenting the relative luc activity obtained from zebrafish embryos microinjected simultaneously with puORFchop‐luc, phRG‐TK, and each of the expression plasmids as indicated followed by analysis of luc activity at 96 hpf. Embryos microinjected with pCS2 vector during normal condition served as a control group (non‐stress; blank column), while the microinjected embryos treated with 40°C at 72 hpf for 1 h were the heat‐shocked group (solid column). The relative luc activity of microinjected embryos shown on the right was normalized by the amount of RNA in the injected embryos shown on the left. The relative luc activity is represented by the fold increase in Fluc/Rluc ratio over that obtained from control group. The luc activity mediated by the huORFchop element was measured by a dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.Histograms presented the luc activity obtained from HEK293T cells co‐transfected with puORFchop‐luc, phRG‐TK, and each one of the plasmids as indicated and treated with either DMSO (control group) or thapsigargin (TH; stress group) followed by analysis of luc activity. The luc activity of cells transfected with pCS2 vector and kept at normal condition served as a control group. The relative luc activity of transfected cells shown on the right was normalized by the amount of RNA in the transfected cells shown on the left. The relative luc activity is represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group normalized as 1. The luc activity mediated by the huORFchop transcript was measured by the dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.Zebrafish embryos at one‐cell stage were microinjected with plasmids as indicated, followed by extraction of embryonic proteins at the 96‐hpf stage. Protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants were detected using Western blot analysis. GAPDH served as an internal control.HEK293T cells were transfected with indicated plasmids followed by analysis of the protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants using Western blot. The α‐tubulin served as an internal control. Protein levels relative to each internal control are presented in each lane.

Schematic representation of zebrafish Endouc protein and its internal deletion mutants.‡
‡

Histograms presenting the relative luc activity obtained from zebrafish embryos microinjected simultaneously with puORFchop‐luc, phRG‐TK, and each of the expression plasmids as indicated followed by analysis of luc activity at 96 hpf. Embryos microinjected with pCS2 vector during normal condition served as a control group (non‐stress; blank column), while the microinjected embryos treated with 40°C at 72 hpf for 1 h were the heat‐shocked group (solid column). The relative luc activity of microinjected embryos shown on the right was normalized by the amount of RNA in the injected embryos shown on the left. The relative luc activity is represented by the fold increase in Fluc/Rluc ratio over that obtained from control group. The luc activity mediated by the huORFchop element was measured by a dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.

Histograms presented the luc activity obtained from HEK293T cells co‐transfected with puORFchop‐luc, phRG‐TK, and each one of the plasmids as indicated and treated with either DMSO (control group) or thapsigargin (TH; stress group) followed by analysis of luc activity. The luc activity of cells transfected with pCS2 vector and kept at normal condition served as a control group. The relative luc activity of transfected cells shown on the right was normalized by the amount of RNA in the transfected cells shown on the left. The relative luc activity is represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group normalized as 1. The luc activity mediated by the huORFchop transcript was measured by the dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.

Zebrafish embryos at one‐cell stage were microinjected with plasmids as indicated, followed by extraction of embryonic proteins at the 96‐hpf stage. Protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants were detected using Western blot analysis. GAPDH served as an internal control.

HEK293T cells were transfected with indicated plasmids followed by analysis of the protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants using Western blot. The α‐tubulin served as an internal control. Protein levels relative to each internal control are presented in each lane.

Source data are available online for this figure.

Compared with control, the luc activity was greatly increased in recipients with intact Endouc, Endouc△1–25, Endouc△26–136, and Endouc△299–310; it was slightly reduced in recipients with Endouc△137–298 (Fig 3B and C; Appendix Fig S2A and B). We noticed that transfection of Endouc△137–298 failed to suppress huORFchop‐mediated mRNA translation in recipients under both normal and stress conditions (Fig 3B). Western blot analysis also demonstrated that the protein levels of p‐eIF2α and CHOP were greatly reduced in cells expressing Endouc△137–298 both in vivo and in vitro (Fig 3D and E) suggesting that the key domain of Endouc for regulating huORFchop‐TI is located at residues 137–298. E243, H244, E249, H259, and K302 are necessary for human ENDOU‐2 endoribonuclease activity (Laneve et al, 2008), while H162 and K224 are necessary for Xenopus XendoU RNase activity (Schwarz & Blower, 2014). Interestingly, the key residues required to suppress huORFchop‐TI of Endouc are all within the 137–298 residues. On the basis of this evidence, we mutated zebrafish Endouc at H181 (EndoucH181A) and K242 (EndoucK242A) to alanine to generate two endoribonuclease‐deficient Endouc proteins (Fig 4A). Cells and embryos transfected with intact Endouc showed evidence of a huORFchop‐TI blockage. However, no luc activity was induced when embryos were transfected with both EndoucH181A and EndoucK242A (Fig 4B and C; Appendix Fig S2C and D). Furthermore, Western blot analysis demonstrated that overexpression of intact Endouc in recipients increased the protein levels of p‐eIF2α and CHOP at normal condition, while overexpression of both EndoucH181A and EndoucK242A, for which endoribonuclease activity was lost in recipients, did not (Fig 4D and E). These results were consistent with both TH‐treated cells and heat‐shock‐treated embryos. Therefore, we conclude that the endoribonuclease activity of zebrafish Endouc is critical to blocking huORFchop‐TI.

The positive effect of Endouc on p‐eIF2α and CHOP expressions is dependent on its endoribonuclease activity

Schematic representation of zebrafish Endouc protein and its internal deletion mutants.‡
Histograms presented the relative luc activity obtained from zebrafish embryos microinjected simultaneously with puORFchop‐luc, phRG‐TK, and each one of the expression plasmids as indicated followed by analysis of luc activity at 96 hpf. Embryos microinjected with pCS2 vector during normal condition (non‐stress; blank column) served as a control group, while the microinjected embryos treated with 40°C at 72 hpf for 1 h were the heat‐shocked group (solid column). The relative luc activity of microinjected embryos shown on the right was normalized by the amount of RNA in the injected embryos shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from control group. The luc activity mediated by the huORFchop transcript was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.Histograms presented the luc activity obtained from HEK293T cells co‐transfected with puORFchop‐luc, phRG‐TK, and each one of the plasmids as indicated; these were treated with either DMSO (control group) or thapsigargin (TH; stress group) followed by analysis of luc activity. The luc activity of cells transfected with pCS2 vector and kept at normal condition served as a control group. The relative luc activity of transfected cells shown on the right was normalized by the amount of RNA in the transfected cells shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group normalized as 1. The luc activity mediated by the huORFchop transcript was measured by the dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.Zebrafish embryos at one‐cell stage were microinjected with plasmids as indicated followed by extraction of embryonic proteins at the 96‐hpf stage. Protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants were detected using Western blot analysis. GAPDH served as an internal control.HEK293T cells were transfected with indicated plasmids followed by analysis of the protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants using Western blot. Here, α‐tubulin served as an internal control. Protein levels relative to each internal control were presented in each lane.

Schematic representation of zebrafish Endouc protein and its internal deletion mutants.‡

Histograms presented the relative luc activity obtained from zebrafish embryos microinjected simultaneously with puORFchop‐luc, phRG‐TK, and each one of the expression plasmids as indicated followed by analysis of luc activity at 96 hpf. Embryos microinjected with pCS2 vector during normal condition (non‐stress; blank column) served as a control group, while the microinjected embryos treated with 40°C at 72 hpf for 1 h were the heat‐shocked group (solid column). The relative luc activity of microinjected embryos shown on the right was normalized by the amount of RNA in the injected embryos shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from control group. The luc activity mediated by the huORFchop transcript was measured by dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.

Histograms presented the luc activity obtained from HEK293T cells co‐transfected with puORFchop‐luc, phRG‐TK, and each one of the plasmids as indicated; these were treated with either DMSO (control group) or thapsigargin (TH; stress group) followed by analysis of luc activity. The luc activity of cells transfected with pCS2 vector and kept at normal condition served as a control group. The relative luc activity of transfected cells shown on the right was normalized by the amount of RNA in the transfected cells shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from pCS2‐transfected control group normalized as 1. The luc activity mediated by the huORFchop transcript was measured by the dual‐luciferase assay, and its corresponding huORFchop‐luc transcript was measured by RT–qPCR.

Zebrafish embryos at one‐cell stage were microinjected with plasmids as indicated followed by extraction of embryonic proteins at the 96‐hpf stage. Protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants were detected using Western blot analysis. GAPDH served as an internal control.

HEK293T cells were transfected with indicated plasmids followed by analysis of the protein levels of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants using Western blot. Here, α‐tubulin served as an internal control. Protein levels relative to each internal control were presented in each lane.

Source data are available online for this figure.

Next, we demonstrated that overexpression of ENDOU‐1 increased the expression levels of p‐eIF2α and CHOP in HEK293T and U373MG cells (Fig EV3A and B). However, when we mutated residue H285 of ENDOU‐1 (responsible for endoribonuclease activity), we found that overexpression of the ENDOU‐1H285A mutant affected neither p‐eIF2α nor CHOP levels (Fig EV3A and B) suggesting that the increase of p‐eIF2α and CHOP levels induced by ENDOU‐1 is dependent on the endoribonuclease activity of ENDOU‐1. We further confirmed that both Endouc/ENDOU‐1 were functionally conserved increasing p‐eIF2α and CHOP expression and playing a suppressive role in huORFchop‐TI through their endoribonuclease activity. The GFP(+)‐expressing cells were zebrafish glial‐like cells, while the U373MG cells were human glioblastoma–astrocytoma cells. Therefore, these results also strengthened our hypothesis that Endouc/ENDOU‐1 plays important roles in CHOP translation in glial‐like cells of zebrafish embryos.

Increase of p‐eIF2α and CHOP expression levels induced by ENDOU‐1 is dependent on the endoribonuclease activity of Endouc

A, B(A) HEK293T and (B) U373MG cells were transfected with plasmid, as indicated, for 24 h, treated with either DMSO (control group) or thapsigargin (TH; stress group) for 2 h, and then submitted to detection of protein levels of CHOP, p‐eIF2α, total eIF2α, and ENDOU‐1 by Western blot analysis. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane as indicated.

(A) HEK293T and (B) U373MG cells were transfected with plasmid, as indicated, for 24 h, treated with either DMSO (control group) or thapsigargin (TH; stress group) for 2 h, and then submitted to detection of protein levels of CHOP, p‐eIF2α, total eIF2α, and ENDOU‐1 by Western blot analysis. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane as indicated.

Source data are available online for this figure.

Increased CHOP protein level by overexpression of Endouc is specific but independent of ER stressTo further prove the specific roles of Endouc/ENDOU‐1 in facilitating CHOP mRNA translation, we first determined if CHOP induced by Endouc/ENDOU‐1 resulted from ER stress caused by Endouc/ENDOU‐1 overexpression. We examined the mRNA level of spliced xbp1 and protein level of ER stress markers in Endouc/ENDOU‐1‐expressing cells. Compared to TH‐treated cells, Endouc/ENDOU‐1‐overexpressed cells only displayed increased GADD34, p‐eIF2α, and CHOP, while spliced xbp1 mRNA and other stress factors were not increased (Fig EV4A, B and D). GADD34 was induced in Endouc‐ and ENDOU‐1‐overexpressing cells (Fig EV4D), which could have resulted from the increase of p‐eIF2α induced by overexpression of Endouc/ENDOU‐1 as supported by Novoa et al (2003) who reported that eIF2α phosphorylation promotes GADD34 expression. Most stress markers were unaffected suggesting the specific induction of p‐eIF2α and CHOP expression by Endouc/ENDOU‐1 again confirming that Endouc/ENDOU‐1 was functionally conserved as described above. Specifically, Endouc/ENDOU‐1 could increase eIF2α phosphorylation and suppress huORFchop‐mediated repression of mRNA translation to increase CHOP expression through their endoribonuclease activities.Figure EV4Increase of p‐eIF2α and CHOP induced by Endouc/ENDOU‐1 overexpression is specific, but independent of ER stress
A–CRT–PCR analysis: (A) Unspliced and spliced forms of xbp1 RNA were indicated. Cells treated with thapsigargin (TH) were used as positive control. GAPDH served as an internal control. The RT–qPCR was used to quantify the amount of (B) spliced xbp1 RNA and (C) CHOP mRNA. The amount of spliced xbp‐1 and CHOP mRNAs in cells transfected with pCS2 vector and kept at normal condition served as a control group, normalized as 1. The values were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).DWestern blot analysis of proteins, as indicated, expressed in HEK293T cells. Cells treated with TH were used as positive control. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane.

Source data are available online for this figure. 

The RT‐quantitative PCR (RT–qPCR) data indicate that the CHOP mRNA level did not increase in cells overexpressing Endouc/ENDOU‐1. Rather, CHOP mRNA was slightly reduced in these cells (Fig EV4C). In contrast, the amount of CHOP protein was increased in cells overexpressing Endouc/ENDOU‐1. Collectively, these data proved that Endouc/ENDOU‐1 positively regulates CHOP expression at the translational level.

Increased CHOP protein level by overexpression of Endouc is specific but independent of ER stress

To further prove the specific roles of Endouc/ENDOU‐1 in facilitating CHOP mRNA translation, we first determined if CHOP induced by Endouc/ENDOU‐1 resulted from ER stress caused by Endouc/ENDOU‐1 overexpression. We examined the mRNA level of spliced xbp1 and protein level of ER stress markers in Endouc/ENDOU‐1‐expressing cells. Compared to TH‐treated cells, Endouc/ENDOU‐1‐overexpressed cells only displayed increased GADD34, p‐eIF2α, and CHOP, while spliced xbp1 mRNA and other stress factors were not increased (Fig EV4A, B and D). GADD34 was induced in Endouc‐ and ENDOU‐1‐overexpressing cells (Fig EV4D), which could have resulted from the increase of p‐eIF2α induced by overexpression of Endouc/ENDOU‐1 as supported by Novoa et al (2003) who reported that eIF2α phosphorylation promotes GADD34 expression. Most stress markers were unaffected suggesting the specific induction of p‐eIF2α and CHOP expression by Endouc/ENDOU‐1 again confirming that Endouc/ENDOU‐1 was functionally conserved as described above. Specifically, Endouc/ENDOU‐1 could increase eIF2α phosphorylation and suppress huORFchop‐mediated repression of mRNA translation to increase CHOP expression through their endoribonuclease activities.

Increase of p‐eIF2α and CHOP induced by Endouc/ENDOU‐1 overexpression is specific, but independent of ER stress

A–CRT–PCR analysis: (A) Unspliced and spliced forms of xbp1 RNA were indicated. Cells treated with thapsigargin (TH) were used as positive control. GAPDH served as an internal control. The RT–qPCR was used to quantify the amount of (B) spliced xbp1 RNA and (C) CHOP mRNA. The amount of spliced xbp‐1 and CHOP mRNAs in cells transfected with pCS2 vector and kept at normal condition served as a control group, normalized as 1. The values were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).DWestern blot analysis of proteins, as indicated, expressed in HEK293T cells. Cells treated with TH were used as positive control. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane.

RT–PCR analysis: (A) Unspliced and spliced forms of xbp1 RNA were indicated. Cells treated with thapsigargin (TH) were used as positive control. GAPDH served as an internal control. The RT–qPCR was used to quantify the amount of (B) spliced xbp1 RNA and (C) CHOP mRNA. The amount of spliced xbp‐1 and CHOP mRNAs in cells transfected with pCS2 vector and kept at normal condition served as a control group, normalized as 1. The values were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).

Western blot analysis of proteins, as indicated, expressed in HEK293T cells. Cells treated with TH were used as positive control. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane.

Source data are available online for this figure.

The RT‐quantitative PCR (RT–qPCR) data indicate that the CHOP mRNA level did not increase in cells overexpressing Endouc/ENDOU‐1. Rather, CHOP mRNA was slightly reduced in these cells (Fig EV4C). In contrast, the amount of CHOP protein was increased in cells overexpressing Endouc/ENDOU‐1. Collectively, these data proved that Endouc/ENDOU‐1 positively regulates CHOP expression at the translational level.

Endouc/ENDOU‐1 induces CHOP translationTo investigate the effect of Endouc/ENDOU‐1 expression and eIF2α phosphorylation on CHOP translation, we first employed HEK293T cells treated with ISRIB—an integrated stress response inhibitor that can reverse the effect of eIF2α phosphorylation thereby inhibiting eIF2 function. HEK293T cells were transfected with an expression vector containing either vector only or endouc cDNA for 24 h; this was then treated with either DMSO or 200 nM ISRIB followed by an analysis of the level of CHOP protein. Similar to the CHOP protein weakly present in the control, CHOP was only slightly detected in ISRIB‐treated cells in the absence of Endouc (lanes 1 and 2 of Fig 5A). However, CHOP was significantly detected in the Endouc‐expressing cells in the absence of ISRIB (lane 3 of Fig 5B). Interestingly, we found that the protein level of CHOP expressed in cells treated with Endouc plus ISRIB was lower than that in cells treated with Endouc plus p‐eIF2α (lanes 3 versus 4 in Fig 5B) suggesting that both Endouc/ENDOU‐1 expression and eIF2α phosphorylation impact CHOP translation.Figure 5Besides p‐eIF2α, Endouc/ENDOU‐1 is another key protein that causes an increase in the CHOP protein level
AHEK293T cells were transfected with plasmids as indicated for 24 h and treated with DMSO or 200 nM ISRIB for 2 h followed by analysis of the protein level of CHOP, p‐eIF2α, total eIF2α, and Endou‐Flag using Western blot. The α‐tubulin served as an internal control. Protein levels relative to each internal control are presented in each lane.BHEK293T cells were transfected with plasmids as indicated for 24 h and treated with DMSO, thapsigargin (TH), ISRIB, or TH plus ISRIB for 2 h followed by analysis of the protein level of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants using Western blot. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane.CMEF wild‐type (WT) and its eIF2αS51A mutant (S51A) cells were treated with 1 μg of TH for 4 h followed by Western blot analysis of p‐eIF2α, total eIF2α, and ENDOU. GAPDH served as an internal control. Protein levels relative to each internal control are presented in each lane.D, EMEF WT and mutant S51A cells were transfected with plasmids as indicated for 24 h and treated with either DMSO or TH for 4 h followed by analysis of the protein levels of CHOP, p‐eIF2α, total eIF2α, GADD34, Endouc‐Flag and ENDOU‐1‐Flag using Western blot. GAPDH served as an internal control. Protein levels relative to each internal control were presented at each lane.FHEK293T cells were transfected with negative control siRNA and ENDOU‐siRNA for 48 h. After treatment, protein lysates were prepared and subjected to Western blot analysis for ENDOU‐1 and CHOP detection using antibodies. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented in each lane.

Source data are available online for this figure.
Next, we studied the control mechanism between Endouc and p‐eIF2α on suppressing huORFchop‐TI under stress. HEK293T cells were transfected with vector containing Endouc and EndoucK242A mutant, and then treated with TH and/or 200 nM ISRIB followed by an analysis of the protein level of CHOP. Compared to cells treated with TH, the CHOP protein was reduced in cells treated with TH plus ISRIB (lanes 2 versus 3 in Fig 5B) suggesting that ISRIB treatment could reduce the CHOP level induced by TH. This result was consistent with that obtained in Endouc‐expressing cells (lanes 4 versus 5 in Fig 5B). The level of CHOP expressed in cells treated with Endouc/TH/ISRIB was higher than that of cells treated with vector/TH/ISRIB (lanes 3 versus 5 in Fig 5B). CHOP was reduced in the EndoucK242A mutant (lanes 4 versus 6 in Fig 5B) compared to Endouc‐expressing cells. When Endouc‐expressing cells and EndoucK242A‐expressing cells were treated with TH and ISRIB, CHOP was significantly reduced in the EndoucK242A‐expressing cells in comparison to the Endouc‐expressing cells (lanes 5 versus 7 in Fig 5B). Unlike the expression profile of CHOP protein, we noticed that p‐eIF2α was always reduced in ISRIB‐treated cells. This suggests that Endouc alone is sufficient to induce CHOP translation independent of the effect of eIF2α phosphorylation.Our next step was to strengthen the hypothesis that Endouc alone is sufficient to induce CHOP translation and does so independent of eIF2α phosphorylation. To accomplish this, we employed mouse embryonic fibroblast (MEF) WT and mutant S51A cells that cannot phosphorylate eIF2α due to an S51A knock‐in mutation at the eIF2α phosphorylation site (Scheuner et al, 2001). First, when MEF WT cells were treated with TH, both p‐eIF2α and ENDOU‐1 were induced compared to DMSO‐treated control cells (Fig 5C) indicating that Endouc/ENDOU‐1 induced by stress is conserved among fish, mice, and humans. Additionally, the pronounced upregulation of Endouc/ENDOU‐1 failed to materialize in MEF S51A mutant cells treated with TH (Fig 5C) suggesting that p‐eIF2α also facilitates Endouc/ENDOU‐1 upregulation and that increased p‐eIF2α is required for the full expression of Endouc/ENDOU‐1 to facilitate CHOP translation during ER stress.Second, we further confirmed if Endouc/ENDOU‐1 could induce CHOP translation in the absence of p‐eIF2α. To address this question, MEF WT and S51A mutant cells were individually transfected with a plasmid harboring cDNA encoded with either Flag‐tagged Endouc/ENDOU‐1 or Endouc/ENDOU‐1K242A/H285A mutant followed by analysis of the protein level of CHOP and p‐eIF2α. In MEF WT cells, the results showed that both CHOP and p‐eIF2α were significantly induced in Endouc/ENDOU‐1‐overexpressing cells compared to untreated control cells, while neither CHOP nor p‐eIF2α was induced by Endouc/ENDOU‐1K242A/H285A mutant (Fig 5D and E). Nevertheless, CHOP was still increased in MEF S51A mutant cells treated with TH, although the level of CHOP was decreased in S51A mutant cells compared to WT cells (Fig 5D and E), suggesting that Endouc/ENDOU‐1 could induce CHOP translation in the absence of p‐eIF2α through its endoribonuclease activity because the endoribonuclease‐deficient mutant Endouc/ENDOU‐1K242A/H285A could not induce CHOP translation. We also noticed that GADD34 could be induced by Endouc/ENDOU‐1 in MEF WT cells but not in MEF S51A mutant cells (Fig 5D and E), suggesting that GADD34 is not directly induced by Endouc/ENDOU‐1. Rather, the GADD34 induced by Endouc/ENDOU‐1 is dependent on p‐eIF2α.CHOP was increased in the TH‐treated MEF S51A mutant cells, and thus, we conclude that (i) endoribonuclease activity of Endouc/ENDOU‐1 is necessary for Endouc/ENDOU‐1 to induce CHOP and p‐eIF2α expressions; (ii) CHOP translation induced by Endouc/ENDOU‐1 during stress is independent of eIF2α phosphorylation unlike the p‐eIF2α‐dependent induction of GADD34; and (iii) CHOP reached a maximal level in the presence of both Endouc/ENDOU‐1 and p‐eIF2α during ER stress.Finally, we performed siRNA to knock down ENDOU‐1 in HEK293T cells. Versus cells transfected with siCTRL‐siRNA after incubation with TH, the CHOP in HEK293T cells transfected with ENDOU‐1‐siRNA was reduced significantly in both the absence and presence of TH (Fig 5F). We conclude that overexpression of Endouc/ENDOU‐1 results in an increase in the p‐eIF2α and CHOP levels—this is specific and independent of ER stress. Nevertheless, both Endouc and p‐eIF2α are required for CHOP mRNA translation to reach a maximal level.

Endouc/ENDOU‐1 induces CHOP translation

To investigate the effect of Endouc/ENDOU‐1 expression and eIF2α phosphorylation on CHOP translation, we first employed HEK293T cells treated with ISRIB—an integrated stress response inhibitor that can reverse the effect of eIF2α phosphorylation thereby inhibiting eIF2 function. HEK293T cells were transfected with an expression vector containing either vector only or endouc cDNA for 24 h; this was then treated with either DMSO or 200 nM ISRIB followed by an analysis of the level of CHOP protein. Similar to the CHOP protein weakly present in the control, CHOP was only slightly detected in ISRIB‐treated cells in the absence of Endouc (lanes 1 and 2 of Fig 5A). However, CHOP was significantly detected in the Endouc‐expressing cells in the absence of ISRIB (lane 3 of Fig 5B). Interestingly, we found that the protein level of CHOP expressed in cells treated with Endouc plus ISRIB was lower than that in cells treated with Endouc plus p‐eIF2α (lanes 3 versus 4 in Fig 5B) suggesting that both Endouc/ENDOU‐1 expression and eIF2α phosphorylation impact CHOP translation.

Besides p‐eIF2α, Endouc/ENDOU‐1 is another key protein that causes an increase in the CHOP protein level

AHEK293T cells were transfected with plasmids as indicated for 24 h and treated with DMSO or 200 nM ISRIB for 2 h followed by analysis of the protein level of CHOP, p‐eIF2α, total eIF2α, and Endou‐Flag using Western blot. The α‐tubulin served as an internal control. Protein levels relative to each internal control are presented in each lane.BHEK293T cells were transfected with plasmids as indicated for 24 h and treated with DMSO, thapsigargin (TH), ISRIB, or TH plus ISRIB for 2 h followed by analysis of the protein level of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants using Western blot. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane.CMEF wild‐type (WT) and its eIF2αS51A mutant (S51A) cells were treated with 1 μg of TH for 4 h followed by Western blot analysis of p‐eIF2α, total eIF2α, and ENDOU. GAPDH served as an internal control. Protein levels relative to each internal control are presented in each lane.D, EMEF WT and mutant S51A cells were transfected with plasmids as indicated for 24 h and treated with either DMSO or TH for 4 h followed by analysis of the protein levels of CHOP, p‐eIF2α, total eIF2α, GADD34, Endouc‐Flag and ENDOU‐1‐Flag using Western blot. GAPDH served as an internal control. Protein levels relative to each internal control were presented at each lane.FHEK293T cells were transfected with negative control siRNA and ENDOU‐siRNA for 48 h. After treatment, protein lysates were prepared and subjected to Western blot analysis for ENDOU‐1 and CHOP detection using antibodies. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented in each lane.

HEK293T cells were transfected with plasmids as indicated for 24 h and treated with DMSO or 200 nM ISRIB for 2 h followed by analysis of the protein level of CHOP, p‐eIF2α, total eIF2α, and Endou‐Flag using Western blot. The α‐tubulin served as an internal control. Protein levels relative to each internal control are presented in each lane.

HEK293T cells were transfected with plasmids as indicated for 24 h and treated with DMSO, thapsigargin (TH), ISRIB, or TH plus ISRIB for 2 h followed by analysis of the protein level of CHOP, p‐eIF2α, total eIF2α, Endouc‐Flag, and its variants using Western blot. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented at each lane.

MEF wild‐type (WT) and its eIF2αS51A mutant (S51A) cells were treated with 1 μg of TH for 4 h followed by Western blot analysis of p‐eIF2α, total eIF2α, and ENDOU. GAPDH served as an internal control. Protein levels relative to each internal control are presented in each lane.

MEF WT and mutant S51A cells were transfected with plasmids as indicated for 24 h and treated with either DMSO or TH for 4 h followed by analysis of the protein levels of CHOP, p‐eIF2α, total eIF2α, GADD34, Endouc‐Flag and ENDOU‐1‐Flag using Western blot. GAPDH served as an internal control. Protein levels relative to each internal control were presented at each lane.

HEK293T cells were transfected with negative control siRNA and ENDOU‐siRNA for 48 h. After treatment, protein lysates were prepared and subjected to Western blot analysis for ENDOU‐1 and CHOP detection using antibodies. The α‐tubulin served as an internal control. Protein levels relative to each internal control were presented in each lane.

Source data are available online for this figure.

Next, we studied the control mechanism between Endouc and p‐eIF2α on suppressing huORFchop‐TI under stress. HEK293T cells were transfected with vector containing Endouc and EndoucK242A mutant, and then treated with TH and/or 200 nM ISRIB followed by an analysis of the protein level of CHOP. Compared to cells treated with TH, the CHOP protein was reduced in cells treated with TH plus ISRIB (lanes 2 versus 3 in Fig 5B) suggesting that ISRIB treatment could reduce the CHOP level induced by TH. This result was consistent with that obtained in Endouc‐expressing cells (lanes 4 versus 5 in Fig 5B). The level of CHOP expressed in cells treated with Endouc/TH/ISRIB was higher than that of cells treated with vector/TH/ISRIB (lanes 3 versus 5 in Fig 5B). CHOP was reduced in the EndoucK242A mutant (lanes 4 versus 6 in Fig 5B) compared to Endouc‐expressing cells. When Endouc‐expressing cells and EndoucK242A‐expressing cells were treated with TH and ISRIB, CHOP was significantly reduced in the EndoucK242A‐expressing cells in comparison to the Endouc‐expressing cells (lanes 5 versus 7 in Fig 5B). Unlike the expression profile of CHOP protein, we noticed that p‐eIF2α was always reduced in ISRIB‐treated cells. This suggests that Endouc alone is sufficient to induce CHOP translation independent of the effect of eIF2α phosphorylation.

Our next step was to strengthen the hypothesis that Endouc alone is sufficient to induce CHOP translation and does so independent of eIF2α phosphorylation. To accomplish this, we employed mouse embryonic fibroblast (MEF) WT and mutant S51A cells that cannot phosphorylate eIF2α due to an S51A knock‐in mutation at the eIF2α phosphorylation site (Scheuner et al, 2001). First, when MEF WT cells were treated with TH, both p‐eIF2α and ENDOU‐1 were induced compared to DMSO‐treated control cells (Fig 5C) indicating that Endouc/ENDOU‐1 induced by stress is conserved among fish, mice, and humans. Additionally, the pronounced upregulation of Endouc/ENDOU‐1 failed to materialize in MEF S51A mutant cells treated with TH (Fig 5C) suggesting that p‐eIF2α also facilitates Endouc/ENDOU‐1 upregulation and that increased p‐eIF2α is required for the full expression of Endouc/ENDOU‐1 to facilitate CHOP translation during ER stress.

Second, we further confirmed if Endouc/ENDOU‐1 could induce CHOP translation in the absence of p‐eIF2α. To address this question, MEF WT and S51A mutant cells were individually transfected with a plasmid harboring cDNA encoded with either Flag‐tagged Endouc/ENDOU‐1 or Endouc/ENDOU‐1K242A/H285A mutant followed by analysis of the protein level of CHOP and p‐eIF2α. In MEF WT cells, the results showed that both CHOP and p‐eIF2α were significantly induced in Endouc/ENDOU‐1‐overexpressing cells compared to untreated control cells, while neither CHOP nor p‐eIF2α was induced by Endouc/ENDOU‐1K242A/H285A mutant (Fig 5D and E). Nevertheless, CHOP was still increased in MEF S51A mutant cells treated with TH, although the level of CHOP was decreased in S51A mutant cells compared to WT cells (Fig 5D and E), suggesting that Endouc/ENDOU‐1 could induce CHOP translation in the absence of p‐eIF2α through its endoribonuclease activity because the endoribonuclease‐deficient mutant Endouc/ENDOU‐1K242A/H285A could not induce CHOP translation. We also noticed that GADD34 could be induced by Endouc/ENDOU‐1 in MEF WT cells but not in MEF S51A mutant cells (Fig 5D and E), suggesting that GADD34 is not directly induced by Endouc/ENDOU‐1. Rather, the GADD34 induced by Endouc/ENDOU‐1 is dependent on p‐eIF2α.

CHOP was increased in the TH‐treated MEF S51A mutant cells, and thus, we conclude that (i) endoribonuclease activity of Endouc/ENDOU‐1 is necessary for Endouc/ENDOU‐1 to induce CHOP and p‐eIF2α expressions; (ii) CHOP translation induced by Endouc/ENDOU‐1 during stress is independent of eIF2α phosphorylation unlike the p‐eIF2α‐dependent induction of GADD34; and (iii) CHOP reached a maximal level in the presence of both Endouc/ENDOU‐1 and p‐eIF2α during ER stress.

Finally, we performed siRNA to knock down ENDOU‐1 in HEK293T cells. Versus cells transfected with siCTRL‐siRNA after incubation with TH, the CHOP in HEK293T cells transfected with ENDOU‐1‐siRNA was reduced significantly in both the absence and presence of TH (Fig 5F). We conclude that overexpression of Endouc/ENDOU‐1 results in an increase in the p‐eIF2α and CHOP levels—this is specific and independent of ER stress. Nevertheless, both Endouc and p‐eIF2α are required for CHOP mRNA translation to reach a maximal level.

Endouc/ENDOU‐1 induces eIF2α phosphorylation through the PKR pathwayThe coronaviral EndoU family protein Nsp15 has been associated with PKR activity in host cells (Kindler et al, 2017). Since Endouc and ENDOU‐1 are members of the EndoU family of proteins, we next studied a possible role of RNA‐activated protein kinase PKR in the induction of eIF2α phosphorylation and translational upregulation of CHOP as a result of ENDOU‐1 expression. We analyzed the p‐PKR level in Endouc/ENDOU‐1‐expressing cells and found that the p‐PKR level was induced significantly in HEK293T, HeLa, and MEF WT cells treated with TH compared to DMSO‐treated control cells (Fig 6A; Appendix Fig S3A and B). Moreover, the p‐PKR level was induced in Endouc/ENDOU‐1‐overexpressing cells without stress treatment. Meanwhile, the endoribonuclease‐deficient EndoucK242A/ENDOU‐1H285A‐overexpressing cells did not induce p‐PKR suggesting that Endouc/ENDOU‐1 induces PKR phosphorylation through its endoribonuclease activity.Figure 6Endouc/ENDOU‐1 induces the phosphorylation of eIF2α through the PKR pathway
MEF wild‐type (WT) cells were transfected with plasmids as indicated for 24 h and treated with either DMSO or thapsigargin (TH) for 4 h followed by analysis of the protein level of p‐PKR, total PKR, Endouc‐Flag and ENDOU‐1‐Flag. GAPDH served as an internal control. Protein levels relative to each internal control were presented at each lane.MEF WT cells were transfected with plasmids as indicated for 24 h and treated with either DMSO or TH for 4 h followed by analysis of the protein levels of p‐eIF2α, total eIF2α, CHOP, Endouc‐Flag and ENDOU‐1‐Flag. GAPDH served as an internal control. Protein levels relative to each internal control were presented at each lane. The results obtained from the two trials are shown on the left and right.

Source data are available online for this figure.
We further employed PKRΔ6, which is a PKR dominant‐negative mutant that lacks six amino acids in its catalytic domain and, thus, fails to phosphorylate eIF2α (Koromilas et al, 1992; Romano et al, 1995; Tu et al, 2012). The level of p‐eIF2α was reduced when Endouc/ENDOU‐1 and PKRΔ6 were co‐expressed in cells (Fig 6B; Appendix Fig S3C and D). Compared to the control cells, even though p‐eIF2α was absent in cells co‐transfected with Endouc/ENDOU‐1 and PKRΔ6, CHOP protein was still present (Fig 6B; Appendix Fig S3C and D), albeit in minimally detectable level suggesting that inhibition of PKR pathway in Endouc‐expressing cells only blocks the induction of p‐eIF2α. Moreover, the cells transfected with mutant EndoucK242A/ENDOU‐1H285A failed to induce PKR, eIF2α phosphorylation, and CHOP protein (Fig 6B; Appendix Fig S3C and D). This line of evidence suggests that Endouc/ENDOU‐1 induces eIF2α phosphorylation through the PKR pathway.Taken together, it can be concluded that (i) the increase of eIF2α phosphorylation induced by Endouc is dependent on PKR phosphorylation; (ii) Endouc/ENDOU‐1 induces the PKR‐eIF2α pathway through its endoribonuclease activity; and (iii) expression between Endouc/ENDOU‐1 and p‐eIF2α exhibits feedback loop control.

Endouc/ENDOU‐1 induces eIF2α phosphorylation through the PKR pathway

The coronaviral EndoU family protein Nsp15 has been associated with PKR activity in host cells (Kindler et al, 2017). Since Endouc and ENDOU‐1 are members of the EndoU family of proteins, we next studied a possible role of RNA‐activated protein kinase PKR in the induction of eIF2α phosphorylation and translational upregulation of CHOP as a result of ENDOU‐1 expression. We analyzed the p‐PKR level in Endouc/ENDOU‐1‐expressing cells and found that the p‐PKR level was induced significantly in HEK293T, HeLa, and MEF WT cells treated with TH compared to DMSO‐treated control cells (Fig 6A; Appendix Fig S3A and B). Moreover, the p‐PKR level was induced in Endouc/ENDOU‐1‐overexpressing cells without stress treatment. Meanwhile, the endoribonuclease‐deficient EndoucK242A/ENDOU‐1H285A‐overexpressing cells did not induce p‐PKR suggesting that Endouc/ENDOU‐1 induces PKR phosphorylation through its endoribonuclease activity.

Endouc/ENDOU‐1 induces the phosphorylation of eIF2α through the PKR pathway

MEF wild‐type (WT) cells were transfected with plasmids as indicated for 24 h and treated with either DMSO or thapsigargin (TH) for 4 h followed by analysis of the protein level of p‐PKR, total PKR, Endouc‐Flag and ENDOU‐1‐Flag. GAPDH served as an internal control. Protein levels relative to each internal control were presented at each lane.MEF WT cells were transfected with plasmids as indicated for 24 h and treated with either DMSO or TH for 4 h followed by analysis of the protein levels of p‐eIF2α, total eIF2α, CHOP, Endouc‐Flag and ENDOU‐1‐Flag. GAPDH served as an internal control. Protein levels relative to each internal control were presented at each lane. The results obtained from the two trials are shown on the left and right.

MEF wild‐type (WT) cells were transfected with plasmids as indicated for 24 h and treated with either DMSO or thapsigargin (TH) for 4 h followed by analysis of the protein level of p‐PKR, total PKR, Endouc‐Flag and ENDOU‐1‐Flag. GAPDH served as an internal control. Protein levels relative to each internal control were presented at each lane.

MEF WT cells were transfected with plasmids as indicated for 24 h and treated with either DMSO or TH for 4 h followed by analysis of the protein levels of p‐eIF2α, total eIF2α, CHOP, Endouc‐Flag and ENDOU‐1‐Flag. GAPDH served as an internal control. Protein levels relative to each internal control were presented at each lane. The results obtained from the two trials are shown on the left and right.

Source data are available online for this figure.

We further employed PKRΔ6, which is a PKR dominant‐negative mutant that lacks six amino acids in its catalytic domain and, thus, fails to phosphorylate eIF2α (Koromilas et al, 1992; Romano et al, 1995; Tu et al, 2012). The level of p‐eIF2α was reduced when Endouc/ENDOU‐1 and PKRΔ6 were co‐expressed in cells (Fig 6B; Appendix Fig S3C and D). Compared to the control cells, even though p‐eIF2α was absent in cells co‐transfected with Endouc/ENDOU‐1 and PKRΔ6, CHOP protein was still present (Fig 6B; Appendix Fig S3C and D), albeit in minimally detectable level suggesting that inhibition of PKR pathway in Endouc‐expressing cells only blocks the induction of p‐eIF2α. Moreover, the cells transfected with mutant EndoucK242A/ENDOU‐1H285A failed to induce PKR, eIF2α phosphorylation, and CHOP protein (Fig 6B; Appendix Fig S3C and D). This line of evidence suggests that Endouc/ENDOU‐1 induces eIF2α phosphorylation through the PKR pathway.

Taken together, it can be concluded that (i) the increase of eIF2α phosphorylation induced by Endouc is dependent on PKR phosphorylation; (ii) Endouc/ENDOU‐1 induces the PKR‐eIF2α pathway through its endoribonuclease activity; and (iii) expression between Endouc/ENDOU‐1 and p‐eIF2α exhibits feedback loop control.

Small RNA fragments cleaved from huORFchop transcript under stress were detected in in vivo and in vitro systemsNorthern blot analysis was performed to further study whether the huORFchop transcript could be cleaved in vivo and in vitro under stress. An increase in small RNA fragments was observed in heat‐shock‐treated huORFZ embryos (Fig 7A), suggesting that the huORFchop‐gfp transcript could be cleaved in vivo under stress. In HEK293T cells, compared to DMSO‐treated control cells, endogenous CHOP mRNAs were increased, but small mRNAs of fewer than 50 nucleotides (nts) were significantly increased in TH‐treated cells. Meanwhile, the internal control gapdh mRNA did not change in either DMSO‐treated or TH‐treated cells (Fig 7B) suggesting that huORFchop transcripts could be cleaved under stress.Figure 7Endouc can bind and cleave the huORFchop transcript
ANorthern blot analysis to detect the endogenous huORFchop‐tagged gfp (uORF‐gfp) mRNA and its cleaved form in zebrafish embryos of transgenic line huORFZ treated with non‐stress condition (NS) or heat‐shocked (HS) stress. Zebrafish 18S rRNA served as an internal control. The RNA fragments cleaved from huORFchop transcript were detected.BNorthern blot analysis to detect endogenous CHOP mRNA in HEK293T cells treated with either DMSO (control group) or thapsigargin (TH). Human GAPDH RNA served as internal control. The RNA fragments cleaved from CHOP mRNA were detected.CThe predicted RNA sequences of Endouc recognition site on huORFchop transcript from 72 to 91 nts are illustrated. The predicted cleavage sites at uridylates (U) are indicated by arrowheads.D, EThe cleavage activity of Endouc was studied using materials presented at each lane. EndoucK242A: Endouc mutant; Extract: incubated in cell extracts; and Buf: reaction buffer. RNA probe without adding reaction buffer served as a negative control. The predicted cleavage sites (U) were mutated to generate mutant huORFchop transcript (unlabeled huORF‐mt) and labeled by biotin (biotin‐huORF‐mt). Unlabeled competitor RNA was added in reactions at five times higher than that of the biotinylated probe.FElectrophoretic mobility shifted assay of RNA. Biotin‐labeled huORFchop transcript was reacted with the increased amount of Endouc. The positions of free RNA (free probe) and RNA–protein complex (shifted band) were indicated on the right. The luc 105‐nt RNA and single‐stranded huORFchop 105‐nt DNA served as negative controls.

Small RNA fragments cleaved from huORFchop transcript under stress were detected in in vivo and in vitro systems

Northern blot analysis was performed to further study whether the huORFchop transcript could be cleaved in vivo and in vitro under stress. An increase in small RNA fragments was observed in heat‐shock‐treated huORFZ embryos (Fig 7A), suggesting that the huORFchop‐gfp transcript could be cleaved in vivo under stress. In HEK293T cells, compared to DMSO‐treated control cells, endogenous CHOP mRNAs were increased, but small mRNAs of fewer than 50 nucleotides (nts) were significantly increased in TH‐treated cells. Meanwhile, the internal control gapdh mRNA did not change in either DMSO‐treated or TH‐treated cells (Fig 7B) suggesting that huORFchop transcripts could be cleaved under stress.

Endouc can bind and cleave the huORFchop transcript

ANorthern blot analysis to detect the endogenous huORFchop‐tagged gfp (uORF‐gfp) mRNA and its cleaved form in zebrafish embryos of transgenic line huORFZ treated with non‐stress condition (NS) or heat‐shocked (HS) stress. Zebrafish 18S rRNA served as an internal control. The RNA fragments cleaved from huORFchop transcript were detected.BNorthern blot analysis to detect endogenous CHOP mRNA in HEK293T cells treated with either DMSO (control group) or thapsigargin (TH). Human GAPDH RNA served as internal control. The RNA fragments cleaved from CHOP mRNA were detected.CThe predicted RNA sequences of Endouc recognition site on huORFchop transcript from 72 to 91 nts are illustrated. The predicted cleavage sites at uridylates (U) are indicated by arrowheads.D, EThe cleavage activity of Endouc was studied using materials presented at each lane. EndoucK242A: Endouc mutant; Extract: incubated in cell extracts; and Buf: reaction buffer. RNA probe without adding reaction buffer served as a negative control. The predicted cleavage sites (U) were mutated to generate mutant huORFchop transcript (unlabeled huORF‐mt) and labeled by biotin (biotin‐huORF‐mt). Unlabeled competitor RNA was added in reactions at five times higher than that of the biotinylated probe.FElectrophoretic mobility shifted assay of RNA. Biotin‐labeled huORFchop transcript was reacted with the increased amount of Endouc. The positions of free RNA (free probe) and RNA–protein complex (shifted band) were indicated on the right. The luc 105‐nt RNA and single‐stranded huORFchop 105‐nt DNA served as negative controls.

Northern blot analysis to detect the endogenous huORFchop‐tagged gfp (uORF‐gfp) mRNA and its cleaved form in zebrafish embryos of transgenic line huORFZ treated with non‐stress condition (NS) or heat‐shocked (HS) stress. Zebrafish 18S rRNA served as an internal control. The RNA fragments cleaved from huORFchop transcript were detected.

Northern blot analysis to detect endogenous CHOP mRNA in HEK293T cells treated with either DMSO (control group) or thapsigargin (TH). Human GAPDH RNA served as internal control. The RNA fragments cleaved from CHOP mRNA were detected.

The predicted RNA sequences of Endouc recognition site on huORFchop transcript from 72 to 91 nts are illustrated. The predicted cleavage sites at uridylates (U) are indicated by arrowheads.

The cleavage activity of Endouc was studied using materials presented at each lane. EndoucK242A: Endouc mutant; Extract: incubated in cell extracts; and Buf: reaction buffer. RNA probe without adding reaction buffer served as a negative control. The predicted cleavage sites (U) were mutated to generate mutant huORFchop transcript (unlabeled huORF‐mt) and labeled by biotin (biotin‐huORF‐mt). Unlabeled competitor RNA was added in reactions at five times higher than that of the biotinylated probe.

Electrophoretic mobility shifted assay of RNA. Biotin‐labeled huORFchop transcript was reacted with the increased amount of Endouc. The positions of free RNA (free probe) and RNA–protein complex (shifted band) were indicated on the right. The luc 105‐nt RNA and single‐stranded huORFchop 105‐nt DNA served as negative controls.

Endouc/ENDOU‐1 cleaves huORFchop RNA at uridylates within a specific structureIt was reported that human ENDOU‐1 could cut at uridylates of oligonucleotide GGAACGUAUCCUUUGGGAG derived from small nucleolar RNAs containing multiple natural cleavage sites for XendoU (Laneve et al, 2003, 2008). Interestingly, this XendoU‐specific cutting sequence corresponds to sequences ranging from 72 to 91 nts of huORFchop transcript. When we used the Rochester Predict secondary structure server, we found that the sequences from 72 to 91 nts of huORFchop transcript per se could plausibly form a stem‐loop structure similar to that described by Laneve et al (2008) (Fig 7C), and that it could be repeatedly presented if the full‐length sequence of huORFchop transcript were used to predict its secondary structure (Appendix Fig S4).ENDOU has a ribonuclease domain that can cleave RNAs at UU dinucleotides or a single U in vitro (Caffarelli et al, 1997; Laneve et al, 2003, 2008; Bhardwaj et al, 2006). Therefore, we performed an RNase activity assay to examine whether zebrafish Endouc could cleave huORFchop transcript. Three fragments were generated in a dose‐dependent manner when biotin was labeled at the 3′‐end of huORFchop transcript (Fig 7D). However, cleaved fragments could not be detected when uridylates were mutated to guanines within 69–92 nt of huORFchop transcript (Fig 7D). Competition experiments also showed that the WT huORFchop transcript, but not mutant RNA, could reduce the cleavage signal (Fig 7E). Furthermore, mutant EndoucK242A failed to digest huORFchop transcript (Fig 7D). This line of evidence suggests that the cleavage of Endouc on huORFchop transcript is specific to uridylates and that the amino acid residue at K242 is required for RNase activity of Endouc.

Endouc/ENDOU‐1 cleaves huORFchop RNA at uridylates within a specific structure

It was reported that human ENDOU‐1 could cut at uridylates of oligonucleotide GGAACGUAUCCUUUGGGAG derived from small nucleolar RNAs containing multiple natural cleavage sites for XendoU (Laneve et al, 2003, 2008). Interestingly, this XendoU‐specific cutting sequence corresponds to sequences ranging from 72 to 91 nts of huORFchop transcript. When we used the Rochester Predict secondary structure server, we found that the sequences from 72 to 91 nts of huORFchop transcript per se could plausibly form a stem‐loop structure similar to that described by Laneve et al (2008) (Fig 7C), and that it could be repeatedly presented if the full‐length sequence of huORFchop transcript were used to predict its secondary structure (Appendix Fig S4).

ENDOU has a ribonuclease domain that can cleave RNAs at UU dinucleotides or a single U in vitro (Caffarelli et al, 1997; Laneve et al, 2003, 2008; Bhardwaj et al, 2006). Therefore, we performed an RNase activity assay to examine whether zebrafish Endouc could cleave huORFchop transcript. Three fragments were generated in a dose‐dependent manner when biotin was labeled at the 3′‐end of huORFchop transcript (Fig 7D). However, cleaved fragments could not be detected when uridylates were mutated to guanines within 69–92 nt of huORFchop transcript (Fig 7D). Competition experiments also showed that the WT huORFchop transcript, but not mutant RNA, could reduce the cleavage signal (Fig 7E). Furthermore, mutant EndoucK242A failed to digest huORFchop transcript (Fig 7D). This line of evidence suggests that the cleavage of Endouc on huORFchop transcript is specific to uridylates and that the amino acid residue at K242 is required for RNase activity of Endouc.

Endouc binds directly with huORFchop transcriptTo confirm binding between Endouc and huORFchop transcript, we performed RNA EMSA using biotin‐labeled, full‐length huORFchop transcript and Endouc. Compared to luc mRNA (Appendix Fig S5A), the results confirmed that Endouc could bind specifically with the huORFchop transcript in a dose‐dependent manner (Fig 7F). However, the band shift shown on the EMSA gel was abolished by the increase of unlabeled huORFchop transcript (Appendix Fig S5B). Moreover, Endouc bound neither luc mRNA nor huORFchop DNA (Fig 7F), suggesting that the binding between huORFchop transcript and Endouc is specific.RNA EMSA demonstrated that Endouc could still bind with the mutated huORFchop transcript (Appendix Fig S5C) although binding efficiency was relatively weak relative to the normal huORFchop transcript. However, Endouc failed to cleave mutated huORFchop transcript (Fig 7E) in which uridylates within 72–91 nts were mutated. Thus, it is clear that binding in the presence of uridylate is permissive for the specific cleavage of huORFchop transcript by Endouc.

Endouc binds directly with huORFchop transcript

To confirm binding between Endouc and huORFchop transcript, we performed RNA EMSA using biotin‐labeled, full‐length huORFchop transcript and Endouc. Compared to luc mRNA (Appendix Fig S5A), the results confirmed that Endouc could bind specifically with the huORFchop transcript in a dose‐dependent manner (Fig 7F). However, the band shift shown on the EMSA gel was abolished by the increase of unlabeled huORFchop transcript (Appendix Fig S5B). Moreover, Endouc bound neither luc mRNA nor huORFchop DNA (Fig 7F), suggesting that the binding between huORFchop transcript and Endouc is specific.

RNA EMSA demonstrated that Endouc could still bind with the mutated huORFchop transcript (Appendix Fig S5C) although binding efficiency was relatively weak relative to the normal huORFchop transcript. However, Endouc failed to cleave mutated huORFchop transcript (Fig 7E) in which uridylates within 72–91 nts were mutated. Thus, it is clear that binding in the presence of uridylate is permissive for the specific cleavage of huORFchop transcript by Endouc.

Endouc facilitates stalled ribosomes to suppress the inhibitory huORFchop element and plays a positive role in the translation of CHOP mRNAWe next performed polysome profiling to directly examine the role of Endouc in regulating huORFchop‐TI. HEK293T cells were transiently transfected with Endouc‐ or EndoucK242A‐expression vector and treated with cycloheximide to arrest elongating ribosomes on mRNAs. Their lysates were then subjected to sucrose gradient ultracentrifugation, and sample fraction were pooled into three groups shown in Figure 8A: free ribosomal subunits (Fractions 1 and 2), monosomes (Fractions 3–6), and light and heavy polysomes (Fractions 7–12). Compared to control cells, overexpression of Endouc caused an inhibition of global translational initiation as documented by the loss of heavy polysomes; overexpression of EndoucK242A did not change the polysome profile (Fig 8A). Western blot analysis demonstrated that both Endouc and EndoucK242A were mostly found in the monosome and light polysome fractions, whereas a small percentage of Endouc was found in the heavy polysome fractions (Fig 8A). Although no difference was observed in the distribution at each polysome fraction between protein levels of Endouc and EndoucK242A, the overexpression of Endouc could reduce global translation, while overexpression of EndoucK242A did not change the polysome profile. Moreover, LC‐MS/MS analysis proved that the major proteins interacting with recombinant Endouc‐Flag were ribosomal proteins and proteins involved in translation (Appendix Table S2). This evidence suggested that blockade of huORFchop‐TI via Endouc is dependent on its endoribonuclease activity.Figure 8Endouc associates with monosomes and plays a key role in regulating CHOP post‐translation
AHEK293T cells were transfected with pCS2 vector (black), pCS2‐Endouc (red), and pCS2‐EndoucK242A (green) for 24 h and subjected to either DMSO (control) or thapsigargin (TH; stress) treatment for 1 h. The lysates of transfected cells were applied to polysome profile analysis (PPA). The sucrose gradient from top to bottom was shown from left to right, respectively. The sedimentation of 40S, 60S, 80S, and polysomes was indicated. Lysates from control, Endouc‐expressing, and EndoucK242A‐expressing cells were applied to PPA. The resulting fractions were TCA‐precipitated, analyzed by SDS–PAGE, and subjected to Western blot using antibodies against Flag and S6. Upper and lower rows of Flag- and S6 blots were Endouc‐ and EndoucK242A‐ expressing cells, respectively.B, CThe lysates from control (pCS2), Endouc‐expressing (Endouc), and EndoucK242A‐expressing (EndoucK242A) cells were applied to PPA. The resulting fractions from 1‐10 collected from sucrose gradient were subsequently subjected to RT–qPCR assay to quantify (B) exogenous huORFchop‐tagged luc (luc mRNA) and (C) endogenous CHOP transcripts (CHOP mRNA). The distribution of relative abundance of luc and CHOP transcripts contained in each fraction was determined.D, EHEK293T cells were co‐transfected with pCS2‐Endouc and phuORFchop‐luc for 24 h and then subjected to DMSO or TH treatment for 2 h. The lysates of transfected cells were then applied to PPA. Total RNAs were isolated from heavy polysome fractions followed by performing 5′RACE to characterize the nucleotide sequences of truncated (D) exogenous huORFchop‐luc and (E) endogenous huORFchop‐CHOP transcripts. The first sequence at N‐terminus of RNA samples from the truncated forms derived from exogenous huORFchop‐luc (n = 14) and endogenous huORFchop‐CHOP transcripts (n = 11) was determined; the percentages of positive colonies containing various 5′UTR lengths (truncated forms) of luc and CHOP transcripts among total colonies examined were calculated.

Source data are available online for this figure.
We next measured the levels of endogenous CHOP and exogenous huORFchop‐tagged luc transcripts among the sucrose fractions. In the control cells, the huORFchop‐tagged luc (Fig 8B) and CHOP mRNAs (Fig 8C) were largely associated with monosomes (Fractions 4 and 5). However, in Endouc‐overexpressing cells, the huORFchop‐tagged luc and CHOP mRNAs displayed a shift of 43 and 37% of transcripts from monosome toward light polysomes (Fractions 6–8), respectively. In contrast, similar to the control group, overexpression of endoribonuclease‐deficient mutant EndoucK242A did not trigger these mRNAs into the light polysome fraction (Fractions 6–8). The addition of Endouc caused a higher percentage of luc reporter and CHOP mRNAs to shift from the monosome toward the light polysome fraction to proceed with continuous translation (Appendix Fig S6A and B).

Endouc facilitates stalled ribosomes to suppress the inhibitory huORFchop element and plays a positive role in the translation of CHOP mRNA

We next performed polysome profiling to directly examine the role of Endouc in regulating huORFchop‐TI. HEK293T cells were transiently transfected with Endouc‐ or EndoucK242A‐expression vector and treated with cycloheximide to arrest elongating ribosomes on mRNAs. Their lysates were then subjected to sucrose gradient ultracentrifugation, and sample fraction were pooled into three groups shown in Figure 8A: free ribosomal subunits (Fractions 1 and 2), monosomes (Fractions 3–6), and light and heavy polysomes (Fractions 7–12). Compared to control cells, overexpression of Endouc caused an inhibition of global translational initiation as documented by the loss of heavy polysomes; overexpression of EndoucK242A did not change the polysome profile (Fig 8A). Western blot analysis demonstrated that both Endouc and EndoucK242A were mostly found in the monosome and light polysome fractions, whereas a small percentage of Endouc was found in the heavy polysome fractions (Fig 8A). Although no difference was observed in the distribution at each polysome fraction between protein levels of Endouc and EndoucK242A, the overexpression of Endouc could reduce global translation, while overexpression of EndoucK242A did not change the polysome profile. Moreover, LC‐MS/MS analysis proved that the major proteins interacting with recombinant Endouc‐Flag were ribosomal proteins and proteins involved in translation (Appendix Table S2). This evidence suggested that blockade of huORFchop‐TI via Endouc is dependent on its endoribonuclease activity.

Endouc associates with monosomes and plays a key role in regulating CHOP post‐translation

AHEK293T cells were transfected with pCS2 vector (black), pCS2‐Endouc (red), and pCS2‐EndoucK242A (green) for 24 h and subjected to either DMSO (control) or thapsigargin (TH; stress) treatment for 1 h. The lysates of transfected cells were applied to polysome profile analysis (PPA). The sucrose gradient from top to bottom was shown from left to right, respectively. The sedimentation of 40S, 60S, 80S, and polysomes was indicated. Lysates from control, Endouc‐expressing, and EndoucK242A‐expressing cells were applied to PPA. The resulting fractions were TCA‐precipitated, analyzed by SDS–PAGE, and subjected to Western blot using antibodies against Flag and S6. Upper and lower rows of Flag- and S6 blots were Endouc‐ and EndoucK242A‐ expressing cells, respectively.B, CThe lysates from control (pCS2), Endouc‐expressing (Endouc), and EndoucK242A‐expressing (EndoucK242A) cells were applied to PPA. The resulting fractions from 1‐10 collected from sucrose gradient were subsequently subjected to RT–qPCR assay to quantify (B) exogenous huORFchop‐tagged luc (luc mRNA) and (C) endogenous CHOP transcripts (CHOP mRNA). The distribution of relative abundance of luc and CHOP transcripts contained in each fraction was determined.D, EHEK293T cells were co‐transfected with pCS2‐Endouc and phuORFchop‐luc for 24 h and then subjected to DMSO or TH treatment for 2 h. The lysates of transfected cells were then applied to PPA. Total RNAs were isolated from heavy polysome fractions followed by performing 5′RACE to characterize the nucleotide sequences of truncated (D) exogenous huORFchop‐luc and (E) endogenous huORFchop‐CHOP transcripts. The first sequence at N‐terminus of RNA samples from the truncated forms derived from exogenous huORFchop‐luc (n = 14) and endogenous huORFchop‐CHOP transcripts (n = 11) was determined; the percentages of positive colonies containing various 5′UTR lengths (truncated forms) of luc and CHOP transcripts among total colonies examined were calculated.

HEK293T cells were transfected with pCS2 vector (black), pCS2‐Endouc (red), and pCS2‐EndoucK242A (green) for 24 h and subjected to either DMSO (control) or thapsigargin (TH; stress) treatment for 1 h. The lysates of transfected cells were applied to polysome profile analysis (PPA). The sucrose gradient from top to bottom was shown from left to right, respectively. The sedimentation of 40S, 60S, 80S, and polysomes was indicated. Lysates from control, Endouc‐expressing, and EndoucK242A‐expressing cells were applied to PPA. The resulting fractions were TCA‐precipitated, analyzed by SDS–PAGE, and subjected to Western blot using antibodies against Flag and S6. Upper and lower rows of Flag- and S6 blots were Endouc‐ and EndoucK242A‐ expressing cells, respectively.

The lysates from control (pCS2), Endouc‐expressing (Endouc), and EndoucK242A‐expressing (EndoucK242A) cells were applied to PPA. The resulting fractions from 1‐10 collected from sucrose gradient were subsequently subjected to RT–qPCR assay to quantify (B) exogenous huORFchop‐tagged luc (luc mRNA) and (C) endogenous CHOP transcripts (CHOP mRNA). The distribution of relative abundance of luc and CHOP transcripts contained in each fraction was determined.

HEK293T cells were co‐transfected with pCS2‐Endouc and phuORFchop‐luc for 24 h and then subjected to DMSO or TH treatment for 2 h. The lysates of transfected cells were then applied to PPA. Total RNAs were isolated from heavy polysome fractions followed by performing 5′RACE to characterize the nucleotide sequences of truncated (D) exogenous huORFchop‐luc and (E) endogenous huORFchop‐CHOP transcripts. The first sequence at N‐terminus of RNA samples from the truncated forms derived from exogenous huORFchop‐luc (n = 14) and endogenous huORFchop‐CHOP transcripts (n = 11) was determined; the percentages of positive colonies containing various 5′UTR lengths (truncated forms) of luc and CHOP transcripts among total colonies examined were calculated.

Source data are available online for this figure.

We next measured the levels of endogenous CHOP and exogenous huORFchop‐tagged luc transcripts among the sucrose fractions. In the control cells, the huORFchop‐tagged luc (Fig 8B) and CHOP mRNAs (Fig 8C) were largely associated with monosomes (Fractions 4 and 5). However, in Endouc‐overexpressing cells, the huORFchop‐tagged luc and CHOP mRNAs displayed a shift of 43 and 37% of transcripts from monosome toward light polysomes (Fractions 6–8), respectively. In contrast, similar to the control group, overexpression of endoribonuclease‐deficient mutant EndoucK242A did not trigger these mRNAs into the light polysome fraction (Fractions 6–8). The addition of Endouc caused a higher percentage of luc reporter and CHOP mRNAs to shift from the monosome toward the light polysome fraction to proceed with continuous translation (Appendix Fig S6A and B).

The RNA fragment cleaved by Endouc at the 80G‐81U of huORFchop transcript is found in polysome fractionsNext, to isolate and characterize CHOP mRNA after nuclease activity, we performed an experiment in which exogenous huORFchop‐luc mRNA and endogenous huORFchop‐CHOP mRNA were isolated from polysome fractions and characterized them after nuclease activity. After the sequences of these isolated mRNAs were confirmed using 5′RACE, we focused on finding truncated sequences derived from the huORFchop transcript in the polysome fractions isolated from both exogenous huORFchop‐luc mRNA and endogenous huORFchop‐CHOP mRNA groups. Specifically, the first nucleotide of major truncated mRNAs was the T at the nucleotide position of 81 (81T) (Fig 8D and E) suggesting the following: First, exogenous huORFchop‐luc (42%, n = 14) and endogenous huORFchop‐CHOP mRNAs (55%, n = 11) were cleaved by Endouc, a uridylate‐specific endoribonuclease, at the nucleotide position between 80G and 81U (80G‐81U). Second, the non‐5′‐capped truncated forms of exogenous huORFchop‐luc transcript and endogenous huORFchop‐CHOP mRNA were undergoing translation.Interestingly, the identified Endouc cleavage site at 80G‐81U was located at the hairpin structure of huORFchop cassette, which is one of the predicted sites recognized and cleaved by Endouc (Fig 7C). Moreover, 80G‐81U sequences were located next to the Ile‐Phe‐Ile sequence (82A‐90A nts), which is an essential sequence that allows the huORFchop motif to act as an inhibitory element to repress the translation of CHOP mRNA at normal state (Young et al, 2016). Therefore, we demonstrated cleavage of both exogenous huORFchop‐luc and endogenous huORFchop‐CHOP transcripts by Endouc at 80G‐81U.

The RNA fragment cleaved by Endouc at the 80G‐81U of huORFchop transcript is found in polysome fractions

Next, to isolate and characterize CHOP mRNA after nuclease activity, we performed an experiment in which exogenous huORFchop‐luc mRNA and endogenous huORFchop‐CHOP mRNA were isolated from polysome fractions and characterized them after nuclease activity. After the sequences of these isolated mRNAs were confirmed using 5′RACE, we focused on finding truncated sequences derived from the huORFchop transcript in the polysome fractions isolated from both exogenous huORFchop‐luc mRNA and endogenous huORFchop‐CHOP mRNA groups. Specifically, the first nucleotide of major truncated mRNAs was the T at the nucleotide position of 81 (81T) (Fig 8D and E) suggesting the following: First, exogenous huORFchop‐luc (42%, n = 14) and endogenous huORFchop‐CHOP mRNAs (55%, n = 11) were cleaved by Endouc, a uridylate‐specific endoribonuclease, at the nucleotide position between 80G and 81U (80G‐81U). Second, the non‐5′‐capped truncated forms of exogenous huORFchop‐luc transcript and endogenous huORFchop‐CHOP mRNA were undergoing translation.

Interestingly, the identified Endouc cleavage site at 80G‐81U was located at the hairpin structure of huORFchop cassette, which is one of the predicted sites recognized and cleaved by Endouc (Fig 7C). Moreover, 80G‐81U sequences were located next to the Ile‐Phe‐Ile sequence (82A‐90A nts), which is an essential sequence that allows the huORFchop motif to act as an inhibitory element to repress the translation of CHOP mRNA at normal state (Young et al, 2016). Therefore, we demonstrated cleavage of both exogenous huORFchop‐luc and endogenous huORFchop‐CHOP transcripts by Endouc at 80G‐81U.

The uncapped truncated transcript derived from huORFchop RNA did not decay immediately in cellsTo determine whether overexpression of Endouc could change the turnover of CHOP mRNA, we treated Endouc‐overexpressing MEF cells with TH followed by actinomycin D treatment. The half‐life of CHOP transcript from the control (no stress treatment) and TH‐treated cells was 2.61 and 2.4 h, respectively (Fig EV5A and B). This is consistent with the previous study reported by Rutkowski et al (2006) and Young et al (2016) who demonstrated that the half‐life of CHOP transcript is 2–3 h, while overexpression of Endouc reduced the half‐life of CHOP transcript to 2.18 h suggesting that overexpression of Endouc did not significantly impact the turnover of CHOP transcript.Figure EV5The truncated form of CHOP mRNA after cleavage does not decay immediately in cells
A, BMEF cells were transfected with plasmid pCS2 vector or pCS2‐Endouc‐Flag for 24 h, cultured either in DMSO or thapsigargin (TH) for 3 h, followed by actinomycin D (10 μg/ml) treatment for 0, 1, 2, or 3 h as indicated. RNAs were then harvested, and the expression level of CHOP mRNA was quantified by RT–qPCR and normalized to 18S rRNA. Data present as bar graph (A) and line graph (B) were calculated from three independent experiments and presented as mean ± SD (n = 3) Student's t‐test was used to determine significant differences between each group (*P < 0.05, **P < 0.005, ***P < 0.001).CMEF cells were transfected with plasmid pCS2‐Endouc‐Flag for 24 h, followed by actinomycin D treatment for 0, 1, 2, or 3 h, as indicated. RNAs were then harvested, and the expression levels of full‐length and cleaved CHOP mRNA were quantified by RT–qPCR and normalized to 18S rRNA. Data were calculated from three independent experiments and presented as mean ± SD (n = 3).DMEF cells were transfected with plasmid pCS2‐Endouc‐Flag for 24 h. Some cells were used to isolate total RNAs, and the expression levels of full‐length and cleaved CHOP mRNA were measured by RT–qPCR and normalized by 18S rRNA. Data were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005). Other cells were used to detect the protein levels of CHOP and Endouc‐Flag proteins by Western blot analysis. The GAPDH served as an internal control.

Source data are available online for this figure. 

We next determined whether this uncapped truncated transcript would decay immediately or remain in cells. To accomplish this, MEF cells were transfected with pCS2‐Endouc‐Flag followed by actinomycin D treatment; an adaptor at the 5′ end was added for cDNA synthesis. We then used primers that could specifically target the adaptor‐linked cleavage of huORFchop cDNA for our qPCR experiment (see Materials and Methods). Compared to the half‐life between capped full‐length huORFchop and uncapped truncated huORFchop transcripts, we found that the half‐life of full‐length huORFchop transcript was 3 h, while the half‐life of uncapped truncated huORFchop transcript was reduced to about 43 min (Fig EV5C), suggesting that the uncapped truncated huORFchop transcript could remain in cells for up to 90 min. Since some translated mRNAs have a half‐life shorter than 1 h (Sharova, et al, 2009) or even shorter than 30 min (Raghavan et al, 2002), the half‐life of huORFchop 81‐105‐nt transcript is reasonably sufficient for CHOP translation. We also demonstrated that more CHOP transcript correlated with more CHOP protein further suggesting that the truncated transcripts cleaved from huORFchop may contribute to the translation of CHOP protein (Fig EV5D).

The uncapped truncated transcript derived from huORFchop RNA did not decay immediately in cells

To determine whether overexpression of Endouc could change the turnover of CHOP mRNA, we treated Endouc‐overexpressing MEF cells with TH followed by actinomycin D treatment. The half‐life of CHOP transcript from the control (no stress treatment) and TH‐treated cells was 2.61 and 2.4 h, respectively (Fig EV5A and B). This is consistent with the previous study reported by Rutkowski et al (2006) and Young et al (2016) who demonstrated that the half‐life of CHOP transcript is 2–3 h, while overexpression of Endouc reduced the half‐life of CHOP transcript to 2.18 h suggesting that overexpression of Endouc did not significantly impact the turnover of CHOP transcript.

The truncated form of CHOP mRNA after cleavage does not decay immediately in cells

A, BMEF cells were transfected with plasmid pCS2 vector or pCS2‐Endouc‐Flag for 24 h, cultured either in DMSO or thapsigargin (TH) for 3 h, followed by actinomycin D (10 μg/ml) treatment for 0, 1, 2, or 3 h as indicated. RNAs were then harvested, and the expression level of CHOP mRNA was quantified by RT–qPCR and normalized to 18S rRNA. Data present as bar graph (A) and line graph (B) were calculated from three independent experiments and presented as mean ± SD (n = 3) Student's t‐test was used to determine significant differences between each group (*P < 0.05, **P < 0.005, ***P < 0.001).CMEF cells were transfected with plasmid pCS2‐Endouc‐Flag for 24 h, followed by actinomycin D treatment for 0, 1, 2, or 3 h, as indicated. RNAs were then harvested, and the expression levels of full‐length and cleaved CHOP mRNA were quantified by RT–qPCR and normalized to 18S rRNA. Data were calculated from three independent experiments and presented as mean ± SD (n = 3).DMEF cells were transfected with plasmid pCS2‐Endouc‐Flag for 24 h. Some cells were used to isolate total RNAs, and the expression levels of full‐length and cleaved CHOP mRNA were measured by RT–qPCR and normalized by 18S rRNA. Data were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005). Other cells were used to detect the protein levels of CHOP and Endouc‐Flag proteins by Western blot analysis. The GAPDH served as an internal control.

MEF cells were transfected with plasmid pCS2 vector or pCS2‐Endouc‐Flag for 24 h, cultured either in DMSO or thapsigargin (TH) for 3 h, followed by actinomycin D (10 μg/ml) treatment for 0, 1, 2, or 3 h as indicated. RNAs were then harvested, and the expression level of CHOP mRNA was quantified by RT–qPCR and normalized to 18S rRNA. Data present as bar graph (A) and line graph (B) were calculated from three independent experiments and presented as mean ± SD (n = 3) Student's t‐test was used to determine significant differences between each group (*P < 0.05, **P < 0.005, ***P < 0.001).

MEF cells were transfected with plasmid pCS2‐Endouc‐Flag for 24 h, followed by actinomycin D treatment for 0, 1, 2, or 3 h, as indicated. RNAs were then harvested, and the expression levels of full‐length and cleaved CHOP mRNA were quantified by RT–qPCR and normalized to 18S rRNA. Data were calculated from three independent experiments and presented as mean ± SD (n = 3).

MEF cells were transfected with plasmid pCS2‐Endouc‐Flag for 24 h. Some cells were used to isolate total RNAs, and the expression levels of full‐length and cleaved CHOP mRNA were measured by RT–qPCR and normalized by 18S rRNA. Data were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005). Other cells were used to detect the protein levels of CHOP and Endouc‐Flag proteins by Western blot analysis. The GAPDH served as an internal control.

Source data are available online for this figure.

We next determined whether this uncapped truncated transcript would decay immediately or remain in cells. To accomplish this, MEF cells were transfected with pCS2‐Endouc‐Flag followed by actinomycin D treatment; an adaptor at the 5′ end was added for cDNA synthesis. We then used primers that could specifically target the adaptor‐linked cleavage of huORFchop cDNA for our qPCR experiment (see Materials and Methods). Compared to the half‐life between capped full‐length huORFchop and uncapped truncated huORFchop transcripts, we found that the half‐life of full‐length huORFchop transcript was 3 h, while the half‐life of uncapped truncated huORFchop transcript was reduced to about 43 min (Fig EV5C), suggesting that the uncapped truncated huORFchop transcript could remain in cells for up to 90 min. Since some translated mRNAs have a half‐life shorter than 1 h (Sharova, et al, 2009) or even shorter than 30 min (Raghavan et al, 2002), the half‐life of huORFchop 81‐105‐nt transcript is reasonably sufficient for CHOP translation. We also demonstrated that more CHOP transcript correlated with more CHOP protein further suggesting that the truncated transcripts cleaved from huORFchop may contribute to the translation of CHOP protein (Fig EV5D).

Truncated huORFchop‐69‐105‐nt transcript contains internal translation activity, but truncated huORFchop‐81‐105‐nt transcript does notLuc and CHOP proteins could be produced from exogenous huORFchop‐luc and endogenous huORFchop‐CHOP mRNAs after Endouc binding and cleavage. Therefore, we hypothesized that the 5′‐region of the truncated huORFchop transcript may have internal translational activity that enables the translation of DCS. The predicted RNA sequences of Endouc recognition, binding, and cleavage sites on the 105‐nt huORFchop RNA were located from 72 to 91 nts (Fig 7C). Thus, we first engineered a truncated form that only contains 69–105 nts RNA fragment (huORFchop−69‐105‐nt), and then employed a pRF/phpRF bicistronic system (Terenin et al, 2017). In the pRF bicistronic reporter system, Renilla luc (Rluc) protein is translated in a cap‐dependent manner from the first cistron through ribosomal scanning, while Firefly luc (Fluc) protein is translated from the second cistron only if a fragment inserted at the upstream region possesses a cap‐independent translation mechanism. Since the Fluc/Rluc ratio represents a cap‐independent translation efficiency, four transcripts were designed to obtain Fluc/Rluc ratios. Appendix Fig S7A indicates the following: (i) RF: No insertion between Rluc‐ and Fluc‐cistrons which served as mock control; (ii) RF‐huORF69‐105 (RF‐huORFchop‐69‐105‐nt; an experimental group); (iii) RF‐cmyc (the 5′UTR of c‐myc which served as positive control); and (iv) RF‐Non105 (a 105‐nt RNA encoding luc that served as negative control). Based on the value of Fluc/Rluc ratio, we determined the cap‐independent translation efficiency of each inserted fragment. Following in vitro synthesis, these four bicistronic mRNAs were transfected individually into HEK293T cells, and their luc activities were determined by a dual‐luc assay. The Fluc/Rluc ratio obtained from transfection of RNA transcribed from vector construct only with no insert (RF; mock group) was set as 1. Compared to the RF group, the Fluc/Rluc ratio from transfection of RF‐Non105 transcript (negative control) did not increase (Appendix Fig S7A) suggesting that the 105‐nt luc RNA did not contain internal translation activity. In contrast, the Fluc/Rluc ratio from transfection of RF‐cmyc transcript (positive control) was dramatically increased ~100‐fold compared to the RF group suggesting that the 5′UTR of c‐myc contained strong internal translation activity consistent with the results reported by Stoneley et al (1998). Interestingly, the Fluc/Rluc ratio from transfection of the RF‐huORF69‐105 transcript was increased 3.2‐fold over that of control RF mRNA (Appendix Fig S7A) suggesting that the RF‐huORF69‐105 RNA fragment also has internal translation activity.The phpRF bicistronic reporter vector system excludes the possibility of either ribosomal reinitiation or read‐through from the stop codon of the Rluc cistron to the start codon of the Fluc cistron. Thus, a stable palindromic sequence is inserted before the start codon of the first ORF of Rluc generating a stable mRNA hairpin (−55 Kcal/mol) that inhibits cap‐dependent translation (Yang & Wang, 2019). Consequently, cap‐dependent translation of the Rluc cistron is diminished, while cap‐independent internal translation driven by downstream Fluc cistron is not affected. As depicted in Appendix Fig S7, four transcripts were designed as hpRF (mock control), hpRF‐huORF69‐105 (an experimental group), hpRF‐cmyc (positive control), and hpRF‐Non105 (negative control). Since the amount of input mRNA was not significantly different (Appendix Fig S7A and C) and Rluc activity was no different among the four groups, the change in the Fluc/Rluc ratio was completely due to the change in the Fluc activity (Appendix Fig S7B and D).As shown in Appendix Fig S7B, the Fluc/Rluc ratio from cells transfected with bicistronic mRNA containing c‐myc (hpRF‐cmyc transcript) was significantly increased: up to ~1,500‐fold greater than the Fluc/Rluc ratio of hpRF mock control and hpRF‐Non105 negative control groups. This increase indicates that the 5′UTR of c‐myc contained pronounced internal translation activity. Interestingly, the Fluc/Rluc ratio of cells transfected with the hpRF‐huORF69‐105 transcript was increased 80.5‐fold over that of the hpRF mock control (Appendix Fig S7C). These findings strongly suggest that the huORFchop‐69‐105‐nt transcript has internal translation activity albeit relatively weaker than c‐myc.To further determine whether the truncated form of huORFchop‐81‐105‐nt derived from Endouc‐cleaved huORFchop also contains internal translation activity, we engineered the 81‐105‐nt segment of huORFchop where the cleavage site of Endouc was at 80G‐81U. We then co‐transfected endouc mRNA with bicistronic RNA fragment containing huORFchop‐69‐105‐nt and huORFchop‐81‐105‐nt into HEK293T cells. As shown in Appendix Fig S8, the Fluc/Rluc ratio in cells transfected with endouc mRNA was slightly increased from 3.4‐ to 5.5‐fold in RF‐huORF69‐105 mRNA (Appendix Fig S8A and B). The ratio was dramatically elevated from 43‐ to 226‐fold in hpRF‐huORF69‐105 mRNA compared to that of control hpRF mRNA (Appendix Fig S8C and D). Surprisingly, the internal translation activity driven by huORFchop‐81‐105‐nt was completely lost suggesting that the truncated Endouc‐cut huORFchop transcript from the T81 cleavage site to the end of the huORFchop transcript (i.e., huORFchop‐81‐105‐nt) does not contain internal translation activity.

Truncated huORFchop‐69‐105‐nt transcript contains internal translation activity, but truncated huORFchop‐81‐105‐nt transcript does not

Luc and CHOP proteins could be produced from exogenous huORFchop‐luc and endogenous huORFchop‐CHOP mRNAs after Endouc binding and cleavage. Therefore, we hypothesized that the 5′‐region of the truncated huORFchop transcript may have internal translational activity that enables the translation of DCS. The predicted RNA sequences of Endouc recognition, binding, and cleavage sites on the 105‐nt huORFchop RNA were located from 72 to 91 nts (Fig 7C). Thus, we first engineered a truncated form that only contains 69–105 nts RNA fragment (huORFchop−69‐105‐nt), and then employed a pRF/phpRF bicistronic system (Terenin et al, 2017). In the pRF bicistronic reporter system, Renilla luc (Rluc) protein is translated in a cap‐dependent manner from the first cistron through ribosomal scanning, while Firefly luc (Fluc) protein is translated from the second cistron only if a fragment inserted at the upstream region possesses a cap‐independent translation mechanism. Since the Fluc/Rluc ratio represents a cap‐independent translation efficiency, four transcripts were designed to obtain Fluc/Rluc ratios. Appendix Fig S7A indicates the following: (i) RF: No insertion between Rluc‐ and Fluc‐cistrons which served as mock control; (ii) RF‐huORF69‐105 (RF‐huORFchop‐69‐105‐nt; an experimental group); (iii) RF‐cmyc (the 5′UTR of c‐myc which served as positive control); and (iv) RF‐Non105 (a 105‐nt RNA encoding luc that served as negative control). Based on the value of Fluc/Rluc ratio, we determined the cap‐independent translation efficiency of each inserted fragment. Following in vitro synthesis, these four bicistronic mRNAs were transfected individually into HEK293T cells, and their luc activities were determined by a dual‐luc assay. The Fluc/Rluc ratio obtained from transfection of RNA transcribed from vector construct only with no insert (RF; mock group) was set as 1. Compared to the RF group, the Fluc/Rluc ratio from transfection of RF‐Non105 transcript (negative control) did not increase (Appendix Fig S7A) suggesting that the 105‐nt luc RNA did not contain internal translation activity. In contrast, the Fluc/Rluc ratio from transfection of RF‐cmyc transcript (positive control) was dramatically increased ~100‐fold compared to the RF group suggesting that the 5′UTR of c‐myc contained strong internal translation activity consistent with the results reported by Stoneley et al (1998). Interestingly, the Fluc/Rluc ratio from transfection of the RF‐huORF69‐105 transcript was increased 3.2‐fold over that of control RF mRNA (Appendix Fig S7A) suggesting that the RF‐huORF69‐105 RNA fragment also has internal translation activity.

The phpRF bicistronic reporter vector system excludes the possibility of either ribosomal reinitiation or read‐through from the stop codon of the Rluc cistron to the start codon of the Fluc cistron. Thus, a stable palindromic sequence is inserted before the start codon of the first ORF of Rluc generating a stable mRNA hairpin (−55 Kcal/mol) that inhibits cap‐dependent translation (Yang & Wang, 2019). Consequently, cap‐dependent translation of the Rluc cistron is diminished, while cap‐independent internal translation driven by downstream Fluc cistron is not affected. As depicted in Appendix Fig S7, four transcripts were designed as hpRF (mock control), hpRF‐huORF69‐105 (an experimental group), hpRF‐cmyc (positive control), and hpRF‐Non105 (negative control). Since the amount of input mRNA was not significantly different (Appendix Fig S7A and C) and Rluc activity was no different among the four groups, the change in the Fluc/Rluc ratio was completely due to the change in the Fluc activity (Appendix Fig S7B and D).

As shown in Appendix Fig S7B, the Fluc/Rluc ratio from cells transfected with bicistronic mRNA containing c‐myc (hpRF‐cmyc transcript) was significantly increased: up to ~1,500‐fold greater than the Fluc/Rluc ratio of hpRF mock control and hpRF‐Non105 negative control groups. This increase indicates that the 5′UTR of c‐myc contained pronounced internal translation activity. Interestingly, the Fluc/Rluc ratio of cells transfected with the hpRF‐huORF69‐105 transcript was increased 80.5‐fold over that of the hpRF mock control (Appendix Fig S7C). These findings strongly suggest that the huORFchop‐69‐105‐nt transcript has internal translation activity albeit relatively weaker than c‐myc.

To further determine whether the truncated form of huORFchop‐81‐105‐nt derived from Endouc‐cleaved huORFchop also contains internal translation activity, we engineered the 81‐105‐nt segment of huORFchop where the cleavage site of Endouc was at 80G‐81U. We then co‐transfected endouc mRNA with bicistronic RNA fragment containing huORFchop‐69‐105‐nt and huORFchop‐81‐105‐nt into HEK293T cells. As shown in Appendix Fig S8, the Fluc/Rluc ratio in cells transfected with endouc mRNA was slightly increased from 3.4‐ to 5.5‐fold in RF‐huORF69‐105 mRNA (Appendix Fig S8A and B). The ratio was dramatically elevated from 43‐ to 226‐fold in hpRF‐huORF69‐105 mRNA compared to that of control hpRF mRNA (Appendix Fig S8C and D). Surprisingly, the internal translation activity driven by huORFchop‐81‐105‐nt was completely lost suggesting that the truncated Endouc‐cut huORFchop transcript from the T81 cleavage site to the end of the huORFchop transcript (i.e., huORFchop‐81‐105‐nt) does not contain internal translation activity.

The internal translation activity possessed by huORFchop‐69‐105‐nt transcript is increased in the presence of EndoucTo investigate whether Endouc increases internal translation‐dependent translation of the huORFchop transcript, we co‐transfected endouc mRNA combined with bicistronic RNA containing huORFchop‐69‐105‐nt transcript into HEK293T cells whereas co‐transfection of gfp mRNA and huORFchop‐69‐105‐nt transcript served as a negative control. In the pRF bicistronic reporter system, overexpression of Endouc could increase the translation of DCS of huORFchop‐69‐105‐nt inserted in the intercistronic region (RF‐huORF69‐105 transcript) because the Fluc/Rluc ratio was raised from 3.9 to 5.3 (Fig 9A). Moreover, overexpression of Endouc could dramatically increase the translation of the DCS reporter within huORFchop‐69‐105‐nt transcript (hpRF‐huORF69‐105) in the phpRF bicistronic reporter system in which a hairpin structure inhibits the cap‐dependent translation (Rluc). This is because Fluc/Rluc ratio was elevated from 41‐ to 219‐fold compared to that of hpRF control mRNA (Fig 9C). Again, the Rluc activity showed no difference among groups (Fig 9B and D) indicating that the change of Fluc/Rluc ratio was completely dependent on the change of Fluc activity. In contrast, if the 80G‐81U site in the huORFchop‐69‐105‐nt transcript was mutated (RF‐huORF69‐105‐T81C/hpRF‐huORF69‐105‐T81C), then the internal translation activity was completely lost (Fig 9A and C). We conclude that (i) overexpressed Endouc increases the internal translation activity driven by huORFchop‐69‐105‐nt transcript and (ii) the 5′‐region of uncapped truncated huORFchop‐69‐105‐nt may also contribute to continuous translation as mediated by internal translation initiation activity.Figure 9The 5′ region of the cleaved huORFchop transcript contains internal translation activity
Schematic representation of RF bicistronic transcript is shown above the bar graphs. Rluc: Renilla
luciferase (blank box); Fluc: Firefly
luciferase (solid box). The luc activity of HEK293T cells transfected with bicistronic mRNA, as indicated, in the presence (+) or absence (−) of endouc‐Flag mRNA was determined. The relative luc activity of transfected cells shown on the right was normalized by the amount of the transfected bicistronic transcript shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from control group, normalized as 1. The luc activity was measured by dual‐luciferase assay, and its corresponding bicistronic mRNAs were measured by RT–qPCR. The values were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).The luc activity of Fluc (solid column) versus Rluc (blank column) in HEK293T cells transfected with bicistronic mRNAs shown in A was presented as an increase in fold over that obtained from control group of RF which was normalized as 1. These data suggested that the change of Fluc/Rluc ratio shown in A was entirely dependent on the change of Fluc activity. The data were averaged from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005).Schematic representation of hpRF bicistronic transcript. Rluc: Renilla
luciferase (blank box); Fluc: Firefly
luciferase (solid box). The luc activity of HEK293T cells transfected with bicistronic mRNA, as indicated, in the presence (+) or absence (−) of endouc‐Flag mRNA was determined. The relative luc activity of transfected cells shown on the right was normalized by the amount of the transfected bicistronic transcript shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from control group, normalized as 1. The luc activity was measured by dual‐luciferase assay, and its corresponding bicistronic mRNAs were measured by RT–qPCR. The values were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).The luc activity of Fluc (solid column) versus Rluc (blank column) in HEK293T cells transfected with bicistronic mRNAs shown in C was presented as an increase in fold over that obtained from control group of hpRF which was normalized as 1. These data suggested that the change of Fluc/Rluc ratio shown in C was entirely dependent on the change of Fluc activity. The data were averaged from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).

The internal translation activity possessed by huORFchop‐69‐105‐nt transcript is increased in the presence of Endouc

To investigate whether Endouc increases internal translation‐dependent translation of the huORFchop transcript, we co‐transfected endouc mRNA combined with bicistronic RNA containing huORFchop‐69‐105‐nt transcript into HEK293T cells whereas co‐transfection of gfp mRNA and huORFchop‐69‐105‐nt transcript served as a negative control. In the pRF bicistronic reporter system, overexpression of Endouc could increase the translation of DCS of huORFchop‐69‐105‐nt inserted in the intercistronic region (RF‐huORF69‐105 transcript) because the Fluc/Rluc ratio was raised from 3.9 to 5.3 (Fig 9A). Moreover, overexpression of Endouc could dramatically increase the translation of the DCS reporter within huORFchop‐69‐105‐nt transcript (hpRF‐huORF69‐105) in the phpRF bicistronic reporter system in which a hairpin structure inhibits the cap‐dependent translation (Rluc). This is because Fluc/Rluc ratio was elevated from 41‐ to 219‐fold compared to that of hpRF control mRNA (Fig 9C). Again, the Rluc activity showed no difference among groups (Fig 9B and D) indicating that the change of Fluc/Rluc ratio was completely dependent on the change of Fluc activity. In contrast, if the 80G‐81U site in the huORFchop‐69‐105‐nt transcript was mutated (RF‐huORF69‐105‐T81C/hpRF‐huORF69‐105‐T81C), then the internal translation activity was completely lost (Fig 9A and C). We conclude that (i) overexpressed Endouc increases the internal translation activity driven by huORFchop‐69‐105‐nt transcript and (ii) the 5′‐region of uncapped truncated huORFchop‐69‐105‐nt may also contribute to continuous translation as mediated by internal translation initiation activity.

The 5′ region of the cleaved huORFchop transcript contains internal translation activity

Schematic representation of RF bicistronic transcript is shown above the bar graphs. Rluc: Renilla
luciferase (blank box); Fluc: Firefly
luciferase (solid box). The luc activity of HEK293T cells transfected with bicistronic mRNA, as indicated, in the presence (+) or absence (−) of endouc‐Flag mRNA was determined. The relative luc activity of transfected cells shown on the right was normalized by the amount of the transfected bicistronic transcript shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from control group, normalized as 1. The luc activity was measured by dual‐luciferase assay, and its corresponding bicistronic mRNAs were measured by RT–qPCR. The values were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).The luc activity of Fluc (solid column) versus Rluc (blank column) in HEK293T cells transfected with bicistronic mRNAs shown in A was presented as an increase in fold over that obtained from control group of RF which was normalized as 1. These data suggested that the change of Fluc/Rluc ratio shown in A was entirely dependent on the change of Fluc activity. The data were averaged from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005).Schematic representation of hpRF bicistronic transcript. Rluc: Renilla
luciferase (blank box); Fluc: Firefly
luciferase (solid box). The luc activity of HEK293T cells transfected with bicistronic mRNA, as indicated, in the presence (+) or absence (−) of endouc‐Flag mRNA was determined. The relative luc activity of transfected cells shown on the right was normalized by the amount of the transfected bicistronic transcript shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from control group, normalized as 1. The luc activity was measured by dual‐luciferase assay, and its corresponding bicistronic mRNAs were measured by RT–qPCR. The values were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).The luc activity of Fluc (solid column) versus Rluc (blank column) in HEK293T cells transfected with bicistronic mRNAs shown in C was presented as an increase in fold over that obtained from control group of hpRF which was normalized as 1. These data suggested that the change of Fluc/Rluc ratio shown in C was entirely dependent on the change of Fluc activity. The data were averaged from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).

Schematic representation of RF bicistronic transcript is shown above the bar graphs. Rluc: Renilla
luciferase (blank box); Fluc: Firefly
luciferase (solid box). The luc activity of HEK293T cells transfected with bicistronic mRNA, as indicated, in the presence (+) or absence (−) of endouc‐Flag mRNA was determined. The relative luc activity of transfected cells shown on the right was normalized by the amount of the transfected bicistronic transcript shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from control group, normalized as 1. The luc activity was measured by dual‐luciferase assay, and its corresponding bicistronic mRNAs were measured by RT–qPCR. The values were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).

The luc activity of Fluc (solid column) versus Rluc (blank column) in HEK293T cells transfected with bicistronic mRNAs shown in A was presented as an increase in fold over that obtained from control group of RF which was normalized as 1. These data suggested that the change of Fluc/Rluc ratio shown in A was entirely dependent on the change of Fluc activity. The data were averaged from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005).

Schematic representation of hpRF bicistronic transcript. Rluc: Renilla
luciferase (blank box); Fluc: Firefly
luciferase (solid box). The luc activity of HEK293T cells transfected with bicistronic mRNA, as indicated, in the presence (+) or absence (−) of endouc‐Flag mRNA was determined. The relative luc activity of transfected cells shown on the right was normalized by the amount of the transfected bicistronic transcript shown on the left. The relative luc activity was represented by the fold increase of Fluc/Rluc ratio over that obtained from control group, normalized as 1. The luc activity was measured by dual‐luciferase assay, and its corresponding bicistronic mRNAs were measured by RT–qPCR. The values were calculated from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).

The luc activity of Fluc (solid column) versus Rluc (blank column) in HEK293T cells transfected with bicistronic mRNAs shown in C was presented as an increase in fold over that obtained from control group of hpRF which was normalized as 1. These data suggested that the change of Fluc/Rluc ratio shown in C was entirely dependent on the change of Fluc activity. The data were averaged from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (***P < 0.001).

The huORFchop‐69‐105‐nt transcript possesses an IRES activityWe used the pRF/phpRF bicistronic system described above to demonstrate that the huORFchop‐69‐105‐nt transcript, but not the huORFchop‐81‐105‐nt transcript, contains internal translation activity. Next, we determined whether this huORFchop‐69‐105‐nt transcript possesses an IRES activity. To make this determination, we performed an in vitro system to transcribe polyA‐tailed Fluc mRNA fused after huORFchop‐69‐105‐nt transcript which was capped with a normal m7Gppp (a 5′‐capped form) or a non‐functional ApppG (a 5′ A‐capped form) at the 5′ end. However, the m7GpppG‐capped mRNAs are translated in both a cap‐dependent and a cap‐independent manner, while A‐capped mRNAs are only translated in a cap‐independent manner. Therefore, to make the distinction between cap‐dependence and cap‐independence, we compared the translation efficiency between m7Gppp‐ and A‐capped mRNAs. To do this, we next used A‐capped cmyc IRES transcripts fused at the upstream of a Fluc reporter to serve as a positive control. The A‐capped 105 nt of luc cDNA (Non 105) served as a negative control. After a two‐hr transfection of these transcripts in HEK293T cells, we found that the translation efficiency of A‐capped cmyc‐Fluc mRNA only reached approximately 15% of that driven by m7GpppG‐capped cmyc‐Fluc mRNA (Fig 10A). As expected, the reporter activity driven by A‐capped Fluc mRNA (Fig 10B) and A‐capped Non105 (Fig 10C) was very low (~1.5%). The A‐capped full‐length huORFchop‐1‐105‐nt reporter transcript (A‐capped huORF1‐105Fluc) exhibited only 1.5% of that driven by m7GpppG‐capped huORF1‐105Fluc (Fig 10D) suggesting that full‐length huORFchop‐1‐105‐nt exhibited no IRES activity. We also found that the huORFchop‐81‐105‐nt transcript (A‐capped huORF81‐105Fluc) showed no IRES activity (Fig 10F). Interestingly, the A‐capped huORF69‐105Fluc RNA fragment exhibited 4.3% reporter activity compared to m7GpppG‐capped huORF69‐105Fluc RNA (Fig 10E) suggesting that the uncapped huORFchop‐69‐105‐nt transcript does have IRES activity.Figure 10Comparison between cap‐dependent and IRES‐dependent translation of the huORFchop transcript containing reporter
A–FSchematic representation of capped (m7G‐capped) and A‐capped (non‐functional capped) monocistronic mRNAs with poly(A)‐tail‐encoded firefly luciferase (Fluc) and fused upstream with (A) c‐myc IRES, (B) no insertion, (C) non‐specific 105‐nt luc RNA (Non105), (D) huORFchop‐1‐105‐nt, (E) huORFchop‐69‐105‐nt, and (F) huORFchop‐81‐105‐nt as shown above each bar graph. The RNA transfection assay of each transcript equipped with m7G‐capped or A‐capped monocistronic mRNA in HEK293T cells was compared. To normalize the luc activities of capped and A‐capped Fluc transcripts, the m7G‐capped Renilla luc (Rluc) mRNA was used as an internal control. The luc activity of cells transfected with cap‐monocistronic mRNA in each graph was set as 100%.GThe luc activity driven by the indicated A‐capped monocistronic mRNA in the presence of Endouc or EndoucK242A in HEK293T cells. The m7G‐capped Rluc mRNA was used as an internal control to normalize the activities of A‐cap Fluc transcripts. The egfp mRNA was used as a negative control, and the luc activity of cells transfected with egfp mRNA and A‐cap Fluc mRNA containing full‐length huORFchop‐1‐105‐nt was set to 1. The data were averaged from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).
To further analyze whether the presence of Endouc impacts huORFchop‐69‐105‐nt IRES activity, we studied the luc activity of A‐capped huORF69‐105Fluc transcript in the presence of Endouc or EndoucK242A in HEK293T cells. The results showed that the luc activity of A‐capped monocistronic mRNAs containing full‐length huORFchop and huORFchop‐69‐105‐nt transcripts was induced in the presence of Endouc, but not in the presence of endoribonuclease‐deficient mutant EndoucK242A (Fig 10G) suggesting that huORFchop IRES activity induced by Endouc is mediated through its endoribonuclease activity. Interestingly, the A‐capped mRNA containing huORFchop‐81‐105‐nt failed to induce luc activity (Fig 10G) suggesting that the truncated uncapped huORFchop‐81‐105‐nt transcript generated from the cleavage of Endouc/‐ENDOU‐1 on the full‐length of huORFchop‐1‐105‐nt transcript at 80G‐81U does not possess IRES activity unlike the huORFchop‐69‐105‐nt transcript.To further analyze whether the presence of Endouc impacts the huORFchop‐69‐105‐nt IRES activity, we studied the luc activity of ApppG‐capped huORF69‐105 Luc transcript in the presence of Endouc or EndoucK242A in HEK293T cells. The results showed that the luc activity of ApppG‐capped monocistronic mRNAs containing full‐length huORFchop and huORFchop‐69‐105‐nt transcripts was induced in the presence of Endouc but not in the presence of endoribonuclease‐deficient mutant EndoucK242A (Fig 10G) suggesting that the huORFchop IRES activity induced by Endouc is mediated through its endoribonuclease activity. Interestingly, the ApppG‐capped mRNA containing huORFchop‐81‐105‐nt transcript failed to induce luc activity (Fig 10G). Additionally, in the absence of Endouc, only huORF69‐105, but not full‐length huORFchop, displayed weak IRES activity (Fig 10G). Strikingly, however, when Endouc was overexpressed in cells, both huORFchop‐69‐105‐nt and full‐length huORFchop transcripts displayed IRES activity (Fig 10G).Following the logic of this line of collective evidence, we suggest that an inhibitory element could be present in huORFchop‐1‐68‐nt causing the repression of IRES activity. In clear contrast, when Endouc is overexpressed in cells, the inhibitory structure of huORFchop transcript is disrupted via endoribonuclease, likely altering the inhibitory element presented in huORFchop‐1‐68‐nt but promoting the IRES activity driven by huORFchop‐69‐105‐nt to then facilitate the translation of CHOP DCS.

The huORFchop‐69‐105‐nt transcript possesses an IRES activity

We used the pRF/phpRF bicistronic system described above to demonstrate that the huORFchop‐69‐105‐nt transcript, but not the huORFchop‐81‐105‐nt transcript, contains internal translation activity. Next, we determined whether this huORFchop‐69‐105‐nt transcript possesses an IRES activity. To make this determination, we performed an in vitro system to transcribe polyA‐tailed Fluc mRNA fused after huORFchop‐69‐105‐nt transcript which was capped with a normal m7Gppp (a 5′‐capped form) or a non‐functional ApppG (a 5′ A‐capped form) at the 5′ end. However, the m7GpppG‐capped mRNAs are translated in both a cap‐dependent and a cap‐independent manner, while A‐capped mRNAs are only translated in a cap‐independent manner. Therefore, to make the distinction between cap‐dependence and cap‐independence, we compared the translation efficiency between m7Gppp‐ and A‐capped mRNAs. To do this, we next used A‐capped cmyc IRES transcripts fused at the upstream of a Fluc reporter to serve as a positive control. The A‐capped 105 nt of luc cDNA (Non 105) served as a negative control. After a two‐hr transfection of these transcripts in HEK293T cells, we found that the translation efficiency of A‐capped cmyc‐Fluc mRNA only reached approximately 15% of that driven by m7GpppG‐capped cmyc‐Fluc mRNA (Fig 10A). As expected, the reporter activity driven by A‐capped Fluc mRNA (Fig 10B) and A‐capped Non105 (Fig 10C) was very low (~1.5%). The A‐capped full‐length huORFchop‐1‐105‐nt reporter transcript (A‐capped huORF1‐105Fluc) exhibited only 1.5% of that driven by m7GpppG‐capped huORF1‐105Fluc (Fig 10D) suggesting that full‐length huORFchop‐1‐105‐nt exhibited no IRES activity. We also found that the huORFchop‐81‐105‐nt transcript (A‐capped huORF81‐105Fluc) showed no IRES activity (Fig 10F). Interestingly, the A‐capped huORF69‐105Fluc RNA fragment exhibited 4.3% reporter activity compared to m7GpppG‐capped huORF69‐105Fluc RNA (Fig 10E) suggesting that the uncapped huORFchop‐69‐105‐nt transcript does have IRES activity.

Comparison between cap‐dependent and IRES‐dependent translation of the huORFchop transcript containing reporter

A–FSchematic representation of capped (m7G‐capped) and A‐capped (non‐functional capped) monocistronic mRNAs with poly(A)‐tail‐encoded firefly luciferase (Fluc) and fused upstream with (A) c‐myc IRES, (B) no insertion, (C) non‐specific 105‐nt luc RNA (Non105), (D) huORFchop‐1‐105‐nt, (E) huORFchop‐69‐105‐nt, and (F) huORFchop‐81‐105‐nt as shown above each bar graph. The RNA transfection assay of each transcript equipped with m7G‐capped or A‐capped monocistronic mRNA in HEK293T cells was compared. To normalize the luc activities of capped and A‐capped Fluc transcripts, the m7G‐capped Renilla luc (Rluc) mRNA was used as an internal control. The luc activity of cells transfected with cap‐monocistronic mRNA in each graph was set as 100%.GThe luc activity driven by the indicated A‐capped monocistronic mRNA in the presence of Endouc or EndoucK242A in HEK293T cells. The m7G‐capped Rluc mRNA was used as an internal control to normalize the activities of A‐cap Fluc transcripts. The egfp mRNA was used as a negative control, and the luc activity of cells transfected with egfp mRNA and A‐cap Fluc mRNA containing full‐length huORFchop‐1‐105‐nt was set to 1. The data were averaged from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).

Schematic representation of capped (m7G‐capped) and A‐capped (non‐functional capped) monocistronic mRNAs with poly(A)‐tail‐encoded firefly luciferase (Fluc) and fused upstream with (A) c‐myc IRES, (B) no insertion, (C) non‐specific 105‐nt luc RNA (Non105), (D) huORFchop‐1‐105‐nt, (E) huORFchop‐69‐105‐nt, and (F) huORFchop‐81‐105‐nt as shown above each bar graph. The RNA transfection assay of each transcript equipped with m7G‐capped or A‐capped monocistronic mRNA in HEK293T cells was compared. To normalize the luc activities of capped and A‐capped Fluc transcripts, the m7G‐capped Renilla luc (Rluc) mRNA was used as an internal control. The luc activity of cells transfected with cap‐monocistronic mRNA in each graph was set as 100%.

The luc activity driven by the indicated A‐capped monocistronic mRNA in the presence of Endouc or EndoucK242A in HEK293T cells. The m7G‐capped Rluc mRNA was used as an internal control to normalize the activities of A‐cap Fluc transcripts. The egfp mRNA was used as a negative control, and the luc activity of cells transfected with egfp mRNA and A‐cap Fluc mRNA containing full‐length huORFchop‐1‐105‐nt was set to 1. The data were averaged from three independent experiments and presented as mean ± SD (n = 3). Student's t‐test was used to determine significant differences between each group (**P < 0.005, ***P < 0.001).

To further analyze whether the presence of Endouc impacts huORFchop‐69‐105‐nt IRES activity, we studied the luc activity of A‐capped huORF69‐105Fluc transcript in the presence of Endouc or EndoucK242A in HEK293T cells. The results showed that the luc activity of A‐capped monocistronic mRNAs containing full‐length huORFchop and huORFchop‐69‐105‐nt transcripts was induced in the presence of Endouc, but not in the presence of endoribonuclease‐deficient mutant EndoucK242A (Fig 10G) suggesting that huORFchop IRES activity induced by Endouc is mediated through its endoribonuclease activity. Interestingly, the A‐capped mRNA containing huORFchop‐81‐105‐nt failed to induce luc activity (Fig 10G) suggesting that the truncated uncapped huORFchop‐81‐105‐nt transcript generated from the cleavage of Endouc/‐ENDOU‐1 on the full‐length of huORFchop‐1‐105‐nt transcript at 80G‐81U does not possess IRES activity unlike the huORFchop‐69‐105‐nt transcript.

To further analyze whether the presence of Endouc impacts the huORFchop‐69‐105‐nt IRES activity, we studied the luc activity of ApppG‐capped huORF69‐105 Luc transcript in the presence of Endouc or EndoucK242A in HEK293T cells. The results showed that the luc activity of ApppG‐capped monocistronic mRNAs containing full‐length huORFchop and huORFchop‐69‐105‐nt transcripts was induced in the presence of Endouc but not in the presence of endoribonuclease‐deficient mutant EndoucK242A (Fig 10G) suggesting that the huORFchop IRES activity induced by Endouc is mediated through its endoribonuclease activity. Interestingly, the ApppG‐capped mRNA containing huORFchop‐81‐105‐nt transcript failed to induce luc activity (Fig 10G). Additionally, in the absence of Endouc, only huORF69‐105, but not full‐length huORFchop, displayed weak IRES activity (Fig 10G). Strikingly, however, when Endouc was overexpressed in cells, both huORFchop‐69‐105‐nt and full‐length huORFchop transcripts displayed IRES activity (Fig 10G).

Following the logic of this line of collective evidence, we suggest that an inhibitory element could be present in huORFchop‐1‐68‐nt causing the repression of IRES activity. In clear contrast, when Endouc is overexpressed in cells, the inhibitory structure of huORFchop transcript is disrupted via endoribonuclease, likely altering the inhibitory element presented in huORFchop‐1‐68‐nt but promoting the IRES activity driven by huORFchop‐69‐105‐nt to then facilitate the translation of CHOP DCS.

DiscussionIn this study, we found an RNA‐binding endoribonuclease Endouc/ENDOU‐1 protein involved in mediating CHOP translation through suppression of the inhibitory huORFchop transcript during ER stress. We made several discoveries during the course of our investigation. (i) Endouc/ENDOU‐1 is a stress‐responsive protein that can rapidly increase in the presence of such stressors as heat‐shock and hypoxia. (ii) Endouc/ENDOU‐1 induces huORFchop‐tag reporter and CHOP translation in an endoribonuclease‐dependent manner. (iii) While the level of p‐eIF2α is increased during CHOP translation, CHOP induced by Endouc/ENDOU‐1 is independent of p‐eIF2α since Endouc/ENDOU‐1 alone causes CHOP be partially translated. (iv) However, Endouc/ENDOU‐1 and p‐eIF2α are under the control of a feedback loop that enforces cooperativity in the translation of CHOP DCS within huORFchop transcript to reach a maximal level. (v) Endouc/ENDOU‐1 can directly bind the huORFchop transcript and cleave it at the 80G‐81U site. (vi) Overexpression of Endouc promotes the shifting of huORFchop‐tagged transcripts from monosome fractions toward polysome fractions. (vii) Unlike the huORFchop‐81‐105‐nt transcript, the truncated uncapped huORFchop‐69‐105‐nt transcript possesses an IRES activity. We began our investigation by questioning the involvement of regulatory proteins in suppressing huORFchop‐TI. Based on the results above, we determined that Endouc/ENDOU‐1 plays a distinctly positive role in post‐transcriptional regulation of the human CHOP gene.Within the huORFchop‐69‐105‐nt transcript, the core sequence (72–91 nts) that contains the Endouc recognition site is highly conserved among different species (Young et al, 2016). However, the polypeptide encoded by huORFchop RNA, especially the carboxyl‐terminal portion, is important for forming the inhibitory function of huORFchop (Jousse et al, 2001; Palam et al, 2011). Young et al (2016) proposed that an IFI sequence at amino acid 28–30 encoded by the 81–90 nts of huORFchop transcript causes ribosomes to stall at the upstream sequence of AUG, which, in turn, prevents translation of the DCS. Unexpectedly, however, they found that this ribosomal stall mediated by the IFI sequence is not regulated in a stress‐dependent manner. Thus, the mechanism that allows stalled ribosomes to reinitiate translation at the DCS of huORFchop transcript during ER stress is still unclear.Nevertheless, our study explains how Endouc/ENDOU‐1 participates in post‐translational control mediated by huORFchop transcript, and we propose a simple potential mechanistic model to illustrate the role of Endouc/ENDOU‐1 in huORFchop‐mediated translational control (Fig 11). The 72–91 nts within the full‐length of huORFchop‐1‐105‐nt transcript in which the IFI sequence and 80G‐81U are included constitute an important segment required for huORFchop transcript to form a unique hairpin structure. We learned that an inhibitory conformation occurs and that ribosomes stall at 81–90 nts of huORFchop transcript (Young et al, 2016) when cells are in the normal condition. Thus, the translation of DCS within the huORFchop transcript is blocked by the huORFchop‐TI due to the low levels of Endouc/ENDOU‐1 and p‐eIF2α. However, Endouc/ENDOU‐1 is quickly induced when cells encounter stress, bind directly to the hairpin structure of huORFchop transcript formed by its 72–91‐nt sequences, and cleave it at the 80G‐81U site. This process blocks huORFchop‐TI by the conformational change of the inhibitory huORFchop structure, in turn, releasing the stalled ribosomes to scan through the huORFchop transcript and reinitiate at AUG of the main DCS within huORFchop transcript. This activity constitutes continuous translation of CHOP mRNA via IRES activity driven by the truncated uncapped form of huORFchop‐69‐105‐nt and by the increase of p‐eIF2α induced by Endouc/ENDOU‐1.Figure 11Schematic representation illustrating how Endouc/ENDOU acts as a positive modulator of human CHOP mRNA translation by suppressing the inhibitory huORFchop transcriptDuring normal conditions, the translation of CHOP protein is blocked by huORFchop‐TI due to ribosomes stalled at 81–90 nts within the 105‐nt huORFchop transcript. However, Endouc/ENDOU is quickly produced when cells encounter environmental stress, e.g., heat‐shock and hypoxia as in the present work. It binds to the hairpin structure of huORFchop transcript formed at 72–91 nts and cleaves it at the 80G‐81U site. This endonuclease activity changes the conformation of inhibitory structure formed by huORFchop transcript, in turn, releasing the stalled ribosomes to scan through the huORFchop transcript and reinitiate at AUG of the main downstream coding sequence (DCS) of huORFchop transcript. Finally, CHOP transcript is continuously translated via the IRES activity driven by huORFchop−69‐105‐nt and the increase of p‐eIF2α induced by Endouc/ENDOU.Using the pRF/phpRF bicistronic system, we demonstrated that the huORFchop‐69‐105‐nt transcript, but not the huORFchop‐81‐105‐nt transcript, contains internal translation activity. Furthermore, by comparing the translation efficiency between m7Gppp‐ and A‐capped mRNAs, we also demonstrated that the uncapped truncated huORFchop‐69‐105‐nt transcript possesses an IRES activity. It is known that RNA structure determines the function of IRES elements (Martinez‐Salas et al., 2018), and thus, the RNA structure of huORFchop‐69‐105‐nt may recruit ribosomes and initiation factors for DCS translation. Meanwhile, polysomal profiling analysis demonstrated that Endouc/ENDOU‐1 associates with ribosomal proteins in translational fractions suggesting that Endouc/ENDOU‐1 also plays a role in recruiting other factors that are necessary for DCS translation processes. On the other hand, we found that the huORFchop‐69‐105‐nt, but not full‐length huORFchop, displayed weak IRES activity in the absence of Endouc/ENDOU‐1. However, both huORFchop‐69‐105‐nt and full‐length huORFchop transcripts displayed an IRES activity in Endouc/ENDOU‐1‐overexpressed cells (Fig 10G) suggesting an inhibitory function of huORFchop‐1‐68‐nt able to repress the IRES activity driven by huORFchop‐69‐105‐nt.Therefore, we propose that the cleavage of Endouc/ENDOU‐1 on the huORFchop transcript accomplishes the following tasks: (i) disrupt the inhibitory structure formed by the 72‐91‐nt sequence of huORFchop transcript, (ii) facilitate RNA conformational changes to alter the inhibitory element of huORFchop‐1‐68‐nt, and (iii) strengthen the IRES activity driven by huORFchop‐69‐105‐nt. When 80G‐81U sequences were mutated, the translation of DCS was not enhanced by the increase of Endouc/ENDOU‐1. This observation strengthens our hypothesis that the disruption of inhibitory conformation by the binding and cleavage of Endouc/ENDOU‐1 is required to reinitiate the translation of the CHOP gene under stress conditions. While beyond the scope of the present paper, additional factors involved in the interaction among Endouc/ENDOU‐1, huORFchop‐1‐68‐nt, and huORFchop‐69‐105‐nt transcripts are worthy of further investigation.The translation initiation factor eIF2 can mitigate cellular injury or, alternatively, can induce apoptosis when cells encounter environmental stresses. During stress, the alpha subunit of eIF2 at S51 can be phosphorylated by a number of related protein kinases. Phosphorylation of eIF2 reduces global translation allowing cells to conserve resources and initiate a reconfiguration of gene expression to effectively manage stress. On the other hand, eIF2 phosphorylation can also induce the translation of specific mRNAs such as ATF4 and CHOP (Harding et al, 2000; Lu et al, 2004; Vattem & Wek, 2004; Palam et al, 2011). Palam et al (2011), proposed that enhanced p‐eIF2 during stress facilitates ribosomal bypass of inhibitory huORFchop elements for continued DCS translation. These results suggest that the increase of p‐eIF2α as mediated by Endouc/ENDOU‐1 can promote a cooperativity required to overcome the inhibitory huORFchop element under stress.The results demonstrated that both Endouc and p‐eIF2α are required for CHOP translation; however, p‐eIF2α is not the only protein able to facilitate CHOP translation to reach its maximal level during stress. This conclusion is supported by other researchers. For example, the level of CHOP remained low when Dey et al (2010) used UV irradiation to elicit p‐eIF2α expression. Similarly, Ghosh et al (2011) reported that the absence of Sirtuin1 in cells resulted in the elevated and prolonged phosphorylation of eIF2α but reduced and delayed the induction of the downstream CHOP protein. Moreover, cells treated with Salubrinal—a drug which acts as a specific inhibitor of eIF2α phosphatase enzymes and prevents dephosphorylation of eIF2α during stress condition—led to an increase of p‐eIF2α expression but caused a decrease in CHOP expression (Luo et al, 2003; Oyadomari & Mori, 2004; Duan et al, 2014). Tyagi et al (2015) reported that a ubiquitous small GTPase, Rheb, could enhance p‐eIF2α and increase ATF4 level. However, their downstream CHOP was not affected. Collectively, this line of evidence leads to the conclusion that trans‐acting factors other than p‐eIF2α might be involved in suppressing the inhibitory effect of huORFchop elements under stress. Indeed, this research was undertaken to identify putative proteins other than known ribosomal proteins such as eIF2α that might be directly involved in suppressing huORFchop‐TI under stress.Accordingly, after performing microarray analysis of cells between suppressing and non‐suppressing huORFchop elements under stress in an in vivo system, we identified an RNA‐binding protein, Endouc/ENDOU‐1, that appears to play a key role in facilitating CHOP translation through the suppression of huORFchop‐TI under stress stimuli. Overexpression of Endouc/ENDOU‐1 can induce eIF2α phosphorylation through the PKR pathway. PKR is a double‐stranded RNA (dsRNA)‐dependent protein kinase that plays multiple roles in response to different stress situations such as virus infection (Dabo & Meurs, 2012). The interaction between PKR and dsRNA is independent of any specific RNA nucleotide motif or sequences (Minks et al, 1979). Moreover, about 30–80 nt of dsRNA is enough to activate PKR activity (Manche et al, 1992; Nanduri et al, 1998). Thus, we speculate that the Endouc‐cleaved huORFchop transcript may form a dsRNA that contributes to PKR activation, but, again, further elucidation of this topic is outside the scope of this work yet worthy of further investigation.Many stress factors contain a uORF cassette on their 5′UTR to control translation in response to environmental stresses (Hinnebusch, 2014). ATF4 is one of the key stress factors, and its mRNA contains a uORF cassette on the 5′UTR (Harding et al, 2000). Although we did not examine the translational effects of Endouc on huORFATF4 using luciferase assay, Western blot analysis showed that the protein level of ATF4 did not change when cells overexpressed either Endouc or ENDOU‐1 suggesting that Endouc does not specifically interact with huORFATF4 (Fig EV4D).Meantime, the stress factor GADD34 also contains an inhibitory uORF sequence and could be induced by Endouc/ENDOU‐1 (Fig EV4D). However, the increase of GADD34 protein is indirectly controlled by Endouc/ENDOU‐1 because the translation of GADD34 is p‐eIF2α‐dependent, and p‐eIF2α can be partially induced by Endouc/ENDOU‐1. Therefore, although the transcripts of CHOP, GADD34, and ATF4 contain inhibitory uORF segments to mediate the translation of DCS during stress, the regulatory mechanisms among these three genes are quite distinct. Our data demonstrate that Endouc/ENDOU‐1 is selective for the uORF segment of the human chop transcript rather than the uORF segments of other genes. We propose that this might result from the unique hairpin structure of huORFchop formed by its 71–91‐nt sequence that enables Endouc/ENDOU‐1 to specifically interact with and cleave huORFchop transcript thus repressing the translation inhibition mediated by huORFchop. However, it is very possible that Endouc might be specific for other targets as well. This will also be the subject of further study.

In this study, we found an RNA‐binding endoribonuclease Endouc/ENDOU‐1 protein involved in mediating CHOP translation through suppression of the inhibitory huORFchop transcript during ER stress. We made several discoveries during the course of our investigation. (i) Endouc/ENDOU‐1 is a stress‐responsive protein that can rapidly increase in the presence of such stressors as heat‐shock and hypoxia. (ii) Endouc/ENDOU‐1 induces huORFchop‐tag reporter and CHOP translation in an endoribonuclease‐dependent manner. (iii) While the level of p‐eIF2α is increased during CHOP translation, CHOP induced by Endouc/ENDOU‐1 is independent of p‐eIF2α since Endouc/ENDOU‐1 alone causes CHOP be partially translated. (iv) However, Endouc/ENDOU‐1 and p‐eIF2α are under the control of a feedback loop that enforces cooperativity in the translation of CHOP DCS within huORFchop transcript to reach a maximal level. (v) Endouc/ENDOU‐1 can directly bind the huORFchop transcript and cleave it at the 80G‐81U site. (vi) Overexpression of Endouc promotes the shifting of huORFchop‐tagged transcripts from monosome fractions toward polysome fractions. (vii) Unlike the huORFchop‐81‐105‐nt transcript, the truncated uncapped huORFchop‐69‐105‐nt transcript possesses an IRES activity. We began our investigation by questioning the involvement of regulatory proteins in suppressing huORFchop‐TI. Based on the results above, we determined that Endouc/ENDOU‐1 plays a distinctly positive role in post‐transcriptional regulation of the human CHOP gene.

Within the huORFchop‐69‐105‐nt transcript, the core sequence (72–91 nts) that contains the Endouc recognition site is highly conserved among different species (Young et al, 2016). However, the polypeptide encoded by huORFchop RNA, especially the carboxyl‐terminal portion, is important for forming the inhibitory function of huORFchop (Jousse et al, 2001; Palam et al, 2011). Young et al (2016) proposed that an IFI sequence at amino acid 28–30 encoded by the 81–90 nts of huORFchop transcript causes ribosomes to stall at the upstream sequence of AUG, which, in turn, prevents translation of the DCS. Unexpectedly, however, they found that this ribosomal stall mediated by the IFI sequence is not regulated in a stress‐dependent manner. Thus, the mechanism that allows stalled ribosomes to reinitiate translation at the DCS of huORFchop transcript during ER stress is still unclear.

Nevertheless, our study explains how Endouc/ENDOU‐1 participates in post‐translational control mediated by huORFchop transcript, and we propose a simple potential mechanistic model to illustrate the role of Endouc/ENDOU‐1 in huORFchop‐mediated translational control (Fig 11). The 72–91 nts within the full‐length of huORFchop‐1‐105‐nt transcript in which the IFI sequence and 80G‐81U are included constitute an important segment required for huORFchop transcript to form a unique hairpin structure. We learned that an inhibitory conformation occurs and that ribosomes stall at 81–90 nts of huORFchop transcript (Young et al, 2016) when cells are in the normal condition. Thus, the translation of DCS within the huORFchop transcript is blocked by the huORFchop‐TI due to the low levels of Endouc/ENDOU‐1 and p‐eIF2α. However, Endouc/ENDOU‐1 is quickly induced when cells encounter stress, bind directly to the hairpin structure of huORFchop transcript formed by its 72–91‐nt sequences, and cleave it at the 80G‐81U site. This process blocks huORFchop‐TI by the conformational change of the inhibitory huORFchop structure, in turn, releasing the stalled ribosomes to scan through the huORFchop transcript and reinitiate at AUG of the main DCS within huORFchop transcript. This activity constitutes continuous translation of CHOP mRNA via IRES activity driven by the truncated uncapped form of huORFchop‐69‐105‐nt and by the increase of p‐eIF2α induced by Endouc/ENDOU‐1.

Schematic representation illustrating how Endouc/ENDOU acts as a positive modulator of human CHOP mRNA translation by suppressing the inhibitory huORFchop transcript

During normal conditions, the translation of CHOP protein is blocked by huORFchop‐TI due to ribosomes stalled at 81–90 nts within the 105‐nt huORFchop transcript. However, Endouc/ENDOU is quickly produced when cells encounter environmental stress, e.g., heat‐shock and hypoxia as in the present work. It binds to the hairpin structure of huORFchop transcript formed at 72–91 nts and cleaves it at the 80G‐81U site. This endonuclease activity changes the conformation of inhibitory structure formed by huORFchop transcript, in turn, releasing the stalled ribosomes to scan through the huORFchop transcript and reinitiate at AUG of the main downstream coding sequence (DCS) of huORFchop transcript. Finally, CHOP transcript is continuously translated via the IRES activity driven by huORFchop−69‐105‐nt and the increase of p‐eIF2α induced by Endouc/ENDOU.

Using the pRF/phpRF bicistronic system, we demonstrated that the huORFchop‐69‐105‐nt transcript, but not the huORFchop‐81‐105‐nt transcript, contains internal translation activity. Furthermore, by comparing the translation efficiency between m7Gppp‐ and A‐capped mRNAs, we also demonstrated that the uncapped truncated huORFchop‐69‐105‐nt transcript possesses an IRES activity. It is known that RNA structure determines the function of IRES elements (Martinez‐Salas et al., 2018), and thus, the RNA structure of huORFchop‐69‐105‐nt may recruit ribosomes and initiation factors for DCS translation. Meanwhile, polysomal profiling analysis demonstrated that Endouc/ENDOU‐1 associates with ribosomal proteins in translational fractions suggesting that Endouc/ENDOU‐1 also plays a role in recruiting other factors that are necessary for DCS translation processes. On the other hand, we found that the huORFchop‐69‐105‐nt, but not full‐length huORFchop, displayed weak IRES activity in the absence of Endouc/ENDOU‐1. However, both huORFchop‐69‐105‐nt and full‐length huORFchop transcripts displayed an IRES activity in Endouc/ENDOU‐1‐overexpressed cells (Fig 10G) suggesting an inhibitory function of huORFchop‐1‐68‐nt able to repress the IRES activity driven by huORFchop‐69‐105‐nt.

Therefore, we propose that the cleavage of Endouc/ENDOU‐1 on the huORFchop transcript accomplishes the following tasks: (i) disrupt the inhibitory structure formed by the 72‐91‐nt sequence of huORFchop transcript, (ii) facilitate RNA conformational changes to alter the inhibitory element of huORFchop‐1‐68‐nt, and (iii) strengthen the IRES activity driven by huORFchop‐69‐105‐nt. When 80G‐81U sequences were mutated, the translation of DCS was not enhanced by the increase of Endouc/ENDOU‐1. This observation strengthens our hypothesis that the disruption of inhibitory conformation by the binding and cleavage of Endouc/ENDOU‐1 is required to reinitiate the translation of the CHOP gene under stress conditions. While beyond the scope of the present paper, additional factors involved in the interaction among Endouc/ENDOU‐1, huORFchop‐1‐68‐nt, and huORFchop‐69‐105‐nt transcripts are worthy of further investigation.

The translation initiation factor eIF2 can mitigate cellular injury or, alternatively, can induce apoptosis when cells encounter environmental stresses. During stress, the alpha subunit of eIF2 at S51 can be phosphorylated by a number of related protein kinases. Phosphorylation of eIF2 reduces global translation allowing cells to conserve resources and initiate a reconfiguration of gene expression to effectively manage stress. On the other hand, eIF2 phosphorylation can also induce the translation of specific mRNAs such as ATF4 and CHOP (Harding et al, 2000; Lu et al, 2004; Vattem & Wek, 2004; Palam et al, 2011). Palam et al (2011), proposed that enhanced p‐eIF2 during stress facilitates ribosomal bypass of inhibitory huORFchop elements for continued DCS translation. These results suggest that the increase of p‐eIF2α as mediated by Endouc/ENDOU‐1 can promote a cooperativity required to overcome the inhibitory huORFchop element under stress.

The results demonstrated that both Endouc and p‐eIF2α are required for CHOP translation; however, p‐eIF2α is not the only protein able to facilitate CHOP translation to reach its maximal level during stress. This conclusion is supported by other researchers. For example, the level of CHOP remained low when Dey et al (2010) used UV irradiation to elicit p‐eIF2α expression. Similarly, Ghosh et al (2011) reported that the absence of Sirtuin1 in cells resulted in the elevated and prolonged phosphorylation of eIF2α but reduced and delayed the induction of the downstream CHOP protein. Moreover, cells treated with Salubrinal—a drug which acts as a specific inhibitor of eIF2α phosphatase enzymes and prevents dephosphorylation of eIF2α during stress condition—led to an increase of p‐eIF2α expression but caused a decrease in CHOP expression (Luo et al, 2003; Oyadomari & Mori, 2004; Duan et al, 2014). Tyagi et al (2015) reported that a ubiquitous small GTPase, Rheb, could enhance p‐eIF2α and increase ATF4 level. However, their downstream CHOP was not affected. Collectively, this line of evidence leads to the conclusion that trans‐acting factors other than p‐eIF2α might be involved in suppressing the inhibitory effect of huORFchop elements under stress. Indeed, this research was undertaken to identify putative proteins other than known ribosomal proteins such as eIF2α that might be directly involved in suppressing huORFchop‐TI under stress.

Accordingly, after performing microarray analysis of cells between suppressing and non‐suppressing huORFchop elements under stress in an in vivo system, we identified an RNA‐binding protein, Endouc/ENDOU‐1, that appears to play a key role in facilitating CHOP translation through the suppression of huORFchop‐TI under stress stimuli. Overexpression of Endouc/ENDOU‐1 can induce eIF2α phosphorylation through the PKR pathway. PKR is a double‐stranded RNA (dsRNA)‐dependent protein kinase that plays multiple roles in response to different stress situations such as virus infection (Dabo & Meurs, 2012). The interaction between PKR and dsRNA is independent of any specific RNA nucleotide motif or sequences (Minks et al, 1979). Moreover, about 30–80 nt of dsRNA is enough to activate PKR activity (Manche et al, 1992; Nanduri et al, 1998). Thus, we speculate that the Endouc‐cleaved huORFchop transcript may form a dsRNA that contributes to PKR activation, but, again, further elucidation of this topic is outside the scope of this work yet worthy of further investigation.

Many stress factors contain a uORF cassette on their 5′UTR to control translation in response to environmental stresses (Hinnebusch, 2014). ATF4 is one of the key stress factors, and its mRNA contains a uORF cassette on the 5′UTR (Harding et al, 2000). Although we did not examine the translational effects of Endouc on huORFATF4 using luciferase assay, Western blot analysis showed that the protein level of ATF4 did not change when cells overexpressed either Endouc or ENDOU‐1 suggesting that Endouc does not specifically interact with huORFATF4 (Fig EV4D).

Meantime, the stress factor GADD34 also contains an inhibitory uORF sequence and could be induced by Endouc/ENDOU‐1 (Fig EV4D). However, the increase of GADD34 protein is indirectly controlled by Endouc/ENDOU‐1 because the translation of GADD34 is p‐eIF2α‐dependent, and p‐eIF2α can be partially induced by Endouc/ENDOU‐1. Therefore, although the transcripts of CHOP, GADD34, and ATF4 contain inhibitory uORF segments to mediate the translation of DCS during stress, the regulatory mechanisms among these three genes are quite distinct. Our data demonstrate that Endouc/ENDOU‐1 is selective for the uORF segment of the human chop transcript rather than the uORF segments of other genes. We propose that this might result from the unique hairpin structure of huORFchop formed by its 71–91‐nt sequence that enables Endouc/ENDOU‐1 to specifically interact with and cleave huORFchop transcript thus repressing the translation inhibition mediated by huORFchop. However, it is very possible that Endouc might be specific for other targets as well. This will also be the subject of further study.

Materials and MethodsZebrafish husbandry, transgenic line, and signal observationZebrafish AB strain and transgenic line were cultured as previously described (Westerfield, 2000; Lee et al, 2014). Transgenic line huORFZ harbors a DNA fragment containing the huORFchop element fused with the downstream GFP cDNA (huORFchop‐gfp) and driven by a cytomegalovirus promoter (Lee et al, 2011). The translation of transgenic huORFchop‐gfp mRNA is absolutely inhibited by huORFchop element in huORFZ embryos during normal condition. However, the downstream GFP translated from huORFchop‐gfp mRNA is exclusively apparent in the CNS of heat‐shock‐ and hypoxia‐treated huORFZ embryos (Lee et al, 2011, 2014; Zeng et al, 2016). Specifically, a subtype cell population, which responds with high sensitivity to ER stress, exists in the CNS and, as a result, exhibits a GFP signal. Therefore, the huORFZ line is a good in vivo system with which to study the molecular mechanism of uORF‐mediated translational control. Fluorescence was visualized with a fluorescent stereomicroscope (Leica) and a confocal spectral microscope (Leica). The experiments and treatments of this zebrafish model have been reviewed and approved by the National Taiwan University and Mackay Medical College Institutional Animal Care and Use Committee with ethics approval number NTU‐102‐EL‐24 and A1050012, respectively.RNA extraction, cDNA synthesis, microinjection, and WISHTotal RNAs were extracted from about 50–100 embryos using 500 μl of TRIzol Reagent (Lee et al, 2011). The cDNA pools were generated by Superscript III Reverse Transcriptase Kit (Invitrogen). Microinjection and WISH were performed as previously described (Lee et al, 2011) with the exception of endouc which was labeled by Digoxigenin (DIG) and used as a probe with anti‐DIG antibody (Roche; 1:8,000).Conditions of heat‐shock and hypoxic stressesSince heat‐shock and hypoxia could cause huORFZ embryos to express strong GFP signals in CNS tissues (Lee et al, 2011), we submitted fish to these two treatments to collect GFP(+) and GFP(−) cells individually. For heat‐shock treatment, we followed the protocol described by Lee et al (2011). Briefly, a 2 ml centrifuge tube filled with 30 dechorionated zebrafish embryos from transgenic line huORFZ at 72 hpf was subjected to 40°C for 1 h. Then, the treated embryos were collected into a 3‐cm Petri dish and incubated at 28.5°C until they developed at 96 hpf. Treatment to induce hypoxia followed the protocol described by Zeng et al (2016). Briefly, a deoxygenated solution was prepared as 80 ml of EM (pH 5.8) and placed in a 100‐ml serum bottle capped with a rubber stopper having a glass tube. It was kept at 28°C for 10 min, followed by pumping 99% nitrogen for 5 min at the gas flow rate of 10–20 l/min. The oxygen concentration in deoxygenated solution is less than 0.5 ppm. A 15 ml centrifuge tube filled with 100 dechorionated huORFZ embryos at 72 hpf was treated with the deoxygenated solution for 2 h at 28°C. After treatment, embryos were collected into a 9‐cm Petri dish and incubated at 28°C until they developed at 96 hpf. These treated 96‐hpf embryos were ready for further experiments.Cell capture and microarrayTo study the genes involved in huORFchop‐TI, zebrafish embryos at 72 hpf were separately treated with hypoxia and heat‐shock, followed by examining GFP signals during 96 hpf. After sections were performed in brain, we employed Laser microdissection (Leica LMD 7000) to separately capture the GFP(+) and GFP(−) cells from the same subtype. Equal numbers of GFP(+) and GFP(−) cells were centrifuged and resuspended in 200 μl of TRIzol Reagent (Invitrogen). Total RNAs were extracted using the RNeasy Plus Micro kit (Qiagen), and the expression levels of genes between GFP(+) and GFP(−) cells were analyzed by Welgene Biotech Co. (Taiwan). The Zebrafish Oligo Microarray kit (Agilent, Taiwan) was used to perform color microarray from the two noted stresses. Output from Agilent’s feature extraction software was imported using GeneSpring GX 7.3.1 software (Agilent Technologies). The ratios of treatment and reference raw intensity data were log‐transformed and normalized using locally weighted least squares regression (Lowess regression) normalization. All these candidates were subcloned, expression patterns were examined, and expression levels were compared between heat‐shocked and WT embryos using WISH.RT–qPCR analysisSamples were added to 50–100 µl of TRIzol reagent (Invitrogen) and stored at −80°C. Total RNAs were isolated according to the manufacturer's instructions. The qPCR was analyzed by Seeding Biotech Co. (Taiwan). The first‐strand cDNA was generated using 1 mg of total RNA. Both cDNA concentrations were adjusted to 200 ng/ml, and RT–qPCR was performed using the 7900HT Fast Real‐Time PCR System (Applied Biosystems, USA) according to the manufacturer's instructions. Forward and reverse primers designed for cloning each gene by PCR are presented in Appendix Table S3. Detection of spliced xbp1 and CHOP mRNAs followed the procedure described previously (van Schadewijk et al, 2012; B'chir et al, 2013). For CHOP mRNA stability assay, HEK293T cells were cultured in the presence or absence of TH for 3 h, followed by actinomycin D (10 μg/ml) treatment for 0, 1, 2, and 3 h. After total RNAs were extracted, CHOP mRNA was then quantified by RT–qPCR and normalized to 18S rRNA expression. To compare the stability of full‐length CHOP transcript and cleaved CHOP mRNA, MEF cells were transfected with plasmid pCS2‐Endouc‐Flag for 24 h, followed by actinomycin D (10 μg/ml) treatment for 0, 1, 2, and 3 h. After total RNAs were extracted, an adaptor linked at the 5′end was added for cDNA synthesis as described in the SMARTer RACE 5′/3′ kit (Takara). Then, we designed specific primers containing adaptor‐linked target sequence which could specifically anneal to either full‐length CHOP or cleaved CHOP cDNA for the RT–qPCR experiment. Primers are listed in Appendix Table S3. The data were normalized by the expression level of 18S rRNA.Cell culture, transfection, and drug treatmentCell lines HEK293T, HeLa, and U373MG were cultured and transfected as previously described (Chen et al, 2010). Insect cells Sf21 were cultured and transfected as described by Fu et al (2012). The MEF WT and eIF2αS51A (S51A) mutant cells were cultured following the protocol described by Scheuner et al (2001). Briefly, MEF WT and S51A cells were maintained in DMEM high glucose (Gibco) plus 10% fetal bovine serum (Gibco); 1% penicillin/streptomycin (Gibco); 1% non‐essential amino acid (Sigma); and 1% sodium pyruvate (Sigma). For RNA transfection assay, HEK293T cells were transfected in a 24‐well plate using Lipofectamine 3000 according to the manufacturer’s instruction (Invitrogen). We transfected 300 ng of capped or A‐capped monocistronic Firefly reporter and 10 ng of capped Renilla RNAs (as a transfection control) in each well. Transfection of 300 ng capped bicistronic mRNA and 50 ng of egfp mRNA served as negative control. Relative luc activity was measured at 2 h post‐transfection. For drug treatment, cultures were replaced by fresh medium, incubated for 2 h, and the following chemicals individually added: Dimethyl sulfoxide (DMSO; Sigma) served as the control group, 1 μM TH served as an ER stress inducer, and 200 nM ISRIB served as an inhibitor of integrated stress response (Sidrauski et al, 2015). Cells were then harvested after treatment for 1–8 h as indicated.Knockdown by small interfering RNA (siRNA) in cell linesAll siRNAs were designed and synthesized by Qiagen. AllStars Negative Control siRNA (Qiagen) was used to validate non‐silencing siRNA.Dual‐luciferase assayDual‐luciferase assay performed on cells (Chen et al, 2010) and zebrafish embryos (Lin et al, 2013) was previously described, except 5 ng/μl of phRG‐TK (internal control) and puORFchop‐luc reporter construct were used. For each experiment in zebrafish, embryos at 96 hpf were collected and divided into three groups to carry out in vivo dual‐luciferase assay (n = 60), RT–qPCR (n = 50), and Western blot analysis (n = 90).Western blotting and immunostainingWestern blot analysis and immunostaining followed the procedures described previously (Lin et al, 2013), except that the following antibodies were used: eIF2α (1:1,000; CST); p‐eIF2α (1:1,000; CST); zebrafish eIF2α (1:500; GeneTex); zebrafish DDIT3 (1:500; Techom); PERK (1:1,000; CST); p‐PERK (1:1,000; Abcam); Bip (1:1,000; CST); p58IPK (1:1,000; CST); Flag (1:30,000; Abcam); ENDOU‐1 (1:1,000; Abcam); GADD34 (1:1,000; ABD Serotec); ATF4 (1:1,000; Abcam), PKR (1:1,000; SC); human pPKR (1:1,000; Abcam); mouse pPKR(1:1,000; Sigma‐Aldrich); GAPDH (1:10,000; Abcam); and α‐tubulin (1∶5,000; Sigma). Fifty μg protein extracts were loaded in each lane to detect the protein level. Ninety μg of extracts were loaded to analyze the specific proteins from zebrafish embryos.Northern blotting, RNase activity assay, RNA‐binding assay, and RNA synthesisNorthern blot analysis was performed as previously described (Hsu et al, 2010; Rio, 2014), except huORF‐5′ antisense and huORF‐3′ antisense were synthesized (Mdbio Bioscience, Taiwan) and labeled with radioisotope p32 to serve as probes. The probes to detect zebrafish 18S RNA and human GAPDH RNA served as internal controls and were PCR‐amplified using primers listed in Appendix Table S3. For the RNase activity assay, a concentration of 10 μg purified recombinant Endouc was mixed with cell extracts (10 µg) in 10‐µl reaction buffer (50 mM NaCl, 25 mM HEPES, pH 7.4, 1 mM DTT, 0.1% glycerol, and 20 U RNasin), in which 5 pmole of biotin‐labeled RNA was contained. Samples were incubated at 37°C for 30–60 min, and RNAs were electrophoresed on 10–15% polyacrylamide gel containing 8 M Urea. For RNA‐binding assay, biotin‐labeled RNA substrate was generated using the Pierce RNA 3′ End Biotinylation kit (Thermo). Recombinant Endouc was incubated with RNA substrate for 30 min at 37°C. The results were analyzed on 7 and 15% polyacrylamide gel containing 8  M urea and stained with the Chemiluminescent Nucleic Acid Detection Module kit (Thermo). Biotin‐labeled nucleic acid probes were detected using the Chemiluminescent Nucleic Acid Detection Module Kit (Thermo). For biotin‐labeled RNA synthesis, full‐length huORFchop RNA (105 nt) was synthesized using the T7 Transcription kit (Thermo). Two pmole of template was used, and reaction was 5 h. Then, these small RNAs were labeled with biotin using the Pierce RNA 3′ End Biotinylation kit (Thermo). Primers T7 + huORF (Full)‐F plus T7 + huORF(Full)‐R, T7 + huORF‐M (Full)‐F plus T7 + huORF‐M (Full)‐R, and T7 + luc 1‐105 fragment‐F plus luc 1‐105 fragment‐R were used to amplify full‐length huORFchop, mutant huORFchop, and the luc 105 bp DNA fragment, respectively. The biotin‐labeled huORFchop template DNA was synthesized by Mdbio Bioscience. To synthesize mRNAs used for cell transfection, we generated capped transcripts using mMessage mMachine T7 Ultra kits (Ambion), precipitated with LiCl in 10 mM Tris–HCl (pH 8.5) buffer, and, finally, examined on agarose gel electrophoresis.Plasmid constructionThe huORFchop fragment and plasmid pcDNA‐GFP (pGFP) were described previously (Lee et al, 2011). The cDNAs encoding zebrafish Endouc and three human ENDOU isoforms were cloned by RT–PCR from the cDNA pools to generate pCS2‐zEndouc, pCS2‐hENDOU‐1, pCS2‐hENDOU‐2 and pCS2‐hENDOU‐3. Zebrafish Endouc contains a signal peptide at N‐terminus and a XendoU domain (amino acid residues 131–285), which is highly conserved among eukaryotes (Lee et al, 2017). To analyze Endouc domain mapping, the constructs of Endouc△1–25 (1–25 single peptide deletion), Endouc△26–136 (26–136 domain deletion), Endouc△137–298 (137–298 domain deletion), and Endouc△299–310 (C‐terminal domain deletion) were generated by PCR. Above PCR products were individually subcloned into pCS2 + to generate pCS2‐zEndouc△1–25, pCS2‐zEndouc△26–136, pCS2‐zEndouc△137–298, and pCS2‐zEndouc△299–310. The ORFs of endouc, endouc△137–298, endouc△299–310, endoucK242A, and ENDOU‐1 fused with FLAG peptide were engineered to construct pCS2‐Endouc‐Flag, pCS2‐Endouc△137–298‐Flag, pCS2‐Endouc△299–310‐Flag, pCS2‐EndoucK242‐Flag, and pCS2‐ENDOU‐1‐Flag, respectively. The C‐terminus of Endouc fused with FLAG was cut by PstI and XhoI and inserted into the plasmids containing zebrafish endouc variant to generate pCS2‐Endouc△1–25‐Flag, pCS2‐Endouc△26–136‐Flag and pCS2‐EndoucH181A‐Flag. The C‐terminal domain of ENDOU‐1 fused with FLAG was cut by NcoI and NotI and inserted into plasmids containing human ENDOU isoforms to generate pCS2‐ENDOU‐2‐Flag and pCS2‐ENDOU‐3‐Flag, while it was cut by EcoRV and NotI and inserted into pCS2‐ENDOU‐1H285A to generate pCS2‐ENDOU‐1H285A‐Flag. To analyze the endoribonuclease activity of Endouc/ENDOU‐1, we previously compared the amino acid sequence between human ENDOU isoforms, zebrafish Endouc, and Xenopus XendoU (Lee et al, 2017). E243, H244, E249, H259, and K302 are necessary for human ENDOU‐2 endoribonuclease activity (Laneve et al, 2008), while H162 and K224 are necessary for Xenopus XendoU RNase activity, corresponding to H244 and K302 of human ENDOU‐2, respectively (Schwarz & Blower, 2014). Zebrafish Endouc was therefore mutated at H181 (EndoucH181A) and K242 (EndoucK242A) to alanine, corresponding to H244 and K302 of human ENDOU‐2, respectively. H285 in human ENDOU‐1, corresponding to H244 in human ENDOUU‐2, was also mutated to alanine. These point‐mutated codons responsible for endonuclease activity were obtained by PCR site‐directed mutagenesis to generate pCS2‐zEndoucK242A, pCS2‐zEndoucH181A, and pCS2‐hENDOU‐1H285A.. Plasmid p3XFlag‐eIF2αS51A and p3XFlag‐eIF2αS51A were obtained by PCR site‐directed mutagenesis from p3XFlag‐eIF2α.The pRF/phpRF bicistronic assay systemThe DNA fragments encoding 81 to 105 nts of huORFchop transcript (huORF81‐105), 69–105 nts of huORFchop cassette (huORF69‐105), and its mutant huORF69‐105‐T81C were obtained by primer annealing, which contained a complete hairpin structure, as we predicted, using Predict a Secondary Structure Web Server (Fig 7C), and inserted between the cDNAs encoding Rluc and Fluc in a dual‐reporter system of pRF and phpRF (Stoneley et al, 1998; Jopling & Willis, 2001) to generate plasmids pRF‐uORF69–105, pRF‐uORF81–105, pRF‐uORF69–105T81C, and phpRF‐uORF69–105, phpRF‐uORF81–105, and phpRF‐uORF69–105T81C, respectively. A 105‐bp DNA fragment obtained by PCR from Renilla gene of pRL was inserted into pRF and phpRF to generate pRF‐Non105 and phpRF‐Non105 (negative controls), respectively. Primers used to construct these plasmids are shown in Appendix Table S3. Above plasmids were linearized with BamHI. The capped and polyA‐tailed bicistronic transcripts were synthesized with the T7 RiboMAX large‐scale RNA production system (Promega) according to the manufacturer's protocol. All the bicistronic mRNAs in vitro transcribed from the desired plasmid constructs were transfected into HEK293T cells for 2 h, followed by analysis of luc activity. The luc activity was measured by dual‐luciferase assay, and its corresponding bicistronic mRNAs were measured by RT–qPCR. The relative luc activity, normalized by the relative mRNA level, was represented by the increase in fold of Fluc/Rluc ratio over that obtained from control group.Purification of recombinant EndoucThe DNA fragment encoding recombinant Endouc‐Flag was inserted into the Baculovirus recombinant transfer vector pVL1392 (BD) to generate pVL1392‐Endouc‐Flag, which was transfected into insect cell line Sf21 to produce recombinant Endouc‐Flag. This Endouc‐Flag protein was purified by anti‐Flag beads (Sigma).BioinformaticsTo predict the RNA secondary structure of huORFchop motif, we used http://rna.urmc.rochester.edu/RNAstructureWeb/index.html
Polysome profile analysisThe use of sucrose gradient centrifugation to perform polysome profiling was described previously (Chen et al, 2010), except that cells were incubated with Cycloheximide (100 μg/ml) for 5 min at 37°C, and the fractionated RNA (500 ng) was subjected to RT–PCR. Primers are listed in Appendix Table S3.5′RACEAfter monosomal and polysomal RNAs were fractionated, we isolated the RNA samples to analyze the 5′‐end sequences of all truncated forms derived from huORFchop transcript. The cDNA synthesis and 5′RACE followed the protocols described in the SMARTer RACE 5′/3′ kit (Takara). The specific primers used for 5′RACE are listed in Appendix Table S3. The conditions of PCR amplification were (i) for uORF‐luc fragment: 94°C 30 s, 72°C 30 s (5 cycles), 94°C 30 s, 70°C 30 s, 72°C 25 s (5 cycles), 94°C 30 s, 68°C 30 s, 72°C 25 s (for 25 cycles); and (ii) for uORF‐CHOP fragment: 94°C 30 s, 72°C 30 s (5 cycles), 94°C 30 s, 70°C 30 s, 72°C 30 s (5 cycles), 94°C 30 s, 68°C 30 s, and 72°C 30 s. After PCR amplification, the RACE products were characterized by electrophoresis and subcloned using the In‐Fusion HD cloning kit (Takara). Finally, the candidate RACE products were sequenced using M13F primers.Statistical analysisData were obtained by the average value from three independent experiments and presented as mean ± SD. Difference levels were analyzed using Student’s t‐test.

Materials and Methods

Zebrafish husbandry, transgenic line, and signal observationZebrafish AB strain and transgenic line were cultured as previously described (Westerfield, 2000; Lee et al, 2014). Transgenic line huORFZ harbors a DNA fragment containing the huORFchop element fused with the downstream GFP cDNA (huORFchop‐gfp) and driven by a cytomegalovirus promoter (Lee et al, 2011). The translation of transgenic huORFchop‐gfp mRNA is absolutely inhibited by huORFchop element in huORFZ embryos during normal condition. However, the downstream GFP translated from huORFchop‐gfp mRNA is exclusively apparent in the CNS of heat‐shock‐ and hypoxia‐treated huORFZ embryos (Lee et al, 2011, 2014; Zeng et al, 2016). Specifically, a subtype cell population, which responds with high sensitivity to ER stress, exists in the CNS and, as a result, exhibits a GFP signal. Therefore, the huORFZ line is a good in vivo system with which to study the molecular mechanism of uORF‐mediated translational control. Fluorescence was visualized with a fluorescent stereomicroscope (Leica) and a confocal spectral microscope (Leica). The experiments and treatments of this zebrafish model have been reviewed and approved by the National Taiwan University and Mackay Medical College Institutional Animal Care and Use Committee with ethics approval number NTU‐102‐EL‐24 and A1050012, respectively.

Zebrafish husbandry, transgenic line, and signal observation

Zebrafish AB strain and transgenic line were cultured as previously described (Westerfield, 2000; Lee et al, 2014). Transgenic line huORFZ harbors a DNA fragment containing the huORFchop element fused with the downstream GFP cDNA (huORFchop‐gfp) and driven by a cytomegalovirus promoter (Lee et al, 2011). The translation of transgenic huORFchop‐gfp mRNA is absolutely inhibited by huORFchop element in huORFZ embryos during normal condition. However, the downstream GFP translated from huORFchop‐gfp mRNA is exclusively apparent in the CNS of heat‐shock‐ and hypoxia‐treated huORFZ embryos (Lee et al, 2011, 2014; Zeng et al, 2016). Specifically, a subtype cell population, which responds with high sensitivity to ER stress, exists in the CNS and, as a result, exhibits a GFP signal. Therefore, the huORFZ line is a good in vivo system with which to study the molecular mechanism of uORF‐mediated translational control. Fluorescence was visualized with a fluorescent stereomicroscope (Leica) and a confocal spectral microscope (Leica). The experiments and treatments of this zebrafish model have been reviewed and approved by the National Taiwan University and Mackay Medical College Institutional Animal Care and Use Committee with ethics approval number NTU‐102‐EL‐24 and A1050012, respectively.

RNA extraction, cDNA synthesis, microinjection, and WISHTotal RNAs were extracted from about 50–100 embryos using 500 μl of TRIzol Reagent (Lee et al, 2011). The cDNA pools were generated by Superscript III Reverse Transcriptase Kit (Invitrogen). Microinjection and WISH were performed as previously described (Lee et al, 2011) with the exception of endouc which was labeled by Digoxigenin (DIG) and used as a probe with anti‐DIG antibody (Roche; 1:8,000).

RNA extraction, cDNA synthesis, microinjection, and WISH

Total RNAs were extracted from about 50–100 embryos using 500 μl of TRIzol Reagent (Lee et al, 2011). The cDNA pools were generated by Superscript III Reverse Transcriptase Kit (Invitrogen). Microinjection and WISH were performed as previously described (Lee et al, 2011) with the exception of endouc which was labeled by Digoxigenin (DIG) and used as a probe with anti‐DIG antibody (Roche; 1:8,000).

Conditions of heat‐shock and hypoxic stressesSince heat‐shock and hypoxia could cause huORFZ embryos to express strong GFP signals in CNS tissues (Lee et al, 2011), we submitted fish to these two treatments to collect GFP(+) and GFP(−) cells individually. For heat‐shock treatment, we followed the protocol described by Lee et al (2011). Briefly, a 2 ml centrifuge tube filled with 30 dechorionated zebrafish embryos from transgenic line huORFZ at 72 hpf was subjected to 40°C for 1 h. Then, the treated embryos were collected into a 3‐cm Petri dish and incubated at 28.5°C until they developed at 96 hpf. Treatment to induce hypoxia followed the protocol described by Zeng et al (2016). Briefly, a deoxygenated solution was prepared as 80 ml of EM (pH 5.8) and placed in a 100‐ml serum bottle capped with a rubber stopper having a glass tube. It was kept at 28°C for 10 min, followed by pumping 99% nitrogen for 5 min at the gas flow rate of 10–20 l/min. The oxygen concentration in deoxygenated solution is less than 0.5 ppm. A 15 ml centrifuge tube filled with 100 dechorionated huORFZ embryos at 72 hpf was treated with the deoxygenated solution for 2 h at 28°C. After treatment, embryos were collected into a 9‐cm Petri dish and incubated at 28°C until they developed at 96 hpf. These treated 96‐hpf embryos were ready for further experiments.

Conditions of heat‐shock and hypoxic stresses

Since heat‐shock and hypoxia could cause huORFZ embryos to express strong GFP signals in CNS tissues (Lee et al, 2011), we submitted fish to these two treatments to collect GFP(+) and GFP(−) cells individually. For heat‐shock treatment, we followed the protocol described by Lee et al (2011). Briefly, a 2 ml centrifuge tube filled with 30 dechorionated zebrafish embryos from transgenic line huORFZ at 72 hpf was subjected to 40°C for 1 h. Then, the treated embryos were collected into a 3‐cm Petri dish and incubated at 28.5°C until they developed at 96 hpf. Treatment to induce hypoxia followed the protocol described by Zeng et al (2016). Briefly, a deoxygenated solution was prepared as 80 ml of EM (pH 5.8) and placed in a 100‐ml serum bottle capped with a rubber stopper having a glass tube. It was kept at 28°C for 10 min, followed by pumping 99% nitrogen for 5 min at the gas flow rate of 10–20 l/min. The oxygen concentration in deoxygenated solution is less than 0.5 ppm. A 15 ml centrifuge tube filled with 100 dechorionated huORFZ embryos at 72 hpf was treated with the deoxygenated solution for 2 h at 28°C. After treatment, embryos were collected into a 9‐cm Petri dish and incubated at 28°C until they developed at 96 hpf. These treated 96‐hpf embryos were ready for further experiments.

Cell capture and microarrayTo study the genes involved in huORFchop‐TI, zebrafish embryos at 72 hpf were separately treated with hypoxia and heat‐shock, followed by examining GFP signals during 96 hpf. After sections were performed in brain, we employed Laser microdissection (Leica LMD 7000) to separately capture the GFP(+) and GFP(−) cells from the same subtype. Equal numbers of GFP(+) and GFP(−) cells were centrifuged and resuspended in 200 μl of TRIzol Reagent (Invitrogen). Total RNAs were extracted using the RNeasy Plus Micro kit (Qiagen), and the expression levels of genes between GFP(+) and GFP(−) cells were analyzed by Welgene Biotech Co. (Taiwan). The Zebrafish Oligo Microarray kit (Agilent, Taiwan) was used to perform color microarray from the two noted stresses. Output from Agilent’s feature extraction software was imported using GeneSpring GX 7.3.1 software (Agilent Technologies). The ratios of treatment and reference raw intensity data were log‐transformed and normalized using locally weighted least squares regression (Lowess regression) normalization. All these candidates were subcloned, expression patterns were examined, and expression levels were compared between heat‐shocked and WT embryos using WISH.

Cell capture and microarray

To study the genes involved in huORFchop‐TI, zebrafish embryos at 72 hpf were separately treated with hypoxia and heat‐shock, followed by examining GFP signals during 96 hpf. After sections were performed in brain, we employed Laser microdissection (Leica LMD 7000) to separately capture the GFP(+) and GFP(−) cells from the same subtype. Equal numbers of GFP(+) and GFP(−) cells were centrifuged and resuspended in 200 μl of TRIzol Reagent (Invitrogen). Total RNAs were extracted using the RNeasy Plus Micro kit (Qiagen), and the expression levels of genes between GFP(+) and GFP(−) cells were analyzed by Welgene Biotech Co. (Taiwan). The Zebrafish Oligo Microarray kit (Agilent, Taiwan) was used to perform color microarray from the two noted stresses. Output from Agilent’s feature extraction software was imported using GeneSpring GX 7.3.1 software (Agilent Technologies). The ratios of treatment and reference raw intensity data were log‐transformed and normalized using locally weighted least squares regression (Lowess regression) normalization. All these candidates were subcloned, expression patterns were examined, and expression levels were compared between heat‐shocked and WT embryos using WISH.

RT–qPCR analysisSamples were added to 50–100 µl of TRIzol reagent (Invitrogen) and stored at −80°C. Total RNAs were isolated according to the manufacturer's instructions. The qPCR was analyzed by Seeding Biotech Co. (Taiwan). The first‐strand cDNA was generated using 1 mg of total RNA. Both cDNA concentrations were adjusted to 200 ng/ml, and RT–qPCR was performed using the 7900HT Fast Real‐Time PCR System (Applied Biosystems, USA) according to the manufacturer's instructions. Forward and reverse primers designed for cloning each gene by PCR are presented in Appendix Table S3. Detection of spliced xbp1 and CHOP mRNAs followed the procedure described previously (van Schadewijk et al, 2012; B'chir et al, 2013). For CHOP mRNA stability assay, HEK293T cells were cultured in the presence or absence of TH for 3 h, followed by actinomycin D (10 μg/ml) treatment for 0, 1, 2, and 3 h. After total RNAs were extracted, CHOP mRNA was then quantified by RT–qPCR and normalized to 18S rRNA expression. To compare the stability of full‐length CHOP transcript and cleaved CHOP mRNA, MEF cells were transfected with plasmid pCS2‐Endouc‐Flag for 24 h, followed by actinomycin D (10 μg/ml) treatment for 0, 1, 2, and 3 h. After total RNAs were extracted, an adaptor linked at the 5′end was added for cDNA synthesis as described in the SMARTer RACE 5′/3′ kit (Takara). Then, we designed specific primers containing adaptor‐linked target sequence which could specifically anneal to either full‐length CHOP or cleaved CHOP cDNA for the RT–qPCR experiment. Primers are listed in Appendix Table S3. The data were normalized by the expression level of 18S rRNA.

RT–qPCR analysis

Samples were added to 50–100 µl of TRIzol reagent (Invitrogen) and stored at −80°C. Total RNAs were isolated according to the manufacturer's instructions. The qPCR was analyzed by Seeding Biotech Co. (Taiwan). The first‐strand cDNA was generated using 1 mg of total RNA. Both cDNA concentrations were adjusted to 200 ng/ml, and RT–qPCR was performed using the 7900HT Fast Real‐Time PCR System (Applied Biosystems, USA) according to the manufacturer's instructions. Forward and reverse primers designed for cloning each gene by PCR are presented in Appendix Table S3. Detection of spliced xbp1 and CHOP mRNAs followed the procedure described previously (van Schadewijk et al, 2012; B'chir et al, 2013). For CHOP mRNA stability assay, HEK293T cells were cultured in the presence or absence of TH for 3 h, followed by actinomycin D (10 μg/ml) treatment for 0, 1, 2, and 3 h. After total RNAs were extracted, CHOP mRNA was then quantified by RT–qPCR and normalized to 18S rRNA expression. To compare the stability of full‐length CHOP transcript and cleaved CHOP mRNA, MEF cells were transfected with plasmid pCS2‐Endouc‐Flag for 24 h, followed by actinomycin D (10 μg/ml) treatment for 0, 1, 2, and 3 h. After total RNAs were extracted, an adaptor linked at the 5′end was added for cDNA synthesis as described in the SMARTer RACE 5′/3′ kit (Takara). Then, we designed specific primers containing adaptor‐linked target sequence which could specifically anneal to either full‐length CHOP or cleaved CHOP cDNA for the RT–qPCR experiment. Primers are listed in Appendix Table S3. The data were normalized by the expression level of 18S rRNA.

Cell culture, transfection, and drug treatmentCell lines HEK293T, HeLa, and U373MG were cultured and transfected as previously described (Chen et al, 2010). Insect cells Sf21 were cultured and transfected as described by Fu et al (2012). The MEF WT and eIF2αS51A (S51A) mutant cells were cultured following the protocol described by Scheuner et al (2001). Briefly, MEF WT and S51A cells were maintained in DMEM high glucose (Gibco) plus 10% fetal bovine serum (Gibco); 1% penicillin/streptomycin (Gibco); 1% non‐essential amino acid (Sigma); and 1% sodium pyruvate (Sigma). For RNA transfection assay, HEK293T cells were transfected in a 24‐well plate using Lipofectamine 3000 according to the manufacturer’s instruction (Invitrogen). We transfected 300 ng of capped or A‐capped monocistronic Firefly reporter and 10 ng of capped Renilla RNAs (as a transfection control) in each well. Transfection of 300 ng capped bicistronic mRNA and 50 ng of egfp mRNA served as negative control. Relative luc activity was measured at 2 h post‐transfection. For drug treatment, cultures were replaced by fresh medium, incubated for 2 h, and the following chemicals individually added: Dimethyl sulfoxide (DMSO; Sigma) served as the control group, 1 μM TH served as an ER stress inducer, and 200 nM ISRIB served as an inhibitor of integrated stress response (Sidrauski et al, 2015). Cells were then harvested after treatment for 1–8 h as indicated.

Cell culture, transfection, and drug treatment

Cell lines HEK293T, HeLa, and U373MG were cultured and transfected as previously described (Chen et al, 2010). Insect cells Sf21 were cultured and transfected as described by Fu et al (2012). The MEF WT and eIF2αS51A (S51A) mutant cells were cultured following the protocol described by Scheuner et al (2001). Briefly, MEF WT and S51A cells were maintained in DMEM high glucose (Gibco) plus 10% fetal bovine serum (Gibco); 1% penicillin/streptomycin (Gibco); 1% non‐essential amino acid (Sigma); and 1% sodium pyruvate (Sigma). For RNA transfection assay, HEK293T cells were transfected in a 24‐well plate using Lipofectamine 3000 according to the manufacturer’s instruction (Invitrogen). We transfected 300 ng of capped or A‐capped monocistronic Firefly reporter and 10 ng of capped Renilla RNAs (as a transfection control) in each well. Transfection of 300 ng capped bicistronic mRNA and 50 ng of egfp mRNA served as negative control. Relative luc activity was measured at 2 h post‐transfection. For drug treatment, cultures were replaced by fresh medium, incubated for 2 h, and the following chemicals individually added: Dimethyl sulfoxide (DMSO; Sigma) served as the control group, 1 μM TH served as an ER stress inducer, and 200 nM ISRIB served as an inhibitor of integrated stress response (Sidrauski et al, 2015). Cells were then harvested after treatment for 1–8 h as indicated.

Knockdown by small interfering RNA (siRNA) in cell linesAll siRNAs were designed and synthesized by Qiagen. AllStars Negative Control siRNA (Qiagen) was used to validate non‐silencing siRNA.

Knockdown by small interfering RNA (siRNA) in cell lines

All siRNAs were designed and synthesized by Qiagen. AllStars Negative Control siRNA (Qiagen) was used to validate non‐silencing siRNA.

Dual‐luciferase assayDual‐luciferase assay performed on cells (Chen et al, 2010) and zebrafish embryos (Lin et al, 2013) was previously described, except 5 ng/μl of phRG‐TK (internal control) and puORFchop‐luc reporter construct were used. For each experiment in zebrafish, embryos at 96 hpf were collected and divided into three groups to carry out in vivo dual‐luciferase assay (n = 60), RT–qPCR (n = 50), and Western blot analysis (n = 90).

Dual‐luciferase assay

Dual‐luciferase assay performed on cells (Chen et al, 2010) and zebrafish embryos (Lin et al, 2013) was previously described, except 5 ng/μl of phRG‐TK (internal control) and puORFchop‐luc reporter construct were used. For each experiment in zebrafish, embryos at 96 hpf were collected and divided into three groups to carry out in vivo dual‐luciferase assay (n = 60), RT–qPCR (n = 50), and Western blot analysis (n = 90).

Western blotting and immunostainingWestern blot analysis and immunostaining followed the procedures described previously (Lin et al, 2013), except that the following antibodies were used: eIF2α (1:1,000; CST); p‐eIF2α (1:1,000; CST); zebrafish eIF2α (1:500; GeneTex); zebrafish DDIT3 (1:500; Techom); PERK (1:1,000; CST); p‐PERK (1:1,000; Abcam); Bip (1:1,000; CST); p58IPK (1:1,000; CST); Flag (1:30,000; Abcam); ENDOU‐1 (1:1,000; Abcam); GADD34 (1:1,000; ABD Serotec); ATF4 (1:1,000; Abcam), PKR (1:1,000; SC); human pPKR (1:1,000; Abcam); mouse pPKR(1:1,000; Sigma‐Aldrich); GAPDH (1:10,000; Abcam); and α‐tubulin (1∶5,000; Sigma). Fifty μg protein extracts were loaded in each lane to detect the protein level. Ninety μg of extracts were loaded to analyze the specific proteins from zebrafish embryos.

Western blotting and immunostaining

Western blot analysis and immunostaining followed the procedures described previously (Lin et al, 2013), except that the following antibodies were used: eIF2α (1:1,000; CST); p‐eIF2α (1:1,000; CST); zebrafish eIF2α (1:500; GeneTex); zebrafish DDIT3 (1:500; Techom); PERK (1:1,000; CST); p‐PERK (1:1,000; Abcam); Bip (1:1,000; CST); p58IPK (1:1,000; CST); Flag (1:30,000; Abcam); ENDOU‐1 (1:1,000; Abcam); GADD34 (1:1,000; ABD Serotec); ATF4 (1:1,000; Abcam), PKR (1:1,000; SC); human pPKR (1:1,000; Abcam); mouse pPKR(1:1,000; Sigma‐Aldrich); GAPDH (1:10,000; Abcam); and α‐tubulin (1∶5,000; Sigma). Fifty μg protein extracts were loaded in each lane to detect the protein level. Ninety μg of extracts were loaded to analyze the specific proteins from zebrafish embryos.

Northern blotting, RNase activity assay, RNA‐binding assay, and RNA synthesisNorthern blot analysis was performed as previously described (Hsu et al, 2010; Rio, 2014), except huORF‐5′ antisense and huORF‐3′ antisense were synthesized (Mdbio Bioscience, Taiwan) and labeled with radioisotope p32 to serve as probes. The probes to detect zebrafish 18S RNA and human GAPDH RNA served as internal controls and were PCR‐amplified using primers listed in Appendix Table S3. For the RNase activity assay, a concentration of 10 μg purified recombinant Endouc was mixed with cell extracts (10 µg) in 10‐µl reaction buffer (50 mM NaCl, 25 mM HEPES, pH 7.4, 1 mM DTT, 0.1% glycerol, and 20 U RNasin), in which 5 pmole of biotin‐labeled RNA was contained. Samples were incubated at 37°C for 30–60 min, and RNAs were electrophoresed on 10–15% polyacrylamide gel containing 8 M Urea. For RNA‐binding assay, biotin‐labeled RNA substrate was generated using the Pierce RNA 3′ End Biotinylation kit (Thermo). Recombinant Endouc was incubated with RNA substrate for 30 min at 37°C. The results were analyzed on 7 and 15% polyacrylamide gel containing 8  M urea and stained with the Chemiluminescent Nucleic Acid Detection Module kit (Thermo). Biotin‐labeled nucleic acid probes were detected using the Chemiluminescent Nucleic Acid Detection Module Kit (Thermo). For biotin‐labeled RNA synthesis, full‐length huORFchop RNA (105 nt) was synthesized using the T7 Transcription kit (Thermo). Two pmole of template was used, and reaction was 5 h. Then, these small RNAs were labeled with biotin using the Pierce RNA 3′ End Biotinylation kit (Thermo). Primers T7 + huORF (Full)‐F plus T7 + huORF(Full)‐R, T7 + huORF‐M (Full)‐F plus T7 + huORF‐M (Full)‐R, and T7 + luc 1‐105 fragment‐F plus luc 1‐105 fragment‐R were used to amplify full‐length huORFchop, mutant huORFchop, and the luc 105 bp DNA fragment, respectively. The biotin‐labeled huORFchop template DNA was synthesized by Mdbio Bioscience. To synthesize mRNAs used for cell transfection, we generated capped transcripts using mMessage mMachine T7 Ultra kits (Ambion), precipitated with LiCl in 10 mM Tris–HCl (pH 8.5) buffer, and, finally, examined on agarose gel electrophoresis.

Northern blotting, RNase activity assay, RNA‐binding assay, and RNA synthesis

Northern blot analysis was performed as previously described (Hsu et al, 2010; Rio, 2014), except huORF‐5′ antisense and huORF‐3′ antisense were synthesized (Mdbio Bioscience, Taiwan) and labeled with radioisotope p32 to serve as probes. The probes to detect zebrafish 18S RNA and human GAPDH RNA served as internal controls and were PCR‐amplified using primers listed in Appendix Table S3. For the RNase activity assay, a concentration of 10 μg purified recombinant Endouc was mixed with cell extracts (10 µg) in 10‐µl reaction buffer (50 mM NaCl, 25 mM HEPES, pH 7.4, 1 mM DTT, 0.1% glycerol, and 20 U RNasin), in which 5 pmole of biotin‐labeled RNA was contained. Samples were incubated at 37°C for 30–60 min, and RNAs were electrophoresed on 10–15% polyacrylamide gel containing 8 M Urea. For RNA‐binding assay, biotin‐labeled RNA substrate was generated using the Pierce RNA 3′ End Biotinylation kit (Thermo). Recombinant Endouc was incubated with RNA substrate for 30 min at 37°C. The results were analyzed on 7 and 15% polyacrylamide gel containing 8  M urea and stained with the Chemiluminescent Nucleic Acid Detection Module kit (Thermo). Biotin‐labeled nucleic acid probes were detected using the Chemiluminescent Nucleic Acid Detection Module Kit (Thermo). For biotin‐labeled RNA synthesis, full‐length huORFchop RNA (105 nt) was synthesized using the T7 Transcription kit (Thermo). Two pmole of template was used, and reaction was 5 h. Then, these small RNAs were labeled with biotin using the Pierce RNA 3′ End Biotinylation kit (Thermo). Primers T7 + huORF (Full)‐F plus T7 + huORF(Full)‐R, T7 + huORF‐M (Full)‐F plus T7 + huORF‐M (Full)‐R, and T7 + luc 1‐105 fragment‐F plus luc 1‐105 fragment‐R were used to amplify full‐length huORFchop, mutant huORFchop, and the luc 105 bp DNA fragment, respectively. The biotin‐labeled huORFchop template DNA was synthesized by Mdbio Bioscience. To synthesize mRNAs used for cell transfection, we generated capped transcripts using mMessage mMachine T7 Ultra kits (Ambion), precipitated with LiCl in 10 mM Tris–HCl (pH 8.5) buffer, and, finally, examined on agarose gel electrophoresis.

Plasmid constructionThe huORFchop fragment and plasmid pcDNA‐GFP (pGFP) were described previously (Lee et al, 2011). The cDNAs encoding zebrafish Endouc and three human ENDOU isoforms were cloned by RT–PCR from the cDNA pools to generate pCS2‐zEndouc, pCS2‐hENDOU‐1, pCS2‐hENDOU‐2 and pCS2‐hENDOU‐3. Zebrafish Endouc contains a signal peptide at N‐terminus and a XendoU domain (amino acid residues 131–285), which is highly conserved among eukaryotes (Lee et al, 2017). To analyze Endouc domain mapping, the constructs of Endouc△1–25 (1–25 single peptide deletion), Endouc△26–136 (26–136 domain deletion), Endouc△137–298 (137–298 domain deletion), and Endouc△299–310 (C‐terminal domain deletion) were generated by PCR. Above PCR products were individually subcloned into pCS2 + to generate pCS2‐zEndouc△1–25, pCS2‐zEndouc△26–136, pCS2‐zEndouc△137–298, and pCS2‐zEndouc△299–310. The ORFs of endouc, endouc△137–298, endouc△299–310, endoucK242A, and ENDOU‐1 fused with FLAG peptide were engineered to construct pCS2‐Endouc‐Flag, pCS2‐Endouc△137–298‐Flag, pCS2‐Endouc△299–310‐Flag, pCS2‐EndoucK242‐Flag, and pCS2‐ENDOU‐1‐Flag, respectively. The C‐terminus of Endouc fused with FLAG was cut by PstI and XhoI and inserted into the plasmids containing zebrafish endouc variant to generate pCS2‐Endouc△1–25‐Flag, pCS2‐Endouc△26–136‐Flag and pCS2‐EndoucH181A‐Flag. The C‐terminal domain of ENDOU‐1 fused with FLAG was cut by NcoI and NotI and inserted into plasmids containing human ENDOU isoforms to generate pCS2‐ENDOU‐2‐Flag and pCS2‐ENDOU‐3‐Flag, while it was cut by EcoRV and NotI and inserted into pCS2‐ENDOU‐1H285A to generate pCS2‐ENDOU‐1H285A‐Flag. To analyze the endoribonuclease activity of Endouc/ENDOU‐1, we previously compared the amino acid sequence between human ENDOU isoforms, zebrafish Endouc, and Xenopus XendoU (Lee et al, 2017). E243, H244, E249, H259, and K302 are necessary for human ENDOU‐2 endoribonuclease activity (Laneve et al, 2008), while H162 and K224 are necessary for Xenopus XendoU RNase activity, corresponding to H244 and K302 of human ENDOU‐2, respectively (Schwarz & Blower, 2014). Zebrafish Endouc was therefore mutated at H181 (EndoucH181A) and K242 (EndoucK242A) to alanine, corresponding to H244 and K302 of human ENDOU‐2, respectively. H285 in human ENDOU‐1, corresponding to H244 in human ENDOUU‐2, was also mutated to alanine. These point‐mutated codons responsible for endonuclease activity were obtained by PCR site‐directed mutagenesis to generate pCS2‐zEndoucK242A, pCS2‐zEndoucH181A, and pCS2‐hENDOU‐1H285A.. Plasmid p3XFlag‐eIF2αS51A and p3XFlag‐eIF2αS51A were obtained by PCR site‐directed mutagenesis from p3XFlag‐eIF2α.

Plasmid construction

The huORFchop fragment and plasmid pcDNA‐GFP (pGFP) were described previously (Lee et al, 2011). The cDNAs encoding zebrafish Endouc and three human ENDOU isoforms were cloned by RT–PCR from the cDNA pools to generate pCS2‐zEndouc, pCS2‐hENDOU‐1, pCS2‐hENDOU‐2 and pCS2‐hENDOU‐3. Zebrafish Endouc contains a signal peptide at N‐terminus and a XendoU domain (amino acid residues 131–285), which is highly conserved among eukaryotes (Lee et al, 2017). To analyze Endouc domain mapping, the constructs of Endouc△1–25 (1–25 single peptide deletion), Endouc△26–136 (26–136 domain deletion), Endouc△137–298 (137–298 domain deletion), and Endouc△299–310 (C‐terminal domain deletion) were generated by PCR. Above PCR products were individually subcloned into pCS2 + to generate pCS2‐zEndouc△1–25, pCS2‐zEndouc△26–136, pCS2‐zEndouc△137–298, and pCS2‐zEndouc△299–310. The ORFs of endouc, endouc△137–298, endouc△299–310, endoucK242A, and ENDOU‐1 fused with FLAG peptide were engineered to construct pCS2‐Endouc‐Flag, pCS2‐Endouc△137–298‐Flag, pCS2‐Endouc△299–310‐Flag, pCS2‐EndoucK242‐Flag, and pCS2‐ENDOU‐1‐Flag, respectively. The C‐terminus of Endouc fused with FLAG was cut by PstI and XhoI and inserted into the plasmids containing zebrafish endouc variant to generate pCS2‐Endouc△1–25‐Flag, pCS2‐Endouc△26–136‐Flag and pCS2‐EndoucH181A‐Flag. The C‐terminal domain of ENDOU‐1 fused with FLAG was cut by NcoI and NotI and inserted into plasmids containing human ENDOU isoforms to generate pCS2‐ENDOU‐2‐Flag and pCS2‐ENDOU‐3‐Flag, while it was cut by EcoRV and NotI and inserted into pCS2‐ENDOU‐1H285A to generate pCS2‐ENDOU‐1H285A‐Flag. To analyze the endoribonuclease activity of Endouc/ENDOU‐1, we previously compared the amino acid sequence between human ENDOU isoforms, zebrafish Endouc, and Xenopus XendoU (Lee et al, 2017). E243, H244, E249, H259, and K302 are necessary for human ENDOU‐2 endoribonuclease activity (Laneve et al, 2008), while H162 and K224 are necessary for Xenopus XendoU RNase activity, corresponding to H244 and K302 of human ENDOU‐2, respectively (Schwarz & Blower, 2014). Zebrafish Endouc was therefore mutated at H181 (EndoucH181A) and K242 (EndoucK242A) to alanine, corresponding to H244 and K302 of human ENDOU‐2, respectively. H285 in human ENDOU‐1, corresponding to H244 in human ENDOUU‐2, was also mutated to alanine. These point‐mutated codons responsible for endonuclease activity were obtained by PCR site‐directed mutagenesis to generate pCS2‐zEndoucK242A, pCS2‐zEndoucH181A, and pCS2‐hENDOU‐1H285A.. Plasmid p3XFlag‐eIF2αS51A and p3XFlag‐eIF2αS51A were obtained by PCR site‐directed mutagenesis from p3XFlag‐eIF2α.

The pRF/phpRF bicistronic assay systemThe DNA fragments encoding 81 to 105 nts of huORFchop transcript (huORF81‐105), 69–105 nts of huORFchop cassette (huORF69‐105), and its mutant huORF69‐105‐T81C were obtained by primer annealing, which contained a complete hairpin structure, as we predicted, using Predict a Secondary Structure Web Server (Fig 7C), and inserted between the cDNAs encoding Rluc and Fluc in a dual‐reporter system of pRF and phpRF (Stoneley et al, 1998; Jopling & Willis, 2001) to generate plasmids pRF‐uORF69–105, pRF‐uORF81–105, pRF‐uORF69–105T81C, and phpRF‐uORF69–105, phpRF‐uORF81–105, and phpRF‐uORF69–105T81C, respectively. A 105‐bp DNA fragment obtained by PCR from Renilla gene of pRL was inserted into pRF and phpRF to generate pRF‐Non105 and phpRF‐Non105 (negative controls), respectively. Primers used to construct these plasmids are shown in Appendix Table S3. Above plasmids were linearized with BamHI. The capped and polyA‐tailed bicistronic transcripts were synthesized with the T7 RiboMAX large‐scale RNA production system (Promega) according to the manufacturer's protocol. All the bicistronic mRNAs in vitro transcribed from the desired plasmid constructs were transfected into HEK293T cells for 2 h, followed by analysis of luc activity. The luc activity was measured by dual‐luciferase assay, and its corresponding bicistronic mRNAs were measured by RT–qPCR. The relative luc activity, normalized by the relative mRNA level, was represented by the increase in fold of Fluc/Rluc ratio over that obtained from control group.

The pRF/phpRF bicistronic assay system

The DNA fragments encoding 81 to 105 nts of huORFchop transcript (huORF81‐105), 69–105 nts of huORFchop cassette (huORF69‐105), and its mutant huORF69‐105‐T81C were obtained by primer annealing, which contained a complete hairpin structure, as we predicted, using Predict a Secondary Structure Web Server (Fig 7C), and inserted between the cDNAs encoding Rluc and Fluc in a dual‐reporter system of pRF and phpRF (Stoneley et al, 1998; Jopling & Willis, 2001) to generate plasmids pRF‐uORF69–105, pRF‐uORF81–105, pRF‐uORF69–105T81C, and phpRF‐uORF69–105, phpRF‐uORF81–105, and phpRF‐uORF69–105T81C, respectively. A 105‐bp DNA fragment obtained by PCR from Renilla gene of pRL was inserted into pRF and phpRF to generate pRF‐Non105 and phpRF‐Non105 (negative controls), respectively. Primers used to construct these plasmids are shown in Appendix Table S3. Above plasmids were linearized with BamHI. The capped and polyA‐tailed bicistronic transcripts were synthesized with the T7 RiboMAX large‐scale RNA production system (Promega) according to the manufacturer's protocol. All the bicistronic mRNAs in vitro transcribed from the desired plasmid constructs were transfected into HEK293T cells for 2 h, followed by analysis of luc activity. The luc activity was measured by dual‐luciferase assay, and its corresponding bicistronic mRNAs were measured by RT–qPCR. The relative luc activity, normalized by the relative mRNA level, was represented by the increase in fold of Fluc/Rluc ratio over that obtained from control group.

Purification of recombinant EndoucThe DNA fragment encoding recombinant Endouc‐Flag was inserted into the Baculovirus recombinant transfer vector pVL1392 (BD) to generate pVL1392‐Endouc‐Flag, which was transfected into insect cell line Sf21 to produce recombinant Endouc‐Flag. This Endouc‐Flag protein was purified by anti‐Flag beads (Sigma).

Purification of recombinant Endouc

The DNA fragment encoding recombinant Endouc‐Flag was inserted into the Baculovirus recombinant transfer vector pVL1392 (BD) to generate pVL1392‐Endouc‐Flag, which was transfected into insect cell line Sf21 to produce recombinant Endouc‐Flag. This Endouc‐Flag protein was purified by anti‐Flag beads (Sigma).

BioinformaticsTo predict the RNA secondary structure of huORFchop motif, we used http://rna.urmc.rochester.edu/RNAstructureWeb/index.html

Bioinformatics

To predict the RNA secondary structure of huORFchop motif, we used http://rna.urmc.rochester.edu/RNAstructureWeb/index.html

Polysome profile analysisThe use of sucrose gradient centrifugation to perform polysome profiling was described previously (Chen et al, 2010), except that cells were incubated with Cycloheximide (100 μg/ml) for 5 min at 37°C, and the fractionated RNA (500 ng) was subjected to RT–PCR. Primers are listed in Appendix Table S3.

Polysome profile analysis

The use of sucrose gradient centrifugation to perform polysome profiling was described previously (Chen et al, 2010), except that cells were incubated with Cycloheximide (100 μg/ml) for 5 min at 37°C, and the fractionated RNA (500 ng) was subjected to RT–PCR. Primers are listed in Appendix Table S3.

5′RACEAfter monosomal and polysomal RNAs were fractionated, we isolated the RNA samples to analyze the 5′‐end sequences of all truncated forms derived from huORFchop transcript. The cDNA synthesis and 5′RACE followed the protocols described in the SMARTer RACE 5′/3′ kit (Takara). The specific primers used for 5′RACE are listed in Appendix Table S3. The conditions of PCR amplification were (i) for uORF‐luc fragment: 94°C 30 s, 72°C 30 s (5 cycles), 94°C 30 s, 70°C 30 s, 72°C 25 s (5 cycles), 94°C 30 s, 68°C 30 s, 72°C 25 s (for 25 cycles); and (ii) for uORF‐CHOP fragment: 94°C 30 s, 72°C 30 s (5 cycles), 94°C 30 s, 70°C 30 s, 72°C 30 s (5 cycles), 94°C 30 s, 68°C 30 s, and 72°C 30 s. After PCR amplification, the RACE products were characterized by electrophoresis and subcloned using the In‐Fusion HD cloning kit (Takara). Finally, the candidate RACE products were sequenced using M13F primers.

After monosomal and polysomal RNAs were fractionated, we isolated the RNA samples to analyze the 5′‐end sequences of all truncated forms derived from huORFchop transcript. The cDNA synthesis and 5′RACE followed the protocols described in the SMARTer RACE 5′/3′ kit (Takara). The specific primers used for 5′RACE are listed in Appendix Table S3. The conditions of PCR amplification were (i) for uORF‐luc fragment: 94°C 30 s, 72°C 30 s (5 cycles), 94°C 30 s, 70°C 30 s, 72°C 25 s (5 cycles), 94°C 30 s, 68°C 30 s, 72°C 25 s (for 25 cycles); and (ii) for uORF‐CHOP fragment: 94°C 30 s, 72°C 30 s (5 cycles), 94°C 30 s, 70°C 30 s, 72°C 30 s (5 cycles), 94°C 30 s, 68°C 30 s, and 72°C 30 s. After PCR amplification, the RACE products were characterized by electrophoresis and subcloned using the In‐Fusion HD cloning kit (Takara). Finally, the candidate RACE products were sequenced using M13F primers.

Statistical analysisData were obtained by the average value from three independent experiments and presented as mean ± SD. Difference levels were analyzed using Student’s t‐test.

Statistical analysis

Data were obtained by the average value from three independent experiments and presented as mean ± SD. Difference levels were analyzed using Student’s t‐test.

H‐CL and C‐YF conducted the main experiments. C‐YL, J‐RH, and T‐YH prepared plasmids, proteins, cell lines, and transgenic fish. H‐CL performed LMD and microarray. K‐YL and T‐YH performed the polysome profile assay. H‐YT and J‐RH performed RNA structure prediction. J‐CS and H‐JT conceptualized and directed the project. H‐JT edited the draft written by H‐CL and C‐YF.

Conflict of interestThe authors declare that they have no conflict of interest.

Conflict of interest

The authors declare that they have no conflict of interest.

Supporting informationAppendixClick here for additional data file.Expanded View Figures PDFClick here for additional data file.Source Data for Expanded View/AppendixClick here for additional data file.Review Process FileClick here for additional data file.Source Data for Figure 2Click here for additional data file.Source Data for Figure 3Click here for additional data file.Source Data for Figure 4Click here for additional data file.Source Data for Figure 5Click here for additional data file.Source Data for Figure 6Click here for additional data file.Source Data for Figure 8Click here for additional data file.

Supporting information

Click here for additional data file.

Expanded View Figures PDF

Click here for additional data file.

Source Data for Expanded View/Appendix

Click here for additional data file.

Review Process File

Click here for additional data file.

Source Data for Figure 2

Click here for additional data file.

Source Data for Figure 3

Click here for additional data file.

Source Data for Figure 4

Click here for additional data file.

Source Data for Figure 5

Click here for additional data file.

Source Data for Figure 6

Click here for additional data file.

Source Data for Figure 8

Click here for additional data file.
