---
pmid: '35427153'
title: Structure of the ciliogenesis-associated CPLANE complex.
authors:
- Langousis G
- Cavadini S
- Boegholm N
- Lorentzen E
- Kempf G
- Matthias P
journal: Sci Adv
year: '2022'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC9012472
doi: 10.1126/sciadv.abn0832
---

# Structure of the ciliogenesis-associated CPLANE complex.
**Authors:** Langousis G, Cavadini S, Boegholm N, Lorentzen E, Kempf G, Matthias P
**Journal:** Sci Adv (2022)
**DOI:** [10.1126/sciadv.abn0832](https://doi.org/10.1126/sciadv.abn0832)
**PMC:** [PMC9012472](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9012472/)

## Abstract

1. Sci Adv. 2022 Apr 15;8(15):eabn0832. doi: 10.1126/sciadv.abn0832. Epub 2022
Apr  15.

Structure of the ciliogenesis-associated CPLANE complex.

Langousis G(1), Cavadini S(1), Boegholm N(2), Lorentzen E(2), Kempf G(1), 
Matthias P(1)(3).

Author information:
(1)Friedrich Miescher Institute for Biomedical Research, 4058 Basel, 
Switzerland.
(2)Department of Molecular Biology and Genetics, Aarhus University, Gustav Wieds 
Vej 10c, DK-8000 Aarhus C, Denmark.
(3)Faculty of Sciences, University of Basel, 4031 Basel, Switzerland.

Dysfunctional cilia cause pleiotropic human diseases termed ciliopathies. These 
hereditary maladies are often caused by defects in cilia assembly, a complex 
event that is regulated by the ciliogenesis and planar polarity effector 
(CPLANE) proteins Wdpcp, Inturned, and Fuzzy. CPLANE proteins are essential for 
building the cilium and are mutated in multiple ciliopathies, yet their 
structure and molecular functions remain elusive. Here, we show that mammalian 
CPLANE proteins comprise a bona fide complex and report the near-atomic 
resolution structures of the human Wdpcp-Inturned-Fuzzy complex and of the mouse 
Wdpcp-Inturned-Fuzzy complex bound to the small guanosine triphosphatase Rsg1. 
Notably, the crescent-shaped CPLANE complex binds phospholipids such as 
phosphatidylinositol 3-phosphate via multiple modules and a CPLANE ciliopathy 
mutant exhibits aberrant lipid binding. Our study provides critical structural 
and functional insights into an enigmatic ciliogenesis-associated complex as 
well as unexpected molecular rationales for ciliopathies.

DOI: 10.1126/sciadv.abn0832
PMCID: PMC9012472
PMID: 35427153

## Full Text

INTRODUCTIONCilia constitute a eukaryotic novelty and can be found in most extant phyla (1). These thread-like structures are anchored to the cell surface via a modified centriole and protrude into the extracellular space. Historically, cilia have been regarded as motors that enable cell propulsion and fluid movement across epithelia (2). In addition to their motility functions, cilia are now considered as important signaling platforms that sense a wide variety of environmental cues and drive cognate cellular responses. In vertebrates, cilium-dependent signaling governs important developmental pathways and physiological processes (3, 4). In line with their crucial functions, defects in cilia give rise to more than 30 inherited human diseases termed ciliopathies. These disorders are characterized by pleiotropic clinical manifestations and exhibit diverse molecular etiologies (5).Aberrations in cilium assembly is a common feature of many ciliopathies. Cilium biogenesis (ciliogenesis) is a complex process that is under strict spatiotemporal regulation and involves dozens of proteins (6). In vertebrates, cilia are thought to assemble via either the extracellular or the intracellular pathway; the main difference is that the mother centriole docks directly to the plasma membrane in the first pathway while the cilium partly assembles in the cytoplasm in the latter (7). The better understood intracellular pathway proceeds via ultrastructurally defined stages. First, distal and subdistal appendages decorate the mother centriole (8, 9). Second, small vesicles traffic and dock on the centriole’s distal appendages where they establish the distal appendage vesicles (DAVs) (10, 11). Third, DAVs fuse to form one large ciliary vesicle (CV) upon the concomitant removal of inhibitory factors from the centriole (12–14). Last, transition zone proteins and intraflagellar transport (IFT) proteins are recruited and promote CV membrane growth and axoneme elongation in synchrony with additional vesicle fusion; this results in the formation of the mature organelle (15–18).Among the most enigmatic ciliogenesis regulators are the proteins Wdpcp, Inturned (Intu), and Fuzzy (Fuz). Broadly conserved in metazoans, they were first characterized in Drosophila where they control assembly of the fly’s actin-based wing hairs as effectors of the planar cell polarity (PCP) pathway. Notably, these proteins in vertebrates are pivotal for cilium assembly while having limited, if any, roles in PCP signaling (19–25). In mice, genetic ablation of Wdpcp, Intu, and Fuz results in marked loss of cilia and embryonic lethality because of severe developmental defects (21–23). Wdpcp, Intu, and Fuz were hence recently termed ciliogenesis and planar polarity effector (CPLANE) proteins (19, 20). CPLANE proteins are thought to govern ciliogenesis via multiple mechanisms including actin regulation (23, 24, 26, 27), septin modulation (25), IFT-A complex assembly (19), and Rab23 activation (28). Moreover, they interact with proteins that promote cilium assembly such as Jbts17 (19) and the atypical Rab-like guanosine triphosphatase (GTPase) Rsg1 (29, 30). Of note, CPLANE proteins likely operate at late phases of ciliogenesis since CPLANE, Rsg1, and Rab23 mutants initiate cilium assembly yet arrest at the CV stage (28, 30). Congruent with their key roles in cilia, CPLANE proteins have been found mutated in ciliopathies such as Bardet-Biedl syndrome (BBS), oral-facial-digital syndrome (OFDS), nephronophthisis, and short-rib polydactyly syndrome (SRPS) (19, 25).The precise molecular functions of CPLANE proteins remain poorly understood. The lack of structural information and the absence of recognizable enzymatic domains make prediction of CPLANE properties difficult. It is also uncertain whether these proteins assemble into a bona fide complex and how they integrate their suggested functions in the context of ciliogenesis. In this study, we provide evidence that CPLANE proteins comprise a biochemically stable entity, which we call the CPLANE complex. Further, we present the cryo–electron microscopy (cryo-EM) structures of the human CPLANE (HsCPLANE) complex and of the mouse CPLANE (MmCPLANE) complex bound to the small GTPase Rsg1 at near-atomic resolution. Last, we report that CPLANE proteins bind phospholipids with a clear preference for phosphatidylinositol 3-phosphate [PI(3)P] and show that a ciliopathy-associated CPLANE mutation results in abnormal PI(3)P binding.

INTRODUCTION

Cilia constitute a eukaryotic novelty and can be found in most extant phyla (1). These thread-like structures are anchored to the cell surface via a modified centriole and protrude into the extracellular space. Historically, cilia have been regarded as motors that enable cell propulsion and fluid movement across epithelia (2). In addition to their motility functions, cilia are now considered as important signaling platforms that sense a wide variety of environmental cues and drive cognate cellular responses. In vertebrates, cilium-dependent signaling governs important developmental pathways and physiological processes (3, 4). In line with their crucial functions, defects in cilia give rise to more than 30 inherited human diseases termed ciliopathies. These disorders are characterized by pleiotropic clinical manifestations and exhibit diverse molecular etiologies (5).

Aberrations in cilium assembly is a common feature of many ciliopathies. Cilium biogenesis (ciliogenesis) is a complex process that is under strict spatiotemporal regulation and involves dozens of proteins (6). In vertebrates, cilia are thought to assemble via either the extracellular or the intracellular pathway; the main difference is that the mother centriole docks directly to the plasma membrane in the first pathway while the cilium partly assembles in the cytoplasm in the latter (7). The better understood intracellular pathway proceeds via ultrastructurally defined stages. First, distal and subdistal appendages decorate the mother centriole (8, 9). Second, small vesicles traffic and dock on the centriole’s distal appendages where they establish the distal appendage vesicles (DAVs) (10, 11). Third, DAVs fuse to form one large ciliary vesicle (CV) upon the concomitant removal of inhibitory factors from the centriole (12–14). Last, transition zone proteins and intraflagellar transport (IFT) proteins are recruited and promote CV membrane growth and axoneme elongation in synchrony with additional vesicle fusion; this results in the formation of the mature organelle (15–18).

Among the most enigmatic ciliogenesis regulators are the proteins Wdpcp, Inturned (Intu), and Fuzzy (Fuz). Broadly conserved in metazoans, they were first characterized in Drosophila where they control assembly of the fly’s actin-based wing hairs as effectors of the planar cell polarity (PCP) pathway. Notably, these proteins in vertebrates are pivotal for cilium assembly while having limited, if any, roles in PCP signaling (19–25). In mice, genetic ablation of Wdpcp, Intu, and Fuz results in marked loss of cilia and embryonic lethality because of severe developmental defects (21–23). Wdpcp, Intu, and Fuz were hence recently termed ciliogenesis and planar polarity effector (CPLANE) proteins (19, 20). CPLANE proteins are thought to govern ciliogenesis via multiple mechanisms including actin regulation (23, 24, 26, 27), septin modulation (25), IFT-A complex assembly (19), and Rab23 activation (28). Moreover, they interact with proteins that promote cilium assembly such as Jbts17 (19) and the atypical Rab-like guanosine triphosphatase (GTPase) Rsg1 (29, 30). Of note, CPLANE proteins likely operate at late phases of ciliogenesis since CPLANE, Rsg1, and Rab23 mutants initiate cilium assembly yet arrest at the CV stage (28, 30). Congruent with their key roles in cilia, CPLANE proteins have been found mutated in ciliopathies such as Bardet-Biedl syndrome (BBS), oral-facial-digital syndrome (OFDS), nephronophthisis, and short-rib polydactyly syndrome (SRPS) (19, 25).

The precise molecular functions of CPLANE proteins remain poorly understood. The lack of structural information and the absence of recognizable enzymatic domains make prediction of CPLANE properties difficult. It is also uncertain whether these proteins assemble into a bona fide complex and how they integrate their suggested functions in the context of ciliogenesis. In this study, we provide evidence that CPLANE proteins comprise a biochemically stable entity, which we call the CPLANE complex. Further, we present the cryo–electron microscopy (cryo-EM) structures of the human CPLANE (HsCPLANE) complex and of the mouse CPLANE (MmCPLANE) complex bound to the small GTPase Rsg1 at near-atomic resolution. Last, we report that CPLANE proteins bind phospholipids with a clear preference for phosphatidylinositol 3-phosphate [PI(3)P] and show that a ciliopathy-associated CPLANE mutation results in abnormal PI(3)P binding.

RESULTSStructure of the mammalian CPLANE complexWe reconstituted the HsCPLANE complex (HsWdpcp-HsIntu-HsFuz) and the MmCPLANE complex bound to the Rab-like GTPase MmRsg1 (MmWdpcp-MmIntu-MmFuz-MmRsg1) using full-length recombinant proteins (Fig. 1, A and B, and fig. S1, A and B). Both complexes are stable in vitro at least up to 1 M salt and we detected binary interactions for MmWdpcp-MmIntu, MmIntu-MmFuz, and MmFuz-MmRsg1 pairs (fig. S1, C to F). Using single-particle cryo-EM, we obtained reconstructions (table S1) at 3.35 Å resolution for both the human (Fig. 1C and fig. S2, A to E) and the mouse complex (Fig. 1D and fig. S3, A to E). These maps allowed us to build models that include the majority of the primary protein sequence (Fig. 1, E and F, and fig. S4, A and B). Notably, we did not observe densities for the N-terminal ~300 residues of HsIntu/MmIntu, which contain a predicted PDZ domain (HsIntu 185 to 263 amino acids) as well as for the C-terminal ~100 residues of HsWdpcp/MmWdpcp; both Intu and Wdpcp unresolved parts are predicted to contain intrinsically disordered regions (RaptorX) (31). Overall, the human and mouse complexes are remarkably similar with a root mean square deviation (RMSD) of 3.8 Å over 93% of the common subunits’ residues (fig. S4C); given its higher local resolution of important parts of the structure, we describe in detail the MmCPLANE-MmRsg1 complex (Fig. 2).Fig. 1.Cryo-EM structures of the mammalian CPLANE complex.(A) Domain organization of HsCPLANE subunits. WD40-repeat (WDR), PDZ, and longin-like domains (LD1 to LD3) are indicated. (B) Domain organization of MmCPLANE subunits and MmRsg1. WD40-repeat (WDR), PDZ, longin-like (LD1 to LD3), and GTPase domains are indicated. (C) Cryo-EM map of HsCPLANE in two views. Map is locally sharpened and contoured at 0.160. (D) Cryo-EM map of MmCPLANE bound to MmRsg1 in two views. Map is locally sharpened and contoured at 0.160. (E) Model of the HsCPLANE complex in cartoon representation in two views. (F) Model of the MmCPLANE-MmRsg1 complex in cartoon representation in two views.Fig. 2.CPLANE subunits have extensive interfaces.Cartoon representation of the MmCPLANE-MmRsg1 model. The three interfaces between subunits (Wdpcp-Intu-Fuz, Intu-Fuz, and Fuz-Rsg1) are indicated by dashed boxes. Domains of a subunit are colored in different shades. Close-up views of interfaces are shown and secondary structure elements involved in interactions are indicated (see fig. S5 for complete annotation scheme). Wdpcp-Intu-Fuz interface (1): WDR, Helical domain of Wdpcp, LD1 to LD3 of Intu, and LD3 of Fuz are shown. Intu-Fuz interface (2): LD1 and LD3 of Intu and Fuz are shown. Fuz-Rsg1 interface (3): LD1 and LD2 of Fuz and RSG1 are shown. G elements 1 to 3 [switch I (G2) and II (G3) loops] are indicated.Wdpcp folds into a seven-bladed β-propeller belonging to the WD40 family, followed by an array of α helices that can be classified as α-solenoid (fig. S5A). Compared to mouse, the human Wdpcp has an N-terminal extension of ~35 residues that is not resolved in the map. The N-terminal end of Wdpcp complements the three-stranded β sheet of the last blade first by an antiparallel (fourth strand) and then by a small parallel β strand aligning with the C-terminal half of the fourth strand. In addition, the N-terminal half of the fourth strand aligns with the detached fourth strand of blade 6. The combination of N- and C-terminal strands in one blade is a regular pattern called “Velcro closure” and helps to stabilize the fold of the β-propeller (32). On the other hand, participation of strands from multiple blades in the Velcro closure and deviation from an antiparallel alignment constitute an uncommon variation. Two helices separated by a loop (~10 amino acids) follow the N-terminal β strands, with the first helix contacting Intu/Fuz and the second helix folding against the first blade of the β-propeller to which it connects. The second and third blades contain poorly resolved insertions of ~20 amino acids that are located on the solvent-exposed side of Wdpcp. These insertions could provide interaction sites with other factors such as Jbts17. The sixth and seventh blades are linked by an extended loop (~20 amino acids) while the last (seventh) blade connects to the C-terminal α-solenoid with a short loop of ~10 amino acids. Both loops are involved in interactions with Intu, which are discussed in detail below. Eight helices of the α-solenoid are resolved that arrange into four bundles each containing two antiparallel aligned helices (fig. S5A).Intu and Fuz were predicted to each have three longin-like domains (LDs). The structure confirms the prediction that LD1 of Intu and Fuz adopts a canonical longin fold, LD2 adopts a roadblock longin-like fold, and LD3 adopts a lamtor longin-like fold (28). Longin, roadblock, and lamtor domains are closely related and adopt a ββαβββαα, αββαβββα, and ββαβββα fold, respectively. In both Intu and Fuz, the LDs are arranged in a nonsymmetric triangular manner with each domain forming contacts to the other two domains (fig. S5, B and C). Helices α2 and α3 of LD1 stack perpendicularly on α1 of LD2. The β sheets of LD2 and LD3 form a continuous β sheet, but the connecting strand between LD2 and LD3 crosses the sheets in a diagonal manner, allowing only a small β-bridge. Helix α1 of Intu LD3 has almost double the length of corresponding helices found in the other LDs and substantially protrudes from the complex. This helix also forms contacts to α2 of LD1. In the case of Intu, an α helix, found in the linker region between LD1 and LD2, is located in the center of the LD trimer. This helix runs parallel to the extended helix α1 of LD3 and further stabilizes the interaction between the three domains. In Fuz, this helix is notably shorter and consists of only one turn. LD2 and LD3 of Intu are separated by a long linker region and about 70 amino acids were not resolved in the map. Additional density was found close to the N terminus of Intu showing features of another β strand aligning with the β sheet of LD2. Because of its short length, it was not possible to unambiguously identify the sequence but we speculate that the strand originates from the part preceding the PDZ domain that is consistent with the AlphaFold (33) prediction (fig. S4D). The three LDs of Fuz show an overall twofold pseudo-symmetrical arrangement with respect to the Intu LDs (fig. S5D) and can be superimposed with an RMSD of 5.7 Å despite low sequence similarity (fig. S5E). The structural conservation is in line with the notion that Intu and Fuz may have originated from a common homodimer-forming ancestor by gene duplication (28).The core structure of Rsg1 follows the typical small GTPase fold characterized by a central six-stranded concave β sheet surrounded by five α helices (Fig. 3A). In addition, Rsg1 contains an N-terminal extension of ~50 amino acids preceding the first β strand that is partially resolved (amino acids 11 to 37) and contains an additional α helix (α0). Ηelix α0 packs against the second and third β strands of the extended interswitch in a parallel manner and the N-terminal tip comes close to the loop between the G1 and G2 motifs. The third β strand, which is part of the interswitch, is disrupted by a proline and the N-terminal half interacts with the G1 motif, while the C-terminal half aligns with helix α0. In GTPases of the Arf family, an N-terminal α helix contacts the short interswitch in a perpendicular manner and has been implicated in regulatory functions (“interswitch toggle”) (34). Since the engagement with the long interswitch of Rsg1 is somewhat different (parallel compared to perpendicular), the putative regulatory mechanism might be distinct. Similar to the related REM GTPases, the Rsg1 G3 motif has an atypical sequence (DCGES) that deviates from the canonical DXXGQ motif found in most small GTPases and lacks the catalytically important glutamine. The EM map strongly indicates the presence of a nucleotide (Fig. 3B) and superposition with other GTPases bound to guanosine triphosphate (GTP) analogs (35) shows that a triphosphate would explain the density best (Fig. 3C). A coordinated magnesium could not be identified, probably owing to the limited local resolution. Since the nucleotide specificity of Rsg1 was obscure (30), we characterized endogenously bound nucleotides in purified Rsg1 and Fuz-Rsg1 using reversed-phase high-performance liquid chromatography (HPLC). In both cases, we could identify GTP but not GDP as the bound nucleotide (Fig. 3D), suggesting a low intrinsic GTP hydrolysis rate. An explanation at the molecular level could be the substitution of catalytic glutamine by serine. This would be in line with the atypical GTPases IFT22 and IFT27 bearing a serine at the same position and for which low GTPase activities have been shown (36, 37). In addition, the histidine 84 side chain (located in the switch I loop) is oriented toward the γ-phosphate and presumably contributes to stabilization of the negative charge (Fig. 3B). Of note, the histidine is not in a position where it could coordinate the catalytic water as seen in EF-Tu (38) and rather overlaps with the position of arginine fingers.Fig. 3.Nucleotide content of MmRsg1.(A) Model of MmRsg1. (B) Map density in MmRsg1 nucleotide pocket. Putative GTP is shown in green. (C) Overlay of Rsg1 (green) and Rad (orange; 3Q72) nucleotide conformations (Rad was superposed on Rsg1). Map is locally sharpened and contoured at 0.257. (D) Reverse-phase HPLC analysis of bound nucleotides in Rsg1 and Fuz-Rsg1 [purified proteins shown in insets and labeled F (Fuz) and R (Rsg1)]. Pure GDP and GTP (100 μM) were run as standards in triplicate. Rsg1 (40 μM) and Fuz-Rsg1 (73 μM) were run in triplicate and quadruplicate, respectively.CPLANE subunits have extensive interfacesBoth HsCPLANE and MmCPLANE complexes adopt a crescent-like architecture that is particularly obvious in the mouse complex (Fig. 1, C and D). On one side of the complex lies Wdpcp that mainly contacts Intu with a large interface area of 2609 Å2. The main interaction platform is provided by Intu LD2, which forms extensive contacts with the α-helical domain of Wdpcp (Fig. 2 and fig. S6, A and D). Helices α3 and α5 of Wdpcp stack in a perpendicular manner onto α0, parts of β1, and loops between β2 and α1 as well as β5 and α2 of Intu LD2. There is a strong charge complementarity between Wdpcp (positively charged patch) and Intu (negatively charged patch; fig. S6D). In particular, Wdpcp H466, R502, R544, and K541 are within hydrogen bonding distance of Intu main-chain carbonyls from L940, L942/S532, A499, and the side chain of D496, respectively. On the distal end of the Wdpcp helical domain, a poorly resolved loop region between α0 and β1 of Intu LD2 folds onto Wdpcp helix α7 including hydrophobic and polar contacts (fig. S6E). Another interaction hotspot is centered around the loop connecting the β-propeller and helical domains of Wdpcp. The loop inserts into a pocket formed between Intu LD2 and LD3, and its hydrophobic tip is stacking against a hydrophobic patch inside of the pocket (fig. S6C). In detail, Wdpcp L450 is surrounded by Intu L940, I813, and V481 and Wdpcp I449 packs against the alkyl chain of Intu K935. In addition to hydrophobic contacts, Wdpcp R452 likely forms hydrogen bonds with main-chain carbonyls of Intu P654 and S653 at the entry of the pocket. While contributing to the pocket surface, the C terminus of Intu at the same time meanders around α1 of the Wdpcp helical domain. On the other far end of the interface, a ternary hub is formed between the Wdpcp β-propeller domain, Fuz LD2 and Intu LD3 (Fig. 2 and fig. S6B). The first α helix located in the N-terminal region of Wdpcp and an extended linker between β-propeller blades 6 and 7 form contacts to helices α2 of the LD2 of Intu and Fuz. The density suggests putative salt bridges between Wdpcp R22 and Fuz D295 as well as between Fuz R401 and Intu D923. The interaction between Wdpcp and Fuz contributes an additional contact area of 351 Å2 and expands the interface to a length of almost 80 Å. In summary, an extensive and intertwined interface is formed between Wdpcp and Intu-Fuz, relying on charge and shape complementarity. Intriguingly, the Wdpcp α-solenoid and Intu longin domain interaction is found in cellular trafficking complexes such as the AP2 clathrin adaptor (39) or the β/δ-COPI complex (40). In both examples, the orientation between the α-solenoid and bound longin domain is similar to Wdpcp-Intu; however, the α-solenoid is remarkably extended and encloses the longin domain.Antipodal to the Wdpcp-Intu interface, Intu contacts Fuz with a similarly large buried surface area of 1898 Å2 (Fig. 2). Formation of the Intu-Fuz heterodimer is mediated by LD1 and LD3. In both cases (Intu_LD1-Fuz_LD1 and Intu_LD3-Fuz_LD3), the LDs interact via their third β strand (β3) to form a continuous β sheet as seen in previous structures (Fig. 2) (41). In case of the LD1 Intu-Fuz heterodimer, the outward-facing helices α1 also align in a parallel manner and form additional, mostly hydrophobic contacts (fig. S6F). The LD3 Intu-Fuz heterodimer is in the same orientation as the LD1 dimer, and thus, helices α1 are buried in the center and form less contacts (fig. S6F). Helices α3 are facing outward and come into reach at their N-terminal tips (~5.8 Å backbone distance) but do not seem to form pronounced contacts. To the best of our knowledge, this is the first structure of a complete mammalian hexalongin scaffold and thus provides a blueprint for homologous complexes of the HerMon family such as Mon1-Czz1 and HPS1-HPS4 (28).CPLANE binds to Rsg1 in an effector-like mannerRsg1 lies on the other end of the complex and interacts solely with Fuz. Of note, the Fuz-Rsg1 interaction was stable in vitro without the addition of exogenous nucleotides. The Fuz-Rsg1 interface spans an area of 1099 Å2 and consists of Fuz LD1 and LD2 engaging the Rsg1 switch I + II regions, a loop between α0 and β1, and the interswitch region that is part of the central β sheet of RSG1 (Fig. 2). Notably, a β sheet completion is observed between β3 of Fuz LD2 and β2 of RSG1 (Fig. 4A). The density suggests that the main-chain hydrogen bonds are extended by a hydrogen bond network between Fuz Q215, Y220 and RSG1 T93, Q91 side chains (Fig. 4A). Nearby, the loop connecting RSG1 α0 to β1 is running close to the β-hairpin between β4 and β5 of Fuz LD2 and presumably forms a short β-bridge. The solvent-exposed interswitch β2 often acts as a platform for effector binding (42) or, in some cases, is contacted by the switch I loop when Rab GTPases are bound by a guanine nucleotide exchange factor (GEF). A common feature of GEFs is remodeling of the switch I + II loops to lower the affinity for nucleotide. This mechanism is observed in the homologous hexalongin complex Mon1-Czz1 bound to YPT7 (41), where the switch I loop is dislodged from the active site and forms a small β-bridge with the interswitch. This is in strong contrast to the CPLANE-Rsg1 complex that shows a substantially different overall binding mode where the interswitch is occupied by Fuz and switch I is ordered around the nucleotide as expected for the GTP-bound state (Fig. 4B). Comparison with other longin domain containing GEF/GTPase complexes, such as DENND1B:Rab35 (43), did not reveal a comparable binding mode. In addition, no indications for other nucleotide exchange mechanisms, such as the insertion of residues into the pocket by the GEF, are evident from the EM map. These structural observations suggest that Intu-Fuz is not a GEF for RSG1.Fig. 4.CPLANE interaction with Rsg1.(A) β sheet completion between the third β strand of Fuz LD2 and the interswitch (InSw) of Rsg1. (B) Interaction of Rsg1 switch I + II with Fuz helix α3. RSG1 H85, E86, and K123 side chains are approaching helix α3. G2 H84 is potentially interacting with the phosphate groups of the nucleotide (green). Conserved G2 T87 and G3 D116 as well as G3 S120 substituting for a conserved catalytic glutamine are shown. Residues lacking sufficient side-chain density are indicated by spheres.Rsg1 switch I and a helical part of switch II occupy a shallow partially hydrophobic groove between LD1 and LD2 and are predominantly in contact with helix α3 of Fuz LD1. Helix α3 is curved in both Rsg1-bound (in mouse CPLANE) and -unbound Fuz (in human CPLANE) and contributes to interface shape complementarity; interestingly, this Fuz helix contains a conserved arginine residue (R129) often provided in trans by GTPase-activating proteins (GAPs; Fig. 4B). The cryo-EM density, however, does not suggest that R129 inserts into the active site and modeling of different rotamer conformations shows that the arginine cannot get close enough to properly align with the phosphate groups. Moreover, accessibility of the phosphate groups is hindered by switch I histidine H84 and presumably by the poorly resolved E86 that could come into hydrogen bond distance (Fig. 4B). Comparison to other GTPase-GAP complexes shows that a backbone shift by at least 3 Å would be necessary to bring the arginine into a competent position (44) and entry to the active site would need to be opened by rearrangement of switch I residues. In other GTP-bound RGK and Rab GTPases (35), the switch I + II loops are completely or partially disordered in the absence of an interaction partner; thus, binding of Fuz might contribute to a stabilization of these G elements as it has been described for IFT22 bound to IFT74/81 (36).The binding mode of known longin-GTPase complexes (45) is notably different from the observed Fuz-Rsg1 interface. We therefore compared the Fuz LD1/Rsg1 complex against the PDB for similar interfaces using the TopSearch webserver (46); this yielded effector and GAP complexes as top ranked hits that, however, do not involve interactions with longin domains. A common feature of the top hits is an α helix at a similar position to α3 of Fuz LD2 that is engaging Rsg1 switch I + II. Overall, on the basis of these observations, we propose that Fuz and hence the CPLANE complex predominantly functions as an effector of Rsg1. We do not rule out an additional function as co-GAP by stabilization of the switch I + II loops, but more biochemical evidence is needed to support this notion.CPLANE proteins bind phospholipidsWe noticed that the domains of CPLANE subunits are characteristic of proteins involved in vesicle trafficking and are commonly found in protocoatomer-associated complexes (47). These domains include the β-propeller/α-solenoid of Wdpcp and the LDs of Intu and Fuz. Moreover, Rsg1 is a Rab-like GTPase and hence expected to be involved in membrane trafficking (47). We were thus prompted to determine whether CPLANE proteins bind phospholipids. Both HsWdpcp-HsIntu-HsFuz and MmWdpcp-MmIntu-MmFuz-MmRsg1 complexes bind to phosphoinositides (PIPs) immobilized on membranes. CPLANE complexes interacted solely with PIPs having a single inositol phosphate and exhibited a clear preference for PI(3)P, followed by PI(5)P and PI(4)P (Fig. 5, A and B). We next asked which CPLANE protein mediates the lipid binding and repeated the assay with individual mouse proteins. Unexpectedly, all three mouse CPLANE subunits (MmWdpcp, MmIntu, and MmFuz) but not MmRsg1 were found to bind PIPs (Fig. 5, C to F). Similar to the CPLANE complex, individual subunits bind PI(3)P with highest affinity, followed by PI(5)P and PI(4)P. MmIntu displayed the strongest binding, followed by MmWdpcp and MmFuz (Fig. 5, C to F). We next mapped the PIP binding domains in MmIntu and MmWdpcp. MmIntu uses the N-terminal ~300 amino acids to bind PIPs, with a 47– to 267–amino acid construct being competent for binding; in addition, we detected weaker PIP binding by the Intu LDs (341 to 942 amino acids; fig. S7A). In contrast, MmWdpcp PIP binding is mediated by the C-terminal ~60 amino acids (fig. S7B); within this fragment, longer deletions show progressively lower binding, indicating an extensive binding interface (fig. S7C).Fig. 5.CPLANE proteins bind PIPs.(A) HsCPLANE complex binding to PIP strip. Input and Western blot (3 nM) (equal exposures) for Flag-tagged (left) and untagged (right) Wdpcp are shown. (B) MmCPLANE + MmRsg1 complex binding to PIP strip. Input and Western blot (4 nM) (equal exposures) for Flag-tagged (left) and untagged (right) Wdpcp are shown. (C) MmIntu binding to PIP strip. Input and Western blot (5 nM) for Flag-tagged Intu are shown. (D) MmWdpcp binding to PIP strip. Input and Western blot (20 nM) for Flag-tagged Wdpcp are shown. (E) MmFuz binding to PIP strip. Input and Western blot (20 nM) for Flag-tagged Fuz are shown. (F) MmRsg1 binding to PIP strip. Input and Western blot (20 nM) for Flag-tagged Rsg1 are shown. Blots in (E) and (F) have equal exposures.Analysis of ciliopathy-associated mutationsMutations in CPLANE proteins are associated with ciliopathies and other developmental defects. In addition to frameshift changes that commonly result in premature stop codons, several missense point mutations have been reported (19, 25, 48, 49). We took advantage of our near-atomic HsCPLANE model and mapped 10 missense point mutations onto HsWdpcp, HsIntu, and HsFuz. With the exception of four mutations (HsWdpcp_L205F, HsWdpcp_S708F, HsIntu_A452T, and HsFuz_D354Y) that were located in unresolved parts, the remaining six could be assigned to map densities (Fig. 6A). Intriguingly, several disease-relevant mutations are located at subunit interfaces (see fig. S6, A to E, for mouse cognate residues). We wondered whether these different mutations affect assembly of the HsCPLANE complex. To answer this, we expressed all recombinant human proteins incorporating the mutations and assessed HsCPLANE complex formation in vitro. We failed to detect any obvious defects in expression of individual subunits or assembly of the HsCPLANE (fig. S8, A to D, F, and G). Moreover, mutations did not inhibit HsCPLANE subunits’ interaction with HsRsg1 (fig. S8, E and H). Mutations in HsRsg1 have not been reported yet, but the mouse pixiebob (pxb) mutant, which exhibits early embryonic lethality, was shown to be due to a valine-169–to–aspartic acid substitution in MmRsg1 (30). We therefore mapped this residue in our structure; notably, V169 is distant from the Fuz-Rsg1 interface and we verified that MmRsg1_V169D interacts with MmFuz in vitro although its expression is reduced compared to MmRsg1_WT (fig. S8, I and J). Given the lipid binding properties of CPLANE proteins, we then asked whether the ciliopathy mutations perturb PIP binding. To address this, we focused on the HsWdpcp_Ser708Phe mutation that is found in the PIP binding site of the unresolved Wdpcp C-terminal tail. This mutation was found to be heterozygous in a patient diagnosed with BBS; notably, no other BBS-related mutations were detected in the affected individual (25). Unexpectedly, the Wdpcp_S708F protein exhibited stronger binding than the wild-type (WT) on PIP strips, but its PIP specificity remained unchanged (Fig. 6B). We confirmed this result using arrays with a concentration gradient of PIPs. Consistent with the previous results, the S708F protein had clearly higher relative binding to PIPs as compared to WT (Fig. 6C). Collectively, these results indicate that the reported ciliopathy mutations do not affect assembly of the recombinant HsCPLANE complex but rather disrupt CPLANE functions such as PIP binding.Fig. 6.Disease-associated CPLANE mutations.(A) Cartoon representation of the HsCPLANE model. Missense mutations are shown as red (Wdpcp), blue (Intu), and orange (Fuz) spheres. Wdpcp (L205F and S708F), Intu (A452T), and Fuz (D354Y) are in unresolved parts of the complex. The associated genetic diseases are shown in the box: oral-facial-digital syndrome (OFDS), Meckel-Gruber syndrome (MKS), Bardet-Biedl syndrome (BBS), Joubert syndrome (JBTS), nephronophthisis (NPHP), short-rib polydactyly syndrome (SRPS), and neural tube defects (NTD). (B) HsWdpcp_485-746 (helical α-solenoid) binding to PIP strip for wild-type (WT) and S708F mutant. Input and Western blot (10 nM) (equal exposures) for Flag-tagged constructs are shown. (C) HsWdpcp_485-746 (helical α-solenoid) binding to PIP array (shown 25 to 100 pmol PIP spots) for WT and S708F mutant. Input and Western blot (12 nM) (equal exposures) for Flag-tagged constructs are shown.

Structure of the mammalian CPLANE complexWe reconstituted the HsCPLANE complex (HsWdpcp-HsIntu-HsFuz) and the MmCPLANE complex bound to the Rab-like GTPase MmRsg1 (MmWdpcp-MmIntu-MmFuz-MmRsg1) using full-length recombinant proteins (Fig. 1, A and B, and fig. S1, A and B). Both complexes are stable in vitro at least up to 1 M salt and we detected binary interactions for MmWdpcp-MmIntu, MmIntu-MmFuz, and MmFuz-MmRsg1 pairs (fig. S1, C to F). Using single-particle cryo-EM, we obtained reconstructions (table S1) at 3.35 Å resolution for both the human (Fig. 1C and fig. S2, A to E) and the mouse complex (Fig. 1D and fig. S3, A to E). These maps allowed us to build models that include the majority of the primary protein sequence (Fig. 1, E and F, and fig. S4, A and B). Notably, we did not observe densities for the N-terminal ~300 residues of HsIntu/MmIntu, which contain a predicted PDZ domain (HsIntu 185 to 263 amino acids) as well as for the C-terminal ~100 residues of HsWdpcp/MmWdpcp; both Intu and Wdpcp unresolved parts are predicted to contain intrinsically disordered regions (RaptorX) (31). Overall, the human and mouse complexes are remarkably similar with a root mean square deviation (RMSD) of 3.8 Å over 93% of the common subunits’ residues (fig. S4C); given its higher local resolution of important parts of the structure, we describe in detail the MmCPLANE-MmRsg1 complex (Fig. 2).Fig. 1.Cryo-EM structures of the mammalian CPLANE complex.(A) Domain organization of HsCPLANE subunits. WD40-repeat (WDR), PDZ, and longin-like domains (LD1 to LD3) are indicated. (B) Domain organization of MmCPLANE subunits and MmRsg1. WD40-repeat (WDR), PDZ, longin-like (LD1 to LD3), and GTPase domains are indicated. (C) Cryo-EM map of HsCPLANE in two views. Map is locally sharpened and contoured at 0.160. (D) Cryo-EM map of MmCPLANE bound to MmRsg1 in two views. Map is locally sharpened and contoured at 0.160. (E) Model of the HsCPLANE complex in cartoon representation in two views. (F) Model of the MmCPLANE-MmRsg1 complex in cartoon representation in two views.Fig. 2.CPLANE subunits have extensive interfaces.Cartoon representation of the MmCPLANE-MmRsg1 model. The three interfaces between subunits (Wdpcp-Intu-Fuz, Intu-Fuz, and Fuz-Rsg1) are indicated by dashed boxes. Domains of a subunit are colored in different shades. Close-up views of interfaces are shown and secondary structure elements involved in interactions are indicated (see fig. S5 for complete annotation scheme). Wdpcp-Intu-Fuz interface (1): WDR, Helical domain of Wdpcp, LD1 to LD3 of Intu, and LD3 of Fuz are shown. Intu-Fuz interface (2): LD1 and LD3 of Intu and Fuz are shown. Fuz-Rsg1 interface (3): LD1 and LD2 of Fuz and RSG1 are shown. G elements 1 to 3 [switch I (G2) and II (G3) loops] are indicated.Wdpcp folds into a seven-bladed β-propeller belonging to the WD40 family, followed by an array of α helices that can be classified as α-solenoid (fig. S5A). Compared to mouse, the human Wdpcp has an N-terminal extension of ~35 residues that is not resolved in the map. The N-terminal end of Wdpcp complements the three-stranded β sheet of the last blade first by an antiparallel (fourth strand) and then by a small parallel β strand aligning with the C-terminal half of the fourth strand. In addition, the N-terminal half of the fourth strand aligns with the detached fourth strand of blade 6. The combination of N- and C-terminal strands in one blade is a regular pattern called “Velcro closure” and helps to stabilize the fold of the β-propeller (32). On the other hand, participation of strands from multiple blades in the Velcro closure and deviation from an antiparallel alignment constitute an uncommon variation. Two helices separated by a loop (~10 amino acids) follow the N-terminal β strands, with the first helix contacting Intu/Fuz and the second helix folding against the first blade of the β-propeller to which it connects. The second and third blades contain poorly resolved insertions of ~20 amino acids that are located on the solvent-exposed side of Wdpcp. These insertions could provide interaction sites with other factors such as Jbts17. The sixth and seventh blades are linked by an extended loop (~20 amino acids) while the last (seventh) blade connects to the C-terminal α-solenoid with a short loop of ~10 amino acids. Both loops are involved in interactions with Intu, which are discussed in detail below. Eight helices of the α-solenoid are resolved that arrange into four bundles each containing two antiparallel aligned helices (fig. S5A).Intu and Fuz were predicted to each have three longin-like domains (LDs). The structure confirms the prediction that LD1 of Intu and Fuz adopts a canonical longin fold, LD2 adopts a roadblock longin-like fold, and LD3 adopts a lamtor longin-like fold (28). Longin, roadblock, and lamtor domains are closely related and adopt a ββαβββαα, αββαβββα, and ββαβββα fold, respectively. In both Intu and Fuz, the LDs are arranged in a nonsymmetric triangular manner with each domain forming contacts to the other two domains (fig. S5, B and C). Helices α2 and α3 of LD1 stack perpendicularly on α1 of LD2. The β sheets of LD2 and LD3 form a continuous β sheet, but the connecting strand between LD2 and LD3 crosses the sheets in a diagonal manner, allowing only a small β-bridge. Helix α1 of Intu LD3 has almost double the length of corresponding helices found in the other LDs and substantially protrudes from the complex. This helix also forms contacts to α2 of LD1. In the case of Intu, an α helix, found in the linker region between LD1 and LD2, is located in the center of the LD trimer. This helix runs parallel to the extended helix α1 of LD3 and further stabilizes the interaction between the three domains. In Fuz, this helix is notably shorter and consists of only one turn. LD2 and LD3 of Intu are separated by a long linker region and about 70 amino acids were not resolved in the map. Additional density was found close to the N terminus of Intu showing features of another β strand aligning with the β sheet of LD2. Because of its short length, it was not possible to unambiguously identify the sequence but we speculate that the strand originates from the part preceding the PDZ domain that is consistent with the AlphaFold (33) prediction (fig. S4D). The three LDs of Fuz show an overall twofold pseudo-symmetrical arrangement with respect to the Intu LDs (fig. S5D) and can be superimposed with an RMSD of 5.7 Å despite low sequence similarity (fig. S5E). The structural conservation is in line with the notion that Intu and Fuz may have originated from a common homodimer-forming ancestor by gene duplication (28).The core structure of Rsg1 follows the typical small GTPase fold characterized by a central six-stranded concave β sheet surrounded by five α helices (Fig. 3A). In addition, Rsg1 contains an N-terminal extension of ~50 amino acids preceding the first β strand that is partially resolved (amino acids 11 to 37) and contains an additional α helix (α0). Ηelix α0 packs against the second and third β strands of the extended interswitch in a parallel manner and the N-terminal tip comes close to the loop between the G1 and G2 motifs. The third β strand, which is part of the interswitch, is disrupted by a proline and the N-terminal half interacts with the G1 motif, while the C-terminal half aligns with helix α0. In GTPases of the Arf family, an N-terminal α helix contacts the short interswitch in a perpendicular manner and has been implicated in regulatory functions (“interswitch toggle”) (34). Since the engagement with the long interswitch of Rsg1 is somewhat different (parallel compared to perpendicular), the putative regulatory mechanism might be distinct. Similar to the related REM GTPases, the Rsg1 G3 motif has an atypical sequence (DCGES) that deviates from the canonical DXXGQ motif found in most small GTPases and lacks the catalytically important glutamine. The EM map strongly indicates the presence of a nucleotide (Fig. 3B) and superposition with other GTPases bound to guanosine triphosphate (GTP) analogs (35) shows that a triphosphate would explain the density best (Fig. 3C). A coordinated magnesium could not be identified, probably owing to the limited local resolution. Since the nucleotide specificity of Rsg1 was obscure (30), we characterized endogenously bound nucleotides in purified Rsg1 and Fuz-Rsg1 using reversed-phase high-performance liquid chromatography (HPLC). In both cases, we could identify GTP but not GDP as the bound nucleotide (Fig. 3D), suggesting a low intrinsic GTP hydrolysis rate. An explanation at the molecular level could be the substitution of catalytic glutamine by serine. This would be in line with the atypical GTPases IFT22 and IFT27 bearing a serine at the same position and for which low GTPase activities have been shown (36, 37). In addition, the histidine 84 side chain (located in the switch I loop) is oriented toward the γ-phosphate and presumably contributes to stabilization of the negative charge (Fig. 3B). Of note, the histidine is not in a position where it could coordinate the catalytic water as seen in EF-Tu (38) and rather overlaps with the position of arginine fingers.Fig. 3.Nucleotide content of MmRsg1.(A) Model of MmRsg1. (B) Map density in MmRsg1 nucleotide pocket. Putative GTP is shown in green. (C) Overlay of Rsg1 (green) and Rad (orange; 3Q72) nucleotide conformations (Rad was superposed on Rsg1). Map is locally sharpened and contoured at 0.257. (D) Reverse-phase HPLC analysis of bound nucleotides in Rsg1 and Fuz-Rsg1 [purified proteins shown in insets and labeled F (Fuz) and R (Rsg1)]. Pure GDP and GTP (100 μM) were run as standards in triplicate. Rsg1 (40 μM) and Fuz-Rsg1 (73 μM) were run in triplicate and quadruplicate, respectively.

Structure of the mammalian CPLANE complex

We reconstituted the HsCPLANE complex (HsWdpcp-HsIntu-HsFuz) and the MmCPLANE complex bound to the Rab-like GTPase MmRsg1 (MmWdpcp-MmIntu-MmFuz-MmRsg1) using full-length recombinant proteins (Fig. 1, A and B, and fig. S1, A and B). Both complexes are stable in vitro at least up to 1 M salt and we detected binary interactions for MmWdpcp-MmIntu, MmIntu-MmFuz, and MmFuz-MmRsg1 pairs (fig. S1, C to F). Using single-particle cryo-EM, we obtained reconstructions (table S1) at 3.35 Å resolution for both the human (Fig. 1C and fig. S2, A to E) and the mouse complex (Fig. 1D and fig. S3, A to E). These maps allowed us to build models that include the majority of the primary protein sequence (Fig. 1, E and F, and fig. S4, A and B). Notably, we did not observe densities for the N-terminal ~300 residues of HsIntu/MmIntu, which contain a predicted PDZ domain (HsIntu 185 to 263 amino acids) as well as for the C-terminal ~100 residues of HsWdpcp/MmWdpcp; both Intu and Wdpcp unresolved parts are predicted to contain intrinsically disordered regions (RaptorX) (31). Overall, the human and mouse complexes are remarkably similar with a root mean square deviation (RMSD) of 3.8 Å over 93% of the common subunits’ residues (fig. S4C); given its higher local resolution of important parts of the structure, we describe in detail the MmCPLANE-MmRsg1 complex (Fig. 2).

Cryo-EM structures of the mammalian CPLANE complex.

(A) Domain organization of HsCPLANE subunits. WD40-repeat (WDR), PDZ, and longin-like domains (LD1 to LD3) are indicated. (B) Domain organization of MmCPLANE subunits and MmRsg1. WD40-repeat (WDR), PDZ, longin-like (LD1 to LD3), and GTPase domains are indicated. (C) Cryo-EM map of HsCPLANE in two views. Map is locally sharpened and contoured at 0.160. (D) Cryo-EM map of MmCPLANE bound to MmRsg1 in two views. Map is locally sharpened and contoured at 0.160. (E) Model of the HsCPLANE complex in cartoon representation in two views. (F) Model of the MmCPLANE-MmRsg1 complex in cartoon representation in two views.

CPLANE subunits have extensive interfaces.

Cartoon representation of the MmCPLANE-MmRsg1 model. The three interfaces between subunits (Wdpcp-Intu-Fuz, Intu-Fuz, and Fuz-Rsg1) are indicated by dashed boxes. Domains of a subunit are colored in different shades. Close-up views of interfaces are shown and secondary structure elements involved in interactions are indicated (see fig. S5 for complete annotation scheme). Wdpcp-Intu-Fuz interface (1): WDR, Helical domain of Wdpcp, LD1 to LD3 of Intu, and LD3 of Fuz are shown. Intu-Fuz interface (2): LD1 and LD3 of Intu and Fuz are shown. Fuz-Rsg1 interface (3): LD1 and LD2 of Fuz and RSG1 are shown. G elements 1 to 3 [switch I (G2) and II (G3) loops] are indicated.

Wdpcp folds into a seven-bladed β-propeller belonging to the WD40 family, followed by an array of α helices that can be classified as α-solenoid (fig. S5A). Compared to mouse, the human Wdpcp has an N-terminal extension of ~35 residues that is not resolved in the map. The N-terminal end of Wdpcp complements the three-stranded β sheet of the last blade first by an antiparallel (fourth strand) and then by a small parallel β strand aligning with the C-terminal half of the fourth strand. In addition, the N-terminal half of the fourth strand aligns with the detached fourth strand of blade 6. The combination of N- and C-terminal strands in one blade is a regular pattern called “Velcro closure” and helps to stabilize the fold of the β-propeller (32). On the other hand, participation of strands from multiple blades in the Velcro closure and deviation from an antiparallel alignment constitute an uncommon variation. Two helices separated by a loop (~10 amino acids) follow the N-terminal β strands, with the first helix contacting Intu/Fuz and the second helix folding against the first blade of the β-propeller to which it connects. The second and third blades contain poorly resolved insertions of ~20 amino acids that are located on the solvent-exposed side of Wdpcp. These insertions could provide interaction sites with other factors such as Jbts17. The sixth and seventh blades are linked by an extended loop (~20 amino acids) while the last (seventh) blade connects to the C-terminal α-solenoid with a short loop of ~10 amino acids. Both loops are involved in interactions with Intu, which are discussed in detail below. Eight helices of the α-solenoid are resolved that arrange into four bundles each containing two antiparallel aligned helices (fig. S5A).

Intu and Fuz were predicted to each have three longin-like domains (LDs). The structure confirms the prediction that LD1 of Intu and Fuz adopts a canonical longin fold, LD2 adopts a roadblock longin-like fold, and LD3 adopts a lamtor longin-like fold (28). Longin, roadblock, and lamtor domains are closely related and adopt a ββαβββαα, αββαβββα, and ββαβββα fold, respectively. In both Intu and Fuz, the LDs are arranged in a nonsymmetric triangular manner with each domain forming contacts to the other two domains (fig. S5, B and C). Helices α2 and α3 of LD1 stack perpendicularly on α1 of LD2. The β sheets of LD2 and LD3 form a continuous β sheet, but the connecting strand between LD2 and LD3 crosses the sheets in a diagonal manner, allowing only a small β-bridge. Helix α1 of Intu LD3 has almost double the length of corresponding helices found in the other LDs and substantially protrudes from the complex. This helix also forms contacts to α2 of LD1. In the case of Intu, an α helix, found in the linker region between LD1 and LD2, is located in the center of the LD trimer. This helix runs parallel to the extended helix α1 of LD3 and further stabilizes the interaction between the three domains. In Fuz, this helix is notably shorter and consists of only one turn. LD2 and LD3 of Intu are separated by a long linker region and about 70 amino acids were not resolved in the map. Additional density was found close to the N terminus of Intu showing features of another β strand aligning with the β sheet of LD2. Because of its short length, it was not possible to unambiguously identify the sequence but we speculate that the strand originates from the part preceding the PDZ domain that is consistent with the AlphaFold (33) prediction (fig. S4D). The three LDs of Fuz show an overall twofold pseudo-symmetrical arrangement with respect to the Intu LDs (fig. S5D) and can be superimposed with an RMSD of 5.7 Å despite low sequence similarity (fig. S5E). The structural conservation is in line with the notion that Intu and Fuz may have originated from a common homodimer-forming ancestor by gene duplication (28).

The core structure of Rsg1 follows the typical small GTPase fold characterized by a central six-stranded concave β sheet surrounded by five α helices (Fig. 3A). In addition, Rsg1 contains an N-terminal extension of ~50 amino acids preceding the first β strand that is partially resolved (amino acids 11 to 37) and contains an additional α helix (α0). Ηelix α0 packs against the second and third β strands of the extended interswitch in a parallel manner and the N-terminal tip comes close to the loop between the G1 and G2 motifs. The third β strand, which is part of the interswitch, is disrupted by a proline and the N-terminal half interacts with the G1 motif, while the C-terminal half aligns with helix α0. In GTPases of the Arf family, an N-terminal α helix contacts the short interswitch in a perpendicular manner and has been implicated in regulatory functions (“interswitch toggle”) (34). Since the engagement with the long interswitch of Rsg1 is somewhat different (parallel compared to perpendicular), the putative regulatory mechanism might be distinct. Similar to the related REM GTPases, the Rsg1 G3 motif has an atypical sequence (DCGES) that deviates from the canonical DXXGQ motif found in most small GTPases and lacks the catalytically important glutamine. The EM map strongly indicates the presence of a nucleotide (Fig. 3B) and superposition with other GTPases bound to guanosine triphosphate (GTP) analogs (35) shows that a triphosphate would explain the density best (Fig. 3C). A coordinated magnesium could not be identified, probably owing to the limited local resolution. Since the nucleotide specificity of Rsg1 was obscure (30), we characterized endogenously bound nucleotides in purified Rsg1 and Fuz-Rsg1 using reversed-phase high-performance liquid chromatography (HPLC). In both cases, we could identify GTP but not GDP as the bound nucleotide (Fig. 3D), suggesting a low intrinsic GTP hydrolysis rate. An explanation at the molecular level could be the substitution of catalytic glutamine by serine. This would be in line with the atypical GTPases IFT22 and IFT27 bearing a serine at the same position and for which low GTPase activities have been shown (36, 37). In addition, the histidine 84 side chain (located in the switch I loop) is oriented toward the γ-phosphate and presumably contributes to stabilization of the negative charge (Fig. 3B). Of note, the histidine is not in a position where it could coordinate the catalytic water as seen in EF-Tu (38) and rather overlaps with the position of arginine fingers.

Nucleotide content of MmRsg1.

(A) Model of MmRsg1. (B) Map density in MmRsg1 nucleotide pocket. Putative GTP is shown in green. (C) Overlay of Rsg1 (green) and Rad (orange; 3Q72) nucleotide conformations (Rad was superposed on Rsg1). Map is locally sharpened and contoured at 0.257. (D) Reverse-phase HPLC analysis of bound nucleotides in Rsg1 and Fuz-Rsg1 [purified proteins shown in insets and labeled F (Fuz) and R (Rsg1)]. Pure GDP and GTP (100 μM) were run as standards in triplicate. Rsg1 (40 μM) and Fuz-Rsg1 (73 μM) were run in triplicate and quadruplicate, respectively.

CPLANE subunits have extensive interfacesBoth HsCPLANE and MmCPLANE complexes adopt a crescent-like architecture that is particularly obvious in the mouse complex (Fig. 1, C and D). On one side of the complex lies Wdpcp that mainly contacts Intu with a large interface area of 2609 Å2. The main interaction platform is provided by Intu LD2, which forms extensive contacts with the α-helical domain of Wdpcp (Fig. 2 and fig. S6, A and D). Helices α3 and α5 of Wdpcp stack in a perpendicular manner onto α0, parts of β1, and loops between β2 and α1 as well as β5 and α2 of Intu LD2. There is a strong charge complementarity between Wdpcp (positively charged patch) and Intu (negatively charged patch; fig. S6D). In particular, Wdpcp H466, R502, R544, and K541 are within hydrogen bonding distance of Intu main-chain carbonyls from L940, L942/S532, A499, and the side chain of D496, respectively. On the distal end of the Wdpcp helical domain, a poorly resolved loop region between α0 and β1 of Intu LD2 folds onto Wdpcp helix α7 including hydrophobic and polar contacts (fig. S6E). Another interaction hotspot is centered around the loop connecting the β-propeller and helical domains of Wdpcp. The loop inserts into a pocket formed between Intu LD2 and LD3, and its hydrophobic tip is stacking against a hydrophobic patch inside of the pocket (fig. S6C). In detail, Wdpcp L450 is surrounded by Intu L940, I813, and V481 and Wdpcp I449 packs against the alkyl chain of Intu K935. In addition to hydrophobic contacts, Wdpcp R452 likely forms hydrogen bonds with main-chain carbonyls of Intu P654 and S653 at the entry of the pocket. While contributing to the pocket surface, the C terminus of Intu at the same time meanders around α1 of the Wdpcp helical domain. On the other far end of the interface, a ternary hub is formed between the Wdpcp β-propeller domain, Fuz LD2 and Intu LD3 (Fig. 2 and fig. S6B). The first α helix located in the N-terminal region of Wdpcp and an extended linker between β-propeller blades 6 and 7 form contacts to helices α2 of the LD2 of Intu and Fuz. The density suggests putative salt bridges between Wdpcp R22 and Fuz D295 as well as between Fuz R401 and Intu D923. The interaction between Wdpcp and Fuz contributes an additional contact area of 351 Å2 and expands the interface to a length of almost 80 Å. In summary, an extensive and intertwined interface is formed between Wdpcp and Intu-Fuz, relying on charge and shape complementarity. Intriguingly, the Wdpcp α-solenoid and Intu longin domain interaction is found in cellular trafficking complexes such as the AP2 clathrin adaptor (39) or the β/δ-COPI complex (40). In both examples, the orientation between the α-solenoid and bound longin domain is similar to Wdpcp-Intu; however, the α-solenoid is remarkably extended and encloses the longin domain.Antipodal to the Wdpcp-Intu interface, Intu contacts Fuz with a similarly large buried surface area of 1898 Å2 (Fig. 2). Formation of the Intu-Fuz heterodimer is mediated by LD1 and LD3. In both cases (Intu_LD1-Fuz_LD1 and Intu_LD3-Fuz_LD3), the LDs interact via their third β strand (β3) to form a continuous β sheet as seen in previous structures (Fig. 2) (41). In case of the LD1 Intu-Fuz heterodimer, the outward-facing helices α1 also align in a parallel manner and form additional, mostly hydrophobic contacts (fig. S6F). The LD3 Intu-Fuz heterodimer is in the same orientation as the LD1 dimer, and thus, helices α1 are buried in the center and form less contacts (fig. S6F). Helices α3 are facing outward and come into reach at their N-terminal tips (~5.8 Å backbone distance) but do not seem to form pronounced contacts. To the best of our knowledge, this is the first structure of a complete mammalian hexalongin scaffold and thus provides a blueprint for homologous complexes of the HerMon family such as Mon1-Czz1 and HPS1-HPS4 (28).

CPLANE subunits have extensive interfaces

Both HsCPLANE and MmCPLANE complexes adopt a crescent-like architecture that is particularly obvious in the mouse complex (Fig. 1, C and D). On one side of the complex lies Wdpcp that mainly contacts Intu with a large interface area of 2609 Å2. The main interaction platform is provided by Intu LD2, which forms extensive contacts with the α-helical domain of Wdpcp (Fig. 2 and fig. S6, A and D). Helices α3 and α5 of Wdpcp stack in a perpendicular manner onto α0, parts of β1, and loops between β2 and α1 as well as β5 and α2 of Intu LD2. There is a strong charge complementarity between Wdpcp (positively charged patch) and Intu (negatively charged patch; fig. S6D). In particular, Wdpcp H466, R502, R544, and K541 are within hydrogen bonding distance of Intu main-chain carbonyls from L940, L942/S532, A499, and the side chain of D496, respectively. On the distal end of the Wdpcp helical domain, a poorly resolved loop region between α0 and β1 of Intu LD2 folds onto Wdpcp helix α7 including hydrophobic and polar contacts (fig. S6E). Another interaction hotspot is centered around the loop connecting the β-propeller and helical domains of Wdpcp. The loop inserts into a pocket formed between Intu LD2 and LD3, and its hydrophobic tip is stacking against a hydrophobic patch inside of the pocket (fig. S6C). In detail, Wdpcp L450 is surrounded by Intu L940, I813, and V481 and Wdpcp I449 packs against the alkyl chain of Intu K935. In addition to hydrophobic contacts, Wdpcp R452 likely forms hydrogen bonds with main-chain carbonyls of Intu P654 and S653 at the entry of the pocket. While contributing to the pocket surface, the C terminus of Intu at the same time meanders around α1 of the Wdpcp helical domain. On the other far end of the interface, a ternary hub is formed between the Wdpcp β-propeller domain, Fuz LD2 and Intu LD3 (Fig. 2 and fig. S6B). The first α helix located in the N-terminal region of Wdpcp and an extended linker between β-propeller blades 6 and 7 form contacts to helices α2 of the LD2 of Intu and Fuz. The density suggests putative salt bridges between Wdpcp R22 and Fuz D295 as well as between Fuz R401 and Intu D923. The interaction between Wdpcp and Fuz contributes an additional contact area of 351 Å2 and expands the interface to a length of almost 80 Å. In summary, an extensive and intertwined interface is formed between Wdpcp and Intu-Fuz, relying on charge and shape complementarity. Intriguingly, the Wdpcp α-solenoid and Intu longin domain interaction is found in cellular trafficking complexes such as the AP2 clathrin adaptor (39) or the β/δ-COPI complex (40). In both examples, the orientation between the α-solenoid and bound longin domain is similar to Wdpcp-Intu; however, the α-solenoid is remarkably extended and encloses the longin domain.

Antipodal to the Wdpcp-Intu interface, Intu contacts Fuz with a similarly large buried surface area of 1898 Å2 (Fig. 2). Formation of the Intu-Fuz heterodimer is mediated by LD1 and LD3. In both cases (Intu_LD1-Fuz_LD1 and Intu_LD3-Fuz_LD3), the LDs interact via their third β strand (β3) to form a continuous β sheet as seen in previous structures (Fig. 2) (41). In case of the LD1 Intu-Fuz heterodimer, the outward-facing helices α1 also align in a parallel manner and form additional, mostly hydrophobic contacts (fig. S6F). The LD3 Intu-Fuz heterodimer is in the same orientation as the LD1 dimer, and thus, helices α1 are buried in the center and form less contacts (fig. S6F). Helices α3 are facing outward and come into reach at their N-terminal tips (~5.8 Å backbone distance) but do not seem to form pronounced contacts. To the best of our knowledge, this is the first structure of a complete mammalian hexalongin scaffold and thus provides a blueprint for homologous complexes of the HerMon family such as Mon1-Czz1 and HPS1-HPS4 (28).

CPLANE binds to Rsg1 in an effector-like mannerRsg1 lies on the other end of the complex and interacts solely with Fuz. Of note, the Fuz-Rsg1 interaction was stable in vitro without the addition of exogenous nucleotides. The Fuz-Rsg1 interface spans an area of 1099 Å2 and consists of Fuz LD1 and LD2 engaging the Rsg1 switch I + II regions, a loop between α0 and β1, and the interswitch region that is part of the central β sheet of RSG1 (Fig. 2). Notably, a β sheet completion is observed between β3 of Fuz LD2 and β2 of RSG1 (Fig. 4A). The density suggests that the main-chain hydrogen bonds are extended by a hydrogen bond network between Fuz Q215, Y220 and RSG1 T93, Q91 side chains (Fig. 4A). Nearby, the loop connecting RSG1 α0 to β1 is running close to the β-hairpin between β4 and β5 of Fuz LD2 and presumably forms a short β-bridge. The solvent-exposed interswitch β2 often acts as a platform for effector binding (42) or, in some cases, is contacted by the switch I loop when Rab GTPases are bound by a guanine nucleotide exchange factor (GEF). A common feature of GEFs is remodeling of the switch I + II loops to lower the affinity for nucleotide. This mechanism is observed in the homologous hexalongin complex Mon1-Czz1 bound to YPT7 (41), where the switch I loop is dislodged from the active site and forms a small β-bridge with the interswitch. This is in strong contrast to the CPLANE-Rsg1 complex that shows a substantially different overall binding mode where the interswitch is occupied by Fuz and switch I is ordered around the nucleotide as expected for the GTP-bound state (Fig. 4B). Comparison with other longin domain containing GEF/GTPase complexes, such as DENND1B:Rab35 (43), did not reveal a comparable binding mode. In addition, no indications for other nucleotide exchange mechanisms, such as the insertion of residues into the pocket by the GEF, are evident from the EM map. These structural observations suggest that Intu-Fuz is not a GEF for RSG1.Fig. 4.CPLANE interaction with Rsg1.(A) β sheet completion between the third β strand of Fuz LD2 and the interswitch (InSw) of Rsg1. (B) Interaction of Rsg1 switch I + II with Fuz helix α3. RSG1 H85, E86, and K123 side chains are approaching helix α3. G2 H84 is potentially interacting with the phosphate groups of the nucleotide (green). Conserved G2 T87 and G3 D116 as well as G3 S120 substituting for a conserved catalytic glutamine are shown. Residues lacking sufficient side-chain density are indicated by spheres.Rsg1 switch I and a helical part of switch II occupy a shallow partially hydrophobic groove between LD1 and LD2 and are predominantly in contact with helix α3 of Fuz LD1. Helix α3 is curved in both Rsg1-bound (in mouse CPLANE) and -unbound Fuz (in human CPLANE) and contributes to interface shape complementarity; interestingly, this Fuz helix contains a conserved arginine residue (R129) often provided in trans by GTPase-activating proteins (GAPs; Fig. 4B). The cryo-EM density, however, does not suggest that R129 inserts into the active site and modeling of different rotamer conformations shows that the arginine cannot get close enough to properly align with the phosphate groups. Moreover, accessibility of the phosphate groups is hindered by switch I histidine H84 and presumably by the poorly resolved E86 that could come into hydrogen bond distance (Fig. 4B). Comparison to other GTPase-GAP complexes shows that a backbone shift by at least 3 Å would be necessary to bring the arginine into a competent position (44) and entry to the active site would need to be opened by rearrangement of switch I residues. In other GTP-bound RGK and Rab GTPases (35), the switch I + II loops are completely or partially disordered in the absence of an interaction partner; thus, binding of Fuz might contribute to a stabilization of these G elements as it has been described for IFT22 bound to IFT74/81 (36).The binding mode of known longin-GTPase complexes (45) is notably different from the observed Fuz-Rsg1 interface. We therefore compared the Fuz LD1/Rsg1 complex against the PDB for similar interfaces using the TopSearch webserver (46); this yielded effector and GAP complexes as top ranked hits that, however, do not involve interactions with longin domains. A common feature of the top hits is an α helix at a similar position to α3 of Fuz LD2 that is engaging Rsg1 switch I + II. Overall, on the basis of these observations, we propose that Fuz and hence the CPLANE complex predominantly functions as an effector of Rsg1. We do not rule out an additional function as co-GAP by stabilization of the switch I + II loops, but more biochemical evidence is needed to support this notion.

CPLANE binds to Rsg1 in an effector-like manner

Rsg1 lies on the other end of the complex and interacts solely with Fuz. Of note, the Fuz-Rsg1 interaction was stable in vitro without the addition of exogenous nucleotides. The Fuz-Rsg1 interface spans an area of 1099 Å2 and consists of Fuz LD1 and LD2 engaging the Rsg1 switch I + II regions, a loop between α0 and β1, and the interswitch region that is part of the central β sheet of RSG1 (Fig. 2). Notably, a β sheet completion is observed between β3 of Fuz LD2 and β2 of RSG1 (Fig. 4A). The density suggests that the main-chain hydrogen bonds are extended by a hydrogen bond network between Fuz Q215, Y220 and RSG1 T93, Q91 side chains (Fig. 4A). Nearby, the loop connecting RSG1 α0 to β1 is running close to the β-hairpin between β4 and β5 of Fuz LD2 and presumably forms a short β-bridge. The solvent-exposed interswitch β2 often acts as a platform for effector binding (42) or, in some cases, is contacted by the switch I loop when Rab GTPases are bound by a guanine nucleotide exchange factor (GEF). A common feature of GEFs is remodeling of the switch I + II loops to lower the affinity for nucleotide. This mechanism is observed in the homologous hexalongin complex Mon1-Czz1 bound to YPT7 (41), where the switch I loop is dislodged from the active site and forms a small β-bridge with the interswitch. This is in strong contrast to the CPLANE-Rsg1 complex that shows a substantially different overall binding mode where the interswitch is occupied by Fuz and switch I is ordered around the nucleotide as expected for the GTP-bound state (Fig. 4B). Comparison with other longin domain containing GEF/GTPase complexes, such as DENND1B:Rab35 (43), did not reveal a comparable binding mode. In addition, no indications for other nucleotide exchange mechanisms, such as the insertion of residues into the pocket by the GEF, are evident from the EM map. These structural observations suggest that Intu-Fuz is not a GEF for RSG1.

CPLANE interaction with Rsg1.

(A) β sheet completion between the third β strand of Fuz LD2 and the interswitch (InSw) of Rsg1. (B) Interaction of Rsg1 switch I + II with Fuz helix α3. RSG1 H85, E86, and K123 side chains are approaching helix α3. G2 H84 is potentially interacting with the phosphate groups of the nucleotide (green). Conserved G2 T87 and G3 D116 as well as G3 S120 substituting for a conserved catalytic glutamine are shown. Residues lacking sufficient side-chain density are indicated by spheres.

Rsg1 switch I and a helical part of switch II occupy a shallow partially hydrophobic groove between LD1 and LD2 and are predominantly in contact with helix α3 of Fuz LD1. Helix α3 is curved in both Rsg1-bound (in mouse CPLANE) and -unbound Fuz (in human CPLANE) and contributes to interface shape complementarity; interestingly, this Fuz helix contains a conserved arginine residue (R129) often provided in trans by GTPase-activating proteins (GAPs; Fig. 4B). The cryo-EM density, however, does not suggest that R129 inserts into the active site and modeling of different rotamer conformations shows that the arginine cannot get close enough to properly align with the phosphate groups. Moreover, accessibility of the phosphate groups is hindered by switch I histidine H84 and presumably by the poorly resolved E86 that could come into hydrogen bond distance (Fig. 4B). Comparison to other GTPase-GAP complexes shows that a backbone shift by at least 3 Å would be necessary to bring the arginine into a competent position (44) and entry to the active site would need to be opened by rearrangement of switch I residues. In other GTP-bound RGK and Rab GTPases (35), the switch I + II loops are completely or partially disordered in the absence of an interaction partner; thus, binding of Fuz might contribute to a stabilization of these G elements as it has been described for IFT22 bound to IFT74/81 (36).

The binding mode of known longin-GTPase complexes (45) is notably different from the observed Fuz-Rsg1 interface. We therefore compared the Fuz LD1/Rsg1 complex against the PDB for similar interfaces using the TopSearch webserver (46); this yielded effector and GAP complexes as top ranked hits that, however, do not involve interactions with longin domains. A common feature of the top hits is an α helix at a similar position to α3 of Fuz LD2 that is engaging Rsg1 switch I + II. Overall, on the basis of these observations, we propose that Fuz and hence the CPLANE complex predominantly functions as an effector of Rsg1. We do not rule out an additional function as co-GAP by stabilization of the switch I + II loops, but more biochemical evidence is needed to support this notion.

CPLANE proteins bind phospholipidsWe noticed that the domains of CPLANE subunits are characteristic of proteins involved in vesicle trafficking and are commonly found in protocoatomer-associated complexes (47). These domains include the β-propeller/α-solenoid of Wdpcp and the LDs of Intu and Fuz. Moreover, Rsg1 is a Rab-like GTPase and hence expected to be involved in membrane trafficking (47). We were thus prompted to determine whether CPLANE proteins bind phospholipids. Both HsWdpcp-HsIntu-HsFuz and MmWdpcp-MmIntu-MmFuz-MmRsg1 complexes bind to phosphoinositides (PIPs) immobilized on membranes. CPLANE complexes interacted solely with PIPs having a single inositol phosphate and exhibited a clear preference for PI(3)P, followed by PI(5)P and PI(4)P (Fig. 5, A and B). We next asked which CPLANE protein mediates the lipid binding and repeated the assay with individual mouse proteins. Unexpectedly, all three mouse CPLANE subunits (MmWdpcp, MmIntu, and MmFuz) but not MmRsg1 were found to bind PIPs (Fig. 5, C to F). Similar to the CPLANE complex, individual subunits bind PI(3)P with highest affinity, followed by PI(5)P and PI(4)P. MmIntu displayed the strongest binding, followed by MmWdpcp and MmFuz (Fig. 5, C to F). We next mapped the PIP binding domains in MmIntu and MmWdpcp. MmIntu uses the N-terminal ~300 amino acids to bind PIPs, with a 47– to 267–amino acid construct being competent for binding; in addition, we detected weaker PIP binding by the Intu LDs (341 to 942 amino acids; fig. S7A). In contrast, MmWdpcp PIP binding is mediated by the C-terminal ~60 amino acids (fig. S7B); within this fragment, longer deletions show progressively lower binding, indicating an extensive binding interface (fig. S7C).Fig. 5.CPLANE proteins bind PIPs.(A) HsCPLANE complex binding to PIP strip. Input and Western blot (3 nM) (equal exposures) for Flag-tagged (left) and untagged (right) Wdpcp are shown. (B) MmCPLANE + MmRsg1 complex binding to PIP strip. Input and Western blot (4 nM) (equal exposures) for Flag-tagged (left) and untagged (right) Wdpcp are shown. (C) MmIntu binding to PIP strip. Input and Western blot (5 nM) for Flag-tagged Intu are shown. (D) MmWdpcp binding to PIP strip. Input and Western blot (20 nM) for Flag-tagged Wdpcp are shown. (E) MmFuz binding to PIP strip. Input and Western blot (20 nM) for Flag-tagged Fuz are shown. (F) MmRsg1 binding to PIP strip. Input and Western blot (20 nM) for Flag-tagged Rsg1 are shown. Blots in (E) and (F) have equal exposures.

CPLANE proteins bind phospholipids

We noticed that the domains of CPLANE subunits are characteristic of proteins involved in vesicle trafficking and are commonly found in protocoatomer-associated complexes (47). These domains include the β-propeller/α-solenoid of Wdpcp and the LDs of Intu and Fuz. Moreover, Rsg1 is a Rab-like GTPase and hence expected to be involved in membrane trafficking (47). We were thus prompted to determine whether CPLANE proteins bind phospholipids. Both HsWdpcp-HsIntu-HsFuz and MmWdpcp-MmIntu-MmFuz-MmRsg1 complexes bind to phosphoinositides (PIPs) immobilized on membranes. CPLANE complexes interacted solely with PIPs having a single inositol phosphate and exhibited a clear preference for PI(3)P, followed by PI(5)P and PI(4)P (Fig. 5, A and B). We next asked which CPLANE protein mediates the lipid binding and repeated the assay with individual mouse proteins. Unexpectedly, all three mouse CPLANE subunits (MmWdpcp, MmIntu, and MmFuz) but not MmRsg1 were found to bind PIPs (Fig. 5, C to F). Similar to the CPLANE complex, individual subunits bind PI(3)P with highest affinity, followed by PI(5)P and PI(4)P. MmIntu displayed the strongest binding, followed by MmWdpcp and MmFuz (Fig. 5, C to F). We next mapped the PIP binding domains in MmIntu and MmWdpcp. MmIntu uses the N-terminal ~300 amino acids to bind PIPs, with a 47– to 267–amino acid construct being competent for binding; in addition, we detected weaker PIP binding by the Intu LDs (341 to 942 amino acids; fig. S7A). In contrast, MmWdpcp PIP binding is mediated by the C-terminal ~60 amino acids (fig. S7B); within this fragment, longer deletions show progressively lower binding, indicating an extensive binding interface (fig. S7C).

CPLANE proteins bind PIPs.

(A) HsCPLANE complex binding to PIP strip. Input and Western blot (3 nM) (equal exposures) for Flag-tagged (left) and untagged (right) Wdpcp are shown. (B) MmCPLANE + MmRsg1 complex binding to PIP strip. Input and Western blot (4 nM) (equal exposures) for Flag-tagged (left) and untagged (right) Wdpcp are shown. (C) MmIntu binding to PIP strip. Input and Western blot (5 nM) for Flag-tagged Intu are shown. (D) MmWdpcp binding to PIP strip. Input and Western blot (20 nM) for Flag-tagged Wdpcp are shown. (E) MmFuz binding to PIP strip. Input and Western blot (20 nM) for Flag-tagged Fuz are shown. (F) MmRsg1 binding to PIP strip. Input and Western blot (20 nM) for Flag-tagged Rsg1 are shown. Blots in (E) and (F) have equal exposures.

Analysis of ciliopathy-associated mutationsMutations in CPLANE proteins are associated with ciliopathies and other developmental defects. In addition to frameshift changes that commonly result in premature stop codons, several missense point mutations have been reported (19, 25, 48, 49). We took advantage of our near-atomic HsCPLANE model and mapped 10 missense point mutations onto HsWdpcp, HsIntu, and HsFuz. With the exception of four mutations (HsWdpcp_L205F, HsWdpcp_S708F, HsIntu_A452T, and HsFuz_D354Y) that were located in unresolved parts, the remaining six could be assigned to map densities (Fig. 6A). Intriguingly, several disease-relevant mutations are located at subunit interfaces (see fig. S6, A to E, for mouse cognate residues). We wondered whether these different mutations affect assembly of the HsCPLANE complex. To answer this, we expressed all recombinant human proteins incorporating the mutations and assessed HsCPLANE complex formation in vitro. We failed to detect any obvious defects in expression of individual subunits or assembly of the HsCPLANE (fig. S8, A to D, F, and G). Moreover, mutations did not inhibit HsCPLANE subunits’ interaction with HsRsg1 (fig. S8, E and H). Mutations in HsRsg1 have not been reported yet, but the mouse pixiebob (pxb) mutant, which exhibits early embryonic lethality, was shown to be due to a valine-169–to–aspartic acid substitution in MmRsg1 (30). We therefore mapped this residue in our structure; notably, V169 is distant from the Fuz-Rsg1 interface and we verified that MmRsg1_V169D interacts with MmFuz in vitro although its expression is reduced compared to MmRsg1_WT (fig. S8, I and J). Given the lipid binding properties of CPLANE proteins, we then asked whether the ciliopathy mutations perturb PIP binding. To address this, we focused on the HsWdpcp_Ser708Phe mutation that is found in the PIP binding site of the unresolved Wdpcp C-terminal tail. This mutation was found to be heterozygous in a patient diagnosed with BBS; notably, no other BBS-related mutations were detected in the affected individual (25). Unexpectedly, the Wdpcp_S708F protein exhibited stronger binding than the wild-type (WT) on PIP strips, but its PIP specificity remained unchanged (Fig. 6B). We confirmed this result using arrays with a concentration gradient of PIPs. Consistent with the previous results, the S708F protein had clearly higher relative binding to PIPs as compared to WT (Fig. 6C). Collectively, these results indicate that the reported ciliopathy mutations do not affect assembly of the recombinant HsCPLANE complex but rather disrupt CPLANE functions such as PIP binding.Fig. 6.Disease-associated CPLANE mutations.(A) Cartoon representation of the HsCPLANE model. Missense mutations are shown as red (Wdpcp), blue (Intu), and orange (Fuz) spheres. Wdpcp (L205F and S708F), Intu (A452T), and Fuz (D354Y) are in unresolved parts of the complex. The associated genetic diseases are shown in the box: oral-facial-digital syndrome (OFDS), Meckel-Gruber syndrome (MKS), Bardet-Biedl syndrome (BBS), Joubert syndrome (JBTS), nephronophthisis (NPHP), short-rib polydactyly syndrome (SRPS), and neural tube defects (NTD). (B) HsWdpcp_485-746 (helical α-solenoid) binding to PIP strip for wild-type (WT) and S708F mutant. Input and Western blot (10 nM) (equal exposures) for Flag-tagged constructs are shown. (C) HsWdpcp_485-746 (helical α-solenoid) binding to PIP array (shown 25 to 100 pmol PIP spots) for WT and S708F mutant. Input and Western blot (12 nM) (equal exposures) for Flag-tagged constructs are shown.

Analysis of ciliopathy-associated mutations

Mutations in CPLANE proteins are associated with ciliopathies and other developmental defects. In addition to frameshift changes that commonly result in premature stop codons, several missense point mutations have been reported (19, 25, 48, 49). We took advantage of our near-atomic HsCPLANE model and mapped 10 missense point mutations onto HsWdpcp, HsIntu, and HsFuz. With the exception of four mutations (HsWdpcp_L205F, HsWdpcp_S708F, HsIntu_A452T, and HsFuz_D354Y) that were located in unresolved parts, the remaining six could be assigned to map densities (Fig. 6A). Intriguingly, several disease-relevant mutations are located at subunit interfaces (see fig. S6, A to E, for mouse cognate residues). We wondered whether these different mutations affect assembly of the HsCPLANE complex. To answer this, we expressed all recombinant human proteins incorporating the mutations and assessed HsCPLANE complex formation in vitro. We failed to detect any obvious defects in expression of individual subunits or assembly of the HsCPLANE (fig. S8, A to D, F, and G). Moreover, mutations did not inhibit HsCPLANE subunits’ interaction with HsRsg1 (fig. S8, E and H). Mutations in HsRsg1 have not been reported yet, but the mouse pixiebob (pxb) mutant, which exhibits early embryonic lethality, was shown to be due to a valine-169–to–aspartic acid substitution in MmRsg1 (30). We therefore mapped this residue in our structure; notably, V169 is distant from the Fuz-Rsg1 interface and we verified that MmRsg1_V169D interacts with MmFuz in vitro although its expression is reduced compared to MmRsg1_WT (fig. S8, I and J). Given the lipid binding properties of CPLANE proteins, we then asked whether the ciliopathy mutations perturb PIP binding. To address this, we focused on the HsWdpcp_Ser708Phe mutation that is found in the PIP binding site of the unresolved Wdpcp C-terminal tail. This mutation was found to be heterozygous in a patient diagnosed with BBS; notably, no other BBS-related mutations were detected in the affected individual (25). Unexpectedly, the Wdpcp_S708F protein exhibited stronger binding than the wild-type (WT) on PIP strips, but its PIP specificity remained unchanged (Fig. 6B). We confirmed this result using arrays with a concentration gradient of PIPs. Consistent with the previous results, the S708F protein had clearly higher relative binding to PIPs as compared to WT (Fig. 6C). Collectively, these results indicate that the reported ciliopathy mutations do not affect assembly of the recombinant HsCPLANE complex but rather disrupt CPLANE functions such as PIP binding.

Disease-associated CPLANE mutations.

(A) Cartoon representation of the HsCPLANE model. Missense mutations are shown as red (Wdpcp), blue (Intu), and orange (Fuz) spheres. Wdpcp (L205F and S708F), Intu (A452T), and Fuz (D354Y) are in unresolved parts of the complex. The associated genetic diseases are shown in the box: oral-facial-digital syndrome (OFDS), Meckel-Gruber syndrome (MKS), Bardet-Biedl syndrome (BBS), Joubert syndrome (JBTS), nephronophthisis (NPHP), short-rib polydactyly syndrome (SRPS), and neural tube defects (NTD). (B) HsWdpcp_485-746 (helical α-solenoid) binding to PIP strip for wild-type (WT) and S708F mutant. Input and Western blot (10 nM) (equal exposures) for Flag-tagged constructs are shown. (C) HsWdpcp_485-746 (helical α-solenoid) binding to PIP array (shown 25 to 100 pmol PIP spots) for WT and S708F mutant. Input and Western blot (12 nM) (equal exposures) for Flag-tagged constructs are shown.

DISCUSSIONHuman ciliopathies are an expanding class of multisystemic inherited diseases that are often caused by defects in cilium assembly. The CPLANE proteins Wdpcp, Intu, and Fuz are crucial for ciliogenesis and, when mutated, are associated with devastating ciliopathies; however, their structure and cellular functions have remained enigmatic (20).Here, we used recombinant human and mouse proteins to show that Wdpcp, Intu, and Fuz assemble in a physical unit that we call the CPLANE complex. Existence of this complex has been suggested by studies in Drosophila and murine cells but direct biochemical evidence was lacking (20). We used cryo-EM to elucidate structures of human CPLANE complex and of mouse CPLANE complex bound to Rsg1. The mammalian CPLANE complex adopts a crescent-like architecture and is characterized by extensive interfaces between subunits. Wdpcp and Rsg1 are located on the ends of the crescent and bind the central Intu-Fuz heterodimer on opposite sides. This creates a linear-like subunit interaction scheme that can be depicted as Wdpcp-Intu-Fuz-Rsg1. Our analysis indicates that Intu and Fuz are structural homologs that use two triplets of LDs to interact with each other and with the other subunits of the complex. We did not detect a direct Wdpcp-Fuz interaction in our in vitro assays, although these subunits share a modest interface in the Wdpcp-Intu-Fuz hub. It is thus possible that the Wdpcp-Fuz interaction is unstable in the absence of Intu. Intriguingly, the CPLANE complex in vitro interacts constitutively with the atypical GTPase Rsg1. Of the five (G1 to G5) motifs that characterize GTPases of the Ras-superfamily, Rsg1 only harbors canonical G1 and G2 while G3 to G5 are divergent (30). Despite the noncanonical G motifs, Rsg1 binds GTP and interacts with the CPLANE subunit Fuz while GTP-bound. Future experiments should clarify whether the CPLANE-Rsg1 interaction is nucleotide-independent or whether CPLANE is an effector of Rsg1 and hence binds only to GTP-loaded Rsg1. Moreover, Rsg1 lacks the catalytic glutamine and it remains to be seen if it exhibits GTPase activity in vivo and how that affects the interaction with the CPLANE complex.Mutations in CPLANE subunits have been linked to noxious developmental defects. Mapping these mutations on our structure did not reveal an informative pattern. To test whether these mutations affect CPLANE complex integrity, we expressed the mutated proteins recombinantly and tested their association. We did not, however, observe any clear impact on protein expression, assembly of the CPLANE complex, or interaction with Rsg1. We surmise that the penetrance of these mutations in vivo might be dependent on modifier genes (19, 25) or affect CPLANE functions that are not recapitulated by recombinant systems or our in vitro conditions. For instance, the Intu A452T and E500A mutations were previously shown to decrease the Intu/Fuz GEF activity toward Rab23 (28). Moreover, CPLANE subunits have been shown to have an extensive interactome (19, 27), and it is conceivable that mutations may affect these interactions in vivo. This is further supported by the fact that the MmRsg1_V169D lethal mutation does not compromise correct Rsg1 localization in vivo (30) or interaction with Fuz in vitro; it is hence possible that it impedes a critical interaction of Rsg1 with an unidentified protein.The domains found in CPLANE subunits are emblematic of vesicle trafficking proteins (47) and we thus assessed PIP binding for the CPLANE complex and its individual subunits. Notably, the CPLANE complex interacts with PI(3)P and to a lesser extent with PI(4)P and PI(5)P. This interaction is mainly provided by Intu but we also detected binding of Wdpcp and Fuz. Rsg1 exhibited PIP interaction only when bound to the CPLANE complex and not as individual protein. This is consistent with the predicted lack of a prenylation motif in the C terminus of Rsg1 and suggests that Rsg1 is brought to membranes by interacting with the CPLANE complex. Except for the PDZ domain of Intu, CPLANE proteins lack canonical lipid binding domains (50) and we thus mapped the required modules in Intu and Wdpcp. The Intu N-terminal region containing PDZ was indeed capable of PIP binding, but Intu LDs also exhibited weak binding. On the other hand, Wdpcp engages PIPs via the last 60 C-terminal residues. Of note, both main PIP-binding modules localize to the only major unresolved parts of the CPLANE complex, the N terminus of Intu and the C terminus of Wdpcp. We noticed that the HsWdpcp_S708F ciliopathy mutation is found in the Wdpcp PIP binding C-terminal tail and contributes additional hydrophobicity to this segment; notably, this mutation confers stronger PIP binding compared to WT and presumably provides a molecular explanation for the ensuing BBS. A parsimonious hypothesis would be that aberrant PI(3)P binding may result in defective vesicle trafficking that perturbs BBS-related protein functions. Such defects could either compromise IFT-A complex assembly and subsequently result in abnormal IFT-B, and hence BBSome ciliary trafficking, or affect other BBS proteins that localize in the cilium transition zone (5, 19, 51). While the mechanistic details remain to be defined, this represents one of the first examples that link perturbed PIP binding with ciliopathies (5). The main ligand of CPLANE, PI(3)P, is an endosomal marker and has not yet been detected in mature cilia. Instead, the membrane of the cilium proper mainly contains PI(4)P whereas PI(4,5)P2 and PI(3,4,5)P3 are restricted to the transition zone (4). PI(3)P has, however, been reported in a vesicular compartment at the cilia base termed pericentriolar recycling endosome (PRE). This local pool of PI(3)P in PRE is thought to be required for activation of the Rab11/Rab8 cascade that culminates in cilium assembly (52). Given the PIPs binding of CPLANE proteins and the CV-stage arrest of CPLANE mutants during ciliogenesis, it is tempting to speculate that the CPLANE complex regulates trafficking of PI(3)P-containing vesicles to the nascent CV to promote membrane growth and/or deliver IFT-A proteins. These vesicles could originate at PRE or another PI(3)P-rich compartment. This hypothesis is supported by the CPLANE interactome (19), which includes IFT-A subunits and proteins involved in vesicle transport and fusion; moreover, IFT-A proteins can associate with vesicles (51) and abnormal vesicle accumulation has been observed in mutants of CPLANE proteins or downstream effectors (21, 26). The CPLANE complex could, for example, use its multiple PI(3)P binding sites to function as tether and bridge vesicles to target membranes before fusion (53). An important prediction of this model would be that CPLANE operates on PI(3)P-rich vesicles with an IFT-A–dependent coat (51). The main CPLANE PI(3)P binding modules contain intrinsically disordered regions, and this presumably could contribute to the regulation of the CPLANE complex’s functions. Another intriguing hypothesis is that CPLANE is involved in autophagy since PI(3)P synthesis is critical for autophagy initiation (54); moreover, autophagic removal of the centriolar satellite pool of OFD1 is required for ciliogenesis (55) and mutations in CPLANE proteins have been associated with OFD syndromes (19, 56). It will also be interesting to determine whether the CPLANE complex impinges on cortical actin clearance, an event that is required for cilium emergence (57). Last, Intu and Fuz were recently identified as the GEF of Rab23 (28). The putative cross-talk between the CPLANE-mediated regulation of Rsg1 and Rab23 as well as how these pathways integrate with the extensive network of ciliary Rab proteins (58) remains to be determined. Nevertheless, comparison of our structure to another member of the HerMon family (Mon1-Czz1) indicates that the hexalongin scaffold can provide multiple nonoverlapping binding sites for small GTPases (Fig. 7A). AlphaFold predicts that CPLANE-Rab23 binding is similar to Mon1-Czz1-YPT7. This indicates that the CPLANE complex could engage Rab23 and Rsg1 simultaneously (Fig. 7, B and C). This is consistent with the functions of the other HerMon family proteins that also act as both GEF and effectors of Rab GTPases (59–62). The structure of the CPLANE complex thus provides a blueprint for how HerMon family members can orchestrate lipid-binding and Rab signaling cascades (Fig. 7D). Further studies will be needed to provide mechanistic insights into CPLANE functions, but our results provide a roadmap to illuminate enigmatic aspects of ciliogenesis and the pathophysiology of human disorders.Fig. 7.Model for CPLANE acting as GEF and effector.(A) Superposition of Intu-Fuz-Rsg1 and Mon1-Ccz1-Ypt7 (PDB 5LDD) complexes reveals dissimilar binding of Rsg1 and Ypt7 GTPases. (B) Prediction of the human Intu/Fuz/Rab23 complex using AlphaFold-Multimer positions Rab23 at the same binding site as Ypt7. The predicted structure (AF) was superposed on the experimental structure (EM). (C) AlphaFold Predicted aligned error (PAE) plots for the top-ranked model (ranking based on overall PAE). Pairwise PAE values were extracted from the result matrix according to the sequence ranges of the subunits and plotted separately for clarity. (D) Model for CPLANE functions. CPLANE might act as a Rab signaling hub by being the effector of Rsg1 (Fuz binding site) and the GEF of Rab23 (Intu-Fuz binding site). The main PI3P binding modules of CPLANE in Wdpcp C terminus and Intu N terminus are indicated.

Human ciliopathies are an expanding class of multisystemic inherited diseases that are often caused by defects in cilium assembly. The CPLANE proteins Wdpcp, Intu, and Fuz are crucial for ciliogenesis and, when mutated, are associated with devastating ciliopathies; however, their structure and cellular functions have remained enigmatic (20).

Here, we used recombinant human and mouse proteins to show that Wdpcp, Intu, and Fuz assemble in a physical unit that we call the CPLANE complex. Existence of this complex has been suggested by studies in Drosophila and murine cells but direct biochemical evidence was lacking (20). We used cryo-EM to elucidate structures of human CPLANE complex and of mouse CPLANE complex bound to Rsg1. The mammalian CPLANE complex adopts a crescent-like architecture and is characterized by extensive interfaces between subunits. Wdpcp and Rsg1 are located on the ends of the crescent and bind the central Intu-Fuz heterodimer on opposite sides. This creates a linear-like subunit interaction scheme that can be depicted as Wdpcp-Intu-Fuz-Rsg1. Our analysis indicates that Intu and Fuz are structural homologs that use two triplets of LDs to interact with each other and with the other subunits of the complex. We did not detect a direct Wdpcp-Fuz interaction in our in vitro assays, although these subunits share a modest interface in the Wdpcp-Intu-Fuz hub. It is thus possible that the Wdpcp-Fuz interaction is unstable in the absence of Intu. Intriguingly, the CPLANE complex in vitro interacts constitutively with the atypical GTPase Rsg1. Of the five (G1 to G5) motifs that characterize GTPases of the Ras-superfamily, Rsg1 only harbors canonical G1 and G2 while G3 to G5 are divergent (30). Despite the noncanonical G motifs, Rsg1 binds GTP and interacts with the CPLANE subunit Fuz while GTP-bound. Future experiments should clarify whether the CPLANE-Rsg1 interaction is nucleotide-independent or whether CPLANE is an effector of Rsg1 and hence binds only to GTP-loaded Rsg1. Moreover, Rsg1 lacks the catalytic glutamine and it remains to be seen if it exhibits GTPase activity in vivo and how that affects the interaction with the CPLANE complex.

Mutations in CPLANE subunits have been linked to noxious developmental defects. Mapping these mutations on our structure did not reveal an informative pattern. To test whether these mutations affect CPLANE complex integrity, we expressed the mutated proteins recombinantly and tested their association. We did not, however, observe any clear impact on protein expression, assembly of the CPLANE complex, or interaction with Rsg1. We surmise that the penetrance of these mutations in vivo might be dependent on modifier genes (19, 25) or affect CPLANE functions that are not recapitulated by recombinant systems or our in vitro conditions. For instance, the Intu A452T and E500A mutations were previously shown to decrease the Intu/Fuz GEF activity toward Rab23 (28). Moreover, CPLANE subunits have been shown to have an extensive interactome (19, 27), and it is conceivable that mutations may affect these interactions in vivo. This is further supported by the fact that the MmRsg1_V169D lethal mutation does not compromise correct Rsg1 localization in vivo (30) or interaction with Fuz in vitro; it is hence possible that it impedes a critical interaction of Rsg1 with an unidentified protein.

The domains found in CPLANE subunits are emblematic of vesicle trafficking proteins (47) and we thus assessed PIP binding for the CPLANE complex and its individual subunits. Notably, the CPLANE complex interacts with PI(3)P and to a lesser extent with PI(4)P and PI(5)P. This interaction is mainly provided by Intu but we also detected binding of Wdpcp and Fuz. Rsg1 exhibited PIP interaction only when bound to the CPLANE complex and not as individual protein. This is consistent with the predicted lack of a prenylation motif in the C terminus of Rsg1 and suggests that Rsg1 is brought to membranes by interacting with the CPLANE complex. Except for the PDZ domain of Intu, CPLANE proteins lack canonical lipid binding domains (50) and we thus mapped the required modules in Intu and Wdpcp. The Intu N-terminal region containing PDZ was indeed capable of PIP binding, but Intu LDs also exhibited weak binding. On the other hand, Wdpcp engages PIPs via the last 60 C-terminal residues. Of note, both main PIP-binding modules localize to the only major unresolved parts of the CPLANE complex, the N terminus of Intu and the C terminus of Wdpcp. We noticed that the HsWdpcp_S708F ciliopathy mutation is found in the Wdpcp PIP binding C-terminal tail and contributes additional hydrophobicity to this segment; notably, this mutation confers stronger PIP binding compared to WT and presumably provides a molecular explanation for the ensuing BBS. A parsimonious hypothesis would be that aberrant PI(3)P binding may result in defective vesicle trafficking that perturbs BBS-related protein functions. Such defects could either compromise IFT-A complex assembly and subsequently result in abnormal IFT-B, and hence BBSome ciliary trafficking, or affect other BBS proteins that localize in the cilium transition zone (5, 19, 51). While the mechanistic details remain to be defined, this represents one of the first examples that link perturbed PIP binding with ciliopathies (5). The main ligand of CPLANE, PI(3)P, is an endosomal marker and has not yet been detected in mature cilia. Instead, the membrane of the cilium proper mainly contains PI(4)P whereas PI(4,5)P2 and PI(3,4,5)P3 are restricted to the transition zone (4). PI(3)P has, however, been reported in a vesicular compartment at the cilia base termed pericentriolar recycling endosome (PRE). This local pool of PI(3)P in PRE is thought to be required for activation of the Rab11/Rab8 cascade that culminates in cilium assembly (52). Given the PIPs binding of CPLANE proteins and the CV-stage arrest of CPLANE mutants during ciliogenesis, it is tempting to speculate that the CPLANE complex regulates trafficking of PI(3)P-containing vesicles to the nascent CV to promote membrane growth and/or deliver IFT-A proteins. These vesicles could originate at PRE or another PI(3)P-rich compartment. This hypothesis is supported by the CPLANE interactome (19), which includes IFT-A subunits and proteins involved in vesicle transport and fusion; moreover, IFT-A proteins can associate with vesicles (51) and abnormal vesicle accumulation has been observed in mutants of CPLANE proteins or downstream effectors (21, 26). The CPLANE complex could, for example, use its multiple PI(3)P binding sites to function as tether and bridge vesicles to target membranes before fusion (53). An important prediction of this model would be that CPLANE operates on PI(3)P-rich vesicles with an IFT-A–dependent coat (51). The main CPLANE PI(3)P binding modules contain intrinsically disordered regions, and this presumably could contribute to the regulation of the CPLANE complex’s functions. Another intriguing hypothesis is that CPLANE is involved in autophagy since PI(3)P synthesis is critical for autophagy initiation (54); moreover, autophagic removal of the centriolar satellite pool of OFD1 is required for ciliogenesis (55) and mutations in CPLANE proteins have been associated with OFD syndromes (19, 56). It will also be interesting to determine whether the CPLANE complex impinges on cortical actin clearance, an event that is required for cilium emergence (57). Last, Intu and Fuz were recently identified as the GEF of Rab23 (28). The putative cross-talk between the CPLANE-mediated regulation of Rsg1 and Rab23 as well as how these pathways integrate with the extensive network of ciliary Rab proteins (58) remains to be determined. Nevertheless, comparison of our structure to another member of the HerMon family (Mon1-Czz1) indicates that the hexalongin scaffold can provide multiple nonoverlapping binding sites for small GTPases (Fig. 7A). AlphaFold predicts that CPLANE-Rab23 binding is similar to Mon1-Czz1-YPT7. This indicates that the CPLANE complex could engage Rab23 and Rsg1 simultaneously (Fig. 7, B and C). This is consistent with the functions of the other HerMon family proteins that also act as both GEF and effectors of Rab GTPases (59–62). The structure of the CPLANE complex thus provides a blueprint for how HerMon family members can orchestrate lipid-binding and Rab signaling cascades (Fig. 7D). Further studies will be needed to provide mechanistic insights into CPLANE functions, but our results provide a roadmap to illuminate enigmatic aspects of ciliogenesis and the pathophysiology of human disorders.

Model for CPLANE acting as GEF and effector.

(A) Superposition of Intu-Fuz-Rsg1 and Mon1-Ccz1-Ypt7 (PDB 5LDD) complexes reveals dissimilar binding of Rsg1 and Ypt7 GTPases. (B) Prediction of the human Intu/Fuz/Rab23 complex using AlphaFold-Multimer positions Rab23 at the same binding site as Ypt7. The predicted structure (AF) was superposed on the experimental structure (EM). (C) AlphaFold Predicted aligned error (PAE) plots for the top-ranked model (ranking based on overall PAE). Pairwise PAE values were extracted from the result matrix according to the sequence ranges of the subunits and plotted separately for clarity. (D) Model for CPLANE functions. CPLANE might act as a Rab signaling hub by being the effector of Rsg1 (Fuz binding site) and the GEF of Rab23 (Intu-Fuz binding site). The main PI3P binding modules of CPLANE in Wdpcp C terminus and Intu N terminus are indicated.

MATERIALS AND METHODSProtein expression and purificationRecombinant protein expression was performed in Escherichia coli bacteria or insect cells. For expression in E. coli, full-length human and mouse Rsg1 were subcloned via Gibson assembly (NEB) in pETDuet-derived (Merck) vectors either untagged or having an N-terminal Twin-Strep tag (IBA) followed by a human rhinovirus 3C protease site. For protein expression, BL21 Star (Thermo Fisher Scientific) cells were transformed and grown in auto-induction (63) medium (4 hours at 37°C, 225 rpm and 20 hours at 20°C, 225 rpm). All subsequent steps were performed at 4°C. Cells were collected by centrifugation, resuspended in buffer (1 ml of buffer per 4 to 15 ml of cell culture) containing 100 mM tris, 200 mM NaCl, 0.2% Igepal CA-630, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8) plus cOmplete EDTA-free protease inhibitor tablets (Roche) and lysed by sonication. For small-scale purifications, 2 to 4 ml of cultures was used. Clarified lysate was incubated with 50 μl of slurry of MagStrep type3 XT beads (IBA) in batch mode. Beads were washed 3 to 5× with 50 mM tris, 200 mM NaCl, 0.1% Tween 20, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8), and bound material was eluted with 50 mM tris, 200 mM NaCl, 0.1% Tween 20, 0.5 mM TCEP, 5 mM MgCl2, and 50 mM biotin (pH 8). For large-scale purifications, 4 to 8 liters of cultures was used. Lysate was clarified by centrifugation (40,000g, 30 min) and filtered (0.45 μm) before loading on affinity columns using the GE Healthcare AKTA purifier FPLC.For expression in insect cells, full-length human and mouse Wdpcp, Intu, Fuz, and Rsg1 were subcloned via Gibson assembly (NEB) in pAC-derived vectors either untagged or having an N-terminal Twin-Strep (IBA) or hexa-histidine tag followed by a human rhinovirus 3C protease site (in certain vectors, the 3C protease site was deleted). Point mutations were introduced by polymerase chain reaction amplification of the gene of interest in two fragments while introducing the desired change via appropriate primers. The vectors were cotransfected with baculovirus DNA and Cellfectin II reagent (Thermo Fisher Scientific) in Spodoptera frugiperda (Sf9) insect cells (Thermo Fisher Scientific) to produce baculoviruses expressing the gene of interest. Sf9 cells were grown in Sf-900 III SFM (Thermo Fisher Scientific) at 27°C, 120 rpm. Baculoviruses were amplified in Sf9 cells. For protein expression, baculoviruses were used to infect Sf9 cultures (2 million cells/ml density) at 1:50 to 1:100 v/v dilution. For small-scale purifications, 8 to 30 ml of Sf9 cultures was used and cells were collected 48 hours after infection. All subsequent steps were performed at 4°C. Cells were resuspended in buffer (1 ml of buffer per 10 ml of cell culture) containing 100 mM tris, 200 mM NaCl, 0.2% Igepal CA-630, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8) plus cOmplete EDTA-free protease inhibitor tablets (Roche) and lysed by sonication. Clarified lysate was incubated with a 50-μl slurry of MagStrep type3 XT beads (IBA) in batch mode. Beads were washed 3 to 5× with 50 mM tris, 200 mM NaCl, 0.1% Tween 20, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8), and bound material was eluted with 50 mM tris, 200 mM NaCl, 0.1% Tween 20, 0.5 mM TCEP, 5 mM MgCl2, and 50 mM biotin (pH 8). For large-scale purifications (see below for details), 2 to 6 liters of Sf9 cultures was used and cells were collected 48 hours after infection. All subsequent steps were performed at 4°C. Cells were resuspended in buffer (100 ml of buffer per liter cell culture) and lysed by sonication. Lysate was clarified by centrifugation (40,000g, 30 min) and filtered (0.45 μm) before loading on affinity columns using the GE Healthcare AKTA purifier FPLC system.
HsWdpcp-HsIntu-HsFuz complex
Full-length HsWdpcp, HsIntu, and HsFuz were subcloned in pAC-derived vectors as untagged (HsFuz) or with an N-terminal Twin-Strep tag (HsWdpcp) or hexa-histidine tag (HsIntu) followed by a human rhinovirus 3C protease site. Sf9 cells were coinfected with three baculoviruses (1:100 v/v dilution each) and resuspended in buffer containing 100 mM tris, 200 mM NaCl, 0.25% Igepal CA-630, 0.5 mM TCEP, 2.5% glycerol, 5 mM MgCl2 (pH 8) plus avidin (IBA), and cOmplete EDTA-free protease inhibitor tablets (Roche). Clarified cell extract was loaded on a Strep-Tactin XT Superflow 5-ml cartridge (IBA), and bound material was washed with 20 mM tris, 200 mM NaCl, and 0.5 mM TCEP (pH 8). Bound material was eluted with 20 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 50 mM biotin (pH 8) and concentrated with Amicon filter (Millipore). Salt concentration was adjusted to 100 mM by adding equal volumes of 20 mM tris and 0.5 mM TCEP (pH 8). Complex was further purified by ion exchange chromatography in a RESOURCE Q 1-ml column (Cytiva). Complex was eluted with a 100 to 500 mM salt gradient (in 20 mM tris and 0.5 mM TCEP, pH 8), and peak fractions were concentrated and injected in a Superdex 200 column (Cytiva) for size exclusion chromatography in 50 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8). Peak fractions were concentrated to approximately 0.7 mg/ml and used for cryo-EM.
MmWdpcp-MmIntu-MmFuz-MmRsg1 complex
Full-length MmWdpcp, MmIntu, MmFuz, and MmRsg1 were subcloned in pAC-derived vectors as untagged (MmFuz and MmRsg1) or with an N-terminal Twin-Strep tag (MmWdpcp) or hexa-histidine tag (MmIntu) followed by a human rhinovirus 3C protease site. Sf9 cells were coinfected with four baculoviruses (1:100 v/v dilution each) and resuspended in buffer containing 100 mM tris, 200 mM NaCl, 0.25% Igepal CA-630, 0.5 mM TCEP, 5 mM MgCl2 (pH 8) plus avidin (IBA), and cOmplete EDTA-free protease inhibitor tablets (Roche). Clarified cell extract was loaded on a Strep-Tactin XT Superflow 5-ml cartridge (IBA), and bound material was washed with 20 mM tris, 500 mM NaCl, and 0.5 mM TCEP (pH 8) and 20 mM tris, 200 mM NaCl, and 0.5 mM TCEP (pH 8). Bound material was eluted with 20 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 50 mM biotin (pH 8) and concentrated with Amicon filter (Millipore). Salt concentration was adjusted to 100 mM by adding equal volumes of 20 mM tris and 0.5 mM TCEP (pH 8). The complex was further purified by ion exchange chromatography in a RESOURCE Q 1-ml column (Cytiva). The complex was eluted with a 100 to 500 mM salt gradient (in 20 mM tris and 0.5 mM TCEP, pH 8), and peak fractions were concentrated and injected in a Superdex 200 column (Cytiva) for size exclusion chromatography in 50 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8). Peak fractions were concentrated to approximately 0.7 mg/ml and used for cryo-EM.Protein samples were resolved by SDS–polyacrylamide gel electrophoresis (PAGE) using NuPAGE 4 to 12% bis-tris gels (Thermo Fisher Scientific). Proteins were stained with QuickBlue (Lubio).Lipid binding assayPIP strips (Echelon) were preblocked with 10 mM tris (pH 8), 150 mM NaCl, 0.1% Tween 20, and 1% w/v skim milk for 1 hour at room temperature. Proteins were obtained by small-scale purifications and incubated at 3 to 20 nM with PIP strips in 10 mM tris (pH 8), 150 mM NaCl, 0.1% Tween 20, and 1% w/v skim milk for 1 hour at room temperature. Strips were washed 3× with 10 mM tris (pH 8), 150 mM NaCl, and 0.1% Tween 20, and mouse anti-Flag antibody (Sigma-Aldrich) was added (1:5000 v/v) in 10 mM tris (pH 8), 150 mM NaCl, 0.1% Tween 20, and 1% w/v skim milk for 1 hour at room temperature. Strips were washed 3× with 10 mM tris (pH 8), 150 mM NaCl, and 0.1% Tween 20, and anti-mouse HRP antibody (Bio-Rad) was added (1:5000 v/v) in 10 mM tris (pH 8), 150 mM NaCl, 0.1% Tween 20, and 1% w/v skim milk for 1 hour at room temperature. Strips were washed 3× with 10 mM tris (pH 8), 150 mM NaCl, and 0.1% Tween 20 and 1× with 10 mM tris (pH 8) and 150 mM NaCl. Strips were incubated with chemiluminescent substrate (SuperSignal West Pico Plus, Thermo Fisher Scientific), and signal was detected by photographic films. To allow comparisons, strips were processed in parallel.HPLC nucleotide analysisThe nucleotide states of purified proteins were verified at 21°C by reversed-phase HPLC using a Vydac 218TP C18 column (HiChrom) fitted with a 218TP C18 5 μm, 4.6 mm Guard Cartridges (HiChrom). Nucleotides were separated by isocratic elution at 21°C with a buffer containing 33 mM K2HPO4, 67 mM KH2PO4, 10 mM tetrabutylammonium bromide, and 7.5% (v/v) acetonitrile (HPLC grade). Nucleotide samples of 100 μM GTP (Roth) or 100 μM GDP (Roche) in 20 mM tris (pH 8), 200 mM NaCl, and 0.5 mM TCEP were used as controls. Elution of nucleotides was detected at 254 nm, and 25 μl of sample was injected per HPLC run. Protein samples were centrifuged at 13,000 RCF for 10 min before injection on the Vydac 218TP C18 column.Cryo-EM sample preparationA total of 3.5 μl of sample at 0.7 to 1 mg/ml (2.5 to 3 μM) was applied to Quantifoil holey carbon grids (R 1.2/1.3 200 mesh, Quantifoil Micro Tools). Glow discharging was carried out in a Solarus plasma cleaner (Gatan) for 15 s in a H2/O2 environment. Grids were blotted for 3 to 4 s at 4°C at 100% humidity in a Vitrobot Mark IV (FEI, Hillsboro, OR, USA) and then immediately plunged into liquid ethane.Cryo-EM data collectionData were collected automatically with EPU (Thermo Fisher Scientific) on a Cs-corrected (CEOS GmbH, Heidelberg, Germany) Titan Krios (Thermo Fisher Scientific) electron microscope at 300 keV. Four sessions for a total of 13,056 zero-energy loss movies of the mouse CPLANE complex were recorded using a Gatan K2 summit direct electron detector (Gatan) operating in counting mode and located after a Quantum-LS energy filter (slit width of 20 eV). The acquisition was performed at a magnification of 130,000× yielding a pixel size of 0.86 Å at the specimen level. Datasets were recorded with dose rates between 3.5 and 5 e−/(px·s), and the exposures were fractionated into 50 frames for an accumulated dose of 52 e−/Å2. The targeted defocus values ranged from −0.8 to −2.5 μm. The data for the human CPLANE complex (5619 movies) were collected on the same Titan Krios but using a Falcon 4 (Thermo Fisher Scientific) instead. The magnification was 96,000× with a pixel size of 0.66 Å. The dose rate was set to 18.3 e−/Å/s, and the exposure time was adjusted to account for an accumulated dose of 50 e−/Å2. The EER frames were grouped to obtain 50 fractions.Cryo-EM image processingReal-time data evaluation along with acquisition with EPU (Thermo Fisher Scientific) was performed with cryoFLARE (64). Within cryoFLARE, drift correction was performed with the Relion 3 motioncor implementation, where a motion-corrected sum of all frames was generated with and without applying a dose weighting scheme. CTF was fitted using GCTF (65) on the non–dose-weighted sums. After CTF estimation, only the data that showed a CTF fit better than 5 Å were exported from cryoFLARE and used for further processing with Relion 3 (66) and cryoSPARCv3 (67).For the human CPLANE complex (fig. S2), the raw movies were imported into Warp pipeline and subjected to alignment and patch motion correction. The resulting motion-corrected and dose-weighted micrographs were imported in cryoSPARC. Blob picking and four rounds of two-dimensional (2D) classification were necessary to obtain a set 2D class averages that was used for template picking. The resulting particles were subjected to three rounds of 2D classification to clean up the dataset and lead a subset of 238,978 particles. For ab initio model generation, 50,000 particles were randomly selected (fig. S2) and used to reconstruct one initial 3D envelope. An initial round of 3D refinement led to a reconstruction at 4.1 Å resolution. 3D variability analysis was used to investigate the presence of discrete subclasses. The dataset was divided into four clusters and the resulting models and particles associated to each of them were subjected to 3D heterogeneous refinement. This led to a homogeneous subset of 129,902 particles that were further refined at the final resolution of 3.35 Å.For the mouse CPLANE complex (fig. S3), raw movies from four datasets were imported into cryoSPARC and subjected to alignment and patch motion correction. CTF parameters were estimated with patch CTF. Particles were picked via blob picker. Each dataset was first processed independently for three rounds of 2D classification. For ab initio model generation, 88,000 particles were randomly selected from dataset 1 (fig. S3) and used to reconstruct four initial 3D maps. The 3D initial map that showed better resolved secondary structural features after refinement to higher resolution was kept for further processing. Then, the best particles from each dataset were merged and subjected to another round of 2D classification leading to 441,620 particles. These particles were used to perform an initial round of 3D refinement leading to a reconstruction at 3.4 Å resolution. These refined particles were exported to Relion to perform 3D classification using local searches. A total of 267,271 particles included in the best 3D class were reimported in cryoSPARC to assess, if present, conformation variability in the datasets. Three discrete structures were identified, and the particles included in the model that showed better refined Rsg1 densities were pooled together and subjected to another round of ab initio model generation to further clean up the dataset before refining the model to the final resolution of 3.35 Å.The resolution values reported for all reconstructions are based on the gold standard Fourier shell correlation (FSC) curve at 0.143 criterion. Maps were sharpened with DeepEMhancer (68), and after having obtained an initial model, sharpening was repeated with Locscale (69) starting from the initial map.Model building, refinement, and validationSince closest homologs had poor sequence identities (11 to 20%) and/or low coverage, we used TrRosetta (70), a deep learning-based structure prediction pipeline, to generate initial models. The β-propeller and α-helical domains predicted for Wdpcp were evident from the map and, after identifying the order, blades and helices extracted from the TrRosetta model could be fitted sequentially. The map also contained prominent features of extended β sheets interspersed by α helices indicative of the LDs predicted for Intu and Fuz. The TrRosetta model of Fuz deviated by ~5.7 Å RMSD from the final model and could be fitted as a rigid body. In contrast, the spatial arrangement of LDs in the predicted Intu model was far off from the final model (~20 Å RMSD). However, the correct prediction of the overall fold of the individual LDs enabled mapping of the domains to the density based on secondary structure and side-chain correlation. The density corresponding to RSG1 showed reduced local resolution compared to the rest of the complex; side chains could, however, still be assigned for most parts. During the course of this study, AlphaFold (AF) v2.0 was released (33). We used the predicted models from AF to cross-validate the cryo-EM models. The RMSDs of the predicted models compared to the final cryo-EM models (MmCPLANE) were 0.8 Å for RSG1 (224 residues), 1.2 Å for Fuz (392 residues), 1.6 Å for Intu (336 residues), and 2.2 Å for Wdpcp (536 residues). Models were docked in COOT (71), or in PHENIX (72). Local backbone, side chain, and register deviations were fixed manually in COOT and ISOLDE/ChimeraX (73, 74), and the model was subjected to an automated structure refinement protocol using ROSETTA as previously described (75). Several further rounds of manual rebuilding in COOT and ISOLDE/ChimeraX followed by Cartesian refinement with the ROSETTA FastRelax protocol in combination with density scoring terms were carried out. To test for overfitting, the structure was refined against the first half-map and the FSC correlation was calculated for both half-maps. B factors were fitted at a final stage using ROSETTA. An in-house pipeline was used to run the ROSETTA FastRelax protocol (https://github.com/fmi-basel/RosEM). A model for the human CPLANE structure was built using the mouse model as a template. The sequences of HsWdpcp, HsIntu, and HsFuzzy were aligned to the mouse homologs using phenix.muscle and threaded onto the template models using rosetta.partial_thread (76). The model was iteratively refined by manual rebuilding in COOT followed by real-space refinement using ROSETTA FastRelax as described above. Both models were validated using PHENIX, MOLPROBITY (77), and EMRINGER (78). RMSDs were calculated with the Cealign method implemented in PyMOL (The PyMOL Molecular Graphics System, Version 2.0 Schrödinger LLC). Figures were prepared with ChimeraX and PyMOL.

MATERIALS AND METHODS

Protein expression and purificationRecombinant protein expression was performed in Escherichia coli bacteria or insect cells. For expression in E. coli, full-length human and mouse Rsg1 were subcloned via Gibson assembly (NEB) in pETDuet-derived (Merck) vectors either untagged or having an N-terminal Twin-Strep tag (IBA) followed by a human rhinovirus 3C protease site. For protein expression, BL21 Star (Thermo Fisher Scientific) cells were transformed and grown in auto-induction (63) medium (4 hours at 37°C, 225 rpm and 20 hours at 20°C, 225 rpm). All subsequent steps were performed at 4°C. Cells were collected by centrifugation, resuspended in buffer (1 ml of buffer per 4 to 15 ml of cell culture) containing 100 mM tris, 200 mM NaCl, 0.2% Igepal CA-630, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8) plus cOmplete EDTA-free protease inhibitor tablets (Roche) and lysed by sonication. For small-scale purifications, 2 to 4 ml of cultures was used. Clarified lysate was incubated with 50 μl of slurry of MagStrep type3 XT beads (IBA) in batch mode. Beads were washed 3 to 5× with 50 mM tris, 200 mM NaCl, 0.1% Tween 20, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8), and bound material was eluted with 50 mM tris, 200 mM NaCl, 0.1% Tween 20, 0.5 mM TCEP, 5 mM MgCl2, and 50 mM biotin (pH 8). For large-scale purifications, 4 to 8 liters of cultures was used. Lysate was clarified by centrifugation (40,000g, 30 min) and filtered (0.45 μm) before loading on affinity columns using the GE Healthcare AKTA purifier FPLC.For expression in insect cells, full-length human and mouse Wdpcp, Intu, Fuz, and Rsg1 were subcloned via Gibson assembly (NEB) in pAC-derived vectors either untagged or having an N-terminal Twin-Strep (IBA) or hexa-histidine tag followed by a human rhinovirus 3C protease site (in certain vectors, the 3C protease site was deleted). Point mutations were introduced by polymerase chain reaction amplification of the gene of interest in two fragments while introducing the desired change via appropriate primers. The vectors were cotransfected with baculovirus DNA and Cellfectin II reagent (Thermo Fisher Scientific) in Spodoptera frugiperda (Sf9) insect cells (Thermo Fisher Scientific) to produce baculoviruses expressing the gene of interest. Sf9 cells were grown in Sf-900 III SFM (Thermo Fisher Scientific) at 27°C, 120 rpm. Baculoviruses were amplified in Sf9 cells. For protein expression, baculoviruses were used to infect Sf9 cultures (2 million cells/ml density) at 1:50 to 1:100 v/v dilution. For small-scale purifications, 8 to 30 ml of Sf9 cultures was used and cells were collected 48 hours after infection. All subsequent steps were performed at 4°C. Cells were resuspended in buffer (1 ml of buffer per 10 ml of cell culture) containing 100 mM tris, 200 mM NaCl, 0.2% Igepal CA-630, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8) plus cOmplete EDTA-free protease inhibitor tablets (Roche) and lysed by sonication. Clarified lysate was incubated with a 50-μl slurry of MagStrep type3 XT beads (IBA) in batch mode. Beads were washed 3 to 5× with 50 mM tris, 200 mM NaCl, 0.1% Tween 20, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8), and bound material was eluted with 50 mM tris, 200 mM NaCl, 0.1% Tween 20, 0.5 mM TCEP, 5 mM MgCl2, and 50 mM biotin (pH 8). For large-scale purifications (see below for details), 2 to 6 liters of Sf9 cultures was used and cells were collected 48 hours after infection. All subsequent steps were performed at 4°C. Cells were resuspended in buffer (100 ml of buffer per liter cell culture) and lysed by sonication. Lysate was clarified by centrifugation (40,000g, 30 min) and filtered (0.45 μm) before loading on affinity columns using the GE Healthcare AKTA purifier FPLC system.
HsWdpcp-HsIntu-HsFuz complex
Full-length HsWdpcp, HsIntu, and HsFuz were subcloned in pAC-derived vectors as untagged (HsFuz) or with an N-terminal Twin-Strep tag (HsWdpcp) or hexa-histidine tag (HsIntu) followed by a human rhinovirus 3C protease site. Sf9 cells were coinfected with three baculoviruses (1:100 v/v dilution each) and resuspended in buffer containing 100 mM tris, 200 mM NaCl, 0.25% Igepal CA-630, 0.5 mM TCEP, 2.5% glycerol, 5 mM MgCl2 (pH 8) plus avidin (IBA), and cOmplete EDTA-free protease inhibitor tablets (Roche). Clarified cell extract was loaded on a Strep-Tactin XT Superflow 5-ml cartridge (IBA), and bound material was washed with 20 mM tris, 200 mM NaCl, and 0.5 mM TCEP (pH 8). Bound material was eluted with 20 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 50 mM biotin (pH 8) and concentrated with Amicon filter (Millipore). Salt concentration was adjusted to 100 mM by adding equal volumes of 20 mM tris and 0.5 mM TCEP (pH 8). Complex was further purified by ion exchange chromatography in a RESOURCE Q 1-ml column (Cytiva). Complex was eluted with a 100 to 500 mM salt gradient (in 20 mM tris and 0.5 mM TCEP, pH 8), and peak fractions were concentrated and injected in a Superdex 200 column (Cytiva) for size exclusion chromatography in 50 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8). Peak fractions were concentrated to approximately 0.7 mg/ml and used for cryo-EM.
MmWdpcp-MmIntu-MmFuz-MmRsg1 complex
Full-length MmWdpcp, MmIntu, MmFuz, and MmRsg1 were subcloned in pAC-derived vectors as untagged (MmFuz and MmRsg1) or with an N-terminal Twin-Strep tag (MmWdpcp) or hexa-histidine tag (MmIntu) followed by a human rhinovirus 3C protease site. Sf9 cells were coinfected with four baculoviruses (1:100 v/v dilution each) and resuspended in buffer containing 100 mM tris, 200 mM NaCl, 0.25% Igepal CA-630, 0.5 mM TCEP, 5 mM MgCl2 (pH 8) plus avidin (IBA), and cOmplete EDTA-free protease inhibitor tablets (Roche). Clarified cell extract was loaded on a Strep-Tactin XT Superflow 5-ml cartridge (IBA), and bound material was washed with 20 mM tris, 500 mM NaCl, and 0.5 mM TCEP (pH 8) and 20 mM tris, 200 mM NaCl, and 0.5 mM TCEP (pH 8). Bound material was eluted with 20 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 50 mM biotin (pH 8) and concentrated with Amicon filter (Millipore). Salt concentration was adjusted to 100 mM by adding equal volumes of 20 mM tris and 0.5 mM TCEP (pH 8). The complex was further purified by ion exchange chromatography in a RESOURCE Q 1-ml column (Cytiva). The complex was eluted with a 100 to 500 mM salt gradient (in 20 mM tris and 0.5 mM TCEP, pH 8), and peak fractions were concentrated and injected in a Superdex 200 column (Cytiva) for size exclusion chromatography in 50 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8). Peak fractions were concentrated to approximately 0.7 mg/ml and used for cryo-EM.Protein samples were resolved by SDS–polyacrylamide gel electrophoresis (PAGE) using NuPAGE 4 to 12% bis-tris gels (Thermo Fisher Scientific). Proteins were stained with QuickBlue (Lubio).

Protein expression and purification

Recombinant protein expression was performed in Escherichia coli bacteria or insect cells. For expression in E. coli, full-length human and mouse Rsg1 were subcloned via Gibson assembly (NEB) in pETDuet-derived (Merck) vectors either untagged or having an N-terminal Twin-Strep tag (IBA) followed by a human rhinovirus 3C protease site. For protein expression, BL21 Star (Thermo Fisher Scientific) cells were transformed and grown in auto-induction (63) medium (4 hours at 37°C, 225 rpm and 20 hours at 20°C, 225 rpm). All subsequent steps were performed at 4°C. Cells were collected by centrifugation, resuspended in buffer (1 ml of buffer per 4 to 15 ml of cell culture) containing 100 mM tris, 200 mM NaCl, 0.2% Igepal CA-630, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8) plus cOmplete EDTA-free protease inhibitor tablets (Roche) and lysed by sonication. For small-scale purifications, 2 to 4 ml of cultures was used. Clarified lysate was incubated with 50 μl of slurry of MagStrep type3 XT beads (IBA) in batch mode. Beads were washed 3 to 5× with 50 mM tris, 200 mM NaCl, 0.1% Tween 20, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8), and bound material was eluted with 50 mM tris, 200 mM NaCl, 0.1% Tween 20, 0.5 mM TCEP, 5 mM MgCl2, and 50 mM biotin (pH 8). For large-scale purifications, 4 to 8 liters of cultures was used. Lysate was clarified by centrifugation (40,000g, 30 min) and filtered (0.45 μm) before loading on affinity columns using the GE Healthcare AKTA purifier FPLC.

For expression in insect cells, full-length human and mouse Wdpcp, Intu, Fuz, and Rsg1 were subcloned via Gibson assembly (NEB) in pAC-derived vectors either untagged or having an N-terminal Twin-Strep (IBA) or hexa-histidine tag followed by a human rhinovirus 3C protease site (in certain vectors, the 3C protease site was deleted). Point mutations were introduced by polymerase chain reaction amplification of the gene of interest in two fragments while introducing the desired change via appropriate primers. The vectors were cotransfected with baculovirus DNA and Cellfectin II reagent (Thermo Fisher Scientific) in Spodoptera frugiperda (Sf9) insect cells (Thermo Fisher Scientific) to produce baculoviruses expressing the gene of interest. Sf9 cells were grown in Sf-900 III SFM (Thermo Fisher Scientific) at 27°C, 120 rpm. Baculoviruses were amplified in Sf9 cells. For protein expression, baculoviruses were used to infect Sf9 cultures (2 million cells/ml density) at 1:50 to 1:100 v/v dilution. For small-scale purifications, 8 to 30 ml of Sf9 cultures was used and cells were collected 48 hours after infection. All subsequent steps were performed at 4°C. Cells were resuspended in buffer (1 ml of buffer per 10 ml of cell culture) containing 100 mM tris, 200 mM NaCl, 0.2% Igepal CA-630, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8) plus cOmplete EDTA-free protease inhibitor tablets (Roche) and lysed by sonication. Clarified lysate was incubated with a 50-μl slurry of MagStrep type3 XT beads (IBA) in batch mode. Beads were washed 3 to 5× with 50 mM tris, 200 mM NaCl, 0.1% Tween 20, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8), and bound material was eluted with 50 mM tris, 200 mM NaCl, 0.1% Tween 20, 0.5 mM TCEP, 5 mM MgCl2, and 50 mM biotin (pH 8). For large-scale purifications (see below for details), 2 to 6 liters of Sf9 cultures was used and cells were collected 48 hours after infection. All subsequent steps were performed at 4°C. Cells were resuspended in buffer (100 ml of buffer per liter cell culture) and lysed by sonication. Lysate was clarified by centrifugation (40,000g, 30 min) and filtered (0.45 μm) before loading on affinity columns using the GE Healthcare AKTA purifier FPLC system.

HsWdpcp-HsIntu-HsFuz complex
Full-length HsWdpcp, HsIntu, and HsFuz were subcloned in pAC-derived vectors as untagged (HsFuz) or with an N-terminal Twin-Strep tag (HsWdpcp) or hexa-histidine tag (HsIntu) followed by a human rhinovirus 3C protease site. Sf9 cells were coinfected with three baculoviruses (1:100 v/v dilution each) and resuspended in buffer containing 100 mM tris, 200 mM NaCl, 0.25% Igepal CA-630, 0.5 mM TCEP, 2.5% glycerol, 5 mM MgCl2 (pH 8) plus avidin (IBA), and cOmplete EDTA-free protease inhibitor tablets (Roche). Clarified cell extract was loaded on a Strep-Tactin XT Superflow 5-ml cartridge (IBA), and bound material was washed with 20 mM tris, 200 mM NaCl, and 0.5 mM TCEP (pH 8). Bound material was eluted with 20 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 50 mM biotin (pH 8) and concentrated with Amicon filter (Millipore). Salt concentration was adjusted to 100 mM by adding equal volumes of 20 mM tris and 0.5 mM TCEP (pH 8). Complex was further purified by ion exchange chromatography in a RESOURCE Q 1-ml column (Cytiva). Complex was eluted with a 100 to 500 mM salt gradient (in 20 mM tris and 0.5 mM TCEP, pH 8), and peak fractions were concentrated and injected in a Superdex 200 column (Cytiva) for size exclusion chromatography in 50 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8). Peak fractions were concentrated to approximately 0.7 mg/ml and used for cryo-EM.

HsWdpcp-HsIntu-HsFuz complex

Full-length HsWdpcp, HsIntu, and HsFuz were subcloned in pAC-derived vectors as untagged (HsFuz) or with an N-terminal Twin-Strep tag (HsWdpcp) or hexa-histidine tag (HsIntu) followed by a human rhinovirus 3C protease site. Sf9 cells were coinfected with three baculoviruses (1:100 v/v dilution each) and resuspended in buffer containing 100 mM tris, 200 mM NaCl, 0.25% Igepal CA-630, 0.5 mM TCEP, 2.5% glycerol, 5 mM MgCl2 (pH 8) plus avidin (IBA), and cOmplete EDTA-free protease inhibitor tablets (Roche). Clarified cell extract was loaded on a Strep-Tactin XT Superflow 5-ml cartridge (IBA), and bound material was washed with 20 mM tris, 200 mM NaCl, and 0.5 mM TCEP (pH 8). Bound material was eluted with 20 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 50 mM biotin (pH 8) and concentrated with Amicon filter (Millipore). Salt concentration was adjusted to 100 mM by adding equal volumes of 20 mM tris and 0.5 mM TCEP (pH 8). Complex was further purified by ion exchange chromatography in a RESOURCE Q 1-ml column (Cytiva). Complex was eluted with a 100 to 500 mM salt gradient (in 20 mM tris and 0.5 mM TCEP, pH 8), and peak fractions were concentrated and injected in a Superdex 200 column (Cytiva) for size exclusion chromatography in 50 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8). Peak fractions were concentrated to approximately 0.7 mg/ml and used for cryo-EM.

MmWdpcp-MmIntu-MmFuz-MmRsg1 complex
Full-length MmWdpcp, MmIntu, MmFuz, and MmRsg1 were subcloned in pAC-derived vectors as untagged (MmFuz and MmRsg1) or with an N-terminal Twin-Strep tag (MmWdpcp) or hexa-histidine tag (MmIntu) followed by a human rhinovirus 3C protease site. Sf9 cells were coinfected with four baculoviruses (1:100 v/v dilution each) and resuspended in buffer containing 100 mM tris, 200 mM NaCl, 0.25% Igepal CA-630, 0.5 mM TCEP, 5 mM MgCl2 (pH 8) plus avidin (IBA), and cOmplete EDTA-free protease inhibitor tablets (Roche). Clarified cell extract was loaded on a Strep-Tactin XT Superflow 5-ml cartridge (IBA), and bound material was washed with 20 mM tris, 500 mM NaCl, and 0.5 mM TCEP (pH 8) and 20 mM tris, 200 mM NaCl, and 0.5 mM TCEP (pH 8). Bound material was eluted with 20 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 50 mM biotin (pH 8) and concentrated with Amicon filter (Millipore). Salt concentration was adjusted to 100 mM by adding equal volumes of 20 mM tris and 0.5 mM TCEP (pH 8). The complex was further purified by ion exchange chromatography in a RESOURCE Q 1-ml column (Cytiva). The complex was eluted with a 100 to 500 mM salt gradient (in 20 mM tris and 0.5 mM TCEP, pH 8), and peak fractions were concentrated and injected in a Superdex 200 column (Cytiva) for size exclusion chromatography in 50 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8). Peak fractions were concentrated to approximately 0.7 mg/ml and used for cryo-EM.Protein samples were resolved by SDS–polyacrylamide gel electrophoresis (PAGE) using NuPAGE 4 to 12% bis-tris gels (Thermo Fisher Scientific). Proteins were stained with QuickBlue (Lubio).

MmWdpcp-MmIntu-MmFuz-MmRsg1 complex

Full-length MmWdpcp, MmIntu, MmFuz, and MmRsg1 were subcloned in pAC-derived vectors as untagged (MmFuz and MmRsg1) or with an N-terminal Twin-Strep tag (MmWdpcp) or hexa-histidine tag (MmIntu) followed by a human rhinovirus 3C protease site. Sf9 cells were coinfected with four baculoviruses (1:100 v/v dilution each) and resuspended in buffer containing 100 mM tris, 200 mM NaCl, 0.25% Igepal CA-630, 0.5 mM TCEP, 5 mM MgCl2 (pH 8) plus avidin (IBA), and cOmplete EDTA-free protease inhibitor tablets (Roche). Clarified cell extract was loaded on a Strep-Tactin XT Superflow 5-ml cartridge (IBA), and bound material was washed with 20 mM tris, 500 mM NaCl, and 0.5 mM TCEP (pH 8) and 20 mM tris, 200 mM NaCl, and 0.5 mM TCEP (pH 8). Bound material was eluted with 20 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 50 mM biotin (pH 8) and concentrated with Amicon filter (Millipore). Salt concentration was adjusted to 100 mM by adding equal volumes of 20 mM tris and 0.5 mM TCEP (pH 8). The complex was further purified by ion exchange chromatography in a RESOURCE Q 1-ml column (Cytiva). The complex was eluted with a 100 to 500 mM salt gradient (in 20 mM tris and 0.5 mM TCEP, pH 8), and peak fractions were concentrated and injected in a Superdex 200 column (Cytiva) for size exclusion chromatography in 50 mM tris, 200 mM NaCl, 0.5 mM TCEP, and 5 mM MgCl2 (pH 8). Peak fractions were concentrated to approximately 0.7 mg/ml and used for cryo-EM.

Protein samples were resolved by SDS–polyacrylamide gel electrophoresis (PAGE) using NuPAGE 4 to 12% bis-tris gels (Thermo Fisher Scientific). Proteins were stained with QuickBlue (Lubio).

Lipid binding assayPIP strips (Echelon) were preblocked with 10 mM tris (pH 8), 150 mM NaCl, 0.1% Tween 20, and 1% w/v skim milk for 1 hour at room temperature. Proteins were obtained by small-scale purifications and incubated at 3 to 20 nM with PIP strips in 10 mM tris (pH 8), 150 mM NaCl, 0.1% Tween 20, and 1% w/v skim milk for 1 hour at room temperature. Strips were washed 3× with 10 mM tris (pH 8), 150 mM NaCl, and 0.1% Tween 20, and mouse anti-Flag antibody (Sigma-Aldrich) was added (1:5000 v/v) in 10 mM tris (pH 8), 150 mM NaCl, 0.1% Tween 20, and 1% w/v skim milk for 1 hour at room temperature. Strips were washed 3× with 10 mM tris (pH 8), 150 mM NaCl, and 0.1% Tween 20, and anti-mouse HRP antibody (Bio-Rad) was added (1:5000 v/v) in 10 mM tris (pH 8), 150 mM NaCl, 0.1% Tween 20, and 1% w/v skim milk for 1 hour at room temperature. Strips were washed 3× with 10 mM tris (pH 8), 150 mM NaCl, and 0.1% Tween 20 and 1× with 10 mM tris (pH 8) and 150 mM NaCl. Strips were incubated with chemiluminescent substrate (SuperSignal West Pico Plus, Thermo Fisher Scientific), and signal was detected by photographic films. To allow comparisons, strips were processed in parallel.

Lipid binding assay

PIP strips (Echelon) were preblocked with 10 mM tris (pH 8), 150 mM NaCl, 0.1% Tween 20, and 1% w/v skim milk for 1 hour at room temperature. Proteins were obtained by small-scale purifications and incubated at 3 to 20 nM with PIP strips in 10 mM tris (pH 8), 150 mM NaCl, 0.1% Tween 20, and 1% w/v skim milk for 1 hour at room temperature. Strips were washed 3× with 10 mM tris (pH 8), 150 mM NaCl, and 0.1% Tween 20, and mouse anti-Flag antibody (Sigma-Aldrich) was added (1:5000 v/v) in 10 mM tris (pH 8), 150 mM NaCl, 0.1% Tween 20, and 1% w/v skim milk for 1 hour at room temperature. Strips were washed 3× with 10 mM tris (pH 8), 150 mM NaCl, and 0.1% Tween 20, and anti-mouse HRP antibody (Bio-Rad) was added (1:5000 v/v) in 10 mM tris (pH 8), 150 mM NaCl, 0.1% Tween 20, and 1% w/v skim milk for 1 hour at room temperature. Strips were washed 3× with 10 mM tris (pH 8), 150 mM NaCl, and 0.1% Tween 20 and 1× with 10 mM tris (pH 8) and 150 mM NaCl. Strips were incubated with chemiluminescent substrate (SuperSignal West Pico Plus, Thermo Fisher Scientific), and signal was detected by photographic films. To allow comparisons, strips were processed in parallel.

HPLC nucleotide analysisThe nucleotide states of purified proteins were verified at 21°C by reversed-phase HPLC using a Vydac 218TP C18 column (HiChrom) fitted with a 218TP C18 5 μm, 4.6 mm Guard Cartridges (HiChrom). Nucleotides were separated by isocratic elution at 21°C with a buffer containing 33 mM K2HPO4, 67 mM KH2PO4, 10 mM tetrabutylammonium bromide, and 7.5% (v/v) acetonitrile (HPLC grade). Nucleotide samples of 100 μM GTP (Roth) or 100 μM GDP (Roche) in 20 mM tris (pH 8), 200 mM NaCl, and 0.5 mM TCEP were used as controls. Elution of nucleotides was detected at 254 nm, and 25 μl of sample was injected per HPLC run. Protein samples were centrifuged at 13,000 RCF for 10 min before injection on the Vydac 218TP C18 column.

HPLC nucleotide analysis

The nucleotide states of purified proteins were verified at 21°C by reversed-phase HPLC using a Vydac 218TP C18 column (HiChrom) fitted with a 218TP C18 5 μm, 4.6 mm Guard Cartridges (HiChrom). Nucleotides were separated by isocratic elution at 21°C with a buffer containing 33 mM K2HPO4, 67 mM KH2PO4, 10 mM tetrabutylammonium bromide, and 7.5% (v/v) acetonitrile (HPLC grade). Nucleotide samples of 100 μM GTP (Roth) or 100 μM GDP (Roche) in 20 mM tris (pH 8), 200 mM NaCl, and 0.5 mM TCEP were used as controls. Elution of nucleotides was detected at 254 nm, and 25 μl of sample was injected per HPLC run. Protein samples were centrifuged at 13,000 RCF for 10 min before injection on the Vydac 218TP C18 column.

Cryo-EM sample preparationA total of 3.5 μl of sample at 0.7 to 1 mg/ml (2.5 to 3 μM) was applied to Quantifoil holey carbon grids (R 1.2/1.3 200 mesh, Quantifoil Micro Tools). Glow discharging was carried out in a Solarus plasma cleaner (Gatan) for 15 s in a H2/O2 environment. Grids were blotted for 3 to 4 s at 4°C at 100% humidity in a Vitrobot Mark IV (FEI, Hillsboro, OR, USA) and then immediately plunged into liquid ethane.

Cryo-EM sample preparation

A total of 3.5 μl of sample at 0.7 to 1 mg/ml (2.5 to 3 μM) was applied to Quantifoil holey carbon grids (R 1.2/1.3 200 mesh, Quantifoil Micro Tools). Glow discharging was carried out in a Solarus plasma cleaner (Gatan) for 15 s in a H2/O2 environment. Grids were blotted for 3 to 4 s at 4°C at 100% humidity in a Vitrobot Mark IV (FEI, Hillsboro, OR, USA) and then immediately plunged into liquid ethane.

Cryo-EM data collectionData were collected automatically with EPU (Thermo Fisher Scientific) on a Cs-corrected (CEOS GmbH, Heidelberg, Germany) Titan Krios (Thermo Fisher Scientific) electron microscope at 300 keV. Four sessions for a total of 13,056 zero-energy loss movies of the mouse CPLANE complex were recorded using a Gatan K2 summit direct electron detector (Gatan) operating in counting mode and located after a Quantum-LS energy filter (slit width of 20 eV). The acquisition was performed at a magnification of 130,000× yielding a pixel size of 0.86 Å at the specimen level. Datasets were recorded with dose rates between 3.5 and 5 e−/(px·s), and the exposures were fractionated into 50 frames for an accumulated dose of 52 e−/Å2. The targeted defocus values ranged from −0.8 to −2.5 μm. The data for the human CPLANE complex (5619 movies) were collected on the same Titan Krios but using a Falcon 4 (Thermo Fisher Scientific) instead. The magnification was 96,000× with a pixel size of 0.66 Å. The dose rate was set to 18.3 e−/Å/s, and the exposure time was adjusted to account for an accumulated dose of 50 e−/Å2. The EER frames were grouped to obtain 50 fractions.

Cryo-EM data collection

Data were collected automatically with EPU (Thermo Fisher Scientific) on a Cs-corrected (CEOS GmbH, Heidelberg, Germany) Titan Krios (Thermo Fisher Scientific) electron microscope at 300 keV. Four sessions for a total of 13,056 zero-energy loss movies of the mouse CPLANE complex were recorded using a Gatan K2 summit direct electron detector (Gatan) operating in counting mode and located after a Quantum-LS energy filter (slit width of 20 eV). The acquisition was performed at a magnification of 130,000× yielding a pixel size of 0.86 Å at the specimen level. Datasets were recorded with dose rates between 3.5 and 5 e−/(px·s), and the exposures were fractionated into 50 frames for an accumulated dose of 52 e−/Å2. The targeted defocus values ranged from −0.8 to −2.5 μm. The data for the human CPLANE complex (5619 movies) were collected on the same Titan Krios but using a Falcon 4 (Thermo Fisher Scientific) instead. The magnification was 96,000× with a pixel size of 0.66 Å. The dose rate was set to 18.3 e−/Å/s, and the exposure time was adjusted to account for an accumulated dose of 50 e−/Å2. The EER frames were grouped to obtain 50 fractions.

Cryo-EM image processingReal-time data evaluation along with acquisition with EPU (Thermo Fisher Scientific) was performed with cryoFLARE (64). Within cryoFLARE, drift correction was performed with the Relion 3 motioncor implementation, where a motion-corrected sum of all frames was generated with and without applying a dose weighting scheme. CTF was fitted using GCTF (65) on the non–dose-weighted sums. After CTF estimation, only the data that showed a CTF fit better than 5 Å were exported from cryoFLARE and used for further processing with Relion 3 (66) and cryoSPARCv3 (67).For the human CPLANE complex (fig. S2), the raw movies were imported into Warp pipeline and subjected to alignment and patch motion correction. The resulting motion-corrected and dose-weighted micrographs were imported in cryoSPARC. Blob picking and four rounds of two-dimensional (2D) classification were necessary to obtain a set 2D class averages that was used for template picking. The resulting particles were subjected to three rounds of 2D classification to clean up the dataset and lead a subset of 238,978 particles. For ab initio model generation, 50,000 particles were randomly selected (fig. S2) and used to reconstruct one initial 3D envelope. An initial round of 3D refinement led to a reconstruction at 4.1 Å resolution. 3D variability analysis was used to investigate the presence of discrete subclasses. The dataset was divided into four clusters and the resulting models and particles associated to each of them were subjected to 3D heterogeneous refinement. This led to a homogeneous subset of 129,902 particles that were further refined at the final resolution of 3.35 Å.For the mouse CPLANE complex (fig. S3), raw movies from four datasets were imported into cryoSPARC and subjected to alignment and patch motion correction. CTF parameters were estimated with patch CTF. Particles were picked via blob picker. Each dataset was first processed independently for three rounds of 2D classification. For ab initio model generation, 88,000 particles were randomly selected from dataset 1 (fig. S3) and used to reconstruct four initial 3D maps. The 3D initial map that showed better resolved secondary structural features after refinement to higher resolution was kept for further processing. Then, the best particles from each dataset were merged and subjected to another round of 2D classification leading to 441,620 particles. These particles were used to perform an initial round of 3D refinement leading to a reconstruction at 3.4 Å resolution. These refined particles were exported to Relion to perform 3D classification using local searches. A total of 267,271 particles included in the best 3D class were reimported in cryoSPARC to assess, if present, conformation variability in the datasets. Three discrete structures were identified, and the particles included in the model that showed better refined Rsg1 densities were pooled together and subjected to another round of ab initio model generation to further clean up the dataset before refining the model to the final resolution of 3.35 Å.The resolution values reported for all reconstructions are based on the gold standard Fourier shell correlation (FSC) curve at 0.143 criterion. Maps were sharpened with DeepEMhancer (68), and after having obtained an initial model, sharpening was repeated with Locscale (69) starting from the initial map.

Cryo-EM image processing

Real-time data evaluation along with acquisition with EPU (Thermo Fisher Scientific) was performed with cryoFLARE (64). Within cryoFLARE, drift correction was performed with the Relion 3 motioncor implementation, where a motion-corrected sum of all frames was generated with and without applying a dose weighting scheme. CTF was fitted using GCTF (65) on the non–dose-weighted sums. After CTF estimation, only the data that showed a CTF fit better than 5 Å were exported from cryoFLARE and used for further processing with Relion 3 (66) and cryoSPARCv3 (67).

For the human CPLANE complex (fig. S2), the raw movies were imported into Warp pipeline and subjected to alignment and patch motion correction. The resulting motion-corrected and dose-weighted micrographs were imported in cryoSPARC. Blob picking and four rounds of two-dimensional (2D) classification were necessary to obtain a set 2D class averages that was used for template picking. The resulting particles were subjected to three rounds of 2D classification to clean up the dataset and lead a subset of 238,978 particles. For ab initio model generation, 50,000 particles were randomly selected (fig. S2) and used to reconstruct one initial 3D envelope. An initial round of 3D refinement led to a reconstruction at 4.1 Å resolution. 3D variability analysis was used to investigate the presence of discrete subclasses. The dataset was divided into four clusters and the resulting models and particles associated to each of them were subjected to 3D heterogeneous refinement. This led to a homogeneous subset of 129,902 particles that were further refined at the final resolution of 3.35 Å.

For the mouse CPLANE complex (fig. S3), raw movies from four datasets were imported into cryoSPARC and subjected to alignment and patch motion correction. CTF parameters were estimated with patch CTF. Particles were picked via blob picker. Each dataset was first processed independently for three rounds of 2D classification. For ab initio model generation, 88,000 particles were randomly selected from dataset 1 (fig. S3) and used to reconstruct four initial 3D maps. The 3D initial map that showed better resolved secondary structural features after refinement to higher resolution was kept for further processing. Then, the best particles from each dataset were merged and subjected to another round of 2D classification leading to 441,620 particles. These particles were used to perform an initial round of 3D refinement leading to a reconstruction at 3.4 Å resolution. These refined particles were exported to Relion to perform 3D classification using local searches. A total of 267,271 particles included in the best 3D class were reimported in cryoSPARC to assess, if present, conformation variability in the datasets. Three discrete structures were identified, and the particles included in the model that showed better refined Rsg1 densities were pooled together and subjected to another round of ab initio model generation to further clean up the dataset before refining the model to the final resolution of 3.35 Å.

The resolution values reported for all reconstructions are based on the gold standard Fourier shell correlation (FSC) curve at 0.143 criterion. Maps were sharpened with DeepEMhancer (68), and after having obtained an initial model, sharpening was repeated with Locscale (69) starting from the initial map.

Model building, refinement, and validationSince closest homologs had poor sequence identities (11 to 20%) and/or low coverage, we used TrRosetta (70), a deep learning-based structure prediction pipeline, to generate initial models. The β-propeller and α-helical domains predicted for Wdpcp were evident from the map and, after identifying the order, blades and helices extracted from the TrRosetta model could be fitted sequentially. The map also contained prominent features of extended β sheets interspersed by α helices indicative of the LDs predicted for Intu and Fuz. The TrRosetta model of Fuz deviated by ~5.7 Å RMSD from the final model and could be fitted as a rigid body. In contrast, the spatial arrangement of LDs in the predicted Intu model was far off from the final model (~20 Å RMSD). However, the correct prediction of the overall fold of the individual LDs enabled mapping of the domains to the density based on secondary structure and side-chain correlation. The density corresponding to RSG1 showed reduced local resolution compared to the rest of the complex; side chains could, however, still be assigned for most parts. During the course of this study, AlphaFold (AF) v2.0 was released (33). We used the predicted models from AF to cross-validate the cryo-EM models. The RMSDs of the predicted models compared to the final cryo-EM models (MmCPLANE) were 0.8 Å for RSG1 (224 residues), 1.2 Å for Fuz (392 residues), 1.6 Å for Intu (336 residues), and 2.2 Å for Wdpcp (536 residues). Models were docked in COOT (71), or in PHENIX (72). Local backbone, side chain, and register deviations were fixed manually in COOT and ISOLDE/ChimeraX (73, 74), and the model was subjected to an automated structure refinement protocol using ROSETTA as previously described (75). Several further rounds of manual rebuilding in COOT and ISOLDE/ChimeraX followed by Cartesian refinement with the ROSETTA FastRelax protocol in combination with density scoring terms were carried out. To test for overfitting, the structure was refined against the first half-map and the FSC correlation was calculated for both half-maps. B factors were fitted at a final stage using ROSETTA. An in-house pipeline was used to run the ROSETTA FastRelax protocol (https://github.com/fmi-basel/RosEM). A model for the human CPLANE structure was built using the mouse model as a template. The sequences of HsWdpcp, HsIntu, and HsFuzzy were aligned to the mouse homologs using phenix.muscle and threaded onto the template models using rosetta.partial_thread (76). The model was iteratively refined by manual rebuilding in COOT followed by real-space refinement using ROSETTA FastRelax as described above. Both models were validated using PHENIX, MOLPROBITY (77), and EMRINGER (78). RMSDs were calculated with the Cealign method implemented in PyMOL (The PyMOL Molecular Graphics System, Version 2.0 Schrödinger LLC). Figures were prepared with ChimeraX and PyMOL.

Model building, refinement, and validation

Since closest homologs had poor sequence identities (11 to 20%) and/or low coverage, we used TrRosetta (70), a deep learning-based structure prediction pipeline, to generate initial models. The β-propeller and α-helical domains predicted for Wdpcp were evident from the map and, after identifying the order, blades and helices extracted from the TrRosetta model could be fitted sequentially. The map also contained prominent features of extended β sheets interspersed by α helices indicative of the LDs predicted for Intu and Fuz. The TrRosetta model of Fuz deviated by ~5.7 Å RMSD from the final model and could be fitted as a rigid body. In contrast, the spatial arrangement of LDs in the predicted Intu model was far off from the final model (~20 Å RMSD). However, the correct prediction of the overall fold of the individual LDs enabled mapping of the domains to the density based on secondary structure and side-chain correlation. The density corresponding to RSG1 showed reduced local resolution compared to the rest of the complex; side chains could, however, still be assigned for most parts. During the course of this study, AlphaFold (AF) v2.0 was released (33). We used the predicted models from AF to cross-validate the cryo-EM models. The RMSDs of the predicted models compared to the final cryo-EM models (MmCPLANE) were 0.8 Å for RSG1 (224 residues), 1.2 Å for Fuz (392 residues), 1.6 Å for Intu (336 residues), and 2.2 Å for Wdpcp (536 residues). Models were docked in COOT (71), or in PHENIX (72). Local backbone, side chain, and register deviations were fixed manually in COOT and ISOLDE/ChimeraX (73, 74), and the model was subjected to an automated structure refinement protocol using ROSETTA as previously described (75). Several further rounds of manual rebuilding in COOT and ISOLDE/ChimeraX followed by Cartesian refinement with the ROSETTA FastRelax protocol in combination with density scoring terms were carried out. To test for overfitting, the structure was refined against the first half-map and the FSC correlation was calculated for both half-maps. B factors were fitted at a final stage using ROSETTA. An in-house pipeline was used to run the ROSETTA FastRelax protocol (https://github.com/fmi-basel/RosEM). A model for the human CPLANE structure was built using the mouse model as a template. The sequences of HsWdpcp, HsIntu, and HsFuzzy were aligned to the mouse homologs using phenix.muscle and threaded onto the template models using rosetta.partial_thread (76). The model was iteratively refined by manual rebuilding in COOT followed by real-space refinement using ROSETTA FastRelax as described above. Both models were validated using PHENIX, MOLPROBITY (77), and EMRINGER (78). RMSDs were calculated with the Cealign method implemented in PyMOL (The PyMOL Molecular Graphics System, Version 2.0 Schrödinger LLC). Figures were prepared with ChimeraX and PyMOL.
