---
pmid: '25355947'
title: COMMD1 is linked to the WASH complex and regulates endosomal trafficking of
  the copper transporter ATP7A.
authors:
- Phillips-Krawczak CA
- Singla A
- Starokadomskyy P
- Deng Z
- Osborne DG
- Li H
- Dick CJ
- Gomez TS
- Koenecke M
- Zhang JS
- Dai H
- Sifuentes-Dominguez LF
- Geng LN
- Kaufmann SH
- Hein MY
- Wallis M
- McGaughran J
- Gecz J
- Sluis Bv
- Billadeau DD
- Burstein E
journal: Mol Biol Cell
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4279232
doi: 10.1091/mbc.E14-06-1073
---

# COMMD1 is linked to the WASH complex and regulates endosomal trafficking of the copper transporter ATP7A.
**Authors:** Phillips-Krawczak CA, Singla A, Starokadomskyy P, Deng Z, Osborne DG, Li H, Dick CJ, Gomez TS, Koenecke M, Zhang JS, Dai H, Sifuentes-Dominguez LF, Geng LN, Kaufmann SH, Hein MY, Wallis M, McGaughran J, Gecz J, Sluis Bv, Billadeau DD, Burstein E
**Journal:** Mol Biol Cell (2015)
**DOI:** [10.1091/mbc.E14-06-1073](https://doi.org/10.1091/mbc.E14-06-1073)
**PMC:** [PMC4279232](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4279232/)

## Abstract

1. Mol Biol Cell. 2015 Jan 1;26(1):91-103. doi: 10.1091/mbc.E14-06-1073. Epub
2014  Oct 29.

COMMD1 is linked to the WASH complex and regulates endosomal trafficking of the 
copper transporter ATP7A.

Phillips-Krawczak CA(1), Singla A(2), Starokadomskyy P(2), Deng Z(3), Osborne 
DG(1), Li H(2), Dick CJ(1), Gomez TS(1), Koenecke M(2), Zhang JS(4), Dai H(5), 
Sifuentes-Dominguez LF(2), Geng LN(2), Kaufmann SH(5), Hein MY(6), Wallis M(7), 
McGaughran J(8), Gecz J(9), Sluis Bv(10), Billadeau DD(11), Burstein E(12).

Author information:
(1)Department of Immunology.
(2)Department of Internal Medicine and.
(3)Department of Immunology, Department of Pathophysiology, Qiqihar Medical 
University, Qiqihar, Heilongjiang 161006, China.
(4)Department of Immunology, School of Pharmaceutical Sciences and Key 
Laboratory of Biotechnology and Pharmaceutical Engineering, Wenzhou Medical 
University, Wenzhou, Zhejiang 325035, China.
(5)Department of Molecular Pharmacology and Experimental Therapeutics, and.
(6)Max Planck Institute of Biochemistry, 82152 Martinsried, Germany.
(7)Genetic Health Queensland at the Royal Brisbane and Women's Hospital, 
Herston, Queensland 4029, Australia.
(8)Genetic Health Queensland at the Royal Brisbane and Women's Hospital, 
Herston, Queensland 4029, Australia School of Medicine, University of 
Queensland, Brisbane, Queensland 4072, Australia.
(9)Robinson Institute and Department of Paediatrics, University of Adelaide, 
Adelaide, South Australia 5005, Australia.
(10)Section of Molecular Genetics at the Department of Pediatrics, University 
Medical Center Groningen, University of Groningen, 9713 Groningen, Netherlands.
(11)Department of Immunology, Department of Biochemistry and Molecular Biology, 
Mayo Clinic College of Medicine, Mayo Clinic, Rochester, MN 55905 
billadeau.daniel@mayo.edu ezra.burstein@utsouthwestern.edu.
(12)Department of Internal Medicine and Department of Molecular Biology, UT 
Southwestern Medical Center, Dallas, TX 75390-9151 billadeau.daniel@mayo.edu 
ezra.burstein@utsouthwestern.edu.

COMMD1 deficiency results in defective copper homeostasis, but the mechanism for 
this has remained elusive. Here we report that COMMD1 is directly linked to 
early endosomes through its interaction with a protein complex containing 
CCDC22, CCDC93, and C16orf62. This COMMD/CCDC22/CCDC93 (CCC) complex interacts 
with the multisubunit WASH complex, an evolutionarily conserved system, which is 
required for endosomal deposition of F-actin and cargo trafficking in 
conjunction with the retromer. Interactions between the WASH complex subunit 
FAM21, and the carboxyl-terminal ends of CCDC22 and CCDC93 are responsible for 
CCC complex recruitment to endosomes. We show that depletion of CCC complex 
components leads to lack of copper-dependent movement of the copper transporter 
ATP7A from endosomes, resulting in intracellular copper accumulation and modest 
alterations in copper homeostasis in humans with CCDC22 mutations. This work 
provides a mechanistic explanation for the role of COMMD1 in copper homeostasis 
and uncovers additional genes involved in the regulation of copper transporter 
recycling.

© 2015 Phillips-Krawczak, Singla, et al. This article is distributed by The 
American Society for Cell Biology under license from the author(s). Two months 
after publication it is available to the public under an 
Attribution–Noncommercial–Share Alike 3.0 Unported Creative Commons License 
(http://creativecommons.org/licenses/by-nc-sa/3.0).

DOI: 10.1091/mbc.E14-06-1073
PMCID: PMC4279232
PMID: 25355947 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONCopper is an essential transition metal that is required for the enzymatic activity of several vital proteins and pathways. However, given the high reactivity of copper, its levels are carefully regulated, and consequently, alterations in pathways that handle copper can result in disease states. In humans, mutations in the copper transporters are the main causes of disorders of copper metabolism (Wang et al., 2011), including copper deficiency, due to mutations in ATP7A, and copper accumulation, also known as Wilson's disease, which is due to mutations in the closely related transporter ATP7B. A key functional aspect of ATP7A and ATP7B is their copper-responsive trafficking between the trans-Golgi network (TGN) and vesicles in the cell periphery, which is the subject of significant study (Wang et al., 2011).In addition to these genetic disorders in humans, several Mendelian copper metabolic disorders in animals have provided critical clues regarding copper homeostatic mechanisms. Among these is an autosomal recessive form of copper accumulation affecting some purebred dogs, most notably Bedlington terriers (van de Sluis et al., 2002), which is characterized by chronic hepatitis and cirrhosis, mimicking several aspects of Wilson's disease. This disorder is due to the deletion of exon 2 of the copper metabolism MURR1 domain–containing 1 (COMMD1) gene. COMMD1 is the founding member of the COMMD protein family, which includes 10 highly conserved factors present in most eukaryotes (Burstein et al., 2005) that are also linked to proinflammatory signaling (Maine et al., 2007; Starokadomskyy et al., 2013; Li et al., 2014a), hypoxia adaptation (van de Sluis et al., 2007, 2010), and electrolyte transport (Biasio et al., 2004; Drevillon et al., 2011). With regard to copper metabolism, COMMD1 has been reported to bind to both ATP7A and ATP7B, but the specific regulation that it provides has not been conclusively elucidated (Tao et al., 2003; Vonk et al., 2011; Materia et al., 2012). According to one report, the retrograde transport of ATP7B from peripheral endosomal vesicles back to the TGN in response to copper deprivation is deregulated in the absence of COMMD1 (Miyayama et al., 2010). However, the precise mechanism by which COMMD1 may affect ATP7A/7B trafficking has remained elusive (Weiss et al., 2008; Miyayama et al., 2010; Materia et al., 2012).Recent studies have shown that movement of the copper transporter ATP7A from endosomal vesicles to the plasma membrane (Steinberg et al., 2013) is dependent on sorting nexin 27 (SNX27) acting in conjunction with the retromer complex. This multisubunit protein complex was initially identified in yeast (Seaman et al., 1998) and has been subsequently identified in a wide range of organisms, including mammals (Haft et al., 2000). Retromer consists of a cargo selection complex comprising a trimer of VPS26, VPS29, and VPS35, which acts in concert with other cellular proteins to orchestrate the movement of selected endosomal cargos. This is accomplished first through the recruitment of sorting nexins (SNX1, 2, 5, and 6), which facilitate membrane deformation and tubulation for the generation of the nascent cargo-loaded vesicles (Cullen and Korswagen, 2012). In addition, retromer recruits the Wiskott–Aldrich syndrome protein and SCAR homologue (WASH) complex to endosomes (Gomez and Billadeau, 2009; Harbour et al., 2012). This pentameric complex, containing WASH, FAM21, Strumpellin, SWIP, and CCDC53, promotes branched F-actin deposition on endosomes by activating the ubiquitously expressed Arp2/3 complex (Derivery et al., 2009; Gomez and Billadeau, 2009). This activity facilitates receptor trafficking and is required to maintain segregation of endosomal sorting domains (Derivery et al., 2009; Gomez and Billadeau, 2009; Gomez et al., 2012; Harbour et al., 2010; Zech et al., 2011).Recently we found that COMMD1 and members of the COMMD protein family bind to coiled-coil domain–containing protein 22 (CCDC22), a poorly understood factor whose encoding gene is mutated in some families with X-linked intellectual disability (Voineagu et al., 2012; Starokadomskyy et al., 2013; Kolanczyk et al., 2014). Of interest, CCDC22 was recently reported to interact with FAM21, a WASH complex component (Harbour et al., 2012), but the functional significance of this interaction has not been investigated. In this study, we report that COMMD1 forms a complex containing CCDC22, as well as the previously uncharacterized factors CCDC93 and C16orf62. This complex, referred to hereafter as the CCC (COMMD/CCDC22/CCDC93) complex, is recruited by FAM21 to endosomes. In the absence of the CCC complex or its recruitment to endosomes, ATP7A is mislocalized and does not traffic in response to copper availability, resulting in altered copper handling.

INTRODUCTION

Copper is an essential transition metal that is required for the enzymatic activity of several vital proteins and pathways. However, given the high reactivity of copper, its levels are carefully regulated, and consequently, alterations in pathways that handle copper can result in disease states. In humans, mutations in the copper transporters are the main causes of disorders of copper metabolism (Wang et al., 2011), including copper deficiency, due to mutations in ATP7A, and copper accumulation, also known as Wilson's disease, which is due to mutations in the closely related transporter ATP7B. A key functional aspect of ATP7A and ATP7B is their copper-responsive trafficking between the trans-Golgi network (TGN) and vesicles in the cell periphery, which is the subject of significant study (Wang et al., 2011).

In addition to these genetic disorders in humans, several Mendelian copper metabolic disorders in animals have provided critical clues regarding copper homeostatic mechanisms. Among these is an autosomal recessive form of copper accumulation affecting some purebred dogs, most notably Bedlington terriers (van de Sluis et al., 2002), which is characterized by chronic hepatitis and cirrhosis, mimicking several aspects of Wilson's disease. This disorder is due to the deletion of exon 2 of the copper metabolism MURR1 domain–containing 1 (COMMD1) gene. COMMD1 is the founding member of the COMMD protein family, which includes 10 highly conserved factors present in most eukaryotes (Burstein et al., 2005) that are also linked to proinflammatory signaling (Maine et al., 2007; Starokadomskyy et al., 2013; Li et al., 2014a), hypoxia adaptation (van de Sluis et al., 2007, 2010), and electrolyte transport (Biasio et al., 2004; Drevillon et al., 2011). With regard to copper metabolism, COMMD1 has been reported to bind to both ATP7A and ATP7B, but the specific regulation that it provides has not been conclusively elucidated (Tao et al., 2003; Vonk et al., 2011; Materia et al., 2012). According to one report, the retrograde transport of ATP7B from peripheral endosomal vesicles back to the TGN in response to copper deprivation is deregulated in the absence of COMMD1 (Miyayama et al., 2010). However, the precise mechanism by which COMMD1 may affect ATP7A/7B trafficking has remained elusive (Weiss et al., 2008; Miyayama et al., 2010; Materia et al., 2012).

Recent studies have shown that movement of the copper transporter ATP7A from endosomal vesicles to the plasma membrane (Steinberg et al., 2013) is dependent on sorting nexin 27 (SNX27) acting in conjunction with the retromer complex. This multisubunit protein complex was initially identified in yeast (Seaman et al., 1998) and has been subsequently identified in a wide range of organisms, including mammals (Haft et al., 2000). Retromer consists of a cargo selection complex comprising a trimer of VPS26, VPS29, and VPS35, which acts in concert with other cellular proteins to orchestrate the movement of selected endosomal cargos. This is accomplished first through the recruitment of sorting nexins (SNX1, 2, 5, and 6), which facilitate membrane deformation and tubulation for the generation of the nascent cargo-loaded vesicles (Cullen and Korswagen, 2012). In addition, retromer recruits the Wiskott–Aldrich syndrome protein and SCAR homologue (WASH) complex to endosomes (Gomez and Billadeau, 2009; Harbour et al., 2012). This pentameric complex, containing WASH, FAM21, Strumpellin, SWIP, and CCDC53, promotes branched F-actin deposition on endosomes by activating the ubiquitously expressed Arp2/3 complex (Derivery et al., 2009; Gomez and Billadeau, 2009). This activity facilitates receptor trafficking and is required to maintain segregation of endosomal sorting domains (Derivery et al., 2009; Gomez and Billadeau, 2009; Gomez et al., 2012; Harbour et al., 2010; Zech et al., 2011).

Recently we found that COMMD1 and members of the COMMD protein family bind to coiled-coil domain–containing protein 22 (CCDC22), a poorly understood factor whose encoding gene is mutated in some families with X-linked intellectual disability (Voineagu et al., 2012; Starokadomskyy et al., 2013; Kolanczyk et al., 2014). Of interest, CCDC22 was recently reported to interact with FAM21, a WASH complex component (Harbour et al., 2012), but the functional significance of this interaction has not been investigated. In this study, we report that COMMD1 forms a complex containing CCDC22, as well as the previously uncharacterized factors CCDC93 and C16orf62. This complex, referred to hereafter as the CCC (COMMD/CCDC22/CCDC93) complex, is recruited by FAM21 to endosomes. In the absence of the CCC complex or its recruitment to endosomes, ATP7A is mislocalized and does not traffic in response to copper availability, resulting in altered copper handling.

RESULTSLoss of CCDC22 or COMMD1 affects copper-dependent ATP7A traffickingATP7A is dynamically recycled between the TGN, where it is present when copper availability is low, and cytosolic vesicles, where it is redistributed under high copper conditions. From these cytosolic vesicles, ATP7A then reaches the plasma membrane and is responsible for cellular copper efflux (Wang et al., 2011). Whereas COMMD1 deficiency leads to copper accumulation (van de Sluis et al., 2002; Burstein et al., 2004; Vonk et al., 2011; Materia et al., 2012), a possible role for CCDC22 in copper handling has not been previously studied.We first examined the subcellular distribution of ATP7A using fibroblasts derived from a patient with a recently described point mutation in CCDC22 (c.49A>G/p.T17A). This mutation results in abnormal mRNA splicing, reduced protein expression, and impaired CCDC22–COMMD1 protein interactions (Voineagu et al., 2012; Starokadomskyy et al., 2013). Whereas control fibroblasts demonstrated dramatic redistribution of ATP7A in response to copper availability, CCDC22 T17A fibroblasts had lost the normal dynamic redistribution upon changes in copper availability (Figure 1A and Supplemental Figure S1). These cells displayed a peripheral distribution of ATP7A in large vesicles irrespective of copper levels and lacked the TGN localization of ATP7A upon copper deprivation (Figure 1, A and B). These alterations in ATP7A distribution were mirrored by COMMD1 deficiency in control fibroblasts after small interfering RNA (siRNA)–mediated silencing (Figure 1C). Consistent with these alterations in ATP7A trafficking, CCDC22 deficiency led to increased intracellular copper levels (Figure 1D), akin to what has been reported for COMMD1 deficiency (Burstein et al., 2004; Spee et al., 2007). However, patients with the CCDC22 T17A mutation did not display overt clinical signs of copper toxicosis, nor did they display biochemical evidence of copper overload, which is primarily determined in clinical settings by increased urinary copper excretion over 24 h (Figure 1, E and F). Instead, some affected individuals had elevated serum copper and serum ceruloplasmin concentrations, resembling the phenotype of the liver-specific Commd1-knockout mouse (Vonk et al., 2011) and the elevated ceruloplasmin levels of Bedlington terriers with COMMD1 mutations (Su et al., 1982). Taken together, these findings indicate an important role for both COMMD1 and CCDC22 in copper homeostasis and ATP7A trafficking.FIGURE 1:Regulation of ATP7A trafficking by CCDC22 and COMMD1. (A, B) ATP7A localization in response to copper was assessed in control and CCDC22 T17A fibroblasts via immuno­fluorescence staining for endogenous ATP7A (red) and GM130 (green). (A) Representative images. (B) Quantification of ATP7A distribution pattern in >50 cells/group. The ATP7A distribution observed in CCDC22 T17A fibroblasts was statistically different from the control line (p < 0.001). Scale bar, 20 μm. (C) ATP7A localization in control dermal fibroblasts was examined as before. Cells were transfected with either control siRNA duplexes or siRNA targeting COMMD1. After copper treatments, cells were stained for ATP7A (red) and nuclei (blue) and imaged by confocal microscopy. (D) Increased levels of cellular Cu were noted after siRNA silencing of CCDC22. (E) Pedigree of a kindred affected by a CCDC22 (c.49A>G/p.T17A) mutation. (F) The concentration of copper (Cu) in serum and urine was determined in selected probands of this kindred. Similarly, serum ceruloplasmin concentration (CP) and urinary excretion of copper over 24 h (Cu/24h) were also ascertained when possible. Normal value ranges are indicated at the top of the table. Abnormal values are marked in red.Identification of the CCC complexWe recently reported that CCDC22 can bind to all 10 members of the COMMD protein family (Starokadomskyy et al., 2013). This analysis not only revealed the interaction between CCDC22 and COMMD proteins, but it also showed that CCDC22 bound to CCDC93, a protein that shares similar domain organization (Schou et al., 2014). Another partner found in this analysis was C16orf62, a protein whose function is also unknown. Of interest, all three proteins display a similar range of evolutionary conservation (Supplemental Figure S2) and are predicted to belong to an evolutionary conserved module that coevolved with the WASH complex (Li et al., 2014b).Given that CCDC93 was also identified in two prior TAP screens using COMMD9 and COMMD10 as baits, we speculated that these proteins associate to form what we refer to as the CCC complex. This possibility was examined directly by coimmunoprecipitation experiments. First, endogenous CCDC22 readily coprecipitated endogenous CCDC93 and C16orf62, and, conversely, the immunoprecipitation of these proteins also brought down the other two components of this complex (Figure 2A). We next examined whether COMMD proteins also interacted with these newly identified partners of CCDC22. First, immunoprecipitation of endogenous CCDC93 resulted in the coprecipitation of endogenous COMMD1 and COMMD6 but not IKK1 (Figure 2B). Second, endogenous COMMD1 coprecipitated endogenous C16orf62 and CCDC93 but not IKK1 (Figure 2C). The reciprocal interactions between CCDC22, CCDC93, C16orf62, and COMMD1 could be readily demonstrated using a coiled-coil protein (NEMO) as an additional negative control (Supplemental Figure S6A). Furthermore, all 10 human COMMD family members fused to glutathione S-transferase (GST) uniformly coprecipitated endogenous CCDC93 (Supplemental Figure S3A). Taken together, the interaction experiments validated the notion that these factors form a multiprotein complex.FIGURE 2:Discovery of the COMMD1–CCDC22 interactome. (A) Coimmunoprecipitation of endogenous CCDC22, CCDC93, and C16orf62 in HEK293T cell lysates confirm their interactions. (B) Endogenous CCDC93 coimmunoprecipitates with COMMD1 and COMMD6 in HEK293T cell lysates. (C) Endogenous COMMD1 coimmunoprecipitates with C16orf62 and CCDC93 in HEK 293 cell lysates. (D) Expression of CCC complex subunits was examined by immunoblotting in CCDC22 T17A fibroblasts. (E) Expression of CCC complex subunits was examined by immunoblotting in Commd1-deficient fibroblasts (Commd1−/−) and its isogenic control (Commd1F/F). (F, G) Domain organization of CCDC93 and CCDC22. Truncation mutants of CCDC93 (F) or CCDC22 (G) were expressed in HEK293T cells and subsequently immunoprecipitated from cell lysates to assess their ability to interact with CCDC22, CCDC93, COMMD1, and C16orf62, as indicated. Glyceraldehyde 3-phosphate dehydrogenase (GAPDH) and NEMO served as specificity controls. The asterisk in G denotes bands resulting from the hemagglutinin (HA) immunoblot and are not the GAPDH band, which migrates faster.CCC subunits stabilize expression of components of the complexFor many stable complexes, including WASH and retromer, expression of individual subunits is required to stabilize other components within the complex. We examined this notion first in fibroblasts from patients with the CCDC22 T17A mutation and found that in addition to the expected reduction in CCDC22 expression, the levels of CCDC93 were drastically reduced in these cells (Figure 2D). This phenomenon was recapitulated by RNA interference–mediated silencing of either CCDC22 or CCDC93 (Supplemental Figure S3B) and could not be explained by corresponding mRNA changes (Supplemental Figure S3C), consistent with a posttranscriptional event. In addition, deficiency of Commd1 in mouse embryo fibroblasts (MEFs) also led to reduced expression of Ccdc22 and Ccdc93 (Figure 2E). On the other hand, COMMD1 expression was not affected by siRNA of either CCDC22 or CCDC93 (Supplemental Figure S3B) or by the CCDC22 T17A mutation (Starokadomskyy et al., 2013). With regard to C16orf62, its expression was minimally affected in Commd1−/− MEFs but was more noticeably reduced in CCDC22 T17A fibroblasts (Figure 2, D and E); conversely, silencing of C16orf62 did not affect expression of other CCC subunits (Supplemental Figure S4C), which suggests that this subunit is less vital to the stability of the complex.Organization of the CCC complexNext we evaluated the protein domains in CCDC22 and CCDC93 that are required for their interaction. As depicted in Figure 2, F and G, both CCDC93 and CCDC22 have a shared amino- terminal domain belonging to the NDC80 and NUF2-calponin homology domain (NN-CH), followed by coiled-coil (CC) domains (Schou et al., 2014). On expressing domain truncation mutants of CCDC93, we identified that the NN-CH and the middle CC domain of CCDC93 (1–430) were required for CCDC22 binding, although some binding was possible to the CC region alone (207–431; Figure 2F). Conversely, the NN-CH region of CCDC22 and an extension just before its carboxy-terminal CC domain (1–447) was required for CCDC93 binding (Figure 2G). Moreover, recombinant CCDC22 1–447 purified from Escherichia coli was able to coprecipitate recombinant CCDC93 1-430 in vitro, consistent with a direct interaction through their N-terminal regions (Supplemental Figure S4A).The domain analysis also indicated that the binding of CCDC22 to COMMD1 was separable from its interaction to CCDC93 (amino acids 1–321 vs. 1–447; Figure 2G). This suggested that CCDC93 could be dispensable for CCDC22–COMMD1 binding, and, indeed, CCDC22 interactions with COMMD1 were not affected by CCDC93 silencing (Supplemental Figure S4B). On the other hand, coprecipitation between CCDC22 and C16orf62 was affected by CCDC93 deficiency, suggesting interdependence between these factors in their ability to form a larger complex. Finally, binding of C16orf62 to CCDC22 or CCDC93 could not be mapped to a single domain (Figure 2, F and G), nor did C16orf62 silencing impair CCDC93 binding with CCDC22 and COMMD1 (Supplemental Figure S4C), suggesting that CCDC22/CCDC93/COMMD1 interactions are not dependent on C16orf62. Taken together, these data indicated that CCDC22–CCDC93 interactions are directly mediated by their amino termini, and similarly, the interactions between COMMD1 and CCDC22 are likely direct and independent of other components. On the other hand, C16orf62 likely requires extensive interactions with both CCDC22 and CCDC93 to be assembled within the CCC complex.The CCC and WASH complexes interact with each otherTo gain further insight into the function of the CCC complex, protein–protein interaction maps were developed using data deposited in the National Center for Biotechnology Information (NCBI). This analysis indicated that the predicted interactome for the human CCC complex includes complexes involved in vesicular sorting, such as the WASH complex, exocyst, and BLOC-1 (Figure 3A). Remarkably similar results were obtained when examining protein–protein interaction networks in Drosophila (Supplemental Figure S5).FIGURE 3:Interaction and colocalization of the CCC and WASH complexes. (A) Interaction map between CCC complex components and other cellular complexes were drawn using data extracted from NCBI. Intracomplex interactions, where useful, are also indicated. Human studies are depicted, and interactions identified in this study are depicted in red. (B) Coimmunoprecipitation of endogenous CCC subunits (CCDC22, CCDC93, and C16orf62) with the WASH components (FAM21 and WASH) was noted using HeLa cell lysates. (C) Colocalization of endogenous COMMD1 (green) and CCDC93 (red) or C16orf62 (red) was examined by immunostaining in HeLa cells. Scale bar, 5 μm. (D) Colocalization of endogenous COMMD1 (green) with WASH (red, top), and the retromer subunit VPS35 (red, bottom) was examined by immunofluorescence staining of HeLa cells. Scale bar, 5 μm.The possible interplay between the CCC and WASH complexes is in agreement with a recent report in which CCDC22 and CCDC93 were noted to interact with the WASH complex subunit FAM21 (Harbour et al., 2012). Moreover, the retromer complex, which is responsible for recruiting and activating WASH, was recently implicated in ATP7A sorting (Steinberg et al., 2013). To examine these possible interactions, we first immunoprecipitated FAM21 or WASH and found that each readily coprecipitated endogenous CCC components (C16orf62, CCDC93, and CCDC22; Figure 3B). Conversely, immunoprecipitation of endogenous CCC components also coprecipitated endogenous WASH (Supplemental Figure S6A) and FAM21 (Supplemental Figure S6B). Of interest, retromer did not appear to be required for WASH–CCC interactions, as demonstrated by the fact that FAM21 could precipitate CCC components to the same extent in control and VPS35-deficient cells (Supplemental Figure S6C).Next we examined whether the WASH complex played any role in the stability of the CCC complex. In this regard, we noted that Wash deficiency in MEFs did not impair the expression of CCC complex subunits or their ability to coimmunoprecipitate (Supplemental Figure S6D), and we observed a similar finding after silencing FAM21 in HeLa cells (Supplemental Figure S6E). These data suggested that despite their interactions, the CCC and WASH complexes are distinct and independent.CCC complex components colocalize with WASH and retromer in the early endosomeThe WASH complex is known to localize primarily to early endosomes, where it is recruited by retromer. In view of this, we next evaluated whether the CCC complex is similarly recruited to early endosomes. First, using a highly specific monoclonal antibody against COMMD1 (Supplemental Figure S7A), we found that this protein has a high degree of colocalization with other CCC complex components, namely CCDC93 and C16orf62 (Figure 3C and Supplemental Figure S7C). COMMD1-containing cytosolic foci demonstrated colocalization with WASH and the retromer subunit VPS35 (Figure 3D and Supplemental Figure S7C). Moreover, COMMD1 foci colocalized with the early endosomal marker RUFY1 and were closely juxtaposed to EEA1+ foci on endomembranes (Supplemental Figure S7, B and C). This finding closely resembles the localization of the WASH complex itself (Gomez and Billadeau, 2009). These structures were clearly distinct from very early endocytic vesicles containing APPL1 and did not colocalize with lysosomal (LAMP1) or TGN (TGN46) markers (Supplemental Figure S7, B and C). Transient expression of fluorescently tagged CCC complex proteins revealed simultaneous colocalization for COMMD1, CCDC22, and CCDC93, as well as for COMMD1, C16orf62, and CCDC93 (Supplemental Figure S7D). Taken together, these data indicate that the CCC complex localizes to a subcompartment of the early endosome that is enriched in retromer and WASH.The CCC complex regulates ATP7A traffickingTaken together, our experiments indicated that COMMD1, CCDC22, CCDC93, and C16orf62 interact to form a higher-order stable complex that colocalizes with retromer and WASH on early endosomes. In view of this, we tested whether other components of this complex also regulate ATP7A trafficking, akin to the effects noted for CCDC22 and COMMD1 (Figure 1). Stable silencing of CCDC93 in HeLa cells (which results in deficiency of both CCDC93 and CCDC22) led to altered ATP7A recycling in response to copper availability, which demonstrated peripheral redistribution of the transporter and loss of its TGN localization upon copper deprivation, a finding that was also seen upon VPS35 silencing (Figure 4, A and B).FIGURE 4:Depletion of CCC complex components impairs trafficking of ATP7A and TGN46. (A) The indicated HeLa control and CCDC93- or VPS35-knockdown cell lines (sh, short hairpin against the indicated target) were maintained in high-copper conditions or transitioned to low copper as indicated in Materials and Methods. The cells were subsequently stained for ATP7A (red) and DNA (blue) and imaged by confocal fluorescence microscopy. Scale bar, 20 μm. (B) Quantification of ATP7A localization in the indicated knockdown cell lines was examined (>50 cells/group were scored). The ATP7A distribution observed in shCCDC93 and shVPS35 was statistically different from the control line (p < 0.001). (C) Plasma membrane expression of ATP7A was determined in the same cell lines after high- and low-copper conditions using biotinylation as described in Materials and Methods. N-Cadherin served as a loading control. (D) The indicated HeLa knockdown cell lines were stained for GM130 (green), TGN46 (red), F-actin (dark blue), and DNA (light blue) and imaged by confocal microscopy. Scale bar, 20 μm. (E) TGN46 localization was quantified in the indicated HeLa knockdown lines (>50 cells/group were scored). The TGN46 distribution observed in shCCDC22, shC16orf62, and shVPS35 was statistically different from that in the control line (p < 0.001).Recently it was reported that retromer, in conjunction with SNX27, also regulates the delivery of ATP7A from endosomes to the plasma membrane (Steinberg et al., 2013). Using biotinylation to mark plasma membrane proteins, we found that VPS35 deficiency, and to a lesser extent CCDC93 deficiency, resulted in decreased levels of ATP7A at the membrane during high copper exposure (Figure 4C). Furthermore, retromer deficiency can result in inappropriate sorting of endosomal cargo to lysosomes, leading to overall reductions in protein expression. However, whereas VPS35-deficient cells were depleted in ATP7A (Supplemental Figure S8, C and D) in agreement with previous results (Steinberg et al., 2013), deficiency of CCC components did not affect overall levels of ATP7A in a variety of models (Supplemental Figure S8, A–C).In addition to ATP7A, retromer and WASH regulate the retrograde transport to the TGN of a variety of endosomal cargo, including TGN46 (Hao et al., 2013). Under control conditions, we found that this protein was localized in close proximity to the cis-Golgi marker GM130 as expected; however, upon silencing of CCC complex components or the retromer subunit VPS35, TGN46 was diffusely localized in peripheral cytosolic vesicles (Figure 4, D and E). Thus loss of the CCC complex phenocopies retromer depletion and demonstrates an important role for the CCC complex in the trafficking of retromer cargo such as ATP7A and TGN46.FAM21 is required for endosomal recruitment of the CCC complexGiven the interaction and colocalization of the CCC and WASH complexes, we examined whether their endosomal localization were interrelated. In this regard, we noted that stable silencing of FAM21 in HeLa cells (shFAM21) drastically reduced cytosolic puncta for COMMD1, as well as for the CCC complex components CCDC93 and C16orf62 (Figure 5A and Supplemental Figure S9A). In contrast to this result, loss of Wash in mouse fibroblasts did not abolish COMMD1 endosomal localization (Supplemental Figure S9B). Instead of discrete puncta, COMMD1 localized to collapsed endosomal structures, which retained FAM21 localization, as previously reported (Gomez et al., 2012). Furthermore, silencing of VPS35, a subunit of retromer that in turn is required for FAM21 recruitment to endosomes, also led to loss of COMMD1 puncta (Supplemental Figure S9C). Conversely, FAM21 localization was unaffected in CCDC22 T17A fibroblasts (Figure 5B); instead, there were dramatic changes in COMMD1 localization, with loss of endosomal-localized COMMD1 and its redistribution with a more diffuse cytosolic and nuclear pattern in these cells. Taken together, these data indicated that FAM21 is required for the recruitment of the CCC complex to the endosome, a process that in turn also depends on CCDC22 expression.FIGURE 5:FAM21 recruits the CCC complex to endosomes, and this is necessary for ATP7A trafficking. (A) Formation of cytosolic puncti of FAM21, COMMD1, CCDC93, and C16orf62 was assessed by immunofluorescence staining after stable silencing of FAM21 (shFAM21). Average number of puncta per cell (of ∼75 imaged cells) for the indicated CCC complex member. Representative images, including EEA1 staining to show the localization of early endosomes. Scale bar, 5 μm. (B) Endogenous COMMD1 (red) and FAM21 (green) localization was evaluated by immunofluorescence staining in control fibroblasts and fibroblasts derived from patients with the CCDC22 T17A mutation, as indicated. Scale bar, 5 μm. (C) Localization of ATP7A (red) in control and CCDC93 deleted cells (CRISPR 93). Localization of retromer (VPS35, green) and the CCC complex (COMMD1, purple) in cytosolic vesicles and their relative colocalization are also depicted. Scale bar, 10 μm. (D, E) The indicated HA-tagged CCDC93 or CCDC22 construct was transfected into HEK293T cells and subsequently immunoprecipitated with anti-FAM21 or control immunoglobulin G (IgG). Associated proteins were identified by immunoblotting with anti-HA. Note that the CCDC93 1–430 fragment was also detected in the IgG immunoprecipitation, indicating that this interaction with FAM21 is likely nonspecific (marked by an arrowhead).To demonstrate the functional significance of these events, we deleted CCDC93 in HeLa cells using the CRISPR/Cas9 system. Deletion of CCDC93 led to concurrent depletion of CCDC22 (Supplemental Figure S10A), as well as diffuse localization of ATP7A at both low- and high-copper conditions (Figure 5C, left), confirming once more that the CCC complex is required for copper-dependent relocalization of ATP7A. In control HeLa cells, peripheral vesicles containing ATP7A frequently colocalized with VPS35 and COMMD1 (Figure 5C, right). However, upon loss of CCDC93, ATP7A continued to be present in VPS35+ vesicles, but COMMD1 no longer colocalized with ATP7A, consistent with the loss of endosomal localization of COMMD1 observed upon depletion of CCDC22 (also noted in CCDC22 T17A fibroblasts in Figure 5B). Thus these data indicate that the loss of endosomal localization of the CCC complex is associated with altered cargo sorting.The carboxy-terminal ends of CCDC22/CCDC93 are responsible for FAM21 bindingGiven the importance of FAM21 in the endosomal targeting of the CCC complex, we next turned our attention to the mechanism mediating those interactions. To that end, we performed domain-mapping experiments to identify the domains in CCDC93 and CCDC22 responsible for FAM21 binding. As shown in Figure 5D, two regions in CCDC93 demonstrated specific binding to FAM21: a middle region encompassing amino acids 207–431, and its carboxyl-terminal end spanning amino acids 411–631. A similar analysis with CCDC22 (Figure 5E) also indicated that two regions bound to FAM21, namely amino acids 1–447 and the carboxyl-terminal region encompassed by 321–627. The specificity of binding between these domains and FAM21 was further validated by demonstrating that these constructs do not coprecipitate with NEMO, an unrelated coiled-coil protein (Supplemental Figure S11, A and B). After accounting for the ability of discrete domains to form CCDC22/CCDC93 heterodimers (namely CCDC93 207–431 and CCDC22 1–447; Figure 2, G and H), these data indicated that the carboxyl-terminal ends of CCDC22 and CCDC93 could interact with FAM21.We next turned our attention to FAM21 itself. This protein has an amino-terminal “head” domain that is responsible for its incorporation into the WASH complex and a long, unstructured tail that contains multiple leucine–phenylalanine acidic motifs (Figure 6A; Jia et al., 2012). Distinct regions within this tail are believed to mediate specific interactions. Domain mapping experiments using previously described suppression/reexpression vectors (Derivery et al., 2009; Gomez and Billadeau, 2009) expressing different regions of FAM21 indicated that the amino-terminal “head” of FAM21, encompassing amino acids 1–356, could not bind to the CCC complex (Figure 6B, left). Instead, the FAM21 tail (356-end) was sufficient for binding to the CCC complex, consistent with a prior report pertaining to CCDC22 and CCDC93 (Harbour et al., 2012). Moreover, the FAM21 tail had to encompass its middle region for CCC complex interaction and could not be sustained by its carboxyl-terminal end (742-end; Figure 6B, right). Using in vitro binding, we found that recombinant maltose-binding protein (MBP)–tagged CCDC93 (amino acids 411–631) could bind to a specific region of the FAM21 tail contained in recombinant GST-FAM21 356–600 (Figure 6C) but did not interact with more-carboxy-terminal regions of the tail encompassed in either amino acids 601–900 or 901–1341. Further truncation mutants of the CCDC93 carboxyl terminus revealed that a construct containing amino acids 448–631, but not 544–631, was capable of directly binding the GST-FAM21 356–600 fragment (Figure 6D). This interaction was recapitulated using surface plasmon resonance, which demonstrated a dose-dependent increase in the binding of CCDC93 (448–631) with immobilized GST-FAM21 356–600 with a mean dissociation constant of 0.3–0.6 nM (Figure 6E). This FAM21 fragment did not interact with MBP (Supplemental Figure S10B). Taken together, these data indicate that the C-terminus of CCDC93 can make a direct and high-affinity interaction with the N-terminal tail region of FAM21.FIGURE 6:CCDC93 interacts directly with the FAM21 tail. (A) Diagram of the fragments of FAM21 tail used in the in vitro binding studies. Black bars in the diagram indicate the location of leucine–phenylalanine acidic motifs in the tail of FAM21. (B) The indicated HA-YFP-FAM21 suppression/reexpression vectors were transfected into HEK293T cells and subsequently immunoprecipitated with anti-HA. Associated CCC complex components were detected by immunoblotting. (C) The indicated GST-FAM21 fragments were incubated with MBP-CCDC93 and stained for Coomassie or immunoblotted for MBP after GSH-agarose pull down. MBP-CCDC93 is denoted by an arrowhead. (D) The GST-FAM21 356–600 protein was incubated with the indicated MBP-CCDC93 fragments, and interactions were detected by Coomassie staining after GSH-agarose pull down. (E) The indicated concentrations of MBP-CCDC93 were assessed for binding with GST-FAM21 306-600 using surface plasmon resonance.Finally, we assessed the functional consequence of these interactions for the ability of CCDC93 to regulate the trafficking of ATP7A. Reintroduction of CCDC93 into HeLa deficient cells (after CRISPR-mediated deletion) demonstrated that only full-length CCDC93 rescued the localization of COMMD1 back to cytosolic puncta and was similarly able to restore the TGN localization of ATP7A under low-copper conditions. Of significance, cells expressing the truncated version of CCDC93 unable to bind FAM21 were unable to rescue the TGN localization of ATP7A or the punctate accumulation of COMMD1 (Figure 7, A and B, and Supplemental Figure S10C).FIGURE 7:Endosomal targeting of the CCC complex is required for ATP7A regulation. (A, B) ATP7A localization (red) under low-copper conditions was examined in CCDC93-deleted cells (CRISPR 93), which were rescued with an empty vector (YFP), full-length CCDC93 (WT), or a truncation mutant unable to bind FAM21 (1–438). In addition to ATP7A, cells were stained for COMMD1 (purple). (A) Representative images. (B) Quantitation of ATP7A distribution (in 30 transfected cells from each construct). Scale bar, 10 μm. (C) Model depicting the trafficking of ATP7A, the organization of CCC complex components, and its interaction with the WASH complex through FAM21.

Loss of CCDC22 or COMMD1 affects copper-dependent ATP7A traffickingATP7A is dynamically recycled between the TGN, where it is present when copper availability is low, and cytosolic vesicles, where it is redistributed under high copper conditions. From these cytosolic vesicles, ATP7A then reaches the plasma membrane and is responsible for cellular copper efflux (Wang et al., 2011). Whereas COMMD1 deficiency leads to copper accumulation (van de Sluis et al., 2002; Burstein et al., 2004; Vonk et al., 2011; Materia et al., 2012), a possible role for CCDC22 in copper handling has not been previously studied.We first examined the subcellular distribution of ATP7A using fibroblasts derived from a patient with a recently described point mutation in CCDC22 (c.49A>G/p.T17A). This mutation results in abnormal mRNA splicing, reduced protein expression, and impaired CCDC22–COMMD1 protein interactions (Voineagu et al., 2012; Starokadomskyy et al., 2013). Whereas control fibroblasts demonstrated dramatic redistribution of ATP7A in response to copper availability, CCDC22 T17A fibroblasts had lost the normal dynamic redistribution upon changes in copper availability (Figure 1A and Supplemental Figure S1). These cells displayed a peripheral distribution of ATP7A in large vesicles irrespective of copper levels and lacked the TGN localization of ATP7A upon copper deprivation (Figure 1, A and B). These alterations in ATP7A distribution were mirrored by COMMD1 deficiency in control fibroblasts after small interfering RNA (siRNA)–mediated silencing (Figure 1C). Consistent with these alterations in ATP7A trafficking, CCDC22 deficiency led to increased intracellular copper levels (Figure 1D), akin to what has been reported for COMMD1 deficiency (Burstein et al., 2004; Spee et al., 2007). However, patients with the CCDC22 T17A mutation did not display overt clinical signs of copper toxicosis, nor did they display biochemical evidence of copper overload, which is primarily determined in clinical settings by increased urinary copper excretion over 24 h (Figure 1, E and F). Instead, some affected individuals had elevated serum copper and serum ceruloplasmin concentrations, resembling the phenotype of the liver-specific Commd1-knockout mouse (Vonk et al., 2011) and the elevated ceruloplasmin levels of Bedlington terriers with COMMD1 mutations (Su et al., 1982). Taken together, these findings indicate an important role for both COMMD1 and CCDC22 in copper homeostasis and ATP7A trafficking.FIGURE 1:Regulation of ATP7A trafficking by CCDC22 and COMMD1. (A, B) ATP7A localization in response to copper was assessed in control and CCDC22 T17A fibroblasts via immuno­fluorescence staining for endogenous ATP7A (red) and GM130 (green). (A) Representative images. (B) Quantification of ATP7A distribution pattern in >50 cells/group. The ATP7A distribution observed in CCDC22 T17A fibroblasts was statistically different from the control line (p < 0.001). Scale bar, 20 μm. (C) ATP7A localization in control dermal fibroblasts was examined as before. Cells were transfected with either control siRNA duplexes or siRNA targeting COMMD1. After copper treatments, cells were stained for ATP7A (red) and nuclei (blue) and imaged by confocal microscopy. (D) Increased levels of cellular Cu were noted after siRNA silencing of CCDC22. (E) Pedigree of a kindred affected by a CCDC22 (c.49A>G/p.T17A) mutation. (F) The concentration of copper (Cu) in serum and urine was determined in selected probands of this kindred. Similarly, serum ceruloplasmin concentration (CP) and urinary excretion of copper over 24 h (Cu/24h) were also ascertained when possible. Normal value ranges are indicated at the top of the table. Abnormal values are marked in red.

Loss of CCDC22 or COMMD1 affects copper-dependent ATP7A trafficking

ATP7A is dynamically recycled between the TGN, where it is present when copper availability is low, and cytosolic vesicles, where it is redistributed under high copper conditions. From these cytosolic vesicles, ATP7A then reaches the plasma membrane and is responsible for cellular copper efflux (Wang et al., 2011). Whereas COMMD1 deficiency leads to copper accumulation (van de Sluis et al., 2002; Burstein et al., 2004; Vonk et al., 2011; Materia et al., 2012), a possible role for CCDC22 in copper handling has not been previously studied.

We first examined the subcellular distribution of ATP7A using fibroblasts derived from a patient with a recently described point mutation in CCDC22 (c.49A>G/p.T17A). This mutation results in abnormal mRNA splicing, reduced protein expression, and impaired CCDC22–COMMD1 protein interactions (Voineagu et al., 2012; Starokadomskyy et al., 2013). Whereas control fibroblasts demonstrated dramatic redistribution of ATP7A in response to copper availability, CCDC22 T17A fibroblasts had lost the normal dynamic redistribution upon changes in copper availability (Figure 1A and Supplemental Figure S1). These cells displayed a peripheral distribution of ATP7A in large vesicles irrespective of copper levels and lacked the TGN localization of ATP7A upon copper deprivation (Figure 1, A and B). These alterations in ATP7A distribution were mirrored by COMMD1 deficiency in control fibroblasts after small interfering RNA (siRNA)–mediated silencing (Figure 1C). Consistent with these alterations in ATP7A trafficking, CCDC22 deficiency led to increased intracellular copper levels (Figure 1D), akin to what has been reported for COMMD1 deficiency (Burstein et al., 2004; Spee et al., 2007). However, patients with the CCDC22 T17A mutation did not display overt clinical signs of copper toxicosis, nor did they display biochemical evidence of copper overload, which is primarily determined in clinical settings by increased urinary copper excretion over 24 h (Figure 1, E and F). Instead, some affected individuals had elevated serum copper and serum ceruloplasmin concentrations, resembling the phenotype of the liver-specific Commd1-knockout mouse (Vonk et al., 2011) and the elevated ceruloplasmin levels of Bedlington terriers with COMMD1 mutations (Su et al., 1982). Taken together, these findings indicate an important role for both COMMD1 and CCDC22 in copper homeostasis and ATP7A trafficking.

Regulation of ATP7A trafficking by CCDC22 and COMMD1. (A, B) ATP7A localization in response to copper was assessed in control and CCDC22 T17A fibroblasts via immuno­fluorescence staining for endogenous ATP7A (red) and GM130 (green). (A) Representative images. (B) Quantification of ATP7A distribution pattern in >50 cells/group. The ATP7A distribution observed in CCDC22 T17A fibroblasts was statistically different from the control line (p < 0.001). Scale bar, 20 μm. (C) ATP7A localization in control dermal fibroblasts was examined as before. Cells were transfected with either control siRNA duplexes or siRNA targeting COMMD1. After copper treatments, cells were stained for ATP7A (red) and nuclei (blue) and imaged by confocal microscopy. (D) Increased levels of cellular Cu were noted after siRNA silencing of CCDC22. (E) Pedigree of a kindred affected by a CCDC22 (c.49A>G/p.T17A) mutation. (F) The concentration of copper (Cu) in serum and urine was determined in selected probands of this kindred. Similarly, serum ceruloplasmin concentration (CP) and urinary excretion of copper over 24 h (Cu/24h) were also ascertained when possible. Normal value ranges are indicated at the top of the table. Abnormal values are marked in red.

Identification of the CCC complexWe recently reported that CCDC22 can bind to all 10 members of the COMMD protein family (Starokadomskyy et al., 2013). This analysis not only revealed the interaction between CCDC22 and COMMD proteins, but it also showed that CCDC22 bound to CCDC93, a protein that shares similar domain organization (Schou et al., 2014). Another partner found in this analysis was C16orf62, a protein whose function is also unknown. Of interest, all three proteins display a similar range of evolutionary conservation (Supplemental Figure S2) and are predicted to belong to an evolutionary conserved module that coevolved with the WASH complex (Li et al., 2014b).Given that CCDC93 was also identified in two prior TAP screens using COMMD9 and COMMD10 as baits, we speculated that these proteins associate to form what we refer to as the CCC complex. This possibility was examined directly by coimmunoprecipitation experiments. First, endogenous CCDC22 readily coprecipitated endogenous CCDC93 and C16orf62, and, conversely, the immunoprecipitation of these proteins also brought down the other two components of this complex (Figure 2A). We next examined whether COMMD proteins also interacted with these newly identified partners of CCDC22. First, immunoprecipitation of endogenous CCDC93 resulted in the coprecipitation of endogenous COMMD1 and COMMD6 but not IKK1 (Figure 2B). Second, endogenous COMMD1 coprecipitated endogenous C16orf62 and CCDC93 but not IKK1 (Figure 2C). The reciprocal interactions between CCDC22, CCDC93, C16orf62, and COMMD1 could be readily demonstrated using a coiled-coil protein (NEMO) as an additional negative control (Supplemental Figure S6A). Furthermore, all 10 human COMMD family members fused to glutathione S-transferase (GST) uniformly coprecipitated endogenous CCDC93 (Supplemental Figure S3A). Taken together, the interaction experiments validated the notion that these factors form a multiprotein complex.FIGURE 2:Discovery of the COMMD1–CCDC22 interactome. (A) Coimmunoprecipitation of endogenous CCDC22, CCDC93, and C16orf62 in HEK293T cell lysates confirm their interactions. (B) Endogenous CCDC93 coimmunoprecipitates with COMMD1 and COMMD6 in HEK293T cell lysates. (C) Endogenous COMMD1 coimmunoprecipitates with C16orf62 and CCDC93 in HEK 293 cell lysates. (D) Expression of CCC complex subunits was examined by immunoblotting in CCDC22 T17A fibroblasts. (E) Expression of CCC complex subunits was examined by immunoblotting in Commd1-deficient fibroblasts (Commd1−/−) and its isogenic control (Commd1F/F). (F, G) Domain organization of CCDC93 and CCDC22. Truncation mutants of CCDC93 (F) or CCDC22 (G) were expressed in HEK293T cells and subsequently immunoprecipitated from cell lysates to assess their ability to interact with CCDC22, CCDC93, COMMD1, and C16orf62, as indicated. Glyceraldehyde 3-phosphate dehydrogenase (GAPDH) and NEMO served as specificity controls. The asterisk in G denotes bands resulting from the hemagglutinin (HA) immunoblot and are not the GAPDH band, which migrates faster.

Identification of the CCC complex

We recently reported that CCDC22 can bind to all 10 members of the COMMD protein family (Starokadomskyy et al., 2013). This analysis not only revealed the interaction between CCDC22 and COMMD proteins, but it also showed that CCDC22 bound to CCDC93, a protein that shares similar domain organization (Schou et al., 2014). Another partner found in this analysis was C16orf62, a protein whose function is also unknown. Of interest, all three proteins display a similar range of evolutionary conservation (Supplemental Figure S2) and are predicted to belong to an evolutionary conserved module that coevolved with the WASH complex (Li et al., 2014b).

Given that CCDC93 was also identified in two prior TAP screens using COMMD9 and COMMD10 as baits, we speculated that these proteins associate to form what we refer to as the CCC complex. This possibility was examined directly by coimmunoprecipitation experiments. First, endogenous CCDC22 readily coprecipitated endogenous CCDC93 and C16orf62, and, conversely, the immunoprecipitation of these proteins also brought down the other two components of this complex (Figure 2A). We next examined whether COMMD proteins also interacted with these newly identified partners of CCDC22. First, immunoprecipitation of endogenous CCDC93 resulted in the coprecipitation of endogenous COMMD1 and COMMD6 but not IKK1 (Figure 2B). Second, endogenous COMMD1 coprecipitated endogenous C16orf62 and CCDC93 but not IKK1 (Figure 2C). The reciprocal interactions between CCDC22, CCDC93, C16orf62, and COMMD1 could be readily demonstrated using a coiled-coil protein (NEMO) as an additional negative control (Supplemental Figure S6A). Furthermore, all 10 human COMMD family members fused to glutathione S-transferase (GST) uniformly coprecipitated endogenous CCDC93 (Supplemental Figure S3A). Taken together, the interaction experiments validated the notion that these factors form a multiprotein complex.

Discovery of the COMMD1–CCDC22 interactome. (A) Coimmunoprecipitation of endogenous CCDC22, CCDC93, and C16orf62 in HEK293T cell lysates confirm their interactions. (B) Endogenous CCDC93 coimmunoprecipitates with COMMD1 and COMMD6 in HEK293T cell lysates. (C) Endogenous COMMD1 coimmunoprecipitates with C16orf62 and CCDC93 in HEK 293 cell lysates. (D) Expression of CCC complex subunits was examined by immunoblotting in CCDC22 T17A fibroblasts. (E) Expression of CCC complex subunits was examined by immunoblotting in Commd1-deficient fibroblasts (Commd1−/−) and its isogenic control (Commd1F/F). (F, G) Domain organization of CCDC93 and CCDC22. Truncation mutants of CCDC93 (F) or CCDC22 (G) were expressed in HEK293T cells and subsequently immunoprecipitated from cell lysates to assess their ability to interact with CCDC22, CCDC93, COMMD1, and C16orf62, as indicated. Glyceraldehyde 3-phosphate dehydrogenase (GAPDH) and NEMO served as specificity controls. The asterisk in G denotes bands resulting from the hemagglutinin (HA) immunoblot and are not the GAPDH band, which migrates faster.

CCC subunits stabilize expression of components of the complexFor many stable complexes, including WASH and retromer, expression of individual subunits is required to stabilize other components within the complex. We examined this notion first in fibroblasts from patients with the CCDC22 T17A mutation and found that in addition to the expected reduction in CCDC22 expression, the levels of CCDC93 were drastically reduced in these cells (Figure 2D). This phenomenon was recapitulated by RNA interference–mediated silencing of either CCDC22 or CCDC93 (Supplemental Figure S3B) and could not be explained by corresponding mRNA changes (Supplemental Figure S3C), consistent with a posttranscriptional event. In addition, deficiency of Commd1 in mouse embryo fibroblasts (MEFs) also led to reduced expression of Ccdc22 and Ccdc93 (Figure 2E). On the other hand, COMMD1 expression was not affected by siRNA of either CCDC22 or CCDC93 (Supplemental Figure S3B) or by the CCDC22 T17A mutation (Starokadomskyy et al., 2013). With regard to C16orf62, its expression was minimally affected in Commd1−/− MEFs but was more noticeably reduced in CCDC22 T17A fibroblasts (Figure 2, D and E); conversely, silencing of C16orf62 did not affect expression of other CCC subunits (Supplemental Figure S4C), which suggests that this subunit is less vital to the stability of the complex.

CCC subunits stabilize expression of components of the complex

For many stable complexes, including WASH and retromer, expression of individual subunits is required to stabilize other components within the complex. We examined this notion first in fibroblasts from patients with the CCDC22 T17A mutation and found that in addition to the expected reduction in CCDC22 expression, the levels of CCDC93 were drastically reduced in these cells (Figure 2D). This phenomenon was recapitulated by RNA interference–mediated silencing of either CCDC22 or CCDC93 (Supplemental Figure S3B) and could not be explained by corresponding mRNA changes (Supplemental Figure S3C), consistent with a posttranscriptional event. In addition, deficiency of Commd1 in mouse embryo fibroblasts (MEFs) also led to reduced expression of Ccdc22 and Ccdc93 (Figure 2E). On the other hand, COMMD1 expression was not affected by siRNA of either CCDC22 or CCDC93 (Supplemental Figure S3B) or by the CCDC22 T17A mutation (Starokadomskyy et al., 2013). With regard to C16orf62, its expression was minimally affected in Commd1−/− MEFs but was more noticeably reduced in CCDC22 T17A fibroblasts (Figure 2, D and E); conversely, silencing of C16orf62 did not affect expression of other CCC subunits (Supplemental Figure S4C), which suggests that this subunit is less vital to the stability of the complex.

Organization of the CCC complexNext we evaluated the protein domains in CCDC22 and CCDC93 that are required for their interaction. As depicted in Figure 2, F and G, both CCDC93 and CCDC22 have a shared amino- terminal domain belonging to the NDC80 and NUF2-calponin homology domain (NN-CH), followed by coiled-coil (CC) domains (Schou et al., 2014). On expressing domain truncation mutants of CCDC93, we identified that the NN-CH and the middle CC domain of CCDC93 (1–430) were required for CCDC22 binding, although some binding was possible to the CC region alone (207–431; Figure 2F). Conversely, the NN-CH region of CCDC22 and an extension just before its carboxy-terminal CC domain (1–447) was required for CCDC93 binding (Figure 2G). Moreover, recombinant CCDC22 1–447 purified from Escherichia coli was able to coprecipitate recombinant CCDC93 1-430 in vitro, consistent with a direct interaction through their N-terminal regions (Supplemental Figure S4A).The domain analysis also indicated that the binding of CCDC22 to COMMD1 was separable from its interaction to CCDC93 (amino acids 1–321 vs. 1–447; Figure 2G). This suggested that CCDC93 could be dispensable for CCDC22–COMMD1 binding, and, indeed, CCDC22 interactions with COMMD1 were not affected by CCDC93 silencing (Supplemental Figure S4B). On the other hand, coprecipitation between CCDC22 and C16orf62 was affected by CCDC93 deficiency, suggesting interdependence between these factors in their ability to form a larger complex. Finally, binding of C16orf62 to CCDC22 or CCDC93 could not be mapped to a single domain (Figure 2, F and G), nor did C16orf62 silencing impair CCDC93 binding with CCDC22 and COMMD1 (Supplemental Figure S4C), suggesting that CCDC22/CCDC93/COMMD1 interactions are not dependent on C16orf62. Taken together, these data indicated that CCDC22–CCDC93 interactions are directly mediated by their amino termini, and similarly, the interactions between COMMD1 and CCDC22 are likely direct and independent of other components. On the other hand, C16orf62 likely requires extensive interactions with both CCDC22 and CCDC93 to be assembled within the CCC complex.

Organization of the CCC complex

Next we evaluated the protein domains in CCDC22 and CCDC93 that are required for their interaction. As depicted in Figure 2, F and G, both CCDC93 and CCDC22 have a shared amino- terminal domain belonging to the NDC80 and NUF2-calponin homology domain (NN-CH), followed by coiled-coil (CC) domains (Schou et al., 2014). On expressing domain truncation mutants of CCDC93, we identified that the NN-CH and the middle CC domain of CCDC93 (1–430) were required for CCDC22 binding, although some binding was possible to the CC region alone (207–431; Figure 2F). Conversely, the NN-CH region of CCDC22 and an extension just before its carboxy-terminal CC domain (1–447) was required for CCDC93 binding (Figure 2G). Moreover, recombinant CCDC22 1–447 purified from Escherichia coli was able to coprecipitate recombinant CCDC93 1-430 in vitro, consistent with a direct interaction through their N-terminal regions (Supplemental Figure S4A).

The domain analysis also indicated that the binding of CCDC22 to COMMD1 was separable from its interaction to CCDC93 (amino acids 1–321 vs. 1–447; Figure 2G). This suggested that CCDC93 could be dispensable for CCDC22–COMMD1 binding, and, indeed, CCDC22 interactions with COMMD1 were not affected by CCDC93 silencing (Supplemental Figure S4B). On the other hand, coprecipitation between CCDC22 and C16orf62 was affected by CCDC93 deficiency, suggesting interdependence between these factors in their ability to form a larger complex. Finally, binding of C16orf62 to CCDC22 or CCDC93 could not be mapped to a single domain (Figure 2, F and G), nor did C16orf62 silencing impair CCDC93 binding with CCDC22 and COMMD1 (Supplemental Figure S4C), suggesting that CCDC22/CCDC93/COMMD1 interactions are not dependent on C16orf62. Taken together, these data indicated that CCDC22–CCDC93 interactions are directly mediated by their amino termini, and similarly, the interactions between COMMD1 and CCDC22 are likely direct and independent of other components. On the other hand, C16orf62 likely requires extensive interactions with both CCDC22 and CCDC93 to be assembled within the CCC complex.

The CCC and WASH complexes interact with each otherTo gain further insight into the function of the CCC complex, protein–protein interaction maps were developed using data deposited in the National Center for Biotechnology Information (NCBI). This analysis indicated that the predicted interactome for the human CCC complex includes complexes involved in vesicular sorting, such as the WASH complex, exocyst, and BLOC-1 (Figure 3A). Remarkably similar results were obtained when examining protein–protein interaction networks in Drosophila (Supplemental Figure S5).FIGURE 3:Interaction and colocalization of the CCC and WASH complexes. (A) Interaction map between CCC complex components and other cellular complexes were drawn using data extracted from NCBI. Intracomplex interactions, where useful, are also indicated. Human studies are depicted, and interactions identified in this study are depicted in red. (B) Coimmunoprecipitation of endogenous CCC subunits (CCDC22, CCDC93, and C16orf62) with the WASH components (FAM21 and WASH) was noted using HeLa cell lysates. (C) Colocalization of endogenous COMMD1 (green) and CCDC93 (red) or C16orf62 (red) was examined by immunostaining in HeLa cells. Scale bar, 5 μm. (D) Colocalization of endogenous COMMD1 (green) with WASH (red, top), and the retromer subunit VPS35 (red, bottom) was examined by immunofluorescence staining of HeLa cells. Scale bar, 5 μm.The possible interplay between the CCC and WASH complexes is in agreement with a recent report in which CCDC22 and CCDC93 were noted to interact with the WASH complex subunit FAM21 (Harbour et al., 2012). Moreover, the retromer complex, which is responsible for recruiting and activating WASH, was recently implicated in ATP7A sorting (Steinberg et al., 2013). To examine these possible interactions, we first immunoprecipitated FAM21 or WASH and found that each readily coprecipitated endogenous CCC components (C16orf62, CCDC93, and CCDC22; Figure 3B). Conversely, immunoprecipitation of endogenous CCC components also coprecipitated endogenous WASH (Supplemental Figure S6A) and FAM21 (Supplemental Figure S6B). Of interest, retromer did not appear to be required for WASH–CCC interactions, as demonstrated by the fact that FAM21 could precipitate CCC components to the same extent in control and VPS35-deficient cells (Supplemental Figure S6C).Next we examined whether the WASH complex played any role in the stability of the CCC complex. In this regard, we noted that Wash deficiency in MEFs did not impair the expression of CCC complex subunits or their ability to coimmunoprecipitate (Supplemental Figure S6D), and we observed a similar finding after silencing FAM21 in HeLa cells (Supplemental Figure S6E). These data suggested that despite their interactions, the CCC and WASH complexes are distinct and independent.

The CCC and WASH complexes interact with each other

To gain further insight into the function of the CCC complex, protein–protein interaction maps were developed using data deposited in the National Center for Biotechnology Information (NCBI). This analysis indicated that the predicted interactome for the human CCC complex includes complexes involved in vesicular sorting, such as the WASH complex, exocyst, and BLOC-1 (Figure 3A). Remarkably similar results were obtained when examining protein–protein interaction networks in Drosophila (Supplemental Figure S5).

Interaction and colocalization of the CCC and WASH complexes. (A) Interaction map between CCC complex components and other cellular complexes were drawn using data extracted from NCBI. Intracomplex interactions, where useful, are also indicated. Human studies are depicted, and interactions identified in this study are depicted in red. (B) Coimmunoprecipitation of endogenous CCC subunits (CCDC22, CCDC93, and C16orf62) with the WASH components (FAM21 and WASH) was noted using HeLa cell lysates. (C) Colocalization of endogenous COMMD1 (green) and CCDC93 (red) or C16orf62 (red) was examined by immunostaining in HeLa cells. Scale bar, 5 μm. (D) Colocalization of endogenous COMMD1 (green) with WASH (red, top), and the retromer subunit VPS35 (red, bottom) was examined by immunofluorescence staining of HeLa cells. Scale bar, 5 μm.

The possible interplay between the CCC and WASH complexes is in agreement with a recent report in which CCDC22 and CCDC93 were noted to interact with the WASH complex subunit FAM21 (Harbour et al., 2012). Moreover, the retromer complex, which is responsible for recruiting and activating WASH, was recently implicated in ATP7A sorting (Steinberg et al., 2013). To examine these possible interactions, we first immunoprecipitated FAM21 or WASH and found that each readily coprecipitated endogenous CCC components (C16orf62, CCDC93, and CCDC22; Figure 3B). Conversely, immunoprecipitation of endogenous CCC components also coprecipitated endogenous WASH (Supplemental Figure S6A) and FAM21 (Supplemental Figure S6B). Of interest, retromer did not appear to be required for WASH–CCC interactions, as demonstrated by the fact that FAM21 could precipitate CCC components to the same extent in control and VPS35-deficient cells (Supplemental Figure S6C).

Next we examined whether the WASH complex played any role in the stability of the CCC complex. In this regard, we noted that Wash deficiency in MEFs did not impair the expression of CCC complex subunits or their ability to coimmunoprecipitate (Supplemental Figure S6D), and we observed a similar finding after silencing FAM21 in HeLa cells (Supplemental Figure S6E). These data suggested that despite their interactions, the CCC and WASH complexes are distinct and independent.

CCC complex components colocalize with WASH and retromer in the early endosomeThe WASH complex is known to localize primarily to early endosomes, where it is recruited by retromer. In view of this, we next evaluated whether the CCC complex is similarly recruited to early endosomes. First, using a highly specific monoclonal antibody against COMMD1 (Supplemental Figure S7A), we found that this protein has a high degree of colocalization with other CCC complex components, namely CCDC93 and C16orf62 (Figure 3C and Supplemental Figure S7C). COMMD1-containing cytosolic foci demonstrated colocalization with WASH and the retromer subunit VPS35 (Figure 3D and Supplemental Figure S7C). Moreover, COMMD1 foci colocalized with the early endosomal marker RUFY1 and were closely juxtaposed to EEA1+ foci on endomembranes (Supplemental Figure S7, B and C). This finding closely resembles the localization of the WASH complex itself (Gomez and Billadeau, 2009). These structures were clearly distinct from very early endocytic vesicles containing APPL1 and did not colocalize with lysosomal (LAMP1) or TGN (TGN46) markers (Supplemental Figure S7, B and C). Transient expression of fluorescently tagged CCC complex proteins revealed simultaneous colocalization for COMMD1, CCDC22, and CCDC93, as well as for COMMD1, C16orf62, and CCDC93 (Supplemental Figure S7D). Taken together, these data indicate that the CCC complex localizes to a subcompartment of the early endosome that is enriched in retromer and WASH.

CCC complex components colocalize with WASH and retromer in the early endosome

The WASH complex is known to localize primarily to early endosomes, where it is recruited by retromer. In view of this, we next evaluated whether the CCC complex is similarly recruited to early endosomes. First, using a highly specific monoclonal antibody against COMMD1 (Supplemental Figure S7A), we found that this protein has a high degree of colocalization with other CCC complex components, namely CCDC93 and C16orf62 (Figure 3C and Supplemental Figure S7C). COMMD1-containing cytosolic foci demonstrated colocalization with WASH and the retromer subunit VPS35 (Figure 3D and Supplemental Figure S7C). Moreover, COMMD1 foci colocalized with the early endosomal marker RUFY1 and were closely juxtaposed to EEA1+ foci on endomembranes (Supplemental Figure S7, B and C). This finding closely resembles the localization of the WASH complex itself (Gomez and Billadeau, 2009). These structures were clearly distinct from very early endocytic vesicles containing APPL1 and did not colocalize with lysosomal (LAMP1) or TGN (TGN46) markers (Supplemental Figure S7, B and C). Transient expression of fluorescently tagged CCC complex proteins revealed simultaneous colocalization for COMMD1, CCDC22, and CCDC93, as well as for COMMD1, C16orf62, and CCDC93 (Supplemental Figure S7D). Taken together, these data indicate that the CCC complex localizes to a subcompartment of the early endosome that is enriched in retromer and WASH.

The CCC complex regulates ATP7A traffickingTaken together, our experiments indicated that COMMD1, CCDC22, CCDC93, and C16orf62 interact to form a higher-order stable complex that colocalizes with retromer and WASH on early endosomes. In view of this, we tested whether other components of this complex also regulate ATP7A trafficking, akin to the effects noted for CCDC22 and COMMD1 (Figure 1). Stable silencing of CCDC93 in HeLa cells (which results in deficiency of both CCDC93 and CCDC22) led to altered ATP7A recycling in response to copper availability, which demonstrated peripheral redistribution of the transporter and loss of its TGN localization upon copper deprivation, a finding that was also seen upon VPS35 silencing (Figure 4, A and B).FIGURE 4:Depletion of CCC complex components impairs trafficking of ATP7A and TGN46. (A) The indicated HeLa control and CCDC93- or VPS35-knockdown cell lines (sh, short hairpin against the indicated target) were maintained in high-copper conditions or transitioned to low copper as indicated in Materials and Methods. The cells were subsequently stained for ATP7A (red) and DNA (blue) and imaged by confocal fluorescence microscopy. Scale bar, 20 μm. (B) Quantification of ATP7A localization in the indicated knockdown cell lines was examined (>50 cells/group were scored). The ATP7A distribution observed in shCCDC93 and shVPS35 was statistically different from the control line (p < 0.001). (C) Plasma membrane expression of ATP7A was determined in the same cell lines after high- and low-copper conditions using biotinylation as described in Materials and Methods. N-Cadherin served as a loading control. (D) The indicated HeLa knockdown cell lines were stained for GM130 (green), TGN46 (red), F-actin (dark blue), and DNA (light blue) and imaged by confocal microscopy. Scale bar, 20 μm. (E) TGN46 localization was quantified in the indicated HeLa knockdown lines (>50 cells/group were scored). The TGN46 distribution observed in shCCDC22, shC16orf62, and shVPS35 was statistically different from that in the control line (p < 0.001).Recently it was reported that retromer, in conjunction with SNX27, also regulates the delivery of ATP7A from endosomes to the plasma membrane (Steinberg et al., 2013). Using biotinylation to mark plasma membrane proteins, we found that VPS35 deficiency, and to a lesser extent CCDC93 deficiency, resulted in decreased levels of ATP7A at the membrane during high copper exposure (Figure 4C). Furthermore, retromer deficiency can result in inappropriate sorting of endosomal cargo to lysosomes, leading to overall reductions in protein expression. However, whereas VPS35-deficient cells were depleted in ATP7A (Supplemental Figure S8, C and D) in agreement with previous results (Steinberg et al., 2013), deficiency of CCC components did not affect overall levels of ATP7A in a variety of models (Supplemental Figure S8, A–C).In addition to ATP7A, retromer and WASH regulate the retrograde transport to the TGN of a variety of endosomal cargo, including TGN46 (Hao et al., 2013). Under control conditions, we found that this protein was localized in close proximity to the cis-Golgi marker GM130 as expected; however, upon silencing of CCC complex components or the retromer subunit VPS35, TGN46 was diffusely localized in peripheral cytosolic vesicles (Figure 4, D and E). Thus loss of the CCC complex phenocopies retromer depletion and demonstrates an important role for the CCC complex in the trafficking of retromer cargo such as ATP7A and TGN46.

The CCC complex regulates ATP7A trafficking

Taken together, our experiments indicated that COMMD1, CCDC22, CCDC93, and C16orf62 interact to form a higher-order stable complex that colocalizes with retromer and WASH on early endosomes. In view of this, we tested whether other components of this complex also regulate ATP7A trafficking, akin to the effects noted for CCDC22 and COMMD1 (Figure 1). Stable silencing of CCDC93 in HeLa cells (which results in deficiency of both CCDC93 and CCDC22) led to altered ATP7A recycling in response to copper availability, which demonstrated peripheral redistribution of the transporter and loss of its TGN localization upon copper deprivation, a finding that was also seen upon VPS35 silencing (Figure 4, A and B).

Depletion of CCC complex components impairs trafficking of ATP7A and TGN46. (A) The indicated HeLa control and CCDC93- or VPS35-knockdown cell lines (sh, short hairpin against the indicated target) were maintained in high-copper conditions or transitioned to low copper as indicated in Materials and Methods. The cells were subsequently stained for ATP7A (red) and DNA (blue) and imaged by confocal fluorescence microscopy. Scale bar, 20 μm. (B) Quantification of ATP7A localization in the indicated knockdown cell lines was examined (>50 cells/group were scored). The ATP7A distribution observed in shCCDC93 and shVPS35 was statistically different from the control line (p < 0.001). (C) Plasma membrane expression of ATP7A was determined in the same cell lines after high- and low-copper conditions using biotinylation as described in Materials and Methods. N-Cadherin served as a loading control. (D) The indicated HeLa knockdown cell lines were stained for GM130 (green), TGN46 (red), F-actin (dark blue), and DNA (light blue) and imaged by confocal microscopy. Scale bar, 20 μm. (E) TGN46 localization was quantified in the indicated HeLa knockdown lines (>50 cells/group were scored). The TGN46 distribution observed in shCCDC22, shC16orf62, and shVPS35 was statistically different from that in the control line (p < 0.001).

Recently it was reported that retromer, in conjunction with SNX27, also regulates the delivery of ATP7A from endosomes to the plasma membrane (Steinberg et al., 2013). Using biotinylation to mark plasma membrane proteins, we found that VPS35 deficiency, and to a lesser extent CCDC93 deficiency, resulted in decreased levels of ATP7A at the membrane during high copper exposure (Figure 4C). Furthermore, retromer deficiency can result in inappropriate sorting of endosomal cargo to lysosomes, leading to overall reductions in protein expression. However, whereas VPS35-deficient cells were depleted in ATP7A (Supplemental Figure S8, C and D) in agreement with previous results (Steinberg et al., 2013), deficiency of CCC components did not affect overall levels of ATP7A in a variety of models (Supplemental Figure S8, A–C).

In addition to ATP7A, retromer and WASH regulate the retrograde transport to the TGN of a variety of endosomal cargo, including TGN46 (Hao et al., 2013). Under control conditions, we found that this protein was localized in close proximity to the cis-Golgi marker GM130 as expected; however, upon silencing of CCC complex components or the retromer subunit VPS35, TGN46 was diffusely localized in peripheral cytosolic vesicles (Figure 4, D and E). Thus loss of the CCC complex phenocopies retromer depletion and demonstrates an important role for the CCC complex in the trafficking of retromer cargo such as ATP7A and TGN46.

FAM21 is required for endosomal recruitment of the CCC complexGiven the interaction and colocalization of the CCC and WASH complexes, we examined whether their endosomal localization were interrelated. In this regard, we noted that stable silencing of FAM21 in HeLa cells (shFAM21) drastically reduced cytosolic puncta for COMMD1, as well as for the CCC complex components CCDC93 and C16orf62 (Figure 5A and Supplemental Figure S9A). In contrast to this result, loss of Wash in mouse fibroblasts did not abolish COMMD1 endosomal localization (Supplemental Figure S9B). Instead of discrete puncta, COMMD1 localized to collapsed endosomal structures, which retained FAM21 localization, as previously reported (Gomez et al., 2012). Furthermore, silencing of VPS35, a subunit of retromer that in turn is required for FAM21 recruitment to endosomes, also led to loss of COMMD1 puncta (Supplemental Figure S9C). Conversely, FAM21 localization was unaffected in CCDC22 T17A fibroblasts (Figure 5B); instead, there were dramatic changes in COMMD1 localization, with loss of endosomal-localized COMMD1 and its redistribution with a more diffuse cytosolic and nuclear pattern in these cells. Taken together, these data indicated that FAM21 is required for the recruitment of the CCC complex to the endosome, a process that in turn also depends on CCDC22 expression.FIGURE 5:FAM21 recruits the CCC complex to endosomes, and this is necessary for ATP7A trafficking. (A) Formation of cytosolic puncti of FAM21, COMMD1, CCDC93, and C16orf62 was assessed by immunofluorescence staining after stable silencing of FAM21 (shFAM21). Average number of puncta per cell (of ∼75 imaged cells) for the indicated CCC complex member. Representative images, including EEA1 staining to show the localization of early endosomes. Scale bar, 5 μm. (B) Endogenous COMMD1 (red) and FAM21 (green) localization was evaluated by immunofluorescence staining in control fibroblasts and fibroblasts derived from patients with the CCDC22 T17A mutation, as indicated. Scale bar, 5 μm. (C) Localization of ATP7A (red) in control and CCDC93 deleted cells (CRISPR 93). Localization of retromer (VPS35, green) and the CCC complex (COMMD1, purple) in cytosolic vesicles and their relative colocalization are also depicted. Scale bar, 10 μm. (D, E) The indicated HA-tagged CCDC93 or CCDC22 construct was transfected into HEK293T cells and subsequently immunoprecipitated with anti-FAM21 or control immunoglobulin G (IgG). Associated proteins were identified by immunoblotting with anti-HA. Note that the CCDC93 1–430 fragment was also detected in the IgG immunoprecipitation, indicating that this interaction with FAM21 is likely nonspecific (marked by an arrowhead).To demonstrate the functional significance of these events, we deleted CCDC93 in HeLa cells using the CRISPR/Cas9 system. Deletion of CCDC93 led to concurrent depletion of CCDC22 (Supplemental Figure S10A), as well as diffuse localization of ATP7A at both low- and high-copper conditions (Figure 5C, left), confirming once more that the CCC complex is required for copper-dependent relocalization of ATP7A. In control HeLa cells, peripheral vesicles containing ATP7A frequently colocalized with VPS35 and COMMD1 (Figure 5C, right). However, upon loss of CCDC93, ATP7A continued to be present in VPS35+ vesicles, but COMMD1 no longer colocalized with ATP7A, consistent with the loss of endosomal localization of COMMD1 observed upon depletion of CCDC22 (also noted in CCDC22 T17A fibroblasts in Figure 5B). Thus these data indicate that the loss of endosomal localization of the CCC complex is associated with altered cargo sorting.

FAM21 is required for endosomal recruitment of the CCC complex

Given the interaction and colocalization of the CCC and WASH complexes, we examined whether their endosomal localization were interrelated. In this regard, we noted that stable silencing of FAM21 in HeLa cells (shFAM21) drastically reduced cytosolic puncta for COMMD1, as well as for the CCC complex components CCDC93 and C16orf62 (Figure 5A and Supplemental Figure S9A). In contrast to this result, loss of Wash in mouse fibroblasts did not abolish COMMD1 endosomal localization (Supplemental Figure S9B). Instead of discrete puncta, COMMD1 localized to collapsed endosomal structures, which retained FAM21 localization, as previously reported (Gomez et al., 2012). Furthermore, silencing of VPS35, a subunit of retromer that in turn is required for FAM21 recruitment to endosomes, also led to loss of COMMD1 puncta (Supplemental Figure S9C). Conversely, FAM21 localization was unaffected in CCDC22 T17A fibroblasts (Figure 5B); instead, there were dramatic changes in COMMD1 localization, with loss of endosomal-localized COMMD1 and its redistribution with a more diffuse cytosolic and nuclear pattern in these cells. Taken together, these data indicated that FAM21 is required for the recruitment of the CCC complex to the endosome, a process that in turn also depends on CCDC22 expression.

FAM21 recruits the CCC complex to endosomes, and this is necessary for ATP7A trafficking. (A) Formation of cytosolic puncti of FAM21, COMMD1, CCDC93, and C16orf62 was assessed by immunofluorescence staining after stable silencing of FAM21 (shFAM21). Average number of puncta per cell (of ∼75 imaged cells) for the indicated CCC complex member. Representative images, including EEA1 staining to show the localization of early endosomes. Scale bar, 5 μm. (B) Endogenous COMMD1 (red) and FAM21 (green) localization was evaluated by immunofluorescence staining in control fibroblasts and fibroblasts derived from patients with the CCDC22 T17A mutation, as indicated. Scale bar, 5 μm. (C) Localization of ATP7A (red) in control and CCDC93 deleted cells (CRISPR 93). Localization of retromer (VPS35, green) and the CCC complex (COMMD1, purple) in cytosolic vesicles and their relative colocalization are also depicted. Scale bar, 10 μm. (D, E) The indicated HA-tagged CCDC93 or CCDC22 construct was transfected into HEK293T cells and subsequently immunoprecipitated with anti-FAM21 or control immunoglobulin G (IgG). Associated proteins were identified by immunoblotting with anti-HA. Note that the CCDC93 1–430 fragment was also detected in the IgG immunoprecipitation, indicating that this interaction with FAM21 is likely nonspecific (marked by an arrowhead).

To demonstrate the functional significance of these events, we deleted CCDC93 in HeLa cells using the CRISPR/Cas9 system. Deletion of CCDC93 led to concurrent depletion of CCDC22 (Supplemental Figure S10A), as well as diffuse localization of ATP7A at both low- and high-copper conditions (Figure 5C, left), confirming once more that the CCC complex is required for copper-dependent relocalization of ATP7A. In control HeLa cells, peripheral vesicles containing ATP7A frequently colocalized with VPS35 and COMMD1 (Figure 5C, right). However, upon loss of CCDC93, ATP7A continued to be present in VPS35+ vesicles, but COMMD1 no longer colocalized with ATP7A, consistent with the loss of endosomal localization of COMMD1 observed upon depletion of CCDC22 (also noted in CCDC22 T17A fibroblasts in Figure 5B). Thus these data indicate that the loss of endosomal localization of the CCC complex is associated with altered cargo sorting.

The carboxy-terminal ends of CCDC22/CCDC93 are responsible for FAM21 bindingGiven the importance of FAM21 in the endosomal targeting of the CCC complex, we next turned our attention to the mechanism mediating those interactions. To that end, we performed domain-mapping experiments to identify the domains in CCDC93 and CCDC22 responsible for FAM21 binding. As shown in Figure 5D, two regions in CCDC93 demonstrated specific binding to FAM21: a middle region encompassing amino acids 207–431, and its carboxyl-terminal end spanning amino acids 411–631. A similar analysis with CCDC22 (Figure 5E) also indicated that two regions bound to FAM21, namely amino acids 1–447 and the carboxyl-terminal region encompassed by 321–627. The specificity of binding between these domains and FAM21 was further validated by demonstrating that these constructs do not coprecipitate with NEMO, an unrelated coiled-coil protein (Supplemental Figure S11, A and B). After accounting for the ability of discrete domains to form CCDC22/CCDC93 heterodimers (namely CCDC93 207–431 and CCDC22 1–447; Figure 2, G and H), these data indicated that the carboxyl-terminal ends of CCDC22 and CCDC93 could interact with FAM21.We next turned our attention to FAM21 itself. This protein has an amino-terminal “head” domain that is responsible for its incorporation into the WASH complex and a long, unstructured tail that contains multiple leucine–phenylalanine acidic motifs (Figure 6A; Jia et al., 2012). Distinct regions within this tail are believed to mediate specific interactions. Domain mapping experiments using previously described suppression/reexpression vectors (Derivery et al., 2009; Gomez and Billadeau, 2009) expressing different regions of FAM21 indicated that the amino-terminal “head” of FAM21, encompassing amino acids 1–356, could not bind to the CCC complex (Figure 6B, left). Instead, the FAM21 tail (356-end) was sufficient for binding to the CCC complex, consistent with a prior report pertaining to CCDC22 and CCDC93 (Harbour et al., 2012). Moreover, the FAM21 tail had to encompass its middle region for CCC complex interaction and could not be sustained by its carboxyl-terminal end (742-end; Figure 6B, right). Using in vitro binding, we found that recombinant maltose-binding protein (MBP)–tagged CCDC93 (amino acids 411–631) could bind to a specific region of the FAM21 tail contained in recombinant GST-FAM21 356–600 (Figure 6C) but did not interact with more-carboxy-terminal regions of the tail encompassed in either amino acids 601–900 or 901–1341. Further truncation mutants of the CCDC93 carboxyl terminus revealed that a construct containing amino acids 448–631, but not 544–631, was capable of directly binding the GST-FAM21 356–600 fragment (Figure 6D). This interaction was recapitulated using surface plasmon resonance, which demonstrated a dose-dependent increase in the binding of CCDC93 (448–631) with immobilized GST-FAM21 356–600 with a mean dissociation constant of 0.3–0.6 nM (Figure 6E). This FAM21 fragment did not interact with MBP (Supplemental Figure S10B). Taken together, these data indicate that the C-terminus of CCDC93 can make a direct and high-affinity interaction with the N-terminal tail region of FAM21.FIGURE 6:CCDC93 interacts directly with the FAM21 tail. (A) Diagram of the fragments of FAM21 tail used in the in vitro binding studies. Black bars in the diagram indicate the location of leucine–phenylalanine acidic motifs in the tail of FAM21. (B) The indicated HA-YFP-FAM21 suppression/reexpression vectors were transfected into HEK293T cells and subsequently immunoprecipitated with anti-HA. Associated CCC complex components were detected by immunoblotting. (C) The indicated GST-FAM21 fragments were incubated with MBP-CCDC93 and stained for Coomassie or immunoblotted for MBP after GSH-agarose pull down. MBP-CCDC93 is denoted by an arrowhead. (D) The GST-FAM21 356–600 protein was incubated with the indicated MBP-CCDC93 fragments, and interactions were detected by Coomassie staining after GSH-agarose pull down. (E) The indicated concentrations of MBP-CCDC93 were assessed for binding with GST-FAM21 306-600 using surface plasmon resonance.Finally, we assessed the functional consequence of these interactions for the ability of CCDC93 to regulate the trafficking of ATP7A. Reintroduction of CCDC93 into HeLa deficient cells (after CRISPR-mediated deletion) demonstrated that only full-length CCDC93 rescued the localization of COMMD1 back to cytosolic puncta and was similarly able to restore the TGN localization of ATP7A under low-copper conditions. Of significance, cells expressing the truncated version of CCDC93 unable to bind FAM21 were unable to rescue the TGN localization of ATP7A or the punctate accumulation of COMMD1 (Figure 7, A and B, and Supplemental Figure S10C).FIGURE 7:Endosomal targeting of the CCC complex is required for ATP7A regulation. (A, B) ATP7A localization (red) under low-copper conditions was examined in CCDC93-deleted cells (CRISPR 93), which were rescued with an empty vector (YFP), full-length CCDC93 (WT), or a truncation mutant unable to bind FAM21 (1–438). In addition to ATP7A, cells were stained for COMMD1 (purple). (A) Representative images. (B) Quantitation of ATP7A distribution (in 30 transfected cells from each construct). Scale bar, 10 μm. (C) Model depicting the trafficking of ATP7A, the organization of CCC complex components, and its interaction with the WASH complex through FAM21.

The carboxy-terminal ends of CCDC22/CCDC93 are responsible for FAM21 binding

Given the importance of FAM21 in the endosomal targeting of the CCC complex, we next turned our attention to the mechanism mediating those interactions. To that end, we performed domain-mapping experiments to identify the domains in CCDC93 and CCDC22 responsible for FAM21 binding. As shown in Figure 5D, two regions in CCDC93 demonstrated specific binding to FAM21: a middle region encompassing amino acids 207–431, and its carboxyl-terminal end spanning amino acids 411–631. A similar analysis with CCDC22 (Figure 5E) also indicated that two regions bound to FAM21, namely amino acids 1–447 and the carboxyl-terminal region encompassed by 321–627. The specificity of binding between these domains and FAM21 was further validated by demonstrating that these constructs do not coprecipitate with NEMO, an unrelated coiled-coil protein (Supplemental Figure S11, A and B). After accounting for the ability of discrete domains to form CCDC22/CCDC93 heterodimers (namely CCDC93 207–431 and CCDC22 1–447; Figure 2, G and H), these data indicated that the carboxyl-terminal ends of CCDC22 and CCDC93 could interact with FAM21.

We next turned our attention to FAM21 itself. This protein has an amino-terminal “head” domain that is responsible for its incorporation into the WASH complex and a long, unstructured tail that contains multiple leucine–phenylalanine acidic motifs (Figure 6A; Jia et al., 2012). Distinct regions within this tail are believed to mediate specific interactions. Domain mapping experiments using previously described suppression/reexpression vectors (Derivery et al., 2009; Gomez and Billadeau, 2009) expressing different regions of FAM21 indicated that the amino-terminal “head” of FAM21, encompassing amino acids 1–356, could not bind to the CCC complex (Figure 6B, left). Instead, the FAM21 tail (356-end) was sufficient for binding to the CCC complex, consistent with a prior report pertaining to CCDC22 and CCDC93 (Harbour et al., 2012). Moreover, the FAM21 tail had to encompass its middle region for CCC complex interaction and could not be sustained by its carboxyl-terminal end (742-end; Figure 6B, right). Using in vitro binding, we found that recombinant maltose-binding protein (MBP)–tagged CCDC93 (amino acids 411–631) could bind to a specific region of the FAM21 tail contained in recombinant GST-FAM21 356–600 (Figure 6C) but did not interact with more-carboxy-terminal regions of the tail encompassed in either amino acids 601–900 or 901–1341. Further truncation mutants of the CCDC93 carboxyl terminus revealed that a construct containing amino acids 448–631, but not 544–631, was capable of directly binding the GST-FAM21 356–600 fragment (Figure 6D). This interaction was recapitulated using surface plasmon resonance, which demonstrated a dose-dependent increase in the binding of CCDC93 (448–631) with immobilized GST-FAM21 356–600 with a mean dissociation constant of 0.3–0.6 nM (Figure 6E). This FAM21 fragment did not interact with MBP (Supplemental Figure S10B). Taken together, these data indicate that the C-terminus of CCDC93 can make a direct and high-affinity interaction with the N-terminal tail region of FAM21.

CCDC93 interacts directly with the FAM21 tail. (A) Diagram of the fragments of FAM21 tail used in the in vitro binding studies. Black bars in the diagram indicate the location of leucine–phenylalanine acidic motifs in the tail of FAM21. (B) The indicated HA-YFP-FAM21 suppression/reexpression vectors were transfected into HEK293T cells and subsequently immunoprecipitated with anti-HA. Associated CCC complex components were detected by immunoblotting. (C) The indicated GST-FAM21 fragments were incubated with MBP-CCDC93 and stained for Coomassie or immunoblotted for MBP after GSH-agarose pull down. MBP-CCDC93 is denoted by an arrowhead. (D) The GST-FAM21 356–600 protein was incubated with the indicated MBP-CCDC93 fragments, and interactions were detected by Coomassie staining after GSH-agarose pull down. (E) The indicated concentrations of MBP-CCDC93 were assessed for binding with GST-FAM21 306-600 using surface plasmon resonance.

Finally, we assessed the functional consequence of these interactions for the ability of CCDC93 to regulate the trafficking of ATP7A. Reintroduction of CCDC93 into HeLa deficient cells (after CRISPR-mediated deletion) demonstrated that only full-length CCDC93 rescued the localization of COMMD1 back to cytosolic puncta and was similarly able to restore the TGN localization of ATP7A under low-copper conditions. Of significance, cells expressing the truncated version of CCDC93 unable to bind FAM21 were unable to rescue the TGN localization of ATP7A or the punctate accumulation of COMMD1 (Figure 7, A and B, and Supplemental Figure S10C).

Endosomal targeting of the CCC complex is required for ATP7A regulation. (A, B) ATP7A localization (red) under low-copper conditions was examined in CCDC93-deleted cells (CRISPR 93), which were rescued with an empty vector (YFP), full-length CCDC93 (WT), or a truncation mutant unable to bind FAM21 (1–438). In addition to ATP7A, cells were stained for COMMD1 (purple). (A) Representative images. (B) Quantitation of ATP7A distribution (in 30 transfected cells from each construct). Scale bar, 10 μm. (C) Model depicting the trafficking of ATP7A, the organization of CCC complex components, and its interaction with the WASH complex through FAM21.

DISCUSSIONThe role of COMMD1 in mammalian copper homeostasis has been appreciated for more than a decade (van de Sluis et al., 2002), but the precise mechanism by which COMMD1 regulates copper transport has remained unclear. The present study provides a mechanistic framework to understand how COMMD1 affects intracellular copper levels. Consistent with prior work that suggested a possible endosomal localization and activity for COMMD1 (Burkhead et al., 2009), we found that this factor, in conjunction with CCDC22, CCDC93, and C16orf62, forms a novel complex that interacts with the WASH complex and is required for the copper-dependent trafficking of ATP7A (Figure 7C).The copper-dependent trafficking of ATP7A is a complex process that is incompletely understood. Under low-copper conditions, the transporter resides in the TGN, and upon copper excess, it is mobilized to cytosolic endosomal vesicles through a process that requires ATOX1, a chaperone protein responsible for copper delivery to these transporters (Hamza et al., 2003; Yi and Kaler, 2014). From these peripheral endosomal vesicles, ATP7A can then reach the plasma membrane to deliver copper to the extracellular space. Deficiency of the CCC complex or the retromer subunit VPS35 led in both cases to reduced plasma membrane translocation of ATP7A, which is in line with the intracellular accumulation of copper noted in CCDC22-deficient cells. These findings are in agreement with the recent report that retromer, acting in conjunction with SNX27, is required for ATP7A trafficking to the plasma membrane (Steinberg et al., 2013) and indicate that the CCC complex is required to facilitate this route of ATP7A trafficking (Figure 7C) or is involved in the recycling of internalized receptor back to the plasma membrane.Furthermore, it is also evident that both retromer and the CCC complex are also required for the retrograde transport of ATP7A to the TGN under low-copper conditions, and the absence of VPS35 or CCC components results in a similar phenotype of peripheral dispersal of the transporter (Figure 7C). Prior studies indicate that the retrograde transport of ATP7A is also controlled by a number of additional complexes, including the clathrin adaptors AP-1 and AP-2 (Ishizaki et al., 2010; Holloway et al., 2013; Martinelli et al., 2013) and the BLOC-1 complex (Ryder et al., 2013). Additional work will be required to precisely position how these differing systems act in sequential or parallel pathways to achieve this process. Furthermore, the dual effects of retromer and the CCC complex under high- and low-copper conditions imply that factors such as SNX27 and perhaps others are needed to switch the directionality of ATP7A endosomal trafficking in response to copper availability.Although this work uncovers the cellular mechanism for the role of COMMD1 in copper homeostasis, it remains unclear why humans and mice do not replicate the phenotype noted in dogs deficient for COMMD1. Liver-specific deletion of Commd1 in mice leads to demonstrable copper accumulation under high-copper diets but to an extent that does not induce liver injury (Vonk et al., 2011). We studied the status of copper stores in a family with CCDC22 deficiency, and other than increased serum ceruloplasmin levels in some probands, we did not find any suggestion of copper toxicosis. This is consistent with prior genetic studies that failed to demonstrate a role for COMMD1 as a cause for non-Wilsonian copper toxicosis or as a modifier of Wilson's disease in humans (Muller et al., 2003; Stuehler et al., 2004; Lovicu et al., 2006). The explanation for the organism-specific differences in liver pathology remains unclear but is in line with the substantial physiological differences in copper metabolism among mammals (Wang et al., 2011), which may highlight adaptive changes driven by dietary exposure to copper. In addition, it is noteworthy that dietary factors are known to affect the penetrance of non-Wilsonian copper toxicosis (Shim and Harris, 2003; Harvey and McArdle, 2008), and it may not be ruled out that under certain environmental conditions, mutations in this pathway could result in overt clinical phenotypes.Our studies identify that the core composition of the CCC complex includes COMMD proteins in association with CCDC22, CCDC93, and C16orf62. This is in agreement with a prior report that identified CCDC22 and CCDC93 as FAM21-interacting proteins (Harbour et al., 2012) and a more recent report that CCDC22 and CCDC93 are unstable when either component is depleted by siRNA (Freeman et al., 2014). We also delineated the domains that mediate the assembly of the CCC complex and demonstrated that a minimal CCDC93 fragment (448–631) could directly interact with a proximal FAM21 tail fragment (356–600). Although the data also indicated that the carboxyl terminus of CCDC22 could interact with FAM21 independently of CCDC93, it remains to be established whether this is also a direct protein–protein interaction. Moreover, we show that CCDC22, CCDC93, and C16orf62 protein stability also depends on the presence of COMMD1 and that the CCC complex is not recruited to endosomes in the absence of FAM21, further implicating the FAM21 unstructured tail as a recruitment node for proteins involved in receptor trafficking from membrane regions where the WASH complex localizes.Although our studies were focused on ATP7A trafficking, it is reasonable to speculate that this complex might regulate more broadly other WASH- and retromer-dependent trafficking events. Indeed, we demonstrate that the retrograde trafficking of TGN46 back to the TGN is dependent not only on retromer but also on the CCC complex. Furthermore, the human and Drosophila interaction network maps support the notion that the CCC complex is closely interconnected with the WASH complex. In agreement with this interpretation, a recent study reported that the WASH complex coevolved in a conserved module with the CCC complex, having a much closer evolutionarily relationship with CCC than with retromer itself (Li et al., 2014b). Additional inspection of the interaction network maps also suggests that the CCC complex, through its putative interaction with the BLOC-1, exocyst, and WAVE complexes, may be involved in processes outside of endosomal sorting such as lysosome biogenesis, exocytosis, and plasma membrane dynamics. All of these will be important areas to explore in future studies. In fact, recent studies have uncovered that the WASH complex is recruited to the BLOC-1 complex, in agreement with a possible interplay of the CCC complex in these processes (Ryder et al., 2013). Finally, one critical question that remains to be addressed is the reason for the conservation of 10 COMMD genes in most eukaryotes. It is noteworthy that COMMD proteins also demonstrate an interdependence, suggesting that their ability to heterodimerize (Burstein et al., 2005) is critical to their biological function. Although the specific contribution that each COMMD protein makes in receptor trafficking remains obscure, it is tempting to speculate that individual members of this family may differentially regulate cargo specificity or subcellular localization of the complex or affect distinct aspects of the receptor trafficking process.In summary, this study uncovers the mechanism by which COMMD1 regulates copper metabolism and identifies its essential role in the stable assembly of the CCC complex, an evolutionarily conserved module that is linked to the WASH complex. Given the large number of endosomal cargoes regulated by WASH and retromer, it can be anticipated that the CCC complex participates in a variety of additional physiological and disease processes, as evident by the complex developmental phenotype resulting from CCDC22 mutations (Voineagu et al., 2012; Starokadomskyy et al., 2013). Thus future work aimed at deciphering the actions of the CCC complex and its role in regulating WASH-dependent receptor trafficking should provide mechanistic insight that will have broad biomedical implications.

The role of COMMD1 in mammalian copper homeostasis has been appreciated for more than a decade (van de Sluis et al., 2002), but the precise mechanism by which COMMD1 regulates copper transport has remained unclear. The present study provides a mechanistic framework to understand how COMMD1 affects intracellular copper levels. Consistent with prior work that suggested a possible endosomal localization and activity for COMMD1 (Burkhead et al., 2009), we found that this factor, in conjunction with CCDC22, CCDC93, and C16orf62, forms a novel complex that interacts with the WASH complex and is required for the copper-dependent trafficking of ATP7A (Figure 7C).

The copper-dependent trafficking of ATP7A is a complex process that is incompletely understood. Under low-copper conditions, the transporter resides in the TGN, and upon copper excess, it is mobilized to cytosolic endosomal vesicles through a process that requires ATOX1, a chaperone protein responsible for copper delivery to these transporters (Hamza et al., 2003; Yi and Kaler, 2014). From these peripheral endosomal vesicles, ATP7A can then reach the plasma membrane to deliver copper to the extracellular space. Deficiency of the CCC complex or the retromer subunit VPS35 led in both cases to reduced plasma membrane translocation of ATP7A, which is in line with the intracellular accumulation of copper noted in CCDC22-deficient cells. These findings are in agreement with the recent report that retromer, acting in conjunction with SNX27, is required for ATP7A trafficking to the plasma membrane (Steinberg et al., 2013) and indicate that the CCC complex is required to facilitate this route of ATP7A trafficking (Figure 7C) or is involved in the recycling of internalized receptor back to the plasma membrane.

Furthermore, it is also evident that both retromer and the CCC complex are also required for the retrograde transport of ATP7A to the TGN under low-copper conditions, and the absence of VPS35 or CCC components results in a similar phenotype of peripheral dispersal of the transporter (Figure 7C). Prior studies indicate that the retrograde transport of ATP7A is also controlled by a number of additional complexes, including the clathrin adaptors AP-1 and AP-2 (Ishizaki et al., 2010; Holloway et al., 2013; Martinelli et al., 2013) and the BLOC-1 complex (Ryder et al., 2013). Additional work will be required to precisely position how these differing systems act in sequential or parallel pathways to achieve this process. Furthermore, the dual effects of retromer and the CCC complex under high- and low-copper conditions imply that factors such as SNX27 and perhaps others are needed to switch the directionality of ATP7A endosomal trafficking in response to copper availability.

Although this work uncovers the cellular mechanism for the role of COMMD1 in copper homeostasis, it remains unclear why humans and mice do not replicate the phenotype noted in dogs deficient for COMMD1. Liver-specific deletion of Commd1 in mice leads to demonstrable copper accumulation under high-copper diets but to an extent that does not induce liver injury (Vonk et al., 2011). We studied the status of copper stores in a family with CCDC22 deficiency, and other than increased serum ceruloplasmin levels in some probands, we did not find any suggestion of copper toxicosis. This is consistent with prior genetic studies that failed to demonstrate a role for COMMD1 as a cause for non-Wilsonian copper toxicosis or as a modifier of Wilson's disease in humans (Muller et al., 2003; Stuehler et al., 2004; Lovicu et al., 2006). The explanation for the organism-specific differences in liver pathology remains unclear but is in line with the substantial physiological differences in copper metabolism among mammals (Wang et al., 2011), which may highlight adaptive changes driven by dietary exposure to copper. In addition, it is noteworthy that dietary factors are known to affect the penetrance of non-Wilsonian copper toxicosis (Shim and Harris, 2003; Harvey and McArdle, 2008), and it may not be ruled out that under certain environmental conditions, mutations in this pathway could result in overt clinical phenotypes.

Our studies identify that the core composition of the CCC complex includes COMMD proteins in association with CCDC22, CCDC93, and C16orf62. This is in agreement with a prior report that identified CCDC22 and CCDC93 as FAM21-interacting proteins (Harbour et al., 2012) and a more recent report that CCDC22 and CCDC93 are unstable when either component is depleted by siRNA (Freeman et al., 2014). We also delineated the domains that mediate the assembly of the CCC complex and demonstrated that a minimal CCDC93 fragment (448–631) could directly interact with a proximal FAM21 tail fragment (356–600). Although the data also indicated that the carboxyl terminus of CCDC22 could interact with FAM21 independently of CCDC93, it remains to be established whether this is also a direct protein–protein interaction. Moreover, we show that CCDC22, CCDC93, and C16orf62 protein stability also depends on the presence of COMMD1 and that the CCC complex is not recruited to endosomes in the absence of FAM21, further implicating the FAM21 unstructured tail as a recruitment node for proteins involved in receptor trafficking from membrane regions where the WASH complex localizes.

Although our studies were focused on ATP7A trafficking, it is reasonable to speculate that this complex might regulate more broadly other WASH- and retromer-dependent trafficking events. Indeed, we demonstrate that the retrograde trafficking of TGN46 back to the TGN is dependent not only on retromer but also on the CCC complex. Furthermore, the human and Drosophila interaction network maps support the notion that the CCC complex is closely interconnected with the WASH complex. In agreement with this interpretation, a recent study reported that the WASH complex coevolved in a conserved module with the CCC complex, having a much closer evolutionarily relationship with CCC than with retromer itself (Li et al., 2014b). Additional inspection of the interaction network maps also suggests that the CCC complex, through its putative interaction with the BLOC-1, exocyst, and WAVE complexes, may be involved in processes outside of endosomal sorting such as lysosome biogenesis, exocytosis, and plasma membrane dynamics. All of these will be important areas to explore in future studies. In fact, recent studies have uncovered that the WASH complex is recruited to the BLOC-1 complex, in agreement with a possible interplay of the CCC complex in these processes (Ryder et al., 2013). Finally, one critical question that remains to be addressed is the reason for the conservation of 10 COMMD genes in most eukaryotes. It is noteworthy that COMMD proteins also demonstrate an interdependence, suggesting that their ability to heterodimerize (Burstein et al., 2005) is critical to their biological function. Although the specific contribution that each COMMD protein makes in receptor trafficking remains obscure, it is tempting to speculate that individual members of this family may differentially regulate cargo specificity or subcellular localization of the complex or affect distinct aspects of the receptor trafficking process.

In summary, this study uncovers the mechanism by which COMMD1 regulates copper metabolism and identifies its essential role in the stable assembly of the CCC complex, an evolutionarily conserved module that is linked to the WASH complex. Given the large number of endosomal cargoes regulated by WASH and retromer, it can be anticipated that the CCC complex participates in a variety of additional physiological and disease processes, as evident by the complex developmental phenotype resulting from CCDC22 mutations (Voineagu et al., 2012; Starokadomskyy et al., 2013). Thus future work aimed at deciphering the actions of the CCC complex and its role in regulating WASH-dependent receptor trafficking should provide mechanistic insight that will have broad biomedical implications.

MATERIALS AND METHODSHuman studiesAll patient-related evaluation was performed with the consent of participating patients and their family members and was reviewed and approved by the pertinent institutional review boards as previously reported (Starokadomskyy et al., 2013).Cell cultureFibroblasts from Commd1 floxed mice, Wash floxed mice, and patients with the CCDC22 T17A mutation have been previously described (Vonk et al., 2011; Gomez et al., 2012; Starokadomskyy et al., 2013). HEK293T cells and HeLa cells were obtained from the American Type Culture Collection (Manassas, VA). All cell lines were maintained in high-glucose DMEM containing 10% fetal bovine serum and supplemented with l-glutamine.Copper treatmentsHigh-copper conditions consisted of supplementing the growth medium with CuCl2 to a final concentration of 200 μM for 48 h. Conversely, in order to examine the reversal of copper-induced trafficking, low-copper conditions were established by adding 200 μM bathocuproine disulfonate to the growth medium for 24 h after the cells had been first exposed to high copper for 24 h (200 μM CuCl2).Immunofluorescence stainingThis was carried out as previously described (Gomez and Billadeau, 2009). Cells were fixed in ice-cold fixative (4% paraformaldehyde and 0.5% glutaraldehyde in phosphate-buffered saline [PBS]) and incubated for 18 min at room temperature in the dark, followed by permeabilization with 0.2% Triton X-100 in PBS for 4 min. Cells were then incubated with 10 μg/ml primary antibodies (Abs) in IF buffer (Tris-buffered saline plus human serum cocktail) overnight at 4°C in a humidifier chamber. After three washes in PBS, cells were incubated with secondary Abs (1:500 dilution in blocking buffer) for 1 h at room temperature or overnight at 4°C in a humidifier chamber. After four washes in PBS and addition of Hoechst 33342 nuclear stain, coverslips were rinsed in water and affixed to slides with SlowFade Antifade reagent (Life Technologies, Grand Island, NY). Alexa Fluor 647–phalloidin (A22287), fluorescein isothiocyanate–phalloidin (F432), and rhodamine–phalloidin (R415) (Life Technologies) were used to visualize F-actin. Images were obtained with an LSM-710 laser scanning confocal microscope with a 100×/1.4 oil Plan-Apochromat objective lens using ZEN software (Carl Zeiss, Oberkochen, Germany). Mean fluorescence intensity measurements and quantitative Pearson correlation coefficients were obtained using Zen 2009 software (Carl Zeiss). Puncta enumeration was measured using the analyze particle tool in ImageJ (National Institutes of Health, Bethesda, MD). A total of 75 cells were quantified, and the data are displayed as the average number of puncta per cell.Plasmids, RNA interference, transfection methods, and lentiviral productionThe plasmids pEBB, pEBB-HA, and pEBB-HA-CCDC22 (including full length and the truncation mutants 1–321 and 321–627) have been previously described (Starokadomskyy et al., 2013). For this study, we PCR amplified the coding sequence of CCDC93 and C16orf62 from the IMAGE clones 4826022 and 6452778, respectively. The various truncation mutants of CCDC22 and CCDC93 noted in Figure 2 were generated by PCR amplification of the respective coding sequences. The vectors used for short hairpin RNA (shRNA) suppression and reexpression of FAM21 domains have been previously described (Gomez and Billadeau, 2009). The plasmids pEBB-AcGFP1, pEBB-E2Crimson, and pEBB-mOrange2 were generated by PCR amplification of the corresponding coding sequences and insertion into the BamHI site of pEBB. Directed subcloning was used to generate the constructs for fluorescent fusions of COMMD1, CCDC22, CCDC93, and C16orf62. These constructs were transfected in HEK293T cells using calcium phosphate as a transfection method (Maine et al., 2009) or into HeLa cells using X-treme Gene HP DNA transfection reagent (Roche, Indianapolis, IN). pCI2.F.YFP constructs expressing CCDC93 wild type and CCDC93 1–438 were generated using standard molecular biology techniques. RNA interference with siRNA was performed using siRNA duplexes obtained from (Sigma-Aldrich, St. Louis, MO; Supplemental Table S1). Transfection of siRNA in HEK293T cells was performed using calcium phosphate, and transfection in HeLa cells was performed using RNAiMAX (Life Technologies). For shRNA-mediated silencing, the lentiviral vector pLKO.TRC vector was used with a standard viral vector production and selection protocol as previously described (Maine et al., 2007). The targeted sequences used are noted in Supplemental Table S1.CRISPR/Cas9-mediated gene deletionA HeLa cell line lacking CCDC93 was generated using an RNA guide sequence (GACTTTCCGGCGATGTAGCTGGG) targeting exon 12 and Cas9/CRISPR technology as described (Mali et al., 2013). Clones were isolated and screened for CCDC93 expression by immunoblot. Gene disruption was validated by Sanger sequencing at the Gene Analysis Shared Resource, which identified an eight–base pair deletion within the exon 12 target (underlined in the guide sequence).Biotinylation of plasma membrane proteinsCell surface biotinylation was performed in cells using Sulfo-NHS-SS-biotin (Pierce, Rockford, IL) in biotinylation buffer (10 mM triethanolamine, 150 mM NaCl, 2 mM CaCl2, pH 8.0). Labeling was allowed for 30 min at 4ºC. The biotinylated antigens were precipitated using NeutrAvidin agarose resin (Pierce). Precipitated protein-containing beads were then resuspended in LDS sample buffer (Life Technologies) with 250 mM dithiothreitol (DTT). Proteins were then subjected to SDS–PAGE, and membranes were probed for ATP7A and N-Cadherin.Protein extraction, immunoblotting, and immunoprecipitationCell lysate preparation, immunoprecipitation, and immunoblotting were performed as previously described (Burstein et al., 2004, 2005; Mao et al., 2009). Most immunoblotting and immunoprecipitation experiments were performed using a Triton X-100 lysis buffer (25 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid [HEPES], 100 mM NaCl, 10 mM DTT, 1 mM EDTA, 10% glycerol, 1% Triton X-100). Coprecipitation experiments involving WASH complex components were performed using MRB buffer (20 mM HEPES, pH 7.2, 50 mM potassium acetate, 1 mM EDTA, 200 mM d-sorbitol, 0.1% Triton X-100).In vitro protein interaction studiesMBP and GST fusion proteins of CCDC93 and CCDC22 were generated by subcloning into pMalmut2 or pGEX-KGmut. The FAM21 GST fusions FM, FN, and FC and HA-YFP-FAM21 suppression/reexpression vectors were previously described (Jia et al., 2012). Fusion proteins were purified from E. coli BL21 (Life Technologies One Shot BL21 [DE3] chemically competent E. coli; 44–0048) as previously described (Ham et al., 2013). GST-fused proteins (40 μg) were bound to 40 μl of GSH agarose in 250 μl MRB buffer and incubated with rotation at 4°C for 1 h and then washed once in MRB buffer. MBP-fused proteins or MBP were added to the agarose-bound GST-CCDC22 or FAM21 tail fragments and incubated with rotation at 4°C for 3 h, washed twice in MRB buffer, and eluted in 20 μl of 4× sample loading buffer. Samples were subsequently run out in duplicate on 8.75% SDS–PAGE gels and stained with Coomassie and/or immunoblotted with anti-MBP.Surface plasmon resonanceSurface plasmon resonance (SPR) was performed as previously described (Dai et al., 2011). In brief, proteins for SPR were purified as indicated, dialyzed against Biacore buffer (10 mM HEPES, pH 7.4, 150 mM NaCl, 0.05 mM EDTA, and 0.005% [wt/vol] polysorbate 20) and stored at 4°C before use. GST-FAM21 FN was immobilized on a CM5 chip, and binding assays were performed at 25°C on a Biacore T200 biosensor (Biacore, Uppsala, Sweden). Biacore buffer containing MBP or MBP-CCDC93 (448–631) at various concentrations was injected at 30 μl/min for 1 min. Bound protein was allowed to dissociate in Biacore buffer at 30 μl/min for 10 min and then desorbed with 3 M MgCl2. Binding kinetics were derived using BIA evaluation software (Biacore).AntibodiesAntibodies to WASH, FAM21, and VPS35 were generated as previously described (Gomez and Billadeau, 2009; Jia et al., 2010). Antibodies to CCDC22, CCDC93, and C16orf62 were generated by immunizing rabbits with purified GST fusion proteins (Cocalico Biologicals, Reamstown, PA): CCDC22 (amino acids [aa] 432–627; GenBank, BC011675), CCDC93 (aa 402–631; NCBI, NM_019044.4), and C16orf62 (aa 73–300; NCBI, NM_020314.5). COMMD1, COMMD6, and COMMD9 antibodies were also generated as described previously (Starokadomskyy et al., 2013). The ATP7A antibody was obtained as a gift from Michael Petris (Biochemistry, University of Missouri, Columbia, MO). All other antibodies used in this study are listed in Supplemental Table S2.Cellular copper measurementsDeterminations of copper concentration were made by atomic absorption measurements of cellular lysates prepared as described previously, with minor adjustments (Vonk et al., 2011). Cell lysates were dissolved in HNO3:HClO4 (ratio 4:1) and incubated at 60°C for 3 h. Copper measurements were performed with a Thermo GF95Z Zeeman Graphite Furnace absorption spectrometer. Copper concentrations were corrected for protein concentrations.Quantitative real-time PCRTotal RNA was extracted utilizing Qiagen RNeasy kits. A Reverse Transcriptase kit (Life Technologies) was used to translate RNA to cDNA. cDNA was then subjected to real-time PCR (Mastercycler ep realplex2 system; Eppendorf, Hauppauge, NY) using SYBR Green master mix and the gene-specific primers listed in Supplemental Table S3.Statistical analysisIn all cases, the mean is presented, and the error bars correspond to the SEM. Statistical comparisons between means were performed using Student's t test with Welch's correction (Figures 1D and 5A); comparisons between distributions were performed using the chi-squared method (Figures 1B and 4, B and E).

MATERIALS AND METHODS

Human studiesAll patient-related evaluation was performed with the consent of participating patients and their family members and was reviewed and approved by the pertinent institutional review boards as previously reported (Starokadomskyy et al., 2013).

Human studies

All patient-related evaluation was performed with the consent of participating patients and their family members and was reviewed and approved by the pertinent institutional review boards as previously reported (Starokadomskyy et al., 2013).

Cell cultureFibroblasts from Commd1 floxed mice, Wash floxed mice, and patients with the CCDC22 T17A mutation have been previously described (Vonk et al., 2011; Gomez et al., 2012; Starokadomskyy et al., 2013). HEK293T cells and HeLa cells were obtained from the American Type Culture Collection (Manassas, VA). All cell lines were maintained in high-glucose DMEM containing 10% fetal bovine serum and supplemented with l-glutamine.

Cell culture

Fibroblasts from Commd1 floxed mice, Wash floxed mice, and patients with the CCDC22 T17A mutation have been previously described (Vonk et al., 2011; Gomez et al., 2012; Starokadomskyy et al., 2013). HEK293T cells and HeLa cells were obtained from the American Type Culture Collection (Manassas, VA). All cell lines were maintained in high-glucose DMEM containing 10% fetal bovine serum and supplemented with l-glutamine.

Copper treatmentsHigh-copper conditions consisted of supplementing the growth medium with CuCl2 to a final concentration of 200 μM for 48 h. Conversely, in order to examine the reversal of copper-induced trafficking, low-copper conditions were established by adding 200 μM bathocuproine disulfonate to the growth medium for 24 h after the cells had been first exposed to high copper for 24 h (200 μM CuCl2).

Copper treatments

High-copper conditions consisted of supplementing the growth medium with CuCl2 to a final concentration of 200 μM for 48 h. Conversely, in order to examine the reversal of copper-induced trafficking, low-copper conditions were established by adding 200 μM bathocuproine disulfonate to the growth medium for 24 h after the cells had been first exposed to high copper for 24 h (200 μM CuCl2).

Immunofluorescence stainingThis was carried out as previously described (Gomez and Billadeau, 2009). Cells were fixed in ice-cold fixative (4% paraformaldehyde and 0.5% glutaraldehyde in phosphate-buffered saline [PBS]) and incubated for 18 min at room temperature in the dark, followed by permeabilization with 0.2% Triton X-100 in PBS for 4 min. Cells were then incubated with 10 μg/ml primary antibodies (Abs) in IF buffer (Tris-buffered saline plus human serum cocktail) overnight at 4°C in a humidifier chamber. After three washes in PBS, cells were incubated with secondary Abs (1:500 dilution in blocking buffer) for 1 h at room temperature or overnight at 4°C in a humidifier chamber. After four washes in PBS and addition of Hoechst 33342 nuclear stain, coverslips were rinsed in water and affixed to slides with SlowFade Antifade reagent (Life Technologies, Grand Island, NY). Alexa Fluor 647–phalloidin (A22287), fluorescein isothiocyanate–phalloidin (F432), and rhodamine–phalloidin (R415) (Life Technologies) were used to visualize F-actin. Images were obtained with an LSM-710 laser scanning confocal microscope with a 100×/1.4 oil Plan-Apochromat objective lens using ZEN software (Carl Zeiss, Oberkochen, Germany). Mean fluorescence intensity measurements and quantitative Pearson correlation coefficients were obtained using Zen 2009 software (Carl Zeiss). Puncta enumeration was measured using the analyze particle tool in ImageJ (National Institutes of Health, Bethesda, MD). A total of 75 cells were quantified, and the data are displayed as the average number of puncta per cell.

Immunofluorescence staining

This was carried out as previously described (Gomez and Billadeau, 2009). Cells were fixed in ice-cold fixative (4% paraformaldehyde and 0.5% glutaraldehyde in phosphate-buffered saline [PBS]) and incubated for 18 min at room temperature in the dark, followed by permeabilization with 0.2% Triton X-100 in PBS for 4 min. Cells were then incubated with 10 μg/ml primary antibodies (Abs) in IF buffer (Tris-buffered saline plus human serum cocktail) overnight at 4°C in a humidifier chamber. After three washes in PBS, cells were incubated with secondary Abs (1:500 dilution in blocking buffer) for 1 h at room temperature or overnight at 4°C in a humidifier chamber. After four washes in PBS and addition of Hoechst 33342 nuclear stain, coverslips were rinsed in water and affixed to slides with SlowFade Antifade reagent (Life Technologies, Grand Island, NY). Alexa Fluor 647–phalloidin (A22287), fluorescein isothiocyanate–phalloidin (F432), and rhodamine–phalloidin (R415) (Life Technologies) were used to visualize F-actin. Images were obtained with an LSM-710 laser scanning confocal microscope with a 100×/1.4 oil Plan-Apochromat objective lens using ZEN software (Carl Zeiss, Oberkochen, Germany). Mean fluorescence intensity measurements and quantitative Pearson correlation coefficients were obtained using Zen 2009 software (Carl Zeiss). Puncta enumeration was measured using the analyze particle tool in ImageJ (National Institutes of Health, Bethesda, MD). A total of 75 cells were quantified, and the data are displayed as the average number of puncta per cell.

Plasmids, RNA interference, transfection methods, and lentiviral productionThe plasmids pEBB, pEBB-HA, and pEBB-HA-CCDC22 (including full length and the truncation mutants 1–321 and 321–627) have been previously described (Starokadomskyy et al., 2013). For this study, we PCR amplified the coding sequence of CCDC93 and C16orf62 from the IMAGE clones 4826022 and 6452778, respectively. The various truncation mutants of CCDC22 and CCDC93 noted in Figure 2 were generated by PCR amplification of the respective coding sequences. The vectors used for short hairpin RNA (shRNA) suppression and reexpression of FAM21 domains have been previously described (Gomez and Billadeau, 2009). The plasmids pEBB-AcGFP1, pEBB-E2Crimson, and pEBB-mOrange2 were generated by PCR amplification of the corresponding coding sequences and insertion into the BamHI site of pEBB. Directed subcloning was used to generate the constructs for fluorescent fusions of COMMD1, CCDC22, CCDC93, and C16orf62. These constructs were transfected in HEK293T cells using calcium phosphate as a transfection method (Maine et al., 2009) or into HeLa cells using X-treme Gene HP DNA transfection reagent (Roche, Indianapolis, IN). pCI2.F.YFP constructs expressing CCDC93 wild type and CCDC93 1–438 were generated using standard molecular biology techniques. RNA interference with siRNA was performed using siRNA duplexes obtained from (Sigma-Aldrich, St. Louis, MO; Supplemental Table S1). Transfection of siRNA in HEK293T cells was performed using calcium phosphate, and transfection in HeLa cells was performed using RNAiMAX (Life Technologies). For shRNA-mediated silencing, the lentiviral vector pLKO.TRC vector was used with a standard viral vector production and selection protocol as previously described (Maine et al., 2007). The targeted sequences used are noted in Supplemental Table S1.

Plasmids, RNA interference, transfection methods, and lentiviral production

The plasmids pEBB, pEBB-HA, and pEBB-HA-CCDC22 (including full length and the truncation mutants 1–321 and 321–627) have been previously described (Starokadomskyy et al., 2013). For this study, we PCR amplified the coding sequence of CCDC93 and C16orf62 from the IMAGE clones 4826022 and 6452778, respectively. The various truncation mutants of CCDC22 and CCDC93 noted in Figure 2 were generated by PCR amplification of the respective coding sequences. The vectors used for short hairpin RNA (shRNA) suppression and reexpression of FAM21 domains have been previously described (Gomez and Billadeau, 2009). The plasmids pEBB-AcGFP1, pEBB-E2Crimson, and pEBB-mOrange2 were generated by PCR amplification of the corresponding coding sequences and insertion into the BamHI site of pEBB. Directed subcloning was used to generate the constructs for fluorescent fusions of COMMD1, CCDC22, CCDC93, and C16orf62. These constructs were transfected in HEK293T cells using calcium phosphate as a transfection method (Maine et al., 2009) or into HeLa cells using X-treme Gene HP DNA transfection reagent (Roche, Indianapolis, IN). pCI2.F.YFP constructs expressing CCDC93 wild type and CCDC93 1–438 were generated using standard molecular biology techniques. RNA interference with siRNA was performed using siRNA duplexes obtained from (Sigma-Aldrich, St. Louis, MO; Supplemental Table S1). Transfection of siRNA in HEK293T cells was performed using calcium phosphate, and transfection in HeLa cells was performed using RNAiMAX (Life Technologies). For shRNA-mediated silencing, the lentiviral vector pLKO.TRC vector was used with a standard viral vector production and selection protocol as previously described (Maine et al., 2007). The targeted sequences used are noted in Supplemental Table S1.

CRISPR/Cas9-mediated gene deletionA HeLa cell line lacking CCDC93 was generated using an RNA guide sequence (GACTTTCCGGCGATGTAGCTGGG) targeting exon 12 and Cas9/CRISPR technology as described (Mali et al., 2013). Clones were isolated and screened for CCDC93 expression by immunoblot. Gene disruption was validated by Sanger sequencing at the Gene Analysis Shared Resource, which identified an eight–base pair deletion within the exon 12 target (underlined in the guide sequence).

CRISPR/Cas9-mediated gene deletion

A HeLa cell line lacking CCDC93 was generated using an RNA guide sequence (GACTTTCCGGCGATGTAGCTGGG) targeting exon 12 and Cas9/CRISPR technology as described (Mali et al., 2013). Clones were isolated and screened for CCDC93 expression by immunoblot. Gene disruption was validated by Sanger sequencing at the Gene Analysis Shared Resource, which identified an eight–base pair deletion within the exon 12 target (underlined in the guide sequence).

Biotinylation of plasma membrane proteinsCell surface biotinylation was performed in cells using Sulfo-NHS-SS-biotin (Pierce, Rockford, IL) in biotinylation buffer (10 mM triethanolamine, 150 mM NaCl, 2 mM CaCl2, pH 8.0). Labeling was allowed for 30 min at 4ºC. The biotinylated antigens were precipitated using NeutrAvidin agarose resin (Pierce). Precipitated protein-containing beads were then resuspended in LDS sample buffer (Life Technologies) with 250 mM dithiothreitol (DTT). Proteins were then subjected to SDS–PAGE, and membranes were probed for ATP7A and N-Cadherin.

Biotinylation of plasma membrane proteins

Cell surface biotinylation was performed in cells using Sulfo-NHS-SS-biotin (Pierce, Rockford, IL) in biotinylation buffer (10 mM triethanolamine, 150 mM NaCl, 2 mM CaCl2, pH 8.0). Labeling was allowed for 30 min at 4ºC. The biotinylated antigens were precipitated using NeutrAvidin agarose resin (Pierce). Precipitated protein-containing beads were then resuspended in LDS sample buffer (Life Technologies) with 250 mM dithiothreitol (DTT). Proteins were then subjected to SDS–PAGE, and membranes were probed for ATP7A and N-Cadherin.

Protein extraction, immunoblotting, and immunoprecipitationCell lysate preparation, immunoprecipitation, and immunoblotting were performed as previously described (Burstein et al., 2004, 2005; Mao et al., 2009). Most immunoblotting and immunoprecipitation experiments were performed using a Triton X-100 lysis buffer (25 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid [HEPES], 100 mM NaCl, 10 mM DTT, 1 mM EDTA, 10% glycerol, 1% Triton X-100). Coprecipitation experiments involving WASH complex components were performed using MRB buffer (20 mM HEPES, pH 7.2, 50 mM potassium acetate, 1 mM EDTA, 200 mM d-sorbitol, 0.1% Triton X-100).

Protein extraction, immunoblotting, and immunoprecipitation

Cell lysate preparation, immunoprecipitation, and immunoblotting were performed as previously described (Burstein et al., 2004, 2005; Mao et al., 2009). Most immunoblotting and immunoprecipitation experiments were performed using a Triton X-100 lysis buffer (25 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid [HEPES], 100 mM NaCl, 10 mM DTT, 1 mM EDTA, 10% glycerol, 1% Triton X-100). Coprecipitation experiments involving WASH complex components were performed using MRB buffer (20 mM HEPES, pH 7.2, 50 mM potassium acetate, 1 mM EDTA, 200 mM d-sorbitol, 0.1% Triton X-100).

In vitro protein interaction studiesMBP and GST fusion proteins of CCDC93 and CCDC22 were generated by subcloning into pMalmut2 or pGEX-KGmut. The FAM21 GST fusions FM, FN, and FC and HA-YFP-FAM21 suppression/reexpression vectors were previously described (Jia et al., 2012). Fusion proteins were purified from E. coli BL21 (Life Technologies One Shot BL21 [DE3] chemically competent E. coli; 44–0048) as previously described (Ham et al., 2013). GST-fused proteins (40 μg) were bound to 40 μl of GSH agarose in 250 μl MRB buffer and incubated with rotation at 4°C for 1 h and then washed once in MRB buffer. MBP-fused proteins or MBP were added to the agarose-bound GST-CCDC22 or FAM21 tail fragments and incubated with rotation at 4°C for 3 h, washed twice in MRB buffer, and eluted in 20 μl of 4× sample loading buffer. Samples were subsequently run out in duplicate on 8.75% SDS–PAGE gels and stained with Coomassie and/or immunoblotted with anti-MBP.

In vitro protein interaction studies

MBP and GST fusion proteins of CCDC93 and CCDC22 were generated by subcloning into pMalmut2 or pGEX-KGmut. The FAM21 GST fusions FM, FN, and FC and HA-YFP-FAM21 suppression/reexpression vectors were previously described (Jia et al., 2012). Fusion proteins were purified from E. coli BL21 (Life Technologies One Shot BL21 [DE3] chemically competent E. coli; 44–0048) as previously described (Ham et al., 2013). GST-fused proteins (40 μg) were bound to 40 μl of GSH agarose in 250 μl MRB buffer and incubated with rotation at 4°C for 1 h and then washed once in MRB buffer. MBP-fused proteins or MBP were added to the agarose-bound GST-CCDC22 or FAM21 tail fragments and incubated with rotation at 4°C for 3 h, washed twice in MRB buffer, and eluted in 20 μl of 4× sample loading buffer. Samples were subsequently run out in duplicate on 8.75% SDS–PAGE gels and stained with Coomassie and/or immunoblotted with anti-MBP.

Surface plasmon resonanceSurface plasmon resonance (SPR) was performed as previously described (Dai et al., 2011). In brief, proteins for SPR were purified as indicated, dialyzed against Biacore buffer (10 mM HEPES, pH 7.4, 150 mM NaCl, 0.05 mM EDTA, and 0.005% [wt/vol] polysorbate 20) and stored at 4°C before use. GST-FAM21 FN was immobilized on a CM5 chip, and binding assays were performed at 25°C on a Biacore T200 biosensor (Biacore, Uppsala, Sweden). Biacore buffer containing MBP or MBP-CCDC93 (448–631) at various concentrations was injected at 30 μl/min for 1 min. Bound protein was allowed to dissociate in Biacore buffer at 30 μl/min for 10 min and then desorbed with 3 M MgCl2. Binding kinetics were derived using BIA evaluation software (Biacore).

Surface plasmon resonance

Surface plasmon resonance (SPR) was performed as previously described (Dai et al., 2011). In brief, proteins for SPR were purified as indicated, dialyzed against Biacore buffer (10 mM HEPES, pH 7.4, 150 mM NaCl, 0.05 mM EDTA, and 0.005% [wt/vol] polysorbate 20) and stored at 4°C before use. GST-FAM21 FN was immobilized on a CM5 chip, and binding assays were performed at 25°C on a Biacore T200 biosensor (Biacore, Uppsala, Sweden). Biacore buffer containing MBP or MBP-CCDC93 (448–631) at various concentrations was injected at 30 μl/min for 1 min. Bound protein was allowed to dissociate in Biacore buffer at 30 μl/min for 10 min and then desorbed with 3 M MgCl2. Binding kinetics were derived using BIA evaluation software (Biacore).

AntibodiesAntibodies to WASH, FAM21, and VPS35 were generated as previously described (Gomez and Billadeau, 2009; Jia et al., 2010). Antibodies to CCDC22, CCDC93, and C16orf62 were generated by immunizing rabbits with purified GST fusion proteins (Cocalico Biologicals, Reamstown, PA): CCDC22 (amino acids [aa] 432–627; GenBank, BC011675), CCDC93 (aa 402–631; NCBI, NM_019044.4), and C16orf62 (aa 73–300; NCBI, NM_020314.5). COMMD1, COMMD6, and COMMD9 antibodies were also generated as described previously (Starokadomskyy et al., 2013). The ATP7A antibody was obtained as a gift from Michael Petris (Biochemistry, University of Missouri, Columbia, MO). All other antibodies used in this study are listed in Supplemental Table S2.

Antibodies to WASH, FAM21, and VPS35 were generated as previously described (Gomez and Billadeau, 2009; Jia et al., 2010). Antibodies to CCDC22, CCDC93, and C16orf62 were generated by immunizing rabbits with purified GST fusion proteins (Cocalico Biologicals, Reamstown, PA): CCDC22 (amino acids [aa] 432–627; GenBank, BC011675), CCDC93 (aa 402–631; NCBI, NM_019044.4), and C16orf62 (aa 73–300; NCBI, NM_020314.5). COMMD1, COMMD6, and COMMD9 antibodies were also generated as described previously (Starokadomskyy et al., 2013). The ATP7A antibody was obtained as a gift from Michael Petris (Biochemistry, University of Missouri, Columbia, MO). All other antibodies used in this study are listed in Supplemental Table S2.

Cellular copper measurementsDeterminations of copper concentration were made by atomic absorption measurements of cellular lysates prepared as described previously, with minor adjustments (Vonk et al., 2011). Cell lysates were dissolved in HNO3:HClO4 (ratio 4:1) and incubated at 60°C for 3 h. Copper measurements were performed with a Thermo GF95Z Zeeman Graphite Furnace absorption spectrometer. Copper concentrations were corrected for protein concentrations.

Cellular copper measurements

Determinations of copper concentration were made by atomic absorption measurements of cellular lysates prepared as described previously, with minor adjustments (Vonk et al., 2011). Cell lysates were dissolved in HNO3:HClO4 (ratio 4:1) and incubated at 60°C for 3 h. Copper measurements were performed with a Thermo GF95Z Zeeman Graphite Furnace absorption spectrometer. Copper concentrations were corrected for protein concentrations.

Quantitative real-time PCRTotal RNA was extracted utilizing Qiagen RNeasy kits. A Reverse Transcriptase kit (Life Technologies) was used to translate RNA to cDNA. cDNA was then subjected to real-time PCR (Mastercycler ep realplex2 system; Eppendorf, Hauppauge, NY) using SYBR Green master mix and the gene-specific primers listed in Supplemental Table S3.

Quantitative real-time PCR

Total RNA was extracted utilizing Qiagen RNeasy kits. A Reverse Transcriptase kit (Life Technologies) was used to translate RNA to cDNA. cDNA was then subjected to real-time PCR (Mastercycler ep realplex2 system; Eppendorf, Hauppauge, NY) using SYBR Green master mix and the gene-specific primers listed in Supplemental Table S3.

Statistical analysisIn all cases, the mean is presented, and the error bars correspond to the SEM. Statistical comparisons between means were performed using Student's t test with Welch's correction (Figures 1D and 5A); comparisons between distributions were performed using the chi-squared method (Figures 1B and 4, B and E).

Statistical analysis

In all cases, the mean is presented, and the error bars correspond to the SEM. Statistical comparisons between means were performed using Student's t test with Welch's correction (Figures 1D and 5A); comparisons between distributions were performed using the chi-squared method (Figures 1B and 4, B and E).

Supplementary MaterialSupplemental Materials

Supplementary Material

Supplemental Materials
