---
pmid: '29887526'
title: Middle East respiratory syndrome coronavirus and bat coronavirus HKU9 both
  can utilize GRP78 for attachment onto host cells.
authors:
- Chu H
- Chan CM
- Zhang X
- Wang Y
- Yuan S
- Zhou J
- Au-Yeung RK
- Sze KH
- Yang D
- Shuai H
- Hou Y
- Li C
- Zhao X
- Poon VK
- Leung SP
- Yeung ML
- Yan J
- Lu G
- Jin DY
- Gao GF
- Chan JF
- Yuen KY
journal: J Biol Chem
year: '2018'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6066311
doi: 10.1074/jbc.RA118.001897
---

# Middle East respiratory syndrome coronavirus and bat coronavirus HKU9 both can utilize GRP78 for attachment onto host cells.
**Authors:** Chu H, Chan CM, Zhang X, Wang Y, Yuan S, Zhou J, Au-Yeung RK, Sze KH, Yang D, Shuai H, Hou Y, Li C, Zhao X, Poon VK, Leung SP, Yeung ML, Yan J, Lu G, Jin DY, Gao GF, Chan JF, Yuen KY
**Journal:** J Biol Chem (2018)
**DOI:** [10.1074/jbc.RA118.001897](https://doi.org/10.1074/jbc.RA118.001897)
**PMC:** [PMC6066311](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6066311/)

## Abstract

1. J Biol Chem. 2018 Jul 27;293(30):11709-11726. doi: 10.1074/jbc.RA118.001897. 
Epub 2018 Jun 10.

Middle East respiratory syndrome coronavirus and bat coronavirus HKU9 both can 
utilize GRP78 for attachment onto host cells.

Chu H(1)(2), Chan CM(1)(2), Zhang X(2), Wang Y(2), Yuan S(2), Zhou J(1)(2), 
Au-Yeung RK(3), Sze KH(1)(2), Yang D(2), Shuai H(2), Hou Y(2), Li C(2), Zhao 
X(2), Poon VK(2), Leung SP(2), Yeung ML(1)(2)(4)(5), Yan J(6), Lu G(7), Jin 
DY(8), Gao GF(6)(9), Chan JF(10)(2)(4)(5), Yuen KY(11)(2)(4)(5)(12).

Author information:
(1)From the State Key Laboratory of Emerging Infectious Diseases.
(2)Departments of Microbiology and.
(3)Pathology.
(4)Research Centre of Infection and Immunology.
(5)Carol Yu Centre for Infection.
(6)the CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute 
of Microbiology, Chinese Academy of Sciences, Beijing 100101.
(7)the West China Hospital Emergency Department, State Key Laboratory of 
Biotherapy, West China Hospital, Sichuan University, and Collaborative 
Innovation Center of Biotherapy, Chengdu, Sichuan 610041, and.
(8)School of Biomedical Sciences, and.
(9)the National Institute for Viral Disease Control and Prevention, Chinese 
Center for Disease Control and Prevention (China CDC), Beijing 102206, China.
(10)From the State Key Laboratory of Emerging Infectious Diseases, 
jfwchan@hku.hk.
(11)From the State Key Laboratory of Emerging Infectious Diseases, 
kyyuen@hku.hk.
(12)Collaborative Innovation Center for Diagnosis and Treatment of Infectious 
Diseases, Li Ka Shing Faculty of Medicine, University of Hong Kong, Pokfulam, 
Hong Kong Special Administrative Region.

Coronavirus tropism is predominantly determined by the interaction between 
coronavirus spikes and the host receptors. In this regard, coronaviruses have 
evolved a complicated receptor-recognition system through their spike proteins. 
Spikes from highly related coronaviruses can recognize distinct receptors, 
whereas spikes of distant coronaviruses can employ the same cell-surface 
molecule for entry. Moreover, coronavirus spikes can recognize a broad range of 
cell-surface molecules in addition to the receptors and thereby can augment 
coronavirus attachment or entry. The receptor of Middle East respiratory 
syndrome coronavirus (MERS-CoV) is dipeptidyl peptidase 4 (DPP4). In this study, 
we identified membrane-associated 78-kDa glucose-regulated protein (GRP78) as an 
additional binding target of the MERS-CoV spike. Further analyses indicated that 
GRP78 could not independently render nonpermissive cells susceptible to MERS-CoV 
infection but could facilitate MERS-CoV entry into permissive cells by 
augmenting virus attachment. More importantly, by exploring potential 
interactions between GRP78 and spikes of other coronaviruses, we discovered that 
the highly conserved human GRP78 could interact with the spike protein of bat 
coronavirus HKU9 (bCoV-HKU9) and facilitate its attachment to the host cell 
surface. Taken together, our study has identified GRP78 as a host factor that 
can interact with the spike proteins of two Betacoronaviruses, the lineage C 
MERS-CoV and the lineage D bCoV-HKU9. The capacity of GRP78 to facilitate 
surface attachment of both a human coronavirus and a phylogenetically related 
bat coronavirus exemplifies the need for continuous surveillance of the 
evolution of animal coronaviruses to monitor their potential for human 
adaptations.

© 2018 Chu et al.

DOI: 10.1074/jbc.RA118.001897
PMCID: PMC6066311
PMID: 29887526 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflicts 
of interest with the contents of this article

## Full Text

IntroductionCoronaviruses are known to infect a broad spectrum of species, ranging from birds to
mammals, including humans (1–3). They are enveloped RNA viruses with large genome sizes of
∼28–32 kb. Currently, coronaviruses are classified into four genera:
Alphacoronavirus, Betacoronavirus, Gammacoronavirus, and
Deltacoronavirus (4).
Among them, six coronaviruses from the Alphacoronavirus genera and
the Betacoronavirus genera are known to cause human infections with
diverse outcomes. On the one hand, human coronavirus 229E (HCoV-229E),5 human coronavirus NL63 (HCoV-NL63),
human coronavirus OC43 (HCoV-OC43), and human coronavirus HKU1 (HCoV-HKU1)
predominantly cause mild and self-limiting upper respiratory tract infections (5, 6). In
stark contrast, severe acute respiratory syndrome coronavirus (SARS-CoV) that caused
the severe acute respiratory syndrome epidemic between 2002 and 2003 was highly
pathogenic, which infected more than 8000 people with a fatality rate of ∼10%
(7, 8). Ten years later, another highly pathogenic human coronavirus, Middle
East respiratory syndrome coronavirus (MERS-CoV), emerged in the Middle East in 2012
(9). MERS-CoV caused severe lower
respiratory tract infections with an exceptionally high fatality rate of
∼35%. Most importantly, despite global efforts trying to control the virus'
dissemination, MERS-CoV still spread to over 27 countries and has been causing
continuous infections in the Middle East since 2012 (10).The interaction between the spike protein and its receptor is the main determinant of
host tropism for coronaviruses (11). Among
the six human coronaviruses, the Alphacoronavirus HCoV-229E spike
binds aminopeptidase N (12), whereas the
lineage C Betacoronavirus the MERS-CoV spike recognizes dipeptidyl
peptidase 4 (DPP4) (13). Intriguingly, the
Alphacoronavirus HCoV-NL63 and the lineage B
Betacoronavirus SARS-CoV both utilize angiotensin-converting
enzyme 2 (ACE2) for cell entry (14, 15). However, the protein receptors for the
lineage A Betacoronavirus HCoV-OC43 and HCoV-HKU1 are currently
unknown. In addition to their designated receptors, coronavirus spikes are known to
recognize a broad array of cell-surface molecules, which serve to facilitate the
attachment or entry of the viruses. For example, HCoV-NL63 and mouse hepatitis virus
both employ heparan sulfate proteoglycans to enhance attachment (16, 17).
Similarly, transmissible gastroenteritis coronavirus, bovine coronavirus, HCoV-OC43,
and HCoV-HKU1 bind to O-acetylated sialic acid as key attachment
molecules (18–21).
Interestingly, in addition to utilizing O-acetylated sialic acid as
a critical binding determinant (21),
HCoV-HKU1 spike also recognizes major histocompatibility complex class I C as
another attachment molecule (22). In the case
of SARS-CoV, dendritic cell–specific intercellular adhesion
molecule-3–grabbing nonintegrin (DC-SIGN) and DC-SIGN–related both
augment virus entry (23, 24). For MERS-CoV, we previously reported
carcinoembryonic antigen-related cell adhesion molecule 5 (CEACAM5) as an attachment
factor that could modulate MERS-CoV entry in permissive cells (25). More recently, tetraspanin CD9 was identified as a host
cell- -surface factor that facilitated MERS-CoV entry by scaffolding host cell
receptors and proteases (26).Knowledge of the interaction between coronavirus spikes and cell-surface host factors
contributes to the understanding of coronavirus biology on many aspects, including
tropism, pathogenicity, as well as potential intervention strategies. To this end,
we aimed to investigate whether additional cell-surface molecules were involved in
the attachment or entry of MERS-CoV. In this study, we reported that the MERS-CoV
spike could recognize a 78-kDa glucose–regulated protein (GRP78). Although
traditionally regarded as an ER protein with chaperone activity, recent discoveries
suggest that GRP78 is also localized to the cell surface, where they carry out
physiological functions that regulate signaling and cellular homeostasis (27). Subsequent experiments demonstrated that
GRP78 did not render nonpermissive cells susceptible to MERS-CoV infection but
played a positive role in augmenting MERS-CoV entry in permissive cells, suggesting
that GRP78 is an attachment factor of MERS-CoV that can modulate MERS-CoV entry in
the presence of the host cell receptor DPP4. Importantly, our data further indicated
that the spike protein of a lineage D Betacoronavirus, bat
coronavirus HKU9 (bCoV-HKU9), also recognized GRP78, which played a key role in the
attachment of HKU9–S-pseudovirus to the bat Rousettus
leschenaulti kidney (RLK) cells. Our findings highlight the importance
of the possible evolution of different animal and human coronaviruses to become
capable of using not just the same host receptors but also the same attachment
factors, which may facilitate animal coronaviruses to jump the interspecies barrier
into human.

Introduction

Coronaviruses are known to infect a broad spectrum of species, ranging from birds to
mammals, including humans (1–3). They are enveloped RNA viruses with large genome sizes of
∼28–32 kb. Currently, coronaviruses are classified into four genera:
Alphacoronavirus, Betacoronavirus, Gammacoronavirus, and
Deltacoronavirus (4).
Among them, six coronaviruses from the Alphacoronavirus genera and
the Betacoronavirus genera are known to cause human infections with
diverse outcomes. On the one hand, human coronavirus 229E (HCoV-229E),5 human coronavirus NL63 (HCoV-NL63),
human coronavirus OC43 (HCoV-OC43), and human coronavirus HKU1 (HCoV-HKU1)
predominantly cause mild and self-limiting upper respiratory tract infections (5, 6). In
stark contrast, severe acute respiratory syndrome coronavirus (SARS-CoV) that caused
the severe acute respiratory syndrome epidemic between 2002 and 2003 was highly
pathogenic, which infected more than 8000 people with a fatality rate of ∼10%
(7, 8). Ten years later, another highly pathogenic human coronavirus, Middle
East respiratory syndrome coronavirus (MERS-CoV), emerged in the Middle East in 2012
(9). MERS-CoV caused severe lower
respiratory tract infections with an exceptionally high fatality rate of
∼35%. Most importantly, despite global efforts trying to control the virus'
dissemination, MERS-CoV still spread to over 27 countries and has been causing
continuous infections in the Middle East since 2012 (10).

The interaction between the spike protein and its receptor is the main determinant of
host tropism for coronaviruses (11). Among
the six human coronaviruses, the Alphacoronavirus HCoV-229E spike
binds aminopeptidase N (12), whereas the
lineage C Betacoronavirus the MERS-CoV spike recognizes dipeptidyl
peptidase 4 (DPP4) (13). Intriguingly, the
Alphacoronavirus HCoV-NL63 and the lineage B
Betacoronavirus SARS-CoV both utilize angiotensin-converting
enzyme 2 (ACE2) for cell entry (14, 15). However, the protein receptors for the
lineage A Betacoronavirus HCoV-OC43 and HCoV-HKU1 are currently
unknown. In addition to their designated receptors, coronavirus spikes are known to
recognize a broad array of cell-surface molecules, which serve to facilitate the
attachment or entry of the viruses. For example, HCoV-NL63 and mouse hepatitis virus
both employ heparan sulfate proteoglycans to enhance attachment (16, 17).
Similarly, transmissible gastroenteritis coronavirus, bovine coronavirus, HCoV-OC43,
and HCoV-HKU1 bind to O-acetylated sialic acid as key attachment
molecules (18–21).
Interestingly, in addition to utilizing O-acetylated sialic acid as
a critical binding determinant (21),
HCoV-HKU1 spike also recognizes major histocompatibility complex class I C as
another attachment molecule (22). In the case
of SARS-CoV, dendritic cell–specific intercellular adhesion
molecule-3–grabbing nonintegrin (DC-SIGN) and DC-SIGN–related both
augment virus entry (23, 24). For MERS-CoV, we previously reported
carcinoembryonic antigen-related cell adhesion molecule 5 (CEACAM5) as an attachment
factor that could modulate MERS-CoV entry in permissive cells (25). More recently, tetraspanin CD9 was identified as a host
cell- -surface factor that facilitated MERS-CoV entry by scaffolding host cell
receptors and proteases (26).

Knowledge of the interaction between coronavirus spikes and cell-surface host factors
contributes to the understanding of coronavirus biology on many aspects, including
tropism, pathogenicity, as well as potential intervention strategies. To this end,
we aimed to investigate whether additional cell-surface molecules were involved in
the attachment or entry of MERS-CoV. In this study, we reported that the MERS-CoV
spike could recognize a 78-kDa glucose–regulated protein (GRP78). Although
traditionally regarded as an ER protein with chaperone activity, recent discoveries
suggest that GRP78 is also localized to the cell surface, where they carry out
physiological functions that regulate signaling and cellular homeostasis (27). Subsequent experiments demonstrated that
GRP78 did not render nonpermissive cells susceptible to MERS-CoV infection but
played a positive role in augmenting MERS-CoV entry in permissive cells, suggesting
that GRP78 is an attachment factor of MERS-CoV that can modulate MERS-CoV entry in
the presence of the host cell receptor DPP4. Importantly, our data further indicated
that the spike protein of a lineage D Betacoronavirus, bat
coronavirus HKU9 (bCoV-HKU9), also recognized GRP78, which played a key role in the
attachment of HKU9–S-pseudovirus to the bat Rousettus
leschenaulti kidney (RLK) cells. Our findings highlight the importance
of the possible evolution of different animal and human coronaviruses to become
capable of using not just the same host receptors but also the same attachment
factors, which may facilitate animal coronaviruses to jump the interspecies barrier
into human.

ResultsGRP78 interacts with MERS-CoV spikeWe previously identified human carcinoembryonic antigen-related cell adhesion
molecule 5 (CEACAM5) as an attachment factor for MERS-CoV (25). In this study, we asked whether additional membrane
proteins could interact with the MERS-CoV spike and facilitate the entry or
attachment of MERS-CoV. To this end, we transfected human bronchus epithelial
cells, BEAS2B, with the MERS-CoV spike and evaluated the membrane proteins that
might bind the MERS-CoV spike in the transfected cells. In brief, membrane
proteins from pcDNA–MERS-CoV–S1–V5-transfected BEAS2B cells
were extracted and sedimented (Fig. 1). To
evaluate the extraction efficiency, the cell extracts were probed for markers of
different cellular fractions, including that of the plasma membrane (epidermal
growth factor receptor (EGFR) and pan-cadherin), endoplasmic reticulum (ER)
(calreticulin), Golgi (giantin), and nucleus (lamin A). Western blotting
analyses revealed that our membrane extracts were enriched with the plasma
membrane markers, EGFR and pan-cadherin. In contrast, only a trace amount of the
ER marker was observed, whereas the signal for Golgi and nucleus was not
detected (Fig. 1D and
Fig.
S1).Figure 1.Identification of GRP78 as a target membrane protein of the
MERS-CoV spike.
A, silver staining of membrane proteins of BEAS2B cells
transfected with pcDNA–MERS-CoV–S1–V5. Membrane
extracts were immunoprecipitated (IP) with V5 antibody
and Sepharose A/G beads, followed by washing and eluting with glycine
(lane 1). Sepharose beads were boiled in sample
buffer after glycine elution (lane 2). Membrane
extracts were immunoprecipitated with mouse isotype control and
Sepharose A/G beads (lane 3). B,
expression of MERS-CoV–S1-V5 was detected by Western blotting
(WB) with an anti-ERS-CoV spike antibody.
C, silver staining of membrane proteins of BEAS2B
cells. The membrane extracts were immunoprecipitated with purified
recombinant MERS-CoV–S1–FLAG protein using anti-FLAG M2
antibody and Sepharose A/G beads, followed by washing and eluting with
3× FLAG peptides (lane 1). Sepharose beads were
boiled in sample buffer after 3× FLAG peptide elution (lane
2). Membrane extracts were immunoprecipitated with mouse
isotype control and Sepharose A/G beads (lane 3).
D, 5 μg of sedimented membrane extracts were
run on SDS-PAGE and subjected to Western blots using antibodies against
the plasma membrane marker (EGFR and pan-cadherin), endoplasmic
reticulum marker (calreticulin), Golgi marker (giantin), and nucleus
marker (lamin A). E, gel fragment indicated by the
red arrowhead in A and
C was excised for LC-MS/MS analysis. MS/MS data
were searched against all mammalian protein databases in NCBI and
Swiss-Prot. The protein was identified as GRP78 with significant hits
over different domains of the sequence.To identify potential proteins that could interact with the MERS-CoV spike, the
membrane extracts were immunoprecipitated with a V5 mAb and protein
A/G-Sepharose. The precipitated beads were then washed, and protein complexes
were eluted with 0.1 m glycine. Co-immunoprecipitated proteins were
revealed in SDS-PAGE after silver staining (Fig.
1A, lane 1). The eluted beads were
resuspended in sample loading buffer, boiled, and assessed for elution
efficiency (Fig. 1A,
lane 2). As a control, the same set of membrane extracts
was immunoprecipitated with isotype antibody and protein A/G-Sepharose (Fig. 1A, lane
3). In parallel, the expression of the MERS-CoV spike in the
immunoprecipitated complexes was validated with Western blotting using a mouse
immune serum against the MERS-CoV spike (Fig.
1B). Specific protein bands that were pulled down by
the V5 antibody but not the isotype control were excised and sent for MS
analysis. The MS/MS result revealed one of the dominant bands (Fig. 1A, lane 1,
arrowhead) to be 78-kDa glucose-regulated protein (GRP78), also
known as heat shock 70-kDa protein 5 (HSPA5) or binding immunoglobulin protein
(BiP) (Fig. 1E).To further verify the interaction between the MERS-CoV spike and GRP78, we
attempted to immunoprecipitate GRP78 with purified the MERS-CoV spike proteins.
To this end, recombinant MERS-CoV–S1–FLAG proteins were expressed,
purified, and immunoprecipitated against the membrane protein extracts from
BEAS2B cells. Notably, silver staining of the SDS-PAGE and the subsequent MS
confirmed the presence of GRP78 in the precipitated
MERS-CoV–S1–FLAG complex (Fig.
1C, lane 1, arrowhead) but not in
the control (Fig. 1C,
lane 3). Taken together, our membrane pulldown assay
identified GRP78 as a potential membrane protein specifically bound by the
MERS-CoV spike.GRP78 is a specific binding target of MERS-CoV spikeNext, to examine the direct interaction between GRP78 and the MERS-CoV spike, we
performed a series of co-immunoprecipitation (co-IP) assays in both
overexpression and endogenous settings. First, BHK21 cells were transfected with
GRP78–V5 or the pcDNA–V5 control vector. The cell lysates of the
transfected cells were then immunoprecipitated with either
MERS-CoV–S1–FLAG or Escherichia coli bacterial
alkaline phosphatase–FLAG (BAP–FLAG) pre-adsorbed on anti-FLAG
M2-agarose beads. The precipitated protein complexes were then detected by
Western blotting with the anti-FLAG or the anti-V5 antibody. As illustrated in
Fig. 2A, GRP78
specifically immunoprecipitated with MERS-CoV–S1 (lower panel,
lane 1) but not the control bait protein, BAP (lower panel,
lane 2). Additionally, GRP78 was not precipitated in cells
transfected with the empty vector (Fig.
2A, lower panel, lane 3). To
confirm the interaction between GRP78 and MERS-CoV–S1, we performed
reciprocal co-IP using GRP78 as the bait protein (Fig. 2B). In this setting, cell lysates of
GRP78–V5 or empty vector transfected BHK21 cells were immunoprecipitated
with anti-V5 pre-adsorbed protein A/G-Sepharose and incubated with purified
MERS-CoV–S1–FLAG or BAP–FLAG. Our result demonstrated that
MERS-CoV–S1–FLAG but not BAP–FLAG was efficiently
immunoprecipitated by GRP78–V5 (Fig.
2B, upper panel, lanes 1 and
2). As a negative control, the expression of
pcDNA–V5 empty vector failed to immunoprecipitate with
MERS-CoV–S1–FLAG (Fig.
2B, upper panel, lane 3). In
parallel, MERS-CoV–S1–FLAG did not co-IP with the abundantly
expressed cell-surface protein EGFR, suggesting the interaction between
MERS-CoV–S1–FLAG and GRP78 was specific (Fig. S2,
A and B). Next, we
evaluated whether the interaction between the MERS-CoV spike and GRP78 could
occur at the cell surface. To this end, we obtained the membrane fraction of
Huh7 cells that was predominantly enriched with the plasma membrane contents of
the cells. We then added MERS-CoV–S1–FLAG protein to the membrane
extracts and performed co-IP between the MERS-CoV spike and GRP78. Our data
showed that the MERS-CoV spike and the endogenous GRP78 in the membrane extract
could efficiently interact with each other (Fig.
2, C and D).Figure 2.GRP78 interacts with the MERS-CoV spike.
A, BHK21 cells were transfected with
pcDNA–GRP78–V5 (lanes 1 and
2) or empty vector (lane 3). The
cell lysate was immunoprecipitated (IP) with either
purified recombinant MERS-CoV–S1–FLAG protein
(lanes 1 and 3) or E.
coli bacterial alkaline phosphatase
(BAP)-FLAG protein (lane 2)
pre-adsorbed onto anti-FLAG M2-agarose beads. The precipitated protein
complex was detected using the anti-FLAG antibody or the anti-V5
antibody. B, reciprocal co-IP was performed using GRP78
as the bait protein. Purified MERS-CoV–S1–FLAG
(lanes 1 and 3) or BAP–FLAG
proteins (lane 2) were immunoprecipitated with
overexpressed GRP78–V5 or pcDNA–V5 proteins pre-adsorbed
on anti-V5 Sepharose beads. The precipitated protein complex was
detected using the anti-FLAG antibody or the anti-GRP78 antibody.
C, membrane fraction of Huh7 cells was extracted
and immunoprecipitated with either
MERS-CoV–S1–FLAG(lanes 1 and
3) or BAP–FLAG (lane 2).
D, reciprocal co-IP was performed using GRP78 as
the bait. Mouse IgG was used in place of the membrane extract as a
negative control. E, endogenous co-IP was performed in
MERS-CoV- or mock-infected Huh7 and BEAS2B cells. Immunoprecipitation
was performed using the anti-GRP78 antibody, the anti-MERS-CoV spike
antibody, or the mouse isotype control. The precipitated protein
complexes were detected with the anti-MERS-CoV spike antibody or the
anti-GRP78 antibody. WB, Western blotting.To further verify the physical interaction between GRP78 and the MERS-CoV spike
in a physiological relevant scenario, we performed endogenous co-IP experiments
in MERS-CoV–infected Huh7 and BEAS2B cells (Fig. 2E). In line with our earlier findings, GRP78
efficiently immunoprecipitated the MERS-CoV spike from cell lysates of the
infected samples. In contrast, the MERS-CoV spike was not detected from the
mock-infected samples or from infected samples immunoprecipitated with a control
isotype antibody. The reciprocal co-IP performed using the MERS-CoV spike as the
bait similarly immunoprecipitated endogenous GRP78 from the infected samples but
not from mock-infected samples or from infected samples immunoprecipitated with
the control isotype antibody (Fig.
2E). Collectively, our co-IP data established GRP78
as a specific binding target of the MERS-CoV spike.GRP78 is abundantly expressed on the surface of human and animal
cellsGRP78 is a highly conserved protein that is traditionally described as an
ER-residing chaperone and plays key roles in facilitating protein folding and
assembly as well as the regulation of ER stress (28). In recent years, multiple functions of GRP78 on the cell
surface have been reported, including a critical role of cell-surface GRP78 on
virus entry (29–31). Because
our earlier data suggested that the MERS-CoV spike could interact with plasma
membrane GRP78, we hypothesized that GRP78 might be involved in modulating
MERS-CoV entry or attachment. To this end, we first analyzed GRP78 expression on
the cell surface of human lung cell lines that are susceptible to MERS-CoV
infection (32, 33). As illustrated in Fig.
3A, GRP78 was readily detected on the cell surface
of human lung cell lines, including A549, BEAS2B, and Calu3. In addition, GRP78
expression was also observed on the cell surface of a broad array of human cell
lines (AD293, Caco2, HeLa, and Huh7) and primary cells (monocyte-derived
macrophage (MDM), T cell) of extrapulmonary origin (Fig. 3B). Intriguingly, surface GRP78
expression was similarly detected in nonhuman cell lines, including BHK21, L929,
VeroE6, and RLK. Quantitative analysis of the expression rate (Fig. 3D) and mean
fluorescent intensity (MFI) (Fig.
3E) from the immunolabeled cells revealed that
surface DPP4 and GRP78 were expressed at comparative levels in most measured
cell lines with the exception of L929. The ubiquitous detection of GRP78 across
cell lines from different species by the human GRP78 antibody could be
attributed to the high degree of GRP78 sequence homology between mammalian
species, suggesting that the protein is well conserved in mammalian cells (Fig. 3F). Altogether, the
surface expression of GRP78 on MERS-CoV–susceptible cells supported the
notion that GRP78 might be involved in modulating MERS-CoV entry. However, the
ubiquitous expression of GRP78, particularly on cells that are not permissive to
MERS-CoV infection, including BHK21 and L929, suggested that GRP78 might play an
auxiliary rather than a determining role in MERS-CoV entry.Figure 3.GRP78 is abundantly expressed on the cell surface of mammalian
cells. Surface GRP78 expression was detected on mammalian
cell lines with flow cytometry with no cell permeabilization. The
immunostaining was performed for human lung cell lines
(A), human extrapulmonary cell lines, human primary
macrophages, and human primary T cells (B), as well as
nonhuman cell lines (C). D, percentage
of GRP78-positive cells quantified with DPP4 included for comparisons.
E, MFI of GRP78 on the cell surface was quantified
with isotype and DPP4 staining included as controls. F,
sequence homology between human GRP78 and GRP78 in other mammals. Gates
in A–C represented the percentage of
GRP78-positive cells. Data in D and E
represented mean and standard deviation from three independent
experiments.GRP78 is co-expressed with DPP4 in human pulmonary and extrapulmonary
tissuesIn order for GRP78 to modulate virus entry, it must be expressed by the
susceptible cells at the site of infection. To explore the potential
physiological relevance of GRP78 during MERS-CoV entry, we examined the
distribution of GRP78 in human lung tissues with confocal microscopy. Our
immunostaining results demonstrated that GRP78 was expressed at multiple regions
of the human lung tissues. In particular, specific GRP78 expression was
abundantly detected on the epithelial cells of the bronchus (Fig. 4A), bronchiole (Fig. 4B), and alveolus
(Fig. 4C). Most
importantly, double immunostaining of DPP4 and GRP78 revealed extensive
co-localization of DPP4 and GRP78 among the epithelial cells lining the human
airways (Fig. 4,
A–C). The co-localization between DPP4 and GRP78 on
the apical side of the epithelial cells indicated the potential of GRP78 in
facilitating MERS-CoV entry or attachment (Fig.
4D, arrows). Interestingly, the
co-expression of DPP4 and GRP78 could also be recognized in extrapulmonary
tissues, including the small intestine (Fig.
S3A) and the kidney (Fig.
S3B). Overall, our data demonstrated that
GRP78 was co-expressed with DPP4 on physiologically relevant cell types in the
human lung and could potentially be involved during MERS-CoV infection in the
lower respiratory tract.Figure 4.Co-expression of GRP78 and DPP4 in human tissues.
Immunostaining of GRP78 and DPP4 was performed on paraffin slides of
normal human tissues. GRP78 was labeled with a polyclonal rabbit
anti-GRP78 antibody, and DPP4 was labeled with a polyclonal goat
anti-DPP4 antibody. Cell nuclei were labeled with DAPI. The
co-expression of GRP78 and DPP4 was detected in the bronchus
(A), bronchiole (B), and alveolus
(C). The co-localization of GRP78 and DPP4 was
examined at a higher magnification in D. Images were
acquired with a Carl Zeiss LSM 710 system. Bars, 50
μm for A–C. Bars, 5 μm for
D.Antibody blocking or siRNA knockdown of GRP78 limits MERS-CoV entryTo investigate the functional role of cell-surface GRP78 during MERS-CoV
infection, we first evaluated the capacity of GRP78 antibody in blocking the
entry of MERS–S-pseudovirus. In this set of experiments, Huh7 and BEAS2B
cells were pre-incubated with a rabbit polyclonal antibody against GRP78 or a
nontargeting rabbit control IgG. After the pre-incubation,
MERS–S-pseudoviruses were added to the cells for 1 h in the presence of
the GRP78 antibody or the control IgG. At 72 h post-inoculation, the cells were
lysed and incubated with luciferase substrate for the quantification of
infectivity. Our results demonstrated that GRP78 antibody but not the control
IgG reduced MERS–S-pseudovirus entry in both Huh7 (Fig. 5A) and BEAS2B cells (Fig. 5B) in a dose-dependent
manner. In stark contrast, the entry of the control vesicular stomatitis virus
glycoprotein (VSV-G)-pseudovirus in both cell lines was not inhibited by GRP78
antibody (Fig. 5, A and
B). Next, we proceeded to validate the antibody blocking
results using infectious MERS-CoV. To this end, Huh7 cells were pre-incubated
with antibodies and subsequently infected with MERS-CoV in the presence of
control IgG, GRP78 antibody, or DPP4 antibody. Our data showed that the
treatment of GRP78 antibody similarly inhibited MERS-CoV entry in a
dose-dependent manner (Fig.
5C). For further verification, we infected Huh7 and
BEAS2B cells with MERS-CoV after siRNA knockdown of GRP78 or DPP4. Western
blotting detection demonstrated that GRP78 knockdown did not affect DPP4 or
CEACAM5 expression (Fig.
5D). In line with the antibody blocking results,
depletion of GRP78 reduced MERS-CoV entry in both Huh7 and BEAS2B cells (Fig. 5E). Because CEACAM5
was expressed in Huh7 but not BEAS2B cells, our data implied that the role of
GRP78 in modulating MERS-CoV entry was independent of CEACAM5 expression. To
further evaluate the role of GRP78 on MERS-CoV replication, we assessed virus
growth in MERS-CoV–infected BEAS2B cells after siRNA knockdown of GRP78
or DPP4. Our data demonstrated that GRP78 depletion decreased MERS-CoV
replication, although to a lesser extent compared with that of DPP4 knockdown
(Fig. 5, F and
G). Next, we asked whether GRP78 could play a role in
MERS-CoV entry in the physiologically relevant primary cells. To this end, we
performed siRNA knockdown of GRP78 (Fig.
5H) in primary human MDM and primary human embryonic
lung fibroblasts (HFL), which are both susceptible to MERS-CoV infection as
reported in our previous studies (32,
34). In agreement with our results
from Huh7 and BEAS2B cells, GRP78 knockdown significantly reduced virus entry
(Fig. 5I) and
replication (Fig. 5, J and
K) in MDM and HFL. Collectively, with antibody blocking and
siRNA knockdown, we demonstrated a significant role of GRP78 during MERS-CoV
entry.Figure 5.GRP78 is involved in MERS-CoV entry. Pseudovirus antibody
blocking assays were performed in Huh7 (A) and BEAS2B
(B) cells. A titration of GRP78 or isotype control
antibodies from 0 to 2.5 μg/ml was added and pre-incubated with
Huh7 and BEAS2B cells for 1 h at 37 °C. MERS–S-pseudovirus
or VSV-G–pseudovirus was subsequently added at a ratio of 100 LP
per cell for 1 h. Luciferase activity was determined at 72 h
post-inoculation and was normalized to that of the mock-treated cells.
C, antibody blocking assay was performed in Huh7
cells using infectious MERS-CoV. Huh7 cells were pre-incubated with
antibodies at the indicated concentration for 1 h at 37 °C. The
cells were then challenged with MERS-CoV at 1 m.o.i. for 1 h at 37
°C in the presence of the antibodies. After 1 h, the cells were
washed and harvested. MERS-CoV entry was assessed with qPCR, and the
result was normalized to that of the mock-treated cells.
D, Huh7 or BEAS2B cells were treated with 75
nm GRP78, DPP4, or scrambled siRNA for 2 consecutive days.
The knockdown efficiency was evaluated with Western blottings.
E, siRNA-treated Huh7 or BEAS2B cells were infected
with MERS-CoV at 1 m.o.i. for 1 h at 37 °C. After 1 h, the cells
were harvested, and virus entry was evaluated with qPCR analysis. The
result was normalized to that of the scrambled siRNA-treated cells.
siRNA-treated BEAS2B cells were infected with MERS-CoV at 0.1 m.o.i. for
1 h at 37 °C. The cell lysates (F) and
supernatants (G) were harvested at 24 and 48 h
post-infection. MERS-CoV replication was evaluated with qPCR analysis.
H, siRNA-treated MDM or HFL was infected with
MERS-CoV at 1 m.o.i. for 2 h at 37 °C. After 2 h, the cells were
harvested, and virus entry was evaluated with qPCR analysis
(I). The result was normalized to that of the
scrambled siRNA-treated cells. siRNA-treated MDM or HFL was infected
with MERS-CoV at 0.1 m.o.i. for 1 h at 37 °C. The cell lysates
(J) and supernatants (K) were
harvested at 24 h post-infection. MERS-CoV replication was evaluated
with qPCR analysis. In all panels, data represented mean and S.D. from
three independent experiments. Statistical analyses were carried out
using Student's t test. Statistical significance was
indicated by asterisks when p <
0.05. ns means not significant.GRP78 is an attachment factor of MERS-CoVOur earlier data supported the notion that cell-surface GRP78 was involved in
MERS-CoV entry. To define the functional role of GRP78 during this process, we
challenged AD293 or BHK21 cells with MERS-CoV after GRP78 overexpression. First,
we sought to evaluate the capacity of GRP78 in facilitating MERS-CoV attachment.
To this end, GRP78-transfected AD293 or BHK21 cells were challenged with
MERS-CoV at 4 °C for 2 h. After the incubation, the cells were washed,
fixed, and immunolabeled for MERS-CoV N. As illustrated in Fig. 6A, GRP78 overexpression significantly
increased virus attachment in both AD293 and BHK21 cells. Interestingly, GRP78
overexpression appeared to induce a more substantial increase in MERS-CoV
attachment in the MERS-CoV–nonsusceptible BHK21 cells than that in the
MERS-CoV–susceptible AD293 cells (Fig.
6B). Next, to address whether GRP78 could
independently facilitate MERS-CoV entry, we assessed the level of MERS-CoV entry
in AD293 and BHK21 cells upon GRP78 overexpression. To this end,
GRP78-transfected AD293 and BHK21 cells were challenged with MERS-CoV at 37
°C for 2 h. After infection, the cells were washed and incubated for
another 4 h before harvesting for flow cytometry. Importantly, our result
demonstrated that the nonpermissive BHK21 cells remained refractory to MERS-CoV
infection despite GRP78 overexpression. In contrast, GRP78 overexpression
further enhanced the entry of MERS-CoV to the permissive AD293 cells (Fig. 6, C and
D). The effect of GRP78 on MERS-CoV entry was not due to ER
stress (Fig.
S4). Overall, our data indicated that GRP78 could not facilitate
MERS-CoV entry independently but could serve as an attachment factor and
modulate MERS-CoV entry in the presence of DPP4.Figure 6.GRP78 is an attachment factor of MERS-CoV.
A, to assess the role of GRP78 on MERS-CoV attachment,
GRP78-overexpressing AD293 and BHK21 cells were challenged with MERS-CoV
at 15 m.o.i. for 2 h at 4 °C. After 2 h, the cells were washed,
detached with 10 mm EDTA on ice, and fixed in 4%
paraformaldehyde before immunolabeling for flow cytometry.
B, percentage of MERS-CoV N–positive AD293
and BHK21 cells was quantified for MERS-CoV attachment.
C, to assess the role of GRP78 on MERS-CoV entry,
GRP78-overexpressing AD293 and BHK21 cells were challenged with MERS-CoV
at 5 m.o.i. for 2 h at 37 °C. After 2 h, the inoculum was replaced
with culture media, and the cells were incubated for another 4 h before
harvesting for flow cytometry. D, percentage of
MERS-CoV N–positive AD293 and BHK21 cells was quantified for
MERS-CoV entry. B and D, percentage of
MERS-CoV N–positive cells among GRP78-transfected
(GRP78+) cells was calculated as
(%GRP78+N+
cells/(%GRP78+N+ cells +
%GRP78+N− cells)) × 100%. The
percentage of MERS-CoV N–positive cells among
GRP78-nontransfected (GRP78−) cells was calculated as
(%GRP78−N+
cells/(%GRP78−N+ cells +
%GRP78−N− cells)) × 100%.
Data represented mean and S.D. derived from three independent
experiments. Statistical analyses were carried out using Student's
t test. Statistical significance was indicated by
asterisks when p < 0.05.
ns means not significant.GRP78 is up-regulated on the surface of MERS-CoV–infected
cellsBecause infections by certain coronaviruses, including infectious bronchitis
virus and SARS-CoV, are known to induce ER stress (35–39), which can
promote GRP78 expression on the cell surface (40–43), we asked whether MERS-CoV infection
could up-regulate GRP78 expression on the cell surface. To address this
question, we infected Huh7 cells with MERS-CoV (Fig. 7, A and B) and harvested
samples for flow cytometry at 24 h post-infection. Our result demonstrated that
although the percentage of surface DPP4-positive cells modestly decreased after
MERS-CoV infection, the percentage of surface GRP78-positive cells significantly
increased from ∼50 to ∼80% after MERS-CoV infection (Fig. 7, C and
D). In this regard, our results highlighted the potential
relevance of GRP78 on MERS-CoV attachment onto the infected cells.Figure 7.GRP78 is up-regulated on the surface of MERS-CoV–infected
cells.
A, Huh7 cells were infected with MERS-CoV at 0.01 and
0.1 m.o.i. and were harvested for flow cytometry analysis at 24 h
post-infection. B, percentage of MERS-CoV
N–positive cells was quantified. C, in parallel,
cell surface and total DPP4 and GRP78 among mock- or
MERS-CoV–infected samples were analyzed by flow cytometry.
D, percentage of DPP4-positive cells and
GRP78-positive cells in mock- or MERS-CoV–infected samples were
quantified. Total DPP4 and GRP78 staining was performed by first
permeabilizing the cells with 0.1% Triton X-100, whereas the surface
DPP4 and GRP78 staining was performed in the absence of cell
permeabilization. The gate in A represented the
percentage of MERS-CoV N–positive cells. The gates in
C represented the percentage of DPP4-
(upper panels) and GRP78 (lower
panels)-positive cells. Data represented mean and S.D.
derived from three independent experiments. Statistical analyses were
carried out using Student's t test. Statistical
significance was indicated by asterisks when
p < 0.05. ns means not
significant.GRP78 facilitates the cell-surface attachment of bCoV-HKU9Coronaviruses have evolved a complicated receptor recognition system through
their spike proteins. Peculiarly, the spike proteins from highly-related
coronaviruses can recognize different cell-surface molecules, whereas the spike
proteins of phylogenetically distant coronaviruses can bind the same
cell-surface molecule for attachment or entry (11). By exploring the potential interaction between GRP78 and the
spike proteins of other coronaviruses, we unexpectedly discovered that GRP78
could interact with the spike protein of bat coronavirus HKU9 (bCoV-HKU9) (Fig. 8A). Interestingly,
despite the capacity of binding the spike proteins of lineage C (MERS-CoV) and
lineage D (bCoV-HKU9) Betacoronavirus, GRP78 did not interact
with the spike protein of SARS-CoV, which is a lineage B
Betacoronavirus (Fig.
8B). In 2007, we reported the first discovery and
genome characterization of bCoV-HKU9, which was identified from Leschenault's
rousette bats (R. leschenaulti) (44). Recently, with structural analysis and surface plasmon
resonance assay, it appeared that the receptor-binding domain of bCoV-HKU9 spike
was incapable of reacting with either human DPP4 or ACE2 (45). In this regard, it would be important to explore the
potential physiological relevance of the interaction between GRP78 and bCoV-HKU9
spike. We first evaluated the cell tropism of HKU9–S-pseudovirus with
MERS–S-pseudovirus included as a control. Remarkably, our data suggested
that among the 10 evaluated mammalian cell lines, HKU9–S-pseudovirus
entry was most pronounced in R. leschenaulti kidney (RLK) cells
(Fig. 8C). Notably,
although MERS–S-pseudovirus entry was evident in RLK cells, culture for
bCoV-HKU9 in RLK or other cell lines has not been successful (44). In line with the pseudovirus entry
result, the surface-binding efficiency of HKU9–S-pseudovirus on RLK cells
was ∼3-fold that on Caco2 cells (Fig.
8D), which is a human colon cell line known to be
permissive for both MERS-CoV and SARS-CoV infection. Notably, overexpression of
human GRP78 in the apparently nonpermissive L929 and BHK21 cells did not render
the cells permissive to HKU9–S-pseudovirus entry, indicating that GRP78
could not function as an independent receptor for bCoV-HKU9 (Fig. 8E). In contrast, with
a flow cytometry-based surface-binding assay, we demonstrated that the GRP78
antibody (Fig. 8G) but not
the control IgG (Fig. 8F)
reduced the binding of HKU9–S-pseudovirus to the cell surface of RLK
cells in a dose-dependent manner, which was evidenced by the drop in the
percentage of HKU9–S-positive cells (Fig.
8H) as well as the decrease in the
HKU9–S-mean fluorescent intensity (Fig.
8I). Taken together, our data identified GRP78 as an
important cell surface–binding protein for both MERS-CoV and bCoV-HKU9 by
serving as an attachment factor.Figure 8.GRP78 interacts with the bCoV-HKU9 spike and serves as an
attachment factor for bCoV-HKU9.
A, BHK21 cells were transfected with
pcDNA–GRP78–V5 (lanes 1 and
2) or empty vector (lane 3). Co-IP
between GRP78 and bCoV-HKU9 spike was performed using GRP78 as the bait
protein. Purified bCoV–HKU9–S1–FLAG (lanes
1 and 3) or BAP–FLAG proteins
(lane 2) were immunoprecipitated
(IP) with overexpressed GRP78–V5 or
pcDNA–V5 proteins pre-adsorbed on anti-V5–Sepharose beads.
The precipitated protein complex was detected using the anti-V5 antibody
or the anti-FLAG antibody. B, co-IP between GRP78 and
SARS-CoV spike was performed using GRP78 as the bait protein. Purified
SARS-CoV–S1–FLAG (lanes 1 and
3) or BAP–FLAG proteins (lane
2) were immunoprecipitated with overexpressed
GRP78–V5 or pcDNA–V5 proteins pre-adsorbed on
anti-V5–Sepharose beads. The precipitated protein complex was
detected using the anti-V5 antibody or the anti-FLAG antibody.
C, HKU9–S-pseudovirus entry assays were
performed in a number of mammalian cell lines. Mock-inoculated and
MERS–S-pseudovirus–inoculated cells were included as
negative and positive controls, respectively. HKU9–S-pseudovirus
and MERS–S-pseudovirus were added at a ratio of 100 LP per cell
for 1 h. Luciferase activity was determined at 72 h post-inoculation.
D, HKU9–S-pseudovirus attachment efficiency
was evaluated in Caco2 and RLK cells. HKU9–S-pseudovirus was
inoculated on Caco2 and RLK cells at 100 LP per cell for 2 h at 4
°C. After 2 h, the cells were washed, fixed, and immunolabeled for
flow cytometry. HKU9–S-pseudovirus binding was identified with an
in-house mouse bCoV-HKU9 spike immune serum. E,
HKU9–S-pseudovirus entry in L929 and BHK21 cells was assessed
with or without GRP78 overexpression. HKU9–S-pseudovirus was
inoculated at 100 LP per cell for 1 h at 37 °C. Luciferase activity
was determined at 72 h post-inoculation. F and
G, antibody-blocking assay for
HKU9–S-pseudovirus binding was performed in RLK cells. RLK cells
were pre-incubated with the rabbit anti-GRP78 antibody and the rabbit
control IgG from 0 to 5 μg/ml. After the pre-incubation,
HKU9–S-pseudovirus was inoculated to the cells at 100 LP per cell
for 2 h at 4 °C. The cells were then washed, fixed, and
immunolabeled for flow cytometry. HKU9–S-pseudovirus binding was
identified with an in-house mouse bCoV-HKU9 spike immune serum. The
percentage of bCoV-HKU9 spike-positive cells was quantified in
H, and the MFI of the bCoV-HKU9 spike on the cell
surface was quantified in I. Gates in D,
F, and G represented the percentage of
HKU9 spike-positive cells. Data represented mean and S.D. derived from
three independent experiments. Statistical analyses were carried out
using Student's t test. Statistical significance was
indicated by asterisks when p <
0.05. WB, Western blot. ns means not
significant.Sialic acids and GRP78 act independently to facilitate the surface attachment
of MERS-CoVSialic acids were recently identified as an attachment determinant of MERS-CoV
(46). To investigate whether GRP78
and sialic acids could act in conjunction with each other in facilitating the
attachment of MERS-CoV, we assessed MERS–S-pseudovirus entry in the
presence of a combination of neuraminidase treatment and GRP78 antibody
blocking. Our results demonstrated that although neuraminidase treatment
decreased MERS–S-pseudovirus entry in a dose-dependent manner, the
addition of GRP78 antibody further enhanced the inhibitory effect (Fig. 9A). However, the entry
of HKU9–S-pseudovirus was inhibited by GRP78 antibody but not
neuraminidase treatment (Fig.
9B). Overall, the additive effect of neuraminidase
treatment and GRP78 antibody on limiting MERS–S-pseudovirus entry
suggested that sialic acids and GRP78 both independently facilitated the
attachment of MERS-CoV onto the cell surface, whereas GRP78 but not sialic acids
played an important role for virus attachment of bCoV-HKU9.Figure 9.Sialic acids and GRP78 act independently to facilitate the surface
attachment of MERS-CoV.
A, Huh7 cells were treated with neuraminidase from
C. perfringens, with or without pre-incubation with
the GRP78 polyclonal antibody. The cells were subsequently challenged
with MERS–S-pseudovirus and assessed at 72 h post-infection for
pseudovirus entry. B, RLK cells were treated with
neuraminidase from Clostridium perfringens, with or
without pre-incubation with the GRP78 polyclonal antibody. The cells
were subsequently challenged with HKU9–S-pseudovirus and assessed
at 72 h post infection for pseudovirus entry. Pseudovirus entry was
quantified using a microplate reader as relative light units
(RLU). Data represented mean and standard deviation
derived from three independent experiments. Statistical analyses were
carried out using Student's t test. Statistical
significance was indicated by asterisk marks when
p < 0.05.

GRP78 interacts with MERS-CoV spikeWe previously identified human carcinoembryonic antigen-related cell adhesion
molecule 5 (CEACAM5) as an attachment factor for MERS-CoV (25). In this study, we asked whether additional membrane
proteins could interact with the MERS-CoV spike and facilitate the entry or
attachment of MERS-CoV. To this end, we transfected human bronchus epithelial
cells, BEAS2B, with the MERS-CoV spike and evaluated the membrane proteins that
might bind the MERS-CoV spike in the transfected cells. In brief, membrane
proteins from pcDNA–MERS-CoV–S1–V5-transfected BEAS2B cells
were extracted and sedimented (Fig. 1). To
evaluate the extraction efficiency, the cell extracts were probed for markers of
different cellular fractions, including that of the plasma membrane (epidermal
growth factor receptor (EGFR) and pan-cadherin), endoplasmic reticulum (ER)
(calreticulin), Golgi (giantin), and nucleus (lamin A). Western blotting
analyses revealed that our membrane extracts were enriched with the plasma
membrane markers, EGFR and pan-cadherin. In contrast, only a trace amount of the
ER marker was observed, whereas the signal for Golgi and nucleus was not
detected (Fig. 1D and
Fig.
S1).Figure 1.Identification of GRP78 as a target membrane protein of the
MERS-CoV spike.
A, silver staining of membrane proteins of BEAS2B cells
transfected with pcDNA–MERS-CoV–S1–V5. Membrane
extracts were immunoprecipitated (IP) with V5 antibody
and Sepharose A/G beads, followed by washing and eluting with glycine
(lane 1). Sepharose beads were boiled in sample
buffer after glycine elution (lane 2). Membrane
extracts were immunoprecipitated with mouse isotype control and
Sepharose A/G beads (lane 3). B,
expression of MERS-CoV–S1-V5 was detected by Western blotting
(WB) with an anti-ERS-CoV spike antibody.
C, silver staining of membrane proteins of BEAS2B
cells. The membrane extracts were immunoprecipitated with purified
recombinant MERS-CoV–S1–FLAG protein using anti-FLAG M2
antibody and Sepharose A/G beads, followed by washing and eluting with
3× FLAG peptides (lane 1). Sepharose beads were
boiled in sample buffer after 3× FLAG peptide elution (lane
2). Membrane extracts were immunoprecipitated with mouse
isotype control and Sepharose A/G beads (lane 3).
D, 5 μg of sedimented membrane extracts were
run on SDS-PAGE and subjected to Western blots using antibodies against
the plasma membrane marker (EGFR and pan-cadherin), endoplasmic
reticulum marker (calreticulin), Golgi marker (giantin), and nucleus
marker (lamin A). E, gel fragment indicated by the
red arrowhead in A and
C was excised for LC-MS/MS analysis. MS/MS data
were searched against all mammalian protein databases in NCBI and
Swiss-Prot. The protein was identified as GRP78 with significant hits
over different domains of the sequence.To identify potential proteins that could interact with the MERS-CoV spike, the
membrane extracts were immunoprecipitated with a V5 mAb and protein
A/G-Sepharose. The precipitated beads were then washed, and protein complexes
were eluted with 0.1 m glycine. Co-immunoprecipitated proteins were
revealed in SDS-PAGE after silver staining (Fig.
1A, lane 1). The eluted beads were
resuspended in sample loading buffer, boiled, and assessed for elution
efficiency (Fig. 1A,
lane 2). As a control, the same set of membrane extracts
was immunoprecipitated with isotype antibody and protein A/G-Sepharose (Fig. 1A, lane
3). In parallel, the expression of the MERS-CoV spike in the
immunoprecipitated complexes was validated with Western blotting using a mouse
immune serum against the MERS-CoV spike (Fig.
1B). Specific protein bands that were pulled down by
the V5 antibody but not the isotype control were excised and sent for MS
analysis. The MS/MS result revealed one of the dominant bands (Fig. 1A, lane 1,
arrowhead) to be 78-kDa glucose-regulated protein (GRP78), also
known as heat shock 70-kDa protein 5 (HSPA5) or binding immunoglobulin protein
(BiP) (Fig. 1E).To further verify the interaction between the MERS-CoV spike and GRP78, we
attempted to immunoprecipitate GRP78 with purified the MERS-CoV spike proteins.
To this end, recombinant MERS-CoV–S1–FLAG proteins were expressed,
purified, and immunoprecipitated against the membrane protein extracts from
BEAS2B cells. Notably, silver staining of the SDS-PAGE and the subsequent MS
confirmed the presence of GRP78 in the precipitated
MERS-CoV–S1–FLAG complex (Fig.
1C, lane 1, arrowhead) but not in
the control (Fig. 1C,
lane 3). Taken together, our membrane pulldown assay
identified GRP78 as a potential membrane protein specifically bound by the
MERS-CoV spike.

GRP78 interacts with MERS-CoV spike

We previously identified human carcinoembryonic antigen-related cell adhesion
molecule 5 (CEACAM5) as an attachment factor for MERS-CoV (25). In this study, we asked whether additional membrane
proteins could interact with the MERS-CoV spike and facilitate the entry or
attachment of MERS-CoV. To this end, we transfected human bronchus epithelial
cells, BEAS2B, with the MERS-CoV spike and evaluated the membrane proteins that
might bind the MERS-CoV spike in the transfected cells. In brief, membrane
proteins from pcDNA–MERS-CoV–S1–V5-transfected BEAS2B cells
were extracted and sedimented (Fig. 1). To
evaluate the extraction efficiency, the cell extracts were probed for markers of
different cellular fractions, including that of the plasma membrane (epidermal
growth factor receptor (EGFR) and pan-cadherin), endoplasmic reticulum (ER)
(calreticulin), Golgi (giantin), and nucleus (lamin A). Western blotting
analyses revealed that our membrane extracts were enriched with the plasma
membrane markers, EGFR and pan-cadherin. In contrast, only a trace amount of the
ER marker was observed, whereas the signal for Golgi and nucleus was not
detected (Fig. 1D and
Fig.
S1).

Identification of GRP78 as a target membrane protein of the
MERS-CoV spike.
A, silver staining of membrane proteins of BEAS2B cells
transfected with pcDNA–MERS-CoV–S1–V5. Membrane
extracts were immunoprecipitated (IP) with V5 antibody
and Sepharose A/G beads, followed by washing and eluting with glycine
(lane 1). Sepharose beads were boiled in sample
buffer after glycine elution (lane 2). Membrane
extracts were immunoprecipitated with mouse isotype control and
Sepharose A/G beads (lane 3). B,
expression of MERS-CoV–S1-V5 was detected by Western blotting
(WB) with an anti-ERS-CoV spike antibody.
C, silver staining of membrane proteins of BEAS2B
cells. The membrane extracts were immunoprecipitated with purified
recombinant MERS-CoV–S1–FLAG protein using anti-FLAG M2
antibody and Sepharose A/G beads, followed by washing and eluting with
3× FLAG peptides (lane 1). Sepharose beads were
boiled in sample buffer after 3× FLAG peptide elution (lane
2). Membrane extracts were immunoprecipitated with mouse
isotype control and Sepharose A/G beads (lane 3).
D, 5 μg of sedimented membrane extracts were
run on SDS-PAGE and subjected to Western blots using antibodies against
the plasma membrane marker (EGFR and pan-cadherin), endoplasmic
reticulum marker (calreticulin), Golgi marker (giantin), and nucleus
marker (lamin A). E, gel fragment indicated by the
red arrowhead in A and
C was excised for LC-MS/MS analysis. MS/MS data
were searched against all mammalian protein databases in NCBI and
Swiss-Prot. The protein was identified as GRP78 with significant hits
over different domains of the sequence.

To identify potential proteins that could interact with the MERS-CoV spike, the
membrane extracts were immunoprecipitated with a V5 mAb and protein
A/G-Sepharose. The precipitated beads were then washed, and protein complexes
were eluted with 0.1 m glycine. Co-immunoprecipitated proteins were
revealed in SDS-PAGE after silver staining (Fig.
1A, lane 1). The eluted beads were
resuspended in sample loading buffer, boiled, and assessed for elution
efficiency (Fig. 1A,
lane 2). As a control, the same set of membrane extracts
was immunoprecipitated with isotype antibody and protein A/G-Sepharose (Fig. 1A, lane
3). In parallel, the expression of the MERS-CoV spike in the
immunoprecipitated complexes was validated with Western blotting using a mouse
immune serum against the MERS-CoV spike (Fig.
1B). Specific protein bands that were pulled down by
the V5 antibody but not the isotype control were excised and sent for MS
analysis. The MS/MS result revealed one of the dominant bands (Fig. 1A, lane 1,
arrowhead) to be 78-kDa glucose-regulated protein (GRP78), also
known as heat shock 70-kDa protein 5 (HSPA5) or binding immunoglobulin protein
(BiP) (Fig. 1E).

To further verify the interaction between the MERS-CoV spike and GRP78, we
attempted to immunoprecipitate GRP78 with purified the MERS-CoV spike proteins.
To this end, recombinant MERS-CoV–S1–FLAG proteins were expressed,
purified, and immunoprecipitated against the membrane protein extracts from
BEAS2B cells. Notably, silver staining of the SDS-PAGE and the subsequent MS
confirmed the presence of GRP78 in the precipitated
MERS-CoV–S1–FLAG complex (Fig.
1C, lane 1, arrowhead) but not in
the control (Fig. 1C,
lane 3). Taken together, our membrane pulldown assay
identified GRP78 as a potential membrane protein specifically bound by the
MERS-CoV spike.

GRP78 is a specific binding target of MERS-CoV spikeNext, to examine the direct interaction between GRP78 and the MERS-CoV spike, we
performed a series of co-immunoprecipitation (co-IP) assays in both
overexpression and endogenous settings. First, BHK21 cells were transfected with
GRP78–V5 or the pcDNA–V5 control vector. The cell lysates of the
transfected cells were then immunoprecipitated with either
MERS-CoV–S1–FLAG or Escherichia coli bacterial
alkaline phosphatase–FLAG (BAP–FLAG) pre-adsorbed on anti-FLAG
M2-agarose beads. The precipitated protein complexes were then detected by
Western blotting with the anti-FLAG or the anti-V5 antibody. As illustrated in
Fig. 2A, GRP78
specifically immunoprecipitated with MERS-CoV–S1 (lower panel,
lane 1) but not the control bait protein, BAP (lower panel,
lane 2). Additionally, GRP78 was not precipitated in cells
transfected with the empty vector (Fig.
2A, lower panel, lane 3). To
confirm the interaction between GRP78 and MERS-CoV–S1, we performed
reciprocal co-IP using GRP78 as the bait protein (Fig. 2B). In this setting, cell lysates of
GRP78–V5 or empty vector transfected BHK21 cells were immunoprecipitated
with anti-V5 pre-adsorbed protein A/G-Sepharose and incubated with purified
MERS-CoV–S1–FLAG or BAP–FLAG. Our result demonstrated that
MERS-CoV–S1–FLAG but not BAP–FLAG was efficiently
immunoprecipitated by GRP78–V5 (Fig.
2B, upper panel, lanes 1 and
2). As a negative control, the expression of
pcDNA–V5 empty vector failed to immunoprecipitate with
MERS-CoV–S1–FLAG (Fig.
2B, upper panel, lane 3). In
parallel, MERS-CoV–S1–FLAG did not co-IP with the abundantly
expressed cell-surface protein EGFR, suggesting the interaction between
MERS-CoV–S1–FLAG and GRP78 was specific (Fig. S2,
A and B). Next, we
evaluated whether the interaction between the MERS-CoV spike and GRP78 could
occur at the cell surface. To this end, we obtained the membrane fraction of
Huh7 cells that was predominantly enriched with the plasma membrane contents of
the cells. We then added MERS-CoV–S1–FLAG protein to the membrane
extracts and performed co-IP between the MERS-CoV spike and GRP78. Our data
showed that the MERS-CoV spike and the endogenous GRP78 in the membrane extract
could efficiently interact with each other (Fig.
2, C and D).Figure 2.GRP78 interacts with the MERS-CoV spike.
A, BHK21 cells were transfected with
pcDNA–GRP78–V5 (lanes 1 and
2) or empty vector (lane 3). The
cell lysate was immunoprecipitated (IP) with either
purified recombinant MERS-CoV–S1–FLAG protein
(lanes 1 and 3) or E.
coli bacterial alkaline phosphatase
(BAP)-FLAG protein (lane 2)
pre-adsorbed onto anti-FLAG M2-agarose beads. The precipitated protein
complex was detected using the anti-FLAG antibody or the anti-V5
antibody. B, reciprocal co-IP was performed using GRP78
as the bait protein. Purified MERS-CoV–S1–FLAG
(lanes 1 and 3) or BAP–FLAG
proteins (lane 2) were immunoprecipitated with
overexpressed GRP78–V5 or pcDNA–V5 proteins pre-adsorbed
on anti-V5 Sepharose beads. The precipitated protein complex was
detected using the anti-FLAG antibody or the anti-GRP78 antibody.
C, membrane fraction of Huh7 cells was extracted
and immunoprecipitated with either
MERS-CoV–S1–FLAG(lanes 1 and
3) or BAP–FLAG (lane 2).
D, reciprocal co-IP was performed using GRP78 as
the bait. Mouse IgG was used in place of the membrane extract as a
negative control. E, endogenous co-IP was performed in
MERS-CoV- or mock-infected Huh7 and BEAS2B cells. Immunoprecipitation
was performed using the anti-GRP78 antibody, the anti-MERS-CoV spike
antibody, or the mouse isotype control. The precipitated protein
complexes were detected with the anti-MERS-CoV spike antibody or the
anti-GRP78 antibody. WB, Western blotting.To further verify the physical interaction between GRP78 and the MERS-CoV spike
in a physiological relevant scenario, we performed endogenous co-IP experiments
in MERS-CoV–infected Huh7 and BEAS2B cells (Fig. 2E). In line with our earlier findings, GRP78
efficiently immunoprecipitated the MERS-CoV spike from cell lysates of the
infected samples. In contrast, the MERS-CoV spike was not detected from the
mock-infected samples or from infected samples immunoprecipitated with a control
isotype antibody. The reciprocal co-IP performed using the MERS-CoV spike as the
bait similarly immunoprecipitated endogenous GRP78 from the infected samples but
not from mock-infected samples or from infected samples immunoprecipitated with
the control isotype antibody (Fig.
2E). Collectively, our co-IP data established GRP78
as a specific binding target of the MERS-CoV spike.

GRP78 is a specific binding target of MERS-CoV spike

Next, to examine the direct interaction between GRP78 and the MERS-CoV spike, we
performed a series of co-immunoprecipitation (co-IP) assays in both
overexpression and endogenous settings. First, BHK21 cells were transfected with
GRP78–V5 or the pcDNA–V5 control vector. The cell lysates of the
transfected cells were then immunoprecipitated with either
MERS-CoV–S1–FLAG or Escherichia coli bacterial
alkaline phosphatase–FLAG (BAP–FLAG) pre-adsorbed on anti-FLAG
M2-agarose beads. The precipitated protein complexes were then detected by
Western blotting with the anti-FLAG or the anti-V5 antibody. As illustrated in
Fig. 2A, GRP78
specifically immunoprecipitated with MERS-CoV–S1 (lower panel,
lane 1) but not the control bait protein, BAP (lower panel,
lane 2). Additionally, GRP78 was not precipitated in cells
transfected with the empty vector (Fig.
2A, lower panel, lane 3). To
confirm the interaction between GRP78 and MERS-CoV–S1, we performed
reciprocal co-IP using GRP78 as the bait protein (Fig. 2B). In this setting, cell lysates of
GRP78–V5 or empty vector transfected BHK21 cells were immunoprecipitated
with anti-V5 pre-adsorbed protein A/G-Sepharose and incubated with purified
MERS-CoV–S1–FLAG or BAP–FLAG. Our result demonstrated that
MERS-CoV–S1–FLAG but not BAP–FLAG was efficiently
immunoprecipitated by GRP78–V5 (Fig.
2B, upper panel, lanes 1 and
2). As a negative control, the expression of
pcDNA–V5 empty vector failed to immunoprecipitate with
MERS-CoV–S1–FLAG (Fig.
2B, upper panel, lane 3). In
parallel, MERS-CoV–S1–FLAG did not co-IP with the abundantly
expressed cell-surface protein EGFR, suggesting the interaction between
MERS-CoV–S1–FLAG and GRP78 was specific (Fig. S2,
A and B). Next, we
evaluated whether the interaction between the MERS-CoV spike and GRP78 could
occur at the cell surface. To this end, we obtained the membrane fraction of
Huh7 cells that was predominantly enriched with the plasma membrane contents of
the cells. We then added MERS-CoV–S1–FLAG protein to the membrane
extracts and performed co-IP between the MERS-CoV spike and GRP78. Our data
showed that the MERS-CoV spike and the endogenous GRP78 in the membrane extract
could efficiently interact with each other (Fig.
2, C and D).

GRP78 interacts with the MERS-CoV spike.
A, BHK21 cells were transfected with
pcDNA–GRP78–V5 (lanes 1 and
2) or empty vector (lane 3). The
cell lysate was immunoprecipitated (IP) with either
purified recombinant MERS-CoV–S1–FLAG protein
(lanes 1 and 3) or E.
coli bacterial alkaline phosphatase
(BAP)-FLAG protein (lane 2)
pre-adsorbed onto anti-FLAG M2-agarose beads. The precipitated protein
complex was detected using the anti-FLAG antibody or the anti-V5
antibody. B, reciprocal co-IP was performed using GRP78
as the bait protein. Purified MERS-CoV–S1–FLAG
(lanes 1 and 3) or BAP–FLAG
proteins (lane 2) were immunoprecipitated with
overexpressed GRP78–V5 or pcDNA–V5 proteins pre-adsorbed
on anti-V5 Sepharose beads. The precipitated protein complex was
detected using the anti-FLAG antibody or the anti-GRP78 antibody.
C, membrane fraction of Huh7 cells was extracted
and immunoprecipitated with either
MERS-CoV–S1–FLAG(lanes 1 and
3) or BAP–FLAG (lane 2).
D, reciprocal co-IP was performed using GRP78 as
the bait. Mouse IgG was used in place of the membrane extract as a
negative control. E, endogenous co-IP was performed in
MERS-CoV- or mock-infected Huh7 and BEAS2B cells. Immunoprecipitation
was performed using the anti-GRP78 antibody, the anti-MERS-CoV spike
antibody, or the mouse isotype control. The precipitated protein
complexes were detected with the anti-MERS-CoV spike antibody or the
anti-GRP78 antibody. WB, Western blotting.

To further verify the physical interaction between GRP78 and the MERS-CoV spike
in a physiological relevant scenario, we performed endogenous co-IP experiments
in MERS-CoV–infected Huh7 and BEAS2B cells (Fig. 2E). In line with our earlier findings, GRP78
efficiently immunoprecipitated the MERS-CoV spike from cell lysates of the
infected samples. In contrast, the MERS-CoV spike was not detected from the
mock-infected samples or from infected samples immunoprecipitated with a control
isotype antibody. The reciprocal co-IP performed using the MERS-CoV spike as the
bait similarly immunoprecipitated endogenous GRP78 from the infected samples but
not from mock-infected samples or from infected samples immunoprecipitated with
the control isotype antibody (Fig.
2E). Collectively, our co-IP data established GRP78
as a specific binding target of the MERS-CoV spike.

GRP78 is abundantly expressed on the surface of human and animal
cellsGRP78 is a highly conserved protein that is traditionally described as an
ER-residing chaperone and plays key roles in facilitating protein folding and
assembly as well as the regulation of ER stress (28). In recent years, multiple functions of GRP78 on the cell
surface have been reported, including a critical role of cell-surface GRP78 on
virus entry (29–31). Because
our earlier data suggested that the MERS-CoV spike could interact with plasma
membrane GRP78, we hypothesized that GRP78 might be involved in modulating
MERS-CoV entry or attachment. To this end, we first analyzed GRP78 expression on
the cell surface of human lung cell lines that are susceptible to MERS-CoV
infection (32, 33). As illustrated in Fig.
3A, GRP78 was readily detected on the cell surface
of human lung cell lines, including A549, BEAS2B, and Calu3. In addition, GRP78
expression was also observed on the cell surface of a broad array of human cell
lines (AD293, Caco2, HeLa, and Huh7) and primary cells (monocyte-derived
macrophage (MDM), T cell) of extrapulmonary origin (Fig. 3B). Intriguingly, surface GRP78
expression was similarly detected in nonhuman cell lines, including BHK21, L929,
VeroE6, and RLK. Quantitative analysis of the expression rate (Fig. 3D) and mean
fluorescent intensity (MFI) (Fig.
3E) from the immunolabeled cells revealed that
surface DPP4 and GRP78 were expressed at comparative levels in most measured
cell lines with the exception of L929. The ubiquitous detection of GRP78 across
cell lines from different species by the human GRP78 antibody could be
attributed to the high degree of GRP78 sequence homology between mammalian
species, suggesting that the protein is well conserved in mammalian cells (Fig. 3F). Altogether, the
surface expression of GRP78 on MERS-CoV–susceptible cells supported the
notion that GRP78 might be involved in modulating MERS-CoV entry. However, the
ubiquitous expression of GRP78, particularly on cells that are not permissive to
MERS-CoV infection, including BHK21 and L929, suggested that GRP78 might play an
auxiliary rather than a determining role in MERS-CoV entry.Figure 3.GRP78 is abundantly expressed on the cell surface of mammalian
cells. Surface GRP78 expression was detected on mammalian
cell lines with flow cytometry with no cell permeabilization. The
immunostaining was performed for human lung cell lines
(A), human extrapulmonary cell lines, human primary
macrophages, and human primary T cells (B), as well as
nonhuman cell lines (C). D, percentage
of GRP78-positive cells quantified with DPP4 included for comparisons.
E, MFI of GRP78 on the cell surface was quantified
with isotype and DPP4 staining included as controls. F,
sequence homology between human GRP78 and GRP78 in other mammals. Gates
in A–C represented the percentage of
GRP78-positive cells. Data in D and E
represented mean and standard deviation from three independent
experiments.

GRP78 is abundantly expressed on the surface of human and animal
cells

GRP78 is a highly conserved protein that is traditionally described as an
ER-residing chaperone and plays key roles in facilitating protein folding and
assembly as well as the regulation of ER stress (28). In recent years, multiple functions of GRP78 on the cell
surface have been reported, including a critical role of cell-surface GRP78 on
virus entry (29–31). Because
our earlier data suggested that the MERS-CoV spike could interact with plasma
membrane GRP78, we hypothesized that GRP78 might be involved in modulating
MERS-CoV entry or attachment. To this end, we first analyzed GRP78 expression on
the cell surface of human lung cell lines that are susceptible to MERS-CoV
infection (32, 33). As illustrated in Fig.
3A, GRP78 was readily detected on the cell surface
of human lung cell lines, including A549, BEAS2B, and Calu3. In addition, GRP78
expression was also observed on the cell surface of a broad array of human cell
lines (AD293, Caco2, HeLa, and Huh7) and primary cells (monocyte-derived
macrophage (MDM), T cell) of extrapulmonary origin (Fig. 3B). Intriguingly, surface GRP78
expression was similarly detected in nonhuman cell lines, including BHK21, L929,
VeroE6, and RLK. Quantitative analysis of the expression rate (Fig. 3D) and mean
fluorescent intensity (MFI) (Fig.
3E) from the immunolabeled cells revealed that
surface DPP4 and GRP78 were expressed at comparative levels in most measured
cell lines with the exception of L929. The ubiquitous detection of GRP78 across
cell lines from different species by the human GRP78 antibody could be
attributed to the high degree of GRP78 sequence homology between mammalian
species, suggesting that the protein is well conserved in mammalian cells (Fig. 3F). Altogether, the
surface expression of GRP78 on MERS-CoV–susceptible cells supported the
notion that GRP78 might be involved in modulating MERS-CoV entry. However, the
ubiquitous expression of GRP78, particularly on cells that are not permissive to
MERS-CoV infection, including BHK21 and L929, suggested that GRP78 might play an
auxiliary rather than a determining role in MERS-CoV entry.

GRP78 is abundantly expressed on the cell surface of mammalian
cells. Surface GRP78 expression was detected on mammalian
cell lines with flow cytometry with no cell permeabilization. The
immunostaining was performed for human lung cell lines
(A), human extrapulmonary cell lines, human primary
macrophages, and human primary T cells (B), as well as
nonhuman cell lines (C). D, percentage
of GRP78-positive cells quantified with DPP4 included for comparisons.
E, MFI of GRP78 on the cell surface was quantified
with isotype and DPP4 staining included as controls. F,
sequence homology between human GRP78 and GRP78 in other mammals. Gates
in A–C represented the percentage of
GRP78-positive cells. Data in D and E
represented mean and standard deviation from three independent
experiments.

GRP78 is co-expressed with DPP4 in human pulmonary and extrapulmonary
tissuesIn order for GRP78 to modulate virus entry, it must be expressed by the
susceptible cells at the site of infection. To explore the potential
physiological relevance of GRP78 during MERS-CoV entry, we examined the
distribution of GRP78 in human lung tissues with confocal microscopy. Our
immunostaining results demonstrated that GRP78 was expressed at multiple regions
of the human lung tissues. In particular, specific GRP78 expression was
abundantly detected on the epithelial cells of the bronchus (Fig. 4A), bronchiole (Fig. 4B), and alveolus
(Fig. 4C). Most
importantly, double immunostaining of DPP4 and GRP78 revealed extensive
co-localization of DPP4 and GRP78 among the epithelial cells lining the human
airways (Fig. 4,
A–C). The co-localization between DPP4 and GRP78 on
the apical side of the epithelial cells indicated the potential of GRP78 in
facilitating MERS-CoV entry or attachment (Fig.
4D, arrows). Interestingly, the
co-expression of DPP4 and GRP78 could also be recognized in extrapulmonary
tissues, including the small intestine (Fig.
S3A) and the kidney (Fig.
S3B). Overall, our data demonstrated that
GRP78 was co-expressed with DPP4 on physiologically relevant cell types in the
human lung and could potentially be involved during MERS-CoV infection in the
lower respiratory tract.Figure 4.Co-expression of GRP78 and DPP4 in human tissues.
Immunostaining of GRP78 and DPP4 was performed on paraffin slides of
normal human tissues. GRP78 was labeled with a polyclonal rabbit
anti-GRP78 antibody, and DPP4 was labeled with a polyclonal goat
anti-DPP4 antibody. Cell nuclei were labeled with DAPI. The
co-expression of GRP78 and DPP4 was detected in the bronchus
(A), bronchiole (B), and alveolus
(C). The co-localization of GRP78 and DPP4 was
examined at a higher magnification in D. Images were
acquired with a Carl Zeiss LSM 710 system. Bars, 50
μm for A–C. Bars, 5 μm for
D.

GRP78 is co-expressed with DPP4 in human pulmonary and extrapulmonary
tissues

In order for GRP78 to modulate virus entry, it must be expressed by the
susceptible cells at the site of infection. To explore the potential
physiological relevance of GRP78 during MERS-CoV entry, we examined the
distribution of GRP78 in human lung tissues with confocal microscopy. Our
immunostaining results demonstrated that GRP78 was expressed at multiple regions
of the human lung tissues. In particular, specific GRP78 expression was
abundantly detected on the epithelial cells of the bronchus (Fig. 4A), bronchiole (Fig. 4B), and alveolus
(Fig. 4C). Most
importantly, double immunostaining of DPP4 and GRP78 revealed extensive
co-localization of DPP4 and GRP78 among the epithelial cells lining the human
airways (Fig. 4,
A–C). The co-localization between DPP4 and GRP78 on
the apical side of the epithelial cells indicated the potential of GRP78 in
facilitating MERS-CoV entry or attachment (Fig.
4D, arrows). Interestingly, the
co-expression of DPP4 and GRP78 could also be recognized in extrapulmonary
tissues, including the small intestine (Fig.
S3A) and the kidney (Fig.
S3B). Overall, our data demonstrated that
GRP78 was co-expressed with DPP4 on physiologically relevant cell types in the
human lung and could potentially be involved during MERS-CoV infection in the
lower respiratory tract.

Co-expression of GRP78 and DPP4 in human tissues.
Immunostaining of GRP78 and DPP4 was performed on paraffin slides of
normal human tissues. GRP78 was labeled with a polyclonal rabbit
anti-GRP78 antibody, and DPP4 was labeled with a polyclonal goat
anti-DPP4 antibody. Cell nuclei were labeled with DAPI. The
co-expression of GRP78 and DPP4 was detected in the bronchus
(A), bronchiole (B), and alveolus
(C). The co-localization of GRP78 and DPP4 was
examined at a higher magnification in D. Images were
acquired with a Carl Zeiss LSM 710 system. Bars, 50
μm for A–C. Bars, 5 μm for
D.

Antibody blocking or siRNA knockdown of GRP78 limits MERS-CoV entryTo investigate the functional role of cell-surface GRP78 during MERS-CoV
infection, we first evaluated the capacity of GRP78 antibody in blocking the
entry of MERS–S-pseudovirus. In this set of experiments, Huh7 and BEAS2B
cells were pre-incubated with a rabbit polyclonal antibody against GRP78 or a
nontargeting rabbit control IgG. After the pre-incubation,
MERS–S-pseudoviruses were added to the cells for 1 h in the presence of
the GRP78 antibody or the control IgG. At 72 h post-inoculation, the cells were
lysed and incubated with luciferase substrate for the quantification of
infectivity. Our results demonstrated that GRP78 antibody but not the control
IgG reduced MERS–S-pseudovirus entry in both Huh7 (Fig. 5A) and BEAS2B cells (Fig. 5B) in a dose-dependent
manner. In stark contrast, the entry of the control vesicular stomatitis virus
glycoprotein (VSV-G)-pseudovirus in both cell lines was not inhibited by GRP78
antibody (Fig. 5, A and
B). Next, we proceeded to validate the antibody blocking
results using infectious MERS-CoV. To this end, Huh7 cells were pre-incubated
with antibodies and subsequently infected with MERS-CoV in the presence of
control IgG, GRP78 antibody, or DPP4 antibody. Our data showed that the
treatment of GRP78 antibody similarly inhibited MERS-CoV entry in a
dose-dependent manner (Fig.
5C). For further verification, we infected Huh7 and
BEAS2B cells with MERS-CoV after siRNA knockdown of GRP78 or DPP4. Western
blotting detection demonstrated that GRP78 knockdown did not affect DPP4 or
CEACAM5 expression (Fig.
5D). In line with the antibody blocking results,
depletion of GRP78 reduced MERS-CoV entry in both Huh7 and BEAS2B cells (Fig. 5E). Because CEACAM5
was expressed in Huh7 but not BEAS2B cells, our data implied that the role of
GRP78 in modulating MERS-CoV entry was independent of CEACAM5 expression. To
further evaluate the role of GRP78 on MERS-CoV replication, we assessed virus
growth in MERS-CoV–infected BEAS2B cells after siRNA knockdown of GRP78
or DPP4. Our data demonstrated that GRP78 depletion decreased MERS-CoV
replication, although to a lesser extent compared with that of DPP4 knockdown
(Fig. 5, F and
G). Next, we asked whether GRP78 could play a role in
MERS-CoV entry in the physiologically relevant primary cells. To this end, we
performed siRNA knockdown of GRP78 (Fig.
5H) in primary human MDM and primary human embryonic
lung fibroblasts (HFL), which are both susceptible to MERS-CoV infection as
reported in our previous studies (32,
34). In agreement with our results
from Huh7 and BEAS2B cells, GRP78 knockdown significantly reduced virus entry
(Fig. 5I) and
replication (Fig. 5, J and
K) in MDM and HFL. Collectively, with antibody blocking and
siRNA knockdown, we demonstrated a significant role of GRP78 during MERS-CoV
entry.Figure 5.GRP78 is involved in MERS-CoV entry. Pseudovirus antibody
blocking assays were performed in Huh7 (A) and BEAS2B
(B) cells. A titration of GRP78 or isotype control
antibodies from 0 to 2.5 μg/ml was added and pre-incubated with
Huh7 and BEAS2B cells for 1 h at 37 °C. MERS–S-pseudovirus
or VSV-G–pseudovirus was subsequently added at a ratio of 100 LP
per cell for 1 h. Luciferase activity was determined at 72 h
post-inoculation and was normalized to that of the mock-treated cells.
C, antibody blocking assay was performed in Huh7
cells using infectious MERS-CoV. Huh7 cells were pre-incubated with
antibodies at the indicated concentration for 1 h at 37 °C. The
cells were then challenged with MERS-CoV at 1 m.o.i. for 1 h at 37
°C in the presence of the antibodies. After 1 h, the cells were
washed and harvested. MERS-CoV entry was assessed with qPCR, and the
result was normalized to that of the mock-treated cells.
D, Huh7 or BEAS2B cells were treated with 75
nm GRP78, DPP4, or scrambled siRNA for 2 consecutive days.
The knockdown efficiency was evaluated with Western blottings.
E, siRNA-treated Huh7 or BEAS2B cells were infected
with MERS-CoV at 1 m.o.i. for 1 h at 37 °C. After 1 h, the cells
were harvested, and virus entry was evaluated with qPCR analysis. The
result was normalized to that of the scrambled siRNA-treated cells.
siRNA-treated BEAS2B cells were infected with MERS-CoV at 0.1 m.o.i. for
1 h at 37 °C. The cell lysates (F) and
supernatants (G) were harvested at 24 and 48 h
post-infection. MERS-CoV replication was evaluated with qPCR analysis.
H, siRNA-treated MDM or HFL was infected with
MERS-CoV at 1 m.o.i. for 2 h at 37 °C. After 2 h, the cells were
harvested, and virus entry was evaluated with qPCR analysis
(I). The result was normalized to that of the
scrambled siRNA-treated cells. siRNA-treated MDM or HFL was infected
with MERS-CoV at 0.1 m.o.i. for 1 h at 37 °C. The cell lysates
(J) and supernatants (K) were
harvested at 24 h post-infection. MERS-CoV replication was evaluated
with qPCR analysis. In all panels, data represented mean and S.D. from
three independent experiments. Statistical analyses were carried out
using Student's t test. Statistical significance was
indicated by asterisks when p <
0.05. ns means not significant.

Antibody blocking or siRNA knockdown of GRP78 limits MERS-CoV entry

To investigate the functional role of cell-surface GRP78 during MERS-CoV
infection, we first evaluated the capacity of GRP78 antibody in blocking the
entry of MERS–S-pseudovirus. In this set of experiments, Huh7 and BEAS2B
cells were pre-incubated with a rabbit polyclonal antibody against GRP78 or a
nontargeting rabbit control IgG. After the pre-incubation,
MERS–S-pseudoviruses were added to the cells for 1 h in the presence of
the GRP78 antibody or the control IgG. At 72 h post-inoculation, the cells were
lysed and incubated with luciferase substrate for the quantification of
infectivity. Our results demonstrated that GRP78 antibody but not the control
IgG reduced MERS–S-pseudovirus entry in both Huh7 (Fig. 5A) and BEAS2B cells (Fig. 5B) in a dose-dependent
manner. In stark contrast, the entry of the control vesicular stomatitis virus
glycoprotein (VSV-G)-pseudovirus in both cell lines was not inhibited by GRP78
antibody (Fig. 5, A and
B). Next, we proceeded to validate the antibody blocking
results using infectious MERS-CoV. To this end, Huh7 cells were pre-incubated
with antibodies and subsequently infected with MERS-CoV in the presence of
control IgG, GRP78 antibody, or DPP4 antibody. Our data showed that the
treatment of GRP78 antibody similarly inhibited MERS-CoV entry in a
dose-dependent manner (Fig.
5C). For further verification, we infected Huh7 and
BEAS2B cells with MERS-CoV after siRNA knockdown of GRP78 or DPP4. Western
blotting detection demonstrated that GRP78 knockdown did not affect DPP4 or
CEACAM5 expression (Fig.
5D). In line with the antibody blocking results,
depletion of GRP78 reduced MERS-CoV entry in both Huh7 and BEAS2B cells (Fig. 5E). Because CEACAM5
was expressed in Huh7 but not BEAS2B cells, our data implied that the role of
GRP78 in modulating MERS-CoV entry was independent of CEACAM5 expression. To
further evaluate the role of GRP78 on MERS-CoV replication, we assessed virus
growth in MERS-CoV–infected BEAS2B cells after siRNA knockdown of GRP78
or DPP4. Our data demonstrated that GRP78 depletion decreased MERS-CoV
replication, although to a lesser extent compared with that of DPP4 knockdown
(Fig. 5, F and
G). Next, we asked whether GRP78 could play a role in
MERS-CoV entry in the physiologically relevant primary cells. To this end, we
performed siRNA knockdown of GRP78 (Fig.
5H) in primary human MDM and primary human embryonic
lung fibroblasts (HFL), which are both susceptible to MERS-CoV infection as
reported in our previous studies (32,
34). In agreement with our results
from Huh7 and BEAS2B cells, GRP78 knockdown significantly reduced virus entry
(Fig. 5I) and
replication (Fig. 5, J and
K) in MDM and HFL. Collectively, with antibody blocking and
siRNA knockdown, we demonstrated a significant role of GRP78 during MERS-CoV
entry.

GRP78 is involved in MERS-CoV entry. Pseudovirus antibody
blocking assays were performed in Huh7 (A) and BEAS2B
(B) cells. A titration of GRP78 or isotype control
antibodies from 0 to 2.5 μg/ml was added and pre-incubated with
Huh7 and BEAS2B cells for 1 h at 37 °C. MERS–S-pseudovirus
or VSV-G–pseudovirus was subsequently added at a ratio of 100 LP
per cell for 1 h. Luciferase activity was determined at 72 h
post-inoculation and was normalized to that of the mock-treated cells.
C, antibody blocking assay was performed in Huh7
cells using infectious MERS-CoV. Huh7 cells were pre-incubated with
antibodies at the indicated concentration for 1 h at 37 °C. The
cells were then challenged with MERS-CoV at 1 m.o.i. for 1 h at 37
°C in the presence of the antibodies. After 1 h, the cells were
washed and harvested. MERS-CoV entry was assessed with qPCR, and the
result was normalized to that of the mock-treated cells.
D, Huh7 or BEAS2B cells were treated with 75
nm GRP78, DPP4, or scrambled siRNA for 2 consecutive days.
The knockdown efficiency was evaluated with Western blottings.
E, siRNA-treated Huh7 or BEAS2B cells were infected
with MERS-CoV at 1 m.o.i. for 1 h at 37 °C. After 1 h, the cells
were harvested, and virus entry was evaluated with qPCR analysis. The
result was normalized to that of the scrambled siRNA-treated cells.
siRNA-treated BEAS2B cells were infected with MERS-CoV at 0.1 m.o.i. for
1 h at 37 °C. The cell lysates (F) and
supernatants (G) were harvested at 24 and 48 h
post-infection. MERS-CoV replication was evaluated with qPCR analysis.
H, siRNA-treated MDM or HFL was infected with
MERS-CoV at 1 m.o.i. for 2 h at 37 °C. After 2 h, the cells were
harvested, and virus entry was evaluated with qPCR analysis
(I). The result was normalized to that of the
scrambled siRNA-treated cells. siRNA-treated MDM or HFL was infected
with MERS-CoV at 0.1 m.o.i. for 1 h at 37 °C. The cell lysates
(J) and supernatants (K) were
harvested at 24 h post-infection. MERS-CoV replication was evaluated
with qPCR analysis. In all panels, data represented mean and S.D. from
three independent experiments. Statistical analyses were carried out
using Student's t test. Statistical significance was
indicated by asterisks when p <
0.05. ns means not significant.

GRP78 is an attachment factor of MERS-CoVOur earlier data supported the notion that cell-surface GRP78 was involved in
MERS-CoV entry. To define the functional role of GRP78 during this process, we
challenged AD293 or BHK21 cells with MERS-CoV after GRP78 overexpression. First,
we sought to evaluate the capacity of GRP78 in facilitating MERS-CoV attachment.
To this end, GRP78-transfected AD293 or BHK21 cells were challenged with
MERS-CoV at 4 °C for 2 h. After the incubation, the cells were washed,
fixed, and immunolabeled for MERS-CoV N. As illustrated in Fig. 6A, GRP78 overexpression significantly
increased virus attachment in both AD293 and BHK21 cells. Interestingly, GRP78
overexpression appeared to induce a more substantial increase in MERS-CoV
attachment in the MERS-CoV–nonsusceptible BHK21 cells than that in the
MERS-CoV–susceptible AD293 cells (Fig.
6B). Next, to address whether GRP78 could
independently facilitate MERS-CoV entry, we assessed the level of MERS-CoV entry
in AD293 and BHK21 cells upon GRP78 overexpression. To this end,
GRP78-transfected AD293 and BHK21 cells were challenged with MERS-CoV at 37
°C for 2 h. After infection, the cells were washed and incubated for
another 4 h before harvesting for flow cytometry. Importantly, our result
demonstrated that the nonpermissive BHK21 cells remained refractory to MERS-CoV
infection despite GRP78 overexpression. In contrast, GRP78 overexpression
further enhanced the entry of MERS-CoV to the permissive AD293 cells (Fig. 6, C and
D). The effect of GRP78 on MERS-CoV entry was not due to ER
stress (Fig.
S4). Overall, our data indicated that GRP78 could not facilitate
MERS-CoV entry independently but could serve as an attachment factor and
modulate MERS-CoV entry in the presence of DPP4.Figure 6.GRP78 is an attachment factor of MERS-CoV.
A, to assess the role of GRP78 on MERS-CoV attachment,
GRP78-overexpressing AD293 and BHK21 cells were challenged with MERS-CoV
at 15 m.o.i. for 2 h at 4 °C. After 2 h, the cells were washed,
detached with 10 mm EDTA on ice, and fixed in 4%
paraformaldehyde before immunolabeling for flow cytometry.
B, percentage of MERS-CoV N–positive AD293
and BHK21 cells was quantified for MERS-CoV attachment.
C, to assess the role of GRP78 on MERS-CoV entry,
GRP78-overexpressing AD293 and BHK21 cells were challenged with MERS-CoV
at 5 m.o.i. for 2 h at 37 °C. After 2 h, the inoculum was replaced
with culture media, and the cells were incubated for another 4 h before
harvesting for flow cytometry. D, percentage of
MERS-CoV N–positive AD293 and BHK21 cells was quantified for
MERS-CoV entry. B and D, percentage of
MERS-CoV N–positive cells among GRP78-transfected
(GRP78+) cells was calculated as
(%GRP78+N+
cells/(%GRP78+N+ cells +
%GRP78+N− cells)) × 100%. The
percentage of MERS-CoV N–positive cells among
GRP78-nontransfected (GRP78−) cells was calculated as
(%GRP78−N+
cells/(%GRP78−N+ cells +
%GRP78−N− cells)) × 100%.
Data represented mean and S.D. derived from three independent
experiments. Statistical analyses were carried out using Student's
t test. Statistical significance was indicated by
asterisks when p < 0.05.
ns means not significant.

GRP78 is an attachment factor of MERS-CoV

Our earlier data supported the notion that cell-surface GRP78 was involved in
MERS-CoV entry. To define the functional role of GRP78 during this process, we
challenged AD293 or BHK21 cells with MERS-CoV after GRP78 overexpression. First,
we sought to evaluate the capacity of GRP78 in facilitating MERS-CoV attachment.
To this end, GRP78-transfected AD293 or BHK21 cells were challenged with
MERS-CoV at 4 °C for 2 h. After the incubation, the cells were washed,
fixed, and immunolabeled for MERS-CoV N. As illustrated in Fig. 6A, GRP78 overexpression significantly
increased virus attachment in both AD293 and BHK21 cells. Interestingly, GRP78
overexpression appeared to induce a more substantial increase in MERS-CoV
attachment in the MERS-CoV–nonsusceptible BHK21 cells than that in the
MERS-CoV–susceptible AD293 cells (Fig.
6B). Next, to address whether GRP78 could
independently facilitate MERS-CoV entry, we assessed the level of MERS-CoV entry
in AD293 and BHK21 cells upon GRP78 overexpression. To this end,
GRP78-transfected AD293 and BHK21 cells were challenged with MERS-CoV at 37
°C for 2 h. After infection, the cells were washed and incubated for
another 4 h before harvesting for flow cytometry. Importantly, our result
demonstrated that the nonpermissive BHK21 cells remained refractory to MERS-CoV
infection despite GRP78 overexpression. In contrast, GRP78 overexpression
further enhanced the entry of MERS-CoV to the permissive AD293 cells (Fig. 6, C and
D). The effect of GRP78 on MERS-CoV entry was not due to ER
stress (Fig.
S4). Overall, our data indicated that GRP78 could not facilitate
MERS-CoV entry independently but could serve as an attachment factor and
modulate MERS-CoV entry in the presence of DPP4.

GRP78 is an attachment factor of MERS-CoV.
A, to assess the role of GRP78 on MERS-CoV attachment,
GRP78-overexpressing AD293 and BHK21 cells were challenged with MERS-CoV
at 15 m.o.i. for 2 h at 4 °C. After 2 h, the cells were washed,
detached with 10 mm EDTA on ice, and fixed in 4%
paraformaldehyde before immunolabeling for flow cytometry.
B, percentage of MERS-CoV N–positive AD293
and BHK21 cells was quantified for MERS-CoV attachment.
C, to assess the role of GRP78 on MERS-CoV entry,
GRP78-overexpressing AD293 and BHK21 cells were challenged with MERS-CoV
at 5 m.o.i. for 2 h at 37 °C. After 2 h, the inoculum was replaced
with culture media, and the cells were incubated for another 4 h before
harvesting for flow cytometry. D, percentage of
MERS-CoV N–positive AD293 and BHK21 cells was quantified for
MERS-CoV entry. B and D, percentage of
MERS-CoV N–positive cells among GRP78-transfected
(GRP78+) cells was calculated as
(%GRP78+N+
cells/(%GRP78+N+ cells +
%GRP78+N− cells)) × 100%. The
percentage of MERS-CoV N–positive cells among
GRP78-nontransfected (GRP78−) cells was calculated as
(%GRP78−N+
cells/(%GRP78−N+ cells +
%GRP78−N− cells)) × 100%.
Data represented mean and S.D. derived from three independent
experiments. Statistical analyses were carried out using Student's
t test. Statistical significance was indicated by
asterisks when p < 0.05.
ns means not significant.

GRP78 is up-regulated on the surface of MERS-CoV–infected
cellsBecause infections by certain coronaviruses, including infectious bronchitis
virus and SARS-CoV, are known to induce ER stress (35–39), which can
promote GRP78 expression on the cell surface (40–43), we asked whether MERS-CoV infection
could up-regulate GRP78 expression on the cell surface. To address this
question, we infected Huh7 cells with MERS-CoV (Fig. 7, A and B) and harvested
samples for flow cytometry at 24 h post-infection. Our result demonstrated that
although the percentage of surface DPP4-positive cells modestly decreased after
MERS-CoV infection, the percentage of surface GRP78-positive cells significantly
increased from ∼50 to ∼80% after MERS-CoV infection (Fig. 7, C and
D). In this regard, our results highlighted the potential
relevance of GRP78 on MERS-CoV attachment onto the infected cells.Figure 7.GRP78 is up-regulated on the surface of MERS-CoV–infected
cells.
A, Huh7 cells were infected with MERS-CoV at 0.01 and
0.1 m.o.i. and were harvested for flow cytometry analysis at 24 h
post-infection. B, percentage of MERS-CoV
N–positive cells was quantified. C, in parallel,
cell surface and total DPP4 and GRP78 among mock- or
MERS-CoV–infected samples were analyzed by flow cytometry.
D, percentage of DPP4-positive cells and
GRP78-positive cells in mock- or MERS-CoV–infected samples were
quantified. Total DPP4 and GRP78 staining was performed by first
permeabilizing the cells with 0.1% Triton X-100, whereas the surface
DPP4 and GRP78 staining was performed in the absence of cell
permeabilization. The gate in A represented the
percentage of MERS-CoV N–positive cells. The gates in
C represented the percentage of DPP4-
(upper panels) and GRP78 (lower
panels)-positive cells. Data represented mean and S.D.
derived from three independent experiments. Statistical analyses were
carried out using Student's t test. Statistical
significance was indicated by asterisks when
p < 0.05. ns means not
significant.

GRP78 is up-regulated on the surface of MERS-CoV–infected
cells

Because infections by certain coronaviruses, including infectious bronchitis
virus and SARS-CoV, are known to induce ER stress (35–39), which can
promote GRP78 expression on the cell surface (40–43), we asked whether MERS-CoV infection
could up-regulate GRP78 expression on the cell surface. To address this
question, we infected Huh7 cells with MERS-CoV (Fig. 7, A and B) and harvested
samples for flow cytometry at 24 h post-infection. Our result demonstrated that
although the percentage of surface DPP4-positive cells modestly decreased after
MERS-CoV infection, the percentage of surface GRP78-positive cells significantly
increased from ∼50 to ∼80% after MERS-CoV infection (Fig. 7, C and
D). In this regard, our results highlighted the potential
relevance of GRP78 on MERS-CoV attachment onto the infected cells.

GRP78 is up-regulated on the surface of MERS-CoV–infected
cells.
A, Huh7 cells were infected with MERS-CoV at 0.01 and
0.1 m.o.i. and were harvested for flow cytometry analysis at 24 h
post-infection. B, percentage of MERS-CoV
N–positive cells was quantified. C, in parallel,
cell surface and total DPP4 and GRP78 among mock- or
MERS-CoV–infected samples were analyzed by flow cytometry.
D, percentage of DPP4-positive cells and
GRP78-positive cells in mock- or MERS-CoV–infected samples were
quantified. Total DPP4 and GRP78 staining was performed by first
permeabilizing the cells with 0.1% Triton X-100, whereas the surface
DPP4 and GRP78 staining was performed in the absence of cell
permeabilization. The gate in A represented the
percentage of MERS-CoV N–positive cells. The gates in
C represented the percentage of DPP4-
(upper panels) and GRP78 (lower
panels)-positive cells. Data represented mean and S.D.
derived from three independent experiments. Statistical analyses were
carried out using Student's t test. Statistical
significance was indicated by asterisks when
p < 0.05. ns means not
significant.

GRP78 facilitates the cell-surface attachment of bCoV-HKU9Coronaviruses have evolved a complicated receptor recognition system through
their spike proteins. Peculiarly, the spike proteins from highly-related
coronaviruses can recognize different cell-surface molecules, whereas the spike
proteins of phylogenetically distant coronaviruses can bind the same
cell-surface molecule for attachment or entry (11). By exploring the potential interaction between GRP78 and the
spike proteins of other coronaviruses, we unexpectedly discovered that GRP78
could interact with the spike protein of bat coronavirus HKU9 (bCoV-HKU9) (Fig. 8A). Interestingly,
despite the capacity of binding the spike proteins of lineage C (MERS-CoV) and
lineage D (bCoV-HKU9) Betacoronavirus, GRP78 did not interact
with the spike protein of SARS-CoV, which is a lineage B
Betacoronavirus (Fig.
8B). In 2007, we reported the first discovery and
genome characterization of bCoV-HKU9, which was identified from Leschenault's
rousette bats (R. leschenaulti) (44). Recently, with structural analysis and surface plasmon
resonance assay, it appeared that the receptor-binding domain of bCoV-HKU9 spike
was incapable of reacting with either human DPP4 or ACE2 (45). In this regard, it would be important to explore the
potential physiological relevance of the interaction between GRP78 and bCoV-HKU9
spike. We first evaluated the cell tropism of HKU9–S-pseudovirus with
MERS–S-pseudovirus included as a control. Remarkably, our data suggested
that among the 10 evaluated mammalian cell lines, HKU9–S-pseudovirus
entry was most pronounced in R. leschenaulti kidney (RLK) cells
(Fig. 8C). Notably,
although MERS–S-pseudovirus entry was evident in RLK cells, culture for
bCoV-HKU9 in RLK or other cell lines has not been successful (44). In line with the pseudovirus entry
result, the surface-binding efficiency of HKU9–S-pseudovirus on RLK cells
was ∼3-fold that on Caco2 cells (Fig.
8D), which is a human colon cell line known to be
permissive for both MERS-CoV and SARS-CoV infection. Notably, overexpression of
human GRP78 in the apparently nonpermissive L929 and BHK21 cells did not render
the cells permissive to HKU9–S-pseudovirus entry, indicating that GRP78
could not function as an independent receptor for bCoV-HKU9 (Fig. 8E). In contrast, with
a flow cytometry-based surface-binding assay, we demonstrated that the GRP78
antibody (Fig. 8G) but not
the control IgG (Fig. 8F)
reduced the binding of HKU9–S-pseudovirus to the cell surface of RLK
cells in a dose-dependent manner, which was evidenced by the drop in the
percentage of HKU9–S-positive cells (Fig.
8H) as well as the decrease in the
HKU9–S-mean fluorescent intensity (Fig.
8I). Taken together, our data identified GRP78 as an
important cell surface–binding protein for both MERS-CoV and bCoV-HKU9 by
serving as an attachment factor.Figure 8.GRP78 interacts with the bCoV-HKU9 spike and serves as an
attachment factor for bCoV-HKU9.
A, BHK21 cells were transfected with
pcDNA–GRP78–V5 (lanes 1 and
2) or empty vector (lane 3). Co-IP
between GRP78 and bCoV-HKU9 spike was performed using GRP78 as the bait
protein. Purified bCoV–HKU9–S1–FLAG (lanes
1 and 3) or BAP–FLAG proteins
(lane 2) were immunoprecipitated
(IP) with overexpressed GRP78–V5 or
pcDNA–V5 proteins pre-adsorbed on anti-V5–Sepharose beads.
The precipitated protein complex was detected using the anti-V5 antibody
or the anti-FLAG antibody. B, co-IP between GRP78 and
SARS-CoV spike was performed using GRP78 as the bait protein. Purified
SARS-CoV–S1–FLAG (lanes 1 and
3) or BAP–FLAG proteins (lane
2) were immunoprecipitated with overexpressed
GRP78–V5 or pcDNA–V5 proteins pre-adsorbed on
anti-V5–Sepharose beads. The precipitated protein complex was
detected using the anti-V5 antibody or the anti-FLAG antibody.
C, HKU9–S-pseudovirus entry assays were
performed in a number of mammalian cell lines. Mock-inoculated and
MERS–S-pseudovirus–inoculated cells were included as
negative and positive controls, respectively. HKU9–S-pseudovirus
and MERS–S-pseudovirus were added at a ratio of 100 LP per cell
for 1 h. Luciferase activity was determined at 72 h post-inoculation.
D, HKU9–S-pseudovirus attachment efficiency
was evaluated in Caco2 and RLK cells. HKU9–S-pseudovirus was
inoculated on Caco2 and RLK cells at 100 LP per cell for 2 h at 4
°C. After 2 h, the cells were washed, fixed, and immunolabeled for
flow cytometry. HKU9–S-pseudovirus binding was identified with an
in-house mouse bCoV-HKU9 spike immune serum. E,
HKU9–S-pseudovirus entry in L929 and BHK21 cells was assessed
with or without GRP78 overexpression. HKU9–S-pseudovirus was
inoculated at 100 LP per cell for 1 h at 37 °C. Luciferase activity
was determined at 72 h post-inoculation. F and
G, antibody-blocking assay for
HKU9–S-pseudovirus binding was performed in RLK cells. RLK cells
were pre-incubated with the rabbit anti-GRP78 antibody and the rabbit
control IgG from 0 to 5 μg/ml. After the pre-incubation,
HKU9–S-pseudovirus was inoculated to the cells at 100 LP per cell
for 2 h at 4 °C. The cells were then washed, fixed, and
immunolabeled for flow cytometry. HKU9–S-pseudovirus binding was
identified with an in-house mouse bCoV-HKU9 spike immune serum. The
percentage of bCoV-HKU9 spike-positive cells was quantified in
H, and the MFI of the bCoV-HKU9 spike on the cell
surface was quantified in I. Gates in D,
F, and G represented the percentage of
HKU9 spike-positive cells. Data represented mean and S.D. derived from
three independent experiments. Statistical analyses were carried out
using Student's t test. Statistical significance was
indicated by asterisks when p <
0.05. WB, Western blot. ns means not
significant.

GRP78 facilitates the cell-surface attachment of bCoV-HKU9

Coronaviruses have evolved a complicated receptor recognition system through
their spike proteins. Peculiarly, the spike proteins from highly-related
coronaviruses can recognize different cell-surface molecules, whereas the spike
proteins of phylogenetically distant coronaviruses can bind the same
cell-surface molecule for attachment or entry (11). By exploring the potential interaction between GRP78 and the
spike proteins of other coronaviruses, we unexpectedly discovered that GRP78
could interact with the spike protein of bat coronavirus HKU9 (bCoV-HKU9) (Fig. 8A). Interestingly,
despite the capacity of binding the spike proteins of lineage C (MERS-CoV) and
lineage D (bCoV-HKU9) Betacoronavirus, GRP78 did not interact
with the spike protein of SARS-CoV, which is a lineage B
Betacoronavirus (Fig.
8B). In 2007, we reported the first discovery and
genome characterization of bCoV-HKU9, which was identified from Leschenault's
rousette bats (R. leschenaulti) (44). Recently, with structural analysis and surface plasmon
resonance assay, it appeared that the receptor-binding domain of bCoV-HKU9 spike
was incapable of reacting with either human DPP4 or ACE2 (45). In this regard, it would be important to explore the
potential physiological relevance of the interaction between GRP78 and bCoV-HKU9
spike. We first evaluated the cell tropism of HKU9–S-pseudovirus with
MERS–S-pseudovirus included as a control. Remarkably, our data suggested
that among the 10 evaluated mammalian cell lines, HKU9–S-pseudovirus
entry was most pronounced in R. leschenaulti kidney (RLK) cells
(Fig. 8C). Notably,
although MERS–S-pseudovirus entry was evident in RLK cells, culture for
bCoV-HKU9 in RLK or other cell lines has not been successful (44). In line with the pseudovirus entry
result, the surface-binding efficiency of HKU9–S-pseudovirus on RLK cells
was ∼3-fold that on Caco2 cells (Fig.
8D), which is a human colon cell line known to be
permissive for both MERS-CoV and SARS-CoV infection. Notably, overexpression of
human GRP78 in the apparently nonpermissive L929 and BHK21 cells did not render
the cells permissive to HKU9–S-pseudovirus entry, indicating that GRP78
could not function as an independent receptor for bCoV-HKU9 (Fig. 8E). In contrast, with
a flow cytometry-based surface-binding assay, we demonstrated that the GRP78
antibody (Fig. 8G) but not
the control IgG (Fig. 8F)
reduced the binding of HKU9–S-pseudovirus to the cell surface of RLK
cells in a dose-dependent manner, which was evidenced by the drop in the
percentage of HKU9–S-positive cells (Fig.
8H) as well as the decrease in the
HKU9–S-mean fluorescent intensity (Fig.
8I). Taken together, our data identified GRP78 as an
important cell surface–binding protein for both MERS-CoV and bCoV-HKU9 by
serving as an attachment factor.

GRP78 interacts with the bCoV-HKU9 spike and serves as an
attachment factor for bCoV-HKU9.
A, BHK21 cells were transfected with
pcDNA–GRP78–V5 (lanes 1 and
2) or empty vector (lane 3). Co-IP
between GRP78 and bCoV-HKU9 spike was performed using GRP78 as the bait
protein. Purified bCoV–HKU9–S1–FLAG (lanes
1 and 3) or BAP–FLAG proteins
(lane 2) were immunoprecipitated
(IP) with overexpressed GRP78–V5 or
pcDNA–V5 proteins pre-adsorbed on anti-V5–Sepharose beads.
The precipitated protein complex was detected using the anti-V5 antibody
or the anti-FLAG antibody. B, co-IP between GRP78 and
SARS-CoV spike was performed using GRP78 as the bait protein. Purified
SARS-CoV–S1–FLAG (lanes 1 and
3) or BAP–FLAG proteins (lane
2) were immunoprecipitated with overexpressed
GRP78–V5 or pcDNA–V5 proteins pre-adsorbed on
anti-V5–Sepharose beads. The precipitated protein complex was
detected using the anti-V5 antibody or the anti-FLAG antibody.
C, HKU9–S-pseudovirus entry assays were
performed in a number of mammalian cell lines. Mock-inoculated and
MERS–S-pseudovirus–inoculated cells were included as
negative and positive controls, respectively. HKU9–S-pseudovirus
and MERS–S-pseudovirus were added at a ratio of 100 LP per cell
for 1 h. Luciferase activity was determined at 72 h post-inoculation.
D, HKU9–S-pseudovirus attachment efficiency
was evaluated in Caco2 and RLK cells. HKU9–S-pseudovirus was
inoculated on Caco2 and RLK cells at 100 LP per cell for 2 h at 4
°C. After 2 h, the cells were washed, fixed, and immunolabeled for
flow cytometry. HKU9–S-pseudovirus binding was identified with an
in-house mouse bCoV-HKU9 spike immune serum. E,
HKU9–S-pseudovirus entry in L929 and BHK21 cells was assessed
with or without GRP78 overexpression. HKU9–S-pseudovirus was
inoculated at 100 LP per cell for 1 h at 37 °C. Luciferase activity
was determined at 72 h post-inoculation. F and
G, antibody-blocking assay for
HKU9–S-pseudovirus binding was performed in RLK cells. RLK cells
were pre-incubated with the rabbit anti-GRP78 antibody and the rabbit
control IgG from 0 to 5 μg/ml. After the pre-incubation,
HKU9–S-pseudovirus was inoculated to the cells at 100 LP per cell
for 2 h at 4 °C. The cells were then washed, fixed, and
immunolabeled for flow cytometry. HKU9–S-pseudovirus binding was
identified with an in-house mouse bCoV-HKU9 spike immune serum. The
percentage of bCoV-HKU9 spike-positive cells was quantified in
H, and the MFI of the bCoV-HKU9 spike on the cell
surface was quantified in I. Gates in D,
F, and G represented the percentage of
HKU9 spike-positive cells. Data represented mean and S.D. derived from
three independent experiments. Statistical analyses were carried out
using Student's t test. Statistical significance was
indicated by asterisks when p <
0.05. WB, Western blot. ns means not
significant.

Sialic acids and GRP78 act independently to facilitate the surface attachment
of MERS-CoVSialic acids were recently identified as an attachment determinant of MERS-CoV
(46). To investigate whether GRP78
and sialic acids could act in conjunction with each other in facilitating the
attachment of MERS-CoV, we assessed MERS–S-pseudovirus entry in the
presence of a combination of neuraminidase treatment and GRP78 antibody
blocking. Our results demonstrated that although neuraminidase treatment
decreased MERS–S-pseudovirus entry in a dose-dependent manner, the
addition of GRP78 antibody further enhanced the inhibitory effect (Fig. 9A). However, the entry
of HKU9–S-pseudovirus was inhibited by GRP78 antibody but not
neuraminidase treatment (Fig.
9B). Overall, the additive effect of neuraminidase
treatment and GRP78 antibody on limiting MERS–S-pseudovirus entry
suggested that sialic acids and GRP78 both independently facilitated the
attachment of MERS-CoV onto the cell surface, whereas GRP78 but not sialic acids
played an important role for virus attachment of bCoV-HKU9.Figure 9.Sialic acids and GRP78 act independently to facilitate the surface
attachment of MERS-CoV.
A, Huh7 cells were treated with neuraminidase from
C. perfringens, with or without pre-incubation with
the GRP78 polyclonal antibody. The cells were subsequently challenged
with MERS–S-pseudovirus and assessed at 72 h post-infection for
pseudovirus entry. B, RLK cells were treated with
neuraminidase from Clostridium perfringens, with or
without pre-incubation with the GRP78 polyclonal antibody. The cells
were subsequently challenged with HKU9–S-pseudovirus and assessed
at 72 h post infection for pseudovirus entry. Pseudovirus entry was
quantified using a microplate reader as relative light units
(RLU). Data represented mean and standard deviation
derived from three independent experiments. Statistical analyses were
carried out using Student's t test. Statistical
significance was indicated by asterisk marks when
p < 0.05.

Sialic acids and GRP78 act independently to facilitate the surface attachment
of MERS-CoV

Sialic acids were recently identified as an attachment determinant of MERS-CoV
(46). To investigate whether GRP78
and sialic acids could act in conjunction with each other in facilitating the
attachment of MERS-CoV, we assessed MERS–S-pseudovirus entry in the
presence of a combination of neuraminidase treatment and GRP78 antibody
blocking. Our results demonstrated that although neuraminidase treatment
decreased MERS–S-pseudovirus entry in a dose-dependent manner, the
addition of GRP78 antibody further enhanced the inhibitory effect (Fig. 9A). However, the entry
of HKU9–S-pseudovirus was inhibited by GRP78 antibody but not
neuraminidase treatment (Fig.
9B). Overall, the additive effect of neuraminidase
treatment and GRP78 antibody on limiting MERS–S-pseudovirus entry
suggested that sialic acids and GRP78 both independently facilitated the
attachment of MERS-CoV onto the cell surface, whereas GRP78 but not sialic acids
played an important role for virus attachment of bCoV-HKU9.

Sialic acids and GRP78 act independently to facilitate the surface
attachment of MERS-CoV.
A, Huh7 cells were treated with neuraminidase from
C. perfringens, with or without pre-incubation with
the GRP78 polyclonal antibody. The cells were subsequently challenged
with MERS–S-pseudovirus and assessed at 72 h post-infection for
pseudovirus entry. B, RLK cells were treated with
neuraminidase from Clostridium perfringens, with or
without pre-incubation with the GRP78 polyclonal antibody. The cells
were subsequently challenged with HKU9–S-pseudovirus and assessed
at 72 h post infection for pseudovirus entry. Pseudovirus entry was
quantified using a microplate reader as relative light units
(RLU). Data represented mean and standard deviation
derived from three independent experiments. Statistical analyses were
carried out using Student's t test. Statistical
significance was indicated by asterisk marks when
p < 0.05.

DiscussionHost tropism is predominantly determined by the interaction between coronavirus
spikes and their corresponding host receptors. In addition, the spike proteins of
coronaviruses can recognize a broad range of cell-surface molecules, which serve to
augment coronavirus attachment or entry. In this study, we identified host GRP78 as
a novel interacting target of the MERS-CoV spike (Figs. 1 and 2). GRP78 was expressed
on the surface of MERS-CoV–susceptible cell lines of pulmonary and
extrapulmonary origin (Fig. 3). At the same
time, immunostaining of human lung tissues identified abundant co-expression of DPP4
and GRP78 in the epithelial cells along the human airways (Fig. 4). Next, with antibody blocking and siRNA knockdown
experiments, our data indicated the involvement of GRP78 in MERS-CoV entry (Fig. 5). Overexpression assays of GRP78 in
MERS-CoV–permissive and MERS-CoV–nonpermissive cells unambiguously
demonstrated that GRP78 did not independently render nonpermissive cells susceptible
to MERS-CoV infection but could facilitate MERS-CoV entry in conjunction with DPP4
by serving as an attachment factor (Fig. 6).
Intriguingly, GRP78 was up-regulated upon MERS-CoV infection, which might further
facilitate virus attachment among the infected cells (Fig. 7). Most importantly, GRP78 was also recognized by the spike
protein of a bat Betacoronavirus, bCoV-HKU9. Our result further
indicated that GRP78 was not the functional receptor of bCoV-HKU9 but could modulate
HKU9–S-pseudovirus attachment to RLK cells (Fig. 8). Simultaneous treatments of neuraminidase and GRP78 antibody
blocking revealed that sialic acids and GRP78 both independently facilitated the
attachment of MERS-CoV onto the cell surface, whereas virus attachment of bCoV-HKU9
was mediated by GRP78 but not sialic acids (Fig.
9). Overall, our study identified GRP78 as an attachment factor that
might modulate virus entry for two phylogenetically related
Betacoronaviruses of different lineages, MERS-CoV and
bCoV-HKU9.GRP78, also referred to as BiP or HSPA5, is traditionally recognized as an ER
chaperone (27). It is involved in a wide
range of physiological processes, including protein folding and assembly,
translocation of newly synthesized polypeptides, degradation of misfolded proteins,
as well as maintaining the ER homeostasis (27). In addition, GRP78 is an essential regulator of ER stress due to its
critical role in the unfolded protein response pathway. Despite its participation in
ER-related functions, GRP78 is also detected in other cellular fractions, including
mitochondria, nucleus, cytosol, and plasma membrane (43). In recent years, an increasing number of studies have described the
physiological role of cell-surface GRP78 during virus entry. For instance, GRP78 was
identified as a co-receptor for coxsackievirus A9 (CVA9) (30) and dengue virus (47). In addition, cell-surface GRP78 also facilitates the entry of Japanese
encephalitis virus (29). Here, we reported
GRP78 as a host factor that could serve as an attachment protein for two
Betacoronaviruses, MERS-CoV and bCoV-HKU9. In its capacity as
an attachment factor, GRP78 may serve to concentrate virus particles on the cell
surface, which may then increase the possibility of receptor-mediated virus entry
for MERS-CoV and bCoV-HKU9. Importantly, MERS-CoV infection resulted in an
up-regulation of GRP78 on the cell surface, which may in turn increase the
attachment of MERS-CoV and further enhance the possibility of virus entry in the
infected cells.Coronaviruses can recognize a wide range of cell-surface molecules, including cell
membrane proteins and sugars in addition to their cellular receptors. As an example,
HCoV-NL63 employs ACE2 for host cell entry (14) but can bind to cell-surface heparan sulfate proteoglycans to
enhance attachment and infection of target cells (16). We have previously reported the identification of CEACAM5 as an
attachment factor of MERS-CoV, which could facilitate MERS-CoV entry in the presence
of DPP4 (25). Recently, CD9 was reported as a
host factor that could augment MERS-CoV entry by bringing the cellular receptor and
proteases into close proximity, thus increasing the infection efficiency (26). In this study, the identification of GRP78
as an attachment factor of MERS-CoV further indicated that the spike protein of
MERS-CoV is highly efficient in engaging multiple cell surface factors to facilitate
virus entry. In contrast to CEACAM5, which is expressed on limited cell types (25), surface GRP78 expression appeared to be
relatively abundant across various cell types of different tissue origin (Fig. 3). In addition, a remarkable level of GRP78
was specifically detected on the epithelial cells along the human airways, where it
was found to colocalize with DPP4 (Fig. 4). In
this regard, it is tempting to speculate that the capacity of the MERS-CoV spike to
utilize multiple host surface proteins, including CEACAM5, CD9, and GRP78, may give
MERS-CoV a physiological advantage in establishing efficient infections, which may
contribute to the high pathogenicity of the virus.Bat coronavirus HKU9 (bCoV-HKU9) is a representative lineage D
Betacoronavirus. The virus was first identified in 2007 in a
territory-wide molecular surveillance study on bat samples from the Guangdong
province of Southern China (44). Subsequent
studies suggested that the virus was widely distributed and was circulating in
different bat species (48–51). Structural and functional features of the receptor-binding domain
(RBD) of bCoV-HKU9 demonstrated that the spike protein of the virus was incapable of
interacting with either DPP4 or ACE2 (45).
However, the RBD of bCoV-HKU9 contained a conserved core structure that was shared
across other Betacoronaviruses, including MERS-CoV, SARS-CoV, and
bat coronavirus HKU4 (bCoV-HKU4) (45).
Notably, recent reports demonstrated that bCoV-HKU4 could recognize human DPP4 as a
functional receptor, indicating the potential of bat coronaviruses in human
adaptations. In this regard, the identification of GRP78 as a shared attachment
factor for MERS-CoV and bCoV-HKU9 is interesting but alarming, which highlighted the
importance of continuous surveillance on the other members of the
Betacoronavirus genus for their capacity of interspecies
transmission.In summary, almost all presently circulating human coronaviruses have a
phylogenetically-related virus partner found in animals. The human
Alphacoronavirus HCoV-NL63 may be a recombinant between
NL63-like viruses in Triaenops bats and 229E-like viruses
circulating in Hipposideros bats (52). Another human Alphacoronavirus HCoV-229E has
closely-related 229E-like coronaviruses recently isolated from dromedary camels
(53). Similarly, the lineage A
Betacoronavirus HCoV-OC43 was postulated to originate from a
bovine coronavirus and jumped into human in the 1890s (2, 54). The lineage B
Betacoronavirus, SARS-CoV, originated from either civets or
bats, which jumped into human in 2003 (55,
56), whereas the lineage C
Betacoronavirus, MERS-CoV, is likely to have jumped from camels
into human in 2012 (57, 58). Because three (lineages A, B, and C) out of four lineages
of animal Betacoronaviruses have independently jumped from animal
into human in the recent past, there is enough reason to suspect that a lineage D
Betacoronavirus may also jump into human one day. Our finding
of the lineage C MERS-CoV and lineage D bCoV-HKU9 sharing the same host attachment
factor GRP78 highlights the importance of monitoring the evolution of bCoV-HKU9,
which may jump the interspecies barrier into human leading to another major epidemic
in the future.

Host tropism is predominantly determined by the interaction between coronavirus
spikes and their corresponding host receptors. In addition, the spike proteins of
coronaviruses can recognize a broad range of cell-surface molecules, which serve to
augment coronavirus attachment or entry. In this study, we identified host GRP78 as
a novel interacting target of the MERS-CoV spike (Figs. 1 and 2). GRP78 was expressed
on the surface of MERS-CoV–susceptible cell lines of pulmonary and
extrapulmonary origin (Fig. 3). At the same
time, immunostaining of human lung tissues identified abundant co-expression of DPP4
and GRP78 in the epithelial cells along the human airways (Fig. 4). Next, with antibody blocking and siRNA knockdown
experiments, our data indicated the involvement of GRP78 in MERS-CoV entry (Fig. 5). Overexpression assays of GRP78 in
MERS-CoV–permissive and MERS-CoV–nonpermissive cells unambiguously
demonstrated that GRP78 did not independently render nonpermissive cells susceptible
to MERS-CoV infection but could facilitate MERS-CoV entry in conjunction with DPP4
by serving as an attachment factor (Fig. 6).
Intriguingly, GRP78 was up-regulated upon MERS-CoV infection, which might further
facilitate virus attachment among the infected cells (Fig. 7). Most importantly, GRP78 was also recognized by the spike
protein of a bat Betacoronavirus, bCoV-HKU9. Our result further
indicated that GRP78 was not the functional receptor of bCoV-HKU9 but could modulate
HKU9–S-pseudovirus attachment to RLK cells (Fig. 8). Simultaneous treatments of neuraminidase and GRP78 antibody
blocking revealed that sialic acids and GRP78 both independently facilitated the
attachment of MERS-CoV onto the cell surface, whereas virus attachment of bCoV-HKU9
was mediated by GRP78 but not sialic acids (Fig.
9). Overall, our study identified GRP78 as an attachment factor that
might modulate virus entry for two phylogenetically related
Betacoronaviruses of different lineages, MERS-CoV and
bCoV-HKU9.

GRP78, also referred to as BiP or HSPA5, is traditionally recognized as an ER
chaperone (27). It is involved in a wide
range of physiological processes, including protein folding and assembly,
translocation of newly synthesized polypeptides, degradation of misfolded proteins,
as well as maintaining the ER homeostasis (27). In addition, GRP78 is an essential regulator of ER stress due to its
critical role in the unfolded protein response pathway. Despite its participation in
ER-related functions, GRP78 is also detected in other cellular fractions, including
mitochondria, nucleus, cytosol, and plasma membrane (43). In recent years, an increasing number of studies have described the
physiological role of cell-surface GRP78 during virus entry. For instance, GRP78 was
identified as a co-receptor for coxsackievirus A9 (CVA9) (30) and dengue virus (47). In addition, cell-surface GRP78 also facilitates the entry of Japanese
encephalitis virus (29). Here, we reported
GRP78 as a host factor that could serve as an attachment protein for two
Betacoronaviruses, MERS-CoV and bCoV-HKU9. In its capacity as
an attachment factor, GRP78 may serve to concentrate virus particles on the cell
surface, which may then increase the possibility of receptor-mediated virus entry
for MERS-CoV and bCoV-HKU9. Importantly, MERS-CoV infection resulted in an
up-regulation of GRP78 on the cell surface, which may in turn increase the
attachment of MERS-CoV and further enhance the possibility of virus entry in the
infected cells.

Coronaviruses can recognize a wide range of cell-surface molecules, including cell
membrane proteins and sugars in addition to their cellular receptors. As an example,
HCoV-NL63 employs ACE2 for host cell entry (14) but can bind to cell-surface heparan sulfate proteoglycans to
enhance attachment and infection of target cells (16). We have previously reported the identification of CEACAM5 as an
attachment factor of MERS-CoV, which could facilitate MERS-CoV entry in the presence
of DPP4 (25). Recently, CD9 was reported as a
host factor that could augment MERS-CoV entry by bringing the cellular receptor and
proteases into close proximity, thus increasing the infection efficiency (26). In this study, the identification of GRP78
as an attachment factor of MERS-CoV further indicated that the spike protein of
MERS-CoV is highly efficient in engaging multiple cell surface factors to facilitate
virus entry. In contrast to CEACAM5, which is expressed on limited cell types (25), surface GRP78 expression appeared to be
relatively abundant across various cell types of different tissue origin (Fig. 3). In addition, a remarkable level of GRP78
was specifically detected on the epithelial cells along the human airways, where it
was found to colocalize with DPP4 (Fig. 4). In
this regard, it is tempting to speculate that the capacity of the MERS-CoV spike to
utilize multiple host surface proteins, including CEACAM5, CD9, and GRP78, may give
MERS-CoV a physiological advantage in establishing efficient infections, which may
contribute to the high pathogenicity of the virus.

Bat coronavirus HKU9 (bCoV-HKU9) is a representative lineage D
Betacoronavirus. The virus was first identified in 2007 in a
territory-wide molecular surveillance study on bat samples from the Guangdong
province of Southern China (44). Subsequent
studies suggested that the virus was widely distributed and was circulating in
different bat species (48–51). Structural and functional features of the receptor-binding domain
(RBD) of bCoV-HKU9 demonstrated that the spike protein of the virus was incapable of
interacting with either DPP4 or ACE2 (45).
However, the RBD of bCoV-HKU9 contained a conserved core structure that was shared
across other Betacoronaviruses, including MERS-CoV, SARS-CoV, and
bat coronavirus HKU4 (bCoV-HKU4) (45).
Notably, recent reports demonstrated that bCoV-HKU4 could recognize human DPP4 as a
functional receptor, indicating the potential of bat coronaviruses in human
adaptations. In this regard, the identification of GRP78 as a shared attachment
factor for MERS-CoV and bCoV-HKU9 is interesting but alarming, which highlighted the
importance of continuous surveillance on the other members of the
Betacoronavirus genus for their capacity of interspecies
transmission.

In summary, almost all presently circulating human coronaviruses have a
phylogenetically-related virus partner found in animals. The human
Alphacoronavirus HCoV-NL63 may be a recombinant between
NL63-like viruses in Triaenops bats and 229E-like viruses
circulating in Hipposideros bats (52). Another human Alphacoronavirus HCoV-229E has
closely-related 229E-like coronaviruses recently isolated from dromedary camels
(53). Similarly, the lineage A
Betacoronavirus HCoV-OC43 was postulated to originate from a
bovine coronavirus and jumped into human in the 1890s (2, 54). The lineage B
Betacoronavirus, SARS-CoV, originated from either civets or
bats, which jumped into human in 2003 (55,
56), whereas the lineage C
Betacoronavirus, MERS-CoV, is likely to have jumped from camels
into human in 2012 (57, 58). Because three (lineages A, B, and C) out of four lineages
of animal Betacoronaviruses have independently jumped from animal
into human in the recent past, there is enough reason to suspect that a lineage D
Betacoronavirus may also jump into human one day. Our finding
of the lineage C MERS-CoV and lineage D bCoV-HKU9 sharing the same host attachment
factor GRP78 highlights the importance of monitoring the evolution of bCoV-HKU9,
which may jump the interspecies barrier into human leading to another major epidemic
in the future.

Experimental proceduresCellsA549, AD293 (a derivative of the commonly used HEK293 cell line, with improved
cell adherence), HeLa, Huh7, Caco2, and VeroE6 cells were maintained in
Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% heat-inactivated
fetal bovine serum (FBS), 100 units/ml penicillin, and 100 μg/ml
streptomycin. BEAS2B (transformed epithelial cells isolated from normal human
bronchial epithelium) and Calu3 cells were maintained in supplemented DMEM/F-12.
BHK21, L929, RLK, and HFL were maintained in supplemented minimum essential
medium. Human primary monocytes were obtained from human peripheral blood
mononuclear cells (PBMCs) as described previously (59). Primary human monocyte-derived macrophages (MDMs) were
differentiated from monocytes in Roswell Park Memorial Institute (RPMI)-1640
media supplemented with 10% FBS, 100 μg/ml streptomycin, 100 units/ml
penicillin, 2 mm glutamine, 1% sodium pyruvate, 1% nonessential amino
acids, and 10 ng/ml recombinant human granulocyte macrophage colony-stimulating
factor (GM-CSF) (R&D Systems) (60).
Human primary T cells were isolated from PBMCs with negative selection using the
Dynabeads Untouched Human T cells kit (Invitrogen) as we described previously
(61). Isolated T cells were
maintained in RPMI 1640 medium supplemented with 10% FBS, 100 μg/ml
streptomycin, 100 units/ml penicillin, 1% sodium pyruvate, and 1% nonessential
amino acids.VirusMERS-CoV was a gift from Dr. Ron Fouchier (Erasmus Medical Center, Rotterdam, the
Netherlands) and cultured in VeroE6 cells in serum-free DMEM. Virus titers were
quantified with plaque assays as described previously (25).AntibodiesMERS-CoV nucleocapsid protein (N) was detected with the in-house guinea pig
anti-MERS–CoV N serum as we described previously (32, 62). The
MERS-CoV spike was detected with either the in-house mouse anti-MERS-CoV spike
immune serum or a rabbit anti-MERS-CoV spike antibody from Sino Biological
(40069-RP02). An in-house mouse anti-bCoV-HKU9-spike immune serum was used to
detect bCoV-HKU9 spike. Primary antibodies, including rabbit anti-DPP4
(ab28340), rabbit anti-GRP78 (ab21685), rabbit anti-pan-cadherin (ab16505),
rabbit anti-calreticulin (ab2907), rabbit anti-GM130 (ab52649), rabbit anti-EGFR
(ab52894), and rabbit anti-CEACAM5 (ab131070), were from Abcam. Rabbit
anti-lamin A was from Sigma (SAB4501764). Rabbit control IgG was from
ThermoFisher Scientific (31235). Mouse anti-GRP78 antibody for Western blotting
was from R&D Systems (MAB4846). Rabbit anti-GRP78 from Novus Biologicals
(NBP1-54318) and goat anti-CD26 from R&D Systems (AF1180) were used for
antibody-blocking experiments. Rabbit anti-giantin was from Biolegend
(A488-114L). The mouse anti-β-actin was from Sigma (A5441). The
recombinant FLAG-conjugated proteins were detected with an anti-FLAG M2 antibody
from Sigma (F1804). The V5-tagged proteins were detected with mouse anti-V5
antibodies from Immnoway (YM3005) or ThermoFisher Scientific (R96025). The
eGFP-tagged proteins were detected with a rabbit anti-eGFP from Abcam (ab290).
Secondary antibodies, including Alexa Fluor 488/647 goat anti-guinea pig
(A11073/A21450) and Alexa Fluor 488/647 goat anti-rabbit (A11008/A21245) from
ThermoFisher Scientific, were used for flow cytometry. The goat anti-mouse HRP
(626520) and goat anti-rabbit HRP (656120) antibodies from ThermoFisher
Scientific were used for Western blottings.Plasmid constructionThe construction of pcDNA–MERS–CoV–S was previously
described (25). Codon-optimized
bCoV-HKU9-spike DNA was synthesized at GeneArt (ThermoFisher Scientific) based
on amino acid sequence of the bCoV-HKU9 (44) and cloned into the pcDNA3.1(+) vector. In parallel,
bCoV-HKU9–S1 was subcloned into the pSFV1 vector with a FLAG sequence
in-frame in the 3′-end for protein expression. The expression construct
for codon-optimized SARS-CoV spike, pcDNA-Sopt9, was a gift from Dr. Chen Zhiwei
and was previously described (63). The
construction of pSFV–MERS-CoV–S1–FLAG was previously
described (25). The same ORF was
PCR-amplified and subcloned into pcDNA3.1(+) vector in-frame with a V5 epitope,
which resulted in pcDNA–MERS-CoV–S1–V5. The GRP78-coding
region, including the N-terminal signal peptide, was obtained with RT-PCR from
BEAS2B cells and cloned into pcDNA3.1(+) vector fused with V5, which resulted in
pcDNA–GRP78–V5.Immunoaffinity purification of MERS-CoV–S1–FLAG proteinExpression of FLAG-tagged recombinant proteins was previously described (64, 65). In brief, linearized pSFV–FLAG plasmids were transcribed
in vitro, and the derived capped RNAs were electroporated
into BHK21 cells. At 15 h post-transfection, the cells were lysed, and the
expressed recombinant proteins were immunopurified using anti-FLAG
M2–coated beads (Sigma) according to the manufacturer's instructions. The
purified S protein was assessed by SDS-PAGE and Western blotting. Protein
concentration was quantified with the Pierce BCA assay (ThermoFisher
Scientific).Membrane extractionpcDNA–MERS-CoV–S1–V5–transfected BEAS2B cells
cultured in 10-cm dishes were harvested by scraping cells into HEPES solution
(10 mm HEPES, pH 7.5, 1.5 mm MgCl2, 1 mm
KCl) and centrifuging briefly at 500 × g for 3 min. Cell
pellets were then homogenized in membrane lysis buffer (20 mm Tris-HCl,
pH 7.5, 150 mm NaCl, 1% Nonidet P-40 (Calbiochem), 1%
n-dodecyl β-maltoside (ThermoFisher Scientific), 5%
glycerol, pH 7.5, with protease inhibitor mixture (Roche Applied Science)) and
incubated on ice for 30 min. Residual cellular debris and nuclei in the
resulting extracts were sedimented by centrifugation at 4 °C for 5 min at
6000 × g. The solubilized membrane proteins in the
supernatant were transferred and subjected to an additional spin at 16,000
× g for 30 min at 4 °C, and the membrane extracts
were then resuspended in lysis buffer. All extracts were quantitated using the
Pierce BCA assay kit (ThermoFisher Scientific) and stored in aliquots at
−80 °C until used.Identification of GRP78 by immunoprecipitation and MSMembrane proteins from
pcDNA–MERS-CoV–S1–V5–transfected BEAS2B cells were
immunoprecipitated with mAb against V5 (ThermoFisher Scientific, R96025) and
Sepharose A/G beads (ThermoFisher Scientific). In parallel, the membrane
proteins from the BEAS2B cells were immunoprecipitated with purified
MERS-CoV–S1–FLAG protein, anti-FLAG M2 antibody (Sigma, F1804),
and Sepharose A/G beads (ThermoFisher Scientific). Pulled down proteins reactive
to anti-V5 beads were washed and incubated with 0.1 m glycine, pH 3.5,
and those reactive to anti-FLAG M2 beads were eluted in 3× FLAG peptide
solution (Sigma, 150 ng/μl final concentration). Eluted samples were
spin-dialyzed in Amicon spin column with 10-kDa cutoff (Millipore) and separated
by SDS-PAGE, stained with SilverQuest kit (ThermoFisher Scientific). The gel
fragment was excised for LC-MS/MS analysis carried out in the Center for Genomic
Sciences, University of Hong Kong. MS/MS data were searched against all
mammalian protein databases in NCBI and Swiss-Prot. The protein was identified
as GRP78 with significant hits over different domains of the sequence.Production of pseudotyped virusesLentivirus-based coronavirus spike pseudoviruses were generated by
co-transfection of 293FT cells with the pcDNA full-length spike plasmids in
combination with the HIV-1 backbone plasmid-bearing luciferase reporter gene,
pNL4–3-ΔE-Luc (obtained from the AIDS Research and Reference
Reagent Program) using Lipofectamine 2000 (ThermoFisher Scientific). Cells
transfected overnight were replenished with fresh medium supplemented with 1
mm sodium pyruvate (ThermoFisher Scientific). Supernatants were
harvested 48 h post-transfection, filtered through a 0.45-μm syringe
filter, and concentrated by ultracentrifugation in 30% sucrose solution in a
Beckman rotor SW32Ti at 32,000 rpm for 1 h at 4 °C. The virus pellets were
resuspended in PBS, aliquoted, and stored at −80 °C. The p24
concentrations were quantified using a p24 enzyme-linked immunoassay kit (Cell
BioLabs). Pseudovirus titer was quantified in units of lentiviral particle (LP)
per ml according to the manufacturer's instruction.Luciferase activity assay for pseudovirus entryCoronavirus spike pseudoviruses were used to infect 5 × 103
target cells in white 96-well plates (Corning-Costar). After incubating the
cultures for 72 h at 37 °C, the cells were first washed with PBS
(ThermoFisher Scientific). The cells were then lysed with the lysis buffer
(Promega) on ice, and luciferase substrate (Promega) was then added immediately.
The infectivity was measured using a microplate reader (Beckman DTX880) as
relative light units (RLU). Uninfected cells were included as mock controls for
all experiments. All experiments were performed in triplicate and repeated at
least two times.Quantitative RT-PCRCells were lysed in RLT buffer with 40 mm DTT and extracted with the
RNeasy mini kit (Qiagen). Viral RNA in the supernatant was extracted with the
PureLink Viral RNA/DNA mini kit (Life Technologies, Inc.). Reverse transcription
(RT) and quantitative PCR (qPCR) were performed with the Transcriptor First
Strand cDNA synthesis kit and LightCycler 480 master mix from Roche Applied
Science as we previously described (34).
In the RT reactions, reverse primers against the N gene of MERS-CoV were used to
detect cDNA complementary to the positive strand of viral genomes. The following
sets of primers were used to detect N in qPCR: forward,
5′-CAAAACCTTCCCTAAGAAGGAAAAG-3′, reverse,
5′-GCTCCTTTGGAGGTTCAGACAT-3′, and probe (6-carboxyfluorescein),
5′-ACAAAAGGCACCAAAAGAAGAATCAACAGACC-3′ BHQ1.Antibody-blocking assay for HKU9–S-pseudovirus bindingRLK cells were pre-incubated with the rabbit anti-GRP78 antibody (Novus
Biologicals, NBP1-54318) or the rabbit control IgG (ThermoFisher Scientific,
31235) for 1 h at 37 °C. After the pre-incubation,
HKU9–S-pseudoviruses were inoculated to the cells for attachment at 100
LP per cell at 4 °C for 2 h. After 2 h of incubation, the cells were washed
twice with chilled PBS and fixed in 4% paraformaldehyde for 15 min. The fixed
cells were immunolabeled for the bCoV-HKU9 spike with the in-house mouse
bCoV-HKU9 spike immune serum and the mouse Alexa Fluor 488 goat anti-mouse
secondary antibody (ThermoFisher Scientific). The binding of the
HKU9–S-pseudoviruses was assessed with flow cytometry.Antibody-blocking assay for MERS-CoV entryHuh7 cells were pre-incubated with rabbit polyclonal anti-GRP78 (Novus
Biologicals, NBP1-54318) at different concentrations ranging from 0 to 5
μg/ml. Goat polyclonal anti-DPP4 at 5 μg/ml (R&D, AF1180) and
rabbit IgG at 5 μg/ml (ThermoFisher Scientific, 31235) were included as
controls. After pre-incubating with the antibodies for 1 h at 37 °C, the
cells were challenged with MERS-CoV at 1 m.o.i. for 1 h at 37 °C in the
presence of antibodies. The cells were subsequently washed with PBS and lysed
with RLT (Qiagen) with 40 mm dithiothreitol (DTT). The virus copy
number was quantified with qPCR as described previously (25).siRNA knockdown and virus entry assessmentON-TARGETplus human GRP78 siRNA (L-008198-00-0005) and ON-TARGETplus nontargeting
siRNA (L-001810-10-0020) were obtained from Dharmacon. Transfection of siRNA on
BEAS2B, Huh7, MDM, or HFL cells was performed using Lipofectamine RNAiMAX
(ThermoFisher Scientific) following the manufacturer's manual. In brief, the
cells were transfected with 75 nm siRNA for 2 consecutive days. At 24 h
after the second siRNA transfection, the cells were counted and harvested in
RIPA for Western blottings. In parallel, siRNA-transfected cells were challenged
with MERS-CoV at 1 m.o.i. for 1 h at 37 °C. Following the incubation, the
cells were washed with PBS and lysed in RLT buffer (Qiagen) with 40 mm
DTT. The virus copy number was determined with qPCR.Neuraminidase treatment and GRP78 antibody blocking for pseudovirus
entryHuh7 and RLK cells grown in 96-well plates were washed twice with PBS
(ThermoFisher Scientific) and incubated with neuraminidase from
Clostridium perfringens (Sigma) diluted in FBS-free growth
medium at 37 °C for 3 h. After the incubation, the cells were washed three
times and challenged with MERS–S- or HKU9–S-pseudoviruses, with or
without pre-incubation with the GRP78 polyclonal antibody (Abcam) for 1 h at 37
°C. Fresh complete medium with 10% FBS was replaced at 18 h post-infection.
Pseudovirus entry was quantified using a microplate reader (Beckman DTX880) as
RLU at 72 h post-infection.Flow cytometryImmunostaining for flow cytometry was performed following standard procedures as
we previously described (61). To
determine the surface-expression level of GRP78 and DPP4, the cells were
detached with 10 mm EDTA in PBS, fixed in 4% paraformaldehyde, followed
by immunolabeling with antibodies against GRP78 (Abcam, 21685) or DPP4 (Abcam,
28340) without cell permeabilization. For experiments with intracellular
stainings, cells were detached with 10 mm EDTA in PBS, fixed in 4%
paraformaldehyde, and permeabilized with 0.1% Triton X-100 in PBS. The flow
cytometry was performed using a FACSCanto II flow cytometer (BD Biosciences),
and data were analyzed using FlowJo version X (Tree Star).Flow cytometry of BHK21 and AD293 cells with GRP78 overexpressionAD293 and BHK21 cells were transfected with pcDNA–GRP78–V5 with
Lipofectamine 3000 (ThermoFisher Scientific). The transfected cells were
inoculated with MERS-CoV at 48 h post- transfection. To determine virus entry,
the cells were inoculated with MERS-CoV at 5 m.o.i. at 37 °C for 2 h. After
2 h, the cells were washed with PBS and incubated for another 4 h. At 6 h
post-infection, the cells were washed extensively with PBS, fixed in 4%
paraformaldehyde, and immunolabeled for flow cytometry. To determine virus
attachment, the cells were inoculated with MERS-CoV at 15 m.o.i. at 4 °C
for 2 h. After 2 h, the cells were washed with PBS, fixed in 4%
paraformaldehyde, and immunolabeled for flow cytometry.Confocal microscopy of human tissuesThis study was approved by the Institutional Review Board of the University of
Hong Kong/Hospital Authority Hong Kong West Cluster. Normal human lung sections
were deparaffinized and rehydrated following standard procedures. Antigen
unmasking was performed by boiling tissue sections with the antigen unmasking
solution from Vector Laboratories. Goat anti-DPP4 was obtained from R&D
Systems (AF1180) and rabbit anti-GRP78 was obtained from Abcam (ab21685). Cell
nuclei were labeled with the DAPI nucleic acid stain from ThermoFisher
Scientific (D21490). Alexa Fluor secondary antibodies were obtained from
ThermoFisher Scientific. Mounting was performed with the Vectashield mounting
medium (Vector Laboratories). Images were acquired with a Carl Zeiss LSM 710
system.Statistical analysisData on figures represent the means and standard deviations. Statistical
comparison between different groups was performed by Student's
t test using GraphPad Prism 6. Differences were considered
statistically significant when p < 0.05.

Experimental procedures

CellsA549, AD293 (a derivative of the commonly used HEK293 cell line, with improved
cell adherence), HeLa, Huh7, Caco2, and VeroE6 cells were maintained in
Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% heat-inactivated
fetal bovine serum (FBS), 100 units/ml penicillin, and 100 μg/ml
streptomycin. BEAS2B (transformed epithelial cells isolated from normal human
bronchial epithelium) and Calu3 cells were maintained in supplemented DMEM/F-12.
BHK21, L929, RLK, and HFL were maintained in supplemented minimum essential
medium. Human primary monocytes were obtained from human peripheral blood
mononuclear cells (PBMCs) as described previously (59). Primary human monocyte-derived macrophages (MDMs) were
differentiated from monocytes in Roswell Park Memorial Institute (RPMI)-1640
media supplemented with 10% FBS, 100 μg/ml streptomycin, 100 units/ml
penicillin, 2 mm glutamine, 1% sodium pyruvate, 1% nonessential amino
acids, and 10 ng/ml recombinant human granulocyte macrophage colony-stimulating
factor (GM-CSF) (R&D Systems) (60).
Human primary T cells were isolated from PBMCs with negative selection using the
Dynabeads Untouched Human T cells kit (Invitrogen) as we described previously
(61). Isolated T cells were
maintained in RPMI 1640 medium supplemented with 10% FBS, 100 μg/ml
streptomycin, 100 units/ml penicillin, 1% sodium pyruvate, and 1% nonessential
amino acids.

A549, AD293 (a derivative of the commonly used HEK293 cell line, with improved
cell adherence), HeLa, Huh7, Caco2, and VeroE6 cells were maintained in
Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% heat-inactivated
fetal bovine serum (FBS), 100 units/ml penicillin, and 100 μg/ml
streptomycin. BEAS2B (transformed epithelial cells isolated from normal human
bronchial epithelium) and Calu3 cells were maintained in supplemented DMEM/F-12.
BHK21, L929, RLK, and HFL were maintained in supplemented minimum essential
medium. Human primary monocytes were obtained from human peripheral blood
mononuclear cells (PBMCs) as described previously (59). Primary human monocyte-derived macrophages (MDMs) were
differentiated from monocytes in Roswell Park Memorial Institute (RPMI)-1640
media supplemented with 10% FBS, 100 μg/ml streptomycin, 100 units/ml
penicillin, 2 mm glutamine, 1% sodium pyruvate, 1% nonessential amino
acids, and 10 ng/ml recombinant human granulocyte macrophage colony-stimulating
factor (GM-CSF) (R&D Systems) (60).
Human primary T cells were isolated from PBMCs with negative selection using the
Dynabeads Untouched Human T cells kit (Invitrogen) as we described previously
(61). Isolated T cells were
maintained in RPMI 1640 medium supplemented with 10% FBS, 100 μg/ml
streptomycin, 100 units/ml penicillin, 1% sodium pyruvate, and 1% nonessential
amino acids.

VirusMERS-CoV was a gift from Dr. Ron Fouchier (Erasmus Medical Center, Rotterdam, the
Netherlands) and cultured in VeroE6 cells in serum-free DMEM. Virus titers were
quantified with plaque assays as described previously (25).

MERS-CoV was a gift from Dr. Ron Fouchier (Erasmus Medical Center, Rotterdam, the
Netherlands) and cultured in VeroE6 cells in serum-free DMEM. Virus titers were
quantified with plaque assays as described previously (25).

AntibodiesMERS-CoV nucleocapsid protein (N) was detected with the in-house guinea pig
anti-MERS–CoV N serum as we described previously (32, 62). The
MERS-CoV spike was detected with either the in-house mouse anti-MERS-CoV spike
immune serum or a rabbit anti-MERS-CoV spike antibody from Sino Biological
(40069-RP02). An in-house mouse anti-bCoV-HKU9-spike immune serum was used to
detect bCoV-HKU9 spike. Primary antibodies, including rabbit anti-DPP4
(ab28340), rabbit anti-GRP78 (ab21685), rabbit anti-pan-cadherin (ab16505),
rabbit anti-calreticulin (ab2907), rabbit anti-GM130 (ab52649), rabbit anti-EGFR
(ab52894), and rabbit anti-CEACAM5 (ab131070), were from Abcam. Rabbit
anti-lamin A was from Sigma (SAB4501764). Rabbit control IgG was from
ThermoFisher Scientific (31235). Mouse anti-GRP78 antibody for Western blotting
was from R&D Systems (MAB4846). Rabbit anti-GRP78 from Novus Biologicals
(NBP1-54318) and goat anti-CD26 from R&D Systems (AF1180) were used for
antibody-blocking experiments. Rabbit anti-giantin was from Biolegend
(A488-114L). The mouse anti-β-actin was from Sigma (A5441). The
recombinant FLAG-conjugated proteins were detected with an anti-FLAG M2 antibody
from Sigma (F1804). The V5-tagged proteins were detected with mouse anti-V5
antibodies from Immnoway (YM3005) or ThermoFisher Scientific (R96025). The
eGFP-tagged proteins were detected with a rabbit anti-eGFP from Abcam (ab290).
Secondary antibodies, including Alexa Fluor 488/647 goat anti-guinea pig
(A11073/A21450) and Alexa Fluor 488/647 goat anti-rabbit (A11008/A21245) from
ThermoFisher Scientific, were used for flow cytometry. The goat anti-mouse HRP
(626520) and goat anti-rabbit HRP (656120) antibodies from ThermoFisher
Scientific were used for Western blottings.

MERS-CoV nucleocapsid protein (N) was detected with the in-house guinea pig
anti-MERS–CoV N serum as we described previously (32, 62). The
MERS-CoV spike was detected with either the in-house mouse anti-MERS-CoV spike
immune serum or a rabbit anti-MERS-CoV spike antibody from Sino Biological
(40069-RP02). An in-house mouse anti-bCoV-HKU9-spike immune serum was used to
detect bCoV-HKU9 spike. Primary antibodies, including rabbit anti-DPP4
(ab28340), rabbit anti-GRP78 (ab21685), rabbit anti-pan-cadherin (ab16505),
rabbit anti-calreticulin (ab2907), rabbit anti-GM130 (ab52649), rabbit anti-EGFR
(ab52894), and rabbit anti-CEACAM5 (ab131070), were from Abcam. Rabbit
anti-lamin A was from Sigma (SAB4501764). Rabbit control IgG was from
ThermoFisher Scientific (31235). Mouse anti-GRP78 antibody for Western blotting
was from R&D Systems (MAB4846). Rabbit anti-GRP78 from Novus Biologicals
(NBP1-54318) and goat anti-CD26 from R&D Systems (AF1180) were used for
antibody-blocking experiments. Rabbit anti-giantin was from Biolegend
(A488-114L). The mouse anti-β-actin was from Sigma (A5441). The
recombinant FLAG-conjugated proteins were detected with an anti-FLAG M2 antibody
from Sigma (F1804). The V5-tagged proteins were detected with mouse anti-V5
antibodies from Immnoway (YM3005) or ThermoFisher Scientific (R96025). The
eGFP-tagged proteins were detected with a rabbit anti-eGFP from Abcam (ab290).
Secondary antibodies, including Alexa Fluor 488/647 goat anti-guinea pig
(A11073/A21450) and Alexa Fluor 488/647 goat anti-rabbit (A11008/A21245) from
ThermoFisher Scientific, were used for flow cytometry. The goat anti-mouse HRP
(626520) and goat anti-rabbit HRP (656120) antibodies from ThermoFisher
Scientific were used for Western blottings.

Plasmid constructionThe construction of pcDNA–MERS–CoV–S was previously
described (25). Codon-optimized
bCoV-HKU9-spike DNA was synthesized at GeneArt (ThermoFisher Scientific) based
on amino acid sequence of the bCoV-HKU9 (44) and cloned into the pcDNA3.1(+) vector. In parallel,
bCoV-HKU9–S1 was subcloned into the pSFV1 vector with a FLAG sequence
in-frame in the 3′-end for protein expression. The expression construct
for codon-optimized SARS-CoV spike, pcDNA-Sopt9, was a gift from Dr. Chen Zhiwei
and was previously described (63). The
construction of pSFV–MERS-CoV–S1–FLAG was previously
described (25). The same ORF was
PCR-amplified and subcloned into pcDNA3.1(+) vector in-frame with a V5 epitope,
which resulted in pcDNA–MERS-CoV–S1–V5. The GRP78-coding
region, including the N-terminal signal peptide, was obtained with RT-PCR from
BEAS2B cells and cloned into pcDNA3.1(+) vector fused with V5, which resulted in
pcDNA–GRP78–V5.

Plasmid construction

The construction of pcDNA–MERS–CoV–S was previously
described (25). Codon-optimized
bCoV-HKU9-spike DNA was synthesized at GeneArt (ThermoFisher Scientific) based
on amino acid sequence of the bCoV-HKU9 (44) and cloned into the pcDNA3.1(+) vector. In parallel,
bCoV-HKU9–S1 was subcloned into the pSFV1 vector with a FLAG sequence
in-frame in the 3′-end for protein expression. The expression construct
for codon-optimized SARS-CoV spike, pcDNA-Sopt9, was a gift from Dr. Chen Zhiwei
and was previously described (63). The
construction of pSFV–MERS-CoV–S1–FLAG was previously
described (25). The same ORF was
PCR-amplified and subcloned into pcDNA3.1(+) vector in-frame with a V5 epitope,
which resulted in pcDNA–MERS-CoV–S1–V5. The GRP78-coding
region, including the N-terminal signal peptide, was obtained with RT-PCR from
BEAS2B cells and cloned into pcDNA3.1(+) vector fused with V5, which resulted in
pcDNA–GRP78–V5.

Immunoaffinity purification of MERS-CoV–S1–FLAG proteinExpression of FLAG-tagged recombinant proteins was previously described (64, 65). In brief, linearized pSFV–FLAG plasmids were transcribed
in vitro, and the derived capped RNAs were electroporated
into BHK21 cells. At 15 h post-transfection, the cells were lysed, and the
expressed recombinant proteins were immunopurified using anti-FLAG
M2–coated beads (Sigma) according to the manufacturer's instructions. The
purified S protein was assessed by SDS-PAGE and Western blotting. Protein
concentration was quantified with the Pierce BCA assay (ThermoFisher
Scientific).

Immunoaffinity purification of MERS-CoV–S1–FLAG protein

Expression of FLAG-tagged recombinant proteins was previously described (64, 65). In brief, linearized pSFV–FLAG plasmids were transcribed
in vitro, and the derived capped RNAs were electroporated
into BHK21 cells. At 15 h post-transfection, the cells were lysed, and the
expressed recombinant proteins were immunopurified using anti-FLAG
M2–coated beads (Sigma) according to the manufacturer's instructions. The
purified S protein was assessed by SDS-PAGE and Western blotting. Protein
concentration was quantified with the Pierce BCA assay (ThermoFisher
Scientific).

Membrane extractionpcDNA–MERS-CoV–S1–V5–transfected BEAS2B cells
cultured in 10-cm dishes were harvested by scraping cells into HEPES solution
(10 mm HEPES, pH 7.5, 1.5 mm MgCl2, 1 mm
KCl) and centrifuging briefly at 500 × g for 3 min. Cell
pellets were then homogenized in membrane lysis buffer (20 mm Tris-HCl,
pH 7.5, 150 mm NaCl, 1% Nonidet P-40 (Calbiochem), 1%
n-dodecyl β-maltoside (ThermoFisher Scientific), 5%
glycerol, pH 7.5, with protease inhibitor mixture (Roche Applied Science)) and
incubated on ice for 30 min. Residual cellular debris and nuclei in the
resulting extracts were sedimented by centrifugation at 4 °C for 5 min at
6000 × g. The solubilized membrane proteins in the
supernatant were transferred and subjected to an additional spin at 16,000
× g for 30 min at 4 °C, and the membrane extracts
were then resuspended in lysis buffer. All extracts were quantitated using the
Pierce BCA assay kit (ThermoFisher Scientific) and stored in aliquots at
−80 °C until used.

Membrane extraction

pcDNA–MERS-CoV–S1–V5–transfected BEAS2B cells
cultured in 10-cm dishes were harvested by scraping cells into HEPES solution
(10 mm HEPES, pH 7.5, 1.5 mm MgCl2, 1 mm
KCl) and centrifuging briefly at 500 × g for 3 min. Cell
pellets were then homogenized in membrane lysis buffer (20 mm Tris-HCl,
pH 7.5, 150 mm NaCl, 1% Nonidet P-40 (Calbiochem), 1%
n-dodecyl β-maltoside (ThermoFisher Scientific), 5%
glycerol, pH 7.5, with protease inhibitor mixture (Roche Applied Science)) and
incubated on ice for 30 min. Residual cellular debris and nuclei in the
resulting extracts were sedimented by centrifugation at 4 °C for 5 min at
6000 × g. The solubilized membrane proteins in the
supernatant were transferred and subjected to an additional spin at 16,000
× g for 30 min at 4 °C, and the membrane extracts
were then resuspended in lysis buffer. All extracts were quantitated using the
Pierce BCA assay kit (ThermoFisher Scientific) and stored in aliquots at
−80 °C until used.

Identification of GRP78 by immunoprecipitation and MSMembrane proteins from
pcDNA–MERS-CoV–S1–V5–transfected BEAS2B cells were
immunoprecipitated with mAb against V5 (ThermoFisher Scientific, R96025) and
Sepharose A/G beads (ThermoFisher Scientific). In parallel, the membrane
proteins from the BEAS2B cells were immunoprecipitated with purified
MERS-CoV–S1–FLAG protein, anti-FLAG M2 antibody (Sigma, F1804),
and Sepharose A/G beads (ThermoFisher Scientific). Pulled down proteins reactive
to anti-V5 beads were washed and incubated with 0.1 m glycine, pH 3.5,
and those reactive to anti-FLAG M2 beads were eluted in 3× FLAG peptide
solution (Sigma, 150 ng/μl final concentration). Eluted samples were
spin-dialyzed in Amicon spin column with 10-kDa cutoff (Millipore) and separated
by SDS-PAGE, stained with SilverQuest kit (ThermoFisher Scientific). The gel
fragment was excised for LC-MS/MS analysis carried out in the Center for Genomic
Sciences, University of Hong Kong. MS/MS data were searched against all
mammalian protein databases in NCBI and Swiss-Prot. The protein was identified
as GRP78 with significant hits over different domains of the sequence.

Identification of GRP78 by immunoprecipitation and MS

Membrane proteins from
pcDNA–MERS-CoV–S1–V5–transfected BEAS2B cells were
immunoprecipitated with mAb against V5 (ThermoFisher Scientific, R96025) and
Sepharose A/G beads (ThermoFisher Scientific). In parallel, the membrane
proteins from the BEAS2B cells were immunoprecipitated with purified
MERS-CoV–S1–FLAG protein, anti-FLAG M2 antibody (Sigma, F1804),
and Sepharose A/G beads (ThermoFisher Scientific). Pulled down proteins reactive
to anti-V5 beads were washed and incubated with 0.1 m glycine, pH 3.5,
and those reactive to anti-FLAG M2 beads were eluted in 3× FLAG peptide
solution (Sigma, 150 ng/μl final concentration). Eluted samples were
spin-dialyzed in Amicon spin column with 10-kDa cutoff (Millipore) and separated
by SDS-PAGE, stained with SilverQuest kit (ThermoFisher Scientific). The gel
fragment was excised for LC-MS/MS analysis carried out in the Center for Genomic
Sciences, University of Hong Kong. MS/MS data were searched against all
mammalian protein databases in NCBI and Swiss-Prot. The protein was identified
as GRP78 with significant hits over different domains of the sequence.

Production of pseudotyped virusesLentivirus-based coronavirus spike pseudoviruses were generated by
co-transfection of 293FT cells with the pcDNA full-length spike plasmids in
combination with the HIV-1 backbone plasmid-bearing luciferase reporter gene,
pNL4–3-ΔE-Luc (obtained from the AIDS Research and Reference
Reagent Program) using Lipofectamine 2000 (ThermoFisher Scientific). Cells
transfected overnight were replenished with fresh medium supplemented with 1
mm sodium pyruvate (ThermoFisher Scientific). Supernatants were
harvested 48 h post-transfection, filtered through a 0.45-μm syringe
filter, and concentrated by ultracentrifugation in 30% sucrose solution in a
Beckman rotor SW32Ti at 32,000 rpm for 1 h at 4 °C. The virus pellets were
resuspended in PBS, aliquoted, and stored at −80 °C. The p24
concentrations were quantified using a p24 enzyme-linked immunoassay kit (Cell
BioLabs). Pseudovirus titer was quantified in units of lentiviral particle (LP)
per ml according to the manufacturer's instruction.

Production of pseudotyped viruses

Lentivirus-based coronavirus spike pseudoviruses were generated by
co-transfection of 293FT cells with the pcDNA full-length spike plasmids in
combination with the HIV-1 backbone plasmid-bearing luciferase reporter gene,
pNL4–3-ΔE-Luc (obtained from the AIDS Research and Reference
Reagent Program) using Lipofectamine 2000 (ThermoFisher Scientific). Cells
transfected overnight were replenished with fresh medium supplemented with 1
mm sodium pyruvate (ThermoFisher Scientific). Supernatants were
harvested 48 h post-transfection, filtered through a 0.45-μm syringe
filter, and concentrated by ultracentrifugation in 30% sucrose solution in a
Beckman rotor SW32Ti at 32,000 rpm for 1 h at 4 °C. The virus pellets were
resuspended in PBS, aliquoted, and stored at −80 °C. The p24
concentrations were quantified using a p24 enzyme-linked immunoassay kit (Cell
BioLabs). Pseudovirus titer was quantified in units of lentiviral particle (LP)
per ml according to the manufacturer's instruction.

Luciferase activity assay for pseudovirus entryCoronavirus spike pseudoviruses were used to infect 5 × 103
target cells in white 96-well plates (Corning-Costar). After incubating the
cultures for 72 h at 37 °C, the cells were first washed with PBS
(ThermoFisher Scientific). The cells were then lysed with the lysis buffer
(Promega) on ice, and luciferase substrate (Promega) was then added immediately.
The infectivity was measured using a microplate reader (Beckman DTX880) as
relative light units (RLU). Uninfected cells were included as mock controls for
all experiments. All experiments were performed in triplicate and repeated at
least two times.

Luciferase activity assay for pseudovirus entry

Coronavirus spike pseudoviruses were used to infect 5 × 103
target cells in white 96-well plates (Corning-Costar). After incubating the
cultures for 72 h at 37 °C, the cells were first washed with PBS
(ThermoFisher Scientific). The cells were then lysed with the lysis buffer
(Promega) on ice, and luciferase substrate (Promega) was then added immediately.
The infectivity was measured using a microplate reader (Beckman DTX880) as
relative light units (RLU). Uninfected cells were included as mock controls for
all experiments. All experiments were performed in triplicate and repeated at
least two times.

Quantitative RT-PCRCells were lysed in RLT buffer with 40 mm DTT and extracted with the
RNeasy mini kit (Qiagen). Viral RNA in the supernatant was extracted with the
PureLink Viral RNA/DNA mini kit (Life Technologies, Inc.). Reverse transcription
(RT) and quantitative PCR (qPCR) were performed with the Transcriptor First
Strand cDNA synthesis kit and LightCycler 480 master mix from Roche Applied
Science as we previously described (34).
In the RT reactions, reverse primers against the N gene of MERS-CoV were used to
detect cDNA complementary to the positive strand of viral genomes. The following
sets of primers were used to detect N in qPCR: forward,
5′-CAAAACCTTCCCTAAGAAGGAAAAG-3′, reverse,
5′-GCTCCTTTGGAGGTTCAGACAT-3′, and probe (6-carboxyfluorescein),
5′-ACAAAAGGCACCAAAAGAAGAATCAACAGACC-3′ BHQ1.

Quantitative RT-PCR

Cells were lysed in RLT buffer with 40 mm DTT and extracted with the
RNeasy mini kit (Qiagen). Viral RNA in the supernatant was extracted with the
PureLink Viral RNA/DNA mini kit (Life Technologies, Inc.). Reverse transcription
(RT) and quantitative PCR (qPCR) were performed with the Transcriptor First
Strand cDNA synthesis kit and LightCycler 480 master mix from Roche Applied
Science as we previously described (34).
In the RT reactions, reverse primers against the N gene of MERS-CoV were used to
detect cDNA complementary to the positive strand of viral genomes. The following
sets of primers were used to detect N in qPCR: forward,
5′-CAAAACCTTCCCTAAGAAGGAAAAG-3′, reverse,
5′-GCTCCTTTGGAGGTTCAGACAT-3′, and probe (6-carboxyfluorescein),
5′-ACAAAAGGCACCAAAAGAAGAATCAACAGACC-3′ BHQ1.

Antibody-blocking assay for HKU9–S-pseudovirus bindingRLK cells were pre-incubated with the rabbit anti-GRP78 antibody (Novus
Biologicals, NBP1-54318) or the rabbit control IgG (ThermoFisher Scientific,
31235) for 1 h at 37 °C. After the pre-incubation,
HKU9–S-pseudoviruses were inoculated to the cells for attachment at 100
LP per cell at 4 °C for 2 h. After 2 h of incubation, the cells were washed
twice with chilled PBS and fixed in 4% paraformaldehyde for 15 min. The fixed
cells were immunolabeled for the bCoV-HKU9 spike with the in-house mouse
bCoV-HKU9 spike immune serum and the mouse Alexa Fluor 488 goat anti-mouse
secondary antibody (ThermoFisher Scientific). The binding of the
HKU9–S-pseudoviruses was assessed with flow cytometry.

Antibody-blocking assay for HKU9–S-pseudovirus binding

RLK cells were pre-incubated with the rabbit anti-GRP78 antibody (Novus
Biologicals, NBP1-54318) or the rabbit control IgG (ThermoFisher Scientific,
31235) for 1 h at 37 °C. After the pre-incubation,
HKU9–S-pseudoviruses were inoculated to the cells for attachment at 100
LP per cell at 4 °C for 2 h. After 2 h of incubation, the cells were washed
twice with chilled PBS and fixed in 4% paraformaldehyde for 15 min. The fixed
cells were immunolabeled for the bCoV-HKU9 spike with the in-house mouse
bCoV-HKU9 spike immune serum and the mouse Alexa Fluor 488 goat anti-mouse
secondary antibody (ThermoFisher Scientific). The binding of the
HKU9–S-pseudoviruses was assessed with flow cytometry.

Antibody-blocking assay for MERS-CoV entryHuh7 cells were pre-incubated with rabbit polyclonal anti-GRP78 (Novus
Biologicals, NBP1-54318) at different concentrations ranging from 0 to 5
μg/ml. Goat polyclonal anti-DPP4 at 5 μg/ml (R&D, AF1180) and
rabbit IgG at 5 μg/ml (ThermoFisher Scientific, 31235) were included as
controls. After pre-incubating with the antibodies for 1 h at 37 °C, the
cells were challenged with MERS-CoV at 1 m.o.i. for 1 h at 37 °C in the
presence of antibodies. The cells were subsequently washed with PBS and lysed
with RLT (Qiagen) with 40 mm dithiothreitol (DTT). The virus copy
number was quantified with qPCR as described previously (25).

Antibody-blocking assay for MERS-CoV entry

Huh7 cells were pre-incubated with rabbit polyclonal anti-GRP78 (Novus
Biologicals, NBP1-54318) at different concentrations ranging from 0 to 5
μg/ml. Goat polyclonal anti-DPP4 at 5 μg/ml (R&D, AF1180) and
rabbit IgG at 5 μg/ml (ThermoFisher Scientific, 31235) were included as
controls. After pre-incubating with the antibodies for 1 h at 37 °C, the
cells were challenged with MERS-CoV at 1 m.o.i. for 1 h at 37 °C in the
presence of antibodies. The cells were subsequently washed with PBS and lysed
with RLT (Qiagen) with 40 mm dithiothreitol (DTT). The virus copy
number was quantified with qPCR as described previously (25).

siRNA knockdown and virus entry assessmentON-TARGETplus human GRP78 siRNA (L-008198-00-0005) and ON-TARGETplus nontargeting
siRNA (L-001810-10-0020) were obtained from Dharmacon. Transfection of siRNA on
BEAS2B, Huh7, MDM, or HFL cells was performed using Lipofectamine RNAiMAX
(ThermoFisher Scientific) following the manufacturer's manual. In brief, the
cells were transfected with 75 nm siRNA for 2 consecutive days. At 24 h
after the second siRNA transfection, the cells were counted and harvested in
RIPA for Western blottings. In parallel, siRNA-transfected cells were challenged
with MERS-CoV at 1 m.o.i. for 1 h at 37 °C. Following the incubation, the
cells were washed with PBS and lysed in RLT buffer (Qiagen) with 40 mm
DTT. The virus copy number was determined with qPCR.

siRNA knockdown and virus entry assessment

ON-TARGETplus human GRP78 siRNA (L-008198-00-0005) and ON-TARGETplus nontargeting
siRNA (L-001810-10-0020) were obtained from Dharmacon. Transfection of siRNA on
BEAS2B, Huh7, MDM, or HFL cells was performed using Lipofectamine RNAiMAX
(ThermoFisher Scientific) following the manufacturer's manual. In brief, the
cells were transfected with 75 nm siRNA for 2 consecutive days. At 24 h
after the second siRNA transfection, the cells were counted and harvested in
RIPA for Western blottings. In parallel, siRNA-transfected cells were challenged
with MERS-CoV at 1 m.o.i. for 1 h at 37 °C. Following the incubation, the
cells were washed with PBS and lysed in RLT buffer (Qiagen) with 40 mm
DTT. The virus copy number was determined with qPCR.

Neuraminidase treatment and GRP78 antibody blocking for pseudovirus
entryHuh7 and RLK cells grown in 96-well plates were washed twice with PBS
(ThermoFisher Scientific) and incubated with neuraminidase from
Clostridium perfringens (Sigma) diluted in FBS-free growth
medium at 37 °C for 3 h. After the incubation, the cells were washed three
times and challenged with MERS–S- or HKU9–S-pseudoviruses, with or
without pre-incubation with the GRP78 polyclonal antibody (Abcam) for 1 h at 37
°C. Fresh complete medium with 10% FBS was replaced at 18 h post-infection.
Pseudovirus entry was quantified using a microplate reader (Beckman DTX880) as
RLU at 72 h post-infection.

Neuraminidase treatment and GRP78 antibody blocking for pseudovirus
entry

Huh7 and RLK cells grown in 96-well plates were washed twice with PBS
(ThermoFisher Scientific) and incubated with neuraminidase from
Clostridium perfringens (Sigma) diluted in FBS-free growth
medium at 37 °C for 3 h. After the incubation, the cells were washed three
times and challenged with MERS–S- or HKU9–S-pseudoviruses, with or
without pre-incubation with the GRP78 polyclonal antibody (Abcam) for 1 h at 37
°C. Fresh complete medium with 10% FBS was replaced at 18 h post-infection.
Pseudovirus entry was quantified using a microplate reader (Beckman DTX880) as
RLU at 72 h post-infection.

Flow cytometryImmunostaining for flow cytometry was performed following standard procedures as
we previously described (61). To
determine the surface-expression level of GRP78 and DPP4, the cells were
detached with 10 mm EDTA in PBS, fixed in 4% paraformaldehyde, followed
by immunolabeling with antibodies against GRP78 (Abcam, 21685) or DPP4 (Abcam,
28340) without cell permeabilization. For experiments with intracellular
stainings, cells were detached with 10 mm EDTA in PBS, fixed in 4%
paraformaldehyde, and permeabilized with 0.1% Triton X-100 in PBS. The flow
cytometry was performed using a FACSCanto II flow cytometer (BD Biosciences),
and data were analyzed using FlowJo version X (Tree Star).

Flow cytometry

Immunostaining for flow cytometry was performed following standard procedures as
we previously described (61). To
determine the surface-expression level of GRP78 and DPP4, the cells were
detached with 10 mm EDTA in PBS, fixed in 4% paraformaldehyde, followed
by immunolabeling with antibodies against GRP78 (Abcam, 21685) or DPP4 (Abcam,
28340) without cell permeabilization. For experiments with intracellular
stainings, cells were detached with 10 mm EDTA in PBS, fixed in 4%
paraformaldehyde, and permeabilized with 0.1% Triton X-100 in PBS. The flow
cytometry was performed using a FACSCanto II flow cytometer (BD Biosciences),
and data were analyzed using FlowJo version X (Tree Star).

Flow cytometry of BHK21 and AD293 cells with GRP78 overexpressionAD293 and BHK21 cells were transfected with pcDNA–GRP78–V5 with
Lipofectamine 3000 (ThermoFisher Scientific). The transfected cells were
inoculated with MERS-CoV at 48 h post- transfection. To determine virus entry,
the cells were inoculated with MERS-CoV at 5 m.o.i. at 37 °C for 2 h. After
2 h, the cells were washed with PBS and incubated for another 4 h. At 6 h
post-infection, the cells were washed extensively with PBS, fixed in 4%
paraformaldehyde, and immunolabeled for flow cytometry. To determine virus
attachment, the cells were inoculated with MERS-CoV at 15 m.o.i. at 4 °C
for 2 h. After 2 h, the cells were washed with PBS, fixed in 4%
paraformaldehyde, and immunolabeled for flow cytometry.

Flow cytometry of BHK21 and AD293 cells with GRP78 overexpression

AD293 and BHK21 cells were transfected with pcDNA–GRP78–V5 with
Lipofectamine 3000 (ThermoFisher Scientific). The transfected cells were
inoculated with MERS-CoV at 48 h post- transfection. To determine virus entry,
the cells were inoculated with MERS-CoV at 5 m.o.i. at 37 °C for 2 h. After
2 h, the cells were washed with PBS and incubated for another 4 h. At 6 h
post-infection, the cells were washed extensively with PBS, fixed in 4%
paraformaldehyde, and immunolabeled for flow cytometry. To determine virus
attachment, the cells were inoculated with MERS-CoV at 15 m.o.i. at 4 °C
for 2 h. After 2 h, the cells were washed with PBS, fixed in 4%
paraformaldehyde, and immunolabeled for flow cytometry.

Confocal microscopy of human tissuesThis study was approved by the Institutional Review Board of the University of
Hong Kong/Hospital Authority Hong Kong West Cluster. Normal human lung sections
were deparaffinized and rehydrated following standard procedures. Antigen
unmasking was performed by boiling tissue sections with the antigen unmasking
solution from Vector Laboratories. Goat anti-DPP4 was obtained from R&D
Systems (AF1180) and rabbit anti-GRP78 was obtained from Abcam (ab21685). Cell
nuclei were labeled with the DAPI nucleic acid stain from ThermoFisher
Scientific (D21490). Alexa Fluor secondary antibodies were obtained from
ThermoFisher Scientific. Mounting was performed with the Vectashield mounting
medium (Vector Laboratories). Images were acquired with a Carl Zeiss LSM 710
system.

Confocal microscopy of human tissues

This study was approved by the Institutional Review Board of the University of
Hong Kong/Hospital Authority Hong Kong West Cluster. Normal human lung sections
were deparaffinized and rehydrated following standard procedures. Antigen
unmasking was performed by boiling tissue sections with the antigen unmasking
solution from Vector Laboratories. Goat anti-DPP4 was obtained from R&D
Systems (AF1180) and rabbit anti-GRP78 was obtained from Abcam (ab21685). Cell
nuclei were labeled with the DAPI nucleic acid stain from ThermoFisher
Scientific (D21490). Alexa Fluor secondary antibodies were obtained from
ThermoFisher Scientific. Mounting was performed with the Vectashield mounting
medium (Vector Laboratories). Images were acquired with a Carl Zeiss LSM 710
system.

Statistical analysisData on figures represent the means and standard deviations. Statistical
comparison between different groups was performed by Student's
t test using GraphPad Prism 6. Differences were considered
statistically significant when p < 0.05.

Statistical analysis

Data on figures represent the means and standard deviations. Statistical
comparison between different groups was performed by Student's
t test using GraphPad Prism 6. Differences were considered
statistically significant when p < 0.05.
