---
pmid: '23474713'
title: Structure of active dimeric human telomerase.
authors:
- Sauerwald A
- Sandin S
- Cristofari G
- Scheres SH
- Lingner J
- Rhodes D
journal: Nat Struct Mol Biol
year: '2013'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3785136
doi: 10.1038/nsmb.2530
---

# Structure of active dimeric human telomerase.
**Authors:** Sauerwald A, Sandin S, Cristofari G, Scheres SH, Lingner J, Rhodes D
**Journal:** Nat Struct Mol Biol (2013)
**DOI:** [10.1038/nsmb.2530](https://doi.org/10.1038/nsmb.2530)
**PMC:** [PMC3785136](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3785136/)

## Abstract

1. Nat Struct Mol Biol. 2013 Apr;20(4):454-60. doi: 10.1038/nsmb.2530. Epub 2013 
Mar 10.

Structure of active dimeric human telomerase.

Sauerwald A(1), Sandin S, Cristofari G, Scheres SH, Lingner J, Rhodes D.

Author information:
(1)Medical Research Council Laboratory of Molecular Biology, Cambridge, UK.

Telomerase contains a large RNA subunit, TER, and a protein catalytic subunit, 
TERT. Whether telomerase functions as a monomer or dimer has been a matter of 
debate. Here we report biochemical and labeling data that show that in 
vivo-assembled human telomerase contains two TERT subunits and binds two 
telomeric DNA substrates. Notably, catalytic activity requires both TERT active 
sites to be functional, which demonstrates that human telomerase functions as a 
dimer. We also present the three-dimensional structure of the active full-length 
human telomerase dimer, determined by single-particle EM in negative stain. 
Telomerase has a bilobal architecture with the two monomers linked by a flexible 
interface. The monomer reconstruction at 23-Å resolution and fitting of the 
atomic structure of the TERT subunit from beetle Tribolium castaneum into the EM 
density reveals the spatial relationship between RNA and protein subunits, 
providing insights into telomerase architecture.

DOI: 10.1038/nsmb.2530
PMCID: PMC3785136
PMID: 23474713 [Indexed for MEDLINE]

## Full Text

IntroductionTelomeres, the protein/DNA complexes that cap the ends of eukaryotic chromosomes are essential for genomic stability and cell viability1. In eukaryotes, telomere length is maintained by the telomerase enzyme, a specialized reverse transcriptase capable of de novo DNA synthesis. Telomere maintenance in the majority of cancer cells involves the activation of telomerase2. Originally discovered by Greider and Blackburn in 1985 in the ciliate T. thermophila3, the telomerase enzyme consists of an RNA subunit TER that contains an internal template for telomeric DNA-repeat synthesis4 and a protein subunit TERT that contains a reverse transcriptase domain that catalyzes nucleotide addition5,6. The telomerase is a processive enzyme: In the first step of G-overhang synthesis, the 3′ end of the telomeric primer, the G-overhang, is positioned in the active site of the telomerase forming base pairs with the alignment region of the RNA template. In a second elongation step, nucleotide addition takes place to synthetize the telomeric DNA repeat, and in a third step the telomerase translocates to restart telomere repeat synthesis7. To accomplish all these catalytic steps, the telomerase enzyme would likely have to undergo substantial conformational changes.Together TER and TERT form a tight complex that is sufficient for telomeric DNA repeat synthesis in vitro8, whilst accessory proteins involved in assembly and localization have been found associated with the telomerase holoenzyme isolated from cells9-12. Whether human telomerase functions as a monomer13 or a dimer10,14-17 is a matter of debate10,18. In vitro reconstitution experiments demonstrated that telomerase contains two telomerase RNA moieties14. However, it was suggested that in vitro reconstitution may lead to formation of non-native multi-subunit complexes and that in vivo assembled human telomerase was monomeric13.A full understanding of telomerase function requires information on the three-dimensional structure of the telomerase. This has been hindered by the difficulty in obtaining sufficient quantities of purified, active and full-length telomerase complex. Therefore, in contrast to other important enzymes, such as the ribosome and RNA/DNA polymerases, knowledge of the telomerase structure has been limited to subdomains of the TER and TERT subunits19-21. Amino acid sequence analysis and mutational studies have established that the TERT subunit contains three domains: an N-terminal G-overhang binding (TEN) domain, the RNA binding domain (TRBD) and a C-terminal domain consisting of the reverse transcriptase (RT) domain (Fig.1a)7,20.The most comprehensive structural information on the TERT subunit comes from the crystal structure of the beetle T. castaneum TERT in complex with a short DNA/RNA helix22,23 confirming a structural conservation with other RTs such as HIV RT24 and revealing the relative location of the TRBD with respect to the RT domains within TERT25. This beetle TERT subunit lacks the TEN domain present at the N-terminus of most TERTs, including hTERT, but the structure of this domain is known from the crystal structure of the isolated TEN domain of the ciliate T. thermophila26.Although the length of the RNA subunit is less well conserved than the protein subunit (from 147 nucleotides in protozoa to almost 2000 nucleotides in budding yeasts), phylogenetic and functional studies have revealed that all TERs contain two conserved structural elements: the catalytically essential pseudoknot/template core domain and a stem terminus element called CR4/CR5 in vertebrates27, both of which have been shown to interact with TERT28,29. Structural information on the TER subunit is restricted to isolated segments of the RNA, including the functionally important pseudoknot21. Despite the structural information on TER and TERT sub-domains, little is known about the overall structure of human telomerase and how the various domains contribute to the architecture of full-length telomerase to form a functional enzyme. Here we present the first three-dimensional structural analysis of active, full-length human telomerase by single particle electron microscopy which reveals a dimeric bilobal structure, as well as experimental evidence that the two TERT subunits in the telomerase dimer cooperate in catalytic activity.

Introduction

Telomeres, the protein/DNA complexes that cap the ends of eukaryotic chromosomes are essential for genomic stability and cell viability1. In eukaryotes, telomere length is maintained by the telomerase enzyme, a specialized reverse transcriptase capable of de novo DNA synthesis. Telomere maintenance in the majority of cancer cells involves the activation of telomerase2. Originally discovered by Greider and Blackburn in 1985 in the ciliate T. thermophila3, the telomerase enzyme consists of an RNA subunit TER that contains an internal template for telomeric DNA-repeat synthesis4 and a protein subunit TERT that contains a reverse transcriptase domain that catalyzes nucleotide addition5,6. The telomerase is a processive enzyme: In the first step of G-overhang synthesis, the 3′ end of the telomeric primer, the G-overhang, is positioned in the active site of the telomerase forming base pairs with the alignment region of the RNA template. In a second elongation step, nucleotide addition takes place to synthetize the telomeric DNA repeat, and in a third step the telomerase translocates to restart telomere repeat synthesis7. To accomplish all these catalytic steps, the telomerase enzyme would likely have to undergo substantial conformational changes.

Together TER and TERT form a tight complex that is sufficient for telomeric DNA repeat synthesis in vitro8, whilst accessory proteins involved in assembly and localization have been found associated with the telomerase holoenzyme isolated from cells9-12. Whether human telomerase functions as a monomer13 or a dimer10,14-17 is a matter of debate10,18. In vitro reconstitution experiments demonstrated that telomerase contains two telomerase RNA moieties14. However, it was suggested that in vitro reconstitution may lead to formation of non-native multi-subunit complexes and that in vivo assembled human telomerase was monomeric13.

A full understanding of telomerase function requires information on the three-dimensional structure of the telomerase. This has been hindered by the difficulty in obtaining sufficient quantities of purified, active and full-length telomerase complex. Therefore, in contrast to other important enzymes, such as the ribosome and RNA/DNA polymerases, knowledge of the telomerase structure has been limited to subdomains of the TER and TERT subunits19-21. Amino acid sequence analysis and mutational studies have established that the TERT subunit contains three domains: an N-terminal G-overhang binding (TEN) domain, the RNA binding domain (TRBD) and a C-terminal domain consisting of the reverse transcriptase (RT) domain (Fig.1a)7,20.The most comprehensive structural information on the TERT subunit comes from the crystal structure of the beetle T. castaneum TERT in complex with a short DNA/RNA helix22,23 confirming a structural conservation with other RTs such as HIV RT24 and revealing the relative location of the TRBD with respect to the RT domains within TERT25. This beetle TERT subunit lacks the TEN domain present at the N-terminus of most TERTs, including hTERT, but the structure of this domain is known from the crystal structure of the isolated TEN domain of the ciliate T. thermophila26.

Although the length of the RNA subunit is less well conserved than the protein subunit (from 147 nucleotides in protozoa to almost 2000 nucleotides in budding yeasts), phylogenetic and functional studies have revealed that all TERs contain two conserved structural elements: the catalytically essential pseudoknot/template core domain and a stem terminus element called CR4/CR5 in vertebrates27, both of which have been shown to interact with TERT28,29. Structural information on the TER subunit is restricted to isolated segments of the RNA, including the functionally important pseudoknot21. Despite the structural information on TER and TERT sub-domains, little is known about the overall structure of human telomerase and how the various domains contribute to the architecture of full-length telomerase to form a functional enzyme. Here we present the first three-dimensional structural analysis of active, full-length human telomerase by single particle electron microscopy which reveals a dimeric bilobal structure, as well as experimental evidence that the two TERT subunits in the telomerase dimer cooperate in catalytic activity.

ResultsIn vivo assembled human telomerase is a dimerTo obtain sufficient quantities of purified, active human telomerase for structural analyses, we made use of HEK293T ‘super-telomerase’ cells which over-express hTER and hTERT from transiently transfected plasmid constructs30, and have 200-fold higher telomerase activity than untransfected HEK293T cells31. The purification exploits the protein-A tag cloned at the N-terminus of hTERT, followed by affinity chromatography and sucrose gradient fractionation. To facilitate the tracking of the telomerase in the sucrose gradient, the telomerase was incubated with a [32P] 5′ labeled G-overhang oligonucleotide (5′-(TTAGGG)2-3′) prior to the sedimentation. Figure 1b shows a well resolved peak that sediments in the same position as the 670 kDa thyroglobulin size marker analysed in a parallel sucrose gradient (fig. S1c). Furthermore, tracking of the telomerase using a direct telomerase activity assay 10 shows that the peak of activity coincides with the position of the 670 kDa peak in the sucrose gradient (Fig. 1b. and 1c). The purified telomerase sample was further analysed for protein content by SDS-PAGE, which shows a band with the MW of hTERT (fig. S1a). Incubation of purified telomerase with 5′ [32P] labeled G-overhang oligonucleotide (5′-(TTAGGG)3-3′) followed by native PAGE shows that the telomerase complex migrates as a very tight band, and hence is of homogeneous MW (fig. S1b). The composition of the purified telomerase sample was analysed by mass spectroscopy. This analysis (table S1) reveals that the purified telomerase contains the hTERT subunits and two accessory proteins, Nop10 and dyskerin32. In conclusion, our data show that the purified human telomerase is active and that both the G-overhang bound and unbound telomerase complexes have a molecular weight consistent with that of a dimer consisting of two hTERT (127 kDa) and two hTER (153 kDa) subunits, as well as the two accessory proteins Nop10 (7.7 kDa) and dyskerin (58 kDA). This observation is in agreement with findings from a previous purification of the human telomerase complex from untransfected HEK239T cells10.Figure 1D shows an electron micrograph of purified, in vivo assembled human telomerase bound to the G-overhang oligonucleotide 5′-(TTAGGG)2-3′. Inspection of the micrograph shows side views of many particles with a bilobal structure. Two- and three-dimensional structural information was obtained by single particle classification (Fig. 1e) and electron tomography (Fig. 1f) respectively (For image processing procedures, see Supplementary Material). These analyses reveal that the telomerase enzyme has an elongated structure, 280 Å in length and 125 Å in width, consisting of two spherical lobes connected by a region of thin density. The size of the telomerase particle is consistent with the estimated molecular weight (670 kDa) and the distribution of density into two lobes of similar size suggests the presence of a homodimeric complex. To establish whether the telomerase complex can bind one or two telomeres, a purified telomerase preparation was incubated with a G-overhang oligonucleotide 5′-(TTAGGG)2-3′ labeled at its 5′ end with biotin and 5nm colloidal gold coated with monovalent streptavidin33. Colloidal gold can clearly be recognized as 5nm black dots bound to telomerase dimers (Fig. 1g, encircled), whereas excess monodisperse gold have a thin streptavidin coat (Fig. 1g, arrows). No particles resembling telomerase could be detected in a control sample containing streptavidin gold (Fig. 1h). Out of 450 telomerase dimers analyzed, 13.5% had no gold bound, 50% had one gold particle bound and 36.5% had two gold particles bound. This analysis suggests that one dimeric telomerase particle can bind up to two G-overhang substrates. This in turn is consistent with the presence of two RNA templates, and hence two TER subunits per telomerase complex.Telomerase is a functional dimerTo address the question of whether active human telomerase contains one or more hTERT subunits, the telomerase enzyme was assembled in vivo from differently tagged TERT subunits. Expression plasmids of WT-hTERT containing either a N-terminal 13×Myc or an N-terminal ZZ(=2-protein A-tags)_(TEVsite)_3×Flag tag were generated and the two plasmids were co-expressed at a ratio of 1:1 or 1:6 together with hTER in HEK293T cells as described above. The differential tagging not only permits affinity purification, but results in TERT subunits of different MWs, which permits unambiguous interpretation of the composition of the in vivo assembled telomerase. Firstly, western blot analysis of cell lysates using an affinity purified polyclonal antibody against the C-terminus of hTERT, shows that the differently tagged TERT subunits are expressed in approximately the ratios expected from the input plasmid concentrations (Fig. 2a, Lanes 3 and 5 and fig.S2a-b). Secondly, the composition of telomerase complexes was investigated by binding ZZ-tagged hTERT to IgG sepharose followed by TEV protease cleavage of the TEV site present in the ZZ-tagged hTERT subunit only. Western blot analysis of the released immunoaffinity purified complexes shows that 13×Myc_hTERT co-purifies with ZZ-tagged hTERT (Fig. 2a, Lanes 7 and 9), whilst in the absence of co-expressed ZZ-tagged hTERT, 13×Myc_hTERT does not bind to IgG sepharose (fig. S2 c-d, Lane 6). Furthermore, the intensity of the bands in the western blot (Fig. 2a, Lanes 7 and 9) is consistent with TEV cleavage releasing 3×Flag_hTERT homodimers and 3×Flag_hTERT/13×Myc hTERT heterodimers in the appropriate concentrations (as determined by the expression plasmid ratios) (Fig. 2a). Thus, the 13×Myc_hTERT subunit must have been in complex with the ZZ-tagged hTERT subunit to co-purify in the immunoaffinity purification. This result provides strong evidence that two TERT subunits are present per human telomerase complex and is consistent with both the G-overhang binding data (Fig. 1g) and the molecular weight estimation of purified telomerase (Fig. 1b).We next asked whether the two TERT subunits in a telomerase complex function independently of each other, or whether they cooperate for catalytic activity. For this analysis we generated a catalytically dead active site mutant (V709A, D710I) 34: (DN)-ZZ_(TEVsite)_3×Flag_hTERT (fig. S2a-b, Lane 4). This mutant, or the corresponding wild type (WT)-ZZ_(TEVsite)_3×Flag_hTERT were co-expressed with (WT)-13×Myc_hTERT at plasmid ratios of 1:1 or 6:1. The western blot analysis (Fig. 2a, Lanes 2 to 5) shows that the WT/DN TERT subunits are expressed in the expected ratios. To test whether TERT subunits function independently or cooperate, telomerase complexes were affinity purified via the ZZ-tag and complexes released by TEV cleavage as described above (Fig. 2a, Lanes 6 to 9). Thus, with the (DN) ZZ-tagged hTERT mutant, the complexes released contain WT hTERT only as DN/WT heterodimer, but not as WT/WT homodimers (Fig. 2a, Lanes 6 and 8). Significantly, quantification of telomerase catalytic activity, by both the telospot and direct telomerase activity assays14 (Fig. 2b-c, Lanes 6 and 8), reveal that the telomerase complexes containing a mixture of catalytically inactive and catalytically active hTERT are totally catalytically inactive for both repeat addition processivity and nucleotide addition processivity (Fig. 2c, Lanes 6 and 8) (see fig. S2 c-d, Lanes 2 to 5, 7 and 8 for quantification). In contrast, the WT/WT TERT containing complexes are active as expected (Fig. 2b-c, Lanes 7 and 9). Therefore, the presence of a catalytically inactive TERT subunit in the telomerase complex has a dominant-negative impact on catalytic activity. These results establish that the human telomerase not only is structurally a dimer, but that telomerase activity requires the cooperation between the two TERT subunits in a dimer.3-D reconstruction of the telomerase dimerFor single particle EM analysis of the telomerase dimer bound to the G-overhang oligonucleotide 5′-(TTAGGG)2-3′, the sucrose gradient fractionation described above (Fig. 1b) was carried out in the presence of 0 to 0.1% glutaraldehyde (GraFix)35. This gentle cross-linking method has previously been demonstrated to improve structural homogeneity, particularly for low-abundance complexes that may otherwise dissociate during the preparation of EM grids. Importantly, parallel sucrose gradient fractionations of purified telomerase samples in the presence or absence of glutaraldehyde, show the same sedimentation profile (Fig. 1b). This indicates that the cross-linking did not effect the structural composition and that the cross-linked telomerase sample represents the active conformation of the enzyme (Fig. 1c). After the sucrose gradient fractionation, the concentration of the purified telomerase was very low and this precluded 3D structure determination by cryo EM. After concentrating the sample onto a freshly prepared carbon coated grid (Supplementary Material), we could detect 30-100 particles/image plate which was sufficient for single particle analysis by negative stain.Figure 3a shows a representative field-view of the raw telomerase particles in negative stain, in which top views and side views of the telomerase dimer can clearly be recognized. A data set containing 20,127 telomerase particles was subjected to 2D class averaging by reference-free procedures using ~200 images/class (For image processing procedures see Supplementary Material). Figure 3b-d shows a gallery of 2D class averages representing top views (Fig. 3b), tilted views (Fig. 3c) and side views (Fig. 3d). Figure 3e shows two representative class average side views after they have been rotationally aligned, displaying a bilobal structure in which the two lobes are connected by a thin region of density. One of the two lobes has a V-shaped opening (Fig. 3e-f indicated with a **) and the other appears closed (Fig. 3e-f, indicated with a *). Inspection of the rotationally aligned class averages revealed similar monomer densities, however the relative orientation of the monomers in the dimer differ. The difference detected is illustrated by a simple movie composed of these two 2D class averages in sequence (Fig. 3e, Supplementary movie S1). This analysis suggests that telomerase has a flexible dimer interface that could act as a hinge region.In addition to 2D classification, the 3D structure of the telomerase dimer was refined using the low-resolution subtomogram average (shown in Fig. 1f) as a reference model for 3D single particle refinement (Online Methods). Distinct conformations were further classified by three-dimensional maximum-likelihood analysis in Fourier space (MLF3D)36. After 25 rounds of non-supervised angular refinement, four reconstructions (Fig. S3c-f) were calculated and the absolute hand and the overall correctness of the dimer structure were assessed using a modified version of tilt-pair validation method37 (Fig. S4). Figure 3f shows different orientations of a representative density map calculated from a sub-population of 3,659 particles at a resolution of 30 Å. Top views, tilted views and side views of the dimer density agree well with independently obtained reference-free 2D class averages in Figure 3b-e. These features include a characteristic V-shaped opening in one of the monomers, whilst the other is more closed. From the location of the nano-gold labelled G-overhang we can calculate that the distance between the two catalytic pockets in a telomerase dimer is 180-190Å (Fig. 3g). To test whether the difference in monomer conformation observed was due to sub-saturated levels of bound G-overhang oligonucleotide, we repeated the 2D classification (as described above) of a telomerase prepared in absence of DNA. No difference could be detected between class averages of telomerase in the presence (Fig. 3b) or absence of bound DNA (data not shown) indicating that the conformation of the monomers as well as the flexibility of the dimer interface is independent of G-overhang binding. In summary, the 3D structural determination of the telomerase dimer establishes that the two monomers in the dimer have different conformations, one open and one closed, and are linked by a flexible dimer interface (Fig. 3h). Future analyses will be required to establish the role of the telomerase structure dynamics in the binding of two telomere substrates and telomeric DNA synthesis.Independently refined monomers and composite dimersTo circumvent refinement problems arising from the flexibility in the telomerase dimer interface, we processed the two halves of the telomerase dimer independently. From each class average of the telomerase dimer (excluding top views), the centre of each monomer was measured manually and the corresponding centres in the raw image stack were calculated. Next sub-images were boxed out as shown in Figure 4a, keeping images corresponding to the open V-shaped monomer separate from the closed monomer. The 3D refinements were carried out by MLF3D as described above. As expected, the resolution of the monomer structures markedly improved: measuring 23 Å for the open monomer and 21 Å for the closed. Although of similar resolutions, the V-shaped open monomer density had more clearly defined density distribution as can be explained by alignment procedures that, in general, work better with pronounced features. The dimer interface was not refined using this procedure and therefore has been removed from side views shown in Figure 4b. Next, the independently refined 3D-monomer reconstructions were re-assembled into composite dimers based on the angles previously measured in the individual 2D class averages. The composite dimer shares many structural features with the density calculated for the telomerase dimer, but are better resolved (Compare Fig 3f. and Fig 4d.). Figure 4c shows a comparison of three 2D class averages, the corresponding composite 3D-reconstruction and projections of composite dimers. The close similarity between features in the 3D EM density of composite dimers and the 2D class averages indicates that the open and closed monomers were well refined.The open monomer reconstruction shows several distinct features that allowed the fitting of the atomic structure of the beetle TERT subunit22,25 into the EM density (Fig. 5a-c). This was done using an automatic six dimensional search in real space, for which only one solution was obtained. The excellent fit is supported both by the similarity in the overall shape of the protein subunit and by the coincidence of the catalytic core channel in the crystal structure with a channel in the EM density (Fig. 5c). These features can also be recognized in the crystal structure of the beetle TERT after filtering to 20Å resolution (Fig. S5b). To validate our assignment of the TERT subunit to the correct region of the EM density map, we first tested different antibodies to pinpoint the protein subunit. Although we could detect binding, the EM images were not informative as the molecular weight of IgG is the same MW range as the TERT subunit. Instead, we produced and purified telomerase (as described above) in which the TERT subunit contained an N-terminal 6×His-tag (ZZ_(TEV)_6×His_5x(AAAKE) ~37Å long alpha Helix_3x_Flag_hTERT). The location of the His-tag was found by using Ni-NTA coated 5nm colloidal gold. Visual inspection of particles showed that the majority of the gold particles were located at the periphery of the telomerase dimer (Fig. 5d). To obtain the distribution of the location of the gold particles in the telomerase dimer, we measured the distance from the centre of 100 gold particles to the dimer interface. The plot in Fig. 5e shows a pronounced peak at 110Å distance, which is in good agreement with the fitted location of the TERT subunit in the EM density of the “open” monomer (Fig. 5a-c).

In vivo assembled human telomerase is a dimerTo obtain sufficient quantities of purified, active human telomerase for structural analyses, we made use of HEK293T ‘super-telomerase’ cells which over-express hTER and hTERT from transiently transfected plasmid constructs30, and have 200-fold higher telomerase activity than untransfected HEK293T cells31. The purification exploits the protein-A tag cloned at the N-terminus of hTERT, followed by affinity chromatography and sucrose gradient fractionation. To facilitate the tracking of the telomerase in the sucrose gradient, the telomerase was incubated with a [32P] 5′ labeled G-overhang oligonucleotide (5′-(TTAGGG)2-3′) prior to the sedimentation. Figure 1b shows a well resolved peak that sediments in the same position as the 670 kDa thyroglobulin size marker analysed in a parallel sucrose gradient (fig. S1c). Furthermore, tracking of the telomerase using a direct telomerase activity assay 10 shows that the peak of activity coincides with the position of the 670 kDa peak in the sucrose gradient (Fig. 1b. and 1c). The purified telomerase sample was further analysed for protein content by SDS-PAGE, which shows a band with the MW of hTERT (fig. S1a). Incubation of purified telomerase with 5′ [32P] labeled G-overhang oligonucleotide (5′-(TTAGGG)3-3′) followed by native PAGE shows that the telomerase complex migrates as a very tight band, and hence is of homogeneous MW (fig. S1b). The composition of the purified telomerase sample was analysed by mass spectroscopy. This analysis (table S1) reveals that the purified telomerase contains the hTERT subunits and two accessory proteins, Nop10 and dyskerin32. In conclusion, our data show that the purified human telomerase is active and that both the G-overhang bound and unbound telomerase complexes have a molecular weight consistent with that of a dimer consisting of two hTERT (127 kDa) and two hTER (153 kDa) subunits, as well as the two accessory proteins Nop10 (7.7 kDa) and dyskerin (58 kDA). This observation is in agreement with findings from a previous purification of the human telomerase complex from untransfected HEK239T cells10.Figure 1D shows an electron micrograph of purified, in vivo assembled human telomerase bound to the G-overhang oligonucleotide 5′-(TTAGGG)2-3′. Inspection of the micrograph shows side views of many particles with a bilobal structure. Two- and three-dimensional structural information was obtained by single particle classification (Fig. 1e) and electron tomography (Fig. 1f) respectively (For image processing procedures, see Supplementary Material). These analyses reveal that the telomerase enzyme has an elongated structure, 280 Å in length and 125 Å in width, consisting of two spherical lobes connected by a region of thin density. The size of the telomerase particle is consistent with the estimated molecular weight (670 kDa) and the distribution of density into two lobes of similar size suggests the presence of a homodimeric complex. To establish whether the telomerase complex can bind one or two telomeres, a purified telomerase preparation was incubated with a G-overhang oligonucleotide 5′-(TTAGGG)2-3′ labeled at its 5′ end with biotin and 5nm colloidal gold coated with monovalent streptavidin33. Colloidal gold can clearly be recognized as 5nm black dots bound to telomerase dimers (Fig. 1g, encircled), whereas excess monodisperse gold have a thin streptavidin coat (Fig. 1g, arrows). No particles resembling telomerase could be detected in a control sample containing streptavidin gold (Fig. 1h). Out of 450 telomerase dimers analyzed, 13.5% had no gold bound, 50% had one gold particle bound and 36.5% had two gold particles bound. This analysis suggests that one dimeric telomerase particle can bind up to two G-overhang substrates. This in turn is consistent with the presence of two RNA templates, and hence two TER subunits per telomerase complex.

In vivo assembled human telomerase is a dimer

To obtain sufficient quantities of purified, active human telomerase for structural analyses, we made use of HEK293T ‘super-telomerase’ cells which over-express hTER and hTERT from transiently transfected plasmid constructs30, and have 200-fold higher telomerase activity than untransfected HEK293T cells31. The purification exploits the protein-A tag cloned at the N-terminus of hTERT, followed by affinity chromatography and sucrose gradient fractionation. To facilitate the tracking of the telomerase in the sucrose gradient, the telomerase was incubated with a [32P] 5′ labeled G-overhang oligonucleotide (5′-(TTAGGG)2-3′) prior to the sedimentation. Figure 1b shows a well resolved peak that sediments in the same position as the 670 kDa thyroglobulin size marker analysed in a parallel sucrose gradient (fig. S1c). Furthermore, tracking of the telomerase using a direct telomerase activity assay 10 shows that the peak of activity coincides with the position of the 670 kDa peak in the sucrose gradient (Fig. 1b. and 1c). The purified telomerase sample was further analysed for protein content by SDS-PAGE, which shows a band with the MW of hTERT (fig. S1a). Incubation of purified telomerase with 5′ [32P] labeled G-overhang oligonucleotide (5′-(TTAGGG)3-3′) followed by native PAGE shows that the telomerase complex migrates as a very tight band, and hence is of homogeneous MW (fig. S1b). The composition of the purified telomerase sample was analysed by mass spectroscopy. This analysis (table S1) reveals that the purified telomerase contains the hTERT subunits and two accessory proteins, Nop10 and dyskerin32. In conclusion, our data show that the purified human telomerase is active and that both the G-overhang bound and unbound telomerase complexes have a molecular weight consistent with that of a dimer consisting of two hTERT (127 kDa) and two hTER (153 kDa) subunits, as well as the two accessory proteins Nop10 (7.7 kDa) and dyskerin (58 kDA). This observation is in agreement with findings from a previous purification of the human telomerase complex from untransfected HEK239T cells10.

Figure 1D shows an electron micrograph of purified, in vivo assembled human telomerase bound to the G-overhang oligonucleotide 5′-(TTAGGG)2-3′. Inspection of the micrograph shows side views of many particles with a bilobal structure. Two- and three-dimensional structural information was obtained by single particle classification (Fig. 1e) and electron tomography (Fig. 1f) respectively (For image processing procedures, see Supplementary Material). These analyses reveal that the telomerase enzyme has an elongated structure, 280 Å in length and 125 Å in width, consisting of two spherical lobes connected by a region of thin density. The size of the telomerase particle is consistent with the estimated molecular weight (670 kDa) and the distribution of density into two lobes of similar size suggests the presence of a homodimeric complex. To establish whether the telomerase complex can bind one or two telomeres, a purified telomerase preparation was incubated with a G-overhang oligonucleotide 5′-(TTAGGG)2-3′ labeled at its 5′ end with biotin and 5nm colloidal gold coated with monovalent streptavidin33. Colloidal gold can clearly be recognized as 5nm black dots bound to telomerase dimers (Fig. 1g, encircled), whereas excess monodisperse gold have a thin streptavidin coat (Fig. 1g, arrows). No particles resembling telomerase could be detected in a control sample containing streptavidin gold (Fig. 1h). Out of 450 telomerase dimers analyzed, 13.5% had no gold bound, 50% had one gold particle bound and 36.5% had two gold particles bound. This analysis suggests that one dimeric telomerase particle can bind up to two G-overhang substrates. This in turn is consistent with the presence of two RNA templates, and hence two TER subunits per telomerase complex.

Telomerase is a functional dimerTo address the question of whether active human telomerase contains one or more hTERT subunits, the telomerase enzyme was assembled in vivo from differently tagged TERT subunits. Expression plasmids of WT-hTERT containing either a N-terminal 13×Myc or an N-terminal ZZ(=2-protein A-tags)_(TEVsite)_3×Flag tag were generated and the two plasmids were co-expressed at a ratio of 1:1 or 1:6 together with hTER in HEK293T cells as described above. The differential tagging not only permits affinity purification, but results in TERT subunits of different MWs, which permits unambiguous interpretation of the composition of the in vivo assembled telomerase. Firstly, western blot analysis of cell lysates using an affinity purified polyclonal antibody against the C-terminus of hTERT, shows that the differently tagged TERT subunits are expressed in approximately the ratios expected from the input plasmid concentrations (Fig. 2a, Lanes 3 and 5 and fig.S2a-b). Secondly, the composition of telomerase complexes was investigated by binding ZZ-tagged hTERT to IgG sepharose followed by TEV protease cleavage of the TEV site present in the ZZ-tagged hTERT subunit only. Western blot analysis of the released immunoaffinity purified complexes shows that 13×Myc_hTERT co-purifies with ZZ-tagged hTERT (Fig. 2a, Lanes 7 and 9), whilst in the absence of co-expressed ZZ-tagged hTERT, 13×Myc_hTERT does not bind to IgG sepharose (fig. S2 c-d, Lane 6). Furthermore, the intensity of the bands in the western blot (Fig. 2a, Lanes 7 and 9) is consistent with TEV cleavage releasing 3×Flag_hTERT homodimers and 3×Flag_hTERT/13×Myc hTERT heterodimers in the appropriate concentrations (as determined by the expression plasmid ratios) (Fig. 2a). Thus, the 13×Myc_hTERT subunit must have been in complex with the ZZ-tagged hTERT subunit to co-purify in the immunoaffinity purification. This result provides strong evidence that two TERT subunits are present per human telomerase complex and is consistent with both the G-overhang binding data (Fig. 1g) and the molecular weight estimation of purified telomerase (Fig. 1b).We next asked whether the two TERT subunits in a telomerase complex function independently of each other, or whether they cooperate for catalytic activity. For this analysis we generated a catalytically dead active site mutant (V709A, D710I) 34: (DN)-ZZ_(TEVsite)_3×Flag_hTERT (fig. S2a-b, Lane 4). This mutant, or the corresponding wild type (WT)-ZZ_(TEVsite)_3×Flag_hTERT were co-expressed with (WT)-13×Myc_hTERT at plasmid ratios of 1:1 or 6:1. The western blot analysis (Fig. 2a, Lanes 2 to 5) shows that the WT/DN TERT subunits are expressed in the expected ratios. To test whether TERT subunits function independently or cooperate, telomerase complexes were affinity purified via the ZZ-tag and complexes released by TEV cleavage as described above (Fig. 2a, Lanes 6 to 9). Thus, with the (DN) ZZ-tagged hTERT mutant, the complexes released contain WT hTERT only as DN/WT heterodimer, but not as WT/WT homodimers (Fig. 2a, Lanes 6 and 8). Significantly, quantification of telomerase catalytic activity, by both the telospot and direct telomerase activity assays14 (Fig. 2b-c, Lanes 6 and 8), reveal that the telomerase complexes containing a mixture of catalytically inactive and catalytically active hTERT are totally catalytically inactive for both repeat addition processivity and nucleotide addition processivity (Fig. 2c, Lanes 6 and 8) (see fig. S2 c-d, Lanes 2 to 5, 7 and 8 for quantification). In contrast, the WT/WT TERT containing complexes are active as expected (Fig. 2b-c, Lanes 7 and 9). Therefore, the presence of a catalytically inactive TERT subunit in the telomerase complex has a dominant-negative impact on catalytic activity. These results establish that the human telomerase not only is structurally a dimer, but that telomerase activity requires the cooperation between the two TERT subunits in a dimer.

Telomerase is a functional dimer

To address the question of whether active human telomerase contains one or more hTERT subunits, the telomerase enzyme was assembled in vivo from differently tagged TERT subunits. Expression plasmids of WT-hTERT containing either a N-terminal 13×Myc or an N-terminal ZZ(=2-protein A-tags)_(TEVsite)_3×Flag tag were generated and the two plasmids were co-expressed at a ratio of 1:1 or 1:6 together with hTER in HEK293T cells as described above. The differential tagging not only permits affinity purification, but results in TERT subunits of different MWs, which permits unambiguous interpretation of the composition of the in vivo assembled telomerase. Firstly, western blot analysis of cell lysates using an affinity purified polyclonal antibody against the C-terminus of hTERT, shows that the differently tagged TERT subunits are expressed in approximately the ratios expected from the input plasmid concentrations (Fig. 2a, Lanes 3 and 5 and fig.S2a-b). Secondly, the composition of telomerase complexes was investigated by binding ZZ-tagged hTERT to IgG sepharose followed by TEV protease cleavage of the TEV site present in the ZZ-tagged hTERT subunit only. Western blot analysis of the released immunoaffinity purified complexes shows that 13×Myc_hTERT co-purifies with ZZ-tagged hTERT (Fig. 2a, Lanes 7 and 9), whilst in the absence of co-expressed ZZ-tagged hTERT, 13×Myc_hTERT does not bind to IgG sepharose (fig. S2 c-d, Lane 6). Furthermore, the intensity of the bands in the western blot (Fig. 2a, Lanes 7 and 9) is consistent with TEV cleavage releasing 3×Flag_hTERT homodimers and 3×Flag_hTERT/13×Myc hTERT heterodimers in the appropriate concentrations (as determined by the expression plasmid ratios) (Fig. 2a). Thus, the 13×Myc_hTERT subunit must have been in complex with the ZZ-tagged hTERT subunit to co-purify in the immunoaffinity purification. This result provides strong evidence that two TERT subunits are present per human telomerase complex and is consistent with both the G-overhang binding data (Fig. 1g) and the molecular weight estimation of purified telomerase (Fig. 1b).

We next asked whether the two TERT subunits in a telomerase complex function independently of each other, or whether they cooperate for catalytic activity. For this analysis we generated a catalytically dead active site mutant (V709A, D710I) 34: (DN)-ZZ_(TEVsite)_3×Flag_hTERT (fig. S2a-b, Lane 4). This mutant, or the corresponding wild type (WT)-ZZ_(TEVsite)_3×Flag_hTERT were co-expressed with (WT)-13×Myc_hTERT at plasmid ratios of 1:1 or 6:1. The western blot analysis (Fig. 2a, Lanes 2 to 5) shows that the WT/DN TERT subunits are expressed in the expected ratios. To test whether TERT subunits function independently or cooperate, telomerase complexes were affinity purified via the ZZ-tag and complexes released by TEV cleavage as described above (Fig. 2a, Lanes 6 to 9). Thus, with the (DN) ZZ-tagged hTERT mutant, the complexes released contain WT hTERT only as DN/WT heterodimer, but not as WT/WT homodimers (Fig. 2a, Lanes 6 and 8). Significantly, quantification of telomerase catalytic activity, by both the telospot and direct telomerase activity assays14 (Fig. 2b-c, Lanes 6 and 8), reveal that the telomerase complexes containing a mixture of catalytically inactive and catalytically active hTERT are totally catalytically inactive for both repeat addition processivity and nucleotide addition processivity (Fig. 2c, Lanes 6 and 8) (see fig. S2 c-d, Lanes 2 to 5, 7 and 8 for quantification). In contrast, the WT/WT TERT containing complexes are active as expected (Fig. 2b-c, Lanes 7 and 9). Therefore, the presence of a catalytically inactive TERT subunit in the telomerase complex has a dominant-negative impact on catalytic activity. These results establish that the human telomerase not only is structurally a dimer, but that telomerase activity requires the cooperation between the two TERT subunits in a dimer.

3-D reconstruction of the telomerase dimerFor single particle EM analysis of the telomerase dimer bound to the G-overhang oligonucleotide 5′-(TTAGGG)2-3′, the sucrose gradient fractionation described above (Fig. 1b) was carried out in the presence of 0 to 0.1% glutaraldehyde (GraFix)35. This gentle cross-linking method has previously been demonstrated to improve structural homogeneity, particularly for low-abundance complexes that may otherwise dissociate during the preparation of EM grids. Importantly, parallel sucrose gradient fractionations of purified telomerase samples in the presence or absence of glutaraldehyde, show the same sedimentation profile (Fig. 1b). This indicates that the cross-linking did not effect the structural composition and that the cross-linked telomerase sample represents the active conformation of the enzyme (Fig. 1c). After the sucrose gradient fractionation, the concentration of the purified telomerase was very low and this precluded 3D structure determination by cryo EM. After concentrating the sample onto a freshly prepared carbon coated grid (Supplementary Material), we could detect 30-100 particles/image plate which was sufficient for single particle analysis by negative stain.Figure 3a shows a representative field-view of the raw telomerase particles in negative stain, in which top views and side views of the telomerase dimer can clearly be recognized. A data set containing 20,127 telomerase particles was subjected to 2D class averaging by reference-free procedures using ~200 images/class (For image processing procedures see Supplementary Material). Figure 3b-d shows a gallery of 2D class averages representing top views (Fig. 3b), tilted views (Fig. 3c) and side views (Fig. 3d). Figure 3e shows two representative class average side views after they have been rotationally aligned, displaying a bilobal structure in which the two lobes are connected by a thin region of density. One of the two lobes has a V-shaped opening (Fig. 3e-f indicated with a **) and the other appears closed (Fig. 3e-f, indicated with a *). Inspection of the rotationally aligned class averages revealed similar monomer densities, however the relative orientation of the monomers in the dimer differ. The difference detected is illustrated by a simple movie composed of these two 2D class averages in sequence (Fig. 3e, Supplementary movie S1). This analysis suggests that telomerase has a flexible dimer interface that could act as a hinge region.In addition to 2D classification, the 3D structure of the telomerase dimer was refined using the low-resolution subtomogram average (shown in Fig. 1f) as a reference model for 3D single particle refinement (Online Methods). Distinct conformations were further classified by three-dimensional maximum-likelihood analysis in Fourier space (MLF3D)36. After 25 rounds of non-supervised angular refinement, four reconstructions (Fig. S3c-f) were calculated and the absolute hand and the overall correctness of the dimer structure were assessed using a modified version of tilt-pair validation method37 (Fig. S4). Figure 3f shows different orientations of a representative density map calculated from a sub-population of 3,659 particles at a resolution of 30 Å. Top views, tilted views and side views of the dimer density agree well with independently obtained reference-free 2D class averages in Figure 3b-e. These features include a characteristic V-shaped opening in one of the monomers, whilst the other is more closed. From the location of the nano-gold labelled G-overhang we can calculate that the distance between the two catalytic pockets in a telomerase dimer is 180-190Å (Fig. 3g). To test whether the difference in monomer conformation observed was due to sub-saturated levels of bound G-overhang oligonucleotide, we repeated the 2D classification (as described above) of a telomerase prepared in absence of DNA. No difference could be detected between class averages of telomerase in the presence (Fig. 3b) or absence of bound DNA (data not shown) indicating that the conformation of the monomers as well as the flexibility of the dimer interface is independent of G-overhang binding. In summary, the 3D structural determination of the telomerase dimer establishes that the two monomers in the dimer have different conformations, one open and one closed, and are linked by a flexible dimer interface (Fig. 3h). Future analyses will be required to establish the role of the telomerase structure dynamics in the binding of two telomere substrates and telomeric DNA synthesis.

3-D reconstruction of the telomerase dimer

For single particle EM analysis of the telomerase dimer bound to the G-overhang oligonucleotide 5′-(TTAGGG)2-3′, the sucrose gradient fractionation described above (Fig. 1b) was carried out in the presence of 0 to 0.1% glutaraldehyde (GraFix)35. This gentle cross-linking method has previously been demonstrated to improve structural homogeneity, particularly for low-abundance complexes that may otherwise dissociate during the preparation of EM grids. Importantly, parallel sucrose gradient fractionations of purified telomerase samples in the presence or absence of glutaraldehyde, show the same sedimentation profile (Fig. 1b). This indicates that the cross-linking did not effect the structural composition and that the cross-linked telomerase sample represents the active conformation of the enzyme (Fig. 1c). After the sucrose gradient fractionation, the concentration of the purified telomerase was very low and this precluded 3D structure determination by cryo EM. After concentrating the sample onto a freshly prepared carbon coated grid (Supplementary Material), we could detect 30-100 particles/image plate which was sufficient for single particle analysis by negative stain.

Figure 3a shows a representative field-view of the raw telomerase particles in negative stain, in which top views and side views of the telomerase dimer can clearly be recognized. A data set containing 20,127 telomerase particles was subjected to 2D class averaging by reference-free procedures using ~200 images/class (For image processing procedures see Supplementary Material). Figure 3b-d shows a gallery of 2D class averages representing top views (Fig. 3b), tilted views (Fig. 3c) and side views (Fig. 3d). Figure 3e shows two representative class average side views after they have been rotationally aligned, displaying a bilobal structure in which the two lobes are connected by a thin region of density. One of the two lobes has a V-shaped opening (Fig. 3e-f indicated with a **) and the other appears closed (Fig. 3e-f, indicated with a *). Inspection of the rotationally aligned class averages revealed similar monomer densities, however the relative orientation of the monomers in the dimer differ. The difference detected is illustrated by a simple movie composed of these two 2D class averages in sequence (Fig. 3e, Supplementary movie S1). This analysis suggests that telomerase has a flexible dimer interface that could act as a hinge region.

In addition to 2D classification, the 3D structure of the telomerase dimer was refined using the low-resolution subtomogram average (shown in Fig. 1f) as a reference model for 3D single particle refinement (Online Methods). Distinct conformations were further classified by three-dimensional maximum-likelihood analysis in Fourier space (MLF3D)36. After 25 rounds of non-supervised angular refinement, four reconstructions (Fig. S3c-f) were calculated and the absolute hand and the overall correctness of the dimer structure were assessed using a modified version of tilt-pair validation method37 (Fig. S4). Figure 3f shows different orientations of a representative density map calculated from a sub-population of 3,659 particles at a resolution of 30 Å. Top views, tilted views and side views of the dimer density agree well with independently obtained reference-free 2D class averages in Figure 3b-e. These features include a characteristic V-shaped opening in one of the monomers, whilst the other is more closed. From the location of the nano-gold labelled G-overhang we can calculate that the distance between the two catalytic pockets in a telomerase dimer is 180-190Å (Fig. 3g). To test whether the difference in monomer conformation observed was due to sub-saturated levels of bound G-overhang oligonucleotide, we repeated the 2D classification (as described above) of a telomerase prepared in absence of DNA. No difference could be detected between class averages of telomerase in the presence (Fig. 3b) or absence of bound DNA (data not shown) indicating that the conformation of the monomers as well as the flexibility of the dimer interface is independent of G-overhang binding. In summary, the 3D structural determination of the telomerase dimer establishes that the two monomers in the dimer have different conformations, one open and one closed, and are linked by a flexible dimer interface (Fig. 3h). Future analyses will be required to establish the role of the telomerase structure dynamics in the binding of two telomere substrates and telomeric DNA synthesis.

Independently refined monomers and composite dimersTo circumvent refinement problems arising from the flexibility in the telomerase dimer interface, we processed the two halves of the telomerase dimer independently. From each class average of the telomerase dimer (excluding top views), the centre of each monomer was measured manually and the corresponding centres in the raw image stack were calculated. Next sub-images were boxed out as shown in Figure 4a, keeping images corresponding to the open V-shaped monomer separate from the closed monomer. The 3D refinements were carried out by MLF3D as described above. As expected, the resolution of the monomer structures markedly improved: measuring 23 Å for the open monomer and 21 Å for the closed. Although of similar resolutions, the V-shaped open monomer density had more clearly defined density distribution as can be explained by alignment procedures that, in general, work better with pronounced features. The dimer interface was not refined using this procedure and therefore has been removed from side views shown in Figure 4b. Next, the independently refined 3D-monomer reconstructions were re-assembled into composite dimers based on the angles previously measured in the individual 2D class averages. The composite dimer shares many structural features with the density calculated for the telomerase dimer, but are better resolved (Compare Fig 3f. and Fig 4d.). Figure 4c shows a comparison of three 2D class averages, the corresponding composite 3D-reconstruction and projections of composite dimers. The close similarity between features in the 3D EM density of composite dimers and the 2D class averages indicates that the open and closed monomers were well refined.The open monomer reconstruction shows several distinct features that allowed the fitting of the atomic structure of the beetle TERT subunit22,25 into the EM density (Fig. 5a-c). This was done using an automatic six dimensional search in real space, for which only one solution was obtained. The excellent fit is supported both by the similarity in the overall shape of the protein subunit and by the coincidence of the catalytic core channel in the crystal structure with a channel in the EM density (Fig. 5c). These features can also be recognized in the crystal structure of the beetle TERT after filtering to 20Å resolution (Fig. S5b). To validate our assignment of the TERT subunit to the correct region of the EM density map, we first tested different antibodies to pinpoint the protein subunit. Although we could detect binding, the EM images were not informative as the molecular weight of IgG is the same MW range as the TERT subunit. Instead, we produced and purified telomerase (as described above) in which the TERT subunit contained an N-terminal 6×His-tag (ZZ_(TEV)_6×His_5x(AAAKE) ~37Å long alpha Helix_3x_Flag_hTERT). The location of the His-tag was found by using Ni-NTA coated 5nm colloidal gold. Visual inspection of particles showed that the majority of the gold particles were located at the periphery of the telomerase dimer (Fig. 5d). To obtain the distribution of the location of the gold particles in the telomerase dimer, we measured the distance from the centre of 100 gold particles to the dimer interface. The plot in Fig. 5e shows a pronounced peak at 110Å distance, which is in good agreement with the fitted location of the TERT subunit in the EM density of the “open” monomer (Fig. 5a-c).

Independently refined monomers and composite dimers

To circumvent refinement problems arising from the flexibility in the telomerase dimer interface, we processed the two halves of the telomerase dimer independently. From each class average of the telomerase dimer (excluding top views), the centre of each monomer was measured manually and the corresponding centres in the raw image stack were calculated. Next sub-images were boxed out as shown in Figure 4a, keeping images corresponding to the open V-shaped monomer separate from the closed monomer. The 3D refinements were carried out by MLF3D as described above. As expected, the resolution of the monomer structures markedly improved: measuring 23 Å for the open monomer and 21 Å for the closed. Although of similar resolutions, the V-shaped open monomer density had more clearly defined density distribution as can be explained by alignment procedures that, in general, work better with pronounced features. The dimer interface was not refined using this procedure and therefore has been removed from side views shown in Figure 4b. Next, the independently refined 3D-monomer reconstructions were re-assembled into composite dimers based on the angles previously measured in the individual 2D class averages. The composite dimer shares many structural features with the density calculated for the telomerase dimer, but are better resolved (Compare Fig 3f. and Fig 4d.). Figure 4c shows a comparison of three 2D class averages, the corresponding composite 3D-reconstruction and projections of composite dimers. The close similarity between features in the 3D EM density of composite dimers and the 2D class averages indicates that the open and closed monomers were well refined.

The open monomer reconstruction shows several distinct features that allowed the fitting of the atomic structure of the beetle TERT subunit22,25 into the EM density (Fig. 5a-c). This was done using an automatic six dimensional search in real space, for which only one solution was obtained. The excellent fit is supported both by the similarity in the overall shape of the protein subunit and by the coincidence of the catalytic core channel in the crystal structure with a channel in the EM density (Fig. 5c). These features can also be recognized in the crystal structure of the beetle TERT after filtering to 20Å resolution (Fig. S5b). To validate our assignment of the TERT subunit to the correct region of the EM density map, we first tested different antibodies to pinpoint the protein subunit. Although we could detect binding, the EM images were not informative as the molecular weight of IgG is the same MW range as the TERT subunit. Instead, we produced and purified telomerase (as described above) in which the TERT subunit contained an N-terminal 6×His-tag (ZZ_(TEV)_6×His_5x(AAAKE) ~37Å long alpha Helix_3x_Flag_hTERT). The location of the His-tag was found by using Ni-NTA coated 5nm colloidal gold. Visual inspection of particles showed that the majority of the gold particles were located at the periphery of the telomerase dimer (Fig. 5d). To obtain the distribution of the location of the gold particles in the telomerase dimer, we measured the distance from the centre of 100 gold particles to the dimer interface. The plot in Fig. 5e shows a pronounced peak at 110Å distance, which is in good agreement with the fitted location of the TERT subunit in the EM density of the “open” monomer (Fig. 5a-c).

DiscussionThe determination of the first 3D-structure of active, full-length human telomerase assembled in vivo shows that it has a bilobal architecture (Fig. 1d-f and Fig.3) and a molecular weight consistent with a dimeric complex (Fig. 1b, fig. 1aS). These data together with our biochemical data provide strong evidence that the bilobal structure arises from the presence of two TERT subunits, and two TER subunits (Fig. 1 and Fig. 2) and is able to bind two telomeric G-overhang substrates (Fig. 3g and h). For the TERT subunit, the excellent fit of the crystal structure of the beetle TERT subunit (ref 23-24) into the EM map reveals the location of the RNA binding TRBD and catalytic RT domains within TERT (Fig. 5 a-c). We also demonstrate that for in vivo assembled telomerase, catalytic activity requires the cooperation between two TERT subunits in the telomerase dimer and that a catalytic pocked mutation in one of the two TERT subunits is dominant negative (Fig. 2). These observations substantially strengthen our previous conclusions derived from in vitro assembled telomerase where an RNA template mutant interfered with wild-type template function when co-assembled into telomerase particles14. They are also in agreement with MW estimations by others10,38. However, our results are in disagreement with publications in which it was concluded that human telomerase is monomeric13,39. Our work also underlines the importance of revisiting the conclusion that short telomere disease-associated telomerase mutations are not dominant-negative13.Why should telomerase function as a dimer? Our work demonstrates that a human telomerase dimer contains two catalytic pockets, located about 180 Å apart (corresponding to about a stretch of 50 nucleotides), that can associate with two telomeric DNA substrates (Fig. 3g and h). This suggests that telomerase may function by extending two telomeric ends in parallel. Interestingly, it has been demonstrated recently that the cohesion of replicated telomeres is important for telomere maintenance by telomerase, a function that is defective in dyskeratosis congenita-associated mutations in TIN240. Therefore, an attractive hypothesis is that telomerase dimers extend aligned sister telomeres in parallel. This mechanism could enable sister telomeres to maintain equal lengths after extension. On the other hand, our data showing that the two TERT subunits in a telomerase dimer co-operate in telomerase activity could also have been explained with a mechanism in which telomerase binds and extends a single DNA substrate in a processive manner, as has been suggested from studies using partly purified Euplotes telomerase41. The observed flexibility of the interface between the two monomer lobes in the telomerase structure (Supplementary movie S1) could be functionally important by providing a hinge mechanism for the intramolecular translocation of the DNA substrate from one catalytic pocket to the other, as has been reported for other DNA/RNA polymerases such as the eukaryotic primosome42. However, since the DN/WT telomerase heterodimers were inactive even for the first round of telomeric DNA synthesis (Fig. 2c), which according to the translocation model should not require two active sites, we favour the parallel extension model.The identification of the location of the TERT subunit in the EM density (Fig. 5 a-c) provides important insights into the domain organization of the telomerase dimer. First of all, the TERT subunit is located at the periphery of the open monomer, where it is separated from the rest of the complex by the V-shaped opening in the EM density. Because the fitted crystal structure of the beetle TERT subunit22 corresponds to two thirds of hTERT, the majority of the EM density that is not occupied by the TERT subunit has to correspond to RNA, i.e. the hTER subunit. This interpretation agrees with the observation that the EM map in that region consists of three continuous regions of tubular density with diameters of 20-30Å (Fig. 5a-c), which is an expected feature of RNA helices (fig. S5a). This interpretation of the telomerase architecture implies that the dimer interface involves RNA-RNA interactions, which is in agreement with previous results43 (Fig. 5F). However, we failed to dissociate dimeric TERT into monomers by RNase treatment (data not shown) and therefore we cannot exclude the involvement of some telomerase accessory protein in the formation of the dimer interface. Because the crystal structure of the beetle TERT subunit was solved in complex with a short DNA/RNA helix23, its fitting into the EM density also pinpoints the location the 3′end of the G-overhang in the catalytic pocket of the RT domain (Fig. 5F). The consequence of this fitting is that the 5′end of the G-overhang points towards the centre of the monomer and a continuous region of density in its vicinity that we have tentatively assigned to the TER subunit (fig. S5), and suggests that the DNA binding TEN domain would be located adjacent to TRDB domain (Fig. 5F) consistent with the TERT domain structure (Fig. 1a). Furthermore, the fitting of the TERT subunit structure into the EM density map of the telomerase dimer reveals that the two catalytic pockets in a telomerase dimer (Fig. 3g-h and Fig. 5 d-e) and 6) are located very far apart, at a distance of 180-190 Å from each other. It will be interesting to unravel the mechanism by which they cooperate for telomerase activity, and indeed whether this distance is required to accommodate to telomeric ends in parallel.Future work will aim at improving the resolution of the telomerase structure in order to reveal the fold of the telomerase RNA and its specific interaction with the various TERT domains as well as how the different conformational states we observe may be related to different functional states of the telomerase catalytic cycle. This study also provides the starting point for investigating the structural basis of telomerase dimerization and its significance for catalysis and function.

The determination of the first 3D-structure of active, full-length human telomerase assembled in vivo shows that it has a bilobal architecture (Fig. 1d-f and Fig.3) and a molecular weight consistent with a dimeric complex (Fig. 1b, fig. 1aS). These data together with our biochemical data provide strong evidence that the bilobal structure arises from the presence of two TERT subunits, and two TER subunits (Fig. 1 and Fig. 2) and is able to bind two telomeric G-overhang substrates (Fig. 3g and h). For the TERT subunit, the excellent fit of the crystal structure of the beetle TERT subunit (ref 23-24) into the EM map reveals the location of the RNA binding TRBD and catalytic RT domains within TERT (Fig. 5 a-c). We also demonstrate that for in vivo assembled telomerase, catalytic activity requires the cooperation between two TERT subunits in the telomerase dimer and that a catalytic pocked mutation in one of the two TERT subunits is dominant negative (Fig. 2). These observations substantially strengthen our previous conclusions derived from in vitro assembled telomerase where an RNA template mutant interfered with wild-type template function when co-assembled into telomerase particles14. They are also in agreement with MW estimations by others10,38. However, our results are in disagreement with publications in which it was concluded that human telomerase is monomeric13,39. Our work also underlines the importance of revisiting the conclusion that short telomere disease-associated telomerase mutations are not dominant-negative13.

Why should telomerase function as a dimer? Our work demonstrates that a human telomerase dimer contains two catalytic pockets, located about 180 Å apart (corresponding to about a stretch of 50 nucleotides), that can associate with two telomeric DNA substrates (Fig. 3g and h). This suggests that telomerase may function by extending two telomeric ends in parallel. Interestingly, it has been demonstrated recently that the cohesion of replicated telomeres is important for telomere maintenance by telomerase, a function that is defective in dyskeratosis congenita-associated mutations in TIN240. Therefore, an attractive hypothesis is that telomerase dimers extend aligned sister telomeres in parallel. This mechanism could enable sister telomeres to maintain equal lengths after extension. On the other hand, our data showing that the two TERT subunits in a telomerase dimer co-operate in telomerase activity could also have been explained with a mechanism in which telomerase binds and extends a single DNA substrate in a processive manner, as has been suggested from studies using partly purified Euplotes telomerase41. The observed flexibility of the interface between the two monomer lobes in the telomerase structure (Supplementary movie S1) could be functionally important by providing a hinge mechanism for the intramolecular translocation of the DNA substrate from one catalytic pocket to the other, as has been reported for other DNA/RNA polymerases such as the eukaryotic primosome42. However, since the DN/WT telomerase heterodimers were inactive even for the first round of telomeric DNA synthesis (Fig. 2c), which according to the translocation model should not require two active sites, we favour the parallel extension model.

The identification of the location of the TERT subunit in the EM density (Fig. 5 a-c) provides important insights into the domain organization of the telomerase dimer. First of all, the TERT subunit is located at the periphery of the open monomer, where it is separated from the rest of the complex by the V-shaped opening in the EM density. Because the fitted crystal structure of the beetle TERT subunit22 corresponds to two thirds of hTERT, the majority of the EM density that is not occupied by the TERT subunit has to correspond to RNA, i.e. the hTER subunit. This interpretation agrees with the observation that the EM map in that region consists of three continuous regions of tubular density with diameters of 20-30Å (Fig. 5a-c), which is an expected feature of RNA helices (fig. S5a). This interpretation of the telomerase architecture implies that the dimer interface involves RNA-RNA interactions, which is in agreement with previous results43 (Fig. 5F). However, we failed to dissociate dimeric TERT into monomers by RNase treatment (data not shown) and therefore we cannot exclude the involvement of some telomerase accessory protein in the formation of the dimer interface. Because the crystal structure of the beetle TERT subunit was solved in complex with a short DNA/RNA helix23, its fitting into the EM density also pinpoints the location the 3′end of the G-overhang in the catalytic pocket of the RT domain (Fig. 5F). The consequence of this fitting is that the 5′end of the G-overhang points towards the centre of the monomer and a continuous region of density in its vicinity that we have tentatively assigned to the TER subunit (fig. S5), and suggests that the DNA binding TEN domain would be located adjacent to TRDB domain (Fig. 5F) consistent with the TERT domain structure (Fig. 1a). Furthermore, the fitting of the TERT subunit structure into the EM density map of the telomerase dimer reveals that the two catalytic pockets in a telomerase dimer (Fig. 3g-h and Fig. 5 d-e) and 6) are located very far apart, at a distance of 180-190 Å from each other. It will be interesting to unravel the mechanism by which they cooperate for telomerase activity, and indeed whether this distance is required to accommodate to telomeric ends in parallel.

Future work will aim at improving the resolution of the telomerase structure in order to reveal the fold of the telomerase RNA and its specific interaction with the various TERT domains as well as how the different conformational states we observe may be related to different functional states of the telomerase catalytic cycle. This study also provides the starting point for investigating the structural basis of telomerase dimerization and its significance for catalysis and function.

Online MethodsTelomerase expression and purificationThe telomerase complex was over-expressed using HEK293T cells1 transiently transfected with hTERT- and hTER-expressing plasmid DNA (pVan145 [pCDNA6-ZZ-(TEV)-3×Flag-hTERT(WT)], [pCDNA6-ZZ-(TEV)-3×Flag- hTERT(DN)], or [pCDNA6-13×Myc-hTERT(WT)], and pBS-U1-hTR) as described2 and whole-cell lysates (WCL) were prepared as described3. For telomerase affinity purification, the WCL was adjusted to a protein concentration of 4 mg/ml with buffer A (20 mM HEPES- KOH pH 7.9, 2 mM MgCl2, 300 mM KCl, 10 % glycerol (v/v), 1 mM DTT, 1 mM EDTA, 0.1 % Triton X-100 (v/v), 1 mM PMSF) and clarified by filtration to produce the Input. To 20 ml Input 1 ml buffer A equilibrated IgG –Sepharose 6 Fast Flow (GE Healthcare #17-0969-01) was added and rotated for 3 hrs at 4°C and washed extensively with ice-cold buffer A. Telomerase was released from 1ml IgG –Sepharose by overnight incubation with 5 ml 0.05 mg/ml TEV protease (>90% efficient) in buffer A at 4°C. In a second affinity purification step, the soluble telomerase containing fraction was applied to a 1 ml HiTrap Heparin HP column (GE Healthcare #17-0406-01) previously equilibrated with buffer A and washed with 10 column volumes of buffer A to remove unbound proteins. Bound proteins were eluted using a gradient of 0.3 – 1 M KCl in buffer A. Fractions containing telomerase were identified using a direct telomerase activity assay (Fig. 1c) as described3 and active fractions were pooled, concentrated, and dialyzed against buffer B (20 mM HEPES-KOH pH 7.9, 2 mM MgCl2, 150 mM KCl, 10 % glycerol (v/v), 1 mM DTT, 1 mM EDTA, 1 mM PMSF). Telomerase yields were quantified by Northern blot analysis using hTR run-off transcripts as standards and the purity was assessed by SDS-PAGE.Telomerase sucrose gradient sedimentation in combination with fixation (GraFix)For EM analysis the telomerase complex was further purified using GraFix4. Briefly, solutions of buffer B containing 5 % and 30 % Sucrose were prepared in Falcon tubes. 0.1% glutaraldehyde (EM grade 25%, Science Services GmbH, Munich, Germany) was added to the high-density solution only. 4 ml gradients for centrifugation using a Beckman SW60 rotor were prepared using a gradient former (Gradient Master 107, BioComp Instruments, Canada). Freshly prepared gradients were kept at 4°C for one hr before sample loading. Subsequently 200 μl, containing 10-30 pmol of purified telomerase bound to 5′-[32P]-(TTAGGG)2-3′ was loaded on top of the gradient and run for 20 hr at max 215,000 × g and 4°C. Sucrose gradients were run in parallel without fixation reagent to assess whether the presence of glutaraldehyde caused any differences in particle sedimentation. 200 μl fractions were collected manually from top to bottom (Fig. 1b). The glutaraldehyde was inactivated after fractionation by adding glycine to a final concentration of 80 mM. For EM analysis, the sucrose from the GraFix fractions was removed by applying the sample to a buffer exchange spin column (Zeba spin columns, Pierce, Rockford, IL, USA) previously equilibrated with buffer C (20 mM Tris – HCl pH 7.6, 150 mM KCl, 1 mM MgCl2).SDS-PAGE and E MSA analysisFor SDS-PAGE analysis, 10 μl of extract were directly boiled for 5 min in Laemmli sample buffer and fractionated on a 4–20 % SDS–PAGE gradient gel. Proteins were silver stained. For the EMSA analysis, a two fold dilution series of IgG-Sepharose/Heparin purified telomerase fraction was incubated with 250 nM 32P-(TTAGGG)3 telomeric DNA in the presence of 5 μM CTAGACCTGTCATCA competitor DNA and 0.1 mg/ml acetylated BSA in heparin binding buffer (20 mM HEPES-KOH pH7.9, 300 mM KCl, 2 mM MgCl2, 1 mM EDTA pH8, 1 mM DTT, 10 % glycerol). Reactions were incubated at RT for 5 min before electrophoresis on a 6.7% acrylamide native gel (80:1 acrylamide:bis acrylamide, 5% glycerol, 0.5× Tris borate-EDTA) run at 200 V for 15 h at 4°C and fixed in MeOH/Acetate. As control, oligo DNA was incubated without telomerase. The gel was exposed to X-ray film.Nano-liquid chromatography tandem mass spectrometry (nanoLC-MS/MS)Analysis of the protein content of the purified telomerase samples was carried out by adding 20 μl of a trypsin solution (10 ng/μl in 10 mM Tris/2 mM CaCl2, pH 8.2) to a 200-μl aliquot (estimated at 0.5 pmol) of the purified telomerase or control samples. Microwave (CEM Discover, CEM Corp., USA) assisted trypsin digestion was performed at 60°C at 5 W for 30 min. After digestion the sample was concentrated to approximately 50 μl in a vacuum centrifuge (Savant Instruments Inc., USA), diluted with 50 μl 0.1% formic acid and 4 μl was loaded onto a nano-UPLC system (nanoAcquity UPLC system, Waters, USA) through a trap-column (Symmetry, C18, 180 μm i.d. x 20 mm length, Waters, USA). Peptides were separated on a C18-column (BEH300 C18, 75 μm i.d. x 150 mm length, Waters, USA) at 250 nl/min using a linear gradient from 1 to 60% solvent B (solvent A: 0.1% formic acid in water, solvent B: 0.1% formic acid in acetonitrile) in 60 min. Mass spectra were recorded on a Synapt G2 HDMS (Waters, USA). Information-dependent data acquisition was carried out using a 0.5-second survey scan from which the five most abundant doubly and triply charged peptides were selected for product ion scans. The resulting data were searched against the SwissProt database using the ProteinLynx Global Server (Waters, USA) search engine (table S1).Direct Telomerase Activity AssaysConventional telomerase assays were carried out for 45 min at 30°C in 20-μl reactions containing 1 μl of cell extract, 50mM Tris– HCl (pH 8.0), 50mM KCl, 1mM spermidine, 1mM β-mercaptoethanol, 1mM MgCl2, 0.5mM dATP, 0.5mM dTTP, 2 mM dGTP, 20 μCi of [α-32P]dGTP (3000 Ci/mmol) and 1 μM of telomeric primer 5′-Biotin(T2AG3)-3′. Reactions were stopped by the addition of 5mM EDTA and 1% SDS. Biotinylated telomeric primers were recovered with Dynabeads® M-280 Streptavidin (Invitrogen) following the instructions provided by the manufacturer. Beads were re-suspended in 98% formamide containing 10mM EDTA and 0.005% xylene cyanol, heated to 95 °C for 5 min, and analyzed on a 12% polyacrylamide-urea sequencing gels.Western analysisFor Western analysis, 4 μl of extract were boiled for 5 min in Laemmli sample buffer and fractionated on 4–15 % SDS–PAGE gradient gels. Immunoblots with standard protocols, using an affinity purified rabbit polyclonal antibody against hTERT (1:10000, R484) 5. As secondary IRDye 800CW Goat anti-Rabbit IgG (Li Cor) was used (1:7000). The Li-Cor Infrared fluorescence detection of IRDye infrared dye substrate and imaging system was used to detect bound antibodies.Telospot assayTelospot telomerase activity assays were performed as previously described6. Briefly, telomerase activity reactions were carried out for 45 min at 30 °C in 20 μl volumes containing 1 μl of cell extracts, 50mM Tris– HCl (pH 8.0), 50mM KCl, 1mM spermidine, 1mM β-mercaptoethanol, 1mM MgCl2, 33 μM dATP, 33 μM dTTP, 33 μM dGTP, and 0.25 mM of telomeric primer (T2AG3)3. Reactions were stopped by the addition of 2 μl of 0.25M EDTA / 0.05% bromophenol blue solution. 0.5 μl of the reaction mix was spotted in quadruplicate on a GeneScreen Plus charged nylon membrane (PerkinElmer). DNA samples were UV crosslinked to the membrane with the auto-crosslink function of a Stratalinker (Stratagene). Without prior denaturation membranes were blocked for 1 hr at 60 °C with church buffer. After overnight hybridization at 60 °C with a randomly labeled probe derived from a 600 bp TTAGGG-repeat containing DNA fragment, membranes were washed twice for 15 min with 2× SSC buffer at room temperature and twice for 30 min with 2× SSC, 0.1 % SDS at 65 °C and exposed to a phosphoimager screen. Spot intensities were quantified using 2D-densitometry and the Aida software (Raytest).EM sample preparation and imagingContinuous carbon-coated grids were freshly prepared and glow-discharged before use. 13 μl of telomerase sample (8-10 nM) were deposited on the grid for 15-30 minutes, blotted with filter paper and negatively stained with 2 drops of 1-2% (w/v) uranyl acetate solution. A homogenous sample preparation for EM was obtained using these conditions although the particle concentration was low (30-100 particles/image). Single particle EM data were collected on a FEI CM12 transmission electron microscope, operated at 120 keV and liquid nitrogen temperature. The nominal magnification was 42,000× (calibrated magnification, 42,550×). Micrographs were recorded on Kodak SO-163 film with an electron dose of 10-12 e/Å2 and a defocus of 1.0–1.5 μm. The films were developed in Kodak developer at full strength for 12 min and digitized with a Zeiss SCAI scanner using a step size of 7 μm. The micrographs were compressed x4 giving a final object pixel size of 6.6Å/pixel. Figures 1d and colloidal gold data were recorded on a 2k TVIPS CCD camera with an object pixel size of 3.25 Å. Single axis tilt series were recorded with SerialEM7 on a FEI Tecnai G2 Polara microscope at 300keV and 3.5 μm underfocus. The specimen was tilted ±60° or ±65° and images were recorded every second degree on a 2k TVIPS CCD detector. The electron dose per image was 10 e/Å2 and the pixel size was 5.65 Å at the specimen level.Particle selection and 2D class averagingA data set of 26,361 telomerase particles bound to oligonucleotide 5′-(TTAGGG)2- 3′ were selected manually with Ximdisp8 using a box size of 78×78 pixels. The image stack was normalised and sorted by statistics in Xmipp9, and 20,127 images were selected for further analysis. A maximum likelihood target function in Fourier space (MLF2D10) was used for multi-reference alignment and 2D classification.Initial low-resolution 3D reconstruction by electron tomographyElectron tomography was used to calculate a low-resolution reference model. Single axis tilt series were compressed x2 giving a pixel size of 11.3 Å at the specimen level. The micrographs were coarsely aligned by cross-correlation and reconstructed by filtered back-projection in IMOD11. An iterative refinement procedure was used to refine the alignment parameters and the reconstruction in EMTIAR12. 50 evenly stained sub-tomograms corresponding to individual telomerase particles were manually selected in Bsoft13 and extracted using a box size of 40×40×40 voxels. An average 3D reconstruction was calculated by missing-wedge weighted, reference-free alignment of sub-tomograms in Xmipp14. 13 sub-tomograms with the highest cross correlation to the average (Fig. 1f and fig. S3a) were included in the final sub-tomogram average.Map refinement by single particle electron microscopyInitial single-particle refinement was performed in EMAN215 using the sub-tomogram average (fig. S3a) as a starting reference. The reference model was initially filtered to 80 Å resolution and refined (fig. S3b) by 20 rounds of multi-reference alignment, classification by multivariate statistical analysis and angular assignment by projection matching. Class averages with 6 degree angular spacing were calculated using 6 averaging iterations. The 3D structure was calculated at 35 Å resolution by direct Fourier inversion. To analyse structural variability in the telomerase dimer we used 3D-classification by maximum-likelihood in Fourier space (MLF3D10). After 25 rounds of non-supervised angular refinement against five reference models, four homogenous subsets of images were selected for further refinement (a fifth class was interpreted as an accumulation of particles of low quality). The selected classes contained 2608, 2631, 3659 and 5519 particles respectively. Subsequent refinement in EMAN2 (as described above) yielded reconstructions with resolutions in the range of 30-40 Å (fig. S3c-f).Map validation and absolute hand determinationThe absolute hand and the overall correctness of the dimer structure in Figure 3f were assessed using the tilt-pair validation method16. From one of the tilt series that was used to calculate electron tomograms for our reference-free initial model calculation, we selected the individual projections of one of the dimer particles (selection of other particles gave similar results). These single-particle projections were then aligned independently against reference projections of the dimer structure. Analysis of the relative tilt angles between these images (fig. S4) validated the structure, while comparison with an identical analysis for a tilt series that was collected on the same microscope on a sample of known handedness (helical assemblies of acetyl choline receptors) served to determine its absolute hand.Independent monomer refinement and reconstruction of composite dimersFrom 2D class averages of telomerase dimer side-views, we calculated the centre of each monomer in the raw image stack; 9,380 sub-images were selected of the open monomer and the closed monomer (4,690 each), and extracted using a box size of 34×34 pixels. These sub-images were used for independent monomer refinement, using a two-reference model procedure, in Xmipp (Fig. 4-5). To re-combine the monomers into composite dimers, reference-free 2D class averages (Fig. 4c top panel), were used to determine the orientation of the open and closed monomers (calculated as the average orientation of all sub-images that were included in the 2D class average). This analysis provided a reconstruction of the relative orientation of the independely refined monomers (Fig. 4c middle panel). Since the analysis was carried out using 2D class averages of side views, the Z-height of the monomers were adjusted by hand and compared to the previously refined intact dimer (Fig. 3f). Visual inspection shows that projections of the assembled composite dimer (Fig. 4c, lower panel) agreed well with the 2D class averages (Fig. 4c top panel).Colloidal gold labeling of telomerase domainsTwo types of 5 nm colloidal gold, coated with either streptavidin (prepared as described 17) or Ni-NTA (Nanoprobes, Inc Yaphank, NY, USA), was used to identify the presence and location of the telomeric G-overhang and the TERT subunit respectively. The binding integrity of the monovalent streptavidin gold complex to 5′-[32P]/biotin 5′-[32P]-(TTAGGG)2-3′ -3′ was analysed by EMSA. After GraFix purification of the telomerase (as described above), streptavidin coated gold was added in a stochiometric ratio of 2:1 to telomerase bound to 5′- Biotin dT-(TTAGGG)2- 3′ and incubated for 30 minutes on ice before preparing EM grids (Fig. 1g). To map the location of the TERT subunit, telomerase N-terminal 6×His-tag (ZZ_(TEV)_6×His_5x(AAAKE) ~37Å long alpha helix_3x_Flag_hTERT) was incubated with Ni-NTA coated gold for 3 hours at room temperature (Fig. 5d). We found more telomerase particles in complex with streptavidin coated gold (87%) compared to Ni-NTA coated gold (16%), which is likely to reflect the high affinity of streptavidin-biotin (not shown).Model fitting and density visualizationThe crystal structure of the TERT protein subunit (pdb code: 3KYL) was docked into the EM density of the monomeric subunit using automatic docking procedures in Chimera18. This program was also used for 3D density visualization and generating a 23 Å density map of the TERT subunit (Fig. S5b).

Online Methods

Telomerase expression and purificationThe telomerase complex was over-expressed using HEK293T cells1 transiently transfected with hTERT- and hTER-expressing plasmid DNA (pVan145 [pCDNA6-ZZ-(TEV)-3×Flag-hTERT(WT)], [pCDNA6-ZZ-(TEV)-3×Flag- hTERT(DN)], or [pCDNA6-13×Myc-hTERT(WT)], and pBS-U1-hTR) as described2 and whole-cell lysates (WCL) were prepared as described3. For telomerase affinity purification, the WCL was adjusted to a protein concentration of 4 mg/ml with buffer A (20 mM HEPES- KOH pH 7.9, 2 mM MgCl2, 300 mM KCl, 10 % glycerol (v/v), 1 mM DTT, 1 mM EDTA, 0.1 % Triton X-100 (v/v), 1 mM PMSF) and clarified by filtration to produce the Input. To 20 ml Input 1 ml buffer A equilibrated IgG –Sepharose 6 Fast Flow (GE Healthcare #17-0969-01) was added and rotated for 3 hrs at 4°C and washed extensively with ice-cold buffer A. Telomerase was released from 1ml IgG –Sepharose by overnight incubation with 5 ml 0.05 mg/ml TEV protease (>90% efficient) in buffer A at 4°C. In a second affinity purification step, the soluble telomerase containing fraction was applied to a 1 ml HiTrap Heparin HP column (GE Healthcare #17-0406-01) previously equilibrated with buffer A and washed with 10 column volumes of buffer A to remove unbound proteins. Bound proteins were eluted using a gradient of 0.3 – 1 M KCl in buffer A. Fractions containing telomerase were identified using a direct telomerase activity assay (Fig. 1c) as described3 and active fractions were pooled, concentrated, and dialyzed against buffer B (20 mM HEPES-KOH pH 7.9, 2 mM MgCl2, 150 mM KCl, 10 % glycerol (v/v), 1 mM DTT, 1 mM EDTA, 1 mM PMSF). Telomerase yields were quantified by Northern blot analysis using hTR run-off transcripts as standards and the purity was assessed by SDS-PAGE.

Telomerase expression and purification

The telomerase complex was over-expressed using HEK293T cells1 transiently transfected with hTERT- and hTER-expressing plasmid DNA (pVan145 [pCDNA6-ZZ-(TEV)-3×Flag-hTERT(WT)], [pCDNA6-ZZ-(TEV)-3×Flag- hTERT(DN)], or [pCDNA6-13×Myc-hTERT(WT)], and pBS-U1-hTR) as described2 and whole-cell lysates (WCL) were prepared as described3. For telomerase affinity purification, the WCL was adjusted to a protein concentration of 4 mg/ml with buffer A (20 mM HEPES- KOH pH 7.9, 2 mM MgCl2, 300 mM KCl, 10 % glycerol (v/v), 1 mM DTT, 1 mM EDTA, 0.1 % Triton X-100 (v/v), 1 mM PMSF) and clarified by filtration to produce the Input. To 20 ml Input 1 ml buffer A equilibrated IgG –Sepharose 6 Fast Flow (GE Healthcare #17-0969-01) was added and rotated for 3 hrs at 4°C and washed extensively with ice-cold buffer A. Telomerase was released from 1ml IgG –Sepharose by overnight incubation with 5 ml 0.05 mg/ml TEV protease (>90% efficient) in buffer A at 4°C. In a second affinity purification step, the soluble telomerase containing fraction was applied to a 1 ml HiTrap Heparin HP column (GE Healthcare #17-0406-01) previously equilibrated with buffer A and washed with 10 column volumes of buffer A to remove unbound proteins. Bound proteins were eluted using a gradient of 0.3 – 1 M KCl in buffer A. Fractions containing telomerase were identified using a direct telomerase activity assay (Fig. 1c) as described3 and active fractions were pooled, concentrated, and dialyzed against buffer B (20 mM HEPES-KOH pH 7.9, 2 mM MgCl2, 150 mM KCl, 10 % glycerol (v/v), 1 mM DTT, 1 mM EDTA, 1 mM PMSF). Telomerase yields were quantified by Northern blot analysis using hTR run-off transcripts as standards and the purity was assessed by SDS-PAGE.

Telomerase sucrose gradient sedimentation in combination with fixation (GraFix)For EM analysis the telomerase complex was further purified using GraFix4. Briefly, solutions of buffer B containing 5 % and 30 % Sucrose were prepared in Falcon tubes. 0.1% glutaraldehyde (EM grade 25%, Science Services GmbH, Munich, Germany) was added to the high-density solution only. 4 ml gradients for centrifugation using a Beckman SW60 rotor were prepared using a gradient former (Gradient Master 107, BioComp Instruments, Canada). Freshly prepared gradients were kept at 4°C for one hr before sample loading. Subsequently 200 μl, containing 10-30 pmol of purified telomerase bound to 5′-[32P]-(TTAGGG)2-3′ was loaded on top of the gradient and run for 20 hr at max 215,000 × g and 4°C. Sucrose gradients were run in parallel without fixation reagent to assess whether the presence of glutaraldehyde caused any differences in particle sedimentation. 200 μl fractions were collected manually from top to bottom (Fig. 1b). The glutaraldehyde was inactivated after fractionation by adding glycine to a final concentration of 80 mM. For EM analysis, the sucrose from the GraFix fractions was removed by applying the sample to a buffer exchange spin column (Zeba spin columns, Pierce, Rockford, IL, USA) previously equilibrated with buffer C (20 mM Tris – HCl pH 7.6, 150 mM KCl, 1 mM MgCl2).

Telomerase sucrose gradient sedimentation in combination with fixation (GraFix)

For EM analysis the telomerase complex was further purified using GraFix4. Briefly, solutions of buffer B containing 5 % and 30 % Sucrose were prepared in Falcon tubes. 0.1% glutaraldehyde (EM grade 25%, Science Services GmbH, Munich, Germany) was added to the high-density solution only. 4 ml gradients for centrifugation using a Beckman SW60 rotor were prepared using a gradient former (Gradient Master 107, BioComp Instruments, Canada). Freshly prepared gradients were kept at 4°C for one hr before sample loading. Subsequently 200 μl, containing 10-30 pmol of purified telomerase bound to 5′-[32P]-(TTAGGG)2-3′ was loaded on top of the gradient and run for 20 hr at max 215,000 × g and 4°C. Sucrose gradients were run in parallel without fixation reagent to assess whether the presence of glutaraldehyde caused any differences in particle sedimentation. 200 μl fractions were collected manually from top to bottom (Fig. 1b). The glutaraldehyde was inactivated after fractionation by adding glycine to a final concentration of 80 mM. For EM analysis, the sucrose from the GraFix fractions was removed by applying the sample to a buffer exchange spin column (Zeba spin columns, Pierce, Rockford, IL, USA) previously equilibrated with buffer C (20 mM Tris – HCl pH 7.6, 150 mM KCl, 1 mM MgCl2).

SDS-PAGE and E MSA analysisFor SDS-PAGE analysis, 10 μl of extract were directly boiled for 5 min in Laemmli sample buffer and fractionated on a 4–20 % SDS–PAGE gradient gel. Proteins were silver stained. For the EMSA analysis, a two fold dilution series of IgG-Sepharose/Heparin purified telomerase fraction was incubated with 250 nM 32P-(TTAGGG)3 telomeric DNA in the presence of 5 μM CTAGACCTGTCATCA competitor DNA and 0.1 mg/ml acetylated BSA in heparin binding buffer (20 mM HEPES-KOH pH7.9, 300 mM KCl, 2 mM MgCl2, 1 mM EDTA pH8, 1 mM DTT, 10 % glycerol). Reactions were incubated at RT for 5 min before electrophoresis on a 6.7% acrylamide native gel (80:1 acrylamide:bis acrylamide, 5% glycerol, 0.5× Tris borate-EDTA) run at 200 V for 15 h at 4°C and fixed in MeOH/Acetate. As control, oligo DNA was incubated without telomerase. The gel was exposed to X-ray film.

SDS-PAGE and E MSA analysis

For SDS-PAGE analysis, 10 μl of extract were directly boiled for 5 min in Laemmli sample buffer and fractionated on a 4–20 % SDS–PAGE gradient gel. Proteins were silver stained. For the EMSA analysis, a two fold dilution series of IgG-Sepharose/Heparin purified telomerase fraction was incubated with 250 nM 32P-(TTAGGG)3 telomeric DNA in the presence of 5 μM CTAGACCTGTCATCA competitor DNA and 0.1 mg/ml acetylated BSA in heparin binding buffer (20 mM HEPES-KOH pH7.9, 300 mM KCl, 2 mM MgCl2, 1 mM EDTA pH8, 1 mM DTT, 10 % glycerol). Reactions were incubated at RT for 5 min before electrophoresis on a 6.7% acrylamide native gel (80:1 acrylamide:bis acrylamide, 5% glycerol, 0.5× Tris borate-EDTA) run at 200 V for 15 h at 4°C and fixed in MeOH/Acetate. As control, oligo DNA was incubated without telomerase. The gel was exposed to X-ray film.

Nano-liquid chromatography tandem mass spectrometry (nanoLC-MS/MS)Analysis of the protein content of the purified telomerase samples was carried out by adding 20 μl of a trypsin solution (10 ng/μl in 10 mM Tris/2 mM CaCl2, pH 8.2) to a 200-μl aliquot (estimated at 0.5 pmol) of the purified telomerase or control samples. Microwave (CEM Discover, CEM Corp., USA) assisted trypsin digestion was performed at 60°C at 5 W for 30 min. After digestion the sample was concentrated to approximately 50 μl in a vacuum centrifuge (Savant Instruments Inc., USA), diluted with 50 μl 0.1% formic acid and 4 μl was loaded onto a nano-UPLC system (nanoAcquity UPLC system, Waters, USA) through a trap-column (Symmetry, C18, 180 μm i.d. x 20 mm length, Waters, USA). Peptides were separated on a C18-column (BEH300 C18, 75 μm i.d. x 150 mm length, Waters, USA) at 250 nl/min using a linear gradient from 1 to 60% solvent B (solvent A: 0.1% formic acid in water, solvent B: 0.1% formic acid in acetonitrile) in 60 min. Mass spectra were recorded on a Synapt G2 HDMS (Waters, USA). Information-dependent data acquisition was carried out using a 0.5-second survey scan from which the five most abundant doubly and triply charged peptides were selected for product ion scans. The resulting data were searched against the SwissProt database using the ProteinLynx Global Server (Waters, USA) search engine (table S1).

Nano-liquid chromatography tandem mass spectrometry (nanoLC-MS/MS)

Analysis of the protein content of the purified telomerase samples was carried out by adding 20 μl of a trypsin solution (10 ng/μl in 10 mM Tris/2 mM CaCl2, pH 8.2) to a 200-μl aliquot (estimated at 0.5 pmol) of the purified telomerase or control samples. Microwave (CEM Discover, CEM Corp., USA) assisted trypsin digestion was performed at 60°C at 5 W for 30 min. After digestion the sample was concentrated to approximately 50 μl in a vacuum centrifuge (Savant Instruments Inc., USA), diluted with 50 μl 0.1% formic acid and 4 μl was loaded onto a nano-UPLC system (nanoAcquity UPLC system, Waters, USA) through a trap-column (Symmetry, C18, 180 μm i.d. x 20 mm length, Waters, USA). Peptides were separated on a C18-column (BEH300 C18, 75 μm i.d. x 150 mm length, Waters, USA) at 250 nl/min using a linear gradient from 1 to 60% solvent B (solvent A: 0.1% formic acid in water, solvent B: 0.1% formic acid in acetonitrile) in 60 min. Mass spectra were recorded on a Synapt G2 HDMS (Waters, USA). Information-dependent data acquisition was carried out using a 0.5-second survey scan from which the five most abundant doubly and triply charged peptides were selected for product ion scans. The resulting data were searched against the SwissProt database using the ProteinLynx Global Server (Waters, USA) search engine (table S1).

Direct Telomerase Activity AssaysConventional telomerase assays were carried out for 45 min at 30°C in 20-μl reactions containing 1 μl of cell extract, 50mM Tris– HCl (pH 8.0), 50mM KCl, 1mM spermidine, 1mM β-mercaptoethanol, 1mM MgCl2, 0.5mM dATP, 0.5mM dTTP, 2 mM dGTP, 20 μCi of [α-32P]dGTP (3000 Ci/mmol) and 1 μM of telomeric primer 5′-Biotin(T2AG3)-3′. Reactions were stopped by the addition of 5mM EDTA and 1% SDS. Biotinylated telomeric primers were recovered with Dynabeads® M-280 Streptavidin (Invitrogen) following the instructions provided by the manufacturer. Beads were re-suspended in 98% formamide containing 10mM EDTA and 0.005% xylene cyanol, heated to 95 °C for 5 min, and analyzed on a 12% polyacrylamide-urea sequencing gels.

Direct Telomerase Activity Assays

Conventional telomerase assays were carried out for 45 min at 30°C in 20-μl reactions containing 1 μl of cell extract, 50mM Tris– HCl (pH 8.0), 50mM KCl, 1mM spermidine, 1mM β-mercaptoethanol, 1mM MgCl2, 0.5mM dATP, 0.5mM dTTP, 2 mM dGTP, 20 μCi of [α-32P]dGTP (3000 Ci/mmol) and 1 μM of telomeric primer 5′-Biotin(T2AG3)-3′. Reactions were stopped by the addition of 5mM EDTA and 1% SDS. Biotinylated telomeric primers were recovered with Dynabeads® M-280 Streptavidin (Invitrogen) following the instructions provided by the manufacturer. Beads were re-suspended in 98% formamide containing 10mM EDTA and 0.005% xylene cyanol, heated to 95 °C for 5 min, and analyzed on a 12% polyacrylamide-urea sequencing gels.

Western analysisFor Western analysis, 4 μl of extract were boiled for 5 min in Laemmli sample buffer and fractionated on 4–15 % SDS–PAGE gradient gels. Immunoblots with standard protocols, using an affinity purified rabbit polyclonal antibody against hTERT (1:10000, R484) 5. As secondary IRDye 800CW Goat anti-Rabbit IgG (Li Cor) was used (1:7000). The Li-Cor Infrared fluorescence detection of IRDye infrared dye substrate and imaging system was used to detect bound antibodies.

Western analysis

For Western analysis, 4 μl of extract were boiled for 5 min in Laemmli sample buffer and fractionated on 4–15 % SDS–PAGE gradient gels. Immunoblots with standard protocols, using an affinity purified rabbit polyclonal antibody against hTERT (1:10000, R484) 5. As secondary IRDye 800CW Goat anti-Rabbit IgG (Li Cor) was used (1:7000). The Li-Cor Infrared fluorescence detection of IRDye infrared dye substrate and imaging system was used to detect bound antibodies.

Telospot assayTelospot telomerase activity assays were performed as previously described6. Briefly, telomerase activity reactions were carried out for 45 min at 30 °C in 20 μl volumes containing 1 μl of cell extracts, 50mM Tris– HCl (pH 8.0), 50mM KCl, 1mM spermidine, 1mM β-mercaptoethanol, 1mM MgCl2, 33 μM dATP, 33 μM dTTP, 33 μM dGTP, and 0.25 mM of telomeric primer (T2AG3)3. Reactions were stopped by the addition of 2 μl of 0.25M EDTA / 0.05% bromophenol blue solution. 0.5 μl of the reaction mix was spotted in quadruplicate on a GeneScreen Plus charged nylon membrane (PerkinElmer). DNA samples were UV crosslinked to the membrane with the auto-crosslink function of a Stratalinker (Stratagene). Without prior denaturation membranes were blocked for 1 hr at 60 °C with church buffer. After overnight hybridization at 60 °C with a randomly labeled probe derived from a 600 bp TTAGGG-repeat containing DNA fragment, membranes were washed twice for 15 min with 2× SSC buffer at room temperature and twice for 30 min with 2× SSC, 0.1 % SDS at 65 °C and exposed to a phosphoimager screen. Spot intensities were quantified using 2D-densitometry and the Aida software (Raytest).

Telospot assay

Telospot telomerase activity assays were performed as previously described6. Briefly, telomerase activity reactions were carried out for 45 min at 30 °C in 20 μl volumes containing 1 μl of cell extracts, 50mM Tris– HCl (pH 8.0), 50mM KCl, 1mM spermidine, 1mM β-mercaptoethanol, 1mM MgCl2, 33 μM dATP, 33 μM dTTP, 33 μM dGTP, and 0.25 mM of telomeric primer (T2AG3)3. Reactions were stopped by the addition of 2 μl of 0.25M EDTA / 0.05% bromophenol blue solution. 0.5 μl of the reaction mix was spotted in quadruplicate on a GeneScreen Plus charged nylon membrane (PerkinElmer). DNA samples were UV crosslinked to the membrane with the auto-crosslink function of a Stratalinker (Stratagene). Without prior denaturation membranes were blocked for 1 hr at 60 °C with church buffer. After overnight hybridization at 60 °C with a randomly labeled probe derived from a 600 bp TTAGGG-repeat containing DNA fragment, membranes were washed twice for 15 min with 2× SSC buffer at room temperature and twice for 30 min with 2× SSC, 0.1 % SDS at 65 °C and exposed to a phosphoimager screen. Spot intensities were quantified using 2D-densitometry and the Aida software (Raytest).

EM sample preparation and imagingContinuous carbon-coated grids were freshly prepared and glow-discharged before use. 13 μl of telomerase sample (8-10 nM) were deposited on the grid for 15-30 minutes, blotted with filter paper and negatively stained with 2 drops of 1-2% (w/v) uranyl acetate solution. A homogenous sample preparation for EM was obtained using these conditions although the particle concentration was low (30-100 particles/image). Single particle EM data were collected on a FEI CM12 transmission electron microscope, operated at 120 keV and liquid nitrogen temperature. The nominal magnification was 42,000× (calibrated magnification, 42,550×). Micrographs were recorded on Kodak SO-163 film with an electron dose of 10-12 e/Å2 and a defocus of 1.0–1.5 μm. The films were developed in Kodak developer at full strength for 12 min and digitized with a Zeiss SCAI scanner using a step size of 7 μm. The micrographs were compressed x4 giving a final object pixel size of 6.6Å/pixel. Figures 1d and colloidal gold data were recorded on a 2k TVIPS CCD camera with an object pixel size of 3.25 Å. Single axis tilt series were recorded with SerialEM7 on a FEI Tecnai G2 Polara microscope at 300keV and 3.5 μm underfocus. The specimen was tilted ±60° or ±65° and images were recorded every second degree on a 2k TVIPS CCD detector. The electron dose per image was 10 e/Å2 and the pixel size was 5.65 Å at the specimen level.

EM sample preparation and imaging

Continuous carbon-coated grids were freshly prepared and glow-discharged before use. 13 μl of telomerase sample (8-10 nM) were deposited on the grid for 15-30 minutes, blotted with filter paper and negatively stained with 2 drops of 1-2% (w/v) uranyl acetate solution. A homogenous sample preparation for EM was obtained using these conditions although the particle concentration was low (30-100 particles/image). Single particle EM data were collected on a FEI CM12 transmission electron microscope, operated at 120 keV and liquid nitrogen temperature. The nominal magnification was 42,000× (calibrated magnification, 42,550×). Micrographs were recorded on Kodak SO-163 film with an electron dose of 10-12 e/Å2 and a defocus of 1.0–1.5 μm. The films were developed in Kodak developer at full strength for 12 min and digitized with a Zeiss SCAI scanner using a step size of 7 μm. The micrographs were compressed x4 giving a final object pixel size of 6.6Å/pixel. Figures 1d and colloidal gold data were recorded on a 2k TVIPS CCD camera with an object pixel size of 3.25 Å. Single axis tilt series were recorded with SerialEM7 on a FEI Tecnai G2 Polara microscope at 300keV and 3.5 μm underfocus. The specimen was tilted ±60° or ±65° and images were recorded every second degree on a 2k TVIPS CCD detector. The electron dose per image was 10 e/Å2 and the pixel size was 5.65 Å at the specimen level.

Particle selection and 2D class averagingA data set of 26,361 telomerase particles bound to oligonucleotide 5′-(TTAGGG)2- 3′ were selected manually with Ximdisp8 using a box size of 78×78 pixels. The image stack was normalised and sorted by statistics in Xmipp9, and 20,127 images were selected for further analysis. A maximum likelihood target function in Fourier space (MLF2D10) was used for multi-reference alignment and 2D classification.

Particle selection and 2D class averaging

A data set of 26,361 telomerase particles bound to oligonucleotide 5′-(TTAGGG)2- 3′ were selected manually with Ximdisp8 using a box size of 78×78 pixels. The image stack was normalised and sorted by statistics in Xmipp9, and 20,127 images were selected for further analysis. A maximum likelihood target function in Fourier space (MLF2D10) was used for multi-reference alignment and 2D classification.

Initial low-resolution 3D reconstruction by electron tomographyElectron tomography was used to calculate a low-resolution reference model. Single axis tilt series were compressed x2 giving a pixel size of 11.3 Å at the specimen level. The micrographs were coarsely aligned by cross-correlation and reconstructed by filtered back-projection in IMOD11. An iterative refinement procedure was used to refine the alignment parameters and the reconstruction in EMTIAR12. 50 evenly stained sub-tomograms corresponding to individual telomerase particles were manually selected in Bsoft13 and extracted using a box size of 40×40×40 voxels. An average 3D reconstruction was calculated by missing-wedge weighted, reference-free alignment of sub-tomograms in Xmipp14. 13 sub-tomograms with the highest cross correlation to the average (Fig. 1f and fig. S3a) were included in the final sub-tomogram average.

Initial low-resolution 3D reconstruction by electron tomography

Electron tomography was used to calculate a low-resolution reference model. Single axis tilt series were compressed x2 giving a pixel size of 11.3 Å at the specimen level. The micrographs were coarsely aligned by cross-correlation and reconstructed by filtered back-projection in IMOD11. An iterative refinement procedure was used to refine the alignment parameters and the reconstruction in EMTIAR12. 50 evenly stained sub-tomograms corresponding to individual telomerase particles were manually selected in Bsoft13 and extracted using a box size of 40×40×40 voxels. An average 3D reconstruction was calculated by missing-wedge weighted, reference-free alignment of sub-tomograms in Xmipp14. 13 sub-tomograms with the highest cross correlation to the average (Fig. 1f and fig. S3a) were included in the final sub-tomogram average.

Map refinement by single particle electron microscopyInitial single-particle refinement was performed in EMAN215 using the sub-tomogram average (fig. S3a) as a starting reference. The reference model was initially filtered to 80 Å resolution and refined (fig. S3b) by 20 rounds of multi-reference alignment, classification by multivariate statistical analysis and angular assignment by projection matching. Class averages with 6 degree angular spacing were calculated using 6 averaging iterations. The 3D structure was calculated at 35 Å resolution by direct Fourier inversion. To analyse structural variability in the telomerase dimer we used 3D-classification by maximum-likelihood in Fourier space (MLF3D10). After 25 rounds of non-supervised angular refinement against five reference models, four homogenous subsets of images were selected for further refinement (a fifth class was interpreted as an accumulation of particles of low quality). The selected classes contained 2608, 2631, 3659 and 5519 particles respectively. Subsequent refinement in EMAN2 (as described above) yielded reconstructions with resolutions in the range of 30-40 Å (fig. S3c-f).

Map refinement by single particle electron microscopy

Initial single-particle refinement was performed in EMAN215 using the sub-tomogram average (fig. S3a) as a starting reference. The reference model was initially filtered to 80 Å resolution and refined (fig. S3b) by 20 rounds of multi-reference alignment, classification by multivariate statistical analysis and angular assignment by projection matching. Class averages with 6 degree angular spacing were calculated using 6 averaging iterations. The 3D structure was calculated at 35 Å resolution by direct Fourier inversion. To analyse structural variability in the telomerase dimer we used 3D-classification by maximum-likelihood in Fourier space (MLF3D10). After 25 rounds of non-supervised angular refinement against five reference models, four homogenous subsets of images were selected for further refinement (a fifth class was interpreted as an accumulation of particles of low quality). The selected classes contained 2608, 2631, 3659 and 5519 particles respectively. Subsequent refinement in EMAN2 (as described above) yielded reconstructions with resolutions in the range of 30-40 Å (fig. S3c-f).

Map validation and absolute hand determinationThe absolute hand and the overall correctness of the dimer structure in Figure 3f were assessed using the tilt-pair validation method16. From one of the tilt series that was used to calculate electron tomograms for our reference-free initial model calculation, we selected the individual projections of one of the dimer particles (selection of other particles gave similar results). These single-particle projections were then aligned independently against reference projections of the dimer structure. Analysis of the relative tilt angles between these images (fig. S4) validated the structure, while comparison with an identical analysis for a tilt series that was collected on the same microscope on a sample of known handedness (helical assemblies of acetyl choline receptors) served to determine its absolute hand.

Map validation and absolute hand determination

The absolute hand and the overall correctness of the dimer structure in Figure 3f were assessed using the tilt-pair validation method16. From one of the tilt series that was used to calculate electron tomograms for our reference-free initial model calculation, we selected the individual projections of one of the dimer particles (selection of other particles gave similar results). These single-particle projections were then aligned independently against reference projections of the dimer structure. Analysis of the relative tilt angles between these images (fig. S4) validated the structure, while comparison with an identical analysis for a tilt series that was collected on the same microscope on a sample of known handedness (helical assemblies of acetyl choline receptors) served to determine its absolute hand.

Independent monomer refinement and reconstruction of composite dimersFrom 2D class averages of telomerase dimer side-views, we calculated the centre of each monomer in the raw image stack; 9,380 sub-images were selected of the open monomer and the closed monomer (4,690 each), and extracted using a box size of 34×34 pixels. These sub-images were used for independent monomer refinement, using a two-reference model procedure, in Xmipp (Fig. 4-5). To re-combine the monomers into composite dimers, reference-free 2D class averages (Fig. 4c top panel), were used to determine the orientation of the open and closed monomers (calculated as the average orientation of all sub-images that were included in the 2D class average). This analysis provided a reconstruction of the relative orientation of the independely refined monomers (Fig. 4c middle panel). Since the analysis was carried out using 2D class averages of side views, the Z-height of the monomers were adjusted by hand and compared to the previously refined intact dimer (Fig. 3f). Visual inspection shows that projections of the assembled composite dimer (Fig. 4c, lower panel) agreed well with the 2D class averages (Fig. 4c top panel).

Independent monomer refinement and reconstruction of composite dimers

From 2D class averages of telomerase dimer side-views, we calculated the centre of each monomer in the raw image stack; 9,380 sub-images were selected of the open monomer and the closed monomer (4,690 each), and extracted using a box size of 34×34 pixels. These sub-images were used for independent monomer refinement, using a two-reference model procedure, in Xmipp (Fig. 4-5). To re-combine the monomers into composite dimers, reference-free 2D class averages (Fig. 4c top panel), were used to determine the orientation of the open and closed monomers (calculated as the average orientation of all sub-images that were included in the 2D class average). This analysis provided a reconstruction of the relative orientation of the independely refined monomers (Fig. 4c middle panel). Since the analysis was carried out using 2D class averages of side views, the Z-height of the monomers were adjusted by hand and compared to the previously refined intact dimer (Fig. 3f). Visual inspection shows that projections of the assembled composite dimer (Fig. 4c, lower panel) agreed well with the 2D class averages (Fig. 4c top panel).

Colloidal gold labeling of telomerase domainsTwo types of 5 nm colloidal gold, coated with either streptavidin (prepared as described 17) or Ni-NTA (Nanoprobes, Inc Yaphank, NY, USA), was used to identify the presence and location of the telomeric G-overhang and the TERT subunit respectively. The binding integrity of the monovalent streptavidin gold complex to 5′-[32P]/biotin 5′-[32P]-(TTAGGG)2-3′ -3′ was analysed by EMSA. After GraFix purification of the telomerase (as described above), streptavidin coated gold was added in a stochiometric ratio of 2:1 to telomerase bound to 5′- Biotin dT-(TTAGGG)2- 3′ and incubated for 30 minutes on ice before preparing EM grids (Fig. 1g). To map the location of the TERT subunit, telomerase N-terminal 6×His-tag (ZZ_(TEV)_6×His_5x(AAAKE) ~37Å long alpha helix_3x_Flag_hTERT) was incubated with Ni-NTA coated gold for 3 hours at room temperature (Fig. 5d). We found more telomerase particles in complex with streptavidin coated gold (87%) compared to Ni-NTA coated gold (16%), which is likely to reflect the high affinity of streptavidin-biotin (not shown).

Colloidal gold labeling of telomerase domains

Two types of 5 nm colloidal gold, coated with either streptavidin (prepared as described 17) or Ni-NTA (Nanoprobes, Inc Yaphank, NY, USA), was used to identify the presence and location of the telomeric G-overhang and the TERT subunit respectively. The binding integrity of the monovalent streptavidin gold complex to 5′-[32P]/biotin 5′-[32P]-(TTAGGG)2-3′ -3′ was analysed by EMSA. After GraFix purification of the telomerase (as described above), streptavidin coated gold was added in a stochiometric ratio of 2:1 to telomerase bound to 5′- Biotin dT-(TTAGGG)2- 3′ and incubated for 30 minutes on ice before preparing EM grids (Fig. 1g). To map the location of the TERT subunit, telomerase N-terminal 6×His-tag (ZZ_(TEV)_6×His_5x(AAAKE) ~37Å long alpha helix_3x_Flag_hTERT) was incubated with Ni-NTA coated gold for 3 hours at room temperature (Fig. 5d). We found more telomerase particles in complex with streptavidin coated gold (87%) compared to Ni-NTA coated gold (16%), which is likely to reflect the high affinity of streptavidin-biotin (not shown).

Model fitting and density visualizationThe crystal structure of the TERT protein subunit (pdb code: 3KYL) was docked into the EM density of the monomeric subunit using automatic docking procedures in Chimera18. This program was also used for 3D density visualization and generating a 23 Å density map of the TERT subunit (Fig. S5b).

Model fitting and density visualization

The crystal structure of the TERT protein subunit (pdb code: 3KYL) was docked into the EM density of the monomeric subunit using automatic docking procedures in Chimera18. This program was also used for 3D density visualization and generating a 23 Å density map of the TERT subunit (Fig. S5b).

Supplementary Material1video1

Supplementary Material
