---
pmid: '19338310'
title: Streamline proteomic approach for characterizing protein-protein interaction
  network in a RAD52 protein complex.
authors:
- Du Y
- Zhou J
- Fan J
- Shen Z
- Chen X
journal: J Proteome Res
year: '2009'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4284944
doi: 10.1021/pr800662x
---

# Streamline proteomic approach for characterizing protein-protein interaction network in a RAD52 protein complex.
**Authors:** Du Y, Zhou J, Fan J, Shen Z, Chen X
**Journal:** J Proteome Res (2009)
**DOI:** [10.1021/pr800662x](https://doi.org/10.1021/pr800662x)
**PMC:** [PMC4284944](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4284944/)

## Abstract

1. J Proteome Res. 2009 May;8(5):2211-7. doi: 10.1021/pr800662x.

Streamline proteomic approach for characterizing protein-protein interaction 
network in a RAD52 protein complex.

Du Y(1), Zhou J, Fan J, Shen Z, Chen X.

Author information:
(1)Department of Biochemistry and Biophysics, University of North Carolina, 
Chapel Hill, North Carolina 27599-7260, USA.

Large-scale identification of protein-protein interactions (PPIs) in functional 
complexes represents an efficient route to elucidate the regulatory rules of 
cellular functions. Whereas many methods have been developed to identify the 
PPIs associated with particular target/bait protein in complexes, little 
information is available about the interaction relationships among all 
components in a complex. Here, we have established a strategy of integrating 
proteomic identification of complex components with mammalian two-hybrid 
screening of their binary relationships to achieve information content of both 
breadth (i.e., identifying all potential interacting partners of the protein of 
interest) and depth (i.e., detailed mapping of the physical interactions of a 
subset of the identified and functionally related proteins) in characterizing 
protein complexes. In the initial phase of quantitative proteomic analysis of 
this streamline, the proteins that specifically complex with the target/bait 
protein were pulled down by immunoprecipitation and identified by mass 
spectrometry (MS)-based "dual-tagging" quantitative proteomic approach. In the 
second phase of in-depth characterizations of binary relationships, the physical 
interactions of a subset of functionally closely related complex components are 
mapped by mammalian two-hybrid assay. The screening for binary relationships of 
complex components not only serves as a validation of the first phase of 
proteomic identification, but also further deepens the understanding of the 
protein complex of interest. With this streamlined approach, we studied the 
protein complexes that are associated with a DNA recombination protein RAD52. In 
the initial phase, multiple proteins both known and unknown to interact with 
RAD52 were identified by the "dual-tagging" proteomic method. In the second 
phase, a complex protein-protein interaction network, which may play important 
roles in coordinating the activity of DNA repair with that of cell division, was 
defined by the mammalian two-hybrid assay.

DOI: 10.1021/pr800662x
PMCID: PMC4284944
PMID: 19338310 [Indexed for MEDLINE]

## Full Text

Abstract

Large-scale identification of protein-protein interactions (PPIs) in functional complexes represents an efficient route to elucidate the regulatory rules of cellular functions. While many methods have been developed to identify the PPIs associated with particular target/bait protein in complexes, little information is available about the interaction relationships among all components in a complex. Here, we have established a strategy of integrating proteomic identification of complex components with mammalian two-hybrid screening of their binary relationships to achieve information content of both breadth ( i.e ., identifying all potential interacting partners of the protein of interest) and depth ( i.e ., detailed mapping of the physical interactions of a subset of the identified and functionally related proteins) in characterizing protein complexes. In the initial phase of quantitative proteomic analysis of this streamline, the proteins that specifically complex with the target/bait protein were pulled down by immunoprecipitation and identified by mass spectrometry (MS)-based “dual-tagging” quantitative proteomic approach. In the second phase of in-depth characterizations of binary relationships, the physical interactions of a subset of functionally closely related complex components are mapped by mammalian two-hybrid assay. The screening for binary relationships of complex components not only serves as a validation of the first phase of proteomic identification, but also further deepens the understanding of the protein complex of interest. With this streamlined approach, we studied the protein complexes that are associated with a DNA recombination protein RAD52. In the initial phase, multiple proteins both known and unknown to interact with RAD52 were identified by the “dual-tagging” proteomic method. In the second phase, a complex protein-protein interaction network, which may play important roles in coordinating the activity of DNA repair with that of cell division, was defined by the mammalian two-hybrid assay.

Introduction

Protein-protein interactions play important roles in regulating signal transduction and other cellular pathways in all living organisms. Thus, identification and characterization of multiprotein complexes including complex components and their interacting relationships involving in cellular pathways are critical steps for elucidating the regulatory mechanisms in different physiological processes in cells. There are many ways to investigate protein complexes and protein-protein interactions, and each method has its own strength and drawbacks. 1 Traditionally, highly purified protein complexes can be obtained by liquid chromatography through multiple columns and multiple steps of purification. 2 The drawback of this method is that it requires a large quantity of starting materials and is usually time-consuming. Yeast two-hybrid screening is now commonly used to identify the interactions of possible gene products. 3 The strength of the method is that the screening can be performed in the high throughput and systematic manner. 4 , 5 However, there are some well-appreciated caveats to the method, including high level of false-positives and false-negatives in identifications. 6 At protein level, tandem affinity purification (TAP) has been widely used in the isolation and purification of protein complexes at close to natural level. This method was originally designed to identify protein complexes in yeast, 7 and was later modified for mammalian cells. 8 , 9 However, its applications in mammalian cells are not as successful as in yeast. One of the factors that is causing problems in applying this method in mammalian cells is the existence of untagged version of the endogenous target/bait protein which could lead to low yield of the protein complex isolated through TAP tagged bait. 8 In addition to the conventional biochemical methods, various proteomics-based methods have also been developed in the past years to address these concerns. 10 , 11

By integrating stable-isotope labeling, affinity purification, and mass spectrometry, we have previously developed a quantitative “dual-tagging” proteomic approach that allows unambiguous identification of protein complexes from cells after a single step affinity pull-down under mild purification conditions. 12 – 14 In this design, the bait protein is epitope-tagged for affinity isolation of the complex (epitope-tagging), and in parallel the whole proteome of the cells expressing the epitope-tagged bait at physiological relevant levels is labeled with stable isotope-enriched heavy amino acids (isotope-tagging). In mass spectrometric measurements, the heavy amino acids incorporated in the cellular proteins provide “in-spectra” quantitative markers so that those proteins showing the increases in their abundances bound to the bait can be unambiguously identified after single-step affinity purification. Because this method does not require multiple steps of binding and washing, it substantially increases the chances of identifying the proteins that weakly associate with the target protein in a complex. Nonetheless, as other affinity-based methods, such a profiling experiment can only provide a list of target/bait-specific interacting proteins without the information about the physical interaction relationships or binary interactions among the complex components in a network.

To not only identify target-specific interacting partners but also reveal their interaction network in a complex, here we present an integrated strategy of quantitative proteomics for complex component identification with mammalian two-hybrid assay for establishing their interaction relationships. Using this approach, we studied the protein complexes that associated with DNA recombination protein RAD52. In the initial phase, multiple RAD52-interacting proteins both known and unknown were identified by our “dual-tagging” proteomic method. In the second phase, a complex protein-protein interaction network, which may play important roles in coordinating the activity of DNA repair with that of cell division, was defined by the mammalian two-hybrid assay.

Immunoprecipitation and Western Blot Analyses further Validate Parts of the Proteomic and Mammalian Two-Hybrid Results

Consistent with the quantitative proteomic data ( Tables 1 ), Western blotting showed that RPA14 and HSP70 were co-precipitated with RAD52 ( Figure 4A ). As actin, HSP70 is abundant cellular protein. The immunoprecipitation/Western results here reassured that HSP70 is a genuine component of RAD52 complex, but not a contamination. We also explored the potential reasons causing the inconsistency between our proteomic data and the previously reported results. For example, RAD51 has been known to interact with RAD52. 25 , 26 However, RAD51 was not in the list of our proteomic proteins identified ( Table 1 ). When large amounts of starting materials (same as used for proteomic analysis) were used for immunoprecipitation, and longer exposure times (20 min) were used in Western blotting, we did detect a weak RAD51 band ( Figure 4B ). Thus, we assume that the failure of detecting RAD51 as a RAD52 interacting protein in our proteomic analysis might result from low quantity of RAD51 in the immunoprecipitate.

Discussion

In this study, we have used a strategy of integrating proteomic identification with mammalian two-hybrid assay to analyze the RAD52 protein complex. Using a quantitative proteomic strategy ( Figure 1B ), we identified multiple proteins associating with RAD52 ( Table 1 ). Most of the proteins identified in the present study have not been reported previously. The identified proteins are distributed in several different biologically functional areas ( Table 1 ). We then picked up the proteins that function in DNA repair, DNA replication, and cell division for further analysis: the data from literature and the proteomic results in the present study suggest that the proteins function in these areas are to interact with each other to form a complex protein-protein interaction network. We performed mammalian two-hybrid assays to systematically analyze the binary protein-protein interactions for a set of closely related factors, including RAD52, RAD51, β-actin, profiling, HSP70, and three members of RPA protein.

In the mammalian two-hybrid assays, in addition to confirming the already reported protein-protein interactions, we identified multiple new interactions that have not been observed previously by other approaches. Among these interactions identified, the interactions that link DNA repair to cytokinesis are of particular interesting. The two-hybrid results demonstrate that RPA (which is known to interact with RAD52) interact with β-actin, and RAD51 interacts with both β-actin and profilin I ( Figures 2 and 3 ). After the anaphase of mitosis in cell division, a contractile ring forms in the middle of the elongated cells. When the ring constricts, the elongated cell will be cleaved into two daughter cells. 19 Actin is a major component of the contractile ring, and profilin I has been shown to play an important regulatory role in the formation of the ring. 18 Profilin I is an essential protein for cell survival and cell division, and Pfn1-null mice are not viable. 30 In fission yeast Schizosaccharomyces pombe , cells harboring Pfn1-null gene cannot form the contractile ring, are arrested in the cell cycle at cytokinesis, and are not viable. 31 On the other hand, profilin I was also found to play a role in DNA metabolism. For example, partial loss of profilin I expression correlates with the tumorigenic phenotype, suggesting that profilin I functions as a tumor suppressor. 32 The dual functions of profilin I in both cytokinesis and DNA metabolism, and the physical interactions between the important proteins functioning in DNA repair (RAD51 and RAD52) and the essential proteins functioning in cytokinesis (actin and profilin I) ( Figures 2 and 3 ), suggest that DNA repair and cytokinesis may be two functionally inter-connected and inter-regulated cellular processes. Consistent with this notion, it was recently reported that BRCA2 deficiency impairs the completion of cell division by cytokinesis, and the BRCA2-deficient tumors might be caused by the abnormal cytokinesis activities. 33 BRCA2 is an important DNA repair protein, and is required for the repair of the DSBs by recombination. 34 Given the high fidelity of DNA repair, DNA replication in cells, and separation of DNA and other cellular materials into two daughter cells, eukaryotic cells may have evolved molecular mechanisms that coordinate the activities of DNA repair with those of cytokinesis, so the cells containing un-repaired DNA or other genetic errors will not be passed into daughter cells. If this mechanism truly exists, the RAD52 protein complex defined in the present study, which contains the proteins functioning the DNA repair, DNA replication, and cytokinesis, may be one of the important components of this coordination machinery.

Proteomic analysis is powerful in identifying interacting partners of the protein of interest, but not good for detailed mapping of the physical interactions of proteins. On the other hand, the two-hybrid system is useful for analyzing detailed physical interactions of proteins, but not good for screening binding partners of the protein of interest at the proteome level. By coupling the two methods together, we have demonstrated in this study that information content of satisfactory breadth ( i.e ., identifying all potential interacting partners of the protein of interest; Table 1 ) and depth ( i.e ., detailed mapping of the physical interactions of a subset of the identified, functionally related proteins; Figures 2 and 3 ) on protein complex of interest can be obtained simultaneously.
