---
pmid: '26801641'
title: Enrichment of Cdk1-cyclins at DNA double-strand breaks stimulates Fun30 phosphorylation
  and DNA end resection.
authors:
- Chen X
- Niu H
- Yu Y
- Wang J
- Zhu S
- Zhou J
- Papusha A
- Cui D
- Pan X
- Kwon Y
- Sung P
- Ira G
journal: Nucleic Acids Res
year: '2016'
pmcid: PMC4824098
doi: 10.1093/nar/gkv1544
---

# Enrichment of Cdk1-cyclins at DNA double-strand breaks stimulates Fun30 phosphorylation and DNA end resection.
**Authors:** Chen X, Niu H, Yu Y, Wang J, Zhu S, Zhou J, Papusha A, Cui D, Pan X, Kwon Y, Sung P, Ira G
**Journal:** Nucleic Acids Res (2016)
**DOI:** [10.1093/nar/gkv1544](https://doi.org/10.1093/nar/gkv1544)
**PMC:** [PMC4824098](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4824098/)

## Abstract

1. Nucleic Acids Res. 2016 Apr 7;44(6):2742-53. doi: 10.1093/nar/gkv1544. Epub
2016  Jan 21.

Enrichment of Cdk1-cyclins at DNA double-strand breaks stimulates Fun30 
phosphorylation and DNA end resection.

Chen X(1), Niu H(2), Yu Y(3), Wang J(4), Zhu S(4), Zhou J(4), Papusha A(3), Cui 
D(3), Pan X(3), Kwon Y(2), Sung P(5), Ira G(6).

Author information:
(1)College of Life Sciences and Institute for Advanced Studies, Wuhan 
University, Wuhan, Hubei 40072, China xfchen@whu.edu.cn.
(2)Department of Biochemistry and Biophysics, Yale University School of 
Medicine, New Haven, CT, USA.
(3)Department of Molecular ad Human Genetics, Baylor College of Medicine, One 
Baylor Plaza, Houston, TX 77030, USA.
(4)College of Life Sciences and Institute for Advanced Studies, Wuhan 
University, Wuhan, Hubei 40072, China.
(5)Department of Biochemistry and Biophysics, Yale University School of 
Medicine, New Haven, CT, USA patrick.sung@yale.edu.
(6)Department of Molecular ad Human Genetics, Baylor College of Medicine, One 
Baylor Plaza, Houston, TX 77030, USA gira@bcm.edu.

DNA double-strand breaks (DSBs) are one of the most cytotoxic types of DNA 
lesion challenging genome integrity. The activity of cyclin-dependent kinase 
Cdk1 is essential for DSB repair by homologous recombination and for DNA damage 
signaling. Here we identify the Fun30 chromatin remodeler as a new target of 
Cdk1. Fun30 is phosphorylated by Cdk1 on Serine 28 to stimulate its functions in 
DNA damage response including resection of DSB ends. Importantly, Cdk1-dependent 
phosphorylation of Fun30-S28 increases upon DNA damage and requires the 
recruitment of Fun30 to DSBs, suggesting that phosphorylation increases in situ 
at the DNA damage. Consistently, we find that Cdk1 and multiple cyclins become 
highly enriched at DSBs and that the recruitment of Cdk1 and cyclins Clb2 and 
Clb5 ensures optimal Fun30 phosphorylation and checkpoint activation. We propose 
that the enrichment of Cdk1-cyclin complexes at DSBs serves as a mechanism for 
enhanced targeting and modulating of the activity of DNA damage response 
proteins.

© The Author(s) 2016. Published by Oxford University Press on behalf of Nucleic 
Acids Research.

DOI: 10.1093/nar/gkv1544
PMCID: PMC4824098
PMID: 26801641 [Indexed for MEDLINE]

## Full Text

DNA double-strand breaks (DSBs) are one of the most cytotoxic types of DNA lesion challenging genome integrity. The activity of cyclin-dependent kinase Cdk1 is essential for DSB repair by homologous recombination and for DNA damage signaling. Here we identify the Fun30 chromatin remodeler as a new target of Cdk1. Fun30 is phosphorylated by Cdk1 on Serine 28 to stimulate its functions in DNA damage response including resection of DSB ends. Importantly, Cdk1-dependent phosphorylation of Fun30-S28 increases upon DNA damage and requires the recruitment of Fun30 to DSBs, suggesting that phosphorylation increases in situ at the DNA damage. Consistently, we find that Cdk1 and multiple cyclins become highly enriched at DSBs and that the recruitment of Cdk1 and cyclins Clb2 and Clb5 ensures optimal Fun30 phosphorylation and checkpoint activation. We propose that the enrichment of Cdk1-cyclin complexes at DSBs serves as a mechanism for enhanced targeting and modulating of the activity of DNA damage response proteins.

INTRODUCTIONRepair of DSBs is essential for maintaining genome integrity and cell viability. Upon DSB formation, cells arrest in the G2/M phase of the cell cycle to provide time for damage repair before cell cycle progression resumes. DNA damage checkpoint proteins are recruited to DSBs via interaction with DNA repair proteins and their co-localization activates kinase signaling cascade, leading to cell cycle arrest (1). The cyclin-dependent kinase, Cdk1, which controls the entry into and progression through the cell cycle, is also needed for the DNA damage response (2).The activity of Cdk1 fluctuates during the cell cycle and is regulated by its association with cell cycle phase-specific cyclins. In yeast, two G1/S-specific cyclins Cln1 and Cln2 (E cyclins in humans) trigger the G1/S transition, the S-specific cyclins Clb5 and Clb6 (A cyclins in humans) help drive DNA replication, while the four M-specific cyclins Clb1–4 (cyclin B in humans) regulate the progression of mitosis. Cyclins activate Cdk1 and target the kinase to specific substrates (3,4). In addition to cyclins, Cks1, a regulatory subunit of Cdk1, facilitates phosphorylation of the substrates with multiple phosphorylation sites to achieve proper signaling output (5,6). Cdk1 preferentially phosphorylates Serine or Threonine within the optimal consensus sequence S/T-P-x-K/R and can also modify a S/T-P sequence, albeit less efficiently. Most Cdk1 target proteins harbor clusters of consensus sites within structurally disordered regions (7).Upon DNA damage budding yeast cells arrest in G2/M phase with high levels of Cdk1 activities, while in fission yeast and mammalian cells Cdk activity decrease (reviewed in 2). Despite this difference, inhibition of Cdk activities results in a severe deficiency in the DSB repair and checkpoint activation in both yeast and mammals (8–10). Thus, active Cdks are generally essential for the proper response to DSBs in eukaryotes. At the molecular level, Cdks promote the nucleolytic processing of DSB ends into single stranded DNA for the recruitment of homologous recombination (HR) and DNA damage checkpoint proteins (8–12). The control of DSB end resection is critically important for determining the choice of DSB repair pathway. In G1 cells, resection is downregulated which favors non-homologous end joining (NHEJ), while in S and G2 cells, it is up-regulated to promote repair by homologous recombination (HR) (reviewed in 13). Resection is initiated by the MRX complex harboring Mre11-Rad50-Xrs2 (MRE11-RAD50-NBS1 or MRN in humans) in conjunction with the Sae2 (CtIP in humans) protein to generate a limited amount of ssDNA. The MRX complex also recruits resection enzymes, including the nucleases Exo1 and Dna2 and the DNA helicase Sgs1, capable of generating extensive ssDNA (14–21). Cdk1 is needed for both the initial and extensive resection steps (22). In yeast, Sae2 and Dna2 are phosphorylated by Cdk1 (22,23) while in vertebrates CtIP, NBS1 and EXO1 have been found to be targets of Cdks (24–28), with all these phosphorylation events stimulating resection and HR. Besides resection enzymes, Cdk1 regulates other DNA damage response proteins. In fission yeast Cdk dependent phosphorylation of NHEJ enzyme Xlf1 (human XLF/Cernunnos) inhibits NHEJ. In budding yeast Cdk1 is needed for crossover recombination (29) and for the full activation of DNA damage checkpoint in a resection-independent manner (1), where the checkpoint adaptor protein Rad9 has been identified as a Cdk1 substrate (30–33). In addition, the helicase Srs2 and nucleases Mus81/Mms4 and Yen1 that participate in the resolution of recombination intermediates are subject to regulation by Cdk1 (34–38). Herein, we demonstrate that Cdk1 and multiple cyclins are enriched at DSBs and provide evidence that the local enrichment of Cdk1 enhances the DNA damage response and repair by targeting the chromatin remodeling factor Fun30.

Repair of DSBs is essential for maintaining genome integrity and cell viability. Upon DSB formation, cells arrest in the G2/M phase of the cell cycle to provide time for damage repair before cell cycle progression resumes. DNA damage checkpoint proteins are recruited to DSBs via interaction with DNA repair proteins and their co-localization activates kinase signaling cascade, leading to cell cycle arrest (1). The cyclin-dependent kinase, Cdk1, which controls the entry into and progression through the cell cycle, is also needed for the DNA damage response (2).

The activity of Cdk1 fluctuates during the cell cycle and is regulated by its association with cell cycle phase-specific cyclins. In yeast, two G1/S-specific cyclins Cln1 and Cln2 (E cyclins in humans) trigger the G1/S transition, the S-specific cyclins Clb5 and Clb6 (A cyclins in humans) help drive DNA replication, while the four M-specific cyclins Clb1–4 (cyclin B in humans) regulate the progression of mitosis. Cyclins activate Cdk1 and target the kinase to specific substrates (3,4). In addition to cyclins, Cks1, a regulatory subunit of Cdk1, facilitates phosphorylation of the substrates with multiple phosphorylation sites to achieve proper signaling output (5,6). Cdk1 preferentially phosphorylates Serine or Threonine within the optimal consensus sequence S/T-P-x-K/R and can also modify a S/T-P sequence, albeit less efficiently. Most Cdk1 target proteins harbor clusters of consensus sites within structurally disordered regions (7).

Upon DNA damage budding yeast cells arrest in G2/M phase with high levels of Cdk1 activities, while in fission yeast and mammalian cells Cdk activity decrease (reviewed in 2). Despite this difference, inhibition of Cdk activities results in a severe deficiency in the DSB repair and checkpoint activation in both yeast and mammals (8–10). Thus, active Cdks are generally essential for the proper response to DSBs in eukaryotes. At the molecular level, Cdks promote the nucleolytic processing of DSB ends into single stranded DNA for the recruitment of homologous recombination (HR) and DNA damage checkpoint proteins (8–12). The control of DSB end resection is critically important for determining the choice of DSB repair pathway. In G1 cells, resection is downregulated which favors non-homologous end joining (NHEJ), while in S and G2 cells, it is up-regulated to promote repair by homologous recombination (HR) (reviewed in 13). Resection is initiated by the MRX complex harboring Mre11-Rad50-Xrs2 (MRE11-RAD50-NBS1 or MRN in humans) in conjunction with the Sae2 (CtIP in humans) protein to generate a limited amount of ssDNA. The MRX complex also recruits resection enzymes, including the nucleases Exo1 and Dna2 and the DNA helicase Sgs1, capable of generating extensive ssDNA (14–21). Cdk1 is needed for both the initial and extensive resection steps (22). In yeast, Sae2 and Dna2 are phosphorylated by Cdk1 (22,23) while in vertebrates CtIP, NBS1 and EXO1 have been found to be targets of Cdks (24–28), with all these phosphorylation events stimulating resection and HR. Besides resection enzymes, Cdk1 regulates other DNA damage response proteins. In fission yeast Cdk dependent phosphorylation of NHEJ enzyme Xlf1 (human XLF/Cernunnos) inhibits NHEJ. In budding yeast Cdk1 is needed for crossover recombination (29) and for the full activation of DNA damage checkpoint in a resection-independent manner (1), where the checkpoint adaptor protein Rad9 has been identified as a Cdk1 substrate (30–33). In addition, the helicase Srs2 and nucleases Mus81/Mms4 and Yen1 that participate in the resolution of recombination intermediates are subject to regulation by Cdk1 (34–38). Herein, we demonstrate that Cdk1 and multiple cyclins are enriched at DSBs and provide evidence that the local enrichment of Cdk1 enhances the DNA damage response and repair by targeting the chromatin remodeling factor Fun30.

MATERIALS AND METHODSYeast strains and plasmidsAll strains used in this study are derivatives of JKM139 (MATa hoΔ hml::ADE1 hmr::ADE1 ade1–100 leu2–3,112 trp1::hisG lys5 ura3–52 ade3::GAL::HO) unless otherwise stated and are listed in Supplementary Table S1. Strains used for experiments shown in each figure are listed in Supplementary Table S2. Standard polymerase chain reaction (PCR) amplification and transformation were used to create knock-out strains. The point mutants fun30-S20A, fun30-S28A, fun30-S125A, fun30-S20A S28A and fun30-S20A S28A S125A were each constructed using fusion PCR and a pair of overlapping PCR primers with the high fidelity Phusion DNA polymerase (NEB). The PCR products were cloned into the vector pXP735 with a TRP1 promoter inserted between pTEF and KanMX coding sequences. Wild type FUN30 gene or its mutant allele including 490 bp of 5′ untranslated region (UTR), the coding sequence and 300 bp of the 3′ UTR was inserted between the pTEF and TRP1 promoter sequences in the vector. The plasmid was subsequently used as the template to amplify an integration cassette encompassing the 5′ UTR, the coding sequence, the 3′ UTR of FUN30 or fun30 allelic mutant genes and the TRP1 promoter, KanMX gene and a tTEF fragment, as described previously (39). The resultant cassette was transformed into a fun30::natMX mutant in JKM139 or YMV80 strain background. The 5′ UTR region of FUN30 or fun30 and tTEF sequence directed the integration of this cassette into the genome by homologous recombination. The transformants were selected for G418 resistance and verified by DNA sequencing. Thus, the resultant strains express the wild type FUN30 gene or its mutant allele under the control of endogenous regulatory elements. The sequences of all oligonucleotide primers used are available upon request.Chromatin immunoprecipitation (ChIP)ChIP was performed as previously described (22) with minor modifications. Briefly, cells were grown to early log phase (≈1 × 107 cells/ml) in YEP-Raffinose. A single DSB at MAT locus on chromosome III was induced by adding galactose to a final concentration of 2%. Cells were fixed by adding 1% formaldehyde and subsequently incubated at room temperature for 10 min with mixing. The reaction was quenched by adding glycine to a final concentration of 125 mM for 5 min at room temperature. Cells were lysed with glass beads on a Mini beadbeater, and the extract was sonicated to shear DNA (on average ≈0.5 kb). IP was performed by using affinity-purified anti-Myc (Sigma M4439), anti-FLAG (Sigma M2) or anti-HA (Abcam) antibodies. The reaction was done at 4°C overnight and then incubated with protein-G agarose resin for 3 h at 4°C. The resin was washed extensively before eluting the bound material. The eluted protein-DNA complexes were incubated overnight at 65°C to reverse crosslinking. After protein digestion by protease K treatment, DNA was purified by phenol extraction and ethanol precipitation. Purified DNA was analyzed by real-time quantitative PCR (ABI 7700) using primers that specifically anneal to DNA sequences located at varying distances from the DSB or at control locus.Analysis of DSB end resectionSouthern blot analysis of resection of the HO break at the MAT locus on chromosome III was performed as previously described (20). Briefly, genomic DNA was isolated using a standard phenol extraction protocol. Purified DNA was digested with EcoRI and separated on a 0.8% agarose gel. Southern blotting and hybridization with radiolabeled DNA probes was performed as described (40). Phosphorimaging was done with a Typhoon TRIO variable mode imager (GE Healthcare) and the intensities of target bands were measured using ImageQuant TL (Amersham Biosciences) and normalized to the TRA1 probe. The value for each time point was further normalized to the value for time ‘0’. Kinetics of resection was measured as previously described (20). At least three independent experiments were performed for each strain.Single strand annealing (SSA) and viability assaysAnalysis of SSA between two partial LEU2 gene repeats was performed as previously described (41). To test the viability in response to DSB, cells were grown in YP-Raffinose overnight to early log phase, and then diluted to a concentration of ≈1 × 103 cells/ml. ≈100 cells were plated on YPED and YP-Gal plates. The viability of cells in response to DSB repaired by SSA was measured by dividing the number of colonies grown on YP-Gal by the number of colonies grown on YPED. Three independent experiments were performed for each strain.Analysis of sensitivity to DNA-damaging agentsYeast cells were grown in YEPD overnight to saturation. 10-fold serial dilutions of tested cultures were spotted onto plates containing camptothecin (CPT), phleomycin or methylmethane sulfonate (MMS) at the indicated concentrations. Plates were incubated at 30 °C for two to three days before taking pictures.In vitro phosphorylation of Fun30 by Cdk1For Fun30 protein expression in Escherichia coli, the FUN30 protein reading frame was amplified and cloned into pET24b to attach a (His)6 tag at the C-terminus of the protein. Site-directed mutagenesis was used to create the expression vector for fun30–3A. The protein expression vectors were transformed into the BL21 Rosetta strain. An overnight culture was diluted into 10L 2xLB media containing 50 μg/ml Kanamycin and 27 μg/ml Chloramphenicol. When the cell density reached OD600 = 0.8, 0.2 mM IPTG was added to induce Fun30 or fun30–3A expression. Cells were cultured at 16°C for 16 h before harvesting. All the subsequent steps were carried out at 4°C. For preparing cell extracts, 50 g of cell pellet was resuspended in 250 ml K buffer (20 mM KH2PO4, pH 7.4, 10% glycerol, 0.5 mM EDTA, 0.01% Igepal, 1 mM DTT) containing 150 mM KCl and a cocktail of protease inhibitors (aprotinin, chymostatin, leupeptin, and pepstatin A at 5 μg/ml each, and also 1 mM phenyl-methylsulfonyl fluoride). The cells were lysed via sonication and the crude extract was clarified by centrifugation at 100 000 x g for 45 min. The clarified supernatant was loaded onto a 5 ml SP Sepharose column, which was washed with 50 ml of K buffer plus 150 mM KCl before being developed with a 50 ml linear gradient from 150 mM to 500 mM KCl, collecting 1 ml fractions. The fractions containing the peak of Fun30 or fun30–3A were pooled and incubated with 1 ml Ni-NTA resin (Qiagen) for 2 h. After washing five times with 10 ml K buffer plus 500 mM KCl, 0.1% NP40 and 20 mM imidazole, the bound proteins were eluted with 3 ml K buffer containing 500 mM KCl and 200 mM imidazole. The eluate was concentrated to 0.3 ml in an Amicon 100K device and loaded onto a column of Superdex 200 10/300GL (GE Healthcare). The column was developed with K buffer plus 500 mM KCl, and the peak fractions containing purified Fun30 or fun30–3A were pooled, concentrated and stored at −80°C as small aliquots.For in vitro kinase reaction, around 250 ng of Fun30 or fun30–3A was incubated with 2 ng of purified Cdk1-Clb2 complex in 10 μl of reaction buffer (40 mM Tris-HCl pH 7.4, 2 mM MgCl2, 2 mM ATP, 100 μg/ml BSA, 1 mM DTT, 50 mM KCl and 5 μCi [γ-32P]-ATP) at 30°C for 20 min. The reaction was stopped by the addition of an equal volume of 2xSDS-PAGE loading buffer and heating at 95°C for 3 min. The reaction mixtures (10 μl) were fractionated by 7.5% SDS-PAGE. The gels were dried and analyzed by phosphorimaging or stained with Coomassie blue.Immunoprecipitation and Western blot analysisWhole cell extracts were prepared using the trichloroacetic acid (TCA) method as previously described (39). Immunoprecipitation was carried out following our published method (39). Proteins from whole-cell extracts or immunoprecipitated samples were resolved on a SDS-PAGE gel and transferred onto a PVDF membrane (Millipore) using a semi-dry transfer cell (Bio-Rad). Mouse monoclonal anti-HA antibody was purchased from Abcam, anti-Myc and anti-FLAG antibodies were purchased from Sigma, and the phosphor-specific rabbit polycolonal antibody against Fun30 phosphorylation on serine 28 was ordered from GenScript (Nanjing). Anti-mouse and rabbit IgG HRP-conjugated secondary antibodies were from Santa Cruz Biotechnology. Blots were developed using the ECL Prime Western Blotting substrate (GE Healthcare).

Yeast strains and plasmidsAll strains used in this study are derivatives of JKM139 (MATa hoΔ hml::ADE1 hmr::ADE1 ade1–100 leu2–3,112 trp1::hisG lys5 ura3–52 ade3::GAL::HO) unless otherwise stated and are listed in Supplementary Table S1. Strains used for experiments shown in each figure are listed in Supplementary Table S2. Standard polymerase chain reaction (PCR) amplification and transformation were used to create knock-out strains. The point mutants fun30-S20A, fun30-S28A, fun30-S125A, fun30-S20A S28A and fun30-S20A S28A S125A were each constructed using fusion PCR and a pair of overlapping PCR primers with the high fidelity Phusion DNA polymerase (NEB). The PCR products were cloned into the vector pXP735 with a TRP1 promoter inserted between pTEF and KanMX coding sequences. Wild type FUN30 gene or its mutant allele including 490 bp of 5′ untranslated region (UTR), the coding sequence and 300 bp of the 3′ UTR was inserted between the pTEF and TRP1 promoter sequences in the vector. The plasmid was subsequently used as the template to amplify an integration cassette encompassing the 5′ UTR, the coding sequence, the 3′ UTR of FUN30 or fun30 allelic mutant genes and the TRP1 promoter, KanMX gene and a tTEF fragment, as described previously (39). The resultant cassette was transformed into a fun30::natMX mutant in JKM139 or YMV80 strain background. The 5′ UTR region of FUN30 or fun30 and tTEF sequence directed the integration of this cassette into the genome by homologous recombination. The transformants were selected for G418 resistance and verified by DNA sequencing. Thus, the resultant strains express the wild type FUN30 gene or its mutant allele under the control of endogenous regulatory elements. The sequences of all oligonucleotide primers used are available upon request.

All strains used in this study are derivatives of JKM139 (MATa hoΔ hml::ADE1 hmr::ADE1 ade1–100 leu2–3,112 trp1::hisG lys5 ura3–52 ade3::GAL::HO) unless otherwise stated and are listed in Supplementary Table S1. Strains used for experiments shown in each figure are listed in Supplementary Table S2. Standard polymerase chain reaction (PCR) amplification and transformation were used to create knock-out strains. The point mutants fun30-S20A, fun30-S28A, fun30-S125A, fun30-S20A S28A and fun30-S20A S28A S125A were each constructed using fusion PCR and a pair of overlapping PCR primers with the high fidelity Phusion DNA polymerase (NEB). The PCR products were cloned into the vector pXP735 with a TRP1 promoter inserted between pTEF and KanMX coding sequences. Wild type FUN30 gene or its mutant allele including 490 bp of 5′ untranslated region (UTR), the coding sequence and 300 bp of the 3′ UTR was inserted between the pTEF and TRP1 promoter sequences in the vector. The plasmid was subsequently used as the template to amplify an integration cassette encompassing the 5′ UTR, the coding sequence, the 3′ UTR of FUN30 or fun30 allelic mutant genes and the TRP1 promoter, KanMX gene and a tTEF fragment, as described previously (39). The resultant cassette was transformed into a fun30::natMX mutant in JKM139 or YMV80 strain background. The 5′ UTR region of FUN30 or fun30 and tTEF sequence directed the integration of this cassette into the genome by homologous recombination. The transformants were selected for G418 resistance and verified by DNA sequencing. Thus, the resultant strains express the wild type FUN30 gene or its mutant allele under the control of endogenous regulatory elements. The sequences of all oligonucleotide primers used are available upon request.

Chromatin immunoprecipitation (ChIP)ChIP was performed as previously described (22) with minor modifications. Briefly, cells were grown to early log phase (≈1 × 107 cells/ml) in YEP-Raffinose. A single DSB at MAT locus on chromosome III was induced by adding galactose to a final concentration of 2%. Cells were fixed by adding 1% formaldehyde and subsequently incubated at room temperature for 10 min with mixing. The reaction was quenched by adding glycine to a final concentration of 125 mM for 5 min at room temperature. Cells were lysed with glass beads on a Mini beadbeater, and the extract was sonicated to shear DNA (on average ≈0.5 kb). IP was performed by using affinity-purified anti-Myc (Sigma M4439), anti-FLAG (Sigma M2) or anti-HA (Abcam) antibodies. The reaction was done at 4°C overnight and then incubated with protein-G agarose resin for 3 h at 4°C. The resin was washed extensively before eluting the bound material. The eluted protein-DNA complexes were incubated overnight at 65°C to reverse crosslinking. After protein digestion by protease K treatment, DNA was purified by phenol extraction and ethanol precipitation. Purified DNA was analyzed by real-time quantitative PCR (ABI 7700) using primers that specifically anneal to DNA sequences located at varying distances from the DSB or at control locus.

ChIP was performed as previously described (22) with minor modifications. Briefly, cells were grown to early log phase (≈1 × 107 cells/ml) in YEP-Raffinose. A single DSB at MAT locus on chromosome III was induced by adding galactose to a final concentration of 2%. Cells were fixed by adding 1% formaldehyde and subsequently incubated at room temperature for 10 min with mixing. The reaction was quenched by adding glycine to a final concentration of 125 mM for 5 min at room temperature. Cells were lysed with glass beads on a Mini beadbeater, and the extract was sonicated to shear DNA (on average ≈0.5 kb). IP was performed by using affinity-purified anti-Myc (Sigma M4439), anti-FLAG (Sigma M2) or anti-HA (Abcam) antibodies. The reaction was done at 4°C overnight and then incubated with protein-G agarose resin for 3 h at 4°C. The resin was washed extensively before eluting the bound material. The eluted protein-DNA complexes were incubated overnight at 65°C to reverse crosslinking. After protein digestion by protease K treatment, DNA was purified by phenol extraction and ethanol precipitation. Purified DNA was analyzed by real-time quantitative PCR (ABI 7700) using primers that specifically anneal to DNA sequences located at varying distances from the DSB or at control locus.

Analysis of DSB end resectionSouthern blot analysis of resection of the HO break at the MAT locus on chromosome III was performed as previously described (20). Briefly, genomic DNA was isolated using a standard phenol extraction protocol. Purified DNA was digested with EcoRI and separated on a 0.8% agarose gel. Southern blotting and hybridization with radiolabeled DNA probes was performed as described (40). Phosphorimaging was done with a Typhoon TRIO variable mode imager (GE Healthcare) and the intensities of target bands were measured using ImageQuant TL (Amersham Biosciences) and normalized to the TRA1 probe. The value for each time point was further normalized to the value for time ‘0’. Kinetics of resection was measured as previously described (20). At least three independent experiments were performed for each strain.

Southern blot analysis of resection of the HO break at the MAT locus on chromosome III was performed as previously described (20). Briefly, genomic DNA was isolated using a standard phenol extraction protocol. Purified DNA was digested with EcoRI and separated on a 0.8% agarose gel. Southern blotting and hybridization with radiolabeled DNA probes was performed as described (40). Phosphorimaging was done with a Typhoon TRIO variable mode imager (GE Healthcare) and the intensities of target bands were measured using ImageQuant TL (Amersham Biosciences) and normalized to the TRA1 probe. The value for each time point was further normalized to the value for time ‘0’. Kinetics of resection was measured as previously described (20). At least three independent experiments were performed for each strain.

Single strand annealing (SSA) and viability assaysAnalysis of SSA between two partial LEU2 gene repeats was performed as previously described (41). To test the viability in response to DSB, cells were grown in YP-Raffinose overnight to early log phase, and then diluted to a concentration of ≈1 × 103 cells/ml. ≈100 cells were plated on YPED and YP-Gal plates. The viability of cells in response to DSB repaired by SSA was measured by dividing the number of colonies grown on YP-Gal by the number of colonies grown on YPED. Three independent experiments were performed for each strain.

Analysis of SSA between two partial LEU2 gene repeats was performed as previously described (41). To test the viability in response to DSB, cells were grown in YP-Raffinose overnight to early log phase, and then diluted to a concentration of ≈1 × 103 cells/ml. ≈100 cells were plated on YPED and YP-Gal plates. The viability of cells in response to DSB repaired by SSA was measured by dividing the number of colonies grown on YP-Gal by the number of colonies grown on YPED. Three independent experiments were performed for each strain.

Analysis of sensitivity to DNA-damaging agentsYeast cells were grown in YEPD overnight to saturation. 10-fold serial dilutions of tested cultures were spotted onto plates containing camptothecin (CPT), phleomycin or methylmethane sulfonate (MMS) at the indicated concentrations. Plates were incubated at 30 °C for two to three days before taking pictures.

Yeast cells were grown in YEPD overnight to saturation. 10-fold serial dilutions of tested cultures were spotted onto plates containing camptothecin (CPT), phleomycin or methylmethane sulfonate (MMS) at the indicated concentrations. Plates were incubated at 30 °C for two to three days before taking pictures.

In vitro phosphorylation of Fun30 by Cdk1For Fun30 protein expression in Escherichia coli, the FUN30 protein reading frame was amplified and cloned into pET24b to attach a (His)6 tag at the C-terminus of the protein. Site-directed mutagenesis was used to create the expression vector for fun30–3A. The protein expression vectors were transformed into the BL21 Rosetta strain. An overnight culture was diluted into 10L 2xLB media containing 50 μg/ml Kanamycin and 27 μg/ml Chloramphenicol. When the cell density reached OD600 = 0.8, 0.2 mM IPTG was added to induce Fun30 or fun30–3A expression. Cells were cultured at 16°C for 16 h before harvesting. All the subsequent steps were carried out at 4°C. For preparing cell extracts, 50 g of cell pellet was resuspended in 250 ml K buffer (20 mM KH2PO4, pH 7.4, 10% glycerol, 0.5 mM EDTA, 0.01% Igepal, 1 mM DTT) containing 150 mM KCl and a cocktail of protease inhibitors (aprotinin, chymostatin, leupeptin, and pepstatin A at 5 μg/ml each, and also 1 mM phenyl-methylsulfonyl fluoride). The cells were lysed via sonication and the crude extract was clarified by centrifugation at 100 000 x g for 45 min. The clarified supernatant was loaded onto a 5 ml SP Sepharose column, which was washed with 50 ml of K buffer plus 150 mM KCl before being developed with a 50 ml linear gradient from 150 mM to 500 mM KCl, collecting 1 ml fractions. The fractions containing the peak of Fun30 or fun30–3A were pooled and incubated with 1 ml Ni-NTA resin (Qiagen) for 2 h. After washing five times with 10 ml K buffer plus 500 mM KCl, 0.1% NP40 and 20 mM imidazole, the bound proteins were eluted with 3 ml K buffer containing 500 mM KCl and 200 mM imidazole. The eluate was concentrated to 0.3 ml in an Amicon 100K device and loaded onto a column of Superdex 200 10/300GL (GE Healthcare). The column was developed with K buffer plus 500 mM KCl, and the peak fractions containing purified Fun30 or fun30–3A were pooled, concentrated and stored at −80°C as small aliquots.For in vitro kinase reaction, around 250 ng of Fun30 or fun30–3A was incubated with 2 ng of purified Cdk1-Clb2 complex in 10 μl of reaction buffer (40 mM Tris-HCl pH 7.4, 2 mM MgCl2, 2 mM ATP, 100 μg/ml BSA, 1 mM DTT, 50 mM KCl and 5 μCi [γ-32P]-ATP) at 30°C for 20 min. The reaction was stopped by the addition of an equal volume of 2xSDS-PAGE loading buffer and heating at 95°C for 3 min. The reaction mixtures (10 μl) were fractionated by 7.5% SDS-PAGE. The gels were dried and analyzed by phosphorimaging or stained with Coomassie blue.

For Fun30 protein expression in Escherichia coli, the FUN30 protein reading frame was amplified and cloned into pET24b to attach a (His)6 tag at the C-terminus of the protein. Site-directed mutagenesis was used to create the expression vector for fun30–3A. The protein expression vectors were transformed into the BL21 Rosetta strain. An overnight culture was diluted into 10L 2xLB media containing 50 μg/ml Kanamycin and 27 μg/ml Chloramphenicol. When the cell density reached OD600 = 0.8, 0.2 mM IPTG was added to induce Fun30 or fun30–3A expression. Cells were cultured at 16°C for 16 h before harvesting. All the subsequent steps were carried out at 4°C. For preparing cell extracts, 50 g of cell pellet was resuspended in 250 ml K buffer (20 mM KH2PO4, pH 7.4, 10% glycerol, 0.5 mM EDTA, 0.01% Igepal, 1 mM DTT) containing 150 mM KCl and a cocktail of protease inhibitors (aprotinin, chymostatin, leupeptin, and pepstatin A at 5 μg/ml each, and also 1 mM phenyl-methylsulfonyl fluoride). The cells were lysed via sonication and the crude extract was clarified by centrifugation at 100 000 x g for 45 min. The clarified supernatant was loaded onto a 5 ml SP Sepharose column, which was washed with 50 ml of K buffer plus 150 mM KCl before being developed with a 50 ml linear gradient from 150 mM to 500 mM KCl, collecting 1 ml fractions. The fractions containing the peak of Fun30 or fun30–3A were pooled and incubated with 1 ml Ni-NTA resin (Qiagen) for 2 h. After washing five times with 10 ml K buffer plus 500 mM KCl, 0.1% NP40 and 20 mM imidazole, the bound proteins were eluted with 3 ml K buffer containing 500 mM KCl and 200 mM imidazole. The eluate was concentrated to 0.3 ml in an Amicon 100K device and loaded onto a column of Superdex 200 10/300GL (GE Healthcare). The column was developed with K buffer plus 500 mM KCl, and the peak fractions containing purified Fun30 or fun30–3A were pooled, concentrated and stored at −80°C as small aliquots.

For in vitro kinase reaction, around 250 ng of Fun30 or fun30–3A was incubated with 2 ng of purified Cdk1-Clb2 complex in 10 μl of reaction buffer (40 mM Tris-HCl pH 7.4, 2 mM MgCl2, 2 mM ATP, 100 μg/ml BSA, 1 mM DTT, 50 mM KCl and 5 μCi [γ-32P]-ATP) at 30°C for 20 min. The reaction was stopped by the addition of an equal volume of 2xSDS-PAGE loading buffer and heating at 95°C for 3 min. The reaction mixtures (10 μl) were fractionated by 7.5% SDS-PAGE. The gels were dried and analyzed by phosphorimaging or stained with Coomassie blue.

Immunoprecipitation and Western blot analysisWhole cell extracts were prepared using the trichloroacetic acid (TCA) method as previously described (39). Immunoprecipitation was carried out following our published method (39). Proteins from whole-cell extracts or immunoprecipitated samples were resolved on a SDS-PAGE gel and transferred onto a PVDF membrane (Millipore) using a semi-dry transfer cell (Bio-Rad). Mouse monoclonal anti-HA antibody was purchased from Abcam, anti-Myc and anti-FLAG antibodies were purchased from Sigma, and the phosphor-specific rabbit polycolonal antibody against Fun30 phosphorylation on serine 28 was ordered from GenScript (Nanjing). Anti-mouse and rabbit IgG HRP-conjugated secondary antibodies were from Santa Cruz Biotechnology. Blots were developed using the ECL Prime Western Blotting substrate (GE Healthcare).

Whole cell extracts were prepared using the trichloroacetic acid (TCA) method as previously described (39). Immunoprecipitation was carried out following our published method (39). Proteins from whole-cell extracts or immunoprecipitated samples were resolved on a SDS-PAGE gel and transferred onto a PVDF membrane (Millipore) using a semi-dry transfer cell (Bio-Rad). Mouse monoclonal anti-HA antibody was purchased from Abcam, anti-Myc and anti-FLAG antibodies were purchased from Sigma, and the phosphor-specific rabbit polycolonal antibody against Fun30 phosphorylation on serine 28 was ordered from GenScript (Nanjing). Anti-mouse and rabbit IgG HRP-conjugated secondary antibodies were from Santa Cruz Biotechnology. Blots were developed using the ECL Prime Western Blotting substrate (GE Healthcare).

ResultsCdk1 phosphorylates Fun30 in vivo and in vitroExtensive resection of 5′ strand at DSB ends is controlled by Cdk1 in yeast and human (22,24). Yeast Dna2 is phosphorylated by Cdk1 to stimulate its nuclear localization and recruitment to DSB ends. However, pseudophosphorylated form of Dna2 which restores nuclear localization of Dna2 is insufficient to bypass the need for Cdk1 for extensive resection, raising the possibility that there are additional targets of Cdk1 important for resection. Fun30, which is required for extensive resection (39,42,43), was identified as a putative Cdk1 target in a genome-wide screen (44). Fun30 and its fission yeast ortholog Fft3 are ATP-dependent chromatin remodeling factors involved also in the assembly of chromatin within centromeres and at silenced mating type loci (45–49). The functions of Fun30 appear to be conserved between yeast and human, as SMARCAD1, the human homolog of Fun30, plays a role in both DSB end resection and silencing (42,50). Yeast Fun30 contains a cluster of Cdk1 consensus sites at its N-terminus. Serine 20 (S20) is conserved among fungi and serine 28 (S28) is well conserved across yeast, mouse and human (Figure 1A), while serine 125 (S125) is not conserved. Serine 28 was identified as a putative target site of Cdk1 in a global proteomics study (44). To verify whether Fun30 is indeed phosphorylated on this evolutionarily conserved site in cells, we raised antibody against phosphorylated Fun30-S28. The antibody is highly specific since it only recognizes the Fun30 with phosphorylated S28 (Figure 1B). Further we observed that S28 phosphorylation occurs in a cell-cycle dependent fashion, in cells arrested in S phase with HU or G2/M phase with nocodazole but not in cells synchronized in G1 using alpha factor or when the conditional allele of Cdk1 kinase (Cdk1-as1) is inhibited by an ATP analogue, 1-NM-PP1 (51) (Figure 1C). To provide biochemical evidence that Cdk1 directly phosphorylates Fun30, we purified Fun30 and fun30–3A proteins and tested them with Cdk1/Clb2 in vitro. Consistent with the in vivo data, we found that Fun30 is efficiently phosphorylated by Cdk1, but little phosphorylation of fun30–3A occurs, indicating that the 3A mutation eliminates the major Cdk1-target site(s) (Figure 1D).Figure 1.Analysis of Fun30-S28 phosphorylation in cell cycle and in response to DNA damage. (A) Alignment of the N-terminal amino acid sequences for Fun30 and its orthologs from indicated species. The evolutionarily conserved serine is marked with an asterisk. (B) Western blot analysis of Fun30-S28 phosphorylation in wild type cells and FUN30 mutants. All FUN30 alleles were fused C-terminally to a sequence encoding 13xMYC. (C) Western blot analysis of Fun30-S28 phosphorylation in cells arrested in different phases of the cell cycle. (D) In vitro phosphorylation of Fun30 and fun30–3A by the Cdk1-Clb2 complex. (E) Western blot analysis of Fun30-S28 phosphorylation in wild type cells with or without drug treatments. (F) Western blot showing Rad53 phosphorylation upon MMS treatment in the wild type strain and indicated mutants. (G,H) Western blot showing Fun30-S28 phosphorylation in wild type cells and indicated mutants upon MMS treatment.Fun30 phosphorylation is needed for its functions in response to DNA damageWe wanted to determine the physiological importance of Fun30 phosphorylation by Cdk1. First, we assessed the role of Fun30 phosphorylation in extensive DSB end resection using the donorless strain wherein the HO-induced DSB cannot be repaired because both sister chromatids are cleaved by HO and the donor sequences HML and HMR used during mating type switching have been deleted. Resection was followed using Southern blots and probes specific for loci at varying distances from the DSB. We measured the rates of extensive resection in wild type cells and in the fun30-S28A, fun30-S20A, fun30-S125A, fun30-S20,28A (fun30–2A), fun30-S20,28,125A (fun30–3A) and fun30Δ mutants. Resection in fun30-S20A, fun30-S28A, fun30–2A and fun30–3A was significantly delayed as compared to wild type cells, nearly as much as in fun30Δ cells. (Figure 2A, Southern blots presented in Supplementary Figure S1). Thus phosphorylation of Fun30 by Cdk1 on S20 and S28 is needed for efficient DSB end resection. To confirm this result, we used Southern blot analysis to assess the impact of the fun30 mutations on DSB repair by single strand annealing (SSA) which requires 25 kb resection between partial leu2 repeats (Figure 2B). To prevent alternative repair pathways the assay was done in the absence of the strand exchange protein, Rad51, which is dispensable for SSA. The results showed that fun30-S28A rad51Δ, fun30-S20A rad51Δ and fun30–2A rad51Δ mutants are unable to complete SSA efficiently (Figure 2C) and consistently viability of these mutants is ≈15 times lower than that of the control rad51Δ cells (Figure 2D). FUN30 mutants that lack key phosphorylation sites (e.g. fun30-S20A, fun30-S28A or fun30-S20A S28A) are sensitive to CPT and MMS, indicating that Fun30 phosphorylation is also needed for the response to DNA damage (Supplementary Figure S2). Extensive resection is also needed for adaptation to DNA damage checkpoint upon the induction of a single non-repairable DSB, such that mutants defective in resection, sgs1Δ, exo1Δ, mec1-ad or fun30Δ mutants are at least partially deficient in adaptation (43,52). Again, we observed that cells harboring non-phosphorylable mutants of FUN30 are partially deficient in the adaptation to the DNA damage checkpoint (Supplementary Figure S2). Together, these results demonstrate the importance of Cdk1-dependent Fun30 phosphorylation in response to DNA damage.Figure 2.Fun30-S28 phosphorylation stimulates resection and DNA damage response. (A) Quantification of extensive resection rates at 28 kb from DSBs in wild type and indicated mutants. Plotted are the average values from three independent experiments. Error bars denote standard deviation. (B) Scheme showing the SSA assay between two partial LEU2 repeats. (C) Southern blot showing the kinetics of DSB repair by SSA in indicated mutants. (D) Viability of indicated mutant cells in response to a DSB repaired by SSA. (E) Analysis of the recruitment of wild type and indicated mutant Fun30 proteins tagged with 13xMyc at 1, 5 or 10 kb from DSB ends. Error bars denote standard deviation (n = 3).Phosphorylation is needed for spreading of Fun30 at DSBPhosphorylation by Cdk1 alters various features of a substrate protein, including its association with partners, subcellular localization, stability and others. We found that the subcellular localization of Fun30 does not change in the cell cycle and it is indistinguishable between the wild type cells and the phosphomutants (Supplementary Figure S3). Using co-immunoprecipitation we found that the known Fun30 interaction with RPA (39), the ssDNA binding protein, is not regulated by Cdk1 (Supplementary Figure S3). Next, we used ChIP to assess whether Fun30 phosphorylation regulates its DSB recruitment. We found that while Fun30, fun30-S28A and fun30–3A are efficiently recruited next to DSB ends, (Figure 2E), the enrichment of phosphomutant proteins is largely impaired or eliminated at 5 kb or 10 kb from the DSB. Based on these results, we conclude that Fun30 phosphorylation by Cdk1 is not needed for initial recruitment to DSBs but rather for its activity and spreading along DNA during resection.Fun30 phosphorylation increases upon DNA damageWe examined Fun30-S28 phosphorylation upon exposure to MMS, and unexpectedly we observed enhanced phosphorylation upon DNA damage (Figure 1E). Similarly, we note that Fun30-S28 phosphorylation was increased in response to the DSB inducing agent phleomycin, but it remained largely unaffected upon treatment with the replication stalling compound hydroxyurea (HU) (Figure 1E). We considered two possibilities to explain these observations. One being that MMS treatment causes cells to arrest in S or G2/M where Cdk1 activity is higher. An alternative explanation is that Fun30 phosphorylation by Cdk1 is enhanced in situ at DNA damage sites. The second model is based on several documented examples where cyclins are recruited to specific subcellular locations where they help target the substrates of Cdks (53,54). Moreover, in human and mouse cells, CDK2 is recruited to DNA damage site via interaction with MRE11 (55). If the first model is correct, we expected to see a correlation between Fun30-S28 phosphorylation and cell cycle arrest upon DNA damage. This possibility is unlikely because Fun30-S28 phosphorylation increases upon MMS treatment even in cells pre-arrested in G2/M with nocodazole (Figure 1E). If the second model holds true, we should find decreased Fun30-S28 phosphorylation in mutants that fail to recruit Fun30 to DNA damage sites. To test this model, we examined Fun30-S28 phosphorylation in mec1Δtel1Δsml1Δ, sgs1Δexo1Δ and h2a-S129A mutants. We previously showed that Fun30 recruitment to DSBs is severely decreased in mec1Δtel1Δsml1Δ and sgs1Δexo1Δ mutants (39) and here we have found that efficient recruitment of Fun30 also requires H2A phosphorylation at serine 129 (Supplementary Figure S4). Previous studies showed that purified Fun30 binds preferentially to the non-phosphorylated form of the H2A in vitro (43). Thus, it is possible that in cells phosphorylated H2A plays an indirect role in Fun30 recruitment. The three aforementioned mutants defective in Fun30 recruitment exhibited distinguishable capacities in DNA damage checkpoint activation as reflected by the phosphorylation of the signaling kinase Rad53 in response to MMS (Figure 1F). In wild type and h2A-S129A cells checkpoint activation is comparable while in mec1Δ tel1Δ sml1Δ and sgs1Δ exo1Δ checkpoint activation is completely or partly deficient. As shown in Figure 1G, decreased Fun30-S28 phosphorylation was observed in all three mutants regardless of their capacity in checkpoint activation, indicating that recruitment of Fun30 to DNA damage sites stimulates its phosphorylation. Together the results favor the second model where Cdk1-dependent phosphorylation of Fun30 increases at DNA damage sites.Multiple Cdk1-cyclin complexes are recruited to DSB endsIf Cdk1 activity is increased at DSBs, we would expect to observe the enrichment of Cdk1 and cyclins. Therefore, we followed the recruitment of Cdk1 and cyclins at the DSB at the MAT locus by chromatin immunoprecipitation (ChIP) coupled with quantitative PCR. Cdk1, Cln2 and Clb1–6 were fused to a three tandem FLAG or HA tag at their C-termini in a strain carrying HO-inducible and unrepairable DSB at MAT. We noted that strain carrying HA-tagged Clb1 was slow growing in YEP-raffinose, therefore it was not included in the analysis. We observed a robust recruitment of Cdk1 and enrichment of multiple cyclins at MAT after DSB induction (Figure 3A,B). Highest enrichment was observed for Cln2, Clb2 and Clb5, while lesser recruitment of Clb4, Clb6 and Clb3 also occurred. We found that the G1/S cyclin Cln2 was present at the DSB 4 h after induction even when most cells are arrested in G2/M. Western blot analysis revealed that there is a low level of Cln2 left by that time (Supplementary Figure S5). Besides cyclins, Cks1 also associates with Cdk1 and plays an important role in targeting substrates that are phosphorylated multiply (5,6). Therefore, we examined the enrichment of Cks1–3xHA protein at the HO-induced DSB site. However, no significant enrichment of Cks1 was observed, suggesting that it does not play a direct role in the DNA damage response (Figure 3C). Together, we conclude that Cdk1 and multiple cyclins are enriched locally at DSBs.Figure 3.Cdk1 and multiple cyclins are recruited to DSBs. (A–C) Analysis of the recruitment of Cdc28–3xFLAG, cyclins and Cks1 to DSB ends at MAT locus by ChIP. ARO1 locus was used as a control. Error bars represent standard deviation (n = 3).Resection, checkpoint response and the cyclins Clb2 and Clb5 are needed for efficient Cdk1 recruitment and Fun30 phosphorylationTo understand how the DSB enrichment of the Cdk1 and cyclins is achieved, we examined their recruitment in mutants deficient in checkpoint signaling or with an altered rate of DSB end resection. We noted that the recruitment of Cdk1 is dramatically reduced in the checkpoint deficient mutant mec1Δ tel1Δ sml1Δ, and it is reduced even more in mre11Δ or sgs1Δ exo1Δ, mutants deficient in either initial or extensive resection (Figure 4A). However, it remains unaffected in the mutants with increased rate of resection, such as yku70Δ or dot1Δ. Given that Cdk activity plays key roles in DNA damage response, we tested whether Cdk1 kinase activity is needed for its own recruitment to DSBs. (51) Inactivation of Cdk1 before DSB induction prevents its own recruitment, likely due to the role of Cdk1 in resection (Figure 4A). Together, these results demonstrate that recruitment of Cdk1 requires formation of ssDNA, damage checkpoint activation, and the kinase activity of Cdk1. It is possible that formation of the 3′ ssDNA at DSBs is indirectly needed for Cdk1 and cyclins recruitment by its role in checkpoint response and/or DNA repair.Figure 4.Genetic requirements for recruitment of Cdk1 and cyclins to DSBs. (A–C) Analysis of recruitment of Cdc28 and cyclins to DSBs by ChIP in indicated mutants. Error bars denote standard deviation (n = 3). *P < 0.05 (n = 3, t-test).Some of the cyclins in mammals play Cdk-independent role in the DNA damage response. For example, cyclin D1 mediates the recruitment of the DNA strand exchange protein RAD51 (56). Thus, we decided to test whether the genetic requirement for the recruitment of Cln2, Clb2 and Clb5 cyclins and Cdk1 is the same or different. Similar to Cdk1, the recruitment for all three cyclins was impaired in mre11Δ, sgs1Δ exo1Δ and mec1Δ tel1Δ sml1Δ cells (Figure 4B). Therefore, Cdk1 and cyclins appear to share the same genetic requirement for their DSB recruitment, suggesting that these proteins are delivered to DSBs as complexes. Further, we assessed whether the robust DSB recruitment of cyclins is important for Cdk1 recruitment. We found that Cdk1 recruitment decreases significantly in the clb2Δ and clb2Δ clb5Δ mutants but not in cln2Δ cells (Figure 4C). Accordingly, we observed that Fun30-S28 phosphorylation is also severely impaired in clb2Δ and clb2Δ clb5Δ cells (Figure 1G). Thus, Clb2 and Clb5 play central roles in Cdk1 recruitment as well as in stimulating Fun30 phosphorylation by Cdk1.Clb2 and Clb5 are needed for resistance of cells to DNA damaging agentsAs cyclins direct Cdk1 toward specific targets, the above results suggest that Cdk1 acts with Cln2, Clb2 and Clb5 to target substrates involved in the DNA damage response. Consistent with a broad role of Cdk1-cyclin complexes, deletion of CLN2, CLB2 or CLB5, the most enriched cyclins at DSBs (Figure 3B), dramatically elevates the sensitivity to DNA damaging agents including MMS, CPT and phleomycin (Figure 5A and ref. 57). Thus, each of the three cyclins plays a unique and indispensable function in the DNA damage response. These results could reflect direct phosphorylation of DNA damage response proteins by the Cdk1-cyclin complexes, or they could stem indirectly from alterations in transcriptional programs or other processes (2). To test whether Cln2, Clb2 and Clb5 share overlapping roles in response to DNA damage, we created cln2Δ clb2Δ, cln2Δ clb5Δ and clb2Δ clb5Δ double mutants and analyzed their response to genotoxins. A mild or lack of genetic interaction in response to DNA damaging agents was observed between the G1 cyclin Cln2 and the S-phase cyclin Clb5 or the M-phase cyclin Clb2. However, we observed both strong positive (CPT, phleomycin) and negative (MMS treatment) genetic interactions between CLB2 and CLB5 (Figure 5B). Once the specific targets of these cyclins are recognized, it should be possible to explain these interactions at the molecular level.Figure 5.The role of cyclins in DNA damage response and DSB end resection. (A,B) Analysis of DNA damage sensitivity for the indicated strains. (C,D) Kinetics of extensive DSB ends resection in wild type or indicated cyclin mutant cells. Error bars denote standard deviation (n = 3).Clb2 and Clb5 cyclins control extensive DSB end resectionClb2, Clb5 and Cln2 appear to be the most important cyclins in the cellular resistance to DNA damaging agents, and they are also most strongly recruited to DSBs. Given the importance of Cdk1 in resection and the DNA damage checkpoint response, we focused on these three cyclins and wished to understand their potential contributions to these processes. We found that clb5Δ cells show a decreased rate of extensive resection (Figure 5C and Supplementary Figure S1). Disruption of the other S phase cyclin Clb6 in clb5Δ cells didn't exacerbate the defect in resection, indicating that Clb5 is the only S-phase cyclin needed for optimal resection (Figure 5D and Supplementary Figure S1). Consistent with the role of Clb5 in promoting resection, we observed that repair by SSA between repeats separated by 25 kb (as in Figure 2B) is decreased by half in clb5Δ cells. Two pathways of extensive resection exist, one being dependent on Exo1 and the other on Sgs1/Dna2. To delineate the contribution of Clb5 in these pathways, we tested resection in clb5Δ exo1Δ and clb5Δ sgs1Δ double mutants and found that both mutants exhibit more severe defect in extensive resection, demonstrating a role of Clb5 in both pathways (Supplementary Figure S6). A possible target of Cdk1-Clb5 that facilitates extensive resection could be the known Cdk1 target, Dna2. Cdk1 phosphorylates Dna2 to control its cellular localization and recruitment to DSBs (22,58). However clb5Δ cells show proper nuclear localization of Dna2-GFP (data not shown). We asked whether Fun30 is a target, but the results showed that deletion of FUN30 in clb5Δ cells exacerbates the resection defect (Supplementary Figure S6). Therefore, Fun30 functions at least partially in a Clb5-independent manner, which is consistent with Clb2 and Clb5 playing redundant role in Fun30 phosphorylation (Figure 1F). Thus there are additional yet unknown Cdk1-Clb5 targets that stimulate extensive DSB end resection. In contrast to clb5Δ, clb2Δ mutant cells have a higher rate of extensive resection (Figure 5C) and the double mutant clb2Δ clb5Δ shows an intermediate phenotype, which could be an underlying mechanism of a positive genetic interaction between Clb5 and Clb2 in response to CPT or phleomycin (Figure 5B). As described below, the higher rate of resection in clb2Δ single mutant results likely from a decreased recruitment of Rad9, a known negative regulator of resection (59).Clb2 and Clb5 are important for proper DNA damage checkpoint activationGiven the importance of Clb2 and Clb5 in the recruitment of Cdk1 to DSB ends and the essential role of Cdk1 in the DNA damage checkpoint, we assessed the role of Clb2 and Clb5 in the checkpoint signaling pathway. To evaluate DNA damage checkpoint activation we followed the phosphorylation of the signaling kinase Rad53. Wild type cells and clb5Δ cells respond properly to the HO-induced DSB as indicated by prompt Rad53 phosphorylation (Figure 6A). However, we found Rad53 phosphorylation is impaired in the clb2Δ mutant and that the defect is more severe in the clb2Δ clb5Δ double mutant. Similarly, phosphorylation of Rad9, an upstream checkpoint adaptor protein, is compromised in clb2Δ and particularly in clb2Δ clb5Δ double mutant cells (Figure 6A). We noted that a low level of Rad9 phosphorylation occurs in the clb2Δ clb5Δ mutant even before HO induction, which likely stems from spontaneous DNA damage other than DSBs. Next, we assessed by ChIP whether Rad9 can be efficiently targeted to chromatin upon DSB induction in the absence of Clb2 or Clb5. We saw efficient recruitment of Rad9–3xFLAG to DSBs in wild type cells, and the recruitment decreased significantly in clb2Δ but not clb5Δ single mutant cells, and it was further reduced in the clb2Δ clb5Δ double mutant (Figure 6B). Decreased Rad9 recruitment to DSB in clb2Δ is likely related to increased resection rates observed in this mutant as Rad9 and its human ortholog 53BP1 are known to inhibit resection (13,59,60). Consistent with the role of Clb2 and Clb5 in checkpoint response, we observed that the number of G2/M cells in the clb2Δ clb5Δ double mutant do not increase in response to DSB induction, unlike that seen in wild type cells or in the clb2Δ or clb5Δ single mutant (Figure 6C). Thus, Clb2 and Clb5 play important roles in the DSB recruitment of Cdk1, control of DSB end resection, and in establishing the DNA damage checkpoint upon DNA damage.Figure 6.Clb2 and Clb5 play redundant roles in checkpoint activation in response to DSBs. (A) Western blot analysis of Rad53 and Rad9 phosphorylation upon DSB induction in indicated strains. (B) ChIP analysis of Rad9 recruitment at 1 kb from DSB. *P < 0.05 (n = 3, t-test). (C) Plot showing the percentage of G2/M arrested cells upon DSB induction.

Cdk1 phosphorylates Fun30 in vivo and in vitroExtensive resection of 5′ strand at DSB ends is controlled by Cdk1 in yeast and human (22,24). Yeast Dna2 is phosphorylated by Cdk1 to stimulate its nuclear localization and recruitment to DSB ends. However, pseudophosphorylated form of Dna2 which restores nuclear localization of Dna2 is insufficient to bypass the need for Cdk1 for extensive resection, raising the possibility that there are additional targets of Cdk1 important for resection. Fun30, which is required for extensive resection (39,42,43), was identified as a putative Cdk1 target in a genome-wide screen (44). Fun30 and its fission yeast ortholog Fft3 are ATP-dependent chromatin remodeling factors involved also in the assembly of chromatin within centromeres and at silenced mating type loci (45–49). The functions of Fun30 appear to be conserved between yeast and human, as SMARCAD1, the human homolog of Fun30, plays a role in both DSB end resection and silencing (42,50). Yeast Fun30 contains a cluster of Cdk1 consensus sites at its N-terminus. Serine 20 (S20) is conserved among fungi and serine 28 (S28) is well conserved across yeast, mouse and human (Figure 1A), while serine 125 (S125) is not conserved. Serine 28 was identified as a putative target site of Cdk1 in a global proteomics study (44). To verify whether Fun30 is indeed phosphorylated on this evolutionarily conserved site in cells, we raised antibody against phosphorylated Fun30-S28. The antibody is highly specific since it only recognizes the Fun30 with phosphorylated S28 (Figure 1B). Further we observed that S28 phosphorylation occurs in a cell-cycle dependent fashion, in cells arrested in S phase with HU or G2/M phase with nocodazole but not in cells synchronized in G1 using alpha factor or when the conditional allele of Cdk1 kinase (Cdk1-as1) is inhibited by an ATP analogue, 1-NM-PP1 (51) (Figure 1C). To provide biochemical evidence that Cdk1 directly phosphorylates Fun30, we purified Fun30 and fun30–3A proteins and tested them with Cdk1/Clb2 in vitro. Consistent with the in vivo data, we found that Fun30 is efficiently phosphorylated by Cdk1, but little phosphorylation of fun30–3A occurs, indicating that the 3A mutation eliminates the major Cdk1-target site(s) (Figure 1D).Figure 1.Analysis of Fun30-S28 phosphorylation in cell cycle and in response to DNA damage. (A) Alignment of the N-terminal amino acid sequences for Fun30 and its orthologs from indicated species. The evolutionarily conserved serine is marked with an asterisk. (B) Western blot analysis of Fun30-S28 phosphorylation in wild type cells and FUN30 mutants. All FUN30 alleles were fused C-terminally to a sequence encoding 13xMYC. (C) Western blot analysis of Fun30-S28 phosphorylation in cells arrested in different phases of the cell cycle. (D) In vitro phosphorylation of Fun30 and fun30–3A by the Cdk1-Clb2 complex. (E) Western blot analysis of Fun30-S28 phosphorylation in wild type cells with or without drug treatments. (F) Western blot showing Rad53 phosphorylation upon MMS treatment in the wild type strain and indicated mutants. (G,H) Western blot showing Fun30-S28 phosphorylation in wild type cells and indicated mutants upon MMS treatment.

Extensive resection of 5′ strand at DSB ends is controlled by Cdk1 in yeast and human (22,24). Yeast Dna2 is phosphorylated by Cdk1 to stimulate its nuclear localization and recruitment to DSB ends. However, pseudophosphorylated form of Dna2 which restores nuclear localization of Dna2 is insufficient to bypass the need for Cdk1 for extensive resection, raising the possibility that there are additional targets of Cdk1 important for resection. Fun30, which is required for extensive resection (39,42,43), was identified as a putative Cdk1 target in a genome-wide screen (44). Fun30 and its fission yeast ortholog Fft3 are ATP-dependent chromatin remodeling factors involved also in the assembly of chromatin within centromeres and at silenced mating type loci (45–49). The functions of Fun30 appear to be conserved between yeast and human, as SMARCAD1, the human homolog of Fun30, plays a role in both DSB end resection and silencing (42,50). Yeast Fun30 contains a cluster of Cdk1 consensus sites at its N-terminus. Serine 20 (S20) is conserved among fungi and serine 28 (S28) is well conserved across yeast, mouse and human (Figure 1A), while serine 125 (S125) is not conserved. Serine 28 was identified as a putative target site of Cdk1 in a global proteomics study (44). To verify whether Fun30 is indeed phosphorylated on this evolutionarily conserved site in cells, we raised antibody against phosphorylated Fun30-S28. The antibody is highly specific since it only recognizes the Fun30 with phosphorylated S28 (Figure 1B). Further we observed that S28 phosphorylation occurs in a cell-cycle dependent fashion, in cells arrested in S phase with HU or G2/M phase with nocodazole but not in cells synchronized in G1 using alpha factor or when the conditional allele of Cdk1 kinase (Cdk1-as1) is inhibited by an ATP analogue, 1-NM-PP1 (51) (Figure 1C). To provide biochemical evidence that Cdk1 directly phosphorylates Fun30, we purified Fun30 and fun30–3A proteins and tested them with Cdk1/Clb2 in vitro. Consistent with the in vivo data, we found that Fun30 is efficiently phosphorylated by Cdk1, but little phosphorylation of fun30–3A occurs, indicating that the 3A mutation eliminates the major Cdk1-target site(s) (Figure 1D).

Analysis of Fun30-S28 phosphorylation in cell cycle and in response to DNA damage. (A) Alignment of the N-terminal amino acid sequences for Fun30 and its orthologs from indicated species. The evolutionarily conserved serine is marked with an asterisk. (B) Western blot analysis of Fun30-S28 phosphorylation in wild type cells and FUN30 mutants. All FUN30 alleles were fused C-terminally to a sequence encoding 13xMYC. (C) Western blot analysis of Fun30-S28 phosphorylation in cells arrested in different phases of the cell cycle. (D) In vitro phosphorylation of Fun30 and fun30–3A by the Cdk1-Clb2 complex. (E) Western blot analysis of Fun30-S28 phosphorylation in wild type cells with or without drug treatments. (F) Western blot showing Rad53 phosphorylation upon MMS treatment in the wild type strain and indicated mutants. (G,H) Western blot showing Fun30-S28 phosphorylation in wild type cells and indicated mutants upon MMS treatment.

Fun30 phosphorylation is needed for its functions in response to DNA damageWe wanted to determine the physiological importance of Fun30 phosphorylation by Cdk1. First, we assessed the role of Fun30 phosphorylation in extensive DSB end resection using the donorless strain wherein the HO-induced DSB cannot be repaired because both sister chromatids are cleaved by HO and the donor sequences HML and HMR used during mating type switching have been deleted. Resection was followed using Southern blots and probes specific for loci at varying distances from the DSB. We measured the rates of extensive resection in wild type cells and in the fun30-S28A, fun30-S20A, fun30-S125A, fun30-S20,28A (fun30–2A), fun30-S20,28,125A (fun30–3A) and fun30Δ mutants. Resection in fun30-S20A, fun30-S28A, fun30–2A and fun30–3A was significantly delayed as compared to wild type cells, nearly as much as in fun30Δ cells. (Figure 2A, Southern blots presented in Supplementary Figure S1). Thus phosphorylation of Fun30 by Cdk1 on S20 and S28 is needed for efficient DSB end resection. To confirm this result, we used Southern blot analysis to assess the impact of the fun30 mutations on DSB repair by single strand annealing (SSA) which requires 25 kb resection between partial leu2 repeats (Figure 2B). To prevent alternative repair pathways the assay was done in the absence of the strand exchange protein, Rad51, which is dispensable for SSA. The results showed that fun30-S28A rad51Δ, fun30-S20A rad51Δ and fun30–2A rad51Δ mutants are unable to complete SSA efficiently (Figure 2C) and consistently viability of these mutants is ≈15 times lower than that of the control rad51Δ cells (Figure 2D). FUN30 mutants that lack key phosphorylation sites (e.g. fun30-S20A, fun30-S28A or fun30-S20A S28A) are sensitive to CPT and MMS, indicating that Fun30 phosphorylation is also needed for the response to DNA damage (Supplementary Figure S2). Extensive resection is also needed for adaptation to DNA damage checkpoint upon the induction of a single non-repairable DSB, such that mutants defective in resection, sgs1Δ, exo1Δ, mec1-ad or fun30Δ mutants are at least partially deficient in adaptation (43,52). Again, we observed that cells harboring non-phosphorylable mutants of FUN30 are partially deficient in the adaptation to the DNA damage checkpoint (Supplementary Figure S2). Together, these results demonstrate the importance of Cdk1-dependent Fun30 phosphorylation in response to DNA damage.Figure 2.Fun30-S28 phosphorylation stimulates resection and DNA damage response. (A) Quantification of extensive resection rates at 28 kb from DSBs in wild type and indicated mutants. Plotted are the average values from three independent experiments. Error bars denote standard deviation. (B) Scheme showing the SSA assay between two partial LEU2 repeats. (C) Southern blot showing the kinetics of DSB repair by SSA in indicated mutants. (D) Viability of indicated mutant cells in response to a DSB repaired by SSA. (E) Analysis of the recruitment of wild type and indicated mutant Fun30 proteins tagged with 13xMyc at 1, 5 or 10 kb from DSB ends. Error bars denote standard deviation (n = 3).

We wanted to determine the physiological importance of Fun30 phosphorylation by Cdk1. First, we assessed the role of Fun30 phosphorylation in extensive DSB end resection using the donorless strain wherein the HO-induced DSB cannot be repaired because both sister chromatids are cleaved by HO and the donor sequences HML and HMR used during mating type switching have been deleted. Resection was followed using Southern blots and probes specific for loci at varying distances from the DSB. We measured the rates of extensive resection in wild type cells and in the fun30-S28A, fun30-S20A, fun30-S125A, fun30-S20,28A (fun30–2A), fun30-S20,28,125A (fun30–3A) and fun30Δ mutants. Resection in fun30-S20A, fun30-S28A, fun30–2A and fun30–3A was significantly delayed as compared to wild type cells, nearly as much as in fun30Δ cells. (Figure 2A, Southern blots presented in Supplementary Figure S1). Thus phosphorylation of Fun30 by Cdk1 on S20 and S28 is needed for efficient DSB end resection. To confirm this result, we used Southern blot analysis to assess the impact of the fun30 mutations on DSB repair by single strand annealing (SSA) which requires 25 kb resection between partial leu2 repeats (Figure 2B). To prevent alternative repair pathways the assay was done in the absence of the strand exchange protein, Rad51, which is dispensable for SSA. The results showed that fun30-S28A rad51Δ, fun30-S20A rad51Δ and fun30–2A rad51Δ mutants are unable to complete SSA efficiently (Figure 2C) and consistently viability of these mutants is ≈15 times lower than that of the control rad51Δ cells (Figure 2D). FUN30 mutants that lack key phosphorylation sites (e.g. fun30-S20A, fun30-S28A or fun30-S20A S28A) are sensitive to CPT and MMS, indicating that Fun30 phosphorylation is also needed for the response to DNA damage (Supplementary Figure S2). Extensive resection is also needed for adaptation to DNA damage checkpoint upon the induction of a single non-repairable DSB, such that mutants defective in resection, sgs1Δ, exo1Δ, mec1-ad or fun30Δ mutants are at least partially deficient in adaptation (43,52). Again, we observed that cells harboring non-phosphorylable mutants of FUN30 are partially deficient in the adaptation to the DNA damage checkpoint (Supplementary Figure S2). Together, these results demonstrate the importance of Cdk1-dependent Fun30 phosphorylation in response to DNA damage.

Fun30-S28 phosphorylation stimulates resection and DNA damage response. (A) Quantification of extensive resection rates at 28 kb from DSBs in wild type and indicated mutants. Plotted are the average values from three independent experiments. Error bars denote standard deviation. (B) Scheme showing the SSA assay between two partial LEU2 repeats. (C) Southern blot showing the kinetics of DSB repair by SSA in indicated mutants. (D) Viability of indicated mutant cells in response to a DSB repaired by SSA. (E) Analysis of the recruitment of wild type and indicated mutant Fun30 proteins tagged with 13xMyc at 1, 5 or 10 kb from DSB ends. Error bars denote standard deviation (n = 3).

Phosphorylation is needed for spreading of Fun30 at DSBPhosphorylation by Cdk1 alters various features of a substrate protein, including its association with partners, subcellular localization, stability and others. We found that the subcellular localization of Fun30 does not change in the cell cycle and it is indistinguishable between the wild type cells and the phosphomutants (Supplementary Figure S3). Using co-immunoprecipitation we found that the known Fun30 interaction with RPA (39), the ssDNA binding protein, is not regulated by Cdk1 (Supplementary Figure S3). Next, we used ChIP to assess whether Fun30 phosphorylation regulates its DSB recruitment. We found that while Fun30, fun30-S28A and fun30–3A are efficiently recruited next to DSB ends, (Figure 2E), the enrichment of phosphomutant proteins is largely impaired or eliminated at 5 kb or 10 kb from the DSB. Based on these results, we conclude that Fun30 phosphorylation by Cdk1 is not needed for initial recruitment to DSBs but rather for its activity and spreading along DNA during resection.

Phosphorylation by Cdk1 alters various features of a substrate protein, including its association with partners, subcellular localization, stability and others. We found that the subcellular localization of Fun30 does not change in the cell cycle and it is indistinguishable between the wild type cells and the phosphomutants (Supplementary Figure S3). Using co-immunoprecipitation we found that the known Fun30 interaction with RPA (39), the ssDNA binding protein, is not regulated by Cdk1 (Supplementary Figure S3). Next, we used ChIP to assess whether Fun30 phosphorylation regulates its DSB recruitment. We found that while Fun30, fun30-S28A and fun30–3A are efficiently recruited next to DSB ends, (Figure 2E), the enrichment of phosphomutant proteins is largely impaired or eliminated at 5 kb or 10 kb from the DSB. Based on these results, we conclude that Fun30 phosphorylation by Cdk1 is not needed for initial recruitment to DSBs but rather for its activity and spreading along DNA during resection.

Fun30 phosphorylation increases upon DNA damageWe examined Fun30-S28 phosphorylation upon exposure to MMS, and unexpectedly we observed enhanced phosphorylation upon DNA damage (Figure 1E). Similarly, we note that Fun30-S28 phosphorylation was increased in response to the DSB inducing agent phleomycin, but it remained largely unaffected upon treatment with the replication stalling compound hydroxyurea (HU) (Figure 1E). We considered two possibilities to explain these observations. One being that MMS treatment causes cells to arrest in S or G2/M where Cdk1 activity is higher. An alternative explanation is that Fun30 phosphorylation by Cdk1 is enhanced in situ at DNA damage sites. The second model is based on several documented examples where cyclins are recruited to specific subcellular locations where they help target the substrates of Cdks (53,54). Moreover, in human and mouse cells, CDK2 is recruited to DNA damage site via interaction with MRE11 (55). If the first model is correct, we expected to see a correlation between Fun30-S28 phosphorylation and cell cycle arrest upon DNA damage. This possibility is unlikely because Fun30-S28 phosphorylation increases upon MMS treatment even in cells pre-arrested in G2/M with nocodazole (Figure 1E). If the second model holds true, we should find decreased Fun30-S28 phosphorylation in mutants that fail to recruit Fun30 to DNA damage sites. To test this model, we examined Fun30-S28 phosphorylation in mec1Δtel1Δsml1Δ, sgs1Δexo1Δ and h2a-S129A mutants. We previously showed that Fun30 recruitment to DSBs is severely decreased in mec1Δtel1Δsml1Δ and sgs1Δexo1Δ mutants (39) and here we have found that efficient recruitment of Fun30 also requires H2A phosphorylation at serine 129 (Supplementary Figure S4). Previous studies showed that purified Fun30 binds preferentially to the non-phosphorylated form of the H2A in vitro (43). Thus, it is possible that in cells phosphorylated H2A plays an indirect role in Fun30 recruitment. The three aforementioned mutants defective in Fun30 recruitment exhibited distinguishable capacities in DNA damage checkpoint activation as reflected by the phosphorylation of the signaling kinase Rad53 in response to MMS (Figure 1F). In wild type and h2A-S129A cells checkpoint activation is comparable while in mec1Δ tel1Δ sml1Δ and sgs1Δ exo1Δ checkpoint activation is completely or partly deficient. As shown in Figure 1G, decreased Fun30-S28 phosphorylation was observed in all three mutants regardless of their capacity in checkpoint activation, indicating that recruitment of Fun30 to DNA damage sites stimulates its phosphorylation. Together the results favor the second model where Cdk1-dependent phosphorylation of Fun30 increases at DNA damage sites.

We examined Fun30-S28 phosphorylation upon exposure to MMS, and unexpectedly we observed enhanced phosphorylation upon DNA damage (Figure 1E). Similarly, we note that Fun30-S28 phosphorylation was increased in response to the DSB inducing agent phleomycin, but it remained largely unaffected upon treatment with the replication stalling compound hydroxyurea (HU) (Figure 1E). We considered two possibilities to explain these observations. One being that MMS treatment causes cells to arrest in S or G2/M where Cdk1 activity is higher. An alternative explanation is that Fun30 phosphorylation by Cdk1 is enhanced in situ at DNA damage sites. The second model is based on several documented examples where cyclins are recruited to specific subcellular locations where they help target the substrates of Cdks (53,54). Moreover, in human and mouse cells, CDK2 is recruited to DNA damage site via interaction with MRE11 (55). If the first model is correct, we expected to see a correlation between Fun30-S28 phosphorylation and cell cycle arrest upon DNA damage. This possibility is unlikely because Fun30-S28 phosphorylation increases upon MMS treatment even in cells pre-arrested in G2/M with nocodazole (Figure 1E). If the second model holds true, we should find decreased Fun30-S28 phosphorylation in mutants that fail to recruit Fun30 to DNA damage sites. To test this model, we examined Fun30-S28 phosphorylation in mec1Δtel1Δsml1Δ, sgs1Δexo1Δ and h2a-S129A mutants. We previously showed that Fun30 recruitment to DSBs is severely decreased in mec1Δtel1Δsml1Δ and sgs1Δexo1Δ mutants (39) and here we have found that efficient recruitment of Fun30 also requires H2A phosphorylation at serine 129 (Supplementary Figure S4). Previous studies showed that purified Fun30 binds preferentially to the non-phosphorylated form of the H2A in vitro (43). Thus, it is possible that in cells phosphorylated H2A plays an indirect role in Fun30 recruitment. The three aforementioned mutants defective in Fun30 recruitment exhibited distinguishable capacities in DNA damage checkpoint activation as reflected by the phosphorylation of the signaling kinase Rad53 in response to MMS (Figure 1F). In wild type and h2A-S129A cells checkpoint activation is comparable while in mec1Δ tel1Δ sml1Δ and sgs1Δ exo1Δ checkpoint activation is completely or partly deficient. As shown in Figure 1G, decreased Fun30-S28 phosphorylation was observed in all three mutants regardless of their capacity in checkpoint activation, indicating that recruitment of Fun30 to DNA damage sites stimulates its phosphorylation. Together the results favor the second model where Cdk1-dependent phosphorylation of Fun30 increases at DNA damage sites.

Multiple Cdk1-cyclin complexes are recruited to DSB endsIf Cdk1 activity is increased at DSBs, we would expect to observe the enrichment of Cdk1 and cyclins. Therefore, we followed the recruitment of Cdk1 and cyclins at the DSB at the MAT locus by chromatin immunoprecipitation (ChIP) coupled with quantitative PCR. Cdk1, Cln2 and Clb1–6 were fused to a three tandem FLAG or HA tag at their C-termini in a strain carrying HO-inducible and unrepairable DSB at MAT. We noted that strain carrying HA-tagged Clb1 was slow growing in YEP-raffinose, therefore it was not included in the analysis. We observed a robust recruitment of Cdk1 and enrichment of multiple cyclins at MAT after DSB induction (Figure 3A,B). Highest enrichment was observed for Cln2, Clb2 and Clb5, while lesser recruitment of Clb4, Clb6 and Clb3 also occurred. We found that the G1/S cyclin Cln2 was present at the DSB 4 h after induction even when most cells are arrested in G2/M. Western blot analysis revealed that there is a low level of Cln2 left by that time (Supplementary Figure S5). Besides cyclins, Cks1 also associates with Cdk1 and plays an important role in targeting substrates that are phosphorylated multiply (5,6). Therefore, we examined the enrichment of Cks1–3xHA protein at the HO-induced DSB site. However, no significant enrichment of Cks1 was observed, suggesting that it does not play a direct role in the DNA damage response (Figure 3C). Together, we conclude that Cdk1 and multiple cyclins are enriched locally at DSBs.Figure 3.Cdk1 and multiple cyclins are recruited to DSBs. (A–C) Analysis of the recruitment of Cdc28–3xFLAG, cyclins and Cks1 to DSB ends at MAT locus by ChIP. ARO1 locus was used as a control. Error bars represent standard deviation (n = 3).

If Cdk1 activity is increased at DSBs, we would expect to observe the enrichment of Cdk1 and cyclins. Therefore, we followed the recruitment of Cdk1 and cyclins at the DSB at the MAT locus by chromatin immunoprecipitation (ChIP) coupled with quantitative PCR. Cdk1, Cln2 and Clb1–6 were fused to a three tandem FLAG or HA tag at their C-termini in a strain carrying HO-inducible and unrepairable DSB at MAT. We noted that strain carrying HA-tagged Clb1 was slow growing in YEP-raffinose, therefore it was not included in the analysis. We observed a robust recruitment of Cdk1 and enrichment of multiple cyclins at MAT after DSB induction (Figure 3A,B). Highest enrichment was observed for Cln2, Clb2 and Clb5, while lesser recruitment of Clb4, Clb6 and Clb3 also occurred. We found that the G1/S cyclin Cln2 was present at the DSB 4 h after induction even when most cells are arrested in G2/M. Western blot analysis revealed that there is a low level of Cln2 left by that time (Supplementary Figure S5). Besides cyclins, Cks1 also associates with Cdk1 and plays an important role in targeting substrates that are phosphorylated multiply (5,6). Therefore, we examined the enrichment of Cks1–3xHA protein at the HO-induced DSB site. However, no significant enrichment of Cks1 was observed, suggesting that it does not play a direct role in the DNA damage response (Figure 3C). Together, we conclude that Cdk1 and multiple cyclins are enriched locally at DSBs.

Cdk1 and multiple cyclins are recruited to DSBs. (A–C) Analysis of the recruitment of Cdc28–3xFLAG, cyclins and Cks1 to DSB ends at MAT locus by ChIP. ARO1 locus was used as a control. Error bars represent standard deviation (n = 3).

Resection, checkpoint response and the cyclins Clb2 and Clb5 are needed for efficient Cdk1 recruitment and Fun30 phosphorylationTo understand how the DSB enrichment of the Cdk1 and cyclins is achieved, we examined their recruitment in mutants deficient in checkpoint signaling or with an altered rate of DSB end resection. We noted that the recruitment of Cdk1 is dramatically reduced in the checkpoint deficient mutant mec1Δ tel1Δ sml1Δ, and it is reduced even more in mre11Δ or sgs1Δ exo1Δ, mutants deficient in either initial or extensive resection (Figure 4A). However, it remains unaffected in the mutants with increased rate of resection, such as yku70Δ or dot1Δ. Given that Cdk activity plays key roles in DNA damage response, we tested whether Cdk1 kinase activity is needed for its own recruitment to DSBs. (51) Inactivation of Cdk1 before DSB induction prevents its own recruitment, likely due to the role of Cdk1 in resection (Figure 4A). Together, these results demonstrate that recruitment of Cdk1 requires formation of ssDNA, damage checkpoint activation, and the kinase activity of Cdk1. It is possible that formation of the 3′ ssDNA at DSBs is indirectly needed for Cdk1 and cyclins recruitment by its role in checkpoint response and/or DNA repair.Figure 4.Genetic requirements for recruitment of Cdk1 and cyclins to DSBs. (A–C) Analysis of recruitment of Cdc28 and cyclins to DSBs by ChIP in indicated mutants. Error bars denote standard deviation (n = 3). *P < 0.05 (n = 3, t-test).Some of the cyclins in mammals play Cdk-independent role in the DNA damage response. For example, cyclin D1 mediates the recruitment of the DNA strand exchange protein RAD51 (56). Thus, we decided to test whether the genetic requirement for the recruitment of Cln2, Clb2 and Clb5 cyclins and Cdk1 is the same or different. Similar to Cdk1, the recruitment for all three cyclins was impaired in mre11Δ, sgs1Δ exo1Δ and mec1Δ tel1Δ sml1Δ cells (Figure 4B). Therefore, Cdk1 and cyclins appear to share the same genetic requirement for their DSB recruitment, suggesting that these proteins are delivered to DSBs as complexes. Further, we assessed whether the robust DSB recruitment of cyclins is important for Cdk1 recruitment. We found that Cdk1 recruitment decreases significantly in the clb2Δ and clb2Δ clb5Δ mutants but not in cln2Δ cells (Figure 4C). Accordingly, we observed that Fun30-S28 phosphorylation is also severely impaired in clb2Δ and clb2Δ clb5Δ cells (Figure 1G). Thus, Clb2 and Clb5 play central roles in Cdk1 recruitment as well as in stimulating Fun30 phosphorylation by Cdk1.

To understand how the DSB enrichment of the Cdk1 and cyclins is achieved, we examined their recruitment in mutants deficient in checkpoint signaling or with an altered rate of DSB end resection. We noted that the recruitment of Cdk1 is dramatically reduced in the checkpoint deficient mutant mec1Δ tel1Δ sml1Δ, and it is reduced even more in mre11Δ or sgs1Δ exo1Δ, mutants deficient in either initial or extensive resection (Figure 4A). However, it remains unaffected in the mutants with increased rate of resection, such as yku70Δ or dot1Δ. Given that Cdk activity plays key roles in DNA damage response, we tested whether Cdk1 kinase activity is needed for its own recruitment to DSBs. (51) Inactivation of Cdk1 before DSB induction prevents its own recruitment, likely due to the role of Cdk1 in resection (Figure 4A). Together, these results demonstrate that recruitment of Cdk1 requires formation of ssDNA, damage checkpoint activation, and the kinase activity of Cdk1. It is possible that formation of the 3′ ssDNA at DSBs is indirectly needed for Cdk1 and cyclins recruitment by its role in checkpoint response and/or DNA repair.

Genetic requirements for recruitment of Cdk1 and cyclins to DSBs. (A–C) Analysis of recruitment of Cdc28 and cyclins to DSBs by ChIP in indicated mutants. Error bars denote standard deviation (n = 3). *P < 0.05 (n = 3, t-test).

Some of the cyclins in mammals play Cdk-independent role in the DNA damage response. For example, cyclin D1 mediates the recruitment of the DNA strand exchange protein RAD51 (56). Thus, we decided to test whether the genetic requirement for the recruitment of Cln2, Clb2 and Clb5 cyclins and Cdk1 is the same or different. Similar to Cdk1, the recruitment for all three cyclins was impaired in mre11Δ, sgs1Δ exo1Δ and mec1Δ tel1Δ sml1Δ cells (Figure 4B). Therefore, Cdk1 and cyclins appear to share the same genetic requirement for their DSB recruitment, suggesting that these proteins are delivered to DSBs as complexes. Further, we assessed whether the robust DSB recruitment of cyclins is important for Cdk1 recruitment. We found that Cdk1 recruitment decreases significantly in the clb2Δ and clb2Δ clb5Δ mutants but not in cln2Δ cells (Figure 4C). Accordingly, we observed that Fun30-S28 phosphorylation is also severely impaired in clb2Δ and clb2Δ clb5Δ cells (Figure 1G). Thus, Clb2 and Clb5 play central roles in Cdk1 recruitment as well as in stimulating Fun30 phosphorylation by Cdk1.

Clb2 and Clb5 are needed for resistance of cells to DNA damaging agentsAs cyclins direct Cdk1 toward specific targets, the above results suggest that Cdk1 acts with Cln2, Clb2 and Clb5 to target substrates involved in the DNA damage response. Consistent with a broad role of Cdk1-cyclin complexes, deletion of CLN2, CLB2 or CLB5, the most enriched cyclins at DSBs (Figure 3B), dramatically elevates the sensitivity to DNA damaging agents including MMS, CPT and phleomycin (Figure 5A and ref. 57). Thus, each of the three cyclins plays a unique and indispensable function in the DNA damage response. These results could reflect direct phosphorylation of DNA damage response proteins by the Cdk1-cyclin complexes, or they could stem indirectly from alterations in transcriptional programs or other processes (2). To test whether Cln2, Clb2 and Clb5 share overlapping roles in response to DNA damage, we created cln2Δ clb2Δ, cln2Δ clb5Δ and clb2Δ clb5Δ double mutants and analyzed their response to genotoxins. A mild or lack of genetic interaction in response to DNA damaging agents was observed between the G1 cyclin Cln2 and the S-phase cyclin Clb5 or the M-phase cyclin Clb2. However, we observed both strong positive (CPT, phleomycin) and negative (MMS treatment) genetic interactions between CLB2 and CLB5 (Figure 5B). Once the specific targets of these cyclins are recognized, it should be possible to explain these interactions at the molecular level.Figure 5.The role of cyclins in DNA damage response and DSB end resection. (A,B) Analysis of DNA damage sensitivity for the indicated strains. (C,D) Kinetics of extensive DSB ends resection in wild type or indicated cyclin mutant cells. Error bars denote standard deviation (n = 3).

As cyclins direct Cdk1 toward specific targets, the above results suggest that Cdk1 acts with Cln2, Clb2 and Clb5 to target substrates involved in the DNA damage response. Consistent with a broad role of Cdk1-cyclin complexes, deletion of CLN2, CLB2 or CLB5, the most enriched cyclins at DSBs (Figure 3B), dramatically elevates the sensitivity to DNA damaging agents including MMS, CPT and phleomycin (Figure 5A and ref. 57). Thus, each of the three cyclins plays a unique and indispensable function in the DNA damage response. These results could reflect direct phosphorylation of DNA damage response proteins by the Cdk1-cyclin complexes, or they could stem indirectly from alterations in transcriptional programs or other processes (2). To test whether Cln2, Clb2 and Clb5 share overlapping roles in response to DNA damage, we created cln2Δ clb2Δ, cln2Δ clb5Δ and clb2Δ clb5Δ double mutants and analyzed their response to genotoxins. A mild or lack of genetic interaction in response to DNA damaging agents was observed between the G1 cyclin Cln2 and the S-phase cyclin Clb5 or the M-phase cyclin Clb2. However, we observed both strong positive (CPT, phleomycin) and negative (MMS treatment) genetic interactions between CLB2 and CLB5 (Figure 5B). Once the specific targets of these cyclins are recognized, it should be possible to explain these interactions at the molecular level.

The role of cyclins in DNA damage response and DSB end resection. (A,B) Analysis of DNA damage sensitivity for the indicated strains. (C,D) Kinetics of extensive DSB ends resection in wild type or indicated cyclin mutant cells. Error bars denote standard deviation (n = 3).

Clb2 and Clb5 cyclins control extensive DSB end resectionClb2, Clb5 and Cln2 appear to be the most important cyclins in the cellular resistance to DNA damaging agents, and they are also most strongly recruited to DSBs. Given the importance of Cdk1 in resection and the DNA damage checkpoint response, we focused on these three cyclins and wished to understand their potential contributions to these processes. We found that clb5Δ cells show a decreased rate of extensive resection (Figure 5C and Supplementary Figure S1). Disruption of the other S phase cyclin Clb6 in clb5Δ cells didn't exacerbate the defect in resection, indicating that Clb5 is the only S-phase cyclin needed for optimal resection (Figure 5D and Supplementary Figure S1). Consistent with the role of Clb5 in promoting resection, we observed that repair by SSA between repeats separated by 25 kb (as in Figure 2B) is decreased by half in clb5Δ cells. Two pathways of extensive resection exist, one being dependent on Exo1 and the other on Sgs1/Dna2. To delineate the contribution of Clb5 in these pathways, we tested resection in clb5Δ exo1Δ and clb5Δ sgs1Δ double mutants and found that both mutants exhibit more severe defect in extensive resection, demonstrating a role of Clb5 in both pathways (Supplementary Figure S6). A possible target of Cdk1-Clb5 that facilitates extensive resection could be the known Cdk1 target, Dna2. Cdk1 phosphorylates Dna2 to control its cellular localization and recruitment to DSBs (22,58). However clb5Δ cells show proper nuclear localization of Dna2-GFP (data not shown). We asked whether Fun30 is a target, but the results showed that deletion of FUN30 in clb5Δ cells exacerbates the resection defect (Supplementary Figure S6). Therefore, Fun30 functions at least partially in a Clb5-independent manner, which is consistent with Clb2 and Clb5 playing redundant role in Fun30 phosphorylation (Figure 1F). Thus there are additional yet unknown Cdk1-Clb5 targets that stimulate extensive DSB end resection. In contrast to clb5Δ, clb2Δ mutant cells have a higher rate of extensive resection (Figure 5C) and the double mutant clb2Δ clb5Δ shows an intermediate phenotype, which could be an underlying mechanism of a positive genetic interaction between Clb5 and Clb2 in response to CPT or phleomycin (Figure 5B). As described below, the higher rate of resection in clb2Δ single mutant results likely from a decreased recruitment of Rad9, a known negative regulator of resection (59).

Clb2, Clb5 and Cln2 appear to be the most important cyclins in the cellular resistance to DNA damaging agents, and they are also most strongly recruited to DSBs. Given the importance of Cdk1 in resection and the DNA damage checkpoint response, we focused on these three cyclins and wished to understand their potential contributions to these processes. We found that clb5Δ cells show a decreased rate of extensive resection (Figure 5C and Supplementary Figure S1). Disruption of the other S phase cyclin Clb6 in clb5Δ cells didn't exacerbate the defect in resection, indicating that Clb5 is the only S-phase cyclin needed for optimal resection (Figure 5D and Supplementary Figure S1). Consistent with the role of Clb5 in promoting resection, we observed that repair by SSA between repeats separated by 25 kb (as in Figure 2B) is decreased by half in clb5Δ cells. Two pathways of extensive resection exist, one being dependent on Exo1 and the other on Sgs1/Dna2. To delineate the contribution of Clb5 in these pathways, we tested resection in clb5Δ exo1Δ and clb5Δ sgs1Δ double mutants and found that both mutants exhibit more severe defect in extensive resection, demonstrating a role of Clb5 in both pathways (Supplementary Figure S6). A possible target of Cdk1-Clb5 that facilitates extensive resection could be the known Cdk1 target, Dna2. Cdk1 phosphorylates Dna2 to control its cellular localization and recruitment to DSBs (22,58). However clb5Δ cells show proper nuclear localization of Dna2-GFP (data not shown). We asked whether Fun30 is a target, but the results showed that deletion of FUN30 in clb5Δ cells exacerbates the resection defect (Supplementary Figure S6). Therefore, Fun30 functions at least partially in a Clb5-independent manner, which is consistent with Clb2 and Clb5 playing redundant role in Fun30 phosphorylation (Figure 1F). Thus there are additional yet unknown Cdk1-Clb5 targets that stimulate extensive DSB end resection. In contrast to clb5Δ, clb2Δ mutant cells have a higher rate of extensive resection (Figure 5C) and the double mutant clb2Δ clb5Δ shows an intermediate phenotype, which could be an underlying mechanism of a positive genetic interaction between Clb5 and Clb2 in response to CPT or phleomycin (Figure 5B). As described below, the higher rate of resection in clb2Δ single mutant results likely from a decreased recruitment of Rad9, a known negative regulator of resection (59).

Clb2 and Clb5 are important for proper DNA damage checkpoint activationGiven the importance of Clb2 and Clb5 in the recruitment of Cdk1 to DSB ends and the essential role of Cdk1 in the DNA damage checkpoint, we assessed the role of Clb2 and Clb5 in the checkpoint signaling pathway. To evaluate DNA damage checkpoint activation we followed the phosphorylation of the signaling kinase Rad53. Wild type cells and clb5Δ cells respond properly to the HO-induced DSB as indicated by prompt Rad53 phosphorylation (Figure 6A). However, we found Rad53 phosphorylation is impaired in the clb2Δ mutant and that the defect is more severe in the clb2Δ clb5Δ double mutant. Similarly, phosphorylation of Rad9, an upstream checkpoint adaptor protein, is compromised in clb2Δ and particularly in clb2Δ clb5Δ double mutant cells (Figure 6A). We noted that a low level of Rad9 phosphorylation occurs in the clb2Δ clb5Δ mutant even before HO induction, which likely stems from spontaneous DNA damage other than DSBs. Next, we assessed by ChIP whether Rad9 can be efficiently targeted to chromatin upon DSB induction in the absence of Clb2 or Clb5. We saw efficient recruitment of Rad9–3xFLAG to DSBs in wild type cells, and the recruitment decreased significantly in clb2Δ but not clb5Δ single mutant cells, and it was further reduced in the clb2Δ clb5Δ double mutant (Figure 6B). Decreased Rad9 recruitment to DSB in clb2Δ is likely related to increased resection rates observed in this mutant as Rad9 and its human ortholog 53BP1 are known to inhibit resection (13,59,60). Consistent with the role of Clb2 and Clb5 in checkpoint response, we observed that the number of G2/M cells in the clb2Δ clb5Δ double mutant do not increase in response to DSB induction, unlike that seen in wild type cells or in the clb2Δ or clb5Δ single mutant (Figure 6C). Thus, Clb2 and Clb5 play important roles in the DSB recruitment of Cdk1, control of DSB end resection, and in establishing the DNA damage checkpoint upon DNA damage.Figure 6.Clb2 and Clb5 play redundant roles in checkpoint activation in response to DSBs. (A) Western blot analysis of Rad53 and Rad9 phosphorylation upon DSB induction in indicated strains. (B) ChIP analysis of Rad9 recruitment at 1 kb from DSB. *P < 0.05 (n = 3, t-test). (C) Plot showing the percentage of G2/M arrested cells upon DSB induction.

Given the importance of Clb2 and Clb5 in the recruitment of Cdk1 to DSB ends and the essential role of Cdk1 in the DNA damage checkpoint, we assessed the role of Clb2 and Clb5 in the checkpoint signaling pathway. To evaluate DNA damage checkpoint activation we followed the phosphorylation of the signaling kinase Rad53. Wild type cells and clb5Δ cells respond properly to the HO-induced DSB as indicated by prompt Rad53 phosphorylation (Figure 6A). However, we found Rad53 phosphorylation is impaired in the clb2Δ mutant and that the defect is more severe in the clb2Δ clb5Δ double mutant. Similarly, phosphorylation of Rad9, an upstream checkpoint adaptor protein, is compromised in clb2Δ and particularly in clb2Δ clb5Δ double mutant cells (Figure 6A). We noted that a low level of Rad9 phosphorylation occurs in the clb2Δ clb5Δ mutant even before HO induction, which likely stems from spontaneous DNA damage other than DSBs. Next, we assessed by ChIP whether Rad9 can be efficiently targeted to chromatin upon DSB induction in the absence of Clb2 or Clb5. We saw efficient recruitment of Rad9–3xFLAG to DSBs in wild type cells, and the recruitment decreased significantly in clb2Δ but not clb5Δ single mutant cells, and it was further reduced in the clb2Δ clb5Δ double mutant (Figure 6B). Decreased Rad9 recruitment to DSB in clb2Δ is likely related to increased resection rates observed in this mutant as Rad9 and its human ortholog 53BP1 are known to inhibit resection (13,59,60). Consistent with the role of Clb2 and Clb5 in checkpoint response, we observed that the number of G2/M cells in the clb2Δ clb5Δ double mutant do not increase in response to DSB induction, unlike that seen in wild type cells or in the clb2Δ or clb5Δ single mutant (Figure 6C). Thus, Clb2 and Clb5 play important roles in the DSB recruitment of Cdk1, control of DSB end resection, and in establishing the DNA damage checkpoint upon DNA damage.

Clb2 and Clb5 play redundant roles in checkpoint activation in response to DSBs. (A) Western blot analysis of Rad53 and Rad9 phosphorylation upon DSB induction in indicated strains. (B) ChIP analysis of Rad9 recruitment at 1 kb from DSB. *P < 0.05 (n = 3, t-test). (C) Plot showing the percentage of G2/M arrested cells upon DSB induction.

DISCUSSIONIn this study, we have shown that Cdk1 and several companion cyclins become highly enriched at DSBs. Formation of ssDNA at DSBs and damage checkpoint activation are needed for efficient recruitment of the Cdk1-cyclin complexes. In particular, Clb2, Clb5 and Cln2 are robustly recruited to DSBs and are the most crucial cyclins for resistance to DNA damaging agents. We found that Clb2 and Clb5 play several roles in DSB repair. While Clb2 and Clb5 upregulate the checkpoint response in a partially redundant manner, they are involved in either negative or positive regulation of resection of DSB ends, respectively. We have shown that the chromatin remodeler Fun30 is a new Cdk1 target in the DNA damage response. In particular, Fun30-S28 phosphorylation plays an important role in DSB end resection. Previously, multiple resection enzymes were shown to be regulated by Cdk1 to promote resection in the S and G2 phases when sister chromatid is present as the ideal template for homologous recombination. Our new finding demonstrates that Cdk1 controls this process also at the level of chromatin remodeling. To our knowledge, this is the first example that a chromatin remodeler is directly controlled in cell cycle.Fun30 phosphorylation on S28 increases upon DNA damage, and optimal Fun30 phosphorylation requires the recruitment of Cdk1 and Fun30 to DSBs. These observations lead us to propose that the recruitment of Cdk1-cyclin complexes to DSBs enhances the phosphorylation of substrates involved in the DNA damage response. Such a mechanism would facilitate the phosphorylation of DNA damage response proteins that are suboptimal Cdk1 targets and regulate the activity of DNA repair enzymes locally instead of globally. Cdk1 is essential for DSB repair by HR and damage checkpoint signaling in all eukaryotes. Paradoxically, the activity of Cdk1 is downregulated upon DNA damage in most of the eukaryotes including fission yeast and humans. Our observation on increased Cdk1 activity at the sites of DNA damage provides a possible explanation for these incompatible observations. Specifically, the DSB recruitment of Cdk1 would provide a high local level of kinase activity necessary for the timely resection, repair and checkpoint activation. Given that the activity of Cdk1-Clb2 and Cdk1-Clb5 complexes toward some targets increases in response to DNA damage, it will be of interest to identify the full complement of their targets in cells challenged with DNA damage.

In this study, we have shown that Cdk1 and several companion cyclins become highly enriched at DSBs. Formation of ssDNA at DSBs and damage checkpoint activation are needed for efficient recruitment of the Cdk1-cyclin complexes. In particular, Clb2, Clb5 and Cln2 are robustly recruited to DSBs and are the most crucial cyclins for resistance to DNA damaging agents. We found that Clb2 and Clb5 play several roles in DSB repair. While Clb2 and Clb5 upregulate the checkpoint response in a partially redundant manner, they are involved in either negative or positive regulation of resection of DSB ends, respectively. We have shown that the chromatin remodeler Fun30 is a new Cdk1 target in the DNA damage response. In particular, Fun30-S28 phosphorylation plays an important role in DSB end resection. Previously, multiple resection enzymes were shown to be regulated by Cdk1 to promote resection in the S and G2 phases when sister chromatid is present as the ideal template for homologous recombination. Our new finding demonstrates that Cdk1 controls this process also at the level of chromatin remodeling. To our knowledge, this is the first example that a chromatin remodeler is directly controlled in cell cycle.

Fun30 phosphorylation on S28 increases upon DNA damage, and optimal Fun30 phosphorylation requires the recruitment of Cdk1 and Fun30 to DSBs. These observations lead us to propose that the recruitment of Cdk1-cyclin complexes to DSBs enhances the phosphorylation of substrates involved in the DNA damage response. Such a mechanism would facilitate the phosphorylation of DNA damage response proteins that are suboptimal Cdk1 targets and regulate the activity of DNA repair enzymes locally instead of globally. Cdk1 is essential for DSB repair by HR and damage checkpoint signaling in all eukaryotes. Paradoxically, the activity of Cdk1 is downregulated upon DNA damage in most of the eukaryotes including fission yeast and humans. Our observation on increased Cdk1 activity at the sites of DNA damage provides a possible explanation for these incompatible observations. Specifically, the DSB recruitment of Cdk1 would provide a high local level of kinase activity necessary for the timely resection, repair and checkpoint activation. Given that the activity of Cdk1-Clb2 and Cdk1-Clb5 complexes toward some targets increases in response to DNA damage, it will be of interest to identify the full complement of their targets in cells challenged with DNA damage.

Supplementary MaterialSUPPLEMENTARY DATA

SUPPLEMENTARY DATASupplementary Data are available at NAR Online.

Supplementary Data are available at NAR Online.

FUNDINGNational Institutes of Health [GM080600 to G.I., ES007061, ES015632 to P.S., ES021441 to H.N.]; Chinese National Natural Science Foundation [31471217]; China Basic Research Program [SQ2015CC030021 to X.C.]. Funding for open access charge: Chinese National Natural Science Foundation [31471217]. National Institutes of Health [GM080600].Conflict of interest statement. None declared.

National Institutes of Health [GM080600 to G.I., ES007061, ES015632 to P.S., ES021441 to H.N.]; Chinese National Natural Science Foundation [31471217]; China Basic Research Program [SQ2015CC030021 to X.C.]. Funding for open access charge: Chinese National Natural Science Foundation [31471217]. National Institutes of Health [GM080600].

Conflict of interest statement. None declared.
