---
pmid: '31576228'
title: The DRS-AIMP2-EPRS subcomplex acts as a pivot in the multi-tRNA synthetase
  complex.
authors:
- Hahn H
- Park SH
- Kim HJ
- Kim S
- Han BW
journal: IUCrJ
year: '2019'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6760448
doi: 10.1107/S2052252519010790
---

# The DRS-AIMP2-EPRS subcomplex acts as a pivot in the multi-tRNA synthetase complex.
**Authors:** Hahn H, Park SH, Kim HJ, Kim S, Han BW
**Journal:** IUCrJ (2019)
**DOI:** [10.1107/S2052252519010790](https://doi.org/10.1107/S2052252519010790)
**PMC:** [PMC6760448](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6760448/)

## Abstract

1. IUCrJ. 2019 Aug 24;6(Pt 5):958-967. doi: 10.1107/S2052252519010790.
eCollection  2019 Sep 1.

The DRS-AIMP2-EPRS subcomplex acts as a pivot in the multi-tRNA synthetase 
complex.

Hahn H(1), Park SH(1), Kim HJ(2), Kim S(3), Han BW(1).

Author information:
(1)Research Institute of Pharmaceutical Sciences, College of Pharmacy, Seoul 
National University, Seoul 08826, Republic of Korea.
(2)College of Pharmacy, Chung-Ang University, Seoul 06974, Republic of Korea.
(3)Medicinal Bioconvergence Research Center, Department of Molecular Medicine 
and Biopharmaceutical Sciences, Seoul National University, Seoul 08826, Republic 
of Korea.

Aminoacyl-tRNA synthetases (ARSs) play essential roles in protein biosynthesis 
as well as in other cellular processes, often using evolutionarily acquired 
domains. For possible cooperativity and synergistic effects, nine ARSs assemble 
into the multi-tRNA synthetase complex (MSC) with three scaffold proteins: 
aminoacyl-tRNA synthetase complex-interacting multifunctional proteins 1, 2 and 
3 (AIMP1, AIMP2 and AIMP3). X-ray crystallographic methods were implemented in 
order to determine the structure of a ternary subcomplex of the MSC comprising 
aspartyl-tRNA synthetase (DRS) and two glutathione S-transferase (GST) domains 
from AIMP2 and glutamyl-prolyl-tRNA synthetase (AIMP2GST and EPRSGST, 
respectively). While AIMP2GST and EPRSGST interact via conventional GST 
heterodimerization, DRS strongly interacts with AIMP2GST via hydrogen bonds 
between the α7-β9 loop of DRS and the β2-α2 loop of AIMP2GST, where Ser156 of 
AIMP2GST is essential for the assembly. Structural analyses of 
DRS-AIMP2GST-EPRSGST reveal its pivotal architecture in the MSC and provide 
valuable insights into the overall assembly and conditionally required 
disassembly of the MSC.

© Hyunggu Hahn et al. 2019.

DOI: 10.1107/S2052252519010790
PMCID: PMC6760448
PMID: 31576228

## Full Text

1.Introduction   Aminoacyl-tRNA synthetases (ARSs) play an essential role in protein biosynthesis through the attachment of amino acids to their cognate tRNAs, which is the first step in the translation process (Berg & Offengand, 1958 ▸). In addition to their primary role in protein synthesis, ARSs of higher eukaryotes additionally play roles in essential nontranslational functions through the acquisition of additional motifs or domains, or by generating alternative splicing variants which exhibit different physiological effects (Guo et al., 2010 ▸; Lo et al., 2014 ▸).Nine of the 20 cytosolic ARSs interact with three scaffold proteins: amino­acyl-tRNA synthetase complex-interacting multifunctional proteins 1, 2 and 3 (AIMP1, AIMP2 and AIMP3). Together, they assemble into the multi-tRNA synthetase complex (MSC), with a molecular weight of ∼1.5 MDa (Rho et al., 1999 ▸; Robinson et al., 2000 ▸). The formation of the MSC has been known to expedite protein synthesis through sequestering and stabilizing the components, which are otherwise rapidly degraded (Han et al., 2006 ▸). Thus, the MSC provides a constant flow of charged tRNA for the translational machinery (Negrutskii & Deutscher, 1991 ▸; Kyriacou & Deutscher, 2008 ▸). Conversely, nontranslational functions of ARSs were often displayed following dissociation from the MSC in response to various stress signals (Lee et al., 2004 ▸). The spectrum of such downstream signal pathways ranges from proliferation (Ko et al., 2000 ▸; Han et al., 2012 ▸) and tumorigenesis (Choi et al., 2011 ▸) to tumor suppression (Han et al., 2008 ▸) and apoptosis (Ko et al., 2001 ▸).Among the three scaffold proteins, AIMP2 possesses the most binding sites for MSC components. The N-terminal region of AIMP2 binds the homodimeric interface of lysyl-tRNA synthetase (KRS; Ofir-Birin et al., 2013 ▸), while the adjacent amphipathic α-helix region makes coiled-coil inter­actions with a stable subcomplex composed of arginyl-tRNA synthetase (RRS), glutaminyl-tRNA synthetase (QRS) and AIMP1 (the RQA1 subcomplex; Ahn et al., 2003 ▸). The C-terminal region of AIMP2 contains a glutathione S-transferase (GST)-homology domain which structurally facilitates a heterotetrameric GST complex with three different GST-homology domains from glutamyl-prolyl-tRNA synthetase (EPRS), AIMP3 and methionyl-tRNA synthetase (MRS) (the A2EA3M subcomplex; Cho et al., 2015 ▸).Aspartyl-tRNA synthetase (DRS) is another known binding partner of AIMP2 in the MSC (Kim et al., 2002 ▸). We previously reported the crystal structure and a potential nontranslational role of DRS released from the MSC (Kim et al., 2013 ▸). However, it remains unclear whether there is a direct interaction between AIMP2 and DRS, which prohibits a systematic understanding of MSC assembly and dis­assembly. Hence, we tried to gain insights into the dissociation process by elucidating the protein–protein interaction network within the MSC.Here, we report the crystal structure of a ternary sub­complex of the MSC, DRS–AIMP2GST–EPRSGST (the DA2E subcomplex). Unique interaction modes between DRS and AIMP2, and an ∼294 kDa decameric DA2EA3M model, were generated from our DA2E subcomplex structure. The structural information on the interactions between DRS, AIMP2 and EPRS, as well as an updated model of the MSC, provides valuable insights into the pivotal role of DRS, into the overall architecture of the MSC in higher eukaryotes and into multi-potent functions as a whole.

Introduction

Aminoacyl-tRNA synthetases (ARSs) play an essential role in protein biosynthesis through the attachment of amino acids to their cognate tRNAs, which is the first step in the translation process (Berg & Offengand, 1958 ▸). In addition to their primary role in protein synthesis, ARSs of higher eukaryotes additionally play roles in essential nontranslational functions through the acquisition of additional motifs or domains, or by generating alternative splicing variants which exhibit different physiological effects (Guo et al., 2010 ▸; Lo et al., 2014 ▸).

Nine of the 20 cytosolic ARSs interact with three scaffold proteins: amino­acyl-tRNA synthetase complex-interacting multifunctional proteins 1, 2 and 3 (AIMP1, AIMP2 and AIMP3). Together, they assemble into the multi-tRNA synthetase complex (MSC), with a molecular weight of ∼1.5 MDa (Rho et al., 1999 ▸; Robinson et al., 2000 ▸). The formation of the MSC has been known to expedite protein synthesis through sequestering and stabilizing the components, which are otherwise rapidly degraded (Han et al., 2006 ▸). Thus, the MSC provides a constant flow of charged tRNA for the translational machinery (Negrutskii & Deutscher, 1991 ▸; Kyriacou & Deutscher, 2008 ▸). Conversely, nontranslational functions of ARSs were often displayed following dissociation from the MSC in response to various stress signals (Lee et al., 2004 ▸). The spectrum of such downstream signal pathways ranges from proliferation (Ko et al., 2000 ▸; Han et al., 2012 ▸) and tumorigenesis (Choi et al., 2011 ▸) to tumor suppression (Han et al., 2008 ▸) and apoptosis (Ko et al., 2001 ▸).

Among the three scaffold proteins, AIMP2 possesses the most binding sites for MSC components. The N-terminal region of AIMP2 binds the homodimeric interface of lysyl-tRNA synthetase (KRS; Ofir-Birin et al., 2013 ▸), while the adjacent amphipathic α-helix region makes coiled-coil inter­actions with a stable subcomplex composed of arginyl-tRNA synthetase (RRS), glutaminyl-tRNA synthetase (QRS) and AIMP1 (the RQA1 subcomplex; Ahn et al., 2003 ▸). The C-terminal region of AIMP2 contains a glutathione S-transferase (GST)-homology domain which structurally facilitates a heterotetrameric GST complex with three different GST-homology domains from glutamyl-prolyl-tRNA synthetase (EPRS), AIMP3 and methionyl-tRNA synthetase (MRS) (the A2EA3M subcomplex; Cho et al., 2015 ▸).

Aspartyl-tRNA synthetase (DRS) is another known binding partner of AIMP2 in the MSC (Kim et al., 2002 ▸). We previously reported the crystal structure and a potential nontranslational role of DRS released from the MSC (Kim et al., 2013 ▸). However, it remains unclear whether there is a direct interaction between AIMP2 and DRS, which prohibits a systematic understanding of MSC assembly and dis­assembly. Hence, we tried to gain insights into the dissociation process by elucidating the protein–protein interaction network within the MSC.

Here, we report the crystal structure of a ternary sub­complex of the MSC, DRS–AIMP2GST–EPRSGST (the DA2E subcomplex). Unique interaction modes between DRS and AIMP2, and an ∼294 kDa decameric DA2EA3M model, were generated from our DA2E subcomplex structure. The structural information on the interactions between DRS, AIMP2 and EPRS, as well as an updated model of the MSC, provides valuable insights into the pivotal role of DRS, into the overall architecture of the MSC in higher eukaryotes and into multi-potent functions as a whole.

2.Materials and methods   2.1.Cloning, expression and purification of the DRS–AIMP2-DX2-S34–EPRSGST ternary complex   N-terminally truncated human DRS (residues Ala21–Pro501), an N-terminally truncated human AIMP2 splice variant lacking exon 2 (residues Ser34–Gln45 and Asp115–Lys320, named AIMP2-DX2-S34) and C-terminally truncated human EPRSGST (Met1–Gln157) were amplified using PCR. Each gene was cloned into pET-28a(+) vector (Novagen) to contain N-terminal, C-terminal and C-terminal His6 tags, respectively. The DRS, AIMP2-DX2-S34 and EPRSGST cloned plasmids were separately transformed into the Rosetta2 (DE3)pLysS, BL21-CodonPlus (DE3)-RIPL and BL21-CodonPlus (DE3)-RIPL Escherichia coli strains, respectively. The transformed cells were grown for 16 h in Luria broth medium without induction for use as seeds in scaled-up mixture cultures. The three uninduced E. coli seeds containing the DRS, AIMP2-DX2-S34 or EPRSGST plasmids were pooled into scaled-up culture medium to give a total seed ratio of 1:50. The scaled-up cultured cells were grown in Luria broth medium and induced by 0.5 mM isopropyl β-d-1-thiogalactopyranoside at an OD600 nm of 0.5, followed by further incubation at 20°C for 16 h. Harvested cells were lysed by a cell sonicator (Sonics) in lysis buffer consisting of 20 mM Tris–HCl pH 7.5, 500 mM NaCl, 35 mM imidazole, 1 mM phenylmethylsulfonyl fluoride. Cell debris was removed by centrifugation at 35 000g for 50 min at 4°C and the supernatant was then filtered with a 0.22 µm filter to remove cell debris and any aggregated proteins. The filtered supernatant was applied onto a HiTrap Chelating HP column (GE Healthcare) equilibrated with lysis buffer for affinity chromatography. Unbound or weakly bound proteins were removed from the HiTrap Chelating HP column by six column-volume elutions with washing buffer consisting of 20 mM Tris–HCl pH 7.5, 500 mM NaCl, 50 mM imidazole. The retained proteins were eluted by gradually increasing addition of a buffer consisting of 20 mM Tris–HCl pH 7.5, 500 mM NaCl, 500 mM imidazole. The eluted protein samples were further purified by size-exclusion chromatography on a HiLoad 16/600 Superdex 200 pg column (GE Healthcare) equilibrated with a buffer consisting of 50 mM HEPES–NaOH pH 8.0, 200 mM NaCl, 5 mM DTT, 1%(v/v) glycerol. The purified DRS–AIMP2-DX2-S34–EPRSGST complex was concentrated to 6.0 mg ml−1 for crystallization.Phospho-ablative (AIMP2GST S156A) and phosphor-mimetic (AIMP2GST S156D and S156E) mutations were introduced into AIMP2-DX2-S34 by site-directed mutagenesis using the QuikChange II Site-Directed Mutagenesis Kit (Agilent Technologies) and the proteins were purified as described above.Metal analysis of the purified DRS–AIMP2-DX2-S34–EPRSGST complex was carried out by inductively coupled plasma mass spectrometry (ICP-MS; 820MS, Analytik Jena).2.2.Crystallization and structure determination   Initial crystals of the DA2E subcomplex were grown at 22°C by the sitting-drop vapor-diffusion method by mixing equal volumes of the protein at 6.0 mg ml−1 and a crystallization solution consisting of 1.2 M sodium phosphate monobasic, 0.8 M potassium phosphate dibasic, 0.1 M CAPS pH 10.5, 0.2 M lithium sulfate. DA2E crystals were optimized for X-ray data collection at 22°C using the hanging-drop vapor-diffusion method by mixing equal volumes of protein solution and crystallization solution consisting of 1.3 M sodium phosphate monobasic, 0.5 M potassium phosphate dibasic, 0.1 M CAPS pH 10.5, 0.5 M lithium sulfate with initial crystals as a seed stock and adding 0.015 mM CYMAL-7.The crystals grown in the sitting drop were already cryoprotected by the reservoir solution and were flash-cooled in a nitrogen-gas stream at 100 K. Data were collected from the crystals to 3.6 Å resolution. Raw X-ray diffraction data were indexed and scaled using the HKL-2000 suite (Otwinowski & Minor, 1997 ▸). The crystal structure of DRS–AIMP2-DX2-S34–EPRSGST was determined by molecular replacement with MOLREP (Vagin & Teplyakov, 2010 ▸) using the refined structures of DRS, AIMP2 and EPRSGST as phasing models.Model building of the crystal structures was carried out with Coot (Emsley et al., 2010 ▸) and further refinement was implemented in LORESTR (Kovalevskiy et al., 2016 ▸) and REFMAC5 (Murshudov et al., 2011 ▸) in the CCP4 suite (Winn et al., 2011 ▸) and in phenix.refine (Afonine et al., 2012 ▸). Extra electron densities found around the modeled DA2E complex were carefully analyzed before manually assigning Zn2+ ions, phosphate ions or water molecules. Zn2+ ions and most of the phosphate ions were modeled based on our ICP-MS results and a binding model of DRS–tRNAAsp predicted from the structure of the yeast DRS–tRNAAsp complex (PDB entry 1asy; Ruff et al., 1991 ▸), respectively. For modeling of water molecules, a few representative programs, such as Coot, REFMAC5 and phenix.refine, were implemented but could not assign water molecules that were commonly confirmed by the programs. Thus, only 12 water molecules were manually assigned by iterative placement and confirmation with subsequent refinement. Validation of the crystal structures was implemented in MolProbity (Chen et al., 2010 ▸) and the PDB Validation Server. Data-collection and refinement statistics are summarized in Table 1 ▸. The coordinates and structure factors for the DA2E subcomplex have been deposited in the Protein Data Bank (http://www.rcsb.org) with code 6iy6.

Materials and methods

2.1.Cloning, expression and purification of the DRS–AIMP2-DX2-S34–EPRSGST ternary complex   N-terminally truncated human DRS (residues Ala21–Pro501), an N-terminally truncated human AIMP2 splice variant lacking exon 2 (residues Ser34–Gln45 and Asp115–Lys320, named AIMP2-DX2-S34) and C-terminally truncated human EPRSGST (Met1–Gln157) were amplified using PCR. Each gene was cloned into pET-28a(+) vector (Novagen) to contain N-terminal, C-terminal and C-terminal His6 tags, respectively. The DRS, AIMP2-DX2-S34 and EPRSGST cloned plasmids were separately transformed into the Rosetta2 (DE3)pLysS, BL21-CodonPlus (DE3)-RIPL and BL21-CodonPlus (DE3)-RIPL Escherichia coli strains, respectively. The transformed cells were grown for 16 h in Luria broth medium without induction for use as seeds in scaled-up mixture cultures. The three uninduced E. coli seeds containing the DRS, AIMP2-DX2-S34 or EPRSGST plasmids were pooled into scaled-up culture medium to give a total seed ratio of 1:50. The scaled-up cultured cells were grown in Luria broth medium and induced by 0.5 mM isopropyl β-d-1-thiogalactopyranoside at an OD600 nm of 0.5, followed by further incubation at 20°C for 16 h. Harvested cells were lysed by a cell sonicator (Sonics) in lysis buffer consisting of 20 mM Tris–HCl pH 7.5, 500 mM NaCl, 35 mM imidazole, 1 mM phenylmethylsulfonyl fluoride. Cell debris was removed by centrifugation at 35 000g for 50 min at 4°C and the supernatant was then filtered with a 0.22 µm filter to remove cell debris and any aggregated proteins. The filtered supernatant was applied onto a HiTrap Chelating HP column (GE Healthcare) equilibrated with lysis buffer for affinity chromatography. Unbound or weakly bound proteins were removed from the HiTrap Chelating HP column by six column-volume elutions with washing buffer consisting of 20 mM Tris–HCl pH 7.5, 500 mM NaCl, 50 mM imidazole. The retained proteins were eluted by gradually increasing addition of a buffer consisting of 20 mM Tris–HCl pH 7.5, 500 mM NaCl, 500 mM imidazole. The eluted protein samples were further purified by size-exclusion chromatography on a HiLoad 16/600 Superdex 200 pg column (GE Healthcare) equilibrated with a buffer consisting of 50 mM HEPES–NaOH pH 8.0, 200 mM NaCl, 5 mM DTT, 1%(v/v) glycerol. The purified DRS–AIMP2-DX2-S34–EPRSGST complex was concentrated to 6.0 mg ml−1 for crystallization.Phospho-ablative (AIMP2GST S156A) and phosphor-mimetic (AIMP2GST S156D and S156E) mutations were introduced into AIMP2-DX2-S34 by site-directed mutagenesis using the QuikChange II Site-Directed Mutagenesis Kit (Agilent Technologies) and the proteins were purified as described above.Metal analysis of the purified DRS–AIMP2-DX2-S34–EPRSGST complex was carried out by inductively coupled plasma mass spectrometry (ICP-MS; 820MS, Analytik Jena).

Cloning, expression and purification of the DRS–AIMP2-DX2-S34–EPRSGST ternary complex

N-terminally truncated human DRS (residues Ala21–Pro501), an N-terminally truncated human AIMP2 splice variant lacking exon 2 (residues Ser34–Gln45 and Asp115–Lys320, named AIMP2-DX2-S34) and C-terminally truncated human EPRSGST (Met1–Gln157) were amplified using PCR. Each gene was cloned into pET-28a(+) vector (Novagen) to contain N-terminal, C-terminal and C-terminal His6 tags, respectively. The DRS, AIMP2-DX2-S34 and EPRSGST cloned plasmids were separately transformed into the Rosetta2 (DE3)pLysS, BL21-CodonPlus (DE3)-RIPL and BL21-CodonPlus (DE3)-RIPL Escherichia coli strains, respectively. The transformed cells were grown for 16 h in Luria broth medium without induction for use as seeds in scaled-up mixture cultures. The three uninduced E. coli seeds containing the DRS, AIMP2-DX2-S34 or EPRSGST plasmids were pooled into scaled-up culture medium to give a total seed ratio of 1:50. The scaled-up cultured cells were grown in Luria broth medium and induced by 0.5 mM isopropyl β-d-1-thiogalactopyranoside at an OD600 nm of 0.5, followed by further incubation at 20°C for 16 h. Harvested cells were lysed by a cell sonicator (Sonics) in lysis buffer consisting of 20 mM Tris–HCl pH 7.5, 500 mM NaCl, 35 mM imidazole, 1 mM phenylmethylsulfonyl fluoride. Cell debris was removed by centrifugation at 35 000g for 50 min at 4°C and the supernatant was then filtered with a 0.22 µm filter to remove cell debris and any aggregated proteins. The filtered supernatant was applied onto a HiTrap Chelating HP column (GE Healthcare) equilibrated with lysis buffer for affinity chromatography. Unbound or weakly bound proteins were removed from the HiTrap Chelating HP column by six column-volume elutions with washing buffer consisting of 20 mM Tris–HCl pH 7.5, 500 mM NaCl, 50 mM imidazole. The retained proteins were eluted by gradually increasing addition of a buffer consisting of 20 mM Tris–HCl pH 7.5, 500 mM NaCl, 500 mM imidazole. The eluted protein samples were further purified by size-exclusion chromatography on a HiLoad 16/600 Superdex 200 pg column (GE Healthcare) equilibrated with a buffer consisting of 50 mM HEPES–NaOH pH 8.0, 200 mM NaCl, 5 mM DTT, 1%(v/v) glycerol. The purified DRS–AIMP2-DX2-S34–EPRSGST complex was concentrated to 6.0 mg ml−1 for crystallization.

Phospho-ablative (AIMP2GST S156A) and phosphor-mimetic (AIMP2GST S156D and S156E) mutations were introduced into AIMP2-DX2-S34 by site-directed mutagenesis using the QuikChange II Site-Directed Mutagenesis Kit (Agilent Technologies) and the proteins were purified as described above.

Metal analysis of the purified DRS–AIMP2-DX2-S34–EPRSGST complex was carried out by inductively coupled plasma mass spectrometry (ICP-MS; 820MS, Analytik Jena).

2.2.Crystallization and structure determination   Initial crystals of the DA2E subcomplex were grown at 22°C by the sitting-drop vapor-diffusion method by mixing equal volumes of the protein at 6.0 mg ml−1 and a crystallization solution consisting of 1.2 M sodium phosphate monobasic, 0.8 M potassium phosphate dibasic, 0.1 M CAPS pH 10.5, 0.2 M lithium sulfate. DA2E crystals were optimized for X-ray data collection at 22°C using the hanging-drop vapor-diffusion method by mixing equal volumes of protein solution and crystallization solution consisting of 1.3 M sodium phosphate monobasic, 0.5 M potassium phosphate dibasic, 0.1 M CAPS pH 10.5, 0.5 M lithium sulfate with initial crystals as a seed stock and adding 0.015 mM CYMAL-7.The crystals grown in the sitting drop were already cryoprotected by the reservoir solution and were flash-cooled in a nitrogen-gas stream at 100 K. Data were collected from the crystals to 3.6 Å resolution. Raw X-ray diffraction data were indexed and scaled using the HKL-2000 suite (Otwinowski & Minor, 1997 ▸). The crystal structure of DRS–AIMP2-DX2-S34–EPRSGST was determined by molecular replacement with MOLREP (Vagin & Teplyakov, 2010 ▸) using the refined structures of DRS, AIMP2 and EPRSGST as phasing models.Model building of the crystal structures was carried out with Coot (Emsley et al., 2010 ▸) and further refinement was implemented in LORESTR (Kovalevskiy et al., 2016 ▸) and REFMAC5 (Murshudov et al., 2011 ▸) in the CCP4 suite (Winn et al., 2011 ▸) and in phenix.refine (Afonine et al., 2012 ▸). Extra electron densities found around the modeled DA2E complex were carefully analyzed before manually assigning Zn2+ ions, phosphate ions or water molecules. Zn2+ ions and most of the phosphate ions were modeled based on our ICP-MS results and a binding model of DRS–tRNAAsp predicted from the structure of the yeast DRS–tRNAAsp complex (PDB entry 1asy; Ruff et al., 1991 ▸), respectively. For modeling of water molecules, a few representative programs, such as Coot, REFMAC5 and phenix.refine, were implemented but could not assign water molecules that were commonly confirmed by the programs. Thus, only 12 water molecules were manually assigned by iterative placement and confirmation with subsequent refinement. Validation of the crystal structures was implemented in MolProbity (Chen et al., 2010 ▸) and the PDB Validation Server. Data-collection and refinement statistics are summarized in Table 1 ▸. The coordinates and structure factors for the DA2E subcomplex have been deposited in the Protein Data Bank (http://www.rcsb.org) with code 6iy6.

Crystallization and structure determination

Initial crystals of the DA2E subcomplex were grown at 22°C by the sitting-drop vapor-diffusion method by mixing equal volumes of the protein at 6.0 mg ml−1 and a crystallization solution consisting of 1.2 M sodium phosphate monobasic, 0.8 M potassium phosphate dibasic, 0.1 M CAPS pH 10.5, 0.2 M lithium sulfate. DA2E crystals were optimized for X-ray data collection at 22°C using the hanging-drop vapor-diffusion method by mixing equal volumes of protein solution and crystallization solution consisting of 1.3 M sodium phosphate monobasic, 0.5 M potassium phosphate dibasic, 0.1 M CAPS pH 10.5, 0.5 M lithium sulfate with initial crystals as a seed stock and adding 0.015 mM CYMAL-7.

The crystals grown in the sitting drop were already cryoprotected by the reservoir solution and were flash-cooled in a nitrogen-gas stream at 100 K. Data were collected from the crystals to 3.6 Å resolution. Raw X-ray diffraction data were indexed and scaled using the HKL-2000 suite (Otwinowski & Minor, 1997 ▸). The crystal structure of DRS–AIMP2-DX2-S34–EPRSGST was determined by molecular replacement with MOLREP (Vagin & Teplyakov, 2010 ▸) using the refined structures of DRS, AIMP2 and EPRSGST as phasing models.

Model building of the crystal structures was carried out with Coot (Emsley et al., 2010 ▸) and further refinement was implemented in LORESTR (Kovalevskiy et al., 2016 ▸) and REFMAC5 (Murshudov et al., 2011 ▸) in the CCP4 suite (Winn et al., 2011 ▸) and in phenix.refine (Afonine et al., 2012 ▸). Extra electron densities found around the modeled DA2E complex were carefully analyzed before manually assigning Zn2+ ions, phosphate ions or water molecules. Zn2+ ions and most of the phosphate ions were modeled based on our ICP-MS results and a binding model of DRS–tRNAAsp predicted from the structure of the yeast DRS–tRNAAsp complex (PDB entry 1asy; Ruff et al., 1991 ▸), respectively. For modeling of water molecules, a few representative programs, such as Coot, REFMAC5 and phenix.refine, were implemented but could not assign water molecules that were commonly confirmed by the programs. Thus, only 12 water molecules were manually assigned by iterative placement and confirmation with subsequent refinement. Validation of the crystal structures was implemented in MolProbity (Chen et al., 2010 ▸) and the PDB Validation Server. Data-collection and refinement statistics are summarized in Table 1 ▸. The coordinates and structure factors for the DA2E subcomplex have been deposited in the Protein Data Bank (http://www.rcsb.org) with code 6iy6.

3.Results   3.1.
In vitro MSC subcomplex formation: attempts to obtain a stable composition   In order to elucidate the overall architecture of the MSC, we attempted the purification and crystallization of various MSC subcomplexes with respect to DRS, the structure of which had already been determined (Kim et al., 2013 ▸). Among the MSC components, the C-terminal domain of AIMP2 is known to bind DRS (Quevillon et al., 1999 ▸), but the actual interaction mode between AIMP2 and DRS remained unclear. We observed that DRS forms binary, tertiary, quaternary and pentamery subcomplexes with the GST domains of AIMP2, EPRS, MRS and AIMP3. Importantly, an AIMP2 splicing variant lacking exon 2 (AIMP2-DX2) was included in our trial because AIMP2-DX2 contains the C-terminal GST domain. To obtain sufficiently pure extracts of AIMP2-DX2 for complex structure determination, an N-terminal thioredoxin (Trx) tag was incorporated. When purified Trx-AIMP2-DX2 was subjected to Trx-tag cleavage by incubation with thrombin, the cleaved product seemed to have a smaller molecular weight (∼24 kDa) than the anticipated AIMP2-DX2 construct (∼28 kDa) as analyzed by SDS–PAGE (data not shown). Further analysis by mass spectrometry revealed that thrombin treatment removed a segment of the N-terminal peptide of the AIMP2-DX2 protein along with the Trx tag. We confirmed that the first five N-terminal residues of the cleaved AIMP2-DX2 were Ser-Tyr-Gly-Pro-Ala by N-terminal sequencing, which corresponds to residues 34–38 of AIMP2. We named the 33-amino-acid-truncated form of AIMP2 AIMP2-DX2-S34. This particular truncated form of AIMP2 was likewise observed in the mass-spectrometric analysis of MSC extracted from a human cell culture (Park et al., 2015 ▸), thus justifying our trial of AIMP2-DX2-S34 incorporation into MSC subcomplexes.Because the AIMP2-DX2-S34 construct was more stable in solution than its full-length counterpart AIMP2-DX2, it was used for MSC subcomplex formation along with DRS, EPRSGST, AIMP3 and MRSGST, each with either N-terminal or C-terminal His6 tags [Fig. 1 ▸(a)]. The plasmids carrying each of the proteins were cloned into separate expression strains and the expression strains were then collected together for a scaled-up culture. The proteins were only allowed to form complexes upon cell disruption by sonication. Nevertheless, the proteins stably formed the following MSC subcomplexes as verified by size-exclusion chromatography and SDS–PAGE analyses [Fig. 1 ▸(b)]: DRS–AIMP2-DX-S34 (DA2), DRS–AIMP2-DX2-S34–EPRSGST (DA2E), DRS–AIMP2-DX2-S34–EPRSGST–AIMP3 (DA2EA3) and DRS–AIMP2-DX2-S34–EPRSGST–AIMP3–MRSGST (DA2EA3M). Of the four MSC subcomplexes that were purified, the DA2E sub­complex (∼192 kDa) was successfully crystallized for structure determination.3.2.The DRS–AIMP2GST–EPRSGST (DA2E) subcomplex is orientated around the DRS homodimer   We successfully determined the tertiary complex structure of DA2E at 3.6 Å resolution by the molecular-replacement (MR) method using the structures of DRS and the AIMP2GST–EPRSGST binary complex (PDB entries 4j15 and 5a34; Cho et al., 2015 ▸; Kim et al., 2013 ▸) as phasing models. Despite our efforts to elucidate the structure of AIMP2 with the exon 1 region, only the GST domain of AIMP2 corresponding to exons 3 and 4 (residues 117–320) could be determined (Supplementary Fig. S2). Therefore, in the following we will use the term AIMP2GST instead of AIMP2-DX2-S34 to refer to the protein. The DA2E crystals belonged to space group P61 and contained two DA2E subcomplexes in the asymmetric unit, each comprising two molecules of DRS, AIMP2GST and EPRSGST.The DA2E components are oriented around the central DRS homodimer in a twofold rotational symmetry with the center of mass of DA2E located in the DRS dimeric interface [Fig. 1 ▸(c)]. The DRS structure in the DA2E complex did not exhibit noticeable structural changes compared with that of DRS alone, in that the anticodon-binding domain (residues 57–146) and the catalytic domain (residues 189–497) of DRS could be well modeled. However, the hinge region between the anticodon-binding domain and the catalytic domain (residues 155–178), as well as two other loop regions (residues 225–248 and 273–281) that are known to bind tRNAAsp in an induced-fit manner (Sauter et al., 2000 ▸), could not be modeled owing to a lack of electron density [Supplementary Fig. S1(a)]. Extra electron density around the DA2E complex could not be clearly discerned owing to the low resolution limit and were manually assigned with discretion considering their environment. Extra spherical electron densities observed at the DRS dimeric interfaces were coordinated by two His204/Glu208 pairs, each from one monomer, and were modeled as Zn2+ ions according to the metal-content analysis results using ICP-MS (Supplementary Fig. S3). Extra electron densities observed at the tRNAAsp-binding sites of DRS molecules (catalytic sites and anticodon-binding sites) were modeled as phosphate ions present in the crystallization solution at high concentrations, which seem to mimic the phosphate moieties of the tRNAAsp backbone (Supplementary Fig. S4). Extra electron densities observed at the DRS–AIMP2GST interfaces were also modeled as phosphate ions which interact with backbone O and N atoms of the nearby DRS. Since the interactions do not seem to be biologically significant and the phosphate ions were only introduced for crystallization after in vitro DA2E complex formation, the phosphate ions might have contributed to further stabilizing the DA2E complexes in the crystal. Modeling water molecules using modeling programs did not produce consistent results owing to the low resolution limit. After a careful iterative process, only a dozen water molecules were modeled.AIMP2GST and EPRSGST both adopt a GST fold consisting of an N-terminal Trx fold and a C-terminal α-helical sub­domain. EPRSGST adopts a canonical four-stranded β-sheet in its Trx subdomain, whereas AIMP2GST has an extra stretch of peptide (residues 148–170) which constitutes a strand–loop–helix motif (β2–α2) following helix α1, thus comprising a characteristic five-stranded β-sheet [Supplementary Fig. S1(b)]. Residues 171–179 and 289–291 of AIMP2GST in the α2–β3 loop and the α6–α7 loop, respectively, could not be modeled owing to a lack of electron density [Supplementary Fig. S1(b)].
Protein Interfaces, Surfaces and Assemblies (PISA) analysis (Krissinel & Henrick, 2007 ▸) of the interfaces in the DA2E subcomplex structure revealed that interactions between DRS and AIMP2GST arise mainly from residues in the α7–β9 loop (residues 338–342) and the C-terminus of helix α9 (residues 383 and 384) of DRS and in the N-terminal subdomain of AIMP2GST, which comprises a β-sheet [Figs. 2 ▸(a) and 2 ▸(b)]. The interaction between AIMP2GST and EPRSGST is mainly through the heterodimerization of GST domains, in which the proteins are related to each other by a twofold rotational symmetry. In the following sections, the interfaces between the components of the DA2E subcomplex will be extensively described.3.3.Ser156 of AIMP2 plays a key role in the interaction between DRS and AIMP2GST via hydrogen bonds   The DA2E structure revealed that the binding interfaces of DRS and AIMP2GST have an average area of 747.2 Å2 as calculated by PISA. The AIMP2-binding motif of DRS is located in the middle of the DRS catalytic domain, discrete from the DRS dimeric interface, the tRNAAsp-binding site and the active site [Fig. 1 ▸(c)], which is in accordance with the fact that ARSs retain aminoacyl-tRNA synthetase activities in the MSC (Mirande et al., 1985 ▸).The complex formation of DRS and AIMP2GST is mainly mediated by two binding interfaces [Fig. 2 ▸(a)]. The major interface is formed by multiple hydrogen bonds between Lys338–Pro342 in the α7–β9 loop of DRS and His153–Ser156 in the β2–α2 loop of AIMP2GST [Fig. 2 ▸(b)], while the other interface involves helix α9 of DRS and the β-sheet of AIMP2GST [Fig. 2 ▸(c)]. The loops in the major interface establish multiple hydrogen bonds between the Cα backbones, which are reminiscent of the hydrogen-bonding networks of an antiparallel β-sheet. Interestingly, Ser156 of AIMP2GST seems to play a crucial role in mediating the interaction between DRS and AIMP2GST. The side chain of Ser156 acts as a hydrogen-bond donor coupled with the backbone amide group of Phe339 of DRS, while the backbone amide group of Ser156 forms a hydrogen bond to the backbone O atom of Phe339 of DRS [Fig. 2 ▸(b)]. Ser156 has previously been shown to be exposed to the solvent and phosphorylated by TGF-β signaling. Phosphorylation of Ser156 induced dissociation of AIMP2 from the MSC, followed by translocation into the cell nucleus, where AIMP2 functions as a tumor suppressor in concert with Smurf2 (Kim et al., 2016 ▸).Two mutants, AIMP2GST S156D and S156E, were generated to mimic the phosphorylation of Ser156, and the AIMP2GST S156A mutant was generated to abolish the side-chain-mediated hydrogen bond. The interaction between the AIMP2GST mutants and DRS was evaluated by size-exclusion chromatography [Fig. 2 ▸(d)]. All three AIMP2GST mutants could not form binary complexes with DRS, which suggests that the hydrogen bond from the side chain of AIMP2 Ser156 is crucial for the interaction with DRS and that Ser156 phosphorylation would disintegrate the MSC assembly.3.4.The DA2E ternary complex accompanies a conformational change in interface 1 between two GST domains   
PISA analysis of the DA2E structure revealed that the binding interfaces of AIMP2GST and EPRSGST have an average area of 1083.3 Å2, which is larger than that between DRS and AIMP2GST. The networks of interactions between heterodimeric GST domains in the MSC have been extensively analyzed previously, and were assigned as Interface 1 and Interface 2 (Cho et al., 2015 ▸). Here, we briefly describe the nature of Interface 1 between AIMP2GST and EPRSGST and intensively analyze a unique conformational change therein elicited by the incorporation of DRS compared with the heterodimeric A2E structure.Interface 1 is formed by the bundling of helices α3 and α4 of AIMP2GST and α2 and α3 of EPRSGST, which establish multiple intermolecular hydrogen bonds and salt bridges. When the structures of AIMP2GST in the DA2E subcomplex and in the heterodimeric A2E subcomplex (PDB entry 5a34; Cho et al., 2015 ▸) were compared, the two structures were very similar over 201 equivalent Cα atoms, with a root-mean-square deviation (r.m.s.d.) of 0.87 Å. Although the constructs used for this comparison were truncated differently, the GST-domain structure was not affected by the truncations. However, a remarkably high r.m.s.d. was observed for residues Phe199–Pro206, with a maximum Cα deviation of 8.78 Å at Thr203 [Fig. 2 ▸(e)]. These residues belong to the β4–β5 loop, the last three residues of which (Met204-Cys205-Pro206) make hydrophobic contacts with helix α4 of EPRSGST [Fig. 2 ▸(c)]. Incorporation of DRS causes a major conformational change in the β4–β5 loop, where the C-terminal end of helix α9 of DRS rests on the β-sheet of AIMP2GST [Fig. 2 ▸(c), dashed red circle]. Superposing the AIMP2 structure from the A2E subcomplex [Fig. 2 ▸(c), white] (PDB entry 5a34) onto our AIMP2GST structure [Fig. 2 ▸(c), orange] reveals that helix α9 of DRS occupies the region where the β4–β5 loop would be in the A2E subcomplex structure. The results of PISA analysis show that residues Glu382-Lys383-Tyr384 on helix α9 of DRS make mainly hydrophobic contacts with Leu119, Val123 and Ile201 of AIMP2GST, while the β4–β5 loop of AIMP2GST is sandwiched between DRS and EPRSGST to adopt a more ordered β-turn structure [Fig. 2 ▸(c), orange]. Consequent rearrangement of the Cα backbone in the β4–β5 loop elicited a remarkable difference in Interface 1 compared with that in the heterodimeric A2E complex.3.5.Adjacent DA2E subcomplexes are bridged via Interface 2 between two EPRSGST molecules   To our surprise, we observed two molecules of EPRSGST interacting via Interface 2, each of which are from adjacent DA2E subcomplexes in the asymmetric unit [Figs. 3 ▸(a) and 3 ▸(b)]. In the case of Interface 2 of EPRSGST and AIMP3 (PDB entry 4bvx; Cho et al., 2015 ▸), residues from helices α7 and the α4–α5 loops from each of the GST domains similarly establish a complex network via hydrogen bonds and salt bridges [Fig. 3 ▸(c)] to the EPRSGST crystallographic interface. Because EPRSGST and AIMP3 have a high overall sequence similarity of 40.1% [Fig. 3 ▸(d)] and share the same structural domain [Figs. 3 ▸(b) and 3 ▸(c)], it is understandable that the exposed surface of EPRSGST would facilitate dimerization via Interface 2 in the absence of the legitimate binding partner AIMP3.During our trial crystallization of MSC subcomplexes, we obtained crystals of the binary DA2 subcomplex, which however only diffracted to 4.5 Å resolution (data not shown), compared with crystals of the ternary DA2E subcomplex, which diffracted to 3.6 Å resolution. While DA2 subcomplex molecules were packed with no apparent noncrystallographic symmetry within the asymmetric unit, the DA2E subcomplex molecules in the asymmetric unit are related to each other by a twofold rotational symmetry [Fig. 3 ▸(a)]. Although the interaction between two EPRSGST molecules from adjacent DA2E subcomplexes within the asymmetric unit would have been a crystallographic artifact, it is clear that this interaction contributed to the stabilization and hence successful structure determination of the DA2E subcomplex.

3.1.
In vitro MSC subcomplex formation: attempts to obtain a stable composition   In order to elucidate the overall architecture of the MSC, we attempted the purification and crystallization of various MSC subcomplexes with respect to DRS, the structure of which had already been determined (Kim et al., 2013 ▸). Among the MSC components, the C-terminal domain of AIMP2 is known to bind DRS (Quevillon et al., 1999 ▸), but the actual interaction mode between AIMP2 and DRS remained unclear. We observed that DRS forms binary, tertiary, quaternary and pentamery subcomplexes with the GST domains of AIMP2, EPRS, MRS and AIMP3. Importantly, an AIMP2 splicing variant lacking exon 2 (AIMP2-DX2) was included in our trial because AIMP2-DX2 contains the C-terminal GST domain. To obtain sufficiently pure extracts of AIMP2-DX2 for complex structure determination, an N-terminal thioredoxin (Trx) tag was incorporated. When purified Trx-AIMP2-DX2 was subjected to Trx-tag cleavage by incubation with thrombin, the cleaved product seemed to have a smaller molecular weight (∼24 kDa) than the anticipated AIMP2-DX2 construct (∼28 kDa) as analyzed by SDS–PAGE (data not shown). Further analysis by mass spectrometry revealed that thrombin treatment removed a segment of the N-terminal peptide of the AIMP2-DX2 protein along with the Trx tag. We confirmed that the first five N-terminal residues of the cleaved AIMP2-DX2 were Ser-Tyr-Gly-Pro-Ala by N-terminal sequencing, which corresponds to residues 34–38 of AIMP2. We named the 33-amino-acid-truncated form of AIMP2 AIMP2-DX2-S34. This particular truncated form of AIMP2 was likewise observed in the mass-spectrometric analysis of MSC extracted from a human cell culture (Park et al., 2015 ▸), thus justifying our trial of AIMP2-DX2-S34 incorporation into MSC subcomplexes.Because the AIMP2-DX2-S34 construct was more stable in solution than its full-length counterpart AIMP2-DX2, it was used for MSC subcomplex formation along with DRS, EPRSGST, AIMP3 and MRSGST, each with either N-terminal or C-terminal His6 tags [Fig. 1 ▸(a)]. The plasmids carrying each of the proteins were cloned into separate expression strains and the expression strains were then collected together for a scaled-up culture. The proteins were only allowed to form complexes upon cell disruption by sonication. Nevertheless, the proteins stably formed the following MSC subcomplexes as verified by size-exclusion chromatography and SDS–PAGE analyses [Fig. 1 ▸(b)]: DRS–AIMP2-DX-S34 (DA2), DRS–AIMP2-DX2-S34–EPRSGST (DA2E), DRS–AIMP2-DX2-S34–EPRSGST–AIMP3 (DA2EA3) and DRS–AIMP2-DX2-S34–EPRSGST–AIMP3–MRSGST (DA2EA3M). Of the four MSC subcomplexes that were purified, the DA2E sub­complex (∼192 kDa) was successfully crystallized for structure determination.

In vitro MSC subcomplex formation: attempts to obtain a stable composition

In order to elucidate the overall architecture of the MSC, we attempted the purification and crystallization of various MSC subcomplexes with respect to DRS, the structure of which had already been determined (Kim et al., 2013 ▸). Among the MSC components, the C-terminal domain of AIMP2 is known to bind DRS (Quevillon et al., 1999 ▸), but the actual interaction mode between AIMP2 and DRS remained unclear. We observed that DRS forms binary, tertiary, quaternary and pentamery subcomplexes with the GST domains of AIMP2, EPRS, MRS and AIMP3. Importantly, an AIMP2 splicing variant lacking exon 2 (AIMP2-DX2) was included in our trial because AIMP2-DX2 contains the C-terminal GST domain. To obtain sufficiently pure extracts of AIMP2-DX2 for complex structure determination, an N-terminal thioredoxin (Trx) tag was incorporated. When purified Trx-AIMP2-DX2 was subjected to Trx-tag cleavage by incubation with thrombin, the cleaved product seemed to have a smaller molecular weight (∼24 kDa) than the anticipated AIMP2-DX2 construct (∼28 kDa) as analyzed by SDS–PAGE (data not shown). Further analysis by mass spectrometry revealed that thrombin treatment removed a segment of the N-terminal peptide of the AIMP2-DX2 protein along with the Trx tag. We confirmed that the first five N-terminal residues of the cleaved AIMP2-DX2 were Ser-Tyr-Gly-Pro-Ala by N-terminal sequencing, which corresponds to residues 34–38 of AIMP2. We named the 33-amino-acid-truncated form of AIMP2 AIMP2-DX2-S34. This particular truncated form of AIMP2 was likewise observed in the mass-spectrometric analysis of MSC extracted from a human cell culture (Park et al., 2015 ▸), thus justifying our trial of AIMP2-DX2-S34 incorporation into MSC subcomplexes.

Because the AIMP2-DX2-S34 construct was more stable in solution than its full-length counterpart AIMP2-DX2, it was used for MSC subcomplex formation along with DRS, EPRSGST, AIMP3 and MRSGST, each with either N-terminal or C-terminal His6 tags [Fig. 1 ▸(a)]. The plasmids carrying each of the proteins were cloned into separate expression strains and the expression strains were then collected together for a scaled-up culture. The proteins were only allowed to form complexes upon cell disruption by sonication. Nevertheless, the proteins stably formed the following MSC subcomplexes as verified by size-exclusion chromatography and SDS–PAGE analyses [Fig. 1 ▸(b)]: DRS–AIMP2-DX-S34 (DA2), DRS–AIMP2-DX2-S34–EPRSGST (DA2E), DRS–AIMP2-DX2-S34–EPRSGST–AIMP3 (DA2EA3) and DRS–AIMP2-DX2-S34–EPRSGST–AIMP3–MRSGST (DA2EA3M). Of the four MSC subcomplexes that were purified, the DA2E sub­complex (∼192 kDa) was successfully crystallized for structure determination.

3.2.The DRS–AIMP2GST–EPRSGST (DA2E) subcomplex is orientated around the DRS homodimer   We successfully determined the tertiary complex structure of DA2E at 3.6 Å resolution by the molecular-replacement (MR) method using the structures of DRS and the AIMP2GST–EPRSGST binary complex (PDB entries 4j15 and 5a34; Cho et al., 2015 ▸; Kim et al., 2013 ▸) as phasing models. Despite our efforts to elucidate the structure of AIMP2 with the exon 1 region, only the GST domain of AIMP2 corresponding to exons 3 and 4 (residues 117–320) could be determined (Supplementary Fig. S2). Therefore, in the following we will use the term AIMP2GST instead of AIMP2-DX2-S34 to refer to the protein. The DA2E crystals belonged to space group P61 and contained two DA2E subcomplexes in the asymmetric unit, each comprising two molecules of DRS, AIMP2GST and EPRSGST.The DA2E components are oriented around the central DRS homodimer in a twofold rotational symmetry with the center of mass of DA2E located in the DRS dimeric interface [Fig. 1 ▸(c)]. The DRS structure in the DA2E complex did not exhibit noticeable structural changes compared with that of DRS alone, in that the anticodon-binding domain (residues 57–146) and the catalytic domain (residues 189–497) of DRS could be well modeled. However, the hinge region between the anticodon-binding domain and the catalytic domain (residues 155–178), as well as two other loop regions (residues 225–248 and 273–281) that are known to bind tRNAAsp in an induced-fit manner (Sauter et al., 2000 ▸), could not be modeled owing to a lack of electron density [Supplementary Fig. S1(a)]. Extra electron density around the DA2E complex could not be clearly discerned owing to the low resolution limit and were manually assigned with discretion considering their environment. Extra spherical electron densities observed at the DRS dimeric interfaces were coordinated by two His204/Glu208 pairs, each from one monomer, and were modeled as Zn2+ ions according to the metal-content analysis results using ICP-MS (Supplementary Fig. S3). Extra electron densities observed at the tRNAAsp-binding sites of DRS molecules (catalytic sites and anticodon-binding sites) were modeled as phosphate ions present in the crystallization solution at high concentrations, which seem to mimic the phosphate moieties of the tRNAAsp backbone (Supplementary Fig. S4). Extra electron densities observed at the DRS–AIMP2GST interfaces were also modeled as phosphate ions which interact with backbone O and N atoms of the nearby DRS. Since the interactions do not seem to be biologically significant and the phosphate ions were only introduced for crystallization after in vitro DA2E complex formation, the phosphate ions might have contributed to further stabilizing the DA2E complexes in the crystal. Modeling water molecules using modeling programs did not produce consistent results owing to the low resolution limit. After a careful iterative process, only a dozen water molecules were modeled.AIMP2GST and EPRSGST both adopt a GST fold consisting of an N-terminal Trx fold and a C-terminal α-helical sub­domain. EPRSGST adopts a canonical four-stranded β-sheet in its Trx subdomain, whereas AIMP2GST has an extra stretch of peptide (residues 148–170) which constitutes a strand–loop–helix motif (β2–α2) following helix α1, thus comprising a characteristic five-stranded β-sheet [Supplementary Fig. S1(b)]. Residues 171–179 and 289–291 of AIMP2GST in the α2–β3 loop and the α6–α7 loop, respectively, could not be modeled owing to a lack of electron density [Supplementary Fig. S1(b)].
Protein Interfaces, Surfaces and Assemblies (PISA) analysis (Krissinel & Henrick, 2007 ▸) of the interfaces in the DA2E subcomplex structure revealed that interactions between DRS and AIMP2GST arise mainly from residues in the α7–β9 loop (residues 338–342) and the C-terminus of helix α9 (residues 383 and 384) of DRS and in the N-terminal subdomain of AIMP2GST, which comprises a β-sheet [Figs. 2 ▸(a) and 2 ▸(b)]. The interaction between AIMP2GST and EPRSGST is mainly through the heterodimerization of GST domains, in which the proteins are related to each other by a twofold rotational symmetry. In the following sections, the interfaces between the components of the DA2E subcomplex will be extensively described.

The DRS–AIMP2GST–EPRSGST (DA2E) subcomplex is orientated around the DRS homodimer

We successfully determined the tertiary complex structure of DA2E at 3.6 Å resolution by the molecular-replacement (MR) method using the structures of DRS and the AIMP2GST–EPRSGST binary complex (PDB entries 4j15 and 5a34; Cho et al., 2015 ▸; Kim et al., 2013 ▸) as phasing models. Despite our efforts to elucidate the structure of AIMP2 with the exon 1 region, only the GST domain of AIMP2 corresponding to exons 3 and 4 (residues 117–320) could be determined (Supplementary Fig. S2). Therefore, in the following we will use the term AIMP2GST instead of AIMP2-DX2-S34 to refer to the protein. The DA2E crystals belonged to space group P61 and contained two DA2E subcomplexes in the asymmetric unit, each comprising two molecules of DRS, AIMP2GST and EPRSGST.

The DA2E components are oriented around the central DRS homodimer in a twofold rotational symmetry with the center of mass of DA2E located in the DRS dimeric interface [Fig. 1 ▸(c)]. The DRS structure in the DA2E complex did not exhibit noticeable structural changes compared with that of DRS alone, in that the anticodon-binding domain (residues 57–146) and the catalytic domain (residues 189–497) of DRS could be well modeled. However, the hinge region between the anticodon-binding domain and the catalytic domain (residues 155–178), as well as two other loop regions (residues 225–248 and 273–281) that are known to bind tRNAAsp in an induced-fit manner (Sauter et al., 2000 ▸), could not be modeled owing to a lack of electron density [Supplementary Fig. S1(a)]. Extra electron density around the DA2E complex could not be clearly discerned owing to the low resolution limit and were manually assigned with discretion considering their environment. Extra spherical electron densities observed at the DRS dimeric interfaces were coordinated by two His204/Glu208 pairs, each from one monomer, and were modeled as Zn2+ ions according to the metal-content analysis results using ICP-MS (Supplementary Fig. S3). Extra electron densities observed at the tRNAAsp-binding sites of DRS molecules (catalytic sites and anticodon-binding sites) were modeled as phosphate ions present in the crystallization solution at high concentrations, which seem to mimic the phosphate moieties of the tRNAAsp backbone (Supplementary Fig. S4). Extra electron densities observed at the DRS–AIMP2GST interfaces were also modeled as phosphate ions which interact with backbone O and N atoms of the nearby DRS. Since the interactions do not seem to be biologically significant and the phosphate ions were only introduced for crystallization after in vitro DA2E complex formation, the phosphate ions might have contributed to further stabilizing the DA2E complexes in the crystal. Modeling water molecules using modeling programs did not produce consistent results owing to the low resolution limit. After a careful iterative process, only a dozen water molecules were modeled.

AIMP2GST and EPRSGST both adopt a GST fold consisting of an N-terminal Trx fold and a C-terminal α-helical sub­domain. EPRSGST adopts a canonical four-stranded β-sheet in its Trx subdomain, whereas AIMP2GST has an extra stretch of peptide (residues 148–170) which constitutes a strand–loop–helix motif (β2–α2) following helix α1, thus comprising a characteristic five-stranded β-sheet [Supplementary Fig. S1(b)]. Residues 171–179 and 289–291 of AIMP2GST in the α2–β3 loop and the α6–α7 loop, respectively, could not be modeled owing to a lack of electron density [Supplementary Fig. S1(b)].

Protein Interfaces, Surfaces and Assemblies (PISA) analysis (Krissinel & Henrick, 2007 ▸) of the interfaces in the DA2E subcomplex structure revealed that interactions between DRS and AIMP2GST arise mainly from residues in the α7–β9 loop (residues 338–342) and the C-terminus of helix α9 (residues 383 and 384) of DRS and in the N-terminal subdomain of AIMP2GST, which comprises a β-sheet [Figs. 2 ▸(a) and 2 ▸(b)]. The interaction between AIMP2GST and EPRSGST is mainly through the heterodimerization of GST domains, in which the proteins are related to each other by a twofold rotational symmetry. In the following sections, the interfaces between the components of the DA2E subcomplex will be extensively described.

3.3.Ser156 of AIMP2 plays a key role in the interaction between DRS and AIMP2GST via hydrogen bonds   The DA2E structure revealed that the binding interfaces of DRS and AIMP2GST have an average area of 747.2 Å2 as calculated by PISA. The AIMP2-binding motif of DRS is located in the middle of the DRS catalytic domain, discrete from the DRS dimeric interface, the tRNAAsp-binding site and the active site [Fig. 1 ▸(c)], which is in accordance with the fact that ARSs retain aminoacyl-tRNA synthetase activities in the MSC (Mirande et al., 1985 ▸).The complex formation of DRS and AIMP2GST is mainly mediated by two binding interfaces [Fig. 2 ▸(a)]. The major interface is formed by multiple hydrogen bonds between Lys338–Pro342 in the α7–β9 loop of DRS and His153–Ser156 in the β2–α2 loop of AIMP2GST [Fig. 2 ▸(b)], while the other interface involves helix α9 of DRS and the β-sheet of AIMP2GST [Fig. 2 ▸(c)]. The loops in the major interface establish multiple hydrogen bonds between the Cα backbones, which are reminiscent of the hydrogen-bonding networks of an antiparallel β-sheet. Interestingly, Ser156 of AIMP2GST seems to play a crucial role in mediating the interaction between DRS and AIMP2GST. The side chain of Ser156 acts as a hydrogen-bond donor coupled with the backbone amide group of Phe339 of DRS, while the backbone amide group of Ser156 forms a hydrogen bond to the backbone O atom of Phe339 of DRS [Fig. 2 ▸(b)]. Ser156 has previously been shown to be exposed to the solvent and phosphorylated by TGF-β signaling. Phosphorylation of Ser156 induced dissociation of AIMP2 from the MSC, followed by translocation into the cell nucleus, where AIMP2 functions as a tumor suppressor in concert with Smurf2 (Kim et al., 2016 ▸).Two mutants, AIMP2GST S156D and S156E, were generated to mimic the phosphorylation of Ser156, and the AIMP2GST S156A mutant was generated to abolish the side-chain-mediated hydrogen bond. The interaction between the AIMP2GST mutants and DRS was evaluated by size-exclusion chromatography [Fig. 2 ▸(d)]. All three AIMP2GST mutants could not form binary complexes with DRS, which suggests that the hydrogen bond from the side chain of AIMP2 Ser156 is crucial for the interaction with DRS and that Ser156 phosphorylation would disintegrate the MSC assembly.

Ser156 of AIMP2 plays a key role in the interaction between DRS and AIMP2GST via hydrogen bonds

The DA2E structure revealed that the binding interfaces of DRS and AIMP2GST have an average area of 747.2 Å2 as calculated by PISA. The AIMP2-binding motif of DRS is located in the middle of the DRS catalytic domain, discrete from the DRS dimeric interface, the tRNAAsp-binding site and the active site [Fig. 1 ▸(c)], which is in accordance with the fact that ARSs retain aminoacyl-tRNA synthetase activities in the MSC (Mirande et al., 1985 ▸).

The complex formation of DRS and AIMP2GST is mainly mediated by two binding interfaces [Fig. 2 ▸(a)]. The major interface is formed by multiple hydrogen bonds between Lys338–Pro342 in the α7–β9 loop of DRS and His153–Ser156 in the β2–α2 loop of AIMP2GST [Fig. 2 ▸(b)], while the other interface involves helix α9 of DRS and the β-sheet of AIMP2GST [Fig. 2 ▸(c)]. The loops in the major interface establish multiple hydrogen bonds between the Cα backbones, which are reminiscent of the hydrogen-bonding networks of an antiparallel β-sheet. Interestingly, Ser156 of AIMP2GST seems to play a crucial role in mediating the interaction between DRS and AIMP2GST. The side chain of Ser156 acts as a hydrogen-bond donor coupled with the backbone amide group of Phe339 of DRS, while the backbone amide group of Ser156 forms a hydrogen bond to the backbone O atom of Phe339 of DRS [Fig. 2 ▸(b)]. Ser156 has previously been shown to be exposed to the solvent and phosphorylated by TGF-β signaling. Phosphorylation of Ser156 induced dissociation of AIMP2 from the MSC, followed by translocation into the cell nucleus, where AIMP2 functions as a tumor suppressor in concert with Smurf2 (Kim et al., 2016 ▸).

Two mutants, AIMP2GST S156D and S156E, were generated to mimic the phosphorylation of Ser156, and the AIMP2GST S156A mutant was generated to abolish the side-chain-mediated hydrogen bond. The interaction between the AIMP2GST mutants and DRS was evaluated by size-exclusion chromatography [Fig. 2 ▸(d)]. All three AIMP2GST mutants could not form binary complexes with DRS, which suggests that the hydrogen bond from the side chain of AIMP2 Ser156 is crucial for the interaction with DRS and that Ser156 phosphorylation would disintegrate the MSC assembly.

3.4.The DA2E ternary complex accompanies a conformational change in interface 1 between two GST domains   
PISA analysis of the DA2E structure revealed that the binding interfaces of AIMP2GST and EPRSGST have an average area of 1083.3 Å2, which is larger than that between DRS and AIMP2GST. The networks of interactions between heterodimeric GST domains in the MSC have been extensively analyzed previously, and were assigned as Interface 1 and Interface 2 (Cho et al., 2015 ▸). Here, we briefly describe the nature of Interface 1 between AIMP2GST and EPRSGST and intensively analyze a unique conformational change therein elicited by the incorporation of DRS compared with the heterodimeric A2E structure.Interface 1 is formed by the bundling of helices α3 and α4 of AIMP2GST and α2 and α3 of EPRSGST, which establish multiple intermolecular hydrogen bonds and salt bridges. When the structures of AIMP2GST in the DA2E subcomplex and in the heterodimeric A2E subcomplex (PDB entry 5a34; Cho et al., 2015 ▸) were compared, the two structures were very similar over 201 equivalent Cα atoms, with a root-mean-square deviation (r.m.s.d.) of 0.87 Å. Although the constructs used for this comparison were truncated differently, the GST-domain structure was not affected by the truncations. However, a remarkably high r.m.s.d. was observed for residues Phe199–Pro206, with a maximum Cα deviation of 8.78 Å at Thr203 [Fig. 2 ▸(e)]. These residues belong to the β4–β5 loop, the last three residues of which (Met204-Cys205-Pro206) make hydrophobic contacts with helix α4 of EPRSGST [Fig. 2 ▸(c)]. Incorporation of DRS causes a major conformational change in the β4–β5 loop, where the C-terminal end of helix α9 of DRS rests on the β-sheet of AIMP2GST [Fig. 2 ▸(c), dashed red circle]. Superposing the AIMP2 structure from the A2E subcomplex [Fig. 2 ▸(c), white] (PDB entry 5a34) onto our AIMP2GST structure [Fig. 2 ▸(c), orange] reveals that helix α9 of DRS occupies the region where the β4–β5 loop would be in the A2E subcomplex structure. The results of PISA analysis show that residues Glu382-Lys383-Tyr384 on helix α9 of DRS make mainly hydrophobic contacts with Leu119, Val123 and Ile201 of AIMP2GST, while the β4–β5 loop of AIMP2GST is sandwiched between DRS and EPRSGST to adopt a more ordered β-turn structure [Fig. 2 ▸(c), orange]. Consequent rearrangement of the Cα backbone in the β4–β5 loop elicited a remarkable difference in Interface 1 compared with that in the heterodimeric A2E complex.

The DA2E ternary complex accompanies a conformational change in interface 1 between two GST domains

PISA analysis of the DA2E structure revealed that the binding interfaces of AIMP2GST and EPRSGST have an average area of 1083.3 Å2, which is larger than that between DRS and AIMP2GST. The networks of interactions between heterodimeric GST domains in the MSC have been extensively analyzed previously, and were assigned as Interface 1 and Interface 2 (Cho et al., 2015 ▸). Here, we briefly describe the nature of Interface 1 between AIMP2GST and EPRSGST and intensively analyze a unique conformational change therein elicited by the incorporation of DRS compared with the heterodimeric A2E structure.

Interface 1 is formed by the bundling of helices α3 and α4 of AIMP2GST and α2 and α3 of EPRSGST, which establish multiple intermolecular hydrogen bonds and salt bridges. When the structures of AIMP2GST in the DA2E subcomplex and in the heterodimeric A2E subcomplex (PDB entry 5a34; Cho et al., 2015 ▸) were compared, the two structures were very similar over 201 equivalent Cα atoms, with a root-mean-square deviation (r.m.s.d.) of 0.87 Å. Although the constructs used for this comparison were truncated differently, the GST-domain structure was not affected by the truncations. However, a remarkably high r.m.s.d. was observed for residues Phe199–Pro206, with a maximum Cα deviation of 8.78 Å at Thr203 [Fig. 2 ▸(e)]. These residues belong to the β4–β5 loop, the last three residues of which (Met204-Cys205-Pro206) make hydrophobic contacts with helix α4 of EPRSGST [Fig. 2 ▸(c)]. Incorporation of DRS causes a major conformational change in the β4–β5 loop, where the C-terminal end of helix α9 of DRS rests on the β-sheet of AIMP2GST [Fig. 2 ▸(c), dashed red circle]. Superposing the AIMP2 structure from the A2E subcomplex [Fig. 2 ▸(c), white] (PDB entry 5a34) onto our AIMP2GST structure [Fig. 2 ▸(c), orange] reveals that helix α9 of DRS occupies the region where the β4–β5 loop would be in the A2E subcomplex structure. The results of PISA analysis show that residues Glu382-Lys383-Tyr384 on helix α9 of DRS make mainly hydrophobic contacts with Leu119, Val123 and Ile201 of AIMP2GST, while the β4–β5 loop of AIMP2GST is sandwiched between DRS and EPRSGST to adopt a more ordered β-turn structure [Fig. 2 ▸(c), orange]. Consequent rearrangement of the Cα backbone in the β4–β5 loop elicited a remarkable difference in Interface 1 compared with that in the heterodimeric A2E complex.

3.5.Adjacent DA2E subcomplexes are bridged via Interface 2 between two EPRSGST molecules   To our surprise, we observed two molecules of EPRSGST interacting via Interface 2, each of which are from adjacent DA2E subcomplexes in the asymmetric unit [Figs. 3 ▸(a) and 3 ▸(b)]. In the case of Interface 2 of EPRSGST and AIMP3 (PDB entry 4bvx; Cho et al., 2015 ▸), residues from helices α7 and the α4–α5 loops from each of the GST domains similarly establish a complex network via hydrogen bonds and salt bridges [Fig. 3 ▸(c)] to the EPRSGST crystallographic interface. Because EPRSGST and AIMP3 have a high overall sequence similarity of 40.1% [Fig. 3 ▸(d)] and share the same structural domain [Figs. 3 ▸(b) and 3 ▸(c)], it is understandable that the exposed surface of EPRSGST would facilitate dimerization via Interface 2 in the absence of the legitimate binding partner AIMP3.During our trial crystallization of MSC subcomplexes, we obtained crystals of the binary DA2 subcomplex, which however only diffracted to 4.5 Å resolution (data not shown), compared with crystals of the ternary DA2E subcomplex, which diffracted to 3.6 Å resolution. While DA2 subcomplex molecules were packed with no apparent noncrystallographic symmetry within the asymmetric unit, the DA2E subcomplex molecules in the asymmetric unit are related to each other by a twofold rotational symmetry [Fig. 3 ▸(a)]. Although the interaction between two EPRSGST molecules from adjacent DA2E subcomplexes within the asymmetric unit would have been a crystallographic artifact, it is clear that this interaction contributed to the stabilization and hence successful structure determination of the DA2E subcomplex.

Adjacent DA2E subcomplexes are bridged via Interface 2 between two EPRSGST molecules

To our surprise, we observed two molecules of EPRSGST interacting via Interface 2, each of which are from adjacent DA2E subcomplexes in the asymmetric unit [Figs. 3 ▸(a) and 3 ▸(b)]. In the case of Interface 2 of EPRSGST and AIMP3 (PDB entry 4bvx; Cho et al., 2015 ▸), residues from helices α7 and the α4–α5 loops from each of the GST domains similarly establish a complex network via hydrogen bonds and salt bridges [Fig. 3 ▸(c)] to the EPRSGST crystallographic interface. Because EPRSGST and AIMP3 have a high overall sequence similarity of 40.1% [Fig. 3 ▸(d)] and share the same structural domain [Figs. 3 ▸(b) and 3 ▸(c)], it is understandable that the exposed surface of EPRSGST would facilitate dimerization via Interface 2 in the absence of the legitimate binding partner AIMP3.

During our trial crystallization of MSC subcomplexes, we obtained crystals of the binary DA2 subcomplex, which however only diffracted to 4.5 Å resolution (data not shown), compared with crystals of the ternary DA2E subcomplex, which diffracted to 3.6 Å resolution. While DA2 subcomplex molecules were packed with no apparent noncrystallographic symmetry within the asymmetric unit, the DA2E subcomplex molecules in the asymmetric unit are related to each other by a twofold rotational symmetry [Fig. 3 ▸(a)]. Although the interaction between two EPRSGST molecules from adjacent DA2E subcomplexes within the asymmetric unit would have been a crystallographic artifact, it is clear that this interaction contributed to the stabilization and hence successful structure determination of the DA2E subcomplex.

4.Discussion   The MSC is a multi-protein complex composed of nine ARSs and three AIMPs. It has been proposed that the congregation of individual ARSs and scaffold proteins would facilitate the translation process by providing a constant flow of amino-acid-charged tRNAs (Negrutskii & Deutscher, 1991 ▸). In this work, we report a pivotal architecture of the MSC comprising two AIMP2GST–EPRSGST heterodimers which extend from the central DRS homodimer. While searching for an optimum construct of AIMP2 for DA2E structure determination, we made the serendipitous discovery of AIMP2-DX2 lacking the first 33 residues (AIMP2-DX2-S34), the incorporation of which into the MSC was known to be biologically significant (Park et al., 2015 ▸). This construct seemed to have greatly contributed to the structure determination of DA2E because the flexible N-terminal region of AIMP2 was removed, which could facilitate the crystallization of the DA2E ternary complex.Following the structure determination of DA2E, the inter­actions among the DA2E components were intensively inspected. The interaction between DRS and AIMP2GST is uniquely mediated by the β-sheet of AIMP2GST and its adjoining loops, while the interaction between AIMP2GST and EPRSGST is mediated via the heterodimeric GST-domain Interface 1, as previously reported (Cho et al., 2015 ▸). By analyzing the interface between DRS and AIMP2GST, we speculated that Ser156 of AIMP2GST is a key residue in complex formation and demonstrated this by size-exclusion chromatography with relevant AIMP2GST mutants. To our surprise, both AIMP2GST phosphor-ablative (AIMP2GST S156A) and phosphor-mimetic (AIMP2GST S156D and S156E) mutants successfully inhibited the interaction between DRS and AIMP2GST [Fig. 2 ▸(d)]. Phosphorylation of Ser156 is known to trigger the release of AIMP2 from the MSC and subsequent translocation into the nucleus for interaction with Smurf2 to suppress the nuclear export of Smurf2 and hence elicit tumor-suppressive TGF-β signaling (Kim et al., 2016 ▸). Since AIMP2 is a key scaffold protein that interconnects most components (Robinson et al., 2000 ▸), the release of AIMP2 from the MSC would cause breakdown of the integral MSC, which subsequently enables nontranslational functions of the dissociated components (Kim et al., 2002 ▸). For example, KRS released from the MSC upon phosphorylation of its Ser206 residue is translocated from the cytoplasm into the nucleus to activate the downstream signaling pathway for diadenosine tetraphosphate (Ap4A) production (Yannay-Cohen et al., 2009 ▸). Similar phenomena are observed for EPRS following interferon-γ activation (Sampath et al., 2004 ▸), suggesting that the release of MSC components by post-translational modification is a measure of switching cellular metabolism from growth to damage control in response to a harsh environment (Lee et al., 2004 ▸).Although the mechanistic detail of AIMP2 release from the MSC is not clear, DRS would also be dissociated from the MSC along with AIMP2 as it is the only binding partner. The release of DRS might allow subsequent post-translational modifications such as acetylation, phosphorylation or ubiquitination (Kim et al., 2013 ▸), which might in turn trigger as yet unknown nontranslational functions of DRS. Overall, post-translational modification and subsequent repositioning of the components of MSC could be a common method of eliciting nontranslational functions.Previous interactome studies have shown that AIMP2 acts as a scaffold in the assembly of the MSC to harbor binding sites for other components (Quevillon et al., 1999 ▸; Robinson et al., 2000 ▸). KRS (Ofir-Birin et al., 2013 ▸), AIMP1 (Ahn et al., 2003 ▸), DRS and EPRSGST described here are all known binding partners of AIMP2 [Fig. 4 ▸(a)]. Using previously reported information about the interactions among the MSC components, we built a model of the MSC subcomplex based on our DA2E structure. Firstly, heterotetrameric GST domains could be reasonably appended to the DA2E structure by sequentially superimposing structures of EA3 (EPRSGST–AIMP3; PDB entry 5bmu) and A3M (AIMP3–MRSGST; PDB entry 4bvy) (Cho et al., 2015 ▸), resulting in the DA2EA3M pentameric subcomplex model [Fig. 4 ▸(b)]. The conformation of the DA2EA3M subcomplex model did not hinder the binding of tRNAAsp, the binding site of which was speculated from the yeast DRS–tRNAAsp complex (Ruff et al., 1991 ▸). For a more comprehensive model of DA2EA3M, canonical domains of ERS, PRS and MRS (ERSCan, PRSCan and MRSCan, respectively) should be placed.Although the structures of the canonical domains are known (Son et al., 2013 ▸), no structural information is available for the linkers between the canonical domains and their N-terminally appending GST domains. However, considering the short lengths of the peptides connecting EPRSGST to ERSCan and MRSGST to MRSCan (approximately 40 amino acids each), we could at least surmise that these canonical domains would reside next to their respective GST domains, as denoted by the blue and red ovals, respectively, in Fig. 4 ▸(c). Assigning the position of the PRSCan homodimer is more precarious. There is no structural information about the 300-amino-acid linker region between ERSCan and PRSCan, except for that on the three tandem WHEP domains elucidated by nuclear magnetic resonance (NMR) spectroscopy (Jeong et al., 2000 ▸). For our model of the DA2EA3M structure, this linker region is illustrated with dashed lines to reflect its high flexibility. The homodimeric PRSCan was positioned above the DRS homodimer, as shown in Fig. 4 ▸(c), in such a way that a theoretical twofold rotational symmetry axis of the DRS homodimer coincides with that of the PRSCan homodimer.In contrast, AIMP1 tethers a stable trimeric arginyl-tRNA (RQA1) subcomplex (Fu et al., 2014 ▸) to DA2EA3M via a coiled-coil interaction with the leucine-zipper motif of AIMP2 (residues 55–76; Quevillon et al., 1999 ▸), as shown in Fig. 4 ▸(c). Two KRS homodimers were additionally included in the MSC subcomplex model by estimating the relative position of the N-terminal region of AIMP2 (Gly2–Pro33), which was present in the structure of the KRS homodimer, to the leucine-zipper motif.However, assembling the components of the MSC onto an AIMP2 molecule according to the above interaction data would only result in one half of the whole MSC complex because most of the components of the MSC, including AIMP2 itself, exist as duplicates (Dias et al., 2013 ▸). Thus far, how two AIMP2-centered subcomplexes are brought together to compose the complete MSC has remained elusive. Our DA2E subcomplex structure with a central DRS homodimer provides evidence that DRS plays a pivotal role in the assembly of the MSC by conjoining two AIMP2 molecules as well as additional components. Taken together, our DA2E subcomplex structure provides valuable new structural information about the DRS-centered DA2E subcomplex and further extends our knowledge on the overall assembly of the MSC to thoroughly elucidate the structure, dynamics and synergistic effects of the MSC.

The MSC is a multi-protein complex composed of nine ARSs and three AIMPs. It has been proposed that the congregation of individual ARSs and scaffold proteins would facilitate the translation process by providing a constant flow of amino-acid-charged tRNAs (Negrutskii & Deutscher, 1991 ▸). In this work, we report a pivotal architecture of the MSC comprising two AIMP2GST–EPRSGST heterodimers which extend from the central DRS homodimer. While searching for an optimum construct of AIMP2 for DA2E structure determination, we made the serendipitous discovery of AIMP2-DX2 lacking the first 33 residues (AIMP2-DX2-S34), the incorporation of which into the MSC was known to be biologically significant (Park et al., 2015 ▸). This construct seemed to have greatly contributed to the structure determination of DA2E because the flexible N-terminal region of AIMP2 was removed, which could facilitate the crystallization of the DA2E ternary complex.

Following the structure determination of DA2E, the inter­actions among the DA2E components were intensively inspected. The interaction between DRS and AIMP2GST is uniquely mediated by the β-sheet of AIMP2GST and its adjoining loops, while the interaction between AIMP2GST and EPRSGST is mediated via the heterodimeric GST-domain Interface 1, as previously reported (Cho et al., 2015 ▸). By analyzing the interface between DRS and AIMP2GST, we speculated that Ser156 of AIMP2GST is a key residue in complex formation and demonstrated this by size-exclusion chromatography with relevant AIMP2GST mutants. To our surprise, both AIMP2GST phosphor-ablative (AIMP2GST S156A) and phosphor-mimetic (AIMP2GST S156D and S156E) mutants successfully inhibited the interaction between DRS and AIMP2GST [Fig. 2 ▸(d)]. Phosphorylation of Ser156 is known to trigger the release of AIMP2 from the MSC and subsequent translocation into the nucleus for interaction with Smurf2 to suppress the nuclear export of Smurf2 and hence elicit tumor-suppressive TGF-β signaling (Kim et al., 2016 ▸). Since AIMP2 is a key scaffold protein that interconnects most components (Robinson et al., 2000 ▸), the release of AIMP2 from the MSC would cause breakdown of the integral MSC, which subsequently enables nontranslational functions of the dissociated components (Kim et al., 2002 ▸). For example, KRS released from the MSC upon phosphorylation of its Ser206 residue is translocated from the cytoplasm into the nucleus to activate the downstream signaling pathway for diadenosine tetraphosphate (Ap4A) production (Yannay-Cohen et al., 2009 ▸). Similar phenomena are observed for EPRS following interferon-γ activation (Sampath et al., 2004 ▸), suggesting that the release of MSC components by post-translational modification is a measure of switching cellular metabolism from growth to damage control in response to a harsh environment (Lee et al., 2004 ▸).

Although the mechanistic detail of AIMP2 release from the MSC is not clear, DRS would also be dissociated from the MSC along with AIMP2 as it is the only binding partner. The release of DRS might allow subsequent post-translational modifications such as acetylation, phosphorylation or ubiquitination (Kim et al., 2013 ▸), which might in turn trigger as yet unknown nontranslational functions of DRS. Overall, post-translational modification and subsequent repositioning of the components of MSC could be a common method of eliciting nontranslational functions.

Previous interactome studies have shown that AIMP2 acts as a scaffold in the assembly of the MSC to harbor binding sites for other components (Quevillon et al., 1999 ▸; Robinson et al., 2000 ▸). KRS (Ofir-Birin et al., 2013 ▸), AIMP1 (Ahn et al., 2003 ▸), DRS and EPRSGST described here are all known binding partners of AIMP2 [Fig. 4 ▸(a)]. Using previously reported information about the interactions among the MSC components, we built a model of the MSC subcomplex based on our DA2E structure. Firstly, heterotetrameric GST domains could be reasonably appended to the DA2E structure by sequentially superimposing structures of EA3 (EPRSGST–AIMP3; PDB entry 5bmu) and A3M (AIMP3–MRSGST; PDB entry 4bvy) (Cho et al., 2015 ▸), resulting in the DA2EA3M pentameric subcomplex model [Fig. 4 ▸(b)]. The conformation of the DA2EA3M subcomplex model did not hinder the binding of tRNAAsp, the binding site of which was speculated from the yeast DRS–tRNAAsp complex (Ruff et al., 1991 ▸). For a more comprehensive model of DA2EA3M, canonical domains of ERS, PRS and MRS (ERSCan, PRSCan and MRSCan, respectively) should be placed.

Although the structures of the canonical domains are known (Son et al., 2013 ▸), no structural information is available for the linkers between the canonical domains and their N-terminally appending GST domains. However, considering the short lengths of the peptides connecting EPRSGST to ERSCan and MRSGST to MRSCan (approximately 40 amino acids each), we could at least surmise that these canonical domains would reside next to their respective GST domains, as denoted by the blue and red ovals, respectively, in Fig. 4 ▸(c). Assigning the position of the PRSCan homodimer is more precarious. There is no structural information about the 300-amino-acid linker region between ERSCan and PRSCan, except for that on the three tandem WHEP domains elucidated by nuclear magnetic resonance (NMR) spectroscopy (Jeong et al., 2000 ▸). For our model of the DA2EA3M structure, this linker region is illustrated with dashed lines to reflect its high flexibility. The homodimeric PRSCan was positioned above the DRS homodimer, as shown in Fig. 4 ▸(c), in such a way that a theoretical twofold rotational symmetry axis of the DRS homodimer coincides with that of the PRSCan homodimer.

In contrast, AIMP1 tethers a stable trimeric arginyl-tRNA (RQA1) subcomplex (Fu et al., 2014 ▸) to DA2EA3M via a coiled-coil interaction with the leucine-zipper motif of AIMP2 (residues 55–76; Quevillon et al., 1999 ▸), as shown in Fig. 4 ▸(c). Two KRS homodimers were additionally included in the MSC subcomplex model by estimating the relative position of the N-terminal region of AIMP2 (Gly2–Pro33), which was present in the structure of the KRS homodimer, to the leucine-zipper motif.

However, assembling the components of the MSC onto an AIMP2 molecule according to the above interaction data would only result in one half of the whole MSC complex because most of the components of the MSC, including AIMP2 itself, exist as duplicates (Dias et al., 2013 ▸). Thus far, how two AIMP2-centered subcomplexes are brought together to compose the complete MSC has remained elusive. Our DA2E subcomplex structure with a central DRS homodimer provides evidence that DRS plays a pivotal role in the assembly of the MSC by conjoining two AIMP2 molecules as well as additional components. Taken together, our DA2E subcomplex structure provides valuable new structural information about the DRS-centered DA2E subcomplex and further extends our knowledge on the overall assembly of the MSC to thoroughly elucidate the structure, dynamics and synergistic effects of the MSC.

Supplementary MaterialPDB reference: human cytosolic aspartyl-tRNA synthetase in complex with glutathione S-transferase domains from aminoacyl-tRNA synthase complex-interacting multifunctional protein 2 and glutamyl-prolyl-tRNA synthetase, 6iy6
Supplementary Figures. DOI: 10.1107/S2052252519010790/lz5028sup1.pdf

Supplementary Material

PDB reference: human cytosolic aspartyl-tRNA synthetase in complex with glutathione S-transferase domains from aminoacyl-tRNA synthase complex-interacting multifunctional protein 2 and glutamyl-prolyl-tRNA synthetase, 6iy6

Supplementary Figures. DOI: 10.1107/S2052252519010790/lz5028sup1.pdf
