---
pmid: '19158363'
title: High-resolution DNA-binding specificity analysis of yeast transcription factors.
authors:
- Zhu C
- Byers KJ
- McCord RP
- Shi Z
- Berger MF
- Newburger DE
- Saulrieta K
- Smith Z
- Shah MV
- Radhakrishnan M
- Philippakis AA
- Hu Y
- De Masi F
- Pacek M
- Rolfs A
- Murthy T
- Labaer J
- Bulyk ML
journal: Genome Res
year: '2009'
full_text_available: false
pmcid: PMC2665775
doi: 10.1101/gr.090233.108
---

# High-resolution DNA-binding specificity analysis of yeast transcription factors.
**Authors:** Zhu C, Byers KJ, McCord RP, Shi Z, Berger MF, Newburger DE, Saulrieta K, Smith Z, Shah MV, Radhakrishnan M, Philippakis AA, Hu Y, De Masi F, Pacek M, Rolfs A, Murthy T, Labaer J, Bulyk ML
**Journal:** Genome Res (2009)
**DOI:** [10.1101/gr.090233.108](https://doi.org/10.1101/gr.090233.108)
**PMC:** [PMC2665775](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2665775/)

## Abstract

1. Genome Res. 2009 Apr;19(4):556-66. doi: 10.1101/gr.090233.108. Epub 2009 Jan
21.

High-resolution DNA-binding specificity analysis of yeast transcription factors.

Zhu C(1), Byers KJ, McCord RP, Shi Z, Berger MF, Newburger DE, Saulrieta K, 
Smith Z, Shah MV, Radhakrishnan M, Philippakis AA, Hu Y, De Masi F, Pacek M, 
Rolfs A, Murthy T, Labaer J, Bulyk ML.

Author information:
(1)Division of Genetics, Department of Medicine, Brigham and Women's Hospital 
and Harvard Medical School, Boston, Massachusetts 02115, USA.

Transcription factors (TFs) regulate the expression of genes through 
sequence-specific interactions with DNA-binding sites. However, despite recent 
progress in identifying in vivo TF binding sites by microarray readout of 
chromatin immunoprecipitation (ChIP-chip), nearly half of all known yeast TFs 
are of unknown DNA-binding specificities, and many additional predicted TFs 
remain uncharacterized. To address these gaps in our knowledge of yeast TFs and 
their cis regulatory sequences, we have determined high-resolution binding 
profiles for 89 known and predicted yeast TFs, over more than 2.3 million gapped 
and ungapped 8-bp sequences ("k-mers"). We report 50 new or significantly 
different direct DNA-binding site motifs for yeast DNA-binding proteins and 
motifs for eight proteins for which only a consensus sequence was previously 
known; in total, this corresponds to over a 50% increase in the number of yeast 
DNA-binding proteins with experimentally determined DNA-binding specificities. 
Among other novel regulators, we discovered proteins that bind the PAC 
(Polymerase A and C) motif (GATGAG) and regulate ribosomal RNA (rRNA) 
transcription and processing, core cellular processes that are constituent to 
ribosome biogenesis. In contrast to earlier data types, these comprehensive 
k-mer binding data permit us to consider the regulatory potential of genomic 
sequence at the individual word level. These k-mer data allowed us to reannotate 
in vivo TF binding targets as direct or indirect and to examine TFs' potential 
effects on gene expression in approximately 1,700 environmental and cellular 
conditions. These approaches could be adapted to identify TFs and cis regulatory 
elements in higher eukaryotes.

DOI: 10.1101/gr.090233.108
PMCID: PMC2665775
PMID: 19158363 [Indexed for MEDLINE]
