---
pmid: '12006496'
title: PIF4, a phytochrome-interacting bHLH factor, functions as a negative regulator
  of phytochrome B signaling in Arabidopsis.
authors:
- Huq E
- Quail PH
journal: EMBO J
year: '2002'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC126004
doi: 10.1093/emboj/21.10.2441
---

# PIF4, a phytochrome-interacting bHLH factor, functions as a negative regulator of phytochrome B signaling in Arabidopsis.
**Authors:** Huq E, Quail PH
**Journal:** EMBO J (2002)
**DOI:** [10.1093/emboj/21.10.2441](https://doi.org/10.1093/emboj/21.10.2441)
**PMC:** [PMC126004](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC126004/)

## Abstract

1. EMBO J. 2002 May 15;21(10):2441-50. doi: 10.1093/emboj/21.10.2441.

PIF4, a phytochrome-interacting bHLH factor, functions as a negative regulator 
of phytochrome B signaling in Arabidopsis.

Huq E(1), Quail PH.

Author information:
(1)Department of Plant and Microbial Biology, University of California, Berkeley 
94720, USA.

Plants sense and respond to red and far-red light using the phytochrome (phy) 
family of photoreceptors. However, the mechanism of light signal transduction is 
not well defined. Here, we report the identification of a new mutant Arabidopsis 
locus, srl2 (short under red-light 2), which confers selective hypersensitivity 
to continuous red, but not far-red, light. This hypersensitivity is eliminated 
in srl2phyB, but not srl2phyA, double mutants, indicating that this locus 
functions selectively and negatively in phyB signaling. The SRL2 gene encodes a 
bHLH factor, designated PIF4 (phytochrome-interacting factor 4), which binds 
selectively to the biologically active Pfr form of phyB, but has little affinity 
for phyA. Despite its hypersensitive morphological phenotype, the srl2 mutant 
displays no perturbation of light-induced expression of marker genes for 
chloroplast development. These data suggest that PIF4 may function specifically 
in a branch of the phyB signaling network that regulates a subset of genes 
involved in cell expansion. Consistent with this proposal, PIF4 localizes to the 
nucleus and can bind to a G-box DNA sequence motif found in various 
light-regulated promoters.

DOI: 10.1093/emboj/21.10.2441
PMCID: PMC126004
PMID: 12006496 [Indexed for MEDLINE]

## Full Text

Abstract

Plants sense and respond to red and far-red light using the phytochrome (phy) family of photoreceptors. However, the mechanism of light signal transduction is not well defined. Here, we report the identification of a new mutant Arabidopsis locus, srl2 ( s hort under r ed- l ight 2 ), which confers selective hypersensitivity to continuous red, but not far-red, light. This hypersensitivity is eliminated in srl2phyB , but not srl2phyA , double mutants, indicating that this locus functions selectively and negatively in phyB signaling. The SRL2 gene encodes a bHLH factor, designated PIF4 ( p hytochrome- i nteracting f actor 4 ), which binds selectively to the biologically active Pfr form of phyB, but has little affinity for phyA. Despite its hypersensitive morphological phenotype, the srl2 mutant displays no perturbation of light-induced expression of marker genes for chloroplast development. These data suggest that PIF4 may function specifically in a branch of the phyB signaling network that regulates a subset of genes involved in cell expansion. Consistent with this proposal, PIF4 localizes to the nucleus and can bind to a G-box DNA sequence motif found in various light-regulated promoters.

Introduction

Being sessile organisms, plants have developed an enormous plasticity to adapt to the constantly changing environment. To keep track of time and spatial position, and to optimize growth and development throughout their life cycle, plants have at least three types of informational photoreceptors: an unknown UV-B receptor, UV-A/blue-light receptors, and phytochromes (phys), which sense the red (R)/far-red (FR) region of the spectrum ( Kendrick and Kronenberg, 1994 ; Smith, 2000 ; Sakai et al., 2001 ). Phys are soluble chromoproteins of ∼125 kDa encoded by a small gene family ( PHYA – PHYE ) in Arabidopsis ( Mathews and Sharrock, 1997 ). Phys can exist in two spectral forms. When synthesized in the inactive Pr form, absorption of red light converts the molecule to its biologically active Pfr form, which can be converted back to the Pr form upon absorption of far-red light. This reversible switching between the two forms is critical for its photosensory and biological function ( Quail, 1997 , 2002 ; Smith, 2000 ).

A phy molecule consists of two domains: an N-terminal globular domain, which anchors the bilin chromophore and is responsible for photosensory function; and a C-terminal domain, which is thought to be responsible for signal transfer ( Quail, 1997 ). Rapid progress has been made in recent years in identifying phy signaling intermediates, as well as in characterizing the biochemical function of phys ( Deng and Quail, 1999 ; Hudson, 2000 ; Fankhauser, 2001 ; Quail, 2002 ). However, definition of the primary biochemical or molecular mechanism linking the absorption of light to activation of gene expression is still lacking.

A number of approaches have been used to investigate phy signal transduction. Biochemical approaches have provided evidence that recombinant oat phyA possesses serine/threonine protein kinase activity ( Yeh and Lagarias, 1998 ). Pharmacological and microinjection approaches have implicated G-proteins, cGMP and Ca 2+ /calmodulin in phy signaling ( Bowler et al., 1994 ). Genetic approaches have identified a number of mutants that are either chromophore-biosynthetic or photoreceptor mutants, or signaling mutants ( Hudson, 2000 ). Analyses of photoreceptor mutants, especially phyA and phyB , suggest that phyA is solely responsible for sensing continuous far-red light, and phyB is the major photoreceptor for red light in seedling de-etiolation ( Quail et al., 1995 ). Analyses of early signaling mutants indicate that they are either specific to each phy or responsive to both phys ( Hudson, 2000 ). In addition, a large group of pleiotropic mutants ( cop / det / fus group) have been characterized whose relevant loci encode proteins that function as general repressors of phy signaling as well as other pathways ( Schwechheimer and Deng, 2000 ). These results suggest that each photoreceptor may have distinct genetically separable pathways, which might converge later in the signal transduction cascade ( Deng and Quail, 1999 ; Hudson, 2000 ; Fankhauser, 2001 ; Quail, 2002 ). A number of genetic loci have been cloned recently and most of them are nuclear proteins ( Fankhauser, 2001 ). In addition, phys are also translocated into the nucleus upon light activation ( Nagy et al., 2000 ), indicating that a major phy signal transduction event occurs in the nucleus.

Yeast two-hybrid screening coupled with reverse genetic approaches have identified a small group of apparently functionally unrelated proteins that are involved in phy signaling ( Quail, 2000 ). In addition, targeted yeast two-hybrid assays showed that CRY1 can interact with phyA ( Ahmad et al., 1998 ), and in vivo FRET assays showed interaction between CRY2 and phyB ( Mas et al., 2000 ), suggesting cross-talk between phy and cryptochrome signaling. The C-terminal domain of phyB has also been shown to interact with ZTL/ADO1 and COP1 ( Jarillo et al., 2001 ; Yang et al., 2001 ). Among the phy interacting factors, phytochrome kinase substrate 1 (PKS1), a cytosolic protein, interacts with the C-terminal domain of both phyA and phyB, and might be involved in retention of phy in the cytosol ( Fankhauser et al., 1999 ). NDPK2 is also a cytosolic enzyme, which interacts with phyA ( Choi et al., 1999 ). ELF3 interacts with phyB ( Liu et al., 2001 ), and is involved in the circadian clock ( Cavington et al., 2001 ). PIF3 is a bHLH protein that interacts with the biologically active Pfr form of both phyA and phyB ( Ni et al., 1998 , 1999 ; Zhu et al., 2000 ). In addition, G-box promoter element-bound PIF3 can interact with the Pfr form of phyB, suggesting a direct pathway for controlling gene expression by phys in response to light ( Martinez-Garcia et al., 2000 ).

Although a number of mutants are available that are selectively defective in photoresponsiveness to red light ( Hudson, 2000 ; Fankhauser, 2001 ), the phyB signaling pathway does not seem to have been screened to saturation. Previously characterized components in this pathway include both positively and negatively acting factors ( Fankhauser, 2001 ). Three of the positively acting loci, POC1 / PIF3 , GI and ELF3 , have been molecularly cloned ( Ni et al., 1998 ; Fowler et al., 1999 ; Halliday et al., 1999 ; Park et al., 1999 ; Huq et al., 2000b ; Hicks et al., 2001 ; Liu et al., 2001 ), and all three are nuclear proteins. In addition, two negatively acting components, PKS1 ( Fankhauser et al., 1999 ) and ATHB2 ( Steindler et al., 1999 ), have been molecularly characterized. PKS1 is a cytosolic protein that interacts with both phyA and phyB ( Fankhauser et al., 1999 ), and ATHB2 is involved in shade avoidance. However, thus far, no mutants have been reported for these or any other molecularly characterized, negatively acting component in the phyB pathway. Here, we have sought to identify early signaling components specific to the phyB-mediated de-etiolation process. Using a genetic approach, we previously isolated mutants that are affected in the phyB signaling pathway and characterized two of them, srl1 and gi-100 ( Huq et al., 2000a , b ). A third mutant isolated from this screen, srl2 ( s hort under r ed- l ight 2 ), is described here. We have cloned the SRL2 gene, and show that SRL2 encodes a protein designated PIF4 ( p hytochrome- i nteracting f actor 4 ), which acts as a negative regulator specific to the phyB signaling pathway.

Discussion

The data presented here provide genetic, photobiological and molecular evidence that PIF4 functions as a nuclear localized, negatively acting component potentially specific to the phyB signaling pathway. Compared with the wild type, both srl2 -mutant and PIF4 -antisense lines display short hypocotyls, whereas, conversely, the PIF4 -overexpression lines show long hypocotyls under Rc (Figure 2 A, D and E). The hypocotyl lengths of all these lines are similar in the dark, establishing that this phenotype is light dependent (Figure 2 A, D and E). Moreover, all these lines also show similar hypocotyl lengths under all fluence rates of FRc (Figure 2 A, D and E), suggesting lack of involvement of PIF4 in phyA signaling. Since phyB is the major photoreceptor for Rc-light responses in seedling de-etiolation ( Quail et al., 1995 ), these photobiological experiments suggest the conclusion that PIF4 is involved selectively or specifically in the phyB signaling pathway. This conclusion is verified by the results of double mutant analyses, which show that phyB plays the major role in the hypersensitive phenotype of the srl2 mutant (Figure 3 ). The alternative possibilities that the Rc-specific phenotype of srl2 -mutant and PIF4 -antisense lines is an indirect consequence of phyB overexpression ( Wagner et al., 1991 ), or a hyperactive phyB mutant molecule ( Kretsch et al., 2000 ), appear unlikely. This is because western blot analysis shows that the phyB level is similar in srl2 and the wild type (Figure 2 C), and the SRL2 gene maps to a locus on chromosome 2, >35 cM south of PHYB .

Importantly, the concomitant Rc enhancement of cotyledon expansion and inhibition of hypocotyl elongation observed in the srl2 mutant compared with the wild type (Figure 2 B) establishes that the srl2 mutation enhances normal light-induced photomorphogenic development, as opposed to causing a general defect in seedling growth and development. This conclusion is based on the well-established reciprocal effects of light on cell expansion in hypocotyls and cotyledons during de-etiolation, whereby hypocotyls are inhibited and cotyledons are stimulated by light ( Kendrick and Kronenberg, 1994 ; Quail, 2002 ). These opposite, organ-specific effects provide a robust phenotypic method of distinguishing between mutants that globally inhibit general cell expansion and those specifically affecting light-regulated cell expansion.

The similar hypersensitive phenotype of the antisense lines and the T-DNA insertion mutant argues for the conclusion that srl2 is a loss-of-function mutant and against the possibility that the srl2 phenotype is due to a dominant-negative effect of the potential truncated PIF4 protein (Figure 1 A and B). This, in combination with the converse hyposensitive phenotype of the transgenic overexpression lines (Figure 2 ), indicates strongly that PIF4 acts as a negative regulatory component in the signaling pathway.

A number of previous studies have reported the identification of other loci that appear to function selectively in Rc signaling in seedling de-etiolation. These include pef2 and pef3 ( Ahmad and Cashmore, 1996 ), red1 ( Wagner et al., 1997 ), poc1 ( Halliday et al., 1999 ), gi ( Huq et al., 2000b ) and elf3 ( Liu et al., 2001 ), which act as positive regulators, and PKS1 ( Fankhauser et al., 1999 ), ATHB2 ( Steindler et al., 1999 ) and srl1 ( Huq et al., 2000a ), which act as negative regulators. Of the positively acting loci, only POC1 / PIF3 ( Ni et al., 1998 ), GI and ELF3 have thus far been molecularly cloned, and all three have been reported to encode nuclear proteins. Of the negatively acting components, PKS1 has been shown to be a cytosolic protein that interacts with both phyA and phyB, and ATHB2 has been implicated in shade avoidance. However, no mutant has been reported for these latter two genes. Thus SRL2 (encoding PIF4) is the first genetically identified locus that acts negatively in phyB-specific signaling.

Although PIF4 and PIF3 are closely related bHLH proteins, both involved in implementing phy-regulated seedling de-etiolation, there appear to be important differences in the functional roles played by each in this process. PIF3 appears to function in both phyA and phyB pathways, and has been shown to be involved in regulating not only the cell expansion responses underlying the visible hypocotyl and cotyledon growth phenotypes, but also the expression of key regulatory genes, such as CCA1 and LHY , which are involved in controlling chloroplast biogenesis and circadian rhythms ( Ni et al., 1998 ; Martinez-Garcia et al., 2000 ). PIF4, by contrast, appears to be specific to the phyB pathway, and, while clearly being involved in the growth responses that constitute the visible phenotype (Figures 2 and 3), does not appear to regulate CCA1 , LHY or other frequently used marker genes of plastid development (data not shown). These observations suggest a possible bifurcation in the phyB signaling pathway, whereby PIF4 may act specifically in phyB-regulated expression of a subset of genes involved in cell expansion, whereas PIF3 may act more globally in controlling multiple facets of seedling de-etiolation (Figure 7 ). It will be necessary to identify phyB-controlled genes that regulate the cell expansion process in response to light in order to investigate this possibility further.

The cellular and molecular mechanisms by which PIF4 exerts its regulatory function remain to be fully elucidated. At one level, the light-induced increase in PIF4 expression (Figure 1 B) raises the possibility that the implied increase in PIF4 protein abundance might be involved in its function. However, because both R and FR light are about equally effective in enhancing PIF4 expression, it is unlikely that this effect provides the apparent specificity of PIF4 in phyB signaling. At another level, the apparent constitutive nuclear localization of PIF4 (Figure 6 ) and its capacity to bind to the G-box DNA-sequence motif (Figure 5 ) indicate strongly that PIF4 is likely to function in regulating gene expression in seedling de-etiolation, consistent with the established role of bHLH factors as transcriptional regulators. However, the apparent lack of involvement of PIF4 in regulating previously characterized G-box-containing genes, such as CCA1 and LHY , which are targets of PIF3 ( Martinez-Garcia et al., 2000 ), suggests that nucleotides outside the core G-box hexanucleotide sequence might provide cis -element specificity enabling different members of the bHLH family to discriminate between target promoters in the living plant cell, as is known for other systems ( Littlewood and Evan, 1998 ). Based on these considerations, it might be anticipated that PIF4 will be selectively targeted, at least in part, to promoters of genes involved in regulating cell expansion.

The role of the physical interaction we have observed here between PIF4 and phyB in phyB signaling also remains unclear. On the one hand, the molecular specificity of the interaction of phyB with PIF4 is similar to that of PIF3 in in vitro pull-down assays. PIF4 binds with strong selectivity for the Pfr form of phyB, and with reduced affinity for signaling-mutant derivatives of phyB (Figure 4 ). These data, along with the phyB-dependent phenotype of the srl2 mutant, are consistent with the notion that this direct interaction is functionally significant to phyB-specific signaling. However, in contrast to PIF3, PIF4 does not appear to retain the capacity to bind to phyB once bound to DNA (Figure 5 ). Thus, in principle, there appear to be at least two hypothetical mechanisms whereby a phyB–PIF4 interaction might operate negatively in phyB signaling: (i) PIF4 might sequester the active form of phyB in a non-productive complex, thereby reducing the photoreceptor’s effective activity elsewhere; (ii) similarly, but conversely, this complex might effectively inhibit the biochemical function of PIF4, counteracting its negative activity. However, there are currently insufficient data to assess the relative contribution, if any, of either of these or other possible mechanisms to the function of PIF4. Thus, despite the compelling evidence that PIF4 is involved in phyB-regulated de-etiolation, the molecular mechanism by which this putative transcriptional regulator participates in this process requires further investigation.
