---
pmid: '35512704'
title: Systematic discovery of mutation-directed neo-protein-protein interactions
  in cancer.
authors:
- Mo X
- Niu Q
- Ivanov AA
- Tsang YH
- Tang C
- Shu C
- Li Q
- Qian K
- Wahafu A
- Doyle SP
- Cicka D
- Yang X
- Fan D
- Reyna MA
- Cooper LAD
- Moreno CS
- Zhou W
- Owonikoko TK
- Lonial S
- Khuri FR
- Du Y
- Ramalingam SS
- Mills GB
- Fu H
journal: Cell
year: '2022'
pmcid: PMC9597701
doi: 10.1016/j.cell.2022.04.014
---

# Systematic discovery of mutation-directed neo-protein-protein interactions in cancer.
**Authors:** Mo X, Niu Q, Ivanov AA, Tsang YH, Tang C, Shu C, Li Q, Qian K, Wahafu A, Doyle SP, Cicka D, Yang X, Fan D, Reyna MA, Cooper LAD, Moreno CS, Zhou W, Owonikoko TK, Lonial S, Khuri FR, Du Y, Ramalingam SS, Mills GB, Fu H
**Journal:** Cell (2022)
**DOI:** [10.1016/j.cell.2022.04.014](https://doi.org/10.1016/j.cell.2022.04.014)
**PMC:** [PMC9597701](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9597701/)

## Abstract

1. Cell. 2022 May 26;185(11):1974-1985.e12. doi: 10.1016/j.cell.2022.04.014. Epub
 2022 May 4.

Systematic discovery of mutation-directed neo-protein-protein interactions in 
cancer.

Mo X(1), Niu Q(2), Ivanov AA(3), Tsang YH(4), Tang C(5), Shu C(6), Li Q(2), Qian 
K(2), Wahafu A(7), Doyle SP(2), Cicka D(2), Yang X(8), Fan D(2), Reyna MA(9), 
Cooper LAD(10), Moreno CS(11), Zhou W(12), Owonikoko TK(13), Lonial S(13), Khuri 
FR(14), Du Y(3), Ramalingam SS(15), Mills GB(4), Fu H(16).

Author information:
(1)Department of Pharmacology and Chemical Biology, Emory University School of 
Medicine, Atlanta, GA 30322, USA; Winship Cancer Institute of Emory University, 
Atlanta, GA 30322, USA.
(2)Department of Pharmacology and Chemical Biology, Emory University School of 
Medicine, Atlanta, GA 30322, USA.
(3)Department of Pharmacology and Chemical Biology, Emory University School of 
Medicine, Atlanta, GA 30322, USA; Emory Chemical Biology Discovery Center, Emory 
University School of Medicine, Atlanta, GA 30322, USA; Winship Cancer Institute 
of Emory University, Atlanta, GA 30322, USA.
(4)Division of Oncologic Science, Oregon Health Sciences University School of 
Medicine, Portland, OR 97239, USA.
(5)Department of Pharmacology and Chemical Biology, Emory University School of 
Medicine, Atlanta, GA 30322, USA; Department of Urology, The First Affiliated 
Hospital, Medical School of Xi'An Jiaotong University, Xi'an, Shannxi 710061, 
PRC.
(6)Department of Pharmacology and Chemical Biology, Emory University School of 
Medicine, Atlanta, GA 30322, USA; Department of Gynecology, The Third Xiangya 
Hospital of Central South University, Changsha, Hunan 410013, PRC.
(7)Department of Pharmacology and Chemical Biology, Emory University School of 
Medicine, Atlanta, GA 30322, USA; Department of Neurosurgery, the First 
Affiliated Hospital of Xi'An Jiaotong University, Xi'an, PRC.
(8)Department of Pharmacology and Chemical Biology, Emory University School of 
Medicine, Atlanta, GA 30322, USA; Emory Chemical Biology Discovery Center, Emory 
University School of Medicine, Atlanta, GA 30322, USA.
(9)Department of Biomedical Informatics, Emory University School of Medicine, 
Atlanta, GA 30322, USA.
(10)Department of Pathology, Northwestern University Feinberg School of 
Medicine, Chicago, IL, USA.
(11)Winship Cancer Institute of Emory University, Atlanta, GA 30322, USA; 
Department of Biomedical Informatics, Emory University School of Medicine, 
Atlanta, GA 30322, USA; Department of Pathology and Laboratory Medicine, Emory 
University School of Medicine, Atlanta, GA 30322, USA.
(12)Department of Pathology and Laboratory Medicine, Emory University School of 
Medicine, Atlanta, GA 30322, USA; Department of Hematology and Medical Oncology, 
Emory University, Atlanta, GA 30322, USA.
(13)Department of Hematology and Medical Oncology, Emory University, Atlanta, GA 
30322, USA.
(14)Department of Hematology and Medical Oncology, Emory University, Atlanta, GA 
30322, USA; Department of Internal Medicine, Division of Hematology and 
Oncology, American University of Beirut, Beirut, 1107-2020, Lebanon.
(15)Winship Cancer Institute of Emory University, Atlanta, GA 30322, USA.
(16)Department of Pharmacology and Chemical Biology, Emory University School of 
Medicine, Atlanta, GA 30322, USA; Emory Chemical Biology Discovery Center, Emory 
University School of Medicine, Atlanta, GA 30322, USA; Winship Cancer Institute 
of Emory University, Atlanta, GA 30322, USA; Department of Hematology and 
Medical Oncology, Emory University, Atlanta, GA 30322, USA. Electronic address: 
hfu@emory.edu.

Comprehensive sequencing of patient tumors reveals genomic mutations across 
tumor types that enable tumorigenesis and progression. A subset of oncogenic 
driver mutations results in neomorphic activity where the mutant protein 
mediates functions not engaged by the parental molecule. Here, we identify 
prevalent variant-enabled neomorph-protein-protein interactions (neoPPI) with a 
quantitative high-throughput differential screening (qHT-dS) platform. The 
coupling of highly sensitive BRET biosensors with miniaturized coexpression in 
an ultra-HTS format allows large-scale monitoring of the interactions of 
wild-type and mutant variant counterparts with a library of cancer-associated 
proteins in live cells. The screening of 17,792 interactions with 2,172,864 data 
points revealed a landscape of gain of interactions encompassing both oncogenic 
and tumor suppressor mutations. For example, the recurrent BRAF V600E lesion 
mediates KEAP1 neoPPI, rewiring a BRAFV600E/KEAP1 signaling axis and creating 
collateral vulnerability to NQO1 substrates, offering a combination therapeutic 
strategy. Thus, cancer genomic alterations can create neo-interactions, 
informing variant-directed therapeutic approaches for precision medicine.

Copyright Â© 2022 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.cell.2022.04.014
PMCID: PMC9597701
PMID: 35512704 [Indexed for MEDLINE]

Conflict of interest statement: Declaration of interests H.F. is scientific 
founder of PiVista Therapeutics.

## Full Text

These authors contributed equally to this work.

AUTHOR CONTRIBUTIONS

X.M., Q.N., A.A.I., Y.T.H., C.T., X.Y., C.S., S.D., D.C., D.F., Q.L., K.Q. and Y.D. conducted the experiments; X.M., Q.N., A.A.I., A.W., M.A.R., L.A.D.C., C.S.M., W.Z., T.O., S.L., F.R.K., Y.D., S.S.R., G.D.M, and H.F. participated in data analysis and interpretation; X.M., Q.N., A.A.I., G.B.M, F.R.K., Y.D. and H.F. participated in discussion and manuscript preparation; X.M., Q.N., A.A.I., D.C., Y.D. and H.F. designed the experiments and wrote the paper; and all were involved in manuscript editing.

Comprehensive sequencing of patient tumors reveals genomic mutations across tumor types that enable tumorigenesis and progression. A subset of oncogenic driver mutations results in neomorphic activity where the mutant protein mediates functions not engaged by the parental molecule. Here, we identify prevalent variant-enabled neomorph-protein-protein interactions (neoPPI) with a quantitative High Throughput differential Screening (qHT-dS) platform. Coupling of highly sensitive BRET biosensors with miniaturized co-expression in an ultra-HTS format allows large-scale monitoring of interactions of wild-type and mutant variant counterparts with a library of cancer-associated proteins in live cells. Screening of 17,792 interactions with 2,172,864 data points revealed a landscape of gain-of-interactions encompassing both oncogenic and tumor suppressor mutations. For example, the recurrent BRAF V600E lesion mediates KEAP1 neoPPI, rewiring a BRAFV600E/KEAP1 signaling axis and creating collateral vulnerability to NQO1 substrates, offering a combination therapeutic strategy. Thus, cancer genomic alterations can create neo-interactions, informing variant-directed therapeutic approaches for precision medicine.

Pan cancer analysis unveils widespread driver-mutation directed protein-protein interactions, informing variant-mediated rewiring of oncogenic programs and therapeutic approaches for precision medicine.
