---
pmid: '34819669'
title: A multi-scale map of cell structure fusing protein images and interactions.
authors:
- Qin Y
- Huttlin EL
- Winsnes CF
- Gosztyla ML
- Wacheul L
- Kelly MR
- Blue SM
- Zheng F
- Chen M
- Schaffer LV
- Licon K
- Bäckström A
- Vaites LP
- Lee JJ
- Ouyang W
- Liu SN
- Zhang T
- Silva E
- Park J
- Pitea A
- Kreisberg JF
- Gygi SP
- Ma J
- Harper JW
- Yeo GW
- Lafontaine DLJ
- Lundberg E
- Ideker T
journal: Nature
year: '2021'
full_text_available: false
pmcid: PMC9053732
doi: 10.1038/s41586-021-04115-9
---

# A multi-scale map of cell structure fusing protein images and interactions.
**Authors:** Qin Y, Huttlin EL, Winsnes CF, Gosztyla ML, Wacheul L, Kelly MR, Blue SM, Zheng F, Chen M, Schaffer LV, Licon K, Bäckström A, Vaites LP, Lee JJ, Ouyang W, Liu SN, Zhang T, Silva E, Park J, Pitea A, Kreisberg JF, Gygi SP, Ma J, Harper JW, Yeo GW, Lafontaine DLJ, Lundberg E, Ideker T
**Journal:** Nature (2021)
**DOI:** [10.1038/s41586-021-04115-9](https://doi.org/10.1038/s41586-021-04115-9)
**PMC:** [PMC9053732](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9053732/)

## Abstract

1. Nature. 2021 Dec;600(7889):536-542. doi: 10.1038/s41586-021-04115-9. Epub 2021
 Nov 24.

A multi-scale map of cell structure fusing protein images and interactions.

Qin Y(1)(2), Huttlin EL(3), Winsnes CF(4), Gosztyla ML(5)(6)(7), Wacheul L(8), 
Kelly MR(1), Blue SM(5)(6)(7), Zheng F(1), Chen M(1), Schaffer LV(1), Licon 
K(1), Bäckström A(4), Vaites LP(3), Lee JJ(1), Ouyang W(4), Liu SN(1), Zhang 
T(3), Silva E(1), Park J(1), Pitea A(1), Kreisberg JF(1), Gygi SP(3), Ma J(9), 
Harper JW(3), Yeo GW(2)(5)(6)(7), Lafontaine DLJ(8), Lundberg E(10)(11)(12), 
Ideker T(13)(14)(15)(16)(17).

Author information:
(1)Department of Medicine, University of California San Diego, La Jolla, CA, 
USA.
(2)Bioinformatics and Systems Biology Program, University of California San 
Diego, La Jolla, CA, USA.
(3)Department of Cell Biology, Harvard Medical School, Boston, MA, USA.
(4)Science for Life Laboratory, School of Engineering Sciences in Chemistry, 
Biotechnology and Health, KTH Royal Institute of Technology, Stockholm, Sweden.
(5)Department of Cellular and Molecular Medicine, University of California San 
Diego, La Jolla, CA, USA.
(6)Stem Cell Program, University of California San Diego, La Jolla, CA, USA.
(7)Institute for Genomic Medicine, University of California San Diego, La Jolla, 
CA, USA.
(8)RNA Molecular Biology, Fonds de la Recherche Scientifique (F.R.S./FNRS), 
Université Libre de Bruxelles (ULB), Charleroi-Gosselies, Belgium.
(9)Institute for Artificial Intelligence, Peking University, Beijing, China.
(10)Science for Life Laboratory, School of Engineering Sciences in Chemistry, 
Biotechnology and Health, KTH Royal Institute of Technology, Stockholm, Sweden. 
emma.lundberg@scilifelab.se.
(11)Department of Genetics, Stanford University, Stanford, CA, USA. 
emma.lundberg@scilifelab.se.
(12)Chan Zuckerberg Biohub, San Francisco, San Francisco, CA, USA. 
emma.lundberg@scilifelab.se.
(13)Department of Medicine, University of California San Diego, La Jolla, CA, 
USA. tideker@ucsd.edu.
(14)Bioinformatics and Systems Biology Program, University of California San 
Diego, La Jolla, CA, USA. tideker@ucsd.edu.
(15)Institute for Genomic Medicine, University of California San Diego, La 
Jolla, CA, USA. tideker@ucsd.edu.
(16)Department of Computer Science and Engineering, University of California San 
Diego, La Jolla, CA, USA. tideker@ucsd.edu.
(17)Department of Bioengineering, University of California San Diego, La Jolla, 
CA, USA. tideker@ucsd.edu.

The cell is a multi-scale structure with modular organization across at least 
four orders of magnitude1. Two central approaches for mapping this 
structure-protein fluorescent imaging and protein biophysical association-each 
generate extensive datasets, but of distinct qualities and resolutions that are 
typically treated separately2,3. Here we integrate immunofluorescence images in 
the Human Protein Atlas4 with affinity purifications in BioPlex5 to create a 
unified hierarchical map of human cell architecture. Integration is achieved by 
configuring each approach as a general measure of protein distance, then 
calibrating the two measures using machine learning. The map, known as the 
multi-scale integrated cell (MuSIC 1.0), resolves 69 subcellular systems, of 
which approximately half are to our knowledge undocumented. Accordingly, we 
perform 134 additional affinity purifications and validate subunit associations 
for the majority of systems. The map reveals a pre-ribosomal RNA processing 
assembly and accessory factors, which we show govern rRNA maturation, and 
functional roles for SRRM1 and FAM120C in chromatin and RPS3A in splicing. By 
integration across scales, MuSIC increases the resolution of imaging while 
giving protein interactions a spatial dimension, paving the way to incorporate 
diverse types of data in proteome-wide cell maps.

© 2021. The Author(s), under exclusive licence to Springer Nature Limited.

DOI: 10.1038/s41586-021-04115-9
PMCID: PMC9053732
PMID: 34819669 [Indexed for MEDLINE]

Conflict of interest statement: Competing interests T.I. is a co-founder of 
Data4Cure, is on the Scientific Advisory Board and has an equity interest. T.I. 
is on the Scientific Advisory Board of Ideaya BioSciences and has an equity 
interest. G.W.Y is a co-founder, a member of the Board of Directors, on the 
Scientific Advisory Board, an equity holder and a paid consultant for Locanabio 
and Eclipse BioInnovations. G.W.Y is a visiting professor at the National 
University of Singapore. The terms of these arrangements have been reviewed and 
approved by the University of California San Diego in accordance with its 
conflict-of-interest policies. E.L is on the Scientific Advisory Boards of 
Cartography Biosciences, Nautilus Biotechnology and Interline Therapeutics, and 
has an equity interest in all of these. J.W.H. is a co-founder of Caraway 
Therapeutics, is on the Scientific Advisory Board and has an equity interest. 
J.W.H. is Founding Scientific Advisor for Interline Therapeutics.
