---
pmid: '17468741'
title: Human INT6/eIF3e is required for nonsense-mediated mRNA decay.
authors:
- Morris C
- Wittmann J
- Jäck HM
- Jalinot P
journal: EMBO Rep
year: '2007'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2002529
doi: 10.1038/sj.embor.7400955
---

# Human INT6/eIF3e is required for nonsense-mediated mRNA decay.
**Authors:** Morris C, Wittmann J, Jäck HM, Jalinot P
**Journal:** EMBO Rep (2007)
**DOI:** [10.1038/sj.embor.7400955](https://doi.org/10.1038/sj.embor.7400955)
**PMC:** [PMC2002529](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2002529/)

## Abstract

1. EMBO Rep. 2007 Jun;8(6):596-602. doi: 10.1038/sj.embor.7400955. Epub 2007 Apr 
20.

Human INT6/eIF3e is required for nonsense-mediated mRNA decay.

Morris C(1), Wittmann J, Jäck HM, Jalinot P.

Author information:
(1)LBMC, UMR5239 CNRS-ENS de Lyon, IFR 128 Biosciences Lyon Gerland, 46 Allée 
d'Italie, 69364 Lyon cedex 07, France.

The mammalian integration site 6 (INT6) protein has been implicated in breast 
carcinogenesis and characterized as the eIF3e non-core subunit of the 
translation initiation factor eIF3, but its role in this complex is not known. 
Here, we show that INT6 knockdown by RNA interference strongly inhibits 
nonsense-mediated messenger RNA decay (NMD), which triggers degradation of mRNAs 
with premature stop codons. In contrast to the eIF3b core subunit, which is 
required for both NMD and general translation, INT6 is only necessary for the 
former process. Consistent with such a role, immunoprecipitation experiments 
showed that INT6 co-purifies with CBP80 and the NMD factor UPF2. In addition, 
several transcripts known to be upregulated by UPF1 or UPF2 depletion were also 
found to be sensitive to INT6 suppression. From these observations, we propose 
that INT6, in association with eIF3, is involved in routing specific mRNAs for 
degradation.

DOI: 10.1038/sj.embor.7400955
PMCID: PMC2002529
PMID: 17468741 [Indexed for MEDLINE]

## Full Text

Abstract

The mammalian integration site 6 (INT6) protein has been implicated in breast carcinogenesis and characterized as the eIF3e non-core subunit of the translation initiation factor eIF3, but its role in this complex is not known. Here, we show that INT6 knockdown by RNA interference strongly inhibits nonsense-mediated messenger RNA decay (NMD), which triggers degradation of mRNAs with premature stop codons. In contrast to the eIF3b core subunit, which is required for both NMD and general translation, INT6 is only necessary for the former process. Consistent with such a role, immunoprecipitation experiments showed that INT6 co-purifies with CBP80 and the NMD factor UPF2. In addition, several transcripts known to be upregulated by UPF1 or UPF2 depletion were also found to be sensitive to INT6 suppression. From these observations, we propose that INT6, in association with eIF3, is involved in routing specific mRNAs for degradation.

Introduction

Integration site 6 protein (INT6), first identified as the product of a murine gene in which integration of mouse mammary tumour virus seems to be sufficient to induce breast carcinoma ( Marchetti et al , 1995 ), has been characterized as a subunit (eIF3e) of the eukaryotic translation initiation factor eIF3 ( Asano et al , 1997 ). In addition, this highly conserved protein can interact with subunits of the COP9 signalosome (CSN) and the 26S proteasome ( Karniol et al , 1998 ; Yahalom et al , 2001 ; Hoareau Alves et al , 2002 ). Consistently, fission yeast INT6 promotes proteasome assembly by interacting with, and mediating nuclear import of, the Rpn5 (regulatory particle non-ATPase 5) proteasomal subunit ( Yen et al , 2003 ).

In contrast to other eIF3 subunits, INT6 is not essential for global translation. Fission yeast INT6 mutants show only a weak inhibition in the rate of protein synthesis and a slight decrease in the polysome content ( Bandyopadhyay et al , 2000 ; Akiyoshi et al , 2001 ; Zhou et al , 2005 ). Budding yeast eIF3 is viewed as a factor comprising a core complex of five stoichiometric subunits required for global translation ( Asano et al , 1998 ; Phan et al , 1998 ). Mammalian eIF3 is probably organized in a similar way, and non-core subunits, one of which is INT6, might have regulatory roles. For example, Kim et al (2004) have shown in Arabidopsis thaliana that eIF3h was required for translation of messenger RNAs containing particular 5′ leader sequences. In addition, a recent study reported that two distinct eIF3 complexes exist in fission yeast: one is defined by the Csn7Bp/eIF3m subunit and is found associated with an important part of mRNAs, whereas the other complex contains INT6 and is associated with a more restricted mRNA subset ( Zhou et al , 2005 ).

Consistent with the idea that INT6 is a regulatory subunit of eIF3, we searched for a role of human INT6 in nonsense-mediated mRNA decay (NMD), as this mRNA surveillance mechanism is translation dependent ( Belgrader et al , 1993 ; Thermann et al , 1998 ). Briefly, NMD degrades transcripts with a premature termination codon (PTC) located more than 50 nucleotides upstream of an exon–exon junction ( Maquat, 2004 ; Baker & Parker, 2004 ; Conti & Izaurralde, 2005 ). The prevalent model for mammalian NMD suggests a first, or pioneer, round of translation, which recognizes the PTC, and the recruitment of the three Up Frameshift (UPF) factors by the exon junction complex and the translation termination factors. Here, we show that INT6 silencing by RNA interference (RNAi) impairs NMD in a manner similar to UPF1 depletion. This effect relates neither to an alteration in UPF1 or UPF2 protein amounts nor to UPF1 phosphorylation. However, co-immunoprecipitation experiments showed an association of INT6 with UPF2. These data establish INT6 as a non-core eIF3 subunit contributing specifically to NMD.

Results

To assess whether INT6 is important in NMD, levels of a β-globin mRNA carrying a PTC (NS39; Thermann et al , 1998 ) were compared with those of the wild-type transcript in HeLa cells treated with INT6 short interfering RNAs (siRNAs). In control cells, levels of the NS39 β-globin mRNA represented only 2–3% of the wild-type levels, owing to its degradation by the NMD process ( Fig 1A ). As expected, UPF1 silencing caused a tenfold increase in NS39 mRNA levels. A similar effect, between six- and sevenfold, was observed by depleting INT6 ( Fig 1A ). It should be noted that RNAi against INT6 is slightly less efficient than that against UPF1. That NMD was found to be impaired using two siRNA duplexes targeting different sequences of INT6 mRNA argues against an off-target effect. Accordingly, expression of a Flag-tagged INT6 from a recombinant mRNA lacking the 3′ untranslated region reverted the negative effect on NMD of a third siRNA couple targeting this non-coding part of INT6 mRNA (supplementary Fig S1 online ). Next we wanted to determine whether the inhibitory effect of INT6 silencing on NMD was specific of this process or if it was a consequence of an arrest of translation. Therefore, the steady-state translation was evaluated by analysing the amount of exogenous green fluorescent protein (GFP) produced in siRNA-treated cells. We found that GFP mRNA translation was not affected either after INT6 silencing or after UPF1 depletion, as expected ( Fig 1B ). In agreement with this, we found that INT6 depletion did not significantly affect the amount of other eIF3 subunits (supplementary Fig S2 online ).

The same experiments were carried out with siRNAs targeting the eIF3b core subunit and the eIF3l non-core subunit, which interacts directly with INT6 ( Morris-Desbois et al , 2001 ). Knockdown of eIF3b increases the NS39 mRNA levels ( Fig 1C ), but because it also strongly inhibits general translation, as shown by immunoblot analyses of GFP and endogenous eIF3l ( Fig 1D ), the inhibitory effect on NMD should relate to the translational arrest. After depletion of eIF3l, neither NMD nor steady-state translation was significantly affected ( Fig 1C,D ). Collectively, these data show that INT6 is specifically required for NMD but is not essential for general translation, as anticipated ( Bandyopadhyay et al , 2000 ; Morris & Jalinot, 2005 ; Zhou et al , 2005 ).

Most capped mRNAs are subjected to NMD soon after their transcription when they are still bound by the nuclear cap binding heterodimer CBP80–CBP20, which drives the pioneer round of translation ( Ishigaki et al , 2001 ; Chiu et al , 2004 ). Hence, we investigated whether INT6 could be found in complexes including the cap-binding protein (CBP) dimer. Indeed, antibodies to INT6 specifically precipitated CBP80, and this association was still detected when RNase was added before precipitation ( Fig 2A ), therefore excluding the possibility that it merely results from the presence of both proteins on the same mRNA. In the same experiment, eIF4G-1 was also detected with a weak intensity, but we failed to detect eIF4E ( Fig 2A ). CBP80 was also detected when precipitation was carried out with an antibody to eIF3 ( Fig 2B ). INT6 has been shown to be involved in protein degradation ( Yen et al , 2003 ); therefore, we examined whether its silencing affects expression of the UPF1 and UPF2 NMD factors. Immunoblot analysis showed that neither UPF1 nor UPF2 concentration was modified in INT6-silenced cells ( Fig 3A ). An interaction between these two proteins is required for NMD; therefore this was studied next, and we found that UPF1 was still present in UPF2 immunoprecipitates after INT6 RNAi ( Fig 3B ). Finally, we investigated whether INT6 depletion inhibits NMD by acting on UPF1 phosphorylation ( Pal et al , 2001 ) mediated by the ataxia telangiectasia mutated (ATM)-like kinase SMG1 ( Denning et al , 2001 ; Yamashita et al , 2001 ). By using an antibody specific for sites phosphorylated by this kinase family, we found that INT6 RNAi did not modify the extent of UPF1 phosphorylation ( Fig 3C ). Together, these observations indicate that NMD impairment caused by INT6 knockdown is apparently not due to an effect on the functioning of UPF1 and UPF2 factors.

To explore further the role of INT6 in NMD, we investigated whether it associates with UPF proteins. UPF1 was weakly detected in the INT6 immunoprecipitates and RNase treatment of cell lysate before precipitation further reduced the amount of UPF1 that was pulled down ( Fig 4A , top). By contrast, UPF2 was efficiently precipitated with INT6 and the interaction was unaffected by RNase treatment ( Fig 4A , middle). Co-purification of UPF2 was also observed in precipitations using antibodies to whole eIF3 or to the eIF3b core subunit ( Fig 4B ).

These observations are in agreement with a specific role of INT6 in NMD. To substantiate this further, we examined whether INT6 depletion influences the expression of physiologic NMD substrates. Several genes were selected from previous genome-wide studies designed to identify mRNAs sensitive to UPF1 or UPF2 depletion ( Mendell et al , 2004 ; Wittmann et al , 2006 ), and their transcript levels were quantified in cells treated with siRNAs targeting INT6; UPF1, was used as comparison. Table 1 , which summarizes our results and data from previous studies, shows that selected genes can be divided into two groups. The first group includes genes in which the mRNA levels were raised in response to UPF1 or INT6 depletion, and the second group includes genes in which the transcript levels were increased by UPF1 silencing but not by INT6 suppression. This set of data is restricted; however, it establishes that some known NMD substrates require INT6 for degradation, whereas others that are prone to UPF1-mediated degradation do not.

Discussion

Our findings show that INT6 is required for degradation of a model NMD-sensitive transcript and also for several cellular mRNAs regulated by this process. Although it has been shown that eIF3 participates in the pioneer round of translation ( Chiu et al , 2004 ), the functional contribution of its different subunits to the NMD process has only been addressed in one study, by Welch & Jacobson (1999) . They reported that mutation of PRT1, the eIF3b yeast homologue, causes increased suppressor of Ty 10 (SPT10) mRNA levels, in a manner similar to UPF1 depletion. In agreement with this study, we found that silencing of eIF3b, a core subunit of the initiation factor, hinders both the pioneer round and steady-state translation. Concerning INT6, our results show that it is not required for general translation, but its silencing inhibits NMD. This was not observed after depletion of eIF3l, a protein also viewed as an eIF3 subunit. eIF3l is found in the cytoplasm in association with INT6 and eIF3 ( Morris-Desbois et al , 2001 ), and in the nucleus, it is nucleolar and acts as a cofactor of RNA PolI ( Seither et al , 2001 ; Yuan et al , 2002 ). By contrast, nuclear INT6 is present in the nucleoplasm and is partly associated with chromatin. As previously reported in other species ( Yahalom et al , 2001 ), cell fractionation experiments showed that all eIF3 subunits are not equally distributed between the nucleus and the cytoplasm (C. Morris, data not shown). Hence, the composition of eIF3 is likely to be dynamic and one possibility is that a specific subcomplex is implicated in the nuclear pioneer round of translation. Co-precipitation of UPF2 with some eIF3 subunits, including INT6, is interesting and might represent a direct link between NMD core factors and eIF3. INT6 within eIF3 has recently been shown to interact in vitro with eIF4G-1 ( LeFebvre et al , 2006 ). Our results show that INT6 interacts with both CBP80 and UPF2, and also less intensely with eIF4G-1, even when RNA is degraded. These proteins include the middle domain of eukaryotic initiation factor 4G (MIF4G) motifs. Collectively, these observations raise the possibility that INT6 binds specifically to MIF4G domains and this would allow efficient recruitment of eIF3 complexes containing INT6 to mRNAs loaded with UPF2, thereby promoting the pioneer round of translation. INT6 was not found associated with eIF4E; therefore, this eIF3 subunit would not intervene in steady-state translation.

In conclusion, our findings support a model in which INT6 acts in CBP80-mediated translation and, in combination with UPF2, directs mRNAs towards degradation rather than active translation. They establish that human INT6, in the eIF3 complex, is involved in the quality control of specific mRNAs, the abundance of which is regulated by NMD.

Methods

Cell culture, transfection of plasmids and short interference RNAs. HeLa cells were cultured in Dulbecco's modified Eagle's medium under standard conditions. Cells were co-transfected with an NMD reporter plasmid (pBluescript-β-globin WT or pBluescript-β-globin NS39; Thermann et al , 1998 ), a reference plasmid (pSG–Flag–GFP) together with different siRNA duplexes listed in Table 2 . Lipofectamine 2000 reagent (Invitrogen, Carlsbad, CA, USA) was used according to the supplier's protocol. Cells were split 24 h after transfection and cultured for another 48 h. HeLa cells treated with siRNAs alone were collected 72 h after transfection. One portion of cells was used as a source of RNA, whereas the other was used as a source of protein.

Nonsense-mediated messenger RNA decay analysis and real-time reverse transcription–PCR. For quantification of exogenous β-globin transcripts or endogenous NMD substrates, total RNA was isolated using the RNeasy Mini kit (Qiagen, Hilden, Germany), and real-time reverse transcription–PCR (RT–PCR) analysis was performed using the QuantiTect SYBR green RT–PCR kit (Qiagen) and a LightCycler apparatus (Roche, Basel, Switzerland) according to cycling conditions specified in the handbook of the kit. Primer sequences were designed using the Primer3 software and are listed in Table 3 .

Immunoprecipitation and immunoblotting. For immunoprecipitation, 1 × 10 7 cells were lysed in 300 μl of buffer N (50 mM Tris–HCl, pH 7.4; 300 mM NaCl; 0.05% NP-40; 0.5 mM TCEP (Sigma, St Louis, MO, USA); 0.5 mM Pefabloc (Roche) and 50 U of RNase inhibitor (Promega, Madison, WI, USA)) by four 10 s sonications using a Bioruptor system (Diagenode, Liège, Belgium) and centrifuged at 10,000 g for 10 min at 4°C. Lysates were precleared by incubation with 40 μl of protein A–agarose beads for 1 h and the supernatants were incubated overnight at 4°C with primary antibodies. An 80 μl volume of protein A–agarose beads was then added for 1 h, washed three times for 10 min and resuspended in 20 μl of 2 × SDS sample buffer. After separation by SDS–polyacrylamide gel electrophoresis, proteins were transferred to a PVDF membrane and immunoblotting was performed using protein A/G coupled to peroxidase. The following antibodies have been previously described: INT6 C-20 and N-19 ( Morris-Desbois et al , 1999 ); eIF3l ( Morris-Desbois et al , 2001 ); CBP80 ( Izaurralde et al , 1994 ); UPF1 ( Applequist et al , 1997 ); UPF2 ( Wittmann et al , 2006 ); eIF3 ( Asano et al , 1997 ); and eIF4E ( Morley & McKendrick, 1997 ). The following antibodies were purchased: Flag (clone M2; Sigma), β-actin (Sigma), eIF3b (A-20; Santa Cruz Biotechnology, Santa Cruz, CA, USA), eIF4G (H-300; Santa Cruz) and anti-phospho-(Ser/Thr) ATM/ATR (ataxia telangiectasia and Rad3-related) substrate antibody (#2851; Cell Signaling Technology, Danvers, MA, USA).

Supplementary information is available at EMBO reports online ( http://www.emboreports.org ).
