---
pmid: '25720963'
title: Lysosomal calcium signalling regulates autophagy through calcineurin and ​TFEB.
authors:
- Medina DL
- Di Paola S
- Peluso I
- Armani A
- De Stefani D
- Venditti R
- Montefusco S
- Scotto-Rosato A
- Prezioso C
- Forrester A
- Settembre C
- Wang W
- Gao Q
- Xu H
- Sandri M
- Rizzuto R
- De Matteis MA
- Ballabio A
journal: Nat Cell Biol
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4801004
doi: 10.1038/ncb3114
---

# Lysosomal calcium signalling regulates autophagy through calcineurin and ​TFEB.
**Authors:** Medina DL, Di Paola S, Peluso I, Armani A, De Stefani D, Venditti R, Montefusco S, Scotto-Rosato A, Prezioso C, Forrester A, Settembre C, Wang W, Gao Q, Xu H, Sandri M, Rizzuto R, De Matteis MA, Ballabio A
**Journal:** Nat Cell Biol (2015)
**DOI:** [10.1038/ncb3114](https://doi.org/10.1038/ncb3114)
**PMC:** [PMC4801004](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4801004/)

## Abstract

1. Nat Cell Biol. 2015 Mar;17(3):288-99. doi: 10.1038/ncb3114.

Lysosomal calcium signalling regulates autophagy through calcineurin and ​TFEB.

Medina DL, Di Paola S, Peluso I, Armani A, De Stefani D, Venditti R, Montefusco 
S, Scotto-Rosato A, Prezioso C, Forrester A, Settembre C, Wang W, Gao Q, Xu H, 
Sandri M, Rizzuto R, De Matteis MA, Ballabio A.

Comment in
    Autophagy. 2015;11(7):1192-5. doi: 10.1080/15548627.2015.1054594.

The view of the lysosome as the terminal end of cellular catabolic pathways has 
been challenged by recent studies showing a central role of this organelle in 
the control of cell function. Here we show that a lysosomal Ca2+ signalling 
mechanism controls the activities of the phosphatase calcineurin and of its 
substrate ​TFEB, a master transcriptional regulator of lysosomal biogenesis and 
autophagy. Lysosomal Ca2+ release through ​mucolipin 1 (​MCOLN1) activates 
calcineurin, which binds and dephosphorylates ​TFEB, thus promoting its nuclear 
translocation. Genetic and pharmacological inhibition of calcineurin suppressed 
​TFEB activity during starvation and physical exercise, while calcineurin 
overexpression and constitutive activation had the opposite effect. Induction of 
autophagy and lysosomal biogenesis through ​TFEB required ​MCOLN1-mediated 
calcineurin activation. These data link lysosomal calcium signalling to both 
calcineurin regulation and autophagy induction and identify the lysosome as a 
hub for the signalling pathways that regulate cellular homeostasis.

DOI: 10.1038/ncb3114
PMCID: PMC4801004
PMID: 25720963 [Indexed for MEDLINE]

## Full Text

Lysosomes are membrane-bound organelles present in all cell types. Their role in degradation and recycling processes has been extensively characterized1–3. The lysosomal lumen acidic pH with the presence of a broad variety of hydrolases able to degrade an ample spectrum of substrates, make these organelles extraordinary machineries for the recycling of cellular waste. Extracellular substrates reach the lysosome mainly via the endocytic and phagocytic pathways, while intracellular substrates are the delivered to the lysosome by the autophagic pathway via the fusion of autophagosomes with lysosomes4, 5. Thus, lysosomes are the “terminal end” of most cellular catabolic pathways. The role of the lysosomes in degradation and recycling processes has always been considered as a cellular “housekeeping” function and little attention has been paid to the regulation of these processes and to the possible influence of environmental cues, such as starvation and physical exercise.

The discovery that the Transcription Factor EB (TFEB) is a master regulator of lysosomal and autophagic function and of energy metabolism6–8 suggested that environmental cues may control lysosomal function via the induction of a broad transcriptional program. TFEB activity is regulated by phosphorylation9–13, which keeps TFEB inactive in the cytoplasm, while dephosphorylated TFEB travels to the nucleus to activate transcriptional target genes. TFEB phosphorylation is mediated by mTORC1, a major kinase complex that positively regulates cell growth and negatively regulates autophagy. Interestingly, it is known mTORC1 exerts its activity on the lysosomal surface and is positively regulated by lysosomal nutrients14, 15. The regulation of TFEB by lysosomal mTORC1 and the shuttling of TFEB to the nucleus revealed a lysosome-to-nucleus signaling mechanism9. Thus, these studies identify the lysosome as a signaling hub that controls cellular homeostasis via both post-translational and transcriptional mechanisms14–17.

Another aspect of lysosomal function underestimated in the past is the ability of lysosomes to store Ca2+ and to participate to calcium signaling processes. Several calcium channels reside on the lysosomal membrane. Recent studies have investigated the role of these lysosomal calcium channels in fundamental cellular processes and their involvement in disease mechanisms18. In addition, the recent discoveries of calcium microdomains, which mediate local calcium signals from several intracellular compartments (e.g. mitochondria)19, further suggest the involvement of the lysosome in intracellular calcium signaling.

In the present study, while searching for a phosphatase that de-phosphorylates TFEB, we discovered another example of lysosomal signaling. We revealed a calcium signaling mechanism that starts at the lysosome and controls autophagy via calcineurin-mediated induction of TFEB.

Calcineurin modulates TFEB subcellular localizationPrevious studies demonstrated that mTOR-mediated phosphorylation of TFEB serine residues Ser142 and Ser211 promotes the interaction of TFEB with the 14-3-3 protein and results in a cytoplasmic localization. Conversely, conditions that lead to mTOR inhibition, such as starvation and lysosomal stress, promote TFEB nuclear translocation and transcriptional activation of lysosomal and autophagic genes6, 7, 9, 14, 15, 17. While the role of the kinases that mediate TFEB phosphorylation has been defined by previous studies9–13, the phosphatase(s) involved in its de-phosphorylation have remained elusive. To identify the phosphatase(s) that de-phosphorylate(s) TFEB we performed a High Content (HC) screening of a phosphatase siRNA library using a cellular assay based on cytoplasm-to-nucleus shuttling of TFEB during starvation9. We tested the effects of the specific inhibition of each of 231 phosphatases or putative phosphatases on TFEB subcellular localization. The most significant hit identified by the primary screening was the calcineurin catalytic subunit isoform beta (PPP3CB; Gene ID:5532)20, thus we focused subsequent studies exclusively on this phosphatase.Fig. 1a shows that inhibition of PPP3CB suppressed starvation-induced nuclear translocation of TFEB. The ability of PPP3CB to inhibit TFEB nuclear translocation was further confirmed by using a set of 3 different PPP3CB siRNAs, as well as the pool of 3, in the HeLaTFEB-GFP cell line (Fig. 1b). In addition, the calcineurin inhibitors Cyclosporin A and FK506, both jointly and individually, reduced TFEB nuclear translocation during starvation (Fig. 1c). Importantly, overexpression of ΔCaN, a constitutively active form of calcineurin, induced TFEB nuclear translocation in normally fed cells, suggesting that calcineurin activity was not only necessary but sufficient to induce TFEB nuclear translocation (Fig. 1d,e). Consistent with its effect on TFEB nuclear translocation, ΔCaN induced the expression of TFEB transcriptional target genes in a TFEB-dependent manner (Fig. 1f). Both starvation and calcineurin overexpression induced nuclear translocation as well as de-phosphorylation of the Nuclear Factor of Activated T cells (NFAT), which is known to depend on calcineurin activity21 (Supplementary Fig. 1). In vivo studies showed that transfection by electroporation of both calcineurin subunit ΔCaN and its essential regulatory subunit PPP3R1 (CnB)22 in adult muscles from wild type mice significantly induced TFEB nuclear translocation (Fig. 1g). In addition, exercise, a physiological stimulus that activates calcineurin23, 24, promoted TFEB nuclear translocation and this effect was blunted when an endogenous calcineurin inhibitor, CAIN, was electroporated into muscle cells25 (Fig. 1g). These loss-of-function and gain-of-function data demonstrate that calcineurin plays a crucial role in the regulation of TFEB subcellular localization, both in cell culture and in skeletal muscle, during energy-demanding conditions, such as starvation and physical exercise.

Calcineurin modulates TFEB subcellular localization

Previous studies demonstrated that mTOR-mediated phosphorylation of TFEB serine residues Ser142 and Ser211 promotes the interaction of TFEB with the 14-3-3 protein and results in a cytoplasmic localization. Conversely, conditions that lead to mTOR inhibition, such as starvation and lysosomal stress, promote TFEB nuclear translocation and transcriptional activation of lysosomal and autophagic genes6, 7, 9, 14, 15, 17. While the role of the kinases that mediate TFEB phosphorylation has been defined by previous studies9–13, the phosphatase(s) involved in its de-phosphorylation have remained elusive. To identify the phosphatase(s) that de-phosphorylate(s) TFEB we performed a High Content (HC) screening of a phosphatase siRNA library using a cellular assay based on cytoplasm-to-nucleus shuttling of TFEB during starvation9. We tested the effects of the specific inhibition of each of 231 phosphatases or putative phosphatases on TFEB subcellular localization. The most significant hit identified by the primary screening was the calcineurin catalytic subunit isoform beta (PPP3CB; Gene ID:5532)20, thus we focused subsequent studies exclusively on this phosphatase.

Fig. 1a shows that inhibition of PPP3CB suppressed starvation-induced nuclear translocation of TFEB. The ability of PPP3CB to inhibit TFEB nuclear translocation was further confirmed by using a set of 3 different PPP3CB siRNAs, as well as the pool of 3, in the HeLaTFEB-GFP cell line (Fig. 1b). In addition, the calcineurin inhibitors Cyclosporin A and FK506, both jointly and individually, reduced TFEB nuclear translocation during starvation (Fig. 1c). Importantly, overexpression of ΔCaN, a constitutively active form of calcineurin, induced TFEB nuclear translocation in normally fed cells, suggesting that calcineurin activity was not only necessary but sufficient to induce TFEB nuclear translocation (Fig. 1d,e). Consistent with its effect on TFEB nuclear translocation, ΔCaN induced the expression of TFEB transcriptional target genes in a TFEB-dependent manner (Fig. 1f). Both starvation and calcineurin overexpression induced nuclear translocation as well as de-phosphorylation of the Nuclear Factor of Activated T cells (NFAT), which is known to depend on calcineurin activity21 (Supplementary Fig. 1). In vivo studies showed that transfection by electroporation of both calcineurin subunit ΔCaN and its essential regulatory subunit PPP3R1 (CnB)22 in adult muscles from wild type mice significantly induced TFEB nuclear translocation (Fig. 1g). In addition, exercise, a physiological stimulus that activates calcineurin23, 24, promoted TFEB nuclear translocation and this effect was blunted when an endogenous calcineurin inhibitor, CAIN, was electroporated into muscle cells25 (Fig. 1g). These loss-of-function and gain-of-function data demonstrate that calcineurin plays a crucial role in the regulation of TFEB subcellular localization, both in cell culture and in skeletal muscle, during energy-demanding conditions, such as starvation and physical exercise.

Calcineurin binds and dephosphorylates TFEBThese data prompted us to investigate the relationship between calcineurin and TFEB. PPP3CB-TFEB interaction was detected by co-immunoprecipitation experiments, performed in both HeLaTFEB-GFP and in HEK-293TFEB-GFP cells (Supplementary Fig. 2a-c) and by a histidine-pulldown assay using purified His-TFEB and purified catalytic and regulatory calcineurin subunits (CaN) (Supplementary Fig. 2d). In the his-pulldown assay the interaction between purified TFEB and CaN plus calmodulin (CaM), a protein known to interact with calcineurin22, appeared to be stronger that the one between TFEB and CaN alone, while no interaction was detected between TFEB and CaM alone (Supplementary Fig. 2d). Notably, the interaction between PPP3CB and TFEB was detected both in fed and starved cells, suggesting that it is independent from the phosphorylation status of TFEB, as previously shown for the binding of calcineurin to NFAT19 (Supplementary Fig. 2a,c). Finally, a physical interaction between PPP3CB and TFEB was also detected by Proximity Ligation Assay (PLA)27 performed on endogenous proteins in non-transfected living cells (Fig. 2a). Similar data were obtained by PLA on exogenous overexpressed proteins (Fig. 2b). The interaction between mTOR and TFEB9 was used as a positive control of the assay, while the absence of interaction between the Golgi protein Golgin97 and TFEB was used as a negative control (Supplementary Fig. 3a). The specificities of the antibodies used in the PLA were tested by both immunofluorescence and immunoblotting (Supplementary Fig. 3b-d). These data demonstrate that calcineurin physically interacts with TFEB and suggest that TFEB is a calcineurin substrate.Thus, we tested whether calcineurin de-phosphorylates TFEB. siRNA-mediated inhibition of calcineurin abolished the downshift in TFEB molecular weight that is associated with TFEB de-phosphorylation during starvation7 (Fig. 2c). In addition, expression of a constitutively active mutant form of calcineurin in normally fed cells resulted in TFEB molecular weight downshift (Fig. 2d). Calcineurin-mediated de-phosphorylation of TFEB was confirmed by an in vitro calcineurin phosphatase assay, which revealed that calcineurin de-phosphorylates two critical serine residues, S211 and S142 known to be involved in TFEB nuclear translocation9–12 (Fig. 2e). These result identify TFEB as a novel calcineurin substrate. Thus, calcineurin de-phosphorylates TFEB and induces its nuclear translocation by a mechanism similar to the one described for NFAT (REF).Starvation and inhibition of mTOR have been shown to promote TFEB nuclear translocation 9–11. However, we found that siRNA-mediated inhibition of calcineurin significantly reduced the nuclear translocation of both endogenous and overexpressed TFEB in HeLa, HeLaTFEB-GFP, and MCF-7 cells, respectively, either starved or treated with Torin1, a strong mTOR inhibitor28, demonstrating that the regulation of TFEB subcellular localization by calcineurin is independent from mTOR activity and suggesting that calcineurin acts downstream of mTOR in the regulation of TFEB (Fig. 2f and Supplementary Fig. 3e).

Calcineurin binds and dephosphorylates TFEB

These data prompted us to investigate the relationship between calcineurin and TFEB. PPP3CB-TFEB interaction was detected by co-immunoprecipitation experiments, performed in both HeLaTFEB-GFP and in HEK-293TFEB-GFP cells (Supplementary Fig. 2a-c) and by a histidine-pulldown assay using purified His-TFEB and purified catalytic and regulatory calcineurin subunits (CaN) (Supplementary Fig. 2d). In the his-pulldown assay the interaction between purified TFEB and CaN plus calmodulin (CaM), a protein known to interact with calcineurin22, appeared to be stronger that the one between TFEB and CaN alone, while no interaction was detected between TFEB and CaM alone (Supplementary Fig. 2d). Notably, the interaction between PPP3CB and TFEB was detected both in fed and starved cells, suggesting that it is independent from the phosphorylation status of TFEB, as previously shown for the binding of calcineurin to NFAT19 (Supplementary Fig. 2a,c). Finally, a physical interaction between PPP3CB and TFEB was also detected by Proximity Ligation Assay (PLA)27 performed on endogenous proteins in non-transfected living cells (Fig. 2a). Similar data were obtained by PLA on exogenous overexpressed proteins (Fig. 2b). The interaction between mTOR and TFEB9 was used as a positive control of the assay, while the absence of interaction between the Golgi protein Golgin97 and TFEB was used as a negative control (Supplementary Fig. 3a). The specificities of the antibodies used in the PLA were tested by both immunofluorescence and immunoblotting (Supplementary Fig. 3b-d). These data demonstrate that calcineurin physically interacts with TFEB and suggest that TFEB is a calcineurin substrate.

Thus, we tested whether calcineurin de-phosphorylates TFEB. siRNA-mediated inhibition of calcineurin abolished the downshift in TFEB molecular weight that is associated with TFEB de-phosphorylation during starvation7 (Fig. 2c). In addition, expression of a constitutively active mutant form of calcineurin in normally fed cells resulted in TFEB molecular weight downshift (Fig. 2d). Calcineurin-mediated de-phosphorylation of TFEB was confirmed by an in vitro calcineurin phosphatase assay, which revealed that calcineurin de-phosphorylates two critical serine residues, S211 and S142 known to be involved in TFEB nuclear translocation9–12 (Fig. 2e). These result identify TFEB as a novel calcineurin substrate. Thus, calcineurin de-phosphorylates TFEB and induces its nuclear translocation by a mechanism similar to the one described for NFAT (REF).

Starvation and inhibition of mTOR have been shown to promote TFEB nuclear translocation 9–11. However, we found that siRNA-mediated inhibition of calcineurin significantly reduced the nuclear translocation of both endogenous and overexpressed TFEB in HeLa, HeLaTFEB-GFP, and MCF-7 cells, respectively, either starved or treated with Torin1, a strong mTOR inhibitor28, demonstrating that the regulation of TFEB subcellular localization by calcineurin is independent from mTOR activity and suggesting that calcineurin acts downstream of mTOR in the regulation of TFEB (Fig. 2f and Supplementary Fig. 3e).

Calcium modulates TFEB localizationSince calcineurin is activated by intracellular calcium20, 29, we assessed the calcium-dependence of TFEB regulation by artificially modulating cytoplasmic Ca2+ using Ca2+ chelators and ionophores. The Ca2+ chelator BAPTA-AM significantly reduced TFEB nuclear translocation in both starved and Torin1-treated cells (Fig. 3a). Both thapsigargin, an inhibitor of ER Ca2+-ATPase, and ionomycin, a Ca2+ ionophore, significantly induced TFEB nuclear translocation in normally fed cells and kept TFEB in the nucleus after starvation/re-feeding (Fig. 3b-d). Thapsigargin treatment also reduced the phosphorylation levels of TFEB serine residues S142 and S211 in fed cells (Fig. 3e,f). Importantly, thapsigargin-induced nuclear translocation of TFEB was suppressed when PPP3CB and its essential regulatory subunit PPP3R1 were silenced (Fig. 3g) and was blunted in PPP3R1−/− cells30 (Supplementary Fig. 3f), thus proving that the effects of Ca2+ on TFEB are mediated by calcineurin. Previous studies reported that long-term (24 hours) thapsigargin treatment inhibited mTOR activity via Ca2+-dependent induction of AMPK31. However, we found that up to 3 hours treatment with thapsigargin did not affect mTOR activity, measured by mTOR-mediated phosphorylation of P70S6 kinase (Supplementary Fig. 3g), indicating that intracellular Ca2+ elevation promotes TFEB nuclear translocation through the activation of calcineurin and further suggesting that calcineurin-dependent TFEB de-phosphorylation acts downstream of mTOR in the TFEB regulatory pathway.

Calcium modulates TFEB localization

Since calcineurin is activated by intracellular calcium20, 29, we assessed the calcium-dependence of TFEB regulation by artificially modulating cytoplasmic Ca2+ using Ca2+ chelators and ionophores. The Ca2+ chelator BAPTA-AM significantly reduced TFEB nuclear translocation in both starved and Torin1-treated cells (Fig. 3a). Both thapsigargin, an inhibitor of ER Ca2+-ATPase, and ionomycin, a Ca2+ ionophore, significantly induced TFEB nuclear translocation in normally fed cells and kept TFEB in the nucleus after starvation/re-feeding (Fig. 3b-d). Thapsigargin treatment also reduced the phosphorylation levels of TFEB serine residues S142 and S211 in fed cells (Fig. 3e,f). Importantly, thapsigargin-induced nuclear translocation of TFEB was suppressed when PPP3CB and its essential regulatory subunit PPP3R1 were silenced (Fig. 3g) and was blunted in PPP3R1−/− cells30 (Supplementary Fig. 3f), thus proving that the effects of Ca2+ on TFEB are mediated by calcineurin. Previous studies reported that long-term (24 hours) thapsigargin treatment inhibited mTOR activity via Ca2+-dependent induction of AMPK31. However, we found that up to 3 hours treatment with thapsigargin did not affect mTOR activity, measured by mTOR-mediated phosphorylation of P70S6 kinase (Supplementary Fig. 3g), indicating that intracellular Ca2+ elevation promotes TFEB nuclear translocation through the activation of calcineurin and further suggesting that calcineurin-dependent TFEB de-phosphorylation acts downstream of mTOR in the TFEB regulatory pathway.

Starvation induces lysosomal Ca2+ release via MCOLN1Our data indicate that starvation-induced TFEB nuclear translocation requires the activity of calcineurin and that an increase of cytosolic calcium induces TFEB nuclear translocation. Thus, we studied the effects of starvation on cytosolic Ca2+. Starvation did not induce changes in the overall levels of cytosolic Ca2+, as recorded by the aequorin-based probe localized in the bulk cytoplasm (Fig. 4a). Furthermore, while BAPTA-AM treatment inhibited TFEB nuclear translocation, a slower Ca2+-chelator (i.e. EGTA-AM)32 did not, suggesting that starvation rapidly triggers local calcium signals (Fig. 4b). The ER Ca2+ content was not significantly affected after starvation, as detected by measuring cytosolic calcium transients evoked by the IP3-mobilizing hormone histamine in fed and starved cells (Fig. 4a,c).We postulated that a calcium release from the lysosome may be responsible for local calcineurin activation. Starvation clearly elicited a Ca2+ signal in HeLa cells transiently transfected with a MCOLN1 Ca2+ sensor linked to the lysosomal calcium channel mucolipin 1 (MCOLN1), also known as Transient Receptor Potential calcium channel MucoLipin subfamily member 1 (TRPML1), which is localized on the lysosomal surface (ML1-GCaMP3) (Figure 4d). Similar results were obtained in HEK293 cells stably overexpressing ML1-GCaMP3 (Figure 4e). Consistent with a lysosomal contribution to this calcium signaling mechanism, the increase in Ca2+ levels induced by starvation detected by ML1-GCaMP3 were not affected by the removal of extracellular Ca2+ (ruling out the contribution of plasma membrane calcium channels), but was blunted by treatment with the lysosomotropic agent GPN (Fig. 4f). Notably, neither the silencing of the lysosomal two-pore channel TPCN233 nor the overexpression of a critical regulator of mitochondrial Ca2+ uptake, MICU134 altered TFEB subcellular localization (Supplementary Fig. 4a,b). These data strongly suggest that starvation specifically induces lysosomal Ca2+ release via MCOLN1 and that this generates a lysosomal calcium microdomain that is responsible for calcineurin activation.We then analyzed the involvement of MCOLN1 in the regulation of TFEB nuclear translocation35–38. ShRNA-mediated inhibition of MCOLN1 significantly reduced cytoplasm-to-nucleus shuttling of both endogenous and transiently transfected TFEB-GFP after starvation, further suggesting that this process requires Ca2+ release from the lysosome (Fig. 5a). Similar results were obtained by nucleus/cytoplasm fractionation (Fig. 5b). Consistently, depletion of MCOLN1 reduced the downshift of endogenous TFEB molecular weight during starvation, suggesting a defective de-phosphorylation (Supplementary Fig. 5a). Mutations in the MCOLN1 gene cause Mucolipidosis type 4 (MLIV), a lysosomal storage disease39, 40. We found that starvation-induced nuclear translocation of TFEB is inhibited in fibroblasts from MLIV patients (Fig. 5c). Furthermore, overexpression of wild type MCOLN1 and of gain of function MCOLN1 mutants in HeLaTFEBGFP cells promoted TFEB nuclear translocation (Fig. 5d) in a calcineurin-dependent fashion (Supplementary Fig. 5b) and similar results were obtained by using a recently described MCOLN1-specific agonist, SF5136 (Fig. 5e, Supplementary movies 1-3 and Supplementary Fig. 5c). Importantly, inhibition of MCOLN1 in skeletal muscles in vivo by transfection of shMCOLN1 suppressed the exercise-mediated nuclear translocation of TFEB (Fig. 5f), demonstrating the physiological relevance of lysosomal calcium signaling on TFEB regulation in vivo. Together these data suggest that while calcineurin-TFEB interaction may occur anywhere in the cytoplasm, focal Ca2+ release from the lysosome via MCOLN1 modulates TFEB activity via local calcineurin activation and suggest that MCOLN1 plays a crucial role in the activation of calcineurin near the lysosomal surface, through the generation of a lysosomal Ca2+ microdomain.

Starvation induces lysosomal Ca2+ release via MCOLN1

Our data indicate that starvation-induced TFEB nuclear translocation requires the activity of calcineurin and that an increase of cytosolic calcium induces TFEB nuclear translocation. Thus, we studied the effects of starvation on cytosolic Ca2+. Starvation did not induce changes in the overall levels of cytosolic Ca2+, as recorded by the aequorin-based probe localized in the bulk cytoplasm (Fig. 4a). Furthermore, while BAPTA-AM treatment inhibited TFEB nuclear translocation, a slower Ca2+-chelator (i.e. EGTA-AM)32 did not, suggesting that starvation rapidly triggers local calcium signals (Fig. 4b). The ER Ca2+ content was not significantly affected after starvation, as detected by measuring cytosolic calcium transients evoked by the IP3-mobilizing hormone histamine in fed and starved cells (Fig. 4a,c).

We postulated that a calcium release from the lysosome may be responsible for local calcineurin activation. Starvation clearly elicited a Ca2+ signal in HeLa cells transiently transfected with a MCOLN1 Ca2+ sensor linked to the lysosomal calcium channel mucolipin 1 (MCOLN1), also known as Transient Receptor Potential calcium channel MucoLipin subfamily member 1 (TRPML1), which is localized on the lysosomal surface (ML1-GCaMP3) (Figure 4d). Similar results were obtained in HEK293 cells stably overexpressing ML1-GCaMP3 (Figure 4e). Consistent with a lysosomal contribution to this calcium signaling mechanism, the increase in Ca2+ levels induced by starvation detected by ML1-GCaMP3 were not affected by the removal of extracellular Ca2+ (ruling out the contribution of plasma membrane calcium channels), but was blunted by treatment with the lysosomotropic agent GPN (Fig. 4f). Notably, neither the silencing of the lysosomal two-pore channel TPCN233 nor the overexpression of a critical regulator of mitochondrial Ca2+ uptake, MICU134 altered TFEB subcellular localization (Supplementary Fig. 4a,b). These data strongly suggest that starvation specifically induces lysosomal Ca2+ release via MCOLN1 and that this generates a lysosomal calcium microdomain that is responsible for calcineurin activation.

We then analyzed the involvement of MCOLN1 in the regulation of TFEB nuclear translocation35–38. ShRNA-mediated inhibition of MCOLN1 significantly reduced cytoplasm-to-nucleus shuttling of both endogenous and transiently transfected TFEB-GFP after starvation, further suggesting that this process requires Ca2+ release from the lysosome (Fig. 5a). Similar results were obtained by nucleus/cytoplasm fractionation (Fig. 5b). Consistently, depletion of MCOLN1 reduced the downshift of endogenous TFEB molecular weight during starvation, suggesting a defective de-phosphorylation (Supplementary Fig. 5a). Mutations in the MCOLN1 gene cause Mucolipidosis type 4 (MLIV), a lysosomal storage disease39, 40. We found that starvation-induced nuclear translocation of TFEB is inhibited in fibroblasts from MLIV patients (Fig. 5c). Furthermore, overexpression of wild type MCOLN1 and of gain of function MCOLN1 mutants in HeLaTFEBGFP cells promoted TFEB nuclear translocation (Fig. 5d) in a calcineurin-dependent fashion (Supplementary Fig. 5b) and similar results were obtained by using a recently described MCOLN1-specific agonist, SF5136 (Fig. 5e, Supplementary movies 1-3 and Supplementary Fig. 5c). Importantly, inhibition of MCOLN1 in skeletal muscles in vivo by transfection of shMCOLN1 suppressed the exercise-mediated nuclear translocation of TFEB (Fig. 5f), demonstrating the physiological relevance of lysosomal calcium signaling on TFEB regulation in vivo. Together these data suggest that while calcineurin-TFEB interaction may occur anywhere in the cytoplasm, focal Ca2+ release from the lysosome via MCOLN1 modulates TFEB activity via local calcineurin activation and suggest that MCOLN1 plays a crucial role in the activation of calcineurin near the lysosomal surface, through the generation of a lysosomal Ca2+ microdomain.

Calcineurin and MCOLN1 regulate autophagy via TFEBAs TFEB transcriptionally regulates autophagy during starvation7, 41, we tested whether the transcriptional response of lysosomal/autophagic genes to starvation required calcineurin. Whole-genome gene expression profiling experiments revealed that starvation perturbed the transcriptomes of both wild type and PPP3R1−/− mouse embryonic fibroblasts (MEFs) in a statistically significant manner. We specifically analyzed the expression behavior of 322 lysosomal-autophagic genes (Supplementary Table 1) in the two cell lines. Remarkably, both the number of genes responsive to starvation and the amplitude of the response were significantly reduced in PPP3R1−/− compared to wild type cells (Fig. 6a, Supplementary Fig. 6a, Supplementary Table 2 and 3). Transcriptome analysis results were confirmed by real time PCR experiments on selected TFEB target genes (Supplementary Fig. 6b). Similar results were obtained in HeLa cells in which both PPP3R1 and PPP3CB were silenced (Supplementary Fig. 6c). Next we tested whether calcineurin-mediated TFEB activity had an effect on lysosomal/autophagic function. Lack of calcineurin in PPP3R1−/− cells suppressed the starvation-induced elevation of the lysosomal marker LAMP1 (Fig. 6b) and autophagosomal protein LC3, and blunted the increase in LC3-II levels and in the number of LC3-positive vesicles (Fig. 6c,d). Similar results were obtained by pharmacological inhibition of calcineurin (Supplementary Fig. 7a).In addition, the silencing of calcineurin, significantly reduced the number of phosphatidylinositol 3-phosphate (PI(3)P)-positive vesicles, indicating an impairment in autophagosome formation42, during starvation (Fig. 6e). Furthermore, silencing of both PPP3R1 and PPP3CB in HeLa cells resulted in a reduction of the autophagic flux, measured by the degradation of the autophagic substrate p62 (Fig. 6f). This autophagy impairment was not due to the inhibition of mTOR signaling, as we observed normal phosphorylation kinetics of two different mTOR substrates (ULK, and P70S6K)(Fig. 6g). Conversely, overexpression of ΔCaN increased LC3II protein levels in a TFEB-dependent manner (Fig. 6h,i). Together these data identify calcineurin as a novel regulator of the lysosomal-autophagic pathway.Consistent with a role of MCOLN1 and lysosomal Ca2+ in the activation of calcineurin, the silencing of MCOLN1, significantly reduced the number of PI(3)P-positive vesicles (Fig.7a). Similar results were observed using immunofluorescence against endogenous WIPI protein, a marker of early steps in autophagosome formation, in cells silenced for MCOLN1 (Supplementary Fig. 7b). In addition, we found that MCOLN1 overexpression resulted in a significant increase in the levels of the autophagosome marker LC3, which were further increased by bafilomycin, indicating an induction of autophagy, rather than an impairment of autophagosome degradation (Fig. 7b,c). Thus, lysosomal calcium signaling via MCOLN1 controls autophagy through calcineurin-mediated activation of TFEB.Interestingly, MCOLN1 is known to be a direct transcriptional target of TFEB6, 38, 41. This suggests the presence of a positive feedback loop by which TFEB regulates the expression of MCOLN1, which in turn promotes TFEB activity via Ca2+-mediated activation of calcineurin. Consistently, overexpression of TFEB dramatically increased the effects of MCOLN1 agonists, likely due to an increase in the number of MCOLN1 channels, as observed with lysosomal patch clamp (Fig. 7d,e).

Calcineurin and MCOLN1 regulate autophagy via TFEB

As TFEB transcriptionally regulates autophagy during starvation7, 41, we tested whether the transcriptional response of lysosomal/autophagic genes to starvation required calcineurin. Whole-genome gene expression profiling experiments revealed that starvation perturbed the transcriptomes of both wild type and PPP3R1−/− mouse embryonic fibroblasts (MEFs) in a statistically significant manner. We specifically analyzed the expression behavior of 322 lysosomal-autophagic genes (Supplementary Table 1) in the two cell lines. Remarkably, both the number of genes responsive to starvation and the amplitude of the response were significantly reduced in PPP3R1−/− compared to wild type cells (Fig. 6a, Supplementary Fig. 6a, Supplementary Table 2 and 3). Transcriptome analysis results were confirmed by real time PCR experiments on selected TFEB target genes (Supplementary Fig. 6b). Similar results were obtained in HeLa cells in which both PPP3R1 and PPP3CB were silenced (Supplementary Fig. 6c). Next we tested whether calcineurin-mediated TFEB activity had an effect on lysosomal/autophagic function. Lack of calcineurin in PPP3R1−/− cells suppressed the starvation-induced elevation of the lysosomal marker LAMP1 (Fig. 6b) and autophagosomal protein LC3, and blunted the increase in LC3-II levels and in the number of LC3-positive vesicles (Fig. 6c,d). Similar results were obtained by pharmacological inhibition of calcineurin (Supplementary Fig. 7a).

In addition, the silencing of calcineurin, significantly reduced the number of phosphatidylinositol 3-phosphate (PI(3)P)-positive vesicles, indicating an impairment in autophagosome formation42, during starvation (Fig. 6e). Furthermore, silencing of both PPP3R1 and PPP3CB in HeLa cells resulted in a reduction of the autophagic flux, measured by the degradation of the autophagic substrate p62 (Fig. 6f). This autophagy impairment was not due to the inhibition of mTOR signaling, as we observed normal phosphorylation kinetics of two different mTOR substrates (ULK, and P70S6K)(Fig. 6g). Conversely, overexpression of ΔCaN increased LC3II protein levels in a TFEB-dependent manner (Fig. 6h,i). Together these data identify calcineurin as a novel regulator of the lysosomal-autophagic pathway.

Consistent with a role of MCOLN1 and lysosomal Ca2+ in the activation of calcineurin, the silencing of MCOLN1, significantly reduced the number of PI(3)P-positive vesicles (Fig.7a). Similar results were observed using immunofluorescence against endogenous WIPI protein, a marker of early steps in autophagosome formation, in cells silenced for MCOLN1 (Supplementary Fig. 7b). In addition, we found that MCOLN1 overexpression resulted in a significant increase in the levels of the autophagosome marker LC3, which were further increased by bafilomycin, indicating an induction of autophagy, rather than an impairment of autophagosome degradation (Fig. 7b,c). Thus, lysosomal calcium signaling via MCOLN1 controls autophagy through calcineurin-mediated activation of TFEB.

Interestingly, MCOLN1 is known to be a direct transcriptional target of TFEB6, 38, 41. This suggests the presence of a positive feedback loop by which TFEB regulates the expression of MCOLN1, which in turn promotes TFEB activity via Ca2+-mediated activation of calcineurin. Consistently, overexpression of TFEB dramatically increased the effects of MCOLN1 agonists, likely due to an increase in the number of MCOLN1 channels, as observed with lysosomal patch clamp (Fig. 7d,e).

DISCUSSIONThe regulation of autophagy is thought to be a biphasic process, with a fast induction that relies on post-translational and protein-protein interaction events, and a sustained response that is mediated by transcriptional mechanisms17. Our results identify a calcium signaling pathway that originates from the lysosome and regulates autophagy by activating a TFEB-mediated transcriptional program. It has been well-established that autophagy is negatively regulated by the mTOR kinase43. The new pathway regulating autophagy identified in our study is mediated by the Ca2+-dependent phosphatase calcineurin and seems to operate independently from mTOR. Fig. 8 shows a model representing how starvation and exercise regulate TFEB via both the mTOR and calcineurin pathways. In normal feeding/sedentary conditions mTORC1 phosphorylates TFEB allowing its interaction with 14-3-3 proteins, thus preventing its nuclear translocation. Conversely, during starvation and physical exercise mTORC1 activity is inhibited and, at the same time, calcineurin activity is locally induced by lysosomal calcium release via MCOLN1. This leads to both a decreased rate of TFEB phosphorylation, via mTORC1 inhibition, and an induction of TFEB de-phosphorylation, via calcineurin induction. De-phosphorylated TFEB is free to travel to the nucleus and to induce the expression of lysosomal and autophagic genes (Fig. 8).Previous studies reported that exercise induces autophagy in muscle 44–46. Our results provide a mechanistic link between calcineurin activation and autophagy induction during exercise. To date, the transcription factors of the NFAT family, master regulators of T cell activation47, have been considered the main transcriptional mediators of calcineurin function. The discovery that TFEB, a master controller of autophagy, is a calcineurin substrate expands the current view on the mechanism of action of calcineurin. Interestingly, we found that starvation induces calcineurin-mediated de-phosphorylation and nuclear translocation of both TFEB and NFAT, suggesting a coordinated regulation of the two different transcriptional networks. Previous studies implicated calcineurin in a variety of both physiological and pathological processes, such as neuronal synaptic transmission, adaptative immune response and muscle remodeling after physical exercise48, 49. Our data suggest that some of these processes may be linked to the lysosomal/autophagic pathway via TFEB. In addition, other calcineurin substrates may also play a role in autophagy.Calcium plays a crucial role in the regulation of the activity of transcription factors via the induction of calcium-dependent kinases and phosphatases50. Until now conflicting results have been reported on the mechanism by which calcium regulates autophagy. Here we dissected a Ca2+-dependent pathway that originates from the lysosome and regulates autophagy at the transcriptional level. Recent studies have revealed the presence of calcium microdomains, mediated by different calcium channels, which activate calcineurin in the vicinity of different cellular compartments (i.e. sarcolemma, mitochondria, sarcoplasmatic reticulum and perinuclear sarcoplasmatic reticulum) in the heart51. Our study demonstrates that local calcineurin activation can also occur near the lysosome via the lysosomal calcium channel MCOLN1. In addition, our data also revealed a positive feedback loop by which the TFEB is activated by the calcium channel MCOLN1, which is known to be a TFEB transcriptional target41.In conclusion, the identification of a mechanism by which lysosomal Ca2+ regulates autophagy via the activation of the phosphatase calcineurin and the transcription factor TFEB further support the role of the lysosome as a signaling hub. It is likely that future studies will lead to the identification of other signaling pathways originating from the lysosomal surface.

The regulation of autophagy is thought to be a biphasic process, with a fast induction that relies on post-translational and protein-protein interaction events, and a sustained response that is mediated by transcriptional mechanisms17. Our results identify a calcium signaling pathway that originates from the lysosome and regulates autophagy by activating a TFEB-mediated transcriptional program. It has been well-established that autophagy is negatively regulated by the mTOR kinase43. The new pathway regulating autophagy identified in our study is mediated by the Ca2+-dependent phosphatase calcineurin and seems to operate independently from mTOR. Fig. 8 shows a model representing how starvation and exercise regulate TFEB via both the mTOR and calcineurin pathways. In normal feeding/sedentary conditions mTORC1 phosphorylates TFEB allowing its interaction with 14-3-3 proteins, thus preventing its nuclear translocation. Conversely, during starvation and physical exercise mTORC1 activity is inhibited and, at the same time, calcineurin activity is locally induced by lysosomal calcium release via MCOLN1. This leads to both a decreased rate of TFEB phosphorylation, via mTORC1 inhibition, and an induction of TFEB de-phosphorylation, via calcineurin induction. De-phosphorylated TFEB is free to travel to the nucleus and to induce the expression of lysosomal and autophagic genes (Fig. 8).

Previous studies reported that exercise induces autophagy in muscle 44–46. Our results provide a mechanistic link between calcineurin activation and autophagy induction during exercise. To date, the transcription factors of the NFAT family, master regulators of T cell activation47, have been considered the main transcriptional mediators of calcineurin function. The discovery that TFEB, a master controller of autophagy, is a calcineurin substrate expands the current view on the mechanism of action of calcineurin. Interestingly, we found that starvation induces calcineurin-mediated de-phosphorylation and nuclear translocation of both TFEB and NFAT, suggesting a coordinated regulation of the two different transcriptional networks. Previous studies implicated calcineurin in a variety of both physiological and pathological processes, such as neuronal synaptic transmission, adaptative immune response and muscle remodeling after physical exercise48, 49. Our data suggest that some of these processes may be linked to the lysosomal/autophagic pathway via TFEB. In addition, other calcineurin substrates may also play a role in autophagy.

Calcium plays a crucial role in the regulation of the activity of transcription factors via the induction of calcium-dependent kinases and phosphatases50. Until now conflicting results have been reported on the mechanism by which calcium regulates autophagy. Here we dissected a Ca2+-dependent pathway that originates from the lysosome and regulates autophagy at the transcriptional level. Recent studies have revealed the presence of calcium microdomains, mediated by different calcium channels, which activate calcineurin in the vicinity of different cellular compartments (i.e. sarcolemma, mitochondria, sarcoplasmatic reticulum and perinuclear sarcoplasmatic reticulum) in the heart51. Our study demonstrates that local calcineurin activation can also occur near the lysosome via the lysosomal calcium channel MCOLN1. In addition, our data also revealed a positive feedback loop by which the TFEB is activated by the calcium channel MCOLN1, which is known to be a TFEB transcriptional target41.

In conclusion, the identification of a mechanism by which lysosomal Ca2+ regulates autophagy via the activation of the phosphatase calcineurin and the transcription factor TFEB further support the role of the lysosome as a signaling hub. It is likely that future studies will lead to the identification of other signaling pathways originating from the lysosomal surface.

METHODSDrugs and cellular treatmentsThe following drugs were used: Torin 1 (1µM, otherwise indicated) from Tocris Bioscience; FK506 (5 µM, otherwise indicated), cyclosporine A (10 µM, otherwise indicated), Chloroquine (100 µM) thapsigargin (300 nM), ionomycin (1µM), and EGTA from Sigma; BAPTA-AM (5-25 µM) from Invitrogen; EGTA-AM (5 µM) from Santa Cruz Biotechnology; Bafilomycin A1 (300 nM); SF51 (200 µM) from VITAS-M Laboratory, LTD. Starvation medium was HBSS, 10 mM Hepes.Cell culture, plasmids and siRNA transfectionHeLa and MCF-7, HEK 293, RPE cells were purchased from ATCC and cultured in DMEM or RPMI 1640 media supplemented with 10% fetal bovine serum, 200 µM L-glutamine, 100 µM sodium pyruvate, 5% CO2 at 37 degrees. Cells were silenced with TFEB siRNA oligonucleotides (Thermo, Dhamacon D-009798-03), PPP3CB siRNA oligonucleotides (Ambion, 4390824), PPP3R1 siRNA oligonucleotides (Invitrogen, HSS108410-3, HSS183046-3, HSS108411-3), or non-targeting siRNAs (Thermo, D-001810-10-05), by direct or reverse transfection, using lipofectamine RNAiMAX reagent (Invitrogen) according to the protocol from manufacturer. siRNA-transfected cells were collected after 72 hours, if not otherwise stated.Immortalized MEFs from conditional PPP3R1flox/flox (PPP3R1+/+) mice, kind gift from J. Molkentin (Cincinnati Children's Hospital Medical Center, USA) were cultured in DMEM, 10% CS, 200 µM L-glutamine, 100µM sodium pyruvate. PPP3R1flox/flox (PPP3R1+/+) MEFs were used as control, while lentiviral-mediated Cre transduction was used to generate PPP3R1−/− MEFs30.Stably HeLashMCOLN1 cell line was described in Medina et al, 201131. Stably HeLaTFEB-GFP cell line was described in Settembre et al, 20129. ML-IV human fibroblasts were a kind gift of G.Borsani.Human full-length TFEB-GFP, and TFEB-FLAG were previously described7. Human full length PPP3CB and MCOLN1 tagged with myc-FLAG were from OriGene. The plasmid caryinng a constitutively active form of human calcineurin catalytic subunit (ΔCan) was donated by B. Rothermel (The University of Texas Southwestern Medical Center, USA). NFATGFP was a gift from T. Baldari52; Micu1 plasmid was described in34. Cells were transfected using Lipofectamine LTX – Plus reagent (Invitrogen) or Trans-IT reagent (Mirus-Bio) according to the protocol from manufacturers.Antibodies and Western blottingFor western blot the following antibodies were used: anti-FLAG (M2, cat. F1804, 1:2000), β-actin (AC-15, cat. A5441, 1:4000), β-tubulin (TUB 2.1, cat.T4026, 1:2000) and -ULK (cat. A7481, 1:1000) from Sigma-Aldrich; anti-LC3 (cat. NB100-2220, 1:1000) from Novus Biologicals; anti-PARP1 (cat. ALX-10-302, 1:4000) from ENZO Life Sciences; anti-TFEB (cat. 4240, 1:1000), -p-Ser 14-3-3 binding motif (cat. 9601, 1:1000), -p-P70S6K (cat. 108D2, 1:1000), -p-ULK (cat. 6888, 1:1000) from Cell Signaling Technologies; anti-HA (16B12, cat. MMS-101P, 1:7500) from Covance; anti-P70S6K (cat. KAP-CC035, 1:1000) from Stressgen; anti-PPP3CB (cat. TA308438, 1:1000) from OriGene; anti-pan 14-3-3 (B11, cat. sc-133232, 1:1000), -GAPDH (6C5, cat. sc-32233, 1:10000) and -c-Myc (9E10, cat. sc-40, 1:1000) from Santa Cruz Biotechnology; anti-GFP (1:10000) was provided by M. A. De Matteis (TIGEM); anti-p-Ser142-TFEB (1:1000) was produced as described previously9.Total cell lysates were prepared by solubilisation in RIPA buffer (1% Triton X-100, 1% sodium deoxycholate, 0.1% SDS, 0.15 M NaCl and 0.01 M sodium phosphate, at pH 7.2) containing protease (Roche) and phosphatase (Sigma) inhibitors. The soluble fraction was recovered in the supernatant after centrifugation at 12,000g for 15 min, and protein concentration was measured by the Bradford method. Nuclear/cytosolic fractions were isolated as previously described9.After SDS–PAGE and immunoblotting, the proteins recognized by the specific antibodies were visualized by chemiluminescence methods (ECL Western Blotting Substrate, Pierce) using peroxidase-conjugated anti-rabbit or anti-mouse secondary antibodies (Calbiochem). Membranes were developed using a Chemidoc UVP imaging system (Ultra-Violet Products Ltd) and densitometric quantification was performed in unsaturated images using ImageJ (NIH).Proximity Ligation AssayPLA assay was performed using the Duolink in situ reagents (Olink Biosciences) according to the manufacturer’s specifications.Golgin 97 polyclonal antibody used for immunofluorescence was raised in rabbit by immunization with recombinant glutathione S-transferase (GST)-fusion full-length protein. The antibody was affinity purified over a column conjugated with a His fusion of the protein. Other antibodies used in this study were from the following sources: mAb TFEB (17, cat. sc-101532, 1:50) from Santa Cruz Biotechnology and anti- mTOR (cat. 7010, 1:200) from Cell Signaling Technology. Alexa-conjugated secondary antibodies and ER tracker were from Life Technologies.Immunofluorescence experiments were performed as previously described53. Briefly, HeLa cells were fixed in 4% paraformaldehyde (PFA) for 10 minutes and permeabilized in 0.1% (w/v) saponin, 0.5% (w/v) BSA and 50 mM NH4Cl in PBS (blocking buffer). Cells were incubated with the indicated primary antibodies for 1 hour and subsequently incubated with secondary antibodies. For confocal imaging, the samples were examined under a Zeiss LSM 700 confocal microscope. Optical sections were obtained under a ×63 immersion objective at a definition of 1024 × 1024 pixels (average of eight or sixteen scans), adjusting the pinhole diameter to 1 Airy unit for each emission channel to have all the intensity values between 1 and 254 (linear range). For image analyses, Mock treated and TFEB-KD HeLa cells were fixed and stained as described above and cells acquired with the same laser parameters using the same magnification.Co-immunoprecipitation assaysHeLa cells expressing the indicated tagged proteins were rinsed twice with ice-cold PBS and lysed in ice-cold lysis buffer (400 mM NaCl, 25 mM Tris-HCl, pH 7.4, 1 mM EDTA and 1% Triton X-100) containing protease and phosphatase inhibitors. The soluble fractions from cell lysates were isolated by centrifugation at 12000g for 15 min in a microfuge. For co-immunoprecipitations, each lysate was incubated with corresponding antibody in binding buffer (200 mM NaCl, 25 mM Tris-HCl, pH 7.4, 1mM EDTA) with constant rotation over-night at 4°C. Then, 40 µl of a 50% slurry of Protein-G beads (Sigma-Aldrich) were added to lysates and incubated with rotation for additional 2h at 4°C. After incubation, the resins were washed 5 times in binding buffer and the samples were eluted in Laemmli buffer and separated by SDS-PAGE.Histidine pull-down assayPurified His-TFEB (300 nM), CaN (ProAlt) and CaM (Enzo Life Sciences) and GST (100 nM each) were incubated in binding buffer (20 mM Tris, pH 8; 100 mM NaCl; 2mM CaCl2; 6 mM MgCl2; 0,2% Triton X-100; 30 mM imidazole; protease and phosphatase inhibitor) over-night at 4°C. His-TFEB was pulled-down by adding 20 µl of Ni-NTA resin (Qiagen) for 2 hrs at 4°C. After incubation, the resins were washed 5 times in binding buffer and the samples were eluted in Laemmli buffer and separated by SDS-PAGE.In vitro calcineurin phosphatase assayPhosphorylated FLAG-TFEB was obtained by immunoprecipitation from fed HeLa cells. Protein-G beads containing FLAG-TFEB were incubated for 40 min at 30°C in 24 µl of corresponding phosphatase reaction buffer (20 mM HEPES, pH 7.5, 10 mM MgCl2, 2 mM DTT, protease inhibitors with 1 mM CaCl2 or 10 mM EGTA) containing purified Calcineurin (100U; ENZO Life Sciences) and Calmodulin (4 mM; ENZO Life Sciences). Samples were eluted by adding 8 µl of 4X Laemmli buffer, resolved by SDS–PAGE and analysed by immunoblotting. Antibodies anti-p-Ser 14-3-3 binding motif10, anti-P-Ser142-TFEB9 and TFEB (Cell Signaling Technologies) were used to detect phosphorylated and total TFEB forms, respectively.RNA extraction and quantitative PCRTotal RNA was extracted from cells using RNeasy Plus Mini Kit (Qiagen). Reverse transcription was performed using QuantiTect Rev Transcription Kit (Qiagen). Real-time quantitative Reverse Transcription PCR (qPCR) was performed using the LightCycler® System 2.0 (Roche Applied Science). HPRT was used for qRT-PCR as reference gene. The parameters of real-time qRT-PCR amplification were according to Roche recommendations.The following primers were used in this study. Mouse primers: PGC1a; forward (fw) gaatcaagccactacagacaccg, reverse (rev) catccctcttgagcctttcgtg, MCOLN1; fw: gcgcctatgacaccatcaa, rev: tatcctggactgctcgat; ATP6V0D1; fw: gcatctcagagcaggaccttga, rev: ggataggacacatggcatcagc, ATP6V1H; fw: gttgctgctcacgatgttggag, rev: tgtagcgaacctgctggtcttc, CTSF; fw: acgcctatgcagccataaag, rev: cttttgccatctgtgctgag, CTSB; fw: ttagcgctctcacttccactacc, rev: tgcttgctaccttcctctggtta, TPP1; fw: aagccaggcctacatagtcaga, rev: ccaagtgcttcctgcagtttaga, DPP7; fw: cgccagcaatactgtctggatac, rev: aaatgatgttgctggctgcttta, WIPI; fw: acaggacgcctgaacttctc, rev: cggcagtttctggatcgt, TcFEB; fw: gcgagagctaacagatgctga, rev: ccggtcattgatgttgaacc, HPRT; fw: caagcttgctggtgaaaagg rev: gtcaagggcatatccaacaac, PPP3R1; fw: tgcctgagttacagcagaacc rev: tcgcctttgacactgaactg, ATG14; fw: gcttcgaaggtcacacatcc, rev: cttgaggtcatggcactgtc, PPP3CA; fw; atcccaagttgtcgacgacc, rev: acactttcttccagcctgcc.
Human primers; PGC1a; fw: catgcaaatcacaatcacagg, rev: ttgtggcttttgctgttgac, MCOLN1; fw: gagtgggtgcgacaagtttc, rev: tgttctcttcccggaatgtc, ATP6V0E1; fw: cattgtgatgagcgtgttctgg, rev: aactccccggttaggaccctta, ATP6V1H; fw: ggaagtgtcagatgatcccca, rev: ccgtttgcctcgtggataat, CTSF; fw: acagaggaggagttccgcacta, rev: gcttgcttcatcttgttgcca, TFEB; fw: caaggccaatgacctggac, rev: agctccctggacttttgcag, HPRT; fw: tggcgtcgtgattagtgatg, rev: aacaccctttccaaatcctca, CTSD; fw: cttcgacaacctgatgcagc, rev: tacttggagtctgtgccacc, PPP3CA; fw: gctgccctgatgaaccaac, rev: gcaggtggttctttgaatcgg, DPP7; fw: gattcggaggaacctgagtg, rev: cggaagcaggatcttctgg, CTSB; fw: agtggagaatggcacacccta , rev: aagagccattgtcacccca, TPP1; fw: gatcccagctctcctcaatac, rev: gccatttttgcaccgtgtg, NEU1; fw: tgaagtgtttgcccctggac, rev: aggcaccatgatcatcgctg.High content nuclear translocation assayHeLaTFEBGFP cells were seeded in 96 or 384-well plates, incubated for 12 h, and treated as indicated in the text, washed, fixed, and stained with DAPI. For the acquisition of the images, at least 5 images fields were acquired per well of the 96/384-well plate by using confocal automated microscopy (Opera high content system; Perkin-Elmer). A dedicated script was developed to perform the analysis of TFEB localization on the different images (Harmony and Acapella software; Perkin-Elmer). The script calculates the ratio value resulting from the average intensity of nuclear TFEB–GFP fluorescence divided by the average of the cytosolic intensity of TFEB–GFP fluorescence. The results were normalized using negative (RPMI medium, or otherwise indicated) and positive (starvation, or otherwise indicated) control samples in the same plate54. At least two independent experiments, and up to 3000 individual cells per treatment from at least two independent wells are routinely analyzed. p-values were calculated on the basis of mean values from independent wells. The data are represented by the percentage of nuclear translocation vs the indicated control using Excel (Microsoft) or Prism software (GraphPad software).The TFEB-GFP nuclear translocation analyses of different population of transfected cells (detected by immunofluorescence of tagged-plasmids) were performed by a modified script that calculates the ratio value resulting from the average intensity of nuclear TFEB–GFP fluorescence divided by the average of the cytosolic intensity of TFEB–GFP fluorescence on both positive-transfected and negative transfected cells.High content LC3-autophagosome assayCells were seeded in 96-well plates, incubated for 12 h, and treated as indicated in the text, washed, fixed in ice-cold methanol, and immunofluorescence against LC3 (anti-LC3, Novus Biologicals, 1:400) was performed. After washing, the 96-well plate was analysed using confocal automated microscopy (Opera high content system; Perkin-Elmer). A dedicated script was developed to perform the analysis of the number of LC3-puncta on at least 5 different images per well (Harmony software; Perkin-Elmer).Time-lapseHeLaTFEBGFP were reverse transfected and plated in Mattek glass bottomed dishes. Time-lapse movies were acquired for 45’. One frame was acquired roughly every 20s with lasers set at 30% power or below. All imaging was performed with a x60 Plan Apo oil immersion lens using a Nikon Eclipse Ti Spinning Disk microscope, and images were annotated and 3D reconstructions made using the NIS Elements 4.20 software.EGFP-2×FYVE AssayHeLa cells were reverse transfected with PPP3CB/R1 or scramble control siRNA using RNAiMax. 48h later the cells transfected with the GFP-2xFYVE construct55 using Lipofectamine LTX and Plus (Invitrogen), and 24h later serum starved and fixed with 4% PFA in PBS. Imaging was performed with a 63x PlanApochromat NA 1.4 DIC oil immersion objective on an LSM700 confocal microscope (Zeiss). Image analysis (spot count per cell, minimum of 15 cells counted per treatment group) was performed using ImageJ.Endolysosomal electrophysiologyEndolysosomal electrophysiology was performed in isolated endolysosomes using a modified patch-clamp method35–37. Cos-1 cells were transfected using Lipofectamine 2000 (Invitrogen) with TFEB-S211A fused to mCherry. To increase the size of isolated endolysosomes, cells were treated with 1 µM vacuolin-1, a lipid-soluble polycyclic triazine that can selectively increase the size of endosomes and lysosomes. Whole-endolysosome recordings were performed on isolated enlarged endolysosomes. In brief, a patch pipette (electrode) was pressed against a cell and quickly pulled away to slice the cell membrane35–37. Enlarged endolysosomes were released into a dish. TFEB-S211A-positive vacuoles were identified by monitoring mCherry fluorescence. Bath (internal/cytoplasmic) solution contained 140 mM K+-Gluconate, 4 mM NaCl, 1 mM EGTA, 2 mM Na2-ATP, 2 mM MgCl2, 0.39 mM CaCl2, 0.1 mM GTP and 10 mM HEPES (pH adjusted with KOH to 7.2; free [Ca 2+]i approximately 100 nM). The pipette (luminal) solution was pH 4.6 standard extracellular solution (modified Tyrode’s) with 145 mM NaCl, 5 mM KCl, 2 mM CaCl2, 1 mM MgCl2, 20 mM HEPES and 10 mM glucose (pH adjusted with NaOH). All bath solutions were applied by means of a fast perfusion system to achieve a complete solution exchange within a few seconds. Data were collected using an Axopatch 2A patch-clamp amplifier, Digidata 1440, and pClamp 10.0 software (Axon Instruments). Whole-endolysosome currents were digitized at 10 kHz and filtered at 2 kHz. All experiments were conducted at room temperature (21 – 23°C), and all recordings were analysed using pCLAMP10 (Axon Instruments) and Origin 8.0 (OriginLab). ML-SA1 was purchased from Princeton BioMolecular Research Inc.GCaMP3 Ca2+ imaging in GCaMP3-ML1-HEK293 cellsGCaMP3 Ca2+ imaging was performed in HEK cells that stably express GCaMP3-ML1, which are lysosome-targeted genetically encoded Ca2+ sensors36. The fluorescence intensity at 488 nm (F488) was recorded with the spinning disk confocal imaging system, which consisted of an Olympus IX81 inverted microscope, a 60X or 100X objective (Olympus), a CSU-X1 scanner (Yokogawa), an iXon EM-CCD camera (Andor), and MetaMorph Advanced Imaging acquisition software v.7.7.8.0 (Molecular Devices). Live imaging of ML1-GCaMP3 was performed on a heated stage.Ratiometric GCaMP3 Ca2+ imaging in HeLa cellsHeLa cells were grown on 24 mm coverslip and transfected with plasmid encoding plasmid for cytosolic-localized GCaMP6s or the perilysosomal-localized ML1-GCaMP3 calcium probes. After 24 or 48 hours, coverslips were placed in 1 mL of complete L-15 medium and imaging was performed on a Zeiss Axiovert 200 microscope equipped with a 40x/1.3 N.A. NeoFluar objective. Excitation was performed with a Deltaram V high speed monochromator (Photon Technology International) equipped with a 75W Xenon Arc lamp. Images were captured with a high sensitivity Evolve 512 Delta EMCCD (Photometrics). The system is controlled by Metamorph 7.5 and was assembled by Crisel Instruments. In order to be truly quantitative, we took advantage of the isosbestic point in GCaMP excitation spectrum: we experimentally determined in living cells that exciting GCaMP at 410 nm leads to fluorescence emission which is not Ca2+-dependent. As a consequence, the ratio between 474 and 410 nm excitation wavelengths is proportional to [Ca2+] while independent on probe expression. HeLa cells were thus alternatively illuminated at 474 and 410 nm and fluorescence was collected through a 515/30 nm bandpass filter (Semrock). For GCaMP6s experiments, exposure time was set to 50 ms at 474 nm and to 150 ms at 410 nm, in order to account for the low quantum yield at the latter wavelength. For the ML1-GCaMP3 experiments, exposure time was set to 300 ms at 474 nm and to 400 ms at 410 nm. Each field was acquired for 20 minutes (1 frame/s) under continuous perfusion with the indicated solutions. Analysis was performed with the Fiji distribution of ImageJ. Both images were background corrected frame by frame by subtracting mean pixel values of a cell-free ROI. Data are presented as the mean ± S.E.Aequorin measurementsFor the measurements of [Ca2+]cyt, HeLa cells grown on 13 mm round glass coverslips at 50% confluence were transfected with the cytosolic (cytAeq) probe. The coverslip with the cells was incubated at 37°C with 5 µM coelenterazine for 1-2 hours in L-15 medium (Life Technologies) supplemented with 10% FBS, and then transferred to the perfusion chamber. Aequorin measurements were carried out in either complete L-15 or HBSS (supplemented with CaCl2 and MgCl2, Life Technologies), as indicated. Agonists and other drugs were added to the same medium, as specified in the text. The experiments were terminated by lysing the cells with 100µM digitonin in a hypotonic Ca2+-rich solution (10 mM CaCl2 in H2O), thus discharging the remaining aequorin pool. The light signal was collected and calibrated into [Ca2+] values by an algorithm based on the Ca2+ response curve of aequorin at physiological conditions of pH, [Mg2+] and ionic strength, as previously described56. Data are presented as mean ± s.d. of the histamine-induced peak in [Ca2+].Animals and in vivo transfection experimentsAnimals were handled by specialized personnel under the control of inspectors of the Veterinary Service of the Local Sanitary Service (ASL 16 - Padova), the local officers of the Ministry of Health. All procedures are specified in the projects approved by the Italian Ministero Salute, Ufficio VI (authorization numbers C65). All experiments were performed on adult 2 month-old male CD1 mice (three mice per group, 28–32 g). In vivo transfection experiments were performed by intramuscular injection of expression plasmids in TA muscle followed by electroporation as previously described57. After 8 days from transfection mice were exercised on a treadmill. Mice performed concentric exercise on a treadmill (LE 8710 Panlab Technology 2B, Biological Instruments), with 10 degrees incline, till they were exhausted and then were sacrificed.Histology and Fluorescence MicroscopyImmunofluorescence staining was performed on cryosections as previously described24 and then monitored with a fluorescence microscope. For nuclear localization studies, cyosections were stained with mouse-anti-Dystrophin-1 IgG2a antibodies (1:200; Novocastra) and DAPI to identify subsarcolemmal position of myonuclei.Generation of the lysosomal-autophagic gene listThe complete gene list includes 322 members (Supplementary Table_S1): 98 lysosomal genes described previously41, 58; 9 novel genes co-localizing with the lysosomal marker LAMP258; the master regulator of lysosomal biogenesis and autophagy, TFEB; all the known lysosomal and not lysosomal protein coding genes with a role in the different LSDs selected by using the Online Mendelian Inheritance in Man (OMIM)59, 60; a subset of lysosomal and autophagy genes predicted by different bioinformatics tools. In details, three different bioinformatics tools were used: AmiGO61, Netview62 and uniPROT63.Microarray data processingThe data discussed in this publication have been deposited in NCBIs Gene Expression Omnibus (GEO)64 and are accessible through GEO Series accession numbers GSE56671 (PPP3R1 WT Starved vs Normal medium) and GSE56672 (PPP3R1−/− Starved vs Normal medium). Statistical comparisons were made using analysis of variance (ANOVA). As threshold for statistically significance we used FDR< 0.05 followed by an additional threshold (p-value < 0.05).Data analysisData are presented as the mean ± s.d. t-test statistical comparisons were made using analysis of variance. A p-value < 0.05 was considered statistically significant. No statistical method was used to predetermine sample size. The experiments were not randomized. The investigators were not blinded to allocation during experiments.

Drugs and cellular treatmentsThe following drugs were used: Torin 1 (1µM, otherwise indicated) from Tocris Bioscience; FK506 (5 µM, otherwise indicated), cyclosporine A (10 µM, otherwise indicated), Chloroquine (100 µM) thapsigargin (300 nM), ionomycin (1µM), and EGTA from Sigma; BAPTA-AM (5-25 µM) from Invitrogen; EGTA-AM (5 µM) from Santa Cruz Biotechnology; Bafilomycin A1 (300 nM); SF51 (200 µM) from VITAS-M Laboratory, LTD. Starvation medium was HBSS, 10 mM Hepes.

Drugs and cellular treatments

The following drugs were used: Torin 1 (1µM, otherwise indicated) from Tocris Bioscience; FK506 (5 µM, otherwise indicated), cyclosporine A (10 µM, otherwise indicated), Chloroquine (100 µM) thapsigargin (300 nM), ionomycin (1µM), and EGTA from Sigma; BAPTA-AM (5-25 µM) from Invitrogen; EGTA-AM (5 µM) from Santa Cruz Biotechnology; Bafilomycin A1 (300 nM); SF51 (200 µM) from VITAS-M Laboratory, LTD. Starvation medium was HBSS, 10 mM Hepes.

Cell culture, plasmids and siRNA transfectionHeLa and MCF-7, HEK 293, RPE cells were purchased from ATCC and cultured in DMEM or RPMI 1640 media supplemented with 10% fetal bovine serum, 200 µM L-glutamine, 100 µM sodium pyruvate, 5% CO2 at 37 degrees. Cells were silenced with TFEB siRNA oligonucleotides (Thermo, Dhamacon D-009798-03), PPP3CB siRNA oligonucleotides (Ambion, 4390824), PPP3R1 siRNA oligonucleotides (Invitrogen, HSS108410-3, HSS183046-3, HSS108411-3), or non-targeting siRNAs (Thermo, D-001810-10-05), by direct or reverse transfection, using lipofectamine RNAiMAX reagent (Invitrogen) according to the protocol from manufacturer. siRNA-transfected cells were collected after 72 hours, if not otherwise stated.Immortalized MEFs from conditional PPP3R1flox/flox (PPP3R1+/+) mice, kind gift from J. Molkentin (Cincinnati Children's Hospital Medical Center, USA) were cultured in DMEM, 10% CS, 200 µM L-glutamine, 100µM sodium pyruvate. PPP3R1flox/flox (PPP3R1+/+) MEFs were used as control, while lentiviral-mediated Cre transduction was used to generate PPP3R1−/− MEFs30.Stably HeLashMCOLN1 cell line was described in Medina et al, 201131. Stably HeLaTFEB-GFP cell line was described in Settembre et al, 20129. ML-IV human fibroblasts were a kind gift of G.Borsani.Human full-length TFEB-GFP, and TFEB-FLAG were previously described7. Human full length PPP3CB and MCOLN1 tagged with myc-FLAG were from OriGene. The plasmid caryinng a constitutively active form of human calcineurin catalytic subunit (ΔCan) was donated by B. Rothermel (The University of Texas Southwestern Medical Center, USA). NFATGFP was a gift from T. Baldari52; Micu1 plasmid was described in34. Cells were transfected using Lipofectamine LTX – Plus reagent (Invitrogen) or Trans-IT reagent (Mirus-Bio) according to the protocol from manufacturers.

Cell culture, plasmids and siRNA transfection

HeLa and MCF-7, HEK 293, RPE cells were purchased from ATCC and cultured in DMEM or RPMI 1640 media supplemented with 10% fetal bovine serum, 200 µM L-glutamine, 100 µM sodium pyruvate, 5% CO2 at 37 degrees. Cells were silenced with TFEB siRNA oligonucleotides (Thermo, Dhamacon D-009798-03), PPP3CB siRNA oligonucleotides (Ambion, 4390824), PPP3R1 siRNA oligonucleotides (Invitrogen, HSS108410-3, HSS183046-3, HSS108411-3), or non-targeting siRNAs (Thermo, D-001810-10-05), by direct or reverse transfection, using lipofectamine RNAiMAX reagent (Invitrogen) according to the protocol from manufacturer. siRNA-transfected cells were collected after 72 hours, if not otherwise stated.

Immortalized MEFs from conditional PPP3R1flox/flox (PPP3R1+/+) mice, kind gift from J. Molkentin (Cincinnati Children's Hospital Medical Center, USA) were cultured in DMEM, 10% CS, 200 µM L-glutamine, 100µM sodium pyruvate. PPP3R1flox/flox (PPP3R1+/+) MEFs were used as control, while lentiviral-mediated Cre transduction was used to generate PPP3R1−/− MEFs30.

Stably HeLashMCOLN1 cell line was described in Medina et al, 201131. Stably HeLaTFEB-GFP cell line was described in Settembre et al, 20129. ML-IV human fibroblasts were a kind gift of G.Borsani.

Human full-length TFEB-GFP, and TFEB-FLAG were previously described7. Human full length PPP3CB and MCOLN1 tagged with myc-FLAG were from OriGene. The plasmid caryinng a constitutively active form of human calcineurin catalytic subunit (ΔCan) was donated by B. Rothermel (The University of Texas Southwestern Medical Center, USA). NFATGFP was a gift from T. Baldari52; Micu1 plasmid was described in34. Cells were transfected using Lipofectamine LTX – Plus reagent (Invitrogen) or Trans-IT reagent (Mirus-Bio) according to the protocol from manufacturers.

Antibodies and Western blottingFor western blot the following antibodies were used: anti-FLAG (M2, cat. F1804, 1:2000), β-actin (AC-15, cat. A5441, 1:4000), β-tubulin (TUB 2.1, cat.T4026, 1:2000) and -ULK (cat. A7481, 1:1000) from Sigma-Aldrich; anti-LC3 (cat. NB100-2220, 1:1000) from Novus Biologicals; anti-PARP1 (cat. ALX-10-302, 1:4000) from ENZO Life Sciences; anti-TFEB (cat. 4240, 1:1000), -p-Ser 14-3-3 binding motif (cat. 9601, 1:1000), -p-P70S6K (cat. 108D2, 1:1000), -p-ULK (cat. 6888, 1:1000) from Cell Signaling Technologies; anti-HA (16B12, cat. MMS-101P, 1:7500) from Covance; anti-P70S6K (cat. KAP-CC035, 1:1000) from Stressgen; anti-PPP3CB (cat. TA308438, 1:1000) from OriGene; anti-pan 14-3-3 (B11, cat. sc-133232, 1:1000), -GAPDH (6C5, cat. sc-32233, 1:10000) and -c-Myc (9E10, cat. sc-40, 1:1000) from Santa Cruz Biotechnology; anti-GFP (1:10000) was provided by M. A. De Matteis (TIGEM); anti-p-Ser142-TFEB (1:1000) was produced as described previously9.Total cell lysates were prepared by solubilisation in RIPA buffer (1% Triton X-100, 1% sodium deoxycholate, 0.1% SDS, 0.15 M NaCl and 0.01 M sodium phosphate, at pH 7.2) containing protease (Roche) and phosphatase (Sigma) inhibitors. The soluble fraction was recovered in the supernatant after centrifugation at 12,000g for 15 min, and protein concentration was measured by the Bradford method. Nuclear/cytosolic fractions were isolated as previously described9.After SDS–PAGE and immunoblotting, the proteins recognized by the specific antibodies were visualized by chemiluminescence methods (ECL Western Blotting Substrate, Pierce) using peroxidase-conjugated anti-rabbit or anti-mouse secondary antibodies (Calbiochem). Membranes were developed using a Chemidoc UVP imaging system (Ultra-Violet Products Ltd) and densitometric quantification was performed in unsaturated images using ImageJ (NIH).

Antibodies and Western blotting

For western blot the following antibodies were used: anti-FLAG (M2, cat. F1804, 1:2000), β-actin (AC-15, cat. A5441, 1:4000), β-tubulin (TUB 2.1, cat.T4026, 1:2000) and -ULK (cat. A7481, 1:1000) from Sigma-Aldrich; anti-LC3 (cat. NB100-2220, 1:1000) from Novus Biologicals; anti-PARP1 (cat. ALX-10-302, 1:4000) from ENZO Life Sciences; anti-TFEB (cat. 4240, 1:1000), -p-Ser 14-3-3 binding motif (cat. 9601, 1:1000), -p-P70S6K (cat. 108D2, 1:1000), -p-ULK (cat. 6888, 1:1000) from Cell Signaling Technologies; anti-HA (16B12, cat. MMS-101P, 1:7500) from Covance; anti-P70S6K (cat. KAP-CC035, 1:1000) from Stressgen; anti-PPP3CB (cat. TA308438, 1:1000) from OriGene; anti-pan 14-3-3 (B11, cat. sc-133232, 1:1000), -GAPDH (6C5, cat. sc-32233, 1:10000) and -c-Myc (9E10, cat. sc-40, 1:1000) from Santa Cruz Biotechnology; anti-GFP (1:10000) was provided by M. A. De Matteis (TIGEM); anti-p-Ser142-TFEB (1:1000) was produced as described previously9.

Total cell lysates were prepared by solubilisation in RIPA buffer (1% Triton X-100, 1% sodium deoxycholate, 0.1% SDS, 0.15 M NaCl and 0.01 M sodium phosphate, at pH 7.2) containing protease (Roche) and phosphatase (Sigma) inhibitors. The soluble fraction was recovered in the supernatant after centrifugation at 12,000g for 15 min, and protein concentration was measured by the Bradford method. Nuclear/cytosolic fractions were isolated as previously described9.

After SDS–PAGE and immunoblotting, the proteins recognized by the specific antibodies were visualized by chemiluminescence methods (ECL Western Blotting Substrate, Pierce) using peroxidase-conjugated anti-rabbit or anti-mouse secondary antibodies (Calbiochem). Membranes were developed using a Chemidoc UVP imaging system (Ultra-Violet Products Ltd) and densitometric quantification was performed in unsaturated images using ImageJ (NIH).

Proximity Ligation AssayPLA assay was performed using the Duolink in situ reagents (Olink Biosciences) according to the manufacturer’s specifications.Golgin 97 polyclonal antibody used for immunofluorescence was raised in rabbit by immunization with recombinant glutathione S-transferase (GST)-fusion full-length protein. The antibody was affinity purified over a column conjugated with a His fusion of the protein. Other antibodies used in this study were from the following sources: mAb TFEB (17, cat. sc-101532, 1:50) from Santa Cruz Biotechnology and anti- mTOR (cat. 7010, 1:200) from Cell Signaling Technology. Alexa-conjugated secondary antibodies and ER tracker were from Life Technologies.Immunofluorescence experiments were performed as previously described53. Briefly, HeLa cells were fixed in 4% paraformaldehyde (PFA) for 10 minutes and permeabilized in 0.1% (w/v) saponin, 0.5% (w/v) BSA and 50 mM NH4Cl in PBS (blocking buffer). Cells were incubated with the indicated primary antibodies for 1 hour and subsequently incubated with secondary antibodies. For confocal imaging, the samples were examined under a Zeiss LSM 700 confocal microscope. Optical sections were obtained under a ×63 immersion objective at a definition of 1024 × 1024 pixels (average of eight or sixteen scans), adjusting the pinhole diameter to 1 Airy unit for each emission channel to have all the intensity values between 1 and 254 (linear range). For image analyses, Mock treated and TFEB-KD HeLa cells were fixed and stained as described above and cells acquired with the same laser parameters using the same magnification.

Proximity Ligation Assay

PLA assay was performed using the Duolink in situ reagents (Olink Biosciences) according to the manufacturer’s specifications.

Golgin 97 polyclonal antibody used for immunofluorescence was raised in rabbit by immunization with recombinant glutathione S-transferase (GST)-fusion full-length protein. The antibody was affinity purified over a column conjugated with a His fusion of the protein. Other antibodies used in this study were from the following sources: mAb TFEB (17, cat. sc-101532, 1:50) from Santa Cruz Biotechnology and anti- mTOR (cat. 7010, 1:200) from Cell Signaling Technology. Alexa-conjugated secondary antibodies and ER tracker were from Life Technologies.

Immunofluorescence experiments were performed as previously described53. Briefly, HeLa cells were fixed in 4% paraformaldehyde (PFA) for 10 minutes and permeabilized in 0.1% (w/v) saponin, 0.5% (w/v) BSA and 50 mM NH4Cl in PBS (blocking buffer). Cells were incubated with the indicated primary antibodies for 1 hour and subsequently incubated with secondary antibodies. For confocal imaging, the samples were examined under a Zeiss LSM 700 confocal microscope. Optical sections were obtained under a ×63 immersion objective at a definition of 1024 × 1024 pixels (average of eight or sixteen scans), adjusting the pinhole diameter to 1 Airy unit for each emission channel to have all the intensity values between 1 and 254 (linear range). For image analyses, Mock treated and TFEB-KD HeLa cells were fixed and stained as described above and cells acquired with the same laser parameters using the same magnification.

Co-immunoprecipitation assaysHeLa cells expressing the indicated tagged proteins were rinsed twice with ice-cold PBS and lysed in ice-cold lysis buffer (400 mM NaCl, 25 mM Tris-HCl, pH 7.4, 1 mM EDTA and 1% Triton X-100) containing protease and phosphatase inhibitors. The soluble fractions from cell lysates were isolated by centrifugation at 12000g for 15 min in a microfuge. For co-immunoprecipitations, each lysate was incubated with corresponding antibody in binding buffer (200 mM NaCl, 25 mM Tris-HCl, pH 7.4, 1mM EDTA) with constant rotation over-night at 4°C. Then, 40 µl of a 50% slurry of Protein-G beads (Sigma-Aldrich) were added to lysates and incubated with rotation for additional 2h at 4°C. After incubation, the resins were washed 5 times in binding buffer and the samples were eluted in Laemmli buffer and separated by SDS-PAGE.

Co-immunoprecipitation assays

HeLa cells expressing the indicated tagged proteins were rinsed twice with ice-cold PBS and lysed in ice-cold lysis buffer (400 mM NaCl, 25 mM Tris-HCl, pH 7.4, 1 mM EDTA and 1% Triton X-100) containing protease and phosphatase inhibitors. The soluble fractions from cell lysates were isolated by centrifugation at 12000g for 15 min in a microfuge. For co-immunoprecipitations, each lysate was incubated with corresponding antibody in binding buffer (200 mM NaCl, 25 mM Tris-HCl, pH 7.4, 1mM EDTA) with constant rotation over-night at 4°C. Then, 40 µl of a 50% slurry of Protein-G beads (Sigma-Aldrich) were added to lysates and incubated with rotation for additional 2h at 4°C. After incubation, the resins were washed 5 times in binding buffer and the samples were eluted in Laemmli buffer and separated by SDS-PAGE.

Histidine pull-down assayPurified His-TFEB (300 nM), CaN (ProAlt) and CaM (Enzo Life Sciences) and GST (100 nM each) were incubated in binding buffer (20 mM Tris, pH 8; 100 mM NaCl; 2mM CaCl2; 6 mM MgCl2; 0,2% Triton X-100; 30 mM imidazole; protease and phosphatase inhibitor) over-night at 4°C. His-TFEB was pulled-down by adding 20 µl of Ni-NTA resin (Qiagen) for 2 hrs at 4°C. After incubation, the resins were washed 5 times in binding buffer and the samples were eluted in Laemmli buffer and separated by SDS-PAGE.

Histidine pull-down assay

Purified His-TFEB (300 nM), CaN (ProAlt) and CaM (Enzo Life Sciences) and GST (100 nM each) were incubated in binding buffer (20 mM Tris, pH 8; 100 mM NaCl; 2mM CaCl2; 6 mM MgCl2; 0,2% Triton X-100; 30 mM imidazole; protease and phosphatase inhibitor) over-night at 4°C. His-TFEB was pulled-down by adding 20 µl of Ni-NTA resin (Qiagen) for 2 hrs at 4°C. After incubation, the resins were washed 5 times in binding buffer and the samples were eluted in Laemmli buffer and separated by SDS-PAGE.

In vitro calcineurin phosphatase assayPhosphorylated FLAG-TFEB was obtained by immunoprecipitation from fed HeLa cells. Protein-G beads containing FLAG-TFEB were incubated for 40 min at 30°C in 24 µl of corresponding phosphatase reaction buffer (20 mM HEPES, pH 7.5, 10 mM MgCl2, 2 mM DTT, protease inhibitors with 1 mM CaCl2 or 10 mM EGTA) containing purified Calcineurin (100U; ENZO Life Sciences) and Calmodulin (4 mM; ENZO Life Sciences). Samples were eluted by adding 8 µl of 4X Laemmli buffer, resolved by SDS–PAGE and analysed by immunoblotting. Antibodies anti-p-Ser 14-3-3 binding motif10, anti-P-Ser142-TFEB9 and TFEB (Cell Signaling Technologies) were used to detect phosphorylated and total TFEB forms, respectively.

In vitro calcineurin phosphatase assay

Phosphorylated FLAG-TFEB was obtained by immunoprecipitation from fed HeLa cells. Protein-G beads containing FLAG-TFEB were incubated for 40 min at 30°C in 24 µl of corresponding phosphatase reaction buffer (20 mM HEPES, pH 7.5, 10 mM MgCl2, 2 mM DTT, protease inhibitors with 1 mM CaCl2 or 10 mM EGTA) containing purified Calcineurin (100U; ENZO Life Sciences) and Calmodulin (4 mM; ENZO Life Sciences). Samples were eluted by adding 8 µl of 4X Laemmli buffer, resolved by SDS–PAGE and analysed by immunoblotting. Antibodies anti-p-Ser 14-3-3 binding motif10, anti-P-Ser142-TFEB9 and TFEB (Cell Signaling Technologies) were used to detect phosphorylated and total TFEB forms, respectively.

RNA extraction and quantitative PCRTotal RNA was extracted from cells using RNeasy Plus Mini Kit (Qiagen). Reverse transcription was performed using QuantiTect Rev Transcription Kit (Qiagen). Real-time quantitative Reverse Transcription PCR (qPCR) was performed using the LightCycler® System 2.0 (Roche Applied Science). HPRT was used for qRT-PCR as reference gene. The parameters of real-time qRT-PCR amplification were according to Roche recommendations.The following primers were used in this study. Mouse primers: PGC1a; forward (fw) gaatcaagccactacagacaccg, reverse (rev) catccctcttgagcctttcgtg, MCOLN1; fw: gcgcctatgacaccatcaa, rev: tatcctggactgctcgat; ATP6V0D1; fw: gcatctcagagcaggaccttga, rev: ggataggacacatggcatcagc, ATP6V1H; fw: gttgctgctcacgatgttggag, rev: tgtagcgaacctgctggtcttc, CTSF; fw: acgcctatgcagccataaag, rev: cttttgccatctgtgctgag, CTSB; fw: ttagcgctctcacttccactacc, rev: tgcttgctaccttcctctggtta, TPP1; fw: aagccaggcctacatagtcaga, rev: ccaagtgcttcctgcagtttaga, DPP7; fw: cgccagcaatactgtctggatac, rev: aaatgatgttgctggctgcttta, WIPI; fw: acaggacgcctgaacttctc, rev: cggcagtttctggatcgt, TcFEB; fw: gcgagagctaacagatgctga, rev: ccggtcattgatgttgaacc, HPRT; fw: caagcttgctggtgaaaagg rev: gtcaagggcatatccaacaac, PPP3R1; fw: tgcctgagttacagcagaacc rev: tcgcctttgacactgaactg, ATG14; fw: gcttcgaaggtcacacatcc, rev: cttgaggtcatggcactgtc, PPP3CA; fw; atcccaagttgtcgacgacc, rev: acactttcttccagcctgcc.
Human primers; PGC1a; fw: catgcaaatcacaatcacagg, rev: ttgtggcttttgctgttgac, MCOLN1; fw: gagtgggtgcgacaagtttc, rev: tgttctcttcccggaatgtc, ATP6V0E1; fw: cattgtgatgagcgtgttctgg, rev: aactccccggttaggaccctta, ATP6V1H; fw: ggaagtgtcagatgatcccca, rev: ccgtttgcctcgtggataat, CTSF; fw: acagaggaggagttccgcacta, rev: gcttgcttcatcttgttgcca, TFEB; fw: caaggccaatgacctggac, rev: agctccctggacttttgcag, HPRT; fw: tggcgtcgtgattagtgatg, rev: aacaccctttccaaatcctca, CTSD; fw: cttcgacaacctgatgcagc, rev: tacttggagtctgtgccacc, PPP3CA; fw: gctgccctgatgaaccaac, rev: gcaggtggttctttgaatcgg, DPP7; fw: gattcggaggaacctgagtg, rev: cggaagcaggatcttctgg, CTSB; fw: agtggagaatggcacacccta , rev: aagagccattgtcacccca, TPP1; fw: gatcccagctctcctcaatac, rev: gccatttttgcaccgtgtg, NEU1; fw: tgaagtgtttgcccctggac, rev: aggcaccatgatcatcgctg.

RNA extraction and quantitative PCR

Total RNA was extracted from cells using RNeasy Plus Mini Kit (Qiagen). Reverse transcription was performed using QuantiTect Rev Transcription Kit (Qiagen). Real-time quantitative Reverse Transcription PCR (qPCR) was performed using the LightCycler® System 2.0 (Roche Applied Science). HPRT was used for qRT-PCR as reference gene. The parameters of real-time qRT-PCR amplification were according to Roche recommendations.

The following primers were used in this study. Mouse primers: PGC1a; forward (fw) gaatcaagccactacagacaccg, reverse (rev) catccctcttgagcctttcgtg, MCOLN1; fw: gcgcctatgacaccatcaa, rev: tatcctggactgctcgat; ATP6V0D1; fw: gcatctcagagcaggaccttga, rev: ggataggacacatggcatcagc, ATP6V1H; fw: gttgctgctcacgatgttggag, rev: tgtagcgaacctgctggtcttc, CTSF; fw: acgcctatgcagccataaag, rev: cttttgccatctgtgctgag, CTSB; fw: ttagcgctctcacttccactacc, rev: tgcttgctaccttcctctggtta, TPP1; fw: aagccaggcctacatagtcaga, rev: ccaagtgcttcctgcagtttaga, DPP7; fw: cgccagcaatactgtctggatac, rev: aaatgatgttgctggctgcttta, WIPI; fw: acaggacgcctgaacttctc, rev: cggcagtttctggatcgt, TcFEB; fw: gcgagagctaacagatgctga, rev: ccggtcattgatgttgaacc, HPRT; fw: caagcttgctggtgaaaagg rev: gtcaagggcatatccaacaac, PPP3R1; fw: tgcctgagttacagcagaacc rev: tcgcctttgacactgaactg, ATG14; fw: gcttcgaaggtcacacatcc, rev: cttgaggtcatggcactgtc, PPP3CA; fw; atcccaagttgtcgacgacc, rev: acactttcttccagcctgcc.
Human primers; PGC1a; fw: catgcaaatcacaatcacagg, rev: ttgtggcttttgctgttgac, MCOLN1; fw: gagtgggtgcgacaagtttc, rev: tgttctcttcccggaatgtc, ATP6V0E1; fw: cattgtgatgagcgtgttctgg, rev: aactccccggttaggaccctta, ATP6V1H; fw: ggaagtgtcagatgatcccca, rev: ccgtttgcctcgtggataat, CTSF; fw: acagaggaggagttccgcacta, rev: gcttgcttcatcttgttgcca, TFEB; fw: caaggccaatgacctggac, rev: agctccctggacttttgcag, HPRT; fw: tggcgtcgtgattagtgatg, rev: aacaccctttccaaatcctca, CTSD; fw: cttcgacaacctgatgcagc, rev: tacttggagtctgtgccacc, PPP3CA; fw: gctgccctgatgaaccaac, rev: gcaggtggttctttgaatcgg, DPP7; fw: gattcggaggaacctgagtg, rev: cggaagcaggatcttctgg, CTSB; fw: agtggagaatggcacacccta , rev: aagagccattgtcacccca, TPP1; fw: gatcccagctctcctcaatac, rev: gccatttttgcaccgtgtg, NEU1; fw: tgaagtgtttgcccctggac, rev: aggcaccatgatcatcgctg.

High content nuclear translocation assayHeLaTFEBGFP cells were seeded in 96 or 384-well plates, incubated for 12 h, and treated as indicated in the text, washed, fixed, and stained with DAPI. For the acquisition of the images, at least 5 images fields were acquired per well of the 96/384-well plate by using confocal automated microscopy (Opera high content system; Perkin-Elmer). A dedicated script was developed to perform the analysis of TFEB localization on the different images (Harmony and Acapella software; Perkin-Elmer). The script calculates the ratio value resulting from the average intensity of nuclear TFEB–GFP fluorescence divided by the average of the cytosolic intensity of TFEB–GFP fluorescence. The results were normalized using negative (RPMI medium, or otherwise indicated) and positive (starvation, or otherwise indicated) control samples in the same plate54. At least two independent experiments, and up to 3000 individual cells per treatment from at least two independent wells are routinely analyzed. p-values were calculated on the basis of mean values from independent wells. The data are represented by the percentage of nuclear translocation vs the indicated control using Excel (Microsoft) or Prism software (GraphPad software).The TFEB-GFP nuclear translocation analyses of different population of transfected cells (detected by immunofluorescence of tagged-plasmids) were performed by a modified script that calculates the ratio value resulting from the average intensity of nuclear TFEB–GFP fluorescence divided by the average of the cytosolic intensity of TFEB–GFP fluorescence on both positive-transfected and negative transfected cells.

High content nuclear translocation assay

HeLaTFEBGFP cells were seeded in 96 or 384-well plates, incubated for 12 h, and treated as indicated in the text, washed, fixed, and stained with DAPI. For the acquisition of the images, at least 5 images fields were acquired per well of the 96/384-well plate by using confocal automated microscopy (Opera high content system; Perkin-Elmer). A dedicated script was developed to perform the analysis of TFEB localization on the different images (Harmony and Acapella software; Perkin-Elmer). The script calculates the ratio value resulting from the average intensity of nuclear TFEB–GFP fluorescence divided by the average of the cytosolic intensity of TFEB–GFP fluorescence. The results were normalized using negative (RPMI medium, or otherwise indicated) and positive (starvation, or otherwise indicated) control samples in the same plate54. At least two independent experiments, and up to 3000 individual cells per treatment from at least two independent wells are routinely analyzed. p-values were calculated on the basis of mean values from independent wells. The data are represented by the percentage of nuclear translocation vs the indicated control using Excel (Microsoft) or Prism software (GraphPad software).

The TFEB-GFP nuclear translocation analyses of different population of transfected cells (detected by immunofluorescence of tagged-plasmids) were performed by a modified script that calculates the ratio value resulting from the average intensity of nuclear TFEB–GFP fluorescence divided by the average of the cytosolic intensity of TFEB–GFP fluorescence on both positive-transfected and negative transfected cells.

High content LC3-autophagosome assayCells were seeded in 96-well plates, incubated for 12 h, and treated as indicated in the text, washed, fixed in ice-cold methanol, and immunofluorescence against LC3 (anti-LC3, Novus Biologicals, 1:400) was performed. After washing, the 96-well plate was analysed using confocal automated microscopy (Opera high content system; Perkin-Elmer). A dedicated script was developed to perform the analysis of the number of LC3-puncta on at least 5 different images per well (Harmony software; Perkin-Elmer).

High content LC3-autophagosome assay

Cells were seeded in 96-well plates, incubated for 12 h, and treated as indicated in the text, washed, fixed in ice-cold methanol, and immunofluorescence against LC3 (anti-LC3, Novus Biologicals, 1:400) was performed. After washing, the 96-well plate was analysed using confocal automated microscopy (Opera high content system; Perkin-Elmer). A dedicated script was developed to perform the analysis of the number of LC3-puncta on at least 5 different images per well (Harmony software; Perkin-Elmer).

Time-lapseHeLaTFEBGFP were reverse transfected and plated in Mattek glass bottomed dishes. Time-lapse movies were acquired for 45’. One frame was acquired roughly every 20s with lasers set at 30% power or below. All imaging was performed with a x60 Plan Apo oil immersion lens using a Nikon Eclipse Ti Spinning Disk microscope, and images were annotated and 3D reconstructions made using the NIS Elements 4.20 software.

HeLaTFEBGFP were reverse transfected and plated in Mattek glass bottomed dishes. Time-lapse movies were acquired for 45’. One frame was acquired roughly every 20s with lasers set at 30% power or below. All imaging was performed with a x60 Plan Apo oil immersion lens using a Nikon Eclipse Ti Spinning Disk microscope, and images were annotated and 3D reconstructions made using the NIS Elements 4.20 software.

EGFP-2×FYVE AssayHeLa cells were reverse transfected with PPP3CB/R1 or scramble control siRNA using RNAiMax. 48h later the cells transfected with the GFP-2xFYVE construct55 using Lipofectamine LTX and Plus (Invitrogen), and 24h later serum starved and fixed with 4% PFA in PBS. Imaging was performed with a 63x PlanApochromat NA 1.4 DIC oil immersion objective on an LSM700 confocal microscope (Zeiss). Image analysis (spot count per cell, minimum of 15 cells counted per treatment group) was performed using ImageJ.

EGFP-2×FYVE Assay

HeLa cells were reverse transfected with PPP3CB/R1 or scramble control siRNA using RNAiMax. 48h later the cells transfected with the GFP-2xFYVE construct55 using Lipofectamine LTX and Plus (Invitrogen), and 24h later serum starved and fixed with 4% PFA in PBS. Imaging was performed with a 63x PlanApochromat NA 1.4 DIC oil immersion objective on an LSM700 confocal microscope (Zeiss). Image analysis (spot count per cell, minimum of 15 cells counted per treatment group) was performed using ImageJ.

Endolysosomal electrophysiologyEndolysosomal electrophysiology was performed in isolated endolysosomes using a modified patch-clamp method35–37. Cos-1 cells were transfected using Lipofectamine 2000 (Invitrogen) with TFEB-S211A fused to mCherry. To increase the size of isolated endolysosomes, cells were treated with 1 µM vacuolin-1, a lipid-soluble polycyclic triazine that can selectively increase the size of endosomes and lysosomes. Whole-endolysosome recordings were performed on isolated enlarged endolysosomes. In brief, a patch pipette (electrode) was pressed against a cell and quickly pulled away to slice the cell membrane35–37. Enlarged endolysosomes were released into a dish. TFEB-S211A-positive vacuoles were identified by monitoring mCherry fluorescence. Bath (internal/cytoplasmic) solution contained 140 mM K+-Gluconate, 4 mM NaCl, 1 mM EGTA, 2 mM Na2-ATP, 2 mM MgCl2, 0.39 mM CaCl2, 0.1 mM GTP and 10 mM HEPES (pH adjusted with KOH to 7.2; free [Ca 2+]i approximately 100 nM). The pipette (luminal) solution was pH 4.6 standard extracellular solution (modified Tyrode’s) with 145 mM NaCl, 5 mM KCl, 2 mM CaCl2, 1 mM MgCl2, 20 mM HEPES and 10 mM glucose (pH adjusted with NaOH). All bath solutions were applied by means of a fast perfusion system to achieve a complete solution exchange within a few seconds. Data were collected using an Axopatch 2A patch-clamp amplifier, Digidata 1440, and pClamp 10.0 software (Axon Instruments). Whole-endolysosome currents were digitized at 10 kHz and filtered at 2 kHz. All experiments were conducted at room temperature (21 – 23°C), and all recordings were analysed using pCLAMP10 (Axon Instruments) and Origin 8.0 (OriginLab). ML-SA1 was purchased from Princeton BioMolecular Research Inc.

Endolysosomal electrophysiology

Endolysosomal electrophysiology was performed in isolated endolysosomes using a modified patch-clamp method35–37. Cos-1 cells were transfected using Lipofectamine 2000 (Invitrogen) with TFEB-S211A fused to mCherry. To increase the size of isolated endolysosomes, cells were treated with 1 µM vacuolin-1, a lipid-soluble polycyclic triazine that can selectively increase the size of endosomes and lysosomes. Whole-endolysosome recordings were performed on isolated enlarged endolysosomes. In brief, a patch pipette (electrode) was pressed against a cell and quickly pulled away to slice the cell membrane35–37. Enlarged endolysosomes were released into a dish. TFEB-S211A-positive vacuoles were identified by monitoring mCherry fluorescence. Bath (internal/cytoplasmic) solution contained 140 mM K+-Gluconate, 4 mM NaCl, 1 mM EGTA, 2 mM Na2-ATP, 2 mM MgCl2, 0.39 mM CaCl2, 0.1 mM GTP and 10 mM HEPES (pH adjusted with KOH to 7.2; free [Ca 2+]i approximately 100 nM). The pipette (luminal) solution was pH 4.6 standard extracellular solution (modified Tyrode’s) with 145 mM NaCl, 5 mM KCl, 2 mM CaCl2, 1 mM MgCl2, 20 mM HEPES and 10 mM glucose (pH adjusted with NaOH). All bath solutions were applied by means of a fast perfusion system to achieve a complete solution exchange within a few seconds. Data were collected using an Axopatch 2A patch-clamp amplifier, Digidata 1440, and pClamp 10.0 software (Axon Instruments). Whole-endolysosome currents were digitized at 10 kHz and filtered at 2 kHz. All experiments were conducted at room temperature (21 – 23°C), and all recordings were analysed using pCLAMP10 (Axon Instruments) and Origin 8.0 (OriginLab). ML-SA1 was purchased from Princeton BioMolecular Research Inc.

GCaMP3 Ca2+ imaging in GCaMP3-ML1-HEK293 cellsGCaMP3 Ca2+ imaging was performed in HEK cells that stably express GCaMP3-ML1, which are lysosome-targeted genetically encoded Ca2+ sensors36. The fluorescence intensity at 488 nm (F488) was recorded with the spinning disk confocal imaging system, which consisted of an Olympus IX81 inverted microscope, a 60X or 100X objective (Olympus), a CSU-X1 scanner (Yokogawa), an iXon EM-CCD camera (Andor), and MetaMorph Advanced Imaging acquisition software v.7.7.8.0 (Molecular Devices). Live imaging of ML1-GCaMP3 was performed on a heated stage.

GCaMP3 Ca2+ imaging in GCaMP3-ML1-HEK293 cells

GCaMP3 Ca2+ imaging was performed in HEK cells that stably express GCaMP3-ML1, which are lysosome-targeted genetically encoded Ca2+ sensors36. The fluorescence intensity at 488 nm (F488) was recorded with the spinning disk confocal imaging system, which consisted of an Olympus IX81 inverted microscope, a 60X or 100X objective (Olympus), a CSU-X1 scanner (Yokogawa), an iXon EM-CCD camera (Andor), and MetaMorph Advanced Imaging acquisition software v.7.7.8.0 (Molecular Devices). Live imaging of ML1-GCaMP3 was performed on a heated stage.

Ratiometric GCaMP3 Ca2+ imaging in HeLa cellsHeLa cells were grown on 24 mm coverslip and transfected with plasmid encoding plasmid for cytosolic-localized GCaMP6s or the perilysosomal-localized ML1-GCaMP3 calcium probes. After 24 or 48 hours, coverslips were placed in 1 mL of complete L-15 medium and imaging was performed on a Zeiss Axiovert 200 microscope equipped with a 40x/1.3 N.A. NeoFluar objective. Excitation was performed with a Deltaram V high speed monochromator (Photon Technology International) equipped with a 75W Xenon Arc lamp. Images were captured with a high sensitivity Evolve 512 Delta EMCCD (Photometrics). The system is controlled by Metamorph 7.5 and was assembled by Crisel Instruments. In order to be truly quantitative, we took advantage of the isosbestic point in GCaMP excitation spectrum: we experimentally determined in living cells that exciting GCaMP at 410 nm leads to fluorescence emission which is not Ca2+-dependent. As a consequence, the ratio between 474 and 410 nm excitation wavelengths is proportional to [Ca2+] while independent on probe expression. HeLa cells were thus alternatively illuminated at 474 and 410 nm and fluorescence was collected through a 515/30 nm bandpass filter (Semrock). For GCaMP6s experiments, exposure time was set to 50 ms at 474 nm and to 150 ms at 410 nm, in order to account for the low quantum yield at the latter wavelength. For the ML1-GCaMP3 experiments, exposure time was set to 300 ms at 474 nm and to 400 ms at 410 nm. Each field was acquired for 20 minutes (1 frame/s) under continuous perfusion with the indicated solutions. Analysis was performed with the Fiji distribution of ImageJ. Both images were background corrected frame by frame by subtracting mean pixel values of a cell-free ROI. Data are presented as the mean ± S.E.

Ratiometric GCaMP3 Ca2+ imaging in HeLa cells

HeLa cells were grown on 24 mm coverslip and transfected with plasmid encoding plasmid for cytosolic-localized GCaMP6s or the perilysosomal-localized ML1-GCaMP3 calcium probes. After 24 or 48 hours, coverslips were placed in 1 mL of complete L-15 medium and imaging was performed on a Zeiss Axiovert 200 microscope equipped with a 40x/1.3 N.A. NeoFluar objective. Excitation was performed with a Deltaram V high speed monochromator (Photon Technology International) equipped with a 75W Xenon Arc lamp. Images were captured with a high sensitivity Evolve 512 Delta EMCCD (Photometrics). The system is controlled by Metamorph 7.5 and was assembled by Crisel Instruments. In order to be truly quantitative, we took advantage of the isosbestic point in GCaMP excitation spectrum: we experimentally determined in living cells that exciting GCaMP at 410 nm leads to fluorescence emission which is not Ca2+-dependent. As a consequence, the ratio between 474 and 410 nm excitation wavelengths is proportional to [Ca2+] while independent on probe expression. HeLa cells were thus alternatively illuminated at 474 and 410 nm and fluorescence was collected through a 515/30 nm bandpass filter (Semrock). For GCaMP6s experiments, exposure time was set to 50 ms at 474 nm and to 150 ms at 410 nm, in order to account for the low quantum yield at the latter wavelength. For the ML1-GCaMP3 experiments, exposure time was set to 300 ms at 474 nm and to 400 ms at 410 nm. Each field was acquired for 20 minutes (1 frame/s) under continuous perfusion with the indicated solutions. Analysis was performed with the Fiji distribution of ImageJ. Both images were background corrected frame by frame by subtracting mean pixel values of a cell-free ROI. Data are presented as the mean ± S.E.

Aequorin measurementsFor the measurements of [Ca2+]cyt, HeLa cells grown on 13 mm round glass coverslips at 50% confluence were transfected with the cytosolic (cytAeq) probe. The coverslip with the cells was incubated at 37°C with 5 µM coelenterazine for 1-2 hours in L-15 medium (Life Technologies) supplemented with 10% FBS, and then transferred to the perfusion chamber. Aequorin measurements were carried out in either complete L-15 or HBSS (supplemented with CaCl2 and MgCl2, Life Technologies), as indicated. Agonists and other drugs were added to the same medium, as specified in the text. The experiments were terminated by lysing the cells with 100µM digitonin in a hypotonic Ca2+-rich solution (10 mM CaCl2 in H2O), thus discharging the remaining aequorin pool. The light signal was collected and calibrated into [Ca2+] values by an algorithm based on the Ca2+ response curve of aequorin at physiological conditions of pH, [Mg2+] and ionic strength, as previously described56. Data are presented as mean ± s.d. of the histamine-induced peak in [Ca2+].

Aequorin measurements

For the measurements of [Ca2+]cyt, HeLa cells grown on 13 mm round glass coverslips at 50% confluence were transfected with the cytosolic (cytAeq) probe. The coverslip with the cells was incubated at 37°C with 5 µM coelenterazine for 1-2 hours in L-15 medium (Life Technologies) supplemented with 10% FBS, and then transferred to the perfusion chamber. Aequorin measurements were carried out in either complete L-15 or HBSS (supplemented with CaCl2 and MgCl2, Life Technologies), as indicated. Agonists and other drugs were added to the same medium, as specified in the text. The experiments were terminated by lysing the cells with 100µM digitonin in a hypotonic Ca2+-rich solution (10 mM CaCl2 in H2O), thus discharging the remaining aequorin pool. The light signal was collected and calibrated into [Ca2+] values by an algorithm based on the Ca2+ response curve of aequorin at physiological conditions of pH, [Mg2+] and ionic strength, as previously described56. Data are presented as mean ± s.d. of the histamine-induced peak in [Ca2+].

Animals and in vivo transfection experimentsAnimals were handled by specialized personnel under the control of inspectors of the Veterinary Service of the Local Sanitary Service (ASL 16 - Padova), the local officers of the Ministry of Health. All procedures are specified in the projects approved by the Italian Ministero Salute, Ufficio VI (authorization numbers C65). All experiments were performed on adult 2 month-old male CD1 mice (three mice per group, 28–32 g). In vivo transfection experiments were performed by intramuscular injection of expression plasmids in TA muscle followed by electroporation as previously described57. After 8 days from transfection mice were exercised on a treadmill. Mice performed concentric exercise on a treadmill (LE 8710 Panlab Technology 2B, Biological Instruments), with 10 degrees incline, till they were exhausted and then were sacrificed.

Animals and in vivo transfection experiments

Animals were handled by specialized personnel under the control of inspectors of the Veterinary Service of the Local Sanitary Service (ASL 16 - Padova), the local officers of the Ministry of Health. All procedures are specified in the projects approved by the Italian Ministero Salute, Ufficio VI (authorization numbers C65). All experiments were performed on adult 2 month-old male CD1 mice (three mice per group, 28–32 g). In vivo transfection experiments were performed by intramuscular injection of expression plasmids in TA muscle followed by electroporation as previously described57. After 8 days from transfection mice were exercised on a treadmill. Mice performed concentric exercise on a treadmill (LE 8710 Panlab Technology 2B, Biological Instruments), with 10 degrees incline, till they were exhausted and then were sacrificed.

Histology and Fluorescence MicroscopyImmunofluorescence staining was performed on cryosections as previously described24 and then monitored with a fluorescence microscope. For nuclear localization studies, cyosections were stained with mouse-anti-Dystrophin-1 IgG2a antibodies (1:200; Novocastra) and DAPI to identify subsarcolemmal position of myonuclei.

Histology and Fluorescence Microscopy

Immunofluorescence staining was performed on cryosections as previously described24 and then monitored with a fluorescence microscope. For nuclear localization studies, cyosections were stained with mouse-anti-Dystrophin-1 IgG2a antibodies (1:200; Novocastra) and DAPI to identify subsarcolemmal position of myonuclei.

Generation of the lysosomal-autophagic gene listThe complete gene list includes 322 members (Supplementary Table_S1): 98 lysosomal genes described previously41, 58; 9 novel genes co-localizing with the lysosomal marker LAMP258; the master regulator of lysosomal biogenesis and autophagy, TFEB; all the known lysosomal and not lysosomal protein coding genes with a role in the different LSDs selected by using the Online Mendelian Inheritance in Man (OMIM)59, 60; a subset of lysosomal and autophagy genes predicted by different bioinformatics tools. In details, three different bioinformatics tools were used: AmiGO61, Netview62 and uniPROT63.

Generation of the lysosomal-autophagic gene list

The complete gene list includes 322 members (Supplementary Table_S1): 98 lysosomal genes described previously41, 58; 9 novel genes co-localizing with the lysosomal marker LAMP258; the master regulator of lysosomal biogenesis and autophagy, TFEB; all the known lysosomal and not lysosomal protein coding genes with a role in the different LSDs selected by using the Online Mendelian Inheritance in Man (OMIM)59, 60; a subset of lysosomal and autophagy genes predicted by different bioinformatics tools. In details, three different bioinformatics tools were used: AmiGO61, Netview62 and uniPROT63.

Microarray data processingThe data discussed in this publication have been deposited in NCBIs Gene Expression Omnibus (GEO)64 and are accessible through GEO Series accession numbers GSE56671 (PPP3R1 WT Starved vs Normal medium) and GSE56672 (PPP3R1−/− Starved vs Normal medium). Statistical comparisons were made using analysis of variance (ANOVA). As threshold for statistically significance we used FDR< 0.05 followed by an additional threshold (p-value < 0.05).

Microarray data processing

The data discussed in this publication have been deposited in NCBIs Gene Expression Omnibus (GEO)64 and are accessible through GEO Series accession numbers GSE56671 (PPP3R1 WT Starved vs Normal medium) and GSE56672 (PPP3R1−/− Starved vs Normal medium). Statistical comparisons were made using analysis of variance (ANOVA). As threshold for statistically significance we used FDR< 0.05 followed by an additional threshold (p-value < 0.05).

Data analysisData are presented as the mean ± s.d. t-test statistical comparisons were made using analysis of variance. A p-value < 0.05 was considered statistically significant. No statistical method was used to predetermine sample size. The experiments were not randomized. The investigators were not blinded to allocation during experiments.

Data analysis

Data are presented as the mean ± s.d. t-test statistical comparisons were made using analysis of variance. A p-value < 0.05 was considered statistically significant. No statistical method was used to predetermine sample size. The experiments were not randomized. The investigators were not blinded to allocation during experiments.

Supplementary Material123456789

Supplementary Material
