---
pmid: '26269953'
title: Posttranslational Modifications of the Master Transcriptional Regulator NPR1
  Enable Dynamic but Tight Control of Plant Immune Responses.
authors:
- Saleh A
- Withers J
- Mohan R
- Marqués J
- Gu Y
- Yan S
- Zavaliev R
- Nomoto M
- Tada Y
- Dong X
journal: Cell Host Microbe
year: '2015'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4537515
doi: 10.1016/j.chom.2015.07.005
---

# Posttranslational Modifications of the Master Transcriptional Regulator NPR1 Enable Dynamic but Tight Control of Plant Immune Responses.
**Authors:** Saleh A, Withers J, Mohan R, Marqués J, Gu Y, Yan S, Zavaliev R, Nomoto M, Tada Y, Dong X
**Journal:** Cell Host Microbe (2015)
**DOI:** [10.1016/j.chom.2015.07.005](https://doi.org/10.1016/j.chom.2015.07.005)
**PMC:** [PMC4537515](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4537515/)

## Abstract

1. Cell Host Microbe. 2015 Aug 12;18(2):169-82. doi: 10.1016/j.chom.2015.07.005.

Posttranslational Modifications of the Master Transcriptional Regulator NPR1 
Enable Dynamic but Tight Control of Plant Immune Responses.

Saleh A(1), Withers J(1), Mohan R(1), Marqués J(1), Gu Y(1), Yan S(1), Zavaliev 
R(1), Nomoto M(2), Tada Y(3), Dong X(4).

Author information:
(1)Howard Hughes Medical Institute-Gordon and Betty Moore Foundation, Department 
of Biology, PO Box 90338, Duke University, Durham, NC 27708, USA.
(2)Division of Biological Science, Graduate School of Science, Nagoya 
University, Chikusa-ku, Nagoya 464-8602, Japan.
(3)Center for Gene Research, Nagoya University, Chikusa-ku, Nagoya 464-8602, 
Japan.
(4)Howard Hughes Medical Institute-Gordon and Betty Moore Foundation, Department 
of Biology, PO Box 90338, Duke University, Durham, NC 27708, USA. Electronic 
address: xdong@duke.edu.

Erratum in
    Cell Host Microbe. 2016 Jan 13;19(1):127-30.

NPR1, a master regulator of basal and systemic acquired resistance in plants, 
confers immunity through a transcriptional cascade, which includes transcription 
activators (e.g., TGA3) and repressors (e.g., WRKY70), leading to the massive 
induction of antimicrobial genes. How this single protein orchestrates 
genome-wide transcriptional reprogramming in response to immune stimulus remains 
a major question. Paradoxically, while NPR1 is essential for defense gene 
induction, its turnover appears to be required for this function, suggesting 
that NPR1 activity and degradation are dynamically regulated. Here we show that 
sumoylation of NPR1 by SUMO3 activates defense gene expression by switching 
NPR1's association with the WRKY transcription repressors to TGA transcription 
activators. Sumoylation also triggers NPR1 degradation, rendering the immune 
induction transient. SUMO modification of NPR1 is inhibited by phosphorylation 
at Ser55/Ser59, which keeps NPR1 stable and quiescent. Thus, posttranslational 
modifications enable dynamic but tight and precise control of plant immune 
responses.

Copyright © 2015 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.chom.2015.07.005
PMCID: PMC4537515
PMID: 26269953 [Indexed for MEDLINE]

## Full Text

Introduction

In plants, pathogen-triggered increases in cellular levels of salicylic acid (SA) and exogenous application of SA both lead to transcription reprogramming and a broad-spectrum defense response known as systemic acquired resistance (SAR) ( Fu and Dong, 2013 ). SAR is predominantly dependent on the activity of NPR1 (nonexpressor of pathogenesis-related ( PR ) genes 1), which was first identified through screens for mutants blocked in SA-mediated PR gene expression and resistance ( Cao et al., 1994 ; Delaney et al., 1995 ; Wang et al., 2006 ). NPR1's central role in plant immunity has been firmly established ( Pieterse et al., 2012 ). Therefore, elucidating its regulatory mechanism is critical for our understanding of plant immunity.

Like the mammalian immune regulator NF-κB, the activity of NPR1 is tightly regulated to ensure proper immune induction with minimal detrimental effects on plant growth. Since NPR1 functions in the nucleus ( Kinkema et al., 2000 ), its activity is regulated in part at the nuclear translocation step controlled by the cellular redox changes triggered by SA ( Mou et al., 2003 ; Tada et al., 2008 ). In the nucleus, NPR1 confers immunity through a transcriptional cascade, including transcription activators (e.g., TGA3) and repressors (e.g., WRKY70), leading to the massive induction of antimicrobial PR genes ( Despres et al., 2000 ; Lebel et al., 1998 ; Spoel et al., 2009 ; Wang et al., 2006 ; Zhang et al., 1999 ; Zhou et al., 2000 ). However, how NPR1 regulates transcription is poorly understood. It has been shown that NPR1 could provide the transactivation activity to the associated TGA transcription factors (TFs) when transiently expressed in plants ( Johnson et al., 2003 ; Rochon et al., 2006 ). The structure of the protein suggests that, like other BTB (bric-a-brac, tramtrack, broad-complex) domain-containing proteins, NPR1 may serve as an adaptor for the CULLIN3 ubiquitin E3 ligase ( Luke-Glaser et al., 2007 ; Petroski and Deshaies, 2005 ; Pintard et al., 2004 ) and be involved in the ubiquitination and possibly the degradation of a transcription repressor. In both scenarios, it is not known whether and how NPR1 interactions with TFs are regulated in plants. In yeast two-hybrid analysis, however, NPR1 has been shown to interact with TGA and NIMIN (NIM1-INTERACTING) TFs constitutively ( Despres et al., 2000 ; Weigel et al., 2001 ; Zhang et al., 1999 ; Zhou et al., 2000 ), with the exception of TGA1 and TGA4 ( Despres et al., 2003 ).

Both posttranslational modifications (PTMs) and protein stability may play a role in controlling NPR1 transcriptional activity. NPR1, while having the structure of an adaptor for the CUL3 E3 ligase complex, is itself regulated by the 26S proteasome in the nucleus ( Spoel et al., 2009 ). Normally, NPR1 is constantly degraded via interaction with the NPR4-CUL3 E3 ligase to reduce the basal level of NPR1. Upon pathogen challenge, NPR1 is phosphorylated at the first IκB-like phosphodegron (Ser11/Ser15), ubiquitinated by the NPR3-CUL3 E3 ligase and degraded ( Fu et al., 2012 ; Spoel et al., 2009 ). Paradoxically, NPR1 turnover appears to be required for its full transcriptional activity in SAR even though it is a positive regulator of defense genes ( Spoel et al., 2009 ). Proteasome-mediated recycling of the transcriptional complexes has been proposed as the underlying mechanism. Alternatively, a PTM that causes NPR1 instability may also be required for its transcriptional activity. However, phosphorylation of NPR1 at Ser11/Ser15 has not been shown to significantly alter its interaction with TGA TFs ( Spoel et al., 2009 ). Therefore, how NPR1 transcriptional activity and degradation are dynamically regulated remains an outstanding question.

Sumoylation is a dynamic and reversible PTM that has not been examined for NPR1 regulation. The SUMO system is conserved in all eukaryotic organisms ( Johnson, 2004 ; Mazur and van den Burg, 2012 ). The process begins with proteolytic cleavage of SUMO at the C-terminal di-glycine motif (GG), activation by SUMO E1 and then transfer to the SUMO E2 conjugating enzyme. Conjugation of SUMO to the lysine residue(s) in the target protein requires either a SUMO E3 ligase or a noncovalent interaction with a SUMO-interaction motif (SIM) ( Johnson, 2004 ; Kerscher, 2007 ; Wang and Dasso, 2009 ). In Arabidopsis , overexpressing the small ubiquitin-like modifier 3 (SUMO3) enhances defense gene expression and disease resistance ( van den Burg et al., 2010 ), but the cellular targets for SUMO3 have yet to be identified. Here we report that NPR1 is sumoylated upon immune induction and this PTM differentially affects NPR1 interactions with positive and negative TFs and its stability. Our study also uncovered an intricate interplay between sumoylation and phosphorylation in controlling NPR1 activity and turnover.

Discussion

Sumoylation has diverse regulatory functions in eukaryotes. For the mammalian immune regulator NF-κB inhibitor (IκB), sumoylation prevents its degradation required for NF-κB activation ( Desterro et al., 1998 ). In Arabidopsis , sumoylation plays a role in DELLA protein stability by sequestering GID1, which is required for degradation of DELLA ( Conti et al., 2014 ). In contrast, sumoylation of human promyelocytic leukemia (PML) facilitates its interactions with many proteins and promotes its degradation ( Lallemand-Breitenbach et al., 2008 ). In this study, we identified SUMO3 as a modifier of NPR1 and showed that NPR1 sumoylation is the final regulatory step required for its transcriptional activity and degradation ( Figures 1 , 2 and 4 - 6 ).

SUMO3 targets NPR1 through binding to SIM3, instead of an E3 ligase ( Figure 1 ). This SIM-dependent sumoylation allows a specific SUMO paralog (e.g., SUMO3 for NPR1) to recognize its substrate ( Wilkinson and Henley, 2010 ). For E3 ligase-independent sumoylation, conjugation may occur at a non-consensus lysine residue as is the case for the human USP25 (ubiquitin specific peptidase 25) and DNA helicase BLM (Bloom syndrome, RecQ helicase-like) proteins ( Wilkinson and Henley, 2010 ). Both of these proteins contain a SIM similar to SIM3 found in NPR1 ( Figure 1B ) ( Meulmeester et al., 2008 ; Zhu et al., 2008 ). We hypothesize that NPR1 is sumoylated at a non-consensus lysine because mutating lysines in the six potential SUMO consensuses did not affect NPR1 interaction with SUMO3 ( Figures S1B and S1C ).

The regulation of NPR1 sumoylation requires further investigation. It is known that the SUMO3 gene expression is induced by SA treatment and pathogen challenge ( van den Burg et al., 2010 ). Moreover, mimicking phosphorylation at Ser55/Ser59 can block NPR1 sumoylation indicating that protein conformation also plays a regulatory role. If sumoylation occurs in the nucleus, NPR1 nuclear translocation may be another rate limiting step for this PTM.

The SIM3 in NPR1 is located in the ankyrin-repeat domain responsible for protein-protein interactions ( Figures 1 , 5 and S1A ). NPR1 modification by SUMO3 may change the conformation of the domain to allow binding to NPR3, NPR4 and TGA3, but block interaction with WRKY70. For NPR3 and NPR4, binding to NPR1 may occur at the SUMO3 moiety via their own SIMs or vice versa. A well-studied example for sumoylation serving as an interaction platform for the recruitment of other SIM-containing proteins is the mammalian PML, which organizes the formation of PML nuclear bodies containing more than 100 proteins ( Shen et al., 2006 ).

The interplay between sumoylation and phosphorylation of NPR1 discovered in this study shed light on the mystery of why degradation appeared to be required for the full activity of NPR1 ( Spoel et al., 2009 ). We found that sumoylation of NPR1 not only facilitates its degradation but is also required for differential interactions with WRKY and TGA TFs. At the PR1 gene promoter, sumoylation of NPR1 is required for SA-triggered dissociation from the inhibitory W-box and for the SA-induced association with the active cis -element as-1 ( Figure 5F ). Which of these two regulatory steps is essential for SA-mediated induction of PR1 will require future investigation. If the function of NPR1 is to provide the transactivation activity to TGA TFs as previously proposed ( Despres et al., 2003 ; Despres et al., 2000 ; Fan and Dong, 2002 ; Johnson et al., 2003 ; Rochon et al., 2006 ), then sumoylation-mediated interaction with TGA is the essential step. However, this does not exclude the possibility that sumoylation of NPR1 is required for the removal of transcription repression by a WRKY TF at the W-box. These two mechanisms are not mutually exclusive.

As summarized in Figure 7 , we hypothesize that at resting state, NPR1 is phosphorylated at Ser55/Ser59, which inhibits sumoylation and promotes interaction with WRKY70 to repress PR1 transcription. At the same time, there must be a residual level of sumoylation responsible for conferring basal defense. SA accumulation promotes dephosphorylation of Ser55/Ser59 through an unknown mechanism and induces sumoylation of NPR1, resulting in dissociation from WRKY70. Modification of NPR1 by SUMO3 is required for phosphorylation at Ser11/Ser15 to form a signal amplification loop to generate more activated NPR1. The activated NPR1 interacts with the TGA3 to induce PR1 gene expression. Subsequently, the modified NPR1 is ubiquitinated and targeted for proteasomal degradation by NPR3. This coupling of protein activation and turnover through sumoylation determines the transient nature of NPR1 activation, which is essential for balancing defense and growth. Our study illustrates how dynamic PTMs enable a single protein to perform its multiple functions through engagements with different partners. The mechanism uncovered in this study may well be extended to other systems.
