---
pmid: '23793029'
title: ANKS6 is a central component of a nephronophthisis module linking NEK8 to INVS
  and NPHP3.
authors:
- Hoff S
- Halbritter J
- Epting D
- Frank V
- Nguyen TM
- van Reeuwijk J
- Boehlke C
- Schell C
- Yasunaga T
- Helmstädter M
- Mergen M
- Filhol E
- Boldt K
- Horn N
- Ueffing M
- Otto EA
- Eisenberger T
- Elting MW
- van Wijk JA
- Bockenhauer D
- Sebire NJ
- Rittig S
- Vyberg M
- Ring T
- Pohl M
- Pape L
- Neuhaus TJ
- Elshakhs NA
- Koon SJ
- Harris PC
- Grahammer F
- Huber TB
- Kuehn EW
- Kramer-Zucker A
- Bolz HJ
- Roepman R
- Saunier S
- Walz G
- Hildebrandt F
- Bergmann C
- Lienkamp SS
journal: Nat Genet
year: '2013'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3786259
doi: 10.1038/ng.2681
---

# ANKS6 is a central component of a nephronophthisis module linking NEK8 to INVS and NPHP3.
**Authors:** Hoff S, Halbritter J, Epting D, Frank V, Nguyen TM, van Reeuwijk J, Boehlke C, Schell C, Yasunaga T, Helmstädter M, Mergen M, Filhol E, Boldt K, Horn N, Ueffing M, Otto EA, Eisenberger T, Elting MW, van Wijk JA, Bockenhauer D, Sebire NJ, Rittig S, Vyberg M, Ring T, Pohl M, Pape L, Neuhaus TJ, Elshakhs NA, Koon SJ, Harris PC, Grahammer F, Huber TB, Kuehn EW, Kramer-Zucker A, Bolz HJ, Roepman R, Saunier S, Walz G, Hildebrandt F, Bergmann C, Lienkamp SS
**Journal:** Nat Genet (2013)
**DOI:** [10.1038/ng.2681](https://doi.org/10.1038/ng.2681)
**PMC:** [PMC3786259](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3786259/)

## Abstract

1. Nat Genet. 2013 Aug;45(8):951-6. doi: 10.1038/ng.2681. Epub 2013 Jun 23.

ANKS6 is a central component of a nephronophthisis module linking NEK8 to INVS 
and NPHP3.

Hoff S(1), Halbritter J, Epting D, Frank V, Nguyen TM, van Reeuwijk J, Boehlke 
C, Schell C, Yasunaga T, Helmstädter M, Mergen M, Filhol E, Boldt K, Horn N, 
Ueffing M, Otto EA, Eisenberger T, Elting MW, van Wijk JA, Bockenhauer D, Sebire 
NJ, Rittig S, Vyberg M, Ring T, Pohl M, Pape L, Neuhaus TJ, Elshakhs NA, Koon 
SJ, Harris PC, Grahammer F, Huber TB, Kuehn EW, Kramer-Zucker A, Bolz HJ, 
Roepman R, Saunier S, Walz G, Hildebrandt F, Bergmann C, Lienkamp SS.

Author information:
(1)Department of Medicine, Renal Division, University of Freiburg Medical 
Center, Freiburg, Germany.

Nephronophthisis is an autosomal recessive cystic kidney disease that leads to 
renal failure in childhood or adolescence. Most NPHP gene products form 
molecular networks. Here we identify ANKS6 as a new NPHP family member that 
connects NEK8 (NPHP9) to INVS (NPHP2) and NPHP3. We show that ANKS6 localizes to 
the proximal cilium and confirm its role in renal development through knockdown 
experiments in zebrafish and Xenopus laevis. We also identify six families with 
ANKS6 mutations affected by nephronophthisis, including severe cardiovascular 
abnormalities, liver fibrosis and situs inversus. The oxygen sensor HIF1AN 
hydroxylates ANKS6 and INVS and alters the composition of the ANKS6-INVS-NPHP3 
module. Knockdown of Hif1an in Xenopus results in a phenotype that resembles 
loss of other NPHP proteins. Network analyses uncovered additional putative NPHP 
proteins and placed ANKS6 at the center of this NPHP module, explaining the 
overlapping disease manifestation caused by mutation in ANKS6, NEK8, INVS or 
NPHP3.

DOI: 10.1038/ng.2681
PMCID: PMC3786259
PMID: 23793029 [Indexed for MEDLINE]

Conflict of interest statement: Competing financial interests The authors 
declare no competing financial interests.

## Full Text

NPH is the most frequent genetic cause of renal failure in children, presenting with cystic kidney disease combined with extrarenal manifestations such as retinitis pigmentosa (Senior-Løken syndrome), liver fibrosis, cerebellar vermis hypoplasia (Joubert-syndrome), situs inversus or cardiac malformations 1,2. Since most NPHP gene products localize to the cilium or its appendages, NPH, the related Joubert-syndrome and Meckel-Gruber syndrome (MKS) have been termed ciliopathies 3. Although more than a dozen causative genes have been identified, a surprisingly large proportion of patients with NPH (approximately 60%) do not have a mutation in any of the known NPHP genes 4. Most NPHPs display domain architectures typical for adaptor molecules involved in protein-protein interactions, and form large protein networks 5,6. Hence, a remaining challenge is the identification of the missing components to understand how these protein complexes exert their developmental and tissue-specific functions. Although NPHP members engage in multiple protein-protein interactions, four distinct sub-networks have been identified, the NPHP1-4-8, the NPHP5-6, the NPHP2-3-9 and the MKS modules 5,6,7. However, how specific complexes are assembled and how the composition of individual complexes is regulated, is currently unknown.

NEK8, a NimA (Never in mitosis A)-related serine-threonine kinase is mutated in NPHP9. INVS recruits NEK8 and NPHP3 to the cilium and has only been shown to interact with NEK8 directly 7,8,9. To obtain insight in the molecular function of NEK8 in NPH, we expressed NEK8 in human embryonic kidney (HEK293T) cells and identified interacting proteins by mass spectrometry (MS) 10. This approach identified ANKS6, a protein containing nine N-terminal ankyrin repeats and a C-terminal sterile alpha motif (SAM), as a potential binding partner (Supplementary Table 1); co-immunoprecipitation assays confirmed the interaction between NEK8 and ANKS6 (Supplementary Fig. 1). An Arg823Trp missense mutation of Anks6 (SamCystin, Pkdr1) has recently been identified as the underlying cause of cystic kidney disease in the Han:SPRD +/Cy rat 11. Anks6 was detected at the proximal segment of the cilium in murine inner medullary collecting duct (IMCD) cells (Fig. 1a
and
Supplementary Fig. 1), similar to the localization of INVS, NPHP3 and NEK8 at this compartment 7,12. To analyze the role of Anks6 during embryogenesis, we used morpholino antisense oligonucleotide (MO)-mediated depletion in zebrafish. Injection of two independent anks6 MOs caused pronephric cyst formation (Fig. 1b, c
and
Supplementary Fig. 2). The cystic phenotype caused by anks6 depletion in the pronephric tubule was identical to nek8 and nphp3 morphants 13,14 (Fig. 1d, e) and combined knockdowns had an additive effect on cyst formation (Supplementary Fig. 2). In addition, laterality defects, detected by cmlc2 staining of early heart looping were observed in anks6 depleted zebrafish, and were comparable to nphp3- and nek8-deficient embryos (Fig. 1f, g).

Since unilateral injections allow a tissue restricted knockdown and analysis of organ specific phenotypes, we turned to the Xenopus model to analyze the developmental events in renal formation in further detail. Both nek8 and anks6 were expressed during Xenopus development, and were enriched within the proximal Xenopus pronephros at later developmental stages (Supplementary Fig. 3). Bilateral knockdown of anks6 by MO (Supplementary Fig. 4) resulted in gross body edema typical for a renal excretory defect (Fig. 2a) 15,16, also observed after nphp3 (Supplementary Fig. 5) and invs depletion 17. Depletion of either nek8 or anks6 resulted in a striking simplification of the proximal pronephros convolute (Fig. 2b, c), a phenotype previously also reported for the knockdown of Invs 17. Co-expression of a MO-insensitive mRNA coding for nek8 or anks6 respectively rescued the abnormalities, supporting the specificity of the observed phenotypes (Fig. 2b
and
Supplementary Fig. 4) The nek8 MO-mediated defects were partially rescued by co-expression of anks6 (Fig. 2c). This suggests that both proteins have common molecular effects, allowing Anks6 to partially substitute for Nek8. Early pronephric progenitor and later segmentation markers were not affected by nek8 or anks6 depletion (Supplementary Fig. 5 and Fig. 2d). The reduction in SGLT-1K- and NKCC2-positive pronephros segments indicated a shortening of the proximal and intermediate tubule (Fig. 2d). These data support the overlapping roles of Nek8 and Anks6 during early tubular morphogenesis, and are consistent with the phenotypic changes after nphp3 (Supplementary Fig. 5) and invs depletion 17.

The striking genetic and phenotypic similarities between NEK8 and ANKS6 suggested that ANKS6 might be involved in human cystic kidney disease, presenting with a nephronophthisis-like clinical syndrome. Mutation analysis of our NPHP cohorts yielded eight patients from six families with six different homozygous ANKS6 mutations (Table 1, Supplementary Fig. 6 and 7): two families with truncating mutations (c.2054_2064del, p.His685Profs*12 [B6794], c.2370_2372delTCA, p.Tyr790* [A649]), two families with splice site mutations (c.1973-3C>G [A3114], c.2512-2A>C [NPH316]) and two families with non-synonymous missense mutations (c.934G>C, p.Ala312Pro [A3121], c.1322A>G, p.Gln441Arg [B7397]). All affected individuals had polycystic kidney disease (PKD) with an early (infantile) onset, except for family NPH316 (juvenile). While patients with missense mutations showed normal-sized cystic kidney disease without extrarenal manifestations, splice site and truncating mutations were associated not only with an enlarged renal size, but also with severe extrarenal defects such as hypertrophic obstructive cardiomyopathy, aortic stenosis, pulmonary stenosis, patent ductus arteriosus, situs inversus and periportal liver fibrosis (Supplementary Fig. 6). Only in family NPH316 (Supplementary Fig. 6, Table 1), a splice site mutation (c.2512-2A>C) did not lead to early onset ESRD and liver involvement, possibly due to the localization in intron 14, affecting splicing of the last exon only.

Structural heart defects are often associated with laterality defects, and can result from defective cardiac looping during embryogenesis, which we observed in Anks6, Nek8 and Nphp3 deficient zebrafish embryos (Fig. 1g, f). INVS and NPHP3 mutations cause ventricular and atrial septal defects, pulmonary and aortic stenosis, and ventricular hypertrophy 18–20. Four of our patients with ANKS6 mutations (A3114, B6794, A649, NPH316) had aortic stenosis, causing obstructive cardiomyopathy in one patient (A649); another one of them (A3114) displayed additional pulmonary stenosis (Table 1), linking these three NPHP proteins by their clinical manifestations. The conserved missense ANKS6 Gln441Arg mutation (B7397) did not alter the interaction with other NPHP proteins nor its ciliary localization (Supplementary Fig. 7), but failed to rescue the renal phenotype in Xenopus, confirming the clinical relevance of this mutation (Supplementary Fig. 6). These observations indicate that ANKS6 is a novel member of the NPHP gene family and part of the phenotypically distinct NEK8-INVS-NPHP3 subgroup, which is characterized by structural heart defects in addition to infantile nephronophthisis and situs inversus.

To determine whether ANKS6 is part of a larger NPHP-associated protein network, we evaluated its interaction with other NPHP proteins. The kinase domain and the region between amino acid 259 and 312 of NEK8 was required for the interaction with the ankyrin repeat domain of ANKS6 (Fig. 3a, b
and
Supplementary Fig. 8), which also recognized NPHP3 and INVS (Fig. 3c), but not the structurally related ankyrin repeat protein Diversin (Supplementary Fig. 8). When NEK8 was precipitated from cells co-expressing INVS, the immobilization of INVS was strongly enhanced by the presence of ANKS6 (Supplementary Fig. 8). Similarly, when NEK8 was co-expressed with NPHP3, ANKS6 connected NEK8 to NPHP3 (Fig. 3d). Analysis of cilia in various tissues and model systems did not detect a role for ANKS6 in cilia formation or length control, but knockdown in Xenopus epidermal cells showed a mild defect of polarized orientation (Supplementary Fig. 9).

Juvenile cystic kidney (jck) mice carry a missense mutation in Nek8. Anks6 localization was identical in wild type and jck mice, even detectable in cyst lining cells (Supplementary Fig. 9). Consistent with a role for INVS to recruit other NPHPs, endogenous Anks6 was lost from the proximal segment of cilia in Invs-depleted IMCD3 cells (Fig. 3e) without affecting total Anks6 protein levels (Supplementary Fig. 8). Thus, ANKS6 not only acts as a central component of a distinct NPHP associated module, but appears to organize its assembly by linking INVS and NPHP3 to NEK8, while INVS is required to localize this complex to the proximal ciliary axoneme.

To further dissect the composition of the ANKS6-based module, we expanded our affinity proteomics screens using ANKS6 and different ANKS6 module partners as baits (Supplementary Table 1). Integration of the ANKS6 screens with data from a proteomics screen of TAP-tagged NPHP-associated proteins (Fig. 4a) retrieved a subset of at least seven proteins with a known mitochondrial localization and/or function, and also identified the NEK family member NEK7, and the ankyrin repeat protein ANKS3 as participants of this network. Since mutations of the mitochondrial X-prolyl aminopeptidase 3 were recently found to cause an NPH-like syndrome 21, these proteins including the mitochondrial components of the ANKS6 module may represent additional candidates for NPH and related ciliopathies.

The MS screens revealed a consistent connection between ANKS6 and the asparaginyl-hydroxylase HIF1AN (FIH, factor inhibiting HIF), which was also identified after affinity purification of INVS (Fig. 4a). HIF1AN is an oxygen-sensor that hydroxylates HIF1α as well as other ankyrin repeat proteins during normoxic conditions 22. We confirmed the interaction of HIF1AN with INVS and ANKS6, but not with the ankyrin repeat protein Diversin (Supplementary Fig. 10). Both INVS and ANKS6 contain well-defined and evolutionarily conserved hydroxylation recognition motifs (Supplementary Fig. 10); MS detected hydroxylated peptides at asparagines N75 of INVS and N129 of Anks6 (Fig. 4b). Co-immunoprecipitation experiments revealed that HIF1AN facilitated the ANKS6/INVS/NPHP3 module formation (Supplementary Fig. 10) and that mutation of the hydroxylation sites in ANKS6 resulted in decreased binding to NEK8 (Fig. 4c). The results suggest that hydroxylation of ANKS6 by HIF1AN mediates the complex formation by altering specific protein binding capacities. Hif1an is expressed in distal renal tubules 23, and knockdown of hif1an in Xenopus resulted in edema formation and tubular shortening (Fig. 4d
and
Supplementary Fig. 10), supporting a role for Hif1an in renal development. Hif1α and its target genes are up-regulated in the cy/+ rat kidneys 24, and VEGF receptor inhibition slows cyst progression in these rats 25, suggesting that activation of the Hif1α-dependent hypoxia pathway contributes to cyst progression in Anks6-deficient tubules. The ANKS6/INVS complex could potentially compete with HIF1α for hydroxylation by HIF1AN and induce a pseudo-hypoxic state when dysregulated. However, levels of HIF1α were not affected by overexpression of Anks6 in renal epithelial cells (HEK293T), excluding the possibility that expression levels of Anks6 alone would affect hypoxia signalling (Supplementary Fig. 10). The response to chemically simulated hypoxia was also not affected in either condition (Supplementary Fig. 10).

Our study identifies ANKS6 as a new NPHP family member that assembles a distinct module of NPHP-associated proteins, encompassing NEK8, INVS and NPHP3. The clinical findings as well as the in vivo data suggest that this network controls normal renal and cardiovascular development. HIF1AN connects the ANKS6 module to oxygen-dependent hydroxylation, which appears to alter the composition of the ANKS6 complex.

MATERIALS and METHODSHigh-throughput mutational analysisMutation analysis was performed by different approaches. First, we used PCR-based 48.48 Access Array microfluidic technology (Fluidigm™) with consecutive next generation sequencing. We applied a 14-fold primer multiplexing approach allowing PCR-based amplification of 672 amplicons (592 exons) for 48 DNA samples simultaneously in 13 known and 19 NPHP candidate genes, including ANKS6. A total of 1056 patients with an NPH-related complex phenotype were analyzed. After 4 rounds of amplification followed by indexing of all 1056 patient-derived products with 384 different 10bp-barcodes in a subsequent PCR, 2×150 bidirectional sequencing was performed on 8 lanes of a GAIIx instrument (Illumina™). Bioinformatic analysis was conducted using CLC-Genomics-Workbench™ software. Second, all exons and adjacent intronic boundaries of 129 genes (including ANKS6, in total 2216 coding exons) known or hypothesized to cause ciliopathies were targeted by a custom SeqCap EZ choice sequence capture library (NimbleGen, Madison, Wisconsin, USA) and sequenced on a Roche 454 GS FLX or an Illumina MiSeq platform (2×150 PE). 268 patients with a (poly)cystic kidney disease phenotype or an NPH-related complex ciliopathy were analyzed with an average coverage of 60-fold (GS FLX) or 120-fold (MiSeq). Bioinformatic analysis was performed using the Roche GS Reference Mapper™ software (v2.6), SeqPilot SeqNext moduleTM (v3.5.2, JSI medical systems, Kippenheim, Germany) and an in-house bioinformatic pipeline. For both approaches, potential mutations were confirmed by Sanger sequencing and shown to segregate. Linkage analysis using Affymetrix 250k SNP arrays was performed on a consanguineous family with NPH (Family NPH316). Four unaffected and two affected children are the offspring of a first-cousin marriage, and one unaffected and two affected children are the offspring of a third-cousin marriage. DNA samples from the third-cousin parents and their two affected children, and a sample from their affected cousin were available for the study. Haplotype analysis was performed with MERLIN software. Mutation analysis of ANKS6 was performed by Sanger sequencing.Animals and maintenanceXenopus laevis embryos were cultured, manipulated and staged as described 17. Zebrafish (strain tg (wt1b:GFP))
26 were bred and maintained under standard conditions at 28.5°C. C57BL/6J-Nek8jck/J were purchased from Jackson Laboratories and homozygous and wildtype mice were derived from respective matings of heterozygous animals. All experiments were approved by the institutional animal committee (Regierungspräsidium Baden-Württemberg).RNA extraction and RT-PCRRNA was isolated from Xenopus and zebrafish embryos following the RNAeasy manual (Qiagen) and cDNA synthesis was performed using the RevertAid H Minus Kit (Fermentas) and the First Strand cDNA Synthesis Kit (Invitrogen).Embryo microinjection manipulationsMicroinjection of 10 nanoliter (nl) into the ventro-lateral-vegetal blastomeres of Xenopus was performed to target the pronephros anlagen at the 4 to 8 cell stage. GFP or RFP mRNA was co-injected as an injection control and only fluorescent embryos were used for further analysis. For zebrafish knockdown experiments, morpholinos and sense RNA were diluted in 0.1M KCl to concentrations of 1–4μg/μl and 0.1μg/μl, respectively. One nl of this dilution was injected through the chorion of 1- or 2-cell stage embryos 27.The sequence of antisense oligonucleotide morpholinos (MOs) (GeneTools, LLC) were: nek8 MO (8ng): 5′-CCCGGATCTTCTTGTATTTCTCCAT-3′; anks6 MO (16ng): 5′-CAGTACACCTGCACTTACCCATGGT-3′; hif1an MO (16ng): CCGGCGTGACAGCACAGACCTGAAA-3′; zf nek8 MO: 5′-CTTCTCATACTTCTCCATGTTTTCG-3′; zf anks6 MO1: 5′-TCAGCGCCGTGTTTCCATCCTCATC-3′; zf anks6 MO2: 5′-AGTGTAGTGTGTAGGGTATAAACCT-3′; control (CTL) MO: 5′-CCTCTTACCTCAG TTACAATTTATA-3′. TNT® Quick coupled transcription/translation system (Promega) was used to confirm MO efficiency. In vitro synthesis of mRNA was performed using mMessage mMachine Kit (Ambion) as followed: xNek8_VF10: Pst1, T7; rat Anks6_VF10: Sal1, T7; hNPHP3_VF10: Sal1, T7; hHIF1AN_VF10: Sal1, T7.Whole mount in situ hybridizationWhole mount in situ hybridization (WISH) was performed with digoxigenin labeled antisense probes as described 17,27. For in situ probes the plasmids were linearized and transcribed with SP6 or T7 (Roche). Anti-digoxigenin antibody conjugated to alkaline phosphatase was used to detect bound probes (Roche). Cmlc2 WISH was performed on 48hpf zebrafish embryos.HistologyZebrafish embryos were embedded in Technovit 7100 (Heraeus), stained with Hematoxylin and Eosin and imaged with an Axioplan2 microscope and AxioVision software (Zeiss).Wildtype and Nek8/jck mice were sacrificed at indicated timepoints and kidneys were collected after perfusion with 4% paraformaldehyde (PFA) via the renal artery and subsequently immersion fixated at 4° C over night. Paraffin sectioning was performed using standard techniques.Immunofluorescence staining and microscopyZebrafish embryos were fixed in 4 % PFA/1 % DMSO overnight at 4° C, equilibrated in 100 % MeOH at −20 °C for 1 hour, Proteinase K-digested (10 μg/ml) for 20 min, treated with icecold aceton for 5 min at −20 °C, incubated in blocking solution (1 % PBSTT, 1 % DMSO, 2 % sheep serum, 1 % BSA) followed by incubation with anti-acetylated-Tubulin.For fluorescent staining of whole Xenopus embryos 3G8 and 4A6 antibodies (European Xenopus stock centre) were used. For Xenopus rescue experiments whole embryos were stained with fluorescein lycopersicon esculentum lectin (LEL) (Vector Laboratories). The kidney length was measured using ImageJ. The difference of kidney length from the uninjected to the MO injected side was calculated. Analysis of basal body polarization in Xenopus epidermal cells was performed as described 28.Immunofluorescence staining of IMCD cells was done using 4% PFA or methanol/acetone (1:1). Cells were permeabilized with 0.1% Triton X-100 in PBS and incubated in blocking solution (5% horse serum or 0.2% gold fish gelatine). Primary antibodies: rabbit anti-ANKS6 (Sigma-Aldrich/Prestige Antibodies, polyclonal anti-rabbit, HPA008355), mouse anti-acetylated-Tubulin (Sigma-Aldrich, T6793, 1:3000), mouse anti-γ-Tubulin (T6557, Sigma-Aldrich) and Hoechst 33342. Antibodies were visualized using Cy5, Cy3- or Alexa-488-labelled secondary antibodies at a dilution of 1:1000 (Jackson Immunoresearch).For the ciliogenesis assay IMCD cells were grown on glass for 6 days, stained for acetylated tubulin and Hoechst, and imaged with a confocal microscope. Z-stacks were done to include all cilia in different z-positions and then projected to one plane (maximum intensity projection). Experiments were conducted three independent times (5 fields of view per N).Confocal imaging was performed with a LSM 510 Duo-Live microscope equipped with a 100x/1.45 NA Plan-Apochromate objective (both Carl Zeiss). Excitation of the fluorophores (Hoechst 33342, Alexa-488, Cy3, Cy5) was performed at 405, 488, 561, and 633 nm respectively. For detection of the emission signal at specified ranges, the photomultiplier channels were used with BP filter 420–480, BP 505–530, BP 575–615, LP 650 nm. Confocal pinhole diameters were adjusted to 1 μm sections. Cilia and the nucleus were projected to one plane.Antigen-retrieval on paraffin slides of mouse kidneys was performed using citrate buffer pH 6.0 (10mM Tri-Sodium-Citrate di-hydrate) in a steamer for 30 min. Following primary antibodies were applied: anti-Anks6 (Sigma-Aldrich, 1:100) and anti-acetylated-Tubulin (Sigma-Aldrich, monoclonal anti-mouse, Clone 6-11B-1 - 1:400). Primary antibodies were used in a consecutive staining procedure to avoid cross-reactions. Secondary antibodies (diluted 1:500): Alexa-488, Alexa-555; Hoechst 33342 (diluted 1:1000). Slides were mounted with ProLong Gold Antifade (Invitrogen). Images were acquired on a confocal Zeiss LSM 510 upright microscope (Zeiss), equipped with a Plan-Apochromat 63x/1.4 Oil M27 objective. All confocal image recording was performed with Zen black Software (Zeiss).Plasmids, Reagents and expression clonesRapid amplification of cDNA ends Kit (Invitrogen) was used to synthesize the 5′end of Xenopus laevis Nek8. Full-length NEK8 (NM_178170.2) and several truncated versions of NEK8 created by PCR and standard cloning techniques were fused to V5 or Flag-tagged pcDNA6 vectors (Invitrogen). Full-length Anks6 cDNA clones (NM_ 001015028.2; NM_173551.3) were synthesized by OriGene and fused to V5 or Flag- tagged pcDNA6 vectors (Invitrogen). For the RTS 100 Wheat Germ continuous cell free system Kit (5Prime) the DNA of interest was cloned into the pIVEX1.4 WG vector. An entry clone for NEK8 was created by PCR using a NEK8 cDNA IMAGE clone corresponding to NCBI refseq nucleotide accession NM_178170.2. A NEK8 expression construct was created using Gateway technology (Invitrogen, Leek, The Netherlands). NEK7 (matching NCBI refseq nucleotide accession NM_133494.2) and INVS (matching Ensembl nucleotide accession ENST00000374921) expression constructs for affinity proteomics were kindly provided by professor N. Katsanis and J. Willer.Cellculture, Co-Immunoprecipitation, Western blotting, AntibodiesHuman embryonic kidney (HEK293T) cells (purchased from ATCC) were transiently transfected and co-immunoprecipitation was carried out as described 28. Briefly, cells were washed with PBS, lysed with lysis buffer (1% Triton X-100, 20mM Tris, pH 7.5, 50mM NaCl, 50mM NaF, 15mM Na4P2O7, 0.1mM EDTA) supplemented with 2mM Na3VO4 and protease inhibitor mix (Roche). Lysates were incubated with anti-Flag M2 agarose Affinity M2 beads for two hours and washed with lysis buffer.For Tetracycline inducible (1μg/ml) Anks6 knockdown (Anks6-i) inner medullary collecting duct (IMCD3) cells were lentiviral transduced with a shRNA targeting base pairs 2304–2324 of the coding sequence (CDS) for mAnks6 (NM_001024136) which was cloned into pLVTH. The efficiency of the knockdown was verified by quantitative real time polymerase chain reaction (pPCR) with the MesaFast qPCR Master mix plus for SYBR Assay (Eurogentec). The Invs IMCD3 knockdown cell line (Invs-i) was lentivirally transduced with a tetracycline inducible shRNA targeting bp 945–965 of the CDS for mInvs (NM_010569) and the efficiency was verified by qPCR. For overexpression rat Anks6 (and rAnks6Q433R) was cloned into pLXSN in-frame with C-terminal Venus and transduced into IMCD3 cells. Hypoxic conditions were mimicked by CoCl2 (Sigma-Aldrich) with a final concentration of 125μmol/L for 8 hours. Western blots were analysed with anti-HIF1α antibody (BD Biosciences), anti-ANKS6 antibody and anti-γ–sTubulin.Scanning electron microscopyFor scanning electron microscopy (SEM) samples were fixated with 4% glutaraldehyde (Sigma-Aldrich, EM quality grade) for 4 days at 4°C and were then subsequently dehydrated (EtOH 50, 70, 80, 90 and 100%; 1:1 EtOH and HMDS for 1 hour and 30 min 100% HMDS, afterwards solvent was allowed to evaporate). After dehydration procedure standard coating was performed with Gold (Zeiss Semco Nanolab7, Polaron Cool Sputter Coater E 5100, Balzer Cpd 020). Image acquisition of respective Xenopus embryos was done using a Leo 1450 VP scanning electron microscope.Affinity proteomicsStrep-Flag tandem affinity purification (SF-TAP) was performed as described 29. Prior to LC-MS/MS analysis, protein precipitates were subjected to tryptic proteolysis. Mass spectrometric analysis of SF-TAP purified samples was performed as described. Digested samples were separated on an UltiMate 3000 RSLCnano system, on-line coupled to a LTQ Orbitrap Velos (Thermo Fisher Scientific, Waltham, USA). All MS/MS samples were analyzed using Mascot (version 2.4, Matrix Science, Boston, MA, USA). Mascot was set up to search the human subset of the Swiss Prot database (Release 2012_05, 20245 entries), assuming trypsin as the digestion enzyme. Mascot was searched with a fragment ion mass tolerance of 1.00 Da and a parent ion tolerance of 10.0 PPM. Oxidation of methionine and was specified as variable modification, iodoacetamide derivative of cysteine as fixed. The Mascot results were loaded in Scaffold (version Scaffold_3.5, Proteome Software Inc., Portland, OR) to validate MS/MS based peptide and protein identifications. Peptide identifications were accepted if they could be established at greater than 80.0% probability as specified by the Peptide Prophet algorithm 30. Protein identifications were accepted if they could be established at greater than 95.0% probability and contained at least 2 identified peptides. Protein probabilities were assigned by the Protein Prophet algorithm 31. Proteins that contained similar peptides and could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony.For Flag-IP, washed immunoprecipitates were incubated with Flag-peptide (Sigma) to elute the precipitated protein from the anti-Flag coated beads. Samples were separated by SDS gel, stained and in gel digest were performed as described in standard protocols. Digests with different proteases (Trypsin, Elastase, Thermolysin) were performed in 0.1M NH4HCO3 (pH 8) overnight at 37°C. For one gel band about 0.1μg of protease was used. Peptides were extracted from the gel slices using 5% formic acid. All LC-MS/MS analyses were performed on an ion trap mass spectrometer (Agilent 6340, Agilent Technologies) coupled to an 1200 Agilent nanoflow system via a HPLC-Chip cube ESI interface. Alternatively, LC-MS/MS analyses were performed on a Q-TOF mass spectrometer (Agilent 6520, Agilent Technologies) coupled to a 1200 Agilent nanoflow system via a HPLC-Chip cube ESI interface. Peptides were elutes with a linear acetonitrile gradient with 1%/min at a flow rate of 300nl/min (starting with 3% acetonitrile). For raw data processing Mascot Distiller 2.4.2 (Matrix Science, London, UK) was used. The data from the Mascot searches was further condensed for the samples (experiment and control) in Scaffold 3.4.9 (Proteome Software, Portland, OR).Statistical analysesSigmaStat software was used to analyse statistical significance. All experiments were performed three times and diagrams show the mean +/− standard error of the mean (SEM). The tests used to calculate the significance are indicated in the corresponding figure legends for the individual experiments. A standard number of embryos (20 to 30 for Xenopus; 100 for zebrafish) were injected per experiment and non-viable embryos were excluded before gastrulation. The number of analysed embryos is given above the bars.

MATERIALS and METHODS

High-throughput mutational analysisMutation analysis was performed by different approaches. First, we used PCR-based 48.48 Access Array microfluidic technology (Fluidigm™) with consecutive next generation sequencing. We applied a 14-fold primer multiplexing approach allowing PCR-based amplification of 672 amplicons (592 exons) for 48 DNA samples simultaneously in 13 known and 19 NPHP candidate genes, including ANKS6. A total of 1056 patients with an NPH-related complex phenotype were analyzed. After 4 rounds of amplification followed by indexing of all 1056 patient-derived products with 384 different 10bp-barcodes in a subsequent PCR, 2×150 bidirectional sequencing was performed on 8 lanes of a GAIIx instrument (Illumina™). Bioinformatic analysis was conducted using CLC-Genomics-Workbench™ software. Second, all exons and adjacent intronic boundaries of 129 genes (including ANKS6, in total 2216 coding exons) known or hypothesized to cause ciliopathies were targeted by a custom SeqCap EZ choice sequence capture library (NimbleGen, Madison, Wisconsin, USA) and sequenced on a Roche 454 GS FLX or an Illumina MiSeq platform (2×150 PE). 268 patients with a (poly)cystic kidney disease phenotype or an NPH-related complex ciliopathy were analyzed with an average coverage of 60-fold (GS FLX) or 120-fold (MiSeq). Bioinformatic analysis was performed using the Roche GS Reference Mapper™ software (v2.6), SeqPilot SeqNext moduleTM (v3.5.2, JSI medical systems, Kippenheim, Germany) and an in-house bioinformatic pipeline. For both approaches, potential mutations were confirmed by Sanger sequencing and shown to segregate. Linkage analysis using Affymetrix 250k SNP arrays was performed on a consanguineous family with NPH (Family NPH316). Four unaffected and two affected children are the offspring of a first-cousin marriage, and one unaffected and two affected children are the offspring of a third-cousin marriage. DNA samples from the third-cousin parents and their two affected children, and a sample from their affected cousin were available for the study. Haplotype analysis was performed with MERLIN software. Mutation analysis of ANKS6 was performed by Sanger sequencing.

High-throughput mutational analysis

Mutation analysis was performed by different approaches. First, we used PCR-based 48.48 Access Array microfluidic technology (Fluidigm™) with consecutive next generation sequencing. We applied a 14-fold primer multiplexing approach allowing PCR-based amplification of 672 amplicons (592 exons) for 48 DNA samples simultaneously in 13 known and 19 NPHP candidate genes, including ANKS6. A total of 1056 patients with an NPH-related complex phenotype were analyzed. After 4 rounds of amplification followed by indexing of all 1056 patient-derived products with 384 different 10bp-barcodes in a subsequent PCR, 2×150 bidirectional sequencing was performed on 8 lanes of a GAIIx instrument (Illumina™). Bioinformatic analysis was conducted using CLC-Genomics-Workbench™ software. Second, all exons and adjacent intronic boundaries of 129 genes (including ANKS6, in total 2216 coding exons) known or hypothesized to cause ciliopathies were targeted by a custom SeqCap EZ choice sequence capture library (NimbleGen, Madison, Wisconsin, USA) and sequenced on a Roche 454 GS FLX or an Illumina MiSeq platform (2×150 PE). 268 patients with a (poly)cystic kidney disease phenotype or an NPH-related complex ciliopathy were analyzed with an average coverage of 60-fold (GS FLX) or 120-fold (MiSeq). Bioinformatic analysis was performed using the Roche GS Reference Mapper™ software (v2.6), SeqPilot SeqNext moduleTM (v3.5.2, JSI medical systems, Kippenheim, Germany) and an in-house bioinformatic pipeline. For both approaches, potential mutations were confirmed by Sanger sequencing and shown to segregate. Linkage analysis using Affymetrix 250k SNP arrays was performed on a consanguineous family with NPH (Family NPH316). Four unaffected and two affected children are the offspring of a first-cousin marriage, and one unaffected and two affected children are the offspring of a third-cousin marriage. DNA samples from the third-cousin parents and their two affected children, and a sample from their affected cousin were available for the study. Haplotype analysis was performed with MERLIN software. Mutation analysis of ANKS6 was performed by Sanger sequencing.

Animals and maintenanceXenopus laevis embryos were cultured, manipulated and staged as described 17. Zebrafish (strain tg (wt1b:GFP))
26 were bred and maintained under standard conditions at 28.5°C. C57BL/6J-Nek8jck/J were purchased from Jackson Laboratories and homozygous and wildtype mice were derived from respective matings of heterozygous animals. All experiments were approved by the institutional animal committee (Regierungspräsidium Baden-Württemberg).

Animals and maintenance

Xenopus laevis embryos were cultured, manipulated and staged as described 17. Zebrafish (strain tg (wt1b:GFP))
26 were bred and maintained under standard conditions at 28.5°C. C57BL/6J-Nek8jck/J were purchased from Jackson Laboratories and homozygous and wildtype mice were derived from respective matings of heterozygous animals. All experiments were approved by the institutional animal committee (Regierungspräsidium Baden-Württemberg).

RNA extraction and RT-PCRRNA was isolated from Xenopus and zebrafish embryos following the RNAeasy manual (Qiagen) and cDNA synthesis was performed using the RevertAid H Minus Kit (Fermentas) and the First Strand cDNA Synthesis Kit (Invitrogen).

RNA extraction and RT-PCR

RNA was isolated from Xenopus and zebrafish embryos following the RNAeasy manual (Qiagen) and cDNA synthesis was performed using the RevertAid H Minus Kit (Fermentas) and the First Strand cDNA Synthesis Kit (Invitrogen).

Embryo microinjection manipulationsMicroinjection of 10 nanoliter (nl) into the ventro-lateral-vegetal blastomeres of Xenopus was performed to target the pronephros anlagen at the 4 to 8 cell stage. GFP or RFP mRNA was co-injected as an injection control and only fluorescent embryos were used for further analysis. For zebrafish knockdown experiments, morpholinos and sense RNA were diluted in 0.1M KCl to concentrations of 1–4μg/μl and 0.1μg/μl, respectively. One nl of this dilution was injected through the chorion of 1- or 2-cell stage embryos 27.The sequence of antisense oligonucleotide morpholinos (MOs) (GeneTools, LLC) were: nek8 MO (8ng): 5′-CCCGGATCTTCTTGTATTTCTCCAT-3′; anks6 MO (16ng): 5′-CAGTACACCTGCACTTACCCATGGT-3′; hif1an MO (16ng): CCGGCGTGACAGCACAGACCTGAAA-3′; zf nek8 MO: 5′-CTTCTCATACTTCTCCATGTTTTCG-3′; zf anks6 MO1: 5′-TCAGCGCCGTGTTTCCATCCTCATC-3′; zf anks6 MO2: 5′-AGTGTAGTGTGTAGGGTATAAACCT-3′; control (CTL) MO: 5′-CCTCTTACCTCAG TTACAATTTATA-3′. TNT® Quick coupled transcription/translation system (Promega) was used to confirm MO efficiency. In vitro synthesis of mRNA was performed using mMessage mMachine Kit (Ambion) as followed: xNek8_VF10: Pst1, T7; rat Anks6_VF10: Sal1, T7; hNPHP3_VF10: Sal1, T7; hHIF1AN_VF10: Sal1, T7.

Embryo microinjection manipulations

Microinjection of 10 nanoliter (nl) into the ventro-lateral-vegetal blastomeres of Xenopus was performed to target the pronephros anlagen at the 4 to 8 cell stage. GFP or RFP mRNA was co-injected as an injection control and only fluorescent embryos were used for further analysis. For zebrafish knockdown experiments, morpholinos and sense RNA were diluted in 0.1M KCl to concentrations of 1–4μg/μl and 0.1μg/μl, respectively. One nl of this dilution was injected through the chorion of 1- or 2-cell stage embryos 27.

The sequence of antisense oligonucleotide morpholinos (MOs) (GeneTools, LLC) were: nek8 MO (8ng): 5′-CCCGGATCTTCTTGTATTTCTCCAT-3′; anks6 MO (16ng): 5′-CAGTACACCTGCACTTACCCATGGT-3′; hif1an MO (16ng): CCGGCGTGACAGCACAGACCTGAAA-3′; zf nek8 MO: 5′-CTTCTCATACTTCTCCATGTTTTCG-3′; zf anks6 MO1: 5′-TCAGCGCCGTGTTTCCATCCTCATC-3′; zf anks6 MO2: 5′-AGTGTAGTGTGTAGGGTATAAACCT-3′; control (CTL) MO: 5′-CCTCTTACCTCAG TTACAATTTATA-3′. TNT® Quick coupled transcription/translation system (Promega) was used to confirm MO efficiency. In vitro synthesis of mRNA was performed using mMessage mMachine Kit (Ambion) as followed: xNek8_VF10: Pst1, T7; rat Anks6_VF10: Sal1, T7; hNPHP3_VF10: Sal1, T7; hHIF1AN_VF10: Sal1, T7.

Whole mount in situ hybridizationWhole mount in situ hybridization (WISH) was performed with digoxigenin labeled antisense probes as described 17,27. For in situ probes the plasmids were linearized and transcribed with SP6 or T7 (Roche). Anti-digoxigenin antibody conjugated to alkaline phosphatase was used to detect bound probes (Roche). Cmlc2 WISH was performed on 48hpf zebrafish embryos.

Whole mount in situ hybridization

Whole mount in situ hybridization (WISH) was performed with digoxigenin labeled antisense probes as described 17,27. For in situ probes the plasmids were linearized and transcribed with SP6 or T7 (Roche). Anti-digoxigenin antibody conjugated to alkaline phosphatase was used to detect bound probes (Roche). Cmlc2 WISH was performed on 48hpf zebrafish embryos.

HistologyZebrafish embryos were embedded in Technovit 7100 (Heraeus), stained with Hematoxylin and Eosin and imaged with an Axioplan2 microscope and AxioVision software (Zeiss).Wildtype and Nek8/jck mice were sacrificed at indicated timepoints and kidneys were collected after perfusion with 4% paraformaldehyde (PFA) via the renal artery and subsequently immersion fixated at 4° C over night. Paraffin sectioning was performed using standard techniques.

Zebrafish embryos were embedded in Technovit 7100 (Heraeus), stained with Hematoxylin and Eosin and imaged with an Axioplan2 microscope and AxioVision software (Zeiss).

Wildtype and Nek8/jck mice were sacrificed at indicated timepoints and kidneys were collected after perfusion with 4% paraformaldehyde (PFA) via the renal artery and subsequently immersion fixated at 4° C over night. Paraffin sectioning was performed using standard techniques.

Immunofluorescence staining and microscopyZebrafish embryos were fixed in 4 % PFA/1 % DMSO overnight at 4° C, equilibrated in 100 % MeOH at −20 °C for 1 hour, Proteinase K-digested (10 μg/ml) for 20 min, treated with icecold aceton for 5 min at −20 °C, incubated in blocking solution (1 % PBSTT, 1 % DMSO, 2 % sheep serum, 1 % BSA) followed by incubation with anti-acetylated-Tubulin.For fluorescent staining of whole Xenopus embryos 3G8 and 4A6 antibodies (European Xenopus stock centre) were used. For Xenopus rescue experiments whole embryos were stained with fluorescein lycopersicon esculentum lectin (LEL) (Vector Laboratories). The kidney length was measured using ImageJ. The difference of kidney length from the uninjected to the MO injected side was calculated. Analysis of basal body polarization in Xenopus epidermal cells was performed as described 28.Immunofluorescence staining of IMCD cells was done using 4% PFA or methanol/acetone (1:1). Cells were permeabilized with 0.1% Triton X-100 in PBS and incubated in blocking solution (5% horse serum or 0.2% gold fish gelatine). Primary antibodies: rabbit anti-ANKS6 (Sigma-Aldrich/Prestige Antibodies, polyclonal anti-rabbit, HPA008355), mouse anti-acetylated-Tubulin (Sigma-Aldrich, T6793, 1:3000), mouse anti-γ-Tubulin (T6557, Sigma-Aldrich) and Hoechst 33342. Antibodies were visualized using Cy5, Cy3- or Alexa-488-labelled secondary antibodies at a dilution of 1:1000 (Jackson Immunoresearch).For the ciliogenesis assay IMCD cells were grown on glass for 6 days, stained for acetylated tubulin and Hoechst, and imaged with a confocal microscope. Z-stacks were done to include all cilia in different z-positions and then projected to one plane (maximum intensity projection). Experiments were conducted three independent times (5 fields of view per N).Confocal imaging was performed with a LSM 510 Duo-Live microscope equipped with a 100x/1.45 NA Plan-Apochromate objective (both Carl Zeiss). Excitation of the fluorophores (Hoechst 33342, Alexa-488, Cy3, Cy5) was performed at 405, 488, 561, and 633 nm respectively. For detection of the emission signal at specified ranges, the photomultiplier channels were used with BP filter 420–480, BP 505–530, BP 575–615, LP 650 nm. Confocal pinhole diameters were adjusted to 1 μm sections. Cilia and the nucleus were projected to one plane.Antigen-retrieval on paraffin slides of mouse kidneys was performed using citrate buffer pH 6.0 (10mM Tri-Sodium-Citrate di-hydrate) in a steamer for 30 min. Following primary antibodies were applied: anti-Anks6 (Sigma-Aldrich, 1:100) and anti-acetylated-Tubulin (Sigma-Aldrich, monoclonal anti-mouse, Clone 6-11B-1 - 1:400). Primary antibodies were used in a consecutive staining procedure to avoid cross-reactions. Secondary antibodies (diluted 1:500): Alexa-488, Alexa-555; Hoechst 33342 (diluted 1:1000). Slides were mounted with ProLong Gold Antifade (Invitrogen). Images were acquired on a confocal Zeiss LSM 510 upright microscope (Zeiss), equipped with a Plan-Apochromat 63x/1.4 Oil M27 objective. All confocal image recording was performed with Zen black Software (Zeiss).

Immunofluorescence staining and microscopy

Zebrafish embryos were fixed in 4 % PFA/1 % DMSO overnight at 4° C, equilibrated in 100 % MeOH at −20 °C for 1 hour, Proteinase K-digested (10 μg/ml) for 20 min, treated with icecold aceton for 5 min at −20 °C, incubated in blocking solution (1 % PBSTT, 1 % DMSO, 2 % sheep serum, 1 % BSA) followed by incubation with anti-acetylated-Tubulin.

For fluorescent staining of whole Xenopus embryos 3G8 and 4A6 antibodies (European Xenopus stock centre) were used. For Xenopus rescue experiments whole embryos were stained with fluorescein lycopersicon esculentum lectin (LEL) (Vector Laboratories). The kidney length was measured using ImageJ. The difference of kidney length from the uninjected to the MO injected side was calculated. Analysis of basal body polarization in Xenopus epidermal cells was performed as described 28.

Immunofluorescence staining of IMCD cells was done using 4% PFA or methanol/acetone (1:1). Cells were permeabilized with 0.1% Triton X-100 in PBS and incubated in blocking solution (5% horse serum or 0.2% gold fish gelatine). Primary antibodies: rabbit anti-ANKS6 (Sigma-Aldrich/Prestige Antibodies, polyclonal anti-rabbit, HPA008355), mouse anti-acetylated-Tubulin (Sigma-Aldrich, T6793, 1:3000), mouse anti-γ-Tubulin (T6557, Sigma-Aldrich) and Hoechst 33342. Antibodies were visualized using Cy5, Cy3- or Alexa-488-labelled secondary antibodies at a dilution of 1:1000 (Jackson Immunoresearch).

For the ciliogenesis assay IMCD cells were grown on glass for 6 days, stained for acetylated tubulin and Hoechst, and imaged with a confocal microscope. Z-stacks were done to include all cilia in different z-positions and then projected to one plane (maximum intensity projection). Experiments were conducted three independent times (5 fields of view per N).

Confocal imaging was performed with a LSM 510 Duo-Live microscope equipped with a 100x/1.45 NA Plan-Apochromate objective (both Carl Zeiss). Excitation of the fluorophores (Hoechst 33342, Alexa-488, Cy3, Cy5) was performed at 405, 488, 561, and 633 nm respectively. For detection of the emission signal at specified ranges, the photomultiplier channels were used with BP filter 420–480, BP 505–530, BP 575–615, LP 650 nm. Confocal pinhole diameters were adjusted to 1 μm sections. Cilia and the nucleus were projected to one plane.

Antigen-retrieval on paraffin slides of mouse kidneys was performed using citrate buffer pH 6.0 (10mM Tri-Sodium-Citrate di-hydrate) in a steamer for 30 min. Following primary antibodies were applied: anti-Anks6 (Sigma-Aldrich, 1:100) and anti-acetylated-Tubulin (Sigma-Aldrich, monoclonal anti-mouse, Clone 6-11B-1 - 1:400). Primary antibodies were used in a consecutive staining procedure to avoid cross-reactions. Secondary antibodies (diluted 1:500): Alexa-488, Alexa-555; Hoechst 33342 (diluted 1:1000). Slides were mounted with ProLong Gold Antifade (Invitrogen). Images were acquired on a confocal Zeiss LSM 510 upright microscope (Zeiss), equipped with a Plan-Apochromat 63x/1.4 Oil M27 objective. All confocal image recording was performed with Zen black Software (Zeiss).

Plasmids, Reagents and expression clonesRapid amplification of cDNA ends Kit (Invitrogen) was used to synthesize the 5′end of Xenopus laevis Nek8. Full-length NEK8 (NM_178170.2) and several truncated versions of NEK8 created by PCR and standard cloning techniques were fused to V5 or Flag-tagged pcDNA6 vectors (Invitrogen). Full-length Anks6 cDNA clones (NM_ 001015028.2; NM_173551.3) were synthesized by OriGene and fused to V5 or Flag- tagged pcDNA6 vectors (Invitrogen). For the RTS 100 Wheat Germ continuous cell free system Kit (5Prime) the DNA of interest was cloned into the pIVEX1.4 WG vector. An entry clone for NEK8 was created by PCR using a NEK8 cDNA IMAGE clone corresponding to NCBI refseq nucleotide accession NM_178170.2. A NEK8 expression construct was created using Gateway technology (Invitrogen, Leek, The Netherlands). NEK7 (matching NCBI refseq nucleotide accession NM_133494.2) and INVS (matching Ensembl nucleotide accession ENST00000374921) expression constructs for affinity proteomics were kindly provided by professor N. Katsanis and J. Willer.

Plasmids, Reagents and expression clones

Rapid amplification of cDNA ends Kit (Invitrogen) was used to synthesize the 5′end of Xenopus laevis Nek8. Full-length NEK8 (NM_178170.2) and several truncated versions of NEK8 created by PCR and standard cloning techniques were fused to V5 or Flag-tagged pcDNA6 vectors (Invitrogen). Full-length Anks6 cDNA clones (NM_ 001015028.2; NM_173551.3) were synthesized by OriGene and fused to V5 or Flag- tagged pcDNA6 vectors (Invitrogen). For the RTS 100 Wheat Germ continuous cell free system Kit (5Prime) the DNA of interest was cloned into the pIVEX1.4 WG vector. An entry clone for NEK8 was created by PCR using a NEK8 cDNA IMAGE clone corresponding to NCBI refseq nucleotide accession NM_178170.2. A NEK8 expression construct was created using Gateway technology (Invitrogen, Leek, The Netherlands). NEK7 (matching NCBI refseq nucleotide accession NM_133494.2) and INVS (matching Ensembl nucleotide accession ENST00000374921) expression constructs for affinity proteomics were kindly provided by professor N. Katsanis and J. Willer.

Cellculture, Co-Immunoprecipitation, Western blotting, AntibodiesHuman embryonic kidney (HEK293T) cells (purchased from ATCC) were transiently transfected and co-immunoprecipitation was carried out as described 28. Briefly, cells were washed with PBS, lysed with lysis buffer (1% Triton X-100, 20mM Tris, pH 7.5, 50mM NaCl, 50mM NaF, 15mM Na4P2O7, 0.1mM EDTA) supplemented with 2mM Na3VO4 and protease inhibitor mix (Roche). Lysates were incubated with anti-Flag M2 agarose Affinity M2 beads for two hours and washed with lysis buffer.For Tetracycline inducible (1μg/ml) Anks6 knockdown (Anks6-i) inner medullary collecting duct (IMCD3) cells were lentiviral transduced with a shRNA targeting base pairs 2304–2324 of the coding sequence (CDS) for mAnks6 (NM_001024136) which was cloned into pLVTH. The efficiency of the knockdown was verified by quantitative real time polymerase chain reaction (pPCR) with the MesaFast qPCR Master mix plus for SYBR Assay (Eurogentec). The Invs IMCD3 knockdown cell line (Invs-i) was lentivirally transduced with a tetracycline inducible shRNA targeting bp 945–965 of the CDS for mInvs (NM_010569) and the efficiency was verified by qPCR. For overexpression rat Anks6 (and rAnks6Q433R) was cloned into pLXSN in-frame with C-terminal Venus and transduced into IMCD3 cells. Hypoxic conditions were mimicked by CoCl2 (Sigma-Aldrich) with a final concentration of 125μmol/L for 8 hours. Western blots were analysed with anti-HIF1α antibody (BD Biosciences), anti-ANKS6 antibody and anti-γ–sTubulin.

Cellculture, Co-Immunoprecipitation, Western blotting, Antibodies

Human embryonic kidney (HEK293T) cells (purchased from ATCC) were transiently transfected and co-immunoprecipitation was carried out as described 28. Briefly, cells were washed with PBS, lysed with lysis buffer (1% Triton X-100, 20mM Tris, pH 7.5, 50mM NaCl, 50mM NaF, 15mM Na4P2O7, 0.1mM EDTA) supplemented with 2mM Na3VO4 and protease inhibitor mix (Roche). Lysates were incubated with anti-Flag M2 agarose Affinity M2 beads for two hours and washed with lysis buffer.

For Tetracycline inducible (1μg/ml) Anks6 knockdown (Anks6-i) inner medullary collecting duct (IMCD3) cells were lentiviral transduced with a shRNA targeting base pairs 2304–2324 of the coding sequence (CDS) for mAnks6 (NM_001024136) which was cloned into pLVTH. The efficiency of the knockdown was verified by quantitative real time polymerase chain reaction (pPCR) with the MesaFast qPCR Master mix plus for SYBR Assay (Eurogentec). The Invs IMCD3 knockdown cell line (Invs-i) was lentivirally transduced with a tetracycline inducible shRNA targeting bp 945–965 of the CDS for mInvs (NM_010569) and the efficiency was verified by qPCR. For overexpression rat Anks6 (and rAnks6Q433R) was cloned into pLXSN in-frame with C-terminal Venus and transduced into IMCD3 cells. Hypoxic conditions were mimicked by CoCl2 (Sigma-Aldrich) with a final concentration of 125μmol/L for 8 hours. Western blots were analysed with anti-HIF1α antibody (BD Biosciences), anti-ANKS6 antibody and anti-γ–sTubulin.

Scanning electron microscopyFor scanning electron microscopy (SEM) samples were fixated with 4% glutaraldehyde (Sigma-Aldrich, EM quality grade) for 4 days at 4°C and were then subsequently dehydrated (EtOH 50, 70, 80, 90 and 100%; 1:1 EtOH and HMDS for 1 hour and 30 min 100% HMDS, afterwards solvent was allowed to evaporate). After dehydration procedure standard coating was performed with Gold (Zeiss Semco Nanolab7, Polaron Cool Sputter Coater E 5100, Balzer Cpd 020). Image acquisition of respective Xenopus embryos was done using a Leo 1450 VP scanning electron microscope.

Scanning electron microscopy

For scanning electron microscopy (SEM) samples were fixated with 4% glutaraldehyde (Sigma-Aldrich, EM quality grade) for 4 days at 4°C and were then subsequently dehydrated (EtOH 50, 70, 80, 90 and 100%; 1:1 EtOH and HMDS for 1 hour and 30 min 100% HMDS, afterwards solvent was allowed to evaporate). After dehydration procedure standard coating was performed with Gold (Zeiss Semco Nanolab7, Polaron Cool Sputter Coater E 5100, Balzer Cpd 020). Image acquisition of respective Xenopus embryos was done using a Leo 1450 VP scanning electron microscope.

Affinity proteomicsStrep-Flag tandem affinity purification (SF-TAP) was performed as described 29. Prior to LC-MS/MS analysis, protein precipitates were subjected to tryptic proteolysis. Mass spectrometric analysis of SF-TAP purified samples was performed as described. Digested samples were separated on an UltiMate 3000 RSLCnano system, on-line coupled to a LTQ Orbitrap Velos (Thermo Fisher Scientific, Waltham, USA). All MS/MS samples were analyzed using Mascot (version 2.4, Matrix Science, Boston, MA, USA). Mascot was set up to search the human subset of the Swiss Prot database (Release 2012_05, 20245 entries), assuming trypsin as the digestion enzyme. Mascot was searched with a fragment ion mass tolerance of 1.00 Da and a parent ion tolerance of 10.0 PPM. Oxidation of methionine and was specified as variable modification, iodoacetamide derivative of cysteine as fixed. The Mascot results were loaded in Scaffold (version Scaffold_3.5, Proteome Software Inc., Portland, OR) to validate MS/MS based peptide and protein identifications. Peptide identifications were accepted if they could be established at greater than 80.0% probability as specified by the Peptide Prophet algorithm 30. Protein identifications were accepted if they could be established at greater than 95.0% probability and contained at least 2 identified peptides. Protein probabilities were assigned by the Protein Prophet algorithm 31. Proteins that contained similar peptides and could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony.For Flag-IP, washed immunoprecipitates were incubated with Flag-peptide (Sigma) to elute the precipitated protein from the anti-Flag coated beads. Samples were separated by SDS gel, stained and in gel digest were performed as described in standard protocols. Digests with different proteases (Trypsin, Elastase, Thermolysin) were performed in 0.1M NH4HCO3 (pH 8) overnight at 37°C. For one gel band about 0.1μg of protease was used. Peptides were extracted from the gel slices using 5% formic acid. All LC-MS/MS analyses were performed on an ion trap mass spectrometer (Agilent 6340, Agilent Technologies) coupled to an 1200 Agilent nanoflow system via a HPLC-Chip cube ESI interface. Alternatively, LC-MS/MS analyses were performed on a Q-TOF mass spectrometer (Agilent 6520, Agilent Technologies) coupled to a 1200 Agilent nanoflow system via a HPLC-Chip cube ESI interface. Peptides were elutes with a linear acetonitrile gradient with 1%/min at a flow rate of 300nl/min (starting with 3% acetonitrile). For raw data processing Mascot Distiller 2.4.2 (Matrix Science, London, UK) was used. The data from the Mascot searches was further condensed for the samples (experiment and control) in Scaffold 3.4.9 (Proteome Software, Portland, OR).

Affinity proteomics

Strep-Flag tandem affinity purification (SF-TAP) was performed as described 29. Prior to LC-MS/MS analysis, protein precipitates were subjected to tryptic proteolysis. Mass spectrometric analysis of SF-TAP purified samples was performed as described. Digested samples were separated on an UltiMate 3000 RSLCnano system, on-line coupled to a LTQ Orbitrap Velos (Thermo Fisher Scientific, Waltham, USA). All MS/MS samples were analyzed using Mascot (version 2.4, Matrix Science, Boston, MA, USA). Mascot was set up to search the human subset of the Swiss Prot database (Release 2012_05, 20245 entries), assuming trypsin as the digestion enzyme. Mascot was searched with a fragment ion mass tolerance of 1.00 Da and a parent ion tolerance of 10.0 PPM. Oxidation of methionine and was specified as variable modification, iodoacetamide derivative of cysteine as fixed. The Mascot results were loaded in Scaffold (version Scaffold_3.5, Proteome Software Inc., Portland, OR) to validate MS/MS based peptide and protein identifications. Peptide identifications were accepted if they could be established at greater than 80.0% probability as specified by the Peptide Prophet algorithm 30. Protein identifications were accepted if they could be established at greater than 95.0% probability and contained at least 2 identified peptides. Protein probabilities were assigned by the Protein Prophet algorithm 31. Proteins that contained similar peptides and could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony.

For Flag-IP, washed immunoprecipitates were incubated with Flag-peptide (Sigma) to elute the precipitated protein from the anti-Flag coated beads. Samples were separated by SDS gel, stained and in gel digest were performed as described in standard protocols. Digests with different proteases (Trypsin, Elastase, Thermolysin) were performed in 0.1M NH4HCO3 (pH 8) overnight at 37°C. For one gel band about 0.1μg of protease was used. Peptides were extracted from the gel slices using 5% formic acid. All LC-MS/MS analyses were performed on an ion trap mass spectrometer (Agilent 6340, Agilent Technologies) coupled to an 1200 Agilent nanoflow system via a HPLC-Chip cube ESI interface. Alternatively, LC-MS/MS analyses were performed on a Q-TOF mass spectrometer (Agilent 6520, Agilent Technologies) coupled to a 1200 Agilent nanoflow system via a HPLC-Chip cube ESI interface. Peptides were elutes with a linear acetonitrile gradient with 1%/min at a flow rate of 300nl/min (starting with 3% acetonitrile). For raw data processing Mascot Distiller 2.4.2 (Matrix Science, London, UK) was used. The data from the Mascot searches was further condensed for the samples (experiment and control) in Scaffold 3.4.9 (Proteome Software, Portland, OR).

Statistical analysesSigmaStat software was used to analyse statistical significance. All experiments were performed three times and diagrams show the mean +/− standard error of the mean (SEM). The tests used to calculate the significance are indicated in the corresponding figure legends for the individual experiments. A standard number of embryos (20 to 30 for Xenopus; 100 for zebrafish) were injected per experiment and non-viable embryos were excluded before gastrulation. The number of analysed embryos is given above the bars.

Statistical analyses

SigmaStat software was used to analyse statistical significance. All experiments were performed three times and diagrams show the mean +/− standard error of the mean (SEM). The tests used to calculate the significance are indicated in the corresponding figure legends for the individual experiments. A standard number of embryos (20 to 30 for Xenopus; 100 for zebrafish) were injected per experiment and non-viable embryos were excluded before gastrulation. The number of analysed embryos is given above the bars.

Supplementary Material12

Supplementary Material
