---
pmid: '22012985'
title: Leucine-rich repeat kinase 2 regulates autophagy through a calcium-dependent
  pathway involving NAADP.
authors:
- Gómez-Suaga P
- Luzón-Toro B
- Churamani D
- Zhang L
- Bloor-Young D
- Patel S
- Woodman PG
- Churchill GC
- Hilfiker S
journal: Hum Mol Genet
year: '2012'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3259011
doi: 10.1093/hmg/ddr481
---

# Leucine-rich repeat kinase 2 regulates autophagy through a calcium-dependent pathway involving NAADP.
**Authors:** Gómez-Suaga P, Luzón-Toro B, Churamani D, Zhang L, Bloor-Young D, Patel S, Woodman PG, Churchill GC, Hilfiker S
**Journal:** Hum Mol Genet (2012)
**DOI:** [10.1093/hmg/ddr481](https://doi.org/10.1093/hmg/ddr481)
**PMC:** [PMC3259011](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3259011/)

## Abstract

1. Hum Mol Genet. 2012 Feb 1;21(3):511-25. doi: 10.1093/hmg/ddr481. Epub 2011 Oct
 19.

Leucine-rich repeat kinase 2 regulates autophagy through a calcium-dependent 
pathway involving NAADP.

Gómez-Suaga P(1), Luzón-Toro B, Churamani D, Zhang L, Bloor-Young D, Patel S, 
Woodman PG, Churchill GC, Hilfiker S.

Author information:
(1)Institute of Parasitology and Biomedicine Lo´pez-Neyra, Consejo Superior de 
Investigaciones Científicas, Avda del Conocimiento s/n, 18100 Granada, Spain.

Mutations in the leucine-rich repeat kinase-2 (LRRK2) gene cause late-onset 
Parkinson's disease, but its physiological function has remained largely 
unknown. Here we report that LRRK2 activates a calcium-dependent protein kinase 
kinase-β (CaMKK-β)/adenosine monophosphate (AMP)-activated protein kinase (AMPK) 
pathway which is followed by a persistent increase in autophagosome formation. 
Simultaneously, LRKR2 overexpression increases the levels of the autophagy 
receptor p62 in a protein synthesis-dependent manner, and decreases the number 
of acidic lysosomes. The LRRK2-mediated effects result in increased sensitivity 
of cells to stressors associated with abnormal protein degradation. These 
effects can be mimicked by the lysosomal Ca(2+)-mobilizing messenger nicotinic 
acid adenine dinucleotide phosphate (NAADP) and can be reverted by an NAADP 
receptor antagonist or expression of dominant-negative receptor constructs. 
Collectively, our data indicate a molecular mechanism for LRRK2 deregulation of 
autophagy and reveal previously unidentified therapeutic targets.

DOI: 10.1093/hmg/ddr481
PMCID: PMC3259011
PMID: 22012985 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONAutosomal-dominant mutations in leucine-rich repeat kinase 2 (LRRK2) cause late-onset familial Parkinson's disease (PD) (1,2) which is symptomatically and neurochemically indistinguishable from sporadic PD. In addition, pathogenic mutations contribute to sporadic PD and variations increase risk for PD, suggesting that LRRK2 may be an important factor for idiopathic disease progression as well (3). The most prominent pathogenic LRRK2 mutation (G2019S), located within the kinase domain, has been consistently shown to enhance kinase activity in vitro (4–7), suggesting that it may display a dominant, gain-of-function phenotype. Overexpression of LRRK2 harbouring this disease-segregating mutation leads to neurotoxicity in vivo, and such neurotoxicity is linked to the kinase activity of LRRK2 (8–11). Thus, the pathogenic role of mutant LRRK2 seems to be dependent on catalytic activity.Overexpression of G2019S-mutant LRRK2 in neuronal cells decreases neurite process length with a concomitant accumulation of autophagic structures (12–14), and an impaired autophagic balance is also observed upon overexpression of LRRK2 in non-neuronal and yeast cells (15,16). This is in agreement with the findings that a significant portion of endogenous LRRK2 is localized to membranous structures including ER and endosomal–lysosomal compartments (17), and implicates LRRK2 as an important regulator of macroautophagy.Macroautophagy (hereafter referred to as autophagy) is a process whereby cytoplasmic constituents are engulfed within specialized double-membrane vesicles called autophagosomes, and subsequently delivered to the lysosome for degradation. Autophagy normally proceeds at a low, basal rate, which plays a key role in the homeostatic clearance of old or damaged organelles and proteins (18,19). Basal autophagy is especially high in neuronal cells, and neurons undergo degeneration when basal autophagic degradation is disrupted (20,21). Autophagic failure seems to underlie a variety of neurodegenerative diseases (22), and deregulation of autophagy is evident in the brains of PD patients (23).Despite the findings linking LRRK2 to deregulated autophagy, the exact mechanisms underlying such deregulation have not been characterized. In this work, we find that overexpression of LRRK2 causes an increase in autophagy induction through Ca2+-dependent activation of a CaMKK/adenosine monophosphate (AMP)-activated protein kinase (AMPK) pathway, which can be inhibited by calcium chelation or ectopic Bcl-2. At the same time, LRRK2 causes a partial increase in lysosomal pH and a decrease of cell survival in the presence of protein aggregation-induced stress. The LRRK2-mediated effects involve activation of nicotinic acid adenine dinucleotide phosphate (NAADP)-sensitive two-pore channels (TPCs) located on acidic stores, and can be blocked by a specific antagonist. Together, our data indicate a molecular mechanism by which LRRK2 may act to regulate lysosomal Ca2+ homeostasis, with downstream effects on autophagy as well as other Ca2+-dependent cellular events.

INTRODUCTION

Autosomal-dominant mutations in leucine-rich repeat kinase 2 (LRRK2) cause late-onset familial Parkinson's disease (PD) (1,2) which is symptomatically and neurochemically indistinguishable from sporadic PD. In addition, pathogenic mutations contribute to sporadic PD and variations increase risk for PD, suggesting that LRRK2 may be an important factor for idiopathic disease progression as well (3). The most prominent pathogenic LRRK2 mutation (G2019S), located within the kinase domain, has been consistently shown to enhance kinase activity in vitro (4–7), suggesting that it may display a dominant, gain-of-function phenotype. Overexpression of LRRK2 harbouring this disease-segregating mutation leads to neurotoxicity in vivo, and such neurotoxicity is linked to the kinase activity of LRRK2 (8–11). Thus, the pathogenic role of mutant LRRK2 seems to be dependent on catalytic activity.

Overexpression of G2019S-mutant LRRK2 in neuronal cells decreases neurite process length with a concomitant accumulation of autophagic structures (12–14), and an impaired autophagic balance is also observed upon overexpression of LRRK2 in non-neuronal and yeast cells (15,16). This is in agreement with the findings that a significant portion of endogenous LRRK2 is localized to membranous structures including ER and endosomal–lysosomal compartments (17), and implicates LRRK2 as an important regulator of macroautophagy.

Macroautophagy (hereafter referred to as autophagy) is a process whereby cytoplasmic constituents are engulfed within specialized double-membrane vesicles called autophagosomes, and subsequently delivered to the lysosome for degradation. Autophagy normally proceeds at a low, basal rate, which plays a key role in the homeostatic clearance of old or damaged organelles and proteins (18,19). Basal autophagy is especially high in neuronal cells, and neurons undergo degeneration when basal autophagic degradation is disrupted (20,21). Autophagic failure seems to underlie a variety of neurodegenerative diseases (22), and deregulation of autophagy is evident in the brains of PD patients (23).

Despite the findings linking LRRK2 to deregulated autophagy, the exact mechanisms underlying such deregulation have not been characterized. In this work, we find that overexpression of LRRK2 causes an increase in autophagy induction through Ca2+-dependent activation of a CaMKK/adenosine monophosphate (AMP)-activated protein kinase (AMPK) pathway, which can be inhibited by calcium chelation or ectopic Bcl-2. At the same time, LRRK2 causes a partial increase in lysosomal pH and a decrease of cell survival in the presence of protein aggregation-induced stress. The LRRK2-mediated effects involve activation of nicotinic acid adenine dinucleotide phosphate (NAADP)-sensitive two-pore channels (TPCs) located on acidic stores, and can be blocked by a specific antagonist. Together, our data indicate a molecular mechanism by which LRRK2 may act to regulate lysosomal Ca2+ homeostasis, with downstream effects on autophagy as well as other Ca2+-dependent cellular events.

RESULTSLRRK2 overexpression causes autophagy inductionTo study the effect of LRRK2 on basal autophagy, we selected a variety of constructs including full-length LRRK2 (fl-kinase) and variants bearing the pathogenic G2019S mutation (fl-G2019S) or the kinase-inactivating K1906M mutation (fl-K1906M), respectively. To evaluate whether effects are due to the kinase domain and activity of LRRK2 (12), we also used the kinase domain (kinase) or variants therof (G2019S, K1906M) on their own, respectively. Overexpressed proteins were mostly cytosolic, wild-type (WT) and mutant proteins displayed similar overexpression levels, and no cellular toxicity was apparent 24 or 48 h upon transfection, as previously described (10) (Supplementary Material, Fig. S1).We next addressed the effects of LRRK2 overexpression on autophagosome formation. LC3 is the mammalian homologue of the yeast Atg8 protein, which is covalently modified and re-distributes to autophagic vacuoles during induction of autophagy (23). This process can be followed by the appearance of LC3 puncta or ring-like structures and by a mobility shift from LC3I to LC3II by SDS–PAGE. Cells transiently transfected with LC3–GFP alone, or co-transfected with empty vector and analysed 48 h after transfection, display few cytoplasmic LC3–GFP punctae (Fig. 1A and B). An increase in LC3–GFP punctae was observed when co-transfecting with full-length wild-type as well as with G2019S-mutant LRRK2, but not with the inactive K1906M mutant (Fig. 1B). A similar increase in LC3–GFP punctae was observed with wild-type or G2019S-mutant, but not K1906M-mutant kinase domain only (Fig. 1A and B), indicating that the effects on autophagy are, at least in part, due to the kinase domain and activity of LRRK2. Wild-type and G2019S-mutant full-length LRRK2 increased autophagosome numbers to a similar degree and comparable with that of wild-type or G2019S-mutant kinase domain only (Fig. 1B), possibly due to saturating levels of overexpression of the various active constructs with respect to downstream targets.
Figure 1.Effects of LRRK2 overexpression on autophagosome formation. (A) Example of HEK293T cells transfected with LC3–GFP and either empty vector (ctrl), catalytically inactive K1906M-mutant LRRK2 kinase domain (K1906M), wild-type LRRK2-kinase domain (kinase) or catalytically hyperactive G2019S-mutant LRRK2 kinase domain (G2019S). LC3–GFP fluorescence (green) and nuclear DAPI staining (blue). Scale bar, 10 μm. (B) Quantification of the number of LC3–GFP punctae per cell in the absence (white bars) or the presence of 3-MA (5 mm, 24 h) (grey bars). Cells were co-transfected with LC3–GFP and the indicated constructs, and LC3–GFP punctae quantified 48 h after transfection. Rapamycin (100 nm, 12 h) was used as positive control. Bars represent mean ± s.e.m. (n = 4); *P < 0.001. (C) Top: cells were transfected with either K1906M or kinase constructs, and extracts analysed for endogenous LC3I and LC3II blotting with an anti-LC3 antibody (Cell Signalling). Bottom: quantification of LC3II/tubulin from different experiments (mean ± s.e.m. (n = 3); *P < 0.05. (D) Representative electron micrographs of cells expressing LRRK2 kinase domain. Arrows point to distinct autophagic structures [scale bar 200 nm (left) or 500 nm (right)]. (E) Cells were co-transfected with EGFP and empty vector (GFP) or kinase domain (kinase), and FACS sorted before processing for electron microscopy. Stereological analysis of the number of intersections per cell, number of intersections per autophagic structures (AVs), percentage of cellular volume occupied by AVs and AV diameter were quantified from three individual cells per condition. *P < 0.05.As a positive control, inhibition of the mammalian target of rapamycin (mTOR) protein kinase by rapamycin was found to induce autophagy to a similar degree (Fig. 1B). The phosphatidylinositol 3-kinase inhibitor 3-methyladenine (3-MA) is thought to suppress autophagy by inhibiting the production of phosphatidylinositol-3-phosphate required for autophagosome formation (24), and was found to block the increase in autophagosome numbers mediated by rapamycin, but not by LRRK2 overexpression (Fig. 1B). Such differential 3-MA sensitivity has previously been described (13), and points towards a possible mechanistic difference between mTOR- and LRRK2-mediated autophagy induction. An increase in autophagosome numbers was further reflected by an increase in the level of endogenous LC3II (Fig. 1C), and transmission electron microscopy confirmed that the induction of LC3-positive vesicles by LRRK2 overexpression correlated with an increase in the number of autophagic structures (Fig. 1D and E).We next used a pH-sensitive tagged LC3 construct consisting of a tandem fusion of the red, acid-insensitive mCherry (pKa < 4.5) and the acid-sensitive GFP (pKa 6.0) (td-tag-LC3) (25,26). Td-tag-LC3 will emit yellow (green merged with red) fluorescence in non-acidic structures, but appear as red only in autolysosomes due to the quenching of GFP in these acidic structures (25,26). Indeed, td-tag-LC3 was found to faithfully track autophagosome maturation in transfected cells, thus allowing for the identification of possible deficits in autophagic maturation (Supplementary Material, Fig. S2). Co-transfection of td-tag-LC3 with G2019S-mutant LRRK2 kinase domain increased both early and late autophagic structures (Fig. 2A and B). Such increase in both yellow and red punctae was also observed with wild-type kinase domain or with full-length LRRK2, but not with the catalytically inactive K1906M mutant (Fig. 2B). As a positive control for blocking autophagosome maturation, we used trichostatin A, an inhibitor of histone deacetylases (HDACs) known to be required for the fusion of autophagosomes with lysosomes (27). Indeed, trichostatin A treatment significantly increased the percentage of yellow punctae (and decreased the number of red punctae), in contrast to LRRK2 expression or autophagy induction by rapamycin (Fig. 2C and D). These data indicate that LRRK2 overexpression enhances autophagosome numbers and at least the initial steps of autophagosome maturation in a manner dependent on kinase activity.
Figure 2.Effects of LRRK2 overexpression on autophagosome maturation. (A) Example of HEK293T cells co-transfected with tandem-tagged-LC3 (td-tag-LC3) and either empty vector (pCMV) or G2019S-mutant LRRK2 kinase domain. White: co-localization mask to highlight yellow punctae. Scale bar, 10 μm. (B) Quantification of the number of early, yellow-only td-tag-LC3 punctae per cell (top), or the overall fluorescence from late, red-only td-tag-LC3 punctae per cell (bottom). Cells were co-transfected with td-tag-LC3 and either empty pCMV vector (ctrl), wild-type or mutant kinase or full-length LRRK2 kinase (fl-kinase) constructs as indicated, and quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (C) Example of cells co-transfected with td-tag-LC3 and LRRK2 kinase domain, or transfected with td-tag-LC3 and treated with trichostatin A (300 nm, 4 h). White: co-localization mask to highlight yellow punctae. Scale bar, 10 μm. (D) Quantification of the percentage of early, yellow td-tag-LC3 punctae per cell. Cells were co-transfected with td-tag-LC3 and either kinase or empty vector. The latter were either left untreated (ctrl), or treated with rapamycin (rap.; 100 nm, 4 h) or trichostatin A (tricho.; 300 nm, 4 h) prior to fixation. Note that the percentage of yellow punctae in trichostatin-treated cells is increased because of the decrease in the number of red punctae. Bars represent mean ± s.e.m. (n = 2); *P < 0.05.LRRK2 overexpression activates the Ca2+/CaMKK/AMPK pathwayThe increase in autophagosome numbers probably results from cytosolic events regulated by LRRK2. As recent studies imply an important role for AMPK in basal autophagy regulation (28), we evaluated the effects of LRRK2 on AMPK activity. Expression of wild-type and G2019S-mutant LRRK2 kinase domain, but not its inactive mutant, increased AMPK activity (Fig. 3A and B). This was not accompanied by inhibition of mTORC1 activity when compared with rapamycin or torin treatment (Fig. 3C and D and Supplementary Material, Fig. S3) (29). Accordingly, inhibition of AMPK by a pharmacological inhibitor (compound C) (30) attenuated the increase in autophagosome numbers induced by LRRK2 kinase domain overexpression (Fig. 3E). Compound C also attenuated the increase in autophagosome numbers observed with ionomycin and ATP, two treatments previously shown to activate AMPK (31), but was without effect on autophagosome numbers induced by rapamycin (Fig. 3E).
Figure 3.Effects of LRRK2 overexpression on AMPK and TORC1 signalling. (A) HEK293T cells were transfected with either empty vector (ctrl) or indicated constructs, and cell extracts analysed for phospho-ACC, phospho-AMPK and AMPK. (B) Quantification of the type of experiments depicted in (A). Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (C) As in (A), but extracts analysed for the indicated TORC1 substrates. (D) Quantification of the type of experiments depicted in (C). Bars represent mean ± s.e.m. (n = 3). (E) Cells were co-transfected with LC3–GFP and either empty vector (ctrl) or LRRK2 kinase domain, and control-transfected cells were treated with rapamycin (100 nm, 12 h), ionomycin (2.5 μm, 12 h) or ATP (100 μm, 12 h) where indicated. Cells were incubated in the absence (white bars) or the presence (grey bars) of compound C (5 μm, 24 h), and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.Further evidence for an involvement of AMPK was gained by expressing wild-type, dominant-negative and constitutively active forms of this kinase (32). Wild-type and mutant AMPKα1 forms were overexpressed to similar degrees and without effect on cellular viability, with AMPKα2 forms expressed to a lesser degree and displaying cellular toxicity (Supplementary Material, Fig. S4). Expression of wild-type and constitutively active, but not dominant-negative, AMPKα1 induced an increase in autophagosome numbers (Fig. 4A). Importantly, the dominant-negative AMPKα1 construct was able to inhibit the increase in autophagosome numbers observed upon active LRRK2 kinase domain co-expression (Fig. 4B). Such inhibition was not observed when autophagic on-rate was increased by distinct means, indicating a specific involvement of the AMPK pathway in the LRRK2-mediated process (Supplementary Material, Fig. S4).
Figure 4.Effects of LRRK2 overexpression on AMPK and intracellular calcium. (A) HEK293T cells were co-transfected with LC3–GFP and either empty vector (ctrl), human wild-type AMPKα1 (1-wt), dominant-negative (1-DN) or constitutively active (1-CA) AMPKα1, and LC3–GFP punctae quantified 48 h after transfection. Rapamycin (100 nm, 12 h) (rap.) was used as control. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (B) Cells were transfected with LC3–GFP along with either empty vector (ctrl), or LRRK2 kinase domain (kin) in the presence or in the absence of wild-type (1-wt) or dominant-negative (1-DN) AMPKα1 constructs as indicated, and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.01. (C) Cells were co-transfected with LC3–GFP and either empty vector (ctrl) or LRRK2 kinase domain (kinase), and control-transfected cells were treated with rapamycin (100 nm, 12 h), ionomycin (2.5 μm, 12 h) or ATP (100 μm, 12 h) where indicated. Cells were incubated in the absence (white bars) or in the presence (grey bars) of STO-609 (20 μm, 24 h), a CaMKKβ inhibitor and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (D) Cells were co-transfected with LC3–GFP and either empty vector (ctrl) or the indicated wild-type or mutant kinase domain or full-length (fl) constructs, and either left untreated (white bars) or treated with BAPTA-AM (5 μm, 2 h) (grey bars) prior to analysis for LC3–GFP punctae 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (E) Cells were transfected with LC3–GFP along with either K1906M mutant (white bars) or wild-type (grey bars) LRRK2 kinase domain and either control empty vector (−) or the indicated chimeric Bcl constructs, and LC3–GFP punctae analysed 24 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.AMPK can be activated by CaMKK-β in response to an increase in the cytosolic-free calcium [Ca2+]c (31). In order to test whether LRRK2 expression regulates the Ca2+/CamKK-β/AMPK pathway, we first treated cells with a CaMKK-α/β inhibitor (STO-609) (33). This compound attenuated the increase in autophagosome formation induced by LRRK2 kinase domain (Fig. 4C). Such inhibition was similar to that observed when cells were treated with two stimuli which increase [Ca2+]c by distinct means: ATP, which acts on P2 purinoreceptors to generate inositol 1,4,5-triphosphate (IP3) that triggers the release of Ca2+ from the ER through IP3 receptor-regulated channels, or ionomycin, a Ca2+ ionophore that serves as a mobile Ca2+ carrier to equilibrate Ca2+ levels across biological membranes, including plasma and ER membranes (31,34,35) (Fig. 4C). In contrast, the CaMKK-α/β inhibitor had no effect on autophagosome numbers elicited by rapamycin treatment (Fig. 4C). Remarkably, addition of the intracellular Ca2+ chelator BAPTA-AM (bis-(O-aminophenoxy)-ethane-N,N,N,N′-tetraacetic acid/tetra(acetoxymethyl)-ester) to LRRK2 kinase domain or full-length LRRK2-transfected cells for the last 2 hours completely abolished the increase in autophagosome numbers (Fig. 4D). BAPTA-AM was further able to block the ionomycin-mediated, but not the rapamycin-mediated, increase in autophagosome numbers (Supplementary Material, Fig. S4). Thus, a Ca2+/CaMKK/AMPK pathway seems to be required for the increased formation of autophagosomes in response to active LRRK2 overexpression.ER-localized Bcl-2 inhibits LRRK2-mediated autophagy inductionBcl-2 has been reported to regulate cellular Ca2+ handling and Ca2+/CaMKK/AMPK-induced autophagy (31,36). Thus, we tested whether expression of bcl-2 could inhibit the LRRK2-mediated increase in autophagosome numbers via its effects on Ca2+ homeostasis. For this purpose, we expressed wild-type bcl-2 (Bcl-WT), which is mostly mitochondrial (36,37), or mutants in which the C-terminal hydrophobic sequence has been either removed (Bcl-cyt), resulting in cytosolic expression, or exchanged to a corresponding membrane anchor from an ER-specific isoform of cytochrome b5 (Bcl-ER) or L. monocytogenes ActA (Bcl-mito), resulting in distinct subcellular localizations (31,38). Western blot analysis confirmed equal expression levels, and only slight effects on cytotoxicity were observed when overexpressing Bcl-wt or Bcl-mito, but not Bcl-cyto or Bcl-ER, respectively (Supplementary Material, Fig. S4). Importantly, only Bcl-ER was able to fully inhibit the LRRK2 kinase domain-mediated increase in autophagosome numbers (Fig. 4E). ER-localized Bcl-2 has previously been demonstrated to lower [Ca2+]ER and agonist-induced Ca2+ leak from the ER (39). Together with the inhibitory effect of BAPTA-AM and inhibition of CaMKK on the LRRK2-mediated increase in autophagosome numbers, these results indicate that active LRRK2 may increase [Ca2+]c by directly or indirectly inducing Ca2+ release from the ER. This release is likely mediated by IP3 receptors, as the increase in autophagosome numbers upon LRRK2 expression could not be blocked by dantrolene, a ryanodine receptor antagonist (Supplementary Material, Fig. S4).LRRK2 alkalinizes a subpopulation of lysosomesOverexpression of LRRK2 has previously been shown to lead to accumulation of multi-vesicular bodies and autophagosomes containing incompletely degraded material and p62 (15), a classical macroautophagy substrate (40), indicating possible defects in autophagic degradation. Indeed, overexpression of wild-type or G2019S-mutant, but not inactive K1906M-mutant, kinase domain led to a significant increase in p62 levels and p62-positive structures (Fig. 5A–C). However, the LRRK2-mediated increase in p62 levels could be blocked by protein synthesis inhibitors (Fig. 5D), indicating that it was not reflecting changes in autophagic flux. Identical results were obtained when using ionomycin or thapsigargin (Supplementary Material, Fig. S5), suggesting that the LRRK2-mediated effects on p62 levels may be due to Ca2+-dependent changes in protein synthesis (41).
Figure 5.Effects of LRRK2 overexpression on p62 levels. (A) HEK293T cells were transfected with either control empty vector (ctrl) or wild-type or mutant kinase domain constructs as indicated, and extracts analysed for endogenous p62 levels. (B) Quantification of the type of experiments as depicted in (A). Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (C) Example of cells co-transfected with LC3–GFP (green, left) and LRRK2 kinase domain and co-stained with anti-p62 (red, middle). Scale bar, 10 μm. (D) Cells were transfected with indicated constructs, and treated with puromycin (1 μm, 6 h) where indicated before analysis and quantification of p62 levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.A possible defect in lysosomal homeostasis, however, became apparent when employing lipid staining with BODIPY 493/503 (42), which revealed an increase in lipid droplet numbers in cells overexpressing LRRK2 kinase domain (Fig. 6A and B). In addition, lysotracker DND-99 or Lysosensor staining to determine lysosomal pH indicated a decrease in the number of positive punctae in cells overexpressing active LRRK2 kinase domain (Fig. 6C and D). This was not due to a decrease in lysosomal content (43), as no change in the levels of LAMP1 or LAMP2 were observed (Supplementary Material, Fig. S6). Interestingly, quantification of the percent colocalization of LAMP2 and lysotracker showed an increase in the number of structures positive for LAMP2, but negative for lysotracker, suggesting that LRRK2 increases pH in a subpopulation of lysosomes, without affecting the processing of lysosomal hydrolases (Fig. 6E and Supplementary Material, Fig. S6).
Figure 6.Effects of LRRK2 on lipid accumulation and lysosomal pH, and effects of NAADP. (A) Example of HEK293T cells co-transfected with mCherry and either K1906M-mutant or wild-type LRRK2 kinase domain, and stained with BODIPY 493/503 (green). Scale bar, 10 μm. (B) Percentage change in the number of lipid droplets quantified from experiments as in (A). Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (C) The number of lysosensor-positive structures in cells expressing K1906M or kinase domain. Bars represent mean ± s.e.m. (n = 4); *P < 0.05. (D) Percentage change in the number of lysotracker-positive structures in cells co-transfected with EGFP and either K1906M or kinase domain constructs. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (E) Quantification of LAMP2-positive structures positive for lysotracker in K1906M or kinase-expressing cells. Ten random cells were analysed per condition per experiment. Bars represent mean ± s.e.m. (n = 2); *P < 0.05. (F) Cells were transfected with LC3–GFP and either left untreated, or treated with the indicated concentrations of NAADP-AM or with 1 μm NED-19 for 12 h, in the absence (white bars) or in the presence (grey bars) of BAPTA-AM (5 μm, 2 h), and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (G) Cells were co-transfected with LC3–GFP and either control empty vector (white bars) or bcl-ER (grey bars), and either left untreated (ctrl), treated with NAADP-AM (100 nm, 4 h), starvation (Earle’s salt solution, 4 h), rapamycin (200 nm, 4 h) or torin (50 nm, 4 h) as indicated, and LC3–GFP punctae analysed 24 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (H) Cells were transfected with LC3–GFP and either left untreated (ctrl), or treated with NAADP-AM (100 nm, 12 h) or rapamycin (100 nm, 12 h) as indicated in the presence or in the absence of 3-MA (5 mm, 24 h) (grey bars). Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (I) Cells were either left untreated, or treated with NAADP-AM (100 nm, 12 h) as indicated, and cell extracts analysed and quantified for endogenous LC3. Bars represent mean ± s.e.m. (n = 2); *P < 0.05. (J) Cells were left untreated or treated with NAADP-AM (100 nm, 12 h) or puromycin (1 μm, 6 h) as indicated, and extracts analysed and quantified for p62 levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (K) Non-treated or NAADP-AM-treated cells were stained with BODIPY 493/503 and analysed for the percentage change in the number of lipid droplets. Bars represent mean ± s.e.m. (n = 3); *P < 0.01. (L) Non-treated or NAADP-AM-treated cells were stained with lysotracker and the percentage change quantified. Bars represent mean ± s.e.m. (n = 3); *P < 0.005.LRRK2 acts through NAADP receptorsThere is considerable evidence that acidic organelles such as endosomes and lysosomes are significant stores of Ca2+ that can be released by the Ca2+ mobilizing messenger NAADP (44,45). NAADP-evoked cytosolic Ca2+ signals are accompanied by partial alkalinization of acidic stores (46) and amplification by ER Ca2+ stores (47), and lysosomal Ca2+ depletion causes lipid accumulation (48). That LRRK2 increases lipid droplet numbers (Fig. 6), causes an increase in lysosomal pH (Fig. 6) and regulates Ca2+-dependent events in the cytosol which likely also involve the ER (Figs. 4 and 5) prompted us to consider the involvement of the NAADP pathway in mediating the effects of LRRK2.Application of NAADP-AM was found to increase autophagosome numbers (Fig. 6F). This increase was abolished at higher NAADP concentrations, in agreement with the notion that NAADP-induced Ca2+ release desensitizes at high ligand concentrations (49,50). The effect of NAADP on autophagosome formation could be blocked by BAPTA-AM (Fig. 6F) and was accompanied by an increase in AMPK activity (Supplementary Material, Fig. S6). The autophagy induction observed with NAADP could be inhibited by Bcl-ER (Fig. 6G), similar to starvation-induced autophagy (36,37), but distinct from autophagy induced by rapamycin or torin treatment (Fig. 6G). In addition, the autophagy induced by NAADP was insensitive to 3-MA, in contrast to TORC1-dependent autophagy inducers (Fig. 6H). Furthermore, NAADP caused an increase in LC3II levels (Fig. 6I), a protein synthesis-dependent increase in p62 levels (Fig. 6J), increased lipid droplet numbers (Fig. 6K) and increased lysosomal pH (Fig. 6L), thus fully mimicking the effects of LRRK2 expression.TPCs have recently emerged as the likely targets for NAADP, with TPC1 localized to both endosomes and lysosomes, and TPC2 mainly to lysosomes (50,51). Point mutants in the putative helix of the pore-forming regions have been described, which seem to largely abolish Ca2+ conductance and uncover dominant-negative effects with respect to NAADP-mediated Ca2+ signals (51,52). Thus, we evaluated whether co-expressing wild-type or mutant TPCs may interfere with LRRK2-mediated autophagosome formation. Expression of mRFP-tagged TPC constructs indicated that wild-type and mutant versions were similarly overexpressed and localized to intracellular compartments, with TPC2 partially co-localizing with the lysosomal marker LAMP2, as previously described (51). None of the TPC constructs displayed cytotoxicity at 24 h upon overexpression, but TPC1 and its mutant were found to increase autophagosome formation (Supplementary Material, Fig. S7). Whilst consistent with the involvement of TPC1 in endolysosomal trafficking (53), this finding precluded further analysis of a possible reversal of the LRRK2-mediated increase in autophagosome numbers by TPC1 constructs. In contrast, TPC2 and its mutant did not display effects on autophagosome numbers when expressed on their own (Fig. 7A). Importantly, mutant but not wild-type TPC2 was able to block the effect of LRRK2 kinase domain on autophagosome numbers (Fig. 7A).
Figure 7.A link between LRRK2 and NAADP-sensitive receptors. (A) HEK 293T cells were transfected with LC3–GFP along with K1906M-mutant or wild-type kinase domain and either empty vector (pCMV), TPC2 or mutant TPC2 as indicated, and LC3–GFP punctae quantified 24 h after transfection. Bars represent mean ± s.e.m. (n = 5); *P < 0.005. (B) Cells were co-transfected with LC3–GFP and the indicated kinase domain or full-length (fl) LRRK2 constructs, and either left untreated (white bars) or treated with NED-19 (1 μm, 12 h) (grey bars), followed by quantification of LC3–GFP punctae 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (C) Transfected cells were treated with NED-19 (1 μm, 12 h) where indicated before analysis for p62 levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (D) As in (C), but extracts analysed and quantified for LC3II levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (E) Transfected cells were treated with NED-19 (1 μm, 12 h) where indicated before staining with lysotracker. Bars represent mean ± s.e.m. (n = 4); *P < 0.05. (F) Cells were transfected with TPC2–GFP or GFP vector, and extracts (1 mg) subjected to immunoprecipitation with an anti-GFP antibody. Note that TPC2–GFP runs as distinct bands corresponding to core glycosylated and mature fully glycosylated versions, respectively (67). Co-immunoprecipitated endogenous LRRK2 was detected using an anti-LRRK2 antibody (MJFF2, Epitomics). Note the additional presence of lower molecular-weight bands, which are likely distinct degradation products of endogenous LRRK2. Inputs (5%) were run along-side the immunoprecipitates. Representative of a total of seven independent experiments.We next evaluated whether NED-19, a newly described NAADP antagonist (54), could reverse the LRRK2-mediated effects on autophagic flux. NED-19 on its own was without effect on autophagosome numbers, but could block the increase in autophagosome numbers mediated by active kinase domain or full-length LRRK2 (Fig. 7B). This was accompanied largely by a reversal in the LRRK2-mediated changes on AMPK activity (Supplementary Material, Fig. S7), p62 levels, LC3II levels and lysotracker-positive structures (Fig. 7C–E). Finally, when probing for an interaction between endogenous LRRK2 and GFP–TPC2, LRRK2 was found to co-immunoprecipitate with GFP–TPC2 and vice versa (Fig. 7F and Supplementary Material, Fig. S7). Together, these data indicate that NAADP mimicks the effects of active LRRK2, whilst NED-19 application can revert those effects, and suggest that NAADP receptors may be direct targets for regulation by LRRK2.LRRK2 sensitizes cells to cell death induced by proteasome inhibitionAs the observed LRRK2-mediated effects were not cytotoxic per se, we next wondered whether they would sensitize cells towards cell death under certain conditions (55). When cells were treated with the proteasomal inhibitor MG-132, cell death was markedly accelerated in cells overexpressing active, but not inactive, LRRK2 kinase domain (Fig. 8A). The increased cell death in the presence of MG-132 in LRRK2 kinase domain-expressing cells could be blocked by NED-19 (Fig. 8B), supporting the involvement of the NAADP pathway in this process. Interestingly, application of rapamycin or torin, two specific TORC1 inhibitors, reverted the LRRK2-mediated effects on cell death in the presence of MG-132, whilst L690,440, a TORC1-independent autophagy-enhancing compound (56), was unable to do so (Fig. 8A and C). Rapamycin and torin also largely reversed the effects of LRRK2 on p62 accumulation and on the decrease in lysotracker staining (Fig. 8D and E). Measurements in dopaminergic neuroendocrine PC12 cells yielded similar results. The NAADP-receptor antagonist NED-19 was able to revert the LRRK2-mediated increase in autophagosome numbers, the LRRK2-mediated decrease in lysosensor-positive structures and the LRRK2-mediated increase in cell death in the presence of MG-132 (Supplementary Material, Fig. S8). Together, these data indicate that additional interference with proteasomal degradation is detrimental to cells overexpressing catalytically active LRRK2, and that blocking NAADP-mediated signalling, or enhancing autophagic flux and/or suppressing protein synthesis or affecting lysosomal homeostasis through a TORC1-dependent pathway can overcome the enhanced susceptibility to cell death induced by LRRK2 in non-neuronal as well as neuroendocrine cells.
Figure 8.Effects of LRRK2 expression on cellular viability in the presence of proteasomal inhibitor. (A) HEK293T cells were co-transfected with GFP and K1906M or kinase domain constructs as indicated and either left untreated (−), treated with rapamycin (100 nm, 4 h), MG-132 (500 nm, 12 h) or both as indicated. Cells were analysed for cell death 48 h after transfection using Hoechst staining. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (B) As in (A), but cells treated with NED-19 (1 μm, 12 h) as indicated. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (C) As in (A), but cells treated with torin (500 nm, 4 h) or L690,440 (10 μm, 4 h) as indicated. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (D) Left: cells were transfected with either empty vector or kinase domain construct, and treated with rapamycin (200 nm, 12 h) where indicated before analysis for p62 levels. Right: quantification of experiments where bars represent mean ± s.e.m. (n = 3); *P < 0.05. (E) Transfected cells were treated as indicated before staining with lysotracker. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.

LRRK2 overexpression causes autophagy inductionTo study the effect of LRRK2 on basal autophagy, we selected a variety of constructs including full-length LRRK2 (fl-kinase) and variants bearing the pathogenic G2019S mutation (fl-G2019S) or the kinase-inactivating K1906M mutation (fl-K1906M), respectively. To evaluate whether effects are due to the kinase domain and activity of LRRK2 (12), we also used the kinase domain (kinase) or variants therof (G2019S, K1906M) on their own, respectively. Overexpressed proteins were mostly cytosolic, wild-type (WT) and mutant proteins displayed similar overexpression levels, and no cellular toxicity was apparent 24 or 48 h upon transfection, as previously described (10) (Supplementary Material, Fig. S1).We next addressed the effects of LRRK2 overexpression on autophagosome formation. LC3 is the mammalian homologue of the yeast Atg8 protein, which is covalently modified and re-distributes to autophagic vacuoles during induction of autophagy (23). This process can be followed by the appearance of LC3 puncta or ring-like structures and by a mobility shift from LC3I to LC3II by SDS–PAGE. Cells transiently transfected with LC3–GFP alone, or co-transfected with empty vector and analysed 48 h after transfection, display few cytoplasmic LC3–GFP punctae (Fig. 1A and B). An increase in LC3–GFP punctae was observed when co-transfecting with full-length wild-type as well as with G2019S-mutant LRRK2, but not with the inactive K1906M mutant (Fig. 1B). A similar increase in LC3–GFP punctae was observed with wild-type or G2019S-mutant, but not K1906M-mutant kinase domain only (Fig. 1A and B), indicating that the effects on autophagy are, at least in part, due to the kinase domain and activity of LRRK2. Wild-type and G2019S-mutant full-length LRRK2 increased autophagosome numbers to a similar degree and comparable with that of wild-type or G2019S-mutant kinase domain only (Fig. 1B), possibly due to saturating levels of overexpression of the various active constructs with respect to downstream targets.
Figure 1.Effects of LRRK2 overexpression on autophagosome formation. (A) Example of HEK293T cells transfected with LC3–GFP and either empty vector (ctrl), catalytically inactive K1906M-mutant LRRK2 kinase domain (K1906M), wild-type LRRK2-kinase domain (kinase) or catalytically hyperactive G2019S-mutant LRRK2 kinase domain (G2019S). LC3–GFP fluorescence (green) and nuclear DAPI staining (blue). Scale bar, 10 μm. (B) Quantification of the number of LC3–GFP punctae per cell in the absence (white bars) or the presence of 3-MA (5 mm, 24 h) (grey bars). Cells were co-transfected with LC3–GFP and the indicated constructs, and LC3–GFP punctae quantified 48 h after transfection. Rapamycin (100 nm, 12 h) was used as positive control. Bars represent mean ± s.e.m. (n = 4); *P < 0.001. (C) Top: cells were transfected with either K1906M or kinase constructs, and extracts analysed for endogenous LC3I and LC3II blotting with an anti-LC3 antibody (Cell Signalling). Bottom: quantification of LC3II/tubulin from different experiments (mean ± s.e.m. (n = 3); *P < 0.05. (D) Representative electron micrographs of cells expressing LRRK2 kinase domain. Arrows point to distinct autophagic structures [scale bar 200 nm (left) or 500 nm (right)]. (E) Cells were co-transfected with EGFP and empty vector (GFP) or kinase domain (kinase), and FACS sorted before processing for electron microscopy. Stereological analysis of the number of intersections per cell, number of intersections per autophagic structures (AVs), percentage of cellular volume occupied by AVs and AV diameter were quantified from three individual cells per condition. *P < 0.05.As a positive control, inhibition of the mammalian target of rapamycin (mTOR) protein kinase by rapamycin was found to induce autophagy to a similar degree (Fig. 1B). The phosphatidylinositol 3-kinase inhibitor 3-methyladenine (3-MA) is thought to suppress autophagy by inhibiting the production of phosphatidylinositol-3-phosphate required for autophagosome formation (24), and was found to block the increase in autophagosome numbers mediated by rapamycin, but not by LRRK2 overexpression (Fig. 1B). Such differential 3-MA sensitivity has previously been described (13), and points towards a possible mechanistic difference between mTOR- and LRRK2-mediated autophagy induction. An increase in autophagosome numbers was further reflected by an increase in the level of endogenous LC3II (Fig. 1C), and transmission electron microscopy confirmed that the induction of LC3-positive vesicles by LRRK2 overexpression correlated with an increase in the number of autophagic structures (Fig. 1D and E).We next used a pH-sensitive tagged LC3 construct consisting of a tandem fusion of the red, acid-insensitive mCherry (pKa < 4.5) and the acid-sensitive GFP (pKa 6.0) (td-tag-LC3) (25,26). Td-tag-LC3 will emit yellow (green merged with red) fluorescence in non-acidic structures, but appear as red only in autolysosomes due to the quenching of GFP in these acidic structures (25,26). Indeed, td-tag-LC3 was found to faithfully track autophagosome maturation in transfected cells, thus allowing for the identification of possible deficits in autophagic maturation (Supplementary Material, Fig. S2). Co-transfection of td-tag-LC3 with G2019S-mutant LRRK2 kinase domain increased both early and late autophagic structures (Fig. 2A and B). Such increase in both yellow and red punctae was also observed with wild-type kinase domain or with full-length LRRK2, but not with the catalytically inactive K1906M mutant (Fig. 2B). As a positive control for blocking autophagosome maturation, we used trichostatin A, an inhibitor of histone deacetylases (HDACs) known to be required for the fusion of autophagosomes with lysosomes (27). Indeed, trichostatin A treatment significantly increased the percentage of yellow punctae (and decreased the number of red punctae), in contrast to LRRK2 expression or autophagy induction by rapamycin (Fig. 2C and D). These data indicate that LRRK2 overexpression enhances autophagosome numbers and at least the initial steps of autophagosome maturation in a manner dependent on kinase activity.
Figure 2.Effects of LRRK2 overexpression on autophagosome maturation. (A) Example of HEK293T cells co-transfected with tandem-tagged-LC3 (td-tag-LC3) and either empty vector (pCMV) or G2019S-mutant LRRK2 kinase domain. White: co-localization mask to highlight yellow punctae. Scale bar, 10 μm. (B) Quantification of the number of early, yellow-only td-tag-LC3 punctae per cell (top), or the overall fluorescence from late, red-only td-tag-LC3 punctae per cell (bottom). Cells were co-transfected with td-tag-LC3 and either empty pCMV vector (ctrl), wild-type or mutant kinase or full-length LRRK2 kinase (fl-kinase) constructs as indicated, and quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (C) Example of cells co-transfected with td-tag-LC3 and LRRK2 kinase domain, or transfected with td-tag-LC3 and treated with trichostatin A (300 nm, 4 h). White: co-localization mask to highlight yellow punctae. Scale bar, 10 μm. (D) Quantification of the percentage of early, yellow td-tag-LC3 punctae per cell. Cells were co-transfected with td-tag-LC3 and either kinase or empty vector. The latter were either left untreated (ctrl), or treated with rapamycin (rap.; 100 nm, 4 h) or trichostatin A (tricho.; 300 nm, 4 h) prior to fixation. Note that the percentage of yellow punctae in trichostatin-treated cells is increased because of the decrease in the number of red punctae. Bars represent mean ± s.e.m. (n = 2); *P < 0.05.

LRRK2 overexpression causes autophagy induction

To study the effect of LRRK2 on basal autophagy, we selected a variety of constructs including full-length LRRK2 (fl-kinase) and variants bearing the pathogenic G2019S mutation (fl-G2019S) or the kinase-inactivating K1906M mutation (fl-K1906M), respectively. To evaluate whether effects are due to the kinase domain and activity of LRRK2 (12), we also used the kinase domain (kinase) or variants therof (G2019S, K1906M) on their own, respectively. Overexpressed proteins were mostly cytosolic, wild-type (WT) and mutant proteins displayed similar overexpression levels, and no cellular toxicity was apparent 24 or 48 h upon transfection, as previously described (10) (Supplementary Material, Fig. S1).

We next addressed the effects of LRRK2 overexpression on autophagosome formation. LC3 is the mammalian homologue of the yeast Atg8 protein, which is covalently modified and re-distributes to autophagic vacuoles during induction of autophagy (23). This process can be followed by the appearance of LC3 puncta or ring-like structures and by a mobility shift from LC3I to LC3II by SDS–PAGE. Cells transiently transfected with LC3–GFP alone, or co-transfected with empty vector and analysed 48 h after transfection, display few cytoplasmic LC3–GFP punctae (Fig. 1A and B). An increase in LC3–GFP punctae was observed when co-transfecting with full-length wild-type as well as with G2019S-mutant LRRK2, but not with the inactive K1906M mutant (Fig. 1B). A similar increase in LC3–GFP punctae was observed with wild-type or G2019S-mutant, but not K1906M-mutant kinase domain only (Fig. 1A and B), indicating that the effects on autophagy are, at least in part, due to the kinase domain and activity of LRRK2. Wild-type and G2019S-mutant full-length LRRK2 increased autophagosome numbers to a similar degree and comparable with that of wild-type or G2019S-mutant kinase domain only (Fig. 1B), possibly due to saturating levels of overexpression of the various active constructs with respect to downstream targets.
Figure 1.Effects of LRRK2 overexpression on autophagosome formation. (A) Example of HEK293T cells transfected with LC3–GFP and either empty vector (ctrl), catalytically inactive K1906M-mutant LRRK2 kinase domain (K1906M), wild-type LRRK2-kinase domain (kinase) or catalytically hyperactive G2019S-mutant LRRK2 kinase domain (G2019S). LC3–GFP fluorescence (green) and nuclear DAPI staining (blue). Scale bar, 10 μm. (B) Quantification of the number of LC3–GFP punctae per cell in the absence (white bars) or the presence of 3-MA (5 mm, 24 h) (grey bars). Cells were co-transfected with LC3–GFP and the indicated constructs, and LC3–GFP punctae quantified 48 h after transfection. Rapamycin (100 nm, 12 h) was used as positive control. Bars represent mean ± s.e.m. (n = 4); *P < 0.001. (C) Top: cells were transfected with either K1906M or kinase constructs, and extracts analysed for endogenous LC3I and LC3II blotting with an anti-LC3 antibody (Cell Signalling). Bottom: quantification of LC3II/tubulin from different experiments (mean ± s.e.m. (n = 3); *P < 0.05. (D) Representative electron micrographs of cells expressing LRRK2 kinase domain. Arrows point to distinct autophagic structures [scale bar 200 nm (left) or 500 nm (right)]. (E) Cells were co-transfected with EGFP and empty vector (GFP) or kinase domain (kinase), and FACS sorted before processing for electron microscopy. Stereological analysis of the number of intersections per cell, number of intersections per autophagic structures (AVs), percentage of cellular volume occupied by AVs and AV diameter were quantified from three individual cells per condition. *P < 0.05.

Effects of LRRK2 overexpression on autophagosome formation. (A) Example of HEK293T cells transfected with LC3–GFP and either empty vector (ctrl), catalytically inactive K1906M-mutant LRRK2 kinase domain (K1906M), wild-type LRRK2-kinase domain (kinase) or catalytically hyperactive G2019S-mutant LRRK2 kinase domain (G2019S). LC3–GFP fluorescence (green) and nuclear DAPI staining (blue). Scale bar, 10 μm. (B) Quantification of the number of LC3–GFP punctae per cell in the absence (white bars) or the presence of 3-MA (5 mm, 24 h) (grey bars). Cells were co-transfected with LC3–GFP and the indicated constructs, and LC3–GFP punctae quantified 48 h after transfection. Rapamycin (100 nm, 12 h) was used as positive control. Bars represent mean ± s.e.m. (n = 4); *P < 0.001. (C) Top: cells were transfected with either K1906M or kinase constructs, and extracts analysed for endogenous LC3I and LC3II blotting with an anti-LC3 antibody (Cell Signalling). Bottom: quantification of LC3II/tubulin from different experiments (mean ± s.e.m. (n = 3); *P < 0.05. (D) Representative electron micrographs of cells expressing LRRK2 kinase domain. Arrows point to distinct autophagic structures [scale bar 200 nm (left) or 500 nm (right)]. (E) Cells were co-transfected with EGFP and empty vector (GFP) or kinase domain (kinase), and FACS sorted before processing for electron microscopy. Stereological analysis of the number of intersections per cell, number of intersections per autophagic structures (AVs), percentage of cellular volume occupied by AVs and AV diameter were quantified from three individual cells per condition. *P < 0.05.

As a positive control, inhibition of the mammalian target of rapamycin (mTOR) protein kinase by rapamycin was found to induce autophagy to a similar degree (Fig. 1B). The phosphatidylinositol 3-kinase inhibitor 3-methyladenine (3-MA) is thought to suppress autophagy by inhibiting the production of phosphatidylinositol-3-phosphate required for autophagosome formation (24), and was found to block the increase in autophagosome numbers mediated by rapamycin, but not by LRRK2 overexpression (Fig. 1B). Such differential 3-MA sensitivity has previously been described (13), and points towards a possible mechanistic difference between mTOR- and LRRK2-mediated autophagy induction. An increase in autophagosome numbers was further reflected by an increase in the level of endogenous LC3II (Fig. 1C), and transmission electron microscopy confirmed that the induction of LC3-positive vesicles by LRRK2 overexpression correlated with an increase in the number of autophagic structures (Fig. 1D and E).

We next used a pH-sensitive tagged LC3 construct consisting of a tandem fusion of the red, acid-insensitive mCherry (pKa < 4.5) and the acid-sensitive GFP (pKa 6.0) (td-tag-LC3) (25,26). Td-tag-LC3 will emit yellow (green merged with red) fluorescence in non-acidic structures, but appear as red only in autolysosomes due to the quenching of GFP in these acidic structures (25,26). Indeed, td-tag-LC3 was found to faithfully track autophagosome maturation in transfected cells, thus allowing for the identification of possible deficits in autophagic maturation (Supplementary Material, Fig. S2). Co-transfection of td-tag-LC3 with G2019S-mutant LRRK2 kinase domain increased both early and late autophagic structures (Fig. 2A and B). Such increase in both yellow and red punctae was also observed with wild-type kinase domain or with full-length LRRK2, but not with the catalytically inactive K1906M mutant (Fig. 2B). As a positive control for blocking autophagosome maturation, we used trichostatin A, an inhibitor of histone deacetylases (HDACs) known to be required for the fusion of autophagosomes with lysosomes (27). Indeed, trichostatin A treatment significantly increased the percentage of yellow punctae (and decreased the number of red punctae), in contrast to LRRK2 expression or autophagy induction by rapamycin (Fig. 2C and D). These data indicate that LRRK2 overexpression enhances autophagosome numbers and at least the initial steps of autophagosome maturation in a manner dependent on kinase activity.
Figure 2.Effects of LRRK2 overexpression on autophagosome maturation. (A) Example of HEK293T cells co-transfected with tandem-tagged-LC3 (td-tag-LC3) and either empty vector (pCMV) or G2019S-mutant LRRK2 kinase domain. White: co-localization mask to highlight yellow punctae. Scale bar, 10 μm. (B) Quantification of the number of early, yellow-only td-tag-LC3 punctae per cell (top), or the overall fluorescence from late, red-only td-tag-LC3 punctae per cell (bottom). Cells were co-transfected with td-tag-LC3 and either empty pCMV vector (ctrl), wild-type or mutant kinase or full-length LRRK2 kinase (fl-kinase) constructs as indicated, and quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (C) Example of cells co-transfected with td-tag-LC3 and LRRK2 kinase domain, or transfected with td-tag-LC3 and treated with trichostatin A (300 nm, 4 h). White: co-localization mask to highlight yellow punctae. Scale bar, 10 μm. (D) Quantification of the percentage of early, yellow td-tag-LC3 punctae per cell. Cells were co-transfected with td-tag-LC3 and either kinase or empty vector. The latter were either left untreated (ctrl), or treated with rapamycin (rap.; 100 nm, 4 h) or trichostatin A (tricho.; 300 nm, 4 h) prior to fixation. Note that the percentage of yellow punctae in trichostatin-treated cells is increased because of the decrease in the number of red punctae. Bars represent mean ± s.e.m. (n = 2); *P < 0.05.

Effects of LRRK2 overexpression on autophagosome maturation. (A) Example of HEK293T cells co-transfected with tandem-tagged-LC3 (td-tag-LC3) and either empty vector (pCMV) or G2019S-mutant LRRK2 kinase domain. White: co-localization mask to highlight yellow punctae. Scale bar, 10 μm. (B) Quantification of the number of early, yellow-only td-tag-LC3 punctae per cell (top), or the overall fluorescence from late, red-only td-tag-LC3 punctae per cell (bottom). Cells were co-transfected with td-tag-LC3 and either empty pCMV vector (ctrl), wild-type or mutant kinase or full-length LRRK2 kinase (fl-kinase) constructs as indicated, and quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (C) Example of cells co-transfected with td-tag-LC3 and LRRK2 kinase domain, or transfected with td-tag-LC3 and treated with trichostatin A (300 nm, 4 h). White: co-localization mask to highlight yellow punctae. Scale bar, 10 μm. (D) Quantification of the percentage of early, yellow td-tag-LC3 punctae per cell. Cells were co-transfected with td-tag-LC3 and either kinase or empty vector. The latter were either left untreated (ctrl), or treated with rapamycin (rap.; 100 nm, 4 h) or trichostatin A (tricho.; 300 nm, 4 h) prior to fixation. Note that the percentage of yellow punctae in trichostatin-treated cells is increased because of the decrease in the number of red punctae. Bars represent mean ± s.e.m. (n = 2); *P < 0.05.

LRRK2 overexpression activates the Ca2+/CaMKK/AMPK pathwayThe increase in autophagosome numbers probably results from cytosolic events regulated by LRRK2. As recent studies imply an important role for AMPK in basal autophagy regulation (28), we evaluated the effects of LRRK2 on AMPK activity. Expression of wild-type and G2019S-mutant LRRK2 kinase domain, but not its inactive mutant, increased AMPK activity (Fig. 3A and B). This was not accompanied by inhibition of mTORC1 activity when compared with rapamycin or torin treatment (Fig. 3C and D and Supplementary Material, Fig. S3) (29). Accordingly, inhibition of AMPK by a pharmacological inhibitor (compound C) (30) attenuated the increase in autophagosome numbers induced by LRRK2 kinase domain overexpression (Fig. 3E). Compound C also attenuated the increase in autophagosome numbers observed with ionomycin and ATP, two treatments previously shown to activate AMPK (31), but was without effect on autophagosome numbers induced by rapamycin (Fig. 3E).
Figure 3.Effects of LRRK2 overexpression on AMPK and TORC1 signalling. (A) HEK293T cells were transfected with either empty vector (ctrl) or indicated constructs, and cell extracts analysed for phospho-ACC, phospho-AMPK and AMPK. (B) Quantification of the type of experiments depicted in (A). Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (C) As in (A), but extracts analysed for the indicated TORC1 substrates. (D) Quantification of the type of experiments depicted in (C). Bars represent mean ± s.e.m. (n = 3). (E) Cells were co-transfected with LC3–GFP and either empty vector (ctrl) or LRRK2 kinase domain, and control-transfected cells were treated with rapamycin (100 nm, 12 h), ionomycin (2.5 μm, 12 h) or ATP (100 μm, 12 h) where indicated. Cells were incubated in the absence (white bars) or the presence (grey bars) of compound C (5 μm, 24 h), and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.Further evidence for an involvement of AMPK was gained by expressing wild-type, dominant-negative and constitutively active forms of this kinase (32). Wild-type and mutant AMPKα1 forms were overexpressed to similar degrees and without effect on cellular viability, with AMPKα2 forms expressed to a lesser degree and displaying cellular toxicity (Supplementary Material, Fig. S4). Expression of wild-type and constitutively active, but not dominant-negative, AMPKα1 induced an increase in autophagosome numbers (Fig. 4A). Importantly, the dominant-negative AMPKα1 construct was able to inhibit the increase in autophagosome numbers observed upon active LRRK2 kinase domain co-expression (Fig. 4B). Such inhibition was not observed when autophagic on-rate was increased by distinct means, indicating a specific involvement of the AMPK pathway in the LRRK2-mediated process (Supplementary Material, Fig. S4).
Figure 4.Effects of LRRK2 overexpression on AMPK and intracellular calcium. (A) HEK293T cells were co-transfected with LC3–GFP and either empty vector (ctrl), human wild-type AMPKα1 (1-wt), dominant-negative (1-DN) or constitutively active (1-CA) AMPKα1, and LC3–GFP punctae quantified 48 h after transfection. Rapamycin (100 nm, 12 h) (rap.) was used as control. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (B) Cells were transfected with LC3–GFP along with either empty vector (ctrl), or LRRK2 kinase domain (kin) in the presence or in the absence of wild-type (1-wt) or dominant-negative (1-DN) AMPKα1 constructs as indicated, and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.01. (C) Cells were co-transfected with LC3–GFP and either empty vector (ctrl) or LRRK2 kinase domain (kinase), and control-transfected cells were treated with rapamycin (100 nm, 12 h), ionomycin (2.5 μm, 12 h) or ATP (100 μm, 12 h) where indicated. Cells were incubated in the absence (white bars) or in the presence (grey bars) of STO-609 (20 μm, 24 h), a CaMKKβ inhibitor and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (D) Cells were co-transfected with LC3–GFP and either empty vector (ctrl) or the indicated wild-type or mutant kinase domain or full-length (fl) constructs, and either left untreated (white bars) or treated with BAPTA-AM (5 μm, 2 h) (grey bars) prior to analysis for LC3–GFP punctae 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (E) Cells were transfected with LC3–GFP along with either K1906M mutant (white bars) or wild-type (grey bars) LRRK2 kinase domain and either control empty vector (−) or the indicated chimeric Bcl constructs, and LC3–GFP punctae analysed 24 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.AMPK can be activated by CaMKK-β in response to an increase in the cytosolic-free calcium [Ca2+]c (31). In order to test whether LRRK2 expression regulates the Ca2+/CamKK-β/AMPK pathway, we first treated cells with a CaMKK-α/β inhibitor (STO-609) (33). This compound attenuated the increase in autophagosome formation induced by LRRK2 kinase domain (Fig. 4C). Such inhibition was similar to that observed when cells were treated with two stimuli which increase [Ca2+]c by distinct means: ATP, which acts on P2 purinoreceptors to generate inositol 1,4,5-triphosphate (IP3) that triggers the release of Ca2+ from the ER through IP3 receptor-regulated channels, or ionomycin, a Ca2+ ionophore that serves as a mobile Ca2+ carrier to equilibrate Ca2+ levels across biological membranes, including plasma and ER membranes (31,34,35) (Fig. 4C). In contrast, the CaMKK-α/β inhibitor had no effect on autophagosome numbers elicited by rapamycin treatment (Fig. 4C). Remarkably, addition of the intracellular Ca2+ chelator BAPTA-AM (bis-(O-aminophenoxy)-ethane-N,N,N,N′-tetraacetic acid/tetra(acetoxymethyl)-ester) to LRRK2 kinase domain or full-length LRRK2-transfected cells for the last 2 hours completely abolished the increase in autophagosome numbers (Fig. 4D). BAPTA-AM was further able to block the ionomycin-mediated, but not the rapamycin-mediated, increase in autophagosome numbers (Supplementary Material, Fig. S4). Thus, a Ca2+/CaMKK/AMPK pathway seems to be required for the increased formation of autophagosomes in response to active LRRK2 overexpression.

LRRK2 overexpression activates the Ca2+/CaMKK/AMPK pathway

The increase in autophagosome numbers probably results from cytosolic events regulated by LRRK2. As recent studies imply an important role for AMPK in basal autophagy regulation (28), we evaluated the effects of LRRK2 on AMPK activity. Expression of wild-type and G2019S-mutant LRRK2 kinase domain, but not its inactive mutant, increased AMPK activity (Fig. 3A and B). This was not accompanied by inhibition of mTORC1 activity when compared with rapamycin or torin treatment (Fig. 3C and D and Supplementary Material, Fig. S3) (29). Accordingly, inhibition of AMPK by a pharmacological inhibitor (compound C) (30) attenuated the increase in autophagosome numbers induced by LRRK2 kinase domain overexpression (Fig. 3E). Compound C also attenuated the increase in autophagosome numbers observed with ionomycin and ATP, two treatments previously shown to activate AMPK (31), but was without effect on autophagosome numbers induced by rapamycin (Fig. 3E).
Figure 3.Effects of LRRK2 overexpression on AMPK and TORC1 signalling. (A) HEK293T cells were transfected with either empty vector (ctrl) or indicated constructs, and cell extracts analysed for phospho-ACC, phospho-AMPK and AMPK. (B) Quantification of the type of experiments depicted in (A). Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (C) As in (A), but extracts analysed for the indicated TORC1 substrates. (D) Quantification of the type of experiments depicted in (C). Bars represent mean ± s.e.m. (n = 3). (E) Cells were co-transfected with LC3–GFP and either empty vector (ctrl) or LRRK2 kinase domain, and control-transfected cells were treated with rapamycin (100 nm, 12 h), ionomycin (2.5 μm, 12 h) or ATP (100 μm, 12 h) where indicated. Cells were incubated in the absence (white bars) or the presence (grey bars) of compound C (5 μm, 24 h), and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.

Effects of LRRK2 overexpression on AMPK and TORC1 signalling. (A) HEK293T cells were transfected with either empty vector (ctrl) or indicated constructs, and cell extracts analysed for phospho-ACC, phospho-AMPK and AMPK. (B) Quantification of the type of experiments depicted in (A). Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (C) As in (A), but extracts analysed for the indicated TORC1 substrates. (D) Quantification of the type of experiments depicted in (C). Bars represent mean ± s.e.m. (n = 3). (E) Cells were co-transfected with LC3–GFP and either empty vector (ctrl) or LRRK2 kinase domain, and control-transfected cells were treated with rapamycin (100 nm, 12 h), ionomycin (2.5 μm, 12 h) or ATP (100 μm, 12 h) where indicated. Cells were incubated in the absence (white bars) or the presence (grey bars) of compound C (5 μm, 24 h), and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.

Further evidence for an involvement of AMPK was gained by expressing wild-type, dominant-negative and constitutively active forms of this kinase (32). Wild-type and mutant AMPKα1 forms were overexpressed to similar degrees and without effect on cellular viability, with AMPKα2 forms expressed to a lesser degree and displaying cellular toxicity (Supplementary Material, Fig. S4). Expression of wild-type and constitutively active, but not dominant-negative, AMPKα1 induced an increase in autophagosome numbers (Fig. 4A). Importantly, the dominant-negative AMPKα1 construct was able to inhibit the increase in autophagosome numbers observed upon active LRRK2 kinase domain co-expression (Fig. 4B). Such inhibition was not observed when autophagic on-rate was increased by distinct means, indicating a specific involvement of the AMPK pathway in the LRRK2-mediated process (Supplementary Material, Fig. S4).
Figure 4.Effects of LRRK2 overexpression on AMPK and intracellular calcium. (A) HEK293T cells were co-transfected with LC3–GFP and either empty vector (ctrl), human wild-type AMPKα1 (1-wt), dominant-negative (1-DN) or constitutively active (1-CA) AMPKα1, and LC3–GFP punctae quantified 48 h after transfection. Rapamycin (100 nm, 12 h) (rap.) was used as control. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (B) Cells were transfected with LC3–GFP along with either empty vector (ctrl), or LRRK2 kinase domain (kin) in the presence or in the absence of wild-type (1-wt) or dominant-negative (1-DN) AMPKα1 constructs as indicated, and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.01. (C) Cells were co-transfected with LC3–GFP and either empty vector (ctrl) or LRRK2 kinase domain (kinase), and control-transfected cells were treated with rapamycin (100 nm, 12 h), ionomycin (2.5 μm, 12 h) or ATP (100 μm, 12 h) where indicated. Cells were incubated in the absence (white bars) or in the presence (grey bars) of STO-609 (20 μm, 24 h), a CaMKKβ inhibitor and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (D) Cells were co-transfected with LC3–GFP and either empty vector (ctrl) or the indicated wild-type or mutant kinase domain or full-length (fl) constructs, and either left untreated (white bars) or treated with BAPTA-AM (5 μm, 2 h) (grey bars) prior to analysis for LC3–GFP punctae 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (E) Cells were transfected with LC3–GFP along with either K1906M mutant (white bars) or wild-type (grey bars) LRRK2 kinase domain and either control empty vector (−) or the indicated chimeric Bcl constructs, and LC3–GFP punctae analysed 24 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.

Effects of LRRK2 overexpression on AMPK and intracellular calcium. (A) HEK293T cells were co-transfected with LC3–GFP and either empty vector (ctrl), human wild-type AMPKα1 (1-wt), dominant-negative (1-DN) or constitutively active (1-CA) AMPKα1, and LC3–GFP punctae quantified 48 h after transfection. Rapamycin (100 nm, 12 h) (rap.) was used as control. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (B) Cells were transfected with LC3–GFP along with either empty vector (ctrl), or LRRK2 kinase domain (kin) in the presence or in the absence of wild-type (1-wt) or dominant-negative (1-DN) AMPKα1 constructs as indicated, and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.01. (C) Cells were co-transfected with LC3–GFP and either empty vector (ctrl) or LRRK2 kinase domain (kinase), and control-transfected cells were treated with rapamycin (100 nm, 12 h), ionomycin (2.5 μm, 12 h) or ATP (100 μm, 12 h) where indicated. Cells were incubated in the absence (white bars) or in the presence (grey bars) of STO-609 (20 μm, 24 h), a CaMKKβ inhibitor and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (D) Cells were co-transfected with LC3–GFP and either empty vector (ctrl) or the indicated wild-type or mutant kinase domain or full-length (fl) constructs, and either left untreated (white bars) or treated with BAPTA-AM (5 μm, 2 h) (grey bars) prior to analysis for LC3–GFP punctae 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (E) Cells were transfected with LC3–GFP along with either K1906M mutant (white bars) or wild-type (grey bars) LRRK2 kinase domain and either control empty vector (−) or the indicated chimeric Bcl constructs, and LC3–GFP punctae analysed 24 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.

AMPK can be activated by CaMKK-β in response to an increase in the cytosolic-free calcium [Ca2+]c (31). In order to test whether LRRK2 expression regulates the Ca2+/CamKK-β/AMPK pathway, we first treated cells with a CaMKK-α/β inhibitor (STO-609) (33). This compound attenuated the increase in autophagosome formation induced by LRRK2 kinase domain (Fig. 4C). Such inhibition was similar to that observed when cells were treated with two stimuli which increase [Ca2+]c by distinct means: ATP, which acts on P2 purinoreceptors to generate inositol 1,4,5-triphosphate (IP3) that triggers the release of Ca2+ from the ER through IP3 receptor-regulated channels, or ionomycin, a Ca2+ ionophore that serves as a mobile Ca2+ carrier to equilibrate Ca2+ levels across biological membranes, including plasma and ER membranes (31,34,35) (Fig. 4C). In contrast, the CaMKK-α/β inhibitor had no effect on autophagosome numbers elicited by rapamycin treatment (Fig. 4C). Remarkably, addition of the intracellular Ca2+ chelator BAPTA-AM (bis-(O-aminophenoxy)-ethane-N,N,N,N′-tetraacetic acid/tetra(acetoxymethyl)-ester) to LRRK2 kinase domain or full-length LRRK2-transfected cells for the last 2 hours completely abolished the increase in autophagosome numbers (Fig. 4D). BAPTA-AM was further able to block the ionomycin-mediated, but not the rapamycin-mediated, increase in autophagosome numbers (Supplementary Material, Fig. S4). Thus, a Ca2+/CaMKK/AMPK pathway seems to be required for the increased formation of autophagosomes in response to active LRRK2 overexpression.

ER-localized Bcl-2 inhibits LRRK2-mediated autophagy inductionBcl-2 has been reported to regulate cellular Ca2+ handling and Ca2+/CaMKK/AMPK-induced autophagy (31,36). Thus, we tested whether expression of bcl-2 could inhibit the LRRK2-mediated increase in autophagosome numbers via its effects on Ca2+ homeostasis. For this purpose, we expressed wild-type bcl-2 (Bcl-WT), which is mostly mitochondrial (36,37), or mutants in which the C-terminal hydrophobic sequence has been either removed (Bcl-cyt), resulting in cytosolic expression, or exchanged to a corresponding membrane anchor from an ER-specific isoform of cytochrome b5 (Bcl-ER) or L. monocytogenes ActA (Bcl-mito), resulting in distinct subcellular localizations (31,38). Western blot analysis confirmed equal expression levels, and only slight effects on cytotoxicity were observed when overexpressing Bcl-wt or Bcl-mito, but not Bcl-cyto or Bcl-ER, respectively (Supplementary Material, Fig. S4). Importantly, only Bcl-ER was able to fully inhibit the LRRK2 kinase domain-mediated increase in autophagosome numbers (Fig. 4E). ER-localized Bcl-2 has previously been demonstrated to lower [Ca2+]ER and agonist-induced Ca2+ leak from the ER (39). Together with the inhibitory effect of BAPTA-AM and inhibition of CaMKK on the LRRK2-mediated increase in autophagosome numbers, these results indicate that active LRRK2 may increase [Ca2+]c by directly or indirectly inducing Ca2+ release from the ER. This release is likely mediated by IP3 receptors, as the increase in autophagosome numbers upon LRRK2 expression could not be blocked by dantrolene, a ryanodine receptor antagonist (Supplementary Material, Fig. S4).

ER-localized Bcl-2 inhibits LRRK2-mediated autophagy induction

Bcl-2 has been reported to regulate cellular Ca2+ handling and Ca2+/CaMKK/AMPK-induced autophagy (31,36). Thus, we tested whether expression of bcl-2 could inhibit the LRRK2-mediated increase in autophagosome numbers via its effects on Ca2+ homeostasis. For this purpose, we expressed wild-type bcl-2 (Bcl-WT), which is mostly mitochondrial (36,37), or mutants in which the C-terminal hydrophobic sequence has been either removed (Bcl-cyt), resulting in cytosolic expression, or exchanged to a corresponding membrane anchor from an ER-specific isoform of cytochrome b5 (Bcl-ER) or L. monocytogenes ActA (Bcl-mito), resulting in distinct subcellular localizations (31,38). Western blot analysis confirmed equal expression levels, and only slight effects on cytotoxicity were observed when overexpressing Bcl-wt or Bcl-mito, but not Bcl-cyto or Bcl-ER, respectively (Supplementary Material, Fig. S4). Importantly, only Bcl-ER was able to fully inhibit the LRRK2 kinase domain-mediated increase in autophagosome numbers (Fig. 4E). ER-localized Bcl-2 has previously been demonstrated to lower [Ca2+]ER and agonist-induced Ca2+ leak from the ER (39). Together with the inhibitory effect of BAPTA-AM and inhibition of CaMKK on the LRRK2-mediated increase in autophagosome numbers, these results indicate that active LRRK2 may increase [Ca2+]c by directly or indirectly inducing Ca2+ release from the ER. This release is likely mediated by IP3 receptors, as the increase in autophagosome numbers upon LRRK2 expression could not be blocked by dantrolene, a ryanodine receptor antagonist (Supplementary Material, Fig. S4).

LRRK2 alkalinizes a subpopulation of lysosomesOverexpression of LRRK2 has previously been shown to lead to accumulation of multi-vesicular bodies and autophagosomes containing incompletely degraded material and p62 (15), a classical macroautophagy substrate (40), indicating possible defects in autophagic degradation. Indeed, overexpression of wild-type or G2019S-mutant, but not inactive K1906M-mutant, kinase domain led to a significant increase in p62 levels and p62-positive structures (Fig. 5A–C). However, the LRRK2-mediated increase in p62 levels could be blocked by protein synthesis inhibitors (Fig. 5D), indicating that it was not reflecting changes in autophagic flux. Identical results were obtained when using ionomycin or thapsigargin (Supplementary Material, Fig. S5), suggesting that the LRRK2-mediated effects on p62 levels may be due to Ca2+-dependent changes in protein synthesis (41).
Figure 5.Effects of LRRK2 overexpression on p62 levels. (A) HEK293T cells were transfected with either control empty vector (ctrl) or wild-type or mutant kinase domain constructs as indicated, and extracts analysed for endogenous p62 levels. (B) Quantification of the type of experiments as depicted in (A). Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (C) Example of cells co-transfected with LC3–GFP (green, left) and LRRK2 kinase domain and co-stained with anti-p62 (red, middle). Scale bar, 10 μm. (D) Cells were transfected with indicated constructs, and treated with puromycin (1 μm, 6 h) where indicated before analysis and quantification of p62 levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.A possible defect in lysosomal homeostasis, however, became apparent when employing lipid staining with BODIPY 493/503 (42), which revealed an increase in lipid droplet numbers in cells overexpressing LRRK2 kinase domain (Fig. 6A and B). In addition, lysotracker DND-99 or Lysosensor staining to determine lysosomal pH indicated a decrease in the number of positive punctae in cells overexpressing active LRRK2 kinase domain (Fig. 6C and D). This was not due to a decrease in lysosomal content (43), as no change in the levels of LAMP1 or LAMP2 were observed (Supplementary Material, Fig. S6). Interestingly, quantification of the percent colocalization of LAMP2 and lysotracker showed an increase in the number of structures positive for LAMP2, but negative for lysotracker, suggesting that LRRK2 increases pH in a subpopulation of lysosomes, without affecting the processing of lysosomal hydrolases (Fig. 6E and Supplementary Material, Fig. S6).
Figure 6.Effects of LRRK2 on lipid accumulation and lysosomal pH, and effects of NAADP. (A) Example of HEK293T cells co-transfected with mCherry and either K1906M-mutant or wild-type LRRK2 kinase domain, and stained with BODIPY 493/503 (green). Scale bar, 10 μm. (B) Percentage change in the number of lipid droplets quantified from experiments as in (A). Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (C) The number of lysosensor-positive structures in cells expressing K1906M or kinase domain. Bars represent mean ± s.e.m. (n = 4); *P < 0.05. (D) Percentage change in the number of lysotracker-positive structures in cells co-transfected with EGFP and either K1906M or kinase domain constructs. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (E) Quantification of LAMP2-positive structures positive for lysotracker in K1906M or kinase-expressing cells. Ten random cells were analysed per condition per experiment. Bars represent mean ± s.e.m. (n = 2); *P < 0.05. (F) Cells were transfected with LC3–GFP and either left untreated, or treated with the indicated concentrations of NAADP-AM or with 1 μm NED-19 for 12 h, in the absence (white bars) or in the presence (grey bars) of BAPTA-AM (5 μm, 2 h), and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (G) Cells were co-transfected with LC3–GFP and either control empty vector (white bars) or bcl-ER (grey bars), and either left untreated (ctrl), treated with NAADP-AM (100 nm, 4 h), starvation (Earle’s salt solution, 4 h), rapamycin (200 nm, 4 h) or torin (50 nm, 4 h) as indicated, and LC3–GFP punctae analysed 24 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (H) Cells were transfected with LC3–GFP and either left untreated (ctrl), or treated with NAADP-AM (100 nm, 12 h) or rapamycin (100 nm, 12 h) as indicated in the presence or in the absence of 3-MA (5 mm, 24 h) (grey bars). Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (I) Cells were either left untreated, or treated with NAADP-AM (100 nm, 12 h) as indicated, and cell extracts analysed and quantified for endogenous LC3. Bars represent mean ± s.e.m. (n = 2); *P < 0.05. (J) Cells were left untreated or treated with NAADP-AM (100 nm, 12 h) or puromycin (1 μm, 6 h) as indicated, and extracts analysed and quantified for p62 levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (K) Non-treated or NAADP-AM-treated cells were stained with BODIPY 493/503 and analysed for the percentage change in the number of lipid droplets. Bars represent mean ± s.e.m. (n = 3); *P < 0.01. (L) Non-treated or NAADP-AM-treated cells were stained with lysotracker and the percentage change quantified. Bars represent mean ± s.e.m. (n = 3); *P < 0.005.

LRRK2 alkalinizes a subpopulation of lysosomes

Overexpression of LRRK2 has previously been shown to lead to accumulation of multi-vesicular bodies and autophagosomes containing incompletely degraded material and p62 (15), a classical macroautophagy substrate (40), indicating possible defects in autophagic degradation. Indeed, overexpression of wild-type or G2019S-mutant, but not inactive K1906M-mutant, kinase domain led to a significant increase in p62 levels and p62-positive structures (Fig. 5A–C). However, the LRRK2-mediated increase in p62 levels could be blocked by protein synthesis inhibitors (Fig. 5D), indicating that it was not reflecting changes in autophagic flux. Identical results were obtained when using ionomycin or thapsigargin (Supplementary Material, Fig. S5), suggesting that the LRRK2-mediated effects on p62 levels may be due to Ca2+-dependent changes in protein synthesis (41).
Figure 5.Effects of LRRK2 overexpression on p62 levels. (A) HEK293T cells were transfected with either control empty vector (ctrl) or wild-type or mutant kinase domain constructs as indicated, and extracts analysed for endogenous p62 levels. (B) Quantification of the type of experiments as depicted in (A). Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (C) Example of cells co-transfected with LC3–GFP (green, left) and LRRK2 kinase domain and co-stained with anti-p62 (red, middle). Scale bar, 10 μm. (D) Cells were transfected with indicated constructs, and treated with puromycin (1 μm, 6 h) where indicated before analysis and quantification of p62 levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.

Effects of LRRK2 overexpression on p62 levels. (A) HEK293T cells were transfected with either control empty vector (ctrl) or wild-type or mutant kinase domain constructs as indicated, and extracts analysed for endogenous p62 levels. (B) Quantification of the type of experiments as depicted in (A). Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (C) Example of cells co-transfected with LC3–GFP (green, left) and LRRK2 kinase domain and co-stained with anti-p62 (red, middle). Scale bar, 10 μm. (D) Cells were transfected with indicated constructs, and treated with puromycin (1 μm, 6 h) where indicated before analysis and quantification of p62 levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.

A possible defect in lysosomal homeostasis, however, became apparent when employing lipid staining with BODIPY 493/503 (42), which revealed an increase in lipid droplet numbers in cells overexpressing LRRK2 kinase domain (Fig. 6A and B). In addition, lysotracker DND-99 or Lysosensor staining to determine lysosomal pH indicated a decrease in the number of positive punctae in cells overexpressing active LRRK2 kinase domain (Fig. 6C and D). This was not due to a decrease in lysosomal content (43), as no change in the levels of LAMP1 or LAMP2 were observed (Supplementary Material, Fig. S6). Interestingly, quantification of the percent colocalization of LAMP2 and lysotracker showed an increase in the number of structures positive for LAMP2, but negative for lysotracker, suggesting that LRRK2 increases pH in a subpopulation of lysosomes, without affecting the processing of lysosomal hydrolases (Fig. 6E and Supplementary Material, Fig. S6).
Figure 6.Effects of LRRK2 on lipid accumulation and lysosomal pH, and effects of NAADP. (A) Example of HEK293T cells co-transfected with mCherry and either K1906M-mutant or wild-type LRRK2 kinase domain, and stained with BODIPY 493/503 (green). Scale bar, 10 μm. (B) Percentage change in the number of lipid droplets quantified from experiments as in (A). Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (C) The number of lysosensor-positive structures in cells expressing K1906M or kinase domain. Bars represent mean ± s.e.m. (n = 4); *P < 0.05. (D) Percentage change in the number of lysotracker-positive structures in cells co-transfected with EGFP and either K1906M or kinase domain constructs. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (E) Quantification of LAMP2-positive structures positive for lysotracker in K1906M or kinase-expressing cells. Ten random cells were analysed per condition per experiment. Bars represent mean ± s.e.m. (n = 2); *P < 0.05. (F) Cells were transfected with LC3–GFP and either left untreated, or treated with the indicated concentrations of NAADP-AM or with 1 μm NED-19 for 12 h, in the absence (white bars) or in the presence (grey bars) of BAPTA-AM (5 μm, 2 h), and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (G) Cells were co-transfected with LC3–GFP and either control empty vector (white bars) or bcl-ER (grey bars), and either left untreated (ctrl), treated with NAADP-AM (100 nm, 4 h), starvation (Earle’s salt solution, 4 h), rapamycin (200 nm, 4 h) or torin (50 nm, 4 h) as indicated, and LC3–GFP punctae analysed 24 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (H) Cells were transfected with LC3–GFP and either left untreated (ctrl), or treated with NAADP-AM (100 nm, 12 h) or rapamycin (100 nm, 12 h) as indicated in the presence or in the absence of 3-MA (5 mm, 24 h) (grey bars). Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (I) Cells were either left untreated, or treated with NAADP-AM (100 nm, 12 h) as indicated, and cell extracts analysed and quantified for endogenous LC3. Bars represent mean ± s.e.m. (n = 2); *P < 0.05. (J) Cells were left untreated or treated with NAADP-AM (100 nm, 12 h) or puromycin (1 μm, 6 h) as indicated, and extracts analysed and quantified for p62 levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (K) Non-treated or NAADP-AM-treated cells were stained with BODIPY 493/503 and analysed for the percentage change in the number of lipid droplets. Bars represent mean ± s.e.m. (n = 3); *P < 0.01. (L) Non-treated or NAADP-AM-treated cells were stained with lysotracker and the percentage change quantified. Bars represent mean ± s.e.m. (n = 3); *P < 0.005.

Effects of LRRK2 on lipid accumulation and lysosomal pH, and effects of NAADP. (A) Example of HEK293T cells co-transfected with mCherry and either K1906M-mutant or wild-type LRRK2 kinase domain, and stained with BODIPY 493/503 (green). Scale bar, 10 μm. (B) Percentage change in the number of lipid droplets quantified from experiments as in (A). Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (C) The number of lysosensor-positive structures in cells expressing K1906M or kinase domain. Bars represent mean ± s.e.m. (n = 4); *P < 0.05. (D) Percentage change in the number of lysotracker-positive structures in cells co-transfected with EGFP and either K1906M or kinase domain constructs. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (E) Quantification of LAMP2-positive structures positive for lysotracker in K1906M or kinase-expressing cells. Ten random cells were analysed per condition per experiment. Bars represent mean ± s.e.m. (n = 2); *P < 0.05. (F) Cells were transfected with LC3–GFP and either left untreated, or treated with the indicated concentrations of NAADP-AM or with 1 μm NED-19 for 12 h, in the absence (white bars) or in the presence (grey bars) of BAPTA-AM (5 μm, 2 h), and LC3–GFP punctae quantified 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (G) Cells were co-transfected with LC3–GFP and either control empty vector (white bars) or bcl-ER (grey bars), and either left untreated (ctrl), treated with NAADP-AM (100 nm, 4 h), starvation (Earle’s salt solution, 4 h), rapamycin (200 nm, 4 h) or torin (50 nm, 4 h) as indicated, and LC3–GFP punctae analysed 24 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (H) Cells were transfected with LC3–GFP and either left untreated (ctrl), or treated with NAADP-AM (100 nm, 12 h) or rapamycin (100 nm, 12 h) as indicated in the presence or in the absence of 3-MA (5 mm, 24 h) (grey bars). Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (I) Cells were either left untreated, or treated with NAADP-AM (100 nm, 12 h) as indicated, and cell extracts analysed and quantified for endogenous LC3. Bars represent mean ± s.e.m. (n = 2); *P < 0.05. (J) Cells were left untreated or treated with NAADP-AM (100 nm, 12 h) or puromycin (1 μm, 6 h) as indicated, and extracts analysed and quantified for p62 levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (K) Non-treated or NAADP-AM-treated cells were stained with BODIPY 493/503 and analysed for the percentage change in the number of lipid droplets. Bars represent mean ± s.e.m. (n = 3); *P < 0.01. (L) Non-treated or NAADP-AM-treated cells were stained with lysotracker and the percentage change quantified. Bars represent mean ± s.e.m. (n = 3); *P < 0.005.

LRRK2 acts through NAADP receptorsThere is considerable evidence that acidic organelles such as endosomes and lysosomes are significant stores of Ca2+ that can be released by the Ca2+ mobilizing messenger NAADP (44,45). NAADP-evoked cytosolic Ca2+ signals are accompanied by partial alkalinization of acidic stores (46) and amplification by ER Ca2+ stores (47), and lysosomal Ca2+ depletion causes lipid accumulation (48). That LRRK2 increases lipid droplet numbers (Fig. 6), causes an increase in lysosomal pH (Fig. 6) and regulates Ca2+-dependent events in the cytosol which likely also involve the ER (Figs. 4 and 5) prompted us to consider the involvement of the NAADP pathway in mediating the effects of LRRK2.Application of NAADP-AM was found to increase autophagosome numbers (Fig. 6F). This increase was abolished at higher NAADP concentrations, in agreement with the notion that NAADP-induced Ca2+ release desensitizes at high ligand concentrations (49,50). The effect of NAADP on autophagosome formation could be blocked by BAPTA-AM (Fig. 6F) and was accompanied by an increase in AMPK activity (Supplementary Material, Fig. S6). The autophagy induction observed with NAADP could be inhibited by Bcl-ER (Fig. 6G), similar to starvation-induced autophagy (36,37), but distinct from autophagy induced by rapamycin or torin treatment (Fig. 6G). In addition, the autophagy induced by NAADP was insensitive to 3-MA, in contrast to TORC1-dependent autophagy inducers (Fig. 6H). Furthermore, NAADP caused an increase in LC3II levels (Fig. 6I), a protein synthesis-dependent increase in p62 levels (Fig. 6J), increased lipid droplet numbers (Fig. 6K) and increased lysosomal pH (Fig. 6L), thus fully mimicking the effects of LRRK2 expression.TPCs have recently emerged as the likely targets for NAADP, with TPC1 localized to both endosomes and lysosomes, and TPC2 mainly to lysosomes (50,51). Point mutants in the putative helix of the pore-forming regions have been described, which seem to largely abolish Ca2+ conductance and uncover dominant-negative effects with respect to NAADP-mediated Ca2+ signals (51,52). Thus, we evaluated whether co-expressing wild-type or mutant TPCs may interfere with LRRK2-mediated autophagosome formation. Expression of mRFP-tagged TPC constructs indicated that wild-type and mutant versions were similarly overexpressed and localized to intracellular compartments, with TPC2 partially co-localizing with the lysosomal marker LAMP2, as previously described (51). None of the TPC constructs displayed cytotoxicity at 24 h upon overexpression, but TPC1 and its mutant were found to increase autophagosome formation (Supplementary Material, Fig. S7). Whilst consistent with the involvement of TPC1 in endolysosomal trafficking (53), this finding precluded further analysis of a possible reversal of the LRRK2-mediated increase in autophagosome numbers by TPC1 constructs. In contrast, TPC2 and its mutant did not display effects on autophagosome numbers when expressed on their own (Fig. 7A). Importantly, mutant but not wild-type TPC2 was able to block the effect of LRRK2 kinase domain on autophagosome numbers (Fig. 7A).
Figure 7.A link between LRRK2 and NAADP-sensitive receptors. (A) HEK 293T cells were transfected with LC3–GFP along with K1906M-mutant or wild-type kinase domain and either empty vector (pCMV), TPC2 or mutant TPC2 as indicated, and LC3–GFP punctae quantified 24 h after transfection. Bars represent mean ± s.e.m. (n = 5); *P < 0.005. (B) Cells were co-transfected with LC3–GFP and the indicated kinase domain or full-length (fl) LRRK2 constructs, and either left untreated (white bars) or treated with NED-19 (1 μm, 12 h) (grey bars), followed by quantification of LC3–GFP punctae 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (C) Transfected cells were treated with NED-19 (1 μm, 12 h) where indicated before analysis for p62 levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (D) As in (C), but extracts analysed and quantified for LC3II levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (E) Transfected cells were treated with NED-19 (1 μm, 12 h) where indicated before staining with lysotracker. Bars represent mean ± s.e.m. (n = 4); *P < 0.05. (F) Cells were transfected with TPC2–GFP or GFP vector, and extracts (1 mg) subjected to immunoprecipitation with an anti-GFP antibody. Note that TPC2–GFP runs as distinct bands corresponding to core glycosylated and mature fully glycosylated versions, respectively (67). Co-immunoprecipitated endogenous LRRK2 was detected using an anti-LRRK2 antibody (MJFF2, Epitomics). Note the additional presence of lower molecular-weight bands, which are likely distinct degradation products of endogenous LRRK2. Inputs (5%) were run along-side the immunoprecipitates. Representative of a total of seven independent experiments.We next evaluated whether NED-19, a newly described NAADP antagonist (54), could reverse the LRRK2-mediated effects on autophagic flux. NED-19 on its own was without effect on autophagosome numbers, but could block the increase in autophagosome numbers mediated by active kinase domain or full-length LRRK2 (Fig. 7B). This was accompanied largely by a reversal in the LRRK2-mediated changes on AMPK activity (Supplementary Material, Fig. S7), p62 levels, LC3II levels and lysotracker-positive structures (Fig. 7C–E). Finally, when probing for an interaction between endogenous LRRK2 and GFP–TPC2, LRRK2 was found to co-immunoprecipitate with GFP–TPC2 and vice versa (Fig. 7F and Supplementary Material, Fig. S7). Together, these data indicate that NAADP mimicks the effects of active LRRK2, whilst NED-19 application can revert those effects, and suggest that NAADP receptors may be direct targets for regulation by LRRK2.

LRRK2 acts through NAADP receptors

There is considerable evidence that acidic organelles such as endosomes and lysosomes are significant stores of Ca2+ that can be released by the Ca2+ mobilizing messenger NAADP (44,45). NAADP-evoked cytosolic Ca2+ signals are accompanied by partial alkalinization of acidic stores (46) and amplification by ER Ca2+ stores (47), and lysosomal Ca2+ depletion causes lipid accumulation (48). That LRRK2 increases lipid droplet numbers (Fig. 6), causes an increase in lysosomal pH (Fig. 6) and regulates Ca2+-dependent events in the cytosol which likely also involve the ER (Figs. 4 and 5) prompted us to consider the involvement of the NAADP pathway in mediating the effects of LRRK2.

Application of NAADP-AM was found to increase autophagosome numbers (Fig. 6F). This increase was abolished at higher NAADP concentrations, in agreement with the notion that NAADP-induced Ca2+ release desensitizes at high ligand concentrations (49,50). The effect of NAADP on autophagosome formation could be blocked by BAPTA-AM (Fig. 6F) and was accompanied by an increase in AMPK activity (Supplementary Material, Fig. S6). The autophagy induction observed with NAADP could be inhibited by Bcl-ER (Fig. 6G), similar to starvation-induced autophagy (36,37), but distinct from autophagy induced by rapamycin or torin treatment (Fig. 6G). In addition, the autophagy induced by NAADP was insensitive to 3-MA, in contrast to TORC1-dependent autophagy inducers (Fig. 6H). Furthermore, NAADP caused an increase in LC3II levels (Fig. 6I), a protein synthesis-dependent increase in p62 levels (Fig. 6J), increased lipid droplet numbers (Fig. 6K) and increased lysosomal pH (Fig. 6L), thus fully mimicking the effects of LRRK2 expression.

TPCs have recently emerged as the likely targets for NAADP, with TPC1 localized to both endosomes and lysosomes, and TPC2 mainly to lysosomes (50,51). Point mutants in the putative helix of the pore-forming regions have been described, which seem to largely abolish Ca2+ conductance and uncover dominant-negative effects with respect to NAADP-mediated Ca2+ signals (51,52). Thus, we evaluated whether co-expressing wild-type or mutant TPCs may interfere with LRRK2-mediated autophagosome formation. Expression of mRFP-tagged TPC constructs indicated that wild-type and mutant versions were similarly overexpressed and localized to intracellular compartments, with TPC2 partially co-localizing with the lysosomal marker LAMP2, as previously described (51). None of the TPC constructs displayed cytotoxicity at 24 h upon overexpression, but TPC1 and its mutant were found to increase autophagosome formation (Supplementary Material, Fig. S7). Whilst consistent with the involvement of TPC1 in endolysosomal trafficking (53), this finding precluded further analysis of a possible reversal of the LRRK2-mediated increase in autophagosome numbers by TPC1 constructs. In contrast, TPC2 and its mutant did not display effects on autophagosome numbers when expressed on their own (Fig. 7A). Importantly, mutant but not wild-type TPC2 was able to block the effect of LRRK2 kinase domain on autophagosome numbers (Fig. 7A).
Figure 7.A link between LRRK2 and NAADP-sensitive receptors. (A) HEK 293T cells were transfected with LC3–GFP along with K1906M-mutant or wild-type kinase domain and either empty vector (pCMV), TPC2 or mutant TPC2 as indicated, and LC3–GFP punctae quantified 24 h after transfection. Bars represent mean ± s.e.m. (n = 5); *P < 0.005. (B) Cells were co-transfected with LC3–GFP and the indicated kinase domain or full-length (fl) LRRK2 constructs, and either left untreated (white bars) or treated with NED-19 (1 μm, 12 h) (grey bars), followed by quantification of LC3–GFP punctae 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (C) Transfected cells were treated with NED-19 (1 μm, 12 h) where indicated before analysis for p62 levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (D) As in (C), but extracts analysed and quantified for LC3II levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (E) Transfected cells were treated with NED-19 (1 μm, 12 h) where indicated before staining with lysotracker. Bars represent mean ± s.e.m. (n = 4); *P < 0.05. (F) Cells were transfected with TPC2–GFP or GFP vector, and extracts (1 mg) subjected to immunoprecipitation with an anti-GFP antibody. Note that TPC2–GFP runs as distinct bands corresponding to core glycosylated and mature fully glycosylated versions, respectively (67). Co-immunoprecipitated endogenous LRRK2 was detected using an anti-LRRK2 antibody (MJFF2, Epitomics). Note the additional presence of lower molecular-weight bands, which are likely distinct degradation products of endogenous LRRK2. Inputs (5%) were run along-side the immunoprecipitates. Representative of a total of seven independent experiments.

A link between LRRK2 and NAADP-sensitive receptors. (A) HEK 293T cells were transfected with LC3–GFP along with K1906M-mutant or wild-type kinase domain and either empty vector (pCMV), TPC2 or mutant TPC2 as indicated, and LC3–GFP punctae quantified 24 h after transfection. Bars represent mean ± s.e.m. (n = 5); *P < 0.005. (B) Cells were co-transfected with LC3–GFP and the indicated kinase domain or full-length (fl) LRRK2 constructs, and either left untreated (white bars) or treated with NED-19 (1 μm, 12 h) (grey bars), followed by quantification of LC3–GFP punctae 48 h after transfection. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (C) Transfected cells were treated with NED-19 (1 μm, 12 h) where indicated before analysis for p62 levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (D) As in (C), but extracts analysed and quantified for LC3II levels. Bars represent mean ± s.e.m. (n = 3); *P < 0.05. (E) Transfected cells were treated with NED-19 (1 μm, 12 h) where indicated before staining with lysotracker. Bars represent mean ± s.e.m. (n = 4); *P < 0.05. (F) Cells were transfected with TPC2–GFP or GFP vector, and extracts (1 mg) subjected to immunoprecipitation with an anti-GFP antibody. Note that TPC2–GFP runs as distinct bands corresponding to core glycosylated and mature fully glycosylated versions, respectively (67). Co-immunoprecipitated endogenous LRRK2 was detected using an anti-LRRK2 antibody (MJFF2, Epitomics). Note the additional presence of lower molecular-weight bands, which are likely distinct degradation products of endogenous LRRK2. Inputs (5%) were run along-side the immunoprecipitates. Representative of a total of seven independent experiments.

We next evaluated whether NED-19, a newly described NAADP antagonist (54), could reverse the LRRK2-mediated effects on autophagic flux. NED-19 on its own was without effect on autophagosome numbers, but could block the increase in autophagosome numbers mediated by active kinase domain or full-length LRRK2 (Fig. 7B). This was accompanied largely by a reversal in the LRRK2-mediated changes on AMPK activity (Supplementary Material, Fig. S7), p62 levels, LC3II levels and lysotracker-positive structures (Fig. 7C–E). Finally, when probing for an interaction between endogenous LRRK2 and GFP–TPC2, LRRK2 was found to co-immunoprecipitate with GFP–TPC2 and vice versa (Fig. 7F and Supplementary Material, Fig. S7). Together, these data indicate that NAADP mimicks the effects of active LRRK2, whilst NED-19 application can revert those effects, and suggest that NAADP receptors may be direct targets for regulation by LRRK2.

LRRK2 sensitizes cells to cell death induced by proteasome inhibitionAs the observed LRRK2-mediated effects were not cytotoxic per se, we next wondered whether they would sensitize cells towards cell death under certain conditions (55). When cells were treated with the proteasomal inhibitor MG-132, cell death was markedly accelerated in cells overexpressing active, but not inactive, LRRK2 kinase domain (Fig. 8A). The increased cell death in the presence of MG-132 in LRRK2 kinase domain-expressing cells could be blocked by NED-19 (Fig. 8B), supporting the involvement of the NAADP pathway in this process. Interestingly, application of rapamycin or torin, two specific TORC1 inhibitors, reverted the LRRK2-mediated effects on cell death in the presence of MG-132, whilst L690,440, a TORC1-independent autophagy-enhancing compound (56), was unable to do so (Fig. 8A and C). Rapamycin and torin also largely reversed the effects of LRRK2 on p62 accumulation and on the decrease in lysotracker staining (Fig. 8D and E). Measurements in dopaminergic neuroendocrine PC12 cells yielded similar results. The NAADP-receptor antagonist NED-19 was able to revert the LRRK2-mediated increase in autophagosome numbers, the LRRK2-mediated decrease in lysosensor-positive structures and the LRRK2-mediated increase in cell death in the presence of MG-132 (Supplementary Material, Fig. S8). Together, these data indicate that additional interference with proteasomal degradation is detrimental to cells overexpressing catalytically active LRRK2, and that blocking NAADP-mediated signalling, or enhancing autophagic flux and/or suppressing protein synthesis or affecting lysosomal homeostasis through a TORC1-dependent pathway can overcome the enhanced susceptibility to cell death induced by LRRK2 in non-neuronal as well as neuroendocrine cells.
Figure 8.Effects of LRRK2 expression on cellular viability in the presence of proteasomal inhibitor. (A) HEK293T cells were co-transfected with GFP and K1906M or kinase domain constructs as indicated and either left untreated (−), treated with rapamycin (100 nm, 4 h), MG-132 (500 nm, 12 h) or both as indicated. Cells were analysed for cell death 48 h after transfection using Hoechst staining. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (B) As in (A), but cells treated with NED-19 (1 μm, 12 h) as indicated. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (C) As in (A), but cells treated with torin (500 nm, 4 h) or L690,440 (10 μm, 4 h) as indicated. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (D) Left: cells were transfected with either empty vector or kinase domain construct, and treated with rapamycin (200 nm, 12 h) where indicated before analysis for p62 levels. Right: quantification of experiments where bars represent mean ± s.e.m. (n = 3); *P < 0.05. (E) Transfected cells were treated as indicated before staining with lysotracker. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.

LRRK2 sensitizes cells to cell death induced by proteasome inhibition

As the observed LRRK2-mediated effects were not cytotoxic per se, we next wondered whether they would sensitize cells towards cell death under certain conditions (55). When cells were treated with the proteasomal inhibitor MG-132, cell death was markedly accelerated in cells overexpressing active, but not inactive, LRRK2 kinase domain (Fig. 8A). The increased cell death in the presence of MG-132 in LRRK2 kinase domain-expressing cells could be blocked by NED-19 (Fig. 8B), supporting the involvement of the NAADP pathway in this process. Interestingly, application of rapamycin or torin, two specific TORC1 inhibitors, reverted the LRRK2-mediated effects on cell death in the presence of MG-132, whilst L690,440, a TORC1-independent autophagy-enhancing compound (56), was unable to do so (Fig. 8A and C). Rapamycin and torin also largely reversed the effects of LRRK2 on p62 accumulation and on the decrease in lysotracker staining (Fig. 8D and E). Measurements in dopaminergic neuroendocrine PC12 cells yielded similar results. The NAADP-receptor antagonist NED-19 was able to revert the LRRK2-mediated increase in autophagosome numbers, the LRRK2-mediated decrease in lysosensor-positive structures and the LRRK2-mediated increase in cell death in the presence of MG-132 (Supplementary Material, Fig. S8). Together, these data indicate that additional interference with proteasomal degradation is detrimental to cells overexpressing catalytically active LRRK2, and that blocking NAADP-mediated signalling, or enhancing autophagic flux and/or suppressing protein synthesis or affecting lysosomal homeostasis through a TORC1-dependent pathway can overcome the enhanced susceptibility to cell death induced by LRRK2 in non-neuronal as well as neuroendocrine cells.
Figure 8.Effects of LRRK2 expression on cellular viability in the presence of proteasomal inhibitor. (A) HEK293T cells were co-transfected with GFP and K1906M or kinase domain constructs as indicated and either left untreated (−), treated with rapamycin (100 nm, 4 h), MG-132 (500 nm, 12 h) or both as indicated. Cells were analysed for cell death 48 h after transfection using Hoechst staining. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (B) As in (A), but cells treated with NED-19 (1 μm, 12 h) as indicated. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (C) As in (A), but cells treated with torin (500 nm, 4 h) or L690,440 (10 μm, 4 h) as indicated. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (D) Left: cells were transfected with either empty vector or kinase domain construct, and treated with rapamycin (200 nm, 12 h) where indicated before analysis for p62 levels. Right: quantification of experiments where bars represent mean ± s.e.m. (n = 3); *P < 0.05. (E) Transfected cells were treated as indicated before staining with lysotracker. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.

Effects of LRRK2 expression on cellular viability in the presence of proteasomal inhibitor. (A) HEK293T cells were co-transfected with GFP and K1906M or kinase domain constructs as indicated and either left untreated (−), treated with rapamycin (100 nm, 4 h), MG-132 (500 nm, 12 h) or both as indicated. Cells were analysed for cell death 48 h after transfection using Hoechst staining. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (B) As in (A), but cells treated with NED-19 (1 μm, 12 h) as indicated. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (C) As in (A), but cells treated with torin (500 nm, 4 h) or L690,440 (10 μm, 4 h) as indicated. Bars represent mean ± s.e.m. (n = 3); *P < 0.005. (D) Left: cells were transfected with either empty vector or kinase domain construct, and treated with rapamycin (200 nm, 12 h) where indicated before analysis for p62 levels. Right: quantification of experiments where bars represent mean ± s.e.m. (n = 3); *P < 0.05. (E) Transfected cells were treated as indicated before staining with lysotracker. Bars represent mean ± s.e.m. (n = 3); *P < 0.05.

DISCUSSIONMutations in LRRK2 are the most common genetic cause of PD, but the function of the protein remains unknown. Our study provides insight into the molecular mechanism of LRRK2 action. We have shown that LRRK2 kinase activity exerts a previously unknown regulatory role on macroautophagy that seems to involve Ca2+ release from lysosomes and the Ca2+/CaMKK/AMPK pathway. These events seem to be mediated by activation of NAADP receptors, which causes Ca2+ efflux from acidic stores and is accompanied by a partial alkalinization of lysosomal store pH (Fig. 9).
Figure 9.Model for the LRRK2-mediated events identified in this study. It is proposed that LRRK2 directly or indirectly activates NAADP-receptors (such as TPC2), resulting in an increase in cytosolic Ca2+. Ca2+ efflux from acidic stores may lead to a concomitant increase in lysosomal pH. The cytosolic increase in Ca2+ may result in Ca2+-induced Ca2+ release from ER stores, and activation of CaMKKβ, followed by activation of AMPK and an increase in autophagosome numbers. These LRRK2-mediated events seem to sensitize cells to cell death in the presence of additional compounds interfering with proteasomal degradation. Detection methods and/or tools used in this study to analyse and modulate the indicated events are depicted in red. dn, dominant-negative.In our experimental conditions, wild-type LRRK2 had a significant effect similar to pathogenic G2019S-mutant LRRK2. As mutations in LRRK2 cause PD in an autosomal-dominant manner, we overexpressed the protein to mimic a gain-of-function phenotype. It is feasible that the pathogenic mechanism of LRRK2 depends on exceeding a threshold level of activity, also supported by the finding that homo- and heterozygous carriers of LRRK2 mutations are clinically indistinguishable (3). In this context, overexpressing both wild-type or G2019S-mutant LRRK2 may reach the threshold level required for LRRK2 function and/or activity to become pathogenic. Alternatively, we may have missed a small but biologically important difference in effect, which nevertheless may have important consequences when present over many years. Finally, it remains possible that the effects described here are a wild-type function of LRRK2 unrelated to PD, even though other studies describing an impaired autophagic balance with G2019S-mutant, but not wild-type LRRK2 (12–16), are consistent with the idea of a threshold effect.Previous studies have indicated that enhanced autophagy may mediate neurite shortening induced by G2019S LRRK2 expression, as molecular inhibition of autophagy reversed, and pharmacological activation of autophagy potentiated the effects of LRRK2 on neurite shortening (12,13). Whilst our data are consistent with the notion that LRRK2 expression causes autophagy induction, we find that pharmacological activation of autophagy is beneficial to cell survival, contradictory to the aggravating effects of rapamycin on neurite length (13). However, the micromolar, but not nanomolar, concentrations of rapamycin employed in this study (13) leave open the possibility of secondary effects unrelated to TORC1 inhibition. In addition, even though neurite length and complexity may be relevant readouts for the cellular effects of LRRK2, enhanced stress sensitivity and cell death are equally informative for our understanding of LRRK2 function in cultured cells (55). The observed effects of LRRK2 are not sufficient to cause cell death, but increase the susceptibility of cells to further insults related to protein degradation-mediated stress, reminiscent of data obtained with LRRK2/α-synuclein double-transgenic mice (57). In addition, transgenic mice expressing G2019S-mutant LRRK2 display progressive age-dependent degeneration of dopaminergic neurons in vivo, accompanied by the accumulation of autophagic structures (14). These data imply an important regulatory role for LRRK2 in autophagic balance, which leads to eventual cell death in the presence of an additional trigger.LRRK2 overexpression induces the formation of autophagosomes through a Ca2+/CaMKK/AMPK pathway. Contradictory data have emerged as to the role of cytosolic Ca2+ and autophagy, with both stimulatory (31) and inhibitory (56) effects on autophagy. Our data indicate that Ca2+ displays complex effects at different stages of the pathway, and is required in distinct intracellular locations. In this context, it is interesting to note that LRRK2 or reagents which increase intracellular Ca2+ were both found to increase p62 protein synthesis, indicating that, apart from regulation of autophagy, the LRRK2-mediated change in cytosolic Ca2+ may, at least in part, underlie some of the protein synthesis-mediated events proposed for LRRK2 (58,59).The effects of LRRK2 on autophagy induction were found to be TORC1 independent, in agreement with recent findings that autophagy under normal nutritional conditions is TORC1 independent and regulated by AMPK and InsP3R-mediated Ca2+ signalling from the ER (28,56,60). Furthermore, the LRRK2-mediated sensitization to cell death in the presence of proteasomal inhibition could be reversed by TORC1 inhibitors. Such reversal may be due to an increase in degradative capacity as autophagic flux is enhanced, a decrease in protein synthesis, an effect on lysosomal homeostasis (61) or a combination thereof. Indeed, evidence for all three events were found with rapamyin. In either case, these data suggest that TORC1 inhibitors may prove beneficial in treating neurodegenerative events related to LRRK2.LRRK2-expressing cells display a decrease in the number of lysosomes with acidic pH. Given that acidic pH is only lost in a fraction of lysosomes, further studies are warranted to elucidate whether there exists a correlation between the affected lysosomal subpopulation and the localization of endogenous NAADP receptors. In either case, the LRRK2-mediated effects on lysosomal pH are mild and do not result in a lysosomal proteolysis phenotype (62,63), and under basal conditions and in the absence of additional protein aggregation stress, these LRRK2-mediated effects do not affect cell survival.NAADP is the most powerful endogenous Ca2+ mobilizing compound known to date. It seems to invoke an initial trigger release from acidic stores which is then amplified by Ca2+-induced Ca2+ release from the ER mediated by Ins3P receptors (47). TPCs are the likely target receptors for NAADP. Whilst still poorly characterized ion channels, they display a broad tissue distribution, indicating that they might play a general role in the mobilization of acidic Ca2+ stores. The observation that a specific NAADP antagonist blocks all LRRK2-mediated effects analysed here, together with the observed interaction between LRRK2 and TPC2, suggests a molecular mechanism for LRRK2 function involving TPC2 on acidic stores. In this context, LRRK2 may regulate TPC2 channel behaviour, even though additional effects at the level of NAADP generation cannot be excluded. To our knowledge, this is the first report which links the effects of LRRK2 through an effect on NAADP receptors. Thus, NAADP receptors may represent possible new targets for interventions for familial and sporadic PD linked to abnormal LRRK2 activity.

Mutations in LRRK2 are the most common genetic cause of PD, but the function of the protein remains unknown. Our study provides insight into the molecular mechanism of LRRK2 action. We have shown that LRRK2 kinase activity exerts a previously unknown regulatory role on macroautophagy that seems to involve Ca2+ release from lysosomes and the Ca2+/CaMKK/AMPK pathway. These events seem to be mediated by activation of NAADP receptors, which causes Ca2+ efflux from acidic stores and is accompanied by a partial alkalinization of lysosomal store pH (Fig. 9).
Figure 9.Model for the LRRK2-mediated events identified in this study. It is proposed that LRRK2 directly or indirectly activates NAADP-receptors (such as TPC2), resulting in an increase in cytosolic Ca2+. Ca2+ efflux from acidic stores may lead to a concomitant increase in lysosomal pH. The cytosolic increase in Ca2+ may result in Ca2+-induced Ca2+ release from ER stores, and activation of CaMKKβ, followed by activation of AMPK and an increase in autophagosome numbers. These LRRK2-mediated events seem to sensitize cells to cell death in the presence of additional compounds interfering with proteasomal degradation. Detection methods and/or tools used in this study to analyse and modulate the indicated events are depicted in red. dn, dominant-negative.

Model for the LRRK2-mediated events identified in this study. It is proposed that LRRK2 directly or indirectly activates NAADP-receptors (such as TPC2), resulting in an increase in cytosolic Ca2+. Ca2+ efflux from acidic stores may lead to a concomitant increase in lysosomal pH. The cytosolic increase in Ca2+ may result in Ca2+-induced Ca2+ release from ER stores, and activation of CaMKKβ, followed by activation of AMPK and an increase in autophagosome numbers. These LRRK2-mediated events seem to sensitize cells to cell death in the presence of additional compounds interfering with proteasomal degradation. Detection methods and/or tools used in this study to analyse and modulate the indicated events are depicted in red. dn, dominant-negative.

In our experimental conditions, wild-type LRRK2 had a significant effect similar to pathogenic G2019S-mutant LRRK2. As mutations in LRRK2 cause PD in an autosomal-dominant manner, we overexpressed the protein to mimic a gain-of-function phenotype. It is feasible that the pathogenic mechanism of LRRK2 depends on exceeding a threshold level of activity, also supported by the finding that homo- and heterozygous carriers of LRRK2 mutations are clinically indistinguishable (3). In this context, overexpressing both wild-type or G2019S-mutant LRRK2 may reach the threshold level required for LRRK2 function and/or activity to become pathogenic. Alternatively, we may have missed a small but biologically important difference in effect, which nevertheless may have important consequences when present over many years. Finally, it remains possible that the effects described here are a wild-type function of LRRK2 unrelated to PD, even though other studies describing an impaired autophagic balance with G2019S-mutant, but not wild-type LRRK2 (12–16), are consistent with the idea of a threshold effect.

Previous studies have indicated that enhanced autophagy may mediate neurite shortening induced by G2019S LRRK2 expression, as molecular inhibition of autophagy reversed, and pharmacological activation of autophagy potentiated the effects of LRRK2 on neurite shortening (12,13). Whilst our data are consistent with the notion that LRRK2 expression causes autophagy induction, we find that pharmacological activation of autophagy is beneficial to cell survival, contradictory to the aggravating effects of rapamycin on neurite length (13). However, the micromolar, but not nanomolar, concentrations of rapamycin employed in this study (13) leave open the possibility of secondary effects unrelated to TORC1 inhibition. In addition, even though neurite length and complexity may be relevant readouts for the cellular effects of LRRK2, enhanced stress sensitivity and cell death are equally informative for our understanding of LRRK2 function in cultured cells (55). The observed effects of LRRK2 are not sufficient to cause cell death, but increase the susceptibility of cells to further insults related to protein degradation-mediated stress, reminiscent of data obtained with LRRK2/α-synuclein double-transgenic mice (57). In addition, transgenic mice expressing G2019S-mutant LRRK2 display progressive age-dependent degeneration of dopaminergic neurons in vivo, accompanied by the accumulation of autophagic structures (14). These data imply an important regulatory role for LRRK2 in autophagic balance, which leads to eventual cell death in the presence of an additional trigger.

LRRK2 overexpression induces the formation of autophagosomes through a Ca2+/CaMKK/AMPK pathway. Contradictory data have emerged as to the role of cytosolic Ca2+ and autophagy, with both stimulatory (31) and inhibitory (56) effects on autophagy. Our data indicate that Ca2+ displays complex effects at different stages of the pathway, and is required in distinct intracellular locations. In this context, it is interesting to note that LRRK2 or reagents which increase intracellular Ca2+ were both found to increase p62 protein synthesis, indicating that, apart from regulation of autophagy, the LRRK2-mediated change in cytosolic Ca2+ may, at least in part, underlie some of the protein synthesis-mediated events proposed for LRRK2 (58,59).

The effects of LRRK2 on autophagy induction were found to be TORC1 independent, in agreement with recent findings that autophagy under normal nutritional conditions is TORC1 independent and regulated by AMPK and InsP3R-mediated Ca2+ signalling from the ER (28,56,60). Furthermore, the LRRK2-mediated sensitization to cell death in the presence of proteasomal inhibition could be reversed by TORC1 inhibitors. Such reversal may be due to an increase in degradative capacity as autophagic flux is enhanced, a decrease in protein synthesis, an effect on lysosomal homeostasis (61) or a combination thereof. Indeed, evidence for all three events were found with rapamyin. In either case, these data suggest that TORC1 inhibitors may prove beneficial in treating neurodegenerative events related to LRRK2.

LRRK2-expressing cells display a decrease in the number of lysosomes with acidic pH. Given that acidic pH is only lost in a fraction of lysosomes, further studies are warranted to elucidate whether there exists a correlation between the affected lysosomal subpopulation and the localization of endogenous NAADP receptors. In either case, the LRRK2-mediated effects on lysosomal pH are mild and do not result in a lysosomal proteolysis phenotype (62,63), and under basal conditions and in the absence of additional protein aggregation stress, these LRRK2-mediated effects do not affect cell survival.

NAADP is the most powerful endogenous Ca2+ mobilizing compound known to date. It seems to invoke an initial trigger release from acidic stores which is then amplified by Ca2+-induced Ca2+ release from the ER mediated by Ins3P receptors (47). TPCs are the likely target receptors for NAADP. Whilst still poorly characterized ion channels, they display a broad tissue distribution, indicating that they might play a general role in the mobilization of acidic Ca2+ stores. The observation that a specific NAADP antagonist blocks all LRRK2-mediated effects analysed here, together with the observed interaction between LRRK2 and TPC2, suggests a molecular mechanism for LRRK2 function involving TPC2 on acidic stores. In this context, LRRK2 may regulate TPC2 channel behaviour, even though additional effects at the level of NAADP generation cannot be excluded. To our knowledge, this is the first report which links the effects of LRRK2 through an effect on NAADP receptors. Thus, NAADP receptors may represent possible new targets for interventions for familial and sporadic PD linked to abnormal LRRK2 activity.

MATERIALS AND METHODSPlasmid constructionsTo generate the LRRK2 kinase domain-GFPemd construct (kinase-GFPemd), the human LRRK2 kinase domain (aa 1844–2143) was PCR amplified using full-length LRRK2 as template, with EcoRI/BamHI sites on either end. VAMP2-GFPemd (64) was digested with EcoRI/BamHI to release VAMP2, and the PCR-amplified kinase domain cloned in-frame with the GFPemd at the C-terminal end.To generate the LRRK2 kinase domain pCMV construct (pCMV-kinase), human LRRK2 kinase domain (amino acid 1844–2143) was PCR amplified using full-length LRRK2 as template, with SacI/BamHI sites on either end. pCMV-VAMP2 (64) was digested with SacI/BamHI to release VAMP2, and religated with the digested, PCR-amplified kinase domain.To generate the full-length human LRRK2 pCMV vector (pCMV-flLRRK2), pGEMT-LRRK2 was digested with SacII/MluI to release fl-LRRK2. Quick Change Site Directed Mutagenesis was performed with the empty pCMV vector to introduce a SacII site 5′ to the MluI site, followed by digestion of the vector with SacII/MluI and introduction of flLRRK2.K1906M and G2019S mutations were introduced into all respective vectors by Quick Change Site Directed Mutagenesis, and the identity of all constructs verified by direct sequencing of the entire coding regions.The double-myc-tagged, full-length wild-type and mutant human LRRK2 constructs were kindly provided by M. Cookson (NIH, Bethesda, USA). Full-length constructs were transformed in Stbl3 E. coli, and DNA prepared using the BACMAX DNA Purification Kit (Epicentre Biotechnologies, Madison, USA).N-terminally myc-tagged constructs for wild-type, dominant-negative and constitutively active AMPKα1, as well as N-terminally myc-tagged constructs for wild-type and dominant-negative AMPKα2 were kindly provided by D. Carling (MRC, Imperial College London, UK). Bcl-2 and its organelle-specific mutants were kindly provided by M. Jäättelä (Institute of Cancer Biology, Danish Cancer Society, Denmark) and D.W. Andrews (McMaster University, Hamilton, Ontario, Canada). Human TPC1-mRFP, TPC1-L273P-mRFP and TPC2-mRFP have been previously described. The L265P mutation in TPC2-mRFP was generated by Quick Change Site Directed Mutagenesis and verified by direct sequencing.The tandem-tagged LC3 construct mCherry-EGFP-LC3B (human) was kindly provided by T. Johansen (University of Tromsø, Norway), and EGFP-LC3 (rat) was provided by T. Yoshimori (National Institute for Basic Biology, Okazaki, Japan) and J. Oliver (Institute of Parasitology and Biomedicine ‘López-Neyra’, CSIC, Granada, Spain). The G120A mutation was introduced into both tagged LC3 constructs by Quick Change Site-Directed Mutagenesis (Stratagene) and verified by direct sequencing.ReagentsNAADP-AM and NED-19 were synthesized as previously described (54,65). Torin was a generous gift of D. Sabatini (Whitehead Institute for Biomedical Research, Cambridge, USA). 3-MA (3-methyladenine), NH4Cl, bafilomycin A1, E-64d, pepstatin A, nigericin, trichostatin A, dantrolene, puromycin and ionomycin were from Sigma. STO-609 (7-oxo-7H-benzimidazo[2,1-a]benz[de]isoquinoline-3-carboxylic acid) and L690,440 (1-[(4-hydroxyphenoxy)ethylidene]bis[[phosphinylidyne bis(oxymethylene)]-2,2-dimethylpropanoate) were from Tocris Bioscience (Missouri, USA). Compound C, cycloheximide and MG-132 were from Calbiochem. BAPTA-AM was from Invitrogen, rapamycin was from LC Laboratories (Woburn, USA) and chloroquine was a generous gift from L. Ruiz Pérez (Institute of Parasitology and Biomedicine ‘López-Neyra’, CSIC, Granada, Spain).Cell culture, transfection and cell lysisUnless otherwise indicated, experiments were performed in HEK293T cells, which were cultured as previously described (66). Where indicated, experiments were also performed in neuroendocrine PC12 cells, which were cultured and transfected as previously described (64). Cells at 70–80% confluency were transfected using Lipofectamine 2000 (Invitrogen) according to manufacturer’s specifications for 4 h, followed by addition of fresh medium. Single transfections were performed using 3.5 μg of plasmid of interest and 10 μl of Lipofectamine 2000. For double-transfections, 2.5 μg of each plasmid was used, and for triple-transfections, 1.7 μg of each plasmid was employed. The DNA ratio for co-transfections in the case of full-length LRRK2 constructs was 1:5, using a total of 10 μg of DNA.Cells were treated with the indicated concentrations of compounds and for the indicated times, and cell extracts prepared 24 or 48 h after transfection. Cells were rinsed once in ice-cold PBS, followed by resuspension in 1 ml of lysis buffer per 100 mm dish (160 μl of lysis buffer per well of a 6-well plate) (1% SDS in PBS containing 1 mm PMSF, 1 mm Na3VO4 and 5 mm NaF). Extracts were boiled for 5 min, sonicated and centrifuged at 13 500 rpm for 10 min at 4°C. Protein concentrations of supernatants were estimated using the BCA assay (Pierce). Whenever possible, extracts were immediately resolved by SDS–PAGE without repeated freeze-thaws, as the latter was found to decrease detection of some phosphorylated proteins.Immunofluorescence staining and confocal microscopyTransfected cells were re-plated at a 1:3 ratio onto coverslips and processed for immunocytochemistry 24 or 48 h after transfection. Lysotracker DND-99, LysoSensor Green DND-189 and Bodipy 493/503 dyes were used essentially according to manufacturer’s instructions (Invitrogen). Fixed cells were mounted using ProLong Gold AntiFade mounting medium (Invitrogen), and images acquired on a Leica TCS-SP5 confocal microscope using a ×100 HCX PL APO CS 1.4 oil UV objective. Images were collected using single excitation for each wavelength separately [488 nm Argon Laser line and a 500–545 nm emission band pass; 543 nm HeNe Laser line and a 556–673 nm emission band pass; 405 nm UV diode and a 422–466 nm emission band pass (12.5% intensity)]. Ten to fifteen image sections of selected areas were acquired with a step size of 0.3 μm, and z-stack images analysed and processed using Leica Applied Systems (LAS AF6000) image acquisition software. The same laser intensity was used for image acquisition of individual experiments. Co-localization analysis was performed using Leica Applied Systems (LAS AF6000) image acquisition software, adjusting thresholds to 28% for each channel.Primary antibodies included a mouse monoclonal anti-LAMP2 antibody (1:50; Santa Cruz Biotechnology), a mouse monoclonal anti-p62 antibody (1:50; BD Transduction Laboratories), a mouse monoclonal anti-myc antibody (1:10 000, Sigma) and a rabbit polyclonal anti-GFP antibody (1:1000; Abcam). Secondary antibodies included goat anti-rabbit or goat anti-mouse AlexaFluor-405-conjugated secondary antibody (1:100; Invitrogen), AlexaFluor-488-conjugated secondary antibody (1:1000; Invitrogen) or AlexaFluor-555-conjugated secondary antibody (1:1000; Invitrogen).For detection of apoptosis, fixed cells were mounted using mounting medium containing DAPI (Vector Laboratories) and visualized on a Zeiss microscope using a ×100 oil-immersion objective. For each experiment, an average of 100 cells from random fields was quantified, and condensed or fragmented nuclei scored as apoptotic cells. Occasionally, apoptotic cells were also quantified using the DeadEnd Fluorometric TUNEL System (Promega) according to manufacturer’s specifications (Promega).Quantitative image analysisAutophagic activity was evaluated by quantifying the average number of GFP–LC3 punctae per cell. For this purpose, z-stacks of random fields of cells were captured at ×100 magnification on a confocal microscope. The number of GFP–LC3 punctae per cell was either obtained by manual counts performed by an observer blinded to condition, or more regularly using an NIH ImageJ macro called GFP–LC3 (13). Alternatively, random fields of transfected cells were imaged on a Zeiss microscope using a ×100 oil-immersion objective by an individual blinded to the experimental conditions, and the number of GFP–LC3 punctae per cell counted manually. Whilst the decreased resolution results in identification of overlapping or clustered punctae as a single large puncta, and thus will underestimate the number of GFP–LC3 punctae per cell, the relative changes observed using this determination were similar to those using the high-resolution analysis described above. For all determinations of LC3–GFP punctae per cell, between 30 and 50 randomly chosen transfected cells were analysed per experiment and condition. For quantification of yellow and red mCherry-EGFP–LC3 (td-tag-LC3) punctae, pictures were captured at ×100 magnification on a confocal microscope, and the number of yellow and red punctae analysed as described above from 40 randomly chosen transfected cells per experiment and condition.Electron microscopyTransfected cells were collected, resuspended in HEPES-buffered media and filtered through a 50 μm filter before FACS sorting. Sorted cells were fixed with 4% paraformaldehyde, 0.1% glutaraldehyde in 0.1 m cacodylate buffer containing 3 µm CaCl2 and 7.5% sucrose for 15 min at room temperature. Fixed cells were washed and pelleted, then cell pellets were cut into small cubes and mixed to randomize the sample. Samples were stained with reduced osmium tetraoxide, dehydrated and embedded in resin. Sections (∼80 nm) were examined with an FEI TECNAI BioTWIN electron microscope. Random cells were imaged.To calculate relative volume densities, Photoshop software was used to combine separate images to obtain a complete view of each cell. Autophagic structures were recognized according to their morphological characteristics. A counting grid was superimposed on images at a random offset and intersections were scored according to the underlying structures, using the ImageJ plug-ins Grids and Cell Counter (NIH, Bethesda, MD, USA). The relative volume density of each structure was calculated by dividing the number of intersections counted for each category by the number counted for the whole cell.Western blottingProteins were resolved by SDS–PAGE, transferred onto polyvinylidene difluoride membranes (Hybond, GEHealthcare) and probed with primary antibodies overnight at 4°C. The following antibodies were employed: a rabbit polyclonal anti-phospho-ACC (Ser79), a rabbit monoclonal anti-phospho-AMPKα (Thr172) (40H9), a rabbit monoclonal anti-AMPKα, a rabbit polyclonal phospho-S6 (S235/236), a rabbit polyclonal phospho-p70 S6 kinase (Thr389), a rabbit polyclonal p70 S6 kinase, a rabbit monoclonal phospho-4E-BP1 (T37/46) (236B4) and a rabbit polyclonal 4E-BP1. All those antibodies were from Cell Signaling Technology, and used at a dilution of 1:500. Additional antibodies included a mouse monoclonal anti-p62 (1:500; BD Transduction Laboratories), a mouse monoclonal anti-LAMP2 (H4B4) (1:200; Santa Cruz Biotechnology), a mouse monoclonal anti-LAMP1 (H4A3) (1:200; Santa Cruz Biotechnology), a goat polyclonal anti-cathepsin D antibody (1:100; Santa Cruz Biotechnology), a mouse monoclonal anti-cathepsin L antibody (1:100; BD Biosciences), a mouse monoclonal anti-LC3 (1:500; MBL, Japan), a rabbit polyclonal anti-LC3 (1:500; Cell Signaling Technology), a rabbit polyclonal anti-GFP (ab6556) (1:500; Abcam), a monoclonal anti-bcl antibody (ab694) (1:10000; Abcam), a monoclonal anti-tubulin antibody (clone DM1A) (1:5000; Sigma), a rabbit polyclonal anti-pan-14-3-3 antibody (K-19) (1:500; Santa Cruz Biotechnology), a rabbit monoclonal anti-LRRK2 antibody (MJFF2, 1:500; Epitomics) or a mouse monoclonal C-terminal anti-LRRK2 antibody (N241A/34, NeuroMab, UCDavis, USA). Membranes were washed and incubated with secondary antibodies [anti-rabbit HRP-conjugated antibody (1:2000) or anti-mouse HRP-conjugated antibody (1:2000) (Dako Cytomation)] for 90 min at room temperature, followed by detection using ECL reagents (Roche). Unless otherwise indicated, 50 μg of cell extracts were resolved on MiniProtean Gels (BioRad Laboratories), whilst for phospho-p70 S6 kinase, phospho-4E-BP1, LC3 and p62, 150 μg of cell extracts were resolved on large PROTEAN II gels (BioRad Laboratories), and westerns developed with enhanced ECL reagents (Roche). Upon chemiluminescence detection, a series of timed exposures were undertaken to ensure that densitometric analyses were performed at exposures within the linear range. Films were scanned, and QuantityOne (Biorad) was employed for densitometric analysis.ImmunoprecipitationTransfected cells were washed in chilled PBS and pelleted by centrifugation. Pellets were resuspended in IP buffer (1 ml/10 cm dish) containing 1% NP-40, 50 mm Tris/HCl, pH 7.4, 150 mm NaCl, phosphatase inhibitor cocktails 2 and 3 (Sigma) and protease inhibitor cocktail (Roche) and lysates centrifuged at 13 500 rpm for 10 min at 4°C. Protein concentration of supernatants was estimated using a BCA assay, and 1 mg total protein was subjected to immunoprecipitation with either anti-GFP (5 μg, Abcam) or anti-LRRK2 (10 μg, N138/6, NeuroMab, UCDavis, USA) antibodies. For immunoprecipitations, lysates were incubated with antibodies for 1 h at 4°C, followed by addition of protein G or protein A Sepharose beads and incubation overnight. Beads were washed five times with IP buffer followed by elution with SDS sample buffer and heating at 50°C for 3 min prior to separation by SDS-PAGE.Statistical analysisData are expressed as means ± s.e.m., and the significance of differences was assessed using unpaired t-tests. Differences were accepted as significant at the 95% level (P < 0.05).

MATERIALS AND METHODS

Plasmid constructionsTo generate the LRRK2 kinase domain-GFPemd construct (kinase-GFPemd), the human LRRK2 kinase domain (aa 1844–2143) was PCR amplified using full-length LRRK2 as template, with EcoRI/BamHI sites on either end. VAMP2-GFPemd (64) was digested with EcoRI/BamHI to release VAMP2, and the PCR-amplified kinase domain cloned in-frame with the GFPemd at the C-terminal end.To generate the LRRK2 kinase domain pCMV construct (pCMV-kinase), human LRRK2 kinase domain (amino acid 1844–2143) was PCR amplified using full-length LRRK2 as template, with SacI/BamHI sites on either end. pCMV-VAMP2 (64) was digested with SacI/BamHI to release VAMP2, and religated with the digested, PCR-amplified kinase domain.To generate the full-length human LRRK2 pCMV vector (pCMV-flLRRK2), pGEMT-LRRK2 was digested with SacII/MluI to release fl-LRRK2. Quick Change Site Directed Mutagenesis was performed with the empty pCMV vector to introduce a SacII site 5′ to the MluI site, followed by digestion of the vector with SacII/MluI and introduction of flLRRK2.K1906M and G2019S mutations were introduced into all respective vectors by Quick Change Site Directed Mutagenesis, and the identity of all constructs verified by direct sequencing of the entire coding regions.The double-myc-tagged, full-length wild-type and mutant human LRRK2 constructs were kindly provided by M. Cookson (NIH, Bethesda, USA). Full-length constructs were transformed in Stbl3 E. coli, and DNA prepared using the BACMAX DNA Purification Kit (Epicentre Biotechnologies, Madison, USA).N-terminally myc-tagged constructs for wild-type, dominant-negative and constitutively active AMPKα1, as well as N-terminally myc-tagged constructs for wild-type and dominant-negative AMPKα2 were kindly provided by D. Carling (MRC, Imperial College London, UK). Bcl-2 and its organelle-specific mutants were kindly provided by M. Jäättelä (Institute of Cancer Biology, Danish Cancer Society, Denmark) and D.W. Andrews (McMaster University, Hamilton, Ontario, Canada). Human TPC1-mRFP, TPC1-L273P-mRFP and TPC2-mRFP have been previously described. The L265P mutation in TPC2-mRFP was generated by Quick Change Site Directed Mutagenesis and verified by direct sequencing.The tandem-tagged LC3 construct mCherry-EGFP-LC3B (human) was kindly provided by T. Johansen (University of Tromsø, Norway), and EGFP-LC3 (rat) was provided by T. Yoshimori (National Institute for Basic Biology, Okazaki, Japan) and J. Oliver (Institute of Parasitology and Biomedicine ‘López-Neyra’, CSIC, Granada, Spain). The G120A mutation was introduced into both tagged LC3 constructs by Quick Change Site-Directed Mutagenesis (Stratagene) and verified by direct sequencing.

Plasmid constructions

To generate the LRRK2 kinase domain-GFPemd construct (kinase-GFPemd), the human LRRK2 kinase domain (aa 1844–2143) was PCR amplified using full-length LRRK2 as template, with EcoRI/BamHI sites on either end. VAMP2-GFPemd (64) was digested with EcoRI/BamHI to release VAMP2, and the PCR-amplified kinase domain cloned in-frame with the GFPemd at the C-terminal end.

To generate the LRRK2 kinase domain pCMV construct (pCMV-kinase), human LRRK2 kinase domain (amino acid 1844–2143) was PCR amplified using full-length LRRK2 as template, with SacI/BamHI sites on either end. pCMV-VAMP2 (64) was digested with SacI/BamHI to release VAMP2, and religated with the digested, PCR-amplified kinase domain.

To generate the full-length human LRRK2 pCMV vector (pCMV-flLRRK2), pGEMT-LRRK2 was digested with SacII/MluI to release fl-LRRK2. Quick Change Site Directed Mutagenesis was performed with the empty pCMV vector to introduce a SacII site 5′ to the MluI site, followed by digestion of the vector with SacII/MluI and introduction of flLRRK2.

K1906M and G2019S mutations were introduced into all respective vectors by Quick Change Site Directed Mutagenesis, and the identity of all constructs verified by direct sequencing of the entire coding regions.

The double-myc-tagged, full-length wild-type and mutant human LRRK2 constructs were kindly provided by M. Cookson (NIH, Bethesda, USA). Full-length constructs were transformed in Stbl3 E. coli, and DNA prepared using the BACMAX DNA Purification Kit (Epicentre Biotechnologies, Madison, USA).

N-terminally myc-tagged constructs for wild-type, dominant-negative and constitutively active AMPKα1, as well as N-terminally myc-tagged constructs for wild-type and dominant-negative AMPKα2 were kindly provided by D. Carling (MRC, Imperial College London, UK). Bcl-2 and its organelle-specific mutants were kindly provided by M. Jäättelä (Institute of Cancer Biology, Danish Cancer Society, Denmark) and D.W. Andrews (McMaster University, Hamilton, Ontario, Canada). Human TPC1-mRFP, TPC1-L273P-mRFP and TPC2-mRFP have been previously described. The L265P mutation in TPC2-mRFP was generated by Quick Change Site Directed Mutagenesis and verified by direct sequencing.

The tandem-tagged LC3 construct mCherry-EGFP-LC3B (human) was kindly provided by T. Johansen (University of Tromsø, Norway), and EGFP-LC3 (rat) was provided by T. Yoshimori (National Institute for Basic Biology, Okazaki, Japan) and J. Oliver (Institute of Parasitology and Biomedicine ‘López-Neyra’, CSIC, Granada, Spain). The G120A mutation was introduced into both tagged LC3 constructs by Quick Change Site-Directed Mutagenesis (Stratagene) and verified by direct sequencing.

ReagentsNAADP-AM and NED-19 were synthesized as previously described (54,65). Torin was a generous gift of D. Sabatini (Whitehead Institute for Biomedical Research, Cambridge, USA). 3-MA (3-methyladenine), NH4Cl, bafilomycin A1, E-64d, pepstatin A, nigericin, trichostatin A, dantrolene, puromycin and ionomycin were from Sigma. STO-609 (7-oxo-7H-benzimidazo[2,1-a]benz[de]isoquinoline-3-carboxylic acid) and L690,440 (1-[(4-hydroxyphenoxy)ethylidene]bis[[phosphinylidyne bis(oxymethylene)]-2,2-dimethylpropanoate) were from Tocris Bioscience (Missouri, USA). Compound C, cycloheximide and MG-132 were from Calbiochem. BAPTA-AM was from Invitrogen, rapamycin was from LC Laboratories (Woburn, USA) and chloroquine was a generous gift from L. Ruiz Pérez (Institute of Parasitology and Biomedicine ‘López-Neyra’, CSIC, Granada, Spain).

NAADP-AM and NED-19 were synthesized as previously described (54,65). Torin was a generous gift of D. Sabatini (Whitehead Institute for Biomedical Research, Cambridge, USA). 3-MA (3-methyladenine), NH4Cl, bafilomycin A1, E-64d, pepstatin A, nigericin, trichostatin A, dantrolene, puromycin and ionomycin were from Sigma. STO-609 (7-oxo-7H-benzimidazo[2,1-a]benz[de]isoquinoline-3-carboxylic acid) and L690,440 (1-[(4-hydroxyphenoxy)ethylidene]bis[[phosphinylidyne bis(oxymethylene)]-2,2-dimethylpropanoate) were from Tocris Bioscience (Missouri, USA). Compound C, cycloheximide and MG-132 were from Calbiochem. BAPTA-AM was from Invitrogen, rapamycin was from LC Laboratories (Woburn, USA) and chloroquine was a generous gift from L. Ruiz Pérez (Institute of Parasitology and Biomedicine ‘López-Neyra’, CSIC, Granada, Spain).

Cell culture, transfection and cell lysisUnless otherwise indicated, experiments were performed in HEK293T cells, which were cultured as previously described (66). Where indicated, experiments were also performed in neuroendocrine PC12 cells, which were cultured and transfected as previously described (64). Cells at 70–80% confluency were transfected using Lipofectamine 2000 (Invitrogen) according to manufacturer’s specifications for 4 h, followed by addition of fresh medium. Single transfections were performed using 3.5 μg of plasmid of interest and 10 μl of Lipofectamine 2000. For double-transfections, 2.5 μg of each plasmid was used, and for triple-transfections, 1.7 μg of each plasmid was employed. The DNA ratio for co-transfections in the case of full-length LRRK2 constructs was 1:5, using a total of 10 μg of DNA.Cells were treated with the indicated concentrations of compounds and for the indicated times, and cell extracts prepared 24 or 48 h after transfection. Cells were rinsed once in ice-cold PBS, followed by resuspension in 1 ml of lysis buffer per 100 mm dish (160 μl of lysis buffer per well of a 6-well plate) (1% SDS in PBS containing 1 mm PMSF, 1 mm Na3VO4 and 5 mm NaF). Extracts were boiled for 5 min, sonicated and centrifuged at 13 500 rpm for 10 min at 4°C. Protein concentrations of supernatants were estimated using the BCA assay (Pierce). Whenever possible, extracts were immediately resolved by SDS–PAGE without repeated freeze-thaws, as the latter was found to decrease detection of some phosphorylated proteins.

Cell culture, transfection and cell lysis

Unless otherwise indicated, experiments were performed in HEK293T cells, which were cultured as previously described (66). Where indicated, experiments were also performed in neuroendocrine PC12 cells, which were cultured and transfected as previously described (64). Cells at 70–80% confluency were transfected using Lipofectamine 2000 (Invitrogen) according to manufacturer’s specifications for 4 h, followed by addition of fresh medium. Single transfections were performed using 3.5 μg of plasmid of interest and 10 μl of Lipofectamine 2000. For double-transfections, 2.5 μg of each plasmid was used, and for triple-transfections, 1.7 μg of each plasmid was employed. The DNA ratio for co-transfections in the case of full-length LRRK2 constructs was 1:5, using a total of 10 μg of DNA.

Cells were treated with the indicated concentrations of compounds and for the indicated times, and cell extracts prepared 24 or 48 h after transfection. Cells were rinsed once in ice-cold PBS, followed by resuspension in 1 ml of lysis buffer per 100 mm dish (160 μl of lysis buffer per well of a 6-well plate) (1% SDS in PBS containing 1 mm PMSF, 1 mm Na3VO4 and 5 mm NaF). Extracts were boiled for 5 min, sonicated and centrifuged at 13 500 rpm for 10 min at 4°C. Protein concentrations of supernatants were estimated using the BCA assay (Pierce). Whenever possible, extracts were immediately resolved by SDS–PAGE without repeated freeze-thaws, as the latter was found to decrease detection of some phosphorylated proteins.

Immunofluorescence staining and confocal microscopyTransfected cells were re-plated at a 1:3 ratio onto coverslips and processed for immunocytochemistry 24 or 48 h after transfection. Lysotracker DND-99, LysoSensor Green DND-189 and Bodipy 493/503 dyes were used essentially according to manufacturer’s instructions (Invitrogen). Fixed cells were mounted using ProLong Gold AntiFade mounting medium (Invitrogen), and images acquired on a Leica TCS-SP5 confocal microscope using a ×100 HCX PL APO CS 1.4 oil UV objective. Images were collected using single excitation for each wavelength separately [488 nm Argon Laser line and a 500–545 nm emission band pass; 543 nm HeNe Laser line and a 556–673 nm emission band pass; 405 nm UV diode and a 422–466 nm emission band pass (12.5% intensity)]. Ten to fifteen image sections of selected areas were acquired with a step size of 0.3 μm, and z-stack images analysed and processed using Leica Applied Systems (LAS AF6000) image acquisition software. The same laser intensity was used for image acquisition of individual experiments. Co-localization analysis was performed using Leica Applied Systems (LAS AF6000) image acquisition software, adjusting thresholds to 28% for each channel.Primary antibodies included a mouse monoclonal anti-LAMP2 antibody (1:50; Santa Cruz Biotechnology), a mouse monoclonal anti-p62 antibody (1:50; BD Transduction Laboratories), a mouse monoclonal anti-myc antibody (1:10 000, Sigma) and a rabbit polyclonal anti-GFP antibody (1:1000; Abcam). Secondary antibodies included goat anti-rabbit or goat anti-mouse AlexaFluor-405-conjugated secondary antibody (1:100; Invitrogen), AlexaFluor-488-conjugated secondary antibody (1:1000; Invitrogen) or AlexaFluor-555-conjugated secondary antibody (1:1000; Invitrogen).For detection of apoptosis, fixed cells were mounted using mounting medium containing DAPI (Vector Laboratories) and visualized on a Zeiss microscope using a ×100 oil-immersion objective. For each experiment, an average of 100 cells from random fields was quantified, and condensed or fragmented nuclei scored as apoptotic cells. Occasionally, apoptotic cells were also quantified using the DeadEnd Fluorometric TUNEL System (Promega) according to manufacturer’s specifications (Promega).

Immunofluorescence staining and confocal microscopy

Transfected cells were re-plated at a 1:3 ratio onto coverslips and processed for immunocytochemistry 24 or 48 h after transfection. Lysotracker DND-99, LysoSensor Green DND-189 and Bodipy 493/503 dyes were used essentially according to manufacturer’s instructions (Invitrogen). Fixed cells were mounted using ProLong Gold AntiFade mounting medium (Invitrogen), and images acquired on a Leica TCS-SP5 confocal microscope using a ×100 HCX PL APO CS 1.4 oil UV objective. Images were collected using single excitation for each wavelength separately [488 nm Argon Laser line and a 500–545 nm emission band pass; 543 nm HeNe Laser line and a 556–673 nm emission band pass; 405 nm UV diode and a 422–466 nm emission band pass (12.5% intensity)]. Ten to fifteen image sections of selected areas were acquired with a step size of 0.3 μm, and z-stack images analysed and processed using Leica Applied Systems (LAS AF6000) image acquisition software. The same laser intensity was used for image acquisition of individual experiments. Co-localization analysis was performed using Leica Applied Systems (LAS AF6000) image acquisition software, adjusting thresholds to 28% for each channel.

Primary antibodies included a mouse monoclonal anti-LAMP2 antibody (1:50; Santa Cruz Biotechnology), a mouse monoclonal anti-p62 antibody (1:50; BD Transduction Laboratories), a mouse monoclonal anti-myc antibody (1:10 000, Sigma) and a rabbit polyclonal anti-GFP antibody (1:1000; Abcam). Secondary antibodies included goat anti-rabbit or goat anti-mouse AlexaFluor-405-conjugated secondary antibody (1:100; Invitrogen), AlexaFluor-488-conjugated secondary antibody (1:1000; Invitrogen) or AlexaFluor-555-conjugated secondary antibody (1:1000; Invitrogen).

For detection of apoptosis, fixed cells were mounted using mounting medium containing DAPI (Vector Laboratories) and visualized on a Zeiss microscope using a ×100 oil-immersion objective. For each experiment, an average of 100 cells from random fields was quantified, and condensed or fragmented nuclei scored as apoptotic cells. Occasionally, apoptotic cells were also quantified using the DeadEnd Fluorometric TUNEL System (Promega) according to manufacturer’s specifications (Promega).

Quantitative image analysisAutophagic activity was evaluated by quantifying the average number of GFP–LC3 punctae per cell. For this purpose, z-stacks of random fields of cells were captured at ×100 magnification on a confocal microscope. The number of GFP–LC3 punctae per cell was either obtained by manual counts performed by an observer blinded to condition, or more regularly using an NIH ImageJ macro called GFP–LC3 (13). Alternatively, random fields of transfected cells were imaged on a Zeiss microscope using a ×100 oil-immersion objective by an individual blinded to the experimental conditions, and the number of GFP–LC3 punctae per cell counted manually. Whilst the decreased resolution results in identification of overlapping or clustered punctae as a single large puncta, and thus will underestimate the number of GFP–LC3 punctae per cell, the relative changes observed using this determination were similar to those using the high-resolution analysis described above. For all determinations of LC3–GFP punctae per cell, between 30 and 50 randomly chosen transfected cells were analysed per experiment and condition. For quantification of yellow and red mCherry-EGFP–LC3 (td-tag-LC3) punctae, pictures were captured at ×100 magnification on a confocal microscope, and the number of yellow and red punctae analysed as described above from 40 randomly chosen transfected cells per experiment and condition.

Quantitative image analysis

Autophagic activity was evaluated by quantifying the average number of GFP–LC3 punctae per cell. For this purpose, z-stacks of random fields of cells were captured at ×100 magnification on a confocal microscope. The number of GFP–LC3 punctae per cell was either obtained by manual counts performed by an observer blinded to condition, or more regularly using an NIH ImageJ macro called GFP–LC3 (13). Alternatively, random fields of transfected cells were imaged on a Zeiss microscope using a ×100 oil-immersion objective by an individual blinded to the experimental conditions, and the number of GFP–LC3 punctae per cell counted manually. Whilst the decreased resolution results in identification of overlapping or clustered punctae as a single large puncta, and thus will underestimate the number of GFP–LC3 punctae per cell, the relative changes observed using this determination were similar to those using the high-resolution analysis described above. For all determinations of LC3–GFP punctae per cell, between 30 and 50 randomly chosen transfected cells were analysed per experiment and condition. For quantification of yellow and red mCherry-EGFP–LC3 (td-tag-LC3) punctae, pictures were captured at ×100 magnification on a confocal microscope, and the number of yellow and red punctae analysed as described above from 40 randomly chosen transfected cells per experiment and condition.

Electron microscopyTransfected cells were collected, resuspended in HEPES-buffered media and filtered through a 50 μm filter before FACS sorting. Sorted cells were fixed with 4% paraformaldehyde, 0.1% glutaraldehyde in 0.1 m cacodylate buffer containing 3 µm CaCl2 and 7.5% sucrose for 15 min at room temperature. Fixed cells were washed and pelleted, then cell pellets were cut into small cubes and mixed to randomize the sample. Samples were stained with reduced osmium tetraoxide, dehydrated and embedded in resin. Sections (∼80 nm) were examined with an FEI TECNAI BioTWIN electron microscope. Random cells were imaged.To calculate relative volume densities, Photoshop software was used to combine separate images to obtain a complete view of each cell. Autophagic structures were recognized according to their morphological characteristics. A counting grid was superimposed on images at a random offset and intersections were scored according to the underlying structures, using the ImageJ plug-ins Grids and Cell Counter (NIH, Bethesda, MD, USA). The relative volume density of each structure was calculated by dividing the number of intersections counted for each category by the number counted for the whole cell.

Electron microscopy

Transfected cells were collected, resuspended in HEPES-buffered media and filtered through a 50 μm filter before FACS sorting. Sorted cells were fixed with 4% paraformaldehyde, 0.1% glutaraldehyde in 0.1 m cacodylate buffer containing 3 µm CaCl2 and 7.5% sucrose for 15 min at room temperature. Fixed cells were washed and pelleted, then cell pellets were cut into small cubes and mixed to randomize the sample. Samples were stained with reduced osmium tetraoxide, dehydrated and embedded in resin. Sections (∼80 nm) were examined with an FEI TECNAI BioTWIN electron microscope. Random cells were imaged.

To calculate relative volume densities, Photoshop software was used to combine separate images to obtain a complete view of each cell. Autophagic structures were recognized according to their morphological characteristics. A counting grid was superimposed on images at a random offset and intersections were scored according to the underlying structures, using the ImageJ plug-ins Grids and Cell Counter (NIH, Bethesda, MD, USA). The relative volume density of each structure was calculated by dividing the number of intersections counted for each category by the number counted for the whole cell.

Western blottingProteins were resolved by SDS–PAGE, transferred onto polyvinylidene difluoride membranes (Hybond, GEHealthcare) and probed with primary antibodies overnight at 4°C. The following antibodies were employed: a rabbit polyclonal anti-phospho-ACC (Ser79), a rabbit monoclonal anti-phospho-AMPKα (Thr172) (40H9), a rabbit monoclonal anti-AMPKα, a rabbit polyclonal phospho-S6 (S235/236), a rabbit polyclonal phospho-p70 S6 kinase (Thr389), a rabbit polyclonal p70 S6 kinase, a rabbit monoclonal phospho-4E-BP1 (T37/46) (236B4) and a rabbit polyclonal 4E-BP1. All those antibodies were from Cell Signaling Technology, and used at a dilution of 1:500. Additional antibodies included a mouse monoclonal anti-p62 (1:500; BD Transduction Laboratories), a mouse monoclonal anti-LAMP2 (H4B4) (1:200; Santa Cruz Biotechnology), a mouse monoclonal anti-LAMP1 (H4A3) (1:200; Santa Cruz Biotechnology), a goat polyclonal anti-cathepsin D antibody (1:100; Santa Cruz Biotechnology), a mouse monoclonal anti-cathepsin L antibody (1:100; BD Biosciences), a mouse monoclonal anti-LC3 (1:500; MBL, Japan), a rabbit polyclonal anti-LC3 (1:500; Cell Signaling Technology), a rabbit polyclonal anti-GFP (ab6556) (1:500; Abcam), a monoclonal anti-bcl antibody (ab694) (1:10000; Abcam), a monoclonal anti-tubulin antibody (clone DM1A) (1:5000; Sigma), a rabbit polyclonal anti-pan-14-3-3 antibody (K-19) (1:500; Santa Cruz Biotechnology), a rabbit monoclonal anti-LRRK2 antibody (MJFF2, 1:500; Epitomics) or a mouse monoclonal C-terminal anti-LRRK2 antibody (N241A/34, NeuroMab, UCDavis, USA). Membranes were washed and incubated with secondary antibodies [anti-rabbit HRP-conjugated antibody (1:2000) or anti-mouse HRP-conjugated antibody (1:2000) (Dako Cytomation)] for 90 min at room temperature, followed by detection using ECL reagents (Roche). Unless otherwise indicated, 50 μg of cell extracts were resolved on MiniProtean Gels (BioRad Laboratories), whilst for phospho-p70 S6 kinase, phospho-4E-BP1, LC3 and p62, 150 μg of cell extracts were resolved on large PROTEAN II gels (BioRad Laboratories), and westerns developed with enhanced ECL reagents (Roche). Upon chemiluminescence detection, a series of timed exposures were undertaken to ensure that densitometric analyses were performed at exposures within the linear range. Films were scanned, and QuantityOne (Biorad) was employed for densitometric analysis.

Western blotting

Proteins were resolved by SDS–PAGE, transferred onto polyvinylidene difluoride membranes (Hybond, GEHealthcare) and probed with primary antibodies overnight at 4°C. The following antibodies were employed: a rabbit polyclonal anti-phospho-ACC (Ser79), a rabbit monoclonal anti-phospho-AMPKα (Thr172) (40H9), a rabbit monoclonal anti-AMPKα, a rabbit polyclonal phospho-S6 (S235/236), a rabbit polyclonal phospho-p70 S6 kinase (Thr389), a rabbit polyclonal p70 S6 kinase, a rabbit monoclonal phospho-4E-BP1 (T37/46) (236B4) and a rabbit polyclonal 4E-BP1. All those antibodies were from Cell Signaling Technology, and used at a dilution of 1:500. Additional antibodies included a mouse monoclonal anti-p62 (1:500; BD Transduction Laboratories), a mouse monoclonal anti-LAMP2 (H4B4) (1:200; Santa Cruz Biotechnology), a mouse monoclonal anti-LAMP1 (H4A3) (1:200; Santa Cruz Biotechnology), a goat polyclonal anti-cathepsin D antibody (1:100; Santa Cruz Biotechnology), a mouse monoclonal anti-cathepsin L antibody (1:100; BD Biosciences), a mouse monoclonal anti-LC3 (1:500; MBL, Japan), a rabbit polyclonal anti-LC3 (1:500; Cell Signaling Technology), a rabbit polyclonal anti-GFP (ab6556) (1:500; Abcam), a monoclonal anti-bcl antibody (ab694) (1:10000; Abcam), a monoclonal anti-tubulin antibody (clone DM1A) (1:5000; Sigma), a rabbit polyclonal anti-pan-14-3-3 antibody (K-19) (1:500; Santa Cruz Biotechnology), a rabbit monoclonal anti-LRRK2 antibody (MJFF2, 1:500; Epitomics) or a mouse monoclonal C-terminal anti-LRRK2 antibody (N241A/34, NeuroMab, UCDavis, USA). Membranes were washed and incubated with secondary antibodies [anti-rabbit HRP-conjugated antibody (1:2000) or anti-mouse HRP-conjugated antibody (1:2000) (Dako Cytomation)] for 90 min at room temperature, followed by detection using ECL reagents (Roche). Unless otherwise indicated, 50 μg of cell extracts were resolved on MiniProtean Gels (BioRad Laboratories), whilst for phospho-p70 S6 kinase, phospho-4E-BP1, LC3 and p62, 150 μg of cell extracts were resolved on large PROTEAN II gels (BioRad Laboratories), and westerns developed with enhanced ECL reagents (Roche). Upon chemiluminescence detection, a series of timed exposures were undertaken to ensure that densitometric analyses were performed at exposures within the linear range. Films were scanned, and QuantityOne (Biorad) was employed for densitometric analysis.

ImmunoprecipitationTransfected cells were washed in chilled PBS and pelleted by centrifugation. Pellets were resuspended in IP buffer (1 ml/10 cm dish) containing 1% NP-40, 50 mm Tris/HCl, pH 7.4, 150 mm NaCl, phosphatase inhibitor cocktails 2 and 3 (Sigma) and protease inhibitor cocktail (Roche) and lysates centrifuged at 13 500 rpm for 10 min at 4°C. Protein concentration of supernatants was estimated using a BCA assay, and 1 mg total protein was subjected to immunoprecipitation with either anti-GFP (5 μg, Abcam) or anti-LRRK2 (10 μg, N138/6, NeuroMab, UCDavis, USA) antibodies. For immunoprecipitations, lysates were incubated with antibodies for 1 h at 4°C, followed by addition of protein G or protein A Sepharose beads and incubation overnight. Beads were washed five times with IP buffer followed by elution with SDS sample buffer and heating at 50°C for 3 min prior to separation by SDS-PAGE.

Immunoprecipitation

Transfected cells were washed in chilled PBS and pelleted by centrifugation. Pellets were resuspended in IP buffer (1 ml/10 cm dish) containing 1% NP-40, 50 mm Tris/HCl, pH 7.4, 150 mm NaCl, phosphatase inhibitor cocktails 2 and 3 (Sigma) and protease inhibitor cocktail (Roche) and lysates centrifuged at 13 500 rpm for 10 min at 4°C. Protein concentration of supernatants was estimated using a BCA assay, and 1 mg total protein was subjected to immunoprecipitation with either anti-GFP (5 μg, Abcam) or anti-LRRK2 (10 μg, N138/6, NeuroMab, UCDavis, USA) antibodies. For immunoprecipitations, lysates were incubated with antibodies for 1 h at 4°C, followed by addition of protein G or protein A Sepharose beads and incubation overnight. Beads were washed five times with IP buffer followed by elution with SDS sample buffer and heating at 50°C for 3 min prior to separation by SDS-PAGE.

Statistical analysisData are expressed as means ± s.e.m., and the significance of differences was assessed using unpaired t-tests. Differences were accepted as significant at the 95% level (P < 0.05).

Statistical analysis

Data are expressed as means ± s.e.m., and the significance of differences was assessed using unpaired t-tests. Differences were accepted as significant at the 95% level (P < 0.05).

SUPPLEMENTARY MATERIALSupplementary Material is available at HMG online.Conflict of Interest statement. None declared.

SUPPLEMENTARY MATERIAL

Supplementary Material is available at HMG online.

Conflict of Interest statement. None declared.

Supplementary MaterialSupplementary Data

Supplementary Material

Supplementary Data
