---
pmid: '16429126'
title: Proteome survey reveals modularity of the yeast cell machinery.
authors:
- Gavin AC
- Aloy P
- Grandi P
- Krause R
- Boesche M
- Marzioch M
- Rau C
- Jensen LJ
- Bastuck S
- Dümpelfeld B
- Edelmann A
- Heurtier MA
- Hoffman V
- Hoefert C
- Klein K
- Hudak M
- Michon AM
- Schelder M
- Schirle M
- Remor M
- Rudi T
- Hooper S
- Bauer A
- Bouwmeester T
- Casari G
- Drewes G
- Neubauer G
- Rick JM
- Kuster B
- Bork P
- Russell RB
- Superti-Furga G
journal: Nature
year: '2006'
pmcid: PMC12334737
doi: 10.1038/nature04532
---

# Proteome survey reveals modularity of the yeast cell machinery.
**Authors:** Gavin AC, Aloy P, Grandi P, Krause R, Boesche M, Marzioch M, Rau C, Jensen LJ, Bastuck S, Dümpelfeld B, Edelmann A, Heurtier MA, Hoffman V, Hoefert C, Klein K, Hudak M, Michon AM, Schelder M, Schirle M, Remor M, Rudi T, Hooper S, Bauer A, Bouwmeester T, Casari G, Drewes G, Neubauer G, Rick JM, Kuster B, Bork P, Russell RB, Superti-Furga G
**Journal:** Nature (2006)
**DOI:** [10.1038/nature04532](https://doi.org/10.1038/nature04532)
**PMC:** [PMC12334737](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12334737/)

## Abstract

1. Nature. 2006 Mar 30;440(7084):631-6. doi: 10.1038/nature04532. Epub 2006 Jan
22.

Proteome survey reveals modularity of the yeast cell machinery.

Gavin AC(1), Aloy P, Grandi P, Krause R, Boesche M, Marzioch M, Rau C, Jensen 
LJ, Bastuck S, Dümpelfeld B, Edelmann A, Heurtier MA, Hoffman V, Hoefert C, 
Klein K, Hudak M, Michon AM, Schelder M, Schirle M, Remor M, Rudi T, Hooper S, 
Bauer A, Bouwmeester T, Casari G, Drewes G, Neubauer G, Rick JM, Kuster B, Bork 
P, Russell RB, Superti-Furga G.

Author information:
(1)Cellzome AG, Meyerhofstrasse 1, 69117 Heidelberg, Germany.

Protein complexes are key molecular entities that integrate multiple gene 
products to perform cellular functions. Here we report the first genome-wide 
screen for complexes in an organism, budding yeast, using affinity purification 
and mass spectrometry. Through systematic tagging of open reading frames (ORFs), 
the majority of complexes were purified several times, suggesting screen 
saturation. The richness of the data set enabled a de novo characterization of 
the composition and organization of the cellular machinery. The ensemble of 
cellular proteins partitions into 491 complexes, of which 257 are novel, that 
differentially combine with additional attachment proteins or protein modules to 
enable a diversification of potential functions. Support for this modular 
organization of the proteome comes from integration with available data on 
expression, localization, function, evolutionary conservation, protein structure 
and binary interactions. This study provides the largest collection of 
physically determined eukaryotic cellular machines so far and a platform for 
biological data integration and modelling.

DOI: 10.1038/nature04532
PMID: 16429126 [Indexed for MEDLINE]

## Full Text

Author Contributions

C.Y., R.M.K., and L.H. conceptualized the overall content. L.H. and
R.M.K. directed the research. C.Y., R.A., and R.M.K. collected the
references. C.Y. and R.M.K. generated the figures. C.Y. produced the table.
All authors contributed to the writing and editing of the manuscript.

Author Information

Clinton Yu; Affiliation: 1Department of
Physiology & Biophysics, University of
California, Irvine, Irvine, CA, 92697, USA.Rithika Adavikolanu; Affiliations: 2Department of
Bioengineering, University of California, San
Francisco, San Francisco, CA, 94158, USA.
3Quantitative Biosciences Institute
(QBI), University of California, San Francisco, San
Francisco, CA, 94158, USA. 3-J. David Gladstone
Institutes, San Francisco, CA, 94158, USA.Robyn M. Kaake; Affiliations: 3Quantitative
Biosciences Institute (QBI), University of
California, San Francisco, San Francisco, CA, 94158,
USA. 3-J. David Gladstone Institutes, San Francisco,
CA, 94158, USA. 4Department of Cellular
and Molecular Pharmacology, University of
California, San Francisco, San Francisco, CA, 94158,
USA.Lan Huang; Affiliation: 1Department of
Physiology & Biophysics, University of
California, Irvine, Irvine, CA, 92697, USA.

Clinton Yu; Affiliation: 1Department of
Physiology & Biophysics, University of
California, Irvine, Irvine, CA, 92697, USA.

Affiliation: 1Department of
Physiology & Biophysics, University of
California, Irvine, Irvine, CA, 92697, USA.

Rithika Adavikolanu; Affiliations: 2Department of
Bioengineering, University of California, San
Francisco, San Francisco, CA, 94158, USA.
3Quantitative Biosciences Institute
(QBI), University of California, San Francisco, San
Francisco, CA, 94158, USA. 3-J. David Gladstone
Institutes, San Francisco, CA, 94158, USA.

Affiliations: 2Department of
Bioengineering, University of California, San
Francisco, San Francisco, CA, 94158, USA.
3Quantitative Biosciences Institute
(QBI), University of California, San Francisco, San
Francisco, CA, 94158, USA. 3-J. David Gladstone
Institutes, San Francisco, CA, 94158, USA.

Robyn M. Kaake; Affiliations: 3Quantitative
Biosciences Institute (QBI), University of
California, San Francisco, San Francisco, CA, 94158,
USA. 3-J. David Gladstone Institutes, San Francisco,
CA, 94158, USA. 4Department of Cellular
and Molecular Pharmacology, University of
California, San Francisco, San Francisco, CA, 94158,
USA.

Affiliations: 3Quantitative
Biosciences Institute (QBI), University of
California, San Francisco, San Francisco, CA, 94158,
USA. 3-J. David Gladstone Institutes, San Francisco,
CA, 94158, USA. 4Department of Cellular
and Molecular Pharmacology, University of
California, San Francisco, San Francisco, CA, 94158,
USA.

Lan Huang; Affiliation: 1Department of
Physiology & Biophysics, University of
California, Irvine, Irvine, CA, 92697, USA.

Affiliation: 1Department of
Physiology & Biophysics, University of
California, Irvine, Irvine, CA, 92697, USA.

There is no conflict of interest.

An intricate network of protein assemblies and protein-protein
interactions (PPIs) underlies nearly every biological process in living systems.
The organization of these cellular networks is highly dynamic and intimately
tied to the genomic and proteomic landscapes of a cell. Disruptions in normal
PPIs can impair cellular functions and contribute to the development of human
diseases. In recent years, targeting PPIs has emerged as an attractive strategy
for drug discovery. Consequently, the identification and characterization of
endogenous PPIs—those occurring naturally under physiological
conditions—has become crucial for unraveling the molecular mechanisms
driving human pathology and for laying the groundwork for novel diagnostics and
therapeutics. Owing to numerous technological advancements, mass spectrometry
(MS)-based proteomics has transformed the study of PPIs at the systems-level.
This review focuses on proteomics approaches that enable the characterization of
physiologically relevant endogenous interactions, spanning complex-centric to
structure-centric analyses. Additionally, their applications to define native
PPIs in the contexts of cancer and viral infectious diseases is highlighted.
