---
pmid: '29656893'
title: DNA Repair Network Analysis Reveals Shieldin as a Key Regulator of NHEJ and
  PARP Inhibitor Sensitivity.
authors:
- Gupta R
- Somyajit K
- Narita T
- Maskey E
- Stanlie A
- Kremer M
- Typas D
- Lammers M
- Mailand N
- Nussenzweig A
- Lukas J
- Choudhary C
journal: Cell
year: '2018'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC8108093
doi: 10.1016/j.cell.2018.03.050
---

# DNA Repair Network Analysis Reveals Shieldin as a Key Regulator of NHEJ and PARP Inhibitor Sensitivity.
**Authors:** Gupta R, Somyajit K, Narita T, Maskey E, Stanlie A, Kremer M, Typas D, Lammers M, Mailand N, Nussenzweig A, Lukas J, Choudhary C
**Journal:** Cell (2018)
**DOI:** [10.1016/j.cell.2018.03.050](https://doi.org/10.1016/j.cell.2018.03.050)
**PMC:** [PMC8108093](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8108093/)

## Abstract

1. Cell. 2018 May 3;173(4):972-988.e23. doi: 10.1016/j.cell.2018.03.050. Epub
2018  Apr 12.

DNA Repair Network Analysis Reveals Shieldin as a Key Regulator of NHEJ and PARP 
Inhibitor Sensitivity.

Gupta R(1), Somyajit K(2), Narita T(1), Maskey E(1), Stanlie A(3), Kremer M(4), 
Typas D(2), Lammers M(5), Mailand N(2), Nussenzweig A(3), Lukas J(2), Choudhary 
C(6).

Author information:
(1)Proteomics Program, the Novo Nordisk Foundation Center for Protein Research, 
Faculty of Health and Medical Sciences, University of Copenhagen, Blegdamsvej 
3B, DK-2200 Copenhagen, Denmark.
(2)Protein Signaling Program, the Novo Nordisk Foundation Center for Protein 
Research, Faculty of Health and Medical Sciences, University of Copenhagen, 
Blegdamsvej 3B, DK-2200 Copenhagen, Denmark.
(3)Laboratory of Genome Integrity, NIH, Bethesda, MD 20892, USA.
(4)Institute for Genetics and Cologne Excellence Cluster on Cellular Stress 
Responses in Aging-Associated Diseases (CECAD), Joseph-Stelzmann-Str. 26, 
University of Cologne, 50931 Cologne, Germany.
(5)Institute for Genetics and Cologne Excellence Cluster on Cellular Stress 
Responses in Aging-Associated Diseases (CECAD), Joseph-Stelzmann-Str. 26, 
University of Cologne, 50931 Cologne, Germany; Institute for Biochemistry, 
Synthetic and Structural Biochemistry, Felix-Hausdorff-Str. 4, University of 
Greifswald, 17487 Greifswald, Germany.
(6)Proteomics Program, the Novo Nordisk Foundation Center for Protein Research, 
Faculty of Health and Medical Sciences, University of Copenhagen, Blegdamsvej 
3B, DK-2200 Copenhagen, Denmark; Center for Chromosome Stability (CCS), Faculty 
of Health and Medical Sciences, University of Copenhagen, Blegdamsvej 3B, 
DK-2200 Copenhagen, Denmark. Electronic address: chuna.choudhary@cpr.ku.dk.

Comment in
    Nat Rev Mol Cell Biol. 2018 Jun;19(6):346-347. doi: 
10.1038/s41580-018-0019-9.

Repair of damaged DNA is essential for maintaining genome integrity and for 
preventing genome-instability-associated diseases, such as cancer. By combining 
proximity labeling with quantitative mass spectrometry, we generated 
high-resolution interaction neighborhood maps of the endogenously expressed DNA 
repair factors 53BP1, BRCA1, and MDC1. Our spatially resolved interaction maps 
reveal rich network intricacies, identify shared and bait-specific interaction 
modules, and implicate previously concealed regulators in this process. We 
identified a novel vertebrate-specific protein complex, shieldin, comprising 
REV7 plus three previously uncharacterized proteins, RINN1 (CTC-534A2.2), RINN2 
(FAM35A), and RINN3 (C20ORF196). Recruitment of shieldin to DSBs, via the 
ATM-RNF8-RNF168-53BP1-RIF1 axis, promotes NHEJ-dependent repair of 
intrachromosomal breaks, immunoglobulin class-switch recombination (CSR), and 
fusion of unprotected telomeres. Shieldin functions as a downstream effector of 
53BP1-RIF1 in restraining DNA end resection and in sensitizing BRCA1-deficient 
cells to PARP inhibitors. These findings have implications for understanding 
cancer-associated PARPi resistance and the evolution of antibody CSR in higher 
vertebrates.

Copyright © 2018 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.cell.2018.03.050
PMCID: PMC8108093
PMID: 29656893 [Indexed for MEDLINE]

Conflict of interest statement: DECLARATION OF INTERESTS The authors have no 
conflict of interest to declare.

## Full Text

INTRODUCTION

DNA damage presents a major threat to genome stability, and cells have therefore evolved elegant mechanisms, collectively referred to as the DNA damage response (DDR), to protect their genome integrity ( Jackson and Bartek, 2009 ). Among different types of damage, DNA double-strand breaks (DSBs) are one of the most severe types of DNA lesions. DSBs are repaired through two major pathways: homology-directed repair (HDR) and non-homologous end joining (NHEJ) ( Ceccaldi et al., 2016 ). The tumor suppressor 53BP1 plays an important role in DSB repair pathway choice and in enhancing the fidelity of DSB repair. In G1, 53BP1 promotes classical NHEJ over more error-prone alternative end joining ( Ceccaldi et al., 2016 ; Chapman et al., 2013 ; Escribano-Díaz et al., 2013 ); in S/G2, it guards DSB against hyper-resection to restrain mutagenic forms of HDR, such as single-strand annealing ( Ochs et al., 2016 ). RIF1 is the most proximal effector of 53BP1 in its anti-resection function, and recent work identified REV7 as a downstream factor in 53BP1-/RIF1-dependent DSB repair pathway choice ( Boersma et al., 2015 ; Xu et al., 2015 ).

In the physiological context, the 53BP1/RIF1/REV7 pathway promotes NHEJ, telomere maintenance, and immunoglobulin class-switch recombination (CSR). In the clinical context, cells lacking key HDR regulators, such the tumor suppressor BRCA1, are hyper-sensitive to PARP inhibitors (PARPi) ( Bryant et al., 2005 ; Farmer et al., 2005 ). Three different PARPi (olaparib, rucaparib, and niraparib) were recently approved for the treatment of HDR-defective cancers ( Lord and Ashworth, 2017 ). Remarkably, genetic depletion of 53BP1 , RIF1 , or REV7 provides synthetic viability to BRCA1 -deficient cells and affords resistance to PARPi in HDR-defective cells ( Boersma et al., 2015 ; Bunting et al., 2010 ; Lord and Ashworth, 2017 ; Xu et al., 2015 ).

Despite their important roles in DSB repair, 53BP1, RIF1, and REV7 contain no known enzymatic activity, no direct interactions are shown between them, and the identity of downstream effector(s) in this pathway is unknown. Identification of 53BP1 downstream effector(s) is crucial for understanding the cellular pathways and molecular mechanisms of DSB repair. A major challenge in investigating DDR networks is that isolation of the “DDR factories” is intractable to most biochemical methods, and the high-stringency conditions required for extracting chromatin-bound complexes are incompatible with preserving physiologically relevant protein-protein interactions.

To circumvent these limitations and to provide a holistic view of the “DSB repairosome,” we combined CRISPR-based genome editing ( Cong et al., 2013 ), APEX-based proximity labeling ( Rhee et al., 2013 ), and quantitative mass spectrometry (MS) to survey protein networks in the neighborhood of the endogenously expressed DDR factors 53BP1, BRCA1, and MDC1. This allowed us to map their interaction landscape in the native environment, revealing new insights into the global DDR networks. Notably, we discovered a vertebrate-specific novel protein complex, shieldin, which functions as a downstream effector in the 53BP1 pathway, regulates NHEJ in various settings, and impacts resistance to PARPi in HDR-defective cells. These findings have implications for understanding the evolution of antibody diversification, as well as for understanding the mechanisms of PARPi-resistance with possible clinical ramifications.

DISCUSSION

This study provides the hitherto most complete, proximity-based quantitative map of the DSB repair networks, demonstrating the potential of the PROX-NET approach in a systematic mapping of neighborhood networks of endogenously expressed, chromatin-associated proteins. A major strength of this approach is that it provides a systems view of spatially resolved neighborhood networks from living cells and has the potential to identify low-affinity, transient, and indirect protein interactions. These strengths of PROX-NET are complementary to the advantages of the traditional AP-MS-based protein-interaction mapping methods, which mostly detect direct protein-protein interactions and thus can provide insights into specific protein complexes and sub-networks within larger interaction networks. We anticipate that the combination of precision nuclease-based APEX tagging of endogenous proteins and PROX-NET will be extendable to map neighborhood networks of other chromatin-associated processes that are “beyond reach” for conventional AP-MS analyses.

Our analyses faithfully captured shared and bait-specific DDR networks, which are highly consistent with previously published literature, and identification of additional DDR factors highlights the usefulness of our dataset in identifying putative novel DDR factors. Notably, our analyses also identified many non-DDR proteins, mostly dominated by proteins related to transcription (for 53BP1) and to centromeres and kinetochores (for BRCA1 and MDC1). In summary, our PROX-NET-based neighborhood networks present a composite picture of DDR and non-DDR-related networks in proximity to 53BP1, BRCA1, and MDC1. The dataset can serve as a resource for further understanding the functions of the bait proteins in DDR and non-DDR related processes.

The function of 53BP1 in DNA damage was reported at the beginning of this millennium ( Rappold et al., 2001 ; Schultz et al., 2000 ), and cumulative research from the past 17 years established that 53BP1-RIF1-REV7 function in a linear pathway to regulate NHEJ and DSB repair pathway choice. Our work makes several important contributions to the understanding of the 53BP1 pathway: (1) we discover three new proteins that collectively double the number of proteins directly involved in the 53BP1 pathway, (2) we show that these three novel proteins interact with REV7 to form the shieldin complex, (3) we delineate the hierarchy in recruitment of shieldin components to DSB sites, (4) we show that shieldin functions as an 53BP1/RIF1 effector in restraining DNA end resection and promoting NHEJ, and (5) interactions among shieldin components present the first high-affinity direct protein-protein interactions within the 53BP1 pathway.

The prevailing model in DSB repair pathway choice posits that the 53BP1-RIF1 module shields DSBs from DNA end resection. Based on our findings, we expand the current model by including the shieldin complex as an essential downstream module in DSB repair pathway choice ( Figure 7F ). We propose that 53BP1-RIF1 act upstream to facilitate the recruitment of the shieldin complex, which we postulate shields DSBs against resection and thereby promotes NHEJ. Notably, RINN1–3 are expressed at very low levels, indicating that shieldin is a rate-limiting effector complex in the 53BP1 pathway. Our findings also raise new questions; for example, how is shieldin connected to the upstream components, and what are the structural and biochemical mechanisms by which it inhibits end resection? Identification of RINN1–3 and the shieldin complex presents an important step toward understanding the mechanisms by which cells make key DSB repair decisions.

The work also provides notable insights into the evolution of the 53BP1 pathway and the emergence of antibody class switching. Budding yeast orthologs of 53BP1, RIF1, and REV7 are implicated in the DDR, but their function in DSB repair pathway choice is not evolutionarily conserved in yeast. We show that vertebrate-specific RINN1 acts upstream of REV7; because recently evolved RINN1–3 have key roles in the 53BP1 pathway, we propose that the function of the 53BP1 pathway in DSB repair decision process was added recently. This is consistent with the observation that, unlike the core NHEJ regulators, such as Ku70/Ku80 and LIG4, 53BP1 is not absolutely essential for DNA end joining. However, 53BP1 is critically important for immunoglobulin CSR ( Manis et al., 2004 ), which represents one of the best-known examples of higher vertebrate-specific, NHEJ-dependent physiological process. Immunoglobulin CSR involves generation of activation-induced cytidine deaminase (AID)-induced DSBs in the S regions and subsequent repair by NHEJ. Previously known CSR-regulating factors are present in jawed vertebrates; however, the mammalian-type S regions only evolved in amphibians, and this was thought to have contributed in the evolution of CSR in amphibians. However, B cells from nurse sharks, which do not contain mammalian-type S regions, undergo antigen-induced, AID-dependent CSR ( Zhu et al., 2012 ), demonstrating that a primitive form of CSR evolved prior to the evolution of mammalian-type S regions. Given the contemporary evolution of shieldin (especially of RINN3), as well as its importance for and co-appearance with CSR, it is interesting to speculate that evolution of shieldin may have contributed to the emergence of CSR. In this context, it is notable that nurse sharks, which contain the earliest form of CSR, contain all the members of the shieldin complex.

The 53BP1 pathway is also an important modulator of PARPi sensitivity in HDR-defective tumors. Our results show that deletion of shieldin components provides resistance to PARPi in BRCA1 -depleted cells. While the clinical relevance of these findings remains to be investigated, our results are entirely consistent with the phenotypes conferred by the previously characterized components of this pathway, including 53BP1, RIF1, and REV7. It would be interesting to investigate the dysregulation and therapeutic relevance of the newly identified shieldin components in cancer.

Collectively, our work provides a panoramic view of the endogenous DDR networks and implicates novel factors in the DDR. The discovery of the shieldin complex provides new insights into the evolution of 53BP1-RIF1-dependent DSB repair pathway choice with potential implications for understanding higher vertebrate-specific evolution of antibody CSR and clinical resistance to PARP inhibitors.
