---
pmid: '25651062'
title: An acetyl-methyl switch drives a conformational change in p53.
authors:
- Tong Q
- Mazur SJ
- Rincon-Arano H
- Rothbart SB
- Kuznetsov DM
- Cui G
- Liu WH
- Gete Y
- Klein BJ
- Jenkins L
- Mer G
- Kutateladze AG
- Strahl BD
- Groudine M
- Appella E
- Kutateladze TG
journal: Structure
year: '2015'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4317571
doi: 10.1016/j.str.2014.12.010
---

# An acetyl-methyl switch drives a conformational change in p53.
**Authors:** Tong Q, Mazur SJ, Rincon-Arano H, Rothbart SB, Kuznetsov DM, Cui G, Liu WH, Gete Y, Klein BJ, Jenkins L, Mer G, Kutateladze AG, Strahl BD, Groudine M, Appella E, Kutateladze TG
**Journal:** Structure (2015)
**DOI:** [10.1016/j.str.2014.12.010](https://doi.org/10.1016/j.str.2014.12.010)
**PMC:** [PMC4317571](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4317571/)

## Abstract

1. Structure. 2015 Feb 3;23(2):322-31. doi: 10.1016/j.str.2014.12.010.

An acetyl-methyl switch drives a conformational change in p53.

Tong Q(1), Mazur SJ(2), Rincon-Arano H(3), Rothbart SB(4), Kuznetsov DM(5), Cui 
G(6), Liu WH(1), Gete Y(2), Klein BJ(1), Jenkins L(2), Mer G(6), Kutateladze 
AG(5), Strahl BD(4), Groudine M(7), Appella E(2), Kutateladze TG(8).

Author information:
(1)Department of Pharmacology, University of Colorado School of Medicine, 
Aurora, CO 80045, USA.
(2)Laboratory of Cell Biology, National Cancer Institute, NIH, Bethesda, MD 
20892, USA.
(3)Basic Science Division, Fred Hutchinson Cancer Research Center, Seattle, WA 
98109, USA.
(4)Department of Biochemistry and Biophysics and the Lineberger Comprehensive 
Cancer Center, University of North Carolina School of Medicine, Chapel Hill, NC 
27599, USA.
(5)Department of Chemistry and Biochemistry, University of Denver, Denver, CO 
80210, USA.
(6)Department of Biochemistry and Molecular Biology, Mayo Clinic College of 
Medicine, Rochester, MN 55905, USA.
(7)Basic Science Division, Fred Hutchinson Cancer Research Center, Seattle, WA 
98109, USA; Department of Radiation Oncology, University Washington School of 
Medicine, Seattle, WA 98109, USA.
(8)Department of Pharmacology, University of Colorado School of Medicine, 
Aurora, CO 80045, USA. Electronic address: tatiana.kutateladze@ucdenver.edu.

Individual posttranslational modifications (PTMs) of p53 mediate diverse 
p53-dependent responses; however, much less is known about the combinatorial 
action of adjacent modifications. Here, we describe crosstalk between the early 
DNA damage response mark p53K382me2 and the surrounding PTMs that modulate 
binding of p53 cofactors, including 53BP1 and p300. The 1.8 Å resolution crystal 
structure of the tandem Tudor domain (TTD) of 53BP1 in complex with p53 peptide 
acetylated at K381 and dimethylated at K382 (p53K381acK382me2) reveals that the 
dual PTM induces a conformational change in p53. The α-helical fold of 
p53K381acK382me2 positions the side chains of R379, K381ac, and K382me2 to 
interact with TTD concurrently, reinforcing a modular design of double PTM 
mimetics. Biochemical and nuclear magnetic resonance analyses show that other 
surrounding PTMs, including phosphorylation of serine/threonine residues of p53, 
affect association with TTD. Our findings suggest a novel PTM-driven 
conformation switch-like mechanism that may regulate p53 interactions with 
binding partners.

Copyright © 2015 Elsevier Ltd. All rights reserved.

DOI: 10.1016/j.str.2014.12.010
PMCID: PMC4317571
PMID: 25651062 [Indexed for MEDLINE]

## Full Text

INTRODUCTION

p53 undergoes numerous posttranslational modifications (PTMs) that mediate function, stability, and subcellular localization of this tumor suppressor. Recent mass-spectrometry analysis has identified 222 PTMs present on 99 residues of endogenous p53 ( DeHart et al., 2014 ). The PTMs are spread throughout the protein, however are particularly enriched in the C-terminal regulatory domain (CTD) of p53. Among the most common PTMs are phosphorylation of serine and threonine residues, methylation of arginine residues, and acetylation, methylation, ubiquitination, sumoylation and neddylation of lysine residues ( Berger, 2010 ; Dai and Gu, 2010 ). Generally, phosphorylation and acetylation are thought to activate or stabilize p53, whereas polyubiquitination targets p53 for proteasomal degradation, and methylation can be either an activating or repressive mark. Although some individual PTMs are linked to a particular p53 response, growing evidence suggests widespread crosstalk between the PTMs, which could be either synergistic or antagonistic in nature.

The p53 CTD contains six lysines within a span of 17 residues, including two pairs of contiguous lysine residues, K372K373 and K381K382. As individual lysine residues can be posttranslationally modified in a variety of ways, crosstalk between these PTMs can provide a mechanism for fine-tuning p53 activities. For example, in response to DNA damage, SET7/9-dependent monomethylation of K372 can promote acetylation of nearby lysine residues, including K373 and K382, and enhance the stability and activity of p53, ultimately upregulating p21 and triggering cell cycle arrest ( Ivanov et al., 2007 ). Methylation of K369 in mouse p53 (K372 in human ortholog) is important for the recruitment of the Tip60 lysine acetyltransferase (KAT) complex to p53 and for the subsequent acetylation of K370 and K379 (K373 and K382 in human p53) ( Kurash et al., 2008 ). In the absence of sustained damage, repressive methylation marks have been proposed to keep p53 in an inactive form, however upon DNA damage, acetylation can replace methylation, promoting p53 transcriptional activity ( Berger, 2010 ; Loewer et al., 2010 ). In agreement, activities of SMYD2 and SET8 lysine methyltransferases (KMTs) responsible for the deposition of the repressive marks p53K370me1 (p53 monomethylated at K370) and p53K382me1 (p53 monomethylated at K382) are reduced following DNA damage, while activity of the CBP/p300 KAT is increased ( Huang et al., 2006 ; Huang et al., 2007 ; Ivanov et al., 2007 ; Loewer et al., 2010 ; Shi et al., 2007 ; West et al., 2010 ). The acetylation-methylation interplay can function as a switch, allowing for distinctly different p53 responses to severe DNA damage as opposed to transient low-level DNA breaks that occur during normal cell processes ( Berger, 2010 ; Loewer et al., 2010 ).

A number of spatial and temporal correlations have been reported within the p53 methylation or acetylation pathways. The SET7/9-produced activating mark p53K372me1 (p53 monomethylated at K372) prevents repressive monomethylation of K370 by inhibiting SMYD2 priming at p53 ( Huang et al., 2006 ; Huang et al., 2007 ). Acetylation of the CTD lysine residues stimulates p53 transactivation through the recruitment of co-factors, inhibits ubiquitin ligase MDM2-mediated ubiquitination, and is essential for p53 tetramer formation ( Barlev et al., 2001 ; Itahana et al., 2009 ; Kawaguchi et al., 2006 ; Luo et al., 2004 ; Mujtaba et al., 2004 ; Yamaguchi et al., 2009 ). Binding of p53 to the transcriptional co-activator PC4 is augmented when K381 and K382 are acetylated ( Debnath et al., 2011 ). Acetylation of p53 by Tip60 selectively impedes MDM2-dependent neddylation, whereas FBXO11-facilitated neddylation suppresses p53 transcriptional activity, possibly via preventing acetylation ( Abida et al., 2007 ; Dohmesen et al., 2008 ). Sumoylation at K386 blocks consecutive acetylation of p53 by p300, while acetylation shows no antagonistic effect on sumoylation at K386 and alleviates sumoylation-induced inhibition of DNA binding. Genotoxic stress triggers phosphorylation of S366, S378 and T387, which differentially affect p53 acetylation ( Ou et al., 2005 ). Phosphorylation of S378 decreases acetylation of K373 and K382, whereas phosphorylation of S366 and T387 increases it ( Ou et al., 2005 ).

Three histone readers, including the malignant brain tumor (MBT) module of the L3MBTL1 protein, the tandem Tudor domain (TTD) of the PHF20 protein, and the TTD of p53-binding protein 1 (53BP1), have been shown to recognize methylated K382 of p53 ( Cui et al., 2012 ; Kachirskaia et al., 2008 ; Roy et al., 2010 ; Shi et al., 2007 ; West et al., 2010 ). p53 monomethylated at K382 is generated by the SET8 KMT, which is active in resting cells but is repressed following DNA damage ( Shi et al., 2007 ). In the absence of stress, the MBT module of L3MBTL1 binds to p53K382me1 at promoters of the highly responsive p53 targets, p21 and PUMA , repressing these genes and compacting chromatin ( Shi et al., 2007 ; West et al., 2010 ). The concomitant decrease in p53K382me1 levels and the increase in p53K382me2 (p53 dimethylated at K382) levels upon DNA damage cause disassociation of L3MBTL1. The dimethylated p53K382 species is recognized by the TTD of 53BP1, a DNA damage response protein and a p53 co-factor that also binds to H4K20me2 (histone 4 dimethylated at K20) ( Botuyan et al., 2006 ; Huang et al., 2007 ; Huyen et al., 2004 ; Kachirskaia et al., 2008 ). 53BP1 association with p53K382me2 has been shown to facilitate p53 accumulation and stabilization at DNA double-strand break (DSB) sites and play a role in DNA repair ( Kachirskaia et al., 2008 ; Roy et al., 2010 ).

In this study, we characterize crosstalk between PTMs in the K381K382 region and the surrounding residues of p53. We demonstrate that dimethylation of p53K382 is rapidly induced following DNA damage, whereas acetylation of p53K381 remains relatively unchanged. The TTD domain of 53BP1 recognizes the p53K381acK382me2 (p53 acetylated at K381 and dimethylated at K382) modification. This dual PTM causes a significant conformational change in the p53 structure, converting a U-shaped p53K382me2 peptide into α-helical p53K381acK382me2. Other PTMs, including phosphorylation of serine and threonine residues of p53 further influence the interaction with 53BP1 TTD. Our data suggest a novel p53 regulatory mechanism in which different combinations of PTMs trigger distinct p53 conformations when bound to specific interactors.
