---
pmid: '32188938'
title: Cryo-EM structure of SWI/SNF complex bound to a nucleosome.
authors:
- Han Y
- Reyes AA
- Malik S
- He Y
journal: Nature
year: '2020'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC7319049
doi: 10.1038/s41586-020-2087-1
---

# Cryo-EM structure of SWI/SNF complex bound to a nucleosome.
**Authors:** Han Y, Reyes AA, Malik S, He Y
**Journal:** Nature (2020)
**DOI:** [10.1038/s41586-020-2087-1](https://doi.org/10.1038/s41586-020-2087-1)
**PMC:** [PMC7319049](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7319049/)

## Abstract

1. Nature. 2020 Mar;579(7799):452-455. doi: 10.1038/s41586-020-2087-1. Epub 2020 
Mar 11.

Cryo-EM structure of SWI/SNF complex bound to a nucleosome.

Han Y(1), Reyes AA(1)(2), Malik S(1), He Y(3)(4)(5)(6).

Author information:
(1)Department of Molecular Biosciences, Northwestern University, Evanston, IL, 
USA.
(2)Interdisciplinary Biological Sciences Program, Northwestern University, 
Evanston, IL, USA.
(3)Department of Molecular Biosciences, Northwestern University, Evanston, IL, 
USA. yuanhe@northwestern.edu.
(4)Interdisciplinary Biological Sciences Program, Northwestern University, 
Evanston, IL, USA. yuanhe@northwestern.edu.
(5)Chemistry of Life Processes Institute, Northwestern University, Evanston, IL, 
USA. yuanhe@northwestern.edu.
(6)Robert H. Lurie Comprehensive Cancer Center of Northwestern University, 
Northwestern University, Chicago, IL, USA. yuanhe@northwestern.edu.

The chromatin-remodelling complex SWI/SNF is highly conserved and has critical 
roles in various cellular processes, including transcription and DNA-damage 
repair1,2. It hydrolyses ATP to remodel chromatin structure by sliding and 
evicting histone octamers3-8, creating DNA regions that become accessible to 
other essential factors. However, our mechanistic understanding of the 
remodelling activity is hindered by the lack of a high-resolution structure of 
complexes from this family. Here we report the cryo-electron microscopy 
structure of Saccharomyces cerevisiae SWI/SNF bound to a nucleosome, at 
near-atomic resolution. In the structure, the actin-related protein (Arp) module 
is sandwiched between the ATPase and the rest of the complex, with the Snf2 
helicase-SANT associated (HSA) domain connecting all modules. The body contains 
an assembly scaffold composed of conserved subunits Snf12 (also known as SMARCD 
or BAF60), Snf5 (also known as SMARCB1, BAF47 or INI1) and an asymmetric dimer 
of Swi3 (also known as SMARCC, BAF155 or BAF170). Another conserved subunit, 
Swi1 (also known as ARID1 or BAF250), resides in the core of SWI/SNF, acting as 
a molecular hub. We also observed interactions between Snf5 and the histones at 
the acidic patch, which could serve as an anchor during active DNA 
translocation. Our structure enables us to map and rationalize a subset of 
cancer-related mutations in the human SWI/SNF complex and to propose a model for 
how SWI/SNF recognizes and remodels the +1 nucleosome to generate 
nucleosome-depleted regions during gene activation9.

DOI: 10.1038/s41586-020-2087-1
PMCID: PMC7319049
PMID: 32188938 [Indexed for MEDLINE]

## Full Text

To gain insight into the molecular mechanisms of how SWI/SNF remodels chromatin, we purified endogenous SWI/SNF from S. cerevisiae, assembled the SWI/SNF-nucleosome complex in vitro (Extended Data Fig. 1) and determined its structure using single particle cryo-EM. The complex was assembled in the presence of the ATP analog ADP-BeFx that mimics the transition state of ATP hydrolysis and was determined to sub-nanometer resolution (Extended Data Fig. 2). We observed that the nucleosome is clamped between two regions of the SWI/SNF complex (Fig. 1, Supplementary Video 1). To improve the resolution, we also assembled the complex in the presence of ATPγS and determined its structure using cryo-EM (Extended Data Fig. 3). Since this structure shows features similar to the ADP-BeFx bound complex, we combined the two data sets and performed further processing (Extended Data Fig. 4). After careful 3D classification, we obtained a reconstruction of the body of SWI/SNF to an average resolution of 4.7 Å (Fig. 1a; Extended Data Fig. 4). This resolution allowed the de novo model building of the SWI/SNF complex (Fig. 1b).

The structure of the SWI/SNF complex shows a modular architecture of this complex (Fig. 2a). The Snf2 ATPase domain binds the nucleosome at super helical location (SHL) 2, the same location shown in the stand-alone structures of the Snf2 ATPase-nucleosome complexes10,11 as well as in SWR112, Chd113,14 and SNF2h15, but quite different from INO8016,17 (Extended Data Fig. 5). The Arp module is composed of Arp7, Arp9, Rtt102 and the HSA domain of Snf2 and is sandwiched between the body and the ATPase module (Fig.1). This architecture has never been observed before and is quite different from other multi-subunit remodeling complexes, including INO80 and SWR112,16,17 (Extended Data Fig. 5). The HSA of Snf2 plays an essential role in connecting the ATPase and Arp modules to the body, extending into the body and anchoring at the opposite side of the complex (Fig. 1). We therefore named this region of Snf2 the Anchor domain. This connection of the Arp module to the body through a single α helix could explain the observed flexibility of the Arp and ATPase modules in the reconstruction as evidenced by lower estimated local resolution (Extended Data Figs. 2, 3). The functional relevance of this flexibility requires further investigation.

The 4.7 Å resolution map of the body shows the helical nature of SWI/SNF (Fig. 1a) and enabled us to build a structural model with the help of prior knowledge of this important complex (Figs. 1b, 2; Methods). We then mapped the crosslinking data for apo SWI/SNF18 onto our model of the body as a validation procedure (Supplementary Table 1). Out of the 35 inter-linking lysine pairs that were mapped onto the body, 27 (77%) have a C⍺-C⍺ distance within 30 Å, while 55 out of the 60 (92%) intra-linking pairs show a C⍺-C⍺ distance within 30 Å. These comparisons demonstrate the accuracy of our model, and also indicates that the structure of the SWI/SNF body does not change drastically upon engaging a nucleosome.

The conserved subunits Swi1/ARID1/BAF250, Swi3/SMARCC/BAF150/177, Snf12/ SMARCD/BAF60 and Snf5/ SMARCB1/BAF47/INI1 assemble into the body of the SWI/SNF complex (Fig. 2), consistent with these proteins forming a core module in the human SWI/SNF complexes19. Based on the positioning of different domains and their functions, we further defined four modules of the body — the Spine, the Hinge, the Arm and the Core (Fig. 2a).

The Spine is composed of Snf12 and the C-terminal regions of Swi3 (Fig. 2b). We identified two copies (named A and B) of Swi3 in our structure, consistent with previous crosslinking data showing multiple same-residue crosslinks within Swi318. The most striking feature of the Spine is the four-helix bundle formed by the two long helices (LH1/2) of Snf12 and the Coiled-coil domains from two Swi3 (Fig. 2b; Extended Data Fig. 6a), consistent with previous finding that the RSC homologs of Snf12 (RSC6) and Swi3 (RSC8) directly interact20. The Coiled-coil domain of Swi3 has clear leucine-zipper properties, containing hydrophobic amino acids separated by 7 residues in a helical region21. Surprisingly, the two Coiled-coil domains of Swi3 have different lengths (Fig. 2b), showing an asymmetric folding (Extended Data Fig. 6b). We speculate that this might be due to the different interactions that the two Coiled-coils are involved in during complex assembly. BAF155/170 (SMARCC1/2), the human homologs of Swi3, have been indicated to form a dimer at the very first step of SWI/SNF complex assembly19. We therefore hypothesize that the two copies of Swi3 are indistinguishable at the early steps of SWI/SNF assembly, and that after engaging with other subunits, especially Snf12/SMARCD/BAF60, the symmetry is broken. Snf12 has been shown to play important roles in SWI/SNF function22, and our structure suggests that it may do so by interacting with Swi3 and contributing to the assembly of the complex. The unassigned density at the tip of the Spine shows clear β-sheet features and is directly connected to the SWIB domain of Snf12 (Extended Data Fig. 6c). This, together with the secondary structure prediction of Snf12, allowed us to assign this density to Snf12.

The Hinge is composed of the two SANT domains of Swi3 and the C-terminal helices of Snf12. SANTB contacts the C-terminal helices of Snf12 and is in close proximity to the Core sub-module, whereas SANTA is located at the top and interacts with a C-terminal segment of Swi3A (Fig. 2c). Both SANT domains contact and sandwich the Snf2 Anchor domain (Fig. 2c), playing a key role in stabilizing the ATPase within the complex.

The Arm is composed of Snf5, the N-terminal SWIRM domains of Swi3 and C-terminus of Swp82. The Snf5 Core repeat (RPT) domains each engages one copy of the Swi3 SWIRM domain (Fig. 2d, Extended Data Fig. 7a, b). Subtle differences in the two RPT-SWIRM interfaces (Extended Data Fig. 7c) are likely due to the α helix N-terminal to RPT1 (H-N) wedging between RPT1 and SWIRMA while the C-terminal region of RPT2 is packed against the opposite side of SWIRMB. The RPT1/SWIRMA connects the Arm module to the Core module by tightly associating with Swi1 (Fig. 2d). Swp82 contains an α helix that runs along Snf5/Swi3 (Fig. 2d), likely further stabilizing the Arm module. The environments that the two molecules of Swi3 experience in both the Hinge and the Arm further establish the asymmetric architecture of this homodimer (Extended Data Fig. 6b).

Swi1/ARID1/BAF250 resides in the Core module of the body, acting as a hub to integrate all other modules (Fig. 2a, e). It clearly folds into an Armadillo (ARM) repeat structure23 (Fig. 2e, Extended Data Fig. 8a). Interestingly, BAF250a, the human homolog of Swi1, was predicted to contain an ARM domain24, consistent with the highly conserved nature of this subunit. The Swi1 ARM repeat domain contains extra insertion sequences (Extended Data Fig. 8a), such as the one between helices H3 and H6. In addition to the neighboring repeats, this long insertion makes extensive contacts with the Snf5 and Swi3 subunits of the Arm as well as both the Spine and the Hinge. It contacts the Arm by wrapping on top of the Swi3 SWIRMA domain and traveling back along the Snf5 H-N (Extended Data Fig. 8b). Interestingly, this insertion also forms an α helix H4 that contacts a surface on SWIRMB, whose corresponding region on SWIRMA engages with Swi1 H1 and Snf5 H-N (Extended Data Fig. 7d), emphasizing the role of Swi1 in associating with the Arm module. In addition to this long insertion associating with the Arm, H1 of Swi1 contacts the SWIRMA domain of Swi3, whereas H3 and H8 interact with Snf5 RPT1 (Extended Data Fig. 8b), thus connecting the Arm to the Core. The Swi1 ARM repeat domain also interacts extensively with the Spine module. The entire top surface of the Swi1 ARM makes contacts with the helix bundle from the Spine, with the C-terminal helices H19 and H20 engaging the SWIB domain of Snf12 (Extended Data Fig. 8c).

The Core is also the major docking point of the Snf2 Anchor domain (Fig. 3a). H11 of Swi1 ARM interacts with an extended region of the Snf2 HSA domain, while H2, H6 and H9 contact the Anchor linker (Extended Data Fig. 9a). These interactions, together with the Hinge region sandwiching the Anchor helices of Snf2 (Extended Data Fig. 9b), further lock the ATPase in the complex. This observation is consistent with ARID1A being the branching subunit connecting the ATPase module with the rest of the SWI/SNF complex in humans19.

The modular architecture of the SWI/SNF complex revealed by our structure agrees well with the modules revealed by previous biochemical and proteomic studies25,26. The conserved SWI/SNF subunits form the structural scaffold within the complex, whereas yeast-specific subunits only occupy peripheral regions. For example, Snf6 was identified to situate at the back of the complex, spanning the Core and wrapping on top of the four-helix bundle of the Spine (Extended Data Fig. 10a). Swp82 is another yeast-specific subunit, and it is also located peripherally, making limited contacts with the rest of the complex (Extended Data Fig. 10b).

Based on our sequence conservation analysis (Supplementary Figures 2-6), we have also mapped a subset of invariant residues from the human cancer mutation database27 onto our SWI/SNF model. Although the majority of the mutations likely compromise structure and folding, many also map to protein-protein interfaces, contributing to different types of the human disease (Fig. 2b-e, 3a, Supplementary Table 2).

Our structure has also enabled us to map the interactions between the SWI/SNF complex and the nucleosome. The ATPase domain of Snf2 binds the nucleosome at SHL2 in the context of the entire complex, as reported previously for the stand-alone Snf210,11. A series of cancer patient mutations map to the Snf2 HSA-DNA interface near SHL-6, likely diminishing the remodeling efficiency by disrupting protein-DNA interactions (Fig. 3a). The yeast-specific subunit Swp82 was found to reside in close proximity to the nucleosomal DNA near SHL-2 (Extended Data Fig. 10b). We also observed that Snf6 is close to the extranucleosomal DNA proximal to the nucleosome, in good agreement with previous site-directed DNA crosslinking experiments28. At a lower threshold, we observed additional density for extranucleosomal DNA running along the body (Extended Data Fig. 10c), suggesting flexibility of this DNA segment. However, when we prepared the SWI/SNF-nucleosome complex using a nucleosome with no overhanging DNA sequence (data not shown), we failed to observe stable complex formation using our nucleosome pull-down approach, suggesting the importance of the extranucleosomal DNA in nucleosome binding to SWI/SNF in our experimental setup. Interestingly, this extranucleosomal DNA also coincides with the possible trajectories of the N-terminal regions of both Swi1 and Snf5 (Extended Data Fig. 10c), which have been shown to interact with acidic transcription activators29. This could explain how SWI/SNF is recruited by transcription activators to its target loci for chromatin remodeling, leading to an activated gene transcription.

A connecting density is observed between the histones and Snf5 C-terminus (Fig. 3b, c), consistent with the histone crosslinking experiments18,28. This density likely corresponds to the highly conserved Snf5 arginine-rich region that interacts with the acidic patch of the histone octamer (Fig. 3b, c), suggesting a conserved mechanism of octamer recognition. Deletion of the RPT domains in Snf5 uncouples ATP hydrolysis by Snf2 with the chromatin remodeling activity18. Our structure suggests an anchoring role of the Arm module during active remodeling, in which Snf5 locks the histones in place as the nucleosomal DNA is being translocated, thus coupling ATP hydrolysis with chromatin remodeling (Fig. 3d). In contrast, this anchoring role is primarily carried out by the Arp module in other large remodeling complexes, including INO80 and SWR112,16,17 (Extended Data Fig. 5).

It has been well documented that the one of the natural substrates for SWI/SNF is the +1 nucleosome situated near the promoter9. Therefore, the extranucleosomal DNA at the exit side of the nucleosome in our structure corresponds to upstream promoter DNA, consistent with SWI/SNF’s function in generating the nucleosome-depleted regions during gene activation. Whether the same model holds true at other positions where Snf2 also binds30 needs further investigation.

MethodsSWI/SNF purification.SWI/SNF complex was purified from a yeast strain containing a TAP tag at the C-terminus of Snf231 (obtained from the High Throughput Analysis Laboratory at Northwestern University). Tandem affinity purification was performed as following. The tagged yeast strain was grown to an optical density at 600nm (OD600) of 4-5 in 12 liters of YPD (3% glucose). Next, cells were harvested by centrifugation and washed with 200 ml of cold TAP Extraction Buffer (40 mM Tris pH 8, 250 mM ammonium sulfate, 1mM EDTA, 10% glycerol, 0.1% Tween 20, 5 mM dithiothreitol [DTT], 2 mM phenylmethylsulfonyl fluoride [PMSF], 0.31 mg/ml benzamidine, 0.3 μg/ml leupeptin, 1.4 μg/ml pepstatin, 2 μg/ml chymostatin). Cells were resuspended in 150 ml cold TAP Extraction Buffer and lysed in a BeadBeater (Biospec Products). Cell debris was removed by centrifugation at 14,000 ×g at 4°C for 1 hr. For the first affinity step, 2 ml IgG Sepharose beads (GE Healthcare) were incubated with the lysate at 4°C overnight. The beads were next washed and resuspended in 4 ml cold TEV (tobacco etch virus) Cleavage Buffer (10 mM Tris pH8, 150 mM NaCl, 0.1% NP-40, 0.5 mM EDTA, 10% glycerol). TEV cleavage using 25 μg of TEV protease was performed at room temperature for 1 hr with gentle shaking. The TEV protease-cleaved products were collected, and the IgG beads were washed with 3 column volumes (~6 ml total) cold Calmodulin Binding Buffer (15 mM HEPES pH7.6, 1 mM magnesium acetate, 1 mM imidazole, 2 mM CaCl2, 0.1% NP-40, 10% glycerol, 200 mM ammonium sulfate, 5 mM DTT, 2 mM PMSF, 0.31 mg/ml benzamidine, 0.3 μg/ml leupeptin, 1.4 μg/ml pepstatin, 2 μg/ml chymostatin). CaCl2 was added to the combined eluate at a final concentration of 2 mM and incubated with 0.8 ml Calmodulin Affinity Resin (Agilent Technologies) at 4°C for 2 hours. After incubation, the beads were washed with cold Calmodulin Binding Buffer and cold Calmodulin Wash Buffer (same as Calmodulin Binding Buffer, but containing 0.01% NP-40), and bound proteins were eluted with Calmodulin Elution Buffer (15 mM HEPES pH 7.6, 1 mM magnesium acetate, 1 mM imidazole, 2 mM EGTA, 10% glycerol, 0.01% NP-40, 200 mM ammonium sulfate) at room temperature. Fractions containing the SWI/SNF complex were combined and concentrated to a final concentration of ~3-4 mg/ml (280 nm absorption) using a concentrator (Amicon Ultra-4 Ultracel 30K, Millipore). Concentrated complex was resolved on a 4-12% Bis-Tris gel (Invitrogen) followed by Coomassie blue staining (Extended Data Fig.1a and Supplementary Figure 1), and aliquoted, flash frozen in liquid nitrogen and stored at −80℃.Nucleosome reconstitution.Mono-nucleosome was reconstituted with Xenopus histones and the 601 DNA32 using the Mini Prep Cell (Bio-rad) as described previously33. The Xenopus histones were obtained from the Histone Source – the Protein Expression and Purification (PEP) Facility at Colorado State University. DNA oligonucleotides containing the 601 sequence were purchased from IDT (Integrated DNA Technology): top strand, 5’-ACCTCCCACTATTTTATGCGCCGGTATTGAACCACGCTTATGCCCAGCATCGTTAATCGATGTATATATCTGACACGTGCCTGGAGACTAGGGAGTAATCCCCTTGGCGGTTAAAACGCGGGGGACAGCGCGTACGTGCGTTTAAGCGGTGCTAGAGCTGTCTACGACCAATTGAGCGGCCTCGGCACCGGGATTCTGAT-3’; bottom strand, 5’-ATCAGAATCCCGGTGCCGAGGCCGCTCAATTGGTCGTAGACAGCTCTAGCACCGCTTAAACGCACGTACGCGCTGTCCCCCGCGTTTTAACCGCCAAGGGGATTACTCCCTAGTCTCCAGGCACGTGTCAGATATATACATCGATTAACGATGCTGGGCATAAGCGTGGTTCAATACCGGCGCAT-3’. The 601 sequence is underlined. The lyophilized DNA oligos were resuspended in water to a final concentration of ~100 μM and mixed at 1:1 molar ratio. Annealing of the DNA was performed by incubating in boiling water for 5 min followed by gradually cooling to room temperature in 2 hours. The reconstituted nucleosome core particle (NCP) was concentrated to ~6μM and annealed with a biotinylated RNA molecule (IDT, 5’-UAGUGGGAGGU-3’-biotin) to the top DNA strand at 1:1.5 (DNA to RNA) molar ratio at 45℃ for 5 min followed by gradually cooling to room temperature in 30-40 min. This resulted in a final concentration of the nucleosome core particle (NCP) at 5.52 μM. The annealed NCP was stored at 4℃.SWI/SNF-NCP assembly.To assemble the SWI/SNF-NCP complex, we modified our approach of reconstituting Pol I/II/III pre-initiation complexes (PIC)34-36 and used the NCP to replace the nucleic acid scaffold. Specifically, 0.4 μl of the biotin-RNA-annealed NCP (0.552 μM, 1/10 of the storage concentration) was first mixed with 1μl of the assembly buffer (12 mM HEPES pH 7.9, 0.12 mM EDTA, 12% glycerol, 8.25 mM MgCl2, 1 mM DTT, 2 mM ADP, 32mM KF, 4mM BeCl2 and 0.05% NP-40 [Roche]). Next, 1 μl of the concentrated SWI/SNF complex was added to this mixture and incubated at room temperature for 2 hours. Assembled complex was immobilized onto the magnetic streptavidin T1 beads (Invitrogen) which had been equilibrated with the assembly buffer plus 60 mM KCl and minus ADP-BeFx. Following washing of the beads two times using a wash buffer (10 mM HEPES, 10 mM Tris, pH 7.9, 5% glycerol, 5 mM MgCl2, 50 mM KCl, 1 mM DTT, 0.05% NP-40, 1 mM ADP, 16mM KF, 2mM BeCl2), the complex was eluted by incubating the beads at room temperature for 30 min with 3μl digestion buffer containing 10 mM HEPES, pH 7.9, 10 mM MgCl2, 50 mM KCl, 1 mM DTT, 5% glycerol, 0.05% NP-40, 1 mM ADP, 16mM KF, 2mM BeCl2 and 0.05 unit/μl RNase H (New England Biolabs). The SWI/SNF-NCP complex assembled in the presence of ATPγS was performed essentially as described above with 1mM (2mM in the first assembly buffer) ATPγS replacing ADP-BeFx in the buffers.Electron microscopy.The assembled SWI/SNF-NCP complex was first crosslinked using 0.05% glutaraldehyde under very low illumination conditions on ice for 5 min before applied onto EM grids. Negative staining sample preparation and data collection were performed as previously described35. For cryo sample preparation, crosslinked complex (~3.3 μl) was applied onto a 400 mesh Quantifoil grid containing 3.5 μm holes and 1 μm spacing (Quantifoil 3.5/1, Electron Microscopy Sciences). A thin carbon film was floated onto the grid before it was plasma cleaned for 10s at 5 W power using a Solarus plasma cleaner (Gatan) equipped with air immediately before sample deposition. The sample was allowed to absorb to the grid for 10 min at 4℃ and 100% humidity in a Vitrobot (FEI) under low illumination conditions, before blotted for 4 s at 10 force and plunge-frozen in liquid ethane. The frozen grids were stored in liquid nitrogen until imaging.Cryo-EM data collection was performed using a JEOL 3200FS transmission electron microscope (JEOL) equipped with a K2 Summit direct electron detector (Gatan) operating at 200kV (Supplementary Table 3). Data were collected using the K2 camera in counting mode at a nominal magnification of 30,000 × (1.12 Å per pixel). Movie series with defocus values ranging from −1.5 to −4.5 μm were collected using Leginon37. 40-frame exposures were taken at 0.3 s per frame (12 s total), using a dose rate of 8 e− per pixel per second, corresponding to a total dose of 76.5 e− Å−2 per movie series. Four datasets with a total number of 7,769 movies on the ADP-BeFx sample and four other datasets with a total number of 6,903 movies on the ATPγS sample were collected.Image processing and three-dimensional reconstruction.Negative stain data pre-processing was performed using the Appion processing environment38. Particles were automatically selected from the micrographs using a difference of Gaussians (DoG) particle picker39. The contract transfer function (CTF) of each micrograph was estimated using CTFFind440, the phases were flipped using CTFFind4, and particle stacks were extracted using a box size of 128 × 128 pixels. Two-dimensional classification was conducted using iterative multivariate statistical analysis and multi-reference alignment analysis (MSA-MRA) within the IMAGIC software41. Three-dimensional (3D) reconstruction of negative stained data was performed using an iterative multi-reference projection-matching approach containing libraries from the EMAN2 software package42. The initial 3D model was generated using cryoSPARC43.Cryo-EM data was pre-processed as follows. Movie frames were aligned using MotionCor244 to correct for specimen motion. Particles were automatically selected from the aligned and dose-weighted micrographs using Gautomatch (developed by Zhang K, MRC Laboratory of Molecular Biology, Cambridge, UK) with 2-fold binnning (corresponding to 2.24Å/pixel). The CTF of each micrograph and of each particle was estimated using Gctf45. All three-dimensional (3D) classification and refinement steps together with postprocess and local resolution estimation were performed within RELION 3.046.For the ADP-BeFx dataset, 891,573 particles were automatically picked and were subjected to an initial round of 3D classification with alignment using the density obtained from negative staining as the initial reference (Extended Data Fig. 2). The “Angular sampling interval”, “Offset search range (pix)” and “Offset search step (pix)” were set to 15 degrees, 10 and 2, respectively, for the first 50 iterations. Next, these values were set back to default (7.5 degrees, 5, 1) and the 3D classification was continued until convergence. This resulted in class 3 with 198,543 particles showing sharp structural features of SWI/SNF and nucleosome. This class was subsequently refined and further classified without alignment into 5 classes with a mask around the Arp module and the nucleosome (Extended Data Fig. 2b). Class 1 with 35,214 particles from this second round of classification showed best features of the nucleosome and was chosen to proceed with 3D auto-refinement, which yielded a structure of SWI/SNF-NCP at an overall resolution of 8.96Å (Extended Data Fig. 2c). All resolutions reported herein correspond to the gold-standard Fourier shell correlation (FSC) using the 0.143 criterion47. The ATPγS dataset with 820,117 particles was processed in a similar manner (Extended Data Fig. 3), resulting in a structure with an overall resolution of 10Å.To focus on the body of SWI/SNF, we combined the particles from both samples after the first round of 3D classification with a total number of 390,573 particles (Extended Data Fig. 4). Next, signal subtraction on the nucleosome and the lower half of the Arp module was performed as previously described48, leaving the SWI/SNF body and the top half of the Arp module intact. Subsequently, a 3D classification was performed with only local alignment turned on. This resulted in class 5 with 61,518 particles showing the best structural features of the body (Extended Data Fig. 4b). Next, we unbinned and refined the original particle stack of this class, and generated masks around the body, the Arp module plus the ATPase density of Snf2, and the nucleosome (Extended Data Fig. 4b). 3D multi-body refinement49 was then performed on this class, which drastically improved the resolution of the body to 4.7Å (Extended Data Fig. 4c). However, the Arp module plus the ATPase of Snf2 and the nucleosome densities resulted in worse resolution. We speculate this could be due to the flexible nature of this complex and that the continuous carbon we used for sample preparation might hinder the signal-to-noise ratio that these regions were resolved at a lower resolution. The subtle conformational differences between the complexes binding to the two ATP analogs might also contribute to the lower resolution of these regions. The core region of the body has a resolution close to 4.3Å (Extended Data Fig. 4b), showing densities of bulky sidechains, which enabled us to partially build the structural model of the body (Fig. 1b). This body map replaced its corresponding region in the ADP-BeFx map to result in the composite map shown in Fig. 1a.Model building.To aid in model building, we performed secondary structure prediction of the SWI/SNF subunits using the Genesilico Metaserver50. Sequence alignment of the conserved SWI/SNF subunits were performed using CLC Sequence Viewer 7 (Supplementary Figures 2-6). To build the structural model of the SWI/SNF body, we first performed rigid body docking of known structures into our 4.7Å body map. The rigid-body docking was performed in UCSF Chimera51,52, which yielded good fit of the following structures: the SNF5 Repeat domains (RPTs) and Swi3 SWIRM domains from the human BAF47/BAF155 complex (PDB ID 5GJK)53, the SANT domain of the yeast Swi3 (PDB ID 2YUS), and the SWIB domain of mouse BAF60a (PDB ID 1UHR). The BAF47/BAF155 heterodimer and the Swi3 SANT domain can be docked in the density map at two distinct locations, indicating two copies of these domains. Indeed, chemical crosslinking combined with mass spectrometry has shown at least two copies of Swi3 in the yeast SWI/SNF complex18, confirming our docking experiment. Yeast Snf5 has been annotated with two SNF5 RPT domains in Pfam (http://pfam.xfam.org/protein/P18480). We observed clear density in our map that connects these two RPT domains. Next, we built homology models of these structures using Modeller54 and replaced the docked PDBs in the body density map. Regions with missing or extra connecting density were then manually deleted or built in Coot55 based also on secondary structure predictions of these proteins.The Snf2 Anchor domain was built manually in Coot. First, the Arp7/Arp9/Rtt102/HSA structure (PDB ID 4I6M)56 was rigid body docked into the full map, which helped in registering the HSA helix in the body map. The HSA helix was then manually extended in Coot, with Y586 matching a sidechain density further confirming the register of this helix. Next, the Anchor domain was manually extended from the end of the HSA by following the connected density of the map. Again, secondary structure prediction was also used as a guide when extending the model in Coot. Bulky sidechain density at Y497, Y533 and W554 further confirmed the model.The ARM repeat domain of Swi1 locates in the core region of the body map with the highest local resolution, therefore enabling de novo model building. First, the helix density corresponding to residues 942-955 of Swi1 was chosen to model because it has the highest local resolution and that it contains a few bulky sidechain densities. Next, two ⍺ helices with poly-alanine sequence were generated in Coot, which allowed us to create a bulky residue (lysine, arginine, histidine, methionine, phenylalanine, tyrosine, and tryptophan) pattern along both directions. Subsequently, these patterns were used to search against the sequences of SWI/SNF subunits on the Sequence pattern search server (http://www-archbac.u-psud.fr/genomics/patternSearch.html), and Swi1 942-955 was one of the best hit. Further extension of this helix into connected density also matched the secondary structures of Swi1. Then, the remaining regions of the Swi1 ARM repeat domain were manually built into the density in Coot based on secondary structure prediction as well as bulky sidechain densities wherever possible. The overall architecture of the ARM repeat domain of Swi1 also matches that of an Armadillo repeat containing protein β-catenin57 (Extended Data Fig. 8a), confirming our model of Swi1.The positioning of the SWIB domain of Snf12 aided us in building the remaining of this protein into the density. First, at the Spine tip, where the SWIB was docked, there is β-sheet like density (Extended Data Fig. 6c). This agrees with the secondary structure prediction of Snf12, which shows β-strands right N-terminus of the SWIB domain. Although the resolution of this region is low, we are confident about its identity. Four long helices belonging to the Spine module directly connect to this region, two of which extending into the Snf12 densities. Therefore, we assigned these two helices to Snf12. This agreed well with the secondary structure prediction of Snf12, which shows that Snf12 contains two long helices. Next, we performed protein sequence pattern search based on the bulky sidechain densities. Based on the search results, we manually built the two helices of Snf12 in Coot.Based on secondary structure prediction, we reasoned that the other two long helices belong to Swi3 C-terminus. This is backed up by the finding that the C-terminus of Swi3 contains a coiled-coil leucine zipper motif21 and there are two copies of Swi3 in SWI/SNF. To facilitate the registering of the sequence in these long helices, we fitted the crystal structure of human OmoMYC homodimer (PDB ID 5I4Z)58 into the density and obtained a good fit (Extended Data Fig. 6a). Based on this fitting, we mapped the hydrophobic residues from Swi3 as indicated before21 and manually built the two helices in Coot. The rest of Swi3 density in the Spine and Hinge cannot be confidently modeled due to lower resolution and missing density, therefore are modeled with poly-alanine.Snf6 was also manually built in Coot based on secondary structure prediction, bulky sidechain density and prior knowledge based on chemical crosslinking and mass spectrometry data18 and site-directed DNA crosslinking experiments28. We cannot confidently model in Swp82, however we were able to assign densities to this yeast specific subunit based on crosslinking experiments18 and mapping by deletions and EM59. The N-terminal region of Swp82 forms a RSC7 homology domain, therefore we speculate that it occupies the globular density near the Hinge; C-terminal region of Swp82 crosslinks to regions near Snf5 RPT domains and Swi3 SWIRM domain, therefore it was assigned to the density close to Snf5 and Swi3 in the Arm module. This also agrees with the mapping by deletion analysis59. There are also several unassigned densities on the solvent exposed surface of the complex. We did not identify Taf14 and Snf11 in the map (Supplementary Table 4).The molecular model of SWI/SNF body was then refined using Namdinator60 (Supplementary Table 3). To obtain the model for the full complex, we rigid-body fitted the body, the Arp module (PDB ID 4I6M)56 and the ATPase-nucleosome bound with ADP-BeFx (PDB ID 5Z3V)11 into the map of the full complexes in Coot. Then, the HSA helix was connected manually, and the DNA sequence was modified to match our sequence. The extra DNA was manually extended by 10bp using B form DNA in Coot. The figures were prepared using UCSF Chimera and ChimeraX61. Cα-Cα distances from crosslinked lysine pairs18 were measured in UCSF Chimera. 30 Å was chosen as the cutoff62. For crosslinks involving the two molecules of Swi3, we picked the combination that gave the shortest distance as the measurement (Supplementary Table 1).Data availabilityCryo-EM density maps have been deposited in the Electron Microscopy Data Bank (EMDB) under accession numbers EMD-20934 (ADP-BeFx), EMD-20935 (ATPγS), EMD-20933 (body). Model coordinates have been deposited in the Protein Data Bank (PDB) under accession numbers 6UXW (ADP-BeFx), 6UXV (body).

SWI/SNF purification.SWI/SNF complex was purified from a yeast strain containing a TAP tag at the C-terminus of Snf231 (obtained from the High Throughput Analysis Laboratory at Northwestern University). Tandem affinity purification was performed as following. The tagged yeast strain was grown to an optical density at 600nm (OD600) of 4-5 in 12 liters of YPD (3% glucose). Next, cells were harvested by centrifugation and washed with 200 ml of cold TAP Extraction Buffer (40 mM Tris pH 8, 250 mM ammonium sulfate, 1mM EDTA, 10% glycerol, 0.1% Tween 20, 5 mM dithiothreitol [DTT], 2 mM phenylmethylsulfonyl fluoride [PMSF], 0.31 mg/ml benzamidine, 0.3 μg/ml leupeptin, 1.4 μg/ml pepstatin, 2 μg/ml chymostatin). Cells were resuspended in 150 ml cold TAP Extraction Buffer and lysed in a BeadBeater (Biospec Products). Cell debris was removed by centrifugation at 14,000 ×g at 4°C for 1 hr. For the first affinity step, 2 ml IgG Sepharose beads (GE Healthcare) were incubated with the lysate at 4°C overnight. The beads were next washed and resuspended in 4 ml cold TEV (tobacco etch virus) Cleavage Buffer (10 mM Tris pH8, 150 mM NaCl, 0.1% NP-40, 0.5 mM EDTA, 10% glycerol). TEV cleavage using 25 μg of TEV protease was performed at room temperature for 1 hr with gentle shaking. The TEV protease-cleaved products were collected, and the IgG beads were washed with 3 column volumes (~6 ml total) cold Calmodulin Binding Buffer (15 mM HEPES pH7.6, 1 mM magnesium acetate, 1 mM imidazole, 2 mM CaCl2, 0.1% NP-40, 10% glycerol, 200 mM ammonium sulfate, 5 mM DTT, 2 mM PMSF, 0.31 mg/ml benzamidine, 0.3 μg/ml leupeptin, 1.4 μg/ml pepstatin, 2 μg/ml chymostatin). CaCl2 was added to the combined eluate at a final concentration of 2 mM and incubated with 0.8 ml Calmodulin Affinity Resin (Agilent Technologies) at 4°C for 2 hours. After incubation, the beads were washed with cold Calmodulin Binding Buffer and cold Calmodulin Wash Buffer (same as Calmodulin Binding Buffer, but containing 0.01% NP-40), and bound proteins were eluted with Calmodulin Elution Buffer (15 mM HEPES pH 7.6, 1 mM magnesium acetate, 1 mM imidazole, 2 mM EGTA, 10% glycerol, 0.01% NP-40, 200 mM ammonium sulfate) at room temperature. Fractions containing the SWI/SNF complex were combined and concentrated to a final concentration of ~3-4 mg/ml (280 nm absorption) using a concentrator (Amicon Ultra-4 Ultracel 30K, Millipore). Concentrated complex was resolved on a 4-12% Bis-Tris gel (Invitrogen) followed by Coomassie blue staining (Extended Data Fig.1a and Supplementary Figure 1), and aliquoted, flash frozen in liquid nitrogen and stored at −80℃.

SWI/SNF purification.

SWI/SNF complex was purified from a yeast strain containing a TAP tag at the C-terminus of Snf231 (obtained from the High Throughput Analysis Laboratory at Northwestern University). Tandem affinity purification was performed as following. The tagged yeast strain was grown to an optical density at 600nm (OD600) of 4-5 in 12 liters of YPD (3% glucose). Next, cells were harvested by centrifugation and washed with 200 ml of cold TAP Extraction Buffer (40 mM Tris pH 8, 250 mM ammonium sulfate, 1mM EDTA, 10% glycerol, 0.1% Tween 20, 5 mM dithiothreitol [DTT], 2 mM phenylmethylsulfonyl fluoride [PMSF], 0.31 mg/ml benzamidine, 0.3 μg/ml leupeptin, 1.4 μg/ml pepstatin, 2 μg/ml chymostatin). Cells were resuspended in 150 ml cold TAP Extraction Buffer and lysed in a BeadBeater (Biospec Products). Cell debris was removed by centrifugation at 14,000 ×g at 4°C for 1 hr. For the first affinity step, 2 ml IgG Sepharose beads (GE Healthcare) were incubated with the lysate at 4°C overnight. The beads were next washed and resuspended in 4 ml cold TEV (tobacco etch virus) Cleavage Buffer (10 mM Tris pH8, 150 mM NaCl, 0.1% NP-40, 0.5 mM EDTA, 10% glycerol). TEV cleavage using 25 μg of TEV protease was performed at room temperature for 1 hr with gentle shaking. The TEV protease-cleaved products were collected, and the IgG beads were washed with 3 column volumes (~6 ml total) cold Calmodulin Binding Buffer (15 mM HEPES pH7.6, 1 mM magnesium acetate, 1 mM imidazole, 2 mM CaCl2, 0.1% NP-40, 10% glycerol, 200 mM ammonium sulfate, 5 mM DTT, 2 mM PMSF, 0.31 mg/ml benzamidine, 0.3 μg/ml leupeptin, 1.4 μg/ml pepstatin, 2 μg/ml chymostatin). CaCl2 was added to the combined eluate at a final concentration of 2 mM and incubated with 0.8 ml Calmodulin Affinity Resin (Agilent Technologies) at 4°C for 2 hours. After incubation, the beads were washed with cold Calmodulin Binding Buffer and cold Calmodulin Wash Buffer (same as Calmodulin Binding Buffer, but containing 0.01% NP-40), and bound proteins were eluted with Calmodulin Elution Buffer (15 mM HEPES pH 7.6, 1 mM magnesium acetate, 1 mM imidazole, 2 mM EGTA, 10% glycerol, 0.01% NP-40, 200 mM ammonium sulfate) at room temperature. Fractions containing the SWI/SNF complex were combined and concentrated to a final concentration of ~3-4 mg/ml (280 nm absorption) using a concentrator (Amicon Ultra-4 Ultracel 30K, Millipore). Concentrated complex was resolved on a 4-12% Bis-Tris gel (Invitrogen) followed by Coomassie blue staining (Extended Data Fig.1a and Supplementary Figure 1), and aliquoted, flash frozen in liquid nitrogen and stored at −80℃.

Nucleosome reconstitution.Mono-nucleosome was reconstituted with Xenopus histones and the 601 DNA32 using the Mini Prep Cell (Bio-rad) as described previously33. The Xenopus histones were obtained from the Histone Source – the Protein Expression and Purification (PEP) Facility at Colorado State University. DNA oligonucleotides containing the 601 sequence were purchased from IDT (Integrated DNA Technology): top strand, 5’-ACCTCCCACTATTTTATGCGCCGGTATTGAACCACGCTTATGCCCAGCATCGTTAATCGATGTATATATCTGACACGTGCCTGGAGACTAGGGAGTAATCCCCTTGGCGGTTAAAACGCGGGGGACAGCGCGTACGTGCGTTTAAGCGGTGCTAGAGCTGTCTACGACCAATTGAGCGGCCTCGGCACCGGGATTCTGAT-3’; bottom strand, 5’-ATCAGAATCCCGGTGCCGAGGCCGCTCAATTGGTCGTAGACAGCTCTAGCACCGCTTAAACGCACGTACGCGCTGTCCCCCGCGTTTTAACCGCCAAGGGGATTACTCCCTAGTCTCCAGGCACGTGTCAGATATATACATCGATTAACGATGCTGGGCATAAGCGTGGTTCAATACCGGCGCAT-3’. The 601 sequence is underlined. The lyophilized DNA oligos were resuspended in water to a final concentration of ~100 μM and mixed at 1:1 molar ratio. Annealing of the DNA was performed by incubating in boiling water for 5 min followed by gradually cooling to room temperature in 2 hours. The reconstituted nucleosome core particle (NCP) was concentrated to ~6μM and annealed with a biotinylated RNA molecule (IDT, 5’-UAGUGGGAGGU-3’-biotin) to the top DNA strand at 1:1.5 (DNA to RNA) molar ratio at 45℃ for 5 min followed by gradually cooling to room temperature in 30-40 min. This resulted in a final concentration of the nucleosome core particle (NCP) at 5.52 μM. The annealed NCP was stored at 4℃.

Nucleosome reconstitution.

Mono-nucleosome was reconstituted with Xenopus histones and the 601 DNA32 using the Mini Prep Cell (Bio-rad) as described previously33. The Xenopus histones were obtained from the Histone Source – the Protein Expression and Purification (PEP) Facility at Colorado State University. DNA oligonucleotides containing the 601 sequence were purchased from IDT (Integrated DNA Technology): top strand, 5’-ACCTCCCACTATTTTATGCGCCGGTATTGAACCACGCTTATGCCCAGCATCGTTAATCGATGTATATATCTGACACGTGCCTGGAGACTAGGGAGTAATCCCCTTGGCGGTTAAAACGCGGGGGACAGCGCGTACGTGCGTTTAAGCGGTGCTAGAGCTGTCTACGACCAATTGAGCGGCCTCGGCACCGGGATTCTGAT-3’; bottom strand, 5’-ATCAGAATCCCGGTGCCGAGGCCGCTCAATTGGTCGTAGACAGCTCTAGCACCGCTTAAACGCACGTACGCGCTGTCCCCCGCGTTTTAACCGCCAAGGGGATTACTCCCTAGTCTCCAGGCACGTGTCAGATATATACATCGATTAACGATGCTGGGCATAAGCGTGGTTCAATACCGGCGCAT-3’. The 601 sequence is underlined. The lyophilized DNA oligos were resuspended in water to a final concentration of ~100 μM and mixed at 1:1 molar ratio. Annealing of the DNA was performed by incubating in boiling water for 5 min followed by gradually cooling to room temperature in 2 hours. The reconstituted nucleosome core particle (NCP) was concentrated to ~6μM and annealed with a biotinylated RNA molecule (IDT, 5’-UAGUGGGAGGU-3’-biotin) to the top DNA strand at 1:1.5 (DNA to RNA) molar ratio at 45℃ for 5 min followed by gradually cooling to room temperature in 30-40 min. This resulted in a final concentration of the nucleosome core particle (NCP) at 5.52 μM. The annealed NCP was stored at 4℃.

SWI/SNF-NCP assembly.To assemble the SWI/SNF-NCP complex, we modified our approach of reconstituting Pol I/II/III pre-initiation complexes (PIC)34-36 and used the NCP to replace the nucleic acid scaffold. Specifically, 0.4 μl of the biotin-RNA-annealed NCP (0.552 μM, 1/10 of the storage concentration) was first mixed with 1μl of the assembly buffer (12 mM HEPES pH 7.9, 0.12 mM EDTA, 12% glycerol, 8.25 mM MgCl2, 1 mM DTT, 2 mM ADP, 32mM KF, 4mM BeCl2 and 0.05% NP-40 [Roche]). Next, 1 μl of the concentrated SWI/SNF complex was added to this mixture and incubated at room temperature for 2 hours. Assembled complex was immobilized onto the magnetic streptavidin T1 beads (Invitrogen) which had been equilibrated with the assembly buffer plus 60 mM KCl and minus ADP-BeFx. Following washing of the beads two times using a wash buffer (10 mM HEPES, 10 mM Tris, pH 7.9, 5% glycerol, 5 mM MgCl2, 50 mM KCl, 1 mM DTT, 0.05% NP-40, 1 mM ADP, 16mM KF, 2mM BeCl2), the complex was eluted by incubating the beads at room temperature for 30 min with 3μl digestion buffer containing 10 mM HEPES, pH 7.9, 10 mM MgCl2, 50 mM KCl, 1 mM DTT, 5% glycerol, 0.05% NP-40, 1 mM ADP, 16mM KF, 2mM BeCl2 and 0.05 unit/μl RNase H (New England Biolabs). The SWI/SNF-NCP complex assembled in the presence of ATPγS was performed essentially as described above with 1mM (2mM in the first assembly buffer) ATPγS replacing ADP-BeFx in the buffers.

SWI/SNF-NCP assembly.

To assemble the SWI/SNF-NCP complex, we modified our approach of reconstituting Pol I/II/III pre-initiation complexes (PIC)34-36 and used the NCP to replace the nucleic acid scaffold. Specifically, 0.4 μl of the biotin-RNA-annealed NCP (0.552 μM, 1/10 of the storage concentration) was first mixed with 1μl of the assembly buffer (12 mM HEPES pH 7.9, 0.12 mM EDTA, 12% glycerol, 8.25 mM MgCl2, 1 mM DTT, 2 mM ADP, 32mM KF, 4mM BeCl2 and 0.05% NP-40 [Roche]). Next, 1 μl of the concentrated SWI/SNF complex was added to this mixture and incubated at room temperature for 2 hours. Assembled complex was immobilized onto the magnetic streptavidin T1 beads (Invitrogen) which had been equilibrated with the assembly buffer plus 60 mM KCl and minus ADP-BeFx. Following washing of the beads two times using a wash buffer (10 mM HEPES, 10 mM Tris, pH 7.9, 5% glycerol, 5 mM MgCl2, 50 mM KCl, 1 mM DTT, 0.05% NP-40, 1 mM ADP, 16mM KF, 2mM BeCl2), the complex was eluted by incubating the beads at room temperature for 30 min with 3μl digestion buffer containing 10 mM HEPES, pH 7.9, 10 mM MgCl2, 50 mM KCl, 1 mM DTT, 5% glycerol, 0.05% NP-40, 1 mM ADP, 16mM KF, 2mM BeCl2 and 0.05 unit/μl RNase H (New England Biolabs). The SWI/SNF-NCP complex assembled in the presence of ATPγS was performed essentially as described above with 1mM (2mM in the first assembly buffer) ATPγS replacing ADP-BeFx in the buffers.

Electron microscopy.The assembled SWI/SNF-NCP complex was first crosslinked using 0.05% glutaraldehyde under very low illumination conditions on ice for 5 min before applied onto EM grids. Negative staining sample preparation and data collection were performed as previously described35. For cryo sample preparation, crosslinked complex (~3.3 μl) was applied onto a 400 mesh Quantifoil grid containing 3.5 μm holes and 1 μm spacing (Quantifoil 3.5/1, Electron Microscopy Sciences). A thin carbon film was floated onto the grid before it was plasma cleaned for 10s at 5 W power using a Solarus plasma cleaner (Gatan) equipped with air immediately before sample deposition. The sample was allowed to absorb to the grid for 10 min at 4℃ and 100% humidity in a Vitrobot (FEI) under low illumination conditions, before blotted for 4 s at 10 force and plunge-frozen in liquid ethane. The frozen grids were stored in liquid nitrogen until imaging.Cryo-EM data collection was performed using a JEOL 3200FS transmission electron microscope (JEOL) equipped with a K2 Summit direct electron detector (Gatan) operating at 200kV (Supplementary Table 3). Data were collected using the K2 camera in counting mode at a nominal magnification of 30,000 × (1.12 Å per pixel). Movie series with defocus values ranging from −1.5 to −4.5 μm were collected using Leginon37. 40-frame exposures were taken at 0.3 s per frame (12 s total), using a dose rate of 8 e− per pixel per second, corresponding to a total dose of 76.5 e− Å−2 per movie series. Four datasets with a total number of 7,769 movies on the ADP-BeFx sample and four other datasets with a total number of 6,903 movies on the ATPγS sample were collected.

Electron microscopy.

The assembled SWI/SNF-NCP complex was first crosslinked using 0.05% glutaraldehyde under very low illumination conditions on ice for 5 min before applied onto EM grids. Negative staining sample preparation and data collection were performed as previously described35. For cryo sample preparation, crosslinked complex (~3.3 μl) was applied onto a 400 mesh Quantifoil grid containing 3.5 μm holes and 1 μm spacing (Quantifoil 3.5/1, Electron Microscopy Sciences). A thin carbon film was floated onto the grid before it was plasma cleaned for 10s at 5 W power using a Solarus plasma cleaner (Gatan) equipped with air immediately before sample deposition. The sample was allowed to absorb to the grid for 10 min at 4℃ and 100% humidity in a Vitrobot (FEI) under low illumination conditions, before blotted for 4 s at 10 force and plunge-frozen in liquid ethane. The frozen grids were stored in liquid nitrogen until imaging.

Cryo-EM data collection was performed using a JEOL 3200FS transmission electron microscope (JEOL) equipped with a K2 Summit direct electron detector (Gatan) operating at 200kV (Supplementary Table 3). Data were collected using the K2 camera in counting mode at a nominal magnification of 30,000 × (1.12 Å per pixel). Movie series with defocus values ranging from −1.5 to −4.5 μm were collected using Leginon37. 40-frame exposures were taken at 0.3 s per frame (12 s total), using a dose rate of 8 e− per pixel per second, corresponding to a total dose of 76.5 e− Å−2 per movie series. Four datasets with a total number of 7,769 movies on the ADP-BeFx sample and four other datasets with a total number of 6,903 movies on the ATPγS sample were collected.

Image processing and three-dimensional reconstruction.Negative stain data pre-processing was performed using the Appion processing environment38. Particles were automatically selected from the micrographs using a difference of Gaussians (DoG) particle picker39. The contract transfer function (CTF) of each micrograph was estimated using CTFFind440, the phases were flipped using CTFFind4, and particle stacks were extracted using a box size of 128 × 128 pixels. Two-dimensional classification was conducted using iterative multivariate statistical analysis and multi-reference alignment analysis (MSA-MRA) within the IMAGIC software41. Three-dimensional (3D) reconstruction of negative stained data was performed using an iterative multi-reference projection-matching approach containing libraries from the EMAN2 software package42. The initial 3D model was generated using cryoSPARC43.Cryo-EM data was pre-processed as follows. Movie frames were aligned using MotionCor244 to correct for specimen motion. Particles were automatically selected from the aligned and dose-weighted micrographs using Gautomatch (developed by Zhang K, MRC Laboratory of Molecular Biology, Cambridge, UK) with 2-fold binnning (corresponding to 2.24Å/pixel). The CTF of each micrograph and of each particle was estimated using Gctf45. All three-dimensional (3D) classification and refinement steps together with postprocess and local resolution estimation were performed within RELION 3.046.For the ADP-BeFx dataset, 891,573 particles were automatically picked and were subjected to an initial round of 3D classification with alignment using the density obtained from negative staining as the initial reference (Extended Data Fig. 2). The “Angular sampling interval”, “Offset search range (pix)” and “Offset search step (pix)” were set to 15 degrees, 10 and 2, respectively, for the first 50 iterations. Next, these values were set back to default (7.5 degrees, 5, 1) and the 3D classification was continued until convergence. This resulted in class 3 with 198,543 particles showing sharp structural features of SWI/SNF and nucleosome. This class was subsequently refined and further classified without alignment into 5 classes with a mask around the Arp module and the nucleosome (Extended Data Fig. 2b). Class 1 with 35,214 particles from this second round of classification showed best features of the nucleosome and was chosen to proceed with 3D auto-refinement, which yielded a structure of SWI/SNF-NCP at an overall resolution of 8.96Å (Extended Data Fig. 2c). All resolutions reported herein correspond to the gold-standard Fourier shell correlation (FSC) using the 0.143 criterion47. The ATPγS dataset with 820,117 particles was processed in a similar manner (Extended Data Fig. 3), resulting in a structure with an overall resolution of 10Å.To focus on the body of SWI/SNF, we combined the particles from both samples after the first round of 3D classification with a total number of 390,573 particles (Extended Data Fig. 4). Next, signal subtraction on the nucleosome and the lower half of the Arp module was performed as previously described48, leaving the SWI/SNF body and the top half of the Arp module intact. Subsequently, a 3D classification was performed with only local alignment turned on. This resulted in class 5 with 61,518 particles showing the best structural features of the body (Extended Data Fig. 4b). Next, we unbinned and refined the original particle stack of this class, and generated masks around the body, the Arp module plus the ATPase density of Snf2, and the nucleosome (Extended Data Fig. 4b). 3D multi-body refinement49 was then performed on this class, which drastically improved the resolution of the body to 4.7Å (Extended Data Fig. 4c). However, the Arp module plus the ATPase of Snf2 and the nucleosome densities resulted in worse resolution. We speculate this could be due to the flexible nature of this complex and that the continuous carbon we used for sample preparation might hinder the signal-to-noise ratio that these regions were resolved at a lower resolution. The subtle conformational differences between the complexes binding to the two ATP analogs might also contribute to the lower resolution of these regions. The core region of the body has a resolution close to 4.3Å (Extended Data Fig. 4b), showing densities of bulky sidechains, which enabled us to partially build the structural model of the body (Fig. 1b). This body map replaced its corresponding region in the ADP-BeFx map to result in the composite map shown in Fig. 1a.

Image processing and three-dimensional reconstruction.

Negative stain data pre-processing was performed using the Appion processing environment38. Particles were automatically selected from the micrographs using a difference of Gaussians (DoG) particle picker39. The contract transfer function (CTF) of each micrograph was estimated using CTFFind440, the phases were flipped using CTFFind4, and particle stacks were extracted using a box size of 128 × 128 pixels. Two-dimensional classification was conducted using iterative multivariate statistical analysis and multi-reference alignment analysis (MSA-MRA) within the IMAGIC software41. Three-dimensional (3D) reconstruction of negative stained data was performed using an iterative multi-reference projection-matching approach containing libraries from the EMAN2 software package42. The initial 3D model was generated using cryoSPARC43.

Cryo-EM data was pre-processed as follows. Movie frames were aligned using MotionCor244 to correct for specimen motion. Particles were automatically selected from the aligned and dose-weighted micrographs using Gautomatch (developed by Zhang K, MRC Laboratory of Molecular Biology, Cambridge, UK) with 2-fold binnning (corresponding to 2.24Å/pixel). The CTF of each micrograph and of each particle was estimated using Gctf45. All three-dimensional (3D) classification and refinement steps together with postprocess and local resolution estimation were performed within RELION 3.046.

For the ADP-BeFx dataset, 891,573 particles were automatically picked and were subjected to an initial round of 3D classification with alignment using the density obtained from negative staining as the initial reference (Extended Data Fig. 2). The “Angular sampling interval”, “Offset search range (pix)” and “Offset search step (pix)” were set to 15 degrees, 10 and 2, respectively, for the first 50 iterations. Next, these values were set back to default (7.5 degrees, 5, 1) and the 3D classification was continued until convergence. This resulted in class 3 with 198,543 particles showing sharp structural features of SWI/SNF and nucleosome. This class was subsequently refined and further classified without alignment into 5 classes with a mask around the Arp module and the nucleosome (Extended Data Fig. 2b). Class 1 with 35,214 particles from this second round of classification showed best features of the nucleosome and was chosen to proceed with 3D auto-refinement, which yielded a structure of SWI/SNF-NCP at an overall resolution of 8.96Å (Extended Data Fig. 2c). All resolutions reported herein correspond to the gold-standard Fourier shell correlation (FSC) using the 0.143 criterion47. The ATPγS dataset with 820,117 particles was processed in a similar manner (Extended Data Fig. 3), resulting in a structure with an overall resolution of 10Å.

To focus on the body of SWI/SNF, we combined the particles from both samples after the first round of 3D classification with a total number of 390,573 particles (Extended Data Fig. 4). Next, signal subtraction on the nucleosome and the lower half of the Arp module was performed as previously described48, leaving the SWI/SNF body and the top half of the Arp module intact. Subsequently, a 3D classification was performed with only local alignment turned on. This resulted in class 5 with 61,518 particles showing the best structural features of the body (Extended Data Fig. 4b). Next, we unbinned and refined the original particle stack of this class, and generated masks around the body, the Arp module plus the ATPase density of Snf2, and the nucleosome (Extended Data Fig. 4b). 3D multi-body refinement49 was then performed on this class, which drastically improved the resolution of the body to 4.7Å (Extended Data Fig. 4c). However, the Arp module plus the ATPase of Snf2 and the nucleosome densities resulted in worse resolution. We speculate this could be due to the flexible nature of this complex and that the continuous carbon we used for sample preparation might hinder the signal-to-noise ratio that these regions were resolved at a lower resolution. The subtle conformational differences between the complexes binding to the two ATP analogs might also contribute to the lower resolution of these regions. The core region of the body has a resolution close to 4.3Å (Extended Data Fig. 4b), showing densities of bulky sidechains, which enabled us to partially build the structural model of the body (Fig. 1b). This body map replaced its corresponding region in the ADP-BeFx map to result in the composite map shown in Fig. 1a.

Model building.To aid in model building, we performed secondary structure prediction of the SWI/SNF subunits using the Genesilico Metaserver50. Sequence alignment of the conserved SWI/SNF subunits were performed using CLC Sequence Viewer 7 (Supplementary Figures 2-6). To build the structural model of the SWI/SNF body, we first performed rigid body docking of known structures into our 4.7Å body map. The rigid-body docking was performed in UCSF Chimera51,52, which yielded good fit of the following structures: the SNF5 Repeat domains (RPTs) and Swi3 SWIRM domains from the human BAF47/BAF155 complex (PDB ID 5GJK)53, the SANT domain of the yeast Swi3 (PDB ID 2YUS), and the SWIB domain of mouse BAF60a (PDB ID 1UHR). The BAF47/BAF155 heterodimer and the Swi3 SANT domain can be docked in the density map at two distinct locations, indicating two copies of these domains. Indeed, chemical crosslinking combined with mass spectrometry has shown at least two copies of Swi3 in the yeast SWI/SNF complex18, confirming our docking experiment. Yeast Snf5 has been annotated with two SNF5 RPT domains in Pfam (http://pfam.xfam.org/protein/P18480). We observed clear density in our map that connects these two RPT domains. Next, we built homology models of these structures using Modeller54 and replaced the docked PDBs in the body density map. Regions with missing or extra connecting density were then manually deleted or built in Coot55 based also on secondary structure predictions of these proteins.The Snf2 Anchor domain was built manually in Coot. First, the Arp7/Arp9/Rtt102/HSA structure (PDB ID 4I6M)56 was rigid body docked into the full map, which helped in registering the HSA helix in the body map. The HSA helix was then manually extended in Coot, with Y586 matching a sidechain density further confirming the register of this helix. Next, the Anchor domain was manually extended from the end of the HSA by following the connected density of the map. Again, secondary structure prediction was also used as a guide when extending the model in Coot. Bulky sidechain density at Y497, Y533 and W554 further confirmed the model.The ARM repeat domain of Swi1 locates in the core region of the body map with the highest local resolution, therefore enabling de novo model building. First, the helix density corresponding to residues 942-955 of Swi1 was chosen to model because it has the highest local resolution and that it contains a few bulky sidechain densities. Next, two ⍺ helices with poly-alanine sequence were generated in Coot, which allowed us to create a bulky residue (lysine, arginine, histidine, methionine, phenylalanine, tyrosine, and tryptophan) pattern along both directions. Subsequently, these patterns were used to search against the sequences of SWI/SNF subunits on the Sequence pattern search server (http://www-archbac.u-psud.fr/genomics/patternSearch.html), and Swi1 942-955 was one of the best hit. Further extension of this helix into connected density also matched the secondary structures of Swi1. Then, the remaining regions of the Swi1 ARM repeat domain were manually built into the density in Coot based on secondary structure prediction as well as bulky sidechain densities wherever possible. The overall architecture of the ARM repeat domain of Swi1 also matches that of an Armadillo repeat containing protein β-catenin57 (Extended Data Fig. 8a), confirming our model of Swi1.The positioning of the SWIB domain of Snf12 aided us in building the remaining of this protein into the density. First, at the Spine tip, where the SWIB was docked, there is β-sheet like density (Extended Data Fig. 6c). This agrees with the secondary structure prediction of Snf12, which shows β-strands right N-terminus of the SWIB domain. Although the resolution of this region is low, we are confident about its identity. Four long helices belonging to the Spine module directly connect to this region, two of which extending into the Snf12 densities. Therefore, we assigned these two helices to Snf12. This agreed well with the secondary structure prediction of Snf12, which shows that Snf12 contains two long helices. Next, we performed protein sequence pattern search based on the bulky sidechain densities. Based on the search results, we manually built the two helices of Snf12 in Coot.Based on secondary structure prediction, we reasoned that the other two long helices belong to Swi3 C-terminus. This is backed up by the finding that the C-terminus of Swi3 contains a coiled-coil leucine zipper motif21 and there are two copies of Swi3 in SWI/SNF. To facilitate the registering of the sequence in these long helices, we fitted the crystal structure of human OmoMYC homodimer (PDB ID 5I4Z)58 into the density and obtained a good fit (Extended Data Fig. 6a). Based on this fitting, we mapped the hydrophobic residues from Swi3 as indicated before21 and manually built the two helices in Coot. The rest of Swi3 density in the Spine and Hinge cannot be confidently modeled due to lower resolution and missing density, therefore are modeled with poly-alanine.Snf6 was also manually built in Coot based on secondary structure prediction, bulky sidechain density and prior knowledge based on chemical crosslinking and mass spectrometry data18 and site-directed DNA crosslinking experiments28. We cannot confidently model in Swp82, however we were able to assign densities to this yeast specific subunit based on crosslinking experiments18 and mapping by deletions and EM59. The N-terminal region of Swp82 forms a RSC7 homology domain, therefore we speculate that it occupies the globular density near the Hinge; C-terminal region of Swp82 crosslinks to regions near Snf5 RPT domains and Swi3 SWIRM domain, therefore it was assigned to the density close to Snf5 and Swi3 in the Arm module. This also agrees with the mapping by deletion analysis59. There are also several unassigned densities on the solvent exposed surface of the complex. We did not identify Taf14 and Snf11 in the map (Supplementary Table 4).The molecular model of SWI/SNF body was then refined using Namdinator60 (Supplementary Table 3). To obtain the model for the full complex, we rigid-body fitted the body, the Arp module (PDB ID 4I6M)56 and the ATPase-nucleosome bound with ADP-BeFx (PDB ID 5Z3V)11 into the map of the full complexes in Coot. Then, the HSA helix was connected manually, and the DNA sequence was modified to match our sequence. The extra DNA was manually extended by 10bp using B form DNA in Coot. The figures were prepared using UCSF Chimera and ChimeraX61. Cα-Cα distances from crosslinked lysine pairs18 were measured in UCSF Chimera. 30 Å was chosen as the cutoff62. For crosslinks involving the two molecules of Swi3, we picked the combination that gave the shortest distance as the measurement (Supplementary Table 1).

Model building.

To aid in model building, we performed secondary structure prediction of the SWI/SNF subunits using the Genesilico Metaserver50. Sequence alignment of the conserved SWI/SNF subunits were performed using CLC Sequence Viewer 7 (Supplementary Figures 2-6). To build the structural model of the SWI/SNF body, we first performed rigid body docking of known structures into our 4.7Å body map. The rigid-body docking was performed in UCSF Chimera51,52, which yielded good fit of the following structures: the SNF5 Repeat domains (RPTs) and Swi3 SWIRM domains from the human BAF47/BAF155 complex (PDB ID 5GJK)53, the SANT domain of the yeast Swi3 (PDB ID 2YUS), and the SWIB domain of mouse BAF60a (PDB ID 1UHR). The BAF47/BAF155 heterodimer and the Swi3 SANT domain can be docked in the density map at two distinct locations, indicating two copies of these domains. Indeed, chemical crosslinking combined with mass spectrometry has shown at least two copies of Swi3 in the yeast SWI/SNF complex18, confirming our docking experiment. Yeast Snf5 has been annotated with two SNF5 RPT domains in Pfam (http://pfam.xfam.org/protein/P18480). We observed clear density in our map that connects these two RPT domains. Next, we built homology models of these structures using Modeller54 and replaced the docked PDBs in the body density map. Regions with missing or extra connecting density were then manually deleted or built in Coot55 based also on secondary structure predictions of these proteins.

The Snf2 Anchor domain was built manually in Coot. First, the Arp7/Arp9/Rtt102/HSA structure (PDB ID 4I6M)56 was rigid body docked into the full map, which helped in registering the HSA helix in the body map. The HSA helix was then manually extended in Coot, with Y586 matching a sidechain density further confirming the register of this helix. Next, the Anchor domain was manually extended from the end of the HSA by following the connected density of the map. Again, secondary structure prediction was also used as a guide when extending the model in Coot. Bulky sidechain density at Y497, Y533 and W554 further confirmed the model.

The ARM repeat domain of Swi1 locates in the core region of the body map with the highest local resolution, therefore enabling de novo model building. First, the helix density corresponding to residues 942-955 of Swi1 was chosen to model because it has the highest local resolution and that it contains a few bulky sidechain densities. Next, two ⍺ helices with poly-alanine sequence were generated in Coot, which allowed us to create a bulky residue (lysine, arginine, histidine, methionine, phenylalanine, tyrosine, and tryptophan) pattern along both directions. Subsequently, these patterns were used to search against the sequences of SWI/SNF subunits on the Sequence pattern search server (http://www-archbac.u-psud.fr/genomics/patternSearch.html), and Swi1 942-955 was one of the best hit. Further extension of this helix into connected density also matched the secondary structures of Swi1. Then, the remaining regions of the Swi1 ARM repeat domain were manually built into the density in Coot based on secondary structure prediction as well as bulky sidechain densities wherever possible. The overall architecture of the ARM repeat domain of Swi1 also matches that of an Armadillo repeat containing protein β-catenin57 (Extended Data Fig. 8a), confirming our model of Swi1.

The positioning of the SWIB domain of Snf12 aided us in building the remaining of this protein into the density. First, at the Spine tip, where the SWIB was docked, there is β-sheet like density (Extended Data Fig. 6c). This agrees with the secondary structure prediction of Snf12, which shows β-strands right N-terminus of the SWIB domain. Although the resolution of this region is low, we are confident about its identity. Four long helices belonging to the Spine module directly connect to this region, two of which extending into the Snf12 densities. Therefore, we assigned these two helices to Snf12. This agreed well with the secondary structure prediction of Snf12, which shows that Snf12 contains two long helices. Next, we performed protein sequence pattern search based on the bulky sidechain densities. Based on the search results, we manually built the two helices of Snf12 in Coot.

Based on secondary structure prediction, we reasoned that the other two long helices belong to Swi3 C-terminus. This is backed up by the finding that the C-terminus of Swi3 contains a coiled-coil leucine zipper motif21 and there are two copies of Swi3 in SWI/SNF. To facilitate the registering of the sequence in these long helices, we fitted the crystal structure of human OmoMYC homodimer (PDB ID 5I4Z)58 into the density and obtained a good fit (Extended Data Fig. 6a). Based on this fitting, we mapped the hydrophobic residues from Swi3 as indicated before21 and manually built the two helices in Coot. The rest of Swi3 density in the Spine and Hinge cannot be confidently modeled due to lower resolution and missing density, therefore are modeled with poly-alanine.

Snf6 was also manually built in Coot based on secondary structure prediction, bulky sidechain density and prior knowledge based on chemical crosslinking and mass spectrometry data18 and site-directed DNA crosslinking experiments28. We cannot confidently model in Swp82, however we were able to assign densities to this yeast specific subunit based on crosslinking experiments18 and mapping by deletions and EM59. The N-terminal region of Swp82 forms a RSC7 homology domain, therefore we speculate that it occupies the globular density near the Hinge; C-terminal region of Swp82 crosslinks to regions near Snf5 RPT domains and Swi3 SWIRM domain, therefore it was assigned to the density close to Snf5 and Swi3 in the Arm module. This also agrees with the mapping by deletion analysis59. There are also several unassigned densities on the solvent exposed surface of the complex. We did not identify Taf14 and Snf11 in the map (Supplementary Table 4).

The molecular model of SWI/SNF body was then refined using Namdinator60 (Supplementary Table 3). To obtain the model for the full complex, we rigid-body fitted the body, the Arp module (PDB ID 4I6M)56 and the ATPase-nucleosome bound with ADP-BeFx (PDB ID 5Z3V)11 into the map of the full complexes in Coot. Then, the HSA helix was connected manually, and the DNA sequence was modified to match our sequence. The extra DNA was manually extended by 10bp using B form DNA in Coot. The figures were prepared using UCSF Chimera and ChimeraX61. Cα-Cα distances from crosslinked lysine pairs18 were measured in UCSF Chimera. 30 Å was chosen as the cutoff62. For crosslinks involving the two molecules of Swi3, we picked the combination that gave the shortest distance as the measurement (Supplementary Table 1).

Data availabilityCryo-EM density maps have been deposited in the Electron Microscopy Data Bank (EMDB) under accession numbers EMD-20934 (ADP-BeFx), EMD-20935 (ATPγS), EMD-20933 (body). Model coordinates have been deposited in the Protein Data Bank (PDB) under accession numbers 6UXW (ADP-BeFx), 6UXV (body).

Data availability

Cryo-EM density maps have been deposited in the Electron Microscopy Data Bank (EMDB) under accession numbers EMD-20934 (ADP-BeFx), EMD-20935 (ATPγS), EMD-20933 (body). Model coordinates have been deposited in the Protein Data Bank (PDB) under accession numbers 6UXW (ADP-BeFx), 6UXV (body).

Extended DataExtended Data Fig. 1Purification and assembly of the SWI/SNF-nucleosome complex.a, A Coomassie stained protein SDS-PAGE gel (4-12% gradient) showing the TAP-purified SWI/SNF complex from the yeast strain bearing a TAP tag at the C-terminus of Snf2. SWI/SNF subunits are labeled based on molecular weight. At least three purifications were performed with similar gel pattern. b, Schematic showing the assembly and purification protocol of the SWI/SNF-nucleosome complex prior to single particle cryo-EM analysis. SWI/SNF is first incubated with reconstituted nucleosome. The nucleosomal DNA contains a single-stranded overhang that is annealed to a biotinylated RNA molecule. Next, the assembled complex is immobilized onto streptavidin coated magnetic beads. Following washes, the complex is eluted using RNase H digestion. The eluted complex is then crosslinked and deposited onto EM grid for vitrification. c, Domain organization of all subunits that has been built in the model from Fig 1.b. Mammalian homologs are shown in parentheses. Newly built or homology regions are highlighted by red lines with residue numbers, whereas previous structures that were rigid body docked in our map are indicated by black lines. Subunits are colored as in Fig 1. Abbreviations: QLQ, Glutamine-Leucine-Glutamine; HSA, Helicase/SANT-associated; SnAC, Snf2 ATP coupling; AT, AT hook DNA-binding motif; RPT, Snf5 core repeat; SWIRM, a protein domain found in SWI3, RSC8 and MOIRA; SANT, SWI3, ADA2, N-CoR and TFIIIB" DNA-binding; ARID, AT-rich interaction domain; ARM repeat, Armadillo repeat; LH1/2, long helix 1/2; SWIB, SWI complex BAF60b.Extended Data Fig. 2Data processing scheme of the ADP-BeFx sample.a, A representative raw micrograph (out of 7,769 total images) of the SWI/SNF-nucleosome complex assembled in the presence of ADP-BeFx. b, Flow chart of the cryo-EM data processing procedure. The particle stack of class 3 (198,543 particles; blue dashed box) after the first sorting was chosen for the combined processing (Extended Data Fig. 4). c, Fourier Shell Correlation curves of the complex showing a final average resolution of 8.96Å (FSC=0.143).Extended Data Fig. 3Data processing scheme of the ATPγS sample.a, A representative raw micrograph (out of 6,903 total images) of the SWI/SNF-nucleosome complex assembled in the presence of ATPγS. b, Flow chart of the cryo-EM data processing procedure. The particle stack of class 5 (192,029 particles; red dashed box) after the first sorting was chosen for the combined processing (Extended Data Fig. 4). c, Fourier Shell Correlation curves of the complex showing a final average resolution of 10Å (FSC=0.143).Extended Data Fig. 4Data processing scheme of the combined dataset.a, Flow chart of the data processing procedure by combining ADP-BeFx (Extended Data Fig. 2) and ATPγS (Extended Data Fig. 3) datasets. b, Fourier Shell Correlation curves of the body showing a final average resolution of 4.7Å (FSC=0.143). c, model-map FSC of SWI/SNF body.Extended Data Fig. 5Comparing the structures of chromatin remodelers from different families.All but the INO80 complex have their ATPase module binding to SHL2/−2. INO80 engages the nucleosomal DNA at SHL −6 to −7. SWI/SNF is different from the INO80/SWR1 family remodelers in that its Arp module (Arp7/9) is sandwiched between the Body and the ATPase modules. The Snf2 ATPase module is connected through the long HSA domain to the rest of the complex, whereas the ATPases INO80 and Swr1 directly contact the main body of the corresponding complexes. All remodelers are aligned based on histone proteins. The ATPase in each complex is colored red, whereas Arp proteins are colored green. PDB codes of the other chromatin remodelers are shown in parentheses.Extended Data Fig. 6Structural features of the Spine sub-module of the SWI/SNF complex.a, The Swi3 Coiled-coil dimer (left) resembles the structure of the dominant-negative allele of MYC (OmoMYC, shown in blue in the middle panel; PDB ID 5I4Z). The OmoMYC structure was rigid body docked in the Spine density corresponding to the Swi3 Coiled-coil and then compared with the Swi3 Coiled-coil. b, Swi3 forms an asymmetric dimer in the SWI/SNF complex. c, The density at the tip of the Spine shows features of β-sheet and is therefore assigned to Snf12 based on closed proximity to Snf12 SWIB domain and secondary structure prediction.Extended Data Fig. 7Structural features of the Arm sub-module of the SWI/SNF complex.a, The Snf5 RPT1 and Swi3 SWIRMA heterodimer was aligned with the human BAF47/BAF155 crystal structure (PDB ID 5GJK). b, The Snf5 RPT2 and Swi3 SWIRMB heterodimer was aligned with the human BAF47/BAF155 crystal structure (PDB ID 5GJK). c, RPT1/SWIRMA interface shows slight difference with the RPT2/SWIRMB interface. RPT1 and 2 was aligned, resulting in the SWIRM domains slightly shifting from each other. d, Comparing the interfaces between the SWIRM domains and Swi1. The two SWIRM domains was aligned, resulting in Swi1 H4 (yellow; contacting SWIRMB) occupying a similar position as Swi1 H1 (gold) and Snf5 H-N on SWIRMA. In all panels, structural elements related to RPT1/SWIRMA is depicted using darker colors, whereas RPT2/SWIRMB associated structures is shown in lighter colors.Extended Data Fig. 8Structural features of the Core sub-module of the SWI/SNF complex.a, The Swi1 ARM repeat domain is aligned with β-catenin (gray; PDB ID 3BCT). The insertions of the Swi1 ARM repeat domain are depicted in magenta. b, Detailed interaction between the Swi1 ARM repeat domain with the Arm and Hinge sub-modules. c, Detailed interaction between the Swi1 ARM repeat domain with the Spine sub-module. The EM density of Swi1 is also shown in b and c as mesh.Extended Data Fig. 9Interactions between the Snf2 Anchor domain and the rest of the SWI/SNF complex.a, The Snf2 Anchor linker region interacts with the Swi1 ARM repeat domain. b, Snf2 Anchor helices 1 and 2 are sandwiched between the two SANT domains of Swi3 in the Hinge region. The EM density of Snf2 Anchor is shown as mesh.Extended Data Fig. 10Locations of yeast specific subunits and the extranucleosomal DNA.a and b, Snf6 (a) and Swp82 (b) are positioned at peripheral locations within SWI/SNF. Map and structural models are shown with Snf6 and Swp82 highlighted in a and b, respectively. Swp82 is in close proximity to the nucleosomal DNA near SHL −2 (b). c, The extranucleosomal DNA density is close to Snf6 and is indicated as dashed lines. The N-termini of Swi1 and Snf5 are also labeled. The N-terminal regions of Swi1 and Snf5, which are highly flexible and therefore not resolved in the structure, could take trajectories close to the extranucleosomal DNA.

Extended Data

Purification and assembly of the SWI/SNF-nucleosome complex.

a, A Coomassie stained protein SDS-PAGE gel (4-12% gradient) showing the TAP-purified SWI/SNF complex from the yeast strain bearing a TAP tag at the C-terminus of Snf2. SWI/SNF subunits are labeled based on molecular weight. At least three purifications were performed with similar gel pattern. b, Schematic showing the assembly and purification protocol of the SWI/SNF-nucleosome complex prior to single particle cryo-EM analysis. SWI/SNF is first incubated with reconstituted nucleosome. The nucleosomal DNA contains a single-stranded overhang that is annealed to a biotinylated RNA molecule. Next, the assembled complex is immobilized onto streptavidin coated magnetic beads. Following washes, the complex is eluted using RNase H digestion. The eluted complex is then crosslinked and deposited onto EM grid for vitrification. c, Domain organization of all subunits that has been built in the model from Fig 1.b. Mammalian homologs are shown in parentheses. Newly built or homology regions are highlighted by red lines with residue numbers, whereas previous structures that were rigid body docked in our map are indicated by black lines. Subunits are colored as in Fig 1. Abbreviations: QLQ, Glutamine-Leucine-Glutamine; HSA, Helicase/SANT-associated; SnAC, Snf2 ATP coupling; AT, AT hook DNA-binding motif; RPT, Snf5 core repeat; SWIRM, a protein domain found in SWI3, RSC8 and MOIRA; SANT, SWI3, ADA2, N-CoR and TFIIIB" DNA-binding; ARID, AT-rich interaction domain; ARM repeat, Armadillo repeat; LH1/2, long helix 1/2; SWIB, SWI complex BAF60b.

Data processing scheme of the ADP-BeFx sample.

a, A representative raw micrograph (out of 7,769 total images) of the SWI/SNF-nucleosome complex assembled in the presence of ADP-BeFx. b, Flow chart of the cryo-EM data processing procedure. The particle stack of class 3 (198,543 particles; blue dashed box) after the first sorting was chosen for the combined processing (Extended Data Fig. 4). c, Fourier Shell Correlation curves of the complex showing a final average resolution of 8.96Å (FSC=0.143).

Data processing scheme of the ATPγS sample.

a, A representative raw micrograph (out of 6,903 total images) of the SWI/SNF-nucleosome complex assembled in the presence of ATPγS. b, Flow chart of the cryo-EM data processing procedure. The particle stack of class 5 (192,029 particles; red dashed box) after the first sorting was chosen for the combined processing (Extended Data Fig. 4). c, Fourier Shell Correlation curves of the complex showing a final average resolution of 10Å (FSC=0.143).

Data processing scheme of the combined dataset.

a, Flow chart of the data processing procedure by combining ADP-BeFx (Extended Data Fig. 2) and ATPγS (Extended Data Fig. 3) datasets. b, Fourier Shell Correlation curves of the body showing a final average resolution of 4.7Å (FSC=0.143). c, model-map FSC of SWI/SNF body.

Comparing the structures of chromatin remodelers from different families.

All but the INO80 complex have their ATPase module binding to SHL2/−2. INO80 engages the nucleosomal DNA at SHL −6 to −7. SWI/SNF is different from the INO80/SWR1 family remodelers in that its Arp module (Arp7/9) is sandwiched between the Body and the ATPase modules. The Snf2 ATPase module is connected through the long HSA domain to the rest of the complex, whereas the ATPases INO80 and Swr1 directly contact the main body of the corresponding complexes. All remodelers are aligned based on histone proteins. The ATPase in each complex is colored red, whereas Arp proteins are colored green. PDB codes of the other chromatin remodelers are shown in parentheses.

Structural features of the Spine sub-module of the SWI/SNF complex.

a, The Swi3 Coiled-coil dimer (left) resembles the structure of the dominant-negative allele of MYC (OmoMYC, shown in blue in the middle panel; PDB ID 5I4Z). The OmoMYC structure was rigid body docked in the Spine density corresponding to the Swi3 Coiled-coil and then compared with the Swi3 Coiled-coil. b, Swi3 forms an asymmetric dimer in the SWI/SNF complex. c, The density at the tip of the Spine shows features of β-sheet and is therefore assigned to Snf12 based on closed proximity to Snf12 SWIB domain and secondary structure prediction.

Structural features of the Arm sub-module of the SWI/SNF complex.

a, The Snf5 RPT1 and Swi3 SWIRMA heterodimer was aligned with the human BAF47/BAF155 crystal structure (PDB ID 5GJK). b, The Snf5 RPT2 and Swi3 SWIRMB heterodimer was aligned with the human BAF47/BAF155 crystal structure (PDB ID 5GJK). c, RPT1/SWIRMA interface shows slight difference with the RPT2/SWIRMB interface. RPT1 and 2 was aligned, resulting in the SWIRM domains slightly shifting from each other. d, Comparing the interfaces between the SWIRM domains and Swi1. The two SWIRM domains was aligned, resulting in Swi1 H4 (yellow; contacting SWIRMB) occupying a similar position as Swi1 H1 (gold) and Snf5 H-N on SWIRMA. In all panels, structural elements related to RPT1/SWIRMA is depicted using darker colors, whereas RPT2/SWIRMB associated structures is shown in lighter colors.

Structural features of the Core sub-module of the SWI/SNF complex.

a, The Swi1 ARM repeat domain is aligned with β-catenin (gray; PDB ID 3BCT). The insertions of the Swi1 ARM repeat domain are depicted in magenta. b, Detailed interaction between the Swi1 ARM repeat domain with the Arm and Hinge sub-modules. c, Detailed interaction between the Swi1 ARM repeat domain with the Spine sub-module. The EM density of Swi1 is also shown in b and c as mesh.

Interactions between the Snf2 Anchor domain and the rest of the SWI/SNF complex.

a, The Snf2 Anchor linker region interacts with the Swi1 ARM repeat domain. b, Snf2 Anchor helices 1 and 2 are sandwiched between the two SANT domains of Swi3 in the Hinge region. The EM density of Snf2 Anchor is shown as mesh.

Locations of yeast specific subunits and the extranucleosomal DNA.

a and b, Snf6 (a) and Swp82 (b) are positioned at peripheral locations within SWI/SNF. Map and structural models are shown with Snf6 and Swp82 highlighted in a and b, respectively. Swp82 is in close proximity to the nucleosomal DNA near SHL −2 (b). c, The extranucleosomal DNA density is close to Snf6 and is indicated as dashed lines. The N-termini of Swi1 and Snf5 are also labeled. The N-terminal regions of Swi1 and Snf5, which are highly flexible and therefore not resolved in the structure, could take trajectories close to the extranucleosomal DNA.

Supplementary Material12Sup_Vid1

Supplementary Material
