---
pmid: '30626964'
title: SETD3 is an actin histidine methyltransferase that prevents primary dystocia.
authors:
- Wilkinson AW
- Diep J
- Dai S
- Liu S
- Ooi YS
- Song D
- Li TM
- Horton JR
- Zhang X
- Liu C
- Trivedi DV
- Ruppel KM
- Vilches-Moure JG
- Casey KM
- Mak J
- Cowan T
- Elias JE
- Nagamine CM
- Spudich JA
- Cheng X
- Carette JE
- Gozani O
journal: Nature
year: '2019'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6511263
doi: 10.1038/s41586-018-0821-8
---

# SETD3 is an actin histidine methyltransferase that prevents primary dystocia.
**Authors:** Wilkinson AW, Diep J, Dai S, Liu S, Ooi YS, Song D, Li TM, Horton JR, Zhang X, Liu C, Trivedi DV, Ruppel KM, Vilches-Moure JG, Casey KM, Mak J, Cowan T, Elias JE, Nagamine CM, Spudich JA, Cheng X, Carette JE, Gozani O
**Journal:** Nature (2019)
**DOI:** [10.1038/s41586-018-0821-8](https://doi.org/10.1038/s41586-018-0821-8)
**PMC:** [PMC6511263](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6511263/)

## Abstract

1. Nature. 2019 Jan;565(7739):372-376. doi: 10.1038/s41586-018-0821-8. Epub 2018 
Dec 10.

SETD3 is an actin histidine methyltransferase that prevents primary dystocia.

Wilkinson AW(#)(1), Diep J(#)(2), Dai S(#)(3), Liu S(1), Ooi YS(2), Song D(4), 
Li TM(1), Horton JR(3), Zhang X(3), Liu C(4), Trivedi DV(4), Ruppel KM(4), 
Vilches-Moure JG(5), Casey KM(5), Mak J(6), Cowan T(7), Elias JE(8), Nagamine 
CM(5), Spudich JA(4), Cheng X(#)(9), Carette JE(#)(10), Gozani O(#)(11).

Author information:
(1)Department of Biology, Stanford University, Stanford, CA, USA.
(2)Department of Microbiology and Immunology, Stanford University School of 
Medicine, Stanford, CA, USA.
(3)Department of Molecular and Cellular Oncology, The University of Texas MD 
Anderson Cancer Center, Houston, TX, USA.
(4)Department of Biochemistry, Stanford University School of Medicine, Stanford, 
CA, USA.
(5)Department of Comparative Medicine, Stanford University School of Medicine, 
Stanford, CA, USA.
(6)Stanford Healthcare, Palo Alto, CA, USA.
(7)Department of Pathology, Stanford University School of Medicine, Stanford, 
CA, USA.
(8)Department of Chemical and Systems Biology, Stanford University School of 
Medicine, Stanford, CA, USA.
(9)Department of Molecular and Cellular Oncology, The University of Texas MD 
Anderson Cancer Center, Houston, TX, USA. xcheng5@mdanderson.org.
(10)Department of Microbiology and Immunology, Stanford University School of 
Medicine, Stanford, CA, USA. carette@stanford.edu.
(11)Department of Biology, Stanford University, Stanford, CA, USA. 
ogozani@stanford.edu.
(#)Contributed equally

Comment in
    Nature. 2019 Jan;565(7739):297-298. doi: 10.1038/d41586-018-07882-0.

For more than 50 years, the methylation of mammalian actin at histidine 73 has 
been known to occur1. Despite the pervasiveness of His73 methylation, which we 
find is conserved in several model animals and plants, its function remains 
unclear and the enzyme that generates this modification is unknown. Here we 
identify SET domain protein 3 (SETD3) as the physiological actin His73 
methyltransferase. Structural studies reveal that an extensive network of 
interactions clamps the actin peptide onto the surface of SETD3 to orient His73 
correctly within the catalytic pocket and to facilitate methyl transfer. His73 
methylation reduces the nucleotide-exchange rate on actin monomers and modestly 
accelerates the assembly of actin filaments. Mice that lack SETD3 show complete 
loss of actin His73 methylation in several tissues, and quantitative proteomics 
analysis shows that actin His73 methylation is the only detectable physiological 
substrate of SETD3. SETD3-deficient female mice have severely decreased litter 
sizes owing to primary maternal dystocia that is refractory to ecbolic induction 
agents. Furthermore, depletion of SETD3 impairs signal-induced contraction in 
primary human uterine smooth muscle cells. Together, our results identify a 
mammalian histidine methyltransferase and uncover a pivotal role for SETD3 and 
actin His73 methylation in the regulation of smooth muscle contractility. Our 
data also support the broader hypothesis that protein histidine methylation acts 
as a common regulatory mechanism.

DOI: 10.1038/s41586-018-0821-8
PMCID: PMC6511263
PMID: 30626964 [Indexed for MEDLINE]

## Full Text

SETD3, a ubiquitously expressed protein, is epigenetically up-regulated in
response to muscle loading and hypertrophy2. SETD3 contains an N-terminal SET domain and a C-terminal domain
homologous to a Rubisco large subunit
methyltransferase (LSMT) interaction
domain (Extended Data Fig. 1a). Proteins containing
the catalytic lysine methyltransferase SET domain are found in all three kingdoms of
life and regulate diverse biological and pathologic processes3. In humans, ~half of the fifty-five SET
family members methylate lysines on histone and/or non-histone protein
substrates3. Enzymatic
activities for the remainder of human SET proteins, including SETD3, are unclear. SETD3
was reported to methylate histone H3 at lysines 4 and 364. However, this conclusion was based on mass
spectrometry data wherin mass shifts were inconsistent with a bona fide methylation
event (see Figure 2C in4). When we tested SETD3 activity, it did not
methylate histones in contrast to two established protein lysine methyltransferases
(PKMTs), NSD2 and MLL23 (Fig. 1a and Extended Data Fig.
1b). Based on these results and the cytoplasmic localization of SETD3 (Extended Data Fig. 1c), we conclude that SETD3 is not
a histone methyltransferase and likely targets a non-histone substrate.

Among active PKMTs, SETD3 is most similar to human SETD65. We generated a putative SETD3 catalytic mutant
based on this homology (Extended Data Fig. 1a) and
performed in vitro methylation assays with 3H-SAM
(S-adenosyl-methionine) on total cytoplasmic extract with wild-type
or mutant SETD3. A ~42 kDa band was more prominent in reactions with wild-type
SETD3 relative to the mutant (Fig. 1b). A
three-step fractionation protocol was used to partially purify the SETD3-dependent
activity for mass spectrometry analysis (Extended Data
Fig. 1d-g). The ten most abundant hits
in the proteomic dataset with molecular weights close to the candidate substrate (Extended Data Fig. 1h; Supplementary Table 1) were expressed as
recombinant proteins and directly tested as SETD3 substrates in vitro
(Fig. 1c). Of these proteins and other
potential substrates, only beta-actin was methylated by SETD3 (Fig. 1c, Extended Data Fig.
1i). SETD6 and 10 additional active PKMTs do not methylate actin (Fig. 1d-e; Extended Data Fig. 1j). Also, SETD3 does not
methylate the SETD6 substrate RelA (Fig.
1d)6. These data
identify SETD3 as the first example of an enzyme to methylate actin in
vitro.

To identify the specific actin residue/s modified by SETD3, in
vitro methylated actin was analyzed by tandem mass spectrometry (MS/MS);
deuterated SAM was used as the methyl donor to distinguish SETD3-catalyzed methylation
from all other sources. Unexpectedly, no lysine methylation events on actin were
observed (Extended Data Fig. 2). We therefore
extended the analysis to search for methylation events at other residues (arginine,
glutamine, cysteine and histidine) and unambiguously identified incorporation of a
single deuterated methyl moiety at histidine 73 of actin (Extended Data Fig. 3a-c). Histidine
methylation is known to exist on a limited number of proteins. The only known histidine
methyltransferase is the yeast enzyme Hpm1p, which methylates Rpl3 (not actin) and bears
no structural resemblance to SETD37,8. However, methylation of actin at
histidine 73 (actin-H73me) is a canonical modification present in mammals but not in
yeast1,7,9. We identified H73me (or
equivalent) on actin purified from ten different model organisms ranging from worms to
plants to humans, which all have a SETD3 homologue, but not in S.
cerevisiae, which lacks SETD3 (Extended Data
Fig. 4). Thus, actin-H73me is an evolutionarily conserved modification
present in a broad range of multicellular eukaryotic organisms.

Mutation of actin H73 to alanine (H73A) prevents methylation by SETD3 (Fig. 2a). Histidine can potentially be methylated on
the nitrogen in position 1 (π, N1) or 3 (τ,
N3) of the imidazole ring (Fig.
2b), with His(3-me) the physiologic modification found on endogenous actin.
Methylation assays performed on actin peptides spanning H73 (residues 66-80 of
beta-actin) and in which H73 was either unmethylated or methylated at the
N1 or N3 position (Extended Data Fig. 3d) showed that SETD3 methylated the unmodified peptide
(kcat 0.4 min−1) and the peptide
harboring H73(1-me) (with a much slower rate), but not H73(3-me), indicating that SETD3
catalyzes methylation of H73 at the physiologically relevant N3
position (Fig. 2c; Extended Data Fig. 3e-g).

To gain insight into the molecular basis of how SETD3 methylates a histidine
substrate, we co-crystallized SETD3 with an actin peptide (residues 66-80) containing
H73 in the presence of S-adenosyl-l-homocysteine (SAH). We
obtained three highly similar crystal forms, focusing on the highest resolution
structure of 1.69 Å (Extended Data Table
1). SETD3 contains two lobes (SET and LSMT-like) forming a V-shaped cleft (Extended Data Fig. 5a). The 15-residue actin-peptide
occupies an extended surface groove of the SET-domain (Fig. 2d), with H73 inserted into the active-site channel positioned to meet
SAH from the other end (Fig. 2e). From the
viewpoint of the cofactor-binding pocket, the methyl-accepting H73 N3
atom is seen at the bottom of the channel (Extended Data
Fig. 5b). Structural superimposition comparing SETD3 and SETD65 reveals that for SETD3, the methyl donor
group of SAM points toward and is within the distance of 1.8 Å of the target
N3 atom of the imidazole ring (Fig.
2f). An intra-molecular hydrogen bond between the main-chain carbonyl oxygen
of E72 and the main-chain amide nitrogen of I75 bends the peptide substrate in a sharp
turn at H73. This configuration allows H73 and G74 (the smallest residue) to enter deep
into the channel and positions the target N3 atom in line with the
methyl group and sulfur atom of SAM (Fig. 2f; Extended Data Fig. 5c-d); a linear arrangement comprising the nucleophile, the methyl group and
the leaving thioester group in the transition state is required for the SN2
reaction mechanism used by SAM-dependent methyltransferases10.

All 15 residues of the actin peptide are bound in the structure (Fig. 2d; Extended Data Fig.
5d), with the SETD3-actin peptide interface consisting of an extensive
network of charged/polar and hydrophobic interactions involving both inter- and
intra-molecular contacts (Extended Data Fig.
5d-n). For example, W79 and I71 of
actin each fit into hydrophobic pockets on the SETD3 surface (Fig. 2g). This long and ordered peptide-substrate recognition
groove for SETD3 is unique compared to other SET domains11, suggesting that SETD3 would inefficiently
accommodate substrates diverging from actin. Structure-guided point mutations in actin
that disrupt the hydrophobic interaction (I71A) or are predicted to interfere with
proper positioning of H73 in the catalytic pocket (G74I) impair the ability of SETD3 to
methylate actin-H73, and the double mutant (I71A/G74I) abolishes activity (Fig. 3a). Further, while a W79E substitution is
tolerated (Extended Data Fig. 5e), SETD3 has no
activity on actin I71A/W79E double mutations (Fig.
3a). These data highlight the sensitivity of SETD3 activity on actin to
changes in the sequence spanning H73. Indeed, the fifteen-residue sequence spanning H73
is highly conserved amongst the six human actin isoforms9,12 with
the only difference being an I/V variation at position 76 that does not impact SETD3
recognition (Extended Data Fig. 5g,o). Accordingly, all actin isoforms are methylated by SETD3
in vitro (Fig. 3b). Finally,
the putative SETD3 catalytic mutant is a conserved tyrosine (Y312) that participates in
forming the active-site pocket (Extended Data Fig.
5d,i,p). This mutant has residual activity; however, mutation of additional
conserved residues important for SAM binding abolished SETD3 activity (Fig. 3c; Extended Data Fig.
5p).

We next raised a state-specific actin-H73me antibody that selectively recognized
H73(3-me) peptides (Extended Data Fig. 6a). The
majority of endogenous actin is marked by H73 methylation in HT1080 cancer cells and the
depletion of endogenous SETD3 by two independent single guide RNAs (sgRNAs) results in
almost complete loss of this modification as determined by Western blotting and mass
spectrometry (Fig. 3d; Extended Data Fig. 6b). We also observed complete loss of
actin-H73me in a clonal HeLa cell line in which the SETD3 alleles were disrupted (Fig. 3e; Extended Data
Fig. 6c). Further, complementation of SETD3-depleted HT1080 cells with
CRISPR-resistant wild-type, but not catalytically-dead SETD3NHY,
reconstituted actin methylation at H73 (Fig. 3f;
Extended Data Fig. 6d). To assess the
physiologic role of SETD3 in regulating actin H73 methylation, we obtained
Setd3+/− mice from the Canadian Mouse Mutant
Repository13 and generated
Setd3 null homozygotes
(Setd3−/−), which are viable. Analysis of
actin purified from nine different tissue types isolated from
Setd3+/+ and
Setd3−/− mice showed that in tissues
expressing SETD3, the majority of actin is methylated, however, this modification is not
detected in any tissue from SETD3 deficient mice (Fig.
3g; Extended Data Fig. 7a-d). Finally, quantitative profiling of serum amino
acids identified depletion of 3-methyl histidine as the lone abnormality in
Setd3−/− mice (Extended Data Fig. 7e). Collectively, these results identify
SETD3 as the physiologic actin-H73 methyltransferase.

Actin is one of the more abundant proteins in the human proteome and H73me is a
high stoichiometry event. Our data argue that SETD3 is the principal enzyme tasked with
generating actin-H73me, however is actin-H73 the only SETD3 substrate or one of many
substrates? There are few known histidine methylated proteins besides actin – one
is myosin, which SETD3 does not methylate (Fig.
3h). In addition, SETD3 has no activity on histones and several other proteins
(see Fig. 1c-d
and Extended Data Fig. 1i). To investigate SETD3
catalytic specificity in a physiologic and unbiased setting, we used quantitative
proteomics to compare the methylome of HeLa cells ±SETD3 (see Fig. 3e). Of the >900 methylation events detected in
the analysis, including ~180 histidine methylated peptides (Supplementary Table 2), actin-H73me was the
only modification quantitatively altered upon SETD3 depletion (Fig. 3i). Together, these results argue that the principal
physiologic enzymatic function of SETD3 is actin-H73 methylation.

Based on elegant studies leveraging actin mutants, H73 and its methylation are
suggested to influence actin polymerization dynamics, in part due to its proximity to
the ATP binding site9,12,14.
However, a direct side-by-side comparison of the properties of actin ±methylation
has not previously been possible. We purified actin ±H73me (Extended Data Fig 8a-b)
from HeLa cells (see Fig. 3e) and found that
methylation promoted actin polymerization kinetics in vitro but did not
affect depolymerization (Fig. 4 a; Extended Data Fig. 8c). Moreover, the rate of ATP nucleotide
exchange was faster with the unmethylated species (Fig.
4b). We note that actin-H73me exhibited a subtle increase in polymerization
when extending off of phalloidin-actin seeds, and made little difference on
polymerization rates in response to Arp2/3 stimulation (Extended Data Fig. 8d-e). To explore
the role of H73 methylation during actin polymerization in cells, mouse embryonic
fibroblasts (MEFs) were isolated from Setd3+/− mice
positive for actin-H73me and Setd3−/− mice
lacking methylation (Fig. 4c). Cells containing
actin-H73me were modestly more efficient at migration than cells deficient for
methylated actin (Fig. 4d; Extended Data Fig. 8f). Together, these data are consistent
with a model where SETD3 methylation of actin positively regulates actin polymerization
in cells.

While Setd3−/− MEFs show a defect in
cell migration, Setd3−/− mice are viable.
Thus, actin-H73me must play a specialized physiologic role in vivo that
is not essential for basal actin functions. The characterization of SETD3 knockouts by
the International Mouse Phenotypic Consortium scored several phenotypes, including
moderate skeletal muscle myopathy, abnormal cardiac EKG, and mild decreased lean
mass13. We did not observe
gross histopathologic changes in muscle tissue from young adult
Setd3−/− mice (Extended Data Fig. 9a-b). While we have not performed a comprehensive phenotypic analysis, we have
observed that the litter size of female
Setd3−/− mice, regardless of the genotype
of the sire, are significantly smaller than litters from
Setd3+/+ and
Setd3+/− females (Extended Data Fig. 9c). To explore this observation further, we analyzed
parturition in Setd3−/− or littermate control
females all sired by wild-type males. Normal parturition was defined as birth at 19 days
post-coitum (dpc) and dystocia (delayed parturition) was defined as either no births
(≥20 dpc) or incomplete delivery with fetuses remaining in utero
(≥20 dpc). Dystocia, a rare phenotype, was observed in 8/9
Setd3−/− mice, whereas normal parturition
was observed in all of the control mice (Fig. 5a).
As expected, in control mice, no fetuses were detected in utero at 20
dpc (Fig. 5b). In contrast,
Setd3−/− females typically had many
fetuses remaining in utero at ≥20 dpc (Fig. 5b-c).

The dystocia observed in the Setd3−/−
females is independent of the genotype of the fetus as (1) wild-type males were used as
sires, (2) pups successfully born to Setd3−/−
mice were viable, and (3) Setd3+/− females had normal
parturition. Thus, the cause of dystocia is of maternal origin. Secondary maternal
dystocia is unlikely as there were no obvious and consistent anatomical pelvic
abnormalities present in Setd3−/− females, and
an anatomic defect would not explain the presence of intact fetuses in
utero. Thus, the dystocia is primary, either due to abnormal myometrium
contraction or a defect in hormone signaling. To distinguish between these
possibilities, early labor was induced in wild-type and
Setd3−/− females with oxytocin
administration at 18 dpc (see schematic Fig. 5d).
Wild-type mice went into early labor and all pups were born within 24 hours of
treatment, whereas the dystocia of Setd3−/−
mice was refractory to oxytocin (Fig. 5e-f). Similar results were observed with prostaglandin
treatment as an independent ecbolic agent15 (Extended Data Fig. 9d-e). These data identify a specific requirement for
SETD3 in the contraction mechanism of the laboring uterus, a mechanism that
fundamentally relies on proper actin behavior16.

As multiple, unique point mutations in actin are implicated in the etiology of
several human smooth muscle myopathies17,18, we postulated a link
between actin-H73me and uterine cell contraction in the primary dystocia of
Setd3−/− females. To test this idea, we
depleted SETD3 and actin-H73me from primary human myometrial cells (Fig. 5g) and measured cellular contraction under normal
conditions and in response to two uterine smooth muscle contraction stimulants. While
depletion of SETD3 had only a marginal impact on intrinsic contraction, the absence of
SETD3 significantly impeded signal-induced contraction by oxytocin and Endothelin-1
(Fig. 5h-i). Finally, the loss of Endothelin-1-induced cellular contraction upon SETD3
knockdown was rescued in the primary human myometrial cells by complementation with
CRISPR-resistant wild-type SETD3, but not by catalytically inactive SETD3NHY
(Fig. 5j-k; Extended Data Fig. 9f). These data
suggest a specific role for actin-H73me in signal-induced contraction of smooth
muscle.

Here, we have identified SETD3 as the first known metazoan protein histidine
methyltransferase and the enzyme responsible for generating the abundant and enigmatic
actin-H73me. SETD3 represents the first SET domain family member to methylate a residue
besides lysine, broadening the current framework of possible chemistries catalyzed by
SET domains. Our analysis revealed histidine methylation – previously detected on
only a handful of proteins – to be surprisingly common, observed with a frequency
similar to that of protein lysine di- and tri-methylation19. Actin mutations are implicated in smooth muscle
myopathies (e.g. 17,18) including a case of a pregnant patient
suffering from uterine muscle atony20.
We postulate that SETD3 deficiency and loss of H73me presents like intrinsic actin
mutations by altering actin responsiveness to polymerization cues, which leads to
defects in smooth muscle contraction. H73me impact on actin-myosin interactions is
marginal (Extended Data Fig. 9g-h), suggesting smooth muscle contraction regulation is
mediated by a different mechanism. Specifically, besides the canonical actin-myosin
pathway, there is a distinct role for dynamic actin polymerization in mediating
mechanical force generation of smooth muscle21. In summary, our work identifies a functional role for
mammalian histidine methylation in regulating actin and smooth muscle contraction and
suggests a broad role for this modification in the regulation of mammalian
proteomes.

MethodsReagents.Biotinylated peptides were synthesized at the UNC High-Throughput
Peptide Synthesis and Array Facility (Chapel Hill, NC)22. Fmoc-L-histidine amino acids with
either (1-me) or (3-me) modifications (Sigma-Aldrich) were used in the synthesis
of these peptides and synthesized peptides were purified by HPLC (>92%
purity). Antibodies used in this study: SETD3 (Abcam), pan-actin (Cytoskeleton),
beta-tubulin (Millipore), H3K4me2 (Cell Signaling), H3K36me2 (Cell Signaling),
H73(3-me) see below, GFP (Invitrogen), Alexa488 anti-rabbit antibody (Life
Technologies), anti-rabbit and anti-mouse peroxidase conjugated antibodies
(Jackson Immunoresearch). Western blots were visualized by chemiluminescence (GE
Healthcare).Anti-H73(3-me) specific antibody generation.Peptide corresponding to beta-actin amino acids 68-78 (KYPIEHGIVTN) with
H(3-me) installed at position 73 was synthesized and HPLC purified (99% purity).
This peptide was conjugated to KLH and used as antigen to immunize rabbits.
Rabbit protocols, peptide conjugation, immunization, and antiserum production
were performed by Pocono Rabbit Farm & Laboratory (Canadensis, PA).
Antiserum was negatively selected against unmodified actin peptide (amino acids
66-80). Final purification with was performed with immobilized antigenic peptide
to select for H73(3-me) specific antibodies.Recombinant protein expression for in vitro
reactions.SETD3 (Q86TU7) and actin were cloned into pGEX-6P-1. MLL2 catalytic
complex contained GST-tagged catalytic SET domain of MLL2, and full-length,
6xHis-tagged ASH2L, WDR5, DPY30, and RBBP5 that were all individually expressed
in bacteria and assembled in stoichiometric proportions to result in a catalytic
complex23. Lysine
methyltransferases used as controls were reacted with previously reported
substrates24-35. Candidate substrate cDNAs were
subcloned from the human ORFeome (Open Biosystems) using an LR recombination
reaction (Life Technologies) into pDEST15. Mutations of all proteins were
generated using Phusion site-directed mutagenesis (Thermo Scientific) according
to manufacturer’s recommendations. All proteins were expressed as
N-terminal GST fusions (except for non-catalytic MLL complex components) in BL21
or BL21(DE3) codon plus bacteria. Expression was performed at 16°C
overnight in the presence of 0.1 mM IPTG. Recombinant GST fusions were purified
with Glutathione Sepharose 4B (GE Healthcare) and eluted in 100 mM Tris pH 8.0,
10 mg/ml reduced glutathione (Sigma-Aldrich). Any cleavage of GST tags was
performed with Prescission protease (GE Healthcare) overnight at 4°C.
Recombinant 6xHis-tagged proteins were purified with Ni-NTA agarose (Life
technologies) and eluted in 100 mM Tris pH 8.0, 250 mM imidazole. Kinetics
experiments and crystallography utilized additional purification described
below.Purification of SETD3 for crystallography.SETD3 was expressed as a GST-fusion as described above. Cleared
bacterial lysate containing soluble glutathione S-transferase (GST) fusion SETD3
was first loaded on a self-packed glutathione-sepharose column and eluted with
buffer containing 100 mM Tris (pH 8.0), 150 mM NaCl, 5% glycerol, 0.5 mM TCEP
and 20 mM reduced glutathione. The GST tag was removed by ~100 μg
prescission protease (purified in-house) at 4°C overnight. The cleaved
protein solution was diluted to ~50 mM NaCl and further purified by
HiTrap Q-HP, GE Healthcare) with a linear gradient of 50 mM to 500 mM NaCl;
SETD3 eluted together with the cleaved GST around 200-300 mM NaCl. To remove
GST, fractions were loaded onto a fresh glutathione-sepharose column and the
flow-through was collected and concentrated to ~2 ml. SETD3 was then
further purified by size exclusion chromatography (Superdex 200, GE Healthcare)
in 20 mM Tris (pH 8.0), 200 mM NaCl, 5% glycerol, 0.5 mM TCEP. All purification
steps were performed at 4°C. The chromatography was conducted in a
BIO-RAD NGCTM system.Cellular fractionation.Cytoplasmic cellular extracts were prepared by established
protocols36. Briefly,
cells were disrupted by hypotonic lysis (25 mM KCl, 1.5 mM MgCl2, 10
mM HEPES–KOH pH 7.9, protease inhibitors) and dounce homogenization. Size
exclusion chromatography (Superose 6) was performed in hypotonic lysis buffer.
Ion-exchange chromatography (Sepharose Q HP) columns were equilibrated in
hypotonic lysis buffer and proteins were eluted on a linear gradient from 25-600
mM KCl. Fractions were concentrated with centrifugal filter units (Amicon, 3000
Da MWCO) before being used as substrate in in vitro reactions
with radiolabeled SAM. Chromatography for substrate enrichment was performed
with an AKTA FPLC (GE Healthcare)In vitro methylation reactions.In vitro methyltransferase reactions were performed
with 5 μg methyltransferase, 1 μg substrate, and 2 μCi
3H-S-adenosylmethionine (American Radiolabeled
Chemicals) in a buffer containing 50 mM Tris pH 8.0, 20 mM KCl, 5 mM
MgCl2 at 30°C overnight. Reactions were resolved by
SDS-PAGE, and 3H-methylation was visualized by autoradiography and
gels were stained with Coomassie as a loading control. Reactions with deuterated
SAM (CDN Isotopes) or normal SAM (Sigma-Aldrich) were performed with SAM at a
final concentration of 80 μM.Steady-state kinetic measurement.Human beta-actin peptide (residues 66-80; TLKYPIEHGIVTNWD) was used as a
substrate for SETD3. The reaction mixture containing 20 mM Tris (pH 8.0), 50 mM
NaCl, 0.1 mg/ml BSA, 1 mM DTT, and 0.18 μM SETD3. To determine the Km
value for peptide, the concentration of S-adenosyl-L-methionine (SAM) was kept
constant at 40 μM, while for determination of the Km for SAM, peptide
concentration was kept constant at 50 μM. Reactions were carried out at
room temperature for 20 min with a total volume of 20 μl, and terminated
by the addition of trifluoroacetic acid (TFA) to 0.1% (v/v). The activity of
SETD3 was measured using a bioluminescence assay (MTase-Glo™, Promega) in
which the reaction by-product S-adenosyl-homocysteine (SAH) is converted into
ATP in a two-step reaction and can be detected via a luciferase
reaction37. In
general, 5 μl of reaction mixture was transferred to a low-volume
384-well plate and the luminescence assay was performed according to the
manufacture’s protocol. A Synergy 4 Multi-Mode Microplate Reader (BioTek)
was used to measure luminescent signal.Time-course of methylation for three β-actin peptides.Typically, a reaction mixture contained 20 mM Tris (pH 8.0), 50 mM NaCl,
0.1 mg/ml BSA, 1 mM DTT, 40 μM SAM and 50 μM peptide. In addition
to the unmodified peptide, 1-methyl-His and 3-methyl-His peptides were used in
the activity assay. For unmodified peptide, 0.18 μM SETD3 was used, while
for modified ones, 80-fold higher amounts of SETD3 (14.4 μM) were used
due to very low activity on these peptides. For each time point, 10 μl of
reaction mixture was taken and adding TFA to 0.1% stopped the reaction. The
luminescence assay was performed as described above.Mass spectrometry.LC-MS/MS was performed using an Orbitrap Elite or Fusion (Thermo
Scientific) and data was analyzed manually or using MaxQuant software38. Peptide identification
searches with MaxQuant were performed with parameters to identify methionine
oxidation, N-acetylation, monomethyl lysine, dimethyl lysine, trimethyl lysine,
methyl histidine, monomethyl arginine, dimethyl arginine, methyl glutamine, and
methyl cysteine. in vitro reactions were directly resolved by
SDS-PAGE before mass spectrometry analysis was performed as above and digested
with trypsin, chymotrypsin or Glu-C (Promega). Quantitative proteomic analysis
was performed with SILAC, light isotopic labels (K0R0) and heavy isotopic labels
(K8R10). In the SETD3-dependent methylomics, no peptides with a SILAC ratio
greater than 1.5 were deemed to be SETD3-dependent methyl events. No peptides
(other than actin) with at least a 1.5-fold ratio in the reverse direction and
NaN ratio in the forward direction were identified at all in the forward
direction. Actin H73 methylation was detected with light (WT cells, K0R0)
labeling in the forward direction and not with heavy labeling (SETD3 knockout
cells, K8R10). We thus concluded that the lack of a calculable ratio
(light/heavy) in the forward direction is due to the clonal nature of the cell
lines being used. We therefore plotted the forward ratio as infinite (Fig. 3i).Targeted quantification of actin methylation was performed by manual
identification of methylated and unmethylated peptides followed by integration
of area under the curve for chromatographic peaks associated with the
corresponding m/z. Percentage of methylated peptides was
determined as a fraction of the sum total area for methylated and unmethylated
species. Actin was isolated from cellular or tissue sources using DNAse I
affinity chromatography39.
DNAse I was immobilized to NHS-activated Sepharose FF (GE Healthcare) overnight
in 100 mM Hepes pH 7.5, 150 mM NaCl, 80 mM CaCl2. Cells/tissues were
homogenized with buffer containing 1M Tris pH 8.0, 600 mM KCl, 0.5 mM
MgCl2, 4% NP-40, 1% Tween-20, 1 mM DTT, 1 mM ATP, and protease
inhibitors40 (Roche).
Cleared lysates were incubated with immobilized DNase I resin overnight, washed,
boiled in SDS sample buffer, and purified actin was resolved by SDS-PAGE. Gels
were silver-stained (Thermo Scientific) then actin bands were excised and
processed for mass spectrometry by in-gel digest with sequencing grade trypsin.
The following sources were used in the analysis of actin methylation across
evolution: S. cerevisiae, mid-log BY4741; A.
thaliana, leaf tissue; N. benthamiana, leaf
tissue; C. elegans, mixed population; D.
melanogaster, larvae; S. frugiperda, Sf9 cells;
D. rerio, 5 days post fertilization embryos; X.
laevis, egg extract; G. gallus, skeletal muscle;
M. musculus, brain/uterine/skeletal muscle tissue;
H. sapiens, cell lines.Crystallography.Crystallization of the SETD3-SAH-actin peptide (residues 66-80;
Genscript) complex was carried out by the hanging-drop vapor-diffusion method at
20°C after mixing SETD3 (~15 mg/ml) with SAH and peptide at a
molar ratio of 1:4:5, and then mixed with an equal amount of well-solution.
Three different crystal forms were obtained. Crystals with space group
P21 were obtained with 0.2 M ammonium acetate,
0.1 M sodium citrate tribasic dihydrate pH 5.6, and 30% (w/v) polyethylene
glycol 4000. Crystals with space group
P212121 were obtained with
two similar conditions, either 0.2 M ammonium sulfate or 0.2 M sodium chloride,
0.1 M BIS-TRIS pH 5.5, 25% (w/v) polyethylene glycol 3350, and 0.1 M BIS-TRIS pH
5.5, 25% (w/v) polyethylene glycol 3350. The first form, like the
P21 crystal, contained two SETD3-SAH-peptide
complexes per asymmetric unit, while the second contained only one complex.For X-ray diffraction data collection, single crystals were flash frozen
in liquid nitrogen by equilibrating in a cryoprotectant buffer containing well
solution with 25% (v/v) ethylene glycol. All datasets were collected at the
SER-CAT beamline 22ID of Advanced Photon Source at Argonne National Laboratory.
Crystallographic datasets were processed with HKL200041. Molecular replacement was performed
with PHENIX PHASER module42 by
using the structure of human SETD3 in complex with SAM (PDB ID 3SMT) as a search
model. PHENIX REFINE43 was
used for refinement with 5% randomly chosen reflections for validation by the
Rfree value. COOT44 was used
for peptide building and model corrections between refinement rounds. Structure
quality was analyzed during PHENIX refinements and later validated by the PDB
validation server. Molecular graphics were generated using PyMol
(Schrödinger, LLC).We note that the high-resolution data at 1.69 Å (PDB 6MBK) were
the result of merging two datasets collected from two different parts of the
same crystal. This crystal did diffract X-rays to higher resolution but was
susceptible to radiation damage, especially in the higher resolution range. The
merge resulted in 98.4% completeness in the highest resolution shell of
1.75-1.69 Å. The somewhat higher R-work and R-free values of the
structure (PDB 6MBK) might be an indication of radiation damage that could not
be modeled and further refinement would not lower these R values substantially
and the structure and map compared well with the quality of the other two
crystal forms.Actin purification for biochemical assays.Actin was purified from wild-type HeLa cells or a clonally-derived HeLa
cell line in which SETD3 was knocked out. Purification was adapted from
established protocols45.
Briefly, HeLa cells were resuspended in lysis buffer (10 mM Tris–HCl, pH
8.0, 11.6 % (w/v) sucrose, 1 mM EGTA, 1 mM ATP, 5 mM (DTT), and protease
inhibitor cocktail) and lysed by sonication. Lysates were cleared by
ultracentrifugation (100,000 × g, 2 hours). Cleared cellular extract was
loaded onto an anion-exchange column (Sepharose Q HP, GE Healthcare)
equilibrated with column buffer (10 mM Tris–HCl, pH 8.0, 0.2 mM
CaCl2 , 0.5 mM ATP, 0.5 mM DTT). Actin protein was eluted on a
linear gradient of column buffer containing 0 mM KCl to column buffer containing
600 mM KCl. Actin-positive fractions were polymerized with 2 mM MgCl2
and 1 mM ATP at room temperature, and polymerized actin was pelleted at 100,000
× g for 2 hours. F-actin was resuspended and depolymerized in 2 mM Tris
pH 8.0, 0.2 mM CaCl2, 0.5 mM DTT, and 0.2 mM ATP for ≥ 2 days.
Residual F-actin was cleared by ultracentrifugation as before and the protein in
the supernatant was used in biochemical assays.Pyrene-based polymerization assays were performed according to
commercially available kit instructions (Cytoskeleton). Purified HeLa actin was
combined in a 10:1 ratio with pyrene-labeled actin from rabbit skeletal muscle
(Cytoskeleton). Polymerization was induced with 2 mM MgCl2, 50 mM
KCl, and 1 mM ATP and monitored by fluorescence in a plate reader (excitation:
355 nm/emission: 410 nm, Biotek Cytation 3, Gen5 v3.04.17 software). Readings
were taken every 60 seconds with orbital shaking. Depolymerization assays were
performed by diluting actin polymerized as above 250-fold in 10 mM Tris pH 8.0,
0.2 mM CaCl2, 1 mM ATP. Depolymerization was monitored by
fluorescence similar to polymerization. Phalloidin-actin seeds were made by
polymerizing monomeric actin into filaments followed by incubation of the
resulting F-actin with a stoichiometric excess of phalloidin (Millipore-Sigma).
F-actin was pelleted for 1h at 100,000 × g. The F-actin was washed and
resuspended in F-buffer (10 mM Tris pH 7.5, 2 mM MgCl2, 50 mM KCl, 1
mM ATP, 1 mM DTT) and then sheared through a 27G needle. Seeds were allowed to
recover overnight at room temperature. Polymerization with seeds was performed
with 2 μM seed, 1 μM monomeric actin, and 0.1 μM
pyrene-actin. Polymerization with Arp2/3 complex (Cytoskeleton) and recombinant
WASP VCA (Cytoskeleton) were used at concentrations of 5 nM and 100 nM
respectively with 1 μM actin and 0.1 μM pyrene-actin.Nucleotide Exchange.Nucleotide exchange assays were performed as previously
reported46. G-actin
monomers were transferred into a buffer without ATP (2 mM Tris pH 7.5, 0.2 mM
CaCl2, 1 mM DTT) by desalting column (GE Healthcare, PD spin
column with G-25). 1,N6-ethenoadenosine-5’-O-triphosphate
(ε-ATP, Axxora) was added to G-actin at a final concentration of 0.3 mM
and incubated on ice for at least 2h. Unbound ε-ATP was removed by
desalting as before. ATP exchange of ε-ATP with normal ATP (100
μM) was performed with 2 μM G-actin and monitored by fluorescence
(excitation, 340 nm; emission, 410 nm) in triplicate.Cell culture.293T (ATCC), HT1080 (ATCC) and H1-HeLa (ATCC) cells were cultured in
Dulbecco’s Modified Eagle’s Medium (DMEM, Life Technologies)
supplemented with 10% Fetal bovine serum, 2 mM L-glutamine, and
penicillin/streptomycin (Life Technologies). Primary uterine smooth muscle cells
(ATCC) were cultured in vascular cell basal media (ATCC) supplemented with the
vascular smooth muscle growth kit (ATCC). All above cell lines were
authenticated and declared mycoplasma free by ATCC. Transfections were performed
with either TransIT-293 or TransIT-LT1 (Mirus) in 293T cells or HT1080 cells,
respectively. Stable cell lines were generated using lentiviral transductions.
293T cells were co-transfected with lentiviral plasmid, pCMV-Δ8.2 and
pCMV-VSVg in a ratio of 9:8:1 by mass. 48 hours after transfection, target cells
were transduced with 0.45 μm filtered viral supernatant supplemented with
8 μg/ml polybrene. Viral concentration was performed as necessary with
Lenti-X lentiviral concentration solution per manufacturer’s
recommendations (Takara). Cells were selected 24 hours after media replacement.
To deplete cells of SETD3, CRISPR/Cas9 components were stably expressed in
HT1080 cells using lentiCRISPRv2 (Addgene) with the following single guide RNAs:
sgControl, 5’- CTTCGAAATGTCCGTTCGGT-3’; sgSETD3-1,
5’-TGTTACAGAATGCAGCAGTC-3’; sgSETD3-2,
5’-GTATGTGCAGATCCGGACTC-3’. sgSETD3-1 was used for reconstitution
experiments and generation of the clonal H1-HeLa cell line. Cell selection was
performed with 2 μg/ml puromycin or 100 μg/ml hygromycin. SETD3
cDNA cloned into pLenti CMV Hygro DEST (W117-1) (Addgene) was made
CRISPR/Cas9-resistant by incorporating a synonymous mutation into the PAM
sequence associated with sgSETD3-1 from CGG to CTG. Reconstitution experiments
were performed by transient transfection in HT1080 cells or stable incorporation
by lentivirus in primary uterine smooth muscle cells. SILAC cell culture was
performed with DMEM for SILAC (Thermo Scientific), supplemented with 10%
dialyzed fetal bovine serum, 115 mg proline (Applichem), 50 mg lysine (light,
lysine HCl [Applichem]; heavy,
13C6,15N2-L-Lysine HCl
[Silantes]), and 50 mg arginine (light, arginine HCl [Applichem]; heavy,
13C6,15N4-L-Arginine HCl
[Silantes]). Cells used for SILAC-based proteomics were cultured in SILAC media
for at least one week before analysis.Immunofluorescence imagingH1-HeLa cells were seeded overnight onto autoclaved 12mm circle micro
cover glass (VWR) at 50,000 cells per well in 24-well plates (Greiner Bio-One).
Samples were fixed with 4% paraformaldehyde (Sigma) for 30 minutes at room
temperature. Each sample was incubated with Anti-GFP antibody (1:300) in
immunofluorescence blocking buffer (1× phosphate buffered saline (PBS)
with 3% BSA, 1% saponin and 1% Triton X-100) overnight at 4°C. Cells were
washed thoroughly with 1xPBS and then incubated for 1 hour at room temperature
with Alexa488 anti-rabbit antibody (1:500) and DAPI stain (Insitus
Biotechnologies, 1:300 dilution) in immunofluorescence blocking buffer. Washed
micro cover glasses were then mounted onto microscope slides (Fisher Scientific)
using Vectashield with DAPI (Vector Laboratories Inc). Images were taken with
Zeiss LSM 700 confocal microscope and processed with Volocity software.MEF cell line generation.Embryos were harvested 13.5 dpc and internal organs were removed.
Remaining embryos were minced and treated with trypsin to dissociate cells for
30 minutes at 37°C. Trypsinization was quenched with growth media
(DMEM+10% fetal bovine serum). Quenched cell suspension was plated and allowed
to grow for 4 days. At this point, cells were frozen or used in experiments.
Genotyping of new cell lines were performed with residual embryonic tissue and
confirmed by western blot analysis.Cell Migration assays.Cell migration assays were performed with Radius cell migration kits
(Cell Biolabs). MEFs (1×105 cells) were plated in 24-well
migration plates and allowed to attach overnight. The Radius gel was removed
according the manufacturer’s protocol. After gel removal, imaging was
performed over a 24h period with a Nikon Eclipse Ti inverted fluorescent
microscope and NIS elements (version 4.60) software. During image acquisition,
the environmental conditions were kept at 5% CO2 and 37°C
(Okolab). After the 24 hour period, cells were fixed and stained with DAPI
according to kit instructions. Cell migration was quantified by re-imaging
migration fields and counting DAPI-stained nuclei within the boundary of the
circular void left by the gel at 0h. Migration assays were performed in
triplicate.Cell Contraction assays.Cellular contraction assays were performed on a collagen substrate (Cell
Biolabs). 450 μl collagen mixture made with the provided 5xPBS was added
to a 24-well plate and allowed to polymerize according to the
manufacturer’s protocol. Primary uterine smooth muscle cells
(5×104 cells) were layered on top of a collagen matrix in
vascular basal medium without the growth kit. Cells were allowed to attach to
the collagen overnight at 37°C. The next day, the collagen disks were
released from the sides of each well. Induction of contraction was performed
with 10 μM oxytocin (Tocris) or 1 μM Endothelin-1 (Tocris).
Contraction was monitored by imaging after 24h of contraction. Quantification of
collagen disk area was performed in ImageJ (NIH). Relative disk area was
determined by normalizing to the area of disks lacking cells. All assays were
performed in triplicate.Ethics statementMice were housed in the Stanford mouse facility accredited by the
Association for Assessment and Accreditation of Laboratory Animal Care. All
experiments were approved by Stanford’s Institutional Animal Care and Use
Committee.Maintenance of mouse colonies.Husbandry is performed in accordance with the Guide for the Care
and Use of Laboratory Animals, 8th
edition (2010) and the 2015 revision of the
Public Health Service Policy on Humane Care and Use of Laboratory
Animals. Room conditions included a temperature of 23°C,
relative humidity of 30% to 40%, and a 12:12-h light:dark cycle (lights on, 0700
h). All mouse colonies are maintained under SPF conditions in irradiated
disposable individually ventilated cages (InnocageR, Innovive, San
Diego, CA) with irradiated, corncob bedding, irradiated food (Teklad 2918 Global
18% Protein Rodent Diet, Envigo) and UV-irradiated, acidified (pH, 2.5 to 3.0),
reverse-osmosis–purified bottled water (AquaviveR, Innovive).
The mouse colonies are monitored for viral, bacterial, and parasitic pathogens
using dirty-bedding sentinels that were tested quarterly and were found to be
free of mouse parvovirus, minute virus of mice, mouse hepatitis virus, mouse
rotavirus, Theiler’s murine encephalomyelitis virus, murine norovirus,
Sendai virus, mouse adenovirus 1 and 2, ectromelia virus, lymphocytic
choriomeningitis virus, pneumonia virus of mice, respiratory enterovirus III,
fur mites, lice, and pinworms. Sample size was maximized as much as possible to
gain highest confidence in the results. No randomization was necessary as only
single variables changed per experiment. Blinding was not necessary as any
phenotypic assessment or other measurements was performed using discrete,
quantitative measurementsMouse Strains.C57BL/6N - Setd3tm1.1(NCOM)Mfgc/Tcp
heterozygous mice were obtained from the Canadian Mouse Mutant Repository (CMMR,
Toronto, Canada). The strain was made at the Toronto Centre for Phenogenomics as
part of the NorCOMM2 project using NorCOMM ES cells47. Progeny were genotyped using either the
Setd3-McKerlie PCR assay to identify either the 212-bp wild-type band (primers:
Setd3_wt_F1- AACCAGCAGTGCACAGAGACAAGCTG; Setd3_wt_R1 -
AACCCAAACTCTGCCAGCCAAAGCAC) and/or a 566 bp tm1.1 band (primers: Setd3_tm1_F4-
AAACTGGTGCTGCTGATAACCTGGGC; GH717 - CACCGACGCCAATCACAAACAC) following
instructions provided by the CMMR or the use of a commercial genotyping service
(Transnetyx, Memphis, TN). For timed pregnant matings, C57BL/6N male mice were
purchased from Charles River Laboratories (Hollister, CA).Quantification of litter size and length of gestation.Quantification of pregnancies and litters for both trio or pair breeding
females were tabulated for Setd3−/−,
Setd3+/−,
Setd3+/+ females. All
litters that were observed (“confirmed”) were counted, even if
pups subsequently died. To determine pups/litter, litters that were lost prior
to when pups could be counted were listed as 0 pups for that litter. The length
of gestation for Setd3−/−,
Setd3+/−,
Setd3+/+ females was
determined by timed pregnant matings. Females were mated to wild type C57BL/6N
males (Charles River Laboratories, Hollister, CA). Whether the female was a
virgin (primigravida) or had a previous pregnancy (multigravida) was recorded.
Presence of a vaginal plug was denoted 0 day post coitum (dpc). Plugged females
were moved to a separate cage and weights taken at intervals to confirm
pregnancy. A few days prior to 19 dpc, the female was placed in a separate cage
and provided with nesting material (Enviro-driR, Shepherd Specialty
Papers) and a paper tube. Cages were checked for births twice daily (morning and
afternoon) starting from 18 dpc.Quantifying dystocia and fetuses in utero.The day of parturition for
Setd3−/− females varied from 19
dpc to ≥20 dpc. Normal parturition was defined as birth at 19 days
post-coitum (dpc) and dystocia (delayed parturition) was defined as either no
births (≥20 dpc) or incomplete delivery with fetuses remaining in
utero (≥20 dpc). To determine whether
Setd3−/− females that delivered on
19 dpc still exhibited a delayed in parturition, females were euthanized on late
20 dpc if they did not deliver on 19 dpc
(Setd3−/−) or 24h after giving
birth if they delivered on 19 dpc
(Setd3−/−,
Setd3+/−,
Setd3+/+) and examined for
the presence of fetuses in utero. We note that the dystocia
occurred in both primigravida and multigravida
Setd3−/− females and that
actin-H73me is not detected in the uterus of late-stage pregnant
Setd3−/− females (data not
shown).Induction of parturition.Oxytocin (VetOneR, MWI Veterinary Supply Co.) was given
subcutaneously on 18 dpc to induce preterm labor (dose = 0.4 U in 0.9% saline
supplemented with 0.1% calcium gluconate and 0.025% dextrose, given 3x, q30 min
over a period of 1.5 h) to Setd3−/−,
and Setd3+/+timed-pregnant females.
The normal day of birth for
Setd3+/+mice of this strain is
19 dpc. All pups born on 18 dpc were removed and euthanized. All mice quantified
for fetuses in utero were euthanized at 1700h on d19.A PGF2α cocktail (1 ml LutalyseR (5 mg
PGF2α/ml), 1 ml 10% calcium gluconate, 0.05 ml of 50% isotonic dextrose,
and 8.0 ml of 0.9% sterile saline) was used to induce parturition in two
Setd3−/− females on 19 dpc.
Females were injected subcutaneously with 1 ml of the cocktail at 3 time points
on 19 dpc (~0900 h, ~1200 h, ~1700 h). The females were
monitored for parturition and euthanized on late 20 dpc at 1700 h.Metabolic analysis.Amino acids were analyzed as underivatized compounds by liquid
chromatography tandem mass spectrometry (LC-MS/MS). Samples (50μL) were
deproteinized with 50μL of 6% sulfosalicylic acid, and centrifuged for 15
min at 17,000 RCF. Supernatants were diluted with an isotopic standard mixture
(Cambridge Isotope Laboratories, Inc) in 2 μM tridecafluoroheptanoic acid
(TDFHA), and then injected into the LC-MS/MS system. Samples were analyzed using
a clinically validated amino acid analysis method48. An Agilent 1200 series liquid
chromatography system equipped with a Thermo Hypercarb trap column (3 μm,
4.6 × 50 mm) and a Waters BEH C18 analytical column (2.5 μm, 2.1
× 100 mm) was used to separate compounds in a trap and reverse-elute
configuration. An Agilent 6460 triple quadrupole mass spectrometer was used in
positive polarity electrospray ionization for detection by dynamic multiple
reaction monitoring (MRM). Mobile phase A was 0.02% perfluoroheptanoic acid and
mobile phase B was acetonitrile. Standard curves were prepared by diluting amino
acid standards (Wako Chemicals USA, Inc); for 1-methylhistidine,
S-aminoethylcystine was used as the internal standard and for 3-methylhistidine,
histidine-13C615N3 was used as
the internal standard. Agilent Masshunter B08 was used for data analysis.Muscle Tissue Histology.Tissues were fixed by immersion in 10% neutral buffered formalin,
routinely processed for paraffin embedding, and 5μm thick sections were
routinely stained with hematoxylin and eosin (H&E).Myosin expression and purificationConstruction, expression, and purification of the wild-type recombinant
human beta-cardiac myosin sS1 and the hypertrophic cardiac myopathy H251N
mutant49 are described
in detail elsewhere49-51. Briefly, a truncated version
of MYH7 (residues 1–808), corresponding to sS1, with a C-terminal
enhanced green fluorescent protein (eGFP), was co-expressed with myosin light
chain 3 (MYL3), encoding human ventricular essential light chain (ELC), and
containing an N-terminal FLAG tag (DYKDDDDK) and tobacco etch virus (TEV)
protease site in mouse myoblast C2C12 cells using the AdEasy Vector System
(Obiogene Inc.). The myosin heavy chain with its associated FLAG-tagged ELC was
first purified from clarified lysate with anti-FLAG resin (Sigma). After
cleaving off the FLAG tag with TEV protease, the human beta-cardiac sS1 was
further purified using anion exchange chromatography on a 1-mL HiTrap Q HP
column (GE Healthcare). Peak fractions were eluted with column buffer (10 mM
imidazole, pH 7.5, ~200 mM NaCl, 4 mM MgCl2, 1 mM DTT, 2 mM ATP, and 10%
sucrose) and concentrated by centrifugation in Amicon Ultra-0.5 10-kDa cutoff
spin filters (EMD Millipore) before being used in assays or stored at −80
°C.in vitro motility.Experiments were conducted following previously described
methods51-54. Briefly, frozen myosin protein was
first subjected to a “deadheading” procedure to remove inactive
heads. Myosin was mixed with 5 – 10x molar excess of unlabeled F-actin on
ice for 5 min, followed by addition of 2 mM ATP for 3 min, then centrifuged at
95k rpm in a TLA-100 rotor (Beckman Coulter) at 4°C for 20 min. The
supernatant was collected for use in the motility. Flow chamber was built on a
nitrocellulose-coated coverslip mounted on a glass slide. Deadheaded myosin was
immobilized via specific binding to the GFP antibody that non-specifically binds
to the coverslip surface, followed by blocking with assay buffer (25 mM
imidazole pH 7.5, 25 mM KCl, 4 mM MgCl2, 1 mM EGTA, 10 mM DTT, and 1
mg/ml BSA). A final solution containing fluorescently labeled bovine cardiac
F-actin, 2 mM ATP, and an oxygen-scavenging system (0.4% glucose, 0.11 mg/ml
glucose oxidase, and 0.018 mg/ml catalase) was subsequently flowed in before
imaging on a total internal reflection fluorescence microscope (Nikon Eclipse
Ti-E) equipped with a 561 nm laser (Cobolt) and a 100x oil-immersion objective
(Nikon)55. Time-lapse
movies were acquired at multiple fields of views for each experimental condition
at 1 Hz sampling rate. Movies were analyzed using the FAST (Fast Automated Spud
Trekker) software56 to obtain
the mean velocities with 20% tolerance. All experiments were conducted at
23°C.Actin-activated Myosin ATPase activity.Purified actin was freshly cycled to F-actin by extensive (four times
over 4 d) dialysis in ATPase buffer (10 mM imidazole, pH 7.5, 5 mM
KCl, 3 mM MgCl2, and 1 mM DTT) to remove any residual ATP. The
monomeric concentration of F-actin was determined by measuring the absorbance of
a serial dilution of the actin in 6 M guanidine hydrochloride both at
290 nm with an extinction coefficient of
26,600 M−1cm−1 and at
280 nm with an extinction coefficient of
45,840 M−1cm−1 in a
spectrophotometer (NanoDrop). Full-length human gelsolin was added to actin at a
ratio of 1:1,000 to reduce the viscosity of the actin and thereby decrease
pipetting error at higher actin concentrations without affecting the ATPase
activity. The steady-state actin-activated ATPase activities of freshly prepared
human beta-cardiac sS1-eGFP myosin using WT and KO actin were determined using a
colorimetric readout of phosphate production57, as described in a previous work51. In this assay, reactions containing sS1
at a final concentration of 0.01 mg/ml, 2 mM ATP, and actin at
concentrations ranging from 5 to 77 μM were performed at
23 °C with shaking using a microplate spectrophotometer (Thermo
Scientific Multiskan GO). The rate of sS1 activity was obtained by linear
fitting the phosphate signal as a function of time and converted to activity
units using a phosphate standard. Each actin condition was performed in
duplicate, The error on each data point represents the s.e.m. of the duplicates.
The Michaelis–Menten equation was fitted to determine the maximal
activity (kcat) and the actin concentration at half-maximum (apparent Km for
actin).Statistics.GraphPad Prism 7.0 or KaldeidaGraph 4.5 software were used for
statistics.Data availability.The X-ray structures (coordinates and structure factor files) of
SETD3 with bound actin peptide have been submitted to PDB under accession
number 6MBJ (P21), 6MBK (Two complexes in P212121) and 6MBL (one complex in
P212121). Source data for this study are provided as supplementary information
(Fig. 4d, 5b, 5e, 5f, 5i, 5k; Extended Data Fig. 7e, 9c, 9h).
Source gel data provided for cropped images (Fig. 1a, 1d, 1e, 2a, 2c, 3a-f,g, 4c, 5g, 5j; Extended Data Fig. 1b)
is provided in Supplementary Figure 1. Mass spectrometry data associated with
Fig. 1c/Extended Data Fig. 1h and Fig. 3i are provided as a Supplementary Tables.
Additional requests can be made to the corresponding authors.

Reagents.Biotinylated peptides were synthesized at the UNC High-Throughput
Peptide Synthesis and Array Facility (Chapel Hill, NC)22. Fmoc-L-histidine amino acids with
either (1-me) or (3-me) modifications (Sigma-Aldrich) were used in the synthesis
of these peptides and synthesized peptides were purified by HPLC (>92%
purity). Antibodies used in this study: SETD3 (Abcam), pan-actin (Cytoskeleton),
beta-tubulin (Millipore), H3K4me2 (Cell Signaling), H3K36me2 (Cell Signaling),
H73(3-me) see below, GFP (Invitrogen), Alexa488 anti-rabbit antibody (Life
Technologies), anti-rabbit and anti-mouse peroxidase conjugated antibodies
(Jackson Immunoresearch). Western blots were visualized by chemiluminescence (GE
Healthcare).

Biotinylated peptides were synthesized at the UNC High-Throughput
Peptide Synthesis and Array Facility (Chapel Hill, NC)22. Fmoc-L-histidine amino acids with
either (1-me) or (3-me) modifications (Sigma-Aldrich) were used in the synthesis
of these peptides and synthesized peptides were purified by HPLC (>92%
purity). Antibodies used in this study: SETD3 (Abcam), pan-actin (Cytoskeleton),
beta-tubulin (Millipore), H3K4me2 (Cell Signaling), H3K36me2 (Cell Signaling),
H73(3-me) see below, GFP (Invitrogen), Alexa488 anti-rabbit antibody (Life
Technologies), anti-rabbit and anti-mouse peroxidase conjugated antibodies
(Jackson Immunoresearch). Western blots were visualized by chemiluminescence (GE
Healthcare).

Anti-H73(3-me) specific antibody generation.Peptide corresponding to beta-actin amino acids 68-78 (KYPIEHGIVTN) with
H(3-me) installed at position 73 was synthesized and HPLC purified (99% purity).
This peptide was conjugated to KLH and used as antigen to immunize rabbits.
Rabbit protocols, peptide conjugation, immunization, and antiserum production
were performed by Pocono Rabbit Farm & Laboratory (Canadensis, PA).
Antiserum was negatively selected against unmodified actin peptide (amino acids
66-80). Final purification with was performed with immobilized antigenic peptide
to select for H73(3-me) specific antibodies.

Anti-H73(3-me) specific antibody generation.

Peptide corresponding to beta-actin amino acids 68-78 (KYPIEHGIVTN) with
H(3-me) installed at position 73 was synthesized and HPLC purified (99% purity).
This peptide was conjugated to KLH and used as antigen to immunize rabbits.
Rabbit protocols, peptide conjugation, immunization, and antiserum production
were performed by Pocono Rabbit Farm & Laboratory (Canadensis, PA).
Antiserum was negatively selected against unmodified actin peptide (amino acids
66-80). Final purification with was performed with immobilized antigenic peptide
to select for H73(3-me) specific antibodies.

Recombinant protein expression for in vitro
reactions.SETD3 (Q86TU7) and actin were cloned into pGEX-6P-1. MLL2 catalytic
complex contained GST-tagged catalytic SET domain of MLL2, and full-length,
6xHis-tagged ASH2L, WDR5, DPY30, and RBBP5 that were all individually expressed
in bacteria and assembled in stoichiometric proportions to result in a catalytic
complex23. Lysine
methyltransferases used as controls were reacted with previously reported
substrates24-35. Candidate substrate cDNAs were
subcloned from the human ORFeome (Open Biosystems) using an LR recombination
reaction (Life Technologies) into pDEST15. Mutations of all proteins were
generated using Phusion site-directed mutagenesis (Thermo Scientific) according
to manufacturer’s recommendations. All proteins were expressed as
N-terminal GST fusions (except for non-catalytic MLL complex components) in BL21
or BL21(DE3) codon plus bacteria. Expression was performed at 16°C
overnight in the presence of 0.1 mM IPTG. Recombinant GST fusions were purified
with Glutathione Sepharose 4B (GE Healthcare) and eluted in 100 mM Tris pH 8.0,
10 mg/ml reduced glutathione (Sigma-Aldrich). Any cleavage of GST tags was
performed with Prescission protease (GE Healthcare) overnight at 4°C.
Recombinant 6xHis-tagged proteins were purified with Ni-NTA agarose (Life
technologies) and eluted in 100 mM Tris pH 8.0, 250 mM imidazole. Kinetics
experiments and crystallography utilized additional purification described
below.

Recombinant protein expression for in vitro
reactions.

SETD3 (Q86TU7) and actin were cloned into pGEX-6P-1. MLL2 catalytic
complex contained GST-tagged catalytic SET domain of MLL2, and full-length,
6xHis-tagged ASH2L, WDR5, DPY30, and RBBP5 that were all individually expressed
in bacteria and assembled in stoichiometric proportions to result in a catalytic
complex23. Lysine
methyltransferases used as controls were reacted with previously reported
substrates24-35. Candidate substrate cDNAs were
subcloned from the human ORFeome (Open Biosystems) using an LR recombination
reaction (Life Technologies) into pDEST15. Mutations of all proteins were
generated using Phusion site-directed mutagenesis (Thermo Scientific) according
to manufacturer’s recommendations. All proteins were expressed as
N-terminal GST fusions (except for non-catalytic MLL complex components) in BL21
or BL21(DE3) codon plus bacteria. Expression was performed at 16°C
overnight in the presence of 0.1 mM IPTG. Recombinant GST fusions were purified
with Glutathione Sepharose 4B (GE Healthcare) and eluted in 100 mM Tris pH 8.0,
10 mg/ml reduced glutathione (Sigma-Aldrich). Any cleavage of GST tags was
performed with Prescission protease (GE Healthcare) overnight at 4°C.
Recombinant 6xHis-tagged proteins were purified with Ni-NTA agarose (Life
technologies) and eluted in 100 mM Tris pH 8.0, 250 mM imidazole. Kinetics
experiments and crystallography utilized additional purification described
below.

Purification of SETD3 for crystallography.SETD3 was expressed as a GST-fusion as described above. Cleared
bacterial lysate containing soluble glutathione S-transferase (GST) fusion SETD3
was first loaded on a self-packed glutathione-sepharose column and eluted with
buffer containing 100 mM Tris (pH 8.0), 150 mM NaCl, 5% glycerol, 0.5 mM TCEP
and 20 mM reduced glutathione. The GST tag was removed by ~100 μg
prescission protease (purified in-house) at 4°C overnight. The cleaved
protein solution was diluted to ~50 mM NaCl and further purified by
HiTrap Q-HP, GE Healthcare) with a linear gradient of 50 mM to 500 mM NaCl;
SETD3 eluted together with the cleaved GST around 200-300 mM NaCl. To remove
GST, fractions were loaded onto a fresh glutathione-sepharose column and the
flow-through was collected and concentrated to ~2 ml. SETD3 was then
further purified by size exclusion chromatography (Superdex 200, GE Healthcare)
in 20 mM Tris (pH 8.0), 200 mM NaCl, 5% glycerol, 0.5 mM TCEP. All purification
steps were performed at 4°C. The chromatography was conducted in a
BIO-RAD NGCTM system.

Purification of SETD3 for crystallography.

SETD3 was expressed as a GST-fusion as described above. Cleared
bacterial lysate containing soluble glutathione S-transferase (GST) fusion SETD3
was first loaded on a self-packed glutathione-sepharose column and eluted with
buffer containing 100 mM Tris (pH 8.0), 150 mM NaCl, 5% glycerol, 0.5 mM TCEP
and 20 mM reduced glutathione. The GST tag was removed by ~100 μg
prescission protease (purified in-house) at 4°C overnight. The cleaved
protein solution was diluted to ~50 mM NaCl and further purified by
HiTrap Q-HP, GE Healthcare) with a linear gradient of 50 mM to 500 mM NaCl;
SETD3 eluted together with the cleaved GST around 200-300 mM NaCl. To remove
GST, fractions were loaded onto a fresh glutathione-sepharose column and the
flow-through was collected and concentrated to ~2 ml. SETD3 was then
further purified by size exclusion chromatography (Superdex 200, GE Healthcare)
in 20 mM Tris (pH 8.0), 200 mM NaCl, 5% glycerol, 0.5 mM TCEP. All purification
steps were performed at 4°C. The chromatography was conducted in a
BIO-RAD NGCTM system.

Cellular fractionation.Cytoplasmic cellular extracts were prepared by established
protocols36. Briefly,
cells were disrupted by hypotonic lysis (25 mM KCl, 1.5 mM MgCl2, 10
mM HEPES–KOH pH 7.9, protease inhibitors) and dounce homogenization. Size
exclusion chromatography (Superose 6) was performed in hypotonic lysis buffer.
Ion-exchange chromatography (Sepharose Q HP) columns were equilibrated in
hypotonic lysis buffer and proteins were eluted on a linear gradient from 25-600
mM KCl. Fractions were concentrated with centrifugal filter units (Amicon, 3000
Da MWCO) before being used as substrate in in vitro reactions
with radiolabeled SAM. Chromatography for substrate enrichment was performed
with an AKTA FPLC (GE Healthcare)

Cellular fractionation.

Cytoplasmic cellular extracts were prepared by established
protocols36. Briefly,
cells were disrupted by hypotonic lysis (25 mM KCl, 1.5 mM MgCl2, 10
mM HEPES–KOH pH 7.9, protease inhibitors) and dounce homogenization. Size
exclusion chromatography (Superose 6) was performed in hypotonic lysis buffer.
Ion-exchange chromatography (Sepharose Q HP) columns were equilibrated in
hypotonic lysis buffer and proteins were eluted on a linear gradient from 25-600
mM KCl. Fractions were concentrated with centrifugal filter units (Amicon, 3000
Da MWCO) before being used as substrate in in vitro reactions
with radiolabeled SAM. Chromatography for substrate enrichment was performed
with an AKTA FPLC (GE Healthcare)

In vitro methylation reactions.In vitro methyltransferase reactions were performed
with 5 μg methyltransferase, 1 μg substrate, and 2 μCi
3H-S-adenosylmethionine (American Radiolabeled
Chemicals) in a buffer containing 50 mM Tris pH 8.0, 20 mM KCl, 5 mM
MgCl2 at 30°C overnight. Reactions were resolved by
SDS-PAGE, and 3H-methylation was visualized by autoradiography and
gels were stained with Coomassie as a loading control. Reactions with deuterated
SAM (CDN Isotopes) or normal SAM (Sigma-Aldrich) were performed with SAM at a
final concentration of 80 μM.

In vitro methylation reactions.

In vitro methyltransferase reactions were performed
with 5 μg methyltransferase, 1 μg substrate, and 2 μCi
3H-S-adenosylmethionine (American Radiolabeled
Chemicals) in a buffer containing 50 mM Tris pH 8.0, 20 mM KCl, 5 mM
MgCl2 at 30°C overnight. Reactions were resolved by
SDS-PAGE, and 3H-methylation was visualized by autoradiography and
gels were stained with Coomassie as a loading control. Reactions with deuterated
SAM (CDN Isotopes) or normal SAM (Sigma-Aldrich) were performed with SAM at a
final concentration of 80 μM.

Steady-state kinetic measurement.Human beta-actin peptide (residues 66-80; TLKYPIEHGIVTNWD) was used as a
substrate for SETD3. The reaction mixture containing 20 mM Tris (pH 8.0), 50 mM
NaCl, 0.1 mg/ml BSA, 1 mM DTT, and 0.18 μM SETD3. To determine the Km
value for peptide, the concentration of S-adenosyl-L-methionine (SAM) was kept
constant at 40 μM, while for determination of the Km for SAM, peptide
concentration was kept constant at 50 μM. Reactions were carried out at
room temperature for 20 min with a total volume of 20 μl, and terminated
by the addition of trifluoroacetic acid (TFA) to 0.1% (v/v). The activity of
SETD3 was measured using a bioluminescence assay (MTase-Glo™, Promega) in
which the reaction by-product S-adenosyl-homocysteine (SAH) is converted into
ATP in a two-step reaction and can be detected via a luciferase
reaction37. In
general, 5 μl of reaction mixture was transferred to a low-volume
384-well plate and the luminescence assay was performed according to the
manufacture’s protocol. A Synergy 4 Multi-Mode Microplate Reader (BioTek)
was used to measure luminescent signal.

Steady-state kinetic measurement.

Human beta-actin peptide (residues 66-80; TLKYPIEHGIVTNWD) was used as a
substrate for SETD3. The reaction mixture containing 20 mM Tris (pH 8.0), 50 mM
NaCl, 0.1 mg/ml BSA, 1 mM DTT, and 0.18 μM SETD3. To determine the Km
value for peptide, the concentration of S-adenosyl-L-methionine (SAM) was kept
constant at 40 μM, while for determination of the Km for SAM, peptide
concentration was kept constant at 50 μM. Reactions were carried out at
room temperature for 20 min with a total volume of 20 μl, and terminated
by the addition of trifluoroacetic acid (TFA) to 0.1% (v/v). The activity of
SETD3 was measured using a bioluminescence assay (MTase-Glo™, Promega) in
which the reaction by-product S-adenosyl-homocysteine (SAH) is converted into
ATP in a two-step reaction and can be detected via a luciferase
reaction37. In
general, 5 μl of reaction mixture was transferred to a low-volume
384-well plate and the luminescence assay was performed according to the
manufacture’s protocol. A Synergy 4 Multi-Mode Microplate Reader (BioTek)
was used to measure luminescent signal.

Time-course of methylation for three β-actin peptides.Typically, a reaction mixture contained 20 mM Tris (pH 8.0), 50 mM NaCl,
0.1 mg/ml BSA, 1 mM DTT, 40 μM SAM and 50 μM peptide. In addition
to the unmodified peptide, 1-methyl-His and 3-methyl-His peptides were used in
the activity assay. For unmodified peptide, 0.18 μM SETD3 was used, while
for modified ones, 80-fold higher amounts of SETD3 (14.4 μM) were used
due to very low activity on these peptides. For each time point, 10 μl of
reaction mixture was taken and adding TFA to 0.1% stopped the reaction. The
luminescence assay was performed as described above.

Time-course of methylation for three β-actin peptides.

Typically, a reaction mixture contained 20 mM Tris (pH 8.0), 50 mM NaCl,
0.1 mg/ml BSA, 1 mM DTT, 40 μM SAM and 50 μM peptide. In addition
to the unmodified peptide, 1-methyl-His and 3-methyl-His peptides were used in
the activity assay. For unmodified peptide, 0.18 μM SETD3 was used, while
for modified ones, 80-fold higher amounts of SETD3 (14.4 μM) were used
due to very low activity on these peptides. For each time point, 10 μl of
reaction mixture was taken and adding TFA to 0.1% stopped the reaction. The
luminescence assay was performed as described above.

Mass spectrometry.LC-MS/MS was performed using an Orbitrap Elite or Fusion (Thermo
Scientific) and data was analyzed manually or using MaxQuant software38. Peptide identification
searches with MaxQuant were performed with parameters to identify methionine
oxidation, N-acetylation, monomethyl lysine, dimethyl lysine, trimethyl lysine,
methyl histidine, monomethyl arginine, dimethyl arginine, methyl glutamine, and
methyl cysteine. in vitro reactions were directly resolved by
SDS-PAGE before mass spectrometry analysis was performed as above and digested
with trypsin, chymotrypsin or Glu-C (Promega). Quantitative proteomic analysis
was performed with SILAC, light isotopic labels (K0R0) and heavy isotopic labels
(K8R10). In the SETD3-dependent methylomics, no peptides with a SILAC ratio
greater than 1.5 were deemed to be SETD3-dependent methyl events. No peptides
(other than actin) with at least a 1.5-fold ratio in the reverse direction and
NaN ratio in the forward direction were identified at all in the forward
direction. Actin H73 methylation was detected with light (WT cells, K0R0)
labeling in the forward direction and not with heavy labeling (SETD3 knockout
cells, K8R10). We thus concluded that the lack of a calculable ratio
(light/heavy) in the forward direction is due to the clonal nature of the cell
lines being used. We therefore plotted the forward ratio as infinite (Fig. 3i).Targeted quantification of actin methylation was performed by manual
identification of methylated and unmethylated peptides followed by integration
of area under the curve for chromatographic peaks associated with the
corresponding m/z. Percentage of methylated peptides was
determined as a fraction of the sum total area for methylated and unmethylated
species. Actin was isolated from cellular or tissue sources using DNAse I
affinity chromatography39.
DNAse I was immobilized to NHS-activated Sepharose FF (GE Healthcare) overnight
in 100 mM Hepes pH 7.5, 150 mM NaCl, 80 mM CaCl2. Cells/tissues were
homogenized with buffer containing 1M Tris pH 8.0, 600 mM KCl, 0.5 mM
MgCl2, 4% NP-40, 1% Tween-20, 1 mM DTT, 1 mM ATP, and protease
inhibitors40 (Roche).
Cleared lysates were incubated with immobilized DNase I resin overnight, washed,
boiled in SDS sample buffer, and purified actin was resolved by SDS-PAGE. Gels
were silver-stained (Thermo Scientific) then actin bands were excised and
processed for mass spectrometry by in-gel digest with sequencing grade trypsin.
The following sources were used in the analysis of actin methylation across
evolution: S. cerevisiae, mid-log BY4741; A.
thaliana, leaf tissue; N. benthamiana, leaf
tissue; C. elegans, mixed population; D.
melanogaster, larvae; S. frugiperda, Sf9 cells;
D. rerio, 5 days post fertilization embryos; X.
laevis, egg extract; G. gallus, skeletal muscle;
M. musculus, brain/uterine/skeletal muscle tissue;
H. sapiens, cell lines.

Mass spectrometry.

LC-MS/MS was performed using an Orbitrap Elite or Fusion (Thermo
Scientific) and data was analyzed manually or using MaxQuant software38. Peptide identification
searches with MaxQuant were performed with parameters to identify methionine
oxidation, N-acetylation, monomethyl lysine, dimethyl lysine, trimethyl lysine,
methyl histidine, monomethyl arginine, dimethyl arginine, methyl glutamine, and
methyl cysteine. in vitro reactions were directly resolved by
SDS-PAGE before mass spectrometry analysis was performed as above and digested
with trypsin, chymotrypsin or Glu-C (Promega). Quantitative proteomic analysis
was performed with SILAC, light isotopic labels (K0R0) and heavy isotopic labels
(K8R10). In the SETD3-dependent methylomics, no peptides with a SILAC ratio
greater than 1.5 were deemed to be SETD3-dependent methyl events. No peptides
(other than actin) with at least a 1.5-fold ratio in the reverse direction and
NaN ratio in the forward direction were identified at all in the forward
direction. Actin H73 methylation was detected with light (WT cells, K0R0)
labeling in the forward direction and not with heavy labeling (SETD3 knockout
cells, K8R10). We thus concluded that the lack of a calculable ratio
(light/heavy) in the forward direction is due to the clonal nature of the cell
lines being used. We therefore plotted the forward ratio as infinite (Fig. 3i).

Targeted quantification of actin methylation was performed by manual
identification of methylated and unmethylated peptides followed by integration
of area under the curve for chromatographic peaks associated with the
corresponding m/z. Percentage of methylated peptides was
determined as a fraction of the sum total area for methylated and unmethylated
species. Actin was isolated from cellular or tissue sources using DNAse I
affinity chromatography39.
DNAse I was immobilized to NHS-activated Sepharose FF (GE Healthcare) overnight
in 100 mM Hepes pH 7.5, 150 mM NaCl, 80 mM CaCl2. Cells/tissues were
homogenized with buffer containing 1M Tris pH 8.0, 600 mM KCl, 0.5 mM
MgCl2, 4% NP-40, 1% Tween-20, 1 mM DTT, 1 mM ATP, and protease
inhibitors40 (Roche).
Cleared lysates were incubated with immobilized DNase I resin overnight, washed,
boiled in SDS sample buffer, and purified actin was resolved by SDS-PAGE. Gels
were silver-stained (Thermo Scientific) then actin bands were excised and
processed for mass spectrometry by in-gel digest with sequencing grade trypsin.
The following sources were used in the analysis of actin methylation across
evolution: S. cerevisiae, mid-log BY4741; A.
thaliana, leaf tissue; N. benthamiana, leaf
tissue; C. elegans, mixed population; D.
melanogaster, larvae; S. frugiperda, Sf9 cells;
D. rerio, 5 days post fertilization embryos; X.
laevis, egg extract; G. gallus, skeletal muscle;
M. musculus, brain/uterine/skeletal muscle tissue;
H. sapiens, cell lines.

Crystallography.Crystallization of the SETD3-SAH-actin peptide (residues 66-80;
Genscript) complex was carried out by the hanging-drop vapor-diffusion method at
20°C after mixing SETD3 (~15 mg/ml) with SAH and peptide at a
molar ratio of 1:4:5, and then mixed with an equal amount of well-solution.
Three different crystal forms were obtained. Crystals with space group
P21 were obtained with 0.2 M ammonium acetate,
0.1 M sodium citrate tribasic dihydrate pH 5.6, and 30% (w/v) polyethylene
glycol 4000. Crystals with space group
P212121 were obtained with
two similar conditions, either 0.2 M ammonium sulfate or 0.2 M sodium chloride,
0.1 M BIS-TRIS pH 5.5, 25% (w/v) polyethylene glycol 3350, and 0.1 M BIS-TRIS pH
5.5, 25% (w/v) polyethylene glycol 3350. The first form, like the
P21 crystal, contained two SETD3-SAH-peptide
complexes per asymmetric unit, while the second contained only one complex.For X-ray diffraction data collection, single crystals were flash frozen
in liquid nitrogen by equilibrating in a cryoprotectant buffer containing well
solution with 25% (v/v) ethylene glycol. All datasets were collected at the
SER-CAT beamline 22ID of Advanced Photon Source at Argonne National Laboratory.
Crystallographic datasets were processed with HKL200041. Molecular replacement was performed
with PHENIX PHASER module42 by
using the structure of human SETD3 in complex with SAM (PDB ID 3SMT) as a search
model. PHENIX REFINE43 was
used for refinement with 5% randomly chosen reflections for validation by the
Rfree value. COOT44 was used
for peptide building and model corrections between refinement rounds. Structure
quality was analyzed during PHENIX refinements and later validated by the PDB
validation server. Molecular graphics were generated using PyMol
(Schrödinger, LLC).We note that the high-resolution data at 1.69 Å (PDB 6MBK) were
the result of merging two datasets collected from two different parts of the
same crystal. This crystal did diffract X-rays to higher resolution but was
susceptible to radiation damage, especially in the higher resolution range. The
merge resulted in 98.4% completeness in the highest resolution shell of
1.75-1.69 Å. The somewhat higher R-work and R-free values of the
structure (PDB 6MBK) might be an indication of radiation damage that could not
be modeled and further refinement would not lower these R values substantially
and the structure and map compared well with the quality of the other two
crystal forms.

Crystallography.

Crystallization of the SETD3-SAH-actin peptide (residues 66-80;
Genscript) complex was carried out by the hanging-drop vapor-diffusion method at
20°C after mixing SETD3 (~15 mg/ml) with SAH and peptide at a
molar ratio of 1:4:5, and then mixed with an equal amount of well-solution.
Three different crystal forms were obtained. Crystals with space group
P21 were obtained with 0.2 M ammonium acetate,
0.1 M sodium citrate tribasic dihydrate pH 5.6, and 30% (w/v) polyethylene
glycol 4000. Crystals with space group
P212121 were obtained with
two similar conditions, either 0.2 M ammonium sulfate or 0.2 M sodium chloride,
0.1 M BIS-TRIS pH 5.5, 25% (w/v) polyethylene glycol 3350, and 0.1 M BIS-TRIS pH
5.5, 25% (w/v) polyethylene glycol 3350. The first form, like the
P21 crystal, contained two SETD3-SAH-peptide
complexes per asymmetric unit, while the second contained only one complex.

For X-ray diffraction data collection, single crystals were flash frozen
in liquid nitrogen by equilibrating in a cryoprotectant buffer containing well
solution with 25% (v/v) ethylene glycol. All datasets were collected at the
SER-CAT beamline 22ID of Advanced Photon Source at Argonne National Laboratory.
Crystallographic datasets were processed with HKL200041. Molecular replacement was performed
with PHENIX PHASER module42 by
using the structure of human SETD3 in complex with SAM (PDB ID 3SMT) as a search
model. PHENIX REFINE43 was
used for refinement with 5% randomly chosen reflections for validation by the
Rfree value. COOT44 was used
for peptide building and model corrections between refinement rounds. Structure
quality was analyzed during PHENIX refinements and later validated by the PDB
validation server. Molecular graphics were generated using PyMol
(Schrödinger, LLC).

We note that the high-resolution data at 1.69 Å (PDB 6MBK) were
the result of merging two datasets collected from two different parts of the
same crystal. This crystal did diffract X-rays to higher resolution but was
susceptible to radiation damage, especially in the higher resolution range. The
merge resulted in 98.4% completeness in the highest resolution shell of
1.75-1.69 Å. The somewhat higher R-work and R-free values of the
structure (PDB 6MBK) might be an indication of radiation damage that could not
be modeled and further refinement would not lower these R values substantially
and the structure and map compared well with the quality of the other two
crystal forms.

Actin purification for biochemical assays.Actin was purified from wild-type HeLa cells or a clonally-derived HeLa
cell line in which SETD3 was knocked out. Purification was adapted from
established protocols45.
Briefly, HeLa cells were resuspended in lysis buffer (10 mM Tris–HCl, pH
8.0, 11.6 % (w/v) sucrose, 1 mM EGTA, 1 mM ATP, 5 mM (DTT), and protease
inhibitor cocktail) and lysed by sonication. Lysates were cleared by
ultracentrifugation (100,000 × g, 2 hours). Cleared cellular extract was
loaded onto an anion-exchange column (Sepharose Q HP, GE Healthcare)
equilibrated with column buffer (10 mM Tris–HCl, pH 8.0, 0.2 mM
CaCl2 , 0.5 mM ATP, 0.5 mM DTT). Actin protein was eluted on a
linear gradient of column buffer containing 0 mM KCl to column buffer containing
600 mM KCl. Actin-positive fractions were polymerized with 2 mM MgCl2
and 1 mM ATP at room temperature, and polymerized actin was pelleted at 100,000
× g for 2 hours. F-actin was resuspended and depolymerized in 2 mM Tris
pH 8.0, 0.2 mM CaCl2, 0.5 mM DTT, and 0.2 mM ATP for ≥ 2 days.
Residual F-actin was cleared by ultracentrifugation as before and the protein in
the supernatant was used in biochemical assays.Pyrene-based polymerization assays were performed according to
commercially available kit instructions (Cytoskeleton). Purified HeLa actin was
combined in a 10:1 ratio with pyrene-labeled actin from rabbit skeletal muscle
(Cytoskeleton). Polymerization was induced with 2 mM MgCl2, 50 mM
KCl, and 1 mM ATP and monitored by fluorescence in a plate reader (excitation:
355 nm/emission: 410 nm, Biotek Cytation 3, Gen5 v3.04.17 software). Readings
were taken every 60 seconds with orbital shaking. Depolymerization assays were
performed by diluting actin polymerized as above 250-fold in 10 mM Tris pH 8.0,
0.2 mM CaCl2, 1 mM ATP. Depolymerization was monitored by
fluorescence similar to polymerization. Phalloidin-actin seeds were made by
polymerizing monomeric actin into filaments followed by incubation of the
resulting F-actin with a stoichiometric excess of phalloidin (Millipore-Sigma).
F-actin was pelleted for 1h at 100,000 × g. The F-actin was washed and
resuspended in F-buffer (10 mM Tris pH 7.5, 2 mM MgCl2, 50 mM KCl, 1
mM ATP, 1 mM DTT) and then sheared through a 27G needle. Seeds were allowed to
recover overnight at room temperature. Polymerization with seeds was performed
with 2 μM seed, 1 μM monomeric actin, and 0.1 μM
pyrene-actin. Polymerization with Arp2/3 complex (Cytoskeleton) and recombinant
WASP VCA (Cytoskeleton) were used at concentrations of 5 nM and 100 nM
respectively with 1 μM actin and 0.1 μM pyrene-actin.

Actin purification for biochemical assays.

Actin was purified from wild-type HeLa cells or a clonally-derived HeLa
cell line in which SETD3 was knocked out. Purification was adapted from
established protocols45.
Briefly, HeLa cells were resuspended in lysis buffer (10 mM Tris–HCl, pH
8.0, 11.6 % (w/v) sucrose, 1 mM EGTA, 1 mM ATP, 5 mM (DTT), and protease
inhibitor cocktail) and lysed by sonication. Lysates were cleared by
ultracentrifugation (100,000 × g, 2 hours). Cleared cellular extract was
loaded onto an anion-exchange column (Sepharose Q HP, GE Healthcare)
equilibrated with column buffer (10 mM Tris–HCl, pH 8.0, 0.2 mM
CaCl2 , 0.5 mM ATP, 0.5 mM DTT). Actin protein was eluted on a
linear gradient of column buffer containing 0 mM KCl to column buffer containing
600 mM KCl. Actin-positive fractions were polymerized with 2 mM MgCl2
and 1 mM ATP at room temperature, and polymerized actin was pelleted at 100,000
× g for 2 hours. F-actin was resuspended and depolymerized in 2 mM Tris
pH 8.0, 0.2 mM CaCl2, 0.5 mM DTT, and 0.2 mM ATP for ≥ 2 days.
Residual F-actin was cleared by ultracentrifugation as before and the protein in
the supernatant was used in biochemical assays.

Pyrene-based polymerization assays were performed according to
commercially available kit instructions (Cytoskeleton). Purified HeLa actin was
combined in a 10:1 ratio with pyrene-labeled actin from rabbit skeletal muscle
(Cytoskeleton). Polymerization was induced with 2 mM MgCl2, 50 mM
KCl, and 1 mM ATP and monitored by fluorescence in a plate reader (excitation:
355 nm/emission: 410 nm, Biotek Cytation 3, Gen5 v3.04.17 software). Readings
were taken every 60 seconds with orbital shaking. Depolymerization assays were
performed by diluting actin polymerized as above 250-fold in 10 mM Tris pH 8.0,
0.2 mM CaCl2, 1 mM ATP. Depolymerization was monitored by
fluorescence similar to polymerization. Phalloidin-actin seeds were made by
polymerizing monomeric actin into filaments followed by incubation of the
resulting F-actin with a stoichiometric excess of phalloidin (Millipore-Sigma).
F-actin was pelleted for 1h at 100,000 × g. The F-actin was washed and
resuspended in F-buffer (10 mM Tris pH 7.5, 2 mM MgCl2, 50 mM KCl, 1
mM ATP, 1 mM DTT) and then sheared through a 27G needle. Seeds were allowed to
recover overnight at room temperature. Polymerization with seeds was performed
with 2 μM seed, 1 μM monomeric actin, and 0.1 μM
pyrene-actin. Polymerization with Arp2/3 complex (Cytoskeleton) and recombinant
WASP VCA (Cytoskeleton) were used at concentrations of 5 nM and 100 nM
respectively with 1 μM actin and 0.1 μM pyrene-actin.

Nucleotide Exchange.Nucleotide exchange assays were performed as previously
reported46. G-actin
monomers were transferred into a buffer without ATP (2 mM Tris pH 7.5, 0.2 mM
CaCl2, 1 mM DTT) by desalting column (GE Healthcare, PD spin
column with G-25). 1,N6-ethenoadenosine-5’-O-triphosphate
(ε-ATP, Axxora) was added to G-actin at a final concentration of 0.3 mM
and incubated on ice for at least 2h. Unbound ε-ATP was removed by
desalting as before. ATP exchange of ε-ATP with normal ATP (100
μM) was performed with 2 μM G-actin and monitored by fluorescence
(excitation, 340 nm; emission, 410 nm) in triplicate.

Nucleotide Exchange.

Nucleotide exchange assays were performed as previously
reported46. G-actin
monomers were transferred into a buffer without ATP (2 mM Tris pH 7.5, 0.2 mM
CaCl2, 1 mM DTT) by desalting column (GE Healthcare, PD spin
column with G-25). 1,N6-ethenoadenosine-5’-O-triphosphate
(ε-ATP, Axxora) was added to G-actin at a final concentration of 0.3 mM
and incubated on ice for at least 2h. Unbound ε-ATP was removed by
desalting as before. ATP exchange of ε-ATP with normal ATP (100
μM) was performed with 2 μM G-actin and monitored by fluorescence
(excitation, 340 nm; emission, 410 nm) in triplicate.

Cell culture.293T (ATCC), HT1080 (ATCC) and H1-HeLa (ATCC) cells were cultured in
Dulbecco’s Modified Eagle’s Medium (DMEM, Life Technologies)
supplemented with 10% Fetal bovine serum, 2 mM L-glutamine, and
penicillin/streptomycin (Life Technologies). Primary uterine smooth muscle cells
(ATCC) were cultured in vascular cell basal media (ATCC) supplemented with the
vascular smooth muscle growth kit (ATCC). All above cell lines were
authenticated and declared mycoplasma free by ATCC. Transfections were performed
with either TransIT-293 or TransIT-LT1 (Mirus) in 293T cells or HT1080 cells,
respectively. Stable cell lines were generated using lentiviral transductions.
293T cells were co-transfected with lentiviral plasmid, pCMV-Δ8.2 and
pCMV-VSVg in a ratio of 9:8:1 by mass. 48 hours after transfection, target cells
were transduced with 0.45 μm filtered viral supernatant supplemented with
8 μg/ml polybrene. Viral concentration was performed as necessary with
Lenti-X lentiviral concentration solution per manufacturer’s
recommendations (Takara). Cells were selected 24 hours after media replacement.
To deplete cells of SETD3, CRISPR/Cas9 components were stably expressed in
HT1080 cells using lentiCRISPRv2 (Addgene) with the following single guide RNAs:
sgControl, 5’- CTTCGAAATGTCCGTTCGGT-3’; sgSETD3-1,
5’-TGTTACAGAATGCAGCAGTC-3’; sgSETD3-2,
5’-GTATGTGCAGATCCGGACTC-3’. sgSETD3-1 was used for reconstitution
experiments and generation of the clonal H1-HeLa cell line. Cell selection was
performed with 2 μg/ml puromycin or 100 μg/ml hygromycin. SETD3
cDNA cloned into pLenti CMV Hygro DEST (W117-1) (Addgene) was made
CRISPR/Cas9-resistant by incorporating a synonymous mutation into the PAM
sequence associated with sgSETD3-1 from CGG to CTG. Reconstitution experiments
were performed by transient transfection in HT1080 cells or stable incorporation
by lentivirus in primary uterine smooth muscle cells. SILAC cell culture was
performed with DMEM for SILAC (Thermo Scientific), supplemented with 10%
dialyzed fetal bovine serum, 115 mg proline (Applichem), 50 mg lysine (light,
lysine HCl [Applichem]; heavy,
13C6,15N2-L-Lysine HCl
[Silantes]), and 50 mg arginine (light, arginine HCl [Applichem]; heavy,
13C6,15N4-L-Arginine HCl
[Silantes]). Cells used for SILAC-based proteomics were cultured in SILAC media
for at least one week before analysis.

Cell culture.

293T (ATCC), HT1080 (ATCC) and H1-HeLa (ATCC) cells were cultured in
Dulbecco’s Modified Eagle’s Medium (DMEM, Life Technologies)
supplemented with 10% Fetal bovine serum, 2 mM L-glutamine, and
penicillin/streptomycin (Life Technologies). Primary uterine smooth muscle cells
(ATCC) were cultured in vascular cell basal media (ATCC) supplemented with the
vascular smooth muscle growth kit (ATCC). All above cell lines were
authenticated and declared mycoplasma free by ATCC. Transfections were performed
with either TransIT-293 or TransIT-LT1 (Mirus) in 293T cells or HT1080 cells,
respectively. Stable cell lines were generated using lentiviral transductions.
293T cells were co-transfected with lentiviral plasmid, pCMV-Δ8.2 and
pCMV-VSVg in a ratio of 9:8:1 by mass. 48 hours after transfection, target cells
were transduced with 0.45 μm filtered viral supernatant supplemented with
8 μg/ml polybrene. Viral concentration was performed as necessary with
Lenti-X lentiviral concentration solution per manufacturer’s
recommendations (Takara). Cells were selected 24 hours after media replacement.
To deplete cells of SETD3, CRISPR/Cas9 components were stably expressed in
HT1080 cells using lentiCRISPRv2 (Addgene) with the following single guide RNAs:
sgControl, 5’- CTTCGAAATGTCCGTTCGGT-3’; sgSETD3-1,
5’-TGTTACAGAATGCAGCAGTC-3’; sgSETD3-2,
5’-GTATGTGCAGATCCGGACTC-3’. sgSETD3-1 was used for reconstitution
experiments and generation of the clonal H1-HeLa cell line. Cell selection was
performed with 2 μg/ml puromycin or 100 μg/ml hygromycin. SETD3
cDNA cloned into pLenti CMV Hygro DEST (W117-1) (Addgene) was made
CRISPR/Cas9-resistant by incorporating a synonymous mutation into the PAM
sequence associated with sgSETD3-1 from CGG to CTG. Reconstitution experiments
were performed by transient transfection in HT1080 cells or stable incorporation
by lentivirus in primary uterine smooth muscle cells. SILAC cell culture was
performed with DMEM for SILAC (Thermo Scientific), supplemented with 10%
dialyzed fetal bovine serum, 115 mg proline (Applichem), 50 mg lysine (light,
lysine HCl [Applichem]; heavy,
13C6,15N2-L-Lysine HCl
[Silantes]), and 50 mg arginine (light, arginine HCl [Applichem]; heavy,
13C6,15N4-L-Arginine HCl
[Silantes]). Cells used for SILAC-based proteomics were cultured in SILAC media
for at least one week before analysis.

Immunofluorescence imagingH1-HeLa cells were seeded overnight onto autoclaved 12mm circle micro
cover glass (VWR) at 50,000 cells per well in 24-well plates (Greiner Bio-One).
Samples were fixed with 4% paraformaldehyde (Sigma) for 30 minutes at room
temperature. Each sample was incubated with Anti-GFP antibody (1:300) in
immunofluorescence blocking buffer (1× phosphate buffered saline (PBS)
with 3% BSA, 1% saponin and 1% Triton X-100) overnight at 4°C. Cells were
washed thoroughly with 1xPBS and then incubated for 1 hour at room temperature
with Alexa488 anti-rabbit antibody (1:500) and DAPI stain (Insitus
Biotechnologies, 1:300 dilution) in immunofluorescence blocking buffer. Washed
micro cover glasses were then mounted onto microscope slides (Fisher Scientific)
using Vectashield with DAPI (Vector Laboratories Inc). Images were taken with
Zeiss LSM 700 confocal microscope and processed with Volocity software.

Immunofluorescence imaging

H1-HeLa cells were seeded overnight onto autoclaved 12mm circle micro
cover glass (VWR) at 50,000 cells per well in 24-well plates (Greiner Bio-One).
Samples were fixed with 4% paraformaldehyde (Sigma) for 30 minutes at room
temperature. Each sample was incubated with Anti-GFP antibody (1:300) in
immunofluorescence blocking buffer (1× phosphate buffered saline (PBS)
with 3% BSA, 1% saponin and 1% Triton X-100) overnight at 4°C. Cells were
washed thoroughly with 1xPBS and then incubated for 1 hour at room temperature
with Alexa488 anti-rabbit antibody (1:500) and DAPI stain (Insitus
Biotechnologies, 1:300 dilution) in immunofluorescence blocking buffer. Washed
micro cover glasses were then mounted onto microscope slides (Fisher Scientific)
using Vectashield with DAPI (Vector Laboratories Inc). Images were taken with
Zeiss LSM 700 confocal microscope and processed with Volocity software.

MEF cell line generation.Embryos were harvested 13.5 dpc and internal organs were removed.
Remaining embryos were minced and treated with trypsin to dissociate cells for
30 minutes at 37°C. Trypsinization was quenched with growth media
(DMEM+10% fetal bovine serum). Quenched cell suspension was plated and allowed
to grow for 4 days. At this point, cells were frozen or used in experiments.
Genotyping of new cell lines were performed with residual embryonic tissue and
confirmed by western blot analysis.

MEF cell line generation.

Embryos were harvested 13.5 dpc and internal organs were removed.
Remaining embryos were minced and treated with trypsin to dissociate cells for
30 minutes at 37°C. Trypsinization was quenched with growth media
(DMEM+10% fetal bovine serum). Quenched cell suspension was plated and allowed
to grow for 4 days. At this point, cells were frozen or used in experiments.
Genotyping of new cell lines were performed with residual embryonic tissue and
confirmed by western blot analysis.

Cell Migration assays.Cell migration assays were performed with Radius cell migration kits
(Cell Biolabs). MEFs (1×105 cells) were plated in 24-well
migration plates and allowed to attach overnight. The Radius gel was removed
according the manufacturer’s protocol. After gel removal, imaging was
performed over a 24h period with a Nikon Eclipse Ti inverted fluorescent
microscope and NIS elements (version 4.60) software. During image acquisition,
the environmental conditions were kept at 5% CO2 and 37°C
(Okolab). After the 24 hour period, cells were fixed and stained with DAPI
according to kit instructions. Cell migration was quantified by re-imaging
migration fields and counting DAPI-stained nuclei within the boundary of the
circular void left by the gel at 0h. Migration assays were performed in
triplicate.

Cell Migration assays.

Cell migration assays were performed with Radius cell migration kits
(Cell Biolabs). MEFs (1×105 cells) were plated in 24-well
migration plates and allowed to attach overnight. The Radius gel was removed
according the manufacturer’s protocol. After gel removal, imaging was
performed over a 24h period with a Nikon Eclipse Ti inverted fluorescent
microscope and NIS elements (version 4.60) software. During image acquisition,
the environmental conditions were kept at 5% CO2 and 37°C
(Okolab). After the 24 hour period, cells were fixed and stained with DAPI
according to kit instructions. Cell migration was quantified by re-imaging
migration fields and counting DAPI-stained nuclei within the boundary of the
circular void left by the gel at 0h. Migration assays were performed in
triplicate.

Cell Contraction assays.Cellular contraction assays were performed on a collagen substrate (Cell
Biolabs). 450 μl collagen mixture made with the provided 5xPBS was added
to a 24-well plate and allowed to polymerize according to the
manufacturer’s protocol. Primary uterine smooth muscle cells
(5×104 cells) were layered on top of a collagen matrix in
vascular basal medium without the growth kit. Cells were allowed to attach to
the collagen overnight at 37°C. The next day, the collagen disks were
released from the sides of each well. Induction of contraction was performed
with 10 μM oxytocin (Tocris) or 1 μM Endothelin-1 (Tocris).
Contraction was monitored by imaging after 24h of contraction. Quantification of
collagen disk area was performed in ImageJ (NIH). Relative disk area was
determined by normalizing to the area of disks lacking cells. All assays were
performed in triplicate.

Cell Contraction assays.

Cellular contraction assays were performed on a collagen substrate (Cell
Biolabs). 450 μl collagen mixture made with the provided 5xPBS was added
to a 24-well plate and allowed to polymerize according to the
manufacturer’s protocol. Primary uterine smooth muscle cells
(5×104 cells) were layered on top of a collagen matrix in
vascular basal medium without the growth kit. Cells were allowed to attach to
the collagen overnight at 37°C. The next day, the collagen disks were
released from the sides of each well. Induction of contraction was performed
with 10 μM oxytocin (Tocris) or 1 μM Endothelin-1 (Tocris).
Contraction was monitored by imaging after 24h of contraction. Quantification of
collagen disk area was performed in ImageJ (NIH). Relative disk area was
determined by normalizing to the area of disks lacking cells. All assays were
performed in triplicate.

Ethics statementMice were housed in the Stanford mouse facility accredited by the
Association for Assessment and Accreditation of Laboratory Animal Care. All
experiments were approved by Stanford’s Institutional Animal Care and Use
Committee.

Ethics statement

Mice were housed in the Stanford mouse facility accredited by the
Association for Assessment and Accreditation of Laboratory Animal Care. All
experiments were approved by Stanford’s Institutional Animal Care and Use
Committee.

Maintenance of mouse colonies.Husbandry is performed in accordance with the Guide for the Care
and Use of Laboratory Animals, 8th
edition (2010) and the 2015 revision of the
Public Health Service Policy on Humane Care and Use of Laboratory
Animals. Room conditions included a temperature of 23°C,
relative humidity of 30% to 40%, and a 12:12-h light:dark cycle (lights on, 0700
h). All mouse colonies are maintained under SPF conditions in irradiated
disposable individually ventilated cages (InnocageR, Innovive, San
Diego, CA) with irradiated, corncob bedding, irradiated food (Teklad 2918 Global
18% Protein Rodent Diet, Envigo) and UV-irradiated, acidified (pH, 2.5 to 3.0),
reverse-osmosis–purified bottled water (AquaviveR, Innovive).
The mouse colonies are monitored for viral, bacterial, and parasitic pathogens
using dirty-bedding sentinels that were tested quarterly and were found to be
free of mouse parvovirus, minute virus of mice, mouse hepatitis virus, mouse
rotavirus, Theiler’s murine encephalomyelitis virus, murine norovirus,
Sendai virus, mouse adenovirus 1 and 2, ectromelia virus, lymphocytic
choriomeningitis virus, pneumonia virus of mice, respiratory enterovirus III,
fur mites, lice, and pinworms. Sample size was maximized as much as possible to
gain highest confidence in the results. No randomization was necessary as only
single variables changed per experiment. Blinding was not necessary as any
phenotypic assessment or other measurements was performed using discrete,
quantitative measurements

Maintenance of mouse colonies.

Husbandry is performed in accordance with the Guide for the Care
and Use of Laboratory Animals, 8th
edition (2010) and the 2015 revision of the
Public Health Service Policy on Humane Care and Use of Laboratory
Animals. Room conditions included a temperature of 23°C,
relative humidity of 30% to 40%, and a 12:12-h light:dark cycle (lights on, 0700
h). All mouse colonies are maintained under SPF conditions in irradiated
disposable individually ventilated cages (InnocageR, Innovive, San
Diego, CA) with irradiated, corncob bedding, irradiated food (Teklad 2918 Global
18% Protein Rodent Diet, Envigo) and UV-irradiated, acidified (pH, 2.5 to 3.0),
reverse-osmosis–purified bottled water (AquaviveR, Innovive).
The mouse colonies are monitored for viral, bacterial, and parasitic pathogens
using dirty-bedding sentinels that were tested quarterly and were found to be
free of mouse parvovirus, minute virus of mice, mouse hepatitis virus, mouse
rotavirus, Theiler’s murine encephalomyelitis virus, murine norovirus,
Sendai virus, mouse adenovirus 1 and 2, ectromelia virus, lymphocytic
choriomeningitis virus, pneumonia virus of mice, respiratory enterovirus III,
fur mites, lice, and pinworms. Sample size was maximized as much as possible to
gain highest confidence in the results. No randomization was necessary as only
single variables changed per experiment. Blinding was not necessary as any
phenotypic assessment or other measurements was performed using discrete,
quantitative measurements

Mouse Strains.C57BL/6N - Setd3tm1.1(NCOM)Mfgc/Tcp
heterozygous mice were obtained from the Canadian Mouse Mutant Repository (CMMR,
Toronto, Canada). The strain was made at the Toronto Centre for Phenogenomics as
part of the NorCOMM2 project using NorCOMM ES cells47. Progeny were genotyped using either the
Setd3-McKerlie PCR assay to identify either the 212-bp wild-type band (primers:
Setd3_wt_F1- AACCAGCAGTGCACAGAGACAAGCTG; Setd3_wt_R1 -
AACCCAAACTCTGCCAGCCAAAGCAC) and/or a 566 bp tm1.1 band (primers: Setd3_tm1_F4-
AAACTGGTGCTGCTGATAACCTGGGC; GH717 - CACCGACGCCAATCACAAACAC) following
instructions provided by the CMMR or the use of a commercial genotyping service
(Transnetyx, Memphis, TN). For timed pregnant matings, C57BL/6N male mice were
purchased from Charles River Laboratories (Hollister, CA).

Mouse Strains.

C57BL/6N - Setd3tm1.1(NCOM)Mfgc/Tcp
heterozygous mice were obtained from the Canadian Mouse Mutant Repository (CMMR,
Toronto, Canada). The strain was made at the Toronto Centre for Phenogenomics as
part of the NorCOMM2 project using NorCOMM ES cells47. Progeny were genotyped using either the
Setd3-McKerlie PCR assay to identify either the 212-bp wild-type band (primers:
Setd3_wt_F1- AACCAGCAGTGCACAGAGACAAGCTG; Setd3_wt_R1 -
AACCCAAACTCTGCCAGCCAAAGCAC) and/or a 566 bp tm1.1 band (primers: Setd3_tm1_F4-
AAACTGGTGCTGCTGATAACCTGGGC; GH717 - CACCGACGCCAATCACAAACAC) following
instructions provided by the CMMR or the use of a commercial genotyping service
(Transnetyx, Memphis, TN). For timed pregnant matings, C57BL/6N male mice were
purchased from Charles River Laboratories (Hollister, CA).

Quantification of litter size and length of gestation.Quantification of pregnancies and litters for both trio or pair breeding
females were tabulated for Setd3−/−,
Setd3+/−,
Setd3+/+ females. All
litters that were observed (“confirmed”) were counted, even if
pups subsequently died. To determine pups/litter, litters that were lost prior
to when pups could be counted were listed as 0 pups for that litter. The length
of gestation for Setd3−/−,
Setd3+/−,
Setd3+/+ females was
determined by timed pregnant matings. Females were mated to wild type C57BL/6N
males (Charles River Laboratories, Hollister, CA). Whether the female was a
virgin (primigravida) or had a previous pregnancy (multigravida) was recorded.
Presence of a vaginal plug was denoted 0 day post coitum (dpc). Plugged females
were moved to a separate cage and weights taken at intervals to confirm
pregnancy. A few days prior to 19 dpc, the female was placed in a separate cage
and provided with nesting material (Enviro-driR, Shepherd Specialty
Papers) and a paper tube. Cages were checked for births twice daily (morning and
afternoon) starting from 18 dpc.

Quantification of litter size and length of gestation.

Quantification of pregnancies and litters for both trio or pair breeding
females were tabulated for Setd3−/−,
Setd3+/−,
Setd3+/+ females. All
litters that were observed (“confirmed”) were counted, even if
pups subsequently died. To determine pups/litter, litters that were lost prior
to when pups could be counted were listed as 0 pups for that litter. The length
of gestation for Setd3−/−,
Setd3+/−,
Setd3+/+ females was
determined by timed pregnant matings. Females were mated to wild type C57BL/6N
males (Charles River Laboratories, Hollister, CA). Whether the female was a
virgin (primigravida) or had a previous pregnancy (multigravida) was recorded.
Presence of a vaginal plug was denoted 0 day post coitum (dpc). Plugged females
were moved to a separate cage and weights taken at intervals to confirm
pregnancy. A few days prior to 19 dpc, the female was placed in a separate cage
and provided with nesting material (Enviro-driR, Shepherd Specialty
Papers) and a paper tube. Cages were checked for births twice daily (morning and
afternoon) starting from 18 dpc.

Quantifying dystocia and fetuses in utero.The day of parturition for
Setd3−/− females varied from 19
dpc to ≥20 dpc. Normal parturition was defined as birth at 19 days
post-coitum (dpc) and dystocia (delayed parturition) was defined as either no
births (≥20 dpc) or incomplete delivery with fetuses remaining in
utero (≥20 dpc). To determine whether
Setd3−/− females that delivered on
19 dpc still exhibited a delayed in parturition, females were euthanized on late
20 dpc if they did not deliver on 19 dpc
(Setd3−/−) or 24h after giving
birth if they delivered on 19 dpc
(Setd3−/−,
Setd3+/−,
Setd3+/+) and examined for
the presence of fetuses in utero. We note that the dystocia
occurred in both primigravida and multigravida
Setd3−/− females and that
actin-H73me is not detected in the uterus of late-stage pregnant
Setd3−/− females (data not
shown).

Quantifying dystocia and fetuses in utero.

The day of parturition for
Setd3−/− females varied from 19
dpc to ≥20 dpc. Normal parturition was defined as birth at 19 days
post-coitum (dpc) and dystocia (delayed parturition) was defined as either no
births (≥20 dpc) or incomplete delivery with fetuses remaining in
utero (≥20 dpc). To determine whether
Setd3−/− females that delivered on
19 dpc still exhibited a delayed in parturition, females were euthanized on late
20 dpc if they did not deliver on 19 dpc
(Setd3−/−) or 24h after giving
birth if they delivered on 19 dpc
(Setd3−/−,
Setd3+/−,
Setd3+/+) and examined for
the presence of fetuses in utero. We note that the dystocia
occurred in both primigravida and multigravida
Setd3−/− females and that
actin-H73me is not detected in the uterus of late-stage pregnant
Setd3−/− females (data not
shown).

Induction of parturition.Oxytocin (VetOneR, MWI Veterinary Supply Co.) was given
subcutaneously on 18 dpc to induce preterm labor (dose = 0.4 U in 0.9% saline
supplemented with 0.1% calcium gluconate and 0.025% dextrose, given 3x, q30 min
over a period of 1.5 h) to Setd3−/−,
and Setd3+/+timed-pregnant females.
The normal day of birth for
Setd3+/+mice of this strain is
19 dpc. All pups born on 18 dpc were removed and euthanized. All mice quantified
for fetuses in utero were euthanized at 1700h on d19.A PGF2α cocktail (1 ml LutalyseR (5 mg
PGF2α/ml), 1 ml 10% calcium gluconate, 0.05 ml of 50% isotonic dextrose,
and 8.0 ml of 0.9% sterile saline) was used to induce parturition in two
Setd3−/− females on 19 dpc.
Females were injected subcutaneously with 1 ml of the cocktail at 3 time points
on 19 dpc (~0900 h, ~1200 h, ~1700 h). The females were
monitored for parturition and euthanized on late 20 dpc at 1700 h.

Induction of parturition.

Oxytocin (VetOneR, MWI Veterinary Supply Co.) was given
subcutaneously on 18 dpc to induce preterm labor (dose = 0.4 U in 0.9% saline
supplemented with 0.1% calcium gluconate and 0.025% dextrose, given 3x, q30 min
over a period of 1.5 h) to Setd3−/−,
and Setd3+/+timed-pregnant females.
The normal day of birth for
Setd3+/+mice of this strain is
19 dpc. All pups born on 18 dpc were removed and euthanized. All mice quantified
for fetuses in utero were euthanized at 1700h on d19.

A PGF2α cocktail (1 ml LutalyseR (5 mg
PGF2α/ml), 1 ml 10% calcium gluconate, 0.05 ml of 50% isotonic dextrose,
and 8.0 ml of 0.9% sterile saline) was used to induce parturition in two
Setd3−/− females on 19 dpc.
Females were injected subcutaneously with 1 ml of the cocktail at 3 time points
on 19 dpc (~0900 h, ~1200 h, ~1700 h). The females were
monitored for parturition and euthanized on late 20 dpc at 1700 h.

Metabolic analysis.Amino acids were analyzed as underivatized compounds by liquid
chromatography tandem mass spectrometry (LC-MS/MS). Samples (50μL) were
deproteinized with 50μL of 6% sulfosalicylic acid, and centrifuged for 15
min at 17,000 RCF. Supernatants were diluted with an isotopic standard mixture
(Cambridge Isotope Laboratories, Inc) in 2 μM tridecafluoroheptanoic acid
(TDFHA), and then injected into the LC-MS/MS system. Samples were analyzed using
a clinically validated amino acid analysis method48. An Agilent 1200 series liquid
chromatography system equipped with a Thermo Hypercarb trap column (3 μm,
4.6 × 50 mm) and a Waters BEH C18 analytical column (2.5 μm, 2.1
× 100 mm) was used to separate compounds in a trap and reverse-elute
configuration. An Agilent 6460 triple quadrupole mass spectrometer was used in
positive polarity electrospray ionization for detection by dynamic multiple
reaction monitoring (MRM). Mobile phase A was 0.02% perfluoroheptanoic acid and
mobile phase B was acetonitrile. Standard curves were prepared by diluting amino
acid standards (Wako Chemicals USA, Inc); for 1-methylhistidine,
S-aminoethylcystine was used as the internal standard and for 3-methylhistidine,
histidine-13C615N3 was used as
the internal standard. Agilent Masshunter B08 was used for data analysis.

Metabolic analysis.

Amino acids were analyzed as underivatized compounds by liquid
chromatography tandem mass spectrometry (LC-MS/MS). Samples (50μL) were
deproteinized with 50μL of 6% sulfosalicylic acid, and centrifuged for 15
min at 17,000 RCF. Supernatants were diluted with an isotopic standard mixture
(Cambridge Isotope Laboratories, Inc) in 2 μM tridecafluoroheptanoic acid
(TDFHA), and then injected into the LC-MS/MS system. Samples were analyzed using
a clinically validated amino acid analysis method48. An Agilent 1200 series liquid
chromatography system equipped with a Thermo Hypercarb trap column (3 μm,
4.6 × 50 mm) and a Waters BEH C18 analytical column (2.5 μm, 2.1
× 100 mm) was used to separate compounds in a trap and reverse-elute
configuration. An Agilent 6460 triple quadrupole mass spectrometer was used in
positive polarity electrospray ionization for detection by dynamic multiple
reaction monitoring (MRM). Mobile phase A was 0.02% perfluoroheptanoic acid and
mobile phase B was acetonitrile. Standard curves were prepared by diluting amino
acid standards (Wako Chemicals USA, Inc); for 1-methylhistidine,
S-aminoethylcystine was used as the internal standard and for 3-methylhistidine,
histidine-13C615N3 was used as
the internal standard. Agilent Masshunter B08 was used for data analysis.

Muscle Tissue Histology.Tissues were fixed by immersion in 10% neutral buffered formalin,
routinely processed for paraffin embedding, and 5μm thick sections were
routinely stained with hematoxylin and eosin (H&E).

Muscle Tissue Histology.

Tissues were fixed by immersion in 10% neutral buffered formalin,
routinely processed for paraffin embedding, and 5μm thick sections were
routinely stained with hematoxylin and eosin (H&E).

Myosin expression and purificationConstruction, expression, and purification of the wild-type recombinant
human beta-cardiac myosin sS1 and the hypertrophic cardiac myopathy H251N
mutant49 are described
in detail elsewhere49-51. Briefly, a truncated version
of MYH7 (residues 1–808), corresponding to sS1, with a C-terminal
enhanced green fluorescent protein (eGFP), was co-expressed with myosin light
chain 3 (MYL3), encoding human ventricular essential light chain (ELC), and
containing an N-terminal FLAG tag (DYKDDDDK) and tobacco etch virus (TEV)
protease site in mouse myoblast C2C12 cells using the AdEasy Vector System
(Obiogene Inc.). The myosin heavy chain with its associated FLAG-tagged ELC was
first purified from clarified lysate with anti-FLAG resin (Sigma). After
cleaving off the FLAG tag with TEV protease, the human beta-cardiac sS1 was
further purified using anion exchange chromatography on a 1-mL HiTrap Q HP
column (GE Healthcare). Peak fractions were eluted with column buffer (10 mM
imidazole, pH 7.5, ~200 mM NaCl, 4 mM MgCl2, 1 mM DTT, 2 mM ATP, and 10%
sucrose) and concentrated by centrifugation in Amicon Ultra-0.5 10-kDa cutoff
spin filters (EMD Millipore) before being used in assays or stored at −80
°C.

Myosin expression and purification

Construction, expression, and purification of the wild-type recombinant
human beta-cardiac myosin sS1 and the hypertrophic cardiac myopathy H251N
mutant49 are described
in detail elsewhere49-51. Briefly, a truncated version
of MYH7 (residues 1–808), corresponding to sS1, with a C-terminal
enhanced green fluorescent protein (eGFP), was co-expressed with myosin light
chain 3 (MYL3), encoding human ventricular essential light chain (ELC), and
containing an N-terminal FLAG tag (DYKDDDDK) and tobacco etch virus (TEV)
protease site in mouse myoblast C2C12 cells using the AdEasy Vector System
(Obiogene Inc.). The myosin heavy chain with its associated FLAG-tagged ELC was
first purified from clarified lysate with anti-FLAG resin (Sigma). After
cleaving off the FLAG tag with TEV protease, the human beta-cardiac sS1 was
further purified using anion exchange chromatography on a 1-mL HiTrap Q HP
column (GE Healthcare). Peak fractions were eluted with column buffer (10 mM
imidazole, pH 7.5, ~200 mM NaCl, 4 mM MgCl2, 1 mM DTT, 2 mM ATP, and 10%
sucrose) and concentrated by centrifugation in Amicon Ultra-0.5 10-kDa cutoff
spin filters (EMD Millipore) before being used in assays or stored at −80
°C.

in vitro motility.Experiments were conducted following previously described
methods51-54. Briefly, frozen myosin protein was
first subjected to a “deadheading” procedure to remove inactive
heads. Myosin was mixed with 5 – 10x molar excess of unlabeled F-actin on
ice for 5 min, followed by addition of 2 mM ATP for 3 min, then centrifuged at
95k rpm in a TLA-100 rotor (Beckman Coulter) at 4°C for 20 min. The
supernatant was collected for use in the motility. Flow chamber was built on a
nitrocellulose-coated coverslip mounted on a glass slide. Deadheaded myosin was
immobilized via specific binding to the GFP antibody that non-specifically binds
to the coverslip surface, followed by blocking with assay buffer (25 mM
imidazole pH 7.5, 25 mM KCl, 4 mM MgCl2, 1 mM EGTA, 10 mM DTT, and 1
mg/ml BSA). A final solution containing fluorescently labeled bovine cardiac
F-actin, 2 mM ATP, and an oxygen-scavenging system (0.4% glucose, 0.11 mg/ml
glucose oxidase, and 0.018 mg/ml catalase) was subsequently flowed in before
imaging on a total internal reflection fluorescence microscope (Nikon Eclipse
Ti-E) equipped with a 561 nm laser (Cobolt) and a 100x oil-immersion objective
(Nikon)55. Time-lapse
movies were acquired at multiple fields of views for each experimental condition
at 1 Hz sampling rate. Movies were analyzed using the FAST (Fast Automated Spud
Trekker) software56 to obtain
the mean velocities with 20% tolerance. All experiments were conducted at
23°C.

in vitro motility.

Experiments were conducted following previously described
methods51-54. Briefly, frozen myosin protein was
first subjected to a “deadheading” procedure to remove inactive
heads. Myosin was mixed with 5 – 10x molar excess of unlabeled F-actin on
ice for 5 min, followed by addition of 2 mM ATP for 3 min, then centrifuged at
95k rpm in a TLA-100 rotor (Beckman Coulter) at 4°C for 20 min. The
supernatant was collected for use in the motility. Flow chamber was built on a
nitrocellulose-coated coverslip mounted on a glass slide. Deadheaded myosin was
immobilized via specific binding to the GFP antibody that non-specifically binds
to the coverslip surface, followed by blocking with assay buffer (25 mM
imidazole pH 7.5, 25 mM KCl, 4 mM MgCl2, 1 mM EGTA, 10 mM DTT, and 1
mg/ml BSA). A final solution containing fluorescently labeled bovine cardiac
F-actin, 2 mM ATP, and an oxygen-scavenging system (0.4% glucose, 0.11 mg/ml
glucose oxidase, and 0.018 mg/ml catalase) was subsequently flowed in before
imaging on a total internal reflection fluorescence microscope (Nikon Eclipse
Ti-E) equipped with a 561 nm laser (Cobolt) and a 100x oil-immersion objective
(Nikon)55. Time-lapse
movies were acquired at multiple fields of views for each experimental condition
at 1 Hz sampling rate. Movies were analyzed using the FAST (Fast Automated Spud
Trekker) software56 to obtain
the mean velocities with 20% tolerance. All experiments were conducted at
23°C.

Actin-activated Myosin ATPase activity.Purified actin was freshly cycled to F-actin by extensive (four times
over 4 d) dialysis in ATPase buffer (10 mM imidazole, pH 7.5, 5 mM
KCl, 3 mM MgCl2, and 1 mM DTT) to remove any residual ATP. The
monomeric concentration of F-actin was determined by measuring the absorbance of
a serial dilution of the actin in 6 M guanidine hydrochloride both at
290 nm with an extinction coefficient of
26,600 M−1cm−1 and at
280 nm with an extinction coefficient of
45,840 M−1cm−1 in a
spectrophotometer (NanoDrop). Full-length human gelsolin was added to actin at a
ratio of 1:1,000 to reduce the viscosity of the actin and thereby decrease
pipetting error at higher actin concentrations without affecting the ATPase
activity. The steady-state actin-activated ATPase activities of freshly prepared
human beta-cardiac sS1-eGFP myosin using WT and KO actin were determined using a
colorimetric readout of phosphate production57, as described in a previous work51. In this assay, reactions containing sS1
at a final concentration of 0.01 mg/ml, 2 mM ATP, and actin at
concentrations ranging from 5 to 77 μM were performed at
23 °C with shaking using a microplate spectrophotometer (Thermo
Scientific Multiskan GO). The rate of sS1 activity was obtained by linear
fitting the phosphate signal as a function of time and converted to activity
units using a phosphate standard. Each actin condition was performed in
duplicate, The error on each data point represents the s.e.m. of the duplicates.
The Michaelis–Menten equation was fitted to determine the maximal
activity (kcat) and the actin concentration at half-maximum (apparent Km for
actin).

Actin-activated Myosin ATPase activity.

Purified actin was freshly cycled to F-actin by extensive (four times
over 4 d) dialysis in ATPase buffer (10 mM imidazole, pH 7.5, 5 mM
KCl, 3 mM MgCl2, and 1 mM DTT) to remove any residual ATP. The
monomeric concentration of F-actin was determined by measuring the absorbance of
a serial dilution of the actin in 6 M guanidine hydrochloride both at
290 nm with an extinction coefficient of
26,600 M−1cm−1 and at
280 nm with an extinction coefficient of
45,840 M−1cm−1 in a
spectrophotometer (NanoDrop). Full-length human gelsolin was added to actin at a
ratio of 1:1,000 to reduce the viscosity of the actin and thereby decrease
pipetting error at higher actin concentrations without affecting the ATPase
activity. The steady-state actin-activated ATPase activities of freshly prepared
human beta-cardiac sS1-eGFP myosin using WT and KO actin were determined using a
colorimetric readout of phosphate production57, as described in a previous work51. In this assay, reactions containing sS1
at a final concentration of 0.01 mg/ml, 2 mM ATP, and actin at
concentrations ranging from 5 to 77 μM were performed at
23 °C with shaking using a microplate spectrophotometer (Thermo
Scientific Multiskan GO). The rate of sS1 activity was obtained by linear
fitting the phosphate signal as a function of time and converted to activity
units using a phosphate standard. Each actin condition was performed in
duplicate, The error on each data point represents the s.e.m. of the duplicates.
The Michaelis–Menten equation was fitted to determine the maximal
activity (kcat) and the actin concentration at half-maximum (apparent Km for
actin).

Statistics.GraphPad Prism 7.0 or KaldeidaGraph 4.5 software were used for
statistics.Data availability.The X-ray structures (coordinates and structure factor files) of
SETD3 with bound actin peptide have been submitted to PDB under accession
number 6MBJ (P21), 6MBK (Two complexes in P212121) and 6MBL (one complex in
P212121). Source data for this study are provided as supplementary information
(Fig. 4d, 5b, 5e, 5f, 5i, 5k; Extended Data Fig. 7e, 9c, 9h).
Source gel data provided for cropped images (Fig. 1a, 1d, 1e, 2a, 2c, 3a-f,g, 4c, 5g, 5j; Extended Data Fig. 1b)
is provided in Supplementary Figure 1. Mass spectrometry data associated with
Fig. 1c/Extended Data Fig. 1h and Fig. 3i are provided as a Supplementary Tables.
Additional requests can be made to the corresponding authors.

Statistics.

GraphPad Prism 7.0 or KaldeidaGraph 4.5 software were used for
statistics.

Data availability.The X-ray structures (coordinates and structure factor files) of
SETD3 with bound actin peptide have been submitted to PDB under accession
number 6MBJ (P21), 6MBK (Two complexes in P212121) and 6MBL (one complex in
P212121). Source data for this study are provided as supplementary information
(Fig. 4d, 5b, 5e, 5f, 5i, 5k; Extended Data Fig. 7e, 9c, 9h).
Source gel data provided for cropped images (Fig. 1a, 1d, 1e, 2a, 2c, 3a-f,g, 4c, 5g, 5j; Extended Data Fig. 1b)
is provided in Supplementary Figure 1. Mass spectrometry data associated with
Fig. 1c/Extended Data Fig. 1h and Fig. 3i are provided as a Supplementary Tables.
Additional requests can be made to the corresponding authors.

Data availability.

The X-ray structures (coordinates and structure factor files) of
SETD3 with bound actin peptide have been submitted to PDB under accession
number 6MBJ (P21), 6MBK (Two complexes in P212121) and 6MBL (one complex in
P212121). Source data for this study are provided as supplementary information
(Fig. 4d, 5b, 5e, 5f, 5i, 5k; Extended Data Fig. 7e, 9c, 9h).
Source gel data provided for cropped images (Fig. 1a, 1d, 1e, 2a, 2c, 3a-f,g, 4c, 5g, 5j; Extended Data Fig. 1b)
is provided in Supplementary Figure 1. Mass spectrometry data associated with
Fig. 1c/Extended Data Fig. 1h and Fig. 3i are provided as a Supplementary Tables.
Additional requests can be made to the corresponding authors.

Extended DataExtended Data Figure 1.Identification of actin as a SETD3 substrate.a, Top: domain structure of SETD3
containing an N-terminal SET domain and a C-terminal domain homologous to
plant Rubisco LSMT. Bottom: alignment of homologous
methyltransferases human SETD3 and human SETD6. Red box, putative catalytic
tyrosine. b, Methylation reactions as in Fig. 1a with non-radiolabeled SAM and analyzed by
Western blot with indicated antibodies. Total H3 shown as a loading control.
c, SETD3 localizes to the cytoplasm. Representative
immunofluorescence images of GFP or GFP-SETD3 localization in HeLa cells
(left) are merged with DAPI counterstaining (right). Scale bar, 7 μm.
d-g, Biochemical enrichment of a candidate SETD3 substrate.
d, Schematic of biochemical strategy to identify methylated
band indicated in Fig. 1b.
e, in vitro methylation reactions using
cell extracts separated by size-exclusion chromatography as a substrate.
Reactions were performed with either wild-type (WT) SETD3 or a putative
catalytic mutant (Y312A). Reactions were analyzed as in Fig. 1. Arrowhead, candidate substrate.
f, Ion-exchange chromatography separates candidate
substrates. Fractions positive for SETD3-specific methylation by
size-exclusion chromatography were further separated by ion-exchange
chromatography and either the flowthrough (FT) or pooled fractions
containing a significantly silver-stained band (arrowhead) at the size of
the candidate substrate were used as substrate in an in
vitro methylation reaction. g, Flowthrough (FT) or
pooled eluent from ion-exchange chromatography were used as substrate for
methylation reactions as in e. Arrowhead, protein band in
eluent that was analyzed by mass spectrometry. h, Top candidate
substrates with molecular weight (MW) similar to the size of SETD3-dependent
band from in vitro reactions with cell extracts. Candidates
identified by mass spectrometry are ranked by abundance determined by MS/MS
count (Supplementary Table
1). i,
in vitro methylation reactions with SETD3 on actin,
recombinant histone H3 (rH3), FOXM1, or a no substrate as a control. Top
panel, 3H-SAM is the methyl donor and methylation visualized by
autoradiography. Bottom panel, Coomassie staining of proteins in the
reaction. j,
in vitro methylation reactions with the indicated enzymes
and substrates. All experiments were repeated at least three times with
similar results.Extended Data Figure 2.SETD3 does not methylate beta-actin lysines.GST-beta-actin was expressed in bacteria, and cleaved using
Prescission protease. Cleaved beta-actin was used in an in
vitro methylation assay with SETD3 and deuterated SAM. Spectra
are representative from experiments independently performed three times with
similar results. MS/MS spectra with identified ions for unmethylated lysine
residues produced with indicated proteases: a, K18 (trypsin)
(Identified N-terminal GPLGS amino acids are residual, vector specific amino
acids from the original GST fusion protein). b, K50 (trypsin);
c, K61 (trypsin); d, K68 (trypsin);
e, K84 (chymotrypsin); f, K113 (trypsin);
g, K118 (Glu-C); h, K191 (trypsin);
i, K213 (trypsin); j, K215 (Glu-C);
k, K238 (chymotrypsin); l, K284/K291 (chymotrypsin);
m, K315/K326 (trypsin); n, K326/K328
(trypsin); o, K336 (trypsin); p, K373
(trypsin).Extended Data Figure 3.SETD3 generates actin-H73(3-me).a-c, Tandem MS/MS identifies H73 of actin as SETD3
methylated residue. a,
Top: Methylated tryptic peptide with indicated b and y
ions. Modifications: me, methylation; ox, oxidation.
Bottom: Table of m/z for b and y ions
identified in MS/MS spectra from methylation reactions with (+SETD3) or
without (-SETD3) using deuterated SAM as the methyl donor (+17.03 Da mass
shift). *, peptides containing H73. b-c, in
vitro methylation reactions with (b) or without
(c, negative control) SETD3 and beta-actin with deuterated
S-adenosyl-methionione (SAM) were analyzed by mass
spectrometry. Left, MS/MS spectra of tryptic peptides
containing H73. Right, b and y ions identified of oxidized
(ox) peptides in each spectrum are indicated. me, methylation.
d, Dot blot loading control for Fig. 2c of biotinylated ACTB peptides (amino acids
66-80) that are unmodified at H73 or methylated in the N3
(3-me) or N1 (1-me) position. Serial dilutions of peptide
were visualized by streptavidin-HRP and chemiluminescence. e-g,
Methylation reaction kinetics for SETD3 and actin peptide. e,
Time-course of SETD3 with His(1-me) or His(3-me) modified peptides as
substrate. Inset: unmodified peptide using 80-fold lower
enzyme concentration as that used for modified peptides. f-g,
Steady-state kinetics of SETD3 for f, human beta-actin peptide
(residues 66-80) and g, SAM. Kinetic measurements were
performed using a bioluminescent methyltransferase assay, MTase-Glo™
(Promega). Means from biological triplicates for each data point are
reported and error bars indicate standard deviation. a-g, Data
are representative of three independent experiments with similar
results.Extended Data Figure 4.Methylation of the conserved actin-H73 residue is present in diverse
organisms.a, Summary of actin histidine methylation on the
conserved H73 residue among model organisms. Abundance of methylation is
reported as a percent of peptide that is methylated. b,
Chromatograms for MH3+ ions (m/z ± 10
ppm) for methylated and unmethylated versions of the indicated peptides. The
unmethylated actin peptide that was analyzed for each species is shown above
the corresponding chromatograms (ox, oxidation). When alpha-actin peptides
were detected, associated chromatograms and quantification are provided on
right. The area of indicated peaks was normalized to the sum area between
peptides (± methylation) and percent abundance is labeled.
Chromatograms quantifying actin histidine methylation in human and mouse can
be found in Extended Data Fig. 6 and
7. a-b, N.D., not
detected. Quantification represents data from two independent experiments
with similar results.Extended Data Figure 5.Structural details of SETD3-actin peptide interactions.a, Overall structure of SETD3 with a V-shaped cleft
constructed by the SET domain (green) and an LSMT-like domain (cyan).
Helices are shown as cylinders and strands as arrows. Actin peptide shown as
a stick model. b, View of the target histidine through the
channel from the SAH binding pocket. c, Omit electron density
of Fo-Fc, contoured at 3.0σ above the mean, is shown for omitting
cofactor SAH and the actin peptide used for co-crystallization.
d, Details of inter- and intramolecular interactions
between SETD3 (green) and actin peptide (yellow). e, W79 of
actin bound in a hydrophobic surface pocket of SETD3. f, T77
and N78 of actin form hydrogen bonds with N153 and Q254 of SETD3.
g, V76 of actin is in van der Waals contact with SETD3
residues H323 and R315, which in turn interacts with E72 of actin.
h, I75 of actin is in van der Waals contact with main-chain
Cα of N255 and Q256 of SETD3. i, G74 of actin is located
in the amino end of a SETD3 helix. The imidazole ring of H73 (the substrate
target) is parallel with the aromatic ring of Y312 of SETD3. j,
SETD3 R315 bridges between the carboxylate oxygen of E72 and the main-chain
carbonyl oxygen of H73 of actin. k, I71 of actin is
accommodated in a surface hydrophobic pocket of SETD3. l, Y69
and P70 of actin interact with a stretch of SETD3 residues from I283 to
L289. m, Y69 of actin packs against P258 of SETD3 and K68 of
actin interacts with E290 of SETD3. n, L67 and P70 of actin
form an intra-molecular interaction and both interact with I283 and T284 of
SETD3. o, Alignment of the amino acids from all six human actin
isoforms corresponding to beta-actin amino acids 66-80. Variant amino acid
highlighted in yellow. p, The cofactor (SAH) binding site
includes residues R253, Y312, N277 and H278 of SETD3.Extended Data Figure 6.SETD3 is required for actin methylation in cells.a, Peptide dot blot spotted with biotinylated
beta-actin peptides (amino acids 66-80) containing H73, H73(3-me), or
H73(1-me). Blots were probed with H73(3-me)-specific antibody or
streptavidin as a loading control. b-d, Chromatograms for
quantification of actin-H73me in human cells. Stoichiometry of oxidized
beta/gamma-actin H73 peptide (YPIEHGIVTNWDDM(ox)EK) with or without
methylation after purification from cells. MH3+
m/z: unmethylated, 654.968 ± 10 ppm; methylated,
659.6401 ± 10 ppm. Alpha-actin peptide was not detected in these
cells. Quantification performed as in Extended
Data Fig. 4. b, Chromatograms for HT1080 cells
treated with CRISPR/Cas9 that is targeted with a control sgRNA or
SETD3-specific sgRNAs. c, Chromatograms for actin H73
methylation in wild-type (WT) HeLa cells or clonal HeLa SETD3 knockout (KO)
cells. N.D., not detected. d, Chromatograms of actin H73
methylation in HT1080 cells treated with CRISPR/Cas9 targeted with a either
a control or SETD3-specific sgRNA and complemented with CRISPR-resistant
SETD3WT, SETD3NHY or control plasmids.
a-d, Experiments were performed three independent times
with similar results.Extended Data Figure 7.SETD3 is required for actin-H73 methylation in mice.a-d, Chromatograms for quantification of actin
histidine methylation in mouse tissues. a, Chromatograms to
determine abundance of histidine methylation from beta/gamma-actin H73
peptide (YPIEHGIVTNWDDM(ox)EK) with or without methylation after
purification from brain tissue of mice with the indicated genotypes.
MH3+
m/z: unmethylated, 654.968 ± 10 ppm; methylated,
659.640 ± 10 ppm. Alpha-actin peptide was not detected in these
cells. Quantification performed as in Extended
Data Fig. 4. b, Chromatograms to determine abundance
of histidine methylation on alpha-actin H75 peptide (YPIEHGIITNWDDM(ox)EK)
from skeletal muscle as in a. MH3+
m/z: unmethylated, 659.640 ± 10 ppm; methylated,
664.312 ± 10 ppm. Beta/gamma-actin peptide was not detected in these
cells. c, Chromatograms to determine abundance of
beta/gamma-actin H73 methylation from uterine tissue as in a.
d, Chromatograms to determine abundance of alpha-actin H75
methylation from uterine tissue as in b. a-d,
N.D., not detected. Quantification was performed three independent times
with similar results. e, Table of quantitative amino acid panel
from murine blood serum. Quantitative profiling of amino acids in plasma is
used clinically to diagnose metabolic disorders. 3-methyl histidine (3MH)
(indicated by asterisk) is one of the amino acids measured in the panel, and
actin is thought to be the primary source of this metabolite.
Setd3−/− (KO,
n = 8) indicated amino acid levels were normalized to
levels of amino acids from animals with normal H73 methylation levels (Total
n = 12;
Setd3+/+ (WT),
n = 5; Setd3+/−
(HET), n = 7). Standard error of the difference between two
means is shown for the indicated n.Extended Data Figure 8.SETD3-dependent actin-H73 methylation modestly regulates
polymerization.a-b, Purification of actin with and without H73
methylation. a, Coomassie stained gel of the actin purified
from HeLa cell lines described in Fig.
3f, Actin(+me) or Actin(-me), and used in biochemical assays
described in Fig. 4a-b and c-e. b, Mass spectrometry
quantification of actin H73 methylation from a. N.D., not
detected. a-b, Representative data from an experiment performed
at least three times with similar results. c, Methylation does
not alter actin depolymerization rates. Actin polymerized as in Fig. 4a was diluted to 0.02 μM and
depolymerization was monitored by fluorescence normalized to initial values.
d, Elongation of the indicated monomeric actin (1
μM, 0.1 μM pyrene-actin) measured in the presence of 2
μM phalloidin-actin seeds made with methylated (circles) or
unmethylated (triangles). e, Arp2/3-induced actin
polymerization performed in the presence of 100 nM WASP VCA and 5 nM Arp2/3
complex with 1 μM of the indicated monomeric actin and 0.1 μM
pyrene-actin. c-e, Mean values plotted with s.e.m. from three
independent biological replicates. f, SETD3 promotes cell
migration. Representative images of cell migration assays performed three
times with similar results using cells from Fig. 4c. A circular void of cells was created at the start of
the assay (0h, dashed red circle). After 24h of migration, cells were fixed
and stained with DAPI. Scale bar, 100 μm.Extended Data Figure 9.Analysis of SETD3 and actin-H73me in parturition and uterine smooth
muscle contraction.a-b, Histology of muscle tissues from
Setd3+/+ and
Setd3−/− mice. a,
Hematoxylin and eosin staining of aorta, colon, heart, tongue, and hind limb
muscle from Setd3+/+ and
Setd3−/− mice. b,
Tongue and hind limb (striated muscle) sections from a were
re-imaged without a condenser to highlight the sarcoplasmic striations
characteristic of this muscle type. a-b, Scale bar: 20
μm. Images from three independent experiments gave similar results.
c, Quantification of pups per litter for
Setd3+/+
(n = 12), Setd3+/−
(n = 43),
Setd3−/− (n
=26) mothers. ***, p-value = <0.001.
d-e, Labor induction at 19 dpc does not rescue dystocia of
Setd3−/− pregnant mice.
d, Schematic of prostaglandin
treatment protocol. PGF2α cocktail was administered at 0, 3, and 8
hours on day 19 with euthanasia and quantification of fetuses at 32h after
first treatment. e, Table quantifying births and postterm
fetuses in utero for the indicated genotypes at the time
shown in the schematic. Note: controls delivered before treatment commenced.
f, Collagen contraction assay as in Fig. 5h with indicated reconstitution cell lines
from Fig. 5j performed with three
independent biological replicates. g-h, Actin-H73me does not
notably alter myosin activity. g,
in vitro actin motility assay. Gliding velocities of actin
filaments prepared from actin described in Fig. 4 were measured using human β-cardiac sS1 myosin or
a hyperactive mutant (H251N). Data are mean velocities with s.e.m. from four
different experiments. h, Actin-activated ATPase of human
β-cardiac sS1 using actin as in g. The Michaelis-Menten
equation is fit to the ATPase data for WT actin (solid line) and KO actin
(dashed line). Points are means from two independent experiments. For gel
source data, see Supplementary Fig. 1.Extended Data Table 1.Summary of X-ray data collection from SERCAT beamline (22-ID) at
wavelength = 1Å and refinement statistics. (*)SETD3Actin
peptideSAHActin
peptideSAHActin
peptideSAHPDB Code6MBJ6MBK6MBL
Data Collection
Space
groupP21P212121P212121Cell
dimensions (Å)60.35,
176.17, 66.5860.28,
115.97, 173.5956.56,
92.83, 111.64α, β, γ (°)90,
92.9, 9090, 90,
9090, 90,
90Resolution (Å)39.07-1.79 (1.85-1.79)39.38-1.69 (1.75-1.69)48.30-2.19 (2.27-2.19)a Rmerge0.146 (0.892)0.129 (0.800)0.125 (0.741)Rpim0.041 (0.436)0.061 (0.592)0.066 (0.548)CC1/2, CC(0.463, 0.795)(0.720, 0.915)(0.402, 0.757)b <I/σI>14.5 (1.0)11.3 (1.7)14.1 (1.5)Completeness (%)94.5 (78.9)99.2 (98.4)99.5 (96.7)Redundancy13.6 (8.0)19.2 (8.9)10.6 (4.7)Observed reflections1,695,3472,597,462323,060Unique
reflections124,387
(10,380)135,382
(13,262)30,434
(2920)
Refinement
Resolution (Å)1.791.692.19No. reflections123,743135,11030,334c Rwork /
d Rfree0.199 / 0.2230.228 / 0.2430.182 / 0.227No. AtomsTwo complexesTwo complexesOne complexProtein782479533923Peptide248252124SAH525226Solvent777933179B-factors (Å2)Protein34.329.647.3Peptide41.434.542.3SAH18.016.732.0Solvent45.239.946.5
R.m.s. deviations
Bond lengths (Å)0.0030.0100.004Bond angles (°)0.5640.9420.683Values in parenthesis correspond to highest resolution
shell.a, Rmerge = Σ ∣ I -
<I>∣ /Σ I, where I is the observed intensity
and <I> is the averaged intensity from multiple
observations.b, <I/σI> = averaged ratio of the intensity
(I) to the error of the intensity (σI).c, Rwork = Σ ∣ Fobs - Fcal ∣
/Σ ∣ Fobs ∣, where Fobs and Fcal are the observed
and calculated structure factors, respectively.d, Rfree was calculated using a randomly chosen subset
(5%) of the reflections not used in refinement.

Extended Data

Identification of actin as a SETD3 substrate.

a, Top: domain structure of SETD3
containing an N-terminal SET domain and a C-terminal domain homologous to
plant Rubisco LSMT. Bottom: alignment of homologous
methyltransferases human SETD3 and human SETD6. Red box, putative catalytic
tyrosine. b, Methylation reactions as in Fig. 1a with non-radiolabeled SAM and analyzed by
Western blot with indicated antibodies. Total H3 shown as a loading control.
c, SETD3 localizes to the cytoplasm. Representative
immunofluorescence images of GFP or GFP-SETD3 localization in HeLa cells
(left) are merged with DAPI counterstaining (right). Scale bar, 7 μm.
d-g, Biochemical enrichment of a candidate SETD3 substrate.
d, Schematic of biochemical strategy to identify methylated
band indicated in Fig. 1b.
e, in vitro methylation reactions using
cell extracts separated by size-exclusion chromatography as a substrate.
Reactions were performed with either wild-type (WT) SETD3 or a putative
catalytic mutant (Y312A). Reactions were analyzed as in Fig. 1. Arrowhead, candidate substrate.
f, Ion-exchange chromatography separates candidate
substrates. Fractions positive for SETD3-specific methylation by
size-exclusion chromatography were further separated by ion-exchange
chromatography and either the flowthrough (FT) or pooled fractions
containing a significantly silver-stained band (arrowhead) at the size of
the candidate substrate were used as substrate in an in
vitro methylation reaction. g, Flowthrough (FT) or
pooled eluent from ion-exchange chromatography were used as substrate for
methylation reactions as in e. Arrowhead, protein band in
eluent that was analyzed by mass spectrometry. h, Top candidate
substrates with molecular weight (MW) similar to the size of SETD3-dependent
band from in vitro reactions with cell extracts. Candidates
identified by mass spectrometry are ranked by abundance determined by MS/MS
count (Supplementary Table
1). i,
in vitro methylation reactions with SETD3 on actin,
recombinant histone H3 (rH3), FOXM1, or a no substrate as a control. Top
panel, 3H-SAM is the methyl donor and methylation visualized by
autoradiography. Bottom panel, Coomassie staining of proteins in the
reaction. j,
in vitro methylation reactions with the indicated enzymes
and substrates. All experiments were repeated at least three times with
similar results.

SETD3 does not methylate beta-actin lysines.

GST-beta-actin was expressed in bacteria, and cleaved using
Prescission protease. Cleaved beta-actin was used in an in
vitro methylation assay with SETD3 and deuterated SAM. Spectra
are representative from experiments independently performed three times with
similar results. MS/MS spectra with identified ions for unmethylated lysine
residues produced with indicated proteases: a, K18 (trypsin)
(Identified N-terminal GPLGS amino acids are residual, vector specific amino
acids from the original GST fusion protein). b, K50 (trypsin);
c, K61 (trypsin); d, K68 (trypsin);
e, K84 (chymotrypsin); f, K113 (trypsin);
g, K118 (Glu-C); h, K191 (trypsin);
i, K213 (trypsin); j, K215 (Glu-C);
k, K238 (chymotrypsin); l, K284/K291 (chymotrypsin);
m, K315/K326 (trypsin); n, K326/K328
(trypsin); o, K336 (trypsin); p, K373
(trypsin).

SETD3 generates actin-H73(3-me).

a-c, Tandem MS/MS identifies H73 of actin as SETD3
methylated residue. a,
Top: Methylated tryptic peptide with indicated b and y
ions. Modifications: me, methylation; ox, oxidation.
Bottom: Table of m/z for b and y ions
identified in MS/MS spectra from methylation reactions with (+SETD3) or
without (-SETD3) using deuterated SAM as the methyl donor (+17.03 Da mass
shift). *, peptides containing H73. b-c, in
vitro methylation reactions with (b) or without
(c, negative control) SETD3 and beta-actin with deuterated
S-adenosyl-methionione (SAM) were analyzed by mass
spectrometry. Left, MS/MS spectra of tryptic peptides
containing H73. Right, b and y ions identified of oxidized
(ox) peptides in each spectrum are indicated. me, methylation.
d, Dot blot loading control for Fig. 2c of biotinylated ACTB peptides (amino acids
66-80) that are unmodified at H73 or methylated in the N3
(3-me) or N1 (1-me) position. Serial dilutions of peptide
were visualized by streptavidin-HRP and chemiluminescence. e-g,
Methylation reaction kinetics for SETD3 and actin peptide. e,
Time-course of SETD3 with His(1-me) or His(3-me) modified peptides as
substrate. Inset: unmodified peptide using 80-fold lower
enzyme concentration as that used for modified peptides. f-g,
Steady-state kinetics of SETD3 for f, human beta-actin peptide
(residues 66-80) and g, SAM. Kinetic measurements were
performed using a bioluminescent methyltransferase assay, MTase-Glo™
(Promega). Means from biological triplicates for each data point are
reported and error bars indicate standard deviation. a-g, Data
are representative of three independent experiments with similar
results.

Methylation of the conserved actin-H73 residue is present in diverse
organisms.

a, Summary of actin histidine methylation on the
conserved H73 residue among model organisms. Abundance of methylation is
reported as a percent of peptide that is methylated. b,
Chromatograms for MH3+ ions (m/z ± 10
ppm) for methylated and unmethylated versions of the indicated peptides. The
unmethylated actin peptide that was analyzed for each species is shown above
the corresponding chromatograms (ox, oxidation). When alpha-actin peptides
were detected, associated chromatograms and quantification are provided on
right. The area of indicated peaks was normalized to the sum area between
peptides (± methylation) and percent abundance is labeled.
Chromatograms quantifying actin histidine methylation in human and mouse can
be found in Extended Data Fig. 6 and
7. a-b, N.D., not
detected. Quantification represents data from two independent experiments
with similar results.

Structural details of SETD3-actin peptide interactions.

a, Overall structure of SETD3 with a V-shaped cleft
constructed by the SET domain (green) and an LSMT-like domain (cyan).
Helices are shown as cylinders and strands as arrows. Actin peptide shown as
a stick model. b, View of the target histidine through the
channel from the SAH binding pocket. c, Omit electron density
of Fo-Fc, contoured at 3.0σ above the mean, is shown for omitting
cofactor SAH and the actin peptide used for co-crystallization.
d, Details of inter- and intramolecular interactions
between SETD3 (green) and actin peptide (yellow). e, W79 of
actin bound in a hydrophobic surface pocket of SETD3. f, T77
and N78 of actin form hydrogen bonds with N153 and Q254 of SETD3.
g, V76 of actin is in van der Waals contact with SETD3
residues H323 and R315, which in turn interacts with E72 of actin.
h, I75 of actin is in van der Waals contact with main-chain
Cα of N255 and Q256 of SETD3. i, G74 of actin is located
in the amino end of a SETD3 helix. The imidazole ring of H73 (the substrate
target) is parallel with the aromatic ring of Y312 of SETD3. j,
SETD3 R315 bridges between the carboxylate oxygen of E72 and the main-chain
carbonyl oxygen of H73 of actin. k, I71 of actin is
accommodated in a surface hydrophobic pocket of SETD3. l, Y69
and P70 of actin interact with a stretch of SETD3 residues from I283 to
L289. m, Y69 of actin packs against P258 of SETD3 and K68 of
actin interacts with E290 of SETD3. n, L67 and P70 of actin
form an intra-molecular interaction and both interact with I283 and T284 of
SETD3. o, Alignment of the amino acids from all six human actin
isoforms corresponding to beta-actin amino acids 66-80. Variant amino acid
highlighted in yellow. p, The cofactor (SAH) binding site
includes residues R253, Y312, N277 and H278 of SETD3.

SETD3 is required for actin methylation in cells.

a, Peptide dot blot spotted with biotinylated
beta-actin peptides (amino acids 66-80) containing H73, H73(3-me), or
H73(1-me). Blots were probed with H73(3-me)-specific antibody or
streptavidin as a loading control. b-d, Chromatograms for
quantification of actin-H73me in human cells. Stoichiometry of oxidized
beta/gamma-actin H73 peptide (YPIEHGIVTNWDDM(ox)EK) with or without
methylation after purification from cells. MH3+
m/z: unmethylated, 654.968 ± 10 ppm; methylated,
659.6401 ± 10 ppm. Alpha-actin peptide was not detected in these
cells. Quantification performed as in Extended
Data Fig. 4. b, Chromatograms for HT1080 cells
treated with CRISPR/Cas9 that is targeted with a control sgRNA or
SETD3-specific sgRNAs. c, Chromatograms for actin H73
methylation in wild-type (WT) HeLa cells or clonal HeLa SETD3 knockout (KO)
cells. N.D., not detected. d, Chromatograms of actin H73
methylation in HT1080 cells treated with CRISPR/Cas9 targeted with a either
a control or SETD3-specific sgRNA and complemented with CRISPR-resistant
SETD3WT, SETD3NHY or control plasmids.
a-d, Experiments were performed three independent times
with similar results.

SETD3 is required for actin-H73 methylation in mice.

a-d, Chromatograms for quantification of actin
histidine methylation in mouse tissues. a, Chromatograms to
determine abundance of histidine methylation from beta/gamma-actin H73
peptide (YPIEHGIVTNWDDM(ox)EK) with or without methylation after
purification from brain tissue of mice with the indicated genotypes.
MH3+
m/z: unmethylated, 654.968 ± 10 ppm; methylated,
659.640 ± 10 ppm. Alpha-actin peptide was not detected in these
cells. Quantification performed as in Extended
Data Fig. 4. b, Chromatograms to determine abundance
of histidine methylation on alpha-actin H75 peptide (YPIEHGIITNWDDM(ox)EK)
from skeletal muscle as in a. MH3+
m/z: unmethylated, 659.640 ± 10 ppm; methylated,
664.312 ± 10 ppm. Beta/gamma-actin peptide was not detected in these
cells. c, Chromatograms to determine abundance of
beta/gamma-actin H73 methylation from uterine tissue as in a.
d, Chromatograms to determine abundance of alpha-actin H75
methylation from uterine tissue as in b. a-d,
N.D., not detected. Quantification was performed three independent times
with similar results. e, Table of quantitative amino acid panel
from murine blood serum. Quantitative profiling of amino acids in plasma is
used clinically to diagnose metabolic disorders. 3-methyl histidine (3MH)
(indicated by asterisk) is one of the amino acids measured in the panel, and
actin is thought to be the primary source of this metabolite.
Setd3−/− (KO,
n = 8) indicated amino acid levels were normalized to
levels of amino acids from animals with normal H73 methylation levels (Total
n = 12;
Setd3+/+ (WT),
n = 5; Setd3+/−
(HET), n = 7). Standard error of the difference between two
means is shown for the indicated n.

SETD3-dependent actin-H73 methylation modestly regulates
polymerization.

a-b, Purification of actin with and without H73
methylation. a, Coomassie stained gel of the actin purified
from HeLa cell lines described in Fig.
3f, Actin(+me) or Actin(-me), and used in biochemical assays
described in Fig. 4a-b and c-e. b, Mass spectrometry
quantification of actin H73 methylation from a. N.D., not
detected. a-b, Representative data from an experiment performed
at least three times with similar results. c, Methylation does
not alter actin depolymerization rates. Actin polymerized as in Fig. 4a was diluted to 0.02 μM and
depolymerization was monitored by fluorescence normalized to initial values.
d, Elongation of the indicated monomeric actin (1
μM, 0.1 μM pyrene-actin) measured in the presence of 2
μM phalloidin-actin seeds made with methylated (circles) or
unmethylated (triangles). e, Arp2/3-induced actin
polymerization performed in the presence of 100 nM WASP VCA and 5 nM Arp2/3
complex with 1 μM of the indicated monomeric actin and 0.1 μM
pyrene-actin. c-e, Mean values plotted with s.e.m. from three
independent biological replicates. f, SETD3 promotes cell
migration. Representative images of cell migration assays performed three
times with similar results using cells from Fig. 4c. A circular void of cells was created at the start of
the assay (0h, dashed red circle). After 24h of migration, cells were fixed
and stained with DAPI. Scale bar, 100 μm.

Analysis of SETD3 and actin-H73me in parturition and uterine smooth
muscle contraction.

a-b, Histology of muscle tissues from
Setd3+/+ and
Setd3−/− mice. a,
Hematoxylin and eosin staining of aorta, colon, heart, tongue, and hind limb
muscle from Setd3+/+ and
Setd3−/− mice. b,
Tongue and hind limb (striated muscle) sections from a were
re-imaged without a condenser to highlight the sarcoplasmic striations
characteristic of this muscle type. a-b, Scale bar: 20
μm. Images from three independent experiments gave similar results.
c, Quantification of pups per litter for
Setd3+/+
(n = 12), Setd3+/−
(n = 43),
Setd3−/− (n
=26) mothers. ***, p-value = <0.001.
d-e, Labor induction at 19 dpc does not rescue dystocia of
Setd3−/− pregnant mice.
d, Schematic of prostaglandin
treatment protocol. PGF2α cocktail was administered at 0, 3, and 8
hours on day 19 with euthanasia and quantification of fetuses at 32h after
first treatment. e, Table quantifying births and postterm
fetuses in utero for the indicated genotypes at the time
shown in the schematic. Note: controls delivered before treatment commenced.
f, Collagen contraction assay as in Fig. 5h with indicated reconstitution cell lines
from Fig. 5j performed with three
independent biological replicates. g-h, Actin-H73me does not
notably alter myosin activity. g,
in vitro actin motility assay. Gliding velocities of actin
filaments prepared from actin described in Fig. 4 were measured using human β-cardiac sS1 myosin or
a hyperactive mutant (H251N). Data are mean velocities with s.e.m. from four
different experiments. h, Actin-activated ATPase of human
β-cardiac sS1 using actin as in g. The Michaelis-Menten
equation is fit to the ATPase data for WT actin (solid line) and KO actin
(dashed line). Points are means from two independent experiments. For gel
source data, see Supplementary Fig. 1.

Summary of X-ray data collection from SERCAT beamline (22-ID) at
wavelength = 1Å and refinement statistics. (*)

Values in parenthesis correspond to highest resolution
shell.

, Rmerge = Σ ∣ I -
<I>∣ /Σ I, where I is the observed intensity
and <I> is the averaged intensity from multiple
observations.

, <I/σI> = averaged ratio of the intensity
(I) to the error of the intensity (σI).

, Rwork = Σ ∣ Fobs - Fcal ∣
/Σ ∣ Fobs ∣, where Fobs and Fcal are the observed
and calculated structure factors, respectively.

, Rfree was calculated using a randomly chosen subset
(5%) of the reflections not used in refinement.

Supplementary Material12Supp Table 1Supp Table 2Supp fig 1Supplementary Data

Supplementary Material
