---
pmid: '33864814'
title: SAGA and SAGA-like SLIK transcriptional coactivators are structurally and biochemically
  equivalent.
authors:
- Adamus K
- Reboul C
- Voss J
- Huang C
- Schittenhelm RB
- Le SN
- Ellisdon AM
- Elmlund H
- Boudes M
- Elmlund D
journal: J Biol Chem
year: '2021'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC8131915
doi: 10.1016/j.jbc.2021.100671
---

# SAGA and SAGA-like SLIK transcriptional coactivators are structurally and biochemically equivalent.
**Authors:** Adamus K, Reboul C, Voss J, Huang C, Schittenhelm RB, Le SN, Ellisdon AM, Elmlund H, Boudes M, Elmlund D
**Journal:** J Biol Chem (2021)
**DOI:** [10.1016/j.jbc.2021.100671](https://doi.org/10.1016/j.jbc.2021.100671)
**PMC:** [PMC8131915](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8131915/)

## Abstract

1. J Biol Chem. 2021 Jan-Jun;296:100671. doi: 10.1016/j.jbc.2021.100671. Epub
2021  Apr 15.

SAGA and SAGA-like SLIK transcriptional coactivators are structurally and 
biochemically equivalent.

Adamus K(1), Reboul C(1), Voss J(1), Huang C(2), Schittenhelm RB(2), Le SN(1), 
Ellisdon AM(1), Elmlund H(1), Boudes M(3), Elmlund D(4).

Author information:
(1)Department of Biochemistry and Molecular Biology, Biomedicine Discovery 
Institute, Monash University, Clayton, Victoria, Australia.
(2)Monash Proteomics & Metabolomics Facility, Department of Biochemistry and 
Molecular Biology, Biomedicine Discovery Institute, Monash University, Clayton, 
Victoria, Australia.
(3)Department of Biochemistry and Molecular Biology, Biomedicine Discovery 
Institute, Monash University, Clayton, Victoria, Australia. Electronic address: 
marion.boudes@monash.edu.
(4)Department of Biochemistry and Molecular Biology, Biomedicine Discovery 
Institute, Monash University, Clayton, Victoria, Australia. Electronic address: 
dominika.elmlund@monash.edu.

The SAGA-like complex SLIK is a modified version of the 
Spt-Ada-Gcn5-Acetyltransferase (SAGA) complex. SLIK is formed through C-terminal 
truncation of the Spt7 SAGA subunit, causing loss of Spt8, one of the subunits 
that interacts with the TATA-binding protein (TBP). SLIK and SAGA are both 
coactivators of RNA polymerase II transcription in yeast, and both SAGA and SLIK 
perform chromatin modifications. The two complexes have been speculated to 
uniquely contribute to transcriptional regulation, but their respective 
contributions are not clear. To investigate, we assayed the chromatin modifying 
functions of SAGA and SLIK, revealing identical kinetics on minimal substrates 
in vitro. We also examined the binding of SAGA and SLIK to TBP and concluded 
that interestingly, both protein complexes have similar affinity for TBP. 
Additionally, despite the loss of Spt8 and C-terminus of Spt7 in SLIK, TBP 
prebound to SLIK is not released in the presence of TATA-box DNA, just like TBP 
prebound to SAGA. Furthermore, we determined a low-resolution cryo-EM structure 
of SLIK, revealing a modular architecture identical to SAGA. Finally, we 
performed a comprehensive study of DNA-binding properties of both coactivators. 
Purified SAGA and SLIK both associate with ssDNA and dsDNA with high affinity 
(KD = 10-17 nM), and the binding is sequence-independent. In conclusion, our 
study shows that the cleavage of Spt7 and the absence of the Spt8 subunit in 
SLIK neither drive any major conformational differences in its structure 
compared with SAGA, nor significantly affect HAT, DUB, or DNA-binding activities 
in vitro.

Copyright © 2021 The Authors. Published by Elsevier Inc. All rights reserved.

DOI: 10.1016/j.jbc.2021.100671
PMCID: PMC8131915
PMID: 33864814 [Indexed for MEDLINE]

Conflict of interest statement: Conflict of interest The authors declare that 
they have no conflicts of interest with the contents of this article.

## Full Text

Spt-Ada-Gcn5 Acetyltransferase (SAGA) (1, 2) is a major transcriptional coactivator complex in eukaryotes. This 1.5 MDa complex is conserved from yeast to humans and performs multiple functions during transcription initiation and elongation by RNA polymerase II: histone acetylation (HAT) (1), histone deubiquitination (DUB) (3), activator binding (1, 2), and TATA-binding protein interaction (4). SAGA-mediated chromatin modifications are important for expression of nearly all genes (5, 6). The role of SAGA extends beyond transcription and includes mRNA export (7, 8), mRNA splicing (8), RNA polymerase II- and III-dependent snRNA transcription (9), and DNA repair (10, 11). SAGA is also important for development in metazoans (12) and tolerance of plants to environmental stress (13). SLIK (also named SALSA for “SAGA altered, Spt8 absent”) was discovered in 2002 in the yeast Saccharomyces cerevisiae (14, 15). In this isoform of SAGA, the 150 kDa subunit Spt7 is C-terminally truncated, resulting in loss of the 65 kDa Spt8 subunit—one of the two main TBP-interacting subunits. Stress conditions trigger passive diffusion of protease Pep4 into the nucleus (16), where it cleaves the Spt7 C-terminus of SAGA, resulting in dissociation of Spt8 and formation of SLIK (17). Little is known about the exact function of SLIK. It is essential for response to nitrogen starvation (17) and was proposed to be involved in the retrograde response signaling pathway of communication between mitochondria and the nucleus (14).

In the recent cryo-EM structure of SAGA in complex with TBP, Papai et al. (18) showed that the Spt3 and Spt8 subunits of SAGA cooperatively inhibit TBP binding to DNA, with Spt8 appearing to have a secondary role in adjusting the orientation of TBP. Earlier in vivo cross-linking studies revealed that the general transcription factor TFIIA competes with Spt8 for TBP binding (19). Papai et al. (18) therefore proposed that TFIIA displaces Spt8 from TBP and promotes TBP-DNA complex formation. In SLIK, the absence of Spt8 could result in a less repressive complex. This hypothesis would reconcile the seemingly conflicting data indicating that SAGA has both a positive and a negative role in expression of several genes (20, 21).

Although high-resolution structures of the DUB module (22, 23) and individual subunits of the HAT module (24) are available, both modules remain at lower resolution within the most recent complete SAGA complex structure, due to their inherent flexibility (18, 25). This limits the possibility to extrapolate the impacts of the loss of Spt8 and C-terminus of Spt7 from the structural information available, especially when considering that HAT module subunits Ada3 and catalytic Gcn5 both cross-link to Spt7 (26) and that the DUB module is close to the C-terminus of Spt7 and Spt8 (18, 25). Additionally, functional studies showed that the DUB module subunit Ubp8 affects the cleavage of Spt7 and may have a regulatory role in SLIK formation (27). Whether the loss of Spt8 would reciprocally affect the DUB activity is unknown.

Clarifying the respective contribution of SAGA and SLIK to transcription regulation is a prerequisite to interpret functional data such as genome-wide transcription analyses, which only see the sum of their individual contributions to gene expression. We therefore initiated a functional and structural study of SLIK, to elucidate how the absence of the C-terminus of Spt7 and loss of Spt8 impacts the complex. We solved a low-resolution cryo-EM structure of SLIK and compared it with currently available maps of SAGA, revealing a modular architecture identical to SAGA. We also used SAGA and SLIK complexes purified to high homogeneity to compare their DUB and HAT activities, which are identical on minimal substrate. Additionally, we compared binding of SAGA and SLIK to TBP and examined competition between SAGA/SLIK, TBP, and TATA-box DNA. We concluded that both complexes behave similarly in their interaction with TBP.

It is well established that the SAGA family of transcriptional coactivators are recruited to gene promoters via interactions with DNA-binding activators (28, 29, 30, 31). Recently, Mittal et al. (32) showed that the HAT activity in SAGA is sensitive to nucleosome-flanking DNA, hinting that SAGA might directly interact with flanking DNA. We therefore performed a comprehensive study of DNA-binding properties of both SAGA and SLIK coactivators. This established that both SAGA and SLIK bind DNA with high affinity in a sequence-independent manner.

The overall structural arrangement, the chromatin-modifying function and the DNA-binding activity are unaffected by the cleavage of Spt7 C-terminus and loss of Spt8 in SAGA. Our data strongly suggest that SAGA and SLIK share the same fundamental functional and structural characteristics, and more gene-specific specific investigations would be required to reveal any distinct regulatory roles associated with their compositional differences.

ResultsSAGA and SLIK purified to high homogeneitySAGA and SLIK were purified from S. cerevisiae cells, using a strain with a TAP-tag on Ubp8. Complexes were purified together from whole-cell extracts by affinity chromatography (IgG and Heparin) and subsequently separated during an ion exchange step, following the previously established protocol (14) (Fig. 1). Mass spectrometry analyses confirmed that all subunits were present. The Rtg2 protein, initially described to be associated with SLIK (14), was not identified by mass spectrometry in neither purified SAGA nor SLIK complexes. Our SLIK complex has seemingly unaffected structural architecture and chromatin-modifying activities (see next sections), which contradicts the hypothesis that Rtg2 would be required for structural integrity of SLIK (14).Figure 1Analysis of purified SAGA and SLIK complexes showing the expected subunits stoichiometry.A, SDS-PAGE gel of purified SLIK and SAGA complexes with all subunits indicated. The Rtg2 subunit associated with SLIK was not identified; B, western blot analysis of purified SLIK and SAGA with anti-Spt7 antibody. ∗ indicates the position of truncated Spt7 subunit in the SLIK complex on both SDS-PAGE gel and WB in panels A and B, respectively.Cross-linking mass spectrometry (XL-MS) reveals identical architecture of SAGA and SLIKWe used cross-linking mass spectrometry to create and compare protein–protein cross-linking maps of SAGA and SLIK. We obtained 128/82 intramolecular and 70/59 intermolecular cross-linked peptides with an e-value below 0.01 for SAGA and SLIK, respectively (Tables S1–S3). These numbers are lower than in the exhaustive XL-MS analysis of SAGA by Han et al., (26) which included several repeats, and do not represent all the possible interactions; therefore the cross-linked peptides for SAGA and SLIK only partially overlap (58 intramolecular and 21 intermolecular cross-linked peptides are shared between SAGA and SLIK) (Fig. 2, Table S1). However, all SAGA and SLIK cross-linkers identified agree with the previously published SAGA complex structures (18, 25) and cross-linking networks (26). We conclude that the modules and subunits are similarly arranged in both complexes (Fig. 2). In both SAGA and SLIK, Tra1 cross-links to Taf12 and Ada1, in agreement with previous studies identifying Ada1 and Taf12 as Tra1 interacting partners forming a hinge region (26, 33). We can therefore conclude that the orientation of the hinge region is retained in SLIK.Figure 2BS3cross-linking map of A, SAGA and B, SLIK complexes. Tra1 (navy blue); core subunits: Ada1, Taf5, Taf6, Taf9, Taf10, Taf12 (dark green); core subunits: Spt3, Spt7, Spt8 (light green); DUB module subunits: Sgf11, Sgf73, Sus1, Ubp8 (light blue); HAT module subunits: Ada2, Ada3, Gcn5, Sgf29 (purple). Intrasubunit cross-links are presented in green, intersubunit cross-links are presented in gray. For specific cross-linking positions within each subunit refer to the Tables S1–S3.SAGA and SLIK have indistinguishable acetylation activity on minimal substrateWe compared the HAT activity of SAGA versus SLIK by measuring their respective initial acetylation rates at increasing concentrations of H3 peptide, in the presence of saturating amounts of acetyl-CoA, as previously described (34). Progress curves of SAGA or SLIK-catalyzed acetylation of H3 peptide were generated by coupling peptide acetylation with NADH formation. Velocity experiments were subsequently used to estimate the steady-state kinetic parameters. Initial rates were plotted as a function of the substrate concentration (Fig. 3A) and fit to the Michaelis–Menten equation. We concluded that kcat and KM values are indistinguishable for SAGA and SLIK (Table 1).Figure 3Comparison of histone-modifying activities of SAGA and SLIK.A, HAT activity of SAGA versus SLIK measured on the H3 peptide (H2N-ARTKQTARKSTGGKAPRKQLA-COOH). Individual data points are plotted. B, DUB activity of SAGA versus SLIK toward the minimal substrate Ub-AMC. Differences are not significant.Table 1Summary of steady-state kinetic parameters, nonlinear least square fitting to Michaelis–Menten equationProtein complexkcat (s−1)KM (μM)kcat/KM (μM−1.s−1)SAGA9.5 ± 1.5186 ± 1050.051 ± 0.014SLIK8.2 ± 1.6114 ± 1050.072 ± 0.015Our results cannot be directly compared with any previously published studies, since none of them used the entire SAGA complex to measure HAT activity on minimal substrate. Ringel et al. (34) used purified HAT module with H3 peptide substrate and obtained a kcat of 2.5 s−1 and a KM of 250 μM (with wild type HAT module and unmodified peptide). These values are similar to ours. Discrepancies may be due to the use of purified module versus full complex. We also used a slightly different experimental setup, including higher buffer osmolarity. Studies that used nucleosomal substrates typically report a much higher acetylation rate due to the increased affinity of SAGA to nucleosomes (32).SAGA and SLIK have identical deubiquitination activity on minimal substrateWe compared the DUB activity of SAGA and SLIK, using the minimal substrate ubiquitin-AMC (Ub-AMC) as previously described (35). Progress curves of SAGA or SLIK-catalyzed hydrolysis of Ub-AMC were generated by monitoring the release of fluorescent dye AMC. Recorded fluorescence intensity signal was converted into the concentration of AMC and plotted as a function of time (Fig. 3B). Generated progress curves show that SAGA and SLIK exhibit the same level of activity. Subsequently, an estimation of the steady-state kinetic parameters was attempted from the initial velocity experiments. Initial rates were plotted as a function of the substrate concentration (Fig. 3B) and fit to the Michaelis–Menten equation. Accurate estimation of the kinetic parameters from collected data was not possible due to the technical limitations: the high KM observed (30–35 μM) in this experimental setup would require substrate concentrations far higher to the ones used here (20 μM maximum). These concentrations could not be used due to an increased aggregation of Ub-AMC above 30 μM, the high percentage of DMSO that would have resulted, as well as extended measurement times to produce complete substrate hydrolysis. However, our kinetic curves indicate that SAGA and SLIK hydrolyze Ub-AMC at the same rate (Fig. 3B). The KM that we observe is within the same range as reported by Samara et al. (35) using minimal substrate and purified DUB module, i.e., 24 μM. Notably, while Samara et al. were able to perform their experiment at 30 °C when using purified DUB module, our experiments were conducted at ambient temperature due to the limited stability of SAGA and SLIK at higher temperatures, thus slowing the reaction rate.SAGA and SLIK bind to ssDNA and dsDNA with high affinityElectrophoretic Mobility Shift Assay (EMSA) confirmed that, as suggested by Mittal et al. (32), purified SAGA binds to dsDNA (Fig. 4C). We therefore initiated a comprehensive study of DNA-binding properties of both SAGA and SLIK, using fluorescence polarization (FP) titrations (Fig. 5). We started to assess binding to DNA fragments that approximately match the size of SAGA and SLIK (27 nm long; top lobe 12–15 nm wide, Tra1 15–18 nm wide). To this end, we used fluorescently labeled 75-mer oligonucleotides that are 25 nm long (http://www.calctool.org/CALC/prof/bio/dna). The DNA sequence used corresponds to the HIS4 TATA-box containing promoter (see Supporting information). Both SAGA and SLIK bind to 75-mer dsDNA with high affinity (KD ∼ 10 nM) and 75-mer ssDNA of the same sequence with slightly lower affinity (KD ∼ 15–17 nM) (Fig. 5, A and B).Figure 4Binding of SAGA and SLIK to TBP.A, gel shift assay using FITC-labeled TBP (10 nM) and SAGA or SLIK at increasing concentrations (0–30 nM). B, quantitation by densitometry of unbound TBP in A, showing similar affinity for both SAGA and SLIK. C, gel-shift assay using TBP, SAGA or SLIK, and Cy3-labeled TATA-box dsDNA; 20-mer (A), 75-mer (B), and 75-mer in presence of 12-fold molar excess TATA-less DNA (C), as indicated. In lanes 10 to 15, DNA was added to a preformed SAGA- or SLIK-TBP complex. The asterisk indicates minor association of SAGA/SLIK without (lanes 4, 7) or with (lanes 10, 13) TBP to 20-mer DNA.Figure 5Characterization of SAGA and SLIK DNA-binding affinity.A, 75-mer dsDNA; KD(SAGA) = 10 ± 4 [nM], KD(SLIK) = 10 ± 4 [nM], B, 75-mer ssDNA; KD(SAGA) = 15 ± 11 [nM], KD(SLIK) = 17 ± 6 [nM], C, 20-mer TATA-less dsDNA; KD(SAGA) = 78 ± 32 [nM], KD(SLIK) = 15 ± 4 [nM], D, 20-mer TATA-box dsDNA; KD(SAGA) = 99 ± 50 [nM], KD(SLIK) = 21 ± 7 [nM], E, competition assay with Poly(dI:dC), F, competition assay with unlabeled 200-mer dsDNA corresponding to RPS5 TATA-less promoter. Individual data points are plotted.To determine whether the binding was sequence-specific, we measured FP of labeled 75-mer dsDNA corresponding to the HIS4 TATA-box containing promoter with SAGA or SLIK, in the presence of increasing amounts of competitors Poly(deoxyinosinic-deoxycytidylic) acid (Poly(dI-dC)) (Fig. 5E), unlabeled 200-mer dsDNA corresponding to RPS5 TATA-less promoter (Fig. 5F), or unlabeled 75-mer of the same sequence (Fig. S1). FP was strongly reduced in all cases (decreased by 95% with ten times excess competitor), revealing that binding of both SAGA and SLIK to DNA is sequence-independent.To better characterize SAGA and SLIK binding to DNA, we assessed SAGA and SLIK binding to shorter DNA fragments (20-mer, i.e., approximately 7 nm in length). The sequence corresponds to the 5’ of the 75-mer ssDNA used previously and does not contain a TATA-box (see Supporting information). SLIK retains binding to 20-mer dsDNA with high affinity (KD ∼ 15 nM) (Fig. 5C). However, binding of SAGA is strongly affected and has a weaker affinity, about five times lower than SLIK (KD ∼ 78 nM). The observed difference in affinity toward shorter DNA fragments may be explained by the removal of the net negative charge carried by the Spt8 subunit, as Spt8 is the most negatively charged subunit in SAGA based on computed isoelectric points. We observed similar differences between SAGA and SLIK when measuring binding to 20-mer dsDNA fragment encompassing a TATA-box (KD(SLIK) ∼ 21 nM; KD(SAGA) ∼ 100 nM) (Fig. 5D). No specific binding to 20-mer ssDNA could be measured for either SAGA or SLIK.SAGA and SLIK bind TBPAlthough Spt8 and C-terminus of Spt7 both interact with TBP (19, 26), Papai et al. (18) recently showed that Spt3 subunit is the main TBP binder, while Spt8 appears to have a secondary role in positioning of TBP. To determine whether the loss of Spt8 and C-terminus of Spt7 impacts the affinity of SAGA for TBP, we investigated its binding by SAGA and SLIK using EMSA (Fig. 4A). The titration reveals no significant difference in binding affinity of SAGA and SLIK to TBP (Fig. 4B).Next, we investigated whether the absence of Spt8 in SLIK would alleviate the steric hindrance that prevents TBP from binding to TATA-box DNA when bound to SAGA and whether it would affect the release of TBP from the complex. We incubated SAGA- or SLIK-bound TBP with 20-mer or 75-mer TATA-box fluorescently labeled dsDNA and examined the DNA shift by EMSA (Fig. 4C, lanes 10–15). Like Papai et al., we did not observe any TBP-DNA complex when using either SAGA- or SLIK-bound TBP (TBP-DNA complexes in the absence of SAGA/SLIK are shown in Fig. S2). However, we highlighted the presence of a ternary SAGA-TBP-DNA and SLIK-TBP-DNA complex in the presence of 75-mer TATA-box DNA (but only minor in the presence of the 20-mer, presumably due to the lower affinity of the complexes for shorter DNA fragments, as shown in Fig. 5, C and D). Furthermore, we used mass spectrometry to confirm that the shifted DNA was associated to the SAGA-TBP and SLIK-TBP complexes rather than SAGA or SLIK alone (Tables S4 and S5). This complex was not observed by Papai et al. in their EMSA as they used a 12-fold molar excess of unlabeled TATA-less DNA with respect to their 100-mer labeled TATA-box DNA. Given that SAGA is able to bind DNA independently of the sequence (Fig. 5, E and F), the excess of unlabeled DNA masked SAGA binding to fluorescent DNA. We observed the same outcome as Papai et al. when adding unlabeled TATA-less DNA, i.e., no shift of labeled DNA. Notably, the DNA binding of SAGA-TBP complex is unlikely to be mediated by TBP, since if this was the case, SAGA-TBP complex would also be bound to 20-mer TATA-box DNA. This agrees with the fact that the TBP DNA-binding cleft is not accessible when TBP is bound to SAGA (18). Our results demonstrate that the absence of Spt8 and C-terminus of Spt7 in SLIK does not significantly alter its affinity for TBP, nor does it alleviate its inhibitory effect on DNA binding by TBP, when the latter is bound to the complex. In conclusion, we demonstrate that both SAGA and SLIK are able to concomitantly bind TBP and DNA—either with or without TATA-box.Cryo-EM of SLIKWe next initiated a cryo-EM study of SLIK, to elucidate the structural differences between SLIK and SAGA. Like SAGA, SLIK is a fragile complex prone to denaturation upon grid preparation and needs to be stabilized by glutaraldehyde cross-linking (18, 25, 33), using either GraFix (36, 37) or AgarFix (38) techniques. Cryo-EM data were collected on purified SLIK complex stabilized by GraFix (36, 37). The initial 3D ab initio model was subjected to 3D classification (Figs. S3 and S4). The best class (22.5 Å resolution) is shown in Figure 6. The recent structure of S. cerevisiae Tra1 and structural core of SAGA (25) were fitted in the map of SLIK, revealing that SLIK and SAGA have identical overall architecture (Fig. 6B). The model of Komagataella phaffii SAGA (18) is also in excellent agreement with our map of SLIK (Fig. S5). Like SAGA, SLIK is organized in two lobes that are flexibly connected, as evidenced by the movement of the two lobes with respect to each other in the 2D class averages (Fig. 6, Fig. S3). The bottom lobe is fully occupied by the recruitment module formed by 433 kDa transcription-associated protein 1 (Tra1) (39), while other modules are gathered in a conformationally flexible upper lobe (33, 39, 40). Both lobes are connected through a narrow hinge region (25, 26, 33). As expected, the Spt8 subunit (resolved at lower resolution in the recent SAGA structures (18, 25)) is absent in SLIK (Fig. 6B, Figs. S3 and S5). The resolution of our SLIK model does not allow determination of the orientation of the individual subunits or deducing whether the octamer-like internal structure of the SAGA core module is retained in SLIK. However, in correlation with XL-MS data (Fig. 2), we conclude that the overall arrangement of the core subunits is highly similar in SAGA and SLIK. As reported for SAGA, HAT and DUB modules are poorly resolved due to their inherent flexibility, evident from the 2D class averages (Fig. S3). However, since our XL-MS data for SLIK is in excellent agreement with previous studies (18, 25, 26), we were able to use published structures (18, 25, 33) to model the DUB module and position the HAT module in the extra density present in the cryo-EM map of SLIK (Fig. 6C).Figure 6Cryo-EM of SLIK.A, best 3D class, B, map of SLIK map superimposed with the structural core of S. cerevisiae SAGA (PDB: 6T9I), which excludes HAT and DUB modules that were resolved at lower resolution due to their flexibility (25). Spt7 is indicated with an arrow (modeled up to residue 1085). Spt8 and C-terminus of Spt7 that connects to Spt8 (residues 1086–1332), are absent in the presented SAGA model, since they could not be resolved due to their important flexibility. Spt8 (but not the C-terminus of Spt7) was resolved in K. phaffii SAGA in complex with TBP (PDB: 6TBM, see also Fig. S5); the gray sphere indicates the position of Spt8 in SAGA-TBP complex (18). Tra1 (bottom lobe) is in gray, structural core (top lobe): Ada1 (navy blue), Spt3 (purple), Spt7 (light blue), Spt20 (yellow), Taf5 (pink), Taf6 (orange), Taf9 (magenta), Taf10 (khaki), Taf12 (dark green), DUB module: Sgf73 (light green). C, SLIK model, based on XL-MS data and available cryo-EM structures of S. cerevisiae and K. phaffii SAGA (PDB: 6T9I and 6TBM). DUB module is shown in purple and HAT module is represented as a sphere, due to the lack of structural information available for the individual subunits.

SAGA and SLIK purified to high homogeneitySAGA and SLIK were purified from S. cerevisiae cells, using a strain with a TAP-tag on Ubp8. Complexes were purified together from whole-cell extracts by affinity chromatography (IgG and Heparin) and subsequently separated during an ion exchange step, following the previously established protocol (14) (Fig. 1). Mass spectrometry analyses confirmed that all subunits were present. The Rtg2 protein, initially described to be associated with SLIK (14), was not identified by mass spectrometry in neither purified SAGA nor SLIK complexes. Our SLIK complex has seemingly unaffected structural architecture and chromatin-modifying activities (see next sections), which contradicts the hypothesis that Rtg2 would be required for structural integrity of SLIK (14).Figure 1Analysis of purified SAGA and SLIK complexes showing the expected subunits stoichiometry.A, SDS-PAGE gel of purified SLIK and SAGA complexes with all subunits indicated. The Rtg2 subunit associated with SLIK was not identified; B, western blot analysis of purified SLIK and SAGA with anti-Spt7 antibody. ∗ indicates the position of truncated Spt7 subunit in the SLIK complex on both SDS-PAGE gel and WB in panels A and B, respectively.

SAGA and SLIK purified to high homogeneity

SAGA and SLIK were purified from S. cerevisiae cells, using a strain with a TAP-tag on Ubp8. Complexes were purified together from whole-cell extracts by affinity chromatography (IgG and Heparin) and subsequently separated during an ion exchange step, following the previously established protocol (14) (Fig. 1). Mass spectrometry analyses confirmed that all subunits were present. The Rtg2 protein, initially described to be associated with SLIK (14), was not identified by mass spectrometry in neither purified SAGA nor SLIK complexes. Our SLIK complex has seemingly unaffected structural architecture and chromatin-modifying activities (see next sections), which contradicts the hypothesis that Rtg2 would be required for structural integrity of SLIK (14).Figure 1Analysis of purified SAGA and SLIK complexes showing the expected subunits stoichiometry.A, SDS-PAGE gel of purified SLIK and SAGA complexes with all subunits indicated. The Rtg2 subunit associated with SLIK was not identified; B, western blot analysis of purified SLIK and SAGA with anti-Spt7 antibody. ∗ indicates the position of truncated Spt7 subunit in the SLIK complex on both SDS-PAGE gel and WB in panels A and B, respectively.

Analysis of purified SAGA and SLIK complexes showing the expected subunits stoichiometry.A, SDS-PAGE gel of purified SLIK and SAGA complexes with all subunits indicated. The Rtg2 subunit associated with SLIK was not identified; B, western blot analysis of purified SLIK and SAGA with anti-Spt7 antibody. ∗ indicates the position of truncated Spt7 subunit in the SLIK complex on both SDS-PAGE gel and WB in panels A and B, respectively.

Cross-linking mass spectrometry (XL-MS) reveals identical architecture of SAGA and SLIKWe used cross-linking mass spectrometry to create and compare protein–protein cross-linking maps of SAGA and SLIK. We obtained 128/82 intramolecular and 70/59 intermolecular cross-linked peptides with an e-value below 0.01 for SAGA and SLIK, respectively (Tables S1–S3). These numbers are lower than in the exhaustive XL-MS analysis of SAGA by Han et al., (26) which included several repeats, and do not represent all the possible interactions; therefore the cross-linked peptides for SAGA and SLIK only partially overlap (58 intramolecular and 21 intermolecular cross-linked peptides are shared between SAGA and SLIK) (Fig. 2, Table S1). However, all SAGA and SLIK cross-linkers identified agree with the previously published SAGA complex structures (18, 25) and cross-linking networks (26). We conclude that the modules and subunits are similarly arranged in both complexes (Fig. 2). In both SAGA and SLIK, Tra1 cross-links to Taf12 and Ada1, in agreement with previous studies identifying Ada1 and Taf12 as Tra1 interacting partners forming a hinge region (26, 33). We can therefore conclude that the orientation of the hinge region is retained in SLIK.Figure 2BS3cross-linking map of A, SAGA and B, SLIK complexes. Tra1 (navy blue); core subunits: Ada1, Taf5, Taf6, Taf9, Taf10, Taf12 (dark green); core subunits: Spt3, Spt7, Spt8 (light green); DUB module subunits: Sgf11, Sgf73, Sus1, Ubp8 (light blue); HAT module subunits: Ada2, Ada3, Gcn5, Sgf29 (purple). Intrasubunit cross-links are presented in green, intersubunit cross-links are presented in gray. For specific cross-linking positions within each subunit refer to the Tables S1–S3.

Cross-linking mass spectrometry (XL-MS) reveals identical architecture of SAGA and SLIK

We used cross-linking mass spectrometry to create and compare protein–protein cross-linking maps of SAGA and SLIK. We obtained 128/82 intramolecular and 70/59 intermolecular cross-linked peptides with an e-value below 0.01 for SAGA and SLIK, respectively (Tables S1–S3). These numbers are lower than in the exhaustive XL-MS analysis of SAGA by Han et al., (26) which included several repeats, and do not represent all the possible interactions; therefore the cross-linked peptides for SAGA and SLIK only partially overlap (58 intramolecular and 21 intermolecular cross-linked peptides are shared between SAGA and SLIK) (Fig. 2, Table S1). However, all SAGA and SLIK cross-linkers identified agree with the previously published SAGA complex structures (18, 25) and cross-linking networks (26). We conclude that the modules and subunits are similarly arranged in both complexes (Fig. 2). In both SAGA and SLIK, Tra1 cross-links to Taf12 and Ada1, in agreement with previous studies identifying Ada1 and Taf12 as Tra1 interacting partners forming a hinge region (26, 33). We can therefore conclude that the orientation of the hinge region is retained in SLIK.Figure 2BS3cross-linking map of A, SAGA and B, SLIK complexes. Tra1 (navy blue); core subunits: Ada1, Taf5, Taf6, Taf9, Taf10, Taf12 (dark green); core subunits: Spt3, Spt7, Spt8 (light green); DUB module subunits: Sgf11, Sgf73, Sus1, Ubp8 (light blue); HAT module subunits: Ada2, Ada3, Gcn5, Sgf29 (purple). Intrasubunit cross-links are presented in green, intersubunit cross-links are presented in gray. For specific cross-linking positions within each subunit refer to the Tables S1–S3.

BS3cross-linking map of A, SAGA and B, SLIK complexes. Tra1 (navy blue); core subunits: Ada1, Taf5, Taf6, Taf9, Taf10, Taf12 (dark green); core subunits: Spt3, Spt7, Spt8 (light green); DUB module subunits: Sgf11, Sgf73, Sus1, Ubp8 (light blue); HAT module subunits: Ada2, Ada3, Gcn5, Sgf29 (purple). Intrasubunit cross-links are presented in green, intersubunit cross-links are presented in gray. For specific cross-linking positions within each subunit refer to the Tables S1–S3.

SAGA and SLIK have indistinguishable acetylation activity on minimal substrateWe compared the HAT activity of SAGA versus SLIK by measuring their respective initial acetylation rates at increasing concentrations of H3 peptide, in the presence of saturating amounts of acetyl-CoA, as previously described (34). Progress curves of SAGA or SLIK-catalyzed acetylation of H3 peptide were generated by coupling peptide acetylation with NADH formation. Velocity experiments were subsequently used to estimate the steady-state kinetic parameters. Initial rates were plotted as a function of the substrate concentration (Fig. 3A) and fit to the Michaelis–Menten equation. We concluded that kcat and KM values are indistinguishable for SAGA and SLIK (Table 1).Figure 3Comparison of histone-modifying activities of SAGA and SLIK.A, HAT activity of SAGA versus SLIK measured on the H3 peptide (H2N-ARTKQTARKSTGGKAPRKQLA-COOH). Individual data points are plotted. B, DUB activity of SAGA versus SLIK toward the minimal substrate Ub-AMC. Differences are not significant.Table 1Summary of steady-state kinetic parameters, nonlinear least square fitting to Michaelis–Menten equationProtein complexkcat (s−1)KM (μM)kcat/KM (μM−1.s−1)SAGA9.5 ± 1.5186 ± 1050.051 ± 0.014SLIK8.2 ± 1.6114 ± 1050.072 ± 0.015Our results cannot be directly compared with any previously published studies, since none of them used the entire SAGA complex to measure HAT activity on minimal substrate. Ringel et al. (34) used purified HAT module with H3 peptide substrate and obtained a kcat of 2.5 s−1 and a KM of 250 μM (with wild type HAT module and unmodified peptide). These values are similar to ours. Discrepancies may be due to the use of purified module versus full complex. We also used a slightly different experimental setup, including higher buffer osmolarity. Studies that used nucleosomal substrates typically report a much higher acetylation rate due to the increased affinity of SAGA to nucleosomes (32).

SAGA and SLIK have indistinguishable acetylation activity on minimal substrate

We compared the HAT activity of SAGA versus SLIK by measuring their respective initial acetylation rates at increasing concentrations of H3 peptide, in the presence of saturating amounts of acetyl-CoA, as previously described (34). Progress curves of SAGA or SLIK-catalyzed acetylation of H3 peptide were generated by coupling peptide acetylation with NADH formation. Velocity experiments were subsequently used to estimate the steady-state kinetic parameters. Initial rates were plotted as a function of the substrate concentration (Fig. 3A) and fit to the Michaelis–Menten equation. We concluded that kcat and KM values are indistinguishable for SAGA and SLIK (Table 1).Figure 3Comparison of histone-modifying activities of SAGA and SLIK.A, HAT activity of SAGA versus SLIK measured on the H3 peptide (H2N-ARTKQTARKSTGGKAPRKQLA-COOH). Individual data points are plotted. B, DUB activity of SAGA versus SLIK toward the minimal substrate Ub-AMC. Differences are not significant.Table 1Summary of steady-state kinetic parameters, nonlinear least square fitting to Michaelis–Menten equationProtein complexkcat (s−1)KM (μM)kcat/KM (μM−1.s−1)SAGA9.5 ± 1.5186 ± 1050.051 ± 0.014SLIK8.2 ± 1.6114 ± 1050.072 ± 0.015

Comparison of histone-modifying activities of SAGA and SLIK.A, HAT activity of SAGA versus SLIK measured on the H3 peptide (H2N-ARTKQTARKSTGGKAPRKQLA-COOH). Individual data points are plotted. B, DUB activity of SAGA versus SLIK toward the minimal substrate Ub-AMC. Differences are not significant.

Summary of steady-state kinetic parameters, nonlinear least square fitting to Michaelis–Menten equation

Our results cannot be directly compared with any previously published studies, since none of them used the entire SAGA complex to measure HAT activity on minimal substrate. Ringel et al. (34) used purified HAT module with H3 peptide substrate and obtained a kcat of 2.5 s−1 and a KM of 250 μM (with wild type HAT module and unmodified peptide). These values are similar to ours. Discrepancies may be due to the use of purified module versus full complex. We also used a slightly different experimental setup, including higher buffer osmolarity. Studies that used nucleosomal substrates typically report a much higher acetylation rate due to the increased affinity of SAGA to nucleosomes (32).

SAGA and SLIK have identical deubiquitination activity on minimal substrateWe compared the DUB activity of SAGA and SLIK, using the minimal substrate ubiquitin-AMC (Ub-AMC) as previously described (35). Progress curves of SAGA or SLIK-catalyzed hydrolysis of Ub-AMC were generated by monitoring the release of fluorescent dye AMC. Recorded fluorescence intensity signal was converted into the concentration of AMC and plotted as a function of time (Fig. 3B). Generated progress curves show that SAGA and SLIK exhibit the same level of activity. Subsequently, an estimation of the steady-state kinetic parameters was attempted from the initial velocity experiments. Initial rates were plotted as a function of the substrate concentration (Fig. 3B) and fit to the Michaelis–Menten equation. Accurate estimation of the kinetic parameters from collected data was not possible due to the technical limitations: the high KM observed (30–35 μM) in this experimental setup would require substrate concentrations far higher to the ones used here (20 μM maximum). These concentrations could not be used due to an increased aggregation of Ub-AMC above 30 μM, the high percentage of DMSO that would have resulted, as well as extended measurement times to produce complete substrate hydrolysis. However, our kinetic curves indicate that SAGA and SLIK hydrolyze Ub-AMC at the same rate (Fig. 3B). The KM that we observe is within the same range as reported by Samara et al. (35) using minimal substrate and purified DUB module, i.e., 24 μM. Notably, while Samara et al. were able to perform their experiment at 30 °C when using purified DUB module, our experiments were conducted at ambient temperature due to the limited stability of SAGA and SLIK at higher temperatures, thus slowing the reaction rate.

SAGA and SLIK have identical deubiquitination activity on minimal substrate

We compared the DUB activity of SAGA and SLIK, using the minimal substrate ubiquitin-AMC (Ub-AMC) as previously described (35). Progress curves of SAGA or SLIK-catalyzed hydrolysis of Ub-AMC were generated by monitoring the release of fluorescent dye AMC. Recorded fluorescence intensity signal was converted into the concentration of AMC and plotted as a function of time (Fig. 3B). Generated progress curves show that SAGA and SLIK exhibit the same level of activity. Subsequently, an estimation of the steady-state kinetic parameters was attempted from the initial velocity experiments. Initial rates were plotted as a function of the substrate concentration (Fig. 3B) and fit to the Michaelis–Menten equation. Accurate estimation of the kinetic parameters from collected data was not possible due to the technical limitations: the high KM observed (30–35 μM) in this experimental setup would require substrate concentrations far higher to the ones used here (20 μM maximum). These concentrations could not be used due to an increased aggregation of Ub-AMC above 30 μM, the high percentage of DMSO that would have resulted, as well as extended measurement times to produce complete substrate hydrolysis. However, our kinetic curves indicate that SAGA and SLIK hydrolyze Ub-AMC at the same rate (Fig. 3B). The KM that we observe is within the same range as reported by Samara et al. (35) using minimal substrate and purified DUB module, i.e., 24 μM. Notably, while Samara et al. were able to perform their experiment at 30 °C when using purified DUB module, our experiments were conducted at ambient temperature due to the limited stability of SAGA and SLIK at higher temperatures, thus slowing the reaction rate.

SAGA and SLIK bind to ssDNA and dsDNA with high affinityElectrophoretic Mobility Shift Assay (EMSA) confirmed that, as suggested by Mittal et al. (32), purified SAGA binds to dsDNA (Fig. 4C). We therefore initiated a comprehensive study of DNA-binding properties of both SAGA and SLIK, using fluorescence polarization (FP) titrations (Fig. 5). We started to assess binding to DNA fragments that approximately match the size of SAGA and SLIK (27 nm long; top lobe 12–15 nm wide, Tra1 15–18 nm wide). To this end, we used fluorescently labeled 75-mer oligonucleotides that are 25 nm long (http://www.calctool.org/CALC/prof/bio/dna). The DNA sequence used corresponds to the HIS4 TATA-box containing promoter (see Supporting information). Both SAGA and SLIK bind to 75-mer dsDNA with high affinity (KD ∼ 10 nM) and 75-mer ssDNA of the same sequence with slightly lower affinity (KD ∼ 15–17 nM) (Fig. 5, A and B).Figure 4Binding of SAGA and SLIK to TBP.A, gel shift assay using FITC-labeled TBP (10 nM) and SAGA or SLIK at increasing concentrations (0–30 nM). B, quantitation by densitometry of unbound TBP in A, showing similar affinity for both SAGA and SLIK. C, gel-shift assay using TBP, SAGA or SLIK, and Cy3-labeled TATA-box dsDNA; 20-mer (A), 75-mer (B), and 75-mer in presence of 12-fold molar excess TATA-less DNA (C), as indicated. In lanes 10 to 15, DNA was added to a preformed SAGA- or SLIK-TBP complex. The asterisk indicates minor association of SAGA/SLIK without (lanes 4, 7) or with (lanes 10, 13) TBP to 20-mer DNA.Figure 5Characterization of SAGA and SLIK DNA-binding affinity.A, 75-mer dsDNA; KD(SAGA) = 10 ± 4 [nM], KD(SLIK) = 10 ± 4 [nM], B, 75-mer ssDNA; KD(SAGA) = 15 ± 11 [nM], KD(SLIK) = 17 ± 6 [nM], C, 20-mer TATA-less dsDNA; KD(SAGA) = 78 ± 32 [nM], KD(SLIK) = 15 ± 4 [nM], D, 20-mer TATA-box dsDNA; KD(SAGA) = 99 ± 50 [nM], KD(SLIK) = 21 ± 7 [nM], E, competition assay with Poly(dI:dC), F, competition assay with unlabeled 200-mer dsDNA corresponding to RPS5 TATA-less promoter. Individual data points are plotted.To determine whether the binding was sequence-specific, we measured FP of labeled 75-mer dsDNA corresponding to the HIS4 TATA-box containing promoter with SAGA or SLIK, in the presence of increasing amounts of competitors Poly(deoxyinosinic-deoxycytidylic) acid (Poly(dI-dC)) (Fig. 5E), unlabeled 200-mer dsDNA corresponding to RPS5 TATA-less promoter (Fig. 5F), or unlabeled 75-mer of the same sequence (Fig. S1). FP was strongly reduced in all cases (decreased by 95% with ten times excess competitor), revealing that binding of both SAGA and SLIK to DNA is sequence-independent.To better characterize SAGA and SLIK binding to DNA, we assessed SAGA and SLIK binding to shorter DNA fragments (20-mer, i.e., approximately 7 nm in length). The sequence corresponds to the 5’ of the 75-mer ssDNA used previously and does not contain a TATA-box (see Supporting information). SLIK retains binding to 20-mer dsDNA with high affinity (KD ∼ 15 nM) (Fig. 5C). However, binding of SAGA is strongly affected and has a weaker affinity, about five times lower than SLIK (KD ∼ 78 nM). The observed difference in affinity toward shorter DNA fragments may be explained by the removal of the net negative charge carried by the Spt8 subunit, as Spt8 is the most negatively charged subunit in SAGA based on computed isoelectric points. We observed similar differences between SAGA and SLIK when measuring binding to 20-mer dsDNA fragment encompassing a TATA-box (KD(SLIK) ∼ 21 nM; KD(SAGA) ∼ 100 nM) (Fig. 5D). No specific binding to 20-mer ssDNA could be measured for either SAGA or SLIK.

SAGA and SLIK bind to ssDNA and dsDNA with high affinity

Electrophoretic Mobility Shift Assay (EMSA) confirmed that, as suggested by Mittal et al. (32), purified SAGA binds to dsDNA (Fig. 4C). We therefore initiated a comprehensive study of DNA-binding properties of both SAGA and SLIK, using fluorescence polarization (FP) titrations (Fig. 5). We started to assess binding to DNA fragments that approximately match the size of SAGA and SLIK (27 nm long; top lobe 12–15 nm wide, Tra1 15–18 nm wide). To this end, we used fluorescently labeled 75-mer oligonucleotides that are 25 nm long (http://www.calctool.org/CALC/prof/bio/dna). The DNA sequence used corresponds to the HIS4 TATA-box containing promoter (see Supporting information). Both SAGA and SLIK bind to 75-mer dsDNA with high affinity (KD ∼ 10 nM) and 75-mer ssDNA of the same sequence with slightly lower affinity (KD ∼ 15–17 nM) (Fig. 5, A and B).Figure 4Binding of SAGA and SLIK to TBP.A, gel shift assay using FITC-labeled TBP (10 nM) and SAGA or SLIK at increasing concentrations (0–30 nM). B, quantitation by densitometry of unbound TBP in A, showing similar affinity for both SAGA and SLIK. C, gel-shift assay using TBP, SAGA or SLIK, and Cy3-labeled TATA-box dsDNA; 20-mer (A), 75-mer (B), and 75-mer in presence of 12-fold molar excess TATA-less DNA (C), as indicated. In lanes 10 to 15, DNA was added to a preformed SAGA- or SLIK-TBP complex. The asterisk indicates minor association of SAGA/SLIK without (lanes 4, 7) or with (lanes 10, 13) TBP to 20-mer DNA.Figure 5Characterization of SAGA and SLIK DNA-binding affinity.A, 75-mer dsDNA; KD(SAGA) = 10 ± 4 [nM], KD(SLIK) = 10 ± 4 [nM], B, 75-mer ssDNA; KD(SAGA) = 15 ± 11 [nM], KD(SLIK) = 17 ± 6 [nM], C, 20-mer TATA-less dsDNA; KD(SAGA) = 78 ± 32 [nM], KD(SLIK) = 15 ± 4 [nM], D, 20-mer TATA-box dsDNA; KD(SAGA) = 99 ± 50 [nM], KD(SLIK) = 21 ± 7 [nM], E, competition assay with Poly(dI:dC), F, competition assay with unlabeled 200-mer dsDNA corresponding to RPS5 TATA-less promoter. Individual data points are plotted.

Binding of SAGA and SLIK to TBP.A, gel shift assay using FITC-labeled TBP (10 nM) and SAGA or SLIK at increasing concentrations (0–30 nM). B, quantitation by densitometry of unbound TBP in A, showing similar affinity for both SAGA and SLIK. C, gel-shift assay using TBP, SAGA or SLIK, and Cy3-labeled TATA-box dsDNA; 20-mer (A), 75-mer (B), and 75-mer in presence of 12-fold molar excess TATA-less DNA (C), as indicated. In lanes 10 to 15, DNA was added to a preformed SAGA- or SLIK-TBP complex. The asterisk indicates minor association of SAGA/SLIK without (lanes 4, 7) or with (lanes 10, 13) TBP to 20-mer DNA.

Characterization of SAGA and SLIK DNA-binding affinity.A, 75-mer dsDNA; KD(SAGA) = 10 ± 4 [nM], KD(SLIK) = 10 ± 4 [nM], B, 75-mer ssDNA; KD(SAGA) = 15 ± 11 [nM], KD(SLIK) = 17 ± 6 [nM], C, 20-mer TATA-less dsDNA; KD(SAGA) = 78 ± 32 [nM], KD(SLIK) = 15 ± 4 [nM], D, 20-mer TATA-box dsDNA; KD(SAGA) = 99 ± 50 [nM], KD(SLIK) = 21 ± 7 [nM], E, competition assay with Poly(dI:dC), F, competition assay with unlabeled 200-mer dsDNA corresponding to RPS5 TATA-less promoter. Individual data points are plotted.

To determine whether the binding was sequence-specific, we measured FP of labeled 75-mer dsDNA corresponding to the HIS4 TATA-box containing promoter with SAGA or SLIK, in the presence of increasing amounts of competitors Poly(deoxyinosinic-deoxycytidylic) acid (Poly(dI-dC)) (Fig. 5E), unlabeled 200-mer dsDNA corresponding to RPS5 TATA-less promoter (Fig. 5F), or unlabeled 75-mer of the same sequence (Fig. S1). FP was strongly reduced in all cases (decreased by 95% with ten times excess competitor), revealing that binding of both SAGA and SLIK to DNA is sequence-independent.

To better characterize SAGA and SLIK binding to DNA, we assessed SAGA and SLIK binding to shorter DNA fragments (20-mer, i.e., approximately 7 nm in length). The sequence corresponds to the 5’ of the 75-mer ssDNA used previously and does not contain a TATA-box (see Supporting information). SLIK retains binding to 20-mer dsDNA with high affinity (KD ∼ 15 nM) (Fig. 5C). However, binding of SAGA is strongly affected and has a weaker affinity, about five times lower than SLIK (KD ∼ 78 nM). The observed difference in affinity toward shorter DNA fragments may be explained by the removal of the net negative charge carried by the Spt8 subunit, as Spt8 is the most negatively charged subunit in SAGA based on computed isoelectric points. We observed similar differences between SAGA and SLIK when measuring binding to 20-mer dsDNA fragment encompassing a TATA-box (KD(SLIK) ∼ 21 nM; KD(SAGA) ∼ 100 nM) (Fig. 5D). No specific binding to 20-mer ssDNA could be measured for either SAGA or SLIK.

SAGA and SLIK bind TBPAlthough Spt8 and C-terminus of Spt7 both interact with TBP (19, 26), Papai et al. (18) recently showed that Spt3 subunit is the main TBP binder, while Spt8 appears to have a secondary role in positioning of TBP. To determine whether the loss of Spt8 and C-terminus of Spt7 impacts the affinity of SAGA for TBP, we investigated its binding by SAGA and SLIK using EMSA (Fig. 4A). The titration reveals no significant difference in binding affinity of SAGA and SLIK to TBP (Fig. 4B).Next, we investigated whether the absence of Spt8 in SLIK would alleviate the steric hindrance that prevents TBP from binding to TATA-box DNA when bound to SAGA and whether it would affect the release of TBP from the complex. We incubated SAGA- or SLIK-bound TBP with 20-mer or 75-mer TATA-box fluorescently labeled dsDNA and examined the DNA shift by EMSA (Fig. 4C, lanes 10–15). Like Papai et al., we did not observe any TBP-DNA complex when using either SAGA- or SLIK-bound TBP (TBP-DNA complexes in the absence of SAGA/SLIK are shown in Fig. S2). However, we highlighted the presence of a ternary SAGA-TBP-DNA and SLIK-TBP-DNA complex in the presence of 75-mer TATA-box DNA (but only minor in the presence of the 20-mer, presumably due to the lower affinity of the complexes for shorter DNA fragments, as shown in Fig. 5, C and D). Furthermore, we used mass spectrometry to confirm that the shifted DNA was associated to the SAGA-TBP and SLIK-TBP complexes rather than SAGA or SLIK alone (Tables S4 and S5). This complex was not observed by Papai et al. in their EMSA as they used a 12-fold molar excess of unlabeled TATA-less DNA with respect to their 100-mer labeled TATA-box DNA. Given that SAGA is able to bind DNA independently of the sequence (Fig. 5, E and F), the excess of unlabeled DNA masked SAGA binding to fluorescent DNA. We observed the same outcome as Papai et al. when adding unlabeled TATA-less DNA, i.e., no shift of labeled DNA. Notably, the DNA binding of SAGA-TBP complex is unlikely to be mediated by TBP, since if this was the case, SAGA-TBP complex would also be bound to 20-mer TATA-box DNA. This agrees with the fact that the TBP DNA-binding cleft is not accessible when TBP is bound to SAGA (18). Our results demonstrate that the absence of Spt8 and C-terminus of Spt7 in SLIK does not significantly alter its affinity for TBP, nor does it alleviate its inhibitory effect on DNA binding by TBP, when the latter is bound to the complex. In conclusion, we demonstrate that both SAGA and SLIK are able to concomitantly bind TBP and DNA—either with or without TATA-box.

SAGA and SLIK bind TBP

Although Spt8 and C-terminus of Spt7 both interact with TBP (19, 26), Papai et al. (18) recently showed that Spt3 subunit is the main TBP binder, while Spt8 appears to have a secondary role in positioning of TBP. To determine whether the loss of Spt8 and C-terminus of Spt7 impacts the affinity of SAGA for TBP, we investigated its binding by SAGA and SLIK using EMSA (Fig. 4A). The titration reveals no significant difference in binding affinity of SAGA and SLIK to TBP (Fig. 4B).

Next, we investigated whether the absence of Spt8 in SLIK would alleviate the steric hindrance that prevents TBP from binding to TATA-box DNA when bound to SAGA and whether it would affect the release of TBP from the complex. We incubated SAGA- or SLIK-bound TBP with 20-mer or 75-mer TATA-box fluorescently labeled dsDNA and examined the DNA shift by EMSA (Fig. 4C, lanes 10–15). Like Papai et al., we did not observe any TBP-DNA complex when using either SAGA- or SLIK-bound TBP (TBP-DNA complexes in the absence of SAGA/SLIK are shown in Fig. S2). However, we highlighted the presence of a ternary SAGA-TBP-DNA and SLIK-TBP-DNA complex in the presence of 75-mer TATA-box DNA (but only minor in the presence of the 20-mer, presumably due to the lower affinity of the complexes for shorter DNA fragments, as shown in Fig. 5, C and D). Furthermore, we used mass spectrometry to confirm that the shifted DNA was associated to the SAGA-TBP and SLIK-TBP complexes rather than SAGA or SLIK alone (Tables S4 and S5). This complex was not observed by Papai et al. in their EMSA as they used a 12-fold molar excess of unlabeled TATA-less DNA with respect to their 100-mer labeled TATA-box DNA. Given that SAGA is able to bind DNA independently of the sequence (Fig. 5, E and F), the excess of unlabeled DNA masked SAGA binding to fluorescent DNA. We observed the same outcome as Papai et al. when adding unlabeled TATA-less DNA, i.e., no shift of labeled DNA. Notably, the DNA binding of SAGA-TBP complex is unlikely to be mediated by TBP, since if this was the case, SAGA-TBP complex would also be bound to 20-mer TATA-box DNA. This agrees with the fact that the TBP DNA-binding cleft is not accessible when TBP is bound to SAGA (18). Our results demonstrate that the absence of Spt8 and C-terminus of Spt7 in SLIK does not significantly alter its affinity for TBP, nor does it alleviate its inhibitory effect on DNA binding by TBP, when the latter is bound to the complex. In conclusion, we demonstrate that both SAGA and SLIK are able to concomitantly bind TBP and DNA—either with or without TATA-box.

Cryo-EM of SLIKWe next initiated a cryo-EM study of SLIK, to elucidate the structural differences between SLIK and SAGA. Like SAGA, SLIK is a fragile complex prone to denaturation upon grid preparation and needs to be stabilized by glutaraldehyde cross-linking (18, 25, 33), using either GraFix (36, 37) or AgarFix (38) techniques. Cryo-EM data were collected on purified SLIK complex stabilized by GraFix (36, 37). The initial 3D ab initio model was subjected to 3D classification (Figs. S3 and S4). The best class (22.5 Å resolution) is shown in Figure 6. The recent structure of S. cerevisiae Tra1 and structural core of SAGA (25) were fitted in the map of SLIK, revealing that SLIK and SAGA have identical overall architecture (Fig. 6B). The model of Komagataella phaffii SAGA (18) is also in excellent agreement with our map of SLIK (Fig. S5). Like SAGA, SLIK is organized in two lobes that are flexibly connected, as evidenced by the movement of the two lobes with respect to each other in the 2D class averages (Fig. 6, Fig. S3). The bottom lobe is fully occupied by the recruitment module formed by 433 kDa transcription-associated protein 1 (Tra1) (39), while other modules are gathered in a conformationally flexible upper lobe (33, 39, 40). Both lobes are connected through a narrow hinge region (25, 26, 33). As expected, the Spt8 subunit (resolved at lower resolution in the recent SAGA structures (18, 25)) is absent in SLIK (Fig. 6B, Figs. S3 and S5). The resolution of our SLIK model does not allow determination of the orientation of the individual subunits or deducing whether the octamer-like internal structure of the SAGA core module is retained in SLIK. However, in correlation with XL-MS data (Fig. 2), we conclude that the overall arrangement of the core subunits is highly similar in SAGA and SLIK. As reported for SAGA, HAT and DUB modules are poorly resolved due to their inherent flexibility, evident from the 2D class averages (Fig. S3). However, since our XL-MS data for SLIK is in excellent agreement with previous studies (18, 25, 26), we were able to use published structures (18, 25, 33) to model the DUB module and position the HAT module in the extra density present in the cryo-EM map of SLIK (Fig. 6C).Figure 6Cryo-EM of SLIK.A, best 3D class, B, map of SLIK map superimposed with the structural core of S. cerevisiae SAGA (PDB: 6T9I), which excludes HAT and DUB modules that were resolved at lower resolution due to their flexibility (25). Spt7 is indicated with an arrow (modeled up to residue 1085). Spt8 and C-terminus of Spt7 that connects to Spt8 (residues 1086–1332), are absent in the presented SAGA model, since they could not be resolved due to their important flexibility. Spt8 (but not the C-terminus of Spt7) was resolved in K. phaffii SAGA in complex with TBP (PDB: 6TBM, see also Fig. S5); the gray sphere indicates the position of Spt8 in SAGA-TBP complex (18). Tra1 (bottom lobe) is in gray, structural core (top lobe): Ada1 (navy blue), Spt3 (purple), Spt7 (light blue), Spt20 (yellow), Taf5 (pink), Taf6 (orange), Taf9 (magenta), Taf10 (khaki), Taf12 (dark green), DUB module: Sgf73 (light green). C, SLIK model, based on XL-MS data and available cryo-EM structures of S. cerevisiae and K. phaffii SAGA (PDB: 6T9I and 6TBM). DUB module is shown in purple and HAT module is represented as a sphere, due to the lack of structural information available for the individual subunits.

Cryo-EM of SLIK

We next initiated a cryo-EM study of SLIK, to elucidate the structural differences between SLIK and SAGA. Like SAGA, SLIK is a fragile complex prone to denaturation upon grid preparation and needs to be stabilized by glutaraldehyde cross-linking (18, 25, 33), using either GraFix (36, 37) or AgarFix (38) techniques. Cryo-EM data were collected on purified SLIK complex stabilized by GraFix (36, 37). The initial 3D ab initio model was subjected to 3D classification (Figs. S3 and S4). The best class (22.5 Å resolution) is shown in Figure 6. The recent structure of S. cerevisiae Tra1 and structural core of SAGA (25) were fitted in the map of SLIK, revealing that SLIK and SAGA have identical overall architecture (Fig. 6B). The model of Komagataella phaffii SAGA (18) is also in excellent agreement with our map of SLIK (Fig. S5). Like SAGA, SLIK is organized in two lobes that are flexibly connected, as evidenced by the movement of the two lobes with respect to each other in the 2D class averages (Fig. 6, Fig. S3). The bottom lobe is fully occupied by the recruitment module formed by 433 kDa transcription-associated protein 1 (Tra1) (39), while other modules are gathered in a conformationally flexible upper lobe (33, 39, 40). Both lobes are connected through a narrow hinge region (25, 26, 33). As expected, the Spt8 subunit (resolved at lower resolution in the recent SAGA structures (18, 25)) is absent in SLIK (Fig. 6B, Figs. S3 and S5). The resolution of our SLIK model does not allow determination of the orientation of the individual subunits or deducing whether the octamer-like internal structure of the SAGA core module is retained in SLIK. However, in correlation with XL-MS data (Fig. 2), we conclude that the overall arrangement of the core subunits is highly similar in SAGA and SLIK. As reported for SAGA, HAT and DUB modules are poorly resolved due to their inherent flexibility, evident from the 2D class averages (Fig. S3). However, since our XL-MS data for SLIK is in excellent agreement with previous studies (18, 25, 26), we were able to use published structures (18, 25, 33) to model the DUB module and position the HAT module in the extra density present in the cryo-EM map of SLIK (Fig. 6C).Figure 6Cryo-EM of SLIK.A, best 3D class, B, map of SLIK map superimposed with the structural core of S. cerevisiae SAGA (PDB: 6T9I), which excludes HAT and DUB modules that were resolved at lower resolution due to their flexibility (25). Spt7 is indicated with an arrow (modeled up to residue 1085). Spt8 and C-terminus of Spt7 that connects to Spt8 (residues 1086–1332), are absent in the presented SAGA model, since they could not be resolved due to their important flexibility. Spt8 (but not the C-terminus of Spt7) was resolved in K. phaffii SAGA in complex with TBP (PDB: 6TBM, see also Fig. S5); the gray sphere indicates the position of Spt8 in SAGA-TBP complex (18). Tra1 (bottom lobe) is in gray, structural core (top lobe): Ada1 (navy blue), Spt3 (purple), Spt7 (light blue), Spt20 (yellow), Taf5 (pink), Taf6 (orange), Taf9 (magenta), Taf10 (khaki), Taf12 (dark green), DUB module: Sgf73 (light green). C, SLIK model, based on XL-MS data and available cryo-EM structures of S. cerevisiae and K. phaffii SAGA (PDB: 6T9I and 6TBM). DUB module is shown in purple and HAT module is represented as a sphere, due to the lack of structural information available for the individual subunits.

Cryo-EM of SLIK.A, best 3D class, B, map of SLIK map superimposed with the structural core of S. cerevisiae SAGA (PDB: 6T9I), which excludes HAT and DUB modules that were resolved at lower resolution due to their flexibility (25). Spt7 is indicated with an arrow (modeled up to residue 1085). Spt8 and C-terminus of Spt7 that connects to Spt8 (residues 1086–1332), are absent in the presented SAGA model, since they could not be resolved due to their important flexibility. Spt8 (but not the C-terminus of Spt7) was resolved in K. phaffii SAGA in complex with TBP (PDB: 6TBM, see also Fig. S5); the gray sphere indicates the position of Spt8 in SAGA-TBP complex (18). Tra1 (bottom lobe) is in gray, structural core (top lobe): Ada1 (navy blue), Spt3 (purple), Spt7 (light blue), Spt20 (yellow), Taf5 (pink), Taf6 (orange), Taf9 (magenta), Taf10 (khaki), Taf12 (dark green), DUB module: Sgf73 (light green). C, SLIK model, based on XL-MS data and available cryo-EM structures of S. cerevisiae and K. phaffii SAGA (PDB: 6T9I and 6TBM). DUB module is shown in purple and HAT module is represented as a sphere, due to the lack of structural information available for the individual subunits.

DiscussionThe Rtg2 protein is not present in our purified SLIK complex. In the original publication reporting the identification of SLIK, Pray-Grant et al. (14) mentioned that Rtg2 is uniquely associated with this complex. In our hands, Rtg2 could not be identified by mass spectrometry. Our purification protocol includes ammonium sulfate precipitation as well as stringent washes of IgG-bound SAGA and SLIK; in particular 25 column volumes of buffer containing 500 mM (NH4)2SO4, resulting in highly pure SAGA and SLIK. It is possible that Rtg2 is less tightly associated than the other subunits and dissociates upon purification. Pray-Grant et al. also revealed that whole-cell extracts prepared from yeast lacking RTG2 were significantly reduced in their amount of SLIK complex (as assessed by HAT assays and western blots) while maintaining similar amounts of SAGA. Therefore, they concluded that Rtg2 was an essential component of SLIK, required for its structural integrity (14). In contrast, our SLIK complex lacking Rtg2 has seemingly unaffected architecture, HAT, DUB, and DNA-binding activities, raising questions about the role of Rtg2 in the integrity of SLIK complex.Recent structures of SAGA showed that Spt7 occupies the edge of the core module interlacing with Taf6, Taf10, and Spt3 subunits and is essential for SAGA integrity (18, 25, 26). In contrast, Spt8 occupies an external position—it protrudes from the core module to which it is flexibly tethered by the unstructured region of the C-terminus of Spt7 (Fig. 6B) (18). Our results further confirm that neither the modular architecture nor the arrangement of the core subunits is affected by the truncation of Spt7 and absence of Spt8 subunit in SLIK (Figs. 2 and 6).Even though Spt8 has a minor role in TBP binding (18), we showed that the affinity of SAGA for TBP is not significantly affected by the loss of Spt8 and C-terminus of Spt7. This is consistent with previous in vivo studies, where spt8 mutants do not significantly alter the TBP-binding properties of SAGA (41, 42). We also observed the same kinetics for DUB and HAT enzymatic activities on minimal substrates (ubiquitin-AMC for DUB, H3 peptide for HAT) for SAGA and SLIK. This is in agreement with the similarity between cryo-EM structures of SAGA and SLIK; even though DUB and HAT modules are flexible and thus poorly defined in all published maps, the overall architecture of the complex does not seem to be affected by the lack of Spt8 and C-terminus of Spt7 (Figs. 2 and 6). In vivo, however, the chromatin-modifying activities of SAGA and SLIK may be differentially regulated by other factors. In particular, cis-histone cross talk pathways are playing a prominent role in modulation of HAT activity (43): SAGA-mediated HAT is facilitated by phosphorylation on H3 serine 10 and threonine 11 (44, 45), binding of Sgf29 tandem Tudor domain to H3K4 trimethylation (34, 46), binding of Gcn5 bromodomain to acetylated H3K14 (47), interaction of the bromodomain of Spt7 with acetylated H3K9 (48). Comprehensive studies using modified peptides, or modified nucleosomes, are therefore required to establish whether SAGA and SLIK have identical HAT activity. Additionally, interactions with DNA flanking the nucleosome (32), or with other proteins such as activators (32) may differentially modulate HAT activity of SAGA versus SLIK. Finally, given the proximity of Spt8 and DUB module (18, 25), differential steric hindrance and charges in SAGA versus SLIK could affect the DUB activity in the context of interaction with a much larger substrate, such as the nucleosome.SAGA and SLIK both bind DNA with high affinity. Both complexes bind dsDNA with higher affinity than ssDNA. The DNA binding is sequence-independent for both SAGA and SLIK, which agrees with the observation made by Mittal et al. (32) that DNA flanking the nucleosome increases its rate of acetylation by SAGA, regardless of the DNA sequence. Several SAGA subunits have been shown or hypothesized to bind DNA. The 2017 cryo-EM structure of Tra1 revealed that this subunit is strikingly similar to DNA-PK catalytic subunit (DNA-PKcs)—an essential DNA double-strand breaks (DSB) repair factor (39, 49). Tra1 has a highly positively charged surface in the region corresponding to the hypothesized DNA-binding site in DNA-PKcs. However, in Tra1 this region does not form the arch required to accommodate a DNA duplex going through the subunit (39). The zinc finger domain of Sgf11 (DUB module) can also weakly bind double-stranded DNA (50). The fact that SAGA and SLIK are concomitantly able to bind TBP and DNA suggests that the binding sites for TBP and DNA are different. The TBP-binding site is located at the periphery of SAGA and does not exclude DNA binding by Tra1 or the DUB module. In addition to having an effect on SAGA’s HAT activity (32), this DNA-binding property could play a role in the context of TBP loading onto TATA-box. We showed that TBP is not released from either SAGA or SLIK in the presence of DNA containing the TATA-box. However, interaction of SAGA-TBP or SLIK-TBP with DNA may play a role in bringing TBP closer to the promoter, where it would be unloaded through interactions with other factors, such as TFIIA (18). Solving the structure of SAGA/SLIK in complex with DNA is of critical importance as it could inform about whether interaction with DNA is playing a role in TBP loading, chromatin-modifying, or DNA repair activities of SAGA and SLIK.We showed that SAGA and SLIK are functionally equivalent coactivator complexes in yeast. Due to the absence of Spt8 subunit in SLIK, it more closely resembles variants of SAGA found in metazoans, where the Spt8 subunit was lost throughout the evolution (51, 52, 53). Our findings strongly suggest that the absence of Spt8 does not exert any significant effect on the HAT, DUB, TBP-, and DNA-binding activities of SLIK. We propose that Spt8 plays an auxiliary role within the SAGA complex, and more gene-specific specific investigations would be required to reveal any distinct regulatory roles associated with compositional differences of SAGA and SLIK.

The Rtg2 protein is not present in our purified SLIK complex. In the original publication reporting the identification of SLIK, Pray-Grant et al. (14) mentioned that Rtg2 is uniquely associated with this complex. In our hands, Rtg2 could not be identified by mass spectrometry. Our purification protocol includes ammonium sulfate precipitation as well as stringent washes of IgG-bound SAGA and SLIK; in particular 25 column volumes of buffer containing 500 mM (NH4)2SO4, resulting in highly pure SAGA and SLIK. It is possible that Rtg2 is less tightly associated than the other subunits and dissociates upon purification. Pray-Grant et al. also revealed that whole-cell extracts prepared from yeast lacking RTG2 were significantly reduced in their amount of SLIK complex (as assessed by HAT assays and western blots) while maintaining similar amounts of SAGA. Therefore, they concluded that Rtg2 was an essential component of SLIK, required for its structural integrity (14). In contrast, our SLIK complex lacking Rtg2 has seemingly unaffected architecture, HAT, DUB, and DNA-binding activities, raising questions about the role of Rtg2 in the integrity of SLIK complex.

Recent structures of SAGA showed that Spt7 occupies the edge of the core module interlacing with Taf6, Taf10, and Spt3 subunits and is essential for SAGA integrity (18, 25, 26). In contrast, Spt8 occupies an external position—it protrudes from the core module to which it is flexibly tethered by the unstructured region of the C-terminus of Spt7 (Fig. 6B) (18). Our results further confirm that neither the modular architecture nor the arrangement of the core subunits is affected by the truncation of Spt7 and absence of Spt8 subunit in SLIK (Figs. 2 and 6).

Even though Spt8 has a minor role in TBP binding (18), we showed that the affinity of SAGA for TBP is not significantly affected by the loss of Spt8 and C-terminus of Spt7. This is consistent with previous in vivo studies, where spt8 mutants do not significantly alter the TBP-binding properties of SAGA (41, 42). We also observed the same kinetics for DUB and HAT enzymatic activities on minimal substrates (ubiquitin-AMC for DUB, H3 peptide for HAT) for SAGA and SLIK. This is in agreement with the similarity between cryo-EM structures of SAGA and SLIK; even though DUB and HAT modules are flexible and thus poorly defined in all published maps, the overall architecture of the complex does not seem to be affected by the lack of Spt8 and C-terminus of Spt7 (Figs. 2 and 6). In vivo, however, the chromatin-modifying activities of SAGA and SLIK may be differentially regulated by other factors. In particular, cis-histone cross talk pathways are playing a prominent role in modulation of HAT activity (43): SAGA-mediated HAT is facilitated by phosphorylation on H3 serine 10 and threonine 11 (44, 45), binding of Sgf29 tandem Tudor domain to H3K4 trimethylation (34, 46), binding of Gcn5 bromodomain to acetylated H3K14 (47), interaction of the bromodomain of Spt7 with acetylated H3K9 (48). Comprehensive studies using modified peptides, or modified nucleosomes, are therefore required to establish whether SAGA and SLIK have identical HAT activity. Additionally, interactions with DNA flanking the nucleosome (32), or with other proteins such as activators (32) may differentially modulate HAT activity of SAGA versus SLIK. Finally, given the proximity of Spt8 and DUB module (18, 25), differential steric hindrance and charges in SAGA versus SLIK could affect the DUB activity in the context of interaction with a much larger substrate, such as the nucleosome.

SAGA and SLIK both bind DNA with high affinity. Both complexes bind dsDNA with higher affinity than ssDNA. The DNA binding is sequence-independent for both SAGA and SLIK, which agrees with the observation made by Mittal et al. (32) that DNA flanking the nucleosome increases its rate of acetylation by SAGA, regardless of the DNA sequence. Several SAGA subunits have been shown or hypothesized to bind DNA. The 2017 cryo-EM structure of Tra1 revealed that this subunit is strikingly similar to DNA-PK catalytic subunit (DNA-PKcs)—an essential DNA double-strand breaks (DSB) repair factor (39, 49). Tra1 has a highly positively charged surface in the region corresponding to the hypothesized DNA-binding site in DNA-PKcs. However, in Tra1 this region does not form the arch required to accommodate a DNA duplex going through the subunit (39). The zinc finger domain of Sgf11 (DUB module) can also weakly bind double-stranded DNA (50). The fact that SAGA and SLIK are concomitantly able to bind TBP and DNA suggests that the binding sites for TBP and DNA are different. The TBP-binding site is located at the periphery of SAGA and does not exclude DNA binding by Tra1 or the DUB module. In addition to having an effect on SAGA’s HAT activity (32), this DNA-binding property could play a role in the context of TBP loading onto TATA-box. We showed that TBP is not released from either SAGA or SLIK in the presence of DNA containing the TATA-box. However, interaction of SAGA-TBP or SLIK-TBP with DNA may play a role in bringing TBP closer to the promoter, where it would be unloaded through interactions with other factors, such as TFIIA (18). Solving the structure of SAGA/SLIK in complex with DNA is of critical importance as it could inform about whether interaction with DNA is playing a role in TBP loading, chromatin-modifying, or DNA repair activities of SAGA and SLIK.

We showed that SAGA and SLIK are functionally equivalent coactivator complexes in yeast. Due to the absence of Spt8 subunit in SLIK, it more closely resembles variants of SAGA found in metazoans, where the Spt8 subunit was lost throughout the evolution (51, 52, 53). Our findings strongly suggest that the absence of Spt8 does not exert any significant effect on the HAT, DUB, TBP-, and DNA-binding activities of SLIK. We propose that Spt8 plays an auxiliary role within the SAGA complex, and more gene-specific specific investigations would be required to reveal any distinct regulatory roles associated with compositional differences of SAGA and SLIK.

Experimental proceduresSAGA and SLIK purificationS. cerevisiae TAP-tagged strains used in this study were purchased from Dharmacon. SAGA used for HAT, DUB, and DNA-binding studies was purified from Spt7-TAP strain (selectively purifies SAGA without SLIK). SLIK was purified from Ubp8-TAP strain. Selective agar plates were streaked from glycerol stocks and incubated for 72 h at 30 °C. Single colonies were then picked to grow overnight culture in YPD media (1% (w/v) yeast extract, 2% (w/v) peptone, 3% (w/v) glucose). Harvested cells were resuspended in equal volume of lysis buffer 400 mM (NH4)2SO4, 200 mM HEPES, pH 7.9, 20 μM ZnSO4, 20% (w/v) glycerol, 3 mM DTT, supplemented with protease inhibitor cocktail: PMSF 3.4% (w/v), leupeptin 5.68% (w/v), pepstatin A 0.0274% (w/v), benzamidine HCl 6.6% (w/v), and lysed in a DYNO-Mill (Multi Lab WAB). Nucleic acids were precipitated with 0.2% (w/v) polyethyleneimine. Lysate was cleared by centrifugation in a JLA8.1 rotor (Beckman Coulter) at 8000× rpm, 4 °C for 1 h 2 M final concentration of ammonium sulfate was used for salt-induced precipitation of total soluble protein fraction from the lysate. Protein pellet was resolubilized in 75 mM (NH4)2SO4, 50 mM HEPES, pH 7.6, 10 μM ZnSO4, 1 mM DTT and further clarified by ultracentrifugation in a Ti-45 rotor (Beckman Coulter) at 45,000× rpm, 4 °C for 1 h. The conductivity of the supernatant was adjusted to ∼50 mS/cm with ammonium sulfate and loaded onto a pre-equilibrated IgG column. The column was prepared by coupling IgG purified from rabbit serum (Scripps Laboratories) to the CNBr-activated sepharose 4B (GE Healthcare) as per manufacturers’ instructions. The bound complex was extensively washed with high and low-salt buffers containing 500 mM and 25 mM ammonium sulfate, respectively and further subjected to the overnight cleavage with 200 μg TEV protease at 4 °C. Complexes were eluted with 200 mM (NH4)2SO4, 50 mM HEPES, pH 6.8/7.6, 10 μM ZnSO4, 1 mM DTT. Selected fractions were concentrated and further purified by affinity chromatography on HiTrap Heparin HP column (GE Healthcare). Elution was performed with a stepped gradient of up to 1 M ammonium sulfate. Salt concentration of the protein fraction was adjusted back to 200 mM ammonium sulfate. Protein fraction was concentrated and subjected to gel filtration chromatography on Superose 6 10/300 column pre-equilibrated in 200 mM (NH4)2SO4, 50 mM HEPES, pH 6.8/7.6, 10 μM ZnSO4, 1 mM DTT. Purification of the SLIK complex involved an additional ion-exchange chromatography step on MonoQ GL 5/50 column (GE Healthcare Life Sciences). Conductivity of the protein sample was adjusted to 200 mM sodium chloride and loaded onto MonoQ equilibrated in low-salt buffer (200 mM NaCl, 50 mM HEPES, pH 6.8, 10 μM ZnSO4, 1 mM DTT). SLIK complex was eluted with the stepped gradient of up to 1 M sodium chloride as previously described (14). Collected fractions were buffer exchanged into 200 mM (NH4)2SO4, 50 mM HEPES, pH 6.8, 10 μM ZnSO4, 1 mM DTT for protein stability.TBP purificationFull-length yeast S. cerevisiae TBP gene was amplified by PCR from genomic DNA and cloned in frame with a cleavable N-terminal His-tag in pProEx vector. Transformed BL21 pLysS cells were grown in LB medium until an OD600 nm of 0.8 was reached. Expression was induced with 1 mM IPTG for 3 h at 37 °C. The cells were collected by centrifugation and frozen. Thawed cells were resuspended in lysis buffer (500 mM NaCl, 20 mM HEPES pH 7.5, 10% glycerol, 1 mM DTT), treated with 1 mg/ml lysozyme for 30 min at room temperature, and lysed by sonication in the presence of protease inhibitors. The resulting lysate was clarified by centrifugation and loaded onto a Protino Ni-NTA 5 ml column. The column was washed with lysis buffer supplemented with 50 mM imidazole and eluted with a gradient from 50 to 500 mM imidazole. The elution was dialyzed overnight at 4 °C in 350 mM NaCl, 20 mM HEPES pH 7.5, 10% glycerol, 3 mM DTT, in the presence of 200 μg TEV protease to cleave the His-tag away. Sample was then loaded on a Protino Ni-NTA 5 ml column and flow-through was collected. NaCl concentration was then adjusted to 200 mM, and the protein was loaded onto a Heparin HiTrap 1 ml column (Cytiva) equilibrated with 200 mM NaCl, 20 mM HEPES pH 7.5, 10% glycerol, 3 mM DTT. TBP was eluted with a gradient to 1 M NaCl. Fractions were analyzed by Coomassie blue staining, pooled, and concentrated in Amicon-Ultra (Millipore) to a final concentration of 3.4 mg/ml.Cross-linking mass spectrometryIn total, 150 μg of freshly purified SAGA/SLIK complex was buffer exchanged into assay buffer (200 mM K2SO4, 50 mM HEPES pH 6.8/7.6, 2 mM MgSO4, 10 μM ZnSO4, 3 mM DTT), on a PD MiniTrap G-25 column (GE Healthcare) as per standard manufacturer’s protocol and split into two 1.5 ml LoBind tubes (Eppendorf) with a total volume of 100 μl each. In total, 2 mg of bis(sulfosuccinimidyl)suberate (BS3) cross-linker (Thermo Scientific) was dissolved in the assay buffer to a final concentration of 100 mM. 20,000× molar excess of BS3 was added, and the samples were incubated for 2 h at room temperature. Controls were prepared with the assay buffer in place of the cross-linker. The reaction was quenched with 100 mM Tris pH 8.0 and incubated for 10 min at room temperature. 0.8 M Urea was added to promote protein unfolding. Protein samples were reduced with 10 mM TCEP (Thermo Scientific) for 45 min at 55 °C and alkylated with 40 mM chloroacetamide (Sigma Aldrich) for 20 min at room temperature in the dark. Samples were digested with trypsin (1:10 (w/w) ratio) (Promega) at 37 °C, 600× rpm overnight. Digestion was stopped with 1% formic acid (Sigma Aldrich) and purified by zip tipping using OMIX C18 100 μl pipette tips (Agilent). The tip was rehydrated with 50% acetonitrile/0.1% formic acid (Buffer B) and equilibrated with 0.1% formic acid (Buffer A). The samples were bound to the resin bed of the tip by repeated pipetting up and down 10 to 15× in the sample vial. The tip was washed three times with Buffer A. Purified peptides were eluted with Buffer B into a new 1.5 ml LoBind tube (Eppendorf). Samples were vacuum concentrated to ∼5 μl in a SpeedVac (Thermo Scientific) and reconstituted in 20 μl of Buffer A. Both cross-linked and control samples were analyzed on a QExactive HF (Thermo Scientific) mass spectrometer coupled to a Dionex Ultimate 3000 RSLCnano system (Thermo Scientific). In detail, peptides were immobilized on an Acclaim PepMap 100 (100 μm × 2 cm, nanoViper, C18, 5 μm, 100 Å; Thermo Scientific) trap column and separated on an Acclaim PepMap RSLC (75 μm × 50 cm, nanoViper, C18, 2 μm, 100 Å; Thermo Scientific) analytical column by increasing concentrations of 80% acetonitrile/0.1% formic acid at a flow of 250 nl/min for 90 min. All samples were analyzed with pLink2 (v2.3.9) (54) using a custom database containing the protein sequences of SAGA or SLIK components in addition to common contaminant sequences (312 database entries). Trypsin/P was selected as protease, up to three missed cleavages were permitted, and precursor and fragment mass tolerances were set to 10 and 20 ppm, respectively. Carbamidomethylation of cysteine residues was selected as fixed modification, while oxidation of methionine residues and acetylation of protein N-termini were selected as variable modifications. Only peptides falling below an e-value cutoff of 0.01 were considered for further analysis, and cross-linked peptides were plotted using xVis.Enzyme kinetics—DUBSteady-state kinetic measurements were performed at room temperature in a CLARIOstar Plus plate reader (BMG Labtech), using 350 nm and 440 nm as an excitation and emission wavelengths, respectively. Deubiquitination (DUB) assays were carried out in a 384-well low-volume polystyrene microplates (CORNING), using Ub-AMC (UbiQ) dissolved in 100% DMSO as a substrate. Reactions were assembled in a total volume of 30 μl in assay buffer containing 100 mM (NH4)2SO4, 50 mM HEPES, pH 7.6, 10 μM ZnSO4, 1 mM DTT. In order to find a point where initial rate was independent of enzyme concentration, enzyme was titrated into a 2 μM fixed concentration of Ub-AMC. Measurement was performed for 2 h and initial rates were plotted against the enzyme concentration. Based on obtained results, the enzyme concentration of 75 nM (2.9 μl of SAGA or SLIK) was determined as optimal for steady-state kinetic measurements (data not shown). For DUB assays the enzyme concentration was kept constant, whereas the concentration of Ub-AMC was varied between 0.5 μM and 20 μM. Final concentration of DMSO was held at 4.44%. Reaction mix for each substrate concentration was prepared and pipetted into the wells. Reactions were initiated by the addition of enzyme and the release of AMC was monitored at 440 nm until the total fluorescence was invariant in time. Appropriate blank reactions were prepared and used for background correction. The calibration curve was generated by plotting fluorescence versus substrate concentration at a point where the substrate was fully converted i.e., total fluorescence was invariant in time. Each experiment was performed in duplicate. Kinetic parameters (Km, Vmax) were generated by fitting initial rates to the Michaelis–Menten equation using GraphPad Prism 8.Enzyme kinetics—HATSteady-state kinetic measurements using H3 peptide substrate were done with a continuous spectrophotometric assay as previously described (34). Each 50 μl reaction contained 50 mM (NH4)2SO4, 50 mM HEPES, pH 7.6, 5 mM MgCl2, 1 mM DTT, 0.2 mM thiamine pyrophosphate, 0.2 mM NAD+, 2.5 mM pyruvate, 1 μl of 0.57 U/mg at 15 mg/ml pyruvate dehydrogenase (Sigma), 100 μM acetyl-CoA, 50 nM SAGA or SLIK, and 0 to 1500 μM histone H3 peptide 1 to 21 (sequence ARTKQTARKSTGGKAPRKQLA, purchased at ≥95% purity from Cayman Chemical).All the reaction components, except for acetyl-CoA, were assembled in clear 96-well plates and incubated at room temperature for 5 min. Reactions were initiated by the addition of acetyl-CoA, and the absorbance at 340 nm was monitored continuously using a FLUOstar Omega plate reader (BMG Labtech) for 5 to 60 min. Absorbance at 340 nm was converted into the molar concentration of NADH using Beer’s Law, assuming ε340 nm = 6220 M−1 cm−1. To calculate initial rates, NADH production was plotted as a function of time and fit to a line where initial velocity conditions were satisfied, typically within the first 20 min. A blank reaction containing the HAT module and acetyl-CoA, but no substrate, was performed for each titration and the rate was subtracted as background from the other reactions. Initial rates were measured in duplicate, normalized to the enzyme concentration, and plotted as a function of substrate concentration. The resulting curve was fit to the Michaelis–Menten equation using nonlinear least squares regression implemented in GraphPad Prism 8.Protein–ligand binding assay by fluorescence polarizationFP titrations were carried out in a 384-well low-volume microplates (CORNING) at room temperature in a PHERAstar FSX plate reader (BMG Labtech), equipped with polarizing fluorescent filters. Protein dilutions (100 nM–3.125 nM, final concentrations) were prepared in the protein storage buffer (200 mM (NH4)2SO4, 50 mM HEPES, pH 6.8, 10 μM ZnSO4, 1 mM DTT). SAGA and SLIK were processed in parallel and adjusted to the same concentration so that an identical volume of protein dilution was added to each reaction (3.9 μl). Reactions were performed in assay buffer (50 mM CH3CO2K, 50 mM HEPES, pH 6.8, 2 mM MgSO4, 10 μM ZnSO4, 1 mM DTT), supplemented with 0.75 pmol per well of oligonucleotides 5’-labeled with Cy3 (IDT) (final concentration 25 nM) (see sequences in Supporting information). Reactions were assembled in a total volume of 30 μl and incubated for 10 min at room temperature. Measurements were carried out using fluorophore-specific excitation/emission wavelengths (540/590 for Cy3). Each titration curve was made in triplicate (except for 20-mer with TATA DNA, that was made in duplicate). Blank reactions without protein were prepared and used for background correction. Binding curves were generated by plotting FP versus protein concentration. Binding parameters were estimated by fitting in Graphpad Prism 8. For the competition experiments, 25 nM of 75-mer dsDNA was used as above, in the same assay buffer. SAGA or SLIK was added to 50 nM concentration (corresponding to the plateau previously characterized in these conditions). Reactions were supplemented with either no competitor, Poly(dI-dC) competitor (1, 10, 100 times weight excess, relative to dsDNA; i.e., 37, 370, or 3700 ng per reaction), 200-mer dsDNA competitor corresponding to the TATA-less promoter of the ribosomal protein 5 gene (RPS5) (1, 5, 10, 100 times weight excess, relative to dsDNA; i.e., 37, 185, 370 or 3700 ng per reaction), or unlabeled 75-mer dsDNA with the same sequence as the DNA probe (1, 5, 10, 50 times weight/molar excess, relative to dsDNA). Conditions were prepared in triplicates (Poly(dI-dC)), duplicates (RPS5), or single repeat (unlabeled 75-mer dsDNA). Blank reactions without protein but with DNA and appropriate amount of Poly(dI-dC) were used for background correction; all blanks had similar FP values. Reactions were assembled in a 384-well low-volume polystyrene microplates (CORNING) at room temperature and incubated for 10 min. Measurements were carried out using 540/590 excitation/emission wavelengths in a PHERAstar FSX plate reader (BMG Labtech), using a polarizing fluorescent filter. FP signal was normalized to 100% for reactions without competitor.Binding to 75-mer dsDNA was virtually identical for SAGA purified from Spt7-TAP strain and SAGA purified from Ubp8-TAP strain (not shown).Gel shift TBP- and DNA-binding assayFor TBP-binding assays, purified TBP was fluorescently labeled with OneQuant Fluorescein Isothiocyanate (FITC) (G-Biosciences). One milligram of FITC was warmed up to room temperature and resuspended in 10 μl DMSO. In total, 250 μl NaHCO3/Na2CO3 buffer at 0.2 M concentration was added, followed by 250 μl TBP at 3.4 mg/ml (final pH 10). Sample was incubated overnight at 4 °C, with gentle rocking, protected from light. Excess dye was removed using a desalting column (PD MiniTrap, Cytiva), followed by a gel filtration column (S75 10/300) in 500 mM NaCl, 20 mM HEPES pH 7.5, 10% glycerol, 3 mM DTT. Protein concentration was estimated by densitometry, by comparing dilutions of FITC-TBP with titrations of unlabeled TBP at known concentrations. Binding assays (10 μl) contained 3 mM Tris-HCl (pH 8.0), 6 mM HEPES-KOH (pH 8.0), 45 mM CH3COOK, 42 mM KCl, 5 mM MgCl2, 3% glycerol, 6% sucrose, 0.01% NP40, 25 μg/ml BSA, 1 mM DTT, plus proteins. FITC-TBP concentration was maintained at 10 nM while SAGA and SLIK were titrated from 0 to 150 nM. Serial dilutions of SAGA and SLIK were done in storage buffer and the same amount of buffer was added to sample without SAGA/SLIK to keep the salt concentration constant. Proteins were incubated together for 1 h at 4 °C, and the samples were loaded onto horizontal agarose-polyacrylamide native gels (0.3% agarose, 2% acrylamide with 0.5X TBE in the gel and running buffer). The gel was run at 90 V for 3 h and scanned using a Typhoon 5 to follow FITC fluorescence of TBP. For DNA-binding assays, DNA probes used were the same as for FP experiments (75-mer and 20-mer dsDNA, both containing a TATA-box, Cy3-labeled). The nonlabeled DNA was 200-mer dsDNA corresponding to the TATA-less promoter of the ribosomal protein 5 gene (RPS5). Binding assays (10 μl) contained 3 mM Tris-HCl (pH 8.0), 6 mM HEPES-KOH (pH 8.0), 45 mM CH3COOK, 42 mM KCl, 5 mM MgCl2, 3% glycerol, 6% sucrose, 0.01% NP40, 25 μg/ml BSA, 1 mM DTT, plus proteins and DNA. The protein concentrations were 75 nM TBP, 300 nM SAGA or SLIK, 75 nM Cy3-DNA, 900 nM nonlabeled TATA-less DNA. TBP was incubated first with a fourfold excess of SAGA or SLIK complex for 1 h at 4 °C. Then DNA was added, and the reaction mixtures were incubated for 20 min at room temperature before they were loaded onto horizontal agarose-polyacrylamide native gels (1% agarose, 1.3% acrylamide with 1X Tris-glycine buffer, 5 mM MgCl2 in the gel and running buffer). The gel was run at 30 V overnight and scanned using a Typhoon 5 to follow Cy3 fluorescence of DNA. Composition of the shifted bands was verified by mass spectrometry (Tables S4 and S5).Sample stabilization with GraFix for cryo-EMFor electron microscopy studies, proteins were stabilized by the optimized GraFix procedure (36); 10 to 40% continuous glycerol gradients were prepared by mixing light solution (500 mM CH3COOK, 25 mM HEPES, pH 6.8, 2 mM MgSO4, 10 μM ZnSO4, 10% (w/v) glycerol) with heavy solution (500 mM CH3COOK, 25 mM HEPES, pH 6.8, 2 mM MgSO4, 10 μM ZnSO4, 40% (w/v) glycerol, 0.3% (v/v) glutaraldehyde) using the Gradient Master (BioComp Instruments). In total, 100 pmol of the SLIK complex was applied on top of the gradient and was subject to ultracentrifugation in an SW60 Ti rotor (Beckman Coulter) at 30,000× rpm, 4 °C for 15 h. Following ultracentrifugation, the gradients were fractionated using the Piston Gradient Fractionator (BioComp Instruments). Obtained UV280 profile indicated that the complex was eluted as a single peak positioned approximately 2/3 down the gradient. Selected peak fractions were pooled and quenched with 50 mM final concentration of ammonium sulfate to prevent further cross-linking.Cryo-EM sample preparationGrids used were ultrathin Carbon film supported by a lacey Carbon film on a 400 mesh, copper (Pelco). In order to remove glycerol required for GraFix, sample was buffer-exchanged on the grid: 2.5 μl sample was applied, manually blotted away; 2.5 μl buffer (500 mM CH3COOK, 25 mM HEPES, pH 6.8, 2 mM MgSO4, 10 μM ZnSO4) was applied and manually blotted away; 2.5 μl buffer was applied, and the grid was vitrified in liquid ethane using a Vitrobot IV (FEI). Cryo-grids were screened under low-dose conditions on the Tecnai T12 Spirit microscope (FEI) at 120 kV, to identify the optimal freezing conditions (ice thickness, contrast, and particles distribution).Cryo-EM data collection and image processingImages were collected on a Titan Krios FEG-TEM electron microscope (FEI) operated at 300 kV and recorded at a magnification of 105,000 using a K2 Summit direct electron detector camera (Gatan) in counting mode at a calibrated pixel size of 1.36 Å. In total, 1894 and 3175 dose-fractionated movies were recorded during two independent collections with the specimen subjected to total accumulated doses of 50.2 and 50.3 e−/Å2 respectively. Individual movies were corrected for stage and beam-induced motion and CTF parameters calculated using SIMPLE3.0 (Figs. S3 and S4). Particles’ positions were automatically identified with template-based picking with the structure of SAGA resolution limited to 20 Å (EMD-10412) (25) and SIMPLE3.0. The initial set of 292,323 particles was refined to final set of 44,021 though iterative rounds of 2D classification (55). The class averages thus generated were used to determine an ab initio 3D model for further single particles refinement using RELION3.0 (Fig. S3, resolution limit of 30 Å) (56, 57). A consensus 3D model was first obtained at 23.6 Å resolution (gold-standard FSC criterion of 0.143) and subjected to 3D classification to yield three classes populated by 16,623, 14,188, and 13,210 particles individually refined to resolutions 22.5, 29.1, and 29.1 Å respectively (Fig. 6, Figs. S3 and S4).Model buildingOur XL-MS data are consistent with published models of SAGA and DUB module (18, 25, 26). We therefore used the structure of the structural core of S. cerevisiae SAGA (PDB: 6T9I) (25) and docked it manually into the map. DUB module of K. phaffii SAGA-TBP (PDB: 6TBM) (18) was docked based on XL-MS data and its location and orientation in K. phaffii SAGA. HAT module was modeled as a sphere due to the absence of structural information for the full module and docked in the density based on its position in published structures (18, 25, 33).

Experimental procedures

SAGA and SLIK purificationS. cerevisiae TAP-tagged strains used in this study were purchased from Dharmacon. SAGA used for HAT, DUB, and DNA-binding studies was purified from Spt7-TAP strain (selectively purifies SAGA without SLIK). SLIK was purified from Ubp8-TAP strain. Selective agar plates were streaked from glycerol stocks and incubated for 72 h at 30 °C. Single colonies were then picked to grow overnight culture in YPD media (1% (w/v) yeast extract, 2% (w/v) peptone, 3% (w/v) glucose). Harvested cells were resuspended in equal volume of lysis buffer 400 mM (NH4)2SO4, 200 mM HEPES, pH 7.9, 20 μM ZnSO4, 20% (w/v) glycerol, 3 mM DTT, supplemented with protease inhibitor cocktail: PMSF 3.4% (w/v), leupeptin 5.68% (w/v), pepstatin A 0.0274% (w/v), benzamidine HCl 6.6% (w/v), and lysed in a DYNO-Mill (Multi Lab WAB). Nucleic acids were precipitated with 0.2% (w/v) polyethyleneimine. Lysate was cleared by centrifugation in a JLA8.1 rotor (Beckman Coulter) at 8000× rpm, 4 °C for 1 h 2 M final concentration of ammonium sulfate was used for salt-induced precipitation of total soluble protein fraction from the lysate. Protein pellet was resolubilized in 75 mM (NH4)2SO4, 50 mM HEPES, pH 7.6, 10 μM ZnSO4, 1 mM DTT and further clarified by ultracentrifugation in a Ti-45 rotor (Beckman Coulter) at 45,000× rpm, 4 °C for 1 h. The conductivity of the supernatant was adjusted to ∼50 mS/cm with ammonium sulfate and loaded onto a pre-equilibrated IgG column. The column was prepared by coupling IgG purified from rabbit serum (Scripps Laboratories) to the CNBr-activated sepharose 4B (GE Healthcare) as per manufacturers’ instructions. The bound complex was extensively washed with high and low-salt buffers containing 500 mM and 25 mM ammonium sulfate, respectively and further subjected to the overnight cleavage with 200 μg TEV protease at 4 °C. Complexes were eluted with 200 mM (NH4)2SO4, 50 mM HEPES, pH 6.8/7.6, 10 μM ZnSO4, 1 mM DTT. Selected fractions were concentrated and further purified by affinity chromatography on HiTrap Heparin HP column (GE Healthcare). Elution was performed with a stepped gradient of up to 1 M ammonium sulfate. Salt concentration of the protein fraction was adjusted back to 200 mM ammonium sulfate. Protein fraction was concentrated and subjected to gel filtration chromatography on Superose 6 10/300 column pre-equilibrated in 200 mM (NH4)2SO4, 50 mM HEPES, pH 6.8/7.6, 10 μM ZnSO4, 1 mM DTT. Purification of the SLIK complex involved an additional ion-exchange chromatography step on MonoQ GL 5/50 column (GE Healthcare Life Sciences). Conductivity of the protein sample was adjusted to 200 mM sodium chloride and loaded onto MonoQ equilibrated in low-salt buffer (200 mM NaCl, 50 mM HEPES, pH 6.8, 10 μM ZnSO4, 1 mM DTT). SLIK complex was eluted with the stepped gradient of up to 1 M sodium chloride as previously described (14). Collected fractions were buffer exchanged into 200 mM (NH4)2SO4, 50 mM HEPES, pH 6.8, 10 μM ZnSO4, 1 mM DTT for protein stability.

SAGA and SLIK purification

S. cerevisiae TAP-tagged strains used in this study were purchased from Dharmacon. SAGA used for HAT, DUB, and DNA-binding studies was purified from Spt7-TAP strain (selectively purifies SAGA without SLIK). SLIK was purified from Ubp8-TAP strain. Selective agar plates were streaked from glycerol stocks and incubated for 72 h at 30 °C. Single colonies were then picked to grow overnight culture in YPD media (1% (w/v) yeast extract, 2% (w/v) peptone, 3% (w/v) glucose). Harvested cells were resuspended in equal volume of lysis buffer 400 mM (NH4)2SO4, 200 mM HEPES, pH 7.9, 20 μM ZnSO4, 20% (w/v) glycerol, 3 mM DTT, supplemented with protease inhibitor cocktail: PMSF 3.4% (w/v), leupeptin 5.68% (w/v), pepstatin A 0.0274% (w/v), benzamidine HCl 6.6% (w/v), and lysed in a DYNO-Mill (Multi Lab WAB). Nucleic acids were precipitated with 0.2% (w/v) polyethyleneimine. Lysate was cleared by centrifugation in a JLA8.1 rotor (Beckman Coulter) at 8000× rpm, 4 °C for 1 h 2 M final concentration of ammonium sulfate was used for salt-induced precipitation of total soluble protein fraction from the lysate. Protein pellet was resolubilized in 75 mM (NH4)2SO4, 50 mM HEPES, pH 7.6, 10 μM ZnSO4, 1 mM DTT and further clarified by ultracentrifugation in a Ti-45 rotor (Beckman Coulter) at 45,000× rpm, 4 °C for 1 h. The conductivity of the supernatant was adjusted to ∼50 mS/cm with ammonium sulfate and loaded onto a pre-equilibrated IgG column. The column was prepared by coupling IgG purified from rabbit serum (Scripps Laboratories) to the CNBr-activated sepharose 4B (GE Healthcare) as per manufacturers’ instructions. The bound complex was extensively washed with high and low-salt buffers containing 500 mM and 25 mM ammonium sulfate, respectively and further subjected to the overnight cleavage with 200 μg TEV protease at 4 °C. Complexes were eluted with 200 mM (NH4)2SO4, 50 mM HEPES, pH 6.8/7.6, 10 μM ZnSO4, 1 mM DTT. Selected fractions were concentrated and further purified by affinity chromatography on HiTrap Heparin HP column (GE Healthcare). Elution was performed with a stepped gradient of up to 1 M ammonium sulfate. Salt concentration of the protein fraction was adjusted back to 200 mM ammonium sulfate. Protein fraction was concentrated and subjected to gel filtration chromatography on Superose 6 10/300 column pre-equilibrated in 200 mM (NH4)2SO4, 50 mM HEPES, pH 6.8/7.6, 10 μM ZnSO4, 1 mM DTT. Purification of the SLIK complex involved an additional ion-exchange chromatography step on MonoQ GL 5/50 column (GE Healthcare Life Sciences). Conductivity of the protein sample was adjusted to 200 mM sodium chloride and loaded onto MonoQ equilibrated in low-salt buffer (200 mM NaCl, 50 mM HEPES, pH 6.8, 10 μM ZnSO4, 1 mM DTT). SLIK complex was eluted with the stepped gradient of up to 1 M sodium chloride as previously described (14). Collected fractions were buffer exchanged into 200 mM (NH4)2SO4, 50 mM HEPES, pH 6.8, 10 μM ZnSO4, 1 mM DTT for protein stability.

TBP purificationFull-length yeast S. cerevisiae TBP gene was amplified by PCR from genomic DNA and cloned in frame with a cleavable N-terminal His-tag in pProEx vector. Transformed BL21 pLysS cells were grown in LB medium until an OD600 nm of 0.8 was reached. Expression was induced with 1 mM IPTG for 3 h at 37 °C. The cells were collected by centrifugation and frozen. Thawed cells were resuspended in lysis buffer (500 mM NaCl, 20 mM HEPES pH 7.5, 10% glycerol, 1 mM DTT), treated with 1 mg/ml lysozyme for 30 min at room temperature, and lysed by sonication in the presence of protease inhibitors. The resulting lysate was clarified by centrifugation and loaded onto a Protino Ni-NTA 5 ml column. The column was washed with lysis buffer supplemented with 50 mM imidazole and eluted with a gradient from 50 to 500 mM imidazole. The elution was dialyzed overnight at 4 °C in 350 mM NaCl, 20 mM HEPES pH 7.5, 10% glycerol, 3 mM DTT, in the presence of 200 μg TEV protease to cleave the His-tag away. Sample was then loaded on a Protino Ni-NTA 5 ml column and flow-through was collected. NaCl concentration was then adjusted to 200 mM, and the protein was loaded onto a Heparin HiTrap 1 ml column (Cytiva) equilibrated with 200 mM NaCl, 20 mM HEPES pH 7.5, 10% glycerol, 3 mM DTT. TBP was eluted with a gradient to 1 M NaCl. Fractions were analyzed by Coomassie blue staining, pooled, and concentrated in Amicon-Ultra (Millipore) to a final concentration of 3.4 mg/ml.

TBP purification

Full-length yeast S. cerevisiae TBP gene was amplified by PCR from genomic DNA and cloned in frame with a cleavable N-terminal His-tag in pProEx vector. Transformed BL21 pLysS cells were grown in LB medium until an OD600 nm of 0.8 was reached. Expression was induced with 1 mM IPTG for 3 h at 37 °C. The cells were collected by centrifugation and frozen. Thawed cells were resuspended in lysis buffer (500 mM NaCl, 20 mM HEPES pH 7.5, 10% glycerol, 1 mM DTT), treated with 1 mg/ml lysozyme for 30 min at room temperature, and lysed by sonication in the presence of protease inhibitors. The resulting lysate was clarified by centrifugation and loaded onto a Protino Ni-NTA 5 ml column. The column was washed with lysis buffer supplemented with 50 mM imidazole and eluted with a gradient from 50 to 500 mM imidazole. The elution was dialyzed overnight at 4 °C in 350 mM NaCl, 20 mM HEPES pH 7.5, 10% glycerol, 3 mM DTT, in the presence of 200 μg TEV protease to cleave the His-tag away. Sample was then loaded on a Protino Ni-NTA 5 ml column and flow-through was collected. NaCl concentration was then adjusted to 200 mM, and the protein was loaded onto a Heparin HiTrap 1 ml column (Cytiva) equilibrated with 200 mM NaCl, 20 mM HEPES pH 7.5, 10% glycerol, 3 mM DTT. TBP was eluted with a gradient to 1 M NaCl. Fractions were analyzed by Coomassie blue staining, pooled, and concentrated in Amicon-Ultra (Millipore) to a final concentration of 3.4 mg/ml.

Cross-linking mass spectrometryIn total, 150 μg of freshly purified SAGA/SLIK complex was buffer exchanged into assay buffer (200 mM K2SO4, 50 mM HEPES pH 6.8/7.6, 2 mM MgSO4, 10 μM ZnSO4, 3 mM DTT), on a PD MiniTrap G-25 column (GE Healthcare) as per standard manufacturer’s protocol and split into two 1.5 ml LoBind tubes (Eppendorf) with a total volume of 100 μl each. In total, 2 mg of bis(sulfosuccinimidyl)suberate (BS3) cross-linker (Thermo Scientific) was dissolved in the assay buffer to a final concentration of 100 mM. 20,000× molar excess of BS3 was added, and the samples were incubated for 2 h at room temperature. Controls were prepared with the assay buffer in place of the cross-linker. The reaction was quenched with 100 mM Tris pH 8.0 and incubated for 10 min at room temperature. 0.8 M Urea was added to promote protein unfolding. Protein samples were reduced with 10 mM TCEP (Thermo Scientific) for 45 min at 55 °C and alkylated with 40 mM chloroacetamide (Sigma Aldrich) for 20 min at room temperature in the dark. Samples were digested with trypsin (1:10 (w/w) ratio) (Promega) at 37 °C, 600× rpm overnight. Digestion was stopped with 1% formic acid (Sigma Aldrich) and purified by zip tipping using OMIX C18 100 μl pipette tips (Agilent). The tip was rehydrated with 50% acetonitrile/0.1% formic acid (Buffer B) and equilibrated with 0.1% formic acid (Buffer A). The samples were bound to the resin bed of the tip by repeated pipetting up and down 10 to 15× in the sample vial. The tip was washed three times with Buffer A. Purified peptides were eluted with Buffer B into a new 1.5 ml LoBind tube (Eppendorf). Samples were vacuum concentrated to ∼5 μl in a SpeedVac (Thermo Scientific) and reconstituted in 20 μl of Buffer A. Both cross-linked and control samples were analyzed on a QExactive HF (Thermo Scientific) mass spectrometer coupled to a Dionex Ultimate 3000 RSLCnano system (Thermo Scientific). In detail, peptides were immobilized on an Acclaim PepMap 100 (100 μm × 2 cm, nanoViper, C18, 5 μm, 100 Å; Thermo Scientific) trap column and separated on an Acclaim PepMap RSLC (75 μm × 50 cm, nanoViper, C18, 2 μm, 100 Å; Thermo Scientific) analytical column by increasing concentrations of 80% acetonitrile/0.1% formic acid at a flow of 250 nl/min for 90 min. All samples were analyzed with pLink2 (v2.3.9) (54) using a custom database containing the protein sequences of SAGA or SLIK components in addition to common contaminant sequences (312 database entries). Trypsin/P was selected as protease, up to three missed cleavages were permitted, and precursor and fragment mass tolerances were set to 10 and 20 ppm, respectively. Carbamidomethylation of cysteine residues was selected as fixed modification, while oxidation of methionine residues and acetylation of protein N-termini were selected as variable modifications. Only peptides falling below an e-value cutoff of 0.01 were considered for further analysis, and cross-linked peptides were plotted using xVis.

Cross-linking mass spectrometry

In total, 150 μg of freshly purified SAGA/SLIK complex was buffer exchanged into assay buffer (200 mM K2SO4, 50 mM HEPES pH 6.8/7.6, 2 mM MgSO4, 10 μM ZnSO4, 3 mM DTT), on a PD MiniTrap G-25 column (GE Healthcare) as per standard manufacturer’s protocol and split into two 1.5 ml LoBind tubes (Eppendorf) with a total volume of 100 μl each. In total, 2 mg of bis(sulfosuccinimidyl)suberate (BS3) cross-linker (Thermo Scientific) was dissolved in the assay buffer to a final concentration of 100 mM. 20,000× molar excess of BS3 was added, and the samples were incubated for 2 h at room temperature. Controls were prepared with the assay buffer in place of the cross-linker. The reaction was quenched with 100 mM Tris pH 8.0 and incubated for 10 min at room temperature. 0.8 M Urea was added to promote protein unfolding. Protein samples were reduced with 10 mM TCEP (Thermo Scientific) for 45 min at 55 °C and alkylated with 40 mM chloroacetamide (Sigma Aldrich) for 20 min at room temperature in the dark. Samples were digested with trypsin (1:10 (w/w) ratio) (Promega) at 37 °C, 600× rpm overnight. Digestion was stopped with 1% formic acid (Sigma Aldrich) and purified by zip tipping using OMIX C18 100 μl pipette tips (Agilent). The tip was rehydrated with 50% acetonitrile/0.1% formic acid (Buffer B) and equilibrated with 0.1% formic acid (Buffer A). The samples were bound to the resin bed of the tip by repeated pipetting up and down 10 to 15× in the sample vial. The tip was washed three times with Buffer A. Purified peptides were eluted with Buffer B into a new 1.5 ml LoBind tube (Eppendorf). Samples were vacuum concentrated to ∼5 μl in a SpeedVac (Thermo Scientific) and reconstituted in 20 μl of Buffer A. Both cross-linked and control samples were analyzed on a QExactive HF (Thermo Scientific) mass spectrometer coupled to a Dionex Ultimate 3000 RSLCnano system (Thermo Scientific). In detail, peptides were immobilized on an Acclaim PepMap 100 (100 μm × 2 cm, nanoViper, C18, 5 μm, 100 Å; Thermo Scientific) trap column and separated on an Acclaim PepMap RSLC (75 μm × 50 cm, nanoViper, C18, 2 μm, 100 Å; Thermo Scientific) analytical column by increasing concentrations of 80% acetonitrile/0.1% formic acid at a flow of 250 nl/min for 90 min. All samples were analyzed with pLink2 (v2.3.9) (54) using a custom database containing the protein sequences of SAGA or SLIK components in addition to common contaminant sequences (312 database entries). Trypsin/P was selected as protease, up to three missed cleavages were permitted, and precursor and fragment mass tolerances were set to 10 and 20 ppm, respectively. Carbamidomethylation of cysteine residues was selected as fixed modification, while oxidation of methionine residues and acetylation of protein N-termini were selected as variable modifications. Only peptides falling below an e-value cutoff of 0.01 were considered for further analysis, and cross-linked peptides were plotted using xVis.

Enzyme kinetics—DUBSteady-state kinetic measurements were performed at room temperature in a CLARIOstar Plus plate reader (BMG Labtech), using 350 nm and 440 nm as an excitation and emission wavelengths, respectively. Deubiquitination (DUB) assays were carried out in a 384-well low-volume polystyrene microplates (CORNING), using Ub-AMC (UbiQ) dissolved in 100% DMSO as a substrate. Reactions were assembled in a total volume of 30 μl in assay buffer containing 100 mM (NH4)2SO4, 50 mM HEPES, pH 7.6, 10 μM ZnSO4, 1 mM DTT. In order to find a point where initial rate was independent of enzyme concentration, enzyme was titrated into a 2 μM fixed concentration of Ub-AMC. Measurement was performed for 2 h and initial rates were plotted against the enzyme concentration. Based on obtained results, the enzyme concentration of 75 nM (2.9 μl of SAGA or SLIK) was determined as optimal for steady-state kinetic measurements (data not shown). For DUB assays the enzyme concentration was kept constant, whereas the concentration of Ub-AMC was varied between 0.5 μM and 20 μM. Final concentration of DMSO was held at 4.44%. Reaction mix for each substrate concentration was prepared and pipetted into the wells. Reactions were initiated by the addition of enzyme and the release of AMC was monitored at 440 nm until the total fluorescence was invariant in time. Appropriate blank reactions were prepared and used for background correction. The calibration curve was generated by plotting fluorescence versus substrate concentration at a point where the substrate was fully converted i.e., total fluorescence was invariant in time. Each experiment was performed in duplicate. Kinetic parameters (Km, Vmax) were generated by fitting initial rates to the Michaelis–Menten equation using GraphPad Prism 8.

Enzyme kinetics—DUB

Steady-state kinetic measurements were performed at room temperature in a CLARIOstar Plus plate reader (BMG Labtech), using 350 nm and 440 nm as an excitation and emission wavelengths, respectively. Deubiquitination (DUB) assays were carried out in a 384-well low-volume polystyrene microplates (CORNING), using Ub-AMC (UbiQ) dissolved in 100% DMSO as a substrate. Reactions were assembled in a total volume of 30 μl in assay buffer containing 100 mM (NH4)2SO4, 50 mM HEPES, pH 7.6, 10 μM ZnSO4, 1 mM DTT. In order to find a point where initial rate was independent of enzyme concentration, enzyme was titrated into a 2 μM fixed concentration of Ub-AMC. Measurement was performed for 2 h and initial rates were plotted against the enzyme concentration. Based on obtained results, the enzyme concentration of 75 nM (2.9 μl of SAGA or SLIK) was determined as optimal for steady-state kinetic measurements (data not shown). For DUB assays the enzyme concentration was kept constant, whereas the concentration of Ub-AMC was varied between 0.5 μM and 20 μM. Final concentration of DMSO was held at 4.44%. Reaction mix for each substrate concentration was prepared and pipetted into the wells. Reactions were initiated by the addition of enzyme and the release of AMC was monitored at 440 nm until the total fluorescence was invariant in time. Appropriate blank reactions were prepared and used for background correction. The calibration curve was generated by plotting fluorescence versus substrate concentration at a point where the substrate was fully converted i.e., total fluorescence was invariant in time. Each experiment was performed in duplicate. Kinetic parameters (Km, Vmax) were generated by fitting initial rates to the Michaelis–Menten equation using GraphPad Prism 8.

Enzyme kinetics—HATSteady-state kinetic measurements using H3 peptide substrate were done with a continuous spectrophotometric assay as previously described (34). Each 50 μl reaction contained 50 mM (NH4)2SO4, 50 mM HEPES, pH 7.6, 5 mM MgCl2, 1 mM DTT, 0.2 mM thiamine pyrophosphate, 0.2 mM NAD+, 2.5 mM pyruvate, 1 μl of 0.57 U/mg at 15 mg/ml pyruvate dehydrogenase (Sigma), 100 μM acetyl-CoA, 50 nM SAGA or SLIK, and 0 to 1500 μM histone H3 peptide 1 to 21 (sequence ARTKQTARKSTGGKAPRKQLA, purchased at ≥95% purity from Cayman Chemical).All the reaction components, except for acetyl-CoA, were assembled in clear 96-well plates and incubated at room temperature for 5 min. Reactions were initiated by the addition of acetyl-CoA, and the absorbance at 340 nm was monitored continuously using a FLUOstar Omega plate reader (BMG Labtech) for 5 to 60 min. Absorbance at 340 nm was converted into the molar concentration of NADH using Beer’s Law, assuming ε340 nm = 6220 M−1 cm−1. To calculate initial rates, NADH production was plotted as a function of time and fit to a line where initial velocity conditions were satisfied, typically within the first 20 min. A blank reaction containing the HAT module and acetyl-CoA, but no substrate, was performed for each titration and the rate was subtracted as background from the other reactions. Initial rates were measured in duplicate, normalized to the enzyme concentration, and plotted as a function of substrate concentration. The resulting curve was fit to the Michaelis–Menten equation using nonlinear least squares regression implemented in GraphPad Prism 8.

Enzyme kinetics—HAT

Steady-state kinetic measurements using H3 peptide substrate were done with a continuous spectrophotometric assay as previously described (34). Each 50 μl reaction contained 50 mM (NH4)2SO4, 50 mM HEPES, pH 7.6, 5 mM MgCl2, 1 mM DTT, 0.2 mM thiamine pyrophosphate, 0.2 mM NAD+, 2.5 mM pyruvate, 1 μl of 0.57 U/mg at 15 mg/ml pyruvate dehydrogenase (Sigma), 100 μM acetyl-CoA, 50 nM SAGA or SLIK, and 0 to 1500 μM histone H3 peptide 1 to 21 (sequence ARTKQTARKSTGGKAPRKQLA, purchased at ≥95% purity from Cayman Chemical).

All the reaction components, except for acetyl-CoA, were assembled in clear 96-well plates and incubated at room temperature for 5 min. Reactions were initiated by the addition of acetyl-CoA, and the absorbance at 340 nm was monitored continuously using a FLUOstar Omega plate reader (BMG Labtech) for 5 to 60 min. Absorbance at 340 nm was converted into the molar concentration of NADH using Beer’s Law, assuming ε340 nm = 6220 M−1 cm−1. To calculate initial rates, NADH production was plotted as a function of time and fit to a line where initial velocity conditions were satisfied, typically within the first 20 min. A blank reaction containing the HAT module and acetyl-CoA, but no substrate, was performed for each titration and the rate was subtracted as background from the other reactions. Initial rates were measured in duplicate, normalized to the enzyme concentration, and plotted as a function of substrate concentration. The resulting curve was fit to the Michaelis–Menten equation using nonlinear least squares regression implemented in GraphPad Prism 8.

Protein–ligand binding assay by fluorescence polarizationFP titrations were carried out in a 384-well low-volume microplates (CORNING) at room temperature in a PHERAstar FSX plate reader (BMG Labtech), equipped with polarizing fluorescent filters. Protein dilutions (100 nM–3.125 nM, final concentrations) were prepared in the protein storage buffer (200 mM (NH4)2SO4, 50 mM HEPES, pH 6.8, 10 μM ZnSO4, 1 mM DTT). SAGA and SLIK were processed in parallel and adjusted to the same concentration so that an identical volume of protein dilution was added to each reaction (3.9 μl). Reactions were performed in assay buffer (50 mM CH3CO2K, 50 mM HEPES, pH 6.8, 2 mM MgSO4, 10 μM ZnSO4, 1 mM DTT), supplemented with 0.75 pmol per well of oligonucleotides 5’-labeled with Cy3 (IDT) (final concentration 25 nM) (see sequences in Supporting information). Reactions were assembled in a total volume of 30 μl and incubated for 10 min at room temperature. Measurements were carried out using fluorophore-specific excitation/emission wavelengths (540/590 for Cy3). Each titration curve was made in triplicate (except for 20-mer with TATA DNA, that was made in duplicate). Blank reactions without protein were prepared and used for background correction. Binding curves were generated by plotting FP versus protein concentration. Binding parameters were estimated by fitting in Graphpad Prism 8. For the competition experiments, 25 nM of 75-mer dsDNA was used as above, in the same assay buffer. SAGA or SLIK was added to 50 nM concentration (corresponding to the plateau previously characterized in these conditions). Reactions were supplemented with either no competitor, Poly(dI-dC) competitor (1, 10, 100 times weight excess, relative to dsDNA; i.e., 37, 370, or 3700 ng per reaction), 200-mer dsDNA competitor corresponding to the TATA-less promoter of the ribosomal protein 5 gene (RPS5) (1, 5, 10, 100 times weight excess, relative to dsDNA; i.e., 37, 185, 370 or 3700 ng per reaction), or unlabeled 75-mer dsDNA with the same sequence as the DNA probe (1, 5, 10, 50 times weight/molar excess, relative to dsDNA). Conditions were prepared in triplicates (Poly(dI-dC)), duplicates (RPS5), or single repeat (unlabeled 75-mer dsDNA). Blank reactions without protein but with DNA and appropriate amount of Poly(dI-dC) were used for background correction; all blanks had similar FP values. Reactions were assembled in a 384-well low-volume polystyrene microplates (CORNING) at room temperature and incubated for 10 min. Measurements were carried out using 540/590 excitation/emission wavelengths in a PHERAstar FSX plate reader (BMG Labtech), using a polarizing fluorescent filter. FP signal was normalized to 100% for reactions without competitor.Binding to 75-mer dsDNA was virtually identical for SAGA purified from Spt7-TAP strain and SAGA purified from Ubp8-TAP strain (not shown).

Protein–ligand binding assay by fluorescence polarization

FP titrations were carried out in a 384-well low-volume microplates (CORNING) at room temperature in a PHERAstar FSX plate reader (BMG Labtech), equipped with polarizing fluorescent filters. Protein dilutions (100 nM–3.125 nM, final concentrations) were prepared in the protein storage buffer (200 mM (NH4)2SO4, 50 mM HEPES, pH 6.8, 10 μM ZnSO4, 1 mM DTT). SAGA and SLIK were processed in parallel and adjusted to the same concentration so that an identical volume of protein dilution was added to each reaction (3.9 μl). Reactions were performed in assay buffer (50 mM CH3CO2K, 50 mM HEPES, pH 6.8, 2 mM MgSO4, 10 μM ZnSO4, 1 mM DTT), supplemented with 0.75 pmol per well of oligonucleotides 5’-labeled with Cy3 (IDT) (final concentration 25 nM) (see sequences in Supporting information). Reactions were assembled in a total volume of 30 μl and incubated for 10 min at room temperature. Measurements were carried out using fluorophore-specific excitation/emission wavelengths (540/590 for Cy3). Each titration curve was made in triplicate (except for 20-mer with TATA DNA, that was made in duplicate). Blank reactions without protein were prepared and used for background correction. Binding curves were generated by plotting FP versus protein concentration. Binding parameters were estimated by fitting in Graphpad Prism 8. For the competition experiments, 25 nM of 75-mer dsDNA was used as above, in the same assay buffer. SAGA or SLIK was added to 50 nM concentration (corresponding to the plateau previously characterized in these conditions). Reactions were supplemented with either no competitor, Poly(dI-dC) competitor (1, 10, 100 times weight excess, relative to dsDNA; i.e., 37, 370, or 3700 ng per reaction), 200-mer dsDNA competitor corresponding to the TATA-less promoter of the ribosomal protein 5 gene (RPS5) (1, 5, 10, 100 times weight excess, relative to dsDNA; i.e., 37, 185, 370 or 3700 ng per reaction), or unlabeled 75-mer dsDNA with the same sequence as the DNA probe (1, 5, 10, 50 times weight/molar excess, relative to dsDNA). Conditions were prepared in triplicates (Poly(dI-dC)), duplicates (RPS5), or single repeat (unlabeled 75-mer dsDNA). Blank reactions without protein but with DNA and appropriate amount of Poly(dI-dC) were used for background correction; all blanks had similar FP values. Reactions were assembled in a 384-well low-volume polystyrene microplates (CORNING) at room temperature and incubated for 10 min. Measurements were carried out using 540/590 excitation/emission wavelengths in a PHERAstar FSX plate reader (BMG Labtech), using a polarizing fluorescent filter. FP signal was normalized to 100% for reactions without competitor.

Binding to 75-mer dsDNA was virtually identical for SAGA purified from Spt7-TAP strain and SAGA purified from Ubp8-TAP strain (not shown).

Gel shift TBP- and DNA-binding assayFor TBP-binding assays, purified TBP was fluorescently labeled with OneQuant Fluorescein Isothiocyanate (FITC) (G-Biosciences). One milligram of FITC was warmed up to room temperature and resuspended in 10 μl DMSO. In total, 250 μl NaHCO3/Na2CO3 buffer at 0.2 M concentration was added, followed by 250 μl TBP at 3.4 mg/ml (final pH 10). Sample was incubated overnight at 4 °C, with gentle rocking, protected from light. Excess dye was removed using a desalting column (PD MiniTrap, Cytiva), followed by a gel filtration column (S75 10/300) in 500 mM NaCl, 20 mM HEPES pH 7.5, 10% glycerol, 3 mM DTT. Protein concentration was estimated by densitometry, by comparing dilutions of FITC-TBP with titrations of unlabeled TBP at known concentrations. Binding assays (10 μl) contained 3 mM Tris-HCl (pH 8.0), 6 mM HEPES-KOH (pH 8.0), 45 mM CH3COOK, 42 mM KCl, 5 mM MgCl2, 3% glycerol, 6% sucrose, 0.01% NP40, 25 μg/ml BSA, 1 mM DTT, plus proteins. FITC-TBP concentration was maintained at 10 nM while SAGA and SLIK were titrated from 0 to 150 nM. Serial dilutions of SAGA and SLIK were done in storage buffer and the same amount of buffer was added to sample without SAGA/SLIK to keep the salt concentration constant. Proteins were incubated together for 1 h at 4 °C, and the samples were loaded onto horizontal agarose-polyacrylamide native gels (0.3% agarose, 2% acrylamide with 0.5X TBE in the gel and running buffer). The gel was run at 90 V for 3 h and scanned using a Typhoon 5 to follow FITC fluorescence of TBP. For DNA-binding assays, DNA probes used were the same as for FP experiments (75-mer and 20-mer dsDNA, both containing a TATA-box, Cy3-labeled). The nonlabeled DNA was 200-mer dsDNA corresponding to the TATA-less promoter of the ribosomal protein 5 gene (RPS5). Binding assays (10 μl) contained 3 mM Tris-HCl (pH 8.0), 6 mM HEPES-KOH (pH 8.0), 45 mM CH3COOK, 42 mM KCl, 5 mM MgCl2, 3% glycerol, 6% sucrose, 0.01% NP40, 25 μg/ml BSA, 1 mM DTT, plus proteins and DNA. The protein concentrations were 75 nM TBP, 300 nM SAGA or SLIK, 75 nM Cy3-DNA, 900 nM nonlabeled TATA-less DNA. TBP was incubated first with a fourfold excess of SAGA or SLIK complex for 1 h at 4 °C. Then DNA was added, and the reaction mixtures were incubated for 20 min at room temperature before they were loaded onto horizontal agarose-polyacrylamide native gels (1% agarose, 1.3% acrylamide with 1X Tris-glycine buffer, 5 mM MgCl2 in the gel and running buffer). The gel was run at 30 V overnight and scanned using a Typhoon 5 to follow Cy3 fluorescence of DNA. Composition of the shifted bands was verified by mass spectrometry (Tables S4 and S5).

Gel shift TBP- and DNA-binding assay

For TBP-binding assays, purified TBP was fluorescently labeled with OneQuant Fluorescein Isothiocyanate (FITC) (G-Biosciences). One milligram of FITC was warmed up to room temperature and resuspended in 10 μl DMSO. In total, 250 μl NaHCO3/Na2CO3 buffer at 0.2 M concentration was added, followed by 250 μl TBP at 3.4 mg/ml (final pH 10). Sample was incubated overnight at 4 °C, with gentle rocking, protected from light. Excess dye was removed using a desalting column (PD MiniTrap, Cytiva), followed by a gel filtration column (S75 10/300) in 500 mM NaCl, 20 mM HEPES pH 7.5, 10% glycerol, 3 mM DTT. Protein concentration was estimated by densitometry, by comparing dilutions of FITC-TBP with titrations of unlabeled TBP at known concentrations. Binding assays (10 μl) contained 3 mM Tris-HCl (pH 8.0), 6 mM HEPES-KOH (pH 8.0), 45 mM CH3COOK, 42 mM KCl, 5 mM MgCl2, 3% glycerol, 6% sucrose, 0.01% NP40, 25 μg/ml BSA, 1 mM DTT, plus proteins. FITC-TBP concentration was maintained at 10 nM while SAGA and SLIK were titrated from 0 to 150 nM. Serial dilutions of SAGA and SLIK were done in storage buffer and the same amount of buffer was added to sample without SAGA/SLIK to keep the salt concentration constant. Proteins were incubated together for 1 h at 4 °C, and the samples were loaded onto horizontal agarose-polyacrylamide native gels (0.3% agarose, 2% acrylamide with 0.5X TBE in the gel and running buffer). The gel was run at 90 V for 3 h and scanned using a Typhoon 5 to follow FITC fluorescence of TBP. For DNA-binding assays, DNA probes used were the same as for FP experiments (75-mer and 20-mer dsDNA, both containing a TATA-box, Cy3-labeled). The nonlabeled DNA was 200-mer dsDNA corresponding to the TATA-less promoter of the ribosomal protein 5 gene (RPS5). Binding assays (10 μl) contained 3 mM Tris-HCl (pH 8.0), 6 mM HEPES-KOH (pH 8.0), 45 mM CH3COOK, 42 mM KCl, 5 mM MgCl2, 3% glycerol, 6% sucrose, 0.01% NP40, 25 μg/ml BSA, 1 mM DTT, plus proteins and DNA. The protein concentrations were 75 nM TBP, 300 nM SAGA or SLIK, 75 nM Cy3-DNA, 900 nM nonlabeled TATA-less DNA. TBP was incubated first with a fourfold excess of SAGA or SLIK complex for 1 h at 4 °C. Then DNA was added, and the reaction mixtures were incubated for 20 min at room temperature before they were loaded onto horizontal agarose-polyacrylamide native gels (1% agarose, 1.3% acrylamide with 1X Tris-glycine buffer, 5 mM MgCl2 in the gel and running buffer). The gel was run at 30 V overnight and scanned using a Typhoon 5 to follow Cy3 fluorescence of DNA. Composition of the shifted bands was verified by mass spectrometry (Tables S4 and S5).

Sample stabilization with GraFix for cryo-EMFor electron microscopy studies, proteins were stabilized by the optimized GraFix procedure (36); 10 to 40% continuous glycerol gradients were prepared by mixing light solution (500 mM CH3COOK, 25 mM HEPES, pH 6.8, 2 mM MgSO4, 10 μM ZnSO4, 10% (w/v) glycerol) with heavy solution (500 mM CH3COOK, 25 mM HEPES, pH 6.8, 2 mM MgSO4, 10 μM ZnSO4, 40% (w/v) glycerol, 0.3% (v/v) glutaraldehyde) using the Gradient Master (BioComp Instruments). In total, 100 pmol of the SLIK complex was applied on top of the gradient and was subject to ultracentrifugation in an SW60 Ti rotor (Beckman Coulter) at 30,000× rpm, 4 °C for 15 h. Following ultracentrifugation, the gradients were fractionated using the Piston Gradient Fractionator (BioComp Instruments). Obtained UV280 profile indicated that the complex was eluted as a single peak positioned approximately 2/3 down the gradient. Selected peak fractions were pooled and quenched with 50 mM final concentration of ammonium sulfate to prevent further cross-linking.

Sample stabilization with GraFix for cryo-EM

For electron microscopy studies, proteins were stabilized by the optimized GraFix procedure (36); 10 to 40% continuous glycerol gradients were prepared by mixing light solution (500 mM CH3COOK, 25 mM HEPES, pH 6.8, 2 mM MgSO4, 10 μM ZnSO4, 10% (w/v) glycerol) with heavy solution (500 mM CH3COOK, 25 mM HEPES, pH 6.8, 2 mM MgSO4, 10 μM ZnSO4, 40% (w/v) glycerol, 0.3% (v/v) glutaraldehyde) using the Gradient Master (BioComp Instruments). In total, 100 pmol of the SLIK complex was applied on top of the gradient and was subject to ultracentrifugation in an SW60 Ti rotor (Beckman Coulter) at 30,000× rpm, 4 °C for 15 h. Following ultracentrifugation, the gradients were fractionated using the Piston Gradient Fractionator (BioComp Instruments). Obtained UV280 profile indicated that the complex was eluted as a single peak positioned approximately 2/3 down the gradient. Selected peak fractions were pooled and quenched with 50 mM final concentration of ammonium sulfate to prevent further cross-linking.

Cryo-EM sample preparationGrids used were ultrathin Carbon film supported by a lacey Carbon film on a 400 mesh, copper (Pelco). In order to remove glycerol required for GraFix, sample was buffer-exchanged on the grid: 2.5 μl sample was applied, manually blotted away; 2.5 μl buffer (500 mM CH3COOK, 25 mM HEPES, pH 6.8, 2 mM MgSO4, 10 μM ZnSO4) was applied and manually blotted away; 2.5 μl buffer was applied, and the grid was vitrified in liquid ethane using a Vitrobot IV (FEI). Cryo-grids were screened under low-dose conditions on the Tecnai T12 Spirit microscope (FEI) at 120 kV, to identify the optimal freezing conditions (ice thickness, contrast, and particles distribution).

Cryo-EM sample preparation

Grids used were ultrathin Carbon film supported by a lacey Carbon film on a 400 mesh, copper (Pelco). In order to remove glycerol required for GraFix, sample was buffer-exchanged on the grid: 2.5 μl sample was applied, manually blotted away; 2.5 μl buffer (500 mM CH3COOK, 25 mM HEPES, pH 6.8, 2 mM MgSO4, 10 μM ZnSO4) was applied and manually blotted away; 2.5 μl buffer was applied, and the grid was vitrified in liquid ethane using a Vitrobot IV (FEI). Cryo-grids were screened under low-dose conditions on the Tecnai T12 Spirit microscope (FEI) at 120 kV, to identify the optimal freezing conditions (ice thickness, contrast, and particles distribution).

Cryo-EM data collection and image processingImages were collected on a Titan Krios FEG-TEM electron microscope (FEI) operated at 300 kV and recorded at a magnification of 105,000 using a K2 Summit direct electron detector camera (Gatan) in counting mode at a calibrated pixel size of 1.36 Å. In total, 1894 and 3175 dose-fractionated movies were recorded during two independent collections with the specimen subjected to total accumulated doses of 50.2 and 50.3 e−/Å2 respectively. Individual movies were corrected for stage and beam-induced motion and CTF parameters calculated using SIMPLE3.0 (Figs. S3 and S4). Particles’ positions were automatically identified with template-based picking with the structure of SAGA resolution limited to 20 Å (EMD-10412) (25) and SIMPLE3.0. The initial set of 292,323 particles was refined to final set of 44,021 though iterative rounds of 2D classification (55). The class averages thus generated were used to determine an ab initio 3D model for further single particles refinement using RELION3.0 (Fig. S3, resolution limit of 30 Å) (56, 57). A consensus 3D model was first obtained at 23.6 Å resolution (gold-standard FSC criterion of 0.143) and subjected to 3D classification to yield three classes populated by 16,623, 14,188, and 13,210 particles individually refined to resolutions 22.5, 29.1, and 29.1 Å respectively (Fig. 6, Figs. S3 and S4).

Cryo-EM data collection and image processing

Images were collected on a Titan Krios FEG-TEM electron microscope (FEI) operated at 300 kV and recorded at a magnification of 105,000 using a K2 Summit direct electron detector camera (Gatan) in counting mode at a calibrated pixel size of 1.36 Å. In total, 1894 and 3175 dose-fractionated movies were recorded during two independent collections with the specimen subjected to total accumulated doses of 50.2 and 50.3 e−/Å2 respectively. Individual movies were corrected for stage and beam-induced motion and CTF parameters calculated using SIMPLE3.0 (Figs. S3 and S4). Particles’ positions were automatically identified with template-based picking with the structure of SAGA resolution limited to 20 Å (EMD-10412) (25) and SIMPLE3.0. The initial set of 292,323 particles was refined to final set of 44,021 though iterative rounds of 2D classification (55). The class averages thus generated were used to determine an ab initio 3D model for further single particles refinement using RELION3.0 (Fig. S3, resolution limit of 30 Å) (56, 57). A consensus 3D model was first obtained at 23.6 Å resolution (gold-standard FSC criterion of 0.143) and subjected to 3D classification to yield three classes populated by 16,623, 14,188, and 13,210 particles individually refined to resolutions 22.5, 29.1, and 29.1 Å respectively (Fig. 6, Figs. S3 and S4).

Model buildingOur XL-MS data are consistent with published models of SAGA and DUB module (18, 25, 26). We therefore used the structure of the structural core of S. cerevisiae SAGA (PDB: 6T9I) (25) and docked it manually into the map. DUB module of K. phaffii SAGA-TBP (PDB: 6TBM) (18) was docked based on XL-MS data and its location and orientation in K. phaffii SAGA. HAT module was modeled as a sphere due to the absence of structural information for the full module and docked in the density based on its position in published structures (18, 25, 33).

Model building

Our XL-MS data are consistent with published models of SAGA and DUB module (18, 25, 26). We therefore used the structure of the structural core of S. cerevisiae SAGA (PDB: 6T9I) (25) and docked it manually into the map. DUB module of K. phaffii SAGA-TBP (PDB: 6TBM) (18) was docked based on XL-MS data and its location and orientation in K. phaffii SAGA. HAT module was modeled as a sphere due to the absence of structural information for the full module and docked in the density based on its position in published structures (18, 25, 33).

Data availabilityThe proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD024640 (58). EM data have been deposited to the Electron Microscopy Data Bank (EMDB) public repository with the entry EMD-23038.

Data availability

The proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD024640 (58). EM data have been deposited to the Electron Microscopy Data Bank (EMDB) public repository with the entry EMD-23038.

Supporting informationThis article contains supporting information.

Supporting information

This article contains supporting information.

Conflict of interestThe authors declare that they have no conflicts of interest with the contents of this article.

Conflict of interest

The authors declare that they have no conflicts of interest with the contents of this article.
