---
pmid: '29263158'
title: Clb6-Cdc28 Promotes Ribonucleotide Reductase Subcellular Redistribution during
  S Phase.
authors:
- Wu X
- An X
- Zhang C
- Huang M
journal: Mol Cell Biol
year: '2018'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC5829487
doi: 10.1128/MCB.00497-17
---

# Clb6-Cdc28 Promotes Ribonucleotide Reductase Subcellular Redistribution during S Phase.
**Authors:** Wu X, An X, Zhang C, Huang M
**Journal:** Mol Cell Biol (2018)
**DOI:** [10.1128/MCB.00497-17](https://doi.org/10.1128/MCB.00497-17)
**PMC:** [PMC5829487](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5829487/)

## Abstract

1. Mol Cell Biol. 2018 Feb 27;38(6):e00497-17. doi: 10.1128/MCB.00497-17. Print 
2018 Mar 15.

Clb6-Cdc28 Promotes Ribonucleotide Reductase Subcellular Redistribution during S 
Phase.

Wu X(1), An X(1), Zhang C(1)(2), Huang M(3)(2).

Author information:
(1)Department of Biochemistry and Molecular Genetics, University of Colorado 
School of Medicine, Aurora, Colorado, USA.
(2)Department of Dermatology, University of Colorado School of Medicine, Aurora, 
Colorado, USA.
(3)Department of Biochemistry and Molecular Genetics, University of Colorado 
School of Medicine, Aurora, Colorado, USA mingxia.huang@ucdenver.edu.

A tightly controlled cellular deoxyribonucleotide (deoxynucleoside triphosphate 
[dNTP]) pool is critical for maintenance of genome integrity. One mode of dNTP 
pool regulation is through subcellular localization of ribonucleotide reductase 
(RNR), the enzyme that catalyzes the rate-limiting step of dNTP biosynthesis. In 
Saccharomyces cerevisiae, the RNR small subunit, Rnr2-Rnr4, is localized to the 
nucleus, whereas the large subunit, Rnr1, is cytoplasmic. As cells enter S phase 
or encounter DNA damage, Rnr2-Rnr4 relocalizes to the cytoplasm to form an 
active holoenzyme complex with Rnr1. Although the DNA damage-induced 
relocalization requires the checkpoint kinases Mec1-Rad53-Dun1, the 
S-phase-specific redistribution does not. Here, we report that the S-phase 
cyclin-cyclin-dependent kinase (CDK) complex Clb6-Cdc28 controls Rnr2-Rnr4 
relocalization in S phase. Rnr2 contains a consensus CDK site and exhibits 
Clb6-dependent phosphorylation in S phase. Deletion of CLB6 or removal of the 
CDK site results in an increased association of Rnr2 with its nuclear anchor 
Wtm1, nuclear retention of Rnr2-Rnr4, and an enhanced sensitivity to the RNR 
inhibitor hydroxyurea. Thus, we propose that Rnr2-Rnr4 redistribution in S phase 
is triggered by Clb6-Cdc28-mediated phosphorylation of Rnr2, which disrupts the 
Rnr2-Wtm1 interaction and promotes the release of Rnr2-Rnr4 from the nucleus.

Copyright © 2018 American Society for Microbiology.

DOI: 10.1128/MCB.00497-17
PMCID: PMC5829487
PMID: 29263158 [Indexed for MEDLINE]

## Full Text

ABSTRACT

A tightly controlled cellular deoxyribonucleotide (deoxynucleoside triphosphate [dNTP]) pool is critical for maintenance of genome integrity. One mode of dNTP pool regulation is through subcellular localization of ribonucleotide reductase (RNR), the enzyme that catalyzes the rate-limiting step of dNTP biosynthesis. In Saccharomyces cerevisiae , the RNR small subunit, Rnr2-Rnr4, is localized to the nucleus, whereas the large subunit, Rnr1, is cytoplasmic. As cells enter S phase or encounter DNA damage, Rnr2-Rnr4 relocalizes to the cytoplasm to form an active holoenzyme complex with Rnr1. Although the DNA damage-induced relocalization requires the checkpoint kinases Mec1-Rad53-Dun1, the S-phase-specific redistribution does not. Here, we report that the S-phase cyclin–cyclin-dependent kinase (CDK) complex Clb6-Cdc28 controls Rnr2-Rnr4 relocalization in S phase. Rnr2 contains a consensus CDK site and exhibits Clb6-dependent phosphorylation in S phase. Deletion of CLB6 or removal of the CDK site results in an increased association of Rnr2 with its nuclear anchor Wtm1, nuclear retention of Rnr2-Rnr4, and an enhanced sensitivity to the RNR inhibitor hydroxyurea. Thus, we propose that Rnr2-Rnr4 redistribution in S phase is triggered by Clb6-Cdc28-mediated phosphorylation of Rnr2, which disrupts the Rnr2-Wtm1 interaction and promotes the release of Rnr2-Rnr4 from the nucleus.

INTRODUCTION

Faithful replication of the genome depends on an adequate and balanced deoxyribonucleotide (deoxynucleoside triphosphate [dNTP]) pool ( 1 ). Ribonucleotide reductase (RNR) catalyzes the rate-limiting step in de novo dNTP synthesis and, thus, is largely responsible for controlling intracellular dNTP pools ( 2 ). All eukaryotic RNRs comprise a large subunit R1 that contains the catalytic and allosteric sites and a small subunit R2 that houses a diferric-tyrosyl radical cofactor that is essential to initiate nucleotide reduction ( 3 , 4 ). In the budding yeast, Saccharomyces cerevisiae , the R1 subunit is a homodimer encoded by the RNR1 gene, while the R2 subunit is a heterodimer encoded by two paralogous genes, RNR2 and RNR4 ( 5 – 7 ).

Adjusting the dNTP pools to meet the demand of DNA synthesis under different conditions is critical, as levels that are too high or too low can cause increased spontaneous mutagenesis and genomic instability, a hallmark of cancer and aging ( 8 – 10 ). Cells have evolved multiple strategies to tightly control the level and activity of RNR, including allostery, transcription, and inhibitor protein association, as well as subcellular localization ( 2 , 11 ). Allosteric regulation occurs through the large subunit R1, which alters substrate specificity based on sensing individual dNTP concentrations and overall activity through a dATP-mediated negative-feedback mechanism ( 2 ).

In response to DNA damage and replication stress, such as low dNTP levels caused by the RNR inhibitor hydroxyurea (HU), transcription of the RNR genes is rapidly induced by activation of the DNA damage checkpoint kinase cascade Med1-Rad53-Dun1, which inactivates the Crt1-Ssn6-tup1 transcriptional repressor complex ( 12 , 13 ). The Mec1-Rad53-Dun1 kinase cascade is also responsible for phosphorylation-mediated proteolysis of the Rnr1 inhibitor protein Sml1, thus allowing further RNR activation ( 14 ).

RNR subcellular localization is an additional level of regulation ( 15 – 18 ). In the budding yeast, Rnr1 resides in the cytoplasm, whereas Rnr2-Rnr4 is confined to the nucleus except when cells enter S phase or encounter genotoxic stress. Nuclear sequestration of Rnr2-Rnr4 is controlled by two negative regulators of RNR: Dif1, which facilitates its nuclear import, and Wtm1, which binds to and retains it within the nucleus ( 19 – 22 ). DNA damage and replication stress elicit the Mec1-Rad53-Dun1 kinase cascade-mediated phosphorylation and degradation of Dif1, which leads to cytoplasmic enrichment of Rnr2-Rnr4, allowing it to form an active holoenzyme with Rnr1 ( 21 , 22 ). The physiological importance of tight RNR regulation is manifested by the fact that the lethality of the mec1 Δ and rad53 Δ checkpoint kinase mutants can be rescued by activation of the RNR pathway, either via overexpression of RNR or via the removal of one of its negative regulators, SML1 , WTM1 , or DIF1 ( 13 , 19 , 21 – 24 ).

Rnr2-Rnr4 also relocalizes from the nucleus to the cytoplasm as cells enter S phase, coinciding with an increased demand of dNTP production for DNA replication ( 17 ). However, it is unclear whether this S-phase-specific redistribution is controlled by the Mec1-Rad53-Dun1 checkpoint kinase cascade or through a yet-unidentified mechanism.

The G 1 -to-S-phase transition in yeast cells is coordinated by two conserved kinase complexes, cyclin-dependent kinase (CDK) and Dbf4-dependent kinase (DDK) ( 25 ), each comprising a constitutive catalytic subunit, Cdc28 and Cdc7, respectively, and a cell cycle-controlled regulatory subunit, cyclins and Dbf4, respectively ( 25 , 26 ). CDK and DDK act cooperatively to phosphorylate specific subunits of the prereplication complex, which triggers conformational changes that allow polymerase recruitment and origin firing in S phase. The S-phase CDK activity is governed by sequential association of Cdc28 with six B-type cyclins, Clb1 to -6, which peak in three successive waves, in the order of the pair of S-phase cyclins Clb5/Clb6 and the two pairs of mitotic cyclins Clb3/Clb4 and Clb1/Clb2, during the S-phase pregression ( 27 ). The Clb5/Clb6-dependent CDK activity peaks at the G 1 -to-S transition and is primarily responsible for cooperation with DDK to activate the firing of replication origins.

In this study, we uncover the mechanism that regulates Rnr2-Rnr4 relocalization during S phase. We show that the S-phase-specific redistribution is independent of the Mec1-Rad53-Dun1 kinase cascade and, instead, is controlled by Clb6-Cdc28. Furthermore, we demonstrate that a consensus CDK phosphorylation site, TPSK, at the N terminus of Rnr2 controls relocalization of Rnr2-Rnr4 through modulating the Wtm1-Rnr2 association. A phosphorylation-deficient substitution in Rnr2 enhances both in vivo Rnr2-Wtm1 association and cellular sensitivity to HU. We present evidence indicating that Clb6-Cdc28-mediated phosphorylation of Rnr2 at the consensus TPSK motif plays a critical role in the nucleus-to-cytoplasm redistribution of Rnr2-Rnr4 in S phase.

Mutation resulting in phosphorylation-defective consensus CDK site in Rnr2 leads to nuclear retention of R2 subunit in S phase.

The consensus sequence for Cdc28 (Cdk1)-type CDK phosphorylation is S/T-P-X-K/R or S/T-P ( 31 , 32 ). Rnr2 contains a single Cdc28 phosphorylation motif, TPSK, at its N terminus, which is conserved in many fungal relatives of S. cerevisiae ( Fig. 5A ). To determine whether the threonine-5 within this TPSK motif is involved in regulating Rnr2 redistribution in S phase, we generated a phosphorylation-deficient mutant allele, rnr2-T5A , and compared the subcellular localization patterns of Rnr2 wild-type and Rnr2-T5A mutant proteins as cells moved from G 1 to S phase. In G 1 -arrested cells, Rnr2-T5A was primarily localized to the nucleus, just like the wild-type protein. However, as cells were released from G 1 to S phase, Rnr2-T5A was much slower than the wild type in leaving the nucleus. At the 30-min time point after release from G 1 , Rnr2-T5A was still retained in the nucleus in >60% of the cells, while only ∼20% of the wild-type cells had a strong nuclear Rnr2 signal ( Fig. 5B ). Anti-Rnr4 antibody immunofluorescence staining revealed similar nuclear retention of Rnr4 in the rnr2-T5A mutant (data not shown). Thus, the consensus CDK phosphorylation site in Rnr2 is required for the nucleus-to-cytoplasm redistribution of Rnr2-Rnr4 in S phase.

DISCUSSION

Optimization of cellular dNTP concentrations is important for high-fidelity DNA replication and repair. A main prerequisite for cells to enter S phase is to enlarge their dNTP pools in order to meet the demand of genome duplication, which is achieved largely through upregulation of RNR. In addition to the increase of RNR gene transcription and proteolysis of the RNR inhibitor Sml1 ( 24 , 33 ) at entry into S phase, the budding yeast further enhances RNR activity by promoting nuclear release of the heterodimeric R2 subunit Rnr2-Rnr4 so it can form the active holoenzyme with the cytoplasmic R1. In this study, we investigated the mechanism underlying the checkpoint-independent, S-phase-specific R2 redistribution. The amount of R2 in the nucleus is a net outcome of its nuclear import, facilitated by Dif1, nuclear retention by Wtm1, and nuclear export. Our results support a model in which the S-phase cyclin-CDK complex Clb6-Cdc28 phosphorylates Rnr2 at its N-terminal TPSK motif, which weakens the Rnr2-Wtm1 interaction and facilitates the release of preexisting Rnr2-Rnr4 from the nucleus ( Fig. 7 ). The nucleus-to-cytoplasm R2 redistribution is further facilitated by S-phase-specific proteolysis of Dif1 ( 21 ), leading to decreased nuclear import of newly translated R2. Consistent with this model, a previous study has shown that deletion of CLB6 makes the mec1Δ sml1 Δ mutant more sensitive to HU ( 34 ), similar to the effect of the phosphorylation-deficient T5A mutant of Rnr2 ( Fig. 6D ). On the other hand, overexpression of CLB6 can partially suppress the HU sensitivity of the chk1Δ dun1 Δ checkpoint double mutant ( 35 ), which may be partially accounted for by an increase in cytoplasmic localization of R2.

Intriguingly, our results demonstrate that the redistribution of R2 in S phase is specifically controlled by Clb6 but not its paralog Clb5, with which it shares 50% primary sequence identity ( 36 ). CLB5 and CLB6 are named S-phase cyclins because they both peak at S-phase entry and play major roles in initiating DNA replication. The Clb5 protein level remains elevated throughout the S phase, whereas Clb6 is rapidly degraded in early unperturbed S phase via an SCF Cdc4 - and anaphase promoter complex (APC)-dependent pathway during normal cell cycle progression ( 37 , 38 ). Consistent with the different expression patterns, Clb5 is required for the firing of both early and late replication origins, while Clb6 is only involved in the firing of the early origins ( 37 ). Although the clb6 Δ mutant shows no apparent S-phase defect, as opposed to the clb5 Δ mutant, the clb5Δ clb6 Δ double mutant exhibits a more severe S-phase delay and an increased HU sensitivity relative to those of each single mutant ( 39 , 40 ), suggesting overlapping albeit distinct roles of the two cyclins ( 39 ). It is likely that the Clb5-Cdc28 and Clb6-Cdc28 kinase complexes differ in their specificities toward a subset of target substrates while sharing most others. A previous study has shown that Clb6-Cdc28 but not Clb5-Cdc28 triggers nuclear export of transcription factor Swi6 in early S phase by specifically phosphorylating Swi6 at serine-160, thus altering Swi4/Swi6 (SBF) and Mbp1/Swi6 (MBF)-mediated transcription at the G 1 /S transition ( 41 ). Our results indicate that the phosphorylation of Rnr2 , like that of Swi6, is specifically dependent on Clb6-Cdc28 and triggers nuclear release of the R2 subunit.

Regulation of the mammalian RNR by phosphorylation of its two R2 subunits, RRM2 and p53R2, has also been reported. RRM2, the main R2 subunit in proliferating cells, peaks in S phase and is degraded in G 2 upon completion of DNA replication. The G 2 proteolysis of RRM2 is triggered by CDK-mediated phosphorylation of Thr-33 ( 42 ). RRM2 contains an additional CDK phosphorylation site, Ser-20 ( 43 ), although the physiological significance of this phosphorylation is unclear. The DNA damage-inducible p53R2 is phosphorylated by ATM at Ser-72 in response to UV irradiation, which increases p53R2's stability and cellular survival under genotoxic stress ( 44 ). p53R2 has also been shown to interact with kinase MEK2 (extracellular signal-regulated kinase [ERK] kinase 2/mitogen-activated protein [MAP] kinase kinase 2), which is required for serum-stimulated increase in RNR activity ( 45 ). It would thus appear that phosphorylation of an RNR subunit is a common theme of RNR regulation both during normal cell cycle progression and in response to DNA damage.
