---
pmid: '38231884'
title: ARID3C Acts as a Regulator of Monocyte-to-Macrophage Differentiation Interacting
  with NPM1.
authors:
- Kim HS
- Kim YI
- Cho JY
journal: J Proteome Res
year: '2024'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC11302414
doi: 10.1021/acs.jproteome.3c00509
---

# ARID3C Acts as a Regulator of Monocyte-to-Macrophage Differentiation Interacting with NPM1.
**Authors:** Kim HS, Kim YI, Cho JY
**Journal:** J Proteome Res (2024)
**DOI:** [10.1021/acs.jproteome.3c00509](https://doi.org/10.1021/acs.jproteome.3c00509)
**PMC:** [PMC11302414](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11302414/)

## Abstract

1. J Proteome Res. 2024 Aug 2;23(8):2882-2892. doi:
10.1021/acs.jproteome.3c00509.  Epub 2024 Jan 17.

ARID3C Acts as a Regulator of Monocyte-to-Macrophage Differentiation Interacting 
with NPM1.

Kim HS(1)(2), Kim YI(1)(2), Cho JY(1)(2).

Author information:
(1)Department of Biochemistry, College of Veterinary Medicine, Research 
Institute for Veterinary Science, and BK21 FOUR Future Veterinary Medicine 
Leading Education and Research Center, Seoul National University, Seoul 08826, 
Republic of Korea.
(2)Comparative Medicine Disease Research Center (CDRC), Science Research Center 
(SRC), Seoul National University, Seoul 08826, Republic of Korea.

ARID3C is a protein located on human chromosome 9 and expressed at low levels in 
various organs, yet its biological function has not been elucidated. In this 
study, we investigated both the cellular localization and function of ARID3C. 
Employing a combination of LC-MS/MS and deep learning techniques, we identified 
NPM1 as a binding partner for ARID3C's nuclear shuttling. ARID3C was found to 
predominantly localize with the nucleus, where it functioned as a transcription 
factor for genes STAT3, STAT1, and JUNB, thereby facilitating 
monocyte-to-macrophage differentiation. The precise binding sites between ARID3C 
and NPM1 were predicted by AlphaFold2. Mutating this binding site prevented 
ARID3C from interacting with NPM1, resulting in its retention in the cytoplasm 
instead of translocation to the nucleus. Consequently, ARID3C lost its ability 
to bind to the promoters of target genes, leading to a loss of 
monocyte-to-macrophage differentiation. Collectively, our findings indicate that 
ARID3C forms a complex with NPM1 to translocate to the nucleus, acting as a 
transcription factor that promotes the expression of the genes involved in 
monocyte-to-macrophage differentiation.

DOI: 10.1021/acs.jproteome.3c00509
PMCID: PMC11302414
PMID: 38231884 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing financial 
interest.

## Full Text

IntroductionProtein analysis plays a pivotal role
in understanding cellular
processes, disease mechanisms, and biomarker discovery. However, according
to the latest version of neXtProt DB, (released on April 18, 2023)
as a result of numerous studies, it has been revealed that 18,397
out of 20,389 coding genes in humans, which constitutes 90% of the
total, have evidence at the protein level. This discovery began with
the Chromosome-centric Human Proteome Project (C-HPP), initiated by
the Human Proteome Organization (HUPO) in 2012, to identify missing
proteins (MPs) and characterize the protein’s role encoded
by each chromosome. Proteins are categorized into five levels according
to their protein existence (PE), ranging from experimental evidence
at the protein level (PE1) to experimental evidence at the transcript
level (PE2), protein inferred from homology (PE3), protein predicted
(PE4), and protein uncertain (PE5). The number of MPs and uncharacterized
PEs still existed, and they are considered to be in the “dark
field” of proteomics research. Over the years, significant
advancements in coupled high-performance liquid chromatography (HPLC)-
and mass spectrometry (MS)-based techniques have revolutionized the
field of protein analysis, enabling more comprehensive and in-depth
investigations.1 MS-based approaches also
provide a quantitative analysis, along with the capability to detect
post-translational modifications2 and analyze
complex mixtures.3 These technological
advancements have facilitated research on proteins that were previously
challenging to study using traditional methods.4 Consequently, the number of proteins with confirmed existence
has increased from 15,649 to 18,397 over the past decade (as of April
18th, 2023 released in neXtProt). Alongside the advancements in MS
technology, there is a growing interest in employing deep learning
approaches for protein analysis.5 Integration
of these two approaches has enhanced our ability to study protein
structure, function, and interactions. In this study, we employed
MS and deep learning-based AlphaFold26,7 to investigate
the interactome and its function of the partially studied protein
ARID3C (AT-rich interactive domain-containing protein 3C).ARID3C
is located in the human chr 9p13.3 and categorized as the
PE1 level, but its molecular function in cells has not been fully
elucidated yet except in B cells.8 ARID3C
is a member of the AT-rich interaction domain 3 (ARID3) transcription
family consisting of two functional domains that play a major role
in its function. The first one is the ARID domain, which is known
to bind to DNA,9 and the other is the REKLES
domain, which is known to be involved in nuclear import/export.10 Other ARID3 family ARID3A localized on chromosome
19 and was reported to have various functions in embryonic development,11 immune,12 and cancers.13−15 Unlike ARID3A, only one study reported that ARID3C was shown to
participate in the immunoglobin gene transcription in B cells.8 Tidwell et al. revealed that the mouse homologue
of human ARID3C, referred to as Brightlike, is expressed in B lineage
lymphocytes, undergoes nuclear-cytoplasmic shuttling, associates with
Bright (ARID3A), and acts as a coactivator for Bright (ARID3A) dependent
IgH transcription. However, the function of ARID3C in monocyte-to-macrophage
differentiation is unknown. In this study, we employed a combination
of LC-MS/MS and a deep learning model to unravel the interacting partners,
cellular localization, and molecular functions of ARID3C during the
process of monocyte-to-macrophage differentiation.

Introduction

Protein analysis plays a pivotal role
in understanding cellular
processes, disease mechanisms, and biomarker discovery. However, according
to the latest version of neXtProt DB, (released on April 18, 2023)
as a result of numerous studies, it has been revealed that 18,397
out of 20,389 coding genes in humans, which constitutes 90% of the
total, have evidence at the protein level. This discovery began with
the Chromosome-centric Human Proteome Project (C-HPP), initiated by
the Human Proteome Organization (HUPO) in 2012, to identify missing
proteins (MPs) and characterize the protein’s role encoded
by each chromosome. Proteins are categorized into five levels according
to their protein existence (PE), ranging from experimental evidence
at the protein level (PE1) to experimental evidence at the transcript
level (PE2), protein inferred from homology (PE3), protein predicted
(PE4), and protein uncertain (PE5). The number of MPs and uncharacterized
PEs still existed, and they are considered to be in the “dark
field” of proteomics research. Over the years, significant
advancements in coupled high-performance liquid chromatography (HPLC)-
and mass spectrometry (MS)-based techniques have revolutionized the
field of protein analysis, enabling more comprehensive and in-depth
investigations.1 MS-based approaches also
provide a quantitative analysis, along with the capability to detect
post-translational modifications2 and analyze
complex mixtures.3 These technological
advancements have facilitated research on proteins that were previously
challenging to study using traditional methods.4 Consequently, the number of proteins with confirmed existence
has increased from 15,649 to 18,397 over the past decade (as of April
18th, 2023 released in neXtProt). Alongside the advancements in MS
technology, there is a growing interest in employing deep learning
approaches for protein analysis.5 Integration
of these two approaches has enhanced our ability to study protein
structure, function, and interactions. In this study, we employed
MS and deep learning-based AlphaFold26,7 to investigate
the interactome and its function of the partially studied protein
ARID3C (AT-rich interactive domain-containing protein 3C).

ARID3C
is located in the human chr 9p13.3 and categorized as the
PE1 level, but its molecular function in cells has not been fully
elucidated yet except in B cells.8 ARID3C
is a member of the AT-rich interaction domain 3 (ARID3) transcription
family consisting of two functional domains that play a major role
in its function. The first one is the ARID domain, which is known
to bind to DNA,9 and the other is the REKLES
domain, which is known to be involved in nuclear import/export.10 Other ARID3 family ARID3A localized on chromosome
19 and was reported to have various functions in embryonic development,11 immune,12 and cancers.13−15 Unlike ARID3A, only one study reported that ARID3C was shown to
participate in the immunoglobin gene transcription in B cells.8 Tidwell et al. revealed that the mouse homologue
of human ARID3C, referred to as Brightlike, is expressed in B lineage
lymphocytes, undergoes nuclear-cytoplasmic shuttling, associates with
Bright (ARID3A), and acts as a coactivator for Bright (ARID3A) dependent
IgH transcription. However, the function of ARID3C in monocyte-to-macrophage
differentiation is unknown. In this study, we employed a combination
of LC-MS/MS and a deep learning model to unravel the interacting partners,
cellular localization, and molecular functions of ARID3C during the
process of monocyte-to-macrophage differentiation.

Experimental SectionCell Culture and DifferentiationHuman embryonic kidney
(HEK) 293T cell lines and leukemia cell lines THP-1 and HL-60 were
obtained from the Korean Cell Line Bank (KCLB; Seoul, Korea). HEK293T
cells were maintained in Dulbecco’s modified Eagle medium (DMEM,
HyClone Laboratories Inc., Logan, Utah, USA), and both leukemia cell
lines were maintained in RPMI-1640 (HyClone Laboratories Inc., Logan,
Utah, USA) containing 10% fetal bovine serum (HyClone, Waltham, MA,
USA) and a 1% antibiotic/antimycotic solution (Gibco, Life Technologies
Corp., NY, USA) at 37 °C in a 5% CO2 condition. The
monocyte-to-macrophage differentiation of THP-1 and HL-60 was induced
with phorbol 12-myristate 13-acetate (PMA). PMA was added to a final
concentration of 0.1 μM for 1 × 106 cells of
THP-1 and 2 × 106 cells of HL-60.RNA Isolation, cDNA Synthesis, and Real-Time PCR AssayThe real-time PCR assay was performed as previously described.16 Briefly, quantitative RT-PCR was performed in
a CFX Connect Real-Time PCR System (Biorad, CA, USA) using Go Taq
polymerase (Promega, WI, USA). The PCR conditions were 5 min at 95
°C, followed by 35 cycles of 95 °C for 30 s, 61 °C
for 30 s, and 72 °C for 30 s. The comparative Ct method was used
for analyzing data with GAPDH serving as an internal normalization
control. For real-time PCR analysis, we designed the primer to amplify
the exon–exon junction of a gene of interest. The specific
primer sets are provided in Table S1.Western Blotting AssayThe Western blotting assay was
performed as previously described.17 Information
on antibodies used in the study is provided in Table S1.MutagenesisTo construct ARID3C proteins with a deletion
in the REKELES β domain and four amino acid substitutions, site-directed
mutagenesis was performed with a Q5 Site-Directed Mutagenesis Kit
following the manufacturer’s instructions (New England Biolabs,
Ipswich, MA, USA). Primers used for Q5 site-directed mutagenesis were
designed using the NEBase Changer tool (https://nebasechanger.neb.com/). In brief, deletion and substitution primers were employed for
exponential amplification followed by the use of the kinase-ligase-Dpn1
(KLD) enzyme to remove the template. Subsequently, plasmids were obtained
through transformation using competent E. coli.The substituted amino acids were chosen based on the principles
of “Grantham’s distance”, “Sneath’s
distance”, and “Miyata’s distance”, selecting
the amino acid with the greatest difference, dissimilarity, and distance,
producing the V381D, F383D, A384W, and R385G mutants.HaloTag Protein PurificationHEK293T cells were transfected
with 10 μg of N-term Halo tagged ARID3C plasmid DNA using a
NEPA21 electroporator (Nepagene, Chiba, Japan) with following the
manufacturer’s instructions (Voltage; 150 V, Pulse; 5 ms).
The detailed structure of the HaloTag ARID3C plasmid is provided in Figure S1, ordered from Promega (ORF clone #FHC02454).
Two days after transfection, HEK293T cells were harvested with a mammalian
lysis buffer (50 mM Tris–HCl, pH 7.5, 150 mM NaCl, 1% Triton
X-100, and 0.1% sodium deoxycholate) supplemented with a protease
inhibitor cocktail. Lysate was cleared by centrifugation, and the
supernatant was incubated with the pre-equilibrated resin for 15 min
at room temperature (RT) with rotation. After incubation, the resin
was washed four times with a mammalian lysis buffer. Then, two different
methods of elution were used for immune blot and interactome analysis.
For the immunoblotting, the resin was resuspended with a SDS elution
buffer (50 mM Tris–HCl, pH 7.5, and 1% SDS) and boiled for
10 min at 100 °C. For interactome analysis, the resin was resuspended
with TEV protease for 20 min with rotation in RT to cleave ARID3C
with HaloTag, and TEV protease was removed by incubation with MagneHis
Ni-Particles. Tag-free proteins were analyzed by LC-MS/MS.Filter-Aided Sample Preparation (FASP) and Mass Spectrometry
(MS) AnalysisInteractome proteins eluted from HaloTag purification
were digested with trypsin following the FASP protocol.18 The peptide was cleaned up and fractionated
using poly(styrene-divinylbenzene)-reversed-phase sulfonate (SDB-RPS)
StageTips performed as previously described.19 Briefly, LC-MS/MS analysis was performed with an Orbitrap Fusion
Lumos mass spectrometer (Thermo Fisher Scientific, IL, USA) coupled
with an EASY-nLC1200 UHPLC (Thermo Fisher Scientific). The sample
was injected into an EASY-Spray column (75 μm i.d. × 50
cm; PepMap RSLC C18 particle, 2 μm particle size, 100 Å
pore size, Thermo Fisher Scientific) followed by a 120 min LC gradient
at a flow rate 300 nL min–1. MS measurements were
acquired by data-dependent mode with a 3 s cycle time during the HPLC
separation. Raw data were processed with MaxQuant (v. 1.6.5.1) software,
and a threshold was set to a 1% false discovery rate (FDR) for both
PSM and protein. The UniProt human proteome (number of entries: 20,423)
was used for database searching. In this study, the criteria that
followed the HUPO guidelines were used for data analysis and protein
identification. The proteins were identified with two or more unique
peptides (>9 amino acids). Raw data was deposited at https://massive.ucsd.edu/ProteoSAFe/static/massive.jsp?task=7c82d864a7a24a7eb8afc824744a7e88 (MassIVE MSV000093593).Chromatin Immunoprecipitation (ChIP)The Halo ChIP
system was performed with the HaloCHIP system following the manufacturer’s
instructions (Promega, WI, USA, #G9410). Briefly, transfected 293T
cells were cross-linked with formaldehyde (final concentration; 1%)
and DNA was sonicated to a size of 500–2,000 bp using a Bioruptor
Pico sonicator (Diagenode, Paris, France) for 10 cycles of 30 s on;
30 s off, followed by centrifugation at a high speed for 5 min. Supernatants
were incubated with pre-equilibrated resin for 2 h with mixing at
RT. The resin was washed subsequently with a mammalian lysis buffer,
nuclease-free water, a high salt wash buffer, and nuclease-free water.
Captured DNA was reverse cross-linked by adding a reversal buffer
for 12 h at 65 °C and eluted DNA was cleaned up with a Fragment
DNA purification Kit (iNtRON Biotechnology, Seongnam, Korea). The
purity and quantity of isolated DNA were measured by the ratio of
260/280 nm absorbance, and 1 ng of DNA was used for PCR.Flow CytometryPMA-treated HL-60 and THP-1 cells were
detached and rinsed with PBS two times. The cell pellets were resuspended
with a cell staining buffer (BioLegend) and incubated with the primary
antibody; CD11b for 15 min at 4 °C. After the cells were washed
with a cell staining buffer, the cells were incubated with the secondary
antibody at 4 °C for 15 min. To remove dead cells, 7-AAD was
added, and the samples were analyzed with FACS AriaII (BD Biosciences,
NJ, USA). Information on antibodies used in study is provided in Table S1.ImmunostainingImmunostaining was done as described
previously.16 Briefly, HEK293T cells were
fixed with 4% paraformaldehyde and permeabilized with 0.5% Triton
X-100 in PBS. After a washing treatment with PBS/T, the cells were
blocked with 5% BSA and incubated primary antibodies for 1 h at room
temperature. Secondary antibodies were incubated for 1 h at room temperature,
and nuclei counterstaining was performed using 1 μg/mL DAPI
(Thermo Fisher Scientific, MA, USA). Information on the used antibodies
is provided in Table S1.Bioinformatic AnalysisProtein–protein interaction
(PPI) network analysis was conducted with the STRING DB (v. 11) (https://string-db.org/). The proteins
were classified using the UniProt database based on their subcellular
location. AlphaFold2 v. 2.2.0 generates a PDB file based on the full
sequence of the protein.7 The best model
from the generated PDB file was utilized for standard molecular visualization.
Structure figures were visualized using UCSF ChimeraX v.1.2.5 developed
by the Resource for Biocomputing, Visualization, and Informatics at
the University of California, San Francisco, with support from National
Institutes of Health R01-GM129325 and the Office of Cyber Infrastructure
and Computational Biology, National Institute of Allergy and Infectious
Diseases.20 The average shortest distance
between protein and protein is 4.0 Å.Statistical AnalysisStatistical analysis was performed
using Prism software (v. 8.0.1, GraphPad Software). For the statistical
significance of multiple groups, 2-way ANOVA combined with Tukey’s
honest significant difference (HSD) test was performed, and Student’s t test was performed for analysis between two groups. The
data represent means ± SEM. Significant differences are indicated
with different letters in each figure legend.

Experimental Section

Cell Culture and DifferentiationHuman embryonic kidney
(HEK) 293T cell lines and leukemia cell lines THP-1 and HL-60 were
obtained from the Korean Cell Line Bank (KCLB; Seoul, Korea). HEK293T
cells were maintained in Dulbecco’s modified Eagle medium (DMEM,
HyClone Laboratories Inc., Logan, Utah, USA), and both leukemia cell
lines were maintained in RPMI-1640 (HyClone Laboratories Inc., Logan,
Utah, USA) containing 10% fetal bovine serum (HyClone, Waltham, MA,
USA) and a 1% antibiotic/antimycotic solution (Gibco, Life Technologies
Corp., NY, USA) at 37 °C in a 5% CO2 condition. The
monocyte-to-macrophage differentiation of THP-1 and HL-60 was induced
with phorbol 12-myristate 13-acetate (PMA). PMA was added to a final
concentration of 0.1 μM for 1 × 106 cells of
THP-1 and 2 × 106 cells of HL-60.

Cell Culture and Differentiation

Human embryonic kidney
(HEK) 293T cell lines and leukemia cell lines THP-1 and HL-60 were
obtained from the Korean Cell Line Bank (KCLB; Seoul, Korea). HEK293T
cells were maintained in Dulbecco’s modified Eagle medium (DMEM,
HyClone Laboratories Inc., Logan, Utah, USA), and both leukemia cell
lines were maintained in RPMI-1640 (HyClone Laboratories Inc., Logan,
Utah, USA) containing 10% fetal bovine serum (HyClone, Waltham, MA,
USA) and a 1% antibiotic/antimycotic solution (Gibco, Life Technologies
Corp., NY, USA) at 37 °C in a 5% CO2 condition. The
monocyte-to-macrophage differentiation of THP-1 and HL-60 was induced
with phorbol 12-myristate 13-acetate (PMA). PMA was added to a final
concentration of 0.1 μM for 1 × 106 cells of
THP-1 and 2 × 106 cells of HL-60.

RNA Isolation, cDNA Synthesis, and Real-Time PCR AssayThe real-time PCR assay was performed as previously described.16 Briefly, quantitative RT-PCR was performed in
a CFX Connect Real-Time PCR System (Biorad, CA, USA) using Go Taq
polymerase (Promega, WI, USA). The PCR conditions were 5 min at 95
°C, followed by 35 cycles of 95 °C for 30 s, 61 °C
for 30 s, and 72 °C for 30 s. The comparative Ct method was used
for analyzing data with GAPDH serving as an internal normalization
control. For real-time PCR analysis, we designed the primer to amplify
the exon–exon junction of a gene of interest. The specific
primer sets are provided in Table S1.

RNA Isolation, cDNA Synthesis, and Real-Time PCR Assay

The real-time PCR assay was performed as previously described.16 Briefly, quantitative RT-PCR was performed in
a CFX Connect Real-Time PCR System (Biorad, CA, USA) using Go Taq
polymerase (Promega, WI, USA). The PCR conditions were 5 min at 95
°C, followed by 35 cycles of 95 °C for 30 s, 61 °C
for 30 s, and 72 °C for 30 s. The comparative Ct method was used
for analyzing data with GAPDH serving as an internal normalization
control. For real-time PCR analysis, we designed the primer to amplify
the exon–exon junction of a gene of interest. The specific
primer sets are provided in Table S1.

Western Blotting AssayThe Western blotting assay was
performed as previously described.17 Information
on antibodies used in the study is provided in Table S1.

Western Blotting Assay

The Western blotting assay was
performed as previously described.17 Information
on antibodies used in the study is provided in Table S1.

MutagenesisTo construct ARID3C proteins with a deletion
in the REKELES β domain and four amino acid substitutions, site-directed
mutagenesis was performed with a Q5 Site-Directed Mutagenesis Kit
following the manufacturer’s instructions (New England Biolabs,
Ipswich, MA, USA). Primers used for Q5 site-directed mutagenesis were
designed using the NEBase Changer tool (https://nebasechanger.neb.com/). In brief, deletion and substitution primers were employed for
exponential amplification followed by the use of the kinase-ligase-Dpn1
(KLD) enzyme to remove the template. Subsequently, plasmids were obtained
through transformation using competent E. coli.The substituted amino acids were chosen based on the principles
of “Grantham’s distance”, “Sneath’s
distance”, and “Miyata’s distance”, selecting
the amino acid with the greatest difference, dissimilarity, and distance,
producing the V381D, F383D, A384W, and R385G mutants.

Mutagenesis

To construct ARID3C proteins with a deletion
in the REKELES β domain and four amino acid substitutions, site-directed
mutagenesis was performed with a Q5 Site-Directed Mutagenesis Kit
following the manufacturer’s instructions (New England Biolabs,
Ipswich, MA, USA). Primers used for Q5 site-directed mutagenesis were
designed using the NEBase Changer tool (https://nebasechanger.neb.com/). In brief, deletion and substitution primers were employed for
exponential amplification followed by the use of the kinase-ligase-Dpn1
(KLD) enzyme to remove the template. Subsequently, plasmids were obtained
through transformation using competent E. coli.

The substituted amino acids were chosen based on the principles
of “Grantham’s distance”, “Sneath’s
distance”, and “Miyata’s distance”, selecting
the amino acid with the greatest difference, dissimilarity, and distance,
producing the V381D, F383D, A384W, and R385G mutants.

HaloTag Protein PurificationHEK293T cells were transfected
with 10 μg of N-term Halo tagged ARID3C plasmid DNA using a
NEPA21 electroporator (Nepagene, Chiba, Japan) with following the
manufacturer’s instructions (Voltage; 150 V, Pulse; 5 ms).
The detailed structure of the HaloTag ARID3C plasmid is provided in Figure S1, ordered from Promega (ORF clone #FHC02454).
Two days after transfection, HEK293T cells were harvested with a mammalian
lysis buffer (50 mM Tris–HCl, pH 7.5, 150 mM NaCl, 1% Triton
X-100, and 0.1% sodium deoxycholate) supplemented with a protease
inhibitor cocktail. Lysate was cleared by centrifugation, and the
supernatant was incubated with the pre-equilibrated resin for 15 min
at room temperature (RT) with rotation. After incubation, the resin
was washed four times with a mammalian lysis buffer. Then, two different
methods of elution were used for immune blot and interactome analysis.
For the immunoblotting, the resin was resuspended with a SDS elution
buffer (50 mM Tris–HCl, pH 7.5, and 1% SDS) and boiled for
10 min at 100 °C. For interactome analysis, the resin was resuspended
with TEV protease for 20 min with rotation in RT to cleave ARID3C
with HaloTag, and TEV protease was removed by incubation with MagneHis
Ni-Particles. Tag-free proteins were analyzed by LC-MS/MS.

HaloTag Protein Purification

HEK293T cells were transfected
with 10 μg of N-term Halo tagged ARID3C plasmid DNA using a
NEPA21 electroporator (Nepagene, Chiba, Japan) with following the
manufacturer’s instructions (Voltage; 150 V, Pulse; 5 ms).
The detailed structure of the HaloTag ARID3C plasmid is provided in Figure S1, ordered from Promega (ORF clone #FHC02454).
Two days after transfection, HEK293T cells were harvested with a mammalian
lysis buffer (50 mM Tris–HCl, pH 7.5, 150 mM NaCl, 1% Triton
X-100, and 0.1% sodium deoxycholate) supplemented with a protease
inhibitor cocktail. Lysate was cleared by centrifugation, and the
supernatant was incubated with the pre-equilibrated resin for 15 min
at room temperature (RT) with rotation. After incubation, the resin
was washed four times with a mammalian lysis buffer. Then, two different
methods of elution were used for immune blot and interactome analysis.
For the immunoblotting, the resin was resuspended with a SDS elution
buffer (50 mM Tris–HCl, pH 7.5, and 1% SDS) and boiled for
10 min at 100 °C. For interactome analysis, the resin was resuspended
with TEV protease for 20 min with rotation in RT to cleave ARID3C
with HaloTag, and TEV protease was removed by incubation with MagneHis
Ni-Particles. Tag-free proteins were analyzed by LC-MS/MS.

Filter-Aided Sample Preparation (FASP) and Mass Spectrometry
(MS) AnalysisInteractome proteins eluted from HaloTag purification
were digested with trypsin following the FASP protocol.18 The peptide was cleaned up and fractionated
using poly(styrene-divinylbenzene)-reversed-phase sulfonate (SDB-RPS)
StageTips performed as previously described.19 Briefly, LC-MS/MS analysis was performed with an Orbitrap Fusion
Lumos mass spectrometer (Thermo Fisher Scientific, IL, USA) coupled
with an EASY-nLC1200 UHPLC (Thermo Fisher Scientific). The sample
was injected into an EASY-Spray column (75 μm i.d. × 50
cm; PepMap RSLC C18 particle, 2 μm particle size, 100 Å
pore size, Thermo Fisher Scientific) followed by a 120 min LC gradient
at a flow rate 300 nL min–1. MS measurements were
acquired by data-dependent mode with a 3 s cycle time during the HPLC
separation. Raw data were processed with MaxQuant (v. 1.6.5.1) software,
and a threshold was set to a 1% false discovery rate (FDR) for both
PSM and protein. The UniProt human proteome (number of entries: 20,423)
was used for database searching. In this study, the criteria that
followed the HUPO guidelines were used for data analysis and protein
identification. The proteins were identified with two or more unique
peptides (>9 amino acids). Raw data was deposited at https://massive.ucsd.edu/ProteoSAFe/static/massive.jsp?task=7c82d864a7a24a7eb8afc824744a7e88 (MassIVE MSV000093593).

Filter-Aided Sample Preparation (FASP) and Mass Spectrometry
(MS) Analysis

Interactome proteins eluted from HaloTag purification
were digested with trypsin following the FASP protocol.18 The peptide was cleaned up and fractionated
using poly(styrene-divinylbenzene)-reversed-phase sulfonate (SDB-RPS)
StageTips performed as previously described.19 Briefly, LC-MS/MS analysis was performed with an Orbitrap Fusion
Lumos mass spectrometer (Thermo Fisher Scientific, IL, USA) coupled
with an EASY-nLC1200 UHPLC (Thermo Fisher Scientific). The sample
was injected into an EASY-Spray column (75 μm i.d. × 50
cm; PepMap RSLC C18 particle, 2 μm particle size, 100 Å
pore size, Thermo Fisher Scientific) followed by a 120 min LC gradient
at a flow rate 300 nL min–1. MS measurements were
acquired by data-dependent mode with a 3 s cycle time during the HPLC
separation. Raw data were processed with MaxQuant (v. 1.6.5.1) software,
and a threshold was set to a 1% false discovery rate (FDR) for both
PSM and protein. The UniProt human proteome (number of entries: 20,423)
was used for database searching. In this study, the criteria that
followed the HUPO guidelines were used for data analysis and protein
identification. The proteins were identified with two or more unique
peptides (>9 amino acids). Raw data was deposited at https://massive.ucsd.edu/ProteoSAFe/static/massive.jsp?task=7c82d864a7a24a7eb8afc824744a7e88 (MassIVE MSV000093593).

Chromatin Immunoprecipitation (ChIP)The Halo ChIP
system was performed with the HaloCHIP system following the manufacturer’s
instructions (Promega, WI, USA, #G9410). Briefly, transfected 293T
cells were cross-linked with formaldehyde (final concentration; 1%)
and DNA was sonicated to a size of 500–2,000 bp using a Bioruptor
Pico sonicator (Diagenode, Paris, France) for 10 cycles of 30 s on;
30 s off, followed by centrifugation at a high speed for 5 min. Supernatants
were incubated with pre-equilibrated resin for 2 h with mixing at
RT. The resin was washed subsequently with a mammalian lysis buffer,
nuclease-free water, a high salt wash buffer, and nuclease-free water.
Captured DNA was reverse cross-linked by adding a reversal buffer
for 12 h at 65 °C and eluted DNA was cleaned up with a Fragment
DNA purification Kit (iNtRON Biotechnology, Seongnam, Korea). The
purity and quantity of isolated DNA were measured by the ratio of
260/280 nm absorbance, and 1 ng of DNA was used for PCR.

Chromatin Immunoprecipitation (ChIP)

The Halo ChIP
system was performed with the HaloCHIP system following the manufacturer’s
instructions (Promega, WI, USA, #G9410). Briefly, transfected 293T
cells were cross-linked with formaldehyde (final concentration; 1%)
and DNA was sonicated to a size of 500–2,000 bp using a Bioruptor
Pico sonicator (Diagenode, Paris, France) for 10 cycles of 30 s on;
30 s off, followed by centrifugation at a high speed for 5 min. Supernatants
were incubated with pre-equilibrated resin for 2 h with mixing at
RT. The resin was washed subsequently with a mammalian lysis buffer,
nuclease-free water, a high salt wash buffer, and nuclease-free water.
Captured DNA was reverse cross-linked by adding a reversal buffer
for 12 h at 65 °C and eluted DNA was cleaned up with a Fragment
DNA purification Kit (iNtRON Biotechnology, Seongnam, Korea). The
purity and quantity of isolated DNA were measured by the ratio of
260/280 nm absorbance, and 1 ng of DNA was used for PCR.

Flow CytometryPMA-treated HL-60 and THP-1 cells were
detached and rinsed with PBS two times. The cell pellets were resuspended
with a cell staining buffer (BioLegend) and incubated with the primary
antibody; CD11b for 15 min at 4 °C. After the cells were washed
with a cell staining buffer, the cells were incubated with the secondary
antibody at 4 °C for 15 min. To remove dead cells, 7-AAD was
added, and the samples were analyzed with FACS AriaII (BD Biosciences,
NJ, USA). Information on antibodies used in study is provided in Table S1.

Flow Cytometry

PMA-treated HL-60 and THP-1 cells were
detached and rinsed with PBS two times. The cell pellets were resuspended
with a cell staining buffer (BioLegend) and incubated with the primary
antibody; CD11b for 15 min at 4 °C. After the cells were washed
with a cell staining buffer, the cells were incubated with the secondary
antibody at 4 °C for 15 min. To remove dead cells, 7-AAD was
added, and the samples were analyzed with FACS AriaII (BD Biosciences,
NJ, USA). Information on antibodies used in study is provided in Table S1.

ImmunostainingImmunostaining was done as described
previously.16 Briefly, HEK293T cells were
fixed with 4% paraformaldehyde and permeabilized with 0.5% Triton
X-100 in PBS. After a washing treatment with PBS/T, the cells were
blocked with 5% BSA and incubated primary antibodies for 1 h at room
temperature. Secondary antibodies were incubated for 1 h at room temperature,
and nuclei counterstaining was performed using 1 μg/mL DAPI
(Thermo Fisher Scientific, MA, USA). Information on the used antibodies
is provided in Table S1.

Immunostaining

Immunostaining was done as described
previously.16 Briefly, HEK293T cells were
fixed with 4% paraformaldehyde and permeabilized with 0.5% Triton
X-100 in PBS. After a washing treatment with PBS/T, the cells were
blocked with 5% BSA and incubated primary antibodies for 1 h at room
temperature. Secondary antibodies were incubated for 1 h at room temperature,
and nuclei counterstaining was performed using 1 μg/mL DAPI
(Thermo Fisher Scientific, MA, USA). Information on the used antibodies
is provided in Table S1.

Bioinformatic AnalysisProtein–protein interaction
(PPI) network analysis was conducted with the STRING DB (v. 11) (https://string-db.org/). The proteins
were classified using the UniProt database based on their subcellular
location. AlphaFold2 v. 2.2.0 generates a PDB file based on the full
sequence of the protein.7 The best model
from the generated PDB file was utilized for standard molecular visualization.
Structure figures were visualized using UCSF ChimeraX v.1.2.5 developed
by the Resource for Biocomputing, Visualization, and Informatics at
the University of California, San Francisco, with support from National
Institutes of Health R01-GM129325 and the Office of Cyber Infrastructure
and Computational Biology, National Institute of Allergy and Infectious
Diseases.20 The average shortest distance
between protein and protein is 4.0 Å.

Bioinformatic Analysis

Protein–protein interaction
(PPI) network analysis was conducted with the STRING DB (v. 11) (https://string-db.org/). The proteins
were classified using the UniProt database based on their subcellular
location. AlphaFold2 v. 2.2.0 generates a PDB file based on the full
sequence of the protein.7 The best model
from the generated PDB file was utilized for standard molecular visualization.
Structure figures were visualized using UCSF ChimeraX v.1.2.5 developed
by the Resource for Biocomputing, Visualization, and Informatics at
the University of California, San Francisco, with support from National
Institutes of Health R01-GM129325 and the Office of Cyber Infrastructure
and Computational Biology, National Institute of Allergy and Infectious
Diseases.20 The average shortest distance
between protein and protein is 4.0 Å.

Statistical AnalysisStatistical analysis was performed
using Prism software (v. 8.0.1, GraphPad Software). For the statistical
significance of multiple groups, 2-way ANOVA combined with Tukey’s
honest significant difference (HSD) test was performed, and Student’s t test was performed for analysis between two groups. The
data represent means ± SEM. Significant differences are indicated
with different letters in each figure legend.

Statistical Analysis

Statistical analysis was performed
using Prism software (v. 8.0.1, GraphPad Software). For the statistical
significance of multiple groups, 2-way ANOVA combined with Tukey’s
honest significant difference (HSD) test was performed, and Student’s t test was performed for analysis between two groups. The
data represent means ± SEM. Significant differences are indicated
with different letters in each figure legend.

ResultsAnalysis of ARI D3C Interactors by Coimmunoprecipitation and
Mass SpectrometryTo identify the interaction partners of
ARID3C, we transiently expressed a Halo-tagged ARID3C protein in 293T
cells and subsequently purified the tagged protein along with its
interactors (Figure 1A and S1A). As a negative control for
protein–protein interactions, we used protein extracts from
cells expressing the free HaloTag (Figure 1B). Protein identifications from the mass
spectrometry were filtered using the criteria of the HUPO guidelines,
then proteins commonly identified in the HaloTag control were removed.
Moreover, false-positive proteins with predicted cellular localization
differing from ARID3C were also excluded.Figure 1Interactome analysis
of ARID3C. (A) Strategy to identify interaction
partners of ARID3C. HEK293T cells used for ARID3C overexpression and
the purified ARID3C complex by HaloTag purification were digested
into peptides using the FASP digestion method for LC-MS/MS. (B) Schematic
illustration of the ARIC3C interactome analysis. To identify the ARID3C
binding partner, we filtered out proteins that were expected to be
false positive under 3 step conditions. (C) Venn diagram showing the
distribution of the total 158 candidates selected. (D) Distribution
of the subcellular localization of selected 129 candidates. (E) Protein–protein
interaction (PPI) network of the proteins specific to ARID3C interactome
and predicted to be present with the nucleus. (F) Representative images
showing the subcellular localization of ARID3C and NPM1. Double immunofluorescence
staining shows colocalization between ARID3C and NPM1 in the nucleus.
Scale bar: 10 μm. Figure 1A,B created with BioRender.com.The IP-MS analysis identified a total of 158 proteins,
with a FDR
calculation below 1% at the global protein level. One hundred twenty-nine
proteins were unique to ARID3C, not overlapping with the HaloTag control
(Figure 1C and Table 1). We detected 20
unique peptides of ARID3C, and the MS spectrum is shown in Figure S2A. We conducted PPI analysis using the
STRING database, dividing proteins into two groups: all 129 proteins
(Figure S3A) and nuclear-localized proteins
(Figure 1E). When analyzing
the PPIs of the 129 proteins associated with ARID3C, we identified
five main clusters: Type III intermediate filament, Proteasome core
complex, Dendritic growth core, Muscle thin filament tropomyosin,
and Acetyl-CoA carboxylase complex. Notably, nuclear proteins exhibited
more specific clustering, particularly involving the lamin filament
and actin filament. We focused on proteins that constitute the lamin
filament, aiming to better elucidate the role of ARID3C as a transcription
factor. Among these lamin filament proteins (NPM1, RPLP0, LMNA, LMNB1,
and HNRNPA2B1), we emphasized NPM1, which occupies the most nodes.
Furthermore, ARID3C and NPM1 are colocalized in the nucleus in the
293T cells (Figure 1E). Thus, in this study, we focused on the interaction between ARID3C
and NPM1.Table 1List of 129 Proteins Identified Only
in ARID3C Log2
(LFQ intensity)    ARID3C_1ARID3C_2Halo_1Halo_2Number of unique peptidesNumber of MS/MS countMajority protein IDs (Uniprot)130.1230.0044NaNNaN19179P08670228.460428.2145NaNNaN13133P62258330.225230.1298NaNNaN17128P13796430.585830.6375NaNNaN10128P62937529.968729.8311NaNNaN7112P62328628.357228.233NaNNaN13110P12814729.40429.3822NaNNaN10100P63104832.057932.0949NaNNaN2096A6NKF2930.280230.2902NaNNaN484P150901029.945329.7907NaNNaN1181P0DP251128.90628.8909NaNNaN778P082381227.104827.0888NaNNaN1858P355791327.915928.046NaNNaN957P294011427.20627.1318NaNNaN1656P550721527.157126.9163NaNNaN1553P533961627.214527.115NaNNaN1552P110211727.257227.2982NaNNaN1151Q9BQE31827.204227.0744NaNNaN950P067331927.480927.3402NaNNaN1149P226262026.702126.792NaNNaN848P040752127.272727.5049NaNNaN1348P146182228.50628.59NaNNaN347P223922327.201327.202NaNNaN844O750832426.431526.3275NaNNaN2144P213332528.589828.4875NaNNaN743Q998802629.06329.0558NaNNaN742P409252726.709126.7141NaNNaN739P409262826.796726.5888NaNNaN637P378022926.433926.5495NaNNaN1537P386463027.082427.0201NaNNaN936P186693127.638227.8432NaNNaN836P235283228.064628.0349NaNNaN436Q140193326.881827.0149NaNNaN635P525653428.616128.5627NaNNaN534P074513529.208629.0912NaNNaN434P629873626.136526.1635NaNNaN834P277973726.920326.8601NaNNaN634P503953828.406828.3238NaNNaN333Q9H2993926.272926.3007NaNNaN632P493274025.733725.7989NaNNaN731P266414127.089627.0654NaNNaN631P619814226.74326.6946NaNNaN531Q011054326.051526.1457NaNNaN730P260384425.942425.9155NaNNaN929P025454526.448726.4448NaNNaN529P481634624.569423.1841NaNNaN228P319464725.92325.8683NaNNaN1227O437074827.368927.5557NaNNaN627P679364926.615426.3999NaNNaN426O148185025.632925.6336NaNNaN626P005585124.837326.393NaNNaN226P077375226.076925.7863NaNNaN526P099605328.589828.5863NaNNaN226P699055426.223826.0614NaNNaN325O753685525.45325.5016NaNNaN1024P136395626.073426.1436NaNNaN524P145505726.051125.9075NaNNaN724P609005822.71422.2538NaNNaN324Q148475925.444325.5144NaNNaN624Q165556025.969426.0793NaNNaN623P606606126.442126.4004NaNNaN522P067536226.164225.9491NaNNaN522P280666325.73226.1175NaNNaN622P629066423.107422.2677NaNNaN521P004506525.879925.8317NaNNaN621P257896625.195324.8988NaNNaN621P477566726.883926.9795NaNNaN519P170666825.653725.7283NaNNaN519P257886925.998626.0869NaNNaN619Q096667025.151224.9405NaNNaN319Q96CX27125.788525.9526NaNNaN418P207007226.050426.279NaNNaN618P217967324.038524.1862NaNNaN918Q138137423.628224.3248NaNNaN417P105997525.471725.2458NaNNaN417P206187625.533125.5971NaNNaN417P300417724.975124.827NaNNaN516P053877825.695225.6926NaNNaN516P273487925.129425.1703NaNNaN516P300508024.522824.6406NaNNaN315P072378125.582525.4838NaNNaN615P079008224.905924.9505NaNNaN415P311468324.742824.6945NaNNaN415P619788425.413425.4558NaNNaN515Q049178525.202125.3364NaNNaN314P067488626.551626.5969NaNNaN214P093828724.499324.392NaNNaN414P176618824.974124.9066NaNNaN514P680368925.681825.5923NaNNaN414Q015189024.179224.2282NaNNaN413O002999124.118623.6956NaNNaN213P074379225.253525.1577NaNNaN513P257869326.289424.4252NaNNaN313P311509425.627525.6403NaNNaN313P840909525.193925.0022NaNNaN212P497209626.333826.1736NaNNaN212P629429724.510624.4904NaNNaN312P629939824.539124.3048NaNNaN712Q006109925.135625.441NaNNaN312Q0146910024.40924.623NaNNaN212Q9268810123.591323.7504NaNNaN411P0049210224.728724.5849NaNNaN511P2570510325.326225.4438NaNNaN311P3008610425.768125.8786NaNNaN211P6331310524.566524.8052NaNNaN310Q96KP410624.062323.8665NaNNaN59P0039010723.20123.1414NaNNaN29P3004410823.760523.5635NaNNaN29P4012110925.579725.7457NaNNaN59P4588011024.604224.509NaNNaN49P5999811124.865824.5847NaNNaN29Q9949711224.146924.2116NaNNaN28P0016711323.000522.9948NaNNaN48P0719511423.270122.9133NaNNaN28P0733911527.199627.0812NaNNaN28P0760211624.401124.3697NaNNaN28P2921811723.431623.0263NaNNaN48P4694011823.698623.7716NaNNaN48P6301011924.509125.1422NaNNaN38Q0855412023.157923.2595NaNNaN57P0538812123.264123.1863NaNNaN27P6108812223.024322.6641NaNNaN26P0279012323.852423.8998NaNNaN36P2352612422.640422.507NaNNaN26P6115812523.68424.0302NaNNaN26Q1584312623.131623.1095NaNNaN35O1514412722.961322.9744NaNNaN35P0673712822.831322.6406NaNNaN35P1512112923.51123.6221NaNNaN25Q9NZT1ARID3C Interacts with NPM1 for Nuclear ShuttlingNext,
we employed AlphaFold2 to predict the 3D structures of the ARID3C
and NPM1 protein (Figure S4A,B) and predicted
its interaction with NPM1 (Figure 2A). Based on the report that NPM1 is involved in nuclear
shuttling,21 we hypothesized that ARID3C
may interact with NPM1 to enter the nucleus. Three-dimensional predictions
by AlphaFold2 unveiled that NPM1 interacts with two sites: the ARID
domain (left) and the REKLES β domain (right) of ARID3C. As
the ARID domain is known to be involved in DNA binding,9 we focused on the REKLES β domain. Four
amino acids, V381, F383, A384, and R385, in the REKLES β domain
of ARID3C are predicted to be involved in the binding to NPM1 (Figure 2B). To explore whether
ARID3C indeed binds to NPM1, we constructed two different mutations
(Figure 2C). The first
mutation deleted the whole REKLES β domain including the four
amino acids (ΔREK), while the second mutation substituted those
four amino acids with different amino acids: V381D, F383D, A384W,
and R385G (Mut). We transiently expressed ARID3C, ΔREK, and
Mut with the NPM1-flag in 293T cells. After IP using the HaloTag of
ARID3C and performing immunoblotting with the Flag antibody, we found
that ARID3C bound to NPM1, while ΔREK did not bind at all. The
Mut showed limited binding to NPM1. Herein, we have demonstrated that
ARID3C interacts with NPM1 and that the specific four amino acids
in the REKLES β domain play a pivotal role in this interaction
(Figure 2D).Figure 2ARID3C interacts
with NPM1 for nuclear shuttling. (A) Predicted
3D structure of ARID3C and NPM1 interaction complex revealed by AlphaFold2.
Blue, ARID3c; Gray, NPM1. Residue–residue alignment confidence
(PAE) contacts were labeled with dark blue. Magnified left image,
binding site between ARID domain of ARID3C and NPM1; right image,
REKLES β domain of ARID3C and NPM1. (B) Structure of REKLES
β domain amino acids of ARID3C and the corresponding amino acids
of NPM1. (C) Structure of the deletion and amino acid substitution
mutated in REKES β domain. (D) Coimmunoprecipitation assay with
ARID3C, ΔREK, and Mut with Halo antibody verified the interaction
between ARID3C and NPM1. (E) Representative images showing the subcellular
localization of mutated ARID3C and NPM1. Scale bar: 10 μm. Figure 2B created with BioRender.com.Next, we investigated the subcellular localization
of mutated ARID3C
and NPM1. We found that both ARID3C and NPM1 exist in the nucleus
of cells, while the deleted ARID3C (ΔREK) is retained in the
cytoplasm (Figure 2E). The substitution form of ARID3C (Mut) was found to have both
nuclear and cytoplasmic localization, which is reminiscent of the
IP-IB result (Figure 2D) with some binding to NPM1 and some not. These observations highlight
that the translocation of ARID3C from the cytoplasm to the nucleus
is contingent upon its interaction with the REKLES β domain
of NPM1, mainly those four amino acids V381, F383, A384, and R385.ARID3C Promotes PMA-Induced Myeloid DifferentiationTo investigate the role of ARID3C in monocyte differentiation, we
used AML patient-derived monocyte cell lines HL-60 and THP-1. HL-60
and THP-1 were treated with PMA to induce monocyte-to-macrophage differentiation
(Figure 3A). A significant
increase was observed in the expression of macrophage differentiation
markers, CD11b and CSF1R (Figure 3B). Next, we also confirmed an increase in the level
of protein expression of CD11b upon PMA-induced differentiation in
both cell types (Figure 3C). Flow cytometric analysis also showed that the number of CD11b-positive
cells increased with PMA treatment (Figure 3D).Figure 3ARID3C induce monocyte-to-macrophage differentiation.
(A) Graphical
representation of monocyte differentiation into macrophages induced
by phorbol-12-myristate-13-acetate (PMA). (B) Relative mRNA expression
of CD11b and CSF1R for monocyte-to-macrophage differentiation was
quantified by qRT-PCR. The results were analyzed using 2−ΔΔCT
and GAPDH as a reference gene. (C) Protein expression level of CD11b
quantified by Western Blot. β-Actin was used as an internal
control. (D) Representative flow cytometry of CD11b of monocyte and
PMA treated monocytes. (E) Halo and ARID3C overexpressed cells were
used for Western Blotting. (F,G) Relative mRNA expression of CD11b,
CD14, and CSF1R was quantified by qRT-PCR. HL-60 (F) and THP-1 (G)
cells were transfected with ARID3C and treated with PMA for 24 h.
The results were analyzed using 2−ΔΔCT and used
GAPDH as a reference gene. All data are presented as mean ± SEM
(n = 3). The Student’s t test
was used to analyze the data, and statistical analysis was performed.
**P < 0.01; n.s., not significant. Figure 3A created with BioRender.com.Subsequently, we conducted transient transfection
of HL-60 and
THP-1 with either the ARID3C or Halo control construct. Transfection
efficiency was confirmed by a Western blot (Figure 3E). After ARID3C transfection, the cells
were treated with PMA for the macrophage differentiation. Intriguingly,
the expression of ARID3C led to a further augmentation in the levels
of CD11b, CD14, and CSF1R mRNA when compared to the control (Figure 3F,G). These findings
collectively indicate that ARID3C experts a positive regulatory effect
on the monocyte-to-macrophage differentiation.As a Transcription Factor, ARID3C Directly Regulates Genes Involved
in Monocyte-to-Macrophage DifferentiationAfter induction
for 24 h with PMA, HL-60 cells that were transfected with ARID3C exhibited
a higher proportion of mature monocytes (Figure 4A). We investigated whether ARID3C binds
to NPM1 and enters to the nucleus to act as a transcription factor.
To begin, we conducted a screening for master transcription factor
genes that orchestrate the process of monocyte-to-macrophage differentiation.22 Subsequent to this screening, we identified
three genes (STAT3, STAT1, and JUNB) that possess an ARID-binding
DNA element within their respective promoters (Figure 4B). Notably, STAT3 is required for monocyte
migration and differentiation,23 while
STAT1 promotes macrophage activation and the production of inflammatory
cytokines.24 Moreover, JUNB is known to
participate in various cellular processes, including immune regulation
and differentiation.25 Together, these
transcription factors contribute to the differentiation of monocytes
into functional macrophages. Based on this information, we infer that
ARID3C acts as a transcription factor that regulates the expression
of three genes, which results in an increase in monocyte-to-macrophage
differentiation. Next, we performed chromatin immunoprecipitation
by the Halo tag followed by PCR (ChIP-PCR). Additionally, we hypothesized
that ARID3C would not function as a transcription factor if it failed
to bind with NPM1 and was retained in the cytoplasm. To validate this
hypothesis, we transfected both the original ARID3C and a mutated
form of ARID3C, and observed their binding to the promoters of STAT3,
STAT1, and JUNB (Figure 4C–E). The results showed that ARID3C bound to the promoters.
We observed that ARID3C binds to the promoters of STAT3, STAT1, and
JUNB, but the ΔREK failed to bind and Mut bound at a very limited
level, indicating that ARID3C requires an intact NPM1-binding site
to act as a transcription factor in differentiation (Figure 4C–E). In Figure 4F, we also detected failure
of binding capacity in mutated ARID3C by ChIP-qPCR. Furthermore, we
confirmed failure of intact between ARID3C and NPM1 leads to decrease
of differentiation by using flow cytometry to measure the level of
CD11b marker in cells transfected with the different ARID3C constructs
(Figure 4G). Upon transfection
of ARID3C in HL-60 cells, the highest expression of CD11b was observed,
whereas the deletion of the REKLES domain in ΔREK resulted in
the lowest CD11b expression.Figure 4As a transcription factor, ARID3C directly regulates
the genes
related to macrophage differentiation. (A) Phase contrast microscopy
shows that PMA induced HL-60 cells. ARID3C overexpressed HL-60 cell
shows morphological characteristics of macrophages. Scale bar: 250
μm. (B) Graphical diagram for selection of transcription factors
with an ARID domain related to macrophage differentiation. (C–E)
Chromatin immunoprecipitation (ChIP) analysis of STAT3, STAT1, and
JUNB in the 293T cells transfected with ARID3C, ΔREK, and Mut.
Genome structures of STAT3, STAT1, and JUNB are shown. Blue histogram
represents the predicted ARID3C binding domain, and the inner black
bar indicates the amplified region by PCR. (F) DNA fold enrichments
were quantified by ChIP-qPCR. (G) Expression levels of CD11b in Halo,
ARID3C, ΔREK, and Mut transfected HL-60 cells were analyzed
by FACS (red, Halo; blue, ARID3C; orange, ΔREK; green, Mut).
All data are presented as mean ± SEM (n = 3).
The statistical analysis is presented. Two-way ANOVA was used to compare
groups. # p < 0.01, * p <
0.05, and a, not significant.These findings indicate that ARID3C promotes macrophage
differentiation
by binding to NPM1 and translocating into the nucleus to act as a
transcription factor. Overall, we showed that ARID3C promotes monocyte
differentiation, but mutated ARID3C reduced differentiation ability.
Accordingly, ARID3C induces monocyte-to-macrophage differentiation
as a transcription factor by regulating the expression of STAT3, STAT1,
and JUNB.

Analysis of ARI D3C Interactors by Coimmunoprecipitation and
Mass SpectrometryTo identify the interaction partners of
ARID3C, we transiently expressed a Halo-tagged ARID3C protein in 293T
cells and subsequently purified the tagged protein along with its
interactors (Figure 1A and S1A). As a negative control for
protein–protein interactions, we used protein extracts from
cells expressing the free HaloTag (Figure 1B). Protein identifications from the mass
spectrometry were filtered using the criteria of the HUPO guidelines,
then proteins commonly identified in the HaloTag control were removed.
Moreover, false-positive proteins with predicted cellular localization
differing from ARID3C were also excluded.Figure 1Interactome analysis
of ARID3C. (A) Strategy to identify interaction
partners of ARID3C. HEK293T cells used for ARID3C overexpression and
the purified ARID3C complex by HaloTag purification were digested
into peptides using the FASP digestion method for LC-MS/MS. (B) Schematic
illustration of the ARIC3C interactome analysis. To identify the ARID3C
binding partner, we filtered out proteins that were expected to be
false positive under 3 step conditions. (C) Venn diagram showing the
distribution of the total 158 candidates selected. (D) Distribution
of the subcellular localization of selected 129 candidates. (E) Protein–protein
interaction (PPI) network of the proteins specific to ARID3C interactome
and predicted to be present with the nucleus. (F) Representative images
showing the subcellular localization of ARID3C and NPM1. Double immunofluorescence
staining shows colocalization between ARID3C and NPM1 in the nucleus.
Scale bar: 10 μm. Figure 1A,B created with BioRender.com.The IP-MS analysis identified a total of 158 proteins,
with a FDR
calculation below 1% at the global protein level. One hundred twenty-nine
proteins were unique to ARID3C, not overlapping with the HaloTag control
(Figure 1C and Table 1). We detected 20
unique peptides of ARID3C, and the MS spectrum is shown in Figure S2A. We conducted PPI analysis using the
STRING database, dividing proteins into two groups: all 129 proteins
(Figure S3A) and nuclear-localized proteins
(Figure 1E). When analyzing
the PPIs of the 129 proteins associated with ARID3C, we identified
five main clusters: Type III intermediate filament, Proteasome core
complex, Dendritic growth core, Muscle thin filament tropomyosin,
and Acetyl-CoA carboxylase complex. Notably, nuclear proteins exhibited
more specific clustering, particularly involving the lamin filament
and actin filament. We focused on proteins that constitute the lamin
filament, aiming to better elucidate the role of ARID3C as a transcription
factor. Among these lamin filament proteins (NPM1, RPLP0, LMNA, LMNB1,
and HNRNPA2B1), we emphasized NPM1, which occupies the most nodes.
Furthermore, ARID3C and NPM1 are colocalized in the nucleus in the
293T cells (Figure 1E). Thus, in this study, we focused on the interaction between ARID3C
and NPM1.Table 1List of 129 Proteins Identified Only
in ARID3C Log2
(LFQ intensity)    ARID3C_1ARID3C_2Halo_1Halo_2Number of unique peptidesNumber of MS/MS countMajority protein IDs (Uniprot)130.1230.0044NaNNaN19179P08670228.460428.2145NaNNaN13133P62258330.225230.1298NaNNaN17128P13796430.585830.6375NaNNaN10128P62937529.968729.8311NaNNaN7112P62328628.357228.233NaNNaN13110P12814729.40429.3822NaNNaN10100P63104832.057932.0949NaNNaN2096A6NKF2930.280230.2902NaNNaN484P150901029.945329.7907NaNNaN1181P0DP251128.90628.8909NaNNaN778P082381227.104827.0888NaNNaN1858P355791327.915928.046NaNNaN957P294011427.20627.1318NaNNaN1656P550721527.157126.9163NaNNaN1553P533961627.214527.115NaNNaN1552P110211727.257227.2982NaNNaN1151Q9BQE31827.204227.0744NaNNaN950P067331927.480927.3402NaNNaN1149P226262026.702126.792NaNNaN848P040752127.272727.5049NaNNaN1348P146182228.50628.59NaNNaN347P223922327.201327.202NaNNaN844O750832426.431526.3275NaNNaN2144P213332528.589828.4875NaNNaN743Q998802629.06329.0558NaNNaN742P409252726.709126.7141NaNNaN739P409262826.796726.5888NaNNaN637P378022926.433926.5495NaNNaN1537P386463027.082427.0201NaNNaN936P186693127.638227.8432NaNNaN836P235283228.064628.0349NaNNaN436Q140193326.881827.0149NaNNaN635P525653428.616128.5627NaNNaN534P074513529.208629.0912NaNNaN434P629873626.136526.1635NaNNaN834P277973726.920326.8601NaNNaN634P503953828.406828.3238NaNNaN333Q9H2993926.272926.3007NaNNaN632P493274025.733725.7989NaNNaN731P266414127.089627.0654NaNNaN631P619814226.74326.6946NaNNaN531Q011054326.051526.1457NaNNaN730P260384425.942425.9155NaNNaN929P025454526.448726.4448NaNNaN529P481634624.569423.1841NaNNaN228P319464725.92325.8683NaNNaN1227O437074827.368927.5557NaNNaN627P679364926.615426.3999NaNNaN426O148185025.632925.6336NaNNaN626P005585124.837326.393NaNNaN226P077375226.076925.7863NaNNaN526P099605328.589828.5863NaNNaN226P699055426.223826.0614NaNNaN325O753685525.45325.5016NaNNaN1024P136395626.073426.1436NaNNaN524P145505726.051125.9075NaNNaN724P609005822.71422.2538NaNNaN324Q148475925.444325.5144NaNNaN624Q165556025.969426.0793NaNNaN623P606606126.442126.4004NaNNaN522P067536226.164225.9491NaNNaN522P280666325.73226.1175NaNNaN622P629066423.107422.2677NaNNaN521P004506525.879925.8317NaNNaN621P257896625.195324.8988NaNNaN621P477566726.883926.9795NaNNaN519P170666825.653725.7283NaNNaN519P257886925.998626.0869NaNNaN619Q096667025.151224.9405NaNNaN319Q96CX27125.788525.9526NaNNaN418P207007226.050426.279NaNNaN618P217967324.038524.1862NaNNaN918Q138137423.628224.3248NaNNaN417P105997525.471725.2458NaNNaN417P206187625.533125.5971NaNNaN417P300417724.975124.827NaNNaN516P053877825.695225.6926NaNNaN516P273487925.129425.1703NaNNaN516P300508024.522824.6406NaNNaN315P072378125.582525.4838NaNNaN615P079008224.905924.9505NaNNaN415P311468324.742824.6945NaNNaN415P619788425.413425.4558NaNNaN515Q049178525.202125.3364NaNNaN314P067488626.551626.5969NaNNaN214P093828724.499324.392NaNNaN414P176618824.974124.9066NaNNaN514P680368925.681825.5923NaNNaN414Q015189024.179224.2282NaNNaN413O002999124.118623.6956NaNNaN213P074379225.253525.1577NaNNaN513P257869326.289424.4252NaNNaN313P311509425.627525.6403NaNNaN313P840909525.193925.0022NaNNaN212P497209626.333826.1736NaNNaN212P629429724.510624.4904NaNNaN312P629939824.539124.3048NaNNaN712Q006109925.135625.441NaNNaN312Q0146910024.40924.623NaNNaN212Q9268810123.591323.7504NaNNaN411P0049210224.728724.5849NaNNaN511P2570510325.326225.4438NaNNaN311P3008610425.768125.8786NaNNaN211P6331310524.566524.8052NaNNaN310Q96KP410624.062323.8665NaNNaN59P0039010723.20123.1414NaNNaN29P3004410823.760523.5635NaNNaN29P4012110925.579725.7457NaNNaN59P4588011024.604224.509NaNNaN49P5999811124.865824.5847NaNNaN29Q9949711224.146924.2116NaNNaN28P0016711323.000522.9948NaNNaN48P0719511423.270122.9133NaNNaN28P0733911527.199627.0812NaNNaN28P0760211624.401124.3697NaNNaN28P2921811723.431623.0263NaNNaN48P4694011823.698623.7716NaNNaN48P6301011924.509125.1422NaNNaN38Q0855412023.157923.2595NaNNaN57P0538812123.264123.1863NaNNaN27P6108812223.024322.6641NaNNaN26P0279012323.852423.8998NaNNaN36P2352612422.640422.507NaNNaN26P6115812523.68424.0302NaNNaN26Q1584312623.131623.1095NaNNaN35O1514412722.961322.9744NaNNaN35P0673712822.831322.6406NaNNaN35P1512112923.51123.6221NaNNaN25Q9NZT1

Analysis of ARI D3C Interactors by Coimmunoprecipitation and
Mass Spectrometry

To identify the interaction partners of
ARID3C, we transiently expressed a Halo-tagged ARID3C protein in 293T
cells and subsequently purified the tagged protein along with its
interactors (Figure 1A and S1A). As a negative control for
protein–protein interactions, we used protein extracts from
cells expressing the free HaloTag (Figure 1B). Protein identifications from the mass
spectrometry were filtered using the criteria of the HUPO guidelines,
then proteins commonly identified in the HaloTag control were removed.
Moreover, false-positive proteins with predicted cellular localization
differing from ARID3C were also excluded.

Interactome analysis
of ARID3C. (A) Strategy to identify interaction
partners of ARID3C. HEK293T cells used for ARID3C overexpression and
the purified ARID3C complex by HaloTag purification were digested
into peptides using the FASP digestion method for LC-MS/MS. (B) Schematic
illustration of the ARIC3C interactome analysis. To identify the ARID3C
binding partner, we filtered out proteins that were expected to be
false positive under 3 step conditions. (C) Venn diagram showing the
distribution of the total 158 candidates selected. (D) Distribution
of the subcellular localization of selected 129 candidates. (E) Protein–protein
interaction (PPI) network of the proteins specific to ARID3C interactome
and predicted to be present with the nucleus. (F) Representative images
showing the subcellular localization of ARID3C and NPM1. Double immunofluorescence
staining shows colocalization between ARID3C and NPM1 in the nucleus.
Scale bar: 10 μm. Figure 1A,B created with BioRender.com.

The IP-MS analysis identified a total of 158 proteins,
with a FDR
calculation below 1% at the global protein level. One hundred twenty-nine
proteins were unique to ARID3C, not overlapping with the HaloTag control
(Figure 1C and Table 1). We detected 20
unique peptides of ARID3C, and the MS spectrum is shown in Figure S2A. We conducted PPI analysis using the
STRING database, dividing proteins into two groups: all 129 proteins
(Figure S3A) and nuclear-localized proteins
(Figure 1E). When analyzing
the PPIs of the 129 proteins associated with ARID3C, we identified
five main clusters: Type III intermediate filament, Proteasome core
complex, Dendritic growth core, Muscle thin filament tropomyosin,
and Acetyl-CoA carboxylase complex. Notably, nuclear proteins exhibited
more specific clustering, particularly involving the lamin filament
and actin filament. We focused on proteins that constitute the lamin
filament, aiming to better elucidate the role of ARID3C as a transcription
factor. Among these lamin filament proteins (NPM1, RPLP0, LMNA, LMNB1,
and HNRNPA2B1), we emphasized NPM1, which occupies the most nodes.
Furthermore, ARID3C and NPM1 are colocalized in the nucleus in the
293T cells (Figure 1E). Thus, in this study, we focused on the interaction between ARID3C
and NPM1.

List of 129 Proteins Identified Only
in ARID3C

ARID3C Interacts with NPM1 for Nuclear ShuttlingNext,
we employed AlphaFold2 to predict the 3D structures of the ARID3C
and NPM1 protein (Figure S4A,B) and predicted
its interaction with NPM1 (Figure 2A). Based on the report that NPM1 is involved in nuclear
shuttling,21 we hypothesized that ARID3C
may interact with NPM1 to enter the nucleus. Three-dimensional predictions
by AlphaFold2 unveiled that NPM1 interacts with two sites: the ARID
domain (left) and the REKLES β domain (right) of ARID3C. As
the ARID domain is known to be involved in DNA binding,9 we focused on the REKLES β domain. Four
amino acids, V381, F383, A384, and R385, in the REKLES β domain
of ARID3C are predicted to be involved in the binding to NPM1 (Figure 2B). To explore whether
ARID3C indeed binds to NPM1, we constructed two different mutations
(Figure 2C). The first
mutation deleted the whole REKLES β domain including the four
amino acids (ΔREK), while the second mutation substituted those
four amino acids with different amino acids: V381D, F383D, A384W,
and R385G (Mut). We transiently expressed ARID3C, ΔREK, and
Mut with the NPM1-flag in 293T cells. After IP using the HaloTag of
ARID3C and performing immunoblotting with the Flag antibody, we found
that ARID3C bound to NPM1, while ΔREK did not bind at all. The
Mut showed limited binding to NPM1. Herein, we have demonstrated that
ARID3C interacts with NPM1 and that the specific four amino acids
in the REKLES β domain play a pivotal role in this interaction
(Figure 2D).Figure 2ARID3C interacts
with NPM1 for nuclear shuttling. (A) Predicted
3D structure of ARID3C and NPM1 interaction complex revealed by AlphaFold2.
Blue, ARID3c; Gray, NPM1. Residue–residue alignment confidence
(PAE) contacts were labeled with dark blue. Magnified left image,
binding site between ARID domain of ARID3C and NPM1; right image,
REKLES β domain of ARID3C and NPM1. (B) Structure of REKLES
β domain amino acids of ARID3C and the corresponding amino acids
of NPM1. (C) Structure of the deletion and amino acid substitution
mutated in REKES β domain. (D) Coimmunoprecipitation assay with
ARID3C, ΔREK, and Mut with Halo antibody verified the interaction
between ARID3C and NPM1. (E) Representative images showing the subcellular
localization of mutated ARID3C and NPM1. Scale bar: 10 μm. Figure 2B created with BioRender.com.Next, we investigated the subcellular localization
of mutated ARID3C
and NPM1. We found that both ARID3C and NPM1 exist in the nucleus
of cells, while the deleted ARID3C (ΔREK) is retained in the
cytoplasm (Figure 2E). The substitution form of ARID3C (Mut) was found to have both
nuclear and cytoplasmic localization, which is reminiscent of the
IP-IB result (Figure 2D) with some binding to NPM1 and some not. These observations highlight
that the translocation of ARID3C from the cytoplasm to the nucleus
is contingent upon its interaction with the REKLES β domain
of NPM1, mainly those four amino acids V381, F383, A384, and R385.

ARID3C Interacts with NPM1 for Nuclear Shuttling

Next,
we employed AlphaFold2 to predict the 3D structures of the ARID3C
and NPM1 protein (Figure S4A,B) and predicted
its interaction with NPM1 (Figure 2A). Based on the report that NPM1 is involved in nuclear
shuttling,21 we hypothesized that ARID3C
may interact with NPM1 to enter the nucleus. Three-dimensional predictions
by AlphaFold2 unveiled that NPM1 interacts with two sites: the ARID
domain (left) and the REKLES β domain (right) of ARID3C. As
the ARID domain is known to be involved in DNA binding,9 we focused on the REKLES β domain. Four
amino acids, V381, F383, A384, and R385, in the REKLES β domain
of ARID3C are predicted to be involved in the binding to NPM1 (Figure 2B). To explore whether
ARID3C indeed binds to NPM1, we constructed two different mutations
(Figure 2C). The first
mutation deleted the whole REKLES β domain including the four
amino acids (ΔREK), while the second mutation substituted those
four amino acids with different amino acids: V381D, F383D, A384W,
and R385G (Mut). We transiently expressed ARID3C, ΔREK, and
Mut with the NPM1-flag in 293T cells. After IP using the HaloTag of
ARID3C and performing immunoblotting with the Flag antibody, we found
that ARID3C bound to NPM1, while ΔREK did not bind at all. The
Mut showed limited binding to NPM1. Herein, we have demonstrated that
ARID3C interacts with NPM1 and that the specific four amino acids
in the REKLES β domain play a pivotal role in this interaction
(Figure 2D).

ARID3C interacts
with NPM1 for nuclear shuttling. (A) Predicted
3D structure of ARID3C and NPM1 interaction complex revealed by AlphaFold2.
Blue, ARID3c; Gray, NPM1. Residue–residue alignment confidence
(PAE) contacts were labeled with dark blue. Magnified left image,
binding site between ARID domain of ARID3C and NPM1; right image,
REKLES β domain of ARID3C and NPM1. (B) Structure of REKLES
β domain amino acids of ARID3C and the corresponding amino acids
of NPM1. (C) Structure of the deletion and amino acid substitution
mutated in REKES β domain. (D) Coimmunoprecipitation assay with
ARID3C, ΔREK, and Mut with Halo antibody verified the interaction
between ARID3C and NPM1. (E) Representative images showing the subcellular
localization of mutated ARID3C and NPM1. Scale bar: 10 μm. Figure 2B created with BioRender.com.

Next, we investigated the subcellular localization
of mutated ARID3C
and NPM1. We found that both ARID3C and NPM1 exist in the nucleus
of cells, while the deleted ARID3C (ΔREK) is retained in the
cytoplasm (Figure 2E). The substitution form of ARID3C (Mut) was found to have both
nuclear and cytoplasmic localization, which is reminiscent of the
IP-IB result (Figure 2D) with some binding to NPM1 and some not. These observations highlight
that the translocation of ARID3C from the cytoplasm to the nucleus
is contingent upon its interaction with the REKLES β domain
of NPM1, mainly those four amino acids V381, F383, A384, and R385.

ARID3C Promotes PMA-Induced Myeloid DifferentiationTo investigate the role of ARID3C in monocyte differentiation, we
used AML patient-derived monocyte cell lines HL-60 and THP-1. HL-60
and THP-1 were treated with PMA to induce monocyte-to-macrophage differentiation
(Figure 3A). A significant
increase was observed in the expression of macrophage differentiation
markers, CD11b and CSF1R (Figure 3B). Next, we also confirmed an increase in the level
of protein expression of CD11b upon PMA-induced differentiation in
both cell types (Figure 3C). Flow cytometric analysis also showed that the number of CD11b-positive
cells increased with PMA treatment (Figure 3D).Figure 3ARID3C induce monocyte-to-macrophage differentiation.
(A) Graphical
representation of monocyte differentiation into macrophages induced
by phorbol-12-myristate-13-acetate (PMA). (B) Relative mRNA expression
of CD11b and CSF1R for monocyte-to-macrophage differentiation was
quantified by qRT-PCR. The results were analyzed using 2−ΔΔCT
and GAPDH as a reference gene. (C) Protein expression level of CD11b
quantified by Western Blot. β-Actin was used as an internal
control. (D) Representative flow cytometry of CD11b of monocyte and
PMA treated monocytes. (E) Halo and ARID3C overexpressed cells were
used for Western Blotting. (F,G) Relative mRNA expression of CD11b,
CD14, and CSF1R was quantified by qRT-PCR. HL-60 (F) and THP-1 (G)
cells were transfected with ARID3C and treated with PMA for 24 h.
The results were analyzed using 2−ΔΔCT and used
GAPDH as a reference gene. All data are presented as mean ± SEM
(n = 3). The Student’s t test
was used to analyze the data, and statistical analysis was performed.
**P < 0.01; n.s., not significant. Figure 3A created with BioRender.com.Subsequently, we conducted transient transfection
of HL-60 and
THP-1 with either the ARID3C or Halo control construct. Transfection
efficiency was confirmed by a Western blot (Figure 3E). After ARID3C transfection, the cells
were treated with PMA for the macrophage differentiation. Intriguingly,
the expression of ARID3C led to a further augmentation in the levels
of CD11b, CD14, and CSF1R mRNA when compared to the control (Figure 3F,G). These findings
collectively indicate that ARID3C experts a positive regulatory effect
on the monocyte-to-macrophage differentiation.

ARID3C Promotes PMA-Induced Myeloid Differentiation

To investigate the role of ARID3C in monocyte differentiation, we
used AML patient-derived monocyte cell lines HL-60 and THP-1. HL-60
and THP-1 were treated with PMA to induce monocyte-to-macrophage differentiation
(Figure 3A). A significant
increase was observed in the expression of macrophage differentiation
markers, CD11b and CSF1R (Figure 3B). Next, we also confirmed an increase in the level
of protein expression of CD11b upon PMA-induced differentiation in
both cell types (Figure 3C). Flow cytometric analysis also showed that the number of CD11b-positive
cells increased with PMA treatment (Figure 3D).

ARID3C induce monocyte-to-macrophage differentiation.
(A) Graphical
representation of monocyte differentiation into macrophages induced
by phorbol-12-myristate-13-acetate (PMA). (B) Relative mRNA expression
of CD11b and CSF1R for monocyte-to-macrophage differentiation was
quantified by qRT-PCR. The results were analyzed using 2−ΔΔCT
and GAPDH as a reference gene. (C) Protein expression level of CD11b
quantified by Western Blot. β-Actin was used as an internal
control. (D) Representative flow cytometry of CD11b of monocyte and
PMA treated monocytes. (E) Halo and ARID3C overexpressed cells were
used for Western Blotting. (F,G) Relative mRNA expression of CD11b,
CD14, and CSF1R was quantified by qRT-PCR. HL-60 (F) and THP-1 (G)
cells were transfected with ARID3C and treated with PMA for 24 h.
The results were analyzed using 2−ΔΔCT and used
GAPDH as a reference gene. All data are presented as mean ± SEM
(n = 3). The Student’s t test
was used to analyze the data, and statistical analysis was performed.
**P < 0.01; n.s., not significant. Figure 3A created with BioRender.com.

Subsequently, we conducted transient transfection
of HL-60 and
THP-1 with either the ARID3C or Halo control construct. Transfection
efficiency was confirmed by a Western blot (Figure 3E). After ARID3C transfection, the cells
were treated with PMA for the macrophage differentiation. Intriguingly,
the expression of ARID3C led to a further augmentation in the levels
of CD11b, CD14, and CSF1R mRNA when compared to the control (Figure 3F,G). These findings
collectively indicate that ARID3C experts a positive regulatory effect
on the monocyte-to-macrophage differentiation.

As a Transcription Factor, ARID3C Directly Regulates Genes Involved
in Monocyte-to-Macrophage DifferentiationAfter induction
for 24 h with PMA, HL-60 cells that were transfected with ARID3C exhibited
a higher proportion of mature monocytes (Figure 4A). We investigated whether ARID3C binds
to NPM1 and enters to the nucleus to act as a transcription factor.
To begin, we conducted a screening for master transcription factor
genes that orchestrate the process of monocyte-to-macrophage differentiation.22 Subsequent to this screening, we identified
three genes (STAT3, STAT1, and JUNB) that possess an ARID-binding
DNA element within their respective promoters (Figure 4B). Notably, STAT3 is required for monocyte
migration and differentiation,23 while
STAT1 promotes macrophage activation and the production of inflammatory
cytokines.24 Moreover, JUNB is known to
participate in various cellular processes, including immune regulation
and differentiation.25 Together, these
transcription factors contribute to the differentiation of monocytes
into functional macrophages. Based on this information, we infer that
ARID3C acts as a transcription factor that regulates the expression
of three genes, which results in an increase in monocyte-to-macrophage
differentiation. Next, we performed chromatin immunoprecipitation
by the Halo tag followed by PCR (ChIP-PCR). Additionally, we hypothesized
that ARID3C would not function as a transcription factor if it failed
to bind with NPM1 and was retained in the cytoplasm. To validate this
hypothesis, we transfected both the original ARID3C and a mutated
form of ARID3C, and observed their binding to the promoters of STAT3,
STAT1, and JUNB (Figure 4C–E). The results showed that ARID3C bound to the promoters.
We observed that ARID3C binds to the promoters of STAT3, STAT1, and
JUNB, but the ΔREK failed to bind and Mut bound at a very limited
level, indicating that ARID3C requires an intact NPM1-binding site
to act as a transcription factor in differentiation (Figure 4C–E). In Figure 4F, we also detected failure
of binding capacity in mutated ARID3C by ChIP-qPCR. Furthermore, we
confirmed failure of intact between ARID3C and NPM1 leads to decrease
of differentiation by using flow cytometry to measure the level of
CD11b marker in cells transfected with the different ARID3C constructs
(Figure 4G). Upon transfection
of ARID3C in HL-60 cells, the highest expression of CD11b was observed,
whereas the deletion of the REKLES domain in ΔREK resulted in
the lowest CD11b expression.Figure 4As a transcription factor, ARID3C directly regulates
the genes
related to macrophage differentiation. (A) Phase contrast microscopy
shows that PMA induced HL-60 cells. ARID3C overexpressed HL-60 cell
shows morphological characteristics of macrophages. Scale bar: 250
μm. (B) Graphical diagram for selection of transcription factors
with an ARID domain related to macrophage differentiation. (C–E)
Chromatin immunoprecipitation (ChIP) analysis of STAT3, STAT1, and
JUNB in the 293T cells transfected with ARID3C, ΔREK, and Mut.
Genome structures of STAT3, STAT1, and JUNB are shown. Blue histogram
represents the predicted ARID3C binding domain, and the inner black
bar indicates the amplified region by PCR. (F) DNA fold enrichments
were quantified by ChIP-qPCR. (G) Expression levels of CD11b in Halo,
ARID3C, ΔREK, and Mut transfected HL-60 cells were analyzed
by FACS (red, Halo; blue, ARID3C; orange, ΔREK; green, Mut).
All data are presented as mean ± SEM (n = 3).
The statistical analysis is presented. Two-way ANOVA was used to compare
groups. # p < 0.01, * p <
0.05, and a, not significant.These findings indicate that ARID3C promotes macrophage
differentiation
by binding to NPM1 and translocating into the nucleus to act as a
transcription factor. Overall, we showed that ARID3C promotes monocyte
differentiation, but mutated ARID3C reduced differentiation ability.
Accordingly, ARID3C induces monocyte-to-macrophage differentiation
as a transcription factor by regulating the expression of STAT3, STAT1,
and JUNB.

As a Transcription Factor, ARID3C Directly Regulates Genes Involved
in Monocyte-to-Macrophage Differentiation

After induction
for 24 h with PMA, HL-60 cells that were transfected with ARID3C exhibited
a higher proportion of mature monocytes (Figure 4A). We investigated whether ARID3C binds
to NPM1 and enters to the nucleus to act as a transcription factor.
To begin, we conducted a screening for master transcription factor
genes that orchestrate the process of monocyte-to-macrophage differentiation.22 Subsequent to this screening, we identified
three genes (STAT3, STAT1, and JUNB) that possess an ARID-binding
DNA element within their respective promoters (Figure 4B). Notably, STAT3 is required for monocyte
migration and differentiation,23 while
STAT1 promotes macrophage activation and the production of inflammatory
cytokines.24 Moreover, JUNB is known to
participate in various cellular processes, including immune regulation
and differentiation.25 Together, these
transcription factors contribute to the differentiation of monocytes
into functional macrophages. Based on this information, we infer that
ARID3C acts as a transcription factor that regulates the expression
of three genes, which results in an increase in monocyte-to-macrophage
differentiation. Next, we performed chromatin immunoprecipitation
by the Halo tag followed by PCR (ChIP-PCR). Additionally, we hypothesized
that ARID3C would not function as a transcription factor if it failed
to bind with NPM1 and was retained in the cytoplasm. To validate this
hypothesis, we transfected both the original ARID3C and a mutated
form of ARID3C, and observed their binding to the promoters of STAT3,
STAT1, and JUNB (Figure 4C–E). The results showed that ARID3C bound to the promoters.
We observed that ARID3C binds to the promoters of STAT3, STAT1, and
JUNB, but the ΔREK failed to bind and Mut bound at a very limited
level, indicating that ARID3C requires an intact NPM1-binding site
to act as a transcription factor in differentiation (Figure 4C–E). In Figure 4F, we also detected failure
of binding capacity in mutated ARID3C by ChIP-qPCR. Furthermore, we
confirmed failure of intact between ARID3C and NPM1 leads to decrease
of differentiation by using flow cytometry to measure the level of
CD11b marker in cells transfected with the different ARID3C constructs
(Figure 4G). Upon transfection
of ARID3C in HL-60 cells, the highest expression of CD11b was observed,
whereas the deletion of the REKLES domain in ΔREK resulted in
the lowest CD11b expression.

As a transcription factor, ARID3C directly regulates
the genes
related to macrophage differentiation. (A) Phase contrast microscopy
shows that PMA induced HL-60 cells. ARID3C overexpressed HL-60 cell
shows morphological characteristics of macrophages. Scale bar: 250
μm. (B) Graphical diagram for selection of transcription factors
with an ARID domain related to macrophage differentiation. (C–E)
Chromatin immunoprecipitation (ChIP) analysis of STAT3, STAT1, and
JUNB in the 293T cells transfected with ARID3C, ΔREK, and Mut.
Genome structures of STAT3, STAT1, and JUNB are shown. Blue histogram
represents the predicted ARID3C binding domain, and the inner black
bar indicates the amplified region by PCR. (F) DNA fold enrichments
were quantified by ChIP-qPCR. (G) Expression levels of CD11b in Halo,
ARID3C, ΔREK, and Mut transfected HL-60 cells were analyzed
by FACS (red, Halo; blue, ARID3C; orange, ΔREK; green, Mut).
All data are presented as mean ± SEM (n = 3).
The statistical analysis is presented. Two-way ANOVA was used to compare
groups. # p < 0.01, * p <
0.05, and a, not significant.

These findings indicate that ARID3C promotes macrophage
differentiation
by binding to NPM1 and translocating into the nucleus to act as a
transcription factor. Overall, we showed that ARID3C promotes monocyte
differentiation, but mutated ARID3C reduced differentiation ability.
Accordingly, ARID3C induces monocyte-to-macrophage differentiation
as a transcription factor by regulating the expression of STAT3, STAT1,
and JUNB.

DiscussionThe number of PE1 proteins has increased
from 15,649 to 18,397
in the past ten years, a remarkable achievement of the Chromosome-based
Human Proteome Project (C-HPP) project.26,27 However, 1,521
(as of April 18th, 2023 released in neXtProt) PE1 proteins still lack
functional annotation, and they are considered to be in the “dark
field” of proteomics research. Here, we not only provided chromosome
9 located protein evidence but also utilized MS plus deep learning
to identify binding motifs of the interactors discovered through IP-MS.
Furthermore, we experimentally validated ARID3C-NPM1 interactions
and reported on the cellular functions during macrophage differentiation.Despite lacking NLS and NES peptide sequences, ARID3C can shuttle
between the nucleus and cytoplasm by binding to NPM1, which we discovered
for the first time. NPM1 is known to be mutated in one-third of adult
acute myeloid leukemia (AML) patients.28 The most common mutation is an insertion of “TCGT”,
which induces a frameshift in the C-terminal and leads to the addition
of one more nuclear export signal (NES). These NPM1 mutation (mtNPM1)
results in being localized in the cytosol rather than the nucleus.29 Cytoplasmic delocalization of mtNPM1 induces
the cytoplasmic retention of several nuclear proteins. In AML patient-derived
cells, NPM1 forms a complex with master transcription factors of monocyte
lineage differentiation, such as PU.1 (also known as SPI1), CEBP1,
and RUNX1, to regulate granulocytic differentiation.30 Therefore, mislocalized mtNPM1 in AML cells prevents monocyte
and granulocyte terminal differentiation by cytoplasmic retention
of PU.1, leading to the disruption of the formation of the PU.1/RUNX1/CEBPA
master transcription factor complex.30 Additionally,
the CCCTC-binding factor (CTCF) has been reported to interact with
both NPM1 and mtNPM1, but the interaction with mtNPM1 also results
in the mislocalization of CTCF.31 Moreover,
mtNPM1 also contributes to the mislocalization of several nuclear
proteins such as MIZ1, Fbwγ, and HEXIM1. Likewise, ARID3C has
two regions that can bind to NPM1. Inferring from the high prevalence
of NPM1 mutations in AML patients, it is reasonable to anticipate
that these mutations may also affect the subcellular mislocalization
of ARID3C. Further investigation into the potential effects of NPM1
mutations on ARID3C’s subcellular localization and its functional
implications in AML pathogenesis would be valuable to understand the
complex interplay between these proteins in cancer development.Furthermore, while our focus was on the interaction partners involved
in the nuclear import of ARID3C, IP-MS data suggest the existence
of additional potential interacting partners that could bind to ARID3C.
Further research is needed to investigate proteins that could potentially
form complexes with ARID3C as a transcription factor. Additionally,
our study has primarily focused on ARID3C expression in 293T cells
and monocytes. Therefore, studies in the cells with higher baseline
expression levels of ARID3C, such as in the liver or skin, will reveal
further function in the future.Previously, mass spectrometry
(MS) could identify protein interactors
but it was challenging to pinpoint the exact binding sites. In this
study, we utilized deep learning to predict the interaction between
ARID3C and NPM1, thereby revealing the precise binding sites, REKLES
β domain with 4 specific 4 amino acids. Thus, by integrating
deep learning with the experimental data, we were able to uncover
a function of ARID3C. We also characterized the function of ARID3C
in monocyte-to-macrophage differentiation and validated its role as
a transcription factor using NPM1 as a nuclear shuttle. ARID3C bound
to NPM1 leads to nucleus translocation and functions as a transcription
factor regulating the expression of master transcription factors involved
in monocyte to macrophage monocytic differentiation, including STAT3,
STAT1, and JUNB.

The number of PE1 proteins has increased
from 15,649 to 18,397
in the past ten years, a remarkable achievement of the Chromosome-based
Human Proteome Project (C-HPP) project.26,27 However, 1,521
(as of April 18th, 2023 released in neXtProt) PE1 proteins still lack
functional annotation, and they are considered to be in the “dark
field” of proteomics research. Here, we not only provided chromosome
9 located protein evidence but also utilized MS plus deep learning
to identify binding motifs of the interactors discovered through IP-MS.
Furthermore, we experimentally validated ARID3C-NPM1 interactions
and reported on the cellular functions during macrophage differentiation.

Despite lacking NLS and NES peptide sequences, ARID3C can shuttle
between the nucleus and cytoplasm by binding to NPM1, which we discovered
for the first time. NPM1 is known to be mutated in one-third of adult
acute myeloid leukemia (AML) patients.28 The most common mutation is an insertion of “TCGT”,
which induces a frameshift in the C-terminal and leads to the addition
of one more nuclear export signal (NES). These NPM1 mutation (mtNPM1)
results in being localized in the cytosol rather than the nucleus.29 Cytoplasmic delocalization of mtNPM1 induces
the cytoplasmic retention of several nuclear proteins. In AML patient-derived
cells, NPM1 forms a complex with master transcription factors of monocyte
lineage differentiation, such as PU.1 (also known as SPI1), CEBP1,
and RUNX1, to regulate granulocytic differentiation.30 Therefore, mislocalized mtNPM1 in AML cells prevents monocyte
and granulocyte terminal differentiation by cytoplasmic retention
of PU.1, leading to the disruption of the formation of the PU.1/RUNX1/CEBPA
master transcription factor complex.30 Additionally,
the CCCTC-binding factor (CTCF) has been reported to interact with
both NPM1 and mtNPM1, but the interaction with mtNPM1 also results
in the mislocalization of CTCF.31 Moreover,
mtNPM1 also contributes to the mislocalization of several nuclear
proteins such as MIZ1, Fbwγ, and HEXIM1. Likewise, ARID3C has
two regions that can bind to NPM1. Inferring from the high prevalence
of NPM1 mutations in AML patients, it is reasonable to anticipate
that these mutations may also affect the subcellular mislocalization
of ARID3C. Further investigation into the potential effects of NPM1
mutations on ARID3C’s subcellular localization and its functional
implications in AML pathogenesis would be valuable to understand the
complex interplay between these proteins in cancer development.

Furthermore, while our focus was on the interaction partners involved
in the nuclear import of ARID3C, IP-MS data suggest the existence
of additional potential interacting partners that could bind to ARID3C.
Further research is needed to investigate proteins that could potentially
form complexes with ARID3C as a transcription factor. Additionally,
our study has primarily focused on ARID3C expression in 293T cells
and monocytes. Therefore, studies in the cells with higher baseline
expression levels of ARID3C, such as in the liver or skin, will reveal
further function in the future.

Previously, mass spectrometry
(MS) could identify protein interactors
but it was challenging to pinpoint the exact binding sites. In this
study, we utilized deep learning to predict the interaction between
ARID3C and NPM1, thereby revealing the precise binding sites, REKLES
β domain with 4 specific 4 amino acids. Thus, by integrating
deep learning with the experimental data, we were able to uncover
a function of ARID3C. We also characterized the function of ARID3C
in monocyte-to-macrophage differentiation and validated its role as
a transcription factor using NPM1 as a nuclear shuttle. ARID3C bound
to NPM1 leads to nucleus translocation and functions as a transcription
factor regulating the expression of master transcription factors involved
in monocyte to macrophage monocytic differentiation, including STAT3,
STAT1, and JUNB.
