---
pmid: '32482714'
title: Progress toward understanding chromosome silencing by Xist RNA.
authors:
- Brockdorff N
- Bowness JS
- Wei G
journal: Genes Dev
year: '2020'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC7263139
doi: 10.1101/gad.337196.120
---

# Progress toward understanding chromosome silencing by Xist RNA.
**Authors:** Brockdorff N, Bowness JS, Wei G
**Journal:** Genes Dev (2020)
**DOI:** [10.1101/gad.337196.120](https://doi.org/10.1101/gad.337196.120)
**PMC:** [PMC7263139](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7263139/)

## Abstract

1. Genes Dev. 2020 Jun 1;34(11-12):733-744. doi: 10.1101/gad.337196.120.

Progress toward understanding chromosome silencing by Xist RNA.

Brockdorff N(1), Bowness JS(1), Wei G(1).

Author information:
(1)Department of Biochemistry, University of Oxford, Oxford OX1 3QU, United 
Kingdom.

The X inactive-specific transcript (Xist) gene is the master regulator of X 
chromosome inactivation in mammals. Xist produces a long noncoding (lnc)RNA that 
accumulates over the entire length of the chromosome from which it is 
transcribed, recruiting factors to modify underlying chromatin and silence 
X-linked genes in cis Recent years have seen significant progress in identifying 
important functional elements in Xist RNA, their associated RNA-binding proteins 
(RBPs), and the downstream pathways for chromatin modification and gene 
silencing. In this review, we summarize progress in understanding both how these 
pathways function in Xist-mediated silencing and the complex interplay between 
them.

© 2020 Brockdorff et al.; Published by Cold Spring Harbor Laboratory Press.

DOI: 10.1101/gad.337196.120
PMCID: PMC7263139
PMID: 32482714 [Indexed for MEDLINE]

## Full Text

X chromosome inactivation, the process that evolved in mammals to balance levels of X-linked gene expression in XX females relative to XY males, is controlled by a master regulator locus, Xist (X inactive-specific transcript), that is located on the X chromosome and functions in cis (for review, see Heard et al. 1997). Xist produces a 15- to 17-kb-long noncoding RNA (lncRNA) that accumulates over the chromosome from which it is transcribed, recruiting factors/complexes that act to modify the underlying chromatin environment and repress X-linked gene expression. Gene knockout and transgenic experiments demonstrated that Xist is both necessary and sufficient for chromosome inactivation (Penny et al. 1996; Lee and Jaenisch 1997).

Studies over many years have defined chromatin features, chromatin-associated factors, and higher-order chromosome organization associated with the inactive X chromosome (Xi) (Fig. 1; Table 1). An important challenge for the field is to understand cause and effect. Which features of Xi are required to establish gene silencing and depend on factors recruited directly by Xist RNA and which are secondary but possibly still important for long-term chromosome silencing through development and into adulthood? To address this issue, studies on early mouse embryos and using mouse embryonic stem cell (mESC) models in vitro defined Xi features that are established early following the onset of Xist RNA expression, interpreted as indicating a role in establishment of X inactivation. Important examples include exclusion of RNA Polymerase II (RNAPII), loss of histone modifications associated with gene activity, and acquisition of histone modifications catalyzed by the Polycomb complexes PRC1 and PRC2. Conversely, Xi features that are established after a delay following Xist RNA expression, for example, CpG island DNA methylation and deposition of the histone variant macroH2A, are thought to function in maintenance rather than establishment of silencing.

Features of the inactive X chromosome. (A) Comparison of chromatin features on the active X chromosome (Xa) and the inactive X chromosome (Xi) at the nucleosomal scale. Xi is characterized by depletion of histone modifications associated with gene activity, shown here histone acetylation, normally enriched at promoters and enhancers, H3K4me3, enriched at promoters and H3K36me3, high levels of which occur within the bodies of active genes. Additionally, Xi chromatin has enhanced levels of histone modifications associated with gene repression; for example, H3K27me3, H2AK119ub1, H3K9me2/3, and high levels of the histone variant macroH2A. DNA methylation is acquired at CpG islands of Xi genes. RNA polymerase II (RNAPII) and associated general transcription factors (GTFs) are depleted from Xi, as are enhancer-bound transcription factors (TFs), as evidenced by reduced chromatin accessibility seen using ATAC-seq. Similarly, there are reduced levels of the insulator protein CTCF and the cohesin complex at insulator/boundary elements. (B) Changes in higher order structure of the Xi chromosome. Key examples are an approximately twofold chromatin compaction relative to Xa, association of Xi with the nuclear and/or nucleolar periphery, reduced TAD and compartment structure (related to reduced CTCF/cohesin binding), with interactions instead occurring within one of two large megadomains, and synchronous replication across most of the chromosome, usually in late S phase.

Chromatin and higher-order chromosome features of the inactive X chromosome

Significant progress toward defining the critical pathways for establishment of gene silencing has come from the identification of functional sequence elements in Xist RNA and the RNA-binding proteins (RBPs) that they bind. Thus, deletion analysis defined the A repeat, one of a number of conserved tandem repeat blocks (repeated X7.5 in mice and X8 in humans) present in Xist RNA sequence (Fig. 2A), as being critical for silencing, while multiple regions of the transcript were defined as functioning redundantly to mediate local accumulation of Xist RNA (Wutz et al. 2002). An important breakthrough toward identifying the factors that interact with the A repeat (and other regions of Xist RNA), came from complementary studies that applied either proteomic or genetic screening strategies (Chu et al. 2015; McHugh et al. 2015; Minajigi et al. 2015; Moindrot et al. 2015; Monfort et al. 2015). Leading candidates that emerged from this work are summarized in Figure 2A,B. The RBP SPEN, which functions through interaction with the histone deacetylase complex NCoR-HDAC3 (and possibly other factors), was shown to bind Xist A repeat directly (Monfort et al. 2015). The closely related RBP RBM15 also binds the A-repeat, and functions at least in part by recruiting the METTL3/14 complex responsible for catalyzing N6-methyladenosine (m6A) on mRNA (Patil et al. 2016). Additionally, the Lamin B receptor (LBR), was identified as binding Xist RNA (McHugh et al. 2015). Although not a known RBP, subsequent work reported that LBR has an unstructured SR domain that mediates interaction with Xist RNA (Chen et al. 2016). Finally the RBP hnRNPK was identified as an abundant Xist-binding protein (Chu et al. 2015), with subsequent studies revealing that it recognizes the Xist B/C-repeat region and directs recruitment of PRC1 and PRC2 Polycomb complexes to Xi (Cooper et al. 2016; Almeida et al. 2017; Pintacuda et al. 2017). It should be noted that the latter studies overturned the prior model for Polycomb recruitment to Xi, which invoked PRC2 interaction with Xist A repeat as the initiating event (Zhao et al. 2008). In this review, we discuss progress toward determining the role of these newly defined factors in establishing features of Xi chromatin and in gene silencing by Xist RNA, including a summary of our understanding of their relative contribution and interplay.

Xist RNA-binding proteins (RBPs) with a role in chromosome silencing. (A) Binding profiles of key RBPs discussed in this article. iCLIP/eCLIP data obtained from various sources (Chen et al. 2016; Lu et al. 2016; Cirillo et al. 2017; Nesterova et al. 2019) is shown against a map of the Xist gene with tandem repeat elements A–F indicated below. A-repeat and B/C-repeat regions are highlighted with pale-blue shading. The m6A track illustrates sites of m6A deposition as determined by m6A-seq (Coker et al. 2020). Gray bar indicates the location of LBS on Xist. (B) Features of the RBPs SPEN, RBM15, LBR, and hnRNPK. Schematics indicate domain architecture with key interacting partners shown at the right. Validated direct interactors are shown in black. (RRM) RNA recognition motif; (RID) RBPJ interaction domain; (SPOC) Spen paralog and ortholog C-terminal; (RS) arginine/serine; (PEMT) phosphatidylethanolamine N-methyltransferase; (KH) K homology; (KI) K interaction.

Assaying Xist-mediated chromosome silencingIn considering recent advances in this field it is important to appreciate the varied models and assays that have been used in analyzing the contribution of different factors to Xist-mediated chromosome silencing, not least because reported findings may otherwise appear contradictory. As summarized in Figure 3A, the principal models that provide access to the developmental stage when X inactivation is established are early mouse embryos and pluripotent embryonic stem cells (mESCs). The latter include XX mESCs, which express Xist RNA from a single X chromosome when differentiated in vitro, XY mESCs with Xist expression driven by a doxycycline-activated promoter, either from the single X chromosome allele, or from an autosomally integrated Xist transgene, and XX mESCs with a doxycycline inducible promoter engineered into one of the two Xist alleles. Models using inducible promoters offer the advantage of regulatable, synchronous, and homogeneous Xist RNA expression within cell populations, albeit at levels not necessarily equivalent to expression from the physiological Xist promoter. Analysis of X inactivation in embryos is important to verify observations in a physiological context. In addition to the aforementioned models, some studies have analyzed XX somatic cells in which X inactivation is already established. However, maintenance of X inactivation in somatic cells has been reported to be Xist-independent (Csankovszki et al. 1999), albeit using relatively crude gene silencing assays, and as such these models may not be ideally suited to studying the establishment of Xist-mediated silencing.Figure 3.Models and assays for X inactivation studies. (A) Early mouse embryos. X inactivation (indicated as condensed bar with Xist RNA represented as green dashed line) proceeds at the two- to four-cell stage (E1.5–E2.5), initially on the paternally inherited X chromosome (Xp), referred to as imprinted X chromosome inactivation (iXCI). iXCI is maintained in trophectoderm (orange) and extraembryonic endoderm (green) derived tissues through development (shown here E3.5–E10.5), whereas in cells of the embryo proper (blue), initial reactivation of Xp at E3.5 is followed by random X inactivation (rXCI) of Xp or the maternal X chromosome (Xm) at around E5.5. (B) In XX mouse embryonic stem cells (mESCs) both X chromosomes are active. Random X inactivation proceeds following the onset of cellular differentiation in vitro. (C,D) X inactivation in mESCs can be uncoupled from differentiation by using an inducible promoter (red line) to drive Xist expression, illustrated on the single X chromosome in XY mESCs (C) or one of two X chromosome in XX mESCs (D). Silencing can be induced with or without cell differentiation in these models. Assays used to assess Xist-mediated silencing in wild-type (+/+) versus silencing factor mutant (−/−) include the following: (E) Phenotype of female versus male embryos/adults. (F,G) Imaging-based assays, notably nascent RNA FISH where abrogated silencing in mutants is indicated by the detection of nascent RNA (large red dot) signal within the Xist RNA domain (gray; F), and single-molecule (sm) FISH to quantify mRNA from an X-linked gene in the presence or absence of Xist RNA induction (G). smFISH illustration shows Xist RNA (gray) induced on the single X chromosome in an XY mESC line with X-linked gene mRNA (small red dots) being strongly reduced in wild-type, but not in silencing factor mutant cells. Finally, molecular assays have been used to directly quantify levels of X-linked mRNAs. (H) RNA-seq analysis using cells in which divergent X chromosomes with high single-nucleotide polymorphism density (orange and black bars), allow precise determination of the Xa:Xi ratio for RNAs (orange and black dots) for most X-linked genes when used in conjunction with nonrandom X inactivation/Xist induction.Assays for Xist-mediated silencing can be grouped into three categories; indirect phenotype/viability assays, direct imaging based assays and direct molecular assays, for example, allelic RNA-seq, illustrated in Figure 3B. Each of the approaches affords distinct advantages, but equally has specific limitations. Phenotypic assays provide evidence for the importance of a given factor in a physiological setting, but do not quantify the underlying deficiency in gene silencing. Imaging-based assays provide a good basis for measuring silencing in individual cells, within a population, for example, in early embryos, but provide limited quantitation and throughput (the number of X-linked genes that is practical to assay). Molecular assays such as allelic RNA-seq provide highly quantitative measurements of silencing for multiple genes, but the data are in most cases averaged values for large and often heterogeneous populations of cells. The increasing application of single-cell RNA-seq technologies has the potential to overcome this latter limitation.

Assaying Xist-mediated chromosome silencing

In considering recent advances in this field it is important to appreciate the varied models and assays that have been used in analyzing the contribution of different factors to Xist-mediated chromosome silencing, not least because reported findings may otherwise appear contradictory. As summarized in Figure 3A, the principal models that provide access to the developmental stage when X inactivation is established are early mouse embryos and pluripotent embryonic stem cells (mESCs). The latter include XX mESCs, which express Xist RNA from a single X chromosome when differentiated in vitro, XY mESCs with Xist expression driven by a doxycycline-activated promoter, either from the single X chromosome allele, or from an autosomally integrated Xist transgene, and XX mESCs with a doxycycline inducible promoter engineered into one of the two Xist alleles. Models using inducible promoters offer the advantage of regulatable, synchronous, and homogeneous Xist RNA expression within cell populations, albeit at levels not necessarily equivalent to expression from the physiological Xist promoter. Analysis of X inactivation in embryos is important to verify observations in a physiological context. In addition to the aforementioned models, some studies have analyzed XX somatic cells in which X inactivation is already established. However, maintenance of X inactivation in somatic cells has been reported to be Xist-independent (Csankovszki et al. 1999), albeit using relatively crude gene silencing assays, and as such these models may not be ideally suited to studying the establishment of Xist-mediated silencing.

Models and assays for X inactivation studies. (A) Early mouse embryos. X inactivation (indicated as condensed bar with Xist RNA represented as green dashed line) proceeds at the two- to four-cell stage (E1.5–E2.5), initially on the paternally inherited X chromosome (Xp), referred to as imprinted X chromosome inactivation (iXCI). iXCI is maintained in trophectoderm (orange) and extraembryonic endoderm (green) derived tissues through development (shown here E3.5–E10.5), whereas in cells of the embryo proper (blue), initial reactivation of Xp at E3.5 is followed by random X inactivation (rXCI) of Xp or the maternal X chromosome (Xm) at around E5.5. (B) In XX mouse embryonic stem cells (mESCs) both X chromosomes are active. Random X inactivation proceeds following the onset of cellular differentiation in vitro. (C,D) X inactivation in mESCs can be uncoupled from differentiation by using an inducible promoter (red line) to drive Xist expression, illustrated on the single X chromosome in XY mESCs (C) or one of two X chromosome in XX mESCs (D). Silencing can be induced with or without cell differentiation in these models. Assays used to assess Xist-mediated silencing in wild-type (+/+) versus silencing factor mutant (−/−) include the following: (E) Phenotype of female versus male embryos/adults. (F,G) Imaging-based assays, notably nascent RNA FISH where abrogated silencing in mutants is indicated by the detection of nascent RNA (large red dot) signal within the Xist RNA domain (gray; F), and single-molecule (sm) FISH to quantify mRNA from an X-linked gene in the presence or absence of Xist RNA induction (G). smFISH illustration shows Xist RNA (gray) induced on the single X chromosome in an XY mESC line with X-linked gene mRNA (small red dots) being strongly reduced in wild-type, but not in silencing factor mutant cells. Finally, molecular assays have been used to directly quantify levels of X-linked mRNAs. (H) RNA-seq analysis using cells in which divergent X chromosomes with high single-nucleotide polymorphism density (orange and black bars), allow precise determination of the Xa:Xi ratio for RNAs (orange and black dots) for most X-linked genes when used in conjunction with nonrandom X inactivation/Xist induction.

Assays for Xist-mediated silencing can be grouped into three categories; indirect phenotype/viability assays, direct imaging based assays and direct molecular assays, for example, allelic RNA-seq, illustrated in Figure 3B. Each of the approaches affords distinct advantages, but equally has specific limitations. Phenotypic assays provide evidence for the importance of a given factor in a physiological setting, but do not quantify the underlying deficiency in gene silencing. Imaging-based assays provide a good basis for measuring silencing in individual cells, within a population, for example, in early embryos, but provide limited quantitation and throughput (the number of X-linked genes that is practical to assay). Molecular assays such as allelic RNA-seq provide highly quantitative measurements of silencing for multiple genes, but the data are in most cases averaged values for large and often heterogeneous populations of cells. The increasing application of single-cell RNA-seq technologies has the potential to overcome this latter limitation.

A central role for SPEN in Xist-mediated silencingThe RBP SPEN was identified as a key factor for establishment of Xist-mediated silencing in several independent studies (Chu et al. 2015; McHugh et al. 2015; Moindrot et al. 2015; Monfort et al. 2015). Recent work has validated this conclusion in a physiological model, preimplantation mouse embryos (Dossin et al. 2020). SPEN (synonyms: MINT and SHARP), >400 kDa, is a conserved RNA-binding protein containing four canonical RRM (RNA recognition motif) domains at the N terminus and a C-terminal SPOC (SPEN paralog and ortholog C-terminal) domain (Fig. 2B). The RRM and SPOC domains are responsible for SPEN's protein–RNA interactions and protein–protein interactions, respectively. EMSA experiments confirmed that a direct interaction occurs between SPEN RRM2–4 and Xist A repeat, with comparable binding affinity to SRA (steroid receptor RNA activator) RNA, previously identified as a SPEN target (Arieti et al. 2014; Monfort et al. 2015). PARIS (psoralen analysis of RNA interactions and structures), together with SPEN RRM iCLIP-seq, suggest that SPEN binding to the Xist A repeat likely occurs at the single-stranded nucleotides 3–5 nt upstream of the interrepeat duplex (Lu et al. 2016). Analysis of the interaction of SPEN with SRA RNA revealed that RRM3 is the principal domain mediating protein–RNA interaction (Arieti et al. 2014).SPEN was originally identified as having a role in the Notch/RBP-J signaling pathway in Drosophila (Oswald et al. 2002), functioning through an interaction with the SMRT/NCoR–HDAC3 histone deacetylase complex, which represses gene transcription (Shi et al. 2001). HDAC3 is a stoichiometric component of the SMRT/NCoR complex whose core enzymatic activity and thus repressive function depends on this interaction (Emmett and Lazar 2019). Similar to SPEN, HDAC3 loss of function strongly abrogates Xist-mediated silencing (McHugh et al. 2015; Żylicz et al. 2019). These findings indicate that SPEN is directly recruited to Xist RNA through binding to the A repeat by its RRM domain, with interaction with SMRT/NCOR–HDAC3 through the SPEN SPOC domain (Ariyoshi and Schwabe 2003; Mikami et al. 2014), resulting in histone deacetylation and repression of X-linked genes. Consistent with this model, a recent study reported that deacetylation of H4 and H3K27 are among the earliest changes in chromatin modification following the onset of Xist RNA expression (Żylicz et al. 2019).While recruitment of NCoR–HDAC3 by SPEN is a key mechanism in Xist-mediated silencing, emerging evidence indicates that SPEN has additional functions in X inactivation. Specifically, deletion of the SPEN RNA-binding domains and/or the entire protein leads to a more dramatic loss of silencing than HDAC3 loss of function (Nesterova et al. 2019; Żylicz et al. 2019; Dossin et al. 2020). Interestingly, deletion of the C-terminal SPOC domain alone results in an intermediate silencing deficit (Dossin et al. 2020). Together, these findings suggest that there are other interacting factors, in addition to HDAC3, important for SPEN function in Xist-mediated silencing. One candidate is the H3K4 methyltransferase coactivator complex KMT2D, which has been reported to interact with the SPEN SPOC domain (Oswald et al. 2016). SPEN and KMT2D also physically interact in Drosophila and permit the chromatin regulation of Notch targets in vivo (Oswald et al. 2016). Sequestration of KMT2D by SPEN could conceivably be linked to the loss of H3K4me3 on Xi. A recent analysis of the SPEN SPOC domain interactome in mESCs (Dossin et al. 2020), has highlighted other candidate factors, notably the NuRD complex, a finding that is consistent with a prior study (Shi et al. 2001), and also RNAPII and associated cofactors. The latter may indicate directed RNAPII inactivation by SPEN, possibly accounting for the reduced RNAPII levels over the Xi chromosome. Further studies are required to determine whether these additional SPEN-linked functions contribute to Xist-mediated silencing.Besides its interactions with chromatin modifying factors, SPEN loss of function has been reported to reduce Xist RNA levels and to disrupt local accumulation over the Xi domain, as has deletion of the Xist A-repeat region (Nesterova et al. 2019), and this also could be a contributory factor in abrogated silencing following SPEN loss of function.

A central role for SPEN in Xist-mediated silencing

The RBP SPEN was identified as a key factor for establishment of Xist-mediated silencing in several independent studies (Chu et al. 2015; McHugh et al. 2015; Moindrot et al. 2015; Monfort et al. 2015). Recent work has validated this conclusion in a physiological model, preimplantation mouse embryos (Dossin et al. 2020). SPEN (synonyms: MINT and SHARP), >400 kDa, is a conserved RNA-binding protein containing four canonical RRM (RNA recognition motif) domains at the N terminus and a C-terminal SPOC (SPEN paralog and ortholog C-terminal) domain (Fig. 2B). The RRM and SPOC domains are responsible for SPEN's protein–RNA interactions and protein–protein interactions, respectively. EMSA experiments confirmed that a direct interaction occurs between SPEN RRM2–4 and Xist A repeat, with comparable binding affinity to SRA (steroid receptor RNA activator) RNA, previously identified as a SPEN target (Arieti et al. 2014; Monfort et al. 2015). PARIS (psoralen analysis of RNA interactions and structures), together with SPEN RRM iCLIP-seq, suggest that SPEN binding to the Xist A repeat likely occurs at the single-stranded nucleotides 3–5 nt upstream of the interrepeat duplex (Lu et al. 2016). Analysis of the interaction of SPEN with SRA RNA revealed that RRM3 is the principal domain mediating protein–RNA interaction (Arieti et al. 2014).

SPEN was originally identified as having a role in the Notch/RBP-J signaling pathway in Drosophila (Oswald et al. 2002), functioning through an interaction with the SMRT/NCoR–HDAC3 histone deacetylase complex, which represses gene transcription (Shi et al. 2001). HDAC3 is a stoichiometric component of the SMRT/NCoR complex whose core enzymatic activity and thus repressive function depends on this interaction (Emmett and Lazar 2019). Similar to SPEN, HDAC3 loss of function strongly abrogates Xist-mediated silencing (McHugh et al. 2015; Żylicz et al. 2019). These findings indicate that SPEN is directly recruited to Xist RNA through binding to the A repeat by its RRM domain, with interaction with SMRT/NCOR–HDAC3 through the SPEN SPOC domain (Ariyoshi and Schwabe 2003; Mikami et al. 2014), resulting in histone deacetylation and repression of X-linked genes. Consistent with this model, a recent study reported that deacetylation of H4 and H3K27 are among the earliest changes in chromatin modification following the onset of Xist RNA expression (Żylicz et al. 2019).

While recruitment of NCoR–HDAC3 by SPEN is a key mechanism in Xist-mediated silencing, emerging evidence indicates that SPEN has additional functions in X inactivation. Specifically, deletion of the SPEN RNA-binding domains and/or the entire protein leads to a more dramatic loss of silencing than HDAC3 loss of function (Nesterova et al. 2019; Żylicz et al. 2019; Dossin et al. 2020). Interestingly, deletion of the C-terminal SPOC domain alone results in an intermediate silencing deficit (Dossin et al. 2020). Together, these findings suggest that there are other interacting factors, in addition to HDAC3, important for SPEN function in Xist-mediated silencing. One candidate is the H3K4 methyltransferase coactivator complex KMT2D, which has been reported to interact with the SPEN SPOC domain (Oswald et al. 2016). SPEN and KMT2D also physically interact in Drosophila and permit the chromatin regulation of Notch targets in vivo (Oswald et al. 2016). Sequestration of KMT2D by SPEN could conceivably be linked to the loss of H3K4me3 on Xi. A recent analysis of the SPEN SPOC domain interactome in mESCs (Dossin et al. 2020), has highlighted other candidate factors, notably the NuRD complex, a finding that is consistent with a prior study (Shi et al. 2001), and also RNAPII and associated cofactors. The latter may indicate directed RNAPII inactivation by SPEN, possibly accounting for the reduced RNAPII levels over the Xi chromosome. Further studies are required to determine whether these additional SPEN-linked functions contribute to Xist-mediated silencing.

Besides its interactions with chromatin modifying factors, SPEN loss of function has been reported to reduce Xist RNA levels and to disrupt local accumulation over the Xi domain, as has deletion of the Xist A-repeat region (Nesterova et al. 2019), and this also could be a contributory factor in abrogated silencing following SPEN loss of function.

A role for RBM15 and the METTL3/14 complex in Xist-mediated chromosome silencingRNA modification has emerged as a new layer in controlling a variety of processes in RNA metabolism (Roundtree et al. 2017). Xist RNA shows high levels of m6A (Ke et al. 2015; Linder et al. 2015; Patil et al. 2016; Coker et al. 2019; Nesterova et al. 2019), an abundant RNA modification in mRNA and noncoding RNA (Linder et al. 2015; Patil et al. 2016). Major m6A peaks in Xist RNA lie immediately downstream from the A repeat in Xist exon I and at downstream locations in Xist exons III/IV and VII (Fig. 2A). The m6A modification is deposited on mRNA by the METTL3/14 complex consisting of a catalytic heterodimer of METTL3–METTL14 and regulatory subunit WTAP, together with accessory proteins including RBM15 and KIAA1429 (Meyer and Jaffrey 2017). YTH-domain-containing proteins YTHDC1/2 and YTHDF1/2/3 serve as m6A readers in vivo (Meyer and Jaffrey 2017).RBM15/15B (synonyms: OTT1/3) derive from the same protein family as SPEN and share a conserved arrangement of N-terminal RRM domains and a C-terminal SPOC domain. RBM15 interacts directly with the Xist A-repeat region as determined by CLIP-seq (Fig. 2A,B; Patil et al. 2016). As noted above, RBM15 (and its direct homolog, RBM15B), have been implicated in targeting the METTL3/14 complex to specific RNAs, including Xist (Patil et al. 2016). However, like SPEN, RBM15 also interacts with other factors/complexes. Examples include components of the RNA export pathway (Zolotukhin et al. 2009), and SET1B (synonym: KMT2G), a subunit of an H3K4 methyltransferase complex that interacts directly with the RBM15 SPOC domain (Lee and Skalnik 2012). Similar to the argument above for SPEN and KMT2D, sequestration of SET1B by RBM15 could contribute to reduced H3K4me3 on Xi. A recent analysis of the RBM15 interactome in mESCs confirmed association with the m6A and the SET1B complexes, and in addition, reported several other potentially interesting links, including RNA splicing factors and, intriguingly, SPEN (Coker et al. 2020). Whether or not these associations involve direct interaction with RBM15 remains to be determined.Perturbation of RBM15/METTL3/14 complex subunits affects Xist-mediated silencing to varying degrees, ranging from little or no silencing deficiency following RNAi-mediated knockdown of the regulatory subunit WTAP (Chu et al. 2015; Moindrot et al. 2015) through to a major silencing deficiency following knockdown of the catalytic subunit METTL3 (Patil et al. 2016). Loss of function of the m6A reader YTHDC1 was also reported to strongly abrogate silencing, and, moreover, tethering YTHDC1 to the 3′ end of XIST was sufficient to rescue its silencing function in the absence of the m6A methylation complex (Patil et al. 2016). Set against these findings, more recent studies reported a relatively modest effect on Xist-mediated silencing following CRISPR/Cas9-mediated deletion of genes encoding RBM15 and subunits of the METTL3/14 complex (Nesterova et al. 2019). This conclusion was supported by analysis of deletions encompassing the major m6A peak located immediately downstream from the Xist A repeat (Nesterova et al. 2019; Coker et al. 2020). The functional role of other Xist m6A sites and how they are established have not been analyzed to date. Gene knockout of RBM15 in mice results in midgestation embryo lethality but with no apparent female bias (Raffel et al. 2009). This finding supports that RBM15 plays only a modest role in Xist-mediated silencing, although possible redundancy with RBM15B may be important in this context.Several factors likely underlie the disparate findings on the role of RBM15/METTL3/14 complex in Xist-meditated silencing. The METTL3/14 complex has a global influence on ∼7000 mRNAs, affecting pre-mRNA processing, RNA stability, and translation (for review, see Yue et al. 2015), so loss of function likely generates significant secondary effects. The manifestation of secondary effects will in turn be influenced by the means for abrogating function (knockdown vs. chronic knockout vs. acute knockout). Additional confounding factors are functional redundancy of METTL3/14 complex subunits (for example, RBM15 and RBM15B) (Patil et al. 2016), and possibly nonessentiality of regulatory subunits (for example, WTAP) in m6A catalysis. Finally, as noted above, use of different models (XY vs. XX mESCs) and silencing assays (single-molecule FISH vs. allelic RNA-seq), is likely a contributory factor for the aforementioned studies coming to different conclusions (see also comments below).While the emerging consensus is that RBM15 and m6A play a relatively minor role in Xist-mediated gene silencing, at least compared with other factors such as SPEN and the Polycomb system, more work is needed to corroborate this conclusion and also to understand the mechanistic basis for the effect that is seen. In the latter case, possibilities include that m6A readers such as YTHDC1 mediate gene silencing, or that m6A modification impacts on Xist RNA folding and/or binding of other RBPs linked to silencing (so-called “m6A switch”) (Liu et al. 2015), for example, SPEN. With this in mind it is interesting to consider how RBM15 and SPEN may affect one another's interaction with A repeat, given both proteins likely compete for common binding sites. A final consideration, given that m6A has been reported to regulate RNA stability (Wang et al. 2014), is that RBM15/m6A may affect Xist RNA turnover, and thereby modify Xist-mediated silencing indirectly.

A role for RBM15 and the METTL3/14 complex in Xist-mediated chromosome silencing

RNA modification has emerged as a new layer in controlling a variety of processes in RNA metabolism (Roundtree et al. 2017). Xist RNA shows high levels of m6A (Ke et al. 2015; Linder et al. 2015; Patil et al. 2016; Coker et al. 2019; Nesterova et al. 2019), an abundant RNA modification in mRNA and noncoding RNA (Linder et al. 2015; Patil et al. 2016). Major m6A peaks in Xist RNA lie immediately downstream from the A repeat in Xist exon I and at downstream locations in Xist exons III/IV and VII (Fig. 2A). The m6A modification is deposited on mRNA by the METTL3/14 complex consisting of a catalytic heterodimer of METTL3–METTL14 and regulatory subunit WTAP, together with accessory proteins including RBM15 and KIAA1429 (Meyer and Jaffrey 2017). YTH-domain-containing proteins YTHDC1/2 and YTHDF1/2/3 serve as m6A readers in vivo (Meyer and Jaffrey 2017).

RBM15/15B (synonyms: OTT1/3) derive from the same protein family as SPEN and share a conserved arrangement of N-terminal RRM domains and a C-terminal SPOC domain. RBM15 interacts directly with the Xist A-repeat region as determined by CLIP-seq (Fig. 2A,B; Patil et al. 2016). As noted above, RBM15 (and its direct homolog, RBM15B), have been implicated in targeting the METTL3/14 complex to specific RNAs, including Xist (Patil et al. 2016). However, like SPEN, RBM15 also interacts with other factors/complexes. Examples include components of the RNA export pathway (Zolotukhin et al. 2009), and SET1B (synonym: KMT2G), a subunit of an H3K4 methyltransferase complex that interacts directly with the RBM15 SPOC domain (Lee and Skalnik 2012). Similar to the argument above for SPEN and KMT2D, sequestration of SET1B by RBM15 could contribute to reduced H3K4me3 on Xi. A recent analysis of the RBM15 interactome in mESCs confirmed association with the m6A and the SET1B complexes, and in addition, reported several other potentially interesting links, including RNA splicing factors and, intriguingly, SPEN (Coker et al. 2020). Whether or not these associations involve direct interaction with RBM15 remains to be determined.

Perturbation of RBM15/METTL3/14 complex subunits affects Xist-mediated silencing to varying degrees, ranging from little or no silencing deficiency following RNAi-mediated knockdown of the regulatory subunit WTAP (Chu et al. 2015; Moindrot et al. 2015) through to a major silencing deficiency following knockdown of the catalytic subunit METTL3 (Patil et al. 2016). Loss of function of the m6A reader YTHDC1 was also reported to strongly abrogate silencing, and, moreover, tethering YTHDC1 to the 3′ end of XIST was sufficient to rescue its silencing function in the absence of the m6A methylation complex (Patil et al. 2016). Set against these findings, more recent studies reported a relatively modest effect on Xist-mediated silencing following CRISPR/Cas9-mediated deletion of genes encoding RBM15 and subunits of the METTL3/14 complex (Nesterova et al. 2019). This conclusion was supported by analysis of deletions encompassing the major m6A peak located immediately downstream from the Xist A repeat (Nesterova et al. 2019; Coker et al. 2020). The functional role of other Xist m6A sites and how they are established have not been analyzed to date. Gene knockout of RBM15 in mice results in midgestation embryo lethality but with no apparent female bias (Raffel et al. 2009). This finding supports that RBM15 plays only a modest role in Xist-mediated silencing, although possible redundancy with RBM15B may be important in this context.

Several factors likely underlie the disparate findings on the role of RBM15/METTL3/14 complex in Xist-meditated silencing. The METTL3/14 complex has a global influence on ∼7000 mRNAs, affecting pre-mRNA processing, RNA stability, and translation (for review, see Yue et al. 2015), so loss of function likely generates significant secondary effects. The manifestation of secondary effects will in turn be influenced by the means for abrogating function (knockdown vs. chronic knockout vs. acute knockout). Additional confounding factors are functional redundancy of METTL3/14 complex subunits (for example, RBM15 and RBM15B) (Patil et al. 2016), and possibly nonessentiality of regulatory subunits (for example, WTAP) in m6A catalysis. Finally, as noted above, use of different models (XY vs. XX mESCs) and silencing assays (single-molecule FISH vs. allelic RNA-seq), is likely a contributory factor for the aforementioned studies coming to different conclusions (see also comments below).

While the emerging consensus is that RBM15 and m6A play a relatively minor role in Xist-mediated gene silencing, at least compared with other factors such as SPEN and the Polycomb system, more work is needed to corroborate this conclusion and also to understand the mechanistic basis for the effect that is seen. In the latter case, possibilities include that m6A readers such as YTHDC1 mediate gene silencing, or that m6A modification impacts on Xist RNA folding and/or binding of other RBPs linked to silencing (so-called “m6A switch”) (Liu et al. 2015), for example, SPEN. With this in mind it is interesting to consider how RBM15 and SPEN may affect one another's interaction with A repeat, given both proteins likely compete for common binding sites. A final consideration, given that m6A has been reported to regulate RNA stability (Wang et al. 2014), is that RBM15/m6A may affect Xist RNA turnover, and thereby modify Xist-mediated silencing indirectly.

A reassessment of the role of LBR in Xist-mediated silencingProteomic screens identified Lamin B receptor (LBR), an integral component of the nuclear lamina, as a direct Xist interactor (McHugh et al. 2015; Minajigi et al. 2015). Consistent with this finding LBR was identified as a candidate mRNA binder in the mRNA-bound proteome (Baltz et al. 2012). More recent work has reported that LBR binds to three sites across Xist RNA as determined by CLIP-seq, with the most prominent, referred to as LBS, located downstream from the major 5′ m6A peak in and encompassing the entire F repeat (Fig. 2A; Chen et al. 2016). The interaction of LBR with Xist RNA was shown to be through an arginine–serine tract (RS) ( Fig. 2B). In addition to major sites of enrichment there is a widespread distribution of LBR across Xist RNA (Chen et al. 2016; Cirillo et al. 2017), in contrast to other well-characterized Xist RBPs such as SPEN, RBM15, and hnRNPK (see Fig. 2A). This may indicate a relatively low sequence specificity in the interaction of LBR with RNA.In initial studies, LBR was reported to be critically important for Xist-mediated silencing, as determined using smFISH (see Fig. 3) following LBR depletion by RNAi (McHugh et al. 2015; Chen et al. 2016). Similar loss of silencing was seen upon deletion of LBS, and moreover, this deficit was complemented by tethering LBR synthetically to the 3′ end of Xist RNA. Further analysis suggested that LBR facilitates silencing by recruiting the inactive X chromosome to the nuclear lamina to enable Xist RNPs to spread to actively transcribed genes (Chen et al. 2016). A more recent study reported contrasting conclusions with only a minor effect on silencing following CRISPR/Cas9-mediated deletion of Lbr or LBS following Xist induction in XX mESCs (Nesterova et al. 2019). This study also analyzed Lbr deletion in an autosomal Xist transgenic XY mESC line and found no detectable effects on Xist-mediated silencing. The minor silencing deficiency observed in XX mESCs disappeared after extended Xist RNA induction (6 d), suggesting that LBR may enhance the rate of Xist-mediated silencing, possibly, as reported by Chen et al. (2016), by facilitating association of Xi with the nuclear periphery. The conclusion that LBR plays a relatively minor role in Xist-mediated silencing is supported by analysis of Lbr gene mutations in mice, for which there are no reported female-specific phenotypes (Shultz et al. 2003; Cohen et al. 2008).The contrasting findings in different studies on the role of LBR in X inactivation are likely attributable to the use of different models and assay systems. In particular, silencing deficiencies determined using the smFISH assay (Fig. 3) appear to be considerably greater than those seen using allelic RNA-seq, also evident in relation to the disparate findings on the role of RBM15 and the METTL3/14 complex discussed above. The quantitative power of allelic RNA-seq, together with the greater coverage of X-linked genes, suggests this assay likely provides a more reliable measure, although this point is open to debate. An additional consideration in relation to studies on LBR is the use of different perturbation strategies, specifically RNAi-mediated knockdown, which is relatively acute, compared with chronic effects in null mESCs generated using CRISPR/Cas9 gene deletion. Again, further studies should address whether this is relevant.

A reassessment of the role of LBR in Xist-mediated silencing

Proteomic screens identified Lamin B receptor (LBR), an integral component of the nuclear lamina, as a direct Xist interactor (McHugh et al. 2015; Minajigi et al. 2015). Consistent with this finding LBR was identified as a candidate mRNA binder in the mRNA-bound proteome (Baltz et al. 2012). More recent work has reported that LBR binds to three sites across Xist RNA as determined by CLIP-seq, with the most prominent, referred to as LBS, located downstream from the major 5′ m6A peak in and encompassing the entire F repeat (Fig. 2A; Chen et al. 2016). The interaction of LBR with Xist RNA was shown to be through an arginine–serine tract (RS) ( Fig. 2B). In addition to major sites of enrichment there is a widespread distribution of LBR across Xist RNA (Chen et al. 2016; Cirillo et al. 2017), in contrast to other well-characterized Xist RBPs such as SPEN, RBM15, and hnRNPK (see Fig. 2A). This may indicate a relatively low sequence specificity in the interaction of LBR with RNA.

In initial studies, LBR was reported to be critically important for Xist-mediated silencing, as determined using smFISH (see Fig. 3) following LBR depletion by RNAi (McHugh et al. 2015; Chen et al. 2016). Similar loss of silencing was seen upon deletion of LBS, and moreover, this deficit was complemented by tethering LBR synthetically to the 3′ end of Xist RNA. Further analysis suggested that LBR facilitates silencing by recruiting the inactive X chromosome to the nuclear lamina to enable Xist RNPs to spread to actively transcribed genes (Chen et al. 2016). A more recent study reported contrasting conclusions with only a minor effect on silencing following CRISPR/Cas9-mediated deletion of Lbr or LBS following Xist induction in XX mESCs (Nesterova et al. 2019). This study also analyzed Lbr deletion in an autosomal Xist transgenic XY mESC line and found no detectable effects on Xist-mediated silencing. The minor silencing deficiency observed in XX mESCs disappeared after extended Xist RNA induction (6 d), suggesting that LBR may enhance the rate of Xist-mediated silencing, possibly, as reported by Chen et al. (2016), by facilitating association of Xi with the nuclear periphery. The conclusion that LBR plays a relatively minor role in Xist-mediated silencing is supported by analysis of Lbr gene mutations in mice, for which there are no reported female-specific phenotypes (Shultz et al. 2003; Cohen et al. 2008).

The contrasting findings in different studies on the role of LBR in X inactivation are likely attributable to the use of different models and assay systems. In particular, silencing deficiencies determined using the smFISH assay (Fig. 3) appear to be considerably greater than those seen using allelic RNA-seq, also evident in relation to the disparate findings on the role of RBM15 and the METTL3/14 complex discussed above. The quantitative power of allelic RNA-seq, together with the greater coverage of X-linked genes, suggests this assay likely provides a more reliable measure, although this point is open to debate. An additional consideration in relation to studies on LBR is the use of different perturbation strategies, specifically RNAi-mediated knockdown, which is relatively acute, compared with chronic effects in null mESCs generated using CRISPR/Cas9 gene deletion. Again, further studies should address whether this is relevant.

Recruitment of hnRNPK and the Polycomb system through B/C repeat contributes to Xist-mediated silencingThe Polycomb system, which comprises several multiprotein complexes that catalyze the histone modifications H2AK119ub1 (canonical and variant PRC1 complexes), and H3K27me3 (PRC2 complexes), is recruited to Xi in an Xist RNA-dependent manner (for review, see Brockdorff 2017). Briefly, the RBP hnRNPK bound to the Xist RNA B/C repeat initiates Polycomb recruitment via direct interaction of the hnRNPK KI domain (Fig. 2B), and the PCGF3/5 subunit of PCGF3/5–PRC1. PCGF3/5–PRC1-mediated H2AK119ub1 further directs concentration of other variant PRC1 complexes, PRC2, and ultimately, canonical PRC1 via binding to PRC2-mediated H3K27me3 (da Rocha et al. 2014; Chu et al. 2015; Cooper et al. 2016; Almeida et al. 2017; Pintacuda et al. 2017). hnRNPK has three RNA-binding KH domains (Fig. 2B) that function cooperatively by each recognizing a C-rich RNA motif (Paziewska et al. 2004). The B/C-repeat region in both humans and mice comprises ∼30 tandem repeats (WGCCC), which could theoretically bind up to 10 hnRNPK molecules. Occupancy of this element by multiple hnRNPK molecules is supported by CLIP-seq (Fig. 2A; Cirillo et al. 2017), and EMSA analyses (Colognori et al. 2019). Perturbation of hnRNPK strongly abrogates Polycomb recruitment, phenocopying deletion of the B/C repeat (Chu et al. 2015; Pintacuda et al. 2017; Colognori et al. 2019), while tethering of hnRNPK to XistΔB/C can rescue recruitment of Polycomb (Pintacuda et al. 2017).Early studies reported that perturbation of PRC2 results in a weak effect on Xist-mediated silencing (Wang et al. 2001; Silva et al. 2003; Kalantry and Magnuson 2006; Kalantry et al. 2006;). More recent studies addressed the role of PRC1 and PRC2 together (deletion of PCGF3/5 or the B/C repeat region), and reported a substantive reduction in silencing efficiency, using both inducible autosomal Xist in XY mESCs and XX mESC models (Almeida et al. 2017; Pintacuda et al. 2017; Nesterova et al. 2019). It was further shown that complete PRC1 loss of function as opposed to PRC2 loss (and as a consequence, canonical but not variant PRC1) is the principal contributor to abrogated silencing (Nesterova et al. 2019). Presumably H2AK119ub1 catalysis is important for this effect, although this remains to be formally proven. A primary role for PRC1 was substantiated through phenotype analysis of Pcgf3/5 null embryos, with female lethality occurring at an earlier stage than in male embryos (E7.5–E9.5 compared with E9.5–E12.5).Other recent studies broadly support that the B/C-repeat element, hnRNPK, and PRC1 mediate the recruitment of Polycomb to Xi (Chu et al. 2015; Bousard et al. 2019; Colognori et al. 2019), although there are some differences in interpretation. Colognori et al. (2019) found that a low level of PRC2-mediated H3K27me3 is retained over Xi following deletion of the catalytic subunits of PRC1, and this was suggested to indicate an independent pathway for PRC2 recruitment by Xist RNA. However, because in this instance deletion of PRC1/Xist B repeat was performed in XX somatic cells (MEFs), after establishment of X inactivation, retention of H3K27me3 could be linked to PRC2 self-templating, as has been described previously (Hansen et al. 2008; Margueron et al. 2009). The study by Bousard et al. (2019), also reported low levels of H3K27me3 enrichment on Xi after induction of Xist-RNA lacking the B/C repeat region, although this was only within the bodies of silenced genes and is likely due to loss of active gene-associated histone modifications, specifically H3K4me3 and H3K36me3, that directly inhibit PRC2 catalytic activity (Schmitges et al. 2011).Bousard et al. (2019) concluded that the Polycomb pathway makes a relatively small contribution to Xist-mediated silencing, in contrast to the conclusions of Pintacuda et al. (2017), Nesterova et al. (2019), and Colognori et al. (2019). This difference could relate to the use of different cell models as Bousard et al. (2019) analyzed XY mESCs with inducible Xist expression on the single X chromosome. There were also differences in the extent of the B/C-repeat region deletions, and the time points following onset of Xist RNA expression. Indeed, Nesterova et al. (2019) reported a more pronounced effect on Xist-mediated silencing following deletion of the B/C repeat region after 6 d (with mESC differentiation) compared with 1 d of Xist RNA induction. Bousard et al. (2019) were not able to analyze later time points as Xist expression triggers rapid cell death in the XY mESC model.The mechanism by which Polycomb represses genes is not well understood, and this applies also in the case of Xist-mediated silencing. As noted above, both H2AK119ub1 and H3K27me3 enrichment occur over gene-rich domains covering much of the Xi chromosome, encompassing regulatory elements (REs), gene bodies, and intergenic regions. Thus, silencing could result from the activity of reader proteins that bind these histone modifications, or alternatively, through effects on chromatin structure/accessibility, for example chromatin compaction limiting TF and RNAPII accessibility at REs.In relation to Polycomb reader proteins, recent work has shown that recruitment of the chromosomal architecture protein SmcHD1 to Xi, which occurs several days after the onset of Xist expression (Gendrel et al. 2012), is dependent on PRC1 activity (Jansz et al. 2018b). SmcHD1 is thought to function in long-term maintenance of X inactivation, being required for DNA methylation at the majority of Xi CpG island promoters, and also for Xi-specific higher-order chromosome structure (Gendrel et al. 2013; Mould et al. 2013; Jansz et al. 2018a; Sakakibara et al. 2018; Wang et al. 2018; Gdula et al. 2019). This pathway may therefore account for the more pronounced contribution of Polycomb to Xist-mediated silencing at later stages of mESC differentiation.A final consideration in how Polycomb impacts Xist-mediated silencing comes from a recent report showing abnormal Xist RNA localization following deletion of the Xist B repeat, hnRNPK, or PRC1 (Colognori et al. 2019). Thus, similar to SPEN and the A repeat, at least some of the observed silencing deficiency in the absence of the Polycomb system may result from perturbation of Xist–RNA localization rather than Xi chromatin modification.

Recruitment of hnRNPK and the Polycomb system through B/C repeat contributes to Xist-mediated silencing

The Polycomb system, which comprises several multiprotein complexes that catalyze the histone modifications H2AK119ub1 (canonical and variant PRC1 complexes), and H3K27me3 (PRC2 complexes), is recruited to Xi in an Xist RNA-dependent manner (for review, see Brockdorff 2017). Briefly, the RBP hnRNPK bound to the Xist RNA B/C repeat initiates Polycomb recruitment via direct interaction of the hnRNPK KI domain (Fig. 2B), and the PCGF3/5 subunit of PCGF3/5–PRC1. PCGF3/5–PRC1-mediated H2AK119ub1 further directs concentration of other variant PRC1 complexes, PRC2, and ultimately, canonical PRC1 via binding to PRC2-mediated H3K27me3 (da Rocha et al. 2014; Chu et al. 2015; Cooper et al. 2016; Almeida et al. 2017; Pintacuda et al. 2017). hnRNPK has three RNA-binding KH domains (Fig. 2B) that function cooperatively by each recognizing a C-rich RNA motif (Paziewska et al. 2004). The B/C-repeat region in both humans and mice comprises ∼30 tandem repeats (WGCCC), which could theoretically bind up to 10 hnRNPK molecules. Occupancy of this element by multiple hnRNPK molecules is supported by CLIP-seq (Fig. 2A; Cirillo et al. 2017), and EMSA analyses (Colognori et al. 2019). Perturbation of hnRNPK strongly abrogates Polycomb recruitment, phenocopying deletion of the B/C repeat (Chu et al. 2015; Pintacuda et al. 2017; Colognori et al. 2019), while tethering of hnRNPK to XistΔB/C can rescue recruitment of Polycomb (Pintacuda et al. 2017).

Early studies reported that perturbation of PRC2 results in a weak effect on Xist-mediated silencing (Wang et al. 2001; Silva et al. 2003; Kalantry and Magnuson 2006; Kalantry et al. 2006;). More recent studies addressed the role of PRC1 and PRC2 together (deletion of PCGF3/5 or the B/C repeat region), and reported a substantive reduction in silencing efficiency, using both inducible autosomal Xist in XY mESCs and XX mESC models (Almeida et al. 2017; Pintacuda et al. 2017; Nesterova et al. 2019). It was further shown that complete PRC1 loss of function as opposed to PRC2 loss (and as a consequence, canonical but not variant PRC1) is the principal contributor to abrogated silencing (Nesterova et al. 2019). Presumably H2AK119ub1 catalysis is important for this effect, although this remains to be formally proven. A primary role for PRC1 was substantiated through phenotype analysis of Pcgf3/5 null embryos, with female lethality occurring at an earlier stage than in male embryos (E7.5–E9.5 compared with E9.5–E12.5).

Other recent studies broadly support that the B/C-repeat element, hnRNPK, and PRC1 mediate the recruitment of Polycomb to Xi (Chu et al. 2015; Bousard et al. 2019; Colognori et al. 2019), although there are some differences in interpretation. Colognori et al. (2019) found that a low level of PRC2-mediated H3K27me3 is retained over Xi following deletion of the catalytic subunits of PRC1, and this was suggested to indicate an independent pathway for PRC2 recruitment by Xist RNA. However, because in this instance deletion of PRC1/Xist B repeat was performed in XX somatic cells (MEFs), after establishment of X inactivation, retention of H3K27me3 could be linked to PRC2 self-templating, as has been described previously (Hansen et al. 2008; Margueron et al. 2009). The study by Bousard et al. (2019), also reported low levels of H3K27me3 enrichment on Xi after induction of Xist-RNA lacking the B/C repeat region, although this was only within the bodies of silenced genes and is likely due to loss of active gene-associated histone modifications, specifically H3K4me3 and H3K36me3, that directly inhibit PRC2 catalytic activity (Schmitges et al. 2011).

Bousard et al. (2019) concluded that the Polycomb pathway makes a relatively small contribution to Xist-mediated silencing, in contrast to the conclusions of Pintacuda et al. (2017), Nesterova et al. (2019), and Colognori et al. (2019). This difference could relate to the use of different cell models as Bousard et al. (2019) analyzed XY mESCs with inducible Xist expression on the single X chromosome. There were also differences in the extent of the B/C-repeat region deletions, and the time points following onset of Xist RNA expression. Indeed, Nesterova et al. (2019) reported a more pronounced effect on Xist-mediated silencing following deletion of the B/C repeat region after 6 d (with mESC differentiation) compared with 1 d of Xist RNA induction. Bousard et al. (2019) were not able to analyze later time points as Xist expression triggers rapid cell death in the XY mESC model.

The mechanism by which Polycomb represses genes is not well understood, and this applies also in the case of Xist-mediated silencing. As noted above, both H2AK119ub1 and H3K27me3 enrichment occur over gene-rich domains covering much of the Xi chromosome, encompassing regulatory elements (REs), gene bodies, and intergenic regions. Thus, silencing could result from the activity of reader proteins that bind these histone modifications, or alternatively, through effects on chromatin structure/accessibility, for example chromatin compaction limiting TF and RNAPII accessibility at REs.

In relation to Polycomb reader proteins, recent work has shown that recruitment of the chromosomal architecture protein SmcHD1 to Xi, which occurs several days after the onset of Xist expression (Gendrel et al. 2012), is dependent on PRC1 activity (Jansz et al. 2018b). SmcHD1 is thought to function in long-term maintenance of X inactivation, being required for DNA methylation at the majority of Xi CpG island promoters, and also for Xi-specific higher-order chromosome structure (Gendrel et al. 2013; Mould et al. 2013; Jansz et al. 2018a; Sakakibara et al. 2018; Wang et al. 2018; Gdula et al. 2019). This pathway may therefore account for the more pronounced contribution of Polycomb to Xist-mediated silencing at later stages of mESC differentiation.

A final consideration in how Polycomb impacts Xist-mediated silencing comes from a recent report showing abnormal Xist RNA localization following deletion of the Xist B repeat, hnRNPK, or PRC1 (Colognori et al. 2019). Thus, similar to SPEN and the A repeat, at least some of the observed silencing deficiency in the absence of the Polycomb system may result from perturbation of Xist–RNA localization rather than Xi chromatin modification.

The interplay of silencing pathways in X inactivationA key question leading on from the identification of pathways for Xist-mediated silencing is their relative contribution and relationship to one another. In addressing this it is important to consider that not all genes on the X chromosome, or for that matter on autosomes silenced by Xist transgenes, respond equivalently. Several studies have reported variation in the silencing rate of individual genes and gene subsets (Okamoto et al. 2004; Lin et al. 2007; Berletch et al. 2015; Marks et al. 2015; Nesterova et al. 2019), and a number of genes show either partial or complete escape from X inactivation, in some cases varying depending on developmental stage and cell type (Berletch et al. 2015).As discussed above, the use of different models, time points, and assays in analyzing key factors means that it is difficult to make meaningful quantitative comparisons between studies. More recently this has been addressed in a systematic analysis of the major pathways, analyzing common time points following the onset of Xist RNA expression (Nesterova et al. 2019). A-repeat/SPEN was found to be the predominant pathway, with only low levels of silencing detected in its absence. The B-repeat/hnRNPK/PRC1 pathway was the second most significant in terms of overall contribution to silencing, with both the RBM15/METTL3/14 complex and LBS/LBR contributing relatively little. A simple interpretation of these observations is that the A-repeat/SPEN pathway establishes silencing of X-linked genes, and that other pathways function downstream, augmenting A-repeat/SPEN-mediated repression and/or stabilizing gene silencing for long-term maintenance of X inactivation. In support of this view, Żylicz et al. (2019) reported that HDAC3 loss of function delays the acquisition of Polycomb-mediated H2AK119ub1/H3K27me3 on Xi. This interpretation nevertheless needs to be tempered taking into consideration that A-repeat/SPEN and B-repeat/hnRNPK/PRC1 have more than one downstream effector and, moreover, that both pathways contribute to correct localization and/or maintaining high levels of Xist RNA on the chromosome.In relation to the importance of different pathways for silencing specific genes or gene subsets, two recent studies applied machine learning to define which features affect the rate or efficiency of silencing of X-linked genes. In both cases, the principal determinants that were identified were 2D/3D proximity relative to the Xist transcription site and pre-existing chromatin features linked to gene activity (reducing silencing efficiency) or gene repression (enhancing silencing efficiency) (Barros de Andrade et al. 2019; Nesterova et al. 2019). The results of the chromatin feature analysis suggest that highly expressed genes are silenced less well and vice versa. Accordingly Nesterova et al. (2019), reported that the SPEN/NCoR-HDAC3 pathway is especially important for silencing highly transcribed genes, a finding that accords with a prior study that analyzed allelic silencing in trophoblast tissues of female embryos following deletion of the A repeat (Sakata et al. 2017). Consistent with these observations, initial deacetylation events linked to SPEN-HDAC3 occur preferentially at transcriptionally active regions (Żylicz et al. 2019). Moreover, a recent study reported that SPEN-binding sites on Xi chromatin correspond to promoters and enhancers of active genes, correlating closely with sites of RNAPII enrichment (Dossin et al. 2020). As silencing progresses, the level of SPEN enrichment reduces proportionately, suggesting that active transcription is required for the ongoing recruitment of SPEN.The B/C-repeat/hnRNPK/PRC1 pathway results in increased deposition of H2AK119ub1 and H3K27me3 chromosome wide, notably over gene-rich domains where Xist RNA is concentrated (Marks et al. 2009; Calabrese et al. 2012; Pinter et al. 2012; Nesterova et al. 2019; Żylicz et al. 2019). Accordingly, recent analyses using allelic RNA-seq indicate that the contribution of Polycomb-mediated silencing is equivalent for most, if not all, of the genes located in cis with expressed Xist RNA (Almeida et al. 2017; Pintacuda et al. 2017; Colognori et al. 2019; Nesterova et al. 2019). Nesterova et al. (2019) did find, however, that genes with pre-existing polycomb marks, which in general are expressed at a relatively low level, tend to be more dependent on PCGF3/5 than those with preactive marks. Consistent with these findings, the placement of the repressive marks H2AK119ub1 and H3K27me3 occurs preferentially at pre-existing repressive environments (Nesterova et al. 2019). The likely reason for this is that positive feedback facilitates amplification of existing Polycomb-mediated chromatin modifications. Conversely, the small contribution made by PRC2 in silencing X-linked genes was found to occur predominantly at highly active genes during later progression of X inactivation, suggesting a role in maintaining or reinforcing depletion of histone modifications associated with gene activity, for example H3K27ac.The differential effects on repression of gene subsets by the two major silencing pathways are relatively subtle, with most genes showing equivalent dependence on both pathways. Similarly, for the RBM15/METTL3/14 and LBR/LBS pathways there were no significant differential effects on gene subsets (Nesterova et al. 2019). Thus, the overall picture is that the different silencing pathways function synergistically, contributing to the silencing of all X-linked genes rather than through division of labor, to distinct gene subsets.

The interplay of silencing pathways in X inactivation

A key question leading on from the identification of pathways for Xist-mediated silencing is their relative contribution and relationship to one another. In addressing this it is important to consider that not all genes on the X chromosome, or for that matter on autosomes silenced by Xist transgenes, respond equivalently. Several studies have reported variation in the silencing rate of individual genes and gene subsets (Okamoto et al. 2004; Lin et al. 2007; Berletch et al. 2015; Marks et al. 2015; Nesterova et al. 2019), and a number of genes show either partial or complete escape from X inactivation, in some cases varying depending on developmental stage and cell type (Berletch et al. 2015).

As discussed above, the use of different models, time points, and assays in analyzing key factors means that it is difficult to make meaningful quantitative comparisons between studies. More recently this has been addressed in a systematic analysis of the major pathways, analyzing common time points following the onset of Xist RNA expression (Nesterova et al. 2019). A-repeat/SPEN was found to be the predominant pathway, with only low levels of silencing detected in its absence. The B-repeat/hnRNPK/PRC1 pathway was the second most significant in terms of overall contribution to silencing, with both the RBM15/METTL3/14 complex and LBS/LBR contributing relatively little. A simple interpretation of these observations is that the A-repeat/SPEN pathway establishes silencing of X-linked genes, and that other pathways function downstream, augmenting A-repeat/SPEN-mediated repression and/or stabilizing gene silencing for long-term maintenance of X inactivation. In support of this view, Żylicz et al. (2019) reported that HDAC3 loss of function delays the acquisition of Polycomb-mediated H2AK119ub1/H3K27me3 on Xi. This interpretation nevertheless needs to be tempered taking into consideration that A-repeat/SPEN and B-repeat/hnRNPK/PRC1 have more than one downstream effector and, moreover, that both pathways contribute to correct localization and/or maintaining high levels of Xist RNA on the chromosome.

In relation to the importance of different pathways for silencing specific genes or gene subsets, two recent studies applied machine learning to define which features affect the rate or efficiency of silencing of X-linked genes. In both cases, the principal determinants that were identified were 2D/3D proximity relative to the Xist transcription site and pre-existing chromatin features linked to gene activity (reducing silencing efficiency) or gene repression (enhancing silencing efficiency) (Barros de Andrade et al. 2019; Nesterova et al. 2019). The results of the chromatin feature analysis suggest that highly expressed genes are silenced less well and vice versa. Accordingly Nesterova et al. (2019), reported that the SPEN/NCoR-HDAC3 pathway is especially important for silencing highly transcribed genes, a finding that accords with a prior study that analyzed allelic silencing in trophoblast tissues of female embryos following deletion of the A repeat (Sakata et al. 2017). Consistent with these observations, initial deacetylation events linked to SPEN-HDAC3 occur preferentially at transcriptionally active regions (Żylicz et al. 2019). Moreover, a recent study reported that SPEN-binding sites on Xi chromatin correspond to promoters and enhancers of active genes, correlating closely with sites of RNAPII enrichment (Dossin et al. 2020). As silencing progresses, the level of SPEN enrichment reduces proportionately, suggesting that active transcription is required for the ongoing recruitment of SPEN.

The B/C-repeat/hnRNPK/PRC1 pathway results in increased deposition of H2AK119ub1 and H3K27me3 chromosome wide, notably over gene-rich domains where Xist RNA is concentrated (Marks et al. 2009; Calabrese et al. 2012; Pinter et al. 2012; Nesterova et al. 2019; Żylicz et al. 2019). Accordingly, recent analyses using allelic RNA-seq indicate that the contribution of Polycomb-mediated silencing is equivalent for most, if not all, of the genes located in cis with expressed Xist RNA (Almeida et al. 2017; Pintacuda et al. 2017; Colognori et al. 2019; Nesterova et al. 2019). Nesterova et al. (2019) did find, however, that genes with pre-existing polycomb marks, which in general are expressed at a relatively low level, tend to be more dependent on PCGF3/5 than those with preactive marks. Consistent with these findings, the placement of the repressive marks H2AK119ub1 and H3K27me3 occurs preferentially at pre-existing repressive environments (Nesterova et al. 2019). The likely reason for this is that positive feedback facilitates amplification of existing Polycomb-mediated chromatin modifications. Conversely, the small contribution made by PRC2 in silencing X-linked genes was found to occur predominantly at highly active genes during later progression of X inactivation, suggesting a role in maintaining or reinforcing depletion of histone modifications associated with gene activity, for example H3K27ac.

The differential effects on repression of gene subsets by the two major silencing pathways are relatively subtle, with most genes showing equivalent dependence on both pathways. Similarly, for the RBM15/METTL3/14 and LBR/LBS pathways there were no significant differential effects on gene subsets (Nesterova et al. 2019). Thus, the overall picture is that the different silencing pathways function synergistically, contributing to the silencing of all X-linked genes rather than through division of labor, to distinct gene subsets.

Future perspectivesRecent years have seen significant progress toward understanding how Xist RNA establishes chromosome-wide silencing. Chromatin modifications linked to SPEN/HDAC3 and hnRNPK/PRC1 provide a compelling explanation for classical features of Xi chromatin, namely, histone hypoacetylation and deposition of H2AK119ub1 and H3K27me3 histone modifications, all of which occur commensurate with the onset of Xist RNA expression (Nesterova et al. 2019; Żylicz et al. 2019). The basis for other classical Xi features linked to establishment of Xist-mediated silencing, specifically depletion of RNAPII and loss of H3K4me2/3 (Chaumeil et al. 2006), remain to be fully resolved. One possibility is that these are indirect consequences of a repressive chromatin state linked to the aforementioned histone modifications, and negative feedback mechanisms such as inhibition of H3K4me3 by PRC2-mediated H3K27me3 (Schmitges et al. 2011). Alternatively, more direct mechanisms may play a role, supported by the findings that the SPEN SPOC domain interacts with RNAPII/associated cofactors (Dossin et al. 2020), and both SPEN and RBM15 have direct interactions with complexes that catalyze H3K4me3 (Lee and Skalnik 2012; Oswald et al. 2016; Coker et al. 2020). The mechanistic basis for the relatively minor contribution of RBM15/m6A and LBR in establishment of silencing is at present a matter of debate but may relate to known roles for the RBM15/m6A pathway in regulating mRNA dynamics, and the role of LBR in associating the Xi with the nuclear periphery. Figure 4 summarizes key findings discussed in this review.Figure 4.Pathways for Xist-mediated silencing. Schematic illustrates Xist RNA with tandem repeats A-F indicated together with associated RBPs and downstream effectors implicated in chromosome silencing. SPEN is linked to promoter/enhancer histone deacetylation through NCoR–HDAC3. hnRNPK recruits PCGF3/5–PRC1, which catalyzes widespread deposition of H2AK119ub1. H2AK119ub1 provides a binding platform for other variant (v) PRC1 complexes and PRC2, with PRC2-mediated H3K27me3, then recruiting canonical (c) PRC1. RBM15 is linked to m6A deposition on Xist RNA via recruitment of the METTL3/14 complex, facilitating silencing through yet to be determined mechanisms. LBR links Xist/Xi with the nuclear envelope and the repressive Lamin-associated environment. Other putative pathways, indicated in gray with dashed gray arrows, are RBM15-SET1B and/or SPEN–KMT2D potentially contributing to loss of H3K4me3 on Xi, SPEN–NuRD, potentially contributing to Xi histone deacetylation, and direct inhibitory interaction of SPEN with RNAPII/cofactors. Circular arrows denote catalytic activity. Both SPEN and PRC1 also contribute to anchoring Xist RNA to the nuclear matrix (dashed pink/green arrows), which may in turn contribute to their role in silencing.As the field has progressed toward defining the pathways for Xist-mediated silencing, new frontiers and challenges have begun to open up. A notable example is the possible role of liquid-liquid phase separation driven by intrinsically disordered proteins and associated nucleic acids, recently recognized as a fundamental mechanism in a wide range of molecular interactions in biology (for review, see Hyman et al. 2014). Several of the RBPs that interact with Xist RNA have intrinsically disordered regions, and thus have the potential to drive formation of liquid-liquid phase separated condensates. This mechanism has been proposed to be important for Xist RNA function, potentially impacting on Xist-mediated silencing or Xist RNA localization (Cerase et al. 2019). A second important challenge is to define the role of key factors/pathways in Xist-mediated silencing versus local accumulation of Xist RNA in cis. Previously these were considered to be mechanistically separable processes (Wutz et al. 2002), but as indicated above (and see Fig. 4), there are emerging examples of factors that have dual roles both in silencing and Xist RNA localization. A third important challenge will be to better understand how pathways that function in the establishment of silencing link to pathways for maintenance of X inactivation, for example, CpG island methylation and deposition of macroH2A, and similarly how silencing and maintenance pathways determine the unique topological organization of Xi. Finally, it has been shown that Xist-dependent silencing occurs only within a restricted window of opportunity in early development (Wutz and Jaenisch 2000), and yet the major silencing pathways discussed herein apparently function throughout development and in adult tissues. Determining why Xist RNA functions, in large part only in early developmental cell types, presents an intriguing and important future challenge.Inspired by the progress toward understanding chromosome silencing by Xist RNA it will be interesting to determine whether there are other examples of genetic regulation using similar mechanisms. In a recent report Airn and Kcnq1ot1 loci, which are required for parental imprinting of gene clusters on mouse chromosome 17 and chromosome 7, respectively, were found to function in long-range recruitment of Polycomb complexes through a mechanism that is reminiscent of Xist (Schertzer et al. 2019). Specifically, lncRNAs derived from the Airn and Kcnq1ot1 loci induce Polycomb-dependent chromatin modifications in cis over domains spanning several mega-bases in trophoblast stem cells in an hnRNPK-dependent pathway. The extent of gene silencing and Polycomb modifications correlates with topological proximity to the lncRNA locus, pre-existing genome architecture, and the abundance of the lncRNA itself. These examples may represent only the tip of the iceberg as many lncRNAs have been implicated in gene silencing by associating with repressive histone-modifying complexes (Khalil et al. 2009; Guttman et al. 2011; Guttman and Rinn 2012).

Future perspectives

Recent years have seen significant progress toward understanding how Xist RNA establishes chromosome-wide silencing. Chromatin modifications linked to SPEN/HDAC3 and hnRNPK/PRC1 provide a compelling explanation for classical features of Xi chromatin, namely, histone hypoacetylation and deposition of H2AK119ub1 and H3K27me3 histone modifications, all of which occur commensurate with the onset of Xist RNA expression (Nesterova et al. 2019; Żylicz et al. 2019). The basis for other classical Xi features linked to establishment of Xist-mediated silencing, specifically depletion of RNAPII and loss of H3K4me2/3 (Chaumeil et al. 2006), remain to be fully resolved. One possibility is that these are indirect consequences of a repressive chromatin state linked to the aforementioned histone modifications, and negative feedback mechanisms such as inhibition of H3K4me3 by PRC2-mediated H3K27me3 (Schmitges et al. 2011). Alternatively, more direct mechanisms may play a role, supported by the findings that the SPEN SPOC domain interacts with RNAPII/associated cofactors (Dossin et al. 2020), and both SPEN and RBM15 have direct interactions with complexes that catalyze H3K4me3 (Lee and Skalnik 2012; Oswald et al. 2016; Coker et al. 2020). The mechanistic basis for the relatively minor contribution of RBM15/m6A and LBR in establishment of silencing is at present a matter of debate but may relate to known roles for the RBM15/m6A pathway in regulating mRNA dynamics, and the role of LBR in associating the Xi with the nuclear periphery. Figure 4 summarizes key findings discussed in this review.

Pathways for Xist-mediated silencing. Schematic illustrates Xist RNA with tandem repeats A-F indicated together with associated RBPs and downstream effectors implicated in chromosome silencing. SPEN is linked to promoter/enhancer histone deacetylation through NCoR–HDAC3. hnRNPK recruits PCGF3/5–PRC1, which catalyzes widespread deposition of H2AK119ub1. H2AK119ub1 provides a binding platform for other variant (v) PRC1 complexes and PRC2, with PRC2-mediated H3K27me3, then recruiting canonical (c) PRC1. RBM15 is linked to m6A deposition on Xist RNA via recruitment of the METTL3/14 complex, facilitating silencing through yet to be determined mechanisms. LBR links Xist/Xi with the nuclear envelope and the repressive Lamin-associated environment. Other putative pathways, indicated in gray with dashed gray arrows, are RBM15-SET1B and/or SPEN–KMT2D potentially contributing to loss of H3K4me3 on Xi, SPEN–NuRD, potentially contributing to Xi histone deacetylation, and direct inhibitory interaction of SPEN with RNAPII/cofactors. Circular arrows denote catalytic activity. Both SPEN and PRC1 also contribute to anchoring Xist RNA to the nuclear matrix (dashed pink/green arrows), which may in turn contribute to their role in silencing.

As the field has progressed toward defining the pathways for Xist-mediated silencing, new frontiers and challenges have begun to open up. A notable example is the possible role of liquid-liquid phase separation driven by intrinsically disordered proteins and associated nucleic acids, recently recognized as a fundamental mechanism in a wide range of molecular interactions in biology (for review, see Hyman et al. 2014). Several of the RBPs that interact with Xist RNA have intrinsically disordered regions, and thus have the potential to drive formation of liquid-liquid phase separated condensates. This mechanism has been proposed to be important for Xist RNA function, potentially impacting on Xist-mediated silencing or Xist RNA localization (Cerase et al. 2019). A second important challenge is to define the role of key factors/pathways in Xist-mediated silencing versus local accumulation of Xist RNA in cis. Previously these were considered to be mechanistically separable processes (Wutz et al. 2002), but as indicated above (and see Fig. 4), there are emerging examples of factors that have dual roles both in silencing and Xist RNA localization. A third important challenge will be to better understand how pathways that function in the establishment of silencing link to pathways for maintenance of X inactivation, for example, CpG island methylation and deposition of macroH2A, and similarly how silencing and maintenance pathways determine the unique topological organization of Xi. Finally, it has been shown that Xist-dependent silencing occurs only within a restricted window of opportunity in early development (Wutz and Jaenisch 2000), and yet the major silencing pathways discussed herein apparently function throughout development and in adult tissues. Determining why Xist RNA functions, in large part only in early developmental cell types, presents an intriguing and important future challenge.

Inspired by the progress toward understanding chromosome silencing by Xist RNA it will be interesting to determine whether there are other examples of genetic regulation using similar mechanisms. In a recent report Airn and Kcnq1ot1 loci, which are required for parental imprinting of gene clusters on mouse chromosome 17 and chromosome 7, respectively, were found to function in long-range recruitment of Polycomb complexes through a mechanism that is reminiscent of Xist (Schertzer et al. 2019). Specifically, lncRNAs derived from the Airn and Kcnq1ot1 loci induce Polycomb-dependent chromatin modifications in cis over domains spanning several mega-bases in trophoblast stem cells in an hnRNPK-dependent pathway. The extent of gene silencing and Polycomb modifications correlates with topological proximity to the lncRNA locus, pre-existing genome architecture, and the abundance of the lncRNA itself. These examples may represent only the tip of the iceberg as many lncRNAs have been implicated in gene silencing by associating with repressive histone-modifying complexes (Khalil et al. 2009; Guttman et al. 2011; Guttman and Rinn 2012).
