---
pmid: '25383541'
title: The mitotic checkpoint complex binds a second CDC20 to inhibit active APC/C.
authors:
- Izawa D
- Pines J
journal: Nature
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4312099
doi: 10.1038/nature13911
---

# The mitotic checkpoint complex binds a second CDC20 to inhibit active APC/C.
**Authors:** Izawa D, Pines J
**Journal:** Nature (2015)
**DOI:** [10.1038/nature13911](https://doi.org/10.1038/nature13911)
**PMC:** [PMC4312099](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4312099/)

## Abstract

1. Nature. 2015 Jan 29;517(7536):631-4. doi: 10.1038/nature13911. Epub 2014 Nov
12.

The mitotic checkpoint complex binds a second CDC20 to inhibit active APC/C.

Izawa D(1), Pines J(1).

Author information:
(1)The Gurdon Institute and Department of Zoology, Tennis Court Road, Cambridge 
CB2 1QN, UK.

The spindle assembly checkpoint (SAC) maintains genomic stability by delaying 
chromosome segregation until the last chromosome has attached to the mitotic 
spindle. The SAC prevents the anaphase promoting complex/cyclosome (APC/C) 
ubiquitin ligase from recognizing cyclin B and securin by catalysing the 
incorporation of the APC/C co-activator, CDC20, into a complex called the 
mitotic checkpoint complex (MCC). The SAC works through unattached kinetochores 
generating a diffusible 'wait anaphase' signal that inhibits the APC/C in the 
cytoplasm, but the nature of this signal remains a key unsolved problem. 
Moreover, the SAC and the APC/C are highly responsive to each other: the APC/C 
quickly targets cyclin B and securin once all the chromosomes attach in 
metaphase, but is rapidly inhibited should kinetochore attachment be perturbed. 
How this is achieved is also unknown. Here, we show that the MCC can inhibit a 
second CDC20 that has already bound and activated the APC/C. We show how the MCC 
inhibits active APC/C and that this is essential for the SAC. Moreover, this 
mechanism can prevent anaphase in the absence of kinetochore signalling. Thus, 
we propose that the diffusible 'wait anaphase' signal could be the MCC itself, 
and explain how reactivating the SAC can rapidly inhibit active APC/C.

DOI: 10.1038/nature13911
PMCID: PMC4312099
PMID: 25383541 [Indexed for MEDLINE]

## Full Text

The MCC is an APC/C inhibitor containing the MAD2, BUBR1 and BUB3 checkpoint proteins in a complex with CDC20 5, where MAD2 and BUBR1 inhibit CDC20 by binding to substrate and APC/C recognition motifs6-8. To elucidate how the SAC inhibits the APC/C we produced recombinant human MCC (rMCC) by co-expressing His6-tagged MAD2, Streptavidin Binding Protein (SBP)-tagged-BUBR1 and untagged CDC20 at a 8:1:2 ratio (Extended Data Fig. 1a-e) in baculovirus-infected Sf9 cells. We co-purified MAD2, BUBR1 and CDC20 in a ‘core MCC’ complex at a 1:1:1 ratio (Extended Data Fig. 1b).

Incubating core rMCC with recombinant His6-tagged CDC20 showed that core MCC could bind a second CDC20 molecule (Fig. 1a & Extended Data Fig. 1f), which was not because CDC20 homodimerised (Fig 1a). NB: including BUB3 in the core rMCC made no difference to the amount of CDC20 that was bound (Extended Data Fig. 2). We note here that Primorac and Musacchio recently speculated that the MCC may contain two molecules of CDC20 9. The mode of binding to the second CDC20 differed from that required to form the core MCC because core MCC could bind to a CDC20ΔKILR mutant unable to bind MAD2 8 (Fig. 1a and Extended Data Fig. 1c). This also excluded the possibility that the second CDC20 had exchanged with CDC20 in the core MCC.

The question arose as to why we did not purify rMCC with two molecules of CDC20. We postulated that the second CDC20 bound less stably than the first CDC20, which is cooperatively bound by MAD2 and BUBR1 6; therefore, limiting amounts of CDC20 would preferentially incorporate into the core MCC. In agreement with this, we purified some core rMCC bound to a second CDC20 from Sf9 cell lysates containing excess CDC20 (50% bound in Extended Data Fig. 1g). We noted that increasing the amount of functional SBPCDC20 enhanced core rMCC binding to the APC/C (Fig. 1b; Extended Data Fig. 1h & i). This indicated that core MCC could bind CDC20 associated with the APC/C, and that core rMCC did not compete with SBPCDC20 for APC/C binding (Fig. 1c). This agreed with our previous finding that the MCC and CDC20 bind to the APC/C through different sites10.

To determine the properties of MCC as an APC/CCDC20 inhibitor we used a reconstituted ubiquitylation assay with APC/C isolated from CDC20-depleted mitotic cells (APC/CΔCDC20), and incubated it with SBPCDC20 and/or core rMCC. Adding CDC20 strongly activated the APC/C, whereas, as expected6,8, core MCC alone only weakly stimulated the APC/C (Fig. 1d). Neither MAD2 nor BUBR1 alone can inhibit the mitotic APC/C 11, and together they require pre-incubation to inhibit interphase APC/CCDC20 (reference 7). By contrast, core MCC was a potent and rapid inhibitor of active APC/CCDC20: as well as preventing CDC20 from activating the APC/C (Fig. 1d; Extended Data Fig. 3a & b), it inhibited active mitotic APC/C within 10 minutes (Fig. 1e - g; Extended Data Fig. 3c).

To gain insight into how core MCC could inhibit active APC/CCDC20, we sought to identify how core MCC bound to a second CDC20. Studies on yeast MAD3/BUBR1 had implicated a number of D-box and KEN box motifs in binding to CDC20, and as important for the SAC 12,13. A D-box bound to the side of the CDC20 ß-propeller domain in the MCC structure, whereas a KEN box bound to the top face 6. We hypothesised that the second CDC20 might bind to the core MCC in a similar manner; therefore, we made mutations in the D-box receptor (D177A; ΔDR), and the KEN-box receptor (N329A/N331A/T377A/R445A; ΔKR) of CDC20. Both these CDC20 mutants bound much less well to core rMCC in vitro (Fig. 2a & b). Since the ΔDR mutant could still be incorporated into the core MCC (Fig. 2c), we tested whether inhibiting a second CDC20 was important for the SAC (Fig. 2d). We replaced endogenous CDC20 with the ΔDR mutant, or the ΔKR mutant as a positive control, and assayed the ability of cells to arrest in response to nocodazole. As expected, the ΔKR mutant abrogated the SAC because it could not form the core MCC (Fig. 2c-e). By contrast, the ΔDR mutant assembled into the core MCC and bound to the APC/C (Fig. 2c & d), yet the SAC was still defective (Fig. 2e). Cells expressing the ΔDR mutant, however, took more time to exit mitosis than those expressing the ΔKR mutant (Fig. 2e). We thought this might be because the ΔDR mutant was less effective at activating the APC/C 14; consistent with this, Cyclin B1 was degraded more slowly in these cells (Extended Data Fig. 4). These data supported the idea that the MCC inhibited a second CDC20 as part of a functional SAC.

Since CDC20 required its D-box and KEN box receptors to bind the core MCC, we identified the D-box and a KEN box on BUBR1 responsible for binding CDC20. The structure of the core MCC implicated a putative D-box6, but BUBR1 has two KEN-boxes: the first (K26EN) is essential to form the core MCC (Extended Data Fig. 1d), whereas the second (K304EN) is not required to form the core MCC but is still important for the SAC 12,13,15,16. We thought the second KEN box a more likely candidate to bind a second CDC20 9; therefore, we mutated the putative D-box (R224xxL: ΔD-box) and the second KEN-box (ΔKEN2) in human BUBR1. Both mutants were incorporated into the core MCC in vitro (Fig. 3a; Extended data Fig. 5a) and in vivo (Fig. 3c); both inhibited the CDC20 within the core MCC (Extended data Fig. 5b), but reduced binding to a second molecule of CDC20 (Fig. 3a & b). (NB: BUBR1 alone did not bind two molecules of CDC20 because neither the D-box nor the second KEN-box was required to bind CDC20 in the absence of MAD2 (Extended Data Fig. 5c).) Furthermore, replacing endogenous BUBR1 with the ΔD-box mutant (Fig. 3c) prevented cells from arresting in mitosis in response to either nocodazole (Fig. 3d), or Taxol where the SAC is much weaker 17 (Extended data Fig. 5d). Thus, the core MCC must inhibit a second CDC20 molecule to impose a functional SAC.

An important test of our idea that the core MCC inhibited active APC/CCDC20 was whether the core MCC could arrest a mitotic cell in which kinetochores could not catalyse further CDC20 incorporation into the core MCC (see Extended Data Fig. 6a). To prevent the core MCC from disassembling we attached a YFP tag to MAD2 (Venus-MAD2) and a GFP-binding domain (GBP)18 to CDC20 (GBP-CDC20). We called this stable complex MCCM2 (see Extended Data Fig. 6 and 7); a similar approach using leucine zippers had been used in budding yeast19. We expressed MCCM2 in cells with normal levels of endogenous CDC20. MCCM2 was able to inhibit the APC/C when the SAC was inactivated in three different ways. i) MCCM2 imposed a metaphase delay (Fig. 4a), in which the kinetochores did not stain for MAD2 (Extended Data Fig. 8a). The extent of the delay correlated with the amount of GBP-CDC20, and thus the amount of MCCM2 (Extended Data Fig. 8b). ii) MCCM2 imposed a delay in cells treated with the Mps1 kinase inhibitor Reversine to prevent core MCC assembly 20 (Fig. 4b). iii) MCCM2 arrested cells in mitosis after depleting the Knl1 kinetochore protein that is required for the SAC21 (Extended Data Fig. 8c-e). These data supported our idea that the core MCC inhibited active APC/CCDC20. Moreover, as the MCC inhibits the APC/C without further signalling from the kinetochores, it has one of the essential properties required of the diffusible wait anaphase inhibitor, although our data do not prove that it is the diffusible inhibitor in vivo.

All the functional components of the core MCC were required for MCCM2 to inhibit APC/CCDC20 because we could not delay cells in mitosis when we stabilised the binding between MAD2 and CDC20 in the absence of BUBR1 (Fig. 4c); nor when we stabilised MAD2 with a CDC20-ΔKILR mutant that cannot form the core MCC (Extended Data Fig. 9a). Finally, we stabilised the binding between MAD2 and CDC20 (MCCM2), but replaced BUBR1 with the ΔD-box mutant to perturb binding to a second CDC20. These complexes were much less effective at inhibiting APC/CCDC20
in vitro (Extended Data Fig. 9b), and unable to delay cells in mitosis (Fig. 4c; model in Extended Data Fig. 9c). Thus, we conclude that to arrest cells in mitosis the core MCC inhibits a second molecule of CDC20 that can even be part of an active APC/CCDC20.

Crucial gaps have remained in our understanding of the SAC: notably, how the wait anaphase signal generated at unattached kinetochores inhibits APC/C activity in the rest of the cell2. Unattached kinetochores appear to catalyse a conformational change in MAD2 to bind CDC20 22 and subsequently promote APC/C-MCC formation in the cytoplasm. However, it is unlikely that all CDC20 could be bound by MAD2 at the kinetochore, therefore additional mechanisms have been proposed to prevent the activation of the APC/C, including cytoplasmic amplification of MAD2-CDC20 binding22, although this now appears unlikely19,23, and phosphorylation of CDC20 by BUB124. We now show how the MCC, formed either at kinetochores or in the cytoplasm, could act as a diffusible inhibitor to inhibit APC/CCDC20 throughout the cell (Extended Data Fig. 10), although our data do not prove that it disseminates the wait anaphase signal in vivo. Han and colleagues7 proposed that the complex between MAD2 and CDC20 will template the formation of the BBC (BUBR1-BUB3-CDC20) complex25 to inhibit CDC20 - although in these experiments p31Comet was depleted, which would alter the levels and behaviour of checkpoint complexes 25-27. While we also find that the BBC is an abundant APC/C inhibitor in cells 27,28, we show here that stabilising the MCC generates a more potent inhibitor than stabilising the BBC (Fig. 4a; MCCR1 see Extended Data Fig. 6b), which agrees with the observation that cells containing a greater proportion of MCC over BBC exhibit stronger SAC activity 17,25. Our results could also solve a further conundrum posed by the SAC. MAD2 and the APC/C bind to the same KILR motif on CDC20 8; therefore, CDC20 must dissociate from the APC/C to bind MAD2. By analogy with measurements on Cdh1 29, CDC20 is predicted to dissociate slowly from the APC/C (t1/2 ~25 min), yet reactivating the SAC can inhibit active APC/C in less than 5 minutes 3,4. Our finding that MCC rapidly inhibits CDC20 already bound to the APC/C can help to explain the close temporal coupling between the SAC and the APC/C. Indeed, our data indicate that the MCC prefers CDC20 that is already bound to the APC/C; the reason for this will be important to determine in the future.

MethodsCell culture and synchronizationHeLa cells were maintained in Advanced D-MEM with 10% FBS. For synchronisation at the beginning of S phase, HeLa cells were treated with 2.5 mM thymidine as previously described8. For prometaphase, cells were released from a thymidine block and 6 hours later treated with nocodazole at a final concentration of 0.33 μM for 6-12 hrs. For SAC inactivated samples, cells were released from a nocodazole block into medium including 1 μM reversine and 10 μM MG132 for a further 1 hour.Transfection siRNA and DNAThe following ON-TARGETplus (Dharmacon, CO, USA) oligos as previously described8 were used: CDC20 50 nM (CGGAAGACCUGCCGUUACAUU); MAD2 20 nM (GGAAGAGUCGGGACCACAGUU); BUBR1 50 nM (GAUGGUGAAUUGUGGAAUA); BLINKIN 50 nM (AAGAUCUGAUUAAGGAUCCACGAAA) and GAPDH (D-001830-01). Cells were transfected with siRNA oligos once or twice at the indicated concentrations using lipofectamine RNAiMax (Invitrogen). To transfect siRNA oligos and DNA plasmids at the same time, cells were treated with lipofectamine 2000 (Invitrogen). An siRNA-resistant of resistant open reading frame (ORF) of BUBR1 is generated by mutating underlined nucleotides (GATGGCGAGCTTUGGAAUA).In vitro reconstituted ubiquitylation assayIn vitro ubiquitylation assays were performed as described previously8 but with modifications to use a fluorescently-labelled substrate developed by Dr Takahiro Matsusaka. Briefly, CDC20 was depleted by siRNA treatment for 48 hr before the APC/C was purified with anti-APC3 (AF3.1) antibody from mitotic HeLa cell extract. Immunoprecipitates were resuspended in ubiquitylation reaction buffer contained E1-ligase, UbcH10 (E2), ubiquitin, ATP, ATP regenerating system, and fluorescently-labelled securin as a substrate in QA buffer (100 mM NaCl, 30 mM Hepes pH 7.8, 2 mM ATP, 2 mM MgCl2, 0.1 μg/μl BSA, 1 mM DTT) at 37°C for the indicated time, and supplied with recombinant CDC20 and/or core rMCC as indicated. Recombinant securin protein was labelled with IRDye680 dye (IRDye 680LT Maleimide Infrared Dye: LiCOR) according to the manufacturers instructions and directly scanned with a Li-COR Odyssey CCD scanner after SDS-PAGE analysis. Ubiquitylation of CDC20, MAD2 and BUBR1 were analysed by quantitative immunoblotting. After blotting with primary antibodies, blots were incubated with fluorescently labelled secondary antibodies and the fluorescence measured using a LI-COR Odyssey CCD scanner according to the manufacturer’s instructions (LI-COR Biosciences, NE, USA).Expression of mCherry-GBP-CDC20, Venus-BUBR1 and Venus-MAD2We used two types of human expression vectors: pcDNA5-3Flag-Venus (inducible CMV promoter) and pmCherry-CAG-C1 (chicken ß-actin promoter). In the pcDNA5-3Flag-Venus, 3Flag-Venus is inserted into the multiple-cloning site of pcDNA5/FRT/TO (Invitrogen). In the pmCherry-CAG-C1 vector, EYFP and CMV promoter of pEYFP-C1 (Clontech) were replaced by mCherry and CAG promoter, respectively. An siRNA-resistant open reading frame (ORF) of CDC20 and GBP-CDC20 were cloned into the pmCherry-CAG-C1 vector, and MAD2 and BUBR1 were cloned into pcDNA5-3Flag-Venus. All constructs were verified by sequencing and sequences are available on request. To co-express mCherry-GBP-CDC20 and Venus-MAD2 or Venus-BUBR1, the indicated plasmids were co-transfected with the indicated siRNA oligos using Lipolipofectamine 2000 (Invitrogen).Inducible cell linesTo generate cell lines expressing 3Flag-CDC20 or 3Flag-Cerulean-BUBR1 proteins from an inducible promoter, siRNA resistant ORF of CDC20 or BUBR1 was cloned into a modified version of pcDNA5/FRT/TO (Invitrogen). Those plasmids were transfected into a HeLa-FRT cell line (gift of Stephen Taylor, University of Manchester) and stable cell lines were generated using the FLIP-in system (Invitrogen). To obtain a cell line expressing 3Flag-Venus-MAD2 from an inducible CMV promoter and mCherry-GBP-CDC20 from a constitutive CMV promoter (used in Fig. 4b), a HeLa-FRT cell line expressing an inducible 3Flag-Venus-MAD2 was transfected with the pmCherry-C1-GBP-CDC20 plasmid and selected with Geneticin (Invitrogen). To induce proteins from the inducible promoter, cells were treated with tetracycline (1 μg/ml, Calbiochem) 36 hr before analysis.Immunoprecipitation and size exclusion chromatographyCells for immunoprecipitation were lysed with HEPES buffer (150 mM KCl, 20 mM Hepes pH 7.8, 10 mM EDTA, 10 % Glycerol, 0.2 % NP-40, 1 mM DTT, Roche complete inhibitor cocktail tablet, 0.2 μM microcystin, 1 mM PMSF) for 10 min on ice and clarified by a 20000 × g spin for 10 min. Protein complexes were immunoprecipitated with antibodies (anti-APC4, anti-APC3 (AF3.1)), anti-GFP or anti-flag M2 epitope) covalently coupled to Protein G Dynabeads (Invitrogen) using HEPES buffer for incubation and washing. For size exclusion chromatography analysis, cells were pelleted then resuspended in buffer A (140 mM NaCl, 20 mM Hepes pH 7.8, 6 mM MgCl2, 5% glycerol, 1 mM dithiothreitol (DTT), Roche complete inhibitor cocktail tablet, 0,2 μM microcystin, 1 mM PMSF) at a 1:1 ratio of buffer to cells, and lysed by nitrogen cavitation (1000 PSI, 30 min, Parr Instruments, USA). Lysed cells were centrifuged at 20,000 × g for 10 min and 259,000 × g for 10 min before loading onto a Superose 6 PC 3.2/30 column (GE Healthcare). The column was run at a flow rate of 25 μl/min−1 in buffer B (140 mM NaCl, 30 mM Hepes 7.8, 5% glycerol, 1 mM DTT) and 50 μl fractions collected.EpifluorescenceCells were seeded into 8-well dishes (Thistle Scientific, UK) to enable experiments to be performed in parallel. Before imaging, the culture medium was replaced with Leibovitz’s L-15 medium (Gibco Life Technologies, UK) supplemented with 10% foetal bovine serum and penicillin/streptomycin. Cells were imaged on a DeltaVision micrososcope equipped with an environmental chamber at 37°C (API, USA) with a QuantEM camera (Photometrics, USA) and Lambda LS illumination (Sutter, USA) as previously described 10, or a spinning disc microscope (Intelligent Imaging Innovations, Colorado, USA) equipped with a CSU-X1 head (Yokogawa, Japan) and a QuantEM:512sc EMCCD camera (Photometrics, USA). In fig. 2e, 3d and fig.4, images of DIC and fluorescence were captured at 6 min intervals and the fluorescence intensities were measured and analysed using ImageJ/Fiji software as previously described 10.AntibodiesThe following antibodies were used at the indicated dilutions. CDC20 (sc-13162, Santa Cruz Biotechnology) 1:500; CDC20 (A301-180A, Bethyl laboratories) 1:500; BUBR1 (612503, BD transduction laboratories); BUBR1 (A300-386A, Bethyl laboratories) 1:500; MAD2 (610679, BD transduction laboratories) 1:500; MAD2 (A300-301A, Bethyl Laboratories) 1:500; BUB3 (611730, BD Transduction Laboratories) 1:500; APC3 (610455, BD Transduction Laboratories) 1:500; APC4 (monoclonal antibody raised against a C-terminal peptide) 1:500; BLINKIN (a kind gift from M. Yanagida and T. Kiyomitsu) 1:50; anti-myc-epitope (9E10, Santa Cruz Biotechnology) 1:500; anti-flag epitope (M2, Sigma) 1:5000; anti-GFP (Clone 3.1 and 7.1, Roche) 1:200.Secondary antibodies: IRDye 680CW donkey anti mouse (926-68072, LI-COR), IRDye 800CW donkey anti mouse (926-32212, LI-COR); IRDye 680CW donkey anti rabbit (926-32223, LI-COR); IRDye 800CW donkey anti rabbit (926-32213, LI-COR) were all used at 1:10000.ImmunofluorescenceCells were fixed in 4% paraformaldehyde and 2% sucrose for 5 minutes. After fixation cells were blocked in 3% BSA-PBS NP-40 0.2% and then incubated with antibodies. All antibodies were diluted in 3% BSA-PBS NP-40 0.2% and washes were performed with PBS NP-40 0.2%. Anti-GFP (Roche), Anti-MAD2 (A300-301A, Bethyl Laboratories), anti-flag M2 (sigma), anti-CDC20 (sc-13162, Santa Cruz Biotechnology) and anti-BUBR1 (A300-301A, Bethyl Laboratories) antibodies were used at 1:400 or 1:200, respectively. Anti-ACA serum (a kind gift from W. Earnshaw, University of Edinburgh, UK) was used at 1:20000. Secondary antibodies conjugated to Alexa Fluor 488, Alexa Fluor 568 or Alexa Fluor 647 (Molecular Probes) were diluted 1:400. DNA was stained with Hoechst-33342.

Cell culture and synchronizationHeLa cells were maintained in Advanced D-MEM with 10% FBS. For synchronisation at the beginning of S phase, HeLa cells were treated with 2.5 mM thymidine as previously described8. For prometaphase, cells were released from a thymidine block and 6 hours later treated with nocodazole at a final concentration of 0.33 μM for 6-12 hrs. For SAC inactivated samples, cells were released from a nocodazole block into medium including 1 μM reversine and 10 μM MG132 for a further 1 hour.

Cell culture and synchronization

HeLa cells were maintained in Advanced D-MEM with 10% FBS. For synchronisation at the beginning of S phase, HeLa cells were treated with 2.5 mM thymidine as previously described8. For prometaphase, cells were released from a thymidine block and 6 hours later treated with nocodazole at a final concentration of 0.33 μM for 6-12 hrs. For SAC inactivated samples, cells were released from a nocodazole block into medium including 1 μM reversine and 10 μM MG132 for a further 1 hour.

Transfection siRNA and DNAThe following ON-TARGETplus (Dharmacon, CO, USA) oligos as previously described8 were used: CDC20 50 nM (CGGAAGACCUGCCGUUACAUU); MAD2 20 nM (GGAAGAGUCGGGACCACAGUU); BUBR1 50 nM (GAUGGUGAAUUGUGGAAUA); BLINKIN 50 nM (AAGAUCUGAUUAAGGAUCCACGAAA) and GAPDH (D-001830-01). Cells were transfected with siRNA oligos once or twice at the indicated concentrations using lipofectamine RNAiMax (Invitrogen). To transfect siRNA oligos and DNA plasmids at the same time, cells were treated with lipofectamine 2000 (Invitrogen). An siRNA-resistant of resistant open reading frame (ORF) of BUBR1 is generated by mutating underlined nucleotides (GATGGCGAGCTTUGGAAUA).

Transfection siRNA and DNA

The following ON-TARGETplus (Dharmacon, CO, USA) oligos as previously described8 were used: CDC20 50 nM (CGGAAGACCUGCCGUUACAUU); MAD2 20 nM (GGAAGAGUCGGGACCACAGUU); BUBR1 50 nM (GAUGGUGAAUUGUGGAAUA); BLINKIN 50 nM (AAGAUCUGAUUAAGGAUCCACGAAA) and GAPDH (D-001830-01). Cells were transfected with siRNA oligos once or twice at the indicated concentrations using lipofectamine RNAiMax (Invitrogen). To transfect siRNA oligos and DNA plasmids at the same time, cells were treated with lipofectamine 2000 (Invitrogen). An siRNA-resistant of resistant open reading frame (ORF) of BUBR1 is generated by mutating underlined nucleotides (GATGGCGAGCTTUGGAAUA).

In vitro reconstituted ubiquitylation assayIn vitro ubiquitylation assays were performed as described previously8 but with modifications to use a fluorescently-labelled substrate developed by Dr Takahiro Matsusaka. Briefly, CDC20 was depleted by siRNA treatment for 48 hr before the APC/C was purified with anti-APC3 (AF3.1) antibody from mitotic HeLa cell extract. Immunoprecipitates were resuspended in ubiquitylation reaction buffer contained E1-ligase, UbcH10 (E2), ubiquitin, ATP, ATP regenerating system, and fluorescently-labelled securin as a substrate in QA buffer (100 mM NaCl, 30 mM Hepes pH 7.8, 2 mM ATP, 2 mM MgCl2, 0.1 μg/μl BSA, 1 mM DTT) at 37°C for the indicated time, and supplied with recombinant CDC20 and/or core rMCC as indicated. Recombinant securin protein was labelled with IRDye680 dye (IRDye 680LT Maleimide Infrared Dye: LiCOR) according to the manufacturers instructions and directly scanned with a Li-COR Odyssey CCD scanner after SDS-PAGE analysis. Ubiquitylation of CDC20, MAD2 and BUBR1 were analysed by quantitative immunoblotting. After blotting with primary antibodies, blots were incubated with fluorescently labelled secondary antibodies and the fluorescence measured using a LI-COR Odyssey CCD scanner according to the manufacturer’s instructions (LI-COR Biosciences, NE, USA).

In vitro reconstituted ubiquitylation assay

In vitro ubiquitylation assays were performed as described previously8 but with modifications to use a fluorescently-labelled substrate developed by Dr Takahiro Matsusaka. Briefly, CDC20 was depleted by siRNA treatment for 48 hr before the APC/C was purified with anti-APC3 (AF3.1) antibody from mitotic HeLa cell extract. Immunoprecipitates were resuspended in ubiquitylation reaction buffer contained E1-ligase, UbcH10 (E2), ubiquitin, ATP, ATP regenerating system, and fluorescently-labelled securin as a substrate in QA buffer (100 mM NaCl, 30 mM Hepes pH 7.8, 2 mM ATP, 2 mM MgCl2, 0.1 μg/μl BSA, 1 mM DTT) at 37°C for the indicated time, and supplied with recombinant CDC20 and/or core rMCC as indicated. Recombinant securin protein was labelled with IRDye680 dye (IRDye 680LT Maleimide Infrared Dye: LiCOR) according to the manufacturers instructions and directly scanned with a Li-COR Odyssey CCD scanner after SDS-PAGE analysis. Ubiquitylation of CDC20, MAD2 and BUBR1 were analysed by quantitative immunoblotting. After blotting with primary antibodies, blots were incubated with fluorescently labelled secondary antibodies and the fluorescence measured using a LI-COR Odyssey CCD scanner according to the manufacturer’s instructions (LI-COR Biosciences, NE, USA).

Expression of mCherry-GBP-CDC20, Venus-BUBR1 and Venus-MAD2We used two types of human expression vectors: pcDNA5-3Flag-Venus (inducible CMV promoter) and pmCherry-CAG-C1 (chicken ß-actin promoter). In the pcDNA5-3Flag-Venus, 3Flag-Venus is inserted into the multiple-cloning site of pcDNA5/FRT/TO (Invitrogen). In the pmCherry-CAG-C1 vector, EYFP and CMV promoter of pEYFP-C1 (Clontech) were replaced by mCherry and CAG promoter, respectively. An siRNA-resistant open reading frame (ORF) of CDC20 and GBP-CDC20 were cloned into the pmCherry-CAG-C1 vector, and MAD2 and BUBR1 were cloned into pcDNA5-3Flag-Venus. All constructs were verified by sequencing and sequences are available on request. To co-express mCherry-GBP-CDC20 and Venus-MAD2 or Venus-BUBR1, the indicated plasmids were co-transfected with the indicated siRNA oligos using Lipolipofectamine 2000 (Invitrogen).

Expression of mCherry-GBP-CDC20, Venus-BUBR1 and Venus-MAD2

We used two types of human expression vectors: pcDNA5-3Flag-Venus (inducible CMV promoter) and pmCherry-CAG-C1 (chicken ß-actin promoter). In the pcDNA5-3Flag-Venus, 3Flag-Venus is inserted into the multiple-cloning site of pcDNA5/FRT/TO (Invitrogen). In the pmCherry-CAG-C1 vector, EYFP and CMV promoter of pEYFP-C1 (Clontech) were replaced by mCherry and CAG promoter, respectively. An siRNA-resistant open reading frame (ORF) of CDC20 and GBP-CDC20 were cloned into the pmCherry-CAG-C1 vector, and MAD2 and BUBR1 were cloned into pcDNA5-3Flag-Venus. All constructs were verified by sequencing and sequences are available on request. To co-express mCherry-GBP-CDC20 and Venus-MAD2 or Venus-BUBR1, the indicated plasmids were co-transfected with the indicated siRNA oligos using Lipolipofectamine 2000 (Invitrogen).

Inducible cell linesTo generate cell lines expressing 3Flag-CDC20 or 3Flag-Cerulean-BUBR1 proteins from an inducible promoter, siRNA resistant ORF of CDC20 or BUBR1 was cloned into a modified version of pcDNA5/FRT/TO (Invitrogen). Those plasmids were transfected into a HeLa-FRT cell line (gift of Stephen Taylor, University of Manchester) and stable cell lines were generated using the FLIP-in system (Invitrogen). To obtain a cell line expressing 3Flag-Venus-MAD2 from an inducible CMV promoter and mCherry-GBP-CDC20 from a constitutive CMV promoter (used in Fig. 4b), a HeLa-FRT cell line expressing an inducible 3Flag-Venus-MAD2 was transfected with the pmCherry-C1-GBP-CDC20 plasmid and selected with Geneticin (Invitrogen). To induce proteins from the inducible promoter, cells were treated with tetracycline (1 μg/ml, Calbiochem) 36 hr before analysis.

Inducible cell lines

To generate cell lines expressing 3Flag-CDC20 or 3Flag-Cerulean-BUBR1 proteins from an inducible promoter, siRNA resistant ORF of CDC20 or BUBR1 was cloned into a modified version of pcDNA5/FRT/TO (Invitrogen). Those plasmids were transfected into a HeLa-FRT cell line (gift of Stephen Taylor, University of Manchester) and stable cell lines were generated using the FLIP-in system (Invitrogen). To obtain a cell line expressing 3Flag-Venus-MAD2 from an inducible CMV promoter and mCherry-GBP-CDC20 from a constitutive CMV promoter (used in Fig. 4b), a HeLa-FRT cell line expressing an inducible 3Flag-Venus-MAD2 was transfected with the pmCherry-C1-GBP-CDC20 plasmid and selected with Geneticin (Invitrogen). To induce proteins from the inducible promoter, cells were treated with tetracycline (1 μg/ml, Calbiochem) 36 hr before analysis.

Immunoprecipitation and size exclusion chromatographyCells for immunoprecipitation were lysed with HEPES buffer (150 mM KCl, 20 mM Hepes pH 7.8, 10 mM EDTA, 10 % Glycerol, 0.2 % NP-40, 1 mM DTT, Roche complete inhibitor cocktail tablet, 0.2 μM microcystin, 1 mM PMSF) for 10 min on ice and clarified by a 20000 × g spin for 10 min. Protein complexes were immunoprecipitated with antibodies (anti-APC4, anti-APC3 (AF3.1)), anti-GFP or anti-flag M2 epitope) covalently coupled to Protein G Dynabeads (Invitrogen) using HEPES buffer for incubation and washing. For size exclusion chromatography analysis, cells were pelleted then resuspended in buffer A (140 mM NaCl, 20 mM Hepes pH 7.8, 6 mM MgCl2, 5% glycerol, 1 mM dithiothreitol (DTT), Roche complete inhibitor cocktail tablet, 0,2 μM microcystin, 1 mM PMSF) at a 1:1 ratio of buffer to cells, and lysed by nitrogen cavitation (1000 PSI, 30 min, Parr Instruments, USA). Lysed cells were centrifuged at 20,000 × g for 10 min and 259,000 × g for 10 min before loading onto a Superose 6 PC 3.2/30 column (GE Healthcare). The column was run at a flow rate of 25 μl/min−1 in buffer B (140 mM NaCl, 30 mM Hepes 7.8, 5% glycerol, 1 mM DTT) and 50 μl fractions collected.

Immunoprecipitation and size exclusion chromatography

Cells for immunoprecipitation were lysed with HEPES buffer (150 mM KCl, 20 mM Hepes pH 7.8, 10 mM EDTA, 10 % Glycerol, 0.2 % NP-40, 1 mM DTT, Roche complete inhibitor cocktail tablet, 0.2 μM microcystin, 1 mM PMSF) for 10 min on ice and clarified by a 20000 × g spin for 10 min. Protein complexes were immunoprecipitated with antibodies (anti-APC4, anti-APC3 (AF3.1)), anti-GFP or anti-flag M2 epitope) covalently coupled to Protein G Dynabeads (Invitrogen) using HEPES buffer for incubation and washing. For size exclusion chromatography analysis, cells were pelleted then resuspended in buffer A (140 mM NaCl, 20 mM Hepes pH 7.8, 6 mM MgCl2, 5% glycerol, 1 mM dithiothreitol (DTT), Roche complete inhibitor cocktail tablet, 0,2 μM microcystin, 1 mM PMSF) at a 1:1 ratio of buffer to cells, and lysed by nitrogen cavitation (1000 PSI, 30 min, Parr Instruments, USA). Lysed cells were centrifuged at 20,000 × g for 10 min and 259,000 × g for 10 min before loading onto a Superose 6 PC 3.2/30 column (GE Healthcare). The column was run at a flow rate of 25 μl/min−1 in buffer B (140 mM NaCl, 30 mM Hepes 7.8, 5% glycerol, 1 mM DTT) and 50 μl fractions collected.

EpifluorescenceCells were seeded into 8-well dishes (Thistle Scientific, UK) to enable experiments to be performed in parallel. Before imaging, the culture medium was replaced with Leibovitz’s L-15 medium (Gibco Life Technologies, UK) supplemented with 10% foetal bovine serum and penicillin/streptomycin. Cells were imaged on a DeltaVision micrososcope equipped with an environmental chamber at 37°C (API, USA) with a QuantEM camera (Photometrics, USA) and Lambda LS illumination (Sutter, USA) as previously described 10, or a spinning disc microscope (Intelligent Imaging Innovations, Colorado, USA) equipped with a CSU-X1 head (Yokogawa, Japan) and a QuantEM:512sc EMCCD camera (Photometrics, USA). In fig. 2e, 3d and fig.4, images of DIC and fluorescence were captured at 6 min intervals and the fluorescence intensities were measured and analysed using ImageJ/Fiji software as previously described 10.

Epifluorescence

Cells were seeded into 8-well dishes (Thistle Scientific, UK) to enable experiments to be performed in parallel. Before imaging, the culture medium was replaced with Leibovitz’s L-15 medium (Gibco Life Technologies, UK) supplemented with 10% foetal bovine serum and penicillin/streptomycin. Cells were imaged on a DeltaVision micrososcope equipped with an environmental chamber at 37°C (API, USA) with a QuantEM camera (Photometrics, USA) and Lambda LS illumination (Sutter, USA) as previously described 10, or a spinning disc microscope (Intelligent Imaging Innovations, Colorado, USA) equipped with a CSU-X1 head (Yokogawa, Japan) and a QuantEM:512sc EMCCD camera (Photometrics, USA). In fig. 2e, 3d and fig.4, images of DIC and fluorescence were captured at 6 min intervals and the fluorescence intensities were measured and analysed using ImageJ/Fiji software as previously described 10.

AntibodiesThe following antibodies were used at the indicated dilutions. CDC20 (sc-13162, Santa Cruz Biotechnology) 1:500; CDC20 (A301-180A, Bethyl laboratories) 1:500; BUBR1 (612503, BD transduction laboratories); BUBR1 (A300-386A, Bethyl laboratories) 1:500; MAD2 (610679, BD transduction laboratories) 1:500; MAD2 (A300-301A, Bethyl Laboratories) 1:500; BUB3 (611730, BD Transduction Laboratories) 1:500; APC3 (610455, BD Transduction Laboratories) 1:500; APC4 (monoclonal antibody raised against a C-terminal peptide) 1:500; BLINKIN (a kind gift from M. Yanagida and T. Kiyomitsu) 1:50; anti-myc-epitope (9E10, Santa Cruz Biotechnology) 1:500; anti-flag epitope (M2, Sigma) 1:5000; anti-GFP (Clone 3.1 and 7.1, Roche) 1:200.Secondary antibodies: IRDye 680CW donkey anti mouse (926-68072, LI-COR), IRDye 800CW donkey anti mouse (926-32212, LI-COR); IRDye 680CW donkey anti rabbit (926-32223, LI-COR); IRDye 800CW donkey anti rabbit (926-32213, LI-COR) were all used at 1:10000.

The following antibodies were used at the indicated dilutions. CDC20 (sc-13162, Santa Cruz Biotechnology) 1:500; CDC20 (A301-180A, Bethyl laboratories) 1:500; BUBR1 (612503, BD transduction laboratories); BUBR1 (A300-386A, Bethyl laboratories) 1:500; MAD2 (610679, BD transduction laboratories) 1:500; MAD2 (A300-301A, Bethyl Laboratories) 1:500; BUB3 (611730, BD Transduction Laboratories) 1:500; APC3 (610455, BD Transduction Laboratories) 1:500; APC4 (monoclonal antibody raised against a C-terminal peptide) 1:500; BLINKIN (a kind gift from M. Yanagida and T. Kiyomitsu) 1:50; anti-myc-epitope (9E10, Santa Cruz Biotechnology) 1:500; anti-flag epitope (M2, Sigma) 1:5000; anti-GFP (Clone 3.1 and 7.1, Roche) 1:200.

Secondary antibodies: IRDye 680CW donkey anti mouse (926-68072, LI-COR), IRDye 800CW donkey anti mouse (926-32212, LI-COR); IRDye 680CW donkey anti rabbit (926-32223, LI-COR); IRDye 800CW donkey anti rabbit (926-32213, LI-COR) were all used at 1:10000.

ImmunofluorescenceCells were fixed in 4% paraformaldehyde and 2% sucrose for 5 minutes. After fixation cells were blocked in 3% BSA-PBS NP-40 0.2% and then incubated with antibodies. All antibodies were diluted in 3% BSA-PBS NP-40 0.2% and washes were performed with PBS NP-40 0.2%. Anti-GFP (Roche), Anti-MAD2 (A300-301A, Bethyl Laboratories), anti-flag M2 (sigma), anti-CDC20 (sc-13162, Santa Cruz Biotechnology) and anti-BUBR1 (A300-301A, Bethyl Laboratories) antibodies were used at 1:400 or 1:200, respectively. Anti-ACA serum (a kind gift from W. Earnshaw, University of Edinburgh, UK) was used at 1:20000. Secondary antibodies conjugated to Alexa Fluor 488, Alexa Fluor 568 or Alexa Fluor 647 (Molecular Probes) were diluted 1:400. DNA was stained with Hoechst-33342.

Immunofluorescence

Cells were fixed in 4% paraformaldehyde and 2% sucrose for 5 minutes. After fixation cells were blocked in 3% BSA-PBS NP-40 0.2% and then incubated with antibodies. All antibodies were diluted in 3% BSA-PBS NP-40 0.2% and washes were performed with PBS NP-40 0.2%. Anti-GFP (Roche), Anti-MAD2 (A300-301A, Bethyl Laboratories), anti-flag M2 (sigma), anti-CDC20 (sc-13162, Santa Cruz Biotechnology) and anti-BUBR1 (A300-301A, Bethyl Laboratories) antibodies were used at 1:400 or 1:200, respectively. Anti-ACA serum (a kind gift from W. Earnshaw, University of Edinburgh, UK) was used at 1:20000. Secondary antibodies conjugated to Alexa Fluor 488, Alexa Fluor 568 or Alexa Fluor 647 (Molecular Probes) were diluted 1:400. DNA was stained with Hoechst-33342.

Extended DataExtended data Figure 1Recombinant human Mitotic Checkpoint Complex binds to a second CDC20a, Schematic illustration of purification steps. Human wild type CDC20 (untagged), SBPBUBR1 and 6HisMAD2 were expressed in baculovirus-infected Sf9 cells. The recombinant core Mitotic Checkpoint Complex (rMCC) was purified by Ni-NTA and streptavidin beads. Purified core rMCC bound to streptavidin beads was used to assay binding to purified recombinant Cdc20.b, Core rMCC consisting of CDC20, SBPBUBR1 and 6HisMAD2 was analysed by SDS-PAGE and Coomassie Blue (CB) R250 staining, followed by quantification at 680 nm on a LiCOR Odyssey scanner. Equal molar amounts of purified SBPBUBR1, SBPCDC20 and SBPMAD2 proteins were used to calibrate the CB staining. The stoichiometry of core rMCC (mean ± s.d. is shown below the panel with SBPBUBR1 set to 1.0) was estimated from three independently purified core rMCC preparations. Molecular mass markers are on the left.c, d, Both the MAD2 Binding Motif of CDC20 and the first KEN-box of BUBR1 are required to assemble rMCC. Core rMCC was pulled down with streptavidin beads from Sf9 cells expressing SBPBUBR1, 6HisMAD2, and either wild-type CDC20 or the K129ILR/AAAA mutant (Δ) (panel c), or 6HisMAD2, wild-type CDC20 plus wild type (WT) SBPBUBR1, or alanine substitution mutants of either KEN Box 1 (KEN1) or KEN Box 2 (KEN2). The proteins retained on streptavidin beads were analysed by immunoblotting with the indicated antibodies.e, Relative expression levels of core rMCC components. Sf9 cells extracts expressing the core rMCC, and the purified rMCC complex, were analysed by quantitative immunoblotting. The ratio of the proteins in the extracts is given, with that of SBP-BUBR1 set to 1.0.f, Schematic illustration of the second CDC20 binding assay in fig. 1a. In lanes 1 & 2, the streptavidin beads were incubated with either 6HisCDC20 wild-type or the ΔKILR (K129ILR/AAAA) mutant. In lanes 3 & 4, the streptavidin beads bound to core rMCC were incubated with the 6HisCDC20 proteins. In lanes 6 & 7, the streptavidin beads bound to 6His-SBPCDC20 were incubated with the 6HisCDC20 proteins.g, Sf9 cell extracts expressing core rMCC or 3Flag-tagged CDC20 were mixed and the core rMCC purified as in panel a. The core rMCC was analysed by quantitative immunoblotting. 51% of the core rMCC was purified bound to a second 3FlagCDC20.h, A functional CDC20 promotes the binding of core rMCC to the APC/C. The APC/C was immunoprecipitated from CDC20-depleted mitotic extracts supplemented with a constant amount of core rMCC and 10-fold excess of recombinant wild-type (WT) SBPCDC20, or the ΔKILR or ΔIR mutants. The co-immunoprecipitates were analysed as in Fig. 1c.i, Schematic of the APC/C-MCC-CDC20 ternary complex. Both core rMCC and CDC20 bind to the APC/C and form a ternary complex (left). The CDC20ΔKILR mutant cannot bind the APC/C directly, nor stimulate core rMCC binding to the APC/C, but CDC20ΔKILR still binds to rMCC (right).Extended data Figure 2Comparison of rMCC with and without BUB3.a, b, Preparation of recombinant core MCC with or without BUB3. Insect cells were infected with viruses expressing core MCC components with and without BUB3, and the rMCC was purified by Ni-NAT and streptavidin beads. The complexes were analysed by CB staining (a) and immunoblotting (b). c, Binding to a second 6HisCDC20 of recombinant core MCC with or without BUB3 was performed and analysed as in Fig. 1a.Extended data Figure 3Molar ratios of rMCC, CDC20 and the APC/C in the in vitro ubiquitylation assays.a & b, Core rMCC and CDC20 from Fig 1d were analysed by quantitative immunoblotting. CDC20, MAD2 and BUBR1 were analysed by quantitative immunoblotting in the input (a) and in the reaction (b). The black filled circles are unconjugated SBPCDC20; red filled circles are ubiquitylated SBPCDC20. c, Core rMCC, rCDC20, and the APC/C immunoprecipitates used in Fig. 1e, plus a purified SBPAPC3 subunit, were analysed by quantitative immunoblotting with the indicated antibodies. The calculated molar ratios of rMCC, rCDC20 and the APC/C are shown below the panels.Extended data Figure 4Cells expressing the D-box and KEN box receptor mutants of CDC20 can degrade Cyclin B1 in nocodazole.Cyclin B1-Venus degradation was analysed in siRNA CDC20-treated cells rescued with siRNA resistant versions of 3×Flag-CDC20, wild-type, or ΔDR, or ΔKR mutants, in the presence of nocodazole (0.33 μM). The fluorescence of individual cells was measured, the value at NEBD set to 1 and the mean ± s.e.m. for all cells plotted. n = number of cells analysed from at least two independent experiments.Extended data Figure 5Characterisation of the MCC containing D-Box or KEN-Box 2 mutants of BUBR1.a, Core rMCC assembled with SBPBUBR1 wild-type, or ΔD-box, or ΔKEN2 mutants, was purified as in Extended Fig. 1a and analysed on a LiCOR Odyssey scanner at 680 nm after SDS-PAGE and CB R250 staining. b, The core rMCC mutants prepared in a were assayed as APC/C inhibitors in an in vitro ubiquitylation assay as in Fig. 1d. c, Insect cell extracts expressing CDC20 with SBPBUBR1, either wild-type, or ΔD-box or ΔKEN2 mutant, were incubated with streptavidin beads. The proteins retained on the streptavidin beads were analysed by quantitative immunoblotting. d, HeLa cells were treated with siRNA against BUBR1 and rescued with 3×Flag-Cerulean-BUBR1, either wild-type or the ΔD-box mutant, and mitosis analysed in 0.116 μM Taxol as in Fig. 3d. The time from NEBD to anaphase (or mitotic exit) was measured and plotted as a box and whisker chart. n = number of analysed cells from two independent biological replicates. e, HeLa cells were treated with siRNA against BUBR1 and rescued with siRNA resistant 3×Flag-Cerulean-BUBR1, either wild-type or the ΔD-box mutant, then analysed by immunostaining. Cells were stained with anti-Flag M2 and anti-ACA antibodies, and Hoechst 33342, and representative images of prometaphase cells are shown. Scale bar = 10 μm.Extended data Figure 6Stabilising the interaction between MAD2 and CDC20a, Schematic of how a stabilised MCC might block cells in metaphase. At prometaphase, when the SAC is ON, CDC20 is inhibited both by incorporation into the MCC and through binding to the MCC. At metaphase when the SAC is OFF, CDC20 is released from the MCC and activates the APC/C. We postulate that stabilising an exogenous MCC to prevent its disassembly should also prevent endogenous CDC20 from activating the APC/C, which results into anaphase delay.b, Schematic of a stabilised MCC. To stabilise the MCC we took advantage of the binding between yellow fluorescent protein (Venus) and GFP-binding protein (GBP), which is a 13kDa domain from a camelid antibody that binds strongly and specifically to GFP and YFP18. MAD2 and BUBR1 were tagged with Venus and the GBP domain was tagged to CDC20. We refer to the MCC containing a stabilised MAD2-CDC20 interaction as MCCM2, and that with stabilised BUBR1-CDC20 as MCCR1. c, GBP- and Venus-fusion proteins bind stably to each other in vivo. HeLa cell lines expressing siRNA-resistant myc-CDC20 or myc-GBP-CDC20 were transfected with plasmids encoding either Venus alone or Venus-MAD2, followed by siRNA treatment against CDC20. After a single thymidine block and release, the cells were arrested at prometaphase by treating with nocodazole, and harvested by mitotic shake-off 48 hr after the siRNA treatment. Proteins were immunoprecipitated with anti-myc epitope antibodies before analysis by quantitative immunoblotting with the indicated antibodies. WT: myc-CDC20; GBP: myc-GBP-CDC20.Extended data Figure 7Stabilising the interaction between MAD2 and CDC20 prevents disassembly of the MCC in vivoa & b, Tethering CDC20 to MAD2 prevents MCC disassembly and release from the APC/C. A, Empty plasmids or plasmids encoding Venus-MAD2 were transfected into HeLa cell lines expressing 3×Flag-GBP-CDC20 and the cells synchronized at prometaphase by thymidine release followed by a nocodazole block. Cells were harvested by mitotic shake off and separated into two cultures after washing once in medium. One culture was harvested immediately (− reversine) and the other resuspended in medium containing 1 μM reversine and 10 μM MG132 (+ reversine) for 1 hr before harvesting. The APC/C was immunoprecipitated with an anti-APC4 antibody and the immunoprecipitates analysed by quantitative immunoblotting. We note that the APC/C preferred to bind endogenous CDC20 over GBP-CDC20 as the co-activator in vivo (see + reversine lane in control cells) but the MCCM2 did not sequester endogenous CDC20 from the APC/C (see + reversine lane in GBP-CDC20+Venus-MAD2 cells). b, Mean ± s.e.m. of the relative amounts of the indicated proteins in the APC4 immunoprecipitates calculated from four independent biological experiments. The amount of protein bound to the APC/C in the absence of reversine was set to 1 (red line)c-e, Tethering CDC20 to MAD2 prevents MCC disassembly and release from the APC/C in the absence of endogenous CDC20.c, Plasmids encoding Venus-MAD2 were transfected into HeLa cell lines expressing the indicated CDC20 fusion proteins following siRNA treatment against CDC20 for 48 hr. Cells were synchronized at prometaphase then treated with reversine, and anti APC4 and anti-GFP immunoprecipitates were analysed as in a. WT: myc-CDC20; GBP: myc-GBP-CDC20. Note that endogenous CDC20 could not be inhibited through exchange into MCCM2 because a core MCC composed of Venus-MAD2 and untagged CDC20 disassembled. d, HeLa cell lines expressing myc-CDC20 (upper blots) or myc-GBP-CDC20 (lower blots) were transfected with a plasmid encoding Venus-MAD2 followed by siRNA treatment against CDC20 for 48 hr. Cells were synchronized at prometaphase and treated with reversine as indicated in a. Total cell extracts were analysed by size-exclusion chromatography on a Sepharose 6 column and fractions were analysed by quantitative immunoblotting against the indicated proteins and the relative amounts of Venus-MAD2 plotted in panel e with the sum of Venus-MAD2 intensities set to 1. The migration of APC/C or APC/C-MCC is annotated below panel d.Extended data Figure 8KNL1 is not required for a stabilised MCC to inhibit anaphase.a, HeLa cells expressing MCCM2 in Fig. 4a were analysed by immunostaining. The cells were stained with anti-GFP, anti-MAD2, anti-ACA and Hoechst 33342, and representative images of prometaphase and metaphase cells are shown. Scale bar = 5 μm. b, The time from NEBD to Anaphase in Fig. 4a was plotted against the intensity of mCherry-GBP-CDC20 (left) or the ratio of V-MAD2 to mCherry-GBP-CDC20 (right). The ratio of V-MAD2 to mCherry-GBP-CDC20 was calibrated by measuring fluorescence intensity of a mCherry-GBP-Venus fusion protein in HeLa cells.c-e, MCCM2 delays anaphase when Blinkin/KNL1 is depleted.c, HeLa cells were treated with siRNA against Blinkin/KNL1 for 72 hrs and total cell extracts were analysed by quantitative immunoblotting with indicated the antibodies. d, HeLa cells treated as in c were analysed by immunostaining. The cells were stained with anti-CDC20, anti-BUBR1, anti-ACA and Hoechst 33342, and representative images of early prometaphase are shown. Scale bar = 10 μme, HeLa cell lines stably expressing mCherry-GBP-CDC20 and an inducible 3×Flag-Venus-MAD2 (expressed from a tetracyclin-inducible promoter) were treated with siRNA against Blinkin/KNL1 as in c. Progression through mitosis was analysed in the presence (+Tet) or absence (−Tet) of tetracyclin, and analysed as in Fig. 4b. n = number of cells from two independent biological replicates.Extended data Figure 9Functional MCCM2 is required to delay anaphasea, HeLa cells were transfected with plasmids encoding Venus-MAD2 and either wild type or a MAD2-binding defective (KILR/AAAA) mutant of CDC20 tagged with mCherry-GBP, and mitotic progression was analysed as in fig. 4a. n = number of cells from three independent biological replicates b, The core rMCC mutants used in Extended Data Fig. 5a were incubated with preformed APC/CCDC20 and assayed as APC/C inhibitors in an in vitro ubiquitylation assay as in Fig. 1e. The extent of APC/C inhibition (incubation of MCCWT set to 1.0) is shown below the securin panel. This result is representative of two independent experiments.c, Schematic of the inhibitory activities of the stabilised MCCs in BUBR1-depleted cells used in Fig. 4c. When BUBR1 is depleted, MAD2 and CDC20 cannot form the MCC to inhibit endogenous CDC20 (left). When rescued with wild-type BUBR1, MCCM2 can form and inhibit endogenous CDC20 to delay anaphase. By contrast, when rescued by the BUBR1 ΔD-box mutant, MCCM2 can only weakly inhibit endogenous CDC20 and cells can proceed into anaphase.Extended data Figure 10Model for how the MCC could disseminate the wait anaphase signal Unattached kinetochores catalyse MCC formation and the MCC disseminates the Wait Anaphase signal through the cytoplasm (black arrows). When the MCC disassembles (blue arrows), this releases CDC20 that, along with newly synthesised CDC20, can have two fates: to be recruited to unattached kinetochores and incorporated into the MCC, or to bind the APC/C to form APC/CCDC20. The MCC is able to inhibit both unliganded CDC20 and CDC20 bound to the APC/C (red bars).

Extended Data

Recombinant human Mitotic Checkpoint Complex binds to a second CDC20

a, Schematic illustration of purification steps. Human wild type CDC20 (untagged), SBPBUBR1 and 6HisMAD2 were expressed in baculovirus-infected Sf9 cells. The recombinant core Mitotic Checkpoint Complex (rMCC) was purified by Ni-NTA and streptavidin beads. Purified core rMCC bound to streptavidin beads was used to assay binding to purified recombinant Cdc20.

b, Core rMCC consisting of CDC20, SBPBUBR1 and 6HisMAD2 was analysed by SDS-PAGE and Coomassie Blue (CB) R250 staining, followed by quantification at 680 nm on a LiCOR Odyssey scanner. Equal molar amounts of purified SBPBUBR1, SBPCDC20 and SBPMAD2 proteins were used to calibrate the CB staining. The stoichiometry of core rMCC (mean ± s.d. is shown below the panel with SBPBUBR1 set to 1.0) was estimated from three independently purified core rMCC preparations. Molecular mass markers are on the left.

c, d, Both the MAD2 Binding Motif of CDC20 and the first KEN-box of BUBR1 are required to assemble rMCC. Core rMCC was pulled down with streptavidin beads from Sf9 cells expressing SBPBUBR1, 6HisMAD2, and either wild-type CDC20 or the K129ILR/AAAA mutant (Δ) (panel c), or 6HisMAD2, wild-type CDC20 plus wild type (WT) SBPBUBR1, or alanine substitution mutants of either KEN Box 1 (KEN1) or KEN Box 2 (KEN2). The proteins retained on streptavidin beads were analysed by immunoblotting with the indicated antibodies.

e, Relative expression levels of core rMCC components. Sf9 cells extracts expressing the core rMCC, and the purified rMCC complex, were analysed by quantitative immunoblotting. The ratio of the proteins in the extracts is given, with that of SBP-BUBR1 set to 1.0.

f, Schematic illustration of the second CDC20 binding assay in fig. 1a. In lanes 1 & 2, the streptavidin beads were incubated with either 6HisCDC20 wild-type or the ΔKILR (K129ILR/AAAA) mutant. In lanes 3 & 4, the streptavidin beads bound to core rMCC were incubated with the 6HisCDC20 proteins. In lanes 6 & 7, the streptavidin beads bound to 6His-SBPCDC20 were incubated with the 6HisCDC20 proteins.

g, Sf9 cell extracts expressing core rMCC or 3Flag-tagged CDC20 were mixed and the core rMCC purified as in panel a. The core rMCC was analysed by quantitative immunoblotting. 51% of the core rMCC was purified bound to a second 3FlagCDC20.

h, A functional CDC20 promotes the binding of core rMCC to the APC/C. The APC/C was immunoprecipitated from CDC20-depleted mitotic extracts supplemented with a constant amount of core rMCC and 10-fold excess of recombinant wild-type (WT) SBPCDC20, or the ΔKILR or ΔIR mutants. The co-immunoprecipitates were analysed as in Fig. 1c.

i, Schematic of the APC/C-MCC-CDC20 ternary complex. Both core rMCC and CDC20 bind to the APC/C and form a ternary complex (left). The CDC20ΔKILR mutant cannot bind the APC/C directly, nor stimulate core rMCC binding to the APC/C, but CDC20ΔKILR still binds to rMCC (right).

Comparison of rMCC with and without BUB3.

a, b, Preparation of recombinant core MCC with or without BUB3. Insect cells were infected with viruses expressing core MCC components with and without BUB3, and the rMCC was purified by Ni-NAT and streptavidin beads. The complexes were analysed by CB staining (a) and immunoblotting (b). c, Binding to a second 6HisCDC20 of recombinant core MCC with or without BUB3 was performed and analysed as in Fig. 1a.

Molar ratios of rMCC, CDC20 and the APC/C in the in vitro ubiquitylation assays.

a & b, Core rMCC and CDC20 from Fig 1d were analysed by quantitative immunoblotting. CDC20, MAD2 and BUBR1 were analysed by quantitative immunoblotting in the input (a) and in the reaction (b). The black filled circles are unconjugated SBPCDC20; red filled circles are ubiquitylated SBPCDC20. c, Core rMCC, rCDC20, and the APC/C immunoprecipitates used in Fig. 1e, plus a purified SBPAPC3 subunit, were analysed by quantitative immunoblotting with the indicated antibodies. The calculated molar ratios of rMCC, rCDC20 and the APC/C are shown below the panels.

Cells expressing the D-box and KEN box receptor mutants of CDC20 can degrade Cyclin B1 in nocodazole.

Cyclin B1-Venus degradation was analysed in siRNA CDC20-treated cells rescued with siRNA resistant versions of 3×Flag-CDC20, wild-type, or ΔDR, or ΔKR mutants, in the presence of nocodazole (0.33 μM). The fluorescence of individual cells was measured, the value at NEBD set to 1 and the mean ± s.e.m. for all cells plotted. n = number of cells analysed from at least two independent experiments.

Characterisation of the MCC containing D-Box or KEN-Box 2 mutants of BUBR1.

a, Core rMCC assembled with SBPBUBR1 wild-type, or ΔD-box, or ΔKEN2 mutants, was purified as in Extended Fig. 1a and analysed on a LiCOR Odyssey scanner at 680 nm after SDS-PAGE and CB R250 staining. b, The core rMCC mutants prepared in a were assayed as APC/C inhibitors in an in vitro ubiquitylation assay as in Fig. 1d. c, Insect cell extracts expressing CDC20 with SBPBUBR1, either wild-type, or ΔD-box or ΔKEN2 mutant, were incubated with streptavidin beads. The proteins retained on the streptavidin beads were analysed by quantitative immunoblotting. d, HeLa cells were treated with siRNA against BUBR1 and rescued with 3×Flag-Cerulean-BUBR1, either wild-type or the ΔD-box mutant, and mitosis analysed in 0.116 μM Taxol as in Fig. 3d. The time from NEBD to anaphase (or mitotic exit) was measured and plotted as a box and whisker chart. n = number of analysed cells from two independent biological replicates. e, HeLa cells were treated with siRNA against BUBR1 and rescued with siRNA resistant 3×Flag-Cerulean-BUBR1, either wild-type or the ΔD-box mutant, then analysed by immunostaining. Cells were stained with anti-Flag M2 and anti-ACA antibodies, and Hoechst 33342, and representative images of prometaphase cells are shown. Scale bar = 10 μm.

Stabilising the interaction between MAD2 and CDC20

a, Schematic of how a stabilised MCC might block cells in metaphase. At prometaphase, when the SAC is ON, CDC20 is inhibited both by incorporation into the MCC and through binding to the MCC. At metaphase when the SAC is OFF, CDC20 is released from the MCC and activates the APC/C. We postulate that stabilising an exogenous MCC to prevent its disassembly should also prevent endogenous CDC20 from activating the APC/C, which results into anaphase delay.

b, Schematic of a stabilised MCC. To stabilise the MCC we took advantage of the binding between yellow fluorescent protein (Venus) and GFP-binding protein (GBP), which is a 13kDa domain from a camelid antibody that binds strongly and specifically to GFP and YFP18. MAD2 and BUBR1 were tagged with Venus and the GBP domain was tagged to CDC20. We refer to the MCC containing a stabilised MAD2-CDC20 interaction as MCCM2, and that with stabilised BUBR1-CDC20 as MCCR1. c, GBP- and Venus-fusion proteins bind stably to each other in vivo. HeLa cell lines expressing siRNA-resistant myc-CDC20 or myc-GBP-CDC20 were transfected with plasmids encoding either Venus alone or Venus-MAD2, followed by siRNA treatment against CDC20. After a single thymidine block and release, the cells were arrested at prometaphase by treating with nocodazole, and harvested by mitotic shake-off 48 hr after the siRNA treatment. Proteins were immunoprecipitated with anti-myc epitope antibodies before analysis by quantitative immunoblotting with the indicated antibodies. WT: myc-CDC20; GBP: myc-GBP-CDC20.

Stabilising the interaction between MAD2 and CDC20 prevents disassembly of the MCC in vivo

a & b, Tethering CDC20 to MAD2 prevents MCC disassembly and release from the APC/C. A, Empty plasmids or plasmids encoding Venus-MAD2 were transfected into HeLa cell lines expressing 3×Flag-GBP-CDC20 and the cells synchronized at prometaphase by thymidine release followed by a nocodazole block. Cells were harvested by mitotic shake off and separated into two cultures after washing once in medium. One culture was harvested immediately (− reversine) and the other resuspended in medium containing 1 μM reversine and 10 μM MG132 (+ reversine) for 1 hr before harvesting. The APC/C was immunoprecipitated with an anti-APC4 antibody and the immunoprecipitates analysed by quantitative immunoblotting. We note that the APC/C preferred to bind endogenous CDC20 over GBP-CDC20 as the co-activator in vivo (see + reversine lane in control cells) but the MCCM2 did not sequester endogenous CDC20 from the APC/C (see + reversine lane in GBP-CDC20+Venus-MAD2 cells). b, Mean ± s.e.m. of the relative amounts of the indicated proteins in the APC4 immunoprecipitates calculated from four independent biological experiments. The amount of protein bound to the APC/C in the absence of reversine was set to 1 (red line)

c-e, Tethering CDC20 to MAD2 prevents MCC disassembly and release from the APC/C in the absence of endogenous CDC20.

c, Plasmids encoding Venus-MAD2 were transfected into HeLa cell lines expressing the indicated CDC20 fusion proteins following siRNA treatment against CDC20 for 48 hr. Cells were synchronized at prometaphase then treated with reversine, and anti APC4 and anti-GFP immunoprecipitates were analysed as in a. WT: myc-CDC20; GBP: myc-GBP-CDC20. Note that endogenous CDC20 could not be inhibited through exchange into MCCM2 because a core MCC composed of Venus-MAD2 and untagged CDC20 disassembled. d, HeLa cell lines expressing myc-CDC20 (upper blots) or myc-GBP-CDC20 (lower blots) were transfected with a plasmid encoding Venus-MAD2 followed by siRNA treatment against CDC20 for 48 hr. Cells were synchronized at prometaphase and treated with reversine as indicated in a. Total cell extracts were analysed by size-exclusion chromatography on a Sepharose 6 column and fractions were analysed by quantitative immunoblotting against the indicated proteins and the relative amounts of Venus-MAD2 plotted in panel e with the sum of Venus-MAD2 intensities set to 1. The migration of APC/C or APC/C-MCC is annotated below panel d.

KNL1 is not required for a stabilised MCC to inhibit anaphase.

a, HeLa cells expressing MCCM2 in Fig. 4a were analysed by immunostaining. The cells were stained with anti-GFP, anti-MAD2, anti-ACA and Hoechst 33342, and representative images of prometaphase and metaphase cells are shown. Scale bar = 5 μm. b, The time from NEBD to Anaphase in Fig. 4a was plotted against the intensity of mCherry-GBP-CDC20 (left) or the ratio of V-MAD2 to mCherry-GBP-CDC20 (right). The ratio of V-MAD2 to mCherry-GBP-CDC20 was calibrated by measuring fluorescence intensity of a mCherry-GBP-Venus fusion protein in HeLa cells.

c-e, MCCM2 delays anaphase when Blinkin/KNL1 is depleted.

c, HeLa cells were treated with siRNA against Blinkin/KNL1 for 72 hrs and total cell extracts were analysed by quantitative immunoblotting with indicated the antibodies. d, HeLa cells treated as in c were analysed by immunostaining. The cells were stained with anti-CDC20, anti-BUBR1, anti-ACA and Hoechst 33342, and representative images of early prometaphase are shown. Scale bar = 10 μm

e, HeLa cell lines stably expressing mCherry-GBP-CDC20 and an inducible 3×Flag-Venus-MAD2 (expressed from a tetracyclin-inducible promoter) were treated with siRNA against Blinkin/KNL1 as in c. Progression through mitosis was analysed in the presence (+Tet) or absence (−Tet) of tetracyclin, and analysed as in Fig. 4b. n = number of cells from two independent biological replicates.

Functional MCCM2 is required to delay anaphase

a, HeLa cells were transfected with plasmids encoding Venus-MAD2 and either wild type or a MAD2-binding defective (KILR/AAAA) mutant of CDC20 tagged with mCherry-GBP, and mitotic progression was analysed as in fig. 4a. n = number of cells from three independent biological replicates b, The core rMCC mutants used in Extended Data Fig. 5a were incubated with preformed APC/CCDC20 and assayed as APC/C inhibitors in an in vitro ubiquitylation assay as in Fig. 1e. The extent of APC/C inhibition (incubation of MCCWT set to 1.0) is shown below the securin panel. This result is representative of two independent experiments.

c, Schematic of the inhibitory activities of the stabilised MCCs in BUBR1-depleted cells used in Fig. 4c. When BUBR1 is depleted, MAD2 and CDC20 cannot form the MCC to inhibit endogenous CDC20 (left). When rescued with wild-type BUBR1, MCCM2 can form and inhibit endogenous CDC20 to delay anaphase. By contrast, when rescued by the BUBR1 ΔD-box mutant, MCCM2 can only weakly inhibit endogenous CDC20 and cells can proceed into anaphase.

Model for how the MCC could disseminate the wait anaphase signal Unattached kinetochores catalyse MCC formation and the MCC disseminates the Wait Anaphase signal through the cytoplasm (black arrows). When the MCC disassembles (blue arrows), this releases CDC20 that, along with newly synthesised CDC20, can have two fates: to be recruited to unattached kinetochores and incorporated into the MCC, or to bind the APC/C to form APC/CCDC20. The MCC is able to inhibit both unliganded CDC20 and CDC20 bound to the APC/C (red bars).
