---
pmid: '29795547'
title: Yeast heterochromatin regulators Sir2 and Sir3 act directly at euchromatic
  DNA replication origins.
authors:
- Hoggard TA
- Chang F
- Perry KR
- Subramanian S
- Kenworthy J
- Chueng J
- Shor E
- Hyland EM
- Boeke JD
- Weinreich M
- Fox CA
journal: PLoS Genet
year: '2018'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC5991416
doi: 10.1371/journal.pgen.1007418
---

# Yeast heterochromatin regulators Sir2 and Sir3 act directly at euchromatic DNA replication origins.
**Authors:** Hoggard TA, Chang F, Perry KR, Subramanian S, Kenworthy J, Chueng J, Shor E, Hyland EM, Boeke JD, Weinreich M, Fox CA
**Journal:** PLoS Genet (2018)
**DOI:** [10.1371/journal.pgen.1007418](https://doi.org/10.1371/journal.pgen.1007418)
**PMC:** [PMC5991416](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5991416/)

## Abstract

1. PLoS Genet. 2018 May 24;14(5):e1007418. doi: 10.1371/journal.pgen.1007418. 
eCollection 2018 May.

Yeast heterochromatin regulators Sir2 and Sir3 act directly at euchromatic DNA 
replication origins.

Hoggard TA(1), Chang F(2), Perry KR(1)(3), Subramanian S(2), Kenworthy J(2), 
Chueng J(3), Shor E(4), Hyland EM(5), Boeke JD(6), Weinreich M(2), Fox CA(1)(3).

Author information:
(1)Department of Biomolecular Chemistry, School of Medicine and Public Health, 
Madison, WI, United States of America.
(2)Laboratory of Genome Integrity and Tumorigenesis, Van Andel Research 
Institute, Grand Rapids, MI, United States of America.
(3)Integrated Program in Biochemistry, School of Medicine and Public Health and 
College of Agricultural Sciences, University of Wisconsin, Madison, WI, United 
States of America.
(4)Public Health Research Institute, New Jersey Medical School, Rutgers 
University, Newark, New Jersey, United States of America.
(5)School of Biological Sciences, Medical Biology Center, Queen's University, 
Belfast, United Kingdom.
(6)Department of Biochemistry and Molecular Pharmacology, Institute for Systems 
Genetics and NYU Langone Health, New York, NY, United States of America.

Most active DNA replication origins are found within euchromatin, while origins 
within heterochromatin are often inactive or inhibited. In yeast, origin 
activity within heterochromatin is negatively controlled by the histone H4K16 
deacetylase, Sir2, and at some heterochromatic loci also by the nucleosome 
binding protein, Sir3. The prevailing view has been that direct functions of 
Sir2 and Sir3 are confined to heterochromatin. However, growth defects in yeast 
mutants compromised for loading the MCM helicase, such as cdc6-4, are suppressed 
by deletion of either SIR2 or SIR3. While these and other observations indicate 
that SIR2,3 can have a negative impact on at least some euchromatic origins, the 
genomic scale of this effect was unknown. It was also unknown whether this 
suppression resulted from direct functions of Sir2,3 within euchromatin, or was 
an indirect effect of their previously established roles within heterochromatin. 
Using MCM ChIP-Seq, we show that a SIR2 deletion rescued MCM complex loading at 
~80% of euchromatic origins in cdc6-4 cells. Therefore, Sir2 exhibited a 
pervasive effect at the majority of euchromatic origins. Using MNase-H4K16ac 
ChIP-Seq, we show that origin-adjacent nucleosomes were depleted for H4K16 
acetylation in a SIR2-dependent manner in wild type (i.e. CDC6) cells. In 
addition, we present evidence that both Sir2 and Sir3 bound to nucleosomes 
adjacent to euchromatic origins. The relative levels of each of these molecular 
hallmarks of yeast heterochromatin-SIR2-dependent H4K16 hypoacetylation, Sir2, 
and Sir3 -correlated with how strongly a SIR2 deletion suppressed the MCM 
loading defect in cdc6-4 cells. Finally, a screen for histone H3 and H4 mutants 
that could suppress the cdc6-4 growth defect identified amino acids that map to 
a surface of the nucleosome important for Sir3 binding. We conclude that 
heterochromatin proteins directly modify the local chromatin environment of 
euchromatic DNA replication origins.

DOI: 10.1371/journal.pgen.1007418
PMCID: PMC5991416
PMID: 29795547 [Indexed for MEDLINE]

Conflict of interest statement: The authors have declared that no competing 
interests exist.

## Full Text

IntroductionIn eukaryotic cells, efficient genome duplication requires the function of multiple DNA replication origins distributed over the length of each chromosome [1–5]. Origins are selected by a series of steps in late M- to G1-phase during which the origin recognition complex (ORC) binds directly to DNA and recruits the Cdc6 protein [6–8]. The ORC-Cdc6-DNA complex recruits Cdt1-MCM to form an MCM double hexamer (dhMCM) encircling double-stranded DNA [9,10]. Several kinases and loading factors then remodel the dhMCM into two active CMG helicases (Cdc45-MCM-GINS) that unwind the DNA bidirectionally from each origin to allow the initiation of DNA synthesis [11]. Thus, dhMCM loading is the event that ‘licenses’ the DNA to function as an origin of replication in S-phase (for recent comprehensive review of yeast replication see [12]). All replication origins exist in the context of chromatin, and origin function is significantly affected by local chromatin structure (reviewed in [13,14]). Typically, the most efficient origins (i.e. the origins that are used in most cell cycles) are found in euchromatin, while less efficient origins are associated with heterochromatin. While there is intense interest in the impact of chromatin structure on origin function, the relevant molecular features of origin-adjacent chromatin and the steps in origin function that they control remain incompletely understood.Yeast heterochromatin is characterized by hypoacetylated nucleosomes and associated heterochromatin regulatory proteins that promote a compact chromatin structure that makes the underlying DNA inaccessible to protein-DNA interactions and processes such as transcription and DNA replication initiation (recently reviewed in [15]). The Sir2 (Silent information regulator) protein, the founding member of a family of conserved NAD-dependent protein deacetylases, removes an acetyl group from lysine 16 of histone H4 (H4K16) and is required for heterochromatin formation in budding yeast [16–19]. Heterochromatin domains form at only a few discrete regions in the yeast genome, rDNA, telomeres, and the HM-mating type loci, and at each of these loci, Sir2 both deacetylates H4K16ac and stably binds to nucleosomes [15]. Heterochromatin formation at the HM loci and telomeres also requires nucleosome binding by the Sir3 and Sir4 proteins.While it is recognized that the function of normally efficient origins can be suppressed when they are engineered within heterochromatic regions of the genome [20,21], several more recent studies reveal that SIR2 can also affect the function of euchromatic origins [22,23]. However, depending on experimental context, SIR2 can be interpreted to exert either positive or negative effects on euchromatic origins. In particular, recent studies reveal that SIR2 acts indirectly as a positive regulator of euchromatic origins by suppressing the function of rDNA origins [24,25]. The tandemly repeated rDNA gene locus on chromosome XII contains ~200 replication origins, but ~80% of these origins are suppressed by Sir2-dependent heterochromatin [24]. Because the rDNA origins account for >30% of all yeast genomic origins, deletion of SIR2 results in activation of many rDNA origins, which then sequester limiting S-phase origin activation factors from euchromatic origins, delaying their activation time in S-phase [26,27]. Thus, the positive role for SIR2 in regulating euchromatic origin function in these reports is explained as a byproduct of a direct inhibition of rDNA origins by Sir2.Interestingly, other studies establish that SIR2 can also exert a negative effect at euchromatic origins. This negative role was revealed by a classic genetic screen that isolated sir2 mutants as suppressors of the temperature-sensitive cdc6-4 allele [23,28,29]. Cdc6, a member of the AAA+ protein family, must bind ATP to load dhMCM [30,31]. The cdc6-4 allele encodes a mutant Cdc6 with a lysine to alanine substitution in the conserved ATP binding motif. Cells with the cdc6-4 allele are viable but have S-phase associated growth defects at the permissive temperature and arrest growth at the non-permissive temperature with a failure to load dhMCM, as assessed by origin-specific ChIPs. Like sir2Δ, sir3Δ is also a robust suppressor of the cdc6-4 growth defect, whereas a sir4Δ mutant is only a very weak suppressor [29]. Thus the suppression by sir2Δ or sir3Δ is not easily explained by defects in classic Sir-heterochromatic gene silencing of known loci because Sir2, Sir3 and Sir4 are each equally essential for HM- and telomere silencing and only Sir2 is required for rDNA silencing. In addition, multiple sir2 alleles specifically defective in rDNA silencing do not suppress cdc6-4 ts lethality, indicating that a loss of rDNA silencing is not sufficient to explain SIR2’s negative effect on euchromatic origins [29]. These findings support a distinct, rDNA- and classic heterochromatin-independent function of SIR2 on euchromatic origins. Importantly, a sir2 catalytic mutant or a histone mutant that converts H4K16 into a residue that mimics acetylated H4K16 (H4K16Q) suppresses cdc6-4, indicating that the Sir2 deacetylase function is critical for its negative impact on euchromatic origins [23,29]. However, these data do not address whether the negative role exerted by SIR2 is due to direct roles for Sir2 within euchromatic regions of the genome, or whether SIR2 and SIR3 affect euchromatic origins through a shared mechanism.In this report we investigated the molecular mechanisms by which SIR2 affected euchromatic origins by performing MCM ChIP-Seq and MNase-H4K16ac ChIP-Seq experiments and also by analyzing several published high-resolution ChIP-Seq experiments [32–34]. Our results provide evidence for a direct role for Sir2 and Sir3 in forming a repressive local chromatin environment around most origins that exist within euchromatin. A sir2Δ mutation was sufficient to restore MCM binding at the majority of euchromatic origins in cdc6-4 cells even at the non-permissive growth temperature for this allele. In wild type cells, nucleosomes immediately adjacent to the majority of euchromatic origins were relatively hypoacetylated on H4K16 compared to non-origin control loci, a behavior unique to this histone acetylation mark. Moreover, this origin-specific H4K16 hypoacetylation was completely dependent on SIR2. In addition, Sir2 and Sir3 were physically associated with euchromatic origins but not non-origin control loci. The levels of these three distinct molecular features of yeast heterochromatin—hypoacetylation of H4K16, Sir2 and Sir3 binding—correlated with how strongly a SIR2 deletion suppressed the MCM loading defect in cdc6-4 cells. Based on these results we propose that the dhMCM loading reaction has evolved to work within a potentially repressive Sir2,3-chromatin environment that forms around most euchromatic origins. Consistent with this model, a screen for suppressors of cdc6-4 temperature-sensitive lethality identified several histone mutants known to disrupt the Sir3-nucleosome binding interface.

Introduction

In eukaryotic cells, efficient genome duplication requires the function of multiple DNA replication origins distributed over the length of each chromosome [1–5]. Origins are selected by a series of steps in late M- to G1-phase during which the origin recognition complex (ORC) binds directly to DNA and recruits the Cdc6 protein [6–8]. The ORC-Cdc6-DNA complex recruits Cdt1-MCM to form an MCM double hexamer (dhMCM) encircling double-stranded DNA [9,10]. Several kinases and loading factors then remodel the dhMCM into two active CMG helicases (Cdc45-MCM-GINS) that unwind the DNA bidirectionally from each origin to allow the initiation of DNA synthesis [11]. Thus, dhMCM loading is the event that ‘licenses’ the DNA to function as an origin of replication in S-phase (for recent comprehensive review of yeast replication see [12]). All replication origins exist in the context of chromatin, and origin function is significantly affected by local chromatin structure (reviewed in [13,14]). Typically, the most efficient origins (i.e. the origins that are used in most cell cycles) are found in euchromatin, while less efficient origins are associated with heterochromatin. While there is intense interest in the impact of chromatin structure on origin function, the relevant molecular features of origin-adjacent chromatin and the steps in origin function that they control remain incompletely understood.

Yeast heterochromatin is characterized by hypoacetylated nucleosomes and associated heterochromatin regulatory proteins that promote a compact chromatin structure that makes the underlying DNA inaccessible to protein-DNA interactions and processes such as transcription and DNA replication initiation (recently reviewed in [15]). The Sir2 (Silent information regulator) protein, the founding member of a family of conserved NAD-dependent protein deacetylases, removes an acetyl group from lysine 16 of histone H4 (H4K16) and is required for heterochromatin formation in budding yeast [16–19]. Heterochromatin domains form at only a few discrete regions in the yeast genome, rDNA, telomeres, and the HM-mating type loci, and at each of these loci, Sir2 both deacetylates H4K16ac and stably binds to nucleosomes [15]. Heterochromatin formation at the HM loci and telomeres also requires nucleosome binding by the Sir3 and Sir4 proteins.

While it is recognized that the function of normally efficient origins can be suppressed when they are engineered within heterochromatic regions of the genome [20,21], several more recent studies reveal that SIR2 can also affect the function of euchromatic origins [22,23]. However, depending on experimental context, SIR2 can be interpreted to exert either positive or negative effects on euchromatic origins. In particular, recent studies reveal that SIR2 acts indirectly as a positive regulator of euchromatic origins by suppressing the function of rDNA origins [24,25]. The tandemly repeated rDNA gene locus on chromosome XII contains ~200 replication origins, but ~80% of these origins are suppressed by Sir2-dependent heterochromatin [24]. Because the rDNA origins account for >30% of all yeast genomic origins, deletion of SIR2 results in activation of many rDNA origins, which then sequester limiting S-phase origin activation factors from euchromatic origins, delaying their activation time in S-phase [26,27]. Thus, the positive role for SIR2 in regulating euchromatic origin function in these reports is explained as a byproduct of a direct inhibition of rDNA origins by Sir2.

Interestingly, other studies establish that SIR2 can also exert a negative effect at euchromatic origins. This negative role was revealed by a classic genetic screen that isolated sir2 mutants as suppressors of the temperature-sensitive cdc6-4 allele [23,28,29]. Cdc6, a member of the AAA+ protein family, must bind ATP to load dhMCM [30,31]. The cdc6-4 allele encodes a mutant Cdc6 with a lysine to alanine substitution in the conserved ATP binding motif. Cells with the cdc6-4 allele are viable but have S-phase associated growth defects at the permissive temperature and arrest growth at the non-permissive temperature with a failure to load dhMCM, as assessed by origin-specific ChIPs. Like sir2Δ, sir3Δ is also a robust suppressor of the cdc6-4 growth defect, whereas a sir4Δ mutant is only a very weak suppressor [29]. Thus the suppression by sir2Δ or sir3Δ is not easily explained by defects in classic Sir-heterochromatic gene silencing of known loci because Sir2, Sir3 and Sir4 are each equally essential for HM- and telomere silencing and only Sir2 is required for rDNA silencing. In addition, multiple sir2 alleles specifically defective in rDNA silencing do not suppress cdc6-4 ts lethality, indicating that a loss of rDNA silencing is not sufficient to explain SIR2’s negative effect on euchromatic origins [29]. These findings support a distinct, rDNA- and classic heterochromatin-independent function of SIR2 on euchromatic origins. Importantly, a sir2 catalytic mutant or a histone mutant that converts H4K16 into a residue that mimics acetylated H4K16 (H4K16Q) suppresses cdc6-4, indicating that the Sir2 deacetylase function is critical for its negative impact on euchromatic origins [23,29]. However, these data do not address whether the negative role exerted by SIR2 is due to direct roles for Sir2 within euchromatic regions of the genome, or whether SIR2 and SIR3 affect euchromatic origins through a shared mechanism.

In this report we investigated the molecular mechanisms by which SIR2 affected euchromatic origins by performing MCM ChIP-Seq and MNase-H4K16ac ChIP-Seq experiments and also by analyzing several published high-resolution ChIP-Seq experiments [32–34]. Our results provide evidence for a direct role for Sir2 and Sir3 in forming a repressive local chromatin environment around most origins that exist within euchromatin. A sir2Δ mutation was sufficient to restore MCM binding at the majority of euchromatic origins in cdc6-4 cells even at the non-permissive growth temperature for this allele. In wild type cells, nucleosomes immediately adjacent to the majority of euchromatic origins were relatively hypoacetylated on H4K16 compared to non-origin control loci, a behavior unique to this histone acetylation mark. Moreover, this origin-specific H4K16 hypoacetylation was completely dependent on SIR2. In addition, Sir2 and Sir3 were physically associated with euchromatic origins but not non-origin control loci. The levels of these three distinct molecular features of yeast heterochromatin—hypoacetylation of H4K16, Sir2 and Sir3 binding—correlated with how strongly a SIR2 deletion suppressed the MCM loading defect in cdc6-4 cells. Based on these results we propose that the dhMCM loading reaction has evolved to work within a potentially repressive Sir2,3-chromatin environment that forms around most euchromatic origins. Consistent with this model, a screen for suppressors of cdc6-4 temperature-sensitive lethality identified several histone mutants known to disrupt the Sir3-nucleosome binding interface.

ResultsSIR2 inhibited MCM loading at most euchromatic origins in cdc6-4 mutant cellsTo examine the extent to which a SIR2 deletion (sir2Δ) suppresses defects in MCM loading caused by the cdc6-4 allele, we assessed MCM binding by ChIP-Seq in four congenic yeast strains: wild type (SIR2 CDC6); sir2Δ; cdc6-4; and sir2Δ cdc6-4. Because sir2Δ rescues the temperature-sensitive growth defect of cdc6-4 [29], the experiment was performed using chromatin from cells incubated at 37°C, the non-permissive growth temperature for cdc6-4. Cells were arrested in M-phase at the permissive growth temperature (25°C), shifted to 37°C, and then released into G1-phase to allow time for MCM loading ((Fig 1A and S1 Fig). MCM ChIP-Seq was performed for each strain using a monoclonal antibody against Mcm2 [23]. Examination of MCM ChIP-Seq signals across the genome, as illustrated for chromosomes III and VI, revealed that MCM distribution in wild type and sir2Δ cells was similar (Fig 1B and 1C, S2 Fig). Cdc6-4 cells failed to produce MCM ChIP-Seq signals above background levels, consistent with the essential role of Cdc6 in MCM loading (Fig 1B and 1C). In contrast, cdc6-4 sir2Δ mutant cells showed MCM association, albeit to varying degrees, at most origins. (Fig 1B and 1C, S2 Fig). Therefore, deletion of SIR2 rescued origin-specific association of MCM at the majority of chromosomal origins in cdc6-4 mutant yeast, consistent with SIR2 having a pervasive negative effect at the majority of euchromatic origins.10.1371/journal.pgen.1007418.g001Fig 1SIR2 inhibited MCM association at the majority of euchromatic origins in cdc6-4 mutant cells.A. Experimental outline: MCM binding to chromosomal DNA was examined by ChIP-Seq from congenic CDC6 SIR2 (wild type), CDC6 sir2Δ (sir2Δ), cdc6-4 SIR2 (cdc6-4), and cdc6-4 sir2Δ cells. Cells were released from a G2/M nocodazole-arrest into G1 phase at 37°C prior to formaldehyde crosslinking. B. MCM ChIP-Seq signals across Chromosome III. The x-axis indicates the chromosomal coordinates, the y-axis the normalized MCM read counts (MCM signal). The scale on the y-axis is the same for each of the four strains. Confirmed origins on Chromosome III are indicated by boxes and numbers. Black boxes indicate origins that were defined as SIR2-responsive because they provided for plasmid replication in cdc6-4 sir2Δ cells at the non-permissive temperature for cdc6-4 [23]. The arrowhead and the line-with-circle indicate ARS305 and ARS307, respectively that produced similar association with MCM in wild type cells but different degrees of MCM association in cdc6-4 sir2Δ cells. These origins are highlighted to illustrate that the wild type pattern of MCM origin distribution was not rescued fully in cdc6-4 sir2Δ cells. The starred origins (ARS301, ARS302, ARS317 and ARS318) are associated with transcriptional silencers that direct the assembly of SIR-heterochromatin at the HML and HMR loci on chromosome III [15]. They were excluded from further analyses in this study for this reason, as discussed in text. C. MCM ChIP-Seq signals across Chromosome VI. The arrowhead and the line-with-circle indicate ARS606 and ARS609, respectively, which behaved analogously to ARS305 and ARS307, respectively, as described in B. D. Origins’ SIR2-responsiveness was defined as the ratio of the MCM signal at the origin in cdc6-4 sir2Δ cells relative to sir2Δ cells. We confined analyses to confirmed origins that produced strong signals in both WT and sir2Δ cells as defined by inclusion among the top 400 chromosomal coordinates of enrichment (S2 Fig). The dotted line indicates the cut-off that captured the euchromatic origins (ARS305, ARS315, ARS603 and ARS606) that had previously been defined as SIR2-responsive origins based on a systematic plasmid-based screen of these chromosomal origins [23].A previous study screened for origins on chromosomes III and VI (Fig 1, gray boxes) that, when cloned onto a plasmid, were functional in cdc6-4 sir2Δ cells growing at the non-permissive temperature for cdc6-4 [23]. Five origins were identified as SIR2-responsive in this screen: ARS317 (the HMR-E silencer origin), ARS305, ARS315, ARS603 and ARS606 (Fig 1B, 1C and 1D, black boxes). ARS317 was not used in subsequent analyses here because it did not produce a robust MCM ChIP-Seq signal in wild type cells, and because we eliminated all origins that exist within heterochromatic domains for deeper analyses of euchromatic origins, as described below. The plasmid-based study suggested that at least 20% of yeast origins were likely to be SIR2-responsive [23]. In contrast, the MCM-ChIP-Seq revealed that ~80% of origins showed MCM binding in cdc6-4 sir2Δ cells (Fig 1B, 1C and S2 Fig). We note that the plasmid-based assay demanded that origin function was rescued to a level in cdc6-4 sir2Δ cells that allowed for colonies to form at 37°C, a selection that might have been more stringent than the screen for an MCM ChIP-Seq signal as in Fig 1. In this regard, it is notable that the origins identified as SIR2-responsive in the plasmid screen were indeed among those that showed the most robust rescue of MCM ChIP-Seq signals in cdc6-4 sir2Δ cells (Fig 1D, black boxes). In addition, the plasmid-based screen assessed only small origin fragments that might not have recapitulated a nucleosome organization required to show SIR2-responsiveness. Indeed, plasmid-born ARS1005 (a top origin identified by MCM ChIP-Seq) exhibited SIR2-regulation only when a larger chromosomal region surrounding ARS1005 was present that could accommodate the adjacent chromosomally directed nucleosomes (S3 Fig).Reduction in rDNA copy number did not explain suppression of cdc6-4 by sir mutantsIn yeast, each rDNA repeat contains a single rDNA origin, and these repeats are present in hundreds of tandem copies per cell on chromosome XII (Fig 2A). Some mutants that affect origin function, such as orc2-1, can be suppressed by reducing the rDNA origin-load [35,36]. Therefore, we tested whether reduced rDNA copy number might account for suppression of cdc6-4 by sir2Δ by analyzing the rDNA locus in the strains used for the MCM ChIP-Seq experiment by qPCR (Fig 2B). While rDNA copy number varied from colony to colony even within a single strain, on average, wild type, sir2Δ and cdc6-4 cells had similar levels of rDNA (Fig 2C). However, cdc6-4 sir2Δ cells showed a 2-fold reduction in rDNA levels relative to cdc6-4 cells (P = 0.02), raising the possibility that suppression of cdc6-4 by sir2Δ was mediated, at least in part, by reductions in rDNA copy number (Fig 2C). To further address this issue, we performed two additional experiments. First, we exploited the previous observation that sir3Δ also suppresses cdc6-4 [29]. In contrast to SIR2, SIR3 has no role in rDNA silencing [37,38]. Cdc6-4 sir3Δ cells grew at the non-permissive temperature for cdc6-4, as expected [29] (Fig 2D). However, the rDNA copy number of cdc6-4 sir3Δ yeast was slightly higher compared to cdc6-4 yeast, indicating that reduced rDNA copy number could not explain this suppression (Fig 2E). Second, we asked whether a reduction in rDNA copy number was sufficient to suppress cdc6-4 by generating cdc6-4 cells with ~35 copies of the rDNA locus (rDNA-35) (Fig 2D and S4 Fig). These cells also contained a FOB1 deletion to help maintain rDNA copy number [39,40]. The cdc6-4 rDNA-35 fob1Δ cells failed to grow at the non-permissive temperature and grew no better than the cdc6-4 rDNA-180 fob1Δ cells, which harbor ~180 copies of the rDNA locus (S4 Fig). Thus, reducing rDNA copy number was neither necessary nor sufficient to explain SIR-mediated suppression of cdc6-4. Cells harboring a reduction in rDNA copy number in cdc6-4 sir2Δ populations likely arose because loss of Sir2 increases recombination frequency within the rDNA array, and reduced rDNA copy number is selected for over passaging in many mutants compromised for replication [35,41,42].10.1371/journal.pgen.1007418.g002Fig 2Reduction in rDNA copy number did not account for the suppression of cdc6-4 by sir2Δ.A. Diagram of a single rDNA repeat is shown with the location of the primer pairs used to assess rDNA copy number by qPCR indicated by colored histogram. The rDNA origin is indicated as the open circle. NTS are the non-transcribed regions in the rDNA locus. RFB is the replication fork block element that binds Fob1. B. Enrichment values are equal to 2- ΔCt where ΔCt = Ct (rDNA locus (purple or teal) or ERV46(aqua))—Ct (single copy control locus, RIM15) [72] and equal the predicted average copy number of the target locus. For each strain, three independent colonies were assessed (biological replicates), each represented by a bar with the standard error. Because we observed substantial variation in rDNA copy number between colonies from the same strains (purple and teal bars represent different primer pairs that amplify the rDNA locus), we also used qPCR to assess the copy number of a single copy gene (ERV46) for the same DNA preparations (aqua). Note the different scale for the single-copy experiment as this value should equal only 1.0. C.
SIR2 effect on the average enrichment values of the rDNA locus for each of the strains assessed in C. D. Growth of 10-fold dilutions of the indicated strains was assessed on solid YPD media at the indicated temperatures. E.
SIR3 effect on the average enrichment values of the rDNA locus for each of the indicated strains.Origin-specific depletion of acetylated H4K16-containing nucleosomesThe pervasive yet origin-specific rescue of the MCM loading defect in cdc6-4 cells by sir2Δ prompted us to consider the possibility that Sir2 might function directly on origin-adjacent nucleosomes within euchromatin. Sir2 is a deacetylase with specificity for acetylated lysine 16 of histone H4 (H4K16ac) [19]. Therefore, we used a comprehensive genome-wide histone modification atlas generated by high resolution MNase-ChIP-Seq of yeast nucleosomes to examine the acetylation status of nucleosomes adjacent to euchromatic origins [32] (Fig 3). To perform this analyses, we focused on two distinct groups of euchromatic loci: (i) experimentally-confirmed origins and (ii) non-origin intergenic regions that contain a match to the ORC binding site [43]. At confirmed origins, ORC and MCM binding, as well as origin activity are experimentally documented, while at non-origin intergenic regions with ORC site matches, ORC and MCM binding are not detectable in vivo, and no origin function has been detected (Fig 3A non-origins n = 179; origins; n = 259). These non-origin intergenic regions with ORC site matches serve as controls for origin-specific as opposed to primarily sequence-directed chromatin signatures [43]. 1201 bp fragments from these two distinct groups were aligned with the T-rich strand of the ORC site on the top strand in the 5’ to 3’ orientation, so that three nucleosomes positioned on either site of the origin could be examined (Fig 3B). Previous genome-scale studies have not identified Sir binding or function at euchromatic origins [34,44,45]. Thus, we expected that if SIR2 was acting on nucleosomes adjacent to euchromatic origins, its effect would be weak compared to its effect on nucleosomes within known SIR2-heterochromatic regions. Therefore, to avoid H4K16ac depletion within known silent chromatin regions masking signals from weaker but potentially physiologically relevant Sir effects on euchromatin, the HM silencer, telomeric (as defined by origins within 15 kb of chromosome ends) and rDNA origins were excluded from these analyses. We then used the average nucleosome occupancy data from this recent nucleosome modification study [32] to confirm that we could recapitulate previously published results about nucleosome occupancy and positioning differences between origin and non-origin loci [43,46]. The results confirmed the conclusion that origin-adjacent nucleosomes show high occupancy at more defined positions around the nucleosome-depleted origins compared to the control non-origin ORC-site containing control loci (Fig 3C). Next, the relative H4K16ac status of nucleosomes surrounding the ORC site for both groups was determined and normalized to the average H4K16ac status from a collection of nucleosomes present in a distinct collection of euchromatin intergenic regions that contained neither origins nor matches to the ORC site (n = 239). This analysis revealed that nucleosomes adjacent to origins were relatively depleted for H4K16ac but that nucleosomes adjacent to the control non-origin nucleosomes were not (Fig 3D). H3K9ac is also a potential substrate for Sir2 but, an H3K9Q substitution fails to suppress cdc6-4 [19,23]. In contrast to H4K16ac, H3K9ac was depleted similarly from nucleosomes adjacent to the origin and non-origin ORC-site control loci (Fig 3D). In fact, H4K16ac was distinct among nucleosome acetylation marks in showing origin-specific depletion (S5 Fig).10.1371/journal.pgen.1007418.g003Fig 3Acetylated H4K16 is depleted from origin-adjacent nucleosomes.A. Two groups of loci were analyzed, origins, defined as experimentally-confirmed origins with a confirmed or high-confidence ORC site, n = 259, and intergenic non-origin loci with ORC site matches but for which no origin activity has been observed, n = 179 [43]. These two different groups of loci were compared to determine whether chromatin-states at origins were specific for origin function as opposed to being the result of underlying AT-rich sequence elements present in origins. “ORC in vitro” refers to loci that were bound by purified ORC in a genomic electrophoretic mobility assay [73]; “ORC in vivo” and “MCM in vivo” refer to the fraction of sites in these two groups that were detected by ORC- or MCM-ChIP, respectively. B. The WebLogo consensus for the ORC sites (or matches) in origins and non-origins, respectively, are shown above the diagram of the fragments used in the analyses of adjacent nucleosomes. Each fragment analyzed was oriented with the T-rich strand of the ORC site 5’ to 3’ on the top strand, and the first nucleotide of the ORC site was designated as position “0”. The fragments were 1201 bp such that six proximal nucleosomes, shown as black ovals, three on each side of the ORC site, were assessed. C. Nucleosome occupancy surrounding the origin and non-origin nucleosome depleted regions are shown using the MNase-ChIP-Seq nucleosome occupancy data from [32]. D. Normalized H4K16ac and H3K9ac for each of the six nucleosomes for the two different groups of loci examined. P-values are derived from Student’s T-test comparing the mean of acetylation status between each nucleosome to the mean acetylation status of nucleosomes from the 239 intergenic control loci. E.
SIR2-responsiveness was defined as the ratio of the MCM ChIP-Seq signal in cdc6-4 sir2Δ to sir2Δ cells. The origins were ranked based on SIR2-responsiveness and then divided into quintiles, with the high quintile containing the most SIR2-responsive origins. F. H4K16ac status for the three quintiles of SIR2-responsive origins indicated in ‘E’ was determined as in ‘D’.If depletion of H4K16ac on origin-adjacent nucleosomes was relevant to SIR2-dependent inhibition of MCM loading in cdc6-4 cells, then the origins most responsive to deletion of SIR2 might be expected to show the greatest depletion of H4K16ac. We defined origin SIR2-responsiveness as the ratio of the MCM ChIP-Seq signal in cdc6-4 sir2Δ cells to that in sir2Δ cells; the most SIR2-responsive origins generated ratios near 1.0, suggesting that deletion of SIR2 substantially rescued the MCM loading defect of cdc6-4 (Fig 3E). Comparison of H4K16ac status of nucleosomes surrounding the low, medium, and high-SIR2 responsive origin quintiles revealed that as a group the most SIR2-responsive origins exhibited the greatest depletion of H416ac (Fig 3F). Thus the varying degrees of SIR2-responsiveness to MCM association generally correlated well with the degree of H4K16 hypoacetylation, with the exception of the -1 nucleosome for the low and medium SIR2-responsive origins.In a separate analysis, H4K16ac status of nucleosomes adjacent to euchromatic origins was also examined relative to the genome-wide average level of H4K16ac so that a comparison to origins within established SIR-heterochromatin domains could be made (S6 Fig). This analysis, presented as box-and-whiskers plots to show the variation in H4K16ac at the relevant nucleosomes in each group, also indicated that the most prominent depletion of H4K16ac occurred on the -1 and +1 nucleosome positions for euchromatic origins, and weakened considerably by the -3 and +3 nucleosomes, as also seen in Fig 3F. As expected, the -1 and +1 nucleosomes of euchromatic origins were less depleted for H4K16ac than the analogous nucleosomes for origins within SIR-heterochromatin, with the median values for the two types of origins differing ~3-fold. The differences between H4K16ac status were more striking between the two types of origins (i.e. euchromatic versus SIR-heterochromatic) at nucleosomes more distal from the origin, consistent with the H4K16ac status at euchromatic origins being more localized than the H4K16ac status within SIR-heterochromatin. Analyses of H4K16ac status at a few individual origins also indicated that the -1 and/or +1 nucleosomes were the most likely to show the greatest depletion of H4K16ac, particularly for the most SIR2-responsive origins. (S7 Fig).The depletion of acetylated H4K16-containing nucleosomes around euchromatic origins required SIR2The data described above raised the possibility that Sir2 was deacetylating H4K16ac nucleosomes adjacent to euchromatic origins. To test this possibility, we performed H4K16ac MNase ChIP-Seq experiments on the exact same wild type (SIR2) and sir2Δ cells used for the MCM ChIP-Seq experiment described in Fig 1 (Fig 4A). Analyses of wild type cells recapitulated the published H4K16ac MNase ChIP-Seq results shown in Fig 3D. We note that the relative level of origin-specific H4K16ac depletion was slightly greater in these new experiments, possibly due to differences in strain backgrounds or growth conditions. Regardless, the key result was that, in contrast to wild type cells, depletion of H4K16ac was lost from origin-adjacent nucleosomes in the sir2Δ cells, while the behavior of non-origin ORC site control nucleosomes was unchanged. Importantly, these effects also correlated with SIR2 responsiveness (Fig 4B). Thus, origin-specific depletion of H4K16ac on nucleosomes adjacent to euchromatic origins required SIR2.10.1371/journal.pgen.1007418.g004Fig 4Depletion of H4K16ac from origin-adjacent nucleosomes required SIR2.A. H4K16 acetylation status of nucleosomes was assessed by MNase ChIP-Seq in SIR2 and sir2Δ cells. These cells were the same as those analyzed for MCM binding in Fig 1. B. The H4K16 acetylation status was plotted for the low, medium and high SIR2-responsive quintiles as defined in Fig 3.Sir2 and Sir3 were detected at originsSir2 and Sir3 are physical components of yeast heterochromatin that have been detected at rDNA (Sir2), telomeres and HM loci (Sir2 and Sir3) by ChIP experiments in multiple studies (reviewed in [15]). However, neither protein has been reported to associate with euchromatic origins [44,45,47]. Depletion of H4K16ac at euchromatic origins was estimated to be between 25–35% of that detected at heterochromatic origins for the +1 nucleosome (S6 Fig). This observation is consistent with a proteomic analysis of nucleosome modifications on a plasmid based origin [48]. However, given that Sir2 is an enzyme, stable binding by Sir2, or even Sir3 if it is required only transiently, might not be required to establish or maintain this modification state. To test this possibility, we used more recently published high-resolution ChIP-Seq Sir2 and Sir3 data sets and excluded all origins from known heterochromatin origins, as described above for Figs 3 and 4 [34,44]. This analysis detected Sir2 and Sir3 ChIP-Seq signals on nucleosomes adjacent to origins but not on nucleosomes adjacent to non-origin controls (Fig 5A and 5B). Because the Sir3 data was generated from a high-resolution MNase ChIP-Seq experiment, we also examined these data at nucleotide resolution across the 1201 bp origin and non-origin fragments described in Fig 3B. To generate the baseline for these analyses, the number of reads that contained a given nucleotide in the ChIP sample was divided by the number of reads that contained that nucleotide in the starting material, for every single nucleotide within the euchromatic genome (i.e. nucleotides from known heterochromatin regions were excluded to form the “0” binding baseline for Sir3), following the method in [49] (Fig 5C). These analyses revealed the strongest Sir3 ChIP-Seq signal at the most proximal origin-adjacent nucleosomes (-1 and +1), but signals above baseline were also detectable at the -3, -2, +2 and +3 nucleosomes. As was true for the depletion of H4K16ac, the Sir3 signals correlated with the degree of SIR2-responsiveness as defined in Fig 3E (except in the opposite direction) (Fig 5D and 5E). Randomizing the origins into three new groups of equivalent numbers prior to plotting the Sir3 ChIP-Seq signals eliminated the correlation (Fig 5F).10.1371/journal.pgen.1007418.g005Fig 5Sir2 and Sir3 were physically associated with nucleosomes adjacent to origins.A. Sir2 or (B) Sir3 binding to origin or non-origin adjacent nucleosomes was assessed as in Fig 3 using data from [33,34]. C. The normalized Sir3 ChIP-Seq signal (log2 Sir3 ChIP signal, y-axis) for each nucleotide was plotted over the 1201 bp origin (blue) or non-origin (green) loci (coordinates on x-axis). The Sir3 ChIP-Seq signal represented total reads for each nucleotide normalized to the depth (total reads) and breadth (number of nucleotides with reads) of the sequencing reactions [49]. Genomic regions previously established as SIR-heterochromatic domains were excluded from the normalization but see S8 Fig for an analysis that includes these nucleotides in establishing the baseline. D. and E. The Sir3 ChIP-Signals were plotted as in ‘C’ or ‘B’, respectively, for the three different SIR2-responsive groups defined in Fig 3. F. The SIR2-responsive origins comprising the three groups assessed in ‘D’ were randomized into three different sets and the Sir3 ChIP-Seq signals determined and plotted as in ‘D’.A separate analyses of Sir3-3xHA ChIP-Seq data from the same study indicated that excluding nucleotides from SIR-heterochromatin when establishing the baseline was required for the Sir3 ChIP-Seq signals at euchromatic origins to rise above the “0” baseline [50] (S8 Fig), in contrast to what was observed for H4K16ac (S6 Fig). Nevertheless, in this independent analysis, Sir3-3xHA (detected with anti-HA) signals at euchromatic origins were clearly greater at the -1 and +1 nucleosomes of origins relative to the comparable nucleosomes at the non-origin ORC-site containing loci that have been used as controls for origin specificity throughout this study, and Sir3-3xHA signals also correlated with SIR2-responsiveness (S8B and S8C Fig).We interpret these results to mean that strong Sir3 association with heterochromatic regions reduced the “0” baseline and obscured comparatively weak Sir3 signals at euchromatic origins. Thus, we conclude that Sir3 associated with euchromatic origins but more transiently and at levels less than Sir3’s association with SIR-heterochromatin regions [50].In summary, several molecular hallmarks of SIR-heterochromatin–Sir2, Sir3 and Sir2-dependent H4K16 hypoacetylation–could be detected at euchromatin origins but not at euchromatic non-origin controls. In addition, the relative levels of each of these molecular hallmarks correlated with how strongly a SIR2 deletion restored an MCM ChIP signal to origins in cdc6-4 cells grown at temperatures that otherwise abolished MCM loading.A cdc6-4 suppressor screen targeting core modifiable histone H3 and H4 residues identified a nucleosome surface important for Sir3 bindingHistones H3 and H4 form a tetramer that binds both to dsDNA and to two dimers of histones H2A and H2B to form the nucleosome. Many conserved residues on the histone H3 and H4 N-terminal tails as well as within the globular core region (“core-modifiable” residues) can be post-translationally modified, and, as discussed above, an H4K16Q substitution that should mimic the acetylated form of this residue (i.e. the form predicted to phenocopy the effect of a sir2Δ), suppressed the temperature-sensitive growth defect of cdc6-4 [23]. To test whether we could identify additional alleles with similar suppressive behavior, we assessed a library of 43 histone H3 and H4 mutations affecting 15 distinct residues for suppression of the cdc6-4 growth defect (Fig 6A). Mutant plasmids harboring a single copy of the HHT2-HHF2 locus marked with TRP1 were transformed into wild type and cdc6-4 cells lacking both chromosomal copies of histone H3 and H4 genes, maintained by a HHT2-HHF2 URA3 ARS CEN plasmid, and colonies were subsequently plated on 5-FOA containing media to select against the wild type HHT2-HHF2 plasmid. Recovered cells were then examined for growth on YPD medium at 25°C and higher temperatures. In agreement with previous studies, none of the histone mutants, with the exception of H4-S47E, caused growth defects in wild type cells [51], and several mutants were identified that inhibited growth of cells harboring the cdc6-4 allele even at the permissive growth temperature (25°C). However, relevant to this study, specific substitutions at only two residues, H3K79 (H3K79A and H3K79Q) and H4K79 (H4K79A and H4K79R) suppressed the temperature-sensitive growth defect of cdc6-4 cells (Fig 6B and S9 Fig). Notably, these residues are important for Sir3 binding to the nucleosome and for Sir3-mediated transcriptional silencing at the HM and telomeric loci [52–54]. H3K79 is a substrate for methylation by Dot1 and Sir3 binds preferentially to unmethylated H3K79 [55,56]. Based on these results, additional substitutions were engineered by site-directed mutagenesis at H3K79 and at four adjacent residues (H3K79E, H3E73A, H3E73K, H374A, H3T80A, H3D81A) (Fig 6B). In addition to revealing that H3K79E was a stronger suppressor of cdc6-4 than H3K79Q, analyses of these mutants also revealed that substitutions of H3E73 and H3T80 also suppressed the temperature-sensitive growth defect of cdc6-4 cells (Fig 6B and S9 Fig). H3E73 and H3T80 are important for HM and telomeric silencing [52], and H3T80 directly contacts the LRS domain of Sir3 [53,54]. In summary, suppression of cdc6-4 temperature-sensitivity was achieved by single substitutions within only four residues (H3E73, H3K79, H3T80, and H4K79) of H3/H4. Each of these residues clustered within a small patch on the nucleosome surface important for binding Sir3 (Fig 6C, highlighted in red) [53], strongly suggesting that disruption of Sir3 binding to nucleosomes suppressed the cdc6-4 growth defect. Together with the ChIP-Seq data, these genetic data supported the conclusion that both Sir2 and Sir3 alter the molecular features of nucleosomes adjacent to euchromatic origins, thus generating a local chromatin environment that can negatively impact the MCM loading reaction.10.1371/journal.pgen.1007418.g006Fig 6Histone H3 and H4 core modifiable alleles that suppress cdc6-4 temperature sensitivity define a known Sir3-nucleosome interface.A. List of histone H3 and H4 substitutions within the mutant library that were screened for their ability to affect the growth of cdc6-4 mutant cells (see also S9 Fig). B. 10-fold serial dilutions of M1345 (cdc6-4) with either wild type H3, H4 or selected histone variants that were from the original library (Fig 6A) or were generated by site-directed mutagenesis (see text), were spotted onto YPD and grown at the indicated temperatures. Asterisks (*) indicate the histone variants that suppressed cdc6-4 temperature-sensitive growth. H4K59A and H4K59R are shown as examples of histone mutants that did not suppress. C. Nucleosome structure (PDB-1ID3) with one H3(dark blue)-H4(light blue) dimer and the H3K79, H3T80, H4E74 and H4K79 residues highlighted in red.

SIR2 inhibited MCM loading at most euchromatic origins in cdc6-4 mutant cellsTo examine the extent to which a SIR2 deletion (sir2Δ) suppresses defects in MCM loading caused by the cdc6-4 allele, we assessed MCM binding by ChIP-Seq in four congenic yeast strains: wild type (SIR2 CDC6); sir2Δ; cdc6-4; and sir2Δ cdc6-4. Because sir2Δ rescues the temperature-sensitive growth defect of cdc6-4 [29], the experiment was performed using chromatin from cells incubated at 37°C, the non-permissive growth temperature for cdc6-4. Cells were arrested in M-phase at the permissive growth temperature (25°C), shifted to 37°C, and then released into G1-phase to allow time for MCM loading ((Fig 1A and S1 Fig). MCM ChIP-Seq was performed for each strain using a monoclonal antibody against Mcm2 [23]. Examination of MCM ChIP-Seq signals across the genome, as illustrated for chromosomes III and VI, revealed that MCM distribution in wild type and sir2Δ cells was similar (Fig 1B and 1C, S2 Fig). Cdc6-4 cells failed to produce MCM ChIP-Seq signals above background levels, consistent with the essential role of Cdc6 in MCM loading (Fig 1B and 1C). In contrast, cdc6-4 sir2Δ mutant cells showed MCM association, albeit to varying degrees, at most origins. (Fig 1B and 1C, S2 Fig). Therefore, deletion of SIR2 rescued origin-specific association of MCM at the majority of chromosomal origins in cdc6-4 mutant yeast, consistent with SIR2 having a pervasive negative effect at the majority of euchromatic origins.10.1371/journal.pgen.1007418.g001Fig 1SIR2 inhibited MCM association at the majority of euchromatic origins in cdc6-4 mutant cells.A. Experimental outline: MCM binding to chromosomal DNA was examined by ChIP-Seq from congenic CDC6 SIR2 (wild type), CDC6 sir2Δ (sir2Δ), cdc6-4 SIR2 (cdc6-4), and cdc6-4 sir2Δ cells. Cells were released from a G2/M nocodazole-arrest into G1 phase at 37°C prior to formaldehyde crosslinking. B. MCM ChIP-Seq signals across Chromosome III. The x-axis indicates the chromosomal coordinates, the y-axis the normalized MCM read counts (MCM signal). The scale on the y-axis is the same for each of the four strains. Confirmed origins on Chromosome III are indicated by boxes and numbers. Black boxes indicate origins that were defined as SIR2-responsive because they provided for plasmid replication in cdc6-4 sir2Δ cells at the non-permissive temperature for cdc6-4 [23]. The arrowhead and the line-with-circle indicate ARS305 and ARS307, respectively that produced similar association with MCM in wild type cells but different degrees of MCM association in cdc6-4 sir2Δ cells. These origins are highlighted to illustrate that the wild type pattern of MCM origin distribution was not rescued fully in cdc6-4 sir2Δ cells. The starred origins (ARS301, ARS302, ARS317 and ARS318) are associated with transcriptional silencers that direct the assembly of SIR-heterochromatin at the HML and HMR loci on chromosome III [15]. They were excluded from further analyses in this study for this reason, as discussed in text. C. MCM ChIP-Seq signals across Chromosome VI. The arrowhead and the line-with-circle indicate ARS606 and ARS609, respectively, which behaved analogously to ARS305 and ARS307, respectively, as described in B. D. Origins’ SIR2-responsiveness was defined as the ratio of the MCM signal at the origin in cdc6-4 sir2Δ cells relative to sir2Δ cells. We confined analyses to confirmed origins that produced strong signals in both WT and sir2Δ cells as defined by inclusion among the top 400 chromosomal coordinates of enrichment (S2 Fig). The dotted line indicates the cut-off that captured the euchromatic origins (ARS305, ARS315, ARS603 and ARS606) that had previously been defined as SIR2-responsive origins based on a systematic plasmid-based screen of these chromosomal origins [23].A previous study screened for origins on chromosomes III and VI (Fig 1, gray boxes) that, when cloned onto a plasmid, were functional in cdc6-4 sir2Δ cells growing at the non-permissive temperature for cdc6-4 [23]. Five origins were identified as SIR2-responsive in this screen: ARS317 (the HMR-E silencer origin), ARS305, ARS315, ARS603 and ARS606 (Fig 1B, 1C and 1D, black boxes). ARS317 was not used in subsequent analyses here because it did not produce a robust MCM ChIP-Seq signal in wild type cells, and because we eliminated all origins that exist within heterochromatic domains for deeper analyses of euchromatic origins, as described below. The plasmid-based study suggested that at least 20% of yeast origins were likely to be SIR2-responsive [23]. In contrast, the MCM-ChIP-Seq revealed that ~80% of origins showed MCM binding in cdc6-4 sir2Δ cells (Fig 1B, 1C and S2 Fig). We note that the plasmid-based assay demanded that origin function was rescued to a level in cdc6-4 sir2Δ cells that allowed for colonies to form at 37°C, a selection that might have been more stringent than the screen for an MCM ChIP-Seq signal as in Fig 1. In this regard, it is notable that the origins identified as SIR2-responsive in the plasmid screen were indeed among those that showed the most robust rescue of MCM ChIP-Seq signals in cdc6-4 sir2Δ cells (Fig 1D, black boxes). In addition, the plasmid-based screen assessed only small origin fragments that might not have recapitulated a nucleosome organization required to show SIR2-responsiveness. Indeed, plasmid-born ARS1005 (a top origin identified by MCM ChIP-Seq) exhibited SIR2-regulation only when a larger chromosomal region surrounding ARS1005 was present that could accommodate the adjacent chromosomally directed nucleosomes (S3 Fig).

SIR2 inhibited MCM loading at most euchromatic origins in cdc6-4 mutant cells

To examine the extent to which a SIR2 deletion (sir2Δ) suppresses defects in MCM loading caused by the cdc6-4 allele, we assessed MCM binding by ChIP-Seq in four congenic yeast strains: wild type (SIR2 CDC6); sir2Δ; cdc6-4; and sir2Δ cdc6-4. Because sir2Δ rescues the temperature-sensitive growth defect of cdc6-4 [29], the experiment was performed using chromatin from cells incubated at 37°C, the non-permissive growth temperature for cdc6-4. Cells were arrested in M-phase at the permissive growth temperature (25°C), shifted to 37°C, and then released into G1-phase to allow time for MCM loading ((Fig 1A and S1 Fig). MCM ChIP-Seq was performed for each strain using a monoclonal antibody against Mcm2 [23]. Examination of MCM ChIP-Seq signals across the genome, as illustrated for chromosomes III and VI, revealed that MCM distribution in wild type and sir2Δ cells was similar (Fig 1B and 1C, S2 Fig). Cdc6-4 cells failed to produce MCM ChIP-Seq signals above background levels, consistent with the essential role of Cdc6 in MCM loading (Fig 1B and 1C). In contrast, cdc6-4 sir2Δ mutant cells showed MCM association, albeit to varying degrees, at most origins. (Fig 1B and 1C, S2 Fig). Therefore, deletion of SIR2 rescued origin-specific association of MCM at the majority of chromosomal origins in cdc6-4 mutant yeast, consistent with SIR2 having a pervasive negative effect at the majority of euchromatic origins.

SIR2 inhibited MCM association at the majority of euchromatic origins in cdc6-4 mutant cells.

A. Experimental outline: MCM binding to chromosomal DNA was examined by ChIP-Seq from congenic CDC6 SIR2 (wild type), CDC6 sir2Δ (sir2Δ), cdc6-4 SIR2 (cdc6-4), and cdc6-4 sir2Δ cells. Cells were released from a G2/M nocodazole-arrest into G1 phase at 37°C prior to formaldehyde crosslinking. B. MCM ChIP-Seq signals across Chromosome III. The x-axis indicates the chromosomal coordinates, the y-axis the normalized MCM read counts (MCM signal). The scale on the y-axis is the same for each of the four strains. Confirmed origins on Chromosome III are indicated by boxes and numbers. Black boxes indicate origins that were defined as SIR2-responsive because they provided for plasmid replication in cdc6-4 sir2Δ cells at the non-permissive temperature for cdc6-4 [23]. The arrowhead and the line-with-circle indicate ARS305 and ARS307, respectively that produced similar association with MCM in wild type cells but different degrees of MCM association in cdc6-4 sir2Δ cells. These origins are highlighted to illustrate that the wild type pattern of MCM origin distribution was not rescued fully in cdc6-4 sir2Δ cells. The starred origins (ARS301, ARS302, ARS317 and ARS318) are associated with transcriptional silencers that direct the assembly of SIR-heterochromatin at the HML and HMR loci on chromosome III [15]. They were excluded from further analyses in this study for this reason, as discussed in text. C. MCM ChIP-Seq signals across Chromosome VI. The arrowhead and the line-with-circle indicate ARS606 and ARS609, respectively, which behaved analogously to ARS305 and ARS307, respectively, as described in B. D. Origins’ SIR2-responsiveness was defined as the ratio of the MCM signal at the origin in cdc6-4 sir2Δ cells relative to sir2Δ cells. We confined analyses to confirmed origins that produced strong signals in both WT and sir2Δ cells as defined by inclusion among the top 400 chromosomal coordinates of enrichment (S2 Fig). The dotted line indicates the cut-off that captured the euchromatic origins (ARS305, ARS315, ARS603 and ARS606) that had previously been defined as SIR2-responsive origins based on a systematic plasmid-based screen of these chromosomal origins [23].

A previous study screened for origins on chromosomes III and VI (Fig 1, gray boxes) that, when cloned onto a plasmid, were functional in cdc6-4 sir2Δ cells growing at the non-permissive temperature for cdc6-4 [23]. Five origins were identified as SIR2-responsive in this screen: ARS317 (the HMR-E silencer origin), ARS305, ARS315, ARS603 and ARS606 (Fig 1B, 1C and 1D, black boxes). ARS317 was not used in subsequent analyses here because it did not produce a robust MCM ChIP-Seq signal in wild type cells, and because we eliminated all origins that exist within heterochromatic domains for deeper analyses of euchromatic origins, as described below. The plasmid-based study suggested that at least 20% of yeast origins were likely to be SIR2-responsive [23]. In contrast, the MCM-ChIP-Seq revealed that ~80% of origins showed MCM binding in cdc6-4 sir2Δ cells (Fig 1B, 1C and S2 Fig). We note that the plasmid-based assay demanded that origin function was rescued to a level in cdc6-4 sir2Δ cells that allowed for colonies to form at 37°C, a selection that might have been more stringent than the screen for an MCM ChIP-Seq signal as in Fig 1. In this regard, it is notable that the origins identified as SIR2-responsive in the plasmid screen were indeed among those that showed the most robust rescue of MCM ChIP-Seq signals in cdc6-4 sir2Δ cells (Fig 1D, black boxes). In addition, the plasmid-based screen assessed only small origin fragments that might not have recapitulated a nucleosome organization required to show SIR2-responsiveness. Indeed, plasmid-born ARS1005 (a top origin identified by MCM ChIP-Seq) exhibited SIR2-regulation only when a larger chromosomal region surrounding ARS1005 was present that could accommodate the adjacent chromosomally directed nucleosomes (S3 Fig).

Reduction in rDNA copy number did not explain suppression of cdc6-4 by sir mutantsIn yeast, each rDNA repeat contains a single rDNA origin, and these repeats are present in hundreds of tandem copies per cell on chromosome XII (Fig 2A). Some mutants that affect origin function, such as orc2-1, can be suppressed by reducing the rDNA origin-load [35,36]. Therefore, we tested whether reduced rDNA copy number might account for suppression of cdc6-4 by sir2Δ by analyzing the rDNA locus in the strains used for the MCM ChIP-Seq experiment by qPCR (Fig 2B). While rDNA copy number varied from colony to colony even within a single strain, on average, wild type, sir2Δ and cdc6-4 cells had similar levels of rDNA (Fig 2C). However, cdc6-4 sir2Δ cells showed a 2-fold reduction in rDNA levels relative to cdc6-4 cells (P = 0.02), raising the possibility that suppression of cdc6-4 by sir2Δ was mediated, at least in part, by reductions in rDNA copy number (Fig 2C). To further address this issue, we performed two additional experiments. First, we exploited the previous observation that sir3Δ also suppresses cdc6-4 [29]. In contrast to SIR2, SIR3 has no role in rDNA silencing [37,38]. Cdc6-4 sir3Δ cells grew at the non-permissive temperature for cdc6-4, as expected [29] (Fig 2D). However, the rDNA copy number of cdc6-4 sir3Δ yeast was slightly higher compared to cdc6-4 yeast, indicating that reduced rDNA copy number could not explain this suppression (Fig 2E). Second, we asked whether a reduction in rDNA copy number was sufficient to suppress cdc6-4 by generating cdc6-4 cells with ~35 copies of the rDNA locus (rDNA-35) (Fig 2D and S4 Fig). These cells also contained a FOB1 deletion to help maintain rDNA copy number [39,40]. The cdc6-4 rDNA-35 fob1Δ cells failed to grow at the non-permissive temperature and grew no better than the cdc6-4 rDNA-180 fob1Δ cells, which harbor ~180 copies of the rDNA locus (S4 Fig). Thus, reducing rDNA copy number was neither necessary nor sufficient to explain SIR-mediated suppression of cdc6-4. Cells harboring a reduction in rDNA copy number in cdc6-4 sir2Δ populations likely arose because loss of Sir2 increases recombination frequency within the rDNA array, and reduced rDNA copy number is selected for over passaging in many mutants compromised for replication [35,41,42].10.1371/journal.pgen.1007418.g002Fig 2Reduction in rDNA copy number did not account for the suppression of cdc6-4 by sir2Δ.A. Diagram of a single rDNA repeat is shown with the location of the primer pairs used to assess rDNA copy number by qPCR indicated by colored histogram. The rDNA origin is indicated as the open circle. NTS are the non-transcribed regions in the rDNA locus. RFB is the replication fork block element that binds Fob1. B. Enrichment values are equal to 2- ΔCt where ΔCt = Ct (rDNA locus (purple or teal) or ERV46(aqua))—Ct (single copy control locus, RIM15) [72] and equal the predicted average copy number of the target locus. For each strain, three independent colonies were assessed (biological replicates), each represented by a bar with the standard error. Because we observed substantial variation in rDNA copy number between colonies from the same strains (purple and teal bars represent different primer pairs that amplify the rDNA locus), we also used qPCR to assess the copy number of a single copy gene (ERV46) for the same DNA preparations (aqua). Note the different scale for the single-copy experiment as this value should equal only 1.0. C.
SIR2 effect on the average enrichment values of the rDNA locus for each of the strains assessed in C. D. Growth of 10-fold dilutions of the indicated strains was assessed on solid YPD media at the indicated temperatures. E.
SIR3 effect on the average enrichment values of the rDNA locus for each of the indicated strains.

Reduction in rDNA copy number did not explain suppression of cdc6-4 by sir mutants

In yeast, each rDNA repeat contains a single rDNA origin, and these repeats are present in hundreds of tandem copies per cell on chromosome XII (Fig 2A). Some mutants that affect origin function, such as orc2-1, can be suppressed by reducing the rDNA origin-load [35,36]. Therefore, we tested whether reduced rDNA copy number might account for suppression of cdc6-4 by sir2Δ by analyzing the rDNA locus in the strains used for the MCM ChIP-Seq experiment by qPCR (Fig 2B). While rDNA copy number varied from colony to colony even within a single strain, on average, wild type, sir2Δ and cdc6-4 cells had similar levels of rDNA (Fig 2C). However, cdc6-4 sir2Δ cells showed a 2-fold reduction in rDNA levels relative to cdc6-4 cells (P = 0.02), raising the possibility that suppression of cdc6-4 by sir2Δ was mediated, at least in part, by reductions in rDNA copy number (Fig 2C). To further address this issue, we performed two additional experiments. First, we exploited the previous observation that sir3Δ also suppresses cdc6-4 [29]. In contrast to SIR2, SIR3 has no role in rDNA silencing [37,38]. Cdc6-4 sir3Δ cells grew at the non-permissive temperature for cdc6-4, as expected [29] (Fig 2D). However, the rDNA copy number of cdc6-4 sir3Δ yeast was slightly higher compared to cdc6-4 yeast, indicating that reduced rDNA copy number could not explain this suppression (Fig 2E). Second, we asked whether a reduction in rDNA copy number was sufficient to suppress cdc6-4 by generating cdc6-4 cells with ~35 copies of the rDNA locus (rDNA-35) (Fig 2D and S4 Fig). These cells also contained a FOB1 deletion to help maintain rDNA copy number [39,40]. The cdc6-4 rDNA-35 fob1Δ cells failed to grow at the non-permissive temperature and grew no better than the cdc6-4 rDNA-180 fob1Δ cells, which harbor ~180 copies of the rDNA locus (S4 Fig). Thus, reducing rDNA copy number was neither necessary nor sufficient to explain SIR-mediated suppression of cdc6-4. Cells harboring a reduction in rDNA copy number in cdc6-4 sir2Δ populations likely arose because loss of Sir2 increases recombination frequency within the rDNA array, and reduced rDNA copy number is selected for over passaging in many mutants compromised for replication [35,41,42].

Reduction in rDNA copy number did not account for the suppression of cdc6-4 by sir2Δ.

A. Diagram of a single rDNA repeat is shown with the location of the primer pairs used to assess rDNA copy number by qPCR indicated by colored histogram. The rDNA origin is indicated as the open circle. NTS are the non-transcribed regions in the rDNA locus. RFB is the replication fork block element that binds Fob1. B. Enrichment values are equal to 2- ΔCt where ΔCt = Ct (rDNA locus (purple or teal) or ERV46(aqua))—Ct (single copy control locus, RIM15) [72] and equal the predicted average copy number of the target locus. For each strain, three independent colonies were assessed (biological replicates), each represented by a bar with the standard error. Because we observed substantial variation in rDNA copy number between colonies from the same strains (purple and teal bars represent different primer pairs that amplify the rDNA locus), we also used qPCR to assess the copy number of a single copy gene (ERV46) for the same DNA preparations (aqua). Note the different scale for the single-copy experiment as this value should equal only 1.0. C.
SIR2 effect on the average enrichment values of the rDNA locus for each of the strains assessed in C. D. Growth of 10-fold dilutions of the indicated strains was assessed on solid YPD media at the indicated temperatures. E.
SIR3 effect on the average enrichment values of the rDNA locus for each of the indicated strains.

Origin-specific depletion of acetylated H4K16-containing nucleosomesThe pervasive yet origin-specific rescue of the MCM loading defect in cdc6-4 cells by sir2Δ prompted us to consider the possibility that Sir2 might function directly on origin-adjacent nucleosomes within euchromatin. Sir2 is a deacetylase with specificity for acetylated lysine 16 of histone H4 (H4K16ac) [19]. Therefore, we used a comprehensive genome-wide histone modification atlas generated by high resolution MNase-ChIP-Seq of yeast nucleosomes to examine the acetylation status of nucleosomes adjacent to euchromatic origins [32] (Fig 3). To perform this analyses, we focused on two distinct groups of euchromatic loci: (i) experimentally-confirmed origins and (ii) non-origin intergenic regions that contain a match to the ORC binding site [43]. At confirmed origins, ORC and MCM binding, as well as origin activity are experimentally documented, while at non-origin intergenic regions with ORC site matches, ORC and MCM binding are not detectable in vivo, and no origin function has been detected (Fig 3A non-origins n = 179; origins; n = 259). These non-origin intergenic regions with ORC site matches serve as controls for origin-specific as opposed to primarily sequence-directed chromatin signatures [43]. 1201 bp fragments from these two distinct groups were aligned with the T-rich strand of the ORC site on the top strand in the 5’ to 3’ orientation, so that three nucleosomes positioned on either site of the origin could be examined (Fig 3B). Previous genome-scale studies have not identified Sir binding or function at euchromatic origins [34,44,45]. Thus, we expected that if SIR2 was acting on nucleosomes adjacent to euchromatic origins, its effect would be weak compared to its effect on nucleosomes within known SIR2-heterochromatic regions. Therefore, to avoid H4K16ac depletion within known silent chromatin regions masking signals from weaker but potentially physiologically relevant Sir effects on euchromatin, the HM silencer, telomeric (as defined by origins within 15 kb of chromosome ends) and rDNA origins were excluded from these analyses. We then used the average nucleosome occupancy data from this recent nucleosome modification study [32] to confirm that we could recapitulate previously published results about nucleosome occupancy and positioning differences between origin and non-origin loci [43,46]. The results confirmed the conclusion that origin-adjacent nucleosomes show high occupancy at more defined positions around the nucleosome-depleted origins compared to the control non-origin ORC-site containing control loci (Fig 3C). Next, the relative H4K16ac status of nucleosomes surrounding the ORC site for both groups was determined and normalized to the average H4K16ac status from a collection of nucleosomes present in a distinct collection of euchromatin intergenic regions that contained neither origins nor matches to the ORC site (n = 239). This analysis revealed that nucleosomes adjacent to origins were relatively depleted for H4K16ac but that nucleosomes adjacent to the control non-origin nucleosomes were not (Fig 3D). H3K9ac is also a potential substrate for Sir2 but, an H3K9Q substitution fails to suppress cdc6-4 [19,23]. In contrast to H4K16ac, H3K9ac was depleted similarly from nucleosomes adjacent to the origin and non-origin ORC-site control loci (Fig 3D). In fact, H4K16ac was distinct among nucleosome acetylation marks in showing origin-specific depletion (S5 Fig).10.1371/journal.pgen.1007418.g003Fig 3Acetylated H4K16 is depleted from origin-adjacent nucleosomes.A. Two groups of loci were analyzed, origins, defined as experimentally-confirmed origins with a confirmed or high-confidence ORC site, n = 259, and intergenic non-origin loci with ORC site matches but for which no origin activity has been observed, n = 179 [43]. These two different groups of loci were compared to determine whether chromatin-states at origins were specific for origin function as opposed to being the result of underlying AT-rich sequence elements present in origins. “ORC in vitro” refers to loci that were bound by purified ORC in a genomic electrophoretic mobility assay [73]; “ORC in vivo” and “MCM in vivo” refer to the fraction of sites in these two groups that were detected by ORC- or MCM-ChIP, respectively. B. The WebLogo consensus for the ORC sites (or matches) in origins and non-origins, respectively, are shown above the diagram of the fragments used in the analyses of adjacent nucleosomes. Each fragment analyzed was oriented with the T-rich strand of the ORC site 5’ to 3’ on the top strand, and the first nucleotide of the ORC site was designated as position “0”. The fragments were 1201 bp such that six proximal nucleosomes, shown as black ovals, three on each side of the ORC site, were assessed. C. Nucleosome occupancy surrounding the origin and non-origin nucleosome depleted regions are shown using the MNase-ChIP-Seq nucleosome occupancy data from [32]. D. Normalized H4K16ac and H3K9ac for each of the six nucleosomes for the two different groups of loci examined. P-values are derived from Student’s T-test comparing the mean of acetylation status between each nucleosome to the mean acetylation status of nucleosomes from the 239 intergenic control loci. E.
SIR2-responsiveness was defined as the ratio of the MCM ChIP-Seq signal in cdc6-4 sir2Δ to sir2Δ cells. The origins were ranked based on SIR2-responsiveness and then divided into quintiles, with the high quintile containing the most SIR2-responsive origins. F. H4K16ac status for the three quintiles of SIR2-responsive origins indicated in ‘E’ was determined as in ‘D’.If depletion of H4K16ac on origin-adjacent nucleosomes was relevant to SIR2-dependent inhibition of MCM loading in cdc6-4 cells, then the origins most responsive to deletion of SIR2 might be expected to show the greatest depletion of H4K16ac. We defined origin SIR2-responsiveness as the ratio of the MCM ChIP-Seq signal in cdc6-4 sir2Δ cells to that in sir2Δ cells; the most SIR2-responsive origins generated ratios near 1.0, suggesting that deletion of SIR2 substantially rescued the MCM loading defect of cdc6-4 (Fig 3E). Comparison of H4K16ac status of nucleosomes surrounding the low, medium, and high-SIR2 responsive origin quintiles revealed that as a group the most SIR2-responsive origins exhibited the greatest depletion of H416ac (Fig 3F). Thus the varying degrees of SIR2-responsiveness to MCM association generally correlated well with the degree of H4K16 hypoacetylation, with the exception of the -1 nucleosome for the low and medium SIR2-responsive origins.In a separate analysis, H4K16ac status of nucleosomes adjacent to euchromatic origins was also examined relative to the genome-wide average level of H4K16ac so that a comparison to origins within established SIR-heterochromatin domains could be made (S6 Fig). This analysis, presented as box-and-whiskers plots to show the variation in H4K16ac at the relevant nucleosomes in each group, also indicated that the most prominent depletion of H4K16ac occurred on the -1 and +1 nucleosome positions for euchromatic origins, and weakened considerably by the -3 and +3 nucleosomes, as also seen in Fig 3F. As expected, the -1 and +1 nucleosomes of euchromatic origins were less depleted for H4K16ac than the analogous nucleosomes for origins within SIR-heterochromatin, with the median values for the two types of origins differing ~3-fold. The differences between H4K16ac status were more striking between the two types of origins (i.e. euchromatic versus SIR-heterochromatic) at nucleosomes more distal from the origin, consistent with the H4K16ac status at euchromatic origins being more localized than the H4K16ac status within SIR-heterochromatin. Analyses of H4K16ac status at a few individual origins also indicated that the -1 and/or +1 nucleosomes were the most likely to show the greatest depletion of H4K16ac, particularly for the most SIR2-responsive origins. (S7 Fig).

Origin-specific depletion of acetylated H4K16-containing nucleosomes

The pervasive yet origin-specific rescue of the MCM loading defect in cdc6-4 cells by sir2Δ prompted us to consider the possibility that Sir2 might function directly on origin-adjacent nucleosomes within euchromatin. Sir2 is a deacetylase with specificity for acetylated lysine 16 of histone H4 (H4K16ac) [19]. Therefore, we used a comprehensive genome-wide histone modification atlas generated by high resolution MNase-ChIP-Seq of yeast nucleosomes to examine the acetylation status of nucleosomes adjacent to euchromatic origins [32] (Fig 3). To perform this analyses, we focused on two distinct groups of euchromatic loci: (i) experimentally-confirmed origins and (ii) non-origin intergenic regions that contain a match to the ORC binding site [43]. At confirmed origins, ORC and MCM binding, as well as origin activity are experimentally documented, while at non-origin intergenic regions with ORC site matches, ORC and MCM binding are not detectable in vivo, and no origin function has been detected (Fig 3A non-origins n = 179; origins; n = 259). These non-origin intergenic regions with ORC site matches serve as controls for origin-specific as opposed to primarily sequence-directed chromatin signatures [43]. 1201 bp fragments from these two distinct groups were aligned with the T-rich strand of the ORC site on the top strand in the 5’ to 3’ orientation, so that three nucleosomes positioned on either site of the origin could be examined (Fig 3B). Previous genome-scale studies have not identified Sir binding or function at euchromatic origins [34,44,45]. Thus, we expected that if SIR2 was acting on nucleosomes adjacent to euchromatic origins, its effect would be weak compared to its effect on nucleosomes within known SIR2-heterochromatic regions. Therefore, to avoid H4K16ac depletion within known silent chromatin regions masking signals from weaker but potentially physiologically relevant Sir effects on euchromatin, the HM silencer, telomeric (as defined by origins within 15 kb of chromosome ends) and rDNA origins were excluded from these analyses. We then used the average nucleosome occupancy data from this recent nucleosome modification study [32] to confirm that we could recapitulate previously published results about nucleosome occupancy and positioning differences between origin and non-origin loci [43,46]. The results confirmed the conclusion that origin-adjacent nucleosomes show high occupancy at more defined positions around the nucleosome-depleted origins compared to the control non-origin ORC-site containing control loci (Fig 3C). Next, the relative H4K16ac status of nucleosomes surrounding the ORC site for both groups was determined and normalized to the average H4K16ac status from a collection of nucleosomes present in a distinct collection of euchromatin intergenic regions that contained neither origins nor matches to the ORC site (n = 239). This analysis revealed that nucleosomes adjacent to origins were relatively depleted for H4K16ac but that nucleosomes adjacent to the control non-origin nucleosomes were not (Fig 3D). H3K9ac is also a potential substrate for Sir2 but, an H3K9Q substitution fails to suppress cdc6-4 [19,23]. In contrast to H4K16ac, H3K9ac was depleted similarly from nucleosomes adjacent to the origin and non-origin ORC-site control loci (Fig 3D). In fact, H4K16ac was distinct among nucleosome acetylation marks in showing origin-specific depletion (S5 Fig).

Acetylated H4K16 is depleted from origin-adjacent nucleosomes.

A. Two groups of loci were analyzed, origins, defined as experimentally-confirmed origins with a confirmed or high-confidence ORC site, n = 259, and intergenic non-origin loci with ORC site matches but for which no origin activity has been observed, n = 179 [43]. These two different groups of loci were compared to determine whether chromatin-states at origins were specific for origin function as opposed to being the result of underlying AT-rich sequence elements present in origins. “ORC in vitro” refers to loci that were bound by purified ORC in a genomic electrophoretic mobility assay [73]; “ORC in vivo” and “MCM in vivo” refer to the fraction of sites in these two groups that were detected by ORC- or MCM-ChIP, respectively. B. The WebLogo consensus for the ORC sites (or matches) in origins and non-origins, respectively, are shown above the diagram of the fragments used in the analyses of adjacent nucleosomes. Each fragment analyzed was oriented with the T-rich strand of the ORC site 5’ to 3’ on the top strand, and the first nucleotide of the ORC site was designated as position “0”. The fragments were 1201 bp such that six proximal nucleosomes, shown as black ovals, three on each side of the ORC site, were assessed. C. Nucleosome occupancy surrounding the origin and non-origin nucleosome depleted regions are shown using the MNase-ChIP-Seq nucleosome occupancy data from [32]. D. Normalized H4K16ac and H3K9ac for each of the six nucleosomes for the two different groups of loci examined. P-values are derived from Student’s T-test comparing the mean of acetylation status between each nucleosome to the mean acetylation status of nucleosomes from the 239 intergenic control loci. E.
SIR2-responsiveness was defined as the ratio of the MCM ChIP-Seq signal in cdc6-4 sir2Δ to sir2Δ cells. The origins were ranked based on SIR2-responsiveness and then divided into quintiles, with the high quintile containing the most SIR2-responsive origins. F. H4K16ac status for the three quintiles of SIR2-responsive origins indicated in ‘E’ was determined as in ‘D’.

If depletion of H4K16ac on origin-adjacent nucleosomes was relevant to SIR2-dependent inhibition of MCM loading in cdc6-4 cells, then the origins most responsive to deletion of SIR2 might be expected to show the greatest depletion of H4K16ac. We defined origin SIR2-responsiveness as the ratio of the MCM ChIP-Seq signal in cdc6-4 sir2Δ cells to that in sir2Δ cells; the most SIR2-responsive origins generated ratios near 1.0, suggesting that deletion of SIR2 substantially rescued the MCM loading defect of cdc6-4 (Fig 3E). Comparison of H4K16ac status of nucleosomes surrounding the low, medium, and high-SIR2 responsive origin quintiles revealed that as a group the most SIR2-responsive origins exhibited the greatest depletion of H416ac (Fig 3F). Thus the varying degrees of SIR2-responsiveness to MCM association generally correlated well with the degree of H4K16 hypoacetylation, with the exception of the -1 nucleosome for the low and medium SIR2-responsive origins.

In a separate analysis, H4K16ac status of nucleosomes adjacent to euchromatic origins was also examined relative to the genome-wide average level of H4K16ac so that a comparison to origins within established SIR-heterochromatin domains could be made (S6 Fig). This analysis, presented as box-and-whiskers plots to show the variation in H4K16ac at the relevant nucleosomes in each group, also indicated that the most prominent depletion of H4K16ac occurred on the -1 and +1 nucleosome positions for euchromatic origins, and weakened considerably by the -3 and +3 nucleosomes, as also seen in Fig 3F. As expected, the -1 and +1 nucleosomes of euchromatic origins were less depleted for H4K16ac than the analogous nucleosomes for origins within SIR-heterochromatin, with the median values for the two types of origins differing ~3-fold. The differences between H4K16ac status were more striking between the two types of origins (i.e. euchromatic versus SIR-heterochromatic) at nucleosomes more distal from the origin, consistent with the H4K16ac status at euchromatic origins being more localized than the H4K16ac status within SIR-heterochromatin. Analyses of H4K16ac status at a few individual origins also indicated that the -1 and/or +1 nucleosomes were the most likely to show the greatest depletion of H4K16ac, particularly for the most SIR2-responsive origins. (S7 Fig).

The depletion of acetylated H4K16-containing nucleosomes around euchromatic origins required SIR2The data described above raised the possibility that Sir2 was deacetylating H4K16ac nucleosomes adjacent to euchromatic origins. To test this possibility, we performed H4K16ac MNase ChIP-Seq experiments on the exact same wild type (SIR2) and sir2Δ cells used for the MCM ChIP-Seq experiment described in Fig 1 (Fig 4A). Analyses of wild type cells recapitulated the published H4K16ac MNase ChIP-Seq results shown in Fig 3D. We note that the relative level of origin-specific H4K16ac depletion was slightly greater in these new experiments, possibly due to differences in strain backgrounds or growth conditions. Regardless, the key result was that, in contrast to wild type cells, depletion of H4K16ac was lost from origin-adjacent nucleosomes in the sir2Δ cells, while the behavior of non-origin ORC site control nucleosomes was unchanged. Importantly, these effects also correlated with SIR2 responsiveness (Fig 4B). Thus, origin-specific depletion of H4K16ac on nucleosomes adjacent to euchromatic origins required SIR2.10.1371/journal.pgen.1007418.g004Fig 4Depletion of H4K16ac from origin-adjacent nucleosomes required SIR2.A. H4K16 acetylation status of nucleosomes was assessed by MNase ChIP-Seq in SIR2 and sir2Δ cells. These cells were the same as those analyzed for MCM binding in Fig 1. B. The H4K16 acetylation status was plotted for the low, medium and high SIR2-responsive quintiles as defined in Fig 3.

The depletion of acetylated H4K16-containing nucleosomes around euchromatic origins required SIR2

The data described above raised the possibility that Sir2 was deacetylating H4K16ac nucleosomes adjacent to euchromatic origins. To test this possibility, we performed H4K16ac MNase ChIP-Seq experiments on the exact same wild type (SIR2) and sir2Δ cells used for the MCM ChIP-Seq experiment described in Fig 1 (Fig 4A). Analyses of wild type cells recapitulated the published H4K16ac MNase ChIP-Seq results shown in Fig 3D. We note that the relative level of origin-specific H4K16ac depletion was slightly greater in these new experiments, possibly due to differences in strain backgrounds or growth conditions. Regardless, the key result was that, in contrast to wild type cells, depletion of H4K16ac was lost from origin-adjacent nucleosomes in the sir2Δ cells, while the behavior of non-origin ORC site control nucleosomes was unchanged. Importantly, these effects also correlated with SIR2 responsiveness (Fig 4B). Thus, origin-specific depletion of H4K16ac on nucleosomes adjacent to euchromatic origins required SIR2.

Depletion of H4K16ac from origin-adjacent nucleosomes required SIR2.

A. H4K16 acetylation status of nucleosomes was assessed by MNase ChIP-Seq in SIR2 and sir2Δ cells. These cells were the same as those analyzed for MCM binding in Fig 1. B. The H4K16 acetylation status was plotted for the low, medium and high SIR2-responsive quintiles as defined in Fig 3.

Sir2 and Sir3 were detected at originsSir2 and Sir3 are physical components of yeast heterochromatin that have been detected at rDNA (Sir2), telomeres and HM loci (Sir2 and Sir3) by ChIP experiments in multiple studies (reviewed in [15]). However, neither protein has been reported to associate with euchromatic origins [44,45,47]. Depletion of H4K16ac at euchromatic origins was estimated to be between 25–35% of that detected at heterochromatic origins for the +1 nucleosome (S6 Fig). This observation is consistent with a proteomic analysis of nucleosome modifications on a plasmid based origin [48]. However, given that Sir2 is an enzyme, stable binding by Sir2, or even Sir3 if it is required only transiently, might not be required to establish or maintain this modification state. To test this possibility, we used more recently published high-resolution ChIP-Seq Sir2 and Sir3 data sets and excluded all origins from known heterochromatin origins, as described above for Figs 3 and 4 [34,44]. This analysis detected Sir2 and Sir3 ChIP-Seq signals on nucleosomes adjacent to origins but not on nucleosomes adjacent to non-origin controls (Fig 5A and 5B). Because the Sir3 data was generated from a high-resolution MNase ChIP-Seq experiment, we also examined these data at nucleotide resolution across the 1201 bp origin and non-origin fragments described in Fig 3B. To generate the baseline for these analyses, the number of reads that contained a given nucleotide in the ChIP sample was divided by the number of reads that contained that nucleotide in the starting material, for every single nucleotide within the euchromatic genome (i.e. nucleotides from known heterochromatin regions were excluded to form the “0” binding baseline for Sir3), following the method in [49] (Fig 5C). These analyses revealed the strongest Sir3 ChIP-Seq signal at the most proximal origin-adjacent nucleosomes (-1 and +1), but signals above baseline were also detectable at the -3, -2, +2 and +3 nucleosomes. As was true for the depletion of H4K16ac, the Sir3 signals correlated with the degree of SIR2-responsiveness as defined in Fig 3E (except in the opposite direction) (Fig 5D and 5E). Randomizing the origins into three new groups of equivalent numbers prior to plotting the Sir3 ChIP-Seq signals eliminated the correlation (Fig 5F).10.1371/journal.pgen.1007418.g005Fig 5Sir2 and Sir3 were physically associated with nucleosomes adjacent to origins.A. Sir2 or (B) Sir3 binding to origin or non-origin adjacent nucleosomes was assessed as in Fig 3 using data from [33,34]. C. The normalized Sir3 ChIP-Seq signal (log2 Sir3 ChIP signal, y-axis) for each nucleotide was plotted over the 1201 bp origin (blue) or non-origin (green) loci (coordinates on x-axis). The Sir3 ChIP-Seq signal represented total reads for each nucleotide normalized to the depth (total reads) and breadth (number of nucleotides with reads) of the sequencing reactions [49]. Genomic regions previously established as SIR-heterochromatic domains were excluded from the normalization but see S8 Fig for an analysis that includes these nucleotides in establishing the baseline. D. and E. The Sir3 ChIP-Signals were plotted as in ‘C’ or ‘B’, respectively, for the three different SIR2-responsive groups defined in Fig 3. F. The SIR2-responsive origins comprising the three groups assessed in ‘D’ were randomized into three different sets and the Sir3 ChIP-Seq signals determined and plotted as in ‘D’.A separate analyses of Sir3-3xHA ChIP-Seq data from the same study indicated that excluding nucleotides from SIR-heterochromatin when establishing the baseline was required for the Sir3 ChIP-Seq signals at euchromatic origins to rise above the “0” baseline [50] (S8 Fig), in contrast to what was observed for H4K16ac (S6 Fig). Nevertheless, in this independent analysis, Sir3-3xHA (detected with anti-HA) signals at euchromatic origins were clearly greater at the -1 and +1 nucleosomes of origins relative to the comparable nucleosomes at the non-origin ORC-site containing loci that have been used as controls for origin specificity throughout this study, and Sir3-3xHA signals also correlated with SIR2-responsiveness (S8B and S8C Fig).We interpret these results to mean that strong Sir3 association with heterochromatic regions reduced the “0” baseline and obscured comparatively weak Sir3 signals at euchromatic origins. Thus, we conclude that Sir3 associated with euchromatic origins but more transiently and at levels less than Sir3’s association with SIR-heterochromatin regions [50].In summary, several molecular hallmarks of SIR-heterochromatin–Sir2, Sir3 and Sir2-dependent H4K16 hypoacetylation–could be detected at euchromatin origins but not at euchromatic non-origin controls. In addition, the relative levels of each of these molecular hallmarks correlated with how strongly a SIR2 deletion restored an MCM ChIP signal to origins in cdc6-4 cells grown at temperatures that otherwise abolished MCM loading.

Sir2 and Sir3 were detected at origins

Sir2 and Sir3 are physical components of yeast heterochromatin that have been detected at rDNA (Sir2), telomeres and HM loci (Sir2 and Sir3) by ChIP experiments in multiple studies (reviewed in [15]). However, neither protein has been reported to associate with euchromatic origins [44,45,47]. Depletion of H4K16ac at euchromatic origins was estimated to be between 25–35% of that detected at heterochromatic origins for the +1 nucleosome (S6 Fig). This observation is consistent with a proteomic analysis of nucleosome modifications on a plasmid based origin [48]. However, given that Sir2 is an enzyme, stable binding by Sir2, or even Sir3 if it is required only transiently, might not be required to establish or maintain this modification state. To test this possibility, we used more recently published high-resolution ChIP-Seq Sir2 and Sir3 data sets and excluded all origins from known heterochromatin origins, as described above for Figs 3 and 4 [34,44]. This analysis detected Sir2 and Sir3 ChIP-Seq signals on nucleosomes adjacent to origins but not on nucleosomes adjacent to non-origin controls (Fig 5A and 5B). Because the Sir3 data was generated from a high-resolution MNase ChIP-Seq experiment, we also examined these data at nucleotide resolution across the 1201 bp origin and non-origin fragments described in Fig 3B. To generate the baseline for these analyses, the number of reads that contained a given nucleotide in the ChIP sample was divided by the number of reads that contained that nucleotide in the starting material, for every single nucleotide within the euchromatic genome (i.e. nucleotides from known heterochromatin regions were excluded to form the “0” binding baseline for Sir3), following the method in [49] (Fig 5C). These analyses revealed the strongest Sir3 ChIP-Seq signal at the most proximal origin-adjacent nucleosomes (-1 and +1), but signals above baseline were also detectable at the -3, -2, +2 and +3 nucleosomes. As was true for the depletion of H4K16ac, the Sir3 signals correlated with the degree of SIR2-responsiveness as defined in Fig 3E (except in the opposite direction) (Fig 5D and 5E). Randomizing the origins into three new groups of equivalent numbers prior to plotting the Sir3 ChIP-Seq signals eliminated the correlation (Fig 5F).

Sir2 and Sir3 were physically associated with nucleosomes adjacent to origins.

A. Sir2 or (B) Sir3 binding to origin or non-origin adjacent nucleosomes was assessed as in Fig 3 using data from [33,34]. C. The normalized Sir3 ChIP-Seq signal (log2 Sir3 ChIP signal, y-axis) for each nucleotide was plotted over the 1201 bp origin (blue) or non-origin (green) loci (coordinates on x-axis). The Sir3 ChIP-Seq signal represented total reads for each nucleotide normalized to the depth (total reads) and breadth (number of nucleotides with reads) of the sequencing reactions [49]. Genomic regions previously established as SIR-heterochromatic domains were excluded from the normalization but see S8 Fig for an analysis that includes these nucleotides in establishing the baseline. D. and E. The Sir3 ChIP-Signals were plotted as in ‘C’ or ‘B’, respectively, for the three different SIR2-responsive groups defined in Fig 3. F. The SIR2-responsive origins comprising the three groups assessed in ‘D’ were randomized into three different sets and the Sir3 ChIP-Seq signals determined and plotted as in ‘D’.

A separate analyses of Sir3-3xHA ChIP-Seq data from the same study indicated that excluding nucleotides from SIR-heterochromatin when establishing the baseline was required for the Sir3 ChIP-Seq signals at euchromatic origins to rise above the “0” baseline [50] (S8 Fig), in contrast to what was observed for H4K16ac (S6 Fig). Nevertheless, in this independent analysis, Sir3-3xHA (detected with anti-HA) signals at euchromatic origins were clearly greater at the -1 and +1 nucleosomes of origins relative to the comparable nucleosomes at the non-origin ORC-site containing loci that have been used as controls for origin specificity throughout this study, and Sir3-3xHA signals also correlated with SIR2-responsiveness (S8B and S8C Fig).We interpret these results to mean that strong Sir3 association with heterochromatic regions reduced the “0” baseline and obscured comparatively weak Sir3 signals at euchromatic origins. Thus, we conclude that Sir3 associated with euchromatic origins but more transiently and at levels less than Sir3’s association with SIR-heterochromatin regions [50].

In summary, several molecular hallmarks of SIR-heterochromatin–Sir2, Sir3 and Sir2-dependent H4K16 hypoacetylation–could be detected at euchromatin origins but not at euchromatic non-origin controls. In addition, the relative levels of each of these molecular hallmarks correlated with how strongly a SIR2 deletion restored an MCM ChIP signal to origins in cdc6-4 cells grown at temperatures that otherwise abolished MCM loading.

A cdc6-4 suppressor screen targeting core modifiable histone H3 and H4 residues identified a nucleosome surface important for Sir3 bindingHistones H3 and H4 form a tetramer that binds both to dsDNA and to two dimers of histones H2A and H2B to form the nucleosome. Many conserved residues on the histone H3 and H4 N-terminal tails as well as within the globular core region (“core-modifiable” residues) can be post-translationally modified, and, as discussed above, an H4K16Q substitution that should mimic the acetylated form of this residue (i.e. the form predicted to phenocopy the effect of a sir2Δ), suppressed the temperature-sensitive growth defect of cdc6-4 [23]. To test whether we could identify additional alleles with similar suppressive behavior, we assessed a library of 43 histone H3 and H4 mutations affecting 15 distinct residues for suppression of the cdc6-4 growth defect (Fig 6A). Mutant plasmids harboring a single copy of the HHT2-HHF2 locus marked with TRP1 were transformed into wild type and cdc6-4 cells lacking both chromosomal copies of histone H3 and H4 genes, maintained by a HHT2-HHF2 URA3 ARS CEN plasmid, and colonies were subsequently plated on 5-FOA containing media to select against the wild type HHT2-HHF2 plasmid. Recovered cells were then examined for growth on YPD medium at 25°C and higher temperatures. In agreement with previous studies, none of the histone mutants, with the exception of H4-S47E, caused growth defects in wild type cells [51], and several mutants were identified that inhibited growth of cells harboring the cdc6-4 allele even at the permissive growth temperature (25°C). However, relevant to this study, specific substitutions at only two residues, H3K79 (H3K79A and H3K79Q) and H4K79 (H4K79A and H4K79R) suppressed the temperature-sensitive growth defect of cdc6-4 cells (Fig 6B and S9 Fig). Notably, these residues are important for Sir3 binding to the nucleosome and for Sir3-mediated transcriptional silencing at the HM and telomeric loci [52–54]. H3K79 is a substrate for methylation by Dot1 and Sir3 binds preferentially to unmethylated H3K79 [55,56]. Based on these results, additional substitutions were engineered by site-directed mutagenesis at H3K79 and at four adjacent residues (H3K79E, H3E73A, H3E73K, H374A, H3T80A, H3D81A) (Fig 6B). In addition to revealing that H3K79E was a stronger suppressor of cdc6-4 than H3K79Q, analyses of these mutants also revealed that substitutions of H3E73 and H3T80 also suppressed the temperature-sensitive growth defect of cdc6-4 cells (Fig 6B and S9 Fig). H3E73 and H3T80 are important for HM and telomeric silencing [52], and H3T80 directly contacts the LRS domain of Sir3 [53,54]. In summary, suppression of cdc6-4 temperature-sensitivity was achieved by single substitutions within only four residues (H3E73, H3K79, H3T80, and H4K79) of H3/H4. Each of these residues clustered within a small patch on the nucleosome surface important for binding Sir3 (Fig 6C, highlighted in red) [53], strongly suggesting that disruption of Sir3 binding to nucleosomes suppressed the cdc6-4 growth defect. Together with the ChIP-Seq data, these genetic data supported the conclusion that both Sir2 and Sir3 alter the molecular features of nucleosomes adjacent to euchromatic origins, thus generating a local chromatin environment that can negatively impact the MCM loading reaction.10.1371/journal.pgen.1007418.g006Fig 6Histone H3 and H4 core modifiable alleles that suppress cdc6-4 temperature sensitivity define a known Sir3-nucleosome interface.A. List of histone H3 and H4 substitutions within the mutant library that were screened for their ability to affect the growth of cdc6-4 mutant cells (see also S9 Fig). B. 10-fold serial dilutions of M1345 (cdc6-4) with either wild type H3, H4 or selected histone variants that were from the original library (Fig 6A) or were generated by site-directed mutagenesis (see text), were spotted onto YPD and grown at the indicated temperatures. Asterisks (*) indicate the histone variants that suppressed cdc6-4 temperature-sensitive growth. H4K59A and H4K59R are shown as examples of histone mutants that did not suppress. C. Nucleosome structure (PDB-1ID3) with one H3(dark blue)-H4(light blue) dimer and the H3K79, H3T80, H4E74 and H4K79 residues highlighted in red.

A cdc6-4 suppressor screen targeting core modifiable histone H3 and H4 residues identified a nucleosome surface important for Sir3 binding

Histones H3 and H4 form a tetramer that binds both to dsDNA and to two dimers of histones H2A and H2B to form the nucleosome. Many conserved residues on the histone H3 and H4 N-terminal tails as well as within the globular core region (“core-modifiable” residues) can be post-translationally modified, and, as discussed above, an H4K16Q substitution that should mimic the acetylated form of this residue (i.e. the form predicted to phenocopy the effect of a sir2Δ), suppressed the temperature-sensitive growth defect of cdc6-4 [23]. To test whether we could identify additional alleles with similar suppressive behavior, we assessed a library of 43 histone H3 and H4 mutations affecting 15 distinct residues for suppression of the cdc6-4 growth defect (Fig 6A). Mutant plasmids harboring a single copy of the HHT2-HHF2 locus marked with TRP1 were transformed into wild type and cdc6-4 cells lacking both chromosomal copies of histone H3 and H4 genes, maintained by a HHT2-HHF2 URA3 ARS CEN plasmid, and colonies were subsequently plated on 5-FOA containing media to select against the wild type HHT2-HHF2 plasmid. Recovered cells were then examined for growth on YPD medium at 25°C and higher temperatures. In agreement with previous studies, none of the histone mutants, with the exception of H4-S47E, caused growth defects in wild type cells [51], and several mutants were identified that inhibited growth of cells harboring the cdc6-4 allele even at the permissive growth temperature (25°C). However, relevant to this study, specific substitutions at only two residues, H3K79 (H3K79A and H3K79Q) and H4K79 (H4K79A and H4K79R) suppressed the temperature-sensitive growth defect of cdc6-4 cells (Fig 6B and S9 Fig). Notably, these residues are important for Sir3 binding to the nucleosome and for Sir3-mediated transcriptional silencing at the HM and telomeric loci [52–54]. H3K79 is a substrate for methylation by Dot1 and Sir3 binds preferentially to unmethylated H3K79 [55,56]. Based on these results, additional substitutions were engineered by site-directed mutagenesis at H3K79 and at four adjacent residues (H3K79E, H3E73A, H3E73K, H374A, H3T80A, H3D81A) (Fig 6B). In addition to revealing that H3K79E was a stronger suppressor of cdc6-4 than H3K79Q, analyses of these mutants also revealed that substitutions of H3E73 and H3T80 also suppressed the temperature-sensitive growth defect of cdc6-4 cells (Fig 6B and S9 Fig). H3E73 and H3T80 are important for HM and telomeric silencing [52], and H3T80 directly contacts the LRS domain of Sir3 [53,54]. In summary, suppression of cdc6-4 temperature-sensitivity was achieved by single substitutions within only four residues (H3E73, H3K79, H3T80, and H4K79) of H3/H4. Each of these residues clustered within a small patch on the nucleosome surface important for binding Sir3 (Fig 6C, highlighted in red) [53], strongly suggesting that disruption of Sir3 binding to nucleosomes suppressed the cdc6-4 growth defect. Together with the ChIP-Seq data, these genetic data supported the conclusion that both Sir2 and Sir3 alter the molecular features of nucleosomes adjacent to euchromatic origins, thus generating a local chromatin environment that can negatively impact the MCM loading reaction.

Histone H3 and H4 core modifiable alleles that suppress cdc6-4 temperature sensitivity define a known Sir3-nucleosome interface.

A. List of histone H3 and H4 substitutions within the mutant library that were screened for their ability to affect the growth of cdc6-4 mutant cells (see also S9 Fig). B. 10-fold serial dilutions of M1345 (cdc6-4) with either wild type H3, H4 or selected histone variants that were from the original library (Fig 6A) or were generated by site-directed mutagenesis (see text), were spotted onto YPD and grown at the indicated temperatures. Asterisks (*) indicate the histone variants that suppressed cdc6-4 temperature-sensitive growth. H4K59A and H4K59R are shown as examples of histone mutants that did not suppress. C. Nucleosome structure (PDB-1ID3) with one H3(dark blue)-H4(light blue) dimer and the H3K79, H3T80, H4E74 and H4K79 residues highlighted in red.

DiscussionSir2 and Sir3 are non-essential proteins known best for their direct functions in heterochromatin-mediated transcriptional gene silencing in budding yeast (reviewed in [15]). This report described a new, pervasive and direct heterochromatin-independent role for these regulators at euchromatic DNA replication origins. The role was pervasive in that Sir2 exerted a negative effect on the MCM loading reaction at most origins in the genome. The evidence in support of this conclusion came from a comparison of MCM loading at origins by ChIP-Seq in cdc6-4 mutant cells and cdc6-4 sir2Δ cells. Cdc6 is essential for the MCM complex loading reaction, and the cdc6-4 allele weakens Cdc6 function such that, under non-permissive growth temperatures, detectable MCM association was abolished from all origins. However, under the same conditions, cdc6-4 sir2Δ cells exhibited MCM association at most euchromatic origins (~80%). Thus, the ability of sir2 inactivating mutations to robustly suppress the temperature-sensitive lethality of cdc6-4, as well as growth defects caused by other replication alleles that reduce or abolish MCM loading [29], is due to genome-scale enhancement of MCM loading at most euchromatic origins.The ability of sir2Δ to enhance MCM complex loading at most euchromatic origins in cdc6-4 cells was striking, but it did not indicate whether this effect was due to Sir2 acting directly within euchromatin in general or at euchromatic origins in particular. In particular, recent reports establish that SIR2 can alter origin function within euchromatin in yeast indirectly because of its function in rDNA heterochromatin formation that suppresses many of the rDNA repeat origins [22,25]. However, analyses of the rDNA locus presented here suggested that a similar rDNA-mediated mechanism could not explain why sir2Δ or sir3Δ suppressed cdc6-4. Instead, the data provided evidence that Sir2 functioned directly at euchromatic origins, and that it was this direct function of Sir2 that made yeast so vulnerable to defects in the MCM complex loading reaction caused by cdc6-4. First, relative hypoacetylation of H4K16 was observed for nucleosomes immediately adjacent to euchromatic origins but not for the analogous nucleosomes at non-origin ORC-site containing control loci. Thus, in an asynchronous population of wild type cells, nucleosomes flanking euchromatic origins were relatively hypoacteylated on H4K16. Of the twelve histone acetylation marks examined [32], H4K16ac was the only one that showed this origin-specific depletion. Second, origin-adjacent nucleosomal H4K16 hypoacetylation required SIR2; a sir2Δ mutant completely lost the nucleosomal H4K16 hypoacetylation pattern at euchromatic origins, without having any obvious effect on nucleosomes adjacent to the non-origin control loci. Third, analyses of previously published high-resolution ChIP-Seq experiments [34,50] provided evidence that Sir2 and Sir3 physically associated with nucleosomes adjacent to euchromatic origins but not non-origin control loci. Sir2 and Sir3 exhibited the same pattern but in the exact opposite direction of H4K16ac, which is what is observed for these proteins and this histone mark at SIR-heterochromatic domains. Lastly, each of these defining molecular features of classic SIR-heterochromatin correlated with SIR2-responsiveness–defined as the relative extent of MCM association rescue at origins in cdc6-4 sir2Δ cells. Because of these observations, and in particular the clear depletion of ChIP-Seq signals around origins generated by multiple histone antibodies, we think it is unlikely that these outcomes result from the non-specific “hyper-ChIPability” that affects certain highly transcribed regions [45,57]. Therefore, to the best of our knowledge, these data provide the first evidence that the heterochromatin proteins Sir2 and Sir3 act directly on the local chromatin environment of euchromatic origins.Despite the common molecular hallmarks shared between the Sir2-3-chromatin at euchromatic origins described in this study and classic SIR-mediated heterochromatin, these two types of chromatin domains are different. For example, unlike SIR-heterochromatin that functions in gene silencing at the HM loci, telomeres or rDNA, there is no evidence that origin-adjacent chromatin functions as a robust transcriptionally silenced domain [58], although we cannot rule out the possibility that Sir2,3-chromatin is having small effects on the transcription of annotated genes near origins or on the expression of non-coding RNAs in these regions. In addition, SIR-heterochromatin affects multiple nucleosomes, defining domains that encompass ~4–10 kb of contiguous chromosomal DNA (reviewed in [15]), whereas the Sir2-3-chromatin at origins encompassed <1 kb of chromosomal DNA adjacent to the origin and affected only four to six nucleosomes at most. While it remains to be determined whether the specific molecular interactions that recruit Sir2 and Sir3 to euchromatic origins are also used to recruit these proteins to heterochromatic regions, their functional and structural impact on nucleosomes at euchromatic origins was substantially attenuated relative to their impact within heterochromatin. Therefore, we suggest that the Sir2-3 chromatin at origins defines a role for these proteins that differs from their roles in canonical Sir-heterochromatin.A specific G1-phase role for H4K16 acetylation at originsAcetylation of histone H3 and H4 N-termini at origin proximal-nucleosomes generally enhances origin function [20,59,60]. However, it has been difficult to assign specific roles of this acetylation to individual histone H3 or H4 lysines. Indeed, in terms of origin activation, while acetylation of histone H3 and H4 tail lysines is clearly important for origin activation during S-phase, no single lysine acetylation event is sufficient [48]. In addition, several different combinations of multiple lysine to arginine substitutions on the histone H3 and H4 N-terminal tails can substantially reduce origin activation, suggesting that some threshold level of nucleosome acetylation, or the concomitant charge neutralization, is what is important for origin activation. In contrast, the initial identification of a sir2 mutation, and subsequently a H4K16Q substitution, as robust suppressors of cdc6-4 temperature-sensitivity and origin-specific MCM loading, indicated that H4K16 might be unique among histone tail lysines in being particularly relevant to origin licensing [23,29]. This study revealed that H4K16 was also unique among histone H3 and H4 tail lysines because H4K16 exhibited SIR2-dependent, origin-specific hypoacetylation on nucleosomes adjacent to euchromatic origins. Consistent with these observations, proteomic analysis of nucleosomes adjacent to a plasmid-origin reveals that H4K16 behaves uniquely among histone H3 and H4 tail acetylation marks [48]. Specifically, in G1-phase, nucleosomes adjacent to a plasmid-based origin were relatively hypoacetylated at H4K16 compared to bulk nucleosomes, whereas the other histone tail lysines analyzed (H3-K9, -K14, K-23; H4-K5, -K8, -K12) showed similar levels of hypoacetylation on bulk and plasmid-based origin adjacent nucleosomes. Because Sir2 and Sir3 also showed origin-specific association with nucleosomes, the distinctive role for H4K16 acetylation in control of the G1-phase MCM loading reaction can now be explained by formation of a Sir2,3 chromatin structure directly at many euchromatic origins, suggesting that origins, perhaps via ORC, can specifically recruit Sir2,3.MCM loading challenges in a native chromatin contextThe data presented here provided evidence that the MCM loading reaction has evolved to work within a naturally repressive chromatin environment established by Sir2,3. Indeed, an otherwise severely crippled licensing reaction caused by the cdc6-4 defect can quite effectively license most origins if the native repressive chromatin environment is abolished by inactivation of Sir2. While the precise mechanistic step(s) of the MCM complex loading reaction that are affected by the Sir2,3-repressive chromatin environment remain to be examined, recent successes in reconstituting DNA replication in chromatin contexts in vitro offer an ideal pathway forward [61–63]. For example, while reconstituted chromatin does not block MCM complex loading in vitro, it would be interesting to learn whether the addition of Sir3 to these biochemical reactions is sufficient for inhibition and, if so, requires Sir3’s ability to bind to nucleosomes [53].However, in terms of yeast physiology, the major challenge moving forward is to understand why such a chromatin-structure capable of inhibiting MCM complex loading may have evolved to exist at so many euchromatic origins in the first place. While a sir2Δ or a sir3Δ have no obvious genome-scale effect on yeast cell division, MCM complex loading or origin activation in cells with wild type MCM complex loading reaction components, the Sir2,3-chromatin structure discussed here does indeed exist in wild type cells, and, based on the strong genetic suppression of cdc6-4 and other alleles that weaken MCM complex loading [29], is clearly capable of exerting a substantial inhibitory effect on this reaction. Indeed, the results presented here suggest that the levels and or activities of the MCM complex loading proteins have evolved to contend with a particularly inhibitory local chromatin structure around euchromatic origins; a temperature-sensitive allele that weakened MCM complex loading, such as cdc6-4, would never have been isolated in yeast cells lacking Sir2 or Sir3 as the mutant Cdc6-4 protein provides for robust function in such contexts. It is possible that the Sir2,3- chromatin at euchromatic origins is simply a byproduct of yeast cells evolving transcriptionally silenced heterochromatic domains, and, as a result of strong evolutionary selection for these regions’ existence, combined with fundamental protein dynamics, Sir2 and Sir3 simply fall off of heterochromatin at some frequency and, when they do, end up concentrating at origins as they stochastically bounce around the nucleus [64–67]. Sir2 and Sir3 may concentrate at origins over other euchromatic elements because of the intrinsic ability of origins to act as silencers, as suggested by some studies [68,69]. Regardless, the outcome is that the MCM complex loading reaction had to evolve more robustly than it would have otherwise. Additionally, and/or alternatively, the Sir2,3-chromatin at euchromatic origins may play an important role in normal yeast physiology, offering a fine-tuning of the MCM complex loading reaction at many individual origins that, while at the first-level of cell growth and origin function in bulk laboratory assays may be hard to measure, nevertheless has important repercussions at the population and/or evolutionary scale for yeast and other eukaryotic organisms. Interestingly, over two decades ago a study reported how over-expression of Sir2 and Sir3 promoted severe chromosome instability and yeast cell lethality [70]. Perhaps these effects were related, at least in part, to Sir2,3 effects on origins. Finally, a role for Sir2 in origin control may indeed be conserved as human SIRT1, the ortholog of yeast Sir2, suppresses the function of dormant origins under conditions of replication stress [71].

Sir2 and Sir3 are non-essential proteins known best for their direct functions in heterochromatin-mediated transcriptional gene silencing in budding yeast (reviewed in [15]). This report described a new, pervasive and direct heterochromatin-independent role for these regulators at euchromatic DNA replication origins. The role was pervasive in that Sir2 exerted a negative effect on the MCM loading reaction at most origins in the genome. The evidence in support of this conclusion came from a comparison of MCM loading at origins by ChIP-Seq in cdc6-4 mutant cells and cdc6-4 sir2Δ cells. Cdc6 is essential for the MCM complex loading reaction, and the cdc6-4 allele weakens Cdc6 function such that, under non-permissive growth temperatures, detectable MCM association was abolished from all origins. However, under the same conditions, cdc6-4 sir2Δ cells exhibited MCM association at most euchromatic origins (~80%). Thus, the ability of sir2 inactivating mutations to robustly suppress the temperature-sensitive lethality of cdc6-4, as well as growth defects caused by other replication alleles that reduce or abolish MCM loading [29], is due to genome-scale enhancement of MCM loading at most euchromatic origins.

The ability of sir2Δ to enhance MCM complex loading at most euchromatic origins in cdc6-4 cells was striking, but it did not indicate whether this effect was due to Sir2 acting directly within euchromatin in general or at euchromatic origins in particular. In particular, recent reports establish that SIR2 can alter origin function within euchromatin in yeast indirectly because of its function in rDNA heterochromatin formation that suppresses many of the rDNA repeat origins [22,25]. However, analyses of the rDNA locus presented here suggested that a similar rDNA-mediated mechanism could not explain why sir2Δ or sir3Δ suppressed cdc6-4. Instead, the data provided evidence that Sir2 functioned directly at euchromatic origins, and that it was this direct function of Sir2 that made yeast so vulnerable to defects in the MCM complex loading reaction caused by cdc6-4. First, relative hypoacetylation of H4K16 was observed for nucleosomes immediately adjacent to euchromatic origins but not for the analogous nucleosomes at non-origin ORC-site containing control loci. Thus, in an asynchronous population of wild type cells, nucleosomes flanking euchromatic origins were relatively hypoacteylated on H4K16. Of the twelve histone acetylation marks examined [32], H4K16ac was the only one that showed this origin-specific depletion. Second, origin-adjacent nucleosomal H4K16 hypoacetylation required SIR2; a sir2Δ mutant completely lost the nucleosomal H4K16 hypoacetylation pattern at euchromatic origins, without having any obvious effect on nucleosomes adjacent to the non-origin control loci. Third, analyses of previously published high-resolution ChIP-Seq experiments [34,50] provided evidence that Sir2 and Sir3 physically associated with nucleosomes adjacent to euchromatic origins but not non-origin control loci. Sir2 and Sir3 exhibited the same pattern but in the exact opposite direction of H4K16ac, which is what is observed for these proteins and this histone mark at SIR-heterochromatic domains. Lastly, each of these defining molecular features of classic SIR-heterochromatin correlated with SIR2-responsiveness–defined as the relative extent of MCM association rescue at origins in cdc6-4 sir2Δ cells. Because of these observations, and in particular the clear depletion of ChIP-Seq signals around origins generated by multiple histone antibodies, we think it is unlikely that these outcomes result from the non-specific “hyper-ChIPability” that affects certain highly transcribed regions [45,57]. Therefore, to the best of our knowledge, these data provide the first evidence that the heterochromatin proteins Sir2 and Sir3 act directly on the local chromatin environment of euchromatic origins.

Despite the common molecular hallmarks shared between the Sir2-3-chromatin at euchromatic origins described in this study and classic SIR-mediated heterochromatin, these two types of chromatin domains are different. For example, unlike SIR-heterochromatin that functions in gene silencing at the HM loci, telomeres or rDNA, there is no evidence that origin-adjacent chromatin functions as a robust transcriptionally silenced domain [58], although we cannot rule out the possibility that Sir2,3-chromatin is having small effects on the transcription of annotated genes near origins or on the expression of non-coding RNAs in these regions. In addition, SIR-heterochromatin affects multiple nucleosomes, defining domains that encompass ~4–10 kb of contiguous chromosomal DNA (reviewed in [15]), whereas the Sir2-3-chromatin at origins encompassed <1 kb of chromosomal DNA adjacent to the origin and affected only four to six nucleosomes at most. While it remains to be determined whether the specific molecular interactions that recruit Sir2 and Sir3 to euchromatic origins are also used to recruit these proteins to heterochromatic regions, their functional and structural impact on nucleosomes at euchromatic origins was substantially attenuated relative to their impact within heterochromatin. Therefore, we suggest that the Sir2-3 chromatin at origins defines a role for these proteins that differs from their roles in canonical Sir-heterochromatin.

A specific G1-phase role for H4K16 acetylation at originsAcetylation of histone H3 and H4 N-termini at origin proximal-nucleosomes generally enhances origin function [20,59,60]. However, it has been difficult to assign specific roles of this acetylation to individual histone H3 or H4 lysines. Indeed, in terms of origin activation, while acetylation of histone H3 and H4 tail lysines is clearly important for origin activation during S-phase, no single lysine acetylation event is sufficient [48]. In addition, several different combinations of multiple lysine to arginine substitutions on the histone H3 and H4 N-terminal tails can substantially reduce origin activation, suggesting that some threshold level of nucleosome acetylation, or the concomitant charge neutralization, is what is important for origin activation. In contrast, the initial identification of a sir2 mutation, and subsequently a H4K16Q substitution, as robust suppressors of cdc6-4 temperature-sensitivity and origin-specific MCM loading, indicated that H4K16 might be unique among histone tail lysines in being particularly relevant to origin licensing [23,29]. This study revealed that H4K16 was also unique among histone H3 and H4 tail lysines because H4K16 exhibited SIR2-dependent, origin-specific hypoacetylation on nucleosomes adjacent to euchromatic origins. Consistent with these observations, proteomic analysis of nucleosomes adjacent to a plasmid-origin reveals that H4K16 behaves uniquely among histone H3 and H4 tail acetylation marks [48]. Specifically, in G1-phase, nucleosomes adjacent to a plasmid-based origin were relatively hypoacetylated at H4K16 compared to bulk nucleosomes, whereas the other histone tail lysines analyzed (H3-K9, -K14, K-23; H4-K5, -K8, -K12) showed similar levels of hypoacetylation on bulk and plasmid-based origin adjacent nucleosomes. Because Sir2 and Sir3 also showed origin-specific association with nucleosomes, the distinctive role for H4K16 acetylation in control of the G1-phase MCM loading reaction can now be explained by formation of a Sir2,3 chromatin structure directly at many euchromatic origins, suggesting that origins, perhaps via ORC, can specifically recruit Sir2,3.

A specific G1-phase role for H4K16 acetylation at origins

Acetylation of histone H3 and H4 N-termini at origin proximal-nucleosomes generally enhances origin function [20,59,60]. However, it has been difficult to assign specific roles of this acetylation to individual histone H3 or H4 lysines. Indeed, in terms of origin activation, while acetylation of histone H3 and H4 tail lysines is clearly important for origin activation during S-phase, no single lysine acetylation event is sufficient [48]. In addition, several different combinations of multiple lysine to arginine substitutions on the histone H3 and H4 N-terminal tails can substantially reduce origin activation, suggesting that some threshold level of nucleosome acetylation, or the concomitant charge neutralization, is what is important for origin activation. In contrast, the initial identification of a sir2 mutation, and subsequently a H4K16Q substitution, as robust suppressors of cdc6-4 temperature-sensitivity and origin-specific MCM loading, indicated that H4K16 might be unique among histone tail lysines in being particularly relevant to origin licensing [23,29]. This study revealed that H4K16 was also unique among histone H3 and H4 tail lysines because H4K16 exhibited SIR2-dependent, origin-specific hypoacetylation on nucleosomes adjacent to euchromatic origins. Consistent with these observations, proteomic analysis of nucleosomes adjacent to a plasmid-origin reveals that H4K16 behaves uniquely among histone H3 and H4 tail acetylation marks [48]. Specifically, in G1-phase, nucleosomes adjacent to a plasmid-based origin were relatively hypoacetylated at H4K16 compared to bulk nucleosomes, whereas the other histone tail lysines analyzed (H3-K9, -K14, K-23; H4-K5, -K8, -K12) showed similar levels of hypoacetylation on bulk and plasmid-based origin adjacent nucleosomes. Because Sir2 and Sir3 also showed origin-specific association with nucleosomes, the distinctive role for H4K16 acetylation in control of the G1-phase MCM loading reaction can now be explained by formation of a Sir2,3 chromatin structure directly at many euchromatic origins, suggesting that origins, perhaps via ORC, can specifically recruit Sir2,3.

MCM loading challenges in a native chromatin contextThe data presented here provided evidence that the MCM loading reaction has evolved to work within a naturally repressive chromatin environment established by Sir2,3. Indeed, an otherwise severely crippled licensing reaction caused by the cdc6-4 defect can quite effectively license most origins if the native repressive chromatin environment is abolished by inactivation of Sir2. While the precise mechanistic step(s) of the MCM complex loading reaction that are affected by the Sir2,3-repressive chromatin environment remain to be examined, recent successes in reconstituting DNA replication in chromatin contexts in vitro offer an ideal pathway forward [61–63]. For example, while reconstituted chromatin does not block MCM complex loading in vitro, it would be interesting to learn whether the addition of Sir3 to these biochemical reactions is sufficient for inhibition and, if so, requires Sir3’s ability to bind to nucleosomes [53].However, in terms of yeast physiology, the major challenge moving forward is to understand why such a chromatin-structure capable of inhibiting MCM complex loading may have evolved to exist at so many euchromatic origins in the first place. While a sir2Δ or a sir3Δ have no obvious genome-scale effect on yeast cell division, MCM complex loading or origin activation in cells with wild type MCM complex loading reaction components, the Sir2,3-chromatin structure discussed here does indeed exist in wild type cells, and, based on the strong genetic suppression of cdc6-4 and other alleles that weaken MCM complex loading [29], is clearly capable of exerting a substantial inhibitory effect on this reaction. Indeed, the results presented here suggest that the levels and or activities of the MCM complex loading proteins have evolved to contend with a particularly inhibitory local chromatin structure around euchromatic origins; a temperature-sensitive allele that weakened MCM complex loading, such as cdc6-4, would never have been isolated in yeast cells lacking Sir2 or Sir3 as the mutant Cdc6-4 protein provides for robust function in such contexts. It is possible that the Sir2,3- chromatin at euchromatic origins is simply a byproduct of yeast cells evolving transcriptionally silenced heterochromatic domains, and, as a result of strong evolutionary selection for these regions’ existence, combined with fundamental protein dynamics, Sir2 and Sir3 simply fall off of heterochromatin at some frequency and, when they do, end up concentrating at origins as they stochastically bounce around the nucleus [64–67]. Sir2 and Sir3 may concentrate at origins over other euchromatic elements because of the intrinsic ability of origins to act as silencers, as suggested by some studies [68,69]. Regardless, the outcome is that the MCM complex loading reaction had to evolve more robustly than it would have otherwise. Additionally, and/or alternatively, the Sir2,3-chromatin at euchromatic origins may play an important role in normal yeast physiology, offering a fine-tuning of the MCM complex loading reaction at many individual origins that, while at the first-level of cell growth and origin function in bulk laboratory assays may be hard to measure, nevertheless has important repercussions at the population and/or evolutionary scale for yeast and other eukaryotic organisms. Interestingly, over two decades ago a study reported how over-expression of Sir2 and Sir3 promoted severe chromosome instability and yeast cell lethality [70]. Perhaps these effects were related, at least in part, to Sir2,3 effects on origins. Finally, a role for Sir2 in origin control may indeed be conserved as human SIRT1, the ortholog of yeast Sir2, suppresses the function of dormant origins under conditions of replication stress [71].

MCM loading challenges in a native chromatin context

The data presented here provided evidence that the MCM loading reaction has evolved to work within a naturally repressive chromatin environment established by Sir2,3. Indeed, an otherwise severely crippled licensing reaction caused by the cdc6-4 defect can quite effectively license most origins if the native repressive chromatin environment is abolished by inactivation of Sir2. While the precise mechanistic step(s) of the MCM complex loading reaction that are affected by the Sir2,3-repressive chromatin environment remain to be examined, recent successes in reconstituting DNA replication in chromatin contexts in vitro offer an ideal pathway forward [61–63]. For example, while reconstituted chromatin does not block MCM complex loading in vitro, it would be interesting to learn whether the addition of Sir3 to these biochemical reactions is sufficient for inhibition and, if so, requires Sir3’s ability to bind to nucleosomes [53].

However, in terms of yeast physiology, the major challenge moving forward is to understand why such a chromatin-structure capable of inhibiting MCM complex loading may have evolved to exist at so many euchromatic origins in the first place. While a sir2Δ or a sir3Δ have no obvious genome-scale effect on yeast cell division, MCM complex loading or origin activation in cells with wild type MCM complex loading reaction components, the Sir2,3-chromatin structure discussed here does indeed exist in wild type cells, and, based on the strong genetic suppression of cdc6-4 and other alleles that weaken MCM complex loading [29], is clearly capable of exerting a substantial inhibitory effect on this reaction. Indeed, the results presented here suggest that the levels and or activities of the MCM complex loading proteins have evolved to contend with a particularly inhibitory local chromatin structure around euchromatic origins; a temperature-sensitive allele that weakened MCM complex loading, such as cdc6-4, would never have been isolated in yeast cells lacking Sir2 or Sir3 as the mutant Cdc6-4 protein provides for robust function in such contexts. It is possible that the Sir2,3- chromatin at euchromatic origins is simply a byproduct of yeast cells evolving transcriptionally silenced heterochromatic domains, and, as a result of strong evolutionary selection for these regions’ existence, combined with fundamental protein dynamics, Sir2 and Sir3 simply fall off of heterochromatin at some frequency and, when they do, end up concentrating at origins as they stochastically bounce around the nucleus [64–67]. Sir2 and Sir3 may concentrate at origins over other euchromatic elements because of the intrinsic ability of origins to act as silencers, as suggested by some studies [68,69]. Regardless, the outcome is that the MCM complex loading reaction had to evolve more robustly than it would have otherwise. Additionally, and/or alternatively, the Sir2,3-chromatin at euchromatic origins may play an important role in normal yeast physiology, offering a fine-tuning of the MCM complex loading reaction at many individual origins that, while at the first-level of cell growth and origin function in bulk laboratory assays may be hard to measure, nevertheless has important repercussions at the population and/or evolutionary scale for yeast and other eukaryotic organisms. Interestingly, over two decades ago a study reported how over-expression of Sir2 and Sir3 promoted severe chromosome instability and yeast cell lethality [70]. Perhaps these effects were related, at least in part, to Sir2,3 effects on origins. Finally, a role for Sir2 in origin control may indeed be conserved as human SIRT1, the ortholog of yeast Sir2, suppresses the function of dormant origins under conditions of replication stress [71].

Materials and methodsYeast strainsThe strains used for plasmid stability measurements and Mcm2 ChIP-Seq were the W303-1A derivatives: M138 (MATa), M386 (MATa
cdc6-4::LEU2), M922 (MATa
cdc6-4::LEU2 sir2Δ::TRP1) (described in Pappas et al. 2004), M1014 (MATa
cdc6-4) and M2126 (MATa
sir2Δ::TRP1). The M1321 (WT) and M1345 (cdc6-4) histone shuffle strains were described previously (Crampton et al., 2008). The strains used for genetic suppression and/or rDNA copy number experiments (Fig 2) include, in addition to M138, M386, M922 and M2126 described above, CFY43 (MATa
FOB1 CDC6 SIR2 sir3Δ::TRP1), CFY4584 (MATa
fob1 Δ::HIS3 CDC6 SIR2 SIR3 rDNA-35), CFY4585 (MATa
fob1 Δ::HIS3 CDC6 SIR2 SIR3 rDNA-180), CFY4603 (MATa
fob1 Δ::HIS3 cdc6-4 SIR2 SIR3 rDNA-35), CFY4604 (MATa
fob1 Δ::HIS3 cdc6-4 SIR2 SIR3 rDNA-180), CFY4613 (MATa
FOB1 cdc6-4 SIR2 sir3Δ::TRP1). CFY4583 and CFY4585 were provided by Jonathan Houseley (Babraham Institute, Cambridge, UK) [40].ChIP-Seq experimentsFor the MCM ChIP-Seq experiments, yeast cells were grown in liquid YPD at 25°C from single colonies until they reached an A600 of 0.2, at which point nocodazole was added to a final concentration of 15ug/ml and the cultures incubated for 2.5 hours, to obtain a uniform G2/M-phase arrest (S1 Fig). Cultures were shifted to 37°C for 30 minutes, the non-permissive growth temperature for cdc6-4, and then released from the nocodazole arrest at 37°C. When the majority of cells had entered G1-phase (55-minutes post-release for CDC6 SIR2, sir2Δ and cdc6-4 sir2Δ and 110 minutes for cdc6-4), OD equivalents of each cell line were cross-linked with formaldehyde for 15 minutes and then harvested, and chromatin was prepared for IP, by sonication for ChIP-Seq. Mcm2 ChIP was performed using a monoclonal antibody raised against yeast Mcm2 (gift of Bruce Stillman, Cold Spring Harbor Laboratory). Three independent biological replicates and two technical replicates for each biological replicate were performed for each strain. The ChIP DNA was prepared for deep sequencing by the UW-Madison Biotechnology Center using http://www.biooscientific.com/Portals/0/Manuals/NGS/5143-01-NEXTflex-ChIP-Seq-Kit.pdf. Sequencing was done on the HiSeq2000 1x100. Quality and quantity of the finished libraries were assessed using an Agilent DNA1000 chip and Qubit dsDNA HS Assay Kit, respectively. Libraries were standardized to 2nM. Cluster generation was performed using standard Cluster Kits and the Illumina Cluster Station. Single-end, 100bp sequencing was performed, using standard SBS chemistry on an Illumina HiSeq2000 sequencer. Images were analyzed using the standard Illumina Pipeline, version 1.8.2. Sequencing data for this project are available at the NCBI BioProject ID PRJNA428768.For the downstream analyses (Fig 1 and S2 Fig), all data for a given strain were combined as each replicate generated virtually identical reads. However, for the cdc6-4 strain, which produced extremely low signal-to-noise data, only two technical replicates from a single biological sample were used. For downstream analyses using MochiView the CG1 format data was collated into 25 bp bins (Fig 1). The combined data from each strain was run through CisGenome to identify the top 1000 peaks, divided into 100 peak bins and the percent of peaks that overlapped with at least one ARS (ARS coordinates used from oriDB) determined. The oriDB lists 410 yeast origins as confirmed (http://cerevisiae.oridb.org/). Based on these data and the above analyses we chose to examine and compare the top 400 peaks from each sample for our analyses (S2 Fig). The raw data for CDC6 SIR2, sir2Δ and cdc6-4 sir2Δ were normalized and scaled to the same cdc6-4 data for to generate the final MCM signal values for downstream analyses.The H4K16-acetylation MNase-ChIP DNA was generated from yeast strains M138 (CDC6 SIR2 strain used in Fig 1) and M2126 (CDC6 sir2Δ strain used in Fig 1) following the protocol described in [32], except a Bio101 Thermo FastPrep FP120 was used to break cells after crosslinking. Purified DNA was submitted to the University of Wisconsin-Madison Biotechnology Center. DNA concentration and sizing were verified using the Qubit dsDNA HS Assay Kit (Invitrogen, Carlsbad, California, USA) and Agilent DNA HS chip (Agilent Technologies, Inc., Santa Clara, CA, USA), respectively. Samples were prepared according the TruSeq ChIP Sample Preparation kit (Illumina Inc., San Diego, California, USA) with minor modifications. Libraries were size selected for an average insert size of 350 bp using SPRI-based bead selection. Quality and quantity of the finished libraries were assessed using an Agilent DNA1000 chip and Qubit dsDNA HS Assay Kit, respectively. Libraries were standardized to 2nM. Paired end 300bp sequencing was performed, using SBS chemistry (v3) on an Illumina MiSeq sequencer. Images were analyzed using the standard Illumina Pipeline, version 1.8.2. Sequencing data for this project are available at the NCBI BioProject ID PRJNA428768. For the data shown in Fig 4, downstream analyses were performed as described (Weiner et al., 2015). For the data presented as histograms relative to a baseline value derived from 239 euchromatic intergenic regions lacking either origins or ORC-site matches (Figs 3D, 3F, 4A, 4B, 5A, 5B, 5E and S5 and S7 Figs), the average signal for six contiguous nucleosomes in these regions was used. Thus each nucleosome assessed was normalized to the same value, and the log2 value of this ratio was plotted.rDNA copy number determinationQuantitative PCR reactions were carried out in sealed 200 ul microplates in a BioRad C1000 Thermocycler, CFX96 Real-Time System. The conditions were: 95°C-3’|95°C -30”, 58°C -15”|x40 cycles Standard Melt Curve Analysis. Each reaction contained: 0.04 ng/uL genomic DNA template, 0.016 nM of each primer, and 1X qPCR master mix (0.1 mM dNTPs, 1.25% formamide, 0.1 mg/mL BSA, 0.5 U Taq DNA polymerase, 5 mM Tris-pH 8.0, 10 mM KCl, 1.5 mM MgCl2, 0.75% Triton and 0.5X SYBR Green DNA stain.) DNA concentration and primer efficiency validation were performed by testing primer efficiency by titrating template concentrations. Serial dilutions of DNA were used to test a range of DNA concentrations from 0.1 ng/uL to 0.0016 ng/uL in technical triplicates. Efficiency values for RIM15, NTS2 and NL primer sets were calculated as 83%, 83%, and 81%, respectively, by the BioRad CFX96 software. The entire range of DNA concentrations was found to be within the linear range of instrument response. Melt curve analysis revealed a single product from each primer pair that was confirmed in each experimental run. The primer pairs were: For rDNA: NTS2: GGGCGATAATGACGGGAAGA-Fwd and TGTCCACTTTCAACCGTCCC-Rev; 35S (D1/D2 loop of 26S rDNA): GCATATCAATAAGCGGAGGAAAAG-Fwd and ACTTTACAAAGAACCGCACTCC-Rev; For single copy control: RIM15: GCCAGAACATTGGGTCAGAT-Fwd and CCGGATACTCGGATGTGTCT-Rev. For the single copy experimental (aqua bars in Fig 2B) ERV46: CACAGCTAGGACACCACCAA-Fwd and AGGGACAAGGATCATCCAAA-Rev. The enrichment values are equal to 2- ΔCt where ΔCt = Ct (rDNA locus (or ERV46))—Ct (single copy locus, RIM15) [72]. This value should equal the copy number of the target locus being assessed.Histone H3 and H4 suppressor screenStrains M1321 (MATa
ade2 ura3 leu2-3,112 trp1 hht1,hhf1::LEU2 hht2,hhf2::HIS3/pDP378 (HHT2,HHF2 CEN6 ARSH4 URA3) and M1345 (M1321, cdc6::ura3 LEU2::cdc6-4) were transformed with the TRP1 histone H3, H4 plasmids pH3H4-WT and mutant derivatives as shown in Fig 6A at 25°C. Multiple transformants were streak purified on FOA medium, recovered to YPD at 25°C and then screened for growth at various temperatures on YPD medium. Histone mutations were also constructed in pDP373 (pRS414 (TRP1) containing HHT2-HHF2 on a Spe1 fragment) using QuikChange. The entire H3 and H4 genes were sequenced for varification, and subsequently transformed in M1321 and M1345 for the yeast growth experiments in Fig 6.

Materials and methods

Yeast strainsThe strains used for plasmid stability measurements and Mcm2 ChIP-Seq were the W303-1A derivatives: M138 (MATa), M386 (MATa
cdc6-4::LEU2), M922 (MATa
cdc6-4::LEU2 sir2Δ::TRP1) (described in Pappas et al. 2004), M1014 (MATa
cdc6-4) and M2126 (MATa
sir2Δ::TRP1). The M1321 (WT) and M1345 (cdc6-4) histone shuffle strains were described previously (Crampton et al., 2008). The strains used for genetic suppression and/or rDNA copy number experiments (Fig 2) include, in addition to M138, M386, M922 and M2126 described above, CFY43 (MATa
FOB1 CDC6 SIR2 sir3Δ::TRP1), CFY4584 (MATa
fob1 Δ::HIS3 CDC6 SIR2 SIR3 rDNA-35), CFY4585 (MATa
fob1 Δ::HIS3 CDC6 SIR2 SIR3 rDNA-180), CFY4603 (MATa
fob1 Δ::HIS3 cdc6-4 SIR2 SIR3 rDNA-35), CFY4604 (MATa
fob1 Δ::HIS3 cdc6-4 SIR2 SIR3 rDNA-180), CFY4613 (MATa
FOB1 cdc6-4 SIR2 sir3Δ::TRP1). CFY4583 and CFY4585 were provided by Jonathan Houseley (Babraham Institute, Cambridge, UK) [40].

Yeast strains

The strains used for plasmid stability measurements and Mcm2 ChIP-Seq were the W303-1A derivatives: M138 (MATa), M386 (MATa
cdc6-4::LEU2), M922 (MATa
cdc6-4::LEU2 sir2Δ::TRP1) (described in Pappas et al. 2004), M1014 (MATa
cdc6-4) and M2126 (MATa
sir2Δ::TRP1). The M1321 (WT) and M1345 (cdc6-4) histone shuffle strains were described previously (Crampton et al., 2008). The strains used for genetic suppression and/or rDNA copy number experiments (Fig 2) include, in addition to M138, M386, M922 and M2126 described above, CFY43 (MATa
FOB1 CDC6 SIR2 sir3Δ::TRP1), CFY4584 (MATa
fob1 Δ::HIS3 CDC6 SIR2 SIR3 rDNA-35), CFY4585 (MATa
fob1 Δ::HIS3 CDC6 SIR2 SIR3 rDNA-180), CFY4603 (MATa
fob1 Δ::HIS3 cdc6-4 SIR2 SIR3 rDNA-35), CFY4604 (MATa
fob1 Δ::HIS3 cdc6-4 SIR2 SIR3 rDNA-180), CFY4613 (MATa
FOB1 cdc6-4 SIR2 sir3Δ::TRP1). CFY4583 and CFY4585 were provided by Jonathan Houseley (Babraham Institute, Cambridge, UK) [40].

ChIP-Seq experimentsFor the MCM ChIP-Seq experiments, yeast cells were grown in liquid YPD at 25°C from single colonies until they reached an A600 of 0.2, at which point nocodazole was added to a final concentration of 15ug/ml and the cultures incubated for 2.5 hours, to obtain a uniform G2/M-phase arrest (S1 Fig). Cultures were shifted to 37°C for 30 minutes, the non-permissive growth temperature for cdc6-4, and then released from the nocodazole arrest at 37°C. When the majority of cells had entered G1-phase (55-minutes post-release for CDC6 SIR2, sir2Δ and cdc6-4 sir2Δ and 110 minutes for cdc6-4), OD equivalents of each cell line were cross-linked with formaldehyde for 15 minutes and then harvested, and chromatin was prepared for IP, by sonication for ChIP-Seq. Mcm2 ChIP was performed using a monoclonal antibody raised against yeast Mcm2 (gift of Bruce Stillman, Cold Spring Harbor Laboratory). Three independent biological replicates and two technical replicates for each biological replicate were performed for each strain. The ChIP DNA was prepared for deep sequencing by the UW-Madison Biotechnology Center using http://www.biooscientific.com/Portals/0/Manuals/NGS/5143-01-NEXTflex-ChIP-Seq-Kit.pdf. Sequencing was done on the HiSeq2000 1x100. Quality and quantity of the finished libraries were assessed using an Agilent DNA1000 chip and Qubit dsDNA HS Assay Kit, respectively. Libraries were standardized to 2nM. Cluster generation was performed using standard Cluster Kits and the Illumina Cluster Station. Single-end, 100bp sequencing was performed, using standard SBS chemistry on an Illumina HiSeq2000 sequencer. Images were analyzed using the standard Illumina Pipeline, version 1.8.2. Sequencing data for this project are available at the NCBI BioProject ID PRJNA428768.For the downstream analyses (Fig 1 and S2 Fig), all data for a given strain were combined as each replicate generated virtually identical reads. However, for the cdc6-4 strain, which produced extremely low signal-to-noise data, only two technical replicates from a single biological sample were used. For downstream analyses using MochiView the CG1 format data was collated into 25 bp bins (Fig 1). The combined data from each strain was run through CisGenome to identify the top 1000 peaks, divided into 100 peak bins and the percent of peaks that overlapped with at least one ARS (ARS coordinates used from oriDB) determined. The oriDB lists 410 yeast origins as confirmed (http://cerevisiae.oridb.org/). Based on these data and the above analyses we chose to examine and compare the top 400 peaks from each sample for our analyses (S2 Fig). The raw data for CDC6 SIR2, sir2Δ and cdc6-4 sir2Δ were normalized and scaled to the same cdc6-4 data for to generate the final MCM signal values for downstream analyses.The H4K16-acetylation MNase-ChIP DNA was generated from yeast strains M138 (CDC6 SIR2 strain used in Fig 1) and M2126 (CDC6 sir2Δ strain used in Fig 1) following the protocol described in [32], except a Bio101 Thermo FastPrep FP120 was used to break cells after crosslinking. Purified DNA was submitted to the University of Wisconsin-Madison Biotechnology Center. DNA concentration and sizing were verified using the Qubit dsDNA HS Assay Kit (Invitrogen, Carlsbad, California, USA) and Agilent DNA HS chip (Agilent Technologies, Inc., Santa Clara, CA, USA), respectively. Samples were prepared according the TruSeq ChIP Sample Preparation kit (Illumina Inc., San Diego, California, USA) with minor modifications. Libraries were size selected for an average insert size of 350 bp using SPRI-based bead selection. Quality and quantity of the finished libraries were assessed using an Agilent DNA1000 chip and Qubit dsDNA HS Assay Kit, respectively. Libraries were standardized to 2nM. Paired end 300bp sequencing was performed, using SBS chemistry (v3) on an Illumina MiSeq sequencer. Images were analyzed using the standard Illumina Pipeline, version 1.8.2. Sequencing data for this project are available at the NCBI BioProject ID PRJNA428768. For the data shown in Fig 4, downstream analyses were performed as described (Weiner et al., 2015). For the data presented as histograms relative to a baseline value derived from 239 euchromatic intergenic regions lacking either origins or ORC-site matches (Figs 3D, 3F, 4A, 4B, 5A, 5B, 5E and S5 and S7 Figs), the average signal for six contiguous nucleosomes in these regions was used. Thus each nucleosome assessed was normalized to the same value, and the log2 value of this ratio was plotted.

ChIP-Seq experiments

For the MCM ChIP-Seq experiments, yeast cells were grown in liquid YPD at 25°C from single colonies until they reached an A600 of 0.2, at which point nocodazole was added to a final concentration of 15ug/ml and the cultures incubated for 2.5 hours, to obtain a uniform G2/M-phase arrest (S1 Fig). Cultures were shifted to 37°C for 30 minutes, the non-permissive growth temperature for cdc6-4, and then released from the nocodazole arrest at 37°C. When the majority of cells had entered G1-phase (55-minutes post-release for CDC6 SIR2, sir2Δ and cdc6-4 sir2Δ and 110 minutes for cdc6-4), OD equivalents of each cell line were cross-linked with formaldehyde for 15 minutes and then harvested, and chromatin was prepared for IP, by sonication for ChIP-Seq. Mcm2 ChIP was performed using a monoclonal antibody raised against yeast Mcm2 (gift of Bruce Stillman, Cold Spring Harbor Laboratory). Three independent biological replicates and two technical replicates for each biological replicate were performed for each strain. The ChIP DNA was prepared for deep sequencing by the UW-Madison Biotechnology Center using http://www.biooscientific.com/Portals/0/Manuals/NGS/5143-01-NEXTflex-ChIP-Seq-Kit.pdf. Sequencing was done on the HiSeq2000 1x100. Quality and quantity of the finished libraries were assessed using an Agilent DNA1000 chip and Qubit dsDNA HS Assay Kit, respectively. Libraries were standardized to 2nM. Cluster generation was performed using standard Cluster Kits and the Illumina Cluster Station. Single-end, 100bp sequencing was performed, using standard SBS chemistry on an Illumina HiSeq2000 sequencer. Images were analyzed using the standard Illumina Pipeline, version 1.8.2. Sequencing data for this project are available at the NCBI BioProject ID PRJNA428768.

For the downstream analyses (Fig 1 and S2 Fig), all data for a given strain were combined as each replicate generated virtually identical reads. However, for the cdc6-4 strain, which produced extremely low signal-to-noise data, only two technical replicates from a single biological sample were used. For downstream analyses using MochiView the CG1 format data was collated into 25 bp bins (Fig 1). The combined data from each strain was run through CisGenome to identify the top 1000 peaks, divided into 100 peak bins and the percent of peaks that overlapped with at least one ARS (ARS coordinates used from oriDB) determined. The oriDB lists 410 yeast origins as confirmed (http://cerevisiae.oridb.org/). Based on these data and the above analyses we chose to examine and compare the top 400 peaks from each sample for our analyses (S2 Fig). The raw data for CDC6 SIR2, sir2Δ and cdc6-4 sir2Δ were normalized and scaled to the same cdc6-4 data for to generate the final MCM signal values for downstream analyses.

The H4K16-acetylation MNase-ChIP DNA was generated from yeast strains M138 (CDC6 SIR2 strain used in Fig 1) and M2126 (CDC6 sir2Δ strain used in Fig 1) following the protocol described in [32], except a Bio101 Thermo FastPrep FP120 was used to break cells after crosslinking. Purified DNA was submitted to the University of Wisconsin-Madison Biotechnology Center. DNA concentration and sizing were verified using the Qubit dsDNA HS Assay Kit (Invitrogen, Carlsbad, California, USA) and Agilent DNA HS chip (Agilent Technologies, Inc., Santa Clara, CA, USA), respectively. Samples were prepared according the TruSeq ChIP Sample Preparation kit (Illumina Inc., San Diego, California, USA) with minor modifications. Libraries were size selected for an average insert size of 350 bp using SPRI-based bead selection. Quality and quantity of the finished libraries were assessed using an Agilent DNA1000 chip and Qubit dsDNA HS Assay Kit, respectively. Libraries were standardized to 2nM. Paired end 300bp sequencing was performed, using SBS chemistry (v3) on an Illumina MiSeq sequencer. Images were analyzed using the standard Illumina Pipeline, version 1.8.2. Sequencing data for this project are available at the NCBI BioProject ID PRJNA428768. For the data shown in Fig 4, downstream analyses were performed as described (Weiner et al., 2015). For the data presented as histograms relative to a baseline value derived from 239 euchromatic intergenic regions lacking either origins or ORC-site matches (Figs 3D, 3F, 4A, 4B, 5A, 5B, 5E and S5 and S7 Figs), the average signal for six contiguous nucleosomes in these regions was used. Thus each nucleosome assessed was normalized to the same value, and the log2 value of this ratio was plotted.

rDNA copy number determinationQuantitative PCR reactions were carried out in sealed 200 ul microplates in a BioRad C1000 Thermocycler, CFX96 Real-Time System. The conditions were: 95°C-3’|95°C -30”, 58°C -15”|x40 cycles Standard Melt Curve Analysis. Each reaction contained: 0.04 ng/uL genomic DNA template, 0.016 nM of each primer, and 1X qPCR master mix (0.1 mM dNTPs, 1.25% formamide, 0.1 mg/mL BSA, 0.5 U Taq DNA polymerase, 5 mM Tris-pH 8.0, 10 mM KCl, 1.5 mM MgCl2, 0.75% Triton and 0.5X SYBR Green DNA stain.) DNA concentration and primer efficiency validation were performed by testing primer efficiency by titrating template concentrations. Serial dilutions of DNA were used to test a range of DNA concentrations from 0.1 ng/uL to 0.0016 ng/uL in technical triplicates. Efficiency values for RIM15, NTS2 and NL primer sets were calculated as 83%, 83%, and 81%, respectively, by the BioRad CFX96 software. The entire range of DNA concentrations was found to be within the linear range of instrument response. Melt curve analysis revealed a single product from each primer pair that was confirmed in each experimental run. The primer pairs were: For rDNA: NTS2: GGGCGATAATGACGGGAAGA-Fwd and TGTCCACTTTCAACCGTCCC-Rev; 35S (D1/D2 loop of 26S rDNA): GCATATCAATAAGCGGAGGAAAAG-Fwd and ACTTTACAAAGAACCGCACTCC-Rev; For single copy control: RIM15: GCCAGAACATTGGGTCAGAT-Fwd and CCGGATACTCGGATGTGTCT-Rev. For the single copy experimental (aqua bars in Fig 2B) ERV46: CACAGCTAGGACACCACCAA-Fwd and AGGGACAAGGATCATCCAAA-Rev. The enrichment values are equal to 2- ΔCt where ΔCt = Ct (rDNA locus (or ERV46))—Ct (single copy locus, RIM15) [72]. This value should equal the copy number of the target locus being assessed.

rDNA copy number determination

Quantitative PCR reactions were carried out in sealed 200 ul microplates in a BioRad C1000 Thermocycler, CFX96 Real-Time System. The conditions were: 95°C-3’|95°C -30”, 58°C -15”|x40 cycles Standard Melt Curve Analysis. Each reaction contained: 0.04 ng/uL genomic DNA template, 0.016 nM of each primer, and 1X qPCR master mix (0.1 mM dNTPs, 1.25% formamide, 0.1 mg/mL BSA, 0.5 U Taq DNA polymerase, 5 mM Tris-pH 8.0, 10 mM KCl, 1.5 mM MgCl2, 0.75% Triton and 0.5X SYBR Green DNA stain.) DNA concentration and primer efficiency validation were performed by testing primer efficiency by titrating template concentrations. Serial dilutions of DNA were used to test a range of DNA concentrations from 0.1 ng/uL to 0.0016 ng/uL in technical triplicates. Efficiency values for RIM15, NTS2 and NL primer sets were calculated as 83%, 83%, and 81%, respectively, by the BioRad CFX96 software. The entire range of DNA concentrations was found to be within the linear range of instrument response. Melt curve analysis revealed a single product from each primer pair that was confirmed in each experimental run. The primer pairs were: For rDNA: NTS2: GGGCGATAATGACGGGAAGA-Fwd and TGTCCACTTTCAACCGTCCC-Rev; 35S (D1/D2 loop of 26S rDNA): GCATATCAATAAGCGGAGGAAAAG-Fwd and ACTTTACAAAGAACCGCACTCC-Rev; For single copy control: RIM15: GCCAGAACATTGGGTCAGAT-Fwd and CCGGATACTCGGATGTGTCT-Rev. For the single copy experimental (aqua bars in Fig 2B) ERV46: CACAGCTAGGACACCACCAA-Fwd and AGGGACAAGGATCATCCAAA-Rev. The enrichment values are equal to 2- ΔCt where ΔCt = Ct (rDNA locus (or ERV46))—Ct (single copy locus, RIM15) [72]. This value should equal the copy number of the target locus being assessed.

Histone H3 and H4 suppressor screenStrains M1321 (MATa
ade2 ura3 leu2-3,112 trp1 hht1,hhf1::LEU2 hht2,hhf2::HIS3/pDP378 (HHT2,HHF2 CEN6 ARSH4 URA3) and M1345 (M1321, cdc6::ura3 LEU2::cdc6-4) were transformed with the TRP1 histone H3, H4 plasmids pH3H4-WT and mutant derivatives as shown in Fig 6A at 25°C. Multiple transformants were streak purified on FOA medium, recovered to YPD at 25°C and then screened for growth at various temperatures on YPD medium. Histone mutations were also constructed in pDP373 (pRS414 (TRP1) containing HHT2-HHF2 on a Spe1 fragment) using QuikChange. The entire H3 and H4 genes were sequenced for varification, and subsequently transformed in M1321 and M1345 for the yeast growth experiments in Fig 6.

Histone H3 and H4 suppressor screen

Strains M1321 (MATa
ade2 ura3 leu2-3,112 trp1 hht1,hhf1::LEU2 hht2,hhf2::HIS3/pDP378 (HHT2,HHF2 CEN6 ARSH4 URA3) and M1345 (M1321, cdc6::ura3 LEU2::cdc6-4) were transformed with the TRP1 histone H3, H4 plasmids pH3H4-WT and mutant derivatives as shown in Fig 6A at 25°C. Multiple transformants were streak purified on FOA medium, recovered to YPD at 25°C and then screened for growth at various temperatures on YPD medium. Histone mutations were also constructed in pDP373 (pRS414 (TRP1) containing HHT2-HHF2 on a Spe1 fragment) using QuikChange. The entire H3 and H4 genes were sequenced for varification, and subsequently transformed in M1321 and M1345 for the yeast growth experiments in Fig 6.

Supporting informationS1 FigFlow cytometry analyses of the four strains used for the MCM-ChIP-Seq experiment as described in Fig 1.Formaldehyde was added to the CDC6 SIR2, CDC6 sir2Δ, cdc6-4 sir2Δ cell cultures at 55 minutes aftert nocodazole release. For cdc6-4 SIR2 cells, formaldehyde was added 105 minutes after nocodazole release.(PDF)Click here for additional data file.S2 FigOverlap between origin data sets was substantial.A. The MCM ChIP-Seq signals that identified confirmed origins in CDC6 SIR2 and CDC6 sir2Δ cells were compared. 340 confirmed origins were identified among the top 400 MCM peaks in CDC6 sir2Δ, while 338 were identified in CDC6 SIR2 cells. Among these, 330 were contained in both the CDC6 SIR2 and CDC6 sir2Δ data sets, indicating >97% overlap. For the top 400 peaks identified in the cdc6-4 sir2Δ, 279 identified confirmed origins. 272 of these were also found in either the CDC6 SIR2 or the CDC6 sir2Δ data sets, indicating that ~83% of confirmed origins otherwise defective in MCM association in cdc6-4 cells were rescued by the sir2Δ mutation. B. The areas defining the MCM ChIP-signals in CDC6 SIR2 cells were determined, and then the corresponding coordinates were used to determine the areas under the corresponding regions in the mutant cells. Comparison of CDC6 sir2Δ and CDC6 SIR2 data generated a strong correspondence, indicating that in otherwise wild type cells, a sir2Δ had a minimal effect on MCM distribution across origins. In contrast, comparison of the SIR2 cdc6-4 and CDC6 SIR2 data sets indicated a poor relationship between the areas of signals for origins, consistent with cdc6-4 abolishing virtually all MCM binding. However, the cdc6-4 sir2Δ cells generated areas more similar to that of wild type CDC6 SIR2 or CDC6 sir2Δ cells, consistent with the substantial amount of MCM binding rescue described in Fig 1 and in (A) of this figure.(PDF)Click here for additional data file.S3 FigSIR2-regulation on an ARS1005 plasmid was observed only with a relatively large chromosomal fragment of ~500 bp.A.
ARS1005 was an origin that showed substantial rescue of MCM binding (by ChIP-Seq) in cdc6-4 sir2Δ cells. The blue line indicates the trace of nucleosome occupancy at ARS1005 and the yellow boxes indicate the extent of ARS1005 sequences inserted in the CEN4 URA3 plasmid pARS1-WT, that replaced ARS1 as described (Chang et al. 2011, NAR). The ACS (ARS Consensus Sequence), the conserved 11-bp core sequence element within the binding site for yeast ORC, is indicated with a red line and was confirmed by mutation in pFJ199 and pMW564 contexts. B. The indicated plasmid clones were transformed into WT (M138), cdc6-4 (M386) and cdc6-4 sir2Δ strains (M922) and then assayed for plasmid stability over approximately 10 generations as described [23] (Crampton et al., 2008, Mol Cell). The high cdc6-4 plasmid loss rate of pFJ197 and pFJ198 (that did not contain chromosomal sequences overlapping positioned nucleosomes) were not significantly rescued by sir2Δ. In contrast, those plasmids that contained larger inserts of DNA that would contain one (pFJ188 and pMW564) or both (pFJ199) positioned nucleosomes were significantly rescued.(PDF)Click here for additional data file.S4 FigAssessing rDNA copy number in the fob1Δ cells used in Fig 2D.The cdc6-4 fob1Δ strains for the experiment in Fig 2D were generated by crossing a cdc6-4 FOB1 and CDC6 fob1Δ: HIS3 rDNA-35 (or fob1Δ fob1Δ: HIS3 rDNA-180, as appropriate, where rDNA-## indicates number of copies of the rDNA locus) parents. The resulting HIS3+ and temperature-sensitive haploids examined in Fig 2D were confirmed to be cdc6-4 by PCR of a 700 bp region followed by a digest with HhaI to generate ~600 and ~100 bp fragments. The rDNA copy number of these haploids was then confirmed by qPCR using the same primers described in Fig 2. The data show the enrichment values for the rDNA copy number of these strains, as indicated, for three technical replicates each and the error-bars indicate standard deviation for the three independent reactions.(PDF)Click here for additional data file.S5 FigAnalyses of acetylation marks on nucleosomes adjacent to the indicated origin groups.Most acetylation marks were depleted similarly from nucleosomes adjacent to origin and non-origin loci relative to the control intergenic loci. Normalized acetylation status was determined for each of the indicated acetylation marks on each of the indicated nucleosomes using the genome-wide histone modification atlas from [32] (Weiner et al., 2015) as described for Fig 3.(PDF)Click here for additional data file.S6 FigBox-and-Whiskers analyses of H4 K16Ac on nucleosomes adjacent to the indicated origin groups.Log2 values for H4K16ac/total (input) ratios (y-axis) were determined for each indicated nucleosome (x-axis) for each origin group indicated (colored columns). The euchromatic origins were subdivided into three subgroups based on their SIR2-reponsiveness (i.e. the ratio of the MCM ChIP signal over the origin coordinates in cdc6-4 sir2Δ mutant cells to the MCM ChIP signal in wild type (CDC6 sir2Δ) cells.) The origin from within the HM SIR-heterochromatic loci (silencers) and from within the SIR-heterochromatic telomeres were also assessed for H4K16ac in this experiment. The “0” binding baseline was generated using all
S. cerevisiae nucleotides (i.e. without pre-excluding nucleotides from within SIR-heterochromatin) using the method described in the main text.(PDF)Click here for additional data file.S7 FigExamples of H4K16ac status at selected individual origins.The H4K16ac status for the indicated origin-adjacent was determined as described for Fig 3D. A. Three origins selected from the quintile containing low SIR2-responsive origins as described for Fig 3E. B. Three origins selected from the quintile containing the high SIR2-responsive origins as described for Fig 3E.(PDF)Click here for additional data file.S8 FigSir3-3xHA ChIP-Seq signals at euchromatic origins using a baseline generated using all S. cerevisiae nucleotides.A. Sir3-3xHA ChIP-Seq signals at euchromatic origins and non-origin ORC-site loci relative to a baseline generated using all using all
S. cerevisiae nucleotides (i.e. without pre-excluding nucleotides from within SIR-heterochromatin) using the method of [49] and as described in the main text. The data are from [50]. These data are for comparison to the Sir3 data shown in Fig 5C of the main text and indicated that excluding SIR-heterochromatin regions shifts the baseline for “0” Sir3 binding, as expected if the vast majority of stable Sir3 binding is confined to the telomeric and HM regions of the genome. However, the relative Sir3-3xHA ChIP-Seq signal differences between origins and non-origin ORC-site control loci are still apparent. B. Sir3-3xHA ChIP-Seq signals at the three groups of euchromatic origins that differ based on their SIR2-responsiveness as shown in Figs 3E and 5D. C. The SIR2-responsive origins comprising the three groups assessed in ‘B’ were randomized into three different sets and the Sir3-3xHA ChIP-Seq signals determined and plotted as in ‘B’.(PDF)Click here for additional data file.S9 FigYeast growth assays as in Fig 6B for additional histone mutants.“WT” refers to cdc6-4. That is, all strains assessed are of the cdc6-4 genotype, as in Fig 6B.(PDF)Click here for additional data file.

Supporting information

Flow cytometry analyses of the four strains used for the MCM-ChIP-Seq experiment as described in Fig 1.

Formaldehyde was added to the CDC6 SIR2, CDC6 sir2Δ, cdc6-4 sir2Δ cell cultures at 55 minutes aftert nocodazole release. For cdc6-4 SIR2 cells, formaldehyde was added 105 minutes after nocodazole release.

Click here for additional data file.

Overlap between origin data sets was substantial.

A. The MCM ChIP-Seq signals that identified confirmed origins in CDC6 SIR2 and CDC6 sir2Δ cells were compared. 340 confirmed origins were identified among the top 400 MCM peaks in CDC6 sir2Δ, while 338 were identified in CDC6 SIR2 cells. Among these, 330 were contained in both the CDC6 SIR2 and CDC6 sir2Δ data sets, indicating >97% overlap. For the top 400 peaks identified in the cdc6-4 sir2Δ, 279 identified confirmed origins. 272 of these were also found in either the CDC6 SIR2 or the CDC6 sir2Δ data sets, indicating that ~83% of confirmed origins otherwise defective in MCM association in cdc6-4 cells were rescued by the sir2Δ mutation. B. The areas defining the MCM ChIP-signals in CDC6 SIR2 cells were determined, and then the corresponding coordinates were used to determine the areas under the corresponding regions in the mutant cells. Comparison of CDC6 sir2Δ and CDC6 SIR2 data generated a strong correspondence, indicating that in otherwise wild type cells, a sir2Δ had a minimal effect on MCM distribution across origins. In contrast, comparison of the SIR2 cdc6-4 and CDC6 SIR2 data sets indicated a poor relationship between the areas of signals for origins, consistent with cdc6-4 abolishing virtually all MCM binding. However, the cdc6-4 sir2Δ cells generated areas more similar to that of wild type CDC6 SIR2 or CDC6 sir2Δ cells, consistent with the substantial amount of MCM binding rescue described in Fig 1 and in (A) of this figure.

Click here for additional data file.

SIR2-regulation on an ARS1005 plasmid was observed only with a relatively large chromosomal fragment of ~500 bp.

A.
ARS1005 was an origin that showed substantial rescue of MCM binding (by ChIP-Seq) in cdc6-4 sir2Δ cells. The blue line indicates the trace of nucleosome occupancy at ARS1005 and the yellow boxes indicate the extent of ARS1005 sequences inserted in the CEN4 URA3 plasmid pARS1-WT, that replaced ARS1 as described (Chang et al. 2011, NAR). The ACS (ARS Consensus Sequence), the conserved 11-bp core sequence element within the binding site for yeast ORC, is indicated with a red line and was confirmed by mutation in pFJ199 and pMW564 contexts. B. The indicated plasmid clones were transformed into WT (M138), cdc6-4 (M386) and cdc6-4 sir2Δ strains (M922) and then assayed for plasmid stability over approximately 10 generations as described [23] (Crampton et al., 2008, Mol Cell). The high cdc6-4 plasmid loss rate of pFJ197 and pFJ198 (that did not contain chromosomal sequences overlapping positioned nucleosomes) were not significantly rescued by sir2Δ. In contrast, those plasmids that contained larger inserts of DNA that would contain one (pFJ188 and pMW564) or both (pFJ199) positioned nucleosomes were significantly rescued.

Click here for additional data file.

Assessing rDNA copy number in the fob1Δ cells used in Fig 2D.

The cdc6-4 fob1Δ strains for the experiment in Fig 2D were generated by crossing a cdc6-4 FOB1 and CDC6 fob1Δ: HIS3 rDNA-35 (or fob1Δ fob1Δ: HIS3 rDNA-180, as appropriate, where rDNA-## indicates number of copies of the rDNA locus) parents. The resulting HIS3+ and temperature-sensitive haploids examined in Fig 2D were confirmed to be cdc6-4 by PCR of a 700 bp region followed by a digest with HhaI to generate ~600 and ~100 bp fragments. The rDNA copy number of these haploids was then confirmed by qPCR using the same primers described in Fig 2. The data show the enrichment values for the rDNA copy number of these strains, as indicated, for three technical replicates each and the error-bars indicate standard deviation for the three independent reactions.

Click here for additional data file.

Analyses of acetylation marks on nucleosomes adjacent to the indicated origin groups.

Most acetylation marks were depleted similarly from nucleosomes adjacent to origin and non-origin loci relative to the control intergenic loci. Normalized acetylation status was determined for each of the indicated acetylation marks on each of the indicated nucleosomes using the genome-wide histone modification atlas from [32] (Weiner et al., 2015) as described for Fig 3.

Click here for additional data file.

Box-and-Whiskers analyses of H4 K16Ac on nucleosomes adjacent to the indicated origin groups.

Log2 values for H4K16ac/total (input) ratios (y-axis) were determined for each indicated nucleosome (x-axis) for each origin group indicated (colored columns). The euchromatic origins were subdivided into three subgroups based on their SIR2-reponsiveness (i.e. the ratio of the MCM ChIP signal over the origin coordinates in cdc6-4 sir2Δ mutant cells to the MCM ChIP signal in wild type (CDC6 sir2Δ) cells.) The origin from within the HM SIR-heterochromatic loci (silencers) and from within the SIR-heterochromatic telomeres were also assessed for H4K16ac in this experiment. The “0” binding baseline was generated using all
S. cerevisiae nucleotides (i.e. without pre-excluding nucleotides from within SIR-heterochromatin) using the method described in the main text.

Click here for additional data file.

Examples of H4K16ac status at selected individual origins.

The H4K16ac status for the indicated origin-adjacent was determined as described for Fig 3D. A. Three origins selected from the quintile containing low SIR2-responsive origins as described for Fig 3E. B. Three origins selected from the quintile containing the high SIR2-responsive origins as described for Fig 3E.

Click here for additional data file.

Sir3-3xHA ChIP-Seq signals at euchromatic origins using a baseline generated using all S. cerevisiae nucleotides.

A. Sir3-3xHA ChIP-Seq signals at euchromatic origins and non-origin ORC-site loci relative to a baseline generated using all using all
S. cerevisiae nucleotides (i.e. without pre-excluding nucleotides from within SIR-heterochromatin) using the method of [49] and as described in the main text. The data are from [50]. These data are for comparison to the Sir3 data shown in Fig 5C of the main text and indicated that excluding SIR-heterochromatin regions shifts the baseline for “0” Sir3 binding, as expected if the vast majority of stable Sir3 binding is confined to the telomeric and HM regions of the genome. However, the relative Sir3-3xHA ChIP-Seq signal differences between origins and non-origin ORC-site control loci are still apparent. B. Sir3-3xHA ChIP-Seq signals at the three groups of euchromatic origins that differ based on their SIR2-responsiveness as shown in Figs 3E and 5D. C. The SIR2-responsive origins comprising the three groups assessed in ‘B’ were randomized into three different sets and the Sir3-3xHA ChIP-Seq signals determined and plotted as in ‘B’.

Click here for additional data file.

Yeast growth assays as in Fig 6B for additional histone mutants.

“WT” refers to cdc6-4. That is, all strains assessed are of the cdc6-4 genotype, as in Fig 6B.

Click here for additional data file.
