---
pmid: '26829474'
title: Acetylation of Aurora B by TIP60 ensures accurate chromosomal segregation.
authors:
- Mo F
- Zhuang X
- Liu X
- Yao PY
- Qin B
- Su Z
- Zang J
- Wang Z
- Zhang J
- Dou Z
- Tian C
- Teng M
- Niu L
- Hill DL
- Fang G
- Ding X
- Fu C
- Yao X
journal: Nat Chem Biol
year: '2016'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4798883
doi: 10.1038/nchembio.2017
---

# Acetylation of Aurora B by TIP60 ensures accurate chromosomal segregation.
**Authors:** Mo F, Zhuang X, Liu X, Yao PY, Qin B, Su Z, Zang J, Wang Z, Zhang J, Dou Z, Tian C, Teng M, Niu L, Hill DL, Fang G, Ding X, Fu C, Yao X
**Journal:** Nat Chem Biol (2016)
**DOI:** [10.1038/nchembio.2017](https://doi.org/10.1038/nchembio.2017)
**PMC:** [PMC4798883](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4798883/)

## Abstract

1. Nat Chem Biol. 2016 Apr;12(4):226-32. doi: 10.1038/nchembio.2017. Epub 2016
Feb  1.

Acetylation of Aurora B by TIP60 ensures accurate chromosomal segregation.

Mo F(1)(2), Zhuang X(1)(2), Liu X(1)(2)(3)(4)(5), Yao PY(5), Qin B(1)(2)(5), Su 
Z(5)(6), Zang J(1)(2)(3)(4), Wang Z(1)(2)(3)(4), Zhang J(1)(2)(7), Dou 
Z(1)(2)(3)(4), Tian C(1)(2)(3)(4), Teng M(1)(2)(3)(4), Niu L(1)(2)(3)(4), Hill 
DL(8), Fang G(1)(2), Ding X(5)(6), Fu C(1)(2)(3)(4), Yao X(1)(2)(3)(4).

Author information:
(1)Anhui Key Laboratory for Cellular Dynamics &Chemical Biology, Hefei, China.
(2)Laboratory for Organelle Dynamics &Plasticity Control, University of Science 
&Technology of China School of Life Sciences, Hefei, China.
(3)Chinese Academy of Sciences Center for Excellence in Molecular Cell Science, 
Hefei, China.
(4)Hefei National Laboratory for Physical Sciences at Nanoscale, Hefei, China.
(5)Molecular Imaging Center, Morehouse School of Medicine, Atlanta, Georgia, 
USA.
(6)Department of Medical Cell Biology, Beijing University of Chinese Medicine, 
Beijing, China.
(7)State Key Laboratory of Respiratory Diseases, Guangzhou, China.
(8)Comprehensive Cancer Center, University of Alabama, Birmingham, Alabama, USA.

Comment in
    Nat Chem Biol. 2016 Apr;12(4):204-5. doi: 10.1038/nchembio.2041.

Faithful segregation of chromosomes in mammalian cells requires bi-orientation 
of sister chromatids, which relies on the sensing of correct attachments between 
spindle microtubules and kinetochores. Although the mechanisms underlying 
cyclin-dependent kinase 1 (CDK1) activation, which triggers mitotic entry, have 
been extensively studied, the regulatory mechanisms that couple CDK1-cyclin B 
activity to chromosome stability are not well understood. Here, we identified a 
signaling axis in which Aurora B activity is modulated by CDK1-cyclin B via the 
acetyltransferase TIP60 in human cell division. CDK1-cyclin B phosphorylates 
Ser90 of TIP60, which elicits TIP60-dependent acetylation of Aurora B and 
promotes accurate chromosome segregation in mitosis. Mechanistically, TIP60 
acetylation of Aurora B at Lys215 protects Aurora B's activation loop from 
dephosphorylation by the phosphatase PP2A to ensure a robust, error-free 
metaphase-anaphase transition. These findings delineate a conserved signaling 
cascade that integrates protein phosphorylation and acetylation with cell cycle 
progression for maintenance of genomic stability.

DOI: 10.1038/nchembio.2017
PMCID: PMC4798883
PMID: 26829474 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing financial 
interests.

## Full Text

INTRODUCTIONMaintenance of chromosome stability is pivotal for cellular homeostasis1,2, wherein sensing of genomic alterations represents a necessary step3,4. Several checkpoint pathways, such as DNA damage response (DDR) and spindle assembly checkpoint (SAC), orchestrate cell cycle progression to satisfy quality control. Central to DNA damage response signaling is the signaling cascade of protein kinase ATM and acetyltransferase TIP605. Binding of TIP60 to Lys9 tri-methylated histone H3 (H3K9me3) promotes TIP60-dependent activation of ATM6. Since H3K9me3 also involves in centromere assembly and chromosome stability, and since some key components of DNA damage response signaling, such as Chk17, Chk28,9, and BRCA19 are essentail for proper mitotic progression, the outstanding questions were how TIP60 function at the centromere was regulated and whether TIP60 also guided chromosome dynamics and stability during mitosis.Aurora B kinase is the catalytic subunit of the chromosome passenger complex (CPC), which localizes to inner-centromeres in early mitosis and transfers to central spindle upon metaphase-anaphase transition. CPC complex governs chromosome segregation by generating spindle checkpoint signals and correcting aberrant kinetochore-microtubule attachments. Although inner-centromere localization of CPC also relies on post-translational modification of histone H3 tails, whether and how Aurora B and TIP60 cooperate in centromere remains unclear.Here, we identified a novel regulatory mechanism underlying Aurora B kinase activation by TIP60 dependent acetylation. We found that TIP60 co-localized with phosphorylated Aurora B at the outer-kinetochore, where TIP60 direct acetylated Aurora B at lysine 215, which was essential for Aurora B activation and chromosome bi-orientation. Biochemical characterization demonstrated that lysine 215 (Lys215) acetylation prevent PP2A dependent dephosphorylation of Aurora B at threonine 232 (Thr232), thus sustained a robust activity for erroneous attachment correction. Interestingly, CDK1-cyclin B phosphorylated TIP60 at serine 90 (S90), which promoted TIP60 activity at kinetochores and elicited TIP60-dependent acetylation of Aurora B. Our study characterized a novel mechanism underlying CDK1-TIP60-Aurora B signaling axis that promoted accurate mitotic process and genomic stability.

INTRODUCTION

Maintenance of chromosome stability is pivotal for cellular homeostasis1,2, wherein sensing of genomic alterations represents a necessary step3,4. Several checkpoint pathways, such as DNA damage response (DDR) and spindle assembly checkpoint (SAC), orchestrate cell cycle progression to satisfy quality control. Central to DNA damage response signaling is the signaling cascade of protein kinase ATM and acetyltransferase TIP605. Binding of TIP60 to Lys9 tri-methylated histone H3 (H3K9me3) promotes TIP60-dependent activation of ATM6. Since H3K9me3 also involves in centromere assembly and chromosome stability, and since some key components of DNA damage response signaling, such as Chk17, Chk28,9, and BRCA19 are essentail for proper mitotic progression, the outstanding questions were how TIP60 function at the centromere was regulated and whether TIP60 also guided chromosome dynamics and stability during mitosis.

Aurora B kinase is the catalytic subunit of the chromosome passenger complex (CPC), which localizes to inner-centromeres in early mitosis and transfers to central spindle upon metaphase-anaphase transition. CPC complex governs chromosome segregation by generating spindle checkpoint signals and correcting aberrant kinetochore-microtubule attachments. Although inner-centromere localization of CPC also relies on post-translational modification of histone H3 tails, whether and how Aurora B and TIP60 cooperate in centromere remains unclear.

Here, we identified a novel regulatory mechanism underlying Aurora B kinase activation by TIP60 dependent acetylation. We found that TIP60 co-localized with phosphorylated Aurora B at the outer-kinetochore, where TIP60 direct acetylated Aurora B at lysine 215, which was essential for Aurora B activation and chromosome bi-orientation. Biochemical characterization demonstrated that lysine 215 (Lys215) acetylation prevent PP2A dependent dephosphorylation of Aurora B at threonine 232 (Thr232), thus sustained a robust activity for erroneous attachment correction. Interestingly, CDK1-cyclin B phosphorylated TIP60 at serine 90 (S90), which promoted TIP60 activity at kinetochores and elicited TIP60-dependent acetylation of Aurora B. Our study characterized a novel mechanism underlying CDK1-TIP60-Aurora B signaling axis that promoted accurate mitotic process and genomic stability.

RESULTSTIP60 is essential for faithful chromosome segregationTo elucidate the function of TIP60 in mitosis, we employed two shRNAs to suppress endogenous TIP60 (Supplementary Results, Supplementary Fig. 1a and b) and examined the resulting phenotype using time-lapse microscopy (Fig. 1a; Supplementary Fig. 1c). Although the control cells progressed through mitosis normally, TIP60-depleted cells exhibited a high frequency of chromosome segregation defects, including anaphase lagging chromosome, chromosome misalignment, and chromosome bridges (Fig. 1b, c and d). The defects seen in the TIP60-deficient cells were largely rescued by expressing an RNAi-resistant, wild-type TIP60, but not an acetyltransferase-deficient form of TIP6010 (Supplementary Fig. 1d, e, f and g), demonstrating the role of TIP60 acetyltransferase activity in mitotic chromosome movements. To eliminate the possibility that the phenotype was a consequence of disruption of TIP60 function in the DNA damage response pathway, we introduced two small molecule inhibitors of TIP60 acetyltransferase (NU905611 and MG14912) into cultured cells immediately after mitotic entry. As expected, TIP60 acetyltransferase activity inhibition in mitosis resulted in increased rates of chromosome mis-segregation (Fig. 1e, f and g; Supplementary Fig. 1h). Thus, TIP60 acetyltransferase activity was essential for chromosome alignment and accurate metaphase-anaphase transition. The function of TIP60 in genomic stability was apparent, as suppression of TIP60 gave rise to micronuclei and polyploidy (Supplementary Fig. 1i, j and k).TIP60 modulates Aurora B kinase activity at kinetochoresTo delineate the mechanism of action underlying TIP60 function in mitosis, we examined the subcellular distribution of TIP60 in mitosis. Consistent with its function, TIP60 localized to kinetochores in early mitosis, as determined by two different TIP60 antibodies (Fig. 2a; Supplementary Fig. 2a). The localization of TIP60 to kinetochores was independent of microtubules, as nocodazole treatment did not alter its localization (Supplementary Fig. 2b). In a line-scan, the fluorescence intensity profiles suggested that TIP60 co-localized with Hec1 at the outer kinetochores (Fig. 2b and Supplementary Fig. 2c). The kinetochore localization of TIP60 was abolished by shRNA-mediated suppression, confirming the specificity of the antibody (Supplementary Fig. 2d). Furthermore, exogenously expressed TIP60-GFP also localized to the kinetochores in prometaphase cells (Supplementary Fig. 2e), confirming that TIP60 was a kinetochore protein.The localization of TIP60 prompted a search, with a battery of kinase inhibitors, for the molecular determinants underlying its localization, the results indicated that a stable localization of TIP60 to the kinetochores specifically required Mps1 kinase activity (Supplementary Fig. 3a, b and c). The Mps1-dependence was confirmed by siRNA treatment in which suppression of Mps1 quantitatively reduced the levels of TIP60 at the kinetochores (Supplementary Fig. 3d). We and two other groups have recently found that Mps1 is recruited to kinetochores by binding to two distinct areas of the Ndc80 complex13–15. To determine whether kinetochore recruitment of TIP60 required adequate Mps1 kinase at kinetochores, we knocked down two components of the Ndc80 complex, Hec1 and Nuf2. The absence of either Nuf2 or Hec1 abolished kinetochore localization of TIP60 (Supplementary Fig. 3e), confirming that Ncd80/Mps1 pathway regulated TIP60 distribution to kinetochore16.Since TIP60 transiently localized to kinetochores in early mitosis when chromosomes attempted for bi-orientation, we questioned whether TIP60 distinguished amphitelic attachment from erroneously attached kinetochores. To this end, we treated HeLa cells with the mitotic kinesin CENP-E inhibitor GSK923295 (IC50 = 3.2 nM)17 to create erroneous kinetochore-microtubule attachments. TIP60 preferentially accumulated to erroneously attached kinetochores near the spindle poles but not to those with apparent amphitelic attachments (Supplementary Fig. 3f). Quantitative analyses of normalized pixel intensities indicated that the level of kinetochore-associated TIP60 was a function of chromosome position to the poles (Fig. 2c). The function of TIP60 in chromosome alignment was further assessed. In a monastrol-washout assay, bi-orientation was achieved only after either of the TIP60 inhibitors was removed (Supplementary Fig. 4a). Consistently, TIP60-depleted cells also failed to undergo accurate chromosome segregation, as lagging chromosomes were frequently observed (Supplementary Fig. 4b), confirming that achievement of chromosome bi-orientation required TIP60 activity. Since Aurora B kinase is a key factor in correcting erroneous attachments18, 19, we sought to examine if the function of TIP60 in promoting accurate kinetochore attachment was mediated via Aurora B signaling. To this end, Aurora B was depleted via siRNA treatment. Although depletion of Aurora B caused a more severe phenotype than that of TIP60 depletion, there was no additive effect when combined suppressing Aurora B and TIP60 (Supplementary Fig. 4c and d), revealing functional relevance between Aurora B and TIP60. Since an active form of Aurora B (Thr232 phosphorylated) localizes to kinetochores20, where it co-localized with TIP60 (Fig. 2d), we further examined the intensity of Thr232-phosphorylated Aurora B in TIP60-depleted cells (Fig. 2e). Quantitative analysis showed that the level of pT232 in TIP60 siRNA-treated cells reduced to 33% of its control value while average Aurora B protein levels exhibited no significant alternation (Supplementary Fig. 4e and f), suggesting that TIP60 regulated activity of kinetochore-bound Aurora B.To probe if Aurora B kinase activity was modulated by TIP60-mediated acetylation, we examined Aurora B activity using phosphorylation of CENP-A at Ser7 (pS7-CENP-A) as a reporter21. Suppressing TIP60 acetyltransferase activity by either TIP60 inhibitors (Fig. 2f) or TIP60 shRNA (Supplementary Fig. 4g and h) reduced CENP-A phosphorylation, suggesting that TIP60-mediated acetylation modulated Aurora B kinase activity in mitotic cells. The effect of TIP60 inhibition on Aurora B activation, together with the co-localization of phosphorylated Aurora B and TIP60, prompted us to determine whether TIP60 formed a complex with Aurora B in vivo. In mitosis, gel filtration assays and reciprocal co-immunoprecipitation confirmed that TIP60 formed a complex with Aurora B (Supplementary Fig. 5a, b and c). Thus, we conclude that TIP60 acetyltransferase modulated Aurora B kinase activity in mitosis.K215 acetylation by TIP60 promotes Aurora B activityWe next determined if TIP60 directly acetylated Aurora B. Western blots for anti-pan-acetyl-lysine (acK) established that Aurora B was a substrate of TIP60 in vitro (Fig. 3a). To identify the physiological substrate sites of TIP60 in Aurora B, purified Aurora B from mitotic HeLa cells was subjected to mass spectrometric analyses which revealed 4 potential acetylation sites, including Lys31, Lys115, Lys215, and Lys231 (Supplementary Fig. 5d). To map the site(s) responsible for TIP60 acetylation, we generated a series of Aurora B mutants, in which the identified acetylation sites were individually mutated to arginine. In the Aurora B Lys215 mutant immunoprecipitated from mitotic cells, Aurora B acetylation was virtually undetectable (Supplementary Fig. 5e), suggesting that acK215 accounted for Aurora B acetylation in mitosis. To characterize the precise cellular function of Lys215 acetylation in mitosis, we generated a Lys215 acetylated Aurora B antibody (acK215-AurB), which specifically recognized both endogenous acetylated Aurora B (Supplementary Fig. 5f and g) and TIP60 acetylated Aurora B in vitro (Supplementary Fig. 5h). To assess whether Aurora B was a cognate substrate of TIP60 in mitosis, we analyzed K215 acetylation in aliquots of asynchronized or nocodazole-synchronized HeLa cells. Lys215 acetylation level was dramatically increased in nocodazole-synchronized cells but reduced after TIP60 inhibition (Supplementary Fig. 5i), indicating that K215 was a cognate substrate of TIP60 in mitotic cells. To examine the precise cellular function of K215 acetylation by TIP60, we knocked-down endogenous TIP60 and assessed the activity of Aurora B by in vitro kinase assays. Aurora B kinase activity was alleviated in TIP60 depleted cells, consistent with the reduced K215 acetylation level (Fig. 3b), indicating that both Lys215 acetylation and kinase activity of Aurora B required TIP60.Since Lys215 is highly conserved among species and locates close to the activation loop of Aurora B (Supplementary Fig. 5j), we hypothesized that TIP60 promoted Aurora B activity through acetylating Aurora B at Lys215. To confirm the hypothesis, we purified FLAG-tagged wild-type (WT), acetylation-mimicking (K215Q) and non-acetylatable (K215R) mutants from mitotic cells for in vitro kinase assay and enzymatic characterization. As reflected by H3 phosphorylation, TIP60 mediated K215 acetylation robustly stimulated Aurora B activity (Fig. 3c and d; Supplementary Fig. 5k).K215 acetylation promotes chromosome bi-orientationSince TIP60 distinguished bi-oriented kinetochores from erroneously attached kinetochores, we next tested if alteration of the kinetochore-microtubule attachment modulated distribution of acetylated Aurora B at the kinetochores. AcK215 staining was apparent in prometaphase when bi-orientation was not achieved in most kinetochores, whereas the staining was diminished in metaphase when most kinetochores were properly attached to microtubules (Fig. 3e; Supplementary Fig. 6a).If acetylation of Lys215 was involved in correction of aberrant kinetochore attachments, the level of acK215 would increase in the misaligned kinetochores. To test this hypothesis, we exposed HeLa cells to GSK923295 to create both bi-oriented and misaligned kinetochores in the same cell, in which the labeling of acK215 was brighter on the misaligned kinetochores (Fig. 3f). Statistical analysis indicated that acK215 levels of mis-aligned kinetochores were about 6-fold higher than those near the equator (Supplementary Fig. 6a). Since pT232 is a reporter of Aurora B activity, we also examined the levels of pT232 in GSK923295-treated cells. In these cells, the levels of pT232 were elevated at the kinetochores near the pole relative to kinetochores near the equator (Supplementary Fig. 6b). Statistical analyses showed that the pT232-AurB levels of misaligned kinetochores were much higher than those near the equator (Supplementary Fig. 6c), supporting the notion that acK215 promoted Aurora B activity at mis-attached and unattached kinetochores. Intriguingly, a brief treatment with the TIP60 inhibitor NU9056 abolished the labeling of acK215 and pT232 at the misaligned kinetochores (Supplementary Fig. 6d and e), indicating that TIP60 regulated Aurora B acetylation and activity on these kinetochores. If elevated acetylation of Aurora B by TIP60 was necessary for chromosome bi-orientation, introduction of the acetylation-mimicking K215Q-Aurora B would partially reverse the chromosome-misalignment seen in TIP60-depleted cells. Indeed, expression of acetylation-mimicking K215Q-Aurora B, but not wild-type or non-acetylatable K215R, partially rescued the chromosome alignment in TIP60-depleted cells (Fig. 3g
and
Supplementary Fig. 6f). Thus, we conclude that Aurora B kinase activity was modulated by acetylation in mitosis and TIP60-dependent acetylation of Aurora B at the kinetochores promoted chromosome bi-orientation.K215 acetylation prevents PP2A-mediated inactivationSince inhibition of TIP60 led to dephosphorylation of pT232, an auto-phosphorylation site essential for Aurora B activity22 and a substrate of PP2A phosphatase23, we next determined whether acetylation of Lys215 promoted phosphorylation of Thr232 through antagonizing PP2A function. In TIP60-inhibited cells, the reduction of pT232 was partially reversed by treatment of okadaic acid (OA), which preferentially inhibits PP2A at lower concentration24, 25 (Fig. 4a). To determine if acK215 prevented PP2A-elicited dephosphorylation of pT232, we conducted an in vitro PP2A assay in which the K215Q mutant attenuated the PP2A-mediated dephosphorylation of Thr232 (Fig. 4b). To directly determine whether Lys215 acetylation antagonized PP2A-mediated dephosphorylation of Thr232, we took advantage of a recently developed, genetically encoding method to yield Lys215-acetylated Aurora B26, 27 and a truncation of INCENP (822–918) 28, 29 from E. coli (Fig. 4c; Supplementary Fig. 7a and b). Both wild-type Aurora B and acetylated Aurora B exhibited comparable Thr232 phosphorylation, but the acK215 antibody specifically recognized the latter (Fig. 4d). Although the Lys215-acetylated Aurora B showed no superior activity toward the wild-type Aurora B indicated as determined by in vitro kinase assay (Fig. 4e), Thr232 phosphorylation was largely preserved after incubation with PP2A phosphatase for 2 h (Fig. 4f), supporting the notion that Lys215 acetylation promoted Aurora B activity through antagonizing PP2A-mediated dephosphorylation.Furthermore, suppression of TIP60 with MG149 retained the Aurora B-PP2A interaction in vivo (Fig. 4g). Consistently, the K215Q mutant that mimicked acetylation also failed to bind PP2A, as determined with a co-immunoprecipitation assay (Fig. 4h). This acetylation-elicited perturbation of the Aurora B-PP2A interaction was confirmed by a pull-down assay in which FLAG-PR65 interacted with wild-type Aurora B rather than Lys215-acetylated Aurora B (Fig. 4i). Thus, the acetylation of Lys215 by TIP60 sustained Aurora B activity by protecting pT232 from PP2A-elicited dephosphorylation.CDK1 phosphorylates TIP60 to ensure accurate mitosisThe phosphorylation of TIP60 at Ser90 by CDK1 has been reported30–32. To assess the spatiotemporal dynamics of CDK1-elicited phosphorylation of TIP60 and delineate its relevance in mitosis, we generated a phospho-S90 specific antibody (pS90), which specifically recognized phosphorylated TIP60 both in vivo and in vitro (Supplementary Fig. 8a and b). We next determined the temporal dynamics of pS90-TIP60 levels relative to those of acK215-Aurora B during mitosis by collecting synchronized HeLa cells at indicated intervals after release from the G1/S phase for immunoprecipitation assays (Fig. 5a; Supplementary Fig. 8c). The temporal dynamics of pS90 were similar to that of cyclin B, suggesting that TIP60 was timely regulated by CDK1-Cyclin B1. Furthermore, pT232-Aurora B and pS7-CENP-A levels in cells correlated with acK215 variation, and were parallel to pS90 with a brief lag, suggesting that Aurora B activity was tightly regulated by TIP60-S90 phosphorylation and TIP60-dependent Aurora B-K215 acetylation. In nocodazole-treated cells, the levels of pS90 and acK215 were increased (Fig. 5b) but were diminished upon inhibition of CDK1 by Roscovitine (combined with MG132 to prevent Cyclin B degradation, and treatment was limited to 20 min to prevent mitotic exit33), suggesting that CDK1 kinase activity controlled TIP60-dependent modulation of Aurora B activity. Furthermore, immunofluorescence assay showed that CDK1 inhibition abolished the signal of pS90 at kinetochores in mitotic cells, indicating that pS90 was a substrate of CDK1 during mitosis (Fig. 5c and Supplementary Fig. 8d). In TIP60-depleted cells, either wild-type or S90D mutant of TIP60, but not the S90A mutant restored K215 acetylation and T232 phosphorylation of Aurora B, suggesting that robust TIP60 activation in mitosis required CDK1-dependent S90 phosphorylation (Fig. 5d). We also evaluated TIP60 acetyltransferase activity by an in vitro assay using 14C-labeled Ac-CoA as the acetyl donor. The acetyltransferase activity was increased by CDK1 phosphorylation but diminished after addition of NU9056, a specific inhibitor of TIP60 (Fig. 5e and Supplementary Fig. 8e), confirming a direct function of CDK1/cyclin B in promoting TIP60 activity via phosphorylation of S90. If elevated phosphorylation of TIP60 by CDK1 was necessary for accurate chromosome segregation, introduction of the phosphorylation-mimicking S90D-TIP60 should reverse the chromosome segregation errors seen in TIP60-suppressed cells. As predicted, expression of non-phosphorylatable S90A mutant failed to rescue Aurora B activity and failed to correct chromosome segregation errors (Fig. 5f and g), revealing a temporal coordination of the CDK1-TIP60-Aurora B signaling axis in linking cell cycle progression to maintenance of genomic stability. Additional experiments to address whether Mps1 would synergize with CDK1 in release of TIP60 from the kinetochores in the presence of MG132 demonstrated that simultaneous inhibition of CDK1 and Mps1 did not result in any additive effects on TIP60 delocalization (Supplementary Fig. 8f and 8g). Thus, we concluded that CDK1-elicited phosphorylation of TIP60 promoted correction of attachment errors in mitosis via acetylation and activation of Aurora B at the kinetochores.

TIP60 is essential for faithful chromosome segregationTo elucidate the function of TIP60 in mitosis, we employed two shRNAs to suppress endogenous TIP60 (Supplementary Results, Supplementary Fig. 1a and b) and examined the resulting phenotype using time-lapse microscopy (Fig. 1a; Supplementary Fig. 1c). Although the control cells progressed through mitosis normally, TIP60-depleted cells exhibited a high frequency of chromosome segregation defects, including anaphase lagging chromosome, chromosome misalignment, and chromosome bridges (Fig. 1b, c and d). The defects seen in the TIP60-deficient cells were largely rescued by expressing an RNAi-resistant, wild-type TIP60, but not an acetyltransferase-deficient form of TIP6010 (Supplementary Fig. 1d, e, f and g), demonstrating the role of TIP60 acetyltransferase activity in mitotic chromosome movements. To eliminate the possibility that the phenotype was a consequence of disruption of TIP60 function in the DNA damage response pathway, we introduced two small molecule inhibitors of TIP60 acetyltransferase (NU905611 and MG14912) into cultured cells immediately after mitotic entry. As expected, TIP60 acetyltransferase activity inhibition in mitosis resulted in increased rates of chromosome mis-segregation (Fig. 1e, f and g; Supplementary Fig. 1h). Thus, TIP60 acetyltransferase activity was essential for chromosome alignment and accurate metaphase-anaphase transition. The function of TIP60 in genomic stability was apparent, as suppression of TIP60 gave rise to micronuclei and polyploidy (Supplementary Fig. 1i, j and k).

TIP60 is essential for faithful chromosome segregation

To elucidate the function of TIP60 in mitosis, we employed two shRNAs to suppress endogenous TIP60 (Supplementary Results, Supplementary Fig. 1a and b) and examined the resulting phenotype using time-lapse microscopy (Fig. 1a; Supplementary Fig. 1c). Although the control cells progressed through mitosis normally, TIP60-depleted cells exhibited a high frequency of chromosome segregation defects, including anaphase lagging chromosome, chromosome misalignment, and chromosome bridges (Fig. 1b, c and d). The defects seen in the TIP60-deficient cells were largely rescued by expressing an RNAi-resistant, wild-type TIP60, but not an acetyltransferase-deficient form of TIP6010 (Supplementary Fig. 1d, e, f and g), demonstrating the role of TIP60 acetyltransferase activity in mitotic chromosome movements. To eliminate the possibility that the phenotype was a consequence of disruption of TIP60 function in the DNA damage response pathway, we introduced two small molecule inhibitors of TIP60 acetyltransferase (NU905611 and MG14912) into cultured cells immediately after mitotic entry. As expected, TIP60 acetyltransferase activity inhibition in mitosis resulted in increased rates of chromosome mis-segregation (Fig. 1e, f and g; Supplementary Fig. 1h). Thus, TIP60 acetyltransferase activity was essential for chromosome alignment and accurate metaphase-anaphase transition. The function of TIP60 in genomic stability was apparent, as suppression of TIP60 gave rise to micronuclei and polyploidy (Supplementary Fig. 1i, j and k).

TIP60 modulates Aurora B kinase activity at kinetochoresTo delineate the mechanism of action underlying TIP60 function in mitosis, we examined the subcellular distribution of TIP60 in mitosis. Consistent with its function, TIP60 localized to kinetochores in early mitosis, as determined by two different TIP60 antibodies (Fig. 2a; Supplementary Fig. 2a). The localization of TIP60 to kinetochores was independent of microtubules, as nocodazole treatment did not alter its localization (Supplementary Fig. 2b). In a line-scan, the fluorescence intensity profiles suggested that TIP60 co-localized with Hec1 at the outer kinetochores (Fig. 2b and Supplementary Fig. 2c). The kinetochore localization of TIP60 was abolished by shRNA-mediated suppression, confirming the specificity of the antibody (Supplementary Fig. 2d). Furthermore, exogenously expressed TIP60-GFP also localized to the kinetochores in prometaphase cells (Supplementary Fig. 2e), confirming that TIP60 was a kinetochore protein.The localization of TIP60 prompted a search, with a battery of kinase inhibitors, for the molecular determinants underlying its localization, the results indicated that a stable localization of TIP60 to the kinetochores specifically required Mps1 kinase activity (Supplementary Fig. 3a, b and c). The Mps1-dependence was confirmed by siRNA treatment in which suppression of Mps1 quantitatively reduced the levels of TIP60 at the kinetochores (Supplementary Fig. 3d). We and two other groups have recently found that Mps1 is recruited to kinetochores by binding to two distinct areas of the Ndc80 complex13–15. To determine whether kinetochore recruitment of TIP60 required adequate Mps1 kinase at kinetochores, we knocked down two components of the Ndc80 complex, Hec1 and Nuf2. The absence of either Nuf2 or Hec1 abolished kinetochore localization of TIP60 (Supplementary Fig. 3e), confirming that Ncd80/Mps1 pathway regulated TIP60 distribution to kinetochore16.Since TIP60 transiently localized to kinetochores in early mitosis when chromosomes attempted for bi-orientation, we questioned whether TIP60 distinguished amphitelic attachment from erroneously attached kinetochores. To this end, we treated HeLa cells with the mitotic kinesin CENP-E inhibitor GSK923295 (IC50 = 3.2 nM)17 to create erroneous kinetochore-microtubule attachments. TIP60 preferentially accumulated to erroneously attached kinetochores near the spindle poles but not to those with apparent amphitelic attachments (Supplementary Fig. 3f). Quantitative analyses of normalized pixel intensities indicated that the level of kinetochore-associated TIP60 was a function of chromosome position to the poles (Fig. 2c). The function of TIP60 in chromosome alignment was further assessed. In a monastrol-washout assay, bi-orientation was achieved only after either of the TIP60 inhibitors was removed (Supplementary Fig. 4a). Consistently, TIP60-depleted cells also failed to undergo accurate chromosome segregation, as lagging chromosomes were frequently observed (Supplementary Fig. 4b), confirming that achievement of chromosome bi-orientation required TIP60 activity. Since Aurora B kinase is a key factor in correcting erroneous attachments18, 19, we sought to examine if the function of TIP60 in promoting accurate kinetochore attachment was mediated via Aurora B signaling. To this end, Aurora B was depleted via siRNA treatment. Although depletion of Aurora B caused a more severe phenotype than that of TIP60 depletion, there was no additive effect when combined suppressing Aurora B and TIP60 (Supplementary Fig. 4c and d), revealing functional relevance between Aurora B and TIP60. Since an active form of Aurora B (Thr232 phosphorylated) localizes to kinetochores20, where it co-localized with TIP60 (Fig. 2d), we further examined the intensity of Thr232-phosphorylated Aurora B in TIP60-depleted cells (Fig. 2e). Quantitative analysis showed that the level of pT232 in TIP60 siRNA-treated cells reduced to 33% of its control value while average Aurora B protein levels exhibited no significant alternation (Supplementary Fig. 4e and f), suggesting that TIP60 regulated activity of kinetochore-bound Aurora B.To probe if Aurora B kinase activity was modulated by TIP60-mediated acetylation, we examined Aurora B activity using phosphorylation of CENP-A at Ser7 (pS7-CENP-A) as a reporter21. Suppressing TIP60 acetyltransferase activity by either TIP60 inhibitors (Fig. 2f) or TIP60 shRNA (Supplementary Fig. 4g and h) reduced CENP-A phosphorylation, suggesting that TIP60-mediated acetylation modulated Aurora B kinase activity in mitotic cells. The effect of TIP60 inhibition on Aurora B activation, together with the co-localization of phosphorylated Aurora B and TIP60, prompted us to determine whether TIP60 formed a complex with Aurora B in vivo. In mitosis, gel filtration assays and reciprocal co-immunoprecipitation confirmed that TIP60 formed a complex with Aurora B (Supplementary Fig. 5a, b and c). Thus, we conclude that TIP60 acetyltransferase modulated Aurora B kinase activity in mitosis.

TIP60 modulates Aurora B kinase activity at kinetochores

To delineate the mechanism of action underlying TIP60 function in mitosis, we examined the subcellular distribution of TIP60 in mitosis. Consistent with its function, TIP60 localized to kinetochores in early mitosis, as determined by two different TIP60 antibodies (Fig. 2a; Supplementary Fig. 2a). The localization of TIP60 to kinetochores was independent of microtubules, as nocodazole treatment did not alter its localization (Supplementary Fig. 2b). In a line-scan, the fluorescence intensity profiles suggested that TIP60 co-localized with Hec1 at the outer kinetochores (Fig. 2b and Supplementary Fig. 2c). The kinetochore localization of TIP60 was abolished by shRNA-mediated suppression, confirming the specificity of the antibody (Supplementary Fig. 2d). Furthermore, exogenously expressed TIP60-GFP also localized to the kinetochores in prometaphase cells (Supplementary Fig. 2e), confirming that TIP60 was a kinetochore protein.

The localization of TIP60 prompted a search, with a battery of kinase inhibitors, for the molecular determinants underlying its localization, the results indicated that a stable localization of TIP60 to the kinetochores specifically required Mps1 kinase activity (Supplementary Fig. 3a, b and c). The Mps1-dependence was confirmed by siRNA treatment in which suppression of Mps1 quantitatively reduced the levels of TIP60 at the kinetochores (Supplementary Fig. 3d). We and two other groups have recently found that Mps1 is recruited to kinetochores by binding to two distinct areas of the Ndc80 complex13–15. To determine whether kinetochore recruitment of TIP60 required adequate Mps1 kinase at kinetochores, we knocked down two components of the Ndc80 complex, Hec1 and Nuf2. The absence of either Nuf2 or Hec1 abolished kinetochore localization of TIP60 (Supplementary Fig. 3e), confirming that Ncd80/Mps1 pathway regulated TIP60 distribution to kinetochore16.

Since TIP60 transiently localized to kinetochores in early mitosis when chromosomes attempted for bi-orientation, we questioned whether TIP60 distinguished amphitelic attachment from erroneously attached kinetochores. To this end, we treated HeLa cells with the mitotic kinesin CENP-E inhibitor GSK923295 (IC50 = 3.2 nM)17 to create erroneous kinetochore-microtubule attachments. TIP60 preferentially accumulated to erroneously attached kinetochores near the spindle poles but not to those with apparent amphitelic attachments (Supplementary Fig. 3f). Quantitative analyses of normalized pixel intensities indicated that the level of kinetochore-associated TIP60 was a function of chromosome position to the poles (Fig. 2c). The function of TIP60 in chromosome alignment was further assessed. In a monastrol-washout assay, bi-orientation was achieved only after either of the TIP60 inhibitors was removed (Supplementary Fig. 4a). Consistently, TIP60-depleted cells also failed to undergo accurate chromosome segregation, as lagging chromosomes were frequently observed (Supplementary Fig. 4b), confirming that achievement of chromosome bi-orientation required TIP60 activity. Since Aurora B kinase is a key factor in correcting erroneous attachments18, 19, we sought to examine if the function of TIP60 in promoting accurate kinetochore attachment was mediated via Aurora B signaling. To this end, Aurora B was depleted via siRNA treatment. Although depletion of Aurora B caused a more severe phenotype than that of TIP60 depletion, there was no additive effect when combined suppressing Aurora B and TIP60 (Supplementary Fig. 4c and d), revealing functional relevance between Aurora B and TIP60. Since an active form of Aurora B (Thr232 phosphorylated) localizes to kinetochores20, where it co-localized with TIP60 (Fig. 2d), we further examined the intensity of Thr232-phosphorylated Aurora B in TIP60-depleted cells (Fig. 2e). Quantitative analysis showed that the level of pT232 in TIP60 siRNA-treated cells reduced to 33% of its control value while average Aurora B protein levels exhibited no significant alternation (Supplementary Fig. 4e and f), suggesting that TIP60 regulated activity of kinetochore-bound Aurora B.

To probe if Aurora B kinase activity was modulated by TIP60-mediated acetylation, we examined Aurora B activity using phosphorylation of CENP-A at Ser7 (pS7-CENP-A) as a reporter21. Suppressing TIP60 acetyltransferase activity by either TIP60 inhibitors (Fig. 2f) or TIP60 shRNA (Supplementary Fig. 4g and h) reduced CENP-A phosphorylation, suggesting that TIP60-mediated acetylation modulated Aurora B kinase activity in mitotic cells. The effect of TIP60 inhibition on Aurora B activation, together with the co-localization of phosphorylated Aurora B and TIP60, prompted us to determine whether TIP60 formed a complex with Aurora B in vivo. In mitosis, gel filtration assays and reciprocal co-immunoprecipitation confirmed that TIP60 formed a complex with Aurora B (Supplementary Fig. 5a, b and c). Thus, we conclude that TIP60 acetyltransferase modulated Aurora B kinase activity in mitosis.

K215 acetylation by TIP60 promotes Aurora B activityWe next determined if TIP60 directly acetylated Aurora B. Western blots for anti-pan-acetyl-lysine (acK) established that Aurora B was a substrate of TIP60 in vitro (Fig. 3a). To identify the physiological substrate sites of TIP60 in Aurora B, purified Aurora B from mitotic HeLa cells was subjected to mass spectrometric analyses which revealed 4 potential acetylation sites, including Lys31, Lys115, Lys215, and Lys231 (Supplementary Fig. 5d). To map the site(s) responsible for TIP60 acetylation, we generated a series of Aurora B mutants, in which the identified acetylation sites were individually mutated to arginine. In the Aurora B Lys215 mutant immunoprecipitated from mitotic cells, Aurora B acetylation was virtually undetectable (Supplementary Fig. 5e), suggesting that acK215 accounted for Aurora B acetylation in mitosis. To characterize the precise cellular function of Lys215 acetylation in mitosis, we generated a Lys215 acetylated Aurora B antibody (acK215-AurB), which specifically recognized both endogenous acetylated Aurora B (Supplementary Fig. 5f and g) and TIP60 acetylated Aurora B in vitro (Supplementary Fig. 5h). To assess whether Aurora B was a cognate substrate of TIP60 in mitosis, we analyzed K215 acetylation in aliquots of asynchronized or nocodazole-synchronized HeLa cells. Lys215 acetylation level was dramatically increased in nocodazole-synchronized cells but reduced after TIP60 inhibition (Supplementary Fig. 5i), indicating that K215 was a cognate substrate of TIP60 in mitotic cells. To examine the precise cellular function of K215 acetylation by TIP60, we knocked-down endogenous TIP60 and assessed the activity of Aurora B by in vitro kinase assays. Aurora B kinase activity was alleviated in TIP60 depleted cells, consistent with the reduced K215 acetylation level (Fig. 3b), indicating that both Lys215 acetylation and kinase activity of Aurora B required TIP60.Since Lys215 is highly conserved among species and locates close to the activation loop of Aurora B (Supplementary Fig. 5j), we hypothesized that TIP60 promoted Aurora B activity through acetylating Aurora B at Lys215. To confirm the hypothesis, we purified FLAG-tagged wild-type (WT), acetylation-mimicking (K215Q) and non-acetylatable (K215R) mutants from mitotic cells for in vitro kinase assay and enzymatic characterization. As reflected by H3 phosphorylation, TIP60 mediated K215 acetylation robustly stimulated Aurora B activity (Fig. 3c and d; Supplementary Fig. 5k).

K215 acetylation by TIP60 promotes Aurora B activity

We next determined if TIP60 directly acetylated Aurora B. Western blots for anti-pan-acetyl-lysine (acK) established that Aurora B was a substrate of TIP60 in vitro (Fig. 3a). To identify the physiological substrate sites of TIP60 in Aurora B, purified Aurora B from mitotic HeLa cells was subjected to mass spectrometric analyses which revealed 4 potential acetylation sites, including Lys31, Lys115, Lys215, and Lys231 (Supplementary Fig. 5d). To map the site(s) responsible for TIP60 acetylation, we generated a series of Aurora B mutants, in which the identified acetylation sites were individually mutated to arginine. In the Aurora B Lys215 mutant immunoprecipitated from mitotic cells, Aurora B acetylation was virtually undetectable (Supplementary Fig. 5e), suggesting that acK215 accounted for Aurora B acetylation in mitosis. To characterize the precise cellular function of Lys215 acetylation in mitosis, we generated a Lys215 acetylated Aurora B antibody (acK215-AurB), which specifically recognized both endogenous acetylated Aurora B (Supplementary Fig. 5f and g) and TIP60 acetylated Aurora B in vitro (Supplementary Fig. 5h). To assess whether Aurora B was a cognate substrate of TIP60 in mitosis, we analyzed K215 acetylation in aliquots of asynchronized or nocodazole-synchronized HeLa cells. Lys215 acetylation level was dramatically increased in nocodazole-synchronized cells but reduced after TIP60 inhibition (Supplementary Fig. 5i), indicating that K215 was a cognate substrate of TIP60 in mitotic cells. To examine the precise cellular function of K215 acetylation by TIP60, we knocked-down endogenous TIP60 and assessed the activity of Aurora B by in vitro kinase assays. Aurora B kinase activity was alleviated in TIP60 depleted cells, consistent with the reduced K215 acetylation level (Fig. 3b), indicating that both Lys215 acetylation and kinase activity of Aurora B required TIP60.

Since Lys215 is highly conserved among species and locates close to the activation loop of Aurora B (Supplementary Fig. 5j), we hypothesized that TIP60 promoted Aurora B activity through acetylating Aurora B at Lys215. To confirm the hypothesis, we purified FLAG-tagged wild-type (WT), acetylation-mimicking (K215Q) and non-acetylatable (K215R) mutants from mitotic cells for in vitro kinase assay and enzymatic characterization. As reflected by H3 phosphorylation, TIP60 mediated K215 acetylation robustly stimulated Aurora B activity (Fig. 3c and d; Supplementary Fig. 5k).

K215 acetylation promotes chromosome bi-orientationSince TIP60 distinguished bi-oriented kinetochores from erroneously attached kinetochores, we next tested if alteration of the kinetochore-microtubule attachment modulated distribution of acetylated Aurora B at the kinetochores. AcK215 staining was apparent in prometaphase when bi-orientation was not achieved in most kinetochores, whereas the staining was diminished in metaphase when most kinetochores were properly attached to microtubules (Fig. 3e; Supplementary Fig. 6a).If acetylation of Lys215 was involved in correction of aberrant kinetochore attachments, the level of acK215 would increase in the misaligned kinetochores. To test this hypothesis, we exposed HeLa cells to GSK923295 to create both bi-oriented and misaligned kinetochores in the same cell, in which the labeling of acK215 was brighter on the misaligned kinetochores (Fig. 3f). Statistical analysis indicated that acK215 levels of mis-aligned kinetochores were about 6-fold higher than those near the equator (Supplementary Fig. 6a). Since pT232 is a reporter of Aurora B activity, we also examined the levels of pT232 in GSK923295-treated cells. In these cells, the levels of pT232 were elevated at the kinetochores near the pole relative to kinetochores near the equator (Supplementary Fig. 6b). Statistical analyses showed that the pT232-AurB levels of misaligned kinetochores were much higher than those near the equator (Supplementary Fig. 6c), supporting the notion that acK215 promoted Aurora B activity at mis-attached and unattached kinetochores. Intriguingly, a brief treatment with the TIP60 inhibitor NU9056 abolished the labeling of acK215 and pT232 at the misaligned kinetochores (Supplementary Fig. 6d and e), indicating that TIP60 regulated Aurora B acetylation and activity on these kinetochores. If elevated acetylation of Aurora B by TIP60 was necessary for chromosome bi-orientation, introduction of the acetylation-mimicking K215Q-Aurora B would partially reverse the chromosome-misalignment seen in TIP60-depleted cells. Indeed, expression of acetylation-mimicking K215Q-Aurora B, but not wild-type or non-acetylatable K215R, partially rescued the chromosome alignment in TIP60-depleted cells (Fig. 3g
and
Supplementary Fig. 6f). Thus, we conclude that Aurora B kinase activity was modulated by acetylation in mitosis and TIP60-dependent acetylation of Aurora B at the kinetochores promoted chromosome bi-orientation.

K215 acetylation promotes chromosome bi-orientation

Since TIP60 distinguished bi-oriented kinetochores from erroneously attached kinetochores, we next tested if alteration of the kinetochore-microtubule attachment modulated distribution of acetylated Aurora B at the kinetochores. AcK215 staining was apparent in prometaphase when bi-orientation was not achieved in most kinetochores, whereas the staining was diminished in metaphase when most kinetochores were properly attached to microtubules (Fig. 3e; Supplementary Fig. 6a).

If acetylation of Lys215 was involved in correction of aberrant kinetochore attachments, the level of acK215 would increase in the misaligned kinetochores. To test this hypothesis, we exposed HeLa cells to GSK923295 to create both bi-oriented and misaligned kinetochores in the same cell, in which the labeling of acK215 was brighter on the misaligned kinetochores (Fig. 3f). Statistical analysis indicated that acK215 levels of mis-aligned kinetochores were about 6-fold higher than those near the equator (Supplementary Fig. 6a). Since pT232 is a reporter of Aurora B activity, we also examined the levels of pT232 in GSK923295-treated cells. In these cells, the levels of pT232 were elevated at the kinetochores near the pole relative to kinetochores near the equator (Supplementary Fig. 6b). Statistical analyses showed that the pT232-AurB levels of misaligned kinetochores were much higher than those near the equator (Supplementary Fig. 6c), supporting the notion that acK215 promoted Aurora B activity at mis-attached and unattached kinetochores. Intriguingly, a brief treatment with the TIP60 inhibitor NU9056 abolished the labeling of acK215 and pT232 at the misaligned kinetochores (Supplementary Fig. 6d and e), indicating that TIP60 regulated Aurora B acetylation and activity on these kinetochores. If elevated acetylation of Aurora B by TIP60 was necessary for chromosome bi-orientation, introduction of the acetylation-mimicking K215Q-Aurora B would partially reverse the chromosome-misalignment seen in TIP60-depleted cells. Indeed, expression of acetylation-mimicking K215Q-Aurora B, but not wild-type or non-acetylatable K215R, partially rescued the chromosome alignment in TIP60-depleted cells (Fig. 3g
and
Supplementary Fig. 6f). Thus, we conclude that Aurora B kinase activity was modulated by acetylation in mitosis and TIP60-dependent acetylation of Aurora B at the kinetochores promoted chromosome bi-orientation.

K215 acetylation prevents PP2A-mediated inactivationSince inhibition of TIP60 led to dephosphorylation of pT232, an auto-phosphorylation site essential for Aurora B activity22 and a substrate of PP2A phosphatase23, we next determined whether acetylation of Lys215 promoted phosphorylation of Thr232 through antagonizing PP2A function. In TIP60-inhibited cells, the reduction of pT232 was partially reversed by treatment of okadaic acid (OA), which preferentially inhibits PP2A at lower concentration24, 25 (Fig. 4a). To determine if acK215 prevented PP2A-elicited dephosphorylation of pT232, we conducted an in vitro PP2A assay in which the K215Q mutant attenuated the PP2A-mediated dephosphorylation of Thr232 (Fig. 4b). To directly determine whether Lys215 acetylation antagonized PP2A-mediated dephosphorylation of Thr232, we took advantage of a recently developed, genetically encoding method to yield Lys215-acetylated Aurora B26, 27 and a truncation of INCENP (822–918) 28, 29 from E. coli (Fig. 4c; Supplementary Fig. 7a and b). Both wild-type Aurora B and acetylated Aurora B exhibited comparable Thr232 phosphorylation, but the acK215 antibody specifically recognized the latter (Fig. 4d). Although the Lys215-acetylated Aurora B showed no superior activity toward the wild-type Aurora B indicated as determined by in vitro kinase assay (Fig. 4e), Thr232 phosphorylation was largely preserved after incubation with PP2A phosphatase for 2 h (Fig. 4f), supporting the notion that Lys215 acetylation promoted Aurora B activity through antagonizing PP2A-mediated dephosphorylation.Furthermore, suppression of TIP60 with MG149 retained the Aurora B-PP2A interaction in vivo (Fig. 4g). Consistently, the K215Q mutant that mimicked acetylation also failed to bind PP2A, as determined with a co-immunoprecipitation assay (Fig. 4h). This acetylation-elicited perturbation of the Aurora B-PP2A interaction was confirmed by a pull-down assay in which FLAG-PR65 interacted with wild-type Aurora B rather than Lys215-acetylated Aurora B (Fig. 4i). Thus, the acetylation of Lys215 by TIP60 sustained Aurora B activity by protecting pT232 from PP2A-elicited dephosphorylation.

K215 acetylation prevents PP2A-mediated inactivation

Since inhibition of TIP60 led to dephosphorylation of pT232, an auto-phosphorylation site essential for Aurora B activity22 and a substrate of PP2A phosphatase23, we next determined whether acetylation of Lys215 promoted phosphorylation of Thr232 through antagonizing PP2A function. In TIP60-inhibited cells, the reduction of pT232 was partially reversed by treatment of okadaic acid (OA), which preferentially inhibits PP2A at lower concentration24, 25 (Fig. 4a). To determine if acK215 prevented PP2A-elicited dephosphorylation of pT232, we conducted an in vitro PP2A assay in which the K215Q mutant attenuated the PP2A-mediated dephosphorylation of Thr232 (Fig. 4b). To directly determine whether Lys215 acetylation antagonized PP2A-mediated dephosphorylation of Thr232, we took advantage of a recently developed, genetically encoding method to yield Lys215-acetylated Aurora B26, 27 and a truncation of INCENP (822–918) 28, 29 from E. coli (Fig. 4c; Supplementary Fig. 7a and b). Both wild-type Aurora B and acetylated Aurora B exhibited comparable Thr232 phosphorylation, but the acK215 antibody specifically recognized the latter (Fig. 4d). Although the Lys215-acetylated Aurora B showed no superior activity toward the wild-type Aurora B indicated as determined by in vitro kinase assay (Fig. 4e), Thr232 phosphorylation was largely preserved after incubation with PP2A phosphatase for 2 h (Fig. 4f), supporting the notion that Lys215 acetylation promoted Aurora B activity through antagonizing PP2A-mediated dephosphorylation.

Furthermore, suppression of TIP60 with MG149 retained the Aurora B-PP2A interaction in vivo (Fig. 4g). Consistently, the K215Q mutant that mimicked acetylation also failed to bind PP2A, as determined with a co-immunoprecipitation assay (Fig. 4h). This acetylation-elicited perturbation of the Aurora B-PP2A interaction was confirmed by a pull-down assay in which FLAG-PR65 interacted with wild-type Aurora B rather than Lys215-acetylated Aurora B (Fig. 4i). Thus, the acetylation of Lys215 by TIP60 sustained Aurora B activity by protecting pT232 from PP2A-elicited dephosphorylation.

CDK1 phosphorylates TIP60 to ensure accurate mitosisThe phosphorylation of TIP60 at Ser90 by CDK1 has been reported30–32. To assess the spatiotemporal dynamics of CDK1-elicited phosphorylation of TIP60 and delineate its relevance in mitosis, we generated a phospho-S90 specific antibody (pS90), which specifically recognized phosphorylated TIP60 both in vivo and in vitro (Supplementary Fig. 8a and b). We next determined the temporal dynamics of pS90-TIP60 levels relative to those of acK215-Aurora B during mitosis by collecting synchronized HeLa cells at indicated intervals after release from the G1/S phase for immunoprecipitation assays (Fig. 5a; Supplementary Fig. 8c). The temporal dynamics of pS90 were similar to that of cyclin B, suggesting that TIP60 was timely regulated by CDK1-Cyclin B1. Furthermore, pT232-Aurora B and pS7-CENP-A levels in cells correlated with acK215 variation, and were parallel to pS90 with a brief lag, suggesting that Aurora B activity was tightly regulated by TIP60-S90 phosphorylation and TIP60-dependent Aurora B-K215 acetylation. In nocodazole-treated cells, the levels of pS90 and acK215 were increased (Fig. 5b) but were diminished upon inhibition of CDK1 by Roscovitine (combined with MG132 to prevent Cyclin B degradation, and treatment was limited to 20 min to prevent mitotic exit33), suggesting that CDK1 kinase activity controlled TIP60-dependent modulation of Aurora B activity. Furthermore, immunofluorescence assay showed that CDK1 inhibition abolished the signal of pS90 at kinetochores in mitotic cells, indicating that pS90 was a substrate of CDK1 during mitosis (Fig. 5c and Supplementary Fig. 8d). In TIP60-depleted cells, either wild-type or S90D mutant of TIP60, but not the S90A mutant restored K215 acetylation and T232 phosphorylation of Aurora B, suggesting that robust TIP60 activation in mitosis required CDK1-dependent S90 phosphorylation (Fig. 5d). We also evaluated TIP60 acetyltransferase activity by an in vitro assay using 14C-labeled Ac-CoA as the acetyl donor. The acetyltransferase activity was increased by CDK1 phosphorylation but diminished after addition of NU9056, a specific inhibitor of TIP60 (Fig. 5e and Supplementary Fig. 8e), confirming a direct function of CDK1/cyclin B in promoting TIP60 activity via phosphorylation of S90. If elevated phosphorylation of TIP60 by CDK1 was necessary for accurate chromosome segregation, introduction of the phosphorylation-mimicking S90D-TIP60 should reverse the chromosome segregation errors seen in TIP60-suppressed cells. As predicted, expression of non-phosphorylatable S90A mutant failed to rescue Aurora B activity and failed to correct chromosome segregation errors (Fig. 5f and g), revealing a temporal coordination of the CDK1-TIP60-Aurora B signaling axis in linking cell cycle progression to maintenance of genomic stability. Additional experiments to address whether Mps1 would synergize with CDK1 in release of TIP60 from the kinetochores in the presence of MG132 demonstrated that simultaneous inhibition of CDK1 and Mps1 did not result in any additive effects on TIP60 delocalization (Supplementary Fig. 8f and 8g). Thus, we concluded that CDK1-elicited phosphorylation of TIP60 promoted correction of attachment errors in mitosis via acetylation and activation of Aurora B at the kinetochores.

CDK1 phosphorylates TIP60 to ensure accurate mitosis

The phosphorylation of TIP60 at Ser90 by CDK1 has been reported30–32. To assess the spatiotemporal dynamics of CDK1-elicited phosphorylation of TIP60 and delineate its relevance in mitosis, we generated a phospho-S90 specific antibody (pS90), which specifically recognized phosphorylated TIP60 both in vivo and in vitro (Supplementary Fig. 8a and b). We next determined the temporal dynamics of pS90-TIP60 levels relative to those of acK215-Aurora B during mitosis by collecting synchronized HeLa cells at indicated intervals after release from the G1/S phase for immunoprecipitation assays (Fig. 5a; Supplementary Fig. 8c). The temporal dynamics of pS90 were similar to that of cyclin B, suggesting that TIP60 was timely regulated by CDK1-Cyclin B1. Furthermore, pT232-Aurora B and pS7-CENP-A levels in cells correlated with acK215 variation, and were parallel to pS90 with a brief lag, suggesting that Aurora B activity was tightly regulated by TIP60-S90 phosphorylation and TIP60-dependent Aurora B-K215 acetylation. In nocodazole-treated cells, the levels of pS90 and acK215 were increased (Fig. 5b) but were diminished upon inhibition of CDK1 by Roscovitine (combined with MG132 to prevent Cyclin B degradation, and treatment was limited to 20 min to prevent mitotic exit33), suggesting that CDK1 kinase activity controlled TIP60-dependent modulation of Aurora B activity. Furthermore, immunofluorescence assay showed that CDK1 inhibition abolished the signal of pS90 at kinetochores in mitotic cells, indicating that pS90 was a substrate of CDK1 during mitosis (Fig. 5c and Supplementary Fig. 8d). In TIP60-depleted cells, either wild-type or S90D mutant of TIP60, but not the S90A mutant restored K215 acetylation and T232 phosphorylation of Aurora B, suggesting that robust TIP60 activation in mitosis required CDK1-dependent S90 phosphorylation (Fig. 5d). We also evaluated TIP60 acetyltransferase activity by an in vitro assay using 14C-labeled Ac-CoA as the acetyl donor. The acetyltransferase activity was increased by CDK1 phosphorylation but diminished after addition of NU9056, a specific inhibitor of TIP60 (Fig. 5e and Supplementary Fig. 8e), confirming a direct function of CDK1/cyclin B in promoting TIP60 activity via phosphorylation of S90. If elevated phosphorylation of TIP60 by CDK1 was necessary for accurate chromosome segregation, introduction of the phosphorylation-mimicking S90D-TIP60 should reverse the chromosome segregation errors seen in TIP60-suppressed cells. As predicted, expression of non-phosphorylatable S90A mutant failed to rescue Aurora B activity and failed to correct chromosome segregation errors (Fig. 5f and g), revealing a temporal coordination of the CDK1-TIP60-Aurora B signaling axis in linking cell cycle progression to maintenance of genomic stability. Additional experiments to address whether Mps1 would synergize with CDK1 in release of TIP60 from the kinetochores in the presence of MG132 demonstrated that simultaneous inhibition of CDK1 and Mps1 did not result in any additive effects on TIP60 delocalization (Supplementary Fig. 8f and 8g). Thus, we concluded that CDK1-elicited phosphorylation of TIP60 promoted correction of attachment errors in mitosis via acetylation and activation of Aurora B at the kinetochores.

DISCUSSIONKinetochores are specialized protein machines involved in maintaining genome stability during mitosis via orchestrating accurate chromosome-microtubule attachments and SAC34. CDK1-cyclin B inhibits PP1 activity in early mitosis, and reduction of cyclin B levels later in mitosis permits PP1 auto-reactivation, which reactivates PP2A for subsequent dephosphorylation of Aurora B at Thr23235. Acetylation of Aurora B by TIP60 therefore provided full activity of Aurora B for optimal correction of errors in kinetochore attachment prior to sister chromatid separation. Our identification of the CDK1-TIP60-Aurora B signaling axis uncovered a new regulatory mechanism by which acetylation of Aurora B prevented PP2A mediated dephosphorylation of T-loop to enhance the activity of Aurora B for control of genomic stability (Fig. 5h). It would be of great interest, in follow-up studies, to characterize additional substrates of TIP60 in mitotic cells and delineate their precise molecular function in mitosis.We identified a mechanism involving the CDK1-TIP60-Aurora B axis that underlain kinetochore sensing of aberrant microtubule attachment and established its connection to error-free metaphase-anaphase transition, as perturbation of the axis gave rise to polyploidy or micronuclei, characteristics of genomic instability36, which highlighted how an acetylation-phosphorylation cascade in sensing and correction of chromosome attachment errors during mitotic progression maintained genomic stability. Alterations in chromatin promote TIP60 binding to chromatin and the concurrent accumulation of tyrosine phosphorylation of TIP60, which in turn induces ATM-mediated signaling and DNA damage checkpoint activation37. Of note, control of centromere plasticity shares common machinery involving the DNA damage response (DDR) such as the MHF/CENP-X complex38–39. Indeed, chromosome segregation errors cause DNA damage and structural chromosome aberrations40. Our present study delineated the cellular function of TIP60 in mitosis but not earlier events by which CDK1-elicited activation of TIP60 orchestrated acetylation of Aurora B at the kinetochores for accurate chromosome alignment and segregation. These results explained how TIP60 serves as a genome guardian via orchestrating chromatin plasticity during DDR processes and generating an optimal Aurora B activity for kinetochore-attachments and error-free metaphase-anaphase transitions in mitosis (Fig. 5h).Together with our findings that PCAF-mediated acetylation controls microtubule plus-end dynamics41 and synergism between TIP60 and PCAF42, these studies provided a unifying view of a previously uncharacterized molecular mechanism that underlain acetyl-regulation in mitosis and defined a signaling axis that integrated protein phosphorylation and acetylation to connect cell cycle progression with genomic stability.

Kinetochores are specialized protein machines involved in maintaining genome stability during mitosis via orchestrating accurate chromosome-microtubule attachments and SAC34. CDK1-cyclin B inhibits PP1 activity in early mitosis, and reduction of cyclin B levels later in mitosis permits PP1 auto-reactivation, which reactivates PP2A for subsequent dephosphorylation of Aurora B at Thr23235. Acetylation of Aurora B by TIP60 therefore provided full activity of Aurora B for optimal correction of errors in kinetochore attachment prior to sister chromatid separation. Our identification of the CDK1-TIP60-Aurora B signaling axis uncovered a new regulatory mechanism by which acetylation of Aurora B prevented PP2A mediated dephosphorylation of T-loop to enhance the activity of Aurora B for control of genomic stability (Fig. 5h). It would be of great interest, in follow-up studies, to characterize additional substrates of TIP60 in mitotic cells and delineate their precise molecular function in mitosis.

We identified a mechanism involving the CDK1-TIP60-Aurora B axis that underlain kinetochore sensing of aberrant microtubule attachment and established its connection to error-free metaphase-anaphase transition, as perturbation of the axis gave rise to polyploidy or micronuclei, characteristics of genomic instability36, which highlighted how an acetylation-phosphorylation cascade in sensing and correction of chromosome attachment errors during mitotic progression maintained genomic stability. Alterations in chromatin promote TIP60 binding to chromatin and the concurrent accumulation of tyrosine phosphorylation of TIP60, which in turn induces ATM-mediated signaling and DNA damage checkpoint activation37. Of note, control of centromere plasticity shares common machinery involving the DNA damage response (DDR) such as the MHF/CENP-X complex38–39. Indeed, chromosome segregation errors cause DNA damage and structural chromosome aberrations40. Our present study delineated the cellular function of TIP60 in mitosis but not earlier events by which CDK1-elicited activation of TIP60 orchestrated acetylation of Aurora B at the kinetochores for accurate chromosome alignment and segregation. These results explained how TIP60 serves as a genome guardian via orchestrating chromatin plasticity during DDR processes and generating an optimal Aurora B activity for kinetochore-attachments and error-free metaphase-anaphase transitions in mitosis (Fig. 5h).

Together with our findings that PCAF-mediated acetylation controls microtubule plus-end dynamics41 and synergism between TIP60 and PCAF42, these studies provided a unifying view of a previously uncharacterized molecular mechanism that underlain acetyl-regulation in mitosis and defined a signaling axis that integrated protein phosphorylation and acetylation to connect cell cycle progression with genomic stability.

ONLINE METHODSPlasmidsSite-specific mutants of FLAG- or EGFP-tagged Aurora B and RNAi-resistant TIP60 were generated by PCR-based, site-directed mutagenesis kit from Vazyme (C212) according to the manufacturer’s instructions. MBP-Aurora B-WT and MBP-Aurora B-K215R plasmids were generated by sub-cloning Aurora B from corresponding EGFP vectors into the pMal-c2 vector. The GST-H3 N-tail construct was obtained by cloning a PCR fragment corresponding to aa 1–15 of human histone H3 into the pGEX-6p1 vector. ACKRS3 and pCDF-pylT plasmids were gifts from the laboratory of Jason Chin. pAB-HIS-INbox was constructed by inserting full-length Aurora B and a truncation of INCENP (aa 822–918) downstream of two adjacent T7 promoters in pCDF-pylT. The corresponding AAG codon in pAB-HIS-INbox was mutated to TAG in order to generate pAB-215TAG-HIS-INbox. All plasmids used were verified by sequencing (Invitrogen).Cell culture, synchronization, and transfectionHEK293T and HeLa cells were purchased from the American Type Culture Collection and maintained as monolayers in advanced DMEM (Invitrogen) with 10% fetal calf serum (HyClone) and 100 units/mL of penicillin plus 100 μg/mL of streptomycin (Invitrogen). The cell lines used were not found in ICLAC for cross-contaminated or otherwise misidentified cell lines. The cells were routinely tested for mycoplasma contamination. For cell cycle synchronization, HeLa cells were first blocked in G1/S with 2.5 mM thymidine (Sigma) for 16 h and then released in fresh culture medium for 8 h to enrich mitotic cells. Plasmid transfections were performed with Lipofectamine 2000 (Invitrogen) according to the manufacturer’s instructions.Inhibitors and treatmentsNocodazole (100 ng/mL, ≥99%), monastrol (50 μM, ≥98%), MG132 (10 μM, ≥90%), OA (500 nM, ≥92%), Reversine (1 μM, ≥98%), Roscovitine (20 μM, ≥98%), NAM (5 mM, ≥99.5%), and TSA (1 μM, ≥98%) were from Sigma. MG149 (100 μM, >99%) was from Axon. NU9056 (20 μM, >98%), ZM447439 (2 μM, >99%) were from Tocris Bioscience. GSK923295 (50 nM, >99%), BI2536 (100 nM, >99%), VX-680 (500 nM, >99%) was from Selleckchem. The protease inhibitors cocktail was from Sigma.RNA interferenceThe lentivirus-based vector PLKO.1 along with pRSV-Rev, pMDLg/pRRE, and pMD2.G were ordered from Addgene and used for producing shRNA-packaged viral particles as previously described43. The nucleotide sequence for shRNA against TIP60 was 5′-CCTCCTATCCTATCGAAGCTA-3′ (#1) and 5′-TCGAATTGTTTGGGCACTGAT-3′ (#2). PLKO.1 vectors with shRNA containing scrambled sequence 5′-CCTAAGGTTAAGTCGCCCTCG-3′ were used to generate a control virus. Previously described siRNA duplexes were used to repress TIP6010, Mps144, Nuf245, Hec146 and Aurora B47. 5’-UUCUCCGAACGUGUCACGUTT-3’ was used as a negative control siRNA. All siRNA duplexes were purchased from Qiagen and were transfected with Lipofectamine 2000 reagent (Invitrogen) according to the manufacturer’s instructions.AntibodiesAnti-TIP60 #1 (H-93, epitope aa 421–513, 1:1000), anti-Mps1 (C-19, 1:1000), and anti-Mad2 (17D10, 1:1000) antibodies were from Santa Cruz. Anti-α-tubulin (ab80779, 1:5000) was from Abcam. Anti-FLAG-tag (M2, 1:2000) antibody was from Sigma. Anti-GST-tag (2625, 1:2000), anti-His-tag (12698, 1:2000), anti-HA-tag (3724,1:2000), anti-MBP-tag (2396, 1:2000), anti-pS10-H3 (3377, 1:5000), anti-RCC1 (5134, 1:1000), anti-pS11-RCC1 Ser11 (5500, 1:1000), anti-acetylated-lysine (9441, 1:1000), and anti-pS7-CENP-A (2187, 1:1000) antibodies were from Cell Signaling Technology. Anti-Aurora B (AIM-1, 1:2000) antibody was from BD Biosciences. Anti-pT232-Aurora B antibody (600-401-677, 1:2000) was from Rockland. Rabbit anti-TIP60 #2 (epitope aa 80–95, 1:1000), anti-pS90-TIP60 (1:1000) and anti-acK215-Aurora B (1:1000) antibodies were generated by YenZym LLC. To generate anti-pS90-TIP60 antibody, a synthetic peptide containing phosphorylated S90 (C-KNGLPGSRPG-pS-PERE) was conjugated to rabbit albumin (Sigma) and injected into rabbits as previously described48. Serum was collected by a standard protocol and preabsorbed by unphosphorylated TIP60 peptide (C-KNGLPGSRPGSPERE) followed by affinity-purification using (C-KNGLPGSRPG-pS-PERE)-conjugated divinylsulfone Sepharose beads. Secondary antibodies were from Jackson Immune Research Laboratory. To generate anti-acK215-Aurora B antibody, peptide containing acetylated K215 (GLKGEL-acK-IADFGWS; synthesized by YenZym) was conjugated to rabbit albumin (Sigma) and immunized into rabbits as previously described48. The serum was collected and preabsorbed by unacetylated Aurora B peptide (GLKGELKIADFGWS) followed by affinity-purification using (GLKGEL-acK-IADFGWS)-conjugated sulftone sepharose beads (Sigma).Recombinant protein preparationThe acetylated protein was produced from E. coli as previously described26, 27. Briefly, E. coli strain Rosetta (DE3) was transformed with pACKRS and pAB-215TAG-HIS-IN plasmids simultaneously. The bacteria were cultured in lysogeny broth medium supplemented with kanamycin (50 mg/mL) and spectinomycin (50 mg/mL) to OD600 of 0.7. Acetyl-lysine (AcK, 10 mM) and NAM (20 mM) were then added, the culture was incubated for 0.5 h, and protein expression was induced with IPTG (0.2 mM) at 37 °C for 4 h. The bacteria were lysed by sonication in Ni-NTA binding buffer (50 mM NaH2PO4, pH 8.0, 300 mM NaCl, 10 mM imidazole) and incubated with Ni-NTA agarose (Qiagen) for 1.5 h at 4 °C. The agarose was washed three times in Ni-NTA binding buffer supplemented with 20 mM imidazole and eluted with Ni-NTA binding buffer supplemented with 250 mM imidazole. The eluted protein was then dialyzed against dialysis buffer (25 mM Tris-HCl, pH 7.4; 100 mM NaCl; 10% glycerol) for 4 h at 4 °C.GST-H3 (aa 1–15), MBP-Aurora B-WT, MBP-Aurora B-K215R were produced from bacteria as previously described41. Basically, the plasmids were transformed into E. coli strain Rosetta (DE3), and protein expression was induced with 0.2 mM IPTG at 16 °C. Bacteria expressing GST-H3 (aa 1–15) were suspended and lysed by sonication in PBS buffer supplemented with 1% Triton X-100. The preparation was incubated with glutathione-Sepharose 4B (GE Healthcare Life Science) for 1.5 h at 4 °C. The resin was washed three times, and GST-H3 protein was eluted with 10 mM glutathione. Bacteria expressing MBP-Aurora B-WT and MBP-Aurora B-K215R were lysed in MBP column buffer (20 mM Tris-HCl, pH 7.4; 200 mM NaCl; 1 mM EDTA) and incubated with amylose resin (New England Biolabs) for 1.5 h at 4 °C. The resin was washed three times in MBP column buffer and eluted with MBP column buffer supplemented with 10 mM maltose. All purification procedures were performed at 4 °C, and protease inhibitor cocktail (Sigma) was added to prevent protein degradation.Immunoprecipitation and pull-down assaysFor immunoprecipitation, cells were treated with indicated reagents before being trypsinized and lysed in EBC buffer (50 mM Tris-HCl, pH 8.0; 120 mM NaCl; 0.5% NP-40) supplemented with protease inhibitor cocktail (Sigma), phosphatase inhibitor cocktail (Sigma), TSA (1 μM), and NAM (10 mM). After pre-clearing with protein A/G resin, the lysate was incubated with Aurora B antibody at 4 °C for 24 h with gentle rotation. Protein A/G resin was then added to the lysates, and they were incubated for another 6 h. The Protein A/G resin was then spun down and washed five times with lysis buffer before being resolved by SDS-PAGE and immunoblotted with the indicated antibodies. For FLAG-tagged protein immunoprecipitation, the FLAG-M2 resin was added to the lysates and incubated for 4 h before washing. For in vitro reactions, the FLAG-beads were further washed twice with dialysis buffer, and the FLAG-tagged protein was eluted with dialysis buffer supplemented with 100 μg/mL 3× FLAG peptide (Sigma).For pull-down assays, FLAG-PR65 was co-expressed with PP2A for 36 h and subjected to FLAG-M2 resin immunoprecipitation. The immunoprecipitated FLAG-PR65 was incubated with either Aurora B or acK215-Aurora B in EBC buffer supplemented with protease inhibitor cocktail (Sigma), phosphatase inhibitor cocktail (Sigma), TSA (1 μM), and NAM (10 mM). 4 h later, the binding fraction was washed with EBC buffer 5 times and analyzed by Western blot.Aurora B kinase assay and characterization of kineticsPurified recombinant GST-H3 (1–15) and Aurora B were incubated in kinase buffer (25 mM HEPES, pH 7.4; 100 mM NaCl; 5 mM MgCl2; 1 mM DTT) supplemented with 100 μM ATP and protease cocktail inhibitor for 20 min at 30 °C. The kinase reactions were stopped by addition of 5× Sample buffer (10% SDS; 0.5% bromophenol blue; 50% glycerol; 1 M DTT) before being resolved by SDS-PAGE and immunoblotted with indicated antibodies.The kinetics of Aurora-B was characterized with Fluorometric Kinase Assay Kits (ATT Bioquest, 31001) following the manufacturer’s instructions. Basically, Aurora B kinase was incubated with GST-histone H3 (1–15) in 20 μl kinetics assay buffer (60 mM HEPES, pH 7.5; 3 mM MgCl2; 3 mM MnCl2; 3 μM Na-orthovanadate) in the presence of 0–100 μM ATP for 0.5 h at 37 °C. Then the ADP sensor and sensor buffer were added, and the preparations were incubated for another 15 min at room temperature. The fluorescence intensities were monitored at Ex/Em = 540/590 nm to determine the formation of ADP. The reaction was repeated, and the Km and kcat values were calculated according to the Michaelis-Menten equation.In vitro PP2A phosphatase assayPurified Aurora B was incubated with PP2A phosphatase (Millipore) in dephosphorylation buffer (50 mM Tris-HCl, pH7.4; 0.1 mM EDTA; 1 mM DTT; 2 mM MgCl2; 0.01% Brij-35) in the presence of EDTA-free protease inhibitor cocktail (Roche) with gentle agitation. The reactions were stopped by addition of 5× Sample buffer and heated at 95 °C for 5 min. The samples were resolved by SDS-PAGE and immunoblotted with indicated antibodies.In vitro acetylation assay and characterization of TIP60 kineticsThe acetylation reaction was performed essentially as previously described41. Basically, purified TIP60 was incubated with Aurora B in 30 μl HAT buffer (20 mM Tris-HCl, pH8.0; 10% glycerol; 100 mM NaCl; 1 mM DTT; 1 mM EDTA; 10 μM TSA; 10 mM NAM) containing 100 μM acetyl-CoA for 2 h at 37 °C. The reaction was stopped by addition of 5× Sample buffer and heated at 95 °C for 5 min before being resolved by SDS-PAGE and immunoblotted with indicated antibodies.Methods for characterizing the acetyltransferase kinetics of TIP60 has been described12. Basically, purified TIP60 was incubated with H4 peptide (aa 1–20) and 14C labeled Ac-CoA in 30 μl HAT buffer at 30 °C. After incubation, the mixture was loaded onto Waterman P81 filter paper and then washed with 50 mM of sodium bicarbonate (pH 9.0) for three times. The radioactive products were quantified by a liquid scintillation spectrometer. The reaction was repeated for three times, and the Km and kcat values were calculated according to the Michaelis-Menten equation.Gel filtration assayHeLa cells synchronized with nocodazole were lysed in EBC buffer (50 mM Tris-HCl, pH 8.0; 120 mM NaCl; 0.5% NP-40) supplemented with protease inhibitor cocktail (Sigma) and phosphatase inhibitor cocktail (Sigma). Lysates were then centrifuged for 30 min at 200,000× g, and the supernatants were concentrated by Centriprep-10 (Millipore). Clarified lysates were filtered through a 0.45-μm membrane before being loaded onto a Superose 6, 10/300 GL column (GE Healthcare Life Sciences). Column fractions were then collected for SDS-PAGE and Western blotting analyses.Immunofluorescence and time-lapse imagingHeLa cells grown on coverslips were fixed by a pre-extraction method using PTEM buffer (60 mM PIPES, pH7.9; 10 mM EGTA; 2 mM MgCl2; 0.2% Triton X-100) supplemented with 3.7% paraformaldehyde. After blocking with PBST (PBS with 0.05% Tween-20) buffer containing 1% bovine serum albumin (Sigma) for 45 min at room temperature, the fixed cells were incubated with primary antibodies in a humidified chamber for 1 h at room temperature or overnight at 4 °C, followed by secondary antibodies for 1 h at 37 °C. The DNA was stained with 4′,6-diamidino-2-phenylindole (DAPI) from Sigma. Images were captured by DeltaVision softWoRx software (Applied Precision) and processed by deconvolution and z-stack projection.For Time-lapse imaging, HeLa cells were cultured in glass-bottom culture dishes (MatTek) and maintained in CO2-independent media (Gibco) supplemented with 10% FBS and 2 mM glutamine49. During imaging, the dishes were placed in a sealed chamber at 37 °C. Images of living cells were taken with a DeltaVision microscopy system.Fluorescence intensity quantificationQuantification of fluorescence intensity of kinetochore-associated proteins was performed as described previously using ImageJ45. In brief, the average pixel intensities from no less than five cells (which were randomly selected) were measured, and background pixel intensities were subtracted. The pixel intensities at each kinetochore pair were then normalized against ACA or Hec1 values to account for any variations in staining or image acquisition.StatisticsAll statistics were described in the figure legends. Two-tailed unpaired Student’s t-test was applied for experimental comparisons, using GraphPad Prism. All Western blotting analyses were taken from three separated experiments. No statistical method was used to predetermine sample size. All data were expected to have normal distribution.

ONLINE METHODS

PlasmidsSite-specific mutants of FLAG- or EGFP-tagged Aurora B and RNAi-resistant TIP60 were generated by PCR-based, site-directed mutagenesis kit from Vazyme (C212) according to the manufacturer’s instructions. MBP-Aurora B-WT and MBP-Aurora B-K215R plasmids were generated by sub-cloning Aurora B from corresponding EGFP vectors into the pMal-c2 vector. The GST-H3 N-tail construct was obtained by cloning a PCR fragment corresponding to aa 1–15 of human histone H3 into the pGEX-6p1 vector. ACKRS3 and pCDF-pylT plasmids were gifts from the laboratory of Jason Chin. pAB-HIS-INbox was constructed by inserting full-length Aurora B and a truncation of INCENP (aa 822–918) downstream of two adjacent T7 promoters in pCDF-pylT. The corresponding AAG codon in pAB-HIS-INbox was mutated to TAG in order to generate pAB-215TAG-HIS-INbox. All plasmids used were verified by sequencing (Invitrogen).

Site-specific mutants of FLAG- or EGFP-tagged Aurora B and RNAi-resistant TIP60 were generated by PCR-based, site-directed mutagenesis kit from Vazyme (C212) according to the manufacturer’s instructions. MBP-Aurora B-WT and MBP-Aurora B-K215R plasmids were generated by sub-cloning Aurora B from corresponding EGFP vectors into the pMal-c2 vector. The GST-H3 N-tail construct was obtained by cloning a PCR fragment corresponding to aa 1–15 of human histone H3 into the pGEX-6p1 vector. ACKRS3 and pCDF-pylT plasmids were gifts from the laboratory of Jason Chin. pAB-HIS-INbox was constructed by inserting full-length Aurora B and a truncation of INCENP (aa 822–918) downstream of two adjacent T7 promoters in pCDF-pylT. The corresponding AAG codon in pAB-HIS-INbox was mutated to TAG in order to generate pAB-215TAG-HIS-INbox. All plasmids used were verified by sequencing (Invitrogen).

Cell culture, synchronization, and transfectionHEK293T and HeLa cells were purchased from the American Type Culture Collection and maintained as monolayers in advanced DMEM (Invitrogen) with 10% fetal calf serum (HyClone) and 100 units/mL of penicillin plus 100 μg/mL of streptomycin (Invitrogen). The cell lines used were not found in ICLAC for cross-contaminated or otherwise misidentified cell lines. The cells were routinely tested for mycoplasma contamination. For cell cycle synchronization, HeLa cells were first blocked in G1/S with 2.5 mM thymidine (Sigma) for 16 h and then released in fresh culture medium for 8 h to enrich mitotic cells. Plasmid transfections were performed with Lipofectamine 2000 (Invitrogen) according to the manufacturer’s instructions.

Cell culture, synchronization, and transfection

HEK293T and HeLa cells were purchased from the American Type Culture Collection and maintained as monolayers in advanced DMEM (Invitrogen) with 10% fetal calf serum (HyClone) and 100 units/mL of penicillin plus 100 μg/mL of streptomycin (Invitrogen). The cell lines used were not found in ICLAC for cross-contaminated or otherwise misidentified cell lines. The cells were routinely tested for mycoplasma contamination. For cell cycle synchronization, HeLa cells were first blocked in G1/S with 2.5 mM thymidine (Sigma) for 16 h and then released in fresh culture medium for 8 h to enrich mitotic cells. Plasmid transfections were performed with Lipofectamine 2000 (Invitrogen) according to the manufacturer’s instructions.

Inhibitors and treatmentsNocodazole (100 ng/mL, ≥99%), monastrol (50 μM, ≥98%), MG132 (10 μM, ≥90%), OA (500 nM, ≥92%), Reversine (1 μM, ≥98%), Roscovitine (20 μM, ≥98%), NAM (5 mM, ≥99.5%), and TSA (1 μM, ≥98%) were from Sigma. MG149 (100 μM, >99%) was from Axon. NU9056 (20 μM, >98%), ZM447439 (2 μM, >99%) were from Tocris Bioscience. GSK923295 (50 nM, >99%), BI2536 (100 nM, >99%), VX-680 (500 nM, >99%) was from Selleckchem. The protease inhibitors cocktail was from Sigma.

Inhibitors and treatments

Nocodazole (100 ng/mL, ≥99%), monastrol (50 μM, ≥98%), MG132 (10 μM, ≥90%), OA (500 nM, ≥92%), Reversine (1 μM, ≥98%), Roscovitine (20 μM, ≥98%), NAM (5 mM, ≥99.5%), and TSA (1 μM, ≥98%) were from Sigma. MG149 (100 μM, >99%) was from Axon. NU9056 (20 μM, >98%), ZM447439 (2 μM, >99%) were from Tocris Bioscience. GSK923295 (50 nM, >99%), BI2536 (100 nM, >99%), VX-680 (500 nM, >99%) was from Selleckchem. The protease inhibitors cocktail was from Sigma.

RNA interferenceThe lentivirus-based vector PLKO.1 along with pRSV-Rev, pMDLg/pRRE, and pMD2.G were ordered from Addgene and used for producing shRNA-packaged viral particles as previously described43. The nucleotide sequence for shRNA against TIP60 was 5′-CCTCCTATCCTATCGAAGCTA-3′ (#1) and 5′-TCGAATTGTTTGGGCACTGAT-3′ (#2). PLKO.1 vectors with shRNA containing scrambled sequence 5′-CCTAAGGTTAAGTCGCCCTCG-3′ were used to generate a control virus. Previously described siRNA duplexes were used to repress TIP6010, Mps144, Nuf245, Hec146 and Aurora B47. 5’-UUCUCCGAACGUGUCACGUTT-3’ was used as a negative control siRNA. All siRNA duplexes were purchased from Qiagen and were transfected with Lipofectamine 2000 reagent (Invitrogen) according to the manufacturer’s instructions.

RNA interference

The lentivirus-based vector PLKO.1 along with pRSV-Rev, pMDLg/pRRE, and pMD2.G were ordered from Addgene and used for producing shRNA-packaged viral particles as previously described43. The nucleotide sequence for shRNA against TIP60 was 5′-CCTCCTATCCTATCGAAGCTA-3′ (#1) and 5′-TCGAATTGTTTGGGCACTGAT-3′ (#2). PLKO.1 vectors with shRNA containing scrambled sequence 5′-CCTAAGGTTAAGTCGCCCTCG-3′ were used to generate a control virus. Previously described siRNA duplexes were used to repress TIP6010, Mps144, Nuf245, Hec146 and Aurora B47. 5’-UUCUCCGAACGUGUCACGUTT-3’ was used as a negative control siRNA. All siRNA duplexes were purchased from Qiagen and were transfected with Lipofectamine 2000 reagent (Invitrogen) according to the manufacturer’s instructions.

AntibodiesAnti-TIP60 #1 (H-93, epitope aa 421–513, 1:1000), anti-Mps1 (C-19, 1:1000), and anti-Mad2 (17D10, 1:1000) antibodies were from Santa Cruz. Anti-α-tubulin (ab80779, 1:5000) was from Abcam. Anti-FLAG-tag (M2, 1:2000) antibody was from Sigma. Anti-GST-tag (2625, 1:2000), anti-His-tag (12698, 1:2000), anti-HA-tag (3724,1:2000), anti-MBP-tag (2396, 1:2000), anti-pS10-H3 (3377, 1:5000), anti-RCC1 (5134, 1:1000), anti-pS11-RCC1 Ser11 (5500, 1:1000), anti-acetylated-lysine (9441, 1:1000), and anti-pS7-CENP-A (2187, 1:1000) antibodies were from Cell Signaling Technology. Anti-Aurora B (AIM-1, 1:2000) antibody was from BD Biosciences. Anti-pT232-Aurora B antibody (600-401-677, 1:2000) was from Rockland. Rabbit anti-TIP60 #2 (epitope aa 80–95, 1:1000), anti-pS90-TIP60 (1:1000) and anti-acK215-Aurora B (1:1000) antibodies were generated by YenZym LLC. To generate anti-pS90-TIP60 antibody, a synthetic peptide containing phosphorylated S90 (C-KNGLPGSRPG-pS-PERE) was conjugated to rabbit albumin (Sigma) and injected into rabbits as previously described48. Serum was collected by a standard protocol and preabsorbed by unphosphorylated TIP60 peptide (C-KNGLPGSRPGSPERE) followed by affinity-purification using (C-KNGLPGSRPG-pS-PERE)-conjugated divinylsulfone Sepharose beads. Secondary antibodies were from Jackson Immune Research Laboratory. To generate anti-acK215-Aurora B antibody, peptide containing acetylated K215 (GLKGEL-acK-IADFGWS; synthesized by YenZym) was conjugated to rabbit albumin (Sigma) and immunized into rabbits as previously described48. The serum was collected and preabsorbed by unacetylated Aurora B peptide (GLKGELKIADFGWS) followed by affinity-purification using (GLKGEL-acK-IADFGWS)-conjugated sulftone sepharose beads (Sigma).

Anti-TIP60 #1 (H-93, epitope aa 421–513, 1:1000), anti-Mps1 (C-19, 1:1000), and anti-Mad2 (17D10, 1:1000) antibodies were from Santa Cruz. Anti-α-tubulin (ab80779, 1:5000) was from Abcam. Anti-FLAG-tag (M2, 1:2000) antibody was from Sigma. Anti-GST-tag (2625, 1:2000), anti-His-tag (12698, 1:2000), anti-HA-tag (3724,1:2000), anti-MBP-tag (2396, 1:2000), anti-pS10-H3 (3377, 1:5000), anti-RCC1 (5134, 1:1000), anti-pS11-RCC1 Ser11 (5500, 1:1000), anti-acetylated-lysine (9441, 1:1000), and anti-pS7-CENP-A (2187, 1:1000) antibodies were from Cell Signaling Technology. Anti-Aurora B (AIM-1, 1:2000) antibody was from BD Biosciences. Anti-pT232-Aurora B antibody (600-401-677, 1:2000) was from Rockland. Rabbit anti-TIP60 #2 (epitope aa 80–95, 1:1000), anti-pS90-TIP60 (1:1000) and anti-acK215-Aurora B (1:1000) antibodies were generated by YenZym LLC. To generate anti-pS90-TIP60 antibody, a synthetic peptide containing phosphorylated S90 (C-KNGLPGSRPG-pS-PERE) was conjugated to rabbit albumin (Sigma) and injected into rabbits as previously described48. Serum was collected by a standard protocol and preabsorbed by unphosphorylated TIP60 peptide (C-KNGLPGSRPGSPERE) followed by affinity-purification using (C-KNGLPGSRPG-pS-PERE)-conjugated divinylsulfone Sepharose beads. Secondary antibodies were from Jackson Immune Research Laboratory. To generate anti-acK215-Aurora B antibody, peptide containing acetylated K215 (GLKGEL-acK-IADFGWS; synthesized by YenZym) was conjugated to rabbit albumin (Sigma) and immunized into rabbits as previously described48. The serum was collected and preabsorbed by unacetylated Aurora B peptide (GLKGELKIADFGWS) followed by affinity-purification using (GLKGEL-acK-IADFGWS)-conjugated sulftone sepharose beads (Sigma).

Recombinant protein preparationThe acetylated protein was produced from E. coli as previously described26, 27. Briefly, E. coli strain Rosetta (DE3) was transformed with pACKRS and pAB-215TAG-HIS-IN plasmids simultaneously. The bacteria were cultured in lysogeny broth medium supplemented with kanamycin (50 mg/mL) and spectinomycin (50 mg/mL) to OD600 of 0.7. Acetyl-lysine (AcK, 10 mM) and NAM (20 mM) were then added, the culture was incubated for 0.5 h, and protein expression was induced with IPTG (0.2 mM) at 37 °C for 4 h. The bacteria were lysed by sonication in Ni-NTA binding buffer (50 mM NaH2PO4, pH 8.0, 300 mM NaCl, 10 mM imidazole) and incubated with Ni-NTA agarose (Qiagen) for 1.5 h at 4 °C. The agarose was washed three times in Ni-NTA binding buffer supplemented with 20 mM imidazole and eluted with Ni-NTA binding buffer supplemented with 250 mM imidazole. The eluted protein was then dialyzed against dialysis buffer (25 mM Tris-HCl, pH 7.4; 100 mM NaCl; 10% glycerol) for 4 h at 4 °C.GST-H3 (aa 1–15), MBP-Aurora B-WT, MBP-Aurora B-K215R were produced from bacteria as previously described41. Basically, the plasmids were transformed into E. coli strain Rosetta (DE3), and protein expression was induced with 0.2 mM IPTG at 16 °C. Bacteria expressing GST-H3 (aa 1–15) were suspended and lysed by sonication in PBS buffer supplemented with 1% Triton X-100. The preparation was incubated with glutathione-Sepharose 4B (GE Healthcare Life Science) for 1.5 h at 4 °C. The resin was washed three times, and GST-H3 protein was eluted with 10 mM glutathione. Bacteria expressing MBP-Aurora B-WT and MBP-Aurora B-K215R were lysed in MBP column buffer (20 mM Tris-HCl, pH 7.4; 200 mM NaCl; 1 mM EDTA) and incubated with amylose resin (New England Biolabs) for 1.5 h at 4 °C. The resin was washed three times in MBP column buffer and eluted with MBP column buffer supplemented with 10 mM maltose. All purification procedures were performed at 4 °C, and protease inhibitor cocktail (Sigma) was added to prevent protein degradation.

Recombinant protein preparation

The acetylated protein was produced from E. coli as previously described26, 27. Briefly, E. coli strain Rosetta (DE3) was transformed with pACKRS and pAB-215TAG-HIS-IN plasmids simultaneously. The bacteria were cultured in lysogeny broth medium supplemented with kanamycin (50 mg/mL) and spectinomycin (50 mg/mL) to OD600 of 0.7. Acetyl-lysine (AcK, 10 mM) and NAM (20 mM) were then added, the culture was incubated for 0.5 h, and protein expression was induced with IPTG (0.2 mM) at 37 °C for 4 h. The bacteria were lysed by sonication in Ni-NTA binding buffer (50 mM NaH2PO4, pH 8.0, 300 mM NaCl, 10 mM imidazole) and incubated with Ni-NTA agarose (Qiagen) for 1.5 h at 4 °C. The agarose was washed three times in Ni-NTA binding buffer supplemented with 20 mM imidazole and eluted with Ni-NTA binding buffer supplemented with 250 mM imidazole. The eluted protein was then dialyzed against dialysis buffer (25 mM Tris-HCl, pH 7.4; 100 mM NaCl; 10% glycerol) for 4 h at 4 °C.

GST-H3 (aa 1–15), MBP-Aurora B-WT, MBP-Aurora B-K215R were produced from bacteria as previously described41. Basically, the plasmids were transformed into E. coli strain Rosetta (DE3), and protein expression was induced with 0.2 mM IPTG at 16 °C. Bacteria expressing GST-H3 (aa 1–15) were suspended and lysed by sonication in PBS buffer supplemented with 1% Triton X-100. The preparation was incubated with glutathione-Sepharose 4B (GE Healthcare Life Science) for 1.5 h at 4 °C. The resin was washed three times, and GST-H3 protein was eluted with 10 mM glutathione. Bacteria expressing MBP-Aurora B-WT and MBP-Aurora B-K215R were lysed in MBP column buffer (20 mM Tris-HCl, pH 7.4; 200 mM NaCl; 1 mM EDTA) and incubated with amylose resin (New England Biolabs) for 1.5 h at 4 °C. The resin was washed three times in MBP column buffer and eluted with MBP column buffer supplemented with 10 mM maltose. All purification procedures were performed at 4 °C, and protease inhibitor cocktail (Sigma) was added to prevent protein degradation.

Immunoprecipitation and pull-down assaysFor immunoprecipitation, cells were treated with indicated reagents before being trypsinized and lysed in EBC buffer (50 mM Tris-HCl, pH 8.0; 120 mM NaCl; 0.5% NP-40) supplemented with protease inhibitor cocktail (Sigma), phosphatase inhibitor cocktail (Sigma), TSA (1 μM), and NAM (10 mM). After pre-clearing with protein A/G resin, the lysate was incubated with Aurora B antibody at 4 °C for 24 h with gentle rotation. Protein A/G resin was then added to the lysates, and they were incubated for another 6 h. The Protein A/G resin was then spun down and washed five times with lysis buffer before being resolved by SDS-PAGE and immunoblotted with the indicated antibodies. For FLAG-tagged protein immunoprecipitation, the FLAG-M2 resin was added to the lysates and incubated for 4 h before washing. For in vitro reactions, the FLAG-beads were further washed twice with dialysis buffer, and the FLAG-tagged protein was eluted with dialysis buffer supplemented with 100 μg/mL 3× FLAG peptide (Sigma).For pull-down assays, FLAG-PR65 was co-expressed with PP2A for 36 h and subjected to FLAG-M2 resin immunoprecipitation. The immunoprecipitated FLAG-PR65 was incubated with either Aurora B or acK215-Aurora B in EBC buffer supplemented with protease inhibitor cocktail (Sigma), phosphatase inhibitor cocktail (Sigma), TSA (1 μM), and NAM (10 mM). 4 h later, the binding fraction was washed with EBC buffer 5 times and analyzed by Western blot.

Immunoprecipitation and pull-down assays

For immunoprecipitation, cells were treated with indicated reagents before being trypsinized and lysed in EBC buffer (50 mM Tris-HCl, pH 8.0; 120 mM NaCl; 0.5% NP-40) supplemented with protease inhibitor cocktail (Sigma), phosphatase inhibitor cocktail (Sigma), TSA (1 μM), and NAM (10 mM). After pre-clearing with protein A/G resin, the lysate was incubated with Aurora B antibody at 4 °C for 24 h with gentle rotation. Protein A/G resin was then added to the lysates, and they were incubated for another 6 h. The Protein A/G resin was then spun down and washed five times with lysis buffer before being resolved by SDS-PAGE and immunoblotted with the indicated antibodies. For FLAG-tagged protein immunoprecipitation, the FLAG-M2 resin was added to the lysates and incubated for 4 h before washing. For in vitro reactions, the FLAG-beads were further washed twice with dialysis buffer, and the FLAG-tagged protein was eluted with dialysis buffer supplemented with 100 μg/mL 3× FLAG peptide (Sigma).

For pull-down assays, FLAG-PR65 was co-expressed with PP2A for 36 h and subjected to FLAG-M2 resin immunoprecipitation. The immunoprecipitated FLAG-PR65 was incubated with either Aurora B or acK215-Aurora B in EBC buffer supplemented with protease inhibitor cocktail (Sigma), phosphatase inhibitor cocktail (Sigma), TSA (1 μM), and NAM (10 mM). 4 h later, the binding fraction was washed with EBC buffer 5 times and analyzed by Western blot.

Aurora B kinase assay and characterization of kineticsPurified recombinant GST-H3 (1–15) and Aurora B were incubated in kinase buffer (25 mM HEPES, pH 7.4; 100 mM NaCl; 5 mM MgCl2; 1 mM DTT) supplemented with 100 μM ATP and protease cocktail inhibitor for 20 min at 30 °C. The kinase reactions were stopped by addition of 5× Sample buffer (10% SDS; 0.5% bromophenol blue; 50% glycerol; 1 M DTT) before being resolved by SDS-PAGE and immunoblotted with indicated antibodies.The kinetics of Aurora-B was characterized with Fluorometric Kinase Assay Kits (ATT Bioquest, 31001) following the manufacturer’s instructions. Basically, Aurora B kinase was incubated with GST-histone H3 (1–15) in 20 μl kinetics assay buffer (60 mM HEPES, pH 7.5; 3 mM MgCl2; 3 mM MnCl2; 3 μM Na-orthovanadate) in the presence of 0–100 μM ATP for 0.5 h at 37 °C. Then the ADP sensor and sensor buffer were added, and the preparations were incubated for another 15 min at room temperature. The fluorescence intensities were monitored at Ex/Em = 540/590 nm to determine the formation of ADP. The reaction was repeated, and the Km and kcat values were calculated according to the Michaelis-Menten equation.

Aurora B kinase assay and characterization of kinetics

Purified recombinant GST-H3 (1–15) and Aurora B were incubated in kinase buffer (25 mM HEPES, pH 7.4; 100 mM NaCl; 5 mM MgCl2; 1 mM DTT) supplemented with 100 μM ATP and protease cocktail inhibitor for 20 min at 30 °C. The kinase reactions were stopped by addition of 5× Sample buffer (10% SDS; 0.5% bromophenol blue; 50% glycerol; 1 M DTT) before being resolved by SDS-PAGE and immunoblotted with indicated antibodies.

The kinetics of Aurora-B was characterized with Fluorometric Kinase Assay Kits (ATT Bioquest, 31001) following the manufacturer’s instructions. Basically, Aurora B kinase was incubated with GST-histone H3 (1–15) in 20 μl kinetics assay buffer (60 mM HEPES, pH 7.5; 3 mM MgCl2; 3 mM MnCl2; 3 μM Na-orthovanadate) in the presence of 0–100 μM ATP for 0.5 h at 37 °C. Then the ADP sensor and sensor buffer were added, and the preparations were incubated for another 15 min at room temperature. The fluorescence intensities were monitored at Ex/Em = 540/590 nm to determine the formation of ADP. The reaction was repeated, and the Km and kcat values were calculated according to the Michaelis-Menten equation.

In vitro PP2A phosphatase assayPurified Aurora B was incubated with PP2A phosphatase (Millipore) in dephosphorylation buffer (50 mM Tris-HCl, pH7.4; 0.1 mM EDTA; 1 mM DTT; 2 mM MgCl2; 0.01% Brij-35) in the presence of EDTA-free protease inhibitor cocktail (Roche) with gentle agitation. The reactions were stopped by addition of 5× Sample buffer and heated at 95 °C for 5 min. The samples were resolved by SDS-PAGE and immunoblotted with indicated antibodies.

In vitro PP2A phosphatase assay

Purified Aurora B was incubated with PP2A phosphatase (Millipore) in dephosphorylation buffer (50 mM Tris-HCl, pH7.4; 0.1 mM EDTA; 1 mM DTT; 2 mM MgCl2; 0.01% Brij-35) in the presence of EDTA-free protease inhibitor cocktail (Roche) with gentle agitation. The reactions were stopped by addition of 5× Sample buffer and heated at 95 °C for 5 min. The samples were resolved by SDS-PAGE and immunoblotted with indicated antibodies.

In vitro acetylation assay and characterization of TIP60 kineticsThe acetylation reaction was performed essentially as previously described41. Basically, purified TIP60 was incubated with Aurora B in 30 μl HAT buffer (20 mM Tris-HCl, pH8.0; 10% glycerol; 100 mM NaCl; 1 mM DTT; 1 mM EDTA; 10 μM TSA; 10 mM NAM) containing 100 μM acetyl-CoA for 2 h at 37 °C. The reaction was stopped by addition of 5× Sample buffer and heated at 95 °C for 5 min before being resolved by SDS-PAGE and immunoblotted with indicated antibodies.Methods for characterizing the acetyltransferase kinetics of TIP60 has been described12. Basically, purified TIP60 was incubated with H4 peptide (aa 1–20) and 14C labeled Ac-CoA in 30 μl HAT buffer at 30 °C. After incubation, the mixture was loaded onto Waterman P81 filter paper and then washed with 50 mM of sodium bicarbonate (pH 9.0) for three times. The radioactive products were quantified by a liquid scintillation spectrometer. The reaction was repeated for three times, and the Km and kcat values were calculated according to the Michaelis-Menten equation.

In vitro acetylation assay and characterization of TIP60 kinetics

The acetylation reaction was performed essentially as previously described41. Basically, purified TIP60 was incubated with Aurora B in 30 μl HAT buffer (20 mM Tris-HCl, pH8.0; 10% glycerol; 100 mM NaCl; 1 mM DTT; 1 mM EDTA; 10 μM TSA; 10 mM NAM) containing 100 μM acetyl-CoA for 2 h at 37 °C. The reaction was stopped by addition of 5× Sample buffer and heated at 95 °C for 5 min before being resolved by SDS-PAGE and immunoblotted with indicated antibodies.

Methods for characterizing the acetyltransferase kinetics of TIP60 has been described12. Basically, purified TIP60 was incubated with H4 peptide (aa 1–20) and 14C labeled Ac-CoA in 30 μl HAT buffer at 30 °C. After incubation, the mixture was loaded onto Waterman P81 filter paper and then washed with 50 mM of sodium bicarbonate (pH 9.0) for three times. The radioactive products were quantified by a liquid scintillation spectrometer. The reaction was repeated for three times, and the Km and kcat values were calculated according to the Michaelis-Menten equation.

Gel filtration assayHeLa cells synchronized with nocodazole were lysed in EBC buffer (50 mM Tris-HCl, pH 8.0; 120 mM NaCl; 0.5% NP-40) supplemented with protease inhibitor cocktail (Sigma) and phosphatase inhibitor cocktail (Sigma). Lysates were then centrifuged for 30 min at 200,000× g, and the supernatants were concentrated by Centriprep-10 (Millipore). Clarified lysates were filtered through a 0.45-μm membrane before being loaded onto a Superose 6, 10/300 GL column (GE Healthcare Life Sciences). Column fractions were then collected for SDS-PAGE and Western blotting analyses.

Gel filtration assay

HeLa cells synchronized with nocodazole were lysed in EBC buffer (50 mM Tris-HCl, pH 8.0; 120 mM NaCl; 0.5% NP-40) supplemented with protease inhibitor cocktail (Sigma) and phosphatase inhibitor cocktail (Sigma). Lysates were then centrifuged for 30 min at 200,000× g, and the supernatants were concentrated by Centriprep-10 (Millipore). Clarified lysates were filtered through a 0.45-μm membrane before being loaded onto a Superose 6, 10/300 GL column (GE Healthcare Life Sciences). Column fractions were then collected for SDS-PAGE and Western blotting analyses.

Immunofluorescence and time-lapse imagingHeLa cells grown on coverslips were fixed by a pre-extraction method using PTEM buffer (60 mM PIPES, pH7.9; 10 mM EGTA; 2 mM MgCl2; 0.2% Triton X-100) supplemented with 3.7% paraformaldehyde. After blocking with PBST (PBS with 0.05% Tween-20) buffer containing 1% bovine serum albumin (Sigma) for 45 min at room temperature, the fixed cells were incubated with primary antibodies in a humidified chamber for 1 h at room temperature or overnight at 4 °C, followed by secondary antibodies for 1 h at 37 °C. The DNA was stained with 4′,6-diamidino-2-phenylindole (DAPI) from Sigma. Images were captured by DeltaVision softWoRx software (Applied Precision) and processed by deconvolution and z-stack projection.For Time-lapse imaging, HeLa cells were cultured in glass-bottom culture dishes (MatTek) and maintained in CO2-independent media (Gibco) supplemented with 10% FBS and 2 mM glutamine49. During imaging, the dishes were placed in a sealed chamber at 37 °C. Images of living cells were taken with a DeltaVision microscopy system.

Immunofluorescence and time-lapse imaging

HeLa cells grown on coverslips were fixed by a pre-extraction method using PTEM buffer (60 mM PIPES, pH7.9; 10 mM EGTA; 2 mM MgCl2; 0.2% Triton X-100) supplemented with 3.7% paraformaldehyde. After blocking with PBST (PBS with 0.05% Tween-20) buffer containing 1% bovine serum albumin (Sigma) for 45 min at room temperature, the fixed cells were incubated with primary antibodies in a humidified chamber for 1 h at room temperature or overnight at 4 °C, followed by secondary antibodies for 1 h at 37 °C. The DNA was stained with 4′,6-diamidino-2-phenylindole (DAPI) from Sigma. Images were captured by DeltaVision softWoRx software (Applied Precision) and processed by deconvolution and z-stack projection.

For Time-lapse imaging, HeLa cells were cultured in glass-bottom culture dishes (MatTek) and maintained in CO2-independent media (Gibco) supplemented with 10% FBS and 2 mM glutamine49. During imaging, the dishes were placed in a sealed chamber at 37 °C. Images of living cells were taken with a DeltaVision microscopy system.

Fluorescence intensity quantificationQuantification of fluorescence intensity of kinetochore-associated proteins was performed as described previously using ImageJ45. In brief, the average pixel intensities from no less than five cells (which were randomly selected) were measured, and background pixel intensities were subtracted. The pixel intensities at each kinetochore pair were then normalized against ACA or Hec1 values to account for any variations in staining or image acquisition.

Fluorescence intensity quantification

Quantification of fluorescence intensity of kinetochore-associated proteins was performed as described previously using ImageJ45. In brief, the average pixel intensities from no less than five cells (which were randomly selected) were measured, and background pixel intensities were subtracted. The pixel intensities at each kinetochore pair were then normalized against ACA or Hec1 values to account for any variations in staining or image acquisition.

StatisticsAll statistics were described in the figure legends. Two-tailed unpaired Student’s t-test was applied for experimental comparisons, using GraphPad Prism. All Western blotting analyses were taken from three separated experiments. No statistical method was used to predetermine sample size. All data were expected to have normal distribution.

All statistics were described in the figure legends. Two-tailed unpaired Student’s t-test was applied for experimental comparisons, using GraphPad Prism. All Western blotting analyses were taken from three separated experiments. No statistical method was used to predetermine sample size. All data were expected to have normal distribution.

Supplementary Material1

Supplementary Material
