---
pmid: '29022589'
title: Dynamic landscape and regulation of RNA editing in mammals.
authors:
- Tan MH
- Li Q
- Shanmugam R
- Piskol R
- Kohler J
- Young AN
- Liu KI
- Zhang R
- Ramaswami G
- Ariyoshi K
- Gupte A
- Keegan LP
- George CX
- Ramu A
- Huang N
- Pollina EA
- Leeman DS
- Rustighi A
- Goh YPS
- GTEx Consortium
- Laboratory, Data Analysis &Coordinating Center (LDACC)—Analysis Working Group
- Statistical Methods groups—Analysis Working Group
- Enhancing GTEx (eGTEx) groups
- NIH Common Fund
- NIH/NCI
- NIH/NHGRI
- NIH/NIMH
- NIH/NIDA
- Biospecimen Collection Source Site—NDRI
- Biospecimen Collection Source Site—RPCI
- Biospecimen Core Resource—VARI
- Brain Bank Repository—University of Miami Brain Endowment Bank
- Leidos Biomedical—Project Management
- ELSI Study
- Genome Browser Data Integration &Visualization—EBI
- Genome Browser Data Integration &Visualization—UCSC Genomics Institute, University
  of California Santa Cruz
- Chawla A
- Del Sal G
- Peltz G
- Brunet A
- Conrad DF
- Samuel CE
- O'Connell MA
- Walkley CR
- Nishikura K
- Li JB
journal: Nature
year: '2017'
full_text_available: false
pmcid: PMC5723435
doi: 10.1038/nature24041
---

# Dynamic landscape and regulation of RNA editing in mammals.
**Authors:** Tan MH, Li Q, Shanmugam R, Piskol R, Kohler J, Young AN, Liu KI, Zhang R, Ramaswami G, Ariyoshi K, Gupte A, Keegan LP, George CX, Ramu A, Huang N, Pollina EA, Leeman DS, Rustighi A, Goh YPS, GTEx Consortium, Laboratory, Data Analysis &Coordinating Center (LDACC)—Analysis Working Group, Statistical Methods groups—Analysis Working Group, Enhancing GTEx (eGTEx) groups, NIH Common Fund, NIH/NCI, NIH/NHGRI, NIH/NIMH, NIH/NIDA, Biospecimen Collection Source Site—NDRI, Biospecimen Collection Source Site—RPCI, Biospecimen Core Resource—VARI, Brain Bank Repository—University of Miami Brain Endowment Bank, Leidos Biomedical—Project Management, ELSI Study, Genome Browser Data Integration &Visualization—EBI, Genome Browser Data Integration &Visualization—UCSC Genomics Institute, University of California Santa Cruz, Chawla A, Del Sal G, Peltz G, Brunet A, Conrad DF, Samuel CE, O'Connell MA, Walkley CR, Nishikura K, Li JB
**Journal:** Nature (2017)
**DOI:** [10.1038/nature24041](https://doi.org/10.1038/nature24041)
**PMC:** [PMC5723435](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5723435/)

## Abstract

1. Nature. 2017 Oct 11;550(7675):249-254. doi: 10.1038/nature24041.

Dynamic landscape and regulation of RNA editing in mammals.

Tan MH(1)(2)(3), Li Q(1), Shanmugam R(2)(3), Piskol R(1), Kohler J(1), Young 
AN(1), Liu KI(3), Zhang R(1), Ramaswami G(1), Ariyoshi K(4), Gupte A(5), Keegan 
LP(6)(7), George CX(8), Ramu A(9)(10), Huang N(9)(10), Pollina EA(1), Leeman 
DS(1), Rustighi A(11), Goh YPS(12); GTEx Consortium; Laboratory, Data Analysis 
&Coordinating Center (LDACC)—Analysis Working Group; Statistical Methods 
groups—Analysis Working Group; Enhancing GTEx (eGTEx) groups; NIH Common Fund; 
NIH/NCI; NIH/NHGRI; NIH/NIMH; NIH/NIDA; Biospecimen Collection Source Site—NDRI; 
Biospecimen Collection Source Site—RPCI; Biospecimen Core Resource—VARI; Brain 
Bank Repository—University of Miami Brain Endowment Bank; Leidos 
Biomedical—Project Management; ELSI Study; Genome Browser Data Integration 
&Visualization—EBI; Genome Browser Data Integration &Visualization—UCSC Genomics 
Institute, University of California Santa Cruz; Chawla A(12), Del Sal G(11), 
Peltz G(13), Brunet A(1), Conrad DF(9)(10), Samuel CE(8), O'Connell MA(6)(7), 
Walkley CR(5)(14), Nishikura K(4), Li JB(1).

Collaborators: Aguet F, Ardlie KG, Cummings BB, Gelfand ET, Getz G, Hadley K, 
Handsaker RE, Huang KH, Kashin S, Karczewski KJ, Lek M, Li X, MacArthur DG, 
Nedzel JL, Nguyen DT, Noble MS, Segrè AV, Trowbridge CA, Tukiainen T, Abell NS, 
Balliu B, Barshir R, Basha O, Battle A, Bogu GK, Brown A, Brown CD, Castel SE, 
Chen LS, Chiang C, Conrad DF, Cox NJ, Damani FN, Davis JR, Delaneau O, 
Dermitzakis ET, Engelhardt BE, Eskin E, Ferreira PG, Frésard L, Gamazon ER, 
Garrido-Martín D, Gewirtz ADH, Gliner G, Gloudemans MJ, Guigo R, Hall IM, Han B, 
He Y, Hormozdiari F, Howald C, Kyung Im H, Jo B, Yong Kang E, Kim Y, 
Kim-Hellmuth S, Lappalainen T, Li G, Li X, Liu B, Mangul S, McCarthy MI, 
McDowell IC, Mohammadi P, Monlong J, Montgomery SB, Muñoz-Aguirre M, Ndungu AW, 
Nicolae DL, Nobel AB, Oliva M, Ongen H, Palowitch JJ, Panousis N, Papasaikas P, 
Park Y, Parsana P, Payne AJ, Peterson CB, Quan J, Reverter F, Sabatti C, Saha A, 
Sammeth M, Scott AJ, Shabalin AA, Sodaei R, Stephens M, Stranger BE, Strober BJ, 
Sul JH, Tsang EK, Urbut S, van de Bunt M, Wang G, Wen X, Wright FA, Xi HS, 
Yeger-Lotem E, Zappala Z, Zaugg JB, Zhou YH, Akey JM, Bates D, Chan J, Chen LS, 
Claussnitzer M, Demanelis K, Diegel M, Doherty JA, Feinberg AP, Fernando MS, 
Halow J, Hansen KD, Haugen E, Hickey PF, Hou L, Jasmine F, Jian R, Jiang L, 
Johnson A, Kaul R, Kellis M, Kibriya MG, Lee K, Li JB, Li Q, Li X, Lin J, Lin S, 
Linder S, Linke C, Liu Y, Maurano MT, Molinie B, Montgomery SB, Nelson J, Neri 
FJ, Oliva M, Park Y, Pierce BL, Rinaldi NJ, Rizzardi LF, Sandstrom R, Skol A, 
Smith KS, Snyder MP, Stamatoyannopoulos J, Stranger BE, Tang H, Tsang EK, Wang 
L, Wang M, Van Wittenberghe N, Wu F, Zhang R, Nierras CR, Branton PA, Carithers 
LJ, Guan P, Moore HM, Rao A, Vaught JB, Gould SE, Lockart NC, Martin C, 
Struewing JP, Volpi S, Addington AM, Koester SE, Little AR, Brigham LE, Hasz R, 
Hunter M, Johns C, Johnson M, Kopen G, Leinweber WF, Lonsdale JT, McDonald A, 
Mestichelli B, Myer K, Roe B, Salvatore M, Shad S, Thomas JA, Walters G, 
Washington M, Wheeler J, Bridge J, Foster BA, Gillard BM, Karasik E, Kumar R, 
Miklos M, Moser MT, Jewell SD, Montroy RG, Rohrer DC, Valley DR, Davis DA, Mash 
DC, Undale AH, Smith AM, Tabor DE, Roche NV, McLean JA, Vatanian N, Robinson KL, 
Sobin L, Barcus ME, Valentino KM, Qi L, Hunter S, Hariharan P, Singh S, Um KS, 
Matose T, Tomaszewski MM, Barker LK, Mosavel M, Siminoff LA, Traino HM, Flicek 
P, Juettemann T, Ruffier M, Sheppard D, Taylor K, Trevanion SJ, Zerbino DR, 
Craft B, Goldman M, Haeussler M, Kent WJ, Lee CM, Paten B, Rosenbloom KR, Vivian 
J, Zhu J, Craft B, Goldman M, Haeussler M, Kent WJ, Lee CM, Paten B, Rosenbloom 
KR, Vivian J, Zhu J.

Author information:
(1)Department of Genetics, Stanford University School of Medicine, Stanford, 
California 94305, USA.
(2)School of Chemical and Biomedical Engineering, Nanyang Technological 
University, Singapore 637459, Singapore.
(3)Genome Institute of Singapore, Agency for Science Technology and Research, 
Singapore 138672, Singapore.
(4)The Wistar Institute, Philadelphia, Pennsylvania 19104, USA.
(5)St Vincent's Institute of Medical Research, Fitzroy, Victoria 3065, 
Australia.
(6)MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, 
University of Edinburgh, Edinburgh EH4 2XU, UK.
(7)Central European Institute of Technology, Masaryk University, Kamenice, Brno 
625 00, Czech Republic.
(8)Department of Molecular, Cellular and Developmental Biology, University of 
California-Santa Barbara, Santa Barbara, California 93106, USA.
(9)Department of Genetics, Washington University School of Medicine, St Louis, 
Missouri 63108, USA.
(10)Department of Pathology &Immunology, Washington University School of 
Medicine, St Louis, Missouri 63108, USA.
(11)Department of Life Sciences, University of Trieste, 34127 Trieste, Italy and 
National Laboratory CIB (LNCIB), Area Science Park, 34149 Trieste, Italy.
(12)Cardiovascular Research Institute, University of California-San Francisco, 
San Francisco, California 94158, USA.
(13)Department of Anesthesia, Stanford University School of Medicine, Stanford, 
California 94305, USA.
(14)Department of Medicine, St. Vincent's Hospital, University of Melbourne, 
Fitzroy, Victoria 3065, Australia.

Comment in
    Nature. 2017 Oct 11;550(7675):190-191. doi: 10.1038/550190a.
    Nature. 2017 Oct 11;550(7675):157. doi: 10.1038/550157a.
    Nature. 2017 Oct 11;550(7675):169-170. doi: 10.1038/550169a.
    Nat Rev Genet. 2017 Dec;18(12):701. doi: 10.1038/nrg.2017.94.

Adenosine-to-inosine (A-to-I) RNA editing is a conserved post-transcriptional 
mechanism mediated by ADAR enzymes that diversifies the transcriptome by 
altering selected nucleotides in RNA molecules. Although many editing sites have 
recently been discovered, the extent to which most sites are edited and how the 
editing is regulated in different biological contexts are not fully understood. 
Here we report dynamic spatiotemporal patterns and new regulators of RNA 
editing, discovered through an extensive profiling of A-to-I RNA editing in 
8,551 human samples (representing 53 body sites from 552 individuals) from the 
Genotype-Tissue Expression (GTEx) project and in hundreds of other primate and 
mouse samples. We show that editing levels in non-repetitive coding regions vary 
more between tissues than editing levels in repetitive regions. Globally, ADAR1 
is the primary editor of repetitive sites and ADAR2 is the primary editor of 
non-repetitive coding sites, whereas the catalytically inactive ADAR3 
predominantly acts as an inhibitor of editing. Cross-species analysis of RNA 
editing in several tissues revealed that species, rather than tissue type, is 
the primary determinant of editing levels, suggesting stronger cis-directed 
regulation of RNA editing for most sites, although the small set of conserved 
coding sites is under stronger trans-regulation. In addition, we curated an 
extensive set of ADAR1 and ADAR2 targets and showed that many editing sites 
display distinct tissue-specific regulation by the ADAR enzymes in vivo. Further 
analysis of the GTEx data revealed several potential regulators of editing, such 
as AIMP2, which reduces editing in muscles by enhancing the degradation of the 
ADAR proteins. Collectively, our work provides insights into the complex cis- 
and trans-regulation of A-to-I editing.

DOI: 10.1038/nature24041
PMCID: PMC5723435
PMID: 29022589 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing financial 
interests. Readers are welcome to comment on the online version of the paper. 
Publisher’s note: Springer Nature remains neutral with regard to jurisdictional 
claims in published maps and institutional affiliations
