---
pmid: '22124327'
title: Positive regulation of p53 stability and activity by the deubiquitinating enzyme
  Otubain 1.
authors:
- Sun XX
- Challagundla KB
- Dai MS
journal: EMBO J
year: '2012'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC3273389
doi: 10.1038/emboj.2011.434
---

# Positive regulation of p53 stability and activity by the deubiquitinating enzyme Otubain 1.
**Authors:** Sun XX, Challagundla KB, Dai MS
**Journal:** EMBO J (2012)
**DOI:** [10.1038/emboj.2011.434](https://doi.org/10.1038/emboj.2011.434)
**PMC:** [PMC3273389](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3273389/)

## Abstract

1. EMBO J. 2012 Feb 1;31(3):576-92. doi: 10.1038/emboj.2011.434. Epub 2011 Nov
29.

Positive regulation of p53 stability and activity by the deubiquitinating enzyme 
Otubain 1.

Sun XX(1), Challagundla KB, Dai MS.

Author information:
(1)Department of Molecular & Medical Genetics, School of Medicine, OHSU Knight 
Cancer Institute, Oregon Health & Science University, Portland, OR 97239, USA.

The ubiquitin (Ub)-proteasome system plays a pivotal role in the regulation of 
p53 protein stability and activity. p53 is ubiquitinated and destabilized by 
MDM2 and several other Ub E3s, whereas it is deubiquitinated and stabilized by 
Ub-specific protease (USP)7 and USP10. Here we show that the ovarian tumour 
domain-containing Ub aldehyde-binding protein 1 (Otub1) is a novel p53 
regulator. Otub1 directly suppresses MDM2-mediated p53 ubiquitination in cells 
and in vitro. Overexpression of Otub1 drastically stabilizes and activates p53, 
leading to apoptosis and marked inhibition of cell proliferation in a 
p53-dependent manner. These effects are independent of its catalytic activity 
but require residue Asp88. Mutation of Asp88 to Ala (Otub1(D88A)) abolishes 
activity of Otub1 to suppress p53 ubiquitination. Further, wild-type Otub1 and 
its catalytic mutant (Otub1(C91S)), but not Otub1(D88A), bind to the MDM2 
cognate E2, UbcH5, and suppress its Ub-conjugating activity in vitro. 
Overexpression of Otub1(D88A) or ablation of endogenous Otub1 by siRNA markedly 
impaired p53 stabilization and activation in response to DNA damage. Together, 
these results reveal a novel function for Otub1 in regulating p53 stability and 
activity.

DOI: 10.1038/emboj.2011.434
PMCID: PMC3273389
PMID: 22124327 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflict 
of interest.

## Full Text

Abstract

The ubiquitin (Ub)–proteasome system plays a pivotal role in the regulation of p53 protein stability and activity. p53 is ubiquitinated and destabilized by MDM2 and several other Ub E3s, whereas it is deubiquitinated and stabilized by Ub-specific protease (USP)7 and USP10. Here we show that the ovarian tumour domain-containing Ub aldehyde-binding protein 1 (Otub1) is a novel p53 regulator. Otub1 directly suppresses MDM2-mediated p53 ubiquitination in cells and in vitro . Overexpression of Otub1 drastically stabilizes and activates p53, leading to apoptosis and marked inhibition of cell proliferation in a p53-dependent manner. These effects are independent of its catalytic activity but require residue Asp88. Mutation of Asp88 to Ala (Otub1 D88A ) abolishes activity of Otub1 to suppress p53 ubiquitination. Further, wild-type Otub1 and its catalytic mutant (Otub1 C91S ), but not Otub1 D88A , bind to the MDM2 cognate E2, UbcH5, and suppress its Ub-conjugating activity in vitro . Overexpression of Otub1 D88A or ablation of endogenous Otub1 by siRNA markedly impaired p53 stabilization and activation in response to DNA damage. Together, these results reveal a novel function for Otub1 in regulating p53 stability and activity.

Introduction

The p53 tumour suppressor plays a key role in maintaining genomic integrity and preventing cells from malignant transformation ( Levine, 1997 ; Vogelstein et al, 2000 ; Oren, 2003 ). Inactivation of p53 function by mutations in the p53 gene or other defects in the p53 signalling pathways is associated with most human cancers ( Vogelstein et al, 2000 ; Oren, 2003 ). p53 is activated in response to diverse stress and induces expression of various genes, whose protein products mediate cell cycle arrest, apoptosis, senescence, autophagy, angiogenesis inhibition, and regulation of energy metabolism ( Vogelstein et al, 2000 ; Oren, 2003 ; Levine and Oren, 2009 ; Vousden and Prives, 2009 ; Vousden and Ryan, 2009 ).

Under physiological conditions, p53 is maintained at low levels primarily by the oncoprotein MDM2. MDM2 binds to the N-terminal transactivation domain (TAD) of p53 ( Chen et al, 1993 ; Oliner et al, 1993 ), directly inhibiting its transcriptional activity ( Momand et al, 1992 ; Chen et al, 1993 ). As a Ring-finger-containing ubiquitin (Ub) ligase (E3) ( Honda et al, 1997 ; Fang et al, 2000 ), MDM2 also promotes p53 ubiquitination and degradation through the proteasome system ( Haupt et al, 1997 ; Kubbutat et al, 1997 ). Further, MDM2 mediates p53 nuclear export ( Freedman and Levine, 1998 ; Roth et al, 1998 ; Li et al, 2003 ). Together, MDM2 suppresses p53-mediated cell growth arrest and apoptosis. Consistently, MDM2 is overexpressed in several types of human cancers, such as soft tissue sarcomas, leukaemia, and breast cancers ( Bueso-Ramos et al, 1993 ; Cordon-Cardo et al, 1994 ; Momand et al, 1998 ; Deb, 2003 ; Dworakowska et al, 2004 ). As MDM2 is transcriptionally induced by p53, the two proteins form an elegant autoregulatory feedback loop ( Barak et al, 1993 ; Picksley and Lane, 1993 ; Wu et al, 1993 ). Genetic disruption of the p53 gene rescues the lethal phenotype of mdm2 knockout mice, firmly validating the notion of the MDM2–p53 feedback loop ( Jones et al, 1995 ; Montes de Oca Luna et al, 1995 ). Mice that are homozygous for a knock-in of an MDM2 E3-inactive mutant, C462A, are also embryonic lethal and can be rescued by deleting p53 as well, providing compelling evidence that the Ub E3 function of MDM2 is indispensible for its suppression of p53 in vivo ( Itahana et al, 2007 ). p53 can also be ubiquitinated by a number of other Ub E3s ( Dai et al, 2006 ), including Pirh2 ( Leng et al, 2003 ), COP1 ( Dornan et al, 2004 ), and ARF-BP1 ( Chen et al, 2005 ), although their exact function in regulating p53 in vivo remains unknown.

Similar to most posttranslational modifications, ubiquitination of p53 can be reversed by counteraction of deubiquitinating enzymes (DUBs). Human genome encodes approximately 95 putative DUBs, categorized into five classes: Ub-specific protease (USP), Ub C-terminal hydrolase (UCH), ovarian tumour (OTU) domain-containing protease, Machado–Joseph disease (MJD) protease, and JAB1/MPN/Mov34 metalloenzyme (JAMM; Nijman et al, 2005 ). The UCH, USP, OTU, and MJD families are cysteine proteases, whereas the JAMMs are zinc metalloproteases ( Nijman et al, 2005 ; Komander et al, 2009 ). Several USP family members have been shown to regulate the MDM2–p53 pathway. USP7 (also called HAUSP) deubiquitinates p53, leading to p53 stabilization and activation ( Li et al, 2002 ). USP7 also deubiquitinates MDM2 and MDMX, an MDM2 homologue also known as MDM4. Interestingly, partial knockdown of USP7 destabilizes p53, whereas substantial knockdown of USP7 stabilizes p53 through destabilization of MDM2 ( Cummins et al, 2004 ; Li et al, 2004 ). DNA-damage-induced phosphorylation of MDMX disrupts its binding to USP7, contributing to the destabilization of MDMX following DNA damage ( Meulmeester et al, 2005 ). Thus, a proper level of USP7 is required for maintaining the molecular ratio of p53–MDM2–MDMX axis. Most recently, USP10 has been shown to specifically deubiquitinate p53, but not MDM2 and MDMX ( Yuan et al, 2010 ). DNA damage triggers ATM-dependent phosphorylation and nuclear translocation of USP10, providing another mechanism for the regulation of p53 stability and activity by deubiquitination. Importantly, USP10 is required for efficient p53 activation in response to DNA damage ( Yuan et al, 2010 ). Similarly, USP29 has recently been shown to deubiquitinate and stabilize p53 in response to oxidative stress ( Liu et al, 2011 ). In contrast, USP2 deubiquitinates MDM2 ( Stevenson et al, 2007 ) and MDMX ( Allende-Vega et al, 2010 ), but not p53, leading to suppression of p53 activity. Knockdown of USP2 results in p53-dependent cell cycle arrest ( Stevenson et al, 2007 ). Thus, deubiquitination plays a crucial role in finely tuning normal homeostasis of the p53–MDM2–MDMX loop as well as its response to stress.

However, it is not known whether p53 is regulated by DUBs other than USP family members. Here we show that the ovarian tumour domain-containing Ub aldehyde-binding protein 1 (Otubain 1, Otub1 thereafter), an OTU family member DUB, is a novel p53 regulator. Otub1 directly suppresses MDM2-mediated p53 ubiquitination in cells and in vitro independent of its catalytic activity. It does so primarily by suppressing the activity of the MDM2 cognate Ub-conjugating enzyme (E2) UbcH5. Overexpression of Otub1 results in marked apoptosis and inhibition of cell proliferation in a p53-dependent manner. Inhibition of Otub1 markedly impaired p53 activation induced by DNA damage. Together, our results reveal a novel and critical role for Otub1 in regulating p53 stability and activity in response to DNA damage.

Discussion

Otub1 is a member of OTU domain-containing cysteine protease and has been shown to catalyse the cleavage of K48-linked poly-Ub in vitro ( Balakirev et al, 2003 ; Soares et al, 2004 ; Edelmann et al, 2009 ; also see Supplementary Figure S5A ). Soares et al (2004) first reported that Otub1 regulates T-cell anergy by enhancing the degradation of a key Ub E3 ligase called GRAIL (gene related to anergy in lymphocytes) independently of its deubiquitinase activity ( Soares et al, 2004 ). Recently, it has been shown that Otub1 deubiquitinates tumour necrosis factor receptor-associated factor (TRAF)-3 and TRAF6, leading to the inhibition of viral-induced INFβ production ( Li et al, 2010 ). Otub1 is ubiquitously expressed in most tissues ( Soares et al, 2004 ), suggesting that it might have a broader function. Indeed, it has been shown that Otub1 deubiquitinates oestrogen receptor-α and regulates its level and activity ( Stanisic et al, 2009 ), deubiquitinates RhoA small GTPase to facilitate the cellular uptake of Yersinia bacteria ( Edelmann et al, 2010 ), and suppresses DNA-damage-dependent chromatin ubiquitination ( Nakada et al, 2010 ). In this study, we found that Otub1 directly suppresses MDM2-mediated p53 ubiquitination in cells and in vitro , leading to stabilization and activation of p53. Overexpression of Otub1 induces p53-dependent apoptosis and drastic cell growth inhibition. Thus, we have identified Otub1 as a novel p53 regulator.

Otub1 contains a catalytic triad consisting of C91, H265, and D268 ( Edelmann et al, 2009 ) and possesses in vitro DUB enzymatic activity towards polyUb chains ( Supplementary Figure S5A ). However, mutation of the catalytic C91 did not abolish the ability of Otub1 to inhibit MDM2-mediated p53 ubiquitination in cells ( Figure 5 ) and in vitro ( Figure 7 ). Point mutations at H265, D268, or all three of the catalytic residues were unable to abolish Otub1 function (data not shown). Our data further revealed that Otub1 and its C91S mutant bind to the UbcH5 and significantly inhibit its Ub-conjugating activity, regardless of the presence or absence of MDM2 ( Figure 7C and D ). Thus, it appears that Otub1 suppresses p53 ubiquitination in cells primarily via non-canonical inhibition of UbcH5, although we cannot exclude the possibility that Otub1 also directly inhibits MDM2 E3 activity. This observation is consistent with a non-canonical mechanism by which Otub1 suppresses DNA-damage-induced chromatin ubiquitination ( Nakada et al, 2010 ). On the other hand, Otub1 possesses weak in vitro DUB enzymatic activity towards polyubiquitinated p53 ( Supplementary Figure S5B ). This weak activity might be due to the Otub1 conformation that is incompatible with typical functional catalytic cores—the imidazole ring of H265 is positioned far from catalytic C91 and sandwiched between a Pro and a Glu ( Edelmann et al, 2009 ). Whether and to what extend this weak DUB activity contributes to the function of Otub1 in regulating p53 in cells is not clear. However, the direct association of Otub1 with p53 ( Figure 3C ) and MDM2 ( Supplementary Figure S6 ) suggest that Otub1 may bind to the p53–MDM2–UbcH5 complex, wherein it suppresses the activity of E2/E3 while it may also cleave ployUb chains attached to p53 ( Figure 10F ). Interestingly, compared with wt Otub1, the Otub1 C91S mutant possesses stronger suppressing effects on MDM2-mediated p53 ubiquitination in cells ( Figure 5C ) and in vitro ( Figure 7A ) and on the Ub-conjugating activity of UbcH5 in the absence of MDM2 ( Figure 7D ). Also, Otub1 C91S binds to UbcH5 stronger ( Figure 7F and Supplementary Figure S7C and D ) than wt Otub1. Thus, it is likely that mutating this residue converts Otub1 to a conformation that favours its tight association with UbcH5 and stronger suppression of its activity.

On the other hand, mutation of the D88 residue abolished the activity of Otub1 to suppress p53 ubiquitination in cells ( Figure 6 ) and in vitro ( Figure 7 ). Consistently, Otub1 D88A failed to bind to and suppress UbcH5 ( Figure 7 ). D88 was initially predicted to be a catalytic residue that is conserved across species and among other OTU family DUBs ( Balakirev et al, 2003 ), and the D88A mutant failed to cleave polyUb chain in vitro ( Supplementary Figure S5A ). Thus D88 is essential for Otub1's DUB activity, albeit it is not among the catalytic triad ( Edelmann et al, 2009 ). D88 is located in a loop closely preceding the C91-containing active-site helix within the OTU domain ( Nanao et al, 2004 ; Edelmann et al, 2009 ). C91 may contact D88 to form a novel oxyanion hole ( Nanao et al, 2004 ; Edelmann et al, 2009 ) essential for stabilization of the oxyanion reaction intermediate in the enzymatic reaction ( Nijman et al, 2005 ). Thus, mutating D88 may disrupt the structure of the OTU domain. Supporting this notion, neither the Otub1 N-terminal fragment lacking the OTU domain nor the C-terminal fragment, which disrupts the OTU domain, can induce p53 when overexpressed even though both bind to p53 ( Supplementary Figure S3B ). Also, the Otub1 mutant with D88A, C91S, and H265A mutations failed to suppress DNA-damage-induced chromatin ubiquitination ( Nakada et al, 2010 ). These results indicate that Otub1 regulation of p53 ubiquitination and its binding to UbcH5 might require the integrity of the OTU domain.

Functionally, overexpression of Otub1 D88A significantly reduced the levels and activity of p53 and suppressed p53 activation in response to DNA damage ( Figure 8 ), suggesting that this mutant may act as a dominant-negative mutant of Otub1, considering that its strong binding to p53 ( Figure 6F ) may interfere with the binding of endogenous Otub1 to p53. Knockdown of endogenous Otub1 drastically attenuated the p53 stabilization and activation in response to DNA damage ( Figure 9 ). DNA damaging treatments markedly increased the interaction between p53 and Otub1 ( Figure 9K and Supplementary Figure S8A ). Importantly, Otub1 is essential for the G2/M- and S-phase checkpoints in response to DNA damage ( Figure 9I and J ). These results indicate that Otub1 plays a critical role in p53 signalling in response to DNA damage.

Interestingly, Otub1 has been suggested to act as an inhibitor of DNA repair by suppressing Ubc13/RNF168 activity. Under physiological condition, Otub1 may set a threshold for RNF168 activity to dynamically control chromatin ubiquitination ( Nakada et al, 2010 ). It is likely that upon DNA damage, Otub1 transiently dissociates from Ubc13–RNF168 complex allowing RNF168 to catalyse ubiquitination of chromatin at DNA damage sites ( Nakada et al, 2010 ), whereas it associates with UbcH5–MDM2 complex to suppress MDM2-mediated p53 ubiquitination, leading to p53 stabilization and activation. This dual action would likely work in concert to promote DNA damage response by initiating cell cycle arrest and subsequent DNA repair. This dual action might also play a role in lethal DNA damage response. In this condition, Otub1 may cause sustained p53 activation and apoptosis by suppressing UbcH5–MDM2 and permanently dissociating from the Ubc13/RNF168. Nevertheless, as knockdown of Otub1 reduces the levels of p53 and impairs p53 induction in response to DNA damage, our results indicate that Otub1 plays a crucial role in maintaining homeostatic levels of p53 under physiological conditions, via stoichiometric interplay with the MDM2–UbcH5 complex. It may also play a role in determining the functional outcome of the p53 activation.

Our results indicate that Otub1 regulates p53 in the cytoplasm and remains in the cytoplasm upon DNA damage ( Figure 10 ). In contrast, USP7 is a nuclear p53 deubiquitinase and USP10 is primarily a cytoplasmic p53 deubiquitinase but a fraction of it translocates into the nucleus following DNA damage ( Yuan et al, 2010 ). Therefore, these three DUBs regulate p53 function in different compartments, ensuring fine control of p53 stability and activity in response to stress. Future studies would examine how Otub1 regulates p53 signalling and whether it is subjected to posttranslational modifications in response to DNA damage. Altogether, our study places Otub1 as one of the central molecules that modulate the p53 stability and activity and suggests that Otub1 might have tumour suppressor function. It will be interesting to test whether expression of Otub1 is altered in human cancers and whether Otub1 suppresses tumourigenesis in vivo .
