---
pmid: '25817994'
title: PqqD is a novel peptide chaperone that forms a ternary complex with the radical
  S-adenosylmethionine protein PqqE in the pyrroloquinoline quinone biosynthetic pathway.
authors:
- Latham JA
- Iavarone AT
- Barr I
- Juthani PV
- Klinman JP
journal: J Biol Chem
year: '2015'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4432305
doi: 10.1074/jbc.M115.646521
---

# PqqD is a novel peptide chaperone that forms a ternary complex with the radical S-adenosylmethionine protein PqqE in the pyrroloquinoline quinone biosynthetic pathway.
**Authors:** Latham JA, Iavarone AT, Barr I, Juthani PV, Klinman JP
**Journal:** J Biol Chem (2015)
**DOI:** [10.1074/jbc.M115.646521](https://doi.org/10.1074/jbc.M115.646521)
**PMC:** [PMC4432305](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4432305/)

## Abstract

1. J Biol Chem. 2015 May 15;290(20):12908-18. doi: 10.1074/jbc.M115.646521. Epub 
2015 Mar 27.

PqqD is a novel peptide chaperone that forms a ternary complex with the radical 
S-adenosylmethionine protein PqqE in the pyrroloquinoline quinone biosynthetic 
pathway.

Latham JA(1), Iavarone AT(2), Barr I(1), Juthani PV(1), Klinman JP(3).

Author information:
(1)From the Departments of Chemistry and Molecular and Cell Biology and 
California Institute for Quantitative Biosciences, University of California, 
Berkeley, California 94720.
(2)From the Departments of Chemistry and California Institute for Quantitative 
Biosciences, University of California, Berkeley, California 94720.
(3)From the Departments of Chemistry and Molecular and Cell Biology and 
California Institute for Quantitative Biosciences, University of California, 
Berkeley, California 94720 klinman@berkeley.edu.

Pyrroloquinoline quinone (PQQ) is a product of a ribosomally synthesized and 
post-translationally modified pathway consisting of five conserved genes, 
pqqA-E. PqqE is a radical S-adenosylmethionine (RS) protein with a C-terminal 
SPASM domain, and is proposed to catalyze the formation of a carbon-carbon bond 
between the glutamate and tyrosine side chains of the peptide substrate PqqA. 
PqqD is a 10-kDa protein with an unknown function, but is essential for PQQ 
production. Recently, in Klebsiella pneumoniae (Kp), PqqD and PqqE were shown to 
interact; however, the stoichiometry and KD were not obtained. Here, we show 
that the PqqE and PqqD interaction transcends species, also occurring in 
Methylobacterium extorquens AM1 (Me). The stoichiometry of the MePqqD and MePqqE 
interaction is 1:1 and the KD, determined by surface plasmon resonance 
spectroscopy (SPR), was found to be ∼12 μm. Moreover, using SPR and isothermal 
calorimetry techniques, we establish for the first time that MePqqD binds MePqqA 
tightly (KD ∼200 nm). The formation of a ternary MePqqA-D-E complex was captured 
by native mass spectrometry and the KD for the MePqqAD-MePqqE interaction was 
found to be ∼5 μm. Finally, using a bioinformatic analysis, we found that PqqD 
orthologues are associated with the RS-SPASM family of proteins (subtilosin, 
pyrroloquinoline quinone, anaerobic sulfatase maturating enzyme, and 
mycofactocin), all of which modify either peptides or proteins. In conclusion, 
we propose that PqqD is a novel peptide chaperone and that PqqD orthologues may 
play a similar role in peptide modification pathways that use an RS-SPASM 
protein.

© 2015 by The American Society for Biochemistry and Molecular Biology, Inc.

DOI: 10.1074/jbc.M115.646521
PMCID: PMC4432305
PMID: 25817994 [Indexed for MEDLINE]

## Full Text

Abstract

Pyrroloquinoline quinone (PQQ) is a product of a ribosomally synthesized and post-translationally modified pathway consisting of five conserved genes, pqqA-E . PqqE is a radical S -adenosylmethionine (RS) protein with a C-terminal SPASM domain, and is proposed to catalyze the formation of a carbon-carbon bond between the glutamate and tyrosine side chains of the peptide substrate PqqA. PqqD is a 10-kDa protein with an unknown function, but is essential for PQQ production. Recently, in Klebsiella pneumoniae (Kp), PqqD and PqqE were shown to interact; however, the stoichiometry and K D were not obtained. Here, we show that the PqqE and PqqD interaction transcends species, also occurring in Methylobacterium extorquens AM1 (Me). The stoichiometry of the MePqqD and MePqqE interaction is 1:1 and the K D , determined by surface plasmon resonance spectroscopy (SPR), was found to be ∼12 μ m . Moreover, using SPR and isothermal calorimetry techniques, we establish for the first time that MePqqD binds MePqqA tightly ( K D ∼200 n m ). The formation of a ternary MePqqA-D-E complex was captured by native mass spectrometry and the K D for the MePqqAD-MePqqE interaction was found to be ∼5 μ m . Finally, using a bioinformatic analysis, we found that PqqD orthologues are associated with the RS-SPASM family of proteins (subtilosin, pyrroloquinoline quinone, anaerobic sulfatase maturating enzyme, and mycofactocin), all of which modify either peptides or proteins. In conclusion, we propose that PqqD is a novel peptide chaperone and that PqqD orthologues may play a similar role in peptide modification pathways that use an RS-SPASM protein.

Introduction

Ribosomally synthesized and post-translationally modified peptides constitute a major class of natural products consisting of bioactive compounds such as antibiotics ( 1 – 3 ), quorum sensing molecules ( 4 , 5 ), metal chelators ( 6 ), and redox cofactors ( 7 ). Certain ribosomally derived products rely on radical S -adenosylmethionine enzymes (herein referred to as RS proteins) to catalyze the formation of α-carbon thioether bonds or carbon-carbon bonds ( 8 , 9 ). In the subtilosin and the pyrroloquinoline quinone (PQQ) 2 pathways, the RS protein also contains a C-terminal extension referred to as a SPASM domain ( s ubtilosin, P QQ, a naerobic s ulfatase maturating enzyme, and m ycofactocin). RS-SPASM proteins are typically found in peptide or protein modification pathways ( 10 , 11 ). In some cases, the RS-SPASM protein is fused to a ∼90 amino acid protein ( e.g. AlbA) or, alternatively, the RS-SPASM gene is located adjacent to a gene encoding for a ∼90 amino acid protein ( e.g. PqqE). The function of the small protein and its relationship to RS-SPASM proteins remains unknown and, its role in the PQQ biosynthetic pathway is the focus of this paper.

PQQ (4,5-dihydro-4,5-dioxo-1 H -pyrrolo[2,3- f ] quinolone-2,7,9-tricarboxylic acid) is a redox active tricyclic o -quinone that serves as a cofactor, predominantly, for prokaryotic alcohol and sugar dehydrogenases ( 12 , 13 ). Biosynthesis of PQQ relies on five strictly conserved gene products, pqqABCDE ( 8 , 9 , 14 – 16 ). In all cases, pqqA encodes for a short peptide, typically 20–30 amino acids in length, containing a conserved glutamate and tyrosine that serve as the backbone in PQQ biogenesis ( Fig. 1 ). The glutamate and tyrosine undergo post-translational modifications to form the intermediate 3 a -(2-amino-2-carboxyethyl)-4,5-dioxo-4,5,6,7,8,9-hexahydroquinoline-7,9-dicarboxylic acid (AHQQ). Of the pqq gene products, PqqC is the most characterized and has been shown to catalyze the eight-electron oxidation and ring cyclization of AHQQ to form PQQ ( 17 , 18 ). The functions of the remaining pqq gene products remain elusive, although, a recent bioinformatic analysis proposed that PqqB, a metallo-β-lactamase-fold member, oxidizes some form of intermediate that leads to the formation of AHQQ ( 15 ). PqqE is an RS-SPASM protein and has been shown to cleave S -adenosylmethionine to methionine and 5′-deoxyadenosine in an uncoupled reaction ( 19 ). PqqE is proposed to catalyze the C-C bond formation between the glutamate and tyrosine; however, efforts to couple S -adenosylmethionine cleavage to the cross-linking have thus far been unsuccessful ( 19 ).

PqqD is a small protein (∼90 amino acids) with no detectable cofactor and few strictly conserved residues. PqqD can be found as an individual protein, as a C-terminal gene fusion to PqqC ( e.g. Methylobacterium extorquens AM1 PqqCD) or as a N-terminal gene fusion to PqqE ( e.g. Methylocystis sp. SC2 PqqDE). A crystal structure has been solved for Xanthomonas campestris PqqD (XcPqqD, PDB number 3G2B). Analysis of the structure indicates that XcPqqD forms a saddle-like dimer under crystallization conditions ( Fig. 2 ) but little functional information could be derived ( 20 ). A bioinformatic analysis on PqqD could not identify functionally relevant residues ( 15 ), adding to the ambiguity of the role that PqqD plays in PQQ biogenesis. However, recent electron paramagnetic resonance (EPR) and hydrogen-deuterium exchange mass spectrometry experiments have shown that Klebsiella pneumonia PqqE (KpPqqE) and PqqD (KpPqqD) interact, suggesting that the formation of a macromolecular complex may lead to function of one or more enzymes ( 21 ).

Although the formation of a KpPqqD-KpPqqE complex has been reported ( 19 ), a detailed biophysical characterization of the interaction has not yet been completed. To learn more about the interaction, we determined the stoichiometry of the KpPqqD-KpPqqE complex using native mass spectrometry experiments. We expanded our investigations into M.extorquens AM1 (Me) and determined the stoichiometry and dissociation constant ( K D ) for the MePqqD-MePqqE interaction. We also determined, through various biophysical techniques, that MePqqD binds the peptide MePqqA with a submicromolar K D independent of the presence of MePqqE. Additionally, we were able to capture the MePqqADE ternary complex by mass spectrometry and establish the K D for the MePqqAD-MePqqE interaction. Last, we use small angle x-ray scattering to demonstrate that the KpPqqD adopts a different oligomerization and conformational state than was found for the XcPqqD crystal.

DISCUSSION

PqqD has been shown, through gene knock-out and bioinformatic studies, to be an essential component for PQQ biosynthesis ( 8 , 14 , 15 ). The crystal structure for XcPqqD indicates that PqqD is a dimer and may form a scaffold for a larger macromolecular complex ( 15 ). When we purified KpPqqD, however, we found that the protein eluted from the SEC column later than was expected for the size of the dimer. Upon further analytical SEC analysis we found a molecular weight for KpPqqD that was closer to that of monomer than a dimer. We therefore used SAXS as a means to determine the solution molecular weight and, thereby, the oligomerization state of KpPqqD. SAXS has been used to determine the molecular weight of proteins with an average error of 5% ( 29 ). This technique provides strong support for the existence of KpPqqD as a monomer.

The difference in oligomerization states for crystallized XcPqqD and solution KpPqqD warranted a further analysis. In the crystal, XcPqqD adopts a saddle-like dimeric conformation with each monomer, containing an α-helix bundle and an extended β1-β2 hairpin feature, interlocked with the another monomer by H-bonds and hydrophobic interactions in an elongated β3 strand region ( 20 ). By contrast, in solution KpPqqD adopts a monomeric globular structure that is quite different from the crystallized dimer or the monomer of XcPqqD. When the XcPqqD monomer is fit into the volume of the KpPqqD solution structure it becomes evident that the β1-β2 hairpin adopts an extended conformation. We postulated that the positions of the β1 and β2 strands are compacted in relationship to the α-helix bundle of KpPqqD and that the dimerization state, or domain swapped state, may either be an innate property to XcPqqD or the result of the crystallographic conditions. We proceeded to generate a model of a compact XcPqqD monomer consisting of the β1 and β2 strands from one polypeptide chain and the β3, α1, α2, and α3 motifs from the second polypeptide chain. The theoretical SAXS profile matches the experimental SAXS profile for KpPqqD quite well. Follow up experiments will be necessary to interrogate whether the crystal structure of XcPqqD has implications for a relevant conformational alteration that could facilitate the formation of a tight complex with PqqA.

As mentioned before, the KpPqqD-KpPqqE interaction had previously been discovered, although the K D , stoichiometry, and how it relates to PQQ biosynthesis were not determined. We re-examined the interaction by using native mass spectrometry to derive the stoichiometric ratios for the KpPqqD-KpPqqE complex and found the molar ratio to be 1:1. We could not probe for other protein-protein interactions within the K. pneumonae PQQ pathway due to solubility issues with KpPqqB and the peptide substrate KpPqqA, prompting the move to the M. extorquens AM1 PQQ pathway. In M. exotorquens AM1, the PQQ biosynthetic pathway consists of the operon pqqAB ( CD ) E where pqqC and pqqD are a gene fusion. All M. extorquens AM1 pqq genes were cloned into E. coli expression vectors and were found to be soluble.

Native mass spectrometry experiments were used to verify a MePqqD-MePqqE complex and clearly demonstrated that the interaction transcends species. Moreover, the mass spectra confirmed the 1:1 molar ratio of the MePqqD-MePqqE complex. The K D for the MePqqD-MePqqE interaction, determined by SPR, is found to be ∼12 μ m . The relatively weak K D may suggest that the MePqqD-MePqqE interaction is transient ( 39 ). We attempted to determine the stoichiometry of the complex by analytical SEC but found that the complex did not remain intact under these conditions. Using the as isolated form of MePqqB, no interactions have been detected between MePqqD or MePqqE through native mass spectrometry. We did not probe for any protein interactions with PqqC because gene knock-out studies have shown that in the absence of PqqC, the intermediate AHQQ is still formed ( 8 ). The fact that AHQQ can be formed decreases the likelihood that a functionally relevant protein complex with PqqC is necessary. The native mass spectral data suggest that the major protein-protein interaction in the PQQ pathway is between MePqqD and MePqqE.

In a brute force effort to discover the binding partner for the peptide PqqA, we utilized SPR and ITC to screen for binding events between MePqqA and MePqqB, MePqqE, and MePqqD. Surprisingly, we find that MePqqD binds MePqqA tightly ( K D ∼ 200–300 n m depending on the construct). In contrast, evidence for peptide binding by MePqqE or MePqqB was not found. Significantly, these experiments provide the first evidence that PqqD is the PqqA binding partner. Moreover, to the best of our knowledge, this is the first instance that a PqqD homologue has been shown to bind a peptide.

Taking into account the interaction between MePqqD and MePqqE, we also probed for a MePqqA-D-E complex. Native mass spectrometry experiments provided evidence for a ternary complex and the molar ratio was determined to be 1:1:1. The K D between the MePqqAD complex and MePqqE is 4.5 μ m , by a dual injection SPR technique, about the same as the K D determined for MePqqD and MePqqE. The similar K D values between MePqqE and MePqqD or MePqqAD suggest that the presence of the peptide does not assist in PqqD-PqqE complex formation. The binding data determined here provides a new picture of the first steps in PQQ biosynthesis. Our new hypothesis is that PqqD first binds PqqA to protect the peptide from premature proteolytic cleavage and/or to present the glutamate and tyrosine to PqqE. PqqE then binds the PqqAD complex and catalyzes the formation of the carbon-carbon bond between the glutamate and tyrosine.

The idea that the RS-SPASM protein directly acts on the peptide is consistent with other post-translational pathways such as in subtilosin biosynthesis. In subtilosin biosynthesis, the peptide is directly modified by AlbA ( 40 ). Moreover, our bioinformatic analysis supports, in part, the notion that a functionally relevant interaction between a PqqD homologue and an RS-SPASM protein exists in peptide modification pathways. Not only does AlbA contain a N-terminal PqqD orthologue, other RS-SPASM proteins can be found with a PqqD orthologue fused to or near the RS-SPASM gene ( e.g. the putative mycofactocin pathway, the Pep1357C pathway, and in the putative 6 cysteines in 45 modification pathway). The exact role of the PqqD homologues in other peptide modification pathways will need to be addressed; however, from the work presented here, we propose that PqqD is a peptide chaperone that forms a stable complex with PqqA.
