---
pmid: '24105743'
title: Oncogenic RAS directs silencing of tumor suppressor genes through ordered recruitment
  of transcriptional repressors.
authors:
- Wajapeyee N
- Malonia SK
- Palakurthy RK
- Green MR
journal: Genes Dev
year: '2013'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3814642
doi: 10.1101/gad.227413.113
---

# Oncogenic RAS directs silencing of tumor suppressor genes through ordered recruitment of transcriptional repressors.
**Authors:** Wajapeyee N, Malonia SK, Palakurthy RK, Green MR
**Journal:** Genes Dev (2013)
**DOI:** [10.1101/gad.227413.113](https://doi.org/10.1101/gad.227413.113)
**PMC:** [PMC3814642](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3814642/)

## Abstract

1. Genes Dev. 2013 Oct 15;27(20):2221-6. doi: 10.1101/gad.227413.113. Epub 2013
Oct  8.

Oncogenic RAS directs silencing of tumor suppressor genes through ordered 
recruitment of transcriptional repressors.

Wajapeyee N(1), Malonia SK, Palakurthy RK, Green MR.

Author information:
(1)Department of Pathology, Yale University School of Medicine, New Haven, 
Connecticut 06520, USA;

We previously identified 28 cofactors through which a RAS oncoprotein directs 
transcriptional silencing of Fas and other tumor suppressor genes (TSGs). Here 
we performed RNAi-based epistasis experiments and found that RAS-directed 
silencing occurs through a highly ordered pathway that is initiated by binding 
of ZFP354B, a sequence-specific DNA-binding protein, and culminates in 
recruitment of the DNA methyltransferase DNMT1. RNAi and pharmacological 
inhibition experiments reveal that silencing requires continuous function of RAS 
and its cofactors and can be rapidly reversed, which may have therapeutic 
implications for reactivation of silenced TSGs in RAS-positive cancers.

DOI: 10.1101/gad.227413.113
PMCID: PMC3814642
PMID: 24105743 [Indexed for MEDLINE]

## Full Text

The conversion of a normal cell to a cancer cell is a stepwise process that typically involves the activation of oncogenes and inactivation of tumor suppressor genes (TSGs) (Hanahan and Weinberg 2011). There are two general mechanisms by which TSGs are inactivated. First, the TSG can acquire a deletion or mutation that abrogates the function of the encoded protein (Berger et al. 2011; Vogelstein et al. 2013). Second, the TSG can become transcriptionally silenced by a process commonly referred to as “epigenetic silencing” (Kulis and Esteller 2010). Transcriptionally silenced TSGs have characteristic features of heterochromatin, including inhibitory histone modifications and hypermethylated DNA regions.

Formally, epigenetic gene regulation refers to a change in gene expression that occurs in the absence of any change in DNA sequence and can be inherited in the absence of the signal (or event) that initiated the change (Ptashne 2007). Whether transcriptional inactivation of TSGs is truly epigenetic (i.e., whether it can be inherited in the absence of the initiating signal) remains to be determined.

As a model system for studying transcriptional inactivation of TSGs, we studied silencing of Fas in oncogenic RAS-transformed cells (Gazin et al. 2007). Expression of a RAS oncoprotein in mouse NIH 3T3 cells transcriptionally silences Fas, thereby preventing Fas ligand-induced apoptosis (Peli et al. 1999). Previously, we performed a genome-wide RNAi screen to identify 28 cofactors required for RAS-mediated silencing of Fas (Gazin et al. 2007). We further showed that a number of these factors are directly associated with specific regions of Fas in Kras-transformed NIH 3T3 cells but not in untransformed NIH 3T3 cells.

Here we used RNAi to perform experiments analogous to classical epistasis analyses to order the 28 cofactors into a pathway. Based on this information, we went on to study the initiation, maintenance, reversibility, and kinetics of RAS-directed silencing of TSGs.

Results and DiscussionWe previously derived and characterized a series of 28 Kras NIH 3T3 knockdown cell lines, each of which is depleted for one of the 28 cofactors required for Fas silencing, resulting in Fas reactivation (Gazin et al. 2007). We further demonstrated that the shRNAs used to derive these cell lines efficiently and specifically knock down their target gene (Supplemental Fig. S1; Gazin et al. 2007). To order the 28 components into a pathway, we monitored binding of 12 cofactors that are stably associated with transcriptionally silenced Fas in the 28 Kras NIH 3T3 knockdown cell lines. We reasoned that if a Kras NIH 3T3 knockdown cell line supports binding of a particular cofactor, then the gene knocked down in that cell line is dispensable for binding and can be placed downstream in the pathway. In contrast, if a Kras NIH 3T3 knockdown cell line fails to support binding of a particular cofactor, then the gene knocked down in the cell line is required for binding and can be placed upstream in the pathway.We monitored cofactor binding to Fas using three sets of promoter-specific primer pairs that cover the entire Fas promoter region: ∼2 kb upstream of the transcription start site (TSS), ∼1 kb upstream of the TSS, or encompassing the core promoter/TSS (Gazin et al. 2007). As specificity controls, we monitored binding of each cofactor to three irrelevant DNA regions and also analyzed enrichment using an irrelevant antibody (Supplemental Fig. S2). The chromatin immunoprecipitation (ChIP) results of Figure 1A show that of the 12 DNA-binding events analyzed, binding of ZFP354B, a zinc finger protein that contains a KRAB transcriptional repressor domain, was dependent on the fewest additional cofactors. Only three cofactors (mitogen-activated protein kinase 1 [MAPK1], PDPK1, and S100Z), all of which have been implicated as cell signaling proteins (Gribenko et al. 2001; Downward 2003), were required for binding of ZFP354B to transcriptionally silenced Fas in Kras NIH 3T3 cells. In contrast, binding of the DNA methyltransferase DNMT1 was dependent on all of the other 27 cofactors. The other 10 DNA-binding events analyzed showed a cofactor dependence that was intermediate to that of ZFP354B and DNMT1. The ChIP results of Figure 1A enabled us to construct a pathway that is summarized in Figure 1B and discussed below.Figure 1.Delineation of a RAS-directed transcriptional silencing pathway. (A) ChIP analysis monitoring binding of 12 cofactors to Fas in each of the Kras NIH 3T3 knockdown (KD) cell lines. The results are shown relative to that obtained in NIH 3T3 cells, which was set to 1. Error bars indicate SEM. (B) Schematic of the RAS-directed silencing pathway. For steps at which the order of cofactors cannot be distinguished, the cofactors are aligned horizontally.Two of the cofactors, EZH2 and EED, are subunits of Polycomb repressive complex 2, which confers transcriptional repression through histone H3 Lys 27 trimethylation (H3K27me3) (Margueron and Reinberg 2011). We therefore measured H3K27me3 levels on Fas in the 28 Kras NIH 3T3 knockdown cell lines. The results of Supplemental Figure S3 indicate that knockdown of EZH2 or EED and all factors upstream of EZH2 and EED resulted in loss of H3K27me3. In contrast, knockdown of components downstream from EZH2 and EED did not affect H3K27me3.In our previous study, we showed that most of the 28 cofactors were also required for transcriptional silencing of several other TSGs in Kras NIH 3T3 cells (Gazin et al. 2007). To determine the generality of the pathway, we performed RNAi-based epistasis experiments for Sfrp1, one of the other TSGs analyzed in our previous study for which 25 of the 28 cofactors were required for silencing.Supplemental Figure S4 shows the results of ChIP experiments analyzing binding of five representative cofactors (ZFP354B, CTCF, EZH2, BMI1, and DNMT1), which act at distinct steps of the pathway, in 25 Kras NIH 3T3 knockdown cell lines. Significantly, in all cases, the results on Sfrp1 were entirely consistent with those obtained with Fas.The order of the RAS-directed transcriptional silencing pathway described above indicated that ZFP354B engages in the first sequence-specific DNA-binding interaction with Fas. We showed previously that expression of activated RAS in NIH 3T3 cells results in a large increase in ZFP354B protein levels (Gazin et al. 2007). These considerations raised the possibility that binding of ZFP354B may be the critical event that is sufficient to initiate and maintain Fas silencing.To test this possibility, we asked whether increasing ZFP354B levels would result in transcriptional silencing of TSGs even in the absence of oncogenic RAS. Consistent with this idea, quantitative RT–PCR (qRT–PCR) analysis showed that ectopic expression of ZFP354B in NIH 3T3 cells (NIH 3T3/ZFP354B cells) (Supplemental Fig. S5A) resulted in substantial transcriptional repression of both Fas (Fig. 2A; see also Supplemental Fig. S5B) and Sfrp1 (Supplemental Fig. S6A).Figure 2.ZFP345B overexpression is sufficient to silence Fas in the absence of RAS. (A,B) qRT–PCR analysis monitoring Fas expression (A) and bisulfite sequence analysis of Fas (B) in NIH 3T3 cells stably expressing vector or ZFP354B. (C,D) qRT–PCR analysis monitoring Fas expression (C) and MeDIP analysis monitoring Fas DNA methylation (D) in NIH 3T3/ZFP354B knockdown (KD) cell lines. As controls, Fas expression and DNA methylation were also monitored in NIH 3T3 and Kras NIH 3T3 cells. (E) ChIP analysis monitoring binding of ZFP354B, CTCF, EZH2, BMI1, and DNMT1 to Fas in NIH 3T3 cells stably expressing vector or ZFP354B. Error bars indicate SEM.We next performed a series of experiments to compare transcriptional repression resulting from ZFP354B overexpression with that resulting from oncogenic RAS. We showed previously that in Kras NIH 3T3 cells, the transcriptionally silenced TSGs are hypermethylated (Gazin et al. 2007). We performed bisulfite sequencing in NIH 3T3/ZFP354B cells, analyzing the same Fas and Sfrp1 regions previously found to be hypermethylated in Kras NIH 3T3 cells (Gazin et al. 2007). Bisulfite sequence analysis showed that in NIH 3T3/ZFP354B cells, both Fas (Fig. 2B) and Sfrp1 (Supplemental Fig. S6B) were hypermethylated.We next asked whether transcriptional repression in NIH 3T3/ZFP354B cells was also dependent on cofactors that functioned downstream from ZFP354B. As in Kras NIH 3T3 cells (Gazin et al. 2007), knockdown of the representative downstream cofactor CTCF, EZH2, BMI1, or DNMT1 in NIH 3T3/ZFP354B cells reactivated Fas (Fig. 2C) and Sfrp1 (Supplemental Fig. S6C).We also assessed DNA methylation following knockdown using a methylated DNA immunoprecipitation (MeDIP) assay. As expected, increased Fas or Sfrp1 transcription following knockdown of CTCF, EZH2, BMI1, or DNMT1 in NIH 3T3/ZFP354B cells was accompanied by decreased DNA methylation (Fig. 2D; Supplemental Fig. S6D). Finally, ChIP analysis showed that in NIH 3T3/ZFP354B cells, as in Kras NIH 3T3 cells, ZFP354B itself as well as CTCF, EZH2, BMI1, and DNMT1 were bound to the transcriptionally silenced Fas (Fig. 2E) and Sfrp1 (Supplemental Fig. S6E) genes. Collectively, the results of Figure 2 and Supplemental Figure S6 demonstrate that overexpression of ZFP354B is sufficient to initiate transcriptional silencing of TSGs through a pathway similar to that directed by oncogenic RAS.Oncogenic RAS stimulates several downstream signaling pathways, including the MAPK and phosphoinositide 3-kinase (PI3K)/AKT pathways (De Luca et al. 2012). To understand in greater detail the basis of RAS-mediated silencing of Fas, we first analyzed activating HRAS mutants that are defective for signaling through either the MAPK pathway [HRAS(12V,40C)] or the PI3K/AKT pathway [HRAS(12V,35S)] (White et al. 1995; Rodriguez-Viciana et al. 1997; Hamad et al. 2002). The results of Figure 3A show that neither mutant was able to promote Fas silencing, indicating the requirement for both the MAPK and PI3K/AKT pathways.Figure 3.Kinetics of RAS-directed transcriptional silencing. (A) qRT–PCR analysis monitoring Fas expression in NIH 3T3 cells expressing HRAS mutants. The results were normalized to Fas expression in NIH 3T3 cells, which was set to 1. (B) qRT–PCR analysis monitoring Fas expression in Kras NIH 3T3 cells treated in the absence and presence of U0126. (C,D) qRT–PCR analysis monitoring Fas expression in Kras NIH 3T3 cells following U0126 addition (C) or withdrawal (D). Fas expression is shown relative to that observed in untreated KRas NIH 3T3 cells, which was set to 1. (E,F) ChIP analysis monitoring binding of ZFP354B, EZH2, and DNMT1 to Fas in Kras NIH 3T3 cells following U0126 addition (E) or withdrawal (F). Binding in Kras NIH 3T3 cells is defined as 100% occupancy. (G,H) ChIP analysis monitoring H3K27me3 and DNA methylation on Fas in Kras NIH 3T3 cells following U0126 addition (G) or withdrawal (H). The results were normalized to that obtained in NIH 3T3 cells, which was set to 1. (I,J) Immunoblot analysis showing ZFP354B, phosphorylated ERK1/2 (p-ERK1/2), and total ERK1/2 (t-ERK1/2) levels in Kras NIH 3T3 cells following U0126 addition (I) or withdrawal (J). Error bars indicate SEM.To confirm this conclusion, we analyzed chemical inhibitors of these signaling pathways. Kras NIH 3T3 cells were treated with a chemical inhibitor of either MAPK signaling (U0126, a selective inhibitor of MEK1 and MEK2) (Favata et al. 1998) or PI3K/AKT signaling (LY294002, a selective PI3K inhibitor) (Vlahos et al. 1994), and Fas expression was analyzed by qRT–PCR. The results of Figure 3B and Supplemental Figure S7A show that both inhibitors reactivated Fas expression, confirming that both the PI3K/AKT and MAPK pathways are required for RAS-directed transcriptional silencing of Fas. Consistent with this conclusion, among the 28 cofactors are PDPK1, a regulator of PI3K/AKT signaling (Raimondi and Falasca 2011), and MAPK1 and MAP3K9, regulators of MAPK signaling (Morrison 2012).The availability of pharmacological inhibitors enabled us to study the reversibility and kinetics of RAS-directed transcriptional silencing. Figure 3C and Supplemental Figure S7B show that following addition of U0126 or LY294002 to Kras NIH 3T3 cells, Fas reactivation occurred within 24 or 36 h, respectively. We next asked whether removal of the drugs would result in restoration of Fas silencing. Figure 3D and Supplemental Figure S7C show that following removal of U0126 or LY294002, Fas silencing was restored over a 48-h time course. These results indicate that RAS-directed silencing of Fas occurs rapidly and is highly reversible.To further study the reversibility and kinetics of RAS-directed silencing of Fas, we analyzed representative cofactor binding as well as H3K27me3 and DNA methylation. The ChIP results of Figure 3E and Supplemental Figure S7D show that following addition of U0126 or LY294002, there was a progressive decrease in binding of ZFP354B, EZH2, and DNMT1 over 24–36 h, which correlated well with the kinetics of transcriptional reactivation (Fig. 3C; Supplemental Fig. S7B). Likewise, Figure 3F and Supplemental Figure S7E show that removal of U0126 or LY294002 resulted in a progressive increase in binding of ZFP354B, EZH2, and DNMT1 over ∼48 h, which correlated well with the kinetics of transcriptional silencing (Fig. 3D; Supplemental Fig. S7C). Notably, the loss of H3K27me3 and DNA methylation following addition of U0126 (Fig. 3G) or LY294002 (Supplemental Fig. S7F) and the acquisition of H3K27me3 and DNA methylation following removal of U0126 (Fig. 3H) or LY294002 (Supplemental Fig. S7G) were also well correlated with transcription and cofactor binding.Finally, as expected, ZFP354B levels decreased following addition of U0126 (Fig. 3I) or LY294002 (Supplemental Fig. S7H) and increased following withdrawal of U0126 (Fig. 3J) or LY294002 (Supplemental Fig. S7H), with kinetics that correlated well with that of transcriptional activity.Two mechanisms for DNA demethylation have been proposed (Wu and Zhang 2010). The first is the so-called passive mechanism, in which methyl groups are lost simply as a result of DNA replication. The second is an active mechanism in which a DNA demethylase catalyzes the removal of the methyl groups. The relatively rapid induction of Fas transcription and demethylation following pharmacological inhibition of RAS suggested that DNA demethylation was unlikely to result from a passive mechanism. Consistent with this possibility, when passive DNA methylation was blocked by addition of the DNA replication inhibitor aphidicolin, U0126 or LY294002 treatment still reactivated Fas expression (Supplemental Fig. S8A), which was accompanied by decreased methylation of Fas (Supplemental Fig. S8B).In this study, we showed how RNAi-based epistasis analysis can be used to order a defined set of components into a molecular pathway. The pathway is initiated by RAS, which then functions through a set of cell signaling proteins (MAPK1, PDPK1, and S100Z). The first DNA-binding event on Fas is by ZFP354B, followed by recruitment of additional DNA-binding proteins, multisubunit complexes, chromatin-modifying activities, and, finally, DNTM1 (Fig. 4). Collectively, these results indicate that RAS-directed silencing of Fas is a highly ordered process that ultimately establishes a platform for DNMT1 recruitment. This pathway of cofactor binding provides the underlying basis for a corresponding ordered establishment of repressive marks, including H3K27me3 and DNA methylation.Figure 4.Model for RAS-directed silencing of TSGs. (Top) RAS signaling through the MAPK and PI3K/AKT pathways results in increased levels of ZFP354B, which binds to the TSG promoter and initiates recruitment of other cofactors (which may or may not bind directly to ZFP354B). Recruitment of DNMT1 leads to methylation of the promoter and silencing of TSG expression. (Bottom) The RAS-directed signaling pathway can be targeted through pharmacological inhibition of MAPK or PI3K/AKT, which leads to reduced levels of ZFP354B. In the absence of ZFP354B binding, association of other cofactors with the promoter cannot be maintained, and, in conjunction with rapid DNA demethylation, the TSG is re-expressed.RAS initiates and maintains silencing by regulating levels of ZFP354B, which is the first cofactor to interact with Fas (Fig. 4). We note that all 28 cofactors may not act directly on Fas. Some cofactors, for example, may function by regulating expression or activity of other cofactors. Accordingly, we showed previously that PDPK1 regulates ZFP354B levels (Gazin et al. 2007).Although we used RAS-transformed NIH 3T3 cells as an experimental system, for several reasons, we believe that our results have relevance to human cancers. For example, as in murine cells, activated RAS silences FAS in human cells (Urquhart et al. 2002; Gazin et al. 2007). Moreover, FAS silencing also occurs in some transformed cells, human tumors, and mouse models of cancer and has been shown to be relevant to both tumor progression (for example, see Hopkins-Donaldson et al. 2003) and chemotherapeutic resistance (Maecker et al. 2002). In addition, we showed previously that this same pathway also mediates silencing of other TSGs, including Lox, Par4/Pawr, and Plagl1, which have been found to be relevant to cellular transformation and cancer (for discussion, see Gazin et al. 2007). Finally, several of the components of the pathway that we describe have been shown to cooperate with RAS in transformation of human cells (Croonquist and Van Ness 2005; Datta et al. 2007) or are overexpressed in human cancers and contribute to the transformed phenotype (Chang and Hung 2012; Jin and Robertson 2013).Our results indicate that RAS-directed transcriptional silencing of TSGs is not truly epigenetic because RAS is required for not only initiation of the pathway but also maintenance of repression. The continual requirement for RAS and the components of the RAS-directed silencing pathway and the rapid reversibility of TSG silencing may have therapeutic implications. The components through which RAS and other oncoproteins direct TSG silencing in human cancers can be identified using functional genomic approaches such as those we described and represent potential anti-cancer targets.

Results and Discussion

We previously derived and characterized a series of 28 Kras NIH 3T3 knockdown cell lines, each of which is depleted for one of the 28 cofactors required for Fas silencing, resulting in Fas reactivation (Gazin et al. 2007). We further demonstrated that the shRNAs used to derive these cell lines efficiently and specifically knock down their target gene (Supplemental Fig. S1; Gazin et al. 2007). To order the 28 components into a pathway, we monitored binding of 12 cofactors that are stably associated with transcriptionally silenced Fas in the 28 Kras NIH 3T3 knockdown cell lines. We reasoned that if a Kras NIH 3T3 knockdown cell line supports binding of a particular cofactor, then the gene knocked down in that cell line is dispensable for binding and can be placed downstream in the pathway. In contrast, if a Kras NIH 3T3 knockdown cell line fails to support binding of a particular cofactor, then the gene knocked down in the cell line is required for binding and can be placed upstream in the pathway.

We monitored cofactor binding to Fas using three sets of promoter-specific primer pairs that cover the entire Fas promoter region: ∼2 kb upstream of the transcription start site (TSS), ∼1 kb upstream of the TSS, or encompassing the core promoter/TSS (Gazin et al. 2007). As specificity controls, we monitored binding of each cofactor to three irrelevant DNA regions and also analyzed enrichment using an irrelevant antibody (Supplemental Fig. S2). The chromatin immunoprecipitation (ChIP) results of Figure 1A show that of the 12 DNA-binding events analyzed, binding of ZFP354B, a zinc finger protein that contains a KRAB transcriptional repressor domain, was dependent on the fewest additional cofactors. Only three cofactors (mitogen-activated protein kinase 1 [MAPK1], PDPK1, and S100Z), all of which have been implicated as cell signaling proteins (Gribenko et al. 2001; Downward 2003), were required for binding of ZFP354B to transcriptionally silenced Fas in Kras NIH 3T3 cells. In contrast, binding of the DNA methyltransferase DNMT1 was dependent on all of the other 27 cofactors. The other 10 DNA-binding events analyzed showed a cofactor dependence that was intermediate to that of ZFP354B and DNMT1. The ChIP results of Figure 1A enabled us to construct a pathway that is summarized in Figure 1B and discussed below.

Delineation of a RAS-directed transcriptional silencing pathway. (A) ChIP analysis monitoring binding of 12 cofactors to Fas in each of the Kras NIH 3T3 knockdown (KD) cell lines. The results are shown relative to that obtained in NIH 3T3 cells, which was set to 1. Error bars indicate SEM. (B) Schematic of the RAS-directed silencing pathway. For steps at which the order of cofactors cannot be distinguished, the cofactors are aligned horizontally.

Two of the cofactors, EZH2 and EED, are subunits of Polycomb repressive complex 2, which confers transcriptional repression through histone H3 Lys 27 trimethylation (H3K27me3) (Margueron and Reinberg 2011). We therefore measured H3K27me3 levels on Fas in the 28 Kras NIH 3T3 knockdown cell lines. The results of Supplemental Figure S3 indicate that knockdown of EZH2 or EED and all factors upstream of EZH2 and EED resulted in loss of H3K27me3. In contrast, knockdown of components downstream from EZH2 and EED did not affect H3K27me3.

In our previous study, we showed that most of the 28 cofactors were also required for transcriptional silencing of several other TSGs in Kras NIH 3T3 cells (Gazin et al. 2007). To determine the generality of the pathway, we performed RNAi-based epistasis experiments for Sfrp1, one of the other TSGs analyzed in our previous study for which 25 of the 28 cofactors were required for silencing.

Supplemental Figure S4 shows the results of ChIP experiments analyzing binding of five representative cofactors (ZFP354B, CTCF, EZH2, BMI1, and DNMT1), which act at distinct steps of the pathway, in 25 Kras NIH 3T3 knockdown cell lines. Significantly, in all cases, the results on Sfrp1 were entirely consistent with those obtained with Fas.

The order of the RAS-directed transcriptional silencing pathway described above indicated that ZFP354B engages in the first sequence-specific DNA-binding interaction with Fas. We showed previously that expression of activated RAS in NIH 3T3 cells results in a large increase in ZFP354B protein levels (Gazin et al. 2007). These considerations raised the possibility that binding of ZFP354B may be the critical event that is sufficient to initiate and maintain Fas silencing.

To test this possibility, we asked whether increasing ZFP354B levels would result in transcriptional silencing of TSGs even in the absence of oncogenic RAS. Consistent with this idea, quantitative RT–PCR (qRT–PCR) analysis showed that ectopic expression of ZFP354B in NIH 3T3 cells (NIH 3T3/ZFP354B cells) (Supplemental Fig. S5A) resulted in substantial transcriptional repression of both Fas (Fig. 2A; see also Supplemental Fig. S5B) and Sfrp1 (Supplemental Fig. S6A).

ZFP345B overexpression is sufficient to silence Fas in the absence of RAS. (A,B) qRT–PCR analysis monitoring Fas expression (A) and bisulfite sequence analysis of Fas (B) in NIH 3T3 cells stably expressing vector or ZFP354B. (C,D) qRT–PCR analysis monitoring Fas expression (C) and MeDIP analysis monitoring Fas DNA methylation (D) in NIH 3T3/ZFP354B knockdown (KD) cell lines. As controls, Fas expression and DNA methylation were also monitored in NIH 3T3 and Kras NIH 3T3 cells. (E) ChIP analysis monitoring binding of ZFP354B, CTCF, EZH2, BMI1, and DNMT1 to Fas in NIH 3T3 cells stably expressing vector or ZFP354B. Error bars indicate SEM.

We next performed a series of experiments to compare transcriptional repression resulting from ZFP354B overexpression with that resulting from oncogenic RAS. We showed previously that in Kras NIH 3T3 cells, the transcriptionally silenced TSGs are hypermethylated (Gazin et al. 2007). We performed bisulfite sequencing in NIH 3T3/ZFP354B cells, analyzing the same Fas and Sfrp1 regions previously found to be hypermethylated in Kras NIH 3T3 cells (Gazin et al. 2007). Bisulfite sequence analysis showed that in NIH 3T3/ZFP354B cells, both Fas (Fig. 2B) and Sfrp1 (Supplemental Fig. S6B) were hypermethylated.

We next asked whether transcriptional repression in NIH 3T3/ZFP354B cells was also dependent on cofactors that functioned downstream from ZFP354B. As in Kras NIH 3T3 cells (Gazin et al. 2007), knockdown of the representative downstream cofactor CTCF, EZH2, BMI1, or DNMT1 in NIH 3T3/ZFP354B cells reactivated Fas (Fig. 2C) and Sfrp1 (Supplemental Fig. S6C).

We also assessed DNA methylation following knockdown using a methylated DNA immunoprecipitation (MeDIP) assay. As expected, increased Fas or Sfrp1 transcription following knockdown of CTCF, EZH2, BMI1, or DNMT1 in NIH 3T3/ZFP354B cells was accompanied by decreased DNA methylation (Fig. 2D; Supplemental Fig. S6D). Finally, ChIP analysis showed that in NIH 3T3/ZFP354B cells, as in Kras NIH 3T3 cells, ZFP354B itself as well as CTCF, EZH2, BMI1, and DNMT1 were bound to the transcriptionally silenced Fas (Fig. 2E) and Sfrp1 (Supplemental Fig. S6E) genes. Collectively, the results of Figure 2 and Supplemental Figure S6 demonstrate that overexpression of ZFP354B is sufficient to initiate transcriptional silencing of TSGs through a pathway similar to that directed by oncogenic RAS.

Oncogenic RAS stimulates several downstream signaling pathways, including the MAPK and phosphoinositide 3-kinase (PI3K)/AKT pathways (De Luca et al. 2012). To understand in greater detail the basis of RAS-mediated silencing of Fas, we first analyzed activating HRAS mutants that are defective for signaling through either the MAPK pathway [HRAS(12V,40C)] or the PI3K/AKT pathway [HRAS(12V,35S)] (White et al. 1995; Rodriguez-Viciana et al. 1997; Hamad et al. 2002). The results of Figure 3A show that neither mutant was able to promote Fas silencing, indicating the requirement for both the MAPK and PI3K/AKT pathways.

Kinetics of RAS-directed transcriptional silencing. (A) qRT–PCR analysis monitoring Fas expression in NIH 3T3 cells expressing HRAS mutants. The results were normalized to Fas expression in NIH 3T3 cells, which was set to 1. (B) qRT–PCR analysis monitoring Fas expression in Kras NIH 3T3 cells treated in the absence and presence of U0126. (C,D) qRT–PCR analysis monitoring Fas expression in Kras NIH 3T3 cells following U0126 addition (C) or withdrawal (D). Fas expression is shown relative to that observed in untreated KRas NIH 3T3 cells, which was set to 1. (E,F) ChIP analysis monitoring binding of ZFP354B, EZH2, and DNMT1 to Fas in Kras NIH 3T3 cells following U0126 addition (E) or withdrawal (F). Binding in Kras NIH 3T3 cells is defined as 100% occupancy. (G,H) ChIP analysis monitoring H3K27me3 and DNA methylation on Fas in Kras NIH 3T3 cells following U0126 addition (G) or withdrawal (H). The results were normalized to that obtained in NIH 3T3 cells, which was set to 1. (I,J) Immunoblot analysis showing ZFP354B, phosphorylated ERK1/2 (p-ERK1/2), and total ERK1/2 (t-ERK1/2) levels in Kras NIH 3T3 cells following U0126 addition (I) or withdrawal (J). Error bars indicate SEM.

To confirm this conclusion, we analyzed chemical inhibitors of these signaling pathways. Kras NIH 3T3 cells were treated with a chemical inhibitor of either MAPK signaling (U0126, a selective inhibitor of MEK1 and MEK2) (Favata et al. 1998) or PI3K/AKT signaling (LY294002, a selective PI3K inhibitor) (Vlahos et al. 1994), and Fas expression was analyzed by qRT–PCR. The results of Figure 3B and Supplemental Figure S7A show that both inhibitors reactivated Fas expression, confirming that both the PI3K/AKT and MAPK pathways are required for RAS-directed transcriptional silencing of Fas. Consistent with this conclusion, among the 28 cofactors are PDPK1, a regulator of PI3K/AKT signaling (Raimondi and Falasca 2011), and MAPK1 and MAP3K9, regulators of MAPK signaling (Morrison 2012).

The availability of pharmacological inhibitors enabled us to study the reversibility and kinetics of RAS-directed transcriptional silencing. Figure 3C and Supplemental Figure S7B show that following addition of U0126 or LY294002 to Kras NIH 3T3 cells, Fas reactivation occurred within 24 or 36 h, respectively. We next asked whether removal of the drugs would result in restoration of Fas silencing. Figure 3D and Supplemental Figure S7C show that following removal of U0126 or LY294002, Fas silencing was restored over a 48-h time course. These results indicate that RAS-directed silencing of Fas occurs rapidly and is highly reversible.

To further study the reversibility and kinetics of RAS-directed silencing of Fas, we analyzed representative cofactor binding as well as H3K27me3 and DNA methylation. The ChIP results of Figure 3E and Supplemental Figure S7D show that following addition of U0126 or LY294002, there was a progressive decrease in binding of ZFP354B, EZH2, and DNMT1 over 24–36 h, which correlated well with the kinetics of transcriptional reactivation (Fig. 3C; Supplemental Fig. S7B). Likewise, Figure 3F and Supplemental Figure S7E show that removal of U0126 or LY294002 resulted in a progressive increase in binding of ZFP354B, EZH2, and DNMT1 over ∼48 h, which correlated well with the kinetics of transcriptional silencing (Fig. 3D; Supplemental Fig. S7C). Notably, the loss of H3K27me3 and DNA methylation following addition of U0126 (Fig. 3G) or LY294002 (Supplemental Fig. S7F) and the acquisition of H3K27me3 and DNA methylation following removal of U0126 (Fig. 3H) or LY294002 (Supplemental Fig. S7G) were also well correlated with transcription and cofactor binding.

Finally, as expected, ZFP354B levels decreased following addition of U0126 (Fig. 3I) or LY294002 (Supplemental Fig. S7H) and increased following withdrawal of U0126 (Fig. 3J) or LY294002 (Supplemental Fig. S7H), with kinetics that correlated well with that of transcriptional activity.

Two mechanisms for DNA demethylation have been proposed (Wu and Zhang 2010). The first is the so-called passive mechanism, in which methyl groups are lost simply as a result of DNA replication. The second is an active mechanism in which a DNA demethylase catalyzes the removal of the methyl groups. The relatively rapid induction of Fas transcription and demethylation following pharmacological inhibition of RAS suggested that DNA demethylation was unlikely to result from a passive mechanism. Consistent with this possibility, when passive DNA methylation was blocked by addition of the DNA replication inhibitor aphidicolin, U0126 or LY294002 treatment still reactivated Fas expression (Supplemental Fig. S8A), which was accompanied by decreased methylation of Fas (Supplemental Fig. S8B).

In this study, we showed how RNAi-based epistasis analysis can be used to order a defined set of components into a molecular pathway. The pathway is initiated by RAS, which then functions through a set of cell signaling proteins (MAPK1, PDPK1, and S100Z). The first DNA-binding event on Fas is by ZFP354B, followed by recruitment of additional DNA-binding proteins, multisubunit complexes, chromatin-modifying activities, and, finally, DNTM1 (Fig. 4). Collectively, these results indicate that RAS-directed silencing of Fas is a highly ordered process that ultimately establishes a platform for DNMT1 recruitment. This pathway of cofactor binding provides the underlying basis for a corresponding ordered establishment of repressive marks, including H3K27me3 and DNA methylation.

Model for RAS-directed silencing of TSGs. (Top) RAS signaling through the MAPK and PI3K/AKT pathways results in increased levels of ZFP354B, which binds to the TSG promoter and initiates recruitment of other cofactors (which may or may not bind directly to ZFP354B). Recruitment of DNMT1 leads to methylation of the promoter and silencing of TSG expression. (Bottom) The RAS-directed signaling pathway can be targeted through pharmacological inhibition of MAPK or PI3K/AKT, which leads to reduced levels of ZFP354B. In the absence of ZFP354B binding, association of other cofactors with the promoter cannot be maintained, and, in conjunction with rapid DNA demethylation, the TSG is re-expressed.

RAS initiates and maintains silencing by regulating levels of ZFP354B, which is the first cofactor to interact with Fas (Fig. 4). We note that all 28 cofactors may not act directly on Fas. Some cofactors, for example, may function by regulating expression or activity of other cofactors. Accordingly, we showed previously that PDPK1 regulates ZFP354B levels (Gazin et al. 2007).

Although we used RAS-transformed NIH 3T3 cells as an experimental system, for several reasons, we believe that our results have relevance to human cancers. For example, as in murine cells, activated RAS silences FAS in human cells (Urquhart et al. 2002; Gazin et al. 2007). Moreover, FAS silencing also occurs in some transformed cells, human tumors, and mouse models of cancer and has been shown to be relevant to both tumor progression (for example, see Hopkins-Donaldson et al. 2003) and chemotherapeutic resistance (Maecker et al. 2002). In addition, we showed previously that this same pathway also mediates silencing of other TSGs, including Lox, Par4/Pawr, and Plagl1, which have been found to be relevant to cellular transformation and cancer (for discussion, see Gazin et al. 2007). Finally, several of the components of the pathway that we describe have been shown to cooperate with RAS in transformation of human cells (Croonquist and Van Ness 2005; Datta et al. 2007) or are overexpressed in human cancers and contribute to the transformed phenotype (Chang and Hung 2012; Jin and Robertson 2013).

Our results indicate that RAS-directed transcriptional silencing of TSGs is not truly epigenetic because RAS is required for not only initiation of the pathway but also maintenance of repression. The continual requirement for RAS and the components of the RAS-directed silencing pathway and the rapid reversibility of TSG silencing may have therapeutic implications. The components through which RAS and other oncoproteins direct TSG silencing in human cancers can be identified using functional genomic approaches such as those we described and represent potential anti-cancer targets.

Materials and methodsCell lines and cultureNIH 3T3 (American Type Culture Collection [ATCC] no. CRL-1658) and Kras NIH 3T3 (ATCC no. CRL-6361) cells were maintained in DMEM supplemented with 10% FCS at 37°C and 5% CO2. To derive NIH 3T3/ZFP354B cells, full-length Zfp354b was PCR-amplified from a cDNA (Open Biosystems, no. BC107400) and cloned into the vector 3xFlag-Myc-CMV-26 (Sigma). The construct was linearized and transfected into NIH 3T3 cells, which were selected with 500 μg/mL neomycin for 3 wk. To construct cell lines stably expressing RAS mutants, HEK293T cells were transfected with 2 μg of pBABE-HRAS(12V), pBABE-HRAS(12V,35S), or pBABE-HRAS(12V,40C) (Addgene). Viral supernatants were collected 48 h later and used to infect NIH 3T3 cells followed by selection with 2 μg/mL puromycin. For drug addition experiments, Kras NIH 3T3 cells were treated with dimethyl sulfoxide (DMSO), 10 μM U0126 (Cell Signaling Technology, Inc.), or 20 μM LY294002 (Calbiochem) for 48 h unless otherwise stated. For drug withdrawal experiments, cells were treated with U0126 or LY294002 for 36 or 48 h and then grown in fresh medium without inhibitors.RNAiIndividual knockdown cell lines were generated by retroviral transduction of 0.6 × 105
Kras NIH 3T3 or NIH 3T3/ZFP354B cells with the target shRNA (Supplemental Table S1).ChIP analysisChIP assays were performed as previously described (Gazin et al. 2007) using ASF1A (Millipore), BMI1 (Abcam), CTCF (Upstate Biotechnology), DNMT1 (Imgenex), EED (Millipore), EZH2 (Cell Signaling Technology), NPM2 (a gift from M.M. Matzuk), SIRT6 (Aviva Systems Biology), SOX14 (Santa Cruz Biotechnology), TRIM37 (a gift from A.E. Lehesjoki), TRIM66 (a gift from R. Losson), ZFP354B (Gazin et al. 2007), or H3K27me3 (Millipore) antibodies. Primer sequences used for amplifying ChIP products are provided in Supplemental Table S2.Normalized Ct (ΔCt) values were calculated by subtracting the Ct obtained with input DNA from that obtained with immunoprecipitated DNA [ΔCt = Ct(IP) − Ct(input)]. Relative fold enrichment of a factor at the target site was then calculated using the formula 2−[ΔCt(T)
−
ΔCt(Actb)], where ΔCt(T) and ΔCt(Actb) are ΔCt values obtained using target and Actb (irrelevant) primers, respectively.qRT–PCRTotal RNA was isolated and reverse-transcribed, and qRT–PCR was performed as described previously (Gazin et al. 2007) using primers listed in Supplemental Table S3.MeDIP analysisMeDIP assays were performed as described (Gazin et al. 2007). Relative quantification of DNA fragments for each region was determined by plotting Ct values on the standard curve. Fold difference of immunoprecipitated over input DNA was calculated to indicate enrichment levels of the target region. All assays were conducted on at least two biological replicates.Bisulfite sequencing analysisBisulfite modification and sequencing were carried out as previously described (Gazin et al. 2007) using primer sequences listed in Supplemental Table S2.Immunoblot analysisCell extracts were prepared as previously described (Santra et al. 2009). Blots were probed with ZFP354B (Gazin et al. 2007), phospho-AKT, total AKT, phospho-ERK1/2, total ERK1/2 (Cell Signaling), or α-tubulin antibodies.

Materials and methods

Cell lines and cultureNIH 3T3 (American Type Culture Collection [ATCC] no. CRL-1658) and Kras NIH 3T3 (ATCC no. CRL-6361) cells were maintained in DMEM supplemented with 10% FCS at 37°C and 5% CO2. To derive NIH 3T3/ZFP354B cells, full-length Zfp354b was PCR-amplified from a cDNA (Open Biosystems, no. BC107400) and cloned into the vector 3xFlag-Myc-CMV-26 (Sigma). The construct was linearized and transfected into NIH 3T3 cells, which were selected with 500 μg/mL neomycin for 3 wk. To construct cell lines stably expressing RAS mutants, HEK293T cells were transfected with 2 μg of pBABE-HRAS(12V), pBABE-HRAS(12V,35S), or pBABE-HRAS(12V,40C) (Addgene). Viral supernatants were collected 48 h later and used to infect NIH 3T3 cells followed by selection with 2 μg/mL puromycin. For drug addition experiments, Kras NIH 3T3 cells were treated with dimethyl sulfoxide (DMSO), 10 μM U0126 (Cell Signaling Technology, Inc.), or 20 μM LY294002 (Calbiochem) for 48 h unless otherwise stated. For drug withdrawal experiments, cells were treated with U0126 or LY294002 for 36 or 48 h and then grown in fresh medium without inhibitors.

Cell lines and culture

NIH 3T3 (American Type Culture Collection [ATCC] no. CRL-1658) and Kras NIH 3T3 (ATCC no. CRL-6361) cells were maintained in DMEM supplemented with 10% FCS at 37°C and 5% CO2. To derive NIH 3T3/ZFP354B cells, full-length Zfp354b was PCR-amplified from a cDNA (Open Biosystems, no. BC107400) and cloned into the vector 3xFlag-Myc-CMV-26 (Sigma). The construct was linearized and transfected into NIH 3T3 cells, which were selected with 500 μg/mL neomycin for 3 wk. To construct cell lines stably expressing RAS mutants, HEK293T cells were transfected with 2 μg of pBABE-HRAS(12V), pBABE-HRAS(12V,35S), or pBABE-HRAS(12V,40C) (Addgene). Viral supernatants were collected 48 h later and used to infect NIH 3T3 cells followed by selection with 2 μg/mL puromycin. For drug addition experiments, Kras NIH 3T3 cells were treated with dimethyl sulfoxide (DMSO), 10 μM U0126 (Cell Signaling Technology, Inc.), or 20 μM LY294002 (Calbiochem) for 48 h unless otherwise stated. For drug withdrawal experiments, cells were treated with U0126 or LY294002 for 36 or 48 h and then grown in fresh medium without inhibitors.

RNAiIndividual knockdown cell lines were generated by retroviral transduction of 0.6 × 105
Kras NIH 3T3 or NIH 3T3/ZFP354B cells with the target shRNA (Supplemental Table S1).

Individual knockdown cell lines were generated by retroviral transduction of 0.6 × 105
Kras NIH 3T3 or NIH 3T3/ZFP354B cells with the target shRNA (Supplemental Table S1).

ChIP analysisChIP assays were performed as previously described (Gazin et al. 2007) using ASF1A (Millipore), BMI1 (Abcam), CTCF (Upstate Biotechnology), DNMT1 (Imgenex), EED (Millipore), EZH2 (Cell Signaling Technology), NPM2 (a gift from M.M. Matzuk), SIRT6 (Aviva Systems Biology), SOX14 (Santa Cruz Biotechnology), TRIM37 (a gift from A.E. Lehesjoki), TRIM66 (a gift from R. Losson), ZFP354B (Gazin et al. 2007), or H3K27me3 (Millipore) antibodies. Primer sequences used for amplifying ChIP products are provided in Supplemental Table S2.Normalized Ct (ΔCt) values were calculated by subtracting the Ct obtained with input DNA from that obtained with immunoprecipitated DNA [ΔCt = Ct(IP) − Ct(input)]. Relative fold enrichment of a factor at the target site was then calculated using the formula 2−[ΔCt(T)
−
ΔCt(Actb)], where ΔCt(T) and ΔCt(Actb) are ΔCt values obtained using target and Actb (irrelevant) primers, respectively.

ChIP analysis

ChIP assays were performed as previously described (Gazin et al. 2007) using ASF1A (Millipore), BMI1 (Abcam), CTCF (Upstate Biotechnology), DNMT1 (Imgenex), EED (Millipore), EZH2 (Cell Signaling Technology), NPM2 (a gift from M.M. Matzuk), SIRT6 (Aviva Systems Biology), SOX14 (Santa Cruz Biotechnology), TRIM37 (a gift from A.E. Lehesjoki), TRIM66 (a gift from R. Losson), ZFP354B (Gazin et al. 2007), or H3K27me3 (Millipore) antibodies. Primer sequences used for amplifying ChIP products are provided in Supplemental Table S2.

Normalized Ct (ΔCt) values were calculated by subtracting the Ct obtained with input DNA from that obtained with immunoprecipitated DNA [ΔCt = Ct(IP) − Ct(input)]. Relative fold enrichment of a factor at the target site was then calculated using the formula 2−[ΔCt(T)
−
ΔCt(Actb)], where ΔCt(T) and ΔCt(Actb) are ΔCt values obtained using target and Actb (irrelevant) primers, respectively.

qRT–PCRTotal RNA was isolated and reverse-transcribed, and qRT–PCR was performed as described previously (Gazin et al. 2007) using primers listed in Supplemental Table S3.

Total RNA was isolated and reverse-transcribed, and qRT–PCR was performed as described previously (Gazin et al. 2007) using primers listed in Supplemental Table S3.

MeDIP analysisMeDIP assays were performed as described (Gazin et al. 2007). Relative quantification of DNA fragments for each region was determined by plotting Ct values on the standard curve. Fold difference of immunoprecipitated over input DNA was calculated to indicate enrichment levels of the target region. All assays were conducted on at least two biological replicates.

MeDIP analysis

MeDIP assays were performed as described (Gazin et al. 2007). Relative quantification of DNA fragments for each region was determined by plotting Ct values on the standard curve. Fold difference of immunoprecipitated over input DNA was calculated to indicate enrichment levels of the target region. All assays were conducted on at least two biological replicates.

Bisulfite sequencing analysisBisulfite modification and sequencing were carried out as previously described (Gazin et al. 2007) using primer sequences listed in Supplemental Table S2.

Bisulfite sequencing analysis

Bisulfite modification and sequencing were carried out as previously described (Gazin et al. 2007) using primer sequences listed in Supplemental Table S2.

Immunoblot analysisCell extracts were prepared as previously described (Santra et al. 2009). Blots were probed with ZFP354B (Gazin et al. 2007), phospho-AKT, total AKT, phospho-ERK1/2, total ERK1/2 (Cell Signaling), or α-tubulin antibodies.

Immunoblot analysis

Cell extracts were prepared as previously described (Santra et al. 2009). Blots were probed with ZFP354B (Gazin et al. 2007), phospho-AKT, total AKT, phospho-ERK1/2, total ERK1/2 (Cell Signaling), or α-tubulin antibodies.
