---
pmid: '32492414'
title: Chaperone-Mediated Protein Disaggregation Triggers Proteolytic Clearance of
  Intra-nuclear Protein Inclusions.
authors:
- den Brave F
- Cairo LV
- Jagadeesan C
- Ruger-Herreros C
- Mogk A
- Bukau B
- Jentsch S
journal: Cell Rep
year: '2020'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC7273177
doi: 10.1016/j.celrep.2020.107680
---

# Chaperone-Mediated Protein Disaggregation Triggers Proteolytic Clearance of Intra-nuclear Protein Inclusions.
**Authors:** den Brave F, Cairo LV, Jagadeesan C, Ruger-Herreros C, Mogk A, Bukau B, Jentsch S
**Journal:** Cell Rep (2020)
**DOI:** [10.1016/j.celrep.2020.107680](https://doi.org/10.1016/j.celrep.2020.107680)
**PMC:** [PMC7273177](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7273177/)

## Abstract

1. Cell Rep. 2020 Jun 2;31(9):107680. doi: 10.1016/j.celrep.2020.107680.

Chaperone-Mediated Protein Disaggregation Triggers Proteolytic Clearance of 
Intra-nuclear Protein Inclusions.

den Brave F(1), Cairo LV(2), Jagadeesan C(2), Ruger-Herreros C(3), Mogk A(3), 
Bukau B(3), Jentsch S(4).

Author information:
(1)Department of Molecular Cell Biology, Max Planck Institute of Biochemistry, 
Am Klopferspitz 18, 82152 Martinsried, Germany. Electronic address: 
denbrave@uni-bonn.de.
(2)Department of Molecular Cell Biology, Max Planck Institute of Biochemistry, 
Am Klopferspitz 18, 82152 Martinsried, Germany; Department of Cellular 
Biochemistry, Max Planck Institute of Biochemistry, Am Klopferspitz 18, 82152 
Martinsried, Germany.
(3)Center for Molecular Biology of Heidelberg University (ZMBH), Im Neuenheimer 
Feld 282, DKFZ-ZMBH Alliance, Heidelberg, Germany; German Cancer Research Center 
(DKFZ), 69120 Heidelberg, Germany.
(4)Department of Molecular Cell Biology, Max Planck Institute of Biochemistry, 
Am Klopferspitz 18, 82152 Martinsried, Germany.

The formation of insoluble inclusions in the cytosol and nucleus is associated 
with impaired protein homeostasis and is a hallmark of several neurodegenerative 
diseases. Due to the absence of the autophagic machinery, nuclear protein 
aggregates require a solubilization step preceding degradation by the 26S 
proteasome. Using yeast, we identify a nuclear protein quality control pathway 
required for the clearance of protein aggregates. The nuclear J-domain protein 
Apj1 supports protein disaggregation together with Hsp70 but independent of the 
canonical disaggregase Hsp104. Disaggregation mediated by Apj1/Hsp70 promotes 
turnover rather than refolding. A loss of Apj1 activity uncouples disaggregation 
from proteasomal turnover, resulting in accumulation of toxic soluble protein 
species. Endogenous substrates of the Apj1/Hsp70 pathway include both nuclear 
and cytoplasmic proteins, which aggregate inside the nucleus upon proteotoxic 
stress. These findings demonstrate the coordinated activity of the Apj1/Hsp70 
disaggregation system with the 26S proteasome in facilitating the clearance of 
toxic inclusions inside the nucleus.

Copyright © 2020 The Authors. Published by Elsevier Inc. All rights reserved.

DOI: 10.1016/j.celrep.2020.107680
PMCID: PMC7273177
PMID: 32492414 [Indexed for MEDLINE]

Conflict of interest statement: Declaration of Interests The authors declare no 
competing interests.

## Full Text

IntroductionCells have evolved complex networks of molecular chaperones and proteolytic systems to maintain protein homeostasis (or proteostasis). Chaperones assist in protein folding and prevent aggregation, refold stress-denatured proteins, and cooperate with the ubiquitin proteasome system (UPS) and autophagy in the degradation of terminally misfolded proteins (Chen et al., 2011, Hipp et al., 2019, Tyedmers et al., 2010). Proteostasis imbalance, as it may occur as part of the aging process, increases the danger of misfolding and frequently results in the accumulation of aggregates that are deposited in insoluble inclusions, a hallmark of many age-dependent neurodegenerative diseases (Douglas and Dillin, 2010, Hipp et al., 2014). Many polyQ diseases are associated with aggregate formation in the nucleus, including Huntington disease and several forms of spinocerebellar ataxia (Chung et al., 2018, Mori et al., 2013). Thus, it is important to understand the machineries and mechanisms that act in the nucleus to maintain proteostasis.The use of model proteins targeted to the nucleus has uncovered quality control mechanisms resulting in the degradation of misfolded proteins or their sequestration into inclusions (Jones and Gardner, 2016, Sontag et al., 2017). Studies in the yeast Saccharomyces cerevisiae identified ubiquitin ligases that recognize misfolded proteins and mediate their proteasomal degradation. The ligase San1 has been shown to recognize and mediate the degradation of soluble terminally misfolded or mutated proteins, but it does not support the clearance of proteins once aggregated (Gardner et al., 2005, Rosenbaum et al., 2011). Likewise, the conserved ligase Ubr1 localizes to the nucleus, where it displays overlapping substrate specificity with San1 in mediating the turnover of several model substrates (Amm and Wolf, 2016, Heck et al., 2010, Khosrow-Khavar et al., 2012, Nillegoda et al., 2010, Prasad et al., 2018, Samant et al., 2018). Degradation mediated by these ligases requires the Hsp40 chaperones Ydj1 and Sis1, which together with Hsp70 are presumed to maintain protein solubility to support proteasomal turnover (Guerriero et al., 2013, Heck et al., 2010, Park et al., 2007, Prasad et al., 2010, Prasad et al., 2018, Summers et al., 2013). In addition, the membrane-embedded ubiquitin ligase Doa10 locates to the endoplasmic reticulum and the nuclear envelope, where it contributes to the turnover of nuclear proteins in a Sis1-dependent manner (Deng and Hochstrasser, 2006, Shiber et al., 2013). At the inner nuclear membrane, the Asi complex targets aberrant and mislocalized proteins for degradation (Foresti et al., 2014, Khmelinskii et al., 2014).When timely degradation fails, as upon overload of the UPS under acute stress conditions, misfolded proteins may be sequestered into cytosolic, intranuclear, and juxtanuclear inclusions (Kaganovich et al., 2008, Malinovska et al., 2012, Miller et al., 2015). This process is regulated by the chaperones Btn2 and Hsp42, which promote inclusion formation in nucleus and cytosol, respectively (Malinovska et al., 2012, Miller et al., 2015, Specht et al., 2011). Sequestration of soluble misfolded proteins inside the nucleus is required to maintain proteostasis when chaperone capacity is low (Ho et al., 2019).The autophagy pathway, which can degrade larger aggregate structures, does not operate in the nucleoplasm (Gatica et al., 2018). Thus, clearance of nuclear inclusions would require the action of disaggregating chaperones, producing soluble protein for degradation by the UPS.The best-studied disaggregation machinery is composed of the yeast AAA+ protein Hsp104, which cooperates with Hsp40 and Hsp70 partner chaperones in recovering proteins from heat-induced amorphous aggregates (Glover and Lindquist, 1998, Parsell et al., 1994). Hsp104 exerts a threading activity on the aggregate, leading to the sequential removal of monomeric protein species, which are predominantly targeted to refolding pathways (Ho et al., 2019, Mogk et al., 2018, Shorter and Southworth, 2019, Wallace et al., 2015). Hsp104 is absent from metazoans where Hsp70 mediates disaggregation in cooperation with Hsp40 co-chaperones and the nucleotide exchange factor (NEF) Hsp110 (Gao et al., 2015, Rampelt et al., 2012, Shorter, 2011). Hsp70 chaperones have a central role in the proteostasis network, based on their ability to bind short hydrophobic peptides in the context of non-native proteins (Balchin et al., 2016, Kampinga and Craig, 2010, Rosenzweig et al., 2019). Functional diversity among Hsp70 members is mainly achieved through a large set of Hsp40 co-chaperones, J-domain proteins, which recruit Hsp70s to specific protein substrates and subcellular locations. Mixed complexes of class A and class B Hsp40 chaperones have been shown to enhance Hsp70-mediated disaggregation (Nillegoda et al., 2015). How disaggregation of aberrant proteins is mediated in the nucleus remains poorly defined. Recent work demonstrated a role of Btn2 in recruiting Sis1 together with Hsp70-Hsp104 to refold nuclear luciferase after heat stress (Ho et al., 2019). However, it has remained unclear if the nucleus contains dedicated machinery to eliminate aggregates containing damaged or terminally misfolded proteins.In the present study, we have used yeast to understand how nuclear protein aggregates are cleared. We have identified the Hsp40 Apj1 as a nuclear chaperone specifically required for the disaggregation of intra-nuclear inclusions. Apj1 is a canonical class A Hsp40, which has previously been observed to localize to nuclear foci, but whose function has remained unknown (Gallina et al., 2015, Tkach et al., 2012). We show that Apj1 functions in concert with Ssa-class Hsp70s and the Hsp70 NEF Hsp110 (Sse1). We demonstrate both in vivo and in vitro that Apj1 supports aggregate solubilization independently of the disaggregase Hsp104. This is consistent with the model that Apj1 is part of an Hsp70-Hsp110-based disaggregation machinery thus far described only in higher eukaryotes. We observe that Apj1 competes with Hsp104 for substrate disaggregation. While Apj1 supports direct turnover of aggregated proteins following disaggregation, Hsp104 appears to rather support refolding of disaggregated substrates. This coordination of protein disaggregation and turnover provides a protective mechanism, which minimizes the occurrence of toxic soluble aberrant proteins created by disaggregation. Proteomic analysis shows that substrates of the Apj1-dependent nuclear quality control pathway include multiple nuclear and cytosolic proteins, which aggregate upon heat stress or proteasome inhibition. These findings highlight a general role for the nucleus as an important hub in the cellular proteostasis network and define Apj1 as a central component of nuclear protein quality control.

Introduction

Cells have evolved complex networks of molecular chaperones and proteolytic systems to maintain protein homeostasis (or proteostasis). Chaperones assist in protein folding and prevent aggregation, refold stress-denatured proteins, and cooperate with the ubiquitin proteasome system (UPS) and autophagy in the degradation of terminally misfolded proteins (Chen et al., 2011, Hipp et al., 2019, Tyedmers et al., 2010). Proteostasis imbalance, as it may occur as part of the aging process, increases the danger of misfolding and frequently results in the accumulation of aggregates that are deposited in insoluble inclusions, a hallmark of many age-dependent neurodegenerative diseases (Douglas and Dillin, 2010, Hipp et al., 2014). Many polyQ diseases are associated with aggregate formation in the nucleus, including Huntington disease and several forms of spinocerebellar ataxia (Chung et al., 2018, Mori et al., 2013). Thus, it is important to understand the machineries and mechanisms that act in the nucleus to maintain proteostasis.

The use of model proteins targeted to the nucleus has uncovered quality control mechanisms resulting in the degradation of misfolded proteins or their sequestration into inclusions (Jones and Gardner, 2016, Sontag et al., 2017). Studies in the yeast Saccharomyces cerevisiae identified ubiquitin ligases that recognize misfolded proteins and mediate their proteasomal degradation. The ligase San1 has been shown to recognize and mediate the degradation of soluble terminally misfolded or mutated proteins, but it does not support the clearance of proteins once aggregated (Gardner et al., 2005, Rosenbaum et al., 2011). Likewise, the conserved ligase Ubr1 localizes to the nucleus, where it displays overlapping substrate specificity with San1 in mediating the turnover of several model substrates (Amm and Wolf, 2016, Heck et al., 2010, Khosrow-Khavar et al., 2012, Nillegoda et al., 2010, Prasad et al., 2018, Samant et al., 2018). Degradation mediated by these ligases requires the Hsp40 chaperones Ydj1 and Sis1, which together with Hsp70 are presumed to maintain protein solubility to support proteasomal turnover (Guerriero et al., 2013, Heck et al., 2010, Park et al., 2007, Prasad et al., 2010, Prasad et al., 2018, Summers et al., 2013). In addition, the membrane-embedded ubiquitin ligase Doa10 locates to the endoplasmic reticulum and the nuclear envelope, where it contributes to the turnover of nuclear proteins in a Sis1-dependent manner (Deng and Hochstrasser, 2006, Shiber et al., 2013). At the inner nuclear membrane, the Asi complex targets aberrant and mislocalized proteins for degradation (Foresti et al., 2014, Khmelinskii et al., 2014).

When timely degradation fails, as upon overload of the UPS under acute stress conditions, misfolded proteins may be sequestered into cytosolic, intranuclear, and juxtanuclear inclusions (Kaganovich et al., 2008, Malinovska et al., 2012, Miller et al., 2015). This process is regulated by the chaperones Btn2 and Hsp42, which promote inclusion formation in nucleus and cytosol, respectively (Malinovska et al., 2012, Miller et al., 2015, Specht et al., 2011). Sequestration of soluble misfolded proteins inside the nucleus is required to maintain proteostasis when chaperone capacity is low (Ho et al., 2019).

The autophagy pathway, which can degrade larger aggregate structures, does not operate in the nucleoplasm (Gatica et al., 2018). Thus, clearance of nuclear inclusions would require the action of disaggregating chaperones, producing soluble protein for degradation by the UPS.

The best-studied disaggregation machinery is composed of the yeast AAA+ protein Hsp104, which cooperates with Hsp40 and Hsp70 partner chaperones in recovering proteins from heat-induced amorphous aggregates (Glover and Lindquist, 1998, Parsell et al., 1994). Hsp104 exerts a threading activity on the aggregate, leading to the sequential removal of monomeric protein species, which are predominantly targeted to refolding pathways (Ho et al., 2019, Mogk et al., 2018, Shorter and Southworth, 2019, Wallace et al., 2015). Hsp104 is absent from metazoans where Hsp70 mediates disaggregation in cooperation with Hsp40 co-chaperones and the nucleotide exchange factor (NEF) Hsp110 (Gao et al., 2015, Rampelt et al., 2012, Shorter, 2011). Hsp70 chaperones have a central role in the proteostasis network, based on their ability to bind short hydrophobic peptides in the context of non-native proteins (Balchin et al., 2016, Kampinga and Craig, 2010, Rosenzweig et al., 2019). Functional diversity among Hsp70 members is mainly achieved through a large set of Hsp40 co-chaperones, J-domain proteins, which recruit Hsp70s to specific protein substrates and subcellular locations. Mixed complexes of class A and class B Hsp40 chaperones have been shown to enhance Hsp70-mediated disaggregation (Nillegoda et al., 2015). How disaggregation of aberrant proteins is mediated in the nucleus remains poorly defined. Recent work demonstrated a role of Btn2 in recruiting Sis1 together with Hsp70-Hsp104 to refold nuclear luciferase after heat stress (Ho et al., 2019). However, it has remained unclear if the nucleus contains dedicated machinery to eliminate aggregates containing damaged or terminally misfolded proteins.

In the present study, we have used yeast to understand how nuclear protein aggregates are cleared. We have identified the Hsp40 Apj1 as a nuclear chaperone specifically required for the disaggregation of intra-nuclear inclusions. Apj1 is a canonical class A Hsp40, which has previously been observed to localize to nuclear foci, but whose function has remained unknown (Gallina et al., 2015, Tkach et al., 2012). We show that Apj1 functions in concert with Ssa-class Hsp70s and the Hsp70 NEF Hsp110 (Sse1). We demonstrate both in vivo and in vitro that Apj1 supports aggregate solubilization independently of the disaggregase Hsp104. This is consistent with the model that Apj1 is part of an Hsp70-Hsp110-based disaggregation machinery thus far described only in higher eukaryotes. We observe that Apj1 competes with Hsp104 for substrate disaggregation. While Apj1 supports direct turnover of aggregated proteins following disaggregation, Hsp104 appears to rather support refolding of disaggregated substrates. This coordination of protein disaggregation and turnover provides a protective mechanism, which minimizes the occurrence of toxic soluble aberrant proteins created by disaggregation. Proteomic analysis shows that substrates of the Apj1-dependent nuclear quality control pathway include multiple nuclear and cytosolic proteins, which aggregate upon heat stress or proteasome inhibition. These findings highlight a general role for the nucleus as an important hub in the cellular proteostasis network and define Apj1 as a central component of nuclear protein quality control.

ResultsThe Nuclear Hsp40 Apj1 Functions in Clearance of Misfolded ProteinsThe Hsp40 chaperone Apj1 has been previously found to localize to the intra-nuclear protein inclusion INQ (Gallina et al., 2015, Tkach et al., 2012). When evaluating the subcellular distribution of GFP-tagged Apj1 (GFP-Apj1) expressed from its endogenous promoter, we observed a strong enrichment in the nucleus (marked by Nup49-mars of the nuclear pore complex) (Figures 1A and S1A). Acute heat stress or proteasome inhibition resulted in coalescence of GFP-Apj1 into nuclear inclusions (Figures 1A and S1A), suggesting that Apj1 might be associated with nuclear protein aggregates. To mark the intra-nuclear quality control compartment INQ, we used heat-aggregated nuclear luciferase (LuciDM-NLS-GFP) (Miller et al., 2015). Indeed, GFP-tagged nuclear luciferase was found to co-localize with mars-tagged Apj1 (mars-Apj1) upon heat stress, confirming the previously published INQ localization of Apj1 (Figures 1B and S1B). Cell fractionation showed that Apj1 was soluble in unstressed cells (Figure 1C, −HS). Notably, the level of endogenous Apj1 strongly increased upon acute heat stress, and the majority of the protein shifted to the insoluble pellet fraction (Figure 1C, +HS), suggesting that Apj1 function is increasingly required under stress conditions that cause protein aggregation. Likewise, we observed increased amounts of the disaggregase Hsp104 as well as the Hsp40 Sis1 in the insoluble fraction upon stress, whereas the most abundant Hsp40, Ydj1, remained entirely soluble (Figure 1C +HS). Because Hsp104 and Sis1 have been previously implicated in protein disaggregation (Ho et al., 2019, Parsell et al., 1994), which might be reflected by their presence in the insoluble fraction, we speculated that Apj1 might function in recovering aggregated proteins from inclusions.Figure 1The Hsp40 Apj1 Mediates Turnover of Aggregation-Prone Nuclear Proteins(A) Nuclear localization of Apj1. GFP-Apj1 was expressed in cells harboring Nup49-mars and lacking the ABC transporter Pdr5 and was analyzed by live cell imaging. Cells were left untreated, subjected for 20 min to 42°C or treated for 2 h with the proteasome inhibitor MG-132. Dashed lines indicate cell boundaries.(B) Apj1 localizes to INQ. mars-Apj1 was co-expressed with GFP-tagged nuclear-targeted mutated luciferase (LuciDM-NLS-GFP). Formation of INQ was induced by 20-min heat shock at 42°C and analyzed as in (A).(C) Fractionation of Apj1 in untreated and heat-stressed cells. Total cell extracts (T) were solubilized and separated by centrifugation in detergent soluble (S) and insoluble pellet (P) fractions. Equal amounts of each fraction were loaded. Fractionation was analyzed using antibodies against endogenous Apj1, Ydj1, Sis1 and Hsp104. Soluble Pgk1 served as a control.(D) Effect of Apj1 on luciferase disaggregation. The indicated strains expressing LuciDM-NLS as in (B) were subjected to 42°C for 20 min to aggregate nuclear luciferase. Disaggregation was determined by measuring luciferase activity at different time points after heat shock as indicated. Luciferase activity measured before heat shock was set to 100%. Quantification shows averages ± SD from three independent experiments.(E) Turnover of LuciDM-NLS-GFP in Apj1-deficient cells. Degradation of LuciDM-NLS-GFP after heat shock was analyzed after addition of cycloheximide (CHX). LuciDM-NLS-GFP was detected using anti-GFP antibodies; Pgk1 serves a loading control. Quantification below shows averages ± SD from three independent experiments.(F) Schematic representation and subcellular distribution of the model substrate NLS-CG∗. ΔssCPY∗ is a mutant version of the secretory protein carboxypeptidase Y lacking the signal sequence; NLS, nuclear localization signal. Cells expressing NLS-CG∗ and Nup49-mars were analyzed by live cell microscopy under unstressed conditions as in (A).(G) Turnover of NLS-CG∗ in Apj1-deficient cells. Degradation of NLS-CG∗ was analyzed after addition of CHX. NLS-CG∗ was detected using anti-GFP antibodies; Pgk1 serves a loading control. Quantification below shows averages ± SD from three independent experiments.To analyze a possible role of Apj1 in recovering aggregated proteins following stress, we probed its role in refolding heat-aggregated nuclear luciferase by using LuciDM-NLS-GFP. We measured the recovery of in vivo luciferase activity after heat-induced luciferase inactivation, which was dependent on the disaggregase Hsp104 (Figure S1C). Surprisingly, we observed that recovery of luciferase activity was accelerated in the absence of Apj1 (Figure 1D). In addition, we observed an increase in luciferase stability after heat shock in cells lacking Apj1 (Figure 1E). Thus, Apj1 might counteract refolding of aggregated luciferase by competing with Hsp104 and Sis1-dependent disaggregation and appears to rather promote protein turnover. To better investigate a possible role of Apj1 in nuclear protein turnover, we used mutant carboxypeptidase Y fused to GFP (CG∗) as a terminally misfolded model protein (Figure 1F). As a secretory protein, carboxypeptidase Y is unable to fold in the reducing environment of the cytosol (Medicherla et al., 2004), and when expressed without a secretory signal sequence (CG∗), it is recognized by cytosolic chaperones and transported to the nucleus for proteasomal degradation (Heck et al., 2010, Park et al., 2007, Park et al., 2013, Prasad et al., 2010). We targeted CG∗ to the nucleus (NLS-CG∗), bypassing the requirement of chaperones for nuclear import (Park et al., 2013). Expression of NLS-CG∗ from the GAL1 promoter resulted in diffuse nuclear staining and formation of GFP-positive inclusions located both inside and outside the nucleus (Figures 1F and S1D). We next analyzed the turnover of NLS-CG∗ upon cycloheximide (CHX) treatment and observed a delayed turnover in cells lacking Apj1 (Figure 1G).In summary, we conclude that Apj1 is recruited to nuclear protein inclusions upon stress and supports the turnover of misfolded nuclear-targeted proteins.Apj1 Acts on Nuclear Protein AggregatesThe localization of Apj1 to INQ and its fractionation to the insoluble pellet upon stress prompted us to investigate a possible role on nuclear protein aggregates. Cell fractionation revealed that about 50% of the Apj1 substrate NLS-CG∗ was insoluble under standard growth conditions (Figure 2A). This allowed us to compare the effect of Apj1 on both soluble and insoluble substrate pools. CHX shut-off followed by cell fractionation showed that a loss of Apj1 function substantially delayed the degradation of insoluble NLS-CG∗ (Figure 2B, pellet), but had only a minor effect on the turnover of soluble NLS-CG∗ (Figure 2B, soluble). Similar results were obtained without CHX addition, by blocking galactose-dependent NLS-CG∗ expression by addition of glucose (Figure S2A). These results suggested a role for Apj1 in clearing protein aggregates, in line with its accumulation in nuclear foci and insolubility upon stress.Figure 2Apj1 Acts on Nuclear Protein Aggregates(A) Fractionation of NLS-CG∗ in unstressed cells was performed as in Figure 1C. NLS-CG∗ was detected using anti-GFP antibodies. Quantification below shows averages ± SD from three independent experiments.(B) Turnover of soluble and insoluble NLS-CG∗ in the absence of Apj1. Degradation of NLS-CG∗ was followed after addition of CHX. Samples of each time point were fractionated as in (A). NLS-CG∗ was detected using anti-GFP antibodies. Soluble Pgk1 serves as a control. Quantification below shows averages ± SD from three independent experiments.(C) Localization of differentially targeted CG∗ variants. Distribution of NLS-CG∗ and NES-CG∗ following acute heat stress (20 min, 42°C) in cells harboring Nup49-mars was analyzed by live cell microscopy. NES, nuclear export signal.(D) Impact of Apj1 on the stability of differentially localized CG∗ variants. Turnover of NLS-CG∗ and NES-CG∗ after acute heat stress (20 min, 42°C) was followed as in Figure 1G. NLS-CG∗ and NES-CG∗ were detected using anti-GFP antibodies. Pgk1 serves as a control. Averages ± SD from three independent experiments are shown.(E) Effect of Apj1 and Btn2 deletion on NLS-CG∗ aggregation. The indicated strains were fractionated and analyzed as in (A). Averages ± SD from three independent experiments are shown.(F) Turnover of NLS-CG∗ in the absence of Btn2. Degradation of NLS-CG∗ was analyzed as in Figure 1G in the indicated strains. Averages ± SD from three independent experiments are shown.The dual localization of NLS-CG∗ foci to the nucleus and the cytoplasm (Figure 1F) raised the question as to which pool of the aggregated substrate is targeted by Apj1. CG∗ is efficiently excluded from the nucleus by addition of a nuclear export sequence (NES), which results in the cytoplasmic localization of the protein and its marked stabilization (Park et al., 2013). To achieve extensive aggregate formation, cells expressing NLS- and NES-CG∗ were exposed to heat stress. NLS-CG∗ formed nuclear and cytoplasmic foci, whereas NES-CG∗ formed exclusively cytoplasmic foci (Figures 2C and S2B). Deletion of Apj1 reduced the turnover of NLS-CG∗ following heat stress but had no effect on the slow degradation of cytoplasmic NES-CG∗ (Figure 2D). CG∗, without additional targeting sequence, is transported to the nucleus for degradation (Park et al., 2013). Importantly, turnover of CG∗ was also delayed in the absence of Apj1, demonstrating that its nuclear localization, but not the NLS itself, renders the substrate Apj1 dependent (Figure S2C).Sequestration of misfolded proteins into nuclear inclusions has been shown to be dependent on the chaperone Btn2 (Miller et al., 2015). Indeed, cells lacking Btn2 contained no visible nuclear NLS-CG∗ foci, independent of the presence of Apj1 (Figure S2D). We compared the amounts of aggregated NLS-CG∗ by using fractionation. As expected, deletion of Apj1 alone resulted in a strong accumulation of NLS-CG∗ in the insoluble pellet fraction (Figure 2E). In contrast, the loss of Btn2 reduced the amount of insoluble NLS-CG∗, in line with its requirement for nuclear aggregate formation (Figure 2E). Additional deletion of the cytosolic aggregase Hsp42 completely abolishes NLS-CG∗ aggregation (Figure 2E). This suggests that the residual amounts of aggregated NLS-CG∗ observed in absence of Btn2 represent cytosolic insoluble NLS-CG∗, consistent with NLS-CG∗ foci present in the cytosol (Figure 1F). Strikingly, additional deletion of Apj1 in cells lacking Btn2 did not result in an accumulation of aggregated NLS-CG∗, as observed in the presence of Btn2 (Figure 2E). In line with this, NLS-CG∗ was degraded with similar kinetics in wild-type (WT) and Δbtn2 and Δbtn2Δapj1 mutant cells (Figure 2F). Thus, deletion of Btn2 bypasses the requirement of Apj1 for efficient NLS-CG∗ degradation, suggesting that Apj1 is specifically required for the turnover of nuclear NLS-CG∗ aggregates generated by the action of Btn2.Collectively, these data support a role of Apj1 in specifically clearing nuclear protein aggregates.Apj1 Cooperates with Hsp70 and Hsp110 in Aggregate ClearanceIn line with the canonical function of Hsp40 proteins, it seemed plausible to assume that Apj1 cooperates with Hsp70. The J-domain of Hsp40 is critical for its interaction with Hsp70, and indeed, we observed an interaction of Apj1 with the Hsp70 proteins Ssa1, Ssa3, and Ssa4 (Ssa2 was not tested due to high similarity with Ssa1) that was lost upon J-domain deletion (Figure S3A). In line with an Hsp70-dependent role of Apj1 in NLS-CG∗ degradation, we observed that Apj1 lacking the J-domain does not support NLS-CG∗ turnover (Figure S3B). Stimulation of the Hsp70 ATPase by Hsp40 and transfer of substrates from Hsp40 to Hsp70 depends on the conserved His-Pro-Asp (HPD)-loop segment in the J-domain of Hsp40 (Kampinga and Craig, 2010). To trap substrates on Apj1, we mutated the HPD-loop residues of Apj1 to alanine (Apj1AAA). Apj1AAA failed to restore the delayed degradation of NLS-CG∗ in Δapj1 cells (Figure 3A), consistent with Apj1 functionally cooperating with Hsp70. Moreover, immunoprecipitation of NLS-CG∗ from cell extracts resulted in efficient co-precipitation of Apj1AAA, whereas WT Apj1 was co-precipitated with lower efficiency (Figure 3B). These data demonstrate that Apj1 transfers substrates to Hsp70 and requires Hsp70 interaction for supporting NLS-CG∗ turnover.Figure 3Apj1-Dependent Turnover Requires Hsp70 and Hsp110(A) Functionality of Apj1AAA. Degradation of NLS-CG∗ was analyzed as in Figure 1G in Δapj1 cells expressing the indicated plasmids. Averages ± SD from three independent experiments are shown.(B) Interaction of Apj1 with NLS-CG∗. NLS-CG∗ or just GFP was co-expressed with the indicated Apj1 variants in Δapj1 cells. NLS-CG∗ was purified from cell extracts using GFP-trap beads. Binding proteins were analyzed by western blotting using GFP and Apj1-specific antibodies. IP, immunoprecipitation.(C) Role of Hsp70 NEFs in degrading soluble and insoluble NLS-CG∗. Degradation of NLS-CG∗ in the indicated strains was analyzed as in Figure 2B.(D) Role of Sse1 localization on NLS-CG∗ turnover. Empty vector or the indicated Sse1 variants were co-expressed with NLS-CG∗ in the indicated strains. Degradation of soluble and insoluble NLS-CG∗ was analyzed as in Figure 2B. Averages ± SD from three independent experiments are shown.We next asked which of the Hsp70 NEFs might be involved in Apj1-dependent turnover of aggregated NLS-CG∗. We screened all known cytosolic/nuclear NEFs for their role in clearing NLS-CG∗ aggregates after heat stress, including the Hsp110 homologs Sse1 and Sse2, and the unrelated NEFs Fes1 and Snl1. Only deletion of Sse1 strongly stabilized NLS-CG∗ (Figure S3C). Similarly we found that in non-stressed cells, Sse1 was the only NEF required for degradation of insoluble NLS-CG∗ (Figure 3C). However, we also observed a stabilization of soluble NLS-CG when Sse1 was deleted (Figure 3C). This points to a more general effect of Sse1 deletion on Hsp70 function, as substrate release from Hsp70 will be strongly impaired, thus reducing total Hsp70 capacity. To address a possible nuclear role of Sse1, we targeted or excluded this NEF by addition of an NLS or NES, respectively. At normal growth temperature (30°C) both NLS- and NES-Sse1 restored the strong growth defect of cells lacking Sse1 (Figure S3D), in line with Sse1 deletion generally impairing cellular Hsp70 function. Importantly, only NLS-Sse1, but not NES-Sse1, fully restored growth at an elevated temperature (37°C), suggesting a nuclear-specific function for Sse1 (Figure S3D). We next analyzed the impact of the different Sse1 variants on NLS-CG∗ turnover. In line with a general impairment of Hsp70 function in Sse1 deficient cells, we observed that NLS- and NES-Sse1 supported the turnover of soluble NLS-CG∗ to the same extent as Sse1 without additional targeting sequences (Figure 3D, soluble). In contrast, only NLS-Sse1 supported the Apj1-dependent clearance of aggregated NLS-CG∗, and the turnover in the presence of NES-Sse1 was similar to Apj1-deficient cells (Figure 3D, pellet).Together, these data are consistent with Apj1 co-operating with Hsp70 and the NEF Sse1 in clearing nuclear proteins aggregates.Apj1 Mediates Hsp104-Independent Protein DisaggregationTo date, all protein disaggregation processes described in yeast depend on Hsp104. Thus, we analyzed the requirement of Hsp104 for the degradation of insoluble NLS-CG∗ recovered in the pellet fraction. Deletion of HSP104 had a stabilizing effect on NLS-CG∗ aggregates, which was surprisingly less pronounced than the stabilization observed in Δapj1 cells (Figure 4A). Importantly, turnover of insoluble NLS-CG∗ was completely blocked when both Apj1 and Hsp104 were deleted simultaneously (Figure 4A). This suggests that Apj1 and Hsp104 function independently in clearing aggregated NLS-CG∗. Because NLS-CG∗ also forms cytoplasmic aggregates, we asked whether both Apj1 and Hsp104 are recruited to aggregates in the nucleus. When co-expressing mars-Apj1AAA with NLS-CG∗, we found that Apj1 co-localized only with nuclear, but not cytoplasmic inclusions, of NLS-CG∗ (Figures 4B and S4A). In contrast, mars-tagged Hsp104 (Hsp104-mars) co-localized with both cytoplasmic and nuclear NLS-CG∗ inclusions (Figures 4B and S4A). These data are consistent with Apj1 and Hsp104 both acting on nuclear protein aggregates.Figure 4Apj1 Supports Disaggregation Independent of Hsp104 and Sis1(A) Effect of Apj1 and Hsp104 on degradation of insoluble NLS-CG∗. Degradation of insoluble NLS-CG∗ (pellet fraction) in the indicated strains was analyzed as in Figure 2B. Averages ± SD from three independent experiments are shown.(B) Co-localization of NLS-CG∗ with Apj1AAA and Hsp104. mars-tagged Apj1AAA was co-expressed with NLS-CG∗ in Δapj1 cells, and genomically tagged Hsp104-mars was co-expressed with NLS-CG∗. Cells were analyzed by live-cell imaging as in Figure 1A. Dashed lines indicate position of nucleus. Nuclei were counterstained with Hoechst. N, nuclear; C, cytoplasmic foci.(C) Interplay between Sis1 and Apj1 in NLS-CG∗ turnover. The indicated strains expressing NLS-CG∗ were grown in presence of doxycycline (Dox) and analyzed as in Figure 2B. Averages ± SD from four independent experiments are shown. Significance was calculated using Student’s t test, ∗ p < 0.05.(D) Synthetic growth defect between Apj1 and Sis1. Serial dilutions of the indicated yeast strains pre-grown with (+) or without (−) Dox were spotted onto YPD plates containing Dox and incubated at the indicated temperatures.(E) Analysis of Apj1-dependent disaggregation in vitro. Luciferase was aggregated at 42°C for 20 min. Luciferase activities were determined after a 240 min incubation at 30°C in the presence of the indicated chaperones. The remaining sample was subjected to high-speed centrifugation, and the soluble supernatant was analyzed by western blotting against luciferase. The activity and solubility of native luciferase was set as 100%. Averages ± SD from three independent experiments are shown. wo HS, without heat shock.To address the possible involvement of other Hsp40 chaperones in NLS-CG∗ turnover, we analyzed the impact of the three cytosolic or nuclear class A Hsp40 chaperones (Apj1, Ydj1, and Xdj1) and three class B Hsp40s (Sis1, Caj1, and Djp1) present in yeast. Hsp40 chaperones may either function in aggregation prevention or in disaggregation. Testing the solubility of NLS-CG∗ in cells deficient for individual Hsp40 chaperones showed that the loss of Ydj1 and Sis1 resulted in complete aggregation of NLS-CG∗, implying a function in maintaining the protein in a soluble state (Figure S4B, −HS). To screen for a possible involvement in clearing aggregated NLS-CG∗, we performed CHX chase experiments upon heat stress, where most NLS-CG∗ is insoluble (Figure S4B, +HS). In addition to Apj1, only the class B Hsp40 Sis1 was required for efficient NLS-CG∗ turnover, whereas the other Hsp40 chaperones tested were dispensable (Figure S4C). In addition to its function in trafficking misfolded proteins to the nucleus, Sis1 has recently been shown to promote refolding of heat-aggregated luciferase (Ho et al., 2019). To test if Apj1 co-operates with Sis1, we analyzed the turnover of aggregated NLS-CG∗ in Sis1-depleted cells with or without Apj1. Although Sis1 depletion alone already strongly delayed removal of insoluble NLS-CG∗, we observed a small but significant increase in stabilization when deleting Apj1 in addition (Figure 4C). These findings are reminiscent of the full stabilization of insoluble NLS-CG∗ upon Apj1 and Hsp104 deletion (Figure 4A). This might suggest independent roles for Apj1 and Sis1 in clearing aggregated NLS-CG∗. In agreement with independent but overlapping actions of the two Hsp40s, we observed a strong negative synthetic growth defect at high temperatures when downregulating Sis1 in cells lacking Apj1 (Figures 4D and S4D).To directly show that Apj1 supports protein disaggregation, we tested for this activity in vitro. We used heat-aggregated luciferase as a substrate and measured its recovery by disaggregation. To this end, we compared the soluble protein and the amount of active luciferase obtained after adding the indicated combination of chaperones and subsequent incubation for 240 min at 30°C. As previously shown, aggregated luciferase is efficiently re-solubilized and refolded by the Hsp70-Hsp104 bi-chaperone system in conjunction with the Hsp40 Sis1 and a NEF (here, Sse1) (Figures 4E and S4E) (Ho et al., 2019). We observed that Sis1 efficiently supported disaggregation and refolding of heat-aggregated luciferase, which was reflected by high amounts of solubilized, active luciferase (Figures 4E and S4E). Luciferase refolding was low with Apj1 as the sole Hsp40 in the reaction; however, a substantial amount of luciferase was detected in the soluble fraction of this sample (Figures 4E and S4E). Strikingly, the observed Apj1 activity in solubilizing luciferase was independent of Hsp104 (Figure 4E). In contrast, the amount of luciferase solubilized by Sis1 activity was strongly reduced in the absence of Hsp104, and its Hsp104-independent disaggregation activity was lower than the activity of Apj1 (Figure 4E). Using equal amounts of Apj1 and Sis1 (with the same final Hsp40 concentration as before) in the disaggregation reaction resulted in comparable amounts of soluble luciferase compared with Sis1 alone, and we observed a reduction in luciferase refolding (Figure 4E). This is in line with both Hsp40 chaperones acting independently, with Apj1 generating non-native, soluble luciferase species.We conclude that two independent pathways for protein disaggregation exist in the nucleus. One involves Sis1 and Hsp104 as previously described, and a second pathway involves Apj1 in concert with Hsp70 and Hsp110 but independent of Hsp104. Importantly, Apj1-dependent disaggregation produces soluble species but does not support refolding, which is consistent with a function in supporting proteolytic turnover of disaggregated proteins.Apj1 Co-ordinates Disaggregation with TurnoverThe observation that Apj1 mediates turnover of nuclear protein aggregates in vivo prompted us to investigate the role of the UPS in this pathway. We observed that both soluble and insoluble pools of NLS-CG∗ are efficiently degraded in a proteasome-dependent manner, as indicated by marked stabilization of NLS-CG∗ upon proteasome inhibition (Figure S5A). Next, we tested the three ubiquitin ligases Ubr1, San1, and Doa10, which have been previously linked to nuclear quality control and turnover of CG∗/NLS-CG∗ (Heck et al., 2010, Samant et al., 2018). Deletion of Apj1 in the absence of San1 or Doa10 resulted in additional stabilization of insoluble NLS-CG∗, suggesting they act on alternative pathways (Figure S5B). In contrast, deletion of Apj1 in cells lacking Ubr1 had no additional effect on the turnover of aggregated NLS-CG∗ (Figure S5B). However, in a triple deletion of Ubr1 with San1 and Doa10, a lack of Apj1 still resulted in a strong stabilization of the insoluble pellet fraction of NLS-CG∗ (Figure 5A). Thus, none of the tested ligases individually or in combination are required for Apj1-dependent turnover of NLS-CG∗. We did not detect an alteration in NLS-CG∗ ubiquitylation in cells lacking Apj1 (Figure S5C). As expected from their known function in degrading misfolded soluble proteins, the soluble fraction of NLS-CG∗ was largely stable when Ubr1, San1, and Doa10 were deleted together (Figure 5A, soluble). In contrast, insoluble NLS-CG∗ was still turned over under this condition (Figure 5A, pellet). As observed before, the turnover of aggregated NLS-CG∗ was dependent on Apj1 and Hsp104, with complete stabilization only observed in the double deletion cells (Figure 5A, pellet; compare Figure 4A). Strikingly, deletion of Apj1 in Δubr1 Δsan1 Δdoa10 cells also resulted in an increase of soluble NLS-CG∗ during CHX shut-off, which was dependent on Hsp104 (Figure 5A, soluble). This further supports our model of two independent pathways competing in disaggregation, with one involving Apj1, which leads to direct degradation, and one dependent on Hsp104, not directly linked to proteolysis and, therefore, resulting in accumulation of soluble substrate in the absence of Apj1 (Figure 5B). We evaluated the physiological consequences of having only Hsp104-dependent disaggregation, a scenario resulting in soluble NLS-CG∗ accumulation in Δubr1 Δsan1 Δdoa10 cells. Deletion of Apj1 in a background lacking the three ubiquitin ligases increased the toxicity of NLS-CG∗ (Figure 5C). Strikingly, this additional toxicity was completely reverted by additional deletion of Hsp104 (Figure 5C), indicating that the observed toxicity is caused by a soluble substrate generated by Hsp104. These data are consistent with a protective effect of Apj1 caused by tightly coupling disaggregation with turnover because complete inhibition of disaggregation reverts the detrimental effect of Apj1 deletion (Figure 5C).Figure 5Apj1 Coordinates Disaggregation with Turnover(A) Role of nuclear protein quality control ligases in turnover of soluble and insoluble NLS-CG∗. Degradation of NLS-CG∗ in the indicated strains was analyzed as in Figure 2B. Averages ± SD from five independent experiments are shown.(B) Model for different outcomes of disaggregation reactions. Apj1-dependent disaggregation results in efficient turnover, whereas Hsp104-dependent disaggregation produces soluble protein. In absence of Apj1, Apj1 substrates are disaggregated by Hsp104 producing soluble protein.(C) Effect of Apj1 on NLS-CG∗ toxicity. Serial dilutions of the indicated strains were spotted on synthetic media containing galactose or glucose for the induction or repression of NLS-CG∗ expression, respectively. Cells were grown for 3 days (glucose) or 5 days (galactose) at 37°C.(D) Apj1 deletion is detrimental in cells with compromised proteasome activity. Serial dilutions of the indicated yeast strains were spotted on YP media containing glucose and grown for 2 days at the indicated temperatures.Our data suggest that Apj1 efficiently targets substrates for degradation. In line with this, we observe a negative synthetic growth defect when deleting Apj1 in a strain carrying the hypomorphic pre1-1 mutation of the 20S proteasomal core, which results in reduced proteasome activity and a growth phenotype at an elevated temperature (Figure 5D). This is consistent with a role for Apj1 in increasing the efficiency of substrate transfer to the proteasome, when proteasomal degradation is inefficient. Again, this effect was specific for Apj1 but was not observed with a deletion of Hsp104 (Figure 5D).In summary, these data suggest that during disaggregation, Apj1 promotes the turnover of resolubilized substrate, whereas Hsp104 fosters substrate refolding. In the absence of Apj1, Hsp104 generates soluble misfolded proteins, which are toxic if not readily degraded by the proteasome.Apj1 Acts on Nuclear Inclusions Containing a Broad Range of Cellular SubstratesTo identify endogenous substrates of Apj1, we analyzed the composition of Apj1-containing nuclear aggregates formed upon stress. We used the Hsp70-binding deficient Apj1 mutant Apj1AAA (Figures 3A and 3B) under conditions of heat stress or proteasomal inhibition, in which formation of intra-nuclear Apj1 foci are observed (Figure 1A). As for GFP-Apj1, the nuclear localization of GFP-Apj1AAA was unchanged under all conditions tested (Figure S6A). Moreover, GFP-Apj1AAA localized to nuclear foci upon stress (Figure S6A). We expressed GFP alone, GFP-Apj1, or GFP-Apj1AAA under conditions of heat stress or proteasome inhibition and performed anti-GFP immunoprecipitation, followed by mass spectrometry and label-free quantitation (Figure 6A). Putative Apj1 substrates were selected based on their significant enrichment with GFP-Apj1AAA compared to GFP alone and increased binding to GFP-Apj1AAA in comparison to GFP-Apj1 (Figure 6A). Based on these criteria, we identified 423 putative Apj1 substrates upon heat stress and 171 after proteasome inhibition, with an overlap of 22 proteins found in both conditions (Figure S6B; Table S1). The majority of putative Apj1 substrates identified upon heat stress were of nuclear origin (or have dual nuclear and cytoplasmic localization), including many nucleolar proteins (Figure 6B). In addition, numerous cytoplasmic proteins were detected (Figure 6B). Strikingly, after proteasome inhibition, more than half of the Apj1-interacting proteins were of cytoplasmic origin, including several mitochondrial proteins (Figure 6B).Figure 6Identification of Endogenous Apj1 Substrates(A) Strategy to identify putative Apj1 substrates by mass spectrometry. GFP, GFP-Apj1, and GFPAAA were purified from cells treated with heat shock or proteasome inhibition as in Figure 1A. Four technical replicates of each sample were analyzed by mass spectrometry by using label-free quantification. Proteins were classified as Apj1 substrates when significantly enriched with GFP-Apj1AAA compared with GFP alone and show increased binding to the substrate trap variant GFP-Apj1AAA compared with GFP-Apj1.(B) Annotated localization of the putative Apj1 substrates. Proteins with an ambiguous localization to both the nucleus and cytoplasm were classified as nuclear. Numbers indicate the quantity of proteins found in each group.(C) Interaction of Cse4 and Orc4 with Apj1 upon stress. C-terminally VN tagged Cse4 or Orc4, respectively, was co-expressed with untagged or VC-tagged Apj1 in cells expressing Nup49-mars and lacking endogenous Apj1 and Pdr5. Cells were analyzed untreated, upon acute heat stress or after 2h of MG132 treatment. Quantification shows averages ± SD from three independent experiments, each replicate represents at least 300 cells.(D) Interaction of Mrpl7 with Apj1 occurs inside the nucleus. Interaction of C-terminally VN-tagged Mrpl7 and Pgk1 was performed as described in (C). Quantification shows averages ± SD from three independent experiments; each replicate represents at least 300 cells.(E) Impact of protein synthesis and mitochondrial import on Apj1-Mrpl7 interaction. Cells expressing VN-Mrpl7 and Apj1-VC were analyzed as in (D) with the following modifications; VN-Mrpl7 synthesis prior to MG-132 addition was shut down by addition of glucose. Mitochondrial import was inhibited by adding carbonyl cyanide m-chlorophenyl hydrazine (CCCP). Quantification shows averages ± SD from three independent experiments; each replicate represents at least 300 cells.Our data suggest that Apj1 acts on a wide range of substrates upon different proteotoxic stresses and highlight the extent to which cytoplasmic proteins are imported into the nucleus under such conditions. To confirm that the observed interactions occur inside the nucleus, we used fluorescence complementation. The N-terminal half of the Venus fluorescent protein was fused to the C terminus of the putative substrate and co-expressed with Apj1 N-terminally fused to the C-terminal half of Venus (VC-Apj1). First, we tested the nuclear substrates Cse4 (Cse4-VN), a centromeric histone H3 variant, and Orc4 (Orc4-VN) of the origin recognition complex. Upon proteasome inhibition, we observed Venus fluorescent foci inside the nucleus in approximately 15% of the cells (Figure 6C), which were infrequently detected in unstressed cells. Likewise, we observed a small but reproducible increase in nuclear Venus foci upon acute heat stress (Figure 6C). Thus, Apj1 interacts with its substrates inside the nucleus under conditions that give rise to nuclear aggregate formation. To probe for nuclear localization of cytoplasmic Apj1 substrates, we used the mitochondrial ribosomal protein Mrpl7 (Mrpl7-VN). We observed Venus foci inside the nucleus in about 8% of the cells upon proteasome inhibition, demonstrating that the interaction between Mrpl7 and Apj1 occurs inside the nuclear interior (Figure 6D). This interaction was dependent on proteotoxic stress induced by proteasome inhibition but was not observed upon acute heat shock, in line with our mass spectrometry analyses (Figure 6D). In contrast, the highly soluble and stable cytoplasmic protein Pgk1 (Pgk1-VN) was barely observed to interact with Apj1 even under stress conditions (Figure 6D). Finally, we asked which pool of the mitochondrial protein Mrpl7 is targeted to nuclear foci. When inhibiting galactose-inducible synthesis of Mrpl7 by adding glucose 30 min before proteasome inhibition, Venus foci formed by VC-Apj1 and Mrpl7-VN were barely detected (Figure 6E). In contrast to this, addition of carbonyl cyanide m-chlorophenyl hydrazine (CCCP) strongly increased the number of cells with Mrpl7 containing Venus foci (Figure 6E). CCCP disrupts the mitochondrial membrane potential, thereby inhibiting ATP synthesis and mitochondrial import. The increase of nuclear Mrpl7 upon CCCP treatment might, therefore, be a consequence of failed mitochondrial import, although we cannot rule out that CCCP has additional effects on cellular physiology. The overall levels of Mrpl7-VN remained largely unchanged upon CCCP treatment (Figure S6C), suggesting that nuclear Mrpl7 represents a non-mitochondrially imported sub-fraction of this protein.In summary, we conclude that proteins of nuclear and cytoplasmic origin are sequestered into nuclear foci upon stress and that mitochondrial proteins present in these inclusions represent species, which failed to properly import into mitochondria. This highlights the general role of the nucleus in cellular proteostasis and the role of Apj1 in this pathway.

The Nuclear Hsp40 Apj1 Functions in Clearance of Misfolded ProteinsThe Hsp40 chaperone Apj1 has been previously found to localize to the intra-nuclear protein inclusion INQ (Gallina et al., 2015, Tkach et al., 2012). When evaluating the subcellular distribution of GFP-tagged Apj1 (GFP-Apj1) expressed from its endogenous promoter, we observed a strong enrichment in the nucleus (marked by Nup49-mars of the nuclear pore complex) (Figures 1A and S1A). Acute heat stress or proteasome inhibition resulted in coalescence of GFP-Apj1 into nuclear inclusions (Figures 1A and S1A), suggesting that Apj1 might be associated with nuclear protein aggregates. To mark the intra-nuclear quality control compartment INQ, we used heat-aggregated nuclear luciferase (LuciDM-NLS-GFP) (Miller et al., 2015). Indeed, GFP-tagged nuclear luciferase was found to co-localize with mars-tagged Apj1 (mars-Apj1) upon heat stress, confirming the previously published INQ localization of Apj1 (Figures 1B and S1B). Cell fractionation showed that Apj1 was soluble in unstressed cells (Figure 1C, −HS). Notably, the level of endogenous Apj1 strongly increased upon acute heat stress, and the majority of the protein shifted to the insoluble pellet fraction (Figure 1C, +HS), suggesting that Apj1 function is increasingly required under stress conditions that cause protein aggregation. Likewise, we observed increased amounts of the disaggregase Hsp104 as well as the Hsp40 Sis1 in the insoluble fraction upon stress, whereas the most abundant Hsp40, Ydj1, remained entirely soluble (Figure 1C +HS). Because Hsp104 and Sis1 have been previously implicated in protein disaggregation (Ho et al., 2019, Parsell et al., 1994), which might be reflected by their presence in the insoluble fraction, we speculated that Apj1 might function in recovering aggregated proteins from inclusions.Figure 1The Hsp40 Apj1 Mediates Turnover of Aggregation-Prone Nuclear Proteins(A) Nuclear localization of Apj1. GFP-Apj1 was expressed in cells harboring Nup49-mars and lacking the ABC transporter Pdr5 and was analyzed by live cell imaging. Cells were left untreated, subjected for 20 min to 42°C or treated for 2 h with the proteasome inhibitor MG-132. Dashed lines indicate cell boundaries.(B) Apj1 localizes to INQ. mars-Apj1 was co-expressed with GFP-tagged nuclear-targeted mutated luciferase (LuciDM-NLS-GFP). Formation of INQ was induced by 20-min heat shock at 42°C and analyzed as in (A).(C) Fractionation of Apj1 in untreated and heat-stressed cells. Total cell extracts (T) were solubilized and separated by centrifugation in detergent soluble (S) and insoluble pellet (P) fractions. Equal amounts of each fraction were loaded. Fractionation was analyzed using antibodies against endogenous Apj1, Ydj1, Sis1 and Hsp104. Soluble Pgk1 served as a control.(D) Effect of Apj1 on luciferase disaggregation. The indicated strains expressing LuciDM-NLS as in (B) were subjected to 42°C for 20 min to aggregate nuclear luciferase. Disaggregation was determined by measuring luciferase activity at different time points after heat shock as indicated. Luciferase activity measured before heat shock was set to 100%. Quantification shows averages ± SD from three independent experiments.(E) Turnover of LuciDM-NLS-GFP in Apj1-deficient cells. Degradation of LuciDM-NLS-GFP after heat shock was analyzed after addition of cycloheximide (CHX). LuciDM-NLS-GFP was detected using anti-GFP antibodies; Pgk1 serves a loading control. Quantification below shows averages ± SD from three independent experiments.(F) Schematic representation and subcellular distribution of the model substrate NLS-CG∗. ΔssCPY∗ is a mutant version of the secretory protein carboxypeptidase Y lacking the signal sequence; NLS, nuclear localization signal. Cells expressing NLS-CG∗ and Nup49-mars were analyzed by live cell microscopy under unstressed conditions as in (A).(G) Turnover of NLS-CG∗ in Apj1-deficient cells. Degradation of NLS-CG∗ was analyzed after addition of CHX. NLS-CG∗ was detected using anti-GFP antibodies; Pgk1 serves a loading control. Quantification below shows averages ± SD from three independent experiments.To analyze a possible role of Apj1 in recovering aggregated proteins following stress, we probed its role in refolding heat-aggregated nuclear luciferase by using LuciDM-NLS-GFP. We measured the recovery of in vivo luciferase activity after heat-induced luciferase inactivation, which was dependent on the disaggregase Hsp104 (Figure S1C). Surprisingly, we observed that recovery of luciferase activity was accelerated in the absence of Apj1 (Figure 1D). In addition, we observed an increase in luciferase stability after heat shock in cells lacking Apj1 (Figure 1E). Thus, Apj1 might counteract refolding of aggregated luciferase by competing with Hsp104 and Sis1-dependent disaggregation and appears to rather promote protein turnover. To better investigate a possible role of Apj1 in nuclear protein turnover, we used mutant carboxypeptidase Y fused to GFP (CG∗) as a terminally misfolded model protein (Figure 1F). As a secretory protein, carboxypeptidase Y is unable to fold in the reducing environment of the cytosol (Medicherla et al., 2004), and when expressed without a secretory signal sequence (CG∗), it is recognized by cytosolic chaperones and transported to the nucleus for proteasomal degradation (Heck et al., 2010, Park et al., 2007, Park et al., 2013, Prasad et al., 2010). We targeted CG∗ to the nucleus (NLS-CG∗), bypassing the requirement of chaperones for nuclear import (Park et al., 2013). Expression of NLS-CG∗ from the GAL1 promoter resulted in diffuse nuclear staining and formation of GFP-positive inclusions located both inside and outside the nucleus (Figures 1F and S1D). We next analyzed the turnover of NLS-CG∗ upon cycloheximide (CHX) treatment and observed a delayed turnover in cells lacking Apj1 (Figure 1G).In summary, we conclude that Apj1 is recruited to nuclear protein inclusions upon stress and supports the turnover of misfolded nuclear-targeted proteins.

The Nuclear Hsp40 Apj1 Functions in Clearance of Misfolded Proteins

The Hsp40 chaperone Apj1 has been previously found to localize to the intra-nuclear protein inclusion INQ (Gallina et al., 2015, Tkach et al., 2012). When evaluating the subcellular distribution of GFP-tagged Apj1 (GFP-Apj1) expressed from its endogenous promoter, we observed a strong enrichment in the nucleus (marked by Nup49-mars of the nuclear pore complex) (Figures 1A and S1A). Acute heat stress or proteasome inhibition resulted in coalescence of GFP-Apj1 into nuclear inclusions (Figures 1A and S1A), suggesting that Apj1 might be associated with nuclear protein aggregates. To mark the intra-nuclear quality control compartment INQ, we used heat-aggregated nuclear luciferase (LuciDM-NLS-GFP) (Miller et al., 2015). Indeed, GFP-tagged nuclear luciferase was found to co-localize with mars-tagged Apj1 (mars-Apj1) upon heat stress, confirming the previously published INQ localization of Apj1 (Figures 1B and S1B). Cell fractionation showed that Apj1 was soluble in unstressed cells (Figure 1C, −HS). Notably, the level of endogenous Apj1 strongly increased upon acute heat stress, and the majority of the protein shifted to the insoluble pellet fraction (Figure 1C, +HS), suggesting that Apj1 function is increasingly required under stress conditions that cause protein aggregation. Likewise, we observed increased amounts of the disaggregase Hsp104 as well as the Hsp40 Sis1 in the insoluble fraction upon stress, whereas the most abundant Hsp40, Ydj1, remained entirely soluble (Figure 1C +HS). Because Hsp104 and Sis1 have been previously implicated in protein disaggregation (Ho et al., 2019, Parsell et al., 1994), which might be reflected by their presence in the insoluble fraction, we speculated that Apj1 might function in recovering aggregated proteins from inclusions.Figure 1The Hsp40 Apj1 Mediates Turnover of Aggregation-Prone Nuclear Proteins(A) Nuclear localization of Apj1. GFP-Apj1 was expressed in cells harboring Nup49-mars and lacking the ABC transporter Pdr5 and was analyzed by live cell imaging. Cells were left untreated, subjected for 20 min to 42°C or treated for 2 h with the proteasome inhibitor MG-132. Dashed lines indicate cell boundaries.(B) Apj1 localizes to INQ. mars-Apj1 was co-expressed with GFP-tagged nuclear-targeted mutated luciferase (LuciDM-NLS-GFP). Formation of INQ was induced by 20-min heat shock at 42°C and analyzed as in (A).(C) Fractionation of Apj1 in untreated and heat-stressed cells. Total cell extracts (T) were solubilized and separated by centrifugation in detergent soluble (S) and insoluble pellet (P) fractions. Equal amounts of each fraction were loaded. Fractionation was analyzed using antibodies against endogenous Apj1, Ydj1, Sis1 and Hsp104. Soluble Pgk1 served as a control.(D) Effect of Apj1 on luciferase disaggregation. The indicated strains expressing LuciDM-NLS as in (B) were subjected to 42°C for 20 min to aggregate nuclear luciferase. Disaggregation was determined by measuring luciferase activity at different time points after heat shock as indicated. Luciferase activity measured before heat shock was set to 100%. Quantification shows averages ± SD from three independent experiments.(E) Turnover of LuciDM-NLS-GFP in Apj1-deficient cells. Degradation of LuciDM-NLS-GFP after heat shock was analyzed after addition of cycloheximide (CHX). LuciDM-NLS-GFP was detected using anti-GFP antibodies; Pgk1 serves a loading control. Quantification below shows averages ± SD from three independent experiments.(F) Schematic representation and subcellular distribution of the model substrate NLS-CG∗. ΔssCPY∗ is a mutant version of the secretory protein carboxypeptidase Y lacking the signal sequence; NLS, nuclear localization signal. Cells expressing NLS-CG∗ and Nup49-mars were analyzed by live cell microscopy under unstressed conditions as in (A).(G) Turnover of NLS-CG∗ in Apj1-deficient cells. Degradation of NLS-CG∗ was analyzed after addition of CHX. NLS-CG∗ was detected using anti-GFP antibodies; Pgk1 serves a loading control. Quantification below shows averages ± SD from three independent experiments.

The Hsp40 Apj1 Mediates Turnover of Aggregation-Prone Nuclear Proteins

(A) Nuclear localization of Apj1. GFP-Apj1 was expressed in cells harboring Nup49-mars and lacking the ABC transporter Pdr5 and was analyzed by live cell imaging. Cells were left untreated, subjected for 20 min to 42°C or treated for 2 h with the proteasome inhibitor MG-132. Dashed lines indicate cell boundaries.

(B) Apj1 localizes to INQ. mars-Apj1 was co-expressed with GFP-tagged nuclear-targeted mutated luciferase (LuciDM-NLS-GFP). Formation of INQ was induced by 20-min heat shock at 42°C and analyzed as in (A).

(C) Fractionation of Apj1 in untreated and heat-stressed cells. Total cell extracts (T) were solubilized and separated by centrifugation in detergent soluble (S) and insoluble pellet (P) fractions. Equal amounts of each fraction were loaded. Fractionation was analyzed using antibodies against endogenous Apj1, Ydj1, Sis1 and Hsp104. Soluble Pgk1 served as a control.

(D) Effect of Apj1 on luciferase disaggregation. The indicated strains expressing LuciDM-NLS as in (B) were subjected to 42°C for 20 min to aggregate nuclear luciferase. Disaggregation was determined by measuring luciferase activity at different time points after heat shock as indicated. Luciferase activity measured before heat shock was set to 100%. Quantification shows averages ± SD from three independent experiments.

(E) Turnover of LuciDM-NLS-GFP in Apj1-deficient cells. Degradation of LuciDM-NLS-GFP after heat shock was analyzed after addition of cycloheximide (CHX). LuciDM-NLS-GFP was detected using anti-GFP antibodies; Pgk1 serves a loading control. Quantification below shows averages ± SD from three independent experiments.

(F) Schematic representation and subcellular distribution of the model substrate NLS-CG∗. ΔssCPY∗ is a mutant version of the secretory protein carboxypeptidase Y lacking the signal sequence; NLS, nuclear localization signal. Cells expressing NLS-CG∗ and Nup49-mars were analyzed by live cell microscopy under unstressed conditions as in (A).

(G) Turnover of NLS-CG∗ in Apj1-deficient cells. Degradation of NLS-CG∗ was analyzed after addition of CHX. NLS-CG∗ was detected using anti-GFP antibodies; Pgk1 serves a loading control. Quantification below shows averages ± SD from three independent experiments.

To analyze a possible role of Apj1 in recovering aggregated proteins following stress, we probed its role in refolding heat-aggregated nuclear luciferase by using LuciDM-NLS-GFP. We measured the recovery of in vivo luciferase activity after heat-induced luciferase inactivation, which was dependent on the disaggregase Hsp104 (Figure S1C). Surprisingly, we observed that recovery of luciferase activity was accelerated in the absence of Apj1 (Figure 1D). In addition, we observed an increase in luciferase stability after heat shock in cells lacking Apj1 (Figure 1E). Thus, Apj1 might counteract refolding of aggregated luciferase by competing with Hsp104 and Sis1-dependent disaggregation and appears to rather promote protein turnover. To better investigate a possible role of Apj1 in nuclear protein turnover, we used mutant carboxypeptidase Y fused to GFP (CG∗) as a terminally misfolded model protein (Figure 1F). As a secretory protein, carboxypeptidase Y is unable to fold in the reducing environment of the cytosol (Medicherla et al., 2004), and when expressed without a secretory signal sequence (CG∗), it is recognized by cytosolic chaperones and transported to the nucleus for proteasomal degradation (Heck et al., 2010, Park et al., 2007, Park et al., 2013, Prasad et al., 2010). We targeted CG∗ to the nucleus (NLS-CG∗), bypassing the requirement of chaperones for nuclear import (Park et al., 2013). Expression of NLS-CG∗ from the GAL1 promoter resulted in diffuse nuclear staining and formation of GFP-positive inclusions located both inside and outside the nucleus (Figures 1F and S1D). We next analyzed the turnover of NLS-CG∗ upon cycloheximide (CHX) treatment and observed a delayed turnover in cells lacking Apj1 (Figure 1G).

In summary, we conclude that Apj1 is recruited to nuclear protein inclusions upon stress and supports the turnover of misfolded nuclear-targeted proteins.

Apj1 Acts on Nuclear Protein AggregatesThe localization of Apj1 to INQ and its fractionation to the insoluble pellet upon stress prompted us to investigate a possible role on nuclear protein aggregates. Cell fractionation revealed that about 50% of the Apj1 substrate NLS-CG∗ was insoluble under standard growth conditions (Figure 2A). This allowed us to compare the effect of Apj1 on both soluble and insoluble substrate pools. CHX shut-off followed by cell fractionation showed that a loss of Apj1 function substantially delayed the degradation of insoluble NLS-CG∗ (Figure 2B, pellet), but had only a minor effect on the turnover of soluble NLS-CG∗ (Figure 2B, soluble). Similar results were obtained without CHX addition, by blocking galactose-dependent NLS-CG∗ expression by addition of glucose (Figure S2A). These results suggested a role for Apj1 in clearing protein aggregates, in line with its accumulation in nuclear foci and insolubility upon stress.Figure 2Apj1 Acts on Nuclear Protein Aggregates(A) Fractionation of NLS-CG∗ in unstressed cells was performed as in Figure 1C. NLS-CG∗ was detected using anti-GFP antibodies. Quantification below shows averages ± SD from three independent experiments.(B) Turnover of soluble and insoluble NLS-CG∗ in the absence of Apj1. Degradation of NLS-CG∗ was followed after addition of CHX. Samples of each time point were fractionated as in (A). NLS-CG∗ was detected using anti-GFP antibodies. Soluble Pgk1 serves as a control. Quantification below shows averages ± SD from three independent experiments.(C) Localization of differentially targeted CG∗ variants. Distribution of NLS-CG∗ and NES-CG∗ following acute heat stress (20 min, 42°C) in cells harboring Nup49-mars was analyzed by live cell microscopy. NES, nuclear export signal.(D) Impact of Apj1 on the stability of differentially localized CG∗ variants. Turnover of NLS-CG∗ and NES-CG∗ after acute heat stress (20 min, 42°C) was followed as in Figure 1G. NLS-CG∗ and NES-CG∗ were detected using anti-GFP antibodies. Pgk1 serves as a control. Averages ± SD from three independent experiments are shown.(E) Effect of Apj1 and Btn2 deletion on NLS-CG∗ aggregation. The indicated strains were fractionated and analyzed as in (A). Averages ± SD from three independent experiments are shown.(F) Turnover of NLS-CG∗ in the absence of Btn2. Degradation of NLS-CG∗ was analyzed as in Figure 1G in the indicated strains. Averages ± SD from three independent experiments are shown.The dual localization of NLS-CG∗ foci to the nucleus and the cytoplasm (Figure 1F) raised the question as to which pool of the aggregated substrate is targeted by Apj1. CG∗ is efficiently excluded from the nucleus by addition of a nuclear export sequence (NES), which results in the cytoplasmic localization of the protein and its marked stabilization (Park et al., 2013). To achieve extensive aggregate formation, cells expressing NLS- and NES-CG∗ were exposed to heat stress. NLS-CG∗ formed nuclear and cytoplasmic foci, whereas NES-CG∗ formed exclusively cytoplasmic foci (Figures 2C and S2B). Deletion of Apj1 reduced the turnover of NLS-CG∗ following heat stress but had no effect on the slow degradation of cytoplasmic NES-CG∗ (Figure 2D). CG∗, without additional targeting sequence, is transported to the nucleus for degradation (Park et al., 2013). Importantly, turnover of CG∗ was also delayed in the absence of Apj1, demonstrating that its nuclear localization, but not the NLS itself, renders the substrate Apj1 dependent (Figure S2C).Sequestration of misfolded proteins into nuclear inclusions has been shown to be dependent on the chaperone Btn2 (Miller et al., 2015). Indeed, cells lacking Btn2 contained no visible nuclear NLS-CG∗ foci, independent of the presence of Apj1 (Figure S2D). We compared the amounts of aggregated NLS-CG∗ by using fractionation. As expected, deletion of Apj1 alone resulted in a strong accumulation of NLS-CG∗ in the insoluble pellet fraction (Figure 2E). In contrast, the loss of Btn2 reduced the amount of insoluble NLS-CG∗, in line with its requirement for nuclear aggregate formation (Figure 2E). Additional deletion of the cytosolic aggregase Hsp42 completely abolishes NLS-CG∗ aggregation (Figure 2E). This suggests that the residual amounts of aggregated NLS-CG∗ observed in absence of Btn2 represent cytosolic insoluble NLS-CG∗, consistent with NLS-CG∗ foci present in the cytosol (Figure 1F). Strikingly, additional deletion of Apj1 in cells lacking Btn2 did not result in an accumulation of aggregated NLS-CG∗, as observed in the presence of Btn2 (Figure 2E). In line with this, NLS-CG∗ was degraded with similar kinetics in wild-type (WT) and Δbtn2 and Δbtn2Δapj1 mutant cells (Figure 2F). Thus, deletion of Btn2 bypasses the requirement of Apj1 for efficient NLS-CG∗ degradation, suggesting that Apj1 is specifically required for the turnover of nuclear NLS-CG∗ aggregates generated by the action of Btn2.Collectively, these data support a role of Apj1 in specifically clearing nuclear protein aggregates.

Apj1 Acts on Nuclear Protein Aggregates

The localization of Apj1 to INQ and its fractionation to the insoluble pellet upon stress prompted us to investigate a possible role on nuclear protein aggregates. Cell fractionation revealed that about 50% of the Apj1 substrate NLS-CG∗ was insoluble under standard growth conditions (Figure 2A). This allowed us to compare the effect of Apj1 on both soluble and insoluble substrate pools. CHX shut-off followed by cell fractionation showed that a loss of Apj1 function substantially delayed the degradation of insoluble NLS-CG∗ (Figure 2B, pellet), but had only a minor effect on the turnover of soluble NLS-CG∗ (Figure 2B, soluble). Similar results were obtained without CHX addition, by blocking galactose-dependent NLS-CG∗ expression by addition of glucose (Figure S2A). These results suggested a role for Apj1 in clearing protein aggregates, in line with its accumulation in nuclear foci and insolubility upon stress.Figure 2Apj1 Acts on Nuclear Protein Aggregates(A) Fractionation of NLS-CG∗ in unstressed cells was performed as in Figure 1C. NLS-CG∗ was detected using anti-GFP antibodies. Quantification below shows averages ± SD from three independent experiments.(B) Turnover of soluble and insoluble NLS-CG∗ in the absence of Apj1. Degradation of NLS-CG∗ was followed after addition of CHX. Samples of each time point were fractionated as in (A). NLS-CG∗ was detected using anti-GFP antibodies. Soluble Pgk1 serves as a control. Quantification below shows averages ± SD from three independent experiments.(C) Localization of differentially targeted CG∗ variants. Distribution of NLS-CG∗ and NES-CG∗ following acute heat stress (20 min, 42°C) in cells harboring Nup49-mars was analyzed by live cell microscopy. NES, nuclear export signal.(D) Impact of Apj1 on the stability of differentially localized CG∗ variants. Turnover of NLS-CG∗ and NES-CG∗ after acute heat stress (20 min, 42°C) was followed as in Figure 1G. NLS-CG∗ and NES-CG∗ were detected using anti-GFP antibodies. Pgk1 serves as a control. Averages ± SD from three independent experiments are shown.(E) Effect of Apj1 and Btn2 deletion on NLS-CG∗ aggregation. The indicated strains were fractionated and analyzed as in (A). Averages ± SD from three independent experiments are shown.(F) Turnover of NLS-CG∗ in the absence of Btn2. Degradation of NLS-CG∗ was analyzed as in Figure 1G in the indicated strains. Averages ± SD from three independent experiments are shown.

Apj1 Acts on Nuclear Protein Aggregates

(A) Fractionation of NLS-CG∗ in unstressed cells was performed as in Figure 1C. NLS-CG∗ was detected using anti-GFP antibodies. Quantification below shows averages ± SD from three independent experiments.

(B) Turnover of soluble and insoluble NLS-CG∗ in the absence of Apj1. Degradation of NLS-CG∗ was followed after addition of CHX. Samples of each time point were fractionated as in (A). NLS-CG∗ was detected using anti-GFP antibodies. Soluble Pgk1 serves as a control. Quantification below shows averages ± SD from three independent experiments.

(C) Localization of differentially targeted CG∗ variants. Distribution of NLS-CG∗ and NES-CG∗ following acute heat stress (20 min, 42°C) in cells harboring Nup49-mars was analyzed by live cell microscopy. NES, nuclear export signal.

(D) Impact of Apj1 on the stability of differentially localized CG∗ variants. Turnover of NLS-CG∗ and NES-CG∗ after acute heat stress (20 min, 42°C) was followed as in Figure 1G. NLS-CG∗ and NES-CG∗ were detected using anti-GFP antibodies. Pgk1 serves as a control. Averages ± SD from three independent experiments are shown.

(E) Effect of Apj1 and Btn2 deletion on NLS-CG∗ aggregation. The indicated strains were fractionated and analyzed as in (A). Averages ± SD from three independent experiments are shown.

(F) Turnover of NLS-CG∗ in the absence of Btn2. Degradation of NLS-CG∗ was analyzed as in Figure 1G in the indicated strains. Averages ± SD from three independent experiments are shown.

The dual localization of NLS-CG∗ foci to the nucleus and the cytoplasm (Figure 1F) raised the question as to which pool of the aggregated substrate is targeted by Apj1. CG∗ is efficiently excluded from the nucleus by addition of a nuclear export sequence (NES), which results in the cytoplasmic localization of the protein and its marked stabilization (Park et al., 2013). To achieve extensive aggregate formation, cells expressing NLS- and NES-CG∗ were exposed to heat stress. NLS-CG∗ formed nuclear and cytoplasmic foci, whereas NES-CG∗ formed exclusively cytoplasmic foci (Figures 2C and S2B). Deletion of Apj1 reduced the turnover of NLS-CG∗ following heat stress but had no effect on the slow degradation of cytoplasmic NES-CG∗ (Figure 2D). CG∗, without additional targeting sequence, is transported to the nucleus for degradation (Park et al., 2013). Importantly, turnover of CG∗ was also delayed in the absence of Apj1, demonstrating that its nuclear localization, but not the NLS itself, renders the substrate Apj1 dependent (Figure S2C).

Sequestration of misfolded proteins into nuclear inclusions has been shown to be dependent on the chaperone Btn2 (Miller et al., 2015). Indeed, cells lacking Btn2 contained no visible nuclear NLS-CG∗ foci, independent of the presence of Apj1 (Figure S2D). We compared the amounts of aggregated NLS-CG∗ by using fractionation. As expected, deletion of Apj1 alone resulted in a strong accumulation of NLS-CG∗ in the insoluble pellet fraction (Figure 2E). In contrast, the loss of Btn2 reduced the amount of insoluble NLS-CG∗, in line with its requirement for nuclear aggregate formation (Figure 2E). Additional deletion of the cytosolic aggregase Hsp42 completely abolishes NLS-CG∗ aggregation (Figure 2E). This suggests that the residual amounts of aggregated NLS-CG∗ observed in absence of Btn2 represent cytosolic insoluble NLS-CG∗, consistent with NLS-CG∗ foci present in the cytosol (Figure 1F). Strikingly, additional deletion of Apj1 in cells lacking Btn2 did not result in an accumulation of aggregated NLS-CG∗, as observed in the presence of Btn2 (Figure 2E). In line with this, NLS-CG∗ was degraded with similar kinetics in wild-type (WT) and Δbtn2 and Δbtn2Δapj1 mutant cells (Figure 2F). Thus, deletion of Btn2 bypasses the requirement of Apj1 for efficient NLS-CG∗ degradation, suggesting that Apj1 is specifically required for the turnover of nuclear NLS-CG∗ aggregates generated by the action of Btn2.

Collectively, these data support a role of Apj1 in specifically clearing nuclear protein aggregates.

Apj1 Cooperates with Hsp70 and Hsp110 in Aggregate ClearanceIn line with the canonical function of Hsp40 proteins, it seemed plausible to assume that Apj1 cooperates with Hsp70. The J-domain of Hsp40 is critical for its interaction with Hsp70, and indeed, we observed an interaction of Apj1 with the Hsp70 proteins Ssa1, Ssa3, and Ssa4 (Ssa2 was not tested due to high similarity with Ssa1) that was lost upon J-domain deletion (Figure S3A). In line with an Hsp70-dependent role of Apj1 in NLS-CG∗ degradation, we observed that Apj1 lacking the J-domain does not support NLS-CG∗ turnover (Figure S3B). Stimulation of the Hsp70 ATPase by Hsp40 and transfer of substrates from Hsp40 to Hsp70 depends on the conserved His-Pro-Asp (HPD)-loop segment in the J-domain of Hsp40 (Kampinga and Craig, 2010). To trap substrates on Apj1, we mutated the HPD-loop residues of Apj1 to alanine (Apj1AAA). Apj1AAA failed to restore the delayed degradation of NLS-CG∗ in Δapj1 cells (Figure 3A), consistent with Apj1 functionally cooperating with Hsp70. Moreover, immunoprecipitation of NLS-CG∗ from cell extracts resulted in efficient co-precipitation of Apj1AAA, whereas WT Apj1 was co-precipitated with lower efficiency (Figure 3B). These data demonstrate that Apj1 transfers substrates to Hsp70 and requires Hsp70 interaction for supporting NLS-CG∗ turnover.Figure 3Apj1-Dependent Turnover Requires Hsp70 and Hsp110(A) Functionality of Apj1AAA. Degradation of NLS-CG∗ was analyzed as in Figure 1G in Δapj1 cells expressing the indicated plasmids. Averages ± SD from three independent experiments are shown.(B) Interaction of Apj1 with NLS-CG∗. NLS-CG∗ or just GFP was co-expressed with the indicated Apj1 variants in Δapj1 cells. NLS-CG∗ was purified from cell extracts using GFP-trap beads. Binding proteins were analyzed by western blotting using GFP and Apj1-specific antibodies. IP, immunoprecipitation.(C) Role of Hsp70 NEFs in degrading soluble and insoluble NLS-CG∗. Degradation of NLS-CG∗ in the indicated strains was analyzed as in Figure 2B.(D) Role of Sse1 localization on NLS-CG∗ turnover. Empty vector or the indicated Sse1 variants were co-expressed with NLS-CG∗ in the indicated strains. Degradation of soluble and insoluble NLS-CG∗ was analyzed as in Figure 2B. Averages ± SD from three independent experiments are shown.We next asked which of the Hsp70 NEFs might be involved in Apj1-dependent turnover of aggregated NLS-CG∗. We screened all known cytosolic/nuclear NEFs for their role in clearing NLS-CG∗ aggregates after heat stress, including the Hsp110 homologs Sse1 and Sse2, and the unrelated NEFs Fes1 and Snl1. Only deletion of Sse1 strongly stabilized NLS-CG∗ (Figure S3C). Similarly we found that in non-stressed cells, Sse1 was the only NEF required for degradation of insoluble NLS-CG∗ (Figure 3C). However, we also observed a stabilization of soluble NLS-CG when Sse1 was deleted (Figure 3C). This points to a more general effect of Sse1 deletion on Hsp70 function, as substrate release from Hsp70 will be strongly impaired, thus reducing total Hsp70 capacity. To address a possible nuclear role of Sse1, we targeted or excluded this NEF by addition of an NLS or NES, respectively. At normal growth temperature (30°C) both NLS- and NES-Sse1 restored the strong growth defect of cells lacking Sse1 (Figure S3D), in line with Sse1 deletion generally impairing cellular Hsp70 function. Importantly, only NLS-Sse1, but not NES-Sse1, fully restored growth at an elevated temperature (37°C), suggesting a nuclear-specific function for Sse1 (Figure S3D). We next analyzed the impact of the different Sse1 variants on NLS-CG∗ turnover. In line with a general impairment of Hsp70 function in Sse1 deficient cells, we observed that NLS- and NES-Sse1 supported the turnover of soluble NLS-CG∗ to the same extent as Sse1 without additional targeting sequences (Figure 3D, soluble). In contrast, only NLS-Sse1 supported the Apj1-dependent clearance of aggregated NLS-CG∗, and the turnover in the presence of NES-Sse1 was similar to Apj1-deficient cells (Figure 3D, pellet).Together, these data are consistent with Apj1 co-operating with Hsp70 and the NEF Sse1 in clearing nuclear proteins aggregates.

Apj1 Cooperates with Hsp70 and Hsp110 in Aggregate Clearance

In line with the canonical function of Hsp40 proteins, it seemed plausible to assume that Apj1 cooperates with Hsp70. The J-domain of Hsp40 is critical for its interaction with Hsp70, and indeed, we observed an interaction of Apj1 with the Hsp70 proteins Ssa1, Ssa3, and Ssa4 (Ssa2 was not tested due to high similarity with Ssa1) that was lost upon J-domain deletion (Figure S3A). In line with an Hsp70-dependent role of Apj1 in NLS-CG∗ degradation, we observed that Apj1 lacking the J-domain does not support NLS-CG∗ turnover (Figure S3B). Stimulation of the Hsp70 ATPase by Hsp40 and transfer of substrates from Hsp40 to Hsp70 depends on the conserved His-Pro-Asp (HPD)-loop segment in the J-domain of Hsp40 (Kampinga and Craig, 2010). To trap substrates on Apj1, we mutated the HPD-loop residues of Apj1 to alanine (Apj1AAA). Apj1AAA failed to restore the delayed degradation of NLS-CG∗ in Δapj1 cells (Figure 3A), consistent with Apj1 functionally cooperating with Hsp70. Moreover, immunoprecipitation of NLS-CG∗ from cell extracts resulted in efficient co-precipitation of Apj1AAA, whereas WT Apj1 was co-precipitated with lower efficiency (Figure 3B). These data demonstrate that Apj1 transfers substrates to Hsp70 and requires Hsp70 interaction for supporting NLS-CG∗ turnover.Figure 3Apj1-Dependent Turnover Requires Hsp70 and Hsp110(A) Functionality of Apj1AAA. Degradation of NLS-CG∗ was analyzed as in Figure 1G in Δapj1 cells expressing the indicated plasmids. Averages ± SD from three independent experiments are shown.(B) Interaction of Apj1 with NLS-CG∗. NLS-CG∗ or just GFP was co-expressed with the indicated Apj1 variants in Δapj1 cells. NLS-CG∗ was purified from cell extracts using GFP-trap beads. Binding proteins were analyzed by western blotting using GFP and Apj1-specific antibodies. IP, immunoprecipitation.(C) Role of Hsp70 NEFs in degrading soluble and insoluble NLS-CG∗. Degradation of NLS-CG∗ in the indicated strains was analyzed as in Figure 2B.(D) Role of Sse1 localization on NLS-CG∗ turnover. Empty vector or the indicated Sse1 variants were co-expressed with NLS-CG∗ in the indicated strains. Degradation of soluble and insoluble NLS-CG∗ was analyzed as in Figure 2B. Averages ± SD from three independent experiments are shown.

Apj1-Dependent Turnover Requires Hsp70 and Hsp110

(A) Functionality of Apj1AAA. Degradation of NLS-CG∗ was analyzed as in Figure 1G in Δapj1 cells expressing the indicated plasmids. Averages ± SD from three independent experiments are shown.

(B) Interaction of Apj1 with NLS-CG∗. NLS-CG∗ or just GFP was co-expressed with the indicated Apj1 variants in Δapj1 cells. NLS-CG∗ was purified from cell extracts using GFP-trap beads. Binding proteins were analyzed by western blotting using GFP and Apj1-specific antibodies. IP, immunoprecipitation.

(C) Role of Hsp70 NEFs in degrading soluble and insoluble NLS-CG∗. Degradation of NLS-CG∗ in the indicated strains was analyzed as in Figure 2B.

(D) Role of Sse1 localization on NLS-CG∗ turnover. Empty vector or the indicated Sse1 variants were co-expressed with NLS-CG∗ in the indicated strains. Degradation of soluble and insoluble NLS-CG∗ was analyzed as in Figure 2B. Averages ± SD from three independent experiments are shown.

We next asked which of the Hsp70 NEFs might be involved in Apj1-dependent turnover of aggregated NLS-CG∗. We screened all known cytosolic/nuclear NEFs for their role in clearing NLS-CG∗ aggregates after heat stress, including the Hsp110 homologs Sse1 and Sse2, and the unrelated NEFs Fes1 and Snl1. Only deletion of Sse1 strongly stabilized NLS-CG∗ (Figure S3C). Similarly we found that in non-stressed cells, Sse1 was the only NEF required for degradation of insoluble NLS-CG∗ (Figure 3C). However, we also observed a stabilization of soluble NLS-CG when Sse1 was deleted (Figure 3C). This points to a more general effect of Sse1 deletion on Hsp70 function, as substrate release from Hsp70 will be strongly impaired, thus reducing total Hsp70 capacity. To address a possible nuclear role of Sse1, we targeted or excluded this NEF by addition of an NLS or NES, respectively. At normal growth temperature (30°C) both NLS- and NES-Sse1 restored the strong growth defect of cells lacking Sse1 (Figure S3D), in line with Sse1 deletion generally impairing cellular Hsp70 function. Importantly, only NLS-Sse1, but not NES-Sse1, fully restored growth at an elevated temperature (37°C), suggesting a nuclear-specific function for Sse1 (Figure S3D). We next analyzed the impact of the different Sse1 variants on NLS-CG∗ turnover. In line with a general impairment of Hsp70 function in Sse1 deficient cells, we observed that NLS- and NES-Sse1 supported the turnover of soluble NLS-CG∗ to the same extent as Sse1 without additional targeting sequences (Figure 3D, soluble). In contrast, only NLS-Sse1 supported the Apj1-dependent clearance of aggregated NLS-CG∗, and the turnover in the presence of NES-Sse1 was similar to Apj1-deficient cells (Figure 3D, pellet).

Together, these data are consistent with Apj1 co-operating with Hsp70 and the NEF Sse1 in clearing nuclear proteins aggregates.

Apj1 Mediates Hsp104-Independent Protein DisaggregationTo date, all protein disaggregation processes described in yeast depend on Hsp104. Thus, we analyzed the requirement of Hsp104 for the degradation of insoluble NLS-CG∗ recovered in the pellet fraction. Deletion of HSP104 had a stabilizing effect on NLS-CG∗ aggregates, which was surprisingly less pronounced than the stabilization observed in Δapj1 cells (Figure 4A). Importantly, turnover of insoluble NLS-CG∗ was completely blocked when both Apj1 and Hsp104 were deleted simultaneously (Figure 4A). This suggests that Apj1 and Hsp104 function independently in clearing aggregated NLS-CG∗. Because NLS-CG∗ also forms cytoplasmic aggregates, we asked whether both Apj1 and Hsp104 are recruited to aggregates in the nucleus. When co-expressing mars-Apj1AAA with NLS-CG∗, we found that Apj1 co-localized only with nuclear, but not cytoplasmic inclusions, of NLS-CG∗ (Figures 4B and S4A). In contrast, mars-tagged Hsp104 (Hsp104-mars) co-localized with both cytoplasmic and nuclear NLS-CG∗ inclusions (Figures 4B and S4A). These data are consistent with Apj1 and Hsp104 both acting on nuclear protein aggregates.Figure 4Apj1 Supports Disaggregation Independent of Hsp104 and Sis1(A) Effect of Apj1 and Hsp104 on degradation of insoluble NLS-CG∗. Degradation of insoluble NLS-CG∗ (pellet fraction) in the indicated strains was analyzed as in Figure 2B. Averages ± SD from three independent experiments are shown.(B) Co-localization of NLS-CG∗ with Apj1AAA and Hsp104. mars-tagged Apj1AAA was co-expressed with NLS-CG∗ in Δapj1 cells, and genomically tagged Hsp104-mars was co-expressed with NLS-CG∗. Cells were analyzed by live-cell imaging as in Figure 1A. Dashed lines indicate position of nucleus. Nuclei were counterstained with Hoechst. N, nuclear; C, cytoplasmic foci.(C) Interplay between Sis1 and Apj1 in NLS-CG∗ turnover. The indicated strains expressing NLS-CG∗ were grown in presence of doxycycline (Dox) and analyzed as in Figure 2B. Averages ± SD from four independent experiments are shown. Significance was calculated using Student’s t test, ∗ p < 0.05.(D) Synthetic growth defect between Apj1 and Sis1. Serial dilutions of the indicated yeast strains pre-grown with (+) or without (−) Dox were spotted onto YPD plates containing Dox and incubated at the indicated temperatures.(E) Analysis of Apj1-dependent disaggregation in vitro. Luciferase was aggregated at 42°C for 20 min. Luciferase activities were determined after a 240 min incubation at 30°C in the presence of the indicated chaperones. The remaining sample was subjected to high-speed centrifugation, and the soluble supernatant was analyzed by western blotting against luciferase. The activity and solubility of native luciferase was set as 100%. Averages ± SD from three independent experiments are shown. wo HS, without heat shock.To address the possible involvement of other Hsp40 chaperones in NLS-CG∗ turnover, we analyzed the impact of the three cytosolic or nuclear class A Hsp40 chaperones (Apj1, Ydj1, and Xdj1) and three class B Hsp40s (Sis1, Caj1, and Djp1) present in yeast. Hsp40 chaperones may either function in aggregation prevention or in disaggregation. Testing the solubility of NLS-CG∗ in cells deficient for individual Hsp40 chaperones showed that the loss of Ydj1 and Sis1 resulted in complete aggregation of NLS-CG∗, implying a function in maintaining the protein in a soluble state (Figure S4B, −HS). To screen for a possible involvement in clearing aggregated NLS-CG∗, we performed CHX chase experiments upon heat stress, where most NLS-CG∗ is insoluble (Figure S4B, +HS). In addition to Apj1, only the class B Hsp40 Sis1 was required for efficient NLS-CG∗ turnover, whereas the other Hsp40 chaperones tested were dispensable (Figure S4C). In addition to its function in trafficking misfolded proteins to the nucleus, Sis1 has recently been shown to promote refolding of heat-aggregated luciferase (Ho et al., 2019). To test if Apj1 co-operates with Sis1, we analyzed the turnover of aggregated NLS-CG∗ in Sis1-depleted cells with or without Apj1. Although Sis1 depletion alone already strongly delayed removal of insoluble NLS-CG∗, we observed a small but significant increase in stabilization when deleting Apj1 in addition (Figure 4C). These findings are reminiscent of the full stabilization of insoluble NLS-CG∗ upon Apj1 and Hsp104 deletion (Figure 4A). This might suggest independent roles for Apj1 and Sis1 in clearing aggregated NLS-CG∗. In agreement with independent but overlapping actions of the two Hsp40s, we observed a strong negative synthetic growth defect at high temperatures when downregulating Sis1 in cells lacking Apj1 (Figures 4D and S4D).To directly show that Apj1 supports protein disaggregation, we tested for this activity in vitro. We used heat-aggregated luciferase as a substrate and measured its recovery by disaggregation. To this end, we compared the soluble protein and the amount of active luciferase obtained after adding the indicated combination of chaperones and subsequent incubation for 240 min at 30°C. As previously shown, aggregated luciferase is efficiently re-solubilized and refolded by the Hsp70-Hsp104 bi-chaperone system in conjunction with the Hsp40 Sis1 and a NEF (here, Sse1) (Figures 4E and S4E) (Ho et al., 2019). We observed that Sis1 efficiently supported disaggregation and refolding of heat-aggregated luciferase, which was reflected by high amounts of solubilized, active luciferase (Figures 4E and S4E). Luciferase refolding was low with Apj1 as the sole Hsp40 in the reaction; however, a substantial amount of luciferase was detected in the soluble fraction of this sample (Figures 4E and S4E). Strikingly, the observed Apj1 activity in solubilizing luciferase was independent of Hsp104 (Figure 4E). In contrast, the amount of luciferase solubilized by Sis1 activity was strongly reduced in the absence of Hsp104, and its Hsp104-independent disaggregation activity was lower than the activity of Apj1 (Figure 4E). Using equal amounts of Apj1 and Sis1 (with the same final Hsp40 concentration as before) in the disaggregation reaction resulted in comparable amounts of soluble luciferase compared with Sis1 alone, and we observed a reduction in luciferase refolding (Figure 4E). This is in line with both Hsp40 chaperones acting independently, with Apj1 generating non-native, soluble luciferase species.We conclude that two independent pathways for protein disaggregation exist in the nucleus. One involves Sis1 and Hsp104 as previously described, and a second pathway involves Apj1 in concert with Hsp70 and Hsp110 but independent of Hsp104. Importantly, Apj1-dependent disaggregation produces soluble species but does not support refolding, which is consistent with a function in supporting proteolytic turnover of disaggregated proteins.

Apj1 Mediates Hsp104-Independent Protein Disaggregation

To date, all protein disaggregation processes described in yeast depend on Hsp104. Thus, we analyzed the requirement of Hsp104 for the degradation of insoluble NLS-CG∗ recovered in the pellet fraction. Deletion of HSP104 had a stabilizing effect on NLS-CG∗ aggregates, which was surprisingly less pronounced than the stabilization observed in Δapj1 cells (Figure 4A). Importantly, turnover of insoluble NLS-CG∗ was completely blocked when both Apj1 and Hsp104 were deleted simultaneously (Figure 4A). This suggests that Apj1 and Hsp104 function independently in clearing aggregated NLS-CG∗. Because NLS-CG∗ also forms cytoplasmic aggregates, we asked whether both Apj1 and Hsp104 are recruited to aggregates in the nucleus. When co-expressing mars-Apj1AAA with NLS-CG∗, we found that Apj1 co-localized only with nuclear, but not cytoplasmic inclusions, of NLS-CG∗ (Figures 4B and S4A). In contrast, mars-tagged Hsp104 (Hsp104-mars) co-localized with both cytoplasmic and nuclear NLS-CG∗ inclusions (Figures 4B and S4A). These data are consistent with Apj1 and Hsp104 both acting on nuclear protein aggregates.Figure 4Apj1 Supports Disaggregation Independent of Hsp104 and Sis1(A) Effect of Apj1 and Hsp104 on degradation of insoluble NLS-CG∗. Degradation of insoluble NLS-CG∗ (pellet fraction) in the indicated strains was analyzed as in Figure 2B. Averages ± SD from three independent experiments are shown.(B) Co-localization of NLS-CG∗ with Apj1AAA and Hsp104. mars-tagged Apj1AAA was co-expressed with NLS-CG∗ in Δapj1 cells, and genomically tagged Hsp104-mars was co-expressed with NLS-CG∗. Cells were analyzed by live-cell imaging as in Figure 1A. Dashed lines indicate position of nucleus. Nuclei were counterstained with Hoechst. N, nuclear; C, cytoplasmic foci.(C) Interplay between Sis1 and Apj1 in NLS-CG∗ turnover. The indicated strains expressing NLS-CG∗ were grown in presence of doxycycline (Dox) and analyzed as in Figure 2B. Averages ± SD from four independent experiments are shown. Significance was calculated using Student’s t test, ∗ p < 0.05.(D) Synthetic growth defect between Apj1 and Sis1. Serial dilutions of the indicated yeast strains pre-grown with (+) or without (−) Dox were spotted onto YPD plates containing Dox and incubated at the indicated temperatures.(E) Analysis of Apj1-dependent disaggregation in vitro. Luciferase was aggregated at 42°C for 20 min. Luciferase activities were determined after a 240 min incubation at 30°C in the presence of the indicated chaperones. The remaining sample was subjected to high-speed centrifugation, and the soluble supernatant was analyzed by western blotting against luciferase. The activity and solubility of native luciferase was set as 100%. Averages ± SD from three independent experiments are shown. wo HS, without heat shock.

Apj1 Supports Disaggregation Independent of Hsp104 and Sis1

(A) Effect of Apj1 and Hsp104 on degradation of insoluble NLS-CG∗. Degradation of insoluble NLS-CG∗ (pellet fraction) in the indicated strains was analyzed as in Figure 2B. Averages ± SD from three independent experiments are shown.

(B) Co-localization of NLS-CG∗ with Apj1AAA and Hsp104. mars-tagged Apj1AAA was co-expressed with NLS-CG∗ in Δapj1 cells, and genomically tagged Hsp104-mars was co-expressed with NLS-CG∗. Cells were analyzed by live-cell imaging as in Figure 1A. Dashed lines indicate position of nucleus. Nuclei were counterstained with Hoechst. N, nuclear; C, cytoplasmic foci.

(C) Interplay between Sis1 and Apj1 in NLS-CG∗ turnover. The indicated strains expressing NLS-CG∗ were grown in presence of doxycycline (Dox) and analyzed as in Figure 2B. Averages ± SD from four independent experiments are shown. Significance was calculated using Student’s t test, ∗ p < 0.05.

(D) Synthetic growth defect between Apj1 and Sis1. Serial dilutions of the indicated yeast strains pre-grown with (+) or without (−) Dox were spotted onto YPD plates containing Dox and incubated at the indicated temperatures.

(E) Analysis of Apj1-dependent disaggregation in vitro. Luciferase was aggregated at 42°C for 20 min. Luciferase activities were determined after a 240 min incubation at 30°C in the presence of the indicated chaperones. The remaining sample was subjected to high-speed centrifugation, and the soluble supernatant was analyzed by western blotting against luciferase. The activity and solubility of native luciferase was set as 100%. Averages ± SD from three independent experiments are shown. wo HS, without heat shock.

To address the possible involvement of other Hsp40 chaperones in NLS-CG∗ turnover, we analyzed the impact of the three cytosolic or nuclear class A Hsp40 chaperones (Apj1, Ydj1, and Xdj1) and three class B Hsp40s (Sis1, Caj1, and Djp1) present in yeast. Hsp40 chaperones may either function in aggregation prevention or in disaggregation. Testing the solubility of NLS-CG∗ in cells deficient for individual Hsp40 chaperones showed that the loss of Ydj1 and Sis1 resulted in complete aggregation of NLS-CG∗, implying a function in maintaining the protein in a soluble state (Figure S4B, −HS). To screen for a possible involvement in clearing aggregated NLS-CG∗, we performed CHX chase experiments upon heat stress, where most NLS-CG∗ is insoluble (Figure S4B, +HS). In addition to Apj1, only the class B Hsp40 Sis1 was required for efficient NLS-CG∗ turnover, whereas the other Hsp40 chaperones tested were dispensable (Figure S4C). In addition to its function in trafficking misfolded proteins to the nucleus, Sis1 has recently been shown to promote refolding of heat-aggregated luciferase (Ho et al., 2019). To test if Apj1 co-operates with Sis1, we analyzed the turnover of aggregated NLS-CG∗ in Sis1-depleted cells with or without Apj1. Although Sis1 depletion alone already strongly delayed removal of insoluble NLS-CG∗, we observed a small but significant increase in stabilization when deleting Apj1 in addition (Figure 4C). These findings are reminiscent of the full stabilization of insoluble NLS-CG∗ upon Apj1 and Hsp104 deletion (Figure 4A). This might suggest independent roles for Apj1 and Sis1 in clearing aggregated NLS-CG∗. In agreement with independent but overlapping actions of the two Hsp40s, we observed a strong negative synthetic growth defect at high temperatures when downregulating Sis1 in cells lacking Apj1 (Figures 4D and S4D).

To directly show that Apj1 supports protein disaggregation, we tested for this activity in vitro. We used heat-aggregated luciferase as a substrate and measured its recovery by disaggregation. To this end, we compared the soluble protein and the amount of active luciferase obtained after adding the indicated combination of chaperones and subsequent incubation for 240 min at 30°C. As previously shown, aggregated luciferase is efficiently re-solubilized and refolded by the Hsp70-Hsp104 bi-chaperone system in conjunction with the Hsp40 Sis1 and a NEF (here, Sse1) (Figures 4E and S4E) (Ho et al., 2019). We observed that Sis1 efficiently supported disaggregation and refolding of heat-aggregated luciferase, which was reflected by high amounts of solubilized, active luciferase (Figures 4E and S4E). Luciferase refolding was low with Apj1 as the sole Hsp40 in the reaction; however, a substantial amount of luciferase was detected in the soluble fraction of this sample (Figures 4E and S4E). Strikingly, the observed Apj1 activity in solubilizing luciferase was independent of Hsp104 (Figure 4E). In contrast, the amount of luciferase solubilized by Sis1 activity was strongly reduced in the absence of Hsp104, and its Hsp104-independent disaggregation activity was lower than the activity of Apj1 (Figure 4E). Using equal amounts of Apj1 and Sis1 (with the same final Hsp40 concentration as before) in the disaggregation reaction resulted in comparable amounts of soluble luciferase compared with Sis1 alone, and we observed a reduction in luciferase refolding (Figure 4E). This is in line with both Hsp40 chaperones acting independently, with Apj1 generating non-native, soluble luciferase species.

We conclude that two independent pathways for protein disaggregation exist in the nucleus. One involves Sis1 and Hsp104 as previously described, and a second pathway involves Apj1 in concert with Hsp70 and Hsp110 but independent of Hsp104. Importantly, Apj1-dependent disaggregation produces soluble species but does not support refolding, which is consistent with a function in supporting proteolytic turnover of disaggregated proteins.

Apj1 Co-ordinates Disaggregation with TurnoverThe observation that Apj1 mediates turnover of nuclear protein aggregates in vivo prompted us to investigate the role of the UPS in this pathway. We observed that both soluble and insoluble pools of NLS-CG∗ are efficiently degraded in a proteasome-dependent manner, as indicated by marked stabilization of NLS-CG∗ upon proteasome inhibition (Figure S5A). Next, we tested the three ubiquitin ligases Ubr1, San1, and Doa10, which have been previously linked to nuclear quality control and turnover of CG∗/NLS-CG∗ (Heck et al., 2010, Samant et al., 2018). Deletion of Apj1 in the absence of San1 or Doa10 resulted in additional stabilization of insoluble NLS-CG∗, suggesting they act on alternative pathways (Figure S5B). In contrast, deletion of Apj1 in cells lacking Ubr1 had no additional effect on the turnover of aggregated NLS-CG∗ (Figure S5B). However, in a triple deletion of Ubr1 with San1 and Doa10, a lack of Apj1 still resulted in a strong stabilization of the insoluble pellet fraction of NLS-CG∗ (Figure 5A). Thus, none of the tested ligases individually or in combination are required for Apj1-dependent turnover of NLS-CG∗. We did not detect an alteration in NLS-CG∗ ubiquitylation in cells lacking Apj1 (Figure S5C). As expected from their known function in degrading misfolded soluble proteins, the soluble fraction of NLS-CG∗ was largely stable when Ubr1, San1, and Doa10 were deleted together (Figure 5A, soluble). In contrast, insoluble NLS-CG∗ was still turned over under this condition (Figure 5A, pellet). As observed before, the turnover of aggregated NLS-CG∗ was dependent on Apj1 and Hsp104, with complete stabilization only observed in the double deletion cells (Figure 5A, pellet; compare Figure 4A). Strikingly, deletion of Apj1 in Δubr1 Δsan1 Δdoa10 cells also resulted in an increase of soluble NLS-CG∗ during CHX shut-off, which was dependent on Hsp104 (Figure 5A, soluble). This further supports our model of two independent pathways competing in disaggregation, with one involving Apj1, which leads to direct degradation, and one dependent on Hsp104, not directly linked to proteolysis and, therefore, resulting in accumulation of soluble substrate in the absence of Apj1 (Figure 5B). We evaluated the physiological consequences of having only Hsp104-dependent disaggregation, a scenario resulting in soluble NLS-CG∗ accumulation in Δubr1 Δsan1 Δdoa10 cells. Deletion of Apj1 in a background lacking the three ubiquitin ligases increased the toxicity of NLS-CG∗ (Figure 5C). Strikingly, this additional toxicity was completely reverted by additional deletion of Hsp104 (Figure 5C), indicating that the observed toxicity is caused by a soluble substrate generated by Hsp104. These data are consistent with a protective effect of Apj1 caused by tightly coupling disaggregation with turnover because complete inhibition of disaggregation reverts the detrimental effect of Apj1 deletion (Figure 5C).Figure 5Apj1 Coordinates Disaggregation with Turnover(A) Role of nuclear protein quality control ligases in turnover of soluble and insoluble NLS-CG∗. Degradation of NLS-CG∗ in the indicated strains was analyzed as in Figure 2B. Averages ± SD from five independent experiments are shown.(B) Model for different outcomes of disaggregation reactions. Apj1-dependent disaggregation results in efficient turnover, whereas Hsp104-dependent disaggregation produces soluble protein. In absence of Apj1, Apj1 substrates are disaggregated by Hsp104 producing soluble protein.(C) Effect of Apj1 on NLS-CG∗ toxicity. Serial dilutions of the indicated strains were spotted on synthetic media containing galactose or glucose for the induction or repression of NLS-CG∗ expression, respectively. Cells were grown for 3 days (glucose) or 5 days (galactose) at 37°C.(D) Apj1 deletion is detrimental in cells with compromised proteasome activity. Serial dilutions of the indicated yeast strains were spotted on YP media containing glucose and grown for 2 days at the indicated temperatures.Our data suggest that Apj1 efficiently targets substrates for degradation. In line with this, we observe a negative synthetic growth defect when deleting Apj1 in a strain carrying the hypomorphic pre1-1 mutation of the 20S proteasomal core, which results in reduced proteasome activity and a growth phenotype at an elevated temperature (Figure 5D). This is consistent with a role for Apj1 in increasing the efficiency of substrate transfer to the proteasome, when proteasomal degradation is inefficient. Again, this effect was specific for Apj1 but was not observed with a deletion of Hsp104 (Figure 5D).In summary, these data suggest that during disaggregation, Apj1 promotes the turnover of resolubilized substrate, whereas Hsp104 fosters substrate refolding. In the absence of Apj1, Hsp104 generates soluble misfolded proteins, which are toxic if not readily degraded by the proteasome.

Apj1 Co-ordinates Disaggregation with Turnover

The observation that Apj1 mediates turnover of nuclear protein aggregates in vivo prompted us to investigate the role of the UPS in this pathway. We observed that both soluble and insoluble pools of NLS-CG∗ are efficiently degraded in a proteasome-dependent manner, as indicated by marked stabilization of NLS-CG∗ upon proteasome inhibition (Figure S5A). Next, we tested the three ubiquitin ligases Ubr1, San1, and Doa10, which have been previously linked to nuclear quality control and turnover of CG∗/NLS-CG∗ (Heck et al., 2010, Samant et al., 2018). Deletion of Apj1 in the absence of San1 or Doa10 resulted in additional stabilization of insoluble NLS-CG∗, suggesting they act on alternative pathways (Figure S5B). In contrast, deletion of Apj1 in cells lacking Ubr1 had no additional effect on the turnover of aggregated NLS-CG∗ (Figure S5B). However, in a triple deletion of Ubr1 with San1 and Doa10, a lack of Apj1 still resulted in a strong stabilization of the insoluble pellet fraction of NLS-CG∗ (Figure 5A). Thus, none of the tested ligases individually or in combination are required for Apj1-dependent turnover of NLS-CG∗. We did not detect an alteration in NLS-CG∗ ubiquitylation in cells lacking Apj1 (Figure S5C). As expected from their known function in degrading misfolded soluble proteins, the soluble fraction of NLS-CG∗ was largely stable when Ubr1, San1, and Doa10 were deleted together (Figure 5A, soluble). In contrast, insoluble NLS-CG∗ was still turned over under this condition (Figure 5A, pellet). As observed before, the turnover of aggregated NLS-CG∗ was dependent on Apj1 and Hsp104, with complete stabilization only observed in the double deletion cells (Figure 5A, pellet; compare Figure 4A). Strikingly, deletion of Apj1 in Δubr1 Δsan1 Δdoa10 cells also resulted in an increase of soluble NLS-CG∗ during CHX shut-off, which was dependent on Hsp104 (Figure 5A, soluble). This further supports our model of two independent pathways competing in disaggregation, with one involving Apj1, which leads to direct degradation, and one dependent on Hsp104, not directly linked to proteolysis and, therefore, resulting in accumulation of soluble substrate in the absence of Apj1 (Figure 5B). We evaluated the physiological consequences of having only Hsp104-dependent disaggregation, a scenario resulting in soluble NLS-CG∗ accumulation in Δubr1 Δsan1 Δdoa10 cells. Deletion of Apj1 in a background lacking the three ubiquitin ligases increased the toxicity of NLS-CG∗ (Figure 5C). Strikingly, this additional toxicity was completely reverted by additional deletion of Hsp104 (Figure 5C), indicating that the observed toxicity is caused by a soluble substrate generated by Hsp104. These data are consistent with a protective effect of Apj1 caused by tightly coupling disaggregation with turnover because complete inhibition of disaggregation reverts the detrimental effect of Apj1 deletion (Figure 5C).Figure 5Apj1 Coordinates Disaggregation with Turnover(A) Role of nuclear protein quality control ligases in turnover of soluble and insoluble NLS-CG∗. Degradation of NLS-CG∗ in the indicated strains was analyzed as in Figure 2B. Averages ± SD from five independent experiments are shown.(B) Model for different outcomes of disaggregation reactions. Apj1-dependent disaggregation results in efficient turnover, whereas Hsp104-dependent disaggregation produces soluble protein. In absence of Apj1, Apj1 substrates are disaggregated by Hsp104 producing soluble protein.(C) Effect of Apj1 on NLS-CG∗ toxicity. Serial dilutions of the indicated strains were spotted on synthetic media containing galactose or glucose for the induction or repression of NLS-CG∗ expression, respectively. Cells were grown for 3 days (glucose) or 5 days (galactose) at 37°C.(D) Apj1 deletion is detrimental in cells with compromised proteasome activity. Serial dilutions of the indicated yeast strains were spotted on YP media containing glucose and grown for 2 days at the indicated temperatures.

Apj1 Coordinates Disaggregation with Turnover

(A) Role of nuclear protein quality control ligases in turnover of soluble and insoluble NLS-CG∗. Degradation of NLS-CG∗ in the indicated strains was analyzed as in Figure 2B. Averages ± SD from five independent experiments are shown.

(B) Model for different outcomes of disaggregation reactions. Apj1-dependent disaggregation results in efficient turnover, whereas Hsp104-dependent disaggregation produces soluble protein. In absence of Apj1, Apj1 substrates are disaggregated by Hsp104 producing soluble protein.

(C) Effect of Apj1 on NLS-CG∗ toxicity. Serial dilutions of the indicated strains were spotted on synthetic media containing galactose or glucose for the induction or repression of NLS-CG∗ expression, respectively. Cells were grown for 3 days (glucose) or 5 days (galactose) at 37°C.

(D) Apj1 deletion is detrimental in cells with compromised proteasome activity. Serial dilutions of the indicated yeast strains were spotted on YP media containing glucose and grown for 2 days at the indicated temperatures.

Our data suggest that Apj1 efficiently targets substrates for degradation. In line with this, we observe a negative synthetic growth defect when deleting Apj1 in a strain carrying the hypomorphic pre1-1 mutation of the 20S proteasomal core, which results in reduced proteasome activity and a growth phenotype at an elevated temperature (Figure 5D). This is consistent with a role for Apj1 in increasing the efficiency of substrate transfer to the proteasome, when proteasomal degradation is inefficient. Again, this effect was specific for Apj1 but was not observed with a deletion of Hsp104 (Figure 5D).

In summary, these data suggest that during disaggregation, Apj1 promotes the turnover of resolubilized substrate, whereas Hsp104 fosters substrate refolding. In the absence of Apj1, Hsp104 generates soluble misfolded proteins, which are toxic if not readily degraded by the proteasome.

Apj1 Acts on Nuclear Inclusions Containing a Broad Range of Cellular SubstratesTo identify endogenous substrates of Apj1, we analyzed the composition of Apj1-containing nuclear aggregates formed upon stress. We used the Hsp70-binding deficient Apj1 mutant Apj1AAA (Figures 3A and 3B) under conditions of heat stress or proteasomal inhibition, in which formation of intra-nuclear Apj1 foci are observed (Figure 1A). As for GFP-Apj1, the nuclear localization of GFP-Apj1AAA was unchanged under all conditions tested (Figure S6A). Moreover, GFP-Apj1AAA localized to nuclear foci upon stress (Figure S6A). We expressed GFP alone, GFP-Apj1, or GFP-Apj1AAA under conditions of heat stress or proteasome inhibition and performed anti-GFP immunoprecipitation, followed by mass spectrometry and label-free quantitation (Figure 6A). Putative Apj1 substrates were selected based on their significant enrichment with GFP-Apj1AAA compared to GFP alone and increased binding to GFP-Apj1AAA in comparison to GFP-Apj1 (Figure 6A). Based on these criteria, we identified 423 putative Apj1 substrates upon heat stress and 171 after proteasome inhibition, with an overlap of 22 proteins found in both conditions (Figure S6B; Table S1). The majority of putative Apj1 substrates identified upon heat stress were of nuclear origin (or have dual nuclear and cytoplasmic localization), including many nucleolar proteins (Figure 6B). In addition, numerous cytoplasmic proteins were detected (Figure 6B). Strikingly, after proteasome inhibition, more than half of the Apj1-interacting proteins were of cytoplasmic origin, including several mitochondrial proteins (Figure 6B).Figure 6Identification of Endogenous Apj1 Substrates(A) Strategy to identify putative Apj1 substrates by mass spectrometry. GFP, GFP-Apj1, and GFPAAA were purified from cells treated with heat shock or proteasome inhibition as in Figure 1A. Four technical replicates of each sample were analyzed by mass spectrometry by using label-free quantification. Proteins were classified as Apj1 substrates when significantly enriched with GFP-Apj1AAA compared with GFP alone and show increased binding to the substrate trap variant GFP-Apj1AAA compared with GFP-Apj1.(B) Annotated localization of the putative Apj1 substrates. Proteins with an ambiguous localization to both the nucleus and cytoplasm were classified as nuclear. Numbers indicate the quantity of proteins found in each group.(C) Interaction of Cse4 and Orc4 with Apj1 upon stress. C-terminally VN tagged Cse4 or Orc4, respectively, was co-expressed with untagged or VC-tagged Apj1 in cells expressing Nup49-mars and lacking endogenous Apj1 and Pdr5. Cells were analyzed untreated, upon acute heat stress or after 2h of MG132 treatment. Quantification shows averages ± SD from three independent experiments, each replicate represents at least 300 cells.(D) Interaction of Mrpl7 with Apj1 occurs inside the nucleus. Interaction of C-terminally VN-tagged Mrpl7 and Pgk1 was performed as described in (C). Quantification shows averages ± SD from three independent experiments; each replicate represents at least 300 cells.(E) Impact of protein synthesis and mitochondrial import on Apj1-Mrpl7 interaction. Cells expressing VN-Mrpl7 and Apj1-VC were analyzed as in (D) with the following modifications; VN-Mrpl7 synthesis prior to MG-132 addition was shut down by addition of glucose. Mitochondrial import was inhibited by adding carbonyl cyanide m-chlorophenyl hydrazine (CCCP). Quantification shows averages ± SD from three independent experiments; each replicate represents at least 300 cells.Our data suggest that Apj1 acts on a wide range of substrates upon different proteotoxic stresses and highlight the extent to which cytoplasmic proteins are imported into the nucleus under such conditions. To confirm that the observed interactions occur inside the nucleus, we used fluorescence complementation. The N-terminal half of the Venus fluorescent protein was fused to the C terminus of the putative substrate and co-expressed with Apj1 N-terminally fused to the C-terminal half of Venus (VC-Apj1). First, we tested the nuclear substrates Cse4 (Cse4-VN), a centromeric histone H3 variant, and Orc4 (Orc4-VN) of the origin recognition complex. Upon proteasome inhibition, we observed Venus fluorescent foci inside the nucleus in approximately 15% of the cells (Figure 6C), which were infrequently detected in unstressed cells. Likewise, we observed a small but reproducible increase in nuclear Venus foci upon acute heat stress (Figure 6C). Thus, Apj1 interacts with its substrates inside the nucleus under conditions that give rise to nuclear aggregate formation. To probe for nuclear localization of cytoplasmic Apj1 substrates, we used the mitochondrial ribosomal protein Mrpl7 (Mrpl7-VN). We observed Venus foci inside the nucleus in about 8% of the cells upon proteasome inhibition, demonstrating that the interaction between Mrpl7 and Apj1 occurs inside the nuclear interior (Figure 6D). This interaction was dependent on proteotoxic stress induced by proteasome inhibition but was not observed upon acute heat shock, in line with our mass spectrometry analyses (Figure 6D). In contrast, the highly soluble and stable cytoplasmic protein Pgk1 (Pgk1-VN) was barely observed to interact with Apj1 even under stress conditions (Figure 6D). Finally, we asked which pool of the mitochondrial protein Mrpl7 is targeted to nuclear foci. When inhibiting galactose-inducible synthesis of Mrpl7 by adding glucose 30 min before proteasome inhibition, Venus foci formed by VC-Apj1 and Mrpl7-VN were barely detected (Figure 6E). In contrast to this, addition of carbonyl cyanide m-chlorophenyl hydrazine (CCCP) strongly increased the number of cells with Mrpl7 containing Venus foci (Figure 6E). CCCP disrupts the mitochondrial membrane potential, thereby inhibiting ATP synthesis and mitochondrial import. The increase of nuclear Mrpl7 upon CCCP treatment might, therefore, be a consequence of failed mitochondrial import, although we cannot rule out that CCCP has additional effects on cellular physiology. The overall levels of Mrpl7-VN remained largely unchanged upon CCCP treatment (Figure S6C), suggesting that nuclear Mrpl7 represents a non-mitochondrially imported sub-fraction of this protein.In summary, we conclude that proteins of nuclear and cytoplasmic origin are sequestered into nuclear foci upon stress and that mitochondrial proteins present in these inclusions represent species, which failed to properly import into mitochondria. This highlights the general role of the nucleus in cellular proteostasis and the role of Apj1 in this pathway.

Apj1 Acts on Nuclear Inclusions Containing a Broad Range of Cellular Substrates

To identify endogenous substrates of Apj1, we analyzed the composition of Apj1-containing nuclear aggregates formed upon stress. We used the Hsp70-binding deficient Apj1 mutant Apj1AAA (Figures 3A and 3B) under conditions of heat stress or proteasomal inhibition, in which formation of intra-nuclear Apj1 foci are observed (Figure 1A). As for GFP-Apj1, the nuclear localization of GFP-Apj1AAA was unchanged under all conditions tested (Figure S6A). Moreover, GFP-Apj1AAA localized to nuclear foci upon stress (Figure S6A). We expressed GFP alone, GFP-Apj1, or GFP-Apj1AAA under conditions of heat stress or proteasome inhibition and performed anti-GFP immunoprecipitation, followed by mass spectrometry and label-free quantitation (Figure 6A). Putative Apj1 substrates were selected based on their significant enrichment with GFP-Apj1AAA compared to GFP alone and increased binding to GFP-Apj1AAA in comparison to GFP-Apj1 (Figure 6A). Based on these criteria, we identified 423 putative Apj1 substrates upon heat stress and 171 after proteasome inhibition, with an overlap of 22 proteins found in both conditions (Figure S6B; Table S1). The majority of putative Apj1 substrates identified upon heat stress were of nuclear origin (or have dual nuclear and cytoplasmic localization), including many nucleolar proteins (Figure 6B). In addition, numerous cytoplasmic proteins were detected (Figure 6B). Strikingly, after proteasome inhibition, more than half of the Apj1-interacting proteins were of cytoplasmic origin, including several mitochondrial proteins (Figure 6B).Figure 6Identification of Endogenous Apj1 Substrates(A) Strategy to identify putative Apj1 substrates by mass spectrometry. GFP, GFP-Apj1, and GFPAAA were purified from cells treated with heat shock or proteasome inhibition as in Figure 1A. Four technical replicates of each sample were analyzed by mass spectrometry by using label-free quantification. Proteins were classified as Apj1 substrates when significantly enriched with GFP-Apj1AAA compared with GFP alone and show increased binding to the substrate trap variant GFP-Apj1AAA compared with GFP-Apj1.(B) Annotated localization of the putative Apj1 substrates. Proteins with an ambiguous localization to both the nucleus and cytoplasm were classified as nuclear. Numbers indicate the quantity of proteins found in each group.(C) Interaction of Cse4 and Orc4 with Apj1 upon stress. C-terminally VN tagged Cse4 or Orc4, respectively, was co-expressed with untagged or VC-tagged Apj1 in cells expressing Nup49-mars and lacking endogenous Apj1 and Pdr5. Cells were analyzed untreated, upon acute heat stress or after 2h of MG132 treatment. Quantification shows averages ± SD from three independent experiments, each replicate represents at least 300 cells.(D) Interaction of Mrpl7 with Apj1 occurs inside the nucleus. Interaction of C-terminally VN-tagged Mrpl7 and Pgk1 was performed as described in (C). Quantification shows averages ± SD from three independent experiments; each replicate represents at least 300 cells.(E) Impact of protein synthesis and mitochondrial import on Apj1-Mrpl7 interaction. Cells expressing VN-Mrpl7 and Apj1-VC were analyzed as in (D) with the following modifications; VN-Mrpl7 synthesis prior to MG-132 addition was shut down by addition of glucose. Mitochondrial import was inhibited by adding carbonyl cyanide m-chlorophenyl hydrazine (CCCP). Quantification shows averages ± SD from three independent experiments; each replicate represents at least 300 cells.

Identification of Endogenous Apj1 Substrates

(A) Strategy to identify putative Apj1 substrates by mass spectrometry. GFP, GFP-Apj1, and GFPAAA were purified from cells treated with heat shock or proteasome inhibition as in Figure 1A. Four technical replicates of each sample were analyzed by mass spectrometry by using label-free quantification. Proteins were classified as Apj1 substrates when significantly enriched with GFP-Apj1AAA compared with GFP alone and show increased binding to the substrate trap variant GFP-Apj1AAA compared with GFP-Apj1.

(B) Annotated localization of the putative Apj1 substrates. Proteins with an ambiguous localization to both the nucleus and cytoplasm were classified as nuclear. Numbers indicate the quantity of proteins found in each group.

(C) Interaction of Cse4 and Orc4 with Apj1 upon stress. C-terminally VN tagged Cse4 or Orc4, respectively, was co-expressed with untagged or VC-tagged Apj1 in cells expressing Nup49-mars and lacking endogenous Apj1 and Pdr5. Cells were analyzed untreated, upon acute heat stress or after 2h of MG132 treatment. Quantification shows averages ± SD from three independent experiments, each replicate represents at least 300 cells.

(D) Interaction of Mrpl7 with Apj1 occurs inside the nucleus. Interaction of C-terminally VN-tagged Mrpl7 and Pgk1 was performed as described in (C). Quantification shows averages ± SD from three independent experiments; each replicate represents at least 300 cells.

(E) Impact of protein synthesis and mitochondrial import on Apj1-Mrpl7 interaction. Cells expressing VN-Mrpl7 and Apj1-VC were analyzed as in (D) with the following modifications; VN-Mrpl7 synthesis prior to MG-132 addition was shut down by addition of glucose. Mitochondrial import was inhibited by adding carbonyl cyanide m-chlorophenyl hydrazine (CCCP). Quantification shows averages ± SD from three independent experiments; each replicate represents at least 300 cells.

Our data suggest that Apj1 acts on a wide range of substrates upon different proteotoxic stresses and highlight the extent to which cytoplasmic proteins are imported into the nucleus under such conditions. To confirm that the observed interactions occur inside the nucleus, we used fluorescence complementation. The N-terminal half of the Venus fluorescent protein was fused to the C terminus of the putative substrate and co-expressed with Apj1 N-terminally fused to the C-terminal half of Venus (VC-Apj1). First, we tested the nuclear substrates Cse4 (Cse4-VN), a centromeric histone H3 variant, and Orc4 (Orc4-VN) of the origin recognition complex. Upon proteasome inhibition, we observed Venus fluorescent foci inside the nucleus in approximately 15% of the cells (Figure 6C), which were infrequently detected in unstressed cells. Likewise, we observed a small but reproducible increase in nuclear Venus foci upon acute heat stress (Figure 6C). Thus, Apj1 interacts with its substrates inside the nucleus under conditions that give rise to nuclear aggregate formation. To probe for nuclear localization of cytoplasmic Apj1 substrates, we used the mitochondrial ribosomal protein Mrpl7 (Mrpl7-VN). We observed Venus foci inside the nucleus in about 8% of the cells upon proteasome inhibition, demonstrating that the interaction between Mrpl7 and Apj1 occurs inside the nuclear interior (Figure 6D). This interaction was dependent on proteotoxic stress induced by proteasome inhibition but was not observed upon acute heat shock, in line with our mass spectrometry analyses (Figure 6D). In contrast, the highly soluble and stable cytoplasmic protein Pgk1 (Pgk1-VN) was barely observed to interact with Apj1 even under stress conditions (Figure 6D). Finally, we asked which pool of the mitochondrial protein Mrpl7 is targeted to nuclear foci. When inhibiting galactose-inducible synthesis of Mrpl7 by adding glucose 30 min before proteasome inhibition, Venus foci formed by VC-Apj1 and Mrpl7-VN were barely detected (Figure 6E). In contrast to this, addition of carbonyl cyanide m-chlorophenyl hydrazine (CCCP) strongly increased the number of cells with Mrpl7 containing Venus foci (Figure 6E). CCCP disrupts the mitochondrial membrane potential, thereby inhibiting ATP synthesis and mitochondrial import. The increase of nuclear Mrpl7 upon CCCP treatment might, therefore, be a consequence of failed mitochondrial import, although we cannot rule out that CCCP has additional effects on cellular physiology. The overall levels of Mrpl7-VN remained largely unchanged upon CCCP treatment (Figure S6C), suggesting that nuclear Mrpl7 represents a non-mitochondrially imported sub-fraction of this protein.

In summary, we conclude that proteins of nuclear and cytoplasmic origin are sequestered into nuclear foci upon stress and that mitochondrial proteins present in these inclusions represent species, which failed to properly import into mitochondria. This highlights the general role of the nucleus in cellular proteostasis and the role of Apj1 in this pathway.

DiscussionIn our current work, we describe a pathway for clearance of protein aggregates inside the nucleus. Using yeast as a model system, we identified the nuclear Hsp40 chaperone Apj1 as a factor specifically mediating proteolytic clearance of intra-nuclear protein inclusions (Figure 7). Apj1 cooperates with Hsp70 and its NEF Sse1 in disaggregation and acts independently of the disaggregase Hsp104. Our work reveals the existence of two alternative nuclear protein disaggregation pathways. Disaggregation by Sis1/Hsp70/Hsp104 predominantly targets solubilized substrates to refolding pathways, and disaggregation depending on Apj1/Hsp70 results in protein turnover. The efficient degradation following Apj1-dependent turnover minimizes the risk of spurious and harmful interactions by soluble misfolded proteins, as produced by Hsp104-dependent disaggregation when the Apj1 pathway is blocked. Moreover, our discovery that not only nuclear but also proteins from different cytoplasmic compartments are targeted by Apj1 implies a central role of this nuclear pathway in cellular proteostasis.Figure 7Nuclear Pathways for Clearing Protein AggregatesMisfolded proteins arising from proteotoxic stress are targeted for degradation by the proteasome depending on the nuclear ubiquitin ligases San1, Ubr1, and Doa10. When degradation fails, nuclear protein inclusions are formed depending on Btn2. Btn2-dependent aggregates can be resolved by the nuclear-specific Hsp40 chaperone Apj1, which uses Hsp70 and its NEF Hsp110 for disaggregation. Apj1-dependent disaggregation results in efficient substrate turnover. In contrast to Apj1, Sis1-Hsp104-dependent disaggregation primarily produces soluble proteins to support refolding.Apj1-Dependent Protein DisaggregationUsing the nuclear-targeted terminally misfolded protein NLS-CG∗, we established that the Hsp40 Apj1 functions in clearance of aggregated proteins. Apj1 specifically supports the turnover of insoluble NLS-CG∗ inside the nucleus but not of soluble or cytoplasmic CG∗ (Figure 2). Likewise, Apj1 localizes to nuclear but not cytoplasmic inclusions. Consequently, abrogating nuclear aggregate formation by deleting the nuclear sequestrase Btn2 renders NLS-CG∗ turnover independent of Apj1 (Figures 2E and 2F). Our in vivo analysis suggests that Apj1 requires Hsp70 and Hsp110 (Sse1) for its activity, whereas it appears to function independently of the disaggregase Hsp104 (Figures 3 and 4A). In principle, stabilization of insoluble NLS-CG∗ can be explained by (1) a loss of holdase activity and (2) impaired disaggregation. As shown in Figure S4B, a loss of Ydj1 causes a complete loss of soluble NLS-CG∗, indicating that Ydj1 holdase activity maintains the substrate in a soluble state. This Ydj1 function has also been described for other misfolded and degradation-prone model substrates (Park et al., 2007, Guerriero et al., 2013, Jones, 2020, McClellan et al., 2005, Prasad et al., 2010). In contrast to this, Apj1 deletion has a negligible effect on the amount of soluble NLS-CG∗, arguing against a major holdase function for Apj1 (Figure 2E; Figure S4B) and suggesting that Apj1 predominantly functions as a disaggregase. Moreover, we showed that the role of Apj1 in the turnover of NLS-CG∗ depends on Btn2-dependent protein aggregation (Figures 2E and 2F). The absence of Btn2 prevents nuclear aggregation of NLS-CG∗, and turnover of soluble NLS-CG∗ becomes independent of Apj1 activity. These findings also suggests that the role of Apj1 in NLS-CG∗ turnover relies on a disaggregation function rather than a holdase activity. Importantly, we could reconstitute an Apj1/Hsp70/Sse1 disaggregation system that functions without Hsp104 in vitro (Figure 4E). Although these data serve as proof of principle, supporting the view that Apj1 is part of an autonomous Hsp70 based disaggregation machinery, further work will be required to elucidate the mechanistic details of Apj1-dependent disaggregation. This will involve dissecting the specific sequence features enabling Apj1 to support Hsp70-dependent disaggregation and analyzing the impact of different NEFs (Sse1 and Fes1) in this process. Furthermore, nucleotide-independent chaperone (holdase) activities of Apj1 need to be analyzed and compared to the major J-domain proteins of yeast, Ydj1 and Sis1.Previously, the NEF Sse1 has been shown to mainly localize to the cytoplasm, with only a minor pool present in the nucleus raising the question whether Sse1 functions inside the nuclear compartment. (Ho et al., 2019, Kaimal et al., 2017). However, the observation that nuclear localization of Sse1 is required for Apj1 function as well as the observation that no other Hsp70 NEF is required for the turnover of aggregated NLS-CG∗ supports the model that Apj1 directly cooperates with Sse1 (Figures 3C and 3D). It should be noted that Apj1 is a very low-abundant protein, and thus, a minor nuclear fraction of Sse1 might be sufficient to support its function.Two Independent Nuclear Disaggregation Pathways Determine Substrate FateThe best described disaggregation machinery involving Hsp70-Hsp104 mainly supports refolding instead of turnover, and likewise, it has been shown that the vast majority of proteins aggregating upon heat stress are subsequently refolded (Mogk et al., 2018, Wallace et al., 2015). In line with this, the Hsp40 Sis1 in concert with Hsp70 and Hsp104 mediates the recovery of heat-aggregated luciferase inside the nucleus (Ho et al., 2019). Instead, the Apj1/Hsp70-dependent disaggregation pathway described here directly targets substrates to degradation. The existence of two independent disaggregation pathways with opposing impacts on substrate fate is supported by several findings. First, the Hsp104-dependent recovery of heat-aggregated luciferase is accelerated when Apj1 is absent, suggesting that both systems compete for substrate binding (Figures 1D and S1C). Second, only the combined deletion of Apj1 with Hsp104 or Sis1 results in complete stabilization of aggregated NLS-CG∗ (Figures 4A and 4C). Third, Apj1 and Sis1 display a strong synthetic growth defect at high temperatures (Figure 4D). The physiological relevance of having a specialized disaggregation system linked to protein turnover is documented by the finding that in the absence of Apj1, Hsp104-dependent disaggregation produces soluble aberrant proteins, which results in toxicity (Figure 5).These in vivo observations are, in part, explained by our in vitro data, where we observe differences in the state of resolubilized luciferase. Here, Sis1-Hsp104-dependent disaggregation results in efficient luciferase refolding (Figure 4E). In contrast, Apj1-dependent disaggregation produces soluble but largely inactive luciferase (Figure 4E). This might explain why in vivo Apj1-dependent disaggregation is always linked to protein turnover, and we speculate that Apj1 keeps its substrates in a conformation, which is more conducive for degradation by the proteasome. The observation of distinct disaggregation machineries with different outcomes parallels previous observations made on Hsp40 chaperones acting on soluble misfolded proteins. For instance, human DNAJB1 appears to predominantly support refolding of its subtrates (Michels et al., 1999, Minami et al., 1996). In contrast, another Hsp40, DNAJB2, has been established to rather target proteins to the UPS for turnover (Howarth et al., 2007, Westhoff et al., 2005). Here, triage decision depends on the presence of a ubiquitin-interacting motif (UIM) domain, and a DNAJB2 mutant lacking a UIM domain supports folding instead of turnover. Likewise, distinct endoplasmic reticulum (ER) luminal Hsp40s have been shown to either support folding or degradation of soluble client proteins (Behnke et al., 2016). Which specific substrate features determine triage decisions by Hsp40 chaperones remains to be determined.We infer that Apj1 plays a critical role in triage decisions during protein disaggregation. Low cellular levels of Apj1 as compared to Sis1 might be important to ensure that the majority of proteins are refolded, as opposed to degraded following heat-induced aggregation (Ho et al., 2018, Wallace et al., 2015). Notably, the degrading Apj1 disaggregation system is restricted to the yeast nucleus, whereas Sis1 localizes to nucleus and cytosol (Park et al., 2013). The major cytosolic Hsp40 Ydj1 also supports disaggregation in concert with Hsp104 (Park et al., 2007, Reidy et al., 2014, Tessarz et al., 2008). It is, therefore, conceivable that misfolded proteins that aggregate in the cytosol are not subjected to triage decision and are primarily refolded.Which substrate parameters determine the preferential recognition by the degrading Apj1 and the refolding Sis1 are currently unknown. These are expected to be critical, as they will determine the outcome of protein disaggregation. Furthermore, it remains an open question of how Apj1 couples protein disaggregation to turnover. Apj1-dependent NLS-CG∗ turnover was independent of the nuclear ubiquitin ligases required for degrading soluble NLS-CG∗ (Figure 5A). This suggests the involvement of an alternative ubiquitin ligase or a different mode of action. Indeed, Hsp70-Sse1 have been previously implicated in mediating ubiquitin-independent proteasomal turnover of misfolded proteins (Kandasamy and Andréasson, 2018).The Role of Apj1 in Cellular ProteostasisBased on observations with model substrates, several studies had previously demonstrated the nuclear degradation of cytosolic proteins (Heck et al., 2010, Park et al., 2013, Prasad et al., 2010). Our findings that Apj1 acts as a chaperone specifically interacting with nuclear protein aggregates allowed us to demonstrate that a large number of cytoplasmic proteins enter the nucleus in response to proteotoxic stress, including conditions of proteasome inhibition. Why cytoplasmic proteins enter the nucleus for degradation is not well understood, but it is in line with the observation that proteasomes are enriched inside the nucleus (Russell et al., 1999, von Mikecz, 2006). A possible explanation is that spatial separation of protein synthesis and degradation helps to prevent premature proteolysis of nascent chains and harmful interactions of misfolded proteins with newly synthesized proteins (Klaips et al., 2018). To buffer the potential negative impact of targeting misfolded proteins into the nucleus, a robust nuclear proteostasis network is required to prevent misfolded proteins from interfering with essential nuclear processes, such as DNA replication and repair, transcription, or ribosome biogenesis. To limit such negative effects of misfolded proteins, they are typically sequestered into inclusions if not readily degraded by the proteasome. It is conceivable that especially cytoplasmic proteins sequestered into nuclear inclusions should be degraded rather than refolded. The Apj1-dependent coordination of disaggregation with turnover as presented here represents a mode of action whereby protein aggregates can be removed without producing toxic soluble intermediates.

In our current work, we describe a pathway for clearance of protein aggregates inside the nucleus. Using yeast as a model system, we identified the nuclear Hsp40 chaperone Apj1 as a factor specifically mediating proteolytic clearance of intra-nuclear protein inclusions (Figure 7). Apj1 cooperates with Hsp70 and its NEF Sse1 in disaggregation and acts independently of the disaggregase Hsp104. Our work reveals the existence of two alternative nuclear protein disaggregation pathways. Disaggregation by Sis1/Hsp70/Hsp104 predominantly targets solubilized substrates to refolding pathways, and disaggregation depending on Apj1/Hsp70 results in protein turnover. The efficient degradation following Apj1-dependent turnover minimizes the risk of spurious and harmful interactions by soluble misfolded proteins, as produced by Hsp104-dependent disaggregation when the Apj1 pathway is blocked. Moreover, our discovery that not only nuclear but also proteins from different cytoplasmic compartments are targeted by Apj1 implies a central role of this nuclear pathway in cellular proteostasis.Figure 7Nuclear Pathways for Clearing Protein AggregatesMisfolded proteins arising from proteotoxic stress are targeted for degradation by the proteasome depending on the nuclear ubiquitin ligases San1, Ubr1, and Doa10. When degradation fails, nuclear protein inclusions are formed depending on Btn2. Btn2-dependent aggregates can be resolved by the nuclear-specific Hsp40 chaperone Apj1, which uses Hsp70 and its NEF Hsp110 for disaggregation. Apj1-dependent disaggregation results in efficient substrate turnover. In contrast to Apj1, Sis1-Hsp104-dependent disaggregation primarily produces soluble proteins to support refolding.

Nuclear Pathways for Clearing Protein Aggregates

Misfolded proteins arising from proteotoxic stress are targeted for degradation by the proteasome depending on the nuclear ubiquitin ligases San1, Ubr1, and Doa10. When degradation fails, nuclear protein inclusions are formed depending on Btn2. Btn2-dependent aggregates can be resolved by the nuclear-specific Hsp40 chaperone Apj1, which uses Hsp70 and its NEF Hsp110 for disaggregation. Apj1-dependent disaggregation results in efficient substrate turnover. In contrast to Apj1, Sis1-Hsp104-dependent disaggregation primarily produces soluble proteins to support refolding.

Apj1-Dependent Protein DisaggregationUsing the nuclear-targeted terminally misfolded protein NLS-CG∗, we established that the Hsp40 Apj1 functions in clearance of aggregated proteins. Apj1 specifically supports the turnover of insoluble NLS-CG∗ inside the nucleus but not of soluble or cytoplasmic CG∗ (Figure 2). Likewise, Apj1 localizes to nuclear but not cytoplasmic inclusions. Consequently, abrogating nuclear aggregate formation by deleting the nuclear sequestrase Btn2 renders NLS-CG∗ turnover independent of Apj1 (Figures 2E and 2F). Our in vivo analysis suggests that Apj1 requires Hsp70 and Hsp110 (Sse1) for its activity, whereas it appears to function independently of the disaggregase Hsp104 (Figures 3 and 4A). In principle, stabilization of insoluble NLS-CG∗ can be explained by (1) a loss of holdase activity and (2) impaired disaggregation. As shown in Figure S4B, a loss of Ydj1 causes a complete loss of soluble NLS-CG∗, indicating that Ydj1 holdase activity maintains the substrate in a soluble state. This Ydj1 function has also been described for other misfolded and degradation-prone model substrates (Park et al., 2007, Guerriero et al., 2013, Jones, 2020, McClellan et al., 2005, Prasad et al., 2010). In contrast to this, Apj1 deletion has a negligible effect on the amount of soluble NLS-CG∗, arguing against a major holdase function for Apj1 (Figure 2E; Figure S4B) and suggesting that Apj1 predominantly functions as a disaggregase. Moreover, we showed that the role of Apj1 in the turnover of NLS-CG∗ depends on Btn2-dependent protein aggregation (Figures 2E and 2F). The absence of Btn2 prevents nuclear aggregation of NLS-CG∗, and turnover of soluble NLS-CG∗ becomes independent of Apj1 activity. These findings also suggests that the role of Apj1 in NLS-CG∗ turnover relies on a disaggregation function rather than a holdase activity. Importantly, we could reconstitute an Apj1/Hsp70/Sse1 disaggregation system that functions without Hsp104 in vitro (Figure 4E). Although these data serve as proof of principle, supporting the view that Apj1 is part of an autonomous Hsp70 based disaggregation machinery, further work will be required to elucidate the mechanistic details of Apj1-dependent disaggregation. This will involve dissecting the specific sequence features enabling Apj1 to support Hsp70-dependent disaggregation and analyzing the impact of different NEFs (Sse1 and Fes1) in this process. Furthermore, nucleotide-independent chaperone (holdase) activities of Apj1 need to be analyzed and compared to the major J-domain proteins of yeast, Ydj1 and Sis1.Previously, the NEF Sse1 has been shown to mainly localize to the cytoplasm, with only a minor pool present in the nucleus raising the question whether Sse1 functions inside the nuclear compartment. (Ho et al., 2019, Kaimal et al., 2017). However, the observation that nuclear localization of Sse1 is required for Apj1 function as well as the observation that no other Hsp70 NEF is required for the turnover of aggregated NLS-CG∗ supports the model that Apj1 directly cooperates with Sse1 (Figures 3C and 3D). It should be noted that Apj1 is a very low-abundant protein, and thus, a minor nuclear fraction of Sse1 might be sufficient to support its function.

Apj1-Dependent Protein Disaggregation

Using the nuclear-targeted terminally misfolded protein NLS-CG∗, we established that the Hsp40 Apj1 functions in clearance of aggregated proteins. Apj1 specifically supports the turnover of insoluble NLS-CG∗ inside the nucleus but not of soluble or cytoplasmic CG∗ (Figure 2). Likewise, Apj1 localizes to nuclear but not cytoplasmic inclusions. Consequently, abrogating nuclear aggregate formation by deleting the nuclear sequestrase Btn2 renders NLS-CG∗ turnover independent of Apj1 (Figures 2E and 2F). Our in vivo analysis suggests that Apj1 requires Hsp70 and Hsp110 (Sse1) for its activity, whereas it appears to function independently of the disaggregase Hsp104 (Figures 3 and 4A). In principle, stabilization of insoluble NLS-CG∗ can be explained by (1) a loss of holdase activity and (2) impaired disaggregation. As shown in Figure S4B, a loss of Ydj1 causes a complete loss of soluble NLS-CG∗, indicating that Ydj1 holdase activity maintains the substrate in a soluble state. This Ydj1 function has also been described for other misfolded and degradation-prone model substrates (Park et al., 2007, Guerriero et al., 2013, Jones, 2020, McClellan et al., 2005, Prasad et al., 2010). In contrast to this, Apj1 deletion has a negligible effect on the amount of soluble NLS-CG∗, arguing against a major holdase function for Apj1 (Figure 2E; Figure S4B) and suggesting that Apj1 predominantly functions as a disaggregase. Moreover, we showed that the role of Apj1 in the turnover of NLS-CG∗ depends on Btn2-dependent protein aggregation (Figures 2E and 2F). The absence of Btn2 prevents nuclear aggregation of NLS-CG∗, and turnover of soluble NLS-CG∗ becomes independent of Apj1 activity. These findings also suggests that the role of Apj1 in NLS-CG∗ turnover relies on a disaggregation function rather than a holdase activity. Importantly, we could reconstitute an Apj1/Hsp70/Sse1 disaggregation system that functions without Hsp104 in vitro (Figure 4E). Although these data serve as proof of principle, supporting the view that Apj1 is part of an autonomous Hsp70 based disaggregation machinery, further work will be required to elucidate the mechanistic details of Apj1-dependent disaggregation. This will involve dissecting the specific sequence features enabling Apj1 to support Hsp70-dependent disaggregation and analyzing the impact of different NEFs (Sse1 and Fes1) in this process. Furthermore, nucleotide-independent chaperone (holdase) activities of Apj1 need to be analyzed and compared to the major J-domain proteins of yeast, Ydj1 and Sis1.

Previously, the NEF Sse1 has been shown to mainly localize to the cytoplasm, with only a minor pool present in the nucleus raising the question whether Sse1 functions inside the nuclear compartment. (Ho et al., 2019, Kaimal et al., 2017). However, the observation that nuclear localization of Sse1 is required for Apj1 function as well as the observation that no other Hsp70 NEF is required for the turnover of aggregated NLS-CG∗ supports the model that Apj1 directly cooperates with Sse1 (Figures 3C and 3D). It should be noted that Apj1 is a very low-abundant protein, and thus, a minor nuclear fraction of Sse1 might be sufficient to support its function.

Two Independent Nuclear Disaggregation Pathways Determine Substrate FateThe best described disaggregation machinery involving Hsp70-Hsp104 mainly supports refolding instead of turnover, and likewise, it has been shown that the vast majority of proteins aggregating upon heat stress are subsequently refolded (Mogk et al., 2018, Wallace et al., 2015). In line with this, the Hsp40 Sis1 in concert with Hsp70 and Hsp104 mediates the recovery of heat-aggregated luciferase inside the nucleus (Ho et al., 2019). Instead, the Apj1/Hsp70-dependent disaggregation pathway described here directly targets substrates to degradation. The existence of two independent disaggregation pathways with opposing impacts on substrate fate is supported by several findings. First, the Hsp104-dependent recovery of heat-aggregated luciferase is accelerated when Apj1 is absent, suggesting that both systems compete for substrate binding (Figures 1D and S1C). Second, only the combined deletion of Apj1 with Hsp104 or Sis1 results in complete stabilization of aggregated NLS-CG∗ (Figures 4A and 4C). Third, Apj1 and Sis1 display a strong synthetic growth defect at high temperatures (Figure 4D). The physiological relevance of having a specialized disaggregation system linked to protein turnover is documented by the finding that in the absence of Apj1, Hsp104-dependent disaggregation produces soluble aberrant proteins, which results in toxicity (Figure 5).These in vivo observations are, in part, explained by our in vitro data, where we observe differences in the state of resolubilized luciferase. Here, Sis1-Hsp104-dependent disaggregation results in efficient luciferase refolding (Figure 4E). In contrast, Apj1-dependent disaggregation produces soluble but largely inactive luciferase (Figure 4E). This might explain why in vivo Apj1-dependent disaggregation is always linked to protein turnover, and we speculate that Apj1 keeps its substrates in a conformation, which is more conducive for degradation by the proteasome. The observation of distinct disaggregation machineries with different outcomes parallels previous observations made on Hsp40 chaperones acting on soluble misfolded proteins. For instance, human DNAJB1 appears to predominantly support refolding of its subtrates (Michels et al., 1999, Minami et al., 1996). In contrast, another Hsp40, DNAJB2, has been established to rather target proteins to the UPS for turnover (Howarth et al., 2007, Westhoff et al., 2005). Here, triage decision depends on the presence of a ubiquitin-interacting motif (UIM) domain, and a DNAJB2 mutant lacking a UIM domain supports folding instead of turnover. Likewise, distinct endoplasmic reticulum (ER) luminal Hsp40s have been shown to either support folding or degradation of soluble client proteins (Behnke et al., 2016). Which specific substrate features determine triage decisions by Hsp40 chaperones remains to be determined.We infer that Apj1 plays a critical role in triage decisions during protein disaggregation. Low cellular levels of Apj1 as compared to Sis1 might be important to ensure that the majority of proteins are refolded, as opposed to degraded following heat-induced aggregation (Ho et al., 2018, Wallace et al., 2015). Notably, the degrading Apj1 disaggregation system is restricted to the yeast nucleus, whereas Sis1 localizes to nucleus and cytosol (Park et al., 2013). The major cytosolic Hsp40 Ydj1 also supports disaggregation in concert with Hsp104 (Park et al., 2007, Reidy et al., 2014, Tessarz et al., 2008). It is, therefore, conceivable that misfolded proteins that aggregate in the cytosol are not subjected to triage decision and are primarily refolded.Which substrate parameters determine the preferential recognition by the degrading Apj1 and the refolding Sis1 are currently unknown. These are expected to be critical, as they will determine the outcome of protein disaggregation. Furthermore, it remains an open question of how Apj1 couples protein disaggregation to turnover. Apj1-dependent NLS-CG∗ turnover was independent of the nuclear ubiquitin ligases required for degrading soluble NLS-CG∗ (Figure 5A). This suggests the involvement of an alternative ubiquitin ligase or a different mode of action. Indeed, Hsp70-Sse1 have been previously implicated in mediating ubiquitin-independent proteasomal turnover of misfolded proteins (Kandasamy and Andréasson, 2018).

Two Independent Nuclear Disaggregation Pathways Determine Substrate Fate

The best described disaggregation machinery involving Hsp70-Hsp104 mainly supports refolding instead of turnover, and likewise, it has been shown that the vast majority of proteins aggregating upon heat stress are subsequently refolded (Mogk et al., 2018, Wallace et al., 2015). In line with this, the Hsp40 Sis1 in concert with Hsp70 and Hsp104 mediates the recovery of heat-aggregated luciferase inside the nucleus (Ho et al., 2019). Instead, the Apj1/Hsp70-dependent disaggregation pathway described here directly targets substrates to degradation. The existence of two independent disaggregation pathways with opposing impacts on substrate fate is supported by several findings. First, the Hsp104-dependent recovery of heat-aggregated luciferase is accelerated when Apj1 is absent, suggesting that both systems compete for substrate binding (Figures 1D and S1C). Second, only the combined deletion of Apj1 with Hsp104 or Sis1 results in complete stabilization of aggregated NLS-CG∗ (Figures 4A and 4C). Third, Apj1 and Sis1 display a strong synthetic growth defect at high temperatures (Figure 4D). The physiological relevance of having a specialized disaggregation system linked to protein turnover is documented by the finding that in the absence of Apj1, Hsp104-dependent disaggregation produces soluble aberrant proteins, which results in toxicity (Figure 5).

These in vivo observations are, in part, explained by our in vitro data, where we observe differences in the state of resolubilized luciferase. Here, Sis1-Hsp104-dependent disaggregation results in efficient luciferase refolding (Figure 4E). In contrast, Apj1-dependent disaggregation produces soluble but largely inactive luciferase (Figure 4E). This might explain why in vivo Apj1-dependent disaggregation is always linked to protein turnover, and we speculate that Apj1 keeps its substrates in a conformation, which is more conducive for degradation by the proteasome. The observation of distinct disaggregation machineries with different outcomes parallels previous observations made on Hsp40 chaperones acting on soluble misfolded proteins. For instance, human DNAJB1 appears to predominantly support refolding of its subtrates (Michels et al., 1999, Minami et al., 1996). In contrast, another Hsp40, DNAJB2, has been established to rather target proteins to the UPS for turnover (Howarth et al., 2007, Westhoff et al., 2005). Here, triage decision depends on the presence of a ubiquitin-interacting motif (UIM) domain, and a DNAJB2 mutant lacking a UIM domain supports folding instead of turnover. Likewise, distinct endoplasmic reticulum (ER) luminal Hsp40s have been shown to either support folding or degradation of soluble client proteins (Behnke et al., 2016). Which specific substrate features determine triage decisions by Hsp40 chaperones remains to be determined.

We infer that Apj1 plays a critical role in triage decisions during protein disaggregation. Low cellular levels of Apj1 as compared to Sis1 might be important to ensure that the majority of proteins are refolded, as opposed to degraded following heat-induced aggregation (Ho et al., 2018, Wallace et al., 2015). Notably, the degrading Apj1 disaggregation system is restricted to the yeast nucleus, whereas Sis1 localizes to nucleus and cytosol (Park et al., 2013). The major cytosolic Hsp40 Ydj1 also supports disaggregation in concert with Hsp104 (Park et al., 2007, Reidy et al., 2014, Tessarz et al., 2008). It is, therefore, conceivable that misfolded proteins that aggregate in the cytosol are not subjected to triage decision and are primarily refolded.

Which substrate parameters determine the preferential recognition by the degrading Apj1 and the refolding Sis1 are currently unknown. These are expected to be critical, as they will determine the outcome of protein disaggregation. Furthermore, it remains an open question of how Apj1 couples protein disaggregation to turnover. Apj1-dependent NLS-CG∗ turnover was independent of the nuclear ubiquitin ligases required for degrading soluble NLS-CG∗ (Figure 5A). This suggests the involvement of an alternative ubiquitin ligase or a different mode of action. Indeed, Hsp70-Sse1 have been previously implicated in mediating ubiquitin-independent proteasomal turnover of misfolded proteins (Kandasamy and Andréasson, 2018).

The Role of Apj1 in Cellular ProteostasisBased on observations with model substrates, several studies had previously demonstrated the nuclear degradation of cytosolic proteins (Heck et al., 2010, Park et al., 2013, Prasad et al., 2010). Our findings that Apj1 acts as a chaperone specifically interacting with nuclear protein aggregates allowed us to demonstrate that a large number of cytoplasmic proteins enter the nucleus in response to proteotoxic stress, including conditions of proteasome inhibition. Why cytoplasmic proteins enter the nucleus for degradation is not well understood, but it is in line with the observation that proteasomes are enriched inside the nucleus (Russell et al., 1999, von Mikecz, 2006). A possible explanation is that spatial separation of protein synthesis and degradation helps to prevent premature proteolysis of nascent chains and harmful interactions of misfolded proteins with newly synthesized proteins (Klaips et al., 2018). To buffer the potential negative impact of targeting misfolded proteins into the nucleus, a robust nuclear proteostasis network is required to prevent misfolded proteins from interfering with essential nuclear processes, such as DNA replication and repair, transcription, or ribosome biogenesis. To limit such negative effects of misfolded proteins, they are typically sequestered into inclusions if not readily degraded by the proteasome. It is conceivable that especially cytoplasmic proteins sequestered into nuclear inclusions should be degraded rather than refolded. The Apj1-dependent coordination of disaggregation with turnover as presented here represents a mode of action whereby protein aggregates can be removed without producing toxic soluble intermediates.

The Role of Apj1 in Cellular Proteostasis

Based on observations with model substrates, several studies had previously demonstrated the nuclear degradation of cytosolic proteins (Heck et al., 2010, Park et al., 2013, Prasad et al., 2010). Our findings that Apj1 acts as a chaperone specifically interacting with nuclear protein aggregates allowed us to demonstrate that a large number of cytoplasmic proteins enter the nucleus in response to proteotoxic stress, including conditions of proteasome inhibition. Why cytoplasmic proteins enter the nucleus for degradation is not well understood, but it is in line with the observation that proteasomes are enriched inside the nucleus (Russell et al., 1999, von Mikecz, 2006). A possible explanation is that spatial separation of protein synthesis and degradation helps to prevent premature proteolysis of nascent chains and harmful interactions of misfolded proteins with newly synthesized proteins (Klaips et al., 2018). To buffer the potential negative impact of targeting misfolded proteins into the nucleus, a robust nuclear proteostasis network is required to prevent misfolded proteins from interfering with essential nuclear processes, such as DNA replication and repair, transcription, or ribosome biogenesis. To limit such negative effects of misfolded proteins, they are typically sequestered into inclusions if not readily degraded by the proteasome. It is conceivable that especially cytoplasmic proteins sequestered into nuclear inclusions should be degraded rather than refolded. The Apj1-dependent coordination of disaggregation with turnover as presented here represents a mode of action whereby protein aggregates can be removed without producing toxic soluble intermediates.

STAR★MethodsKey Resources TableREAGENT or RESOURCESOURCEIDENTIFIERAntibodiesmouse monoclonal anti-GFPSanta CruzCat. #: sc-9996; RRID: AB_627695mouse monoclonal anti-GFPBioLegendCat. #: 902605; RRID: AB_2734671mouse monoclonal anti-Pgk1InvitrogenCat. #: 459250; RRID: AB_2532235rabbit polyclonal anti-Hsp104Enzo Life ScienceCat. #: ADI-SPA-1040-D; RRID: AB_2039208rabbit polyclonal anti-Apj1this paperN/Amouse-monoclonal anti-Ydj1Sigma AldrichCat. #: SAB5200011rabbit polyclonal anti-Sis1Cosmo BioCat. #: cop-080051; RRID: AB_10709957rabbit polyclonal anti-Atp4M. Escobar-HenriquesN/Amouse monoclonal anti-UbiquitinSanta CruzCat. #: sc-8017; RRID: AB_628423goat anti-mouse IgG, IRDye 800CWLi-CorCat. #: 926-32210; RRID: AB_621842goat anti-rabbit IgG, IRDye 800CWLi-CorCat. #: 926-32211; RRID: AB_621843goat anti-mouse IgG, IRDye 680RDLi-CorCat. #: 926-68070; RRID: AB_10956588goat anti-rabbit IgG, IRDye 680RDLi-CorCat. #: 926-68071; RRID: AB_10956166Bacterial and Virus StrainsBL21 Competent E. coliNEBCat. #: C2530HStellar Competent Cells (E. coli)TakaraCat. #: 636763XL1 blueLab stockN/AChemicals, Peptides, and Recombinant ProteinsGFP-Trap® Magnetic AgaroseChromoTekCat. #: gtma-10NiNTA-Agarose-beadsQIAGENCat. #: 1018236)Experimental Models: Organisms/StrainspSIS1::kanR-tet07-TATA, URA3::CMV-tTA, ∆apj1::his3MX6this studyCRY055his3Δ1, leu2Δ0, lys2Δ0, ura3Δ0, S288chttp://www.euroscarf.deindex.php?name=NewsyFA1791Δapj1::NatNT2, Δpdr5::hphNT2, S288cthis studyyFA3566Nup49-mars::KanMX6, S288cthis studyyFA2251Δapj1::NatNT2, S288cthis studyyFA1837Δapj1::NatNT2, Nup49-mars::KanMX6, S288cthis studyyFA2249Δbtn2::NatNT2, S288cthis studyyFA1913Δapj1::hphNT2, Δbtn2::NatNT2, S288cthis studyyFA3222Δhsp42::KanMX, Δbtn2::NatNt2, S288cthis studyyFA2082Δapj1::hphNT2, Δbtn2::NatNt2, Δhsp42::KanMX, S288cthis studyyFA3207Δhsp104::KanMX6, S288cthis studyyFA3258Δapj1::NatNT2, Δhsp104::KanMX6, S288cthis studyyFA3325Hsp104-mars::KanMX6, S288cthis studyyFA3887Δapj1::NatNT2, Hsp104-mars::KanMX6, S288cthis studyyFA3889Δsse1::hphNT2, S288cthis studyyFA1890Δsse2::kanMX, S288cthis studyyFA2161Δfes1::kanMX, S288cthis studyyFA2165Δsnl1::kanMX, S288cthis studyyFA2169Δsse1::hphNT2, Δapj1::NatNT2, S288cthis studyyFA1895Δpdr5::NatNT2, S288cthis studyyFA3562pre1-1::KanMX6, S288cLi et al., 2011yFA0371pre1-1::KanMX6, Δapj1::NatNT2, S288cthis studyyFA2272pre1-1::KanMX6, Δhsp104::KanMX6, S288cthis studyyFA4616pre1-1::KanMX6, Δhsp104::KanMX6, Δapj1::NatNT2, S288cthis studyyFA4617Δsan1::NatNT2, S288cthis studyyFA2476Δapj1::hphNT2, Δsan1::NatNT2, S288cthis studyyFA2589Δubr1::KanMX6, S288cthis studyyFA3613Δapj1::hphNT2, Δubr1::NatNT2, S288cthis studyyFA2585Δdoa10::LEU2MX6, S288cthis studyyFA4701Δapj1::NatNT2, Δdoa10::KanMX6, S288cthis studyyFA4702Δapj1, Δpdr5, Nup49-mars, S288cthis studyyFA4618URA3::CMV-tTA, pSIS11::kanR-tet07-TATA, S288cOpen BiosystemsyFA3520URA3::CMV-tTA, Δapj1::NatNt2, pSIS11::kanR-tet07-TATA, S288cthis studyyFA3524Δxdj1::His3MX6, S288cthis studyyFA4061Δcaj1::hphNT2, S288cthis studyyFA1518Δdjp1::KanMX, S288cthis studyyFA4136ydj1-2::His3, Leu2::ydj1-151, prc1-1, W303Park et al., 2007yFA3519ydj1-2::His3, Leu2::ydj1-151, prc1-1, Δapj1::NatNt2, W303this studyyFA3554Δsan1::NatNT2, Δubr1::KanMX6, Δdoa10::His3, S288cthis studyyFA4693Δsan1::NatNT2, Δubr1::KanMX6, Δdoa10::His3, Δapj1::Leu2MX6, S288cthis studyyFA4694Δhsp104::hphNT2, Δubr1::KanmX6, Δsan1::NatNT2, Δdoa10::His3, S288cthis studyyFA4699Δhsp104::hphNT2, Δubr1::KanmX6, Δsan1::NatNT2, Δapj1::His3, Δdoa10::LEU2MX6, S288cthis studyyFA4700Recombinant DNAprs315Sikorski and Hieter, 1989pFA0026prs315 pApj1 GFP-Apj1this studypFA0825prs315 pApj1 GFP-Apj1AAAthis studypFA0876pcu426 LuciDM-NLS-GFPthis studypFA0379pcu426 pGAL1 NLS-CG∗this studypFA0762p413 pGAL1 CG∗Park et al., 2013pFA0765p413 pGAL1 NLS-CG∗Park et al., 2013pFA0766p413 pGAL1 NES-CG∗Park et al., 2013pFA0872prs315 pApj1 Apj1this studypFA0824prs315 pApj1 Apj1AAAthis studypFA0809prs315 pApj1 mars-Apj1AAAthis studypFA0875prs315 pAPJ1 VC-Apj1this studypFA1024p413 pGAL1 Cse4-VNthis studypFA1025p413 pGAL1 Mrpl7-VNthis studypFA1026p413 pGAL1 Pgk1-VNthis studypFA1033p413 pGAL1 Orc4-VNthis studypFA1034p415Mumberg et al., 1995pFA0173p415 pSSE1 mars-Sse1this studypFA1027p415 pSSE1 NLS-mars-Sse1this studypFA1029p415 pSSE1 NES-mars-Sse1this studypFA1031pADH 8His-UbiquitinM. GlickmanpFA1040Software and AlgorithmsPrismGraphPadhttps://www.graphpad.com/scientific-software/FijiImageJhttps://imagej.net/Fiji/DownloadsPerseusMaxQuanthttps://maxquant.net/perseus/Resource AvailabilityLead ContactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Fabian den Brave (denbrave@uni-bonn.de).Materials AvailabilityAll unique reagents generated in this study are available from the Lead Contact without restriction.Data and Code AvailabilityThe published article includes all datasets generated or analyzed during this study.Experimental Model and Subject DetailsStandard protocols were used for yeast manipulations. Yeast cultures were inoculated from overnight cultures, grown using standard growth conditions and media. If not otherwise indicated, cells were cultured at 30°C in YPD-media or synthetic medium with auxotrophic supplements containing 2% glucose, raffinose or galactose as carbon source, as indicated. For proteasomal inhibition cells were treated with 100 μM MG132 in DMSO. For Sis1 depletion cells were grown in presence of 10 μg/ml Doxycycline. Mitochondrial protein import was inhibited by adding 25 μM CCCP. Standard cloning and site-directed mutagenesis techniques were used. All plasmids used are listed in Table S2. Chromosomally tagged strains and knockout strains were constructed by a PCR-based strategy. All strains used are listed in Table S3.Method DetailsLive cell imagingYeast cells were grown overnight in synthetic media to the exponential phase and analyzed by epifluorescence microscopy (Axioplan 2; Carl Zeiss MicroImaging, Inc., Germany) using a 63x oil-immersion objective. Images were acquired with a camera (AxioCam MRm, Carl Zeiss MicroImaging, Inc.) and processed with Axiovision 4.7 (Carl Zeiss MicroImaging, Inc.) and ImageJ (V2.0.0). For statistical analyses and representative images, images were aquired with same exposure times and processed using the same parameters for comparison.In vivo Luciferase disaggregationExponentially growing yeast cells expressing LuciferaseDM-NLS-GFP were treated with 200 μg/ml cycloheximide and subjected to 42°C for 20 min to induce Luciferase aggregation. Cells were then shifted to 30°C and Luciferase activity was measured using a Teacan plate reader.Expression shut-off assayYeast cells expressing indicated proteins were inoculated from overnight cultures using synthetic medium with auxotrophic supplements and 2% galactose as the carbon source. Cells were grown at 30°C to log phase in the same medium. Expression was stopped by addition of 2% glucose and 200μg/ml cycloheximide. Samples were taken at the indicated time points and total cell extracts or fractionations were analyzed by immunoblotting with the indicated antibodies.Fractionation of soluble and insoluble proteinFor cell fractionation assays, total yeast cell extracts were prepared by cell disruption using bead-beating in fractionation buffer (100 mM HEPES, 1% Triton X-100, 300 mM NaCl, protease inhibitors) with zirconia/silica beads, pre-cleared for 5 min at 100 g and fractionated at 16,000g for 10 min to separate proteins into soluble (S) and insoluble pellet (P) fractions. Equal amounts of the total cell lysate soluble fraction (S) and insoluble pellet fraction (P) were loaded onto gels and analyzed by immunoblotting.Co-immunoprecipitationNative yeast extracts were prepared by cell disruption on a multitube bead-beater (MM301 from Retsch GmbH) in lysis buffer (25 mM Tris pH 7.5, 50 mM KCl, 10 mM MgCl2, 1 mM EDTA, 5% glycerol, 0.5% Triton X-100, protease inhibitors) with zirconia/silica beads. For immunoprecipitation GFP-Trap_A matrix (ChromoTek GmbH) was used. Binding was performed for 2 h with rotation at 4°C and followed by stringent washing steps to remove nonspecific background binding. The binding proteins were then eluted by adding HU loading buffer and incubated at 65°C for 10 min. Samples were then analyzed by immunoblotting with the indicated antibodies.In vitro Luciferase disaggregationHsp104, Ssa1, Sis1, Sse1 and Luciferase were purified from E. coli BL21 (NEB) or XL1 blue (lab stock) cells (Ho et al., 2019, Rampelt et al., 2012). Apj1 was purified as 6His-SUMO fusion from BL21 cells grown at 24°C. Apj1 was purified in Lysis buffer (40 mM Tris-HCl pH 7.5, 500 mM NaCl, 5mM MgCl2, 20 mM Imidazole, 5mM DTT) using NiNTA-Agarose-beads (QIAGEN #1018236). Protein was eluted using elution buffer (40 mM Tris-HCl pH 7.5, 500 mM NaCl, 5mM MgCl2, 500 mM Imidazole) and dialysed into storage buffer (40 mM Tris-HCl pH 7.5, 500 mM NaCl, 5mM MgCl2, 10% glycerol, 5mM DTT). The SUMO tag was cleaved by addition of 6His-Ulp1 and 6His-SUMO and 6His-Ulp1 were removed by NiNTA-Agarose-beads.For disaggregation assays Luciferase (0.1 μM) was incubated at 42°C for 20 min in buffer A (50 mM HEPES, pH 7.5, 150 mM NaCl, 20 mM MgCl2, 2 mM DTT). Samples were shifted to 30°C and mixed with an ATP regenerating system (2 mM ATP, 3 mM phosphoenolpyruvate, 20 ng/μl pyruvate kinase) and disaggregating chaperones (1 μM Ssa1, 0.25 μM Sis1, 0.25 μM Apj1, 0.05 μM Sse1 or 0.5 μM Fes1, 0.5 μM Hsp104, 50 nM final concentration of Luciferase). When Apj1 and Sis1 were both present in the same reaction, concentration of each chaperone was reduced by 50% so that the total Hsp40 concentration stayed constant. Luciferase activities were determined using a Paradigm Plate Reader. Soluble and insoluble Luciferase were separated by centrifugation (30 min, 4°C, 16,000 g).Denaturing Ni-NTA pulldownsNLS-CG∗ was co-expressed with 6His-Ubiquitin and denaturing Ni-NTA pulldowns were performed as previously described (Psakhye and Jentsch, 2016).Mass-spectrometryNative yeast extracts were prepared by cell disruption in an ultra centrifugal mill (ZM200 from Retsch GmbH) in lysis buffer (25 mM Tris pH 7.5, 50 mM KCl, 10 mM MgCl2, 1 mM EDTA, 5% glycerol, 0.5% Triton X-100, protease inhibitors). For immunoprecipitations as described above GFP-Trap_A matrix (ChromoTek GmbH) was used. Samples were run on SDS-PAGE. Gel lanes were cut into roughly 1X1 mm size pieces and destained in 50% ethanol and 50 mM ammonium bicarbonate buffer twice. The samples were then dehydraded and re-hydrated with enzyme solution containing 12.5 ng/ul of trypsin (Promega) in 50 mM ammonium bicarbonate buffer and digested overnight. Peptides were extracted using 30% acetonitrile and 3% trifluoroacetic acid solution twice. The pH of the extracted peptides were adjuted to be above 6 and incubated with 10 mM TCEP and 40 mM chloroacetamide solution for reduction alkylation of cysteines and then purified by StageTips. Purified peptides were then subjected to LCMSMS anaylsis on a Q Exactive mass spectrometer. All raw data were processed using Maxqaunt software and peak lists were searched against yeast proteome and filtered at 1% FDR at both peptide and protein group level.AntibodiesMonoclonal antibodies against GFP (clone B-2, 1:2,000 dilution) and Ubiquitin (clone P4D1, 1:2,000 dilution) were purchased from Santa Cruz Biotechnology, monoclonal Pgk1 antibodies (clone 22C5D8, 1:5,000 dilution) were from Invitrogen and monoclonal Ydj1 antibodies (1:5,000 dilution) were from Sigma Aldrich. Monoclonal antibodies against GFP (Clone B34, 1:2,000 dilution) were used to detect the n-terminal part of split-venus (VN). Polyclonal Hsp104 antibodies (1:1,000 dilution) were purchased from Enzo Life sciences and polyclonal Sis1 antibodies (1:10,000 dilution) were from Cosmo Bio. Polyclonal Apj1 antibodies were raised in rabbit against full length Apj1. Fluorescently labeled secondary antibodies (IRDye 800CW, anti-mouse IgG (goat) and IRDye 800CW, anti-rabbit (goat), IRDye 680RD, anti-mouse IgG (goat) and IRDye 680RD, anti-rabbit (goat); each used at 1:10,000 for immunodetection using a Li-Cor system) were from Li-Cor.Quantification and Statistical AnalysisQuantification of western blots was performed using Li-Cor Image Studio software. Statistical analysis of protein levels and microscopic data was performed using GraphPad Prism software. Statistical analysis of mass-spectrometry data was performed using Perseus software. Quantifications represent averages ± SD of at least three independent experiments. Number of replicates of individual experiments is described in the figure legends. The p value for the difference in NLS-CG∗ degradation in Figure 4C was calculated using GraphPad Prism software (Student’s t test). P values ∗p < 0.05, ∗∗p < 0.01, ns > 0.05.

STAR★Methods

Key Resources TableREAGENT or RESOURCESOURCEIDENTIFIERAntibodiesmouse monoclonal anti-GFPSanta CruzCat. #: sc-9996; RRID: AB_627695mouse monoclonal anti-GFPBioLegendCat. #: 902605; RRID: AB_2734671mouse monoclonal anti-Pgk1InvitrogenCat. #: 459250; RRID: AB_2532235rabbit polyclonal anti-Hsp104Enzo Life ScienceCat. #: ADI-SPA-1040-D; RRID: AB_2039208rabbit polyclonal anti-Apj1this paperN/Amouse-monoclonal anti-Ydj1Sigma AldrichCat. #: SAB5200011rabbit polyclonal anti-Sis1Cosmo BioCat. #: cop-080051; RRID: AB_10709957rabbit polyclonal anti-Atp4M. Escobar-HenriquesN/Amouse monoclonal anti-UbiquitinSanta CruzCat. #: sc-8017; RRID: AB_628423goat anti-mouse IgG, IRDye 800CWLi-CorCat. #: 926-32210; RRID: AB_621842goat anti-rabbit IgG, IRDye 800CWLi-CorCat. #: 926-32211; RRID: AB_621843goat anti-mouse IgG, IRDye 680RDLi-CorCat. #: 926-68070; RRID: AB_10956588goat anti-rabbit IgG, IRDye 680RDLi-CorCat. #: 926-68071; RRID: AB_10956166Bacterial and Virus StrainsBL21 Competent E. coliNEBCat. #: C2530HStellar Competent Cells (E. coli)TakaraCat. #: 636763XL1 blueLab stockN/AChemicals, Peptides, and Recombinant ProteinsGFP-Trap® Magnetic AgaroseChromoTekCat. #: gtma-10NiNTA-Agarose-beadsQIAGENCat. #: 1018236)Experimental Models: Organisms/StrainspSIS1::kanR-tet07-TATA, URA3::CMV-tTA, ∆apj1::his3MX6this studyCRY055his3Δ1, leu2Δ0, lys2Δ0, ura3Δ0, S288chttp://www.euroscarf.deindex.php?name=NewsyFA1791Δapj1::NatNT2, Δpdr5::hphNT2, S288cthis studyyFA3566Nup49-mars::KanMX6, S288cthis studyyFA2251Δapj1::NatNT2, S288cthis studyyFA1837Δapj1::NatNT2, Nup49-mars::KanMX6, S288cthis studyyFA2249Δbtn2::NatNT2, S288cthis studyyFA1913Δapj1::hphNT2, Δbtn2::NatNT2, S288cthis studyyFA3222Δhsp42::KanMX, Δbtn2::NatNt2, S288cthis studyyFA2082Δapj1::hphNT2, Δbtn2::NatNt2, Δhsp42::KanMX, S288cthis studyyFA3207Δhsp104::KanMX6, S288cthis studyyFA3258Δapj1::NatNT2, Δhsp104::KanMX6, S288cthis studyyFA3325Hsp104-mars::KanMX6, S288cthis studyyFA3887Δapj1::NatNT2, Hsp104-mars::KanMX6, S288cthis studyyFA3889Δsse1::hphNT2, S288cthis studyyFA1890Δsse2::kanMX, S288cthis studyyFA2161Δfes1::kanMX, S288cthis studyyFA2165Δsnl1::kanMX, S288cthis studyyFA2169Δsse1::hphNT2, Δapj1::NatNT2, S288cthis studyyFA1895Δpdr5::NatNT2, S288cthis studyyFA3562pre1-1::KanMX6, S288cLi et al., 2011yFA0371pre1-1::KanMX6, Δapj1::NatNT2, S288cthis studyyFA2272pre1-1::KanMX6, Δhsp104::KanMX6, S288cthis studyyFA4616pre1-1::KanMX6, Δhsp104::KanMX6, Δapj1::NatNT2, S288cthis studyyFA4617Δsan1::NatNT2, S288cthis studyyFA2476Δapj1::hphNT2, Δsan1::NatNT2, S288cthis studyyFA2589Δubr1::KanMX6, S288cthis studyyFA3613Δapj1::hphNT2, Δubr1::NatNT2, S288cthis studyyFA2585Δdoa10::LEU2MX6, S288cthis studyyFA4701Δapj1::NatNT2, Δdoa10::KanMX6, S288cthis studyyFA4702Δapj1, Δpdr5, Nup49-mars, S288cthis studyyFA4618URA3::CMV-tTA, pSIS11::kanR-tet07-TATA, S288cOpen BiosystemsyFA3520URA3::CMV-tTA, Δapj1::NatNt2, pSIS11::kanR-tet07-TATA, S288cthis studyyFA3524Δxdj1::His3MX6, S288cthis studyyFA4061Δcaj1::hphNT2, S288cthis studyyFA1518Δdjp1::KanMX, S288cthis studyyFA4136ydj1-2::His3, Leu2::ydj1-151, prc1-1, W303Park et al., 2007yFA3519ydj1-2::His3, Leu2::ydj1-151, prc1-1, Δapj1::NatNt2, W303this studyyFA3554Δsan1::NatNT2, Δubr1::KanMX6, Δdoa10::His3, S288cthis studyyFA4693Δsan1::NatNT2, Δubr1::KanMX6, Δdoa10::His3, Δapj1::Leu2MX6, S288cthis studyyFA4694Δhsp104::hphNT2, Δubr1::KanmX6, Δsan1::NatNT2, Δdoa10::His3, S288cthis studyyFA4699Δhsp104::hphNT2, Δubr1::KanmX6, Δsan1::NatNT2, Δapj1::His3, Δdoa10::LEU2MX6, S288cthis studyyFA4700Recombinant DNAprs315Sikorski and Hieter, 1989pFA0026prs315 pApj1 GFP-Apj1this studypFA0825prs315 pApj1 GFP-Apj1AAAthis studypFA0876pcu426 LuciDM-NLS-GFPthis studypFA0379pcu426 pGAL1 NLS-CG∗this studypFA0762p413 pGAL1 CG∗Park et al., 2013pFA0765p413 pGAL1 NLS-CG∗Park et al., 2013pFA0766p413 pGAL1 NES-CG∗Park et al., 2013pFA0872prs315 pApj1 Apj1this studypFA0824prs315 pApj1 Apj1AAAthis studypFA0809prs315 pApj1 mars-Apj1AAAthis studypFA0875prs315 pAPJ1 VC-Apj1this studypFA1024p413 pGAL1 Cse4-VNthis studypFA1025p413 pGAL1 Mrpl7-VNthis studypFA1026p413 pGAL1 Pgk1-VNthis studypFA1033p413 pGAL1 Orc4-VNthis studypFA1034p415Mumberg et al., 1995pFA0173p415 pSSE1 mars-Sse1this studypFA1027p415 pSSE1 NLS-mars-Sse1this studypFA1029p415 pSSE1 NES-mars-Sse1this studypFA1031pADH 8His-UbiquitinM. GlickmanpFA1040Software and AlgorithmsPrismGraphPadhttps://www.graphpad.com/scientific-software/FijiImageJhttps://imagej.net/Fiji/DownloadsPerseusMaxQuanthttps://maxquant.net/perseus/

Key Resources Table

REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesmouse monoclonal anti-GFPSanta CruzCat. #: sc-9996; RRID: AB_627695mouse monoclonal anti-GFPBioLegendCat. #: 902605; RRID: AB_2734671mouse monoclonal anti-Pgk1InvitrogenCat. #: 459250; RRID: AB_2532235rabbit polyclonal anti-Hsp104Enzo Life ScienceCat. #: ADI-SPA-1040-D; RRID: AB_2039208rabbit polyclonal anti-Apj1this paperN/Amouse-monoclonal anti-Ydj1Sigma AldrichCat. #: SAB5200011rabbit polyclonal anti-Sis1Cosmo BioCat. #: cop-080051; RRID: AB_10709957rabbit polyclonal anti-Atp4M. Escobar-HenriquesN/Amouse monoclonal anti-UbiquitinSanta CruzCat. #: sc-8017; RRID: AB_628423goat anti-mouse IgG, IRDye 800CWLi-CorCat. #: 926-32210; RRID: AB_621842goat anti-rabbit IgG, IRDye 800CWLi-CorCat. #: 926-32211; RRID: AB_621843goat anti-mouse IgG, IRDye 680RDLi-CorCat. #: 926-68070; RRID: AB_10956588goat anti-rabbit IgG, IRDye 680RDLi-CorCat. #: 926-68071; RRID: AB_10956166Bacterial and Virus StrainsBL21 Competent E. coliNEBCat. #: C2530HStellar Competent Cells (E. coli)TakaraCat. #: 636763XL1 blueLab stockN/AChemicals, Peptides, and Recombinant ProteinsGFP-Trap® Magnetic AgaroseChromoTekCat. #: gtma-10NiNTA-Agarose-beadsQIAGENCat. #: 1018236)Experimental Models: Organisms/StrainspSIS1::kanR-tet07-TATA, URA3::CMV-tTA, ∆apj1::his3MX6this studyCRY055his3Δ1, leu2Δ0, lys2Δ0, ura3Δ0, S288chttp://www.euroscarf.deindex.php?name=NewsyFA1791Δapj1::NatNT2, Δpdr5::hphNT2, S288cthis studyyFA3566Nup49-mars::KanMX6, S288cthis studyyFA2251Δapj1::NatNT2, S288cthis studyyFA1837Δapj1::NatNT2, Nup49-mars::KanMX6, S288cthis studyyFA2249Δbtn2::NatNT2, S288cthis studyyFA1913Δapj1::hphNT2, Δbtn2::NatNT2, S288cthis studyyFA3222Δhsp42::KanMX, Δbtn2::NatNt2, S288cthis studyyFA2082Δapj1::hphNT2, Δbtn2::NatNt2, Δhsp42::KanMX, S288cthis studyyFA3207Δhsp104::KanMX6, S288cthis studyyFA3258Δapj1::NatNT2, Δhsp104::KanMX6, S288cthis studyyFA3325Hsp104-mars::KanMX6, S288cthis studyyFA3887Δapj1::NatNT2, Hsp104-mars::KanMX6, S288cthis studyyFA3889Δsse1::hphNT2, S288cthis studyyFA1890Δsse2::kanMX, S288cthis studyyFA2161Δfes1::kanMX, S288cthis studyyFA2165Δsnl1::kanMX, S288cthis studyyFA2169Δsse1::hphNT2, Δapj1::NatNT2, S288cthis studyyFA1895Δpdr5::NatNT2, S288cthis studyyFA3562pre1-1::KanMX6, S288cLi et al., 2011yFA0371pre1-1::KanMX6, Δapj1::NatNT2, S288cthis studyyFA2272pre1-1::KanMX6, Δhsp104::KanMX6, S288cthis studyyFA4616pre1-1::KanMX6, Δhsp104::KanMX6, Δapj1::NatNT2, S288cthis studyyFA4617Δsan1::NatNT2, S288cthis studyyFA2476Δapj1::hphNT2, Δsan1::NatNT2, S288cthis studyyFA2589Δubr1::KanMX6, S288cthis studyyFA3613Δapj1::hphNT2, Δubr1::NatNT2, S288cthis studyyFA2585Δdoa10::LEU2MX6, S288cthis studyyFA4701Δapj1::NatNT2, Δdoa10::KanMX6, S288cthis studyyFA4702Δapj1, Δpdr5, Nup49-mars, S288cthis studyyFA4618URA3::CMV-tTA, pSIS11::kanR-tet07-TATA, S288cOpen BiosystemsyFA3520URA3::CMV-tTA, Δapj1::NatNt2, pSIS11::kanR-tet07-TATA, S288cthis studyyFA3524Δxdj1::His3MX6, S288cthis studyyFA4061Δcaj1::hphNT2, S288cthis studyyFA1518Δdjp1::KanMX, S288cthis studyyFA4136ydj1-2::His3, Leu2::ydj1-151, prc1-1, W303Park et al., 2007yFA3519ydj1-2::His3, Leu2::ydj1-151, prc1-1, Δapj1::NatNt2, W303this studyyFA3554Δsan1::NatNT2, Δubr1::KanMX6, Δdoa10::His3, S288cthis studyyFA4693Δsan1::NatNT2, Δubr1::KanMX6, Δdoa10::His3, Δapj1::Leu2MX6, S288cthis studyyFA4694Δhsp104::hphNT2, Δubr1::KanmX6, Δsan1::NatNT2, Δdoa10::His3, S288cthis studyyFA4699Δhsp104::hphNT2, Δubr1::KanmX6, Δsan1::NatNT2, Δapj1::His3, Δdoa10::LEU2MX6, S288cthis studyyFA4700Recombinant DNAprs315Sikorski and Hieter, 1989pFA0026prs315 pApj1 GFP-Apj1this studypFA0825prs315 pApj1 GFP-Apj1AAAthis studypFA0876pcu426 LuciDM-NLS-GFPthis studypFA0379pcu426 pGAL1 NLS-CG∗this studypFA0762p413 pGAL1 CG∗Park et al., 2013pFA0765p413 pGAL1 NLS-CG∗Park et al., 2013pFA0766p413 pGAL1 NES-CG∗Park et al., 2013pFA0872prs315 pApj1 Apj1this studypFA0824prs315 pApj1 Apj1AAAthis studypFA0809prs315 pApj1 mars-Apj1AAAthis studypFA0875prs315 pAPJ1 VC-Apj1this studypFA1024p413 pGAL1 Cse4-VNthis studypFA1025p413 pGAL1 Mrpl7-VNthis studypFA1026p413 pGAL1 Pgk1-VNthis studypFA1033p413 pGAL1 Orc4-VNthis studypFA1034p415Mumberg et al., 1995pFA0173p415 pSSE1 mars-Sse1this studypFA1027p415 pSSE1 NLS-mars-Sse1this studypFA1029p415 pSSE1 NES-mars-Sse1this studypFA1031pADH 8His-UbiquitinM. GlickmanpFA1040Software and AlgorithmsPrismGraphPadhttps://www.graphpad.com/scientific-software/FijiImageJhttps://imagej.net/Fiji/DownloadsPerseusMaxQuanthttps://maxquant.net/perseus/

Resource AvailabilityLead ContactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Fabian den Brave (denbrave@uni-bonn.de).Materials AvailabilityAll unique reagents generated in this study are available from the Lead Contact without restriction.Data and Code AvailabilityThe published article includes all datasets generated or analyzed during this study.

Resource Availability

Lead ContactFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Fabian den Brave (denbrave@uni-bonn.de).

Lead Contact

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Fabian den Brave (denbrave@uni-bonn.de).

Materials AvailabilityAll unique reagents generated in this study are available from the Lead Contact without restriction.

Materials Availability

All unique reagents generated in this study are available from the Lead Contact without restriction.

Data and Code AvailabilityThe published article includes all datasets generated or analyzed during this study.

Data and Code Availability

The published article includes all datasets generated or analyzed during this study.

Experimental Model and Subject DetailsStandard protocols were used for yeast manipulations. Yeast cultures were inoculated from overnight cultures, grown using standard growth conditions and media. If not otherwise indicated, cells were cultured at 30°C in YPD-media or synthetic medium with auxotrophic supplements containing 2% glucose, raffinose or galactose as carbon source, as indicated. For proteasomal inhibition cells were treated with 100 μM MG132 in DMSO. For Sis1 depletion cells were grown in presence of 10 μg/ml Doxycycline. Mitochondrial protein import was inhibited by adding 25 μM CCCP. Standard cloning and site-directed mutagenesis techniques were used. All plasmids used are listed in Table S2. Chromosomally tagged strains and knockout strains were constructed by a PCR-based strategy. All strains used are listed in Table S3.

Experimental Model and Subject Details

Standard protocols were used for yeast manipulations. Yeast cultures were inoculated from overnight cultures, grown using standard growth conditions and media. If not otherwise indicated, cells were cultured at 30°C in YPD-media or synthetic medium with auxotrophic supplements containing 2% glucose, raffinose or galactose as carbon source, as indicated. For proteasomal inhibition cells were treated with 100 μM MG132 in DMSO. For Sis1 depletion cells were grown in presence of 10 μg/ml Doxycycline. Mitochondrial protein import was inhibited by adding 25 μM CCCP. Standard cloning and site-directed mutagenesis techniques were used. All plasmids used are listed in Table S2. Chromosomally tagged strains and knockout strains were constructed by a PCR-based strategy. All strains used are listed in Table S3.

Method DetailsLive cell imagingYeast cells were grown overnight in synthetic media to the exponential phase and analyzed by epifluorescence microscopy (Axioplan 2; Carl Zeiss MicroImaging, Inc., Germany) using a 63x oil-immersion objective. Images were acquired with a camera (AxioCam MRm, Carl Zeiss MicroImaging, Inc.) and processed with Axiovision 4.7 (Carl Zeiss MicroImaging, Inc.) and ImageJ (V2.0.0). For statistical analyses and representative images, images were aquired with same exposure times and processed using the same parameters for comparison.In vivo Luciferase disaggregationExponentially growing yeast cells expressing LuciferaseDM-NLS-GFP were treated with 200 μg/ml cycloheximide and subjected to 42°C for 20 min to induce Luciferase aggregation. Cells were then shifted to 30°C and Luciferase activity was measured using a Teacan plate reader.Expression shut-off assayYeast cells expressing indicated proteins were inoculated from overnight cultures using synthetic medium with auxotrophic supplements and 2% galactose as the carbon source. Cells were grown at 30°C to log phase in the same medium. Expression was stopped by addition of 2% glucose and 200μg/ml cycloheximide. Samples were taken at the indicated time points and total cell extracts or fractionations were analyzed by immunoblotting with the indicated antibodies.Fractionation of soluble and insoluble proteinFor cell fractionation assays, total yeast cell extracts were prepared by cell disruption using bead-beating in fractionation buffer (100 mM HEPES, 1% Triton X-100, 300 mM NaCl, protease inhibitors) with zirconia/silica beads, pre-cleared for 5 min at 100 g and fractionated at 16,000g for 10 min to separate proteins into soluble (S) and insoluble pellet (P) fractions. Equal amounts of the total cell lysate soluble fraction (S) and insoluble pellet fraction (P) were loaded onto gels and analyzed by immunoblotting.Co-immunoprecipitationNative yeast extracts were prepared by cell disruption on a multitube bead-beater (MM301 from Retsch GmbH) in lysis buffer (25 mM Tris pH 7.5, 50 mM KCl, 10 mM MgCl2, 1 mM EDTA, 5% glycerol, 0.5% Triton X-100, protease inhibitors) with zirconia/silica beads. For immunoprecipitation GFP-Trap_A matrix (ChromoTek GmbH) was used. Binding was performed for 2 h with rotation at 4°C and followed by stringent washing steps to remove nonspecific background binding. The binding proteins were then eluted by adding HU loading buffer and incubated at 65°C for 10 min. Samples were then analyzed by immunoblotting with the indicated antibodies.In vitro Luciferase disaggregationHsp104, Ssa1, Sis1, Sse1 and Luciferase were purified from E. coli BL21 (NEB) or XL1 blue (lab stock) cells (Ho et al., 2019, Rampelt et al., 2012). Apj1 was purified as 6His-SUMO fusion from BL21 cells grown at 24°C. Apj1 was purified in Lysis buffer (40 mM Tris-HCl pH 7.5, 500 mM NaCl, 5mM MgCl2, 20 mM Imidazole, 5mM DTT) using NiNTA-Agarose-beads (QIAGEN #1018236). Protein was eluted using elution buffer (40 mM Tris-HCl pH 7.5, 500 mM NaCl, 5mM MgCl2, 500 mM Imidazole) and dialysed into storage buffer (40 mM Tris-HCl pH 7.5, 500 mM NaCl, 5mM MgCl2, 10% glycerol, 5mM DTT). The SUMO tag was cleaved by addition of 6His-Ulp1 and 6His-SUMO and 6His-Ulp1 were removed by NiNTA-Agarose-beads.For disaggregation assays Luciferase (0.1 μM) was incubated at 42°C for 20 min in buffer A (50 mM HEPES, pH 7.5, 150 mM NaCl, 20 mM MgCl2, 2 mM DTT). Samples were shifted to 30°C and mixed with an ATP regenerating system (2 mM ATP, 3 mM phosphoenolpyruvate, 20 ng/μl pyruvate kinase) and disaggregating chaperones (1 μM Ssa1, 0.25 μM Sis1, 0.25 μM Apj1, 0.05 μM Sse1 or 0.5 μM Fes1, 0.5 μM Hsp104, 50 nM final concentration of Luciferase). When Apj1 and Sis1 were both present in the same reaction, concentration of each chaperone was reduced by 50% so that the total Hsp40 concentration stayed constant. Luciferase activities were determined using a Paradigm Plate Reader. Soluble and insoluble Luciferase were separated by centrifugation (30 min, 4°C, 16,000 g).Denaturing Ni-NTA pulldownsNLS-CG∗ was co-expressed with 6His-Ubiquitin and denaturing Ni-NTA pulldowns were performed as previously described (Psakhye and Jentsch, 2016).Mass-spectrometryNative yeast extracts were prepared by cell disruption in an ultra centrifugal mill (ZM200 from Retsch GmbH) in lysis buffer (25 mM Tris pH 7.5, 50 mM KCl, 10 mM MgCl2, 1 mM EDTA, 5% glycerol, 0.5% Triton X-100, protease inhibitors). For immunoprecipitations as described above GFP-Trap_A matrix (ChromoTek GmbH) was used. Samples were run on SDS-PAGE. Gel lanes were cut into roughly 1X1 mm size pieces and destained in 50% ethanol and 50 mM ammonium bicarbonate buffer twice. The samples were then dehydraded and re-hydrated with enzyme solution containing 12.5 ng/ul of trypsin (Promega) in 50 mM ammonium bicarbonate buffer and digested overnight. Peptides were extracted using 30% acetonitrile and 3% trifluoroacetic acid solution twice. The pH of the extracted peptides were adjuted to be above 6 and incubated with 10 mM TCEP and 40 mM chloroacetamide solution for reduction alkylation of cysteines and then purified by StageTips. Purified peptides were then subjected to LCMSMS anaylsis on a Q Exactive mass spectrometer. All raw data were processed using Maxqaunt software and peak lists were searched against yeast proteome and filtered at 1% FDR at both peptide and protein group level.AntibodiesMonoclonal antibodies against GFP (clone B-2, 1:2,000 dilution) and Ubiquitin (clone P4D1, 1:2,000 dilution) were purchased from Santa Cruz Biotechnology, monoclonal Pgk1 antibodies (clone 22C5D8, 1:5,000 dilution) were from Invitrogen and monoclonal Ydj1 antibodies (1:5,000 dilution) were from Sigma Aldrich. Monoclonal antibodies against GFP (Clone B34, 1:2,000 dilution) were used to detect the n-terminal part of split-venus (VN). Polyclonal Hsp104 antibodies (1:1,000 dilution) were purchased from Enzo Life sciences and polyclonal Sis1 antibodies (1:10,000 dilution) were from Cosmo Bio. Polyclonal Apj1 antibodies were raised in rabbit against full length Apj1. Fluorescently labeled secondary antibodies (IRDye 800CW, anti-mouse IgG (goat) and IRDye 800CW, anti-rabbit (goat), IRDye 680RD, anti-mouse IgG (goat) and IRDye 680RD, anti-rabbit (goat); each used at 1:10,000 for immunodetection using a Li-Cor system) were from Li-Cor.

Method Details

Live cell imagingYeast cells were grown overnight in synthetic media to the exponential phase and analyzed by epifluorescence microscopy (Axioplan 2; Carl Zeiss MicroImaging, Inc., Germany) using a 63x oil-immersion objective. Images were acquired with a camera (AxioCam MRm, Carl Zeiss MicroImaging, Inc.) and processed with Axiovision 4.7 (Carl Zeiss MicroImaging, Inc.) and ImageJ (V2.0.0). For statistical analyses and representative images, images were aquired with same exposure times and processed using the same parameters for comparison.

Live cell imaging

Yeast cells were grown overnight in synthetic media to the exponential phase and analyzed by epifluorescence microscopy (Axioplan 2; Carl Zeiss MicroImaging, Inc., Germany) using a 63x oil-immersion objective. Images were acquired with a camera (AxioCam MRm, Carl Zeiss MicroImaging, Inc.) and processed with Axiovision 4.7 (Carl Zeiss MicroImaging, Inc.) and ImageJ (V2.0.0). For statistical analyses and representative images, images were aquired with same exposure times and processed using the same parameters for comparison.

In vivo Luciferase disaggregationExponentially growing yeast cells expressing LuciferaseDM-NLS-GFP were treated with 200 μg/ml cycloheximide and subjected to 42°C for 20 min to induce Luciferase aggregation. Cells were then shifted to 30°C and Luciferase activity was measured using a Teacan plate reader.

In vivo Luciferase disaggregation

Exponentially growing yeast cells expressing LuciferaseDM-NLS-GFP were treated with 200 μg/ml cycloheximide and subjected to 42°C for 20 min to induce Luciferase aggregation. Cells were then shifted to 30°C and Luciferase activity was measured using a Teacan plate reader.

Expression shut-off assayYeast cells expressing indicated proteins were inoculated from overnight cultures using synthetic medium with auxotrophic supplements and 2% galactose as the carbon source. Cells were grown at 30°C to log phase in the same medium. Expression was stopped by addition of 2% glucose and 200μg/ml cycloheximide. Samples were taken at the indicated time points and total cell extracts or fractionations were analyzed by immunoblotting with the indicated antibodies.

Expression shut-off assay

Yeast cells expressing indicated proteins were inoculated from overnight cultures using synthetic medium with auxotrophic supplements and 2% galactose as the carbon source. Cells were grown at 30°C to log phase in the same medium. Expression was stopped by addition of 2% glucose and 200μg/ml cycloheximide. Samples were taken at the indicated time points and total cell extracts or fractionations were analyzed by immunoblotting with the indicated antibodies.

Fractionation of soluble and insoluble proteinFor cell fractionation assays, total yeast cell extracts were prepared by cell disruption using bead-beating in fractionation buffer (100 mM HEPES, 1% Triton X-100, 300 mM NaCl, protease inhibitors) with zirconia/silica beads, pre-cleared for 5 min at 100 g and fractionated at 16,000g for 10 min to separate proteins into soluble (S) and insoluble pellet (P) fractions. Equal amounts of the total cell lysate soluble fraction (S) and insoluble pellet fraction (P) were loaded onto gels and analyzed by immunoblotting.

Fractionation of soluble and insoluble protein

For cell fractionation assays, total yeast cell extracts were prepared by cell disruption using bead-beating in fractionation buffer (100 mM HEPES, 1% Triton X-100, 300 mM NaCl, protease inhibitors) with zirconia/silica beads, pre-cleared for 5 min at 100 g and fractionated at 16,000g for 10 min to separate proteins into soluble (S) and insoluble pellet (P) fractions. Equal amounts of the total cell lysate soluble fraction (S) and insoluble pellet fraction (P) were loaded onto gels and analyzed by immunoblotting.

Co-immunoprecipitationNative yeast extracts were prepared by cell disruption on a multitube bead-beater (MM301 from Retsch GmbH) in lysis buffer (25 mM Tris pH 7.5, 50 mM KCl, 10 mM MgCl2, 1 mM EDTA, 5% glycerol, 0.5% Triton X-100, protease inhibitors) with zirconia/silica beads. For immunoprecipitation GFP-Trap_A matrix (ChromoTek GmbH) was used. Binding was performed for 2 h with rotation at 4°C and followed by stringent washing steps to remove nonspecific background binding. The binding proteins were then eluted by adding HU loading buffer and incubated at 65°C for 10 min. Samples were then analyzed by immunoblotting with the indicated antibodies.

Co-immunoprecipitation

Native yeast extracts were prepared by cell disruption on a multitube bead-beater (MM301 from Retsch GmbH) in lysis buffer (25 mM Tris pH 7.5, 50 mM KCl, 10 mM MgCl2, 1 mM EDTA, 5% glycerol, 0.5% Triton X-100, protease inhibitors) with zirconia/silica beads. For immunoprecipitation GFP-Trap_A matrix (ChromoTek GmbH) was used. Binding was performed for 2 h with rotation at 4°C and followed by stringent washing steps to remove nonspecific background binding. The binding proteins were then eluted by adding HU loading buffer and incubated at 65°C for 10 min. Samples were then analyzed by immunoblotting with the indicated antibodies.

In vitro Luciferase disaggregationHsp104, Ssa1, Sis1, Sse1 and Luciferase were purified from E. coli BL21 (NEB) or XL1 blue (lab stock) cells (Ho et al., 2019, Rampelt et al., 2012). Apj1 was purified as 6His-SUMO fusion from BL21 cells grown at 24°C. Apj1 was purified in Lysis buffer (40 mM Tris-HCl pH 7.5, 500 mM NaCl, 5mM MgCl2, 20 mM Imidazole, 5mM DTT) using NiNTA-Agarose-beads (QIAGEN #1018236). Protein was eluted using elution buffer (40 mM Tris-HCl pH 7.5, 500 mM NaCl, 5mM MgCl2, 500 mM Imidazole) and dialysed into storage buffer (40 mM Tris-HCl pH 7.5, 500 mM NaCl, 5mM MgCl2, 10% glycerol, 5mM DTT). The SUMO tag was cleaved by addition of 6His-Ulp1 and 6His-SUMO and 6His-Ulp1 were removed by NiNTA-Agarose-beads.For disaggregation assays Luciferase (0.1 μM) was incubated at 42°C for 20 min in buffer A (50 mM HEPES, pH 7.5, 150 mM NaCl, 20 mM MgCl2, 2 mM DTT). Samples were shifted to 30°C and mixed with an ATP regenerating system (2 mM ATP, 3 mM phosphoenolpyruvate, 20 ng/μl pyruvate kinase) and disaggregating chaperones (1 μM Ssa1, 0.25 μM Sis1, 0.25 μM Apj1, 0.05 μM Sse1 or 0.5 μM Fes1, 0.5 μM Hsp104, 50 nM final concentration of Luciferase). When Apj1 and Sis1 were both present in the same reaction, concentration of each chaperone was reduced by 50% so that the total Hsp40 concentration stayed constant. Luciferase activities were determined using a Paradigm Plate Reader. Soluble and insoluble Luciferase were separated by centrifugation (30 min, 4°C, 16,000 g).

In vitro Luciferase disaggregation

Hsp104, Ssa1, Sis1, Sse1 and Luciferase were purified from E. coli BL21 (NEB) or XL1 blue (lab stock) cells (Ho et al., 2019, Rampelt et al., 2012). Apj1 was purified as 6His-SUMO fusion from BL21 cells grown at 24°C. Apj1 was purified in Lysis buffer (40 mM Tris-HCl pH 7.5, 500 mM NaCl, 5mM MgCl2, 20 mM Imidazole, 5mM DTT) using NiNTA-Agarose-beads (QIAGEN #1018236). Protein was eluted using elution buffer (40 mM Tris-HCl pH 7.5, 500 mM NaCl, 5mM MgCl2, 500 mM Imidazole) and dialysed into storage buffer (40 mM Tris-HCl pH 7.5, 500 mM NaCl, 5mM MgCl2, 10% glycerol, 5mM DTT). The SUMO tag was cleaved by addition of 6His-Ulp1 and 6His-SUMO and 6His-Ulp1 were removed by NiNTA-Agarose-beads.

For disaggregation assays Luciferase (0.1 μM) was incubated at 42°C for 20 min in buffer A (50 mM HEPES, pH 7.5, 150 mM NaCl, 20 mM MgCl2, 2 mM DTT). Samples were shifted to 30°C and mixed with an ATP regenerating system (2 mM ATP, 3 mM phosphoenolpyruvate, 20 ng/μl pyruvate kinase) and disaggregating chaperones (1 μM Ssa1, 0.25 μM Sis1, 0.25 μM Apj1, 0.05 μM Sse1 or 0.5 μM Fes1, 0.5 μM Hsp104, 50 nM final concentration of Luciferase). When Apj1 and Sis1 were both present in the same reaction, concentration of each chaperone was reduced by 50% so that the total Hsp40 concentration stayed constant. Luciferase activities were determined using a Paradigm Plate Reader. Soluble and insoluble Luciferase were separated by centrifugation (30 min, 4°C, 16,000 g).

Denaturing Ni-NTA pulldownsNLS-CG∗ was co-expressed with 6His-Ubiquitin and denaturing Ni-NTA pulldowns were performed as previously described (Psakhye and Jentsch, 2016).

Denaturing Ni-NTA pulldowns

NLS-CG∗ was co-expressed with 6His-Ubiquitin and denaturing Ni-NTA pulldowns were performed as previously described (Psakhye and Jentsch, 2016).

Mass-spectrometryNative yeast extracts were prepared by cell disruption in an ultra centrifugal mill (ZM200 from Retsch GmbH) in lysis buffer (25 mM Tris pH 7.5, 50 mM KCl, 10 mM MgCl2, 1 mM EDTA, 5% glycerol, 0.5% Triton X-100, protease inhibitors). For immunoprecipitations as described above GFP-Trap_A matrix (ChromoTek GmbH) was used. Samples were run on SDS-PAGE. Gel lanes were cut into roughly 1X1 mm size pieces and destained in 50% ethanol and 50 mM ammonium bicarbonate buffer twice. The samples were then dehydraded and re-hydrated with enzyme solution containing 12.5 ng/ul of trypsin (Promega) in 50 mM ammonium bicarbonate buffer and digested overnight. Peptides were extracted using 30% acetonitrile and 3% trifluoroacetic acid solution twice. The pH of the extracted peptides were adjuted to be above 6 and incubated with 10 mM TCEP and 40 mM chloroacetamide solution for reduction alkylation of cysteines and then purified by StageTips. Purified peptides were then subjected to LCMSMS anaylsis on a Q Exactive mass spectrometer. All raw data were processed using Maxqaunt software and peak lists were searched against yeast proteome and filtered at 1% FDR at both peptide and protein group level.

Mass-spectrometry

Native yeast extracts were prepared by cell disruption in an ultra centrifugal mill (ZM200 from Retsch GmbH) in lysis buffer (25 mM Tris pH 7.5, 50 mM KCl, 10 mM MgCl2, 1 mM EDTA, 5% glycerol, 0.5% Triton X-100, protease inhibitors). For immunoprecipitations as described above GFP-Trap_A matrix (ChromoTek GmbH) was used. Samples were run on SDS-PAGE. Gel lanes were cut into roughly 1X1 mm size pieces and destained in 50% ethanol and 50 mM ammonium bicarbonate buffer twice. The samples were then dehydraded and re-hydrated with enzyme solution containing 12.5 ng/ul of trypsin (Promega) in 50 mM ammonium bicarbonate buffer and digested overnight. Peptides were extracted using 30% acetonitrile and 3% trifluoroacetic acid solution twice. The pH of the extracted peptides were adjuted to be above 6 and incubated with 10 mM TCEP and 40 mM chloroacetamide solution for reduction alkylation of cysteines and then purified by StageTips. Purified peptides were then subjected to LCMSMS anaylsis on a Q Exactive mass spectrometer. All raw data were processed using Maxqaunt software and peak lists were searched against yeast proteome and filtered at 1% FDR at both peptide and protein group level.

AntibodiesMonoclonal antibodies against GFP (clone B-2, 1:2,000 dilution) and Ubiquitin (clone P4D1, 1:2,000 dilution) were purchased from Santa Cruz Biotechnology, monoclonal Pgk1 antibodies (clone 22C5D8, 1:5,000 dilution) were from Invitrogen and monoclonal Ydj1 antibodies (1:5,000 dilution) were from Sigma Aldrich. Monoclonal antibodies against GFP (Clone B34, 1:2,000 dilution) were used to detect the n-terminal part of split-venus (VN). Polyclonal Hsp104 antibodies (1:1,000 dilution) were purchased from Enzo Life sciences and polyclonal Sis1 antibodies (1:10,000 dilution) were from Cosmo Bio. Polyclonal Apj1 antibodies were raised in rabbit against full length Apj1. Fluorescently labeled secondary antibodies (IRDye 800CW, anti-mouse IgG (goat) and IRDye 800CW, anti-rabbit (goat), IRDye 680RD, anti-mouse IgG (goat) and IRDye 680RD, anti-rabbit (goat); each used at 1:10,000 for immunodetection using a Li-Cor system) were from Li-Cor.

Monoclonal antibodies against GFP (clone B-2, 1:2,000 dilution) and Ubiquitin (clone P4D1, 1:2,000 dilution) were purchased from Santa Cruz Biotechnology, monoclonal Pgk1 antibodies (clone 22C5D8, 1:5,000 dilution) were from Invitrogen and monoclonal Ydj1 antibodies (1:5,000 dilution) were from Sigma Aldrich. Monoclonal antibodies against GFP (Clone B34, 1:2,000 dilution) were used to detect the n-terminal part of split-venus (VN). Polyclonal Hsp104 antibodies (1:1,000 dilution) were purchased from Enzo Life sciences and polyclonal Sis1 antibodies (1:10,000 dilution) were from Cosmo Bio. Polyclonal Apj1 antibodies were raised in rabbit against full length Apj1. Fluorescently labeled secondary antibodies (IRDye 800CW, anti-mouse IgG (goat) and IRDye 800CW, anti-rabbit (goat), IRDye 680RD, anti-mouse IgG (goat) and IRDye 680RD, anti-rabbit (goat); each used at 1:10,000 for immunodetection using a Li-Cor system) were from Li-Cor.

Quantification and Statistical AnalysisQuantification of western blots was performed using Li-Cor Image Studio software. Statistical analysis of protein levels and microscopic data was performed using GraphPad Prism software. Statistical analysis of mass-spectrometry data was performed using Perseus software. Quantifications represent averages ± SD of at least three independent experiments. Number of replicates of individual experiments is described in the figure legends. The p value for the difference in NLS-CG∗ degradation in Figure 4C was calculated using GraphPad Prism software (Student’s t test). P values ∗p < 0.05, ∗∗p < 0.01, ns > 0.05.

Quantification and Statistical Analysis

Quantification of western blots was performed using Li-Cor Image Studio software. Statistical analysis of protein levels and microscopic data was performed using GraphPad Prism software. Statistical analysis of mass-spectrometry data was performed using Perseus software. Quantifications represent averages ± SD of at least three independent experiments. Number of replicates of individual experiments is described in the figure legends. The p value for the difference in NLS-CG∗ degradation in Figure 4C was calculated using GraphPad Prism software (Student’s t test). P values ∗p < 0.05, ∗∗p < 0.01, ns > 0.05.
