---
pmid: '12199906'
title: Epstein-Barr virus encoded nuclear protein EBNA-3 binds a novel human uridine
  kinase/uracil phosphoribosyltransferase.
authors:
- Kashuba E
- Kashuba V
- Sandalova T
- Klein G
- Szekely L
journal: BMC Cell Biol
year: '2002'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC126255
doi: 10.1186/1471-2121-3-23
---

# Epstein-Barr virus encoded nuclear protein EBNA-3 binds a novel human uridine kinase/uracil phosphoribosyltransferase.
**Authors:** Kashuba E, Kashuba V, Sandalova T, Klein G, Szekely L
**Journal:** BMC Cell Biol (2002)
**DOI:** [10.1186/1471-2121-3-23](https://doi.org/10.1186/1471-2121-3-23)
**PMC:** [PMC126255](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC126255/)

## Abstract

1. BMC Cell Biol. 2002 Aug 29;3:23. doi: 10.1186/1471-2121-3-23. Epub 2002 Aug
29.

Epstein-Barr virus encoded nuclear protein EBNA-3 binds a novel human uridine 
kinase/uracil phosphoribosyltransferase.

Kashuba E(1), Kashuba V, Sandalova T, Klein G, Szekely L.

Author information:
(1)Microbiology and Tumor Biology Centre, Karolinska Institute, S-171 77, 
Stockholm, Sweden. elena.kashuba@mtc.ki.se

BACKGROUND: Epstein-Barr virus (EBV) infects resting B-lymphocytes and 
transforms them into immortal proliferating lymphoblastoid cell lines (LCLs) in 
vitro. The transformed immunoblasts may grow up as immunoblastic lymphomas in 
immuno-suppressed hosts.
RESULTS: In order to identify cellular protein targets that may be involved in 
Epstein-Barr virus mediated B-cell transformation, human LCL cDNA library was 
screened with one of the transformation associated nuclear antigens, EBNA-3 
(also called EBNA-3A), using the yeast two-hybrid system. A clone encoding a 
fragment of a novel human protein was isolated (clone 538). The interaction was 
confirmed using in vitro binding assays. A full-length cDNA clone (F538) was 
isolated. Sequence alignment with known proteins and 3D structure predictions 
suggest that F538 is a novel human uridine kinase/uracil 
phosphoribosyltransferase. The GFP-F538 fluorescent fusion protein showed a 
preferentially cytoplasmic distribution but translocated to the nucleus upon 
co-expression of EBNA-3. A naturally occurring splice variant of F538, that 
lacks the C-terminal uracil phosphoribosyltransferase part but maintain uridine 
kinase domain, did not translocate to the nucleus in the presence of EBNA3. 
Antibody that was raised against the bacterially produced GST-538 protein showed 
cytoplasmic staining in EBV negative Burkitt lymphomas but gave a predominantly 
nuclear staining in EBV positive LCL-s and stable transfected cells expressing 
EBNA-3.
CONCLUSION: We suggest that EBNA-3 by direct protein-protein interaction induces 
the nuclear accumulation of a novel enzyme, that is part of the ribonucleotide 
salvage pathway. Increased intranuclear levels of UK/UPRT may contribute to the 
metabolic build-up that is needed for blast transformation and rapid 
proliferation.

DOI: 10.1186/1471-2121-3-23
PMCID: PMC126255
PMID: 12199906 [Indexed for MEDLINE]

## Full Text

BackgroundEBV is a gamma-herpes virus that is present in nearly all humans in form of lifelong infection. EBV infection in adolescence causes infectious mononucleosis. The virus associates with several human malignancies, particularly, with Burkitt lymphoma (BL), nasopharyngeal carcinoma (NPC) and post transplant lymphoproliferative disease [1].In vitro EBV transforms B-cells into lymphoblastoid cell lines (LCLs) that express 9 virally encoded proteins – the nuclear proteins EBNA 1–6 and the membrane proteins LMP1, 2A and 2B. Six of them – EBNA-1, -2, -3, -5 and -6 and LMP-1 are required for immortalisation of B-cells [1]. EBNA-3 (also called EBNA-3A) is a member of the EBNA-3-protein family, designated as EBNA-3, 4 and 6, or in the alternative nomenclature, EBNA-3A, B and C. These three proteins bear a major responsibility for the induction of the immune rejection response that makes mononucleosis a self-limiting disease. All three proteins can bind to the DNA binding protein RBP-Jk [2-8].The role of the EBNA-3 family in the viral strategy is not well understood [9,10]. We have previously identified two cellular proteins that can bind to EBNA-3 in yeast two-hybrid system. The ε-subunit of the TCP1 chaperonin complex may assist the initial folding of the nascent EBNA-3 [11]. The Xap-2 protein is a minor subunit of the aryl hydrocarbon receptor complex [12]. It is also a cellular target for the Hepatitis B virus encoded X antigen. HBX is believed to be involved in HBV associated carcinogenesis [13].In this paper we have identified yet another human protein, designated F538, through its binding to EBNA-3. We have found that it is homologous to human and mouse uridine kinases, human uridine-cytidine kinase, and to uracil phosphoribosyltransferases of Toxoplasma gondii, C. elegans and Cryptosporidium parvum .

EBV is a gamma-herpes virus that is present in nearly all humans in form of lifelong infection. EBV infection in adolescence causes infectious mononucleosis. The virus associates with several human malignancies, particularly, with Burkitt lymphoma (BL), nasopharyngeal carcinoma (NPC) and post transplant lymphoproliferative disease [1].

In vitro EBV transforms B-cells into lymphoblastoid cell lines (LCLs) that express 9 virally encoded proteins – the nuclear proteins EBNA 1–6 and the membrane proteins LMP1, 2A and 2B. Six of them – EBNA-1, -2, -3, -5 and -6 and LMP-1 are required for immortalisation of B-cells [1]. EBNA-3 (also called EBNA-3A) is a member of the EBNA-3-protein family, designated as EBNA-3, 4 and 6, or in the alternative nomenclature, EBNA-3A, B and C. These three proteins bear a major responsibility for the induction of the immune rejection response that makes mononucleosis a self-limiting disease. All three proteins can bind to the DNA binding protein RBP-Jk [2-8].

The role of the EBNA-3 family in the viral strategy is not well understood [9,10]. We have previously identified two cellular proteins that can bind to EBNA-3 in yeast two-hybrid system. The ε-subunit of the TCP1 chaperonin complex may assist the initial folding of the nascent EBNA-3 [11]. The Xap-2 protein is a minor subunit of the aryl hydrocarbon receptor complex [12]. It is also a cellular target for the Hepatitis B virus encoded X antigen. HBX is believed to be involved in HBV associated carcinogenesis [13].

In this paper we have identified yet another human protein, designated F538, through its binding to EBNA-3. We have found that it is homologous to human and mouse uridine kinases, human uridine-cytidine kinase, and to uracil phosphoribosyltransferases of Toxoplasma gondii, C. elegans and Cryptosporidium parvum .

ResultsRBP-Jk is one of the known interacting partners of EBNA-3. It binds to the N-terminal part of EBNA-3. In order to find additional targets of this large viral protein we used an N-terminus truncated EBNA-3 cDNA clone (encoding amino acids 127–945) for screening of a human lymphoblast cDNA library. We identified an interactive clone (designated clone 538) that could grow on His, Leu and Trp deficient medium and expressed β-galactosidase from an interaction dependent reporter locus. Plasmid rescue from the yeast to E. coli produced a clone with a ~800 bp long insert at its XhoI site.Retransformation of clone 538 DNA together with the EBNA-3 construct into the yeast tester strain (SFY526) confirmed the interaction. Plasmids containing EBNA-1, ΔEBNA-4, EBNA-5, EBNA-6 and the empty pGBT9/BR vector were also retransformed together with the clone 538 DNA as negative controls. None of them showed any significant interaction in β-galactosidase test (Table 1). The strongly interacting p53/SV40LT protein pair was used as positive control.Table 1Specific activity in β-galactosidase units of 538 co-transformants in yeast strain SFY526Target DNA in BD vectorβGU *EBNA-10,6EBNA-36,7ΔEBNA-40,8EBNA-50,8EBNA-60,9BD0,6* – Large T-p53 activity is 72,8 (30% of standard deviation after 4 independent experiments)XhoI cleaved insert of clone 538 was used as a probe on Northern blots. The 2,4 kb transcript was detected ubiquitously in all tissues with highest level expression in skeletal muscle, heart and kidney (Figure 1). We have detected the same transcript in Burkitt lymphoma (BL) lines DG75, DG75-EBNA-3, DG75-EBNA-5 and freshly established LCL (data not shown).Figure 1Northern blot for 2.4 kb F538 mRNA (top) shows an ubiquitous expression in different human tissues. β-actin probe is used as loading control on the same filter (bottom).Sequencing of the insert showed that clone 538 was identical to a part of a putative human gene with GenBank accession number of NM_017859.1, encoding the hypothetical gene product FLJ20517 (accession number NP_060329). The gene is consisted of fifteen exons that are distributed on a 19580 nucleotide long region on chromosome 20 (1325898–1306319).To obtain the full-length cDNA (F538) we used a human heart cDNA library for PCR-amplification. The full-length cDNA was sequenced and cloned into green fluorescence protein fusion vector pEGFP-C1 (GFP) and AD vectors. The cDNA clone contained 1630 bp, encoding a 538 aa long protein. The first 10 amino acids were omitted from the cloning strategy because no selective primers could be designed for this region.We have also identified an alternatively spliced variant of this gene (F538ΔC) where exon 12 was spliced to the middle of exon 11. This splice created a frameshift that led to a stop-codon. The encoded protein product is truncated from residue 386 with 10 extra amino acids prior to the stop-codon (Figure 2). The alternatively spliced variant was also recovered as EST from GenBank as mRNA expressed in different normal tissues such as optic nerve (BM702108), retina (BM693202) and testis (AA382318). The exon structure of the gene and the location of PCR primers are shown on Figure 3.Figure 2Sequence of F538 that is identical to the hypothetical protein FLJ20517. The sequence of the cDNA fragment isolated from the yeast two hybrid screen is underlined (top) Sequence of the C terminal part of the alternatively spliced variant with the additional ten amino acids preceding the newly created stop-codon (bottom).Figure 3Exon structure and cloning strategy for the full length F538 and the C terminal truncated splice variantTo analyse the protein-protein interaction in vitro, the insert from yeast 538 clone (the coding region corresponding to amino acids 216–473) was cloned into glutathione-S-transferase bacterial expression vector (GST-2TK). Upon induction, a 56 kD fusion protein was detected on silver or Coomassie blue stained SDS acryl amide gels.This and several other control GST proteins were used to precipitate interacting proteins from lysates of CV-1 cells that were infected with recombinant vaccinia virus expressing full length EBNA-3. GST-538, but not GST or GST-EBNA-5 could precipitated EBNA-3 (Figure 4). GST-Full436 containing the Xap-2 gene was used as positive interaction control [12]. Lysates of cells, infected with recombinant vaccinia virus that expressed EBNA-2 were included as non-specific precipitation controls.Figure 4GST-538, but not different control fusion proteins precipitates EBNA-3 from CV-1 cell lysates infected with recombinant vaccinia virus as detected by Western blotting.To study the subcellular localization of the protein, GFP-F538 and GFP-F538ΔC constructs were transfected into CV1 cells. Protein expression was already detectable after 4–6 hours by direct fluorescence. At this time the protein was homogeneously distributed in the cytoplasm. 24 hours after the transfection the protein started to form granular precipitates of varying size that were restricted to the cytoplasm. After 48 hours the protein formed large cytoplasmic inclusion bodies as the result of overexpression (Figure 5).Figure 5GFP-F538 localize to the cytoplasm of transfected CV1 cells. Depending on the level of expression it shows homogeneous (5–8 hours after transfection – top row), speckled (24 hours after transfection – middle row) or massively granular distribution (48 hours after transfection – bottom row). DNA staining with Hoechst 33258 is blue.EBNA-3 is a nuclear protein. In order to test if EBNA-3 has any effect on the subcellular distribution of F538, CV-1 cells that were transfected with GFP-F538 or GFP-F538ΔC, were superinfected with recombinant vaccinia virus expressing EBNA-3 or EBNA-5. EBNA-3 but not EBNA-5 induced nuclear translocation of GFP-F538. C-terminally truncated protein remained in cytoplasm. EBNA-3 re-distributed in the nucleus and formed nuclear precipitates together with GFP-F538. These two proteins showed a high degree of co-localization (Figure 6A,6B,6C,6D,6E,6F,6G,6H,6I).Figure 6Expression of EBNA-3 (B, E, H – red) from recombinant vaccinia virus in cells that were transfected with GFP-F538 (A, D, G – green) leads to the accumulation of GFP-F538 in the nucleus in parallel with the redistribution of EBNA-3 from homogeneous nucleoplasmic pattern to well circumscribed nuclear granules. In these granules EBNA-3 (H) shows high levels of co-localization (I) with GFP-F538 (G). Expression of EBNA-5 (C – red) from recombinant vaccinia virus does not change the cytoplasmic localization of GFP-F538 (C – green). Immunofluorescence staining of endogeneous F538 protein with rabbit polyclonal antibodies shows cytoplasmic distribution in the EBV negative BL cells DG75 (J) but gives predominantly nuclear staining in EBV positive BL cells Raji (K) or EBV transformed lymphoblastoid cell line 940110 (L). DG75 cells that express EBNA-5 show the same cytoplasmic distribution as the parental cells (M) whereas EBNA-3 expressing DG75 cells demonstrate nuclear accumulation of the F538 protein (N). High magnification image of the nucleus show that F538 preferentially accumulates in low DNA density areas that corresponds to the euchromatin (O) DNA staining with Hoechst 33258 is blue.We raised rabbit polyclonal antibodies against the bacterially produced GST-538 protein. Immunofluorescence staining detected an almost exclusively cytoplasmic distribution of F538 in the EBV negative BL cells DG75 (Figure 6J) and BL21 but gave a predominantly nuclear staining in the EBV positive BL Raji (Figure 6K) or EBV transformed lymphoblastoid cell lines Nadja, IARC171 and 940110 (Figure 6L).Importantly the stable transfected clone of DG75 that constitutively expressed EBNA-3 in more than 95% of the cells showed nuclear accumulation of the F538 protein (Figure 6N) whereas F538 remained cytoplasmic in DG75 cells stably transfected with EBNA-5 (Figure 6M). Optical sections taken at high magnification using fluorescence 3D microscope showed that F538 preferentially accumulated in the low DNA density areas of the nucleus of the EBNA-3 expressing cells (Fig 6O).We have built a 3D-structural model for the C-terminal part of F538, starting from residue 309, using the SWISSMODEL program [14-16]. With very high probability E-value (1e-51) it corresponded to the crystal structure of Toxoplasma gondii uracil phosphoribosyltransferase (UPRT) [17] (Figure 7). UPRT (T. gondii) belongs to PRTase-like superfamily, phosphoribosyltransferases (PRTases) family with PRTase-like fold (SCOP classification). This second half of F538 (309–548 aa) shows high sequence similarity of three of the four conserved regions (I, II and IV) where the binding sites for ribose (R) and uracil (U) are located (Figure 8). Furthermore the predicted 3D structure shows an almost identical folding even for the region III despite the sequence difference. Structure and composition of the active centre in UPRT (T. gondii) and in F538 shows also very high similarity (Figure 9). The flexible loop that is involved in the binding to another monomeric unit is present as well. The 3D model of F538 suggests that the flexible loop is folded in a similar fashion as in UPRT (T. gondii) where it is involved in the formation of homo-dimer as well as homo-tetramer, that is the active form of the enzyme (Figure 10).Figure 7Predicted 3D structure of F538 (aa 309–542) as space fill model in comparison to the crystal structure of T. gondii UPRT (2.5 Å resolution – Schumacher et al, 1998). The bound UMP is red.Figure 8Protein sequence alignment of F538 and UPRT of Toxoplasma gondii. The four conserved regions are underlined. R – ribose binding residue, U – uracil binding residues.Figure 9The structure of T. gondii UPRT active center with coordinated UMP (white balls and sticks) as compared to the predicted SWISSModel structure of F538. The UPRT conserved region III that shows the highest difference between the two proteins still exhibits the same folding pattern (black line on the backbone)Figure 10The structure of the predicted dimer derived from SWISSModel for F538. Reaction product – UMP is shown in red. N – and C-termini are indicated.The N-terminal part contains an ATP/GTP-binding site (P-loop) suggesting that F538 also has a uridine kinase activity. High (37–42) aligning scores to the known uridine kinases support this proposition (Figure 11). F538 most probably consists of two domains with separate enzymatic activity in a similar fashion as the C. elegans UK/UPRT enzyme (Figure 12).Figure 11Multiple alignments of the N terminus of F538, mouse uridine kinase (P52263), UK/UPRT of Cryptosporidium parvum (AAG53652) and UK/UPRT of C. elegans (CAA93459). P-loop (ATP/GTP-binding site) is underlined.Figure 12Multiple alignments of the C terminus of F538, T. gondii phosphoribosyltransferase (Q26998), UK/UPRT of Cryptosporidium parvum (AAG53652) and UK/UPRT of C. elegans (CAA93459).

RBP-Jk is one of the known interacting partners of EBNA-3. It binds to the N-terminal part of EBNA-3. In order to find additional targets of this large viral protein we used an N-terminus truncated EBNA-3 cDNA clone (encoding amino acids 127–945) for screening of a human lymphoblast cDNA library. We identified an interactive clone (designated clone 538) that could grow on His, Leu and Trp deficient medium and expressed β-galactosidase from an interaction dependent reporter locus. Plasmid rescue from the yeast to E. coli produced a clone with a ~800 bp long insert at its XhoI site.

Retransformation of clone 538 DNA together with the EBNA-3 construct into the yeast tester strain (SFY526) confirmed the interaction. Plasmids containing EBNA-1, ΔEBNA-4, EBNA-5, EBNA-6 and the empty pGBT9/BR vector were also retransformed together with the clone 538 DNA as negative controls. None of them showed any significant interaction in β-galactosidase test (Table 1). The strongly interacting p53/SV40LT protein pair was used as positive control.

Specific activity in β-galactosidase units of 538 co-transformants in yeast strain SFY526

* – Large T-p53 activity is 72,8 (30% of standard deviation after 4 independent experiments)

XhoI cleaved insert of clone 538 was used as a probe on Northern blots. The 2,4 kb transcript was detected ubiquitously in all tissues with highest level expression in skeletal muscle, heart and kidney (Figure 1). We have detected the same transcript in Burkitt lymphoma (BL) lines DG75, DG75-EBNA-3, DG75-EBNA-5 and freshly established LCL (data not shown).

Northern blot for 2.4 kb F538 mRNA (top) shows an ubiquitous expression in different human tissues. β-actin probe is used as loading control on the same filter (bottom).

Sequencing of the insert showed that clone 538 was identical to a part of a putative human gene with GenBank accession number of NM_017859.1, encoding the hypothetical gene product FLJ20517 (accession number NP_060329). The gene is consisted of fifteen exons that are distributed on a 19580 nucleotide long region on chromosome 20 (1325898–1306319).

To obtain the full-length cDNA (F538) we used a human heart cDNA library for PCR-amplification. The full-length cDNA was sequenced and cloned into green fluorescence protein fusion vector pEGFP-C1 (GFP) and AD vectors. The cDNA clone contained 1630 bp, encoding a 538 aa long protein. The first 10 amino acids were omitted from the cloning strategy because no selective primers could be designed for this region.

We have also identified an alternatively spliced variant of this gene (F538ΔC) where exon 12 was spliced to the middle of exon 11. This splice created a frameshift that led to a stop-codon. The encoded protein product is truncated from residue 386 with 10 extra amino acids prior to the stop-codon (Figure 2). The alternatively spliced variant was also recovered as EST from GenBank as mRNA expressed in different normal tissues such as optic nerve (BM702108), retina (BM693202) and testis (AA382318). The exon structure of the gene and the location of PCR primers are shown on Figure 3.

Sequence of F538 that is identical to the hypothetical protein FLJ20517. The sequence of the cDNA fragment isolated from the yeast two hybrid screen is underlined (top) Sequence of the C terminal part of the alternatively spliced variant with the additional ten amino acids preceding the newly created stop-codon (bottom).

Exon structure and cloning strategy for the full length F538 and the C terminal truncated splice variant

To analyse the protein-protein interaction in vitro, the insert from yeast 538 clone (the coding region corresponding to amino acids 216–473) was cloned into glutathione-S-transferase bacterial expression vector (GST-2TK). Upon induction, a 56 kD fusion protein was detected on silver or Coomassie blue stained SDS acryl amide gels.

This and several other control GST proteins were used to precipitate interacting proteins from lysates of CV-1 cells that were infected with recombinant vaccinia virus expressing full length EBNA-3. GST-538, but not GST or GST-EBNA-5 could precipitated EBNA-3 (Figure 4). GST-Full436 containing the Xap-2 gene was used as positive interaction control [12]. Lysates of cells, infected with recombinant vaccinia virus that expressed EBNA-2 were included as non-specific precipitation controls.

GST-538, but not different control fusion proteins precipitates EBNA-3 from CV-1 cell lysates infected with recombinant vaccinia virus as detected by Western blotting.

To study the subcellular localization of the protein, GFP-F538 and GFP-F538ΔC constructs were transfected into CV1 cells. Protein expression was already detectable after 4–6 hours by direct fluorescence. At this time the protein was homogeneously distributed in the cytoplasm. 24 hours after the transfection the protein started to form granular precipitates of varying size that were restricted to the cytoplasm. After 48 hours the protein formed large cytoplasmic inclusion bodies as the result of overexpression (Figure 5).

GFP-F538 localize to the cytoplasm of transfected CV1 cells. Depending on the level of expression it shows homogeneous (5–8 hours after transfection – top row), speckled (24 hours after transfection – middle row) or massively granular distribution (48 hours after transfection – bottom row). DNA staining with Hoechst 33258 is blue.

EBNA-3 is a nuclear protein. In order to test if EBNA-3 has any effect on the subcellular distribution of F538, CV-1 cells that were transfected with GFP-F538 or GFP-F538ΔC, were superinfected with recombinant vaccinia virus expressing EBNA-3 or EBNA-5. EBNA-3 but not EBNA-5 induced nuclear translocation of GFP-F538. C-terminally truncated protein remained in cytoplasm. EBNA-3 re-distributed in the nucleus and formed nuclear precipitates together with GFP-F538. These two proteins showed a high degree of co-localization (Figure 6A,6B,6C,6D,6E,6F,6G,6H,6I).

Expression of EBNA-3 (B, E, H – red) from recombinant vaccinia virus in cells that were transfected with GFP-F538 (A, D, G – green) leads to the accumulation of GFP-F538 in the nucleus in parallel with the redistribution of EBNA-3 from homogeneous nucleoplasmic pattern to well circumscribed nuclear granules. In these granules EBNA-3 (H) shows high levels of co-localization (I) with GFP-F538 (G). Expression of EBNA-5 (C – red) from recombinant vaccinia virus does not change the cytoplasmic localization of GFP-F538 (C – green). Immunofluorescence staining of endogeneous F538 protein with rabbit polyclonal antibodies shows cytoplasmic distribution in the EBV negative BL cells DG75 (J) but gives predominantly nuclear staining in EBV positive BL cells Raji (K) or EBV transformed lymphoblastoid cell line 940110 (L). DG75 cells that express EBNA-5 show the same cytoplasmic distribution as the parental cells (M) whereas EBNA-3 expressing DG75 cells demonstrate nuclear accumulation of the F538 protein (N). High magnification image of the nucleus show that F538 preferentially accumulates in low DNA density areas that corresponds to the euchromatin (O) DNA staining with Hoechst 33258 is blue.

We raised rabbit polyclonal antibodies against the bacterially produced GST-538 protein. Immunofluorescence staining detected an almost exclusively cytoplasmic distribution of F538 in the EBV negative BL cells DG75 (Figure 6J) and BL21 but gave a predominantly nuclear staining in the EBV positive BL Raji (Figure 6K) or EBV transformed lymphoblastoid cell lines Nadja, IARC171 and 940110 (Figure 6L).

Importantly the stable transfected clone of DG75 that constitutively expressed EBNA-3 in more than 95% of the cells showed nuclear accumulation of the F538 protein (Figure 6N) whereas F538 remained cytoplasmic in DG75 cells stably transfected with EBNA-5 (Figure 6M). Optical sections taken at high magnification using fluorescence 3D microscope showed that F538 preferentially accumulated in the low DNA density areas of the nucleus of the EBNA-3 expressing cells (Fig 6O).

We have built a 3D-structural model for the C-terminal part of F538, starting from residue 309, using the SWISSMODEL program [14-16]. With very high probability E-value (1e-51) it corresponded to the crystal structure of Toxoplasma gondii uracil phosphoribosyltransferase (UPRT) [17] (Figure 7). UPRT (T. gondii) belongs to PRTase-like superfamily, phosphoribosyltransferases (PRTases) family with PRTase-like fold (SCOP classification). This second half of F538 (309–548 aa) shows high sequence similarity of three of the four conserved regions (I, II and IV) where the binding sites for ribose (R) and uracil (U) are located (Figure 8). Furthermore the predicted 3D structure shows an almost identical folding even for the region III despite the sequence difference. Structure and composition of the active centre in UPRT (T. gondii) and in F538 shows also very high similarity (Figure 9). The flexible loop that is involved in the binding to another monomeric unit is present as well. The 3D model of F538 suggests that the flexible loop is folded in a similar fashion as in UPRT (T. gondii) where it is involved in the formation of homo-dimer as well as homo-tetramer, that is the active form of the enzyme (Figure 10).

Predicted 3D structure of F538 (aa 309–542) as space fill model in comparison to the crystal structure of T. gondii UPRT (2.5 Å resolution – Schumacher et al, 1998). The bound UMP is red.

Protein sequence alignment of F538 and UPRT of Toxoplasma gondii. The four conserved regions are underlined. R – ribose binding residue, U – uracil binding residues.

The structure of T. gondii UPRT active center with coordinated UMP (white balls and sticks) as compared to the predicted SWISSModel structure of F538. The UPRT conserved region III that shows the highest difference between the two proteins still exhibits the same folding pattern (black line on the backbone)

The structure of the predicted dimer derived from SWISSModel for F538. Reaction product – UMP is shown in red. N – and C-termini are indicated.

The N-terminal part contains an ATP/GTP-binding site (P-loop) suggesting that F538 also has a uridine kinase activity. High (37–42) aligning scores to the known uridine kinases support this proposition (Figure 11). F538 most probably consists of two domains with separate enzymatic activity in a similar fashion as the C. elegans UK/UPRT enzyme (Figure 12).

Multiple alignments of the N terminus of F538, mouse uridine kinase (P52263), UK/UPRT of Cryptosporidium parvum (AAG53652) and UK/UPRT of C. elegans (CAA93459). P-loop (ATP/GTP-binding site) is underlined.

Multiple alignments of the C terminus of F538, T. gondii phosphoribosyltransferase (Q26998), UK/UPRT of Cryptosporidium parvum (AAG53652) and UK/UPRT of C. elegans (CAA93459).

DiscussionIn the cell nucleotides are made either by the de novo pathway, or by the salvage pathway. The nucleosides can be salvaged by nucleoside kinases or cleaved by nucleoside phosphorylases to yield free bases and ribose-1-phosphate (desoxyribose-1-phosphate). The bases can be either salvaged by phosphoribosyltransferases or catabolized further.Uridine kinase (UK) phosphorylates uridine to uridine mono-phosphate (UMP) using ATP as a phosphate donor:Uridine+ATP → UMP+ADPUracil phosphoribosyltransferase (UPRT) can salvage uracil to UMP:PRPP+uracil → UMP+PPi,where PRPP is 5-phosphoribosyl-1-pyrophosphate and PPi is pyrophosphate. Uridine kinases, also called ATP uridine 5' phosphotransferases, are rate-limiting enzymes in the salvage pathway. Recently it was demonstrated that uracil salvage might take place also in mammalian (rat) cells, but the responsible enzyme was still unidentified [18]. It was also shown that the activity of uridine kinases is increased 5–13 fold in human colon tumors [19] ovarian carcinomas, hepatocellular carcinomas [20] and damaged tissues [21]. The level of expression was however very low in normal tissues [22]. At present, two human uridine kinases are known: human uridine kinase 1 (HUCK1 – AAK49122) and human uridine kinase 2 (HUCK2 – AAK14053).The N-terminal half of F538 shows high sequence similarity to both UCK1 and UCK2. UCK1 and UCK2 share 70% of identity at the protein level. F538 shows 38% identity to the UK/UPRT of Cryptosporidium parvum (accession number AAG53652), 50% identity to the UPRT of Toxoplasma gondii (accession number Q26998) and 51% identity to the UK/UPRT of C. elegans (accession number CAA93459).Three of four conservative domains and binding sites are present in F538 and UK/UPRT as well as in UPRT.On the basis of the high homology to UK and UK/UPRT and the similar 3D structure of the C-terminal part of F538 to the T. gondii UPRT, we propose that F538 is a novel uridine kinas/uracil phosphoribosyltransferase (UK/UPRT) – an enzyme with double catalytic activity. The protein coded by F538ΔC lacks the UPRT domain but may function as uridine kinase.Uridine kinases and UPRTs play an important role in the salvage pathway of RNA-synthesis. It was suggested [23] that de novo synthesized UTP is preferentially used for the production of UDP-sugars and phospholipids, while UTP, made by the salvage pathway, is used for RNA-synthesis. Uridine kinase activity is increased in the tumor cells. The increase of the nuclear UTP pool may be an important factor for maintenance of high proliferation rate. We suggest that the nuclear accumulation of this novel human UK/UPRT may be part of the viral strategy to convert a resting B cell to continuously proliferating lymphoblasts by securing a salvage pathway enzyme at the site of the RNA synthesis.We have shown that F538 interacts specifically with EBNA-3 in the yeast two-hybrid system and in GST pull down assays. The predominantly cytoplasmic GFP-F538 translocates to the nucleus when EBNA-3 is expressed. GFP-F538ΔC is not targeted to the nucleus by EBNA-3. This together with the sequence position of the cDNA fragment recovered from the yeast two hybrid screen indicates that binding site of F538 to EBNA-3 is located between residue 368 and 473, where the conserved domains of UPRT are located.

In the cell nucleotides are made either by the de novo pathway, or by the salvage pathway. The nucleosides can be salvaged by nucleoside kinases or cleaved by nucleoside phosphorylases to yield free bases and ribose-1-phosphate (desoxyribose-1-phosphate). The bases can be either salvaged by phosphoribosyltransferases or catabolized further.

Uridine kinase (UK) phosphorylates uridine to uridine mono-phosphate (UMP) using ATP as a phosphate donor:

Uridine+ATP → UMP+ADP

Uracil phosphoribosyltransferase (UPRT) can salvage uracil to UMP:

PRPP+uracil → UMP+PPi,

where PRPP is 5-phosphoribosyl-1-pyrophosphate and PPi is pyrophosphate. Uridine kinases, also called ATP uridine 5' phosphotransferases, are rate-limiting enzymes in the salvage pathway. Recently it was demonstrated that uracil salvage might take place also in mammalian (rat) cells, but the responsible enzyme was still unidentified [18]. It was also shown that the activity of uridine kinases is increased 5–13 fold in human colon tumors [19] ovarian carcinomas, hepatocellular carcinomas [20] and damaged tissues [21]. The level of expression was however very low in normal tissues [22]. At present, two human uridine kinases are known: human uridine kinase 1 (HUCK1 – AAK49122) and human uridine kinase 2 (HUCK2 – AAK14053).

The N-terminal half of F538 shows high sequence similarity to both UCK1 and UCK2. UCK1 and UCK2 share 70% of identity at the protein level. F538 shows 38% identity to the UK/UPRT of Cryptosporidium parvum (accession number AAG53652), 50% identity to the UPRT of Toxoplasma gondii (accession number Q26998) and 51% identity to the UK/UPRT of C. elegans (accession number CAA93459).

Three of four conservative domains and binding sites are present in F538 and UK/UPRT as well as in UPRT.

On the basis of the high homology to UK and UK/UPRT and the similar 3D structure of the C-terminal part of F538 to the T. gondii UPRT, we propose that F538 is a novel uridine kinas/uracil phosphoribosyltransferase (UK/UPRT) – an enzyme with double catalytic activity. The protein coded by F538ΔC lacks the UPRT domain but may function as uridine kinase.

Uridine kinases and UPRTs play an important role in the salvage pathway of RNA-synthesis. It was suggested [23] that de novo synthesized UTP is preferentially used for the production of UDP-sugars and phospholipids, while UTP, made by the salvage pathway, is used for RNA-synthesis. Uridine kinase activity is increased in the tumor cells. The increase of the nuclear UTP pool may be an important factor for maintenance of high proliferation rate. We suggest that the nuclear accumulation of this novel human UK/UPRT may be part of the viral strategy to convert a resting B cell to continuously proliferating lymphoblasts by securing a salvage pathway enzyme at the site of the RNA synthesis.

We have shown that F538 interacts specifically with EBNA-3 in the yeast two-hybrid system and in GST pull down assays. The predominantly cytoplasmic GFP-F538 translocates to the nucleus when EBNA-3 is expressed. GFP-F538ΔC is not targeted to the nucleus by EBNA-3. This together with the sequence position of the cDNA fragment recovered from the yeast two hybrid screen indicates that binding site of F538 to EBNA-3 is located between residue 368 and 473, where the conserved domains of UPRT are located.

ConclusionsOur data suggest that EBNA-3 may bring an enzyme that participate in the uridine salvage pathway to the euchromatic part of the nucleus. Through this, EBNA-3 may contribute to the increase of UTP nuclear pool, that in turn is needed for increased cell proliferation.

Conclusions

Our data suggest that EBNA-3 may bring an enzyme that participate in the uridine salvage pathway to the euchromatic part of the nucleus. Through this, EBNA-3 may contribute to the increase of UTP nuclear pool, that in turn is needed for increased cell proliferation.

MethodsPlasmidsThe polylinker region of the GAL-4 DNA-binding domain containing pGBT9 vector (Clontech) was modified to make it compatible with the GST-2TK expression vector (designated as BD). For AD-F538 cloning we have used pGAD424 vector from Clontech.Construction of the plasmids Full436/GST-2TK, NEBNA-1/BD, ΔEBNA-4/BD, EBNA-5/BD, EBNA-5/GST-2TK, was previously described [12].N-terminus truncated mouse p53 in pGBT9 (pVA3) and SV40 Large T-antigen in pGAD10 (pTD1, both from Clontech) were used as positive interaction controls.The GFP-F538 and AD-F538 were generated by inserting the 1642 bp PCR-product into the respective vectors (primers are described below) corresponding to amino acids 11 – 548 of the NP_060329.Yeast strains and cDNA library screeningThe Saccharomyces cerevisiae HF7c strain was used for library screening and SFY526 for confirmation of the interaction upon retransformation. Human lymphocyte MATCHMAKER cDNA library in pACT GAL-4 transcriptional activation domain vector along with the yeast strains were obtained from Clontech. Library screening was done according the Clontech protocol. Interacting clones were selected on SD plates lacking His, Leu and Trp. The fastest growing clones were further tested for β-galactosidase activity by ONPG test as described [24]. Specific activity of the given clones was calculated as percentage of β-galactosidase units of the positive control. The samples were incubated with ONPG at 30°C for 2 hours.SequencingSequencing was done using capillary Apply BioSystem sequence machine (Perkin-Elmer).Northern blottingNorthern blotting was carried out according to manufacturer protocol on ready m-RNA blot (Cat. #7760-1, Clontech).PCRPCR was carried using Perkin-Elmer or Idahotech thermocyclers.Primers to amplify the F538 sequence:5' primers: CGGGATCCGATCCTTCGCCCACGTCGCC (for GFP), GGAATTCGATCCTTCGCCCACGTCGCC (for pGAD424).3' primer: GGAATTCTCACTGGGCAGCTAACCCGT (for GFP), CGGGATCCTCACTGGGCAGCTAACCCGT (for pGAD424).Sequencing primers:5' primer: CCACCCTCAAGAAGCTGAAG3' primer:GGTAAGCTGGTTGGTCTGGAT.Primers were obtained from GIBCO BRL.Cells and cell cultureAll cell lines were cultured at 37°C, in Iscove's medium containing 10% fetal bovine serum. Periodic staining with Hoechst 33258 was used to monitore the absence of mycoplasma. CV1 cells were infected at high multiplicity with recombinant vaccinia virus encoding EBNA-3 or EBNA-5 as described [12]. CV1 was transfected with GFP-F538 or GFP-F538ΔC construct using Lipofectamine Plus Reagent (Life Technology) according to the manufacturer's protocol. Infection with recombinant vaccinia viruses was done as described [25].GST pull down assayGST pull down assay was performed as described [12]. All cell lysates contained 0,5 % of BSA as non-specific competitor.Immunofluorescence stainingImmunostaining and digital image capturing was carried out as described [26-28] using the anti-EBNA-3A monoclonal antibody T2.78–19 (kind gift of M. Rowe) and anti-EBNA-5 monoclonal antibody JF186. The polyclonal rabbit antibodies against GST-538 were produced by ASLA, Ltd, Riga, Latvia.Molecular modelling3D modelling was run using SWISS-MODEL, an automated comparative protein modelling server [14-16]. The software was obtained from ExPASy molecular biology server .Multiple alignmentMultiple aligning was run using Clustalw program [29] obtained from EMBL, European Bioinformatics Institute . We used the default parameters with BLOSUM matrix (gap opening value 10, gap extension value 0,05). Similar results were obtained using another aligning program – MAXHOM [30].

PlasmidsThe polylinker region of the GAL-4 DNA-binding domain containing pGBT9 vector (Clontech) was modified to make it compatible with the GST-2TK expression vector (designated as BD). For AD-F538 cloning we have used pGAD424 vector from Clontech.Construction of the plasmids Full436/GST-2TK, NEBNA-1/BD, ΔEBNA-4/BD, EBNA-5/BD, EBNA-5/GST-2TK, was previously described [12].N-terminus truncated mouse p53 in pGBT9 (pVA3) and SV40 Large T-antigen in pGAD10 (pTD1, both from Clontech) were used as positive interaction controls.The GFP-F538 and AD-F538 were generated by inserting the 1642 bp PCR-product into the respective vectors (primers are described below) corresponding to amino acids 11 – 548 of the NP_060329.

The polylinker region of the GAL-4 DNA-binding domain containing pGBT9 vector (Clontech) was modified to make it compatible with the GST-2TK expression vector (designated as BD). For AD-F538 cloning we have used pGAD424 vector from Clontech.

Construction of the plasmids Full436/GST-2TK, NEBNA-1/BD, ΔEBNA-4/BD, EBNA-5/BD, EBNA-5/GST-2TK, was previously described [12].

N-terminus truncated mouse p53 in pGBT9 (pVA3) and SV40 Large T-antigen in pGAD10 (pTD1, both from Clontech) were used as positive interaction controls.

The GFP-F538 and AD-F538 were generated by inserting the 1642 bp PCR-product into the respective vectors (primers are described below) corresponding to amino acids 11 – 548 of the NP_060329.

Yeast strains and cDNA library screeningThe Saccharomyces cerevisiae HF7c strain was used for library screening and SFY526 for confirmation of the interaction upon retransformation. Human lymphocyte MATCHMAKER cDNA library in pACT GAL-4 transcriptional activation domain vector along with the yeast strains were obtained from Clontech. Library screening was done according the Clontech protocol. Interacting clones were selected on SD plates lacking His, Leu and Trp. The fastest growing clones were further tested for β-galactosidase activity by ONPG test as described [24]. Specific activity of the given clones was calculated as percentage of β-galactosidase units of the positive control. The samples were incubated with ONPG at 30°C for 2 hours.

Yeast strains and cDNA library screening

The Saccharomyces cerevisiae HF7c strain was used for library screening and SFY526 for confirmation of the interaction upon retransformation. Human lymphocyte MATCHMAKER cDNA library in pACT GAL-4 transcriptional activation domain vector along with the yeast strains were obtained from Clontech. Library screening was done according the Clontech protocol. Interacting clones were selected on SD plates lacking His, Leu and Trp. The fastest growing clones were further tested for β-galactosidase activity by ONPG test as described [24]. Specific activity of the given clones was calculated as percentage of β-galactosidase units of the positive control. The samples were incubated with ONPG at 30°C for 2 hours.

SequencingSequencing was done using capillary Apply BioSystem sequence machine (Perkin-Elmer).

Sequencing was done using capillary Apply BioSystem sequence machine (Perkin-Elmer).

Northern blottingNorthern blotting was carried out according to manufacturer protocol on ready m-RNA blot (Cat. #7760-1, Clontech).

Northern blotting

Northern blotting was carried out according to manufacturer protocol on ready m-RNA blot (Cat. #7760-1, Clontech).

PCRPCR was carried using Perkin-Elmer or Idahotech thermocyclers.Primers to amplify the F538 sequence:5' primers: CGGGATCCGATCCTTCGCCCACGTCGCC (for GFP), GGAATTCGATCCTTCGCCCACGTCGCC (for pGAD424).3' primer: GGAATTCTCACTGGGCAGCTAACCCGT (for GFP), CGGGATCCTCACTGGGCAGCTAACCCGT (for pGAD424).Sequencing primers:5' primer: CCACCCTCAAGAAGCTGAAG3' primer:GGTAAGCTGGTTGGTCTGGAT.Primers were obtained from GIBCO BRL.

PCR was carried using Perkin-Elmer or Idahotech thermocyclers.

Primers to amplify the F538 sequence:

5' primers: CGGGATCCGATCCTTCGCCCACGTCGCC (for GFP), GGAATTCGATCCTTCGCCCACGTCGCC (for pGAD424).

3' primer: GGAATTCTCACTGGGCAGCTAACCCGT (for GFP), CGGGATCCTCACTGGGCAGCTAACCCGT (for pGAD424).

Sequencing primers:

5' primer: CCACCCTCAAGAAGCTGAAG

3' primer:GGTAAGCTGGTTGGTCTGGAT.

Primers were obtained from GIBCO BRL.

Cells and cell cultureAll cell lines were cultured at 37°C, in Iscove's medium containing 10% fetal bovine serum. Periodic staining with Hoechst 33258 was used to monitore the absence of mycoplasma. CV1 cells were infected at high multiplicity with recombinant vaccinia virus encoding EBNA-3 or EBNA-5 as described [12]. CV1 was transfected with GFP-F538 or GFP-F538ΔC construct using Lipofectamine Plus Reagent (Life Technology) according to the manufacturer's protocol. Infection with recombinant vaccinia viruses was done as described [25].

Cells and cell culture

All cell lines were cultured at 37°C, in Iscove's medium containing 10% fetal bovine serum. Periodic staining with Hoechst 33258 was used to monitore the absence of mycoplasma. CV1 cells were infected at high multiplicity with recombinant vaccinia virus encoding EBNA-3 or EBNA-5 as described [12]. CV1 was transfected with GFP-F538 or GFP-F538ΔC construct using Lipofectamine Plus Reagent (Life Technology) according to the manufacturer's protocol. Infection with recombinant vaccinia viruses was done as described [25].

GST pull down assayGST pull down assay was performed as described [12]. All cell lysates contained 0,5 % of BSA as non-specific competitor.

GST pull down assay

GST pull down assay was performed as described [12]. All cell lysates contained 0,5 % of BSA as non-specific competitor.

Immunofluorescence stainingImmunostaining and digital image capturing was carried out as described [26-28] using the anti-EBNA-3A monoclonal antibody T2.78–19 (kind gift of M. Rowe) and anti-EBNA-5 monoclonal antibody JF186. The polyclonal rabbit antibodies against GST-538 were produced by ASLA, Ltd, Riga, Latvia.

Immunofluorescence staining

Immunostaining and digital image capturing was carried out as described [26-28] using the anti-EBNA-3A monoclonal antibody T2.78–19 (kind gift of M. Rowe) and anti-EBNA-5 monoclonal antibody JF186. The polyclonal rabbit antibodies against GST-538 were produced by ASLA, Ltd, Riga, Latvia.

Molecular modelling3D modelling was run using SWISS-MODEL, an automated comparative protein modelling server [14-16]. The software was obtained from ExPASy molecular biology server .

Molecular modelling

3D modelling was run using SWISS-MODEL, an automated comparative protein modelling server [14-16]. The software was obtained from ExPASy molecular biology server .

Multiple alignmentMultiple aligning was run using Clustalw program [29] obtained from EMBL, European Bioinformatics Institute . We used the default parameters with BLOSUM matrix (gap opening value 10, gap extension value 0,05). Similar results were obtained using another aligning program – MAXHOM [30].

Multiple alignment

Multiple aligning was run using Clustalw program [29] obtained from EMBL, European Bioinformatics Institute . We used the default parameters with BLOSUM matrix (gap opening value 10, gap extension value 0,05). Similar results were obtained using another aligning program – MAXHOM [30].

Authors' contributionsEK carried out most of the experiments and drafted the manuscript. VK participated in the cloning of the full-length cDNA. TS participated in the 3D structure predictions. GK participated in the design of the study. LS conceived of the study, and participated in its design and coordination.All authors read and approved the final manuscript.

Authors' contributions

EK carried out most of the experiments and drafted the manuscript. VK participated in the cloning of the full-length cDNA. TS participated in the 3D structure predictions. GK participated in the design of the study. LS conceived of the study, and participated in its design and coordination.

All authors read and approved the final manuscript.
