---
pmid: '29892012'
title: An interactome perturbation framework prioritizes damaging missense mutations
  for developmental disorders.
authors:
- Chen S
- Fragoza R
- Klei L
- Liu Y
- Wang J
- Roeder K
- Devlin B
- Yu H
journal: Nat Genet
year: '2018'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC6314957
doi: 10.1038/s41588-018-0130-z
---

# An interactome perturbation framework prioritizes damaging missense mutations for developmental disorders.
**Authors:** Chen S, Fragoza R, Klei L, Liu Y, Wang J, Roeder K, Devlin B, Yu H
**Journal:** Nat Genet (2018)
**DOI:** [10.1038/s41588-018-0130-z](https://doi.org/10.1038/s41588-018-0130-z)
**PMC:** [PMC6314957](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6314957/)

## Abstract

1. Nat Genet. 2018 Jul;50(7):1032-1040. doi: 10.1038/s41588-018-0130-z. Epub 2018
 Jun 11.

An interactome perturbation framework prioritizes damaging missense mutations 
for developmental disorders.

Chen S(1)(2)(3), Fragoza R(1)(2)(3), Klei L(4), Liu Y(1)(2), Wang J(5), Roeder 
K(6)(7), Devlin B(8), Yu H(9)(10).

Author information:
(1)Department of Biological Statistics and Computational Biology, Cornell 
University, Ithaca, NY, USA.
(2)Weill Institute for Cell and Molecular Biology, Cornell University, Ithaca, 
NY, USA.
(3)Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY, 
USA.
(4)Department of Psychiatry, University of Pittsburgh School of Medicine, 
Pittsburgh, PA, USA.
(5)Department of Statistics and Data Science, Carnegie Mellon University, 
Pittsburgh, PA, USA.
(6)Department of Statistics and Data Science, Carnegie Mellon University, 
Pittsburgh, PA, USA. roeder@andrew.cmu.edu.
(7)Computational Biology Department, Carnegie Mellon University, Pittsburgh, PA, 
USA. roeder@andrew.cmu.edu.
(8)Department of Psychiatry, University of Pittsburgh School of Medicine, 
Pittsburgh, PA, USA. devlinbj@upmc.edu.
(9)Department of Biological Statistics and Computational Biology, Cornell 
University, Ithaca, NY, USA. haiyuan.yu@cornell.edu.
(10)Weill Institute for Cell and Molecular Biology, Cornell University, Ithaca, 
NY, USA. haiyuan.yu@cornell.edu.

Identifying disease-associated missense mutations remains a challenge, 
especially in large-scale sequencing studies. Here we establish an 
experimentally and computationally integrated approach to investigate the 
functional impact of missense mutations in the context of the human interactome 
network and test our approach by analyzing ~2,000 de novo missense mutations 
found in autism subjects and their unaffected siblings. Interaction-disrupting 
de novo missense mutations are more common in autism probands, principally 
affect hub proteins, and disrupt a significantly higher fraction of hub 
interactions than in unaffected siblings. Moreover, they tend to disrupt 
interactions involving genes previously implicated in autism, providing 
complementary evidence that strengthens previously identified associations and 
enhances the discovery of new ones. Importantly, by analyzing de novo missense 
mutation data from six disorders, we demonstrate that our interactome 
perturbation approach offers a generalizable framework for identifying and 
prioritizing missense mutations that contribute to the risk of human disease.

DOI: 10.1038/s41588-018-0130-z
PMCID: PMC6314957
PMID: 29892012 [Indexed for MEDLINE]

Conflict of interest statement: Competing Interests The authors declare no 
competing financial interests.

## Full Text

Abstract

Identifying disease-associated missense mutations remains a challenge, especially in large-scale sequencing studies. Here we establish an experimentally and computationally integrated approach to investigate the functional impact of missense mutations in the context of the human interactome network and test our approach by analyzing ~2,000 de novo missense (dnMis) mutations found in autism subjects and their unaffected siblings. Interaction-disrupting dnMis mutations are more common in autism probands, these mutations principally affect hub proteins, and they disrupt a significantly higher fraction of hub interactions than in unaffected siblings. Additionally, they tend to disrupt interactions involving genes previously implicated in autism, providing complementary evidence that strengthens previously identified associations and enhances discovery of new ones. Importantly, by analyzing dnMis data from six disorders, we demonstrate that our interactome perturbation approach offers a generalizable framework for identifying and prioritizing missense mutations that contribute risk to human disease.

Introduction

Mutations disrupting the function of proteins are recognized as an important source of risk for developmental disorders (DDs), such as intellectual disability 1 , 2 , autism spectrum disorder (ASD) 3 , and congenital heart defects 4 . Whole-exome sequencing (WES) has produced a boon of findings linking de novo mutations to risk for DDs 5 – 19 . Not all mutations are simple to interpret as causing a loss of gene function. Missense mutations are especially difficult; although there are bioinformatics tools to predict the level of damage 20 – 22 , these annotators are far from perfect. This is a critical deficiency because the majority of coding mutations are missense. Here we show that one key feature in evaluating the disruptiveness of mutations is whether they fall in known or predicted protein-protein interaction interfaces and their likelihood to disrupt these interactions.

Large-scale studies of known disease-associated mutations have already reported a strong association with binding interfaces of protein interactions 23 , 24 . The major bottleneck for wide application of this feature is limited knowledge about the set of interactions and the binding interfaces of all interactions. To experimentally evaluate the impact of mutations on protein interactions, we establish a high-throughput mutagenesis and interactome-scanning pipeline for generating site-specific mutant clones and testing corresponding mutant protein interactions. Such a pipeline, however, cannot evaluate the impact of missense mutations on many interactions, because high-throughput interaction assays are limited in their coverage 25 – 27 . For this reason, we also explore a computational approach for systematically examining the functional impact of missense mutations on protein interactions. This approach builds on our newly-established full-interactome interface predictions 28 to computationally predict the impact of all missense mutations on all associated interactions. Here we apply our experimental and computational approaches in tandem, which can be applied to any WES study.

To evaluate the effectiveness of our integrated experimental-computational approach, we focus on 2,821 de novo missense (dnMis) mutations identified from WES of ~2,500 families from the Simons Simplex Collection (SSC) 29 ( Supplementary Table 1 ). The SSC targets the study of ASD through a cohort of parent-offspring trios or quads with two unaffected parents, an ASD proband and, for most families, an unaffected sibling 30 . Previous analyses of the SSC data have reported significantly higher de novo mutation rates in ASD probands versus unaffected siblings across various mutation types, from copy number variants (CNVs) 29 , 31 , 32 , frameshift indels 33 , to missense mutations 12 – 14 , 16 . While a number of risk de novo copy number 29 , 31 , 32 , 34 , 35 and protein truncating 12 – 14 , 16 , 33 variants have been identified, exactly which dnMis mutations play a role and to what extent are open questions. We applied our integrated framework to evaluate the effect of 1,733 dnMis mutations within a protein interactome framework aiming to identify potentially disease-contributing dnMis mutations. Though there are many ways by which a missense mutation can impact a protein’s function, such as by destabilizing protein folding, we evaluate the disruptiveness of a mutation within our framework exclusively on its capacity to disrupt protein interactions, measured experimentally or through prediction. We further compare the network properties of proteins impacted by interaction-disrupting and non-disrupting dnMis mutations, using unaffected siblings as negative controls throughout. While our analyses focus on dnMis mutations in ASD, the integrated experimental-computational approach provides a generalizable framework for investigating the impact of missense mutations uncovered by WES for human diseases.

Discussion

Here we demonstrated that dnMis mutations can contribute to ASD risk by disrupting protein-protein interactions and that our interactome perturbation framework offers a novel and effective way to identify ASD risk dnMis mutations. Because only a small fraction of dnMis mutations found in ASD subjects are believed to be functional 12 , this framework helps overcome a significant challenge in identifying risk dnMis mutations. Our analyses focused on dnMis mutations from the SSC families because the information on unaffected siblings in the dataset provides robust negative controls. Our results demonstrated that interaction-disrupting dnMis mutations in ASD probands preferentially impact proteins that have many interaction partners in the interactome network (i.e., hubs) and disrupt these interactions at a significantly higher rate than those in unaffected siblings. Our results also lend evidence to previously reported ASD-associated genes and pathways by showing that interaction-disrupting dnMis mutations are closely clustered to proteins in ASD-associated functional classes in the interactome network. Thus, characterizing interactome perturbation provides additional and potentially orthogonal information to strengthen previously identified genetic associations and helps discover new genes that contribute to ASD risk.

Integration of computational predictions with experimental data imbued far more meaning onto missense mutations found in ASD probands and their siblings. Thus the prediction model alone can enhance researchers’ ability to prioritize damaging missense mutations and can be applied across a wide range of human disease studies. We emphasize that the strength of this prediction model is rooted in its integration of PPH2 scores and Interactome INSIDER interface predictions. To demonstrate this, we repeated all analyses using PPH2 and Interactome INSIDER separately. The results show that neither method individually is sufficient to reproduce most signals towards identifying disease-contributing dnMis mutations in ASD ( Supplementary Fig. 2 and Supplementary Fig. 3 ). This confirms that our two-tiered predictor, which evaluates the disruptiveness of a variant on protein interactions, greatly improves the effectiveness of predicting functional missense mutations. We also note that our predictor is robust at different PPH2 score cutoffs ( Supplementary Fig. 4 ). Taken together, we demonstrate that our computational prediction approach can serve as an effective and robust method to identify disease-contributing missense mutations.

Stronger associations between ASD proband mutations and clinical data can be established by filtering out ASD dnMis mutations that are also identified as population variants in the Exome Aggregation Consortium (ExAC) 56 since these variants represent standing variations in the population and are less likely to be deleterious as a result 68 . Adopting this same principle, we found that dnMis mutations in DDs are significantly more enriched on interaction interfaces when mutations coinciding with ExAC are filtered out in comparison to the non-filtered set ( Supplementary Fig. 5 and Supplementary Note ). Importantly, our results show that the characteristic network and haploinsufficiency properties of disruptive proband dnMis mutations are not unique to ASD but are shared features across different DDs ( Supplementary Fig. 6 ), indicating that our interactome perturbation framework is generalizable to prioritize dnMis mutations across a wide range of DDs ( Supplementary Note ).

Our analyses indicate that network properties are important in interpreting the functional impact of dnMis mutations and their relevance towards disease etiology. However, we recognize that the human interactome with which these analyses are performed is currently incomplete. As a result, certain classes of protein interactions, for example interactions mediated by membrane-bound proteins, may be under-represented in the current interactome, limiting potential insights from such proteins. Moreover, literature-derived segments of the human interactome are subject to sampling bias present in small-scale studies 38 , 69 . Therefore, we re-examined the network topology analyses across a chronologically-ordered series of unbiased high-throughput (HT)-derived human interactomes ( Supplementary Note ). We show that not only are our results robust across all HT-derived interactomes, more importantly, we also demonstrate that the topological differences between interaction-disrupting and non-disrupting dnMis mutations in probands becomes more significant as the interactome coverage increases ( Supplementary Fig. 7 ). Moreover, we show that all our results remain the same when we expanded our disruptiveness predictions to include all dnMis mutations in the current human interactome ( Supplementary Fig. 8 and Supplementary Note ). Taken together, we fully expect that as increasingly more human protein-protein interactions and mutations are uncovered, our interactome perturbation framework can be applied to these new interactions and mutations to identify new or currently under-characterized disease-associated mutations and genes.

As large-scale WES studies continue to produce mutation data at ever-increasing scales, our interaction-disruption prediction approach can greatly extend the reach of interactome perturbation studies for investigating complex genotype-phenotype relationships and improving our understanding of how genetic variation affects disease risk through the alteration of topological and community structures of networks.
