---
pmid: '21224850'
title: Large ring polymers align FtsZ polymers for normal septum formation.
authors:
- Gündoğdu ME
- Kawai Y
- Pavlendova N
- Ogasawara N
- Errington J
- Scheffers DJ
- Hamoen LW
journal: EMBO J
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3034016
doi: 10.1038/emboj.2010.345
---

# Large ring polymers align FtsZ polymers for normal septum formation.
**Authors:** Gündoğdu ME, Kawai Y, Pavlendova N, Ogasawara N, Errington J, Scheffers DJ, Hamoen LW
**Journal:** EMBO J (2011)
**DOI:** [10.1038/emboj.2010.345](https://doi.org/10.1038/emboj.2010.345)
**PMC:** [PMC3034016](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3034016/)

## Abstract

1. EMBO J. 2011 Feb 2;30(3):617-26. doi: 10.1038/emboj.2010.345. Epub 2011 Jan
11.

Large ring polymers align FtsZ polymers for normal septum formation.

Gündoğdu ME(1), Kawai Y, Pavlendova N, Ogasawara N, Errington J, Scheffers DJ, 
Hamoen LW.

Author information:
(1)Centre for Bacterial Cell Biology, Institute for Cell and Molecular 
Biosciences, Newcastle University, Newcastle, UK.

Cytokinesis in bacteria is initiated by polymerization of the tubulin homologue 
FtsZ into a circular structure at midcell, the Z-ring. This structure functions 
as a scaffold for all other cell division proteins. Several proteins support 
assembly of the Z-ring, and one such protein, SepF, is required for normal cell 
division in Gram-positive bacteria and cyanobacteria. Mutation of sepF results 
in deformed division septa. It is unclear how SepF contributes to the synthesis 
of normal septa. We have studied SepF by electron microscopy (EM) and found that 
the protein assembles into very large (∼50 nm diameter) rings. These rings were 
able to bundle FtsZ protofilaments into strikingly long and regular tubular 
structures reminiscent of eukaryotic microtubules. SepF mutants that disturb 
interaction with FtsZ or that impair ring formation are no longer able to align 
FtsZ filaments in vitro, and fail to support normal cell division in vivo. We 
propose that SepF rings are required for the regular arrangement of FtsZ 
filaments. Absence of this ordered state could explain the grossly distorted 
septal morphologies seen in sepF mutants.

DOI: 10.1038/emboj.2010.345
PMCID: PMC3034016
PMID: 21224850 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflict 
of interest.

## Full Text

IntroductionThe earliest known event in bacterial cell division is the assembly of the tubulin-like protein FtsZ into a circular structure at midcell (Adams and Errington, 2009). This so-called Z-ring recruits the other proteins needed for synthesis of the division septum. How the Z-ring is structured in the cell is not really known. Electron microscopy (EM) studies have shown that, depending on the reaction conditions, purified FtsZ can polymerize into long bundles, and structures like sheets, mini-rings, and helices (Bramhill and Thompson, 1994; Mukherjee and Lutkenhaus, 1999; Lu et al, 2000; Popp et al, 2009). Assembly of FtsZ into a stable Z-ring at the site of cell division involves several other proteins. One key player is FtsA, which binds FtsZ and links the Z-ring to the membrane via an amphipathic α-helical domain (Jensen et al, 2005; Pichoff and Lutkenhaus, 2005). Another conserved protein, ZapA, forms a link between FtsZ protofilaments and stimulates polymerization (Gueiros-Filho and Losick, 2002; Small et al, 2007). In rod-shaped bacteria the Min proteins exercise a regulatory role, and inhibit polymerization of FtsZ close to cell poles (Hu et al, 1999; Scheffers, 2008). Gram-positive bacteria use another FtsZ regulator, the integral membrane protein EzrA (Levin et al, 1999). Deletion of this protein leads to extra Z-rings, and it is therefore considered a negative regulator. In this study we investigated SepF (YlmF), a Z-ring-associated protein that is highly conserved in Gram-positive bacteria and cyanobacteria (Miyagishima et al, 2005; Hamoen et al, 2006; Ishikawa et al, 2006).The first indications that SepF has a role in cell division came from studies with Streptococcus pneumoniae and the cyanobacterium Synechococcus elongatus, which showed that mutations in sepF lead to severe cell division defects (Fadda et al, 2003; Miyagishima et al, 2005). Further studies with Bacillus subtilis revealed that SepF localizes to the division site. This localization depended on the presence of FtsZ, and yeast two-hybrid experiments showed a direct interaction between both proteins (Hamoen et al, 2006; Ishikawa et al, 2006). Deletion of sepF results in grossly deformed division septa in B. subtilis. Interestingly, deformed septa are not observed with other cell division mutants. SepF has an apparent functional overlap with FtsA. FtsA is not essential in B. subtilis, although B. subtilis ftsA mutants grow slower and form filamentous cells (Beall and Lutkenhaus, 1992; Jensen et al, 2005). This phenotype can be restored by overexpression of sepF. A double disruption of sepF and ftsA completely eliminates Z-ring formation and is lethal. This functional overlap with FtsA led Ishikawa et al (2006) to conclude that, like FtsA, SepF is involved in the early stages of Z-ring assembly. However, these results seem to contradict our earlier data on SepF. If SepF stimulates polymerization of FtsZ one may assume that a sepF mutant becomes sensitive for reduced FtsZ levels, but this is not the case (Hamoen et al, 2006). Furthermore, if SepF supports Z-ring formation it is unlikely that deletion of a negative regulator of FtsZ polymerization would cause problems, and yet introduction of a sepF mutation in an ezrA deletion background proved to be lethal.It is unclear at what stage of the division process SepF is active, and to gain more information on this we isolated dominant negative sepF mutants. To test whether the mutations affected the interaction with FtsZ, we purified SepF. This resulted in a rather striking observation. It turned out that SepF assembles into very large rings that can bundle FtsZ protofilaments into long and regular tubular structures. The dominant negative sepF mutants were unable to form these tubules. A mutation that blocked ring formation was also unable to align FtsZ protofilaments. The results support a new model in which SepF forms regular ring polymers that organize FtsZ protofilaments into higher order structures required for a smooth invagination of the Gram-positive septal wall. Thus, SepF is more than a simple positive regulator of Z-ring formation, which might explain why its absence is synthetic lethal in an ftsA mutant as well as in an ezrA mutant.

Introduction

The earliest known event in bacterial cell division is the assembly of the tubulin-like protein FtsZ into a circular structure at midcell (Adams and Errington, 2009). This so-called Z-ring recruits the other proteins needed for synthesis of the division septum. How the Z-ring is structured in the cell is not really known. Electron microscopy (EM) studies have shown that, depending on the reaction conditions, purified FtsZ can polymerize into long bundles, and structures like sheets, mini-rings, and helices (Bramhill and Thompson, 1994; Mukherjee and Lutkenhaus, 1999; Lu et al, 2000; Popp et al, 2009). Assembly of FtsZ into a stable Z-ring at the site of cell division involves several other proteins. One key player is FtsA, which binds FtsZ and links the Z-ring to the membrane via an amphipathic α-helical domain (Jensen et al, 2005; Pichoff and Lutkenhaus, 2005). Another conserved protein, ZapA, forms a link between FtsZ protofilaments and stimulates polymerization (Gueiros-Filho and Losick, 2002; Small et al, 2007). In rod-shaped bacteria the Min proteins exercise a regulatory role, and inhibit polymerization of FtsZ close to cell poles (Hu et al, 1999; Scheffers, 2008). Gram-positive bacteria use another FtsZ regulator, the integral membrane protein EzrA (Levin et al, 1999). Deletion of this protein leads to extra Z-rings, and it is therefore considered a negative regulator. In this study we investigated SepF (YlmF), a Z-ring-associated protein that is highly conserved in Gram-positive bacteria and cyanobacteria (Miyagishima et al, 2005; Hamoen et al, 2006; Ishikawa et al, 2006).

The first indications that SepF has a role in cell division came from studies with Streptococcus pneumoniae and the cyanobacterium Synechococcus elongatus, which showed that mutations in sepF lead to severe cell division defects (Fadda et al, 2003; Miyagishima et al, 2005). Further studies with Bacillus subtilis revealed that SepF localizes to the division site. This localization depended on the presence of FtsZ, and yeast two-hybrid experiments showed a direct interaction between both proteins (Hamoen et al, 2006; Ishikawa et al, 2006). Deletion of sepF results in grossly deformed division septa in B. subtilis. Interestingly, deformed septa are not observed with other cell division mutants. SepF has an apparent functional overlap with FtsA. FtsA is not essential in B. subtilis, although B. subtilis ftsA mutants grow slower and form filamentous cells (Beall and Lutkenhaus, 1992; Jensen et al, 2005). This phenotype can be restored by overexpression of sepF. A double disruption of sepF and ftsA completely eliminates Z-ring formation and is lethal. This functional overlap with FtsA led Ishikawa et al (2006) to conclude that, like FtsA, SepF is involved in the early stages of Z-ring assembly. However, these results seem to contradict our earlier data on SepF. If SepF stimulates polymerization of FtsZ one may assume that a sepF mutant becomes sensitive for reduced FtsZ levels, but this is not the case (Hamoen et al, 2006). Furthermore, if SepF supports Z-ring formation it is unlikely that deletion of a negative regulator of FtsZ polymerization would cause problems, and yet introduction of a sepF mutation in an ezrA deletion background proved to be lethal.

It is unclear at what stage of the division process SepF is active, and to gain more information on this we isolated dominant negative sepF mutants. To test whether the mutations affected the interaction with FtsZ, we purified SepF. This resulted in a rather striking observation. It turned out that SepF assembles into very large rings that can bundle FtsZ protofilaments into long and regular tubular structures. The dominant negative sepF mutants were unable to form these tubules. A mutation that blocked ring formation was also unable to align FtsZ protofilaments. The results support a new model in which SepF forms regular ring polymers that organize FtsZ protofilaments into higher order structures required for a smooth invagination of the Gram-positive septal wall. Thus, SepF is more than a simple positive regulator of Z-ring formation, which might explain why its absence is synthetic lethal in an ftsA mutant as well as in an ezrA mutant.

ResultsSepF polymerizes into very large ringsTo investigate in detail how SepF influences the polymerization of FtsZ, we decided to purify both proteins for biochemical analysis. SepF was purified using maltose-binding protein (MBP) as affinity tag. The MBP moiety was cleaved from SepF and removed by ion-exchange chromatography. When subjected to size exclusion chromatography, SepF eluted in the void volume, suggesting that the protein formed aggregates. To confirm this, we examined the protein sample by EM, yet the EM images revealed a very surprising result. It proved that SepF polymerizes into very large regular ring structures with an average diameter of about 50 nm (Figure 1). These rings are so wide that, theoretically, they can encompass two whole ribosomes. The formation of these rings seemed a robust process as they were readily formed at high salt concentrations (500 mM), and in the presence of other proteins (e.g., BSA). Although the buffer used in Figure 1 contains Mg2+ (see below), this is not required for ring formation. Rings were still visible at SepF concentrations as low as 0.1 μM, but their number was greatly reduced, and they were difficult to find on the EM grid. It was therefore not possible to determine the critical concentration at which rings were formed. The ring structures display a limited variability in ring diameter (Figure 1). This suggests that SepF filaments, presumptive precursors of the closed rings, are rather inflexible and always close on themselves.SepF is an abundant proteinThe striking SepF rings begged the question what effect they would have on FtsZ polymerization. To analyze this under physiological conditions, we first determined the ratio of both proteins in the cell. Antibodies against purified SepF were raised and used to perform a quantitative western blot analysis. For exponentially growing cells, we estimated that the cellular concentrations of SepF and FtsZ were ∼6 and 8 μM, respectively (Supplementary Figure S1). This roughly correspond to 8000 molecules of SepF per cell, and 11 000 molecules for FtsZ, assuming a cell volume of 2.6 μm3 (Henriques et al, 1998). FtsZ is an abundant protein in the cell (Wang and Lutkenhaus, 1993), and the fact that the amount of SepF is almost as high as that of FtsZ suggests that the SepF rings have a structural role rather than a regulatory role in Z-ring formation.SepF-ring formation requires physiological reaction conditionsFtsZ polymerization can be followed by pelleting and light scattering assays (Mukherjee and Lutkenhaus, 1998; Scheffers, 2008). When purified SepF was included in an FtsZ polymerization experiment a small increase in the amount of pelleted FtsZ was observed, suggesting that SepF stimulates polymerization of FtsZ (Figure 2A, lanes 1 and 2). However, we noticed a problem with the solubility of SepF. FtsZ polymerization assays are generally performed at a relatively low pH of around pH 6.5, as this stimulates the lateral association of FtsZ protofilaments (Mukherjee and Lutkenhaus, 1999). It emerged that the low pH resulted in precipitation of SepF, and EM images of SepF in a buffer of pH 6.5 showed that protein rings were no longer present (Figure 1D). Precipitation of SepF was reduced by increasing the salt concentration (300 mM KCl) and using a buffer with a more physiological pH of 7.4 (Booth, 1985; Breeuwer et al, 1996) (Supplementary Figure S2B). This effect did not depend on the type of buffer used, and the same result was obtained with a Tris, HEPES or MES buffer (data not shown). At pH 7.4 and 300 mM salt, the total amount of pelleted FtsZ was substantially less, but the stimulatory effect of SepF on pelleting was again detectable (Figure 2A, lanes 3 and 4). In light scattering assays using standard polymerization buffer of pH 6.5, B. subtilis FtsZ gave a classic response upon the addition of GTP, with a strong increase in light scatter signal that decreased over time (Supplementary Figure S3) (Mukherjee and Lutkenhaus, 1999). The light scattering signal was strongly reduced by a shift to pH 7.4, and almost eliminated by increased salt concentrations (Supplementary Figure S3), indicating that lateral interactions between FtsZ protofilaments are greatly diminished under these conditions. The addition of SepF did result in a shift in light scattering signal (Figure 2B) but not the gradual rise that is normally associated with bundling of FtsZ protofilaments. SepF had no measurable effect on the GTPase activity of FtsZ (Figure 2C).SepF bundles FtsZ into tubulesWe returned to EM to investigate whether the SepF rings had any effect on FtsZ filaments. In a buffer of pH 7.4 with 300 mM KCl, FtsZ forms clear protofilaments when GTP is present (Figure 3A). The addition of SepF had a remarkable result and long regular tubular structures became visible (Figure 3B). The average diameter of these structures was about 48 nm (Figure 3C), close to the diameter of SepF rings. Tubule formation was not influenced by the order in which the proteins or GTP were added, but depended on the concentration of FtsZ. When half the amount of FtsZ was used (5 μM) less tubules were detected. Increasing the SepF concentration did not compensate for this. At half the SepF concentration (3 μM) tubules were still visible, but at 1 μM SepF and 10 μM FtsZ no tubules could be identified on the EM grids. When GDP instead of GTP was used no tubules were formed. We never observed tubules in polymerization buffer of pH 6.5, which may explain why they were not observed in a previous study (Singh et al, 2008). We used 10 mM Mg2+ in the polymerization buffers, which is commonly used for FtsZ polymerization studies, but 1 mM Mg2+ was sufficient to see tubules. Without Mg2+ no tubules were formed. At higher magnifications the tubules appear to consist of straight longitudinal filaments, most likely FtsZ polymers (Figure 3D, narrow arrows), with evident transverse bands that presumably represent SepF rings (Figure 3D, wide arrows). From the EM images it seems that the SepF rings form the core of the FtsZ–SepF tubules, but attempts to resolve this question using thin section EM or Cryo-EM were unsuccessful. The length of the tubules increased with time and tubules could grow up to micrometres in length (Figure 3E; Supplementary Figure S4). Interestingly, during the first 5 min of the reaction filaments emanating from the ends of the short tubules tended to be straight (Figure 4A and B), whereas after 10 min, some tubules had splayed tips with highly curved filaments (Figure 4C–E). In the later samples (20 min), tubules with different kind of bifurcations, including branches and rings, were evident (Figures 3E and 4F–I).Screen for SepF mutations that inhibit cell divisionAlthough the formation of FtsZ–SepF tubules appeared a reproducible and robust process in vitro, it was important to test its physiological relevance. We investigated whether it was possible to isolate dominant negative mutations in SepF that would affect the interaction of SepF with FtsZ and so disrupt tubule formation. The screen took advantage of the fact that a combination of sepF and ftsA null mutations is lethal (Ishikawa et al, 2006). Randomly mutagenized alleles of sepF were expressed from a xylose inducible Pxyl-promoter in cells carrying a wild-type sepF gene and a deletion of ftsA. After testing several thousands of transformants two clones were identified that were unable to grow when the sepF mutant allele was overexpressed by the addition of xylose to the growth medium (Figure 5A). Sequencing of the mutated sepF alleles revealed two different mutations affecting conserved residues: A98V and F124S (Figure 6). When these SepF mutants were (over)expressed in a strain devoid of wild-type SepF, cells remained elongated, and there was no indication that the mutant proteins could compensate for the absence of wild-type protein. To test whether the dominant negative sepF mutants affected Z-ring formation, we examined the localization of a gfp–ftsZ fusion. As shown in Figure 5B (upper panel), the overexpression of wild-type SepF in an ftsA mutant background had no effect on Z-ring formation. However, when the mutants A98V or F124S were induced (Figure 5B, lower panels), Z-ring formation was abolished, indicating that the mutant proteins inhibit cell division by interfering with Z-ring formation.Dominant negative SepF mutants do not form tubulesThe inability of the SepF mutants to support growth could be a consequence of mutated FtsZ-binding sites or possibly a failure to form ring structures. To test this, the mutant SepF proteins were purified and first analyzed by size exclusion chromatography. Wild-type SepF showed a large peak in the void volume (>2 MDa), probably corresponding to SepF rings (Figure 7A). The F124S mutant showed a comparable elution profile to the wild-type protein. However, the A98V mutant was almost absent from the void volume, suggesting that this mutant affects ring formation. This protein showed an increased elution at around 14 ml between the peaks for the reference proteins thyroglobulin (669 kDa) and aldolase (158 kDa). Thus, the A98V mutation still gives oligomeric structures and does not reduce the protein to its monomeric state of 17 kDa. EM analysis showed normal ring formation for the F124S mutant (Figure 7B). In case of the A98V mutant it was difficult to find rings on the EM grids, indicating that this mutant protein is indeed disturbed in ring formation.We then examined the ability of the mutant proteins to support the formation of FtsZ tubules. Importantly, neither of the mutant proteins formed any detectable tubular structures under conditions in which wild-type SepF supported abundant tubulation (Supplementary Figure S5). As the F124S protein still forms rings, it is possible that this mutant is unable to interact with FtsZ. To test this, we performed an FtsZ–SepF co-elution experiment using MBP–SepF fusions. MBP–SepF, when bound to amylose resin, is capable of selectively binding FtsZ from an extract of Escherichia coli cells expressing B. subtilis FtsZ (Figures 7C and 8A). MBP–SepF mutant fusions, bound to amylose resin, retained significantly less FtsZ compared with wild-type protein. In fact, the amount of FtsZ that interacted with these mutant proteins was only detectable by western blot analysis. Thus, both A98V and F124S mutations impair the interaction with FtsZ protein. GFP fusions with these SepF mutants showed diffuse fluorescence in cells, in agreement with a loss of FtsZ-binding activity (data not shown).The F124S mutant largely retained the ability to form rings, and it is likely that its phenotype is manifested mainly through impaired interaction with FtsZ. Possibly, this protein interferes with wild-type SepF by creating hybrid rings that are less efficient in bundling FtsZ protofilaments. The A98V mutant could have a similar effect. We have tried to visualize hybrid rings by employing SepF fusion proteins, such as an N-terminal MBP–SepF fusion or a C-terminal SepF–Intein fusion, but both fusion proteins were unable to form rings by themselves. This could also explain why GFP fusions of SepF are not active in vivo. Indeed, when we examined purified SepF–GFP by EM no protein rings were detected (data not shown).Breaking the ring abolishes tubule formationThe above results suggest that alignment of FtsZ protofilaments is a key activity of SepF. However, it is still unclear whether the formation of rings is important for this activity. The inability of a SepF–GFP fusion to form rings indicates that the C-terminus is probably important for ring formation. Indeed, removal of the putative α-helical domain at the C-terminus (mutant Δ134) abolishes formation of SepF rings and of SepF–FtsZ tubules, even though the truncated SepF can still bind FtsZ (Figure 8A, data not shown). The first residue in this 15 amino acid C-terminal domain is a conserved glycine (Figure 6). Mutation of this glycine to asparagine (G135N) had no consequences for SepF–FtsZ interaction as the mutant protein still bound FtsZ in vitro (Figure 8A), and localized to the Z-ring in vivo (Figure 8B). When we analyzed purified G135N mutant protein by EM no protein rings were detectable, instead the protein appeared to assemble into long filamentous structures (Figure 8C, left panel). Importantly, mixing the mutant protein with purified FtsZ did not result in tubular structures (Figure 8C, right panel). These data suggest that the C-terminus of SepF is required for the formation of rings, and that SepF rings are required for the alignment of FtsZ polymers. To test whether the sepF-ring mutant was active in vivo, we replaced wild-type sepF with sepF–G135N in a B. subtilis strain that contains ftsA under control of the IPTG-inducible Pspac promoter. When the resulting strain was grown on plates without IPTG, no colonies appeared (Figure 8D), despite the fact that expression levels of SepF–G135N were normal (Figure 8E). The mutant, when overexpressed in an ftsA deletion strain, showed no dominant negative effect like the SepF mutants A98V and F124S. The finding that SepF–G135N cannot support cell growth in the absence of FtsA indicates that SepF-ring formation is critical for the function of SepF, and lends further support to the conclusion that the organization of FtsZ protofilaments by SepF rings is important for correct cell division.

SepF polymerizes into very large ringsTo investigate in detail how SepF influences the polymerization of FtsZ, we decided to purify both proteins for biochemical analysis. SepF was purified using maltose-binding protein (MBP) as affinity tag. The MBP moiety was cleaved from SepF and removed by ion-exchange chromatography. When subjected to size exclusion chromatography, SepF eluted in the void volume, suggesting that the protein formed aggregates. To confirm this, we examined the protein sample by EM, yet the EM images revealed a very surprising result. It proved that SepF polymerizes into very large regular ring structures with an average diameter of about 50 nm (Figure 1). These rings are so wide that, theoretically, they can encompass two whole ribosomes. The formation of these rings seemed a robust process as they were readily formed at high salt concentrations (500 mM), and in the presence of other proteins (e.g., BSA). Although the buffer used in Figure 1 contains Mg2+ (see below), this is not required for ring formation. Rings were still visible at SepF concentrations as low as 0.1 μM, but their number was greatly reduced, and they were difficult to find on the EM grid. It was therefore not possible to determine the critical concentration at which rings were formed. The ring structures display a limited variability in ring diameter (Figure 1). This suggests that SepF filaments, presumptive precursors of the closed rings, are rather inflexible and always close on themselves.

SepF polymerizes into very large rings

To investigate in detail how SepF influences the polymerization of FtsZ, we decided to purify both proteins for biochemical analysis. SepF was purified using maltose-binding protein (MBP) as affinity tag. The MBP moiety was cleaved from SepF and removed by ion-exchange chromatography. When subjected to size exclusion chromatography, SepF eluted in the void volume, suggesting that the protein formed aggregates. To confirm this, we examined the protein sample by EM, yet the EM images revealed a very surprising result. It proved that SepF polymerizes into very large regular ring structures with an average diameter of about 50 nm (Figure 1). These rings are so wide that, theoretically, they can encompass two whole ribosomes. The formation of these rings seemed a robust process as they were readily formed at high salt concentrations (500 mM), and in the presence of other proteins (e.g., BSA). Although the buffer used in Figure 1 contains Mg2+ (see below), this is not required for ring formation. Rings were still visible at SepF concentrations as low as 0.1 μM, but their number was greatly reduced, and they were difficult to find on the EM grid. It was therefore not possible to determine the critical concentration at which rings were formed. The ring structures display a limited variability in ring diameter (Figure 1). This suggests that SepF filaments, presumptive precursors of the closed rings, are rather inflexible and always close on themselves.

SepF is an abundant proteinThe striking SepF rings begged the question what effect they would have on FtsZ polymerization. To analyze this under physiological conditions, we first determined the ratio of both proteins in the cell. Antibodies against purified SepF were raised and used to perform a quantitative western blot analysis. For exponentially growing cells, we estimated that the cellular concentrations of SepF and FtsZ were ∼6 and 8 μM, respectively (Supplementary Figure S1). This roughly correspond to 8000 molecules of SepF per cell, and 11 000 molecules for FtsZ, assuming a cell volume of 2.6 μm3 (Henriques et al, 1998). FtsZ is an abundant protein in the cell (Wang and Lutkenhaus, 1993), and the fact that the amount of SepF is almost as high as that of FtsZ suggests that the SepF rings have a structural role rather than a regulatory role in Z-ring formation.

SepF is an abundant protein

The striking SepF rings begged the question what effect they would have on FtsZ polymerization. To analyze this under physiological conditions, we first determined the ratio of both proteins in the cell. Antibodies against purified SepF were raised and used to perform a quantitative western blot analysis. For exponentially growing cells, we estimated that the cellular concentrations of SepF and FtsZ were ∼6 and 8 μM, respectively (Supplementary Figure S1). This roughly correspond to 8000 molecules of SepF per cell, and 11 000 molecules for FtsZ, assuming a cell volume of 2.6 μm3 (Henriques et al, 1998). FtsZ is an abundant protein in the cell (Wang and Lutkenhaus, 1993), and the fact that the amount of SepF is almost as high as that of FtsZ suggests that the SepF rings have a structural role rather than a regulatory role in Z-ring formation.

SepF-ring formation requires physiological reaction conditionsFtsZ polymerization can be followed by pelleting and light scattering assays (Mukherjee and Lutkenhaus, 1998; Scheffers, 2008). When purified SepF was included in an FtsZ polymerization experiment a small increase in the amount of pelleted FtsZ was observed, suggesting that SepF stimulates polymerization of FtsZ (Figure 2A, lanes 1 and 2). However, we noticed a problem with the solubility of SepF. FtsZ polymerization assays are generally performed at a relatively low pH of around pH 6.5, as this stimulates the lateral association of FtsZ protofilaments (Mukherjee and Lutkenhaus, 1999). It emerged that the low pH resulted in precipitation of SepF, and EM images of SepF in a buffer of pH 6.5 showed that protein rings were no longer present (Figure 1D). Precipitation of SepF was reduced by increasing the salt concentration (300 mM KCl) and using a buffer with a more physiological pH of 7.4 (Booth, 1985; Breeuwer et al, 1996) (Supplementary Figure S2B). This effect did not depend on the type of buffer used, and the same result was obtained with a Tris, HEPES or MES buffer (data not shown). At pH 7.4 and 300 mM salt, the total amount of pelleted FtsZ was substantially less, but the stimulatory effect of SepF on pelleting was again detectable (Figure 2A, lanes 3 and 4). In light scattering assays using standard polymerization buffer of pH 6.5, B. subtilis FtsZ gave a classic response upon the addition of GTP, with a strong increase in light scatter signal that decreased over time (Supplementary Figure S3) (Mukherjee and Lutkenhaus, 1999). The light scattering signal was strongly reduced by a shift to pH 7.4, and almost eliminated by increased salt concentrations (Supplementary Figure S3), indicating that lateral interactions between FtsZ protofilaments are greatly diminished under these conditions. The addition of SepF did result in a shift in light scattering signal (Figure 2B) but not the gradual rise that is normally associated with bundling of FtsZ protofilaments. SepF had no measurable effect on the GTPase activity of FtsZ (Figure 2C).

SepF-ring formation requires physiological reaction conditions

FtsZ polymerization can be followed by pelleting and light scattering assays (Mukherjee and Lutkenhaus, 1998; Scheffers, 2008). When purified SepF was included in an FtsZ polymerization experiment a small increase in the amount of pelleted FtsZ was observed, suggesting that SepF stimulates polymerization of FtsZ (Figure 2A, lanes 1 and 2). However, we noticed a problem with the solubility of SepF. FtsZ polymerization assays are generally performed at a relatively low pH of around pH 6.5, as this stimulates the lateral association of FtsZ protofilaments (Mukherjee and Lutkenhaus, 1999). It emerged that the low pH resulted in precipitation of SepF, and EM images of SepF in a buffer of pH 6.5 showed that protein rings were no longer present (Figure 1D). Precipitation of SepF was reduced by increasing the salt concentration (300 mM KCl) and using a buffer with a more physiological pH of 7.4 (Booth, 1985; Breeuwer et al, 1996) (Supplementary Figure S2B). This effect did not depend on the type of buffer used, and the same result was obtained with a Tris, HEPES or MES buffer (data not shown). At pH 7.4 and 300 mM salt, the total amount of pelleted FtsZ was substantially less, but the stimulatory effect of SepF on pelleting was again detectable (Figure 2A, lanes 3 and 4). In light scattering assays using standard polymerization buffer of pH 6.5, B. subtilis FtsZ gave a classic response upon the addition of GTP, with a strong increase in light scatter signal that decreased over time (Supplementary Figure S3) (Mukherjee and Lutkenhaus, 1999). The light scattering signal was strongly reduced by a shift to pH 7.4, and almost eliminated by increased salt concentrations (Supplementary Figure S3), indicating that lateral interactions between FtsZ protofilaments are greatly diminished under these conditions. The addition of SepF did result in a shift in light scattering signal (Figure 2B) but not the gradual rise that is normally associated with bundling of FtsZ protofilaments. SepF had no measurable effect on the GTPase activity of FtsZ (Figure 2C).

SepF bundles FtsZ into tubulesWe returned to EM to investigate whether the SepF rings had any effect on FtsZ filaments. In a buffer of pH 7.4 with 300 mM KCl, FtsZ forms clear protofilaments when GTP is present (Figure 3A). The addition of SepF had a remarkable result and long regular tubular structures became visible (Figure 3B). The average diameter of these structures was about 48 nm (Figure 3C), close to the diameter of SepF rings. Tubule formation was not influenced by the order in which the proteins or GTP were added, but depended on the concentration of FtsZ. When half the amount of FtsZ was used (5 μM) less tubules were detected. Increasing the SepF concentration did not compensate for this. At half the SepF concentration (3 μM) tubules were still visible, but at 1 μM SepF and 10 μM FtsZ no tubules could be identified on the EM grids. When GDP instead of GTP was used no tubules were formed. We never observed tubules in polymerization buffer of pH 6.5, which may explain why they were not observed in a previous study (Singh et al, 2008). We used 10 mM Mg2+ in the polymerization buffers, which is commonly used for FtsZ polymerization studies, but 1 mM Mg2+ was sufficient to see tubules. Without Mg2+ no tubules were formed. At higher magnifications the tubules appear to consist of straight longitudinal filaments, most likely FtsZ polymers (Figure 3D, narrow arrows), with evident transverse bands that presumably represent SepF rings (Figure 3D, wide arrows). From the EM images it seems that the SepF rings form the core of the FtsZ–SepF tubules, but attempts to resolve this question using thin section EM or Cryo-EM were unsuccessful. The length of the tubules increased with time and tubules could grow up to micrometres in length (Figure 3E; Supplementary Figure S4). Interestingly, during the first 5 min of the reaction filaments emanating from the ends of the short tubules tended to be straight (Figure 4A and B), whereas after 10 min, some tubules had splayed tips with highly curved filaments (Figure 4C–E). In the later samples (20 min), tubules with different kind of bifurcations, including branches and rings, were evident (Figures 3E and 4F–I).

SepF bundles FtsZ into tubules

We returned to EM to investigate whether the SepF rings had any effect on FtsZ filaments. In a buffer of pH 7.4 with 300 mM KCl, FtsZ forms clear protofilaments when GTP is present (Figure 3A). The addition of SepF had a remarkable result and long regular tubular structures became visible (Figure 3B). The average diameter of these structures was about 48 nm (Figure 3C), close to the diameter of SepF rings. Tubule formation was not influenced by the order in which the proteins or GTP were added, but depended on the concentration of FtsZ. When half the amount of FtsZ was used (5 μM) less tubules were detected. Increasing the SepF concentration did not compensate for this. At half the SepF concentration (3 μM) tubules were still visible, but at 1 μM SepF and 10 μM FtsZ no tubules could be identified on the EM grids. When GDP instead of GTP was used no tubules were formed. We never observed tubules in polymerization buffer of pH 6.5, which may explain why they were not observed in a previous study (Singh et al, 2008). We used 10 mM Mg2+ in the polymerization buffers, which is commonly used for FtsZ polymerization studies, but 1 mM Mg2+ was sufficient to see tubules. Without Mg2+ no tubules were formed. At higher magnifications the tubules appear to consist of straight longitudinal filaments, most likely FtsZ polymers (Figure 3D, narrow arrows), with evident transverse bands that presumably represent SepF rings (Figure 3D, wide arrows). From the EM images it seems that the SepF rings form the core of the FtsZ–SepF tubules, but attempts to resolve this question using thin section EM or Cryo-EM were unsuccessful. The length of the tubules increased with time and tubules could grow up to micrometres in length (Figure 3E; Supplementary Figure S4). Interestingly, during the first 5 min of the reaction filaments emanating from the ends of the short tubules tended to be straight (Figure 4A and B), whereas after 10 min, some tubules had splayed tips with highly curved filaments (Figure 4C–E). In the later samples (20 min), tubules with different kind of bifurcations, including branches and rings, were evident (Figures 3E and 4F–I).

Screen for SepF mutations that inhibit cell divisionAlthough the formation of FtsZ–SepF tubules appeared a reproducible and robust process in vitro, it was important to test its physiological relevance. We investigated whether it was possible to isolate dominant negative mutations in SepF that would affect the interaction of SepF with FtsZ and so disrupt tubule formation. The screen took advantage of the fact that a combination of sepF and ftsA null mutations is lethal (Ishikawa et al, 2006). Randomly mutagenized alleles of sepF were expressed from a xylose inducible Pxyl-promoter in cells carrying a wild-type sepF gene and a deletion of ftsA. After testing several thousands of transformants two clones were identified that were unable to grow when the sepF mutant allele was overexpressed by the addition of xylose to the growth medium (Figure 5A). Sequencing of the mutated sepF alleles revealed two different mutations affecting conserved residues: A98V and F124S (Figure 6). When these SepF mutants were (over)expressed in a strain devoid of wild-type SepF, cells remained elongated, and there was no indication that the mutant proteins could compensate for the absence of wild-type protein. To test whether the dominant negative sepF mutants affected Z-ring formation, we examined the localization of a gfp–ftsZ fusion. As shown in Figure 5B (upper panel), the overexpression of wild-type SepF in an ftsA mutant background had no effect on Z-ring formation. However, when the mutants A98V or F124S were induced (Figure 5B, lower panels), Z-ring formation was abolished, indicating that the mutant proteins inhibit cell division by interfering with Z-ring formation.

Screen for SepF mutations that inhibit cell division

Although the formation of FtsZ–SepF tubules appeared a reproducible and robust process in vitro, it was important to test its physiological relevance. We investigated whether it was possible to isolate dominant negative mutations in SepF that would affect the interaction of SepF with FtsZ and so disrupt tubule formation. The screen took advantage of the fact that a combination of sepF and ftsA null mutations is lethal (Ishikawa et al, 2006). Randomly mutagenized alleles of sepF were expressed from a xylose inducible Pxyl-promoter in cells carrying a wild-type sepF gene and a deletion of ftsA. After testing several thousands of transformants two clones were identified that were unable to grow when the sepF mutant allele was overexpressed by the addition of xylose to the growth medium (Figure 5A). Sequencing of the mutated sepF alleles revealed two different mutations affecting conserved residues: A98V and F124S (Figure 6). When these SepF mutants were (over)expressed in a strain devoid of wild-type SepF, cells remained elongated, and there was no indication that the mutant proteins could compensate for the absence of wild-type protein. To test whether the dominant negative sepF mutants affected Z-ring formation, we examined the localization of a gfp–ftsZ fusion. As shown in Figure 5B (upper panel), the overexpression of wild-type SepF in an ftsA mutant background had no effect on Z-ring formation. However, when the mutants A98V or F124S were induced (Figure 5B, lower panels), Z-ring formation was abolished, indicating that the mutant proteins inhibit cell division by interfering with Z-ring formation.

Dominant negative SepF mutants do not form tubulesThe inability of the SepF mutants to support growth could be a consequence of mutated FtsZ-binding sites or possibly a failure to form ring structures. To test this, the mutant SepF proteins were purified and first analyzed by size exclusion chromatography. Wild-type SepF showed a large peak in the void volume (>2 MDa), probably corresponding to SepF rings (Figure 7A). The F124S mutant showed a comparable elution profile to the wild-type protein. However, the A98V mutant was almost absent from the void volume, suggesting that this mutant affects ring formation. This protein showed an increased elution at around 14 ml between the peaks for the reference proteins thyroglobulin (669 kDa) and aldolase (158 kDa). Thus, the A98V mutation still gives oligomeric structures and does not reduce the protein to its monomeric state of 17 kDa. EM analysis showed normal ring formation for the F124S mutant (Figure 7B). In case of the A98V mutant it was difficult to find rings on the EM grids, indicating that this mutant protein is indeed disturbed in ring formation.We then examined the ability of the mutant proteins to support the formation of FtsZ tubules. Importantly, neither of the mutant proteins formed any detectable tubular structures under conditions in which wild-type SepF supported abundant tubulation (Supplementary Figure S5). As the F124S protein still forms rings, it is possible that this mutant is unable to interact with FtsZ. To test this, we performed an FtsZ–SepF co-elution experiment using MBP–SepF fusions. MBP–SepF, when bound to amylose resin, is capable of selectively binding FtsZ from an extract of Escherichia coli cells expressing B. subtilis FtsZ (Figures 7C and 8A). MBP–SepF mutant fusions, bound to amylose resin, retained significantly less FtsZ compared with wild-type protein. In fact, the amount of FtsZ that interacted with these mutant proteins was only detectable by western blot analysis. Thus, both A98V and F124S mutations impair the interaction with FtsZ protein. GFP fusions with these SepF mutants showed diffuse fluorescence in cells, in agreement with a loss of FtsZ-binding activity (data not shown).The F124S mutant largely retained the ability to form rings, and it is likely that its phenotype is manifested mainly through impaired interaction with FtsZ. Possibly, this protein interferes with wild-type SepF by creating hybrid rings that are less efficient in bundling FtsZ protofilaments. The A98V mutant could have a similar effect. We have tried to visualize hybrid rings by employing SepF fusion proteins, such as an N-terminal MBP–SepF fusion or a C-terminal SepF–Intein fusion, but both fusion proteins were unable to form rings by themselves. This could also explain why GFP fusions of SepF are not active in vivo. Indeed, when we examined purified SepF–GFP by EM no protein rings were detected (data not shown).

Dominant negative SepF mutants do not form tubules

The inability of the SepF mutants to support growth could be a consequence of mutated FtsZ-binding sites or possibly a failure to form ring structures. To test this, the mutant SepF proteins were purified and first analyzed by size exclusion chromatography. Wild-type SepF showed a large peak in the void volume (>2 MDa), probably corresponding to SepF rings (Figure 7A). The F124S mutant showed a comparable elution profile to the wild-type protein. However, the A98V mutant was almost absent from the void volume, suggesting that this mutant affects ring formation. This protein showed an increased elution at around 14 ml between the peaks for the reference proteins thyroglobulin (669 kDa) and aldolase (158 kDa). Thus, the A98V mutation still gives oligomeric structures and does not reduce the protein to its monomeric state of 17 kDa. EM analysis showed normal ring formation for the F124S mutant (Figure 7B). In case of the A98V mutant it was difficult to find rings on the EM grids, indicating that this mutant protein is indeed disturbed in ring formation.

We then examined the ability of the mutant proteins to support the formation of FtsZ tubules. Importantly, neither of the mutant proteins formed any detectable tubular structures under conditions in which wild-type SepF supported abundant tubulation (Supplementary Figure S5). As the F124S protein still forms rings, it is possible that this mutant is unable to interact with FtsZ. To test this, we performed an FtsZ–SepF co-elution experiment using MBP–SepF fusions. MBP–SepF, when bound to amylose resin, is capable of selectively binding FtsZ from an extract of Escherichia coli cells expressing B. subtilis FtsZ (Figures 7C and 8A). MBP–SepF mutant fusions, bound to amylose resin, retained significantly less FtsZ compared with wild-type protein. In fact, the amount of FtsZ that interacted with these mutant proteins was only detectable by western blot analysis. Thus, both A98V and F124S mutations impair the interaction with FtsZ protein. GFP fusions with these SepF mutants showed diffuse fluorescence in cells, in agreement with a loss of FtsZ-binding activity (data not shown).

The F124S mutant largely retained the ability to form rings, and it is likely that its phenotype is manifested mainly through impaired interaction with FtsZ. Possibly, this protein interferes with wild-type SepF by creating hybrid rings that are less efficient in bundling FtsZ protofilaments. The A98V mutant could have a similar effect. We have tried to visualize hybrid rings by employing SepF fusion proteins, such as an N-terminal MBP–SepF fusion or a C-terminal SepF–Intein fusion, but both fusion proteins were unable to form rings by themselves. This could also explain why GFP fusions of SepF are not active in vivo. Indeed, when we examined purified SepF–GFP by EM no protein rings were detected (data not shown).

Breaking the ring abolishes tubule formationThe above results suggest that alignment of FtsZ protofilaments is a key activity of SepF. However, it is still unclear whether the formation of rings is important for this activity. The inability of a SepF–GFP fusion to form rings indicates that the C-terminus is probably important for ring formation. Indeed, removal of the putative α-helical domain at the C-terminus (mutant Δ134) abolishes formation of SepF rings and of SepF–FtsZ tubules, even though the truncated SepF can still bind FtsZ (Figure 8A, data not shown). The first residue in this 15 amino acid C-terminal domain is a conserved glycine (Figure 6). Mutation of this glycine to asparagine (G135N) had no consequences for SepF–FtsZ interaction as the mutant protein still bound FtsZ in vitro (Figure 8A), and localized to the Z-ring in vivo (Figure 8B). When we analyzed purified G135N mutant protein by EM no protein rings were detectable, instead the protein appeared to assemble into long filamentous structures (Figure 8C, left panel). Importantly, mixing the mutant protein with purified FtsZ did not result in tubular structures (Figure 8C, right panel). These data suggest that the C-terminus of SepF is required for the formation of rings, and that SepF rings are required for the alignment of FtsZ polymers. To test whether the sepF-ring mutant was active in vivo, we replaced wild-type sepF with sepF–G135N in a B. subtilis strain that contains ftsA under control of the IPTG-inducible Pspac promoter. When the resulting strain was grown on plates without IPTG, no colonies appeared (Figure 8D), despite the fact that expression levels of SepF–G135N were normal (Figure 8E). The mutant, when overexpressed in an ftsA deletion strain, showed no dominant negative effect like the SepF mutants A98V and F124S. The finding that SepF–G135N cannot support cell growth in the absence of FtsA indicates that SepF-ring formation is critical for the function of SepF, and lends further support to the conclusion that the organization of FtsZ protofilaments by SepF rings is important for correct cell division.

Breaking the ring abolishes tubule formation

The above results suggest that alignment of FtsZ protofilaments is a key activity of SepF. However, it is still unclear whether the formation of rings is important for this activity. The inability of a SepF–GFP fusion to form rings indicates that the C-terminus is probably important for ring formation. Indeed, removal of the putative α-helical domain at the C-terminus (mutant Δ134) abolishes formation of SepF rings and of SepF–FtsZ tubules, even though the truncated SepF can still bind FtsZ (Figure 8A, data not shown). The first residue in this 15 amino acid C-terminal domain is a conserved glycine (Figure 6). Mutation of this glycine to asparagine (G135N) had no consequences for SepF–FtsZ interaction as the mutant protein still bound FtsZ in vitro (Figure 8A), and localized to the Z-ring in vivo (Figure 8B). When we analyzed purified G135N mutant protein by EM no protein rings were detectable, instead the protein appeared to assemble into long filamentous structures (Figure 8C, left panel). Importantly, mixing the mutant protein with purified FtsZ did not result in tubular structures (Figure 8C, right panel). These data suggest that the C-terminus of SepF is required for the formation of rings, and that SepF rings are required for the alignment of FtsZ polymers. To test whether the sepF-ring mutant was active in vivo, we replaced wild-type sepF with sepF–G135N in a B. subtilis strain that contains ftsA under control of the IPTG-inducible Pspac promoter. When the resulting strain was grown on plates without IPTG, no colonies appeared (Figure 8D), despite the fact that expression levels of SepF–G135N were normal (Figure 8E). The mutant, when overexpressed in an ftsA deletion strain, showed no dominant negative effect like the SepF mutants A98V and F124S. The finding that SepF–G135N cannot support cell growth in the absence of FtsA indicates that SepF-ring formation is critical for the function of SepF, and lends further support to the conclusion that the organization of FtsZ protofilaments by SepF rings is important for correct cell division.

DiscussionThis is the first report of a protein polymer that directly supports bundling of FtsZ polymers. Even more surprising is the fact that this polymer closes into a ring. There are other cell division proteins that have been shown to promote bundling of FtsZ filaments, including ZapA of B. subtilis and E. coli (Gueiros-Filho and Losick, 2002; Small et al, 2007), and the E. coli protein ZipA (RayChaudhuri, 1999; Hale et al, 2000), but the FtsZ bundles formed by these proteins do not display the regular, highly organized structure seen for FtsZ–SepF tubules.The formation and shape of FtsZ–SepF tubules is reminiscent of eukaryotic microtubules, although with a width of about 25 nm (Desai and Mitchison, 1997), microtubules are considerably thinner than FtsZ–SepF tubules. The switch from polymerization to depolymerization in microtubules is accompanied by a transition from ends with straight protofilaments to ends with protofilaments curved outwards (Mandelkow et al, 1991; Desai and Mitchison, 1997). A similar phenomenon is observed with FtsZ–SepF tubules. During the first minutes of the reaction, filaments emanating from the ends of short tubules tended to be straight, but when the reaction progressed tubules were seen with splayed tips with filaments curving outwards (Figure 4). This is likely due to a shift from GTP to GDP bound FtsZ, as FtsZ protofilaments with bound GTP favour a straight conformation, whereas they tend to become curved when GTP is hydrolyzed to GDP (Lu et al, 2000). This curvature of FtsZ–SepF tubules did not result in depolymerization of the tubules, like the depolymerization phase (dynamic instability) of microtubules, and after prolonged incubation the tubules disintegrate over their entire length (data not shown). In yeast, the Dam1 kinetochore complex involved in chromosome segregation forms ring-like structures around microtubules (Wang et al, 2007). Interestingly, the diameter of these rings is about 50 nm, close to that of SepF rings. However, the Dam1 subunits are composed of 10 different proteins. We are unaware of any other protein that can make such large regular ring structures by itself.FtsZ is an abundant protein, yet only a third of the FtsZ molecules in B. subtilis cells is part of the Z-ring, and it has been estimated that the Z-ring can be 2–3 protofilaments thick when division initiates (Anderson et al, 2004). Considerably more filaments are visible in the FtsZ–SepF tubules (Figure 3D), so there seems to be insufficient FtsZ in the cell to make complete tubules when the Z-ring is formed. Possibly, at a later stage in division, when the diameter of the Z-ring is much smaller, a more tubular configuration appears. Although this is speculation, it is clear that SepF rings support the alignment of FtsZ filaments which, considering the malformed septa in sepF mutants (Hamoen et al, 2006), seems to be crucial for proper septum synthesis.The striking appearance of SepF rings raises the question whether there is a biological advantage in employing a protein that polymerizes into a circular configuration. Why would the cell not use linear protein polymers to organize the alignment of FtsZ protofilaments? A possible reason is depicted in Figure 9. Firstly, it is very difficult to control the length of a linear polymer (red rods), and when such polymer becomes exceedingly long the FtsZ polymers (grey rods) will be spaced too widely (Figure 9A). Polymerization into a ring prevents this polymer length issue (Figure 9B). However, even when the linear polymers are somehow of fixed length, it would still be difficult to control the width of the assembled super structure, as FtsZ protofilaments alignment would be able to continue sideways (arrows). This problem is also circumvented when the supporting polymer adopts a circular configuration. In Figure 9B SepF is depicted as rings. Importantly, in this model, even if SepF polymerization is incomplete, and shorter curved polymers are formed, binding of these SepF arcs would still result in a tubular alignment of FtsZ protofilaments.SepF is found only in Gram-positive bacteria and cyanobacteria, and not in Gram negatives. One clear difference between these groups of organisms lies in the thickness of their cell wall and the way division septa are formed. Gram-negative bacteria have a thin layer of peptidoglycan, which follows the constriction of the inner membrane during division. Gram-positive bacteria have a thick cell wall and synthesize a continuous thick septum along the division site. The absence of SepF results in septa which are unusually thick and often badly malformed (Figure 9C; Supplementary Figure S6). This seems to be characteristic for sepF mutants, and is not observed with other cell division mutants, including minC, and ezrA (Barak et al, 1998; Hamoen et al, 2006). We have also checked whether an ftsA deletion causes irregular septa in B. subtilis, and although the frequency of septation was clearly reduced, the septa looked normal (Supplementary Figure S6). We propose that SepF rings provide a mechanism for arranging FtsZ protofilaments into higher order structures of defined width, required for the proper synthesis of the thick division septa in Gram-positive bacteria and cyanobacteria.

This is the first report of a protein polymer that directly supports bundling of FtsZ polymers. Even more surprising is the fact that this polymer closes into a ring. There are other cell division proteins that have been shown to promote bundling of FtsZ filaments, including ZapA of B. subtilis and E. coli (Gueiros-Filho and Losick, 2002; Small et al, 2007), and the E. coli protein ZipA (RayChaudhuri, 1999; Hale et al, 2000), but the FtsZ bundles formed by these proteins do not display the regular, highly organized structure seen for FtsZ–SepF tubules.

The formation and shape of FtsZ–SepF tubules is reminiscent of eukaryotic microtubules, although with a width of about 25 nm (Desai and Mitchison, 1997), microtubules are considerably thinner than FtsZ–SepF tubules. The switch from polymerization to depolymerization in microtubules is accompanied by a transition from ends with straight protofilaments to ends with protofilaments curved outwards (Mandelkow et al, 1991; Desai and Mitchison, 1997). A similar phenomenon is observed with FtsZ–SepF tubules. During the first minutes of the reaction, filaments emanating from the ends of short tubules tended to be straight, but when the reaction progressed tubules were seen with splayed tips with filaments curving outwards (Figure 4). This is likely due to a shift from GTP to GDP bound FtsZ, as FtsZ protofilaments with bound GTP favour a straight conformation, whereas they tend to become curved when GTP is hydrolyzed to GDP (Lu et al, 2000). This curvature of FtsZ–SepF tubules did not result in depolymerization of the tubules, like the depolymerization phase (dynamic instability) of microtubules, and after prolonged incubation the tubules disintegrate over their entire length (data not shown). In yeast, the Dam1 kinetochore complex involved in chromosome segregation forms ring-like structures around microtubules (Wang et al, 2007). Interestingly, the diameter of these rings is about 50 nm, close to that of SepF rings. However, the Dam1 subunits are composed of 10 different proteins. We are unaware of any other protein that can make such large regular ring structures by itself.

FtsZ is an abundant protein, yet only a third of the FtsZ molecules in B. subtilis cells is part of the Z-ring, and it has been estimated that the Z-ring can be 2–3 protofilaments thick when division initiates (Anderson et al, 2004). Considerably more filaments are visible in the FtsZ–SepF tubules (Figure 3D), so there seems to be insufficient FtsZ in the cell to make complete tubules when the Z-ring is formed. Possibly, at a later stage in division, when the diameter of the Z-ring is much smaller, a more tubular configuration appears. Although this is speculation, it is clear that SepF rings support the alignment of FtsZ filaments which, considering the malformed septa in sepF mutants (Hamoen et al, 2006), seems to be crucial for proper septum synthesis.

The striking appearance of SepF rings raises the question whether there is a biological advantage in employing a protein that polymerizes into a circular configuration. Why would the cell not use linear protein polymers to organize the alignment of FtsZ protofilaments? A possible reason is depicted in Figure 9. Firstly, it is very difficult to control the length of a linear polymer (red rods), and when such polymer becomes exceedingly long the FtsZ polymers (grey rods) will be spaced too widely (Figure 9A). Polymerization into a ring prevents this polymer length issue (Figure 9B). However, even when the linear polymers are somehow of fixed length, it would still be difficult to control the width of the assembled super structure, as FtsZ protofilaments alignment would be able to continue sideways (arrows). This problem is also circumvented when the supporting polymer adopts a circular configuration. In Figure 9B SepF is depicted as rings. Importantly, in this model, even if SepF polymerization is incomplete, and shorter curved polymers are formed, binding of these SepF arcs would still result in a tubular alignment of FtsZ protofilaments.

SepF is found only in Gram-positive bacteria and cyanobacteria, and not in Gram negatives. One clear difference between these groups of organisms lies in the thickness of their cell wall and the way division septa are formed. Gram-negative bacteria have a thin layer of peptidoglycan, which follows the constriction of the inner membrane during division. Gram-positive bacteria have a thick cell wall and synthesize a continuous thick septum along the division site. The absence of SepF results in septa which are unusually thick and often badly malformed (Figure 9C; Supplementary Figure S6). This seems to be characteristic for sepF mutants, and is not observed with other cell division mutants, including minC, and ezrA (Barak et al, 1998; Hamoen et al, 2006). We have also checked whether an ftsA deletion causes irregular septa in B. subtilis, and although the frequency of septation was clearly reduced, the septa looked normal (Supplementary Figure S6). We propose that SepF rings provide a mechanism for arranging FtsZ protofilaments into higher order structures of defined width, required for the proper synthesis of the thick division septa in Gram-positive bacteria and cyanobacteria.

Materials and methodsBacterial strains, growth conditions, and mediaThe B. subtilis strains and primers used in this study are listed in Supplementary Tables S1 and S2. Bacillus strains were grown at 30 and 37°C in LB medium, Difco antibiotic medium 3 (PAB), or casein hydrolysate (CH) medium (Sterlini and Mandelstam, 1969) supplemented with adenine and guanosine (required for growth of CRK6000; 20 μg/ml each). Bacillus transformants were selected on PAB plates supplemented with 50 μg/ml spectinomycin, 5 μg/ml chloramphenicol, 1 μg/ml erythromycin, and/or 25 μg/ml lincomycin.Protein purificationB. subtilis sepF was cloned into pMal-c2X (New England Biolabs), using restriction free cloning (van den Ent and Lowe, 2006). The Xa-cleavage site was positioned immediately upstream of SepF, so that native SepF, with no additional residues, remains after cleavage. Overnight cultures of E. coli BL21 (DE3) carrying pMal–SepF were diluted 1:100 in fresh 2 × TY medium. Cultures were grown to an OD600 of 0.4, and MBP–SepF expression was induced with IPTG for 3 h. Cell lysates were obtained using a French Press. In total, 1 mM PMSF and 7.5 unit/ml Benzonase (New England Biolabs) were added, and cell debris was removed by centrifugation. The total protein fraction was loaded onto an amylose column, and MBP–SepF was eluted with elution buffer (20 mM Tris–HCl pH 7.4, 200 mM NaCl, 0.5 mM DTT, and 10 mM maltose). MBP was cleaved from SepF with Factor Xa (1 μg per 250 μg fusion protein) overnight at 4°C in elution buffer containing 2 mM CaCl2. The final purification was performed on an MonoQ column followed by a protein concentration step using Millipore Amicon Ultra protein concentration tubes with a 10-kDa cutoff. Small aliquots were frozen in liquid N2.B. subtilis FtsZ was purified as described previously (Wang and Lutkenhaus, 1993; Scheffers, 2008).Size exclusion chromatographyPurified SepF (70 μg in 200 μl) was loaded onto a Superose 6 10/300 column (GE Healthcare), pre-equilibrated with buffer containing 20 mM Tris–HCl pH 7.4, 250 mM KCl, and 1 mM EDTA. Samples were eluted with 25 ml buffer at a flow rate of 0.3 ml/min. Elution was monitored at 280 nm. The experiments were repeated three times for each SepF variant, and a representative chromatogram is shown. The Superose 6 column was calibrated with thyroglobulin (669 kDa), aldolase (158 kDa), and RNaseA (13.7 kDa) as molecular standards (HMV Gel Filtration Calibration Kit, Amersham Pharmacia).Intracellular concentration of SepF and FtsZThe intracellular concentrations of SepF and FtsZ were determined using quantitative western blot analyses as previously described (Ishikawa et al, 2006). We raised rabbit polyclonal antiserum against purified SepF. B. subtilis cells were grown in LB medium at 37°C, and at OD600 of 0.3 a 10-ml sample was taken. After centrifugation, the cells were broken by sonication, mixed with SDS sample buffer, and after heating, loaded onto a SDS–PAA gel for western blot analysis. Titrations of purified SepF and FtsZ were loaded onto the same gel (Supplementary Figure S1). The intensities of the bands were quantified using ImageJ software. Molecules per cell were determined based on CFU of the cultures (2.39E+07/ml), and assuming a cell volume of 2.6 μm3 (Henriques et al, 1998).Sedimentation assaysFtsZ sedimentation assays were performed as previously described (Mukherjee and Lutkenhaus, 1998; Scheffers, 2008). Buffers used were standard pH 6.5 polymerization buffer (50 mM MES–NaOH pH 6.5, 50 mM KCl, 10 mM MgCl2), and polymerization buffer of pH 7.4 (50 mM Tris–HCl pH 7.4, 300 mM KCl, 10 mM MgCl2). FtsZ (10 μM) and SepF (6 μM) were incubated for 5 min at room temperature in 45 μl polymerization buffer. FtsZ polymerization was initiated by the addition of 1 mM GTP and samples were incubated for 5–20 min at 30°C. High molecular weight assemblies were spun down (10 min, 80 000 r.p.m., 25°C) in a Beckman ultracentrifuge with a TLA-100 rotor. Supernatant and pellet fractions were separated by SDS–PAA gels followed by colloidal Coomassie staining. Band intensities were analyzed by ImageJ. The increase in pelleting was determined by subtracting the amount of pelleted protein in fractions without GTP from fractions that contained GTP (to stimulate FtsZ polymerization).Light scattering assaysLight scattering assays were performed as previously described (Mukherjee and Lutkenhaus, 1999; Scheffers, 2008), using a Varian Cary Eclipse fluorescence spectrophotometer, and the same reaction conditions as described for the sedimentation assays. In all, 1.5 or 2.5 nm excitation and emission slit values were used.GTP hydrolysis assaysGTPase assays were performed as previously described (Lanzetta et al, 1979; Scheffers, 2008), using Malachite green to detect the released phosphate. The reaction conditions were the same as used for the sedimentation assays.Electron microscopyFtsZ polymerization was performed as described above, and SepF protein samples were prepared in polymerization buffers (10 μM FtsZ and 6 μM SepF). In all, 2 μl samples were applied to glow-discharged 200 mesh carbon coated grids. Grids were negatively stained, using 100 μl uranyl-acetate (2%) and imaged in a Philips CM100 electron microscope.For transmission electron microscopy, cultures were grown in CH medium at 37°C to mid-exponential phase and fixed by the addition of gluteraldehyde to a final concentration of 2%. Cells were then pelleted and fixation was continued overnight at 4°C. Pellets were washed 3 times with 100 mM phosphate buffer for 15 min, post-fixed with 1% osmium tetroxide in 100 mM phosphate buffer, and incubated for 1 h at 4°C. Pellets were then washed in phosphate buffer, dehydrated in acetone, and embedded in resin that was allowed to polymerize overnight at 65°C. Sections of 80 nm were cut using an ultracut microtome (Reichert and Jung), negatively stained with 2.5% uranyl-acetate and examined with a Philips CM-100 electron microscope.Library constructionFor PCR mutagenesis, the sepF gene containing the Shine–Dalgarno sequence was amplified by PCR with TaKaRa Ex Taq™ (Takara), using SDYlmFF and ylmFgwR (Ishikawa et al, 2006). The reaction was carried out in the reaction mixture containing 0.1 mM MnCl2 (Cromie et al, 1999). Plasmid library was constructed using the Gateway cloning technology (Invitrogen). The pX-GW plasmid (K Hiramatsu and S Ishikawa, unpublished) was employed to insert the mutagenized sepF fragments, under control of the xylose-inducible promoter Pxyl, into the amyE locus of B. subtilis.Construction of SepF mutantsFor the GFP or MBP-fusions, the A98V, F124S, G135N, and Δ134 mutations were generated using the Quickchange site-directed mutagenesis method (Stratagene). Briefly, plasmid pSepF–GFP (Hamoen et al, 2006) and pMal–SepF were used as templates for temperature-cycled amplification with primers designed to introduce mutations into these codons. Amplification was carried out with Pfu polymerase (Strategene) and primers (Supplementary Table S2, position of mutation underlined). The product was treated with DpnI to digest unamplified DNA, and transformed into competent DH5α cells. The resulting plasmids with the desired mutations were verified by sequence analysis. To introduce sepF–G135N into B. subtilis, the C-terminal part of sepF was amplified by PCR using pMal–sepFG135N and pMal–sepF (for wild-type control) as template and primers LH224 and LH225 (Supplementary Table S2). The PCR products were digested with EcoRI and BamHI and cloned into the Campbell-integration vector pMutin4 (Vagner et al, 1998), creating pM–sepFG135 and pM–sepF.The SepF mutations were introduced into different B. subtilis backgrounds by transformation of competent cells as described by Moriya et al (1998). The resulting strains are listed in Supplementary Table S1.Microscopic imagingFor fluorescence microscopy, overnight cultures were diluted into fresh PAB medium (for GFP–FtsZ) or CH medium (for SepF–GFP), and grown to exponential phase at 30°C. Cells were mounted on microscope slides covered with a thin film of 1.2% agarose in water. Images were acquired with a Sony Cool-Snap HQ cooled CCD camera (Roper Scientific) attached to a Zeiss Axiovert 200M microscope. The images were analyzed with METAMORPH version 6 software.FtsZ co-elutionCell pellets from 30 ml E. coli cultures expressing MBP–SepF mutants were resuspended in 4 ml of buffer A (20 mM Tris–HCl pH 7.4, 200 mM KCl, 1 mM EDTA) with 1 mM PMSF. The suspension was then sonicated (10 W, 4 s pulses) for a total period of 4 min. The resulting suspension was centrifuged at 13 000 r.p.m. for 30 min to remove cell debris. Supernatants containing the different MBP–SepF fusion proteins were added to 1.0 ml of amylose resins (BioLabs) pre-equilibrated with buffer A, and the suspensions were gently mixed at 4°C for 30 min. After incubation, the resins were allowed to settle down by gravity flow and washed six times with 2 ml of buffer A. After the last wash step the resin was mixed with 0.5 ml buffer A and one third of the suspension (0.5 ml) was transferred to empty column, to determine the amount of bound MBP–SepF. MBP–SepF was eluted with 2 × 0.5 ml elution buffer (buffer A containing 10 mM maltose and 1% SDS, 10 min incubation), and the protein concentration was determined with BCA Protein Assay kit (Thermo Scientific Pierce). For the FtsZ co-elution experiments the resin volume was adjusted so that equal amounts of MBP–SepF were used. The MBP–SepF containing amylose resin was loaded in empty columns and 4 ml of cell extract from E. coli cells expressing B. subtilis FtsZ was added and incubated for 30 min at 4°C. After incubation, the columns were washed six times with buffer A before elution with 1.5 ml elution buffer. The flow through, first wash step, second wash step, last wash step, and elution fractions were analyzed by SDS–PAGE. When necessary, the presence of FtsZ in the elution fraction was detected with western blotting using FtsZ antiserum.

Materials and methods

Bacterial strains, growth conditions, and mediaThe B. subtilis strains and primers used in this study are listed in Supplementary Tables S1 and S2. Bacillus strains were grown at 30 and 37°C in LB medium, Difco antibiotic medium 3 (PAB), or casein hydrolysate (CH) medium (Sterlini and Mandelstam, 1969) supplemented with adenine and guanosine (required for growth of CRK6000; 20 μg/ml each). Bacillus transformants were selected on PAB plates supplemented with 50 μg/ml spectinomycin, 5 μg/ml chloramphenicol, 1 μg/ml erythromycin, and/or 25 μg/ml lincomycin.

Bacterial strains, growth conditions, and media

The B. subtilis strains and primers used in this study are listed in Supplementary Tables S1 and S2. Bacillus strains were grown at 30 and 37°C in LB medium, Difco antibiotic medium 3 (PAB), or casein hydrolysate (CH) medium (Sterlini and Mandelstam, 1969) supplemented with adenine and guanosine (required for growth of CRK6000; 20 μg/ml each). Bacillus transformants were selected on PAB plates supplemented with 50 μg/ml spectinomycin, 5 μg/ml chloramphenicol, 1 μg/ml erythromycin, and/or 25 μg/ml lincomycin.

Protein purificationB. subtilis sepF was cloned into pMal-c2X (New England Biolabs), using restriction free cloning (van den Ent and Lowe, 2006). The Xa-cleavage site was positioned immediately upstream of SepF, so that native SepF, with no additional residues, remains after cleavage. Overnight cultures of E. coli BL21 (DE3) carrying pMal–SepF were diluted 1:100 in fresh 2 × TY medium. Cultures were grown to an OD600 of 0.4, and MBP–SepF expression was induced with IPTG for 3 h. Cell lysates were obtained using a French Press. In total, 1 mM PMSF and 7.5 unit/ml Benzonase (New England Biolabs) were added, and cell debris was removed by centrifugation. The total protein fraction was loaded onto an amylose column, and MBP–SepF was eluted with elution buffer (20 mM Tris–HCl pH 7.4, 200 mM NaCl, 0.5 mM DTT, and 10 mM maltose). MBP was cleaved from SepF with Factor Xa (1 μg per 250 μg fusion protein) overnight at 4°C in elution buffer containing 2 mM CaCl2. The final purification was performed on an MonoQ column followed by a protein concentration step using Millipore Amicon Ultra protein concentration tubes with a 10-kDa cutoff. Small aliquots were frozen in liquid N2.B. subtilis FtsZ was purified as described previously (Wang and Lutkenhaus, 1993; Scheffers, 2008).

Protein purification

B. subtilis sepF was cloned into pMal-c2X (New England Biolabs), using restriction free cloning (van den Ent and Lowe, 2006). The Xa-cleavage site was positioned immediately upstream of SepF, so that native SepF, with no additional residues, remains after cleavage. Overnight cultures of E. coli BL21 (DE3) carrying pMal–SepF were diluted 1:100 in fresh 2 × TY medium. Cultures were grown to an OD600 of 0.4, and MBP–SepF expression was induced with IPTG for 3 h. Cell lysates were obtained using a French Press. In total, 1 mM PMSF and 7.5 unit/ml Benzonase (New England Biolabs) were added, and cell debris was removed by centrifugation. The total protein fraction was loaded onto an amylose column, and MBP–SepF was eluted with elution buffer (20 mM Tris–HCl pH 7.4, 200 mM NaCl, 0.5 mM DTT, and 10 mM maltose). MBP was cleaved from SepF with Factor Xa (1 μg per 250 μg fusion protein) overnight at 4°C in elution buffer containing 2 mM CaCl2. The final purification was performed on an MonoQ column followed by a protein concentration step using Millipore Amicon Ultra protein concentration tubes with a 10-kDa cutoff. Small aliquots were frozen in liquid N2.

B. subtilis FtsZ was purified as described previously (Wang and Lutkenhaus, 1993; Scheffers, 2008).

Size exclusion chromatographyPurified SepF (70 μg in 200 μl) was loaded onto a Superose 6 10/300 column (GE Healthcare), pre-equilibrated with buffer containing 20 mM Tris–HCl pH 7.4, 250 mM KCl, and 1 mM EDTA. Samples were eluted with 25 ml buffer at a flow rate of 0.3 ml/min. Elution was monitored at 280 nm. The experiments were repeated three times for each SepF variant, and a representative chromatogram is shown. The Superose 6 column was calibrated with thyroglobulin (669 kDa), aldolase (158 kDa), and RNaseA (13.7 kDa) as molecular standards (HMV Gel Filtration Calibration Kit, Amersham Pharmacia).

Size exclusion chromatography

Purified SepF (70 μg in 200 μl) was loaded onto a Superose 6 10/300 column (GE Healthcare), pre-equilibrated with buffer containing 20 mM Tris–HCl pH 7.4, 250 mM KCl, and 1 mM EDTA. Samples were eluted with 25 ml buffer at a flow rate of 0.3 ml/min. Elution was monitored at 280 nm. The experiments were repeated three times for each SepF variant, and a representative chromatogram is shown. The Superose 6 column was calibrated with thyroglobulin (669 kDa), aldolase (158 kDa), and RNaseA (13.7 kDa) as molecular standards (HMV Gel Filtration Calibration Kit, Amersham Pharmacia).

Intracellular concentration of SepF and FtsZThe intracellular concentrations of SepF and FtsZ were determined using quantitative western blot analyses as previously described (Ishikawa et al, 2006). We raised rabbit polyclonal antiserum against purified SepF. B. subtilis cells were grown in LB medium at 37°C, and at OD600 of 0.3 a 10-ml sample was taken. After centrifugation, the cells were broken by sonication, mixed with SDS sample buffer, and after heating, loaded onto a SDS–PAA gel for western blot analysis. Titrations of purified SepF and FtsZ were loaded onto the same gel (Supplementary Figure S1). The intensities of the bands were quantified using ImageJ software. Molecules per cell were determined based on CFU of the cultures (2.39E+07/ml), and assuming a cell volume of 2.6 μm3 (Henriques et al, 1998).

Intracellular concentration of SepF and FtsZ

The intracellular concentrations of SepF and FtsZ were determined using quantitative western blot analyses as previously described (Ishikawa et al, 2006). We raised rabbit polyclonal antiserum against purified SepF. B. subtilis cells were grown in LB medium at 37°C, and at OD600 of 0.3 a 10-ml sample was taken. After centrifugation, the cells were broken by sonication, mixed with SDS sample buffer, and after heating, loaded onto a SDS–PAA gel for western blot analysis. Titrations of purified SepF and FtsZ were loaded onto the same gel (Supplementary Figure S1). The intensities of the bands were quantified using ImageJ software. Molecules per cell were determined based on CFU of the cultures (2.39E+07/ml), and assuming a cell volume of 2.6 μm3 (Henriques et al, 1998).

Sedimentation assaysFtsZ sedimentation assays were performed as previously described (Mukherjee and Lutkenhaus, 1998; Scheffers, 2008). Buffers used were standard pH 6.5 polymerization buffer (50 mM MES–NaOH pH 6.5, 50 mM KCl, 10 mM MgCl2), and polymerization buffer of pH 7.4 (50 mM Tris–HCl pH 7.4, 300 mM KCl, 10 mM MgCl2). FtsZ (10 μM) and SepF (6 μM) were incubated for 5 min at room temperature in 45 μl polymerization buffer. FtsZ polymerization was initiated by the addition of 1 mM GTP and samples were incubated for 5–20 min at 30°C. High molecular weight assemblies were spun down (10 min, 80 000 r.p.m., 25°C) in a Beckman ultracentrifuge with a TLA-100 rotor. Supernatant and pellet fractions were separated by SDS–PAA gels followed by colloidal Coomassie staining. Band intensities were analyzed by ImageJ. The increase in pelleting was determined by subtracting the amount of pelleted protein in fractions without GTP from fractions that contained GTP (to stimulate FtsZ polymerization).

Sedimentation assays

FtsZ sedimentation assays were performed as previously described (Mukherjee and Lutkenhaus, 1998; Scheffers, 2008). Buffers used were standard pH 6.5 polymerization buffer (50 mM MES–NaOH pH 6.5, 50 mM KCl, 10 mM MgCl2), and polymerization buffer of pH 7.4 (50 mM Tris–HCl pH 7.4, 300 mM KCl, 10 mM MgCl2). FtsZ (10 μM) and SepF (6 μM) were incubated for 5 min at room temperature in 45 μl polymerization buffer. FtsZ polymerization was initiated by the addition of 1 mM GTP and samples were incubated for 5–20 min at 30°C. High molecular weight assemblies were spun down (10 min, 80 000 r.p.m., 25°C) in a Beckman ultracentrifuge with a TLA-100 rotor. Supernatant and pellet fractions were separated by SDS–PAA gels followed by colloidal Coomassie staining. Band intensities were analyzed by ImageJ. The increase in pelleting was determined by subtracting the amount of pelleted protein in fractions without GTP from fractions that contained GTP (to stimulate FtsZ polymerization).

Light scattering assaysLight scattering assays were performed as previously described (Mukherjee and Lutkenhaus, 1999; Scheffers, 2008), using a Varian Cary Eclipse fluorescence spectrophotometer, and the same reaction conditions as described for the sedimentation assays. In all, 1.5 or 2.5 nm excitation and emission slit values were used.

Light scattering assays

Light scattering assays were performed as previously described (Mukherjee and Lutkenhaus, 1999; Scheffers, 2008), using a Varian Cary Eclipse fluorescence spectrophotometer, and the same reaction conditions as described for the sedimentation assays. In all, 1.5 or 2.5 nm excitation and emission slit values were used.

GTP hydrolysis assaysGTPase assays were performed as previously described (Lanzetta et al, 1979; Scheffers, 2008), using Malachite green to detect the released phosphate. The reaction conditions were the same as used for the sedimentation assays.

GTP hydrolysis assays

GTPase assays were performed as previously described (Lanzetta et al, 1979; Scheffers, 2008), using Malachite green to detect the released phosphate. The reaction conditions were the same as used for the sedimentation assays.

Electron microscopyFtsZ polymerization was performed as described above, and SepF protein samples were prepared in polymerization buffers (10 μM FtsZ and 6 μM SepF). In all, 2 μl samples were applied to glow-discharged 200 mesh carbon coated grids. Grids were negatively stained, using 100 μl uranyl-acetate (2%) and imaged in a Philips CM100 electron microscope.For transmission electron microscopy, cultures were grown in CH medium at 37°C to mid-exponential phase and fixed by the addition of gluteraldehyde to a final concentration of 2%. Cells were then pelleted and fixation was continued overnight at 4°C. Pellets were washed 3 times with 100 mM phosphate buffer for 15 min, post-fixed with 1% osmium tetroxide in 100 mM phosphate buffer, and incubated for 1 h at 4°C. Pellets were then washed in phosphate buffer, dehydrated in acetone, and embedded in resin that was allowed to polymerize overnight at 65°C. Sections of 80 nm were cut using an ultracut microtome (Reichert and Jung), negatively stained with 2.5% uranyl-acetate and examined with a Philips CM-100 electron microscope.

Electron microscopy

FtsZ polymerization was performed as described above, and SepF protein samples were prepared in polymerization buffers (10 μM FtsZ and 6 μM SepF). In all, 2 μl samples were applied to glow-discharged 200 mesh carbon coated grids. Grids were negatively stained, using 100 μl uranyl-acetate (2%) and imaged in a Philips CM100 electron microscope.

For transmission electron microscopy, cultures were grown in CH medium at 37°C to mid-exponential phase and fixed by the addition of gluteraldehyde to a final concentration of 2%. Cells were then pelleted and fixation was continued overnight at 4°C. Pellets were washed 3 times with 100 mM phosphate buffer for 15 min, post-fixed with 1% osmium tetroxide in 100 mM phosphate buffer, and incubated for 1 h at 4°C. Pellets were then washed in phosphate buffer, dehydrated in acetone, and embedded in resin that was allowed to polymerize overnight at 65°C. Sections of 80 nm were cut using an ultracut microtome (Reichert and Jung), negatively stained with 2.5% uranyl-acetate and examined with a Philips CM-100 electron microscope.

Library constructionFor PCR mutagenesis, the sepF gene containing the Shine–Dalgarno sequence was amplified by PCR with TaKaRa Ex Taq™ (Takara), using SDYlmFF and ylmFgwR (Ishikawa et al, 2006). The reaction was carried out in the reaction mixture containing 0.1 mM MnCl2 (Cromie et al, 1999). Plasmid library was constructed using the Gateway cloning technology (Invitrogen). The pX-GW plasmid (K Hiramatsu and S Ishikawa, unpublished) was employed to insert the mutagenized sepF fragments, under control of the xylose-inducible promoter Pxyl, into the amyE locus of B. subtilis.

Library construction

For PCR mutagenesis, the sepF gene containing the Shine–Dalgarno sequence was amplified by PCR with TaKaRa Ex Taq™ (Takara), using SDYlmFF and ylmFgwR (Ishikawa et al, 2006). The reaction was carried out in the reaction mixture containing 0.1 mM MnCl2 (Cromie et al, 1999). Plasmid library was constructed using the Gateway cloning technology (Invitrogen). The pX-GW plasmid (K Hiramatsu and S Ishikawa, unpublished) was employed to insert the mutagenized sepF fragments, under control of the xylose-inducible promoter Pxyl, into the amyE locus of B. subtilis.

Construction of SepF mutantsFor the GFP or MBP-fusions, the A98V, F124S, G135N, and Δ134 mutations were generated using the Quickchange site-directed mutagenesis method (Stratagene). Briefly, plasmid pSepF–GFP (Hamoen et al, 2006) and pMal–SepF were used as templates for temperature-cycled amplification with primers designed to introduce mutations into these codons. Amplification was carried out with Pfu polymerase (Strategene) and primers (Supplementary Table S2, position of mutation underlined). The product was treated with DpnI to digest unamplified DNA, and transformed into competent DH5α cells. The resulting plasmids with the desired mutations were verified by sequence analysis. To introduce sepF–G135N into B. subtilis, the C-terminal part of sepF was amplified by PCR using pMal–sepFG135N and pMal–sepF (for wild-type control) as template and primers LH224 and LH225 (Supplementary Table S2). The PCR products were digested with EcoRI and BamHI and cloned into the Campbell-integration vector pMutin4 (Vagner et al, 1998), creating pM–sepFG135 and pM–sepF.The SepF mutations were introduced into different B. subtilis backgrounds by transformation of competent cells as described by Moriya et al (1998). The resulting strains are listed in Supplementary Table S1.

Construction of SepF mutants

For the GFP or MBP-fusions, the A98V, F124S, G135N, and Δ134 mutations were generated using the Quickchange site-directed mutagenesis method (Stratagene). Briefly, plasmid pSepF–GFP (Hamoen et al, 2006) and pMal–SepF were used as templates for temperature-cycled amplification with primers designed to introduce mutations into these codons. Amplification was carried out with Pfu polymerase (Strategene) and primers (Supplementary Table S2, position of mutation underlined). The product was treated with DpnI to digest unamplified DNA, and transformed into competent DH5α cells. The resulting plasmids with the desired mutations were verified by sequence analysis. To introduce sepF–G135N into B. subtilis, the C-terminal part of sepF was amplified by PCR using pMal–sepFG135N and pMal–sepF (for wild-type control) as template and primers LH224 and LH225 (Supplementary Table S2). The PCR products were digested with EcoRI and BamHI and cloned into the Campbell-integration vector pMutin4 (Vagner et al, 1998), creating pM–sepFG135 and pM–sepF.

The SepF mutations were introduced into different B. subtilis backgrounds by transformation of competent cells as described by Moriya et al (1998). The resulting strains are listed in Supplementary Table S1.

Microscopic imagingFor fluorescence microscopy, overnight cultures were diluted into fresh PAB medium (for GFP–FtsZ) or CH medium (for SepF–GFP), and grown to exponential phase at 30°C. Cells were mounted on microscope slides covered with a thin film of 1.2% agarose in water. Images were acquired with a Sony Cool-Snap HQ cooled CCD camera (Roper Scientific) attached to a Zeiss Axiovert 200M microscope. The images were analyzed with METAMORPH version 6 software.

Microscopic imaging

For fluorescence microscopy, overnight cultures were diluted into fresh PAB medium (for GFP–FtsZ) or CH medium (for SepF–GFP), and grown to exponential phase at 30°C. Cells were mounted on microscope slides covered with a thin film of 1.2% agarose in water. Images were acquired with a Sony Cool-Snap HQ cooled CCD camera (Roper Scientific) attached to a Zeiss Axiovert 200M microscope. The images were analyzed with METAMORPH version 6 software.

FtsZ co-elutionCell pellets from 30 ml E. coli cultures expressing MBP–SepF mutants were resuspended in 4 ml of buffer A (20 mM Tris–HCl pH 7.4, 200 mM KCl, 1 mM EDTA) with 1 mM PMSF. The suspension was then sonicated (10 W, 4 s pulses) for a total period of 4 min. The resulting suspension was centrifuged at 13 000 r.p.m. for 30 min to remove cell debris. Supernatants containing the different MBP–SepF fusion proteins were added to 1.0 ml of amylose resins (BioLabs) pre-equilibrated with buffer A, and the suspensions were gently mixed at 4°C for 30 min. After incubation, the resins were allowed to settle down by gravity flow and washed six times with 2 ml of buffer A. After the last wash step the resin was mixed with 0.5 ml buffer A and one third of the suspension (0.5 ml) was transferred to empty column, to determine the amount of bound MBP–SepF. MBP–SepF was eluted with 2 × 0.5 ml elution buffer (buffer A containing 10 mM maltose and 1% SDS, 10 min incubation), and the protein concentration was determined with BCA Protein Assay kit (Thermo Scientific Pierce). For the FtsZ co-elution experiments the resin volume was adjusted so that equal amounts of MBP–SepF were used. The MBP–SepF containing amylose resin was loaded in empty columns and 4 ml of cell extract from E. coli cells expressing B. subtilis FtsZ was added and incubated for 30 min at 4°C. After incubation, the columns were washed six times with buffer A before elution with 1.5 ml elution buffer. The flow through, first wash step, second wash step, last wash step, and elution fractions were analyzed by SDS–PAGE. When necessary, the presence of FtsZ in the elution fraction was detected with western blotting using FtsZ antiserum.

FtsZ co-elution

Cell pellets from 30 ml E. coli cultures expressing MBP–SepF mutants were resuspended in 4 ml of buffer A (20 mM Tris–HCl pH 7.4, 200 mM KCl, 1 mM EDTA) with 1 mM PMSF. The suspension was then sonicated (10 W, 4 s pulses) for a total period of 4 min. The resulting suspension was centrifuged at 13 000 r.p.m. for 30 min to remove cell debris. Supernatants containing the different MBP–SepF fusion proteins were added to 1.0 ml of amylose resins (BioLabs) pre-equilibrated with buffer A, and the suspensions were gently mixed at 4°C for 30 min. After incubation, the resins were allowed to settle down by gravity flow and washed six times with 2 ml of buffer A. After the last wash step the resin was mixed with 0.5 ml buffer A and one third of the suspension (0.5 ml) was transferred to empty column, to determine the amount of bound MBP–SepF. MBP–SepF was eluted with 2 × 0.5 ml elution buffer (buffer A containing 10 mM maltose and 1% SDS, 10 min incubation), and the protein concentration was determined with BCA Protein Assay kit (Thermo Scientific Pierce). For the FtsZ co-elution experiments the resin volume was adjusted so that equal amounts of MBP–SepF were used. The MBP–SepF containing amylose resin was loaded in empty columns and 4 ml of cell extract from E. coli cells expressing B. subtilis FtsZ was added and incubated for 30 min at 4°C. After incubation, the columns were washed six times with buffer A before elution with 1.5 ml elution buffer. The flow through, first wash step, second wash step, last wash step, and elution fractions were analyzed by SDS–PAGE. When necessary, the presence of FtsZ in the elution fraction was detected with western blotting using FtsZ antiserum.

Supplementary MaterialSupplementary InformationReview Process File

Supplementary Material

Supplementary Information

Review Process File
