---
pmid: '27225933'
title: Structural evidence for Nap1-dependent H2A-H2B deposition and nucleosome assembly.
authors:
- Aguilar-Gurrieri C
- Larabi A
- Vinayachandran V
- Patel NA
- Yen K
- Reja R
- Ebong IO
- Schoehn G
- Robinson CV
- Pugh BF
- Panne D
journal: EMBO J
year: '2016'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4931181
doi: 10.15252/embj.201694105
---

# Structural evidence for Nap1-dependent H2A-H2B deposition and nucleosome assembly.
**Authors:** Aguilar-Gurrieri C, Larabi A, Vinayachandran V, Patel NA, Yen K, Reja R, Ebong IO, Schoehn G, Robinson CV, Pugh BF, Panne D
**Journal:** EMBO J (2016)
**DOI:** [10.15252/embj.201694105](https://doi.org/10.15252/embj.201694105)
**PMC:** [PMC4931181](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4931181/)

## Abstract

1. EMBO J. 2016 Jul 1;35(13):1465-82. doi: 10.15252/embj.201694105. Epub 2016 May
 25.

Structural evidence for Nap1-dependent H2A-H2B deposition and nucleosome 
assembly.

Aguilar-Gurrieri C(1), Larabi A(1), Vinayachandran V(2), Patel NA(3), Yen K(4), 
Reja R(2), Ebong IO(3), Schoehn G(5), Robinson CV(3), Pugh BF(2), Panne D(6).

Author information:
(1)European Molecular Biology Laboratory, Grenoble, France Unit for Virus 
Host-Cell Interactions, Univ. Grenoble Alpes-EMBL-CNRS, Grenoble, France.
(2)Center for Eukaryotic Gene Regulation, Department of Biochemistry and 
Molecular Biology, The Pennsylvania State University, University Park, PA, USA.
(3)Department of Chemistry, University of Oxford, Oxford, UK.
(4)Department of Cell Biology, Southern Medical University, Guangzhou, China.
(5)Unit for Virus Host-Cell Interactions, Univ. Grenoble Alpes-EMBL-CNRS, 
Grenoble, France Université Grenoble-Alpes, Grenoble, France Centre National de 
la Recherche Scientifique (CNRS) IBS, Grenoble, France CEA, IBS, Grenoble, 
France.
(6)European Molecular Biology Laboratory, Grenoble, France Unit for Virus 
Host-Cell Interactions, Univ. Grenoble Alpes-EMBL-CNRS, Grenoble, France 
panne@embl.fr.

Nap1 is a histone chaperone involved in the nuclear import of H2A-H2B and 
nucleosome assembly. Here, we report the crystal structure of Nap1 bound to 
H2A-H2B together with in vitro and in vivo functional studies that elucidate the 
principles underlying Nap1-mediated H2A-H2B chaperoning and nucleosome assembly. 
A Nap1 dimer provides an acidic binding surface and asymmetrically engages a 
single H2A-H2B heterodimer. Oligomerization of the Nap1-H2A-H2B complex results 
in burial of surfaces required for deposition of H2A-H2B into nucleosomes. 
Chromatin immunoprecipitation-exonuclease (ChIP-exo) analysis shows that Nap1 is 
required for H2A-H2B deposition across the genome. Mutants that interfere with 
Nap1 oligomerization exhibit severe nucleosome assembly defects showing that 
oligomerization is essential for the chaperone function. These findings 
establish the molecular basis for Nap1-mediated H2A-H2B deposition and 
nucleosome assembly.

© 2016 The Authors. Published under the terms of the CC BY NC ND 4.0 license.

DOI: 10.15252/embj.201694105
PMCID: PMC4931181
PMID: 27225933 [Indexed for MEDLINE]

## Full Text

IntroductionEukaryotic genomes are organized into chromatin, a higher‐order structure comprising histones and DNA. The basic repeating unit of chromatin is the nucleosome, which consists of a histone octamer enwrapped by 147 bp of DNA, which is organized into two superhelical turns (Luger et al, 1997). Spatial and temporal regulation of specific chromatin loci by dynamic assembly of nucleosomes is essential for the control of DNA‐templated processes such as replication, DNA damage repair, and gene regulation (Venkatesh & Workman, 2015). The nucleosome serves as a general transcriptional repressor, and promoter nucleosome removal is an early step in gene activation (Becker & Horz, 2002; Bryant et al, 2008). Thus, the ordered disassembly and reassembly of nucleosomes is an important paradigm in gene regulation. Chromatin remodeling depends on the recruitment of acetyltransferases that acetylate the four core histones, on the activity of chromatin‐remodeling complexes and on the capacity of histone chaperones to act as a histone acceptors, facilitating their removal from DNA to stimulate transcription (Ito et al, 2000; Asahara et al, 2002; Lorch et al, 2006; Adkins et al, 2007; Sharma & Nyborg, 2008).
In vitro studies have revealed that the nucleosome assembly is a two‐step process, involving the initial binding of an (H3–H4)2 tetramer to the DNA, followed by the binding of two H2A–H2B dimers to complete the octameric nucleosome (Smith & Stillman, 1991). In vivo, the assembly process is catalyzed by histone chaperones, a diverse class of proteins that is involved in histone turnover including transport, transfer, and storage (De Koning et al, 2007). Structural studies of histone chaperones bound to cognate histones have revealed that histone chaperones frequently protect hydrophobic histone interfaces that become solvent‐exposed outside of the context of the nucleosome or neutralize excess positive charge by providing an acidic binding pocket to directly compete with non‐specific electrostatic histone–DNA interactions (English et al, 2006; Zhou et al, 2008, 2011; Cho & Harrison, 2011; Hu et al, 2011; Elsasser et al, 2012; Liu et al, 2012; Hondele et al, 2013; Obri et al, 2014; Huang et al, 2015). However, how chaperones mediate the final deposition onto DNA to assemble nucleosomes remains poorly understood (Loyola & Almouzni, 2004; De Koning et al, 2007; Elsasser & D'Arcy, 2012). Nucleosome assembly is central for DNA replication and possibly a rate‐limiting factor for the control of gene expression. It is therefore important to understand how histone chaperones, in conjunction with chromatin modifiers and remodelers, mediate and control this process.Nap1 is universally conserved among eukaryotes and is the founding member of the large NAP family of histone chaperone proteins. While Nap1 is one of the best‐studied histone chaperones, the molecular determinants of its role in histone chaperoning and nucleosome assembly have remained elusive. In the work reported here, we find that Nap1 is required for H2A–H2B deposition on a genomic scale. The structure of yeast Nap1 in complex with H2A–H2B, determined at 6.7 Å resolution, demonstrates that Nap1 uses an acidic binding surface to engage histone H2A of a single H2A–H2B heterodimer. Oligomerization of the Nap1–H2A–H2B protomer results in burial of surfaces required for the deposition of H2A–H2B into nucleosomes and provides an additional function, presumably histone transport or storage. Disruption of oligomerization results in a strong nucleosome assembly defects across the genome indicating that there are two separable functions of Nap1: histone transport/storage and nucleosome assembly.

Introduction

Eukaryotic genomes are organized into chromatin, a higher‐order structure comprising histones and DNA. The basic repeating unit of chromatin is the nucleosome, which consists of a histone octamer enwrapped by 147 bp of DNA, which is organized into two superhelical turns (Luger et al, 1997). Spatial and temporal regulation of specific chromatin loci by dynamic assembly of nucleosomes is essential for the control of DNA‐templated processes such as replication, DNA damage repair, and gene regulation (Venkatesh & Workman, 2015). The nucleosome serves as a general transcriptional repressor, and promoter nucleosome removal is an early step in gene activation (Becker & Horz, 2002; Bryant et al, 2008). Thus, the ordered disassembly and reassembly of nucleosomes is an important paradigm in gene regulation. Chromatin remodeling depends on the recruitment of acetyltransferases that acetylate the four core histones, on the activity of chromatin‐remodeling complexes and on the capacity of histone chaperones to act as a histone acceptors, facilitating their removal from DNA to stimulate transcription (Ito et al, 2000; Asahara et al, 2002; Lorch et al, 2006; Adkins et al, 2007; Sharma & Nyborg, 2008).

In vitro studies have revealed that the nucleosome assembly is a two‐step process, involving the initial binding of an (H3–H4)2 tetramer to the DNA, followed by the binding of two H2A–H2B dimers to complete the octameric nucleosome (Smith & Stillman, 1991). In vivo, the assembly process is catalyzed by histone chaperones, a diverse class of proteins that is involved in histone turnover including transport, transfer, and storage (De Koning et al, 2007). Structural studies of histone chaperones bound to cognate histones have revealed that histone chaperones frequently protect hydrophobic histone interfaces that become solvent‐exposed outside of the context of the nucleosome or neutralize excess positive charge by providing an acidic binding pocket to directly compete with non‐specific electrostatic histone–DNA interactions (English et al, 2006; Zhou et al, 2008, 2011; Cho & Harrison, 2011; Hu et al, 2011; Elsasser et al, 2012; Liu et al, 2012; Hondele et al, 2013; Obri et al, 2014; Huang et al, 2015). However, how chaperones mediate the final deposition onto DNA to assemble nucleosomes remains poorly understood (Loyola & Almouzni, 2004; De Koning et al, 2007; Elsasser & D'Arcy, 2012). Nucleosome assembly is central for DNA replication and possibly a rate‐limiting factor for the control of gene expression. It is therefore important to understand how histone chaperones, in conjunction with chromatin modifiers and remodelers, mediate and control this process.

Nap1 is universally conserved among eukaryotes and is the founding member of the large NAP family of histone chaperone proteins. While Nap1 is one of the best‐studied histone chaperones, the molecular determinants of its role in histone chaperoning and nucleosome assembly have remained elusive. In the work reported here, we find that Nap1 is required for H2A–H2B deposition on a genomic scale. The structure of yeast Nap1 in complex with H2A–H2B, determined at 6.7 Å resolution, demonstrates that Nap1 uses an acidic binding surface to engage histone H2A of a single H2A–H2B heterodimer. Oligomerization of the Nap1–H2A–H2B protomer results in burial of surfaces required for the deposition of H2A–H2B into nucleosomes and provides an additional function, presumably histone transport or storage. Disruption of oligomerization results in a strong nucleosome assembly defects across the genome indicating that there are two separable functions of Nap1: histone transport/storage and nucleosome assembly.

ResultsNap1 is required for nucleosome assembly in vivo
To determine the contribution of yNap1 to chromatin assembly in vivo, we performed an MNase digestion assay on chromatin from the wild‐type BY4741 (WT) and a yNap1 deletion strain nap1Δ. Gel electrophoresis of chromatin, partially digested with MNase, showed a nucleosomal ladder (Fig 1A). The distance between each band corresponds to the distance between the most distal nucleosome ends within an MNase‐cleaved array of an integral number of nucleosomes. For oligonucleosomes, the size of each band includes the sum of the nucleosome core DNA lengths plus linker lengths (147 bp + 20 bp) (Fig 1A). Consequently, the interband distance corresponds to a population average spacing between adjacent nucleosomes.Figure 1Role of yNap1 in nucleosome assembly on a genomic scale
Analysis of MNase‐digested chromatin DNA fragments from wild‐type (WT, sizes indicated in black) and nap1Δ (N, sizes indicated in red) cells by agarose gel electrophoresis. Diagram illustrating reduction in array length in nap1Δ cells.Frequency distribution of nucleosomal DNA size (WT in gray fill with nap1Δ in blue traces) from H3 ChIP‐seq.Calculation of nucleosome fuzziness as the standard deviation of nucleosome (or tag) positions. See also Appendix Fig S1.
When compared to BY4741 (WT), the population distribution of nucleosomal sizes in the nap1Δ strain was more enriched with smaller DNA fragments, indicating that linker regions were more accessible to MNase. The mononucleosome‐sized MNase fragments electrophoresed in a smaller size range (~145 bp) with respect to the wild type (~170 bp) (Fig 1A, Appendix Fig S1A), indicating that yNap1 has an impact on nucleosome assembly in vivo. This shorter fragment size is more consistent with an assembly intermediate such as a hexasome rather than a full nucleosome, although other types of nucleosomal and subnucleosomal structures are not excluded.MNase digestions indicate that the nucleosomal arrays maintained approximate wild‐type spacing (or ~165 bp interband distance) to the extent that can be measured by gel electrophoresis, indicating that nucleosome spacing in bulk chromatin was generally unaltered in the nap1Δ strain. However, the di‐ and trinucleosome arrays were shorter by a constant ~25 bp in the nap1Δ strain. We interpret the fixed size reduction as follows: MNase is expected to cleave more frequently where linkers are longer, and this is expected to occur in the nap1Δ strain where H2A–H2B is lost (i.e., linker DNA plus the nucleosomal DNA that is vacated by H2A–H2B). Having additional cleavage opportunities where H2A–H2B are lost produces the fixed size reduction in the array. If H2A–H2B assembly into nucleosomes is dynamic even in the nap1Δ strain (but partially shifted toward dissociation), then nucleosomes that exist within cleaved arrays are likely to be full nucleosomes, since their spacing appears to be unchanged. In other words, spacing is set by the predominance of full nucleosomes, with cleavage occurring where there is a relatively infrequent dissociation of H2A–H2B.To further investigate the impact of yNap1 on chromatin assembly, we mapped nucleosome positions on a genomewide scale. This assay involves formaldehyde‐cross‐linking and chromatin fragmentation to mononucleosomes using MNase followed by ChIP. DNA fragments that immunoprecipitate with a H3 antibody were then detected by deep sequencing. To take into account the underlying intrinsic nucleosome occupancy levels, we first measured the overall histone levels by immunoblotting (Appendix Fig S1B). These data showed that the overall histone levels between the wild type and nap1Δ strain were similar. Therefore, we normalized the total tag counts in both samples to be equal. In these single‐read sequencing experiments, nucleosome sizes (including subnucleosomes) were deduced by pairing adjacent peaks located on opposite strands and in the 3′ direction. Confirming the in‐gel observation, these structures were approximately hexasomal in size in the nap1Δ strain (Fig 1B, ~100–130 bp range, compared to ~150 bp for wild type). In this population analysis, they also had fuzzier locations (higher standard deviation associated with peak calling) (Fig 1C). This suggests that the hexasomal‐sized nucleosomes were positionally more dispersed than full nucleosomes. Accumulation of these putative hexasomes indicates that yNap1 preferentially deposits a second copy of H2A–H2B dimers into nucleosomal arrays. Together, these data indicate that yNap1 contributes to nucleosome assembly in vivo.Structure of the yNap1–H2A–H2B complexTo understand the molecular basis for how Nap1 chaperones and deposits H2A–H2B into chromatin, we crystallized Saccharomyces cerevisiae Nap1 (yNap1) bound to Xenopus laevis H2A–H2B. Crystals were obtained with the full‐length proteins but diffracted poorly. In the crystal structure of full‐length yNap1 (Park & Luger, 2006), the N‐ and C‐terminal regions are disordered and we were able to improve diffraction to a minimum Bragg spacing of ~6.7 Å by N‐ and C‐terminal truncation of yNap1 to residues 75–365 and N‐terminal truncation of the histones H2A (14–130) and H2B (28–126) (Fig 2A). This core domain of yNAP1 (yNap1c) is sufficient to bind the H2A–H2B dimer in solution as well as to assemble chromatin from DNA and histone components (Fujii‐Nakata et al, 1992; Park et al, 2005). The structure was determined by molecular replacement with the individual structures of the yNap1c homodimer and the histone H2A–H2B heterodimer (Fig 2B and C) (Park et al, 2008). Molecular replacement without H2A–H2B unambiguously showed the presence of the missing histones with an overall fold that was clearly visible (Figs 2 and EV1A and B). Recent method advances allow low‐resolution refinement and improvement of the low‐resolution model and electron density map by deriving deformable elastic network (DEN) restraints from known high‐resolution models (Schroder et al, 2010). Parameter optimization for DEN refinement covered 510 individual computations and allowed us to reduce the Rfree to 0.31 while maintaining an overall good coordinate accuracy (Table EV1).Figure 2Crystal structure of the yNap1–H2A–H2B complex
Domain architecture of yNap1, H2A, and H2B. The Histone binding region (HBR1 and HBR2) of yNap1 and the yNap1 binding region (NBR1 and NBR2) of H2A are indicated. Regions involved in higher‐order oligomerization are shown (IF1 and IF2).Ribbon view of the asymmetric crystallographic unit containing a (yNap1c2–H2AΔ14–H2BΔ28)6 assembly. For clarity, only layer 1 of the complex is shown. Layer 1 contains three protomers A–C that are related by two perpendicular non‐crystallographic dyads ().Side view of the complex. The layer 2 omitted from panel (B) is shown in gray. The yNap1 dimer is shown in green and yellow, H2A in red, and H2B in blue. See also Fig EV1.View of the yNap1–H2A–H2B protomer complex with labeling of secondary structure elements.
Figure EV1DNA–H2A–H2B binding and electrostatics of the yNap1–H2A–H2B complexRelated to Fig 2.
A, B2F0–Fc omit electron density map for the region comprising H2A–H2B contoured at 1σ in absence (A) or presence (B) of H2A/H2B in ribbon representation. Similar views as in Fig 2A–C are shown.C–EDNA binding of the H2A–H2B heterodimer.FView of the ternary complex with H2A–H2B in ribbon representation. APBS‐calculated electrostatics (−7 kT to +7 kT) were mapped onto yNap1 dimer surface.GHBR1.HHBR2.
The low‐resolution diffraction of the crystals may be reflective of the dynamic oligomerization of the yNap1c2–H2A–H2B complex in solution; however, the availability of the individual yNap1c and H2A–H2B structures effectively provides atomic‐resolution details to the structure of the complex. Previous low‐resolution hydrogen–deuterium exchange experiments suggested that a yNap1 dimer accommodates two copies of H2A–H2B in a tetrameric conformation (D'Arcy et al, 2013). Instead, we found that only a single H2A–H2B heterodimer can be accommodated in the binding pocket of a Nap1 dimer. Six copies of this yNap1c2–H2A–H2B complex form the asymmetric crystallographic unit, which belong to space group P21 (Table EV1).The yNap1 dimer is arranged in an antiparallel fashion such that the globular domains are positioned on opposite ends of the dimer (Park & Luger, 2006). The concave face of the dome‐shaped yNap1 homodimer constitutes the binding site for the H2A–H2B heterodimer. This protein–protein interface is mediated by both yNap1 subunits and almost exclusively by H2A and results in the burial of ~761 Å2 of surface area. The H2A–H2B heterodimer retains the configuration as seen in the nucleosome structure (Luger et al, 1997). In the nucleosome, the H2A–H2B heterodimer binds 2.5 turns of DNA double helix, which arcs around them along their long axes to generate a ~140° bend (Figs 2 and EV1C–E). We find that the yNap1 dimer mimics the structure and electrostatics of nucleosomal DNA and provides a concave acidic binding surface for H2A–H2B (Figs 2D and EV1F–H). As the H2A–H2B heterodimer is positioned asymmetrically in the electronegative binding cavity, yNap1 engages two of the three DNA binding surfaces of H2A–H2B with the majority of contacts toward H2A (Fig 3A). In the center of the heterodimer, contacts occur between H2A α1 and histone binding region 1 (HBR1) of yNap1, which spans the C‐terminus of α4 to the N‐terminus of α6 (residues 194–205) (Fig 3B). The unique N‐terminal helix αN of H2A (residues 16–21) complements this binding interface by packing against E310 of yNap1, which is located in the region C‐terminal to the disordered β‐hairpin (β5–β6). The second DNA binding interface of H2A–H2B, containing binding loops L1 of H2B and L2 of H2A, packs against histone binding region 2 (HBR2), which comprises the N‐terminus of helix α8 of the second yNap1 monomer (residues 328–336) (Fig 3C). The majority of contacts occur between H2A L2 residues K75–R81 and E328–E336 of helix α8 of yNap1. The positive dipole moments of helix α2 of H2B and α3 of H2A, which are directed toward the DNA phosphodiester backbone in context of the nucleosome, are capped by this negatively charged patch on helix α8 of yNap1. The third DNA binding interface of the H2A–H2B heterodimer containing the H2A L1 and H2B L2 loops does not engage yNap1 (Fig 3A).Figure 3Interactions of yNap1 with the basic DNA binding surface of the H2A–H2B heterodimer
Overview of the yNap1c2–H2AΔ14–H2BΔ28 complex.Residues R17, R20, R29, R32, R35, and K36 of H2A make contacts with the acidic HBR1 in yNap1 comprising residues D201, D205, and E310.Residues K75, R77, and R81 of H2A make polar contacts with the HBR2 comprising residues E332, D333, and E339 in α8 of yNap1.
In vitro GST‐pulldown assay showing the role of residues in the HBR interfaces in H2A–H2B binding. The mutant‐labeled HBR1 (lane 6) contains the triple mutation D201R, D205R, and E310R, and HBR2 (lane 12), the triple mutations E332, D333, and E336. HBR1 + 2 (lane 13) contains all six mutations. The mutant‐labeled HBR1 + 2A (lane 14) has alanine replacements in the six HBR residues. Top panel: Bound. Bottom panel: Input.Disruption of the Nap1–histone interaction by H2A mutation. Bottom panel: Inputs of the coexpressed proteins (soluble extract) and top panel the material bound to the glutathione affinity column were analyzed by SDS–PAGE and Coomassie staining.Controls showing input (bottom) and bound (top) fractions. See also Fig EV2.

Source data are available online for this figure.
To assess the contribution of the HBR interfaces to yNap1–H2A–H2B complex formation, we created mutants targeting conserved, surface‐exposed residues in the histone binding interfaces and tested H2A–H2B binding using a GST‐pulldown assay (data are summarized in Appendix Table S2). Single, double, or triple (D201R, D205R, E310R) mutations in the HBR1 interface did not abolish H2A–H2B binding (Fig 3D, lanes 1–6). In the HBR2 interface, single, double, or triple (E332, D333, E336) mutation or replacement of residues 319–337 with a single Gly amino acid residue, a modification ablating the segment of α8 comprising the HBR2 region (ΔHBR2), also did not abolish H2A–H2B binding (Fig 3D, lanes 7–12). However, when both HBR interfaces contained charge reversal or alanine mutations, we observed reduced H2A–H2B binding (Fig 3D, lanes 13–15). We therefore conclude that both HBR interfaces contribute synergistically to binding of H2A–H2B. Residues in the HBR interfaces are conserved in Nap1 from yeast to humans but not in Vps75 and SET indicating that these structurally related histone chaperones have evolved different modes of histone binding (Fig EV2A–D).Figure EV2Sequence conservationRelated to Fig 3. 
The yNap1 dimer is color‐coded according to the sequence conservation.HBR2.HBR1. The higher the conservation, the darker the color. Similar views as in Fig 3A–C are shown.Nap1‐like histone chaperons with highlighted conserved residues. Secondary structure elements are shown above the amino acid sequences. The histone binding region 1 or 2 (HBR1 or HBR2) as well as the oligomerization interface (IF1) of yNap1 are highlighted in green. Key residues in the HBR1 and HBR2 (●) as well as IF1 (▵) are indicated. The structure‐based sequence alignment was performed using Pymol (Schrodinger, 2010) and ClustalW (Thompson et al, 1994). The figure was generated using Espript (Gouet et al, 2003).
We also tested the impact of mutations on the ability to copurify overexpressed yNap1 and H2A–H2B from E. coli cells. GST‐yNap1 wild type (Fig 3E, lane 1) but not the yNap1 HBR1 + 2 mutant was able to copurify H2A–H2B on the glutathione affinity resin (Fig 3F, lane 2). As expected, expression of yNap1 alone did not show a band at the expected position for H2A–H2B (Fig 3F, lane 1) and H2A–H2B alone was expressed but did not bind to the resin (Fig 3E, lane 14). To assess the contribution of residues at the histone side of the interface, we introduced mutations in the Nap1 binding regions NBR1 and NBR2 of H2A. Mutation of single amino acid residues in NBR1 or NBR2 of H2A (Fig 3E, lanes 2–6) or mutation of the NBR2 region alone was not sufficient to fully abolish binding (Fig 3E, lane 11). However, variants containing several mutations in NBR1 and NBR2 reduced H2A–H2B binding (Fig 3E, lanes 7–13). In particular, the triple NBR1 mutant (R30E, R33E, K37E) or a sextuple mutant NBR1 + 2 that also has the NBR2 mutations K75E, R78E, R82E showed drastically reduced binding (Fig 3E, lanes 12–13). Variants containing combinations of mutations in both interfaces generally showed greatly reduced histone binding (Fig 3E, lane 7–9). A mutant designed to disrupt the H2A oligomerization interface IF2 (N94E, G98D, R99D, T101D) reduced but did not fully abolish H2A–H2B binding, indicating that oligomerization through this interface is not essential for complex formation but contributes to binding avidity (Fig 3E, lane 10). Together, mutations in yNap1 and H2A that were designed to decrease histone H2A–H2B binding do indeed weaken the yNap1–histone H2A–H2B interaction. A charge‐swap experiment provides further proof for the validity of the interface: In the yNap1 sextuple mutant HBR1 + 2 and the H2A sextuple mutant NBR1 + 2, the charged interfacial residues are swapped. While each mutant tested against the wild‐type counterpart loses binding (Fig 3D, lane 10 and Fig 3E, lane 13), binding is restored, when these mutants are tested against each other (Fig 3F, lane 3).Oligomerization of the yNap1–H2A–H2B complexOligomerization of various Nap1 orthologs occurs in vivo and under physiological conditions in vitro (Ishimi et al, 1983, 1984; Fujii‐Nakata et al, 1992; Chang et al, 1997; Mosammaparast et al, 2002; McBryant & Peersen, 2004; Toth et al, 2005; Park et al, 2008; Noda et al, 2011; Newman et al, 2012). The biological role of Nap1 oligomerization has so far remained unclear. Depending on the method of analysis, the ionic strength, and concentration used, different oligomerization states of Nap1 dimers have been reported. In size‐exclusion chromatography, purified yNap1 alone or in complex with H2A–H2B elutes as a broad peak indicative of a heterogeneous mixtures of various oligomeric species (Mosammaparast et al, 2002; Park et al, 2008). The crystal structure of yNap1c in the absence of histones revealed an extended β5–β6 hairpin that is involved in crystal packing (Park et al, 2008). The same packing interface is not available in crystals of full‐length yNap1, but mutation of this packing interface reduces yNap1 oligomerization in vitro (Park et al, 2008). In the presence of histones, yNap1 shows similar propensities to oligomerize (Toth et al, 2005; Noda et al, 2011). Xenopus laevis Nap1 in the presence of H2A–H2B or H3–H4 forms heterogeneous ring‐like particles in the 0.5–1.0 MDa size range as visualized by electron microscopy (Newman et al, 2012). The underlying compositional heterogeneity has made it difficult to ascertain the exact stoichiometry and to understand the structural basis and biological role of Nap1–histone oligomerization. We therefore asked whether our crystal structure is relevant for Nap1 oligomerization. The 6 copies of the yNap1c2–H2A–H2B complex in the crystallographic asymmetric unit form two layers, each containing three protomers (A–C). Protomers A and B (A' and B' in layer 2) are related by a ~102° rotation and ~8 Å displacement along the central axis (Fig 4A). Two non‐crystallographic dyads relate protomer C to A and B in each layer, but because of displacement along the central axis, protomers C and C' pack through alternate protein interfaces: Protomer C is located below the plane of protomers A and B, whereas protomer C' is located above A' and B'. Because of the alternating arrangement of layers 1 and 2 in the crystal (Fig 4F), packing of the two layers results in a pseudohelical assembly.Figure 4Oligomerization interfaces
Cartoon illustrating the arrangement of the two layers of the yNap1c2–H2A–H2B oligomer.A ribbon view of the protomers B and C comprising the IF1 interface.Close‐up view of interactions in the IF1 oligomerization interface.Ribbon view of protomers A and A' comprising the interlayer oligomerization interface IF2.Close‐up view of interactions in the IF2 interface.Crystal packing interactions. To illustrate packing, the two layers were moved apart. Packing of the two layers in the crystallographic asymmetric unit is stabilized by two interlayer IF2 interfaces (indicated by two arrows facing each other) between protomers A–A' and B–B'. The IF2 interface between protomers C provides for packing to the neighboring unit cells. H2A is colored in red, H2B in blue, and yNap1 in green and yellow. See also Fig EV3.
To investigate the yNap1–H2A–H2B complexes further, we visualized our preparations by negative‐stain EM. Electron micrographs showed that complexes containing full‐length yNap1 populated a series of oligomeric states (Fig EV3D and E). No such particles were observed in the absence of histones indicating that particle formation is dependent on H2A–H2B. Electron micrographs of yNap1c in complex with H2AΔ14–H2BΔ28 revealed monodisperse particles, about 15 nm in diameter, similar to those observed previously (Newman et al, 2012). Varying structural features indicated different orientations of the complex on the carbon support film (Fig 5A). A single particle reconstruction of the yNap1c2–H2AΔ14–H2BΔ28 complex matched well with the low‐pass filtered projections from the crystal structure showing that the oligomer seen in the crystals is relevant for assembly in solution (Fig 5B and C).Figure EV3yNap1–H2A–H2B oligomerizationRelated to Fig 4. 
ANative mass spectra of yNap1c against varying concentrations of H2AΔ14–H2BΔ28.BNative PAGE assay of yNap1 binding to H2A–H2B (lanes 1–5), yNap1c binding to H2A–H2B (lanes 6–10), or of yNap1_IF1 to H2A–H2B (lanes 11–15).CNative PAGE assay of H2AΔ14_IF2–H2BΔ28 binding to yNap1 (lanes 1–5) or yNap1c (lanes 6–10) or yNap1_IF1 (lanes 11–15). Samples were separated by 6% native PAGE and visualized by Coomassie staining.D, EEM analysis of the full‐length yNap1–H2A–H2B complex.
Figure 5Native mass spectrometry and negative‐stain EM of the yNap1–H2A–H2B particle
AEM analysis of the yNap1c2–H2AΔ14–H2BΔ28 complex. Representative raw image of negatively stained particles.BRepresentative particles (1st row), class averages coming from the projection matching procedure (2nd row) and comparison of the class averages with projections from a final model generated after four projection matching cycles (3rd row).CFrom left to right: Crystal structure of the (yNap1c2–H2A–H2B)6 complex in two different orientations (brown); 60 Å‐filtered structure of the complex (magenta); 3D reconstruction of the complex from negatively stained particles (green) and fit of the crystal structure into the EM map.D–IMass spectra of yNap1 variants in the absence and presence of H2A–H2B. yNap1 in complex in the absence (D) or presence of H2A–H2B (E). yNap1c in the absence (F) or bound to H2A–H2B (G). The oligomerization mutant yNap1_IF1 alone (H) or in complex with H2A_IF2Δ14–H2BΔ28 (I). yNap1 is shown in green circles and H2A–H2B as red and blue asterisks.
To analyze the relevance of the oligomerization interfaces, we prepared a series of yNap1 variants and compared their propensity to oligomerize. Previous analysis by ensemble averaging techniques such as size exclusion chromatography, sucrose gradients, analytical ultracentrifugation, and small‐angle X‐ray scattering did not converge on a unique solution for Nap1 oligomerization and histone binding stoichiometry (Ishimi et al, 1983, 1984; Fujii‐Nakata et al, 1992; Chang et al, 1997; Mosammaparast et al, 2002; McBryant & Peersen, 2004; Toth et al, 2005; Park et al, 2008; Noda et al, 2011; Newman et al, 2012). Instead, we used nanoflow electrospray ionization–mass spectrometry under native conditions, a technique that affords unparalleled mass accuracy and enables unambiguous determination of protein stoichiometry. Mass spectra of yNap1 alone showed three major series of resolved peaks with different charge states corresponding mostly to yNap1 dimers, tetramers, and a minor hexameric fraction (Fig 5D and Table EV2). In the presence of H2A–H2B, we obtained a series of larger complexes corresponding to yNap1 dimers, tetramers, hexamers, and octamers bound to one, two, three, or four copies of H2A–H2B, respectively (Fig 5E and Table EV2). For yNap1c alone, we observed dimers, tetramers, and a hexameric fraction (Fig 5F and Table EV2). In the presence of H2AΔ14–H2BΔ28, we observed assemblies containing up to 6 copies of the yNap1c2–H2A–H2B complex (Fig 5G and Table EV2). Titration of yNap1c with increasing amounts of H2AΔ14–H2BΔ28 revealed oligomers containing up to 6 copies even when concentrations of H2A–H2B were limiting (Fig EV3A). H2A–H2B binding enhanced higher‐order oligomerization, but the flexible histone tails did not appear to contribute to self‐assembly as we observed a similar degree of oligomerization in their presence or absence (Table EV2). Also, the flexible, non‐conserved N‐terminal and highly negatively charged C‐terminal regions of yNap1 did not contribute to oligomerization of isolated yNap1, in agreement with data reported previously (Park et al, 2008). However, in the presence of H2A–H2B, self‐assembly was more pronounced when these flexible tails were absent. In all experiments, and in agreement with our structure, we found that a single H2A–H2B dimer binds to a yNap1 dimer and we could not confirm the stoichiometry proposed by D'Arcy et al (2013). These data confirm that a single yNap1 dimer binds to a single H2A–H2B heterodimer and that this complex self‐assembles into higher‐order oligomers.To investigate the relevance of the oligomerization interfaces for yNap1 function, we performed a set of in vitro and in vivo experiments. We identified and characterized the largest interfaces that drive packing of individual protomers. In layer 1, protomers B and C pack through a polar interface (IF1) that is composed of yNap1 residues from the linker connecting β1 and β2 and the beginning of β3 (Fig 4B and C). A second major interface (IF2) occurs between the two layers and comprises the C‐terminal docking domains of H2A, which involves residues of α3 and αC in protomers A–A' and B–B' (Fig 4D–F). This double‐layered assembly packs into the crystal by forming IF2‐type interactions between protomers C and C' (Fig 4F). To analyze the contribution of the oligomerization interfaces, we mutated four residues, G225D, S237D, T244D, and D246R, in the IF1 interface of yNap1 (yNap1_IF1) and four residues, N94E, G98D, R99D, and T101D, in the IF2 interface of tail‐less H2AΔ14 (H2AA Δ14_IF2). Native mass spectra of yNap1_IF1 showed mostly a dimer with a minor tetrameric fraction (Fig 5H; Table EV2). In the presence of H2AΔ14_IF2–H2BΔ28, we observed mostly a yNap1_IF1 dimer bound to a single histone heterodimer (Fig 5I; Table EV2). The β5–β6 hairpin that comprises a nuclear localization sequence (NLS) and that has been previously implicated in oligomerization of isolated yNap1 (Park et al, 2008) does not appear to contribute to oligomerization of the H2A–H2B‐bound complex: It does not form the same crystal packing interface reported previously and is solvent‐exposed and disordered in our structure. To analyze the contribution of the β‐hairpin, we generated an in‐frame deletion of the β5–β6 region (yNap1cΔ), a modification that does not disrupt structural integrity, and found similar oligomerization properties as yNap1c, while a construct (yNap1cΔ_G225D) containing in addition the IF1 mutation G225D remained dimeric (Table EV2). In the presence of H2A–H2B, yNap1cΔ oligomerized to a similar degree than yNap1c showing that the β5–β6 region does not contribute to oligomerization of the histone‐bound complex. In contrast, the yNap1cΔ_G225D mutant did not oligomerize upon addition of H2A–H2B, thus reconfirming the relevance of the IF1 interface to oligomerization of the yNap12–H2A–H2B complex. More qualitative native PAGE experiments also showed that the IF1 mutant has a reduced propensity to oligomerize compared to yNap1 or yNap1c both with wild‐type H2A–H2B (Fig EV3B) and with tail‐less H2A–H2B containing the IF2 mutation (Fig EV3C). The IF2 interface also contributes to oligomerization as yNap1 and yNap1c showed reduced oligomerization upon addition of H2AΔ14_IF2–H2BΔ28 (Fig EV3C). We conclude that the IF1 and IF2 interfaces are relevant for oligomerization of the yNap12–H2A–H2B complexes and that H2A–H2B binding to yNap1 and oligomerization contributes synergistically to the binding free energy of complex formation. As oligomerization of the yNap12–H2A–H2B complex results in burial of a segment that is required for docking onto H3–H4 tetrasomes, we argue that it is inhibitory for H2A–H2B deposition and nucleosome formation. Such an inhibitory conformation could be relevant for histone transport or storage. The pseudohelical arrangement of the yNap12–H2A–H2B complex probably forms the basis for the dynamic oligomerization that has been observed by us and others (McBryant & Peersen, 2004; Toth et al, 2005; Noda et al, 2011; Newman et al, 2012).Nucleosome assembly activity of yNap1 in vitro and in vivo
To investigate the contribution of yNap1 to nucleosome assembly, we established a quantitative multiple fluorescence relative affinity assay (Man & Stormo, 2001). This method constitutes a competition assay where fluorescently labeled free DNA (AF647) or (H3–H4)2–DNA tetrasome complexes (AF488) compete for the same pool of H2A–H2B, which is provided free or in complex with yNap1 (Fig 6A). This competition assay allowed us to measure the relative affinities of H2A–H2B toward (H3–H4)2–DNA tetrasome complexes and toward free DNA and assess the impact of yNap1 on the relative rates of nucleosome formation and non‐specific DNA binding of H2A–H2B. As a DNA substrate, we used a 147‐bp fragment containing the 601 nucleosome positioning sequence (Lowary & Widom, 1998). The bound and unbound fractions were separated using an electrophoretic mobility shift assay (EMSA) and quantified to derive an apparent dissociation constant of yNap1‐mediated H2A–H2B deposition and nucleosome assembly. We found that yNap1 specifically deposited H2A–H2B onto (H3–H4)2–DNA tetrasome complexes resulting in nucleosome formation (K
d ~940 nM; Fig 6A and B) while preventing non‐specific deposition of H2A–H2B onto free DNA (Fig 6A). In the absence of yNap1, H2A–H2B preferentially bound to the (H3–H4)2–DNA tetrasome complexes to form nucleosomes (K
d ~29 nM) but also interacted non‐specifically with the free DNA at higher concentrations (Fig EV4A and B). The yNap1c and yNap1_IF1 mutants are not affected in this nucleosome assembly assay indicating that the flexible N‐ and C‐terminal regions and oligomerization of yNap1 are not essential for H2A–H2B deposition (Fig EV4C and D). Free H2A–H2B preferentially interacted with tetrasomes and only once these were exhausted from the reaction H2A–H2B bound non‐specifically to the free DNA. In accordance with our structure and previously published data (Andrews et al, 2010), yNap1 is suppressing non‐specific DNA binding of H2A–H2B and thereby facilitates specific deposition of H2A–H2B and nucleosome assembly.Figure 6Nucleosome assembly in vitro and in vivo

Native PAGE gel showing the result of an affinity shift assay scanned to detect AF488‐ (left) or AF647‐labeled DNA (right); 31 nM–4 μM of yNap1–H2A–H2B (lanes 6–13) was incubated with a mixture of 0.8 μM AF488‐labeled H3–H4–DNA tetrasome complexes and 0.8 μM AF647‐labeled DNA (lane 5). Lanes 1–3 show migration of tetrasomes (tetra), nucleosomes (nucl), or free H2A–H2B obtained by salt deposition onto AF647‐labeled DNA.Intensity of the nucleosome band (nucl) is plotted. The value at 4 μM of yNap1–H2A–H2B (lane 13) was set to 100%. For non‐specific DNA binding, the value in the absence of H2A–H2B (lane 5) was set to 100% intensity and disappearance of the free DNA AF647 band was plotted. Each curve is representative of three independent experiments performed on different days. Standard deviations are shown.Analysis of MNase‐digested chromatin DNA fragments from wild‐type (BY4741), nap1Δ, and nap1Δ cells complemented with different yNap1 expression plasmids. Total MNase units per 50 ml of culture are indicated on the bottom of the lanes. Kb+: DNA marker.Positional representation of MNase‐titrated nucleosomal arrays from panel (C), comparing with BY4741 (blue traces) with NAP1 deletion (green traces) and other mutants (HBR1—black, HBR2—red, HBR1 + 2—purple, and IF1—gold traces). Color gradients indicate MNase concentrations (from high to low), and the yellow box represents mononucleosomes. Dashed lines represent nucleosome dyad with respect to BY4741 (WT). Traces are offset vertically for ease of visibility. See also Fig EV4I.
Figure EV4Nucleosome assembly in vitro
Related to Fig 6. 
ANative PAGE gel showing the result of a multiple fluorescence relative affinity shift assay in the absence of yNap1; 31 nM–8 μM H2A–H2B (lanes 4–12) were incubated with a mixture of 0.7 μM AF488‐labeled H3–H4–DNA tetrasome complexes and 0.7 μM AF647‐labeled DNA. The gel was scanned to detect AF488‐ (left) or AF647‐labeled DNA (right). Lane 1 shows migration of tetrasomes (tetra). Appearance of nucleosomes (nucl) is indicated.BTo quantify nucleosome assembly, the intensity of the band (nucl) was plotted as a function of H2A–H2B concentration. To measure non‐specific DNA binding, the disappearance of the free DNA AF647 band was plotted. The value in the absence of H2A–H2B (lane 1) was set to 100% intensity. The curve is representative of three independent experiments performed on different days. Standard deviations are indicated.C, DSame experiment as in (A) for the yNap1_IF1–H2A–H2B and for the yNap1c–H2A–H2B.ENucleosome assembly rescue assay. Increasing amounts 1 μM–4 μM H2A–H2B were added to 0.7 μM AF488‐labeled H3–H4–DNA tetrasome complexes (lane 1) and resulted in non‐specific aggregation at 2–4 μM H2A–H2B (lanes 2–4). Incubation of 4 μM H2A–H2B with increasing amounts 0.31 μM–5 μM of yNap1 (lanes 5–9) resulted in a rescue of nucleosome assembly and prevented non‐specific DNA binding of H2A–H2B. The yNap1_HBR1 + 2 (lanes 10–14) show a defect in this rescue assay.FSame experiment as in (E) for the yNap1_HBR1 (lanes 5–9) and yNap1_HBR2 (lanes 10–14).GSame experiment as in (E) for the yNap1c (lanes 5–9) and yNap1_IF1 (lanes 10–14).HQuantification of the “rescue” activity of yNap1 variants. Each data point is the mean of three independent measurements. The error bar indicates the standard deviation of the data.ITo show the composition of the salt assembled tetrasome and nucleosome preparations, the same sample was run on a native and SDS–PAGE gel.JExpression controls for yNap1 mutants. Nap1 pool in BY4741, Nap1Δ, and Nap1 mutants compared with TBP (TATA binding protein) as loading control.
To investigate the impact of the HBR mutants on yNap1 function, we added excess H2A–H2B to tetrasomes to investigate how the yNap1 variants compete with non‐specifically DNA‐bound H2A–H2B to “rescue” nucleosome assembly. As expected, the addition of excess H2A–H2B resulted in non‐specific aggregation (Fig EV4E and F; lanes 2–4). Subsequent addition of wild‐type yNap1 resolved such non‐specific DNA binding (Fig EV4E; lanes 5–9). yNap1 constructs containing the HBR1, HBR2, or HBR1 + 2 mutations were deficient in resolving such aggregates as indicated by the higher concentration required to restore nucleosome assembly (Fig EV4E and F). Thus in contrast to the GST‐pulldown assay which measures direct yNap1–H2A–H2B binding and where only the HBR1 + 2 mutant showed a defect (Fig 3D), in this competition assay, where the yNap1 variants compete non‐specifically bound H2A–H2B from DNA to restore nucleosome assembly, we also observed a phenotype for the HBR1 and HBR2 mutants. yNap1c and yNap1–IF1 also showed decreased rescue activity indicating that the flexible N‐ and C‐terminal regions and oligomerization are required for efficient chaperone activity (Fig EV4G). To qualitatively compare the “rescue” activity of the different yNap1 variants, the fluorescence intensity of the nucleosome band at 1 μM concentration of yNap1 was quantified (Fig EV4H). To monitor integrity of the tetrasome and nucleosome preparations, the same samples were run on a native and SDS–PAGE gel and confirm the composition and stoichiometry of these preparations (Fig EV4I). In order to further examine the physiological relevance, we analyzed the impact of yNap1 mutants on nucleosome assembly in vivo. As yNap1 deletion results in accumulation of nucleosome assembly intermediates (Fig 1), we complemented the nap1Δ strain with yNap1 variants and characterized their impact on nucleosomal structures using MNase digests (Appendix Table S1). Western blot analysis of the mutants confirmed that all proteins were expressed to wild‐type levels (Fig EV4J). Like wild‐type yNap1, the HBR1 and HBR2 mutant fully or partially complemented the nap1Δ strain (Fig 6C and D, comparing mononucleosomes). The HBR1 + 2 mutants were increasingly defective, as expected from the binding data. In contrast, the IF1 mutant, which showed only modest impact on H2A–H2B binding in vitro, showed the greatest defect on nucleosome assembly. The impact was stronger than that seen in the nap1Δ strain, indicative of a dominant negative effect. Together, these data show the functional importance of H2A–H2B engagement of yNap1 and histone binding induced oligomerization of the yNap1–H2A–H2B complex.

Nap1 is required for nucleosome assembly in vivo
To determine the contribution of yNap1 to chromatin assembly in vivo, we performed an MNase digestion assay on chromatin from the wild‐type BY4741 (WT) and a yNap1 deletion strain nap1Δ. Gel electrophoresis of chromatin, partially digested with MNase, showed a nucleosomal ladder (Fig 1A). The distance between each band corresponds to the distance between the most distal nucleosome ends within an MNase‐cleaved array of an integral number of nucleosomes. For oligonucleosomes, the size of each band includes the sum of the nucleosome core DNA lengths plus linker lengths (147 bp + 20 bp) (Fig 1A). Consequently, the interband distance corresponds to a population average spacing between adjacent nucleosomes.Figure 1Role of yNap1 in nucleosome assembly on a genomic scale
Analysis of MNase‐digested chromatin DNA fragments from wild‐type (WT, sizes indicated in black) and nap1Δ (N, sizes indicated in red) cells by agarose gel electrophoresis. Diagram illustrating reduction in array length in nap1Δ cells.Frequency distribution of nucleosomal DNA size (WT in gray fill with nap1Δ in blue traces) from H3 ChIP‐seq.Calculation of nucleosome fuzziness as the standard deviation of nucleosome (or tag) positions. See also Appendix Fig S1.
When compared to BY4741 (WT), the population distribution of nucleosomal sizes in the nap1Δ strain was more enriched with smaller DNA fragments, indicating that linker regions were more accessible to MNase. The mononucleosome‐sized MNase fragments electrophoresed in a smaller size range (~145 bp) with respect to the wild type (~170 bp) (Fig 1A, Appendix Fig S1A), indicating that yNap1 has an impact on nucleosome assembly in vivo. This shorter fragment size is more consistent with an assembly intermediate such as a hexasome rather than a full nucleosome, although other types of nucleosomal and subnucleosomal structures are not excluded.MNase digestions indicate that the nucleosomal arrays maintained approximate wild‐type spacing (or ~165 bp interband distance) to the extent that can be measured by gel electrophoresis, indicating that nucleosome spacing in bulk chromatin was generally unaltered in the nap1Δ strain. However, the di‐ and trinucleosome arrays were shorter by a constant ~25 bp in the nap1Δ strain. We interpret the fixed size reduction as follows: MNase is expected to cleave more frequently where linkers are longer, and this is expected to occur in the nap1Δ strain where H2A–H2B is lost (i.e., linker DNA plus the nucleosomal DNA that is vacated by H2A–H2B). Having additional cleavage opportunities where H2A–H2B are lost produces the fixed size reduction in the array. If H2A–H2B assembly into nucleosomes is dynamic even in the nap1Δ strain (but partially shifted toward dissociation), then nucleosomes that exist within cleaved arrays are likely to be full nucleosomes, since their spacing appears to be unchanged. In other words, spacing is set by the predominance of full nucleosomes, with cleavage occurring where there is a relatively infrequent dissociation of H2A–H2B.To further investigate the impact of yNap1 on chromatin assembly, we mapped nucleosome positions on a genomewide scale. This assay involves formaldehyde‐cross‐linking and chromatin fragmentation to mononucleosomes using MNase followed by ChIP. DNA fragments that immunoprecipitate with a H3 antibody were then detected by deep sequencing. To take into account the underlying intrinsic nucleosome occupancy levels, we first measured the overall histone levels by immunoblotting (Appendix Fig S1B). These data showed that the overall histone levels between the wild type and nap1Δ strain were similar. Therefore, we normalized the total tag counts in both samples to be equal. In these single‐read sequencing experiments, nucleosome sizes (including subnucleosomes) were deduced by pairing adjacent peaks located on opposite strands and in the 3′ direction. Confirming the in‐gel observation, these structures were approximately hexasomal in size in the nap1Δ strain (Fig 1B, ~100–130 bp range, compared to ~150 bp for wild type). In this population analysis, they also had fuzzier locations (higher standard deviation associated with peak calling) (Fig 1C). This suggests that the hexasomal‐sized nucleosomes were positionally more dispersed than full nucleosomes. Accumulation of these putative hexasomes indicates that yNap1 preferentially deposits a second copy of H2A–H2B dimers into nucleosomal arrays. Together, these data indicate that yNap1 contributes to nucleosome assembly in vivo.

Nap1 is required for nucleosome assembly in vivo

To determine the contribution of yNap1 to chromatin assembly in vivo, we performed an MNase digestion assay on chromatin from the wild‐type BY4741 (WT) and a yNap1 deletion strain nap1Δ. Gel electrophoresis of chromatin, partially digested with MNase, showed a nucleosomal ladder (Fig 1A). The distance between each band corresponds to the distance between the most distal nucleosome ends within an MNase‐cleaved array of an integral number of nucleosomes. For oligonucleosomes, the size of each band includes the sum of the nucleosome core DNA lengths plus linker lengths (147 bp + 20 bp) (Fig 1A). Consequently, the interband distance corresponds to a population average spacing between adjacent nucleosomes.

Role of yNap1 in nucleosome assembly on a genomic scale

Analysis of MNase‐digested chromatin DNA fragments from wild‐type (WT, sizes indicated in black) and nap1Δ (N, sizes indicated in red) cells by agarose gel electrophoresis. Diagram illustrating reduction in array length in nap1Δ cells.Frequency distribution of nucleosomal DNA size (WT in gray fill with nap1Δ in blue traces) from H3 ChIP‐seq.Calculation of nucleosome fuzziness as the standard deviation of nucleosome (or tag) positions. See also Appendix Fig S1.

Analysis of MNase‐digested chromatin DNA fragments from wild‐type (WT, sizes indicated in black) and nap1Δ (N, sizes indicated in red) cells by agarose gel electrophoresis. Diagram illustrating reduction in array length in nap1Δ cells.

Frequency distribution of nucleosomal DNA size (WT in gray fill with nap1Δ in blue traces) from H3 ChIP‐seq.

Calculation of nucleosome fuzziness as the standard deviation of nucleosome (or tag) positions. See also Appendix Fig S1.

When compared to BY4741 (WT), the population distribution of nucleosomal sizes in the nap1Δ strain was more enriched with smaller DNA fragments, indicating that linker regions were more accessible to MNase. The mononucleosome‐sized MNase fragments electrophoresed in a smaller size range (~145 bp) with respect to the wild type (~170 bp) (Fig 1A, Appendix Fig S1A), indicating that yNap1 has an impact on nucleosome assembly in vivo. This shorter fragment size is more consistent with an assembly intermediate such as a hexasome rather than a full nucleosome, although other types of nucleosomal and subnucleosomal structures are not excluded.

MNase digestions indicate that the nucleosomal arrays maintained approximate wild‐type spacing (or ~165 bp interband distance) to the extent that can be measured by gel electrophoresis, indicating that nucleosome spacing in bulk chromatin was generally unaltered in the nap1Δ strain. However, the di‐ and trinucleosome arrays were shorter by a constant ~25 bp in the nap1Δ strain. We interpret the fixed size reduction as follows: MNase is expected to cleave more frequently where linkers are longer, and this is expected to occur in the nap1Δ strain where H2A–H2B is lost (i.e., linker DNA plus the nucleosomal DNA that is vacated by H2A–H2B). Having additional cleavage opportunities where H2A–H2B are lost produces the fixed size reduction in the array. If H2A–H2B assembly into nucleosomes is dynamic even in the nap1Δ strain (but partially shifted toward dissociation), then nucleosomes that exist within cleaved arrays are likely to be full nucleosomes, since their spacing appears to be unchanged. In other words, spacing is set by the predominance of full nucleosomes, with cleavage occurring where there is a relatively infrequent dissociation of H2A–H2B.

To further investigate the impact of yNap1 on chromatin assembly, we mapped nucleosome positions on a genomewide scale. This assay involves formaldehyde‐cross‐linking and chromatin fragmentation to mononucleosomes using MNase followed by ChIP. DNA fragments that immunoprecipitate with a H3 antibody were then detected by deep sequencing. To take into account the underlying intrinsic nucleosome occupancy levels, we first measured the overall histone levels by immunoblotting (Appendix Fig S1B). These data showed that the overall histone levels between the wild type and nap1Δ strain were similar. Therefore, we normalized the total tag counts in both samples to be equal. In these single‐read sequencing experiments, nucleosome sizes (including subnucleosomes) were deduced by pairing adjacent peaks located on opposite strands and in the 3′ direction. Confirming the in‐gel observation, these structures were approximately hexasomal in size in the nap1Δ strain (Fig 1B, ~100–130 bp range, compared to ~150 bp for wild type). In this population analysis, they also had fuzzier locations (higher standard deviation associated with peak calling) (Fig 1C). This suggests that the hexasomal‐sized nucleosomes were positionally more dispersed than full nucleosomes. Accumulation of these putative hexasomes indicates that yNap1 preferentially deposits a second copy of H2A–H2B dimers into nucleosomal arrays. Together, these data indicate that yNap1 contributes to nucleosome assembly in vivo.

Structure of the yNap1–H2A–H2B complexTo understand the molecular basis for how Nap1 chaperones and deposits H2A–H2B into chromatin, we crystallized Saccharomyces cerevisiae Nap1 (yNap1) bound to Xenopus laevis H2A–H2B. Crystals were obtained with the full‐length proteins but diffracted poorly. In the crystal structure of full‐length yNap1 (Park & Luger, 2006), the N‐ and C‐terminal regions are disordered and we were able to improve diffraction to a minimum Bragg spacing of ~6.7 Å by N‐ and C‐terminal truncation of yNap1 to residues 75–365 and N‐terminal truncation of the histones H2A (14–130) and H2B (28–126) (Fig 2A). This core domain of yNAP1 (yNap1c) is sufficient to bind the H2A–H2B dimer in solution as well as to assemble chromatin from DNA and histone components (Fujii‐Nakata et al, 1992; Park et al, 2005). The structure was determined by molecular replacement with the individual structures of the yNap1c homodimer and the histone H2A–H2B heterodimer (Fig 2B and C) (Park et al, 2008). Molecular replacement without H2A–H2B unambiguously showed the presence of the missing histones with an overall fold that was clearly visible (Figs 2 and EV1A and B). Recent method advances allow low‐resolution refinement and improvement of the low‐resolution model and electron density map by deriving deformable elastic network (DEN) restraints from known high‐resolution models (Schroder et al, 2010). Parameter optimization for DEN refinement covered 510 individual computations and allowed us to reduce the Rfree to 0.31 while maintaining an overall good coordinate accuracy (Table EV1).Figure 2Crystal structure of the yNap1–H2A–H2B complex
Domain architecture of yNap1, H2A, and H2B. The Histone binding region (HBR1 and HBR2) of yNap1 and the yNap1 binding region (NBR1 and NBR2) of H2A are indicated. Regions involved in higher‐order oligomerization are shown (IF1 and IF2).Ribbon view of the asymmetric crystallographic unit containing a (yNap1c2–H2AΔ14–H2BΔ28)6 assembly. For clarity, only layer 1 of the complex is shown. Layer 1 contains three protomers A–C that are related by two perpendicular non‐crystallographic dyads ().Side view of the complex. The layer 2 omitted from panel (B) is shown in gray. The yNap1 dimer is shown in green and yellow, H2A in red, and H2B in blue. See also Fig EV1.View of the yNap1–H2A–H2B protomer complex with labeling of secondary structure elements.
Figure EV1DNA–H2A–H2B binding and electrostatics of the yNap1–H2A–H2B complexRelated to Fig 2.
A, B2F0–Fc omit electron density map for the region comprising H2A–H2B contoured at 1σ in absence (A) or presence (B) of H2A/H2B in ribbon representation. Similar views as in Fig 2A–C are shown.C–EDNA binding of the H2A–H2B heterodimer.FView of the ternary complex with H2A–H2B in ribbon representation. APBS‐calculated electrostatics (−7 kT to +7 kT) were mapped onto yNap1 dimer surface.GHBR1.HHBR2.
The low‐resolution diffraction of the crystals may be reflective of the dynamic oligomerization of the yNap1c2–H2A–H2B complex in solution; however, the availability of the individual yNap1c and H2A–H2B structures effectively provides atomic‐resolution details to the structure of the complex. Previous low‐resolution hydrogen–deuterium exchange experiments suggested that a yNap1 dimer accommodates two copies of H2A–H2B in a tetrameric conformation (D'Arcy et al, 2013). Instead, we found that only a single H2A–H2B heterodimer can be accommodated in the binding pocket of a Nap1 dimer. Six copies of this yNap1c2–H2A–H2B complex form the asymmetric crystallographic unit, which belong to space group P21 (Table EV1).The yNap1 dimer is arranged in an antiparallel fashion such that the globular domains are positioned on opposite ends of the dimer (Park & Luger, 2006). The concave face of the dome‐shaped yNap1 homodimer constitutes the binding site for the H2A–H2B heterodimer. This protein–protein interface is mediated by both yNap1 subunits and almost exclusively by H2A and results in the burial of ~761 Å2 of surface area. The H2A–H2B heterodimer retains the configuration as seen in the nucleosome structure (Luger et al, 1997). In the nucleosome, the H2A–H2B heterodimer binds 2.5 turns of DNA double helix, which arcs around them along their long axes to generate a ~140° bend (Figs 2 and EV1C–E). We find that the yNap1 dimer mimics the structure and electrostatics of nucleosomal DNA and provides a concave acidic binding surface for H2A–H2B (Figs 2D and EV1F–H). As the H2A–H2B heterodimer is positioned asymmetrically in the electronegative binding cavity, yNap1 engages two of the three DNA binding surfaces of H2A–H2B with the majority of contacts toward H2A (Fig 3A). In the center of the heterodimer, contacts occur between H2A α1 and histone binding region 1 (HBR1) of yNap1, which spans the C‐terminus of α4 to the N‐terminus of α6 (residues 194–205) (Fig 3B). The unique N‐terminal helix αN of H2A (residues 16–21) complements this binding interface by packing against E310 of yNap1, which is located in the region C‐terminal to the disordered β‐hairpin (β5–β6). The second DNA binding interface of H2A–H2B, containing binding loops L1 of H2B and L2 of H2A, packs against histone binding region 2 (HBR2), which comprises the N‐terminus of helix α8 of the second yNap1 monomer (residues 328–336) (Fig 3C). The majority of contacts occur between H2A L2 residues K75–R81 and E328–E336 of helix α8 of yNap1. The positive dipole moments of helix α2 of H2B and α3 of H2A, which are directed toward the DNA phosphodiester backbone in context of the nucleosome, are capped by this negatively charged patch on helix α8 of yNap1. The third DNA binding interface of the H2A–H2B heterodimer containing the H2A L1 and H2B L2 loops does not engage yNap1 (Fig 3A).Figure 3Interactions of yNap1 with the basic DNA binding surface of the H2A–H2B heterodimer
Overview of the yNap1c2–H2AΔ14–H2BΔ28 complex.Residues R17, R20, R29, R32, R35, and K36 of H2A make contacts with the acidic HBR1 in yNap1 comprising residues D201, D205, and E310.Residues K75, R77, and R81 of H2A make polar contacts with the HBR2 comprising residues E332, D333, and E339 in α8 of yNap1.
In vitro GST‐pulldown assay showing the role of residues in the HBR interfaces in H2A–H2B binding. The mutant‐labeled HBR1 (lane 6) contains the triple mutation D201R, D205R, and E310R, and HBR2 (lane 12), the triple mutations E332, D333, and E336. HBR1 + 2 (lane 13) contains all six mutations. The mutant‐labeled HBR1 + 2A (lane 14) has alanine replacements in the six HBR residues. Top panel: Bound. Bottom panel: Input.Disruption of the Nap1–histone interaction by H2A mutation. Bottom panel: Inputs of the coexpressed proteins (soluble extract) and top panel the material bound to the glutathione affinity column were analyzed by SDS–PAGE and Coomassie staining.Controls showing input (bottom) and bound (top) fractions. See also Fig EV2.

Source data are available online for this figure.
To assess the contribution of the HBR interfaces to yNap1–H2A–H2B complex formation, we created mutants targeting conserved, surface‐exposed residues in the histone binding interfaces and tested H2A–H2B binding using a GST‐pulldown assay (data are summarized in Appendix Table S2). Single, double, or triple (D201R, D205R, E310R) mutations in the HBR1 interface did not abolish H2A–H2B binding (Fig 3D, lanes 1–6). In the HBR2 interface, single, double, or triple (E332, D333, E336) mutation or replacement of residues 319–337 with a single Gly amino acid residue, a modification ablating the segment of α8 comprising the HBR2 region (ΔHBR2), also did not abolish H2A–H2B binding (Fig 3D, lanes 7–12). However, when both HBR interfaces contained charge reversal or alanine mutations, we observed reduced H2A–H2B binding (Fig 3D, lanes 13–15). We therefore conclude that both HBR interfaces contribute synergistically to binding of H2A–H2B. Residues in the HBR interfaces are conserved in Nap1 from yeast to humans but not in Vps75 and SET indicating that these structurally related histone chaperones have evolved different modes of histone binding (Fig EV2A–D).Figure EV2Sequence conservationRelated to Fig 3. 
The yNap1 dimer is color‐coded according to the sequence conservation.HBR2.HBR1. The higher the conservation, the darker the color. Similar views as in Fig 3A–C are shown.Nap1‐like histone chaperons with highlighted conserved residues. Secondary structure elements are shown above the amino acid sequences. The histone binding region 1 or 2 (HBR1 or HBR2) as well as the oligomerization interface (IF1) of yNap1 are highlighted in green. Key residues in the HBR1 and HBR2 (●) as well as IF1 (▵) are indicated. The structure‐based sequence alignment was performed using Pymol (Schrodinger, 2010) and ClustalW (Thompson et al, 1994). The figure was generated using Espript (Gouet et al, 2003).
We also tested the impact of mutations on the ability to copurify overexpressed yNap1 and H2A–H2B from E. coli cells. GST‐yNap1 wild type (Fig 3E, lane 1) but not the yNap1 HBR1 + 2 mutant was able to copurify H2A–H2B on the glutathione affinity resin (Fig 3F, lane 2). As expected, expression of yNap1 alone did not show a band at the expected position for H2A–H2B (Fig 3F, lane 1) and H2A–H2B alone was expressed but did not bind to the resin (Fig 3E, lane 14). To assess the contribution of residues at the histone side of the interface, we introduced mutations in the Nap1 binding regions NBR1 and NBR2 of H2A. Mutation of single amino acid residues in NBR1 or NBR2 of H2A (Fig 3E, lanes 2–6) or mutation of the NBR2 region alone was not sufficient to fully abolish binding (Fig 3E, lane 11). However, variants containing several mutations in NBR1 and NBR2 reduced H2A–H2B binding (Fig 3E, lanes 7–13). In particular, the triple NBR1 mutant (R30E, R33E, K37E) or a sextuple mutant NBR1 + 2 that also has the NBR2 mutations K75E, R78E, R82E showed drastically reduced binding (Fig 3E, lanes 12–13). Variants containing combinations of mutations in both interfaces generally showed greatly reduced histone binding (Fig 3E, lane 7–9). A mutant designed to disrupt the H2A oligomerization interface IF2 (N94E, G98D, R99D, T101D) reduced but did not fully abolish H2A–H2B binding, indicating that oligomerization through this interface is not essential for complex formation but contributes to binding avidity (Fig 3E, lane 10). Together, mutations in yNap1 and H2A that were designed to decrease histone H2A–H2B binding do indeed weaken the yNap1–histone H2A–H2B interaction. A charge‐swap experiment provides further proof for the validity of the interface: In the yNap1 sextuple mutant HBR1 + 2 and the H2A sextuple mutant NBR1 + 2, the charged interfacial residues are swapped. While each mutant tested against the wild‐type counterpart loses binding (Fig 3D, lane 10 and Fig 3E, lane 13), binding is restored, when these mutants are tested against each other (Fig 3F, lane 3).

Structure of the yNap1–H2A–H2B complex

To understand the molecular basis for how Nap1 chaperones and deposits H2A–H2B into chromatin, we crystallized Saccharomyces cerevisiae Nap1 (yNap1) bound to Xenopus laevis H2A–H2B. Crystals were obtained with the full‐length proteins but diffracted poorly. In the crystal structure of full‐length yNap1 (Park & Luger, 2006), the N‐ and C‐terminal regions are disordered and we were able to improve diffraction to a minimum Bragg spacing of ~6.7 Å by N‐ and C‐terminal truncation of yNap1 to residues 75–365 and N‐terminal truncation of the histones H2A (14–130) and H2B (28–126) (Fig 2A). This core domain of yNAP1 (yNap1c) is sufficient to bind the H2A–H2B dimer in solution as well as to assemble chromatin from DNA and histone components (Fujii‐Nakata et al, 1992; Park et al, 2005). The structure was determined by molecular replacement with the individual structures of the yNap1c homodimer and the histone H2A–H2B heterodimer (Fig 2B and C) (Park et al, 2008). Molecular replacement without H2A–H2B unambiguously showed the presence of the missing histones with an overall fold that was clearly visible (Figs 2 and EV1A and B). Recent method advances allow low‐resolution refinement and improvement of the low‐resolution model and electron density map by deriving deformable elastic network (DEN) restraints from known high‐resolution models (Schroder et al, 2010). Parameter optimization for DEN refinement covered 510 individual computations and allowed us to reduce the Rfree to 0.31 while maintaining an overall good coordinate accuracy (Table EV1).

Crystal structure of the yNap1–H2A–H2B complex

Domain architecture of yNap1, H2A, and H2B. The Histone binding region (HBR1 and HBR2) of yNap1 and the yNap1 binding region (NBR1 and NBR2) of H2A are indicated. Regions involved in higher‐order oligomerization are shown (IF1 and IF2).Ribbon view of the asymmetric crystallographic unit containing a (yNap1c2–H2AΔ14–H2BΔ28)6 assembly. For clarity, only layer 1 of the complex is shown. Layer 1 contains three protomers A–C that are related by two perpendicular non‐crystallographic dyads ().Side view of the complex. The layer 2 omitted from panel (B) is shown in gray. The yNap1 dimer is shown in green and yellow, H2A in red, and H2B in blue. See also Fig EV1.View of the yNap1–H2A–H2B protomer complex with labeling of secondary structure elements.

Domain architecture of yNap1, H2A, and H2B. The Histone binding region (HBR1 and HBR2) of yNap1 and the yNap1 binding region (NBR1 and NBR2) of H2A are indicated. Regions involved in higher‐order oligomerization are shown (IF1 and IF2).

Ribbon view of the asymmetric crystallographic unit containing a (yNap1c2–H2AΔ14–H2BΔ28)6 assembly. For clarity, only layer 1 of the complex is shown. Layer 1 contains three protomers A–C that are related by two perpendicular non‐crystallographic dyads ().

Side view of the complex. The layer 2 omitted from panel (B) is shown in gray. The yNap1 dimer is shown in green and yellow, H2A in red, and H2B in blue. See also Fig EV1.

View of the yNap1–H2A–H2B protomer complex with labeling of secondary structure elements.

DNA–H2A–H2B binding and electrostatics of the yNap1–H2A–H2B complex

Related to Fig 2.
A, B2F0–Fc omit electron density map for the region comprising H2A–H2B contoured at 1σ in absence (A) or presence (B) of H2A/H2B in ribbon representation. Similar views as in Fig 2A–C are shown.C–EDNA binding of the H2A–H2B heterodimer.FView of the ternary complex with H2A–H2B in ribbon representation. APBS‐calculated electrostatics (−7 kT to +7 kT) were mapped onto yNap1 dimer surface.GHBR1.HHBR2.

2F0–Fc omit electron density map for the region comprising H2A–H2B contoured at 1σ in absence (A) or presence (B) of H2A/H2B in ribbon representation. Similar views as in Fig 2A–C are shown.

DNA binding of the H2A–H2B heterodimer.

View of the ternary complex with H2A–H2B in ribbon representation. APBS‐calculated electrostatics (−7 kT to +7 kT) were mapped onto yNap1 dimer surface.

The low‐resolution diffraction of the crystals may be reflective of the dynamic oligomerization of the yNap1c2–H2A–H2B complex in solution; however, the availability of the individual yNap1c and H2A–H2B structures effectively provides atomic‐resolution details to the structure of the complex. Previous low‐resolution hydrogen–deuterium exchange experiments suggested that a yNap1 dimer accommodates two copies of H2A–H2B in a tetrameric conformation (D'Arcy et al, 2013). Instead, we found that only a single H2A–H2B heterodimer can be accommodated in the binding pocket of a Nap1 dimer. Six copies of this yNap1c2–H2A–H2B complex form the asymmetric crystallographic unit, which belong to space group P21 (Table EV1).

The yNap1 dimer is arranged in an antiparallel fashion such that the globular domains are positioned on opposite ends of the dimer (Park & Luger, 2006). The concave face of the dome‐shaped yNap1 homodimer constitutes the binding site for the H2A–H2B heterodimer. This protein–protein interface is mediated by both yNap1 subunits and almost exclusively by H2A and results in the burial of ~761 Å2 of surface area. The H2A–H2B heterodimer retains the configuration as seen in the nucleosome structure (Luger et al, 1997). In the nucleosome, the H2A–H2B heterodimer binds 2.5 turns of DNA double helix, which arcs around them along their long axes to generate a ~140° bend (Figs 2 and EV1C–E). We find that the yNap1 dimer mimics the structure and electrostatics of nucleosomal DNA and provides a concave acidic binding surface for H2A–H2B (Figs 2D and EV1F–H). As the H2A–H2B heterodimer is positioned asymmetrically in the electronegative binding cavity, yNap1 engages two of the three DNA binding surfaces of H2A–H2B with the majority of contacts toward H2A (Fig 3A). In the center of the heterodimer, contacts occur between H2A α1 and histone binding region 1 (HBR1) of yNap1, which spans the C‐terminus of α4 to the N‐terminus of α6 (residues 194–205) (Fig 3B). The unique N‐terminal helix αN of H2A (residues 16–21) complements this binding interface by packing against E310 of yNap1, which is located in the region C‐terminal to the disordered β‐hairpin (β5–β6). The second DNA binding interface of H2A–H2B, containing binding loops L1 of H2B and L2 of H2A, packs against histone binding region 2 (HBR2), which comprises the N‐terminus of helix α8 of the second yNap1 monomer (residues 328–336) (Fig 3C). The majority of contacts occur between H2A L2 residues K75–R81 and E328–E336 of helix α8 of yNap1. The positive dipole moments of helix α2 of H2B and α3 of H2A, which are directed toward the DNA phosphodiester backbone in context of the nucleosome, are capped by this negatively charged patch on helix α8 of yNap1. The third DNA binding interface of the H2A–H2B heterodimer containing the H2A L1 and H2B L2 loops does not engage yNap1 (Fig 3A).

Interactions of yNap1 with the basic DNA binding surface of the H2A–H2B heterodimer

Overview of the yNap1c2–H2AΔ14–H2BΔ28 complex.Residues R17, R20, R29, R32, R35, and K36 of H2A make contacts with the acidic HBR1 in yNap1 comprising residues D201, D205, and E310.Residues K75, R77, and R81 of H2A make polar contacts with the HBR2 comprising residues E332, D333, and E339 in α8 of yNap1.
In vitro GST‐pulldown assay showing the role of residues in the HBR interfaces in H2A–H2B binding. The mutant‐labeled HBR1 (lane 6) contains the triple mutation D201R, D205R, and E310R, and HBR2 (lane 12), the triple mutations E332, D333, and E336. HBR1 + 2 (lane 13) contains all six mutations. The mutant‐labeled HBR1 + 2A (lane 14) has alanine replacements in the six HBR residues. Top panel: Bound. Bottom panel: Input.Disruption of the Nap1–histone interaction by H2A mutation. Bottom panel: Inputs of the coexpressed proteins (soluble extract) and top panel the material bound to the glutathione affinity column were analyzed by SDS–PAGE and Coomassie staining.Controls showing input (bottom) and bound (top) fractions. See also Fig EV2.

Overview of the yNap1c2–H2AΔ14–H2BΔ28 complex.

Residues R17, R20, R29, R32, R35, and K36 of H2A make contacts with the acidic HBR1 in yNap1 comprising residues D201, D205, and E310.

Residues K75, R77, and R81 of H2A make polar contacts with the HBR2 comprising residues E332, D333, and E339 in α8 of yNap1.

In vitro GST‐pulldown assay showing the role of residues in the HBR interfaces in H2A–H2B binding. The mutant‐labeled HBR1 (lane 6) contains the triple mutation D201R, D205R, and E310R, and HBR2 (lane 12), the triple mutations E332, D333, and E336. HBR1 + 2 (lane 13) contains all six mutations. The mutant‐labeled HBR1 + 2A (lane 14) has alanine replacements in the six HBR residues. Top panel: Bound. Bottom panel: Input.

Disruption of the Nap1–histone interaction by H2A mutation. Bottom panel: Inputs of the coexpressed proteins (soluble extract) and top panel the material bound to the glutathione affinity column were analyzed by SDS–PAGE and Coomassie staining.

Controls showing input (bottom) and bound (top) fractions. See also Fig EV2.

Source data are available online for this figure.

To assess the contribution of the HBR interfaces to yNap1–H2A–H2B complex formation, we created mutants targeting conserved, surface‐exposed residues in the histone binding interfaces and tested H2A–H2B binding using a GST‐pulldown assay (data are summarized in Appendix Table S2). Single, double, or triple (D201R, D205R, E310R) mutations in the HBR1 interface did not abolish H2A–H2B binding (Fig 3D, lanes 1–6). In the HBR2 interface, single, double, or triple (E332, D333, E336) mutation or replacement of residues 319–337 with a single Gly amino acid residue, a modification ablating the segment of α8 comprising the HBR2 region (ΔHBR2), also did not abolish H2A–H2B binding (Fig 3D, lanes 7–12). However, when both HBR interfaces contained charge reversal or alanine mutations, we observed reduced H2A–H2B binding (Fig 3D, lanes 13–15). We therefore conclude that both HBR interfaces contribute synergistically to binding of H2A–H2B. Residues in the HBR interfaces are conserved in Nap1 from yeast to humans but not in Vps75 and SET indicating that these structurally related histone chaperones have evolved different modes of histone binding (Fig EV2A–D).

Sequence conservation

Related to Fig 3. 
The yNap1 dimer is color‐coded according to the sequence conservation.HBR2.HBR1. The higher the conservation, the darker the color. Similar views as in Fig 3A–C are shown.Nap1‐like histone chaperons with highlighted conserved residues. Secondary structure elements are shown above the amino acid sequences. The histone binding region 1 or 2 (HBR1 or HBR2) as well as the oligomerization interface (IF1) of yNap1 are highlighted in green. Key residues in the HBR1 and HBR2 (●) as well as IF1 (▵) are indicated. The structure‐based sequence alignment was performed using Pymol (Schrodinger, 2010) and ClustalW (Thompson et al, 1994). The figure was generated using Espript (Gouet et al, 2003).

The yNap1 dimer is color‐coded according to the sequence conservation.

HBR1. The higher the conservation, the darker the color. Similar views as in Fig 3A–C are shown.

Nap1‐like histone chaperons with highlighted conserved residues. Secondary structure elements are shown above the amino acid sequences. The histone binding region 1 or 2 (HBR1 or HBR2) as well as the oligomerization interface (IF1) of yNap1 are highlighted in green. Key residues in the HBR1 and HBR2 (●) as well as IF1 (▵) are indicated. The structure‐based sequence alignment was performed using Pymol (Schrodinger, 2010) and ClustalW (Thompson et al, 1994). The figure was generated using Espript (Gouet et al, 2003).

We also tested the impact of mutations on the ability to copurify overexpressed yNap1 and H2A–H2B from E. coli cells. GST‐yNap1 wild type (Fig 3E, lane 1) but not the yNap1 HBR1 + 2 mutant was able to copurify H2A–H2B on the glutathione affinity resin (Fig 3F, lane 2). As expected, expression of yNap1 alone did not show a band at the expected position for H2A–H2B (Fig 3F, lane 1) and H2A–H2B alone was expressed but did not bind to the resin (Fig 3E, lane 14). To assess the contribution of residues at the histone side of the interface, we introduced mutations in the Nap1 binding regions NBR1 and NBR2 of H2A. Mutation of single amino acid residues in NBR1 or NBR2 of H2A (Fig 3E, lanes 2–6) or mutation of the NBR2 region alone was not sufficient to fully abolish binding (Fig 3E, lane 11). However, variants containing several mutations in NBR1 and NBR2 reduced H2A–H2B binding (Fig 3E, lanes 7–13). In particular, the triple NBR1 mutant (R30E, R33E, K37E) or a sextuple mutant NBR1 + 2 that also has the NBR2 mutations K75E, R78E, R82E showed drastically reduced binding (Fig 3E, lanes 12–13). Variants containing combinations of mutations in both interfaces generally showed greatly reduced histone binding (Fig 3E, lane 7–9). A mutant designed to disrupt the H2A oligomerization interface IF2 (N94E, G98D, R99D, T101D) reduced but did not fully abolish H2A–H2B binding, indicating that oligomerization through this interface is not essential for complex formation but contributes to binding avidity (Fig 3E, lane 10). Together, mutations in yNap1 and H2A that were designed to decrease histone H2A–H2B binding do indeed weaken the yNap1–histone H2A–H2B interaction. A charge‐swap experiment provides further proof for the validity of the interface: In the yNap1 sextuple mutant HBR1 + 2 and the H2A sextuple mutant NBR1 + 2, the charged interfacial residues are swapped. While each mutant tested against the wild‐type counterpart loses binding (Fig 3D, lane 10 and Fig 3E, lane 13), binding is restored, when these mutants are tested against each other (Fig 3F, lane 3).

Oligomerization of the yNap1–H2A–H2B complexOligomerization of various Nap1 orthologs occurs in vivo and under physiological conditions in vitro (Ishimi et al, 1983, 1984; Fujii‐Nakata et al, 1992; Chang et al, 1997; Mosammaparast et al, 2002; McBryant & Peersen, 2004; Toth et al, 2005; Park et al, 2008; Noda et al, 2011; Newman et al, 2012). The biological role of Nap1 oligomerization has so far remained unclear. Depending on the method of analysis, the ionic strength, and concentration used, different oligomerization states of Nap1 dimers have been reported. In size‐exclusion chromatography, purified yNap1 alone or in complex with H2A–H2B elutes as a broad peak indicative of a heterogeneous mixtures of various oligomeric species (Mosammaparast et al, 2002; Park et al, 2008). The crystal structure of yNap1c in the absence of histones revealed an extended β5–β6 hairpin that is involved in crystal packing (Park et al, 2008). The same packing interface is not available in crystals of full‐length yNap1, but mutation of this packing interface reduces yNap1 oligomerization in vitro (Park et al, 2008). In the presence of histones, yNap1 shows similar propensities to oligomerize (Toth et al, 2005; Noda et al, 2011). Xenopus laevis Nap1 in the presence of H2A–H2B or H3–H4 forms heterogeneous ring‐like particles in the 0.5–1.0 MDa size range as visualized by electron microscopy (Newman et al, 2012). The underlying compositional heterogeneity has made it difficult to ascertain the exact stoichiometry and to understand the structural basis and biological role of Nap1–histone oligomerization. We therefore asked whether our crystal structure is relevant for Nap1 oligomerization. The 6 copies of the yNap1c2–H2A–H2B complex in the crystallographic asymmetric unit form two layers, each containing three protomers (A–C). Protomers A and B (A' and B' in layer 2) are related by a ~102° rotation and ~8 Å displacement along the central axis (Fig 4A). Two non‐crystallographic dyads relate protomer C to A and B in each layer, but because of displacement along the central axis, protomers C and C' pack through alternate protein interfaces: Protomer C is located below the plane of protomers A and B, whereas protomer C' is located above A' and B'. Because of the alternating arrangement of layers 1 and 2 in the crystal (Fig 4F), packing of the two layers results in a pseudohelical assembly.Figure 4Oligomerization interfaces
Cartoon illustrating the arrangement of the two layers of the yNap1c2–H2A–H2B oligomer.A ribbon view of the protomers B and C comprising the IF1 interface.Close‐up view of interactions in the IF1 oligomerization interface.Ribbon view of protomers A and A' comprising the interlayer oligomerization interface IF2.Close‐up view of interactions in the IF2 interface.Crystal packing interactions. To illustrate packing, the two layers were moved apart. Packing of the two layers in the crystallographic asymmetric unit is stabilized by two interlayer IF2 interfaces (indicated by two arrows facing each other) between protomers A–A' and B–B'. The IF2 interface between protomers C provides for packing to the neighboring unit cells. H2A is colored in red, H2B in blue, and yNap1 in green and yellow. See also Fig EV3.
To investigate the yNap1–H2A–H2B complexes further, we visualized our preparations by negative‐stain EM. Electron micrographs showed that complexes containing full‐length yNap1 populated a series of oligomeric states (Fig EV3D and E). No such particles were observed in the absence of histones indicating that particle formation is dependent on H2A–H2B. Electron micrographs of yNap1c in complex with H2AΔ14–H2BΔ28 revealed monodisperse particles, about 15 nm in diameter, similar to those observed previously (Newman et al, 2012). Varying structural features indicated different orientations of the complex on the carbon support film (Fig 5A). A single particle reconstruction of the yNap1c2–H2AΔ14–H2BΔ28 complex matched well with the low‐pass filtered projections from the crystal structure showing that the oligomer seen in the crystals is relevant for assembly in solution (Fig 5B and C).Figure EV3yNap1–H2A–H2B oligomerizationRelated to Fig 4. 
ANative mass spectra of yNap1c against varying concentrations of H2AΔ14–H2BΔ28.BNative PAGE assay of yNap1 binding to H2A–H2B (lanes 1–5), yNap1c binding to H2A–H2B (lanes 6–10), or of yNap1_IF1 to H2A–H2B (lanes 11–15).CNative PAGE assay of H2AΔ14_IF2–H2BΔ28 binding to yNap1 (lanes 1–5) or yNap1c (lanes 6–10) or yNap1_IF1 (lanes 11–15). Samples were separated by 6% native PAGE and visualized by Coomassie staining.D, EEM analysis of the full‐length yNap1–H2A–H2B complex.
Figure 5Native mass spectrometry and negative‐stain EM of the yNap1–H2A–H2B particle
AEM analysis of the yNap1c2–H2AΔ14–H2BΔ28 complex. Representative raw image of negatively stained particles.BRepresentative particles (1st row), class averages coming from the projection matching procedure (2nd row) and comparison of the class averages with projections from a final model generated after four projection matching cycles (3rd row).CFrom left to right: Crystal structure of the (yNap1c2–H2A–H2B)6 complex in two different orientations (brown); 60 Å‐filtered structure of the complex (magenta); 3D reconstruction of the complex from negatively stained particles (green) and fit of the crystal structure into the EM map.D–IMass spectra of yNap1 variants in the absence and presence of H2A–H2B. yNap1 in complex in the absence (D) or presence of H2A–H2B (E). yNap1c in the absence (F) or bound to H2A–H2B (G). The oligomerization mutant yNap1_IF1 alone (H) or in complex with H2A_IF2Δ14–H2BΔ28 (I). yNap1 is shown in green circles and H2A–H2B as red and blue asterisks.
To analyze the relevance of the oligomerization interfaces, we prepared a series of yNap1 variants and compared their propensity to oligomerize. Previous analysis by ensemble averaging techniques such as size exclusion chromatography, sucrose gradients, analytical ultracentrifugation, and small‐angle X‐ray scattering did not converge on a unique solution for Nap1 oligomerization and histone binding stoichiometry (Ishimi et al, 1983, 1984; Fujii‐Nakata et al, 1992; Chang et al, 1997; Mosammaparast et al, 2002; McBryant & Peersen, 2004; Toth et al, 2005; Park et al, 2008; Noda et al, 2011; Newman et al, 2012). Instead, we used nanoflow electrospray ionization–mass spectrometry under native conditions, a technique that affords unparalleled mass accuracy and enables unambiguous determination of protein stoichiometry. Mass spectra of yNap1 alone showed three major series of resolved peaks with different charge states corresponding mostly to yNap1 dimers, tetramers, and a minor hexameric fraction (Fig 5D and Table EV2). In the presence of H2A–H2B, we obtained a series of larger complexes corresponding to yNap1 dimers, tetramers, hexamers, and octamers bound to one, two, three, or four copies of H2A–H2B, respectively (Fig 5E and Table EV2). For yNap1c alone, we observed dimers, tetramers, and a hexameric fraction (Fig 5F and Table EV2). In the presence of H2AΔ14–H2BΔ28, we observed assemblies containing up to 6 copies of the yNap1c2–H2A–H2B complex (Fig 5G and Table EV2). Titration of yNap1c with increasing amounts of H2AΔ14–H2BΔ28 revealed oligomers containing up to 6 copies even when concentrations of H2A–H2B were limiting (Fig EV3A). H2A–H2B binding enhanced higher‐order oligomerization, but the flexible histone tails did not appear to contribute to self‐assembly as we observed a similar degree of oligomerization in their presence or absence (Table EV2). Also, the flexible, non‐conserved N‐terminal and highly negatively charged C‐terminal regions of yNap1 did not contribute to oligomerization of isolated yNap1, in agreement with data reported previously (Park et al, 2008). However, in the presence of H2A–H2B, self‐assembly was more pronounced when these flexible tails were absent. In all experiments, and in agreement with our structure, we found that a single H2A–H2B dimer binds to a yNap1 dimer and we could not confirm the stoichiometry proposed by D'Arcy et al (2013). These data confirm that a single yNap1 dimer binds to a single H2A–H2B heterodimer and that this complex self‐assembles into higher‐order oligomers.To investigate the relevance of the oligomerization interfaces for yNap1 function, we performed a set of in vitro and in vivo experiments. We identified and characterized the largest interfaces that drive packing of individual protomers. In layer 1, protomers B and C pack through a polar interface (IF1) that is composed of yNap1 residues from the linker connecting β1 and β2 and the beginning of β3 (Fig 4B and C). A second major interface (IF2) occurs between the two layers and comprises the C‐terminal docking domains of H2A, which involves residues of α3 and αC in protomers A–A' and B–B' (Fig 4D–F). This double‐layered assembly packs into the crystal by forming IF2‐type interactions between protomers C and C' (Fig 4F). To analyze the contribution of the oligomerization interfaces, we mutated four residues, G225D, S237D, T244D, and D246R, in the IF1 interface of yNap1 (yNap1_IF1) and four residues, N94E, G98D, R99D, and T101D, in the IF2 interface of tail‐less H2AΔ14 (H2AA Δ14_IF2). Native mass spectra of yNap1_IF1 showed mostly a dimer with a minor tetrameric fraction (Fig 5H; Table EV2). In the presence of H2AΔ14_IF2–H2BΔ28, we observed mostly a yNap1_IF1 dimer bound to a single histone heterodimer (Fig 5I; Table EV2). The β5–β6 hairpin that comprises a nuclear localization sequence (NLS) and that has been previously implicated in oligomerization of isolated yNap1 (Park et al, 2008) does not appear to contribute to oligomerization of the H2A–H2B‐bound complex: It does not form the same crystal packing interface reported previously and is solvent‐exposed and disordered in our structure. To analyze the contribution of the β‐hairpin, we generated an in‐frame deletion of the β5–β6 region (yNap1cΔ), a modification that does not disrupt structural integrity, and found similar oligomerization properties as yNap1c, while a construct (yNap1cΔ_G225D) containing in addition the IF1 mutation G225D remained dimeric (Table EV2). In the presence of H2A–H2B, yNap1cΔ oligomerized to a similar degree than yNap1c showing that the β5–β6 region does not contribute to oligomerization of the histone‐bound complex. In contrast, the yNap1cΔ_G225D mutant did not oligomerize upon addition of H2A–H2B, thus reconfirming the relevance of the IF1 interface to oligomerization of the yNap12–H2A–H2B complex. More qualitative native PAGE experiments also showed that the IF1 mutant has a reduced propensity to oligomerize compared to yNap1 or yNap1c both with wild‐type H2A–H2B (Fig EV3B) and with tail‐less H2A–H2B containing the IF2 mutation (Fig EV3C). The IF2 interface also contributes to oligomerization as yNap1 and yNap1c showed reduced oligomerization upon addition of H2AΔ14_IF2–H2BΔ28 (Fig EV3C). We conclude that the IF1 and IF2 interfaces are relevant for oligomerization of the yNap12–H2A–H2B complexes and that H2A–H2B binding to yNap1 and oligomerization contributes synergistically to the binding free energy of complex formation. As oligomerization of the yNap12–H2A–H2B complex results in burial of a segment that is required for docking onto H3–H4 tetrasomes, we argue that it is inhibitory for H2A–H2B deposition and nucleosome formation. Such an inhibitory conformation could be relevant for histone transport or storage. The pseudohelical arrangement of the yNap12–H2A–H2B complex probably forms the basis for the dynamic oligomerization that has been observed by us and others (McBryant & Peersen, 2004; Toth et al, 2005; Noda et al, 2011; Newman et al, 2012).

Oligomerization of the yNap1–H2A–H2B complex

Oligomerization of various Nap1 orthologs occurs in vivo and under physiological conditions in vitro (Ishimi et al, 1983, 1984; Fujii‐Nakata et al, 1992; Chang et al, 1997; Mosammaparast et al, 2002; McBryant & Peersen, 2004; Toth et al, 2005; Park et al, 2008; Noda et al, 2011; Newman et al, 2012). The biological role of Nap1 oligomerization has so far remained unclear. Depending on the method of analysis, the ionic strength, and concentration used, different oligomerization states of Nap1 dimers have been reported. In size‐exclusion chromatography, purified yNap1 alone or in complex with H2A–H2B elutes as a broad peak indicative of a heterogeneous mixtures of various oligomeric species (Mosammaparast et al, 2002; Park et al, 2008). The crystal structure of yNap1c in the absence of histones revealed an extended β5–β6 hairpin that is involved in crystal packing (Park et al, 2008). The same packing interface is not available in crystals of full‐length yNap1, but mutation of this packing interface reduces yNap1 oligomerization in vitro (Park et al, 2008). In the presence of histones, yNap1 shows similar propensities to oligomerize (Toth et al, 2005; Noda et al, 2011). Xenopus laevis Nap1 in the presence of H2A–H2B or H3–H4 forms heterogeneous ring‐like particles in the 0.5–1.0 MDa size range as visualized by electron microscopy (Newman et al, 2012). The underlying compositional heterogeneity has made it difficult to ascertain the exact stoichiometry and to understand the structural basis and biological role of Nap1–histone oligomerization. We therefore asked whether our crystal structure is relevant for Nap1 oligomerization. The 6 copies of the yNap1c2–H2A–H2B complex in the crystallographic asymmetric unit form two layers, each containing three protomers (A–C). Protomers A and B (A' and B' in layer 2) are related by a ~102° rotation and ~8 Å displacement along the central axis (Fig 4A). Two non‐crystallographic dyads relate protomer C to A and B in each layer, but because of displacement along the central axis, protomers C and C' pack through alternate protein interfaces: Protomer C is located below the plane of protomers A and B, whereas protomer C' is located above A' and B'. Because of the alternating arrangement of layers 1 and 2 in the crystal (Fig 4F), packing of the two layers results in a pseudohelical assembly.

Oligomerization interfaces

Cartoon illustrating the arrangement of the two layers of the yNap1c2–H2A–H2B oligomer.A ribbon view of the protomers B and C comprising the IF1 interface.Close‐up view of interactions in the IF1 oligomerization interface.Ribbon view of protomers A and A' comprising the interlayer oligomerization interface IF2.Close‐up view of interactions in the IF2 interface.Crystal packing interactions. To illustrate packing, the two layers were moved apart. Packing of the two layers in the crystallographic asymmetric unit is stabilized by two interlayer IF2 interfaces (indicated by two arrows facing each other) between protomers A–A' and B–B'. The IF2 interface between protomers C provides for packing to the neighboring unit cells. H2A is colored in red, H2B in blue, and yNap1 in green and yellow. See also Fig EV3.

Cartoon illustrating the arrangement of the two layers of the yNap1c2–H2A–H2B oligomer.

A ribbon view of the protomers B and C comprising the IF1 interface.

Close‐up view of interactions in the IF1 oligomerization interface.

Ribbon view of protomers A and A' comprising the interlayer oligomerization interface IF2.

Close‐up view of interactions in the IF2 interface.

Crystal packing interactions. To illustrate packing, the two layers were moved apart. Packing of the two layers in the crystallographic asymmetric unit is stabilized by two interlayer IF2 interfaces (indicated by two arrows facing each other) between protomers A–A' and B–B'. The IF2 interface between protomers C provides for packing to the neighboring unit cells. H2A is colored in red, H2B in blue, and yNap1 in green and yellow. See also Fig EV3.

To investigate the yNap1–H2A–H2B complexes further, we visualized our preparations by negative‐stain EM. Electron micrographs showed that complexes containing full‐length yNap1 populated a series of oligomeric states (Fig EV3D and E). No such particles were observed in the absence of histones indicating that particle formation is dependent on H2A–H2B. Electron micrographs of yNap1c in complex with H2AΔ14–H2BΔ28 revealed monodisperse particles, about 15 nm in diameter, similar to those observed previously (Newman et al, 2012). Varying structural features indicated different orientations of the complex on the carbon support film (Fig 5A). A single particle reconstruction of the yNap1c2–H2AΔ14–H2BΔ28 complex matched well with the low‐pass filtered projections from the crystal structure showing that the oligomer seen in the crystals is relevant for assembly in solution (Fig 5B and C).

yNap1–H2A–H2B oligomerization

Related to Fig 4. 
ANative mass spectra of yNap1c against varying concentrations of H2AΔ14–H2BΔ28.BNative PAGE assay of yNap1 binding to H2A–H2B (lanes 1–5), yNap1c binding to H2A–H2B (lanes 6–10), or of yNap1_IF1 to H2A–H2B (lanes 11–15).CNative PAGE assay of H2AΔ14_IF2–H2BΔ28 binding to yNap1 (lanes 1–5) or yNap1c (lanes 6–10) or yNap1_IF1 (lanes 11–15). Samples were separated by 6% native PAGE and visualized by Coomassie staining.D, EEM analysis of the full‐length yNap1–H2A–H2B complex.

Native mass spectra of yNap1c against varying concentrations of H2AΔ14–H2BΔ28.

Native PAGE assay of yNap1 binding to H2A–H2B (lanes 1–5), yNap1c binding to H2A–H2B (lanes 6–10), or of yNap1_IF1 to H2A–H2B (lanes 11–15).

Native PAGE assay of H2AΔ14_IF2–H2BΔ28 binding to yNap1 (lanes 1–5) or yNap1c (lanes 6–10) or yNap1_IF1 (lanes 11–15). Samples were separated by 6% native PAGE and visualized by Coomassie staining.

EM analysis of the full‐length yNap1–H2A–H2B complex.

Native mass spectrometry and negative‐stain EM of the yNap1–H2A–H2B particle

AEM analysis of the yNap1c2–H2AΔ14–H2BΔ28 complex. Representative raw image of negatively stained particles.BRepresentative particles (1st row), class averages coming from the projection matching procedure (2nd row) and comparison of the class averages with projections from a final model generated after four projection matching cycles (3rd row).CFrom left to right: Crystal structure of the (yNap1c2–H2A–H2B)6 complex in two different orientations (brown); 60 Å‐filtered structure of the complex (magenta); 3D reconstruction of the complex from negatively stained particles (green) and fit of the crystal structure into the EM map.D–IMass spectra of yNap1 variants in the absence and presence of H2A–H2B. yNap1 in complex in the absence (D) or presence of H2A–H2B (E). yNap1c in the absence (F) or bound to H2A–H2B (G). The oligomerization mutant yNap1_IF1 alone (H) or in complex with H2A_IF2Δ14–H2BΔ28 (I). yNap1 is shown in green circles and H2A–H2B as red and blue asterisks.

EM analysis of the yNap1c2–H2AΔ14–H2BΔ28 complex. Representative raw image of negatively stained particles.

Representative particles (1st row), class averages coming from the projection matching procedure (2nd row) and comparison of the class averages with projections from a final model generated after four projection matching cycles (3rd row).

From left to right: Crystal structure of the (yNap1c2–H2A–H2B)6 complex in two different orientations (brown); 60 Å‐filtered structure of the complex (magenta); 3D reconstruction of the complex from negatively stained particles (green) and fit of the crystal structure into the EM map.

Mass spectra of yNap1 variants in the absence and presence of H2A–H2B. yNap1 in complex in the absence (D) or presence of H2A–H2B (E). yNap1c in the absence (F) or bound to H2A–H2B (G). The oligomerization mutant yNap1_IF1 alone (H) or in complex with H2A_IF2Δ14–H2BΔ28 (I). yNap1 is shown in green circles and H2A–H2B as red and blue asterisks.

To analyze the relevance of the oligomerization interfaces, we prepared a series of yNap1 variants and compared their propensity to oligomerize. Previous analysis by ensemble averaging techniques such as size exclusion chromatography, sucrose gradients, analytical ultracentrifugation, and small‐angle X‐ray scattering did not converge on a unique solution for Nap1 oligomerization and histone binding stoichiometry (Ishimi et al, 1983, 1984; Fujii‐Nakata et al, 1992; Chang et al, 1997; Mosammaparast et al, 2002; McBryant & Peersen, 2004; Toth et al, 2005; Park et al, 2008; Noda et al, 2011; Newman et al, 2012). Instead, we used nanoflow electrospray ionization–mass spectrometry under native conditions, a technique that affords unparalleled mass accuracy and enables unambiguous determination of protein stoichiometry. Mass spectra of yNap1 alone showed three major series of resolved peaks with different charge states corresponding mostly to yNap1 dimers, tetramers, and a minor hexameric fraction (Fig 5D and Table EV2). In the presence of H2A–H2B, we obtained a series of larger complexes corresponding to yNap1 dimers, tetramers, hexamers, and octamers bound to one, two, three, or four copies of H2A–H2B, respectively (Fig 5E and Table EV2). For yNap1c alone, we observed dimers, tetramers, and a hexameric fraction (Fig 5F and Table EV2). In the presence of H2AΔ14–H2BΔ28, we observed assemblies containing up to 6 copies of the yNap1c2–H2A–H2B complex (Fig 5G and Table EV2). Titration of yNap1c with increasing amounts of H2AΔ14–H2BΔ28 revealed oligomers containing up to 6 copies even when concentrations of H2A–H2B were limiting (Fig EV3A). H2A–H2B binding enhanced higher‐order oligomerization, but the flexible histone tails did not appear to contribute to self‐assembly as we observed a similar degree of oligomerization in their presence or absence (Table EV2). Also, the flexible, non‐conserved N‐terminal and highly negatively charged C‐terminal regions of yNap1 did not contribute to oligomerization of isolated yNap1, in agreement with data reported previously (Park et al, 2008). However, in the presence of H2A–H2B, self‐assembly was more pronounced when these flexible tails were absent. In all experiments, and in agreement with our structure, we found that a single H2A–H2B dimer binds to a yNap1 dimer and we could not confirm the stoichiometry proposed by D'Arcy et al (2013). These data confirm that a single yNap1 dimer binds to a single H2A–H2B heterodimer and that this complex self‐assembles into higher‐order oligomers.

To investigate the relevance of the oligomerization interfaces for yNap1 function, we performed a set of in vitro and in vivo experiments. We identified and characterized the largest interfaces that drive packing of individual protomers. In layer 1, protomers B and C pack through a polar interface (IF1) that is composed of yNap1 residues from the linker connecting β1 and β2 and the beginning of β3 (Fig 4B and C). A second major interface (IF2) occurs between the two layers and comprises the C‐terminal docking domains of H2A, which involves residues of α3 and αC in protomers A–A' and B–B' (Fig 4D–F). This double‐layered assembly packs into the crystal by forming IF2‐type interactions between protomers C and C' (Fig 4F). To analyze the contribution of the oligomerization interfaces, we mutated four residues, G225D, S237D, T244D, and D246R, in the IF1 interface of yNap1 (yNap1_IF1) and four residues, N94E, G98D, R99D, and T101D, in the IF2 interface of tail‐less H2AΔ14 (H2AA Δ14_IF2). Native mass spectra of yNap1_IF1 showed mostly a dimer with a minor tetrameric fraction (Fig 5H; Table EV2). In the presence of H2AΔ14_IF2–H2BΔ28, we observed mostly a yNap1_IF1 dimer bound to a single histone heterodimer (Fig 5I; Table EV2). The β5–β6 hairpin that comprises a nuclear localization sequence (NLS) and that has been previously implicated in oligomerization of isolated yNap1 (Park et al, 2008) does not appear to contribute to oligomerization of the H2A–H2B‐bound complex: It does not form the same crystal packing interface reported previously and is solvent‐exposed and disordered in our structure. To analyze the contribution of the β‐hairpin, we generated an in‐frame deletion of the β5–β6 region (yNap1cΔ), a modification that does not disrupt structural integrity, and found similar oligomerization properties as yNap1c, while a construct (yNap1cΔ_G225D) containing in addition the IF1 mutation G225D remained dimeric (Table EV2). In the presence of H2A–H2B, yNap1cΔ oligomerized to a similar degree than yNap1c showing that the β5–β6 region does not contribute to oligomerization of the histone‐bound complex. In contrast, the yNap1cΔ_G225D mutant did not oligomerize upon addition of H2A–H2B, thus reconfirming the relevance of the IF1 interface to oligomerization of the yNap12–H2A–H2B complex. More qualitative native PAGE experiments also showed that the IF1 mutant has a reduced propensity to oligomerize compared to yNap1 or yNap1c both with wild‐type H2A–H2B (Fig EV3B) and with tail‐less H2A–H2B containing the IF2 mutation (Fig EV3C). The IF2 interface also contributes to oligomerization as yNap1 and yNap1c showed reduced oligomerization upon addition of H2AΔ14_IF2–H2BΔ28 (Fig EV3C). We conclude that the IF1 and IF2 interfaces are relevant for oligomerization of the yNap12–H2A–H2B complexes and that H2A–H2B binding to yNap1 and oligomerization contributes synergistically to the binding free energy of complex formation. As oligomerization of the yNap12–H2A–H2B complex results in burial of a segment that is required for docking onto H3–H4 tetrasomes, we argue that it is inhibitory for H2A–H2B deposition and nucleosome formation. Such an inhibitory conformation could be relevant for histone transport or storage. The pseudohelical arrangement of the yNap12–H2A–H2B complex probably forms the basis for the dynamic oligomerization that has been observed by us and others (McBryant & Peersen, 2004; Toth et al, 2005; Noda et al, 2011; Newman et al, 2012).

Nucleosome assembly activity of yNap1 in vitro and in vivo
To investigate the contribution of yNap1 to nucleosome assembly, we established a quantitative multiple fluorescence relative affinity assay (Man & Stormo, 2001). This method constitutes a competition assay where fluorescently labeled free DNA (AF647) or (H3–H4)2–DNA tetrasome complexes (AF488) compete for the same pool of H2A–H2B, which is provided free or in complex with yNap1 (Fig 6A). This competition assay allowed us to measure the relative affinities of H2A–H2B toward (H3–H4)2–DNA tetrasome complexes and toward free DNA and assess the impact of yNap1 on the relative rates of nucleosome formation and non‐specific DNA binding of H2A–H2B. As a DNA substrate, we used a 147‐bp fragment containing the 601 nucleosome positioning sequence (Lowary & Widom, 1998). The bound and unbound fractions were separated using an electrophoretic mobility shift assay (EMSA) and quantified to derive an apparent dissociation constant of yNap1‐mediated H2A–H2B deposition and nucleosome assembly. We found that yNap1 specifically deposited H2A–H2B onto (H3–H4)2–DNA tetrasome complexes resulting in nucleosome formation (K
d ~940 nM; Fig 6A and B) while preventing non‐specific deposition of H2A–H2B onto free DNA (Fig 6A). In the absence of yNap1, H2A–H2B preferentially bound to the (H3–H4)2–DNA tetrasome complexes to form nucleosomes (K
d ~29 nM) but also interacted non‐specifically with the free DNA at higher concentrations (Fig EV4A and B). The yNap1c and yNap1_IF1 mutants are not affected in this nucleosome assembly assay indicating that the flexible N‐ and C‐terminal regions and oligomerization of yNap1 are not essential for H2A–H2B deposition (Fig EV4C and D). Free H2A–H2B preferentially interacted with tetrasomes and only once these were exhausted from the reaction H2A–H2B bound non‐specifically to the free DNA. In accordance with our structure and previously published data (Andrews et al, 2010), yNap1 is suppressing non‐specific DNA binding of H2A–H2B and thereby facilitates specific deposition of H2A–H2B and nucleosome assembly.Figure 6Nucleosome assembly in vitro and in vivo

Native PAGE gel showing the result of an affinity shift assay scanned to detect AF488‐ (left) or AF647‐labeled DNA (right); 31 nM–4 μM of yNap1–H2A–H2B (lanes 6–13) was incubated with a mixture of 0.8 μM AF488‐labeled H3–H4–DNA tetrasome complexes and 0.8 μM AF647‐labeled DNA (lane 5). Lanes 1–3 show migration of tetrasomes (tetra), nucleosomes (nucl), or free H2A–H2B obtained by salt deposition onto AF647‐labeled DNA.Intensity of the nucleosome band (nucl) is plotted. The value at 4 μM of yNap1–H2A–H2B (lane 13) was set to 100%. For non‐specific DNA binding, the value in the absence of H2A–H2B (lane 5) was set to 100% intensity and disappearance of the free DNA AF647 band was plotted. Each curve is representative of three independent experiments performed on different days. Standard deviations are shown.Analysis of MNase‐digested chromatin DNA fragments from wild‐type (BY4741), nap1Δ, and nap1Δ cells complemented with different yNap1 expression plasmids. Total MNase units per 50 ml of culture are indicated on the bottom of the lanes. Kb+: DNA marker.Positional representation of MNase‐titrated nucleosomal arrays from panel (C), comparing with BY4741 (blue traces) with NAP1 deletion (green traces) and other mutants (HBR1—black, HBR2—red, HBR1 + 2—purple, and IF1—gold traces). Color gradients indicate MNase concentrations (from high to low), and the yellow box represents mononucleosomes. Dashed lines represent nucleosome dyad with respect to BY4741 (WT). Traces are offset vertically for ease of visibility. See also Fig EV4I.
Figure EV4Nucleosome assembly in vitro
Related to Fig 6. 
ANative PAGE gel showing the result of a multiple fluorescence relative affinity shift assay in the absence of yNap1; 31 nM–8 μM H2A–H2B (lanes 4–12) were incubated with a mixture of 0.7 μM AF488‐labeled H3–H4–DNA tetrasome complexes and 0.7 μM AF647‐labeled DNA. The gel was scanned to detect AF488‐ (left) or AF647‐labeled DNA (right). Lane 1 shows migration of tetrasomes (tetra). Appearance of nucleosomes (nucl) is indicated.BTo quantify nucleosome assembly, the intensity of the band (nucl) was plotted as a function of H2A–H2B concentration. To measure non‐specific DNA binding, the disappearance of the free DNA AF647 band was plotted. The value in the absence of H2A–H2B (lane 1) was set to 100% intensity. The curve is representative of three independent experiments performed on different days. Standard deviations are indicated.C, DSame experiment as in (A) for the yNap1_IF1–H2A–H2B and for the yNap1c–H2A–H2B.ENucleosome assembly rescue assay. Increasing amounts 1 μM–4 μM H2A–H2B were added to 0.7 μM AF488‐labeled H3–H4–DNA tetrasome complexes (lane 1) and resulted in non‐specific aggregation at 2–4 μM H2A–H2B (lanes 2–4). Incubation of 4 μM H2A–H2B with increasing amounts 0.31 μM–5 μM of yNap1 (lanes 5–9) resulted in a rescue of nucleosome assembly and prevented non‐specific DNA binding of H2A–H2B. The yNap1_HBR1 + 2 (lanes 10–14) show a defect in this rescue assay.FSame experiment as in (E) for the yNap1_HBR1 (lanes 5–9) and yNap1_HBR2 (lanes 10–14).GSame experiment as in (E) for the yNap1c (lanes 5–9) and yNap1_IF1 (lanes 10–14).HQuantification of the “rescue” activity of yNap1 variants. Each data point is the mean of three independent measurements. The error bar indicates the standard deviation of the data.ITo show the composition of the salt assembled tetrasome and nucleosome preparations, the same sample was run on a native and SDS–PAGE gel.JExpression controls for yNap1 mutants. Nap1 pool in BY4741, Nap1Δ, and Nap1 mutants compared with TBP (TATA binding protein) as loading control.
To investigate the impact of the HBR mutants on yNap1 function, we added excess H2A–H2B to tetrasomes to investigate how the yNap1 variants compete with non‐specifically DNA‐bound H2A–H2B to “rescue” nucleosome assembly. As expected, the addition of excess H2A–H2B resulted in non‐specific aggregation (Fig EV4E and F; lanes 2–4). Subsequent addition of wild‐type yNap1 resolved such non‐specific DNA binding (Fig EV4E; lanes 5–9). yNap1 constructs containing the HBR1, HBR2, or HBR1 + 2 mutations were deficient in resolving such aggregates as indicated by the higher concentration required to restore nucleosome assembly (Fig EV4E and F). Thus in contrast to the GST‐pulldown assay which measures direct yNap1–H2A–H2B binding and where only the HBR1 + 2 mutant showed a defect (Fig 3D), in this competition assay, where the yNap1 variants compete non‐specifically bound H2A–H2B from DNA to restore nucleosome assembly, we also observed a phenotype for the HBR1 and HBR2 mutants. yNap1c and yNap1–IF1 also showed decreased rescue activity indicating that the flexible N‐ and C‐terminal regions and oligomerization are required for efficient chaperone activity (Fig EV4G). To qualitatively compare the “rescue” activity of the different yNap1 variants, the fluorescence intensity of the nucleosome band at 1 μM concentration of yNap1 was quantified (Fig EV4H). To monitor integrity of the tetrasome and nucleosome preparations, the same samples were run on a native and SDS–PAGE gel and confirm the composition and stoichiometry of these preparations (Fig EV4I). In order to further examine the physiological relevance, we analyzed the impact of yNap1 mutants on nucleosome assembly in vivo. As yNap1 deletion results in accumulation of nucleosome assembly intermediates (Fig 1), we complemented the nap1Δ strain with yNap1 variants and characterized their impact on nucleosomal structures using MNase digests (Appendix Table S1). Western blot analysis of the mutants confirmed that all proteins were expressed to wild‐type levels (Fig EV4J). Like wild‐type yNap1, the HBR1 and HBR2 mutant fully or partially complemented the nap1Δ strain (Fig 6C and D, comparing mononucleosomes). The HBR1 + 2 mutants were increasingly defective, as expected from the binding data. In contrast, the IF1 mutant, which showed only modest impact on H2A–H2B binding in vitro, showed the greatest defect on nucleosome assembly. The impact was stronger than that seen in the nap1Δ strain, indicative of a dominant negative effect. Together, these data show the functional importance of H2A–H2B engagement of yNap1 and histone binding induced oligomerization of the yNap1–H2A–H2B complex.

Nucleosome assembly activity of yNap1 in vitro and in vivo

To investigate the contribution of yNap1 to nucleosome assembly, we established a quantitative multiple fluorescence relative affinity assay (Man & Stormo, 2001). This method constitutes a competition assay where fluorescently labeled free DNA (AF647) or (H3–H4)2–DNA tetrasome complexes (AF488) compete for the same pool of H2A–H2B, which is provided free or in complex with yNap1 (Fig 6A). This competition assay allowed us to measure the relative affinities of H2A–H2B toward (H3–H4)2–DNA tetrasome complexes and toward free DNA and assess the impact of yNap1 on the relative rates of nucleosome formation and non‐specific DNA binding of H2A–H2B. As a DNA substrate, we used a 147‐bp fragment containing the 601 nucleosome positioning sequence (Lowary & Widom, 1998). The bound and unbound fractions were separated using an electrophoretic mobility shift assay (EMSA) and quantified to derive an apparent dissociation constant of yNap1‐mediated H2A–H2B deposition and nucleosome assembly. We found that yNap1 specifically deposited H2A–H2B onto (H3–H4)2–DNA tetrasome complexes resulting in nucleosome formation (K
d ~940 nM; Fig 6A and B) while preventing non‐specific deposition of H2A–H2B onto free DNA (Fig 6A). In the absence of yNap1, H2A–H2B preferentially bound to the (H3–H4)2–DNA tetrasome complexes to form nucleosomes (K
d ~29 nM) but also interacted non‐specifically with the free DNA at higher concentrations (Fig EV4A and B). The yNap1c and yNap1_IF1 mutants are not affected in this nucleosome assembly assay indicating that the flexible N‐ and C‐terminal regions and oligomerization of yNap1 are not essential for H2A–H2B deposition (Fig EV4C and D). Free H2A–H2B preferentially interacted with tetrasomes and only once these were exhausted from the reaction H2A–H2B bound non‐specifically to the free DNA. In accordance with our structure and previously published data (Andrews et al, 2010), yNap1 is suppressing non‐specific DNA binding of H2A–H2B and thereby facilitates specific deposition of H2A–H2B and nucleosome assembly.

Nucleosome assembly in vitro and in vivo

Native PAGE gel showing the result of an affinity shift assay scanned to detect AF488‐ (left) or AF647‐labeled DNA (right); 31 nM–4 μM of yNap1–H2A–H2B (lanes 6–13) was incubated with a mixture of 0.8 μM AF488‐labeled H3–H4–DNA tetrasome complexes and 0.8 μM AF647‐labeled DNA (lane 5). Lanes 1–3 show migration of tetrasomes (tetra), nucleosomes (nucl), or free H2A–H2B obtained by salt deposition onto AF647‐labeled DNA.Intensity of the nucleosome band (nucl) is plotted. The value at 4 μM of yNap1–H2A–H2B (lane 13) was set to 100%. For non‐specific DNA binding, the value in the absence of H2A–H2B (lane 5) was set to 100% intensity and disappearance of the free DNA AF647 band was plotted. Each curve is representative of three independent experiments performed on different days. Standard deviations are shown.Analysis of MNase‐digested chromatin DNA fragments from wild‐type (BY4741), nap1Δ, and nap1Δ cells complemented with different yNap1 expression plasmids. Total MNase units per 50 ml of culture are indicated on the bottom of the lanes. Kb+: DNA marker.Positional representation of MNase‐titrated nucleosomal arrays from panel (C), comparing with BY4741 (blue traces) with NAP1 deletion (green traces) and other mutants (HBR1—black, HBR2—red, HBR1 + 2—purple, and IF1—gold traces). Color gradients indicate MNase concentrations (from high to low), and the yellow box represents mononucleosomes. Dashed lines represent nucleosome dyad with respect to BY4741 (WT). Traces are offset vertically for ease of visibility. See also Fig EV4I.

Native PAGE gel showing the result of an affinity shift assay scanned to detect AF488‐ (left) or AF647‐labeled DNA (right); 31 nM–4 μM of yNap1–H2A–H2B (lanes 6–13) was incubated with a mixture of 0.8 μM AF488‐labeled H3–H4–DNA tetrasome complexes and 0.8 μM AF647‐labeled DNA (lane 5). Lanes 1–3 show migration of tetrasomes (tetra), nucleosomes (nucl), or free H2A–H2B obtained by salt deposition onto AF647‐labeled DNA.

Intensity of the nucleosome band (nucl) is plotted. The value at 4 μM of yNap1–H2A–H2B (lane 13) was set to 100%. For non‐specific DNA binding, the value in the absence of H2A–H2B (lane 5) was set to 100% intensity and disappearance of the free DNA AF647 band was plotted. Each curve is representative of three independent experiments performed on different days. Standard deviations are shown.

Analysis of MNase‐digested chromatin DNA fragments from wild‐type (BY4741), nap1Δ, and nap1Δ cells complemented with different yNap1 expression plasmids. Total MNase units per 50 ml of culture are indicated on the bottom of the lanes. Kb+: DNA marker.

Positional representation of MNase‐titrated nucleosomal arrays from panel (C), comparing with BY4741 (blue traces) with NAP1 deletion (green traces) and other mutants (HBR1—black, HBR2—red, HBR1 + 2—purple, and IF1—gold traces). Color gradients indicate MNase concentrations (from high to low), and the yellow box represents mononucleosomes. Dashed lines represent nucleosome dyad with respect to BY4741 (WT). Traces are offset vertically for ease of visibility. See also Fig EV4I.

Nucleosome assembly in vitro

Related to Fig 6. 
ANative PAGE gel showing the result of a multiple fluorescence relative affinity shift assay in the absence of yNap1; 31 nM–8 μM H2A–H2B (lanes 4–12) were incubated with a mixture of 0.7 μM AF488‐labeled H3–H4–DNA tetrasome complexes and 0.7 μM AF647‐labeled DNA. The gel was scanned to detect AF488‐ (left) or AF647‐labeled DNA (right). Lane 1 shows migration of tetrasomes (tetra). Appearance of nucleosomes (nucl) is indicated.BTo quantify nucleosome assembly, the intensity of the band (nucl) was plotted as a function of H2A–H2B concentration. To measure non‐specific DNA binding, the disappearance of the free DNA AF647 band was plotted. The value in the absence of H2A–H2B (lane 1) was set to 100% intensity. The curve is representative of three independent experiments performed on different days. Standard deviations are indicated.C, DSame experiment as in (A) for the yNap1_IF1–H2A–H2B and for the yNap1c–H2A–H2B.ENucleosome assembly rescue assay. Increasing amounts 1 μM–4 μM H2A–H2B were added to 0.7 μM AF488‐labeled H3–H4–DNA tetrasome complexes (lane 1) and resulted in non‐specific aggregation at 2–4 μM H2A–H2B (lanes 2–4). Incubation of 4 μM H2A–H2B with increasing amounts 0.31 μM–5 μM of yNap1 (lanes 5–9) resulted in a rescue of nucleosome assembly and prevented non‐specific DNA binding of H2A–H2B. The yNap1_HBR1 + 2 (lanes 10–14) show a defect in this rescue assay.FSame experiment as in (E) for the yNap1_HBR1 (lanes 5–9) and yNap1_HBR2 (lanes 10–14).GSame experiment as in (E) for the yNap1c (lanes 5–9) and yNap1_IF1 (lanes 10–14).HQuantification of the “rescue” activity of yNap1 variants. Each data point is the mean of three independent measurements. The error bar indicates the standard deviation of the data.ITo show the composition of the salt assembled tetrasome and nucleosome preparations, the same sample was run on a native and SDS–PAGE gel.JExpression controls for yNap1 mutants. Nap1 pool in BY4741, Nap1Δ, and Nap1 mutants compared with TBP (TATA binding protein) as loading control.

Native PAGE gel showing the result of a multiple fluorescence relative affinity shift assay in the absence of yNap1; 31 nM–8 μM H2A–H2B (lanes 4–12) were incubated with a mixture of 0.7 μM AF488‐labeled H3–H4–DNA tetrasome complexes and 0.7 μM AF647‐labeled DNA. The gel was scanned to detect AF488‐ (left) or AF647‐labeled DNA (right). Lane 1 shows migration of tetrasomes (tetra). Appearance of nucleosomes (nucl) is indicated.

To quantify nucleosome assembly, the intensity of the band (nucl) was plotted as a function of H2A–H2B concentration. To measure non‐specific DNA binding, the disappearance of the free DNA AF647 band was plotted. The value in the absence of H2A–H2B (lane 1) was set to 100% intensity. The curve is representative of three independent experiments performed on different days. Standard deviations are indicated.

Same experiment as in (A) for the yNap1_IF1–H2A–H2B and for the yNap1c–H2A–H2B.

Nucleosome assembly rescue assay. Increasing amounts 1 μM–4 μM H2A–H2B were added to 0.7 μM AF488‐labeled H3–H4–DNA tetrasome complexes (lane 1) and resulted in non‐specific aggregation at 2–4 μM H2A–H2B (lanes 2–4). Incubation of 4 μM H2A–H2B with increasing amounts 0.31 μM–5 μM of yNap1 (lanes 5–9) resulted in a rescue of nucleosome assembly and prevented non‐specific DNA binding of H2A–H2B. The yNap1_HBR1 + 2 (lanes 10–14) show a defect in this rescue assay.

Same experiment as in (E) for the yNap1_HBR1 (lanes 5–9) and yNap1_HBR2 (lanes 10–14).

Same experiment as in (E) for the yNap1c (lanes 5–9) and yNap1_IF1 (lanes 10–14).

Quantification of the “rescue” activity of yNap1 variants. Each data point is the mean of three independent measurements. The error bar indicates the standard deviation of the data.

To show the composition of the salt assembled tetrasome and nucleosome preparations, the same sample was run on a native and SDS–PAGE gel.

Expression controls for yNap1 mutants. Nap1 pool in BY4741, Nap1Δ, and Nap1 mutants compared with TBP (TATA binding protein) as loading control.

To investigate the impact of the HBR mutants on yNap1 function, we added excess H2A–H2B to tetrasomes to investigate how the yNap1 variants compete with non‐specifically DNA‐bound H2A–H2B to “rescue” nucleosome assembly. As expected, the addition of excess H2A–H2B resulted in non‐specific aggregation (Fig EV4E and F; lanes 2–4). Subsequent addition of wild‐type yNap1 resolved such non‐specific DNA binding (Fig EV4E; lanes 5–9). yNap1 constructs containing the HBR1, HBR2, or HBR1 + 2 mutations were deficient in resolving such aggregates as indicated by the higher concentration required to restore nucleosome assembly (Fig EV4E and F). Thus in contrast to the GST‐pulldown assay which measures direct yNap1–H2A–H2B binding and where only the HBR1 + 2 mutant showed a defect (Fig 3D), in this competition assay, where the yNap1 variants compete non‐specifically bound H2A–H2B from DNA to restore nucleosome assembly, we also observed a phenotype for the HBR1 and HBR2 mutants. yNap1c and yNap1–IF1 also showed decreased rescue activity indicating that the flexible N‐ and C‐terminal regions and oligomerization are required for efficient chaperone activity (Fig EV4G). To qualitatively compare the “rescue” activity of the different yNap1 variants, the fluorescence intensity of the nucleosome band at 1 μM concentration of yNap1 was quantified (Fig EV4H). To monitor integrity of the tetrasome and nucleosome preparations, the same samples were run on a native and SDS–PAGE gel and confirm the composition and stoichiometry of these preparations (Fig EV4I). In order to further examine the physiological relevance, we analyzed the impact of yNap1 mutants on nucleosome assembly in vivo. As yNap1 deletion results in accumulation of nucleosome assembly intermediates (Fig 1), we complemented the nap1Δ strain with yNap1 variants and characterized their impact on nucleosomal structures using MNase digests (Appendix Table S1). Western blot analysis of the mutants confirmed that all proteins were expressed to wild‐type levels (Fig EV4J). Like wild‐type yNap1, the HBR1 and HBR2 mutant fully or partially complemented the nap1Δ strain (Fig 6C and D, comparing mononucleosomes). The HBR1 + 2 mutants were increasingly defective, as expected from the binding data. In contrast, the IF1 mutant, which showed only modest impact on H2A–H2B binding in vitro, showed the greatest defect on nucleosome assembly. The impact was stronger than that seen in the nap1Δ strain, indicative of a dominant negative effect. Together, these data show the functional importance of H2A–H2B engagement of yNap1 and histone binding induced oligomerization of the yNap1–H2A–H2B complex.

DiscussionDynamic assembly of nucleosomes is essential for the control of DNA‐templated processes such as replication, DNA damage repair, and gene regulation (Venkatesh & Workman, 2015). Nucleosome assembly is a two‐step process requiring the initial binding of an (H3–H4)2 tetramer to the DNA followed by deposition of two H2A–H2B dimers (Smith & Stillman, 1991). How chaperones orchestrate this process remains unclear (Loyola & Almouzni, 2004; De Koning et al, 2007; Elsasser & D'Arcy, 2012). yNap1 is a conserved histone chaperone that is well known to assemble nucleosomes in vitro but if yNap1 contributes to nucleosome assembly, in vivo has remained unknown (Ishimi & Kikuchi, 1991; Walter et al, 1995; Chang et al, 1997; Park et al, 2005; Lorch et al, 2006; Mazurkiewicz et al, 2006; Andrews et al, 2010). Our genomewide mapping of nucleosome positioning in a nap1Δ strain revealed significantly smaller nucleosomes (subnucleosomes) on average (~120 bp) as compared to those of the wild‐type strain (~150 bp). As one copy of H2A–H2B organizes about ~30 bp of nucleosomal DNA (Luger et al, 1997), it is likely that these smaller nucleosomes correspond to hexasomes, assembly intermediates lacking one copy of H2A–H2B. yNap1 is involved in nuclear transport of H2A–H2B (Mosammaparast et al, 2002), but we exclude the possibility that the nucleosome assembly defect we observe in the nap1Δ strain is due to lack of nuclear H2A–H2B. Nap1 deletion only modestly reduces the amounts of nuclear H2A–H2B indicating that there are redundant H2A–H2B transport mechanisms (Mosammaparast et al, 2001). Instead, as previous analysis in a nap1Δ strain reported significant non‐nucleosomal histone–DNA interaction of H2A–H2B, we favor the model that yNap1 prevents such non‐nucleosomal binding and thereby contributes to H2A–H2B deposition and nucleosome assembly (Andrews et al, 2010).Our structure shows how yNap1 prevents non‐specific DNA binding: yNap1 engages the structurally conserved, basic DNA binding regions of the histone fold of H2A. The DNA binding surfaces are conserved in other H2A variants such as H2A.Z, and therefore, our structure also explains why Nap1 can assist remodeling complexes such as SWR1 in incorporating H2A.Z into chromatin (Kobor et al, 2004; Mizuguchi et al, 2004; Park et al, 2005; Luk et al, 2007). This DNA binding region is conserved in H3–H4 as well, which likely explains why yNap1 can interact with all four core histones in vitro (McQuibban et al, 1998; Mosammaparast et al, 2001; Nakagawa et al, 2001; Bowman et al, 2011). However, we find no evidence that yNap1 chaperones H3–H4 in vivo. Our observation that hexasome‐sized nucleosomes accumulate in a nap1Δ strain indicates a defect in deposition of H2A–H2B but not of H3–H4. Our results therefore agree with studies showing that Nap1 preferentially interacts with H2A–H2B in vivo (Ito et al, 1996; Chang et al, 1997).As we consistently observed hexasomes and not H3–H4 tetrasomes in the nap1Δ strain, we suggest that yNap1 is required for specific deposition of the second copy of H2A–H2B. yNap1 also removes the second copy of H2A–H2B resulting in accumulation of hexasomes in RSC‐mediated nucleosome remodeling (Kuryan et al, 2012). Thus, while yNap1 preferentially targets the second copy of H2A–H2B, other H2A–H2B chaperones such as nucleoplasmin or FACT might be involved in exchanging the first copy of H2A–H2B (Gurard‐Levin et al, 2014). Preference for the second copy of H2A–H2B is probably explained by the positioning of H2A–H2B in the yNap1 binding pocket: Deposition of a second H2A–H2B dimer requires, in addition to H3–H4 binding through the H2A docking domain, contact formation between the two H2A L1 loops, an interface that is important for deposition of the second but not the first copy of H2A–H2B (Luger et al, 1997). Due to the asymmetric positioning, yNap1 does not engage the L1 loop of H2A and therefore would enable contact formation between two H2A–H2A copies, thus explaining the preference for the second copy of H2A–H2B. Because H2A–H2B loading requires yNap1 residues that are invariant across eukaryotes, we suggest that the yNap1‐dependent H2A–H2B loading we describe is a general feature of the nucleosome assembly reaction.The exact binding stoichiometry of the Nap1–H2A–H2B complex has been controversial. A stoichiometry of both one and two histone dimers to a single Nap1 dimer has been proposed (McBryant et al, 2003; Toth et al, 2005; Andrews et al, 2008; Newman et al, 2012; D'Arcy et al, 2013). By using hydrogen–deuterium exchange, D'Arcy et al (2013) reported that a yNap1 dimer accommodates two copies of H2A–H2B in an unconventional tetramer conformation. Such low‐resolution techniques can produce confounding results, especially with self‐assembling systems where it is difficult to determine from which interface (Nap1–Nap1, Nap1–histone, or histone–histone) protection arises. Our structure and extensive native mass spectrometry experiments now demonstrate that only a single H2A–H2B heterodimer is accommodated in the binding pocket of a yNap1 dimer. D'Arcy et al (2013) performed extensive mutagenesis of the acidic underside of yNap1 with the goal to identify the histone binding region. Our structure now shows that the mutations identified to disrupt histone binding are not located in the histone binding interface. Instead, the reported mutations are located in the buried segments in the helical underside of yNap1 and most likely indirectly interfere with H2A–H2B binding by affecting the structural integrity of yNap1. Our structure shows that histone binding through the yNap1 HBR regions and oligomerization through the IF1 and IF2 interfaces of the complex synergistically contribute to histone binding. Individual mutations in the HBR or NBR interfaces did not abolish H2A–H2B binding (Fig 3D and E) presumably due to the low nanomolar binding affinity between yNap1 and H2A–H2B (Park & Luger, 2006; Andrews et al, 2008). We observed reduced H2A–H2B binding only when both HBR regions contained charge reversal or alanine mutations (Fig 3D). Oligomerization through the IF1 and IF2 interfaces further contributes to the high binding avidity of the Nap1–H2A–H2B complex. A rigorous test for the specific yNap1–H2A–H2B interaction comes from a charge‐swap experiment: Replacement of the basic HBR1 + 2 interface in yNap1 with cationic residues abolishes binding to H2A–H2B. Also replacement of the basic NBR1 + 2 residues of H2A with acidic residues abolishes binding to yNap1. Binding is restored when such mutants are tested against each other (Fig 3F).Possible role of Nap1 oligomerization in histone transportThere is ample evidence that oligomerization of isolated Nap1 and Nap1–histone complexes occurs under physiological conditions in vitro and in vivo (Ishimi et al, 1983, 1984; Fujii‐Nakata et al, 1992; Mosammaparast et al, 2002; Toth et al, 2005; Newman et al, 2012). The physiological role of such oligomerization has remained unclear. yNap1 and Nap1‐like proteins show dynamic nuclear‐cytoplasmic localization in a variety of organisms and are known histone import factors (Ito et al, 1996; Rodriguez et al, 2000; Mosammaparast et al, 2002). One possibility is that oligomerization plays a role in such nucleocytoplasmic shuttling. A nuclear export sequence (NES) has been identified and experimentally confirmed in residues 88–102 (Mosammaparast et al, 2002; Miyaji‐Yamaguchi et al, 2003). Due to continuous NES‐dependent export from the nucleus, the majority of yNAP1 is present in the cytoplasm (Mosammaparast et al, 2002; Miyaji‐Yamaguchi et al, 2003). The karyopherin Kap114p binds a region of yNap1 that contains putative nuclear localization sequence (NLS) in residues 290–295 (Mosammaparast et al, 2002; Miyaji‐Yamaguchi et al, 2003). The region comprising this putative NLS has been implicated in oligomerization of isolated yNap1 presumably resulting in a form that is inhibitory for nuclear transport as burial of the NLS is predicted to prevent interaction with Kap114p (Park et al, 2008). We found that in the H2A–H2B‐bound yNap1 oligomer, the NLS region is disordered and solvent accessible. A solvent‐accessible NLS might allow binding of the karyopherin Kap114p and nuclear transport of the histone‐bound complex (Mosammaparast et al, 2002). We propose that in the absence of histones, yNap1 resides in the cytoplasm as the NLS is masked by oligomerization. Upon H2A–H2B binding in the cytoplasm, the NLS becomes accessible allowing Kap114p‐mediated nuclear import of the yNap1–H2A–H2B complex (Fig 7). Thus, oligomerization through the NLS in isolated yNap1 and through the IF1 and IF2 interfaces in the histone‐bound complex could be directly responsible for nucleocytoplasmic transport of yNap1 complexes. In accordance with the model that the yNap1–H2A–H2B oligomer seen in our structure is critical for yNap1 function, mutation of the yNap1_IF1 oligomerization interface results in severe nucleosome assembly defects, more severe than the defects observed with the HBR mutants or upon deletion yNap1 (Fig 6C and D). This observation at first seems counterintuitive: As the histone‐bound oligomer is inhibitory for H2A–H2B deposition, mutagenesis of the IF1 interface might have been expected to result in a gain‐of‐function phenotype. One plausible interpretation is that the yNap1_IF1 mutant acts as a dominant negative by sequestering H2A–H2B from other chaperones or by interfering with nucleocytoplasmic transport, thus causing the severe assembly defect observed here. Future studies need to experimentally demonstrate the role of the NLS and how defective oligomerization or histone loading interferes with the yNap1 functional cycle. As oligomerization results in burial of surfaces required for deposition of H2A–H2B into nucleosomes, we predict that it must be dynamically regulated, possibly by post‐translational modifications. The transport of histones into or out of the nucleus is an essential step in chromatin assembly and the masking and unmasking of NLS sequences, as a function of histone binding and oligomerization may be a mechanism for regulating subcellular localization of yNAP1 and histones.Figure 7Model for yNap1 as a H2A–H2B transport and chromatin assembly factorH2A–H2B synthesis in the cytoplasm results in binding to yNap1. The yNap1–H2A–H2B complex oligomerizes such that the NLS becomes accessible to the karyopherin Kap114p, resulting in nuclear transport. Presumably the oligomer needs to be disassembled for nuclear transport, a function that could be provided by phosphorylation (e.g., CK2). In the nucleus, yNap1 deposits H2A–H2B onto hexasome intermediates. After release of H2A–H2B, apo yNap1 can oligomerize through a region involving the NLS. Burial of the NLS results in cytoplasmic localization.

Dynamic assembly of nucleosomes is essential for the control of DNA‐templated processes such as replication, DNA damage repair, and gene regulation (Venkatesh & Workman, 2015). Nucleosome assembly is a two‐step process requiring the initial binding of an (H3–H4)2 tetramer to the DNA followed by deposition of two H2A–H2B dimers (Smith & Stillman, 1991). How chaperones orchestrate this process remains unclear (Loyola & Almouzni, 2004; De Koning et al, 2007; Elsasser & D'Arcy, 2012). yNap1 is a conserved histone chaperone that is well known to assemble nucleosomes in vitro but if yNap1 contributes to nucleosome assembly, in vivo has remained unknown (Ishimi & Kikuchi, 1991; Walter et al, 1995; Chang et al, 1997; Park et al, 2005; Lorch et al, 2006; Mazurkiewicz et al, 2006; Andrews et al, 2010). Our genomewide mapping of nucleosome positioning in a nap1Δ strain revealed significantly smaller nucleosomes (subnucleosomes) on average (~120 bp) as compared to those of the wild‐type strain (~150 bp). As one copy of H2A–H2B organizes about ~30 bp of nucleosomal DNA (Luger et al, 1997), it is likely that these smaller nucleosomes correspond to hexasomes, assembly intermediates lacking one copy of H2A–H2B. yNap1 is involved in nuclear transport of H2A–H2B (Mosammaparast et al, 2002), but we exclude the possibility that the nucleosome assembly defect we observe in the nap1Δ strain is due to lack of nuclear H2A–H2B. Nap1 deletion only modestly reduces the amounts of nuclear H2A–H2B indicating that there are redundant H2A–H2B transport mechanisms (Mosammaparast et al, 2001). Instead, as previous analysis in a nap1Δ strain reported significant non‐nucleosomal histone–DNA interaction of H2A–H2B, we favor the model that yNap1 prevents such non‐nucleosomal binding and thereby contributes to H2A–H2B deposition and nucleosome assembly (Andrews et al, 2010).

Our structure shows how yNap1 prevents non‐specific DNA binding: yNap1 engages the structurally conserved, basic DNA binding regions of the histone fold of H2A. The DNA binding surfaces are conserved in other H2A variants such as H2A.Z, and therefore, our structure also explains why Nap1 can assist remodeling complexes such as SWR1 in incorporating H2A.Z into chromatin (Kobor et al, 2004; Mizuguchi et al, 2004; Park et al, 2005; Luk et al, 2007). This DNA binding region is conserved in H3–H4 as well, which likely explains why yNap1 can interact with all four core histones in vitro (McQuibban et al, 1998; Mosammaparast et al, 2001; Nakagawa et al, 2001; Bowman et al, 2011). However, we find no evidence that yNap1 chaperones H3–H4 in vivo. Our observation that hexasome‐sized nucleosomes accumulate in a nap1Δ strain indicates a defect in deposition of H2A–H2B but not of H3–H4. Our results therefore agree with studies showing that Nap1 preferentially interacts with H2A–H2B in vivo (Ito et al, 1996; Chang et al, 1997).

As we consistently observed hexasomes and not H3–H4 tetrasomes in the nap1Δ strain, we suggest that yNap1 is required for specific deposition of the second copy of H2A–H2B. yNap1 also removes the second copy of H2A–H2B resulting in accumulation of hexasomes in RSC‐mediated nucleosome remodeling (Kuryan et al, 2012). Thus, while yNap1 preferentially targets the second copy of H2A–H2B, other H2A–H2B chaperones such as nucleoplasmin or FACT might be involved in exchanging the first copy of H2A–H2B (Gurard‐Levin et al, 2014). Preference for the second copy of H2A–H2B is probably explained by the positioning of H2A–H2B in the yNap1 binding pocket: Deposition of a second H2A–H2B dimer requires, in addition to H3–H4 binding through the H2A docking domain, contact formation between the two H2A L1 loops, an interface that is important for deposition of the second but not the first copy of H2A–H2B (Luger et al, 1997). Due to the asymmetric positioning, yNap1 does not engage the L1 loop of H2A and therefore would enable contact formation between two H2A–H2A copies, thus explaining the preference for the second copy of H2A–H2B. Because H2A–H2B loading requires yNap1 residues that are invariant across eukaryotes, we suggest that the yNap1‐dependent H2A–H2B loading we describe is a general feature of the nucleosome assembly reaction.

The exact binding stoichiometry of the Nap1–H2A–H2B complex has been controversial. A stoichiometry of both one and two histone dimers to a single Nap1 dimer has been proposed (McBryant et al, 2003; Toth et al, 2005; Andrews et al, 2008; Newman et al, 2012; D'Arcy et al, 2013). By using hydrogen–deuterium exchange, D'Arcy et al (2013) reported that a yNap1 dimer accommodates two copies of H2A–H2B in an unconventional tetramer conformation. Such low‐resolution techniques can produce confounding results, especially with self‐assembling systems where it is difficult to determine from which interface (Nap1–Nap1, Nap1–histone, or histone–histone) protection arises. Our structure and extensive native mass spectrometry experiments now demonstrate that only a single H2A–H2B heterodimer is accommodated in the binding pocket of a yNap1 dimer. D'Arcy et al (2013) performed extensive mutagenesis of the acidic underside of yNap1 with the goal to identify the histone binding region. Our structure now shows that the mutations identified to disrupt histone binding are not located in the histone binding interface. Instead, the reported mutations are located in the buried segments in the helical underside of yNap1 and most likely indirectly interfere with H2A–H2B binding by affecting the structural integrity of yNap1. Our structure shows that histone binding through the yNap1 HBR regions and oligomerization through the IF1 and IF2 interfaces of the complex synergistically contribute to histone binding. Individual mutations in the HBR or NBR interfaces did not abolish H2A–H2B binding (Fig 3D and E) presumably due to the low nanomolar binding affinity between yNap1 and H2A–H2B (Park & Luger, 2006; Andrews et al, 2008). We observed reduced H2A–H2B binding only when both HBR regions contained charge reversal or alanine mutations (Fig 3D). Oligomerization through the IF1 and IF2 interfaces further contributes to the high binding avidity of the Nap1–H2A–H2B complex. A rigorous test for the specific yNap1–H2A–H2B interaction comes from a charge‐swap experiment: Replacement of the basic HBR1 + 2 interface in yNap1 with cationic residues abolishes binding to H2A–H2B. Also replacement of the basic NBR1 + 2 residues of H2A with acidic residues abolishes binding to yNap1. Binding is restored when such mutants are tested against each other (Fig 3F).

Possible role of Nap1 oligomerization in histone transportThere is ample evidence that oligomerization of isolated Nap1 and Nap1–histone complexes occurs under physiological conditions in vitro and in vivo (Ishimi et al, 1983, 1984; Fujii‐Nakata et al, 1992; Mosammaparast et al, 2002; Toth et al, 2005; Newman et al, 2012). The physiological role of such oligomerization has remained unclear. yNap1 and Nap1‐like proteins show dynamic nuclear‐cytoplasmic localization in a variety of organisms and are known histone import factors (Ito et al, 1996; Rodriguez et al, 2000; Mosammaparast et al, 2002). One possibility is that oligomerization plays a role in such nucleocytoplasmic shuttling. A nuclear export sequence (NES) has been identified and experimentally confirmed in residues 88–102 (Mosammaparast et al, 2002; Miyaji‐Yamaguchi et al, 2003). Due to continuous NES‐dependent export from the nucleus, the majority of yNAP1 is present in the cytoplasm (Mosammaparast et al, 2002; Miyaji‐Yamaguchi et al, 2003). The karyopherin Kap114p binds a region of yNap1 that contains putative nuclear localization sequence (NLS) in residues 290–295 (Mosammaparast et al, 2002; Miyaji‐Yamaguchi et al, 2003). The region comprising this putative NLS has been implicated in oligomerization of isolated yNap1 presumably resulting in a form that is inhibitory for nuclear transport as burial of the NLS is predicted to prevent interaction with Kap114p (Park et al, 2008). We found that in the H2A–H2B‐bound yNap1 oligomer, the NLS region is disordered and solvent accessible. A solvent‐accessible NLS might allow binding of the karyopherin Kap114p and nuclear transport of the histone‐bound complex (Mosammaparast et al, 2002). We propose that in the absence of histones, yNap1 resides in the cytoplasm as the NLS is masked by oligomerization. Upon H2A–H2B binding in the cytoplasm, the NLS becomes accessible allowing Kap114p‐mediated nuclear import of the yNap1–H2A–H2B complex (Fig 7). Thus, oligomerization through the NLS in isolated yNap1 and through the IF1 and IF2 interfaces in the histone‐bound complex could be directly responsible for nucleocytoplasmic transport of yNap1 complexes. In accordance with the model that the yNap1–H2A–H2B oligomer seen in our structure is critical for yNap1 function, mutation of the yNap1_IF1 oligomerization interface results in severe nucleosome assembly defects, more severe than the defects observed with the HBR mutants or upon deletion yNap1 (Fig 6C and D). This observation at first seems counterintuitive: As the histone‐bound oligomer is inhibitory for H2A–H2B deposition, mutagenesis of the IF1 interface might have been expected to result in a gain‐of‐function phenotype. One plausible interpretation is that the yNap1_IF1 mutant acts as a dominant negative by sequestering H2A–H2B from other chaperones or by interfering with nucleocytoplasmic transport, thus causing the severe assembly defect observed here. Future studies need to experimentally demonstrate the role of the NLS and how defective oligomerization or histone loading interferes with the yNap1 functional cycle. As oligomerization results in burial of surfaces required for deposition of H2A–H2B into nucleosomes, we predict that it must be dynamically regulated, possibly by post‐translational modifications. The transport of histones into or out of the nucleus is an essential step in chromatin assembly and the masking and unmasking of NLS sequences, as a function of histone binding and oligomerization may be a mechanism for regulating subcellular localization of yNAP1 and histones.Figure 7Model for yNap1 as a H2A–H2B transport and chromatin assembly factorH2A–H2B synthesis in the cytoplasm results in binding to yNap1. The yNap1–H2A–H2B complex oligomerizes such that the NLS becomes accessible to the karyopherin Kap114p, resulting in nuclear transport. Presumably the oligomer needs to be disassembled for nuclear transport, a function that could be provided by phosphorylation (e.g., CK2). In the nucleus, yNap1 deposits H2A–H2B onto hexasome intermediates. After release of H2A–H2B, apo yNap1 can oligomerize through a region involving the NLS. Burial of the NLS results in cytoplasmic localization.

Possible role of Nap1 oligomerization in histone transport

There is ample evidence that oligomerization of isolated Nap1 and Nap1–histone complexes occurs under physiological conditions in vitro and in vivo (Ishimi et al, 1983, 1984; Fujii‐Nakata et al, 1992; Mosammaparast et al, 2002; Toth et al, 2005; Newman et al, 2012). The physiological role of such oligomerization has remained unclear. yNap1 and Nap1‐like proteins show dynamic nuclear‐cytoplasmic localization in a variety of organisms and are known histone import factors (Ito et al, 1996; Rodriguez et al, 2000; Mosammaparast et al, 2002). One possibility is that oligomerization plays a role in such nucleocytoplasmic shuttling. A nuclear export sequence (NES) has been identified and experimentally confirmed in residues 88–102 (Mosammaparast et al, 2002; Miyaji‐Yamaguchi et al, 2003). Due to continuous NES‐dependent export from the nucleus, the majority of yNAP1 is present in the cytoplasm (Mosammaparast et al, 2002; Miyaji‐Yamaguchi et al, 2003). The karyopherin Kap114p binds a region of yNap1 that contains putative nuclear localization sequence (NLS) in residues 290–295 (Mosammaparast et al, 2002; Miyaji‐Yamaguchi et al, 2003). The region comprising this putative NLS has been implicated in oligomerization of isolated yNap1 presumably resulting in a form that is inhibitory for nuclear transport as burial of the NLS is predicted to prevent interaction with Kap114p (Park et al, 2008). We found that in the H2A–H2B‐bound yNap1 oligomer, the NLS region is disordered and solvent accessible. A solvent‐accessible NLS might allow binding of the karyopherin Kap114p and nuclear transport of the histone‐bound complex (Mosammaparast et al, 2002). We propose that in the absence of histones, yNap1 resides in the cytoplasm as the NLS is masked by oligomerization. Upon H2A–H2B binding in the cytoplasm, the NLS becomes accessible allowing Kap114p‐mediated nuclear import of the yNap1–H2A–H2B complex (Fig 7). Thus, oligomerization through the NLS in isolated yNap1 and through the IF1 and IF2 interfaces in the histone‐bound complex could be directly responsible for nucleocytoplasmic transport of yNap1 complexes. In accordance with the model that the yNap1–H2A–H2B oligomer seen in our structure is critical for yNap1 function, mutation of the yNap1_IF1 oligomerization interface results in severe nucleosome assembly defects, more severe than the defects observed with the HBR mutants or upon deletion yNap1 (Fig 6C and D). This observation at first seems counterintuitive: As the histone‐bound oligomer is inhibitory for H2A–H2B deposition, mutagenesis of the IF1 interface might have been expected to result in a gain‐of‐function phenotype. One plausible interpretation is that the yNap1_IF1 mutant acts as a dominant negative by sequestering H2A–H2B from other chaperones or by interfering with nucleocytoplasmic transport, thus causing the severe assembly defect observed here. Future studies need to experimentally demonstrate the role of the NLS and how defective oligomerization or histone loading interferes with the yNap1 functional cycle. As oligomerization results in burial of surfaces required for deposition of H2A–H2B into nucleosomes, we predict that it must be dynamically regulated, possibly by post‐translational modifications. The transport of histones into or out of the nucleus is an essential step in chromatin assembly and the masking and unmasking of NLS sequences, as a function of histone binding and oligomerization may be a mechanism for regulating subcellular localization of yNAP1 and histones.

Model for yNap1 as a H2A–H2B transport and chromatin assembly factor

H2A–H2B synthesis in the cytoplasm results in binding to yNap1. The yNap1–H2A–H2B complex oligomerizes such that the NLS becomes accessible to the karyopherin Kap114p, resulting in nuclear transport. Presumably the oligomer needs to be disassembled for nuclear transport, a function that could be provided by phosphorylation (e.g., CK2). In the nucleus, yNap1 deposits H2A–H2B onto hexasome intermediates. After release of H2A–H2B, apo yNap1 can oligomerize through a region involving the NLS. Burial of the NLS results in cytoplasmic localization.

Materials and MethodsConstructsThe Saccharomyces cerevisiae Nap1 full‐length (residues 1–417), core (residues 75–365), and coreΔ 284–308 variants were cloned into pACYCDuet1 with a TEV‐cleavable N‐terminal His‐tag or into pETM33 (EMBL) with a TEV‐cleavable N‐terminal GST tag. The Xenopus laevis histones H2A and H2B were cloned into pET‐Duet1 or pACYCDuet1 (Novagen). All point mutants were generated using the QuikChange Lightning Site‐directed mutagenesis kit (Agilent Technologies). All constructs were confirmed by DNA sequencing.Protein expression and purificationRecombinant histones were purified and refolded according to the standard procedures (Luger et al, 1999). yNap1 protein variants were expressed in E. coli BL21(DE3). For co‐expression of yNap1 and H2A–H2B, GST‐tagged yNap1 in pETM33 was co‐transformed with a pACYCDuet1 vector encoding H2A and H2B into E. coli BL21(DE3). For binding assays, cells were grown to mid‐log phase and induced by addition of 0.5 mM IPTG for 3 h at 37°C. 5 × 109 cells were harvested and frozen at −20°C. For preparative scale purifications, cells were resuspended in buffer 1 (20 mM Tris, pH 8.0, 150 mM NaCl, 20 mM imidazole) and lysed using a Microfluidizer (Microfluidics). The lysate was clarified by centrifugation and applied to a 5 ml HisTrap FF crude column according to the instructions by the manufacturer (GE Healthcare). The resin was washed with buffer 1 and eluted with buffer 1 containing 500 mM imidazole. Eluted sample was then dialyzed against buffer 1 for 14–16 h at 4°C after addition of His‐tagged TEV protease (1:100 w/w). Subtractive Ni‐NTA chromatography was then employed to remove uncleaved protein and TEV protease. The untagged protein was further purified by gel filtration on a High Load 16/60 Superdex 200 column (GE Healthcare) equilibrated in buffer 2 (20 mM HEPES, pH 7.5, 150 mM NaCl, 1 mM dithiothreitol (DTT)). For GST‐tagged yNap1, cells were resuspended in 0.5 ml of lysis buffer (20 mM Tris, pH 8.0, 500 mM NaCl, 1 mM dithiothreitol), complemented with COMPLETE protease inhibitors (Roche), 12 units of Benzonase (Sigma), 1 mM EDTA, 2 μg of lysozyme (Thermo fisher), and lysed using a sonicator (Misonix). The lysates were incubated for 15 min at room temperature and clarified by centrifugation. The supernatants applied to 20 μl of 50% Glutathione Sepharose 4 Fast Flow resin (GE Healthcare) for 1 h at 4°C on a wheel. The resin was washed 4 times with 1 ml of lysis buffer complemented by 1% Triton X‐100 and eluted with 40 μl of the same buffer containing 10 mM reduced glutathione. Protein concentrations were determined by measuring the absorbance at 280 nm using a molar extinction coefficient calculated from the amino acid sequence. The final protein was concentrated to 10 mg ml−1, and aliquots were flash‐frozen in liquid N2 and stored at −80°C.Reconstitution of protein complexesH2A–H2B heterodimers were assembled by dissolving equimolar amounts of each histone in unfolding buffer (20 mM Tris, pH 7.5, 7 M guanidinium, 5 mM β‐mercaptoethanol) and incubated at room temperature for 30 min. After mixing H2A–H2B, the sample was incubated for 1 h, followed by dialysis for 16–18 h at 4°C against refolding buffer (10 mM Tris, pH 7.5, 2 M NaCl, 1 mM Na‐EDTA, 5 mM β‐mercaptoethanol). The sample was then run in refolding buffer on a pre‐equilibrated HiLoad 16/60 Superdex 75 column (GE Healthcare). Aliquots were stored in 50% glycerol at −20°C. To prepare Nap1–H2A–H2B complexes, reconstituted H2A–H2B dimers were added at a twofold molar excess to Nap1. The mixture was incubated for 30 min on ice and then loaded on a pre‐equilibrated Superose 6 10/300 GL (GE Healthcare) in buffer 2.Crystallization and structure determinationThe yNap1–H2A–H2B complex at 10 mg ml−1 was mixed with an equal volume of crystallization solution containing 10–15% polyethylene glycol 3350 and 0.2 M NaCl, and crystals were grown by hanging‐drop vapor diffusion at 20°C. Crystals were further refined by dehydration by increasing the PEG concentrations in the reservoir up to 35%. Crystals were cryoprotected in 20–25% glycerol and frozen in liquid nitrogen. Crystals typically diffracted to a Bragg spacing of ~8–15 Å resolution (~600 samples analyzed). Construct engineering by N‐ and C‐terminal truncation of yNap1 to residues 75–365 and N‐terminal truncation of the histones H2A (14–130) and H2B (28–126) allowed us to obtain crystals diffracting to a Bragg spacing of 6.7 Å. We collected native diffraction data at the ESRF on beamline ID14–4, under a nitrogen gas stream at 100 K, at a wavelength of 0.974 Å. We processed the data with XDS (Kabsch, 2010). Five percent of the reflections were marked as a test set. The structure of the yNap1c2–H2A–H2B complex was determined by molecular replacement using Phaser (McCoy et al, 2007). The search model was 2Z2R for yNap1 (Park et al, 2008), and models for Xenopus laevis H2A–H2B were used from the nucleosome structure 1KX4 (Davey et al, 2002). After a unique solution was identified for yNap1, a subsequent search with H2A–H2B resulted in one solution containing yNap1 bound to H2A–H2B. This solution was fixed and used as a search model, which allowed the identification of the remaining five copies of the complex. To address the issue of potential model bias, molecular replacement was repeated with H2A–H2B omitted from the search model. Each of the 24 protein chains were defined as individual rigid bodies and their position refined by rigid body refinement using the L–BFGS optimization method as implemented in phenix.refine (Afonine et al, 2012). B factors from the high‐resolution search models were maintained during the initial cycles of refinement and the initial model was used as both the starting and reference model for subsequent Deformable Elastic Network (DEN) refinement using CNS over a grid‐enabled web server hosted by SBGrid (Schroder et al, 2010; O'Donovan et al, 2012). The refinement protocol was similar to that used previously (Brunger et al, 2012) with the following non‐default setting: Only one overall anisotropic B factor refinement per chain was carried out with the starting B factors maintained from the original high‐resolution models. DEN restraints and non‐crystallographic symmetry (NCS) restraints were maintained throughout the refinement procedure. Seven different temperatures (from 0 to 3,000 K) for the slow‐cooling‐simulated annealing scheme were tested. Out of the resulting models, the one with the lowest Rfree value was used for subsequent analysis. The final structure was visualized in PyMol (Schrodinger, 2010). The resulting models have no Ramachandran outliers, good stereochemical parameters, and low crystallographic Rwork/Rfree (Table EV1), indicating a good agreement with the diffraction data.MNase analysisA summary of strains used is shown in Appendix Table S1. Saccharomyces cerevisiae strains BY4741 and nap1Δ strains (Fig 1) were grown in YPD media at 25°C to exponential phase (OD600 ~0.8). Other NAP1 mutants (Fig 6) were grown in synthetic complete (SC) –ura dropout media at similar OD and temperature. Yeast cells were then cross‐linked with 1% formaldehyde at 25°C for 15 min and quenched with 125 mM glycine. Yeast cells were harvested by centrifugation and ruptured by bead beating using zirconia 0.5 mm beads at 4°C for 3 cycles of 3 min each. For immunoblots of total cells, one milliliter cell equivalents were harvested and resuspended in Laemmli buffer, heat‐denatured, and electrophoresed on a 10% polyacrylamide gel under denaturing conditions. After blotting to 0.2‐micron nitrocellulose membrane blotted bands were immunodetected with Nap1 and TBP (TATA binding protein) antibodies. For micrococcal nuclease (MNase) digestion (Yen et al, 2012), the chromatin pellets were washed with NPS buffer (50 mM NaCl, 10 mM Tris, pH 7.5, 5 mM MgCl2, 1 mM CaCl2). For MNase‐seq (Fig 1), NPS also contained 0.5 mM spermidine, 0.075% IGEPAL. Mononucleosomes were solubilized via digestion with MNase to ~80% completion by gel analysis (as indicated in Appendix Fig S1). Mononucleosomes from 50 ml culture were immunoprecipitated using 10 μl of Protein A Mag Sepharose beads (GE) conjugated with 10 μg of H3 antibody (Abcam, ab1791) before ligation of the sequencing adaptor. DNA was eluted before ligation‐mediated PCR (LM‐PCR), followed by gel purification of approximately 250‐bp nucleosomal libraries. For gel electrophoretic analysis (Fig 6), chromatin was digested with 25 U, 50 U, 75 U, and 100 U of MNase for 15 min at 37°C. Digestions were quenched using 10 mM EDTA. The resulting variously digested chromatin was phenol‐extracted, and the DNA precipitated with ethanol. DNA fragments were electrophoresed on a 1.5% agarose gel at 120 V for 0.5–2 h. For each lane in the gel image, Image J (http://imagej.nih.gov/ij/) was used to extract the X and Y coordinates for the density histogram. The X and Y coordinates were plotted and the data were smoothed using a moving average window of size 500.GST pulldownAbout 10 μM GST‐tagged yNap1 variants were incubated for 30 min at 4°C with 40 μl of 50% Glutathione Sepharose 4 Fast Flow resin (GE Healthcare), pre‐equilibrated in buffer 3 (20 mM Tris, pH 8.0, 500 mM NaCl, 1 mM DTT, 0.1% Tween‐20). The resin was then washed three times with buffer 3, and H2A–H2B dimers were added and incubated overnight at 4°C. The resin was then washed five times with buffer 3, and bound protein was detected by boiling the resin in SDS–PAGE loading buffer, followed by SDS–PAGE and staining with Coomassie blue.Non‐denaturing mass spectrometryPrior to non‐denaturing MS analysis, 30 μl of sample was buffer‐exchanged into 150 mM ammonium acetate, pH 7.5 using micro Bio‐Spin6 columns (Bio‐Rad) or 10 kDa Vivaspin concentrators (Sartorius). For all measurements, 2 μl of sample was loaded into a capillary needle and spectra were recorded. The data were acquired on a quadrupole time‐of‐flight mass spectrometer (QTof II) modified for the transmission of large macromolecular assemblies (Sobott et al, 2002). Typical instrument conditions used for non‐denaturing MS conditions were as follows: capillary voltages 1.7 kV and cone voltage up to 120 V; for collision‐induced dissociation experiments, the collision voltage was increased up to 180–200 V, extractor voltage 5 V, analyzer pressure 8.15 × 10−4 mbar, and ToF 8.42 × 10−6 mbar. All mass spectra were calibrated externally using 100 mg ml−1 cesium iodide in water and data were acquired and processed with MassLynx v4.1 software (Waters) with minimal smoothing and no background subtraction.Negative‐stain electron microscopyFour microliters of the yNap1–H2A–H2B sample (~0.1 mg ml−1) were loaded between the mica and carbon interface as described in (Franzetti et al, 2002). The sample was stained using 2% uranyl acetate pH 4.5 and air‐dried. Images were taken under low‐dose conditions in a CM12 Philips electron microscope working at 120 kV and with a nominal magnification of 40,000 using an Orius SC1000 CCD camera. A total of 1,200 individual particles were boxed from 30 images and CTF‐corrected (phase flipped) (Tosi et al, 2014). The crystal structure of yNap1–H2A–H2B was low‐pass filtered to 60 Å, reprojected into 372 equally spaced views, and used as the starting model for 3D classification. Because of the limited number of images and as the sample was negatively stained, only four projection matching cycles were carried out in SPIDER (Frank et al, 1996; Franzetti et al, 2002). Fitting of the crystal structure into the EM map was done with Chimera (Pettersen et al, 2004).Gel shift assayAlexa Fluor 488‐ or Alexa Fluor 647 5′‐labeled (*) oligonucleotides 5′*CTGGAGAATCCCGGTGCCG3′ and 5′ACAGGATGTATATATCTGACACG3′ were used to PCR amplify the 147‐bp fragment containing the Widom 601 nucleosome positioning sequence from the plasmid pGEM‐3z/601 (Lowary & Widom, 1998). Labeled DNA fragments were gel‐purified and used in binding reactions. For salt deposition of histones, established procedures were used (Dyer et al, 2004). Briefly, H2A–H2B dimers and H3–H4 tetramers were mixed with DNA at 20 mM Tris, pH 8.0, 2 M NaCl, 1 mM EDTA, 1 mM DTT and dialyzed against 1.5 M NaCl buffer for 2–3 h at 4°C. The samples were then consecutively transferred to lower (first 1 M, 0.5 M, 0.25 M, and then 0.15 M) NaCl concentration buffer for 2 h each with the last dialysis being an overnight step. The binding reactions were analyzed on a 6% polyacrylamide 0.5× TBE gel. The gel was scanned by Typhoon Variable Mode Imager (GE Healthcare) using excitation laser at 488 nm, emission filter 526 nm, output voltage 580 V (for AF488 measurement), and excitation laser at 633 nm, emission filter 670 nm, output voltage 580 V (for AF647 measurement). The fluorescence intensities were quantified by ImageJ. The apparent dissociation constant (K
d) was obtained by quantifying the nucleosome complex.Native PAGE assayAbout 20 μM yNap1 and 0, 12.5, 25, 50, or 100 μM H2A–H2B were mixed in 20 mM HEPES pH 8.0, 150 mM NaCl, 1 mM DTT, 10% glycerol and incubated for 1 h on ice. Samples were then applied to a pre‐run 6% polyacrylamide gels in 0.5× TBE at 100 V, 21°C for 120 min. Protein was visualized using Coomassie Blue.

Materials and Methods

ConstructsThe Saccharomyces cerevisiae Nap1 full‐length (residues 1–417), core (residues 75–365), and coreΔ 284–308 variants were cloned into pACYCDuet1 with a TEV‐cleavable N‐terminal His‐tag or into pETM33 (EMBL) with a TEV‐cleavable N‐terminal GST tag. The Xenopus laevis histones H2A and H2B were cloned into pET‐Duet1 or pACYCDuet1 (Novagen). All point mutants were generated using the QuikChange Lightning Site‐directed mutagenesis kit (Agilent Technologies). All constructs were confirmed by DNA sequencing.

The Saccharomyces cerevisiae Nap1 full‐length (residues 1–417), core (residues 75–365), and coreΔ 284–308 variants were cloned into pACYCDuet1 with a TEV‐cleavable N‐terminal His‐tag or into pETM33 (EMBL) with a TEV‐cleavable N‐terminal GST tag. The Xenopus laevis histones H2A and H2B were cloned into pET‐Duet1 or pACYCDuet1 (Novagen). All point mutants were generated using the QuikChange Lightning Site‐directed mutagenesis kit (Agilent Technologies). All constructs were confirmed by DNA sequencing.

Protein expression and purificationRecombinant histones were purified and refolded according to the standard procedures (Luger et al, 1999). yNap1 protein variants were expressed in E. coli BL21(DE3). For co‐expression of yNap1 and H2A–H2B, GST‐tagged yNap1 in pETM33 was co‐transformed with a pACYCDuet1 vector encoding H2A and H2B into E. coli BL21(DE3). For binding assays, cells were grown to mid‐log phase and induced by addition of 0.5 mM IPTG for 3 h at 37°C. 5 × 109 cells were harvested and frozen at −20°C. For preparative scale purifications, cells were resuspended in buffer 1 (20 mM Tris, pH 8.0, 150 mM NaCl, 20 mM imidazole) and lysed using a Microfluidizer (Microfluidics). The lysate was clarified by centrifugation and applied to a 5 ml HisTrap FF crude column according to the instructions by the manufacturer (GE Healthcare). The resin was washed with buffer 1 and eluted with buffer 1 containing 500 mM imidazole. Eluted sample was then dialyzed against buffer 1 for 14–16 h at 4°C after addition of His‐tagged TEV protease (1:100 w/w). Subtractive Ni‐NTA chromatography was then employed to remove uncleaved protein and TEV protease. The untagged protein was further purified by gel filtration on a High Load 16/60 Superdex 200 column (GE Healthcare) equilibrated in buffer 2 (20 mM HEPES, pH 7.5, 150 mM NaCl, 1 mM dithiothreitol (DTT)). For GST‐tagged yNap1, cells were resuspended in 0.5 ml of lysis buffer (20 mM Tris, pH 8.0, 500 mM NaCl, 1 mM dithiothreitol), complemented with COMPLETE protease inhibitors (Roche), 12 units of Benzonase (Sigma), 1 mM EDTA, 2 μg of lysozyme (Thermo fisher), and lysed using a sonicator (Misonix). The lysates were incubated for 15 min at room temperature and clarified by centrifugation. The supernatants applied to 20 μl of 50% Glutathione Sepharose 4 Fast Flow resin (GE Healthcare) for 1 h at 4°C on a wheel. The resin was washed 4 times with 1 ml of lysis buffer complemented by 1% Triton X‐100 and eluted with 40 μl of the same buffer containing 10 mM reduced glutathione. Protein concentrations were determined by measuring the absorbance at 280 nm using a molar extinction coefficient calculated from the amino acid sequence. The final protein was concentrated to 10 mg ml−1, and aliquots were flash‐frozen in liquid N2 and stored at −80°C.

Protein expression and purification

Recombinant histones were purified and refolded according to the standard procedures (Luger et al, 1999). yNap1 protein variants were expressed in E. coli BL21(DE3). For co‐expression of yNap1 and H2A–H2B, GST‐tagged yNap1 in pETM33 was co‐transformed with a pACYCDuet1 vector encoding H2A and H2B into E. coli BL21(DE3). For binding assays, cells were grown to mid‐log phase and induced by addition of 0.5 mM IPTG for 3 h at 37°C. 5 × 109 cells were harvested and frozen at −20°C. For preparative scale purifications, cells were resuspended in buffer 1 (20 mM Tris, pH 8.0, 150 mM NaCl, 20 mM imidazole) and lysed using a Microfluidizer (Microfluidics). The lysate was clarified by centrifugation and applied to a 5 ml HisTrap FF crude column according to the instructions by the manufacturer (GE Healthcare). The resin was washed with buffer 1 and eluted with buffer 1 containing 500 mM imidazole. Eluted sample was then dialyzed against buffer 1 for 14–16 h at 4°C after addition of His‐tagged TEV protease (1:100 w/w). Subtractive Ni‐NTA chromatography was then employed to remove uncleaved protein and TEV protease. The untagged protein was further purified by gel filtration on a High Load 16/60 Superdex 200 column (GE Healthcare) equilibrated in buffer 2 (20 mM HEPES, pH 7.5, 150 mM NaCl, 1 mM dithiothreitol (DTT)). For GST‐tagged yNap1, cells were resuspended in 0.5 ml of lysis buffer (20 mM Tris, pH 8.0, 500 mM NaCl, 1 mM dithiothreitol), complemented with COMPLETE protease inhibitors (Roche), 12 units of Benzonase (Sigma), 1 mM EDTA, 2 μg of lysozyme (Thermo fisher), and lysed using a sonicator (Misonix). The lysates were incubated for 15 min at room temperature and clarified by centrifugation. The supernatants applied to 20 μl of 50% Glutathione Sepharose 4 Fast Flow resin (GE Healthcare) for 1 h at 4°C on a wheel. The resin was washed 4 times with 1 ml of lysis buffer complemented by 1% Triton X‐100 and eluted with 40 μl of the same buffer containing 10 mM reduced glutathione. Protein concentrations were determined by measuring the absorbance at 280 nm using a molar extinction coefficient calculated from the amino acid sequence. The final protein was concentrated to 10 mg ml−1, and aliquots were flash‐frozen in liquid N2 and stored at −80°C.

Reconstitution of protein complexesH2A–H2B heterodimers were assembled by dissolving equimolar amounts of each histone in unfolding buffer (20 mM Tris, pH 7.5, 7 M guanidinium, 5 mM β‐mercaptoethanol) and incubated at room temperature for 30 min. After mixing H2A–H2B, the sample was incubated for 1 h, followed by dialysis for 16–18 h at 4°C against refolding buffer (10 mM Tris, pH 7.5, 2 M NaCl, 1 mM Na‐EDTA, 5 mM β‐mercaptoethanol). The sample was then run in refolding buffer on a pre‐equilibrated HiLoad 16/60 Superdex 75 column (GE Healthcare). Aliquots were stored in 50% glycerol at −20°C. To prepare Nap1–H2A–H2B complexes, reconstituted H2A–H2B dimers were added at a twofold molar excess to Nap1. The mixture was incubated for 30 min on ice and then loaded on a pre‐equilibrated Superose 6 10/300 GL (GE Healthcare) in buffer 2.

Reconstitution of protein complexes

H2A–H2B heterodimers were assembled by dissolving equimolar amounts of each histone in unfolding buffer (20 mM Tris, pH 7.5, 7 M guanidinium, 5 mM β‐mercaptoethanol) and incubated at room temperature for 30 min. After mixing H2A–H2B, the sample was incubated for 1 h, followed by dialysis for 16–18 h at 4°C against refolding buffer (10 mM Tris, pH 7.5, 2 M NaCl, 1 mM Na‐EDTA, 5 mM β‐mercaptoethanol). The sample was then run in refolding buffer on a pre‐equilibrated HiLoad 16/60 Superdex 75 column (GE Healthcare). Aliquots were stored in 50% glycerol at −20°C. To prepare Nap1–H2A–H2B complexes, reconstituted H2A–H2B dimers were added at a twofold molar excess to Nap1. The mixture was incubated for 30 min on ice and then loaded on a pre‐equilibrated Superose 6 10/300 GL (GE Healthcare) in buffer 2.

Crystallization and structure determinationThe yNap1–H2A–H2B complex at 10 mg ml−1 was mixed with an equal volume of crystallization solution containing 10–15% polyethylene glycol 3350 and 0.2 M NaCl, and crystals were grown by hanging‐drop vapor diffusion at 20°C. Crystals were further refined by dehydration by increasing the PEG concentrations in the reservoir up to 35%. Crystals were cryoprotected in 20–25% glycerol and frozen in liquid nitrogen. Crystals typically diffracted to a Bragg spacing of ~8–15 Å resolution (~600 samples analyzed). Construct engineering by N‐ and C‐terminal truncation of yNap1 to residues 75–365 and N‐terminal truncation of the histones H2A (14–130) and H2B (28–126) allowed us to obtain crystals diffracting to a Bragg spacing of 6.7 Å. We collected native diffraction data at the ESRF on beamline ID14–4, under a nitrogen gas stream at 100 K, at a wavelength of 0.974 Å. We processed the data with XDS (Kabsch, 2010). Five percent of the reflections were marked as a test set. The structure of the yNap1c2–H2A–H2B complex was determined by molecular replacement using Phaser (McCoy et al, 2007). The search model was 2Z2R for yNap1 (Park et al, 2008), and models for Xenopus laevis H2A–H2B were used from the nucleosome structure 1KX4 (Davey et al, 2002). After a unique solution was identified for yNap1, a subsequent search with H2A–H2B resulted in one solution containing yNap1 bound to H2A–H2B. This solution was fixed and used as a search model, which allowed the identification of the remaining five copies of the complex. To address the issue of potential model bias, molecular replacement was repeated with H2A–H2B omitted from the search model. Each of the 24 protein chains were defined as individual rigid bodies and their position refined by rigid body refinement using the L–BFGS optimization method as implemented in phenix.refine (Afonine et al, 2012). B factors from the high‐resolution search models were maintained during the initial cycles of refinement and the initial model was used as both the starting and reference model for subsequent Deformable Elastic Network (DEN) refinement using CNS over a grid‐enabled web server hosted by SBGrid (Schroder et al, 2010; O'Donovan et al, 2012). The refinement protocol was similar to that used previously (Brunger et al, 2012) with the following non‐default setting: Only one overall anisotropic B factor refinement per chain was carried out with the starting B factors maintained from the original high‐resolution models. DEN restraints and non‐crystallographic symmetry (NCS) restraints were maintained throughout the refinement procedure. Seven different temperatures (from 0 to 3,000 K) for the slow‐cooling‐simulated annealing scheme were tested. Out of the resulting models, the one with the lowest Rfree value was used for subsequent analysis. The final structure was visualized in PyMol (Schrodinger, 2010). The resulting models have no Ramachandran outliers, good stereochemical parameters, and low crystallographic Rwork/Rfree (Table EV1), indicating a good agreement with the diffraction data.

Crystallization and structure determination

The yNap1–H2A–H2B complex at 10 mg ml−1 was mixed with an equal volume of crystallization solution containing 10–15% polyethylene glycol 3350 and 0.2 M NaCl, and crystals were grown by hanging‐drop vapor diffusion at 20°C. Crystals were further refined by dehydration by increasing the PEG concentrations in the reservoir up to 35%. Crystals were cryoprotected in 20–25% glycerol and frozen in liquid nitrogen. Crystals typically diffracted to a Bragg spacing of ~8–15 Å resolution (~600 samples analyzed). Construct engineering by N‐ and C‐terminal truncation of yNap1 to residues 75–365 and N‐terminal truncation of the histones H2A (14–130) and H2B (28–126) allowed us to obtain crystals diffracting to a Bragg spacing of 6.7 Å. We collected native diffraction data at the ESRF on beamline ID14–4, under a nitrogen gas stream at 100 K, at a wavelength of 0.974 Å. We processed the data with XDS (Kabsch, 2010). Five percent of the reflections were marked as a test set. The structure of the yNap1c2–H2A–H2B complex was determined by molecular replacement using Phaser (McCoy et al, 2007). The search model was 2Z2R for yNap1 (Park et al, 2008), and models for Xenopus laevis H2A–H2B were used from the nucleosome structure 1KX4 (Davey et al, 2002). After a unique solution was identified for yNap1, a subsequent search with H2A–H2B resulted in one solution containing yNap1 bound to H2A–H2B. This solution was fixed and used as a search model, which allowed the identification of the remaining five copies of the complex. To address the issue of potential model bias, molecular replacement was repeated with H2A–H2B omitted from the search model. Each of the 24 protein chains were defined as individual rigid bodies and their position refined by rigid body refinement using the L–BFGS optimization method as implemented in phenix.refine (Afonine et al, 2012). B factors from the high‐resolution search models were maintained during the initial cycles of refinement and the initial model was used as both the starting and reference model for subsequent Deformable Elastic Network (DEN) refinement using CNS over a grid‐enabled web server hosted by SBGrid (Schroder et al, 2010; O'Donovan et al, 2012). The refinement protocol was similar to that used previously (Brunger et al, 2012) with the following non‐default setting: Only one overall anisotropic B factor refinement per chain was carried out with the starting B factors maintained from the original high‐resolution models. DEN restraints and non‐crystallographic symmetry (NCS) restraints were maintained throughout the refinement procedure. Seven different temperatures (from 0 to 3,000 K) for the slow‐cooling‐simulated annealing scheme were tested. Out of the resulting models, the one with the lowest Rfree value was used for subsequent analysis. The final structure was visualized in PyMol (Schrodinger, 2010). The resulting models have no Ramachandran outliers, good stereochemical parameters, and low crystallographic Rwork/Rfree (Table EV1), indicating a good agreement with the diffraction data.

MNase analysisA summary of strains used is shown in Appendix Table S1. Saccharomyces cerevisiae strains BY4741 and nap1Δ strains (Fig 1) were grown in YPD media at 25°C to exponential phase (OD600 ~0.8). Other NAP1 mutants (Fig 6) were grown in synthetic complete (SC) –ura dropout media at similar OD and temperature. Yeast cells were then cross‐linked with 1% formaldehyde at 25°C for 15 min and quenched with 125 mM glycine. Yeast cells were harvested by centrifugation and ruptured by bead beating using zirconia 0.5 mm beads at 4°C for 3 cycles of 3 min each. For immunoblots of total cells, one milliliter cell equivalents were harvested and resuspended in Laemmli buffer, heat‐denatured, and electrophoresed on a 10% polyacrylamide gel under denaturing conditions. After blotting to 0.2‐micron nitrocellulose membrane blotted bands were immunodetected with Nap1 and TBP (TATA binding protein) antibodies. For micrococcal nuclease (MNase) digestion (Yen et al, 2012), the chromatin pellets were washed with NPS buffer (50 mM NaCl, 10 mM Tris, pH 7.5, 5 mM MgCl2, 1 mM CaCl2). For MNase‐seq (Fig 1), NPS also contained 0.5 mM spermidine, 0.075% IGEPAL. Mononucleosomes were solubilized via digestion with MNase to ~80% completion by gel analysis (as indicated in Appendix Fig S1). Mononucleosomes from 50 ml culture were immunoprecipitated using 10 μl of Protein A Mag Sepharose beads (GE) conjugated with 10 μg of H3 antibody (Abcam, ab1791) before ligation of the sequencing adaptor. DNA was eluted before ligation‐mediated PCR (LM‐PCR), followed by gel purification of approximately 250‐bp nucleosomal libraries. For gel electrophoretic analysis (Fig 6), chromatin was digested with 25 U, 50 U, 75 U, and 100 U of MNase for 15 min at 37°C. Digestions were quenched using 10 mM EDTA. The resulting variously digested chromatin was phenol‐extracted, and the DNA precipitated with ethanol. DNA fragments were electrophoresed on a 1.5% agarose gel at 120 V for 0.5–2 h. For each lane in the gel image, Image J (http://imagej.nih.gov/ij/) was used to extract the X and Y coordinates for the density histogram. The X and Y coordinates were plotted and the data were smoothed using a moving average window of size 500.

MNase analysis

A summary of strains used is shown in Appendix Table S1. Saccharomyces cerevisiae strains BY4741 and nap1Δ strains (Fig 1) were grown in YPD media at 25°C to exponential phase (OD600 ~0.8). Other NAP1 mutants (Fig 6) were grown in synthetic complete (SC) –ura dropout media at similar OD and temperature. Yeast cells were then cross‐linked with 1% formaldehyde at 25°C for 15 min and quenched with 125 mM glycine. Yeast cells were harvested by centrifugation and ruptured by bead beating using zirconia 0.5 mm beads at 4°C for 3 cycles of 3 min each. For immunoblots of total cells, one milliliter cell equivalents were harvested and resuspended in Laemmli buffer, heat‐denatured, and electrophoresed on a 10% polyacrylamide gel under denaturing conditions. After blotting to 0.2‐micron nitrocellulose membrane blotted bands were immunodetected with Nap1 and TBP (TATA binding protein) antibodies. For micrococcal nuclease (MNase) digestion (Yen et al, 2012), the chromatin pellets were washed with NPS buffer (50 mM NaCl, 10 mM Tris, pH 7.5, 5 mM MgCl2, 1 mM CaCl2). For MNase‐seq (Fig 1), NPS also contained 0.5 mM spermidine, 0.075% IGEPAL. Mononucleosomes were solubilized via digestion with MNase to ~80% completion by gel analysis (as indicated in Appendix Fig S1). Mononucleosomes from 50 ml culture were immunoprecipitated using 10 μl of Protein A Mag Sepharose beads (GE) conjugated with 10 μg of H3 antibody (Abcam, ab1791) before ligation of the sequencing adaptor. DNA was eluted before ligation‐mediated PCR (LM‐PCR), followed by gel purification of approximately 250‐bp nucleosomal libraries. For gel electrophoretic analysis (Fig 6), chromatin was digested with 25 U, 50 U, 75 U, and 100 U of MNase for 15 min at 37°C. Digestions were quenched using 10 mM EDTA. The resulting variously digested chromatin was phenol‐extracted, and the DNA precipitated with ethanol. DNA fragments were electrophoresed on a 1.5% agarose gel at 120 V for 0.5–2 h. For each lane in the gel image, Image J (http://imagej.nih.gov/ij/) was used to extract the X and Y coordinates for the density histogram. The X and Y coordinates were plotted and the data were smoothed using a moving average window of size 500.

GST pulldownAbout 10 μM GST‐tagged yNap1 variants were incubated for 30 min at 4°C with 40 μl of 50% Glutathione Sepharose 4 Fast Flow resin (GE Healthcare), pre‐equilibrated in buffer 3 (20 mM Tris, pH 8.0, 500 mM NaCl, 1 mM DTT, 0.1% Tween‐20). The resin was then washed three times with buffer 3, and H2A–H2B dimers were added and incubated overnight at 4°C. The resin was then washed five times with buffer 3, and bound protein was detected by boiling the resin in SDS–PAGE loading buffer, followed by SDS–PAGE and staining with Coomassie blue.

GST pulldown

About 10 μM GST‐tagged yNap1 variants were incubated for 30 min at 4°C with 40 μl of 50% Glutathione Sepharose 4 Fast Flow resin (GE Healthcare), pre‐equilibrated in buffer 3 (20 mM Tris, pH 8.0, 500 mM NaCl, 1 mM DTT, 0.1% Tween‐20). The resin was then washed three times with buffer 3, and H2A–H2B dimers were added and incubated overnight at 4°C. The resin was then washed five times with buffer 3, and bound protein was detected by boiling the resin in SDS–PAGE loading buffer, followed by SDS–PAGE and staining with Coomassie blue.

Non‐denaturing mass spectrometryPrior to non‐denaturing MS analysis, 30 μl of sample was buffer‐exchanged into 150 mM ammonium acetate, pH 7.5 using micro Bio‐Spin6 columns (Bio‐Rad) or 10 kDa Vivaspin concentrators (Sartorius). For all measurements, 2 μl of sample was loaded into a capillary needle and spectra were recorded. The data were acquired on a quadrupole time‐of‐flight mass spectrometer (QTof II) modified for the transmission of large macromolecular assemblies (Sobott et al, 2002). Typical instrument conditions used for non‐denaturing MS conditions were as follows: capillary voltages 1.7 kV and cone voltage up to 120 V; for collision‐induced dissociation experiments, the collision voltage was increased up to 180–200 V, extractor voltage 5 V, analyzer pressure 8.15 × 10−4 mbar, and ToF 8.42 × 10−6 mbar. All mass spectra were calibrated externally using 100 mg ml−1 cesium iodide in water and data were acquired and processed with MassLynx v4.1 software (Waters) with minimal smoothing and no background subtraction.

Non‐denaturing mass spectrometry

Prior to non‐denaturing MS analysis, 30 μl of sample was buffer‐exchanged into 150 mM ammonium acetate, pH 7.5 using micro Bio‐Spin6 columns (Bio‐Rad) or 10 kDa Vivaspin concentrators (Sartorius). For all measurements, 2 μl of sample was loaded into a capillary needle and spectra were recorded. The data were acquired on a quadrupole time‐of‐flight mass spectrometer (QTof II) modified for the transmission of large macromolecular assemblies (Sobott et al, 2002). Typical instrument conditions used for non‐denaturing MS conditions were as follows: capillary voltages 1.7 kV and cone voltage up to 120 V; for collision‐induced dissociation experiments, the collision voltage was increased up to 180–200 V, extractor voltage 5 V, analyzer pressure 8.15 × 10−4 mbar, and ToF 8.42 × 10−6 mbar. All mass spectra were calibrated externally using 100 mg ml−1 cesium iodide in water and data were acquired and processed with MassLynx v4.1 software (Waters) with minimal smoothing and no background subtraction.

Negative‐stain electron microscopyFour microliters of the yNap1–H2A–H2B sample (~0.1 mg ml−1) were loaded between the mica and carbon interface as described in (Franzetti et al, 2002). The sample was stained using 2% uranyl acetate pH 4.5 and air‐dried. Images were taken under low‐dose conditions in a CM12 Philips electron microscope working at 120 kV and with a nominal magnification of 40,000 using an Orius SC1000 CCD camera. A total of 1,200 individual particles were boxed from 30 images and CTF‐corrected (phase flipped) (Tosi et al, 2014). The crystal structure of yNap1–H2A–H2B was low‐pass filtered to 60 Å, reprojected into 372 equally spaced views, and used as the starting model for 3D classification. Because of the limited number of images and as the sample was negatively stained, only four projection matching cycles were carried out in SPIDER (Frank et al, 1996; Franzetti et al, 2002). Fitting of the crystal structure into the EM map was done with Chimera (Pettersen et al, 2004).

Negative‐stain electron microscopy

Four microliters of the yNap1–H2A–H2B sample (~0.1 mg ml−1) were loaded between the mica and carbon interface as described in (Franzetti et al, 2002). The sample was stained using 2% uranyl acetate pH 4.5 and air‐dried. Images were taken under low‐dose conditions in a CM12 Philips electron microscope working at 120 kV and with a nominal magnification of 40,000 using an Orius SC1000 CCD camera. A total of 1,200 individual particles were boxed from 30 images and CTF‐corrected (phase flipped) (Tosi et al, 2014). The crystal structure of yNap1–H2A–H2B was low‐pass filtered to 60 Å, reprojected into 372 equally spaced views, and used as the starting model for 3D classification. Because of the limited number of images and as the sample was negatively stained, only four projection matching cycles were carried out in SPIDER (Frank et al, 1996; Franzetti et al, 2002). Fitting of the crystal structure into the EM map was done with Chimera (Pettersen et al, 2004).

Gel shift assayAlexa Fluor 488‐ or Alexa Fluor 647 5′‐labeled (*) oligonucleotides 5′*CTGGAGAATCCCGGTGCCG3′ and 5′ACAGGATGTATATATCTGACACG3′ were used to PCR amplify the 147‐bp fragment containing the Widom 601 nucleosome positioning sequence from the plasmid pGEM‐3z/601 (Lowary & Widom, 1998). Labeled DNA fragments were gel‐purified and used in binding reactions. For salt deposition of histones, established procedures were used (Dyer et al, 2004). Briefly, H2A–H2B dimers and H3–H4 tetramers were mixed with DNA at 20 mM Tris, pH 8.0, 2 M NaCl, 1 mM EDTA, 1 mM DTT and dialyzed against 1.5 M NaCl buffer for 2–3 h at 4°C. The samples were then consecutively transferred to lower (first 1 M, 0.5 M, 0.25 M, and then 0.15 M) NaCl concentration buffer for 2 h each with the last dialysis being an overnight step. The binding reactions were analyzed on a 6% polyacrylamide 0.5× TBE gel. The gel was scanned by Typhoon Variable Mode Imager (GE Healthcare) using excitation laser at 488 nm, emission filter 526 nm, output voltage 580 V (for AF488 measurement), and excitation laser at 633 nm, emission filter 670 nm, output voltage 580 V (for AF647 measurement). The fluorescence intensities were quantified by ImageJ. The apparent dissociation constant (K
d) was obtained by quantifying the nucleosome complex.

Gel shift assay

Alexa Fluor 488‐ or Alexa Fluor 647 5′‐labeled (*) oligonucleotides 5′*CTGGAGAATCCCGGTGCCG3′ and 5′ACAGGATGTATATATCTGACACG3′ were used to PCR amplify the 147‐bp fragment containing the Widom 601 nucleosome positioning sequence from the plasmid pGEM‐3z/601 (Lowary & Widom, 1998). Labeled DNA fragments were gel‐purified and used in binding reactions. For salt deposition of histones, established procedures were used (Dyer et al, 2004). Briefly, H2A–H2B dimers and H3–H4 tetramers were mixed with DNA at 20 mM Tris, pH 8.0, 2 M NaCl, 1 mM EDTA, 1 mM DTT and dialyzed against 1.5 M NaCl buffer for 2–3 h at 4°C. The samples were then consecutively transferred to lower (first 1 M, 0.5 M, 0.25 M, and then 0.15 M) NaCl concentration buffer for 2 h each with the last dialysis being an overnight step. The binding reactions were analyzed on a 6% polyacrylamide 0.5× TBE gel. The gel was scanned by Typhoon Variable Mode Imager (GE Healthcare) using excitation laser at 488 nm, emission filter 526 nm, output voltage 580 V (for AF488 measurement), and excitation laser at 633 nm, emission filter 670 nm, output voltage 580 V (for AF647 measurement). The fluorescence intensities were quantified by ImageJ. The apparent dissociation constant (K
d) was obtained by quantifying the nucleosome complex.

Native PAGE assayAbout 20 μM yNap1 and 0, 12.5, 25, 50, or 100 μM H2A–H2B were mixed in 20 mM HEPES pH 8.0, 150 mM NaCl, 1 mM DTT, 10% glycerol and incubated for 1 h on ice. Samples were then applied to a pre‐run 6% polyacrylamide gels in 0.5× TBE at 100 V, 21°C for 120 min. Protein was visualized using Coomassie Blue.

Native PAGE assay

About 20 μM yNap1 and 0, 12.5, 25, 50, or 100 μM H2A–H2B were mixed in 20 mM HEPES pH 8.0, 150 mM NaCl, 1 mM DTT, 10% glycerol and incubated for 1 h on ice. Samples were then applied to a pre‐run 6% polyacrylamide gels in 0.5× TBE at 100 V, 21°C for 120 min. Protein was visualized using Coomassie Blue.

CA–G, AL, and DP designed and performed the experiments except as noted below. KY conducted MNase‐seq, VV performed MNase gel analysis, RR wrote scripts for gel quantification, under the supervision of BFP. NAP, I‐OE, and CVR performed native mass spectrometry experiments, and GS performed negative‐stain EM experiments. DP advised and assisted on all aspects of the project and wrote the manuscript. All authors discussed the results and commented on the manuscript.

Conflict of interestThe authors declare that they have no conflict of interest.

Conflict of interest

The authors declare that they have no conflict of interest.

Supporting informationAppendixClick here for additional data file.Expanded View Figures PDFClick here for additional data file.Table EV1Click here for additional data file.Table EV2Click here for additional data file.Review Process FileClick here for additional data file.Source Data for Figure 3Click here for additional data file.

Supporting information

Click here for additional data file.

Expanded View Figures PDF

Click here for additional data file.

Click here for additional data file.

Click here for additional data file.

Review Process File

Click here for additional data file.

Source Data for Figure 3

Click here for additional data file.
