---
pmid: '29924966'
title: A Proteomic Variant Approach (ProVarA) for Personalized Medicine of Inherited
  and Somatic Disease.
authors:
- Hutt DM
- Loguercio S
- Campos AR
- Balch WE
journal: J Mol Biol
year: '2018'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC6097907
doi: 10.1016/j.jmb.2018.06.017
---

# A Proteomic Variant Approach (ProVarA) for Personalized Medicine of Inherited and Somatic Disease.
**Authors:** Hutt DM, Loguercio S, Campos AR, Balch WE
**Journal:** J Mol Biol (2018)
**DOI:** [10.1016/j.jmb.2018.06.017](https://doi.org/10.1016/j.jmb.2018.06.017)
**PMC:** [PMC6097907](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6097907/)

## Abstract

1. J Mol Biol. 2018 Sep 14;430(18 Pt A):2951-2973. doi:
10.1016/j.jmb.2018.06.017.  Epub 2018 Jun 18.

A Proteomic Variant Approach (ProVarA) for Personalized Medicine of Inherited 
and Somatic Disease.

Hutt DM(1), Loguercio S(1), Campos AR(2), Balch WE(3).

Author information:
(1)The Scripps Research Institute, Department of Molecular Medicine, 10550 North 
Torrey Pines Rd, La Jolla, CA 92037, USA.
(2)Sanford Burnham Prebys Medical Discovery Institute Proteomic Core, 10901 
North Torrey Pines Road, La Jolla, CA 92037, USA.
(3)The Scripps Research Institute, Department of Molecular Medicine, 10550 North 
Torrey Pines Rd, La Jolla, CA 92037, USA; Integrative Structural and 
Computational Biology, 10550 North Torrey Pines Rd, La Jolla, CA 92037, USA; The 
Skaggs Institute for Chemical Biology, 10550 North Torrey Pines Rd, La Jolla, CA 
92037, USA. Electronic address: webalch@scripps.edu.

The advent of precision medicine for genetic diseases has been hampered by the 
large number of variants that cause familial and somatic disease, a complexity 
that is further confounded by the impact of genetic modifiers. To begin to 
understand differences in onset, progression and therapeutic response that exist 
among disease-causing variants, we present the proteomic variant approach 
(ProVarA), a proteomic method that integrates mass spectrometry with genomic 
tools to dissect the etiology of disease. To illustrate its value, we examined 
the impact of variation in cystic fibrosis (CF), where 2025 disease-associated 
mutations in the CF transmembrane conductance regulator (CFTR) gene have been 
annotated and where individual genotypes exhibit phenotypic heterogeneity and 
response to therapeutic intervention. A comparative analysis of variant-specific 
proteomics allows us to identify a number of protein interactions contributing 
to the basic defects associated with F508del- and G551D-CFTR, two of the most 
common disease-associated variants in the patient population. We demonstrate 
that a number of these causal interactions are significantly altered in response 
to treatment with Vx809 and Vx770, small-molecule therapeutics that respectively 
target the F508del and G551D variants. ProVarA represents the first comparative 
proteomic analysis among multiple disease-causing mutations, thereby providing a 
methodological approach that provides a significant advancement to existing 
proteomic efforts in understanding the impact of variation in CF disease. We 
posit that the implementation of ProVarA for any familial or somatic mutation 
will provide a substantial increase in the knowledge base needed to implement a 
precision medicine-based approach for clinical management of disease.

Copyright © 2018 Elsevier Ltd. All rights reserved.

DOI: 10.1016/j.jmb.2018.06.017
PMCID: PMC6097907
PMID: 29924966 [Indexed for MEDLINE]

## Full Text

Abstract

The advent of precision medicine for genetic diseases has been hampered by the large number of variants that cause familial and somatic disease, a complexity that is further confounded by the impact of genetic modifiers. To begin to understand differences in onset, progression and therapeutic response that exist among disease-causing variants, we present the p roteomic v ariant a pproach (ProVarA), a proteomic method that integrates mass spectrometry with genomic tools to dissect the etiology of disease. To illustrate its value, we examined the impact of variation in c ystic f ibrosis (CF), where 2025 disease-associated mutations in the CF t ransmembrane conductance r egulator (CFTR) gene have been annotated and where individual genotypes exhibit phenotypic heterogeneity and response to therapeutic intervention. A comparative analysis of variant-specific proteomics allows us to identify a number of protein interactions contributing to the basic defects associated with F508del- and G551D-CFTR, 2 of the most common disease-associated variants in the patient population. We demonstrate that a number of these causal interactions are significantly altered in response to treatment with Vx809 and Vx770, small molecule therapeutics that respectively target the F508del and G551D variants. ProVarA represents the first comparative proteomic analysis among multiple disease-causing mutations, thereby providing a methodological approach that provides a significant advancement to existing proteomic efforts in understanding the impact of variation in CF disease. We posit that the implementation of ProVarA for any familial or somatic mutation will provide a substantial increase in the knowledge base needed to implement a precision medicine-based approach for clinical management of disease.

Introduction

Disease-causing familial variation recorded in GnomAD [ 1 ] and ClinVar [ 2 ] now include more than 200,000 annotated variants in the human population that contribute to genetic diversity and healthspan [ 2 – 10 ]. Like rapidly evolving somatic variations observed in cancer, these mutations uncouple the affected protein from its normal community of interacting proteins, which are critical for its biogenesis and function in the cell. The inability of many disease-associated variants to properly interact with their respective components results in altered functional profiles leading to the disease phenotype. Therapeutic development in these genetic diseases is complicated by the fact that many are caused by different variants in a single protein and the fact that even patients with the same disease-causing allele are differentially impacted by modifier genes leading to a heterogeneity of responses to existing therapeutics [ 5 ]. These observations have led to the advent of high definition medicine [ 11 ] to better serve affected individuals from the perspective of the precision medicine initiative. However, the implementation of such a personalized medicine approach often suffers due to a lack of information pertaining to the etiology of the disease as well as a sparsity of data pertaining to the impact of the disease-causing mutation on the affected protein and its function.

Herein, we introduce the development of a proteomic methodology that captures critical information required to understand the onset and progression of variant-specific disease, referred to as the p roteomic v ariant a pproach (ProVarA). Using affinity purification mass spectrometry (AP-MS) combined with genomic screening, we illustrate the utility of ProVarA in differentiating the impact of individual variants on disease etiology. To illustrate the utility of ProVarA to serve as a proteomic methodology for a complex familial disease, we focus on c ystic f ibrosis (CF). CF is caused by mutations in the CF t ransmembrane conductance r egulator (CFTR) gene, which codes for a cAMP-regulated chloride channel expressed at the apical surface of epithelial cells [ 12 – 17 ] and is critical for the maintenance of proper chloride and bicarbonate balance in nearly all tissues. While more than 70% of CF patients carry at least one allele of a three-base pair deletion (delCTT) [ 18 ] resulting in the loss of phenylalanine at position 508 (F508del-CFTR), there are currently 2025 known CF-causing mutations in the patient population ( http://www.genet.sickkids.on.ca ; www.cftr2.org ). These mutations are grouped into 1 of 6 classes including mutations that lead to a loss of CFTR production (Class I), misfolding and/or premature degradation (Class II), functional impairment (Class III), obstruction of the channel pore (Class IV), a reduction in the amount of CFTR produced (Class V) and destabilization of CFTR at the cell surface (Class VI) [ 19 ]. Recently a seventh class was proposed, which re-categorized those class I mutations that did not produce any mRNA into a class VII to reflect their lack of correctability by small molecule therapeutics [ 20 ]. Marson et al extended upon this 7 class suggestion by proposing that class VII be renamed as class IA and that class I proposed by De Boeck and Amaral be labelled as class IB to maintain the progressive decrease in disease severity historically associated with increasing class numbers in this CFTR classification system [ 21 ].

Herein we show how the application of ProVarA to CFTR variants can be used to interrogate the impact of these different CF-causing mutations on the functional interactions of the variant polypeptide chain with the binding proteins required for its normal function. These variants include the Class II variants, F508del, G85E, R560T and N1303K as well as the Class III variant G551D CFTR [ 19 ]. Profiling interactions through ProVarA allows us to identify both common and unique interactions that contribute to the development of CF disease in ER-restricted variants in comparison to those trafficking to the cell surface but exhibiting impaired function. We also generate p rotein i nteraction p rofiles (PIPs) in the presence of therapeutics to assess their impact on the profiles of the disease-causing variants. The high definition PIP of each variant and their causal impact on function provides a critical example of how ProVarA can be utilized for the advancement of a personalized medicine for any familial and/or somatic disease where alterations in the sequence impact human health.

The ProVarA Methodology

To identify the cohort of proteins that are differentially interacting with a broad spectrum of CFTR variants we employed the co -purifying p rotein i dentification t echnology (CoPIT) methodology [ 23 ], an immunoprecipitation based proteomic approach we have developed to compare the cohort of recovered protein with each of the CFTR variants of interest.

In order to validate our transient transduction system, we first compared our WT- and F508del-CFTR PIPs ( Table S1 & S2 ) with the recently published WT- and F508del interactomes generated using CFBE41o- cell lines stably expressing WT or F508del transgenes, respectively [ 23 ]. A pair-wise comparison of the protein ID revealed a 37.2% and 30.7% overlap between WT and F508del PIPs respectively. In our previous study, we selected 52 proteins exhibiting a higher affinity for F508del- relative to WT-CFTR for siRNA-mediated knockdown experiments to assess their impact on F508del-CFTR trafficking and function, with 31 siRNAs correcting the trafficking defect associated with this disease-associated variant [ 23 ]. Our current F508del PIP recovered 19 (36.5%) of these proteins, 16 of which exhibit a higher affinity for F508del- relative to WT-CFTR, including 10 of the corrective siRNA targets. Furthermore, we identified 6 of 8 targets shown to restore F508del-CFTR chloride channel conductance in patient-derived primary human bronchial epithelial cells (hBE). Taken together, our current F508del PIP recapitulates many of the key features seen in CFBE41o- cell lines stably expressing WT- or F508del-CFTR transgenes [ 20 ]. The differences seen likely reflect the aberrantly elevated expression of the F508del transgene relative to the expression of the WT-CFTR transgene in the stable cell lines [ 20 ] a condition inconsistent with the low levels of F508del expression relative to that of WT-CFTR seen in patient derived primary human bronchial epithelial (hBE) cells but which is captured in our AVV transient expression system ( Figure 1A ) and reflected in our new F508del PIP.

To compare the composition of the PIPs of the CFTR variants ( Tables S1 – 6 ), we used the list of proteins that exhibit a statistically significant difference in recovery relative to the control CFTR immunoprecipitation in GFP transduced CFBE41o- null cells to calculate the pairwise Jaccard similarity coefficients, a score which reflects the number of common proteins in the two datasets relative to the total proteins in the merged datasets where identical data sets would score as a 1 and completely unique datasets as a 0. Our analysis revealed that the Class II variants are more closely related to one another than to either WT- or G551D-CFTR ( Figure 1B ). An expected observation given the respective trafficking agendas of these variant that limit ER export, a feature evident by their accumulation in the ER-restricted band B N-linked glycoform and their lack of the post-Golgi band C glycoform relative to that seen with WT-CFTR ( Figure 1A ). To more accurately compare the PIPs of these CF-associated mutations we normalized the recovery of all proteins in each PIP of a given variant to the amount of the CFTR variant recovered ( Tables S1 – 6 ). This bait normalization (BN) eliminates the differential recovery of common proteins due to differences seen in CFTR expression and recovery by immunoprecipitation among the variants ( Figure 1A ). A heatmap of the BN PIPs ( Figure 1C ) reveal clear differences between the ER-restricted Class II variants and both WT- and G551D-CFTR. While the G551D-CFTR variant exhibits a similar expression and trafficking profile to that seen for WT-CFTR ( Figure 1A ), we note that the composition of their interactomes are significantly different as determined by the Jaccard similarity index ( Figure 1B & C ). These results illustrate that PIPs can capture differences in the protein fold that reflect a WT-like spatial distribution of a non-functional variant.

An analysis of the composition of the pairwise differential BN-PIPs ( Tables S2 – 6 ), revealed 588 common proteins ( Table 1 , Figure 2A ). These data indicate that many features of the CFTR fold are retained despite the presence of different disease-causing variations suggesting that a limited number of intermediate steps are impacting the ability of these mutants to achieve a functional fold. However, when we filtered the data for proteins that exhibit a statistically significant difference (p < 0.05) in the BN peptide intensity relative to that seen with WT-CFTR ( Tables S2 – 6 ), we observed clear differences among the variants with 70 common proteins exhibiting a statistically significant difference in their affinity for the variants relative to that seen for WT-CFTR ( Table 1 , Figure 2B ). A separation of these proteins into those that exhibit a significant increase ( Figure 2C ) or decrease ( Figure 2D ) in affinity for the variants, relative to WT-CFTR, highlight an additional layer of divergence between Class II and Class III variants. We observe that there are more proteins recovered with Class II variants that exhibit a higher binding affinity for the variant compared to WT ( Figure 2C & D ). This is consistent with the idea that these ER-restricted variants suffer from either a preponderance of aberrant interactions that WT-CFTR rarely or never sees, such as degradation machinery components [ 13 , 16 , 23 , 24 , 26 – 36 ] or that these variants suffer from an inability to navigate key folding intermediates and the PN components engaged in these key biogenesis steps are found in higher abundance with the variants [ 22 , 39 , 40 ]. Additionally, we also observe that the G551D variant suffers from an overall reduction in binding affinity relative to that seen with WT-CFTR ( Figure 2C & D ), suggesting that this variant is lacking key interactions despite its WT-like trafficking ability, which could, at least in part, explain its gating defect. These data also explain how both class II and class III variants can exhibit a low similarity to both WT-CFTR and to one another.

These data support the hypothesis that these PIPs can accurately characterize differences among diverse CF-causing variants and across multiple disease classes, an approach that will be useful to identify therapeutic targets for correction of either individual variants or for binning of variants with common PIP features, reflecting a common folding defect, an important step in determining the etiology of these respective variant-linked CF diseases. This represents a key step in advancing personalized medicine for CF as described below.

Discussion

Herein we describe the utility of ProVarA as a novel proteomic approach to provide a deeper understanding of how variant-specific PIP influence disease onset and progression and its response to FDA-approved therapeutics. ProVarA generates PIP based on binding affinities of proteins allowing us to classify disease causative mutations, determine causative events associated with these mutations, identify mutations that may be targeted by drugs as well as assess the mechanism of action of these small molecule therapeutics ( Figure 7 ). In essence, ProVarA can be viewed as a protein-interaction extension of the Connectivity Map concept, which utilizes transcriptomic-based cellular signatures to catalog responses to genetic perturbations and pharmacological interventions [ 105 , 106 ].

Our results highlight the analysis of 6 CF-causing variants of CFTR to demonstrate how ProVarA can be utilized to inform on the etiology of CF disease and to gain insight into key proteomic changes that are responsible, at least in part, for the therapeutic activity of the FDA-approved pharmaceuticals Ivacaftor (Vx770) and Lumacaftor (Vx809). Below we discuss the value of ProVarA for the CF community in evaluating the impact of variants on the disease state(s) and probable causes for response to therapeutics. We also briefly discuss the use of ProVarA for other familial/somatic diseases where mutation is an important feature of onset, progression, and therapeutic intervention reflecting the emergent genetic diversity in the population and its implications for precision medicine-based initiatives.
