---
pmid: '39423811'
title: PLD3 and PLD4 synthesize S,S-BMP, a key phospholipid enabling lipid degradation
  in lysosomes.
authors:
- Singh S
- Dransfeld UE
- Ambaw YA
- Lopez-Scarim J
- Farese RV Jr
- Walther TC
journal: Cell
year: '2024'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC12055030
doi: 10.1016/j.cell.2024.09.036
---

# PLD3 and PLD4 synthesize S,S-BMP, a key phospholipid enabling lipid degradation in lysosomes.
**Authors:** Singh S, Dransfeld UE, Ambaw YA, Lopez-Scarim J, Farese RV Jr, Walther TC
**Journal:** Cell (2024)
**DOI:** [10.1016/j.cell.2024.09.036](https://doi.org/10.1016/j.cell.2024.09.036)
**PMC:** [PMC12055030](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12055030/)

## Abstract

1. Cell. 2024 Nov 27;187(24):6820-6834.e24. doi: 10.1016/j.cell.2024.09.036. Epub
 2024 Oct 17.

PLD3 and PLD4 synthesize S,S-BMP, a key phospholipid enabling lipid degradation 
in lysosomes.

Singh S(1), Dransfeld UE(2), Ambaw YA(1), Lopez-Scarim J(3), Farese RV Jr(4), 
Walther TC(5).

Author information:
(1)Cell Biology Program, Sloan Kettering Institute, MSKCC, New York, NY, USA.
(2)Department of Molecular Metabolism, Harvard T.H. Chan School of Public 
Health, Boston, MA, USA.
(3)Weill Cornell/Rockefeller/Sloan Kettering Tri-Institutional MD-PhD Program, 
New York, NY, USA.
(4)Cell Biology Program, Sloan Kettering Institute, MSKCC, New York, NY, USA. 
Electronic address: rfarese@mskcc.org.
(5)Cell Biology Program, Sloan Kettering Institute, MSKCC, New York, NY, USA; 
Howard Hughes Medical Institute, New York, NY, USA. Electronic address: 
twalther@mskcc.org.

Update of
    bioRxiv. 2024 Mar 21:2024.03.21.586175. doi: 10.1101/2024.03.21.586175.

Bis(monoacylglycero)phosphate (BMP) is an abundant lysosomal phospholipid 
required for degradation of lipids, particularly gangliosides. Alterations in 
BMP levels are associated with neurodegenerative diseases. Unlike typical 
glycerophospholipids, lysosomal BMP has two chiral glycerol carbons in the S 
(rather than the R) stereo-conformation, protecting it from lysosomal 
degradation. How this unusual and yet crucial S,S-stereochemistry is achieved is 
unknown. Here, we report that phospholipases D3 and D4 (PLD3 and PLD4) 
synthesize lysosomal S,S-BMP, with either enzyme catalyzing the critical 
glycerol stereo-inversion reaction in vitro. Deletion of PLD3 or PLD4 markedly 
reduced BMP levels in cells or in murine tissues where either enzyme is highly 
expressed (brain for PLD3; spleen for PLD4), leading to gangliosidosis and 
lysosomal abnormalities. PLD3 mutants associated with neurodegenerative 
diseases, including risk of Alzheimer's disease, diminished PLD3 catalytic 
activity. We conclude that PLD3/4 enzymes synthesize lysosomal S,S-BMP, a 
crucial lipid for maintaining brain health.

Copyright © 2024 The Author(s). Published by Elsevier Inc. All rights reserved.

DOI: 10.1016/j.cell.2024.09.036
PMCID: PMC12055030
PMID: 39423811 [Indexed for MEDLINE]

Conflict of interest statement: Declaration of interests R.V.F. serves gratis as 
a board member of the Bluefield Project to Cure FTD.

## Full Text

INTRODUCTIONThe degradation of lipids (for instance, from lipoproteins, lipid droplets, or cell membranes) by acid hydrolases is a major function of lysosomes. Lysosomal lipid degradation is thought to occur predominantly in the lysosomal lumen because the delimiting membrane of lysosomes is protected from acid hydrolases by an extensive glycocalyx. The degradation of membrane lipids in the lysosomal lumen requires the formation of intralumenal vesicles (ILVs) to provide acid hydrolases access to membrane lipids. ILV membranes differ in composition from other cell membranes. They are enriched with a specific phospholipid, bis(monoacylglycero)phosphate (BMP), which constitutes up to 70 mol % of ILV phospholipids1 and may be involved in ILV formation.2,3 A key feature of BMP is that it is negatively charged at lysosomal pH (4.5–5.0), which is thought to enable binding of positively charged regions of specific acid hydrolases to ILV membranes.4–7 BMP-mediated lipid degradation is crucial for cells. For example, BMP deficiency leads to defects in the degradation of ganglioside lipids (sialic-acid-containing glycosphingolipids that are abundant in the brain),8 and excessive ganglioside accumulation appears to be toxic to the nervous system.9,10 Additionally, alterations in BMP-mediated lysosomal lipid degradation are increasingly recognized as a hallmark of neurodevelopmental and neurodegenerative diseases.11–13To maintain BMP levels in ILVs, BMP must avoid degradation by lysosomal phospholipases. How does this lipid resist lysosomal degradation? This property is apparently provided by the unusual stereochemistry of lysosomal BMP. BMP has two glycerol molecules, and lysosomal BMP has both glycerol chiral carbons in the S- rather than the R-stereo-conformation, which is typically found in glycerophospholipids (Figure S1A).14–17 The S,S-configuration is thought to protect BMP from degradation by lysosomal acid phospholipases.5,18–20The key question for understanding BMP synthesis is, therefore, how cells synthesize the S,S-stereo-isoform of BMP. BMP is thought to be synthesized in a multistep pathway from the precursor R,S-phosphatidylglycerol (PG)21,22 (Figure 1A). Inasmuch as the precursor lipid PG is an isomer of BMP, its synthesis pathway can be viewed as requiring two principal steps: a rearrangement of acyl moieties around the glycerol-phosphate-glycerol backbone and a stereo-conversion of an R-glycerol to an S-glycerol to generate a molecule with both glycerols in the S-configuration. The latter step is crucial to yield S,S-BMP, which is the form found in mammalian lysosomes.14–17,23Here, we report the identification of the lysosomal PLD3 and PLD4 as enzymes that mediate the synthesis of S,S-BMP. We show that these enzymes, expressed in cultured cells or purified, catalyze BMP synthesis and that the product of these enzymes is lysosomally stable S,S-BMP. Further, these enzymes are required to maintain BMP levels in cells or tissues where they are expressed, and the loss of these enzymes leads to BMP deficiency, ganglioside accumulation, and lysosomal abnormalities.

INTRODUCTION

The degradation of lipids (for instance, from lipoproteins, lipid droplets, or cell membranes) by acid hydrolases is a major function of lysosomes. Lysosomal lipid degradation is thought to occur predominantly in the lysosomal lumen because the delimiting membrane of lysosomes is protected from acid hydrolases by an extensive glycocalyx. The degradation of membrane lipids in the lysosomal lumen requires the formation of intralumenal vesicles (ILVs) to provide acid hydrolases access to membrane lipids. ILV membranes differ in composition from other cell membranes. They are enriched with a specific phospholipid, bis(monoacylglycero)phosphate (BMP), which constitutes up to 70 mol % of ILV phospholipids1 and may be involved in ILV formation.2,3 A key feature of BMP is that it is negatively charged at lysosomal pH (4.5–5.0), which is thought to enable binding of positively charged regions of specific acid hydrolases to ILV membranes.4–7 BMP-mediated lipid degradation is crucial for cells. For example, BMP deficiency leads to defects in the degradation of ganglioside lipids (sialic-acid-containing glycosphingolipids that are abundant in the brain),8 and excessive ganglioside accumulation appears to be toxic to the nervous system.9,10 Additionally, alterations in BMP-mediated lysosomal lipid degradation are increasingly recognized as a hallmark of neurodevelopmental and neurodegenerative diseases.11–13

To maintain BMP levels in ILVs, BMP must avoid degradation by lysosomal phospholipases. How does this lipid resist lysosomal degradation? This property is apparently provided by the unusual stereochemistry of lysosomal BMP. BMP has two glycerol molecules, and lysosomal BMP has both glycerol chiral carbons in the S- rather than the R-stereo-conformation, which is typically found in glycerophospholipids (Figure S1A).14–17 The S,S-configuration is thought to protect BMP from degradation by lysosomal acid phospholipases.5,18–20

The key question for understanding BMP synthesis is, therefore, how cells synthesize the S,S-stereo-isoform of BMP. BMP is thought to be synthesized in a multistep pathway from the precursor R,S-phosphatidylglycerol (PG)21,22 (Figure 1A). Inasmuch as the precursor lipid PG is an isomer of BMP, its synthesis pathway can be viewed as requiring two principal steps: a rearrangement of acyl moieties around the glycerol-phosphate-glycerol backbone and a stereo-conversion of an R-glycerol to an S-glycerol to generate a molecule with both glycerols in the S-configuration. The latter step is crucial to yield S,S-BMP, which is the form found in mammalian lysosomes.14–17,23

Here, we report the identification of the lysosomal PLD3 and PLD4 as enzymes that mediate the synthesis of S,S-BMP. We show that these enzymes, expressed in cultured cells or purified, catalyze BMP synthesis and that the product of these enzymes is lysosomally stable S,S-BMP. Further, these enzymes are required to maintain BMP levels in cells or tissues where they are expressed, and the loss of these enzymes leads to BMP deficiency, ganglioside accumulation, and lysosomal abnormalities.

RESULTSHuman cells synthesize BMP from PG or lyso-PGPrior studies reported that BMP may be synthesized from PG.15,24–26 We thus tested this hypothesis in human microglia clone 3 (HMC3) cells. Microglial cells were chosen as being particularly relevant to ganglioside degradation in the central nervous system. Incubation of cells with medium containing 18:1/18:1 PG resulted in an ~2-fold increase in cellular BMP levels, as assessed by immunofluorescence microscopy (Figures 1B and 1C). To independently measure changes in BMP under these conditions, we developed and validated a chromatographic separation and liquid chromatography-tandem mass spectrometry (LC-MS/MS) protocol that reliably distinguished between PG and BMP lipid species, which are isobaric and thus often incorrectly measured together (Figures S1B–S1G). As expected, the analysis of cellular lipids by LC-MS/MS revealed a ~10-fold increase of 18:1/18:1 PG and a ~20-fold increase of lyso-PG in response to 18:1/18:1 PG incubation (Figure 1D). Additionally, total BMP levels increased by 2–3-fold, and analysis of BMP species revealed that numerous 18:1-containing BMP species were increased (Figure 1E). We also detected accumulation of PG species with different fatty acyl moieties, suggesting that re-acylation of 18:1 lyso-PG to PG occurred during the assays (Figure S2A). The same experiments performed in another human cell line (HEK293T) yielded similar results (Figures S2B and S2C).Our data suggested that BMP is synthesized from lyso-PG that is generated by PG hydrolysis. Indeed, incubation of either HMC3 or HEK293T cells with medium containing 18:1 lyso-PG increased lyso-PG levels and resulted in ~2–3-fold increases of different BMP species (Figures 1F, 1G, S2D, and S2E). In contrast, incubation of cells with a variety of other glycerophospholipids did not increase cellular BMP levels and instead increased levels of lysophospholipid derivatives (Figures S2F–S2K). Taken together, these data indicate that lyso-PG is a precursor of cellular BMP in mammalian cells.Human cells synthesize BMP by transphosphatidylation of lyso-PG in lysosomesA key aspect of BMP synthesis is the generation of the lysosomally stable S,S-form of BMP. We therefore sought to verify that S,S-BMP is more stable in purified lysosomes than other stereo-isoforms of BMP, as proposed.18–20,23,27 To test this, we incubated R,R-, R,S-, or S,S-stereoisomers of 18:1/18:1 BMP for 1 h with lysosomal extracts from HMC3 cells. Indeed, both the R,R- and R,S- stereoisomers of BMP were degraded to lyso-PG in the lysosomal extracts of HMC3 cells, whereas S,S-BMP was much more resistant to degradation (Figures 2A and S2L).The synthesis of S,S-BMP from R,S-lyso-PG in mammals requires the stereochemical inversion of a chiral carbon in at least one glycerol, resulting in the S-stereo-conformation of both glycerol moieties. To test whether this stereo-inversion may involve a glycerol exchange or a rearrangement of carbon substituents, we incubated lysosomal extracts with R,rac-lyso-PG and sn-1-(rac)-monoacylglycerol (MAG) containing a deuterium-labeled headgroup glycerol (18:1 d5-MAG; Figure 2B). This resulted in increased levels of 18:1/18:1 d5-BMP (and the corresponding d5 fragment; Figures 2C and 2D), which is the product of a transphosphatidylation reaction that exchanged one glycerol of lyso-PG with deuterated glycerol from MAG. The increase in d5-BMP synthesis was observed for incubations at pH 5.4 and not at neutral pH. In contrast, we found only minor increases in BMP levels above background at either pH via transacylation (generating BMP without deuterium label), where one acyl chain is transferred from a different glycerolipid to a lyso-PG molecule (Figure 2D). In vitro BMP synthesis activity was greatest for the substrates lyso-PG and MAG (Figure 2E). Other tested substrates did not yield products predicted for transphosphatidylation (Figure 2F). Incubation of HMC3 cells with 18:1 MAG alone did not significantly increase BMP levels; however, when cells were incubated with both 18:1 lyso-PG and 18:1 MAG, BMP levels were ~3-fold higher than in cells incubated with 18:1 lyso-PG alone (Figure S2M). Thus, cells and isolated lysosomes produce BMP by transphosphatidylation of lyso-PG and MAG substrates, which could lead to stereo-inversion of a glycerol moiety, a step required for S,S-BMP synthesis.PLD3 or PLD4 overexpression increases BMP levels in lysosomesWe sought to identify enzymes that catalyze the transphosphatidylation reaction to produce S,S-BMP. Previous studies showed that members of the phospholipase D (PLD) class of enzymes catalyze such reactions. For example, the bacterial phosphatidylserine synthase and cardiolipin synthases are members of the PLD enzyme family that use this catalytic mechanism.28,29 We therefore focused on PLD enzymes as candidates. Mammals have six different PLD enzymes and, consistent with previous reports,30,31 three of them (PLD1, PLD3, and PLD4) localized to lysosomes when expressed in HMC3 cells (Figures 3A and 3B). In contrast, PLD2 localized at the plasma membrane (Figure 3B), as described previously,32 and PLD5 and PLD6 localized to mitochondria (Figure S3A). The catalytic activity of each of these enzymes is not completely understood, with, for instance, PLD3 reported as a PLD33,34 and an exonuclease.35,36To test whether lysosomal PLDs synthesize BMP, we overexpressed them in HEK293T cells expressing TMEM192–3xHA, which allows for isolation of lysosomes by rapid affinity purification.37 We tested the resultant lysosomal extracts for BMP synthesis activity and found that overexpression of PLD3 or PLD4, but not PLD1, increased BMP synthesis (measured as levels above background) nearly 4-fold with added lyso-PG and MAG substrates (Figure 3C).PLDs have two conserved phosphodiesterase domains, each possessing a catalytic triad of histidine, lysine, and aspartic acid residues predicted to be essential for activity (Figures S3B and S3C). Mutations of any of these residues to alanine in PLD3 or PLD4 abolished lysosomal BMP synthesis activity (Figures 3D and 3E), but did not impair the expression or localization of the mutant proteins to lysosomes (Figures S3D and S3E). In contrast, a conservative change of E423D in PLD3 or E433D in PLD4 (aspartic acid is found at this position in other PLDs) had no detectable effect on expression, localization, or BMP synthesis activity (Figures 3D, 3E, S3D, and S3E).In humans, mutations in PLD3 are implicated as causing dominant spinocerebellar ataxia (SCA) 46.33,38 We tested one such missense mutation, PLD3 L308P, and found reduced lysosomal localization and no detectable increase in BMP levels compared with mock transfection control (Figures 3F and 3G). Immunoblotting also revealed decreased processing of the enzyme to its soluble form (Figure 3F). Another mutation, PLD3 V232M, increases the risk for Alzheimer’s disease.39 PLD3 V232M localized normally to lysosomes but exhibited ~50% reduced BMP synthesis activity as compared with wild-type (WT) PLD3 (Figures 3F and 3G).Purified PLD3 or PLD4 catalyze S,S-BMP synthesis in vitro via a transphosphatidylation reactionTo test whether PLD3 or PLD4 are sufficient for catalyzing BMP synthesis, we overexpressed and purified each protein from HEK293FT cells (Figures 4A and 4B). Proteomic analyses of the purified protein samples revealed no contaminants likely to mediate BMP synthesis (including transphosphatidylation) reactions (Table S1). Either purified PLD3 or PLD4 catalyzed the synthesis of BMP from lyso-PG and MAG substrates in vitro (Figures 4C and 4D). In contrast, purified enzymes with mutations in the presumed catalytic sites (PLD3 H416A or PLD4 H214A) lacked BMP synthesis activity. Both PLD3 and PLD4 utilized sn-1-MAG or sn-2-MAG as substrates (Figures 4C and 4D). Additionally, PLD3 and PLD4 preferred S,R-lyso-PG (lyso-PG with acylation of the S-glycerol) compared with R,racemic-lyso-PG (lyso-PG with acylation of R-glycerol), although the enzymes could utilize either lyso-PG as a substrate. Neither purified enzyme was active with R,racemic-PG as a substrate.To determine whether the products of PLD3 and PLD4 reactions were the physiological S,S-BMP stereoisomer, we established methods to distinguish different BMP stereoisomers. First, we employed chiral derivatization (using (R)-(−)-1-(1-naphthyl) ethyl isocyanate [R-NIC]) and chromatographic separation to measure different BMP stereoisomers (Figures 4E and S3F). Analyzing PLD reaction products, we found that derivatized products of PLD3 and PLD4 eluted at retention times (~23.5 min) consistent with S,S-BMP rather than R,R-BMP (~21.8 min) or R,S-BMP (~22.6 min; Figure 4F). Second, we assayed whether the products of PLD3 or PLD4 were substrates for PLA2G15, predicted to cleave glycerolipids in the R-configuration. BMP products of either PLD3 or PLD4 reactions were resistant to 15 min of PLA2G15-mediated hydrolysis, similar to S,S-BMP, whereas R,R-BMP or R,S-BMP were readily cleaved to lyso-PG. In contrast, incubation of the different BMPs with a control enzyme, PLA1 from Aspergillus oryzae, resulted in hydrolysis of all stereo-conformations of BMP, including the products of PLD3 and PLD4 (Figures 4G, S3G, and S3H). These results indicate that BMP synthesized by PLD3 or PLD4 is in the S,S-stereo-conformation (Figures 4E–4G, S3G, and S3H).PLD3 and PLD4 deficiency in human cells results in decreased BMP levels, ganglioside accumulation, and other lysosomal abnormalitiesWe tested whether PLD3 and PLD4 are required for BMP synthesis in cells. HMC3 and HEK293T cells express PLD3 but not PLD4 (Figure S4A). To analyze PLD3/4 function in BMP synthesis, we therefore generated PLD3 knockout cells by CRISPR-Cas9-mediated genome engineering (Figures 5A and S4B). BMP levels in PLD3-deficient HMC3 or HEK293T cells were 70%–80% lower than those in control cells (Figures 5B, 5C, and S4C–S4G; Table S2). BMP levels were restored by stably re-expressing PLD3 (Figures 5B and 5C, “add-back,” and S4C–S4G; Table S2). Levels of lyso-PG or PG were similar in PLD3 knockout cells (Figures S4H and S4I).Because BMP is crucial for lipid degradation, reducing its levels may result in accumulation of lysosomal lipids, such as gangliosides.12,13,18 Indeed, deletion of PLD3 from HMC3 or HEK293T cells resulted in accumulation of numerous species of gangliosides (Figures 5D and S4J), and the ganglioside accumulation could be rescued, either by adding back PLD3 or by feeding its reaction product S,S-BMP to cells (Figures 5D and S4J). The transcript and protein levels of ganglioside metabolic enzymes were not significantly changed in PLD3 knockout cells or cells fed with BMP (Figures S5A–S5C). Additionally, in vitro activities of ganglioside catabolic enzymes were unchanged with PLD3 knockout (Figure S5D). These data indicate that ganglioside accumulation in PLD3 knockout cells is likely a consequence of low BMP levels rather than due to an indirect effect, such as altered gene expression.We also examined other aspects of lysosome function in PLD3 knockout cells. Exogenous addition of lipids to cells and in vitro studies have demonstrated that BMP can promote ILV biogenesis.2 Consistent with this hypothesis, electron micrographs from PLD3 knockout cells appeared to have reduced numbers of ILVs (Figure 5E) compared with WT or PLD3 add-back cells. BMP may also facilitate cholesterol egress from lysosomes.40 Analysis of cholesterol levels revealed that PLD3 knockout cells had ~1.5-fold increased levels of cholesterol in lysosomes but no change in the level of whole-cell cholesterol (Figure 5F). Additionally, there were no detectable changes in lysosomal pH in PLD3 knockout cells (Figures S5E and S5F).We also examined PLD3 knockout cells for lysosomal membrane defects. PLD3 knockout cells exhibited increased recruitment of galectin-3 to lysosomes, indicating a response to lysosomal damage that exposed glycans (Figures 5G and 5H). We also detected a modest reduction in lysosomal protease activity, assessed by fluorescent dequenching of DQ-BSA, in PLD3 knockout cells (Figures S5G and S5H). However, the number of lysosomes per cell was similar in WT, PLD3 knockout, and PLD3 add-back cells (Figure S5I), and PLD3 knockout cells had normal levels of lysosomal proteins (Figure S5J).PLD3 was reported to function in nucleic acid degradation in lysosomes, with evidence that PLD3 can bind and cleave single-stranded DNA (ssDNA).35,36,41 Indeed, we found that the reported ssDNA substrate competed with the BMP synthesis activity of PLD3 in dose-dependent manner but required concentrations ~10-fold higher than lipid substrates to inhibit BMP synthesis by 50% (Figure S5K). Deficiency of PLD3 was also shown to cause accumulation of mitochondrial DNA in lysosomes.41 At baseline, we found no differences in mitochondrial nucleic acid levels in the lysosomes of PLD3 knockout HMC3 cells compared with WT cells (Figure S5L), but induction of mitophagy by mitochondrial division inhibitor 1 (Mdivi-1) treatment increased mitochondrial DNA more in lysosomes of PLD3 knockout cells than in WT cells, as reported.41 However, for each of the genes assayed, this effect of PLD3 deficiency was reversed by S,S-BMP addition (Figure S5L), consistent with this phenotype being a consequence of lowered BMP levels.PLD3 or PLD4 knockout mice exhibit reduced tissue levels of BMP and elevated ganglioside levelsIn vivo, PLD3 and PLD4 appear to perform redundant functions because mice with either gene knocked out are viable but mice that lack both enzymes die before weaning.35 Consistent with this hypothesis, systematic transcriptomic studies of mice42 show that expression patterns of the enzymes vary greatly between tissues, with some tissues expressing predominantly one enzyme and other tissues having both enzymes.According to systematic gene expression analyses in mice,42 PLD3 expression is highest in the central nervous system. To test the function of PLD3 in the brain, we studied global PLD3 knockout mice. Brains of PLD3 knockout mice had markedly reduced PLD3 mRNA and protein levels (Figures 6A, 6B, S6A, and S6B). PLD3 knockout mice had a minor increase in levels of PLD4 but no change in levels of other detected PLDs (Figure S6C). In general, 8- to 12-week-old PLD3 knockout mice appeared healthy and exhibited normal behavior.Brains from 8- to 12-week-old PLD3 knockout mice had markedly reduced (~70%) BMP levels and an accumulation of hexosylceramides in the brain compared with WT or heterozygous littermate controls (Figures 6C, 6D, S6D, and S6E; Table S3). In contrast, there were a few changes of BMP lipid levels in peripheral tissues, such as the spleen, where PLD3 is less expressed42 (Figure S6F). BMP deficiency results in defects in ganglioside degradation12,13 and, indeed, brains of PLD3 knockout mice had accumulations of several different gangliosides, including GM1, GM2, GM3, GD1, and GD3 (Figure 6E). In young (8- to 12-week-old) PLD3 knockout mice, we did not observe obvious neurological phenotypes; it will be interesting to see how BMP deficiency affects aged mice.In contrast to PLD3, PLD4 mRNA expression is highest in murine spleen and myeloid cells.42 To test the function of PLD4 in vivo, we studied PLD4 knockout mice generated by CRISPR-Cas9-mediated genome engineering. Knockout mice had markedly reduced PLD4 mRNA and protein levels in the spleen (Figures 7A, 7B, S6G, and S6H). There were no differences in levels of other detected PLDs (Figure S6I).Lipidomic analyses of PLD4 knockout mice revealed that PLD4 deficiency led to markedly decreased BMP levels (~80%) in the spleen (Figures 7C, 7D, S6J, and S6K; Table S4), whereas the deletion had little effect on BMP levels in the brain (Figure S6L). The reduction of BMP levels in the spleen correlated with an ~2-fold increase in the levels of GM3, GD1a, GD2, GT1, and GT3 gangliosides compared with WT or heterozygous littermate controls (Figure 7E). In agreement with previous reports,35,36 PLD4 knockout mice exhibited splenomegaly (Figures S6M and S6N).

Human cells synthesize BMP from PG or lyso-PGPrior studies reported that BMP may be synthesized from PG.15,24–26 We thus tested this hypothesis in human microglia clone 3 (HMC3) cells. Microglial cells were chosen as being particularly relevant to ganglioside degradation in the central nervous system. Incubation of cells with medium containing 18:1/18:1 PG resulted in an ~2-fold increase in cellular BMP levels, as assessed by immunofluorescence microscopy (Figures 1B and 1C). To independently measure changes in BMP under these conditions, we developed and validated a chromatographic separation and liquid chromatography-tandem mass spectrometry (LC-MS/MS) protocol that reliably distinguished between PG and BMP lipid species, which are isobaric and thus often incorrectly measured together (Figures S1B–S1G). As expected, the analysis of cellular lipids by LC-MS/MS revealed a ~10-fold increase of 18:1/18:1 PG and a ~20-fold increase of lyso-PG in response to 18:1/18:1 PG incubation (Figure 1D). Additionally, total BMP levels increased by 2–3-fold, and analysis of BMP species revealed that numerous 18:1-containing BMP species were increased (Figure 1E). We also detected accumulation of PG species with different fatty acyl moieties, suggesting that re-acylation of 18:1 lyso-PG to PG occurred during the assays (Figure S2A). The same experiments performed in another human cell line (HEK293T) yielded similar results (Figures S2B and S2C).Our data suggested that BMP is synthesized from lyso-PG that is generated by PG hydrolysis. Indeed, incubation of either HMC3 or HEK293T cells with medium containing 18:1 lyso-PG increased lyso-PG levels and resulted in ~2–3-fold increases of different BMP species (Figures 1F, 1G, S2D, and S2E). In contrast, incubation of cells with a variety of other glycerophospholipids did not increase cellular BMP levels and instead increased levels of lysophospholipid derivatives (Figures S2F–S2K). Taken together, these data indicate that lyso-PG is a precursor of cellular BMP in mammalian cells.

Human cells synthesize BMP from PG or lyso-PG

Prior studies reported that BMP may be synthesized from PG.15,24–26 We thus tested this hypothesis in human microglia clone 3 (HMC3) cells. Microglial cells were chosen as being particularly relevant to ganglioside degradation in the central nervous system. Incubation of cells with medium containing 18:1/18:1 PG resulted in an ~2-fold increase in cellular BMP levels, as assessed by immunofluorescence microscopy (Figures 1B and 1C). To independently measure changes in BMP under these conditions, we developed and validated a chromatographic separation and liquid chromatography-tandem mass spectrometry (LC-MS/MS) protocol that reliably distinguished between PG and BMP lipid species, which are isobaric and thus often incorrectly measured together (Figures S1B–S1G). As expected, the analysis of cellular lipids by LC-MS/MS revealed a ~10-fold increase of 18:1/18:1 PG and a ~20-fold increase of lyso-PG in response to 18:1/18:1 PG incubation (Figure 1D). Additionally, total BMP levels increased by 2–3-fold, and analysis of BMP species revealed that numerous 18:1-containing BMP species were increased (Figure 1E). We also detected accumulation of PG species with different fatty acyl moieties, suggesting that re-acylation of 18:1 lyso-PG to PG occurred during the assays (Figure S2A). The same experiments performed in another human cell line (HEK293T) yielded similar results (Figures S2B and S2C).

Our data suggested that BMP is synthesized from lyso-PG that is generated by PG hydrolysis. Indeed, incubation of either HMC3 or HEK293T cells with medium containing 18:1 lyso-PG increased lyso-PG levels and resulted in ~2–3-fold increases of different BMP species (Figures 1F, 1G, S2D, and S2E). In contrast, incubation of cells with a variety of other glycerophospholipids did not increase cellular BMP levels and instead increased levels of lysophospholipid derivatives (Figures S2F–S2K). Taken together, these data indicate that lyso-PG is a precursor of cellular BMP in mammalian cells.

Human cells synthesize BMP by transphosphatidylation of lyso-PG in lysosomesA key aspect of BMP synthesis is the generation of the lysosomally stable S,S-form of BMP. We therefore sought to verify that S,S-BMP is more stable in purified lysosomes than other stereo-isoforms of BMP, as proposed.18–20,23,27 To test this, we incubated R,R-, R,S-, or S,S-stereoisomers of 18:1/18:1 BMP for 1 h with lysosomal extracts from HMC3 cells. Indeed, both the R,R- and R,S- stereoisomers of BMP were degraded to lyso-PG in the lysosomal extracts of HMC3 cells, whereas S,S-BMP was much more resistant to degradation (Figures 2A and S2L).The synthesis of S,S-BMP from R,S-lyso-PG in mammals requires the stereochemical inversion of a chiral carbon in at least one glycerol, resulting in the S-stereo-conformation of both glycerol moieties. To test whether this stereo-inversion may involve a glycerol exchange or a rearrangement of carbon substituents, we incubated lysosomal extracts with R,rac-lyso-PG and sn-1-(rac)-monoacylglycerol (MAG) containing a deuterium-labeled headgroup glycerol (18:1 d5-MAG; Figure 2B). This resulted in increased levels of 18:1/18:1 d5-BMP (and the corresponding d5 fragment; Figures 2C and 2D), which is the product of a transphosphatidylation reaction that exchanged one glycerol of lyso-PG with deuterated glycerol from MAG. The increase in d5-BMP synthesis was observed for incubations at pH 5.4 and not at neutral pH. In contrast, we found only minor increases in BMP levels above background at either pH via transacylation (generating BMP without deuterium label), where one acyl chain is transferred from a different glycerolipid to a lyso-PG molecule (Figure 2D). In vitro BMP synthesis activity was greatest for the substrates lyso-PG and MAG (Figure 2E). Other tested substrates did not yield products predicted for transphosphatidylation (Figure 2F). Incubation of HMC3 cells with 18:1 MAG alone did not significantly increase BMP levels; however, when cells were incubated with both 18:1 lyso-PG and 18:1 MAG, BMP levels were ~3-fold higher than in cells incubated with 18:1 lyso-PG alone (Figure S2M). Thus, cells and isolated lysosomes produce BMP by transphosphatidylation of lyso-PG and MAG substrates, which could lead to stereo-inversion of a glycerol moiety, a step required for S,S-BMP synthesis.

Human cells synthesize BMP by transphosphatidylation of lyso-PG in lysosomes

A key aspect of BMP synthesis is the generation of the lysosomally stable S,S-form of BMP. We therefore sought to verify that S,S-BMP is more stable in purified lysosomes than other stereo-isoforms of BMP, as proposed.18–20,23,27 To test this, we incubated R,R-, R,S-, or S,S-stereoisomers of 18:1/18:1 BMP for 1 h with lysosomal extracts from HMC3 cells. Indeed, both the R,R- and R,S- stereoisomers of BMP were degraded to lyso-PG in the lysosomal extracts of HMC3 cells, whereas S,S-BMP was much more resistant to degradation (Figures 2A and S2L).

The synthesis of S,S-BMP from R,S-lyso-PG in mammals requires the stereochemical inversion of a chiral carbon in at least one glycerol, resulting in the S-stereo-conformation of both glycerol moieties. To test whether this stereo-inversion may involve a glycerol exchange or a rearrangement of carbon substituents, we incubated lysosomal extracts with R,rac-lyso-PG and sn-1-(rac)-monoacylglycerol (MAG) containing a deuterium-labeled headgroup glycerol (18:1 d5-MAG; Figure 2B). This resulted in increased levels of 18:1/18:1 d5-BMP (and the corresponding d5 fragment; Figures 2C and 2D), which is the product of a transphosphatidylation reaction that exchanged one glycerol of lyso-PG with deuterated glycerol from MAG. The increase in d5-BMP synthesis was observed for incubations at pH 5.4 and not at neutral pH. In contrast, we found only minor increases in BMP levels above background at either pH via transacylation (generating BMP without deuterium label), where one acyl chain is transferred from a different glycerolipid to a lyso-PG molecule (Figure 2D). In vitro BMP synthesis activity was greatest for the substrates lyso-PG and MAG (Figure 2E). Other tested substrates did not yield products predicted for transphosphatidylation (Figure 2F). Incubation of HMC3 cells with 18:1 MAG alone did not significantly increase BMP levels; however, when cells were incubated with both 18:1 lyso-PG and 18:1 MAG, BMP levels were ~3-fold higher than in cells incubated with 18:1 lyso-PG alone (Figure S2M). Thus, cells and isolated lysosomes produce BMP by transphosphatidylation of lyso-PG and MAG substrates, which could lead to stereo-inversion of a glycerol moiety, a step required for S,S-BMP synthesis.

PLD3 or PLD4 overexpression increases BMP levels in lysosomesWe sought to identify enzymes that catalyze the transphosphatidylation reaction to produce S,S-BMP. Previous studies showed that members of the phospholipase D (PLD) class of enzymes catalyze such reactions. For example, the bacterial phosphatidylserine synthase and cardiolipin synthases are members of the PLD enzyme family that use this catalytic mechanism.28,29 We therefore focused on PLD enzymes as candidates. Mammals have six different PLD enzymes and, consistent with previous reports,30,31 three of them (PLD1, PLD3, and PLD4) localized to lysosomes when expressed in HMC3 cells (Figures 3A and 3B). In contrast, PLD2 localized at the plasma membrane (Figure 3B), as described previously,32 and PLD5 and PLD6 localized to mitochondria (Figure S3A). The catalytic activity of each of these enzymes is not completely understood, with, for instance, PLD3 reported as a PLD33,34 and an exonuclease.35,36To test whether lysosomal PLDs synthesize BMP, we overexpressed them in HEK293T cells expressing TMEM192–3xHA, which allows for isolation of lysosomes by rapid affinity purification.37 We tested the resultant lysosomal extracts for BMP synthesis activity and found that overexpression of PLD3 or PLD4, but not PLD1, increased BMP synthesis (measured as levels above background) nearly 4-fold with added lyso-PG and MAG substrates (Figure 3C).PLDs have two conserved phosphodiesterase domains, each possessing a catalytic triad of histidine, lysine, and aspartic acid residues predicted to be essential for activity (Figures S3B and S3C). Mutations of any of these residues to alanine in PLD3 or PLD4 abolished lysosomal BMP synthesis activity (Figures 3D and 3E), but did not impair the expression or localization of the mutant proteins to lysosomes (Figures S3D and S3E). In contrast, a conservative change of E423D in PLD3 or E433D in PLD4 (aspartic acid is found at this position in other PLDs) had no detectable effect on expression, localization, or BMP synthesis activity (Figures 3D, 3E, S3D, and S3E).In humans, mutations in PLD3 are implicated as causing dominant spinocerebellar ataxia (SCA) 46.33,38 We tested one such missense mutation, PLD3 L308P, and found reduced lysosomal localization and no detectable increase in BMP levels compared with mock transfection control (Figures 3F and 3G). Immunoblotting also revealed decreased processing of the enzyme to its soluble form (Figure 3F). Another mutation, PLD3 V232M, increases the risk for Alzheimer’s disease.39 PLD3 V232M localized normally to lysosomes but exhibited ~50% reduced BMP synthesis activity as compared with wild-type (WT) PLD3 (Figures 3F and 3G).

PLD3 or PLD4 overexpression increases BMP levels in lysosomes

We sought to identify enzymes that catalyze the transphosphatidylation reaction to produce S,S-BMP. Previous studies showed that members of the phospholipase D (PLD) class of enzymes catalyze such reactions. For example, the bacterial phosphatidylserine synthase and cardiolipin synthases are members of the PLD enzyme family that use this catalytic mechanism.28,29 We therefore focused on PLD enzymes as candidates. Mammals have six different PLD enzymes and, consistent with previous reports,30,31 three of them (PLD1, PLD3, and PLD4) localized to lysosomes when expressed in HMC3 cells (Figures 3A and 3B). In contrast, PLD2 localized at the plasma membrane (Figure 3B), as described previously,32 and PLD5 and PLD6 localized to mitochondria (Figure S3A). The catalytic activity of each of these enzymes is not completely understood, with, for instance, PLD3 reported as a PLD33,34 and an exonuclease.35,36

To test whether lysosomal PLDs synthesize BMP, we overexpressed them in HEK293T cells expressing TMEM192–3xHA, which allows for isolation of lysosomes by rapid affinity purification.37 We tested the resultant lysosomal extracts for BMP synthesis activity and found that overexpression of PLD3 or PLD4, but not PLD1, increased BMP synthesis (measured as levels above background) nearly 4-fold with added lyso-PG and MAG substrates (Figure 3C).

PLDs have two conserved phosphodiesterase domains, each possessing a catalytic triad of histidine, lysine, and aspartic acid residues predicted to be essential for activity (Figures S3B and S3C). Mutations of any of these residues to alanine in PLD3 or PLD4 abolished lysosomal BMP synthesis activity (Figures 3D and 3E), but did not impair the expression or localization of the mutant proteins to lysosomes (Figures S3D and S3E). In contrast, a conservative change of E423D in PLD3 or E433D in PLD4 (aspartic acid is found at this position in other PLDs) had no detectable effect on expression, localization, or BMP synthesis activity (Figures 3D, 3E, S3D, and S3E).

In humans, mutations in PLD3 are implicated as causing dominant spinocerebellar ataxia (SCA) 46.33,38 We tested one such missense mutation, PLD3 L308P, and found reduced lysosomal localization and no detectable increase in BMP levels compared with mock transfection control (Figures 3F and 3G). Immunoblotting also revealed decreased processing of the enzyme to its soluble form (Figure 3F). Another mutation, PLD3 V232M, increases the risk for Alzheimer’s disease.39 PLD3 V232M localized normally to lysosomes but exhibited ~50% reduced BMP synthesis activity as compared with wild-type (WT) PLD3 (Figures 3F and 3G).

Purified PLD3 or PLD4 catalyze S,S-BMP synthesis in vitro via a transphosphatidylation reactionTo test whether PLD3 or PLD4 are sufficient for catalyzing BMP synthesis, we overexpressed and purified each protein from HEK293FT cells (Figures 4A and 4B). Proteomic analyses of the purified protein samples revealed no contaminants likely to mediate BMP synthesis (including transphosphatidylation) reactions (Table S1). Either purified PLD3 or PLD4 catalyzed the synthesis of BMP from lyso-PG and MAG substrates in vitro (Figures 4C and 4D). In contrast, purified enzymes with mutations in the presumed catalytic sites (PLD3 H416A or PLD4 H214A) lacked BMP synthesis activity. Both PLD3 and PLD4 utilized sn-1-MAG or sn-2-MAG as substrates (Figures 4C and 4D). Additionally, PLD3 and PLD4 preferred S,R-lyso-PG (lyso-PG with acylation of the S-glycerol) compared with R,racemic-lyso-PG (lyso-PG with acylation of R-glycerol), although the enzymes could utilize either lyso-PG as a substrate. Neither purified enzyme was active with R,racemic-PG as a substrate.To determine whether the products of PLD3 and PLD4 reactions were the physiological S,S-BMP stereoisomer, we established methods to distinguish different BMP stereoisomers. First, we employed chiral derivatization (using (R)-(−)-1-(1-naphthyl) ethyl isocyanate [R-NIC]) and chromatographic separation to measure different BMP stereoisomers (Figures 4E and S3F). Analyzing PLD reaction products, we found that derivatized products of PLD3 and PLD4 eluted at retention times (~23.5 min) consistent with S,S-BMP rather than R,R-BMP (~21.8 min) or R,S-BMP (~22.6 min; Figure 4F). Second, we assayed whether the products of PLD3 or PLD4 were substrates for PLA2G15, predicted to cleave glycerolipids in the R-configuration. BMP products of either PLD3 or PLD4 reactions were resistant to 15 min of PLA2G15-mediated hydrolysis, similar to S,S-BMP, whereas R,R-BMP or R,S-BMP were readily cleaved to lyso-PG. In contrast, incubation of the different BMPs with a control enzyme, PLA1 from Aspergillus oryzae, resulted in hydrolysis of all stereo-conformations of BMP, including the products of PLD3 and PLD4 (Figures 4G, S3G, and S3H). These results indicate that BMP synthesized by PLD3 or PLD4 is in the S,S-stereo-conformation (Figures 4E–4G, S3G, and S3H).

Purified PLD3 or PLD4 catalyze S,S-BMP synthesis in vitro via a transphosphatidylation reaction

To test whether PLD3 or PLD4 are sufficient for catalyzing BMP synthesis, we overexpressed and purified each protein from HEK293FT cells (Figures 4A and 4B). Proteomic analyses of the purified protein samples revealed no contaminants likely to mediate BMP synthesis (including transphosphatidylation) reactions (Table S1). Either purified PLD3 or PLD4 catalyzed the synthesis of BMP from lyso-PG and MAG substrates in vitro (Figures 4C and 4D). In contrast, purified enzymes with mutations in the presumed catalytic sites (PLD3 H416A or PLD4 H214A) lacked BMP synthesis activity. Both PLD3 and PLD4 utilized sn-1-MAG or sn-2-MAG as substrates (Figures 4C and 4D). Additionally, PLD3 and PLD4 preferred S,R-lyso-PG (lyso-PG with acylation of the S-glycerol) compared with R,racemic-lyso-PG (lyso-PG with acylation of R-glycerol), although the enzymes could utilize either lyso-PG as a substrate. Neither purified enzyme was active with R,racemic-PG as a substrate.

To determine whether the products of PLD3 and PLD4 reactions were the physiological S,S-BMP stereoisomer, we established methods to distinguish different BMP stereoisomers. First, we employed chiral derivatization (using (R)-(−)-1-(1-naphthyl) ethyl isocyanate [R-NIC]) and chromatographic separation to measure different BMP stereoisomers (Figures 4E and S3F). Analyzing PLD reaction products, we found that derivatized products of PLD3 and PLD4 eluted at retention times (~23.5 min) consistent with S,S-BMP rather than R,R-BMP (~21.8 min) or R,S-BMP (~22.6 min; Figure 4F). Second, we assayed whether the products of PLD3 or PLD4 were substrates for PLA2G15, predicted to cleave glycerolipids in the R-configuration. BMP products of either PLD3 or PLD4 reactions were resistant to 15 min of PLA2G15-mediated hydrolysis, similar to S,S-BMP, whereas R,R-BMP or R,S-BMP were readily cleaved to lyso-PG. In contrast, incubation of the different BMPs with a control enzyme, PLA1 from Aspergillus oryzae, resulted in hydrolysis of all stereo-conformations of BMP, including the products of PLD3 and PLD4 (Figures 4G, S3G, and S3H). These results indicate that BMP synthesized by PLD3 or PLD4 is in the S,S-stereo-conformation (Figures 4E–4G, S3G, and S3H).

PLD3 and PLD4 deficiency in human cells results in decreased BMP levels, ganglioside accumulation, and other lysosomal abnormalitiesWe tested whether PLD3 and PLD4 are required for BMP synthesis in cells. HMC3 and HEK293T cells express PLD3 but not PLD4 (Figure S4A). To analyze PLD3/4 function in BMP synthesis, we therefore generated PLD3 knockout cells by CRISPR-Cas9-mediated genome engineering (Figures 5A and S4B). BMP levels in PLD3-deficient HMC3 or HEK293T cells were 70%–80% lower than those in control cells (Figures 5B, 5C, and S4C–S4G; Table S2). BMP levels were restored by stably re-expressing PLD3 (Figures 5B and 5C, “add-back,” and S4C–S4G; Table S2). Levels of lyso-PG or PG were similar in PLD3 knockout cells (Figures S4H and S4I).Because BMP is crucial for lipid degradation, reducing its levels may result in accumulation of lysosomal lipids, such as gangliosides.12,13,18 Indeed, deletion of PLD3 from HMC3 or HEK293T cells resulted in accumulation of numerous species of gangliosides (Figures 5D and S4J), and the ganglioside accumulation could be rescued, either by adding back PLD3 or by feeding its reaction product S,S-BMP to cells (Figures 5D and S4J). The transcript and protein levels of ganglioside metabolic enzymes were not significantly changed in PLD3 knockout cells or cells fed with BMP (Figures S5A–S5C). Additionally, in vitro activities of ganglioside catabolic enzymes were unchanged with PLD3 knockout (Figure S5D). These data indicate that ganglioside accumulation in PLD3 knockout cells is likely a consequence of low BMP levels rather than due to an indirect effect, such as altered gene expression.We also examined other aspects of lysosome function in PLD3 knockout cells. Exogenous addition of lipids to cells and in vitro studies have demonstrated that BMP can promote ILV biogenesis.2 Consistent with this hypothesis, electron micrographs from PLD3 knockout cells appeared to have reduced numbers of ILVs (Figure 5E) compared with WT or PLD3 add-back cells. BMP may also facilitate cholesterol egress from lysosomes.40 Analysis of cholesterol levels revealed that PLD3 knockout cells had ~1.5-fold increased levels of cholesterol in lysosomes but no change in the level of whole-cell cholesterol (Figure 5F). Additionally, there were no detectable changes in lysosomal pH in PLD3 knockout cells (Figures S5E and S5F).We also examined PLD3 knockout cells for lysosomal membrane defects. PLD3 knockout cells exhibited increased recruitment of galectin-3 to lysosomes, indicating a response to lysosomal damage that exposed glycans (Figures 5G and 5H). We also detected a modest reduction in lysosomal protease activity, assessed by fluorescent dequenching of DQ-BSA, in PLD3 knockout cells (Figures S5G and S5H). However, the number of lysosomes per cell was similar in WT, PLD3 knockout, and PLD3 add-back cells (Figure S5I), and PLD3 knockout cells had normal levels of lysosomal proteins (Figure S5J).PLD3 was reported to function in nucleic acid degradation in lysosomes, with evidence that PLD3 can bind and cleave single-stranded DNA (ssDNA).35,36,41 Indeed, we found that the reported ssDNA substrate competed with the BMP synthesis activity of PLD3 in dose-dependent manner but required concentrations ~10-fold higher than lipid substrates to inhibit BMP synthesis by 50% (Figure S5K). Deficiency of PLD3 was also shown to cause accumulation of mitochondrial DNA in lysosomes.41 At baseline, we found no differences in mitochondrial nucleic acid levels in the lysosomes of PLD3 knockout HMC3 cells compared with WT cells (Figure S5L), but induction of mitophagy by mitochondrial division inhibitor 1 (Mdivi-1) treatment increased mitochondrial DNA more in lysosomes of PLD3 knockout cells than in WT cells, as reported.41 However, for each of the genes assayed, this effect of PLD3 deficiency was reversed by S,S-BMP addition (Figure S5L), consistent with this phenotype being a consequence of lowered BMP levels.

PLD3 and PLD4 deficiency in human cells results in decreased BMP levels, ganglioside accumulation, and other lysosomal abnormalities

We tested whether PLD3 and PLD4 are required for BMP synthesis in cells. HMC3 and HEK293T cells express PLD3 but not PLD4 (Figure S4A). To analyze PLD3/4 function in BMP synthesis, we therefore generated PLD3 knockout cells by CRISPR-Cas9-mediated genome engineering (Figures 5A and S4B). BMP levels in PLD3-deficient HMC3 or HEK293T cells were 70%–80% lower than those in control cells (Figures 5B, 5C, and S4C–S4G; Table S2). BMP levels were restored by stably re-expressing PLD3 (Figures 5B and 5C, “add-back,” and S4C–S4G; Table S2). Levels of lyso-PG or PG were similar in PLD3 knockout cells (Figures S4H and S4I).

Because BMP is crucial for lipid degradation, reducing its levels may result in accumulation of lysosomal lipids, such as gangliosides.12,13,18 Indeed, deletion of PLD3 from HMC3 or HEK293T cells resulted in accumulation of numerous species of gangliosides (Figures 5D and S4J), and the ganglioside accumulation could be rescued, either by adding back PLD3 or by feeding its reaction product S,S-BMP to cells (Figures 5D and S4J). The transcript and protein levels of ganglioside metabolic enzymes were not significantly changed in PLD3 knockout cells or cells fed with BMP (Figures S5A–S5C). Additionally, in vitro activities of ganglioside catabolic enzymes were unchanged with PLD3 knockout (Figure S5D). These data indicate that ganglioside accumulation in PLD3 knockout cells is likely a consequence of low BMP levels rather than due to an indirect effect, such as altered gene expression.

We also examined other aspects of lysosome function in PLD3 knockout cells. Exogenous addition of lipids to cells and in vitro studies have demonstrated that BMP can promote ILV biogenesis.2 Consistent with this hypothesis, electron micrographs from PLD3 knockout cells appeared to have reduced numbers of ILVs (Figure 5E) compared with WT or PLD3 add-back cells. BMP may also facilitate cholesterol egress from lysosomes.40 Analysis of cholesterol levels revealed that PLD3 knockout cells had ~1.5-fold increased levels of cholesterol in lysosomes but no change in the level of whole-cell cholesterol (Figure 5F). Additionally, there were no detectable changes in lysosomal pH in PLD3 knockout cells (Figures S5E and S5F).

We also examined PLD3 knockout cells for lysosomal membrane defects. PLD3 knockout cells exhibited increased recruitment of galectin-3 to lysosomes, indicating a response to lysosomal damage that exposed glycans (Figures 5G and 5H). We also detected a modest reduction in lysosomal protease activity, assessed by fluorescent dequenching of DQ-BSA, in PLD3 knockout cells (Figures S5G and S5H). However, the number of lysosomes per cell was similar in WT, PLD3 knockout, and PLD3 add-back cells (Figure S5I), and PLD3 knockout cells had normal levels of lysosomal proteins (Figure S5J).

PLD3 was reported to function in nucleic acid degradation in lysosomes, with evidence that PLD3 can bind and cleave single-stranded DNA (ssDNA).35,36,41 Indeed, we found that the reported ssDNA substrate competed with the BMP synthesis activity of PLD3 in dose-dependent manner but required concentrations ~10-fold higher than lipid substrates to inhibit BMP synthesis by 50% (Figure S5K). Deficiency of PLD3 was also shown to cause accumulation of mitochondrial DNA in lysosomes.41 At baseline, we found no differences in mitochondrial nucleic acid levels in the lysosomes of PLD3 knockout HMC3 cells compared with WT cells (Figure S5L), but induction of mitophagy by mitochondrial division inhibitor 1 (Mdivi-1) treatment increased mitochondrial DNA more in lysosomes of PLD3 knockout cells than in WT cells, as reported.41 However, for each of the genes assayed, this effect of PLD3 deficiency was reversed by S,S-BMP addition (Figure S5L), consistent with this phenotype being a consequence of lowered BMP levels.

PLD3 or PLD4 knockout mice exhibit reduced tissue levels of BMP and elevated ganglioside levelsIn vivo, PLD3 and PLD4 appear to perform redundant functions because mice with either gene knocked out are viable but mice that lack both enzymes die before weaning.35 Consistent with this hypothesis, systematic transcriptomic studies of mice42 show that expression patterns of the enzymes vary greatly between tissues, with some tissues expressing predominantly one enzyme and other tissues having both enzymes.According to systematic gene expression analyses in mice,42 PLD3 expression is highest in the central nervous system. To test the function of PLD3 in the brain, we studied global PLD3 knockout mice. Brains of PLD3 knockout mice had markedly reduced PLD3 mRNA and protein levels (Figures 6A, 6B, S6A, and S6B). PLD3 knockout mice had a minor increase in levels of PLD4 but no change in levels of other detected PLDs (Figure S6C). In general, 8- to 12-week-old PLD3 knockout mice appeared healthy and exhibited normal behavior.Brains from 8- to 12-week-old PLD3 knockout mice had markedly reduced (~70%) BMP levels and an accumulation of hexosylceramides in the brain compared with WT or heterozygous littermate controls (Figures 6C, 6D, S6D, and S6E; Table S3). In contrast, there were a few changes of BMP lipid levels in peripheral tissues, such as the spleen, where PLD3 is less expressed42 (Figure S6F). BMP deficiency results in defects in ganglioside degradation12,13 and, indeed, brains of PLD3 knockout mice had accumulations of several different gangliosides, including GM1, GM2, GM3, GD1, and GD3 (Figure 6E). In young (8- to 12-week-old) PLD3 knockout mice, we did not observe obvious neurological phenotypes; it will be interesting to see how BMP deficiency affects aged mice.In contrast to PLD3, PLD4 mRNA expression is highest in murine spleen and myeloid cells.42 To test the function of PLD4 in vivo, we studied PLD4 knockout mice generated by CRISPR-Cas9-mediated genome engineering. Knockout mice had markedly reduced PLD4 mRNA and protein levels in the spleen (Figures 7A, 7B, S6G, and S6H). There were no differences in levels of other detected PLDs (Figure S6I).Lipidomic analyses of PLD4 knockout mice revealed that PLD4 deficiency led to markedly decreased BMP levels (~80%) in the spleen (Figures 7C, 7D, S6J, and S6K; Table S4), whereas the deletion had little effect on BMP levels in the brain (Figure S6L). The reduction of BMP levels in the spleen correlated with an ~2-fold increase in the levels of GM3, GD1a, GD2, GT1, and GT3 gangliosides compared with WT or heterozygous littermate controls (Figure 7E). In agreement with previous reports,35,36 PLD4 knockout mice exhibited splenomegaly (Figures S6M and S6N).

PLD3 or PLD4 knockout mice exhibit reduced tissue levels of BMP and elevated ganglioside levels

In vivo, PLD3 and PLD4 appear to perform redundant functions because mice with either gene knocked out are viable but mice that lack both enzymes die before weaning.35 Consistent with this hypothesis, systematic transcriptomic studies of mice42 show that expression patterns of the enzymes vary greatly between tissues, with some tissues expressing predominantly one enzyme and other tissues having both enzymes.

According to systematic gene expression analyses in mice,42 PLD3 expression is highest in the central nervous system. To test the function of PLD3 in the brain, we studied global PLD3 knockout mice. Brains of PLD3 knockout mice had markedly reduced PLD3 mRNA and protein levels (Figures 6A, 6B, S6A, and S6B). PLD3 knockout mice had a minor increase in levels of PLD4 but no change in levels of other detected PLDs (Figure S6C). In general, 8- to 12-week-old PLD3 knockout mice appeared healthy and exhibited normal behavior.

Brains from 8- to 12-week-old PLD3 knockout mice had markedly reduced (~70%) BMP levels and an accumulation of hexosylceramides in the brain compared with WT or heterozygous littermate controls (Figures 6C, 6D, S6D, and S6E; Table S3). In contrast, there were a few changes of BMP lipid levels in peripheral tissues, such as the spleen, where PLD3 is less expressed42 (Figure S6F). BMP deficiency results in defects in ganglioside degradation12,13 and, indeed, brains of PLD3 knockout mice had accumulations of several different gangliosides, including GM1, GM2, GM3, GD1, and GD3 (Figure 6E). In young (8- to 12-week-old) PLD3 knockout mice, we did not observe obvious neurological phenotypes; it will be interesting to see how BMP deficiency affects aged mice.

In contrast to PLD3, PLD4 mRNA expression is highest in murine spleen and myeloid cells.42 To test the function of PLD4 in vivo, we studied PLD4 knockout mice generated by CRISPR-Cas9-mediated genome engineering. Knockout mice had markedly reduced PLD4 mRNA and protein levels in the spleen (Figures 7A, 7B, S6G, and S6H). There were no differences in levels of other detected PLDs (Figure S6I).

Lipidomic analyses of PLD4 knockout mice revealed that PLD4 deficiency led to markedly decreased BMP levels (~80%) in the spleen (Figures 7C, 7D, S6J, and S6K; Table S4), whereas the deletion had little effect on BMP levels in the brain (Figure S6L). The reduction of BMP levels in the spleen correlated with an ~2-fold increase in the levels of GM3, GD1a, GD2, GT1, and GT3 gangliosides compared with WT or heterozygous littermate controls (Figure 7E). In agreement with previous reports,35,36 PLD4 knockout mice exhibited splenomegaly (Figures S6M and S6N).

DISCUSSIONS,S-BMP is the major species found in lysosomes and is crucial for lysosomal lipid degradation. How it is produced in the lysosome has been a long-standing mystery. Here, we report that the lysosomal enzymes PLD3 and PLD4 catalyze S,S-BMP synthesis by mediating the crucial stereo-inversion step of the biosynthetic pathway. These enzymes were both sufficient and required to generate lysosomal BMP: purified enzymes were sufficient to catalyze S,S-BMP synthesis in vitro via a transphosphatidylation reaction, utilizing S,R-lyso-PG and MAG as substrates, and PLD3 and PLD4 were required for maintaining normal BMP levels in human cells and murine tissues.Our findings provide a more complete picture of the possible S,S-BMP biosynthetic pathway. In the current model, the starting lipid is R,S-PG, a phospholipid predominantly synthesized in mitochondria.43 How PG reaches the lysosome is unclear. In the lysosome, PG may be converted to lyso-PG by phospholipases, such as PLA2G15,44 and lyso-PG could then serve as a precursor for BMP synthesis. In the final step, PLD3 or PLD4 perform the crucial stereo-inversion to yield S,S-BMP, using R-lyso-PG and MAG as substrates (see pathway model in Figure 7F). The source of MAG for this pathway is yet uncertain, but possibilities include that it is derived from the hydrolysis of triglycerides (TG) or diacylglycerol (DAG) by lysosomal acid lipase or from a dephosphorylation of lysophospholipids by phosphatases or phospholipase C.A recent report suggests that lysosomal CLN5 may be a BMP synthase, using two R,racemic-lyso-PG molecules as substrates in a transacylation reaction that generates R,racemic-BMP.45 We purified CLN5 and found that, in vitro, it can use R,racemic-lyso-PG to synthesize R,racemic-BMP (Figure S7) but not S,S-BMP. Possibly R,S-BMP or R,R-BMP made by CLN5 may be an intermediate that would be converted to lyso-PG by one of the abundant lysosomal phospholipases, such as PLA2G15. We also deleted CLN5 in HEK293T cells and, in contrast to what was reported for this cell type,45 we found no significant changes in cellular BMP levels due to CLN5 deletion (Figure S7). The reason for the discrepancy in results between studies is unclear but could relate to differences in cells or gene manipulation or different methods for measuring BMP. In any event, we question whether any transacylase is the crucial enzyme for BMP synthesis because it would not mediate the glycerol stereo-inversion step required for S,S-BMP synthesis.Previous studies reported that PLD3 and PLD4 provide exonuclease activities in the lysosome.35,36 Although our studies do not exclude this possibility, we found no differences in nucleic acid levels in lysosomes of PLD3 knockout cells at baseline. As reported,41 we found that the induction of mitophagy increased mitochondrial DNA levels in PLD3 knockout lysosomes. However, this increase was reversed by addition of exogenous BMP, as was ganglioside accumulation (Figure S5L), suggesting that the loss of BMP is the primary defect in PLD3 deficiency. Alternatively, PLD3/4 enzymes may act as both lipases and nucleases, as reported for mitochondrial transmembrane PLD6.46–52 We did find that ssDNA could compete as a substrate for the purified PLD3/4 enzymes in vitro but only at concentrations well above lipid substrate concentrations. In vivo, PLD3/4 may prefer lipid substrates, either by binding to membranes or by an unidentified co-factor that presents lipid substrates.Our findings may be relevant to understanding neurodegenerative disease pathogenesis. The accumulation of gangliosides and other complex sphingolipids in lysosomes due to mutations of specific hydrolases has long been known to cause severe neurodevelopmental diseases, such as Gaucher’s, Tay-Sachs, and Sandhoff’s diseases.53–55 More recently, accumulation of lysosomal lipids has been increasingly recognized as being linked to adult neurodegenerative diseases, such as frontotemporal dementia (FTD) (GRN mutations13), motor neuron disease (Vps54 mutations56), or Parkinson’s disease (GBA mutations53). The brain and neurons are particularly rich in complex sphingolipids, such as gangliosides,57 and degradation of these lipids is an important function of lysosomes in the brain.7,9 Thus, an understanding of the processes that maintain lysosomal BMP levels may illuminate a better understanding of disease mechanisms and provide avenues for their treatment. Consistent with this notion, reduced BMP levels in the PLD3-deficient murine brain were associated with marked accumulation of gangliosides. This is similar to findings in progranulin-deficient FTD, where BMP deficiency leads to accumulation of gangliosides,13 and treatment with BMP rescued the ganglioside levels in cultured cells.PLD3 mutations in humans have also been linked to human neurodegeneration. For example, the PLD3 V232M mutation is associated with increased Alzheimer’s disease risk.39,58 We found that this mutation reduced BMP synthesis levels by more than 50%, suggesting that reduced levels of BMP may contribute to the increased Alzheimer’s disease risk. Another mutation, PLD3 L308P, was reported to cause autosomal-dominant SCA 46.33 We found impaired PLD3 trafficking (consistent with a previous report59) and reduced BMP synthesis for the PLD3 L308P mutant protein in vitro. Thus, altered BMP synthesis may also contribute to some forms of SCA. How the PLD3 L308P mutation would act dominantly is currently unclear. PLD3 is reported to form homodimers,60,61 so it is possible that the mutant protein may affect the trafficking of WT PLD3 to the lysosome.Our results for PLD3/4 enzymes playing an essential role for BMP synthesis and the elucidation of the BMP synthesis pathway in general should have broad implications, both for clarifying mechanisms of lysosomal lipid degradation and for deciphering the contribution of BMP and lysosomal lipid degradation to neurodegenerative diseases and other diseases of lysosomal storage.Limitations of the studyOur study has several limitations. First, we focus on the stereo-inversion step of BMP synthesis, and many questions remain about the overall BMP synthesis pathway, including enzymes acting upstream of PLD3/PLD4 or the sources of substrates for the various reactions. Second, how PLD3/4 recognize and prefer lipid substrates is not fully understood. We also have not yet studied the effects of PLD3 deficiency–and consequently BMP deficiency and ganglioside levels in aged mice–or how PLD3 and PLD4 compensate for each other in tissues.

S,S-BMP is the major species found in lysosomes and is crucial for lysosomal lipid degradation. How it is produced in the lysosome has been a long-standing mystery. Here, we report that the lysosomal enzymes PLD3 and PLD4 catalyze S,S-BMP synthesis by mediating the crucial stereo-inversion step of the biosynthetic pathway. These enzymes were both sufficient and required to generate lysosomal BMP: purified enzymes were sufficient to catalyze S,S-BMP synthesis in vitro via a transphosphatidylation reaction, utilizing S,R-lyso-PG and MAG as substrates, and PLD3 and PLD4 were required for maintaining normal BMP levels in human cells and murine tissues.

Our findings provide a more complete picture of the possible S,S-BMP biosynthetic pathway. In the current model, the starting lipid is R,S-PG, a phospholipid predominantly synthesized in mitochondria.43 How PG reaches the lysosome is unclear. In the lysosome, PG may be converted to lyso-PG by phospholipases, such as PLA2G15,44 and lyso-PG could then serve as a precursor for BMP synthesis. In the final step, PLD3 or PLD4 perform the crucial stereo-inversion to yield S,S-BMP, using R-lyso-PG and MAG as substrates (see pathway model in Figure 7F). The source of MAG for this pathway is yet uncertain, but possibilities include that it is derived from the hydrolysis of triglycerides (TG) or diacylglycerol (DAG) by lysosomal acid lipase or from a dephosphorylation of lysophospholipids by phosphatases or phospholipase C.

A recent report suggests that lysosomal CLN5 may be a BMP synthase, using two R,racemic-lyso-PG molecules as substrates in a transacylation reaction that generates R,racemic-BMP.45 We purified CLN5 and found that, in vitro, it can use R,racemic-lyso-PG to synthesize R,racemic-BMP (Figure S7) but not S,S-BMP. Possibly R,S-BMP or R,R-BMP made by CLN5 may be an intermediate that would be converted to lyso-PG by one of the abundant lysosomal phospholipases, such as PLA2G15. We also deleted CLN5 in HEK293T cells and, in contrast to what was reported for this cell type,45 we found no significant changes in cellular BMP levels due to CLN5 deletion (Figure S7). The reason for the discrepancy in results between studies is unclear but could relate to differences in cells or gene manipulation or different methods for measuring BMP. In any event, we question whether any transacylase is the crucial enzyme for BMP synthesis because it would not mediate the glycerol stereo-inversion step required for S,S-BMP synthesis.

Previous studies reported that PLD3 and PLD4 provide exonuclease activities in the lysosome.35,36 Although our studies do not exclude this possibility, we found no differences in nucleic acid levels in lysosomes of PLD3 knockout cells at baseline. As reported,41 we found that the induction of mitophagy increased mitochondrial DNA levels in PLD3 knockout lysosomes. However, this increase was reversed by addition of exogenous BMP, as was ganglioside accumulation (Figure S5L), suggesting that the loss of BMP is the primary defect in PLD3 deficiency. Alternatively, PLD3/4 enzymes may act as both lipases and nucleases, as reported for mitochondrial transmembrane PLD6.46–52 We did find that ssDNA could compete as a substrate for the purified PLD3/4 enzymes in vitro but only at concentrations well above lipid substrate concentrations. In vivo, PLD3/4 may prefer lipid substrates, either by binding to membranes or by an unidentified co-factor that presents lipid substrates.

Our findings may be relevant to understanding neurodegenerative disease pathogenesis. The accumulation of gangliosides and other complex sphingolipids in lysosomes due to mutations of specific hydrolases has long been known to cause severe neurodevelopmental diseases, such as Gaucher’s, Tay-Sachs, and Sandhoff’s diseases.53–55 More recently, accumulation of lysosomal lipids has been increasingly recognized as being linked to adult neurodegenerative diseases, such as frontotemporal dementia (FTD) (GRN mutations13), motor neuron disease (Vps54 mutations56), or Parkinson’s disease (GBA mutations53). The brain and neurons are particularly rich in complex sphingolipids, such as gangliosides,57 and degradation of these lipids is an important function of lysosomes in the brain.7,9 Thus, an understanding of the processes that maintain lysosomal BMP levels may illuminate a better understanding of disease mechanisms and provide avenues for their treatment. Consistent with this notion, reduced BMP levels in the PLD3-deficient murine brain were associated with marked accumulation of gangliosides. This is similar to findings in progranulin-deficient FTD, where BMP deficiency leads to accumulation of gangliosides,13 and treatment with BMP rescued the ganglioside levels in cultured cells.

PLD3 mutations in humans have also been linked to human neurodegeneration. For example, the PLD3 V232M mutation is associated with increased Alzheimer’s disease risk.39,58 We found that this mutation reduced BMP synthesis levels by more than 50%, suggesting that reduced levels of BMP may contribute to the increased Alzheimer’s disease risk. Another mutation, PLD3 L308P, was reported to cause autosomal-dominant SCA 46.33 We found impaired PLD3 trafficking (consistent with a previous report59) and reduced BMP synthesis for the PLD3 L308P mutant protein in vitro. Thus, altered BMP synthesis may also contribute to some forms of SCA. How the PLD3 L308P mutation would act dominantly is currently unclear. PLD3 is reported to form homodimers,60,61 so it is possible that the mutant protein may affect the trafficking of WT PLD3 to the lysosome.

Our results for PLD3/4 enzymes playing an essential role for BMP synthesis and the elucidation of the BMP synthesis pathway in general should have broad implications, both for clarifying mechanisms of lysosomal lipid degradation and for deciphering the contribution of BMP and lysosomal lipid degradation to neurodegenerative diseases and other diseases of lysosomal storage.

Limitations of the studyOur study has several limitations. First, we focus on the stereo-inversion step of BMP synthesis, and many questions remain about the overall BMP synthesis pathway, including enzymes acting upstream of PLD3/PLD4 or the sources of substrates for the various reactions. Second, how PLD3/4 recognize and prefer lipid substrates is not fully understood. We also have not yet studied the effects of PLD3 deficiency–and consequently BMP deficiency and ganglioside levels in aged mice–or how PLD3 and PLD4 compensate for each other in tissues.

Limitations of the study

Our study has several limitations. First, we focus on the stereo-inversion step of BMP synthesis, and many questions remain about the overall BMP synthesis pathway, including enzymes acting upstream of PLD3/PLD4 or the sources of substrates for the various reactions. Second, how PLD3/4 recognize and prefer lipid substrates is not fully understood. We also have not yet studied the effects of PLD3 deficiency–and consequently BMP deficiency and ganglioside levels in aged mice–or how PLD3 and PLD4 compensate for each other in tissues.

RESOURCE AVAILABILITYLead contactFurther requests and information concerning this study should be addressed to the lead contact, Tobias C. Walther (twalther@mskcc.org).Materials availabilityAll the reagents are available on request to the lead contact, Tobias C. Walther (twalther@mskcc.org).Data and code availabilityThere was no new code written as the part of this study. Proteomics data are deposited in the ProteomeXchangeConsortium through PRIDE partner repository with dataset identifier PRIDE: PXD046505. Lipidomics data are deposited in Dryad with a digital object identifier (DOI): https://doi.org/10.5061/dryad.1g1jwsv37. All reagents, plasmids, antibodies, and cell lines generated here are available from the Farese & Walther laboratory upon request. Other data generated are available from the corresponding authors on request.

RESOURCE AVAILABILITY

Lead contactFurther requests and information concerning this study should be addressed to the lead contact, Tobias C. Walther (twalther@mskcc.org).

Lead contact

Further requests and information concerning this study should be addressed to the lead contact, Tobias C. Walther (twalther@mskcc.org).

Materials availabilityAll the reagents are available on request to the lead contact, Tobias C. Walther (twalther@mskcc.org).

Materials availability

All the reagents are available on request to the lead contact, Tobias C. Walther (twalther@mskcc.org).

Data and code availabilityThere was no new code written as the part of this study. Proteomics data are deposited in the ProteomeXchangeConsortium through PRIDE partner repository with dataset identifier PRIDE: PXD046505. Lipidomics data are deposited in Dryad with a digital object identifier (DOI): https://doi.org/10.5061/dryad.1g1jwsv37. All reagents, plasmids, antibodies, and cell lines generated here are available from the Farese & Walther laboratory upon request. Other data generated are available from the corresponding authors on request.

Data and code availability

There was no new code written as the part of this study. Proteomics data are deposited in the ProteomeXchangeConsortium through PRIDE partner repository with dataset identifier PRIDE: PXD046505. Lipidomics data are deposited in Dryad with a digital object identifier (DOI): https://doi.org/10.5061/dryad.1g1jwsv37. All reagents, plasmids, antibodies, and cell lines generated here are available from the Farese & Walther laboratory upon request. Other data generated are available from the corresponding authors on request.

STAR★METHODSEXPERIMENTAL MODEL AND STUDY PARTICIPANT DETAILSMouse linesPLD3 heterozygous (HET) on C57BL/6N and PLD4 heterozygous (HET) on C57BL/6J mice were obtained from the International Mouse Phenotyping Consortium (University of California, Davis). Heterozygous PLD3 or PLD4 mice were bred to generate homozygous PLD3 or PLD4 knockout (KO) and wildtype (WT) mice. Mice were genotyped for PLD3 alleles with the following primers: PLD3-forward 5′-TTCCTGTGTGCCTGTCTTGC-3′, PLD3-reverse 5′-AGCTCCTTTCTCCGTCCCTC-3′, and PLD3-transverse 5′-TCGTGGTATCGTTATGCGCC -3′. PCR amplification strategy yielded a 492-bp product for WT and a 178-bp product for KO allele. PLD4 mice were genotyped using the following primers: PLD4-forward 5′- CCAGCTTGAGGTAGCTTCTCATGG -3′, PLD4-reverse 5′- CCACTGATCAAGGCTGTTGCAG-3′, and PLD4-transverse 5′- GCTGACTAGGATAGGCTGGTTGGG -3′. PCR amplification yielded a 1162-bp product for WT and a 497-bp product for KO allele. All mice were housed in a pathogen-free barrier facility with a 12 h light/12 h dark cycle and ad libitum access to water and food. Equal number of male and female mice 2–3 months of age were sacrificed for experiments. Animal procedures were approved by the Institutional Animal Care and Use Committee of the Memorial Sloan Kettering Cancer Centre on animals, following NIH guidelines.METHOD DETAILSCell cultureHMC3 (ATCC #CRL-3304), HEK293T (ATCC #CRL-3216) and their genetically manipulated versions were grown at 37°C, 5% (v/v) CO2 in Dulbecco’s Modified Eagles Medium (DMEM) (Gibco #11965126), supplemented with 10% (w/v) heat-inactivated foetal bovine serum (FBS) (Gibco #10438026) and 1x penicillin-streptomycin (Gibco #15140163). HEK293FT cells (Invitrogen #R70007) were cultured and maintained in FreeStyle™ 293 Expression Medium (Gibco #12338026) supplemented with 5% (v/v) heat-inactivated FBS (Gibco #10438026) in a shaking incubator at 37°C, 5% (v/v) CO2. Cells were periodically tested for mycoplasma contamination with a PCR detection kit (abm #G238). All cell lines were verified by short tandem repeat profiling.RT-PCRTotal RNA was isolated from cells grown in six-well plates at 70–80% confluency with an RNA extraction kit (Qiagen #74104). cDNA was synthesized using a sensiFAST cDNA synthesis kit (Meridian Bioscience #BIO-65053). Genes were amplified for melting curves using Power SYBR™ Green PCR Master Mix (Applied Biosystems™, #4367659). Primers used for the tested genes are listed in Table S5.Molecular cloningAll plasmids used either were procured from GenScript or cDNA clones were obtained from Dharmacon – Mammalian Gene Collection (listed in the reagent sheet). All truncated or tagged versions of clones were generated using Q5 high-fidelity DNA polymerase (New England Biolabs #M0515) and NEB-Gibson Assembly (New England Biolabs #E2611S). All point mutations in PLD3 and PLD4 were generated using Q5-site directed mutagenesis kit (New England Biolabs #E0554S).Lentivirus productionHEK293T cells were transfected with pLJC5-hPLD3 or pLJC5-TMEM192–3xHA (Addgene #102930) envelope proteinpCMV-VSV-G (Addgene #8454), packaging plasmid - pCMV-dR8.2 dvpr (Addgene #8455) in a 6:4:1 ratio using PEI-max (Polysciences #24765–100). Virus-containing medium was harvested two days post-transfection, filtered through a 0.45-μm syringe filter, and stored at −80 °C or used to transduce cells for stable cell line generation.Gene editingCRISPR/Cas9 mediated gene editing of HEK293T and HMC3 cells was performed as described.13 Briefly, human PLD3 sgRNA (5′-3′) was cloned into pSpCa9(BB)-2A-Puro (PX459) V2.0 (Addgene #62988) as described.62 The sequences of sgRNA are as follows: forward – CTTGACGCCACGCTCGTAGG, reverse - CCTACGAGCGTGGCGTCAAG. Cells were transfected with 2 μg of plasmids with FuGene HD transfection reagent (Promega #E2312) overnight. Thereafter, cells were selected with puromycin (1 μg/mL) until all non-transfected control cells died (2–3 days). Selected cells were serially diluted to obtain knock out (KO) clones. To generate PLD3 rescue cell lines (also called PLD3 add-back cells), PLD3 KO HMC3 and HEK293T cells were transduced with lentivirus harbouring a PLD3 insert gene using polybrene at a final concentration of 10 μg/mL. TMEM192–3x-HA tagged (also called Lyso-Tagged) cells were generated as reported.37Lipid incubationsCells were plated in six-well dishes and grown to 70–80% confluency. Thereafter, medium was changed with fresh complete DMEM medium supplemented with 25 μM of phosphatidylglycerol, phosphatidylserine, phosphatidic acid, cardiolipin, or lysophosphatidylglycerol. Phosphatidylcholine and phosphatidylethanolamine were difficult to feed, and so, their concentrations were raised to 100 μM in the medium. Lipid-supplemented medium were prepared by drying lipids in a glass vial under N2 stream. Dried lipids were resuspended in complete DMEM using a water bath sonicator for ~30 mins. Cells were maintained in lipid-supplemented medium for 6 h, and thereafter, cells were washed with ice-chilled Dulbecco’s phosphate-buffered saline (DPBS) (Gibco #14190144) six times and processed for lipid extraction or for immunofluorescence assays. For rescue experiments with BMP, knockout cells grown at ~40% confluency were treated with 25 μM of S,S-BMP (prepared as described above) for 24 h, and thereafter, cells were washed with DPBS and processed for lipid extraction. All lipids used were procured from Avanti Polar Lipids.ImmunostainingCells grown on glass cover slips were washed six times with DPBS, fixed with 4% (w/v) formaldehyde in PBS (BosterBio #AR1068) in DPBS for 15 min in the dark at room temperature (RT). Thereafter, cells were washed again three times with DPBS, permeabilized with permeabilization buffer containing 0.02% (w/v) saponin (Sigma-Aldrich #47036) and 1% (w/v) bovine serum albumin (Sigma-Aldrich #A6003) in DPBS at RT. Cells were incubated with blocking buffer containing 1% (w/v) bovine serum albumin in DPBS at RT, then incubated with primary antibody (1:100 dilution in blocking buffer) for 2 h at RT, washed three times with DPBS and incubated with secondary antibody (1:1000 dilution in blocking buffer) for 45 mins at RT. Cover slips were washed three times with DPBS and mounted on glass slides with Fluoromount Aqueous Mounting Medium (Sigma-Aldrich #F4680). Slides were dried at RT in the dark overnight before imaging. Information on the antibodies used is available in the reagent sheet.DQ-BSA dequenching assaysTo measure lysosomal protease activity, DQ-BSA (Invitrogen #D12050) was dissolved at 1 mg/mL in DPBS by water-bath sonication. Cells were plated in glass-bottom 12-well plates and grown to 60–70% confluency. Cells were washed with DPBS and fed with DQ-BSA (Invitrogen #D12050) at 100 μg/mL in complete DMEM medium for 4 h and treated with LysoTracker Red DND-99 (Invitrogen #L7528) for 15 min and subsequently imaged on a Nikon CSU-W1 SoRa microscope. Cells were maintained in a live cell chamber at 37°C, 85% humidity and 5% CO2.Lysosomal pH measurementsCells were seeded at 50,000 cells per well in 96-well glass bottom plate. Cells were allowed to adhere for 6 hours at 37°C, 85% humidity, and 5% CO2. Cells were incubated with LysoSensor™ Green DND-189 (Invitrogen #L7535) or LysoSensor™ Yellow/Blue DND-160 (Invitrogen #L7545) for 1 h at 37°C. Thereafter, cells were washed three times with DPBS, and replenished with fresh complete DMEM media with no Phenol Red, and subsequently fluorescence was measured on Tecan Spark multimode microplate reader with excitation and emission set at 440 and 500 nm for LysoSensor™ Green DND-189 and 340/380 nm excitation and 440/540 nm excitation for LysoSensor™ Yellow/Blue DND-160. Emission intensities were plotted as it as for the LysoSensor™ Green DND-189. For LysoSensor™ Yellow/Blue DND-160 dye, ratio of emission intensities at 440 nm and 540 nm was used for analysis. For positive control, cells were treated with 100 nM Bafilomycin for 4 hours prior to addition of LysoSensor™ Green DND-189 (Invitrogen #L7535).Cholesterol measurementsLysosomes were immune-isolated from cells as described in lysosome purifications section. Lysosomal lipids were extracted as described in lipid extraction section and cholesterol was measured using Amplex™-Red Cholesterol Assay Kit (Invitrogen #A12216) as per manufacturer’s instructions. Briefly, lysosomal or cellular lipid extracts were dried under inert N2 stream and re-suspended in 50 μL of reaction buffer (0.1 mM potassium phosphate, pH 7.4, 0.05 M NaCl, 5 mM cholic acid, 0.1% Triton X-100). This was mixed with 50 μL of enzyme mixture that contained 2 U/mL of horseradish peroxidase, 2 U/mL cholesterol oxidase, 300 μM Amplex Red. Reaction mixture was incubated at 37 °C in dark for 30 mins. Fluorescence was measured on Tecan Spark multimode microplate reader with excitation and emission set at 540 and 560 nm. Background fluorescence was calculated by incubating lipid extracts with reaction buffer without enzyme mixture. Linearity of the assay was determined by running standard curves for 0 to 10 μg/mL of cholesterol standard. Fluorescence values were normalized to the amount of the protein, and fold change was calculated by dividing each measurement to the mean of values for WT samples.Microscopy and image analysisFluorescence images were acquired on a Nikon CSU-W1 SoRa spinning-disc microscope fitted with an ORCA-Fusion BT Digital CMOS camera (Hamamatsu #C15440–20IP). For cells, images were acquired through a 100x SR HP Apo 1.49 NA objective (Nikon #MRD71970). Green fluorophores (IgG-Alexa-488) were exited using a 488-nm laser and red fluorophores (IgG-Alexa-562) were exited using a 561-nm laser. Emission filters were for green signal = 525/50 and for red signal = 605/52. Images were processed and prepared for figures using Fiji-ImageJ. Signals in the lysosomes (e.g., BMP, DQ-BSA or galectin-3) were quantified by masking the signal for the lysosomal marker LAMP1 or LysoTracker Red DND-99 (Invitrogen #L7528).Western blot analysisCells were scraped into ice-chilled DPBS and pelleted at 1000 g for 2 min. Cell pellets were resuspended in DPBS, lysed by probe sonication (1 sec on, 3 sec off, 60% amplitude), boiled with 4X loading dye at 95°C for 30 min for denaturation. Denatured proteins were separated on 4–15% gradient SDS-PAGE gels at 120 V and transferred to methanol-activated PVDF blotting membrane at 60 V for 3 h at 4°C. Thereafter, PVDF membranes were blocked with 5% (w/v) skim milk with 0.1% (v/v) Tween-20 in DPBS (blocking buffer) for 1 h at RT. Blocked membranes were incubated with primary antibodies (1:1000 diluted in blocking buffer) overnight at 4° C, washed three times with 0.1% (w/v) Tween-20 in DPBS (washing buffer), incubated with secondary antibody (Santa Cruz Biotechnology #sc516102 #sc2357 – 1:5000 diluted in blocking buffer) for 1 h at RT. Membranes were washed three times with washing buffer and developed with ECL chemiluminescence substrates and imaged on a BioRad ChemiDoc Imager.Lysosome purificationsLysosomes were immunoisolated as reported.37 TMEM192–3X HA-tagged cells from one 150-mm plate at 70% confluency were scraped into ice-chilled DPBS supplemented and pelleted at 1000 g for 2 min. The cell pellet was Dounce-homogenized in 1 mL ice-chilled KPBS buffer containing 136 mM KCl, 10 mM KH2PO4 adjusted to pH 7.25 with KOH. Cell lysates were spun down at 1000 g for 2 min. Supernatant was collected and incubated with anti-HA magnetic beads, washed and pre-equilibrated with KPBS buffer for 15 min at 4° C. Beads were washed with excess KPBS buffer 10 times. Proteins were eluted from the beads using lysis buffer containing 50 mM Tris, 150 mM NaCl, 1 mM EDTA, 1% (v/v) Triton X-100 adjusted to pH 7.4.Recombinant protein expression and purificationHMC3 or HEK293T cells were grown at ~50% confluency in complete DMEM medium and transfected with pSPORT6 expression plasmids carrying human versions of PLD1, PLD2, PLD3, PLD4, PLD5, and PLD6 genes, using FuGene HD transfection reagent (Promega #E2312) for 24 h and thereafter processed for immunostaining or for lysosomal isolations and western blots. For PLD3, PLD4, PLA2G15 and CLN5 purifications, HEK293FT cells were grown at 1 × 106 cells/mL and transfected with C-terminal FLAG-tagged WT or active-site mutants cloned in pcDNA3.1 using PEI-max (Polysciences #24765–100). Cells were treated with 10 mM sodium butyrate for ~16 h after transfection and cultured for two additional days. Cells were collected by centrifugation at 2500 g for 15 min, and cell pellets were rinsed with DPBS. Cell pellets were resuspended in 50 mM Tris (pH 7.5), 200 mM NaCL, 5 mM MgCL2, 1 mM EDTA, 5 mM TCEP (resuspension buffer) and lysed by probe sonication (60% amplitude, 1 sec on, 3 sec off for 10 min). Lysates were centrifuged at 25,000 g for 1 h at 4° C. Supernatants were collected, syringe filtered with a 0.45-μM syringe filter, and incubated with anti-FLAG beads (Sigma-Aldrich #A2220) pre-equilibrated with resuspension buffer without TCEP. Beads were washed 10 times with 10 column volumes of resuspension buffer without TCEP. Proteins were eluted with 50 μg/mL of 1x FLAG peptide (Sigma-Aldrich #F3290), concentrated using 30 kDa centricons (Millipore #UFC9030). Concentrated proteins were buffer exchanged with sodium citrate buffer, pH 5.4, using button dialysis, followed by centrifugation at 50,000 g for 30 min to pellet any aggregates. Protein concentration in the supernatant fraction was accessed by nanodrop spectrophotometry, and purity of the protein was accessed using 10% Coomassie-stained SDS-PAGE gel and label-free mass spectrometry.In vitro enzymatic assaysFor BMP synthesis assays, lipid substrates were prepared by resuspending the required amount of lipids in sodium citrate buffer at pH 5.4 using water-bath sonication to a final concentration of 50 μM of phospholipids and 25 μM of mono- or diacylglycerol (Avanti Polar Lipids). Substrates were incubated with extracts of affinity isolated lysosomes on anti-HA magnetic beads (3–5 μg of lysosomal proteins) or 1 μg of purified enzyme (final volume = 100 μL) for 45 min, 37 °C with constant shaking in glass vials. Thereafter, the reaction was quenched with 250 μL of 2:1 chloroform:methanol spiked with 14:0/14:0 BMP as internal standard. The mixture was vortexed and spun at 1000 g for 5 min, and the bottom organic layer was collected in a separate glass vial. Thereafter, the aqueous fraction was acidified with 2 μL of formic acid, vortexed and extracted again with 250 μL of chloroform. Organic layers from both steps were pooled, dried under a N2-stream and analyzed by LC-MS/MS. BMP formation was monitored and normalized to internal standards, the amount of protein used and the time of the reaction. For nuclease competition assay, 50 μM of S,R-lyso-PG and 25 μM of monoacylglycerol was incubated with 1 μg of purified PLD3 along with increasing concentrations of single-stranded DNA (5 μM to 5 mM). Sequence of single-stranded DNA was as reported previously36 : 5’–TCCATGACGTTCCTGATGCTAAGTATGCACTTCATCGTCAAGCAATGCTATGCAT–3’.To test the stability of BMP stereoisomers in lysosomes 100 μM of 18:1/18:1 BMP stereoisomers were incubated with 5 μg of lysosomal lysates for 1 h at 37 °C with constant shaking in glass vials. To test the stability of BMP synthesized by PLD3 or PLD4, enzymatic products were incubated with 1 μg of PLA2G15 or 5 μg of phospholipase A1 from Aspergillus oryzae (Sigma-Aldrich #L3295) for 15 mins at 37 °C. This incubation time was reduced to accommodate for the low amount of BMP synthesized by PLD3 or PLD4 compared to their commercially available synthetic standards. Reaction was quenched with 250 μL of 2:1 chloroform: methanol; and lipids were extracted as described for the BMP synthesis assay. Hydrolysis of BMP stereoisomer standards was monitored by TLC and reaction rates were measured by LC-MS/MS. PLA2G15 used in the reactions was obtained by overexpressing C-terminally FLAG-tagged human PLA2G15 in HEK293FT cells and affinity-isolation using anti-FLAG magnetic beads (Sigma-Aldrich #M8823).To test for lysosomal catabolic activity (e.g., ganglioside catabolism), a fluorescence plate reader assay was setup as described.13 Artificial substrate mimetics for lysosomal enzymes were dissolved in DMSO to stock concentrations of 100 μM. Cells were Dounce-homogenized in ice-cold DPBS and probe sonicated for 1 min (60% amplitude, 1 sec on, 3 sec off). Protein concentrations were estimated using Pierce BCA Protein Assay Kit (Pierce #23225) with BSA as control for standard curves. Lysates (25 μg) were incubated with 1 mM of fluorogenic substrates in sodium citrate buffer (total volume = 100 μL) at pH 5.4 at 37 °C for 30 min. Thereafter, reactions were quenched with 300 μL glycine-buffer, pH 10.4. Fluorescence was measured in a Tecan Spark multimode microplate reader with excitation and emission set at 350 and 440 nm. A standard curve was drawn using 4-methylumbelliferone (Sigma-Aldrich #M1381) as standards dissolved in reaction buffer + quenching buffer in ratios used for the assay. Substrates used were as follows: 4-methylumbelliferyl-β-D-galactoside for β-galactosidase, 4-methylumbelliferyl-β-D-glucopyranoside for β-glucocerebrosidase, 4-nethylumbelliferyl-2-acetamido-2-deoxy-β-D-glucopyranoside for β-hexosaminidase, 7-(α-D-glucopyranosyloxy)-4-methyl-2H-1-benzopyran-2-one for α-glucosidase, 7-(α-D-galactopyranosyloxy)-4-methyl-2H-1-benzopyran-2-one for α-galactosidase, 7-[[2-(acetylamino)-2-deoxy-α-D-galactopyranosyl]oxy]-4-methyl-2H-1-benzopyran-2-one for β-galactopyranosaminidase activity, hexadecanoic acid, 4-methyl-2-oxo-2H-1-benzopyran-7-yl ester for lysosomal acid lipase, and 4-methyl-2-oxo-2H-1-benzopyran-7-yl 5-(acetylamino)-3,5-dideoxy-α-neuraminic acid for neuraminidases. All artificial substrates were procured from Cayman Chemicals.Preparation of S,R-lyso-PGR,S-BMP (100 μM) was incubated with 5 μg of human PLA2G15 at 37 °C in glass vials with constant shaking. The reaction was monitored by running small aliquots on TLC. The reaction was quenched and extracted with excess 2:1 (v/v) chloroform:methanol. Extracts were dried under an N2 stream, and the lyso-PG mixture was separated on a C18 reverse phase column (Pheneomenex, 5 μm, 50 × 4.6 mm) using the following solvents: Solvent A= 95:5 (v/v) H2O/MeOH + 0.1% (v/v) NH4OH, solvent B= 60:35:5 (v/v) IPA/MeOH/H2O + 0.1% (v/v) NH4OH. The liquid chromatography (LC) gradient was 30 mins, with a stable flow rate of 0.5 mL/min, starting 5% B for 5 min, linear gradient of B from 5–100% over 23 min, 100% B from min 23–25, and re-equilibration with 5% B for 5 min. The isolated lyso-PG was pooled and dried under N2, weighed and used for BMP synthesis assays.Chiral derivatization and separation of BMP stereoisomersStandards of BMP stereoisomers or enzymatic products were mixed with derivatization agent (R)-(−)-1-(1-naphthyl)ethyl isocyanate (Sigma-Aldrich #220442) in a ratio of 1:10 in 1 mL of dichloromethanol (VWR BDH23373.400) and 50 μL of pyridine (Sigma-Aldrich #270970) overnight (14–16 h) at RT. The reaction mixture was dried under a stream of N2 and extracted with 2:1:1 chloroform:methanol:water. The bottom organic layer was collected and separated on YMC Chiral ART Cellulose-SZ, analytical column (YMC America #KSZ99S03–2502WT). Solvent A = 60:40 (v/v) ACN/water + 0.1% (v/v) formic acid + 10 mM ammonium formate, Solvent B = 90:10 (v/v) IPA: ACN + 0.1% (v/v) formic acid + 10 mM ammonium formate. The LC gradient was 45 min, with a stable flow rate of 0.2 mL/min, starting with 40% B for 5 min, linear gradient of 40–70% B for min 5–30, maintained at 100% B for mins 30–33, re-equilibrated with 40% B for min 33–45 at 0.3 mL/min. The column temperature was maintained at 10 °C. MS parameters were as described in LC-MS/MS analysis of lipids section.Lipid extractionLipids were extracted from cells or tissues using established protocols.63 Briefly, cells were lysed in DPBS using a probe sonicator, or tissues were Dounce-homogenized in DPBS, and lipids were extracted with a chloroform and methanol mixture, with spiked-in lipid splash mix and/or BMP 14:0/14:0, as internal standards. The final ratio of chloroform:methanol:DPBS in the extraction mixture was 2:1:1. The extraction mixture was vortexed and spun down at 1000 g for 10 min. The bottom organic phase was collected in a glass tube. The top aqueous phase was acidified with 0.5% (v/v) formic acid, vortexed and re-extracted with chloroform by spinning down as in the previous step. Both organic phases were pooled and dried under a stream of N2 and analysed by LC-MS/MS.Ganglioside extractionTissues or cells were Dounce-homogenized with ice-chilled DPBS, and methanol was added to the final ratio of 90:10 methanol: water. The extraction mixture was spun down at 1000 g for 20 min to pellet proteins, and the solvent layers containing gangliosides and other metabolites were collected in glass tubes and dried under an N2 stream. Dried extract was reconstituted in 1 mL of LC-MS-grade water and desalted by Sola HRP SPE 30mg/2MI 96-well plate 1EA (Thermo Scientific #60509–001). Desalting cartridges were cleaned three times with 1 mL of methanol, equilibrated three times with LC-MS-grade water, and then the extracts dissolved in water were loaded onto the cartridge and washed three times with water, and finally, gangliosides were eluted by 3 mL of methanol. Eluates were dried under N2 and reconstituted in 300:150:50 of MeOH:water:chloroform.TLC for lipidsExtracted lipids were separated on high-performance thin layer chromatography (Sigma-Aldrich #1056310001) in the first dimension using 70:30:3:2 (v/v/v/v) chloroform:methanol:water:concentrated ammonia as the running phase. For BMP separation, plates were dried at RT for 1 hour and run in the second dimension using 65:35:5 (v/v/v) of chloroform:methanol:water as the running phase. Plates were dried and sprayed with 5% (w/v) copper sulfate dissolved in 25% (v/v) ethanol and 8.5% (v/v) phosphoric acid, followed by charring at 60 °C for 1 h. Plates were imaged on BioRad ChemiDoc Imager.LC-MS/MS analysis of lipidsAll samples were analysed on a Vanquish UHPLC (Thermo Scientific) coupled to a Orbitrap Exploris 240 mass spectrometer (Thermo Scientific #BRE725535). For routine lipid analysis, extracts were separated on a C18 reverse phase column (Pheneomenex, 5 μm, 50 × 4.6 mm #00B-4435-E0) as described.63 Briefly, for positive mode: solvent A= 95:5 (v/v) H2O/MeOH + 0.1% formic acid + 10 mM ammonium formate, solvent B= 60:35:5 (v/v/v) isopropanol (IPA)/MeOH/H2O + 0.1% (v/v) formic acid + 10 mM ammonium formate. For negative mode, solvent A= 95:5 (v/v) H2O/MeOH + 0.1% (v/v) NH4OH, solvent B= 60:35:5 (v/v) IPA/MeOH/H2O + 0.1% (v/v) NH4OH. LC gradient was 60 min, with stable flow rate of 0.5 mL/min starting 5% B for 5 min, linear gradient of B from 5–100% over 50 min, 100% B from min 50–55, and re-equilibration with 5% B for 5 min. The column temperature was maintained at 55 °C. MS parameters were as follows: - heated ESI mode of ionization, sheath gas = 40 units, aux gas = 8 units, sweep gas =1 units, spray voltage = static, positive ion =3,400 V, negative ion = 2500 V, ion transfer tube temp= 320 °C, vaporizer temp=275 °C. Orbitrap resolution at MS1 was set to 120000, scan range 250–1800, RF lens = 60%, AGC target = standard, Orbitrap resolution at MS2 was set to 30000. EASY-IC™ was used for internal calibration. Lipids were searched and aligned for annotation in LipidSearch 5.0 (Thermo Scientific #OPTON-30880) with precursor mass error tolerance of 5 ppm and product mass error tolerance of 10 ppm, retention time deviation tolerance of 0.5 min.LC-MS/MS analysis of BMPAll samples were analysed on a Vanquish UHPLC (Thermo Scientific) coupled to a Orbitrap Exploris 240 mass spectrometer (Thermo Scientific #BRE725535). Chromatographic separation of BMP and PG was achieved on a C-30 reverse phase column (Thermo Scientific #078664, 3 mm 250 × 2.1 mm). Solvent = 60:40 (v/v) ACN/water + 0.1% (v/v) formic acid + 10 mM ammonium formate, Solvent B = 90:10 (v/v) IPA: ACN + 0.1% (v/v) formic acid + 10 mM ammonium formate. The LC gradient was 45 min, with a stable flow rate of 0.2 mL/min starting 40% B, linear gradient of 45–55% over mins 1–7, maintained at 65% B for mins 8–12, linear gradient of 65–70% B for mins 12–30, maintained at 100% B for mins 30–33, re-equilibrated with 40% for mins 33–45. Column temperature was maintained at 50 °C. MS parameters were as follows: - heated ESI mode of ionization, sheath gas = 40 units, aux gas = 8 units, sweep gas =1 units, spray voltage = static, positive ion =3,400 V, negative ion = 2500 V, ion transfer tube temp= 320 °C, vaporizer temp=275 °C. Orbitrap resolution at MS1 was set to 120000, RF lens = 60%, Orbitrap resolution at MS2 was set to 30000. Orbitrap was set up with an unscheduled targeted inclusion mass list corresponding to PG and BMP. EASY-IC™ was used for internal calibration. To avoid source fragmentation of cardiolipin confounding measurements, we inactivated source fragmentation during data acquisition with mass spectrometry. Using this method, an analysis of cardiolipin standards (CL 14:0/14:0/14:0/14:0) showed no notable source fragmentation of CL into PG or its fragment ions.PG and BMP peaks were identified using Excalibur 3.1 (Thermo Scientific #OPTON-30965), the area under the curve was normalized to the area under the curve for internal standards (14:0/14:0 BMP for cells and 15:0/18:1(d7) PG for mice tissues) and amount of protein/wet weight of tissue used. Samples for BMP analysis were run in both negative and positive ion mode. All the quantification presented in the figures of the manuscript come from the negative-mode analysis. Positive-mode analysis of BMP was used only to confirm the identify of BMP species. Targeted precursor masses for BMP, and their predicted/observed fragments are as follows:
Negative ModePositive ModeSpeciesParent IonFragment IonFragment IonParent IonFragment IonFragment IonBMP 16:0/16:0721.51255.23255.23740.54313.27313.27BMP 16:0/18:1747.52255.23281.25766.56313.27339.23BMP 16:0/18:0749.54255.23283.26768.58313.27341.31BMP 16:0/20:1775.55255.23309.28794.59313.27367.32BMP 16:0/20:4769.51255.23303.23788.54313.27361.27BMP 16:0/22:6793.51255.23327.23812.54313.27385.27BMP 16:0/17:1733.51255.23267.23752.54313.27325.27BMP 16:0/12:0665.44255.23199.17684.48313.27257.21BMP 16:0/14:0693.47255.23227.20712.51313.27285.24BMP 16:0/18:2745.51255.23279.23764.54313.27337.27BMP 16:0/20:0777.57255.23311.30796.61313.27369.34BMP 16:0/22:0805.60255.23339.33824.64313.27397.37BMP 16:0/22:1803.58255.23337.31822.62313.27395.35BMP 16:0/22:4797.54255.23331.26816.58313.27389.31BMP 16:0/24:0833.63255.23367.36852.67313.27425.4BMP 16:0/24:1831.62255.23365.34850.65313.27423.38BMP 18:1/18:1773.54281.25281.25792.58339.23339.23BMP 18:1/18:0775.55281.25283.26794.59339.23341.31BMP 18:1/20:1801.57281.25309.28820.61339.23367.32BMP 18:1/20:4795.52281.25303.23814.56339.23361.27BMP 18:1/22:6819.52281.25327.23838.56339.23385.27BMP 18:1/17:1759.52281.25267.23778.56339.23325.27BMP 18:1/12:0691.46281.25199.17710.50339.23257.21BMP 18:1/14:0719.49281.25227.20738.53339.23285.24BMP 18:1/18:2771.52281.25279.23790.56339.23337.27BMP 18:1/20:0803.58281.25311.30822.62339.23369.34BMP 18:1/22:0831.62281.25339.33850.65339.23397.37BMP 18:1/22:1829.60281.25337.31848.64339.23395.35BMP 18:1/22:4823.55281.25331.26842.59339.23389.31BMP 18:1/24:0859.65281.25367.36878.68339.23425.4BMP 18:1/24:1857.63281.25365.34876.67339.23423.38BMP 18:2/18:2769.51279.23279.23788.54337.27337.27BMP 18:2/20:1799.55279.23309.28818.59337.27367.32BMP 18:2/20:4793.51279.23303.23812.54337.27361.27BMP 18:2/22:1827.58279.23337.31846.62337.27395.35BMP 18:2/22:4821.54279.23331.26840.58337.27389.31BMP 18:2/22:6817.51279.23327.23836.54337.27385.27BMP 18:2/24:1855.62279.23365.34874.65337.27423.38BMP 20:1/20:1829.60309.28309.28848.64367.32367.32BMP 20:1/20:4823.55309.28303.23842.59367.32361.27BMP 20:1/22:1857.63309.28337.31876.67367.32395.35BMP 20:1/22:4851.58309.28331.26870.62367.32389.31BMP 20:1/22:6847.55309.28327.23866.59367.32385.27BMP 20:1/24:1885.66309.28365.34904.70367.32423.38BMP 20:4/20:4817.51303.23303.23836.54361.27361.27BMP 20:4/22:1851.58303.23337.31870.62361.27395.35BMP 20:4/22:4845.54303.23331.26864.58361.27389.31BMP 20:4/22:6841.51303.23327.23860.54361.27385.27BMP 20:4/24:1879.62303.23365.34898.65361.27423.38BMP 22:1/22:1885.66337.31337.31904.70395.35395.35BMP 22:1/22:4879.62337.31331.26898.65395.35389.31BMP 22:1/22:6875.58337.31327.23894.62395.35385.27BMP 22:1/24:1913.69337.31365.34932.73395.35423.38BMP 22:4/22:4873.57331.26331.26892.61389.31389.31BMP 22:4/22:6869.54331.26327.23888.58389.31385.27BMP 22:6/22:6865.51327.23327.23884.54385.27385.27BMP 22:6/24:1903.62327.23365.34922.65385.27423.38BMP 24:1/22:4907.65365.34331.26926.68423.38389.31BMP 24:1/24:1941.73365.34365.34960.76423.38423.38LC-MS/MS analysis of gangliosidesFor LC-MS/MS analysis of gangliosides, samples were analyzed on a Vanquish UHPLC (Thermo Scientific) coupled to a Orbitrap Exploris 240 mass spectrometer (Thermo Scientific #BRE725535). Extracts were separated on a Kinetex HILIC column (Phenomenex #00D-4461-AN, 2.6 μm,100 × 2.1 mm). Solvent A= acetonitrile with 0.2% (v/v) acetic acid, solvent B = water with 10 mM ammonium acetate, pH 6.1, adjusted with acetic acid. The column temperature was maintained at 50 °C. The LC gradient started with a constant flow rate of 0.6 mL/min with 12.3% B at 0 min, linear gradient of 12.3% B to 22.1% B for min 1–15. The column was equilibrated between runs by 12.3% B for 5 mins. The spray voltage was set to −4.5 kV, and the heated capillary and the HESI were held at 300 °C and 250 °C, respectively. MS parameters were as follows: - heated ESI mode of ionization, sheath gas = 40 units, aux gas = 5 units, sweep gas =1 units, spray voltage = static, negative ion =−4,500 V, negative ion = 2500 V, ion transfer tube temp= 300 °C, vaporizer temp=250 °C. Orbitrap resolution at MS1 was set to 120000, scan range 250–1800, RF lens = 60%, AGC target = standard, Orbitrap resolution at MS2 was set to 30000. EASY-IC™ was used for internal calibration. Analysis was performed in Excalibur (Thermo Scientific #OPTON-30965). The area under the curve was normalized to GM3-d5 standards, followed by normalization to amount of protein/wet weight of tissue used.ProteomicsCellular lysates (50 μL) at 1 mg/mL were mixed with 50 μL of 8 M urea, 50 mM EPPS. The sample was then incubated with 5 mM TCEP for 30 min at 25 °C at 1000 rpm. Then it was incubated with 10 mM IAA during 30 min in the dark at RT before adding 5 mM DTT for 15 min at RT. The sample was digested with Lys-C with a 1:200 ratio enzyme:protein for 1 h at 25 °C at 1000 rpm. The sample was finally diluted half with 50 mM ammonium bicarbonate before digesting with trypsin with a 1:100 ratio protein:enzyme at 37 °C overnight. After the overnight incubation, the samples were desalted using C18 stage tips. Peptides were separated on a C18 reverse phase column (25-cm length, 75-μm diameter and 1.7-μm particle size from IonOptic Aurora 3 #1801220) using a gradient of 2–95% B over 90 min at 50 °C with a flow rate of 300 nL/min using a NanoElute 2 system (Bruker). Solvent A= 0.1% (w/v) formic acid in HPLC-grade water; solvent B = 99.5% (v/v) acetonitrile + 0.1% (v/v) formic acid. MS data were acquired on a timsToF (Bruker) with a captive spray source in data-independent acquisition mode. The mass range was set from 100 to 1700 m/z, the ion mobility range was set from 0.60 V.s/cm2 (collision energy 20eV) to 1.6 V.s/cm2 (collision energy 59 eV) with a ramp time of 100 ms and an accumulation time of 100 ms. Dia-PASEF settings were mass range 400.0 Da to 1201.0 Da, mobility range 0.60–1.60 and a cycle time estimate of 1.80 sec. The dia-PASEF windows were set with a mass width of 26.00 Da, mass overlap 1.00 Da, mass steps per cycle 32. Dia windows are as listed:
#MS TypeCycle IdStart IM [1/K0]End IM [1/K0]Start Mass [m/z]End Mass [m/z]CE [eV]MS10-----PASEF10.911.6800826-PASEF10.60.904400426-PASEF20.92361.6825851-PASEF20.60.9174425451-PASEF30.93721.6850876-PASEF30.60.9308450476-PASEF40.95071.6875901-PASEF40.60.9442475501-PASEF50.96431.6900926-PASEF50.60.9576500526-PASEF60.97791.6925951-PASEF60.60.971525551-PASEF70.99151.6950976-PASEF70.60.9844550576-PASEF81.00511.69751001-PASEF80.60.9979575601-PASEF91.01861.610001026-PASEF90.61.0113600626-PASEF101.03221.610251051-PASEF100.61.0247625651-PASEF111.04581.610501076-PASEF110.61.0381650676-PASEF121.05941.610751101-PASEF120.61.0515675701-PASEF131.0731.611001126-PASEF130.61.0649700726-PASEF141.08651.611251151-PASEF140.61.0783725751-PASEF151.10011.611501176-PASEF150.61.0918750776-PASEF161.11371.611751201-PASEF160.61.1052775801-Mitophagy induction and nucleic acid measurement in lysosomesMitochondrial DNA in the lysosomes was quantified as described previously.41 Briefly, lyso-tagged WT and PLD3 knock out HMC3 cells were treated with DMSO as vehicle or 50 μM Mdivi-I or with 5 μM S,S-BMP + 50 μM Mdivi-I for 24 hours. S,S-BMP treatment to cells started a passage before the Mdivi-1 treatment and continued till the cells were harvested. To increase yield of lysosomal DNA, cells from five 150 mm × 20 mm plates were pooled as one replicate and processed for lysosome isolation as described. Isolated lysosomes were processed through a genomic DNA extraction kit (ThermoFisher Scientific #K0512) and nucleic acids were eluted in 100 μL water. Isolated DNA was PCR amplified for 60 cycles in technical duplicates and quantified by melting curves as described in the RT-PCR section. Primers used were as follows:mtATP6-Forward AATCCAAGCCTACGTTTTCACAmtATP6-Reverse AGTATGAGGAGCGTTATGGAGTmtCO2-Forward AATCGAGTAGTACTCCCGATTGmtCO2-Reverse TTCTAGGACGATGGGCATGAAAmtND1-Forward CTATCACCCTATTAACCACTCAmtND1-Reverse TTCGCCTGTAATATTGAACGTAmt-DLoop-Forward CACCCAAGAACAGGGTTTGTmt-DLoop -Reverse TGGCCATGGGTATGTTGTTAAQUANTIFICATION AND STATISTICAL ANALYSISAll data in the bar graphs are presented as mean ± standard deviation. N number in figure legends indicate biological replicates until otherwise mentioned. All statistical analysis were conducted using Prism 10 (GraphPad, San Diego, CA) or Matlab R2022b. Details of methods used to calculate statistical significance are mentioned in the figure legends.

STAR★METHODS

EXPERIMENTAL MODEL AND STUDY PARTICIPANT DETAILSMouse linesPLD3 heterozygous (HET) on C57BL/6N and PLD4 heterozygous (HET) on C57BL/6J mice were obtained from the International Mouse Phenotyping Consortium (University of California, Davis). Heterozygous PLD3 or PLD4 mice were bred to generate homozygous PLD3 or PLD4 knockout (KO) and wildtype (WT) mice. Mice were genotyped for PLD3 alleles with the following primers: PLD3-forward 5′-TTCCTGTGTGCCTGTCTTGC-3′, PLD3-reverse 5′-AGCTCCTTTCTCCGTCCCTC-3′, and PLD3-transverse 5′-TCGTGGTATCGTTATGCGCC -3′. PCR amplification strategy yielded a 492-bp product for WT and a 178-bp product for KO allele. PLD4 mice were genotyped using the following primers: PLD4-forward 5′- CCAGCTTGAGGTAGCTTCTCATGG -3′, PLD4-reverse 5′- CCACTGATCAAGGCTGTTGCAG-3′, and PLD4-transverse 5′- GCTGACTAGGATAGGCTGGTTGGG -3′. PCR amplification yielded a 1162-bp product for WT and a 497-bp product for KO allele. All mice were housed in a pathogen-free barrier facility with a 12 h light/12 h dark cycle and ad libitum access to water and food. Equal number of male and female mice 2–3 months of age were sacrificed for experiments. Animal procedures were approved by the Institutional Animal Care and Use Committee of the Memorial Sloan Kettering Cancer Centre on animals, following NIH guidelines.

EXPERIMENTAL MODEL AND STUDY PARTICIPANT DETAILS

Mouse linesPLD3 heterozygous (HET) on C57BL/6N and PLD4 heterozygous (HET) on C57BL/6J mice were obtained from the International Mouse Phenotyping Consortium (University of California, Davis). Heterozygous PLD3 or PLD4 mice were bred to generate homozygous PLD3 or PLD4 knockout (KO) and wildtype (WT) mice. Mice were genotyped for PLD3 alleles with the following primers: PLD3-forward 5′-TTCCTGTGTGCCTGTCTTGC-3′, PLD3-reverse 5′-AGCTCCTTTCTCCGTCCCTC-3′, and PLD3-transverse 5′-TCGTGGTATCGTTATGCGCC -3′. PCR amplification strategy yielded a 492-bp product for WT and a 178-bp product for KO allele. PLD4 mice were genotyped using the following primers: PLD4-forward 5′- CCAGCTTGAGGTAGCTTCTCATGG -3′, PLD4-reverse 5′- CCACTGATCAAGGCTGTTGCAG-3′, and PLD4-transverse 5′- GCTGACTAGGATAGGCTGGTTGGG -3′. PCR amplification yielded a 1162-bp product for WT and a 497-bp product for KO allele. All mice were housed in a pathogen-free barrier facility with a 12 h light/12 h dark cycle and ad libitum access to water and food. Equal number of male and female mice 2–3 months of age were sacrificed for experiments. Animal procedures were approved by the Institutional Animal Care and Use Committee of the Memorial Sloan Kettering Cancer Centre on animals, following NIH guidelines.

Mouse lines

PLD3 heterozygous (HET) on C57BL/6N and PLD4 heterozygous (HET) on C57BL/6J mice were obtained from the International Mouse Phenotyping Consortium (University of California, Davis). Heterozygous PLD3 or PLD4 mice were bred to generate homozygous PLD3 or PLD4 knockout (KO) and wildtype (WT) mice. Mice were genotyped for PLD3 alleles with the following primers: PLD3-forward 5′-TTCCTGTGTGCCTGTCTTGC-3′, PLD3-reverse 5′-AGCTCCTTTCTCCGTCCCTC-3′, and PLD3-transverse 5′-TCGTGGTATCGTTATGCGCC -3′. PCR amplification strategy yielded a 492-bp product for WT and a 178-bp product for KO allele. PLD4 mice were genotyped using the following primers: PLD4-forward 5′- CCAGCTTGAGGTAGCTTCTCATGG -3′, PLD4-reverse 5′- CCACTGATCAAGGCTGTTGCAG-3′, and PLD4-transverse 5′- GCTGACTAGGATAGGCTGGTTGGG -3′. PCR amplification yielded a 1162-bp product for WT and a 497-bp product for KO allele. All mice were housed in a pathogen-free barrier facility with a 12 h light/12 h dark cycle and ad libitum access to water and food. Equal number of male and female mice 2–3 months of age were sacrificed for experiments. Animal procedures were approved by the Institutional Animal Care and Use Committee of the Memorial Sloan Kettering Cancer Centre on animals, following NIH guidelines.

METHOD DETAILSCell cultureHMC3 (ATCC #CRL-3304), HEK293T (ATCC #CRL-3216) and their genetically manipulated versions were grown at 37°C, 5% (v/v) CO2 in Dulbecco’s Modified Eagles Medium (DMEM) (Gibco #11965126), supplemented with 10% (w/v) heat-inactivated foetal bovine serum (FBS) (Gibco #10438026) and 1x penicillin-streptomycin (Gibco #15140163). HEK293FT cells (Invitrogen #R70007) were cultured and maintained in FreeStyle™ 293 Expression Medium (Gibco #12338026) supplemented with 5% (v/v) heat-inactivated FBS (Gibco #10438026) in a shaking incubator at 37°C, 5% (v/v) CO2. Cells were periodically tested for mycoplasma contamination with a PCR detection kit (abm #G238). All cell lines were verified by short tandem repeat profiling.RT-PCRTotal RNA was isolated from cells grown in six-well plates at 70–80% confluency with an RNA extraction kit (Qiagen #74104). cDNA was synthesized using a sensiFAST cDNA synthesis kit (Meridian Bioscience #BIO-65053). Genes were amplified for melting curves using Power SYBR™ Green PCR Master Mix (Applied Biosystems™, #4367659). Primers used for the tested genes are listed in Table S5.Molecular cloningAll plasmids used either were procured from GenScript or cDNA clones were obtained from Dharmacon – Mammalian Gene Collection (listed in the reagent sheet). All truncated or tagged versions of clones were generated using Q5 high-fidelity DNA polymerase (New England Biolabs #M0515) and NEB-Gibson Assembly (New England Biolabs #E2611S). All point mutations in PLD3 and PLD4 were generated using Q5-site directed mutagenesis kit (New England Biolabs #E0554S).Lentivirus productionHEK293T cells were transfected with pLJC5-hPLD3 or pLJC5-TMEM192–3xHA (Addgene #102930) envelope proteinpCMV-VSV-G (Addgene #8454), packaging plasmid - pCMV-dR8.2 dvpr (Addgene #8455) in a 6:4:1 ratio using PEI-max (Polysciences #24765–100). Virus-containing medium was harvested two days post-transfection, filtered through a 0.45-μm syringe filter, and stored at −80 °C or used to transduce cells for stable cell line generation.Gene editingCRISPR/Cas9 mediated gene editing of HEK293T and HMC3 cells was performed as described.13 Briefly, human PLD3 sgRNA (5′-3′) was cloned into pSpCa9(BB)-2A-Puro (PX459) V2.0 (Addgene #62988) as described.62 The sequences of sgRNA are as follows: forward – CTTGACGCCACGCTCGTAGG, reverse - CCTACGAGCGTGGCGTCAAG. Cells were transfected with 2 μg of plasmids with FuGene HD transfection reagent (Promega #E2312) overnight. Thereafter, cells were selected with puromycin (1 μg/mL) until all non-transfected control cells died (2–3 days). Selected cells were serially diluted to obtain knock out (KO) clones. To generate PLD3 rescue cell lines (also called PLD3 add-back cells), PLD3 KO HMC3 and HEK293T cells were transduced with lentivirus harbouring a PLD3 insert gene using polybrene at a final concentration of 10 μg/mL. TMEM192–3x-HA tagged (also called Lyso-Tagged) cells were generated as reported.37Lipid incubationsCells were plated in six-well dishes and grown to 70–80% confluency. Thereafter, medium was changed with fresh complete DMEM medium supplemented with 25 μM of phosphatidylglycerol, phosphatidylserine, phosphatidic acid, cardiolipin, or lysophosphatidylglycerol. Phosphatidylcholine and phosphatidylethanolamine were difficult to feed, and so, their concentrations were raised to 100 μM in the medium. Lipid-supplemented medium were prepared by drying lipids in a glass vial under N2 stream. Dried lipids were resuspended in complete DMEM using a water bath sonicator for ~30 mins. Cells were maintained in lipid-supplemented medium for 6 h, and thereafter, cells were washed with ice-chilled Dulbecco’s phosphate-buffered saline (DPBS) (Gibco #14190144) six times and processed for lipid extraction or for immunofluorescence assays. For rescue experiments with BMP, knockout cells grown at ~40% confluency were treated with 25 μM of S,S-BMP (prepared as described above) for 24 h, and thereafter, cells were washed with DPBS and processed for lipid extraction. All lipids used were procured from Avanti Polar Lipids.ImmunostainingCells grown on glass cover slips were washed six times with DPBS, fixed with 4% (w/v) formaldehyde in PBS (BosterBio #AR1068) in DPBS for 15 min in the dark at room temperature (RT). Thereafter, cells were washed again three times with DPBS, permeabilized with permeabilization buffer containing 0.02% (w/v) saponin (Sigma-Aldrich #47036) and 1% (w/v) bovine serum albumin (Sigma-Aldrich #A6003) in DPBS at RT. Cells were incubated with blocking buffer containing 1% (w/v) bovine serum albumin in DPBS at RT, then incubated with primary antibody (1:100 dilution in blocking buffer) for 2 h at RT, washed three times with DPBS and incubated with secondary antibody (1:1000 dilution in blocking buffer) for 45 mins at RT. Cover slips were washed three times with DPBS and mounted on glass slides with Fluoromount Aqueous Mounting Medium (Sigma-Aldrich #F4680). Slides were dried at RT in the dark overnight before imaging. Information on the antibodies used is available in the reagent sheet.DQ-BSA dequenching assaysTo measure lysosomal protease activity, DQ-BSA (Invitrogen #D12050) was dissolved at 1 mg/mL in DPBS by water-bath sonication. Cells were plated in glass-bottom 12-well plates and grown to 60–70% confluency. Cells were washed with DPBS and fed with DQ-BSA (Invitrogen #D12050) at 100 μg/mL in complete DMEM medium for 4 h and treated with LysoTracker Red DND-99 (Invitrogen #L7528) for 15 min and subsequently imaged on a Nikon CSU-W1 SoRa microscope. Cells were maintained in a live cell chamber at 37°C, 85% humidity and 5% CO2.Lysosomal pH measurementsCells were seeded at 50,000 cells per well in 96-well glass bottom plate. Cells were allowed to adhere for 6 hours at 37°C, 85% humidity, and 5% CO2. Cells were incubated with LysoSensor™ Green DND-189 (Invitrogen #L7535) or LysoSensor™ Yellow/Blue DND-160 (Invitrogen #L7545) for 1 h at 37°C. Thereafter, cells were washed three times with DPBS, and replenished with fresh complete DMEM media with no Phenol Red, and subsequently fluorescence was measured on Tecan Spark multimode microplate reader with excitation and emission set at 440 and 500 nm for LysoSensor™ Green DND-189 and 340/380 nm excitation and 440/540 nm excitation for LysoSensor™ Yellow/Blue DND-160. Emission intensities were plotted as it as for the LysoSensor™ Green DND-189. For LysoSensor™ Yellow/Blue DND-160 dye, ratio of emission intensities at 440 nm and 540 nm was used for analysis. For positive control, cells were treated with 100 nM Bafilomycin for 4 hours prior to addition of LysoSensor™ Green DND-189 (Invitrogen #L7535).Cholesterol measurementsLysosomes were immune-isolated from cells as described in lysosome purifications section. Lysosomal lipids were extracted as described in lipid extraction section and cholesterol was measured using Amplex™-Red Cholesterol Assay Kit (Invitrogen #A12216) as per manufacturer’s instructions. Briefly, lysosomal or cellular lipid extracts were dried under inert N2 stream and re-suspended in 50 μL of reaction buffer (0.1 mM potassium phosphate, pH 7.4, 0.05 M NaCl, 5 mM cholic acid, 0.1% Triton X-100). This was mixed with 50 μL of enzyme mixture that contained 2 U/mL of horseradish peroxidase, 2 U/mL cholesterol oxidase, 300 μM Amplex Red. Reaction mixture was incubated at 37 °C in dark for 30 mins. Fluorescence was measured on Tecan Spark multimode microplate reader with excitation and emission set at 540 and 560 nm. Background fluorescence was calculated by incubating lipid extracts with reaction buffer without enzyme mixture. Linearity of the assay was determined by running standard curves for 0 to 10 μg/mL of cholesterol standard. Fluorescence values were normalized to the amount of the protein, and fold change was calculated by dividing each measurement to the mean of values for WT samples.Microscopy and image analysisFluorescence images were acquired on a Nikon CSU-W1 SoRa spinning-disc microscope fitted with an ORCA-Fusion BT Digital CMOS camera (Hamamatsu #C15440–20IP). For cells, images were acquired through a 100x SR HP Apo 1.49 NA objective (Nikon #MRD71970). Green fluorophores (IgG-Alexa-488) were exited using a 488-nm laser and red fluorophores (IgG-Alexa-562) were exited using a 561-nm laser. Emission filters were for green signal = 525/50 and for red signal = 605/52. Images were processed and prepared for figures using Fiji-ImageJ. Signals in the lysosomes (e.g., BMP, DQ-BSA or galectin-3) were quantified by masking the signal for the lysosomal marker LAMP1 or LysoTracker Red DND-99 (Invitrogen #L7528).Western blot analysisCells were scraped into ice-chilled DPBS and pelleted at 1000 g for 2 min. Cell pellets were resuspended in DPBS, lysed by probe sonication (1 sec on, 3 sec off, 60% amplitude), boiled with 4X loading dye at 95°C for 30 min for denaturation. Denatured proteins were separated on 4–15% gradient SDS-PAGE gels at 120 V and transferred to methanol-activated PVDF blotting membrane at 60 V for 3 h at 4°C. Thereafter, PVDF membranes were blocked with 5% (w/v) skim milk with 0.1% (v/v) Tween-20 in DPBS (blocking buffer) for 1 h at RT. Blocked membranes were incubated with primary antibodies (1:1000 diluted in blocking buffer) overnight at 4° C, washed three times with 0.1% (w/v) Tween-20 in DPBS (washing buffer), incubated with secondary antibody (Santa Cruz Biotechnology #sc516102 #sc2357 – 1:5000 diluted in blocking buffer) for 1 h at RT. Membranes were washed three times with washing buffer and developed with ECL chemiluminescence substrates and imaged on a BioRad ChemiDoc Imager.Lysosome purificationsLysosomes were immunoisolated as reported.37 TMEM192–3X HA-tagged cells from one 150-mm plate at 70% confluency were scraped into ice-chilled DPBS supplemented and pelleted at 1000 g for 2 min. The cell pellet was Dounce-homogenized in 1 mL ice-chilled KPBS buffer containing 136 mM KCl, 10 mM KH2PO4 adjusted to pH 7.25 with KOH. Cell lysates were spun down at 1000 g for 2 min. Supernatant was collected and incubated with anti-HA magnetic beads, washed and pre-equilibrated with KPBS buffer for 15 min at 4° C. Beads were washed with excess KPBS buffer 10 times. Proteins were eluted from the beads using lysis buffer containing 50 mM Tris, 150 mM NaCl, 1 mM EDTA, 1% (v/v) Triton X-100 adjusted to pH 7.4.Recombinant protein expression and purificationHMC3 or HEK293T cells were grown at ~50% confluency in complete DMEM medium and transfected with pSPORT6 expression plasmids carrying human versions of PLD1, PLD2, PLD3, PLD4, PLD5, and PLD6 genes, using FuGene HD transfection reagent (Promega #E2312) for 24 h and thereafter processed for immunostaining or for lysosomal isolations and western blots. For PLD3, PLD4, PLA2G15 and CLN5 purifications, HEK293FT cells were grown at 1 × 106 cells/mL and transfected with C-terminal FLAG-tagged WT or active-site mutants cloned in pcDNA3.1 using PEI-max (Polysciences #24765–100). Cells were treated with 10 mM sodium butyrate for ~16 h after transfection and cultured for two additional days. Cells were collected by centrifugation at 2500 g for 15 min, and cell pellets were rinsed with DPBS. Cell pellets were resuspended in 50 mM Tris (pH 7.5), 200 mM NaCL, 5 mM MgCL2, 1 mM EDTA, 5 mM TCEP (resuspension buffer) and lysed by probe sonication (60% amplitude, 1 sec on, 3 sec off for 10 min). Lysates were centrifuged at 25,000 g for 1 h at 4° C. Supernatants were collected, syringe filtered with a 0.45-μM syringe filter, and incubated with anti-FLAG beads (Sigma-Aldrich #A2220) pre-equilibrated with resuspension buffer without TCEP. Beads were washed 10 times with 10 column volumes of resuspension buffer without TCEP. Proteins were eluted with 50 μg/mL of 1x FLAG peptide (Sigma-Aldrich #F3290), concentrated using 30 kDa centricons (Millipore #UFC9030). Concentrated proteins were buffer exchanged with sodium citrate buffer, pH 5.4, using button dialysis, followed by centrifugation at 50,000 g for 30 min to pellet any aggregates. Protein concentration in the supernatant fraction was accessed by nanodrop spectrophotometry, and purity of the protein was accessed using 10% Coomassie-stained SDS-PAGE gel and label-free mass spectrometry.In vitro enzymatic assaysFor BMP synthesis assays, lipid substrates were prepared by resuspending the required amount of lipids in sodium citrate buffer at pH 5.4 using water-bath sonication to a final concentration of 50 μM of phospholipids and 25 μM of mono- or diacylglycerol (Avanti Polar Lipids). Substrates were incubated with extracts of affinity isolated lysosomes on anti-HA magnetic beads (3–5 μg of lysosomal proteins) or 1 μg of purified enzyme (final volume = 100 μL) for 45 min, 37 °C with constant shaking in glass vials. Thereafter, the reaction was quenched with 250 μL of 2:1 chloroform:methanol spiked with 14:0/14:0 BMP as internal standard. The mixture was vortexed and spun at 1000 g for 5 min, and the bottom organic layer was collected in a separate glass vial. Thereafter, the aqueous fraction was acidified with 2 μL of formic acid, vortexed and extracted again with 250 μL of chloroform. Organic layers from both steps were pooled, dried under a N2-stream and analyzed by LC-MS/MS. BMP formation was monitored and normalized to internal standards, the amount of protein used and the time of the reaction. For nuclease competition assay, 50 μM of S,R-lyso-PG and 25 μM of monoacylglycerol was incubated with 1 μg of purified PLD3 along with increasing concentrations of single-stranded DNA (5 μM to 5 mM). Sequence of single-stranded DNA was as reported previously36 : 5’–TCCATGACGTTCCTGATGCTAAGTATGCACTTCATCGTCAAGCAATGCTATGCAT–3’.To test the stability of BMP stereoisomers in lysosomes 100 μM of 18:1/18:1 BMP stereoisomers were incubated with 5 μg of lysosomal lysates for 1 h at 37 °C with constant shaking in glass vials. To test the stability of BMP synthesized by PLD3 or PLD4, enzymatic products were incubated with 1 μg of PLA2G15 or 5 μg of phospholipase A1 from Aspergillus oryzae (Sigma-Aldrich #L3295) for 15 mins at 37 °C. This incubation time was reduced to accommodate for the low amount of BMP synthesized by PLD3 or PLD4 compared to their commercially available synthetic standards. Reaction was quenched with 250 μL of 2:1 chloroform: methanol; and lipids were extracted as described for the BMP synthesis assay. Hydrolysis of BMP stereoisomer standards was monitored by TLC and reaction rates were measured by LC-MS/MS. PLA2G15 used in the reactions was obtained by overexpressing C-terminally FLAG-tagged human PLA2G15 in HEK293FT cells and affinity-isolation using anti-FLAG magnetic beads (Sigma-Aldrich #M8823).To test for lysosomal catabolic activity (e.g., ganglioside catabolism), a fluorescence plate reader assay was setup as described.13 Artificial substrate mimetics for lysosomal enzymes were dissolved in DMSO to stock concentrations of 100 μM. Cells were Dounce-homogenized in ice-cold DPBS and probe sonicated for 1 min (60% amplitude, 1 sec on, 3 sec off). Protein concentrations were estimated using Pierce BCA Protein Assay Kit (Pierce #23225) with BSA as control for standard curves. Lysates (25 μg) were incubated with 1 mM of fluorogenic substrates in sodium citrate buffer (total volume = 100 μL) at pH 5.4 at 37 °C for 30 min. Thereafter, reactions were quenched with 300 μL glycine-buffer, pH 10.4. Fluorescence was measured in a Tecan Spark multimode microplate reader with excitation and emission set at 350 and 440 nm. A standard curve was drawn using 4-methylumbelliferone (Sigma-Aldrich #M1381) as standards dissolved in reaction buffer + quenching buffer in ratios used for the assay. Substrates used were as follows: 4-methylumbelliferyl-β-D-galactoside for β-galactosidase, 4-methylumbelliferyl-β-D-glucopyranoside for β-glucocerebrosidase, 4-nethylumbelliferyl-2-acetamido-2-deoxy-β-D-glucopyranoside for β-hexosaminidase, 7-(α-D-glucopyranosyloxy)-4-methyl-2H-1-benzopyran-2-one for α-glucosidase, 7-(α-D-galactopyranosyloxy)-4-methyl-2H-1-benzopyran-2-one for α-galactosidase, 7-[[2-(acetylamino)-2-deoxy-α-D-galactopyranosyl]oxy]-4-methyl-2H-1-benzopyran-2-one for β-galactopyranosaminidase activity, hexadecanoic acid, 4-methyl-2-oxo-2H-1-benzopyran-7-yl ester for lysosomal acid lipase, and 4-methyl-2-oxo-2H-1-benzopyran-7-yl 5-(acetylamino)-3,5-dideoxy-α-neuraminic acid for neuraminidases. All artificial substrates were procured from Cayman Chemicals.Preparation of S,R-lyso-PGR,S-BMP (100 μM) was incubated with 5 μg of human PLA2G15 at 37 °C in glass vials with constant shaking. The reaction was monitored by running small aliquots on TLC. The reaction was quenched and extracted with excess 2:1 (v/v) chloroform:methanol. Extracts were dried under an N2 stream, and the lyso-PG mixture was separated on a C18 reverse phase column (Pheneomenex, 5 μm, 50 × 4.6 mm) using the following solvents: Solvent A= 95:5 (v/v) H2O/MeOH + 0.1% (v/v) NH4OH, solvent B= 60:35:5 (v/v) IPA/MeOH/H2O + 0.1% (v/v) NH4OH. The liquid chromatography (LC) gradient was 30 mins, with a stable flow rate of 0.5 mL/min, starting 5% B for 5 min, linear gradient of B from 5–100% over 23 min, 100% B from min 23–25, and re-equilibration with 5% B for 5 min. The isolated lyso-PG was pooled and dried under N2, weighed and used for BMP synthesis assays.Chiral derivatization and separation of BMP stereoisomersStandards of BMP stereoisomers or enzymatic products were mixed with derivatization agent (R)-(−)-1-(1-naphthyl)ethyl isocyanate (Sigma-Aldrich #220442) in a ratio of 1:10 in 1 mL of dichloromethanol (VWR BDH23373.400) and 50 μL of pyridine (Sigma-Aldrich #270970) overnight (14–16 h) at RT. The reaction mixture was dried under a stream of N2 and extracted with 2:1:1 chloroform:methanol:water. The bottom organic layer was collected and separated on YMC Chiral ART Cellulose-SZ, analytical column (YMC America #KSZ99S03–2502WT). Solvent A = 60:40 (v/v) ACN/water + 0.1% (v/v) formic acid + 10 mM ammonium formate, Solvent B = 90:10 (v/v) IPA: ACN + 0.1% (v/v) formic acid + 10 mM ammonium formate. The LC gradient was 45 min, with a stable flow rate of 0.2 mL/min, starting with 40% B for 5 min, linear gradient of 40–70% B for min 5–30, maintained at 100% B for mins 30–33, re-equilibrated with 40% B for min 33–45 at 0.3 mL/min. The column temperature was maintained at 10 °C. MS parameters were as described in LC-MS/MS analysis of lipids section.Lipid extractionLipids were extracted from cells or tissues using established protocols.63 Briefly, cells were lysed in DPBS using a probe sonicator, or tissues were Dounce-homogenized in DPBS, and lipids were extracted with a chloroform and methanol mixture, with spiked-in lipid splash mix and/or BMP 14:0/14:0, as internal standards. The final ratio of chloroform:methanol:DPBS in the extraction mixture was 2:1:1. The extraction mixture was vortexed and spun down at 1000 g for 10 min. The bottom organic phase was collected in a glass tube. The top aqueous phase was acidified with 0.5% (v/v) formic acid, vortexed and re-extracted with chloroform by spinning down as in the previous step. Both organic phases were pooled and dried under a stream of N2 and analysed by LC-MS/MS.Ganglioside extractionTissues or cells were Dounce-homogenized with ice-chilled DPBS, and methanol was added to the final ratio of 90:10 methanol: water. The extraction mixture was spun down at 1000 g for 20 min to pellet proteins, and the solvent layers containing gangliosides and other metabolites were collected in glass tubes and dried under an N2 stream. Dried extract was reconstituted in 1 mL of LC-MS-grade water and desalted by Sola HRP SPE 30mg/2MI 96-well plate 1EA (Thermo Scientific #60509–001). Desalting cartridges were cleaned three times with 1 mL of methanol, equilibrated three times with LC-MS-grade water, and then the extracts dissolved in water were loaded onto the cartridge and washed three times with water, and finally, gangliosides were eluted by 3 mL of methanol. Eluates were dried under N2 and reconstituted in 300:150:50 of MeOH:water:chloroform.TLC for lipidsExtracted lipids were separated on high-performance thin layer chromatography (Sigma-Aldrich #1056310001) in the first dimension using 70:30:3:2 (v/v/v/v) chloroform:methanol:water:concentrated ammonia as the running phase. For BMP separation, plates were dried at RT for 1 hour and run in the second dimension using 65:35:5 (v/v/v) of chloroform:methanol:water as the running phase. Plates were dried and sprayed with 5% (w/v) copper sulfate dissolved in 25% (v/v) ethanol and 8.5% (v/v) phosphoric acid, followed by charring at 60 °C for 1 h. Plates were imaged on BioRad ChemiDoc Imager.LC-MS/MS analysis of lipidsAll samples were analysed on a Vanquish UHPLC (Thermo Scientific) coupled to a Orbitrap Exploris 240 mass spectrometer (Thermo Scientific #BRE725535). For routine lipid analysis, extracts were separated on a C18 reverse phase column (Pheneomenex, 5 μm, 50 × 4.6 mm #00B-4435-E0) as described.63 Briefly, for positive mode: solvent A= 95:5 (v/v) H2O/MeOH + 0.1% formic acid + 10 mM ammonium formate, solvent B= 60:35:5 (v/v/v) isopropanol (IPA)/MeOH/H2O + 0.1% (v/v) formic acid + 10 mM ammonium formate. For negative mode, solvent A= 95:5 (v/v) H2O/MeOH + 0.1% (v/v) NH4OH, solvent B= 60:35:5 (v/v) IPA/MeOH/H2O + 0.1% (v/v) NH4OH. LC gradient was 60 min, with stable flow rate of 0.5 mL/min starting 5% B for 5 min, linear gradient of B from 5–100% over 50 min, 100% B from min 50–55, and re-equilibration with 5% B for 5 min. The column temperature was maintained at 55 °C. MS parameters were as follows: - heated ESI mode of ionization, sheath gas = 40 units, aux gas = 8 units, sweep gas =1 units, spray voltage = static, positive ion =3,400 V, negative ion = 2500 V, ion transfer tube temp= 320 °C, vaporizer temp=275 °C. Orbitrap resolution at MS1 was set to 120000, scan range 250–1800, RF lens = 60%, AGC target = standard, Orbitrap resolution at MS2 was set to 30000. EASY-IC™ was used for internal calibration. Lipids were searched and aligned for annotation in LipidSearch 5.0 (Thermo Scientific #OPTON-30880) with precursor mass error tolerance of 5 ppm and product mass error tolerance of 10 ppm, retention time deviation tolerance of 0.5 min.LC-MS/MS analysis of BMPAll samples were analysed on a Vanquish UHPLC (Thermo Scientific) coupled to a Orbitrap Exploris 240 mass spectrometer (Thermo Scientific #BRE725535). Chromatographic separation of BMP and PG was achieved on a C-30 reverse phase column (Thermo Scientific #078664, 3 mm 250 × 2.1 mm). Solvent = 60:40 (v/v) ACN/water + 0.1% (v/v) formic acid + 10 mM ammonium formate, Solvent B = 90:10 (v/v) IPA: ACN + 0.1% (v/v) formic acid + 10 mM ammonium formate. The LC gradient was 45 min, with a stable flow rate of 0.2 mL/min starting 40% B, linear gradient of 45–55% over mins 1–7, maintained at 65% B for mins 8–12, linear gradient of 65–70% B for mins 12–30, maintained at 100% B for mins 30–33, re-equilibrated with 40% for mins 33–45. Column temperature was maintained at 50 °C. MS parameters were as follows: - heated ESI mode of ionization, sheath gas = 40 units, aux gas = 8 units, sweep gas =1 units, spray voltage = static, positive ion =3,400 V, negative ion = 2500 V, ion transfer tube temp= 320 °C, vaporizer temp=275 °C. Orbitrap resolution at MS1 was set to 120000, RF lens = 60%, Orbitrap resolution at MS2 was set to 30000. Orbitrap was set up with an unscheduled targeted inclusion mass list corresponding to PG and BMP. EASY-IC™ was used for internal calibration. To avoid source fragmentation of cardiolipin confounding measurements, we inactivated source fragmentation during data acquisition with mass spectrometry. Using this method, an analysis of cardiolipin standards (CL 14:0/14:0/14:0/14:0) showed no notable source fragmentation of CL into PG or its fragment ions.PG and BMP peaks were identified using Excalibur 3.1 (Thermo Scientific #OPTON-30965), the area under the curve was normalized to the area under the curve for internal standards (14:0/14:0 BMP for cells and 15:0/18:1(d7) PG for mice tissues) and amount of protein/wet weight of tissue used. Samples for BMP analysis were run in both negative and positive ion mode. All the quantification presented in the figures of the manuscript come from the negative-mode analysis. Positive-mode analysis of BMP was used only to confirm the identify of BMP species. Targeted precursor masses for BMP, and their predicted/observed fragments are as follows:
Negative ModePositive ModeSpeciesParent IonFragment IonFragment IonParent IonFragment IonFragment IonBMP 16:0/16:0721.51255.23255.23740.54313.27313.27BMP 16:0/18:1747.52255.23281.25766.56313.27339.23BMP 16:0/18:0749.54255.23283.26768.58313.27341.31BMP 16:0/20:1775.55255.23309.28794.59313.27367.32BMP 16:0/20:4769.51255.23303.23788.54313.27361.27BMP 16:0/22:6793.51255.23327.23812.54313.27385.27BMP 16:0/17:1733.51255.23267.23752.54313.27325.27BMP 16:0/12:0665.44255.23199.17684.48313.27257.21BMP 16:0/14:0693.47255.23227.20712.51313.27285.24BMP 16:0/18:2745.51255.23279.23764.54313.27337.27BMP 16:0/20:0777.57255.23311.30796.61313.27369.34BMP 16:0/22:0805.60255.23339.33824.64313.27397.37BMP 16:0/22:1803.58255.23337.31822.62313.27395.35BMP 16:0/22:4797.54255.23331.26816.58313.27389.31BMP 16:0/24:0833.63255.23367.36852.67313.27425.4BMP 16:0/24:1831.62255.23365.34850.65313.27423.38BMP 18:1/18:1773.54281.25281.25792.58339.23339.23BMP 18:1/18:0775.55281.25283.26794.59339.23341.31BMP 18:1/20:1801.57281.25309.28820.61339.23367.32BMP 18:1/20:4795.52281.25303.23814.56339.23361.27BMP 18:1/22:6819.52281.25327.23838.56339.23385.27BMP 18:1/17:1759.52281.25267.23778.56339.23325.27BMP 18:1/12:0691.46281.25199.17710.50339.23257.21BMP 18:1/14:0719.49281.25227.20738.53339.23285.24BMP 18:1/18:2771.52281.25279.23790.56339.23337.27BMP 18:1/20:0803.58281.25311.30822.62339.23369.34BMP 18:1/22:0831.62281.25339.33850.65339.23397.37BMP 18:1/22:1829.60281.25337.31848.64339.23395.35BMP 18:1/22:4823.55281.25331.26842.59339.23389.31BMP 18:1/24:0859.65281.25367.36878.68339.23425.4BMP 18:1/24:1857.63281.25365.34876.67339.23423.38BMP 18:2/18:2769.51279.23279.23788.54337.27337.27BMP 18:2/20:1799.55279.23309.28818.59337.27367.32BMP 18:2/20:4793.51279.23303.23812.54337.27361.27BMP 18:2/22:1827.58279.23337.31846.62337.27395.35BMP 18:2/22:4821.54279.23331.26840.58337.27389.31BMP 18:2/22:6817.51279.23327.23836.54337.27385.27BMP 18:2/24:1855.62279.23365.34874.65337.27423.38BMP 20:1/20:1829.60309.28309.28848.64367.32367.32BMP 20:1/20:4823.55309.28303.23842.59367.32361.27BMP 20:1/22:1857.63309.28337.31876.67367.32395.35BMP 20:1/22:4851.58309.28331.26870.62367.32389.31BMP 20:1/22:6847.55309.28327.23866.59367.32385.27BMP 20:1/24:1885.66309.28365.34904.70367.32423.38BMP 20:4/20:4817.51303.23303.23836.54361.27361.27BMP 20:4/22:1851.58303.23337.31870.62361.27395.35BMP 20:4/22:4845.54303.23331.26864.58361.27389.31BMP 20:4/22:6841.51303.23327.23860.54361.27385.27BMP 20:4/24:1879.62303.23365.34898.65361.27423.38BMP 22:1/22:1885.66337.31337.31904.70395.35395.35BMP 22:1/22:4879.62337.31331.26898.65395.35389.31BMP 22:1/22:6875.58337.31327.23894.62395.35385.27BMP 22:1/24:1913.69337.31365.34932.73395.35423.38BMP 22:4/22:4873.57331.26331.26892.61389.31389.31BMP 22:4/22:6869.54331.26327.23888.58389.31385.27BMP 22:6/22:6865.51327.23327.23884.54385.27385.27BMP 22:6/24:1903.62327.23365.34922.65385.27423.38BMP 24:1/22:4907.65365.34331.26926.68423.38389.31BMP 24:1/24:1941.73365.34365.34960.76423.38423.38LC-MS/MS analysis of gangliosidesFor LC-MS/MS analysis of gangliosides, samples were analyzed on a Vanquish UHPLC (Thermo Scientific) coupled to a Orbitrap Exploris 240 mass spectrometer (Thermo Scientific #BRE725535). Extracts were separated on a Kinetex HILIC column (Phenomenex #00D-4461-AN, 2.6 μm,100 × 2.1 mm). Solvent A= acetonitrile with 0.2% (v/v) acetic acid, solvent B = water with 10 mM ammonium acetate, pH 6.1, adjusted with acetic acid. The column temperature was maintained at 50 °C. The LC gradient started with a constant flow rate of 0.6 mL/min with 12.3% B at 0 min, linear gradient of 12.3% B to 22.1% B for min 1–15. The column was equilibrated between runs by 12.3% B for 5 mins. The spray voltage was set to −4.5 kV, and the heated capillary and the HESI were held at 300 °C and 250 °C, respectively. MS parameters were as follows: - heated ESI mode of ionization, sheath gas = 40 units, aux gas = 5 units, sweep gas =1 units, spray voltage = static, negative ion =−4,500 V, negative ion = 2500 V, ion transfer tube temp= 300 °C, vaporizer temp=250 °C. Orbitrap resolution at MS1 was set to 120000, scan range 250–1800, RF lens = 60%, AGC target = standard, Orbitrap resolution at MS2 was set to 30000. EASY-IC™ was used for internal calibration. Analysis was performed in Excalibur (Thermo Scientific #OPTON-30965). The area under the curve was normalized to GM3-d5 standards, followed by normalization to amount of protein/wet weight of tissue used.ProteomicsCellular lysates (50 μL) at 1 mg/mL were mixed with 50 μL of 8 M urea, 50 mM EPPS. The sample was then incubated with 5 mM TCEP for 30 min at 25 °C at 1000 rpm. Then it was incubated with 10 mM IAA during 30 min in the dark at RT before adding 5 mM DTT for 15 min at RT. The sample was digested with Lys-C with a 1:200 ratio enzyme:protein for 1 h at 25 °C at 1000 rpm. The sample was finally diluted half with 50 mM ammonium bicarbonate before digesting with trypsin with a 1:100 ratio protein:enzyme at 37 °C overnight. After the overnight incubation, the samples were desalted using C18 stage tips. Peptides were separated on a C18 reverse phase column (25-cm length, 75-μm diameter and 1.7-μm particle size from IonOptic Aurora 3 #1801220) using a gradient of 2–95% B over 90 min at 50 °C with a flow rate of 300 nL/min using a NanoElute 2 system (Bruker). Solvent A= 0.1% (w/v) formic acid in HPLC-grade water; solvent B = 99.5% (v/v) acetonitrile + 0.1% (v/v) formic acid. MS data were acquired on a timsToF (Bruker) with a captive spray source in data-independent acquisition mode. The mass range was set from 100 to 1700 m/z, the ion mobility range was set from 0.60 V.s/cm2 (collision energy 20eV) to 1.6 V.s/cm2 (collision energy 59 eV) with a ramp time of 100 ms and an accumulation time of 100 ms. Dia-PASEF settings were mass range 400.0 Da to 1201.0 Da, mobility range 0.60–1.60 and a cycle time estimate of 1.80 sec. The dia-PASEF windows were set with a mass width of 26.00 Da, mass overlap 1.00 Da, mass steps per cycle 32. Dia windows are as listed:
#MS TypeCycle IdStart IM [1/K0]End IM [1/K0]Start Mass [m/z]End Mass [m/z]CE [eV]MS10-----PASEF10.911.6800826-PASEF10.60.904400426-PASEF20.92361.6825851-PASEF20.60.9174425451-PASEF30.93721.6850876-PASEF30.60.9308450476-PASEF40.95071.6875901-PASEF40.60.9442475501-PASEF50.96431.6900926-PASEF50.60.9576500526-PASEF60.97791.6925951-PASEF60.60.971525551-PASEF70.99151.6950976-PASEF70.60.9844550576-PASEF81.00511.69751001-PASEF80.60.9979575601-PASEF91.01861.610001026-PASEF90.61.0113600626-PASEF101.03221.610251051-PASEF100.61.0247625651-PASEF111.04581.610501076-PASEF110.61.0381650676-PASEF121.05941.610751101-PASEF120.61.0515675701-PASEF131.0731.611001126-PASEF130.61.0649700726-PASEF141.08651.611251151-PASEF140.61.0783725751-PASEF151.10011.611501176-PASEF150.61.0918750776-PASEF161.11371.611751201-PASEF160.61.1052775801-Mitophagy induction and nucleic acid measurement in lysosomesMitochondrial DNA in the lysosomes was quantified as described previously.41 Briefly, lyso-tagged WT and PLD3 knock out HMC3 cells were treated with DMSO as vehicle or 50 μM Mdivi-I or with 5 μM S,S-BMP + 50 μM Mdivi-I for 24 hours. S,S-BMP treatment to cells started a passage before the Mdivi-1 treatment and continued till the cells were harvested. To increase yield of lysosomal DNA, cells from five 150 mm × 20 mm plates were pooled as one replicate and processed for lysosome isolation as described. Isolated lysosomes were processed through a genomic DNA extraction kit (ThermoFisher Scientific #K0512) and nucleic acids were eluted in 100 μL water. Isolated DNA was PCR amplified for 60 cycles in technical duplicates and quantified by melting curves as described in the RT-PCR section. Primers used were as follows:mtATP6-Forward AATCCAAGCCTACGTTTTCACAmtATP6-Reverse AGTATGAGGAGCGTTATGGAGTmtCO2-Forward AATCGAGTAGTACTCCCGATTGmtCO2-Reverse TTCTAGGACGATGGGCATGAAAmtND1-Forward CTATCACCCTATTAACCACTCAmtND1-Reverse TTCGCCTGTAATATTGAACGTAmt-DLoop-Forward CACCCAAGAACAGGGTTTGTmt-DLoop -Reverse TGGCCATGGGTATGTTGTTAA

METHOD DETAILS

Cell cultureHMC3 (ATCC #CRL-3304), HEK293T (ATCC #CRL-3216) and their genetically manipulated versions were grown at 37°C, 5% (v/v) CO2 in Dulbecco’s Modified Eagles Medium (DMEM) (Gibco #11965126), supplemented with 10% (w/v) heat-inactivated foetal bovine serum (FBS) (Gibco #10438026) and 1x penicillin-streptomycin (Gibco #15140163). HEK293FT cells (Invitrogen #R70007) were cultured and maintained in FreeStyle™ 293 Expression Medium (Gibco #12338026) supplemented with 5% (v/v) heat-inactivated FBS (Gibco #10438026) in a shaking incubator at 37°C, 5% (v/v) CO2. Cells were periodically tested for mycoplasma contamination with a PCR detection kit (abm #G238). All cell lines were verified by short tandem repeat profiling.

Cell culture

HMC3 (ATCC #CRL-3304), HEK293T (ATCC #CRL-3216) and their genetically manipulated versions were grown at 37°C, 5% (v/v) CO2 in Dulbecco’s Modified Eagles Medium (DMEM) (Gibco #11965126), supplemented with 10% (w/v) heat-inactivated foetal bovine serum (FBS) (Gibco #10438026) and 1x penicillin-streptomycin (Gibco #15140163). HEK293FT cells (Invitrogen #R70007) were cultured and maintained in FreeStyle™ 293 Expression Medium (Gibco #12338026) supplemented with 5% (v/v) heat-inactivated FBS (Gibco #10438026) in a shaking incubator at 37°C, 5% (v/v) CO2. Cells were periodically tested for mycoplasma contamination with a PCR detection kit (abm #G238). All cell lines were verified by short tandem repeat profiling.

RT-PCRTotal RNA was isolated from cells grown in six-well plates at 70–80% confluency with an RNA extraction kit (Qiagen #74104). cDNA was synthesized using a sensiFAST cDNA synthesis kit (Meridian Bioscience #BIO-65053). Genes were amplified for melting curves using Power SYBR™ Green PCR Master Mix (Applied Biosystems™, #4367659). Primers used for the tested genes are listed in Table S5.

Total RNA was isolated from cells grown in six-well plates at 70–80% confluency with an RNA extraction kit (Qiagen #74104). cDNA was synthesized using a sensiFAST cDNA synthesis kit (Meridian Bioscience #BIO-65053). Genes were amplified for melting curves using Power SYBR™ Green PCR Master Mix (Applied Biosystems™, #4367659). Primers used for the tested genes are listed in Table S5.

Molecular cloningAll plasmids used either were procured from GenScript or cDNA clones were obtained from Dharmacon – Mammalian Gene Collection (listed in the reagent sheet). All truncated or tagged versions of clones were generated using Q5 high-fidelity DNA polymerase (New England Biolabs #M0515) and NEB-Gibson Assembly (New England Biolabs #E2611S). All point mutations in PLD3 and PLD4 were generated using Q5-site directed mutagenesis kit (New England Biolabs #E0554S).

Molecular cloning

All plasmids used either were procured from GenScript or cDNA clones were obtained from Dharmacon – Mammalian Gene Collection (listed in the reagent sheet). All truncated or tagged versions of clones were generated using Q5 high-fidelity DNA polymerase (New England Biolabs #M0515) and NEB-Gibson Assembly (New England Biolabs #E2611S). All point mutations in PLD3 and PLD4 were generated using Q5-site directed mutagenesis kit (New England Biolabs #E0554S).

Lentivirus productionHEK293T cells were transfected with pLJC5-hPLD3 or pLJC5-TMEM192–3xHA (Addgene #102930) envelope proteinpCMV-VSV-G (Addgene #8454), packaging plasmid - pCMV-dR8.2 dvpr (Addgene #8455) in a 6:4:1 ratio using PEI-max (Polysciences #24765–100). Virus-containing medium was harvested two days post-transfection, filtered through a 0.45-μm syringe filter, and stored at −80 °C or used to transduce cells for stable cell line generation.

Lentivirus production

HEK293T cells were transfected with pLJC5-hPLD3 or pLJC5-TMEM192–3xHA (Addgene #102930) envelope proteinpCMV-VSV-G (Addgene #8454), packaging plasmid - pCMV-dR8.2 dvpr (Addgene #8455) in a 6:4:1 ratio using PEI-max (Polysciences #24765–100). Virus-containing medium was harvested two days post-transfection, filtered through a 0.45-μm syringe filter, and stored at −80 °C or used to transduce cells for stable cell line generation.

Gene editingCRISPR/Cas9 mediated gene editing of HEK293T and HMC3 cells was performed as described.13 Briefly, human PLD3 sgRNA (5′-3′) was cloned into pSpCa9(BB)-2A-Puro (PX459) V2.0 (Addgene #62988) as described.62 The sequences of sgRNA are as follows: forward – CTTGACGCCACGCTCGTAGG, reverse - CCTACGAGCGTGGCGTCAAG. Cells were transfected with 2 μg of plasmids with FuGene HD transfection reagent (Promega #E2312) overnight. Thereafter, cells were selected with puromycin (1 μg/mL) until all non-transfected control cells died (2–3 days). Selected cells were serially diluted to obtain knock out (KO) clones. To generate PLD3 rescue cell lines (also called PLD3 add-back cells), PLD3 KO HMC3 and HEK293T cells were transduced with lentivirus harbouring a PLD3 insert gene using polybrene at a final concentration of 10 μg/mL. TMEM192–3x-HA tagged (also called Lyso-Tagged) cells were generated as reported.37

Gene editing

CRISPR/Cas9 mediated gene editing of HEK293T and HMC3 cells was performed as described.13 Briefly, human PLD3 sgRNA (5′-3′) was cloned into pSpCa9(BB)-2A-Puro (PX459) V2.0 (Addgene #62988) as described.62 The sequences of sgRNA are as follows: forward – CTTGACGCCACGCTCGTAGG, reverse - CCTACGAGCGTGGCGTCAAG. Cells were transfected with 2 μg of plasmids with FuGene HD transfection reagent (Promega #E2312) overnight. Thereafter, cells were selected with puromycin (1 μg/mL) until all non-transfected control cells died (2–3 days). Selected cells were serially diluted to obtain knock out (KO) clones. To generate PLD3 rescue cell lines (also called PLD3 add-back cells), PLD3 KO HMC3 and HEK293T cells were transduced with lentivirus harbouring a PLD3 insert gene using polybrene at a final concentration of 10 μg/mL. TMEM192–3x-HA tagged (also called Lyso-Tagged) cells were generated as reported.37

Lipid incubationsCells were plated in six-well dishes and grown to 70–80% confluency. Thereafter, medium was changed with fresh complete DMEM medium supplemented with 25 μM of phosphatidylglycerol, phosphatidylserine, phosphatidic acid, cardiolipin, or lysophosphatidylglycerol. Phosphatidylcholine and phosphatidylethanolamine were difficult to feed, and so, their concentrations were raised to 100 μM in the medium. Lipid-supplemented medium were prepared by drying lipids in a glass vial under N2 stream. Dried lipids were resuspended in complete DMEM using a water bath sonicator for ~30 mins. Cells were maintained in lipid-supplemented medium for 6 h, and thereafter, cells were washed with ice-chilled Dulbecco’s phosphate-buffered saline (DPBS) (Gibco #14190144) six times and processed for lipid extraction or for immunofluorescence assays. For rescue experiments with BMP, knockout cells grown at ~40% confluency were treated with 25 μM of S,S-BMP (prepared as described above) for 24 h, and thereafter, cells were washed with DPBS and processed for lipid extraction. All lipids used were procured from Avanti Polar Lipids.

Lipid incubations

Cells were plated in six-well dishes and grown to 70–80% confluency. Thereafter, medium was changed with fresh complete DMEM medium supplemented with 25 μM of phosphatidylglycerol, phosphatidylserine, phosphatidic acid, cardiolipin, or lysophosphatidylglycerol. Phosphatidylcholine and phosphatidylethanolamine were difficult to feed, and so, their concentrations were raised to 100 μM in the medium. Lipid-supplemented medium were prepared by drying lipids in a glass vial under N2 stream. Dried lipids were resuspended in complete DMEM using a water bath sonicator for ~30 mins. Cells were maintained in lipid-supplemented medium for 6 h, and thereafter, cells were washed with ice-chilled Dulbecco’s phosphate-buffered saline (DPBS) (Gibco #14190144) six times and processed for lipid extraction or for immunofluorescence assays. For rescue experiments with BMP, knockout cells grown at ~40% confluency were treated with 25 μM of S,S-BMP (prepared as described above) for 24 h, and thereafter, cells were washed with DPBS and processed for lipid extraction. All lipids used were procured from Avanti Polar Lipids.

ImmunostainingCells grown on glass cover slips were washed six times with DPBS, fixed with 4% (w/v) formaldehyde in PBS (BosterBio #AR1068) in DPBS for 15 min in the dark at room temperature (RT). Thereafter, cells were washed again three times with DPBS, permeabilized with permeabilization buffer containing 0.02% (w/v) saponin (Sigma-Aldrich #47036) and 1% (w/v) bovine serum albumin (Sigma-Aldrich #A6003) in DPBS at RT. Cells were incubated with blocking buffer containing 1% (w/v) bovine serum albumin in DPBS at RT, then incubated with primary antibody (1:100 dilution in blocking buffer) for 2 h at RT, washed three times with DPBS and incubated with secondary antibody (1:1000 dilution in blocking buffer) for 45 mins at RT. Cover slips were washed three times with DPBS and mounted on glass slides with Fluoromount Aqueous Mounting Medium (Sigma-Aldrich #F4680). Slides were dried at RT in the dark overnight before imaging. Information on the antibodies used is available in the reagent sheet.

Immunostaining

Cells grown on glass cover slips were washed six times with DPBS, fixed with 4% (w/v) formaldehyde in PBS (BosterBio #AR1068) in DPBS for 15 min in the dark at room temperature (RT). Thereafter, cells were washed again three times with DPBS, permeabilized with permeabilization buffer containing 0.02% (w/v) saponin (Sigma-Aldrich #47036) and 1% (w/v) bovine serum albumin (Sigma-Aldrich #A6003) in DPBS at RT. Cells were incubated with blocking buffer containing 1% (w/v) bovine serum albumin in DPBS at RT, then incubated with primary antibody (1:100 dilution in blocking buffer) for 2 h at RT, washed three times with DPBS and incubated with secondary antibody (1:1000 dilution in blocking buffer) for 45 mins at RT. Cover slips were washed three times with DPBS and mounted on glass slides with Fluoromount Aqueous Mounting Medium (Sigma-Aldrich #F4680). Slides were dried at RT in the dark overnight before imaging. Information on the antibodies used is available in the reagent sheet.

DQ-BSA dequenching assaysTo measure lysosomal protease activity, DQ-BSA (Invitrogen #D12050) was dissolved at 1 mg/mL in DPBS by water-bath sonication. Cells were plated in glass-bottom 12-well plates and grown to 60–70% confluency. Cells were washed with DPBS and fed with DQ-BSA (Invitrogen #D12050) at 100 μg/mL in complete DMEM medium for 4 h and treated with LysoTracker Red DND-99 (Invitrogen #L7528) for 15 min and subsequently imaged on a Nikon CSU-W1 SoRa microscope. Cells were maintained in a live cell chamber at 37°C, 85% humidity and 5% CO2.

DQ-BSA dequenching assays

To measure lysosomal protease activity, DQ-BSA (Invitrogen #D12050) was dissolved at 1 mg/mL in DPBS by water-bath sonication. Cells were plated in glass-bottom 12-well plates and grown to 60–70% confluency. Cells were washed with DPBS and fed with DQ-BSA (Invitrogen #D12050) at 100 μg/mL in complete DMEM medium for 4 h and treated with LysoTracker Red DND-99 (Invitrogen #L7528) for 15 min and subsequently imaged on a Nikon CSU-W1 SoRa microscope. Cells were maintained in a live cell chamber at 37°C, 85% humidity and 5% CO2.

Lysosomal pH measurementsCells were seeded at 50,000 cells per well in 96-well glass bottom plate. Cells were allowed to adhere for 6 hours at 37°C, 85% humidity, and 5% CO2. Cells were incubated with LysoSensor™ Green DND-189 (Invitrogen #L7535) or LysoSensor™ Yellow/Blue DND-160 (Invitrogen #L7545) for 1 h at 37°C. Thereafter, cells were washed three times with DPBS, and replenished with fresh complete DMEM media with no Phenol Red, and subsequently fluorescence was measured on Tecan Spark multimode microplate reader with excitation and emission set at 440 and 500 nm for LysoSensor™ Green DND-189 and 340/380 nm excitation and 440/540 nm excitation for LysoSensor™ Yellow/Blue DND-160. Emission intensities were plotted as it as for the LysoSensor™ Green DND-189. For LysoSensor™ Yellow/Blue DND-160 dye, ratio of emission intensities at 440 nm and 540 nm was used for analysis. For positive control, cells were treated with 100 nM Bafilomycin for 4 hours prior to addition of LysoSensor™ Green DND-189 (Invitrogen #L7535).

Lysosomal pH measurements

Cells were seeded at 50,000 cells per well in 96-well glass bottom plate. Cells were allowed to adhere for 6 hours at 37°C, 85% humidity, and 5% CO2. Cells were incubated with LysoSensor™ Green DND-189 (Invitrogen #L7535) or LysoSensor™ Yellow/Blue DND-160 (Invitrogen #L7545) for 1 h at 37°C. Thereafter, cells were washed three times with DPBS, and replenished with fresh complete DMEM media with no Phenol Red, and subsequently fluorescence was measured on Tecan Spark multimode microplate reader with excitation and emission set at 440 and 500 nm for LysoSensor™ Green DND-189 and 340/380 nm excitation and 440/540 nm excitation for LysoSensor™ Yellow/Blue DND-160. Emission intensities were plotted as it as for the LysoSensor™ Green DND-189. For LysoSensor™ Yellow/Blue DND-160 dye, ratio of emission intensities at 440 nm and 540 nm was used for analysis. For positive control, cells were treated with 100 nM Bafilomycin for 4 hours prior to addition of LysoSensor™ Green DND-189 (Invitrogen #L7535).

Cholesterol measurementsLysosomes were immune-isolated from cells as described in lysosome purifications section. Lysosomal lipids were extracted as described in lipid extraction section and cholesterol was measured using Amplex™-Red Cholesterol Assay Kit (Invitrogen #A12216) as per manufacturer’s instructions. Briefly, lysosomal or cellular lipid extracts were dried under inert N2 stream and re-suspended in 50 μL of reaction buffer (0.1 mM potassium phosphate, pH 7.4, 0.05 M NaCl, 5 mM cholic acid, 0.1% Triton X-100). This was mixed with 50 μL of enzyme mixture that contained 2 U/mL of horseradish peroxidase, 2 U/mL cholesterol oxidase, 300 μM Amplex Red. Reaction mixture was incubated at 37 °C in dark for 30 mins. Fluorescence was measured on Tecan Spark multimode microplate reader with excitation and emission set at 540 and 560 nm. Background fluorescence was calculated by incubating lipid extracts with reaction buffer without enzyme mixture. Linearity of the assay was determined by running standard curves for 0 to 10 μg/mL of cholesterol standard. Fluorescence values were normalized to the amount of the protein, and fold change was calculated by dividing each measurement to the mean of values for WT samples.

Cholesterol measurements

Lysosomes were immune-isolated from cells as described in lysosome purifications section. Lysosomal lipids were extracted as described in lipid extraction section and cholesterol was measured using Amplex™-Red Cholesterol Assay Kit (Invitrogen #A12216) as per manufacturer’s instructions. Briefly, lysosomal or cellular lipid extracts were dried under inert N2 stream and re-suspended in 50 μL of reaction buffer (0.1 mM potassium phosphate, pH 7.4, 0.05 M NaCl, 5 mM cholic acid, 0.1% Triton X-100). This was mixed with 50 μL of enzyme mixture that contained 2 U/mL of horseradish peroxidase, 2 U/mL cholesterol oxidase, 300 μM Amplex Red. Reaction mixture was incubated at 37 °C in dark for 30 mins. Fluorescence was measured on Tecan Spark multimode microplate reader with excitation and emission set at 540 and 560 nm. Background fluorescence was calculated by incubating lipid extracts with reaction buffer without enzyme mixture. Linearity of the assay was determined by running standard curves for 0 to 10 μg/mL of cholesterol standard. Fluorescence values were normalized to the amount of the protein, and fold change was calculated by dividing each measurement to the mean of values for WT samples.

Microscopy and image analysisFluorescence images were acquired on a Nikon CSU-W1 SoRa spinning-disc microscope fitted with an ORCA-Fusion BT Digital CMOS camera (Hamamatsu #C15440–20IP). For cells, images were acquired through a 100x SR HP Apo 1.49 NA objective (Nikon #MRD71970). Green fluorophores (IgG-Alexa-488) were exited using a 488-nm laser and red fluorophores (IgG-Alexa-562) were exited using a 561-nm laser. Emission filters were for green signal = 525/50 and for red signal = 605/52. Images were processed and prepared for figures using Fiji-ImageJ. Signals in the lysosomes (e.g., BMP, DQ-BSA or galectin-3) were quantified by masking the signal for the lysosomal marker LAMP1 or LysoTracker Red DND-99 (Invitrogen #L7528).

Microscopy and image analysis

Fluorescence images were acquired on a Nikon CSU-W1 SoRa spinning-disc microscope fitted with an ORCA-Fusion BT Digital CMOS camera (Hamamatsu #C15440–20IP). For cells, images were acquired through a 100x SR HP Apo 1.49 NA objective (Nikon #MRD71970). Green fluorophores (IgG-Alexa-488) were exited using a 488-nm laser and red fluorophores (IgG-Alexa-562) were exited using a 561-nm laser. Emission filters were for green signal = 525/50 and for red signal = 605/52. Images were processed and prepared for figures using Fiji-ImageJ. Signals in the lysosomes (e.g., BMP, DQ-BSA or galectin-3) were quantified by masking the signal for the lysosomal marker LAMP1 or LysoTracker Red DND-99 (Invitrogen #L7528).

Western blot analysisCells were scraped into ice-chilled DPBS and pelleted at 1000 g for 2 min. Cell pellets were resuspended in DPBS, lysed by probe sonication (1 sec on, 3 sec off, 60% amplitude), boiled with 4X loading dye at 95°C for 30 min for denaturation. Denatured proteins were separated on 4–15% gradient SDS-PAGE gels at 120 V and transferred to methanol-activated PVDF blotting membrane at 60 V for 3 h at 4°C. Thereafter, PVDF membranes were blocked with 5% (w/v) skim milk with 0.1% (v/v) Tween-20 in DPBS (blocking buffer) for 1 h at RT. Blocked membranes were incubated with primary antibodies (1:1000 diluted in blocking buffer) overnight at 4° C, washed three times with 0.1% (w/v) Tween-20 in DPBS (washing buffer), incubated with secondary antibody (Santa Cruz Biotechnology #sc516102 #sc2357 – 1:5000 diluted in blocking buffer) for 1 h at RT. Membranes were washed three times with washing buffer and developed with ECL chemiluminescence substrates and imaged on a BioRad ChemiDoc Imager.

Western blot analysis

Cells were scraped into ice-chilled DPBS and pelleted at 1000 g for 2 min. Cell pellets were resuspended in DPBS, lysed by probe sonication (1 sec on, 3 sec off, 60% amplitude), boiled with 4X loading dye at 95°C for 30 min for denaturation. Denatured proteins were separated on 4–15% gradient SDS-PAGE gels at 120 V and transferred to methanol-activated PVDF blotting membrane at 60 V for 3 h at 4°C. Thereafter, PVDF membranes were blocked with 5% (w/v) skim milk with 0.1% (v/v) Tween-20 in DPBS (blocking buffer) for 1 h at RT. Blocked membranes were incubated with primary antibodies (1:1000 diluted in blocking buffer) overnight at 4° C, washed three times with 0.1% (w/v) Tween-20 in DPBS (washing buffer), incubated with secondary antibody (Santa Cruz Biotechnology #sc516102 #sc2357 – 1:5000 diluted in blocking buffer) for 1 h at RT. Membranes were washed three times with washing buffer and developed with ECL chemiluminescence substrates and imaged on a BioRad ChemiDoc Imager.

Lysosome purificationsLysosomes were immunoisolated as reported.37 TMEM192–3X HA-tagged cells from one 150-mm plate at 70% confluency were scraped into ice-chilled DPBS supplemented and pelleted at 1000 g for 2 min. The cell pellet was Dounce-homogenized in 1 mL ice-chilled KPBS buffer containing 136 mM KCl, 10 mM KH2PO4 adjusted to pH 7.25 with KOH. Cell lysates were spun down at 1000 g for 2 min. Supernatant was collected and incubated with anti-HA magnetic beads, washed and pre-equilibrated with KPBS buffer for 15 min at 4° C. Beads were washed with excess KPBS buffer 10 times. Proteins were eluted from the beads using lysis buffer containing 50 mM Tris, 150 mM NaCl, 1 mM EDTA, 1% (v/v) Triton X-100 adjusted to pH 7.4.

Lysosome purifications

Lysosomes were immunoisolated as reported.37 TMEM192–3X HA-tagged cells from one 150-mm plate at 70% confluency were scraped into ice-chilled DPBS supplemented and pelleted at 1000 g for 2 min. The cell pellet was Dounce-homogenized in 1 mL ice-chilled KPBS buffer containing 136 mM KCl, 10 mM KH2PO4 adjusted to pH 7.25 with KOH. Cell lysates were spun down at 1000 g for 2 min. Supernatant was collected and incubated with anti-HA magnetic beads, washed and pre-equilibrated with KPBS buffer for 15 min at 4° C. Beads were washed with excess KPBS buffer 10 times. Proteins were eluted from the beads using lysis buffer containing 50 mM Tris, 150 mM NaCl, 1 mM EDTA, 1% (v/v) Triton X-100 adjusted to pH 7.4.

Recombinant protein expression and purificationHMC3 or HEK293T cells were grown at ~50% confluency in complete DMEM medium and transfected with pSPORT6 expression plasmids carrying human versions of PLD1, PLD2, PLD3, PLD4, PLD5, and PLD6 genes, using FuGene HD transfection reagent (Promega #E2312) for 24 h and thereafter processed for immunostaining or for lysosomal isolations and western blots. For PLD3, PLD4, PLA2G15 and CLN5 purifications, HEK293FT cells were grown at 1 × 106 cells/mL and transfected with C-terminal FLAG-tagged WT or active-site mutants cloned in pcDNA3.1 using PEI-max (Polysciences #24765–100). Cells were treated with 10 mM sodium butyrate for ~16 h after transfection and cultured for two additional days. Cells were collected by centrifugation at 2500 g for 15 min, and cell pellets were rinsed with DPBS. Cell pellets were resuspended in 50 mM Tris (pH 7.5), 200 mM NaCL, 5 mM MgCL2, 1 mM EDTA, 5 mM TCEP (resuspension buffer) and lysed by probe sonication (60% amplitude, 1 sec on, 3 sec off for 10 min). Lysates were centrifuged at 25,000 g for 1 h at 4° C. Supernatants were collected, syringe filtered with a 0.45-μM syringe filter, and incubated with anti-FLAG beads (Sigma-Aldrich #A2220) pre-equilibrated with resuspension buffer without TCEP. Beads were washed 10 times with 10 column volumes of resuspension buffer without TCEP. Proteins were eluted with 50 μg/mL of 1x FLAG peptide (Sigma-Aldrich #F3290), concentrated using 30 kDa centricons (Millipore #UFC9030). Concentrated proteins were buffer exchanged with sodium citrate buffer, pH 5.4, using button dialysis, followed by centrifugation at 50,000 g for 30 min to pellet any aggregates. Protein concentration in the supernatant fraction was accessed by nanodrop spectrophotometry, and purity of the protein was accessed using 10% Coomassie-stained SDS-PAGE gel and label-free mass spectrometry.

Recombinant protein expression and purification

HMC3 or HEK293T cells were grown at ~50% confluency in complete DMEM medium and transfected with pSPORT6 expression plasmids carrying human versions of PLD1, PLD2, PLD3, PLD4, PLD5, and PLD6 genes, using FuGene HD transfection reagent (Promega #E2312) for 24 h and thereafter processed for immunostaining or for lysosomal isolations and western blots. For PLD3, PLD4, PLA2G15 and CLN5 purifications, HEK293FT cells were grown at 1 × 106 cells/mL and transfected with C-terminal FLAG-tagged WT or active-site mutants cloned in pcDNA3.1 using PEI-max (Polysciences #24765–100). Cells were treated with 10 mM sodium butyrate for ~16 h after transfection and cultured for two additional days. Cells were collected by centrifugation at 2500 g for 15 min, and cell pellets were rinsed with DPBS. Cell pellets were resuspended in 50 mM Tris (pH 7.5), 200 mM NaCL, 5 mM MgCL2, 1 mM EDTA, 5 mM TCEP (resuspension buffer) and lysed by probe sonication (60% amplitude, 1 sec on, 3 sec off for 10 min). Lysates were centrifuged at 25,000 g for 1 h at 4° C. Supernatants were collected, syringe filtered with a 0.45-μM syringe filter, and incubated with anti-FLAG beads (Sigma-Aldrich #A2220) pre-equilibrated with resuspension buffer without TCEP. Beads were washed 10 times with 10 column volumes of resuspension buffer without TCEP. Proteins were eluted with 50 μg/mL of 1x FLAG peptide (Sigma-Aldrich #F3290), concentrated using 30 kDa centricons (Millipore #UFC9030). Concentrated proteins were buffer exchanged with sodium citrate buffer, pH 5.4, using button dialysis, followed by centrifugation at 50,000 g for 30 min to pellet any aggregates. Protein concentration in the supernatant fraction was accessed by nanodrop spectrophotometry, and purity of the protein was accessed using 10% Coomassie-stained SDS-PAGE gel and label-free mass spectrometry.

In vitro enzymatic assaysFor BMP synthesis assays, lipid substrates were prepared by resuspending the required amount of lipids in sodium citrate buffer at pH 5.4 using water-bath sonication to a final concentration of 50 μM of phospholipids and 25 μM of mono- or diacylglycerol (Avanti Polar Lipids). Substrates were incubated with extracts of affinity isolated lysosomes on anti-HA magnetic beads (3–5 μg of lysosomal proteins) or 1 μg of purified enzyme (final volume = 100 μL) for 45 min, 37 °C with constant shaking in glass vials. Thereafter, the reaction was quenched with 250 μL of 2:1 chloroform:methanol spiked with 14:0/14:0 BMP as internal standard. The mixture was vortexed and spun at 1000 g for 5 min, and the bottom organic layer was collected in a separate glass vial. Thereafter, the aqueous fraction was acidified with 2 μL of formic acid, vortexed and extracted again with 250 μL of chloroform. Organic layers from both steps were pooled, dried under a N2-stream and analyzed by LC-MS/MS. BMP formation was monitored and normalized to internal standards, the amount of protein used and the time of the reaction. For nuclease competition assay, 50 μM of S,R-lyso-PG and 25 μM of monoacylglycerol was incubated with 1 μg of purified PLD3 along with increasing concentrations of single-stranded DNA (5 μM to 5 mM). Sequence of single-stranded DNA was as reported previously36 : 5’–TCCATGACGTTCCTGATGCTAAGTATGCACTTCATCGTCAAGCAATGCTATGCAT–3’.To test the stability of BMP stereoisomers in lysosomes 100 μM of 18:1/18:1 BMP stereoisomers were incubated with 5 μg of lysosomal lysates for 1 h at 37 °C with constant shaking in glass vials. To test the stability of BMP synthesized by PLD3 or PLD4, enzymatic products were incubated with 1 μg of PLA2G15 or 5 μg of phospholipase A1 from Aspergillus oryzae (Sigma-Aldrich #L3295) for 15 mins at 37 °C. This incubation time was reduced to accommodate for the low amount of BMP synthesized by PLD3 or PLD4 compared to their commercially available synthetic standards. Reaction was quenched with 250 μL of 2:1 chloroform: methanol; and lipids were extracted as described for the BMP synthesis assay. Hydrolysis of BMP stereoisomer standards was monitored by TLC and reaction rates were measured by LC-MS/MS. PLA2G15 used in the reactions was obtained by overexpressing C-terminally FLAG-tagged human PLA2G15 in HEK293FT cells and affinity-isolation using anti-FLAG magnetic beads (Sigma-Aldrich #M8823).To test for lysosomal catabolic activity (e.g., ganglioside catabolism), a fluorescence plate reader assay was setup as described.13 Artificial substrate mimetics for lysosomal enzymes were dissolved in DMSO to stock concentrations of 100 μM. Cells were Dounce-homogenized in ice-cold DPBS and probe sonicated for 1 min (60% amplitude, 1 sec on, 3 sec off). Protein concentrations were estimated using Pierce BCA Protein Assay Kit (Pierce #23225) with BSA as control for standard curves. Lysates (25 μg) were incubated with 1 mM of fluorogenic substrates in sodium citrate buffer (total volume = 100 μL) at pH 5.4 at 37 °C for 30 min. Thereafter, reactions were quenched with 300 μL glycine-buffer, pH 10.4. Fluorescence was measured in a Tecan Spark multimode microplate reader with excitation and emission set at 350 and 440 nm. A standard curve was drawn using 4-methylumbelliferone (Sigma-Aldrich #M1381) as standards dissolved in reaction buffer + quenching buffer in ratios used for the assay. Substrates used were as follows: 4-methylumbelliferyl-β-D-galactoside for β-galactosidase, 4-methylumbelliferyl-β-D-glucopyranoside for β-glucocerebrosidase, 4-nethylumbelliferyl-2-acetamido-2-deoxy-β-D-glucopyranoside for β-hexosaminidase, 7-(α-D-glucopyranosyloxy)-4-methyl-2H-1-benzopyran-2-one for α-glucosidase, 7-(α-D-galactopyranosyloxy)-4-methyl-2H-1-benzopyran-2-one for α-galactosidase, 7-[[2-(acetylamino)-2-deoxy-α-D-galactopyranosyl]oxy]-4-methyl-2H-1-benzopyran-2-one for β-galactopyranosaminidase activity, hexadecanoic acid, 4-methyl-2-oxo-2H-1-benzopyran-7-yl ester for lysosomal acid lipase, and 4-methyl-2-oxo-2H-1-benzopyran-7-yl 5-(acetylamino)-3,5-dideoxy-α-neuraminic acid for neuraminidases. All artificial substrates were procured from Cayman Chemicals.

In vitro enzymatic assays

For BMP synthesis assays, lipid substrates were prepared by resuspending the required amount of lipids in sodium citrate buffer at pH 5.4 using water-bath sonication to a final concentration of 50 μM of phospholipids and 25 μM of mono- or diacylglycerol (Avanti Polar Lipids). Substrates were incubated with extracts of affinity isolated lysosomes on anti-HA magnetic beads (3–5 μg of lysosomal proteins) or 1 μg of purified enzyme (final volume = 100 μL) for 45 min, 37 °C with constant shaking in glass vials. Thereafter, the reaction was quenched with 250 μL of 2:1 chloroform:methanol spiked with 14:0/14:0 BMP as internal standard. The mixture was vortexed and spun at 1000 g for 5 min, and the bottom organic layer was collected in a separate glass vial. Thereafter, the aqueous fraction was acidified with 2 μL of formic acid, vortexed and extracted again with 250 μL of chloroform. Organic layers from both steps were pooled, dried under a N2-stream and analyzed by LC-MS/MS. BMP formation was monitored and normalized to internal standards, the amount of protein used and the time of the reaction. For nuclease competition assay, 50 μM of S,R-lyso-PG and 25 μM of monoacylglycerol was incubated with 1 μg of purified PLD3 along with increasing concentrations of single-stranded DNA (5 μM to 5 mM). Sequence of single-stranded DNA was as reported previously36 : 5’–TCCATGACGTTCCTGATGCTAAGTATGCACTTCATCGTCAAGCAATGCTATGCAT–3’.

To test the stability of BMP stereoisomers in lysosomes 100 μM of 18:1/18:1 BMP stereoisomers were incubated with 5 μg of lysosomal lysates for 1 h at 37 °C with constant shaking in glass vials. To test the stability of BMP synthesized by PLD3 or PLD4, enzymatic products were incubated with 1 μg of PLA2G15 or 5 μg of phospholipase A1 from Aspergillus oryzae (Sigma-Aldrich #L3295) for 15 mins at 37 °C. This incubation time was reduced to accommodate for the low amount of BMP synthesized by PLD3 or PLD4 compared to their commercially available synthetic standards. Reaction was quenched with 250 μL of 2:1 chloroform: methanol; and lipids were extracted as described for the BMP synthesis assay. Hydrolysis of BMP stereoisomer standards was monitored by TLC and reaction rates were measured by LC-MS/MS. PLA2G15 used in the reactions was obtained by overexpressing C-terminally FLAG-tagged human PLA2G15 in HEK293FT cells and affinity-isolation using anti-FLAG magnetic beads (Sigma-Aldrich #M8823).

To test for lysosomal catabolic activity (e.g., ganglioside catabolism), a fluorescence plate reader assay was setup as described.13 Artificial substrate mimetics for lysosomal enzymes were dissolved in DMSO to stock concentrations of 100 μM. Cells were Dounce-homogenized in ice-cold DPBS and probe sonicated for 1 min (60% amplitude, 1 sec on, 3 sec off). Protein concentrations were estimated using Pierce BCA Protein Assay Kit (Pierce #23225) with BSA as control for standard curves. Lysates (25 μg) were incubated with 1 mM of fluorogenic substrates in sodium citrate buffer (total volume = 100 μL) at pH 5.4 at 37 °C for 30 min. Thereafter, reactions were quenched with 300 μL glycine-buffer, pH 10.4. Fluorescence was measured in a Tecan Spark multimode microplate reader with excitation and emission set at 350 and 440 nm. A standard curve was drawn using 4-methylumbelliferone (Sigma-Aldrich #M1381) as standards dissolved in reaction buffer + quenching buffer in ratios used for the assay. Substrates used were as follows: 4-methylumbelliferyl-β-D-galactoside for β-galactosidase, 4-methylumbelliferyl-β-D-glucopyranoside for β-glucocerebrosidase, 4-nethylumbelliferyl-2-acetamido-2-deoxy-β-D-glucopyranoside for β-hexosaminidase, 7-(α-D-glucopyranosyloxy)-4-methyl-2H-1-benzopyran-2-one for α-glucosidase, 7-(α-D-galactopyranosyloxy)-4-methyl-2H-1-benzopyran-2-one for α-galactosidase, 7-[[2-(acetylamino)-2-deoxy-α-D-galactopyranosyl]oxy]-4-methyl-2H-1-benzopyran-2-one for β-galactopyranosaminidase activity, hexadecanoic acid, 4-methyl-2-oxo-2H-1-benzopyran-7-yl ester for lysosomal acid lipase, and 4-methyl-2-oxo-2H-1-benzopyran-7-yl 5-(acetylamino)-3,5-dideoxy-α-neuraminic acid for neuraminidases. All artificial substrates were procured from Cayman Chemicals.

Preparation of S,R-lyso-PGR,S-BMP (100 μM) was incubated with 5 μg of human PLA2G15 at 37 °C in glass vials with constant shaking. The reaction was monitored by running small aliquots on TLC. The reaction was quenched and extracted with excess 2:1 (v/v) chloroform:methanol. Extracts were dried under an N2 stream, and the lyso-PG mixture was separated on a C18 reverse phase column (Pheneomenex, 5 μm, 50 × 4.6 mm) using the following solvents: Solvent A= 95:5 (v/v) H2O/MeOH + 0.1% (v/v) NH4OH, solvent B= 60:35:5 (v/v) IPA/MeOH/H2O + 0.1% (v/v) NH4OH. The liquid chromatography (LC) gradient was 30 mins, with a stable flow rate of 0.5 mL/min, starting 5% B for 5 min, linear gradient of B from 5–100% over 23 min, 100% B from min 23–25, and re-equilibration with 5% B for 5 min. The isolated lyso-PG was pooled and dried under N2, weighed and used for BMP synthesis assays.

Preparation of S,R-lyso-PG

R,S-BMP (100 μM) was incubated with 5 μg of human PLA2G15 at 37 °C in glass vials with constant shaking. The reaction was monitored by running small aliquots on TLC. The reaction was quenched and extracted with excess 2:1 (v/v) chloroform:methanol. Extracts were dried under an N2 stream, and the lyso-PG mixture was separated on a C18 reverse phase column (Pheneomenex, 5 μm, 50 × 4.6 mm) using the following solvents: Solvent A= 95:5 (v/v) H2O/MeOH + 0.1% (v/v) NH4OH, solvent B= 60:35:5 (v/v) IPA/MeOH/H2O + 0.1% (v/v) NH4OH. The liquid chromatography (LC) gradient was 30 mins, with a stable flow rate of 0.5 mL/min, starting 5% B for 5 min, linear gradient of B from 5–100% over 23 min, 100% B from min 23–25, and re-equilibration with 5% B for 5 min. The isolated lyso-PG was pooled and dried under N2, weighed and used for BMP synthesis assays.

Chiral derivatization and separation of BMP stereoisomersStandards of BMP stereoisomers or enzymatic products were mixed with derivatization agent (R)-(−)-1-(1-naphthyl)ethyl isocyanate (Sigma-Aldrich #220442) in a ratio of 1:10 in 1 mL of dichloromethanol (VWR BDH23373.400) and 50 μL of pyridine (Sigma-Aldrich #270970) overnight (14–16 h) at RT. The reaction mixture was dried under a stream of N2 and extracted with 2:1:1 chloroform:methanol:water. The bottom organic layer was collected and separated on YMC Chiral ART Cellulose-SZ, analytical column (YMC America #KSZ99S03–2502WT). Solvent A = 60:40 (v/v) ACN/water + 0.1% (v/v) formic acid + 10 mM ammonium formate, Solvent B = 90:10 (v/v) IPA: ACN + 0.1% (v/v) formic acid + 10 mM ammonium formate. The LC gradient was 45 min, with a stable flow rate of 0.2 mL/min, starting with 40% B for 5 min, linear gradient of 40–70% B for min 5–30, maintained at 100% B for mins 30–33, re-equilibrated with 40% B for min 33–45 at 0.3 mL/min. The column temperature was maintained at 10 °C. MS parameters were as described in LC-MS/MS analysis of lipids section.

Chiral derivatization and separation of BMP stereoisomers

Standards of BMP stereoisomers or enzymatic products were mixed with derivatization agent (R)-(−)-1-(1-naphthyl)ethyl isocyanate (Sigma-Aldrich #220442) in a ratio of 1:10 in 1 mL of dichloromethanol (VWR BDH23373.400) and 50 μL of pyridine (Sigma-Aldrich #270970) overnight (14–16 h) at RT. The reaction mixture was dried under a stream of N2 and extracted with 2:1:1 chloroform:methanol:water. The bottom organic layer was collected and separated on YMC Chiral ART Cellulose-SZ, analytical column (YMC America #KSZ99S03–2502WT). Solvent A = 60:40 (v/v) ACN/water + 0.1% (v/v) formic acid + 10 mM ammonium formate, Solvent B = 90:10 (v/v) IPA: ACN + 0.1% (v/v) formic acid + 10 mM ammonium formate. The LC gradient was 45 min, with a stable flow rate of 0.2 mL/min, starting with 40% B for 5 min, linear gradient of 40–70% B for min 5–30, maintained at 100% B for mins 30–33, re-equilibrated with 40% B for min 33–45 at 0.3 mL/min. The column temperature was maintained at 10 °C. MS parameters were as described in LC-MS/MS analysis of lipids section.

Lipid extractionLipids were extracted from cells or tissues using established protocols.63 Briefly, cells were lysed in DPBS using a probe sonicator, or tissues were Dounce-homogenized in DPBS, and lipids were extracted with a chloroform and methanol mixture, with spiked-in lipid splash mix and/or BMP 14:0/14:0, as internal standards. The final ratio of chloroform:methanol:DPBS in the extraction mixture was 2:1:1. The extraction mixture was vortexed and spun down at 1000 g for 10 min. The bottom organic phase was collected in a glass tube. The top aqueous phase was acidified with 0.5% (v/v) formic acid, vortexed and re-extracted with chloroform by spinning down as in the previous step. Both organic phases were pooled and dried under a stream of N2 and analysed by LC-MS/MS.

Lipid extraction

Lipids were extracted from cells or tissues using established protocols.63 Briefly, cells were lysed in DPBS using a probe sonicator, or tissues were Dounce-homogenized in DPBS, and lipids were extracted with a chloroform and methanol mixture, with spiked-in lipid splash mix and/or BMP 14:0/14:0, as internal standards. The final ratio of chloroform:methanol:DPBS in the extraction mixture was 2:1:1. The extraction mixture was vortexed and spun down at 1000 g for 10 min. The bottom organic phase was collected in a glass tube. The top aqueous phase was acidified with 0.5% (v/v) formic acid, vortexed and re-extracted with chloroform by spinning down as in the previous step. Both organic phases were pooled and dried under a stream of N2 and analysed by LC-MS/MS.

Ganglioside extractionTissues or cells were Dounce-homogenized with ice-chilled DPBS, and methanol was added to the final ratio of 90:10 methanol: water. The extraction mixture was spun down at 1000 g for 20 min to pellet proteins, and the solvent layers containing gangliosides and other metabolites were collected in glass tubes and dried under an N2 stream. Dried extract was reconstituted in 1 mL of LC-MS-grade water and desalted by Sola HRP SPE 30mg/2MI 96-well plate 1EA (Thermo Scientific #60509–001). Desalting cartridges were cleaned three times with 1 mL of methanol, equilibrated three times with LC-MS-grade water, and then the extracts dissolved in water were loaded onto the cartridge and washed three times with water, and finally, gangliosides were eluted by 3 mL of methanol. Eluates were dried under N2 and reconstituted in 300:150:50 of MeOH:water:chloroform.

Ganglioside extraction

Tissues or cells were Dounce-homogenized with ice-chilled DPBS, and methanol was added to the final ratio of 90:10 methanol: water. The extraction mixture was spun down at 1000 g for 20 min to pellet proteins, and the solvent layers containing gangliosides and other metabolites were collected in glass tubes and dried under an N2 stream. Dried extract was reconstituted in 1 mL of LC-MS-grade water and desalted by Sola HRP SPE 30mg/2MI 96-well plate 1EA (Thermo Scientific #60509–001). Desalting cartridges were cleaned three times with 1 mL of methanol, equilibrated three times with LC-MS-grade water, and then the extracts dissolved in water were loaded onto the cartridge and washed three times with water, and finally, gangliosides were eluted by 3 mL of methanol. Eluates were dried under N2 and reconstituted in 300:150:50 of MeOH:water:chloroform.

TLC for lipidsExtracted lipids were separated on high-performance thin layer chromatography (Sigma-Aldrich #1056310001) in the first dimension using 70:30:3:2 (v/v/v/v) chloroform:methanol:water:concentrated ammonia as the running phase. For BMP separation, plates were dried at RT for 1 hour and run in the second dimension using 65:35:5 (v/v/v) of chloroform:methanol:water as the running phase. Plates were dried and sprayed with 5% (w/v) copper sulfate dissolved in 25% (v/v) ethanol and 8.5% (v/v) phosphoric acid, followed by charring at 60 °C for 1 h. Plates were imaged on BioRad ChemiDoc Imager.

TLC for lipids

Extracted lipids were separated on high-performance thin layer chromatography (Sigma-Aldrich #1056310001) in the first dimension using 70:30:3:2 (v/v/v/v) chloroform:methanol:water:concentrated ammonia as the running phase. For BMP separation, plates were dried at RT for 1 hour and run in the second dimension using 65:35:5 (v/v/v) of chloroform:methanol:water as the running phase. Plates were dried and sprayed with 5% (w/v) copper sulfate dissolved in 25% (v/v) ethanol and 8.5% (v/v) phosphoric acid, followed by charring at 60 °C for 1 h. Plates were imaged on BioRad ChemiDoc Imager.

LC-MS/MS analysis of lipidsAll samples were analysed on a Vanquish UHPLC (Thermo Scientific) coupled to a Orbitrap Exploris 240 mass spectrometer (Thermo Scientific #BRE725535). For routine lipid analysis, extracts were separated on a C18 reverse phase column (Pheneomenex, 5 μm, 50 × 4.6 mm #00B-4435-E0) as described.63 Briefly, for positive mode: solvent A= 95:5 (v/v) H2O/MeOH + 0.1% formic acid + 10 mM ammonium formate, solvent B= 60:35:5 (v/v/v) isopropanol (IPA)/MeOH/H2O + 0.1% (v/v) formic acid + 10 mM ammonium formate. For negative mode, solvent A= 95:5 (v/v) H2O/MeOH + 0.1% (v/v) NH4OH, solvent B= 60:35:5 (v/v) IPA/MeOH/H2O + 0.1% (v/v) NH4OH. LC gradient was 60 min, with stable flow rate of 0.5 mL/min starting 5% B for 5 min, linear gradient of B from 5–100% over 50 min, 100% B from min 50–55, and re-equilibration with 5% B for 5 min. The column temperature was maintained at 55 °C. MS parameters were as follows: - heated ESI mode of ionization, sheath gas = 40 units, aux gas = 8 units, sweep gas =1 units, spray voltage = static, positive ion =3,400 V, negative ion = 2500 V, ion transfer tube temp= 320 °C, vaporizer temp=275 °C. Orbitrap resolution at MS1 was set to 120000, scan range 250–1800, RF lens = 60%, AGC target = standard, Orbitrap resolution at MS2 was set to 30000. EASY-IC™ was used for internal calibration. Lipids were searched and aligned for annotation in LipidSearch 5.0 (Thermo Scientific #OPTON-30880) with precursor mass error tolerance of 5 ppm and product mass error tolerance of 10 ppm, retention time deviation tolerance of 0.5 min.

LC-MS/MS analysis of lipids

All samples were analysed on a Vanquish UHPLC (Thermo Scientific) coupled to a Orbitrap Exploris 240 mass spectrometer (Thermo Scientific #BRE725535). For routine lipid analysis, extracts were separated on a C18 reverse phase column (Pheneomenex, 5 μm, 50 × 4.6 mm #00B-4435-E0) as described.63 Briefly, for positive mode: solvent A= 95:5 (v/v) H2O/MeOH + 0.1% formic acid + 10 mM ammonium formate, solvent B= 60:35:5 (v/v/v) isopropanol (IPA)/MeOH/H2O + 0.1% (v/v) formic acid + 10 mM ammonium formate. For negative mode, solvent A= 95:5 (v/v) H2O/MeOH + 0.1% (v/v) NH4OH, solvent B= 60:35:5 (v/v) IPA/MeOH/H2O + 0.1% (v/v) NH4OH. LC gradient was 60 min, with stable flow rate of 0.5 mL/min starting 5% B for 5 min, linear gradient of B from 5–100% over 50 min, 100% B from min 50–55, and re-equilibration with 5% B for 5 min. The column temperature was maintained at 55 °C. MS parameters were as follows: - heated ESI mode of ionization, sheath gas = 40 units, aux gas = 8 units, sweep gas =1 units, spray voltage = static, positive ion =3,400 V, negative ion = 2500 V, ion transfer tube temp= 320 °C, vaporizer temp=275 °C. Orbitrap resolution at MS1 was set to 120000, scan range 250–1800, RF lens = 60%, AGC target = standard, Orbitrap resolution at MS2 was set to 30000. EASY-IC™ was used for internal calibration. Lipids were searched and aligned for annotation in LipidSearch 5.0 (Thermo Scientific #OPTON-30880) with precursor mass error tolerance of 5 ppm and product mass error tolerance of 10 ppm, retention time deviation tolerance of 0.5 min.

LC-MS/MS analysis of BMPAll samples were analysed on a Vanquish UHPLC (Thermo Scientific) coupled to a Orbitrap Exploris 240 mass spectrometer (Thermo Scientific #BRE725535). Chromatographic separation of BMP and PG was achieved on a C-30 reverse phase column (Thermo Scientific #078664, 3 mm 250 × 2.1 mm). Solvent = 60:40 (v/v) ACN/water + 0.1% (v/v) formic acid + 10 mM ammonium formate, Solvent B = 90:10 (v/v) IPA: ACN + 0.1% (v/v) formic acid + 10 mM ammonium formate. The LC gradient was 45 min, with a stable flow rate of 0.2 mL/min starting 40% B, linear gradient of 45–55% over mins 1–7, maintained at 65% B for mins 8–12, linear gradient of 65–70% B for mins 12–30, maintained at 100% B for mins 30–33, re-equilibrated with 40% for mins 33–45. Column temperature was maintained at 50 °C. MS parameters were as follows: - heated ESI mode of ionization, sheath gas = 40 units, aux gas = 8 units, sweep gas =1 units, spray voltage = static, positive ion =3,400 V, negative ion = 2500 V, ion transfer tube temp= 320 °C, vaporizer temp=275 °C. Orbitrap resolution at MS1 was set to 120000, RF lens = 60%, Orbitrap resolution at MS2 was set to 30000. Orbitrap was set up with an unscheduled targeted inclusion mass list corresponding to PG and BMP. EASY-IC™ was used for internal calibration. To avoid source fragmentation of cardiolipin confounding measurements, we inactivated source fragmentation during data acquisition with mass spectrometry. Using this method, an analysis of cardiolipin standards (CL 14:0/14:0/14:0/14:0) showed no notable source fragmentation of CL into PG or its fragment ions.PG and BMP peaks were identified using Excalibur 3.1 (Thermo Scientific #OPTON-30965), the area under the curve was normalized to the area under the curve for internal standards (14:0/14:0 BMP for cells and 15:0/18:1(d7) PG for mice tissues) and amount of protein/wet weight of tissue used. Samples for BMP analysis were run in both negative and positive ion mode. All the quantification presented in the figures of the manuscript come from the negative-mode analysis. Positive-mode analysis of BMP was used only to confirm the identify of BMP species. Targeted precursor masses for BMP, and their predicted/observed fragments are as follows:
Negative ModePositive ModeSpeciesParent IonFragment IonFragment IonParent IonFragment IonFragment IonBMP 16:0/16:0721.51255.23255.23740.54313.27313.27BMP 16:0/18:1747.52255.23281.25766.56313.27339.23BMP 16:0/18:0749.54255.23283.26768.58313.27341.31BMP 16:0/20:1775.55255.23309.28794.59313.27367.32BMP 16:0/20:4769.51255.23303.23788.54313.27361.27BMP 16:0/22:6793.51255.23327.23812.54313.27385.27BMP 16:0/17:1733.51255.23267.23752.54313.27325.27BMP 16:0/12:0665.44255.23199.17684.48313.27257.21BMP 16:0/14:0693.47255.23227.20712.51313.27285.24BMP 16:0/18:2745.51255.23279.23764.54313.27337.27BMP 16:0/20:0777.57255.23311.30796.61313.27369.34BMP 16:0/22:0805.60255.23339.33824.64313.27397.37BMP 16:0/22:1803.58255.23337.31822.62313.27395.35BMP 16:0/22:4797.54255.23331.26816.58313.27389.31BMP 16:0/24:0833.63255.23367.36852.67313.27425.4BMP 16:0/24:1831.62255.23365.34850.65313.27423.38BMP 18:1/18:1773.54281.25281.25792.58339.23339.23BMP 18:1/18:0775.55281.25283.26794.59339.23341.31BMP 18:1/20:1801.57281.25309.28820.61339.23367.32BMP 18:1/20:4795.52281.25303.23814.56339.23361.27BMP 18:1/22:6819.52281.25327.23838.56339.23385.27BMP 18:1/17:1759.52281.25267.23778.56339.23325.27BMP 18:1/12:0691.46281.25199.17710.50339.23257.21BMP 18:1/14:0719.49281.25227.20738.53339.23285.24BMP 18:1/18:2771.52281.25279.23790.56339.23337.27BMP 18:1/20:0803.58281.25311.30822.62339.23369.34BMP 18:1/22:0831.62281.25339.33850.65339.23397.37BMP 18:1/22:1829.60281.25337.31848.64339.23395.35BMP 18:1/22:4823.55281.25331.26842.59339.23389.31BMP 18:1/24:0859.65281.25367.36878.68339.23425.4BMP 18:1/24:1857.63281.25365.34876.67339.23423.38BMP 18:2/18:2769.51279.23279.23788.54337.27337.27BMP 18:2/20:1799.55279.23309.28818.59337.27367.32BMP 18:2/20:4793.51279.23303.23812.54337.27361.27BMP 18:2/22:1827.58279.23337.31846.62337.27395.35BMP 18:2/22:4821.54279.23331.26840.58337.27389.31BMP 18:2/22:6817.51279.23327.23836.54337.27385.27BMP 18:2/24:1855.62279.23365.34874.65337.27423.38BMP 20:1/20:1829.60309.28309.28848.64367.32367.32BMP 20:1/20:4823.55309.28303.23842.59367.32361.27BMP 20:1/22:1857.63309.28337.31876.67367.32395.35BMP 20:1/22:4851.58309.28331.26870.62367.32389.31BMP 20:1/22:6847.55309.28327.23866.59367.32385.27BMP 20:1/24:1885.66309.28365.34904.70367.32423.38BMP 20:4/20:4817.51303.23303.23836.54361.27361.27BMP 20:4/22:1851.58303.23337.31870.62361.27395.35BMP 20:4/22:4845.54303.23331.26864.58361.27389.31BMP 20:4/22:6841.51303.23327.23860.54361.27385.27BMP 20:4/24:1879.62303.23365.34898.65361.27423.38BMP 22:1/22:1885.66337.31337.31904.70395.35395.35BMP 22:1/22:4879.62337.31331.26898.65395.35389.31BMP 22:1/22:6875.58337.31327.23894.62395.35385.27BMP 22:1/24:1913.69337.31365.34932.73395.35423.38BMP 22:4/22:4873.57331.26331.26892.61389.31389.31BMP 22:4/22:6869.54331.26327.23888.58389.31385.27BMP 22:6/22:6865.51327.23327.23884.54385.27385.27BMP 22:6/24:1903.62327.23365.34922.65385.27423.38BMP 24:1/22:4907.65365.34331.26926.68423.38389.31BMP 24:1/24:1941.73365.34365.34960.76423.38423.38

LC-MS/MS analysis of BMP

All samples were analysed on a Vanquish UHPLC (Thermo Scientific) coupled to a Orbitrap Exploris 240 mass spectrometer (Thermo Scientific #BRE725535). Chromatographic separation of BMP and PG was achieved on a C-30 reverse phase column (Thermo Scientific #078664, 3 mm 250 × 2.1 mm). Solvent = 60:40 (v/v) ACN/water + 0.1% (v/v) formic acid + 10 mM ammonium formate, Solvent B = 90:10 (v/v) IPA: ACN + 0.1% (v/v) formic acid + 10 mM ammonium formate. The LC gradient was 45 min, with a stable flow rate of 0.2 mL/min starting 40% B, linear gradient of 45–55% over mins 1–7, maintained at 65% B for mins 8–12, linear gradient of 65–70% B for mins 12–30, maintained at 100% B for mins 30–33, re-equilibrated with 40% for mins 33–45. Column temperature was maintained at 50 °C. MS parameters were as follows: - heated ESI mode of ionization, sheath gas = 40 units, aux gas = 8 units, sweep gas =1 units, spray voltage = static, positive ion =3,400 V, negative ion = 2500 V, ion transfer tube temp= 320 °C, vaporizer temp=275 °C. Orbitrap resolution at MS1 was set to 120000, RF lens = 60%, Orbitrap resolution at MS2 was set to 30000. Orbitrap was set up with an unscheduled targeted inclusion mass list corresponding to PG and BMP. EASY-IC™ was used for internal calibration. To avoid source fragmentation of cardiolipin confounding measurements, we inactivated source fragmentation during data acquisition with mass spectrometry. Using this method, an analysis of cardiolipin standards (CL 14:0/14:0/14:0/14:0) showed no notable source fragmentation of CL into PG or its fragment ions.

PG and BMP peaks were identified using Excalibur 3.1 (Thermo Scientific #OPTON-30965), the area under the curve was normalized to the area under the curve for internal standards (14:0/14:0 BMP for cells and 15:0/18:1(d7) PG for mice tissues) and amount of protein/wet weight of tissue used. Samples for BMP analysis were run in both negative and positive ion mode. All the quantification presented in the figures of the manuscript come from the negative-mode analysis. Positive-mode analysis of BMP was used only to confirm the identify of BMP species. Targeted precursor masses for BMP, and their predicted/observed fragments are as follows:
Negative ModePositive ModeSpeciesParent IonFragment IonFragment IonParent IonFragment IonFragment IonBMP 16:0/16:0721.51255.23255.23740.54313.27313.27BMP 16:0/18:1747.52255.23281.25766.56313.27339.23BMP 16:0/18:0749.54255.23283.26768.58313.27341.31BMP 16:0/20:1775.55255.23309.28794.59313.27367.32BMP 16:0/20:4769.51255.23303.23788.54313.27361.27BMP 16:0/22:6793.51255.23327.23812.54313.27385.27BMP 16:0/17:1733.51255.23267.23752.54313.27325.27BMP 16:0/12:0665.44255.23199.17684.48313.27257.21BMP 16:0/14:0693.47255.23227.20712.51313.27285.24BMP 16:0/18:2745.51255.23279.23764.54313.27337.27BMP 16:0/20:0777.57255.23311.30796.61313.27369.34BMP 16:0/22:0805.60255.23339.33824.64313.27397.37BMP 16:0/22:1803.58255.23337.31822.62313.27395.35BMP 16:0/22:4797.54255.23331.26816.58313.27389.31BMP 16:0/24:0833.63255.23367.36852.67313.27425.4BMP 16:0/24:1831.62255.23365.34850.65313.27423.38BMP 18:1/18:1773.54281.25281.25792.58339.23339.23BMP 18:1/18:0775.55281.25283.26794.59339.23341.31BMP 18:1/20:1801.57281.25309.28820.61339.23367.32BMP 18:1/20:4795.52281.25303.23814.56339.23361.27BMP 18:1/22:6819.52281.25327.23838.56339.23385.27BMP 18:1/17:1759.52281.25267.23778.56339.23325.27BMP 18:1/12:0691.46281.25199.17710.50339.23257.21BMP 18:1/14:0719.49281.25227.20738.53339.23285.24BMP 18:1/18:2771.52281.25279.23790.56339.23337.27BMP 18:1/20:0803.58281.25311.30822.62339.23369.34BMP 18:1/22:0831.62281.25339.33850.65339.23397.37BMP 18:1/22:1829.60281.25337.31848.64339.23395.35BMP 18:1/22:4823.55281.25331.26842.59339.23389.31BMP 18:1/24:0859.65281.25367.36878.68339.23425.4BMP 18:1/24:1857.63281.25365.34876.67339.23423.38BMP 18:2/18:2769.51279.23279.23788.54337.27337.27BMP 18:2/20:1799.55279.23309.28818.59337.27367.32BMP 18:2/20:4793.51279.23303.23812.54337.27361.27BMP 18:2/22:1827.58279.23337.31846.62337.27395.35BMP 18:2/22:4821.54279.23331.26840.58337.27389.31BMP 18:2/22:6817.51279.23327.23836.54337.27385.27BMP 18:2/24:1855.62279.23365.34874.65337.27423.38BMP 20:1/20:1829.60309.28309.28848.64367.32367.32BMP 20:1/20:4823.55309.28303.23842.59367.32361.27BMP 20:1/22:1857.63309.28337.31876.67367.32395.35BMP 20:1/22:4851.58309.28331.26870.62367.32389.31BMP 20:1/22:6847.55309.28327.23866.59367.32385.27BMP 20:1/24:1885.66309.28365.34904.70367.32423.38BMP 20:4/20:4817.51303.23303.23836.54361.27361.27BMP 20:4/22:1851.58303.23337.31870.62361.27395.35BMP 20:4/22:4845.54303.23331.26864.58361.27389.31BMP 20:4/22:6841.51303.23327.23860.54361.27385.27BMP 20:4/24:1879.62303.23365.34898.65361.27423.38BMP 22:1/22:1885.66337.31337.31904.70395.35395.35BMP 22:1/22:4879.62337.31331.26898.65395.35389.31BMP 22:1/22:6875.58337.31327.23894.62395.35385.27BMP 22:1/24:1913.69337.31365.34932.73395.35423.38BMP 22:4/22:4873.57331.26331.26892.61389.31389.31BMP 22:4/22:6869.54331.26327.23888.58389.31385.27BMP 22:6/22:6865.51327.23327.23884.54385.27385.27BMP 22:6/24:1903.62327.23365.34922.65385.27423.38BMP 24:1/22:4907.65365.34331.26926.68423.38389.31BMP 24:1/24:1941.73365.34365.34960.76423.38423.38

LC-MS/MS analysis of gangliosidesFor LC-MS/MS analysis of gangliosides, samples were analyzed on a Vanquish UHPLC (Thermo Scientific) coupled to a Orbitrap Exploris 240 mass spectrometer (Thermo Scientific #BRE725535). Extracts were separated on a Kinetex HILIC column (Phenomenex #00D-4461-AN, 2.6 μm,100 × 2.1 mm). Solvent A= acetonitrile with 0.2% (v/v) acetic acid, solvent B = water with 10 mM ammonium acetate, pH 6.1, adjusted with acetic acid. The column temperature was maintained at 50 °C. The LC gradient started with a constant flow rate of 0.6 mL/min with 12.3% B at 0 min, linear gradient of 12.3% B to 22.1% B for min 1–15. The column was equilibrated between runs by 12.3% B for 5 mins. The spray voltage was set to −4.5 kV, and the heated capillary and the HESI were held at 300 °C and 250 °C, respectively. MS parameters were as follows: - heated ESI mode of ionization, sheath gas = 40 units, aux gas = 5 units, sweep gas =1 units, spray voltage = static, negative ion =−4,500 V, negative ion = 2500 V, ion transfer tube temp= 300 °C, vaporizer temp=250 °C. Orbitrap resolution at MS1 was set to 120000, scan range 250–1800, RF lens = 60%, AGC target = standard, Orbitrap resolution at MS2 was set to 30000. EASY-IC™ was used for internal calibration. Analysis was performed in Excalibur (Thermo Scientific #OPTON-30965). The area under the curve was normalized to GM3-d5 standards, followed by normalization to amount of protein/wet weight of tissue used.

LC-MS/MS analysis of gangliosides

For LC-MS/MS analysis of gangliosides, samples were analyzed on a Vanquish UHPLC (Thermo Scientific) coupled to a Orbitrap Exploris 240 mass spectrometer (Thermo Scientific #BRE725535). Extracts were separated on a Kinetex HILIC column (Phenomenex #00D-4461-AN, 2.6 μm,100 × 2.1 mm). Solvent A= acetonitrile with 0.2% (v/v) acetic acid, solvent B = water with 10 mM ammonium acetate, pH 6.1, adjusted with acetic acid. The column temperature was maintained at 50 °C. The LC gradient started with a constant flow rate of 0.6 mL/min with 12.3% B at 0 min, linear gradient of 12.3% B to 22.1% B for min 1–15. The column was equilibrated between runs by 12.3% B for 5 mins. The spray voltage was set to −4.5 kV, and the heated capillary and the HESI were held at 300 °C and 250 °C, respectively. MS parameters were as follows: - heated ESI mode of ionization, sheath gas = 40 units, aux gas = 5 units, sweep gas =1 units, spray voltage = static, negative ion =−4,500 V, negative ion = 2500 V, ion transfer tube temp= 300 °C, vaporizer temp=250 °C. Orbitrap resolution at MS1 was set to 120000, scan range 250–1800, RF lens = 60%, AGC target = standard, Orbitrap resolution at MS2 was set to 30000. EASY-IC™ was used for internal calibration. Analysis was performed in Excalibur (Thermo Scientific #OPTON-30965). The area under the curve was normalized to GM3-d5 standards, followed by normalization to amount of protein/wet weight of tissue used.

ProteomicsCellular lysates (50 μL) at 1 mg/mL were mixed with 50 μL of 8 M urea, 50 mM EPPS. The sample was then incubated with 5 mM TCEP for 30 min at 25 °C at 1000 rpm. Then it was incubated with 10 mM IAA during 30 min in the dark at RT before adding 5 mM DTT for 15 min at RT. The sample was digested with Lys-C with a 1:200 ratio enzyme:protein for 1 h at 25 °C at 1000 rpm. The sample was finally diluted half with 50 mM ammonium bicarbonate before digesting with trypsin with a 1:100 ratio protein:enzyme at 37 °C overnight. After the overnight incubation, the samples were desalted using C18 stage tips. Peptides were separated on a C18 reverse phase column (25-cm length, 75-μm diameter and 1.7-μm particle size from IonOptic Aurora 3 #1801220) using a gradient of 2–95% B over 90 min at 50 °C with a flow rate of 300 nL/min using a NanoElute 2 system (Bruker). Solvent A= 0.1% (w/v) formic acid in HPLC-grade water; solvent B = 99.5% (v/v) acetonitrile + 0.1% (v/v) formic acid. MS data were acquired on a timsToF (Bruker) with a captive spray source in data-independent acquisition mode. The mass range was set from 100 to 1700 m/z, the ion mobility range was set from 0.60 V.s/cm2 (collision energy 20eV) to 1.6 V.s/cm2 (collision energy 59 eV) with a ramp time of 100 ms and an accumulation time of 100 ms. Dia-PASEF settings were mass range 400.0 Da to 1201.0 Da, mobility range 0.60–1.60 and a cycle time estimate of 1.80 sec. The dia-PASEF windows were set with a mass width of 26.00 Da, mass overlap 1.00 Da, mass steps per cycle 32. Dia windows are as listed:
#MS TypeCycle IdStart IM [1/K0]End IM [1/K0]Start Mass [m/z]End Mass [m/z]CE [eV]MS10-----PASEF10.911.6800826-PASEF10.60.904400426-PASEF20.92361.6825851-PASEF20.60.9174425451-PASEF30.93721.6850876-PASEF30.60.9308450476-PASEF40.95071.6875901-PASEF40.60.9442475501-PASEF50.96431.6900926-PASEF50.60.9576500526-PASEF60.97791.6925951-PASEF60.60.971525551-PASEF70.99151.6950976-PASEF70.60.9844550576-PASEF81.00511.69751001-PASEF80.60.9979575601-PASEF91.01861.610001026-PASEF90.61.0113600626-PASEF101.03221.610251051-PASEF100.61.0247625651-PASEF111.04581.610501076-PASEF110.61.0381650676-PASEF121.05941.610751101-PASEF120.61.0515675701-PASEF131.0731.611001126-PASEF130.61.0649700726-PASEF141.08651.611251151-PASEF140.61.0783725751-PASEF151.10011.611501176-PASEF150.61.0918750776-PASEF161.11371.611751201-PASEF160.61.1052775801-

Cellular lysates (50 μL) at 1 mg/mL were mixed with 50 μL of 8 M urea, 50 mM EPPS. The sample was then incubated with 5 mM TCEP for 30 min at 25 °C at 1000 rpm. Then it was incubated with 10 mM IAA during 30 min in the dark at RT before adding 5 mM DTT for 15 min at RT. The sample was digested with Lys-C with a 1:200 ratio enzyme:protein for 1 h at 25 °C at 1000 rpm. The sample was finally diluted half with 50 mM ammonium bicarbonate before digesting with trypsin with a 1:100 ratio protein:enzyme at 37 °C overnight. After the overnight incubation, the samples were desalted using C18 stage tips. Peptides were separated on a C18 reverse phase column (25-cm length, 75-μm diameter and 1.7-μm particle size from IonOptic Aurora 3 #1801220) using a gradient of 2–95% B over 90 min at 50 °C with a flow rate of 300 nL/min using a NanoElute 2 system (Bruker). Solvent A= 0.1% (w/v) formic acid in HPLC-grade water; solvent B = 99.5% (v/v) acetonitrile + 0.1% (v/v) formic acid. MS data were acquired on a timsToF (Bruker) with a captive spray source in data-independent acquisition mode. The mass range was set from 100 to 1700 m/z, the ion mobility range was set from 0.60 V.s/cm2 (collision energy 20eV) to 1.6 V.s/cm2 (collision energy 59 eV) with a ramp time of 100 ms and an accumulation time of 100 ms. Dia-PASEF settings were mass range 400.0 Da to 1201.0 Da, mobility range 0.60–1.60 and a cycle time estimate of 1.80 sec. The dia-PASEF windows were set with a mass width of 26.00 Da, mass overlap 1.00 Da, mass steps per cycle 32. Dia windows are as listed:
#MS TypeCycle IdStart IM [1/K0]End IM [1/K0]Start Mass [m/z]End Mass [m/z]CE [eV]MS10-----PASEF10.911.6800826-PASEF10.60.904400426-PASEF20.92361.6825851-PASEF20.60.9174425451-PASEF30.93721.6850876-PASEF30.60.9308450476-PASEF40.95071.6875901-PASEF40.60.9442475501-PASEF50.96431.6900926-PASEF50.60.9576500526-PASEF60.97791.6925951-PASEF60.60.971525551-PASEF70.99151.6950976-PASEF70.60.9844550576-PASEF81.00511.69751001-PASEF80.60.9979575601-PASEF91.01861.610001026-PASEF90.61.0113600626-PASEF101.03221.610251051-PASEF100.61.0247625651-PASEF111.04581.610501076-PASEF110.61.0381650676-PASEF121.05941.610751101-PASEF120.61.0515675701-PASEF131.0731.611001126-PASEF130.61.0649700726-PASEF141.08651.611251151-PASEF140.61.0783725751-PASEF151.10011.611501176-PASEF150.61.0918750776-PASEF161.11371.611751201-PASEF160.61.1052775801-

Mitophagy induction and nucleic acid measurement in lysosomesMitochondrial DNA in the lysosomes was quantified as described previously.41 Briefly, lyso-tagged WT and PLD3 knock out HMC3 cells were treated with DMSO as vehicle or 50 μM Mdivi-I or with 5 μM S,S-BMP + 50 μM Mdivi-I for 24 hours. S,S-BMP treatment to cells started a passage before the Mdivi-1 treatment and continued till the cells were harvested. To increase yield of lysosomal DNA, cells from five 150 mm × 20 mm plates were pooled as one replicate and processed for lysosome isolation as described. Isolated lysosomes were processed through a genomic DNA extraction kit (ThermoFisher Scientific #K0512) and nucleic acids were eluted in 100 μL water. Isolated DNA was PCR amplified for 60 cycles in technical duplicates and quantified by melting curves as described in the RT-PCR section. Primers used were as follows:mtATP6-Forward AATCCAAGCCTACGTTTTCACAmtATP6-Reverse AGTATGAGGAGCGTTATGGAGTmtCO2-Forward AATCGAGTAGTACTCCCGATTGmtCO2-Reverse TTCTAGGACGATGGGCATGAAAmtND1-Forward CTATCACCCTATTAACCACTCAmtND1-Reverse TTCGCCTGTAATATTGAACGTAmt-DLoop-Forward CACCCAAGAACAGGGTTTGTmt-DLoop -Reverse TGGCCATGGGTATGTTGTTAA

Mitophagy induction and nucleic acid measurement in lysosomes

Mitochondrial DNA in the lysosomes was quantified as described previously.41 Briefly, lyso-tagged WT and PLD3 knock out HMC3 cells were treated with DMSO as vehicle or 50 μM Mdivi-I or with 5 μM S,S-BMP + 50 μM Mdivi-I for 24 hours. S,S-BMP treatment to cells started a passage before the Mdivi-1 treatment and continued till the cells were harvested. To increase yield of lysosomal DNA, cells from five 150 mm × 20 mm plates were pooled as one replicate and processed for lysosome isolation as described. Isolated lysosomes were processed through a genomic DNA extraction kit (ThermoFisher Scientific #K0512) and nucleic acids were eluted in 100 μL water. Isolated DNA was PCR amplified for 60 cycles in technical duplicates and quantified by melting curves as described in the RT-PCR section. Primers used were as follows:

mtATP6-Forward AATCCAAGCCTACGTTTTCACA

mtATP6-Reverse AGTATGAGGAGCGTTATGGAGT

mtCO2-Forward AATCGAGTAGTACTCCCGATTG

mtCO2-Reverse TTCTAGGACGATGGGCATGAAA

mtND1-Forward CTATCACCCTATTAACCACTCA

mtND1-Reverse TTCGCCTGTAATATTGAACGTA

mt-DLoop-Forward CACCCAAGAACAGGGTTTGT

mt-DLoop -Reverse TGGCCATGGGTATGTTGTTAA

QUANTIFICATION AND STATISTICAL ANALYSISAll data in the bar graphs are presented as mean ± standard deviation. N number in figure legends indicate biological replicates until otherwise mentioned. All statistical analysis were conducted using Prism 10 (GraphPad, San Diego, CA) or Matlab R2022b. Details of methods used to calculate statistical significance are mentioned in the figure legends.

QUANTIFICATION AND STATISTICAL ANALYSIS

All data in the bar graphs are presented as mean ± standard deviation. N number in figure legends indicate biological replicates until otherwise mentioned. All statistical analysis were conducted using Prism 10 (GraphPad, San Diego, CA) or Matlab R2022b. Details of methods used to calculate statistical significance are mentioned in the figure legends.

Supplementary MaterialMMC1MMC3MMC4MMC5MMC26

Supplementary Material
