---
pmid: '26458771'
title: Loss of Tifab, a del(5q) MDS gene, alters hematopoiesis through derepression
  of Toll-like receptor-TRAF6 signaling.
authors:
- Varney ME
- Niederkorn M
- Konno H
- Matsumura T
- Gohda J
- Yoshida N
- Akiyama T
- Christie S
- Fang J
- Miller D
- Jerez A
- Karsan A
- Maciejewski JP
- Meetei RA
- Inoue J
- Starczynowski DT
journal: J Exp Med
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4612089
doi: 10.1084/jem.20141898
---

# Loss of Tifab, a del(5q) MDS gene, alters hematopoiesis through derepression of Toll-like receptor-TRAF6 signaling.
**Authors:** Varney ME, Niederkorn M, Konno H, Matsumura T, Gohda J, Yoshida N, Akiyama T, Christie S, Fang J, Miller D, Jerez A, Karsan A, Maciejewski JP, Meetei RA, Inoue J, Starczynowski DT
**Journal:** J Exp Med (2015)
**DOI:** [10.1084/jem.20141898](https://doi.org/10.1084/jem.20141898)
**PMC:** [PMC4612089](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4612089/)

## Abstract

1. J Exp Med. 2015 Oct 19;212(11):1967-85. doi: 10.1084/jem.20141898. Epub 2015
Oct  12.

Loss of Tifab, a del(5q) MDS gene, alters hematopoiesis through derepression of 
Toll-like receptor-TRAF6 signaling.

Varney ME(1), Niederkorn M(2), Konno H(3), Matsumura T(3), Gohda J(3), Yoshida 
N(3), Akiyama T(3), Christie S(1), Fang J(1), Miller D(1), Jerez A(4), Karsan 
A(5), Maciejewski JP(4), Meetei RA(1), Inoue J(3), Starczynowski DT(6).

Author information:
(1)Division of Experimental Hematology and Cancer Biology, Cincinnati Children's 
Hospital Medical Center, Cincinnati, OH 45229.
(2)Division of Experimental Hematology and Cancer Biology, Cincinnati Children's 
Hospital Medical Center, Cincinnati, OH 45229 Department of Biological Sciences, 
University of Cincinnati, Cincinnati, OH 45267.
(3)Division of Cellular and Molecular Biology, Department of Cancer Biology and 
Laboratory of Developmental Genetics, Center for Experimental Medicine and 
Systems Biology, The Institute of Medical Science, the University of Tokyo, 
Shirokanedai, Minato-ku, Tokyo 108-8639, Japan.
(4)Department of Translational Hematology and Oncology Research, Taussig Cancer 
Institute, Cleveland Clinic, Cleveland, OH.
(5)Michael Smith Genome Sciences Centre and Department of Pathology and 
Laboratory Medicine, BC Cancer Research Centre, Vancouver, BC V5Z 1L3, Canada 
Michael Smith Genome Sciences Centre and Department of Pathology and Laboratory 
Medicine, BC Cancer Research Centre, Vancouver, BC V5Z 1L3, Canada.
(6)Division of Experimental Hematology and Cancer Biology, Cincinnati Children's 
Hospital Medical Center, Cincinnati, OH 45229 Department of Cancer Biology, 
University of Cincinnati College of Medicine, Cincinnati, OH 45267 
Daniel.Starczynowski@cchmc.org.

TRAF-interacting protein with forkhead-associated domain B (TIFAB) is a 
haploinsufficient gene in del(5q) myelodysplastic syndrome (MDS). Deletion of 
Tifab results in progressive bone marrow (BM) and blood defects, including 
skewed hematopoietic stem/progenitor cell (HSPC) proportions and altered myeloid 
differentiation. A subset of mice transplanted with Tifab knockout (KO) HSPCs 
develop a BM failure with neutrophil dysplasia and cytopenia. In competitive 
transplants, Tifab KO HSPCs are out-competed by wild-type (WT) cells, suggesting 
a cell-intrinsic defect. Gene expression analysis of Tifab KO HSPCs identified 
dysregulation of immune-related signatures, and hypersensitivity to TLR4 
stimulation. TIFAB forms a complex with TRAF6, a mediator of immune signaling, 
and reduces TRAF6 protein stability by a lysosome-dependent mechanism. In 
contrast, TIFAB loss increases TRAF6 protein and the dynamic range of TLR4 
signaling, contributing to ineffective hematopoiesis. Moreover, combined 
deletion of TIFAB and miR-146a, two genes associated with del(5q) MDS/AML, 
results in a cooperative increase in TRAF6 expression and hematopoietic 
dysfunction. Re-expression of TIFAB in del(5q) MDS/AML cells results in 
attenuated TLR4 signaling and reduced viability. These findings underscore the 
importance of efficient regulation of innate immune/TRAF6 signaling within HSPCs 
by TIFAB, and its cooperation with miR-146a as it relates to the pathogenesis of 
hematopoietic malignancies, such as del(5q) MDS/AML.

© 2015 Varney et al.

DOI: 10.1084/jem.20141898
PMCID: PMC4612089
PMID: 26458771 [Indexed for MEDLINE]

## Full Text

Myelodysplastic syndrome (MDS) refers to a group of hematopoietic stem cell (HSC) disorders associated with ineffective hematopoiesis, blood cytopenias, myeloid dysplasia, genomic instability, and predisposition to acute myeloid leukemia (AML) or bone marrow failure (BMF). Recent classifications of MDS describe eight subtypes based on biological, genetic, and morphological features (Cazzola and Malcovati, 2010). Independent of classification, MDS is propagated by rare and defective HSCs, and is defined by recurring cytogenetic changes and somatic point mutations. The most common cytogenetic alteration in MDS is deletion of chromosome (chr) 5q (del(5q)). In the absence of other cytogenetic alterations, MDS patients with a del(5q) exhibit refractory anemia, neutropenia, and elevated platelets associated with hypolobulated megakaryocytes (Giagounidis et al., 2006). Del(5q) is found in 25% of therapy-related MDS cases, which occur as a result of treatment with alkylating agents for unrelated conditions, and is strongly correlated with progression to AML (Haase et al., 2007; Haase, 2008; Qian et al., 2010). Although two commonly deleted regions (CDRs) have been mapped on chr 5q each spanning ∼1 Mb, a distal locus at 5q33.1, and a proximal locus at 5q31.1(Zhao et al., 1997), there are multiple genes that likely contribute to the pathogenesis of del(5q) MDS (Le Beau et al., 1989; Willman et al., 1993; Boultwood et al., 1994, 1997, 2000, 2002, 2007; Jaju et al., 1998). Clonal dominance of del(5q) cells is driven by haploid expression of CSNK1A1 (Schneider et al., 2014), whereas the erythroid defect has been attributed to RPS14 genes located within the distal CDR (Ebert et al., 2008; Barlow et al., 2010). Thrombocytosis associated with megakaryocytic dysplasia, neutropenia, and clonal dominance, are caused by loss of two miRNAs, miR-145 and miR-146a, in del(5q) MDS patients (Kumar et al., 2009; Starczynowski et al., 2010). Germline knockout of mouse miR-146a results in an early onset of myeloid expansion in the BM, and progression to more aggressive diseases such as lymphomas, BMF, and myeloid leukemia (Lu et al., 2010; Boldin et al., 2011; Zhao et al., 2011). Furthermore, overexpression of TRAF6, a miR-146a target gene, in mouse HSPCs mimics certain hematopoietic defects observed in miR-146a–deficient mice, including neutropenia, dysplasia, and myeloid leukemia. Overexpression of TRAF6, however, also results in elevated platelets. Some of the effects are mediated by IL-6, as overexpression of TRAF6 in Il6-deficient HSPCs restores platelets and neutrophil counts. DIAPH1, which encodes mDia1, is located on 5q31.3, which resides between the two commonly deleted regions in del(5q) MDS. mDia1-deficient mice exhibit an age-dependent granulocytopenia, and myeloid dysplasia in the BM, in part through increased TLR4 signaling. The proximal CDR at 5q31.1 is associated with aggressive forms of del(5q) and includes ∼12 coding genes. Two of these genes, CTNNA1 and EGR1, are thought to contribute to cell survival and myeloproliferation, respectively (Joslin et al., 2007; Liu et al., 2007). HSPA9 and PPP2CA have also been implicated in aspects of the del(5q) MDS/AML phenotype. Despite this recent progress, detailed molecular, cellular, and genetic analyses of candidate genes on chr 5q are required to completely understand the pathogenesis of del(5q) MDS/AML.

Derepression of TRAF6, as a result of miR-146a haploinsufficiency, is one molecular consequence of del(5q) (Starczynowski et al., 2010; Boldin et al., 2011). TRAF6 is an E3 ubiquitin ligase and signal transducer of the innate immune signaling pathway in response to pathogens and host damage-associated molecules (Wu and Arron, 2003). A search of annotated genes within or near the CDR in del(5q) revealed a relatively uncharacterized gene, TRAF-interacting protein with forkhead-associated domain B (TIFAB). TIFAB resides within the proximal CDR on band 5q31.1, and belongs to a family of forkhead-associated domain proteins that also includes TIFA. TIFA was originally identified as a TRAF6-interacting protein in a yeast two-hybrid screen (Kanamori et al., 2002; Takatsuna et al., 2003), whereas TIFAB was identified as a TIFA-related protein by an in silico homology screen (Matsumura et al., 2004, 2009). To investigate whether loss of TIFAB is important to the pathophysiology of del(5q) MDS/AML, in this study, we characterized a germline Tifab knockout (KO) mouse. Tifab−/− mice exhibit progressive hematopoietic defects, including skewed HSPC proportions, altered myeloid differentiation, and a BMF-like disease associated with BM dysplasia and cytopenia. Tifab−/− BM cells are hypersensitive to Toll-like receptor 4 (TLR4) stimulation, suggesting that loss of TIFAB alters the innate immune pathway. Independent of TRAF6 mRNA, TIFAB loss results in stabilization of TRAF6 protein. Moreover, combined deletion of TIFAB and miR-146a results in a cooperative increase in TRAF6 expression and hematopoietic dysfunction in vivo. This provides a potential molecular explanation for altered TLR4 sensitivity and the BMF phenotype. Collectively, our results provide evidence that deletion of TIFAB contributes to an MDS-like hematopoietic phenotype in mice by changing the dynamic range of the innate immune pathway in HSPCs through the regulation of TRAF6 protein stability.

RESULTSTIFAB resides within the minimally deleted region on chromosome 5q in MDS and AMLChronic innate immune signaling is associated with MDS HSPCs, partly due to deletion of miR-146a and DIAPH1. Given that the innate immune pathway is tightly regulated by negative feedback loops, we hypothesized that other genes on chr 5q cooperate to regulate this pathway. To identify such genes, we searched all annotated genes within or near the proximal and distal CDRs using the current assembly of the human genome (GRCh37/hg19), which revealed TIFAB (NM_001099221) on band 5q31.1 (chr5:134,784,560-134,788,089 bp; Fig. 1 A). The genomic location of TIFAB resides within the proximal CDR adjacent to CXCL14 (Fig. 1 A; Le Beau et al., 1989). Previous studies have delineated the boundaries of the 5q31.1 CDR. Although the smallest reported interval spans ∼1.5 Mb (from 136.25 – 137.80 Mb; Zhao et al., 1997), boundaries extending from IL-4 (132.00 Mb) to UBE2D2 (139.01 Mb) are more commonly reported (Fig. 1 A; Le Beau et al., 1989, 1993). Cytogenetic data from The Cancer Genome Atlas (TCGA) confirms that chr 5q deletions in human AML encompass the TIFAB locus (Fig. 1 A). According to these findings, TIFAB is one of the genes that is deleted in nearly all reported cases of the aggressive subtypes of del(5q) MDS and AML. To determine whether TIFAB is a relevant candidate gene within the 5q31.1 CDR, we first examined the normal expression of TIFAB mRNA in HSPC populations from humans and mice. TIFAB mRNA is expressed highest in human CD34+ cells and the equivalent mouse Lineage-negative (Lin−) BM cells (Fig. 1 B). To determine whether TIFAB mRNA is affected by del(5q) in MDS, we examined publically available gene expression data obtained from MDS CD34+ patient samples (Pellagatti et al., 2006) and by qRT-PCR in an independent cohort of MDS patient samples. TIFAB mRNA was significantly lower in CD34+ and BM mononuclear (MNC) cells isolated from MDS patients with del(5q) as compared with cells from MDS patients diploid at chr 5q (Fig. 1 C). Furthermore, TIFAB expression was significantly lower in del(5q) AML as compared with del(5q) MDS MNC (<60%; Fig. 1 D). These findings suggest that TIFAB may be a relevant gene in the pathogenesis of del(5q) MDS and/or AML.Figure 1.TIFAB deletion in del(5q) MDS/AML and mouse BM cells. (A) Schematic representation showing the genomic position of the TIFAB locus at chr 5q31.1. Below, blue/purple horizontal bars show extent of chr 5q deletions in human AML. (B) qPCR analysis of TIFAB mRNA in human CD34+ and CD34− (left), and mouse lineage positive (Lin+) and lineage negative (Lin−; right) BM cells. (C) TIFAB mRNA expression in normal CD34+ BM cells (n = 17), refractory anemia, and refractory anemia with excess blasts MDS CD34+ cells with del(5q) (n = 47) or diploid 5q status (n = 127) adapted from public gene expression studies (Pellagatti et al., 2006; left). TIFAB mRNA expression was independently verified in normal, MDS dip(5q) (n = 4), and del(5q) (n = 4) BM mononuclear cells (right). *, P < 0.05; Student’s t test. (D) TIFAB mRNA expression by qRT-PCR in MDS (n = 12) and AML (n = 4) mononuclear cells with del(5q). (E) Schematic representation of targeted Tifab allele. Exon 3, which contains the entire coding sequence, is deleted and replaced with a β-galactosidase and neomycin-resistant gene cassette floxed by loxP sites. The position of the 3′ probe for Southern blot is shown in blue. (F) 3′ Southern blot is shown to confirm germline deletion of Tifab. (G) Immunoblot analysis of BM lysates for TIFAB and tubulin protein expression from Tifab+/+ (WT), Tifab+/− (HET), and Tifab−/− (KO) mice. (H) Tifab mRNA was determined by Northern blot analysis (left) and by qRT-PCR (right). (I) BM isolated from Tifab WT, HET, and KO mice was transplanted into lethally irradiated syngeneic recipient BoyJ mice. PB counts from mice transplanted with Tifab WT, HET, and KO BM cells at the indicated time points (months; n > 9 per group) from three independent transplants. *, P < 0.05; Student’s t test. WBC, white blood cell; RBC, red blood cell; and PLT, platelet. (J) PB counts from mice transplanted with Tifab WT, HET, and KO BM cells at 6 mo after transplant (n > 9 per group) from three independent experiments. *, P < 0.05; Student’s t test. NE, neutrophils; LY, lymphocytes; and MO, monocytes. (K) BM cellularity was determined for two femurs and one tibia (2F1T) for WT (n = 3) and KO (n = 3) mice. (L) Wright-Giemsa–stained BM cytospins from representative mice transplanted with Tifab WT, HET, and KO BM cells. Percent neutrophil dysplasia is shown for below the images for WT (n = 3) and KO (n = 3) BM cells. Bars show BM cytospins at low (20 µm) and high (7.5 µm) magnification. Error bars are mean ± SEM values.Tifab deletion results in hematopoietic defects upon BM transplantationTo determine whether loss of TIFAB contributes to hematopoietic defects, we generated a constitutional knockout mouse model. The Tifab locus was targeted in embryonic stem cells with a construct containing LacZ in place of exon 3, which contains the entire coding sequence (Fig. 1 E). Germline deletion of the Tifab locus corresponded with loss of Tifab mRNA and protein (Fig. 1, F–H). Heterozygous (Tifab+/−) and homozygous (Tifab−/−) Tifab-deleted mice were born at expected Mendelian ratios and appeared physically normal. Peripheral blood (PB) counts at 6 and 12 mo of age were normal for all mice (unpublished data). Mature and immature BM cell populations of young or old Tifab-deficient (Tifab+/− and Tifab−/−) mice, as well as spleen sizes, were indistinguishable from WT (Tifab+/+) mice (unpublished data). However, a small subset of Tifab−/− and Tifab+/− mice, when aged beyond 1 yr, became moribund, presenting with enlarged spleens and evidence of BMF (cytopenia; unpublished data). Overall, these observations indicate that TIFAB is not required for steady-state hematopoiesis. To investigate the cell-autonomous function of Tifab-deficient HSPCs upon BM transplantation, BM cells from WT, Tifab+/−, and Tifab−/− mice were transplanted into lethally radiated syngeneic recipient mice. Mice receiving Tifab-deficient BM exhibited a progressive decline in peripheral white blood cell (WBC) counts (4.5 K/µl vs. 12.9 K/µl; P = 0.0014), without changes in red blood cell or platelet counts (Fig. 1 I). At 6 mo after transplant, mice transplanted with Tifab−/− BM cells exhibited significant reduction in neutrophils (0.99 K/µl vs. 1.62 K/µl; P < 0.0001), lymphocytes (4.93 K/µl vs. 8.18 K/µl; P < 0.001), and monocytes (0.38 K/µl vs. 0.61 K/µl; P = 0.0043; Fig. 1 J). Mice transplanted with Tifab−/− BM cells had a normal/hypocellular BM (Fig. 1 K); however, they exhibited dysplastic hypersegmented neutrophils, consistent with a granulocyte defect (10.8 ± 3.0% vs. 4.0 ± 1.6%; P = 0.027; Fig. 1 L).Mice transplanted with Tifab-deficient HSPCs develop a bone marrow failure-like diseaseGiven that mice transplanted with Tifab-deficient BM cells developed worsening cytopenias, the mice were aged beyond 1 yr to determine whether they progress to a hematologic disease. By 6 mo, a subset of mice transplanted with Tifab−/− BM cells became moribund, and within 16 mo, ∼40% of Tifab−/− mice (4/11 mice; P = 0.06), and 25% of Tifab+− mice (4/15 mice; P = 0.15) were moribund, whereas no control mice died (0/9; Fig. 2 A). Moribund Tifab-deficient mice had significantly lower WBC counts, including fewer neutrophils (0.95 K/µl vs. 3.85 K/µl; P = 0.013), monocytes (0.43 K/µl vs. 1.05 K/µl; P = 0.12), and lymphocytes (2.70 K/µl vs. 15.1 K/µl; P = 0.0001; Fig. 2 B). However, RBC counts, hematocrit, and platelet counts in moribund Tifab-deficient mice were comparable to age-matched control mice (Fig. 2 B). Moribund Tifab-deficient mice also had reduced BM cellularity (Fig. 2 C) and smaller spleens (Fig. 2 D) as compared with control mice, suggestive of declining hematopoietic function. Morphological examination of BM cells from moribund Tifab-deficient mice did not indicate consistent evidence of myeloid dysplasia (Fig. 2 C). PB, BM, and spleen analysis of moribund Tifab-deficient mice showed heterogeneous expansion or diminution of myeloid, lymphoid, and erythroid proportions, which also varied between moribund mice (Fig. 2, E–G). Despite the variation in cell proportions, moribund Tifab-deficient mice clearly displayed abnormal composition of mature myeloid, lymphoid, and erythroid cells in the PB and BM. Although PB cytopenia were observed in all mice at time of death, HSPC proportions of Tifab-deficient BM were maintained, suggesting, that the cytopenia was not simply a result of loss of HSCs (Fig. 2 H). Therefore, the progressive cytopenia and death of a subset of mice receiving Tifab-deficient BM cells suggests impaired HSPC function after loss of TIFAB.Figure 2.Transplantation of Tifab-deleted BM cells results in bone marrow failure. (A) Kaplan-Meier survival curves for transplanted mice reconstituted with Tifab WT (n = 9), HET (n = 15), or KO (n = 11) BM cells. Summary from three independent transplants. (B) PB counts from age-matched WT (n = 3) or moribund Tifab KO and HET (n = 4) mice. *, P < 0.05; Student’s t test. WBC, white blood cell; RBC, red blood cell; PLT, platelet; NE, neutrophils; LY, lymphocytes; and MO, monocytes. (C) H&E-stained femur and spleen, and Wright-Giemsa–stained BM cytospins from a representative moribund Tifab KO mouse. Bars: (BM) 100 µm; (cytospins) 20 µm; (spleen) 100 µm. (D) Representative spleens from age-matched WT and moribund Tifab KO mice. (E–G) Flow cytometric analysis of PB (E), BM (F), and spleen (G) cells isolated from a WT and representative moribund Tifab KO transplant mouse. (H) Flow cytometric analysis of HSPC proportions in the BM of Tifab WT (n = 3) and moribund KO (n = 2) transplanted mice. Error bars are mean ± SEM values. *, P < 0.05.TIFAB regulates HSPC function and myeloid differentiationBecause TIFAB deletion results in a progressive decline in hematopoietic function (Fig. 1) and in marrow failure (Fig. 2), we examined the consequences of TIFAB loss before overt disease. Although HSPC populations of nontransplanted Tifab-deficient (Tifab+/− and Tifab−/−) mice were similar to WT mice (Fig. 3 A), after BMT, mice receiving Tifab−/− BM cells exhibited increased long-term HSCs (LT-HSC; P < 0.0001), lineage−Sca1+cKit+ (LSK; P = 0.0036), and common myeloid progenitors (CMP; P = 0.018), whereas a reduction in megakaryocyte-erythroid progenitors was observed (MEP; P = 0.0058; Fig. 3 B). Although HPSC proportions significantly differed between WT and Tifab-deficient mice, the proportion of mature BM cells was similar between WT and Tifab−/− transplanted mice (Fig. 3 C). However, the production of Gr1+ myeloid cells in the PB was significantly increased in mice transplanted with Tifab−/− BM cells (P = 0.037; Fig. 3 D). To assess whether TIFAB is important for hematopoietic progenitor differentiation in vitro, LSK isolated from WT or Tifab-deficient (Tifab+/− and Tifab−/−) mice were examined for progenitor colony formation in methylcellulose. LSK from Tifab-deficient mice formed ∼50% more CFU-G colonies as compared with WT LSK (P = 0.009; Fig. 3 E), underscoring a functional defect in Tifab-deficient HSPCs and consistent with a role of TIFAB in myeloid differentiation.Figure 3.Tifab−/− BM cells exhibit altered hematopoietic proportions after transplantation. (A) LSK, LT-HSC (LSK+ CD150+CD48−), MEP, CMP, and GMP in BM cells from Tifab WT or KO mice (n = 7). (B) LSK, LT-HSC (LSK+ CD150+CD48−), MEP, CMP, and GMP in BM from mice transplanted with Tifab WT or KO BM cells from two independent experiments (n = 6). *, P < 0.05; Student’s t test. (C and D) BM (C) and PB (D) proportions from mice transplanted with Tifab WT or KO BM cells (n = 6) from three independent experiments. *, P < 0.05; Student’s t test. (E) Numbers of colony-forming cells in methylcellulose from LSK cells from Tifab WT, HET, or KO BM mice (n = 6; 2 independent BMT). *, P < 0.05; Student’s t test. GM, granulocyte-monocyte; G, granulocyte; M, monocyte; E, erythroid (BFU-E). (F) 3 × 105 BM isolated from Tifab WT, HET, and KO mice (CD45.2) were mixed with equal numbers of competitor BoyJ BM cells (CD45.1), and then transplanted into lethally radiated syngeneic recipient BoyJ mice (CD45.1). PB and BM chimerism was determined at 4 and 8 mo after transplantation. Calculated chimerism is based on the ratio of CD45.1 and CD45.2 mononuclear cells. Three independent experiments were performed. (G) Representative flow cytometric blots of PB chimerism at 8 mo after transplantation on myeloid (Gr1+CD11b−) and lymphoid (B220+) gated populations. (H and I) Flow cytometric analysis of myeloid (CD11b and Gr1), and lymphoid (CD3 and B220) donor-derived CD45.2+ proportions within the PB (H) and BM (I) of Tifab WT (n = 4), HET (n = 4), and KO (n = 5) competitively transplanted mice. *, P < 0.05; Student’s t test. (J) Flow cytometric analysis of HSPC donor-derived CD45.2+ proportions within the BM of Tifab WT (n = 4), HET (n = 4), and KO (n = 5) competitively transplanted mice at 8 mo. *, P < 0.05; Student’s t test and Mann-Whitney test. Error bars are mean ± SEM values.Competitive repopulation assays were performed to determine whether loss of Tifab affects HSPC function in vivo. Whole BM cells (3 × 105) from 8-wk-old CD45.2 WT, Tifab+/−, or Tifab−/− mice were transplanted with equal numbers of 8-wk-old CD45.1 competitor BM cells into lethally irradiated CD45.1 recipient mice. PB chimerism was evaluated for 50 wk and was calculated based on the proportion of CD45.2 mononuclear cells (CD45.2/[CD45.2+CD45.1]). Consistent with the persistent cytopenias after transplantation of Tifab-deficient BM cells, we observed a reduction in hematopoietic repopulation within the BM and PB, indicating that Tifab−/− HSPCs have a significant disadvantage in competitive transplantation and/or homing (Fig. 3 F). Lower chimerism of Tifab-deficient cells was caused by a reduction in lymphoid (CD3+ and B220+) and myeloid (CD11b+ and Gr1+) contribution to the PB (Fig. 3 G). To gain insight into the effects of TIFAB deletion on hematopoietic differentiation, donor-derived (CD45.2-positive) proportions were evaluated in the BM and PB from the competitive transplantation experiments. Tifab deficiency contributes to increased myeloid (CD11b+; P = 0.015) and reduced lymphoid (B220+ and CD3+; P = 0.004) donor-derived proportions in the PB (Fig. 3 H), with minor consequences on myeloid and lymphoid donor-derived proportions in the BM (Fig. 3 I). In addition, donor-derived Tifab−/− cells resulted in increased CMP (P = 0.03), and slightly elevated LT-HSCs (P = 0.09) and GMP (P = 0.08) proportions (Fig. 3 J). The hematopoietic differences within mice competitively transplanted with Tifab+/− BM cells were consistent by some measures with that of Tifab−/−-transplanted mice for CD11b-expansion and lymphoid-reduction in the PB (Fig. 3 H), and for expansion of the GMP compartment in the BM (Fig. 3 J). These findings suggest that Tifab deficiency affects myeloid/lymphoid-lineage differentiation and HSPC function, before development of overt hematologic disease consistent with BMF.TIFAB regulates immune and survival gene signatures in HSPCsTo understand the molecular basis for impaired HSPC function and BMF after TIFAB deletion, we performed an expression analysis on sorted LSK isolated from 3-mo-old mice transplanted with WT or Tifab−/− BM cells (n = 3 mice/group). We selected this time point to capture the gene expression profile of Tifab−/− LSK after engraftment but before overt hematopoietic failure. Complete blood counts before isolation of LSK cells confirmed absence of BMF in all mice. Analysis of the gene expression data identified differentially expressed genes in Tifab−/− LSK cells, including 32 up-regulated (>1.7-fold) and 22 down-regulated (<1.7-fold; Fig. 4 A and Table S1). To elucidate the relevant signaling pathway regulated after Tifab loss in HSPCs, we performed a gene set enrichment analysis (GSEA; Subramanian et al., 2005), which revealed overexpression of immune and infection response signatures, particularly related to interferon-regulated genes (Fig. 4 B). Down-regulated gene signatures in Tifab−/− LSK also included immune response signatures, but these were related to enterotoxin and LPS (Fig. 4, A and B). Interestingly, the immune-related gene signatures show a clear disparity between various immune insults, in which certain signatures (i.e., viral and interferon) are enriched, whereas other signatures (i.e., TNF and endo/enterotoxin) are down-regulated. Select genes from the microarray analysis that were overexpressed (Il6 and Bcl2a1) and down-regulated (Tnfaip3 and Sod2) in Tifab−/− LSK were confirmed by qRT-PCR (Fig. 4 C).Figure 4.TIFAB suppresses immune signaling and NF-κB activation downstream of the TLR4 receptor in BM HSPCs. (A) Heat map generated from gene set enrichment analysis (GSEA) showing gene expression differences in LSK isolated from 3-mo-old mice transplanted with Tifab WT or KO BM cells (n = 3 mice/group). (B) GSEA of statistically significant gene sets enriched in the LSK cells of transplanted Tifab−/− cells (up in KO) or WT cells (down in KO). (C) Validation of gene expression of the indicated genes in BM MNC by qRT-PCR. *, P < 0.05; Student’s t test. (D and E) Expression analysis of the indicated genes in BM MNC by qRT-PCR after stimulation with LPS (D; 100 ng/ml) or TNF (E; 10 ng/ml) for the indicated time points. Expression was normalized to time 0 (1.0). *, P < 0.05; Student’s t test.Although dysregulation of immune gene signatures is evident even in resting Tifab−/− HPSC, these findings prompted us to examine the role of TIFAB in regulating inducible gene expression. We selected two stimuli (TNF and LPS) that are associated with enriched GSEA signatures in Tifab-deficient HSPCs (Fig. 4 B). BM mononuclear cells (MNC) from WT and Tifab−/− mice were stimulated with TNF (10 ng/ml) or LPS (100 ng/ml) at various time points (0, 1, and 4 h) and examined for target gene expression by qRT-PCR. As expected, stimulation of WT BM cells with either LPS or TNF resulted in a time-dependent increase in Il6, Bcl2a1, Tnfaip3, and Sod2 expression (Fig. 4, D and E). Although expression of certain genes was higher (Il6 and Bcl2a1) or lower (Tnfaip3 and Sod2) in Tifab−/− than WT BM cells at basal levels, LPS stimulation resulted in a significant and acute increase of all genes examined in Tifab−/− BM cells (Fig. 4 D). In contrast, TNF stimulation of Tifab−/− BM cells resulted in a delayed response of Tnfaip3 and Sod2, and no dramatic differences of Il6 and Bcl2a1 expression (Fig. 4 E). These findings reveal that TIFAB controls the dynamic range of immune pathway activation downstream of TLRs, but not the TNF receptor (TNFR).TIFAB forms a complex with TRAF6 and inhibits NF-κBThe molecular function of TIFAB that restricts immune signaling in MDS/AML remains to be determined. To uncover the molecular function of TIFAB, we performed a global proteome analysis to identify TIFAB-associated cofactors in a del(5q) leukemic cell line. We generated stable HL60 lines expressing human TIFAB tagged with both FLAG and HA epitopes. TIFAB and its associated partners were purified from total cell lysates derived from HL60 cells using tandem affinity immunoprecipitation, as previously described (Singh et al., 2008; Fig. 5 A). Mass spectrometry (MS) analysis identified ∼300 unique protein partners of TIFAB. An MS-identified partner of TIFAB is TRAF6, a major component of the TLR signaling pathway and a functionally relevant protein in the pathogenesis of MDS/AML. Other TLR pathway molecules were also uncovered: SQSTM1/p62 and TIFA. To further investigate the TIFAB–TRAF6 interaction, HL60 cells expressing TIFAB were fractionated into cytoplasmic and nuclear fractions. The TIFAB–TRAF6 complex is formed predominantly within the cytoplasm (Fig. 5 B), which is consistent with the known role of TRAF6 signaling during TLR activation. TIFAB (double tagged with FLAG and HA) was also cotransfected with TRAF6 (tagged with FLAG) into HEK293 cells, and immunoprecipitated (anti-HA). In support of the MS analysis, immunoblotting of TRAF6 (anti-FLAG) confirmed that TIFAB co-immunoprecipitates with TRAF6 (Fig. 5 C).Figure 5.TIFAB binds to TRAF6 and suppresses NF-κB activation. (A) A representative silver stain after tandem affinity purification of FLAG-HA-TIFAB from HL60 cells. (B) TIFAB-expressing HL60 cells were lysed and fractionated into cytoplasmic (C) and nuclear (N) fractions, which were then subjected to coimmunoprecipitation of FLAG-HA-TIFAB. Immunoblotting for TRAF6, TIFAB, PARP (nuclear loading control), and GAPDH (cytosolic loading control) is shown. (C) Coimmunoprecipitation analysis of HEK293 transfected with empty vector, HA-TIFAB, FLAG-TRAF6, or HA-TIFAB and FLAG-TRAF6. Lysates were immunoprecipitated with anti-HA (TIFAB), and then blotted for FLAG (TRAF6). (D) Immunoblot analysis of THP1 and HL60 cells transduced with empty vector (Vec) or FLAG-TIFAB for phosphorylated IKKβ (pIKKβ), total IKKβ, and GAPDH. (E) Overview of TLR4 and TNFR-mediated activation of NF-κB, and the proposed regulation by TIFAB. (F–I) NF-κB activation was measured by κB-site containing reporter assays in HEK293 cells transfected with empty vector, TIFAB, and/or IKKβ (F), p65 (G), TRAF6 (H), or TRAF2 (I). Values are normalized to Renilla-luciferase and empty vector (1.0). Summary of three independent experiments. Error bars are mean ± SEM values. *, P < 0.05; Student’s t test.Stimulation of TLRs mediates signals to several transcription factors, one of which is the NF-κB family. Because TIFAB expression is low in THP1 and HL60 cells, whereas NF-κB activation is high, we first determined the consequences of TIFAB reexpression on NF-κB signaling in these relevant human cells. THP1 and HL60 cells were transduced with a vector encoding TIFAB or control (empty vector), and examined for phosphorylated IKKβ (pIKKβ), an indication of NF-κB activation. Overexpression of TIFAB resulted in significant reduction in pIKKβ, indicating that TIFAB reexpression is sufficient to inhibit NF-κB (Fig. 5 D). Because TLR and TNF receptor 2 (TNFR2) use different signaling mediators to activate NF-κB (Fig. 5 E), we wanted to determine the precise step at which TIFAB regulates receptor-induced NF-κB signaling. To examine this mechanism, we performed κB-site containing reporter assays transfected with various activators of NF-κB (IKKβ, RelA/p65) and ones downstream of TLR4 (TRAF6) or TNFR2 (TRAF2; Fig. 5 E). Co-transfection of TIFAB did not suppress IKKβ- (Fig. 5 F) or p65-mediated (Fig. 5 G) κB-site luciferase activity, indicating that TIFAB inhibits NF-κB signaling upstream of the NF-κB kinase complex (IKK) and its transcription factor (RelA/p65). However, co-transfection of TIFAB significantly repressed κB-site luciferase activity by TRAF6 (Fig. 5 H). In contrast, TIFAB did not repress κB-site activation after transfection of TRAF2, a functionally related paralog of TRAF6 (Fig. 5 I). TIFAB represses TRAF6-, but not TRAF2-mediated NF-κB activation, implying that TIFAB selectively inhibits TLR/TRAF6-dependent NF-κB stimuli while having no effect on TNFR2/TRAF2-dependent NF-κB stimuli.TIFAB restricts TLR activation by destabilizing TRAF6 proteinTo investigate the mechanism by which TIFAB inhibits TRAF6-mediated signaling, we overexpressed TIFAB in HEK293 or HL60 cells. We observed reduced levels of transfected (HEK293: 60% decrease) and endogenous (HL60: 40% decrease) TRAF6 protein (Fig. 6 A). Moreover, expression of TIFAB reduced levels of cotransfected TRAF6, whereas the levels of TRAF2 were not affected (Fig. 6 B). Conversely, THP1 cells after knockdown of TIFAB (Fig. 6 C), as well as Tifab−/− BM cells after BM transplantation (Fig. 6 D), exhibited increased expression of TRAF6 protein. The effects of TIFAB on TRAF6 protein are independent of TRAF6 mRNA, as overexpression of TIFAB did not reduce TRAF6 mRNA in HEK293 cells (Fig. 6 E), nor did Tifab deficiency increase Traf6 mRNA in BM cells (Fig. 6 F). To explore the possibility that TIFAB induces TRAF6 protein degradation, we used selective inhibitors of the proteasome (MG-132) or lysosome (3-MA). Treatment of HEK293 cells expressing TRAF6 and TIFAB with MG-132 did not restore TRAF6 protein levels (Fig. 6 G), indicating that TIFAB-mediated degradation of TRAF6 does not occur through the ubiquitin–proteasome pathway (Fig. 6 H). However, treatment with 3-MA partially restored TRAF6 protein expression in TIFAB-expressing HEK293 cells (Fig. 6 G), suggesting that TIFAB-mediated degradation may occur through the lysosome–autophagy pathway (Fig. 6 H).Figure 6.TIFAB inhibits TLR4 signaling by inducing degradation of TRAF6 protein via lysosomes. (A) Immunoblot analysis of HEK293 cells transfected with empty vector, TRAF6, or FLAG-TIFAB (left), and HL60 cells transduced with empty vector (pMSCV-pGK-GFP) or FLAG-TIFAB (right). Images are representative from at least three independent experiments. (B) Immunoblot analysis of HEK293 transfected with empty vector, TIFAB, and/or TRAF6 (left) or TRAF2 (right). Images are representative from at least two independent experiments. (C) Immunoblot analysis of human THP1 cells transduced with scrambled control shRNA (shCTL) or an shRNA targeting human TIFAB (shTIFAB). (D) Immunoblot analysis for TRAF6 on Tifab WT or KO BM cells from independent pre and post-transplanted mice. (E) Expression analysis of TRAF6 mRNA by qRT-PCR in HEK293 cells transfected with empty vector, TRAF6, and/or FLAG-TIFAB. (F) Expression analysis of Tifab or Traf6 mRNA by qRT-PCR in Tifab WT and KO BM cells. (G) Immunoblot analysis of HEK293 transfected with empty vector, TIFAB, and/or TRAF6, and then treated with a proteasome inhibitor (MG132, 10 µM for 12 h) or a lysosome inhibitor (3-MA, 5 mM for 24 h). Images are representative from at least three independent experiments. (H) HEK293 cells were treated with MG132 or 3-MA as in G, and immunoblotted for total ubiquitin (Ub) and LC3B, respectively. (I) Expression analysis of Traf6 mRNA in Tifab WT or KO-transduced BM MNC expressing a scrambled control shRNA (shCTL) or an shRNA targeting mouse Traf6 (shTraf6). (J and K) QRT-PCR expression analysis of Tnfaip3 (J) and Sod2 (K) in Tifab WT or KO BM MNC expressing a scrambled control shRNA (shCTL) or an shRNA-targeting mouse Traf6 (shTraf6) after stimulation with LPS (100 ng/ml) for 4 h. Data are summarized from biological and technical replicates. Error bars are mean ± SEM values. *, P < 0.05; Student’s t test.TRAF6 is required for the hyperimmune response in Tifab-deficient hematopoietic cellsTo establish whether Tifab-deficient BM cells exhibit increased LPS-response through TRAF6, rescue experiments were performed by depleting Traf6 in Tifab−/− BM cells. WT and Tifab−/− BM cells were transduced with shTraf6 (shT6) or control shRNA (shCTL; Fig. 6 I), and then stimulated with LPS for up to 4 h. As in Fig. 4 D, LPS stimulation of WT BM cells expressing the control shRNA exhibited an increase in Tnfaip3 and Sod2 expression, whereas Tifab−/− BM stimulated with LPS exhibited a further increase in both Tnfaip3 and Sod2 levels (Fig. 6, J and K). Knockdown of Traf6 in both WT and Tifab−/− BM cells completely abolished LPS-induced expression of Tnfaip3 and Sod2 (Fig. 6, J and K). Collectively, these findings have uncovered that TIFAB controls TLR signaling by binding and regulating TRAF6 protein stability through lysosomal degradation, but does not affect TNFR–TRAF2 signaling.TIFAB and miR-146a cooperate to regulate TRAF6 expression and myeloid cell differentiationHuman miR-146a and TIFAB both reside on chr 5q and are co-deleted in ∼80% of del(5q) MDS and AML (Gondek et al., 2008). miR-146a−/− mice develop BMF and myeloid leukemia, in part because of derepression of TRAF6 mRNA and protein (Lu et al., 2010; Boldin et al., 2011; Zhao et al., 2011). Given that TIFAB and miR-146a are codeleted in MDS/AML, we examined whether there is a functional cooperation between TIFAB and miR-146a. miR-146a inhibits TRAF6 protein expression and reduces its mRNA levels, whereas herein we report that TIFAB inhibits TRAF6 by destabilizing its protein. Because miR-146a is an NF-κB target gene responsive to LPS, miR-146a expression was investigated in TIFAB-deficient BM cells. At basal levels and after LPS stimulation, miR-146a expression is significantly elevated in Tifab−/− BM MNC (2-fold and 15-fold at baseline and after stimulation, respectively; Fig. 7 A). Although TRAF6 protein is stabilized after BM transplantation in Tifab−/− BM cells (Fig. 6 D), Traf6 mRNA is significantly reduced in Tifab−/− BM cells as compared with WT cells (Fig. 6 F and Fig. 7 B), suggesting that increased miR-146a expression in Tifab-deficient cells may suppress TRAF6 mRNA. To examine a cooperative effect of TIFAB and miR-146a deficiency on TRAF6 mRNA and protein expression, we generated double Tifab- and miR-146a–deficient mice. Combined deletion of Tifab and miR-146a restored TRAF6 mRNA expression to WT levels (Fig. 7 B). In addition, TRAF6 protein expression was synergistically increased in BM MNC from Tifab−/−;miR-146a−/− mice (dKO) after BM transplantation as compared with single-knockout BM cells (Fig. 7 C). Collectively, miR-146a and TIFAB cooperate to suppress TRAF6 mRNA and protein expression in BM HSPCs.Figure 7.TIFAB and miR-146a cooperate to regulate TRAF6 and hematopoietic function. (A) Expression analysis of miR-146a in WT and KO BM MNC by qRT-PCR after stimulation with LPS (100 ng/ml) for the indicated time points. Expression was normalized to time 0 (1.0). *, P < 0.05; Student’s t test. (B) Expression analysis of Traf6 mRNA by qRT-PCR in WT, Tifab KO, miR-146a KO, and double KO (dKO) BM cells. *, P < 0.05; Student’s t test. (C) Immunoblot analysis for TRAF6 on WT, Tifab KO, miR-146a KO, and dKO BM cells from transplanted mice. (D) Total colony-forming cells in methylcellulose from BM MNC from WT, Tifab KO, miR-146a KO, and dKO BM (n = 5–6; two independent experiments). *, P < 0.05; Student’s t test. (E) Proportion of colonies formed from D. GM, granulocyte-monocyte; G, granulocyte; M, monocyte; E, erythroid (BFU-E). **, P < 0.01. (F) PB counts from mice transplanted with WT, Tifab KO, miR-146a KO, and dKO BM cells at 4 mo after transplant (n > 8 per group). Two independent transplantations experiments were performed. NE, neutrophils; LY, lymphocytes; and RBC, red blood cells. Error bars are mean ± SEM values. *, P < 0.05; **, P < 0.01; ***, P < 0.001, Student’s t test.To assess whether TIFAB and miR-146a cooperate to regulate hematopoietic progenitor differentiation in vitro, BM HSPCs isolated from WT, Tifab−/−, miR-146a−/−, or dKO mice were examined for progenitor colony formation in methylcellulose. Total colony formation was increased in dKO as compared with the single KO mice (Fig. 7 D). In addition, combined deletion of TIFAB and miR-146a resulted in an additive increase in CFU-G colony formation (P = 0.007), underscoring a functional cooperation between TIFAB and miR-146a during myeloid differentiation (Fig. 7 E). To determine whether combined deletion of TIFAB and miR-146a accentuates the cytopenia in vivo, BM cells from WT, Tifab−/−, miR-146−/−, and dKO mice were transplanted into lethally radiated syngeneic recipient mice. Mice reconstituted with Tifab-deficient BM exhibited a decline in peripheral neutrophils (2.1 vs. 3.0 K/µl; P = 0.014), lymphocytes (10.5 vs. 13.0 K/µl; P < 0.006), and RBCs (9.5 vs. 10.5 K/µl; P = 0.01) by 4 mo. Deletion of miR-146a had only minor effects on neutrophil (P = 0.08), lymphocyte (P = 0.45), or RBC (P = 0.012) counts (Fig. 7 F). In contrast, combined deletion of TIFAB and miR-146a resulted in a cooperative decrease in peripheral neutrophil (1.3 vs. 3.0 K/µl; P < 0.0001) and lymphocyte (7.5 vs. 13.0 K/µl; P < 0.0001) production, while not contributing further to RBC defects during the same time period (Fig. 7 F). Collectively, these findings suggest that two 5q genes, miR-146a and TIFAB, cooperate to regulate TRAF6 expression, whereas loss of both genes accentuates hematopoietic myeloid differentiation defects in vitro and in vivo.Regulation of TRAF6 is dependent on the C-terminal domain of TIFABTIFAB is a 131 aa protein consisting of a conserved 55-aa forkhead-associated (FHA) domain (aa 36–91), whereas the residues flanking the FHA domain do not contain conserved motifs. To determine whether the FHA domain is required for regulating TRAF6 protein stability and signaling, we created deletion mutants of TIFAB. Three key mutants were created that removed the FHA domain (Δ35–91), the N-terminal domain (Δ1–34), or the C-terminal domain (Δ92–161). Additional TIFAB mutants were generated that deleted a combination of the aforementioned domains (Δ1–91 and Δ35–161; Fig. 8 A). The effects of TIFAB mutants on basal NF-κB activation were measured by κB-site reporter assays. Overexpression of TIFAB repressed basal κB-luciferase activation by ∼30% (P = 0.01; Fig. 8 B). Of the TIFAB deletion mutants, only Δ1–91 mutant retained its ability to repress κB-luciferase activation (P = 1.7 × 10−4; Fig. 8 B). We next evaluated the effects of the TIFAB mutants on TRAF6-mediated NF-κB activation. Overexpression of TRAF6 results in ∼40-fold increase in κB-luciferase activation (Fig. 8 C). Co-expression of full-length TIFAB resulted in >90-fold repression of TRAF6-mediated κB-luciferase activation (P < 0.01; Fig. 8 C). Surprisingly, deletion of the FHA domain (Δ35–91) still repressed TRAF6-mediated κB-luciferase activation and reduced TRAF6 protein levels (Fig. 8, C and D), suggesting that the FHA domain is not essential for inhibiting TRAF6-induced NF-κB signaling. Consistent with the effects on basal NF-κB activation, the Δ1–91 mutant also inhibited TRAF6-mediated NF-κB activation (P = 0.002; Fig. 8 C). To identify the essential residues within aa 92–161 of the Δ1–90 mutants that are responsible for inhibiting TRAF6-mediated NF-κB activation, we created mutants that eliminated a portion of the C-terminal domain; Δ126–161 is missing the distal region of the C-terminal domain, and Δ91-126 is missing the proximal region of the C-terminal domain (Fig. 8 A). Both mutants retained their inhibitory effects on NF-κB, suggesting that multiple regions within this domain are essential for TIFAB function. In parallel, we also examined the effects of the TIFAB mutants on TRAF6 protein stability. Congruent with the NF-κB reporter assays, TIFAB mutant Δ1–91 suppressed TRAF6 protein nearly as efficiently as full-length TIFAB, strongly indicating that <70 aa within the C-terminal domain of TIFAB elicit TRAF6 degradation (Fig. 8, A and D).Figure 8.Inhibition of TRAF6 and NF-κB signaling depends on the TIFAB C-terminal domain and is independent of the FHA domain. (A) Overview of WT TIFAB and various TIFAB deletion mutants. Summary of NF-κB activation in HEK293 cells after expression of TIFAB mutants under basal (B) and TRAF6-induced conditions (C). Summary of TRAF6 protein expression in HEK293 cells after expression of TIFAB mutants (D). FHA, fork-head associated domain; FLAG, FLAG motif cloned at the N-terminal region of TIFAB mutants; nd, not determined. −, repression; +, normal levels; +/−, moderate repression. (B) NF-κB activation was measured by a κB-site containing reporter assays in HEK293 cells transfected with empty vector, TIFAB, or the indicated TIFAB deletion mutants. Values are normalized to Renilla-luciferase and empty vector (1.0). *, P < 0.05; Student’s t test. (C) NF-κB activation was measured by κB-site containing reporter assays in HEK293 cells transfected with empty vector, TRAF6, and TIFAB or the indicated TIFAB deletion mutants. Values are normalized to Renilla-luciferase and empty vector (1.0). *, P < 0.05; **, P < 0.01; ***, P < 0.001; Student’s t test. (D) Immunoblot analysis of HEK293 transfected with empty vector, TRAF6, and TIFAB or TIFAB deletion mutants. Error bars are mean ± SEM values.Restoring TIFAB expression in MDS/AML cells induces apoptosis and impairs leukemic functionTo establish the relevance of TIFAB in MDS and AML, TIFAB was reexpressed in three human MDS/AML cell lines (MDSL, HL60, and THP1) with low endogenous TIFAB, and assessed for leukemic progenitor function and survival. The level of TIFAB reexpression in the indicated cell lines was consistent with physiological TIFAB expression in primary normal and del(5q) AML BM MNC (Fig. 9 A). Ectopic expression of TIFAB in all three cell lines significantly reduced leukemic colony formation (Fig. 9 B) and survival (Fig. 9, C and D). To examine the cellular effects of restoring just the minimal TIFAB domain that retains TRAF6 inhibition in disease-relevant cells, we transduced the three MDS/AML cell lines with Δ1-91–expressing vectors. As a negative control, Δ92–161 was also expressed, as this mutant has lost its TRAF6/NF-κB inhibitory function, yet it still retains the FHA domain (Fig. 8 A). Restoring only the C-terminal domain (Δ1–91) suppressed MDS/AML leukemic progenitor function at a level comparable to WT TIFAB (Fig. 9 B). In stark contrast, Δ92–161 expression did not inhibit MDS/AML cell colony formation. Thus, restoring only the C-terminal domain of TIFAB is sufficient to destabilize TRAF6 protein, suppress NF-κB activation, and impair MDS/AML cell function.Figure 9.TIFAB reexpression suppresses MDS/AML cells. (A) Relative TIFAB mRNA expression determined by qRT-PCR from normal and del(5q) AML patient BM MNC, and transduced cell lines with empty vector or FLAG-TIFAB. (B) Colony formation in methylcellulose of MDSL, HL60, and THP1 cells transduced with empty vector, FLAG-TIFAB, FLAG-TIFABΔ1-91, FLAG-TIFABΔ92-161 was measured after 7–9 d. *, P < 0.05; **, P < 0.01; Student’s t test. (C) Representative Annexin V staining for MDSL, HL60, and THP1 cells transduced with empty vector or FLAG-TIFAB (top). (D) The corresponding summary for Annexin V staining is shown for more than two independent experiments (bottom). *, P < 0.05; Student’s t test. (E) MDSL cells transduced with vector or TIFAB were engrafted into NSGS mice (n = 6/group) and monitored for BM chimerism at the indicated time points. BM engraftment is shown for individual mice transplanted with vector (#1 and #3)-MDSL and TIFAB (#7 and #8)-MDSL cells at 4 and 8–12 wk (human versus mouse CD45). Percent of human cell (MDSL) engraftment is shown for the representative mice. (F) Summary of BM engraftment for mice transplanted with vector- and TIFAB-MDSL cells (n > 5 mice per group). *, P < 0.05; Student’s t test.To examine the effects of TIFAB reexpression on MDS/AML cell function in vivo, MDSL cells were transduced with TIFAB or empty vector (vector) and then xenografted into NSGS mice. Vector- and TIFAB-expressing MDSL cells efficiently engrafted into the BM of recipient mice within 4 wk (Fig. 9, E and F). However, beyond 8 wk after transplant, the BM engraftment of TIFAB-expressing MDSL cells dramatically diminished (P = 0.04), whereas control MDSL cells increased their engraftment during this time (Fig. 9, E and F). Thus, restoring TIFAB expression in MDS/AML cells induces apoptosis and impairs disease-propagating function in vitro and in vivo.

TIFAB resides within the minimally deleted region on chromosome 5q in MDS and AMLChronic innate immune signaling is associated with MDS HSPCs, partly due to deletion of miR-146a and DIAPH1. Given that the innate immune pathway is tightly regulated by negative feedback loops, we hypothesized that other genes on chr 5q cooperate to regulate this pathway. To identify such genes, we searched all annotated genes within or near the proximal and distal CDRs using the current assembly of the human genome (GRCh37/hg19), which revealed TIFAB (NM_001099221) on band 5q31.1 (chr5:134,784,560-134,788,089 bp; Fig. 1 A). The genomic location of TIFAB resides within the proximal CDR adjacent to CXCL14 (Fig. 1 A; Le Beau et al., 1989). Previous studies have delineated the boundaries of the 5q31.1 CDR. Although the smallest reported interval spans ∼1.5 Mb (from 136.25 – 137.80 Mb; Zhao et al., 1997), boundaries extending from IL-4 (132.00 Mb) to UBE2D2 (139.01 Mb) are more commonly reported (Fig. 1 A; Le Beau et al., 1989, 1993). Cytogenetic data from The Cancer Genome Atlas (TCGA) confirms that chr 5q deletions in human AML encompass the TIFAB locus (Fig. 1 A). According to these findings, TIFAB is one of the genes that is deleted in nearly all reported cases of the aggressive subtypes of del(5q) MDS and AML. To determine whether TIFAB is a relevant candidate gene within the 5q31.1 CDR, we first examined the normal expression of TIFAB mRNA in HSPC populations from humans and mice. TIFAB mRNA is expressed highest in human CD34+ cells and the equivalent mouse Lineage-negative (Lin−) BM cells (Fig. 1 B). To determine whether TIFAB mRNA is affected by del(5q) in MDS, we examined publically available gene expression data obtained from MDS CD34+ patient samples (Pellagatti et al., 2006) and by qRT-PCR in an independent cohort of MDS patient samples. TIFAB mRNA was significantly lower in CD34+ and BM mononuclear (MNC) cells isolated from MDS patients with del(5q) as compared with cells from MDS patients diploid at chr 5q (Fig. 1 C). Furthermore, TIFAB expression was significantly lower in del(5q) AML as compared with del(5q) MDS MNC (<60%; Fig. 1 D). These findings suggest that TIFAB may be a relevant gene in the pathogenesis of del(5q) MDS and/or AML.Figure 1.TIFAB deletion in del(5q) MDS/AML and mouse BM cells. (A) Schematic representation showing the genomic position of the TIFAB locus at chr 5q31.1. Below, blue/purple horizontal bars show extent of chr 5q deletions in human AML. (B) qPCR analysis of TIFAB mRNA in human CD34+ and CD34− (left), and mouse lineage positive (Lin+) and lineage negative (Lin−; right) BM cells. (C) TIFAB mRNA expression in normal CD34+ BM cells (n = 17), refractory anemia, and refractory anemia with excess blasts MDS CD34+ cells with del(5q) (n = 47) or diploid 5q status (n = 127) adapted from public gene expression studies (Pellagatti et al., 2006; left). TIFAB mRNA expression was independently verified in normal, MDS dip(5q) (n = 4), and del(5q) (n = 4) BM mononuclear cells (right). *, P < 0.05; Student’s t test. (D) TIFAB mRNA expression by qRT-PCR in MDS (n = 12) and AML (n = 4) mononuclear cells with del(5q). (E) Schematic representation of targeted Tifab allele. Exon 3, which contains the entire coding sequence, is deleted and replaced with a β-galactosidase and neomycin-resistant gene cassette floxed by loxP sites. The position of the 3′ probe for Southern blot is shown in blue. (F) 3′ Southern blot is shown to confirm germline deletion of Tifab. (G) Immunoblot analysis of BM lysates for TIFAB and tubulin protein expression from Tifab+/+ (WT), Tifab+/− (HET), and Tifab−/− (KO) mice. (H) Tifab mRNA was determined by Northern blot analysis (left) and by qRT-PCR (right). (I) BM isolated from Tifab WT, HET, and KO mice was transplanted into lethally irradiated syngeneic recipient BoyJ mice. PB counts from mice transplanted with Tifab WT, HET, and KO BM cells at the indicated time points (months; n > 9 per group) from three independent transplants. *, P < 0.05; Student’s t test. WBC, white blood cell; RBC, red blood cell; and PLT, platelet. (J) PB counts from mice transplanted with Tifab WT, HET, and KO BM cells at 6 mo after transplant (n > 9 per group) from three independent experiments. *, P < 0.05; Student’s t test. NE, neutrophils; LY, lymphocytes; and MO, monocytes. (K) BM cellularity was determined for two femurs and one tibia (2F1T) for WT (n = 3) and KO (n = 3) mice. (L) Wright-Giemsa–stained BM cytospins from representative mice transplanted with Tifab WT, HET, and KO BM cells. Percent neutrophil dysplasia is shown for below the images for WT (n = 3) and KO (n = 3) BM cells. Bars show BM cytospins at low (20 µm) and high (7.5 µm) magnification. Error bars are mean ± SEM values.

TIFAB resides within the minimally deleted region on chromosome 5q in MDS and AML

Chronic innate immune signaling is associated with MDS HSPCs, partly due to deletion of miR-146a and DIAPH1. Given that the innate immune pathway is tightly regulated by negative feedback loops, we hypothesized that other genes on chr 5q cooperate to regulate this pathway. To identify such genes, we searched all annotated genes within or near the proximal and distal CDRs using the current assembly of the human genome (GRCh37/hg19), which revealed TIFAB (NM_001099221) on band 5q31.1 (chr5:134,784,560-134,788,089 bp; Fig. 1 A). The genomic location of TIFAB resides within the proximal CDR adjacent to CXCL14 (Fig. 1 A; Le Beau et al., 1989). Previous studies have delineated the boundaries of the 5q31.1 CDR. Although the smallest reported interval spans ∼1.5 Mb (from 136.25 – 137.80 Mb; Zhao et al., 1997), boundaries extending from IL-4 (132.00 Mb) to UBE2D2 (139.01 Mb) are more commonly reported (Fig. 1 A; Le Beau et al., 1989, 1993). Cytogenetic data from The Cancer Genome Atlas (TCGA) confirms that chr 5q deletions in human AML encompass the TIFAB locus (Fig. 1 A). According to these findings, TIFAB is one of the genes that is deleted in nearly all reported cases of the aggressive subtypes of del(5q) MDS and AML. To determine whether TIFAB is a relevant candidate gene within the 5q31.1 CDR, we first examined the normal expression of TIFAB mRNA in HSPC populations from humans and mice. TIFAB mRNA is expressed highest in human CD34+ cells and the equivalent mouse Lineage-negative (Lin−) BM cells (Fig. 1 B). To determine whether TIFAB mRNA is affected by del(5q) in MDS, we examined publically available gene expression data obtained from MDS CD34+ patient samples (Pellagatti et al., 2006) and by qRT-PCR in an independent cohort of MDS patient samples. TIFAB mRNA was significantly lower in CD34+ and BM mononuclear (MNC) cells isolated from MDS patients with del(5q) as compared with cells from MDS patients diploid at chr 5q (Fig. 1 C). Furthermore, TIFAB expression was significantly lower in del(5q) AML as compared with del(5q) MDS MNC (<60%; Fig. 1 D). These findings suggest that TIFAB may be a relevant gene in the pathogenesis of del(5q) MDS and/or AML.

TIFAB deletion in del(5q) MDS/AML and mouse BM cells. (A) Schematic representation showing the genomic position of the TIFAB locus at chr 5q31.1. Below, blue/purple horizontal bars show extent of chr 5q deletions in human AML. (B) qPCR analysis of TIFAB mRNA in human CD34+ and CD34− (left), and mouse lineage positive (Lin+) and lineage negative (Lin−; right) BM cells. (C) TIFAB mRNA expression in normal CD34+ BM cells (n = 17), refractory anemia, and refractory anemia with excess blasts MDS CD34+ cells with del(5q) (n = 47) or diploid 5q status (n = 127) adapted from public gene expression studies (Pellagatti et al., 2006; left). TIFAB mRNA expression was independently verified in normal, MDS dip(5q) (n = 4), and del(5q) (n = 4) BM mononuclear cells (right). *, P < 0.05; Student’s t test. (D) TIFAB mRNA expression by qRT-PCR in MDS (n = 12) and AML (n = 4) mononuclear cells with del(5q). (E) Schematic representation of targeted Tifab allele. Exon 3, which contains the entire coding sequence, is deleted and replaced with a β-galactosidase and neomycin-resistant gene cassette floxed by loxP sites. The position of the 3′ probe for Southern blot is shown in blue. (F) 3′ Southern blot is shown to confirm germline deletion of Tifab. (G) Immunoblot analysis of BM lysates for TIFAB and tubulin protein expression from Tifab+/+ (WT), Tifab+/− (HET), and Tifab−/− (KO) mice. (H) Tifab mRNA was determined by Northern blot analysis (left) and by qRT-PCR (right). (I) BM isolated from Tifab WT, HET, and KO mice was transplanted into lethally irradiated syngeneic recipient BoyJ mice. PB counts from mice transplanted with Tifab WT, HET, and KO BM cells at the indicated time points (months; n > 9 per group) from three independent transplants. *, P < 0.05; Student’s t test. WBC, white blood cell; RBC, red blood cell; and PLT, platelet. (J) PB counts from mice transplanted with Tifab WT, HET, and KO BM cells at 6 mo after transplant (n > 9 per group) from three independent experiments. *, P < 0.05; Student’s t test. NE, neutrophils; LY, lymphocytes; and MO, monocytes. (K) BM cellularity was determined for two femurs and one tibia (2F1T) for WT (n = 3) and KO (n = 3) mice. (L) Wright-Giemsa–stained BM cytospins from representative mice transplanted with Tifab WT, HET, and KO BM cells. Percent neutrophil dysplasia is shown for below the images for WT (n = 3) and KO (n = 3) BM cells. Bars show BM cytospins at low (20 µm) and high (7.5 µm) magnification. Error bars are mean ± SEM values.

Tifab deletion results in hematopoietic defects upon BM transplantationTo determine whether loss of TIFAB contributes to hematopoietic defects, we generated a constitutional knockout mouse model. The Tifab locus was targeted in embryonic stem cells with a construct containing LacZ in place of exon 3, which contains the entire coding sequence (Fig. 1 E). Germline deletion of the Tifab locus corresponded with loss of Tifab mRNA and protein (Fig. 1, F–H). Heterozygous (Tifab+/−) and homozygous (Tifab−/−) Tifab-deleted mice were born at expected Mendelian ratios and appeared physically normal. Peripheral blood (PB) counts at 6 and 12 mo of age were normal for all mice (unpublished data). Mature and immature BM cell populations of young or old Tifab-deficient (Tifab+/− and Tifab−/−) mice, as well as spleen sizes, were indistinguishable from WT (Tifab+/+) mice (unpublished data). However, a small subset of Tifab−/− and Tifab+/− mice, when aged beyond 1 yr, became moribund, presenting with enlarged spleens and evidence of BMF (cytopenia; unpublished data). Overall, these observations indicate that TIFAB is not required for steady-state hematopoiesis. To investigate the cell-autonomous function of Tifab-deficient HSPCs upon BM transplantation, BM cells from WT, Tifab+/−, and Tifab−/− mice were transplanted into lethally radiated syngeneic recipient mice. Mice receiving Tifab-deficient BM exhibited a progressive decline in peripheral white blood cell (WBC) counts (4.5 K/µl vs. 12.9 K/µl; P = 0.0014), without changes in red blood cell or platelet counts (Fig. 1 I). At 6 mo after transplant, mice transplanted with Tifab−/− BM cells exhibited significant reduction in neutrophils (0.99 K/µl vs. 1.62 K/µl; P < 0.0001), lymphocytes (4.93 K/µl vs. 8.18 K/µl; P < 0.001), and monocytes (0.38 K/µl vs. 0.61 K/µl; P = 0.0043; Fig. 1 J). Mice transplanted with Tifab−/− BM cells had a normal/hypocellular BM (Fig. 1 K); however, they exhibited dysplastic hypersegmented neutrophils, consistent with a granulocyte defect (10.8 ± 3.0% vs. 4.0 ± 1.6%; P = 0.027; Fig. 1 L).

Tifab deletion results in hematopoietic defects upon BM transplantation

To determine whether loss of TIFAB contributes to hematopoietic defects, we generated a constitutional knockout mouse model. The Tifab locus was targeted in embryonic stem cells with a construct containing LacZ in place of exon 3, which contains the entire coding sequence (Fig. 1 E). Germline deletion of the Tifab locus corresponded with loss of Tifab mRNA and protein (Fig. 1, F–H). Heterozygous (Tifab+/−) and homozygous (Tifab−/−) Tifab-deleted mice were born at expected Mendelian ratios and appeared physically normal. Peripheral blood (PB) counts at 6 and 12 mo of age were normal for all mice (unpublished data). Mature and immature BM cell populations of young or old Tifab-deficient (Tifab+/− and Tifab−/−) mice, as well as spleen sizes, were indistinguishable from WT (Tifab+/+) mice (unpublished data). However, a small subset of Tifab−/− and Tifab+/− mice, when aged beyond 1 yr, became moribund, presenting with enlarged spleens and evidence of BMF (cytopenia; unpublished data). Overall, these observations indicate that TIFAB is not required for steady-state hematopoiesis. To investigate the cell-autonomous function of Tifab-deficient HSPCs upon BM transplantation, BM cells from WT, Tifab+/−, and Tifab−/− mice were transplanted into lethally radiated syngeneic recipient mice. Mice receiving Tifab-deficient BM exhibited a progressive decline in peripheral white blood cell (WBC) counts (4.5 K/µl vs. 12.9 K/µl; P = 0.0014), without changes in red blood cell or platelet counts (Fig. 1 I). At 6 mo after transplant, mice transplanted with Tifab−/− BM cells exhibited significant reduction in neutrophils (0.99 K/µl vs. 1.62 K/µl; P < 0.0001), lymphocytes (4.93 K/µl vs. 8.18 K/µl; P < 0.001), and monocytes (0.38 K/µl vs. 0.61 K/µl; P = 0.0043; Fig. 1 J). Mice transplanted with Tifab−/− BM cells had a normal/hypocellular BM (Fig. 1 K); however, they exhibited dysplastic hypersegmented neutrophils, consistent with a granulocyte defect (10.8 ± 3.0% vs. 4.0 ± 1.6%; P = 0.027; Fig. 1 L).

Mice transplanted with Tifab-deficient HSPCs develop a bone marrow failure-like diseaseGiven that mice transplanted with Tifab-deficient BM cells developed worsening cytopenias, the mice were aged beyond 1 yr to determine whether they progress to a hematologic disease. By 6 mo, a subset of mice transplanted with Tifab−/− BM cells became moribund, and within 16 mo, ∼40% of Tifab−/− mice (4/11 mice; P = 0.06), and 25% of Tifab+− mice (4/15 mice; P = 0.15) were moribund, whereas no control mice died (0/9; Fig. 2 A). Moribund Tifab-deficient mice had significantly lower WBC counts, including fewer neutrophils (0.95 K/µl vs. 3.85 K/µl; P = 0.013), monocytes (0.43 K/µl vs. 1.05 K/µl; P = 0.12), and lymphocytes (2.70 K/µl vs. 15.1 K/µl; P = 0.0001; Fig. 2 B). However, RBC counts, hematocrit, and platelet counts in moribund Tifab-deficient mice were comparable to age-matched control mice (Fig. 2 B). Moribund Tifab-deficient mice also had reduced BM cellularity (Fig. 2 C) and smaller spleens (Fig. 2 D) as compared with control mice, suggestive of declining hematopoietic function. Morphological examination of BM cells from moribund Tifab-deficient mice did not indicate consistent evidence of myeloid dysplasia (Fig. 2 C). PB, BM, and spleen analysis of moribund Tifab-deficient mice showed heterogeneous expansion or diminution of myeloid, lymphoid, and erythroid proportions, which also varied between moribund mice (Fig. 2, E–G). Despite the variation in cell proportions, moribund Tifab-deficient mice clearly displayed abnormal composition of mature myeloid, lymphoid, and erythroid cells in the PB and BM. Although PB cytopenia were observed in all mice at time of death, HSPC proportions of Tifab-deficient BM were maintained, suggesting, that the cytopenia was not simply a result of loss of HSCs (Fig. 2 H). Therefore, the progressive cytopenia and death of a subset of mice receiving Tifab-deficient BM cells suggests impaired HSPC function after loss of TIFAB.Figure 2.Transplantation of Tifab-deleted BM cells results in bone marrow failure. (A) Kaplan-Meier survival curves for transplanted mice reconstituted with Tifab WT (n = 9), HET (n = 15), or KO (n = 11) BM cells. Summary from three independent transplants. (B) PB counts from age-matched WT (n = 3) or moribund Tifab KO and HET (n = 4) mice. *, P < 0.05; Student’s t test. WBC, white blood cell; RBC, red blood cell; PLT, platelet; NE, neutrophils; LY, lymphocytes; and MO, monocytes. (C) H&E-stained femur and spleen, and Wright-Giemsa–stained BM cytospins from a representative moribund Tifab KO mouse. Bars: (BM) 100 µm; (cytospins) 20 µm; (spleen) 100 µm. (D) Representative spleens from age-matched WT and moribund Tifab KO mice. (E–G) Flow cytometric analysis of PB (E), BM (F), and spleen (G) cells isolated from a WT and representative moribund Tifab KO transplant mouse. (H) Flow cytometric analysis of HSPC proportions in the BM of Tifab WT (n = 3) and moribund KO (n = 2) transplanted mice. Error bars are mean ± SEM values. *, P < 0.05.

Mice transplanted with Tifab-deficient HSPCs develop a bone marrow failure-like disease

Given that mice transplanted with Tifab-deficient BM cells developed worsening cytopenias, the mice were aged beyond 1 yr to determine whether they progress to a hematologic disease. By 6 mo, a subset of mice transplanted with Tifab−/− BM cells became moribund, and within 16 mo, ∼40% of Tifab−/− mice (4/11 mice; P = 0.06), and 25% of Tifab+− mice (4/15 mice; P = 0.15) were moribund, whereas no control mice died (0/9; Fig. 2 A). Moribund Tifab-deficient mice had significantly lower WBC counts, including fewer neutrophils (0.95 K/µl vs. 3.85 K/µl; P = 0.013), monocytes (0.43 K/µl vs. 1.05 K/µl; P = 0.12), and lymphocytes (2.70 K/µl vs. 15.1 K/µl; P = 0.0001; Fig. 2 B). However, RBC counts, hematocrit, and platelet counts in moribund Tifab-deficient mice were comparable to age-matched control mice (Fig. 2 B). Moribund Tifab-deficient mice also had reduced BM cellularity (Fig. 2 C) and smaller spleens (Fig. 2 D) as compared with control mice, suggestive of declining hematopoietic function. Morphological examination of BM cells from moribund Tifab-deficient mice did not indicate consistent evidence of myeloid dysplasia (Fig. 2 C). PB, BM, and spleen analysis of moribund Tifab-deficient mice showed heterogeneous expansion or diminution of myeloid, lymphoid, and erythroid proportions, which also varied between moribund mice (Fig. 2, E–G). Despite the variation in cell proportions, moribund Tifab-deficient mice clearly displayed abnormal composition of mature myeloid, lymphoid, and erythroid cells in the PB and BM. Although PB cytopenia were observed in all mice at time of death, HSPC proportions of Tifab-deficient BM were maintained, suggesting, that the cytopenia was not simply a result of loss of HSCs (Fig. 2 H). Therefore, the progressive cytopenia and death of a subset of mice receiving Tifab-deficient BM cells suggests impaired HSPC function after loss of TIFAB.

Transplantation of Tifab-deleted BM cells results in bone marrow failure. (A) Kaplan-Meier survival curves for transplanted mice reconstituted with Tifab WT (n = 9), HET (n = 15), or KO (n = 11) BM cells. Summary from three independent transplants. (B) PB counts from age-matched WT (n = 3) or moribund Tifab KO and HET (n = 4) mice. *, P < 0.05; Student’s t test. WBC, white blood cell; RBC, red blood cell; PLT, platelet; NE, neutrophils; LY, lymphocytes; and MO, monocytes. (C) H&E-stained femur and spleen, and Wright-Giemsa–stained BM cytospins from a representative moribund Tifab KO mouse. Bars: (BM) 100 µm; (cytospins) 20 µm; (spleen) 100 µm. (D) Representative spleens from age-matched WT and moribund Tifab KO mice. (E–G) Flow cytometric analysis of PB (E), BM (F), and spleen (G) cells isolated from a WT and representative moribund Tifab KO transplant mouse. (H) Flow cytometric analysis of HSPC proportions in the BM of Tifab WT (n = 3) and moribund KO (n = 2) transplanted mice. Error bars are mean ± SEM values. *, P < 0.05.

TIFAB regulates HSPC function and myeloid differentiationBecause TIFAB deletion results in a progressive decline in hematopoietic function (Fig. 1) and in marrow failure (Fig. 2), we examined the consequences of TIFAB loss before overt disease. Although HSPC populations of nontransplanted Tifab-deficient (Tifab+/− and Tifab−/−) mice were similar to WT mice (Fig. 3 A), after BMT, mice receiving Tifab−/− BM cells exhibited increased long-term HSCs (LT-HSC; P < 0.0001), lineage−Sca1+cKit+ (LSK; P = 0.0036), and common myeloid progenitors (CMP; P = 0.018), whereas a reduction in megakaryocyte-erythroid progenitors was observed (MEP; P = 0.0058; Fig. 3 B). Although HPSC proportions significantly differed between WT and Tifab-deficient mice, the proportion of mature BM cells was similar between WT and Tifab−/− transplanted mice (Fig. 3 C). However, the production of Gr1+ myeloid cells in the PB was significantly increased in mice transplanted with Tifab−/− BM cells (P = 0.037; Fig. 3 D). To assess whether TIFAB is important for hematopoietic progenitor differentiation in vitro, LSK isolated from WT or Tifab-deficient (Tifab+/− and Tifab−/−) mice were examined for progenitor colony formation in methylcellulose. LSK from Tifab-deficient mice formed ∼50% more CFU-G colonies as compared with WT LSK (P = 0.009; Fig. 3 E), underscoring a functional defect in Tifab-deficient HSPCs and consistent with a role of TIFAB in myeloid differentiation.Figure 3.Tifab−/− BM cells exhibit altered hematopoietic proportions after transplantation. (A) LSK, LT-HSC (LSK+ CD150+CD48−), MEP, CMP, and GMP in BM cells from Tifab WT or KO mice (n = 7). (B) LSK, LT-HSC (LSK+ CD150+CD48−), MEP, CMP, and GMP in BM from mice transplanted with Tifab WT or KO BM cells from two independent experiments (n = 6). *, P < 0.05; Student’s t test. (C and D) BM (C) and PB (D) proportions from mice transplanted with Tifab WT or KO BM cells (n = 6) from three independent experiments. *, P < 0.05; Student’s t test. (E) Numbers of colony-forming cells in methylcellulose from LSK cells from Tifab WT, HET, or KO BM mice (n = 6; 2 independent BMT). *, P < 0.05; Student’s t test. GM, granulocyte-monocyte; G, granulocyte; M, monocyte; E, erythroid (BFU-E). (F) 3 × 105 BM isolated from Tifab WT, HET, and KO mice (CD45.2) were mixed with equal numbers of competitor BoyJ BM cells (CD45.1), and then transplanted into lethally radiated syngeneic recipient BoyJ mice (CD45.1). PB and BM chimerism was determined at 4 and 8 mo after transplantation. Calculated chimerism is based on the ratio of CD45.1 and CD45.2 mononuclear cells. Three independent experiments were performed. (G) Representative flow cytometric blots of PB chimerism at 8 mo after transplantation on myeloid (Gr1+CD11b−) and lymphoid (B220+) gated populations. (H and I) Flow cytometric analysis of myeloid (CD11b and Gr1), and lymphoid (CD3 and B220) donor-derived CD45.2+ proportions within the PB (H) and BM (I) of Tifab WT (n = 4), HET (n = 4), and KO (n = 5) competitively transplanted mice. *, P < 0.05; Student’s t test. (J) Flow cytometric analysis of HSPC donor-derived CD45.2+ proportions within the BM of Tifab WT (n = 4), HET (n = 4), and KO (n = 5) competitively transplanted mice at 8 mo. *, P < 0.05; Student’s t test and Mann-Whitney test. Error bars are mean ± SEM values.Competitive repopulation assays were performed to determine whether loss of Tifab affects HSPC function in vivo. Whole BM cells (3 × 105) from 8-wk-old CD45.2 WT, Tifab+/−, or Tifab−/− mice were transplanted with equal numbers of 8-wk-old CD45.1 competitor BM cells into lethally irradiated CD45.1 recipient mice. PB chimerism was evaluated for 50 wk and was calculated based on the proportion of CD45.2 mononuclear cells (CD45.2/[CD45.2+CD45.1]). Consistent with the persistent cytopenias after transplantation of Tifab-deficient BM cells, we observed a reduction in hematopoietic repopulation within the BM and PB, indicating that Tifab−/− HSPCs have a significant disadvantage in competitive transplantation and/or homing (Fig. 3 F). Lower chimerism of Tifab-deficient cells was caused by a reduction in lymphoid (CD3+ and B220+) and myeloid (CD11b+ and Gr1+) contribution to the PB (Fig. 3 G). To gain insight into the effects of TIFAB deletion on hematopoietic differentiation, donor-derived (CD45.2-positive) proportions were evaluated in the BM and PB from the competitive transplantation experiments. Tifab deficiency contributes to increased myeloid (CD11b+; P = 0.015) and reduced lymphoid (B220+ and CD3+; P = 0.004) donor-derived proportions in the PB (Fig. 3 H), with minor consequences on myeloid and lymphoid donor-derived proportions in the BM (Fig. 3 I). In addition, donor-derived Tifab−/− cells resulted in increased CMP (P = 0.03), and slightly elevated LT-HSCs (P = 0.09) and GMP (P = 0.08) proportions (Fig. 3 J). The hematopoietic differences within mice competitively transplanted with Tifab+/− BM cells were consistent by some measures with that of Tifab−/−-transplanted mice for CD11b-expansion and lymphoid-reduction in the PB (Fig. 3 H), and for expansion of the GMP compartment in the BM (Fig. 3 J). These findings suggest that Tifab deficiency affects myeloid/lymphoid-lineage differentiation and HSPC function, before development of overt hematologic disease consistent with BMF.

TIFAB regulates HSPC function and myeloid differentiation

Because TIFAB deletion results in a progressive decline in hematopoietic function (Fig. 1) and in marrow failure (Fig. 2), we examined the consequences of TIFAB loss before overt disease. Although HSPC populations of nontransplanted Tifab-deficient (Tifab+/− and Tifab−/−) mice were similar to WT mice (Fig. 3 A), after BMT, mice receiving Tifab−/− BM cells exhibited increased long-term HSCs (LT-HSC; P < 0.0001), lineage−Sca1+cKit+ (LSK; P = 0.0036), and common myeloid progenitors (CMP; P = 0.018), whereas a reduction in megakaryocyte-erythroid progenitors was observed (MEP; P = 0.0058; Fig. 3 B). Although HPSC proportions significantly differed between WT and Tifab-deficient mice, the proportion of mature BM cells was similar between WT and Tifab−/− transplanted mice (Fig. 3 C). However, the production of Gr1+ myeloid cells in the PB was significantly increased in mice transplanted with Tifab−/− BM cells (P = 0.037; Fig. 3 D). To assess whether TIFAB is important for hematopoietic progenitor differentiation in vitro, LSK isolated from WT or Tifab-deficient (Tifab+/− and Tifab−/−) mice were examined for progenitor colony formation in methylcellulose. LSK from Tifab-deficient mice formed ∼50% more CFU-G colonies as compared with WT LSK (P = 0.009; Fig. 3 E), underscoring a functional defect in Tifab-deficient HSPCs and consistent with a role of TIFAB in myeloid differentiation.

Tifab−/− BM cells exhibit altered hematopoietic proportions after transplantation. (A) LSK, LT-HSC (LSK+ CD150+CD48−), MEP, CMP, and GMP in BM cells from Tifab WT or KO mice (n = 7). (B) LSK, LT-HSC (LSK+ CD150+CD48−), MEP, CMP, and GMP in BM from mice transplanted with Tifab WT or KO BM cells from two independent experiments (n = 6). *, P < 0.05; Student’s t test. (C and D) BM (C) and PB (D) proportions from mice transplanted with Tifab WT or KO BM cells (n = 6) from three independent experiments. *, P < 0.05; Student’s t test. (E) Numbers of colony-forming cells in methylcellulose from LSK cells from Tifab WT, HET, or KO BM mice (n = 6; 2 independent BMT). *, P < 0.05; Student’s t test. GM, granulocyte-monocyte; G, granulocyte; M, monocyte; E, erythroid (BFU-E). (F) 3 × 105 BM isolated from Tifab WT, HET, and KO mice (CD45.2) were mixed with equal numbers of competitor BoyJ BM cells (CD45.1), and then transplanted into lethally radiated syngeneic recipient BoyJ mice (CD45.1). PB and BM chimerism was determined at 4 and 8 mo after transplantation. Calculated chimerism is based on the ratio of CD45.1 and CD45.2 mononuclear cells. Three independent experiments were performed. (G) Representative flow cytometric blots of PB chimerism at 8 mo after transplantation on myeloid (Gr1+CD11b−) and lymphoid (B220+) gated populations. (H and I) Flow cytometric analysis of myeloid (CD11b and Gr1), and lymphoid (CD3 and B220) donor-derived CD45.2+ proportions within the PB (H) and BM (I) of Tifab WT (n = 4), HET (n = 4), and KO (n = 5) competitively transplanted mice. *, P < 0.05; Student’s t test. (J) Flow cytometric analysis of HSPC donor-derived CD45.2+ proportions within the BM of Tifab WT (n = 4), HET (n = 4), and KO (n = 5) competitively transplanted mice at 8 mo. *, P < 0.05; Student’s t test and Mann-Whitney test. Error bars are mean ± SEM values.

Competitive repopulation assays were performed to determine whether loss of Tifab affects HSPC function in vivo. Whole BM cells (3 × 105) from 8-wk-old CD45.2 WT, Tifab+/−, or Tifab−/− mice were transplanted with equal numbers of 8-wk-old CD45.1 competitor BM cells into lethally irradiated CD45.1 recipient mice. PB chimerism was evaluated for 50 wk and was calculated based on the proportion of CD45.2 mononuclear cells (CD45.2/[CD45.2+CD45.1]). Consistent with the persistent cytopenias after transplantation of Tifab-deficient BM cells, we observed a reduction in hematopoietic repopulation within the BM and PB, indicating that Tifab−/− HSPCs have a significant disadvantage in competitive transplantation and/or homing (Fig. 3 F). Lower chimerism of Tifab-deficient cells was caused by a reduction in lymphoid (CD3+ and B220+) and myeloid (CD11b+ and Gr1+) contribution to the PB (Fig. 3 G). To gain insight into the effects of TIFAB deletion on hematopoietic differentiation, donor-derived (CD45.2-positive) proportions were evaluated in the BM and PB from the competitive transplantation experiments. Tifab deficiency contributes to increased myeloid (CD11b+; P = 0.015) and reduced lymphoid (B220+ and CD3+; P = 0.004) donor-derived proportions in the PB (Fig. 3 H), with minor consequences on myeloid and lymphoid donor-derived proportions in the BM (Fig. 3 I). In addition, donor-derived Tifab−/− cells resulted in increased CMP (P = 0.03), and slightly elevated LT-HSCs (P = 0.09) and GMP (P = 0.08) proportions (Fig. 3 J). The hematopoietic differences within mice competitively transplanted with Tifab+/− BM cells were consistent by some measures with that of Tifab−/−-transplanted mice for CD11b-expansion and lymphoid-reduction in the PB (Fig. 3 H), and for expansion of the GMP compartment in the BM (Fig. 3 J). These findings suggest that Tifab deficiency affects myeloid/lymphoid-lineage differentiation and HSPC function, before development of overt hematologic disease consistent with BMF.

TIFAB regulates immune and survival gene signatures in HSPCsTo understand the molecular basis for impaired HSPC function and BMF after TIFAB deletion, we performed an expression analysis on sorted LSK isolated from 3-mo-old mice transplanted with WT or Tifab−/− BM cells (n = 3 mice/group). We selected this time point to capture the gene expression profile of Tifab−/− LSK after engraftment but before overt hematopoietic failure. Complete blood counts before isolation of LSK cells confirmed absence of BMF in all mice. Analysis of the gene expression data identified differentially expressed genes in Tifab−/− LSK cells, including 32 up-regulated (>1.7-fold) and 22 down-regulated (<1.7-fold; Fig. 4 A and Table S1). To elucidate the relevant signaling pathway regulated after Tifab loss in HSPCs, we performed a gene set enrichment analysis (GSEA; Subramanian et al., 2005), which revealed overexpression of immune and infection response signatures, particularly related to interferon-regulated genes (Fig. 4 B). Down-regulated gene signatures in Tifab−/− LSK also included immune response signatures, but these were related to enterotoxin and LPS (Fig. 4, A and B). Interestingly, the immune-related gene signatures show a clear disparity between various immune insults, in which certain signatures (i.e., viral and interferon) are enriched, whereas other signatures (i.e., TNF and endo/enterotoxin) are down-regulated. Select genes from the microarray analysis that were overexpressed (Il6 and Bcl2a1) and down-regulated (Tnfaip3 and Sod2) in Tifab−/− LSK were confirmed by qRT-PCR (Fig. 4 C).Figure 4.TIFAB suppresses immune signaling and NF-κB activation downstream of the TLR4 receptor in BM HSPCs. (A) Heat map generated from gene set enrichment analysis (GSEA) showing gene expression differences in LSK isolated from 3-mo-old mice transplanted with Tifab WT or KO BM cells (n = 3 mice/group). (B) GSEA of statistically significant gene sets enriched in the LSK cells of transplanted Tifab−/− cells (up in KO) or WT cells (down in KO). (C) Validation of gene expression of the indicated genes in BM MNC by qRT-PCR. *, P < 0.05; Student’s t test. (D and E) Expression analysis of the indicated genes in BM MNC by qRT-PCR after stimulation with LPS (D; 100 ng/ml) or TNF (E; 10 ng/ml) for the indicated time points. Expression was normalized to time 0 (1.0). *, P < 0.05; Student’s t test.Although dysregulation of immune gene signatures is evident even in resting Tifab−/− HPSC, these findings prompted us to examine the role of TIFAB in regulating inducible gene expression. We selected two stimuli (TNF and LPS) that are associated with enriched GSEA signatures in Tifab-deficient HSPCs (Fig. 4 B). BM mononuclear cells (MNC) from WT and Tifab−/− mice were stimulated with TNF (10 ng/ml) or LPS (100 ng/ml) at various time points (0, 1, and 4 h) and examined for target gene expression by qRT-PCR. As expected, stimulation of WT BM cells with either LPS or TNF resulted in a time-dependent increase in Il6, Bcl2a1, Tnfaip3, and Sod2 expression (Fig. 4, D and E). Although expression of certain genes was higher (Il6 and Bcl2a1) or lower (Tnfaip3 and Sod2) in Tifab−/− than WT BM cells at basal levels, LPS stimulation resulted in a significant and acute increase of all genes examined in Tifab−/− BM cells (Fig. 4 D). In contrast, TNF stimulation of Tifab−/− BM cells resulted in a delayed response of Tnfaip3 and Sod2, and no dramatic differences of Il6 and Bcl2a1 expression (Fig. 4 E). These findings reveal that TIFAB controls the dynamic range of immune pathway activation downstream of TLRs, but not the TNF receptor (TNFR).

TIFAB regulates immune and survival gene signatures in HSPCs

To understand the molecular basis for impaired HSPC function and BMF after TIFAB deletion, we performed an expression analysis on sorted LSK isolated from 3-mo-old mice transplanted with WT or Tifab−/− BM cells (n = 3 mice/group). We selected this time point to capture the gene expression profile of Tifab−/− LSK after engraftment but before overt hematopoietic failure. Complete blood counts before isolation of LSK cells confirmed absence of BMF in all mice. Analysis of the gene expression data identified differentially expressed genes in Tifab−/− LSK cells, including 32 up-regulated (>1.7-fold) and 22 down-regulated (<1.7-fold; Fig. 4 A and Table S1). To elucidate the relevant signaling pathway regulated after Tifab loss in HSPCs, we performed a gene set enrichment analysis (GSEA; Subramanian et al., 2005), which revealed overexpression of immune and infection response signatures, particularly related to interferon-regulated genes (Fig. 4 B). Down-regulated gene signatures in Tifab−/− LSK also included immune response signatures, but these were related to enterotoxin and LPS (Fig. 4, A and B). Interestingly, the immune-related gene signatures show a clear disparity between various immune insults, in which certain signatures (i.e., viral and interferon) are enriched, whereas other signatures (i.e., TNF and endo/enterotoxin) are down-regulated. Select genes from the microarray analysis that were overexpressed (Il6 and Bcl2a1) and down-regulated (Tnfaip3 and Sod2) in Tifab−/− LSK were confirmed by qRT-PCR (Fig. 4 C).

TIFAB suppresses immune signaling and NF-κB activation downstream of the TLR4 receptor in BM HSPCs. (A) Heat map generated from gene set enrichment analysis (GSEA) showing gene expression differences in LSK isolated from 3-mo-old mice transplanted with Tifab WT or KO BM cells (n = 3 mice/group). (B) GSEA of statistically significant gene sets enriched in the LSK cells of transplanted Tifab−/− cells (up in KO) or WT cells (down in KO). (C) Validation of gene expression of the indicated genes in BM MNC by qRT-PCR. *, P < 0.05; Student’s t test. (D and E) Expression analysis of the indicated genes in BM MNC by qRT-PCR after stimulation with LPS (D; 100 ng/ml) or TNF (E; 10 ng/ml) for the indicated time points. Expression was normalized to time 0 (1.0). *, P < 0.05; Student’s t test.

Although dysregulation of immune gene signatures is evident even in resting Tifab−/− HPSC, these findings prompted us to examine the role of TIFAB in regulating inducible gene expression. We selected two stimuli (TNF and LPS) that are associated with enriched GSEA signatures in Tifab-deficient HSPCs (Fig. 4 B). BM mononuclear cells (MNC) from WT and Tifab−/− mice were stimulated with TNF (10 ng/ml) or LPS (100 ng/ml) at various time points (0, 1, and 4 h) and examined for target gene expression by qRT-PCR. As expected, stimulation of WT BM cells with either LPS or TNF resulted in a time-dependent increase in Il6, Bcl2a1, Tnfaip3, and Sod2 expression (Fig. 4, D and E). Although expression of certain genes was higher (Il6 and Bcl2a1) or lower (Tnfaip3 and Sod2) in Tifab−/− than WT BM cells at basal levels, LPS stimulation resulted in a significant and acute increase of all genes examined in Tifab−/− BM cells (Fig. 4 D). In contrast, TNF stimulation of Tifab−/− BM cells resulted in a delayed response of Tnfaip3 and Sod2, and no dramatic differences of Il6 and Bcl2a1 expression (Fig. 4 E). These findings reveal that TIFAB controls the dynamic range of immune pathway activation downstream of TLRs, but not the TNF receptor (TNFR).

TIFAB forms a complex with TRAF6 and inhibits NF-κBThe molecular function of TIFAB that restricts immune signaling in MDS/AML remains to be determined. To uncover the molecular function of TIFAB, we performed a global proteome analysis to identify TIFAB-associated cofactors in a del(5q) leukemic cell line. We generated stable HL60 lines expressing human TIFAB tagged with both FLAG and HA epitopes. TIFAB and its associated partners were purified from total cell lysates derived from HL60 cells using tandem affinity immunoprecipitation, as previously described (Singh et al., 2008; Fig. 5 A). Mass spectrometry (MS) analysis identified ∼300 unique protein partners of TIFAB. An MS-identified partner of TIFAB is TRAF6, a major component of the TLR signaling pathway and a functionally relevant protein in the pathogenesis of MDS/AML. Other TLR pathway molecules were also uncovered: SQSTM1/p62 and TIFA. To further investigate the TIFAB–TRAF6 interaction, HL60 cells expressing TIFAB were fractionated into cytoplasmic and nuclear fractions. The TIFAB–TRAF6 complex is formed predominantly within the cytoplasm (Fig. 5 B), which is consistent with the known role of TRAF6 signaling during TLR activation. TIFAB (double tagged with FLAG and HA) was also cotransfected with TRAF6 (tagged with FLAG) into HEK293 cells, and immunoprecipitated (anti-HA). In support of the MS analysis, immunoblotting of TRAF6 (anti-FLAG) confirmed that TIFAB co-immunoprecipitates with TRAF6 (Fig. 5 C).Figure 5.TIFAB binds to TRAF6 and suppresses NF-κB activation. (A) A representative silver stain after tandem affinity purification of FLAG-HA-TIFAB from HL60 cells. (B) TIFAB-expressing HL60 cells were lysed and fractionated into cytoplasmic (C) and nuclear (N) fractions, which were then subjected to coimmunoprecipitation of FLAG-HA-TIFAB. Immunoblotting for TRAF6, TIFAB, PARP (nuclear loading control), and GAPDH (cytosolic loading control) is shown. (C) Coimmunoprecipitation analysis of HEK293 transfected with empty vector, HA-TIFAB, FLAG-TRAF6, or HA-TIFAB and FLAG-TRAF6. Lysates were immunoprecipitated with anti-HA (TIFAB), and then blotted for FLAG (TRAF6). (D) Immunoblot analysis of THP1 and HL60 cells transduced with empty vector (Vec) or FLAG-TIFAB for phosphorylated IKKβ (pIKKβ), total IKKβ, and GAPDH. (E) Overview of TLR4 and TNFR-mediated activation of NF-κB, and the proposed regulation by TIFAB. (F–I) NF-κB activation was measured by κB-site containing reporter assays in HEK293 cells transfected with empty vector, TIFAB, and/or IKKβ (F), p65 (G), TRAF6 (H), or TRAF2 (I). Values are normalized to Renilla-luciferase and empty vector (1.0). Summary of three independent experiments. Error bars are mean ± SEM values. *, P < 0.05; Student’s t test.Stimulation of TLRs mediates signals to several transcription factors, one of which is the NF-κB family. Because TIFAB expression is low in THP1 and HL60 cells, whereas NF-κB activation is high, we first determined the consequences of TIFAB reexpression on NF-κB signaling in these relevant human cells. THP1 and HL60 cells were transduced with a vector encoding TIFAB or control (empty vector), and examined for phosphorylated IKKβ (pIKKβ), an indication of NF-κB activation. Overexpression of TIFAB resulted in significant reduction in pIKKβ, indicating that TIFAB reexpression is sufficient to inhibit NF-κB (Fig. 5 D). Because TLR and TNF receptor 2 (TNFR2) use different signaling mediators to activate NF-κB (Fig. 5 E), we wanted to determine the precise step at which TIFAB regulates receptor-induced NF-κB signaling. To examine this mechanism, we performed κB-site containing reporter assays transfected with various activators of NF-κB (IKKβ, RelA/p65) and ones downstream of TLR4 (TRAF6) or TNFR2 (TRAF2; Fig. 5 E). Co-transfection of TIFAB did not suppress IKKβ- (Fig. 5 F) or p65-mediated (Fig. 5 G) κB-site luciferase activity, indicating that TIFAB inhibits NF-κB signaling upstream of the NF-κB kinase complex (IKK) and its transcription factor (RelA/p65). However, co-transfection of TIFAB significantly repressed κB-site luciferase activity by TRAF6 (Fig. 5 H). In contrast, TIFAB did not repress κB-site activation after transfection of TRAF2, a functionally related paralog of TRAF6 (Fig. 5 I). TIFAB represses TRAF6-, but not TRAF2-mediated NF-κB activation, implying that TIFAB selectively inhibits TLR/TRAF6-dependent NF-κB stimuli while having no effect on TNFR2/TRAF2-dependent NF-κB stimuli.

TIFAB forms a complex with TRAF6 and inhibits NF-κB

The molecular function of TIFAB that restricts immune signaling in MDS/AML remains to be determined. To uncover the molecular function of TIFAB, we performed a global proteome analysis to identify TIFAB-associated cofactors in a del(5q) leukemic cell line. We generated stable HL60 lines expressing human TIFAB tagged with both FLAG and HA epitopes. TIFAB and its associated partners were purified from total cell lysates derived from HL60 cells using tandem affinity immunoprecipitation, as previously described (Singh et al., 2008; Fig. 5 A). Mass spectrometry (MS) analysis identified ∼300 unique protein partners of TIFAB. An MS-identified partner of TIFAB is TRAF6, a major component of the TLR signaling pathway and a functionally relevant protein in the pathogenesis of MDS/AML. Other TLR pathway molecules were also uncovered: SQSTM1/p62 and TIFA. To further investigate the TIFAB–TRAF6 interaction, HL60 cells expressing TIFAB were fractionated into cytoplasmic and nuclear fractions. The TIFAB–TRAF6 complex is formed predominantly within the cytoplasm (Fig. 5 B), which is consistent with the known role of TRAF6 signaling during TLR activation. TIFAB (double tagged with FLAG and HA) was also cotransfected with TRAF6 (tagged with FLAG) into HEK293 cells, and immunoprecipitated (anti-HA). In support of the MS analysis, immunoblotting of TRAF6 (anti-FLAG) confirmed that TIFAB co-immunoprecipitates with TRAF6 (Fig. 5 C).

TIFAB binds to TRAF6 and suppresses NF-κB activation. (A) A representative silver stain after tandem affinity purification of FLAG-HA-TIFAB from HL60 cells. (B) TIFAB-expressing HL60 cells were lysed and fractionated into cytoplasmic (C) and nuclear (N) fractions, which were then subjected to coimmunoprecipitation of FLAG-HA-TIFAB. Immunoblotting for TRAF6, TIFAB, PARP (nuclear loading control), and GAPDH (cytosolic loading control) is shown. (C) Coimmunoprecipitation analysis of HEK293 transfected with empty vector, HA-TIFAB, FLAG-TRAF6, or HA-TIFAB and FLAG-TRAF6. Lysates were immunoprecipitated with anti-HA (TIFAB), and then blotted for FLAG (TRAF6). (D) Immunoblot analysis of THP1 and HL60 cells transduced with empty vector (Vec) or FLAG-TIFAB for phosphorylated IKKβ (pIKKβ), total IKKβ, and GAPDH. (E) Overview of TLR4 and TNFR-mediated activation of NF-κB, and the proposed regulation by TIFAB. (F–I) NF-κB activation was measured by κB-site containing reporter assays in HEK293 cells transfected with empty vector, TIFAB, and/or IKKβ (F), p65 (G), TRAF6 (H), or TRAF2 (I). Values are normalized to Renilla-luciferase and empty vector (1.0). Summary of three independent experiments. Error bars are mean ± SEM values. *, P < 0.05; Student’s t test.

Stimulation of TLRs mediates signals to several transcription factors, one of which is the NF-κB family. Because TIFAB expression is low in THP1 and HL60 cells, whereas NF-κB activation is high, we first determined the consequences of TIFAB reexpression on NF-κB signaling in these relevant human cells. THP1 and HL60 cells were transduced with a vector encoding TIFAB or control (empty vector), and examined for phosphorylated IKKβ (pIKKβ), an indication of NF-κB activation. Overexpression of TIFAB resulted in significant reduction in pIKKβ, indicating that TIFAB reexpression is sufficient to inhibit NF-κB (Fig. 5 D). Because TLR and TNF receptor 2 (TNFR2) use different signaling mediators to activate NF-κB (Fig. 5 E), we wanted to determine the precise step at which TIFAB regulates receptor-induced NF-κB signaling. To examine this mechanism, we performed κB-site containing reporter assays transfected with various activators of NF-κB (IKKβ, RelA/p65) and ones downstream of TLR4 (TRAF6) or TNFR2 (TRAF2; Fig. 5 E). Co-transfection of TIFAB did not suppress IKKβ- (Fig. 5 F) or p65-mediated (Fig. 5 G) κB-site luciferase activity, indicating that TIFAB inhibits NF-κB signaling upstream of the NF-κB kinase complex (IKK) and its transcription factor (RelA/p65). However, co-transfection of TIFAB significantly repressed κB-site luciferase activity by TRAF6 (Fig. 5 H). In contrast, TIFAB did not repress κB-site activation after transfection of TRAF2, a functionally related paralog of TRAF6 (Fig. 5 I). TIFAB represses TRAF6-, but not TRAF2-mediated NF-κB activation, implying that TIFAB selectively inhibits TLR/TRAF6-dependent NF-κB stimuli while having no effect on TNFR2/TRAF2-dependent NF-κB stimuli.

TIFAB restricts TLR activation by destabilizing TRAF6 proteinTo investigate the mechanism by which TIFAB inhibits TRAF6-mediated signaling, we overexpressed TIFAB in HEK293 or HL60 cells. We observed reduced levels of transfected (HEK293: 60% decrease) and endogenous (HL60: 40% decrease) TRAF6 protein (Fig. 6 A). Moreover, expression of TIFAB reduced levels of cotransfected TRAF6, whereas the levels of TRAF2 were not affected (Fig. 6 B). Conversely, THP1 cells after knockdown of TIFAB (Fig. 6 C), as well as Tifab−/− BM cells after BM transplantation (Fig. 6 D), exhibited increased expression of TRAF6 protein. The effects of TIFAB on TRAF6 protein are independent of TRAF6 mRNA, as overexpression of TIFAB did not reduce TRAF6 mRNA in HEK293 cells (Fig. 6 E), nor did Tifab deficiency increase Traf6 mRNA in BM cells (Fig. 6 F). To explore the possibility that TIFAB induces TRAF6 protein degradation, we used selective inhibitors of the proteasome (MG-132) or lysosome (3-MA). Treatment of HEK293 cells expressing TRAF6 and TIFAB with MG-132 did not restore TRAF6 protein levels (Fig. 6 G), indicating that TIFAB-mediated degradation of TRAF6 does not occur through the ubiquitin–proteasome pathway (Fig. 6 H). However, treatment with 3-MA partially restored TRAF6 protein expression in TIFAB-expressing HEK293 cells (Fig. 6 G), suggesting that TIFAB-mediated degradation may occur through the lysosome–autophagy pathway (Fig. 6 H).Figure 6.TIFAB inhibits TLR4 signaling by inducing degradation of TRAF6 protein via lysosomes. (A) Immunoblot analysis of HEK293 cells transfected with empty vector, TRAF6, or FLAG-TIFAB (left), and HL60 cells transduced with empty vector (pMSCV-pGK-GFP) or FLAG-TIFAB (right). Images are representative from at least three independent experiments. (B) Immunoblot analysis of HEK293 transfected with empty vector, TIFAB, and/or TRAF6 (left) or TRAF2 (right). Images are representative from at least two independent experiments. (C) Immunoblot analysis of human THP1 cells transduced with scrambled control shRNA (shCTL) or an shRNA targeting human TIFAB (shTIFAB). (D) Immunoblot analysis for TRAF6 on Tifab WT or KO BM cells from independent pre and post-transplanted mice. (E) Expression analysis of TRAF6 mRNA by qRT-PCR in HEK293 cells transfected with empty vector, TRAF6, and/or FLAG-TIFAB. (F) Expression analysis of Tifab or Traf6 mRNA by qRT-PCR in Tifab WT and KO BM cells. (G) Immunoblot analysis of HEK293 transfected with empty vector, TIFAB, and/or TRAF6, and then treated with a proteasome inhibitor (MG132, 10 µM for 12 h) or a lysosome inhibitor (3-MA, 5 mM for 24 h). Images are representative from at least three independent experiments. (H) HEK293 cells were treated with MG132 or 3-MA as in G, and immunoblotted for total ubiquitin (Ub) and LC3B, respectively. (I) Expression analysis of Traf6 mRNA in Tifab WT or KO-transduced BM MNC expressing a scrambled control shRNA (shCTL) or an shRNA targeting mouse Traf6 (shTraf6). (J and K) QRT-PCR expression analysis of Tnfaip3 (J) and Sod2 (K) in Tifab WT or KO BM MNC expressing a scrambled control shRNA (shCTL) or an shRNA-targeting mouse Traf6 (shTraf6) after stimulation with LPS (100 ng/ml) for 4 h. Data are summarized from biological and technical replicates. Error bars are mean ± SEM values. *, P < 0.05; Student’s t test.

TIFAB restricts TLR activation by destabilizing TRAF6 protein

To investigate the mechanism by which TIFAB inhibits TRAF6-mediated signaling, we overexpressed TIFAB in HEK293 or HL60 cells. We observed reduced levels of transfected (HEK293: 60% decrease) and endogenous (HL60: 40% decrease) TRAF6 protein (Fig. 6 A). Moreover, expression of TIFAB reduced levels of cotransfected TRAF6, whereas the levels of TRAF2 were not affected (Fig. 6 B). Conversely, THP1 cells after knockdown of TIFAB (Fig. 6 C), as well as Tifab−/− BM cells after BM transplantation (Fig. 6 D), exhibited increased expression of TRAF6 protein. The effects of TIFAB on TRAF6 protein are independent of TRAF6 mRNA, as overexpression of TIFAB did not reduce TRAF6 mRNA in HEK293 cells (Fig. 6 E), nor did Tifab deficiency increase Traf6 mRNA in BM cells (Fig. 6 F). To explore the possibility that TIFAB induces TRAF6 protein degradation, we used selective inhibitors of the proteasome (MG-132) or lysosome (3-MA). Treatment of HEK293 cells expressing TRAF6 and TIFAB with MG-132 did not restore TRAF6 protein levels (Fig. 6 G), indicating that TIFAB-mediated degradation of TRAF6 does not occur through the ubiquitin–proteasome pathway (Fig. 6 H). However, treatment with 3-MA partially restored TRAF6 protein expression in TIFAB-expressing HEK293 cells (Fig. 6 G), suggesting that TIFAB-mediated degradation may occur through the lysosome–autophagy pathway (Fig. 6 H).

TIFAB inhibits TLR4 signaling by inducing degradation of TRAF6 protein via lysosomes. (A) Immunoblot analysis of HEK293 cells transfected with empty vector, TRAF6, or FLAG-TIFAB (left), and HL60 cells transduced with empty vector (pMSCV-pGK-GFP) or FLAG-TIFAB (right). Images are representative from at least three independent experiments. (B) Immunoblot analysis of HEK293 transfected with empty vector, TIFAB, and/or TRAF6 (left) or TRAF2 (right). Images are representative from at least two independent experiments. (C) Immunoblot analysis of human THP1 cells transduced with scrambled control shRNA (shCTL) or an shRNA targeting human TIFAB (shTIFAB). (D) Immunoblot analysis for TRAF6 on Tifab WT or KO BM cells from independent pre and post-transplanted mice. (E) Expression analysis of TRAF6 mRNA by qRT-PCR in HEK293 cells transfected with empty vector, TRAF6, and/or FLAG-TIFAB. (F) Expression analysis of Tifab or Traf6 mRNA by qRT-PCR in Tifab WT and KO BM cells. (G) Immunoblot analysis of HEK293 transfected with empty vector, TIFAB, and/or TRAF6, and then treated with a proteasome inhibitor (MG132, 10 µM for 12 h) or a lysosome inhibitor (3-MA, 5 mM for 24 h). Images are representative from at least three independent experiments. (H) HEK293 cells were treated with MG132 or 3-MA as in G, and immunoblotted for total ubiquitin (Ub) and LC3B, respectively. (I) Expression analysis of Traf6 mRNA in Tifab WT or KO-transduced BM MNC expressing a scrambled control shRNA (shCTL) or an shRNA targeting mouse Traf6 (shTraf6). (J and K) QRT-PCR expression analysis of Tnfaip3 (J) and Sod2 (K) in Tifab WT or KO BM MNC expressing a scrambled control shRNA (shCTL) or an shRNA-targeting mouse Traf6 (shTraf6) after stimulation with LPS (100 ng/ml) for 4 h. Data are summarized from biological and technical replicates. Error bars are mean ± SEM values. *, P < 0.05; Student’s t test.

TRAF6 is required for the hyperimmune response in Tifab-deficient hematopoietic cellsTo establish whether Tifab-deficient BM cells exhibit increased LPS-response through TRAF6, rescue experiments were performed by depleting Traf6 in Tifab−/− BM cells. WT and Tifab−/− BM cells were transduced with shTraf6 (shT6) or control shRNA (shCTL; Fig. 6 I), and then stimulated with LPS for up to 4 h. As in Fig. 4 D, LPS stimulation of WT BM cells expressing the control shRNA exhibited an increase in Tnfaip3 and Sod2 expression, whereas Tifab−/− BM stimulated with LPS exhibited a further increase in both Tnfaip3 and Sod2 levels (Fig. 6, J and K). Knockdown of Traf6 in both WT and Tifab−/− BM cells completely abolished LPS-induced expression of Tnfaip3 and Sod2 (Fig. 6, J and K). Collectively, these findings have uncovered that TIFAB controls TLR signaling by binding and regulating TRAF6 protein stability through lysosomal degradation, but does not affect TNFR–TRAF2 signaling.

TRAF6 is required for the hyperimmune response in Tifab-deficient hematopoietic cells

To establish whether Tifab-deficient BM cells exhibit increased LPS-response through TRAF6, rescue experiments were performed by depleting Traf6 in Tifab−/− BM cells. WT and Tifab−/− BM cells were transduced with shTraf6 (shT6) or control shRNA (shCTL; Fig. 6 I), and then stimulated with LPS for up to 4 h. As in Fig. 4 D, LPS stimulation of WT BM cells expressing the control shRNA exhibited an increase in Tnfaip3 and Sod2 expression, whereas Tifab−/− BM stimulated with LPS exhibited a further increase in both Tnfaip3 and Sod2 levels (Fig. 6, J and K). Knockdown of Traf6 in both WT and Tifab−/− BM cells completely abolished LPS-induced expression of Tnfaip3 and Sod2 (Fig. 6, J and K). Collectively, these findings have uncovered that TIFAB controls TLR signaling by binding and regulating TRAF6 protein stability through lysosomal degradation, but does not affect TNFR–TRAF2 signaling.

TIFAB and miR-146a cooperate to regulate TRAF6 expression and myeloid cell differentiationHuman miR-146a and TIFAB both reside on chr 5q and are co-deleted in ∼80% of del(5q) MDS and AML (Gondek et al., 2008). miR-146a−/− mice develop BMF and myeloid leukemia, in part because of derepression of TRAF6 mRNA and protein (Lu et al., 2010; Boldin et al., 2011; Zhao et al., 2011). Given that TIFAB and miR-146a are codeleted in MDS/AML, we examined whether there is a functional cooperation between TIFAB and miR-146a. miR-146a inhibits TRAF6 protein expression and reduces its mRNA levels, whereas herein we report that TIFAB inhibits TRAF6 by destabilizing its protein. Because miR-146a is an NF-κB target gene responsive to LPS, miR-146a expression was investigated in TIFAB-deficient BM cells. At basal levels and after LPS stimulation, miR-146a expression is significantly elevated in Tifab−/− BM MNC (2-fold and 15-fold at baseline and after stimulation, respectively; Fig. 7 A). Although TRAF6 protein is stabilized after BM transplantation in Tifab−/− BM cells (Fig. 6 D), Traf6 mRNA is significantly reduced in Tifab−/− BM cells as compared with WT cells (Fig. 6 F and Fig. 7 B), suggesting that increased miR-146a expression in Tifab-deficient cells may suppress TRAF6 mRNA. To examine a cooperative effect of TIFAB and miR-146a deficiency on TRAF6 mRNA and protein expression, we generated double Tifab- and miR-146a–deficient mice. Combined deletion of Tifab and miR-146a restored TRAF6 mRNA expression to WT levels (Fig. 7 B). In addition, TRAF6 protein expression was synergistically increased in BM MNC from Tifab−/−;miR-146a−/− mice (dKO) after BM transplantation as compared with single-knockout BM cells (Fig. 7 C). Collectively, miR-146a and TIFAB cooperate to suppress TRAF6 mRNA and protein expression in BM HSPCs.Figure 7.TIFAB and miR-146a cooperate to regulate TRAF6 and hematopoietic function. (A) Expression analysis of miR-146a in WT and KO BM MNC by qRT-PCR after stimulation with LPS (100 ng/ml) for the indicated time points. Expression was normalized to time 0 (1.0). *, P < 0.05; Student’s t test. (B) Expression analysis of Traf6 mRNA by qRT-PCR in WT, Tifab KO, miR-146a KO, and double KO (dKO) BM cells. *, P < 0.05; Student’s t test. (C) Immunoblot analysis for TRAF6 on WT, Tifab KO, miR-146a KO, and dKO BM cells from transplanted mice. (D) Total colony-forming cells in methylcellulose from BM MNC from WT, Tifab KO, miR-146a KO, and dKO BM (n = 5–6; two independent experiments). *, P < 0.05; Student’s t test. (E) Proportion of colonies formed from D. GM, granulocyte-monocyte; G, granulocyte; M, monocyte; E, erythroid (BFU-E). **, P < 0.01. (F) PB counts from mice transplanted with WT, Tifab KO, miR-146a KO, and dKO BM cells at 4 mo after transplant (n > 8 per group). Two independent transplantations experiments were performed. NE, neutrophils; LY, lymphocytes; and RBC, red blood cells. Error bars are mean ± SEM values. *, P < 0.05; **, P < 0.01; ***, P < 0.001, Student’s t test.To assess whether TIFAB and miR-146a cooperate to regulate hematopoietic progenitor differentiation in vitro, BM HSPCs isolated from WT, Tifab−/−, miR-146a−/−, or dKO mice were examined for progenitor colony formation in methylcellulose. Total colony formation was increased in dKO as compared with the single KO mice (Fig. 7 D). In addition, combined deletion of TIFAB and miR-146a resulted in an additive increase in CFU-G colony formation (P = 0.007), underscoring a functional cooperation between TIFAB and miR-146a during myeloid differentiation (Fig. 7 E). To determine whether combined deletion of TIFAB and miR-146a accentuates the cytopenia in vivo, BM cells from WT, Tifab−/−, miR-146−/−, and dKO mice were transplanted into lethally radiated syngeneic recipient mice. Mice reconstituted with Tifab-deficient BM exhibited a decline in peripheral neutrophils (2.1 vs. 3.0 K/µl; P = 0.014), lymphocytes (10.5 vs. 13.0 K/µl; P < 0.006), and RBCs (9.5 vs. 10.5 K/µl; P = 0.01) by 4 mo. Deletion of miR-146a had only minor effects on neutrophil (P = 0.08), lymphocyte (P = 0.45), or RBC (P = 0.012) counts (Fig. 7 F). In contrast, combined deletion of TIFAB and miR-146a resulted in a cooperative decrease in peripheral neutrophil (1.3 vs. 3.0 K/µl; P < 0.0001) and lymphocyte (7.5 vs. 13.0 K/µl; P < 0.0001) production, while not contributing further to RBC defects during the same time period (Fig. 7 F). Collectively, these findings suggest that two 5q genes, miR-146a and TIFAB, cooperate to regulate TRAF6 expression, whereas loss of both genes accentuates hematopoietic myeloid differentiation defects in vitro and in vivo.

TIFAB and miR-146a cooperate to regulate TRAF6 expression and myeloid cell differentiation

Human miR-146a and TIFAB both reside on chr 5q and are co-deleted in ∼80% of del(5q) MDS and AML (Gondek et al., 2008). miR-146a−/− mice develop BMF and myeloid leukemia, in part because of derepression of TRAF6 mRNA and protein (Lu et al., 2010; Boldin et al., 2011; Zhao et al., 2011). Given that TIFAB and miR-146a are codeleted in MDS/AML, we examined whether there is a functional cooperation between TIFAB and miR-146a. miR-146a inhibits TRAF6 protein expression and reduces its mRNA levels, whereas herein we report that TIFAB inhibits TRAF6 by destabilizing its protein. Because miR-146a is an NF-κB target gene responsive to LPS, miR-146a expression was investigated in TIFAB-deficient BM cells. At basal levels and after LPS stimulation, miR-146a expression is significantly elevated in Tifab−/− BM MNC (2-fold and 15-fold at baseline and after stimulation, respectively; Fig. 7 A). Although TRAF6 protein is stabilized after BM transplantation in Tifab−/− BM cells (Fig. 6 D), Traf6 mRNA is significantly reduced in Tifab−/− BM cells as compared with WT cells (Fig. 6 F and Fig. 7 B), suggesting that increased miR-146a expression in Tifab-deficient cells may suppress TRAF6 mRNA. To examine a cooperative effect of TIFAB and miR-146a deficiency on TRAF6 mRNA and protein expression, we generated double Tifab- and miR-146a–deficient mice. Combined deletion of Tifab and miR-146a restored TRAF6 mRNA expression to WT levels (Fig. 7 B). In addition, TRAF6 protein expression was synergistically increased in BM MNC from Tifab−/−;miR-146a−/− mice (dKO) after BM transplantation as compared with single-knockout BM cells (Fig. 7 C). Collectively, miR-146a and TIFAB cooperate to suppress TRAF6 mRNA and protein expression in BM HSPCs.

TIFAB and miR-146a cooperate to regulate TRAF6 and hematopoietic function. (A) Expression analysis of miR-146a in WT and KO BM MNC by qRT-PCR after stimulation with LPS (100 ng/ml) for the indicated time points. Expression was normalized to time 0 (1.0). *, P < 0.05; Student’s t test. (B) Expression analysis of Traf6 mRNA by qRT-PCR in WT, Tifab KO, miR-146a KO, and double KO (dKO) BM cells. *, P < 0.05; Student’s t test. (C) Immunoblot analysis for TRAF6 on WT, Tifab KO, miR-146a KO, and dKO BM cells from transplanted mice. (D) Total colony-forming cells in methylcellulose from BM MNC from WT, Tifab KO, miR-146a KO, and dKO BM (n = 5–6; two independent experiments). *, P < 0.05; Student’s t test. (E) Proportion of colonies formed from D. GM, granulocyte-monocyte; G, granulocyte; M, monocyte; E, erythroid (BFU-E). **, P < 0.01. (F) PB counts from mice transplanted with WT, Tifab KO, miR-146a KO, and dKO BM cells at 4 mo after transplant (n > 8 per group). Two independent transplantations experiments were performed. NE, neutrophils; LY, lymphocytes; and RBC, red blood cells. Error bars are mean ± SEM values. *, P < 0.05; **, P < 0.01; ***, P < 0.001, Student’s t test.

To assess whether TIFAB and miR-146a cooperate to regulate hematopoietic progenitor differentiation in vitro, BM HSPCs isolated from WT, Tifab−/−, miR-146a−/−, or dKO mice were examined for progenitor colony formation in methylcellulose. Total colony formation was increased in dKO as compared with the single KO mice (Fig. 7 D). In addition, combined deletion of TIFAB and miR-146a resulted in an additive increase in CFU-G colony formation (P = 0.007), underscoring a functional cooperation between TIFAB and miR-146a during myeloid differentiation (Fig. 7 E). To determine whether combined deletion of TIFAB and miR-146a accentuates the cytopenia in vivo, BM cells from WT, Tifab−/−, miR-146−/−, and dKO mice were transplanted into lethally radiated syngeneic recipient mice. Mice reconstituted with Tifab-deficient BM exhibited a decline in peripheral neutrophils (2.1 vs. 3.0 K/µl; P = 0.014), lymphocytes (10.5 vs. 13.0 K/µl; P < 0.006), and RBCs (9.5 vs. 10.5 K/µl; P = 0.01) by 4 mo. Deletion of miR-146a had only minor effects on neutrophil (P = 0.08), lymphocyte (P = 0.45), or RBC (P = 0.012) counts (Fig. 7 F). In contrast, combined deletion of TIFAB and miR-146a resulted in a cooperative decrease in peripheral neutrophil (1.3 vs. 3.0 K/µl; P < 0.0001) and lymphocyte (7.5 vs. 13.0 K/µl; P < 0.0001) production, while not contributing further to RBC defects during the same time period (Fig. 7 F). Collectively, these findings suggest that two 5q genes, miR-146a and TIFAB, cooperate to regulate TRAF6 expression, whereas loss of both genes accentuates hematopoietic myeloid differentiation defects in vitro and in vivo.

Regulation of TRAF6 is dependent on the C-terminal domain of TIFABTIFAB is a 131 aa protein consisting of a conserved 55-aa forkhead-associated (FHA) domain (aa 36–91), whereas the residues flanking the FHA domain do not contain conserved motifs. To determine whether the FHA domain is required for regulating TRAF6 protein stability and signaling, we created deletion mutants of TIFAB. Three key mutants were created that removed the FHA domain (Δ35–91), the N-terminal domain (Δ1–34), or the C-terminal domain (Δ92–161). Additional TIFAB mutants were generated that deleted a combination of the aforementioned domains (Δ1–91 and Δ35–161; Fig. 8 A). The effects of TIFAB mutants on basal NF-κB activation were measured by κB-site reporter assays. Overexpression of TIFAB repressed basal κB-luciferase activation by ∼30% (P = 0.01; Fig. 8 B). Of the TIFAB deletion mutants, only Δ1–91 mutant retained its ability to repress κB-luciferase activation (P = 1.7 × 10−4; Fig. 8 B). We next evaluated the effects of the TIFAB mutants on TRAF6-mediated NF-κB activation. Overexpression of TRAF6 results in ∼40-fold increase in κB-luciferase activation (Fig. 8 C). Co-expression of full-length TIFAB resulted in >90-fold repression of TRAF6-mediated κB-luciferase activation (P < 0.01; Fig. 8 C). Surprisingly, deletion of the FHA domain (Δ35–91) still repressed TRAF6-mediated κB-luciferase activation and reduced TRAF6 protein levels (Fig. 8, C and D), suggesting that the FHA domain is not essential for inhibiting TRAF6-induced NF-κB signaling. Consistent with the effects on basal NF-κB activation, the Δ1–91 mutant also inhibited TRAF6-mediated NF-κB activation (P = 0.002; Fig. 8 C). To identify the essential residues within aa 92–161 of the Δ1–90 mutants that are responsible for inhibiting TRAF6-mediated NF-κB activation, we created mutants that eliminated a portion of the C-terminal domain; Δ126–161 is missing the distal region of the C-terminal domain, and Δ91-126 is missing the proximal region of the C-terminal domain (Fig. 8 A). Both mutants retained their inhibitory effects on NF-κB, suggesting that multiple regions within this domain are essential for TIFAB function. In parallel, we also examined the effects of the TIFAB mutants on TRAF6 protein stability. Congruent with the NF-κB reporter assays, TIFAB mutant Δ1–91 suppressed TRAF6 protein nearly as efficiently as full-length TIFAB, strongly indicating that <70 aa within the C-terminal domain of TIFAB elicit TRAF6 degradation (Fig. 8, A and D).Figure 8.Inhibition of TRAF6 and NF-κB signaling depends on the TIFAB C-terminal domain and is independent of the FHA domain. (A) Overview of WT TIFAB and various TIFAB deletion mutants. Summary of NF-κB activation in HEK293 cells after expression of TIFAB mutants under basal (B) and TRAF6-induced conditions (C). Summary of TRAF6 protein expression in HEK293 cells after expression of TIFAB mutants (D). FHA, fork-head associated domain; FLAG, FLAG motif cloned at the N-terminal region of TIFAB mutants; nd, not determined. −, repression; +, normal levels; +/−, moderate repression. (B) NF-κB activation was measured by a κB-site containing reporter assays in HEK293 cells transfected with empty vector, TIFAB, or the indicated TIFAB deletion mutants. Values are normalized to Renilla-luciferase and empty vector (1.0). *, P < 0.05; Student’s t test. (C) NF-κB activation was measured by κB-site containing reporter assays in HEK293 cells transfected with empty vector, TRAF6, and TIFAB or the indicated TIFAB deletion mutants. Values are normalized to Renilla-luciferase and empty vector (1.0). *, P < 0.05; **, P < 0.01; ***, P < 0.001; Student’s t test. (D) Immunoblot analysis of HEK293 transfected with empty vector, TRAF6, and TIFAB or TIFAB deletion mutants. Error bars are mean ± SEM values.

Regulation of TRAF6 is dependent on the C-terminal domain of TIFAB

TIFAB is a 131 aa protein consisting of a conserved 55-aa forkhead-associated (FHA) domain (aa 36–91), whereas the residues flanking the FHA domain do not contain conserved motifs. To determine whether the FHA domain is required for regulating TRAF6 protein stability and signaling, we created deletion mutants of TIFAB. Three key mutants were created that removed the FHA domain (Δ35–91), the N-terminal domain (Δ1–34), or the C-terminal domain (Δ92–161). Additional TIFAB mutants were generated that deleted a combination of the aforementioned domains (Δ1–91 and Δ35–161; Fig. 8 A). The effects of TIFAB mutants on basal NF-κB activation were measured by κB-site reporter assays. Overexpression of TIFAB repressed basal κB-luciferase activation by ∼30% (P = 0.01; Fig. 8 B). Of the TIFAB deletion mutants, only Δ1–91 mutant retained its ability to repress κB-luciferase activation (P = 1.7 × 10−4; Fig. 8 B). We next evaluated the effects of the TIFAB mutants on TRAF6-mediated NF-κB activation. Overexpression of TRAF6 results in ∼40-fold increase in κB-luciferase activation (Fig. 8 C). Co-expression of full-length TIFAB resulted in >90-fold repression of TRAF6-mediated κB-luciferase activation (P < 0.01; Fig. 8 C). Surprisingly, deletion of the FHA domain (Δ35–91) still repressed TRAF6-mediated κB-luciferase activation and reduced TRAF6 protein levels (Fig. 8, C and D), suggesting that the FHA domain is not essential for inhibiting TRAF6-induced NF-κB signaling. Consistent with the effects on basal NF-κB activation, the Δ1–91 mutant also inhibited TRAF6-mediated NF-κB activation (P = 0.002; Fig. 8 C). To identify the essential residues within aa 92–161 of the Δ1–90 mutants that are responsible for inhibiting TRAF6-mediated NF-κB activation, we created mutants that eliminated a portion of the C-terminal domain; Δ126–161 is missing the distal region of the C-terminal domain, and Δ91-126 is missing the proximal region of the C-terminal domain (Fig. 8 A). Both mutants retained their inhibitory effects on NF-κB, suggesting that multiple regions within this domain are essential for TIFAB function. In parallel, we also examined the effects of the TIFAB mutants on TRAF6 protein stability. Congruent with the NF-κB reporter assays, TIFAB mutant Δ1–91 suppressed TRAF6 protein nearly as efficiently as full-length TIFAB, strongly indicating that <70 aa within the C-terminal domain of TIFAB elicit TRAF6 degradation (Fig. 8, A and D).

Inhibition of TRAF6 and NF-κB signaling depends on the TIFAB C-terminal domain and is independent of the FHA domain. (A) Overview of WT TIFAB and various TIFAB deletion mutants. Summary of NF-κB activation in HEK293 cells after expression of TIFAB mutants under basal (B) and TRAF6-induced conditions (C). Summary of TRAF6 protein expression in HEK293 cells after expression of TIFAB mutants (D). FHA, fork-head associated domain; FLAG, FLAG motif cloned at the N-terminal region of TIFAB mutants; nd, not determined. −, repression; +, normal levels; +/−, moderate repression. (B) NF-κB activation was measured by a κB-site containing reporter assays in HEK293 cells transfected with empty vector, TIFAB, or the indicated TIFAB deletion mutants. Values are normalized to Renilla-luciferase and empty vector (1.0). *, P < 0.05; Student’s t test. (C) NF-κB activation was measured by κB-site containing reporter assays in HEK293 cells transfected with empty vector, TRAF6, and TIFAB or the indicated TIFAB deletion mutants. Values are normalized to Renilla-luciferase and empty vector (1.0). *, P < 0.05; **, P < 0.01; ***, P < 0.001; Student’s t test. (D) Immunoblot analysis of HEK293 transfected with empty vector, TRAF6, and TIFAB or TIFAB deletion mutants. Error bars are mean ± SEM values.

Restoring TIFAB expression in MDS/AML cells induces apoptosis and impairs leukemic functionTo establish the relevance of TIFAB in MDS and AML, TIFAB was reexpressed in three human MDS/AML cell lines (MDSL, HL60, and THP1) with low endogenous TIFAB, and assessed for leukemic progenitor function and survival. The level of TIFAB reexpression in the indicated cell lines was consistent with physiological TIFAB expression in primary normal and del(5q) AML BM MNC (Fig. 9 A). Ectopic expression of TIFAB in all three cell lines significantly reduced leukemic colony formation (Fig. 9 B) and survival (Fig. 9, C and D). To examine the cellular effects of restoring just the minimal TIFAB domain that retains TRAF6 inhibition in disease-relevant cells, we transduced the three MDS/AML cell lines with Δ1-91–expressing vectors. As a negative control, Δ92–161 was also expressed, as this mutant has lost its TRAF6/NF-κB inhibitory function, yet it still retains the FHA domain (Fig. 8 A). Restoring only the C-terminal domain (Δ1–91) suppressed MDS/AML leukemic progenitor function at a level comparable to WT TIFAB (Fig. 9 B). In stark contrast, Δ92–161 expression did not inhibit MDS/AML cell colony formation. Thus, restoring only the C-terminal domain of TIFAB is sufficient to destabilize TRAF6 protein, suppress NF-κB activation, and impair MDS/AML cell function.Figure 9.TIFAB reexpression suppresses MDS/AML cells. (A) Relative TIFAB mRNA expression determined by qRT-PCR from normal and del(5q) AML patient BM MNC, and transduced cell lines with empty vector or FLAG-TIFAB. (B) Colony formation in methylcellulose of MDSL, HL60, and THP1 cells transduced with empty vector, FLAG-TIFAB, FLAG-TIFABΔ1-91, FLAG-TIFABΔ92-161 was measured after 7–9 d. *, P < 0.05; **, P < 0.01; Student’s t test. (C) Representative Annexin V staining for MDSL, HL60, and THP1 cells transduced with empty vector or FLAG-TIFAB (top). (D) The corresponding summary for Annexin V staining is shown for more than two independent experiments (bottom). *, P < 0.05; Student’s t test. (E) MDSL cells transduced with vector or TIFAB were engrafted into NSGS mice (n = 6/group) and monitored for BM chimerism at the indicated time points. BM engraftment is shown for individual mice transplanted with vector (#1 and #3)-MDSL and TIFAB (#7 and #8)-MDSL cells at 4 and 8–12 wk (human versus mouse CD45). Percent of human cell (MDSL) engraftment is shown for the representative mice. (F) Summary of BM engraftment for mice transplanted with vector- and TIFAB-MDSL cells (n > 5 mice per group). *, P < 0.05; Student’s t test.To examine the effects of TIFAB reexpression on MDS/AML cell function in vivo, MDSL cells were transduced with TIFAB or empty vector (vector) and then xenografted into NSGS mice. Vector- and TIFAB-expressing MDSL cells efficiently engrafted into the BM of recipient mice within 4 wk (Fig. 9, E and F). However, beyond 8 wk after transplant, the BM engraftment of TIFAB-expressing MDSL cells dramatically diminished (P = 0.04), whereas control MDSL cells increased their engraftment during this time (Fig. 9, E and F). Thus, restoring TIFAB expression in MDS/AML cells induces apoptosis and impairs disease-propagating function in vitro and in vivo.

Restoring TIFAB expression in MDS/AML cells induces apoptosis and impairs leukemic function

To establish the relevance of TIFAB in MDS and AML, TIFAB was reexpressed in three human MDS/AML cell lines (MDSL, HL60, and THP1) with low endogenous TIFAB, and assessed for leukemic progenitor function and survival. The level of TIFAB reexpression in the indicated cell lines was consistent with physiological TIFAB expression in primary normal and del(5q) AML BM MNC (Fig. 9 A). Ectopic expression of TIFAB in all three cell lines significantly reduced leukemic colony formation (Fig. 9 B) and survival (Fig. 9, C and D). To examine the cellular effects of restoring just the minimal TIFAB domain that retains TRAF6 inhibition in disease-relevant cells, we transduced the three MDS/AML cell lines with Δ1-91–expressing vectors. As a negative control, Δ92–161 was also expressed, as this mutant has lost its TRAF6/NF-κB inhibitory function, yet it still retains the FHA domain (Fig. 8 A). Restoring only the C-terminal domain (Δ1–91) suppressed MDS/AML leukemic progenitor function at a level comparable to WT TIFAB (Fig. 9 B). In stark contrast, Δ92–161 expression did not inhibit MDS/AML cell colony formation. Thus, restoring only the C-terminal domain of TIFAB is sufficient to destabilize TRAF6 protein, suppress NF-κB activation, and impair MDS/AML cell function.

TIFAB reexpression suppresses MDS/AML cells. (A) Relative TIFAB mRNA expression determined by qRT-PCR from normal and del(5q) AML patient BM MNC, and transduced cell lines with empty vector or FLAG-TIFAB. (B) Colony formation in methylcellulose of MDSL, HL60, and THP1 cells transduced with empty vector, FLAG-TIFAB, FLAG-TIFABΔ1-91, FLAG-TIFABΔ92-161 was measured after 7–9 d. *, P < 0.05; **, P < 0.01; Student’s t test. (C) Representative Annexin V staining for MDSL, HL60, and THP1 cells transduced with empty vector or FLAG-TIFAB (top). (D) The corresponding summary for Annexin V staining is shown for more than two independent experiments (bottom). *, P < 0.05; Student’s t test. (E) MDSL cells transduced with vector or TIFAB were engrafted into NSGS mice (n = 6/group) and monitored for BM chimerism at the indicated time points. BM engraftment is shown for individual mice transplanted with vector (#1 and #3)-MDSL and TIFAB (#7 and #8)-MDSL cells at 4 and 8–12 wk (human versus mouse CD45). Percent of human cell (MDSL) engraftment is shown for the representative mice. (F) Summary of BM engraftment for mice transplanted with vector- and TIFAB-MDSL cells (n > 5 mice per group). *, P < 0.05; Student’s t test.

To examine the effects of TIFAB reexpression on MDS/AML cell function in vivo, MDSL cells were transduced with TIFAB or empty vector (vector) and then xenografted into NSGS mice. Vector- and TIFAB-expressing MDSL cells efficiently engrafted into the BM of recipient mice within 4 wk (Fig. 9, E and F). However, beyond 8 wk after transplant, the BM engraftment of TIFAB-expressing MDSL cells dramatically diminished (P = 0.04), whereas control MDSL cells increased their engraftment during this time (Fig. 9, E and F). Thus, restoring TIFAB expression in MDS/AML cells induces apoptosis and impairs disease-propagating function in vitro and in vivo.

DISCUSSIONIdentification and investigation of relevant del(5q) genes is vital to advancing our knowledge and treatment approach for del(5q) MDS/AML. Herein, we describe a TRAF6-interacting protein, TIFAB, which resides within the minimally deleted segment on 5q31.1. Hematopoietic-specific deletion of Tifab results in hematopoietic defects, including skewed HSPC proportions, altered myeloid differentiation, and cytopenia. A subset of mice transplanted with Tifab-deficient hematopoietic cells develop a BMF-like disease associated with BM dysplasia and cytopenia, which occurs through cell-intrinsic defects in HSPCs. Gene expression analysis of Tifab-deficient HSCs identified dysregulation of HSC- and immune-related gene signatures, and hypersensitivity to immune pathway stimulation downstream of TLR4. As therapeutic insight, reexpression of TIFAB results in an attenuated immune response and impaired del(5q) leukemic cell function.Loss of TIFAB in HSPCs increases TRAF6 protein stability, which results in altered innate immune responsiveness and hematopoietic dysfunction. Evidence that TRAF6 is associated with MDS came from work related to miR-146a (Starczynowski et al., 2010, 2011). TRAF6 and IRAK1 are two key targets of miR-146a. In mouse HSPCs, overexpression of miR-146a results in reduced endogenous TRAF6 and IRAK1 protein, and conversely, knockdown of miR-146a results in derepression of TRAF6 and IRAK1 protein. Human miR-146a also resides on chr 5q and is deleted in ∼80% of del(5q) MDS and AML (Gondek et al., 2008). miR-146a−/− mice develop early onset myeloid expansion in the BM, and progress to more aggressive diseases such as lymphomas, BMF, and myeloid leukemia (Lu et al., 2010; Boldin et al., 2011; Zhao et al., 2011), in part due to derepression of TRAF6 and IRAK1. Given that TIFAB and miR-146a reside within the deleted region on chr 5q and are co-deleted in the majority of del(5q) MDS/AML patients, loss of TIFAB and miR-146a may cooperate to induce TRAF6 signaling. Deletion of miR-146a increases translation of TRAF6, whereas loss of TIFAB increases TRAF6 protein stability. Consistent with these previous observations, our combined deletion of TIFAB and miR-146a resulted in an additive increase in TRAF6 protein expression and subsequent innate immune signaling. Moreover, transplanted Tifab−/−;miR-146a−/− BM cells exhibit worse cytopenias as compared with the individual gene knockout mice. In del(5q) MDS and AML, miR-146a deficiency also results in overexpression of a neighboring haploid gene, SQSTM1 (p62), from the intact 5q allele to sustain TRAF6-mediated activation of NF-κB, and leukemic cell survival (Fang et al., 2014). This recent finding provides evidence that myeloid malignancies with del(5q) acquire a partial dependency on TRAF6 activation.DIAPH1, which encodes mDia1, is located within the deleted segment on chr 5q and also regulates innate immune signaling. mDia1 belongs to the formin protein family and plays an important role in linear actin polymerization (Tominaga et al., 2000). mDia1-deficient cells express high levels of CD14, the co-receptor for the TLR4 and MD-2 complex, which detects LPS and damage-associated molecular pattern molecules (DAMP). mDia1-deficient mice exhibit an age-dependent granulocytopenia, and myeloid dysplasia, in part through increased TLR4 signaling. This underscores the connection between mDia1 and innate immune signaling in del(5q) MDS. Collectively, at least three verified haploinsufficient genes on chr 5q (miR-146a, TIFAB, and DIAPH1) regulate innate immune signaling and contribute to aspects of del(5q) MDS.Mounting evidence also implicates increased innate immune signaling within HSPCs in the pathogenesis of nondel(5q) MDS. Chronic immune stimulation increases the risk for MDS and AML (Kristinsson et al., 2011). Overexpression of IL1RAP and TLRs in MDS mediates the activation of TRAF6 and IRAK1 (Barreyro et al., 2012; Wei et al., 2012). Somatic activating mutation variants of TLR2 in MDS mediate hyperactivation of IRAK1 and NF-κB (Wei et al., 2012). Through noncell autonomous mechanisms, expansion of myeloid derived suppressor cells expressing S100A9 induces MDS in mice by activating TLR signaling (Chen et al., 2013). Lastly, the importance of TRAF6 in primary MDS is described in our recent publication, in which genetic and pharmacologic inhibition of IRAK1 suppresses TRAF6 activation and the MDS clone (Rhyasen et al., 2013a,b). Although NF-κB is one pathway activated by the innate immune complex, signaling networks regulated by TRAF6 in MDS and overt leukemia may be distinct, particularly as they relate to disease initiation versus maintenance. Further investigation into deciphering the importance of TRAF6-dependent NF-κB and non-NF-κB signaling networks during disease evolution and within distinct MDS/AML subtypes is warranted. Notably, constitutive NF-κB activation is not sufficient to induce MDS in mice (Beg et al., 1995; Rupec et al., 2005), suggesting additional pathways downstream of TRAF6 contribute to MDS initiation, whereas NF-κB may contribute to the maintenance of the disease (Breccia and Alimena, 2010).TIFA was first identified as a TRAF6-interacting protein in a yeast two-hybrid screen (Kanamori et al., 2002; Takatsuna et al., 2003). TIFA interacts with TRAF6 through a consensus TRAF6-binding motif (Takatsuna et al., 2003) to promote TRAF6 oligomerization and subsequent activation of NF-κB (Ea et al., 2004). A mutation in TIFA that prevents binding to TRAF6 abolishes its ability to induce TRAF6 oligomerization and ubiquitination, indicating that this process requires direct interaction between TIFA and TRAF6 (Ea et al., 2004). TIFAB was identified as a TIFA-related protein by an in silico homology screen (Matsumura et al., 2004, 2009). Consistent with TIFA, we show by coimmunoprecipitation experiments that TIFAB and TRAF6 also form a complex in cells, and that this interaction primarily occurs in the cytoplasm. However, in contrast to the positive regulation of TIFA on TRAF6/NF-κB, herein we provide evidence that TIFAB is a negative regulator of innate immune signaling in HSPCs by inducing TRAF6 protein degradation through a lysosome-dependent mechanism. Regulation of TRAF proteins by autophagy/lysosomes has been previously described. Recently, we have reported that proteosome inhibition induces autophagy/lysosome-dependent degradation of TRAF6 protein in MDS and AML cells (Fang et al., 2012). In this study, treatment with proteasome inhibitors reduced TRAF6 protein, which coincided with apoptosis of MDS/AML cells. TRAF6 is also degraded by autophagy in cells treated with double-stranded RNA, which can be blocked by addition of 3-MA (Inomata et al., 2012). Collectively, TRAF6 protein stability is controlled by lysosome-mediated degradation, in part through a TIFAB-dependent mechanism.Both TIFA and TIFAB contain a FHA domain, a phosphopeptide recognition motif associated with DNA damage, signal transduction, vesicular transport, and protein degradation (Mahajan et al., 2008). Interestingly, deletion of the TIFAB FHA domain was not sufficient to prevent TIFAB-mediated degradation of TRAF6 or inhibition of NF-κB signaling. Thus it is possible that TIFAB is involved in several molecular functions: some that depend on its FHA domain, whereas others do not. In addition to changes in immune-related gene signatures, dysregulation of p53 gene signatures was also observed in Tifab−/− HSPCs. TIFAB may regulate both p53- and immune-related pathways as the TIFAB-deficient phenotype occurs primarily after hematopoietic stress. In conclusion, TIFAB is not only associated in the pathogenesis of del(5q) MDS/AML but also in regulating TLR-mediated signaling during inflammation.

Identification and investigation of relevant del(5q) genes is vital to advancing our knowledge and treatment approach for del(5q) MDS/AML. Herein, we describe a TRAF6-interacting protein, TIFAB, which resides within the minimally deleted segment on 5q31.1. Hematopoietic-specific deletion of Tifab results in hematopoietic defects, including skewed HSPC proportions, altered myeloid differentiation, and cytopenia. A subset of mice transplanted with Tifab-deficient hematopoietic cells develop a BMF-like disease associated with BM dysplasia and cytopenia, which occurs through cell-intrinsic defects in HSPCs. Gene expression analysis of Tifab-deficient HSCs identified dysregulation of HSC- and immune-related gene signatures, and hypersensitivity to immune pathway stimulation downstream of TLR4. As therapeutic insight, reexpression of TIFAB results in an attenuated immune response and impaired del(5q) leukemic cell function.

Loss of TIFAB in HSPCs increases TRAF6 protein stability, which results in altered innate immune responsiveness and hematopoietic dysfunction. Evidence that TRAF6 is associated with MDS came from work related to miR-146a (Starczynowski et al., 2010, 2011). TRAF6 and IRAK1 are two key targets of miR-146a. In mouse HSPCs, overexpression of miR-146a results in reduced endogenous TRAF6 and IRAK1 protein, and conversely, knockdown of miR-146a results in derepression of TRAF6 and IRAK1 protein. Human miR-146a also resides on chr 5q and is deleted in ∼80% of del(5q) MDS and AML (Gondek et al., 2008). miR-146a−/− mice develop early onset myeloid expansion in the BM, and progress to more aggressive diseases such as lymphomas, BMF, and myeloid leukemia (Lu et al., 2010; Boldin et al., 2011; Zhao et al., 2011), in part due to derepression of TRAF6 and IRAK1. Given that TIFAB and miR-146a reside within the deleted region on chr 5q and are co-deleted in the majority of del(5q) MDS/AML patients, loss of TIFAB and miR-146a may cooperate to induce TRAF6 signaling. Deletion of miR-146a increases translation of TRAF6, whereas loss of TIFAB increases TRAF6 protein stability. Consistent with these previous observations, our combined deletion of TIFAB and miR-146a resulted in an additive increase in TRAF6 protein expression and subsequent innate immune signaling. Moreover, transplanted Tifab−/−;miR-146a−/− BM cells exhibit worse cytopenias as compared with the individual gene knockout mice. In del(5q) MDS and AML, miR-146a deficiency also results in overexpression of a neighboring haploid gene, SQSTM1 (p62), from the intact 5q allele to sustain TRAF6-mediated activation of NF-κB, and leukemic cell survival (Fang et al., 2014). This recent finding provides evidence that myeloid malignancies with del(5q) acquire a partial dependency on TRAF6 activation.

DIAPH1, which encodes mDia1, is located within the deleted segment on chr 5q and also regulates innate immune signaling. mDia1 belongs to the formin protein family and plays an important role in linear actin polymerization (Tominaga et al., 2000). mDia1-deficient cells express high levels of CD14, the co-receptor for the TLR4 and MD-2 complex, which detects LPS and damage-associated molecular pattern molecules (DAMP). mDia1-deficient mice exhibit an age-dependent granulocytopenia, and myeloid dysplasia, in part through increased TLR4 signaling. This underscores the connection between mDia1 and innate immune signaling in del(5q) MDS. Collectively, at least three verified haploinsufficient genes on chr 5q (miR-146a, TIFAB, and DIAPH1) regulate innate immune signaling and contribute to aspects of del(5q) MDS.

Mounting evidence also implicates increased innate immune signaling within HSPCs in the pathogenesis of nondel(5q) MDS. Chronic immune stimulation increases the risk for MDS and AML (Kristinsson et al., 2011). Overexpression of IL1RAP and TLRs in MDS mediates the activation of TRAF6 and IRAK1 (Barreyro et al., 2012; Wei et al., 2012). Somatic activating mutation variants of TLR2 in MDS mediate hyperactivation of IRAK1 and NF-κB (Wei et al., 2012). Through noncell autonomous mechanisms, expansion of myeloid derived suppressor cells expressing S100A9 induces MDS in mice by activating TLR signaling (Chen et al., 2013). Lastly, the importance of TRAF6 in primary MDS is described in our recent publication, in which genetic and pharmacologic inhibition of IRAK1 suppresses TRAF6 activation and the MDS clone (Rhyasen et al., 2013a,b). Although NF-κB is one pathway activated by the innate immune complex, signaling networks regulated by TRAF6 in MDS and overt leukemia may be distinct, particularly as they relate to disease initiation versus maintenance. Further investigation into deciphering the importance of TRAF6-dependent NF-κB and non-NF-κB signaling networks during disease evolution and within distinct MDS/AML subtypes is warranted. Notably, constitutive NF-κB activation is not sufficient to induce MDS in mice (Beg et al., 1995; Rupec et al., 2005), suggesting additional pathways downstream of TRAF6 contribute to MDS initiation, whereas NF-κB may contribute to the maintenance of the disease (Breccia and Alimena, 2010).

TIFA was first identified as a TRAF6-interacting protein in a yeast two-hybrid screen (Kanamori et al., 2002; Takatsuna et al., 2003). TIFA interacts with TRAF6 through a consensus TRAF6-binding motif (Takatsuna et al., 2003) to promote TRAF6 oligomerization and subsequent activation of NF-κB (Ea et al., 2004). A mutation in TIFA that prevents binding to TRAF6 abolishes its ability to induce TRAF6 oligomerization and ubiquitination, indicating that this process requires direct interaction between TIFA and TRAF6 (Ea et al., 2004). TIFAB was identified as a TIFA-related protein by an in silico homology screen (Matsumura et al., 2004, 2009). Consistent with TIFA, we show by coimmunoprecipitation experiments that TIFAB and TRAF6 also form a complex in cells, and that this interaction primarily occurs in the cytoplasm. However, in contrast to the positive regulation of TIFA on TRAF6/NF-κB, herein we provide evidence that TIFAB is a negative regulator of innate immune signaling in HSPCs by inducing TRAF6 protein degradation through a lysosome-dependent mechanism. Regulation of TRAF proteins by autophagy/lysosomes has been previously described. Recently, we have reported that proteosome inhibition induces autophagy/lysosome-dependent degradation of TRAF6 protein in MDS and AML cells (Fang et al., 2012). In this study, treatment with proteasome inhibitors reduced TRAF6 protein, which coincided with apoptosis of MDS/AML cells. TRAF6 is also degraded by autophagy in cells treated with double-stranded RNA, which can be blocked by addition of 3-MA (Inomata et al., 2012). Collectively, TRAF6 protein stability is controlled by lysosome-mediated degradation, in part through a TIFAB-dependent mechanism.

Both TIFA and TIFAB contain a FHA domain, a phosphopeptide recognition motif associated with DNA damage, signal transduction, vesicular transport, and protein degradation (Mahajan et al., 2008). Interestingly, deletion of the TIFAB FHA domain was not sufficient to prevent TIFAB-mediated degradation of TRAF6 or inhibition of NF-κB signaling. Thus it is possible that TIFAB is involved in several molecular functions: some that depend on its FHA domain, whereas others do not. In addition to changes in immune-related gene signatures, dysregulation of p53 gene signatures was also observed in Tifab−/− HSPCs. TIFAB may regulate both p53- and immune-related pathways as the TIFAB-deficient phenotype occurs primarily after hematopoietic stress. In conclusion, TIFAB is not only associated in the pathogenesis of del(5q) MDS/AML but also in regulating TLR-mediated signaling during inflammation.

MATERIALS AND METHODSKnockout mice.A target vector was designed to replace exon 3, which contains all coding region of Tifab, with a β-galactosidase-loxP-neomycin resistant gene cassette-loxP fragment. Diphtheria toxin A gene cassette was inserted at the downstream end of the short arm. The targeting construct was electroporated into a 129-derived embryonic stem (ES) cell line. G418-resistant ES clones were screened for homologous recombination by Southern blot analysis. ES clones containing the correctly targeted Tifab locus were injected into C57BL/6 blastocysts. Male chimeric mice were mated with C57BL/6 J, and agouti pups carrying the correct target locus were used as F1 mice. F1 mice were crossed with transgenic mice expressing CRE under control of CAG promoter (Sakai and Miyazaki, 1997). Pups lacking neomycin-resistant gene cassette were further backcrossed into the C57BL/6 J background. miR-146a−/− C57BL/6 mice were obtained from D. Baltimore (California Institute of Technology, Pasadena, CA) as previously described (Fang et al., 2014). Animals were bred and housed in the Association for Assessment and Accreditation of Laboratory Animal Care-accredited animal facility of Cincinnati Children’s Hospital Medical Center.Bone marrow transplantation.For noncompetitive BM transplantation, BM mononuclear cells (0.3–3 × 106) from C7BL/6 Tifab+/+, Tifab+/−, Tifab−/−, miR-146a−/−, or Tifab−/−;miR-146a−/− mice were injected i.v. into lethally radiated (7.0 Gy and 4.75 Gy after 3 h) syngeneic recipient Boy/J mice. For competitive BM transplants, BM mononuclear cells (3 × 105) from 8-wk-old CD45.2 Tifab+/+, Tifab+/−, or Tifab−/− mice were mixed with equal numbers of 8-wk-old BM mononuclear cells from BoyJ (CD45.1) mice and then injected i.v. into lethally irradiated (7.0 Gy and 4.75 Gy after 3 h) CD45.1 recipient BoyJ mice.BM and PB analysis.For cytospins, 2.5–5 × 105 BM cells were spun onto slide at 300 rpm for 5 min, and then stained with Wright-Giemsa. For BM sections, tibia were fixed in formalin and then stained with hematoxylin and eosin. Complete blood counts were performed on PB isolated from the tail vein every 4 wk and analyzed using the Drew Scientific Hemavet 950.BM stimulation with LPS and TNF.BM mononuclear cells were harvested from Tifab+/+ and Tifab−/− mice and cultured in the presence of vehicle control (dH2O for LPS or 0.1% BSA in 1X PBS for TNF), ultrapure LPS (100 ng/ml; Invitrogen), or TNF (10 ng/ml; Gibco) for 0, 1, 4, and 8 h. RNA extraction, cDNA conversion, and qRT-PCR analysis were performed as described below.Quantitative PCR analysis.RNA was extracted from cells and purified with TRIzol Reagent (Life Technologies) or Quick-RNA MiniPrep kit (Zymo Research). cDNA was generated using a high capacity RNA to cDNA kit (Life Technologies 4387406 or 4368814). qPCR was performed using TaqMan Master mix and the following probes (Life Technologies): TIFAB (Hs04185733_m1), Tifab (Mm0421026_m1), TRAF6 (Hs00371512_g1), Traf6 (Mm00493836_m1), Il6 (Mm00446190_m1), Bcl2a1 (Mm03646861_mH), Tnfaip3 (Mm00437121_m1), Sod2 (Mm01313000_m1), GAPDH (Hs02758991_g1), and Gapdh (Mm99999915_g1). qPCR was performed on an Applied Biosystems StepOne Plus Real-Time PCR System. miR-146a expression was performed as previously described (Starczynowski et al., 2011).Clonogenic progenitor assays.Hematopoietic clonogenic progenitor frequencies were determined by plating 2 × 105 BM cells/ml isolated from Tifab+/+, Tifab+/−, Tifab−/−, miR-146a−/−, or Tifab−/−;miR-146a−/− mice in methylcellulose media containing human erythropoietin, murine SCF, murine IL-3, and human IL-6 (Methocult M3434; Stem Cell Technologies). For human leukemic colony formation, transduced MDSL, HL-60, and THP1 cells were plated at 2–3 × 104/ml in methylcellulose (Methocult H4434; Stem Cell Technologies). Colonies were scored after 10 d.Cell lines.Human AML cell lines HL60 and THP1 were purchased from ATCC. The myelodysplastic cell line, MDSL, was provided by K. Tohyama (Kawasaki Medical School, Okayama, Japan). HEK293 cells were also purchased from ATCC. HL-60 and THP1 cell lines were cultured in RPMI-1640 medium with 10% FBS and 1% penicillin-streptomycin. HEK293 cells were cultured in DMEM with 10% FBS and 1% penicillin-streptomycin. MDSL were cultured in 10% FBS, 1% penicillin-streptomycin, 20 ng/ml IL-3, and 50 µM β-mercaptoethanol. Human marrow-derived mononuclear cells were obtained from the Translational Research Development Support Laboratory of Cincinnati Children’s Hospital (Cincinnati, OH) under an approved Institutional Review Board protocol. Trypan blue exclusion was performed to determined cell proliferation and viability.Gene expression profiling.Gene expression analysis (MoGene 2.0 ST Array) was performed on sorted LSK isolated from 3-mo-old mice transplanted with Tifab+/+ or Tifab−/− BM cells (n = 3 mice/group). Total RNA was extracted and purified with Quick RNA extraction kit (Zymo Research). Total RNA was reverse transcribed, labeled, and hybridized onto the GeneChip MoGene 2.0 ST Array (Affymetrix). Scanning was performed with GeneChip Version 3.2 Scanner 3000 7G (Affymetrix) and evaluated with GeneChip operating software (Affymetrix). Data analysis was performed using GeneSpring GX 11.5 software (Agilent Technologies). Gene set enrichment analysis was performed on a JAVA-based dataset supported by the Broad Institute (Subramanian et al., 2005).Flow cytometry analysis.Antibodies used for flow cytometric analysis of BM and PB cells are listed in Table S2). All FACS analyses were performed on FACSAria or FACSCanto machines (BD). FACS data analysis was performed using FACSDiva (BD) or FlowJo (Tree Star) software.Immunoprecipitation and immunoblotting.Total protein was isolated from cells by lysing samples in RIPA buffer (50 mM Tris-HCl, 150 mM NaCl, 1 mM EDTA, 1% Triton X-100, and 0.1% SDS), in the presence of PMSF, sodium orthovanadate, and protease and phosphatase inhibitors. For immunoprecipitation, TRAF6 or hemagglutinin (HA) antibodies were added to cell lysates for 2 h at 4°C and captured by the addition of Protein A/G Plus beads (sc-2003; Santa Cruz Biotechnology, Inc.). The immune complexes were washed with RIPA lysis buffer, and resuspended in SDS sample buffer. Proteins were separated by SDS-PAGE, transferred to nitrocellulose membranes, and analyzed by immunoblotting. Immunoblotting was performed with the following antibodies: TRAF6 (sc-7221; Santa Cruz Biotechnology, Inc.), FLAG (F3165; Sigma-Aldrich), HA (H6908; Sigma-Aldrich) GAPDH (5174; Cell Signaling Technology), phospho-IKKβ (2697; Cell Signaling Technology), IKKβ (2370; Cell Signaling Technology), and Actin (4968; Cell Signaling Technology).Plasmids and transfection.WT and mutant TIFAB cDNA were created by custom gene synthesis (IDT). FLAG and/or HA tags were inserted in frame at the 5′-end of human TIFAB. FLAG-TIFAB was cloned into pcDNA3.0 with EcoRI and HindIII, and into pMSCV-pGK-GFP with EcoRI and XhoI. pcDNA3.0-FLAG-IKKβ and pcDNA3.0-FLAG-RelA were purchased from Addgene. pcDNA3.0-FLAG-TRAF2 was a gift from H. Habelhah (University of Iowa, Iowa City, IA). pcDNA3.0-FLAG-TRAF6 was created by subcloning TRAF6 from MIY-TRAF6 using EcoRI. FLAG-TRAF6 was cloned into pMSCV-pGK-GFP with EcoRI and XhoI. Transfections were performed using TransIT-LT1 transfection reagent (Mirus MIR 2306). HEK293 cells were transfected for 24–48 h with κB-luciferase and TK Renilla plasmids along with TRAF6, TIFAB, TRAF2, IKKβ, or p65. Lysates were analyzed for NF-κB reporter activity using the dual luciferase reporter assay system (Promega E1910). Retro- and lentiviral transduction of cells has been previous described (Fang et al., 2014). For HL60 and THP1, cells were transduced with virus encoding vectors coexpressing GFP (pMSCV-pGK-GFP or pLeGo-iG2). GFP-positive cells were FACS sorted using a FACSAria (BD). For knockdown of Traf6 in Tifab+/+ or Tifab−/− BM mononuclear cells, BM cells were isolated and transduced with pLKO.1 lentiviral vectors (OpenBiosystems) encoding shRNAs targeting mouse Traf6 (TRCN0000040735), as previously described (Fang et al., 2014).Purification and mass spectrometry analysis of TIFAB-interacting proteins.TIFAB complexes were isolated from cytosolic and nuclear extracts of HL60 cells transduced with HA/FLAG-TIFAB using a two-step affinity purification procedure. TIFAB-interacting proteins were identified by mass spectrometry analysis, using a previously described procedure (Singh et al., 2008).Xenograftment of NSGS with MDSL cells.MDSL cells were transduced to overexpress TIFAB (pGK-TIFAB-GFP), and GFP-positive cells were isolated by flow cytometry. 1 × 106 MDSL cells were intravenously injected into 8-wk old-conditioned (Busulfan) NOD/SCID-IL2Rγ3GS (NSGS) mice as previously described (Rhyasen et al., 2013a, 2014). BM aspirates were performed to measure MDSL engraftment by flow cytometry (human CD45.1 versus mouse CD45.1).Microarray data.Microarray data can be found in the Gene Expression Omnibus database under accession no. GSE72936.Patient samples.Informed consent was obtained according to protocols approved by the review board of Cleveland Clinic. Diagnoses were reviewed at Cleveland Clinic and adapted, when required, to WHO 2008 criteria. For qRT-PCR analysis of TIFAB transcript, BM mononuclear cells were isolated from MDS/AML patients and age-matched healthy controls, as previously described (Fang et al., 2014).Online supplemental material.Table S1 lists genes differentially expressed in Tifab-deficient LSK by >1.7 fold. Table S2 lists antibodies for flow cytometry. Online supplemental material is available at http://www.jem.org/cgi/content/full/jem.20141898/DC1.

MATERIALS AND METHODS

Knockout mice.A target vector was designed to replace exon 3, which contains all coding region of Tifab, with a β-galactosidase-loxP-neomycin resistant gene cassette-loxP fragment. Diphtheria toxin A gene cassette was inserted at the downstream end of the short arm. The targeting construct was electroporated into a 129-derived embryonic stem (ES) cell line. G418-resistant ES clones were screened for homologous recombination by Southern blot analysis. ES clones containing the correctly targeted Tifab locus were injected into C57BL/6 blastocysts. Male chimeric mice were mated with C57BL/6 J, and agouti pups carrying the correct target locus were used as F1 mice. F1 mice were crossed with transgenic mice expressing CRE under control of CAG promoter (Sakai and Miyazaki, 1997). Pups lacking neomycin-resistant gene cassette were further backcrossed into the C57BL/6 J background. miR-146a−/− C57BL/6 mice were obtained from D. Baltimore (California Institute of Technology, Pasadena, CA) as previously described (Fang et al., 2014). Animals were bred and housed in the Association for Assessment and Accreditation of Laboratory Animal Care-accredited animal facility of Cincinnati Children’s Hospital Medical Center.Bone marrow transplantation.For noncompetitive BM transplantation, BM mononuclear cells (0.3–3 × 106) from C7BL/6 Tifab+/+, Tifab+/−, Tifab−/−, miR-146a−/−, or Tifab−/−;miR-146a−/− mice were injected i.v. into lethally radiated (7.0 Gy and 4.75 Gy after 3 h) syngeneic recipient Boy/J mice. For competitive BM transplants, BM mononuclear cells (3 × 105) from 8-wk-old CD45.2 Tifab+/+, Tifab+/−, or Tifab−/− mice were mixed with equal numbers of 8-wk-old BM mononuclear cells from BoyJ (CD45.1) mice and then injected i.v. into lethally irradiated (7.0 Gy and 4.75 Gy after 3 h) CD45.1 recipient BoyJ mice.BM and PB analysis.For cytospins, 2.5–5 × 105 BM cells were spun onto slide at 300 rpm for 5 min, and then stained with Wright-Giemsa. For BM sections, tibia were fixed in formalin and then stained with hematoxylin and eosin. Complete blood counts were performed on PB isolated from the tail vein every 4 wk and analyzed using the Drew Scientific Hemavet 950.BM stimulation with LPS and TNF.BM mononuclear cells were harvested from Tifab+/+ and Tifab−/− mice and cultured in the presence of vehicle control (dH2O for LPS or 0.1% BSA in 1X PBS for TNF), ultrapure LPS (100 ng/ml; Invitrogen), or TNF (10 ng/ml; Gibco) for 0, 1, 4, and 8 h. RNA extraction, cDNA conversion, and qRT-PCR analysis were performed as described below.Quantitative PCR analysis.RNA was extracted from cells and purified with TRIzol Reagent (Life Technologies) or Quick-RNA MiniPrep kit (Zymo Research). cDNA was generated using a high capacity RNA to cDNA kit (Life Technologies 4387406 or 4368814). qPCR was performed using TaqMan Master mix and the following probes (Life Technologies): TIFAB (Hs04185733_m1), Tifab (Mm0421026_m1), TRAF6 (Hs00371512_g1), Traf6 (Mm00493836_m1), Il6 (Mm00446190_m1), Bcl2a1 (Mm03646861_mH), Tnfaip3 (Mm00437121_m1), Sod2 (Mm01313000_m1), GAPDH (Hs02758991_g1), and Gapdh (Mm99999915_g1). qPCR was performed on an Applied Biosystems StepOne Plus Real-Time PCR System. miR-146a expression was performed as previously described (Starczynowski et al., 2011).Clonogenic progenitor assays.Hematopoietic clonogenic progenitor frequencies were determined by plating 2 × 105 BM cells/ml isolated from Tifab+/+, Tifab+/−, Tifab−/−, miR-146a−/−, or Tifab−/−;miR-146a−/− mice in methylcellulose media containing human erythropoietin, murine SCF, murine IL-3, and human IL-6 (Methocult M3434; Stem Cell Technologies). For human leukemic colony formation, transduced MDSL, HL-60, and THP1 cells were plated at 2–3 × 104/ml in methylcellulose (Methocult H4434; Stem Cell Technologies). Colonies were scored after 10 d.Cell lines.Human AML cell lines HL60 and THP1 were purchased from ATCC. The myelodysplastic cell line, MDSL, was provided by K. Tohyama (Kawasaki Medical School, Okayama, Japan). HEK293 cells were also purchased from ATCC. HL-60 and THP1 cell lines were cultured in RPMI-1640 medium with 10% FBS and 1% penicillin-streptomycin. HEK293 cells were cultured in DMEM with 10% FBS and 1% penicillin-streptomycin. MDSL were cultured in 10% FBS, 1% penicillin-streptomycin, 20 ng/ml IL-3, and 50 µM β-mercaptoethanol. Human marrow-derived mononuclear cells were obtained from the Translational Research Development Support Laboratory of Cincinnati Children’s Hospital (Cincinnati, OH) under an approved Institutional Review Board protocol. Trypan blue exclusion was performed to determined cell proliferation and viability.Gene expression profiling.Gene expression analysis (MoGene 2.0 ST Array) was performed on sorted LSK isolated from 3-mo-old mice transplanted with Tifab+/+ or Tifab−/− BM cells (n = 3 mice/group). Total RNA was extracted and purified with Quick RNA extraction kit (Zymo Research). Total RNA was reverse transcribed, labeled, and hybridized onto the GeneChip MoGene 2.0 ST Array (Affymetrix). Scanning was performed with GeneChip Version 3.2 Scanner 3000 7G (Affymetrix) and evaluated with GeneChip operating software (Affymetrix). Data analysis was performed using GeneSpring GX 11.5 software (Agilent Technologies). Gene set enrichment analysis was performed on a JAVA-based dataset supported by the Broad Institute (Subramanian et al., 2005).Flow cytometry analysis.Antibodies used for flow cytometric analysis of BM and PB cells are listed in Table S2). All FACS analyses were performed on FACSAria or FACSCanto machines (BD). FACS data analysis was performed using FACSDiva (BD) or FlowJo (Tree Star) software.Immunoprecipitation and immunoblotting.Total protein was isolated from cells by lysing samples in RIPA buffer (50 mM Tris-HCl, 150 mM NaCl, 1 mM EDTA, 1% Triton X-100, and 0.1% SDS), in the presence of PMSF, sodium orthovanadate, and protease and phosphatase inhibitors. For immunoprecipitation, TRAF6 or hemagglutinin (HA) antibodies were added to cell lysates for 2 h at 4°C and captured by the addition of Protein A/G Plus beads (sc-2003; Santa Cruz Biotechnology, Inc.). The immune complexes were washed with RIPA lysis buffer, and resuspended in SDS sample buffer. Proteins were separated by SDS-PAGE, transferred to nitrocellulose membranes, and analyzed by immunoblotting. Immunoblotting was performed with the following antibodies: TRAF6 (sc-7221; Santa Cruz Biotechnology, Inc.), FLAG (F3165; Sigma-Aldrich), HA (H6908; Sigma-Aldrich) GAPDH (5174; Cell Signaling Technology), phospho-IKKβ (2697; Cell Signaling Technology), IKKβ (2370; Cell Signaling Technology), and Actin (4968; Cell Signaling Technology).Plasmids and transfection.WT and mutant TIFAB cDNA were created by custom gene synthesis (IDT). FLAG and/or HA tags were inserted in frame at the 5′-end of human TIFAB. FLAG-TIFAB was cloned into pcDNA3.0 with EcoRI and HindIII, and into pMSCV-pGK-GFP with EcoRI and XhoI. pcDNA3.0-FLAG-IKKβ and pcDNA3.0-FLAG-RelA were purchased from Addgene. pcDNA3.0-FLAG-TRAF2 was a gift from H. Habelhah (University of Iowa, Iowa City, IA). pcDNA3.0-FLAG-TRAF6 was created by subcloning TRAF6 from MIY-TRAF6 using EcoRI. FLAG-TRAF6 was cloned into pMSCV-pGK-GFP with EcoRI and XhoI. Transfections were performed using TransIT-LT1 transfection reagent (Mirus MIR 2306). HEK293 cells were transfected for 24–48 h with κB-luciferase and TK Renilla plasmids along with TRAF6, TIFAB, TRAF2, IKKβ, or p65. Lysates were analyzed for NF-κB reporter activity using the dual luciferase reporter assay system (Promega E1910). Retro- and lentiviral transduction of cells has been previous described (Fang et al., 2014). For HL60 and THP1, cells were transduced with virus encoding vectors coexpressing GFP (pMSCV-pGK-GFP or pLeGo-iG2). GFP-positive cells were FACS sorted using a FACSAria (BD). For knockdown of Traf6 in Tifab+/+ or Tifab−/− BM mononuclear cells, BM cells were isolated and transduced with pLKO.1 lentiviral vectors (OpenBiosystems) encoding shRNAs targeting mouse Traf6 (TRCN0000040735), as previously described (Fang et al., 2014).Purification and mass spectrometry analysis of TIFAB-interacting proteins.TIFAB complexes were isolated from cytosolic and nuclear extracts of HL60 cells transduced with HA/FLAG-TIFAB using a two-step affinity purification procedure. TIFAB-interacting proteins were identified by mass spectrometry analysis, using a previously described procedure (Singh et al., 2008).Xenograftment of NSGS with MDSL cells.MDSL cells were transduced to overexpress TIFAB (pGK-TIFAB-GFP), and GFP-positive cells were isolated by flow cytometry. 1 × 106 MDSL cells were intravenously injected into 8-wk old-conditioned (Busulfan) NOD/SCID-IL2Rγ3GS (NSGS) mice as previously described (Rhyasen et al., 2013a, 2014). BM aspirates were performed to measure MDSL engraftment by flow cytometry (human CD45.1 versus mouse CD45.1).Microarray data.Microarray data can be found in the Gene Expression Omnibus database under accession no. GSE72936.Patient samples.Informed consent was obtained according to protocols approved by the review board of Cleveland Clinic. Diagnoses were reviewed at Cleveland Clinic and adapted, when required, to WHO 2008 criteria. For qRT-PCR analysis of TIFAB transcript, BM mononuclear cells were isolated from MDS/AML patients and age-matched healthy controls, as previously described (Fang et al., 2014).Online supplemental material.Table S1 lists genes differentially expressed in Tifab-deficient LSK by >1.7 fold. Table S2 lists antibodies for flow cytometry. Online supplemental material is available at http://www.jem.org/cgi/content/full/jem.20141898/DC1.

Knockout mice.A target vector was designed to replace exon 3, which contains all coding region of Tifab, with a β-galactosidase-loxP-neomycin resistant gene cassette-loxP fragment. Diphtheria toxin A gene cassette was inserted at the downstream end of the short arm. The targeting construct was electroporated into a 129-derived embryonic stem (ES) cell line. G418-resistant ES clones were screened for homologous recombination by Southern blot analysis. ES clones containing the correctly targeted Tifab locus were injected into C57BL/6 blastocysts. Male chimeric mice were mated with C57BL/6 J, and agouti pups carrying the correct target locus were used as F1 mice. F1 mice were crossed with transgenic mice expressing CRE under control of CAG promoter (Sakai and Miyazaki, 1997). Pups lacking neomycin-resistant gene cassette were further backcrossed into the C57BL/6 J background. miR-146a−/− C57BL/6 mice were obtained from D. Baltimore (California Institute of Technology, Pasadena, CA) as previously described (Fang et al., 2014). Animals were bred and housed in the Association for Assessment and Accreditation of Laboratory Animal Care-accredited animal facility of Cincinnati Children’s Hospital Medical Center.

Knockout mice.

A target vector was designed to replace exon 3, which contains all coding region of Tifab, with a β-galactosidase-loxP-neomycin resistant gene cassette-loxP fragment. Diphtheria toxin A gene cassette was inserted at the downstream end of the short arm. The targeting construct was electroporated into a 129-derived embryonic stem (ES) cell line. G418-resistant ES clones were screened for homologous recombination by Southern blot analysis. ES clones containing the correctly targeted Tifab locus were injected into C57BL/6 blastocysts. Male chimeric mice were mated with C57BL/6 J, and agouti pups carrying the correct target locus were used as F1 mice. F1 mice were crossed with transgenic mice expressing CRE under control of CAG promoter (Sakai and Miyazaki, 1997). Pups lacking neomycin-resistant gene cassette were further backcrossed into the C57BL/6 J background. miR-146a−/− C57BL/6 mice were obtained from D. Baltimore (California Institute of Technology, Pasadena, CA) as previously described (Fang et al., 2014). Animals were bred and housed in the Association for Assessment and Accreditation of Laboratory Animal Care-accredited animal facility of Cincinnati Children’s Hospital Medical Center.

Bone marrow transplantation.For noncompetitive BM transplantation, BM mononuclear cells (0.3–3 × 106) from C7BL/6 Tifab+/+, Tifab+/−, Tifab−/−, miR-146a−/−, or Tifab−/−;miR-146a−/− mice were injected i.v. into lethally radiated (7.0 Gy and 4.75 Gy after 3 h) syngeneic recipient Boy/J mice. For competitive BM transplants, BM mononuclear cells (3 × 105) from 8-wk-old CD45.2 Tifab+/+, Tifab+/−, or Tifab−/− mice were mixed with equal numbers of 8-wk-old BM mononuclear cells from BoyJ (CD45.1) mice and then injected i.v. into lethally irradiated (7.0 Gy and 4.75 Gy after 3 h) CD45.1 recipient BoyJ mice.

Bone marrow transplantation.

For noncompetitive BM transplantation, BM mononuclear cells (0.3–3 × 106) from C7BL/6 Tifab+/+, Tifab+/−, Tifab−/−, miR-146a−/−, or Tifab−/−;miR-146a−/− mice were injected i.v. into lethally radiated (7.0 Gy and 4.75 Gy after 3 h) syngeneic recipient Boy/J mice. For competitive BM transplants, BM mononuclear cells (3 × 105) from 8-wk-old CD45.2 Tifab+/+, Tifab+/−, or Tifab−/− mice were mixed with equal numbers of 8-wk-old BM mononuclear cells from BoyJ (CD45.1) mice and then injected i.v. into lethally irradiated (7.0 Gy and 4.75 Gy after 3 h) CD45.1 recipient BoyJ mice.

BM and PB analysis.For cytospins, 2.5–5 × 105 BM cells were spun onto slide at 300 rpm for 5 min, and then stained with Wright-Giemsa. For BM sections, tibia were fixed in formalin and then stained with hematoxylin and eosin. Complete blood counts were performed on PB isolated from the tail vein every 4 wk and analyzed using the Drew Scientific Hemavet 950.

BM and PB analysis.

For cytospins, 2.5–5 × 105 BM cells were spun onto slide at 300 rpm for 5 min, and then stained with Wright-Giemsa. For BM sections, tibia were fixed in formalin and then stained with hematoxylin and eosin. Complete blood counts were performed on PB isolated from the tail vein every 4 wk and analyzed using the Drew Scientific Hemavet 950.

BM stimulation with LPS and TNF.BM mononuclear cells were harvested from Tifab+/+ and Tifab−/− mice and cultured in the presence of vehicle control (dH2O for LPS or 0.1% BSA in 1X PBS for TNF), ultrapure LPS (100 ng/ml; Invitrogen), or TNF (10 ng/ml; Gibco) for 0, 1, 4, and 8 h. RNA extraction, cDNA conversion, and qRT-PCR analysis were performed as described below.

BM stimulation with LPS and TNF.

BM mononuclear cells were harvested from Tifab+/+ and Tifab−/− mice and cultured in the presence of vehicle control (dH2O for LPS or 0.1% BSA in 1X PBS for TNF), ultrapure LPS (100 ng/ml; Invitrogen), or TNF (10 ng/ml; Gibco) for 0, 1, 4, and 8 h. RNA extraction, cDNA conversion, and qRT-PCR analysis were performed as described below.

Quantitative PCR analysis.RNA was extracted from cells and purified with TRIzol Reagent (Life Technologies) or Quick-RNA MiniPrep kit (Zymo Research). cDNA was generated using a high capacity RNA to cDNA kit (Life Technologies 4387406 or 4368814). qPCR was performed using TaqMan Master mix and the following probes (Life Technologies): TIFAB (Hs04185733_m1), Tifab (Mm0421026_m1), TRAF6 (Hs00371512_g1), Traf6 (Mm00493836_m1), Il6 (Mm00446190_m1), Bcl2a1 (Mm03646861_mH), Tnfaip3 (Mm00437121_m1), Sod2 (Mm01313000_m1), GAPDH (Hs02758991_g1), and Gapdh (Mm99999915_g1). qPCR was performed on an Applied Biosystems StepOne Plus Real-Time PCR System. miR-146a expression was performed as previously described (Starczynowski et al., 2011).

Quantitative PCR analysis.

RNA was extracted from cells and purified with TRIzol Reagent (Life Technologies) or Quick-RNA MiniPrep kit (Zymo Research). cDNA was generated using a high capacity RNA to cDNA kit (Life Technologies 4387406 or 4368814). qPCR was performed using TaqMan Master mix and the following probes (Life Technologies): TIFAB (Hs04185733_m1), Tifab (Mm0421026_m1), TRAF6 (Hs00371512_g1), Traf6 (Mm00493836_m1), Il6 (Mm00446190_m1), Bcl2a1 (Mm03646861_mH), Tnfaip3 (Mm00437121_m1), Sod2 (Mm01313000_m1), GAPDH (Hs02758991_g1), and Gapdh (Mm99999915_g1). qPCR was performed on an Applied Biosystems StepOne Plus Real-Time PCR System. miR-146a expression was performed as previously described (Starczynowski et al., 2011).

Clonogenic progenitor assays.Hematopoietic clonogenic progenitor frequencies were determined by plating 2 × 105 BM cells/ml isolated from Tifab+/+, Tifab+/−, Tifab−/−, miR-146a−/−, or Tifab−/−;miR-146a−/− mice in methylcellulose media containing human erythropoietin, murine SCF, murine IL-3, and human IL-6 (Methocult M3434; Stem Cell Technologies). For human leukemic colony formation, transduced MDSL, HL-60, and THP1 cells were plated at 2–3 × 104/ml in methylcellulose (Methocult H4434; Stem Cell Technologies). Colonies were scored after 10 d.

Clonogenic progenitor assays.

Hematopoietic clonogenic progenitor frequencies were determined by plating 2 × 105 BM cells/ml isolated from Tifab+/+, Tifab+/−, Tifab−/−, miR-146a−/−, or Tifab−/−;miR-146a−/− mice in methylcellulose media containing human erythropoietin, murine SCF, murine IL-3, and human IL-6 (Methocult M3434; Stem Cell Technologies). For human leukemic colony formation, transduced MDSL, HL-60, and THP1 cells were plated at 2–3 × 104/ml in methylcellulose (Methocult H4434; Stem Cell Technologies). Colonies were scored after 10 d.

Cell lines.Human AML cell lines HL60 and THP1 were purchased from ATCC. The myelodysplastic cell line, MDSL, was provided by K. Tohyama (Kawasaki Medical School, Okayama, Japan). HEK293 cells were also purchased from ATCC. HL-60 and THP1 cell lines were cultured in RPMI-1640 medium with 10% FBS and 1% penicillin-streptomycin. HEK293 cells were cultured in DMEM with 10% FBS and 1% penicillin-streptomycin. MDSL were cultured in 10% FBS, 1% penicillin-streptomycin, 20 ng/ml IL-3, and 50 µM β-mercaptoethanol. Human marrow-derived mononuclear cells were obtained from the Translational Research Development Support Laboratory of Cincinnati Children’s Hospital (Cincinnati, OH) under an approved Institutional Review Board protocol. Trypan blue exclusion was performed to determined cell proliferation and viability.

Cell lines.

Human AML cell lines HL60 and THP1 were purchased from ATCC. The myelodysplastic cell line, MDSL, was provided by K. Tohyama (Kawasaki Medical School, Okayama, Japan). HEK293 cells were also purchased from ATCC. HL-60 and THP1 cell lines were cultured in RPMI-1640 medium with 10% FBS and 1% penicillin-streptomycin. HEK293 cells were cultured in DMEM with 10% FBS and 1% penicillin-streptomycin. MDSL were cultured in 10% FBS, 1% penicillin-streptomycin, 20 ng/ml IL-3, and 50 µM β-mercaptoethanol. Human marrow-derived mononuclear cells were obtained from the Translational Research Development Support Laboratory of Cincinnati Children’s Hospital (Cincinnati, OH) under an approved Institutional Review Board protocol. Trypan blue exclusion was performed to determined cell proliferation and viability.

Gene expression profiling.Gene expression analysis (MoGene 2.0 ST Array) was performed on sorted LSK isolated from 3-mo-old mice transplanted with Tifab+/+ or Tifab−/− BM cells (n = 3 mice/group). Total RNA was extracted and purified with Quick RNA extraction kit (Zymo Research). Total RNA was reverse transcribed, labeled, and hybridized onto the GeneChip MoGene 2.0 ST Array (Affymetrix). Scanning was performed with GeneChip Version 3.2 Scanner 3000 7G (Affymetrix) and evaluated with GeneChip operating software (Affymetrix). Data analysis was performed using GeneSpring GX 11.5 software (Agilent Technologies). Gene set enrichment analysis was performed on a JAVA-based dataset supported by the Broad Institute (Subramanian et al., 2005).

Gene expression profiling.

Gene expression analysis (MoGene 2.0 ST Array) was performed on sorted LSK isolated from 3-mo-old mice transplanted with Tifab+/+ or Tifab−/− BM cells (n = 3 mice/group). Total RNA was extracted and purified with Quick RNA extraction kit (Zymo Research). Total RNA was reverse transcribed, labeled, and hybridized onto the GeneChip MoGene 2.0 ST Array (Affymetrix). Scanning was performed with GeneChip Version 3.2 Scanner 3000 7G (Affymetrix) and evaluated with GeneChip operating software (Affymetrix). Data analysis was performed using GeneSpring GX 11.5 software (Agilent Technologies). Gene set enrichment analysis was performed on a JAVA-based dataset supported by the Broad Institute (Subramanian et al., 2005).

Flow cytometry analysis.Antibodies used for flow cytometric analysis of BM and PB cells are listed in Table S2). All FACS analyses were performed on FACSAria or FACSCanto machines (BD). FACS data analysis was performed using FACSDiva (BD) or FlowJo (Tree Star) software.

Flow cytometry analysis.

Antibodies used for flow cytometric analysis of BM and PB cells are listed in Table S2). All FACS analyses were performed on FACSAria or FACSCanto machines (BD). FACS data analysis was performed using FACSDiva (BD) or FlowJo (Tree Star) software.

Immunoprecipitation and immunoblotting.Total protein was isolated from cells by lysing samples in RIPA buffer (50 mM Tris-HCl, 150 mM NaCl, 1 mM EDTA, 1% Triton X-100, and 0.1% SDS), in the presence of PMSF, sodium orthovanadate, and protease and phosphatase inhibitors. For immunoprecipitation, TRAF6 or hemagglutinin (HA) antibodies were added to cell lysates for 2 h at 4°C and captured by the addition of Protein A/G Plus beads (sc-2003; Santa Cruz Biotechnology, Inc.). The immune complexes were washed with RIPA lysis buffer, and resuspended in SDS sample buffer. Proteins were separated by SDS-PAGE, transferred to nitrocellulose membranes, and analyzed by immunoblotting. Immunoblotting was performed with the following antibodies: TRAF6 (sc-7221; Santa Cruz Biotechnology, Inc.), FLAG (F3165; Sigma-Aldrich), HA (H6908; Sigma-Aldrich) GAPDH (5174; Cell Signaling Technology), phospho-IKKβ (2697; Cell Signaling Technology), IKKβ (2370; Cell Signaling Technology), and Actin (4968; Cell Signaling Technology).

Immunoprecipitation and immunoblotting.

Total protein was isolated from cells by lysing samples in RIPA buffer (50 mM Tris-HCl, 150 mM NaCl, 1 mM EDTA, 1% Triton X-100, and 0.1% SDS), in the presence of PMSF, sodium orthovanadate, and protease and phosphatase inhibitors. For immunoprecipitation, TRAF6 or hemagglutinin (HA) antibodies were added to cell lysates for 2 h at 4°C and captured by the addition of Protein A/G Plus beads (sc-2003; Santa Cruz Biotechnology, Inc.). The immune complexes were washed with RIPA lysis buffer, and resuspended in SDS sample buffer. Proteins were separated by SDS-PAGE, transferred to nitrocellulose membranes, and analyzed by immunoblotting. Immunoblotting was performed with the following antibodies: TRAF6 (sc-7221; Santa Cruz Biotechnology, Inc.), FLAG (F3165; Sigma-Aldrich), HA (H6908; Sigma-Aldrich) GAPDH (5174; Cell Signaling Technology), phospho-IKKβ (2697; Cell Signaling Technology), IKKβ (2370; Cell Signaling Technology), and Actin (4968; Cell Signaling Technology).

Plasmids and transfection.WT and mutant TIFAB cDNA were created by custom gene synthesis (IDT). FLAG and/or HA tags were inserted in frame at the 5′-end of human TIFAB. FLAG-TIFAB was cloned into pcDNA3.0 with EcoRI and HindIII, and into pMSCV-pGK-GFP with EcoRI and XhoI. pcDNA3.0-FLAG-IKKβ and pcDNA3.0-FLAG-RelA were purchased from Addgene. pcDNA3.0-FLAG-TRAF2 was a gift from H. Habelhah (University of Iowa, Iowa City, IA). pcDNA3.0-FLAG-TRAF6 was created by subcloning TRAF6 from MIY-TRAF6 using EcoRI. FLAG-TRAF6 was cloned into pMSCV-pGK-GFP with EcoRI and XhoI. Transfections were performed using TransIT-LT1 transfection reagent (Mirus MIR 2306). HEK293 cells were transfected for 24–48 h with κB-luciferase and TK Renilla plasmids along with TRAF6, TIFAB, TRAF2, IKKβ, or p65. Lysates were analyzed for NF-κB reporter activity using the dual luciferase reporter assay system (Promega E1910). Retro- and lentiviral transduction of cells has been previous described (Fang et al., 2014). For HL60 and THP1, cells were transduced with virus encoding vectors coexpressing GFP (pMSCV-pGK-GFP or pLeGo-iG2). GFP-positive cells were FACS sorted using a FACSAria (BD). For knockdown of Traf6 in Tifab+/+ or Tifab−/− BM mononuclear cells, BM cells were isolated and transduced with pLKO.1 lentiviral vectors (OpenBiosystems) encoding shRNAs targeting mouse Traf6 (TRCN0000040735), as previously described (Fang et al., 2014).

Plasmids and transfection.

WT and mutant TIFAB cDNA were created by custom gene synthesis (IDT). FLAG and/or HA tags were inserted in frame at the 5′-end of human TIFAB. FLAG-TIFAB was cloned into pcDNA3.0 with EcoRI and HindIII, and into pMSCV-pGK-GFP with EcoRI and XhoI. pcDNA3.0-FLAG-IKKβ and pcDNA3.0-FLAG-RelA were purchased from Addgene. pcDNA3.0-FLAG-TRAF2 was a gift from H. Habelhah (University of Iowa, Iowa City, IA). pcDNA3.0-FLAG-TRAF6 was created by subcloning TRAF6 from MIY-TRAF6 using EcoRI. FLAG-TRAF6 was cloned into pMSCV-pGK-GFP with EcoRI and XhoI. Transfections were performed using TransIT-LT1 transfection reagent (Mirus MIR 2306). HEK293 cells were transfected for 24–48 h with κB-luciferase and TK Renilla plasmids along with TRAF6, TIFAB, TRAF2, IKKβ, or p65. Lysates were analyzed for NF-κB reporter activity using the dual luciferase reporter assay system (Promega E1910). Retro- and lentiviral transduction of cells has been previous described (Fang et al., 2014). For HL60 and THP1, cells were transduced with virus encoding vectors coexpressing GFP (pMSCV-pGK-GFP or pLeGo-iG2). GFP-positive cells were FACS sorted using a FACSAria (BD). For knockdown of Traf6 in Tifab+/+ or Tifab−/− BM mononuclear cells, BM cells were isolated and transduced with pLKO.1 lentiviral vectors (OpenBiosystems) encoding shRNAs targeting mouse Traf6 (TRCN0000040735), as previously described (Fang et al., 2014).

Purification and mass spectrometry analysis of TIFAB-interacting proteins.TIFAB complexes were isolated from cytosolic and nuclear extracts of HL60 cells transduced with HA/FLAG-TIFAB using a two-step affinity purification procedure. TIFAB-interacting proteins were identified by mass spectrometry analysis, using a previously described procedure (Singh et al., 2008).

Purification and mass spectrometry analysis of TIFAB-interacting proteins.

TIFAB complexes were isolated from cytosolic and nuclear extracts of HL60 cells transduced with HA/FLAG-TIFAB using a two-step affinity purification procedure. TIFAB-interacting proteins were identified by mass spectrometry analysis, using a previously described procedure (Singh et al., 2008).

Xenograftment of NSGS with MDSL cells.MDSL cells were transduced to overexpress TIFAB (pGK-TIFAB-GFP), and GFP-positive cells were isolated by flow cytometry. 1 × 106 MDSL cells were intravenously injected into 8-wk old-conditioned (Busulfan) NOD/SCID-IL2Rγ3GS (NSGS) mice as previously described (Rhyasen et al., 2013a, 2014). BM aspirates were performed to measure MDSL engraftment by flow cytometry (human CD45.1 versus mouse CD45.1).

Xenograftment of NSGS with MDSL cells.

MDSL cells were transduced to overexpress TIFAB (pGK-TIFAB-GFP), and GFP-positive cells were isolated by flow cytometry. 1 × 106 MDSL cells were intravenously injected into 8-wk old-conditioned (Busulfan) NOD/SCID-IL2Rγ3GS (NSGS) mice as previously described (Rhyasen et al., 2013a, 2014). BM aspirates were performed to measure MDSL engraftment by flow cytometry (human CD45.1 versus mouse CD45.1).

Microarray data.Microarray data can be found in the Gene Expression Omnibus database under accession no. GSE72936.

Microarray data.

Microarray data can be found in the Gene Expression Omnibus database under accession no. GSE72936.

Patient samples.Informed consent was obtained according to protocols approved by the review board of Cleveland Clinic. Diagnoses were reviewed at Cleveland Clinic and adapted, when required, to WHO 2008 criteria. For qRT-PCR analysis of TIFAB transcript, BM mononuclear cells were isolated from MDS/AML patients and age-matched healthy controls, as previously described (Fang et al., 2014).

Patient samples.

Informed consent was obtained according to protocols approved by the review board of Cleveland Clinic. Diagnoses were reviewed at Cleveland Clinic and adapted, when required, to WHO 2008 criteria. For qRT-PCR analysis of TIFAB transcript, BM mononuclear cells were isolated from MDS/AML patients and age-matched healthy controls, as previously described (Fang et al., 2014).

Online supplemental material.Table S1 lists genes differentially expressed in Tifab-deficient LSK by >1.7 fold. Table S2 lists antibodies for flow cytometry. Online supplemental material is available at http://www.jem.org/cgi/content/full/jem.20141898/DC1.

Online supplemental material.

Table S1 lists genes differentially expressed in Tifab-deficient LSK by >1.7 fold. Table S2 lists antibodies for flow cytometry. Online supplemental material is available at http://www.jem.org/cgi/content/full/jem.20141898/DC1.

Supplementary MaterialSupplemental Material

Supplementary Material

Supplemental Material
