---
pmid: '10085294'
title: The cortical localization of the microtubule orientation protein, Kar9p, is
  dependent upon actin and proteins required for polarization.
authors:
- Miller RK
- Matheos D
- Rose MD
journal: J Cell Biol
year: '1999'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2148208
doi: 10.1083/jcb.144.5.963
---

# The cortical localization of the microtubule orientation protein, Kar9p, is dependent upon actin and proteins required for polarization.
**Authors:** Miller RK, Matheos D, Rose MD
**Journal:** J Cell Biol (1999)
**DOI:** [10.1083/jcb.144.5.963](https://doi.org/10.1083/jcb.144.5.963)
**PMC:** [PMC2148208](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2148208/)

## Abstract

1. J Cell Biol. 1999 Mar 8;144(5):963-75. doi: 10.1083/jcb.144.5.963.

The cortical localization of the microtubule orientation protein, Kar9p, is 
dependent upon actin and proteins required for polarization.

Miller RK(1), Matheos D, Rose MD.

Author information:
(1)Department of Molecular Biology, Lewis Thomas Laboratory, Princeton 
University, Princeton, New Jersey 08544, USA.

In the yeast Saccharomyces cerevisiae, positioning of the mitotic spindle 
requires both the cytoplasmic microtubules and actin. Kar9p is a novel cortical 
protein that is required for the correct position of the mitotic spindle and the 
orientation of the cytoplasmic microtubules. Green fluorescent protein (GFP)- 
Kar9p localizes to a single spot at the tip of the growing bud and the mating 
projection. However, the cortical localization of Kar9p does not require 
microtubules (Miller, R.K., and M.D. Rose. 1998. J. Cell Biol. 140: 377), 
suggesting that Kar9p interacts with other proteins at the cortex. To 
investigate Kar9p's cortical interactions, we treated cells with the 
actin-depolymerizing drug, latrunculin-A. In both shmoos and mitotic cells, 
Kar9p's cortical localization was completely dependent on polymerized actin. 
Kar9p localization was also altered by mutations in four genes, spa2Delta, 
pea2Delta, bud6Delta, and bni1Delta, required for normal polarization and actin 
cytoskeleton functions and, of these, bni1Delta affected Kar9p localization most 
severely. Like kar9Delta, bni1Delta mutants exhibited nuclear positioning 
defects during mitosis and in shmoos. Furthermore, like kar9Delta, the bni1Delta 
mutant exhibited misoriented cytoplasmic microtubules in shmoos. Genetic 
analysis placed BNI1 in the KAR9 pathway for nuclear migration. However, 
analysis of kar9Delta bni1Delta double mutants suggested that Kar9p retained 
some function in bni1Delta mitotic cells. Unlike the polarization mutants, 
kar9Delta shmoos had a normal morphology and diploids budded in the correct 
bipolar pattern. Furthermore, Bni1p localized normally in kar9Delta. We conclude 
that Kar9p's function is specific for cytoplasmic microtubule orientation and 
that Kar9p's role in nuclear positioning is to coordinate the interactions 
between the actin and microtubule networks.

DOI: 10.1083/jcb.144.5.963
PMCID: PMC2148208
PMID: 10085294 [Indexed for MEDLINE]

## Full Text

The simple budding yeast, Saccharomyces cerevisiae, is a highly polarized cell. During mitosis, almost all cell surface growth and secretion occur in the nascent bud (Field and Schekman, 1980). Underlying the polarized growth and secretion is a highly polarized actin cytoskeleton (Bretscher et al., 1994; Welch et al., 1994). Cortical patches of actin are found clustered in the bud and actin cables run from the mother cell towards the bud. At each division, the choice of position of the nascent bud is under strict genetic control. Haploids preferentially bud at the end marked by their birth scar, the axial pattern, and diploids tend to alternate between the two ends of the cell, the bipolar pattern (Freifelder, 1960; Chant and Pringle, 1995; Yang et al., 1997). During mating, the established axis of haploid cell division is perturbed such that the new growth is directed towards the tip of the mating projection. In response to a gradient of mating pheromone, the axis of growth re-orients towards the highest concentration of mating pheromone (Segall, 1993). Coincident with the change in the growth axis, the actin cytoskeleton redistributes such that actin cortical patches become clustered in the tip of the projection and cables run to the tip (Hasek et al., 1987; Read et al., 1992).

For efficient cell division and conjugation, the nucleus must also become oriented with respect to the growth axis of the cell. In haploid cells, the telophase of the previous cell cycle leaves the nuclear envelope–embedded microtubule organizing center (called the spindle pole body or SPB)1 oriented away from the next site of bud emergence. Thus, the haploid nucleus must rotate up to 180° each cell cycle. Before anaphase, the nucleus also migrates up to the neck of the nascent bud. The result of these movements is that the nuclear spindle becomes aligned with the growth axis adjacent to the bud, so that anaphase spindle elongation occurs through the neck and into the bud (Kahana et al., 1995; Yeh et al., 1995). Defects in nuclear orientation and migration result in misoriented spindles, anaphase in the mother cell, and binucleate mother cells.

During mating, the nucleus becomes oriented with respect to the mating projection. The nucleus moves up close to the mating projection with the spindle pole body pointing towards the tip of the projection (Byers and Goetsch, 1975; Hasek et al., 1987; Rose and Fink, 1987; Meluh and Rose, 1990). Defects in nuclear orientation or migration during mating lead to zygotes in which the two haploid nuclei fail to congress and fuse (for review see Rose, 1996).

Nuclear orientation and migration are dependent on microtubules that are attached to the nucleus at the SPB (Byers, 1981). During mitosis, mutations in the gene for β-tubulin, TUB2, were used to demonstrate that the cytoplasmic microtubules specifically were required to position and maintain the nucleus near the bud neck (Palmer et al., 1992; Sullivan and Huffaker, 1992). During mating as well, microtubule depolymerizing drugs and mutations have shown the requirement for microtubules in nuclear orientation and migration (for review see Rose, 1996). In each case, cytoplasmic microtubules are seen to enter the bud or mating projection and apparently make contact with cortical sites (Carminati and Stearns, 1997; Miller and Rose, 1998). Thus, it seems likely that interactions between cortical proteins and the cytoplasmic microtubules are a key element in nuclear migration and orientation.

In addition, the actin cytoskeleton has also been implicated in nuclear migration. Using a temperature-sensitive act1-4 allele, Koshland and colleagues found that preoriented spindles become misoriented upon loss of actin structures (Palmer et al., 1992). They suggested that the dependence of spindle orientation upon both microtubules and actin was consistent with cytoplasmic microtubules becoming tethered to specific sites on the cell cortex (Palmer et al., 1992). Similar models have been used to explain related phenomena in other organisms, including the early centrosomal rotations in Caenorhabditis elegans (Hyman and White, 1987) and the asymmetric positioning of the mitotic spindle in Chaetopterus oocytes (Lutz et al., 1988).

Several lines of evidence point to Kar9p as a good candidate to mediate the proposed interaction between actin and microtubules. Originally identified in a screen for bilateral karyogamy mutants (Kurihara et al., 1994), Kar9p plays an important role in nuclear migration in both mating and mitosis (Miller et al., 1998). Mutations in kar9 result in specific defects in cytoplasmic microtubule orientation into the mating projection and the bud. Green fluorescent protein (GFP)–Kar9p localizes as a single dot at the tip of the mating projection and at the tip of the growing bud, coincident with the tip of a bundle of cortically directed cytoplasmic microtubules. However, Kar9p localization at the cortex is independent of microtubules.

In addition to Kar9p, several microtubule motor proteins have been implicated in nuclear migration. The kinesin-related protein, Kip3p is involved in the initial alignment of the preanaphase nucleus at the bud neck (DeZwaan et al., 1997). Because kip3Δ causes very long and misoriented cytoplasmic microtubules (Cottingham and Hoyt, 1997; Miller et al., 1998), Kip3p may act by the destabilization of cytoplasmic microtubules. Cytoplasmic dynein function is required after Kip3p for the translocation of the nucleus through the bud neck (Kahana et al., 1995; Yeh et al., 1995; DeZwaan et al., 1997). Components of the dynein-activating (dynactin) complex including Jnm1p (McMillan and Tatchell, 1994), Act5p (Clark and Meyer, 1994; Muhua et al., 1994), and Nip100p (Kahana et al., 1998) are likely to contribute to the dynein-directed step in nuclear migration. Finally, another kinesin-related motor protein, Kip2p, is required for nuclear migration (Cottingham and Hoyt, 1997; Miller et al., 1998). Since kip2 mutants have very short or absent cytoplasmic microtubules, Kip2p may function to stabilize microtubules in vivo and appears to act in opposition to Kip3p.

Abundant genetic data suggests that there are at least two major, partially independent pathways for nuclear orientation and migration (Miller et al., 1998; Miller and Rose, 1998). Notably, with the exception of actin and tubulin, none of the genes involved in nuclear migration are essential for viability, and single deletion mutations do not result in severe defects. However, crosses between the various mutants suggest that Kar9p and Kip3p act in one pathway, whereas dynein, components of the dynactin complex, and Kip2p act in a second pathway.

In this paper, we examine the determinants for the localization of Kar9p to the cortex. We find that Kar9p localization is strongly dependent upon actin. In addition, Kar9p localization is dependent upon several actin-interacting polarization proteins, Spa2p, Pea2p, Bud6p, and Bni1p. Each of these proteins is found localized at the tip of the shmoo projection and tip of the growing bud (Snyder, 1989; Jansen et al., 1996; Valtz and Herskowitz, 1996; Amberg et al., 1997; Evangelista et al., 1997). Several lines of evidence suggest that these proteins act together in the cell. First, mutations in these genes cause similar defects in the diploid bipolar budding pattern (Snyder, 1989; Valtz and Herskowitz, 1996; Zahner et al., 1996; Amberg et al., 1997). Second, biochemical methods demonstrate that Spa2p, Pea2p, and Bud6p are associated with each other in a complex in vivo (Sheu et al., 1998). Interestingly, Spa2p has also been implicated in the signaling pathways for two mitogen-activated protein kinase cascades (Sheu et al., 1998). Bud6p was found to interact with actin by two-hybrid studies (Amberg et al., 1997), raising the possibility that these three proteins may function together to regulate the actin cytoskeleton (Amberg et al., 1997; Sheu et al., 1998).

Bni1p shows the strongest effects on Kar9p localization. Bni1p is a member of the widely conserved formin family of proteins, which includes mouse limb deformity (Woychik et al., 1990), Drosophila diaphanous (Castrillon and Wasserman, 1994) and cappuccino (Emmons et al., 1995), Aspergillus SepA (Harris et al., 1997) and FigA (Marhoul and Adams, 1995), and Schizosaccharomyces pombe Fus1 (Petersen et al., 1998). Bni1p binds to the actin-binding proteins, Bud6p and profilin (Evangelista et al., 1997; Imamura et al., 1997) and is a target of the rho-related GTPases, Cdc42p and Rho1p (Kohno et al., 1996; Evangelista et al., 1997; Imamura et al., 1997). Thus, Bni1p plays an important role in regulating the actin cytoskeleton.

In this and an accompanying paper (Lee et al., 1999), it is shown that bni1Δ mutants also exhibit a defect in the migration of the nucleus to the bud neck, consistent with their role in Kar9p localization. We also demonstrate the presence of microtubule orientation defects in mating bni1Δ mutants. Furthermore, genetic analysis places BNI1 in the KAR9 pathway for nuclear migration. However, our data suggest that Kar9p is at least partially active in bni1Δ mitotic cells. Furthermore, Kar9p is not required for at least two of Bni1p's functions in the cell. Taken together, these data suggest that Kar9p interacts with Bni1p as part of the link between microtubule-dependent nuclear movement and the cortical actin-dependent positional information.

Materials and MethodsCell Culture and Fixation MethodsCells were grown in yeast extract peptone dextrose (YPD) or in synthetic complete (SC) media as previously described (Miller and Rose, 1998). For experiments in which cells were induced with mating pheromone, synthetic α-factor (Department of Molecular Biology, Syn/Seq facility, Princeton University) was added to a final concentration of 10 μg/ml.For experiments localizing GFP-Kar9p, cultures of each strain containing the plasmid pMR3465 were grown to saturation in SC media without leucine (SC-leucine). The precultures were then diluted 500-fold into SC-leucine media containing 2% raffinose instead of glucose and grown for 15–18 h at 30°C to early exponential phase. Expression of GFP-Kar9p was induced for 2–2.5 h at 30°C by the addition of galactose to 2%. To express GFP-Kar9p in shmoos, α-factor was added at the time of galactose induction. Cells were fixed by the addition of formaldehyde to 3.7% for 10 min and washed twice with PBS.For localization of GFP-Bni1p, cells were treated as described above, with the exception that they were grown in SC media without uracil and the expression of GFP-Bni1p was induced by the addition of galactose for 4 h.For determination of the cell-cycle distribution and nuclear position in different mutants, saturated overnight cultures were subcultured into YPD preincubated at either 30° or 14°C. Cells were grown to mid-exponential phase and fixed in methanol:acetic acid (3:1), washed twice in PBS, and stained with 4′,6-diamidino-2-phenylindole (DAPI; Accurate Chemical and Scientific Corp.).For the analysis of the budding pattern, a/α diploid cells homozygous for the indicated mutations were grown to late exponential phase in YPD at 30°C. Cells were then stained with Calcofluor white (Sigma Chemical Co.) to visualize bud scars as described (Pringle, 1991). Only those cells with three or more bud scars were scored. If the bud scars were located at both poles, the cells were scored as bipolar. All other patterns were scored as random.Latrunculin-A TreatmentThe wild-type strain, MS1556, expressing the GFP-Kar9p plasmid was grown to early exponential phase as described above. Mitotic cells or pheromone-treated cells (shmoos) were concentrated 10-fold, and latrunculin-A (LAT-A) was added to 200 μm (Molecular Probes, Inc.) in DMSO for 10 min. Control cells were treated with an equivalent volume of DMSO. Cells were then collected by centrifugation and fixed for observation of the GFP fluorescence as described above or for staining of actin with rhodamine-phalloidin. For rhodamine-phalloidin staining, cells were collected by centrifugation and resuspended in 0.1% Triton X-100, 3.7% formaldehyde in PBS at room temperature for 30 min. Cells were washed once in PBS and resuspended in 3.7% formaldehyde/PBS at 30°C for 2 h. Cells were washed twice in PBS and resuspended in 90 μl PBS. Cells were stained by the addition of 30 μl rhodamine-phalloidin (0.2 U/μl; Molecular Probes, Inc.) in methanol at room temperature in the dark for 90–120 min. Cells were then washed once in PBS and scored by fluorescence microscopy.Strain ConstructionYeast strains used in the course of this study are listed in Table I. Plasmids used in strain construction are listed in Table II. Isogenic deletion mutants were constructed in the wild-type strain, MS1556, derived from S288C. All oligonucleotides used for deletion construction were made by the Department of Molecular Biology Syn/Seq Facility, Princeton University. The bni1Δ was made by cutting p321 (Evangelista et al., 1997) with HindIII and XhoI and transforming into MS1556. This resulted in the bni1Δ strains, MS5212 and MS5340. To construct bnr1Δ::HIS3, all but the first codon of BNR1 was replaced with HIS3 using the one-step gene replacement method (Rothstein, 1983). The disruption fragment was generated by PCR using the following two oligonucleotides (BNR1 sequence is shown in normal font and HIS3 sequence is shown in italics): 5′ TTTTGAAGATTACATAGTGATGAT GATCGTGACACAAAAGCAGATAAAAAAATAGCACAATCATCAGCGATGCCGTTTTAAGAGCTTGGTG 3′ and 5′ AGCGATTGCGAATATTGTCCATTTCTTTATATAAGCTCCACAACTACATAAATACTAAGTC T T CACTA CGAG TTCAAGAGAAAAAAAAAG 3′ . Plasmid pRS403 (Sikorski and Hieter, 1989) was used as the template for the HIS3 portion of the construct. MS1556 was transformed to HIS3 prototrophy to create MS5794. Replacement of BNR1 was confirmed by PCR analysis.To construct bud6Δ::HIS3, all but the first codon of BUD6 wasreplaced by the HIS3 gene using the one step gene replacement method (Rothstein, 1983). The disruption fragment was generated by PCR using the following forward and reverse primers (BUD6 sequence is in normal font and HIS3 sequence is shown in italics): 5′ ACCCAAGAAAAG- GGAAAAAGAGTAGAAGCTGGCTGCCAAATTGGTGTAGTAATCCT CGTATTATTTTAAAATTAGATG CCG TTTT AAGAGC TT G-GTG 3′ and 5′ TATTGCCGCAAACTTTGTAATAAGCCAAAAG-CACTAA T C T C TTTTCCGTTAGC TTTCATAAAATTAGTGTATTTACGAGTTCAAGAGAAAAAAAAAG 3′. Plasmid, pRS403 (Sikorski and Hieter, 1989), was used as the template for the HIS3 portion of the disruption construct. MS1556 was transformed to yield the BUD6 deletion strain, MS5849 and MS5850. The deletion was confirmed at both the 5′ and 3′ ends by PCR analysis.To construct a pea2Δ mutation, pNV44 (Valtz and Herskowitz, 1996) was digested with KpnI and BamH1. The resulting 2.0-kb band was gel purified and transformed into the wild-type strain, MS1556, resulting in disruption of the endogenous PEA2 locus by the one-step gene replacement (Rothstein, 1983). Two independent colonies, MS5229 and MS5230, were isolated.A spa2Δ mutation was made by the one-step gene replacement method. p210 (in which the SacI/SphI fragment of SPA2 was replaced by URA3), a kind gift from Michael Snyder (Yale University, New Haven, CT), was digested with HindIII and SalI. A 4.0-kb band was gel purified and transformed into the wild-type strain, MS1556, creating the SPA2 disruptants, MS5208 and MS5209.MicroscopyMicroscopy was carried out on an Axiophot microscope equipped with a 1.4 NA 100× Neofluor lens (Carl Zeiss, Inc.) or a 1.3 NA 100× UplanFl iris lens (Olympus Corp.). Images were recorded using a SIT Video Camera 3200 with a camera controller C2400 (Hamamatsu Corp.). Images were initially processed using Omnex Image processing unit (Imagen) and captured to computer disk using a Scion image capture board. Adobe Photoshop 4.0 was used to further optimize contrast.

Materials and Methods

Cell Culture and Fixation MethodsCells were grown in yeast extract peptone dextrose (YPD) or in synthetic complete (SC) media as previously described (Miller and Rose, 1998). For experiments in which cells were induced with mating pheromone, synthetic α-factor (Department of Molecular Biology, Syn/Seq facility, Princeton University) was added to a final concentration of 10 μg/ml.For experiments localizing GFP-Kar9p, cultures of each strain containing the plasmid pMR3465 were grown to saturation in SC media without leucine (SC-leucine). The precultures were then diluted 500-fold into SC-leucine media containing 2% raffinose instead of glucose and grown for 15–18 h at 30°C to early exponential phase. Expression of GFP-Kar9p was induced for 2–2.5 h at 30°C by the addition of galactose to 2%. To express GFP-Kar9p in shmoos, α-factor was added at the time of galactose induction. Cells were fixed by the addition of formaldehyde to 3.7% for 10 min and washed twice with PBS.For localization of GFP-Bni1p, cells were treated as described above, with the exception that they were grown in SC media without uracil and the expression of GFP-Bni1p was induced by the addition of galactose for 4 h.For determination of the cell-cycle distribution and nuclear position in different mutants, saturated overnight cultures were subcultured into YPD preincubated at either 30° or 14°C. Cells were grown to mid-exponential phase and fixed in methanol:acetic acid (3:1), washed twice in PBS, and stained with 4′,6-diamidino-2-phenylindole (DAPI; Accurate Chemical and Scientific Corp.).For the analysis of the budding pattern, a/α diploid cells homozygous for the indicated mutations were grown to late exponential phase in YPD at 30°C. Cells were then stained with Calcofluor white (Sigma Chemical Co.) to visualize bud scars as described (Pringle, 1991). Only those cells with three or more bud scars were scored. If the bud scars were located at both poles, the cells were scored as bipolar. All other patterns were scored as random.

Cell Culture and Fixation Methods

Cells were grown in yeast extract peptone dextrose (YPD) or in synthetic complete (SC) media as previously described (Miller and Rose, 1998). For experiments in which cells were induced with mating pheromone, synthetic α-factor (Department of Molecular Biology, Syn/Seq facility, Princeton University) was added to a final concentration of 10 μg/ml.

For experiments localizing GFP-Kar9p, cultures of each strain containing the plasmid pMR3465 were grown to saturation in SC media without leucine (SC-leucine). The precultures were then diluted 500-fold into SC-leucine media containing 2% raffinose instead of glucose and grown for 15–18 h at 30°C to early exponential phase. Expression of GFP-Kar9p was induced for 2–2.5 h at 30°C by the addition of galactose to 2%. To express GFP-Kar9p in shmoos, α-factor was added at the time of galactose induction. Cells were fixed by the addition of formaldehyde to 3.7% for 10 min and washed twice with PBS.

For localization of GFP-Bni1p, cells were treated as described above, with the exception that they were grown in SC media without uracil and the expression of GFP-Bni1p was induced by the addition of galactose for 4 h.

For determination of the cell-cycle distribution and nuclear position in different mutants, saturated overnight cultures were subcultured into YPD preincubated at either 30° or 14°C. Cells were grown to mid-exponential phase and fixed in methanol:acetic acid (3:1), washed twice in PBS, and stained with 4′,6-diamidino-2-phenylindole (DAPI; Accurate Chemical and Scientific Corp.).

For the analysis of the budding pattern, a/α diploid cells homozygous for the indicated mutations were grown to late exponential phase in YPD at 30°C. Cells were then stained with Calcofluor white (Sigma Chemical Co.) to visualize bud scars as described (Pringle, 1991). Only those cells with three or more bud scars were scored. If the bud scars were located at both poles, the cells were scored as bipolar. All other patterns were scored as random.

Latrunculin-A TreatmentThe wild-type strain, MS1556, expressing the GFP-Kar9p plasmid was grown to early exponential phase as described above. Mitotic cells or pheromone-treated cells (shmoos) were concentrated 10-fold, and latrunculin-A (LAT-A) was added to 200 μm (Molecular Probes, Inc.) in DMSO for 10 min. Control cells were treated with an equivalent volume of DMSO. Cells were then collected by centrifugation and fixed for observation of the GFP fluorescence as described above or for staining of actin with rhodamine-phalloidin. For rhodamine-phalloidin staining, cells were collected by centrifugation and resuspended in 0.1% Triton X-100, 3.7% formaldehyde in PBS at room temperature for 30 min. Cells were washed once in PBS and resuspended in 3.7% formaldehyde/PBS at 30°C for 2 h. Cells were washed twice in PBS and resuspended in 90 μl PBS. Cells were stained by the addition of 30 μl rhodamine-phalloidin (0.2 U/μl; Molecular Probes, Inc.) in methanol at room temperature in the dark for 90–120 min. Cells were then washed once in PBS and scored by fluorescence microscopy.

Latrunculin-A Treatment

The wild-type strain, MS1556, expressing the GFP-Kar9p plasmid was grown to early exponential phase as described above. Mitotic cells or pheromone-treated cells (shmoos) were concentrated 10-fold, and latrunculin-A (LAT-A) was added to 200 μm (Molecular Probes, Inc.) in DMSO for 10 min. Control cells were treated with an equivalent volume of DMSO. Cells were then collected by centrifugation and fixed for observation of the GFP fluorescence as described above or for staining of actin with rhodamine-phalloidin. For rhodamine-phalloidin staining, cells were collected by centrifugation and resuspended in 0.1% Triton X-100, 3.7% formaldehyde in PBS at room temperature for 30 min. Cells were washed once in PBS and resuspended in 3.7% formaldehyde/PBS at 30°C for 2 h. Cells were washed twice in PBS and resuspended in 90 μl PBS. Cells were stained by the addition of 30 μl rhodamine-phalloidin (0.2 U/μl; Molecular Probes, Inc.) in methanol at room temperature in the dark for 90–120 min. Cells were then washed once in PBS and scored by fluorescence microscopy.

Strain ConstructionYeast strains used in the course of this study are listed in Table I. Plasmids used in strain construction are listed in Table II. Isogenic deletion mutants were constructed in the wild-type strain, MS1556, derived from S288C. All oligonucleotides used for deletion construction were made by the Department of Molecular Biology Syn/Seq Facility, Princeton University. The bni1Δ was made by cutting p321 (Evangelista et al., 1997) with HindIII and XhoI and transforming into MS1556. This resulted in the bni1Δ strains, MS5212 and MS5340. To construct bnr1Δ::HIS3, all but the first codon of BNR1 was replaced with HIS3 using the one-step gene replacement method (Rothstein, 1983). The disruption fragment was generated by PCR using the following two oligonucleotides (BNR1 sequence is shown in normal font and HIS3 sequence is shown in italics): 5′ TTTTGAAGATTACATAGTGATGAT GATCGTGACACAAAAGCAGATAAAAAAATAGCACAATCATCAGCGATGCCGTTTTAAGAGCTTGGTG 3′ and 5′ AGCGATTGCGAATATTGTCCATTTCTTTATATAAGCTCCACAACTACATAAATACTAAGTC T T CACTA CGAG TTCAAGAGAAAAAAAAAG 3′ . Plasmid pRS403 (Sikorski and Hieter, 1989) was used as the template for the HIS3 portion of the construct. MS1556 was transformed to HIS3 prototrophy to create MS5794. Replacement of BNR1 was confirmed by PCR analysis.To construct bud6Δ::HIS3, all but the first codon of BUD6 wasreplaced by the HIS3 gene using the one step gene replacement method (Rothstein, 1983). The disruption fragment was generated by PCR using the following forward and reverse primers (BUD6 sequence is in normal font and HIS3 sequence is shown in italics): 5′ ACCCAAGAAAAG- GGAAAAAGAGTAGAAGCTGGCTGCCAAATTGGTGTAGTAATCCT CGTATTATTTTAAAATTAGATG CCG TTTT AAGAGC TT G-GTG 3′ and 5′ TATTGCCGCAAACTTTGTAATAAGCCAAAAG-CACTAA T C T C TTTTCCGTTAGC TTTCATAAAATTAGTGTATTTACGAGTTCAAGAGAAAAAAAAAG 3′. Plasmid, pRS403 (Sikorski and Hieter, 1989), was used as the template for the HIS3 portion of the disruption construct. MS1556 was transformed to yield the BUD6 deletion strain, MS5849 and MS5850. The deletion was confirmed at both the 5′ and 3′ ends by PCR analysis.To construct a pea2Δ mutation, pNV44 (Valtz and Herskowitz, 1996) was digested with KpnI and BamH1. The resulting 2.0-kb band was gel purified and transformed into the wild-type strain, MS1556, resulting in disruption of the endogenous PEA2 locus by the one-step gene replacement (Rothstein, 1983). Two independent colonies, MS5229 and MS5230, were isolated.A spa2Δ mutation was made by the one-step gene replacement method. p210 (in which the SacI/SphI fragment of SPA2 was replaced by URA3), a kind gift from Michael Snyder (Yale University, New Haven, CT), was digested with HindIII and SalI. A 4.0-kb band was gel purified and transformed into the wild-type strain, MS1556, creating the SPA2 disruptants, MS5208 and MS5209.

Strain Construction

Yeast strains used in the course of this study are listed in Table I. Plasmids used in strain construction are listed in Table II. Isogenic deletion mutants were constructed in the wild-type strain, MS1556, derived from S288C. All oligonucleotides used for deletion construction were made by the Department of Molecular Biology Syn/Seq Facility, Princeton University. The bni1Δ was made by cutting p321 (Evangelista et al., 1997) with HindIII and XhoI and transforming into MS1556. This resulted in the bni1Δ strains, MS5212 and MS5340.

To construct bnr1Δ::HIS3, all but the first codon of BNR1 was replaced with HIS3 using the one-step gene replacement method (Rothstein, 1983). The disruption fragment was generated by PCR using the following two oligonucleotides (BNR1 sequence is shown in normal font and HIS3 sequence is shown in italics): 5′ TTTTGAAGATTACATAGTGATGAT GATCGTGACACAAAAGCAGATAAAAAAATAGCACAATCATCAGCGATGCCGTTTTAAGAGCTTGGTG 3′ and 5′ AGCGATTGCGAATATTGTCCATTTCTTTATATAAGCTCCACAACTACATAAATACTAAGTC T T CACTA CGAG TTCAAGAGAAAAAAAAAG 3′ . Plasmid pRS403 (Sikorski and Hieter, 1989) was used as the template for the HIS3 portion of the construct. MS1556 was transformed to HIS3 prototrophy to create MS5794. Replacement of BNR1 was confirmed by PCR analysis.

To construct bud6Δ::HIS3, all but the first codon of BUD6 wasreplaced by the HIS3 gene using the one step gene replacement method (Rothstein, 1983). The disruption fragment was generated by PCR using the following forward and reverse primers (BUD6 sequence is in normal font and HIS3 sequence is shown in italics): 5′ ACCCAAGAAAAG- GGAAAAAGAGTAGAAGCTGGCTGCCAAATTGGTGTAGTAATCCT CGTATTATTTTAAAATTAGATG CCG TTTT AAGAGC TT G-GTG 3′ and 5′ TATTGCCGCAAACTTTGTAATAAGCCAAAAG-CACTAA T C T C TTTTCCGTTAGC TTTCATAAAATTAGTGTATTTACGAGTTCAAGAGAAAAAAAAAG 3′. Plasmid, pRS403 (Sikorski and Hieter, 1989), was used as the template for the HIS3 portion of the disruption construct. MS1556 was transformed to yield the BUD6 deletion strain, MS5849 and MS5850. The deletion was confirmed at both the 5′ and 3′ ends by PCR analysis.

To construct a pea2Δ mutation, pNV44 (Valtz and Herskowitz, 1996) was digested with KpnI and BamH1. The resulting 2.0-kb band was gel purified and transformed into the wild-type strain, MS1556, resulting in disruption of the endogenous PEA2 locus by the one-step gene replacement (Rothstein, 1983). Two independent colonies, MS5229 and MS5230, were isolated.

A spa2Δ mutation was made by the one-step gene replacement method. p210 (in which the SacI/SphI fragment of SPA2 was replaced by URA3), a kind gift from Michael Snyder (Yale University, New Haven, CT), was digested with HindIII and SalI. A 4.0-kb band was gel purified and transformed into the wild-type strain, MS1556, creating the SPA2 disruptants, MS5208 and MS5209.

MicroscopyMicroscopy was carried out on an Axiophot microscope equipped with a 1.4 NA 100× Neofluor lens (Carl Zeiss, Inc.) or a 1.3 NA 100× UplanFl iris lens (Olympus Corp.). Images were recorded using a SIT Video Camera 3200 with a camera controller C2400 (Hamamatsu Corp.). Images were initially processed using Omnex Image processing unit (Imagen) and captured to computer disk using a Scion image capture board. Adobe Photoshop 4.0 was used to further optimize contrast.

Microscopy was carried out on an Axiophot microscope equipped with a 1.4 NA 100× Neofluor lens (Carl Zeiss, Inc.) or a 1.3 NA 100× UplanFl iris lens (Olympus Corp.). Images were recorded using a SIT Video Camera 3200 with a camera controller C2400 (Hamamatsu Corp.). Images were initially processed using Omnex Image processing unit (Imagen) and captured to computer disk using a Scion image capture board. Adobe Photoshop 4.0 was used to further optimize contrast.

ResultsKar9p Localization Is Actin DependentKar9p is a cortical protein that plays an important role in orienting cytoplasmic microtubules. To understand the mechanism of Kar9p microtubule orientation, we sought to identify the cellular components involved in localizing Kar9p to its cortical site. Previous work demonstrated that Kar9p's cortical localization is independent of microtubules (Miller and Rose, 1998). Therefore, Kar9p's cortical localization must be dependent on other peripheral determinants.As a first step, we used rhodamine-phalloidin to determine whether GFP-Kar9p colocalizes with cortical actin in pheromone-treated cells. In every case (n = 20), we found GFP-Kar9p coincident with actin at the tip of the shmoo (data not shown). Although in several cases we observed a well-resolved actin dot that coincided with the GFP-Kar9p, most often the cortical actin dots were too concentrated to resolve individual structures. These results suggested that Kar9p might be associated with cortical actin structures.To determine whether the actin cytoskeleton is required for Kar9p localization, we used the actin-depolymerizing drug LAT-A. Confirming earlier work (Ayscough et al., 1997), we found that the actin network was 100% depolymerized within 10 min after treatment with LAT-A, as shown by staining with rhodamine-phalloidin (data not shown). We first examined the localization of a functional GFP-Kar9p construct in pheromone-treated cells. In the control experiments, 84% of shmoos exhibited a single dot of GFP-Kar9p fluorescence at the tip of the mating projection (Fig. 1 A). In contrast, when shmoos were treated with LAT-A, only 8% of the shmoos exhibited the cortical dot at the tip of the projection. Instead, in 74% of the LAT-A–treated shmoos, GFP-Kar9p fluorescence was found in a line extending from the nucleus. Based on previous experiments, we expect that this corresponds to cytoplasmic microtubules (Miller and Rose, 1998). In addition, 19% of the LAT-A–treated shmoos exhibited no localization of the GFP-Kar9p fluorescence. Thus, depolymerization of the actin cytoskeleton results in rapid loss of the cortical localization of Kar9p and redistribution onto the cytoplasmic microtubules. We next examined the requirement for polymerized actin for Kar9p localization in mitotically dividing cells. Unbudded and most post-anaphase large budded cells do not exhibit localized GFP-Kar9p (Miller and Rose, 1998), so we restricted our observations to cells with medium to large buds. In the control cells, 72% displayed a single dot of GFP-Kar9p fluorescence at the tip of the bud (Fig. 1 B). When mitotic cells were treated with LAT-A, only 13% exhibited the dot at the tip of the bid. Instead, 59% of the cells exhibited a dot of GFP-Kar9p fluorescence at two alternate locations, coincident with the DAPI staining and/ or close to the mother-bud neck (Fig. 1 B). The GFP fluorescence coincident with the nuclear DAPI stain is likely to correspond to SPB localization. Another 21% of the LAT-A–treated mitotic cells exhibited no localized GFP-Kar9p fluorescence. Thus, in >80% of LAT-A–treated cells, GFP-Kar9p localization was not found at the tip of the bud. From these results, we conclude that GFP-Kar9p localization in both shmoos and mitotically dividing cells is dependent on polymerized actin.Kar9p Localization Is Affected by the Polarization Mutants, spa2Δ, pea2Δ, bud6Δ, and bni1ΔAlthough Kar9p's localization was actin dependent, it was found only as a single dot and therefore did not colocalize with other cortical actin patches at the shmoo tip and bud cortex. Therefore, we reasoned that additional cortical actin-associated proteins might restrict Kar9p localization to one site. In this case, mutations in a subset of specific actin-associated proteins would eliminate or alter Kar9p localization. To test this idea, we used pheromone-treated cells to assay GFP-Kar9p localization in mutants that affect actin, polarization, and/or mating. Most mutations had no effect on GFP-Kar9p localization to the tip of the shmoo projection. These included genes affecting actin function, ABP1 (Drubin et al., 1988, 1990) and SLA1 (Holtzman et al., 1993); genes involved in GTPase signaling, BOI2 (Bender et al., 1996), RSR1/BUD1 (Bender and Pringle, 1989), RGA1 (Stevenson et al., 1995), and MSB1 (Bender and Pringle, 1989); and genes affecting cell fusion, RVS161 (Brizzio et al., 1998), FUS1 (Trueheart and Fink, 1989), and FUS2 (Trueheart et al., 1987). Furthermore, GFP-Kar9p localization in mitotic cells was not affected by mutations in the genes encoding two microtubule-dependent motor proteins, KIP3 and DHC1 (Miller et al., 1998).However, GFP-Kar9p localization in shmoos was affected by four mutations implicated in actin organization and polarization, spa2, pea2, bud6, and bni1. SPA2 and PEA2 are required for polarization in mating cells and mutations result in broader peanut-shaped shmoo projections (Valtz and Herskowitz, 1996). In mitotic cells, SPA2 and PEA2 are required for the diploid-specific bipolar budding pattern. Spa2p and Pea2p physically interact with each other and are interdependent for their own localization to the tips of shmoo projections (Valtz and Herskowitz, 1996). In the wild-type strain, 86% of the cells displayed GFP-Kar9p localization as a single dot at the tip of the shmoo projection. In contrast, only 15% of spa2Δ and 11% of pea2Δ shmoos exhibited GFP-Kar9p fluorescence as a single dot. Instead, 50% of spa2Δ and 55% of pea2Δ shmoos showed a new pattern in which GFP-Kar9p was found in a broad region over the shmoo tip. In addition, both mutants showed a significant number of cells with GFP-Kar9p in a line or in multiple “speckles” (Fig. 2 A). 
BUD6 is also required for the diploid-specific bipolar budding pattern and Bud6p (Aip3p) has been shown to interact with actin (Amberg et al., 1997). In the bud6Δ strain, only 41% of shmoos exhibited Kar9p fluorescence as a single dot (Fig. 2 A). Instead, a large percentage of the bud6Δ shmoos exhibited GFP-Kar9p fluorescence as a line (37%, compared with 8% in wild type, Fig. 2 A).
BNI1 encodes a formin homologue that has been implicated in the regulation of the actin cytoskeleton and cell polarization (Kohno et al., 1996; Evangelista et al., 1997), as well as bipolar budding (Zahner et al., 1996). Bni1-GFP also localizes to the shmoo tip (Evangelista et al., 1997). When GFP-Kar9p localization was examined in bni1Δ shmoos, only 12% exhibited Kar9p fluorescence as a cortical dot. Instead, 37% of the bni1Δ shmoos exhibited GFP-Kar9p in a line pattern and 41% in a speckled pattern throughout the shmoo (Fig 2 A).Because Kar9p also functions to orient cytoplasmic microtubules during vegetative growth (Miller and Rose, 1998), we sought to determine whether these four mutants also affected Kar9p localization during mitosis. In 85% of wild-type medium-to-large budded cells, GFP-Kar9p localized to a single dot at the tip of the bud. In the spa2Δ and pea2Δ cells, there was a modest reduction in the frequency of GFP-Kar9p localization to the bud tip, down to 75 and 74%, respectively (Fig. 2 B). Although modest, the reduction was observed in three independent experiments. Only 38% of the bud6Δ cells had a dot of GFP-Kar9p fluorescence at the tip of the bud. In 41% of the bud6Δ cells, GFP-Kar9p was mislocalized (Fig. 2 B) to the mother-bud neck and/or a region coincident with DAPI staining. Finally, in the bni1Δ cells, only 14% of cells had a dot of GFP-Kar9p at the tip of the bud. Instead, 70% of these cells had Kar9p fluorescence at the mother-bud neck and/ or a region coincident with DAPI staining (Fig. 2 B). Therefore, all four mutations affected Kar9p localization, although only in the bni1Δ strain was the magnitude similar to that of actin depolymerization.Because the deletion of BNI1 had such a strong effect on Kar9p localization, we next determined whether the two proteins are interdependent for localization. Accordingly, we examined the localization of Bni1p using a fully functional GFP-Bni1p fusion (a generous gift from Charlie Boone, Queens University, Kingston, Canada) in kar9Δ mutant shmoos and mitotic cells. In comparing kar9Δ to wild type, there were no obvious differences in the localization pattern of GFP-Bni1p to the presumptive bud site or the tip of the growing bud (data not shown). Therefore, Bni1p's association with the cortex is not dependent on Kar9p.
BNI1-related (BNR1) is another yeast gene encoding a formin homology domain protein. Bnr1p shares some functions with Bni1p (Imamura et al., 1997). We therefore determined whether BNR1 was also required for GFP-Kar9p localization. In both shmoos and mitotic cells, GFP-Kar9p localization in an isogenic bnr1Δ strain was indistinguishable from wild type (data not shown). Thus, localization of Kar9p to the cortex is a specific function of Bni1p.bni1Δ Mutants Exhibit Nuclear Migration DefectsSince spa2Δ, pea2Δ, bud6Δ, and bni1Δ mutants resulted in an altered localization for GFP-Kar9p, we wanted to determine whether these mutants also exhibited defects in nuclear migration like kar9Δ. Nuclear position was scored in asynchronous cultures using the DNA-specific fluorescent dye, DAPI (Table III). In the wild-type control, 20% of the cultures were large budded and, of these, 96% showed normal nuclear positions (i.e., located at the bud neck, or in anaphase across the bud neck, or in telophase with one nucleus each in the mother and bud). In contrast, 40% of kar9Δ mutants were large-budded (Table III), suggesting that kar9Δ has a cell cycle delay in mitosis. Consistent with our previous report (Miller and Rose, 1998), nuclear positioning was aberrant in up to 50% of the kar9Δ large-budded cells (i.e., nuclei not located at the bud neck, or anaphase or telophase occurring in the mother cell). The spa2Δ, pea2Δ, and bud6Δ mutants exhibited no obvious defects in nuclear position within the large-budded cells. However, like kar9Δ, the bni1Δ strain exhibited a significant defect in nuclear position; in 23% of the large-budded cells, anaphase was occurring within the mother cell. In contrast, only 2% of wild-type large-budded cells exhibited this phenotype (Table III). Unlike kar9Δ, only 5% of bni1Δ large-budded cells contained two nuclei within the mother cell (compared with 18% in kar9Δ). Similar results were obtained when the spa2Δ, pea2Δ, bud6Δ, and bni1Δ cultures were grown at 14°C (data not shown). Therefore, we conclude that bni1Δ is a nuclear migration mutant. However, the nuclear migration defect of bni1Δ is not as severe as that of kar9Δ. We next examined nuclear position in cells treated with mating pheromone. In wild type, the nuclei migrated to the base of the shmoo projection in 66% of the cells. As previously reported (Miller and Rose, 1998), the kar9Δ mutants exhibited a severe defect in nuclear migration to the shmoo projection, with 71% of the nuclei found in the center of the shmoo body (Table IV). The spa2Δ and pea2Δ mutants did not exhibit a nuclear migration defect, and may have localized better than the wild type, consistent with the continued localization of Kar9p at the shmoo tip. However, in 58% of the bud6Δ shmoos and 72% of the bni1Δ shmoos, the nuclei failed to migrate into the shmoo projection and were instead positioned in the center of the shmoo body (Table IV). Therefore, during mating, bud6Δ mutants exhibited a moderate defect and bni1Δ mutants exhibited a major defect in nuclear positioning. Microtubule Morphologies in MutantsWe next examined the microtubule morphologies of the mutants. In >90% of wild-type shmoos, the cytoplasmic microtubules were present as a single bundle that extended from the nucleus to the tip of the shmoo projection (Fig. 3). As previously reported, only 15% of the kar9Δ shmoos contained cytoplasmic microtubules that extended to the shmoo tip. In most kar9Δ shmoos, the cytoplasmic microtubule bundle was misoriented and extended in random directions. In the spa2Δ mutant shmoos, only 52% showed a single bundle of microtubules oriented towards the shmoo tip. Instead, 29% of the spa2Δ mutant shmoos exhibited a “spray” of microtubules extending to the shmoo tip, and 10% showed supernumerary microtubules oriented away from the shmoo tip (“forked pattern,” Fig. 3). The pea2Δ mutant showed a similar pattern, albeit significantly less severe than that of spa2Δ. The presence of additional cytoplasmic microtubules making contact with the cortex is consistent with Kar9p localizing over a broader region of the shmoo tip. The bni1Δ strain also showed a significant defect in microtubule orientation in the shmoo. Like spa2Δ, only 50% of bni1Δ shmoos exhibited the normal microtubule pattern. However, unlike spa2Δ, 39% of the bni1Δ mutant shmoos exhibited misoriented microtubules similar to the pattern seen with the kar9Δ (Fig. 3). Thus, in mating cells, the defect in microtubule orientation in the bni1Δ mutants is consistent with the aberrant localization of Kar9p.We next examined the microtubule morphology in bni1Δ mitotic cells. For this purpose, we focused on those cells exhibiting a nuclear positioning defect such that anaphase was occurring within the mother cell. As reported previously (Miller and Rose, 1998), when the rare wild-type cells with anaphase in the mother were examined, the cytoplasmic microtubules nearly always extended into the bud (90%, Table V). In contrast, in the kar9Δ mutant, the cytoplasmic microtubule bundles infrequently extended into the bud (31%) and were more often found in the mother cell (48%). However, in 79% of the bni1Δ mutant, the cytoplasmic microtubules were correctly extended into the bud. This result suggests that either the bni1Δ mutation also suppresses the microtubule orientation defect created by mislocalized Kar9p or that enough Kar9p was correctly localized in the bni1Δ mutant to facilitate microtubule orientation. To differentiate between these two models, we created kar9Δ bni1Δ double mutants and examined them for defects in nuclear positioning and cytoplasmic microtubule orientation. In 49% of the kar9Δ bni1Δ double mutant cells, cytoplasmic microtubules were misoriented (Table V) like the kar9Δ mutant. Therefore, Kar9p was still responsible for the normal microtubule orientation found in the bni1Δ single mutant cells, in spite of the defect in Kar9p localization. These results suggest that Kar9p localization must be partially independent of Bni1p. This result would also be consistent with the weaker defect in nuclear migration seen for bni1Δ compared with kar9Δ.We next examined the double mutants for defects in nuclear positioning. Compared with the kar9Δ single mutant, the kar9Δ bni1Δ double mutant large-budded cells were slightly more severe for defects in nuclear positioning (47 vs. 33% aberrant, Table VI). In unbudded cells, however, a significantly more severe defect was observed. In unbudded cells, very low percentages of abnormal cells were found in wild-type, kar9Δ, and bni1Δ single mutants (0, 4, and 0.5%, respectively). However, in the kar9Δ bni1Δ double mutant, 22% of the cells were either multinucleate or anucleate. Therefore, the double mutant cells often appeared to progress through cytokinesis, which almost never occurred with the single mutants. Genetic Analysis Places Bni1p in the Kar9p Pathway for Nuclear MigrationBecause of the synthetic lethality between kar9Δ and dhc1Δ, Kar9p and dynein are thought to function in separate, partially redundant pathways for nuclear migration (Miller and Rose, 1998). Because spa2Δ, pea2Δ, bud6Δ, and bni1Δ mutants each altered GFP-Kar9p localization, we wanted to know whether any of these mutations would show genetic interactions similar to those observed with kar9Δ. Isogenic marked deletions of each gene were created and crossed to marked kar9 and dhc1 deletions. The viability of the double mutants was assayed for growth 2–3 d after spore germination. The results of the crosses are summarized in Table VII. When spa2Δ, pea2Δ, bud6Δ, or bni1Δ were crossed with kar9Δ, all four double-mutant combinations were viable (Table VII, 1–4). Similarly, the spa2Δ dhc1Δ, and pea2Δ dhc1Δ double mutants were viable (Table VII, 6–8). The bud6Δ dhc1Δ double mutants were viable, but exibited a mild growth defect (Table VII, 8). However, when bni1Δ was crossed to dhc1Δ, all the predicted double mutants were barely viable and formed microcolonies (Table VII, 9). Similar microcolonies were also observed in synthetic lethal genetic interactions seen with kar9Δ, including dhc1Δ, bik1Δ, and kip2Δ (Miller et al., 1998; Miller and Rose, 1998). Therefore, we consider this to be a synthetic lethal interaction and conclude that Bni1p acts in the Kar9p pathway for nuclear migration. As a control, crosses were performed using a deletion of the related formin homology domain protein, Bnr1p. The bnr1Δ mutation was not synthetically lethal in combination with kar9Δ or dhc1Δ (Table VII, 5 and 10). Therefore, the interaction seen between dhc1Δ and bni1Δ is specific to Bni1p and not a general feature of formin homology proteins.In previous work, Kip3p was shown to function in the KAR9 pathway for nuclear migration (Miller et al., 1998). Because Bni1p also appears to function in the KAR9 pathway, we crossed the bni1Δ to the kip3Δ to determine whether Bni1p and Kip3p might also function in a common pathway. We found that the bni1Δ kip3Δ double mutant is viable, as was also reported by Lee et al. (1999). This suggests that Kar9p, Kip3p, and Bni1p function in a common pathway. Accordingly, the kar9Δ bni1Δ kip3Δ triple mutant would also be predicted to be viable. We found that the kar9Δ bni1Δ kip3Δ triple mutant was indeed viable (Table VII, 12), supporting the hypothesis that Kar9p, Kip3p, and Bni1p function together in a common pathway.kar9Δ Does Not Exhibit Bipolar Budding DefectsThe four mutations that affect Kar9p localization, spa2Δ, pea2Δ, bud6Δ, and bni1Δ, share an additional common phenotype of being defective in the diploid-specific bipolar budding pattern (Snyder, 1989; Valtz and Herskowitz, 1996; Zahner et al., 1996; Amberg et al., 1997). This provided an additional way of addressing whether Kar9p is simply a Bni1p-associated protein or functions specifically for microtubule orientation. Therefore, we examined whether kar9Δ mutants are also defective for bipolar budding. Diploid cells with three or more bud scars were scored to determine whether they showed the bipolar or random budding pattern (see Materials and Methods). In the wild-type control strain, 95% of the cells exhibited a bipolar budding pattern and 5% showed a random pattern (n = 263). In the pea2Δ strain, only 28% of the cells exhibited the bipolar budding pattern, while 72% budded randomly (n = 201). Unlike the pea2Δ mutant, the kar9Δ mutant exhibited the wild-type budding pattern (90% bipolar, n = 300). The kar9Δ mutants also did not display a defect in the haploid-specific axial budding pattern (data not shown). Taken together with the observation that kar9Δ mutant shmoos do not exhibit polarization defects, and that Bni1p localizes normally in the kar9Δ mutants, these results argue strongly that Kar9p's function with Bni1p is specific to the nuclear migration pathway.

Kar9p Localization Is Actin DependentKar9p is a cortical protein that plays an important role in orienting cytoplasmic microtubules. To understand the mechanism of Kar9p microtubule orientation, we sought to identify the cellular components involved in localizing Kar9p to its cortical site. Previous work demonstrated that Kar9p's cortical localization is independent of microtubules (Miller and Rose, 1998). Therefore, Kar9p's cortical localization must be dependent on other peripheral determinants.As a first step, we used rhodamine-phalloidin to determine whether GFP-Kar9p colocalizes with cortical actin in pheromone-treated cells. In every case (n = 20), we found GFP-Kar9p coincident with actin at the tip of the shmoo (data not shown). Although in several cases we observed a well-resolved actin dot that coincided with the GFP-Kar9p, most often the cortical actin dots were too concentrated to resolve individual structures. These results suggested that Kar9p might be associated with cortical actin structures.To determine whether the actin cytoskeleton is required for Kar9p localization, we used the actin-depolymerizing drug LAT-A. Confirming earlier work (Ayscough et al., 1997), we found that the actin network was 100% depolymerized within 10 min after treatment with LAT-A, as shown by staining with rhodamine-phalloidin (data not shown). We first examined the localization of a functional GFP-Kar9p construct in pheromone-treated cells. In the control experiments, 84% of shmoos exhibited a single dot of GFP-Kar9p fluorescence at the tip of the mating projection (Fig. 1 A). In contrast, when shmoos were treated with LAT-A, only 8% of the shmoos exhibited the cortical dot at the tip of the projection. Instead, in 74% of the LAT-A–treated shmoos, GFP-Kar9p fluorescence was found in a line extending from the nucleus. Based on previous experiments, we expect that this corresponds to cytoplasmic microtubules (Miller and Rose, 1998). In addition, 19% of the LAT-A–treated shmoos exhibited no localization of the GFP-Kar9p fluorescence. Thus, depolymerization of the actin cytoskeleton results in rapid loss of the cortical localization of Kar9p and redistribution onto the cytoplasmic microtubules. We next examined the requirement for polymerized actin for Kar9p localization in mitotically dividing cells. Unbudded and most post-anaphase large budded cells do not exhibit localized GFP-Kar9p (Miller and Rose, 1998), so we restricted our observations to cells with medium to large buds. In the control cells, 72% displayed a single dot of GFP-Kar9p fluorescence at the tip of the bud (Fig. 1 B). When mitotic cells were treated with LAT-A, only 13% exhibited the dot at the tip of the bid. Instead, 59% of the cells exhibited a dot of GFP-Kar9p fluorescence at two alternate locations, coincident with the DAPI staining and/ or close to the mother-bud neck (Fig. 1 B). The GFP fluorescence coincident with the nuclear DAPI stain is likely to correspond to SPB localization. Another 21% of the LAT-A–treated mitotic cells exhibited no localized GFP-Kar9p fluorescence. Thus, in >80% of LAT-A–treated cells, GFP-Kar9p localization was not found at the tip of the bud. From these results, we conclude that GFP-Kar9p localization in both shmoos and mitotically dividing cells is dependent on polymerized actin.

Kar9p Localization Is Actin Dependent

Kar9p is a cortical protein that plays an important role in orienting cytoplasmic microtubules. To understand the mechanism of Kar9p microtubule orientation, we sought to identify the cellular components involved in localizing Kar9p to its cortical site. Previous work demonstrated that Kar9p's cortical localization is independent of microtubules (Miller and Rose, 1998). Therefore, Kar9p's cortical localization must be dependent on other peripheral determinants.

As a first step, we used rhodamine-phalloidin to determine whether GFP-Kar9p colocalizes with cortical actin in pheromone-treated cells. In every case (n = 20), we found GFP-Kar9p coincident with actin at the tip of the shmoo (data not shown). Although in several cases we observed a well-resolved actin dot that coincided with the GFP-Kar9p, most often the cortical actin dots were too concentrated to resolve individual structures. These results suggested that Kar9p might be associated with cortical actin structures.

To determine whether the actin cytoskeleton is required for Kar9p localization, we used the actin-depolymerizing drug LAT-A. Confirming earlier work (Ayscough et al., 1997), we found that the actin network was 100% depolymerized within 10 min after treatment with LAT-A, as shown by staining with rhodamine-phalloidin (data not shown). We first examined the localization of a functional GFP-Kar9p construct in pheromone-treated cells. In the control experiments, 84% of shmoos exhibited a single dot of GFP-Kar9p fluorescence at the tip of the mating projection (Fig. 1 A). In contrast, when shmoos were treated with LAT-A, only 8% of the shmoos exhibited the cortical dot at the tip of the projection. Instead, in 74% of the LAT-A–treated shmoos, GFP-Kar9p fluorescence was found in a line extending from the nucleus. Based on previous experiments, we expect that this corresponds to cytoplasmic microtubules (Miller and Rose, 1998). In addition, 19% of the LAT-A–treated shmoos exhibited no localization of the GFP-Kar9p fluorescence. Thus, depolymerization of the actin cytoskeleton results in rapid loss of the cortical localization of Kar9p and redistribution onto the cytoplasmic microtubules.

We next examined the requirement for polymerized actin for Kar9p localization in mitotically dividing cells. Unbudded and most post-anaphase large budded cells do not exhibit localized GFP-Kar9p (Miller and Rose, 1998), so we restricted our observations to cells with medium to large buds. In the control cells, 72% displayed a single dot of GFP-Kar9p fluorescence at the tip of the bud (Fig. 1 B). When mitotic cells were treated with LAT-A, only 13% exhibited the dot at the tip of the bid. Instead, 59% of the cells exhibited a dot of GFP-Kar9p fluorescence at two alternate locations, coincident with the DAPI staining and/ or close to the mother-bud neck (Fig. 1 B). The GFP fluorescence coincident with the nuclear DAPI stain is likely to correspond to SPB localization. Another 21% of the LAT-A–treated mitotic cells exhibited no localized GFP-Kar9p fluorescence. Thus, in >80% of LAT-A–treated cells, GFP-Kar9p localization was not found at the tip of the bud. From these results, we conclude that GFP-Kar9p localization in both shmoos and mitotically dividing cells is dependent on polymerized actin.

Kar9p Localization Is Affected by the Polarization Mutants, spa2Δ, pea2Δ, bud6Δ, and bni1ΔAlthough Kar9p's localization was actin dependent, it was found only as a single dot and therefore did not colocalize with other cortical actin patches at the shmoo tip and bud cortex. Therefore, we reasoned that additional cortical actin-associated proteins might restrict Kar9p localization to one site. In this case, mutations in a subset of specific actin-associated proteins would eliminate or alter Kar9p localization. To test this idea, we used pheromone-treated cells to assay GFP-Kar9p localization in mutants that affect actin, polarization, and/or mating. Most mutations had no effect on GFP-Kar9p localization to the tip of the shmoo projection. These included genes affecting actin function, ABP1 (Drubin et al., 1988, 1990) and SLA1 (Holtzman et al., 1993); genes involved in GTPase signaling, BOI2 (Bender et al., 1996), RSR1/BUD1 (Bender and Pringle, 1989), RGA1 (Stevenson et al., 1995), and MSB1 (Bender and Pringle, 1989); and genes affecting cell fusion, RVS161 (Brizzio et al., 1998), FUS1 (Trueheart and Fink, 1989), and FUS2 (Trueheart et al., 1987). Furthermore, GFP-Kar9p localization in mitotic cells was not affected by mutations in the genes encoding two microtubule-dependent motor proteins, KIP3 and DHC1 (Miller et al., 1998).However, GFP-Kar9p localization in shmoos was affected by four mutations implicated in actin organization and polarization, spa2, pea2, bud6, and bni1. SPA2 and PEA2 are required for polarization in mating cells and mutations result in broader peanut-shaped shmoo projections (Valtz and Herskowitz, 1996). In mitotic cells, SPA2 and PEA2 are required for the diploid-specific bipolar budding pattern. Spa2p and Pea2p physically interact with each other and are interdependent for their own localization to the tips of shmoo projections (Valtz and Herskowitz, 1996). In the wild-type strain, 86% of the cells displayed GFP-Kar9p localization as a single dot at the tip of the shmoo projection. In contrast, only 15% of spa2Δ and 11% of pea2Δ shmoos exhibited GFP-Kar9p fluorescence as a single dot. Instead, 50% of spa2Δ and 55% of pea2Δ shmoos showed a new pattern in which GFP-Kar9p was found in a broad region over the shmoo tip. In addition, both mutants showed a significant number of cells with GFP-Kar9p in a line or in multiple “speckles” (Fig. 2 A). 
BUD6 is also required for the diploid-specific bipolar budding pattern and Bud6p (Aip3p) has been shown to interact with actin (Amberg et al., 1997). In the bud6Δ strain, only 41% of shmoos exhibited Kar9p fluorescence as a single dot (Fig. 2 A). Instead, a large percentage of the bud6Δ shmoos exhibited GFP-Kar9p fluorescence as a line (37%, compared with 8% in wild type, Fig. 2 A).
BNI1 encodes a formin homologue that has been implicated in the regulation of the actin cytoskeleton and cell polarization (Kohno et al., 1996; Evangelista et al., 1997), as well as bipolar budding (Zahner et al., 1996). Bni1-GFP also localizes to the shmoo tip (Evangelista et al., 1997). When GFP-Kar9p localization was examined in bni1Δ shmoos, only 12% exhibited Kar9p fluorescence as a cortical dot. Instead, 37% of the bni1Δ shmoos exhibited GFP-Kar9p in a line pattern and 41% in a speckled pattern throughout the shmoo (Fig 2 A).Because Kar9p also functions to orient cytoplasmic microtubules during vegetative growth (Miller and Rose, 1998), we sought to determine whether these four mutants also affected Kar9p localization during mitosis. In 85% of wild-type medium-to-large budded cells, GFP-Kar9p localized to a single dot at the tip of the bud. In the spa2Δ and pea2Δ cells, there was a modest reduction in the frequency of GFP-Kar9p localization to the bud tip, down to 75 and 74%, respectively (Fig. 2 B). Although modest, the reduction was observed in three independent experiments. Only 38% of the bud6Δ cells had a dot of GFP-Kar9p fluorescence at the tip of the bud. In 41% of the bud6Δ cells, GFP-Kar9p was mislocalized (Fig. 2 B) to the mother-bud neck and/or a region coincident with DAPI staining. Finally, in the bni1Δ cells, only 14% of cells had a dot of GFP-Kar9p at the tip of the bud. Instead, 70% of these cells had Kar9p fluorescence at the mother-bud neck and/ or a region coincident with DAPI staining (Fig. 2 B). Therefore, all four mutations affected Kar9p localization, although only in the bni1Δ strain was the magnitude similar to that of actin depolymerization.Because the deletion of BNI1 had such a strong effect on Kar9p localization, we next determined whether the two proteins are interdependent for localization. Accordingly, we examined the localization of Bni1p using a fully functional GFP-Bni1p fusion (a generous gift from Charlie Boone, Queens University, Kingston, Canada) in kar9Δ mutant shmoos and mitotic cells. In comparing kar9Δ to wild type, there were no obvious differences in the localization pattern of GFP-Bni1p to the presumptive bud site or the tip of the growing bud (data not shown). Therefore, Bni1p's association with the cortex is not dependent on Kar9p.
BNI1-related (BNR1) is another yeast gene encoding a formin homology domain protein. Bnr1p shares some functions with Bni1p (Imamura et al., 1997). We therefore determined whether BNR1 was also required for GFP-Kar9p localization. In both shmoos and mitotic cells, GFP-Kar9p localization in an isogenic bnr1Δ strain was indistinguishable from wild type (data not shown). Thus, localization of Kar9p to the cortex is a specific function of Bni1p.

Kar9p Localization Is Affected by the Polarization Mutants, spa2Δ, pea2Δ, bud6Δ, and bni1Δ

Although Kar9p's localization was actin dependent, it was found only as a single dot and therefore did not colocalize with other cortical actin patches at the shmoo tip and bud cortex. Therefore, we reasoned that additional cortical actin-associated proteins might restrict Kar9p localization to one site. In this case, mutations in a subset of specific actin-associated proteins would eliminate or alter Kar9p localization. To test this idea, we used pheromone-treated cells to assay GFP-Kar9p localization in mutants that affect actin, polarization, and/or mating. Most mutations had no effect on GFP-Kar9p localization to the tip of the shmoo projection. These included genes affecting actin function, ABP1 (Drubin et al., 1988, 1990) and SLA1 (Holtzman et al., 1993); genes involved in GTPase signaling, BOI2 (Bender et al., 1996), RSR1/BUD1 (Bender and Pringle, 1989), RGA1 (Stevenson et al., 1995), and MSB1 (Bender and Pringle, 1989); and genes affecting cell fusion, RVS161 (Brizzio et al., 1998), FUS1 (Trueheart and Fink, 1989), and FUS2 (Trueheart et al., 1987). Furthermore, GFP-Kar9p localization in mitotic cells was not affected by mutations in the genes encoding two microtubule-dependent motor proteins, KIP3 and DHC1 (Miller et al., 1998).

However, GFP-Kar9p localization in shmoos was affected by four mutations implicated in actin organization and polarization, spa2, pea2, bud6, and bni1. SPA2 and PEA2 are required for polarization in mating cells and mutations result in broader peanut-shaped shmoo projections (Valtz and Herskowitz, 1996). In mitotic cells, SPA2 and PEA2 are required for the diploid-specific bipolar budding pattern. Spa2p and Pea2p physically interact with each other and are interdependent for their own localization to the tips of shmoo projections (Valtz and Herskowitz, 1996). In the wild-type strain, 86% of the cells displayed GFP-Kar9p localization as a single dot at the tip of the shmoo projection. In contrast, only 15% of spa2Δ and 11% of pea2Δ shmoos exhibited GFP-Kar9p fluorescence as a single dot. Instead, 50% of spa2Δ and 55% of pea2Δ shmoos showed a new pattern in which GFP-Kar9p was found in a broad region over the shmoo tip. In addition, both mutants showed a significant number of cells with GFP-Kar9p in a line or in multiple “speckles” (Fig. 2 A).

BUD6 is also required for the diploid-specific bipolar budding pattern and Bud6p (Aip3p) has been shown to interact with actin (Amberg et al., 1997). In the bud6Δ strain, only 41% of shmoos exhibited Kar9p fluorescence as a single dot (Fig. 2 A). Instead, a large percentage of the bud6Δ shmoos exhibited GFP-Kar9p fluorescence as a line (37%, compared with 8% in wild type, Fig. 2 A).

BNI1 encodes a formin homologue that has been implicated in the regulation of the actin cytoskeleton and cell polarization (Kohno et al., 1996; Evangelista et al., 1997), as well as bipolar budding (Zahner et al., 1996). Bni1-GFP also localizes to the shmoo tip (Evangelista et al., 1997). When GFP-Kar9p localization was examined in bni1Δ shmoos, only 12% exhibited Kar9p fluorescence as a cortical dot. Instead, 37% of the bni1Δ shmoos exhibited GFP-Kar9p in a line pattern and 41% in a speckled pattern throughout the shmoo (Fig 2 A).

Because Kar9p also functions to orient cytoplasmic microtubules during vegetative growth (Miller and Rose, 1998), we sought to determine whether these four mutants also affected Kar9p localization during mitosis. In 85% of wild-type medium-to-large budded cells, GFP-Kar9p localized to a single dot at the tip of the bud. In the spa2Δ and pea2Δ cells, there was a modest reduction in the frequency of GFP-Kar9p localization to the bud tip, down to 75 and 74%, respectively (Fig. 2 B). Although modest, the reduction was observed in three independent experiments. Only 38% of the bud6Δ cells had a dot of GFP-Kar9p fluorescence at the tip of the bud. In 41% of the bud6Δ cells, GFP-Kar9p was mislocalized (Fig. 2 B) to the mother-bud neck and/or a region coincident with DAPI staining. Finally, in the bni1Δ cells, only 14% of cells had a dot of GFP-Kar9p at the tip of the bud. Instead, 70% of these cells had Kar9p fluorescence at the mother-bud neck and/ or a region coincident with DAPI staining (Fig. 2 B). Therefore, all four mutations affected Kar9p localization, although only in the bni1Δ strain was the magnitude similar to that of actin depolymerization.

Because the deletion of BNI1 had such a strong effect on Kar9p localization, we next determined whether the two proteins are interdependent for localization. Accordingly, we examined the localization of Bni1p using a fully functional GFP-Bni1p fusion (a generous gift from Charlie Boone, Queens University, Kingston, Canada) in kar9Δ mutant shmoos and mitotic cells. In comparing kar9Δ to wild type, there were no obvious differences in the localization pattern of GFP-Bni1p to the presumptive bud site or the tip of the growing bud (data not shown). Therefore, Bni1p's association with the cortex is not dependent on Kar9p.

BNI1-related (BNR1) is another yeast gene encoding a formin homology domain protein. Bnr1p shares some functions with Bni1p (Imamura et al., 1997). We therefore determined whether BNR1 was also required for GFP-Kar9p localization. In both shmoos and mitotic cells, GFP-Kar9p localization in an isogenic bnr1Δ strain was indistinguishable from wild type (data not shown). Thus, localization of Kar9p to the cortex is a specific function of Bni1p.

bni1Δ Mutants Exhibit Nuclear Migration DefectsSince spa2Δ, pea2Δ, bud6Δ, and bni1Δ mutants resulted in an altered localization for GFP-Kar9p, we wanted to determine whether these mutants also exhibited defects in nuclear migration like kar9Δ. Nuclear position was scored in asynchronous cultures using the DNA-specific fluorescent dye, DAPI (Table III). In the wild-type control, 20% of the cultures were large budded and, of these, 96% showed normal nuclear positions (i.e., located at the bud neck, or in anaphase across the bud neck, or in telophase with one nucleus each in the mother and bud). In contrast, 40% of kar9Δ mutants were large-budded (Table III), suggesting that kar9Δ has a cell cycle delay in mitosis. Consistent with our previous report (Miller and Rose, 1998), nuclear positioning was aberrant in up to 50% of the kar9Δ large-budded cells (i.e., nuclei not located at the bud neck, or anaphase or telophase occurring in the mother cell). The spa2Δ, pea2Δ, and bud6Δ mutants exhibited no obvious defects in nuclear position within the large-budded cells. However, like kar9Δ, the bni1Δ strain exhibited a significant defect in nuclear position; in 23% of the large-budded cells, anaphase was occurring within the mother cell. In contrast, only 2% of wild-type large-budded cells exhibited this phenotype (Table III). Unlike kar9Δ, only 5% of bni1Δ large-budded cells contained two nuclei within the mother cell (compared with 18% in kar9Δ). Similar results were obtained when the spa2Δ, pea2Δ, bud6Δ, and bni1Δ cultures were grown at 14°C (data not shown). Therefore, we conclude that bni1Δ is a nuclear migration mutant. However, the nuclear migration defect of bni1Δ is not as severe as that of kar9Δ. We next examined nuclear position in cells treated with mating pheromone. In wild type, the nuclei migrated to the base of the shmoo projection in 66% of the cells. As previously reported (Miller and Rose, 1998), the kar9Δ mutants exhibited a severe defect in nuclear migration to the shmoo projection, with 71% of the nuclei found in the center of the shmoo body (Table IV). The spa2Δ and pea2Δ mutants did not exhibit a nuclear migration defect, and may have localized better than the wild type, consistent with the continued localization of Kar9p at the shmoo tip. However, in 58% of the bud6Δ shmoos and 72% of the bni1Δ shmoos, the nuclei failed to migrate into the shmoo projection and were instead positioned in the center of the shmoo body (Table IV). Therefore, during mating, bud6Δ mutants exhibited a moderate defect and bni1Δ mutants exhibited a major defect in nuclear positioning.

bni1Δ Mutants Exhibit Nuclear Migration Defects

Since spa2Δ, pea2Δ, bud6Δ, and bni1Δ mutants resulted in an altered localization for GFP-Kar9p, we wanted to determine whether these mutants also exhibited defects in nuclear migration like kar9Δ. Nuclear position was scored in asynchronous cultures using the DNA-specific fluorescent dye, DAPI (Table III). In the wild-type control, 20% of the cultures were large budded and, of these, 96% showed normal nuclear positions (i.e., located at the bud neck, or in anaphase across the bud neck, or in telophase with one nucleus each in the mother and bud). In contrast, 40% of kar9Δ mutants were large-budded (Table III), suggesting that kar9Δ has a cell cycle delay in mitosis. Consistent with our previous report (Miller and Rose, 1998), nuclear positioning was aberrant in up to 50% of the kar9Δ large-budded cells (i.e., nuclei not located at the bud neck, or anaphase or telophase occurring in the mother cell). The spa2Δ, pea2Δ, and bud6Δ mutants exhibited no obvious defects in nuclear position within the large-budded cells. However, like kar9Δ, the bni1Δ strain exhibited a significant defect in nuclear position; in 23% of the large-budded cells, anaphase was occurring within the mother cell. In contrast, only 2% of wild-type large-budded cells exhibited this phenotype (Table III). Unlike kar9Δ, only 5% of bni1Δ large-budded cells contained two nuclei within the mother cell (compared with 18% in kar9Δ). Similar results were obtained when the spa2Δ, pea2Δ, bud6Δ, and bni1Δ cultures were grown at 14°C (data not shown). Therefore, we conclude that bni1Δ is a nuclear migration mutant. However, the nuclear migration defect of bni1Δ is not as severe as that of kar9Δ.

We next examined nuclear position in cells treated with mating pheromone. In wild type, the nuclei migrated to the base of the shmoo projection in 66% of the cells. As previously reported (Miller and Rose, 1998), the kar9Δ mutants exhibited a severe defect in nuclear migration to the shmoo projection, with 71% of the nuclei found in the center of the shmoo body (Table IV). The spa2Δ and pea2Δ mutants did not exhibit a nuclear migration defect, and may have localized better than the wild type, consistent with the continued localization of Kar9p at the shmoo tip. However, in 58% of the bud6Δ shmoos and 72% of the bni1Δ shmoos, the nuclei failed to migrate into the shmoo projection and were instead positioned in the center of the shmoo body (Table IV). Therefore, during mating, bud6Δ mutants exhibited a moderate defect and bni1Δ mutants exhibited a major defect in nuclear positioning.

Microtubule Morphologies in MutantsWe next examined the microtubule morphologies of the mutants. In >90% of wild-type shmoos, the cytoplasmic microtubules were present as a single bundle that extended from the nucleus to the tip of the shmoo projection (Fig. 3). As previously reported, only 15% of the kar9Δ shmoos contained cytoplasmic microtubules that extended to the shmoo tip. In most kar9Δ shmoos, the cytoplasmic microtubule bundle was misoriented and extended in random directions. In the spa2Δ mutant shmoos, only 52% showed a single bundle of microtubules oriented towards the shmoo tip. Instead, 29% of the spa2Δ mutant shmoos exhibited a “spray” of microtubules extending to the shmoo tip, and 10% showed supernumerary microtubules oriented away from the shmoo tip (“forked pattern,” Fig. 3). The pea2Δ mutant showed a similar pattern, albeit significantly less severe than that of spa2Δ. The presence of additional cytoplasmic microtubules making contact with the cortex is consistent with Kar9p localizing over a broader region of the shmoo tip. The bni1Δ strain also showed a significant defect in microtubule orientation in the shmoo. Like spa2Δ, only 50% of bni1Δ shmoos exhibited the normal microtubule pattern. However, unlike spa2Δ, 39% of the bni1Δ mutant shmoos exhibited misoriented microtubules similar to the pattern seen with the kar9Δ (Fig. 3). Thus, in mating cells, the defect in microtubule orientation in the bni1Δ mutants is consistent with the aberrant localization of Kar9p.We next examined the microtubule morphology in bni1Δ mitotic cells. For this purpose, we focused on those cells exhibiting a nuclear positioning defect such that anaphase was occurring within the mother cell. As reported previously (Miller and Rose, 1998), when the rare wild-type cells with anaphase in the mother were examined, the cytoplasmic microtubules nearly always extended into the bud (90%, Table V). In contrast, in the kar9Δ mutant, the cytoplasmic microtubule bundles infrequently extended into the bud (31%) and were more often found in the mother cell (48%). However, in 79% of the bni1Δ mutant, the cytoplasmic microtubules were correctly extended into the bud. This result suggests that either the bni1Δ mutation also suppresses the microtubule orientation defect created by mislocalized Kar9p or that enough Kar9p was correctly localized in the bni1Δ mutant to facilitate microtubule orientation. To differentiate between these two models, we created kar9Δ bni1Δ double mutants and examined them for defects in nuclear positioning and cytoplasmic microtubule orientation. In 49% of the kar9Δ bni1Δ double mutant cells, cytoplasmic microtubules were misoriented (Table V) like the kar9Δ mutant. Therefore, Kar9p was still responsible for the normal microtubule orientation found in the bni1Δ single mutant cells, in spite of the defect in Kar9p localization. These results suggest that Kar9p localization must be partially independent of Bni1p. This result would also be consistent with the weaker defect in nuclear migration seen for bni1Δ compared with kar9Δ.We next examined the double mutants for defects in nuclear positioning. Compared with the kar9Δ single mutant, the kar9Δ bni1Δ double mutant large-budded cells were slightly more severe for defects in nuclear positioning (47 vs. 33% aberrant, Table VI). In unbudded cells, however, a significantly more severe defect was observed. In unbudded cells, very low percentages of abnormal cells were found in wild-type, kar9Δ, and bni1Δ single mutants (0, 4, and 0.5%, respectively). However, in the kar9Δ bni1Δ double mutant, 22% of the cells were either multinucleate or anucleate. Therefore, the double mutant cells often appeared to progress through cytokinesis, which almost never occurred with the single mutants.

Microtubule Morphologies in Mutants

We next examined the microtubule morphologies of the mutants. In >90% of wild-type shmoos, the cytoplasmic microtubules were present as a single bundle that extended from the nucleus to the tip of the shmoo projection (Fig. 3). As previously reported, only 15% of the kar9Δ shmoos contained cytoplasmic microtubules that extended to the shmoo tip. In most kar9Δ shmoos, the cytoplasmic microtubule bundle was misoriented and extended in random directions. In the spa2Δ mutant shmoos, only 52% showed a single bundle of microtubules oriented towards the shmoo tip. Instead, 29% of the spa2Δ mutant shmoos exhibited a “spray” of microtubules extending to the shmoo tip, and 10% showed supernumerary microtubules oriented away from the shmoo tip (“forked pattern,” Fig. 3). The pea2Δ mutant showed a similar pattern, albeit significantly less severe than that of spa2Δ. The presence of additional cytoplasmic microtubules making contact with the cortex is consistent with Kar9p localizing over a broader region of the shmoo tip.

The bni1Δ strain also showed a significant defect in microtubule orientation in the shmoo. Like spa2Δ, only 50% of bni1Δ shmoos exhibited the normal microtubule pattern. However, unlike spa2Δ, 39% of the bni1Δ mutant shmoos exhibited misoriented microtubules similar to the pattern seen with the kar9Δ (Fig. 3). Thus, in mating cells, the defect in microtubule orientation in the bni1Δ mutants is consistent with the aberrant localization of Kar9p.

We next examined the microtubule morphology in bni1Δ mitotic cells. For this purpose, we focused on those cells exhibiting a nuclear positioning defect such that anaphase was occurring within the mother cell. As reported previously (Miller and Rose, 1998), when the rare wild-type cells with anaphase in the mother were examined, the cytoplasmic microtubules nearly always extended into the bud (90%, Table V). In contrast, in the kar9Δ mutant, the cytoplasmic microtubule bundles infrequently extended into the bud (31%) and were more often found in the mother cell (48%). However, in 79% of the bni1Δ mutant, the cytoplasmic microtubules were correctly extended into the bud.

This result suggests that either the bni1Δ mutation also suppresses the microtubule orientation defect created by mislocalized Kar9p or that enough Kar9p was correctly localized in the bni1Δ mutant to facilitate microtubule orientation. To differentiate between these two models, we created kar9Δ bni1Δ double mutants and examined them for defects in nuclear positioning and cytoplasmic microtubule orientation. In 49% of the kar9Δ bni1Δ double mutant cells, cytoplasmic microtubules were misoriented (Table V) like the kar9Δ mutant. Therefore, Kar9p was still responsible for the normal microtubule orientation found in the bni1Δ single mutant cells, in spite of the defect in Kar9p localization. These results suggest that Kar9p localization must be partially independent of Bni1p. This result would also be consistent with the weaker defect in nuclear migration seen for bni1Δ compared with kar9Δ.

We next examined the double mutants for defects in nuclear positioning. Compared with the kar9Δ single mutant, the kar9Δ bni1Δ double mutant large-budded cells were slightly more severe for defects in nuclear positioning (47 vs. 33% aberrant, Table VI). In unbudded cells, however, a significantly more severe defect was observed. In unbudded cells, very low percentages of abnormal cells were found in wild-type, kar9Δ, and bni1Δ single mutants (0, 4, and 0.5%, respectively). However, in the kar9Δ bni1Δ double mutant, 22% of the cells were either multinucleate or anucleate. Therefore, the double mutant cells often appeared to progress through cytokinesis, which almost never occurred with the single mutants.

Genetic Analysis Places Bni1p in the Kar9p Pathway for Nuclear MigrationBecause of the synthetic lethality between kar9Δ and dhc1Δ, Kar9p and dynein are thought to function in separate, partially redundant pathways for nuclear migration (Miller and Rose, 1998). Because spa2Δ, pea2Δ, bud6Δ, and bni1Δ mutants each altered GFP-Kar9p localization, we wanted to know whether any of these mutations would show genetic interactions similar to those observed with kar9Δ. Isogenic marked deletions of each gene were created and crossed to marked kar9 and dhc1 deletions. The viability of the double mutants was assayed for growth 2–3 d after spore germination. The results of the crosses are summarized in Table VII. When spa2Δ, pea2Δ, bud6Δ, or bni1Δ were crossed with kar9Δ, all four double-mutant combinations were viable (Table VII, 1–4). Similarly, the spa2Δ dhc1Δ, and pea2Δ dhc1Δ double mutants were viable (Table VII, 6–8). The bud6Δ dhc1Δ double mutants were viable, but exibited a mild growth defect (Table VII, 8). However, when bni1Δ was crossed to dhc1Δ, all the predicted double mutants were barely viable and formed microcolonies (Table VII, 9). Similar microcolonies were also observed in synthetic lethal genetic interactions seen with kar9Δ, including dhc1Δ, bik1Δ, and kip2Δ (Miller et al., 1998; Miller and Rose, 1998). Therefore, we consider this to be a synthetic lethal interaction and conclude that Bni1p acts in the Kar9p pathway for nuclear migration. As a control, crosses were performed using a deletion of the related formin homology domain protein, Bnr1p. The bnr1Δ mutation was not synthetically lethal in combination with kar9Δ or dhc1Δ (Table VII, 5 and 10). Therefore, the interaction seen between dhc1Δ and bni1Δ is specific to Bni1p and not a general feature of formin homology proteins.In previous work, Kip3p was shown to function in the KAR9 pathway for nuclear migration (Miller et al., 1998). Because Bni1p also appears to function in the KAR9 pathway, we crossed the bni1Δ to the kip3Δ to determine whether Bni1p and Kip3p might also function in a common pathway. We found that the bni1Δ kip3Δ double mutant is viable, as was also reported by Lee et al. (1999). This suggests that Kar9p, Kip3p, and Bni1p function in a common pathway. Accordingly, the kar9Δ bni1Δ kip3Δ triple mutant would also be predicted to be viable. We found that the kar9Δ bni1Δ kip3Δ triple mutant was indeed viable (Table VII, 12), supporting the hypothesis that Kar9p, Kip3p, and Bni1p function together in a common pathway.

Genetic Analysis Places Bni1p in the Kar9p Pathway for Nuclear Migration

Because of the synthetic lethality between kar9Δ and dhc1Δ, Kar9p and dynein are thought to function in separate, partially redundant pathways for nuclear migration (Miller and Rose, 1998). Because spa2Δ, pea2Δ, bud6Δ, and bni1Δ mutants each altered GFP-Kar9p localization, we wanted to know whether any of these mutations would show genetic interactions similar to those observed with kar9Δ. Isogenic marked deletions of each gene were created and crossed to marked kar9 and dhc1 deletions. The viability of the double mutants was assayed for growth 2–3 d after spore germination. The results of the crosses are summarized in Table VII. When spa2Δ, pea2Δ, bud6Δ, or bni1Δ were crossed with kar9Δ, all four double-mutant combinations were viable (Table VII, 1–4). Similarly, the spa2Δ dhc1Δ, and pea2Δ dhc1Δ double mutants were viable (Table VII, 6–8). The bud6Δ dhc1Δ double mutants were viable, but exibited a mild growth defect (Table VII, 8). However, when bni1Δ was crossed to dhc1Δ, all the predicted double mutants were barely viable and formed microcolonies (Table VII, 9). Similar microcolonies were also observed in synthetic lethal genetic interactions seen with kar9Δ, including dhc1Δ, bik1Δ, and kip2Δ (Miller et al., 1998; Miller and Rose, 1998). Therefore, we consider this to be a synthetic lethal interaction and conclude that Bni1p acts in the Kar9p pathway for nuclear migration.

As a control, crosses were performed using a deletion of the related formin homology domain protein, Bnr1p. The bnr1Δ mutation was not synthetically lethal in combination with kar9Δ or dhc1Δ (Table VII, 5 and 10). Therefore, the interaction seen between dhc1Δ and bni1Δ is specific to Bni1p and not a general feature of formin homology proteins.

In previous work, Kip3p was shown to function in the KAR9 pathway for nuclear migration (Miller et al., 1998). Because Bni1p also appears to function in the KAR9 pathway, we crossed the bni1Δ to the kip3Δ to determine whether Bni1p and Kip3p might also function in a common pathway. We found that the bni1Δ kip3Δ double mutant is viable, as was also reported by Lee et al. (1999). This suggests that Kar9p, Kip3p, and Bni1p function in a common pathway. Accordingly, the kar9Δ bni1Δ kip3Δ triple mutant would also be predicted to be viable. We found that the kar9Δ bni1Δ kip3Δ triple mutant was indeed viable (Table VII, 12), supporting the hypothesis that Kar9p, Kip3p, and Bni1p function together in a common pathway.

kar9Δ Does Not Exhibit Bipolar Budding DefectsThe four mutations that affect Kar9p localization, spa2Δ, pea2Δ, bud6Δ, and bni1Δ, share an additional common phenotype of being defective in the diploid-specific bipolar budding pattern (Snyder, 1989; Valtz and Herskowitz, 1996; Zahner et al., 1996; Amberg et al., 1997). This provided an additional way of addressing whether Kar9p is simply a Bni1p-associated protein or functions specifically for microtubule orientation. Therefore, we examined whether kar9Δ mutants are also defective for bipolar budding. Diploid cells with three or more bud scars were scored to determine whether they showed the bipolar or random budding pattern (see Materials and Methods). In the wild-type control strain, 95% of the cells exhibited a bipolar budding pattern and 5% showed a random pattern (n = 263). In the pea2Δ strain, only 28% of the cells exhibited the bipolar budding pattern, while 72% budded randomly (n = 201). Unlike the pea2Δ mutant, the kar9Δ mutant exhibited the wild-type budding pattern (90% bipolar, n = 300). The kar9Δ mutants also did not display a defect in the haploid-specific axial budding pattern (data not shown). Taken together with the observation that kar9Δ mutant shmoos do not exhibit polarization defects, and that Bni1p localizes normally in the kar9Δ mutants, these results argue strongly that Kar9p's function with Bni1p is specific to the nuclear migration pathway.

kar9Δ Does Not Exhibit Bipolar Budding Defects

The four mutations that affect Kar9p localization, spa2Δ, pea2Δ, bud6Δ, and bni1Δ, share an additional common phenotype of being defective in the diploid-specific bipolar budding pattern (Snyder, 1989; Valtz and Herskowitz, 1996; Zahner et al., 1996; Amberg et al., 1997). This provided an additional way of addressing whether Kar9p is simply a Bni1p-associated protein or functions specifically for microtubule orientation. Therefore, we examined whether kar9Δ mutants are also defective for bipolar budding. Diploid cells with three or more bud scars were scored to determine whether they showed the bipolar or random budding pattern (see Materials and Methods). In the wild-type control strain, 95% of the cells exhibited a bipolar budding pattern and 5% showed a random pattern (n = 263). In the pea2Δ strain, only 28% of the cells exhibited the bipolar budding pattern, while 72% budded randomly (n = 201). Unlike the pea2Δ mutant, the kar9Δ mutant exhibited the wild-type budding pattern (90% bipolar, n = 300). The kar9Δ mutants also did not display a defect in the haploid-specific axial budding pattern (data not shown). Taken together with the observation that kar9Δ mutant shmoos do not exhibit polarization defects, and that Bni1p localizes normally in the kar9Δ mutants, these results argue strongly that Kar9p's function with Bni1p is specific to the nuclear migration pathway.

DiscussionIn yeast, both actin and the cytoplasmic microtubules have been implicated in normal spindle orientation and movement, suggesting a possible interaction between these two cytoskeletal networks (Palmer et al., 1992; Read et al., 1992; Sullivan and Huffaker, 1992). However, the mechanism by which the actin cytoskeleton might contribute to spindle orientation has not been clear. Studies on centrosomal rotation in C. elegans (Hyman, 1989) and nuclear movement in yeast (Carminati and Stearns, 1997) have also shown the involvement of cortical factors in microtubule positioning and orientation. However, to date, only Kar9p has been identified as a likely candidate for one of the cortical proteins required for cytoplasmic microtubule orientation. In this study, we demonstrated that Kar9p localization is dependent upon the actin cytoskeleton. We have also shown that Kar9p localization is strongly influenced by Bni1p, a formin involved in actin cytoskeletal functions, polarization, budding, and cell fusion processes. Like Kar9p, Bni1p was found to participate in nuclear migration processes during both shmooing and mitosis. Genetic analysis suggests that Bni1p functions in the Kar9p pathway rather than the dynein pathway for nuclear migration. Taken together, these data suggest that Kar9p and Bni1p functionally link the actin and microtubule networks to promote proper spindle orientation and nuclear migration.Three other proteins were found to influence Kar9p localization in mating cells, Bud6p, Spa2p, and Pea2p. Spa2p and Pea2p are required for polarization in mating cells and in their absence the shmoo tip is broader. In these two mutants Kar9p localization was spread over a large area of the shmoo tip. Thus, it seems likely that Pea2p and Spa2p and other proteins help to cluster Kar9p into a single focus at the shmoo tip.Taken together, these observations suggest a model for nuclear orientation in mitotic and mating cells. As the nascent bud emerges, actin patches become associated with the bud cortex. Similarly, actin patches associate with the cortex at the tip of the mating projection. One or a subset of actin patches becomes integrated into a specialized site that we will refer to as the “polarization complex,” normally at or near the tip of the bud and the mating projection. Additional actin-associated proteins playing roles in polarized growth would be recruited to the specialized polarization complex. Among these would be Bni1p, Spa2p, and Pea2p. Each of these proteins would interact with a subset of other components in the complex and each may have cortical associations that are independent of actin. Some of the proteins, perhaps including Bni1p, may play key roles in cross-linking multiple proteins and thereby play a major “scaffolding” role. Loss of such proteins would affect the localization of several proteins to the complex. The specific proteins required might also differ between mating and mitosis. Finally, other proteins, such as Kar9p, would be recruited to more peripheral locations on the polarization complex. Such peripheral proteins may serve as “docking sites” or “adapters” for extrinsic elements such as the microtubules that need to identify and interact with the cortical polarization complex. Thus, the assembly and maintenance of the actin-associated polarization complex provides a unique molecular “address” within the cell that can be “read” by different cellular systems through specific adapter molecules.BNI1 Is Required for Nuclear MigrationWe have identified Bni1p as being required for nuclear migration in mating and mitosis because of its role in Kar9p localization. In pheromone-treated cells, the bni1Δ mutant exhibits a strong defect in microtubule orientation, consistent with the defect in Kar9p localization.The role of Bni1p in mitotic cells is more complex. Although Kar9p was mislocalized away from the bud tip in mitotic bni1Δ mutant cells, the overall effect was less dramatic. In the mitotic cells, Kar9p localization was shifted to secondary sites at the SPB and/or bud site and the cytoplasmic microtubules were not grossly misoriented by the onset of anaphase. Several explanations for this are possible. First, some 14% of the bni1Δ cells showed detectable Kar9p at the tip of the bud. It is possible that different levels of Kar9p at the bud tip are required for microtubule orientation and subsequent nuclear migration. Perhaps there is enough Kar9p at the bud tip to allow correct microtubule alignment, but not enough Kar9p to activate some other component necessary for nuclear migration, such as Kip3p. Second, although Kar9p was not localized to the bud tip in the mutant, it was localized near the bud neck. Microtubules contacting Kar9p at the bud neck would be properly oriented to eventually contact the bud cortex after continued assembly. This would be consistent with the analysis of the double mutant that indicated that in the bni1Δ mutant the orientation of the cytoplasmic microtubules was still dependent on Kar9p.The residual localization of GFP-Kar9p in the bni1Δ, the dependence of microtubule orientation on Kar9p in the bni1Δ, and the fact that the kar9Δ mutant exhibits a more severe nuclear migration defect than the bni1Δ mutant suggests that Kar9p is at least partially active in the absence of Bni1p. Taken together, the simplest explanation for these observations is that Kar9p has substantial residual localization to the cortex, independent of Bni1p. In the absence of Bni1p, the level of GFP-Kar9p at the cortex may simply fall below the limit of detection in most cells. Nevertheless, the reduction in Kar9p localization would be sufficient to compromise the Kar9p pathway for nuclear migration. One prediction of these results is that Kar9p localization would be partially dependent on additional cortical proteins.Independently, Lee et al. (1999) have also identified Bni1p as being required for nuclear migration. In their work, dynamic studies of nuclear migration and microtubule dynamics clearly show a defect in cytoplasmic microtubule function. In addition, they also find that Kip3 and Bni1p are likely to be part of the same nuclear migration pathway.In contrast to bni1Δ, there was no obvious defect in nuclear migration in the spa2Δ, pea2Δ, and bud6Δ cells. This is not surprising given their milder defects in Kar9p localization. Our results showing that bud6Δ does not exhibit a significant nuclear migration defect are in contrast to results previously reported (Amberg et al., 1997). This may be due to differences in strain background or partially toxic effects of the disruption mutant used in that study. In support of this, we find that the bud6Δ::URA3 strain (DBY7055) did exhibit a synthetic phenotype and temperature sensitivity in combination with dhc1Δ.Bni1p Has Functions Distinct from Kar9pAlthough kar9 and bni1 mutants exhibit several similarities and genetically lie in the same pathway for nuclear migration, several lines of evidence suggest that they also have distinct functions. First, unlike bni1Δ mutants (Evangelista et al., 1997), kar9Δ mutants do not exhibit defects in actin localization (Miller and Rose, 1998). Second, kar9Δ mutants do not exhibit defects in the bipolar budding pattern. Third, unlike bni1Δ mutants (Evangelista et al., 1997), kar9Δ did not form round-shaped shmoos or show other obvious defects in polarization. Fourth, unlike bni1Δ mutants, kar9Δ mutants do not exhibit a severe mating defect (Kurihara et al., 1994). Fifth, although Bni1p is required for efficient Kar9p localization, Kar9p is not required for GFP-Bni1p localization in mitotically dividing cells or shmoos. Thus, Bni1p mediates a series of functions in mating and mitosis that are not part of Kar9p's repertoire of functions.If Bni1p's only function in nuclear migration were to localize Kar9p, then the bni1Δ kar9ΔΔ double mutant should not be more severe than the kar9Δ single mutant. However, the double mutant has a more severe phenotype than either single mutant alone, as evidenced both by a slight increase in the number of aberrant large-budded cells and by an increase in the percentage of multinucleate and anucleate unbudded cells (Table VI). Thus, Bni1p may have a secondary role in nuclear migration in addition to the localization of Kar9p. However, if Bni1p does have a secondary role independent of Kar9p, its contribution to nuclear migration must be relatively minor. Otherwise, the additive effects of the two independent defects should lead to a severe growth defect for the double mutant. Instead, we find that the double mutant grows no worse than the single mutants, identical to the wild type.That anucleate cells are observed in the bni1Δ kar9Δ double mutant combination raises the additional possibility that Kar9p and/or Bni1p may play an inhibitory role in cytokinesis. This might be true if one or the other is part of the checkpoint mechanism that coordinates nuclear migration and cytokinesis. In addition to bud tip localization, both Bni1p and Kar9p are also localized at the mother-bud neck in a minor fraction of large budded cells (Evangelista and Boone, personal communication; Fujiwara et al., 1998; Miller and Rose, 1998). More likely, the increased severity of the double mutant may cause some cells to eventually complete cytokinesis without proper nuclear migration.Kar9p Localization by Bni1p Is Highly SpecificConsistent with the observation that they all contribute to polarization, Spa2p, Pea2p, and Bud6p have been isolated as a stable complex (Sheu et al., 1998). Furthermore, Spa2p interacts with Bni1p by two-hybrid analysis (Fujiwara et al., 1998). Yet mutations in these proteins affected Kar9p localization differently, with only mild mitotic defects associated with spa2Δ and pea2Δ, and none apparent for bud6Δ. In shmoos and mitotic cells, the most severe effect on Kar9p localization occurred in the bni1Δ mutants. Nevertheless, all four of these mutants affect polarization in a similar manner and to a similar degree. Thus, the effects of bni1Δ on Kar9p localization are not likely to be due to general defects in polarization.All four mutants, spa2Δ, pea2Δ, bud6Δ, and bni1Δ, also affect cell fusion and bipolar budding. However, another bipolar budding mutant, rvs161Δ, exhibited no defects in Kar9p localization. In addition, Kar9p localization was not altered in the cell fusion mutants fus1Δ and fus2Δ. Thus, alterations in Kar9p localization are not due to general disruptions in either bipolar budding or the cell fusion process.Several other observations speak to the specificity of the effects of these mutations on Kar9p localization. Bnr1p is another formin family protein that shares overlapping functions with Bni1p (Imamura et al., 1997). Kar9p localized normally in bnr1Δ mutants (data not shown) and also did not show any genetic interactions with kar9Δ or dhc1Δ, indicating that the effects on Kar9p localization are not general to all formin proteins, but specific to Bni1p.Other mutants such as sla1Δ that affect the assembly of the actin cytoskeleton (Holtzman et al., 1993) also did not alter Kar9p localization in shmoos. This argues that the specialized interaction of Spa2p, Pea2p, Bud6p, and Bni1p with the actin cytoskeleton is the important feature for Kar9p localization.Differences between Mitotic and Mating CellsThe localization patterns for Spa2p and Pea2p are interdependent, suggesting that their functions are closely interrelated (Valtz and Herskowitz, 1996). Consistent with this, we find that GFP-Kar9p mislocalization was very similar in the spa2Δ and pea2Δ mutants. Thus, with respect to Kar9p localization, these two mutants can be referred to as the “spa2 class.”Interestingly, the spa2 class showed a dramatic difference in the severity of Kar9p mislocalization comparing shmoos and mitotically dividing cells. In mitotic cells, mislocalization was apparent in only ∼10% of cells. In contrast, in most shmoos, Kar9p localization was altered from a single dot to a cap-like pattern. Thus, Spa2p and Pea2p likely play very different roles in polarization and microtubule orientation in mating cells.The finding that Kar9p localization is often spread out over a broader area in spa2Δ or pea2Δ shmoos is consistent with the proposed role for Spa2p and Pea2p in restricting the zone of polarization in shmoos (Valtz and Herskowitz, 1996). One prediction from such a model would be that a narrowed site of polarization would restrict the area of microtubule interaction with the cortex by restricting Kar9p localization to a dot. Consistent with this, we found a “splayed-out” microtubule pattern in a subset of spa2Δ mutants.Two models might explain the altered patterns of Kar9p localization. In one model, Spa2p and Pea2p may play a direct role in limiting Kar9p localization to a tight focus by helping aggregate the Bni1p/Kar9p complex. Alternatively, Spa2p and Pea2p might play an indirect role in Kar9p localization via their effects on shmoo morphology. In either case, correct polarization might be required to define a specialized region of the mating projection that is primed for cell and nuclear fusion. Such a primed region would facilitate the alignment of cytoplasmic microtubules, allowing nuclear congression to follow rapidly after cell fusion. In spa2Δ and pea2Δ mutant shmoos, the primed region would be more diffuse over the blunter mutant shmoo projection. In support of the indirect model, in spa2Δ prezygotes the clustering of vesicles along the zone of cell fusion is disrupted (Gammie et al., 1998). Regardless of the specific mechanism, these results highlight the differences in cell polarization between mating and mitotic cells.In contrast, both the bni1Δ mutation and actin depolymerization resulted in strong defects in Kar9p localization in both shmoos and mitotic cells. Although the bni1Δ mutant was defective for nuclear migration in both shmoos and mitotic cells, there were differences in the effects on microtubule orientation. In bni1Δ shmoos, the microtubules were clearly misoriented, whereas mitotic cells did not exhibit an obvious defect. One interpretation might be that correct Kar9p localization is more important for microtubule orientation in mating cells than in mitotic cells. Alternatively, the difference between the two cells may arise from the different patterns of mislocalized Kar9p. In the bni1Δ shmoos, GFP-Kar9p was present in a diffuse speckled pattern. In the bni1Δ mutant, GFP-Kar9p mislocalized to secondary sites at the SPB and bud neck, where it might still be able to provide some orientation to the microtubules.Coordinating Cytoskeletal SystemsAbundant evidence has shown that Bni1p, specifically, and formins, generally, are involved in organizing the actin cytoskeleton. In addition to Spa2p and Bud6p, Bni1p interacts with profilin and the small G-protein Rho1p (Kohno et al., 1996; Evangelista et al., 1997). It seems likely that formins may play a general role in linking the actin and microtubule cytoskeletons. For example, mutations in the Drosophila formin, cappuccino result in mislocalized molecular determinants and abnormal microtubule patterns (Theurkauf, 1994; Emmons et al., 1995).In conclusion, we report here that Kar9p localization is dependent upon actin and the polarization protein, Bni1p. This provides the first evidence in yeast for a functional link between the actin and microtubule cytoskeletons. These data suggest a mechanism by which the polarized information contained within Kar9p is transferred into the vectorial movement of spindle orientation and nuclear movement into the bud. This work provides an avenue into future investigations as to how nuclear positioning is coordinated with the growth of the bud and cytokinesis.

In yeast, both actin and the cytoplasmic microtubules have been implicated in normal spindle orientation and movement, suggesting a possible interaction between these two cytoskeletal networks (Palmer et al., 1992; Read et al., 1992; Sullivan and Huffaker, 1992). However, the mechanism by which the actin cytoskeleton might contribute to spindle orientation has not been clear. Studies on centrosomal rotation in C. elegans (Hyman, 1989) and nuclear movement in yeast (Carminati and Stearns, 1997) have also shown the involvement of cortical factors in microtubule positioning and orientation. However, to date, only Kar9p has been identified as a likely candidate for one of the cortical proteins required for cytoplasmic microtubule orientation. In this study, we demonstrated that Kar9p localization is dependent upon the actin cytoskeleton. We have also shown that Kar9p localization is strongly influenced by Bni1p, a formin involved in actin cytoskeletal functions, polarization, budding, and cell fusion processes. Like Kar9p, Bni1p was found to participate in nuclear migration processes during both shmooing and mitosis. Genetic analysis suggests that Bni1p functions in the Kar9p pathway rather than the dynein pathway for nuclear migration. Taken together, these data suggest that Kar9p and Bni1p functionally link the actin and microtubule networks to promote proper spindle orientation and nuclear migration.

Three other proteins were found to influence Kar9p localization in mating cells, Bud6p, Spa2p, and Pea2p. Spa2p and Pea2p are required for polarization in mating cells and in their absence the shmoo tip is broader. In these two mutants Kar9p localization was spread over a large area of the shmoo tip. Thus, it seems likely that Pea2p and Spa2p and other proteins help to cluster Kar9p into a single focus at the shmoo tip.

Taken together, these observations suggest a model for nuclear orientation in mitotic and mating cells. As the nascent bud emerges, actin patches become associated with the bud cortex. Similarly, actin patches associate with the cortex at the tip of the mating projection. One or a subset of actin patches becomes integrated into a specialized site that we will refer to as the “polarization complex,” normally at or near the tip of the bud and the mating projection. Additional actin-associated proteins playing roles in polarized growth would be recruited to the specialized polarization complex. Among these would be Bni1p, Spa2p, and Pea2p. Each of these proteins would interact with a subset of other components in the complex and each may have cortical associations that are independent of actin. Some of the proteins, perhaps including Bni1p, may play key roles in cross-linking multiple proteins and thereby play a major “scaffolding” role. Loss of such proteins would affect the localization of several proteins to the complex. The specific proteins required might also differ between mating and mitosis. Finally, other proteins, such as Kar9p, would be recruited to more peripheral locations on the polarization complex. Such peripheral proteins may serve as “docking sites” or “adapters” for extrinsic elements such as the microtubules that need to identify and interact with the cortical polarization complex. Thus, the assembly and maintenance of the actin-associated polarization complex provides a unique molecular “address” within the cell that can be “read” by different cellular systems through specific adapter molecules.

BNI1 Is Required for Nuclear MigrationWe have identified Bni1p as being required for nuclear migration in mating and mitosis because of its role in Kar9p localization. In pheromone-treated cells, the bni1Δ mutant exhibits a strong defect in microtubule orientation, consistent with the defect in Kar9p localization.The role of Bni1p in mitotic cells is more complex. Although Kar9p was mislocalized away from the bud tip in mitotic bni1Δ mutant cells, the overall effect was less dramatic. In the mitotic cells, Kar9p localization was shifted to secondary sites at the SPB and/or bud site and the cytoplasmic microtubules were not grossly misoriented by the onset of anaphase. Several explanations for this are possible. First, some 14% of the bni1Δ cells showed detectable Kar9p at the tip of the bud. It is possible that different levels of Kar9p at the bud tip are required for microtubule orientation and subsequent nuclear migration. Perhaps there is enough Kar9p at the bud tip to allow correct microtubule alignment, but not enough Kar9p to activate some other component necessary for nuclear migration, such as Kip3p. Second, although Kar9p was not localized to the bud tip in the mutant, it was localized near the bud neck. Microtubules contacting Kar9p at the bud neck would be properly oriented to eventually contact the bud cortex after continued assembly. This would be consistent with the analysis of the double mutant that indicated that in the bni1Δ mutant the orientation of the cytoplasmic microtubules was still dependent on Kar9p.The residual localization of GFP-Kar9p in the bni1Δ, the dependence of microtubule orientation on Kar9p in the bni1Δ, and the fact that the kar9Δ mutant exhibits a more severe nuclear migration defect than the bni1Δ mutant suggests that Kar9p is at least partially active in the absence of Bni1p. Taken together, the simplest explanation for these observations is that Kar9p has substantial residual localization to the cortex, independent of Bni1p. In the absence of Bni1p, the level of GFP-Kar9p at the cortex may simply fall below the limit of detection in most cells. Nevertheless, the reduction in Kar9p localization would be sufficient to compromise the Kar9p pathway for nuclear migration. One prediction of these results is that Kar9p localization would be partially dependent on additional cortical proteins.Independently, Lee et al. (1999) have also identified Bni1p as being required for nuclear migration. In their work, dynamic studies of nuclear migration and microtubule dynamics clearly show a defect in cytoplasmic microtubule function. In addition, they also find that Kip3 and Bni1p are likely to be part of the same nuclear migration pathway.In contrast to bni1Δ, there was no obvious defect in nuclear migration in the spa2Δ, pea2Δ, and bud6Δ cells. This is not surprising given their milder defects in Kar9p localization. Our results showing that bud6Δ does not exhibit a significant nuclear migration defect are in contrast to results previously reported (Amberg et al., 1997). This may be due to differences in strain background or partially toxic effects of the disruption mutant used in that study. In support of this, we find that the bud6Δ::URA3 strain (DBY7055) did exhibit a synthetic phenotype and temperature sensitivity in combination with dhc1Δ.

BNI1 Is Required for Nuclear Migration

We have identified Bni1p as being required for nuclear migration in mating and mitosis because of its role in Kar9p localization. In pheromone-treated cells, the bni1Δ mutant exhibits a strong defect in microtubule orientation, consistent with the defect in Kar9p localization.

The role of Bni1p in mitotic cells is more complex. Although Kar9p was mislocalized away from the bud tip in mitotic bni1Δ mutant cells, the overall effect was less dramatic. In the mitotic cells, Kar9p localization was shifted to secondary sites at the SPB and/or bud site and the cytoplasmic microtubules were not grossly misoriented by the onset of anaphase. Several explanations for this are possible. First, some 14% of the bni1Δ cells showed detectable Kar9p at the tip of the bud. It is possible that different levels of Kar9p at the bud tip are required for microtubule orientation and subsequent nuclear migration. Perhaps there is enough Kar9p at the bud tip to allow correct microtubule alignment, but not enough Kar9p to activate some other component necessary for nuclear migration, such as Kip3p. Second, although Kar9p was not localized to the bud tip in the mutant, it was localized near the bud neck. Microtubules contacting Kar9p at the bud neck would be properly oriented to eventually contact the bud cortex after continued assembly. This would be consistent with the analysis of the double mutant that indicated that in the bni1Δ mutant the orientation of the cytoplasmic microtubules was still dependent on Kar9p.

The residual localization of GFP-Kar9p in the bni1Δ, the dependence of microtubule orientation on Kar9p in the bni1Δ, and the fact that the kar9Δ mutant exhibits a more severe nuclear migration defect than the bni1Δ mutant suggests that Kar9p is at least partially active in the absence of Bni1p. Taken together, the simplest explanation for these observations is that Kar9p has substantial residual localization to the cortex, independent of Bni1p. In the absence of Bni1p, the level of GFP-Kar9p at the cortex may simply fall below the limit of detection in most cells. Nevertheless, the reduction in Kar9p localization would be sufficient to compromise the Kar9p pathway for nuclear migration. One prediction of these results is that Kar9p localization would be partially dependent on additional cortical proteins.

Independently, Lee et al. (1999) have also identified Bni1p as being required for nuclear migration. In their work, dynamic studies of nuclear migration and microtubule dynamics clearly show a defect in cytoplasmic microtubule function. In addition, they also find that Kip3 and Bni1p are likely to be part of the same nuclear migration pathway.

In contrast to bni1Δ, there was no obvious defect in nuclear migration in the spa2Δ, pea2Δ, and bud6Δ cells. This is not surprising given their milder defects in Kar9p localization. Our results showing that bud6Δ does not exhibit a significant nuclear migration defect are in contrast to results previously reported (Amberg et al., 1997). This may be due to differences in strain background or partially toxic effects of the disruption mutant used in that study. In support of this, we find that the bud6Δ::URA3 strain (DBY7055) did exhibit a synthetic phenotype and temperature sensitivity in combination with dhc1Δ.

Bni1p Has Functions Distinct from Kar9pAlthough kar9 and bni1 mutants exhibit several similarities and genetically lie in the same pathway for nuclear migration, several lines of evidence suggest that they also have distinct functions. First, unlike bni1Δ mutants (Evangelista et al., 1997), kar9Δ mutants do not exhibit defects in actin localization (Miller and Rose, 1998). Second, kar9Δ mutants do not exhibit defects in the bipolar budding pattern. Third, unlike bni1Δ mutants (Evangelista et al., 1997), kar9Δ did not form round-shaped shmoos or show other obvious defects in polarization. Fourth, unlike bni1Δ mutants, kar9Δ mutants do not exhibit a severe mating defect (Kurihara et al., 1994). Fifth, although Bni1p is required for efficient Kar9p localization, Kar9p is not required for GFP-Bni1p localization in mitotically dividing cells or shmoos. Thus, Bni1p mediates a series of functions in mating and mitosis that are not part of Kar9p's repertoire of functions.If Bni1p's only function in nuclear migration were to localize Kar9p, then the bni1Δ kar9ΔΔ double mutant should not be more severe than the kar9Δ single mutant. However, the double mutant has a more severe phenotype than either single mutant alone, as evidenced both by a slight increase in the number of aberrant large-budded cells and by an increase in the percentage of multinucleate and anucleate unbudded cells (Table VI). Thus, Bni1p may have a secondary role in nuclear migration in addition to the localization of Kar9p. However, if Bni1p does have a secondary role independent of Kar9p, its contribution to nuclear migration must be relatively minor. Otherwise, the additive effects of the two independent defects should lead to a severe growth defect for the double mutant. Instead, we find that the double mutant grows no worse than the single mutants, identical to the wild type.That anucleate cells are observed in the bni1Δ kar9Δ double mutant combination raises the additional possibility that Kar9p and/or Bni1p may play an inhibitory role in cytokinesis. This might be true if one or the other is part of the checkpoint mechanism that coordinates nuclear migration and cytokinesis. In addition to bud tip localization, both Bni1p and Kar9p are also localized at the mother-bud neck in a minor fraction of large budded cells (Evangelista and Boone, personal communication; Fujiwara et al., 1998; Miller and Rose, 1998). More likely, the increased severity of the double mutant may cause some cells to eventually complete cytokinesis without proper nuclear migration.

Bni1p Has Functions Distinct from Kar9p

Although kar9 and bni1 mutants exhibit several similarities and genetically lie in the same pathway for nuclear migration, several lines of evidence suggest that they also have distinct functions. First, unlike bni1Δ mutants (Evangelista et al., 1997), kar9Δ mutants do not exhibit defects in actin localization (Miller and Rose, 1998). Second, kar9Δ mutants do not exhibit defects in the bipolar budding pattern. Third, unlike bni1Δ mutants (Evangelista et al., 1997), kar9Δ did not form round-shaped shmoos or show other obvious defects in polarization. Fourth, unlike bni1Δ mutants, kar9Δ mutants do not exhibit a severe mating defect (Kurihara et al., 1994). Fifth, although Bni1p is required for efficient Kar9p localization, Kar9p is not required for GFP-Bni1p localization in mitotically dividing cells or shmoos. Thus, Bni1p mediates a series of functions in mating and mitosis that are not part of Kar9p's repertoire of functions.

If Bni1p's only function in nuclear migration were to localize Kar9p, then the bni1Δ kar9ΔΔ double mutant should not be more severe than the kar9Δ single mutant. However, the double mutant has a more severe phenotype than either single mutant alone, as evidenced both by a slight increase in the number of aberrant large-budded cells and by an increase in the percentage of multinucleate and anucleate unbudded cells (Table VI). Thus, Bni1p may have a secondary role in nuclear migration in addition to the localization of Kar9p. However, if Bni1p does have a secondary role independent of Kar9p, its contribution to nuclear migration must be relatively minor. Otherwise, the additive effects of the two independent defects should lead to a severe growth defect for the double mutant. Instead, we find that the double mutant grows no worse than the single mutants, identical to the wild type.

That anucleate cells are observed in the bni1Δ kar9Δ double mutant combination raises the additional possibility that Kar9p and/or Bni1p may play an inhibitory role in cytokinesis. This might be true if one or the other is part of the checkpoint mechanism that coordinates nuclear migration and cytokinesis. In addition to bud tip localization, both Bni1p and Kar9p are also localized at the mother-bud neck in a minor fraction of large budded cells (Evangelista and Boone, personal communication; Fujiwara et al., 1998; Miller and Rose, 1998). More likely, the increased severity of the double mutant may cause some cells to eventually complete cytokinesis without proper nuclear migration.

Kar9p Localization by Bni1p Is Highly SpecificConsistent with the observation that they all contribute to polarization, Spa2p, Pea2p, and Bud6p have been isolated as a stable complex (Sheu et al., 1998). Furthermore, Spa2p interacts with Bni1p by two-hybrid analysis (Fujiwara et al., 1998). Yet mutations in these proteins affected Kar9p localization differently, with only mild mitotic defects associated with spa2Δ and pea2Δ, and none apparent for bud6Δ. In shmoos and mitotic cells, the most severe effect on Kar9p localization occurred in the bni1Δ mutants. Nevertheless, all four of these mutants affect polarization in a similar manner and to a similar degree. Thus, the effects of bni1Δ on Kar9p localization are not likely to be due to general defects in polarization.All four mutants, spa2Δ, pea2Δ, bud6Δ, and bni1Δ, also affect cell fusion and bipolar budding. However, another bipolar budding mutant, rvs161Δ, exhibited no defects in Kar9p localization. In addition, Kar9p localization was not altered in the cell fusion mutants fus1Δ and fus2Δ. Thus, alterations in Kar9p localization are not due to general disruptions in either bipolar budding or the cell fusion process.Several other observations speak to the specificity of the effects of these mutations on Kar9p localization. Bnr1p is another formin family protein that shares overlapping functions with Bni1p (Imamura et al., 1997). Kar9p localized normally in bnr1Δ mutants (data not shown) and also did not show any genetic interactions with kar9Δ or dhc1Δ, indicating that the effects on Kar9p localization are not general to all formin proteins, but specific to Bni1p.Other mutants such as sla1Δ that affect the assembly of the actin cytoskeleton (Holtzman et al., 1993) also did not alter Kar9p localization in shmoos. This argues that the specialized interaction of Spa2p, Pea2p, Bud6p, and Bni1p with the actin cytoskeleton is the important feature for Kar9p localization.

Kar9p Localization by Bni1p Is Highly Specific

Consistent with the observation that they all contribute to polarization, Spa2p, Pea2p, and Bud6p have been isolated as a stable complex (Sheu et al., 1998). Furthermore, Spa2p interacts with Bni1p by two-hybrid analysis (Fujiwara et al., 1998). Yet mutations in these proteins affected Kar9p localization differently, with only mild mitotic defects associated with spa2Δ and pea2Δ, and none apparent for bud6Δ. In shmoos and mitotic cells, the most severe effect on Kar9p localization occurred in the bni1Δ mutants. Nevertheless, all four of these mutants affect polarization in a similar manner and to a similar degree. Thus, the effects of bni1Δ on Kar9p localization are not likely to be due to general defects in polarization.

All four mutants, spa2Δ, pea2Δ, bud6Δ, and bni1Δ, also affect cell fusion and bipolar budding. However, another bipolar budding mutant, rvs161Δ, exhibited no defects in Kar9p localization. In addition, Kar9p localization was not altered in the cell fusion mutants fus1Δ and fus2Δ. Thus, alterations in Kar9p localization are not due to general disruptions in either bipolar budding or the cell fusion process.

Several other observations speak to the specificity of the effects of these mutations on Kar9p localization. Bnr1p is another formin family protein that shares overlapping functions with Bni1p (Imamura et al., 1997). Kar9p localized normally in bnr1Δ mutants (data not shown) and also did not show any genetic interactions with kar9Δ or dhc1Δ, indicating that the effects on Kar9p localization are not general to all formin proteins, but specific to Bni1p.

Other mutants such as sla1Δ that affect the assembly of the actin cytoskeleton (Holtzman et al., 1993) also did not alter Kar9p localization in shmoos. This argues that the specialized interaction of Spa2p, Pea2p, Bud6p, and Bni1p with the actin cytoskeleton is the important feature for Kar9p localization.

Differences between Mitotic and Mating CellsThe localization patterns for Spa2p and Pea2p are interdependent, suggesting that their functions are closely interrelated (Valtz and Herskowitz, 1996). Consistent with this, we find that GFP-Kar9p mislocalization was very similar in the spa2Δ and pea2Δ mutants. Thus, with respect to Kar9p localization, these two mutants can be referred to as the “spa2 class.”Interestingly, the spa2 class showed a dramatic difference in the severity of Kar9p mislocalization comparing shmoos and mitotically dividing cells. In mitotic cells, mislocalization was apparent in only ∼10% of cells. In contrast, in most shmoos, Kar9p localization was altered from a single dot to a cap-like pattern. Thus, Spa2p and Pea2p likely play very different roles in polarization and microtubule orientation in mating cells.The finding that Kar9p localization is often spread out over a broader area in spa2Δ or pea2Δ shmoos is consistent with the proposed role for Spa2p and Pea2p in restricting the zone of polarization in shmoos (Valtz and Herskowitz, 1996). One prediction from such a model would be that a narrowed site of polarization would restrict the area of microtubule interaction with the cortex by restricting Kar9p localization to a dot. Consistent with this, we found a “splayed-out” microtubule pattern in a subset of spa2Δ mutants.Two models might explain the altered patterns of Kar9p localization. In one model, Spa2p and Pea2p may play a direct role in limiting Kar9p localization to a tight focus by helping aggregate the Bni1p/Kar9p complex. Alternatively, Spa2p and Pea2p might play an indirect role in Kar9p localization via their effects on shmoo morphology. In either case, correct polarization might be required to define a specialized region of the mating projection that is primed for cell and nuclear fusion. Such a primed region would facilitate the alignment of cytoplasmic microtubules, allowing nuclear congression to follow rapidly after cell fusion. In spa2Δ and pea2Δ mutant shmoos, the primed region would be more diffuse over the blunter mutant shmoo projection. In support of the indirect model, in spa2Δ prezygotes the clustering of vesicles along the zone of cell fusion is disrupted (Gammie et al., 1998). Regardless of the specific mechanism, these results highlight the differences in cell polarization between mating and mitotic cells.In contrast, both the bni1Δ mutation and actin depolymerization resulted in strong defects in Kar9p localization in both shmoos and mitotic cells. Although the bni1Δ mutant was defective for nuclear migration in both shmoos and mitotic cells, there were differences in the effects on microtubule orientation. In bni1Δ shmoos, the microtubules were clearly misoriented, whereas mitotic cells did not exhibit an obvious defect. One interpretation might be that correct Kar9p localization is more important for microtubule orientation in mating cells than in mitotic cells. Alternatively, the difference between the two cells may arise from the different patterns of mislocalized Kar9p. In the bni1Δ shmoos, GFP-Kar9p was present in a diffuse speckled pattern. In the bni1Δ mutant, GFP-Kar9p mislocalized to secondary sites at the SPB and bud neck, where it might still be able to provide some orientation to the microtubules.

Differences between Mitotic and Mating Cells

The localization patterns for Spa2p and Pea2p are interdependent, suggesting that their functions are closely interrelated (Valtz and Herskowitz, 1996). Consistent with this, we find that GFP-Kar9p mislocalization was very similar in the spa2Δ and pea2Δ mutants. Thus, with respect to Kar9p localization, these two mutants can be referred to as the “spa2 class.”

Interestingly, the spa2 class showed a dramatic difference in the severity of Kar9p mislocalization comparing shmoos and mitotically dividing cells. In mitotic cells, mislocalization was apparent in only ∼10% of cells. In contrast, in most shmoos, Kar9p localization was altered from a single dot to a cap-like pattern. Thus, Spa2p and Pea2p likely play very different roles in polarization and microtubule orientation in mating cells.

The finding that Kar9p localization is often spread out over a broader area in spa2Δ or pea2Δ shmoos is consistent with the proposed role for Spa2p and Pea2p in restricting the zone of polarization in shmoos (Valtz and Herskowitz, 1996). One prediction from such a model would be that a narrowed site of polarization would restrict the area of microtubule interaction with the cortex by restricting Kar9p localization to a dot. Consistent with this, we found a “splayed-out” microtubule pattern in a subset of spa2Δ mutants.

Two models might explain the altered patterns of Kar9p localization. In one model, Spa2p and Pea2p may play a direct role in limiting Kar9p localization to a tight focus by helping aggregate the Bni1p/Kar9p complex. Alternatively, Spa2p and Pea2p might play an indirect role in Kar9p localization via their effects on shmoo morphology. In either case, correct polarization might be required to define a specialized region of the mating projection that is primed for cell and nuclear fusion. Such a primed region would facilitate the alignment of cytoplasmic microtubules, allowing nuclear congression to follow rapidly after cell fusion. In spa2Δ and pea2Δ mutant shmoos, the primed region would be more diffuse over the blunter mutant shmoo projection. In support of the indirect model, in spa2Δ prezygotes the clustering of vesicles along the zone of cell fusion is disrupted (Gammie et al., 1998). Regardless of the specific mechanism, these results highlight the differences in cell polarization between mating and mitotic cells.

In contrast, both the bni1Δ mutation and actin depolymerization resulted in strong defects in Kar9p localization in both shmoos and mitotic cells. Although the bni1Δ mutant was defective for nuclear migration in both shmoos and mitotic cells, there were differences in the effects on microtubule orientation. In bni1Δ shmoos, the microtubules were clearly misoriented, whereas mitotic cells did not exhibit an obvious defect. One interpretation might be that correct Kar9p localization is more important for microtubule orientation in mating cells than in mitotic cells. Alternatively, the difference between the two cells may arise from the different patterns of mislocalized Kar9p. In the bni1Δ shmoos, GFP-Kar9p was present in a diffuse speckled pattern. In the bni1Δ mutant, GFP-Kar9p mislocalized to secondary sites at the SPB and bud neck, where it might still be able to provide some orientation to the microtubules.

Coordinating Cytoskeletal SystemsAbundant evidence has shown that Bni1p, specifically, and formins, generally, are involved in organizing the actin cytoskeleton. In addition to Spa2p and Bud6p, Bni1p interacts with profilin and the small G-protein Rho1p (Kohno et al., 1996; Evangelista et al., 1997). It seems likely that formins may play a general role in linking the actin and microtubule cytoskeletons. For example, mutations in the Drosophila formin, cappuccino result in mislocalized molecular determinants and abnormal microtubule patterns (Theurkauf, 1994; Emmons et al., 1995).In conclusion, we report here that Kar9p localization is dependent upon actin and the polarization protein, Bni1p. This provides the first evidence in yeast for a functional link between the actin and microtubule cytoskeletons. These data suggest a mechanism by which the polarized information contained within Kar9p is transferred into the vectorial movement of spindle orientation and nuclear movement into the bud. This work provides an avenue into future investigations as to how nuclear positioning is coordinated with the growth of the bud and cytokinesis.

Coordinating Cytoskeletal Systems

Abundant evidence has shown that Bni1p, specifically, and formins, generally, are involved in organizing the actin cytoskeleton. In addition to Spa2p and Bud6p, Bni1p interacts with profilin and the small G-protein Rho1p (Kohno et al., 1996; Evangelista et al., 1997). It seems likely that formins may play a general role in linking the actin and microtubule cytoskeletons. For example, mutations in the Drosophila formin, cappuccino result in mislocalized molecular determinants and abnormal microtubule patterns (Theurkauf, 1994; Emmons et al., 1995).

In conclusion, we report here that Kar9p localization is dependent upon actin and the polarization protein, Bni1p. This provides the first evidence in yeast for a functional link between the actin and microtubule cytoskeletons. These data suggest a mechanism by which the polarized information contained within Kar9p is transferred into the vectorial movement of spindle orientation and nuclear movement into the bud. This work provides an avenue into future investigations as to how nuclear positioning is coordinated with the growth of the bud and cytokinesis.
