---
pmid: '24336208'
title: Rare coding variants in the phospholipase D3 gene confer risk for Alzheimer's
  disease.
authors:
- Cruchaga C
- Karch CM
- Jin SC
- Benitez BA
- Cai Y
- Guerreiro R
- Harari O
- Norton J
- Budde J
- Bertelsen S
- Jeng AT
- Cooper B
- Skorupa T
- Carrell D
- Levitch D
- Hsu S
- Choi J
- Ryten M
- Sassi C
- Bras J
- Gibbs RJ
- Hernandez DG
- Lupton MK
- Powell J
- Forabosco P
- Ridge PG
- Corcoran CD
- Tschanz JT
- Norton MC
- Munger RG
- Schmutz C
- Leary M
- Demirci FY
- Bamne MN
- Wang X
- Lopez OL
- Ganguli M
- Medway C
- Turton J
- Lord J
- Braae A
- Barber I
- Brown K
- Alzheimer's Research UK (ARUK) Consortium
- Pastor P
- Lorenzo-Betancor O
- Brkanac Z
- Scott E
- Topol E
- Morgan K
- Rogaeva E
- Singleton A
- Hardy J
- Kamboh MI
- George-Hyslop PS
- Cairns N
- Morris JC
- Kauwe JSK
- Goate AM
journal: Nature
year: '2014'
full_text_available: false
full_text_extraction_method: html_abstract_only
pmcid: PMC4050701
doi: 10.1038/nature12825
---

# Rare coding variants in the phospholipase D3 gene confer risk for Alzheimer's disease.
**Authors:** Cruchaga C, Karch CM, Jin SC, Benitez BA, Cai Y, Guerreiro R, Harari O, Norton J, Budde J, Bertelsen S, Jeng AT, Cooper B, Skorupa T, Carrell D, Levitch D, Hsu S, Choi J, Ryten M, Sassi C, Bras J, Gibbs RJ, Hernandez DG, Lupton MK, Powell J, Forabosco P, Ridge PG, Corcoran CD, Tschanz JT, Norton MC, Munger RG, Schmutz C, Leary M, Demirci FY, Bamne MN, Wang X, Lopez OL, Ganguli M, Medway C, Turton J, Lord J, Braae A, Barber I, Brown K, Alzheimer's Research UK (ARUK) Consortium, Pastor P, Lorenzo-Betancor O, Brkanac Z, Scott E, Topol E, Morgan K, Rogaeva E, Singleton A, Hardy J, Kamboh MI, George-Hyslop PS, Cairns N, Morris JC, Kauwe JSK, Goate AM
**Journal:** Nature (2014)
**DOI:** [10.1038/nature12825](https://doi.org/10.1038/nature12825)
**PMC:** [PMC4050701](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4050701/)

## Abstract

1. Nature. 2014 Jan 23;505(7484):550-554. doi: 10.1038/nature12825. Epub 2013 Dec
 11.

Rare coding variants in the phospholipase D3 gene confer risk for Alzheimer's 
disease.

Cruchaga C(1)(2), Karch CM(#)(1)(2), Jin SC(#)(1), Benitez BA(1), Cai Y(1), 
Guerreiro R(3)(4), Harari O(1), Norton J(1), Budde J(1), Bertelsen S(1), Jeng 
AT(1), Cooper B(1), Skorupa T(1), Carrell D(1), Levitch D(1), Hsu S(1), Choi 
J(1), Ryten M(3)(5), Sassi C(3)(4), Bras J(3), Gibbs RJ(3)(4), Hernandez 
DG(3)(4), Lupton MK(6)(7), Powell J(6), Forabosco P(8), Ridge PG(9), Corcoran 
CD(10)(11), Tschanz JT(11)(12), Norton MC(11)(12)(13), Munger RG(13)(14), 
Schmutz C(9), Leary M(9), Demirci FY(15), Bamne MN(15), Wang X(15), Lopez 
OL(16)(17), Ganguli M(18), Medway C(19), Turton J(19), Lord J(19), Braae A(19), 
Barber I(19), Brown K(19); Alzheimer's Research UK (ARUK) Consortium; Pastor 
P(20)(21)(22), Lorenzo-Betancor O(20), Brkanac Z(23), Scott E(24), Topol E(24), 
Morgan K(19), Rogaeva E(25), Singleton A(4), Hardy J(3), Kamboh MI(16)(16)(18), 
George-Hyslop PS(25)(26), Cairns N(2)(27), Morris JC(27)(28)(29), Kauwe JSK(9), 
Goate AM(1)(2)(28)(29)(30).

Author information:
(1)Department of Psychiatry, Washington University, St. Louis, MO, USA.
(2)Hope Center Program on Protein Aggregation and Neurodegeneration, Washington 
University St. Louis, MO, USA.
(3)Department of Molecular Neuroscience, UCL Institute of Neurology, London WC1N 
3BG, UK.
(4)Laboratory of Neurogenetics, National Institute on Aging, National Institutes 
of Health, Bethesda, Maryland, United States of America.
(5)on behalf of UKBEC (UK Brain Expression Consortium).
(6)Institute of Psychiatry, King's College London, London, UK.
(7)Neuroimaging Genetics, QIMR Berghofer Medical Research Institute, Brisbane, 
Australia.
(8)Istituto di Genetica delle Popolazioni - CNR, Sassari, Italy.
(9)Department of Biology, Brigham Young University, Provo, UT, 84602.
(10)Department of Mathematics and Statistics, Utah State University, Logan, UT.
(11)Center for Epidemiologic Studies, Utah State University, Logan, UT.
(12)Department of Psychology, Utah State University, Logan, UT.
(13)Department of Family Consumer and Human Development, Utah State University, 
Logan, UT.
(14)Department of Nutrition, Dietetics, and Food Sciences, Utah State 
University, Logan, UT.
(15)Department of Human Genetics, University of Pittsburgh, Pittsburgh, PA.
(16)Alzheimer's Disease Research Center, University of Pittsburgh, Pittsburgh, 
PA.
(17)Department of Neurology, University of Pittsburgh, Pittsburgh, PA.
(18)Department of Psychiatry, University of Pittsburgh, Pittsburgh, PA.
(19)Human Genetics, School of Molecular Medical Sciences, University of 
Nottingham, Nottingham, NG7 2UH, UK.
(20)Neurogenetics Laboratory, Division of Neurosciences, Center for Applied 
Medical Research, University of Navarra, Pamplona, Spain.
(21)Department of Neurology, Clínica Universidad de Navarra, School of Medicine, 
University of Navarra, Pamplona, Spain.
(22)CIBERNED, Centro de Investigación Biomédica en Red de Enfermedades 
Neurodegenerativas, Instituto de Salud Carlos III, Spain.
(23)University of Washington. Seattle, WA.
(24)The Scripps Research Institute, La Jolla, CA, US.
(25)Tanz Centre for Research in Neurodegenerative Diseases, University of 
Toronto.
(26)Cambridge Institute for Medical Research, and the Department of Clinical 
Neurosciences, University of Cambridge.
(27)Pathology and Immunology, Washington University, St. Louis, MO, USA.
(28)Department of Neurology, Washington University, St. Louis, MO, USA.
(29)Knight ADRC, Washington University, St. Louis, MO, USA.
(30)Department of Genetics, Washington University, St. Louis, MO, USA.
(#)Contributed equally

Comment in
    Nature. 2015 Apr 2;520(7545):E1. doi: 10.1038/nature14036.
    Nature. 2015 Apr 2;520(7545):E10. doi: 10.1038/nature14041.
    Nature. 2015 Apr 2;520(7545):E2-3. doi: 10.1038/nature14038.
    Nature. 2015 Apr 2;520(7545):E3-5. doi: 10.1038/nature14039.
    Nature. 2015 Apr 2;520(7545):E5-6. doi: 10.1038/nature14037.
    Nature. 2015 Apr 2;520(7545):E7-8. doi: 10.1038/nature14040.

Genome-wide association studies (GWAS) have identified several risk variants for 
late-onset Alzheimer's disease (LOAD). These common variants have replicable but 
small effects on LOAD risk and generally do not have obvious functional effects. 
Low-frequency coding variants, not detected by GWAS, are predicted to include 
functional variants with larger effects on risk. To identify low-frequency 
coding variants with large effects on LOAD risk, we carried out whole-exome 
sequencing (WES) in 14 large LOAD families and follow-up analyses of the 
candidate variants in several large LOAD case-control data sets. A rare variant 
in PLD3 (phospholipase D3; Val232Met) segregated with disease status in two 
independent families and doubled risk for Alzheimer's disease in seven 
independent case-control series with a total of more than 11,000 cases and 
controls of European descent. Gene-based burden analyses in 4,387 cases and 
controls of European descent and 302 African American cases and controls, with 
complete sequence data for PLD3, reveal that several variants in this gene 
increase risk for Alzheimer's disease in both populations. PLD3 is highly 
expressed in brain regions that are vulnerable to Alzheimer's disease pathology, 
including hippocampus and cortex, and is expressed at significantly lower levels 
in neurons from Alzheimer's disease brains compared to control brains. 
Overexpression of PLD3 leads to a significant decrease in intracellular 
amyloid-β precursor protein (APP) and extracellular Aβ42 and Aβ40 (the 42- and 
40-residue isoforms of the amyloid-β peptide), and knockdown of PLD3 leads to a 
significant increase in extracellular Aβ42 and Aβ40. Together, our genetic and 
functional data indicate that carriers of PLD3 coding variants have a twofold 
increased risk for LOAD and that PLD3 influences APP processing. This study 
provides an example of how densely affected families may help to identify rare 
variants with large effects on risk for disease or other complex traits.

DOI: 10.1038/nature12825
PMCID: PMC4050701
PMID: 24336208 [Indexed for MEDLINE]

## Full Text

Abstract

Genome-wide association studies (GWAS) have identified several risk
variants for late-onset Alzheimer's disease (LOAD) 1 , 2 . These
common variants have replicable but small effects on LOAD risk and generally do
not have obvious functional effects. Low-frequency coding variants, not detected
by GWAS, are predicted to include functional variants with larger effects on
risk. To identify low frequency coding variants with large effects on LOAD risk,
we performed whole exome-sequencing (WES) in 14 large LOAD families and
follow-up analyses of the candidate variants in several large case-control
datasets. A rare variant in PLD3 (phospholipase-D family,
member 3, rs145999145; V232M) segregated with disease status in two independent
families and doubled risk for AD in seven independent case-control series (V232M
meta-analysis; OR= 2.10, CI=1.47-2.99; p= 2.93×10 -5 , 11,354
cases and controls of European-descent). Gene-based burden analyses in 4,387
cases and controls of European-descent and 302 African American cases and
controls, with complete sequence data for PLD3, indicate that
several variants in this gene increase risk for AD in both populations (EA: OR=
2.75, CI=2.05-3.68; p=1.44×10 -11 , AA: OR= 5.48, CI=1.77-16.92;
p=1.40×10 -3 ). PLD3 is highly expressed in
brain regions vulnerable to AD pathology, including hippocampus and cortex, and
is expressed at lower levels in neurons from AD brains compared to control
brains (p=8.10×10 -10 ). Over-expression of PLD3 leads to a
significant decrease in intracellular APP and extracellular Aβ42 and
Aβ40, while knock-down of PLD3 leads to a significant increase in
extracellular Aβ42 and Aβ40. Together, our genetic and functional
data indicate that carriers of PLD3 coding variants have a
two-fold increased risk for LOAD and that PLD3 influences APP
processing. This study provides an example of how densely affected families may
be used to identify rare variants with large effects on risk for disease or
other complex traits.
