---
pmid: '38242633'
title: SpoVAF and FigP assemble into oligomeric ion channels that enhance spore germination.
authors:
- Gao Y
- Amon JD
- Brogan AP
- Artzi L
- Ramírez-Guadiana FH
- Cofsky JC
- Kruse AC
- Rudner DZ
journal: Genes Dev
year: '2024'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC10903944
doi: 10.1101/gad.351353.123
---

# SpoVAF and FigP assemble into oligomeric ion channels that enhance spore germination.
**Authors:** Gao Y, Amon JD, Brogan AP, Artzi L, Ramírez-Guadiana FH, Cofsky JC, Kruse AC, Rudner DZ
**Journal:** Genes Dev (2024)
**DOI:** [10.1101/gad.351353.123](https://doi.org/10.1101/gad.351353.123)
**PMC:** [PMC10903944](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10903944/)

## Abstract

1. Genes Dev. 2024 Feb 13;38(1-2):31-45. doi: 10.1101/gad.351353.123.

SpoVAF and FigP assemble into oligomeric ion channels that enhance spore 
germination.

Gao Y(1), Amon JD(1), Brogan AP(1), Artzi L(1), Ramírez-Guadiana FH(1), Cofsky 
JC(2), Kruse AC(2), Rudner DZ(3).

Author information:
(1)Department of Microbiology, Harvard Medical School, Boston, Massachusetts 
02115, USA.
(2)Department of Biological Chemistry and Molecular Pharmacology, Harvard 
Medical School, Boston, Massachusetts 02115, USA.
(3)Department of Microbiology, Harvard Medical School, Boston, Massachusetts 
02115, USA; rudner@hms.harvard.edu.

Comment in
    Genes Dev. 2024 Feb 13;38(1-2):1-3. doi: 10.1101/gad.351554.124.

Bacterial spores can remain dormant for decades yet rapidly germinate and resume 
growth in response to nutrients. GerA family receptors that sense and respond to 
these signals have recently been shown to oligomerize into nutrient-gated ion 
channels. Ion release initiates exit from dormancy. Here, we report that a 
distinct ion channel, composed of SpoVAF (5AF) and its newly discovered partner 
protein, YqhR (FigP), amplifies the response. At high germinant concentrations, 
5AF/FigP accelerate germination; at low concentrations, this complex becomes 
critical for exit from dormancy. 5AF is homologous to the channel-forming 
subunit of GerA family receptors and is predicted to oligomerize around a 
central pore. 5AF mutations predicted to widen the channel cause constitutive 
germination during spore formation and membrane depolarization in vegetative 
cells. Narrow-channel mutants are impaired in germination. A screen for 
suppressors of a constitutively germinating 5AF mutant identified FigP as an 
essential cofactor of 5AF activity. We demonstrate that 5AF and FigP interact 
and colocalize with GerA family receptors in spores. Finally, we show that 
5AF/FigP accelerate germination in B. subtilis spores that have nutrient 
receptors from another species. Our data support a model in which 
nutrient-triggered ion release by GerA family receptors activates 5AF/FigP ion 
release, amplifying the response to germinant signals.

© 2024 Gao et al.; Published by Cold Spring Harbor Laboratory Press.

DOI: 10.1101/gad.351353.123
PMCID: PMC10903944
PMID: 38242633 [Indexed for MEDLINE]

## Full Text

Bacteria in the orders Bacillales and Clostridiales are responsible for more than 1 million infections each year and account for huge monetary losses to the food industry (Mallozzi et al. 2010; Andre et al. 2017). These pathogens resist sterilization by differentiating into stress-resistant spores. Spores are metabolically inactive and can remain dormant for decades yet rapidly resume growth upon exposure to nutrients, causing food spoilage, food-borne illness, or life-threatening disease (Setlow 2014). The exit from dormancy, called germination, is a key target in combatting these pathogens. The germination program of most spore-forming bacteria involves a common series of chemical steps catalyzed by a small set of broadly conserved factors (Moir and Cooper 2015; Setlow et al. 2017). Germinant receptors embedded in the spore membrane monitor the environment for germination signals, typically amino acids, sugars, and/or nucleosides. These receptors are nutrient-gated ion channels, and detection of specific nutrients triggers the release of ions from the spore core (Gao et al. 2023). Evidence suggests that ion release activates the transport of large stores of dipicolinic acid (DPA) from the spore via the SpoVA (Spo5A or 5A) transport complex (Vepachedu and Setlow 2007; Gao et al. 2022, 2023). Released DPA in turn activates cell wall hydrolases that degrade the specialized peptidoglycan that encases the spore, allowing core rehydration, macromolecular synthesis, and resumption of growth (Setlow 2003; Moir 2006).

The prototypical germinant receptor in Bacillus subtilis, GerA, is composed of three broadly conserved subunits: GerAA, GerAB, and GerAC (Paredes-Sabja et al. 2011). GerAA and GerAB are polytopic membrane proteins, and GerAC is a lipoprotein (Moir and Cooper 2015). GerAA is predicted to oligomerize into a high-confidence pentamer or hexamer with a membrane channel at its oligomeric interface (Gao et al. 2023; Kilian and Bischofs 2023). Each GerAA protomer in the complex interacts with a GerAB subunit that is responsible for L-alanine recognition (Artzi et al. 2021; Gao et al. 2023). Accordingly, a ring of ligand-binding subunits encircles the oligomeric channel. GerAC protomers are also predicted to oligomerize. Each GerAC protomer binds GerAB and GerAA and bridges neighboring GerAA protomers (Gao et al. 2023). In response to L-alanine, the nutrient-gated ion channel opens, releasing ions. Ion release in turn triggers expulsion of DPA from the spore core via a separate membrane complex encoded in the spo5A (or 5A) locus (Fort and Errington 1985; Gao et al. 2022). The B. subtilis 5A locus consists of seven genes (5AA, 5AB, 5AC, 5AD, 5AEb, 5AEa, and 5AF), of which only three (5AC, 5AEb, and 5AD) are required for DPA export (Gao et al. 2022). The functions of the other genes in the 5A operon are poorly understood. Interestingly, 5AF encodes a homolog of the pore-forming GerAA protein (Errington 1993), suggesting that 5AF may also conduct ions or even play a role in germination-associated ion release. However, it is unclear how an additional ion channel would fit into a model that already contains nutrient-gated ion release by GerA family receptors.

Here, we report that the 5AF protein functions as an ion channel that amplifies nutrient-activated ion release in germinating spores. We show that spores lacking 5AF are impaired in ion release and subsequent steps in spore germination. In an unbiased screen for 5AF mutants that constitutively germinate, we identified amino acid substitutions in 5AF that are predicted to widen the membrane channel or stabilize it in an open conformation. Reciprocally, we engineered 5AF variants predicted to narrow the channel, which exhibited impaired ion release. Using one of the constitutive germination alleles, we screened for suppressors and identified YqhR (FigP) as an essential cofactor of 5AF. AlphaFold2-Multimer predicted that 5AF and FigP form a heterodimer that can oligomerize in a manner similar to the GerA complex. We demonstrate that 5AF and FigP interact, form oligomers in vivo, and colocalize with GerA family receptors in spores. We further show that expression of the channel-widening 5AF allele during vegetative growth is toxic and causes loss of membrane potential, provided FigP is coexpressed. Finally, we show that 5AF/FigP amplify germination responses in B. subtilis spores even when germinant receptors or DPA transporters are replaced by homologs from Bacillus megaterium, Clostridiodes difficile, or Bacillus cereus. Collectively, our data support a model in which 5AF and FigP form an oligomeric membrane channel that releases ions in response to ion release by GerA family nutrient receptors, amplifying the germination response.

Results5AF is predicted to oligomerize into a membrane channel and is required for ion release during germinationThe last gene in the B. subtilis 5A locus, 5AF, encodes a homolog of the channel-forming A subunits of GerA family receptors (Errington 1993; Perez-Valdespino et al. 2014). The AlphaFold-predicted structure of 5AF resembles the GerAA model, with a TM-align score of 0.69 and RMSD of 2.58 Å within the high-confidence regions of the model (Fig. 1A; Supplemental Fig. S1A). Furthermore, AlphaFold2-Multimer (Jumper et al. 2021; Evans et al. 2022) predicts that 5AF monomers assemble into a high-confidence pentamer surrounding a central channel, similar to the GerAA pentamer model (Fig. 1B; Supplemental Fig. S1B; Gao et al. 2023). AlphaFold2-Multimer also predicts that GerAA and, separately, 5AF can form high-confidence hexamers (Supplemental Fig. S1C; Kilian and Bischofs 2023). For clarity, we present the pentamer model throughout this article but are currently unable to experimentally distinguish between the two models.Figure 1.Spo5AF (5AF) is predicted to form an oligomeric membrane channel and is required for efficient ion release during germination. (A) Structural alignment by TM-align of the AlphaFold-predicted structures of GerAA (magenta) and 5AF (cyan). (B) AlphaFold-Multimer predictions of 5AF and GerAA pentamers viewed from outside the spore. Protomers are shown in dark and light gray and cyan (5AF) or magenta (GerAA). (C) Purified spores from wild type, Δ5AF, or Δ5AF complemented with 5AF in trans were mixed with 10 mM L-alanine, and optical density was analyzed over 90 min. (D,E) The germination exudates from the same spores used in C were collected at the indicated times and analyzed for DPA (D) and K+ (E). The percentage release at each time point, relative to the amount released by WT spores at 60 min, is shown. A complete analysis of ion release and biological replicates is shown in Supplemental Figure S3. (F) Purified WT and Δ5AF spores were incubated with and without 20 µM L-alanine without agitation for the indicated times in hours at 37°C. The percentage of DPA released relative to total is shown. Time points analyzed by phase-contrast microscopy are indicated (red arrows). (G) Representative phase-contrast images of WT and Δ5AF spores incubated with 20 µM L-alanine for the indicated times. (H) A functional 5AF-GFP fusion colocalizes with GerAA-mScarlet in dormant spores. Green and red arrowheads indicate colocalized germinosome foci. (I) GerD is required for 5AF-GFP foci in dormant spores. Scale bars, 2 µm.The structural similarity of 5AF to GerAA motivated us to perform a more detailed analysis of 5AF. Although the 5AF gene is in the spo5A locus, it is not required for DPA import into developing spores (Perez-Valdespino et al. 2014; Gao et al. 2022), and its role in germination is unclear. A deletion of 5AF and the gene adjacent to it, 5AEa, was previously reported to cause a delay in germination (Perez-Valdespino et al. 2014). Our analysis of in-frame deletions in 5AF and separately in 5AEa revealed that the loss of 5AF was solely responsible for the delayed germination, as assessed by DPA release and optical density (Fig. 1C,D; Supplemental Fig. S2A–D). Furthermore, normal germination kinetics were restored to the Δ5AF mutant when 5AF was expressed in trans.To determine whether 5AF is involved in ion release, we exposed wild-type and Δ5AF spores to 10 mM L-alanine and analyzed the germination exudates using inductively coupled plasma mass spectrometry (ICP-MS). As shown in Figure 1E, Δ5AF spores were delayed in ion release, and expression of 5AF in trans restored wild-type release rates (Fig. 1E; Supplemental Fig. S3). These findings are consistent with the idea that 5AF oligomers function as ion channels and release ions in response nutrients, thereby accelerating exit from dormancy. Finally, we note that the correlation between delayed release of ions and delayed release of DPA provides additional support for the prevailing model that ion release is the activating signal for DPA export (Gao et al. 2023).Spores lacking GerA are unable to germinate upon exposure to L-alanine (Moir and Cooper 2015; Setlow et al. 2017). However, ΔgerA spores germinate efficiently in the presence of asparagine, glucose, fructose, and K+ ions (AGFK), which signal through the homologous GerB and GerK nutrient receptors. Similarly, ΔgerA spores germinate and form colonies on LB agar plates at efficiencies similar to that of wild type, but spores lacking GerA, GerB, and GerK are reduced in colony formation by ∼500-fold (Supplemental Fig. S2E). To investigate whether 5AF responds to nutrients, we analyzed colony formation of Δ5AF spores and spores lacking 5AF and all GerA family loci (Δger5): gerA, gerB, gerK, and two additional ger loci (yndDEF and yfkQRT) of unknown function (Paidhungat and Setlow 2000). Spores lacking 5AF formed colonies at efficiencies similar to that of wild type, and spores lacking 5AF and all GerA family receptors formed colonies at efficiencies similar to that of the Δger5 gerA family deletion strain (Supplemental Fig. S2E). The absence of any further reduction in colony formation suggests that 5AF does not directly respond to nutrients and instead is activated by the GerA family receptors.Δ5AF spores exhibit a relatively modest delay in germination (Fig. 1C). We wondered whether our standard germination assays, in which we exposed spores to 1–10 mM L-alanine, were masking a more important role for this putative ion channel under conditions that more closely mimic those encountered in the environment. To explore this possibility, we incubated wild-type and Δ5AF spores with low concentrations of nutrients (20 µM L-alanine) without agitation and monitored germination by phase-contrast microscopy and DPA release over 24 h. Approximately 60% of the total DPA in wild-type spores was released over the 24-h time course compared with ∼10% in spores lacking 5AF (Fig. 1F). Phase-contrast microscopy revealed a similar reduction of phase-dark germinated spores in the Δ5AF mutant compared with wild type (Fig. 1G). These data argue that 5AF can significantly increase the number of spores that germinate over 24 h. Collectively, these findings support the idea that 5AF-dependent ion release amplifies GerA's response to L-alanine, which is even more critical under nutrient-poor conditions.GerA family receptors have been shown to colocalize in foci called germinosomes in dormant spores (Griffiths et al. 2011). To investigate whether 5AF is also present in germinosomes, we generated a functional GFP fusion (Supplemental Fig. S4) and analyzed its localization. The 5AF-GFP fusion formed discrete foci that colocalized with GerAA-mScarlet (Fig. 1H). Importantly, the 5AF-GFP foci were lost in spores lacking GerD, a lipoprotein required for germinosome formation (Fig. 1I; Mongkolthanaruk et al. 2009; Griffiths et al. 2011). Altogether, we conclude that 5AF proteins are predicted to assemble into oligomeric ion channels and colocalize with GerA family nutrient-gated ion channels and are required for efficient ion release and exit from dormancy in response to nutrients.An unbiased screen for 5AF mutations that cause premature germinationIn a previous study, we identified constitutively active mutations in gerAA that caused developing spores to prematurely germinate. These alleles mapped to residues within or adjacent to the lumen of the predicted membrane channel of the GerAA oligomer, providing strong genetic support for the AlphaFold prediction (Gao et al. 2023). Accordingly, we performed an unbiased screen for constitutively active 5AF alleles (Supplemental Fig. S5). Briefly, the 5AF gene was PCR-mutagenized and transformed into a Δ5AF mutant, and the transformants were pooled and sporulated. We isolated prematurely germinated spores from denser dormant spores on a histodenz step gradient and inoculated them into rich medium that supports growth and efficient sporulation upon nutrient exhaustion. After a second round of enrichment, the prematurely germinated spores were plated onto sporulation agar plates, and the resulting colonies were sporulated for 96 h at 37°C. Spores from >200 individual colonies were analyzed by phase-contrast microscopy, and two mutants were identified that had high frequencies of premature germination: A360V and L367S (Fig. 2A; Supplemental Fig. S6A). Sporulated cultures of both mutants also displayed severe reductions in heat-resistant colony-forming units (CFUs) (Fig. 2A). The reduction in heat-resistant CFUs is particularly striking given that the Δ5AF mutant is not impaired in spore formation. Importantly, these mutant phenotypes did not require the GerA family receptors (Supplemental Fig. S6A). The L367S mutant mapped within the lumen of the channel model and is predicted to widen the pore. The A360V mutant mapped adjacent to the lumen and is predicted to stabilize an open conformation of the channel (Fig. 2B; Supplemental Fig. S7).Figure 2.Characterization of 5AF mutants predicted to widen or narrow the membrane channel. (A) Representative phase-contrast images of sporulated cultures of strains with 5AF(WT), Δ5AF, and the indicated constitutively active 5AF alleles. Heat-resistant CFUs of each strain compared with wild type are shown below the images. (B) Side and top views of the three transmembrane helices that surround the membrane channel in the 5AF pentamer model. Residues identified in the screen and those that are predicted to line the channel are indicated. (C) Representative phase-contrast images of sporulated cultures of strains with 5AF alleles predicted to narrow the channel. Percentages of heat-resistant CFUs are indicated below the images. (D) Purified spores from the indicated strains were mixed with 10 mM L-alanine, and the germination exudates were collected at the indicated times and analyzed for K+ and Ca2+. The percentage released at each time point, relative to the amount released by 5AF(WT) spores at 60 min, is shown. The channel-narrowing mutant 5AF(V371L) phenocopies Δ5FA. Similar results were obtained for all steps in the germination pathway and with 5AF(Q375L) and are shown in Supplemental Figure S8. (E) Purified spores of the indicated strains were germinated with 20 µM L-alanine without agitation, and the percentage of DPA released relative to total was monitored over time (in hours). Similar differences between WT and Δ5AF were observed at lower concentrations of L-alanine with agitation. (F) Immunoblots from lysates of the purified spores used in D. Functional ProteinC (ProC) fusions to 5AF(WT), 5AF(V371L), and 5AF(Q375L) are stable. σA (SigA) controlled for loading. YqhR-His is stable in the mutant spores. (G) Representative fluorescence images of functional GFP fusions to 5AF(WT) and 5AF(V371L) in dormant spores. Red arrowheads indicate germinosome foci.To further investigate the channel model, we substituted lumen-facing residues V371 and Q375 with leucines, which we predicted would narrow or occlude the channel (Fig. 2B). Both mutants formed phase-bright spores that were heat-resistant (Fig. 2C), similar to wild type and the Δ5AF mutant. However, when these spores were exposed to nutrients, they were delayed in ion release and all subsequent steps of germination (Fig. 2D; Supplemental Fig. S8). Similarly, when the mutant spores were exposed to low concentrations of L-alanine, they were impaired in exit from dormancy (Fig. 2E). Importantly, the mutant 5AF proteins were stable in dormant spores (Fig. 2F) and localized in germinosome foci (Fig. 2G), suggesting that the lumen-facing mutants assemble into complexes but are unable to amplify the GerA response to L-alanine. The discovery of germination-controlling mutations within the predicted membrane channel supports the AlphaFold model that 5AF oligomerizes into an ion channel.Identification of YqhR (FigP) as a 5AF partner proteinGerAA resides in a complex with the nutrient-sensing subunit GerAB and the lipoprotein GerAC (Artzi et al. 2021; Amon et al. 2022; Gao et al. 2023). These three proteins depend on each other for stability in dormant spores and require each other for channel function in vegetatively growing cells (Cooper and Moir 2011; Mongkolthanaruk et al. 2011; Gao et al. 2023). We therefore wondered whether 5AF has one or more partner proteins. To search for potential interacting proteins, we screened for loss-of-function mutations that suppress the sporulation defect associated with the constitutive 5AF allele A360V. We transposon-mutagenized cells harboring 5AF(A360V) and then pooled and sporulated the transposon (Tn) library. The sporulated culture was subjected to 80°C for 20 min to kill vegetative cells that had failed to sporulate and spores that had prematurely germinated. The mixture was then plated on rich medium, allowing the heat-resistant spores to germinate and form colonies. The colonies were pooled and the Tn insertions were mapped by deep sequencing (Tn-seq). The screen identified five genes (yqhR, ymfJ, ymfD, ytpA, and yktB) that were enriched for Tn insertions compared with the input library (Fig. 3A; Supplemental Fig. S9A). The yqhR, ymfJ, and ymfD genes are expressed in the forespore under SigF or SigG control (Nicolas et al. 2012; Arrieta-Ortiz et al. 2015). yqhR encodes an uncharacterized membrane protein, ymfJ encodes a cytoplasmic protein with a DUF3243 domain, and ymfD encodes a putative transporter of the major facilitator superfamily. ytpA and yktB are expressed during vegetative growth and are not specifically induced during sporulation. ytpA encodes a putative phospholipase, and yktB encodes a cytoplasmic protein with a DUF1054 domain.Figure 3.Identification and characterization of FigP, an essential 5AF cofactor. (A) A transposon sequencing (Tn-seq) screen for mutations that suppress the premature germination of 5AF(A360V) identified yqhR (figP). Transposon insertion profiles from the yqhR region of the genome are shown. Tn libraries from wild type (WT) and 5AF(A360V) were sporulated, incubated for 20 min at 80°C, and plated on LB agar. The colonies were pooled and Tn sites were mapped by deep sequencing. Each line represents a Tn insertion site; the height reflects its abundance in the output library. The maximum number of reads was set at 10,000. Tn insertions in yqhR (figP) were overrepresented in 5AF(A360V) compared with WT. (B) Representative phase-contrast images of the indicated strains. ΔyqhR suppresses the premature germination phenotype of 5AF(A360V). The percentages of heat-resistant spores compared with wild type are indicated in the bottom right corners of the images. (C) Purified spores from the indicated strains were mixed with 1 mM L-alanine, and DPA release was monitored over 90 min. (D) The AlphaFold-predicted complex of YqhR (FigP) is in purple and 5AF is in cyan. (E) 5AF and FigP depend on each other for stability in dormant spores. Representative immunoblots from lysates of dormant spores of the indicated strains. (Top) 5AF-His requires FigP for stability. (Bottom) FigP-His requires 5AF for stability. GerBC controlled for loading. (F) 5AF and FigP copurified from detergent-solubilized membrane preparations derived from exponentially growing B. subtilis cells engineered to express FigP-His and 5AF-ProC. (Top) Coomassie-stained gel of load (L) and elution (E) from anti-ProC resin. (Bottom) Immunoblots from the same purification probing for 5AF-ProC and FigP-His. (G) 5AF-GFP and FigP-mScarlet colocalize in dormant spores. Representative fluorescence images of spores with 5AF-GFP and FigP-mScarlet. Colocalizing germinosome foci are indicated by green and red arrowheads.We tested in-frame deletions of all five genes and found that, to varying degrees, all five mutants suppressed the premature germination phenotype and the defect in heat-resistant CFUs of the 5AF(V360A) mutant (Fig 3B; Supplemental Fig. S9B). Among the five mutants, the strongest suppressor was ΔyqhR, which largely restored wild-type percentages of phase-bright spores and heat-resistant CFUs to those of the constitutively active 5AF mutant. Analysis of spore germination of each mutant in a wild-type (5AF+) background revealed that ΔyqhR phenocopied Δ5AF mutant spores, while ΔymfJ, ΔyktB, and ΔymfD had no impact on spore germination, and ΔytpA had an intermediate phenotype (Fig. 3C; Supplemental Fig. S9C). Finally, only the Δ5AF ΔyqhR double mutant had a germination defect that resembled the single mutants (Fig. 3C; Supplemental Fig. S10). These data suggest that 5AF and yqhR reside in the same genetic pathway and could function as partner proteins. We therefore focused on YqhR, which we have renamed FigP (for 5AF-interacting germination protein).AlphaFold2-Multimer (Jumper et al. 2021; Evans et al. 2022) predicted a high-confidence interaction between 5AF and FigP, consistent with the idea that the two proteins form a complex (Fig. 3D; Supplemental Fig. S11). In addition, we found that the two proteins depend on each other for stability in dormant spores (Fig. 3E). The instability of 5AF in the absence of FigP provides an explanation for how figP mutants suppress the premature germination of 5AF(A360V). Further support for the idea that the two proteins interact comes from copurification of FigP-His with 5AF-ProteinC (5AF-ProC) from detergent-solubilized membrane preparations of B. subtilis cells expressing the two functional fusion proteins (Fig. 3F; Supplemental Fig. S4). Finally, a functional FigP-mScarlet fusion colocalized with 5AF-GFP in germinosome foci in dormant spores (Fig. 3G). Altogether, we conclude that FigP resides in a complex with 5AF and likely functions with 5AF to amplify the germination response.5AF/FigP assemble into an oligomeric complexIn the AlphaFold models, FigP binds to a site on 5AF that is analogous to GerAB's binding site on GerAA, though FigP is not homologous to GerAB (Supplemental Fig. S12). AlphaFold-Multimer predicted that the GerAA/GerAB/GerAC heterotrimer can oligomerize without clashes between protomers (Gao et al. 2023). Similarly, AlphaFold2-Multimer predicted that the 5AF/FigP dimer can assemble into a high-confidence oligomer of heterodimers without clashes between subunits (Fig. 4A; Supplemental Fig. S11). These data are consistent with the idea that FigP functions in complex with 5AF.Figure 4.5AF and FigP reside in an oligomeric complex that has properties of ion channels. (A) AlphaFold2-Multimer prediction of the 5AF/FigP oligomer. 5AF protomers are shown in cyan and light/dark gray. FigP protomers are in purple. (B) Expression of the channel-widening 5AF(L367S) is toxic to B. subtilis, provided FigP is coexpressed. Representative spot dilutions of the indicated strains on LB agar in the absence and presence of 50 µM IPTG. (C) Representative immunoblots from the indicated exponentially growing B. subtilis strain. 5AF-ProC is stable in the presence and absence of FigP during vegetive growth, unlike dormant spores. EzrA controlled for loading. (D) Representative growth curves of the indicated strains after the addition of IPTG at time 0. Both the 5AF(WT) and 5AF(L367S) alleles are under the control of the IPTG-regulated promoter Phyperspank, and FigP was expressed constitutively. (E) Representative fluorescence and phase-contrast images of the cells in D at the indicated times after IPTG addition using the potentiometric dye DiSC3(5). Membrane potential drops over time in cells expressing 5AF(L367S) and FigP. Cell lysis, as assayed by phase contrast, only occurs at later time points. A more complete time course, including staining with propidium iodide, is shown in Supplemental Figure S13. (F) Differentially tagged 5AF proteins copurify, provided FigP is present. Immunoblots of the load (L) and elution (E) of the indicated strains expressing 5AF-ProC, 5AF-FLAG, and FigP-His. Detergent-solubilized membrane proteins from exponentially growing B. subtilis cells were subjected to immunoaffinity purification using anti-ProC resin. EzrA served as a negative control. (G) Representative fluorescence images of exponentially growing B. subtilis cells expressing 5AF-mYpet in the presence and absence of FigP. Cells expressing GerAA-mYpet with GerAB and GerAC are shown for comparison. Discrete foci of 5AF-mYpet require FigP. The foci are of intensities similar to that of GerAA-mYpet in the presence of GerAB and GerAC. Scale bars, 5 µm. (H) Quantification of fluorescence intensities of GerAA-mYpet and 5AF-mYpet foci in the presence of their partner proteins. Three biological replicates (>500 cells for each) were quantified and are plotted in different colors. No significant (ns) difference in focus intensity was observed.Evidence that the GerA complex acts as an ion channel comes from experiments in which vegetative expression of constitutively active alleles of gerAA were toxic due to membrane depolarization, provided gerAB and gerAC were coexpressed (Gao et al. 2023). To investigate whether 5AF and FigP can similarly form a toxic membrane channel, we expressed wild-type 5AF and the channel-widening 5AF allele, 5AF(L367S), under an IPTG-regulated promoter in the presence and absence of FigP. 5AF(L367S) was toxic only if FigP was coexpressed (Fig. 4B). Importantly, unlike the situation in dormant spores, 5AF was stable in the absence of FigP in vegetatively growing cells (Fig. 4C). Thus, the lack of toxicity when 5AF(L367S) was expressed on its own argues that FigP is required for 5AF to assemble into a stable oligomer and/or form a functional channel.To investigate whether 5AF(L367S)/FigP toxicity was due to loss of membrane potential, we induced the expression of 5AF(WT)/FigP, 5AF(L367S), or 5AF(L367S)/FigP in rich medium and analyzed membrane potential over time using the potentiometric dye DiSC3(5). Eighty minutes after expression of 5AF(L367S)/FigP, the culture stopped increasing in optical density, whereas at later time points the optical density began to drop, consistent with lysis (Fig. 4D). In contrast, cells expressing 5AF(L367S) without FigP or 5AF(WT) with FigP continued to grow for the full time course. Importantly, membrane potential of cells expressing 5AF(L367S)/FigP gradually decreased over time, and the cells eventually lysed (Fig. 4E; Supplemental Fig. S13). Analysis of membrane permeability using propidium iodide indicated that the loss of membrane potential preceded the loss of membrane integrity (Supplemental Fig. S13). Cells expressing 5AF(WT)/FigP or 5AF(L367S) without FigP maintained membrane potential and membrane integrity throughout the induction time course (Fig. 4E; Supplemental Fig. S13). These experiments support the model that the 5AF/FigP complex forms an ion channel.5AF oligomerizes in vivoTo directly test whether 5AF oligomerizes in vivo, we first tested whether FLAG-tagged 5AF could be copurified with 5AF-ProC from vegetatively growing B. subtilis. As shown in Figure 4F, anti-ProC resin efficiently copurified 5AF-ProC and 5AF-FLAG from cells that coexpressed FigP. In the absence of FigP, 5AF-FLAG was not copurified with 5AF-ProC (Fig. 4F). These data indicate that at least two 5AF protomers reside in a complex that depends on FigP. In a second experiment, we analyzed 5AF-mYpet localization in exponentially growing B. subtilis. We previously showed that expression of GerAA-mYpet in vegetative cells forms fluorescent foci when GerAB and GerAC are coexpressed (Fig. 4G; Gao et al. 2023). Importantly, increasing GerAA-mYpet expression in the presence of GerAB and GerAC increased the number of foci but not their fluorescence intensity, suggesting that each focus represents a discrete oligomeric complex (Supplemental Fig. S14). Here, we coexpressed 5AF-mYpet in the presence and absence of FigP. 5AF-mYpet formed discrete foci, provided FigP was present, and the number of foci but not their intensity increased with increasing 5AF-mYpet expression (Fig. 4G; Supplemental Figs. S14, S15). Importantly, both qualitatively and quantitatively, the 5AF-mYpet/FigP foci resembled GerAA-mYpet/GerAB/GerAC foci (Fig. 4H), arguing that 5AF/FigP form discrete oligomeric complexes that are similar in stoichiometry to GerA complexes. Finally, to investigate whether 5AF forms an oligomeric complex in spores, we coexpressed wild-type 5AF and the channel-blocking mutant 5AF(V371L) in sporulating cells. The resulting spores displayed a delay in germination that was similar to Δ5AF and 5AF(V371L) mutant spores (Supplemental Fig. S16). These data indicate that 5AF(V371L) is a strong dominant-negative allele, consistent with this channel-blocking mutant poisoning wild-type 5AF protomers in an oligomeric complex. Collectively, these experiments provide strong support for the model that 5AF/FigP assembles into an oligomeric complex around a central channel.5AF/FigP enhances germination triggered by a nonnative GerA family receptor and DPA release by nonnative DPA transportersOur data support a model in which the 5AF/FigP complex is activated by ion release from GerA family receptors, and the release of ions by both complexes promotes DPA export by the 5A transporter. In this model, both the activation of 5AF/FigP by GerA receptors and the activation of the DPA transport complex do not involve protein–protein interactions and are instead mediated by chemical or physical changes in the spore. To further test this model, we took advantage of our previous findings that the GerA family receptor GerUV from Bacillus megaterium is functional in B. subtilis spores (Christie and Lowe 2007; Gao et al. 2023). Similarly, we previously showed that the DPA transport complexes from Clostridiodes difficile and Bacillus cereus can replace the native B. subtilis 5A locus. Importantly neither the C. difficile 5A operon nor the B. cereus 5A-2 locus encodes a 5AF homolog (Berendsen et al. 2016; Donnelly et al. 2016; Baloh and Sorg 2021). We analyzed spore germination rates in strains complemented by these nonnative loci in the presence and absence of 5AF. As shown in Figure 5, spores harboring gerUV(Bm) and lacking all GerA family receptors released DPA faster in response to glucose, leucine, proline, and K+ (GLPK) in the presence of 5AF than in its absence. Similarly, spores harboring 5A(Cd) or 5A-2(Bc) released DPA faster when 5AF was present than in its absence (Fig. 5B; Supplemental Fig. S17). These data are consistent with a model in which initial nutrient-stimulated ion release by GerA family channels triggers ion release by the 5AF/FigP complex. Faster ion release more efficiently activates DPA expulsion from the 5A transport complex (Fig. 6).Figure 5.5AF enhances germination of B. subtilis spores with a nonnative germination receptor or nonnative DPA transport complexes. (A) Purified B. subtilis spores lacking all five endogenous germinant receptor loci and harboring the gerUV(Bm) or gerA(Bs) locus were incubated with 10 mM glucose, leucine, proline, and K+ (GLPK), and DPA release was monitored over 90 min. Cells lacking 5AF germinated more slowly. (B) Purified spores lacking the native 5A operon and harboring an ectopic copy of the 5A locus from C. difficile (Cd) or the 5A2 locus from B. cereus (Bc) were incubated with 1 mM L-alanine, and DPA release was monitored over 90 min. Neither 5A2(Bc) nor 5A(Cd) encodes a 5AF homolog. B. subtilis 5AF was expressed in trans in the indicated strains.Figure 6.5AF/FigP amplifies the germination response by releasing ions. Schematic model of the germination pathway in B. subtilis. (Top panel) Nutrients are detected by the B subunits (cyan) of GerA family receptors. Ligand binding causes channel opening of the A subunits and ion release. Monovalent cations (blue pluses) are shown, but the identity of the cation (or anion) that triggers downstream steps is currently unknown. (Middle panel) Ion release by GerA family receptors induces channel opening of 5AF in the 5AF/FigP complex, resulting in ion release and amplification of the response. (Bottom panel) A drop in core ion concentration and/or a change in membrane potential causes “unplugging” of 5D (D) from the 5C/5Eb (C/Eb) channel and release of DPA. DPA activates the cortex-degrading enzyme CwlJ (not shown). A second cortex-degrading enzyme, SleB (also not shown), is activated by an unknown mechanism. Together, these enzymes degrade the cortex, enabling full hydration of the core and the onset of macromolecular synthesis.

5AF is predicted to oligomerize into a membrane channel and is required for ion release during germinationThe last gene in the B. subtilis 5A locus, 5AF, encodes a homolog of the channel-forming A subunits of GerA family receptors (Errington 1993; Perez-Valdespino et al. 2014). The AlphaFold-predicted structure of 5AF resembles the GerAA model, with a TM-align score of 0.69 and RMSD of 2.58 Å within the high-confidence regions of the model (Fig. 1A; Supplemental Fig. S1A). Furthermore, AlphaFold2-Multimer (Jumper et al. 2021; Evans et al. 2022) predicts that 5AF monomers assemble into a high-confidence pentamer surrounding a central channel, similar to the GerAA pentamer model (Fig. 1B; Supplemental Fig. S1B; Gao et al. 2023). AlphaFold2-Multimer also predicts that GerAA and, separately, 5AF can form high-confidence hexamers (Supplemental Fig. S1C; Kilian and Bischofs 2023). For clarity, we present the pentamer model throughout this article but are currently unable to experimentally distinguish between the two models.Figure 1.Spo5AF (5AF) is predicted to form an oligomeric membrane channel and is required for efficient ion release during germination. (A) Structural alignment by TM-align of the AlphaFold-predicted structures of GerAA (magenta) and 5AF (cyan). (B) AlphaFold-Multimer predictions of 5AF and GerAA pentamers viewed from outside the spore. Protomers are shown in dark and light gray and cyan (5AF) or magenta (GerAA). (C) Purified spores from wild type, Δ5AF, or Δ5AF complemented with 5AF in trans were mixed with 10 mM L-alanine, and optical density was analyzed over 90 min. (D,E) The germination exudates from the same spores used in C were collected at the indicated times and analyzed for DPA (D) and K+ (E). The percentage release at each time point, relative to the amount released by WT spores at 60 min, is shown. A complete analysis of ion release and biological replicates is shown in Supplemental Figure S3. (F) Purified WT and Δ5AF spores were incubated with and without 20 µM L-alanine without agitation for the indicated times in hours at 37°C. The percentage of DPA released relative to total is shown. Time points analyzed by phase-contrast microscopy are indicated (red arrows). (G) Representative phase-contrast images of WT and Δ5AF spores incubated with 20 µM L-alanine for the indicated times. (H) A functional 5AF-GFP fusion colocalizes with GerAA-mScarlet in dormant spores. Green and red arrowheads indicate colocalized germinosome foci. (I) GerD is required for 5AF-GFP foci in dormant spores. Scale bars, 2 µm.The structural similarity of 5AF to GerAA motivated us to perform a more detailed analysis of 5AF. Although the 5AF gene is in the spo5A locus, it is not required for DPA import into developing spores (Perez-Valdespino et al. 2014; Gao et al. 2022), and its role in germination is unclear. A deletion of 5AF and the gene adjacent to it, 5AEa, was previously reported to cause a delay in germination (Perez-Valdespino et al. 2014). Our analysis of in-frame deletions in 5AF and separately in 5AEa revealed that the loss of 5AF was solely responsible for the delayed germination, as assessed by DPA release and optical density (Fig. 1C,D; Supplemental Fig. S2A–D). Furthermore, normal germination kinetics were restored to the Δ5AF mutant when 5AF was expressed in trans.To determine whether 5AF is involved in ion release, we exposed wild-type and Δ5AF spores to 10 mM L-alanine and analyzed the germination exudates using inductively coupled plasma mass spectrometry (ICP-MS). As shown in Figure 1E, Δ5AF spores were delayed in ion release, and expression of 5AF in trans restored wild-type release rates (Fig. 1E; Supplemental Fig. S3). These findings are consistent with the idea that 5AF oligomers function as ion channels and release ions in response nutrients, thereby accelerating exit from dormancy. Finally, we note that the correlation between delayed release of ions and delayed release of DPA provides additional support for the prevailing model that ion release is the activating signal for DPA export (Gao et al. 2023).Spores lacking GerA are unable to germinate upon exposure to L-alanine (Moir and Cooper 2015; Setlow et al. 2017). However, ΔgerA spores germinate efficiently in the presence of asparagine, glucose, fructose, and K+ ions (AGFK), which signal through the homologous GerB and GerK nutrient receptors. Similarly, ΔgerA spores germinate and form colonies on LB agar plates at efficiencies similar to that of wild type, but spores lacking GerA, GerB, and GerK are reduced in colony formation by ∼500-fold (Supplemental Fig. S2E). To investigate whether 5AF responds to nutrients, we analyzed colony formation of Δ5AF spores and spores lacking 5AF and all GerA family loci (Δger5): gerA, gerB, gerK, and two additional ger loci (yndDEF and yfkQRT) of unknown function (Paidhungat and Setlow 2000). Spores lacking 5AF formed colonies at efficiencies similar to that of wild type, and spores lacking 5AF and all GerA family receptors formed colonies at efficiencies similar to that of the Δger5 gerA family deletion strain (Supplemental Fig. S2E). The absence of any further reduction in colony formation suggests that 5AF does not directly respond to nutrients and instead is activated by the GerA family receptors.Δ5AF spores exhibit a relatively modest delay in germination (Fig. 1C). We wondered whether our standard germination assays, in which we exposed spores to 1–10 mM L-alanine, were masking a more important role for this putative ion channel under conditions that more closely mimic those encountered in the environment. To explore this possibility, we incubated wild-type and Δ5AF spores with low concentrations of nutrients (20 µM L-alanine) without agitation and monitored germination by phase-contrast microscopy and DPA release over 24 h. Approximately 60% of the total DPA in wild-type spores was released over the 24-h time course compared with ∼10% in spores lacking 5AF (Fig. 1F). Phase-contrast microscopy revealed a similar reduction of phase-dark germinated spores in the Δ5AF mutant compared with wild type (Fig. 1G). These data argue that 5AF can significantly increase the number of spores that germinate over 24 h. Collectively, these findings support the idea that 5AF-dependent ion release amplifies GerA's response to L-alanine, which is even more critical under nutrient-poor conditions.GerA family receptors have been shown to colocalize in foci called germinosomes in dormant spores (Griffiths et al. 2011). To investigate whether 5AF is also present in germinosomes, we generated a functional GFP fusion (Supplemental Fig. S4) and analyzed its localization. The 5AF-GFP fusion formed discrete foci that colocalized with GerAA-mScarlet (Fig. 1H). Importantly, the 5AF-GFP foci were lost in spores lacking GerD, a lipoprotein required for germinosome formation (Fig. 1I; Mongkolthanaruk et al. 2009; Griffiths et al. 2011). Altogether, we conclude that 5AF proteins are predicted to assemble into oligomeric ion channels and colocalize with GerA family nutrient-gated ion channels and are required for efficient ion release and exit from dormancy in response to nutrients.

5AF is predicted to oligomerize into a membrane channel and is required for ion release during germination

The last gene in the B. subtilis 5A locus, 5AF, encodes a homolog of the channel-forming A subunits of GerA family receptors (Errington 1993; Perez-Valdespino et al. 2014). The AlphaFold-predicted structure of 5AF resembles the GerAA model, with a TM-align score of 0.69 and RMSD of 2.58 Å within the high-confidence regions of the model (Fig. 1A; Supplemental Fig. S1A). Furthermore, AlphaFold2-Multimer (Jumper et al. 2021; Evans et al. 2022) predicts that 5AF monomers assemble into a high-confidence pentamer surrounding a central channel, similar to the GerAA pentamer model (Fig. 1B; Supplemental Fig. S1B; Gao et al. 2023). AlphaFold2-Multimer also predicts that GerAA and, separately, 5AF can form high-confidence hexamers (Supplemental Fig. S1C; Kilian and Bischofs 2023). For clarity, we present the pentamer model throughout this article but are currently unable to experimentally distinguish between the two models.

Spo5AF (5AF) is predicted to form an oligomeric membrane channel and is required for efficient ion release during germination. (A) Structural alignment by TM-align of the AlphaFold-predicted structures of GerAA (magenta) and 5AF (cyan). (B) AlphaFold-Multimer predictions of 5AF and GerAA pentamers viewed from outside the spore. Protomers are shown in dark and light gray and cyan (5AF) or magenta (GerAA). (C) Purified spores from wild type, Δ5AF, or Δ5AF complemented with 5AF in trans were mixed with 10 mM L-alanine, and optical density was analyzed over 90 min. (D,E) The germination exudates from the same spores used in C were collected at the indicated times and analyzed for DPA (D) and K+ (E). The percentage release at each time point, relative to the amount released by WT spores at 60 min, is shown. A complete analysis of ion release and biological replicates is shown in Supplemental Figure S3. (F) Purified WT and Δ5AF spores were incubated with and without 20 µM L-alanine without agitation for the indicated times in hours at 37°C. The percentage of DPA released relative to total is shown. Time points analyzed by phase-contrast microscopy are indicated (red arrows). (G) Representative phase-contrast images of WT and Δ5AF spores incubated with 20 µM L-alanine for the indicated times. (H) A functional 5AF-GFP fusion colocalizes with GerAA-mScarlet in dormant spores. Green and red arrowheads indicate colocalized germinosome foci. (I) GerD is required for 5AF-GFP foci in dormant spores. Scale bars, 2 µm.

The structural similarity of 5AF to GerAA motivated us to perform a more detailed analysis of 5AF. Although the 5AF gene is in the spo5A locus, it is not required for DPA import into developing spores (Perez-Valdespino et al. 2014; Gao et al. 2022), and its role in germination is unclear. A deletion of 5AF and the gene adjacent to it, 5AEa, was previously reported to cause a delay in germination (Perez-Valdespino et al. 2014). Our analysis of in-frame deletions in 5AF and separately in 5AEa revealed that the loss of 5AF was solely responsible for the delayed germination, as assessed by DPA release and optical density (Fig. 1C,D; Supplemental Fig. S2A–D). Furthermore, normal germination kinetics were restored to the Δ5AF mutant when 5AF was expressed in trans.

To determine whether 5AF is involved in ion release, we exposed wild-type and Δ5AF spores to 10 mM L-alanine and analyzed the germination exudates using inductively coupled plasma mass spectrometry (ICP-MS). As shown in Figure 1E, Δ5AF spores were delayed in ion release, and expression of 5AF in trans restored wild-type release rates (Fig. 1E; Supplemental Fig. S3). These findings are consistent with the idea that 5AF oligomers function as ion channels and release ions in response nutrients, thereby accelerating exit from dormancy. Finally, we note that the correlation between delayed release of ions and delayed release of DPA provides additional support for the prevailing model that ion release is the activating signal for DPA export (Gao et al. 2023).

Spores lacking GerA are unable to germinate upon exposure to L-alanine (Moir and Cooper 2015; Setlow et al. 2017). However, ΔgerA spores germinate efficiently in the presence of asparagine, glucose, fructose, and K+ ions (AGFK), which signal through the homologous GerB and GerK nutrient receptors. Similarly, ΔgerA spores germinate and form colonies on LB agar plates at efficiencies similar to that of wild type, but spores lacking GerA, GerB, and GerK are reduced in colony formation by ∼500-fold (Supplemental Fig. S2E). To investigate whether 5AF responds to nutrients, we analyzed colony formation of Δ5AF spores and spores lacking 5AF and all GerA family loci (Δger5): gerA, gerB, gerK, and two additional ger loci (yndDEF and yfkQRT) of unknown function (Paidhungat and Setlow 2000). Spores lacking 5AF formed colonies at efficiencies similar to that of wild type, and spores lacking 5AF and all GerA family receptors formed colonies at efficiencies similar to that of the Δger5 gerA family deletion strain (Supplemental Fig. S2E). The absence of any further reduction in colony formation suggests that 5AF does not directly respond to nutrients and instead is activated by the GerA family receptors.

Δ5AF spores exhibit a relatively modest delay in germination (Fig. 1C). We wondered whether our standard germination assays, in which we exposed spores to 1–10 mM L-alanine, were masking a more important role for this putative ion channel under conditions that more closely mimic those encountered in the environment. To explore this possibility, we incubated wild-type and Δ5AF spores with low concentrations of nutrients (20 µM L-alanine) without agitation and monitored germination by phase-contrast microscopy and DPA release over 24 h. Approximately 60% of the total DPA in wild-type spores was released over the 24-h time course compared with ∼10% in spores lacking 5AF (Fig. 1F). Phase-contrast microscopy revealed a similar reduction of phase-dark germinated spores in the Δ5AF mutant compared with wild type (Fig. 1G). These data argue that 5AF can significantly increase the number of spores that germinate over 24 h. Collectively, these findings support the idea that 5AF-dependent ion release amplifies GerA's response to L-alanine, which is even more critical under nutrient-poor conditions.

GerA family receptors have been shown to colocalize in foci called germinosomes in dormant spores (Griffiths et al. 2011). To investigate whether 5AF is also present in germinosomes, we generated a functional GFP fusion (Supplemental Fig. S4) and analyzed its localization. The 5AF-GFP fusion formed discrete foci that colocalized with GerAA-mScarlet (Fig. 1H). Importantly, the 5AF-GFP foci were lost in spores lacking GerD, a lipoprotein required for germinosome formation (Fig. 1I; Mongkolthanaruk et al. 2009; Griffiths et al. 2011). Altogether, we conclude that 5AF proteins are predicted to assemble into oligomeric ion channels and colocalize with GerA family nutrient-gated ion channels and are required for efficient ion release and exit from dormancy in response to nutrients.

An unbiased screen for 5AF mutations that cause premature germinationIn a previous study, we identified constitutively active mutations in gerAA that caused developing spores to prematurely germinate. These alleles mapped to residues within or adjacent to the lumen of the predicted membrane channel of the GerAA oligomer, providing strong genetic support for the AlphaFold prediction (Gao et al. 2023). Accordingly, we performed an unbiased screen for constitutively active 5AF alleles (Supplemental Fig. S5). Briefly, the 5AF gene was PCR-mutagenized and transformed into a Δ5AF mutant, and the transformants were pooled and sporulated. We isolated prematurely germinated spores from denser dormant spores on a histodenz step gradient and inoculated them into rich medium that supports growth and efficient sporulation upon nutrient exhaustion. After a second round of enrichment, the prematurely germinated spores were plated onto sporulation agar plates, and the resulting colonies were sporulated for 96 h at 37°C. Spores from >200 individual colonies were analyzed by phase-contrast microscopy, and two mutants were identified that had high frequencies of premature germination: A360V and L367S (Fig. 2A; Supplemental Fig. S6A). Sporulated cultures of both mutants also displayed severe reductions in heat-resistant colony-forming units (CFUs) (Fig. 2A). The reduction in heat-resistant CFUs is particularly striking given that the Δ5AF mutant is not impaired in spore formation. Importantly, these mutant phenotypes did not require the GerA family receptors (Supplemental Fig. S6A). The L367S mutant mapped within the lumen of the channel model and is predicted to widen the pore. The A360V mutant mapped adjacent to the lumen and is predicted to stabilize an open conformation of the channel (Fig. 2B; Supplemental Fig. S7).Figure 2.Characterization of 5AF mutants predicted to widen or narrow the membrane channel. (A) Representative phase-contrast images of sporulated cultures of strains with 5AF(WT), Δ5AF, and the indicated constitutively active 5AF alleles. Heat-resistant CFUs of each strain compared with wild type are shown below the images. (B) Side and top views of the three transmembrane helices that surround the membrane channel in the 5AF pentamer model. Residues identified in the screen and those that are predicted to line the channel are indicated. (C) Representative phase-contrast images of sporulated cultures of strains with 5AF alleles predicted to narrow the channel. Percentages of heat-resistant CFUs are indicated below the images. (D) Purified spores from the indicated strains were mixed with 10 mM L-alanine, and the germination exudates were collected at the indicated times and analyzed for K+ and Ca2+. The percentage released at each time point, relative to the amount released by 5AF(WT) spores at 60 min, is shown. The channel-narrowing mutant 5AF(V371L) phenocopies Δ5FA. Similar results were obtained for all steps in the germination pathway and with 5AF(Q375L) and are shown in Supplemental Figure S8. (E) Purified spores of the indicated strains were germinated with 20 µM L-alanine without agitation, and the percentage of DPA released relative to total was monitored over time (in hours). Similar differences between WT and Δ5AF were observed at lower concentrations of L-alanine with agitation. (F) Immunoblots from lysates of the purified spores used in D. Functional ProteinC (ProC) fusions to 5AF(WT), 5AF(V371L), and 5AF(Q375L) are stable. σA (SigA) controlled for loading. YqhR-His is stable in the mutant spores. (G) Representative fluorescence images of functional GFP fusions to 5AF(WT) and 5AF(V371L) in dormant spores. Red arrowheads indicate germinosome foci.To further investigate the channel model, we substituted lumen-facing residues V371 and Q375 with leucines, which we predicted would narrow or occlude the channel (Fig. 2B). Both mutants formed phase-bright spores that were heat-resistant (Fig. 2C), similar to wild type and the Δ5AF mutant. However, when these spores were exposed to nutrients, they were delayed in ion release and all subsequent steps of germination (Fig. 2D; Supplemental Fig. S8). Similarly, when the mutant spores were exposed to low concentrations of L-alanine, they were impaired in exit from dormancy (Fig. 2E). Importantly, the mutant 5AF proteins were stable in dormant spores (Fig. 2F) and localized in germinosome foci (Fig. 2G), suggesting that the lumen-facing mutants assemble into complexes but are unable to amplify the GerA response to L-alanine. The discovery of germination-controlling mutations within the predicted membrane channel supports the AlphaFold model that 5AF oligomerizes into an ion channel.

An unbiased screen for 5AF mutations that cause premature germination

In a previous study, we identified constitutively active mutations in gerAA that caused developing spores to prematurely germinate. These alleles mapped to residues within or adjacent to the lumen of the predicted membrane channel of the GerAA oligomer, providing strong genetic support for the AlphaFold prediction (Gao et al. 2023). Accordingly, we performed an unbiased screen for constitutively active 5AF alleles (Supplemental Fig. S5). Briefly, the 5AF gene was PCR-mutagenized and transformed into a Δ5AF mutant, and the transformants were pooled and sporulated. We isolated prematurely germinated spores from denser dormant spores on a histodenz step gradient and inoculated them into rich medium that supports growth and efficient sporulation upon nutrient exhaustion. After a second round of enrichment, the prematurely germinated spores were plated onto sporulation agar plates, and the resulting colonies were sporulated for 96 h at 37°C. Spores from >200 individual colonies were analyzed by phase-contrast microscopy, and two mutants were identified that had high frequencies of premature germination: A360V and L367S (Fig. 2A; Supplemental Fig. S6A). Sporulated cultures of both mutants also displayed severe reductions in heat-resistant colony-forming units (CFUs) (Fig. 2A). The reduction in heat-resistant CFUs is particularly striking given that the Δ5AF mutant is not impaired in spore formation. Importantly, these mutant phenotypes did not require the GerA family receptors (Supplemental Fig. S6A). The L367S mutant mapped within the lumen of the channel model and is predicted to widen the pore. The A360V mutant mapped adjacent to the lumen and is predicted to stabilize an open conformation of the channel (Fig. 2B; Supplemental Fig. S7).

Characterization of 5AF mutants predicted to widen or narrow the membrane channel. (A) Representative phase-contrast images of sporulated cultures of strains with 5AF(WT), Δ5AF, and the indicated constitutively active 5AF alleles. Heat-resistant CFUs of each strain compared with wild type are shown below the images. (B) Side and top views of the three transmembrane helices that surround the membrane channel in the 5AF pentamer model. Residues identified in the screen and those that are predicted to line the channel are indicated. (C) Representative phase-contrast images of sporulated cultures of strains with 5AF alleles predicted to narrow the channel. Percentages of heat-resistant CFUs are indicated below the images. (D) Purified spores from the indicated strains were mixed with 10 mM L-alanine, and the germination exudates were collected at the indicated times and analyzed for K+ and Ca2+. The percentage released at each time point, relative to the amount released by 5AF(WT) spores at 60 min, is shown. The channel-narrowing mutant 5AF(V371L) phenocopies Δ5FA. Similar results were obtained for all steps in the germination pathway and with 5AF(Q375L) and are shown in Supplemental Figure S8. (E) Purified spores of the indicated strains were germinated with 20 µM L-alanine without agitation, and the percentage of DPA released relative to total was monitored over time (in hours). Similar differences between WT and Δ5AF were observed at lower concentrations of L-alanine with agitation. (F) Immunoblots from lysates of the purified spores used in D. Functional ProteinC (ProC) fusions to 5AF(WT), 5AF(V371L), and 5AF(Q375L) are stable. σA (SigA) controlled for loading. YqhR-His is stable in the mutant spores. (G) Representative fluorescence images of functional GFP fusions to 5AF(WT) and 5AF(V371L) in dormant spores. Red arrowheads indicate germinosome foci.

To further investigate the channel model, we substituted lumen-facing residues V371 and Q375 with leucines, which we predicted would narrow or occlude the channel (Fig. 2B). Both mutants formed phase-bright spores that were heat-resistant (Fig. 2C), similar to wild type and the Δ5AF mutant. However, when these spores were exposed to nutrients, they were delayed in ion release and all subsequent steps of germination (Fig. 2D; Supplemental Fig. S8). Similarly, when the mutant spores were exposed to low concentrations of L-alanine, they were impaired in exit from dormancy (Fig. 2E). Importantly, the mutant 5AF proteins were stable in dormant spores (Fig. 2F) and localized in germinosome foci (Fig. 2G), suggesting that the lumen-facing mutants assemble into complexes but are unable to amplify the GerA response to L-alanine. The discovery of germination-controlling mutations within the predicted membrane channel supports the AlphaFold model that 5AF oligomerizes into an ion channel.

Identification of YqhR (FigP) as a 5AF partner proteinGerAA resides in a complex with the nutrient-sensing subunit GerAB and the lipoprotein GerAC (Artzi et al. 2021; Amon et al. 2022; Gao et al. 2023). These three proteins depend on each other for stability in dormant spores and require each other for channel function in vegetatively growing cells (Cooper and Moir 2011; Mongkolthanaruk et al. 2011; Gao et al. 2023). We therefore wondered whether 5AF has one or more partner proteins. To search for potential interacting proteins, we screened for loss-of-function mutations that suppress the sporulation defect associated with the constitutive 5AF allele A360V. We transposon-mutagenized cells harboring 5AF(A360V) and then pooled and sporulated the transposon (Tn) library. The sporulated culture was subjected to 80°C for 20 min to kill vegetative cells that had failed to sporulate and spores that had prematurely germinated. The mixture was then plated on rich medium, allowing the heat-resistant spores to germinate and form colonies. The colonies were pooled and the Tn insertions were mapped by deep sequencing (Tn-seq). The screen identified five genes (yqhR, ymfJ, ymfD, ytpA, and yktB) that were enriched for Tn insertions compared with the input library (Fig. 3A; Supplemental Fig. S9A). The yqhR, ymfJ, and ymfD genes are expressed in the forespore under SigF or SigG control (Nicolas et al. 2012; Arrieta-Ortiz et al. 2015). yqhR encodes an uncharacterized membrane protein, ymfJ encodes a cytoplasmic protein with a DUF3243 domain, and ymfD encodes a putative transporter of the major facilitator superfamily. ytpA and yktB are expressed during vegetative growth and are not specifically induced during sporulation. ytpA encodes a putative phospholipase, and yktB encodes a cytoplasmic protein with a DUF1054 domain.Figure 3.Identification and characterization of FigP, an essential 5AF cofactor. (A) A transposon sequencing (Tn-seq) screen for mutations that suppress the premature germination of 5AF(A360V) identified yqhR (figP). Transposon insertion profiles from the yqhR region of the genome are shown. Tn libraries from wild type (WT) and 5AF(A360V) were sporulated, incubated for 20 min at 80°C, and plated on LB agar. The colonies were pooled and Tn sites were mapped by deep sequencing. Each line represents a Tn insertion site; the height reflects its abundance in the output library. The maximum number of reads was set at 10,000. Tn insertions in yqhR (figP) were overrepresented in 5AF(A360V) compared with WT. (B) Representative phase-contrast images of the indicated strains. ΔyqhR suppresses the premature germination phenotype of 5AF(A360V). The percentages of heat-resistant spores compared with wild type are indicated in the bottom right corners of the images. (C) Purified spores from the indicated strains were mixed with 1 mM L-alanine, and DPA release was monitored over 90 min. (D) The AlphaFold-predicted complex of YqhR (FigP) is in purple and 5AF is in cyan. (E) 5AF and FigP depend on each other for stability in dormant spores. Representative immunoblots from lysates of dormant spores of the indicated strains. (Top) 5AF-His requires FigP for stability. (Bottom) FigP-His requires 5AF for stability. GerBC controlled for loading. (F) 5AF and FigP copurified from detergent-solubilized membrane preparations derived from exponentially growing B. subtilis cells engineered to express FigP-His and 5AF-ProC. (Top) Coomassie-stained gel of load (L) and elution (E) from anti-ProC resin. (Bottom) Immunoblots from the same purification probing for 5AF-ProC and FigP-His. (G) 5AF-GFP and FigP-mScarlet colocalize in dormant spores. Representative fluorescence images of spores with 5AF-GFP and FigP-mScarlet. Colocalizing germinosome foci are indicated by green and red arrowheads.We tested in-frame deletions of all five genes and found that, to varying degrees, all five mutants suppressed the premature germination phenotype and the defect in heat-resistant CFUs of the 5AF(V360A) mutant (Fig 3B; Supplemental Fig. S9B). Among the five mutants, the strongest suppressor was ΔyqhR, which largely restored wild-type percentages of phase-bright spores and heat-resistant CFUs to those of the constitutively active 5AF mutant. Analysis of spore germination of each mutant in a wild-type (5AF+) background revealed that ΔyqhR phenocopied Δ5AF mutant spores, while ΔymfJ, ΔyktB, and ΔymfD had no impact on spore germination, and ΔytpA had an intermediate phenotype (Fig. 3C; Supplemental Fig. S9C). Finally, only the Δ5AF ΔyqhR double mutant had a germination defect that resembled the single mutants (Fig. 3C; Supplemental Fig. S10). These data suggest that 5AF and yqhR reside in the same genetic pathway and could function as partner proteins. We therefore focused on YqhR, which we have renamed FigP (for 5AF-interacting germination protein).AlphaFold2-Multimer (Jumper et al. 2021; Evans et al. 2022) predicted a high-confidence interaction between 5AF and FigP, consistent with the idea that the two proteins form a complex (Fig. 3D; Supplemental Fig. S11). In addition, we found that the two proteins depend on each other for stability in dormant spores (Fig. 3E). The instability of 5AF in the absence of FigP provides an explanation for how figP mutants suppress the premature germination of 5AF(A360V). Further support for the idea that the two proteins interact comes from copurification of FigP-His with 5AF-ProteinC (5AF-ProC) from detergent-solubilized membrane preparations of B. subtilis cells expressing the two functional fusion proteins (Fig. 3F; Supplemental Fig. S4). Finally, a functional FigP-mScarlet fusion colocalized with 5AF-GFP in germinosome foci in dormant spores (Fig. 3G). Altogether, we conclude that FigP resides in a complex with 5AF and likely functions with 5AF to amplify the germination response.

Identification of YqhR (FigP) as a 5AF partner protein

GerAA resides in a complex with the nutrient-sensing subunit GerAB and the lipoprotein GerAC (Artzi et al. 2021; Amon et al. 2022; Gao et al. 2023). These three proteins depend on each other for stability in dormant spores and require each other for channel function in vegetatively growing cells (Cooper and Moir 2011; Mongkolthanaruk et al. 2011; Gao et al. 2023). We therefore wondered whether 5AF has one or more partner proteins. To search for potential interacting proteins, we screened for loss-of-function mutations that suppress the sporulation defect associated with the constitutive 5AF allele A360V. We transposon-mutagenized cells harboring 5AF(A360V) and then pooled and sporulated the transposon (Tn) library. The sporulated culture was subjected to 80°C for 20 min to kill vegetative cells that had failed to sporulate and spores that had prematurely germinated. The mixture was then plated on rich medium, allowing the heat-resistant spores to germinate and form colonies. The colonies were pooled and the Tn insertions were mapped by deep sequencing (Tn-seq). The screen identified five genes (yqhR, ymfJ, ymfD, ytpA, and yktB) that were enriched for Tn insertions compared with the input library (Fig. 3A; Supplemental Fig. S9A). The yqhR, ymfJ, and ymfD genes are expressed in the forespore under SigF or SigG control (Nicolas et al. 2012; Arrieta-Ortiz et al. 2015). yqhR encodes an uncharacterized membrane protein, ymfJ encodes a cytoplasmic protein with a DUF3243 domain, and ymfD encodes a putative transporter of the major facilitator superfamily. ytpA and yktB are expressed during vegetative growth and are not specifically induced during sporulation. ytpA encodes a putative phospholipase, and yktB encodes a cytoplasmic protein with a DUF1054 domain.

Identification and characterization of FigP, an essential 5AF cofactor. (A) A transposon sequencing (Tn-seq) screen for mutations that suppress the premature germination of 5AF(A360V) identified yqhR (figP). Transposon insertion profiles from the yqhR region of the genome are shown. Tn libraries from wild type (WT) and 5AF(A360V) were sporulated, incubated for 20 min at 80°C, and plated on LB agar. The colonies were pooled and Tn sites were mapped by deep sequencing. Each line represents a Tn insertion site; the height reflects its abundance in the output library. The maximum number of reads was set at 10,000. Tn insertions in yqhR (figP) were overrepresented in 5AF(A360V) compared with WT. (B) Representative phase-contrast images of the indicated strains. ΔyqhR suppresses the premature germination phenotype of 5AF(A360V). The percentages of heat-resistant spores compared with wild type are indicated in the bottom right corners of the images. (C) Purified spores from the indicated strains were mixed with 1 mM L-alanine, and DPA release was monitored over 90 min. (D) The AlphaFold-predicted complex of YqhR (FigP) is in purple and 5AF is in cyan. (E) 5AF and FigP depend on each other for stability in dormant spores. Representative immunoblots from lysates of dormant spores of the indicated strains. (Top) 5AF-His requires FigP for stability. (Bottom) FigP-His requires 5AF for stability. GerBC controlled for loading. (F) 5AF and FigP copurified from detergent-solubilized membrane preparations derived from exponentially growing B. subtilis cells engineered to express FigP-His and 5AF-ProC. (Top) Coomassie-stained gel of load (L) and elution (E) from anti-ProC resin. (Bottom) Immunoblots from the same purification probing for 5AF-ProC and FigP-His. (G) 5AF-GFP and FigP-mScarlet colocalize in dormant spores. Representative fluorescence images of spores with 5AF-GFP and FigP-mScarlet. Colocalizing germinosome foci are indicated by green and red arrowheads.

We tested in-frame deletions of all five genes and found that, to varying degrees, all five mutants suppressed the premature germination phenotype and the defect in heat-resistant CFUs of the 5AF(V360A) mutant (Fig 3B; Supplemental Fig. S9B). Among the five mutants, the strongest suppressor was ΔyqhR, which largely restored wild-type percentages of phase-bright spores and heat-resistant CFUs to those of the constitutively active 5AF mutant. Analysis of spore germination of each mutant in a wild-type (5AF+) background revealed that ΔyqhR phenocopied Δ5AF mutant spores, while ΔymfJ, ΔyktB, and ΔymfD had no impact on spore germination, and ΔytpA had an intermediate phenotype (Fig. 3C; Supplemental Fig. S9C). Finally, only the Δ5AF ΔyqhR double mutant had a germination defect that resembled the single mutants (Fig. 3C; Supplemental Fig. S10). These data suggest that 5AF and yqhR reside in the same genetic pathway and could function as partner proteins. We therefore focused on YqhR, which we have renamed FigP (for 5AF-interacting germination protein).

AlphaFold2-Multimer (Jumper et al. 2021; Evans et al. 2022) predicted a high-confidence interaction between 5AF and FigP, consistent with the idea that the two proteins form a complex (Fig. 3D; Supplemental Fig. S11). In addition, we found that the two proteins depend on each other for stability in dormant spores (Fig. 3E). The instability of 5AF in the absence of FigP provides an explanation for how figP mutants suppress the premature germination of 5AF(A360V). Further support for the idea that the two proteins interact comes from copurification of FigP-His with 5AF-ProteinC (5AF-ProC) from detergent-solubilized membrane preparations of B. subtilis cells expressing the two functional fusion proteins (Fig. 3F; Supplemental Fig. S4). Finally, a functional FigP-mScarlet fusion colocalized with 5AF-GFP in germinosome foci in dormant spores (Fig. 3G). Altogether, we conclude that FigP resides in a complex with 5AF and likely functions with 5AF to amplify the germination response.

5AF/FigP assemble into an oligomeric complexIn the AlphaFold models, FigP binds to a site on 5AF that is analogous to GerAB's binding site on GerAA, though FigP is not homologous to GerAB (Supplemental Fig. S12). AlphaFold-Multimer predicted that the GerAA/GerAB/GerAC heterotrimer can oligomerize without clashes between protomers (Gao et al. 2023). Similarly, AlphaFold2-Multimer predicted that the 5AF/FigP dimer can assemble into a high-confidence oligomer of heterodimers without clashes between subunits (Fig. 4A; Supplemental Fig. S11). These data are consistent with the idea that FigP functions in complex with 5AF.Figure 4.5AF and FigP reside in an oligomeric complex that has properties of ion channels. (A) AlphaFold2-Multimer prediction of the 5AF/FigP oligomer. 5AF protomers are shown in cyan and light/dark gray. FigP protomers are in purple. (B) Expression of the channel-widening 5AF(L367S) is toxic to B. subtilis, provided FigP is coexpressed. Representative spot dilutions of the indicated strains on LB agar in the absence and presence of 50 µM IPTG. (C) Representative immunoblots from the indicated exponentially growing B. subtilis strain. 5AF-ProC is stable in the presence and absence of FigP during vegetive growth, unlike dormant spores. EzrA controlled for loading. (D) Representative growth curves of the indicated strains after the addition of IPTG at time 0. Both the 5AF(WT) and 5AF(L367S) alleles are under the control of the IPTG-regulated promoter Phyperspank, and FigP was expressed constitutively. (E) Representative fluorescence and phase-contrast images of the cells in D at the indicated times after IPTG addition using the potentiometric dye DiSC3(5). Membrane potential drops over time in cells expressing 5AF(L367S) and FigP. Cell lysis, as assayed by phase contrast, only occurs at later time points. A more complete time course, including staining with propidium iodide, is shown in Supplemental Figure S13. (F) Differentially tagged 5AF proteins copurify, provided FigP is present. Immunoblots of the load (L) and elution (E) of the indicated strains expressing 5AF-ProC, 5AF-FLAG, and FigP-His. Detergent-solubilized membrane proteins from exponentially growing B. subtilis cells were subjected to immunoaffinity purification using anti-ProC resin. EzrA served as a negative control. (G) Representative fluorescence images of exponentially growing B. subtilis cells expressing 5AF-mYpet in the presence and absence of FigP. Cells expressing GerAA-mYpet with GerAB and GerAC are shown for comparison. Discrete foci of 5AF-mYpet require FigP. The foci are of intensities similar to that of GerAA-mYpet in the presence of GerAB and GerAC. Scale bars, 5 µm. (H) Quantification of fluorescence intensities of GerAA-mYpet and 5AF-mYpet foci in the presence of their partner proteins. Three biological replicates (>500 cells for each) were quantified and are plotted in different colors. No significant (ns) difference in focus intensity was observed.Evidence that the GerA complex acts as an ion channel comes from experiments in which vegetative expression of constitutively active alleles of gerAA were toxic due to membrane depolarization, provided gerAB and gerAC were coexpressed (Gao et al. 2023). To investigate whether 5AF and FigP can similarly form a toxic membrane channel, we expressed wild-type 5AF and the channel-widening 5AF allele, 5AF(L367S), under an IPTG-regulated promoter in the presence and absence of FigP. 5AF(L367S) was toxic only if FigP was coexpressed (Fig. 4B). Importantly, unlike the situation in dormant spores, 5AF was stable in the absence of FigP in vegetatively growing cells (Fig. 4C). Thus, the lack of toxicity when 5AF(L367S) was expressed on its own argues that FigP is required for 5AF to assemble into a stable oligomer and/or form a functional channel.To investigate whether 5AF(L367S)/FigP toxicity was due to loss of membrane potential, we induced the expression of 5AF(WT)/FigP, 5AF(L367S), or 5AF(L367S)/FigP in rich medium and analyzed membrane potential over time using the potentiometric dye DiSC3(5). Eighty minutes after expression of 5AF(L367S)/FigP, the culture stopped increasing in optical density, whereas at later time points the optical density began to drop, consistent with lysis (Fig. 4D). In contrast, cells expressing 5AF(L367S) without FigP or 5AF(WT) with FigP continued to grow for the full time course. Importantly, membrane potential of cells expressing 5AF(L367S)/FigP gradually decreased over time, and the cells eventually lysed (Fig. 4E; Supplemental Fig. S13). Analysis of membrane permeability using propidium iodide indicated that the loss of membrane potential preceded the loss of membrane integrity (Supplemental Fig. S13). Cells expressing 5AF(WT)/FigP or 5AF(L367S) without FigP maintained membrane potential and membrane integrity throughout the induction time course (Fig. 4E; Supplemental Fig. S13). These experiments support the model that the 5AF/FigP complex forms an ion channel.

5AF/FigP assemble into an oligomeric complex

In the AlphaFold models, FigP binds to a site on 5AF that is analogous to GerAB's binding site on GerAA, though FigP is not homologous to GerAB (Supplemental Fig. S12). AlphaFold-Multimer predicted that the GerAA/GerAB/GerAC heterotrimer can oligomerize without clashes between protomers (Gao et al. 2023). Similarly, AlphaFold2-Multimer predicted that the 5AF/FigP dimer can assemble into a high-confidence oligomer of heterodimers without clashes between subunits (Fig. 4A; Supplemental Fig. S11). These data are consistent with the idea that FigP functions in complex with 5AF.

5AF and FigP reside in an oligomeric complex that has properties of ion channels. (A) AlphaFold2-Multimer prediction of the 5AF/FigP oligomer. 5AF protomers are shown in cyan and light/dark gray. FigP protomers are in purple. (B) Expression of the channel-widening 5AF(L367S) is toxic to B. subtilis, provided FigP is coexpressed. Representative spot dilutions of the indicated strains on LB agar in the absence and presence of 50 µM IPTG. (C) Representative immunoblots from the indicated exponentially growing B. subtilis strain. 5AF-ProC is stable in the presence and absence of FigP during vegetive growth, unlike dormant spores. EzrA controlled for loading. (D) Representative growth curves of the indicated strains after the addition of IPTG at time 0. Both the 5AF(WT) and 5AF(L367S) alleles are under the control of the IPTG-regulated promoter Phyperspank, and FigP was expressed constitutively. (E) Representative fluorescence and phase-contrast images of the cells in D at the indicated times after IPTG addition using the potentiometric dye DiSC3(5). Membrane potential drops over time in cells expressing 5AF(L367S) and FigP. Cell lysis, as assayed by phase contrast, only occurs at later time points. A more complete time course, including staining with propidium iodide, is shown in Supplemental Figure S13. (F) Differentially tagged 5AF proteins copurify, provided FigP is present. Immunoblots of the load (L) and elution (E) of the indicated strains expressing 5AF-ProC, 5AF-FLAG, and FigP-His. Detergent-solubilized membrane proteins from exponentially growing B. subtilis cells were subjected to immunoaffinity purification using anti-ProC resin. EzrA served as a negative control. (G) Representative fluorescence images of exponentially growing B. subtilis cells expressing 5AF-mYpet in the presence and absence of FigP. Cells expressing GerAA-mYpet with GerAB and GerAC are shown for comparison. Discrete foci of 5AF-mYpet require FigP. The foci are of intensities similar to that of GerAA-mYpet in the presence of GerAB and GerAC. Scale bars, 5 µm. (H) Quantification of fluorescence intensities of GerAA-mYpet and 5AF-mYpet foci in the presence of their partner proteins. Three biological replicates (>500 cells for each) were quantified and are plotted in different colors. No significant (ns) difference in focus intensity was observed.

Evidence that the GerA complex acts as an ion channel comes from experiments in which vegetative expression of constitutively active alleles of gerAA were toxic due to membrane depolarization, provided gerAB and gerAC were coexpressed (Gao et al. 2023). To investigate whether 5AF and FigP can similarly form a toxic membrane channel, we expressed wild-type 5AF and the channel-widening 5AF allele, 5AF(L367S), under an IPTG-regulated promoter in the presence and absence of FigP. 5AF(L367S) was toxic only if FigP was coexpressed (Fig. 4B). Importantly, unlike the situation in dormant spores, 5AF was stable in the absence of FigP in vegetatively growing cells (Fig. 4C). Thus, the lack of toxicity when 5AF(L367S) was expressed on its own argues that FigP is required for 5AF to assemble into a stable oligomer and/or form a functional channel.

To investigate whether 5AF(L367S)/FigP toxicity was due to loss of membrane potential, we induced the expression of 5AF(WT)/FigP, 5AF(L367S), or 5AF(L367S)/FigP in rich medium and analyzed membrane potential over time using the potentiometric dye DiSC3(5). Eighty minutes after expression of 5AF(L367S)/FigP, the culture stopped increasing in optical density, whereas at later time points the optical density began to drop, consistent with lysis (Fig. 4D). In contrast, cells expressing 5AF(L367S) without FigP or 5AF(WT) with FigP continued to grow for the full time course. Importantly, membrane potential of cells expressing 5AF(L367S)/FigP gradually decreased over time, and the cells eventually lysed (Fig. 4E; Supplemental Fig. S13). Analysis of membrane permeability using propidium iodide indicated that the loss of membrane potential preceded the loss of membrane integrity (Supplemental Fig. S13). Cells expressing 5AF(WT)/FigP or 5AF(L367S) without FigP maintained membrane potential and membrane integrity throughout the induction time course (Fig. 4E; Supplemental Fig. S13). These experiments support the model that the 5AF/FigP complex forms an ion channel.

5AF oligomerizes in vivoTo directly test whether 5AF oligomerizes in vivo, we first tested whether FLAG-tagged 5AF could be copurified with 5AF-ProC from vegetatively growing B. subtilis. As shown in Figure 4F, anti-ProC resin efficiently copurified 5AF-ProC and 5AF-FLAG from cells that coexpressed FigP. In the absence of FigP, 5AF-FLAG was not copurified with 5AF-ProC (Fig. 4F). These data indicate that at least two 5AF protomers reside in a complex that depends on FigP. In a second experiment, we analyzed 5AF-mYpet localization in exponentially growing B. subtilis. We previously showed that expression of GerAA-mYpet in vegetative cells forms fluorescent foci when GerAB and GerAC are coexpressed (Fig. 4G; Gao et al. 2023). Importantly, increasing GerAA-mYpet expression in the presence of GerAB and GerAC increased the number of foci but not their fluorescence intensity, suggesting that each focus represents a discrete oligomeric complex (Supplemental Fig. S14). Here, we coexpressed 5AF-mYpet in the presence and absence of FigP. 5AF-mYpet formed discrete foci, provided FigP was present, and the number of foci but not their intensity increased with increasing 5AF-mYpet expression (Fig. 4G; Supplemental Figs. S14, S15). Importantly, both qualitatively and quantitatively, the 5AF-mYpet/FigP foci resembled GerAA-mYpet/GerAB/GerAC foci (Fig. 4H), arguing that 5AF/FigP form discrete oligomeric complexes that are similar in stoichiometry to GerA complexes. Finally, to investigate whether 5AF forms an oligomeric complex in spores, we coexpressed wild-type 5AF and the channel-blocking mutant 5AF(V371L) in sporulating cells. The resulting spores displayed a delay in germination that was similar to Δ5AF and 5AF(V371L) mutant spores (Supplemental Fig. S16). These data indicate that 5AF(V371L) is a strong dominant-negative allele, consistent with this channel-blocking mutant poisoning wild-type 5AF protomers in an oligomeric complex. Collectively, these experiments provide strong support for the model that 5AF/FigP assembles into an oligomeric complex around a central channel.

5AF oligomerizes in vivo

To directly test whether 5AF oligomerizes in vivo, we first tested whether FLAG-tagged 5AF could be copurified with 5AF-ProC from vegetatively growing B. subtilis. As shown in Figure 4F, anti-ProC resin efficiently copurified 5AF-ProC and 5AF-FLAG from cells that coexpressed FigP. In the absence of FigP, 5AF-FLAG was not copurified with 5AF-ProC (Fig. 4F). These data indicate that at least two 5AF protomers reside in a complex that depends on FigP. In a second experiment, we analyzed 5AF-mYpet localization in exponentially growing B. subtilis. We previously showed that expression of GerAA-mYpet in vegetative cells forms fluorescent foci when GerAB and GerAC are coexpressed (Fig. 4G; Gao et al. 2023). Importantly, increasing GerAA-mYpet expression in the presence of GerAB and GerAC increased the number of foci but not their fluorescence intensity, suggesting that each focus represents a discrete oligomeric complex (Supplemental Fig. S14). Here, we coexpressed 5AF-mYpet in the presence and absence of FigP. 5AF-mYpet formed discrete foci, provided FigP was present, and the number of foci but not their intensity increased with increasing 5AF-mYpet expression (Fig. 4G; Supplemental Figs. S14, S15). Importantly, both qualitatively and quantitatively, the 5AF-mYpet/FigP foci resembled GerAA-mYpet/GerAB/GerAC foci (Fig. 4H), arguing that 5AF/FigP form discrete oligomeric complexes that are similar in stoichiometry to GerA complexes. Finally, to investigate whether 5AF forms an oligomeric complex in spores, we coexpressed wild-type 5AF and the channel-blocking mutant 5AF(V371L) in sporulating cells. The resulting spores displayed a delay in germination that was similar to Δ5AF and 5AF(V371L) mutant spores (Supplemental Fig. S16). These data indicate that 5AF(V371L) is a strong dominant-negative allele, consistent with this channel-blocking mutant poisoning wild-type 5AF protomers in an oligomeric complex. Collectively, these experiments provide strong support for the model that 5AF/FigP assembles into an oligomeric complex around a central channel.

5AF/FigP enhances germination triggered by a nonnative GerA family receptor and DPA release by nonnative DPA transportersOur data support a model in which the 5AF/FigP complex is activated by ion release from GerA family receptors, and the release of ions by both complexes promotes DPA export by the 5A transporter. In this model, both the activation of 5AF/FigP by GerA receptors and the activation of the DPA transport complex do not involve protein–protein interactions and are instead mediated by chemical or physical changes in the spore. To further test this model, we took advantage of our previous findings that the GerA family receptor GerUV from Bacillus megaterium is functional in B. subtilis spores (Christie and Lowe 2007; Gao et al. 2023). Similarly, we previously showed that the DPA transport complexes from Clostridiodes difficile and Bacillus cereus can replace the native B. subtilis 5A locus. Importantly neither the C. difficile 5A operon nor the B. cereus 5A-2 locus encodes a 5AF homolog (Berendsen et al. 2016; Donnelly et al. 2016; Baloh and Sorg 2021). We analyzed spore germination rates in strains complemented by these nonnative loci in the presence and absence of 5AF. As shown in Figure 5, spores harboring gerUV(Bm) and lacking all GerA family receptors released DPA faster in response to glucose, leucine, proline, and K+ (GLPK) in the presence of 5AF than in its absence. Similarly, spores harboring 5A(Cd) or 5A-2(Bc) released DPA faster when 5AF was present than in its absence (Fig. 5B; Supplemental Fig. S17). These data are consistent with a model in which initial nutrient-stimulated ion release by GerA family channels triggers ion release by the 5AF/FigP complex. Faster ion release more efficiently activates DPA expulsion from the 5A transport complex (Fig. 6).Figure 5.5AF enhances germination of B. subtilis spores with a nonnative germination receptor or nonnative DPA transport complexes. (A) Purified B. subtilis spores lacking all five endogenous germinant receptor loci and harboring the gerUV(Bm) or gerA(Bs) locus were incubated with 10 mM glucose, leucine, proline, and K+ (GLPK), and DPA release was monitored over 90 min. Cells lacking 5AF germinated more slowly. (B) Purified spores lacking the native 5A operon and harboring an ectopic copy of the 5A locus from C. difficile (Cd) or the 5A2 locus from B. cereus (Bc) were incubated with 1 mM L-alanine, and DPA release was monitored over 90 min. Neither 5A2(Bc) nor 5A(Cd) encodes a 5AF homolog. B. subtilis 5AF was expressed in trans in the indicated strains.Figure 6.5AF/FigP amplifies the germination response by releasing ions. Schematic model of the germination pathway in B. subtilis. (Top panel) Nutrients are detected by the B subunits (cyan) of GerA family receptors. Ligand binding causes channel opening of the A subunits and ion release. Monovalent cations (blue pluses) are shown, but the identity of the cation (or anion) that triggers downstream steps is currently unknown. (Middle panel) Ion release by GerA family receptors induces channel opening of 5AF in the 5AF/FigP complex, resulting in ion release and amplification of the response. (Bottom panel) A drop in core ion concentration and/or a change in membrane potential causes “unplugging” of 5D (D) from the 5C/5Eb (C/Eb) channel and release of DPA. DPA activates the cortex-degrading enzyme CwlJ (not shown). A second cortex-degrading enzyme, SleB (also not shown), is activated by an unknown mechanism. Together, these enzymes degrade the cortex, enabling full hydration of the core and the onset of macromolecular synthesis.

5AF/FigP enhances germination triggered by a nonnative GerA family receptor and DPA release by nonnative DPA transporters

Our data support a model in which the 5AF/FigP complex is activated by ion release from GerA family receptors, and the release of ions by both complexes promotes DPA export by the 5A transporter. In this model, both the activation of 5AF/FigP by GerA receptors and the activation of the DPA transport complex do not involve protein–protein interactions and are instead mediated by chemical or physical changes in the spore. To further test this model, we took advantage of our previous findings that the GerA family receptor GerUV from Bacillus megaterium is functional in B. subtilis spores (Christie and Lowe 2007; Gao et al. 2023). Similarly, we previously showed that the DPA transport complexes from Clostridiodes difficile and Bacillus cereus can replace the native B. subtilis 5A locus. Importantly neither the C. difficile 5A operon nor the B. cereus 5A-2 locus encodes a 5AF homolog (Berendsen et al. 2016; Donnelly et al. 2016; Baloh and Sorg 2021). We analyzed spore germination rates in strains complemented by these nonnative loci in the presence and absence of 5AF. As shown in Figure 5, spores harboring gerUV(Bm) and lacking all GerA family receptors released DPA faster in response to glucose, leucine, proline, and K+ (GLPK) in the presence of 5AF than in its absence. Similarly, spores harboring 5A(Cd) or 5A-2(Bc) released DPA faster when 5AF was present than in its absence (Fig. 5B; Supplemental Fig. S17). These data are consistent with a model in which initial nutrient-stimulated ion release by GerA family channels triggers ion release by the 5AF/FigP complex. Faster ion release more efficiently activates DPA expulsion from the 5A transport complex (Fig. 6).

5AF enhances germination of B. subtilis spores with a nonnative germination receptor or nonnative DPA transport complexes. (A) Purified B. subtilis spores lacking all five endogenous germinant receptor loci and harboring the gerUV(Bm) or gerA(Bs) locus were incubated with 10 mM glucose, leucine, proline, and K+ (GLPK), and DPA release was monitored over 90 min. Cells lacking 5AF germinated more slowly. (B) Purified spores lacking the native 5A operon and harboring an ectopic copy of the 5A locus from C. difficile (Cd) or the 5A2 locus from B. cereus (Bc) were incubated with 1 mM L-alanine, and DPA release was monitored over 90 min. Neither 5A2(Bc) nor 5A(Cd) encodes a 5AF homolog. B. subtilis 5AF was expressed in trans in the indicated strains.

5AF/FigP amplifies the germination response by releasing ions. Schematic model of the germination pathway in B. subtilis. (Top panel) Nutrients are detected by the B subunits (cyan) of GerA family receptors. Ligand binding causes channel opening of the A subunits and ion release. Monovalent cations (blue pluses) are shown, but the identity of the cation (or anion) that triggers downstream steps is currently unknown. (Middle panel) Ion release by GerA family receptors induces channel opening of 5AF in the 5AF/FigP complex, resulting in ion release and amplification of the response. (Bottom panel) A drop in core ion concentration and/or a change in membrane potential causes “unplugging” of 5D (D) from the 5C/5Eb (C/Eb) channel and release of DPA. DPA activates the cortex-degrading enzyme CwlJ (not shown). A second cortex-degrading enzyme, SleB (also not shown), is activated by an unknown mechanism. Together, these enzymes degrade the cortex, enabling full hydration of the core and the onset of macromolecular synthesis.

DiscussionSpores can remain dormant for decades yet constantly survey their environment for nutrients and the opportunity to germinate and resume growth. Here, we have shown that the 5AF/FigP complex functions to amplify the germination response when nutrients are detected. When spores are exposed to high concentrations of germinants, this complex plays a relatively minor role, accelerating germination by 10–20 min. However, in the presence of low concentrations of nutrients, 5AF/FigP can significantly increase the number of spores that germinate over 24 h. Our data suggest that 5AF/FigP enhances nutrient detection by releasing ions in response to ion release by GerA family receptors (Fig. 6). Faster release of ions by these related ion channels more efficiently triggers DPA export by the 5A complex, leading to activation of cortex lytic enzymes. The 5AF/FigP signal amplifier is not essential for germination and is not as broadly conserved as GerA family receptors or the core components of the DPA transport complex encoded in the 5A operon. However, FigP homologs are present in most Bacilli and co-occur with the poorly characterized 5AA and 5AB proteins encoded in the 5A locus (Supplemental Fig. S18). Our data indicate that 5AF/FigP can accelerate germination of spores harboring the B. megaterium GerUV receptor and, separately, spores harboring C. difficile or B. cereus 5AC/5AD/5AEb. We therefore suspect that 5AF/FigP complexes both sense ion release and use ion release to activate DPA export in all spore formers that encode them.The mechanism by which 5AF/FigP complexes are activated by GerA-type receptors remains to be elucidated. One possible model relates to the colocalization of these complexes in spores. Previous studies have shown that ΔgerD spores that lack clustered nutrient receptors germinate more slowly (Li et al. 2014). This finding suggests that receptor clustering could amplify nutrient detection by conformational spreading within the oligomeric array, increasing ligand affinity, as has been proposed for chemotaxis receptors (Rao et al. 2004; Sourjik and Berg 2004; Briegel et al. 2014). In the context of this model, 5AF/FigP could be activated by conformational spreading within the germinosome. However, we found that ΔgerD mutant spores are further impaired in germination if they also lack 5AF (Supplemental Fig. S19), suggesting that 5AF can be activated by GerA receptors without clustering with them. We favor an alternative model in which the 5AF/FigP oligomer functions as a voltage-gated ion channel (Bezanilla 2008, 2018) that is activated when an initial GerA-catalyzed ion release changes the membrane voltage.It is noteworthy that the AlphaFold-predicted FigP model is structurally homologous to INSIG1 (insulin-induced gene 1), which is involved in cholesterol homeostasis in mammals (Yang et al. 2002). Mammalian INSIG1 has been shown to bind 5-hydroxycholesterol and function with sterol response element binding protein (SREBP) cleavage-activating protein (SCAP) to control SREBP-mediated transcription (Hua et al. 1996). The predicted FigP structure is most similar to the mycobacterial INSIG1 homolog MvINS from Mycobacterium vanbaalenii (Supplemental Fig. S20), which has not been characterized in vivo (Ren et al. 2015). Intriguingly, a crystal structure of MvINS was determined with diacylglycerol bound in its conserved membrane pocket, raising the possibility that FigP also binds a lipid (Supplemental Fig. S20). Amino acid substitutions in either of two tyrosines in FigP that are predicted to face the conserved pocket partially impaired FigP function, slowing the rate of germination (Supplemental Fig. S20). Both mutant proteins were stable in spores and stabilized 5AF (Supplemental Fig. S20). These results are consistent with the idea that FigP binds a lipid and argue that FigP has a function beyond simply stabilizing the 5AF oligomer. It is possible that lipid binding by FigP could function as the trigger for channel opening. If this model is correct, the activating lipids would have to be kept away from the 5AF/FigP oligomers during spore formation and dormancy and would only access the complexes after the onset of germination, perhaps due to changes in membrane fluidity (Cowan et al. 2003; Flores et al. 2023). Alternatively, we speculate that changes in membrane voltage due to ion release by GerA family receptors are transduced through the lipid–FigP interaction. The role of lipid–FigP binding and how ion release activates 5AF/FigP and in turn triggers DPA release by the 5A complex remain important and outstanding questions for future investigation.

Spores can remain dormant for decades yet constantly survey their environment for nutrients and the opportunity to germinate and resume growth. Here, we have shown that the 5AF/FigP complex functions to amplify the germination response when nutrients are detected. When spores are exposed to high concentrations of germinants, this complex plays a relatively minor role, accelerating germination by 10–20 min. However, in the presence of low concentrations of nutrients, 5AF/FigP can significantly increase the number of spores that germinate over 24 h. Our data suggest that 5AF/FigP enhances nutrient detection by releasing ions in response to ion release by GerA family receptors (Fig. 6). Faster release of ions by these related ion channels more efficiently triggers DPA export by the 5A complex, leading to activation of cortex lytic enzymes. The 5AF/FigP signal amplifier is not essential for germination and is not as broadly conserved as GerA family receptors or the core components of the DPA transport complex encoded in the 5A operon. However, FigP homologs are present in most Bacilli and co-occur with the poorly characterized 5AA and 5AB proteins encoded in the 5A locus (Supplemental Fig. S18). Our data indicate that 5AF/FigP can accelerate germination of spores harboring the B. megaterium GerUV receptor and, separately, spores harboring C. difficile or B. cereus 5AC/5AD/5AEb. We therefore suspect that 5AF/FigP complexes both sense ion release and use ion release to activate DPA export in all spore formers that encode them.

The mechanism by which 5AF/FigP complexes are activated by GerA-type receptors remains to be elucidated. One possible model relates to the colocalization of these complexes in spores. Previous studies have shown that ΔgerD spores that lack clustered nutrient receptors germinate more slowly (Li et al. 2014). This finding suggests that receptor clustering could amplify nutrient detection by conformational spreading within the oligomeric array, increasing ligand affinity, as has been proposed for chemotaxis receptors (Rao et al. 2004; Sourjik and Berg 2004; Briegel et al. 2014). In the context of this model, 5AF/FigP could be activated by conformational spreading within the germinosome. However, we found that ΔgerD mutant spores are further impaired in germination if they also lack 5AF (Supplemental Fig. S19), suggesting that 5AF can be activated by GerA receptors without clustering with them. We favor an alternative model in which the 5AF/FigP oligomer functions as a voltage-gated ion channel (Bezanilla 2008, 2018) that is activated when an initial GerA-catalyzed ion release changes the membrane voltage.

It is noteworthy that the AlphaFold-predicted FigP model is structurally homologous to INSIG1 (insulin-induced gene 1), which is involved in cholesterol homeostasis in mammals (Yang et al. 2002). Mammalian INSIG1 has been shown to bind 5-hydroxycholesterol and function with sterol response element binding protein (SREBP) cleavage-activating protein (SCAP) to control SREBP-mediated transcription (Hua et al. 1996). The predicted FigP structure is most similar to the mycobacterial INSIG1 homolog MvINS from Mycobacterium vanbaalenii (Supplemental Fig. S20), which has not been characterized in vivo (Ren et al. 2015). Intriguingly, a crystal structure of MvINS was determined with diacylglycerol bound in its conserved membrane pocket, raising the possibility that FigP also binds a lipid (Supplemental Fig. S20). Amino acid substitutions in either of two tyrosines in FigP that are predicted to face the conserved pocket partially impaired FigP function, slowing the rate of germination (Supplemental Fig. S20). Both mutant proteins were stable in spores and stabilized 5AF (Supplemental Fig. S20). These results are consistent with the idea that FigP binds a lipid and argue that FigP has a function beyond simply stabilizing the 5AF oligomer. It is possible that lipid binding by FigP could function as the trigger for channel opening. If this model is correct, the activating lipids would have to be kept away from the 5AF/FigP oligomers during spore formation and dormancy and would only access the complexes after the onset of germination, perhaps due to changes in membrane fluidity (Cowan et al. 2003; Flores et al. 2023). Alternatively, we speculate that changes in membrane voltage due to ion release by GerA family receptors are transduced through the lipid–FigP interaction. The role of lipid–FigP binding and how ion release activates 5AF/FigP and in turn triggers DPA release by the 5A complex remain important and outstanding questions for future investigation.

Materials and methodsGeneral methodsAll strains used in this study were derived from Bacillus subtilis 168 (Zeigler et al. 2008). Sporulation was induced by nutrient exhaustion in complete Difco sporulation medium (DSM) for 30 h at 37°C (Schaeffer et al. 1965), and sporulation efficiency was determined by comparing heat-resistant (for 20 min at 80°C) colony-forming units (CFUs) of mutants with wild-type B. subtilis 168. All in-frame deletion mutants were derived from Bacillus knockout collection (BKE) (Koo et al. 2017) or constructed by direct transformation of isothermal assembly products into competent B. subtilis. Antibiotic cassettes were excised using a temperature-sensitive plasmid that constitutively expresses Cre recombinase (Meeske et al. 2015). Site-directed mutants were generated using a modified QuickChange protocol. Strains, plasmids, and oligonucleotide primers used in this study are in Supplemental Tables S1–S3. Experiments presented in the text and supplemental figures were from one of at least three biological replicates, with the exception of Figures 2F, 3E, and 4C and Supplemental Figures S16A, S18E, and S19, which were from two biological replicates.Spore purificationTo generate spores used for immunoblot analysis or to determine the total DPA content, cells were grown in DSM for 3 h at 37°C to an OD600 of 0.2–0.3 and then spread on DSM agar plates, followed by incubation for 96 h at 37°C. Spores were scraped from the plates, washed three times with ddH2O, and then resuspended in PBS supplemented with 1.5 mg/mL lysozyme. After incubation for 1 h at 37°C, SDS was added to the final concentration of 2% (w/v), and the spores were incubated for 30 min at 37°C and then washed five times with ddH2O.To generate purified phase-bright spores for germination assays and microscopy analysis, spores were scraped from DSM agar plates as described above. The spores were washed three times with ddH2O and then resuspended in 350 µL of 20% histodenz (Sigma-Aldrich). The suspension was layered on top of 1 mL of 50% histodenz in a microfuge tube, and the step gradient was centrifuged at 15,000 rpm for 30 min. The pellet fraction containing the phase-bright spores was collected and washed five times with ddH2O.DPA quantificationPurified spores were normalized to an OD600 of 1 in 1 mL of ddH2O, and the spore suspension was incubated for 30 min at 100°C to release DPA. After centrifugation at 15,000 rpm for 5 min, 300 µL of the supernatant was collected and mixed with 300 µL of 100 µM TbCl3. Next, 150 µL of the mixture was transferred to a black, flat-bottom, 96-well plate, and the fluorescence signal was measured at 545 nm with excitation at 272 nm using an Infinite M Plex plate reader (Tecan). Each sample was analyzed in technical triplicate and compared with a standard curve generated using purified DPA (Sigma-Aldrich).DPA release assayHistodenz-purified phase-bright spores were normalized to an OD600 of 1 in 25 mM HEPES (pH 7.4) and heat-activated for 30 min at 70°C, followed by incubation for 15 min on ice. Seventy-five microliters of the spore suspension was transferred to a black, flat-bottom, 96-well plate, and an equal volume of 25 mM HEPES (pH 7.4; buffer), 2 or 20 mM L-alanine, 20 mM AGFK (20 mM L-asparagine, D-glucose, fructose, and KCl), and 20 mM GLPK (20 mM D-glucose, L-leucine, L-proline, and KBr) resuspended in 25 mM HEPES (pH 7.4) was added to the spores. All nutrients and buffer contained 100 µM TbCl3, resulting in a final concentration of 50 µM. The fluorescence was monitored at 545 nm with excitation at 272 nm every 2 min for 2 h in an Infinite M Plex plate reader (Tecan). The 96-well plate was maintained at 30°C and agitated between measurements. All spore samples and conditions were tested in technical triplicate and compared with a standard curve.Reduction in optical density (OD600) assayHistodenz-purified phase-bright spores were normalized to an OD600 of 1.2 in 25 mM HEPES (pH 7.4) and heat-activated for 30 min at 70°C, followed by incubation for 15 min on ice. One-hundred microliters of spore suspension was then transferred to a clear, flat-bottom, 96-well plate. An equal volume of nutrients or buffer as described above was added to the spore suspension for a final OD600 of 0.6. The OD600 was monitored every 2 min for 4 h using an infinite M Plex plate reader (Tecan). The plate was maintained at 37°C with agitation between measurements. All samples were analyzed in technical triplicate.DPA release assay with low concentration of L-alanineHistodenz-purified phase-bright spores were normalized to an OD600 of 1 in ddH2O or 20 µM L-alanine and then incubated at 37°C without agitation. Five-hundred microliters of the spore suspension was collected at the indicated times, followed by centrifugation at 15,000 rpm for 5 min. Next, 450 µL of supernatant was mixed with an equal volume of 100 µM TbCl3 and analyzed for DPA as described above. The spore pellets were analyzed by phase-contrast microscopy.Cation release assay using inductively coupled plasma mass spectrometry (ICP-MS)Histodenz-purified phase-bright spores were normalized to an OD600 of 5 in 20 mL of ddH2O and heat-activated for 30 min at 70°C, followed by incubation for 15 min on ice. One milliliter of the spore suspension was mixed with 1 mL of ddH2O and boiled for 30 min to release all DPA and ions from the spore core. The supernatant was collected and stored on ice. The remaining 19 mL of spore suspension was prewarmed for 20 min at 37°C, followed by the addition of an equal volume of 20 mM L-alanine. The mixture was vortexed and incubated at 37°C. At the indicated time points, 2 mL of the spore suspension was collected, and the germination exudate (the supernatant) was collected by centrifugation at 15,000 rpm for 30 sec. Next, 1.8 mL of the supernatant was transferred to a fresh microfuge tube (avoiding the spore pellet) and stored on ice until all samples were collected. In all cases, the spore suspension was collected ∼30 sec prior to the indicated time point.Upon completion of the time course, the germination exudates were separately passed through 0.2-µm syringe filters to remove all particulates. Next, 150 µL of the filtrate was used for DPA quantification by mixing with an equal volume of 100 µM TbCl3 and then analyzed as described above. The remaining 1.6 mL of filtered germination exudate was mixed with 400 µL of 10% nitric acid. The concentrations of K+, Mg2+, and Ca2+ in these samples were quantified using an Agilent 7900 inductively coupled plasma mass spectrometer in the Center for Environmental Health Sciences Bioanalytical Core Facility at Massachusetts Institute of Technology. The instrument was operated in helium mode. All samples were analyzed in biological triplicate. A standard curve for each cation was generated prior to analyzing the germination exudates for each ICP-MS experiment using ultrapure stocks of KCl, MgCl2, and CaCl2 (Avantor). The concentration of ions present prior to L-alanine addition was subtracted from the concentrations at all time points after addition. The data were plotted as a percentage of ion released based on the concentration released by WT spores at 60 min after L-alanine addition.Screen for constitutively active 5AF mutantsThe spoVAF (5AF) gene was PCR-amplified with primers oYG417 and oYG418 using an error-prone Pfu polymerase (Biles and Connolly 2004), and pYG156 harboring the 5AF gene and a kan cassette were used as templates. The mutagenized PCR product was cloned into pCB179 [yhdG::spec (amp)], cut with SpeI and XhoI by isothermal assembly, and transformed into E. coli DH5α. Approximately 130,000 transformants were pooled and stored −80°C. Plasmids from 20 individual transformants were isolated and subjected to Sanger sequencing using oYG380 and oYG17. Approximately 30 kb was sequenced and 37 mutations were identified for an average of ∼1.2 mutations per kilobase (1.85 mutations per gene). The plasmid library was isolated using a minipreparation kit (Zymo Research) and transformed into the host strain bYG516 (ΔsleB::lox72 ΔspoVAF::lox72 yhdG::erm), and ∼80,000 transformants were pooled and stored in 15% glycerol stock at −80°C.To screen for constitutively active 5AF mutants, the pooled B. subtilis library was grown in liquid DSM for 30 h at 37°C. The sporulated cultures were washed three times with ddH2O and layered on a histodenz step gradient. The supernatant “step” fraction containing vegetatively growing cells and germinated phase-gray spores lacking DPA was isolated and resuspended in PBS supplemented with 1.5 mg/mL lysozyme. After incubation for 1 h at 37°C, SDS was added to a final concentration of 2% (w/v), and cells were incubated for 30 min at 37°C and then washed five times with ddH2O. The phase-gray spores were further purified on a histodenz step gradient. The purified spores were incubated with freshly prepared Ca2+-DPA (equal volumes of 120 mM CaCl2 and 120 mM DPA in Tris-HCl at pH 8.0). The mixture was rolled for 2 h at 22°C to activate the cortex-lytic enzyme CwlJ. The mixture was then serially diluted and spread on LB agar followed by incubation overnight at 37°C. Approximately 16,000 colonies were pooled for another round of enrichment as described above.After a second round of enrichment, >200 individual colonies were picked and streaked on sporulation (DSM) agar and incubated for 96 h at 37°C, followed by analysis by phase-contrast microscopy to screen for phase-gray spores, indicative of premature germination. Genomic DNAs (gDNAs) from mutants with phase-gray spores were then isolated and transformed into the parental strain (bYG516) for validation. The gDNAs were also used to transform bYG25 (ΔspoVAF::lox72 yhdG::erm), and the resulting strains were tested for premature germination and the production of phase-dark and lysed spores. The 5AF gene was PCR-amplified from the gDNAs of positive hits using oCB63 and oYG381, and the product was analyzed by Sanger sequencing using oYG380 and oYG17.MicroscopySamples were concentrated by centrifugation at 8000 rpm and immobilized on 1.5% agarose pads. Phase-contrast microscopy and fluorescence microcopy were performed using a Nikon TE2000 inverted microscope equipped with a plan apo 100×/1.4 oil Ph3 DM objective lens and a CoolSNAP HQ2 monochrome CCD camera (Photometrics). For all sporulation cultures and purified phase-bright spores, the exposure time for both GFP and mScarlet fusions was 800 msec. For exponentially growing cells, the exposure time for both 5AF-GFP and 5AF-mYpet was 500 msec. Image analysis and processing were performed using Fiji or MetaMorph software (version 7.7, Molecular Devices).Quantification of fluorescence intensities of 5AF-mYpet and GerAA-mYpet fociTo quantify the fluorescent foci of 5AF-mYpet and GerAA-mYpet in the presence of FigP or GerAB and GerAC, respectively, the strains (bYG1664 and bYG1179) were grown in LB medium supplemented with 1 mM IPTG for 3–4 h, and fluorescent images were captured as described above and analyzed using an ImageJ plug-in in MicrobeJ (Ducret et al. 2016). Briefly, all single cells in the phase-contrast channel were detected based on the following parameters: cell area: >1.1 µm2, cell length: >1 µm, cell width: between 0.5 and 2 µm, and cell angularity: <0.5 rad. All 5AF-mYpet or GerAA-mYpet foci were detected in each cell by setting the fluorescent tolerance value in “Maxima” to 200. Both the focus intensity (mean value) and the average fluorescence intensity (mean value) per cell were quantified. In a subset of cells, the foci were too crowded to accurately detect and quantify individual foci. To enable the analysis, cells with average fluorescence intensity >800 were removed, and only foci in cells with an average fluorescence intensity <800 were analyzed. The data were plotted in GraphPad Prism.Analysis of membrane potential and membrane permeabilityStrains were grown in 30 mL of LB medium at 37°C. When the OD600 reached 0.2–0.3, IPTG was added to a final concentration of 50 µM. One milliliter of culture was collected at the indicated time points, 200 µL of culture was stained with the potentiometric fluorescent dye 3,3′-dipropylthiadicarbocyanine iodide [DiSC3(5); 1 µM final; Invitrogen], and 600 µL of culture was mixed with propidium iodide (PI; 0.5 µM final; Sigma-Aldrich). The cells were then immobilized on 1.5% agarose pads and analyzed by fluorescence microscope using the RFP/mCherry filter set on a Nikon TE2000 inverted microscope equipped with a plan apo 100×/1.4 oil Ph3 DM objective lens and a CoolSNAP HQ2 monochrome CCD camera (Photometrics). The exposure time for both DiSC3(5) and PI was 200 msec. Image analysis and processing were performed using Fiji or GraphPad Prism [version 9.4.1(458)].Coexpression and purification of 5AF and FigP from exponentially growing B. subtilisB. subtilis strains (bYG1144, bYG1201, bYG1184, bYG1280, and bYG1265) harboring IPTG- and xylose-regulated alleles of 5AF and FigP were precultured in LB at 37°C to an OD600 of 0.6 and then diluted into 1 L of LB medium supplemented with 1 mM IPTG and 33 mM xylose at an OD600 of 0.01. Cells were grown with aeration for ∼4 h at 37°C and harvested at an OD600 of ∼1 by centrifugation at 8000 rpm for 15 min. The pellets were washed twice with 1× SMM (0.5 M sucrose, 20 mM maleic acid, 20 mM MgCl2, adjusted to pH 6.5) and resuspended in in 40 mL of 1× SMM containing 0.5 mg/mL lysozyme. The cells were gently agitated for ∼60 min at room temperature until >95% of the cells had converted into protoplasts as determined by phase-contrast microscopy.The protoplasts were pelleted and lysed in 45 mL of cold hypotonic buffer (50 mM HEPES at pH 7.6, 150 mM NaCl, 20 mM MgCl2, 1 mM DTT) supplemented with 5 U/mL of benzonase (Sigma-Aldrich) and 1× complete protease inhibitor (Roche). The lysate was subjected to ultracentrifugation at 35,000 rpm for 60 min at 4°C. The membrane pellet was resuspended in 50 mL of homogenization buffer (20 mM HEPES at pH 7.6, 150 mM NaCl, 20% glycerol) supplemented with 1% n-dodecyl-β-D-maltopyranoside (DDM) using a glass homogenizer. The solubilized suspension was rotated for 2 h at 4°C, followed by ultracentrifugation at 35,000 rpm for 60 min at 4°C. The supernatant containing detergent-solubilized membrane proteins was supplemented with 2 mM CaCl2 and loaded onto a Bio-Rad column containing 1.5 mL of homemade anti-ProteinC antibody resin. The anti-ProteinC antibody resin was made by incubating anti-ProteinC monoclonal antibodies with CNBr-activated Sepharose 4B (Cytiva). The resin was washed with 25 column volumes (CV) of wash buffer (20 mM HEPES at pH 7.6, 150 mM NaCl, 20% glycerol, 2 mM CaCl2, 0.1% DDM), and proteins were eluted with 5 CVs of elution buffer (20 mM HEPES at pH 7.6, 150 mM NaCl, 10% glycerol, 0.1% DDM, 5 mM EDTA at pH 8.0, 0.4 mg/mL ProC peptide [EDQVDPRLIDGK, Genscript]). The eluted proteins were resolved by SDS-PAGE on 17.5% acrylamide gels, followed by Coomassie blue (InstantBlue) staining or analysis by immunoblot as described below.Transposon sequencing (Tn-seq) screenTransposon insertion sequencing (Tn-seq) was performed on two independently generated libraries as described previously (Meeske et al. 2015). Briefly, the transposon plasmid pFR38 harboring the Himar1 C9 transposase erm and the spec cassette flanked by inverted terminal repeats, one of which contained an MmeI site, was transformed into WT and bYG584 [Δ5AF::lox72 ycgO::PVA-spoVAF(A360V); referred to here as 5AF*]. Erythromycin/lincomycin-resistant transformants were grown in LB medium supplemented with 100 µg/mL spectinomycin overnight at 20°C. The cultures were aliquoted and frozen in 15% glycerol. An aliquot was thawed and plated on LB agar plates supplemented with 100 µg/mL spectinomycin and incubated overnight at 42°C. Approximately 1 million colonies for each Tn library were scraped, pooled, aliquoted, and frozen in 15% glycerol.An aliquot of the library was thawed, washed in sporulation (DSM complete) medium, and diluted into 50 mL of DSM at an OD600 of 0.05. The cultures were incubated at 37°C with agitation. Samples were harvested at an early stage of starvation (input) and 36 h later (T36). The T36 samples were incubated for 20 min at 80°C to kill nonsporulated cells and prematurely germinated spores and then plated on LB agar. Heat-resistant spores that could germinate and form colonies overnight at 37°C were then pooled (∼750,000 colonies). Genomic DNA was isolated from the input and output samples and digested with MmeI, followed by adaptor ligation. The transposon–chromosome junctions were amplified in 18 PCR cycles, and the PCR products were gel-purified and sequenced on the Illumina HiSeq platform using TruSeq reagents (Tufts University Core Facility [TUCF]). The reads were mapped to the B. subtilis 168 genome (NCBI NC_000964.3) using Bowtie 1.0.0. The genes in which reads were statistically overrepresented or underrepresented in the 5AF*(output) library compared with the WT (output) library were visualized using Artemis (version 18.1.0; Sanger Institute).Lysates and immunoblot analysisSpore lysates were generated from spores purified on a histodenz step gradient or by lysozyme-SDS as described above. Spores were resuspended in 500 µL of cold PBS supplemented with 1 mM phenylmethylsulfonyl fluoride (PMSF) and transferred to 2-mL tubes containing lysis matrix B (MP Biomedicals). After incubation for 15 min on ice, the spores were lysed using FastPrep (MP Biomedicals) at 6.5 m/sec for 60 sec. An equal volume of 2× sample buffer (4% SDS, 250 mM Tris at pH 6.8, 20% glycerol, 10 mM EDTA, Bromophenol blue) containing 10% β-mercaptoethanol was immediately added to the tubes and vortexed. After centrifugation at 15,000 rpm for 5 min, the supernatant was collected and total protein was quantified by a noninterfering protein assay (G-Biosciences). The normalized protein samples were resolved by SDS-PAGE on 17.5% polyacrylamide gels and transferred to an Immobilon-P membrane (Millipore).Vegetative cell lysates were generated from B. subtilis cultures grown in LB medium. Briefly, cells were harvested at an OD600 of 0.5–0.8 and normalized to an OD600 of 0.5. One milliliter of normalized culture was centrifuged, and the cell pellet was resuspended in 50 µL of lysis buffer (20 mM Tris at pH 7.5, 10 mM EDTA, 1 mg/mL lysozyme [Sigma-Aldrich], 1 mM PMSF, 10 µg/mL DNase I [NEB], 100 µg/mL RNase A [NEB], 10 µg/mL leupeptin [Fisher Scientific], 10 µg/mL pepstatin [Fisher Scientific]) and incubated for 10 min at 37°C, followed by addition of an equal volume of 2× sample buffer containing 10% β-mercaptoethanol. The lysates were resolved by SDS-PAGE on 17.5% acrylamide gels and transferred to an Immobilon-P membrane (Millipore).The PVDF membranes were blocked in 5% nonfat milk dissolved in 1× PBS with 0.5% Tween-20 (PBST) and then probed with anti-His (1:4000; GenScript), anti-ProC (1:1000) (Gao et al. 2023), anti-FLAG (1:5000; Sigma-Aldrich), anti-SigA (1:10,000) (Fujita and Sadaie 1998), anti-EzrA (1:10,000) (Levin et al. 1999), anti-GerBC (1:5000) (Stewart et al. 2012), or anti-SpoVAD (1:10,000) (Vepachedu and Setlow 2005) antibodies diluted in 3% BSA dissolved in PBST. After washing in PBST, the primary antibodies were detected with goat antirabbit or antimouse antibodies coupled to horseradish peroxidase (Bio-Rad) and detected and visualized by Western Lightning ECL reagent (PerkinElmer).Structural modeling using AlphaFold-MultimerAll predicted structures were generated using AlphaFold-Multimer-v2 or AlphaFold-Multimer-v3 and ColabFold (Jumper et al. 2021; Evans et al. 2022), run locally on the Harvard Medical School O2 computing cluster or the Lambda Cloud computing server. General parameters were as follows: Multiple sequence alignments (MSA) were built using mmseqs2. Sequences from the same operon were paired in the MSA, and both paired and unpaired sequences were used to generate models. Five models were generated in each run, and each model was relaxed using AMBER and ranked by pTM score. Homologous templates found in the PDB were used. For the SpoVAF pentamer and the SpoVAF-FigP pentamer of dimers, the maximum number of model recycling was set to 12, and the top-ranked models were used throughout the text.Data availabilityAll data are available here and in the Supplementary Material.

Materials and methods

General methodsAll strains used in this study were derived from Bacillus subtilis 168 (Zeigler et al. 2008). Sporulation was induced by nutrient exhaustion in complete Difco sporulation medium (DSM) for 30 h at 37°C (Schaeffer et al. 1965), and sporulation efficiency was determined by comparing heat-resistant (for 20 min at 80°C) colony-forming units (CFUs) of mutants with wild-type B. subtilis 168. All in-frame deletion mutants were derived from Bacillus knockout collection (BKE) (Koo et al. 2017) or constructed by direct transformation of isothermal assembly products into competent B. subtilis. Antibiotic cassettes were excised using a temperature-sensitive plasmid that constitutively expresses Cre recombinase (Meeske et al. 2015). Site-directed mutants were generated using a modified QuickChange protocol. Strains, plasmids, and oligonucleotide primers used in this study are in Supplemental Tables S1–S3. Experiments presented in the text and supplemental figures were from one of at least three biological replicates, with the exception of Figures 2F, 3E, and 4C and Supplemental Figures S16A, S18E, and S19, which were from two biological replicates.

General methods

All strains used in this study were derived from Bacillus subtilis 168 (Zeigler et al. 2008). Sporulation was induced by nutrient exhaustion in complete Difco sporulation medium (DSM) for 30 h at 37°C (Schaeffer et al. 1965), and sporulation efficiency was determined by comparing heat-resistant (for 20 min at 80°C) colony-forming units (CFUs) of mutants with wild-type B. subtilis 168. All in-frame deletion mutants were derived from Bacillus knockout collection (BKE) (Koo et al. 2017) or constructed by direct transformation of isothermal assembly products into competent B. subtilis. Antibiotic cassettes were excised using a temperature-sensitive plasmid that constitutively expresses Cre recombinase (Meeske et al. 2015). Site-directed mutants were generated using a modified QuickChange protocol. Strains, plasmids, and oligonucleotide primers used in this study are in Supplemental Tables S1–S3. Experiments presented in the text and supplemental figures were from one of at least three biological replicates, with the exception of Figures 2F, 3E, and 4C and Supplemental Figures S16A, S18E, and S19, which were from two biological replicates.

Spore purificationTo generate spores used for immunoblot analysis or to determine the total DPA content, cells were grown in DSM for 3 h at 37°C to an OD600 of 0.2–0.3 and then spread on DSM agar plates, followed by incubation for 96 h at 37°C. Spores were scraped from the plates, washed three times with ddH2O, and then resuspended in PBS supplemented with 1.5 mg/mL lysozyme. After incubation for 1 h at 37°C, SDS was added to the final concentration of 2% (w/v), and the spores were incubated for 30 min at 37°C and then washed five times with ddH2O.To generate purified phase-bright spores for germination assays and microscopy analysis, spores were scraped from DSM agar plates as described above. The spores were washed three times with ddH2O and then resuspended in 350 µL of 20% histodenz (Sigma-Aldrich). The suspension was layered on top of 1 mL of 50% histodenz in a microfuge tube, and the step gradient was centrifuged at 15,000 rpm for 30 min. The pellet fraction containing the phase-bright spores was collected and washed five times with ddH2O.

Spore purification

To generate spores used for immunoblot analysis or to determine the total DPA content, cells were grown in DSM for 3 h at 37°C to an OD600 of 0.2–0.3 and then spread on DSM agar plates, followed by incubation for 96 h at 37°C. Spores were scraped from the plates, washed three times with ddH2O, and then resuspended in PBS supplemented with 1.5 mg/mL lysozyme. After incubation for 1 h at 37°C, SDS was added to the final concentration of 2% (w/v), and the spores were incubated for 30 min at 37°C and then washed five times with ddH2O.

To generate purified phase-bright spores for germination assays and microscopy analysis, spores were scraped from DSM agar plates as described above. The spores were washed three times with ddH2O and then resuspended in 350 µL of 20% histodenz (Sigma-Aldrich). The suspension was layered on top of 1 mL of 50% histodenz in a microfuge tube, and the step gradient was centrifuged at 15,000 rpm for 30 min. The pellet fraction containing the phase-bright spores was collected and washed five times with ddH2O.

DPA quantificationPurified spores were normalized to an OD600 of 1 in 1 mL of ddH2O, and the spore suspension was incubated for 30 min at 100°C to release DPA. After centrifugation at 15,000 rpm for 5 min, 300 µL of the supernatant was collected and mixed with 300 µL of 100 µM TbCl3. Next, 150 µL of the mixture was transferred to a black, flat-bottom, 96-well plate, and the fluorescence signal was measured at 545 nm with excitation at 272 nm using an Infinite M Plex plate reader (Tecan). Each sample was analyzed in technical triplicate and compared with a standard curve generated using purified DPA (Sigma-Aldrich).

DPA quantification

Purified spores were normalized to an OD600 of 1 in 1 mL of ddH2O, and the spore suspension was incubated for 30 min at 100°C to release DPA. After centrifugation at 15,000 rpm for 5 min, 300 µL of the supernatant was collected and mixed with 300 µL of 100 µM TbCl3. Next, 150 µL of the mixture was transferred to a black, flat-bottom, 96-well plate, and the fluorescence signal was measured at 545 nm with excitation at 272 nm using an Infinite M Plex plate reader (Tecan). Each sample was analyzed in technical triplicate and compared with a standard curve generated using purified DPA (Sigma-Aldrich).

DPA release assayHistodenz-purified phase-bright spores were normalized to an OD600 of 1 in 25 mM HEPES (pH 7.4) and heat-activated for 30 min at 70°C, followed by incubation for 15 min on ice. Seventy-five microliters of the spore suspension was transferred to a black, flat-bottom, 96-well plate, and an equal volume of 25 mM HEPES (pH 7.4; buffer), 2 or 20 mM L-alanine, 20 mM AGFK (20 mM L-asparagine, D-glucose, fructose, and KCl), and 20 mM GLPK (20 mM D-glucose, L-leucine, L-proline, and KBr) resuspended in 25 mM HEPES (pH 7.4) was added to the spores. All nutrients and buffer contained 100 µM TbCl3, resulting in a final concentration of 50 µM. The fluorescence was monitored at 545 nm with excitation at 272 nm every 2 min for 2 h in an Infinite M Plex plate reader (Tecan). The 96-well plate was maintained at 30°C and agitated between measurements. All spore samples and conditions were tested in technical triplicate and compared with a standard curve.

DPA release assay

Histodenz-purified phase-bright spores were normalized to an OD600 of 1 in 25 mM HEPES (pH 7.4) and heat-activated for 30 min at 70°C, followed by incubation for 15 min on ice. Seventy-five microliters of the spore suspension was transferred to a black, flat-bottom, 96-well plate, and an equal volume of 25 mM HEPES (pH 7.4; buffer), 2 or 20 mM L-alanine, 20 mM AGFK (20 mM L-asparagine, D-glucose, fructose, and KCl), and 20 mM GLPK (20 mM D-glucose, L-leucine, L-proline, and KBr) resuspended in 25 mM HEPES (pH 7.4) was added to the spores. All nutrients and buffer contained 100 µM TbCl3, resulting in a final concentration of 50 µM. The fluorescence was monitored at 545 nm with excitation at 272 nm every 2 min for 2 h in an Infinite M Plex plate reader (Tecan). The 96-well plate was maintained at 30°C and agitated between measurements. All spore samples and conditions were tested in technical triplicate and compared with a standard curve.

Reduction in optical density (OD600) assayHistodenz-purified phase-bright spores were normalized to an OD600 of 1.2 in 25 mM HEPES (pH 7.4) and heat-activated for 30 min at 70°C, followed by incubation for 15 min on ice. One-hundred microliters of spore suspension was then transferred to a clear, flat-bottom, 96-well plate. An equal volume of nutrients or buffer as described above was added to the spore suspension for a final OD600 of 0.6. The OD600 was monitored every 2 min for 4 h using an infinite M Plex plate reader (Tecan). The plate was maintained at 37°C with agitation between measurements. All samples were analyzed in technical triplicate.

Reduction in optical density (OD600) assay

Histodenz-purified phase-bright spores were normalized to an OD600 of 1.2 in 25 mM HEPES (pH 7.4) and heat-activated for 30 min at 70°C, followed by incubation for 15 min on ice. One-hundred microliters of spore suspension was then transferred to a clear, flat-bottom, 96-well plate. An equal volume of nutrients or buffer as described above was added to the spore suspension for a final OD600 of 0.6. The OD600 was monitored every 2 min for 4 h using an infinite M Plex plate reader (Tecan). The plate was maintained at 37°C with agitation between measurements. All samples were analyzed in technical triplicate.

DPA release assay with low concentration of L-alanineHistodenz-purified phase-bright spores were normalized to an OD600 of 1 in ddH2O or 20 µM L-alanine and then incubated at 37°C without agitation. Five-hundred microliters of the spore suspension was collected at the indicated times, followed by centrifugation at 15,000 rpm for 5 min. Next, 450 µL of supernatant was mixed with an equal volume of 100 µM TbCl3 and analyzed for DPA as described above. The spore pellets were analyzed by phase-contrast microscopy.

DPA release assay with low concentration of L-alanine

Histodenz-purified phase-bright spores were normalized to an OD600 of 1 in ddH2O or 20 µM L-alanine and then incubated at 37°C without agitation. Five-hundred microliters of the spore suspension was collected at the indicated times, followed by centrifugation at 15,000 rpm for 5 min. Next, 450 µL of supernatant was mixed with an equal volume of 100 µM TbCl3 and analyzed for DPA as described above. The spore pellets were analyzed by phase-contrast microscopy.

Cation release assay using inductively coupled plasma mass spectrometry (ICP-MS)Histodenz-purified phase-bright spores were normalized to an OD600 of 5 in 20 mL of ddH2O and heat-activated for 30 min at 70°C, followed by incubation for 15 min on ice. One milliliter of the spore suspension was mixed with 1 mL of ddH2O and boiled for 30 min to release all DPA and ions from the spore core. The supernatant was collected and stored on ice. The remaining 19 mL of spore suspension was prewarmed for 20 min at 37°C, followed by the addition of an equal volume of 20 mM L-alanine. The mixture was vortexed and incubated at 37°C. At the indicated time points, 2 mL of the spore suspension was collected, and the germination exudate (the supernatant) was collected by centrifugation at 15,000 rpm for 30 sec. Next, 1.8 mL of the supernatant was transferred to a fresh microfuge tube (avoiding the spore pellet) and stored on ice until all samples were collected. In all cases, the spore suspension was collected ∼30 sec prior to the indicated time point.Upon completion of the time course, the germination exudates were separately passed through 0.2-µm syringe filters to remove all particulates. Next, 150 µL of the filtrate was used for DPA quantification by mixing with an equal volume of 100 µM TbCl3 and then analyzed as described above. The remaining 1.6 mL of filtered germination exudate was mixed with 400 µL of 10% nitric acid. The concentrations of K+, Mg2+, and Ca2+ in these samples were quantified using an Agilent 7900 inductively coupled plasma mass spectrometer in the Center for Environmental Health Sciences Bioanalytical Core Facility at Massachusetts Institute of Technology. The instrument was operated in helium mode. All samples were analyzed in biological triplicate. A standard curve for each cation was generated prior to analyzing the germination exudates for each ICP-MS experiment using ultrapure stocks of KCl, MgCl2, and CaCl2 (Avantor). The concentration of ions present prior to L-alanine addition was subtracted from the concentrations at all time points after addition. The data were plotted as a percentage of ion released based on the concentration released by WT spores at 60 min after L-alanine addition.

Cation release assay using inductively coupled plasma mass spectrometry (ICP-MS)

Histodenz-purified phase-bright spores were normalized to an OD600 of 5 in 20 mL of ddH2O and heat-activated for 30 min at 70°C, followed by incubation for 15 min on ice. One milliliter of the spore suspension was mixed with 1 mL of ddH2O and boiled for 30 min to release all DPA and ions from the spore core. The supernatant was collected and stored on ice. The remaining 19 mL of spore suspension was prewarmed for 20 min at 37°C, followed by the addition of an equal volume of 20 mM L-alanine. The mixture was vortexed and incubated at 37°C. At the indicated time points, 2 mL of the spore suspension was collected, and the germination exudate (the supernatant) was collected by centrifugation at 15,000 rpm for 30 sec. Next, 1.8 mL of the supernatant was transferred to a fresh microfuge tube (avoiding the spore pellet) and stored on ice until all samples were collected. In all cases, the spore suspension was collected ∼30 sec prior to the indicated time point.

Upon completion of the time course, the germination exudates were separately passed through 0.2-µm syringe filters to remove all particulates. Next, 150 µL of the filtrate was used for DPA quantification by mixing with an equal volume of 100 µM TbCl3 and then analyzed as described above. The remaining 1.6 mL of filtered germination exudate was mixed with 400 µL of 10% nitric acid. The concentrations of K+, Mg2+, and Ca2+ in these samples were quantified using an Agilent 7900 inductively coupled plasma mass spectrometer in the Center for Environmental Health Sciences Bioanalytical Core Facility at Massachusetts Institute of Technology. The instrument was operated in helium mode. All samples were analyzed in biological triplicate. A standard curve for each cation was generated prior to analyzing the germination exudates for each ICP-MS experiment using ultrapure stocks of KCl, MgCl2, and CaCl2 (Avantor). The concentration of ions present prior to L-alanine addition was subtracted from the concentrations at all time points after addition. The data were plotted as a percentage of ion released based on the concentration released by WT spores at 60 min after L-alanine addition.

Screen for constitutively active 5AF mutantsThe spoVAF (5AF) gene was PCR-amplified with primers oYG417 and oYG418 using an error-prone Pfu polymerase (Biles and Connolly 2004), and pYG156 harboring the 5AF gene and a kan cassette were used as templates. The mutagenized PCR product was cloned into pCB179 [yhdG::spec (amp)], cut with SpeI and XhoI by isothermal assembly, and transformed into E. coli DH5α. Approximately 130,000 transformants were pooled and stored −80°C. Plasmids from 20 individual transformants were isolated and subjected to Sanger sequencing using oYG380 and oYG17. Approximately 30 kb was sequenced and 37 mutations were identified for an average of ∼1.2 mutations per kilobase (1.85 mutations per gene). The plasmid library was isolated using a minipreparation kit (Zymo Research) and transformed into the host strain bYG516 (ΔsleB::lox72 ΔspoVAF::lox72 yhdG::erm), and ∼80,000 transformants were pooled and stored in 15% glycerol stock at −80°C.To screen for constitutively active 5AF mutants, the pooled B. subtilis library was grown in liquid DSM for 30 h at 37°C. The sporulated cultures were washed three times with ddH2O and layered on a histodenz step gradient. The supernatant “step” fraction containing vegetatively growing cells and germinated phase-gray spores lacking DPA was isolated and resuspended in PBS supplemented with 1.5 mg/mL lysozyme. After incubation for 1 h at 37°C, SDS was added to a final concentration of 2% (w/v), and cells were incubated for 30 min at 37°C and then washed five times with ddH2O. The phase-gray spores were further purified on a histodenz step gradient. The purified spores were incubated with freshly prepared Ca2+-DPA (equal volumes of 120 mM CaCl2 and 120 mM DPA in Tris-HCl at pH 8.0). The mixture was rolled for 2 h at 22°C to activate the cortex-lytic enzyme CwlJ. The mixture was then serially diluted and spread on LB agar followed by incubation overnight at 37°C. Approximately 16,000 colonies were pooled for another round of enrichment as described above.After a second round of enrichment, >200 individual colonies were picked and streaked on sporulation (DSM) agar and incubated for 96 h at 37°C, followed by analysis by phase-contrast microscopy to screen for phase-gray spores, indicative of premature germination. Genomic DNAs (gDNAs) from mutants with phase-gray spores were then isolated and transformed into the parental strain (bYG516) for validation. The gDNAs were also used to transform bYG25 (ΔspoVAF::lox72 yhdG::erm), and the resulting strains were tested for premature germination and the production of phase-dark and lysed spores. The 5AF gene was PCR-amplified from the gDNAs of positive hits using oCB63 and oYG381, and the product was analyzed by Sanger sequencing using oYG380 and oYG17.

Screen for constitutively active 5AF mutants

The spoVAF (5AF) gene was PCR-amplified with primers oYG417 and oYG418 using an error-prone Pfu polymerase (Biles and Connolly 2004), and pYG156 harboring the 5AF gene and a kan cassette were used as templates. The mutagenized PCR product was cloned into pCB179 [yhdG::spec (amp)], cut with SpeI and XhoI by isothermal assembly, and transformed into E. coli DH5α. Approximately 130,000 transformants were pooled and stored −80°C. Plasmids from 20 individual transformants were isolated and subjected to Sanger sequencing using oYG380 and oYG17. Approximately 30 kb was sequenced and 37 mutations were identified for an average of ∼1.2 mutations per kilobase (1.85 mutations per gene). The plasmid library was isolated using a minipreparation kit (Zymo Research) and transformed into the host strain bYG516 (ΔsleB::lox72 ΔspoVAF::lox72 yhdG::erm), and ∼80,000 transformants were pooled and stored in 15% glycerol stock at −80°C.

To screen for constitutively active 5AF mutants, the pooled B. subtilis library was grown in liquid DSM for 30 h at 37°C. The sporulated cultures were washed three times with ddH2O and layered on a histodenz step gradient. The supernatant “step” fraction containing vegetatively growing cells and germinated phase-gray spores lacking DPA was isolated and resuspended in PBS supplemented with 1.5 mg/mL lysozyme. After incubation for 1 h at 37°C, SDS was added to a final concentration of 2% (w/v), and cells were incubated for 30 min at 37°C and then washed five times with ddH2O. The phase-gray spores were further purified on a histodenz step gradient. The purified spores were incubated with freshly prepared Ca2+-DPA (equal volumes of 120 mM CaCl2 and 120 mM DPA in Tris-HCl at pH 8.0). The mixture was rolled for 2 h at 22°C to activate the cortex-lytic enzyme CwlJ. The mixture was then serially diluted and spread on LB agar followed by incubation overnight at 37°C. Approximately 16,000 colonies were pooled for another round of enrichment as described above.

After a second round of enrichment, >200 individual colonies were picked and streaked on sporulation (DSM) agar and incubated for 96 h at 37°C, followed by analysis by phase-contrast microscopy to screen for phase-gray spores, indicative of premature germination. Genomic DNAs (gDNAs) from mutants with phase-gray spores were then isolated and transformed into the parental strain (bYG516) for validation. The gDNAs were also used to transform bYG25 (ΔspoVAF::lox72 yhdG::erm), and the resulting strains were tested for premature germination and the production of phase-dark and lysed spores. The 5AF gene was PCR-amplified from the gDNAs of positive hits using oCB63 and oYG381, and the product was analyzed by Sanger sequencing using oYG380 and oYG17.

MicroscopySamples were concentrated by centrifugation at 8000 rpm and immobilized on 1.5% agarose pads. Phase-contrast microscopy and fluorescence microcopy were performed using a Nikon TE2000 inverted microscope equipped with a plan apo 100×/1.4 oil Ph3 DM objective lens and a CoolSNAP HQ2 monochrome CCD camera (Photometrics). For all sporulation cultures and purified phase-bright spores, the exposure time for both GFP and mScarlet fusions was 800 msec. For exponentially growing cells, the exposure time for both 5AF-GFP and 5AF-mYpet was 500 msec. Image analysis and processing were performed using Fiji or MetaMorph software (version 7.7, Molecular Devices).

Samples were concentrated by centrifugation at 8000 rpm and immobilized on 1.5% agarose pads. Phase-contrast microscopy and fluorescence microcopy were performed using a Nikon TE2000 inverted microscope equipped with a plan apo 100×/1.4 oil Ph3 DM objective lens and a CoolSNAP HQ2 monochrome CCD camera (Photometrics). For all sporulation cultures and purified phase-bright spores, the exposure time for both GFP and mScarlet fusions was 800 msec. For exponentially growing cells, the exposure time for both 5AF-GFP and 5AF-mYpet was 500 msec. Image analysis and processing were performed using Fiji or MetaMorph software (version 7.7, Molecular Devices).

Quantification of fluorescence intensities of 5AF-mYpet and GerAA-mYpet fociTo quantify the fluorescent foci of 5AF-mYpet and GerAA-mYpet in the presence of FigP or GerAB and GerAC, respectively, the strains (bYG1664 and bYG1179) were grown in LB medium supplemented with 1 mM IPTG for 3–4 h, and fluorescent images were captured as described above and analyzed using an ImageJ plug-in in MicrobeJ (Ducret et al. 2016). Briefly, all single cells in the phase-contrast channel were detected based on the following parameters: cell area: >1.1 µm2, cell length: >1 µm, cell width: between 0.5 and 2 µm, and cell angularity: <0.5 rad. All 5AF-mYpet or GerAA-mYpet foci were detected in each cell by setting the fluorescent tolerance value in “Maxima” to 200. Both the focus intensity (mean value) and the average fluorescence intensity (mean value) per cell were quantified. In a subset of cells, the foci were too crowded to accurately detect and quantify individual foci. To enable the analysis, cells with average fluorescence intensity >800 were removed, and only foci in cells with an average fluorescence intensity <800 were analyzed. The data were plotted in GraphPad Prism.

Quantification of fluorescence intensities of 5AF-mYpet and GerAA-mYpet foci

To quantify the fluorescent foci of 5AF-mYpet and GerAA-mYpet in the presence of FigP or GerAB and GerAC, respectively, the strains (bYG1664 and bYG1179) were grown in LB medium supplemented with 1 mM IPTG for 3–4 h, and fluorescent images were captured as described above and analyzed using an ImageJ plug-in in MicrobeJ (Ducret et al. 2016). Briefly, all single cells in the phase-contrast channel were detected based on the following parameters: cell area: >1.1 µm2, cell length: >1 µm, cell width: between 0.5 and 2 µm, and cell angularity: <0.5 rad. All 5AF-mYpet or GerAA-mYpet foci were detected in each cell by setting the fluorescent tolerance value in “Maxima” to 200. Both the focus intensity (mean value) and the average fluorescence intensity (mean value) per cell were quantified. In a subset of cells, the foci were too crowded to accurately detect and quantify individual foci. To enable the analysis, cells with average fluorescence intensity >800 were removed, and only foci in cells with an average fluorescence intensity <800 were analyzed. The data were plotted in GraphPad Prism.

Analysis of membrane potential and membrane permeabilityStrains were grown in 30 mL of LB medium at 37°C. When the OD600 reached 0.2–0.3, IPTG was added to a final concentration of 50 µM. One milliliter of culture was collected at the indicated time points, 200 µL of culture was stained with the potentiometric fluorescent dye 3,3′-dipropylthiadicarbocyanine iodide [DiSC3(5); 1 µM final; Invitrogen], and 600 µL of culture was mixed with propidium iodide (PI; 0.5 µM final; Sigma-Aldrich). The cells were then immobilized on 1.5% agarose pads and analyzed by fluorescence microscope using the RFP/mCherry filter set on a Nikon TE2000 inverted microscope equipped with a plan apo 100×/1.4 oil Ph3 DM objective lens and a CoolSNAP HQ2 monochrome CCD camera (Photometrics). The exposure time for both DiSC3(5) and PI was 200 msec. Image analysis and processing were performed using Fiji or GraphPad Prism [version 9.4.1(458)].

Analysis of membrane potential and membrane permeability

Strains were grown in 30 mL of LB medium at 37°C. When the OD600 reached 0.2–0.3, IPTG was added to a final concentration of 50 µM. One milliliter of culture was collected at the indicated time points, 200 µL of culture was stained with the potentiometric fluorescent dye 3,3′-dipropylthiadicarbocyanine iodide [DiSC3(5); 1 µM final; Invitrogen], and 600 µL of culture was mixed with propidium iodide (PI; 0.5 µM final; Sigma-Aldrich). The cells were then immobilized on 1.5% agarose pads and analyzed by fluorescence microscope using the RFP/mCherry filter set on a Nikon TE2000 inverted microscope equipped with a plan apo 100×/1.4 oil Ph3 DM objective lens and a CoolSNAP HQ2 monochrome CCD camera (Photometrics). The exposure time for both DiSC3(5) and PI was 200 msec. Image analysis and processing were performed using Fiji or GraphPad Prism [version 9.4.1(458)].

Coexpression and purification of 5AF and FigP from exponentially growing B. subtilisB. subtilis strains (bYG1144, bYG1201, bYG1184, bYG1280, and bYG1265) harboring IPTG- and xylose-regulated alleles of 5AF and FigP were precultured in LB at 37°C to an OD600 of 0.6 and then diluted into 1 L of LB medium supplemented with 1 mM IPTG and 33 mM xylose at an OD600 of 0.01. Cells were grown with aeration for ∼4 h at 37°C and harvested at an OD600 of ∼1 by centrifugation at 8000 rpm for 15 min. The pellets were washed twice with 1× SMM (0.5 M sucrose, 20 mM maleic acid, 20 mM MgCl2, adjusted to pH 6.5) and resuspended in in 40 mL of 1× SMM containing 0.5 mg/mL lysozyme. The cells were gently agitated for ∼60 min at room temperature until >95% of the cells had converted into protoplasts as determined by phase-contrast microscopy.The protoplasts were pelleted and lysed in 45 mL of cold hypotonic buffer (50 mM HEPES at pH 7.6, 150 mM NaCl, 20 mM MgCl2, 1 mM DTT) supplemented with 5 U/mL of benzonase (Sigma-Aldrich) and 1× complete protease inhibitor (Roche). The lysate was subjected to ultracentrifugation at 35,000 rpm for 60 min at 4°C. The membrane pellet was resuspended in 50 mL of homogenization buffer (20 mM HEPES at pH 7.6, 150 mM NaCl, 20% glycerol) supplemented with 1% n-dodecyl-β-D-maltopyranoside (DDM) using a glass homogenizer. The solubilized suspension was rotated for 2 h at 4°C, followed by ultracentrifugation at 35,000 rpm for 60 min at 4°C. The supernatant containing detergent-solubilized membrane proteins was supplemented with 2 mM CaCl2 and loaded onto a Bio-Rad column containing 1.5 mL of homemade anti-ProteinC antibody resin. The anti-ProteinC antibody resin was made by incubating anti-ProteinC monoclonal antibodies with CNBr-activated Sepharose 4B (Cytiva). The resin was washed with 25 column volumes (CV) of wash buffer (20 mM HEPES at pH 7.6, 150 mM NaCl, 20% glycerol, 2 mM CaCl2, 0.1% DDM), and proteins were eluted with 5 CVs of elution buffer (20 mM HEPES at pH 7.6, 150 mM NaCl, 10% glycerol, 0.1% DDM, 5 mM EDTA at pH 8.0, 0.4 mg/mL ProC peptide [EDQVDPRLIDGK, Genscript]). The eluted proteins were resolved by SDS-PAGE on 17.5% acrylamide gels, followed by Coomassie blue (InstantBlue) staining or analysis by immunoblot as described below.

Coexpression and purification of 5AF and FigP from exponentially growing B. subtilis

B. subtilis strains (bYG1144, bYG1201, bYG1184, bYG1280, and bYG1265) harboring IPTG- and xylose-regulated alleles of 5AF and FigP were precultured in LB at 37°C to an OD600 of 0.6 and then diluted into 1 L of LB medium supplemented with 1 mM IPTG and 33 mM xylose at an OD600 of 0.01. Cells were grown with aeration for ∼4 h at 37°C and harvested at an OD600 of ∼1 by centrifugation at 8000 rpm for 15 min. The pellets were washed twice with 1× SMM (0.5 M sucrose, 20 mM maleic acid, 20 mM MgCl2, adjusted to pH 6.5) and resuspended in in 40 mL of 1× SMM containing 0.5 mg/mL lysozyme. The cells were gently agitated for ∼60 min at room temperature until >95% of the cells had converted into protoplasts as determined by phase-contrast microscopy.

The protoplasts were pelleted and lysed in 45 mL of cold hypotonic buffer (50 mM HEPES at pH 7.6, 150 mM NaCl, 20 mM MgCl2, 1 mM DTT) supplemented with 5 U/mL of benzonase (Sigma-Aldrich) and 1× complete protease inhibitor (Roche). The lysate was subjected to ultracentrifugation at 35,000 rpm for 60 min at 4°C. The membrane pellet was resuspended in 50 mL of homogenization buffer (20 mM HEPES at pH 7.6, 150 mM NaCl, 20% glycerol) supplemented with 1% n-dodecyl-β-D-maltopyranoside (DDM) using a glass homogenizer. The solubilized suspension was rotated for 2 h at 4°C, followed by ultracentrifugation at 35,000 rpm for 60 min at 4°C. The supernatant containing detergent-solubilized membrane proteins was supplemented with 2 mM CaCl2 and loaded onto a Bio-Rad column containing 1.5 mL of homemade anti-ProteinC antibody resin. The anti-ProteinC antibody resin was made by incubating anti-ProteinC monoclonal antibodies with CNBr-activated Sepharose 4B (Cytiva). The resin was washed with 25 column volumes (CV) of wash buffer (20 mM HEPES at pH 7.6, 150 mM NaCl, 20% glycerol, 2 mM CaCl2, 0.1% DDM), and proteins were eluted with 5 CVs of elution buffer (20 mM HEPES at pH 7.6, 150 mM NaCl, 10% glycerol, 0.1% DDM, 5 mM EDTA at pH 8.0, 0.4 mg/mL ProC peptide [EDQVDPRLIDGK, Genscript]). The eluted proteins were resolved by SDS-PAGE on 17.5% acrylamide gels, followed by Coomassie blue (InstantBlue) staining or analysis by immunoblot as described below.

Transposon sequencing (Tn-seq) screenTransposon insertion sequencing (Tn-seq) was performed on two independently generated libraries as described previously (Meeske et al. 2015). Briefly, the transposon plasmid pFR38 harboring the Himar1 C9 transposase erm and the spec cassette flanked by inverted terminal repeats, one of which contained an MmeI site, was transformed into WT and bYG584 [Δ5AF::lox72 ycgO::PVA-spoVAF(A360V); referred to here as 5AF*]. Erythromycin/lincomycin-resistant transformants were grown in LB medium supplemented with 100 µg/mL spectinomycin overnight at 20°C. The cultures were aliquoted and frozen in 15% glycerol. An aliquot was thawed and plated on LB agar plates supplemented with 100 µg/mL spectinomycin and incubated overnight at 42°C. Approximately 1 million colonies for each Tn library were scraped, pooled, aliquoted, and frozen in 15% glycerol.An aliquot of the library was thawed, washed in sporulation (DSM complete) medium, and diluted into 50 mL of DSM at an OD600 of 0.05. The cultures were incubated at 37°C with agitation. Samples were harvested at an early stage of starvation (input) and 36 h later (T36). The T36 samples were incubated for 20 min at 80°C to kill nonsporulated cells and prematurely germinated spores and then plated on LB agar. Heat-resistant spores that could germinate and form colonies overnight at 37°C were then pooled (∼750,000 colonies). Genomic DNA was isolated from the input and output samples and digested with MmeI, followed by adaptor ligation. The transposon–chromosome junctions were amplified in 18 PCR cycles, and the PCR products were gel-purified and sequenced on the Illumina HiSeq platform using TruSeq reagents (Tufts University Core Facility [TUCF]). The reads were mapped to the B. subtilis 168 genome (NCBI NC_000964.3) using Bowtie 1.0.0. The genes in which reads were statistically overrepresented or underrepresented in the 5AF*(output) library compared with the WT (output) library were visualized using Artemis (version 18.1.0; Sanger Institute).

Transposon sequencing (Tn-seq) screen

Transposon insertion sequencing (Tn-seq) was performed on two independently generated libraries as described previously (Meeske et al. 2015). Briefly, the transposon plasmid pFR38 harboring the Himar1 C9 transposase erm and the spec cassette flanked by inverted terminal repeats, one of which contained an MmeI site, was transformed into WT and bYG584 [Δ5AF::lox72 ycgO::PVA-spoVAF(A360V); referred to here as 5AF*]. Erythromycin/lincomycin-resistant transformants were grown in LB medium supplemented with 100 µg/mL spectinomycin overnight at 20°C. The cultures were aliquoted and frozen in 15% glycerol. An aliquot was thawed and plated on LB agar plates supplemented with 100 µg/mL spectinomycin and incubated overnight at 42°C. Approximately 1 million colonies for each Tn library were scraped, pooled, aliquoted, and frozen in 15% glycerol.

An aliquot of the library was thawed, washed in sporulation (DSM complete) medium, and diluted into 50 mL of DSM at an OD600 of 0.05. The cultures were incubated at 37°C with agitation. Samples were harvested at an early stage of starvation (input) and 36 h later (T36). The T36 samples were incubated for 20 min at 80°C to kill nonsporulated cells and prematurely germinated spores and then plated on LB agar. Heat-resistant spores that could germinate and form colonies overnight at 37°C were then pooled (∼750,000 colonies). Genomic DNA was isolated from the input and output samples and digested with MmeI, followed by adaptor ligation. The transposon–chromosome junctions were amplified in 18 PCR cycles, and the PCR products were gel-purified and sequenced on the Illumina HiSeq platform using TruSeq reagents (Tufts University Core Facility [TUCF]). The reads were mapped to the B. subtilis 168 genome (NCBI NC_000964.3) using Bowtie 1.0.0. The genes in which reads were statistically overrepresented or underrepresented in the 5AF*(output) library compared with the WT (output) library were visualized using Artemis (version 18.1.0; Sanger Institute).

Lysates and immunoblot analysisSpore lysates were generated from spores purified on a histodenz step gradient or by lysozyme-SDS as described above. Spores were resuspended in 500 µL of cold PBS supplemented with 1 mM phenylmethylsulfonyl fluoride (PMSF) and transferred to 2-mL tubes containing lysis matrix B (MP Biomedicals). After incubation for 15 min on ice, the spores were lysed using FastPrep (MP Biomedicals) at 6.5 m/sec for 60 sec. An equal volume of 2× sample buffer (4% SDS, 250 mM Tris at pH 6.8, 20% glycerol, 10 mM EDTA, Bromophenol blue) containing 10% β-mercaptoethanol was immediately added to the tubes and vortexed. After centrifugation at 15,000 rpm for 5 min, the supernatant was collected and total protein was quantified by a noninterfering protein assay (G-Biosciences). The normalized protein samples were resolved by SDS-PAGE on 17.5% polyacrylamide gels and transferred to an Immobilon-P membrane (Millipore).Vegetative cell lysates were generated from B. subtilis cultures grown in LB medium. Briefly, cells were harvested at an OD600 of 0.5–0.8 and normalized to an OD600 of 0.5. One milliliter of normalized culture was centrifuged, and the cell pellet was resuspended in 50 µL of lysis buffer (20 mM Tris at pH 7.5, 10 mM EDTA, 1 mg/mL lysozyme [Sigma-Aldrich], 1 mM PMSF, 10 µg/mL DNase I [NEB], 100 µg/mL RNase A [NEB], 10 µg/mL leupeptin [Fisher Scientific], 10 µg/mL pepstatin [Fisher Scientific]) and incubated for 10 min at 37°C, followed by addition of an equal volume of 2× sample buffer containing 10% β-mercaptoethanol. The lysates were resolved by SDS-PAGE on 17.5% acrylamide gels and transferred to an Immobilon-P membrane (Millipore).The PVDF membranes were blocked in 5% nonfat milk dissolved in 1× PBS with 0.5% Tween-20 (PBST) and then probed with anti-His (1:4000; GenScript), anti-ProC (1:1000) (Gao et al. 2023), anti-FLAG (1:5000; Sigma-Aldrich), anti-SigA (1:10,000) (Fujita and Sadaie 1998), anti-EzrA (1:10,000) (Levin et al. 1999), anti-GerBC (1:5000) (Stewart et al. 2012), or anti-SpoVAD (1:10,000) (Vepachedu and Setlow 2005) antibodies diluted in 3% BSA dissolved in PBST. After washing in PBST, the primary antibodies were detected with goat antirabbit or antimouse antibodies coupled to horseradish peroxidase (Bio-Rad) and detected and visualized by Western Lightning ECL reagent (PerkinElmer).

Lysates and immunoblot analysis

Spore lysates were generated from spores purified on a histodenz step gradient or by lysozyme-SDS as described above. Spores were resuspended in 500 µL of cold PBS supplemented with 1 mM phenylmethylsulfonyl fluoride (PMSF) and transferred to 2-mL tubes containing lysis matrix B (MP Biomedicals). After incubation for 15 min on ice, the spores were lysed using FastPrep (MP Biomedicals) at 6.5 m/sec for 60 sec. An equal volume of 2× sample buffer (4% SDS, 250 mM Tris at pH 6.8, 20% glycerol, 10 mM EDTA, Bromophenol blue) containing 10% β-mercaptoethanol was immediately added to the tubes and vortexed. After centrifugation at 15,000 rpm for 5 min, the supernatant was collected and total protein was quantified by a noninterfering protein assay (G-Biosciences). The normalized protein samples were resolved by SDS-PAGE on 17.5% polyacrylamide gels and transferred to an Immobilon-P membrane (Millipore).

Vegetative cell lysates were generated from B. subtilis cultures grown in LB medium. Briefly, cells were harvested at an OD600 of 0.5–0.8 and normalized to an OD600 of 0.5. One milliliter of normalized culture was centrifuged, and the cell pellet was resuspended in 50 µL of lysis buffer (20 mM Tris at pH 7.5, 10 mM EDTA, 1 mg/mL lysozyme [Sigma-Aldrich], 1 mM PMSF, 10 µg/mL DNase I [NEB], 100 µg/mL RNase A [NEB], 10 µg/mL leupeptin [Fisher Scientific], 10 µg/mL pepstatin [Fisher Scientific]) and incubated for 10 min at 37°C, followed by addition of an equal volume of 2× sample buffer containing 10% β-mercaptoethanol. The lysates were resolved by SDS-PAGE on 17.5% acrylamide gels and transferred to an Immobilon-P membrane (Millipore).

The PVDF membranes were blocked in 5% nonfat milk dissolved in 1× PBS with 0.5% Tween-20 (PBST) and then probed with anti-His (1:4000; GenScript), anti-ProC (1:1000) (Gao et al. 2023), anti-FLAG (1:5000; Sigma-Aldrich), anti-SigA (1:10,000) (Fujita and Sadaie 1998), anti-EzrA (1:10,000) (Levin et al. 1999), anti-GerBC (1:5000) (Stewart et al. 2012), or anti-SpoVAD (1:10,000) (Vepachedu and Setlow 2005) antibodies diluted in 3% BSA dissolved in PBST. After washing in PBST, the primary antibodies were detected with goat antirabbit or antimouse antibodies coupled to horseradish peroxidase (Bio-Rad) and detected and visualized by Western Lightning ECL reagent (PerkinElmer).

Structural modeling using AlphaFold-MultimerAll predicted structures were generated using AlphaFold-Multimer-v2 or AlphaFold-Multimer-v3 and ColabFold (Jumper et al. 2021; Evans et al. 2022), run locally on the Harvard Medical School O2 computing cluster or the Lambda Cloud computing server. General parameters were as follows: Multiple sequence alignments (MSA) were built using mmseqs2. Sequences from the same operon were paired in the MSA, and both paired and unpaired sequences were used to generate models. Five models were generated in each run, and each model was relaxed using AMBER and ranked by pTM score. Homologous templates found in the PDB were used. For the SpoVAF pentamer and the SpoVAF-FigP pentamer of dimers, the maximum number of model recycling was set to 12, and the top-ranked models were used throughout the text.

Structural modeling using AlphaFold-Multimer

All predicted structures were generated using AlphaFold-Multimer-v2 or AlphaFold-Multimer-v3 and ColabFold (Jumper et al. 2021; Evans et al. 2022), run locally on the Harvard Medical School O2 computing cluster or the Lambda Cloud computing server. General parameters were as follows: Multiple sequence alignments (MSA) were built using mmseqs2. Sequences from the same operon were paired in the MSA, and both paired and unpaired sequences were used to generate models. Five models were generated in each run, and each model was relaxed using AMBER and ranked by pTM score. Homologous templates found in the PDB were used. For the SpoVAF pentamer and the SpoVAF-FigP pentamer of dimers, the maximum number of model recycling was set to 12, and the top-ranked models were used throughout the text.

Data availabilityAll data are available here and in the Supplementary Material.

Data availability

All data are available here and in the Supplementary Material.

Supplementary MaterialSupplement 1

Supplementary Material
