---
pmid: '22500027'
title: Intrinsic protein-protein interaction-mediated and chaperonin-assisted sequential
  assembly of stable bardet-biedl syndrome protein complex, the BBSome.
authors:
- Zhang Q
- Yu D
- Seo S
- Stone EM
- Sheffield VC
journal: J Biol Chem
year: '2012'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3370246
doi: 10.1074/jbc.M112.341487
---

# Intrinsic protein-protein interaction-mediated and chaperonin-assisted sequential assembly of stable bardet-biedl syndrome protein complex, the BBSome.
**Authors:** Zhang Q, Yu D, Seo S, Stone EM, Sheffield VC
**Journal:** J Biol Chem (2012)
**DOI:** [10.1074/jbc.M112.341487](https://doi.org/10.1074/jbc.M112.341487)
**PMC:** [PMC3370246](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3370246/)

## Abstract

1. J Biol Chem. 2012 Jun 8;287(24):20625-35. doi: 10.1074/jbc.M112.341487. Epub 
2012 Apr 12.

Intrinsic protein-protein interaction-mediated and chaperonin-assisted 
sequential assembly of stable bardet-biedl syndrome protein complex, the BBSome.

Zhang Q(1), Yu D, Seo S, Stone EM, Sheffield VC.

Author information:
(1)Department of Pediatrics, Division of Medical Genetics and Howard Hughes 
Medical Institute, University of Iowa, Iowa City 52242, USA.

The pleiotropic features of obesity, retinal degeneration, polydactyly, kidney 
abnormalities, cognitive impairment, hypertension, and diabetes found in 
Bardet-Biedl syndrome (BBS) make this disorder an important model disorder for 
identifying molecular mechanisms involved in common human diseases. To date, 16 
BBS genes have been reported, seven of which (BBS1, 2, 4, 5, 7, 8, and 9) code 
for proteins that form a complex known as the BBSome. The function of the BBSome 
involves ciliary membrane biogenesis. Three additional BBS genes (BBS6, BBS10, 
and BBS12) have homology to type II chaperonins and interact with CCT/TRiC 
proteins and BBS7 to form a complex termed the BBS-chaperonin complex. This 
complex is required for BBSome assembly. Little is known about the process and 
the regulation of BBSome formation. We utilized point mutations and null alleles 
of BBS proteins to disrupt assembly of the BBSome leading to the accumulation of 
BBSome assembly intermediates. By characterizing BBSome assembly intermediates, 
we show that the BBS-chaperonin complex plays a role in BBS7 stability. BBS7 
interacts with BBS2 and becomes part of a BBS7-BBS2-BBS9 assembly intermediate 
referred to as the BBSome core complex because it forms the core of the BBSome. 
BBS1, BBS5, BBS8, and finally BBS4 are added to the BBSome core to form the 
complete BBSome.

DOI: 10.1074/jbc.M112.341487
PMCID: PMC3370246
PMID: 22500027 [Indexed for MEDLINE]

## Full Text

IntroductionMacromolecular complexes, including the ribosome, the 26 S proteasome, the nuclear pore complex, and intraflagellar transport complexes, are responsible for essential functions, leading to a broad interest in their characterization and the process by which they are assembled (1–6). Protein complex assembly provides an important, if underappreciated, means by which cells can regulate function. Moreover, better understanding of protein complex assembly can provide insight into genetically heterogeneous and multigenic diseases and can potentially provide a novel means of pharmacological modification of function, thereby resulting in new drug designs.The pleiotropic features of Bardet-Biedl syndrome (BBS)4 including obesity, retinal degeneration, polydactyly, kidney abnormalities, cognitive impairment, hypertension, and diabetes make BBS an appealing model for identifying molecular mechanisms involved in common diseases (7, 8). To date, 16 BBS-causing genes have been identified (9). Seven of the BBS proteins (BBS1, 2, 4, 5, 7, 8, and 9) have been shown to form a complex known as the BBSome (10). The function of the BBSome involves ciliary membrane biogenesis through the small GTPase Rab8 and its interacting protein, Rabin8. The BBSome also genetically interacts with the intraflagellar transport pathway to regulate sonic hedgehog signal transduction (11). BBSome subunits contain domains known to mediate protein-protein interactions. Specifically, BBS1, BBS2, BBS7, and BBS9 contain β propeller domains; BBS4 and BBS8 contain multiple tetratricopeptide repeat domains; and BBS5 contains two pleckstrin homology domains. BBS3/ARL6 (ADP-ribosylation like factor 6) is a member of the Ras superfamily of small GTP-binding proteins and is required for the ciliary localization of the BBSome (12–15). BBS6, BBS10, and BBS12 have type II chaperonin homology (16–19). Previous studies have demonstrated that BBS6, BBS12, and BBS7 interact with CCT/TRiC proteins to form a complex that we will hereafter refer to as the BBS-chaperonin complex. The BBS-chaperonin complex facilitates BBSome assembly (20). Little is known about the sequential process of BBSome formation, the regulation of this process, or the transition of BBS7 from the BBS-chaperonin complex to the BBSome. In vivo, BBSome proteins are found abundantly in the BBSome complex as opposed to being in the individual free form or within smaller complexes (10), suggesting that BBSome assembly intermediates are transient and short-lived.Understanding the process of BBSome assembly may aid in identifying new molecular components, mutations in which either cause or modify BBS phenotypes. In addition, understanding the process of BBSome formation may provide information about individual subunit function, therefore providing novel insights regarding BBSome function. Furthermore, characterization of BBSome assembly will provide efficient cell-based assays to evaluate point mutations identified in patients. Finally, understanding the process of BBSome assembly may lead to novel treatments for BBS and BBS-related phenotypes.We used multiple approaches to identify BBSome assembly intermediates including introducing point mutations found in BBS patients into individual BBS proteins, the utilization of knock-out mice, and the use of RNAi against BBS genes to disrupt BBSome formation. In addition, we overexpressed specific BBSome subunits that resulted in the assembly and accumulation of distinct BBSome intermediates. By characterizing these intermediates, we demonstrate that BBSome assembly is an ordered process involving the chaperonin-like BBS proteins (BBS6, BBS10, and BBS12) and intrinsic protein-protein interactions.

Introduction

Macromolecular complexes, including the ribosome, the 26 S proteasome, the nuclear pore complex, and intraflagellar transport complexes, are responsible for essential functions, leading to a broad interest in their characterization and the process by which they are assembled (1–6). Protein complex assembly provides an important, if underappreciated, means by which cells can regulate function. Moreover, better understanding of protein complex assembly can provide insight into genetically heterogeneous and multigenic diseases and can potentially provide a novel means of pharmacological modification of function, thereby resulting in new drug designs.

The pleiotropic features of Bardet-Biedl syndrome (BBS)4 including obesity, retinal degeneration, polydactyly, kidney abnormalities, cognitive impairment, hypertension, and diabetes make BBS an appealing model for identifying molecular mechanisms involved in common diseases (7, 8). To date, 16 BBS-causing genes have been identified (9). Seven of the BBS proteins (BBS1, 2, 4, 5, 7, 8, and 9) have been shown to form a complex known as the BBSome (10). The function of the BBSome involves ciliary membrane biogenesis through the small GTPase Rab8 and its interacting protein, Rabin8. The BBSome also genetically interacts with the intraflagellar transport pathway to regulate sonic hedgehog signal transduction (11). BBSome subunits contain domains known to mediate protein-protein interactions. Specifically, BBS1, BBS2, BBS7, and BBS9 contain β propeller domains; BBS4 and BBS8 contain multiple tetratricopeptide repeat domains; and BBS5 contains two pleckstrin homology domains. BBS3/ARL6 (ADP-ribosylation like factor 6) is a member of the Ras superfamily of small GTP-binding proteins and is required for the ciliary localization of the BBSome (12–15). BBS6, BBS10, and BBS12 have type II chaperonin homology (16–19). Previous studies have demonstrated that BBS6, BBS12, and BBS7 interact with CCT/TRiC proteins to form a complex that we will hereafter refer to as the BBS-chaperonin complex. The BBS-chaperonin complex facilitates BBSome assembly (20). Little is known about the sequential process of BBSome formation, the regulation of this process, or the transition of BBS7 from the BBS-chaperonin complex to the BBSome. In vivo, BBSome proteins are found abundantly in the BBSome complex as opposed to being in the individual free form or within smaller complexes (10), suggesting that BBSome assembly intermediates are transient and short-lived.

Understanding the process of BBSome assembly may aid in identifying new molecular components, mutations in which either cause or modify BBS phenotypes. In addition, understanding the process of BBSome formation may provide information about individual subunit function, therefore providing novel insights regarding BBSome function. Furthermore, characterization of BBSome assembly will provide efficient cell-based assays to evaluate point mutations identified in patients. Finally, understanding the process of BBSome assembly may lead to novel treatments for BBS and BBS-related phenotypes.

We used multiple approaches to identify BBSome assembly intermediates including introducing point mutations found in BBS patients into individual BBS proteins, the utilization of knock-out mice, and the use of RNAi against BBS genes to disrupt BBSome formation. In addition, we overexpressed specific BBSome subunits that resulted in the assembly and accumulation of distinct BBSome intermediates. By characterizing these intermediates, we demonstrate that BBSome assembly is an ordered process involving the chaperonin-like BBS proteins (BBS6, BBS10, and BBS12) and intrinsic protein-protein interactions.

EXPERIMENTAL PROCEDURESFor a detailed description of materials and methods, see supplemental “Materials and Methods.”Sucrose Gradient FractionationThe tissues were disrupted by TISSUEMISER (Fisher) in lysis buffer (1- PBS, 0.5% Triton X-100 with protease inhibitors). The lysates were centrifuged at 20,000 × g for 20 min. The supernatants were loaded onto a 20–60% sucrose gradient. The gradient was centrifuged at 100,000 × g for 14 h using a TH-660 rotor Thermo Scientific (Asheville, NC). Two hundred-microliter fractions were taken from the top and precipitated by cold acetone. Precipitated samples were spun at 20,000 × g for 15 min. The pellets were dissolved in SDS-PAGE sample buffer.Protease Sensitivity AssayThe cells were lysed in assay buffer (50 mm Tris-HCl, pH 8.0, 50 mm KCl, 10 mm CaCl2, 10% glycerol). Equal amount of proteins were mixed with 200 μg/ml thermolysin at 4 °C for the times indicated, and the reactions were stopped by the addition of EDTA to achieve a final concentration of 10 mm.

EXPERIMENTAL PROCEDURES

For a detailed description of materials and methods, see supplemental “Materials and Methods.”

Sucrose Gradient FractionationThe tissues were disrupted by TISSUEMISER (Fisher) in lysis buffer (1- PBS, 0.5% Triton X-100 with protease inhibitors). The lysates were centrifuged at 20,000 × g for 20 min. The supernatants were loaded onto a 20–60% sucrose gradient. The gradient was centrifuged at 100,000 × g for 14 h using a TH-660 rotor Thermo Scientific (Asheville, NC). Two hundred-microliter fractions were taken from the top and precipitated by cold acetone. Precipitated samples were spun at 20,000 × g for 15 min. The pellets were dissolved in SDS-PAGE sample buffer.Protease Sensitivity AssayThe cells were lysed in assay buffer (50 mm Tris-HCl, pH 8.0, 50 mm KCl, 10 mm CaCl2, 10% glycerol). Equal amount of proteins were mixed with 200 μg/ml thermolysin at 4 °C for the times indicated, and the reactions were stopped by the addition of EDTA to achieve a final concentration of 10 mm.

Sucrose Gradient FractionationThe tissues were disrupted by TISSUEMISER (Fisher) in lysis buffer (1- PBS, 0.5% Triton X-100 with protease inhibitors). The lysates were centrifuged at 20,000 × g for 20 min. The supernatants were loaded onto a 20–60% sucrose gradient. The gradient was centrifuged at 100,000 × g for 14 h using a TH-660 rotor Thermo Scientific (Asheville, NC). Two hundred-microliter fractions were taken from the top and precipitated by cold acetone. Precipitated samples were spun at 20,000 × g for 15 min. The pellets were dissolved in SDS-PAGE sample buffer.Protease Sensitivity AssayThe cells were lysed in assay buffer (50 mm Tris-HCl, pH 8.0, 50 mm KCl, 10 mm CaCl2, 10% glycerol). Equal amount of proteins were mixed with 200 μg/ml thermolysin at 4 °C for the times indicated, and the reactions were stopped by the addition of EDTA to achieve a final concentration of 10 mm.

Sucrose Gradient FractionationThe tissues were disrupted by TISSUEMISER (Fisher) in lysis buffer (1- PBS, 0.5% Triton X-100 with protease inhibitors). The lysates were centrifuged at 20,000 × g for 20 min. The supernatants were loaded onto a 20–60% sucrose gradient. The gradient was centrifuged at 100,000 × g for 14 h using a TH-660 rotor Thermo Scientific (Asheville, NC). Two hundred-microliter fractions were taken from the top and precipitated by cold acetone. Precipitated samples were spun at 20,000 × g for 15 min. The pellets were dissolved in SDS-PAGE sample buffer.

Sucrose Gradient Fractionation

The tissues were disrupted by TISSUEMISER (Fisher) in lysis buffer (1- PBS, 0.5% Triton X-100 with protease inhibitors). The lysates were centrifuged at 20,000 × g for 20 min. The supernatants were loaded onto a 20–60% sucrose gradient. The gradient was centrifuged at 100,000 × g for 14 h using a TH-660 rotor Thermo Scientific (Asheville, NC). Two hundred-microliter fractions were taken from the top and precipitated by cold acetone. Precipitated samples were spun at 20,000 × g for 15 min. The pellets were dissolved in SDS-PAGE sample buffer.

Protease Sensitivity AssayThe cells were lysed in assay buffer (50 mm Tris-HCl, pH 8.0, 50 mm KCl, 10 mm CaCl2, 10% glycerol). Equal amount of proteins were mixed with 200 μg/ml thermolysin at 4 °C for the times indicated, and the reactions were stopped by the addition of EDTA to achieve a final concentration of 10 mm.

Protease Sensitivity Assay

The cells were lysed in assay buffer (50 mm Tris-HCl, pH 8.0, 50 mm KCl, 10 mm CaCl2, 10% glycerol). Equal amount of proteins were mixed with 200 μg/ml thermolysin at 4 °C for the times indicated, and the reactions were stopped by the addition of EDTA to achieve a final concentration of 10 mm.

RESULTSBBS2, BBS7, and BBS9 Form Ternary Assembly IntermediatePrevious studies have demonstrated that BBS7, which is an integral part of the BBSome, interacts with the BBS-chaperonin complex (20). The fact that all components of the BBS-chaperonin complex are chaperonin type proteins except for BBS7 suggests that BBS7 may serve as a substrate for the BBS-chaperonin complex and be stabilized by the complex. How BBS7 makes the transition from the BBS-chaperonin complex to the BBSome complex is currently unknown. To address this question, we utilized several approaches that result in the accumulation of distinct BBSome assembly intermediates. Point mutations utilized in this study are summarized in supplemental Fig. S1. When we overexpress FLAG-tagged BBS7 containing the point mutation T211I in 293T cells, the mutant BBS7T211I has a decreased ability to interact with endogenous BBSome subunits compared with WT BBS7 as determined by co-immunoprecipitation using FLAG antibody. However, immunoprecipitation of BBS7T211I pulls down relatively more BBS2 than other BBSome proteins (e.g., BBS1 or BBS4) (Fig. 1A), indicating that there is a binary interaction between BBS7T211I and BBS2 that is trapped (does not proceed to the next step in assembly).FIGURE 1.BBS2, BBS7, and BBS9 form the BBSome core complex.
A, FLAG-tagged BBS7 containing a point mutation (T211I) and WT BBS7 were transfected into 293T cells. Both WT and mutant BBS7 were pulled down by FLAG beads. The existence of other endogenous BBSome subunits is detected by Western blot. B, the absence of BBS2 affects the interaction of BBS7 with BBS1, whereas the loss of BBS1 does not affect the interaction of BBS7 with BBS2. C, FLAG-tagged BBS5, BBS8, BBS9, and BBIP10 were transfected into 293T cells. Tagged proteins were immunoprecipitated (IP) by FLAG beads, and the existence of other endogenous BBSome subunits was detected by Western blot. D, BBS9 RNAi was used to knock down endogenous BBS9 expression in a stable cell line expressing FLAG-BBS2. The BBSome was immunoprecipitated by FLAG beads, and the existence of endogenous BBSome subunits was detected by Western blot.We next knocked down BBS2 to determine the effect this has on BBSome assembly. Knockdown of BBS2 decreases the association of BBS7 with BBS1, whereas knockdown of BBS1 does not affect the association of BBS7 with BBS2 (Fig. 1B). This indicates that BBS2 and BBS7 can associate with each other independent of BBS1. Similar results showing stable interaction of BBS2 with BBS7 are obtained with tissue from mutant mice that express a mutant form of BBS1 with an arginine for methionine mutation at amino acid 390 (Bbs1M390) (Fig. 2C). This finding further supports the concept that BBS7 binds to BBS2 independent of binding to BBS1 and other BBSome proteins. Consistent with this result, BBS7 interacts strongly with BBS2 and weakly with BBS1 in pairwise interaction assays (supplemental Fig. S2A).FIGURE 2.BBS1 is incorporated into the BBSome before BBS4.
A, Western blots of protein lysates from WT and Bbs1M390R/M390R mice demonstrate that the absence of BBS1 does not affect other BBSome subunits protein levels, whereas BBS1 protein levels are greatly decreased in samples from Bbs1M390R/M390R mice as indicated by the arrow. This antibody also recognizes several additional bands that do not change in intensity when compared between WT and Bbs1M390R/M390R mice, indicating that these bands are nonspecific. B, sucrose gradient analysis of BBSome formation in Bbs1M390R testes demonstrates that most BBS4 is dissociated from the subcomplex consisting of BBS2, BBS5, BBS7, BBS8, and BBS9. C, immunoprecipitation (IP) using antibody against BBS2 from testes protein lysates shows that BBS4 is dissociated from the subcomplex in the absence of BBS1. These results are similar to those shown by sucrose gradient analysis. D, reciprocal immunoprecipitation from transfected 293T cells demonstrates that BBS4 dissociation from BBS1M390R is due to decreased interaction between BBS1M390R and BBS4 compared with WT BBS1. E, diagram showing the sequential order of BBS1 and BBS4 incorporation into the BBSome.Next, we overexpressed FLAG-tagged BBS9 in 293T cells and used BBS9 to pull down endogenous BBSome subunits. Immunoprecipitation of exogenously expressed BBS9 results in immunoprecipitation of substantial amounts of BBS2 and BBS7, whereas only small amounts of BBS1 and BBS4 are pulled down under these conditions (Fig. 1C). This result indicates that BBS9 forms a ternary complex with BBS2 and BBS7. To further support the existence of a BBS2-BBS7-BBS9 ternary complex, we knock down BBS9 expression using RNAi. Knockdown of BBS9 does not affect the association of BBS7 with BBS2, indicating that the BBS7/BBS2 interaction is not dependent on BBS9. However, knockdown of BBS9 greatly decreases the association of BBS1, BBS4, and BBS8 with BBS2 (Fig. 1D).Consistent with this result, pairwise interaction studies show that BBS2 directly interacts with BBS7 and BBS9, whereas BBS7 and BBS9 do not directly interact, rather they only interact in the presence of BBS2 (supplemental Fig. S2B). We refer to the ternary complex consisting of BBS2-BBS7-BBS9 as the BBSome core complex because this complex appears to be a central component required for the formation of the complete BBSome.BBS1 and BBS4 Are Peripheral Subunits of BBSomeOur pairwise interaction studies showed that BBS9 directly interacts with several BBSome subunits including BBS2, BBS1, BBS5, and BBS8, establishing BBS9 as a central scaffold of the BBSome (10). To determine the interdependence with which other BBSome subunits are incorporated into the BBSome, we utilized tissues from BBS mutant mice generated in our laboratory (21–24). Western blotting of testicular tissue (which expresses large amounts of BBS proteins) showed that Bbs1M390R/M390R mice have decreased BBS1 protein levels, but other BBSome subunits protein levels are similar to WT levels (Fig. 2A). Sucrose gradient analysis reveals that the BBS1M390R mutation associates normally with all BBSome proteins, except for BBS4 (Fig. 2B). The majority of BBS4 is dissociated from the BBSome in Bbs1M390R/M390R mutant mouse tissue. To confirm this result, we immunoprecipitated the endogenous BBSome from Bbs1M390R/M390R mutant mice using antibody against BBS2. As expected, BBS1 is missing from the complex along with BBS4 (Fig. 2C).To confirm that BBS1M390R mutant protein affects BBS4 incorporation into the BBSome, we co-transfected Myc-tagged BBS1 with FLAG-tagged BBSome subunits into 293T cells. BBS1M390R mutant protein has greatly reduced interaction with BBS4 (Fig. 2D), but normal interactions with BBS9 and BBS2. Taken together, these data indicate that BBS1 is incorporated into the BBSome before BBS4 (Fig. 2E). Using a similar approach, we demonstrate that the loss of BBS4 does not affect the incorporation of other BBSome subunits (including BBS1) into the BBSome (Fig. 3, A and B), indicating that BBS4 is the last subunit to be added to the BBSome (Fig. 3C).FIGURE 3.BBS4 is likely the last subunit to incorporate into the BBSome.
A, sucrose gradient analysis of BBSome formation in Bbs4−/− testes lysates demonstrates that all other BBSome subunits can still form a complex in the absence of BBS4. B, immunoprecipitation using anti-BBS2 antibody from testes protein lysates indicates that BBS4 is the last subunit to incorporate into the BBSome. C, diagram showing the sequential order of BBS components incorporation into the BBSome.To determine whether the absence of other BBS proteins allows for partial BBSome assembly, we knocked down expression of BBS5 and BBS8 using RNAi in RPE1 cells. The results reveal that other BBSome subunits can still incorporate into the BBSome in the absence of BBS5, as well as in the absence of BBS8 (25), indicating that like BBS1, BBS5 and BBS8 are incorporated independently into the BBSome. These findings are consistent with the pairwise interaction data showing that BBS1, BBS5, BBS8, and BBS2 directly interact with BBS9 (10).BBS4 Binds to PCM1 and Localizes to Centrosome/Pericentriolar Satellites Independent of Other BBSome SubunitsThe fact that BBS4 is among the last subunits to be incorporated into the BBSome puts it in a favorable position to interact with non-BBSome proteins including BBSome cargos. Previous studies have demonstrated that the BBSome moves from the cytoplasm to pericentriolar satellites then to cilia. BBS4 localizes to centrosome/pericentriolar satellites and interacts with the C terminus of PCM1 (10, 26). The functional relevance of this interaction is not clear. To determine whether the BBS4-PCM1 interaction is required for centrosome/pericentriolar satellite localization of BBS4, GFP-tagged WT and two mutant forms of BBS4 (BBS4G277E and BBS4R295P) were independently co-transfected with a HA-tagged C-terminal fragment of PCM1. Each of the two point mutations disrupted the interaction of BBS4 with PCM1 (Fig. 4A). When transfected into human RPE1 cells, both BBS4G277E and BBS4R295P lose the ability to localize to centrosome/pericentriolar satellites (Fig. 4B). These data suggest that the BBS4-PCM1 interaction is required for BBS4 centrosome/pericentriolar satellite localization.FIGURE 4.BBS4-PCM1 interaction is required for the BBSome to localize to centrosome/pericentriolar satellite.
A, BBS4 point mutations (G277E and R295P) disrupt the interaction between BBS4 and the C terminus of PCM1. GFP-tagged BBS4 (WT or mutants) and HA-tagged PCM1 C terminus were co-transfected into 293T cells, and co-immunoprecipitation (IP) was performed using antibody against HA and GFP. B, BBS4 point mutations (G277E and R295P) disrupt centrosome/pericentriolar satellite localization of BBS4. Shown is immunofluorescent staining of GFP-BBS4 (WT and mutants) in transfected RPE1 cells. γ-Tubulin (red) is used as a centrosomal marker. C, kidney primary cells were transfected with GFP-BBS4, and transfected cells were co-stained with γ-tubulin. The bar represents 10 μm.To determine whether the intact BBSome is required for BBS4 to localize to centrosome/pericentriolar satellites, we transfected cultured kidney cells from BBS knock-out mice with GFP-BBS4. Absence of BBS2, BBS6, or BBS7 does not affect BBS4 centrosome/pericentriolar satellite localization (Fig. 4C), indicating that this localization is an intrinsic feature of free BBS4 (occurs independent of the BBSome). To further characterize BBS4 interactions, we mapped the BBS4 domain that interacts with PCM1 to the N-terminal 380 amino acids (supplemental Fig. S3A). The N-terminal 380 amino acids of BBS4 do not contain the domain that binds with BBIP10, an integral BBSome protein that binds to the complex through BBS4 (27). Furthermore, we show that PCM1 can interact with BBIP10 only when BBS4 is present (supplemental Fig. S3B).BBS10 Regulates Interaction of BBS6-BBS12-BBS7 with CCT/TRiC Proteins to Form BBS-Chaperonin ComplexAs mentioned previously, studies showed that BBS7 form a complex with BBS6, BBS12, and CCT/TRiC proteins to form the BBS-chaperonin complex (20). The fact that BBS7 is the only non-chaperonin component in the BBS-chaperoinin complex suggests that BBS7 may be the substrate for BBS-chaperonin. One possible function of the BBS-chaperonin complex is the stabilization of BBS7. BBS10 is associated with this complex at substoichiometric levels (less than a 1:1 ratio). This finding suggests that BBS10 is not a structural component of the BBS-chaperonin complex but rather plays a role in regulating complex formation. To investigate the role of BBS10 in BBS-chaperonin complex formation, we utilized RNAi to knock down BBS10 in 293T cells expressing FLAG-BBS6, which serves as a convenient tool for immunoprecipitation of the BBS-chaperonin complex. Absence of BBS10 decreases the interaction of BBS6 with TCPα and TCPβ, two CCT/TRiC proteins (Fig. 5A). Furthermore, overexpression of BBS10 promotes the interaction of BBS6 with TCPα and TCPβ (Fig. 5B). These results demonstrate that BBS10 regulates the formation of the BBS-chaperonin complex.FIGURE 5.BBS10 is required for the interaction of BBS6-BBS12-BBS7 with the CCT/TRiC proteins to form the BBS-chaperonin complex.
A, BBS10 RNAi is used to knock down the expression of BBS10 in 293T cells expressing FLAG-BBS6, which was used to pull down the BBS-chaperonin complex. Antibodies against BBS7 and the CCT/TRiC proteins were used to detect the existence of these proteins in the complex. B, overexpression of BBS10 facilitates the association of BBS6 with the CCT/TRiC proteins. FLAG-BBS10 was co-transfected with empty vector or Myc-BBS6, and Myc antibody was used to pull down the BBS-chaperonin complex. Antibodies against BBS7, TCPα, and TCPβ (two subunits of the CCT/TRiC proteins) were used to detect these proteins in the complex. IP, immunoprecipitation.Release of BBS7 from BBS-Chaperonin Complex Is Coordinated with BBSome Core Complex FormationThe mature form of the BBSome does not contain BBS6, BBS12, or CCT/TRiC proteins, suggesting these chaperonin components must be dissociated from assembly intermediates at some point. To determine when these components are released from the assembly intermediates, we overexpressed FLAG-tagged BBS6 in 293T cells and immunoprecipitated with FLAG antibody. As expected, other BBS-chaperonin complex components, as well as endogenous BBS7 were co-immunoprecipitated along with BBS6 (Fig. 6A). In addition, BBS6 pulled down some BBS2. However, the amount of BBS2 pulled down relative to BBS7 was less than the amounts of BBS2 found in the BBSome (as shown by pulling down the BBSome with BBS5). This indicates that the interaction of BBS2 with BBS7 is coupled with the release of BBS6 and BBS12 from the BBS-chaperonin complex (supplemental Fig. S4).FIGURE 6.The release of BBS7 from the BBS-chaperonin complex is coordinated with BBSome core complex formation.
A, FLAG-BBS6 was overexpressed in 239T cells to trap BBSome intermediates. We pulled down BBS6 using FLAG beads and detected endogenous BBSome subunits by Western blot. FLAG-BBS5 was used as a positive control, and FLAG-GFP was used as a negative control. B, individual BBSome subunits were overexpressed in 293T cells and were used to determine specific associations with the CCT/TRiC proteins. Only BBS7 and BBS2 can associate with CCT/TRiC proteins. FLAG-VHL was used as a positive control. FLAG-GFP and FLAG-BBS3 were used as negative controls. IP, immunoprecipitation.Next we examined the release of the CCT/TRiC proteins from the assembly intermediates. We overexpressed each BBSome subunit in 293T cells and determined whether any BBSome subunit can immunoprecipitate the CCT/TRiC proteins. BBS2 and BBS7 showed robust pulldown of CCT/TRiC proteins. BBS9 showed relatively less pulldown of the CCT/TRiC proteins compared with BBS2 and BBS7 (Fig. 6B). This result indicates that the incorporation of BBS9 to form the BBSome core complex is coupled with the release of BBS2/BBS7 from CCT/TRiC proteins.Stability of BBS2 Is Dependent on Its Interaction with Other ProteinsWe have demonstrated that BBS2, BBS7, and BBS9 form a core complex (Fig. 1). To determine the importance of this complex, we examined the protein level of BBS2 in the absence of BBS7 and BBS9, as well as in the absence of the BBS-chaperonin complex. Absence of BBS7 greatly decreases BBS2 protein levels in testes tissue from Bbs7−/− mice. Furthermore, the absence of BBS6 (a component of the BBS-chaperonin complex) in Bbs6−/− mice also results in decreased amounts of BBS2 (Fig. 7A). Because we do not have Bbs10 knock-out mice, we utilize skin fibroblast cells from a human patient who is homozygous for the common BBS10 mutation, c91fs95. This mutation is predicted to result in the total absence of the BBS10 protein. In our study, using human fibroblast cells, the absence of BBS10 does not affect cilia formation (supplemental Fig. S5), in contrast to a previous report (28). However, the absence of BBS10 lowers BBS2 protein levels (Fig. 7A) and inhibits BBSome formation (supplemental Fig. S6).FIGURE 7.BBS2 is ubiquitinated and degraded by the proteasome.
A, the stability of BBS2 is dependent on the BBSome core complex, as well as the BBS-chaperonin complex. Proteins from BBS knock-out mouse tissue (testis) or from RNAi-treated cell lysates were separated by SDS-PAGE. Anti-BBS2 antibody was used to detect BBS2 protein levels. B, BBS2 is sensitive to protease treatment. Primary cells from WT and Bbs6−/− kidney were infected with adenovirus expressing FLAG-BBS2. Equal amounts of total proteins were mixed with protease thermolysin at 4 °C for the indicated time. Compared with WT cells, BBS2 degrades faster in Bbs6−/− cells, suggesting that BBS2 is not stable in the absence of BBS6. C, 293T cells and FLAG-BBS2 stable-expressing cells were treated with the proteasome inhibitor MG132. BBS2 was immunoprecipitated (IP) by FLAG beads or anti-BBS2 antibody, and ubiquitinated BBS2 was detected by anti-ubiquitin antibody. D, HA-ubiquitin was transfected into 293T cells, and endogenous BBS2 was immunoprecipitated by anti-BBS2 antibody. Ubiquitinated BBS2 was detected by anti-HA antibody or anti-ubiquitin antibody. E, 293T cells were treated with 50 μg/ml of cycloheximide (Chx) with or without 20 μm proteasome inhibitor MG132 or MG115 for 8 h. Endogenous BBS2 in the cell lysates was detected by anti-BBS2 antibody.Next, we used RNAi in RPE1 cells to individually reduce expression of BBS9 (BBSome core complex component), BBS12 and TCPβ (BBS-chaperonin complex components). Reduction of each of these proteins greatly decreases BBS2 protein levels. In contrast, the absence of the normal BBSome proteins BBS4 or BBS1 (using tissue from Bbs4−/− and Bbs1M390R mutant mice, respectively) does not affect BBS2 protein levels (Fig. 7A). These data indicate that the stability of BBS2, a component of the BBSome core complex, requires the BBS-chaperonin complex, as well as the other two components of the BBSome core complex (BBS7 and BBS9). Similarly, the stability of BBS7 requires the BBS-chaperonin complex, as well as the other two components of the BBSome core complex (BBS2 and BBS9) (supplemental Fig. S7).To determine how the BBS-chaperonin complex influences BBS2 protein stability, we infected Bbs6−/− kidney cells and WT kidney cells with adenovirus expressing FLAG-BBS2. The same multiplicity of infection was used to infect WT and Bbs6−/− kidney cells, and equal amounts of proteins were used for the analysis. The BBS2 protein level is lower in Bbs6−/− cells than in WT cells (Fig. 7B, 0'), consistent with the finding that the BBS-chaperonin complex is required for BBS2 protein stability. The remaining BBS2 protein in Bbs6−/− cells is more sensitive to protease thermolysin treatment. The half-life of BBS2 in Bbs6−/− cells is reduced ∼3-fold (4 min in WT cells versus 12 min in Bbs6−/− cells) (Fig. 7B). Similar results are obtained when we reintroduce FLAG-BBS2 into Bbs7−/− kidney cells (supplemental Fig. S7). These data indicate that the BBS-chaperonin complex and BBS7 play a role in BBS2 stability.The Ubiquitin-Proteasome Pathway Is Involved in BBS2 DegradationTo investigate the mechanism by which BBS2 is degraded in cells, we examined whether the ubiquitin-proteasome pathway is involved in this process. Both exogenously expressed and endogenously expressed BBS2 are ubiquitinated when the proteasome is inhibited by MG132 (Fig. 7C). We further demonstrate that endogenous BBS2 is ubiquitinated even without inhibition of the proteasome when we overexpressed HA-ubiquitin in 293T cells (Fig. 7D). Treatment of 293T cells with the protein synthesis inhibitor cycloheximide resulted in reduced BBS2 protein levels. The reduced BBS2 protein levels in cycloheximide-treated cells are increased when cycloheximide-treated cells are also treated with the proteasome inhibitors MG132 or MG115 (Fig. 7E). These results demonstrate that free BBS2 is ubiquitinated and degraded by the proteasome.BBSome Assembly ModelBased on the data presented above, we propose a model in which the BBSome forms in an ordered sequential process assisted by both intrinsic protein-protein interactions between BBSome proteins, as well as by the non-BBSome chaperonin-like BBS proteins (BBS6, BBS10, and BBS12) (Fig. 8). The BBS-chaperonin complex interacts with BBS7 to stabilize it. BBS7 transitions from the BBS-chaperonin complex to the BBSome core complex, a complex consisting of BBS7-BBS2-BBS9. The formation of the BBSome core complex is accompanied by the removal of the BBS-chaperonin complex. Intrinsic protein-protein interactions promote the incorporation of BBS1, BBS5, and BBS8. BBS4 is added to complete the BBSome, a step requiring the presence of BBS1. The BBS-chaperonin complex is required for the individual stability of BBS2 and BBS7 proteins prior to these proteins participating with BBS9 to form the BBSome core complex.FIGURE 8.Intrinsic protein-protein interactions and chaperonin mediated sequential assembly of the BBSome. Shown are diagrams of our current model for BBSome assembly. BBS7 is stabilized by the BBS-chaperonin complex (BBS6-BBS12-BBS7-CCT) and makes a transition to the BBSome core complex (BBS7-BBS2-BBS9). This transition requires BBS10. The intrinsic protein-protein interactions guide the incorporation of BBS1, BBS5, and BBS8. BBS4 requires BBS1 to incorporate into the complex and is the last subunit to be added to the complex.

BBS2, BBS7, and BBS9 Form Ternary Assembly IntermediatePrevious studies have demonstrated that BBS7, which is an integral part of the BBSome, interacts with the BBS-chaperonin complex (20). The fact that all components of the BBS-chaperonin complex are chaperonin type proteins except for BBS7 suggests that BBS7 may serve as a substrate for the BBS-chaperonin complex and be stabilized by the complex. How BBS7 makes the transition from the BBS-chaperonin complex to the BBSome complex is currently unknown. To address this question, we utilized several approaches that result in the accumulation of distinct BBSome assembly intermediates. Point mutations utilized in this study are summarized in supplemental Fig. S1. When we overexpress FLAG-tagged BBS7 containing the point mutation T211I in 293T cells, the mutant BBS7T211I has a decreased ability to interact with endogenous BBSome subunits compared with WT BBS7 as determined by co-immunoprecipitation using FLAG antibody. However, immunoprecipitation of BBS7T211I pulls down relatively more BBS2 than other BBSome proteins (e.g., BBS1 or BBS4) (Fig. 1A), indicating that there is a binary interaction between BBS7T211I and BBS2 that is trapped (does not proceed to the next step in assembly).FIGURE 1.BBS2, BBS7, and BBS9 form the BBSome core complex.
A, FLAG-tagged BBS7 containing a point mutation (T211I) and WT BBS7 were transfected into 293T cells. Both WT and mutant BBS7 were pulled down by FLAG beads. The existence of other endogenous BBSome subunits is detected by Western blot. B, the absence of BBS2 affects the interaction of BBS7 with BBS1, whereas the loss of BBS1 does not affect the interaction of BBS7 with BBS2. C, FLAG-tagged BBS5, BBS8, BBS9, and BBIP10 were transfected into 293T cells. Tagged proteins were immunoprecipitated (IP) by FLAG beads, and the existence of other endogenous BBSome subunits was detected by Western blot. D, BBS9 RNAi was used to knock down endogenous BBS9 expression in a stable cell line expressing FLAG-BBS2. The BBSome was immunoprecipitated by FLAG beads, and the existence of endogenous BBSome subunits was detected by Western blot.We next knocked down BBS2 to determine the effect this has on BBSome assembly. Knockdown of BBS2 decreases the association of BBS7 with BBS1, whereas knockdown of BBS1 does not affect the association of BBS7 with BBS2 (Fig. 1B). This indicates that BBS2 and BBS7 can associate with each other independent of BBS1. Similar results showing stable interaction of BBS2 with BBS7 are obtained with tissue from mutant mice that express a mutant form of BBS1 with an arginine for methionine mutation at amino acid 390 (Bbs1M390) (Fig. 2C). This finding further supports the concept that BBS7 binds to BBS2 independent of binding to BBS1 and other BBSome proteins. Consistent with this result, BBS7 interacts strongly with BBS2 and weakly with BBS1 in pairwise interaction assays (supplemental Fig. S2A).FIGURE 2.BBS1 is incorporated into the BBSome before BBS4.
A, Western blots of protein lysates from WT and Bbs1M390R/M390R mice demonstrate that the absence of BBS1 does not affect other BBSome subunits protein levels, whereas BBS1 protein levels are greatly decreased in samples from Bbs1M390R/M390R mice as indicated by the arrow. This antibody also recognizes several additional bands that do not change in intensity when compared between WT and Bbs1M390R/M390R mice, indicating that these bands are nonspecific. B, sucrose gradient analysis of BBSome formation in Bbs1M390R testes demonstrates that most BBS4 is dissociated from the subcomplex consisting of BBS2, BBS5, BBS7, BBS8, and BBS9. C, immunoprecipitation (IP) using antibody against BBS2 from testes protein lysates shows that BBS4 is dissociated from the subcomplex in the absence of BBS1. These results are similar to those shown by sucrose gradient analysis. D, reciprocal immunoprecipitation from transfected 293T cells demonstrates that BBS4 dissociation from BBS1M390R is due to decreased interaction between BBS1M390R and BBS4 compared with WT BBS1. E, diagram showing the sequential order of BBS1 and BBS4 incorporation into the BBSome.Next, we overexpressed FLAG-tagged BBS9 in 293T cells and used BBS9 to pull down endogenous BBSome subunits. Immunoprecipitation of exogenously expressed BBS9 results in immunoprecipitation of substantial amounts of BBS2 and BBS7, whereas only small amounts of BBS1 and BBS4 are pulled down under these conditions (Fig. 1C). This result indicates that BBS9 forms a ternary complex with BBS2 and BBS7. To further support the existence of a BBS2-BBS7-BBS9 ternary complex, we knock down BBS9 expression using RNAi. Knockdown of BBS9 does not affect the association of BBS7 with BBS2, indicating that the BBS7/BBS2 interaction is not dependent on BBS9. However, knockdown of BBS9 greatly decreases the association of BBS1, BBS4, and BBS8 with BBS2 (Fig. 1D).Consistent with this result, pairwise interaction studies show that BBS2 directly interacts with BBS7 and BBS9, whereas BBS7 and BBS9 do not directly interact, rather they only interact in the presence of BBS2 (supplemental Fig. S2B). We refer to the ternary complex consisting of BBS2-BBS7-BBS9 as the BBSome core complex because this complex appears to be a central component required for the formation of the complete BBSome.BBS1 and BBS4 Are Peripheral Subunits of BBSomeOur pairwise interaction studies showed that BBS9 directly interacts with several BBSome subunits including BBS2, BBS1, BBS5, and BBS8, establishing BBS9 as a central scaffold of the BBSome (10). To determine the interdependence with which other BBSome subunits are incorporated into the BBSome, we utilized tissues from BBS mutant mice generated in our laboratory (21–24). Western blotting of testicular tissue (which expresses large amounts of BBS proteins) showed that Bbs1M390R/M390R mice have decreased BBS1 protein levels, but other BBSome subunits protein levels are similar to WT levels (Fig. 2A). Sucrose gradient analysis reveals that the BBS1M390R mutation associates normally with all BBSome proteins, except for BBS4 (Fig. 2B). The majority of BBS4 is dissociated from the BBSome in Bbs1M390R/M390R mutant mouse tissue. To confirm this result, we immunoprecipitated the endogenous BBSome from Bbs1M390R/M390R mutant mice using antibody against BBS2. As expected, BBS1 is missing from the complex along with BBS4 (Fig. 2C).To confirm that BBS1M390R mutant protein affects BBS4 incorporation into the BBSome, we co-transfected Myc-tagged BBS1 with FLAG-tagged BBSome subunits into 293T cells. BBS1M390R mutant protein has greatly reduced interaction with BBS4 (Fig. 2D), but normal interactions with BBS9 and BBS2. Taken together, these data indicate that BBS1 is incorporated into the BBSome before BBS4 (Fig. 2E). Using a similar approach, we demonstrate that the loss of BBS4 does not affect the incorporation of other BBSome subunits (including BBS1) into the BBSome (Fig. 3, A and B), indicating that BBS4 is the last subunit to be added to the BBSome (Fig. 3C).FIGURE 3.BBS4 is likely the last subunit to incorporate into the BBSome.
A, sucrose gradient analysis of BBSome formation in Bbs4−/− testes lysates demonstrates that all other BBSome subunits can still form a complex in the absence of BBS4. B, immunoprecipitation using anti-BBS2 antibody from testes protein lysates indicates that BBS4 is the last subunit to incorporate into the BBSome. C, diagram showing the sequential order of BBS components incorporation into the BBSome.To determine whether the absence of other BBS proteins allows for partial BBSome assembly, we knocked down expression of BBS5 and BBS8 using RNAi in RPE1 cells. The results reveal that other BBSome subunits can still incorporate into the BBSome in the absence of BBS5, as well as in the absence of BBS8 (25), indicating that like BBS1, BBS5 and BBS8 are incorporated independently into the BBSome. These findings are consistent with the pairwise interaction data showing that BBS1, BBS5, BBS8, and BBS2 directly interact with BBS9 (10).BBS4 Binds to PCM1 and Localizes to Centrosome/Pericentriolar Satellites Independent of Other BBSome SubunitsThe fact that BBS4 is among the last subunits to be incorporated into the BBSome puts it in a favorable position to interact with non-BBSome proteins including BBSome cargos. Previous studies have demonstrated that the BBSome moves from the cytoplasm to pericentriolar satellites then to cilia. BBS4 localizes to centrosome/pericentriolar satellites and interacts with the C terminus of PCM1 (10, 26). The functional relevance of this interaction is not clear. To determine whether the BBS4-PCM1 interaction is required for centrosome/pericentriolar satellite localization of BBS4, GFP-tagged WT and two mutant forms of BBS4 (BBS4G277E and BBS4R295P) were independently co-transfected with a HA-tagged C-terminal fragment of PCM1. Each of the two point mutations disrupted the interaction of BBS4 with PCM1 (Fig. 4A). When transfected into human RPE1 cells, both BBS4G277E and BBS4R295P lose the ability to localize to centrosome/pericentriolar satellites (Fig. 4B). These data suggest that the BBS4-PCM1 interaction is required for BBS4 centrosome/pericentriolar satellite localization.FIGURE 4.BBS4-PCM1 interaction is required for the BBSome to localize to centrosome/pericentriolar satellite.
A, BBS4 point mutations (G277E and R295P) disrupt the interaction between BBS4 and the C terminus of PCM1. GFP-tagged BBS4 (WT or mutants) and HA-tagged PCM1 C terminus were co-transfected into 293T cells, and co-immunoprecipitation (IP) was performed using antibody against HA and GFP. B, BBS4 point mutations (G277E and R295P) disrupt centrosome/pericentriolar satellite localization of BBS4. Shown is immunofluorescent staining of GFP-BBS4 (WT and mutants) in transfected RPE1 cells. γ-Tubulin (red) is used as a centrosomal marker. C, kidney primary cells were transfected with GFP-BBS4, and transfected cells were co-stained with γ-tubulin. The bar represents 10 μm.To determine whether the intact BBSome is required for BBS4 to localize to centrosome/pericentriolar satellites, we transfected cultured kidney cells from BBS knock-out mice with GFP-BBS4. Absence of BBS2, BBS6, or BBS7 does not affect BBS4 centrosome/pericentriolar satellite localization (Fig. 4C), indicating that this localization is an intrinsic feature of free BBS4 (occurs independent of the BBSome). To further characterize BBS4 interactions, we mapped the BBS4 domain that interacts with PCM1 to the N-terminal 380 amino acids (supplemental Fig. S3A). The N-terminal 380 amino acids of BBS4 do not contain the domain that binds with BBIP10, an integral BBSome protein that binds to the complex through BBS4 (27). Furthermore, we show that PCM1 can interact with BBIP10 only when BBS4 is present (supplemental Fig. S3B).BBS10 Regulates Interaction of BBS6-BBS12-BBS7 with CCT/TRiC Proteins to Form BBS-Chaperonin ComplexAs mentioned previously, studies showed that BBS7 form a complex with BBS6, BBS12, and CCT/TRiC proteins to form the BBS-chaperonin complex (20). The fact that BBS7 is the only non-chaperonin component in the BBS-chaperoinin complex suggests that BBS7 may be the substrate for BBS-chaperonin. One possible function of the BBS-chaperonin complex is the stabilization of BBS7. BBS10 is associated with this complex at substoichiometric levels (less than a 1:1 ratio). This finding suggests that BBS10 is not a structural component of the BBS-chaperonin complex but rather plays a role in regulating complex formation. To investigate the role of BBS10 in BBS-chaperonin complex formation, we utilized RNAi to knock down BBS10 in 293T cells expressing FLAG-BBS6, which serves as a convenient tool for immunoprecipitation of the BBS-chaperonin complex. Absence of BBS10 decreases the interaction of BBS6 with TCPα and TCPβ, two CCT/TRiC proteins (Fig. 5A). Furthermore, overexpression of BBS10 promotes the interaction of BBS6 with TCPα and TCPβ (Fig. 5B). These results demonstrate that BBS10 regulates the formation of the BBS-chaperonin complex.FIGURE 5.BBS10 is required for the interaction of BBS6-BBS12-BBS7 with the CCT/TRiC proteins to form the BBS-chaperonin complex.
A, BBS10 RNAi is used to knock down the expression of BBS10 in 293T cells expressing FLAG-BBS6, which was used to pull down the BBS-chaperonin complex. Antibodies against BBS7 and the CCT/TRiC proteins were used to detect the existence of these proteins in the complex. B, overexpression of BBS10 facilitates the association of BBS6 with the CCT/TRiC proteins. FLAG-BBS10 was co-transfected with empty vector or Myc-BBS6, and Myc antibody was used to pull down the BBS-chaperonin complex. Antibodies against BBS7, TCPα, and TCPβ (two subunits of the CCT/TRiC proteins) were used to detect these proteins in the complex. IP, immunoprecipitation.Release of BBS7 from BBS-Chaperonin Complex Is Coordinated with BBSome Core Complex FormationThe mature form of the BBSome does not contain BBS6, BBS12, or CCT/TRiC proteins, suggesting these chaperonin components must be dissociated from assembly intermediates at some point. To determine when these components are released from the assembly intermediates, we overexpressed FLAG-tagged BBS6 in 293T cells and immunoprecipitated with FLAG antibody. As expected, other BBS-chaperonin complex components, as well as endogenous BBS7 were co-immunoprecipitated along with BBS6 (Fig. 6A). In addition, BBS6 pulled down some BBS2. However, the amount of BBS2 pulled down relative to BBS7 was less than the amounts of BBS2 found in the BBSome (as shown by pulling down the BBSome with BBS5). This indicates that the interaction of BBS2 with BBS7 is coupled with the release of BBS6 and BBS12 from the BBS-chaperonin complex (supplemental Fig. S4).FIGURE 6.The release of BBS7 from the BBS-chaperonin complex is coordinated with BBSome core complex formation.
A, FLAG-BBS6 was overexpressed in 239T cells to trap BBSome intermediates. We pulled down BBS6 using FLAG beads and detected endogenous BBSome subunits by Western blot. FLAG-BBS5 was used as a positive control, and FLAG-GFP was used as a negative control. B, individual BBSome subunits were overexpressed in 293T cells and were used to determine specific associations with the CCT/TRiC proteins. Only BBS7 and BBS2 can associate with CCT/TRiC proteins. FLAG-VHL was used as a positive control. FLAG-GFP and FLAG-BBS3 were used as negative controls. IP, immunoprecipitation.Next we examined the release of the CCT/TRiC proteins from the assembly intermediates. We overexpressed each BBSome subunit in 293T cells and determined whether any BBSome subunit can immunoprecipitate the CCT/TRiC proteins. BBS2 and BBS7 showed robust pulldown of CCT/TRiC proteins. BBS9 showed relatively less pulldown of the CCT/TRiC proteins compared with BBS2 and BBS7 (Fig. 6B). This result indicates that the incorporation of BBS9 to form the BBSome core complex is coupled with the release of BBS2/BBS7 from CCT/TRiC proteins.Stability of BBS2 Is Dependent on Its Interaction with Other ProteinsWe have demonstrated that BBS2, BBS7, and BBS9 form a core complex (Fig. 1). To determine the importance of this complex, we examined the protein level of BBS2 in the absence of BBS7 and BBS9, as well as in the absence of the BBS-chaperonin complex. Absence of BBS7 greatly decreases BBS2 protein levels in testes tissue from Bbs7−/− mice. Furthermore, the absence of BBS6 (a component of the BBS-chaperonin complex) in Bbs6−/− mice also results in decreased amounts of BBS2 (Fig. 7A). Because we do not have Bbs10 knock-out mice, we utilize skin fibroblast cells from a human patient who is homozygous for the common BBS10 mutation, c91fs95. This mutation is predicted to result in the total absence of the BBS10 protein. In our study, using human fibroblast cells, the absence of BBS10 does not affect cilia formation (supplemental Fig. S5), in contrast to a previous report (28). However, the absence of BBS10 lowers BBS2 protein levels (Fig. 7A) and inhibits BBSome formation (supplemental Fig. S6).FIGURE 7.BBS2 is ubiquitinated and degraded by the proteasome.
A, the stability of BBS2 is dependent on the BBSome core complex, as well as the BBS-chaperonin complex. Proteins from BBS knock-out mouse tissue (testis) or from RNAi-treated cell lysates were separated by SDS-PAGE. Anti-BBS2 antibody was used to detect BBS2 protein levels. B, BBS2 is sensitive to protease treatment. Primary cells from WT and Bbs6−/− kidney were infected with adenovirus expressing FLAG-BBS2. Equal amounts of total proteins were mixed with protease thermolysin at 4 °C for the indicated time. Compared with WT cells, BBS2 degrades faster in Bbs6−/− cells, suggesting that BBS2 is not stable in the absence of BBS6. C, 293T cells and FLAG-BBS2 stable-expressing cells were treated with the proteasome inhibitor MG132. BBS2 was immunoprecipitated (IP) by FLAG beads or anti-BBS2 antibody, and ubiquitinated BBS2 was detected by anti-ubiquitin antibody. D, HA-ubiquitin was transfected into 293T cells, and endogenous BBS2 was immunoprecipitated by anti-BBS2 antibody. Ubiquitinated BBS2 was detected by anti-HA antibody or anti-ubiquitin antibody. E, 293T cells were treated with 50 μg/ml of cycloheximide (Chx) with or without 20 μm proteasome inhibitor MG132 or MG115 for 8 h. Endogenous BBS2 in the cell lysates was detected by anti-BBS2 antibody.Next, we used RNAi in RPE1 cells to individually reduce expression of BBS9 (BBSome core complex component), BBS12 and TCPβ (BBS-chaperonin complex components). Reduction of each of these proteins greatly decreases BBS2 protein levels. In contrast, the absence of the normal BBSome proteins BBS4 or BBS1 (using tissue from Bbs4−/− and Bbs1M390R mutant mice, respectively) does not affect BBS2 protein levels (Fig. 7A). These data indicate that the stability of BBS2, a component of the BBSome core complex, requires the BBS-chaperonin complex, as well as the other two components of the BBSome core complex (BBS7 and BBS9). Similarly, the stability of BBS7 requires the BBS-chaperonin complex, as well as the other two components of the BBSome core complex (BBS2 and BBS9) (supplemental Fig. S7).To determine how the BBS-chaperonin complex influences BBS2 protein stability, we infected Bbs6−/− kidney cells and WT kidney cells with adenovirus expressing FLAG-BBS2. The same multiplicity of infection was used to infect WT and Bbs6−/− kidney cells, and equal amounts of proteins were used for the analysis. The BBS2 protein level is lower in Bbs6−/− cells than in WT cells (Fig. 7B, 0'), consistent with the finding that the BBS-chaperonin complex is required for BBS2 protein stability. The remaining BBS2 protein in Bbs6−/− cells is more sensitive to protease thermolysin treatment. The half-life of BBS2 in Bbs6−/− cells is reduced ∼3-fold (4 min in WT cells versus 12 min in Bbs6−/− cells) (Fig. 7B). Similar results are obtained when we reintroduce FLAG-BBS2 into Bbs7−/− kidney cells (supplemental Fig. S7). These data indicate that the BBS-chaperonin complex and BBS7 play a role in BBS2 stability.The Ubiquitin-Proteasome Pathway Is Involved in BBS2 DegradationTo investigate the mechanism by which BBS2 is degraded in cells, we examined whether the ubiquitin-proteasome pathway is involved in this process. Both exogenously expressed and endogenously expressed BBS2 are ubiquitinated when the proteasome is inhibited by MG132 (Fig. 7C). We further demonstrate that endogenous BBS2 is ubiquitinated even without inhibition of the proteasome when we overexpressed HA-ubiquitin in 293T cells (Fig. 7D). Treatment of 293T cells with the protein synthesis inhibitor cycloheximide resulted in reduced BBS2 protein levels. The reduced BBS2 protein levels in cycloheximide-treated cells are increased when cycloheximide-treated cells are also treated with the proteasome inhibitors MG132 or MG115 (Fig. 7E). These results demonstrate that free BBS2 is ubiquitinated and degraded by the proteasome.BBSome Assembly ModelBased on the data presented above, we propose a model in which the BBSome forms in an ordered sequential process assisted by both intrinsic protein-protein interactions between BBSome proteins, as well as by the non-BBSome chaperonin-like BBS proteins (BBS6, BBS10, and BBS12) (Fig. 8). The BBS-chaperonin complex interacts with BBS7 to stabilize it. BBS7 transitions from the BBS-chaperonin complex to the BBSome core complex, a complex consisting of BBS7-BBS2-BBS9. The formation of the BBSome core complex is accompanied by the removal of the BBS-chaperonin complex. Intrinsic protein-protein interactions promote the incorporation of BBS1, BBS5, and BBS8. BBS4 is added to complete the BBSome, a step requiring the presence of BBS1. The BBS-chaperonin complex is required for the individual stability of BBS2 and BBS7 proteins prior to these proteins participating with BBS9 to form the BBSome core complex.FIGURE 8.Intrinsic protein-protein interactions and chaperonin mediated sequential assembly of the BBSome. Shown are diagrams of our current model for BBSome assembly. BBS7 is stabilized by the BBS-chaperonin complex (BBS6-BBS12-BBS7-CCT) and makes a transition to the BBSome core complex (BBS7-BBS2-BBS9). This transition requires BBS10. The intrinsic protein-protein interactions guide the incorporation of BBS1, BBS5, and BBS8. BBS4 requires BBS1 to incorporate into the complex and is the last subunit to be added to the complex.

BBS2, BBS7, and BBS9 Form Ternary Assembly IntermediatePrevious studies have demonstrated that BBS7, which is an integral part of the BBSome, interacts with the BBS-chaperonin complex (20). The fact that all components of the BBS-chaperonin complex are chaperonin type proteins except for BBS7 suggests that BBS7 may serve as a substrate for the BBS-chaperonin complex and be stabilized by the complex. How BBS7 makes the transition from the BBS-chaperonin complex to the BBSome complex is currently unknown. To address this question, we utilized several approaches that result in the accumulation of distinct BBSome assembly intermediates. Point mutations utilized in this study are summarized in supplemental Fig. S1. When we overexpress FLAG-tagged BBS7 containing the point mutation T211I in 293T cells, the mutant BBS7T211I has a decreased ability to interact with endogenous BBSome subunits compared with WT BBS7 as determined by co-immunoprecipitation using FLAG antibody. However, immunoprecipitation of BBS7T211I pulls down relatively more BBS2 than other BBSome proteins (e.g., BBS1 or BBS4) (Fig. 1A), indicating that there is a binary interaction between BBS7T211I and BBS2 that is trapped (does not proceed to the next step in assembly).FIGURE 1.BBS2, BBS7, and BBS9 form the BBSome core complex.
A, FLAG-tagged BBS7 containing a point mutation (T211I) and WT BBS7 were transfected into 293T cells. Both WT and mutant BBS7 were pulled down by FLAG beads. The existence of other endogenous BBSome subunits is detected by Western blot. B, the absence of BBS2 affects the interaction of BBS7 with BBS1, whereas the loss of BBS1 does not affect the interaction of BBS7 with BBS2. C, FLAG-tagged BBS5, BBS8, BBS9, and BBIP10 were transfected into 293T cells. Tagged proteins were immunoprecipitated (IP) by FLAG beads, and the existence of other endogenous BBSome subunits was detected by Western blot. D, BBS9 RNAi was used to knock down endogenous BBS9 expression in a stable cell line expressing FLAG-BBS2. The BBSome was immunoprecipitated by FLAG beads, and the existence of endogenous BBSome subunits was detected by Western blot.We next knocked down BBS2 to determine the effect this has on BBSome assembly. Knockdown of BBS2 decreases the association of BBS7 with BBS1, whereas knockdown of BBS1 does not affect the association of BBS7 with BBS2 (Fig. 1B). This indicates that BBS2 and BBS7 can associate with each other independent of BBS1. Similar results showing stable interaction of BBS2 with BBS7 are obtained with tissue from mutant mice that express a mutant form of BBS1 with an arginine for methionine mutation at amino acid 390 (Bbs1M390) (Fig. 2C). This finding further supports the concept that BBS7 binds to BBS2 independent of binding to BBS1 and other BBSome proteins. Consistent with this result, BBS7 interacts strongly with BBS2 and weakly with BBS1 in pairwise interaction assays (supplemental Fig. S2A).FIGURE 2.BBS1 is incorporated into the BBSome before BBS4.
A, Western blots of protein lysates from WT and Bbs1M390R/M390R mice demonstrate that the absence of BBS1 does not affect other BBSome subunits protein levels, whereas BBS1 protein levels are greatly decreased in samples from Bbs1M390R/M390R mice as indicated by the arrow. This antibody also recognizes several additional bands that do not change in intensity when compared between WT and Bbs1M390R/M390R mice, indicating that these bands are nonspecific. B, sucrose gradient analysis of BBSome formation in Bbs1M390R testes demonstrates that most BBS4 is dissociated from the subcomplex consisting of BBS2, BBS5, BBS7, BBS8, and BBS9. C, immunoprecipitation (IP) using antibody against BBS2 from testes protein lysates shows that BBS4 is dissociated from the subcomplex in the absence of BBS1. These results are similar to those shown by sucrose gradient analysis. D, reciprocal immunoprecipitation from transfected 293T cells demonstrates that BBS4 dissociation from BBS1M390R is due to decreased interaction between BBS1M390R and BBS4 compared with WT BBS1. E, diagram showing the sequential order of BBS1 and BBS4 incorporation into the BBSome.Next, we overexpressed FLAG-tagged BBS9 in 293T cells and used BBS9 to pull down endogenous BBSome subunits. Immunoprecipitation of exogenously expressed BBS9 results in immunoprecipitation of substantial amounts of BBS2 and BBS7, whereas only small amounts of BBS1 and BBS4 are pulled down under these conditions (Fig. 1C). This result indicates that BBS9 forms a ternary complex with BBS2 and BBS7. To further support the existence of a BBS2-BBS7-BBS9 ternary complex, we knock down BBS9 expression using RNAi. Knockdown of BBS9 does not affect the association of BBS7 with BBS2, indicating that the BBS7/BBS2 interaction is not dependent on BBS9. However, knockdown of BBS9 greatly decreases the association of BBS1, BBS4, and BBS8 with BBS2 (Fig. 1D).Consistent with this result, pairwise interaction studies show that BBS2 directly interacts with BBS7 and BBS9, whereas BBS7 and BBS9 do not directly interact, rather they only interact in the presence of BBS2 (supplemental Fig. S2B). We refer to the ternary complex consisting of BBS2-BBS7-BBS9 as the BBSome core complex because this complex appears to be a central component required for the formation of the complete BBSome.BBS1 and BBS4 Are Peripheral Subunits of BBSomeOur pairwise interaction studies showed that BBS9 directly interacts with several BBSome subunits including BBS2, BBS1, BBS5, and BBS8, establishing BBS9 as a central scaffold of the BBSome (10). To determine the interdependence with which other BBSome subunits are incorporated into the BBSome, we utilized tissues from BBS mutant mice generated in our laboratory (21–24). Western blotting of testicular tissue (which expresses large amounts of BBS proteins) showed that Bbs1M390R/M390R mice have decreased BBS1 protein levels, but other BBSome subunits protein levels are similar to WT levels (Fig. 2A). Sucrose gradient analysis reveals that the BBS1M390R mutation associates normally with all BBSome proteins, except for BBS4 (Fig. 2B). The majority of BBS4 is dissociated from the BBSome in Bbs1M390R/M390R mutant mouse tissue. To confirm this result, we immunoprecipitated the endogenous BBSome from Bbs1M390R/M390R mutant mice using antibody against BBS2. As expected, BBS1 is missing from the complex along with BBS4 (Fig. 2C).To confirm that BBS1M390R mutant protein affects BBS4 incorporation into the BBSome, we co-transfected Myc-tagged BBS1 with FLAG-tagged BBSome subunits into 293T cells. BBS1M390R mutant protein has greatly reduced interaction with BBS4 (Fig. 2D), but normal interactions with BBS9 and BBS2. Taken together, these data indicate that BBS1 is incorporated into the BBSome before BBS4 (Fig. 2E). Using a similar approach, we demonstrate that the loss of BBS4 does not affect the incorporation of other BBSome subunits (including BBS1) into the BBSome (Fig. 3, A and B), indicating that BBS4 is the last subunit to be added to the BBSome (Fig. 3C).FIGURE 3.BBS4 is likely the last subunit to incorporate into the BBSome.
A, sucrose gradient analysis of BBSome formation in Bbs4−/− testes lysates demonstrates that all other BBSome subunits can still form a complex in the absence of BBS4. B, immunoprecipitation using anti-BBS2 antibody from testes protein lysates indicates that BBS4 is the last subunit to incorporate into the BBSome. C, diagram showing the sequential order of BBS components incorporation into the BBSome.To determine whether the absence of other BBS proteins allows for partial BBSome assembly, we knocked down expression of BBS5 and BBS8 using RNAi in RPE1 cells. The results reveal that other BBSome subunits can still incorporate into the BBSome in the absence of BBS5, as well as in the absence of BBS8 (25), indicating that like BBS1, BBS5 and BBS8 are incorporated independently into the BBSome. These findings are consistent with the pairwise interaction data showing that BBS1, BBS5, BBS8, and BBS2 directly interact with BBS9 (10).BBS4 Binds to PCM1 and Localizes to Centrosome/Pericentriolar Satellites Independent of Other BBSome SubunitsThe fact that BBS4 is among the last subunits to be incorporated into the BBSome puts it in a favorable position to interact with non-BBSome proteins including BBSome cargos. Previous studies have demonstrated that the BBSome moves from the cytoplasm to pericentriolar satellites then to cilia. BBS4 localizes to centrosome/pericentriolar satellites and interacts with the C terminus of PCM1 (10, 26). The functional relevance of this interaction is not clear. To determine whether the BBS4-PCM1 interaction is required for centrosome/pericentriolar satellite localization of BBS4, GFP-tagged WT and two mutant forms of BBS4 (BBS4G277E and BBS4R295P) were independently co-transfected with a HA-tagged C-terminal fragment of PCM1. Each of the two point mutations disrupted the interaction of BBS4 with PCM1 (Fig. 4A). When transfected into human RPE1 cells, both BBS4G277E and BBS4R295P lose the ability to localize to centrosome/pericentriolar satellites (Fig. 4B). These data suggest that the BBS4-PCM1 interaction is required for BBS4 centrosome/pericentriolar satellite localization.FIGURE 4.BBS4-PCM1 interaction is required for the BBSome to localize to centrosome/pericentriolar satellite.
A, BBS4 point mutations (G277E and R295P) disrupt the interaction between BBS4 and the C terminus of PCM1. GFP-tagged BBS4 (WT or mutants) and HA-tagged PCM1 C terminus were co-transfected into 293T cells, and co-immunoprecipitation (IP) was performed using antibody against HA and GFP. B, BBS4 point mutations (G277E and R295P) disrupt centrosome/pericentriolar satellite localization of BBS4. Shown is immunofluorescent staining of GFP-BBS4 (WT and mutants) in transfected RPE1 cells. γ-Tubulin (red) is used as a centrosomal marker. C, kidney primary cells were transfected with GFP-BBS4, and transfected cells were co-stained with γ-tubulin. The bar represents 10 μm.To determine whether the intact BBSome is required for BBS4 to localize to centrosome/pericentriolar satellites, we transfected cultured kidney cells from BBS knock-out mice with GFP-BBS4. Absence of BBS2, BBS6, or BBS7 does not affect BBS4 centrosome/pericentriolar satellite localization (Fig. 4C), indicating that this localization is an intrinsic feature of free BBS4 (occurs independent of the BBSome). To further characterize BBS4 interactions, we mapped the BBS4 domain that interacts with PCM1 to the N-terminal 380 amino acids (supplemental Fig. S3A). The N-terminal 380 amino acids of BBS4 do not contain the domain that binds with BBIP10, an integral BBSome protein that binds to the complex through BBS4 (27). Furthermore, we show that PCM1 can interact with BBIP10 only when BBS4 is present (supplemental Fig. S3B).BBS10 Regulates Interaction of BBS6-BBS12-BBS7 with CCT/TRiC Proteins to Form BBS-Chaperonin ComplexAs mentioned previously, studies showed that BBS7 form a complex with BBS6, BBS12, and CCT/TRiC proteins to form the BBS-chaperonin complex (20). The fact that BBS7 is the only non-chaperonin component in the BBS-chaperoinin complex suggests that BBS7 may be the substrate for BBS-chaperonin. One possible function of the BBS-chaperonin complex is the stabilization of BBS7. BBS10 is associated with this complex at substoichiometric levels (less than a 1:1 ratio). This finding suggests that BBS10 is not a structural component of the BBS-chaperonin complex but rather plays a role in regulating complex formation. To investigate the role of BBS10 in BBS-chaperonin complex formation, we utilized RNAi to knock down BBS10 in 293T cells expressing FLAG-BBS6, which serves as a convenient tool for immunoprecipitation of the BBS-chaperonin complex. Absence of BBS10 decreases the interaction of BBS6 with TCPα and TCPβ, two CCT/TRiC proteins (Fig. 5A). Furthermore, overexpression of BBS10 promotes the interaction of BBS6 with TCPα and TCPβ (Fig. 5B). These results demonstrate that BBS10 regulates the formation of the BBS-chaperonin complex.FIGURE 5.BBS10 is required for the interaction of BBS6-BBS12-BBS7 with the CCT/TRiC proteins to form the BBS-chaperonin complex.
A, BBS10 RNAi is used to knock down the expression of BBS10 in 293T cells expressing FLAG-BBS6, which was used to pull down the BBS-chaperonin complex. Antibodies against BBS7 and the CCT/TRiC proteins were used to detect the existence of these proteins in the complex. B, overexpression of BBS10 facilitates the association of BBS6 with the CCT/TRiC proteins. FLAG-BBS10 was co-transfected with empty vector or Myc-BBS6, and Myc antibody was used to pull down the BBS-chaperonin complex. Antibodies against BBS7, TCPα, and TCPβ (two subunits of the CCT/TRiC proteins) were used to detect these proteins in the complex. IP, immunoprecipitation.Release of BBS7 from BBS-Chaperonin Complex Is Coordinated with BBSome Core Complex FormationThe mature form of the BBSome does not contain BBS6, BBS12, or CCT/TRiC proteins, suggesting these chaperonin components must be dissociated from assembly intermediates at some point. To determine when these components are released from the assembly intermediates, we overexpressed FLAG-tagged BBS6 in 293T cells and immunoprecipitated with FLAG antibody. As expected, other BBS-chaperonin complex components, as well as endogenous BBS7 were co-immunoprecipitated along with BBS6 (Fig. 6A). In addition, BBS6 pulled down some BBS2. However, the amount of BBS2 pulled down relative to BBS7 was less than the amounts of BBS2 found in the BBSome (as shown by pulling down the BBSome with BBS5). This indicates that the interaction of BBS2 with BBS7 is coupled with the release of BBS6 and BBS12 from the BBS-chaperonin complex (supplemental Fig. S4).FIGURE 6.The release of BBS7 from the BBS-chaperonin complex is coordinated with BBSome core complex formation.
A, FLAG-BBS6 was overexpressed in 239T cells to trap BBSome intermediates. We pulled down BBS6 using FLAG beads and detected endogenous BBSome subunits by Western blot. FLAG-BBS5 was used as a positive control, and FLAG-GFP was used as a negative control. B, individual BBSome subunits were overexpressed in 293T cells and were used to determine specific associations with the CCT/TRiC proteins. Only BBS7 and BBS2 can associate with CCT/TRiC proteins. FLAG-VHL was used as a positive control. FLAG-GFP and FLAG-BBS3 were used as negative controls. IP, immunoprecipitation.Next we examined the release of the CCT/TRiC proteins from the assembly intermediates. We overexpressed each BBSome subunit in 293T cells and determined whether any BBSome subunit can immunoprecipitate the CCT/TRiC proteins. BBS2 and BBS7 showed robust pulldown of CCT/TRiC proteins. BBS9 showed relatively less pulldown of the CCT/TRiC proteins compared with BBS2 and BBS7 (Fig. 6B). This result indicates that the incorporation of BBS9 to form the BBSome core complex is coupled with the release of BBS2/BBS7 from CCT/TRiC proteins.Stability of BBS2 Is Dependent on Its Interaction with Other ProteinsWe have demonstrated that BBS2, BBS7, and BBS9 form a core complex (Fig. 1). To determine the importance of this complex, we examined the protein level of BBS2 in the absence of BBS7 and BBS9, as well as in the absence of the BBS-chaperonin complex. Absence of BBS7 greatly decreases BBS2 protein levels in testes tissue from Bbs7−/− mice. Furthermore, the absence of BBS6 (a component of the BBS-chaperonin complex) in Bbs6−/− mice also results in decreased amounts of BBS2 (Fig. 7A). Because we do not have Bbs10 knock-out mice, we utilize skin fibroblast cells from a human patient who is homozygous for the common BBS10 mutation, c91fs95. This mutation is predicted to result in the total absence of the BBS10 protein. In our study, using human fibroblast cells, the absence of BBS10 does not affect cilia formation (supplemental Fig. S5), in contrast to a previous report (28). However, the absence of BBS10 lowers BBS2 protein levels (Fig. 7A) and inhibits BBSome formation (supplemental Fig. S6).FIGURE 7.BBS2 is ubiquitinated and degraded by the proteasome.
A, the stability of BBS2 is dependent on the BBSome core complex, as well as the BBS-chaperonin complex. Proteins from BBS knock-out mouse tissue (testis) or from RNAi-treated cell lysates were separated by SDS-PAGE. Anti-BBS2 antibody was used to detect BBS2 protein levels. B, BBS2 is sensitive to protease treatment. Primary cells from WT and Bbs6−/− kidney were infected with adenovirus expressing FLAG-BBS2. Equal amounts of total proteins were mixed with protease thermolysin at 4 °C for the indicated time. Compared with WT cells, BBS2 degrades faster in Bbs6−/− cells, suggesting that BBS2 is not stable in the absence of BBS6. C, 293T cells and FLAG-BBS2 stable-expressing cells were treated with the proteasome inhibitor MG132. BBS2 was immunoprecipitated (IP) by FLAG beads or anti-BBS2 antibody, and ubiquitinated BBS2 was detected by anti-ubiquitin antibody. D, HA-ubiquitin was transfected into 293T cells, and endogenous BBS2 was immunoprecipitated by anti-BBS2 antibody. Ubiquitinated BBS2 was detected by anti-HA antibody or anti-ubiquitin antibody. E, 293T cells were treated with 50 μg/ml of cycloheximide (Chx) with or without 20 μm proteasome inhibitor MG132 or MG115 for 8 h. Endogenous BBS2 in the cell lysates was detected by anti-BBS2 antibody.Next, we used RNAi in RPE1 cells to individually reduce expression of BBS9 (BBSome core complex component), BBS12 and TCPβ (BBS-chaperonin complex components). Reduction of each of these proteins greatly decreases BBS2 protein levels. In contrast, the absence of the normal BBSome proteins BBS4 or BBS1 (using tissue from Bbs4−/− and Bbs1M390R mutant mice, respectively) does not affect BBS2 protein levels (Fig. 7A). These data indicate that the stability of BBS2, a component of the BBSome core complex, requires the BBS-chaperonin complex, as well as the other two components of the BBSome core complex (BBS7 and BBS9). Similarly, the stability of BBS7 requires the BBS-chaperonin complex, as well as the other two components of the BBSome core complex (BBS2 and BBS9) (supplemental Fig. S7).To determine how the BBS-chaperonin complex influences BBS2 protein stability, we infected Bbs6−/− kidney cells and WT kidney cells with adenovirus expressing FLAG-BBS2. The same multiplicity of infection was used to infect WT and Bbs6−/− kidney cells, and equal amounts of proteins were used for the analysis. The BBS2 protein level is lower in Bbs6−/− cells than in WT cells (Fig. 7B, 0'), consistent with the finding that the BBS-chaperonin complex is required for BBS2 protein stability. The remaining BBS2 protein in Bbs6−/− cells is more sensitive to protease thermolysin treatment. The half-life of BBS2 in Bbs6−/− cells is reduced ∼3-fold (4 min in WT cells versus 12 min in Bbs6−/− cells) (Fig. 7B). Similar results are obtained when we reintroduce FLAG-BBS2 into Bbs7−/− kidney cells (supplemental Fig. S7). These data indicate that the BBS-chaperonin complex and BBS7 play a role in BBS2 stability.The Ubiquitin-Proteasome Pathway Is Involved in BBS2 DegradationTo investigate the mechanism by which BBS2 is degraded in cells, we examined whether the ubiquitin-proteasome pathway is involved in this process. Both exogenously expressed and endogenously expressed BBS2 are ubiquitinated when the proteasome is inhibited by MG132 (Fig. 7C). We further demonstrate that endogenous BBS2 is ubiquitinated even without inhibition of the proteasome when we overexpressed HA-ubiquitin in 293T cells (Fig. 7D). Treatment of 293T cells with the protein synthesis inhibitor cycloheximide resulted in reduced BBS2 protein levels. The reduced BBS2 protein levels in cycloheximide-treated cells are increased when cycloheximide-treated cells are also treated with the proteasome inhibitors MG132 or MG115 (Fig. 7E). These results demonstrate that free BBS2 is ubiquitinated and degraded by the proteasome.BBSome Assembly ModelBased on the data presented above, we propose a model in which the BBSome forms in an ordered sequential process assisted by both intrinsic protein-protein interactions between BBSome proteins, as well as by the non-BBSome chaperonin-like BBS proteins (BBS6, BBS10, and BBS12) (Fig. 8). The BBS-chaperonin complex interacts with BBS7 to stabilize it. BBS7 transitions from the BBS-chaperonin complex to the BBSome core complex, a complex consisting of BBS7-BBS2-BBS9. The formation of the BBSome core complex is accompanied by the removal of the BBS-chaperonin complex. Intrinsic protein-protein interactions promote the incorporation of BBS1, BBS5, and BBS8. BBS4 is added to complete the BBSome, a step requiring the presence of BBS1. The BBS-chaperonin complex is required for the individual stability of BBS2 and BBS7 proteins prior to these proteins participating with BBS9 to form the BBSome core complex.FIGURE 8.Intrinsic protein-protein interactions and chaperonin mediated sequential assembly of the BBSome. Shown are diagrams of our current model for BBSome assembly. BBS7 is stabilized by the BBS-chaperonin complex (BBS6-BBS12-BBS7-CCT) and makes a transition to the BBSome core complex (BBS7-BBS2-BBS9). This transition requires BBS10. The intrinsic protein-protein interactions guide the incorporation of BBS1, BBS5, and BBS8. BBS4 requires BBS1 to incorporate into the complex and is the last subunit to be added to the complex.

BBS2, BBS7, and BBS9 Form Ternary Assembly IntermediatePrevious studies have demonstrated that BBS7, which is an integral part of the BBSome, interacts with the BBS-chaperonin complex (20). The fact that all components of the BBS-chaperonin complex are chaperonin type proteins except for BBS7 suggests that BBS7 may serve as a substrate for the BBS-chaperonin complex and be stabilized by the complex. How BBS7 makes the transition from the BBS-chaperonin complex to the BBSome complex is currently unknown. To address this question, we utilized several approaches that result in the accumulation of distinct BBSome assembly intermediates. Point mutations utilized in this study are summarized in supplemental Fig. S1. When we overexpress FLAG-tagged BBS7 containing the point mutation T211I in 293T cells, the mutant BBS7T211I has a decreased ability to interact with endogenous BBSome subunits compared with WT BBS7 as determined by co-immunoprecipitation using FLAG antibody. However, immunoprecipitation of BBS7T211I pulls down relatively more BBS2 than other BBSome proteins (e.g., BBS1 or BBS4) (Fig. 1A), indicating that there is a binary interaction between BBS7T211I and BBS2 that is trapped (does not proceed to the next step in assembly).FIGURE 1.BBS2, BBS7, and BBS9 form the BBSome core complex.
A, FLAG-tagged BBS7 containing a point mutation (T211I) and WT BBS7 were transfected into 293T cells. Both WT and mutant BBS7 were pulled down by FLAG beads. The existence of other endogenous BBSome subunits is detected by Western blot. B, the absence of BBS2 affects the interaction of BBS7 with BBS1, whereas the loss of BBS1 does not affect the interaction of BBS7 with BBS2. C, FLAG-tagged BBS5, BBS8, BBS9, and BBIP10 were transfected into 293T cells. Tagged proteins were immunoprecipitated (IP) by FLAG beads, and the existence of other endogenous BBSome subunits was detected by Western blot. D, BBS9 RNAi was used to knock down endogenous BBS9 expression in a stable cell line expressing FLAG-BBS2. The BBSome was immunoprecipitated by FLAG beads, and the existence of endogenous BBSome subunits was detected by Western blot.We next knocked down BBS2 to determine the effect this has on BBSome assembly. Knockdown of BBS2 decreases the association of BBS7 with BBS1, whereas knockdown of BBS1 does not affect the association of BBS7 with BBS2 (Fig. 1B). This indicates that BBS2 and BBS7 can associate with each other independent of BBS1. Similar results showing stable interaction of BBS2 with BBS7 are obtained with tissue from mutant mice that express a mutant form of BBS1 with an arginine for methionine mutation at amino acid 390 (Bbs1M390) (Fig. 2C). This finding further supports the concept that BBS7 binds to BBS2 independent of binding to BBS1 and other BBSome proteins. Consistent with this result, BBS7 interacts strongly with BBS2 and weakly with BBS1 in pairwise interaction assays (supplemental Fig. S2A).FIGURE 2.BBS1 is incorporated into the BBSome before BBS4.
A, Western blots of protein lysates from WT and Bbs1M390R/M390R mice demonstrate that the absence of BBS1 does not affect other BBSome subunits protein levels, whereas BBS1 protein levels are greatly decreased in samples from Bbs1M390R/M390R mice as indicated by the arrow. This antibody also recognizes several additional bands that do not change in intensity when compared between WT and Bbs1M390R/M390R mice, indicating that these bands are nonspecific. B, sucrose gradient analysis of BBSome formation in Bbs1M390R testes demonstrates that most BBS4 is dissociated from the subcomplex consisting of BBS2, BBS5, BBS7, BBS8, and BBS9. C, immunoprecipitation (IP) using antibody against BBS2 from testes protein lysates shows that BBS4 is dissociated from the subcomplex in the absence of BBS1. These results are similar to those shown by sucrose gradient analysis. D, reciprocal immunoprecipitation from transfected 293T cells demonstrates that BBS4 dissociation from BBS1M390R is due to decreased interaction between BBS1M390R and BBS4 compared with WT BBS1. E, diagram showing the sequential order of BBS1 and BBS4 incorporation into the BBSome.Next, we overexpressed FLAG-tagged BBS9 in 293T cells and used BBS9 to pull down endogenous BBSome subunits. Immunoprecipitation of exogenously expressed BBS9 results in immunoprecipitation of substantial amounts of BBS2 and BBS7, whereas only small amounts of BBS1 and BBS4 are pulled down under these conditions (Fig. 1C). This result indicates that BBS9 forms a ternary complex with BBS2 and BBS7. To further support the existence of a BBS2-BBS7-BBS9 ternary complex, we knock down BBS9 expression using RNAi. Knockdown of BBS9 does not affect the association of BBS7 with BBS2, indicating that the BBS7/BBS2 interaction is not dependent on BBS9. However, knockdown of BBS9 greatly decreases the association of BBS1, BBS4, and BBS8 with BBS2 (Fig. 1D).Consistent with this result, pairwise interaction studies show that BBS2 directly interacts with BBS7 and BBS9, whereas BBS7 and BBS9 do not directly interact, rather they only interact in the presence of BBS2 (supplemental Fig. S2B). We refer to the ternary complex consisting of BBS2-BBS7-BBS9 as the BBSome core complex because this complex appears to be a central component required for the formation of the complete BBSome.

BBS2, BBS7, and BBS9 Form Ternary Assembly Intermediate

Previous studies have demonstrated that BBS7, which is an integral part of the BBSome, interacts with the BBS-chaperonin complex (20). The fact that all components of the BBS-chaperonin complex are chaperonin type proteins except for BBS7 suggests that BBS7 may serve as a substrate for the BBS-chaperonin complex and be stabilized by the complex. How BBS7 makes the transition from the BBS-chaperonin complex to the BBSome complex is currently unknown. To address this question, we utilized several approaches that result in the accumulation of distinct BBSome assembly intermediates. Point mutations utilized in this study are summarized in supplemental Fig. S1. When we overexpress FLAG-tagged BBS7 containing the point mutation T211I in 293T cells, the mutant BBS7T211I has a decreased ability to interact with endogenous BBSome subunits compared with WT BBS7 as determined by co-immunoprecipitation using FLAG antibody. However, immunoprecipitation of BBS7T211I pulls down relatively more BBS2 than other BBSome proteins (e.g., BBS1 or BBS4) (Fig. 1A), indicating that there is a binary interaction between BBS7T211I and BBS2 that is trapped (does not proceed to the next step in assembly).

BBS2, BBS7, and BBS9 form the BBSome core complex.
A, FLAG-tagged BBS7 containing a point mutation (T211I) and WT BBS7 were transfected into 293T cells. Both WT and mutant BBS7 were pulled down by FLAG beads. The existence of other endogenous BBSome subunits is detected by Western blot. B, the absence of BBS2 affects the interaction of BBS7 with BBS1, whereas the loss of BBS1 does not affect the interaction of BBS7 with BBS2. C, FLAG-tagged BBS5, BBS8, BBS9, and BBIP10 were transfected into 293T cells. Tagged proteins were immunoprecipitated (IP) by FLAG beads, and the existence of other endogenous BBSome subunits was detected by Western blot. D, BBS9 RNAi was used to knock down endogenous BBS9 expression in a stable cell line expressing FLAG-BBS2. The BBSome was immunoprecipitated by FLAG beads, and the existence of endogenous BBSome subunits was detected by Western blot.

We next knocked down BBS2 to determine the effect this has on BBSome assembly. Knockdown of BBS2 decreases the association of BBS7 with BBS1, whereas knockdown of BBS1 does not affect the association of BBS7 with BBS2 (Fig. 1B). This indicates that BBS2 and BBS7 can associate with each other independent of BBS1. Similar results showing stable interaction of BBS2 with BBS7 are obtained with tissue from mutant mice that express a mutant form of BBS1 with an arginine for methionine mutation at amino acid 390 (Bbs1M390) (Fig. 2C). This finding further supports the concept that BBS7 binds to BBS2 independent of binding to BBS1 and other BBSome proteins. Consistent with this result, BBS7 interacts strongly with BBS2 and weakly with BBS1 in pairwise interaction assays (supplemental Fig. S2A).

BBS1 is incorporated into the BBSome before BBS4.
A, Western blots of protein lysates from WT and Bbs1M390R/M390R mice demonstrate that the absence of BBS1 does not affect other BBSome subunits protein levels, whereas BBS1 protein levels are greatly decreased in samples from Bbs1M390R/M390R mice as indicated by the arrow. This antibody also recognizes several additional bands that do not change in intensity when compared between WT and Bbs1M390R/M390R mice, indicating that these bands are nonspecific. B, sucrose gradient analysis of BBSome formation in Bbs1M390R testes demonstrates that most BBS4 is dissociated from the subcomplex consisting of BBS2, BBS5, BBS7, BBS8, and BBS9. C, immunoprecipitation (IP) using antibody against BBS2 from testes protein lysates shows that BBS4 is dissociated from the subcomplex in the absence of BBS1. These results are similar to those shown by sucrose gradient analysis. D, reciprocal immunoprecipitation from transfected 293T cells demonstrates that BBS4 dissociation from BBS1M390R is due to decreased interaction between BBS1M390R and BBS4 compared with WT BBS1. E, diagram showing the sequential order of BBS1 and BBS4 incorporation into the BBSome.

Next, we overexpressed FLAG-tagged BBS9 in 293T cells and used BBS9 to pull down endogenous BBSome subunits. Immunoprecipitation of exogenously expressed BBS9 results in immunoprecipitation of substantial amounts of BBS2 and BBS7, whereas only small amounts of BBS1 and BBS4 are pulled down under these conditions (Fig. 1C). This result indicates that BBS9 forms a ternary complex with BBS2 and BBS7. To further support the existence of a BBS2-BBS7-BBS9 ternary complex, we knock down BBS9 expression using RNAi. Knockdown of BBS9 does not affect the association of BBS7 with BBS2, indicating that the BBS7/BBS2 interaction is not dependent on BBS9. However, knockdown of BBS9 greatly decreases the association of BBS1, BBS4, and BBS8 with BBS2 (Fig. 1D).

Consistent with this result, pairwise interaction studies show that BBS2 directly interacts with BBS7 and BBS9, whereas BBS7 and BBS9 do not directly interact, rather they only interact in the presence of BBS2 (supplemental Fig. S2B). We refer to the ternary complex consisting of BBS2-BBS7-BBS9 as the BBSome core complex because this complex appears to be a central component required for the formation of the complete BBSome.

BBS1 and BBS4 Are Peripheral Subunits of BBSomeOur pairwise interaction studies showed that BBS9 directly interacts with several BBSome subunits including BBS2, BBS1, BBS5, and BBS8, establishing BBS9 as a central scaffold of the BBSome (10). To determine the interdependence with which other BBSome subunits are incorporated into the BBSome, we utilized tissues from BBS mutant mice generated in our laboratory (21–24). Western blotting of testicular tissue (which expresses large amounts of BBS proteins) showed that Bbs1M390R/M390R mice have decreased BBS1 protein levels, but other BBSome subunits protein levels are similar to WT levels (Fig. 2A). Sucrose gradient analysis reveals that the BBS1M390R mutation associates normally with all BBSome proteins, except for BBS4 (Fig. 2B). The majority of BBS4 is dissociated from the BBSome in Bbs1M390R/M390R mutant mouse tissue. To confirm this result, we immunoprecipitated the endogenous BBSome from Bbs1M390R/M390R mutant mice using antibody against BBS2. As expected, BBS1 is missing from the complex along with BBS4 (Fig. 2C).To confirm that BBS1M390R mutant protein affects BBS4 incorporation into the BBSome, we co-transfected Myc-tagged BBS1 with FLAG-tagged BBSome subunits into 293T cells. BBS1M390R mutant protein has greatly reduced interaction with BBS4 (Fig. 2D), but normal interactions with BBS9 and BBS2. Taken together, these data indicate that BBS1 is incorporated into the BBSome before BBS4 (Fig. 2E). Using a similar approach, we demonstrate that the loss of BBS4 does not affect the incorporation of other BBSome subunits (including BBS1) into the BBSome (Fig. 3, A and B), indicating that BBS4 is the last subunit to be added to the BBSome (Fig. 3C).FIGURE 3.BBS4 is likely the last subunit to incorporate into the BBSome.
A, sucrose gradient analysis of BBSome formation in Bbs4−/− testes lysates demonstrates that all other BBSome subunits can still form a complex in the absence of BBS4. B, immunoprecipitation using anti-BBS2 antibody from testes protein lysates indicates that BBS4 is the last subunit to incorporate into the BBSome. C, diagram showing the sequential order of BBS components incorporation into the BBSome.To determine whether the absence of other BBS proteins allows for partial BBSome assembly, we knocked down expression of BBS5 and BBS8 using RNAi in RPE1 cells. The results reveal that other BBSome subunits can still incorporate into the BBSome in the absence of BBS5, as well as in the absence of BBS8 (25), indicating that like BBS1, BBS5 and BBS8 are incorporated independently into the BBSome. These findings are consistent with the pairwise interaction data showing that BBS1, BBS5, BBS8, and BBS2 directly interact with BBS9 (10).

BBS1 and BBS4 Are Peripheral Subunits of BBSome

Our pairwise interaction studies showed that BBS9 directly interacts with several BBSome subunits including BBS2, BBS1, BBS5, and BBS8, establishing BBS9 as a central scaffold of the BBSome (10). To determine the interdependence with which other BBSome subunits are incorporated into the BBSome, we utilized tissues from BBS mutant mice generated in our laboratory (21–24). Western blotting of testicular tissue (which expresses large amounts of BBS proteins) showed that Bbs1M390R/M390R mice have decreased BBS1 protein levels, but other BBSome subunits protein levels are similar to WT levels (Fig. 2A). Sucrose gradient analysis reveals that the BBS1M390R mutation associates normally with all BBSome proteins, except for BBS4 (Fig. 2B). The majority of BBS4 is dissociated from the BBSome in Bbs1M390R/M390R mutant mouse tissue. To confirm this result, we immunoprecipitated the endogenous BBSome from Bbs1M390R/M390R mutant mice using antibody against BBS2. As expected, BBS1 is missing from the complex along with BBS4 (Fig. 2C).

To confirm that BBS1M390R mutant protein affects BBS4 incorporation into the BBSome, we co-transfected Myc-tagged BBS1 with FLAG-tagged BBSome subunits into 293T cells. BBS1M390R mutant protein has greatly reduced interaction with BBS4 (Fig. 2D), but normal interactions with BBS9 and BBS2. Taken together, these data indicate that BBS1 is incorporated into the BBSome before BBS4 (Fig. 2E). Using a similar approach, we demonstrate that the loss of BBS4 does not affect the incorporation of other BBSome subunits (including BBS1) into the BBSome (Fig. 3, A and B), indicating that BBS4 is the last subunit to be added to the BBSome (Fig. 3C).

BBS4 is likely the last subunit to incorporate into the BBSome.
A, sucrose gradient analysis of BBSome formation in Bbs4−/− testes lysates demonstrates that all other BBSome subunits can still form a complex in the absence of BBS4. B, immunoprecipitation using anti-BBS2 antibody from testes protein lysates indicates that BBS4 is the last subunit to incorporate into the BBSome. C, diagram showing the sequential order of BBS components incorporation into the BBSome.

To determine whether the absence of other BBS proteins allows for partial BBSome assembly, we knocked down expression of BBS5 and BBS8 using RNAi in RPE1 cells. The results reveal that other BBSome subunits can still incorporate into the BBSome in the absence of BBS5, as well as in the absence of BBS8 (25), indicating that like BBS1, BBS5 and BBS8 are incorporated independently into the BBSome. These findings are consistent with the pairwise interaction data showing that BBS1, BBS5, BBS8, and BBS2 directly interact with BBS9 (10).

BBS4 Binds to PCM1 and Localizes to Centrosome/Pericentriolar Satellites Independent of Other BBSome SubunitsThe fact that BBS4 is among the last subunits to be incorporated into the BBSome puts it in a favorable position to interact with non-BBSome proteins including BBSome cargos. Previous studies have demonstrated that the BBSome moves from the cytoplasm to pericentriolar satellites then to cilia. BBS4 localizes to centrosome/pericentriolar satellites and interacts with the C terminus of PCM1 (10, 26). The functional relevance of this interaction is not clear. To determine whether the BBS4-PCM1 interaction is required for centrosome/pericentriolar satellite localization of BBS4, GFP-tagged WT and two mutant forms of BBS4 (BBS4G277E and BBS4R295P) were independently co-transfected with a HA-tagged C-terminal fragment of PCM1. Each of the two point mutations disrupted the interaction of BBS4 with PCM1 (Fig. 4A). When transfected into human RPE1 cells, both BBS4G277E and BBS4R295P lose the ability to localize to centrosome/pericentriolar satellites (Fig. 4B). These data suggest that the BBS4-PCM1 interaction is required for BBS4 centrosome/pericentriolar satellite localization.FIGURE 4.BBS4-PCM1 interaction is required for the BBSome to localize to centrosome/pericentriolar satellite.
A, BBS4 point mutations (G277E and R295P) disrupt the interaction between BBS4 and the C terminus of PCM1. GFP-tagged BBS4 (WT or mutants) and HA-tagged PCM1 C terminus were co-transfected into 293T cells, and co-immunoprecipitation (IP) was performed using antibody against HA and GFP. B, BBS4 point mutations (G277E and R295P) disrupt centrosome/pericentriolar satellite localization of BBS4. Shown is immunofluorescent staining of GFP-BBS4 (WT and mutants) in transfected RPE1 cells. γ-Tubulin (red) is used as a centrosomal marker. C, kidney primary cells were transfected with GFP-BBS4, and transfected cells were co-stained with γ-tubulin. The bar represents 10 μm.To determine whether the intact BBSome is required for BBS4 to localize to centrosome/pericentriolar satellites, we transfected cultured kidney cells from BBS knock-out mice with GFP-BBS4. Absence of BBS2, BBS6, or BBS7 does not affect BBS4 centrosome/pericentriolar satellite localization (Fig. 4C), indicating that this localization is an intrinsic feature of free BBS4 (occurs independent of the BBSome). To further characterize BBS4 interactions, we mapped the BBS4 domain that interacts with PCM1 to the N-terminal 380 amino acids (supplemental Fig. S3A). The N-terminal 380 amino acids of BBS4 do not contain the domain that binds with BBIP10, an integral BBSome protein that binds to the complex through BBS4 (27). Furthermore, we show that PCM1 can interact with BBIP10 only when BBS4 is present (supplemental Fig. S3B).

BBS4 Binds to PCM1 and Localizes to Centrosome/Pericentriolar Satellites Independent of Other BBSome Subunits

The fact that BBS4 is among the last subunits to be incorporated into the BBSome puts it in a favorable position to interact with non-BBSome proteins including BBSome cargos. Previous studies have demonstrated that the BBSome moves from the cytoplasm to pericentriolar satellites then to cilia. BBS4 localizes to centrosome/pericentriolar satellites and interacts with the C terminus of PCM1 (10, 26). The functional relevance of this interaction is not clear. To determine whether the BBS4-PCM1 interaction is required for centrosome/pericentriolar satellite localization of BBS4, GFP-tagged WT and two mutant forms of BBS4 (BBS4G277E and BBS4R295P) were independently co-transfected with a HA-tagged C-terminal fragment of PCM1. Each of the two point mutations disrupted the interaction of BBS4 with PCM1 (Fig. 4A). When transfected into human RPE1 cells, both BBS4G277E and BBS4R295P lose the ability to localize to centrosome/pericentriolar satellites (Fig. 4B). These data suggest that the BBS4-PCM1 interaction is required for BBS4 centrosome/pericentriolar satellite localization.

BBS4-PCM1 interaction is required for the BBSome to localize to centrosome/pericentriolar satellite.
A, BBS4 point mutations (G277E and R295P) disrupt the interaction between BBS4 and the C terminus of PCM1. GFP-tagged BBS4 (WT or mutants) and HA-tagged PCM1 C terminus were co-transfected into 293T cells, and co-immunoprecipitation (IP) was performed using antibody against HA and GFP. B, BBS4 point mutations (G277E and R295P) disrupt centrosome/pericentriolar satellite localization of BBS4. Shown is immunofluorescent staining of GFP-BBS4 (WT and mutants) in transfected RPE1 cells. γ-Tubulin (red) is used as a centrosomal marker. C, kidney primary cells were transfected with GFP-BBS4, and transfected cells were co-stained with γ-tubulin. The bar represents 10 μm.

To determine whether the intact BBSome is required for BBS4 to localize to centrosome/pericentriolar satellites, we transfected cultured kidney cells from BBS knock-out mice with GFP-BBS4. Absence of BBS2, BBS6, or BBS7 does not affect BBS4 centrosome/pericentriolar satellite localization (Fig. 4C), indicating that this localization is an intrinsic feature of free BBS4 (occurs independent of the BBSome). To further characterize BBS4 interactions, we mapped the BBS4 domain that interacts with PCM1 to the N-terminal 380 amino acids (supplemental Fig. S3A). The N-terminal 380 amino acids of BBS4 do not contain the domain that binds with BBIP10, an integral BBSome protein that binds to the complex through BBS4 (27). Furthermore, we show that PCM1 can interact with BBIP10 only when BBS4 is present (supplemental Fig. S3B).

BBS10 Regulates Interaction of BBS6-BBS12-BBS7 with CCT/TRiC Proteins to Form BBS-Chaperonin ComplexAs mentioned previously, studies showed that BBS7 form a complex with BBS6, BBS12, and CCT/TRiC proteins to form the BBS-chaperonin complex (20). The fact that BBS7 is the only non-chaperonin component in the BBS-chaperoinin complex suggests that BBS7 may be the substrate for BBS-chaperonin. One possible function of the BBS-chaperonin complex is the stabilization of BBS7. BBS10 is associated with this complex at substoichiometric levels (less than a 1:1 ratio). This finding suggests that BBS10 is not a structural component of the BBS-chaperonin complex but rather plays a role in regulating complex formation. To investigate the role of BBS10 in BBS-chaperonin complex formation, we utilized RNAi to knock down BBS10 in 293T cells expressing FLAG-BBS6, which serves as a convenient tool for immunoprecipitation of the BBS-chaperonin complex. Absence of BBS10 decreases the interaction of BBS6 with TCPα and TCPβ, two CCT/TRiC proteins (Fig. 5A). Furthermore, overexpression of BBS10 promotes the interaction of BBS6 with TCPα and TCPβ (Fig. 5B). These results demonstrate that BBS10 regulates the formation of the BBS-chaperonin complex.FIGURE 5.BBS10 is required for the interaction of BBS6-BBS12-BBS7 with the CCT/TRiC proteins to form the BBS-chaperonin complex.
A, BBS10 RNAi is used to knock down the expression of BBS10 in 293T cells expressing FLAG-BBS6, which was used to pull down the BBS-chaperonin complex. Antibodies against BBS7 and the CCT/TRiC proteins were used to detect the existence of these proteins in the complex. B, overexpression of BBS10 facilitates the association of BBS6 with the CCT/TRiC proteins. FLAG-BBS10 was co-transfected with empty vector or Myc-BBS6, and Myc antibody was used to pull down the BBS-chaperonin complex. Antibodies against BBS7, TCPα, and TCPβ (two subunits of the CCT/TRiC proteins) were used to detect these proteins in the complex. IP, immunoprecipitation.

BBS10 Regulates Interaction of BBS6-BBS12-BBS7 with CCT/TRiC Proteins to Form BBS-Chaperonin Complex

As mentioned previously, studies showed that BBS7 form a complex with BBS6, BBS12, and CCT/TRiC proteins to form the BBS-chaperonin complex (20). The fact that BBS7 is the only non-chaperonin component in the BBS-chaperoinin complex suggests that BBS7 may be the substrate for BBS-chaperonin. One possible function of the BBS-chaperonin complex is the stabilization of BBS7. BBS10 is associated with this complex at substoichiometric levels (less than a 1:1 ratio). This finding suggests that BBS10 is not a structural component of the BBS-chaperonin complex but rather plays a role in regulating complex formation. To investigate the role of BBS10 in BBS-chaperonin complex formation, we utilized RNAi to knock down BBS10 in 293T cells expressing FLAG-BBS6, which serves as a convenient tool for immunoprecipitation of the BBS-chaperonin complex. Absence of BBS10 decreases the interaction of BBS6 with TCPα and TCPβ, two CCT/TRiC proteins (Fig. 5A). Furthermore, overexpression of BBS10 promotes the interaction of BBS6 with TCPα and TCPβ (Fig. 5B). These results demonstrate that BBS10 regulates the formation of the BBS-chaperonin complex.

BBS10 is required for the interaction of BBS6-BBS12-BBS7 with the CCT/TRiC proteins to form the BBS-chaperonin complex.
A, BBS10 RNAi is used to knock down the expression of BBS10 in 293T cells expressing FLAG-BBS6, which was used to pull down the BBS-chaperonin complex. Antibodies against BBS7 and the CCT/TRiC proteins were used to detect the existence of these proteins in the complex. B, overexpression of BBS10 facilitates the association of BBS6 with the CCT/TRiC proteins. FLAG-BBS10 was co-transfected with empty vector or Myc-BBS6, and Myc antibody was used to pull down the BBS-chaperonin complex. Antibodies against BBS7, TCPα, and TCPβ (two subunits of the CCT/TRiC proteins) were used to detect these proteins in the complex. IP, immunoprecipitation.

Release of BBS7 from BBS-Chaperonin Complex Is Coordinated with BBSome Core Complex FormationThe mature form of the BBSome does not contain BBS6, BBS12, or CCT/TRiC proteins, suggesting these chaperonin components must be dissociated from assembly intermediates at some point. To determine when these components are released from the assembly intermediates, we overexpressed FLAG-tagged BBS6 in 293T cells and immunoprecipitated with FLAG antibody. As expected, other BBS-chaperonin complex components, as well as endogenous BBS7 were co-immunoprecipitated along with BBS6 (Fig. 6A). In addition, BBS6 pulled down some BBS2. However, the amount of BBS2 pulled down relative to BBS7 was less than the amounts of BBS2 found in the BBSome (as shown by pulling down the BBSome with BBS5). This indicates that the interaction of BBS2 with BBS7 is coupled with the release of BBS6 and BBS12 from the BBS-chaperonin complex (supplemental Fig. S4).FIGURE 6.The release of BBS7 from the BBS-chaperonin complex is coordinated with BBSome core complex formation.
A, FLAG-BBS6 was overexpressed in 239T cells to trap BBSome intermediates. We pulled down BBS6 using FLAG beads and detected endogenous BBSome subunits by Western blot. FLAG-BBS5 was used as a positive control, and FLAG-GFP was used as a negative control. B, individual BBSome subunits were overexpressed in 293T cells and were used to determine specific associations with the CCT/TRiC proteins. Only BBS7 and BBS2 can associate with CCT/TRiC proteins. FLAG-VHL was used as a positive control. FLAG-GFP and FLAG-BBS3 were used as negative controls. IP, immunoprecipitation.Next we examined the release of the CCT/TRiC proteins from the assembly intermediates. We overexpressed each BBSome subunit in 293T cells and determined whether any BBSome subunit can immunoprecipitate the CCT/TRiC proteins. BBS2 and BBS7 showed robust pulldown of CCT/TRiC proteins. BBS9 showed relatively less pulldown of the CCT/TRiC proteins compared with BBS2 and BBS7 (Fig. 6B). This result indicates that the incorporation of BBS9 to form the BBSome core complex is coupled with the release of BBS2/BBS7 from CCT/TRiC proteins.

Release of BBS7 from BBS-Chaperonin Complex Is Coordinated with BBSome Core Complex Formation

The mature form of the BBSome does not contain BBS6, BBS12, or CCT/TRiC proteins, suggesting these chaperonin components must be dissociated from assembly intermediates at some point. To determine when these components are released from the assembly intermediates, we overexpressed FLAG-tagged BBS6 in 293T cells and immunoprecipitated with FLAG antibody. As expected, other BBS-chaperonin complex components, as well as endogenous BBS7 were co-immunoprecipitated along with BBS6 (Fig. 6A). In addition, BBS6 pulled down some BBS2. However, the amount of BBS2 pulled down relative to BBS7 was less than the amounts of BBS2 found in the BBSome (as shown by pulling down the BBSome with BBS5). This indicates that the interaction of BBS2 with BBS7 is coupled with the release of BBS6 and BBS12 from the BBS-chaperonin complex (supplemental Fig. S4).

The release of BBS7 from the BBS-chaperonin complex is coordinated with BBSome core complex formation.
A, FLAG-BBS6 was overexpressed in 239T cells to trap BBSome intermediates. We pulled down BBS6 using FLAG beads and detected endogenous BBSome subunits by Western blot. FLAG-BBS5 was used as a positive control, and FLAG-GFP was used as a negative control. B, individual BBSome subunits were overexpressed in 293T cells and were used to determine specific associations with the CCT/TRiC proteins. Only BBS7 and BBS2 can associate with CCT/TRiC proteins. FLAG-VHL was used as a positive control. FLAG-GFP and FLAG-BBS3 were used as negative controls. IP, immunoprecipitation.

Next we examined the release of the CCT/TRiC proteins from the assembly intermediates. We overexpressed each BBSome subunit in 293T cells and determined whether any BBSome subunit can immunoprecipitate the CCT/TRiC proteins. BBS2 and BBS7 showed robust pulldown of CCT/TRiC proteins. BBS9 showed relatively less pulldown of the CCT/TRiC proteins compared with BBS2 and BBS7 (Fig. 6B). This result indicates that the incorporation of BBS9 to form the BBSome core complex is coupled with the release of BBS2/BBS7 from CCT/TRiC proteins.

Stability of BBS2 Is Dependent on Its Interaction with Other ProteinsWe have demonstrated that BBS2, BBS7, and BBS9 form a core complex (Fig. 1). To determine the importance of this complex, we examined the protein level of BBS2 in the absence of BBS7 and BBS9, as well as in the absence of the BBS-chaperonin complex. Absence of BBS7 greatly decreases BBS2 protein levels in testes tissue from Bbs7−/− mice. Furthermore, the absence of BBS6 (a component of the BBS-chaperonin complex) in Bbs6−/− mice also results in decreased amounts of BBS2 (Fig. 7A). Because we do not have Bbs10 knock-out mice, we utilize skin fibroblast cells from a human patient who is homozygous for the common BBS10 mutation, c91fs95. This mutation is predicted to result in the total absence of the BBS10 protein. In our study, using human fibroblast cells, the absence of BBS10 does not affect cilia formation (supplemental Fig. S5), in contrast to a previous report (28). However, the absence of BBS10 lowers BBS2 protein levels (Fig. 7A) and inhibits BBSome formation (supplemental Fig. S6).FIGURE 7.BBS2 is ubiquitinated and degraded by the proteasome.
A, the stability of BBS2 is dependent on the BBSome core complex, as well as the BBS-chaperonin complex. Proteins from BBS knock-out mouse tissue (testis) or from RNAi-treated cell lysates were separated by SDS-PAGE. Anti-BBS2 antibody was used to detect BBS2 protein levels. B, BBS2 is sensitive to protease treatment. Primary cells from WT and Bbs6−/− kidney were infected with adenovirus expressing FLAG-BBS2. Equal amounts of total proteins were mixed with protease thermolysin at 4 °C for the indicated time. Compared with WT cells, BBS2 degrades faster in Bbs6−/− cells, suggesting that BBS2 is not stable in the absence of BBS6. C, 293T cells and FLAG-BBS2 stable-expressing cells were treated with the proteasome inhibitor MG132. BBS2 was immunoprecipitated (IP) by FLAG beads or anti-BBS2 antibody, and ubiquitinated BBS2 was detected by anti-ubiquitin antibody. D, HA-ubiquitin was transfected into 293T cells, and endogenous BBS2 was immunoprecipitated by anti-BBS2 antibody. Ubiquitinated BBS2 was detected by anti-HA antibody or anti-ubiquitin antibody. E, 293T cells were treated with 50 μg/ml of cycloheximide (Chx) with or without 20 μm proteasome inhibitor MG132 or MG115 for 8 h. Endogenous BBS2 in the cell lysates was detected by anti-BBS2 antibody.Next, we used RNAi in RPE1 cells to individually reduce expression of BBS9 (BBSome core complex component), BBS12 and TCPβ (BBS-chaperonin complex components). Reduction of each of these proteins greatly decreases BBS2 protein levels. In contrast, the absence of the normal BBSome proteins BBS4 or BBS1 (using tissue from Bbs4−/− and Bbs1M390R mutant mice, respectively) does not affect BBS2 protein levels (Fig. 7A). These data indicate that the stability of BBS2, a component of the BBSome core complex, requires the BBS-chaperonin complex, as well as the other two components of the BBSome core complex (BBS7 and BBS9). Similarly, the stability of BBS7 requires the BBS-chaperonin complex, as well as the other two components of the BBSome core complex (BBS2 and BBS9) (supplemental Fig. S7).To determine how the BBS-chaperonin complex influences BBS2 protein stability, we infected Bbs6−/− kidney cells and WT kidney cells with adenovirus expressing FLAG-BBS2. The same multiplicity of infection was used to infect WT and Bbs6−/− kidney cells, and equal amounts of proteins were used for the analysis. The BBS2 protein level is lower in Bbs6−/− cells than in WT cells (Fig. 7B, 0'), consistent with the finding that the BBS-chaperonin complex is required for BBS2 protein stability. The remaining BBS2 protein in Bbs6−/− cells is more sensitive to protease thermolysin treatment. The half-life of BBS2 in Bbs6−/− cells is reduced ∼3-fold (4 min in WT cells versus 12 min in Bbs6−/− cells) (Fig. 7B). Similar results are obtained when we reintroduce FLAG-BBS2 into Bbs7−/− kidney cells (supplemental Fig. S7). These data indicate that the BBS-chaperonin complex and BBS7 play a role in BBS2 stability.

Stability of BBS2 Is Dependent on Its Interaction with Other Proteins

We have demonstrated that BBS2, BBS7, and BBS9 form a core complex (Fig. 1). To determine the importance of this complex, we examined the protein level of BBS2 in the absence of BBS7 and BBS9, as well as in the absence of the BBS-chaperonin complex. Absence of BBS7 greatly decreases BBS2 protein levels in testes tissue from Bbs7−/− mice. Furthermore, the absence of BBS6 (a component of the BBS-chaperonin complex) in Bbs6−/− mice also results in decreased amounts of BBS2 (Fig. 7A). Because we do not have Bbs10 knock-out mice, we utilize skin fibroblast cells from a human patient who is homozygous for the common BBS10 mutation, c91fs95. This mutation is predicted to result in the total absence of the BBS10 protein. In our study, using human fibroblast cells, the absence of BBS10 does not affect cilia formation (supplemental Fig. S5), in contrast to a previous report (28). However, the absence of BBS10 lowers BBS2 protein levels (Fig. 7A) and inhibits BBSome formation (supplemental Fig. S6).

BBS2 is ubiquitinated and degraded by the proteasome.
A, the stability of BBS2 is dependent on the BBSome core complex, as well as the BBS-chaperonin complex. Proteins from BBS knock-out mouse tissue (testis) or from RNAi-treated cell lysates were separated by SDS-PAGE. Anti-BBS2 antibody was used to detect BBS2 protein levels. B, BBS2 is sensitive to protease treatment. Primary cells from WT and Bbs6−/− kidney were infected with adenovirus expressing FLAG-BBS2. Equal amounts of total proteins were mixed with protease thermolysin at 4 °C for the indicated time. Compared with WT cells, BBS2 degrades faster in Bbs6−/− cells, suggesting that BBS2 is not stable in the absence of BBS6. C, 293T cells and FLAG-BBS2 stable-expressing cells were treated with the proteasome inhibitor MG132. BBS2 was immunoprecipitated (IP) by FLAG beads or anti-BBS2 antibody, and ubiquitinated BBS2 was detected by anti-ubiquitin antibody. D, HA-ubiquitin was transfected into 293T cells, and endogenous BBS2 was immunoprecipitated by anti-BBS2 antibody. Ubiquitinated BBS2 was detected by anti-HA antibody or anti-ubiquitin antibody. E, 293T cells were treated with 50 μg/ml of cycloheximide (Chx) with or without 20 μm proteasome inhibitor MG132 or MG115 for 8 h. Endogenous BBS2 in the cell lysates was detected by anti-BBS2 antibody.

Next, we used RNAi in RPE1 cells to individually reduce expression of BBS9 (BBSome core complex component), BBS12 and TCPβ (BBS-chaperonin complex components). Reduction of each of these proteins greatly decreases BBS2 protein levels. In contrast, the absence of the normal BBSome proteins BBS4 or BBS1 (using tissue from Bbs4−/− and Bbs1M390R mutant mice, respectively) does not affect BBS2 protein levels (Fig. 7A). These data indicate that the stability of BBS2, a component of the BBSome core complex, requires the BBS-chaperonin complex, as well as the other two components of the BBSome core complex (BBS7 and BBS9). Similarly, the stability of BBS7 requires the BBS-chaperonin complex, as well as the other two components of the BBSome core complex (BBS2 and BBS9) (supplemental Fig. S7).

To determine how the BBS-chaperonin complex influences BBS2 protein stability, we infected Bbs6−/− kidney cells and WT kidney cells with adenovirus expressing FLAG-BBS2. The same multiplicity of infection was used to infect WT and Bbs6−/− kidney cells, and equal amounts of proteins were used for the analysis. The BBS2 protein level is lower in Bbs6−/− cells than in WT cells (Fig. 7B, 0'), consistent with the finding that the BBS-chaperonin complex is required for BBS2 protein stability. The remaining BBS2 protein in Bbs6−/− cells is more sensitive to protease thermolysin treatment. The half-life of BBS2 in Bbs6−/− cells is reduced ∼3-fold (4 min in WT cells versus 12 min in Bbs6−/− cells) (Fig. 7B). Similar results are obtained when we reintroduce FLAG-BBS2 into Bbs7−/− kidney cells (supplemental Fig. S7). These data indicate that the BBS-chaperonin complex and BBS7 play a role in BBS2 stability.

The Ubiquitin-Proteasome Pathway Is Involved in BBS2 DegradationTo investigate the mechanism by which BBS2 is degraded in cells, we examined whether the ubiquitin-proteasome pathway is involved in this process. Both exogenously expressed and endogenously expressed BBS2 are ubiquitinated when the proteasome is inhibited by MG132 (Fig. 7C). We further demonstrate that endogenous BBS2 is ubiquitinated even without inhibition of the proteasome when we overexpressed HA-ubiquitin in 293T cells (Fig. 7D). Treatment of 293T cells with the protein synthesis inhibitor cycloheximide resulted in reduced BBS2 protein levels. The reduced BBS2 protein levels in cycloheximide-treated cells are increased when cycloheximide-treated cells are also treated with the proteasome inhibitors MG132 or MG115 (Fig. 7E). These results demonstrate that free BBS2 is ubiquitinated and degraded by the proteasome.

The Ubiquitin-Proteasome Pathway Is Involved in BBS2 Degradation

To investigate the mechanism by which BBS2 is degraded in cells, we examined whether the ubiquitin-proteasome pathway is involved in this process. Both exogenously expressed and endogenously expressed BBS2 are ubiquitinated when the proteasome is inhibited by MG132 (Fig. 7C). We further demonstrate that endogenous BBS2 is ubiquitinated even without inhibition of the proteasome when we overexpressed HA-ubiquitin in 293T cells (Fig. 7D). Treatment of 293T cells with the protein synthesis inhibitor cycloheximide resulted in reduced BBS2 protein levels. The reduced BBS2 protein levels in cycloheximide-treated cells are increased when cycloheximide-treated cells are also treated with the proteasome inhibitors MG132 or MG115 (Fig. 7E). These results demonstrate that free BBS2 is ubiquitinated and degraded by the proteasome.

BBSome Assembly ModelBased on the data presented above, we propose a model in which the BBSome forms in an ordered sequential process assisted by both intrinsic protein-protein interactions between BBSome proteins, as well as by the non-BBSome chaperonin-like BBS proteins (BBS6, BBS10, and BBS12) (Fig. 8). The BBS-chaperonin complex interacts with BBS7 to stabilize it. BBS7 transitions from the BBS-chaperonin complex to the BBSome core complex, a complex consisting of BBS7-BBS2-BBS9. The formation of the BBSome core complex is accompanied by the removal of the BBS-chaperonin complex. Intrinsic protein-protein interactions promote the incorporation of BBS1, BBS5, and BBS8. BBS4 is added to complete the BBSome, a step requiring the presence of BBS1. The BBS-chaperonin complex is required for the individual stability of BBS2 and BBS7 proteins prior to these proteins participating with BBS9 to form the BBSome core complex.FIGURE 8.Intrinsic protein-protein interactions and chaperonin mediated sequential assembly of the BBSome. Shown are diagrams of our current model for BBSome assembly. BBS7 is stabilized by the BBS-chaperonin complex (BBS6-BBS12-BBS7-CCT) and makes a transition to the BBSome core complex (BBS7-BBS2-BBS9). This transition requires BBS10. The intrinsic protein-protein interactions guide the incorporation of BBS1, BBS5, and BBS8. BBS4 requires BBS1 to incorporate into the complex and is the last subunit to be added to the complex.

BBSome Assembly Model

Based on the data presented above, we propose a model in which the BBSome forms in an ordered sequential process assisted by both intrinsic protein-protein interactions between BBSome proteins, as well as by the non-BBSome chaperonin-like BBS proteins (BBS6, BBS10, and BBS12) (Fig. 8). The BBS-chaperonin complex interacts with BBS7 to stabilize it. BBS7 transitions from the BBS-chaperonin complex to the BBSome core complex, a complex consisting of BBS7-BBS2-BBS9. The formation of the BBSome core complex is accompanied by the removal of the BBS-chaperonin complex. Intrinsic protein-protein interactions promote the incorporation of BBS1, BBS5, and BBS8. BBS4 is added to complete the BBSome, a step requiring the presence of BBS1. The BBS-chaperonin complex is required for the individual stability of BBS2 and BBS7 proteins prior to these proteins participating with BBS9 to form the BBSome core complex.

Intrinsic protein-protein interactions and chaperonin mediated sequential assembly of the BBSome. Shown are diagrams of our current model for BBSome assembly. BBS7 is stabilized by the BBS-chaperonin complex (BBS6-BBS12-BBS7-CCT) and makes a transition to the BBSome core complex (BBS7-BBS2-BBS9). This transition requires BBS10. The intrinsic protein-protein interactions guide the incorporation of BBS1, BBS5, and BBS8. BBS4 requires BBS1 to incorporate into the complex and is the last subunit to be added to the complex.

DISCUSSIONThe high prevalence of obesity, diabetes, and retinal degeneration in the general population makes the study of BBS clinically relevant beyond those with this relatively rare syndrome. Indeed, BBS gene variants have been associated with both childhood and adult common obesity in at least one population (29). Further understanding the molecular pathophysiology of BBS will provide useful information related to these common disorders. At least a subset of the functions of individual BBS proteins are dependent on the formation of the BBSome, a stable multi-subunit complex consisting of BBS1, BBS2, BBS4, BBS5, BBS7, BBS8, BBS9, and BBIP10, a protein in which mutations have yet to be implicated in disease. Previous studies of BBSome function have been focused on the identification of proteins or cargos that interact with the BBSome (14, 15, 26, 30–32). Here, we approached the molecular pathophysiology of BBS from the standpoint of the BBSome assembly. Our study directly links the BBS-chaperonin complex (consisting of BBS6, BBS12, BBS7, and the CCT/TRiC proteins) to the BBSome, two complexes identified through biochemical analysis.Intrinsic Protein-Protein Interactions and Chaperonin-like BBS Proteins Guide Ordered BBSome AssemblyIn general, large protein complex assembly is an ordered process, usually requiring multiple intermediate steps (1, 3, 5). It is often difficult to identify the nature of the assembly intermediates because of their transient nature and short half-life. The specific assembly order of protein complexes can potentially play an important role in determining the nature of phenotypes resulting from mutations in components of the complex. In some instances, a mutation in one protein of a complex could result in accumulation of an assembly intermediate that maintains residual function or a gain of function, whereas a different assembly order could result in a complete lack of assembly and a total loss of function.In this study, we applied multiple approaches and resources to characterize BBSome assembly intermediates. Point mutations in specific BBS proteins, null mutations completely lacking some BBS proteins, RNA interference to reduce BBS protein levels, as well as overexpression of specific BBSome subunits result in the intracellular accumulation of distinct assembly intermediates. The assembly of the BBSome is dictated by the intrinsic protein-protein interactions among BBSome proteins and is also guided as by the chaperonin-like BBS proteins BBS6, BBS10, and BBS12. Structural analysis reveals that BBS1, BBS2, BBS7, and BBS9 contain β propeller domains, BBS4 and BBS8 contain multiple tetratricopeptide repeat domains, and BBS5 has two pleckstrin homology domains. These domains typically mediate protein-protein interactions. Pairwise co-immunoprecipitation assays showed that BBS7 interacts with BBS2 and that BBS9 interacts with BBS1, BBS2, BBS5, and BBS8. BBS1 was shown to interact with BBS2 and BBS4. The BBSome assembly process reflects these protein-protein interactions. The chaperonin-like BBS proteins (BBS6, BBS10, and BBS12) are essential in initiating the first step of the BBSome formation by forming the BBS-chaperonin complex to stabilize BBS7.BBS2, BBS7, and BBS9 Form BBSome Core ComplexOur study identified BBS2-BBS7-BBS9 as an important intermediate complex in the assembly of the mature BBSome. Formation of this BBSome core complex is coordinated with the release of the BBS-chaperonin complex from BBS7. One function of this BBS-chaperonin complex appears to be to stabilize BBS7 prior to its association with BBS2 and BBS9. The absence of BBS2, BBS7, BBS9, or any of the BBS-chaperonin complex components decreases the levels of BBS2 and BBS7, whereas the absence or mutation of BBS1 or BBS4 has a minimal effect on BBS2 and BBS7 protein stability. Free BBS2 is degraded by the ubiquitin-proteasome pathway. The ubiquitin ligase for BBS2 is currently unknown.BBS11/TRIM32 is an E3 ligase for several proteins including actin and dysbindin (33, 34). The specific mutation of P130S in one family causes BBS phenotypes (35). It is intriguing to speculate that the putative E3 ligase for BBS2 might be BBS11/TRIM32.Regulation of BBSome FunctionThere are various possible mechanisms by which the regulation of BBSome function could occur. One possibility is that regulation occurs after the BBSome is fully assembled, and the movement of the BBSome into its functional cellular compartment (i.e., cilia) is regulated. Both positive (RAB8) and negative (LZTFL1) BBSome interactors have been identified (10, 25). However, regulation of assembly is also a likely possibility. Based on our study, regulation of assembly would most likely occur at the level of the formation of the BBS-chaperonin complex, at the level of the formation of the BBSome core complex, and/or at the level of the release BBS6, BBS12, and CCT/TRiC proteins from BBS7. BBS10 is likely a key regulatory component for the BBS-chaperonin complex.Physiological and Clinical Significance of BBSome Intermediates: A Role for BBSome Assembly Intermediates in BBS Phenotype VariabilityUnder normal conditions, there appears to be a limited free pool of individual BBSome subunits with the vast majority of the subunits being in BBSome complex form. However, BBSome assembly intermediates such as the BBSome core complex may form under conditions such as occur when specific null or point mutations exist in BBS patients. The phenotypes of BBS vary significantly between different families and even within members of the same family (individuals with the same specific mutation) (36–38). These observations suggest the possibility of genetic modifiers. Phenotype modification can occur when a heterozygous variant in one gene influences the phenotypic manifestation of another causative gene. Because some mutations may lead to the accumulation of stable assembly intermediate, the presence of the intermediate may explain some of the phenotypic differences observed among different BBS gene mutations. One interesting example is the hypertension phenotype that is manifested in Bbs4−/− mice, whereas Bbs2−/− mice do not have hypertension (39, 40). Furthermore, physiological differences among human BBS patients caused by mutations in distinct BBS genes have also been reported (37). The mechanism by which this might occur will require further investigation.The elucidation of the stepwise manner by which the BBSome is formed suggests the possibility of cell type-specific forms of the BBSome. In other words, the fact that some BBSome proteins (e.g., BBS1, BBS5, and BBS8) are added independently to a stable intermediate complex allows for the possibility that BBSome heterogeneity exists and that the heterogeneous forms of the BBSome may have functional roles in specific tissues.

The high prevalence of obesity, diabetes, and retinal degeneration in the general population makes the study of BBS clinically relevant beyond those with this relatively rare syndrome. Indeed, BBS gene variants have been associated with both childhood and adult common obesity in at least one population (29). Further understanding the molecular pathophysiology of BBS will provide useful information related to these common disorders. At least a subset of the functions of individual BBS proteins are dependent on the formation of the BBSome, a stable multi-subunit complex consisting of BBS1, BBS2, BBS4, BBS5, BBS7, BBS8, BBS9, and BBIP10, a protein in which mutations have yet to be implicated in disease. Previous studies of BBSome function have been focused on the identification of proteins or cargos that interact with the BBSome (14, 15, 26, 30–32). Here, we approached the molecular pathophysiology of BBS from the standpoint of the BBSome assembly. Our study directly links the BBS-chaperonin complex (consisting of BBS6, BBS12, BBS7, and the CCT/TRiC proteins) to the BBSome, two complexes identified through biochemical analysis.

Intrinsic Protein-Protein Interactions and Chaperonin-like BBS Proteins Guide Ordered BBSome AssemblyIn general, large protein complex assembly is an ordered process, usually requiring multiple intermediate steps (1, 3, 5). It is often difficult to identify the nature of the assembly intermediates because of their transient nature and short half-life. The specific assembly order of protein complexes can potentially play an important role in determining the nature of phenotypes resulting from mutations in components of the complex. In some instances, a mutation in one protein of a complex could result in accumulation of an assembly intermediate that maintains residual function or a gain of function, whereas a different assembly order could result in a complete lack of assembly and a total loss of function.In this study, we applied multiple approaches and resources to characterize BBSome assembly intermediates. Point mutations in specific BBS proteins, null mutations completely lacking some BBS proteins, RNA interference to reduce BBS protein levels, as well as overexpression of specific BBSome subunits result in the intracellular accumulation of distinct assembly intermediates. The assembly of the BBSome is dictated by the intrinsic protein-protein interactions among BBSome proteins and is also guided as by the chaperonin-like BBS proteins BBS6, BBS10, and BBS12. Structural analysis reveals that BBS1, BBS2, BBS7, and BBS9 contain β propeller domains, BBS4 and BBS8 contain multiple tetratricopeptide repeat domains, and BBS5 has two pleckstrin homology domains. These domains typically mediate protein-protein interactions. Pairwise co-immunoprecipitation assays showed that BBS7 interacts with BBS2 and that BBS9 interacts with BBS1, BBS2, BBS5, and BBS8. BBS1 was shown to interact with BBS2 and BBS4. The BBSome assembly process reflects these protein-protein interactions. The chaperonin-like BBS proteins (BBS6, BBS10, and BBS12) are essential in initiating the first step of the BBSome formation by forming the BBS-chaperonin complex to stabilize BBS7.BBS2, BBS7, and BBS9 Form BBSome Core ComplexOur study identified BBS2-BBS7-BBS9 as an important intermediate complex in the assembly of the mature BBSome. Formation of this BBSome core complex is coordinated with the release of the BBS-chaperonin complex from BBS7. One function of this BBS-chaperonin complex appears to be to stabilize BBS7 prior to its association with BBS2 and BBS9. The absence of BBS2, BBS7, BBS9, or any of the BBS-chaperonin complex components decreases the levels of BBS2 and BBS7, whereas the absence or mutation of BBS1 or BBS4 has a minimal effect on BBS2 and BBS7 protein stability. Free BBS2 is degraded by the ubiquitin-proteasome pathway. The ubiquitin ligase for BBS2 is currently unknown.BBS11/TRIM32 is an E3 ligase for several proteins including actin and dysbindin (33, 34). The specific mutation of P130S in one family causes BBS phenotypes (35). It is intriguing to speculate that the putative E3 ligase for BBS2 might be BBS11/TRIM32.Regulation of BBSome FunctionThere are various possible mechanisms by which the regulation of BBSome function could occur. One possibility is that regulation occurs after the BBSome is fully assembled, and the movement of the BBSome into its functional cellular compartment (i.e., cilia) is regulated. Both positive (RAB8) and negative (LZTFL1) BBSome interactors have been identified (10, 25). However, regulation of assembly is also a likely possibility. Based on our study, regulation of assembly would most likely occur at the level of the formation of the BBS-chaperonin complex, at the level of the formation of the BBSome core complex, and/or at the level of the release BBS6, BBS12, and CCT/TRiC proteins from BBS7. BBS10 is likely a key regulatory component for the BBS-chaperonin complex.Physiological and Clinical Significance of BBSome Intermediates: A Role for BBSome Assembly Intermediates in BBS Phenotype VariabilityUnder normal conditions, there appears to be a limited free pool of individual BBSome subunits with the vast majority of the subunits being in BBSome complex form. However, BBSome assembly intermediates such as the BBSome core complex may form under conditions such as occur when specific null or point mutations exist in BBS patients. The phenotypes of BBS vary significantly between different families and even within members of the same family (individuals with the same specific mutation) (36–38). These observations suggest the possibility of genetic modifiers. Phenotype modification can occur when a heterozygous variant in one gene influences the phenotypic manifestation of another causative gene. Because some mutations may lead to the accumulation of stable assembly intermediate, the presence of the intermediate may explain some of the phenotypic differences observed among different BBS gene mutations. One interesting example is the hypertension phenotype that is manifested in Bbs4−/− mice, whereas Bbs2−/− mice do not have hypertension (39, 40). Furthermore, physiological differences among human BBS patients caused by mutations in distinct BBS genes have also been reported (37). The mechanism by which this might occur will require further investigation.The elucidation of the stepwise manner by which the BBSome is formed suggests the possibility of cell type-specific forms of the BBSome. In other words, the fact that some BBSome proteins (e.g., BBS1, BBS5, and BBS8) are added independently to a stable intermediate complex allows for the possibility that BBSome heterogeneity exists and that the heterogeneous forms of the BBSome may have functional roles in specific tissues.

Intrinsic Protein-Protein Interactions and Chaperonin-like BBS Proteins Guide Ordered BBSome AssemblyIn general, large protein complex assembly is an ordered process, usually requiring multiple intermediate steps (1, 3, 5). It is often difficult to identify the nature of the assembly intermediates because of their transient nature and short half-life. The specific assembly order of protein complexes can potentially play an important role in determining the nature of phenotypes resulting from mutations in components of the complex. In some instances, a mutation in one protein of a complex could result in accumulation of an assembly intermediate that maintains residual function or a gain of function, whereas a different assembly order could result in a complete lack of assembly and a total loss of function.In this study, we applied multiple approaches and resources to characterize BBSome assembly intermediates. Point mutations in specific BBS proteins, null mutations completely lacking some BBS proteins, RNA interference to reduce BBS protein levels, as well as overexpression of specific BBSome subunits result in the intracellular accumulation of distinct assembly intermediates. The assembly of the BBSome is dictated by the intrinsic protein-protein interactions among BBSome proteins and is also guided as by the chaperonin-like BBS proteins BBS6, BBS10, and BBS12. Structural analysis reveals that BBS1, BBS2, BBS7, and BBS9 contain β propeller domains, BBS4 and BBS8 contain multiple tetratricopeptide repeat domains, and BBS5 has two pleckstrin homology domains. These domains typically mediate protein-protein interactions. Pairwise co-immunoprecipitation assays showed that BBS7 interacts with BBS2 and that BBS9 interacts with BBS1, BBS2, BBS5, and BBS8. BBS1 was shown to interact with BBS2 and BBS4. The BBSome assembly process reflects these protein-protein interactions. The chaperonin-like BBS proteins (BBS6, BBS10, and BBS12) are essential in initiating the first step of the BBSome formation by forming the BBS-chaperonin complex to stabilize BBS7.BBS2, BBS7, and BBS9 Form BBSome Core ComplexOur study identified BBS2-BBS7-BBS9 as an important intermediate complex in the assembly of the mature BBSome. Formation of this BBSome core complex is coordinated with the release of the BBS-chaperonin complex from BBS7. One function of this BBS-chaperonin complex appears to be to stabilize BBS7 prior to its association with BBS2 and BBS9. The absence of BBS2, BBS7, BBS9, or any of the BBS-chaperonin complex components decreases the levels of BBS2 and BBS7, whereas the absence or mutation of BBS1 or BBS4 has a minimal effect on BBS2 and BBS7 protein stability. Free BBS2 is degraded by the ubiquitin-proteasome pathway. The ubiquitin ligase for BBS2 is currently unknown.BBS11/TRIM32 is an E3 ligase for several proteins including actin and dysbindin (33, 34). The specific mutation of P130S in one family causes BBS phenotypes (35). It is intriguing to speculate that the putative E3 ligase for BBS2 might be BBS11/TRIM32.Regulation of BBSome FunctionThere are various possible mechanisms by which the regulation of BBSome function could occur. One possibility is that regulation occurs after the BBSome is fully assembled, and the movement of the BBSome into its functional cellular compartment (i.e., cilia) is regulated. Both positive (RAB8) and negative (LZTFL1) BBSome interactors have been identified (10, 25). However, regulation of assembly is also a likely possibility. Based on our study, regulation of assembly would most likely occur at the level of the formation of the BBS-chaperonin complex, at the level of the formation of the BBSome core complex, and/or at the level of the release BBS6, BBS12, and CCT/TRiC proteins from BBS7. BBS10 is likely a key regulatory component for the BBS-chaperonin complex.Physiological and Clinical Significance of BBSome Intermediates: A Role for BBSome Assembly Intermediates in BBS Phenotype VariabilityUnder normal conditions, there appears to be a limited free pool of individual BBSome subunits with the vast majority of the subunits being in BBSome complex form. However, BBSome assembly intermediates such as the BBSome core complex may form under conditions such as occur when specific null or point mutations exist in BBS patients. The phenotypes of BBS vary significantly between different families and even within members of the same family (individuals with the same specific mutation) (36–38). These observations suggest the possibility of genetic modifiers. Phenotype modification can occur when a heterozygous variant in one gene influences the phenotypic manifestation of another causative gene. Because some mutations may lead to the accumulation of stable assembly intermediate, the presence of the intermediate may explain some of the phenotypic differences observed among different BBS gene mutations. One interesting example is the hypertension phenotype that is manifested in Bbs4−/− mice, whereas Bbs2−/− mice do not have hypertension (39, 40). Furthermore, physiological differences among human BBS patients caused by mutations in distinct BBS genes have also been reported (37). The mechanism by which this might occur will require further investigation.The elucidation of the stepwise manner by which the BBSome is formed suggests the possibility of cell type-specific forms of the BBSome. In other words, the fact that some BBSome proteins (e.g., BBS1, BBS5, and BBS8) are added independently to a stable intermediate complex allows for the possibility that BBSome heterogeneity exists and that the heterogeneous forms of the BBSome may have functional roles in specific tissues.

Intrinsic Protein-Protein Interactions and Chaperonin-like BBS Proteins Guide Ordered BBSome AssemblyIn general, large protein complex assembly is an ordered process, usually requiring multiple intermediate steps (1, 3, 5). It is often difficult to identify the nature of the assembly intermediates because of their transient nature and short half-life. The specific assembly order of protein complexes can potentially play an important role in determining the nature of phenotypes resulting from mutations in components of the complex. In some instances, a mutation in one protein of a complex could result in accumulation of an assembly intermediate that maintains residual function or a gain of function, whereas a different assembly order could result in a complete lack of assembly and a total loss of function.In this study, we applied multiple approaches and resources to characterize BBSome assembly intermediates. Point mutations in specific BBS proteins, null mutations completely lacking some BBS proteins, RNA interference to reduce BBS protein levels, as well as overexpression of specific BBSome subunits result in the intracellular accumulation of distinct assembly intermediates. The assembly of the BBSome is dictated by the intrinsic protein-protein interactions among BBSome proteins and is also guided as by the chaperonin-like BBS proteins BBS6, BBS10, and BBS12. Structural analysis reveals that BBS1, BBS2, BBS7, and BBS9 contain β propeller domains, BBS4 and BBS8 contain multiple tetratricopeptide repeat domains, and BBS5 has two pleckstrin homology domains. These domains typically mediate protein-protein interactions. Pairwise co-immunoprecipitation assays showed that BBS7 interacts with BBS2 and that BBS9 interacts with BBS1, BBS2, BBS5, and BBS8. BBS1 was shown to interact with BBS2 and BBS4. The BBSome assembly process reflects these protein-protein interactions. The chaperonin-like BBS proteins (BBS6, BBS10, and BBS12) are essential in initiating the first step of the BBSome formation by forming the BBS-chaperonin complex to stabilize BBS7.

Intrinsic Protein-Protein Interactions and Chaperonin-like BBS Proteins Guide Ordered BBSome Assembly

In general, large protein complex assembly is an ordered process, usually requiring multiple intermediate steps (1, 3, 5). It is often difficult to identify the nature of the assembly intermediates because of their transient nature and short half-life. The specific assembly order of protein complexes can potentially play an important role in determining the nature of phenotypes resulting from mutations in components of the complex. In some instances, a mutation in one protein of a complex could result in accumulation of an assembly intermediate that maintains residual function or a gain of function, whereas a different assembly order could result in a complete lack of assembly and a total loss of function.

In this study, we applied multiple approaches and resources to characterize BBSome assembly intermediates. Point mutations in specific BBS proteins, null mutations completely lacking some BBS proteins, RNA interference to reduce BBS protein levels, as well as overexpression of specific BBSome subunits result in the intracellular accumulation of distinct assembly intermediates. The assembly of the BBSome is dictated by the intrinsic protein-protein interactions among BBSome proteins and is also guided as by the chaperonin-like BBS proteins BBS6, BBS10, and BBS12. Structural analysis reveals that BBS1, BBS2, BBS7, and BBS9 contain β propeller domains, BBS4 and BBS8 contain multiple tetratricopeptide repeat domains, and BBS5 has two pleckstrin homology domains. These domains typically mediate protein-protein interactions. Pairwise co-immunoprecipitation assays showed that BBS7 interacts with BBS2 and that BBS9 interacts with BBS1, BBS2, BBS5, and BBS8. BBS1 was shown to interact with BBS2 and BBS4. The BBSome assembly process reflects these protein-protein interactions. The chaperonin-like BBS proteins (BBS6, BBS10, and BBS12) are essential in initiating the first step of the BBSome formation by forming the BBS-chaperonin complex to stabilize BBS7.

BBS2, BBS7, and BBS9 Form BBSome Core ComplexOur study identified BBS2-BBS7-BBS9 as an important intermediate complex in the assembly of the mature BBSome. Formation of this BBSome core complex is coordinated with the release of the BBS-chaperonin complex from BBS7. One function of this BBS-chaperonin complex appears to be to stabilize BBS7 prior to its association with BBS2 and BBS9. The absence of BBS2, BBS7, BBS9, or any of the BBS-chaperonin complex components decreases the levels of BBS2 and BBS7, whereas the absence or mutation of BBS1 or BBS4 has a minimal effect on BBS2 and BBS7 protein stability. Free BBS2 is degraded by the ubiquitin-proteasome pathway. The ubiquitin ligase for BBS2 is currently unknown.BBS11/TRIM32 is an E3 ligase for several proteins including actin and dysbindin (33, 34). The specific mutation of P130S in one family causes BBS phenotypes (35). It is intriguing to speculate that the putative E3 ligase for BBS2 might be BBS11/TRIM32.

BBS2, BBS7, and BBS9 Form BBSome Core Complex

Our study identified BBS2-BBS7-BBS9 as an important intermediate complex in the assembly of the mature BBSome. Formation of this BBSome core complex is coordinated with the release of the BBS-chaperonin complex from BBS7. One function of this BBS-chaperonin complex appears to be to stabilize BBS7 prior to its association with BBS2 and BBS9. The absence of BBS2, BBS7, BBS9, or any of the BBS-chaperonin complex components decreases the levels of BBS2 and BBS7, whereas the absence or mutation of BBS1 or BBS4 has a minimal effect on BBS2 and BBS7 protein stability. Free BBS2 is degraded by the ubiquitin-proteasome pathway. The ubiquitin ligase for BBS2 is currently unknown.

BBS11/TRIM32 is an E3 ligase for several proteins including actin and dysbindin (33, 34). The specific mutation of P130S in one family causes BBS phenotypes (35). It is intriguing to speculate that the putative E3 ligase for BBS2 might be BBS11/TRIM32.

Regulation of BBSome FunctionThere are various possible mechanisms by which the regulation of BBSome function could occur. One possibility is that regulation occurs after the BBSome is fully assembled, and the movement of the BBSome into its functional cellular compartment (i.e., cilia) is regulated. Both positive (RAB8) and negative (LZTFL1) BBSome interactors have been identified (10, 25). However, regulation of assembly is also a likely possibility. Based on our study, regulation of assembly would most likely occur at the level of the formation of the BBS-chaperonin complex, at the level of the formation of the BBSome core complex, and/or at the level of the release BBS6, BBS12, and CCT/TRiC proteins from BBS7. BBS10 is likely a key regulatory component for the BBS-chaperonin complex.

Regulation of BBSome Function

There are various possible mechanisms by which the regulation of BBSome function could occur. One possibility is that regulation occurs after the BBSome is fully assembled, and the movement of the BBSome into its functional cellular compartment (i.e., cilia) is regulated. Both positive (RAB8) and negative (LZTFL1) BBSome interactors have been identified (10, 25). However, regulation of assembly is also a likely possibility. Based on our study, regulation of assembly would most likely occur at the level of the formation of the BBS-chaperonin complex, at the level of the formation of the BBSome core complex, and/or at the level of the release BBS6, BBS12, and CCT/TRiC proteins from BBS7. BBS10 is likely a key regulatory component for the BBS-chaperonin complex.

Physiological and Clinical Significance of BBSome Intermediates: A Role for BBSome Assembly Intermediates in BBS Phenotype VariabilityUnder normal conditions, there appears to be a limited free pool of individual BBSome subunits with the vast majority of the subunits being in BBSome complex form. However, BBSome assembly intermediates such as the BBSome core complex may form under conditions such as occur when specific null or point mutations exist in BBS patients. The phenotypes of BBS vary significantly between different families and even within members of the same family (individuals with the same specific mutation) (36–38). These observations suggest the possibility of genetic modifiers. Phenotype modification can occur when a heterozygous variant in one gene influences the phenotypic manifestation of another causative gene. Because some mutations may lead to the accumulation of stable assembly intermediate, the presence of the intermediate may explain some of the phenotypic differences observed among different BBS gene mutations. One interesting example is the hypertension phenotype that is manifested in Bbs4−/− mice, whereas Bbs2−/− mice do not have hypertension (39, 40). Furthermore, physiological differences among human BBS patients caused by mutations in distinct BBS genes have also been reported (37). The mechanism by which this might occur will require further investigation.The elucidation of the stepwise manner by which the BBSome is formed suggests the possibility of cell type-specific forms of the BBSome. In other words, the fact that some BBSome proteins (e.g., BBS1, BBS5, and BBS8) are added independently to a stable intermediate complex allows for the possibility that BBSome heterogeneity exists and that the heterogeneous forms of the BBSome may have functional roles in specific tissues.

Physiological and Clinical Significance of BBSome Intermediates: A Role for BBSome Assembly Intermediates in BBS Phenotype Variability

Under normal conditions, there appears to be a limited free pool of individual BBSome subunits with the vast majority of the subunits being in BBSome complex form. However, BBSome assembly intermediates such as the BBSome core complex may form under conditions such as occur when specific null or point mutations exist in BBS patients. The phenotypes of BBS vary significantly between different families and even within members of the same family (individuals with the same specific mutation) (36–38). These observations suggest the possibility of genetic modifiers. Phenotype modification can occur when a heterozygous variant in one gene influences the phenotypic manifestation of another causative gene. Because some mutations may lead to the accumulation of stable assembly intermediate, the presence of the intermediate may explain some of the phenotypic differences observed among different BBS gene mutations. One interesting example is the hypertension phenotype that is manifested in Bbs4−/− mice, whereas Bbs2−/− mice do not have hypertension (39, 40). Furthermore, physiological differences among human BBS patients caused by mutations in distinct BBS genes have also been reported (37). The mechanism by which this might occur will require further investigation.

The elucidation of the stepwise manner by which the BBSome is formed suggests the possibility of cell type-specific forms of the BBSome. In other words, the fact that some BBSome proteins (e.g., BBS1, BBS5, and BBS8) are added independently to a stable intermediate complex allows for the possibility that BBSome heterogeneity exists and that the heterogeneous forms of the BBSome may have functional roles in specific tissues.
