---
pmid: '40442316'
title: Structure and activation of the human autophagy-initiating ULK1C:PI3KC3-C1
  supercomplex.
authors:
- Chen M
- Nguyen TN
- Ren X
- Khuu G
- Cook ASI
- Zhao Y
- Yildiz A
- Lazarou M
- Hurley JH
journal: Nat Struct Mol Biol
year: '2025'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC12440827
doi: 10.1038/s41594-025-01557-x
---

# Structure and activation of the human autophagy-initiating ULK1C:PI3KC3-C1 supercomplex.
**Authors:** Chen M, Nguyen TN, Ren X, Khuu G, Cook ASI, Zhao Y, Yildiz A, Lazarou M, Hurley JH
**Journal:** Nat Struct Mol Biol (2025)
**DOI:** [10.1038/s41594-025-01557-x](https://doi.org/10.1038/s41594-025-01557-x)
**PMC:** [PMC12440827](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12440827/)

## Abstract

1. Nat Struct Mol Biol. 2025 Sep;32(9):1596-1605. doi:
10.1038/s41594-025-01557-x.  Epub 2025 May 29.

Structure and activation of the human autophagy-initiating ULK1C:PI3KC3-C1 
supercomplex.

Chen M(1)(2)(3), Nguyen TN(3)(4)(5)(6), Ren X(1)(2)(3), Khuu G(3)(4)(5)(6), Cook 
ASI(2)(3)(7), Zhao Y(1)(8), Yildiz A(1)(7)(8), Lazarou M(3)(4)(5)(6), Hurley 
JH(9)(10)(11)(12)(13).

Author information:
(1)Department of Molecular and Cell Biology, University of California, Berkeley, 
Berkeley, CA, USA.
(2)California Institute for Quantitative Biosciences, University of California, 
Berkeley, Berkeley, CA, USA.
(3)Aligning Science Across Parkinson's (ASAP) Collaborative Research Network, 
Chevy Chase, MD, USA.
(4)Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, 
Australia.
(5)Department of Medical Biology, University of Melbourne, Melbourne, Victoria, 
Australia.
(6)Department of Biochemistry and Molecular Biology, Biomedicine Discovery 
Institute, Monash University, Melbourne, Victoria, Australia.
(7)Graduate Group in Biophysics, University of California, Berkeley, Berkeley, 
CA, USA.
(8)Physics Department, University of California, Berkeley, Berkeley, CA, USA.
(9)Department of Molecular and Cell Biology, University of California, Berkeley, 
Berkeley, CA, USA. jimhurley@berkeley.edu.
(10)California Institute for Quantitative Biosciences, University of California, 
Berkeley, Berkeley, CA, USA. jimhurley@berkeley.edu.
(11)Aligning Science Across Parkinson's (ASAP) Collaborative Research Network, 
Chevy Chase, MD, USA. jimhurley@berkeley.edu.
(12)Graduate Group in Biophysics, University of California, Berkeley, Berkeley, 
CA, USA. jimhurley@berkeley.edu.
(13)Helen Wills Neuroscience Institute, University of California, Berkeley, 
Berkeley, CA, USA. jimhurley@berkeley.edu.

The Unc-51-like kinase protein kinase complex (ULK1C) is the most upstream and 
central player in the initiation of macroautophagy in mammals. Here, we 
determined the cryo-electron microscopy structure of the human ULK1C core at 
amino-acid-level resolution. We also determined a moderate-resolution structure 
of the ULK1C core in complex with another autophagy core complex, the class III 
phosphatidylinositol 3-OH kinase complex I (PI3KC3-C1). We show that the two 
complexes coassemble through extensive contacts between the FIP200 scaffold 
subunit of ULK1C and the VPS15, ATG14 and BECN1 subunits of PI3KC3-C1. The 
FIP200:ATG13:ULK1 core of ULK1C undergoes a rearrangement from 2:1:1 to 2:2:2 
stoichiometry in the presence of PI3KC3-C1. This suggests a structural mechanism 
for the initiation of autophagy through formation of a ULK1C:PI3KC3-C1 
supercomplex and dimerization of ULK1 on the FIP200 scaffold.

© 2025. The Author(s).

DOI: 10.1038/s41594-025-01557-x
PMCID: PMC12440827
PMID: 40442316 [Indexed for MEDLINE]

Conflict of interest statement: Competing interests: J.H.H. is a cofounder of 
Casma Therapeutics and receives research funding from Hoffmann-La Roche. M.L. is 
a cofounder and member of the scientific advisory board of Automera. The other 
authors declare no competing interests.

## Full Text

MainMacroautophagy (hereafter, ‘autophagy’) is the main cellular mechanism for the disposal of molecular aggregates and damaged or unneeded organelles1. While autophagy was first characterized as a bulk response to starvation, it is now clear that many forms of bulk and selective autophagy are central in development and cellular homeostasis1. The homeostatic necessity of autophagy is clearest in neurons, which are postmitotic and, thus, uniquely susceptible to toxic aggregates and damaged organelles2. Autophagic dysfunction has been linked to all major neurodegenerative diseases2, with the clearest genetic linkage to Parkinson’s disease3.All forms of canonical autophagy, bulk and selective, are initiated upon the recruitment and activation of the FIP200 protein4 and the class III phosphatidylinositol 3-OH kinase complex I (PI3KC3-C1)5–7. FIP200 serves as the central scaffolding subunit of the Unc-51-like kinase protein kinase complex (ULK1C), providing a platform for the other three subunits: the ULK1 kinase, ATG13 and ATG101 (refs. 8–12). PI3KC3-C1 contains one copy each of the lipid kinase VPS34, the pseudokinase VPS15 and the regulatory subunits BECN1 and ATG14 (refs. 5,13). The former three subunits are also present in PI3KC3-C2 involved in endosomal sorting and late steps in autophagy, while ATG14 is uniquely involved in autophagy initiation5–7. Atomic-resolution and near-atomic-resolution structures are known for various fragments of these complexes14–16 and structures with low-to-moderate resolution are known for human PI3KC3-C1 (refs. 17,18) and the core of human ULK1C (ref. 19).The means by which ULK1C and PI3KC3-C1 activities are switched on and off are critically important for the physiology of autophagy initiation and for therapeutic interventions targeting neurodegeneration20. Yet these mechanisms have thus far been hidden because of the limitations of the available fragmentary or low-resolution structures. While ULK1C and PI3KC3-C1 are both critical for autophagy initiation and recruited at the earliest stages, beyond the presence of ULK1 phosphorylation sites on PI3KC3-C1 subunits21–24, it has been unclear how their activities are coordinated. Here, we report the structures of the human ULK1C core at resolutions adequate for amino-acid-level interpretation. We also show that ULK1C and PI3KC3-C1 form a physical supercomplex, determine its structure and show that ULK1C can undergo a stoichiometric switch leading to the recruitment of two copies of the ULK1 kinase itself.

Macroautophagy (hereafter, ‘autophagy’) is the main cellular mechanism for the disposal of molecular aggregates and damaged or unneeded organelles1. While autophagy was first characterized as a bulk response to starvation, it is now clear that many forms of bulk and selective autophagy are central in development and cellular homeostasis1. The homeostatic necessity of autophagy is clearest in neurons, which are postmitotic and, thus, uniquely susceptible to toxic aggregates and damaged organelles2. Autophagic dysfunction has been linked to all major neurodegenerative diseases2, with the clearest genetic linkage to Parkinson’s disease3.

All forms of canonical autophagy, bulk and selective, are initiated upon the recruitment and activation of the FIP200 protein4 and the class III phosphatidylinositol 3-OH kinase complex I (PI3KC3-C1)5–7. FIP200 serves as the central scaffolding subunit of the Unc-51-like kinase protein kinase complex (ULK1C), providing a platform for the other three subunits: the ULK1 kinase, ATG13 and ATG101 (refs. 8–12). PI3KC3-C1 contains one copy each of the lipid kinase VPS34, the pseudokinase VPS15 and the regulatory subunits BECN1 and ATG14 (refs. 5,13). The former three subunits are also present in PI3KC3-C2 involved in endosomal sorting and late steps in autophagy, while ATG14 is uniquely involved in autophagy initiation5–7. Atomic-resolution and near-atomic-resolution structures are known for various fragments of these complexes14–16 and structures with low-to-moderate resolution are known for human PI3KC3-C1 (refs. 17,18) and the core of human ULK1C (ref. 19).

The means by which ULK1C and PI3KC3-C1 activities are switched on and off are critically important for the physiology of autophagy initiation and for therapeutic interventions targeting neurodegeneration20. Yet these mechanisms have thus far been hidden because of the limitations of the available fragmentary or low-resolution structures. While ULK1C and PI3KC3-C1 are both critical for autophagy initiation and recruited at the earliest stages, beyond the presence of ULK1 phosphorylation sites on PI3KC3-C1 subunits21–24, it has been unclear how their activities are coordinated. Here, we report the structures of the human ULK1C core at resolutions adequate for amino-acid-level interpretation. We also show that ULK1C and PI3KC3-C1 form a physical supercomplex, determine its structure and show that ULK1C can undergo a stoichiometric switch leading to the recruitment of two copies of the ULK1 kinase itself.

ResultsCryo-electron microscopy (cryo-EM) structure of a 2:1:1 stoichiometric ULK1C coreThe ordered core of ULK1C was purified in a form consisting of the FIP200 N-terminal domain (NTD; 1–640) and the ULK1 C-terminal microtubule-interacting and transport (MIT) domain (836–1059) fused with the ATG13 residues (363–517) responsible for binding to FIP200 and ULK1 (ref. 19) (Fig. 1a). The peripheral domains, including the coiled-coil (CC; 791–1498) and claw (1499–1594) domains of FIP200, the kinase domain of ULK1 (1–278) and the ATG13 (1–190):ATG101 (1–218) HORMA (Hop1p/Rev7p/MAD2) domain, were removed from the construct because they are flexibly connected to the ULK1C core through disordered loops and not known to participate in the ordered core assembly. Deletion of these domains did not affect the stability of the ULK1C core and facilitated its structure determination. A monodisperse peak from size-exclusion chromatography (SEC) was collected and used for cryo-EM data collection. Image processing and three-dimensional (3D) reconstruction resulted in a cryo-EM density map with a local resolution of 3.35 Å in the best regions, including the FIP200:ATG13:ULK1 interface (Extended Data Fig. 1a–g). The distal tips of the FIP200NTD molecules are mobile and the density there is less defined. The ATG13:ULK1 unit resembles the yeast Atg13MIM (MIT-interacting motif):Atg1MIT complex25; thus, we adopted the MIT/MIM terminology for the human complex. The quality of the density allowed assignment of amino acid residues of the ULK1MIT domain and ATG13MIM (Extended Data Fig. 1h–l). The structure confirmed the previous observation of a 2:1:1 FIP200:ATG13:ULK1 complex19, while defining how ULK1MIT and ATG13MIM bind to FIP200 in amino-acid-level detail.Fig. 1Structure of the ULK1C core.a, Domain organization of the ULK1C core. Gray indicates the regions truncated in this study. CTD, C-terminal domain; KD, kinase domain. b, Representative reference-free 2D class averages of the ULK1C core. Scale bar, 20 nm. c, Overview of the EM map (top) and the coordinates (bottom) of the ULK1C core. Subunits are depicted in the same color as in a. The map is contoured at 12σ. d, Pulldown assay of FIP200NTD with ULK1MIT and ATG13MIM. Strep-Tactin resin was loaded with TSF-tagged FIP200NTD to pull down MBP-tagged ULK1MIT and GST-tagged ATG13MIM in various combinations as indicated above the lanes. The experiment was repeated three times with similar results. e, Close-up view of the interface between ULK1MIT and ATG13MIM and the corresponding Strep pulldown assay. FIP200NTD–TSF was used to pull down various GST-tagged ATG13 mutants (F466D;F470D, L476D;L479D, L495D;L499D and F509D;F512D) and MBP-tagged WT ULK1MIT. The experiment was repeated four times with similar results. f–h, Close-up views of the interface between ULK1MIT and FIP200. Key residues for the binding interface are indicated, with hydrogen bonds shown as yellow dotted lines. i, Strep pulldown assay for the FIP200–ULK1 interface. Various FIP200NTD–TSF mutants (R334D, R326D and V314D) were used to pull down WT ULK1MIT and ATG13MIM. All pulldown results were visualized by SDS–PAGE and Coomassie blue staining. The experiment was repeated four times with similar results.Source dataThe ULK1C core has dimensions of 180 × 130 × 90 Å and contains two molecules of FIP200NTD and one molecule each of ULK1 and ATG13 (Fig. 1b,c). FIP200NTD forms a C-shaped dimer as seen at low resolution19, which can now be described in detail with sequence assigned. The two arms of FIP200 consist of 120-Å-long bundles formed by three twisted helixes (81–495). The two arms are connected by an 80-Å-long dimerization domain (496–599) bent at nearly 90° to the arms, resulting in the C shape. The structure after residue 599 was not resolved because of presumed disorder. The FIP200 dimer is in some ways reminiscent of the structure of its yeast counterpart, Atg17 (ref. 26), although the protein folds are distinct. Atg17 also dimerizes through two arms that are in the same plane, although the arms are arranged in an S shape instead of a C (Extended Data Fig. 2a). Residues 1–80 of FIP200 form a ubiquitin-like domain (ULD) located at the middle of the arm domain on the inner side of the C shape. The higher-resolution analysis here confirms the previous observation that the ULD and the arm domain of FIP200 have the same fold as the scaffold domain of the TANK-binding kinase 1 (TBK1)19 (Extended Data Fig. 2b), which is itself central to the initiation of some forms of autophagy27.Cryo-EM density for one copy of a ULK1MIT:ATG13MIM heterodimer was observed on one ‘shoulder’ of the FIP200 dimer. The ATG13MIM binds collaboratively to both FIP200 and ULK1MIT (Fig. 1d), consistent with the role of ATG13 in recruiting ULK1 (ref. 28) downstream of FIP200. As seen for yeast Atg13:Atg1 (ref. 25), the structure consists of two four-helix bundles (Extended Data Fig. 2c). Three helices of each are from the ULK1MIT and one from the ATG13MIM. Four pairs of hydrophobic residues (F466 + F470, L476 + L479, L495 + L499 and F509 + F512) from the ATG13MIM form prominent direct interactions with ULK1MIT (Extended Data Fig. 2d,e). Mutagenesis and pulldown assays confirmed that the latter two pairs, L495 + L499 and F509 + F512, are critical for the binding (Fig. 1e). The ULK1MIT:ATG13MIM heterodimer binds to FIP200 through two main interfaces (Fig. 1f–h). Three FIP200 residues, V314, R326 and R334, were identified by cryo-EM as potentially important for the ULK1:ATG13 interaction, which was confirmed by mutagenesis and pulldown assays (Fig. 1i).The ATG13IDR:FIP200NTD interfaceThe C-terminal portion of the ATG13 intrinsically disordered region (ATG13IDR), corresponding to residues 363–460, was previously demonstrated to bind to the FIP200NTD (ref. 19) but was not visualized with sufficient continuity to assign amino acid sequence on the basis of the density alone. In this study, we used AlphaFold2 (ref. 29), as corroborated by hydrogen–deuterium exchange mass spectrometry (HDX–MS)19, to map four potential binding sites on ATG13 (site 1, 370–387; site 2, 392–408; site 3, 409–414; site 4, 450–460) (Fig. 2a,b and Extended Data Fig. 3). The sites augment the role of the ATG13MIM (460–517), referred to as site M hereafter. The interaction of a single ATG13IDR extends through both protomers of the FIP200NTD dimer. Sites 1 and 4 are unique in ATG13 but their binding sites on two FIP200 protomers are structurally symmetric. One FIP200 molecule is occupied by ATG13 site 4 and the ULK1MIT:ATG13MIM heterodimer and the other FIP200 molecule is occupied by sites 1–3 of the ATG13IDR loop (Fig. 2b). This explains the unusual 2:1:1 stoichiometry of the ULK1C by showing in atomistic detail how the FIP200NTD dimer binds a single copy of ATG13 in the complex.Fig. 2Binding sites of ATG13IDR on FIP200NTD.a, AlphaFold2 prediction model of the ULK1C core. b, Schematic diagram of the ULK1C core. c, Truncation and mutagenesis screening conducted by pulldown assay. Strep-Tactin resin was loaded with WT FIP200NTD–TSF to pull down various ATG13 fragments (WT 363–460, 363–398, 363–392 and 363–450 with F394D;F397D;E403K substitutions (363–450 M1)) or loaded with FIP200NTD–TSF variant (L324D;R543D;R548D) to pull down WT ATG13 fragment 363–450. The experiment was repeated four times with similar results. d,e, Close-up views of binding sites 1 and 2 of the AlphaFold2 model with the key binding residues shown. The map is contoured at 11σ. f, Strep pulldown assay. The WT FIP200NTD–TSF was used to pull down various ATG13 fragments (WT 402–517, 402–450, 450–517 and 450–517 with A452D;F453D substitutions (450–517 M2)) or FIP200NTD–TSF variant (V44D) was used to pull down WT ATG13 fragment 450–517. The results from the pulldown assays (c,f) were visualized by SDS–PAGE and Coomassie blue staining. The experiment was repeated four times with similar results. g, Close-up view of binding site 4 of the AlphaFold2 model. h, Close-up view of the ULK1MIT-binding site (site M) of ATG13MIM determined by cryo-EM. The key binding residues are shown. i,j, ATG13-KO cells stably expressing Halo–LC3B without rescue or rescued with different versions of HA–ATG13 were treated with 50 nM TMR-conjugated Halo ligand for 15 min. Following that, cells were washed with 1× PBS and treated with EBSS for the indicated time periods, harvested and analyzed by immunoblotting with indicated antibodies (i) and the percentage of the cleaved Halo band was quantified (j). Data in j are the mean ± s.d. from three independent experiments. NS, not significant. ****P < 0.0001 (two-way ANOVA). The substitution sites of each variant and their corresponding residues are shown in the insert. k, ATG13-KO cells expressing Halo–LC3B without rescue or rescued with indicated versions of HA–ATG13 were left untreated or treated with EBSS for 30 min or 45 min. Cells were harvested and analyzed by immunoblotting with the indicated antibodies. l, The ratio of phosphorylated ATG14 (p-ATG14) relative to untreated rescue WT ATG13 samples was quantified. Data in l are the mean ± s.d. from three independent experiments. ****P < 0.0001 (two-way ANOVA). The definition of mutants (bottom right) applies to i–l.Source dataWe screened a series of truncations to map the FIP200-binding determinants of ATG13IDR. Two ATG13 fragments, 363–398 (site 2) and 450–460 (site 4) were independently sufficient for FIP200 binding (Fig. 2c), consistent with the previous observation that these regions are protected from HDX–MS19 (Extended Data Fig. 3b). Point mutagenesis showed that ATG13 residues F394, F397 and E403 and FIP200 residues L324, R543 and R548 are required for binding through site 2 (Fig. 2c–e), whereas ATG13 residues A452 and F453 and FIP200 V44 are key for binding through site 4 (Fig. 2f,g). We tested whether disrupting the binding between ATG13 and FIP200 would affect bulk autophagy in response to starvation. ATG13 variants (Fig. 2d,e,g,h) were introduced into ATG13-knockout (KO) HeLa cells (Extended Data Fig. 4). Mutations impacting site 2 (F394D;M396D;F397D;E403K) alone did not have a significant effect. Mutations impacting sites 4 and M (site 4, F449D;A452D;F453D;S454A; site M, L495D;L499D) led to a substantial reduction in flux, whereas mutations impacting sites 2, 4 and M together nearly eliminated flux (Fig. 2i,j). To investigate whether the reduction in autophagy is because of the decreased activity of ULK1C, we measured the phosphorylation level of ATG14S29, a substrate of ULK1 kinase that has an important role in regulating the function of PI3KC3-C1 (ref. 23). The results of the phosphorylation assay showed that reduced ULK1 activity correlates with reductions in autophagy flux. Mutations impacting site 2 alone had no significant effect, whereas combined mutations impacting sites 2, 4 and M eliminated ULK1 activity (Fig. 2k,l).ULK1C and PI3KC3-C1 form a supercomplexPI3KC3-C1 is a substrate of ULK1 (refs. 21,23). However, it has been unclear whether PI3KC3-C1 is recruited coordinately with ULK1C in autophagy initiation. To determine whether there was a direct interaction and, if so, with which portion of the complex, we assessed binding of PI3KC3-C1 to the ULK1C core and the ATG13–ATG101 HORMA dimer. Direct binding between FIP200 and PI3KC3-C1 was observed using a bead-binding assay (Fig. 3a,b). The ATG13–ATG101 HORMA dimer did not interact with PI3KC3-C1 under these conditions (Fig. 3a,b). ULK1C and PI3KC3-C1 were mixed and visualized by cryo-EM (Extended Data Fig. 5a). A small population of ULK1C core in complex with PI3KC3-C1 was observed in this dataset (Extended Data Fig. 5b). The mixture was then further purified by pulldown with PI3KC3-C1 (Fig. 3c) and subjected to cryo-EM (Fig. 4 and Extended Data Fig. 6). Two-dimensional (2D) classes of the ULK1C:PI3KC3-C1 supercomplex were visualized, in which the distinctive C-shaped density of the ULK1C core and the V-shaped density of PI3KC3-C1 were both clearly observed (Fig. 4a). Further 3D reconstruction resulted in an EM map containing the crescent FIP200 dimer and all four subunits (VPS34, VPS15, BECN1 and ATG14) of PI3KC3-C1 (Extended Data Fig. 6). The peripheral regions, including the FIP200 molecule binding with ULK1MIT:ATG13MIM and VPS34HELCAT (helical and catalytic domains), were not modeled because of their high mobility. In contrast, the regions containing VPS15HSD (helical solenoid domain), BECN1CC–ATG14CC, BECN1BH3 and the FIP200 arm not bound to ULK1MIT:ATG13MIM (hereafter, the ‘proximal arm’) were the best resolved parts, with local resolution at 6.84 Å (Extended Data Fig. 6h). This enabled unambiguous identification of the secondary structures identified in the atomistic structures of the PI3KC3-C1 core30 and the proximal arm of FIP200 (Fig. 4b and Extended Data Fig. 7a).Fig. 3Direct interaction between ULK1C and PI3KC3-C1.a, Bead-binding assay of ULK1C core and PI3KC3-C1. GSH beads coated with GST tag alone or GST-tagged PI3KC3-C1 was used to recruit various combination of GFP-tagged proteins. The schematic drawing illustrates the bait and prey samples used for each condition. b, Complementary bead-binding assay for confirming the interaction between FIP200 and PI3KC3-C1. GST-tagged various proteins were used to recruit the mCherry-tagged PI3KC3-C1. c, SDS–PAGE of the ULK1C:PI3KC3-C1 pulldown sample. Strep resin was used to pull down the TSF-tagged VPS15. The eluate was obtained by washing the resin with buffer containing 10 mM d-desthiobiotin and visualized by SDS–PAGE and Coomassie blue staining. The experiment was repeated two times with similar results.Source dataFig. 4Structure of the ULK1C:PI3KC3-C1 supercomplex.a, Representative 2D class average of the ULK1C:PI3KC3-C1 supercomplex. Scale bar, 20 nm. b, Overview of the EM map and the coordinates of ULK1C:PI3KC3-C1 supercomplex superposed with the EM map contoured at 8σ. Note: only one FIP200 molecule of the ULK1C was modeled (blue). c, Close-up view of the FIP200-binding interface of PI3KC3-C1. FIP200 is shown in surface representation and PI3KC3-C1 is shown in ribbon representation. The FIP200ULD, VPS15HSD, BECN1, CC and zinc-finger domains formed by BECN1–ATG14 are indicated. d, Close-up view of the unassigned EM densities at sites 1 and 2. The local refinement map is shown at 22σ. The assigned map is colored according to the molecule (blue, FIP200; pink, VPS15; green, BECN1; yellow, ATG14). The extra densities observed at site 1 are shown in gray and indicated by arrows.Two main interfaces were identified between FIP200 and PI3KC3-C1 (Fig. 4c). The first interface is formed between the FIP200(166–181, 473–485) and BECN1BH3 (Extended Data Fig. 7b), a well-studied Bcl-2/Bcl-XL-binding region that regulates autophagy and apoptosis31–34. A zinc-finger motif, composed of BECN1C137, C140 and ATG14C43, C46, is located behind the BH3 domain and supports the conformation. The BECN1–ATG14 zinc-finger and the BECN1 BH3 region were not visualized in previous PI3KC3-C1 structures; thus, it appears that the presence of the FIP200 proximal arm induces their ordering. The second interface is formed by the globular ULD domain of FIP200 and the curved HSD domain of VPS15 (Extended Data Fig. 7c). The VPS15HSD adjoins FIP200 sites 1 and 2, forming a large pocket with the FIP200 proximal arm and ULD. Fragmented EM density is observed in this pocket (Fig. 4d). Superposition of a predicted model of the ULK1C core onto the supercomplex structure shows that the unassigned density could be contributed by ATG13371–385; however, the resolution is insufficient for unambiguous assignment. Overall, the structure shows that PI3KC3-C1 and FIP200 interact extensively and intimately. It is striking that the BECN1BH3 and ATG14–BECN1 zinc-finger regions become ordered and wrap more than halfway around the FIP200 proximal arm and that these regions closely approach ATG13 sites 1 and 2.FIP200 scaffolding of ULK1 dimerizationIn the same ULK1C:PI3KC3-C1 mixture dataset described above, a particle class of ULK1C showed clear 2:2:2 stoichiometry (Fig. 5a and Extended Data Fig. 5c). The presence of the ULK1MIT:ATG13MIM heterodimer was confirmed on both FIP200 shoulders, demonstrating that the stoichiometry of the ULK1 complex was altered in the cryo-EM sample by the presence of PI3KC3-C1.Fig. 5ULK1 dimerization on the FIP200 scaffold.a, Representative 2D class average and the EM map of the ULK1C (2:2:2) core in the presence of PI3KC3-C1. The ULK1MIT:ATG13MIM domains are indicated with arrows. Scale bar, 20 nm. The EM map is contoured at 7σ. b, Representative 2D class average and EM map of the ULK1C (2:2:2) core of the ATG13450–517 truncation mutant. The EM map is contoured at 14σ. c, Repeated 2D class averages and EM map of the ULK1C (2:1:1) core as shown in Fig. 1b,c for comparison. The EM map is contoured at 12σ.ATG13 site 2 is not involved in the 2:2:2 complex; therefore, we hypothesized that this site might function as a brake to negatively regulate formation of this complex. To test this hypothesis, we truncated ATG13IDR at residue 449 and determined the cryo-EM structure of an ATG13450–517-containing version of the ULK1C core in the absence of PI3KC3-C1. The 2D averages clearly show densities of the ULK1MIT:ATG13MIM heterodimer on both sides of the shoulder (Fig. 5b), which was confirmed by 3D reconstruction at 4.46 Å (Extended Data Fig. 5d–h). The stoichiometry was investigated in solution by SEC with in-line multiangle light scattering (MALS) (Extended Data Fig. 8a) and mass photometry (Fig. 6). The ULK1C cores in the presence and absence of the ATG13IDR region displayed masses consistent with 2:1:1 and 2:2:2 complexes, respectively, corroborating the cryo-EM data. Furthermore, the ATG13 F394D;F397D;E403K mutant, designed to disrupt site 2, also formed a complex with 2:2:2 stoichiometry (Extended Data Fig. 8b). This observation demonstrates that site 2 of ATG13IDR opposes the recruitment of the second ULK1 kinase to the FIP200 dimer.Fig. 6Concentration-dependent stoichiometry shift of ULK1C.Mass photometry measurements of (i) FIP200NTD, (ii) ULK1MIT:ATG13 (363–517), (iii) full-length PI3KC1-C1, (iv) FIP200NTD with truncated ULK1MIT:ATG13 (450–517; note: the molecular weight shifted because of the truncation), (v,vi) FIP200NTD with ULK1MIT:ATG13 (363–517) crosslinked at 10 nM in the absence or presence of 5 nM PI3KC3-C1, (vii,viii) FIP200NTD with ULK1MIT:ATG13 (363–517) crosslinked at 200 nM in the absence or presence of 100 nM PI3KC3-C1 and (ix,x) FIP200NTD with ULK1MIT:ATG13 (363–517) at crosslinked 1,000 nM in the absence or presence of 500 nM PI3KC3-C1. Solid curves represent fits to multiple Gaussian to estimate the average mass and the percentage of each population (Extended Data Table 1). The percentage was calculated from the area of the peaks in the range of 0–500 kDa. The schematic drawing illustrates the setting of each measurement (left) and the predicted assemblies (bottom). Left, the theoretical molecular weights of each assembly are shown next to the diagrams (i–iv). Right, the crosslinking concentrations of FIP200 and ULK1:ATG13 used for each measurement are indicated with red labels (v–x).Source dataWe sought to determine whether PI3KC3-C1 also promoted the formation of the 2:2:2 complex in solution. Using mass photometry, we observed that the ULK1C core alone at 10 nM concentration formed a 2:1:1 complex (Fig. 6(v)), consistent with the SEC–MALS results. The presence of PI3KC3-C1 did not alter the stoichiometry in solution (Fig. 6(vi)). We hypothesized that PI3KC3-C1 might facilitate the membrane recruitment and increased local concentration of ULK1C in autophagy initiation, which might manifest in a cryo-EM experiment as recruitment and concentration of ULK1 at the air–water interface on EM grids. We found that, upon increasing the concentration from 200 nM to 1 μM, abundant 2:2:2 complex appeared in solution, roughly coequal to the 2:1:1 complex (Fig. 6(ix)). Addition of PI3KC3-C1 did not lead to a further enhancement of the 2:2:2-to-2:1:1 ratio in solution (Fig. 6(x)). This observation shows that mass action alone can drive the 2:1:1-to-2:2:2 state conversion, even in the absence of PI3KC3-C1 and for constructs containing ATG13 site 2.

Cryo-electron microscopy (cryo-EM) structure of a 2:1:1 stoichiometric ULK1C coreThe ordered core of ULK1C was purified in a form consisting of the FIP200 N-terminal domain (NTD; 1–640) and the ULK1 C-terminal microtubule-interacting and transport (MIT) domain (836–1059) fused with the ATG13 residues (363–517) responsible for binding to FIP200 and ULK1 (ref. 19) (Fig. 1a). The peripheral domains, including the coiled-coil (CC; 791–1498) and claw (1499–1594) domains of FIP200, the kinase domain of ULK1 (1–278) and the ATG13 (1–190):ATG101 (1–218) HORMA (Hop1p/Rev7p/MAD2) domain, were removed from the construct because they are flexibly connected to the ULK1C core through disordered loops and not known to participate in the ordered core assembly. Deletion of these domains did not affect the stability of the ULK1C core and facilitated its structure determination. A monodisperse peak from size-exclusion chromatography (SEC) was collected and used for cryo-EM data collection. Image processing and three-dimensional (3D) reconstruction resulted in a cryo-EM density map with a local resolution of 3.35 Å in the best regions, including the FIP200:ATG13:ULK1 interface (Extended Data Fig. 1a–g). The distal tips of the FIP200NTD molecules are mobile and the density there is less defined. The ATG13:ULK1 unit resembles the yeast Atg13MIM (MIT-interacting motif):Atg1MIT complex25; thus, we adopted the MIT/MIM terminology for the human complex. The quality of the density allowed assignment of amino acid residues of the ULK1MIT domain and ATG13MIM (Extended Data Fig. 1h–l). The structure confirmed the previous observation of a 2:1:1 FIP200:ATG13:ULK1 complex19, while defining how ULK1MIT and ATG13MIM bind to FIP200 in amino-acid-level detail.Fig. 1Structure of the ULK1C core.a, Domain organization of the ULK1C core. Gray indicates the regions truncated in this study. CTD, C-terminal domain; KD, kinase domain. b, Representative reference-free 2D class averages of the ULK1C core. Scale bar, 20 nm. c, Overview of the EM map (top) and the coordinates (bottom) of the ULK1C core. Subunits are depicted in the same color as in a. The map is contoured at 12σ. d, Pulldown assay of FIP200NTD with ULK1MIT and ATG13MIM. Strep-Tactin resin was loaded with TSF-tagged FIP200NTD to pull down MBP-tagged ULK1MIT and GST-tagged ATG13MIM in various combinations as indicated above the lanes. The experiment was repeated three times with similar results. e, Close-up view of the interface between ULK1MIT and ATG13MIM and the corresponding Strep pulldown assay. FIP200NTD–TSF was used to pull down various GST-tagged ATG13 mutants (F466D;F470D, L476D;L479D, L495D;L499D and F509D;F512D) and MBP-tagged WT ULK1MIT. The experiment was repeated four times with similar results. f–h, Close-up views of the interface between ULK1MIT and FIP200. Key residues for the binding interface are indicated, with hydrogen bonds shown as yellow dotted lines. i, Strep pulldown assay for the FIP200–ULK1 interface. Various FIP200NTD–TSF mutants (R334D, R326D and V314D) were used to pull down WT ULK1MIT and ATG13MIM. All pulldown results were visualized by SDS–PAGE and Coomassie blue staining. The experiment was repeated four times with similar results.Source dataThe ULK1C core has dimensions of 180 × 130 × 90 Å and contains two molecules of FIP200NTD and one molecule each of ULK1 and ATG13 (Fig. 1b,c). FIP200NTD forms a C-shaped dimer as seen at low resolution19, which can now be described in detail with sequence assigned. The two arms of FIP200 consist of 120-Å-long bundles formed by three twisted helixes (81–495). The two arms are connected by an 80-Å-long dimerization domain (496–599) bent at nearly 90° to the arms, resulting in the C shape. The structure after residue 599 was not resolved because of presumed disorder. The FIP200 dimer is in some ways reminiscent of the structure of its yeast counterpart, Atg17 (ref. 26), although the protein folds are distinct. Atg17 also dimerizes through two arms that are in the same plane, although the arms are arranged in an S shape instead of a C (Extended Data Fig. 2a). Residues 1–80 of FIP200 form a ubiquitin-like domain (ULD) located at the middle of the arm domain on the inner side of the C shape. The higher-resolution analysis here confirms the previous observation that the ULD and the arm domain of FIP200 have the same fold as the scaffold domain of the TANK-binding kinase 1 (TBK1)19 (Extended Data Fig. 2b), which is itself central to the initiation of some forms of autophagy27.Cryo-EM density for one copy of a ULK1MIT:ATG13MIM heterodimer was observed on one ‘shoulder’ of the FIP200 dimer. The ATG13MIM binds collaboratively to both FIP200 and ULK1MIT (Fig. 1d), consistent with the role of ATG13 in recruiting ULK1 (ref. 28) downstream of FIP200. As seen for yeast Atg13:Atg1 (ref. 25), the structure consists of two four-helix bundles (Extended Data Fig. 2c). Three helices of each are from the ULK1MIT and one from the ATG13MIM. Four pairs of hydrophobic residues (F466 + F470, L476 + L479, L495 + L499 and F509 + F512) from the ATG13MIM form prominent direct interactions with ULK1MIT (Extended Data Fig. 2d,e). Mutagenesis and pulldown assays confirmed that the latter two pairs, L495 + L499 and F509 + F512, are critical for the binding (Fig. 1e). The ULK1MIT:ATG13MIM heterodimer binds to FIP200 through two main interfaces (Fig. 1f–h). Three FIP200 residues, V314, R326 and R334, were identified by cryo-EM as potentially important for the ULK1:ATG13 interaction, which was confirmed by mutagenesis and pulldown assays (Fig. 1i).

Cryo-electron microscopy (cryo-EM) structure of a 2:1:1 stoichiometric ULK1C core

The ordered core of ULK1C was purified in a form consisting of the FIP200 N-terminal domain (NTD; 1–640) and the ULK1 C-terminal microtubule-interacting and transport (MIT) domain (836–1059) fused with the ATG13 residues (363–517) responsible for binding to FIP200 and ULK1 (ref. 19) (Fig. 1a). The peripheral domains, including the coiled-coil (CC; 791–1498) and claw (1499–1594) domains of FIP200, the kinase domain of ULK1 (1–278) and the ATG13 (1–190):ATG101 (1–218) HORMA (Hop1p/Rev7p/MAD2) domain, were removed from the construct because they are flexibly connected to the ULK1C core through disordered loops and not known to participate in the ordered core assembly. Deletion of these domains did not affect the stability of the ULK1C core and facilitated its structure determination. A monodisperse peak from size-exclusion chromatography (SEC) was collected and used for cryo-EM data collection. Image processing and three-dimensional (3D) reconstruction resulted in a cryo-EM density map with a local resolution of 3.35 Å in the best regions, including the FIP200:ATG13:ULK1 interface (Extended Data Fig. 1a–g). The distal tips of the FIP200NTD molecules are mobile and the density there is less defined. The ATG13:ULK1 unit resembles the yeast Atg13MIM (MIT-interacting motif):Atg1MIT complex25; thus, we adopted the MIT/MIM terminology for the human complex. The quality of the density allowed assignment of amino acid residues of the ULK1MIT domain and ATG13MIM (Extended Data Fig. 1h–l). The structure confirmed the previous observation of a 2:1:1 FIP200:ATG13:ULK1 complex19, while defining how ULK1MIT and ATG13MIM bind to FIP200 in amino-acid-level detail.Fig. 1Structure of the ULK1C core.a, Domain organization of the ULK1C core. Gray indicates the regions truncated in this study. CTD, C-terminal domain; KD, kinase domain. b, Representative reference-free 2D class averages of the ULK1C core. Scale bar, 20 nm. c, Overview of the EM map (top) and the coordinates (bottom) of the ULK1C core. Subunits are depicted in the same color as in a. The map is contoured at 12σ. d, Pulldown assay of FIP200NTD with ULK1MIT and ATG13MIM. Strep-Tactin resin was loaded with TSF-tagged FIP200NTD to pull down MBP-tagged ULK1MIT and GST-tagged ATG13MIM in various combinations as indicated above the lanes. The experiment was repeated three times with similar results. e, Close-up view of the interface between ULK1MIT and ATG13MIM and the corresponding Strep pulldown assay. FIP200NTD–TSF was used to pull down various GST-tagged ATG13 mutants (F466D;F470D, L476D;L479D, L495D;L499D and F509D;F512D) and MBP-tagged WT ULK1MIT. The experiment was repeated four times with similar results. f–h, Close-up views of the interface between ULK1MIT and FIP200. Key residues for the binding interface are indicated, with hydrogen bonds shown as yellow dotted lines. i, Strep pulldown assay for the FIP200–ULK1 interface. Various FIP200NTD–TSF mutants (R334D, R326D and V314D) were used to pull down WT ULK1MIT and ATG13MIM. All pulldown results were visualized by SDS–PAGE and Coomassie blue staining. The experiment was repeated four times with similar results.Source data

Structure of the ULK1C core.

a, Domain organization of the ULK1C core. Gray indicates the regions truncated in this study. CTD, C-terminal domain; KD, kinase domain. b, Representative reference-free 2D class averages of the ULK1C core. Scale bar, 20 nm. c, Overview of the EM map (top) and the coordinates (bottom) of the ULK1C core. Subunits are depicted in the same color as in a. The map is contoured at 12σ. d, Pulldown assay of FIP200NTD with ULK1MIT and ATG13MIM. Strep-Tactin resin was loaded with TSF-tagged FIP200NTD to pull down MBP-tagged ULK1MIT and GST-tagged ATG13MIM in various combinations as indicated above the lanes. The experiment was repeated three times with similar results. e, Close-up view of the interface between ULK1MIT and ATG13MIM and the corresponding Strep pulldown assay. FIP200NTD–TSF was used to pull down various GST-tagged ATG13 mutants (F466D;F470D, L476D;L479D, L495D;L499D and F509D;F512D) and MBP-tagged WT ULK1MIT. The experiment was repeated four times with similar results. f–h, Close-up views of the interface between ULK1MIT and FIP200. Key residues for the binding interface are indicated, with hydrogen bonds shown as yellow dotted lines. i, Strep pulldown assay for the FIP200–ULK1 interface. Various FIP200NTD–TSF mutants (R334D, R326D and V314D) were used to pull down WT ULK1MIT and ATG13MIM. All pulldown results were visualized by SDS–PAGE and Coomassie blue staining. The experiment was repeated four times with similar results.

Source data

The ULK1C core has dimensions of 180 × 130 × 90 Å and contains two molecules of FIP200NTD and one molecule each of ULK1 and ATG13 (Fig. 1b,c). FIP200NTD forms a C-shaped dimer as seen at low resolution19, which can now be described in detail with sequence assigned. The two arms of FIP200 consist of 120-Å-long bundles formed by three twisted helixes (81–495). The two arms are connected by an 80-Å-long dimerization domain (496–599) bent at nearly 90° to the arms, resulting in the C shape. The structure after residue 599 was not resolved because of presumed disorder. The FIP200 dimer is in some ways reminiscent of the structure of its yeast counterpart, Atg17 (ref. 26), although the protein folds are distinct. Atg17 also dimerizes through two arms that are in the same plane, although the arms are arranged in an S shape instead of a C (Extended Data Fig. 2a). Residues 1–80 of FIP200 form a ubiquitin-like domain (ULD) located at the middle of the arm domain on the inner side of the C shape. The higher-resolution analysis here confirms the previous observation that the ULD and the arm domain of FIP200 have the same fold as the scaffold domain of the TANK-binding kinase 1 (TBK1)19 (Extended Data Fig. 2b), which is itself central to the initiation of some forms of autophagy27.

Cryo-EM density for one copy of a ULK1MIT:ATG13MIM heterodimer was observed on one ‘shoulder’ of the FIP200 dimer. The ATG13MIM binds collaboratively to both FIP200 and ULK1MIT (Fig. 1d), consistent with the role of ATG13 in recruiting ULK1 (ref. 28) downstream of FIP200. As seen for yeast Atg13:Atg1 (ref. 25), the structure consists of two four-helix bundles (Extended Data Fig. 2c). Three helices of each are from the ULK1MIT and one from the ATG13MIM. Four pairs of hydrophobic residues (F466 + F470, L476 + L479, L495 + L499 and F509 + F512) from the ATG13MIM form prominent direct interactions with ULK1MIT (Extended Data Fig. 2d,e). Mutagenesis and pulldown assays confirmed that the latter two pairs, L495 + L499 and F509 + F512, are critical for the binding (Fig. 1e). The ULK1MIT:ATG13MIM heterodimer binds to FIP200 through two main interfaces (Fig. 1f–h). Three FIP200 residues, V314, R326 and R334, were identified by cryo-EM as potentially important for the ULK1:ATG13 interaction, which was confirmed by mutagenesis and pulldown assays (Fig. 1i).

The ATG13IDR:FIP200NTD interfaceThe C-terminal portion of the ATG13 intrinsically disordered region (ATG13IDR), corresponding to residues 363–460, was previously demonstrated to bind to the FIP200NTD (ref. 19) but was not visualized with sufficient continuity to assign amino acid sequence on the basis of the density alone. In this study, we used AlphaFold2 (ref. 29), as corroborated by hydrogen–deuterium exchange mass spectrometry (HDX–MS)19, to map four potential binding sites on ATG13 (site 1, 370–387; site 2, 392–408; site 3, 409–414; site 4, 450–460) (Fig. 2a,b and Extended Data Fig. 3). The sites augment the role of the ATG13MIM (460–517), referred to as site M hereafter. The interaction of a single ATG13IDR extends through both protomers of the FIP200NTD dimer. Sites 1 and 4 are unique in ATG13 but their binding sites on two FIP200 protomers are structurally symmetric. One FIP200 molecule is occupied by ATG13 site 4 and the ULK1MIT:ATG13MIM heterodimer and the other FIP200 molecule is occupied by sites 1–3 of the ATG13IDR loop (Fig. 2b). This explains the unusual 2:1:1 stoichiometry of the ULK1C by showing in atomistic detail how the FIP200NTD dimer binds a single copy of ATG13 in the complex.Fig. 2Binding sites of ATG13IDR on FIP200NTD.a, AlphaFold2 prediction model of the ULK1C core. b, Schematic diagram of the ULK1C core. c, Truncation and mutagenesis screening conducted by pulldown assay. Strep-Tactin resin was loaded with WT FIP200NTD–TSF to pull down various ATG13 fragments (WT 363–460, 363–398, 363–392 and 363–450 with F394D;F397D;E403K substitutions (363–450 M1)) or loaded with FIP200NTD–TSF variant (L324D;R543D;R548D) to pull down WT ATG13 fragment 363–450. The experiment was repeated four times with similar results. d,e, Close-up views of binding sites 1 and 2 of the AlphaFold2 model with the key binding residues shown. The map is contoured at 11σ. f, Strep pulldown assay. The WT FIP200NTD–TSF was used to pull down various ATG13 fragments (WT 402–517, 402–450, 450–517 and 450–517 with A452D;F453D substitutions (450–517 M2)) or FIP200NTD–TSF variant (V44D) was used to pull down WT ATG13 fragment 450–517. The results from the pulldown assays (c,f) were visualized by SDS–PAGE and Coomassie blue staining. The experiment was repeated four times with similar results. g, Close-up view of binding site 4 of the AlphaFold2 model. h, Close-up view of the ULK1MIT-binding site (site M) of ATG13MIM determined by cryo-EM. The key binding residues are shown. i,j, ATG13-KO cells stably expressing Halo–LC3B without rescue or rescued with different versions of HA–ATG13 were treated with 50 nM TMR-conjugated Halo ligand for 15 min. Following that, cells were washed with 1× PBS and treated with EBSS for the indicated time periods, harvested and analyzed by immunoblotting with indicated antibodies (i) and the percentage of the cleaved Halo band was quantified (j). Data in j are the mean ± s.d. from three independent experiments. NS, not significant. ****P < 0.0001 (two-way ANOVA). The substitution sites of each variant and their corresponding residues are shown in the insert. k, ATG13-KO cells expressing Halo–LC3B without rescue or rescued with indicated versions of HA–ATG13 were left untreated or treated with EBSS for 30 min or 45 min. Cells were harvested and analyzed by immunoblotting with the indicated antibodies. l, The ratio of phosphorylated ATG14 (p-ATG14) relative to untreated rescue WT ATG13 samples was quantified. Data in l are the mean ± s.d. from three independent experiments. ****P < 0.0001 (two-way ANOVA). The definition of mutants (bottom right) applies to i–l.Source dataWe screened a series of truncations to map the FIP200-binding determinants of ATG13IDR. Two ATG13 fragments, 363–398 (site 2) and 450–460 (site 4) were independently sufficient for FIP200 binding (Fig. 2c), consistent with the previous observation that these regions are protected from HDX–MS19 (Extended Data Fig. 3b). Point mutagenesis showed that ATG13 residues F394, F397 and E403 and FIP200 residues L324, R543 and R548 are required for binding through site 2 (Fig. 2c–e), whereas ATG13 residues A452 and F453 and FIP200 V44 are key for binding through site 4 (Fig. 2f,g). We tested whether disrupting the binding between ATG13 and FIP200 would affect bulk autophagy in response to starvation. ATG13 variants (Fig. 2d,e,g,h) were introduced into ATG13-knockout (KO) HeLa cells (Extended Data Fig. 4). Mutations impacting site 2 (F394D;M396D;F397D;E403K) alone did not have a significant effect. Mutations impacting sites 4 and M (site 4, F449D;A452D;F453D;S454A; site M, L495D;L499D) led to a substantial reduction in flux, whereas mutations impacting sites 2, 4 and M together nearly eliminated flux (Fig. 2i,j). To investigate whether the reduction in autophagy is because of the decreased activity of ULK1C, we measured the phosphorylation level of ATG14S29, a substrate of ULK1 kinase that has an important role in regulating the function of PI3KC3-C1 (ref. 23). The results of the phosphorylation assay showed that reduced ULK1 activity correlates with reductions in autophagy flux. Mutations impacting site 2 alone had no significant effect, whereas combined mutations impacting sites 2, 4 and M eliminated ULK1 activity (Fig. 2k,l).

The ATG13IDR:FIP200NTD interface

The C-terminal portion of the ATG13 intrinsically disordered region (ATG13IDR), corresponding to residues 363–460, was previously demonstrated to bind to the FIP200NTD (ref. 19) but was not visualized with sufficient continuity to assign amino acid sequence on the basis of the density alone. In this study, we used AlphaFold2 (ref. 29), as corroborated by hydrogen–deuterium exchange mass spectrometry (HDX–MS)19, to map four potential binding sites on ATG13 (site 1, 370–387; site 2, 392–408; site 3, 409–414; site 4, 450–460) (Fig. 2a,b and Extended Data Fig. 3). The sites augment the role of the ATG13MIM (460–517), referred to as site M hereafter. The interaction of a single ATG13IDR extends through both protomers of the FIP200NTD dimer. Sites 1 and 4 are unique in ATG13 but their binding sites on two FIP200 protomers are structurally symmetric. One FIP200 molecule is occupied by ATG13 site 4 and the ULK1MIT:ATG13MIM heterodimer and the other FIP200 molecule is occupied by sites 1–3 of the ATG13IDR loop (Fig. 2b). This explains the unusual 2:1:1 stoichiometry of the ULK1C by showing in atomistic detail how the FIP200NTD dimer binds a single copy of ATG13 in the complex.Fig. 2Binding sites of ATG13IDR on FIP200NTD.a, AlphaFold2 prediction model of the ULK1C core. b, Schematic diagram of the ULK1C core. c, Truncation and mutagenesis screening conducted by pulldown assay. Strep-Tactin resin was loaded with WT FIP200NTD–TSF to pull down various ATG13 fragments (WT 363–460, 363–398, 363–392 and 363–450 with F394D;F397D;E403K substitutions (363–450 M1)) or loaded with FIP200NTD–TSF variant (L324D;R543D;R548D) to pull down WT ATG13 fragment 363–450. The experiment was repeated four times with similar results. d,e, Close-up views of binding sites 1 and 2 of the AlphaFold2 model with the key binding residues shown. The map is contoured at 11σ. f, Strep pulldown assay. The WT FIP200NTD–TSF was used to pull down various ATG13 fragments (WT 402–517, 402–450, 450–517 and 450–517 with A452D;F453D substitutions (450–517 M2)) or FIP200NTD–TSF variant (V44D) was used to pull down WT ATG13 fragment 450–517. The results from the pulldown assays (c,f) were visualized by SDS–PAGE and Coomassie blue staining. The experiment was repeated four times with similar results. g, Close-up view of binding site 4 of the AlphaFold2 model. h, Close-up view of the ULK1MIT-binding site (site M) of ATG13MIM determined by cryo-EM. The key binding residues are shown. i,j, ATG13-KO cells stably expressing Halo–LC3B without rescue or rescued with different versions of HA–ATG13 were treated with 50 nM TMR-conjugated Halo ligand for 15 min. Following that, cells were washed with 1× PBS and treated with EBSS for the indicated time periods, harvested and analyzed by immunoblotting with indicated antibodies (i) and the percentage of the cleaved Halo band was quantified (j). Data in j are the mean ± s.d. from three independent experiments. NS, not significant. ****P < 0.0001 (two-way ANOVA). The substitution sites of each variant and their corresponding residues are shown in the insert. k, ATG13-KO cells expressing Halo–LC3B without rescue or rescued with indicated versions of HA–ATG13 were left untreated or treated with EBSS for 30 min or 45 min. Cells were harvested and analyzed by immunoblotting with the indicated antibodies. l, The ratio of phosphorylated ATG14 (p-ATG14) relative to untreated rescue WT ATG13 samples was quantified. Data in l are the mean ± s.d. from three independent experiments. ****P < 0.0001 (two-way ANOVA). The definition of mutants (bottom right) applies to i–l.Source data

Binding sites of ATG13IDR on FIP200NTD.

a, AlphaFold2 prediction model of the ULK1C core. b, Schematic diagram of the ULK1C core. c, Truncation and mutagenesis screening conducted by pulldown assay. Strep-Tactin resin was loaded with WT FIP200NTD–TSF to pull down various ATG13 fragments (WT 363–460, 363–398, 363–392 and 363–450 with F394D;F397D;E403K substitutions (363–450 M1)) or loaded with FIP200NTD–TSF variant (L324D;R543D;R548D) to pull down WT ATG13 fragment 363–450. The experiment was repeated four times with similar results. d,e, Close-up views of binding sites 1 and 2 of the AlphaFold2 model with the key binding residues shown. The map is contoured at 11σ. f, Strep pulldown assay. The WT FIP200NTD–TSF was used to pull down various ATG13 fragments (WT 402–517, 402–450, 450–517 and 450–517 with A452D;F453D substitutions (450–517 M2)) or FIP200NTD–TSF variant (V44D) was used to pull down WT ATG13 fragment 450–517. The results from the pulldown assays (c,f) were visualized by SDS–PAGE and Coomassie blue staining. The experiment was repeated four times with similar results. g, Close-up view of binding site 4 of the AlphaFold2 model. h, Close-up view of the ULK1MIT-binding site (site M) of ATG13MIM determined by cryo-EM. The key binding residues are shown. i,j, ATG13-KO cells stably expressing Halo–LC3B without rescue or rescued with different versions of HA–ATG13 were treated with 50 nM TMR-conjugated Halo ligand for 15 min. Following that, cells were washed with 1× PBS and treated with EBSS for the indicated time periods, harvested and analyzed by immunoblotting with indicated antibodies (i) and the percentage of the cleaved Halo band was quantified (j). Data in j are the mean ± s.d. from three independent experiments. NS, not significant. ****P < 0.0001 (two-way ANOVA). The substitution sites of each variant and their corresponding residues are shown in the insert. k, ATG13-KO cells expressing Halo–LC3B without rescue or rescued with indicated versions of HA–ATG13 were left untreated or treated with EBSS for 30 min or 45 min. Cells were harvested and analyzed by immunoblotting with the indicated antibodies. l, The ratio of phosphorylated ATG14 (p-ATG14) relative to untreated rescue WT ATG13 samples was quantified. Data in l are the mean ± s.d. from three independent experiments. ****P < 0.0001 (two-way ANOVA). The definition of mutants (bottom right) applies to i–l.

Source data

We screened a series of truncations to map the FIP200-binding determinants of ATG13IDR. Two ATG13 fragments, 363–398 (site 2) and 450–460 (site 4) were independently sufficient for FIP200 binding (Fig. 2c), consistent with the previous observation that these regions are protected from HDX–MS19 (Extended Data Fig. 3b). Point mutagenesis showed that ATG13 residues F394, F397 and E403 and FIP200 residues L324, R543 and R548 are required for binding through site 2 (Fig. 2c–e), whereas ATG13 residues A452 and F453 and FIP200 V44 are key for binding through site 4 (Fig. 2f,g). We tested whether disrupting the binding between ATG13 and FIP200 would affect bulk autophagy in response to starvation. ATG13 variants (Fig. 2d,e,g,h) were introduced into ATG13-knockout (KO) HeLa cells (Extended Data Fig. 4). Mutations impacting site 2 (F394D;M396D;F397D;E403K) alone did not have a significant effect. Mutations impacting sites 4 and M (site 4, F449D;A452D;F453D;S454A; site M, L495D;L499D) led to a substantial reduction in flux, whereas mutations impacting sites 2, 4 and M together nearly eliminated flux (Fig. 2i,j). To investigate whether the reduction in autophagy is because of the decreased activity of ULK1C, we measured the phosphorylation level of ATG14S29, a substrate of ULK1 kinase that has an important role in regulating the function of PI3KC3-C1 (ref. 23). The results of the phosphorylation assay showed that reduced ULK1 activity correlates with reductions in autophagy flux. Mutations impacting site 2 alone had no significant effect, whereas combined mutations impacting sites 2, 4 and M eliminated ULK1 activity (Fig. 2k,l).

ULK1C and PI3KC3-C1 form a supercomplexPI3KC3-C1 is a substrate of ULK1 (refs. 21,23). However, it has been unclear whether PI3KC3-C1 is recruited coordinately with ULK1C in autophagy initiation. To determine whether there was a direct interaction and, if so, with which portion of the complex, we assessed binding of PI3KC3-C1 to the ULK1C core and the ATG13–ATG101 HORMA dimer. Direct binding between FIP200 and PI3KC3-C1 was observed using a bead-binding assay (Fig. 3a,b). The ATG13–ATG101 HORMA dimer did not interact with PI3KC3-C1 under these conditions (Fig. 3a,b). ULK1C and PI3KC3-C1 were mixed and visualized by cryo-EM (Extended Data Fig. 5a). A small population of ULK1C core in complex with PI3KC3-C1 was observed in this dataset (Extended Data Fig. 5b). The mixture was then further purified by pulldown with PI3KC3-C1 (Fig. 3c) and subjected to cryo-EM (Fig. 4 and Extended Data Fig. 6). Two-dimensional (2D) classes of the ULK1C:PI3KC3-C1 supercomplex were visualized, in which the distinctive C-shaped density of the ULK1C core and the V-shaped density of PI3KC3-C1 were both clearly observed (Fig. 4a). Further 3D reconstruction resulted in an EM map containing the crescent FIP200 dimer and all four subunits (VPS34, VPS15, BECN1 and ATG14) of PI3KC3-C1 (Extended Data Fig. 6). The peripheral regions, including the FIP200 molecule binding with ULK1MIT:ATG13MIM and VPS34HELCAT (helical and catalytic domains), were not modeled because of their high mobility. In contrast, the regions containing VPS15HSD (helical solenoid domain), BECN1CC–ATG14CC, BECN1BH3 and the FIP200 arm not bound to ULK1MIT:ATG13MIM (hereafter, the ‘proximal arm’) were the best resolved parts, with local resolution at 6.84 Å (Extended Data Fig. 6h). This enabled unambiguous identification of the secondary structures identified in the atomistic structures of the PI3KC3-C1 core30 and the proximal arm of FIP200 (Fig. 4b and Extended Data Fig. 7a).Fig. 3Direct interaction between ULK1C and PI3KC3-C1.a, Bead-binding assay of ULK1C core and PI3KC3-C1. GSH beads coated with GST tag alone or GST-tagged PI3KC3-C1 was used to recruit various combination of GFP-tagged proteins. The schematic drawing illustrates the bait and prey samples used for each condition. b, Complementary bead-binding assay for confirming the interaction between FIP200 and PI3KC3-C1. GST-tagged various proteins were used to recruit the mCherry-tagged PI3KC3-C1. c, SDS–PAGE of the ULK1C:PI3KC3-C1 pulldown sample. Strep resin was used to pull down the TSF-tagged VPS15. The eluate was obtained by washing the resin with buffer containing 10 mM d-desthiobiotin and visualized by SDS–PAGE and Coomassie blue staining. The experiment was repeated two times with similar results.Source dataFig. 4Structure of the ULK1C:PI3KC3-C1 supercomplex.a, Representative 2D class average of the ULK1C:PI3KC3-C1 supercomplex. Scale bar, 20 nm. b, Overview of the EM map and the coordinates of ULK1C:PI3KC3-C1 supercomplex superposed with the EM map contoured at 8σ. Note: only one FIP200 molecule of the ULK1C was modeled (blue). c, Close-up view of the FIP200-binding interface of PI3KC3-C1. FIP200 is shown in surface representation and PI3KC3-C1 is shown in ribbon representation. The FIP200ULD, VPS15HSD, BECN1, CC and zinc-finger domains formed by BECN1–ATG14 are indicated. d, Close-up view of the unassigned EM densities at sites 1 and 2. The local refinement map is shown at 22σ. The assigned map is colored according to the molecule (blue, FIP200; pink, VPS15; green, BECN1; yellow, ATG14). The extra densities observed at site 1 are shown in gray and indicated by arrows.Two main interfaces were identified between FIP200 and PI3KC3-C1 (Fig. 4c). The first interface is formed between the FIP200(166–181, 473–485) and BECN1BH3 (Extended Data Fig. 7b), a well-studied Bcl-2/Bcl-XL-binding region that regulates autophagy and apoptosis31–34. A zinc-finger motif, composed of BECN1C137, C140 and ATG14C43, C46, is located behind the BH3 domain and supports the conformation. The BECN1–ATG14 zinc-finger and the BECN1 BH3 region were not visualized in previous PI3KC3-C1 structures; thus, it appears that the presence of the FIP200 proximal arm induces their ordering. The second interface is formed by the globular ULD domain of FIP200 and the curved HSD domain of VPS15 (Extended Data Fig. 7c). The VPS15HSD adjoins FIP200 sites 1 and 2, forming a large pocket with the FIP200 proximal arm and ULD. Fragmented EM density is observed in this pocket (Fig. 4d). Superposition of a predicted model of the ULK1C core onto the supercomplex structure shows that the unassigned density could be contributed by ATG13371–385; however, the resolution is insufficient for unambiguous assignment. Overall, the structure shows that PI3KC3-C1 and FIP200 interact extensively and intimately. It is striking that the BECN1BH3 and ATG14–BECN1 zinc-finger regions become ordered and wrap more than halfway around the FIP200 proximal arm and that these regions closely approach ATG13 sites 1 and 2.

ULK1C and PI3KC3-C1 form a supercomplex

PI3KC3-C1 is a substrate of ULK1 (refs. 21,23). However, it has been unclear whether PI3KC3-C1 is recruited coordinately with ULK1C in autophagy initiation. To determine whether there was a direct interaction and, if so, with which portion of the complex, we assessed binding of PI3KC3-C1 to the ULK1C core and the ATG13–ATG101 HORMA dimer. Direct binding between FIP200 and PI3KC3-C1 was observed using a bead-binding assay (Fig. 3a,b). The ATG13–ATG101 HORMA dimer did not interact with PI3KC3-C1 under these conditions (Fig. 3a,b). ULK1C and PI3KC3-C1 were mixed and visualized by cryo-EM (Extended Data Fig. 5a). A small population of ULK1C core in complex with PI3KC3-C1 was observed in this dataset (Extended Data Fig. 5b). The mixture was then further purified by pulldown with PI3KC3-C1 (Fig. 3c) and subjected to cryo-EM (Fig. 4 and Extended Data Fig. 6). Two-dimensional (2D) classes of the ULK1C:PI3KC3-C1 supercomplex were visualized, in which the distinctive C-shaped density of the ULK1C core and the V-shaped density of PI3KC3-C1 were both clearly observed (Fig. 4a). Further 3D reconstruction resulted in an EM map containing the crescent FIP200 dimer and all four subunits (VPS34, VPS15, BECN1 and ATG14) of PI3KC3-C1 (Extended Data Fig. 6). The peripheral regions, including the FIP200 molecule binding with ULK1MIT:ATG13MIM and VPS34HELCAT (helical and catalytic domains), were not modeled because of their high mobility. In contrast, the regions containing VPS15HSD (helical solenoid domain), BECN1CC–ATG14CC, BECN1BH3 and the FIP200 arm not bound to ULK1MIT:ATG13MIM (hereafter, the ‘proximal arm’) were the best resolved parts, with local resolution at 6.84 Å (Extended Data Fig. 6h). This enabled unambiguous identification of the secondary structures identified in the atomistic structures of the PI3KC3-C1 core30 and the proximal arm of FIP200 (Fig. 4b and Extended Data Fig. 7a).Fig. 3Direct interaction between ULK1C and PI3KC3-C1.a, Bead-binding assay of ULK1C core and PI3KC3-C1. GSH beads coated with GST tag alone or GST-tagged PI3KC3-C1 was used to recruit various combination of GFP-tagged proteins. The schematic drawing illustrates the bait and prey samples used for each condition. b, Complementary bead-binding assay for confirming the interaction between FIP200 and PI3KC3-C1. GST-tagged various proteins were used to recruit the mCherry-tagged PI3KC3-C1. c, SDS–PAGE of the ULK1C:PI3KC3-C1 pulldown sample. Strep resin was used to pull down the TSF-tagged VPS15. The eluate was obtained by washing the resin with buffer containing 10 mM d-desthiobiotin and visualized by SDS–PAGE and Coomassie blue staining. The experiment was repeated two times with similar results.Source dataFig. 4Structure of the ULK1C:PI3KC3-C1 supercomplex.a, Representative 2D class average of the ULK1C:PI3KC3-C1 supercomplex. Scale bar, 20 nm. b, Overview of the EM map and the coordinates of ULK1C:PI3KC3-C1 supercomplex superposed with the EM map contoured at 8σ. Note: only one FIP200 molecule of the ULK1C was modeled (blue). c, Close-up view of the FIP200-binding interface of PI3KC3-C1. FIP200 is shown in surface representation and PI3KC3-C1 is shown in ribbon representation. The FIP200ULD, VPS15HSD, BECN1, CC and zinc-finger domains formed by BECN1–ATG14 are indicated. d, Close-up view of the unassigned EM densities at sites 1 and 2. The local refinement map is shown at 22σ. The assigned map is colored according to the molecule (blue, FIP200; pink, VPS15; green, BECN1; yellow, ATG14). The extra densities observed at site 1 are shown in gray and indicated by arrows.

Direct interaction between ULK1C and PI3KC3-C1.

a, Bead-binding assay of ULK1C core and PI3KC3-C1. GSH beads coated with GST tag alone or GST-tagged PI3KC3-C1 was used to recruit various combination of GFP-tagged proteins. The schematic drawing illustrates the bait and prey samples used for each condition. b, Complementary bead-binding assay for confirming the interaction between FIP200 and PI3KC3-C1. GST-tagged various proteins were used to recruit the mCherry-tagged PI3KC3-C1. c, SDS–PAGE of the ULK1C:PI3KC3-C1 pulldown sample. Strep resin was used to pull down the TSF-tagged VPS15. The eluate was obtained by washing the resin with buffer containing 10 mM d-desthiobiotin and visualized by SDS–PAGE and Coomassie blue staining. The experiment was repeated two times with similar results.

Source data

Structure of the ULK1C:PI3KC3-C1 supercomplex.

a, Representative 2D class average of the ULK1C:PI3KC3-C1 supercomplex. Scale bar, 20 nm. b, Overview of the EM map and the coordinates of ULK1C:PI3KC3-C1 supercomplex superposed with the EM map contoured at 8σ. Note: only one FIP200 molecule of the ULK1C was modeled (blue). c, Close-up view of the FIP200-binding interface of PI3KC3-C1. FIP200 is shown in surface representation and PI3KC3-C1 is shown in ribbon representation. The FIP200ULD, VPS15HSD, BECN1, CC and zinc-finger domains formed by BECN1–ATG14 are indicated. d, Close-up view of the unassigned EM densities at sites 1 and 2. The local refinement map is shown at 22σ. The assigned map is colored according to the molecule (blue, FIP200; pink, VPS15; green, BECN1; yellow, ATG14). The extra densities observed at site 1 are shown in gray and indicated by arrows.

Two main interfaces were identified between FIP200 and PI3KC3-C1 (Fig. 4c). The first interface is formed between the FIP200(166–181, 473–485) and BECN1BH3 (Extended Data Fig. 7b), a well-studied Bcl-2/Bcl-XL-binding region that regulates autophagy and apoptosis31–34. A zinc-finger motif, composed of BECN1C137, C140 and ATG14C43, C46, is located behind the BH3 domain and supports the conformation. The BECN1–ATG14 zinc-finger and the BECN1 BH3 region were not visualized in previous PI3KC3-C1 structures; thus, it appears that the presence of the FIP200 proximal arm induces their ordering. The second interface is formed by the globular ULD domain of FIP200 and the curved HSD domain of VPS15 (Extended Data Fig. 7c). The VPS15HSD adjoins FIP200 sites 1 and 2, forming a large pocket with the FIP200 proximal arm and ULD. Fragmented EM density is observed in this pocket (Fig. 4d). Superposition of a predicted model of the ULK1C core onto the supercomplex structure shows that the unassigned density could be contributed by ATG13371–385; however, the resolution is insufficient for unambiguous assignment. Overall, the structure shows that PI3KC3-C1 and FIP200 interact extensively and intimately. It is striking that the BECN1BH3 and ATG14–BECN1 zinc-finger regions become ordered and wrap more than halfway around the FIP200 proximal arm and that these regions closely approach ATG13 sites 1 and 2.

FIP200 scaffolding of ULK1 dimerizationIn the same ULK1C:PI3KC3-C1 mixture dataset described above, a particle class of ULK1C showed clear 2:2:2 stoichiometry (Fig. 5a and Extended Data Fig. 5c). The presence of the ULK1MIT:ATG13MIM heterodimer was confirmed on both FIP200 shoulders, demonstrating that the stoichiometry of the ULK1 complex was altered in the cryo-EM sample by the presence of PI3KC3-C1.Fig. 5ULK1 dimerization on the FIP200 scaffold.a, Representative 2D class average and the EM map of the ULK1C (2:2:2) core in the presence of PI3KC3-C1. The ULK1MIT:ATG13MIM domains are indicated with arrows. Scale bar, 20 nm. The EM map is contoured at 7σ. b, Representative 2D class average and EM map of the ULK1C (2:2:2) core of the ATG13450–517 truncation mutant. The EM map is contoured at 14σ. c, Repeated 2D class averages and EM map of the ULK1C (2:1:1) core as shown in Fig. 1b,c for comparison. The EM map is contoured at 12σ.ATG13 site 2 is not involved in the 2:2:2 complex; therefore, we hypothesized that this site might function as a brake to negatively regulate formation of this complex. To test this hypothesis, we truncated ATG13IDR at residue 449 and determined the cryo-EM structure of an ATG13450–517-containing version of the ULK1C core in the absence of PI3KC3-C1. The 2D averages clearly show densities of the ULK1MIT:ATG13MIM heterodimer on both sides of the shoulder (Fig. 5b), which was confirmed by 3D reconstruction at 4.46 Å (Extended Data Fig. 5d–h). The stoichiometry was investigated in solution by SEC with in-line multiangle light scattering (MALS) (Extended Data Fig. 8a) and mass photometry (Fig. 6). The ULK1C cores in the presence and absence of the ATG13IDR region displayed masses consistent with 2:1:1 and 2:2:2 complexes, respectively, corroborating the cryo-EM data. Furthermore, the ATG13 F394D;F397D;E403K mutant, designed to disrupt site 2, also formed a complex with 2:2:2 stoichiometry (Extended Data Fig. 8b). This observation demonstrates that site 2 of ATG13IDR opposes the recruitment of the second ULK1 kinase to the FIP200 dimer.Fig. 6Concentration-dependent stoichiometry shift of ULK1C.Mass photometry measurements of (i) FIP200NTD, (ii) ULK1MIT:ATG13 (363–517), (iii) full-length PI3KC1-C1, (iv) FIP200NTD with truncated ULK1MIT:ATG13 (450–517; note: the molecular weight shifted because of the truncation), (v,vi) FIP200NTD with ULK1MIT:ATG13 (363–517) crosslinked at 10 nM in the absence or presence of 5 nM PI3KC3-C1, (vii,viii) FIP200NTD with ULK1MIT:ATG13 (363–517) crosslinked at 200 nM in the absence or presence of 100 nM PI3KC3-C1 and (ix,x) FIP200NTD with ULK1MIT:ATG13 (363–517) at crosslinked 1,000 nM in the absence or presence of 500 nM PI3KC3-C1. Solid curves represent fits to multiple Gaussian to estimate the average mass and the percentage of each population (Extended Data Table 1). The percentage was calculated from the area of the peaks in the range of 0–500 kDa. The schematic drawing illustrates the setting of each measurement (left) and the predicted assemblies (bottom). Left, the theoretical molecular weights of each assembly are shown next to the diagrams (i–iv). Right, the crosslinking concentrations of FIP200 and ULK1:ATG13 used for each measurement are indicated with red labels (v–x).Source dataWe sought to determine whether PI3KC3-C1 also promoted the formation of the 2:2:2 complex in solution. Using mass photometry, we observed that the ULK1C core alone at 10 nM concentration formed a 2:1:1 complex (Fig. 6(v)), consistent with the SEC–MALS results. The presence of PI3KC3-C1 did not alter the stoichiometry in solution (Fig. 6(vi)). We hypothesized that PI3KC3-C1 might facilitate the membrane recruitment and increased local concentration of ULK1C in autophagy initiation, which might manifest in a cryo-EM experiment as recruitment and concentration of ULK1 at the air–water interface on EM grids. We found that, upon increasing the concentration from 200 nM to 1 μM, abundant 2:2:2 complex appeared in solution, roughly coequal to the 2:1:1 complex (Fig. 6(ix)). Addition of PI3KC3-C1 did not lead to a further enhancement of the 2:2:2-to-2:1:1 ratio in solution (Fig. 6(x)). This observation shows that mass action alone can drive the 2:1:1-to-2:2:2 state conversion, even in the absence of PI3KC3-C1 and for constructs containing ATG13 site 2.

FIP200 scaffolding of ULK1 dimerization

In the same ULK1C:PI3KC3-C1 mixture dataset described above, a particle class of ULK1C showed clear 2:2:2 stoichiometry (Fig. 5a and Extended Data Fig. 5c). The presence of the ULK1MIT:ATG13MIM heterodimer was confirmed on both FIP200 shoulders, demonstrating that the stoichiometry of the ULK1 complex was altered in the cryo-EM sample by the presence of PI3KC3-C1.Fig. 5ULK1 dimerization on the FIP200 scaffold.a, Representative 2D class average and the EM map of the ULK1C (2:2:2) core in the presence of PI3KC3-C1. The ULK1MIT:ATG13MIM domains are indicated with arrows. Scale bar, 20 nm. The EM map is contoured at 7σ. b, Representative 2D class average and EM map of the ULK1C (2:2:2) core of the ATG13450–517 truncation mutant. The EM map is contoured at 14σ. c, Repeated 2D class averages and EM map of the ULK1C (2:1:1) core as shown in Fig. 1b,c for comparison. The EM map is contoured at 12σ.

ULK1 dimerization on the FIP200 scaffold.

a, Representative 2D class average and the EM map of the ULK1C (2:2:2) core in the presence of PI3KC3-C1. The ULK1MIT:ATG13MIM domains are indicated with arrows. Scale bar, 20 nm. The EM map is contoured at 7σ. b, Representative 2D class average and EM map of the ULK1C (2:2:2) core of the ATG13450–517 truncation mutant. The EM map is contoured at 14σ. c, Repeated 2D class averages and EM map of the ULK1C (2:1:1) core as shown in Fig. 1b,c for comparison. The EM map is contoured at 12σ.

ATG13 site 2 is not involved in the 2:2:2 complex; therefore, we hypothesized that this site might function as a brake to negatively regulate formation of this complex. To test this hypothesis, we truncated ATG13IDR at residue 449 and determined the cryo-EM structure of an ATG13450–517-containing version of the ULK1C core in the absence of PI3KC3-C1. The 2D averages clearly show densities of the ULK1MIT:ATG13MIM heterodimer on both sides of the shoulder (Fig. 5b), which was confirmed by 3D reconstruction at 4.46 Å (Extended Data Fig. 5d–h). The stoichiometry was investigated in solution by SEC with in-line multiangle light scattering (MALS) (Extended Data Fig. 8a) and mass photometry (Fig. 6). The ULK1C cores in the presence and absence of the ATG13IDR region displayed masses consistent with 2:1:1 and 2:2:2 complexes, respectively, corroborating the cryo-EM data. Furthermore, the ATG13 F394D;F397D;E403K mutant, designed to disrupt site 2, also formed a complex with 2:2:2 stoichiometry (Extended Data Fig. 8b). This observation demonstrates that site 2 of ATG13IDR opposes the recruitment of the second ULK1 kinase to the FIP200 dimer.Fig. 6Concentration-dependent stoichiometry shift of ULK1C.Mass photometry measurements of (i) FIP200NTD, (ii) ULK1MIT:ATG13 (363–517), (iii) full-length PI3KC1-C1, (iv) FIP200NTD with truncated ULK1MIT:ATG13 (450–517; note: the molecular weight shifted because of the truncation), (v,vi) FIP200NTD with ULK1MIT:ATG13 (363–517) crosslinked at 10 nM in the absence or presence of 5 nM PI3KC3-C1, (vii,viii) FIP200NTD with ULK1MIT:ATG13 (363–517) crosslinked at 200 nM in the absence or presence of 100 nM PI3KC3-C1 and (ix,x) FIP200NTD with ULK1MIT:ATG13 (363–517) at crosslinked 1,000 nM in the absence or presence of 500 nM PI3KC3-C1. Solid curves represent fits to multiple Gaussian to estimate the average mass and the percentage of each population (Extended Data Table 1). The percentage was calculated from the area of the peaks in the range of 0–500 kDa. The schematic drawing illustrates the setting of each measurement (left) and the predicted assemblies (bottom). Left, the theoretical molecular weights of each assembly are shown next to the diagrams (i–iv). Right, the crosslinking concentrations of FIP200 and ULK1:ATG13 used for each measurement are indicated with red labels (v–x).Source data

Concentration-dependent stoichiometry shift of ULK1C.

Mass photometry measurements of (i) FIP200NTD, (ii) ULK1MIT:ATG13 (363–517), (iii) full-length PI3KC1-C1, (iv) FIP200NTD with truncated ULK1MIT:ATG13 (450–517; note: the molecular weight shifted because of the truncation), (v,vi) FIP200NTD with ULK1MIT:ATG13 (363–517) crosslinked at 10 nM in the absence or presence of 5 nM PI3KC3-C1, (vii,viii) FIP200NTD with ULK1MIT:ATG13 (363–517) crosslinked at 200 nM in the absence or presence of 100 nM PI3KC3-C1 and (ix,x) FIP200NTD with ULK1MIT:ATG13 (363–517) at crosslinked 1,000 nM in the absence or presence of 500 nM PI3KC3-C1. Solid curves represent fits to multiple Gaussian to estimate the average mass and the percentage of each population (Extended Data Table 1). The percentage was calculated from the area of the peaks in the range of 0–500 kDa. The schematic drawing illustrates the setting of each measurement (left) and the predicted assemblies (bottom). Left, the theoretical molecular weights of each assembly are shown next to the diagrams (i–iv). Right, the crosslinking concentrations of FIP200 and ULK1:ATG13 used for each measurement are indicated with red labels (v–x).

Source data

We sought to determine whether PI3KC3-C1 also promoted the formation of the 2:2:2 complex in solution. Using mass photometry, we observed that the ULK1C core alone at 10 nM concentration formed a 2:1:1 complex (Fig. 6(v)), consistent with the SEC–MALS results. The presence of PI3KC3-C1 did not alter the stoichiometry in solution (Fig. 6(vi)). We hypothesized that PI3KC3-C1 might facilitate the membrane recruitment and increased local concentration of ULK1C in autophagy initiation, which might manifest in a cryo-EM experiment as recruitment and concentration of ULK1 at the air–water interface on EM grids. We found that, upon increasing the concentration from 200 nM to 1 μM, abundant 2:2:2 complex appeared in solution, roughly coequal to the 2:1:1 complex (Fig. 6(ix)). Addition of PI3KC3-C1 did not lead to a further enhancement of the 2:2:2-to-2:1:1 ratio in solution (Fig. 6(x)). This observation shows that mass action alone can drive the 2:1:1-to-2:2:2 state conversion, even in the absence of PI3KC3-C1 and for constructs containing ATG13 site 2.

DiscussionDespite ULK1C and PI3KC3-C1 being at the heart of human autophagy initiation35, there have been major gaps in understanding how autophagy is switched on by these two complexes. The available structures of ULK1 complex components have been either too fragmentary or at inadequate resolution to draw mechanistic conclusions about the integrated regulation of the entire complexes. Beyond the ability of ULK1 to phosphorylate all four subunits of PI3KC3-C1 (refs. 21–24), it has been unclear how the activities of the two complexes are coordinated. Here, we obtained a cryo-EM reconstruction of the ULK1C core that permitted amino-acid-level interpretation. We then found that ULK1C and PI3KC3-C1 form a physical complex mediated by ULK1C FIP200 and PI3KC3-C1 VPS15, BECN1 and ATG14 subunits. The ULK1C core binds full-length PI3KC3-C1 stably enough to yield a cryo-EM reconstruction. Our findings differ from a report indicating that ULK1C binds to a BECN1:ATG14 subcomplex of PI3KC3-C1 with a full-length HORMA domain-containing ATG13 and ATG101 (ref. 36), which can be attributed to the use of the intact PI3KC3-C1 in this study versus the BECN1:ATG14 subcomplex used in the previous study36. In yeast, the Atg1 complex (the yeast counterpart of human ULK1C) and PI3KC3-C1 are also reported to be in direct physical contact, which mediates the recruitment of PI3KC3-C1 to the phagophore initiation site in yeast37. We note that the ULK1C:PI3KC3-C1 interface is distributed over a large region of the surfaces of these complexes, which has made it challenging to engineer stable mutants that selectively block the interaction. The determinants of binding, however, differ between yeast and humans. Among other differences, Atg38 is important for PI3KC3-C1 recruitment in yeast37, while we found that the human supercomplex assembles independently of the human Atg38 ortholog NRBF2. The same surface region of VPS15HSD binds to both FIP200 and NRBF2 (Extended Data Fig. 7c,d)17; therefore, the relationship between these interactions remains to be investigated further. The most important conclusions are that ULK1C and PI3KC3-C1 form a physical complex through interactions that involve the autophagy-specific ATG14 subunit of PI3KC3-C1 responsible for autophagy-specific functions.We previously found that TBK1 contains a scaffold domain that closely resembles the arm of FIP200 involved in binding to VPS15 (Extended Data Fig. 2b) and TBK1 has been shown to bind directly to PI3KC3-C1 (ref. 27). Most but not all autophagy initiation pathways require ULK1 and/or ULK2 (ULK1/2). In OPTN-mediated Parkin-dependent mitophagy, TBK1 replaces the requirement for ULK1/2 (ref. 27). It will be interesting to determine whether TBK1 uses its FIP200 arm-like domain to bypass the need for ULK1/2 in OPTN mitophagy. The role of the VPS15 pseudokinase domain in PI3KC3 function has been a mystery since its initial identification in yeast38. We recently elucidated how VPS15 regulates the lipid kinase activity of PI3KC3 (ref. 30). Here, we established a role for VPS15 as the major bridge to FIP200 in ULK1C recruitment. The N-terminal predicted zinc-finger domain of ATG14 is crucial for autophagy initiation and the recruitment of PI3KC3-C1 to initiation sites5,15. Yet the molecular function of the ATG14 N-terminal region in the context of PI3KC3-C1 has been elusive and it was not visualized in previous PI3KC3-C1 structures. Here, we found that this region cofolds with the BECN1 NTD and both wrap around the FIP200 proximal arm, consistent with and potentially explaining the function of this region in autophagy initiation. A role for BECN1BH3 in regulating autophagy through interactions with Bcl-1 is also well established39–41. As a limitation, we note that it was not possible to test complex formation by N-terminally truncated forms of ATG14 and BECN1 because of loss of stability. Moreover, the local resolution of the interface in the region involving ATG14 and BECN1BH3 was insufficient to atomistically model molecular contacts. Our observation that BECN1BH3 participates in ULK1 supercomplex formation suggests that BECN1BH3 could have a more direct function in autophagy initiation than previously appreciated.With respect to the regulation of autophagy initiation, we observed a rearrangement of ULK1C from a 2:1:1 to a 2:2:2 complex. ULK1 kinase activity requires autophosphorylation at T180 (ref. 42), as does its yeast ortholog, Atg1 (ref. 43). Artificially induced dimerization of yeast Atg1 promotes its activation44,45. The dimerization and subsequent autophosphorylation of receptor-linked tyrosine kinases (RTKs) is the central paradigm for kinase activation in growth factor signaling46. These findings suggest that PI3KC3-C1-regulated and ATG13-regulated ULK1 kinase dimerization could be an autophagic cognate of the dimerization-based RTK activation paradigm. Here, we observed that the presence of PI3KC3-C1 induced 2:2:2 conversion on cryo-EM grids but not in solution. We established that mass action can drive 2:2:2 conversion in solution in the absence of PI3KC3-C1. We found that ATG13IDR site 2 regulates 2:2:2 conversion both on cryo-EM grids and in solution, demonstrating a common molecular mechanism in both settings. In the supercomplex structure, the N-terminal portions of PI3KC3-C1 subunits BECN1 and ATG14 closely approach ATG13IDR sites 1 and 2 (Extended Data Fig. 7e,f). ULK1 initiates autophagy in the context of puncta of >30 molecules per cluster that are colocalized with the ATG14 subunit of PI3KC3-C1 (refs. 47,48). However, the small number of molecules involved and the transient nature of autophagy initiation have thus far precluded direct imaging of ULK1 dimerization. The elevated local concentration of ULK1 and associated molecules under conditions of corecruitment and colocalization with PI3KC3-C1 would be expected to help drive formation of the 2:2:2 complex. Taking all of these observations together, the data suggest the existence of regulated ULK1 dimerization on the FIP200 scaffold, which could be a versatile mechanism for ULK1 activation (Fig. 7).Fig. 7Model for ULK1 activation.a, Binding and recruitment of the ULK1C to the phagophore membrane by PI3KC3-C1. b, Formation of ULK1C:PI3KC3-C1 supercomplex. c, Increase in the local concentration of ULK1C facilitated by PI3KC3-C1. d, Formation of ULK1 (2:2:2) complex.Given the large interface between PI3KC3-C1 and FIP200 and the multiple contacts between ATG13IDR and FIP200, it seems likely that various upstream kinases and other signaling inputs could regulate formation of the ULK1C:PI3KC3-C1 supercomplex and the activated 2:2:2 ULK1C. While mammalian target of rapamycin C1 is one candidate to modulate these complexes, these mechanisms are, in principle, general and could be regulated by other kinases such as AMP-activated protein kinase49 or TBK1 (ref. 50) or by increased local concentration and colocalization driven by selective autophagy cargo receptor clustering51 or condensate formation52. The new activation-related interfaces identified in this study are also suggestive of new concepts for the therapeutic upregulation of autophagy. Thus, the concept of ULK1C:PI3KC3-C1 supercomplex formation at the heart of autophagy initiation puts both the signaling and therapeutic aspects of autophagy initiation into new perspective.

Despite ULK1C and PI3KC3-C1 being at the heart of human autophagy initiation35, there have been major gaps in understanding how autophagy is switched on by these two complexes. The available structures of ULK1 complex components have been either too fragmentary or at inadequate resolution to draw mechanistic conclusions about the integrated regulation of the entire complexes. Beyond the ability of ULK1 to phosphorylate all four subunits of PI3KC3-C1 (refs. 21–24), it has been unclear how the activities of the two complexes are coordinated. Here, we obtained a cryo-EM reconstruction of the ULK1C core that permitted amino-acid-level interpretation. We then found that ULK1C and PI3KC3-C1 form a physical complex mediated by ULK1C FIP200 and PI3KC3-C1 VPS15, BECN1 and ATG14 subunits. The ULK1C core binds full-length PI3KC3-C1 stably enough to yield a cryo-EM reconstruction. Our findings differ from a report indicating that ULK1C binds to a BECN1:ATG14 subcomplex of PI3KC3-C1 with a full-length HORMA domain-containing ATG13 and ATG101 (ref. 36), which can be attributed to the use of the intact PI3KC3-C1 in this study versus the BECN1:ATG14 subcomplex used in the previous study36. In yeast, the Atg1 complex (the yeast counterpart of human ULK1C) and PI3KC3-C1 are also reported to be in direct physical contact, which mediates the recruitment of PI3KC3-C1 to the phagophore initiation site in yeast37. We note that the ULK1C:PI3KC3-C1 interface is distributed over a large region of the surfaces of these complexes, which has made it challenging to engineer stable mutants that selectively block the interaction. The determinants of binding, however, differ between yeast and humans. Among other differences, Atg38 is important for PI3KC3-C1 recruitment in yeast37, while we found that the human supercomplex assembles independently of the human Atg38 ortholog NRBF2. The same surface region of VPS15HSD binds to both FIP200 and NRBF2 (Extended Data Fig. 7c,d)17; therefore, the relationship between these interactions remains to be investigated further. The most important conclusions are that ULK1C and PI3KC3-C1 form a physical complex through interactions that involve the autophagy-specific ATG14 subunit of PI3KC3-C1 responsible for autophagy-specific functions.

We previously found that TBK1 contains a scaffold domain that closely resembles the arm of FIP200 involved in binding to VPS15 (Extended Data Fig. 2b) and TBK1 has been shown to bind directly to PI3KC3-C1 (ref. 27). Most but not all autophagy initiation pathways require ULK1 and/or ULK2 (ULK1/2). In OPTN-mediated Parkin-dependent mitophagy, TBK1 replaces the requirement for ULK1/2 (ref. 27). It will be interesting to determine whether TBK1 uses its FIP200 arm-like domain to bypass the need for ULK1/2 in OPTN mitophagy. The role of the VPS15 pseudokinase domain in PI3KC3 function has been a mystery since its initial identification in yeast38. We recently elucidated how VPS15 regulates the lipid kinase activity of PI3KC3 (ref. 30). Here, we established a role for VPS15 as the major bridge to FIP200 in ULK1C recruitment. The N-terminal predicted zinc-finger domain of ATG14 is crucial for autophagy initiation and the recruitment of PI3KC3-C1 to initiation sites5,15. Yet the molecular function of the ATG14 N-terminal region in the context of PI3KC3-C1 has been elusive and it was not visualized in previous PI3KC3-C1 structures. Here, we found that this region cofolds with the BECN1 NTD and both wrap around the FIP200 proximal arm, consistent with and potentially explaining the function of this region in autophagy initiation. A role for BECN1BH3 in regulating autophagy through interactions with Bcl-1 is also well established39–41. As a limitation, we note that it was not possible to test complex formation by N-terminally truncated forms of ATG14 and BECN1 because of loss of stability. Moreover, the local resolution of the interface in the region involving ATG14 and BECN1BH3 was insufficient to atomistically model molecular contacts. Our observation that BECN1BH3 participates in ULK1 supercomplex formation suggests that BECN1BH3 could have a more direct function in autophagy initiation than previously appreciated.

With respect to the regulation of autophagy initiation, we observed a rearrangement of ULK1C from a 2:1:1 to a 2:2:2 complex. ULK1 kinase activity requires autophosphorylation at T180 (ref. 42), as does its yeast ortholog, Atg1 (ref. 43). Artificially induced dimerization of yeast Atg1 promotes its activation44,45. The dimerization and subsequent autophosphorylation of receptor-linked tyrosine kinases (RTKs) is the central paradigm for kinase activation in growth factor signaling46. These findings suggest that PI3KC3-C1-regulated and ATG13-regulated ULK1 kinase dimerization could be an autophagic cognate of the dimerization-based RTK activation paradigm. Here, we observed that the presence of PI3KC3-C1 induced 2:2:2 conversion on cryo-EM grids but not in solution. We established that mass action can drive 2:2:2 conversion in solution in the absence of PI3KC3-C1. We found that ATG13IDR site 2 regulates 2:2:2 conversion both on cryo-EM grids and in solution, demonstrating a common molecular mechanism in both settings. In the supercomplex structure, the N-terminal portions of PI3KC3-C1 subunits BECN1 and ATG14 closely approach ATG13IDR sites 1 and 2 (Extended Data Fig. 7e,f). ULK1 initiates autophagy in the context of puncta of >30 molecules per cluster that are colocalized with the ATG14 subunit of PI3KC3-C1 (refs. 47,48). However, the small number of molecules involved and the transient nature of autophagy initiation have thus far precluded direct imaging of ULK1 dimerization. The elevated local concentration of ULK1 and associated molecules under conditions of corecruitment and colocalization with PI3KC3-C1 would be expected to help drive formation of the 2:2:2 complex. Taking all of these observations together, the data suggest the existence of regulated ULK1 dimerization on the FIP200 scaffold, which could be a versatile mechanism for ULK1 activation (Fig. 7).Fig. 7Model for ULK1 activation.a, Binding and recruitment of the ULK1C to the phagophore membrane by PI3KC3-C1. b, Formation of ULK1C:PI3KC3-C1 supercomplex. c, Increase in the local concentration of ULK1C facilitated by PI3KC3-C1. d, Formation of ULK1 (2:2:2) complex.

Model for ULK1 activation.

a, Binding and recruitment of the ULK1C to the phagophore membrane by PI3KC3-C1. b, Formation of ULK1C:PI3KC3-C1 supercomplex. c, Increase in the local concentration of ULK1C facilitated by PI3KC3-C1. d, Formation of ULK1 (2:2:2) complex.

Given the large interface between PI3KC3-C1 and FIP200 and the multiple contacts between ATG13IDR and FIP200, it seems likely that various upstream kinases and other signaling inputs could regulate formation of the ULK1C:PI3KC3-C1 supercomplex and the activated 2:2:2 ULK1C. While mammalian target of rapamycin C1 is one candidate to modulate these complexes, these mechanisms are, in principle, general and could be regulated by other kinases such as AMP-activated protein kinase49 or TBK1 (ref. 50) or by increased local concentration and colocalization driven by selective autophagy cargo receptor clustering51 or condensate formation52. The new activation-related interfaces identified in this study are also suggestive of new concepts for the therapeutic upregulation of autophagy. Thus, the concept of ULK1C:PI3KC3-C1 supercomplex formation at the heart of autophagy initiation puts both the signaling and therapeutic aspects of autophagy initiation into new perspective.

MethodsPlasmid constructionThe sequences of all DNAs encoding components of human ULK1C were codon-optimized, synthesized and then subcloned into the pCAG vector. The fusion construct of the ATG13 C-terminal domain with ULK1MIT domain was subcloned with a TwinStrep-Flag (TSF), tobacco etch virus (TEV) cleavage site, ATG13 (363–517 or 450–517) and 5-aa linker (GSDEA) followed by ULK1 (836–1050) into the pCAG vector. Proteins were tagged with GST (glutathione S-transferase), MBP (maltose-binding protein) or TSF for affinity purification, pulldown or GSH (glutathione) bead assay. All constructs were verified by DNA sequencing. Details are shown in Supplementary Table 1. A detailed procedure is described at 10.17504/protocols.io.bp2l6x3b5lqe/v1.Protein expression and purificationFor cryo-EM samples of FIP200NTD:ATG13363–517–ULK1MIT complexes, pCAG-FIP200NTD (1–640) was cotransfected with pCAG-TSF-ATG13363–517-ULK1MIT (836–1050) using the polyethylenimine (PEI) (Polysciences) transfection system. FIP200NTD–MBP:ATG13363–517–ULK1MIT–TSF was coexpressed in HEK293 GnTi− cells for the FIP200:PI3KC3-C1 cryo-EM study. Cells were transfected at a concentration of 2 × 106 cells per ml. After 48 h, cells were pelleted at 500g for 10 min, washed with PBS once and then stored at −80 °C. Cell pellets were lysed at room temperature for 20 min with lysis buffer (25 mM HEPES pH 7.5, 200 mM NaCl, 2 mM MgCl2, 1 mM TCEP and 10% glycerol) with 5 mM EDTA, 1% Triton X-100 and protease inhibitor cocktail (Thermo Fisher Scientific) before being cleared at 17,000 rpm for 35 min at 4 °C. The clarified supernatant was purified on Strep-Tactin Sepharose resin (IBA Lifesciences) and then eluted in the lysis buffer with 4 mM desthiobiotin (Sigma). After His6–TEV cleavage at 4 °C overnight, samples were concentrated then load onto a Superose 6 Increase 10/300 GL column (Cytiva) in a buffer of 25 mM HEPES pH 7.5, 150 mM NaCl, 1 mM MgCl2 and 1 mM TCEP. A detailed procedure is described at 10.17504/protocols.io.bp2l6x3b5lqe/v1.The PI3KC3-C1 complex was expressed in HEK293 GnTi− cells by PEI transfection from codon-optimized pCAG-VPS15-TSF, pCAG-VPS34, pCAG-ATG14 and pCAG-Beclin-1. pCAG-GST-ATG14 or pCAG-mCherry-ATG14 was used in PI3KC3-C1 expression for GST pulldown or microscopy-based GSH bead assay. Cells were transfected at a concentration of 2 × 106 cells per ml and harvested 48 h after transfection. Pellets were homogenized 20 times by a Pyrex douncer (Corning) in lysis buffer with 25 mM TCEP and proteinase inhibitors (Thermo Fisher Scientific), before adding 10% Triton X-100 stock to a final 1% concentration. After rocking at 4 °C for 1 h, lysates were clarified by centrifugation (43,714g for 40 min at 4 °C) and incubated with Strep-Tactin Sepharose (IBA Lifesciences) at 4 °C overnight. After eluting with lysis buffer and 4 mM desthiobiotin, samples were concentrated and then loaded onto a Superose 6 Increase 10/300 GL column (Cytiva) in 25 mM HEPES pH 7.5, 300 mM NaCl, 1 mM MgCl2 and 25 mM TCEP. FIP200NTD constructs used in Fig. 3a,b were purified using Strep-Tactin Sepharose resin as described above and then loaded onto a Superose 6 Increase 10/300 GL column (Cytiva). ATG101:ATG13N HORMA-related proteins in Fig. 3a,b were purified using Strep resin and a Superdex 200 Increase 10/300 GL column (Cytiva) equilibrated in 25 mM HEPES pH 7.5, 150 mM NaCl, 1 mM MgCl2 and 1 mM TCEP. A detailed procedure is described at 10.17504/protocols.io.5qpvorjezv4o/v1.Strep pulldown assayFIP200NTD–TSF wild type (WT) or mutants were cotransfected with GST–ATG13363–517 and/or ULK1MIT–MBP in 10 ml of HEK293 GnTi− cells. The cells were harvested 48 h after transfection. The pellets were homogenized in 0.5 ml of lysis buffer, protease inhibitors and 1% Triton X-100 and clarified at 40,000g for 15 min. The lysate was incubated with 30 µl of Strep-Tactin Sepharose resin (IBA Lifesciences) at 4 °C for 3 h. The beads were washed four times and then eluted in 50 µl of lysis buffer and 4 mM desthiobiotin. Then, 18 ml of eluent was mixed with lithium dodecylsulfate (LDS)–β-mercaptoethanol (BME) buffer, heated at 60 °C for 5 min and subjected to SDS–PAGE. The gel was then stained with Coomassie brilliant blue G250. For loading input control, 0.3% lysate per sample was mixed with 1× LDS sample buffer (Life Technologies) containing 100 mM BME, boiled at 95 °C for 5 min and then analyzed with 4–12% Bis–Tris gels (Life Technologies). Gels were electrotransferred to polyvinyldifluoride (PVDF) membranes and immunoblotted with various antibodies. FIP200-NTD–TSF lysate input was visualized using horseradish-peroxidase-conjugated anti-Flag antibody (Abcam, ab49763) and SuperSignal West Femto maximum sensitivity substrate enhanced chemiluminescence kit (Thermo Fisher Scientific, 34095). GST–ATG13c lysate input was analyzed using goat anti-GST antibody (Cytiva, 27457701) and donkey anti-goat IgG (Cy5) (Abcam, ab6566). ULKc–MBP lysate was analyzed using rabbit anti-MBP antibody (Thermo Fisher Scientific, PA1-989) and Alexa Fluor 488 goat anti-rabbit IgG (Invitrogen, A11034). The chemiluminescence or fluorescence signals were detected using a ChemiDocMP Imaging system (Biorad). A detailed procedure is described at 10.17504/protocols.io.3byl4jmpolo5/v1.Microscopy-based GSH bead protein–protein interaction assayA mixture of 1 µM purified GST-tagged protein and 500 nM purified fluorescent protein in a total volume of 70 µl was incubated with 9 µl of preblocked glutathione Sepharose beads (Cytiva) in a reaction buffer containing 25 mM HEPES at pH 7.5, 150 mM NaCl, 1 mM MgCl2 and 1 mM TCEP. After incubation at room temperature for 30 min, samples were mixed with an additional 100 µl of reaction buffer and then transferred to the observation chamber for imaging. Images were acquired on a Nikon A1 confocal microscope with a Nikon Plan APO VC ×20 0.75 numerical aperture ultraviolet microscope objective. Three biological replicates were performed for each experimental condition. A detailed procedure is described at 10.17504/protocols.io.4r3l27xdxg1y/v1.SEC–MALSThe purified FIP200NTD:ATG13–ULK1MIT complexes were concentrated to 4–5 mg ml−1. SEC–MALS experiments were performed using an Agilent 1200 high-performance liquid chromatography system (Agilent Technologies), coupled to a Wyatt DAWN HELEOS-II MALS instrument and a Wyatt Optilab rEX differential refractometer (Wyatt). For chromatographic separation, a WTC-050S5 size-exclusion column (Wyatt) with a 40-μl sample loop was used at a flow rate of 0.3 ml min−1 in the buffer of 25 mM HEPES pH 7.5, 200 mM NaCl, 1 mM MgCl2 and 2 mM TCEP. The outputs were analyzed using ASTRA V software (Wyatt). MALS signals, combined with the protein concentration determined by refractive index, were used to calculate the molecular mass of the complex. A detailed procedure is described at 10.17504/protocols.io.j8nlkom4xv5r/v1.Sample preparation of FIP200:PI3KC3-C1 complex for cryo-EMFIP200NTD–MBP:ATG13363–517–ULK1MIT–TSF (final 5 µM) was mixed with PI3KC3-C1–TSF complex at a 1.5:1 molar ratio in a total volume of 200 µl, rocking at 4 °C overnight. On the next day, the sample was incubated with 50 µl of Strep-Tactin Sepharose resin (IBA Lifesciences) at 4 °C for 3 h. After one wash, the beads were eluted in 50 µl of 10 mM d-desthiobiotin with the buffer of 25 mM HEPES pH 7.5, 200 mM NaCl, 1 mM MgCl2 and 10 mM TCEP. A detailed procedure is described at 10.17504/protocols.io.5qpvorjezv4o/v1.Sample vitrification and cryo-EM data acquisitionFor cryo-EM sample preparation, 3 μl of protein solution was applied onto a grid freshly glow-discharged in PELCO easiGlow system (Ted Pella). In-house graphene grids were prepared from Trivial Transfer graphene sheets (ACS Material) and Quantifoil R2/1 300-mesh gold (EM Sciences) following a protocol introduced by Ahn et al.53. The graphene grids were used for the ULK1C (2:1:1) core sample and holey carbon grids (Quantifoil R1.2/1.3 or R2/1 300-mesh; EM Sciences) were sufficient for the other samples. The samples were vitrified with a Vitrobot cryo-plunger (Thermo Fisher Scientific) in plunging conditions optimized previously. Then, 0.05% (w/v) of n-octyl-β-d-glucopyranoside was added in the sample solution as a surfactant before vitrification.The datasets of the ULK1C (2:1:1) core, the ULK1C:PI3KC3-C1 mixture and the ULK1C:PI3KC3-C1 pulldown were recorded at a 300 kV Titan Krios microscope (Thermo Fisher Scientific) equipped with X-FEG and an energy filter set to a width of 20 eV. Automated data acquisition was achieved using SerialEM54 on a K3 Summit direct detection camera (Gatan) at a magnification of ×81,000 and a corresponding pixel size of 0.525 Å in super-resolution mode with a defocus range of −0.8 to −2.0 μm. The ULK1C:PI3KC3-C1 pulldown sample was collected on a specimen stage tilted at 0°, 20° and 30°. Image stacks with 50 frames was collected with a total dose of 50 e− per Å2. The dataset of the ULK1C (2:2:2) core was recorded on a 200-kV Talos Arctica microscope (Thermo Fisher Scientific) equipped with the K3 Summit camera in super-resolution correlated double-sampling mode. The magnification and the pixel size were ×36,000 and 0.5575 Å in super-resolution mode, respectively. Other details of the dataset collection are summarized in Table 1. A detailed procedure is described at 10.17504/protocols.io.kqdg39rreg25/v1.Table 1Cryo-EM data collection, refinement and validation statisticsULK1C core(2:1:1) stoichiometryULK1C:PI3KC3-C1 supercomplexULK1C core(2:2:2) stoichiometryin PI3KC3-C1 mixtureULK1C core(2:2:2) stoichiometryATG13450–517 mutant(EMD-40658), (PDB 8SOI)(EMD-45297), (PDB 9C82)(EMD-40715), (PDB 8SQZ)(EMD-40735), (PDB 8SRM)Data collection and processingMagnification×81,000×81,000×81,000×36,000Voltage (kV)300300300200Electron exposure (e− per Å2)50505050Defocus range (μm)−0.8 to −2.0−0.8 to −2.0−0.8 to −2.0−0.8 to −2.0Pixel size (Å)1.051.051.051.115Symmetry imposedC1C1C1C1Images8,468589 (0° tilt)6,973 (20° tilt)3,740 (30° tilt)5,3832,286Initial particle images (no.)2,156,5164,473,551502,985968,884Final particle images (no.)267,71921,93797,987149,409Map resolution, global (Å)4.28.035.284.46 FSC threshold0.1430.1430.1430.143Map resolution range (Å)3.48–63.837.15–23.074.96–81.033.77–70.64RefinementInitial model used (PDB code)AlphaFold2 modelAlphaFold2 model8SOI8SOIModel resolution (Å)3.78.76.86.5 FSC threshold0.50.50.50.5Model resolution range (Å)3.48–22.497.15–16.904.98–16.473.78–11.37Map sharpening B factor (A2)143.2752.0431.6155.4Model composition Nonhydrogen atoms10,24311,1727,4385,773 Protein residues1,2762,2541,4981,163 Ligands0000B factors (Å2) Protein126.431,099.03319.79208.69 Ligands----Root-mean-square deviations Bond lengths (Å)0.0030.0020.0040.002 Bond angles (°)0.5980.6070.6330.405Validation MolProbity score1.72.481.951.51 Clashscore9.5230.518.074.44 Poor rotamers (%)1.5---Ramachandran plot Favored (%)96.7591.1791.4195.87 Allowed (%)3.178.838.594.13 Disallowed (%)0.008000PDB, Protein Data Bank; FSC, Fourier shell correlation.Image processing and 3D reconstructionThe datasets were processed by following the workflow in cryoSPARC55. In brief, the super-resolution video stacks were motion-corrected and binned 2× by Fourier cropping using Patch motion correction. Contrast transfer function (CTF) determination was performed using Patch CTF estimation, followed by manual removal of the outlier micrographs on the basis of the estimated defocus and resolution value. Single particles were automatically picked by Topaz56 using a manually trained model and extracted with a window size 1.5 times larger than the target particle before further binning 2× to 4× to facilitate subsequent processing. A 2D classification was then used to remove obvious junk particles. The initial models were obtained using ab initio reconstruction. In case the reconstruction job failed to give a healthy initial model, the classes displaying high-resolution features in the 2D classification step were selected and used for ab initio reconstruction. Further classification was performed at the 3D level by multiple rounds of heterogeneous refinement until a clean substack was obtained. The particles were re-extracted with the refined coordinates on micrographs at the original 2× bin pixel size and used for homogeneous refinement for multiple rounds until the final resolution converged. To further improve the quality of the map, local refinement was applied to the datasets of the ULK1C (2:1:1) core and the ULK1C:PI3KC3-C1 pulldown sample. Masking areas were decided by 3D Variability or 3D Flex and the masks were created by UCSF ChimeraX57 and Volume Tools. Each local map was aligned to the consensus map and composed using EMAN2 (ref. 58). The composed maps were then used for model building. The details of data processing are summarized in Table 1. A detailed procedure is described in 10.17504/protocols.io.x54v9d99mg3e/v1.Model building, validation and visualizationThe in silico models of the ULK1C (2:1:1) core and the full-length PI3KC3-C1 were generated by AlphaFold2 prediction29. The resolution of the observed maps enabled amino acid sequence assignment. The primary and secondary structure of the predicted models agreed well with the EM maps. A flexible model fitting using the real-time molecular dynamics simulation-based program ISOLDE59 implemented in the visualization software UCSF ChimeraX was performed, followed by iterative refinement by using the model-editing software Coot60 manually and real-space refinement in PHENIX61 automatically.The two ULK1C (2:2:2) core models were created by aligning two copies of FIP200NTD:ATG13363–517:ULK1MIT subtrimeric complex to the FIP200 dimerization domain, followed by flexible model fitting with using ISOLDE in ChimeraX. The side chains were removed because of the moderate resolution of these maps. The ULK1C:PI3KC3-C1 supercomplex coordinates were generated by fitting of the individual structures of the ULK1C (2:1:1) and the PI3KC3-C1 to the cryo-EM map. The side chains were removed. The quality of the models was validated by using the validation tools in PHENIX and the online validation service provided by wwPDB62,63. The details of the model quality assessment are summarized in Table 1. All figures and videos were made using UCSF ChimeraX. The detailed procedure is described at 10.17504/protocols.io.j8nlkw77wl5r/v1.AlphaFold2 model predictionModels of the FIP200NTD:ATG13MIT:ULKMIM core domain were generated using the ColabFold implementation of AlphaFold2 (refs. 29,64–67). The default MMseqs2 (ref. 68) pipeline was used to generate and pair multiple-sequence alignments for each protein complex predicted. We predicted the following complexes: dimeric FIP200 (1–640), ULK1 (828–1050) and ATG13 (363–517); dimeric FIP200 (1–640) and ATG13 (363–450); dimeric FIP200 (1–640), ATG13 (450–517) and ULK (828–1050); monomeric FIP200 (1–640) and ATG13 (363–450); monomeric FIP200 (1–640), ATG13 (450–517) and ULK (828–1050). In each case, the models were assessed on the basis of the global predicted template modeling score (pTM) and the interfacial PTM (iPTM) score, with iPTM > 0.5 meriting further inspection. The predicted contacts involving ATG13, FIP200 and ULK1 were manually inspected in UCSF ChimeraX and the local predicted local distance difference test (pLDDT) score of the contacting residues was used in conjunction with an assessment of the physiological environment of the interfaces to judge the quality of the predicted residue contacts. The highest-scoring local pLDDT interfaces were used for figure making and interpretation of HDX data.Generation of KO lines using CRISPR–Cas9CRISPR guide RNAs (gRNAs) targeting ATG13 (5′-CCGCGAGTTTGATGCCTTTG-3′ (ref. 69) and 5′-TTGCTTCATGTGTAACCTCTGGG-3′) were cloned into a BbsI-linearized pSpCas9(BB)-2A-GFP vector70 (a gift from F. Zhang; Addgene, plasmid 48138)70 using a Gibson cloning kit (New England Biolabs). gRNA-containing constructs were then sequence-verified and transfected into WT HeLa cells with X-tremeGENE 9 (Roche) overnight and GFP-positive single cells were sorted by fluorescence-activated cell sorting into 96-well plates. Single-cell colonies were left to grow and screened by immunoblotting for loss of the targeted gene product. This procedure is described in detail at 10.17504/protocols.io.j8nlkkzo6l5r/v1.Cloning and generation of stable cell linespBMN-HA-WT ATG13 was previously described (Addgene, 186223)71. Complementary DNAs encoding ATG13 mutants (site 2, F394D;M396D;F397D;E403K; site 4, F449D;A452D;F453D;S454A; site M, L495D;L499D; sites 2, 4 and M, all substitutions) were synthesized by Integrated DNA Technologies and cloned into pBMN-HA-C1 (Addgene, 188645) using a Gibson cloning kit (New England Biolabs). Generation of stable cell lines was previously described72 and the detailed procedure is available at 10.17504/protocols.io.dm6gpjzm8gzp/v1. pMRX-IP-HaloTag7-LC3 (Addgene, 184899) and HA-tagged ATG13 variants were stably expressed in ATG13-KO cells using retroviral transduction.Halo assay to quantify starvation induced autophagyWe measured autophagic flux using a previously described method72,73. In brief, 350,000–400,000 cells were seeded the day before treatment in six-well plates. Cells were treated with 50 nM TMR ligand (Promega) in growth medium (DMEM with 10% FBS, 4.5 g L−1 glucose (Sigma), 1× GlutaMAXTM (Thermo Fisher Scientific), 1× nonessential amino acids (Thermo Fisher Scientific) and 25 mM HEPES. After 15 min, cells were washed three times with 1× Dulbecco’s PBS and treated with Earle’s balanced salt solution (EBSS; Gibco) for the indicated time periods. Following treatment, samples were analyzed by immunoblotting as described below. This procedure is described at 10.17504/protocols.io.3byl4qexzvo5/v1.ImmunoblottingFollowing feeding and treatment periods, the cells were washed with ice-cold 1× PBS, harvested using cell scrapers and lysed in lysis buffer (1× LDS sample buffer (Life Technologies) containing 100 mM DTT (Sigma)). Samples were boiled at 99 °C with shaking for 7 min and 25 μg (for Halo assays) and 70 μg (for the remaining blots) of protein per sample was analyzed with 4–12% Bis–Tris gels (Life Technologies) according to the manufacturer’s instructions. Gels were electrotransferred to PVDF membranes and immunoblotted with the indicated antibodies. For quantification, band intensities were measured using ImageLab 5.2.1 (BioRad). Statistical significance was calculated from at least three independent experiments using a two-way analysis of variance (ANOVA). Error bars are the means ± s.d. The procedure for immunoblotting is described in detail at 10.17504/protocols.io.dm6gpjzm8gzp/v1.Mass photometryHigh-precision coverslips (Azer Scientific) were cleaned by alternating between isopropanol and water three times in a bath sonicator, each 3 min, followed by air drying. The gasket was similarly cleaned with isopropanol and water three times without sonication and then air-dried. A volume of 19 µl of mass photometry buffer (30 mM HEPES pH 7.4, 5 mM MgSO4 and 1 mM EGTA) was added to the well for autofocus. The protein sample was then diluted in the mass photometry buffer in the wells to a concentration of 5–20 nM before data collection. If crosslinking was performed, 1 μl of 0.1% (v/v) glutaraldehyde in mass photometry buffer was added to 9 μl of protein sample and incubated for 10 min. The reaction was then quenched by adding 10 μl of 1 M Tris-HCl pH 7.4. All samples were diluted to 10 nM for the measurement of mass photometry. Protein contrast counts were acquired using a Refeyn TwoMP mass photometer, with three technical replicates. Mass calibration was performed using a standard mix of conalbumin, aldolase and thyroglobulin. Mass photometry profiles were analyzed and fitted to multiple Gaussian peaks, with the mean, s.d. and percentages calculated using DiscoverMP software (Refeyn). This procedure is described at 10.17504/protocols.io.kqdg3keq7v25/v1.Reporting summaryFurther information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

Plasmid constructionThe sequences of all DNAs encoding components of human ULK1C were codon-optimized, synthesized and then subcloned into the pCAG vector. The fusion construct of the ATG13 C-terminal domain with ULK1MIT domain was subcloned with a TwinStrep-Flag (TSF), tobacco etch virus (TEV) cleavage site, ATG13 (363–517 or 450–517) and 5-aa linker (GSDEA) followed by ULK1 (836–1050) into the pCAG vector. Proteins were tagged with GST (glutathione S-transferase), MBP (maltose-binding protein) or TSF for affinity purification, pulldown or GSH (glutathione) bead assay. All constructs were verified by DNA sequencing. Details are shown in Supplementary Table 1. A detailed procedure is described at 10.17504/protocols.io.bp2l6x3b5lqe/v1.

Plasmid construction

The sequences of all DNAs encoding components of human ULK1C were codon-optimized, synthesized and then subcloned into the pCAG vector. The fusion construct of the ATG13 C-terminal domain with ULK1MIT domain was subcloned with a TwinStrep-Flag (TSF), tobacco etch virus (TEV) cleavage site, ATG13 (363–517 or 450–517) and 5-aa linker (GSDEA) followed by ULK1 (836–1050) into the pCAG vector. Proteins were tagged with GST (glutathione S-transferase), MBP (maltose-binding protein) or TSF for affinity purification, pulldown or GSH (glutathione) bead assay. All constructs were verified by DNA sequencing. Details are shown in Supplementary Table 1. A detailed procedure is described at 10.17504/protocols.io.bp2l6x3b5lqe/v1.

Protein expression and purificationFor cryo-EM samples of FIP200NTD:ATG13363–517–ULK1MIT complexes, pCAG-FIP200NTD (1–640) was cotransfected with pCAG-TSF-ATG13363–517-ULK1MIT (836–1050) using the polyethylenimine (PEI) (Polysciences) transfection system. FIP200NTD–MBP:ATG13363–517–ULK1MIT–TSF was coexpressed in HEK293 GnTi− cells for the FIP200:PI3KC3-C1 cryo-EM study. Cells were transfected at a concentration of 2 × 106 cells per ml. After 48 h, cells were pelleted at 500g for 10 min, washed with PBS once and then stored at −80 °C. Cell pellets were lysed at room temperature for 20 min with lysis buffer (25 mM HEPES pH 7.5, 200 mM NaCl, 2 mM MgCl2, 1 mM TCEP and 10% glycerol) with 5 mM EDTA, 1% Triton X-100 and protease inhibitor cocktail (Thermo Fisher Scientific) before being cleared at 17,000 rpm for 35 min at 4 °C. The clarified supernatant was purified on Strep-Tactin Sepharose resin (IBA Lifesciences) and then eluted in the lysis buffer with 4 mM desthiobiotin (Sigma). After His6–TEV cleavage at 4 °C overnight, samples were concentrated then load onto a Superose 6 Increase 10/300 GL column (Cytiva) in a buffer of 25 mM HEPES pH 7.5, 150 mM NaCl, 1 mM MgCl2 and 1 mM TCEP. A detailed procedure is described at 10.17504/protocols.io.bp2l6x3b5lqe/v1.The PI3KC3-C1 complex was expressed in HEK293 GnTi− cells by PEI transfection from codon-optimized pCAG-VPS15-TSF, pCAG-VPS34, pCAG-ATG14 and pCAG-Beclin-1. pCAG-GST-ATG14 or pCAG-mCherry-ATG14 was used in PI3KC3-C1 expression for GST pulldown or microscopy-based GSH bead assay. Cells were transfected at a concentration of 2 × 106 cells per ml and harvested 48 h after transfection. Pellets were homogenized 20 times by a Pyrex douncer (Corning) in lysis buffer with 25 mM TCEP and proteinase inhibitors (Thermo Fisher Scientific), before adding 10% Triton X-100 stock to a final 1% concentration. After rocking at 4 °C for 1 h, lysates were clarified by centrifugation (43,714g for 40 min at 4 °C) and incubated with Strep-Tactin Sepharose (IBA Lifesciences) at 4 °C overnight. After eluting with lysis buffer and 4 mM desthiobiotin, samples were concentrated and then loaded onto a Superose 6 Increase 10/300 GL column (Cytiva) in 25 mM HEPES pH 7.5, 300 mM NaCl, 1 mM MgCl2 and 25 mM TCEP. FIP200NTD constructs used in Fig. 3a,b were purified using Strep-Tactin Sepharose resin as described above and then loaded onto a Superose 6 Increase 10/300 GL column (Cytiva). ATG101:ATG13N HORMA-related proteins in Fig. 3a,b were purified using Strep resin and a Superdex 200 Increase 10/300 GL column (Cytiva) equilibrated in 25 mM HEPES pH 7.5, 150 mM NaCl, 1 mM MgCl2 and 1 mM TCEP. A detailed procedure is described at 10.17504/protocols.io.5qpvorjezv4o/v1.

Protein expression and purification

For cryo-EM samples of FIP200NTD:ATG13363–517–ULK1MIT complexes, pCAG-FIP200NTD (1–640) was cotransfected with pCAG-TSF-ATG13363–517-ULK1MIT (836–1050) using the polyethylenimine (PEI) (Polysciences) transfection system. FIP200NTD–MBP:ATG13363–517–ULK1MIT–TSF was coexpressed in HEK293 GnTi− cells for the FIP200:PI3KC3-C1 cryo-EM study. Cells were transfected at a concentration of 2 × 106 cells per ml. After 48 h, cells were pelleted at 500g for 10 min, washed with PBS once and then stored at −80 °C. Cell pellets were lysed at room temperature for 20 min with lysis buffer (25 mM HEPES pH 7.5, 200 mM NaCl, 2 mM MgCl2, 1 mM TCEP and 10% glycerol) with 5 mM EDTA, 1% Triton X-100 and protease inhibitor cocktail (Thermo Fisher Scientific) before being cleared at 17,000 rpm for 35 min at 4 °C. The clarified supernatant was purified on Strep-Tactin Sepharose resin (IBA Lifesciences) and then eluted in the lysis buffer with 4 mM desthiobiotin (Sigma). After His6–TEV cleavage at 4 °C overnight, samples were concentrated then load onto a Superose 6 Increase 10/300 GL column (Cytiva) in a buffer of 25 mM HEPES pH 7.5, 150 mM NaCl, 1 mM MgCl2 and 1 mM TCEP. A detailed procedure is described at 10.17504/protocols.io.bp2l6x3b5lqe/v1.

The PI3KC3-C1 complex was expressed in HEK293 GnTi− cells by PEI transfection from codon-optimized pCAG-VPS15-TSF, pCAG-VPS34, pCAG-ATG14 and pCAG-Beclin-1. pCAG-GST-ATG14 or pCAG-mCherry-ATG14 was used in PI3KC3-C1 expression for GST pulldown or microscopy-based GSH bead assay. Cells were transfected at a concentration of 2 × 106 cells per ml and harvested 48 h after transfection. Pellets were homogenized 20 times by a Pyrex douncer (Corning) in lysis buffer with 25 mM TCEP and proteinase inhibitors (Thermo Fisher Scientific), before adding 10% Triton X-100 stock to a final 1% concentration. After rocking at 4 °C for 1 h, lysates were clarified by centrifugation (43,714g for 40 min at 4 °C) and incubated with Strep-Tactin Sepharose (IBA Lifesciences) at 4 °C overnight. After eluting with lysis buffer and 4 mM desthiobiotin, samples were concentrated and then loaded onto a Superose 6 Increase 10/300 GL column (Cytiva) in 25 mM HEPES pH 7.5, 300 mM NaCl, 1 mM MgCl2 and 25 mM TCEP. FIP200NTD constructs used in Fig. 3a,b were purified using Strep-Tactin Sepharose resin as described above and then loaded onto a Superose 6 Increase 10/300 GL column (Cytiva). ATG101:ATG13N HORMA-related proteins in Fig. 3a,b were purified using Strep resin and a Superdex 200 Increase 10/300 GL column (Cytiva) equilibrated in 25 mM HEPES pH 7.5, 150 mM NaCl, 1 mM MgCl2 and 1 mM TCEP. A detailed procedure is described at 10.17504/protocols.io.5qpvorjezv4o/v1.

Strep pulldown assayFIP200NTD–TSF wild type (WT) or mutants were cotransfected with GST–ATG13363–517 and/or ULK1MIT–MBP in 10 ml of HEK293 GnTi− cells. The cells were harvested 48 h after transfection. The pellets were homogenized in 0.5 ml of lysis buffer, protease inhibitors and 1% Triton X-100 and clarified at 40,000g for 15 min. The lysate was incubated with 30 µl of Strep-Tactin Sepharose resin (IBA Lifesciences) at 4 °C for 3 h. The beads were washed four times and then eluted in 50 µl of lysis buffer and 4 mM desthiobiotin. Then, 18 ml of eluent was mixed with lithium dodecylsulfate (LDS)–β-mercaptoethanol (BME) buffer, heated at 60 °C for 5 min and subjected to SDS–PAGE. The gel was then stained with Coomassie brilliant blue G250. For loading input control, 0.3% lysate per sample was mixed with 1× LDS sample buffer (Life Technologies) containing 100 mM BME, boiled at 95 °C for 5 min and then analyzed with 4–12% Bis–Tris gels (Life Technologies). Gels were electrotransferred to polyvinyldifluoride (PVDF) membranes and immunoblotted with various antibodies. FIP200-NTD–TSF lysate input was visualized using horseradish-peroxidase-conjugated anti-Flag antibody (Abcam, ab49763) and SuperSignal West Femto maximum sensitivity substrate enhanced chemiluminescence kit (Thermo Fisher Scientific, 34095). GST–ATG13c lysate input was analyzed using goat anti-GST antibody (Cytiva, 27457701) and donkey anti-goat IgG (Cy5) (Abcam, ab6566). ULKc–MBP lysate was analyzed using rabbit anti-MBP antibody (Thermo Fisher Scientific, PA1-989) and Alexa Fluor 488 goat anti-rabbit IgG (Invitrogen, A11034). The chemiluminescence or fluorescence signals were detected using a ChemiDocMP Imaging system (Biorad). A detailed procedure is described at 10.17504/protocols.io.3byl4jmpolo5/v1.

Strep pulldown assay

FIP200NTD–TSF wild type (WT) or mutants were cotransfected with GST–ATG13363–517 and/or ULK1MIT–MBP in 10 ml of HEK293 GnTi− cells. The cells were harvested 48 h after transfection. The pellets were homogenized in 0.5 ml of lysis buffer, protease inhibitors and 1% Triton X-100 and clarified at 40,000g for 15 min. The lysate was incubated with 30 µl of Strep-Tactin Sepharose resin (IBA Lifesciences) at 4 °C for 3 h. The beads were washed four times and then eluted in 50 µl of lysis buffer and 4 mM desthiobiotin. Then, 18 ml of eluent was mixed with lithium dodecylsulfate (LDS)–β-mercaptoethanol (BME) buffer, heated at 60 °C for 5 min and subjected to SDS–PAGE. The gel was then stained with Coomassie brilliant blue G250. For loading input control, 0.3% lysate per sample was mixed with 1× LDS sample buffer (Life Technologies) containing 100 mM BME, boiled at 95 °C for 5 min and then analyzed with 4–12% Bis–Tris gels (Life Technologies). Gels were electrotransferred to polyvinyldifluoride (PVDF) membranes and immunoblotted with various antibodies. FIP200-NTD–TSF lysate input was visualized using horseradish-peroxidase-conjugated anti-Flag antibody (Abcam, ab49763) and SuperSignal West Femto maximum sensitivity substrate enhanced chemiluminescence kit (Thermo Fisher Scientific, 34095). GST–ATG13c lysate input was analyzed using goat anti-GST antibody (Cytiva, 27457701) and donkey anti-goat IgG (Cy5) (Abcam, ab6566). ULKc–MBP lysate was analyzed using rabbit anti-MBP antibody (Thermo Fisher Scientific, PA1-989) and Alexa Fluor 488 goat anti-rabbit IgG (Invitrogen, A11034). The chemiluminescence or fluorescence signals were detected using a ChemiDocMP Imaging system (Biorad). A detailed procedure is described at 10.17504/protocols.io.3byl4jmpolo5/v1.

Microscopy-based GSH bead protein–protein interaction assayA mixture of 1 µM purified GST-tagged protein and 500 nM purified fluorescent protein in a total volume of 70 µl was incubated with 9 µl of preblocked glutathione Sepharose beads (Cytiva) in a reaction buffer containing 25 mM HEPES at pH 7.5, 150 mM NaCl, 1 mM MgCl2 and 1 mM TCEP. After incubation at room temperature for 30 min, samples were mixed with an additional 100 µl of reaction buffer and then transferred to the observation chamber for imaging. Images were acquired on a Nikon A1 confocal microscope with a Nikon Plan APO VC ×20 0.75 numerical aperture ultraviolet microscope objective. Three biological replicates were performed for each experimental condition. A detailed procedure is described at 10.17504/protocols.io.4r3l27xdxg1y/v1.

Microscopy-based GSH bead protein–protein interaction assay

A mixture of 1 µM purified GST-tagged protein and 500 nM purified fluorescent protein in a total volume of 70 µl was incubated with 9 µl of preblocked glutathione Sepharose beads (Cytiva) in a reaction buffer containing 25 mM HEPES at pH 7.5, 150 mM NaCl, 1 mM MgCl2 and 1 mM TCEP. After incubation at room temperature for 30 min, samples were mixed with an additional 100 µl of reaction buffer and then transferred to the observation chamber for imaging. Images were acquired on a Nikon A1 confocal microscope with a Nikon Plan APO VC ×20 0.75 numerical aperture ultraviolet microscope objective. Three biological replicates were performed for each experimental condition. A detailed procedure is described at 10.17504/protocols.io.4r3l27xdxg1y/v1.

SEC–MALSThe purified FIP200NTD:ATG13–ULK1MIT complexes were concentrated to 4–5 mg ml−1. SEC–MALS experiments were performed using an Agilent 1200 high-performance liquid chromatography system (Agilent Technologies), coupled to a Wyatt DAWN HELEOS-II MALS instrument and a Wyatt Optilab rEX differential refractometer (Wyatt). For chromatographic separation, a WTC-050S5 size-exclusion column (Wyatt) with a 40-μl sample loop was used at a flow rate of 0.3 ml min−1 in the buffer of 25 mM HEPES pH 7.5, 200 mM NaCl, 1 mM MgCl2 and 2 mM TCEP. The outputs were analyzed using ASTRA V software (Wyatt). MALS signals, combined with the protein concentration determined by refractive index, were used to calculate the molecular mass of the complex. A detailed procedure is described at 10.17504/protocols.io.j8nlkom4xv5r/v1.

The purified FIP200NTD:ATG13–ULK1MIT complexes were concentrated to 4–5 mg ml−1. SEC–MALS experiments were performed using an Agilent 1200 high-performance liquid chromatography system (Agilent Technologies), coupled to a Wyatt DAWN HELEOS-II MALS instrument and a Wyatt Optilab rEX differential refractometer (Wyatt). For chromatographic separation, a WTC-050S5 size-exclusion column (Wyatt) with a 40-μl sample loop was used at a flow rate of 0.3 ml min−1 in the buffer of 25 mM HEPES pH 7.5, 200 mM NaCl, 1 mM MgCl2 and 2 mM TCEP. The outputs were analyzed using ASTRA V software (Wyatt). MALS signals, combined with the protein concentration determined by refractive index, were used to calculate the molecular mass of the complex. A detailed procedure is described at 10.17504/protocols.io.j8nlkom4xv5r/v1.

Sample preparation of FIP200:PI3KC3-C1 complex for cryo-EMFIP200NTD–MBP:ATG13363–517–ULK1MIT–TSF (final 5 µM) was mixed with PI3KC3-C1–TSF complex at a 1.5:1 molar ratio in a total volume of 200 µl, rocking at 4 °C overnight. On the next day, the sample was incubated with 50 µl of Strep-Tactin Sepharose resin (IBA Lifesciences) at 4 °C for 3 h. After one wash, the beads were eluted in 50 µl of 10 mM d-desthiobiotin with the buffer of 25 mM HEPES pH 7.5, 200 mM NaCl, 1 mM MgCl2 and 10 mM TCEP. A detailed procedure is described at 10.17504/protocols.io.5qpvorjezv4o/v1.

Sample preparation of FIP200:PI3KC3-C1 complex for cryo-EM

FIP200NTD–MBP:ATG13363–517–ULK1MIT–TSF (final 5 µM) was mixed with PI3KC3-C1–TSF complex at a 1.5:1 molar ratio in a total volume of 200 µl, rocking at 4 °C overnight. On the next day, the sample was incubated with 50 µl of Strep-Tactin Sepharose resin (IBA Lifesciences) at 4 °C for 3 h. After one wash, the beads were eluted in 50 µl of 10 mM d-desthiobiotin with the buffer of 25 mM HEPES pH 7.5, 200 mM NaCl, 1 mM MgCl2 and 10 mM TCEP. A detailed procedure is described at 10.17504/protocols.io.5qpvorjezv4o/v1.

Sample vitrification and cryo-EM data acquisitionFor cryo-EM sample preparation, 3 μl of protein solution was applied onto a grid freshly glow-discharged in PELCO easiGlow system (Ted Pella). In-house graphene grids were prepared from Trivial Transfer graphene sheets (ACS Material) and Quantifoil R2/1 300-mesh gold (EM Sciences) following a protocol introduced by Ahn et al.53. The graphene grids were used for the ULK1C (2:1:1) core sample and holey carbon grids (Quantifoil R1.2/1.3 or R2/1 300-mesh; EM Sciences) were sufficient for the other samples. The samples were vitrified with a Vitrobot cryo-plunger (Thermo Fisher Scientific) in plunging conditions optimized previously. Then, 0.05% (w/v) of n-octyl-β-d-glucopyranoside was added in the sample solution as a surfactant before vitrification.The datasets of the ULK1C (2:1:1) core, the ULK1C:PI3KC3-C1 mixture and the ULK1C:PI3KC3-C1 pulldown were recorded at a 300 kV Titan Krios microscope (Thermo Fisher Scientific) equipped with X-FEG and an energy filter set to a width of 20 eV. Automated data acquisition was achieved using SerialEM54 on a K3 Summit direct detection camera (Gatan) at a magnification of ×81,000 and a corresponding pixel size of 0.525 Å in super-resolution mode with a defocus range of −0.8 to −2.0 μm. The ULK1C:PI3KC3-C1 pulldown sample was collected on a specimen stage tilted at 0°, 20° and 30°. Image stacks with 50 frames was collected with a total dose of 50 e− per Å2. The dataset of the ULK1C (2:2:2) core was recorded on a 200-kV Talos Arctica microscope (Thermo Fisher Scientific) equipped with the K3 Summit camera in super-resolution correlated double-sampling mode. The magnification and the pixel size were ×36,000 and 0.5575 Å in super-resolution mode, respectively. Other details of the dataset collection are summarized in Table 1. A detailed procedure is described at 10.17504/protocols.io.kqdg39rreg25/v1.Table 1Cryo-EM data collection, refinement and validation statisticsULK1C core(2:1:1) stoichiometryULK1C:PI3KC3-C1 supercomplexULK1C core(2:2:2) stoichiometryin PI3KC3-C1 mixtureULK1C core(2:2:2) stoichiometryATG13450–517 mutant(EMD-40658), (PDB 8SOI)(EMD-45297), (PDB 9C82)(EMD-40715), (PDB 8SQZ)(EMD-40735), (PDB 8SRM)Data collection and processingMagnification×81,000×81,000×81,000×36,000Voltage (kV)300300300200Electron exposure (e− per Å2)50505050Defocus range (μm)−0.8 to −2.0−0.8 to −2.0−0.8 to −2.0−0.8 to −2.0Pixel size (Å)1.051.051.051.115Symmetry imposedC1C1C1C1Images8,468589 (0° tilt)6,973 (20° tilt)3,740 (30° tilt)5,3832,286Initial particle images (no.)2,156,5164,473,551502,985968,884Final particle images (no.)267,71921,93797,987149,409Map resolution, global (Å)4.28.035.284.46 FSC threshold0.1430.1430.1430.143Map resolution range (Å)3.48–63.837.15–23.074.96–81.033.77–70.64RefinementInitial model used (PDB code)AlphaFold2 modelAlphaFold2 model8SOI8SOIModel resolution (Å)3.78.76.86.5 FSC threshold0.50.50.50.5Model resolution range (Å)3.48–22.497.15–16.904.98–16.473.78–11.37Map sharpening B factor (A2)143.2752.0431.6155.4Model composition Nonhydrogen atoms10,24311,1727,4385,773 Protein residues1,2762,2541,4981,163 Ligands0000B factors (Å2) Protein126.431,099.03319.79208.69 Ligands----Root-mean-square deviations Bond lengths (Å)0.0030.0020.0040.002 Bond angles (°)0.5980.6070.6330.405Validation MolProbity score1.72.481.951.51 Clashscore9.5230.518.074.44 Poor rotamers (%)1.5---Ramachandran plot Favored (%)96.7591.1791.4195.87 Allowed (%)3.178.838.594.13 Disallowed (%)0.008000PDB, Protein Data Bank; FSC, Fourier shell correlation.

Sample vitrification and cryo-EM data acquisition

For cryo-EM sample preparation, 3 μl of protein solution was applied onto a grid freshly glow-discharged in PELCO easiGlow system (Ted Pella). In-house graphene grids were prepared from Trivial Transfer graphene sheets (ACS Material) and Quantifoil R2/1 300-mesh gold (EM Sciences) following a protocol introduced by Ahn et al.53. The graphene grids were used for the ULK1C (2:1:1) core sample and holey carbon grids (Quantifoil R1.2/1.3 or R2/1 300-mesh; EM Sciences) were sufficient for the other samples. The samples were vitrified with a Vitrobot cryo-plunger (Thermo Fisher Scientific) in plunging conditions optimized previously. Then, 0.05% (w/v) of n-octyl-β-d-glucopyranoside was added in the sample solution as a surfactant before vitrification.

The datasets of the ULK1C (2:1:1) core, the ULK1C:PI3KC3-C1 mixture and the ULK1C:PI3KC3-C1 pulldown were recorded at a 300 kV Titan Krios microscope (Thermo Fisher Scientific) equipped with X-FEG and an energy filter set to a width of 20 eV. Automated data acquisition was achieved using SerialEM54 on a K3 Summit direct detection camera (Gatan) at a magnification of ×81,000 and a corresponding pixel size of 0.525 Å in super-resolution mode with a defocus range of −0.8 to −2.0 μm. The ULK1C:PI3KC3-C1 pulldown sample was collected on a specimen stage tilted at 0°, 20° and 30°. Image stacks with 50 frames was collected with a total dose of 50 e− per Å2. The dataset of the ULK1C (2:2:2) core was recorded on a 200-kV Talos Arctica microscope (Thermo Fisher Scientific) equipped with the K3 Summit camera in super-resolution correlated double-sampling mode. The magnification and the pixel size were ×36,000 and 0.5575 Å in super-resolution mode, respectively. Other details of the dataset collection are summarized in Table 1. A detailed procedure is described at 10.17504/protocols.io.kqdg39rreg25/v1.Table 1Cryo-EM data collection, refinement and validation statisticsULK1C core(2:1:1) stoichiometryULK1C:PI3KC3-C1 supercomplexULK1C core(2:2:2) stoichiometryin PI3KC3-C1 mixtureULK1C core(2:2:2) stoichiometryATG13450–517 mutant(EMD-40658), (PDB 8SOI)(EMD-45297), (PDB 9C82)(EMD-40715), (PDB 8SQZ)(EMD-40735), (PDB 8SRM)Data collection and processingMagnification×81,000×81,000×81,000×36,000Voltage (kV)300300300200Electron exposure (e− per Å2)50505050Defocus range (μm)−0.8 to −2.0−0.8 to −2.0−0.8 to −2.0−0.8 to −2.0Pixel size (Å)1.051.051.051.115Symmetry imposedC1C1C1C1Images8,468589 (0° tilt)6,973 (20° tilt)3,740 (30° tilt)5,3832,286Initial particle images (no.)2,156,5164,473,551502,985968,884Final particle images (no.)267,71921,93797,987149,409Map resolution, global (Å)4.28.035.284.46 FSC threshold0.1430.1430.1430.143Map resolution range (Å)3.48–63.837.15–23.074.96–81.033.77–70.64RefinementInitial model used (PDB code)AlphaFold2 modelAlphaFold2 model8SOI8SOIModel resolution (Å)3.78.76.86.5 FSC threshold0.50.50.50.5Model resolution range (Å)3.48–22.497.15–16.904.98–16.473.78–11.37Map sharpening B factor (A2)143.2752.0431.6155.4Model composition Nonhydrogen atoms10,24311,1727,4385,773 Protein residues1,2762,2541,4981,163 Ligands0000B factors (Å2) Protein126.431,099.03319.79208.69 Ligands----Root-mean-square deviations Bond lengths (Å)0.0030.0020.0040.002 Bond angles (°)0.5980.6070.6330.405Validation MolProbity score1.72.481.951.51 Clashscore9.5230.518.074.44 Poor rotamers (%)1.5---Ramachandran plot Favored (%)96.7591.1791.4195.87 Allowed (%)3.178.838.594.13 Disallowed (%)0.008000PDB, Protein Data Bank; FSC, Fourier shell correlation.

Cryo-EM data collection, refinement and validation statistics

589 (0° tilt)

6,973 (20° tilt)

3,740 (30° tilt)

PDB, Protein Data Bank; FSC, Fourier shell correlation.

Image processing and 3D reconstructionThe datasets were processed by following the workflow in cryoSPARC55. In brief, the super-resolution video stacks were motion-corrected and binned 2× by Fourier cropping using Patch motion correction. Contrast transfer function (CTF) determination was performed using Patch CTF estimation, followed by manual removal of the outlier micrographs on the basis of the estimated defocus and resolution value. Single particles were automatically picked by Topaz56 using a manually trained model and extracted with a window size 1.5 times larger than the target particle before further binning 2× to 4× to facilitate subsequent processing. A 2D classification was then used to remove obvious junk particles. The initial models were obtained using ab initio reconstruction. In case the reconstruction job failed to give a healthy initial model, the classes displaying high-resolution features in the 2D classification step were selected and used for ab initio reconstruction. Further classification was performed at the 3D level by multiple rounds of heterogeneous refinement until a clean substack was obtained. The particles were re-extracted with the refined coordinates on micrographs at the original 2× bin pixel size and used for homogeneous refinement for multiple rounds until the final resolution converged. To further improve the quality of the map, local refinement was applied to the datasets of the ULK1C (2:1:1) core and the ULK1C:PI3KC3-C1 pulldown sample. Masking areas were decided by 3D Variability or 3D Flex and the masks were created by UCSF ChimeraX57 and Volume Tools. Each local map was aligned to the consensus map and composed using EMAN2 (ref. 58). The composed maps were then used for model building. The details of data processing are summarized in Table 1. A detailed procedure is described in 10.17504/protocols.io.x54v9d99mg3e/v1.

Image processing and 3D reconstruction

The datasets were processed by following the workflow in cryoSPARC55. In brief, the super-resolution video stacks were motion-corrected and binned 2× by Fourier cropping using Patch motion correction. Contrast transfer function (CTF) determination was performed using Patch CTF estimation, followed by manual removal of the outlier micrographs on the basis of the estimated defocus and resolution value. Single particles were automatically picked by Topaz56 using a manually trained model and extracted with a window size 1.5 times larger than the target particle before further binning 2× to 4× to facilitate subsequent processing. A 2D classification was then used to remove obvious junk particles. The initial models were obtained using ab initio reconstruction. In case the reconstruction job failed to give a healthy initial model, the classes displaying high-resolution features in the 2D classification step were selected and used for ab initio reconstruction. Further classification was performed at the 3D level by multiple rounds of heterogeneous refinement until a clean substack was obtained. The particles were re-extracted with the refined coordinates on micrographs at the original 2× bin pixel size and used for homogeneous refinement for multiple rounds until the final resolution converged. To further improve the quality of the map, local refinement was applied to the datasets of the ULK1C (2:1:1) core and the ULK1C:PI3KC3-C1 pulldown sample. Masking areas were decided by 3D Variability or 3D Flex and the masks were created by UCSF ChimeraX57 and Volume Tools. Each local map was aligned to the consensus map and composed using EMAN2 (ref. 58). The composed maps were then used for model building. The details of data processing are summarized in Table 1. A detailed procedure is described in 10.17504/protocols.io.x54v9d99mg3e/v1.

Model building, validation and visualizationThe in silico models of the ULK1C (2:1:1) core and the full-length PI3KC3-C1 were generated by AlphaFold2 prediction29. The resolution of the observed maps enabled amino acid sequence assignment. The primary and secondary structure of the predicted models agreed well with the EM maps. A flexible model fitting using the real-time molecular dynamics simulation-based program ISOLDE59 implemented in the visualization software UCSF ChimeraX was performed, followed by iterative refinement by using the model-editing software Coot60 manually and real-space refinement in PHENIX61 automatically.The two ULK1C (2:2:2) core models were created by aligning two copies of FIP200NTD:ATG13363–517:ULK1MIT subtrimeric complex to the FIP200 dimerization domain, followed by flexible model fitting with using ISOLDE in ChimeraX. The side chains were removed because of the moderate resolution of these maps. The ULK1C:PI3KC3-C1 supercomplex coordinates were generated by fitting of the individual structures of the ULK1C (2:1:1) and the PI3KC3-C1 to the cryo-EM map. The side chains were removed. The quality of the models was validated by using the validation tools in PHENIX and the online validation service provided by wwPDB62,63. The details of the model quality assessment are summarized in Table 1. All figures and videos were made using UCSF ChimeraX. The detailed procedure is described at 10.17504/protocols.io.j8nlkw77wl5r/v1.

Model building, validation and visualization

The in silico models of the ULK1C (2:1:1) core and the full-length PI3KC3-C1 were generated by AlphaFold2 prediction29. The resolution of the observed maps enabled amino acid sequence assignment. The primary and secondary structure of the predicted models agreed well with the EM maps. A flexible model fitting using the real-time molecular dynamics simulation-based program ISOLDE59 implemented in the visualization software UCSF ChimeraX was performed, followed by iterative refinement by using the model-editing software Coot60 manually and real-space refinement in PHENIX61 automatically.

The two ULK1C (2:2:2) core models were created by aligning two copies of FIP200NTD:ATG13363–517:ULK1MIT subtrimeric complex to the FIP200 dimerization domain, followed by flexible model fitting with using ISOLDE in ChimeraX. The side chains were removed because of the moderate resolution of these maps. The ULK1C:PI3KC3-C1 supercomplex coordinates were generated by fitting of the individual structures of the ULK1C (2:1:1) and the PI3KC3-C1 to the cryo-EM map. The side chains were removed. The quality of the models was validated by using the validation tools in PHENIX and the online validation service provided by wwPDB62,63. The details of the model quality assessment are summarized in Table 1. All figures and videos were made using UCSF ChimeraX. The detailed procedure is described at 10.17504/protocols.io.j8nlkw77wl5r/v1.

AlphaFold2 model predictionModels of the FIP200NTD:ATG13MIT:ULKMIM core domain were generated using the ColabFold implementation of AlphaFold2 (refs. 29,64–67). The default MMseqs2 (ref. 68) pipeline was used to generate and pair multiple-sequence alignments for each protein complex predicted. We predicted the following complexes: dimeric FIP200 (1–640), ULK1 (828–1050) and ATG13 (363–517); dimeric FIP200 (1–640) and ATG13 (363–450); dimeric FIP200 (1–640), ATG13 (450–517) and ULK (828–1050); monomeric FIP200 (1–640) and ATG13 (363–450); monomeric FIP200 (1–640), ATG13 (450–517) and ULK (828–1050). In each case, the models were assessed on the basis of the global predicted template modeling score (pTM) and the interfacial PTM (iPTM) score, with iPTM > 0.5 meriting further inspection. The predicted contacts involving ATG13, FIP200 and ULK1 were manually inspected in UCSF ChimeraX and the local predicted local distance difference test (pLDDT) score of the contacting residues was used in conjunction with an assessment of the physiological environment of the interfaces to judge the quality of the predicted residue contacts. The highest-scoring local pLDDT interfaces were used for figure making and interpretation of HDX data.

AlphaFold2 model prediction

Models of the FIP200NTD:ATG13MIT:ULKMIM core domain were generated using the ColabFold implementation of AlphaFold2 (refs. 29,64–67). The default MMseqs2 (ref. 68) pipeline was used to generate and pair multiple-sequence alignments for each protein complex predicted. We predicted the following complexes: dimeric FIP200 (1–640), ULK1 (828–1050) and ATG13 (363–517); dimeric FIP200 (1–640) and ATG13 (363–450); dimeric FIP200 (1–640), ATG13 (450–517) and ULK (828–1050); monomeric FIP200 (1–640) and ATG13 (363–450); monomeric FIP200 (1–640), ATG13 (450–517) and ULK (828–1050). In each case, the models were assessed on the basis of the global predicted template modeling score (pTM) and the interfacial PTM (iPTM) score, with iPTM > 0.5 meriting further inspection. The predicted contacts involving ATG13, FIP200 and ULK1 were manually inspected in UCSF ChimeraX and the local predicted local distance difference test (pLDDT) score of the contacting residues was used in conjunction with an assessment of the physiological environment of the interfaces to judge the quality of the predicted residue contacts. The highest-scoring local pLDDT interfaces were used for figure making and interpretation of HDX data.

Generation of KO lines using CRISPR–Cas9CRISPR guide RNAs (gRNAs) targeting ATG13 (5′-CCGCGAGTTTGATGCCTTTG-3′ (ref. 69) and 5′-TTGCTTCATGTGTAACCTCTGGG-3′) were cloned into a BbsI-linearized pSpCas9(BB)-2A-GFP vector70 (a gift from F. Zhang; Addgene, plasmid 48138)70 using a Gibson cloning kit (New England Biolabs). gRNA-containing constructs were then sequence-verified and transfected into WT HeLa cells with X-tremeGENE 9 (Roche) overnight and GFP-positive single cells were sorted by fluorescence-activated cell sorting into 96-well plates. Single-cell colonies were left to grow and screened by immunoblotting for loss of the targeted gene product. This procedure is described in detail at 10.17504/protocols.io.j8nlkkzo6l5r/v1.

Generation of KO lines using CRISPR–Cas9

CRISPR guide RNAs (gRNAs) targeting ATG13 (5′-CCGCGAGTTTGATGCCTTTG-3′ (ref. 69) and 5′-TTGCTTCATGTGTAACCTCTGGG-3′) were cloned into a BbsI-linearized pSpCas9(BB)-2A-GFP vector70 (a gift from F. Zhang; Addgene, plasmid 48138)70 using a Gibson cloning kit (New England Biolabs). gRNA-containing constructs were then sequence-verified and transfected into WT HeLa cells with X-tremeGENE 9 (Roche) overnight and GFP-positive single cells were sorted by fluorescence-activated cell sorting into 96-well plates. Single-cell colonies were left to grow and screened by immunoblotting for loss of the targeted gene product. This procedure is described in detail at 10.17504/protocols.io.j8nlkkzo6l5r/v1.

Cloning and generation of stable cell linespBMN-HA-WT ATG13 was previously described (Addgene, 186223)71. Complementary DNAs encoding ATG13 mutants (site 2, F394D;M396D;F397D;E403K; site 4, F449D;A452D;F453D;S454A; site M, L495D;L499D; sites 2, 4 and M, all substitutions) were synthesized by Integrated DNA Technologies and cloned into pBMN-HA-C1 (Addgene, 188645) using a Gibson cloning kit (New England Biolabs). Generation of stable cell lines was previously described72 and the detailed procedure is available at 10.17504/protocols.io.dm6gpjzm8gzp/v1. pMRX-IP-HaloTag7-LC3 (Addgene, 184899) and HA-tagged ATG13 variants were stably expressed in ATG13-KO cells using retroviral transduction.

Cloning and generation of stable cell lines

pBMN-HA-WT ATG13 was previously described (Addgene, 186223)71. Complementary DNAs encoding ATG13 mutants (site 2, F394D;M396D;F397D;E403K; site 4, F449D;A452D;F453D;S454A; site M, L495D;L499D; sites 2, 4 and M, all substitutions) were synthesized by Integrated DNA Technologies and cloned into pBMN-HA-C1 (Addgene, 188645) using a Gibson cloning kit (New England Biolabs). Generation of stable cell lines was previously described72 and the detailed procedure is available at 10.17504/protocols.io.dm6gpjzm8gzp/v1. pMRX-IP-HaloTag7-LC3 (Addgene, 184899) and HA-tagged ATG13 variants were stably expressed in ATG13-KO cells using retroviral transduction.

Halo assay to quantify starvation induced autophagyWe measured autophagic flux using a previously described method72,73. In brief, 350,000–400,000 cells were seeded the day before treatment in six-well plates. Cells were treated with 50 nM TMR ligand (Promega) in growth medium (DMEM with 10% FBS, 4.5 g L−1 glucose (Sigma), 1× GlutaMAXTM (Thermo Fisher Scientific), 1× nonessential amino acids (Thermo Fisher Scientific) and 25 mM HEPES. After 15 min, cells were washed three times with 1× Dulbecco’s PBS and treated with Earle’s balanced salt solution (EBSS; Gibco) for the indicated time periods. Following treatment, samples were analyzed by immunoblotting as described below. This procedure is described at 10.17504/protocols.io.3byl4qexzvo5/v1.

Halo assay to quantify starvation induced autophagy

We measured autophagic flux using a previously described method72,73. In brief, 350,000–400,000 cells were seeded the day before treatment in six-well plates. Cells were treated with 50 nM TMR ligand (Promega) in growth medium (DMEM with 10% FBS, 4.5 g L−1 glucose (Sigma), 1× GlutaMAXTM (Thermo Fisher Scientific), 1× nonessential amino acids (Thermo Fisher Scientific) and 25 mM HEPES. After 15 min, cells were washed three times with 1× Dulbecco’s PBS and treated with Earle’s balanced salt solution (EBSS; Gibco) for the indicated time periods. Following treatment, samples were analyzed by immunoblotting as described below. This procedure is described at 10.17504/protocols.io.3byl4qexzvo5/v1.

ImmunoblottingFollowing feeding and treatment periods, the cells were washed with ice-cold 1× PBS, harvested using cell scrapers and lysed in lysis buffer (1× LDS sample buffer (Life Technologies) containing 100 mM DTT (Sigma)). Samples were boiled at 99 °C with shaking for 7 min and 25 μg (for Halo assays) and 70 μg (for the remaining blots) of protein per sample was analyzed with 4–12% Bis–Tris gels (Life Technologies) according to the manufacturer’s instructions. Gels were electrotransferred to PVDF membranes and immunoblotted with the indicated antibodies. For quantification, band intensities were measured using ImageLab 5.2.1 (BioRad). Statistical significance was calculated from at least three independent experiments using a two-way analysis of variance (ANOVA). Error bars are the means ± s.d. The procedure for immunoblotting is described in detail at 10.17504/protocols.io.dm6gpjzm8gzp/v1.

Immunoblotting

Following feeding and treatment periods, the cells were washed with ice-cold 1× PBS, harvested using cell scrapers and lysed in lysis buffer (1× LDS sample buffer (Life Technologies) containing 100 mM DTT (Sigma)). Samples were boiled at 99 °C with shaking for 7 min and 25 μg (for Halo assays) and 70 μg (for the remaining blots) of protein per sample was analyzed with 4–12% Bis–Tris gels (Life Technologies) according to the manufacturer’s instructions. Gels were electrotransferred to PVDF membranes and immunoblotted with the indicated antibodies. For quantification, band intensities were measured using ImageLab 5.2.1 (BioRad). Statistical significance was calculated from at least three independent experiments using a two-way analysis of variance (ANOVA). Error bars are the means ± s.d. The procedure for immunoblotting is described in detail at 10.17504/protocols.io.dm6gpjzm8gzp/v1.

Mass photometryHigh-precision coverslips (Azer Scientific) were cleaned by alternating between isopropanol and water three times in a bath sonicator, each 3 min, followed by air drying. The gasket was similarly cleaned with isopropanol and water three times without sonication and then air-dried. A volume of 19 µl of mass photometry buffer (30 mM HEPES pH 7.4, 5 mM MgSO4 and 1 mM EGTA) was added to the well for autofocus. The protein sample was then diluted in the mass photometry buffer in the wells to a concentration of 5–20 nM before data collection. If crosslinking was performed, 1 μl of 0.1% (v/v) glutaraldehyde in mass photometry buffer was added to 9 μl of protein sample and incubated for 10 min. The reaction was then quenched by adding 10 μl of 1 M Tris-HCl pH 7.4. All samples were diluted to 10 nM for the measurement of mass photometry. Protein contrast counts were acquired using a Refeyn TwoMP mass photometer, with three technical replicates. Mass calibration was performed using a standard mix of conalbumin, aldolase and thyroglobulin. Mass photometry profiles were analyzed and fitted to multiple Gaussian peaks, with the mean, s.d. and percentages calculated using DiscoverMP software (Refeyn). This procedure is described at 10.17504/protocols.io.kqdg3keq7v25/v1.

Mass photometry

High-precision coverslips (Azer Scientific) were cleaned by alternating between isopropanol and water three times in a bath sonicator, each 3 min, followed by air drying. The gasket was similarly cleaned with isopropanol and water three times without sonication and then air-dried. A volume of 19 µl of mass photometry buffer (30 mM HEPES pH 7.4, 5 mM MgSO4 and 1 mM EGTA) was added to the well for autofocus. The protein sample was then diluted in the mass photometry buffer in the wells to a concentration of 5–20 nM before data collection. If crosslinking was performed, 1 μl of 0.1% (v/v) glutaraldehyde in mass photometry buffer was added to 9 μl of protein sample and incubated for 10 min. The reaction was then quenched by adding 10 μl of 1 M Tris-HCl pH 7.4. All samples were diluted to 10 nM for the measurement of mass photometry. Protein contrast counts were acquired using a Refeyn TwoMP mass photometer, with three technical replicates. Mass calibration was performed using a standard mix of conalbumin, aldolase and thyroglobulin. Mass photometry profiles were analyzed and fitted to multiple Gaussian peaks, with the mean, s.d. and percentages calculated using DiscoverMP software (Refeyn). This procedure is described at 10.17504/protocols.io.kqdg3keq7v25/v1.

Reporting summaryFurther information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

Reporting summary

Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

Online content

Supplementary information
Reporting Summary
Peer Review File
Supplementary Table 1Key resources.

Supplementary information

Reporting Summary
Peer Review File
Supplementary Table 1Key resources.

Reporting Summary

Peer Review File

Key resources.

Source data
Source Data Fig. 1Unprocessed Coomassie blue-stained gels and western blots.
Source Data Fig. 2Unprocessed Coomassie blue-stained gels and western blots.
Source Data Fig. 3Unprocessed Coomassie blue-stained gel.
Source Data Fig. 2Statistical source data.
Source Data Fig. 6Statistical source data.
Source Data Extended Data Fig. 4Unprocessed western blots.

Source data

Source Data Fig. 1Unprocessed Coomassie blue-stained gels and western blots.
Source Data Fig. 2Unprocessed Coomassie blue-stained gels and western blots.
Source Data Fig. 3Unprocessed Coomassie blue-stained gel.
Source Data Fig. 2Statistical source data.
Source Data Fig. 6Statistical source data.
Source Data Extended Data Fig. 4Unprocessed western blots.

Unprocessed Coomassie blue-stained gels and western blots.

Unprocessed Coomassie blue-stained gels and western blots.

Unprocessed Coomassie blue-stained gel.

Statistical source data.

Statistical source data.

Unprocessed western blots.
