---
pmid: '21148293'
title: The endoplasmic reticulum-associated Hsp40 DNAJB12 and Hsc70 cooperate to facilitate
  RMA1 E3-dependent degradation of nascent CFTRDeltaF508.
authors:
- Grove DE
- Fan CY
- Ren HY
- Cyr DM
journal: Mol Biol Cell
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3031462
doi: 10.1091/mbc.E10-09-0760
---

# The endoplasmic reticulum-associated Hsp40 DNAJB12 and Hsc70 cooperate to facilitate RMA1 E3-dependent degradation of nascent CFTRDeltaF508.
**Authors:** Grove DE, Fan CY, Ren HY, Cyr DM
**Journal:** Mol Biol Cell (2011)
**DOI:** [10.1091/mbc.E10-09-0760](https://doi.org/10.1091/mbc.E10-09-0760)
**PMC:** [PMC3031462](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031462/)

## Abstract

1. Mol Biol Cell. 2011 Feb 1;22(3):301-14. doi: 10.1091/mbc.E10-09-0760. Epub
2010  Dec 9.

The endoplasmic reticulum-associated Hsp40 DNAJB12 and Hsc70 cooperate to 
facilitate RMA1 E3-dependent degradation of nascent CFTRDeltaF508.

Grove DE(1), Fan CY, Ren HY, Cyr DM.

Author information:
(1)Department of Cell and Developmental Biology, University of North Carolina at 
Chapel Hill, Chapel Hill, NC 27599, USA.

Relative contributions of folding kinetics versus protein quality control (QC) 
activity in the partitioning of non-native proteins between life and death are 
not clear. Cystic fibrosis transmembrane conductance regulator (CFTR) biogenesis 
serves as an excellent model to study this question because folding of nascent 
CFTR is inefficient and deletion of F508 causes accumulation of CFTRΔF508 in a 
kinetically trapped, but foldable state. Herein, a novel endoplasmic reticulum 
(ER)-associated Hsp40, DNAJB12 (JB12) is demonstrated to play a role in control 
of CFTR folding efficiency. JB12 cooperates with cytosolic Hsc70 and the 
ubiquitin ligase RMA1 to target CFTR and CFTRΔF508 for degradation. Modest 
elevation of JB12 decreased nascent CFTR and CFTRΔF508 accumulation while 
increasing association of Hsc70 with ER forms of CFTR and the RMA1 E3 complex. 
Depletion of JB12 increased CFTR folding efficiency up to threefold and 
permitted a pool of CFTRΔF508 to fold and escape the ER. Introduction of the 
V510D misfolding suppressor mutation into CFTRΔF508 modestly increased folding 
efficiency, whereas combined inactivation of JB12 and suppression of intrinsic 
folding defects permitted CFTRΔF508 to fold at 50% of wild-type efficiency. 
Therapeutic correction of CFTRΔF508 misfolding in cystic fibrosis patients may 
require repair of defective folding kinetics and suppression of ER QC factors, 
such as JB12.

DOI: 10.1091/mbc.E10-09-0760
PMCID: PMC3031462
PMID: 21148293 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONThe fatal lung disease cystic fibrosis (CF) is a loss-of-protein-function disorder caused by misfolding and premature degradation of the cystic fibrosis transmembrane conductance regulator (CFTR). CFTR is a Cl- channel that controls hydration of epithelial cell surfaces in airways and glands (Rowe et al., 2005). Most CF patients inherit the CFTRΔF508 mutant allele the protein product of which exhibits subtle folding defects that lead almost all nascent forms to be degraded (Cyr, 2005). Patients who exhibit partial CFTR function have mild CF symptoms, so restoration of CFTRΔF508 activity to modest levels is a therapeutic goal.CFTR contains 1480 amino acid residues and has two membrane-spanning domains (MSDs), two nucleotide-binding domains (NBD), and a regulatory domain (Riordan et al., 1989). It takes ∼10 min to synthesize one CFTR molecule, and folding requires co- and post-translational assembly events (Du and Lukacs, 2009). CFTR folding involves formation of an intricate network of interdomain contacts between the N- and C-terminal membrane and cytosolic subdomains (Serohijos et al., 2008a), and folding progresses through an ensemble of intermediates (Lukacs et al., 1994; Du et al., 2005). Consequently, CFTR folding is inefficient, with the majority of the newly synthesized protein partitioned toward a degradation pathway (Cyr, 2005).F508 is located in NBD1 and is not essential for Cl- conductance (Rowe et al., 2005), but its deletion leads pools of nascent CFTRΔF508 to accumulate in a foldable, but kinetically trapped, conformation (Denning et al., 1992; Younger et al., 2004). CFTRΔF508 misfolding appears to involve subtle defects in NBD1 folding (Thibodeau et al., 2005; Serohijos et al., 2008a) that cause cotranslational misassembly of an intermediate that is degraded rapidly and may not accumulate (Younger et al., 2006). Folding of the small pool of nascent CFTRΔF508 that is spared initial degradation is arrested because of defective contact formation between NBD1 and regions that include intracellular loops exposed by MSD2 (Serohijos et al., 2008a) and misfolding of NBD2 (Du et al., 2005). Consequently, ∼99% of CFTRΔF508 is degraded prematurely by the ubiquitin–proteasome system (Jensen et al., 1995; Ward et al., 1995).CFTR folding is assisted by several different molecular chaperones. The endoplasmic reticulum (ER)-associated Hsp40 Hdj-2 (DNAJA1) helps attract cytosolic Hsc70 to the ER membrane surface to facilitate cotranslational folding and assembly of NBD1 (Strickland et al., 1997; Meacham et al., 1999). The ER luminal chaperone cal­nexin appears to act after Hdj-2 to facilitate association of regions within the CFTR’s membrane-spanning and cytosolic domains (Pind et al., 1994; Rosser et al., 2008). Glycosylation of CFTR is required for calnexin binding, and this modification appears to be required for stabilization of transmembrane regions in MSD2 in the ER membrane (Glozman et al., 2009). Terminal steps in folding of full-length CFTR are facilitated by Hsp90 and its associated cofactors (Loo et al., 1998). Modulation of the Hsp90 cofactor AHA1 enhances CFTRΔF508 folding and is sufficient to permit small pools of functional CFTRΔF508 to accumulate at the cell surface (Wang et al., 2006).Paradoxically, the selection of nascent forms of CFTR and CFTRΔF508 for proteasomal degradation is also facilitated by molecular chaperones. The cytosolic E3 ubiquitin ligase CHIP interacts with Hsc70 and/or Hsp70 to form a quality control (QC) machine that uses the polypeptide-binding activity of Hsc/Hsp70 to target misfolded CFTR for proteasomal degradation (Meacham et al., 2001). (From here on, Hsc70 and Hsp70 will be treated as the same protein and generally referred to as Hsc70.) In addition, the ER-associated E3 RMA1/RNF5 acts in association with Derlin-1 and the E2 Ubc6e to ubiquitinate CFTR (Younger et al., 2006). Ubiquitinated CFTR is delivered to the proteasome in a pathway that involves Gp78 (Morito et al., 2008), BAP31 (Wang et al., 2008), and p97 (Dalal et al., 2004). How selection of CFTR for degradation by the RMA1 E3 machinery and CHIP E3 complex are synergized is not entirely clear. The RMA1 E3 complex, however, may act coincident with translation to recognize folding defects in CFTR that involve the misfolding and defective assembly of NBD1 into a complex with the R-domain (Younger et al., 2006; Rosser et al., 2008). In contrast, the CHIP E3 may act post-translationally to recognize misfolded regions of CFTR that include NBD2 (Younger et al., 2006).Deletion of F508 renders CFTR highly sensitive to changes in RMA1 activity, which suggests that the RMA1 E3 plays a critical role in selection of CFTRΔF508 for premature degradation (Younger et al., 2006). It is unclear, however, how the RMA1 complex distinguishes between on- and off-pathway forms of nascent CFTR. RMA1 interacts with the transmembrane QC factor Derlin-1, and small interfering RNA (siRNA) knockdown (kd) of Derlin-1 enhances cell surface expression of CFTR (Sun et al., 2006; Younger et al., 2006). Derlin-1 can form complexes with MSD1 of CFTR, and Derlin-1 overexpression promotes ER retention and proteasomal degradation of CFTR (Younger et al., 2006). Therefore it is possible that Derlin-1 is part of a complex that acts as a membrane chaperone to scan the assembly status of CFTR’s membrane regions and targets misassembled forms to RMA1 for ubiquitination. This model seems logical, except that the region in CFTR that is ubiquitinated by RMA1 is cytosolic. Therefore QC factors in addition to Derlin-1 may assist RMA1 in the selection of misfolded CFTR for ubiquitination.Clues to the identity of such a factor come from the analysis of the Hsp40/DnaJ family of Hsc70 cochaperones (Kampinga and Craig, 2010). Hsp40 proteins use a conserved J-domain to regulate Hsc70 ATPase activity and specify client proteins of Hsc70. One mechanism for specification of Hsc70 function is for a specialized Hsp40 to attract Hsc70 to function in a discrete cellular location. Studies in yeast identify a unique Hsp40 subtype that is integrated into the ER membrane and exposes its J-domain to the cytosol (Walsh et al., 2004). This subfamily is represented in yeast by HLJ1 (high copy lethal DnaJ 1), and deletion analysis suggests that HLJ1 functions with cytosolic Hsc70 in ER QC (Youker et al., 2004; Han et al., 2007). RMA1 is not expressed in yeast, so we pondered whether an HLJ1-like protein might act with RMA1 in human cells to assist in the selection of CFTR for degradation.Indeed, HLJ1-like Hsp40s are conserved in the human genome, and herein we report on the characterization of human DNAJB12 (JB12). JB12 is a Type II Hsp40 that contains a J-domain and G/F-like region and is similar to HLJ1 in that it is localized to the ER membrane and exposes its J-domain to the cytosol. JB12 differs from HLJ1, however, in that it contains N- and C-terminal extensions that were added during the course of evolution. Data presented demonstrate that JB12 acts with Hsc70 and RMA1 to facilitate proteasomal degradation of CFTR and CFTRΔF508. JB12 forms complexes that contain RMA1 and Derlin-1, and increasing JB12 levels dramatically increases association of Hsc70 with ER-localized forms of CFTR and RMA1. Depletion of endogenous JB12 results in a significant threefold increase in the folding efficiency of CFTR and permits a pool of CFTRΔF508 to escape the ER. JB12 appears to direct Hsc70 to function with the E3 RMA1 in degradation of nascent CFTRΔF508 and CFTR. As fluctuations in its activity have an impact on the fate of nascent CFTR, JB12 is capable of exerting control over the folding efficiency of nascent polytopic membrane proteins.

INTRODUCTION

The fatal lung disease cystic fibrosis (CF) is a loss-of-protein-function disorder caused by misfolding and premature degradation of the cystic fibrosis transmembrane conductance regulator (CFTR). CFTR is a Cl- channel that controls hydration of epithelial cell surfaces in airways and glands (Rowe et al., 2005). Most CF patients inherit the CFTRΔF508 mutant allele the protein product of which exhibits subtle folding defects that lead almost all nascent forms to be degraded (Cyr, 2005). Patients who exhibit partial CFTR function have mild CF symptoms, so restoration of CFTRΔF508 activity to modest levels is a therapeutic goal.

CFTR contains 1480 amino acid residues and has two membrane-spanning domains (MSDs), two nucleotide-binding domains (NBD), and a regulatory domain (Riordan et al., 1989). It takes ∼10 min to synthesize one CFTR molecule, and folding requires co- and post-translational assembly events (Du and Lukacs, 2009). CFTR folding involves formation of an intricate network of interdomain contacts between the N- and C-terminal membrane and cytosolic subdomains (Serohijos et al., 2008a), and folding progresses through an ensemble of intermediates (Lukacs et al., 1994; Du et al., 2005). Consequently, CFTR folding is inefficient, with the majority of the newly synthesized protein partitioned toward a degradation pathway (Cyr, 2005).

F508 is located in NBD1 and is not essential for Cl- conductance (Rowe et al., 2005), but its deletion leads pools of nascent CFTRΔF508 to accumulate in a foldable, but kinetically trapped, conformation (Denning et al., 1992; Younger et al., 2004). CFTRΔF508 misfolding appears to involve subtle defects in NBD1 folding (Thibodeau et al., 2005; Serohijos et al., 2008a) that cause cotranslational misassembly of an intermediate that is degraded rapidly and may not accumulate (Younger et al., 2006). Folding of the small pool of nascent CFTRΔF508 that is spared initial degradation is arrested because of defective contact formation between NBD1 and regions that include intracellular loops exposed by MSD2 (Serohijos et al., 2008a) and misfolding of NBD2 (Du et al., 2005). Consequently, ∼99% of CFTRΔF508 is degraded prematurely by the ubiquitin–proteasome system (Jensen et al., 1995; Ward et al., 1995).

CFTR folding is assisted by several different molecular chaperones. The endoplasmic reticulum (ER)-associated Hsp40 Hdj-2 (DNAJA1) helps attract cytosolic Hsc70 to the ER membrane surface to facilitate cotranslational folding and assembly of NBD1 (Strickland et al., 1997; Meacham et al., 1999). The ER luminal chaperone cal­nexin appears to act after Hdj-2 to facilitate association of regions within the CFTR’s membrane-spanning and cytosolic domains (Pind et al., 1994; Rosser et al., 2008). Glycosylation of CFTR is required for calnexin binding, and this modification appears to be required for stabilization of transmembrane regions in MSD2 in the ER membrane (Glozman et al., 2009). Terminal steps in folding of full-length CFTR are facilitated by Hsp90 and its associated cofactors (Loo et al., 1998). Modulation of the Hsp90 cofactor AHA1 enhances CFTRΔF508 folding and is sufficient to permit small pools of functional CFTRΔF508 to accumulate at the cell surface (Wang et al., 2006).

Paradoxically, the selection of nascent forms of CFTR and CFTRΔF508 for proteasomal degradation is also facilitated by molecular chaperones. The cytosolic E3 ubiquitin ligase CHIP interacts with Hsc70 and/or Hsp70 to form a quality control (QC) machine that uses the polypeptide-binding activity of Hsc/Hsp70 to target misfolded CFTR for proteasomal degradation (Meacham et al., 2001). (From here on, Hsc70 and Hsp70 will be treated as the same protein and generally referred to as Hsc70.) In addition, the ER-associated E3 RMA1/RNF5 acts in association with Derlin-1 and the E2 Ubc6e to ubiquitinate CFTR (Younger et al., 2006). Ubiquitinated CFTR is delivered to the proteasome in a pathway that involves Gp78 (Morito et al., 2008), BAP31 (Wang et al., 2008), and p97 (Dalal et al., 2004). How selection of CFTR for degradation by the RMA1 E3 machinery and CHIP E3 complex are synergized is not entirely clear. The RMA1 E3 complex, however, may act coincident with translation to recognize folding defects in CFTR that involve the misfolding and defective assembly of NBD1 into a complex with the R-domain (Younger et al., 2006; Rosser et al., 2008). In contrast, the CHIP E3 may act post-translationally to recognize misfolded regions of CFTR that include NBD2 (Younger et al., 2006).

Deletion of F508 renders CFTR highly sensitive to changes in RMA1 activity, which suggests that the RMA1 E3 plays a critical role in selection of CFTRΔF508 for premature degradation (Younger et al., 2006). It is unclear, however, how the RMA1 complex distinguishes between on- and off-pathway forms of nascent CFTR. RMA1 interacts with the transmembrane QC factor Derlin-1, and small interfering RNA (siRNA) knockdown (kd) of Derlin-1 enhances cell surface expression of CFTR (Sun et al., 2006; Younger et al., 2006). Derlin-1 can form complexes with MSD1 of CFTR, and Derlin-1 overexpression promotes ER retention and proteasomal degradation of CFTR (Younger et al., 2006). Therefore it is possible that Derlin-1 is part of a complex that acts as a membrane chaperone to scan the assembly status of CFTR’s membrane regions and targets misassembled forms to RMA1 for ubiquitination. This model seems logical, except that the region in CFTR that is ubiquitinated by RMA1 is cytosolic. Therefore QC factors in addition to Derlin-1 may assist RMA1 in the selection of misfolded CFTR for ubiquitination.

Clues to the identity of such a factor come from the analysis of the Hsp40/DnaJ family of Hsc70 cochaperones (Kampinga and Craig, 2010). Hsp40 proteins use a conserved J-domain to regulate Hsc70 ATPase activity and specify client proteins of Hsc70. One mechanism for specification of Hsc70 function is for a specialized Hsp40 to attract Hsc70 to function in a discrete cellular location. Studies in yeast identify a unique Hsp40 subtype that is integrated into the ER membrane and exposes its J-domain to the cytosol (Walsh et al., 2004). This subfamily is represented in yeast by HLJ1 (high copy lethal DnaJ 1), and deletion analysis suggests that HLJ1 functions with cytosolic Hsc70 in ER QC (Youker et al., 2004; Han et al., 2007). RMA1 is not expressed in yeast, so we pondered whether an HLJ1-like protein might act with RMA1 in human cells to assist in the selection of CFTR for degradation.

Indeed, HLJ1-like Hsp40s are conserved in the human genome, and herein we report on the characterization of human DNAJB12 (JB12). JB12 is a Type II Hsp40 that contains a J-domain and G/F-like region and is similar to HLJ1 in that it is localized to the ER membrane and exposes its J-domain to the cytosol. JB12 differs from HLJ1, however, in that it contains N- and C-terminal extensions that were added during the course of evolution. Data presented demonstrate that JB12 acts with Hsc70 and RMA1 to facilitate proteasomal degradation of CFTR and CFTRΔF508. JB12 forms complexes that contain RMA1 and Derlin-1, and increasing JB12 levels dramatically increases association of Hsc70 with ER-localized forms of CFTR and RMA1. Depletion of endogenous JB12 results in a significant threefold increase in the folding efficiency of CFTR and permits a pool of CFTRΔF508 to escape the ER. JB12 appears to direct Hsc70 to function with the E3 RMA1 in degradation of nascent CFTRΔF508 and CFTR. As fluctuations in its activity have an impact on the fate of nascent CFTR, JB12 is capable of exerting control over the folding efficiency of nascent polytopic membrane proteins.

RESULTSJB12 is an ER-associated transmembrane Hsp40JB12 (GI:294862531) is a 375-amino-acid protein that contains a J-domain, a glycine/phenylalanine-rich region, and a single transmembrane domain (Figure 1A). The J-domain of JB12 is predicted to be located in the cytosol and is defined by the characteristic Hsc70 interaction sequence, HPD. The J-domain of JB12 is flanked by a 112-residue N-terminal extension and a 111-amino-acid residue C-terminal domain borders the transmembrane domain. The function of these respective domains in JB12 is unknown. Fluorescence microscopy of green fluorescent protein (GFP)-JB12 demonstrates that it colocalizes with the dye ER-tracker in a perinuclear location indicative of ER localization (Figure 1B).FIGURE 1:JB12 is an ER-associated transmembrane Hsp40. (A) JB12 is predicted to contain an N-terminal J-domain, a glycine/phenylalanine (G/F)-rich region, and a transmembrane (TM) domain. The J-domain of JB12 contains the characteristic Hsc70 interaction sequence, HPD. (B) Localization of GFP-JB12 with the ER in transiently transfected HEK293 cells by fluorescence microscopy. HEK293 cells transfected with 100 ng of pcDNA3.1(+)-GFP-JB12 were stained with ER-Tracker dye to visualize the localization of ER relative to that of GFP-JB12. (C) Fractionation of HA-JB12 with membranes that contain the ER marker calnexin. Membrane fractions isolated from HEK293 cells that were transfected with 0.05 μg of pcDNA3.1(+)-HA-JB12 were split into equal amount fractions and subjected to washes with 1.0 M NaCl, 0.1 M sodium carbonate, or 1.0% Triton. Membranes were then reisolated for each sample; S and P denote the supernatant and pellets of the centrifuged samples. The total (T) sample is the soluble fraction from a low-speed spin early in the fractionation process and is used to compare to levels of JB12 in the wash steps. The sodium carbonate data were performed as a separate experiment and are shown separately from the other samples. (D) Association of JB12 with Hsc70, but not BiP. HA-JB12 was immunoprecipitated from soluble cell lysates prepared with PBS-Triton (1%) and the cytosolic Hsc70, but not the ER-resident Hsc70 BiP, was found to exist in the isolated JB12-containing complexes. (E) HPD-dependent co-IP of endogenous Hsc70 with JB12-myc. JB12-myc or JB12-QPD-myc was immunoprecipitated from 35S-labeled cell extracts prepared with PBS-Triton (1%). (F) Cooperation of JB12 1–243 with Hsc70 in suppression of heat-denatured luciferase (LUC). Reactions were heated at 42ºC for 10 min followed by centrifugation to isolate the supernatant (S) and pellet (P) fractions from the total (T) sample. (G) Purified JB12 1–243 is unable to cooperate with Hsc70 to refold denatured luciferase. Luciferase was denatured in 6 M guanidinium HCl and then diluted into samples containing the indicated purified protein combinations. The folding reactions were incubated for 1 h at 25ºC, and luciferase activity was measured on a Turner luminometer. The amount of luciferase activity for each sample was normalized to the control sample, Hsc70 + Ydj1. The bar graph represents the average values obtained from two independent experiments.In support of these data, JB12 cofractionates with the ER marker calnexin in HEK293 cells and behaves like an integral membrane protein in that it is not extracted from ER membrane with sodium carbonate (Figure 1C). Cytosolic Hsc70, but not the ER luminal protein BiP, is detected by Western blot in native JB12 immunoprecipitates. This detection supports the prediction that its J-domain resides in the cytosol (Figure 1D). The observed interaction of JB12 with Hsc70 was dependent on JB12’s J-domain because JB12-QPD does not coprecipitate with 35S-Hsc70 (Figure 1E).JB12 can functionally interact with Hsc70 as a purified JB12 fragment (1–243), which contains the J-domain but lacks the transmembrane and ER luminal segments, stimulates Hsc70 ATPase activity from 5 to 15 nm ATP hydrolyzed · mg Hsc70 · min. This functional interaction is J-domain dependent as rates of ATP hydrolysis by Hsc70 in the presence of JB12-QPD 1–243 do not increase (unpublished data). Purified JB12 1–243 also cooperates with purified Hsc70 to suppress the aggregation of thermally denatured luciferase (Figure 1F). JB12 lacks conserved regions in Hsp40s known to bind soluble non-native polypeptides (Fan et al., 2004). This appears to explain why JB12 was incapable of acting independently to suppress luciferase aggregation (Figure 1F). Lack of intrinsic chaperone activity toward water-soluble proteins may also explain why JB12 1–243 is unable to act like the Type I Hsp40 Ydj1 and cooperate with Hsc70 to refold denatured luciferase (Figure 1G). These collective data indicate that JB12 is an ER-localized Hsp40 that uses its J-domain to regulate Hsc70s ATPase activity and enhance Hsc70 chaperone activity.JB12 mediates proteasome-dependent CFTR degradationDue to JB12’s localization to the ER, we investigated whether JB12 interacts with Hsc70 to facilitate CFTR and CFTRΔF508 folding and/or degradation. Increasing JB12 levels prevented accumulation of the ER-localized B-form and the maturely glycosylated and plasma membrane–localized C-form of CFTR (Figure 2A). Elevation of JB12 also caused similar decreases in the accumulation of the B-form of CFTRΔF508. Overexpressed JB12-QPD only moderately decreased CFTR folding and CFTRΔF508 accumulation, suggesting that JB12’s action appears to require Hsc70. Structure function analysis shows that JB12 requires association with the ER membrane to impact CFTR levels because deletion of its transmembrane domain rendered it inactive (Figure 2B). The N- and C-terminal domains of JB12 that flank the J-domain and transmembrane region, however, are not required for overexpressed JB12 to partition CFTR out of its folding pathway (Figure 2B). The results are evident when comparing CFTR levels under conditions where JB12 1–243, JB12 1–264, and JB12 112–375 are overexpressed to similar levels as full-length JB12 (Figure 2B). Thus overexpressed JB12 uses its J-domain and transmembrane domain to decrease the accumulation of CFTRΔF508 and CFTR.FIGURE 2:JB12 mediates proteasome-dependent CFTR degradation. (A) JB12 overexpression decreases CFTR accumulation as detected by Western blots of HEK293 cell extracts that were transiently transfected with indicated combinations of expression plasmids for CFTR, CFTRΔF508, JB12-myc, or JB12-QPD-myc. (B) JB12-mediated degradation of CFTR requires the transmembrane domain but not the N- or C-terminal extensions. HEK293 cells transiently transfected with pcDNA3.1(+)-CFTR (1 μg) and increasing amounts of pcDNA3.1(+)-JB12-myc or pcDNA3.1(+)-JB12-myc truncation mutants (which ranged from 5 to 75 ng) were isolated and subjected to Western blot analysis with the indicated antibodies. (C) JB12-mediated decrease in CFTR accumulation is sensitive to the proteasome inhibitor bortezomib (10 μM) that was added 4 h prior to analyzing the samples by Western blot. (D) Ubiquitinated forms of CFTR and CFTRΔF508 accumulate when their JB12-stimulated degradation is hindered via expression of a dominant negative form of p97, p97QQ. CFTR or CFTRΔF508 was immunoprecipitated from extracts of cells that express the indicated proteins. Ubiquitinated CFTR or CFTR in immunoprecipitated material was detected by Western blot. When p97QQ interfered with the ERAD pathway and JB12 was overexpressed, the level of ubiquitinated CFTR detected was 92% of the quantity that accumulated when only p97QQ was expressed. Levels of ubiquitinated CFTRΔF508 in the presence of JB12 and p97QQ were 45% of levels detected when only p97QQ was expressed, and this value reflected the relative level of CFTRΔF508 under the indicated experimental condition. (See Results for an explanation.) Where indicated, bands B and C represent the immaturely glycosylated and maturely glycosylated forms of CFTR, respectively. Loading consistency is indicated relative to levels of tubulin in different panels.The reduction of total CFTR accumulation resulting from JB12 overexpression is proteasome dependent as it is blocked by the proteasome inhibitor bortezomib (Figure 2C). When JB12 is overexpressed and the proteasome is inhibited, severalfold more of the B-form of CFTR and CFTRΔF508 accumulates and a portion of the B-form migrates on gels as a low-mobility smear reminiscent of a polyubiquitinated species. Thus JB12 appears to reduce accumulation of the C-form of CFTR and the B-form of CFTRΔF508 via a proteasome-dependent pathway.When the ER-associated protein degradation (ERAD) pathway was inhibited by overexpression of a dominant negative form of the ER retrotranslocation factor p97 (p97QQ), inhibition of JB12-dependent degradation of CFTR and CFTRΔF508 was also observed (Figure 2D). In addition, the degradation intermediates of the B-form of CFTR and CFTRΔF508 that p97QQ drove to accumulate were demonstrated via immunoprecipitation (IP) and Western blot analysis to be ubiquitinated. p97QQ is a dominant negative form of p97 that binds (but cannot release) substrates, and blocks extraction of ubiquitinated proteins from the ER (Hirsch et al., 2009). Yet p97 is an abundant protein, so its function in CFTR degradation was not blocked completely by p97QQ overexpression, and p97QQ was not as effective as bortezomib in blocking JB12-dependent CFTR degradation.In the presence of p97QQ and JB12, levels of CFTRΔF508 were not restored to those observed in the absence of JB12. In contrast, when bortezomib was used to inhibit the proteasome in the presence of JB12, CFTRΔF508 levels were dramatically increased and levels of CFTRΔF508 were similar to those detected in bortezomib-treated cells in which JB12 was not elevated (Figure 2C). Therefore proper function of p97 may be required to stabilize the B-form of CFTRΔF508 during the process of JB12-dependent proteasomal degradation, but this hypothesis remains to be tested.Knockdown of endogenous JB12 increases CFTR folding efficiencyTo evaluate the role of endogenous JB12 in CFTR biogenesis, siRNA experiments were performed and the effect that kd of JB12 had on biosynthetic maturation of CFTR and CFTRΔF508 in HEK293 cells was evaluated by pulse-chase and Western blot analysis (Figure 3, A and B). Endogenous JB12 levels were reduced by >90% in HEK293 cells, and the pair of siRNAs used were specific in that the individual siRNAs were both capable of independently performing this task (unpublished data). JB12 kd also had no detectable impact on the levels of other QC factors that modulate CFTR folding or degradation (Supplemental Figure S1). Data from pulse-chase studies show that reduction in JB12 levels leads to a dramatic threefold increase in the quantity of nascent B-form CFTR that is converted to the C-form by the end of the pulse-chase time course. Likewise, the half-life of CFTRΔF508 was increased upon JB12 kd, but accumulation of its folded C-form was not detected.FIGURE 3:Knockdown of endogenous JB12 increases CFTR folding efficiency. Pulse chase (A) and Western blot (B) analyses indicate that the folding efficiency of CFTR is dramatically enhanced whereas the stability of CFTRΔF508 is modestly increased in JB12 siRNA HEK293 cells. The folding efficiency of CFTR increases from approximately 15 ± 2% to 45 ± 3% of the B-form at t = 0 upon kd of JB12 (n = 3). Under control conditions (si-Cont), the half-life of CFTRΔF508 was 40 min (SD ± 2 min) and in the absence of JB12 increased 1.5-fold to 60 min (SD ± 2 min; n = 3). The band marked with an * is a background band in the IP reactions in (B). (C) JB12 kd in human CFBE cells also increases accumulation of CFTR and CFTRΔF508. The immaturely glycosylated B-form and maturely glycosylated C-form of CFTR are indicated. Results were quantified by densitometry and were either normalized to the relative amount of CFTR B-band at t = 0 for each condition (A) or the CFTR B- and C-band levels were normalized to the si-Cont reactions (B and C).When steady-state levels of CFTR were assayed 72 h after JB12 kd, severalfold more of the C-form accumulated, and this accumulation appeared to be due to increased folding efficiency as it was not accompanied by a corresponding increase in the B-form levels. We could also see an increase in the accumulation of the B-form of CFTRΔF508, but the effects of JB12 kd on CFTRΔF508 were less dramatic than those of CFTR.To demonstrate that the function of JB12 in CFTR biogenesis in HEK293 cells is relevant to what occurs in lung epithelial cells, the impact of JB12 kd on CFTR and CFTRΔF508 biogenesis in immortalized human bronchial epithelial (CFBE) cells was determined (Figure 3C). JB12 levels could be reduced by >70% in CFBE cells, and this reduction was accompanied by a severalfold increase in accumulation of the C-form of CFTR. Again, we also observed JB12 kd to permit increased accumulation of the B-form of CFTRΔF508, but the C-form was difficult to detect.Changes in JB12 levels correlate inversely with CFTR and CFTRΔF508 levels in both HEK293 and CFBE cells. CFTR folding is dramatically increased by JB12 kd, but increases in folded CFTRΔF508 are less dramatic. Thus JB12 action appears to limit CFTR folding efficiency, whereas other cellular factors or kinetic folding defects limit the folding of CFTRΔF508. The effect of CFTRΔF508 behavior upon JB12 kd fits with the concept that the majority of CFTRΔF508 accumulates in a kinetically trapped non-native state that is difficult to bring back onto a folding pathway (Younger et al., 2004).Knockdown of JB12 and Hdj-2 have opposite effects on CFTR foldingJB12 and the cytosolic Type I Hsp40 Hdj-2 (Meacham et al., 1999) both act on the cytoplasmic face of the ER but appear to differentially direct Hsc70 to promote the life or death of CFTR. If this interpretation is true, then Hdj-2 kd should hinder CFTR folding and accelerate degradation of the B-form of CFTR and CFTRΔF508. To test this supposition, Hdj-2 levels were reduced by small hairpin RNA (shRNA) and the impact that this reduction had on CFTR levels was determined. Hdj-2 is highly abundant, but shRNAs were able to reduce its levels by ∼70%, which was associated with a 70% reduction in accumulation of the folded C-form of CFTR and a 50% reduction in accumulation of the immature B-form of CFTRΔF508 (Figure 4A). Data from pulse-chase studies suggest that decreased accumulation of folded CFTR detected by Western blot when Hdj-2 levels are low results from decreased folding and stability of the nascent B-form (Figure 4B). Hdj-2 kd decreased CFTR folding efficiency severalfold and decreased stability of the nascent B-form of CFTRΔF508. These data support the notion that Hdj-2 and JB12 direct Hsc70 to play opposing roles in CFTR folding and degradation, respectively.FIGURE 4:Knockdown of JB12 and Hdj-2 has opposite effects on CFTR folding. (A and B) Hdj-2 kd decreases folding and stability of CFTR. Hdj-2 shRNA (sh-Hdj2) or cont-shRNA (sh-Cont) HEK293 cells transfected with pcDNA3.1(+)-CFTR or pcDNA3.1(+)-CFTRΔF508 were collected for Western blot analysis of CFTR steady-state levels (A) or 35S-labeled for CFTR pulse-chase analysis (B). The steady-state levels (C) and half-life (D) of either ApoB or TCRα are not affected by JB12 siRNA (si-JB12). Results that were quantified by densitometry were either normalized to the control reactions (A and C) or normalized to the relative amount of 35S-labeled material at t = 0 for each condition (B and D).Modulating JB12 levels does not appear to cause gross changes in ER protein QC activity in HEK293 cells because JB12 kd does not impact degradation of two different topologically distinct ERAD substrates (Figure 4, C and D). TCRα is a subunit of the T-cell receptor, has a large luminal domain and single transmembrane domain, and is degraded rapidly via ERAD when expressed alone because it lacks assembly partners (Tiwari and Weissman, 2001). ApoB48 is inserted across the ER membrane and is an Hsc70 client but is degraded rapidly in HEK293 cells because they don’t express specialized folding factors required for proper biogenesis (Fisher et al., 1997). JB12 kd does not alter the steady-state levels of TCRα or ApoB48 (Figure 4C), and pulse-chase studies show that JB12 kd does not alter the biosynthesis or half-life of either (Figure 4D). Thus stabilization of the B-form of CFTR upon JB12 kd is not associated with a general stabilization of other ERAD substrates that have different domain structures.JB12 drives Hsc70 to associate with CFTR and the RMA1 E3 complexTo explore the mechanism for JB12 action, its presence in complexes that contain RMA1 and/or CHIP was explored (Figure 5A). Protein–protein interactions were evaluated by asking whether JB12 and Hsc70 coprecipitate under native buffer conditions with either RMA1 or CHIP. JB12 and Hsc70 were pulled down with RMA1, but CHIP was not detected in complexes with JB12. In addition, elevation of JB12 levels increased the quantity of Hsc70 that coprecipitated with flag-RMA1 (Figure 5A).FIGURE 5:JB12 drives Hsc70 to associate with CFTR and the RMA1 E3 complex. (A) JB12 and Hsc70 coprecipitate with the RMA1 E3 ligase, but the E3 ligase CHIP was not observed in a complex with JB12. Flag-RMA1 was immunoprecipitated from soluble cell lysates prepared with PBS-Triton (1%), and both Hsc70 and JB12-myc were found in association with RMA1-containing complexes. In contrast, IP of HA-JB12 from soluble cell lysates isolated Hsc70, but not CHIP-myc, in JB12-containing complexes. The presence of indicated proteins in native immunoprecipitates was determined by Western blot. (B) Elevation of JB12 increases the quantity of 35S-Hsc70 that associates with CFTRΔF508 in a nucleotide-dependent manner. HEK 293 cells transfected with pcDNA3.1(+)-CFTRΔF508 and pcDNA3.1(+)-JB12-myc or pcDNA3.1(+)-JB12-QPD-myc, as indicated, were labeled with 35S-methinone, and co- and re-IP reactions were conducted as described in Materials and Methods. Briefly, the samples were lysed in PBS-Triton (1%) with either an ATP regeneration system (+ ATP) or with apyrase (- ATP), and CFTR was immunoprecipitated from the soluble cell lysates. The reactions were then subjected to a secondary re-IP with α-Hsc70 antibody. Western blots indicate the levels of CFTRΔF508 when JB12 or JB12-QPD is overexpressed. (C) Hsc70, flag-RMA1, Derlin-1, and CFTRΔF508 coprecipitate with HA-JB12. HEK 293 cells overexpressing HA-JB12, flag-RMA1, Derlin-1, and CFTRΔF508 were labeled with 35S-methinone and solubilized with PBS-Triton (1%). Co-IP and re-IP reactions were conducted as described in Materials and Methods. In general, HA-JB12 was isolated from the cell lysates, and secondary re-IP steps identified flag-RMA1, Derlin-1, endogenous Hsc70, and CFTRΔF508 as being part of a JB12-containing complex. A 3-d film exposure was sufficient to visualize proteins bound in initial co-IP reactions (C), whereas a 7-d exposure was necessary to visualize bands in the reimmunoprecipitated samples (B and C).Next we evaluated the impact that elevation of JB12 levels had on Hsc70 binding to nascent 35S-CFTRΔF508 (Figure 5B). CFTRΔF508 was immunoprecipitated from native cell extracts in the presence or absence of overexpressed JB12. The Hsc70 present with CFTRΔF508 in initial precipitates was determined by reimmunoprecipitation (re-IP). The quantity of 35S-Hsc70 coprecipitated with CFTRΔF508 when ATP levels were elevated was low, but an increase was detected upon expression of HA-JB12. This finding is impressive as the quantity of CFTRΔF508 in cells was significantly lower when JB12 was elevated. Depletion of ATP resulted in a more dramatic, approximately 10-fold increase in JB12-dependent Hsc70 binding to the B-form of CFTRΔF508 (Figure 5B, bottom panel). HA-JB12-QPD expression had little impact on accumulation of the B-form of CFTRΔF508 and did not cause an increase in Hsc70 binding to CFTRΔF508. Similar results were observed with the B-form of CFTR (unpublished data). These data indicate that JB12 acts in an ATP- and J-domain–dependent manner to increase association of Hsc70 with complexes that contain the nascent B-form of CFTR.JB12 is present in complexes that contain RMA1 and can increase Hsc70 binding to the B-form of CFTR and CFTRΔF508. Thus JB12 appears to act with RMA1 in degradation of the misfolded B-form of CFTR and CFTRΔF508. If so, JB12 should be present in complexes that contain CFTRΔF508 as well as RMA1, Hsc70, and Derlin-1 (Younger et al., 2006). This supposition was tested in 35S-labeled cells that expressed HA-JB12, flag-RMA1, Derlin-1, and CFTRΔF508 and that were lysed under native conditions (Figure 5C). HA-JB12 was immunoprecipitated, and the ERQC factors of interest were identified by re-IP. A large number of radiolabeled proteins the sizes of which spanned the range of molecular weights resolved on a 12.5% gel were found to coprecipitate HA-JB12 but not with the beads used in precipitations. The large number of bands associated with JB12 is consistent with it functioning with Hsc70 to facilitate the biogenesis of a range of substrates. Re-IPs identified Hsc70, RMA1, Derlin-1, and CFTRΔF508 in precipitates with JB12 (Figure 5C). We performed similar experiments with CHIP-myc but never detected the overexpressed or endogenous CHIP in a complex with JB12 (unpublished data).Of note is that in each of the re-IPs with the indicated antibodies a quantity of JB12 was present. The proteins of interest are membrane proteins that aggregate at elevated temperatures, so samples were heated at 55ºC, instead of 100ºC, before the re-IP step to disrupt interactions between JB12 and the antibody used for initial precipitations. In the absence of boiling, however, this process is incomplete; thus JB12 is present in the Hsc70, RMA1, and Derlin-1 re-IPs. To complicate things further, CFTRΔF508 can aggregate when heated >37ºC in an SDS sample buffer, so the samples used for re-IPs of CFTR were incubated at 37ºC instead of 55ºC. Therefore the original antibody–JB12 interactions were not disrupted, and we observed more JB12 in the CFTRΔF508 re-IPs than in other re-IPs. Controls demonstrate, however, that the presence of Hsc70, RMA1, Derlin-1, and CFTRΔF508 in re-IPs occurs only if antibodies specific to these individual proteins are present.Derlin-1 interacts with multiple E3 ubiquitin ligases (Hirsch et al., 2009), and yeast JB12 is found to cooperate with Hsc70 Ssa1 and the ER-associated E3 Doa10 in the degradation of P-type ATPases and other model ERAD substrates (Han et al., 2007; Nakatsukasa et al., 2008). Thus we explored the specificity of interactions between JB12 with Derlin-1 and RMA1 via determination of whether JB12 could coprecipitate in complexes with the following ER-localized E3 ubiquitin ligases: Gp78 (Morito et al., 2008), March V (Bartee et al., 2004), HRD1 (Hirsch et al., 2009), and SEL1 (Hirsch et al., 2009) (Supplemental Figure S2). Association of JB12 with Gp78, a known interaction partner of RMA1, was detected (Morito et al., 2008), but complexes with MarchV, HRD1, or SEL1 were not observed. These data suggest that JB12 selectively interacts with a subset of ER-associated E3 ligases (Supplemental Figure S2). Overall, data obtained on protein–protein interactions with JB12 demonstrate that it selectively associates with components of the RMA1 E3 complex and can facilitate the association of Hsc70 with nascent CFTR/CFTRΔF508 and RMA1.RMA1 is required for JB12 to promote CFTR degradationOn the basis of data presented thus far (Figure 5), we suggest that JB12 functionally interacts with RMA1, but not CHIP, in degradation of misfolded forms of CFTR. This interpretation is based on the presence of JB12 in complexes with RMA1, Derlin-1, Hsc70, and CFTR and an inability to detect JB12 in complexes that contain Hsc70 and CHIP. If this is true, then RMA1, but not CHIP, should be required for JB12-dependent partitioning of nascent CFTR out of its folding pathway. To further evaluate the potential for functional interactions between CHIP and RMA1, we compared the impact of elevating JB12 approximately onefold over endogenous pools on CFTR biogenesis when either RMA1 or CHIP levels were depleted (Figure 6A). In cells treated with a control siRNA, elevation of JB12 reduced accumulation of the B- and C-forms of CFTR to a barely detectable level (Figure 6A). As predicted, kd of RMA1 hindered the ability of overexpressed JB12 to promote CFTR degradation. It is important to note, however, that in the absence of RMA1, JB12 still retained CFTR in the ER (Figure 6A, top panel). Similarly, when RMA1 is knocked down, overexpressed JB12 did not decrease the accumulation of CFTRΔF508 (Figure 6A, bottom panel). In contrast, JB12 was still fully capable of promoting degradation of CFTR and CFTRΔF508 when CHIP was depleted (Figure 6A).FIGURE 6:RMA1 is required for JB12 to promote CFTR degradation. (A) The effect that RMA1 or CHIP kd has on JB12-dependent decreases in CFTR and CFTRΔF508 accumulation as detected by Western blot. Antibodies directed against RMA1 and CHIP were used to detect the extent of kd of endogenous RMA1 and CHIP levels. (B) Impact of RMA1 and JB12 action alone and in combination on accumulation of CFTR and CFTRΔF508 levels. RMA1 and JB12 blots are included to show the level of overexpression of these proteins relative to their endogenous levels. Tubulin blots indicate loading consistency.Further support for the concept of functional interaction between JB12 and RMA1 in CFTR degradation comes from the observation that these proteins can synergize to perform this function (Figure 6B). Coexpressing JB12-myc and flag-RMA1 at levels where individually they have a modest effect on steady-state levels of CFTR (Figure 6B, top panel) or CFTRΔF508 (Figure 6B, bottom panel) leads to dramatic decreases in the folded C-form of CFTR and the B-form of CFTRΔF508. The decrease observed is blocked via treatment of cells with the proteasome inhibitor bortezomib, suggesting that JB12 and RMA1 cooperate in targeting CFTR for proteasomal degradation. The collective data from siRNA and overexpression studies, when combined with data from co-IP experiments, suggest that JB12 cooperates with Hsc70 to mediate RMA1-dependent CFTR degradation.JB12 monitors the folding status of CFTR N-terminal regionsChaperone-dependent steps in CFTR folding and degradation have been uncovered via analysis of the assembly of CFTR N- and C-terminal fragments (CFTR 837X and CFTR 837–1480) into a complex that escapes the ER and functions as a Cl- channel (Chan et al., 2000). Studies with CFTR 837X suggest that deletion of F508 causes a severe defect in formation of a folding intermediate that contains MSD1, NBD1, and the R-domain that is critical for the folding progression of CFTR (Rosser et al., 2008). Recognition of folding defects in amino-terminal regions of CFTRΔF508 by the RMA1 E3 complex is proposed to be a major cause of premature CFTR degradation (Younger et al., 2006; Rosser et al., 2008). Thus we sought to obtain support for the conclusion that JB12 participates with RMA1 and Derlin-1 in selecting CFTR for degradation by asking if JB12 associates with CFTR 837X and determining whether it facilitates CFTR 837X degradation (Figure 7).FIGURE 7:JB12 monitors the folding status of CFTR N-terminal regions. (A) Endogenous JB12 interacts with the N-terminal half of CFTR. HEK293 cells transfected with either pcDNA3.1(+)-CFTR 837X or pcDNA3.1(+)-CFTR 837–1480 were lysed with PBS-Triton (1%), and endogenous JB12 was immunoprecipitated. Western blots of precipitates were probed with either α-CFTR N-terminal tail or α-CFTR-NBD2. (B–D) Transfections with JB12 or RMA1 siRNA oligos and pcDNA3.1(+)-CFTR 837X, pcDNA3.1(+)-CFTR 837XΔF508, or pcDNA3.1(+)-CFTR 837–1480 were performed as described in Materials and Methods. Levels of indicated proteins in cell extracts were determined by Western blot. CFTR 837X or CFTR 837XΔF508 levels were quantified by densitometry, and levels were normalized to control siRNA (si-Cont) reactions for each fragment. Antibodies directed against RMA1 and JB12 were used to monitor the impact of RNA interference addition on endogenous RMA1 and JB12 levels, respectively. Maturely glycosylated CFTR 837–1480 is designated as C-band, and the immaturely glycosylated form is indicated as B-band.To analyze the association between JB12 and CFTR 837X, HEK293 cells were transfected with either CFTR 837X (which is composed of MSD1, NBD1, and the R-domain) or the MSD2- and NBD2-containing C-terminal CFTR fragment 837–1480. Endogenous JB12 was then immunoprecipitated and via Western blot was found to coprecipitate with CFTR 837X (Figure 7A). This complex appears to be a result of a specific interaction, as a complex that contains endogenous JB12 and CFTR 837–1480 was not isolated. Similar results were obtained with CFTR 837XΔF508 (unpublished data). Thus endogenous JB12 associates with an amino-terminal region of CFTR, the conformation of which is surveyed by components of the RMA1 E3 complex.Deletion of F508 hinders the folding of CFTR 837X, and this fragment is unstable (Rosser et al., 2008). Knockdown of JB12 or RMA1 results in a two- to threefold increase in the accumulation of CFTR 837XΔF508 with the greatest effect observed upon JB12 kd (Figure 7B). This increase in accumulation is related to recognition of misfolded CFTR 837XΔF508 by RMA1 and JB12 as accumulation of CFTR 837X, which is highly stable (Meacham et al., 1999), was only modestly increased upon kd of JB12 or RMA1. Furthermore, accumulation of the C-terminal CFTR 837–1480 fragment is not affected by kd of JB12 (Figure 7C). We do observe, however, that the levels of CFTR 837–1480 increase in the absence of RMA1 (Figure 7D). Thus RMA1 may also have additional functions that are independent of JB12.CFTR 837X, but not CFTR 837XΔF508, assembles into a complex with CFTR 837–1480 that escapes the ER as indicated by its accumulation as a glycosylated species that is resistant to Endo H treatment (Rosser et al., 2008). Intriguingly, productive interactions between the two halves of CFTR when JB12 (Figure 7C) or RMA1 (Figure 7D) levels are knocked down are enhanced such that significantly more folded, maturely glycosylated CFTR 837–1480 accumulates. Thus JB12 or RMA1 kd allows for the increased association and proper assembly of N- and C-terminal regions of CFTR. Kd of these ERQC factors, however, does not permit CFTR 837XΔF508 and CFTR 837–1480 to assemble into a complex that escapes the ER as indicated by the lack of CFTR 837–1480 accumulation as a maturely glycosylated species (Figure 7, C and D). Thus JB12 can act at an early stage of CFTR biogenesis, which is coincident with RMA1 action, to modulate the assembly of N- and C-terminal CFTR fragments into a complex that can escape the ER.Stepwise correction of CFTRΔF508 folding to wild-type levels through depletion of JB12 and intragenic suppression of intrinsic folding defectsThe contribution that ERQC activities versus intrinsic folding kinetics play in governing membrane protein folding is unclear, and this information is required to develop approaches to correct CFTRΔF508 misfolding. Therefore we investigated the relative contribution that folding kinetics and the basal ERQC activity contribute to premature CFTRΔF508 degradation. SiRNA kd of JB12 can increase CFTR folding efficiency threefold, whereas elevation of JB12 limits the folding progression of CFTR (Figures 2 and 3). In contrast, siRNA kd of JB12 permits only a small quantity of CFTRΔF508 to accumulate in the folded C-form (Figure 3) and does not enhance assembly of CFTR 837XΔF508 into a folded channel (Figure 7, C and D). Thus folding of CFTR and CFTRΔF508 appear to differ in that ERQC activity limits efficiency of CFTR folding, whereas some other unknown ERQC factor and/or intrinsic folding defects caused by deletion of F508 limits CFTRΔF508.To explore this issue, we evaluated the relative contribution of JB12 action and defective folding kinetics on escape of CFTRΔF508 from the ER. We compared the extent to which the introduction of V510D, a misfolding suppressor mutation, into CFTRΔF508 versus inactivation of JB12 enhanced CFTRΔF508 folding (Figure 8). Residue V510 is located near F508 on the surface of NBD1, and its mutation to an Asp helps to increase cell surface expression of CFTRΔF508 by a mechanism that is proposed to involve enhancement of interdomain contacts (Mornon et al., 2008). The folding of CFTR, CFTRΔF508, and CFTRΔF508 V510D was compared under control conditions, upon JB12 depletion, when cells were treated with the folding corrector Corr-4a (Pedemonte et al., 2005), or with a combination of the above conditions. CFTR folding efficiency was evaluated in Western blots by measuring the ratio of the C-band and B-band that accumulated (Figure 8A). In parallel, we carried out pulse-chase studies and compared the efficiency of B-band conversion to C-band at different chase times (Figure 8B). Data obtained from both sets of experiments are mutually supportive.FIGURE 8:Stepwise correction of CFTRΔF508 folding to wild-type levels through depletion of JB12 and intragenic suppression of intrinsic folding defects. (A) Western blot analysis of the C- and B-form of CFTR. (B) Pulse-chase analysis of CFTR biosynthetic maturation. Transfections with JB12 siRNA oligos and pcDNA3.1(+)-CFTR, pcDNA3.1(+)-CFTRΔF508, or pcDNA3.1(+)-CFTRΔF508 V510D were performed as described in Materials and Methods. The siRNA samples were treated with DMSO or Corr-4a (5 μM) 24 h prior to being harvested. Steady-state levels of CFTR and CFTR mutants were determined by Western blot. In panel B, cells were labeled with 35S-methionine for 20 min, and 35S-CFTR was immunoprecipitated at the indicated time points. Bands B and C represent the immature and maturely glycosylated forms of CFTR, respectively. The B- and C-band levels for each reaction were quantified by densitometry. The normalized C/B-band ratio is shown for (A), whereas in (B) the results were normalized to the relative amount of CFTR B-band at t = 0 for each condition.JB12 siRNA permits approximately fourfold more CFTRΔF508 to accumulate in the C-form, and the ratio of the C- to B-form increased from not being detectable to 0.08 (Figure 8A). CFTRΔF508 folding is improved to a point at which the C/B ratio is further increased to ∼0.2 when JB12 is depleted and Corr-4a is present. Folding of CFTRΔF508 increased dramatically upon introduction of the V510D suppressor mutation because CFTRΔF508 V510D accumulated at a C/B ratio of 0.2. Strikingly, depletion of JB12 increased the C/B ratio of CFTRΔF508 V510D more than eightfold to an impressive value near 1.6, which is 50% of the ratio observed for wild-type CFTR. In pulse-chase experiments, we also observed depletion of JB12 to permit CFTRΔF508 V510D to fold with ∼50% of wild-type efficiency (Figure 8B).The combination of JB12 kd and Corr-4a treatment enabled CFTRΔF508 V510D to obtain a C/B ratio of 3.5, which is the same as the 3.4 value observed with wild-type CFTR. In pulse-chase experiments, we also observed that these conditions permit the B-form of CFTRΔF508 V510D to convert to the C-form with an efficiency that was greater than CFTR (Figure 8B). Corr-4a is able to stabilize transmembrane regions of CFTR (Wang et al., 2007; Grove et al., 2009), which may explain the twofold increase in CFTRΔF508 V510D folding efficiency that occurs when Corr-4a is present in JB12-depleted cells. When JB12 activity is attenuated and intrinsic folding defects are corrected, CFTRΔF508 can fold and escape the ER at levels similar to wild-type CFTR. Thus it appears that activity of JB12 and intrinsic folding kinetics both play important roles in limiting the escape of CFTRΔF508 from the ER.

JB12 is an ER-associated transmembrane Hsp40JB12 (GI:294862531) is a 375-amino-acid protein that contains a J-domain, a glycine/phenylalanine-rich region, and a single transmembrane domain (Figure 1A). The J-domain of JB12 is predicted to be located in the cytosol and is defined by the characteristic Hsc70 interaction sequence, HPD. The J-domain of JB12 is flanked by a 112-residue N-terminal extension and a 111-amino-acid residue C-terminal domain borders the transmembrane domain. The function of these respective domains in JB12 is unknown. Fluorescence microscopy of green fluorescent protein (GFP)-JB12 demonstrates that it colocalizes with the dye ER-tracker in a perinuclear location indicative of ER localization (Figure 1B).FIGURE 1:JB12 is an ER-associated transmembrane Hsp40. (A) JB12 is predicted to contain an N-terminal J-domain, a glycine/phenylalanine (G/F)-rich region, and a transmembrane (TM) domain. The J-domain of JB12 contains the characteristic Hsc70 interaction sequence, HPD. (B) Localization of GFP-JB12 with the ER in transiently transfected HEK293 cells by fluorescence microscopy. HEK293 cells transfected with 100 ng of pcDNA3.1(+)-GFP-JB12 were stained with ER-Tracker dye to visualize the localization of ER relative to that of GFP-JB12. (C) Fractionation of HA-JB12 with membranes that contain the ER marker calnexin. Membrane fractions isolated from HEK293 cells that were transfected with 0.05 μg of pcDNA3.1(+)-HA-JB12 were split into equal amount fractions and subjected to washes with 1.0 M NaCl, 0.1 M sodium carbonate, or 1.0% Triton. Membranes were then reisolated for each sample; S and P denote the supernatant and pellets of the centrifuged samples. The total (T) sample is the soluble fraction from a low-speed spin early in the fractionation process and is used to compare to levels of JB12 in the wash steps. The sodium carbonate data were performed as a separate experiment and are shown separately from the other samples. (D) Association of JB12 with Hsc70, but not BiP. HA-JB12 was immunoprecipitated from soluble cell lysates prepared with PBS-Triton (1%) and the cytosolic Hsc70, but not the ER-resident Hsc70 BiP, was found to exist in the isolated JB12-containing complexes. (E) HPD-dependent co-IP of endogenous Hsc70 with JB12-myc. JB12-myc or JB12-QPD-myc was immunoprecipitated from 35S-labeled cell extracts prepared with PBS-Triton (1%). (F) Cooperation of JB12 1–243 with Hsc70 in suppression of heat-denatured luciferase (LUC). Reactions were heated at 42ºC for 10 min followed by centrifugation to isolate the supernatant (S) and pellet (P) fractions from the total (T) sample. (G) Purified JB12 1–243 is unable to cooperate with Hsc70 to refold denatured luciferase. Luciferase was denatured in 6 M guanidinium HCl and then diluted into samples containing the indicated purified protein combinations. The folding reactions were incubated for 1 h at 25ºC, and luciferase activity was measured on a Turner luminometer. The amount of luciferase activity for each sample was normalized to the control sample, Hsc70 + Ydj1. The bar graph represents the average values obtained from two independent experiments.In support of these data, JB12 cofractionates with the ER marker calnexin in HEK293 cells and behaves like an integral membrane protein in that it is not extracted from ER membrane with sodium carbonate (Figure 1C). Cytosolic Hsc70, but not the ER luminal protein BiP, is detected by Western blot in native JB12 immunoprecipitates. This detection supports the prediction that its J-domain resides in the cytosol (Figure 1D). The observed interaction of JB12 with Hsc70 was dependent on JB12’s J-domain because JB12-QPD does not coprecipitate with 35S-Hsc70 (Figure 1E).JB12 can functionally interact with Hsc70 as a purified JB12 fragment (1–243), which contains the J-domain but lacks the transmembrane and ER luminal segments, stimulates Hsc70 ATPase activity from 5 to 15 nm ATP hydrolyzed · mg Hsc70 · min. This functional interaction is J-domain dependent as rates of ATP hydrolysis by Hsc70 in the presence of JB12-QPD 1–243 do not increase (unpublished data). Purified JB12 1–243 also cooperates with purified Hsc70 to suppress the aggregation of thermally denatured luciferase (Figure 1F). JB12 lacks conserved regions in Hsp40s known to bind soluble non-native polypeptides (Fan et al., 2004). This appears to explain why JB12 was incapable of acting independently to suppress luciferase aggregation (Figure 1F). Lack of intrinsic chaperone activity toward water-soluble proteins may also explain why JB12 1–243 is unable to act like the Type I Hsp40 Ydj1 and cooperate with Hsc70 to refold denatured luciferase (Figure 1G). These collective data indicate that JB12 is an ER-localized Hsp40 that uses its J-domain to regulate Hsc70s ATPase activity and enhance Hsc70 chaperone activity.

JB12 is an ER-associated transmembrane Hsp40

JB12 (GI:294862531) is a 375-amino-acid protein that contains a J-domain, a glycine/phenylalanine-rich region, and a single transmembrane domain (Figure 1A). The J-domain of JB12 is predicted to be located in the cytosol and is defined by the characteristic Hsc70 interaction sequence, HPD. The J-domain of JB12 is flanked by a 112-residue N-terminal extension and a 111-amino-acid residue C-terminal domain borders the transmembrane domain. The function of these respective domains in JB12 is unknown. Fluorescence microscopy of green fluorescent protein (GFP)-JB12 demonstrates that it colocalizes with the dye ER-tracker in a perinuclear location indicative of ER localization (Figure 1B).

JB12 is an ER-associated transmembrane Hsp40. (A) JB12 is predicted to contain an N-terminal J-domain, a glycine/phenylalanine (G/F)-rich region, and a transmembrane (TM) domain. The J-domain of JB12 contains the characteristic Hsc70 interaction sequence, HPD. (B) Localization of GFP-JB12 with the ER in transiently transfected HEK293 cells by fluorescence microscopy. HEK293 cells transfected with 100 ng of pcDNA3.1(+)-GFP-JB12 were stained with ER-Tracker dye to visualize the localization of ER relative to that of GFP-JB12. (C) Fractionation of HA-JB12 with membranes that contain the ER marker calnexin. Membrane fractions isolated from HEK293 cells that were transfected with 0.05 μg of pcDNA3.1(+)-HA-JB12 were split into equal amount fractions and subjected to washes with 1.0 M NaCl, 0.1 M sodium carbonate, or 1.0% Triton. Membranes were then reisolated for each sample; S and P denote the supernatant and pellets of the centrifuged samples. The total (T) sample is the soluble fraction from a low-speed spin early in the fractionation process and is used to compare to levels of JB12 in the wash steps. The sodium carbonate data were performed as a separate experiment and are shown separately from the other samples. (D) Association of JB12 with Hsc70, but not BiP. HA-JB12 was immunoprecipitated from soluble cell lysates prepared with PBS-Triton (1%) and the cytosolic Hsc70, but not the ER-resident Hsc70 BiP, was found to exist in the isolated JB12-containing complexes. (E) HPD-dependent co-IP of endogenous Hsc70 with JB12-myc. JB12-myc or JB12-QPD-myc was immunoprecipitated from 35S-labeled cell extracts prepared with PBS-Triton (1%). (F) Cooperation of JB12 1–243 with Hsc70 in suppression of heat-denatured luciferase (LUC). Reactions were heated at 42ºC for 10 min followed by centrifugation to isolate the supernatant (S) and pellet (P) fractions from the total (T) sample. (G) Purified JB12 1–243 is unable to cooperate with Hsc70 to refold denatured luciferase. Luciferase was denatured in 6 M guanidinium HCl and then diluted into samples containing the indicated purified protein combinations. The folding reactions were incubated for 1 h at 25ºC, and luciferase activity was measured on a Turner luminometer. The amount of luciferase activity for each sample was normalized to the control sample, Hsc70 + Ydj1. The bar graph represents the average values obtained from two independent experiments.

In support of these data, JB12 cofractionates with the ER marker calnexin in HEK293 cells and behaves like an integral membrane protein in that it is not extracted from ER membrane with sodium carbonate (Figure 1C). Cytosolic Hsc70, but not the ER luminal protein BiP, is detected by Western blot in native JB12 immunoprecipitates. This detection supports the prediction that its J-domain resides in the cytosol (Figure 1D). The observed interaction of JB12 with Hsc70 was dependent on JB12’s J-domain because JB12-QPD does not coprecipitate with 35S-Hsc70 (Figure 1E).

JB12 can functionally interact with Hsc70 as a purified JB12 fragment (1–243), which contains the J-domain but lacks the transmembrane and ER luminal segments, stimulates Hsc70 ATPase activity from 5 to 15 nm ATP hydrolyzed · mg Hsc70 · min. This functional interaction is J-domain dependent as rates of ATP hydrolysis by Hsc70 in the presence of JB12-QPD 1–243 do not increase (unpublished data). Purified JB12 1–243 also cooperates with purified Hsc70 to suppress the aggregation of thermally denatured luciferase (Figure 1F). JB12 lacks conserved regions in Hsp40s known to bind soluble non-native polypeptides (Fan et al., 2004). This appears to explain why JB12 was incapable of acting independently to suppress luciferase aggregation (Figure 1F). Lack of intrinsic chaperone activity toward water-soluble proteins may also explain why JB12 1–243 is unable to act like the Type I Hsp40 Ydj1 and cooperate with Hsc70 to refold denatured luciferase (Figure 1G). These collective data indicate that JB12 is an ER-localized Hsp40 that uses its J-domain to regulate Hsc70s ATPase activity and enhance Hsc70 chaperone activity.

JB12 mediates proteasome-dependent CFTR degradationDue to JB12’s localization to the ER, we investigated whether JB12 interacts with Hsc70 to facilitate CFTR and CFTRΔF508 folding and/or degradation. Increasing JB12 levels prevented accumulation of the ER-localized B-form and the maturely glycosylated and plasma membrane–localized C-form of CFTR (Figure 2A). Elevation of JB12 also caused similar decreases in the accumulation of the B-form of CFTRΔF508. Overexpressed JB12-QPD only moderately decreased CFTR folding and CFTRΔF508 accumulation, suggesting that JB12’s action appears to require Hsc70. Structure function analysis shows that JB12 requires association with the ER membrane to impact CFTR levels because deletion of its transmembrane domain rendered it inactive (Figure 2B). The N- and C-terminal domains of JB12 that flank the J-domain and transmembrane region, however, are not required for overexpressed JB12 to partition CFTR out of its folding pathway (Figure 2B). The results are evident when comparing CFTR levels under conditions where JB12 1–243, JB12 1–264, and JB12 112–375 are overexpressed to similar levels as full-length JB12 (Figure 2B). Thus overexpressed JB12 uses its J-domain and transmembrane domain to decrease the accumulation of CFTRΔF508 and CFTR.FIGURE 2:JB12 mediates proteasome-dependent CFTR degradation. (A) JB12 overexpression decreases CFTR accumulation as detected by Western blots of HEK293 cell extracts that were transiently transfected with indicated combinations of expression plasmids for CFTR, CFTRΔF508, JB12-myc, or JB12-QPD-myc. (B) JB12-mediated degradation of CFTR requires the transmembrane domain but not the N- or C-terminal extensions. HEK293 cells transiently transfected with pcDNA3.1(+)-CFTR (1 μg) and increasing amounts of pcDNA3.1(+)-JB12-myc or pcDNA3.1(+)-JB12-myc truncation mutants (which ranged from 5 to 75 ng) were isolated and subjected to Western blot analysis with the indicated antibodies. (C) JB12-mediated decrease in CFTR accumulation is sensitive to the proteasome inhibitor bortezomib (10 μM) that was added 4 h prior to analyzing the samples by Western blot. (D) Ubiquitinated forms of CFTR and CFTRΔF508 accumulate when their JB12-stimulated degradation is hindered via expression of a dominant negative form of p97, p97QQ. CFTR or CFTRΔF508 was immunoprecipitated from extracts of cells that express the indicated proteins. Ubiquitinated CFTR or CFTR in immunoprecipitated material was detected by Western blot. When p97QQ interfered with the ERAD pathway and JB12 was overexpressed, the level of ubiquitinated CFTR detected was 92% of the quantity that accumulated when only p97QQ was expressed. Levels of ubiquitinated CFTRΔF508 in the presence of JB12 and p97QQ were 45% of levels detected when only p97QQ was expressed, and this value reflected the relative level of CFTRΔF508 under the indicated experimental condition. (See Results for an explanation.) Where indicated, bands B and C represent the immaturely glycosylated and maturely glycosylated forms of CFTR, respectively. Loading consistency is indicated relative to levels of tubulin in different panels.The reduction of total CFTR accumulation resulting from JB12 overexpression is proteasome dependent as it is blocked by the proteasome inhibitor bortezomib (Figure 2C). When JB12 is overexpressed and the proteasome is inhibited, severalfold more of the B-form of CFTR and CFTRΔF508 accumulates and a portion of the B-form migrates on gels as a low-mobility smear reminiscent of a polyubiquitinated species. Thus JB12 appears to reduce accumulation of the C-form of CFTR and the B-form of CFTRΔF508 via a proteasome-dependent pathway.When the ER-associated protein degradation (ERAD) pathway was inhibited by overexpression of a dominant negative form of the ER retrotranslocation factor p97 (p97QQ), inhibition of JB12-dependent degradation of CFTR and CFTRΔF508 was also observed (Figure 2D). In addition, the degradation intermediates of the B-form of CFTR and CFTRΔF508 that p97QQ drove to accumulate were demonstrated via immunoprecipitation (IP) and Western blot analysis to be ubiquitinated. p97QQ is a dominant negative form of p97 that binds (but cannot release) substrates, and blocks extraction of ubiquitinated proteins from the ER (Hirsch et al., 2009). Yet p97 is an abundant protein, so its function in CFTR degradation was not blocked completely by p97QQ overexpression, and p97QQ was not as effective as bortezomib in blocking JB12-dependent CFTR degradation.In the presence of p97QQ and JB12, levels of CFTRΔF508 were not restored to those observed in the absence of JB12. In contrast, when bortezomib was used to inhibit the proteasome in the presence of JB12, CFTRΔF508 levels were dramatically increased and levels of CFTRΔF508 were similar to those detected in bortezomib-treated cells in which JB12 was not elevated (Figure 2C). Therefore proper function of p97 may be required to stabilize the B-form of CFTRΔF508 during the process of JB12-dependent proteasomal degradation, but this hypothesis remains to be tested.

JB12 mediates proteasome-dependent CFTR degradation

Due to JB12’s localization to the ER, we investigated whether JB12 interacts with Hsc70 to facilitate CFTR and CFTRΔF508 folding and/or degradation. Increasing JB12 levels prevented accumulation of the ER-localized B-form and the maturely glycosylated and plasma membrane–localized C-form of CFTR (Figure 2A). Elevation of JB12 also caused similar decreases in the accumulation of the B-form of CFTRΔF508. Overexpressed JB12-QPD only moderately decreased CFTR folding and CFTRΔF508 accumulation, suggesting that JB12’s action appears to require Hsc70. Structure function analysis shows that JB12 requires association with the ER membrane to impact CFTR levels because deletion of its transmembrane domain rendered it inactive (Figure 2B). The N- and C-terminal domains of JB12 that flank the J-domain and transmembrane region, however, are not required for overexpressed JB12 to partition CFTR out of its folding pathway (Figure 2B). The results are evident when comparing CFTR levels under conditions where JB12 1–243, JB12 1–264, and JB12 112–375 are overexpressed to similar levels as full-length JB12 (Figure 2B). Thus overexpressed JB12 uses its J-domain and transmembrane domain to decrease the accumulation of CFTRΔF508 and CFTR.

JB12 mediates proteasome-dependent CFTR degradation. (A) JB12 overexpression decreases CFTR accumulation as detected by Western blots of HEK293 cell extracts that were transiently transfected with indicated combinations of expression plasmids for CFTR, CFTRΔF508, JB12-myc, or JB12-QPD-myc. (B) JB12-mediated degradation of CFTR requires the transmembrane domain but not the N- or C-terminal extensions. HEK293 cells transiently transfected with pcDNA3.1(+)-CFTR (1 μg) and increasing amounts of pcDNA3.1(+)-JB12-myc or pcDNA3.1(+)-JB12-myc truncation mutants (which ranged from 5 to 75 ng) were isolated and subjected to Western blot analysis with the indicated antibodies. (C) JB12-mediated decrease in CFTR accumulation is sensitive to the proteasome inhibitor bortezomib (10 μM) that was added 4 h prior to analyzing the samples by Western blot. (D) Ubiquitinated forms of CFTR and CFTRΔF508 accumulate when their JB12-stimulated degradation is hindered via expression of a dominant negative form of p97, p97QQ. CFTR or CFTRΔF508 was immunoprecipitated from extracts of cells that express the indicated proteins. Ubiquitinated CFTR or CFTR in immunoprecipitated material was detected by Western blot. When p97QQ interfered with the ERAD pathway and JB12 was overexpressed, the level of ubiquitinated CFTR detected was 92% of the quantity that accumulated when only p97QQ was expressed. Levels of ubiquitinated CFTRΔF508 in the presence of JB12 and p97QQ were 45% of levels detected when only p97QQ was expressed, and this value reflected the relative level of CFTRΔF508 under the indicated experimental condition. (See Results for an explanation.) Where indicated, bands B and C represent the immaturely glycosylated and maturely glycosylated forms of CFTR, respectively. Loading consistency is indicated relative to levels of tubulin in different panels.

The reduction of total CFTR accumulation resulting from JB12 overexpression is proteasome dependent as it is blocked by the proteasome inhibitor bortezomib (Figure 2C). When JB12 is overexpressed and the proteasome is inhibited, severalfold more of the B-form of CFTR and CFTRΔF508 accumulates and a portion of the B-form migrates on gels as a low-mobility smear reminiscent of a polyubiquitinated species. Thus JB12 appears to reduce accumulation of the C-form of CFTR and the B-form of CFTRΔF508 via a proteasome-dependent pathway.

When the ER-associated protein degradation (ERAD) pathway was inhibited by overexpression of a dominant negative form of the ER retrotranslocation factor p97 (p97QQ), inhibition of JB12-dependent degradation of CFTR and CFTRΔF508 was also observed (Figure 2D). In addition, the degradation intermediates of the B-form of CFTR and CFTRΔF508 that p97QQ drove to accumulate were demonstrated via immunoprecipitation (IP) and Western blot analysis to be ubiquitinated. p97QQ is a dominant negative form of p97 that binds (but cannot release) substrates, and blocks extraction of ubiquitinated proteins from the ER (Hirsch et al., 2009). Yet p97 is an abundant protein, so its function in CFTR degradation was not blocked completely by p97QQ overexpression, and p97QQ was not as effective as bortezomib in blocking JB12-dependent CFTR degradation.

In the presence of p97QQ and JB12, levels of CFTRΔF508 were not restored to those observed in the absence of JB12. In contrast, when bortezomib was used to inhibit the proteasome in the presence of JB12, CFTRΔF508 levels were dramatically increased and levels of CFTRΔF508 were similar to those detected in bortezomib-treated cells in which JB12 was not elevated (Figure 2C). Therefore proper function of p97 may be required to stabilize the B-form of CFTRΔF508 during the process of JB12-dependent proteasomal degradation, but this hypothesis remains to be tested.

Knockdown of endogenous JB12 increases CFTR folding efficiencyTo evaluate the role of endogenous JB12 in CFTR biogenesis, siRNA experiments were performed and the effect that kd of JB12 had on biosynthetic maturation of CFTR and CFTRΔF508 in HEK293 cells was evaluated by pulse-chase and Western blot analysis (Figure 3, A and B). Endogenous JB12 levels were reduced by >90% in HEK293 cells, and the pair of siRNAs used were specific in that the individual siRNAs were both capable of independently performing this task (unpublished data). JB12 kd also had no detectable impact on the levels of other QC factors that modulate CFTR folding or degradation (Supplemental Figure S1). Data from pulse-chase studies show that reduction in JB12 levels leads to a dramatic threefold increase in the quantity of nascent B-form CFTR that is converted to the C-form by the end of the pulse-chase time course. Likewise, the half-life of CFTRΔF508 was increased upon JB12 kd, but accumulation of its folded C-form was not detected.FIGURE 3:Knockdown of endogenous JB12 increases CFTR folding efficiency. Pulse chase (A) and Western blot (B) analyses indicate that the folding efficiency of CFTR is dramatically enhanced whereas the stability of CFTRΔF508 is modestly increased in JB12 siRNA HEK293 cells. The folding efficiency of CFTR increases from approximately 15 ± 2% to 45 ± 3% of the B-form at t = 0 upon kd of JB12 (n = 3). Under control conditions (si-Cont), the half-life of CFTRΔF508 was 40 min (SD ± 2 min) and in the absence of JB12 increased 1.5-fold to 60 min (SD ± 2 min; n = 3). The band marked with an * is a background band in the IP reactions in (B). (C) JB12 kd in human CFBE cells also increases accumulation of CFTR and CFTRΔF508. The immaturely glycosylated B-form and maturely glycosylated C-form of CFTR are indicated. Results were quantified by densitometry and were either normalized to the relative amount of CFTR B-band at t = 0 for each condition (A) or the CFTR B- and C-band levels were normalized to the si-Cont reactions (B and C).When steady-state levels of CFTR were assayed 72 h after JB12 kd, severalfold more of the C-form accumulated, and this accumulation appeared to be due to increased folding efficiency as it was not accompanied by a corresponding increase in the B-form levels. We could also see an increase in the accumulation of the B-form of CFTRΔF508, but the effects of JB12 kd on CFTRΔF508 were less dramatic than those of CFTR.To demonstrate that the function of JB12 in CFTR biogenesis in HEK293 cells is relevant to what occurs in lung epithelial cells, the impact of JB12 kd on CFTR and CFTRΔF508 biogenesis in immortalized human bronchial epithelial (CFBE) cells was determined (Figure 3C). JB12 levels could be reduced by >70% in CFBE cells, and this reduction was accompanied by a severalfold increase in accumulation of the C-form of CFTR. Again, we also observed JB12 kd to permit increased accumulation of the B-form of CFTRΔF508, but the C-form was difficult to detect.Changes in JB12 levels correlate inversely with CFTR and CFTRΔF508 levels in both HEK293 and CFBE cells. CFTR folding is dramatically increased by JB12 kd, but increases in folded CFTRΔF508 are less dramatic. Thus JB12 action appears to limit CFTR folding efficiency, whereas other cellular factors or kinetic folding defects limit the folding of CFTRΔF508. The effect of CFTRΔF508 behavior upon JB12 kd fits with the concept that the majority of CFTRΔF508 accumulates in a kinetically trapped non-native state that is difficult to bring back onto a folding pathway (Younger et al., 2004).

Knockdown of endogenous JB12 increases CFTR folding efficiency

To evaluate the role of endogenous JB12 in CFTR biogenesis, siRNA experiments were performed and the effect that kd of JB12 had on biosynthetic maturation of CFTR and CFTRΔF508 in HEK293 cells was evaluated by pulse-chase and Western blot analysis (Figure 3, A and B). Endogenous JB12 levels were reduced by >90% in HEK293 cells, and the pair of siRNAs used were specific in that the individual siRNAs were both capable of independently performing this task (unpublished data). JB12 kd also had no detectable impact on the levels of other QC factors that modulate CFTR folding or degradation (Supplemental Figure S1). Data from pulse-chase studies show that reduction in JB12 levels leads to a dramatic threefold increase in the quantity of nascent B-form CFTR that is converted to the C-form by the end of the pulse-chase time course. Likewise, the half-life of CFTRΔF508 was increased upon JB12 kd, but accumulation of its folded C-form was not detected.

Knockdown of endogenous JB12 increases CFTR folding efficiency. Pulse chase (A) and Western blot (B) analyses indicate that the folding efficiency of CFTR is dramatically enhanced whereas the stability of CFTRΔF508 is modestly increased in JB12 siRNA HEK293 cells. The folding efficiency of CFTR increases from approximately 15 ± 2% to 45 ± 3% of the B-form at t = 0 upon kd of JB12 (n = 3). Under control conditions (si-Cont), the half-life of CFTRΔF508 was 40 min (SD ± 2 min) and in the absence of JB12 increased 1.5-fold to 60 min (SD ± 2 min; n = 3). The band marked with an * is a background band in the IP reactions in (B). (C) JB12 kd in human CFBE cells also increases accumulation of CFTR and CFTRΔF508. The immaturely glycosylated B-form and maturely glycosylated C-form of CFTR are indicated. Results were quantified by densitometry and were either normalized to the relative amount of CFTR B-band at t = 0 for each condition (A) or the CFTR B- and C-band levels were normalized to the si-Cont reactions (B and C).

When steady-state levels of CFTR were assayed 72 h after JB12 kd, severalfold more of the C-form accumulated, and this accumulation appeared to be due to increased folding efficiency as it was not accompanied by a corresponding increase in the B-form levels. We could also see an increase in the accumulation of the B-form of CFTRΔF508, but the effects of JB12 kd on CFTRΔF508 were less dramatic than those of CFTR.

To demonstrate that the function of JB12 in CFTR biogenesis in HEK293 cells is relevant to what occurs in lung epithelial cells, the impact of JB12 kd on CFTR and CFTRΔF508 biogenesis in immortalized human bronchial epithelial (CFBE) cells was determined (Figure 3C). JB12 levels could be reduced by >70% in CFBE cells, and this reduction was accompanied by a severalfold increase in accumulation of the C-form of CFTR. Again, we also observed JB12 kd to permit increased accumulation of the B-form of CFTRΔF508, but the C-form was difficult to detect.

Changes in JB12 levels correlate inversely with CFTR and CFTRΔF508 levels in both HEK293 and CFBE cells. CFTR folding is dramatically increased by JB12 kd, but increases in folded CFTRΔF508 are less dramatic. Thus JB12 action appears to limit CFTR folding efficiency, whereas other cellular factors or kinetic folding defects limit the folding of CFTRΔF508. The effect of CFTRΔF508 behavior upon JB12 kd fits with the concept that the majority of CFTRΔF508 accumulates in a kinetically trapped non-native state that is difficult to bring back onto a folding pathway (Younger et al., 2004).

Knockdown of JB12 and Hdj-2 have opposite effects on CFTR foldingJB12 and the cytosolic Type I Hsp40 Hdj-2 (Meacham et al., 1999) both act on the cytoplasmic face of the ER but appear to differentially direct Hsc70 to promote the life or death of CFTR. If this interpretation is true, then Hdj-2 kd should hinder CFTR folding and accelerate degradation of the B-form of CFTR and CFTRΔF508. To test this supposition, Hdj-2 levels were reduced by small hairpin RNA (shRNA) and the impact that this reduction had on CFTR levels was determined. Hdj-2 is highly abundant, but shRNAs were able to reduce its levels by ∼70%, which was associated with a 70% reduction in accumulation of the folded C-form of CFTR and a 50% reduction in accumulation of the immature B-form of CFTRΔF508 (Figure 4A). Data from pulse-chase studies suggest that decreased accumulation of folded CFTR detected by Western blot when Hdj-2 levels are low results from decreased folding and stability of the nascent B-form (Figure 4B). Hdj-2 kd decreased CFTR folding efficiency severalfold and decreased stability of the nascent B-form of CFTRΔF508. These data support the notion that Hdj-2 and JB12 direct Hsc70 to play opposing roles in CFTR folding and degradation, respectively.FIGURE 4:Knockdown of JB12 and Hdj-2 has opposite effects on CFTR folding. (A and B) Hdj-2 kd decreases folding and stability of CFTR. Hdj-2 shRNA (sh-Hdj2) or cont-shRNA (sh-Cont) HEK293 cells transfected with pcDNA3.1(+)-CFTR or pcDNA3.1(+)-CFTRΔF508 were collected for Western blot analysis of CFTR steady-state levels (A) or 35S-labeled for CFTR pulse-chase analysis (B). The steady-state levels (C) and half-life (D) of either ApoB or TCRα are not affected by JB12 siRNA (si-JB12). Results that were quantified by densitometry were either normalized to the control reactions (A and C) or normalized to the relative amount of 35S-labeled material at t = 0 for each condition (B and D).Modulating JB12 levels does not appear to cause gross changes in ER protein QC activity in HEK293 cells because JB12 kd does not impact degradation of two different topologically distinct ERAD substrates (Figure 4, C and D). TCRα is a subunit of the T-cell receptor, has a large luminal domain and single transmembrane domain, and is degraded rapidly via ERAD when expressed alone because it lacks assembly partners (Tiwari and Weissman, 2001). ApoB48 is inserted across the ER membrane and is an Hsc70 client but is degraded rapidly in HEK293 cells because they don’t express specialized folding factors required for proper biogenesis (Fisher et al., 1997). JB12 kd does not alter the steady-state levels of TCRα or ApoB48 (Figure 4C), and pulse-chase studies show that JB12 kd does not alter the biosynthesis or half-life of either (Figure 4D). Thus stabilization of the B-form of CFTR upon JB12 kd is not associated with a general stabilization of other ERAD substrates that have different domain structures.

Knockdown of JB12 and Hdj-2 have opposite effects on CFTR folding

JB12 and the cytosolic Type I Hsp40 Hdj-2 (Meacham et al., 1999) both act on the cytoplasmic face of the ER but appear to differentially direct Hsc70 to promote the life or death of CFTR. If this interpretation is true, then Hdj-2 kd should hinder CFTR folding and accelerate degradation of the B-form of CFTR and CFTRΔF508. To test this supposition, Hdj-2 levels were reduced by small hairpin RNA (shRNA) and the impact that this reduction had on CFTR levels was determined. Hdj-2 is highly abundant, but shRNAs were able to reduce its levels by ∼70%, which was associated with a 70% reduction in accumulation of the folded C-form of CFTR and a 50% reduction in accumulation of the immature B-form of CFTRΔF508 (Figure 4A). Data from pulse-chase studies suggest that decreased accumulation of folded CFTR detected by Western blot when Hdj-2 levels are low results from decreased folding and stability of the nascent B-form (Figure 4B). Hdj-2 kd decreased CFTR folding efficiency severalfold and decreased stability of the nascent B-form of CFTRΔF508. These data support the notion that Hdj-2 and JB12 direct Hsc70 to play opposing roles in CFTR folding and degradation, respectively.

Knockdown of JB12 and Hdj-2 has opposite effects on CFTR folding. (A and B) Hdj-2 kd decreases folding and stability of CFTR. Hdj-2 shRNA (sh-Hdj2) or cont-shRNA (sh-Cont) HEK293 cells transfected with pcDNA3.1(+)-CFTR or pcDNA3.1(+)-CFTRΔF508 were collected for Western blot analysis of CFTR steady-state levels (A) or 35S-labeled for CFTR pulse-chase analysis (B). The steady-state levels (C) and half-life (D) of either ApoB or TCRα are not affected by JB12 siRNA (si-JB12). Results that were quantified by densitometry were either normalized to the control reactions (A and C) or normalized to the relative amount of 35S-labeled material at t = 0 for each condition (B and D).

Modulating JB12 levels does not appear to cause gross changes in ER protein QC activity in HEK293 cells because JB12 kd does not impact degradation of two different topologically distinct ERAD substrates (Figure 4, C and D). TCRα is a subunit of the T-cell receptor, has a large luminal domain and single transmembrane domain, and is degraded rapidly via ERAD when expressed alone because it lacks assembly partners (Tiwari and Weissman, 2001). ApoB48 is inserted across the ER membrane and is an Hsc70 client but is degraded rapidly in HEK293 cells because they don’t express specialized folding factors required for proper biogenesis (Fisher et al., 1997). JB12 kd does not alter the steady-state levels of TCRα or ApoB48 (Figure 4C), and pulse-chase studies show that JB12 kd does not alter the biosynthesis or half-life of either (Figure 4D). Thus stabilization of the B-form of CFTR upon JB12 kd is not associated with a general stabilization of other ERAD substrates that have different domain structures.

JB12 drives Hsc70 to associate with CFTR and the RMA1 E3 complexTo explore the mechanism for JB12 action, its presence in complexes that contain RMA1 and/or CHIP was explored (Figure 5A). Protein–protein interactions were evaluated by asking whether JB12 and Hsc70 coprecipitate under native buffer conditions with either RMA1 or CHIP. JB12 and Hsc70 were pulled down with RMA1, but CHIP was not detected in complexes with JB12. In addition, elevation of JB12 levels increased the quantity of Hsc70 that coprecipitated with flag-RMA1 (Figure 5A).FIGURE 5:JB12 drives Hsc70 to associate with CFTR and the RMA1 E3 complex. (A) JB12 and Hsc70 coprecipitate with the RMA1 E3 ligase, but the E3 ligase CHIP was not observed in a complex with JB12. Flag-RMA1 was immunoprecipitated from soluble cell lysates prepared with PBS-Triton (1%), and both Hsc70 and JB12-myc were found in association with RMA1-containing complexes. In contrast, IP of HA-JB12 from soluble cell lysates isolated Hsc70, but not CHIP-myc, in JB12-containing complexes. The presence of indicated proteins in native immunoprecipitates was determined by Western blot. (B) Elevation of JB12 increases the quantity of 35S-Hsc70 that associates with CFTRΔF508 in a nucleotide-dependent manner. HEK 293 cells transfected with pcDNA3.1(+)-CFTRΔF508 and pcDNA3.1(+)-JB12-myc or pcDNA3.1(+)-JB12-QPD-myc, as indicated, were labeled with 35S-methinone, and co- and re-IP reactions were conducted as described in Materials and Methods. Briefly, the samples were lysed in PBS-Triton (1%) with either an ATP regeneration system (+ ATP) or with apyrase (- ATP), and CFTR was immunoprecipitated from the soluble cell lysates. The reactions were then subjected to a secondary re-IP with α-Hsc70 antibody. Western blots indicate the levels of CFTRΔF508 when JB12 or JB12-QPD is overexpressed. (C) Hsc70, flag-RMA1, Derlin-1, and CFTRΔF508 coprecipitate with HA-JB12. HEK 293 cells overexpressing HA-JB12, flag-RMA1, Derlin-1, and CFTRΔF508 were labeled with 35S-methinone and solubilized with PBS-Triton (1%). Co-IP and re-IP reactions were conducted as described in Materials and Methods. In general, HA-JB12 was isolated from the cell lysates, and secondary re-IP steps identified flag-RMA1, Derlin-1, endogenous Hsc70, and CFTRΔF508 as being part of a JB12-containing complex. A 3-d film exposure was sufficient to visualize proteins bound in initial co-IP reactions (C), whereas a 7-d exposure was necessary to visualize bands in the reimmunoprecipitated samples (B and C).Next we evaluated the impact that elevation of JB12 levels had on Hsc70 binding to nascent 35S-CFTRΔF508 (Figure 5B). CFTRΔF508 was immunoprecipitated from native cell extracts in the presence or absence of overexpressed JB12. The Hsc70 present with CFTRΔF508 in initial precipitates was determined by reimmunoprecipitation (re-IP). The quantity of 35S-Hsc70 coprecipitated with CFTRΔF508 when ATP levels were elevated was low, but an increase was detected upon expression of HA-JB12. This finding is impressive as the quantity of CFTRΔF508 in cells was significantly lower when JB12 was elevated. Depletion of ATP resulted in a more dramatic, approximately 10-fold increase in JB12-dependent Hsc70 binding to the B-form of CFTRΔF508 (Figure 5B, bottom panel). HA-JB12-QPD expression had little impact on accumulation of the B-form of CFTRΔF508 and did not cause an increase in Hsc70 binding to CFTRΔF508. Similar results were observed with the B-form of CFTR (unpublished data). These data indicate that JB12 acts in an ATP- and J-domain–dependent manner to increase association of Hsc70 with complexes that contain the nascent B-form of CFTR.JB12 is present in complexes that contain RMA1 and can increase Hsc70 binding to the B-form of CFTR and CFTRΔF508. Thus JB12 appears to act with RMA1 in degradation of the misfolded B-form of CFTR and CFTRΔF508. If so, JB12 should be present in complexes that contain CFTRΔF508 as well as RMA1, Hsc70, and Derlin-1 (Younger et al., 2006). This supposition was tested in 35S-labeled cells that expressed HA-JB12, flag-RMA1, Derlin-1, and CFTRΔF508 and that were lysed under native conditions (Figure 5C). HA-JB12 was immunoprecipitated, and the ERQC factors of interest were identified by re-IP. A large number of radiolabeled proteins the sizes of which spanned the range of molecular weights resolved on a 12.5% gel were found to coprecipitate HA-JB12 but not with the beads used in precipitations. The large number of bands associated with JB12 is consistent with it functioning with Hsc70 to facilitate the biogenesis of a range of substrates. Re-IPs identified Hsc70, RMA1, Derlin-1, and CFTRΔF508 in precipitates with JB12 (Figure 5C). We performed similar experiments with CHIP-myc but never detected the overexpressed or endogenous CHIP in a complex with JB12 (unpublished data).Of note is that in each of the re-IPs with the indicated antibodies a quantity of JB12 was present. The proteins of interest are membrane proteins that aggregate at elevated temperatures, so samples were heated at 55ºC, instead of 100ºC, before the re-IP step to disrupt interactions between JB12 and the antibody used for initial precipitations. In the absence of boiling, however, this process is incomplete; thus JB12 is present in the Hsc70, RMA1, and Derlin-1 re-IPs. To complicate things further, CFTRΔF508 can aggregate when heated >37ºC in an SDS sample buffer, so the samples used for re-IPs of CFTR were incubated at 37ºC instead of 55ºC. Therefore the original antibody–JB12 interactions were not disrupted, and we observed more JB12 in the CFTRΔF508 re-IPs than in other re-IPs. Controls demonstrate, however, that the presence of Hsc70, RMA1, Derlin-1, and CFTRΔF508 in re-IPs occurs only if antibodies specific to these individual proteins are present.Derlin-1 interacts with multiple E3 ubiquitin ligases (Hirsch et al., 2009), and yeast JB12 is found to cooperate with Hsc70 Ssa1 and the ER-associated E3 Doa10 in the degradation of P-type ATPases and other model ERAD substrates (Han et al., 2007; Nakatsukasa et al., 2008). Thus we explored the specificity of interactions between JB12 with Derlin-1 and RMA1 via determination of whether JB12 could coprecipitate in complexes with the following ER-localized E3 ubiquitin ligases: Gp78 (Morito et al., 2008), March V (Bartee et al., 2004), HRD1 (Hirsch et al., 2009), and SEL1 (Hirsch et al., 2009) (Supplemental Figure S2). Association of JB12 with Gp78, a known interaction partner of RMA1, was detected (Morito et al., 2008), but complexes with MarchV, HRD1, or SEL1 were not observed. These data suggest that JB12 selectively interacts with a subset of ER-associated E3 ligases (Supplemental Figure S2). Overall, data obtained on protein–protein interactions with JB12 demonstrate that it selectively associates with components of the RMA1 E3 complex and can facilitate the association of Hsc70 with nascent CFTR/CFTRΔF508 and RMA1.

JB12 drives Hsc70 to associate with CFTR and the RMA1 E3 complex

To explore the mechanism for JB12 action, its presence in complexes that contain RMA1 and/or CHIP was explored (Figure 5A). Protein–protein interactions were evaluated by asking whether JB12 and Hsc70 coprecipitate under native buffer conditions with either RMA1 or CHIP. JB12 and Hsc70 were pulled down with RMA1, but CHIP was not detected in complexes with JB12. In addition, elevation of JB12 levels increased the quantity of Hsc70 that coprecipitated with flag-RMA1 (Figure 5A).

JB12 drives Hsc70 to associate with CFTR and the RMA1 E3 complex. (A) JB12 and Hsc70 coprecipitate with the RMA1 E3 ligase, but the E3 ligase CHIP was not observed in a complex with JB12. Flag-RMA1 was immunoprecipitated from soluble cell lysates prepared with PBS-Triton (1%), and both Hsc70 and JB12-myc were found in association with RMA1-containing complexes. In contrast, IP of HA-JB12 from soluble cell lysates isolated Hsc70, but not CHIP-myc, in JB12-containing complexes. The presence of indicated proteins in native immunoprecipitates was determined by Western blot. (B) Elevation of JB12 increases the quantity of 35S-Hsc70 that associates with CFTRΔF508 in a nucleotide-dependent manner. HEK 293 cells transfected with pcDNA3.1(+)-CFTRΔF508 and pcDNA3.1(+)-JB12-myc or pcDNA3.1(+)-JB12-QPD-myc, as indicated, were labeled with 35S-methinone, and co- and re-IP reactions were conducted as described in Materials and Methods. Briefly, the samples were lysed in PBS-Triton (1%) with either an ATP regeneration system (+ ATP) or with apyrase (- ATP), and CFTR was immunoprecipitated from the soluble cell lysates. The reactions were then subjected to a secondary re-IP with α-Hsc70 antibody. Western blots indicate the levels of CFTRΔF508 when JB12 or JB12-QPD is overexpressed. (C) Hsc70, flag-RMA1, Derlin-1, and CFTRΔF508 coprecipitate with HA-JB12. HEK 293 cells overexpressing HA-JB12, flag-RMA1, Derlin-1, and CFTRΔF508 were labeled with 35S-methinone and solubilized with PBS-Triton (1%). Co-IP and re-IP reactions were conducted as described in Materials and Methods. In general, HA-JB12 was isolated from the cell lysates, and secondary re-IP steps identified flag-RMA1, Derlin-1, endogenous Hsc70, and CFTRΔF508 as being part of a JB12-containing complex. A 3-d film exposure was sufficient to visualize proteins bound in initial co-IP reactions (C), whereas a 7-d exposure was necessary to visualize bands in the reimmunoprecipitated samples (B and C).

Next we evaluated the impact that elevation of JB12 levels had on Hsc70 binding to nascent 35S-CFTRΔF508 (Figure 5B). CFTRΔF508 was immunoprecipitated from native cell extracts in the presence or absence of overexpressed JB12. The Hsc70 present with CFTRΔF508 in initial precipitates was determined by reimmunoprecipitation (re-IP). The quantity of 35S-Hsc70 coprecipitated with CFTRΔF508 when ATP levels were elevated was low, but an increase was detected upon expression of HA-JB12. This finding is impressive as the quantity of CFTRΔF508 in cells was significantly lower when JB12 was elevated. Depletion of ATP resulted in a more dramatic, approximately 10-fold increase in JB12-dependent Hsc70 binding to the B-form of CFTRΔF508 (Figure 5B, bottom panel). HA-JB12-QPD expression had little impact on accumulation of the B-form of CFTRΔF508 and did not cause an increase in Hsc70 binding to CFTRΔF508. Similar results were observed with the B-form of CFTR (unpublished data). These data indicate that JB12 acts in an ATP- and J-domain–dependent manner to increase association of Hsc70 with complexes that contain the nascent B-form of CFTR.

JB12 is present in complexes that contain RMA1 and can increase Hsc70 binding to the B-form of CFTR and CFTRΔF508. Thus JB12 appears to act with RMA1 in degradation of the misfolded B-form of CFTR and CFTRΔF508. If so, JB12 should be present in complexes that contain CFTRΔF508 as well as RMA1, Hsc70, and Derlin-1 (Younger et al., 2006). This supposition was tested in 35S-labeled cells that expressed HA-JB12, flag-RMA1, Derlin-1, and CFTRΔF508 and that were lysed under native conditions (Figure 5C). HA-JB12 was immunoprecipitated, and the ERQC factors of interest were identified by re-IP. A large number of radiolabeled proteins the sizes of which spanned the range of molecular weights resolved on a 12.5% gel were found to coprecipitate HA-JB12 but not with the beads used in precipitations. The large number of bands associated with JB12 is consistent with it functioning with Hsc70 to facilitate the biogenesis of a range of substrates. Re-IPs identified Hsc70, RMA1, Derlin-1, and CFTRΔF508 in precipitates with JB12 (Figure 5C). We performed similar experiments with CHIP-myc but never detected the overexpressed or endogenous CHIP in a complex with JB12 (unpublished data).

Of note is that in each of the re-IPs with the indicated antibodies a quantity of JB12 was present. The proteins of interest are membrane proteins that aggregate at elevated temperatures, so samples were heated at 55ºC, instead of 100ºC, before the re-IP step to disrupt interactions between JB12 and the antibody used for initial precipitations. In the absence of boiling, however, this process is incomplete; thus JB12 is present in the Hsc70, RMA1, and Derlin-1 re-IPs. To complicate things further, CFTRΔF508 can aggregate when heated >37ºC in an SDS sample buffer, so the samples used for re-IPs of CFTR were incubated at 37ºC instead of 55ºC. Therefore the original antibody–JB12 interactions were not disrupted, and we observed more JB12 in the CFTRΔF508 re-IPs than in other re-IPs. Controls demonstrate, however, that the presence of Hsc70, RMA1, Derlin-1, and CFTRΔF508 in re-IPs occurs only if antibodies specific to these individual proteins are present.

Derlin-1 interacts with multiple E3 ubiquitin ligases (Hirsch et al., 2009), and yeast JB12 is found to cooperate with Hsc70 Ssa1 and the ER-associated E3 Doa10 in the degradation of P-type ATPases and other model ERAD substrates (Han et al., 2007; Nakatsukasa et al., 2008). Thus we explored the specificity of interactions between JB12 with Derlin-1 and RMA1 via determination of whether JB12 could coprecipitate in complexes with the following ER-localized E3 ubiquitin ligases: Gp78 (Morito et al., 2008), March V (Bartee et al., 2004), HRD1 (Hirsch et al., 2009), and SEL1 (Hirsch et al., 2009) (Supplemental Figure S2). Association of JB12 with Gp78, a known interaction partner of RMA1, was detected (Morito et al., 2008), but complexes with MarchV, HRD1, or SEL1 were not observed. These data suggest that JB12 selectively interacts with a subset of ER-associated E3 ligases (Supplemental Figure S2). Overall, data obtained on protein–protein interactions with JB12 demonstrate that it selectively associates with components of the RMA1 E3 complex and can facilitate the association of Hsc70 with nascent CFTR/CFTRΔF508 and RMA1.

RMA1 is required for JB12 to promote CFTR degradationOn the basis of data presented thus far (Figure 5), we suggest that JB12 functionally interacts with RMA1, but not CHIP, in degradation of misfolded forms of CFTR. This interpretation is based on the presence of JB12 in complexes with RMA1, Derlin-1, Hsc70, and CFTR and an inability to detect JB12 in complexes that contain Hsc70 and CHIP. If this is true, then RMA1, but not CHIP, should be required for JB12-dependent partitioning of nascent CFTR out of its folding pathway. To further evaluate the potential for functional interactions between CHIP and RMA1, we compared the impact of elevating JB12 approximately onefold over endogenous pools on CFTR biogenesis when either RMA1 or CHIP levels were depleted (Figure 6A). In cells treated with a control siRNA, elevation of JB12 reduced accumulation of the B- and C-forms of CFTR to a barely detectable level (Figure 6A). As predicted, kd of RMA1 hindered the ability of overexpressed JB12 to promote CFTR degradation. It is important to note, however, that in the absence of RMA1, JB12 still retained CFTR in the ER (Figure 6A, top panel). Similarly, when RMA1 is knocked down, overexpressed JB12 did not decrease the accumulation of CFTRΔF508 (Figure 6A, bottom panel). In contrast, JB12 was still fully capable of promoting degradation of CFTR and CFTRΔF508 when CHIP was depleted (Figure 6A).FIGURE 6:RMA1 is required for JB12 to promote CFTR degradation. (A) The effect that RMA1 or CHIP kd has on JB12-dependent decreases in CFTR and CFTRΔF508 accumulation as detected by Western blot. Antibodies directed against RMA1 and CHIP were used to detect the extent of kd of endogenous RMA1 and CHIP levels. (B) Impact of RMA1 and JB12 action alone and in combination on accumulation of CFTR and CFTRΔF508 levels. RMA1 and JB12 blots are included to show the level of overexpression of these proteins relative to their endogenous levels. Tubulin blots indicate loading consistency.Further support for the concept of functional interaction between JB12 and RMA1 in CFTR degradation comes from the observation that these proteins can synergize to perform this function (Figure 6B). Coexpressing JB12-myc and flag-RMA1 at levels where individually they have a modest effect on steady-state levels of CFTR (Figure 6B, top panel) or CFTRΔF508 (Figure 6B, bottom panel) leads to dramatic decreases in the folded C-form of CFTR and the B-form of CFTRΔF508. The decrease observed is blocked via treatment of cells with the proteasome inhibitor bortezomib, suggesting that JB12 and RMA1 cooperate in targeting CFTR for proteasomal degradation. The collective data from siRNA and overexpression studies, when combined with data from co-IP experiments, suggest that JB12 cooperates with Hsc70 to mediate RMA1-dependent CFTR degradation.

RMA1 is required for JB12 to promote CFTR degradation

On the basis of data presented thus far (Figure 5), we suggest that JB12 functionally interacts with RMA1, but not CHIP, in degradation of misfolded forms of CFTR. This interpretation is based on the presence of JB12 in complexes with RMA1, Derlin-1, Hsc70, and CFTR and an inability to detect JB12 in complexes that contain Hsc70 and CHIP. If this is true, then RMA1, but not CHIP, should be required for JB12-dependent partitioning of nascent CFTR out of its folding pathway. To further evaluate the potential for functional interactions between CHIP and RMA1, we compared the impact of elevating JB12 approximately onefold over endogenous pools on CFTR biogenesis when either RMA1 or CHIP levels were depleted (Figure 6A). In cells treated with a control siRNA, elevation of JB12 reduced accumulation of the B- and C-forms of CFTR to a barely detectable level (Figure 6A). As predicted, kd of RMA1 hindered the ability of overexpressed JB12 to promote CFTR degradation. It is important to note, however, that in the absence of RMA1, JB12 still retained CFTR in the ER (Figure 6A, top panel). Similarly, when RMA1 is knocked down, overexpressed JB12 did not decrease the accumulation of CFTRΔF508 (Figure 6A, bottom panel). In contrast, JB12 was still fully capable of promoting degradation of CFTR and CFTRΔF508 when CHIP was depleted (Figure 6A).

RMA1 is required for JB12 to promote CFTR degradation. (A) The effect that RMA1 or CHIP kd has on JB12-dependent decreases in CFTR and CFTRΔF508 accumulation as detected by Western blot. Antibodies directed against RMA1 and CHIP were used to detect the extent of kd of endogenous RMA1 and CHIP levels. (B) Impact of RMA1 and JB12 action alone and in combination on accumulation of CFTR and CFTRΔF508 levels. RMA1 and JB12 blots are included to show the level of overexpression of these proteins relative to their endogenous levels. Tubulin blots indicate loading consistency.

Further support for the concept of functional interaction between JB12 and RMA1 in CFTR degradation comes from the observation that these proteins can synergize to perform this function (Figure 6B). Coexpressing JB12-myc and flag-RMA1 at levels where individually they have a modest effect on steady-state levels of CFTR (Figure 6B, top panel) or CFTRΔF508 (Figure 6B, bottom panel) leads to dramatic decreases in the folded C-form of CFTR and the B-form of CFTRΔF508. The decrease observed is blocked via treatment of cells with the proteasome inhibitor bortezomib, suggesting that JB12 and RMA1 cooperate in targeting CFTR for proteasomal degradation. The collective data from siRNA and overexpression studies, when combined with data from co-IP experiments, suggest that JB12 cooperates with Hsc70 to mediate RMA1-dependent CFTR degradation.

JB12 monitors the folding status of CFTR N-terminal regionsChaperone-dependent steps in CFTR folding and degradation have been uncovered via analysis of the assembly of CFTR N- and C-terminal fragments (CFTR 837X and CFTR 837–1480) into a complex that escapes the ER and functions as a Cl- channel (Chan et al., 2000). Studies with CFTR 837X suggest that deletion of F508 causes a severe defect in formation of a folding intermediate that contains MSD1, NBD1, and the R-domain that is critical for the folding progression of CFTR (Rosser et al., 2008). Recognition of folding defects in amino-terminal regions of CFTRΔF508 by the RMA1 E3 complex is proposed to be a major cause of premature CFTR degradation (Younger et al., 2006; Rosser et al., 2008). Thus we sought to obtain support for the conclusion that JB12 participates with RMA1 and Derlin-1 in selecting CFTR for degradation by asking if JB12 associates with CFTR 837X and determining whether it facilitates CFTR 837X degradation (Figure 7).FIGURE 7:JB12 monitors the folding status of CFTR N-terminal regions. (A) Endogenous JB12 interacts with the N-terminal half of CFTR. HEK293 cells transfected with either pcDNA3.1(+)-CFTR 837X or pcDNA3.1(+)-CFTR 837–1480 were lysed with PBS-Triton (1%), and endogenous JB12 was immunoprecipitated. Western blots of precipitates were probed with either α-CFTR N-terminal tail or α-CFTR-NBD2. (B–D) Transfections with JB12 or RMA1 siRNA oligos and pcDNA3.1(+)-CFTR 837X, pcDNA3.1(+)-CFTR 837XΔF508, or pcDNA3.1(+)-CFTR 837–1480 were performed as described in Materials and Methods. Levels of indicated proteins in cell extracts were determined by Western blot. CFTR 837X or CFTR 837XΔF508 levels were quantified by densitometry, and levels were normalized to control siRNA (si-Cont) reactions for each fragment. Antibodies directed against RMA1 and JB12 were used to monitor the impact of RNA interference addition on endogenous RMA1 and JB12 levels, respectively. Maturely glycosylated CFTR 837–1480 is designated as C-band, and the immaturely glycosylated form is indicated as B-band.To analyze the association between JB12 and CFTR 837X, HEK293 cells were transfected with either CFTR 837X (which is composed of MSD1, NBD1, and the R-domain) or the MSD2- and NBD2-containing C-terminal CFTR fragment 837–1480. Endogenous JB12 was then immunoprecipitated and via Western blot was found to coprecipitate with CFTR 837X (Figure 7A). This complex appears to be a result of a specific interaction, as a complex that contains endogenous JB12 and CFTR 837–1480 was not isolated. Similar results were obtained with CFTR 837XΔF508 (unpublished data). Thus endogenous JB12 associates with an amino-terminal region of CFTR, the conformation of which is surveyed by components of the RMA1 E3 complex.Deletion of F508 hinders the folding of CFTR 837X, and this fragment is unstable (Rosser et al., 2008). Knockdown of JB12 or RMA1 results in a two- to threefold increase in the accumulation of CFTR 837XΔF508 with the greatest effect observed upon JB12 kd (Figure 7B). This increase in accumulation is related to recognition of misfolded CFTR 837XΔF508 by RMA1 and JB12 as accumulation of CFTR 837X, which is highly stable (Meacham et al., 1999), was only modestly increased upon kd of JB12 or RMA1. Furthermore, accumulation of the C-terminal CFTR 837–1480 fragment is not affected by kd of JB12 (Figure 7C). We do observe, however, that the levels of CFTR 837–1480 increase in the absence of RMA1 (Figure 7D). Thus RMA1 may also have additional functions that are independent of JB12.CFTR 837X, but not CFTR 837XΔF508, assembles into a complex with CFTR 837–1480 that escapes the ER as indicated by its accumulation as a glycosylated species that is resistant to Endo H treatment (Rosser et al., 2008). Intriguingly, productive interactions between the two halves of CFTR when JB12 (Figure 7C) or RMA1 (Figure 7D) levels are knocked down are enhanced such that significantly more folded, maturely glycosylated CFTR 837–1480 accumulates. Thus JB12 or RMA1 kd allows for the increased association and proper assembly of N- and C-terminal regions of CFTR. Kd of these ERQC factors, however, does not permit CFTR 837XΔF508 and CFTR 837–1480 to assemble into a complex that escapes the ER as indicated by the lack of CFTR 837–1480 accumulation as a maturely glycosylated species (Figure 7, C and D). Thus JB12 can act at an early stage of CFTR biogenesis, which is coincident with RMA1 action, to modulate the assembly of N- and C-terminal CFTR fragments into a complex that can escape the ER.

JB12 monitors the folding status of CFTR N-terminal regions

Chaperone-dependent steps in CFTR folding and degradation have been uncovered via analysis of the assembly of CFTR N- and C-terminal fragments (CFTR 837X and CFTR 837–1480) into a complex that escapes the ER and functions as a Cl- channel (Chan et al., 2000). Studies with CFTR 837X suggest that deletion of F508 causes a severe defect in formation of a folding intermediate that contains MSD1, NBD1, and the R-domain that is critical for the folding progression of CFTR (Rosser et al., 2008). Recognition of folding defects in amino-terminal regions of CFTRΔF508 by the RMA1 E3 complex is proposed to be a major cause of premature CFTR degradation (Younger et al., 2006; Rosser et al., 2008). Thus we sought to obtain support for the conclusion that JB12 participates with RMA1 and Derlin-1 in selecting CFTR for degradation by asking if JB12 associates with CFTR 837X and determining whether it facilitates CFTR 837X degradation (Figure 7).

JB12 monitors the folding status of CFTR N-terminal regions. (A) Endogenous JB12 interacts with the N-terminal half of CFTR. HEK293 cells transfected with either pcDNA3.1(+)-CFTR 837X or pcDNA3.1(+)-CFTR 837–1480 were lysed with PBS-Triton (1%), and endogenous JB12 was immunoprecipitated. Western blots of precipitates were probed with either α-CFTR N-terminal tail or α-CFTR-NBD2. (B–D) Transfections with JB12 or RMA1 siRNA oligos and pcDNA3.1(+)-CFTR 837X, pcDNA3.1(+)-CFTR 837XΔF508, or pcDNA3.1(+)-CFTR 837–1480 were performed as described in Materials and Methods. Levels of indicated proteins in cell extracts were determined by Western blot. CFTR 837X or CFTR 837XΔF508 levels were quantified by densitometry, and levels were normalized to control siRNA (si-Cont) reactions for each fragment. Antibodies directed against RMA1 and JB12 were used to monitor the impact of RNA interference addition on endogenous RMA1 and JB12 levels, respectively. Maturely glycosylated CFTR 837–1480 is designated as C-band, and the immaturely glycosylated form is indicated as B-band.

To analyze the association between JB12 and CFTR 837X, HEK293 cells were transfected with either CFTR 837X (which is composed of MSD1, NBD1, and the R-domain) or the MSD2- and NBD2-containing C-terminal CFTR fragment 837–1480. Endogenous JB12 was then immunoprecipitated and via Western blot was found to coprecipitate with CFTR 837X (Figure 7A). This complex appears to be a result of a specific interaction, as a complex that contains endogenous JB12 and CFTR 837–1480 was not isolated. Similar results were obtained with CFTR 837XΔF508 (unpublished data). Thus endogenous JB12 associates with an amino-terminal region of CFTR, the conformation of which is surveyed by components of the RMA1 E3 complex.

Deletion of F508 hinders the folding of CFTR 837X, and this fragment is unstable (Rosser et al., 2008). Knockdown of JB12 or RMA1 results in a two- to threefold increase in the accumulation of CFTR 837XΔF508 with the greatest effect observed upon JB12 kd (Figure 7B). This increase in accumulation is related to recognition of misfolded CFTR 837XΔF508 by RMA1 and JB12 as accumulation of CFTR 837X, which is highly stable (Meacham et al., 1999), was only modestly increased upon kd of JB12 or RMA1. Furthermore, accumulation of the C-terminal CFTR 837–1480 fragment is not affected by kd of JB12 (Figure 7C). We do observe, however, that the levels of CFTR 837–1480 increase in the absence of RMA1 (Figure 7D). Thus RMA1 may also have additional functions that are independent of JB12.

CFTR 837X, but not CFTR 837XΔF508, assembles into a complex with CFTR 837–1480 that escapes the ER as indicated by its accumulation as a glycosylated species that is resistant to Endo H treatment (Rosser et al., 2008). Intriguingly, productive interactions between the two halves of CFTR when JB12 (Figure 7C) or RMA1 (Figure 7D) levels are knocked down are enhanced such that significantly more folded, maturely glycosylated CFTR 837–1480 accumulates. Thus JB12 or RMA1 kd allows for the increased association and proper assembly of N- and C-terminal regions of CFTR. Kd of these ERQC factors, however, does not permit CFTR 837XΔF508 and CFTR 837–1480 to assemble into a complex that escapes the ER as indicated by the lack of CFTR 837–1480 accumulation as a maturely glycosylated species (Figure 7, C and D). Thus JB12 can act at an early stage of CFTR biogenesis, which is coincident with RMA1 action, to modulate the assembly of N- and C-terminal CFTR fragments into a complex that can escape the ER.

Stepwise correction of CFTRΔF508 folding to wild-type levels through depletion of JB12 and intragenic suppression of intrinsic folding defectsThe contribution that ERQC activities versus intrinsic folding kinetics play in governing membrane protein folding is unclear, and this information is required to develop approaches to correct CFTRΔF508 misfolding. Therefore we investigated the relative contribution that folding kinetics and the basal ERQC activity contribute to premature CFTRΔF508 degradation. SiRNA kd of JB12 can increase CFTR folding efficiency threefold, whereas elevation of JB12 limits the folding progression of CFTR (Figures 2 and 3). In contrast, siRNA kd of JB12 permits only a small quantity of CFTRΔF508 to accumulate in the folded C-form (Figure 3) and does not enhance assembly of CFTR 837XΔF508 into a folded channel (Figure 7, C and D). Thus folding of CFTR and CFTRΔF508 appear to differ in that ERQC activity limits efficiency of CFTR folding, whereas some other unknown ERQC factor and/or intrinsic folding defects caused by deletion of F508 limits CFTRΔF508.To explore this issue, we evaluated the relative contribution of JB12 action and defective folding kinetics on escape of CFTRΔF508 from the ER. We compared the extent to which the introduction of V510D, a misfolding suppressor mutation, into CFTRΔF508 versus inactivation of JB12 enhanced CFTRΔF508 folding (Figure 8). Residue V510 is located near F508 on the surface of NBD1, and its mutation to an Asp helps to increase cell surface expression of CFTRΔF508 by a mechanism that is proposed to involve enhancement of interdomain contacts (Mornon et al., 2008). The folding of CFTR, CFTRΔF508, and CFTRΔF508 V510D was compared under control conditions, upon JB12 depletion, when cells were treated with the folding corrector Corr-4a (Pedemonte et al., 2005), or with a combination of the above conditions. CFTR folding efficiency was evaluated in Western blots by measuring the ratio of the C-band and B-band that accumulated (Figure 8A). In parallel, we carried out pulse-chase studies and compared the efficiency of B-band conversion to C-band at different chase times (Figure 8B). Data obtained from both sets of experiments are mutually supportive.FIGURE 8:Stepwise correction of CFTRΔF508 folding to wild-type levels through depletion of JB12 and intragenic suppression of intrinsic folding defects. (A) Western blot analysis of the C- and B-form of CFTR. (B) Pulse-chase analysis of CFTR biosynthetic maturation. Transfections with JB12 siRNA oligos and pcDNA3.1(+)-CFTR, pcDNA3.1(+)-CFTRΔF508, or pcDNA3.1(+)-CFTRΔF508 V510D were performed as described in Materials and Methods. The siRNA samples were treated with DMSO or Corr-4a (5 μM) 24 h prior to being harvested. Steady-state levels of CFTR and CFTR mutants were determined by Western blot. In panel B, cells were labeled with 35S-methionine for 20 min, and 35S-CFTR was immunoprecipitated at the indicated time points. Bands B and C represent the immature and maturely glycosylated forms of CFTR, respectively. The B- and C-band levels for each reaction were quantified by densitometry. The normalized C/B-band ratio is shown for (A), whereas in (B) the results were normalized to the relative amount of CFTR B-band at t = 0 for each condition.JB12 siRNA permits approximately fourfold more CFTRΔF508 to accumulate in the C-form, and the ratio of the C- to B-form increased from not being detectable to 0.08 (Figure 8A). CFTRΔF508 folding is improved to a point at which the C/B ratio is further increased to ∼0.2 when JB12 is depleted and Corr-4a is present. Folding of CFTRΔF508 increased dramatically upon introduction of the V510D suppressor mutation because CFTRΔF508 V510D accumulated at a C/B ratio of 0.2. Strikingly, depletion of JB12 increased the C/B ratio of CFTRΔF508 V510D more than eightfold to an impressive value near 1.6, which is 50% of the ratio observed for wild-type CFTR. In pulse-chase experiments, we also observed depletion of JB12 to permit CFTRΔF508 V510D to fold with ∼50% of wild-type efficiency (Figure 8B).The combination of JB12 kd and Corr-4a treatment enabled CFTRΔF508 V510D to obtain a C/B ratio of 3.5, which is the same as the 3.4 value observed with wild-type CFTR. In pulse-chase experiments, we also observed that these conditions permit the B-form of CFTRΔF508 V510D to convert to the C-form with an efficiency that was greater than CFTR (Figure 8B). Corr-4a is able to stabilize transmembrane regions of CFTR (Wang et al., 2007; Grove et al., 2009), which may explain the twofold increase in CFTRΔF508 V510D folding efficiency that occurs when Corr-4a is present in JB12-depleted cells. When JB12 activity is attenuated and intrinsic folding defects are corrected, CFTRΔF508 can fold and escape the ER at levels similar to wild-type CFTR. Thus it appears that activity of JB12 and intrinsic folding kinetics both play important roles in limiting the escape of CFTRΔF508 from the ER.

Stepwise correction of CFTRΔF508 folding to wild-type levels through depletion of JB12 and intragenic suppression of intrinsic folding defects

The contribution that ERQC activities versus intrinsic folding kinetics play in governing membrane protein folding is unclear, and this information is required to develop approaches to correct CFTRΔF508 misfolding. Therefore we investigated the relative contribution that folding kinetics and the basal ERQC activity contribute to premature CFTRΔF508 degradation. SiRNA kd of JB12 can increase CFTR folding efficiency threefold, whereas elevation of JB12 limits the folding progression of CFTR (Figures 2 and 3). In contrast, siRNA kd of JB12 permits only a small quantity of CFTRΔF508 to accumulate in the folded C-form (Figure 3) and does not enhance assembly of CFTR 837XΔF508 into a folded channel (Figure 7, C and D). Thus folding of CFTR and CFTRΔF508 appear to differ in that ERQC activity limits efficiency of CFTR folding, whereas some other unknown ERQC factor and/or intrinsic folding defects caused by deletion of F508 limits CFTRΔF508.

To explore this issue, we evaluated the relative contribution of JB12 action and defective folding kinetics on escape of CFTRΔF508 from the ER. We compared the extent to which the introduction of V510D, a misfolding suppressor mutation, into CFTRΔF508 versus inactivation of JB12 enhanced CFTRΔF508 folding (Figure 8). Residue V510 is located near F508 on the surface of NBD1, and its mutation to an Asp helps to increase cell surface expression of CFTRΔF508 by a mechanism that is proposed to involve enhancement of interdomain contacts (Mornon et al., 2008). The folding of CFTR, CFTRΔF508, and CFTRΔF508 V510D was compared under control conditions, upon JB12 depletion, when cells were treated with the folding corrector Corr-4a (Pedemonte et al., 2005), or with a combination of the above conditions. CFTR folding efficiency was evaluated in Western blots by measuring the ratio of the C-band and B-band that accumulated (Figure 8A). In parallel, we carried out pulse-chase studies and compared the efficiency of B-band conversion to C-band at different chase times (Figure 8B). Data obtained from both sets of experiments are mutually supportive.

Stepwise correction of CFTRΔF508 folding to wild-type levels through depletion of JB12 and intragenic suppression of intrinsic folding defects. (A) Western blot analysis of the C- and B-form of CFTR. (B) Pulse-chase analysis of CFTR biosynthetic maturation. Transfections with JB12 siRNA oligos and pcDNA3.1(+)-CFTR, pcDNA3.1(+)-CFTRΔF508, or pcDNA3.1(+)-CFTRΔF508 V510D were performed as described in Materials and Methods. The siRNA samples were treated with DMSO or Corr-4a (5 μM) 24 h prior to being harvested. Steady-state levels of CFTR and CFTR mutants were determined by Western blot. In panel B, cells were labeled with 35S-methionine for 20 min, and 35S-CFTR was immunoprecipitated at the indicated time points. Bands B and C represent the immature and maturely glycosylated forms of CFTR, respectively. The B- and C-band levels for each reaction were quantified by densitometry. The normalized C/B-band ratio is shown for (A), whereas in (B) the results were normalized to the relative amount of CFTR B-band at t = 0 for each condition.

JB12 siRNA permits approximately fourfold more CFTRΔF508 to accumulate in the C-form, and the ratio of the C- to B-form increased from not being detectable to 0.08 (Figure 8A). CFTRΔF508 folding is improved to a point at which the C/B ratio is further increased to ∼0.2 when JB12 is depleted and Corr-4a is present. Folding of CFTRΔF508 increased dramatically upon introduction of the V510D suppressor mutation because CFTRΔF508 V510D accumulated at a C/B ratio of 0.2. Strikingly, depletion of JB12 increased the C/B ratio of CFTRΔF508 V510D more than eightfold to an impressive value near 1.6, which is 50% of the ratio observed for wild-type CFTR. In pulse-chase experiments, we also observed depletion of JB12 to permit CFTRΔF508 V510D to fold with ∼50% of wild-type efficiency (Figure 8B).

The combination of JB12 kd and Corr-4a treatment enabled CFTRΔF508 V510D to obtain a C/B ratio of 3.5, which is the same as the 3.4 value observed with wild-type CFTR. In pulse-chase experiments, we also observed that these conditions permit the B-form of CFTRΔF508 V510D to convert to the C-form with an efficiency that was greater than CFTR (Figure 8B). Corr-4a is able to stabilize transmembrane regions of CFTR (Wang et al., 2007; Grove et al., 2009), which may explain the twofold increase in CFTRΔF508 V510D folding efficiency that occurs when Corr-4a is present in JB12-depleted cells. When JB12 activity is attenuated and intrinsic folding defects are corrected, CFTRΔF508 can fold and escape the ER at levels similar to wild-type CFTR. Thus it appears that activity of JB12 and intrinsic folding kinetics both play important roles in limiting the escape of CFTRΔF508 from the ER.

DISCUSSIONJB12 is identified as an ER-associated Hsp40 with a cytosolic J-domain that functions with Hsc70 and the RMA1 E3 ligase complex to mediate proteasomal degradation of CFTR and CFTRΔF508. JB12 exerts control over the fate of nascent CFTR and possibly other polytopic proteins because modulation of its activity within a narrow range has dramatic positive and negative effects on CFTR folding efficiency. JB12 exhibits characteristics of a rheostat that helps control the fate of nascent CFTR and CFTRΔF508 by impacting the partitioning of nascent forms between life and death.JB12 depletion strongly increased CFTR folding efficiency but has modest positive effects on CFTRΔF508 folding. These data are consistent with the concept that the folding pathways of CFTR and CFTRΔF508 have different rate-limiting steps (Lukacs et al., 1994; Thibodeau et al., 2005; Serohijos et al., 2008b). A major obstacle that limits folding of nascent CFTR appears to be basal activity of ERQC factors, such as JB12 and other components of the RMA1 E3 ubiquitin ligase complex (Grove et al., 2009). In contrast, the major limiting factor for CFTRΔF508 folding are defects in NBD1 folding/assembly that lead it to accumulate in a kinetically trapped state (Younger et al., 2004; Aleksandrov et al., 2010; Wang et al., 2010). One such folding defect is proposed to be related to defective assembly of ΔF508-NBD1 into a complex with MSD1 and the R-domain (Rosser et al., 2008), and another is failure of ΔF508-NBD1 to make interdomain contacts with intracellular loops on MSD2 (Serohijos et al., 2008a). The small molecule Corr-4a (Pedemonte et al., 2005) and intragenic suppressor mutations (Pissarra et al., 2008; Loo et al., 2010) partially suppress kinetic defects in CFTRΔF508 folding. Action of JB12 and the RMA1 E3 complex, however, remain an obstacle to highly efficient folding of “corrected” CFTRΔF508. Thus combined attenuation of ERQC activity and suppression of kinetic defects in folding may be required to achieve therapeutically relevant increases in cell surface expression of CFTRΔF508.JB12 and Hdj-2 are Hsp40 family members, and the domain structures and localization of each enable them to target Hsc70 to act in highly specific and opposing roles in the biogenesis of CFTR and CFTRΔF508. JB12 uses its J-domain and transmembrane region to facilitate nascent CFTR degradation via increasing Hsc70 association with the B-form of CFTR and the RMA1 E3 complex. In stark contrast, Hdj-2, which is farnesylated and localized to the cytosolic face of the ER, directs Hsc70 to facilitate CFTR folding via transient association with ribosome-associated CFTR translation intermediates (Meacham et al., 1999). These data provide an example of how specialized Hsp40s can specify Hsc70 function at discrete steps in folding and degradation of the same polypeptide. Data presented also identify RMA1 as a member of an expanding family of E3 ubiquitin ligases that cooperate with Hsc70 to mediate aspects of protein QC (Youker et al., 2004; Han et al., 2007).The following observations suggest that JB12 acts in conjunction with Hsc70 on the cytosolic face of the ER to select CFTR for RMA1-dependent proteasomal degradation. First, elevation of JB12 approximately onefold over endogenous levels caused nearly all nascent CFTR and CFTRΔF508 to be targeted for proteasomal degradation. Second, JB12 kd increased CFTR folding efficiency almost threefold and permitted a small pool of CFTRΔF508 to fold and escape the ER. Third, elevation of JB12 levels increased Hsc70 binding to CFTRΔF508 severalfold. Fourth, mutation of the HPD motif in the J-domain, which is required for JB12 interactions with Hsc70, hindered JB12 function in CFTR degradation. Finally, JB12 action is linked to RMA1 because it is present in complexes that contain RMA1 and RMA1 is required to observe JB12-dependent degradation of CFTR.JB12 clearly plays a role in CFTR degradation, but its exact mechanism of action is not clear. One possibility is that JB12 and Hsc70 act in the selection of misfolded CFTR and CFTRΔF508 for RMA1-dependent degradation. Alternatively, JB12 and Hsc70 could act indirectly to regulate RMA1 E3 action. Based on several observations presented, we favor an active role for JB12 in the selection of misfolded CFTR for degradation. JB12 acts in a J-domain–dependent manner to retain the B-form of CFTR in the ER under conditions in which proteasome activity is inhibited and when RMA1 levels are depleted. Thus JB12 action is not dependent on RMA1, and JB12 appears to act prior to RMA1. JB12 does not appear to be required in the folding of proteins involved in CFTR folding or degradation because steady-state levels of the proteins such as Derlin-1, RMA1, Hsc70, and CHIP are unchanged upon JB12 kd. JB12-dependent accumulation of CFTR in the ER correlates with a severalfold increase in Hsc70 binding to CFTR. Thus we propose that JB12 acts with Hsc70 to facilitate ER retention and delivery of misfolded CFTR to components of the RMA1 E3 complex.Function of JB12 in ERQC raises questions about how it enters into complexes that contain nascent CFTR and CFTRΔF508. JB12 contains a transmembrane domain and a J-domain that are required for it to promote CFTR degradation, but its N- and C-terminal extensions are dispensable for this function. Therefore the form of JB12 that is active in CFTR degradation does not appear to contain a canonical polypeptide-binding domain that recognizes water-soluble proteins. Instead, JB12 may use its transmembrane region to associate with membrane-inserted domains of CFTR. Alternatively, JB12 may be a modular Hsp40 (Mokranjac et al., 2003) that interacts with other partners that function in substrate binding. JB12 and Derlin-1 are isolated in complexes with Hsc70 and Derlin-1 binding to CFTR’s membrane domains has been implicated in the selection of CFTR and CFTRΔF508 for RMA1-dependent degradation (Younger et al., 2006). Thus it is conceivable that JB12 and Derlin-1 act together as a modular Hsp40 to bind unassembled transmembrane domains of CFTR and to bring adjacent cytosolic domains into contact with Hsc70. This concept requires additional experimental support, however.On the basis of the cooperation of Hsc70 and CHIP in CFTR degradation (Meacham et al., 2001), we initially surmised that JB12 would assist in CHIP-dependent aspects of CFTR degradation. This interaction with CHIP remains a possibility, but we were unable to detect endogenous or overexpressed CHIP in complexes with JB12 and Hsc70. In addition, overexpressed JB12 could fully stimulate CFTR degradation upon kd of CHIP, whereas kd of RMA1 hindered JB12 action. Comparison of experiments in which different proteins are knocked down is difficult because the threshold at which depletion of a specific protein elicits an effect on the process under study is never clear. Thus we do not want to exclude the possibility that JB12 assists in CHIP-dependent steps in CFTR degradation, but we don’t have evidence to support a direct functional interaction between JB12 and CHIP.JB12 and Hdj-2 are related to the respective yeast Hsp40s Hlj1 and Ydj1 (Walsh et al., 2004). When CFTR is expressed in yeast it is unable to fold and escape the ER, and all of the wild type and mutant forms expressed are degraded (Zhang et al., 2001). Analysis of deletion mutants reveals that the individual loss of Hlj1 or Ydj1 has no effect on CFTR degradation, but deletion of both Hlj1 and Ydj1 slows turnover (Youker et al., 2004). These results differ from data obtained with mammalian cells where depletion of JB12 alone dramatically slows CFTR turnover and increases folding and where depletion of Hdj-2 alone blocks CFTR folding and accelerates CFTRΔF508 degradation. To help explain these data, it should be noted that an RMA1 homologue is not found in yeast, and the yeast ER folding environment is not sufficient to support CFTR folding. Nevertheless, it is clear that Type I Hsp40s, such as Hdj-2 and Ydj1, help stabilize non-native CFTR in the ER. In addition, the ability of JB12-related proteins to function in QC of polytopic membrane proteins is conserved from yeast to humans.

JB12 is identified as an ER-associated Hsp40 with a cytosolic J-domain that functions with Hsc70 and the RMA1 E3 ligase complex to mediate proteasomal degradation of CFTR and CFTRΔF508. JB12 exerts control over the fate of nascent CFTR and possibly other polytopic proteins because modulation of its activity within a narrow range has dramatic positive and negative effects on CFTR folding efficiency. JB12 exhibits characteristics of a rheostat that helps control the fate of nascent CFTR and CFTRΔF508 by impacting the partitioning of nascent forms between life and death.

JB12 depletion strongly increased CFTR folding efficiency but has modest positive effects on CFTRΔF508 folding. These data are consistent with the concept that the folding pathways of CFTR and CFTRΔF508 have different rate-limiting steps (Lukacs et al., 1994; Thibodeau et al., 2005; Serohijos et al., 2008b). A major obstacle that limits folding of nascent CFTR appears to be basal activity of ERQC factors, such as JB12 and other components of the RMA1 E3 ubiquitin ligase complex (Grove et al., 2009). In contrast, the major limiting factor for CFTRΔF508 folding are defects in NBD1 folding/assembly that lead it to accumulate in a kinetically trapped state (Younger et al., 2004; Aleksandrov et al., 2010; Wang et al., 2010). One such folding defect is proposed to be related to defective assembly of ΔF508-NBD1 into a complex with MSD1 and the R-domain (Rosser et al., 2008), and another is failure of ΔF508-NBD1 to make interdomain contacts with intracellular loops on MSD2 (Serohijos et al., 2008a). The small molecule Corr-4a (Pedemonte et al., 2005) and intragenic suppressor mutations (Pissarra et al., 2008; Loo et al., 2010) partially suppress kinetic defects in CFTRΔF508 folding. Action of JB12 and the RMA1 E3 complex, however, remain an obstacle to highly efficient folding of “corrected” CFTRΔF508. Thus combined attenuation of ERQC activity and suppression of kinetic defects in folding may be required to achieve therapeutically relevant increases in cell surface expression of CFTRΔF508.

JB12 and Hdj-2 are Hsp40 family members, and the domain structures and localization of each enable them to target Hsc70 to act in highly specific and opposing roles in the biogenesis of CFTR and CFTRΔF508. JB12 uses its J-domain and transmembrane region to facilitate nascent CFTR degradation via increasing Hsc70 association with the B-form of CFTR and the RMA1 E3 complex. In stark contrast, Hdj-2, which is farnesylated and localized to the cytosolic face of the ER, directs Hsc70 to facilitate CFTR folding via transient association with ribosome-associated CFTR translation intermediates (Meacham et al., 1999). These data provide an example of how specialized Hsp40s can specify Hsc70 function at discrete steps in folding and degradation of the same polypeptide. Data presented also identify RMA1 as a member of an expanding family of E3 ubiquitin ligases that cooperate with Hsc70 to mediate aspects of protein QC (Youker et al., 2004; Han et al., 2007).

The following observations suggest that JB12 acts in conjunction with Hsc70 on the cytosolic face of the ER to select CFTR for RMA1-dependent proteasomal degradation. First, elevation of JB12 approximately onefold over endogenous levels caused nearly all nascent CFTR and CFTRΔF508 to be targeted for proteasomal degradation. Second, JB12 kd increased CFTR folding efficiency almost threefold and permitted a small pool of CFTRΔF508 to fold and escape the ER. Third, elevation of JB12 levels increased Hsc70 binding to CFTRΔF508 severalfold. Fourth, mutation of the HPD motif in the J-domain, which is required for JB12 interactions with Hsc70, hindered JB12 function in CFTR degradation. Finally, JB12 action is linked to RMA1 because it is present in complexes that contain RMA1 and RMA1 is required to observe JB12-dependent degradation of CFTR.

JB12 clearly plays a role in CFTR degradation, but its exact mechanism of action is not clear. One possibility is that JB12 and Hsc70 act in the selection of misfolded CFTR and CFTRΔF508 for RMA1-dependent degradation. Alternatively, JB12 and Hsc70 could act indirectly to regulate RMA1 E3 action. Based on several observations presented, we favor an active role for JB12 in the selection of misfolded CFTR for degradation. JB12 acts in a J-domain–dependent manner to retain the B-form of CFTR in the ER under conditions in which proteasome activity is inhibited and when RMA1 levels are depleted. Thus JB12 action is not dependent on RMA1, and JB12 appears to act prior to RMA1. JB12 does not appear to be required in the folding of proteins involved in CFTR folding or degradation because steady-state levels of the proteins such as Derlin-1, RMA1, Hsc70, and CHIP are unchanged upon JB12 kd. JB12-dependent accumulation of CFTR in the ER correlates with a severalfold increase in Hsc70 binding to CFTR. Thus we propose that JB12 acts with Hsc70 to facilitate ER retention and delivery of misfolded CFTR to components of the RMA1 E3 complex.

Function of JB12 in ERQC raises questions about how it enters into complexes that contain nascent CFTR and CFTRΔF508. JB12 contains a transmembrane domain and a J-domain that are required for it to promote CFTR degradation, but its N- and C-terminal extensions are dispensable for this function. Therefore the form of JB12 that is active in CFTR degradation does not appear to contain a canonical polypeptide-binding domain that recognizes water-soluble proteins. Instead, JB12 may use its transmembrane region to associate with membrane-inserted domains of CFTR. Alternatively, JB12 may be a modular Hsp40 (Mokranjac et al., 2003) that interacts with other partners that function in substrate binding. JB12 and Derlin-1 are isolated in complexes with Hsc70 and Derlin-1 binding to CFTR’s membrane domains has been implicated in the selection of CFTR and CFTRΔF508 for RMA1-dependent degradation (Younger et al., 2006). Thus it is conceivable that JB12 and Derlin-1 act together as a modular Hsp40 to bind unassembled transmembrane domains of CFTR and to bring adjacent cytosolic domains into contact with Hsc70. This concept requires additional experimental support, however.

On the basis of the cooperation of Hsc70 and CHIP in CFTR degradation (Meacham et al., 2001), we initially surmised that JB12 would assist in CHIP-dependent aspects of CFTR degradation. This interaction with CHIP remains a possibility, but we were unable to detect endogenous or overexpressed CHIP in complexes with JB12 and Hsc70. In addition, overexpressed JB12 could fully stimulate CFTR degradation upon kd of CHIP, whereas kd of RMA1 hindered JB12 action. Comparison of experiments in which different proteins are knocked down is difficult because the threshold at which depletion of a specific protein elicits an effect on the process under study is never clear. Thus we do not want to exclude the possibility that JB12 assists in CHIP-dependent steps in CFTR degradation, but we don’t have evidence to support a direct functional interaction between JB12 and CHIP.

JB12 and Hdj-2 are related to the respective yeast Hsp40s Hlj1 and Ydj1 (Walsh et al., 2004). When CFTR is expressed in yeast it is unable to fold and escape the ER, and all of the wild type and mutant forms expressed are degraded (Zhang et al., 2001). Analysis of deletion mutants reveals that the individual loss of Hlj1 or Ydj1 has no effect on CFTR degradation, but deletion of both Hlj1 and Ydj1 slows turnover (Youker et al., 2004). These results differ from data obtained with mammalian cells where depletion of JB12 alone dramatically slows CFTR turnover and increases folding and where depletion of Hdj-2 alone blocks CFTR folding and accelerates CFTRΔF508 degradation. To help explain these data, it should be noted that an RMA1 homologue is not found in yeast, and the yeast ER folding environment is not sufficient to support CFTR folding. Nevertheless, it is clear that Type I Hsp40s, such as Hdj-2 and Ydj1, help stabilize non-native CFTR in the ER. In addition, the ability of JB12-related proteins to function in QC of polytopic membrane proteins is conserved from yeast to humans.

MATERIALS AND METHODSPlasmids, antibodies, and reagentsThe plasmids used for cell transfection were as follows: pcDNA3.1(+)-CFTR, pcDNA3.1(+)-CFTRΔF508, pcDNA3.1(+)-flag-RMA1, pcDNA3.1(+)-Derlin-1, pcDNA3.1(+)-CHIP-myc, pcDM8 2B4 TCRα, pcDNA-ApoB48 (Younger et al., 2006), and pcDNA3.1(+)-HA-p97QQ (Younger et al., 2004). JB12 cDNA was obtained from Open Biosystems (Huntsville, AL), and the coding sequence of JB12 was subcloned into the pcDNA3.1(+) expression vector to generate pcDNA3.1(+)-HA-JB12, pcDNA3.1(+)-JB12-myc, and pcDNA3.1(+)-GFP-JB12 overexpression constructs. Point and truncation mutants of JB12 or CFTR were made using the QuikChange protocol (Stratagene, La Jolla, CA). Antibodies used in this study were as follows: α-CFTR MM13–4 (N-terminal tail epitope) and α-CFTR M3A7 (NBD2 epitope) were obtained from Millipore (Billerica, MA); α-RMA1 (sc81716) was obtained from Santa Cruz Biotechnology (Santa Cruz, CA); α-tubulin, α-Derlin-1, α-HA, α-myc, and α-flag were purchased from Sigma (St. Louis, MO); α-Hdj2 was obtained from Thermo Fisher Scientific (St. Charles, MO); and α-Hsp/c70 (SPA-757) was obtained from Nventa Biopharmaceuticals (San Diego, CA). JB12 antibody used in our studies was generated against JB12 1–243. The polyclonal α-CHIP antibody used was made against full-length recombinant CHIP protein. Polyclonal α-CFTR, generated against a GST fusion protein that contained CFTR residues 1–79, was a gift from Kevin Kirk (University of Alabama, Birmingham). Rabbit polyclonal α-BiP serum was provided to us by Christopher Nicchitta (Duke University). Bortezomib was purchased from LC Laboratories (Woburn, MA) and used at a final concentration of 10 μM. Corr-4a was obtained from the North American Cystic Fibrosis Foundation through Robert Bridges (Rosalind Franklin University).Protein purification proceduresThe following plasmids were used for overexpression of the indicated proteins in the BL21 (DE3) Escherichia coli strain: pET11a-Hsc70 (Meacham et al., 1999), pET21a-JB12 1–243His6, and pET21a-JB12 1–243-QPDHis6. Hsc70 was purified as described previously (Meacham et al., 1999). JB12 1–243 and JB12 1–243-QPD were expressed by inducing BL21 (DE3) cells containing each construct with 0.2 mM isopropyl 1-thio-β-d-galactopyranoside for 16 h at 30ºC. Cells were collected by centrifugation at 4000 rpm for 10 min and then lysed in 20 mM Tris-HCl (pH 8.0), 150 mM NaCl. JB12 1–243His6 was purified by metal chelate chromatography. All of these proteins were dialyzed into 20 mM K-HEPES (pH 7.4) containing 150 mM NaCl.Analysis of purified JB12 activityLuciferase aggregation assays. Luciferase aggregation assays were carried out as previously described (Rosser et al., 2007). Briefly, firefly luciferase (Promega, Madison, WI) was diluted to a final concentration of 100 nM in reactions containing 2 μM Hsc70 and 4–8 μM JB12 1–243, where indicated, and 25 mM K-HEPES (pH 7.4), 25 mM KCl, 5 mM MgCl2, 5 mM dithiothreitol (DTT), and 2 mM ATP. Aliquots of each reaction were removed and stored on ice to determine total (T) protein levels. Samples were incubated at room temperature for 10 min followed by a 10-min incubation at 42ºC. The supernatant (S) and pellet (P) samples were isolated by centrifugation at 20,000 rpm (Beckman Allegra 64R centrifuge; 4ºC, Beckman Coulter, Brea, CA). Samples were applied to a nitrocellulose membrane using a Bio-Rad slot blot apparatus (Hercules, CA), and luciferase-containing fractions were identified by probing the membrane with a α-luciferase antibody (Cortex Biochemicals, San Leandro, CA).Luciferase refolding assay. Refolding of chemically denatured firefly luciferase was conducted as previously described (Rosser et al., 2007). Briefly, luciferase was chemically denatured in 25 mM K-HEPES (pH 7.4), 50 mM KCl, 5 mM MgCl2, and 5 mM DTT containing 6 M guanidinium HCl. After being denatured, the luciferase-containing solution was diluted into a refolding buffer (25 mM K-HEPES [pH 7.4], 50 mM KCl, 5 mM MgCl2, and 1 mM ATP) that was supplemented with 1.6 μM Hsc70 and 3.2 μM JB12 1–243 or Ydj1, where indicated. Aliquots were removed from the folding reaction and mixed with luciferase assay reagent (Promega). Luciferase activity was measured with a Turner TD-20/20 Luminometer (Promega).Cell culture and transfectionHEK293 cells (Stratagene, Santa Clara, CA) were maintained in DMEM (GIBCO) supplemented with 10% fetal bovine serum (FBS; Hyclone, Logan, UT) and antibiotics (penicillin at 100 U/ml and streptomycin at 100 μg/ml; GIBCO). CFBE cells stably expressing CFTR or CFTRΔF508 were cultured in minimum essential medium (MEM; GIBCO) containing 10% FBS, penicillin at 50 U/ml, streptomycin at 50 μg/ml, and puromycin at 1 μg/ml to maintain expression of CFTR. All cell types were incubated at 37°C in an atmosphere of 5% CO2. Cell transfections were performed using Effectene reagent (Qiagen, Chatsworth, CA) or Lipofectamine 2000 (Invitrogen). The empty pcDNA3.1(+) vector was used to ensure that equal microgram quantities of DNA were used in all transfection reactions. Unless otherwise indicated, each individual Western blot, pulse-chase time point, or IP reaction was transfected into one well of a six-well plate.Fluorescence microscopyHEK293 cells were grown on coverslips and transfected with pcDNA3.1(+)-GFP-JB12 (100 ng). Eighteen hours post-transfection the cells were washed with phosphate-buffered saline (PBS), stained with ER-Tracker in PBS (0.5 μM; Invitrogen) for 15 min, fixed with 4% paraformaldehyde in PBS for 12 min, and washed twice with PBS. The localization of GFP-JB12 and the ER were visualized using an Olympus IX81 motorized inverted microscope. Images were processed with MetaMorph and Adobe Photoshop.Subcellular localization of JB12pcDNA3.1(+)-HA-JB12 (0.05 μg) was transfected into three wells of a six-well plate of HEK293 cells. The transfected cells were lysed 18 h post-transfection with 1 ml of a buffer containing PBS, Complete Protease Inhibitor Cocktail (Roche, Basel, Switzerland), 1 mM phenylmethylsulfonyl fluoride (PMSF), and 2 mM DTT. Cell extracts were homogenized with 50 strokes by a Wheaton homogenizer on ice. The lysates were subjected to centrifugation at 12,000 × g for 20 min at 4ºC, and the 12,000 × g supernatant was used for subsequent analysis. An aliquot was removed from the 12,000 × g supernatant to show the level of total (T) HA-JB12 present. Next membranes were pelleted via centrifugation at 100,000 × g for 30 min at 4ºC. The membrane pellet was resuspended in PBS, and equivalent samples were incubated with 1.0 M NaCl, 0.1 M Na2CO3, or 1% Triton. Each sample was incubated on ice for 30 min prior to centrifugation at 100,000 × g for 30 min at 4ºC. The supernatant (S) and pellet (P) fractions and the total (T) sample were analyzed by Western blot analysis with HA-antibody. Calnexin was used as a marker for ER membranes.Analysis of CFTR biogenesisSteady-state levels of CFTR and its mutants were determined by Western blot analysis. HEK293 cells were transiently transfected with 1 μg of the indicated pcDNA3.1(+)-CFTR plasmids. To analyze the effect of JB12 on CFTR levels, 5 ng of pcDNA3.1(+)-JB12-myc (which results in approximately a 1:1 ratio of overexpressed/endogenous JB12 levels) was also introduced into HEK293 cells. Twenty-four hours after transfection, the cells were harvested, diluted with 2× SDS sample buffer (100 mM Tris-HCl, pH 6.8; 4% SDS; 0.05% bromophenol blue; 20% glycerol), sonicated, and heated at 37°C prior to resolving the proteins on SDS–PAGE gels. The proteins were transferred to nitrocellulose membranes, and the membranes were probed with the designated antibodies. α-tubulin was used to indicate loading controls. Where indicated, bortezomib (10 μM final concentration), Corr-4a (5 μM final concentration), or dimethyl sulfoxide (DMSO) was added to the cells 18 h after transfection, and the cells were incubated for 4 h with bortezomib and for 24 h with Corr-4a or DMSO prior to being analyzed by Western blot.CFTR processing efficiency was measured by pulse-chase analysis. Eighteen hours after transfection, HEK293 cells were starved in methionine-free MEM (Sigma) for 20 min, pulse labeled for 20 min with 35S-methionine (100 μCi per 35-mm well; 1200 Ci/mmol; MP Biomedicals, Irvine, CA), and then chased for the indicated amount of time. Cells were then lysed in PBS buffer supplemented with 1% Triton (PBS-Tr 1%), 1 mM PMSF, and Complete Protease Inhibitor Cocktail (Roche). Soluble lysates were obtained by centrifugation at 20,000 rpm for 10 min in a Beckman Allegra 64R centrifuge. Equal microgram quantities of cell lysate were subjected to IP by incubation with a polyclonal α-CFTR antibody directed against the N terminus followed by addition of a 50% Protein G bead slurry. The beads were washed with PBS-Tr (1%) supplemented with 0.2% SDS, the bound CFTR was eluted with 2× SDS sample buffer, and the samples were heated at 55°C for 10 min. The samples were analyzed by SDS–PAGE and visualized by autoradiography.RNA interference analysisHEK293 cells were transfected with oligonucleotides directed at either JB12 (sequence 1, CUAUCCUCAUCCUGAUUCU; sequence 2, CGCUAUACCUACCAGCAAA) for a final concentration of 200 nM or with a final concentration of 100 nM against RMA1 (sequence 1, GCGCGACCUUCGAAUGUAA; sequence 2, CGGCAAGAGUGUCCAGUAU) or CHIP (sequence 1, GGAGCAGGGCAAUCGUCUG; sequence 2, CCAAGCACGACAAGUACAU) by using the transfection reagent Lipofectamine 2000 (Invitrogen). A similar final concentration of a nonspecific siRNA control duplex (Dharmacon, Lafayette, CO) was used for comparison for each siRNA reaction. Forty-eight hours later, 1 μg of pcDNA3.1(+)-CFTR or pcDNA3.1(+)-CFTR mutants was introduced into the cells using Effectene as the transfection reagent. For steady-state analysis, the cells were harvested 24 h after the second transfection. SDS sample buffer (2×) was added to cell pellets, and after sonication the samples were normalized to contain the same total amount of protein. The reactions were resolved on SDS–PAGE gels and transferred to nitrocellulose membranes. Then the membranes were probed with the indicated antibodies. α-Tubulin was used to indicate loading controls. For pulse-chase analysis, the cells were allowed to recover for 18 h after the second transfection. The pulse-chase procedure used is similar to that described in the CFTR biogenesis section.Similar to that described earlier in the text for HEK293 cells, CFBE cells were transfected with either the nonspecific siRNA control duplex or siRNA oligos directed against JB12 using Lipofectamine 2000. The CFBE cells were harvested 72 h after transfection, and CFTR steady-state levels were analyzed by Western blot.The Hdj-2 shRNA construct (pGIPZ Hdj-2; V2LHS-132206) and the nonsilencing pGIPZ control (RHS4346) were purchased from Open Biosystems, and 6 μg of plasmid was transfected into HEK293 cells per single well of a six-well plate by using Lipofectamine 2000 as the transfection reagent. Puromycin (25 μg/ml) was added to the cells 18 h after transfection to isolate those cells that took up the pGIPZ constructs. Cells were grown in puromycin-containing medium for 3 d before pcDNA3.1(+)-CFTR (1 μg) was introduced into these cells by a second transfection using Effectene. Eighteen hours later, the cells were either harvested for Western blot analysis or 35S-labeled for pulse-chase reactions.CFTR ubiquitination analysisHEK293 cells were transiently transfected with pcDNA3.1(+)-CFTR (1 μg) or pcDNA3.1(+)-CFTRΔF508 (1 μg) and combinations of pcDNA3.1(+)-JB12-myc (5 ng) and pcDNA3.1(+)-HA-p97QQ (0.2 μg). The cells were lysed 18 h post-transfection with PBS-Tr (1%) buffer that contained 1 mM PMSF and Complete Protease Inhibitor Cocktail. Soluble cell lysates were obtained by centrifugation at 20,000 rpm for 10 min in an Allegra 64R centrifuge. Input samples were taken to compare the total amount of protein in each sample. CFTR was isolated by IP with a polyclonal α-CFTR antibody followed by addition of a 50% Protein G bead slurry. The isolated CFTR–Protein G bead complex was washed with PBS-Tr (1%) supplemented with 0.2% SDS, the bound CFTR was eluted with 2× sample buffer, and the samples were heated at 55ºC for 10 min. Samples were resolved on a 10% SDS–PAGE gel and transferred to a nitrocellulose membrane. The nitrocellulose membranes were probed with either α-CFTR or α-Ub to detect the ubiquitinated CFTR.Co-IP of JB12 with ER QC factorsWestern blot co-IPs. HEK293 cells were cotransfected with either pcDNA3.1(+)-JB12-myc (0.2 μg) and pcDNA3.1(+)-flag-RMA1 (0.2 μg) or with pcDNA3.1(+)-HA-JB12 (0.2 μg) and pcDNA3.1(+)-CHIP-myc (0.2 μg). The cells were lysed in a co-IP buffer (PBS-Tr [1%], Complete Protease Inhibitor Cocktail, and 1 mM PMSF) 18 h post-transfection, and the soluble cell lysates were obtained by centrifugation at 20,000 rpm for 10 min in an Allegra 64R centrifuge. Input samples were taken for each reaction. Complexes were isolated by IP with the indicated antibodies followed by addition of a 50% Protein G bead slurry. The isolated complexes were washed with PBS-Tr (1%), eluted with 2× SDS sample buffer, and heated at 55ºC for 10 min. The interacting proteins were separated on a SDS–PAGE gel and transferred to a nitrocellulose membrane. Then the membranes were probed with the designated antibodies.35S-labeled co-IPs. Influence of HA-JB12 (0.05 μg) on CFTRΔF508 (1 μg) and endogenous Hsc70 association in HEK293 cells was visualized via co-IP/re-IP analysis similar to that described previously (Meacham et al., 1999). 35S-labeled cells were lysed in PBS-Tr (1%) supplemented with either 80 mM phosphocreatine, creatine phosphokinase at 500 μg/ml, and 5 mM Mg-ATP (+ ATP condition) or apyrase (- ATP condition) (Meacham et al., 1999). CFTRΔF508 was immunoprecipitated from cell extracts, and the levels of bound Hsc70 were visualized by re-IP of endogenous Hsc70.Isolation of 35S-labeled JB12/RMA1/Derlin-1/Hsc70 degradation complex. Expression plasmids for HA-JB12 (0.05 μg), flag-RMA1 (0.5 μg), Derlin-1 (0.005 μg), and CFTRΔF508 (1 μg) were cotransfected into HEK293 cells. Next, 18 h post-transfection the cells were starved in methionine-free MEM (Sigma) for 30 min followed by a pulse-labeling period of 60 min with 35S-methionine. Cells from 36 individual 35-mm wells were pooled and lysed at 4ºC for 60 min in a co-IP buffer (PBS-Tr [1%], Complete Protease Inhibitor, and PMSF) prior to centrifugation at 20,000 rpm for 10 min in a Beckman Allegra 64R centrifuge. The cleared lysates were then divided with 10% of the total volume to be used for co-IP analysis while the remaining 90% was used for re-IP studies. The reactions were incubated at 4ºC for 30 min in the absence or presence of the α-HA antibody. This incubation was followed by the addition of a 50% Protein G slurry, and incubations with beads were carried out for 30 min. For the co-IP reactions, the Protein G pellets were washed three times with PBS-Tr (1%), and the immunoprecipitated proteins were eluted in 2× SDS sample buffer at 55ºC for 10 min. The re-IP samples were washed three times with PBS-Tr (1%), samples were separated into five equal volume aliquots, and 40 μl of 2× SDS sample buffer was added to each sample. Four of the samples were incubated at 55ºC, and the fifth sample was incubated at 37ºC to elute the interacting proteins. Each sample was further diluted in 1 ml of PBS-Tr (1%) supplemented with Complete Protease Inhibitor, PMSF, 0.2% SDS, and 0.5% bovine serum albumin. The four samples that were heated at 55ºC were incubated with α-flag, α-Derlin-1, α-Hsp/c70, or no antibody, and the sample heated previously at 37ºC was incubated with α-CFTR (CFTR may aggregate when heated to >37ºC in SDS-sample buffer, possibly affecting CFTR interactions with the α-CFTR antibody). Samples were incubated with antibody for 30 min followed by a 30-min incubation with a 50% Protein G slurry at 4ºC. The beads were washed twice with PBS-Tr (1%) supplemented with 0.2% SDS, and the immunoprecipitated proteins were eluted in 2× sample buffer at 55ºC for 10 min. The co-IP and re-IP samples were then analyzed by SDS–PAGE and visualized by autoradiography.Image processingX-ray film was exposed to detect signals on Western blots and autoradiograms. It was then scanned with a Bio-Rad GS-670 Imaging Densitometer. The film exposures used were in the linear range of Kodak Biomax XAR film and were quantitated with Bio-Rad Quant-1 Software. Images were exported to Adobe Photoshop and processed. Images were then imported into Adobe Illustrator and used to build figures. In instances where lanes of gels are not shown in the order they were run, a white line has been placed in the figures. Titrations were carried out to assure that Westerns blots were developed in the linear range of the primary and secondary antibodies.

MATERIALS AND METHODS

Plasmids, antibodies, and reagentsThe plasmids used for cell transfection were as follows: pcDNA3.1(+)-CFTR, pcDNA3.1(+)-CFTRΔF508, pcDNA3.1(+)-flag-RMA1, pcDNA3.1(+)-Derlin-1, pcDNA3.1(+)-CHIP-myc, pcDM8 2B4 TCRα, pcDNA-ApoB48 (Younger et al., 2006), and pcDNA3.1(+)-HA-p97QQ (Younger et al., 2004). JB12 cDNA was obtained from Open Biosystems (Huntsville, AL), and the coding sequence of JB12 was subcloned into the pcDNA3.1(+) expression vector to generate pcDNA3.1(+)-HA-JB12, pcDNA3.1(+)-JB12-myc, and pcDNA3.1(+)-GFP-JB12 overexpression constructs. Point and truncation mutants of JB12 or CFTR were made using the QuikChange protocol (Stratagene, La Jolla, CA). Antibodies used in this study were as follows: α-CFTR MM13–4 (N-terminal tail epitope) and α-CFTR M3A7 (NBD2 epitope) were obtained from Millipore (Billerica, MA); α-RMA1 (sc81716) was obtained from Santa Cruz Biotechnology (Santa Cruz, CA); α-tubulin, α-Derlin-1, α-HA, α-myc, and α-flag were purchased from Sigma (St. Louis, MO); α-Hdj2 was obtained from Thermo Fisher Scientific (St. Charles, MO); and α-Hsp/c70 (SPA-757) was obtained from Nventa Biopharmaceuticals (San Diego, CA). JB12 antibody used in our studies was generated against JB12 1–243. The polyclonal α-CHIP antibody used was made against full-length recombinant CHIP protein. Polyclonal α-CFTR, generated against a GST fusion protein that contained CFTR residues 1–79, was a gift from Kevin Kirk (University of Alabama, Birmingham). Rabbit polyclonal α-BiP serum was provided to us by Christopher Nicchitta (Duke University). Bortezomib was purchased from LC Laboratories (Woburn, MA) and used at a final concentration of 10 μM. Corr-4a was obtained from the North American Cystic Fibrosis Foundation through Robert Bridges (Rosalind Franklin University).

Plasmids, antibodies, and reagents

The plasmids used for cell transfection were as follows: pcDNA3.1(+)-CFTR, pcDNA3.1(+)-CFTRΔF508, pcDNA3.1(+)-flag-RMA1, pcDNA3.1(+)-Derlin-1, pcDNA3.1(+)-CHIP-myc, pcDM8 2B4 TCRα, pcDNA-ApoB48 (Younger et al., 2006), and pcDNA3.1(+)-HA-p97QQ (Younger et al., 2004). JB12 cDNA was obtained from Open Biosystems (Huntsville, AL), and the coding sequence of JB12 was subcloned into the pcDNA3.1(+) expression vector to generate pcDNA3.1(+)-HA-JB12, pcDNA3.1(+)-JB12-myc, and pcDNA3.1(+)-GFP-JB12 overexpression constructs. Point and truncation mutants of JB12 or CFTR were made using the QuikChange protocol (Stratagene, La Jolla, CA). Antibodies used in this study were as follows: α-CFTR MM13–4 (N-terminal tail epitope) and α-CFTR M3A7 (NBD2 epitope) were obtained from Millipore (Billerica, MA); α-RMA1 (sc81716) was obtained from Santa Cruz Biotechnology (Santa Cruz, CA); α-tubulin, α-Derlin-1, α-HA, α-myc, and α-flag were purchased from Sigma (St. Louis, MO); α-Hdj2 was obtained from Thermo Fisher Scientific (St. Charles, MO); and α-Hsp/c70 (SPA-757) was obtained from Nventa Biopharmaceuticals (San Diego, CA). JB12 antibody used in our studies was generated against JB12 1–243. The polyclonal α-CHIP antibody used was made against full-length recombinant CHIP protein. Polyclonal α-CFTR, generated against a GST fusion protein that contained CFTR residues 1–79, was a gift from Kevin Kirk (University of Alabama, Birmingham). Rabbit polyclonal α-BiP serum was provided to us by Christopher Nicchitta (Duke University). Bortezomib was purchased from LC Laboratories (Woburn, MA) and used at a final concentration of 10 μM. Corr-4a was obtained from the North American Cystic Fibrosis Foundation through Robert Bridges (Rosalind Franklin University).

Protein purification proceduresThe following plasmids were used for overexpression of the indicated proteins in the BL21 (DE3) Escherichia coli strain: pET11a-Hsc70 (Meacham et al., 1999), pET21a-JB12 1–243His6, and pET21a-JB12 1–243-QPDHis6. Hsc70 was purified as described previously (Meacham et al., 1999). JB12 1–243 and JB12 1–243-QPD were expressed by inducing BL21 (DE3) cells containing each construct with 0.2 mM isopropyl 1-thio-β-d-galactopyranoside for 16 h at 30ºC. Cells were collected by centrifugation at 4000 rpm for 10 min and then lysed in 20 mM Tris-HCl (pH 8.0), 150 mM NaCl. JB12 1–243His6 was purified by metal chelate chromatography. All of these proteins were dialyzed into 20 mM K-HEPES (pH 7.4) containing 150 mM NaCl.

Protein purification procedures

The following plasmids were used for overexpression of the indicated proteins in the BL21 (DE3) Escherichia coli strain: pET11a-Hsc70 (Meacham et al., 1999), pET21a-JB12 1–243His6, and pET21a-JB12 1–243-QPDHis6. Hsc70 was purified as described previously (Meacham et al., 1999). JB12 1–243 and JB12 1–243-QPD were expressed by inducing BL21 (DE3) cells containing each construct with 0.2 mM isopropyl 1-thio-β-d-galactopyranoside for 16 h at 30ºC. Cells were collected by centrifugation at 4000 rpm for 10 min and then lysed in 20 mM Tris-HCl (pH 8.0), 150 mM NaCl. JB12 1–243His6 was purified by metal chelate chromatography. All of these proteins were dialyzed into 20 mM K-HEPES (pH 7.4) containing 150 mM NaCl.

Analysis of purified JB12 activityLuciferase aggregation assays. Luciferase aggregation assays were carried out as previously described (Rosser et al., 2007). Briefly, firefly luciferase (Promega, Madison, WI) was diluted to a final concentration of 100 nM in reactions containing 2 μM Hsc70 and 4–8 μM JB12 1–243, where indicated, and 25 mM K-HEPES (pH 7.4), 25 mM KCl, 5 mM MgCl2, 5 mM dithiothreitol (DTT), and 2 mM ATP. Aliquots of each reaction were removed and stored on ice to determine total (T) protein levels. Samples were incubated at room temperature for 10 min followed by a 10-min incubation at 42ºC. The supernatant (S) and pellet (P) samples were isolated by centrifugation at 20,000 rpm (Beckman Allegra 64R centrifuge; 4ºC, Beckman Coulter, Brea, CA). Samples were applied to a nitrocellulose membrane using a Bio-Rad slot blot apparatus (Hercules, CA), and luciferase-containing fractions were identified by probing the membrane with a α-luciferase antibody (Cortex Biochemicals, San Leandro, CA).Luciferase refolding assay. Refolding of chemically denatured firefly luciferase was conducted as previously described (Rosser et al., 2007). Briefly, luciferase was chemically denatured in 25 mM K-HEPES (pH 7.4), 50 mM KCl, 5 mM MgCl2, and 5 mM DTT containing 6 M guanidinium HCl. After being denatured, the luciferase-containing solution was diluted into a refolding buffer (25 mM K-HEPES [pH 7.4], 50 mM KCl, 5 mM MgCl2, and 1 mM ATP) that was supplemented with 1.6 μM Hsc70 and 3.2 μM JB12 1–243 or Ydj1, where indicated. Aliquots were removed from the folding reaction and mixed with luciferase assay reagent (Promega). Luciferase activity was measured with a Turner TD-20/20 Luminometer (Promega).

Analysis of purified JB12 activity

Luciferase aggregation assays. Luciferase aggregation assays were carried out as previously described (Rosser et al., 2007). Briefly, firefly luciferase (Promega, Madison, WI) was diluted to a final concentration of 100 nM in reactions containing 2 μM Hsc70 and 4–8 μM JB12 1–243, where indicated, and 25 mM K-HEPES (pH 7.4), 25 mM KCl, 5 mM MgCl2, 5 mM dithiothreitol (DTT), and 2 mM ATP. Aliquots of each reaction were removed and stored on ice to determine total (T) protein levels. Samples were incubated at room temperature for 10 min followed by a 10-min incubation at 42ºC. The supernatant (S) and pellet (P) samples were isolated by centrifugation at 20,000 rpm (Beckman Allegra 64R centrifuge; 4ºC, Beckman Coulter, Brea, CA). Samples were applied to a nitrocellulose membrane using a Bio-Rad slot blot apparatus (Hercules, CA), and luciferase-containing fractions were identified by probing the membrane with a α-luciferase antibody (Cortex Biochemicals, San Leandro, CA).

Luciferase refolding assay. Refolding of chemically denatured firefly luciferase was conducted as previously described (Rosser et al., 2007). Briefly, luciferase was chemically denatured in 25 mM K-HEPES (pH 7.4), 50 mM KCl, 5 mM MgCl2, and 5 mM DTT containing 6 M guanidinium HCl. After being denatured, the luciferase-containing solution was diluted into a refolding buffer (25 mM K-HEPES [pH 7.4], 50 mM KCl, 5 mM MgCl2, and 1 mM ATP) that was supplemented with 1.6 μM Hsc70 and 3.2 μM JB12 1–243 or Ydj1, where indicated. Aliquots were removed from the folding reaction and mixed with luciferase assay reagent (Promega). Luciferase activity was measured with a Turner TD-20/20 Luminometer (Promega).

Cell culture and transfectionHEK293 cells (Stratagene, Santa Clara, CA) were maintained in DMEM (GIBCO) supplemented with 10% fetal bovine serum (FBS; Hyclone, Logan, UT) and antibiotics (penicillin at 100 U/ml and streptomycin at 100 μg/ml; GIBCO). CFBE cells stably expressing CFTR or CFTRΔF508 were cultured in minimum essential medium (MEM; GIBCO) containing 10% FBS, penicillin at 50 U/ml, streptomycin at 50 μg/ml, and puromycin at 1 μg/ml to maintain expression of CFTR. All cell types were incubated at 37°C in an atmosphere of 5% CO2. Cell transfections were performed using Effectene reagent (Qiagen, Chatsworth, CA) or Lipofectamine 2000 (Invitrogen). The empty pcDNA3.1(+) vector was used to ensure that equal microgram quantities of DNA were used in all transfection reactions. Unless otherwise indicated, each individual Western blot, pulse-chase time point, or IP reaction was transfected into one well of a six-well plate.

Cell culture and transfection

HEK293 cells (Stratagene, Santa Clara, CA) were maintained in DMEM (GIBCO) supplemented with 10% fetal bovine serum (FBS; Hyclone, Logan, UT) and antibiotics (penicillin at 100 U/ml and streptomycin at 100 μg/ml; GIBCO). CFBE cells stably expressing CFTR or CFTRΔF508 were cultured in minimum essential medium (MEM; GIBCO) containing 10% FBS, penicillin at 50 U/ml, streptomycin at 50 μg/ml, and puromycin at 1 μg/ml to maintain expression of CFTR. All cell types were incubated at 37°C in an atmosphere of 5% CO2. Cell transfections were performed using Effectene reagent (Qiagen, Chatsworth, CA) or Lipofectamine 2000 (Invitrogen). The empty pcDNA3.1(+) vector was used to ensure that equal microgram quantities of DNA were used in all transfection reactions. Unless otherwise indicated, each individual Western blot, pulse-chase time point, or IP reaction was transfected into one well of a six-well plate.

Fluorescence microscopyHEK293 cells were grown on coverslips and transfected with pcDNA3.1(+)-GFP-JB12 (100 ng). Eighteen hours post-transfection the cells were washed with phosphate-buffered saline (PBS), stained with ER-Tracker in PBS (0.5 μM; Invitrogen) for 15 min, fixed with 4% paraformaldehyde in PBS for 12 min, and washed twice with PBS. The localization of GFP-JB12 and the ER were visualized using an Olympus IX81 motorized inverted microscope. Images were processed with MetaMorph and Adobe Photoshop.

Fluorescence microscopy

HEK293 cells were grown on coverslips and transfected with pcDNA3.1(+)-GFP-JB12 (100 ng). Eighteen hours post-transfection the cells were washed with phosphate-buffered saline (PBS), stained with ER-Tracker in PBS (0.5 μM; Invitrogen) for 15 min, fixed with 4% paraformaldehyde in PBS for 12 min, and washed twice with PBS. The localization of GFP-JB12 and the ER were visualized using an Olympus IX81 motorized inverted microscope. Images were processed with MetaMorph and Adobe Photoshop.

Subcellular localization of JB12pcDNA3.1(+)-HA-JB12 (0.05 μg) was transfected into three wells of a six-well plate of HEK293 cells. The transfected cells were lysed 18 h post-transfection with 1 ml of a buffer containing PBS, Complete Protease Inhibitor Cocktail (Roche, Basel, Switzerland), 1 mM phenylmethylsulfonyl fluoride (PMSF), and 2 mM DTT. Cell extracts were homogenized with 50 strokes by a Wheaton homogenizer on ice. The lysates were subjected to centrifugation at 12,000 × g for 20 min at 4ºC, and the 12,000 × g supernatant was used for subsequent analysis. An aliquot was removed from the 12,000 × g supernatant to show the level of total (T) HA-JB12 present. Next membranes were pelleted via centrifugation at 100,000 × g for 30 min at 4ºC. The membrane pellet was resuspended in PBS, and equivalent samples were incubated with 1.0 M NaCl, 0.1 M Na2CO3, or 1% Triton. Each sample was incubated on ice for 30 min prior to centrifugation at 100,000 × g for 30 min at 4ºC. The supernatant (S) and pellet (P) fractions and the total (T) sample were analyzed by Western blot analysis with HA-antibody. Calnexin was used as a marker for ER membranes.

Subcellular localization of JB12

pcDNA3.1(+)-HA-JB12 (0.05 μg) was transfected into three wells of a six-well plate of HEK293 cells. The transfected cells were lysed 18 h post-transfection with 1 ml of a buffer containing PBS, Complete Protease Inhibitor Cocktail (Roche, Basel, Switzerland), 1 mM phenylmethylsulfonyl fluoride (PMSF), and 2 mM DTT. Cell extracts were homogenized with 50 strokes by a Wheaton homogenizer on ice. The lysates were subjected to centrifugation at 12,000 × g for 20 min at 4ºC, and the 12,000 × g supernatant was used for subsequent analysis. An aliquot was removed from the 12,000 × g supernatant to show the level of total (T) HA-JB12 present. Next membranes were pelleted via centrifugation at 100,000 × g for 30 min at 4ºC. The membrane pellet was resuspended in PBS, and equivalent samples were incubated with 1.0 M NaCl, 0.1 M Na2CO3, or 1% Triton. Each sample was incubated on ice for 30 min prior to centrifugation at 100,000 × g for 30 min at 4ºC. The supernatant (S) and pellet (P) fractions and the total (T) sample were analyzed by Western blot analysis with HA-antibody. Calnexin was used as a marker for ER membranes.

Analysis of CFTR biogenesisSteady-state levels of CFTR and its mutants were determined by Western blot analysis. HEK293 cells were transiently transfected with 1 μg of the indicated pcDNA3.1(+)-CFTR plasmids. To analyze the effect of JB12 on CFTR levels, 5 ng of pcDNA3.1(+)-JB12-myc (which results in approximately a 1:1 ratio of overexpressed/endogenous JB12 levels) was also introduced into HEK293 cells. Twenty-four hours after transfection, the cells were harvested, diluted with 2× SDS sample buffer (100 mM Tris-HCl, pH 6.8; 4% SDS; 0.05% bromophenol blue; 20% glycerol), sonicated, and heated at 37°C prior to resolving the proteins on SDS–PAGE gels. The proteins were transferred to nitrocellulose membranes, and the membranes were probed with the designated antibodies. α-tubulin was used to indicate loading controls. Where indicated, bortezomib (10 μM final concentration), Corr-4a (5 μM final concentration), or dimethyl sulfoxide (DMSO) was added to the cells 18 h after transfection, and the cells were incubated for 4 h with bortezomib and for 24 h with Corr-4a or DMSO prior to being analyzed by Western blot.CFTR processing efficiency was measured by pulse-chase analysis. Eighteen hours after transfection, HEK293 cells were starved in methionine-free MEM (Sigma) for 20 min, pulse labeled for 20 min with 35S-methionine (100 μCi per 35-mm well; 1200 Ci/mmol; MP Biomedicals, Irvine, CA), and then chased for the indicated amount of time. Cells were then lysed in PBS buffer supplemented with 1% Triton (PBS-Tr 1%), 1 mM PMSF, and Complete Protease Inhibitor Cocktail (Roche). Soluble lysates were obtained by centrifugation at 20,000 rpm for 10 min in a Beckman Allegra 64R centrifuge. Equal microgram quantities of cell lysate were subjected to IP by incubation with a polyclonal α-CFTR antibody directed against the N terminus followed by addition of a 50% Protein G bead slurry. The beads were washed with PBS-Tr (1%) supplemented with 0.2% SDS, the bound CFTR was eluted with 2× SDS sample buffer, and the samples were heated at 55°C for 10 min. The samples were analyzed by SDS–PAGE and visualized by autoradiography.

Analysis of CFTR biogenesis

Steady-state levels of CFTR and its mutants were determined by Western blot analysis. HEK293 cells were transiently transfected with 1 μg of the indicated pcDNA3.1(+)-CFTR plasmids. To analyze the effect of JB12 on CFTR levels, 5 ng of pcDNA3.1(+)-JB12-myc (which results in approximately a 1:1 ratio of overexpressed/endogenous JB12 levels) was also introduced into HEK293 cells. Twenty-four hours after transfection, the cells were harvested, diluted with 2× SDS sample buffer (100 mM Tris-HCl, pH 6.8; 4% SDS; 0.05% bromophenol blue; 20% glycerol), sonicated, and heated at 37°C prior to resolving the proteins on SDS–PAGE gels. The proteins were transferred to nitrocellulose membranes, and the membranes were probed with the designated antibodies. α-tubulin was used to indicate loading controls. Where indicated, bortezomib (10 μM final concentration), Corr-4a (5 μM final concentration), or dimethyl sulfoxide (DMSO) was added to the cells 18 h after transfection, and the cells were incubated for 4 h with bortezomib and for 24 h with Corr-4a or DMSO prior to being analyzed by Western blot.

CFTR processing efficiency was measured by pulse-chase analysis. Eighteen hours after transfection, HEK293 cells were starved in methionine-free MEM (Sigma) for 20 min, pulse labeled for 20 min with 35S-methionine (100 μCi per 35-mm well; 1200 Ci/mmol; MP Biomedicals, Irvine, CA), and then chased for the indicated amount of time. Cells were then lysed in PBS buffer supplemented with 1% Triton (PBS-Tr 1%), 1 mM PMSF, and Complete Protease Inhibitor Cocktail (Roche). Soluble lysates were obtained by centrifugation at 20,000 rpm for 10 min in a Beckman Allegra 64R centrifuge. Equal microgram quantities of cell lysate were subjected to IP by incubation with a polyclonal α-CFTR antibody directed against the N terminus followed by addition of a 50% Protein G bead slurry. The beads were washed with PBS-Tr (1%) supplemented with 0.2% SDS, the bound CFTR was eluted with 2× SDS sample buffer, and the samples were heated at 55°C for 10 min. The samples were analyzed by SDS–PAGE and visualized by autoradiography.

RNA interference analysisHEK293 cells were transfected with oligonucleotides directed at either JB12 (sequence 1, CUAUCCUCAUCCUGAUUCU; sequence 2, CGCUAUACCUACCAGCAAA) for a final concentration of 200 nM or with a final concentration of 100 nM against RMA1 (sequence 1, GCGCGACCUUCGAAUGUAA; sequence 2, CGGCAAGAGUGUCCAGUAU) or CHIP (sequence 1, GGAGCAGGGCAAUCGUCUG; sequence 2, CCAAGCACGACAAGUACAU) by using the transfection reagent Lipofectamine 2000 (Invitrogen). A similar final concentration of a nonspecific siRNA control duplex (Dharmacon, Lafayette, CO) was used for comparison for each siRNA reaction. Forty-eight hours later, 1 μg of pcDNA3.1(+)-CFTR or pcDNA3.1(+)-CFTR mutants was introduced into the cells using Effectene as the transfection reagent. For steady-state analysis, the cells were harvested 24 h after the second transfection. SDS sample buffer (2×) was added to cell pellets, and after sonication the samples were normalized to contain the same total amount of protein. The reactions were resolved on SDS–PAGE gels and transferred to nitrocellulose membranes. Then the membranes were probed with the indicated antibodies. α-Tubulin was used to indicate loading controls. For pulse-chase analysis, the cells were allowed to recover for 18 h after the second transfection. The pulse-chase procedure used is similar to that described in the CFTR biogenesis section.Similar to that described earlier in the text for HEK293 cells, CFBE cells were transfected with either the nonspecific siRNA control duplex or siRNA oligos directed against JB12 using Lipofectamine 2000. The CFBE cells were harvested 72 h after transfection, and CFTR steady-state levels were analyzed by Western blot.The Hdj-2 shRNA construct (pGIPZ Hdj-2; V2LHS-132206) and the nonsilencing pGIPZ control (RHS4346) were purchased from Open Biosystems, and 6 μg of plasmid was transfected into HEK293 cells per single well of a six-well plate by using Lipofectamine 2000 as the transfection reagent. Puromycin (25 μg/ml) was added to the cells 18 h after transfection to isolate those cells that took up the pGIPZ constructs. Cells were grown in puromycin-containing medium for 3 d before pcDNA3.1(+)-CFTR (1 μg) was introduced into these cells by a second transfection using Effectene. Eighteen hours later, the cells were either harvested for Western blot analysis or 35S-labeled for pulse-chase reactions.

RNA interference analysis

HEK293 cells were transfected with oligonucleotides directed at either JB12 (sequence 1, CUAUCCUCAUCCUGAUUCU; sequence 2, CGCUAUACCUACCAGCAAA) for a final concentration of 200 nM or with a final concentration of 100 nM against RMA1 (sequence 1, GCGCGACCUUCGAAUGUAA; sequence 2, CGGCAAGAGUGUCCAGUAU) or CHIP (sequence 1, GGAGCAGGGCAAUCGUCUG; sequence 2, CCAAGCACGACAAGUACAU) by using the transfection reagent Lipofectamine 2000 (Invitrogen). A similar final concentration of a nonspecific siRNA control duplex (Dharmacon, Lafayette, CO) was used for comparison for each siRNA reaction. Forty-eight hours later, 1 μg of pcDNA3.1(+)-CFTR or pcDNA3.1(+)-CFTR mutants was introduced into the cells using Effectene as the transfection reagent. For steady-state analysis, the cells were harvested 24 h after the second transfection. SDS sample buffer (2×) was added to cell pellets, and after sonication the samples were normalized to contain the same total amount of protein. The reactions were resolved on SDS–PAGE gels and transferred to nitrocellulose membranes. Then the membranes were probed with the indicated antibodies. α-Tubulin was used to indicate loading controls. For pulse-chase analysis, the cells were allowed to recover for 18 h after the second transfection. The pulse-chase procedure used is similar to that described in the CFTR biogenesis section.

Similar to that described earlier in the text for HEK293 cells, CFBE cells were transfected with either the nonspecific siRNA control duplex or siRNA oligos directed against JB12 using Lipofectamine 2000. The CFBE cells were harvested 72 h after transfection, and CFTR steady-state levels were analyzed by Western blot.

The Hdj-2 shRNA construct (pGIPZ Hdj-2; V2LHS-132206) and the nonsilencing pGIPZ control (RHS4346) were purchased from Open Biosystems, and 6 μg of plasmid was transfected into HEK293 cells per single well of a six-well plate by using Lipofectamine 2000 as the transfection reagent. Puromycin (25 μg/ml) was added to the cells 18 h after transfection to isolate those cells that took up the pGIPZ constructs. Cells were grown in puromycin-containing medium for 3 d before pcDNA3.1(+)-CFTR (1 μg) was introduced into these cells by a second transfection using Effectene. Eighteen hours later, the cells were either harvested for Western blot analysis or 35S-labeled for pulse-chase reactions.

CFTR ubiquitination analysisHEK293 cells were transiently transfected with pcDNA3.1(+)-CFTR (1 μg) or pcDNA3.1(+)-CFTRΔF508 (1 μg) and combinations of pcDNA3.1(+)-JB12-myc (5 ng) and pcDNA3.1(+)-HA-p97QQ (0.2 μg). The cells were lysed 18 h post-transfection with PBS-Tr (1%) buffer that contained 1 mM PMSF and Complete Protease Inhibitor Cocktail. Soluble cell lysates were obtained by centrifugation at 20,000 rpm for 10 min in an Allegra 64R centrifuge. Input samples were taken to compare the total amount of protein in each sample. CFTR was isolated by IP with a polyclonal α-CFTR antibody followed by addition of a 50% Protein G bead slurry. The isolated CFTR–Protein G bead complex was washed with PBS-Tr (1%) supplemented with 0.2% SDS, the bound CFTR was eluted with 2× sample buffer, and the samples were heated at 55ºC for 10 min. Samples were resolved on a 10% SDS–PAGE gel and transferred to a nitrocellulose membrane. The nitrocellulose membranes were probed with either α-CFTR or α-Ub to detect the ubiquitinated CFTR.

CFTR ubiquitination analysis

HEK293 cells were transiently transfected with pcDNA3.1(+)-CFTR (1 μg) or pcDNA3.1(+)-CFTRΔF508 (1 μg) and combinations of pcDNA3.1(+)-JB12-myc (5 ng) and pcDNA3.1(+)-HA-p97QQ (0.2 μg). The cells were lysed 18 h post-transfection with PBS-Tr (1%) buffer that contained 1 mM PMSF and Complete Protease Inhibitor Cocktail. Soluble cell lysates were obtained by centrifugation at 20,000 rpm for 10 min in an Allegra 64R centrifuge. Input samples were taken to compare the total amount of protein in each sample. CFTR was isolated by IP with a polyclonal α-CFTR antibody followed by addition of a 50% Protein G bead slurry. The isolated CFTR–Protein G bead complex was washed with PBS-Tr (1%) supplemented with 0.2% SDS, the bound CFTR was eluted with 2× sample buffer, and the samples were heated at 55ºC for 10 min. Samples were resolved on a 10% SDS–PAGE gel and transferred to a nitrocellulose membrane. The nitrocellulose membranes were probed with either α-CFTR or α-Ub to detect the ubiquitinated CFTR.

Co-IP of JB12 with ER QC factorsWestern blot co-IPs. HEK293 cells were cotransfected with either pcDNA3.1(+)-JB12-myc (0.2 μg) and pcDNA3.1(+)-flag-RMA1 (0.2 μg) or with pcDNA3.1(+)-HA-JB12 (0.2 μg) and pcDNA3.1(+)-CHIP-myc (0.2 μg). The cells were lysed in a co-IP buffer (PBS-Tr [1%], Complete Protease Inhibitor Cocktail, and 1 mM PMSF) 18 h post-transfection, and the soluble cell lysates were obtained by centrifugation at 20,000 rpm for 10 min in an Allegra 64R centrifuge. Input samples were taken for each reaction. Complexes were isolated by IP with the indicated antibodies followed by addition of a 50% Protein G bead slurry. The isolated complexes were washed with PBS-Tr (1%), eluted with 2× SDS sample buffer, and heated at 55ºC for 10 min. The interacting proteins were separated on a SDS–PAGE gel and transferred to a nitrocellulose membrane. Then the membranes were probed with the designated antibodies.35S-labeled co-IPs. Influence of HA-JB12 (0.05 μg) on CFTRΔF508 (1 μg) and endogenous Hsc70 association in HEK293 cells was visualized via co-IP/re-IP analysis similar to that described previously (Meacham et al., 1999). 35S-labeled cells were lysed in PBS-Tr (1%) supplemented with either 80 mM phosphocreatine, creatine phosphokinase at 500 μg/ml, and 5 mM Mg-ATP (+ ATP condition) or apyrase (- ATP condition) (Meacham et al., 1999). CFTRΔF508 was immunoprecipitated from cell extracts, and the levels of bound Hsc70 were visualized by re-IP of endogenous Hsc70.Isolation of 35S-labeled JB12/RMA1/Derlin-1/Hsc70 degradation complex. Expression plasmids for HA-JB12 (0.05 μg), flag-RMA1 (0.5 μg), Derlin-1 (0.005 μg), and CFTRΔF508 (1 μg) were cotransfected into HEK293 cells. Next, 18 h post-transfection the cells were starved in methionine-free MEM (Sigma) for 30 min followed by a pulse-labeling period of 60 min with 35S-methionine. Cells from 36 individual 35-mm wells were pooled and lysed at 4ºC for 60 min in a co-IP buffer (PBS-Tr [1%], Complete Protease Inhibitor, and PMSF) prior to centrifugation at 20,000 rpm for 10 min in a Beckman Allegra 64R centrifuge. The cleared lysates were then divided with 10% of the total volume to be used for co-IP analysis while the remaining 90% was used for re-IP studies. The reactions were incubated at 4ºC for 30 min in the absence or presence of the α-HA antibody. This incubation was followed by the addition of a 50% Protein G slurry, and incubations with beads were carried out for 30 min. For the co-IP reactions, the Protein G pellets were washed three times with PBS-Tr (1%), and the immunoprecipitated proteins were eluted in 2× SDS sample buffer at 55ºC for 10 min. The re-IP samples were washed three times with PBS-Tr (1%), samples were separated into five equal volume aliquots, and 40 μl of 2× SDS sample buffer was added to each sample. Four of the samples were incubated at 55ºC, and the fifth sample was incubated at 37ºC to elute the interacting proteins. Each sample was further diluted in 1 ml of PBS-Tr (1%) supplemented with Complete Protease Inhibitor, PMSF, 0.2% SDS, and 0.5% bovine serum albumin. The four samples that were heated at 55ºC were incubated with α-flag, α-Derlin-1, α-Hsp/c70, or no antibody, and the sample heated previously at 37ºC was incubated with α-CFTR (CFTR may aggregate when heated to >37ºC in SDS-sample buffer, possibly affecting CFTR interactions with the α-CFTR antibody). Samples were incubated with antibody for 30 min followed by a 30-min incubation with a 50% Protein G slurry at 4ºC. The beads were washed twice with PBS-Tr (1%) supplemented with 0.2% SDS, and the immunoprecipitated proteins were eluted in 2× sample buffer at 55ºC for 10 min. The co-IP and re-IP samples were then analyzed by SDS–PAGE and visualized by autoradiography.

Co-IP of JB12 with ER QC factors

Western blot co-IPs. HEK293 cells were cotransfected with either pcDNA3.1(+)-JB12-myc (0.2 μg) and pcDNA3.1(+)-flag-RMA1 (0.2 μg) or with pcDNA3.1(+)-HA-JB12 (0.2 μg) and pcDNA3.1(+)-CHIP-myc (0.2 μg). The cells were lysed in a co-IP buffer (PBS-Tr [1%], Complete Protease Inhibitor Cocktail, and 1 mM PMSF) 18 h post-transfection, and the soluble cell lysates were obtained by centrifugation at 20,000 rpm for 10 min in an Allegra 64R centrifuge. Input samples were taken for each reaction. Complexes were isolated by IP with the indicated antibodies followed by addition of a 50% Protein G bead slurry. The isolated complexes were washed with PBS-Tr (1%), eluted with 2× SDS sample buffer, and heated at 55ºC for 10 min. The interacting proteins were separated on a SDS–PAGE gel and transferred to a nitrocellulose membrane. Then the membranes were probed with the designated antibodies.

35S-labeled co-IPs. Influence of HA-JB12 (0.05 μg) on CFTRΔF508 (1 μg) and endogenous Hsc70 association in HEK293 cells was visualized via co-IP/re-IP analysis similar to that described previously (Meacham et al., 1999). 35S-labeled cells were lysed in PBS-Tr (1%) supplemented with either 80 mM phosphocreatine, creatine phosphokinase at 500 μg/ml, and 5 mM Mg-ATP (+ ATP condition) or apyrase (- ATP condition) (Meacham et al., 1999). CFTRΔF508 was immunoprecipitated from cell extracts, and the levels of bound Hsc70 were visualized by re-IP of endogenous Hsc70.

Isolation of 35S-labeled JB12/RMA1/Derlin-1/Hsc70 degradation complex. Expression plasmids for HA-JB12 (0.05 μg), flag-RMA1 (0.5 μg), Derlin-1 (0.005 μg), and CFTRΔF508 (1 μg) were cotransfected into HEK293 cells. Next, 18 h post-transfection the cells were starved in methionine-free MEM (Sigma) for 30 min followed by a pulse-labeling period of 60 min with 35S-methionine. Cells from 36 individual 35-mm wells were pooled and lysed at 4ºC for 60 min in a co-IP buffer (PBS-Tr [1%], Complete Protease Inhibitor, and PMSF) prior to centrifugation at 20,000 rpm for 10 min in a Beckman Allegra 64R centrifuge. The cleared lysates were then divided with 10% of the total volume to be used for co-IP analysis while the remaining 90% was used for re-IP studies. The reactions were incubated at 4ºC for 30 min in the absence or presence of the α-HA antibody. This incubation was followed by the addition of a 50% Protein G slurry, and incubations with beads were carried out for 30 min. For the co-IP reactions, the Protein G pellets were washed three times with PBS-Tr (1%), and the immunoprecipitated proteins were eluted in 2× SDS sample buffer at 55ºC for 10 min. The re-IP samples were washed three times with PBS-Tr (1%), samples were separated into five equal volume aliquots, and 40 μl of 2× SDS sample buffer was added to each sample. Four of the samples were incubated at 55ºC, and the fifth sample was incubated at 37ºC to elute the interacting proteins. Each sample was further diluted in 1 ml of PBS-Tr (1%) supplemented with Complete Protease Inhibitor, PMSF, 0.2% SDS, and 0.5% bovine serum albumin. The four samples that were heated at 55ºC were incubated with α-flag, α-Derlin-1, α-Hsp/c70, or no antibody, and the sample heated previously at 37ºC was incubated with α-CFTR (CFTR may aggregate when heated to >37ºC in SDS-sample buffer, possibly affecting CFTR interactions with the α-CFTR antibody). Samples were incubated with antibody for 30 min followed by a 30-min incubation with a 50% Protein G slurry at 4ºC. The beads were washed twice with PBS-Tr (1%) supplemented with 0.2% SDS, and the immunoprecipitated proteins were eluted in 2× sample buffer at 55ºC for 10 min. The co-IP and re-IP samples were then analyzed by SDS–PAGE and visualized by autoradiography.

Image processingX-ray film was exposed to detect signals on Western blots and autoradiograms. It was then scanned with a Bio-Rad GS-670 Imaging Densitometer. The film exposures used were in the linear range of Kodak Biomax XAR film and were quantitated with Bio-Rad Quant-1 Software. Images were exported to Adobe Photoshop and processed. Images were then imported into Adobe Illustrator and used to build figures. In instances where lanes of gels are not shown in the order they were run, a white line has been placed in the figures. Titrations were carried out to assure that Westerns blots were developed in the linear range of the primary and secondary antibodies.

Image processing

X-ray film was exposed to detect signals on Western blots and autoradiograms. It was then scanned with a Bio-Rad GS-670 Imaging Densitometer. The film exposures used were in the linear range of Kodak Biomax XAR film and were quantitated with Bio-Rad Quant-1 Software. Images were exported to Adobe Photoshop and processed. Images were then imported into Adobe Illustrator and used to build figures. In instances where lanes of gels are not shown in the order they were run, a white line has been placed in the figures. Titrations were carried out to assure that Westerns blots were developed in the linear range of the primary and secondary antibodies.

Supplementary Material[Supplemental Materials]

Supplementary Material

[Supplemental Materials]
