---
pmid: '25961502'
title: Degradation of lipid droplet-associated proteins by chaperone-mediated autophagy
  facilitates lipolysis.
authors:
- Kaushik S
- Cuervo AM
journal: Nat Cell Biol
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4449813
doi: 10.1038/ncb3166
---

# Degradation of lipid droplet-associated proteins by chaperone-mediated autophagy facilitates lipolysis.
**Authors:** Kaushik S, Cuervo AM
**Journal:** Nat Cell Biol (2015)
**DOI:** [10.1038/ncb3166](https://doi.org/10.1038/ncb3166)
**PMC:** [PMC4449813](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4449813/)

## Abstract

1. Nat Cell Biol. 2015 Jun;17(6):759-70. doi: 10.1038/ncb3166. Epub 2015 May 11.

Degradation of lipid droplet-associated proteins by chaperone-mediated autophagy 
facilitates lipolysis.

Kaushik S(1), Cuervo AM(1).

Author information:
(1)1] Department of Developmental and Molecular Biology, Albert Einstein College 
of Medicine, Bronx, New York 10461, USA [2] Institute for Aging Studies, Albert 
Einstein College of Medicine, Bronx, New York 10461, USA.

Comment in
    Cell Metab. 2015 Jul 7;22(1):60-1. doi: 10.1016/j.cmet.2015.06.017.

Chaperone-mediated autophagy (CMA) selectively degrades a subset of cytosolic 
proteins in lysosomes. A potent physiological activator of CMA is nutrient 
deprivation, a condition in which intracellular triglyceride stores or lipid 
droplets (LDs) also undergo hydrolysis (lipolysis) to generate free fatty acids 
for energetic purposes. Here we report that the LD-associated proteins perilipin 
2 (PLIN2) and perilipin 3 (PLIN3) are CMA substrates and their degradation 
through CMA precedes lipolysis. In vivo studies revealed that CMA degradation of 
PLIN2 and PLIN3 was enhanced during starvation, concurrent with elevated levels 
of cytosolic adipose triglyceride lipase (ATGL) and macroautophagy proteins on 
LDs. CMA blockage both in cultured cells and mouse liver or expression of 
CMA-resistant PLINs leads to reduced association of ATGL and 
macrolipophagy-related proteins with LDs and the subsequent decrease in lipid 
oxidation and accumulation of LDs. We propose a role for CMA in LD biology and 
in the maintenance of lipid homeostasis.

DOI: 10.1038/ncb3166
PMCID: PMC4449813
PMID: 25961502 [Indexed for MEDLINE]

Conflict of interest statement: Competing Financial Interest The authors declare 
that they have no competing interests.

## Full Text

IntroductionAutophagy maintains cellular homeostasis by degrading proteins, lipids and organelles in lysosomes1, 2. Turnover of nonfunctional cellular components by autophagy ensures quality control and recycling of degraded products provides energy3. In chaperone-mediated autophagy (CMA) heat shock cognate protein of 70kDa (hsc70) recognizes proteins with a pentapeptide motif4 and delivers them to the lysosome surface5 for binding to lysosome-associated membrane protein 2A (LAMP-2A; L2A), the rate-limiting component of CMA6. L2A organizes into a multimeric complex7 and in cooperation with lysosomal hsc708 mediates lysosomal translocation of the unfolded substrates9 for degradation. CMA occurs in most mammalian cell-types10, 11 and is maximally activated following prolonged starvation12, mild oxidative stress13, hypoxia14 and lipogenic stressors15. Decrease in lysosomal stability of LAMP-2A results in reduced CMA activity in old organisms16, 17.Added to the well-characterized turnover of proteins by autophagy, lipids can also undergo degradation by macroautophagy, by sequestration into double membrane vesicles (autophagosomes) that then fuse with lysosomes18, 19. Autophagy-mediated lipolysis (macrolipophagy) selectively targets lipid droplets (LD), intracellular lipid stores that serve as energy source through hydrolysis of triglycerides (TG) into free fatty acids (FFA). LD are surrounded by structural proteins of the perilipin (PLIN) family, with PLIN1 being primarily an adipocyte protein and PLIN2 and PLIN3 expressed ubiquitously20. Lipolysis can occur via cytosolic lipases, such as adipose triglyceride lipase (ATGL)21, or by lysosome luminal lipases when autophagosomes, formed through assembly of autophagy-related proteins (ATGs) on the LD surface, engulf portions of LD and target them to lysosomes18.Although CMA can degrade only proteins and no lipids, mice with constitutive blockage of CMA in liver exhibit pronounced steatosis, even though non-selective macroautophagy was intact22. The discrete increase in lipogenesis, due to reduced degradation of lipogenic enzymes through CMA22, could not account alone for the massive hepatic lipid accumulation in these animals. This finding combined with the facts that, (i) in tissues, such as liver, lipid influx, lipolysis and CMA activation occur at maximal rates during starvation12, 23; (ii) lipogenic challenges initially upregulate CMA activity15; and (iii) conditions when CMA activity decreases such as aging16 often associate with intracellular lipid accumulation24, made us to further investigate the role of CMA in regulating lipid homeostasis.Here, we demonstrate that blocking CMA both in cultured cells and in mouse liver in vivo decreases LD breakdown. Under conditions that promote lipolysis, CMA degrades LD proteins PLIN2 and PLIN3, and this facilitates the LD association of cytosolic lipase ATGL and of macroautophagy ATGs. Reduced CMA precludes recruitment of the lipolytic machinery to the LD, thereby positioning CMA as a critical upstream regulator of both macrolipophagy and cytosolic lipolysis.

Introduction

Autophagy maintains cellular homeostasis by degrading proteins, lipids and organelles in lysosomes1, 2. Turnover of nonfunctional cellular components by autophagy ensures quality control and recycling of degraded products provides energy3. In chaperone-mediated autophagy (CMA) heat shock cognate protein of 70kDa (hsc70) recognizes proteins with a pentapeptide motif4 and delivers them to the lysosome surface5 for binding to lysosome-associated membrane protein 2A (LAMP-2A; L2A), the rate-limiting component of CMA6. L2A organizes into a multimeric complex7 and in cooperation with lysosomal hsc708 mediates lysosomal translocation of the unfolded substrates9 for degradation. CMA occurs in most mammalian cell-types10, 11 and is maximally activated following prolonged starvation12, mild oxidative stress13, hypoxia14 and lipogenic stressors15. Decrease in lysosomal stability of LAMP-2A results in reduced CMA activity in old organisms16, 17.

Added to the well-characterized turnover of proteins by autophagy, lipids can also undergo degradation by macroautophagy, by sequestration into double membrane vesicles (autophagosomes) that then fuse with lysosomes18, 19. Autophagy-mediated lipolysis (macrolipophagy) selectively targets lipid droplets (LD), intracellular lipid stores that serve as energy source through hydrolysis of triglycerides (TG) into free fatty acids (FFA). LD are surrounded by structural proteins of the perilipin (PLIN) family, with PLIN1 being primarily an adipocyte protein and PLIN2 and PLIN3 expressed ubiquitously20. Lipolysis can occur via cytosolic lipases, such as adipose triglyceride lipase (ATGL)21, or by lysosome luminal lipases when autophagosomes, formed through assembly of autophagy-related proteins (ATGs) on the LD surface, engulf portions of LD and target them to lysosomes18.

Although CMA can degrade only proteins and no lipids, mice with constitutive blockage of CMA in liver exhibit pronounced steatosis, even though non-selective macroautophagy was intact22. The discrete increase in lipogenesis, due to reduced degradation of lipogenic enzymes through CMA22, could not account alone for the massive hepatic lipid accumulation in these animals. This finding combined with the facts that, (i) in tissues, such as liver, lipid influx, lipolysis and CMA activation occur at maximal rates during starvation12, 23; (ii) lipogenic challenges initially upregulate CMA activity15; and (iii) conditions when CMA activity decreases such as aging16 often associate with intracellular lipid accumulation24, made us to further investigate the role of CMA in regulating lipid homeostasis.

Here, we demonstrate that blocking CMA both in cultured cells and in mouse liver in vivo decreases LD breakdown. Under conditions that promote lipolysis, CMA degrades LD proteins PLIN2 and PLIN3, and this facilitates the LD association of cytosolic lipase ATGL and of macroautophagy ATGs. Reduced CMA precludes recruitment of the lipolytic machinery to the LD, thereby positioning CMA as a critical upstream regulator of both macrolipophagy and cytosolic lipolysis.

ResultsLAMP-2A-deficient cells accumulate LDUsing both, livers from mice conditionally knock-out for LAMP-2A (L2A) in hepatocytes22 (L2AKO) and mouse fibroblasts (NIH3T3 cells) knocked down for L2A (L2A(−)) to block CMA25 we confirmed that, despite lower dependence of fibroblasts on lipid metabolism when compared to hepatocytes, L2A-deficient fibroblasts accumulated significantly more triglycerides (TG) than control fibroblasts (Fig. 1a). These differences in TG content were even higher when intracellular lipid usage was forced by reducing glucose in the media or after a lipogenic stimulus (oleate; OL) (Fig. 1a).As in the L2AKO mice22, we only found a slight trend towards higher TG synthesis in L2A(−) cells compared to control cells (Fig. 1b). In contrast, L2A(−) cells showed significantly reduced β-oxidation rates, an event downstream of TG hydrolysis, both under basal and lipogenic conditions (Fig. 1c), and failed to increase oxygen consumption rates (OCR) upon OL exposure (Fig. 1d; CTR cells: Δ20.9+1.7pmol/min; L2A(−) cells: Δ−1.3+0.2pmol/min OCR change). Decreased β-oxidation rates in L2A(−) cells was not due to defective mitochondria per se, since the fraction of mitochondria with intact membrane potential (Supplementary Figure 1a) and mitochondria turnover (detected with pH-sensitive MitoKeima reporter26; Supplementary Figure 1b) were comparable to CTR cells.. Differences in β-oxidation rates between control and L2A(−) cells were minimized in presence of etomoxir (inhibitor of mitochondrial FFA uptake) (Fig. 1d; upon OL addition, OCR in CTR cells was 48.4+2.1pmol/min higher than in L2A(−) cells, but after adding etomoxir the difference between both cell-types was reduced to 17.2+1.6pmol/min) suggesting a problem in lipid mobilization and not in its oxidation per se.Consistent with the pronounced hepatosteatosis of L2AKO mice (Supplementary Figure 1c), L2A(−) fibroblasts accumulate more and bigger lipid droplets (LD) visible with either BODIPY493/503 (Fig. 1e,
Supplementary Figure 1d) or DPH (1,6-diphenylhexatriene27) (Supplementary Figure 1e,f), both under basal and OL-treated conditions. Unlike control cells, etomoxir-treated L2A(−) cells failed to increase LD supporting defective lipid mobilization (Supplementary Figure 1g). To further test this hypothesis, we used different experimental paradigms to force lipolysis in fibroblasts. OL-challenged control cells, upon removal of OL (OL>S+) showed a decrease in LD number and size, even more evident if maintained in serum-free media (OL>S−) (Fig. 1e, Supplementary Figure 1d–f). In contrast, OL-challenged L2A(−) cells maintained elevated LD number and size throughout. Similarly, culturing cells in low-glucose or in serum-free media, to facilitate cellular usage of lipid stores, revealed significantly reduced ability of L2A(−) cells to decrease their TG levels and LD content compared to control cells (Fig. 1a, Supplementary Figure 1h). Electron microscopy confirmed that LD number, average size, and occupied cellular area were significantly higher in CMA-deficient cells than in control cells in all conditions tested (Fig. 1f). LD accumulation in L2A(−) cells was a direct consequence of L2A loss since expressing L2A in these cells (Supplementary Figure 1i,j) was sufficient to reduce LD to control cell levels (Fig. 1g). Together, these results indicate that CMA blockage results in LD accumulation and this is primarily a consequence of decreased LD breakdown.LD proteins PLIN2 and PLIN3 are CMA substratesSince only proteins and not lipids can be CMA substrates, we next investigated a role of CMA in degradation of LD proteins, which could explain defective LD utilization in CMA-deficient cells. Immunoblotting (Fig. 2a, Supplementary Figure 2a) and immunofluorescence (Fig. 2b) for the ubiquitously expressed PLIN2 and PLIN3 showed significantly higher levels under basal, OL treatment and post-OL treatment in L2A(−) cells. Consistent with the described PLIN2 turnover by the ubiquitin-proteasome pathway28,29, proteasomal inhibitors increased LD number (Supplementary Figure 2b) and PLIN2 levels (Supplementary Figure 2c) in control cells but not in CMA-defective cells, although proteasomal degradation of ubiquitinated proteins and a degron-GFP reporter30 were preserved in these cells (Supplementary Figure 2c–e). Lipid accumulation upon proteasomal or lysosomal inhibition were comparable (Supplementary Figure 2b), leaving open the possibility that PLINs could also be amenable for lysosomal degradation.To examine the possible contribution of CMA to PLIN2 and PLIN3 degradation, we first investigated their association with lysosomes. In support of PLINs lysosomal turnover, inhibiting lysosomal degradation with ammonium chloride and leupeptin in control cells significantly increased the colocalization of PLIN2 or PLIN3 with the lysosomal marker (LAMP1), and more so upon OL addition (Fig. 2c,d, Supplementary Figure 2f,g). In contrast, L2A(−) cells showed decreased association of PLIN2 and PLIN3 with lysosomes and impaired flux with lysosomal inhibitors (Fig. 2c,d, Supplementary Figure 2f,g). PLIN2 and PLIN3 were enriched in lysosomes active for CMA (CMA+, containing hsc70 and LAMP-2A31) as compared to lysosomes with lower CMA activity (CMA−, containing LAMP-2A but deficient in hsc7031) (Fig. 2e, glyceraldehyde 3-phosphate dehydrogenase (GAPDH), a CMA substrate32, is shown as a positive control). Lysosomal levels of both PLINs were reduced in L2A(−) cells (Fig. 2f). We confirmed that lysosomal association of PLIN2 and PLIN3 led to their degradation in vivo, since blocking lysosomal proteolysis in starved rats with leupeptin for 2h was sufficient to increase levels of these LD proteins in CMA-active lysosomes (Fig. 2g, Supplementary Figure 2h). CMA degradation of PLIN2 was also detected in fed mice, albeit at lower levels than in starved animals (Fig. 2g), whereas lysosomal degradation of PLIN3 in fed conditions was negligible (Fig. 2g). Some degradation of PLINs was observed in CMA- lysosomes, likely due to LD degradation by macrolipophagy, since these lysosomes preferentially fuse with autophagosomes33. Collectively, results from our in vitro and in vivo studies indicate that CMA degrades PLIN2 and PLIN3.PLINs interact with CMA chaperone hsc70The first step in CMA is substrate interaction with hsc70 for subsequent lysosomal targeting. We found hsc70 in isolated rat liver LD and its levels increased during starvation, when hepatic lipolysis is highly active, coinciding with a decrease in LD levels of PLIN2 and PLIN3 (Fig. 3a). Immunofluorescence confirmed hsc70 colocalization with each PLIN on LD, which increased upon OL-challenge that induces lipolysis (Fig. 3b,c, Supplementary Figure 3a). Forcing lipid mobilization by placing cells in serum-free media post-OL challenge reduced association of hsc70 with LD (Supplementary Figure 3b). Remarkably, L2A(−) cells exhibited higher hsc70 colocalization with PLIN2 or PLIN3 in LD under all conditions (Fig. 3b,c, Supplementary Figure 3a,b). Similar higher abundance of hsc70 was also observed in LD isolated from livers of L2AKO mice compared to control littermates (Fig. 3d).We found direct interaction of hsc70 with PLIN2 and with PLIN3 in cultured cells. Hsc70 was recovered in PLIN2 and 3 pull-downs (Fig. 3e,f) and both PLINs were also detected in hsc70 pull-downs (Supplementary Figure 3c,d). For the same amount of PLINs pulled-down, we consistently observed higher levels of bound hsc70 in L2A(−) cells (Fig. 3e,f). Increased binding to hsc70 is characteristic of CMA substrates in these cells where disruption of CMA occurs at the level of the lysosomal receptor (L2A) while substrate recognition by hsc70 is intact (Supplementary Figure 3e shows the same effect in two other CMA substrates). This enhanced binding of hsc70 for multiple substrates explains why higher levels of PLINs (a single substrate) are not observed in hsc70 pull-downs (Supplementary Figure 3c,d). Upon treatment of L2A(−) cells with increasing OL concentrations previously shown to activate CMA15, hsc70 bound to PLIN2 or PLIN3 further increased whereas in control cells they remained constant, supporting continuous turnover of LD proteins by CMA (Fig. 3e,f, Supplementary Figure 3f). Since contrary to the almost exclusive LD location of PLIN2, PLIN3 can be present in other cellular compartments, we isolated mouse liver LD and confirmed that interaction of hsc70 with PLINs occurs in LD (Fig. 3g). These results are compatible with PLIN2 and PLIN3 being recognized by the CMA chaperone hsc70 at the LD surface before undergoing CMA-mediated degradation.Modifications of CMA-targeted PLINsTo further confirm the involvement of CMA in PLINs turnover, we activated CMA in OL-challenged cells using an atypical retinoic acid derivative (aRAd)34. Chemical activation of CMA significantly reduced PLIN2 levels (Fig. 4a) and LD content (Fig. 4b) in control cells but not in L2A(−) cells. In fact, restoration of L2A levels in L2A(−) cells through transient transfection was sufficient to bring PLIN2 levels close to control cell values (Fig. 4b).To identify possible posttranslational modifications that trigger PLIN2 for CMA degradation we performed bidimensional electrophoresis and immunoblot and found two very prominent forms of PLIN2 of 5.6 and 5.9 isoelectric point (pI) and a less abundant isoform of pI 5.3. When lipolysis and subsequent PLIN2 degradation was induced with OL, the less positively charged isoform was no longer observed in control cells but it persisted in L2A(−) cells (Fig. 4c). Phosphatase treatment confirmed that PLIN2 was phosphorylated during lipolysis in CTR cells but not in L2A(−) cells that accumulated a form of PLIN2 unchanged by the phosphatase (Fig. 4c). Analysis of LD by electrophoresis using Phos-tag (for resolution of phosphorylated variants) confirmed the presence of a phosphorylated form of PLIN2 in control cells almost completely absent in L2A(−) cells (Fig. 4d). Although further studies are needed to elucidate the reasons for reduced levels of phospho-PLIN2 in absence of functional CMA, our findings suggest that phosphorylation of PLIN2 may be required to trigger its CMA degradation.CMA-resistant PLIN2 causes steatosisHsc70 binds CMA substrates through a pentapeptide motif4. PLIN3 bears a canonical CMA-motif (100LDRLQ) and PLIN2 a putative motif (414SLKVQ). To analyze the consequences of preventing CMA degradation of PLIN2, independently from other changes that may occur during CMA blockage, we mutated the PLIN2 CMA-targeting motif to 414SLKAA. Expression of wild-type (WT) and CMA-mutant (MT) PLIN2-GFP in control cells led to higher levels of MT PLIN2-GFP (Fig. 5a) and a ~2.7-fold increase in LD content and size as compared to WT PLIN2-expressing cells under basal conditions (Fig. 5b) and upon OL-challenge (Fig. 5c). MT PLIN2-GFP accumulation was mainly due to its reduced lysosomal degradation as, treatment with lysosomal inhibitors (NL) increased WT but not MT PLIN2-GFP levels (Fig. 5c) and, unlike WT PLIN2, failed to increase colocalization of MT PLIN2 with the lysosomal marker LAMP1 (Fig. 5d).We confirmed that disruption of the CMA-targeting motif in PLIN2 almost completely abrogates its binding to hsc70 (Fig. 5e), explaining the reduced colocalization of hsc70 with LD in cells expressing MT PLIN2 (Fig. 5f,g; 3D reconstruction of Z-sections shown in Supplementary Figure 4a and Supplementary Video 1). Persistence of MT PLIN2 on LD altered their cellular dynamics and their interplay with lysosomes. Thus, live-cell imaging of cells co-expressing WT PLIN2-GFP and LAMP1-RFP revealed that WT PLIN2-GFP LD were predominantly small and very mobile as reported earlier35, 36 (Fig. 5h–j, Supplementary Video 2 (green channel only)) and showed dynamic inter-LD associations and kiss-and-run events between LD and lysosomes (Fig. 5h–j, Supplementary Video 3 (merged channels)). In striking contrast, majority of LD in MT PLIN2-expressing cells formed larger, less dynamic clusters and the very few smaller LD rarely associated with lysosomes (Fig. 5h–j; Z-sections stack shown in Supplementary Figure 4b, Supplementary Video 2,3). In summary, preventing degradation of PLIN2 by CMA is sufficient to induce the intracellular lipid accumulation observed in CMA-deficient cells (Fig. 1 and Fig. 2) and to reduce LD/lysosome interactions.Defective CMA of PLINs blocks lipolysisTG in LD are broken down into FFA either by cytosolic lipases21, 23 or by lysosomal lipases that gain access to these cytosolic stores via macrolipophagy18. To assess which lipolytic pathway failed to mobilize LD in L2A(−) cells (Fig. 1), we treated cells with a lipase inhibitor, diethylumbelliferyl phosphate (Deup), and with lysosomal inhibitors (NL) and found that βoxidation decreased in control cells but L2A(−) cells remained unresponsive (Fig. 6a). Overall these findings suggested that blocking CMA results in impaired lipolysis by cytosolic lipases and macrolipophagy.The rate-limiting cytosolic lipase in most cell-types, adipose triglyceride lipase (ATGL)21, colocalizes with LD (lipid- or protein-labeled) during lipogenic stimulus (OL) and in OL>S- or OL>low-glucose S+ conditions to induce lipolysis (Fig. 6b,c, Supplementary Figure 5a,b). Similarly, chemical activation of CMA increased colocalization of ATGL with LD (Fig. 6d). In contrast, CMA-defective cells displayed significantly reduced association of ATGL with LD in all conditions (Fig. 6b,c, Supplementary Figure 5a,b) despite total ATGL levels remaining unchanged (Fig. 6e), pointing towards defective localization of this lipase on LD. Reduced lipolytic activity of ATGL in CMA-deficient cells could not be attributed to changes in its association with endogenous modulators because, on the contrary, pull-down for ATGL in L2A(−) cells revealed reduced binding to its inhibitor G0/G1 switch gene 2 (G0S2) and enhanced association with its co-activator comparative gene identification 58 (CGI-58) (Fig. 6f,g), likely reflecting cellular compensatory efforts to sustain lipolysis. Similar changes occur in vivo as association of ATGL with rat liver LD was enhanced upon starvation, which activates both lipolysis and CMA (Fig. 6h), but LD isolated from livers of starved L2AKO mice22 displayed a discrete but consistent reduction in ATGL levels and a more pronounced decrease in levels of its co-activator CGI-58 when compared to LD isolated from control mice (Fig. 6i, Supplementary Figure 5c). Furthermore, expression of the CMA-resistant PLIN2, but not the WT variant, was sufficient to significantly reduce the association of ATGL with LD (Fig. 6j). Thus, the inability to remove PLIN2 from LD in CMA-deficient cells leads to reduced lipolysis, in part because it impairs the association of the cytosolic lipase ATGL with LD.To investigate the reasons behind reduced lysosome-dependent lipophagy in CMA-defective cells (Fig. 6a), we first analyzed changes in LD abundance upon macroautophagy blockage (Fig. 7a, Supplementary Figure 5d). Addition of 3-methyladenine was efficient in reducing macroautophagy in both control and L2A(−) cells (Supplementary Figure 5e) but resulted in increased LD number only in control cells (Fig. 7a, Supplementary Figure 5d). Similarly, upregulation of macroautophagy with rapamycin resulted in a more pronounced decrease in LD content in control than in L2A(−) cells (Fig. 7b,c). This reduction in macrolipophagy in cells with defective CMA contrasts with the previously reported increase in non-selective macroautophagy in these cells25, that we also confirmed by LC3 staining (Supplementary Figure 5f). To investigate the reasons behind the macrolipophagy failure, we analyzed the recruitment of macroautophagy proteins (ATGs) to LD that occurs in liver when lipolysis is activated by prolonged starvation (Fig. 7d) and also in cultured fibroblasts upon OL challenge (Supplementary Figure 6a,b). Co-immunostaining in L2A(−) cells revealed a significant decrease in colocalization with LD of components of macroautophagy initiation complex (Beclin1, ATG14, vps15 and vps34), proteins involved in autophagosome elongation (ATG5) and the structural autophagosome component (LC3) (Fig. 7e,f, Supplementary Figure 6c,d). We also analyzed two cargo-recognition proteins and found that LD in L2A(−) cells show lower colocalization with NBR1 (Fig. 7e,f), the one that normally increases on LD during lipolysis (Fig. 7d, Supplementary Figure 6a,b), whereas colocalization with p62, that does not change during lipolysis, remained also unchanged in L2A(−) cells (Supplementary Figure 6e). Colocalization between LD and ATG7, the ligase that mediates elongation, remained unchanged whereas other ATG proteins such as ULK1, ATG9 and UVRAG did not colocalize with LD either in control or L2A(−) cells (Supplementary Figure 6e,f). Colocalization of Rab7 with LD, required for lysosome-mediated lipolysis37, was also significantly reduced in L2A(−) cells (Fig. 7g). In vivo analysis using isolated liver LD confirmed that levels of LC3, ATG5, Beclin1 and NBR1, were reduced in L2A-defective mice as compared to control mice (Fig. 7h). Furthermore, even in the context of functional CMA, expression of the CMA-resistant PLIN2 mutant was sufficient to significantly reduce the amount of LC3, ATG5 and Beclin1 associated to LD (Fig. 7i, Supplementary Figure 7a,b and Supplementary Video 4 and 3D reconstructions of Z-sections shown in Supplementary Figure 7c and Supplementary Video 5), confirming that inability to degrade LD-associated PLINs via CMA inhibited the association of the macroautophagy machinery to LD. We propose that failure to anchor ATGs at the LD surface prevents initiation of macrolipophagy. Ultrastructural analysis confirmed that the membranous structures observed in large LD in control cells upon induction of lipolysis (by OL challenge or starvation) and previously proposed as signature of macrolipophagy of portions of LD18, were almost completely absent in LD of L2A(−) cells (Supplementary Figure 7d). Collectively, these results indicate that removal of PLINs via CMA precedes macrolipophagy.Altogether, we propose that degradation of LD proteins PLIN2 and PLIN3 by CMA is required for the LD association of both the cytosolic lipase and the macroautophagy effector proteins, and for subsequent lipolysis, explaining the intracellular lipid accumulation observed upon CMA compromise (Fig. 8).

LAMP-2A-deficient cells accumulate LDUsing both, livers from mice conditionally knock-out for LAMP-2A (L2A) in hepatocytes22 (L2AKO) and mouse fibroblasts (NIH3T3 cells) knocked down for L2A (L2A(−)) to block CMA25 we confirmed that, despite lower dependence of fibroblasts on lipid metabolism when compared to hepatocytes, L2A-deficient fibroblasts accumulated significantly more triglycerides (TG) than control fibroblasts (Fig. 1a). These differences in TG content were even higher when intracellular lipid usage was forced by reducing glucose in the media or after a lipogenic stimulus (oleate; OL) (Fig. 1a).As in the L2AKO mice22, we only found a slight trend towards higher TG synthesis in L2A(−) cells compared to control cells (Fig. 1b). In contrast, L2A(−) cells showed significantly reduced β-oxidation rates, an event downstream of TG hydrolysis, both under basal and lipogenic conditions (Fig. 1c), and failed to increase oxygen consumption rates (OCR) upon OL exposure (Fig. 1d; CTR cells: Δ20.9+1.7pmol/min; L2A(−) cells: Δ−1.3+0.2pmol/min OCR change). Decreased β-oxidation rates in L2A(−) cells was not due to defective mitochondria per se, since the fraction of mitochondria with intact membrane potential (Supplementary Figure 1a) and mitochondria turnover (detected with pH-sensitive MitoKeima reporter26; Supplementary Figure 1b) were comparable to CTR cells.. Differences in β-oxidation rates between control and L2A(−) cells were minimized in presence of etomoxir (inhibitor of mitochondrial FFA uptake) (Fig. 1d; upon OL addition, OCR in CTR cells was 48.4+2.1pmol/min higher than in L2A(−) cells, but after adding etomoxir the difference between both cell-types was reduced to 17.2+1.6pmol/min) suggesting a problem in lipid mobilization and not in its oxidation per se.Consistent with the pronounced hepatosteatosis of L2AKO mice (Supplementary Figure 1c), L2A(−) fibroblasts accumulate more and bigger lipid droplets (LD) visible with either BODIPY493/503 (Fig. 1e,
Supplementary Figure 1d) or DPH (1,6-diphenylhexatriene27) (Supplementary Figure 1e,f), both under basal and OL-treated conditions. Unlike control cells, etomoxir-treated L2A(−) cells failed to increase LD supporting defective lipid mobilization (Supplementary Figure 1g). To further test this hypothesis, we used different experimental paradigms to force lipolysis in fibroblasts. OL-challenged control cells, upon removal of OL (OL>S+) showed a decrease in LD number and size, even more evident if maintained in serum-free media (OL>S−) (Fig. 1e, Supplementary Figure 1d–f). In contrast, OL-challenged L2A(−) cells maintained elevated LD number and size throughout. Similarly, culturing cells in low-glucose or in serum-free media, to facilitate cellular usage of lipid stores, revealed significantly reduced ability of L2A(−) cells to decrease their TG levels and LD content compared to control cells (Fig. 1a, Supplementary Figure 1h). Electron microscopy confirmed that LD number, average size, and occupied cellular area were significantly higher in CMA-deficient cells than in control cells in all conditions tested (Fig. 1f). LD accumulation in L2A(−) cells was a direct consequence of L2A loss since expressing L2A in these cells (Supplementary Figure 1i,j) was sufficient to reduce LD to control cell levels (Fig. 1g). Together, these results indicate that CMA blockage results in LD accumulation and this is primarily a consequence of decreased LD breakdown.

LAMP-2A-deficient cells accumulate LD

Using both, livers from mice conditionally knock-out for LAMP-2A (L2A) in hepatocytes22 (L2AKO) and mouse fibroblasts (NIH3T3 cells) knocked down for L2A (L2A(−)) to block CMA25 we confirmed that, despite lower dependence of fibroblasts on lipid metabolism when compared to hepatocytes, L2A-deficient fibroblasts accumulated significantly more triglycerides (TG) than control fibroblasts (Fig. 1a). These differences in TG content were even higher when intracellular lipid usage was forced by reducing glucose in the media or after a lipogenic stimulus (oleate; OL) (Fig. 1a).

As in the L2AKO mice22, we only found a slight trend towards higher TG synthesis in L2A(−) cells compared to control cells (Fig. 1b). In contrast, L2A(−) cells showed significantly reduced β-oxidation rates, an event downstream of TG hydrolysis, both under basal and lipogenic conditions (Fig. 1c), and failed to increase oxygen consumption rates (OCR) upon OL exposure (Fig. 1d; CTR cells: Δ20.9+1.7pmol/min; L2A(−) cells: Δ−1.3+0.2pmol/min OCR change). Decreased β-oxidation rates in L2A(−) cells was not due to defective mitochondria per se, since the fraction of mitochondria with intact membrane potential (Supplementary Figure 1a) and mitochondria turnover (detected with pH-sensitive MitoKeima reporter26; Supplementary Figure 1b) were comparable to CTR cells.. Differences in β-oxidation rates between control and L2A(−) cells were minimized in presence of etomoxir (inhibitor of mitochondrial FFA uptake) (Fig. 1d; upon OL addition, OCR in CTR cells was 48.4+2.1pmol/min higher than in L2A(−) cells, but after adding etomoxir the difference between both cell-types was reduced to 17.2+1.6pmol/min) suggesting a problem in lipid mobilization and not in its oxidation per se.

Consistent with the pronounced hepatosteatosis of L2AKO mice (Supplementary Figure 1c), L2A(−) fibroblasts accumulate more and bigger lipid droplets (LD) visible with either BODIPY493/503 (Fig. 1e,
Supplementary Figure 1d) or DPH (1,6-diphenylhexatriene27) (Supplementary Figure 1e,f), both under basal and OL-treated conditions. Unlike control cells, etomoxir-treated L2A(−) cells failed to increase LD supporting defective lipid mobilization (Supplementary Figure 1g). To further test this hypothesis, we used different experimental paradigms to force lipolysis in fibroblasts. OL-challenged control cells, upon removal of OL (OL>S+) showed a decrease in LD number and size, even more evident if maintained in serum-free media (OL>S−) (Fig. 1e, Supplementary Figure 1d–f). In contrast, OL-challenged L2A(−) cells maintained elevated LD number and size throughout. Similarly, culturing cells in low-glucose or in serum-free media, to facilitate cellular usage of lipid stores, revealed significantly reduced ability of L2A(−) cells to decrease their TG levels and LD content compared to control cells (Fig. 1a, Supplementary Figure 1h). Electron microscopy confirmed that LD number, average size, and occupied cellular area were significantly higher in CMA-deficient cells than in control cells in all conditions tested (Fig. 1f). LD accumulation in L2A(−) cells was a direct consequence of L2A loss since expressing L2A in these cells (Supplementary Figure 1i,j) was sufficient to reduce LD to control cell levels (Fig. 1g). Together, these results indicate that CMA blockage results in LD accumulation and this is primarily a consequence of decreased LD breakdown.

LD proteins PLIN2 and PLIN3 are CMA substratesSince only proteins and not lipids can be CMA substrates, we next investigated a role of CMA in degradation of LD proteins, which could explain defective LD utilization in CMA-deficient cells. Immunoblotting (Fig. 2a, Supplementary Figure 2a) and immunofluorescence (Fig. 2b) for the ubiquitously expressed PLIN2 and PLIN3 showed significantly higher levels under basal, OL treatment and post-OL treatment in L2A(−) cells. Consistent with the described PLIN2 turnover by the ubiquitin-proteasome pathway28,29, proteasomal inhibitors increased LD number (Supplementary Figure 2b) and PLIN2 levels (Supplementary Figure 2c) in control cells but not in CMA-defective cells, although proteasomal degradation of ubiquitinated proteins and a degron-GFP reporter30 were preserved in these cells (Supplementary Figure 2c–e). Lipid accumulation upon proteasomal or lysosomal inhibition were comparable (Supplementary Figure 2b), leaving open the possibility that PLINs could also be amenable for lysosomal degradation.To examine the possible contribution of CMA to PLIN2 and PLIN3 degradation, we first investigated their association with lysosomes. In support of PLINs lysosomal turnover, inhibiting lysosomal degradation with ammonium chloride and leupeptin in control cells significantly increased the colocalization of PLIN2 or PLIN3 with the lysosomal marker (LAMP1), and more so upon OL addition (Fig. 2c,d, Supplementary Figure 2f,g). In contrast, L2A(−) cells showed decreased association of PLIN2 and PLIN3 with lysosomes and impaired flux with lysosomal inhibitors (Fig. 2c,d, Supplementary Figure 2f,g). PLIN2 and PLIN3 were enriched in lysosomes active for CMA (CMA+, containing hsc70 and LAMP-2A31) as compared to lysosomes with lower CMA activity (CMA−, containing LAMP-2A but deficient in hsc7031) (Fig. 2e, glyceraldehyde 3-phosphate dehydrogenase (GAPDH), a CMA substrate32, is shown as a positive control). Lysosomal levels of both PLINs were reduced in L2A(−) cells (Fig. 2f). We confirmed that lysosomal association of PLIN2 and PLIN3 led to their degradation in vivo, since blocking lysosomal proteolysis in starved rats with leupeptin for 2h was sufficient to increase levels of these LD proteins in CMA-active lysosomes (Fig. 2g, Supplementary Figure 2h). CMA degradation of PLIN2 was also detected in fed mice, albeit at lower levels than in starved animals (Fig. 2g), whereas lysosomal degradation of PLIN3 in fed conditions was negligible (Fig. 2g). Some degradation of PLINs was observed in CMA- lysosomes, likely due to LD degradation by macrolipophagy, since these lysosomes preferentially fuse with autophagosomes33. Collectively, results from our in vitro and in vivo studies indicate that CMA degrades PLIN2 and PLIN3.

LD proteins PLIN2 and PLIN3 are CMA substrates

Since only proteins and not lipids can be CMA substrates, we next investigated a role of CMA in degradation of LD proteins, which could explain defective LD utilization in CMA-deficient cells. Immunoblotting (Fig. 2a, Supplementary Figure 2a) and immunofluorescence (Fig. 2b) for the ubiquitously expressed PLIN2 and PLIN3 showed significantly higher levels under basal, OL treatment and post-OL treatment in L2A(−) cells. Consistent with the described PLIN2 turnover by the ubiquitin-proteasome pathway28,29, proteasomal inhibitors increased LD number (Supplementary Figure 2b) and PLIN2 levels (Supplementary Figure 2c) in control cells but not in CMA-defective cells, although proteasomal degradation of ubiquitinated proteins and a degron-GFP reporter30 were preserved in these cells (Supplementary Figure 2c–e). Lipid accumulation upon proteasomal or lysosomal inhibition were comparable (Supplementary Figure 2b), leaving open the possibility that PLINs could also be amenable for lysosomal degradation.

To examine the possible contribution of CMA to PLIN2 and PLIN3 degradation, we first investigated their association with lysosomes. In support of PLINs lysosomal turnover, inhibiting lysosomal degradation with ammonium chloride and leupeptin in control cells significantly increased the colocalization of PLIN2 or PLIN3 with the lysosomal marker (LAMP1), and more so upon OL addition (Fig. 2c,d, Supplementary Figure 2f,g). In contrast, L2A(−) cells showed decreased association of PLIN2 and PLIN3 with lysosomes and impaired flux with lysosomal inhibitors (Fig. 2c,d, Supplementary Figure 2f,g). PLIN2 and PLIN3 were enriched in lysosomes active for CMA (CMA+, containing hsc70 and LAMP-2A31) as compared to lysosomes with lower CMA activity (CMA−, containing LAMP-2A but deficient in hsc7031) (Fig. 2e, glyceraldehyde 3-phosphate dehydrogenase (GAPDH), a CMA substrate32, is shown as a positive control). Lysosomal levels of both PLINs were reduced in L2A(−) cells (Fig. 2f). We confirmed that lysosomal association of PLIN2 and PLIN3 led to their degradation in vivo, since blocking lysosomal proteolysis in starved rats with leupeptin for 2h was sufficient to increase levels of these LD proteins in CMA-active lysosomes (Fig. 2g, Supplementary Figure 2h). CMA degradation of PLIN2 was also detected in fed mice, albeit at lower levels than in starved animals (Fig. 2g), whereas lysosomal degradation of PLIN3 in fed conditions was negligible (Fig. 2g). Some degradation of PLINs was observed in CMA- lysosomes, likely due to LD degradation by macrolipophagy, since these lysosomes preferentially fuse with autophagosomes33. Collectively, results from our in vitro and in vivo studies indicate that CMA degrades PLIN2 and PLIN3.

PLINs interact with CMA chaperone hsc70The first step in CMA is substrate interaction with hsc70 for subsequent lysosomal targeting. We found hsc70 in isolated rat liver LD and its levels increased during starvation, when hepatic lipolysis is highly active, coinciding with a decrease in LD levels of PLIN2 and PLIN3 (Fig. 3a). Immunofluorescence confirmed hsc70 colocalization with each PLIN on LD, which increased upon OL-challenge that induces lipolysis (Fig. 3b,c, Supplementary Figure 3a). Forcing lipid mobilization by placing cells in serum-free media post-OL challenge reduced association of hsc70 with LD (Supplementary Figure 3b). Remarkably, L2A(−) cells exhibited higher hsc70 colocalization with PLIN2 or PLIN3 in LD under all conditions (Fig. 3b,c, Supplementary Figure 3a,b). Similar higher abundance of hsc70 was also observed in LD isolated from livers of L2AKO mice compared to control littermates (Fig. 3d).We found direct interaction of hsc70 with PLIN2 and with PLIN3 in cultured cells. Hsc70 was recovered in PLIN2 and 3 pull-downs (Fig. 3e,f) and both PLINs were also detected in hsc70 pull-downs (Supplementary Figure 3c,d). For the same amount of PLINs pulled-down, we consistently observed higher levels of bound hsc70 in L2A(−) cells (Fig. 3e,f). Increased binding to hsc70 is characteristic of CMA substrates in these cells where disruption of CMA occurs at the level of the lysosomal receptor (L2A) while substrate recognition by hsc70 is intact (Supplementary Figure 3e shows the same effect in two other CMA substrates). This enhanced binding of hsc70 for multiple substrates explains why higher levels of PLINs (a single substrate) are not observed in hsc70 pull-downs (Supplementary Figure 3c,d). Upon treatment of L2A(−) cells with increasing OL concentrations previously shown to activate CMA15, hsc70 bound to PLIN2 or PLIN3 further increased whereas in control cells they remained constant, supporting continuous turnover of LD proteins by CMA (Fig. 3e,f, Supplementary Figure 3f). Since contrary to the almost exclusive LD location of PLIN2, PLIN3 can be present in other cellular compartments, we isolated mouse liver LD and confirmed that interaction of hsc70 with PLINs occurs in LD (Fig. 3g). These results are compatible with PLIN2 and PLIN3 being recognized by the CMA chaperone hsc70 at the LD surface before undergoing CMA-mediated degradation.

PLINs interact with CMA chaperone hsc70

The first step in CMA is substrate interaction with hsc70 for subsequent lysosomal targeting. We found hsc70 in isolated rat liver LD and its levels increased during starvation, when hepatic lipolysis is highly active, coinciding with a decrease in LD levels of PLIN2 and PLIN3 (Fig. 3a). Immunofluorescence confirmed hsc70 colocalization with each PLIN on LD, which increased upon OL-challenge that induces lipolysis (Fig. 3b,c, Supplementary Figure 3a). Forcing lipid mobilization by placing cells in serum-free media post-OL challenge reduced association of hsc70 with LD (Supplementary Figure 3b). Remarkably, L2A(−) cells exhibited higher hsc70 colocalization with PLIN2 or PLIN3 in LD under all conditions (Fig. 3b,c, Supplementary Figure 3a,b). Similar higher abundance of hsc70 was also observed in LD isolated from livers of L2AKO mice compared to control littermates (Fig. 3d).

We found direct interaction of hsc70 with PLIN2 and with PLIN3 in cultured cells. Hsc70 was recovered in PLIN2 and 3 pull-downs (Fig. 3e,f) and both PLINs were also detected in hsc70 pull-downs (Supplementary Figure 3c,d). For the same amount of PLINs pulled-down, we consistently observed higher levels of bound hsc70 in L2A(−) cells (Fig. 3e,f). Increased binding to hsc70 is characteristic of CMA substrates in these cells where disruption of CMA occurs at the level of the lysosomal receptor (L2A) while substrate recognition by hsc70 is intact (Supplementary Figure 3e shows the same effect in two other CMA substrates). This enhanced binding of hsc70 for multiple substrates explains why higher levels of PLINs (a single substrate) are not observed in hsc70 pull-downs (Supplementary Figure 3c,d). Upon treatment of L2A(−) cells with increasing OL concentrations previously shown to activate CMA15, hsc70 bound to PLIN2 or PLIN3 further increased whereas in control cells they remained constant, supporting continuous turnover of LD proteins by CMA (Fig. 3e,f, Supplementary Figure 3f). Since contrary to the almost exclusive LD location of PLIN2, PLIN3 can be present in other cellular compartments, we isolated mouse liver LD and confirmed that interaction of hsc70 with PLINs occurs in LD (Fig. 3g). These results are compatible with PLIN2 and PLIN3 being recognized by the CMA chaperone hsc70 at the LD surface before undergoing CMA-mediated degradation.

Modifications of CMA-targeted PLINsTo further confirm the involvement of CMA in PLINs turnover, we activated CMA in OL-challenged cells using an atypical retinoic acid derivative (aRAd)34. Chemical activation of CMA significantly reduced PLIN2 levels (Fig. 4a) and LD content (Fig. 4b) in control cells but not in L2A(−) cells. In fact, restoration of L2A levels in L2A(−) cells through transient transfection was sufficient to bring PLIN2 levels close to control cell values (Fig. 4b).To identify possible posttranslational modifications that trigger PLIN2 for CMA degradation we performed bidimensional electrophoresis and immunoblot and found two very prominent forms of PLIN2 of 5.6 and 5.9 isoelectric point (pI) and a less abundant isoform of pI 5.3. When lipolysis and subsequent PLIN2 degradation was induced with OL, the less positively charged isoform was no longer observed in control cells but it persisted in L2A(−) cells (Fig. 4c). Phosphatase treatment confirmed that PLIN2 was phosphorylated during lipolysis in CTR cells but not in L2A(−) cells that accumulated a form of PLIN2 unchanged by the phosphatase (Fig. 4c). Analysis of LD by electrophoresis using Phos-tag (for resolution of phosphorylated variants) confirmed the presence of a phosphorylated form of PLIN2 in control cells almost completely absent in L2A(−) cells (Fig. 4d). Although further studies are needed to elucidate the reasons for reduced levels of phospho-PLIN2 in absence of functional CMA, our findings suggest that phosphorylation of PLIN2 may be required to trigger its CMA degradation.

Modifications of CMA-targeted PLINs

To further confirm the involvement of CMA in PLINs turnover, we activated CMA in OL-challenged cells using an atypical retinoic acid derivative (aRAd)34. Chemical activation of CMA significantly reduced PLIN2 levels (Fig. 4a) and LD content (Fig. 4b) in control cells but not in L2A(−) cells. In fact, restoration of L2A levels in L2A(−) cells through transient transfection was sufficient to bring PLIN2 levels close to control cell values (Fig. 4b).

To identify possible posttranslational modifications that trigger PLIN2 for CMA degradation we performed bidimensional electrophoresis and immunoblot and found two very prominent forms of PLIN2 of 5.6 and 5.9 isoelectric point (pI) and a less abundant isoform of pI 5.3. When lipolysis and subsequent PLIN2 degradation was induced with OL, the less positively charged isoform was no longer observed in control cells but it persisted in L2A(−) cells (Fig. 4c). Phosphatase treatment confirmed that PLIN2 was phosphorylated during lipolysis in CTR cells but not in L2A(−) cells that accumulated a form of PLIN2 unchanged by the phosphatase (Fig. 4c). Analysis of LD by electrophoresis using Phos-tag (for resolution of phosphorylated variants) confirmed the presence of a phosphorylated form of PLIN2 in control cells almost completely absent in L2A(−) cells (Fig. 4d). Although further studies are needed to elucidate the reasons for reduced levels of phospho-PLIN2 in absence of functional CMA, our findings suggest that phosphorylation of PLIN2 may be required to trigger its CMA degradation.

CMA-resistant PLIN2 causes steatosisHsc70 binds CMA substrates through a pentapeptide motif4. PLIN3 bears a canonical CMA-motif (100LDRLQ) and PLIN2 a putative motif (414SLKVQ). To analyze the consequences of preventing CMA degradation of PLIN2, independently from other changes that may occur during CMA blockage, we mutated the PLIN2 CMA-targeting motif to 414SLKAA. Expression of wild-type (WT) and CMA-mutant (MT) PLIN2-GFP in control cells led to higher levels of MT PLIN2-GFP (Fig. 5a) and a ~2.7-fold increase in LD content and size as compared to WT PLIN2-expressing cells under basal conditions (Fig. 5b) and upon OL-challenge (Fig. 5c). MT PLIN2-GFP accumulation was mainly due to its reduced lysosomal degradation as, treatment with lysosomal inhibitors (NL) increased WT but not MT PLIN2-GFP levels (Fig. 5c) and, unlike WT PLIN2, failed to increase colocalization of MT PLIN2 with the lysosomal marker LAMP1 (Fig. 5d).We confirmed that disruption of the CMA-targeting motif in PLIN2 almost completely abrogates its binding to hsc70 (Fig. 5e), explaining the reduced colocalization of hsc70 with LD in cells expressing MT PLIN2 (Fig. 5f,g; 3D reconstruction of Z-sections shown in Supplementary Figure 4a and Supplementary Video 1). Persistence of MT PLIN2 on LD altered their cellular dynamics and their interplay with lysosomes. Thus, live-cell imaging of cells co-expressing WT PLIN2-GFP and LAMP1-RFP revealed that WT PLIN2-GFP LD were predominantly small and very mobile as reported earlier35, 36 (Fig. 5h–j, Supplementary Video 2 (green channel only)) and showed dynamic inter-LD associations and kiss-and-run events between LD and lysosomes (Fig. 5h–j, Supplementary Video 3 (merged channels)). In striking contrast, majority of LD in MT PLIN2-expressing cells formed larger, less dynamic clusters and the very few smaller LD rarely associated with lysosomes (Fig. 5h–j; Z-sections stack shown in Supplementary Figure 4b, Supplementary Video 2,3). In summary, preventing degradation of PLIN2 by CMA is sufficient to induce the intracellular lipid accumulation observed in CMA-deficient cells (Fig. 1 and Fig. 2) and to reduce LD/lysosome interactions.

CMA-resistant PLIN2 causes steatosis

Hsc70 binds CMA substrates through a pentapeptide motif4. PLIN3 bears a canonical CMA-motif (100LDRLQ) and PLIN2 a putative motif (414SLKVQ). To analyze the consequences of preventing CMA degradation of PLIN2, independently from other changes that may occur during CMA blockage, we mutated the PLIN2 CMA-targeting motif to 414SLKAA. Expression of wild-type (WT) and CMA-mutant (MT) PLIN2-GFP in control cells led to higher levels of MT PLIN2-GFP (Fig. 5a) and a ~2.7-fold increase in LD content and size as compared to WT PLIN2-expressing cells under basal conditions (Fig. 5b) and upon OL-challenge (Fig. 5c). MT PLIN2-GFP accumulation was mainly due to its reduced lysosomal degradation as, treatment with lysosomal inhibitors (NL) increased WT but not MT PLIN2-GFP levels (Fig. 5c) and, unlike WT PLIN2, failed to increase colocalization of MT PLIN2 with the lysosomal marker LAMP1 (Fig. 5d).

We confirmed that disruption of the CMA-targeting motif in PLIN2 almost completely abrogates its binding to hsc70 (Fig. 5e), explaining the reduced colocalization of hsc70 with LD in cells expressing MT PLIN2 (Fig. 5f,g; 3D reconstruction of Z-sections shown in Supplementary Figure 4a and Supplementary Video 1). Persistence of MT PLIN2 on LD altered their cellular dynamics and their interplay with lysosomes. Thus, live-cell imaging of cells co-expressing WT PLIN2-GFP and LAMP1-RFP revealed that WT PLIN2-GFP LD were predominantly small and very mobile as reported earlier35, 36 (Fig. 5h–j, Supplementary Video 2 (green channel only)) and showed dynamic inter-LD associations and kiss-and-run events between LD and lysosomes (Fig. 5h–j, Supplementary Video 3 (merged channels)). In striking contrast, majority of LD in MT PLIN2-expressing cells formed larger, less dynamic clusters and the very few smaller LD rarely associated with lysosomes (Fig. 5h–j; Z-sections stack shown in Supplementary Figure 4b, Supplementary Video 2,3). In summary, preventing degradation of PLIN2 by CMA is sufficient to induce the intracellular lipid accumulation observed in CMA-deficient cells (Fig. 1 and Fig. 2) and to reduce LD/lysosome interactions.

Defective CMA of PLINs blocks lipolysisTG in LD are broken down into FFA either by cytosolic lipases21, 23 or by lysosomal lipases that gain access to these cytosolic stores via macrolipophagy18. To assess which lipolytic pathway failed to mobilize LD in L2A(−) cells (Fig. 1), we treated cells with a lipase inhibitor, diethylumbelliferyl phosphate (Deup), and with lysosomal inhibitors (NL) and found that βoxidation decreased in control cells but L2A(−) cells remained unresponsive (Fig. 6a). Overall these findings suggested that blocking CMA results in impaired lipolysis by cytosolic lipases and macrolipophagy.The rate-limiting cytosolic lipase in most cell-types, adipose triglyceride lipase (ATGL)21, colocalizes with LD (lipid- or protein-labeled) during lipogenic stimulus (OL) and in OL>S- or OL>low-glucose S+ conditions to induce lipolysis (Fig. 6b,c, Supplementary Figure 5a,b). Similarly, chemical activation of CMA increased colocalization of ATGL with LD (Fig. 6d). In contrast, CMA-defective cells displayed significantly reduced association of ATGL with LD in all conditions (Fig. 6b,c, Supplementary Figure 5a,b) despite total ATGL levels remaining unchanged (Fig. 6e), pointing towards defective localization of this lipase on LD. Reduced lipolytic activity of ATGL in CMA-deficient cells could not be attributed to changes in its association with endogenous modulators because, on the contrary, pull-down for ATGL in L2A(−) cells revealed reduced binding to its inhibitor G0/G1 switch gene 2 (G0S2) and enhanced association with its co-activator comparative gene identification 58 (CGI-58) (Fig. 6f,g), likely reflecting cellular compensatory efforts to sustain lipolysis. Similar changes occur in vivo as association of ATGL with rat liver LD was enhanced upon starvation, which activates both lipolysis and CMA (Fig. 6h), but LD isolated from livers of starved L2AKO mice22 displayed a discrete but consistent reduction in ATGL levels and a more pronounced decrease in levels of its co-activator CGI-58 when compared to LD isolated from control mice (Fig. 6i, Supplementary Figure 5c). Furthermore, expression of the CMA-resistant PLIN2, but not the WT variant, was sufficient to significantly reduce the association of ATGL with LD (Fig. 6j). Thus, the inability to remove PLIN2 from LD in CMA-deficient cells leads to reduced lipolysis, in part because it impairs the association of the cytosolic lipase ATGL with LD.To investigate the reasons behind reduced lysosome-dependent lipophagy in CMA-defective cells (Fig. 6a), we first analyzed changes in LD abundance upon macroautophagy blockage (Fig. 7a, Supplementary Figure 5d). Addition of 3-methyladenine was efficient in reducing macroautophagy in both control and L2A(−) cells (Supplementary Figure 5e) but resulted in increased LD number only in control cells (Fig. 7a, Supplementary Figure 5d). Similarly, upregulation of macroautophagy with rapamycin resulted in a more pronounced decrease in LD content in control than in L2A(−) cells (Fig. 7b,c). This reduction in macrolipophagy in cells with defective CMA contrasts with the previously reported increase in non-selective macroautophagy in these cells25, that we also confirmed by LC3 staining (Supplementary Figure 5f). To investigate the reasons behind the macrolipophagy failure, we analyzed the recruitment of macroautophagy proteins (ATGs) to LD that occurs in liver when lipolysis is activated by prolonged starvation (Fig. 7d) and also in cultured fibroblasts upon OL challenge (Supplementary Figure 6a,b). Co-immunostaining in L2A(−) cells revealed a significant decrease in colocalization with LD of components of macroautophagy initiation complex (Beclin1, ATG14, vps15 and vps34), proteins involved in autophagosome elongation (ATG5) and the structural autophagosome component (LC3) (Fig. 7e,f, Supplementary Figure 6c,d). We also analyzed two cargo-recognition proteins and found that LD in L2A(−) cells show lower colocalization with NBR1 (Fig. 7e,f), the one that normally increases on LD during lipolysis (Fig. 7d, Supplementary Figure 6a,b), whereas colocalization with p62, that does not change during lipolysis, remained also unchanged in L2A(−) cells (Supplementary Figure 6e). Colocalization between LD and ATG7, the ligase that mediates elongation, remained unchanged whereas other ATG proteins such as ULK1, ATG9 and UVRAG did not colocalize with LD either in control or L2A(−) cells (Supplementary Figure 6e,f). Colocalization of Rab7 with LD, required for lysosome-mediated lipolysis37, was also significantly reduced in L2A(−) cells (Fig. 7g). In vivo analysis using isolated liver LD confirmed that levels of LC3, ATG5, Beclin1 and NBR1, were reduced in L2A-defective mice as compared to control mice (Fig. 7h). Furthermore, even in the context of functional CMA, expression of the CMA-resistant PLIN2 mutant was sufficient to significantly reduce the amount of LC3, ATG5 and Beclin1 associated to LD (Fig. 7i, Supplementary Figure 7a,b and Supplementary Video 4 and 3D reconstructions of Z-sections shown in Supplementary Figure 7c and Supplementary Video 5), confirming that inability to degrade LD-associated PLINs via CMA inhibited the association of the macroautophagy machinery to LD. We propose that failure to anchor ATGs at the LD surface prevents initiation of macrolipophagy. Ultrastructural analysis confirmed that the membranous structures observed in large LD in control cells upon induction of lipolysis (by OL challenge or starvation) and previously proposed as signature of macrolipophagy of portions of LD18, were almost completely absent in LD of L2A(−) cells (Supplementary Figure 7d). Collectively, these results indicate that removal of PLINs via CMA precedes macrolipophagy.Altogether, we propose that degradation of LD proteins PLIN2 and PLIN3 by CMA is required for the LD association of both the cytosolic lipase and the macroautophagy effector proteins, and for subsequent lipolysis, explaining the intracellular lipid accumulation observed upon CMA compromise (Fig. 8).

Defective CMA of PLINs blocks lipolysis

TG in LD are broken down into FFA either by cytosolic lipases21, 23 or by lysosomal lipases that gain access to these cytosolic stores via macrolipophagy18. To assess which lipolytic pathway failed to mobilize LD in L2A(−) cells (Fig. 1), we treated cells with a lipase inhibitor, diethylumbelliferyl phosphate (Deup), and with lysosomal inhibitors (NL) and found that βoxidation decreased in control cells but L2A(−) cells remained unresponsive (Fig. 6a). Overall these findings suggested that blocking CMA results in impaired lipolysis by cytosolic lipases and macrolipophagy.

The rate-limiting cytosolic lipase in most cell-types, adipose triglyceride lipase (ATGL)21, colocalizes with LD (lipid- or protein-labeled) during lipogenic stimulus (OL) and in OL>S- or OL>low-glucose S+ conditions to induce lipolysis (Fig. 6b,c, Supplementary Figure 5a,b). Similarly, chemical activation of CMA increased colocalization of ATGL with LD (Fig. 6d). In contrast, CMA-defective cells displayed significantly reduced association of ATGL with LD in all conditions (Fig. 6b,c, Supplementary Figure 5a,b) despite total ATGL levels remaining unchanged (Fig. 6e), pointing towards defective localization of this lipase on LD. Reduced lipolytic activity of ATGL in CMA-deficient cells could not be attributed to changes in its association with endogenous modulators because, on the contrary, pull-down for ATGL in L2A(−) cells revealed reduced binding to its inhibitor G0/G1 switch gene 2 (G0S2) and enhanced association with its co-activator comparative gene identification 58 (CGI-58) (Fig. 6f,g), likely reflecting cellular compensatory efforts to sustain lipolysis. Similar changes occur in vivo as association of ATGL with rat liver LD was enhanced upon starvation, which activates both lipolysis and CMA (Fig. 6h), but LD isolated from livers of starved L2AKO mice22 displayed a discrete but consistent reduction in ATGL levels and a more pronounced decrease in levels of its co-activator CGI-58 when compared to LD isolated from control mice (Fig. 6i, Supplementary Figure 5c). Furthermore, expression of the CMA-resistant PLIN2, but not the WT variant, was sufficient to significantly reduce the association of ATGL with LD (Fig. 6j). Thus, the inability to remove PLIN2 from LD in CMA-deficient cells leads to reduced lipolysis, in part because it impairs the association of the cytosolic lipase ATGL with LD.

To investigate the reasons behind reduced lysosome-dependent lipophagy in CMA-defective cells (Fig. 6a), we first analyzed changes in LD abundance upon macroautophagy blockage (Fig. 7a, Supplementary Figure 5d). Addition of 3-methyladenine was efficient in reducing macroautophagy in both control and L2A(−) cells (Supplementary Figure 5e) but resulted in increased LD number only in control cells (Fig. 7a, Supplementary Figure 5d). Similarly, upregulation of macroautophagy with rapamycin resulted in a more pronounced decrease in LD content in control than in L2A(−) cells (Fig. 7b,c). This reduction in macrolipophagy in cells with defective CMA contrasts with the previously reported increase in non-selective macroautophagy in these cells25, that we also confirmed by LC3 staining (Supplementary Figure 5f). To investigate the reasons behind the macrolipophagy failure, we analyzed the recruitment of macroautophagy proteins (ATGs) to LD that occurs in liver when lipolysis is activated by prolonged starvation (Fig. 7d) and also in cultured fibroblasts upon OL challenge (Supplementary Figure 6a,b). Co-immunostaining in L2A(−) cells revealed a significant decrease in colocalization with LD of components of macroautophagy initiation complex (Beclin1, ATG14, vps15 and vps34), proteins involved in autophagosome elongation (ATG5) and the structural autophagosome component (LC3) (Fig. 7e,f, Supplementary Figure 6c,d). We also analyzed two cargo-recognition proteins and found that LD in L2A(−) cells show lower colocalization with NBR1 (Fig. 7e,f), the one that normally increases on LD during lipolysis (Fig. 7d, Supplementary Figure 6a,b), whereas colocalization with p62, that does not change during lipolysis, remained also unchanged in L2A(−) cells (Supplementary Figure 6e). Colocalization between LD and ATG7, the ligase that mediates elongation, remained unchanged whereas other ATG proteins such as ULK1, ATG9 and UVRAG did not colocalize with LD either in control or L2A(−) cells (Supplementary Figure 6e,f). Colocalization of Rab7 with LD, required for lysosome-mediated lipolysis37, was also significantly reduced in L2A(−) cells (Fig. 7g). In vivo analysis using isolated liver LD confirmed that levels of LC3, ATG5, Beclin1 and NBR1, were reduced in L2A-defective mice as compared to control mice (Fig. 7h). Furthermore, even in the context of functional CMA, expression of the CMA-resistant PLIN2 mutant was sufficient to significantly reduce the amount of LC3, ATG5 and Beclin1 associated to LD (Fig. 7i, Supplementary Figure 7a,b and Supplementary Video 4 and 3D reconstructions of Z-sections shown in Supplementary Figure 7c and Supplementary Video 5), confirming that inability to degrade LD-associated PLINs via CMA inhibited the association of the macroautophagy machinery to LD. We propose that failure to anchor ATGs at the LD surface prevents initiation of macrolipophagy. Ultrastructural analysis confirmed that the membranous structures observed in large LD in control cells upon induction of lipolysis (by OL challenge or starvation) and previously proposed as signature of macrolipophagy of portions of LD18, were almost completely absent in LD of L2A(−) cells (Supplementary Figure 7d). Collectively, these results indicate that removal of PLINs via CMA precedes macrolipophagy.

Altogether, we propose that degradation of LD proteins PLIN2 and PLIN3 by CMA is required for the LD association of both the cytosolic lipase and the macroautophagy effector proteins, and for subsequent lipolysis, explaining the intracellular lipid accumulation observed upon CMA compromise (Fig. 8).

DiscussionIn this study, we demonstrate the role of CMA in regulating mobilization of intracellular lipid stores by degrading LD proteins PLIN2 and PLIN3. Removal of these LD proteins is a prerequisite for subsequent lipolysis to occur, since failure to remove these proteins via CMA results in decreased LD association of components of both arms of the lipolytic machinery – cytosolic lipases and macrolipophagy (Fig. 8).TG in LD are broken down by cytosolic lipases to generate FFA that undergo β-oxidation in mitochondria21, 23 and by lysosomal lipases upon sequestration of complete or fractions of LD by macrolipophagy18, 19. ATGs and ATG-positive membranes are detected in LD during lipolysis, suggesting that the autophagosome limiting membrane forms in situ on LD surface. In fact, recent studies have shown that LD are a site for autophagosome biogenesis38. We have uncovered that CMA is required prior to the selective sequestration of LD by the autophagosome, suggesting that for ATGs to associate and initiate autophagosome biogenesis, LD peripheral proteins would have to be removed from the LD surface. Consequently, the only residual lipolysis observed in CMA-deficient cells could be attributable to in-bulk macroautophagy, shown upregulated in these cells25, and where cargo recognition is not required.Cytosolic lipolysis is regulated by complex interactions between PLINs and lipases39. ATGL associates with LD under maximal lipolytic conditions39 and PLINs at the LD surface exert a ‘shielding effect’ that modulates the lipase’s accessibility to TG. In this regard, overexpression of PLIN2 reduces LD association of ATGL and hence, lipolysis39, 40. We propose that CMA facilitates lipolysis by removing PLINs from LD since we found decreased association of ATGL on LD in cells with elevated PLIN2 levels, either due to impaired CMA or due to expression of the CMA mutant. In addition, PLIN2 removal could also facilitate binding of the coactivator to ATGL already on LD, since CMA blockage reduced levels of CGI-58 in LD.Although PLIN2 has been reported to be a proteasome substrate28, 29, both lysosomes and proteasomes can degrade the same protein depending on the cellular requirement or the cell-type, as reported, for example, for PLIN141–43. In our study, we observed both lysosomal and proteasomal degradation of PLIN2, but interestingly, CMA blockage also prevented PLIN2 proteasomal degradation. Although it is plausible that CMA might indirectly facilitate proteasomal degradation of PLIN2, our data disfavors this option since (i) PLIN2 directly interacts with hsc70; (ii) we detect PLIN2 in CMA-active lysosomes and its degradation in this compartment; and (iii) eliminating the CMA-targeting motif retains PLIN2 in LD. We propose thus that PLIN2 is directly identified by hsc70 in LD and targeted for CMA degradation. Proteasome may only be able to access LD PLIN2 once its removal has been initiated by CMA or alternatively, the PLIN2 amenable to proteasomal degradation could be located in a different compartment, as described before29.We describe PLIN2 and PLIN3 as the first lipid-anchored proteins to be CMA substrates. Presence of hsc70 on LD, also described before44–47 and its persistence at the LD surface when CMA is blocked, together with the altered LD and lysosomal dynamics in these conditions, suggests the presence of a microenvironment at the LD surface, wherein lysosomes and LD are in close proximity. Dynamic and transient LD interactions, similar to the ones we propose here with lysosomes, have been described between LD and other organelles, such as mitochondria, and peroxisomes48–50, to channel FFA from lipolysis to sites of oxidation20. Our studies with live-cell microscopy support marked reduction in kiss-and-run events between LD and lysosomes in conditions that compromise PLIN degradation by CMA such as mutations in the CMA-targeting motif of PLIN2 that prevent hsc70 interaction with this protein.We thus, propose that degradation of PLINs via CMA occurs in discrete areas on LD to facilitate punctual removal of PLINs from the LD surface for ATGL and ATGs to access the stored TG. ATGL rapidly cycles on and off of LD to catalyze lipolysis39 and similarly, association of LC3 with LD is polarized and occurs in discrete areas on the LD surface from where the limiting membrane starts forming18. Although the mechanisms that determine removal of only a fraction of PLINs in a localized region of LD require future investigation, we speculate that changes in PLINs phosphorylation may facilitate the release of a specific fraction of PLINs from LD. In fact, PLIN2 has been already identified as part of the phosphoproteome of LD51.We propose that the pronounced hepatic lipid accumulation observed in L2AKO mice results from a combination of: increased levels of lipogenic enzymes degraded through CMA22, reduced VLDL secretion22, and reduced lipolysis due to the inability of cytosolic lipases and ATGs to bind to LD when PLINs are not degraded by CMA. We have previously shown that chronic lipid loading inhibits CMA activity15. In light of our current findings of regulation of lipid stores by CMA, it is conceivable that a compromise in CMA would set up a viscous cycle of lipid overloading in cells. CMA activity decreases with age16, and it is plausible that changes in CMA would impact cellular lipohomeostasis and the steatosis observed during aging, which in turn would further suppress CMA activity. Manipulating CMA could thus be a viable therapeutic option against excessive lipid accumulation.

In this study, we demonstrate the role of CMA in regulating mobilization of intracellular lipid stores by degrading LD proteins PLIN2 and PLIN3. Removal of these LD proteins is a prerequisite for subsequent lipolysis to occur, since failure to remove these proteins via CMA results in decreased LD association of components of both arms of the lipolytic machinery – cytosolic lipases and macrolipophagy (Fig. 8).

TG in LD are broken down by cytosolic lipases to generate FFA that undergo β-oxidation in mitochondria21, 23 and by lysosomal lipases upon sequestration of complete or fractions of LD by macrolipophagy18, 19. ATGs and ATG-positive membranes are detected in LD during lipolysis, suggesting that the autophagosome limiting membrane forms in situ on LD surface. In fact, recent studies have shown that LD are a site for autophagosome biogenesis38. We have uncovered that CMA is required prior to the selective sequestration of LD by the autophagosome, suggesting that for ATGs to associate and initiate autophagosome biogenesis, LD peripheral proteins would have to be removed from the LD surface. Consequently, the only residual lipolysis observed in CMA-deficient cells could be attributable to in-bulk macroautophagy, shown upregulated in these cells25, and where cargo recognition is not required.

Cytosolic lipolysis is regulated by complex interactions between PLINs and lipases39. ATGL associates with LD under maximal lipolytic conditions39 and PLINs at the LD surface exert a ‘shielding effect’ that modulates the lipase’s accessibility to TG. In this regard, overexpression of PLIN2 reduces LD association of ATGL and hence, lipolysis39, 40. We propose that CMA facilitates lipolysis by removing PLINs from LD since we found decreased association of ATGL on LD in cells with elevated PLIN2 levels, either due to impaired CMA or due to expression of the CMA mutant. In addition, PLIN2 removal could also facilitate binding of the coactivator to ATGL already on LD, since CMA blockage reduced levels of CGI-58 in LD.

Although PLIN2 has been reported to be a proteasome substrate28, 29, both lysosomes and proteasomes can degrade the same protein depending on the cellular requirement or the cell-type, as reported, for example, for PLIN141–43. In our study, we observed both lysosomal and proteasomal degradation of PLIN2, but interestingly, CMA blockage also prevented PLIN2 proteasomal degradation. Although it is plausible that CMA might indirectly facilitate proteasomal degradation of PLIN2, our data disfavors this option since (i) PLIN2 directly interacts with hsc70; (ii) we detect PLIN2 in CMA-active lysosomes and its degradation in this compartment; and (iii) eliminating the CMA-targeting motif retains PLIN2 in LD. We propose thus that PLIN2 is directly identified by hsc70 in LD and targeted for CMA degradation. Proteasome may only be able to access LD PLIN2 once its removal has been initiated by CMA or alternatively, the PLIN2 amenable to proteasomal degradation could be located in a different compartment, as described before29.

We describe PLIN2 and PLIN3 as the first lipid-anchored proteins to be CMA substrates. Presence of hsc70 on LD, also described before44–47 and its persistence at the LD surface when CMA is blocked, together with the altered LD and lysosomal dynamics in these conditions, suggests the presence of a microenvironment at the LD surface, wherein lysosomes and LD are in close proximity. Dynamic and transient LD interactions, similar to the ones we propose here with lysosomes, have been described between LD and other organelles, such as mitochondria, and peroxisomes48–50, to channel FFA from lipolysis to sites of oxidation20. Our studies with live-cell microscopy support marked reduction in kiss-and-run events between LD and lysosomes in conditions that compromise PLIN degradation by CMA such as mutations in the CMA-targeting motif of PLIN2 that prevent hsc70 interaction with this protein.

We thus, propose that degradation of PLINs via CMA occurs in discrete areas on LD to facilitate punctual removal of PLINs from the LD surface for ATGL and ATGs to access the stored TG. ATGL rapidly cycles on and off of LD to catalyze lipolysis39 and similarly, association of LC3 with LD is polarized and occurs in discrete areas on the LD surface from where the limiting membrane starts forming18. Although the mechanisms that determine removal of only a fraction of PLINs in a localized region of LD require future investigation, we speculate that changes in PLINs phosphorylation may facilitate the release of a specific fraction of PLINs from LD. In fact, PLIN2 has been already identified as part of the phosphoproteome of LD51.

We propose that the pronounced hepatic lipid accumulation observed in L2AKO mice results from a combination of: increased levels of lipogenic enzymes degraded through CMA22, reduced VLDL secretion22, and reduced lipolysis due to the inability of cytosolic lipases and ATGs to bind to LD when PLINs are not degraded by CMA. We have previously shown that chronic lipid loading inhibits CMA activity15. In light of our current findings of regulation of lipid stores by CMA, it is conceivable that a compromise in CMA would set up a viscous cycle of lipid overloading in cells. CMA activity decreases with age16, and it is plausible that changes in CMA would impact cellular lipohomeostasis and the steatosis observed during aging, which in turn would further suppress CMA activity. Manipulating CMA could thus be a viable therapeutic option against excessive lipid accumulation.

Online MethodsCell Culture and TreatmentsNIH3T3 mouse fibroblasts (American Type Culture Collection) were cultured in complete Dulbecco's modified Eagle's medium (DMEM, Sigma-Aldrich), 10% heat-inactivated newborn calf serum (HyClone), 1% penicillin/streptomycin/fungizone (Invitrogen) and maintained at 37°C/5% CO2. NIH3T3 clones stably RNA-interfered for LAMP-2A (L2A(−)) were generated as described previously25. All the cell lines were tested for mycoplasma contamination using DNA staining protocol with Hoechst 33258 dye. Oleic acid (Sigma-Aldrich) was conjugated to albumin as described52, and cells were treated with 0.06 mM oleate (OL) for 24 h unless otherwise stated. Where indicated, OL was washed off with Hanks’ Balanced Salt Solution (Invitrogen) and replaced by DMEM or low-glucose DMEM with or without serum (OL>S+, OL>S+ Low Glc, OL>S-, respectively) for 16 h. Cells were treated with lysosomal inhibitors ammonium chloride (20 mM; American Bioanalyticals) and leupeptin (100 μM; Fisher Scientific), macroautophagy activator rapamycin (100 nM; Sigma-Aldrich), proteasomal inhibitors lactacystin (5 μM; Enzo Life Sciences) and MG132 (5 μM; Sigma-Aldrich), mitochondrial depolarizer FCCP (2 μM; Sigma-Aldrich) for 16 h unless indicated otherwise. For imaging experiments, cells were treated with etomoxir (10 μM; Sigma-Aldrich) for 6 h.Plasmids and ReagentsMouse PLIN2-tGFP and G0S2-tGFP plasmids were from Origene, rat LAMP1-RFP plasmid from Addgene plasmid repository, human dsRed-LC3 plasmid53, mtKeima plasmid26, GFP-dgn/GFP-degFS30 plasmids as described before and human L2A (PGK-hL2A, plasmid includes the GFP transgene, to facilitate identification of transfected cells) was generated in our laboratory54. Mutagenesis was performed via the QuikChange site-directed mutagenesis kit (Agilent Biotechnologies). Cells were transfected with 1 μg DNA plasmid (or 0.5 μg of each plasmid in case of cotransfection) using Lipofectamine 2000 transfection reagent (Invitrogen) according to manufacturer's instructions. Sources of chemicals were as previously described18, 25, 55.AntibodiesThe dilutions and sources of antibodies used for immunoblot (IB) and immunofluorescence (IF) in this study are: PLIN2 (1:2000 (IB); 1:100 (IF); Progen Biotechnik; GP40), PLIN3 (1:2000 (IB); 1:100 (IF); ProSci Inc; 3881), L2A (1:5000 (1B); Life Technologies; 512200), mouse LAMP1 (1:3000 (IB); 1:100 (IF); Developmental Studies Hybridoma Bank; 1D4B), rat LAMP1 (1:3000 (IB); Enzo Life Sciences; Ly1C6), human LAMP2 (1:3000 (IB); 1:100 (IF); Developmental Studies Hybridoma Bank; H4B4), LAMP-2B (1:5000 (IB); generated in our laboratory), Hsc70 (1:5000 (IB); 1:100 (IF); Novus Biologicals; 13D3), GAPDH (1:3000 (IB); Abcam; 6C5), hexokinase (1:1000 (IB); Cell Signaling; C35C4), tGFP (1:2000 (IB); Origene; 2H8), ATGL (1:1000 (IB); 1:100 (IF); Cell Signaling; 30A4), CGI58 (1:3000 (IB); ProSci Inc; 45186), ATG5 (1:2000 (IB); 1:100 (IF); Novus Biologicals; 53818), Beclin1 (1:2000 (IB); 1:100 (IF); Novus Biologicals; 500249), LC3 (1:1000 (IB); 1:100 (IF); Cell Signaling; 2775), Atg14 (1:100 (IF); MBL; PD026), Atg7 (1:100 (IF); Cell Signaling; 2631), vps15 (1:100 (IF); Novus Biologicals; NBP130463), vps34 (1:2000 (IB); 1:100 (IF); Life Technologies; 382100), p62 (1:2000 (IB); Enzo Life Sciences; 9860), NBR1 (1:3000 (IB); 1:100 (IF); Abnova; A01), Rab7 (1:100 (IF); Abcam; EPR7589), ULK1-P (1:100 (IF); Cell Signaling; 5869), Atg9 (1:100 (IF); Novus Biologicals; 56893), UVRAG (1:100 (IF); Abcam; 174550), K48-specific ubiquitin (1:2000 (IB); Millipore; Apu2), IkB (1:5000 (IB); Santa Cruz Biotechnology; C21), and Actin (1:10,000 (IB); Abcam; AC15).Animals and TreatmentsAdult Wistar male rats (25) and C57BL/6 male mice (12) (Charles River Laboratories) were used. L2A knockout mice were generated as described22. Where indicated, rats or mice were starved for 48 h or 24 h before organelle isolation by removing food but maintaining water ad libitum. Animals were injected intraperitoneally with leupeptin (2 mg/100 g body weight) 2 h prior to organelle isolation. All animal studies were approved by the Institutional Animal Care and Use Committee. Randomization between control and L2AKO mice was not possible as animal group is based on genotype. In the case of starvation and treatment with vehicle or leupeptin, animals were randomly attributed to each group. The investigators were not blinded to allocation during experiments and outcome assessment. No statistical method was used to predetermine sample size.TG MeasurementTriglycerides (TG) were measured by a commercial kit (Sigma-Aldrich) according to the manufacturer’s instructions.TG SynthesisTG synthesis was measured as described before18. Briefly, synthesis of triglycerides derived from [14C]-oleate (PerkinElmer) complexed with BSA was measured by incubation of untreated or OL-treated cells with [14C]-oleate-BSA (0.2 μCi, 4 h), followed by extraction of cellular lipids and analysis by thin-layer chromatography. Incorporation of radioactivity into TG was determined using a Storm Gel and Blot Imaging System (GE Healthcare) followed by densitometry using ImageJ. The rate of TG synthesis was calculated as the amount of incorporated [14C]-oleate in relative units per mg protein.Beta-oxidationRates of fatty acid beta-oxidation were determined by metabolic radioactive labeling as described earlier18 and by respiratory measurements. Briefly, the rate of carbon dioxide production resulting from the oxidation of [14C]-oleate was measured. Cells pretreated with or without OL (0.06 mM) were treated with [14C]-oleate-BSA (0.8 μCi, 4 h). Where indicated inhibitors, ammonium chloride (20 mM) and leupeptin (100 μM), and diethylumbelliferyl phosphate (Deup) (100 μM) were added with the radiolabeled oleate for 4 h. The released [14C]-carbon dioxide was trapped at 37°C for 1 h onto filter paper pre-soaked in 100 mM sodium hydroxide. The rate of beta-oxidation was calculated as the amount of trapped [14C]-carbon dioxide in relative units produced per mg protein per hour. Equal numbers of cells were plated in XF96 plates (Seahorse Bioscience) in low-glucose DMEM (GIBCO) supplemented with serum. Cells were treated with OL (0.12 mM) for 16 h following which cell respiration was assayed as time-resolved measurements of oxygen consumption rate (OCR) in a respirometer (Seahorse Bioscience). Fatty acid beta-oxidation was calculated as specified by the manufacturer by injecting etomoxir (50 μM) in the buffered assay medium.Cell ImagingImmunofluorescence microscopyCells grown on coverslips were fixed with 4% paraformaldehyde (or pre-chilled methanol in case of staining for Hsc70, in order to highlight the membrane-associated protein) for 30 min, permeabilized and blocked with 1% BSA, 0.01% Triton X-100 in phosphate buffer saline (PBS). Incubation with primary and secondary antibody conjugated to Alexa Fluor 488 or Cy5 (Invitrogen) in 0.1% BSA in PBS was performed at room temperature for 1 h each. For LD staining, cells were incubated with BODIPY 493/503 (Invitrogen, 20 μg/ml) or 1,6-diphenyl-1,3,5-hexatriene (DPH, Sigma-Aldrich, 5 μM; pseudo-colored to red or green) for 20 min at 37°C prior to fixation. For mitochondria staining, cells were incubated with MitoTracker (50 mM) and MitoTracker CMXRos (50 nM) (Invitrogen) for 20 min 37°C prior to fixation. Coverslips were mounted in DAPI-Fluoromount-G (Southern Biotechnology) or Draq5 (Invitrogen; pseudo-colored to blue) to stain the nucleus. All images were acquired with Axiovert 200 fluorescence microscope (Carl Zeiss Microscopy) with ×63 objective and 1.4 numerical aperture, mounted with an ApoTome.2 slider, and prepared using Adobe Photoshop CS3 (Adobe Systems) and ImageJ (NIH). Where indicated, cells plated in glass-bottom 96-well plates were treated, stained with BODIPY493/503, fixed and images were acquired using a high-content microscope (Operetta, Perkin Elmer). Images of 9 different fields per well were captured, resulting in an average of 1500–2000 cells. Nuclei and puncta were identified using the manufacturer’s software. The number of particles/puncta per cell was quantified using ‘analyze particles’ function of ImageJ after thresholding in non-saturated images. The percentage of colocalization was determined by ‘JACoP’ plugin in ImageJ after thresholding of individual frames56. Where indicated, the colocalized pixels were highlighted using ‘colocalization’ plugin of ImageJ. 3D reconstruction images were modeled from Z-sections as mixed renderings using the Inside4D module for AxioVision Rel. 4.8. after applying the Nyquist-Shannon sampling theorem. Immunofluorescence and direct fluorescence quantifications of number of lipid droplets, number of puncta and percentage colocalization, were performed blinded.Live-cell imagingLive-cell imaging was performed in cells grown in glass-bottom dishes (MatTek) 24 h post-transfection (with 8 h OL treatment) at 37°C in a CO2-enriched atmosphere in recording medium (phenol red-free DMEM (GIBCO), 3.7 mg/ml NaHCO3, and 25 mM HEPES, pH 7.4) using a 4D spinning-disk confocal microscope (PerkinElmer, TE2000S) with 60x NA 1.4 oil objective and ORCA-ERA camera (Hamamatsu). Images were acquired as a stack of 50–50 Z optical planes (0.4 μm) encompassing the entire cell width at a fixed rate of maximum speed for ~20 min. Image sequences were processed with Volocity and ImageJ. Using Volocity, small and large LD clusters were identified as <1 μm3 and >1 μm3, respectively and were tracked using the software to calculate the displacement (straight line distance between the start and the end point of the track), displacement rate (displacement/(time of last time point in the track – time of the first time point of the track)) and tracking velocity (average speed over the whole track).Electron microscopyCells were fixed with 2.5% glutaraldehyde and 2% paraformaldehyde in 0.1 M sodium cacodylate buffer pH 7.4 at 4°C for 3 h. Post-fixation with 1% osmium tetroxide was followed by dehydration with graded series of ethanol, infiltration and embedding in EMbed 812 (Electron Microscopy Sciences). 80 nm sections were cut with ultramicrotome (Leica), stained with uranyl acetate followed by lead citrate and viewed in JEM-1200EX transmission electron microscope (JEOL) at 80 kV. Morphometric analysis was done using ImageJ with LD identified as electron translucent structures lacking surrounding membrane. Micrographs were independently annotated by two people and the average of their scoring was used for each micrograph.Cellular subfractionationLysosome isolationRat liver lysosomes were isolated from a light mitochondrial-lysosomal fraction by centrifugation in a discontinuous metrizamide density gradient57.LD isolationRat and mouse liver LD were isolated as described previously18. Livers homogenized in 0.25 M sucrose were centrifuged at 6,800 g for 5 min at 4°C. The supernatant and the fatty layer were centrifuged at 17,000 g for 10 min at 4°C to pellet lysosomes. The supernatant was adjusted to 20% sucrose (LD-enriched fraction) and centrifuged in a discontinuous sucrose density gradient at 28,000 g for 60 min at 4°C. The LD fraction collected was delipidated, solubilized in SDS, and equal protein amount was analyzed by immunoblot.CoimmunoprecipitationCoimmunoprecipitation was performed in total cell lysates or LD-enriched fractions prepared in 25 mM Tris (pH 7.2), 150 mM NaCl, 5 mM MgCl2, 0.5% NP-40, 1 mM DTT, 5% glycerol and protease inhibitors using standard procedures. Equal protein amount of each fraction was incubated with primary antibody (1 μg) overnight followed by incubation with protein A-conjugated sepharose for 1 h, centrifugation and washes.General MethodsTotal cell lysates were prepared in RIPA buffer (150 mM NaCl, 1% NP-40, 0.5% NaDoc, 0.1% SDS, 50 mM Tris pH 8) containing protease and phosphatase inhibitors. Protein concentration was measured using bovine serum albumin as a standard58. Isoelectric focusing (IEF) was done using the Protean IEF Cell (Bio-Rad) at 20°C with rapid ramping to voltage 10,000 V at a current limit of 50 μA using IPG Strips with 4–7pH range (Bio-Rad). Where indicated total cellular fractions were treated with λ-phosphatase according to manufacturer’s instructions (New England BioLabs) before isoelectric focusing. Phos-tag PAGE was performed according to manufacturer’s instructions (Wako) using 100 μM Phos-tag. Samples were subjected to SDS-PAGE, transferred to nitrocellulose membrane, blocked with low-fat milk and incubated with primary antibody overnight. The proteins were visualized by using peroxidase-conjugated secondary antibodies and chemiluminescent reagent (PerkinElmer) in LAS-3000 Imaging System (Fujifilm). Densitometric quantification was performed on unsaturated images using ImageJ.Sample size, Randomization and Statistical AnalysesIn the studies performed in cell lines in culture, all experiments were repeated at least 3 times and often with duplicate blots in the same experiment. All numerical results are reported as mean ± standard error of the mean (SEM). Statistical significance of the difference between experimental groups was analyzed by two-tailed unpaired Student’s t-test using MS Excel. Differences were considered statistically significant for P<0.05. For those experiments in which an estimate of variation was possible based on previous studies or published work by others, we have confirmed that the variance is similar between the control and experimental groups compared. For those instances in which previous information was not available, we have compared with other conditions in which changes in cellular components resulted in changes in CMA.

Online Methods

Cell Culture and TreatmentsNIH3T3 mouse fibroblasts (American Type Culture Collection) were cultured in complete Dulbecco's modified Eagle's medium (DMEM, Sigma-Aldrich), 10% heat-inactivated newborn calf serum (HyClone), 1% penicillin/streptomycin/fungizone (Invitrogen) and maintained at 37°C/5% CO2. NIH3T3 clones stably RNA-interfered for LAMP-2A (L2A(−)) were generated as described previously25. All the cell lines were tested for mycoplasma contamination using DNA staining protocol with Hoechst 33258 dye. Oleic acid (Sigma-Aldrich) was conjugated to albumin as described52, and cells were treated with 0.06 mM oleate (OL) for 24 h unless otherwise stated. Where indicated, OL was washed off with Hanks’ Balanced Salt Solution (Invitrogen) and replaced by DMEM or low-glucose DMEM with or without serum (OL>S+, OL>S+ Low Glc, OL>S-, respectively) for 16 h. Cells were treated with lysosomal inhibitors ammonium chloride (20 mM; American Bioanalyticals) and leupeptin (100 μM; Fisher Scientific), macroautophagy activator rapamycin (100 nM; Sigma-Aldrich), proteasomal inhibitors lactacystin (5 μM; Enzo Life Sciences) and MG132 (5 μM; Sigma-Aldrich), mitochondrial depolarizer FCCP (2 μM; Sigma-Aldrich) for 16 h unless indicated otherwise. For imaging experiments, cells were treated with etomoxir (10 μM; Sigma-Aldrich) for 6 h.

Cell Culture and Treatments

NIH3T3 mouse fibroblasts (American Type Culture Collection) were cultured in complete Dulbecco's modified Eagle's medium (DMEM, Sigma-Aldrich), 10% heat-inactivated newborn calf serum (HyClone), 1% penicillin/streptomycin/fungizone (Invitrogen) and maintained at 37°C/5% CO2. NIH3T3 clones stably RNA-interfered for LAMP-2A (L2A(−)) were generated as described previously25. All the cell lines were tested for mycoplasma contamination using DNA staining protocol with Hoechst 33258 dye. Oleic acid (Sigma-Aldrich) was conjugated to albumin as described52, and cells were treated with 0.06 mM oleate (OL) for 24 h unless otherwise stated. Where indicated, OL was washed off with Hanks’ Balanced Salt Solution (Invitrogen) and replaced by DMEM or low-glucose DMEM with or without serum (OL>S+, OL>S+ Low Glc, OL>S-, respectively) for 16 h. Cells were treated with lysosomal inhibitors ammonium chloride (20 mM; American Bioanalyticals) and leupeptin (100 μM; Fisher Scientific), macroautophagy activator rapamycin (100 nM; Sigma-Aldrich), proteasomal inhibitors lactacystin (5 μM; Enzo Life Sciences) and MG132 (5 μM; Sigma-Aldrich), mitochondrial depolarizer FCCP (2 μM; Sigma-Aldrich) for 16 h unless indicated otherwise. For imaging experiments, cells were treated with etomoxir (10 μM; Sigma-Aldrich) for 6 h.

Plasmids and ReagentsMouse PLIN2-tGFP and G0S2-tGFP plasmids were from Origene, rat LAMP1-RFP plasmid from Addgene plasmid repository, human dsRed-LC3 plasmid53, mtKeima plasmid26, GFP-dgn/GFP-degFS30 plasmids as described before and human L2A (PGK-hL2A, plasmid includes the GFP transgene, to facilitate identification of transfected cells) was generated in our laboratory54. Mutagenesis was performed via the QuikChange site-directed mutagenesis kit (Agilent Biotechnologies). Cells were transfected with 1 μg DNA plasmid (or 0.5 μg of each plasmid in case of cotransfection) using Lipofectamine 2000 transfection reagent (Invitrogen) according to manufacturer's instructions. Sources of chemicals were as previously described18, 25, 55.

Plasmids and Reagents

Mouse PLIN2-tGFP and G0S2-tGFP plasmids were from Origene, rat LAMP1-RFP plasmid from Addgene plasmid repository, human dsRed-LC3 plasmid53, mtKeima plasmid26, GFP-dgn/GFP-degFS30 plasmids as described before and human L2A (PGK-hL2A, plasmid includes the GFP transgene, to facilitate identification of transfected cells) was generated in our laboratory54. Mutagenesis was performed via the QuikChange site-directed mutagenesis kit (Agilent Biotechnologies). Cells were transfected with 1 μg DNA plasmid (or 0.5 μg of each plasmid in case of cotransfection) using Lipofectamine 2000 transfection reagent (Invitrogen) according to manufacturer's instructions. Sources of chemicals were as previously described18, 25, 55.

AntibodiesThe dilutions and sources of antibodies used for immunoblot (IB) and immunofluorescence (IF) in this study are: PLIN2 (1:2000 (IB); 1:100 (IF); Progen Biotechnik; GP40), PLIN3 (1:2000 (IB); 1:100 (IF); ProSci Inc; 3881), L2A (1:5000 (1B); Life Technologies; 512200), mouse LAMP1 (1:3000 (IB); 1:100 (IF); Developmental Studies Hybridoma Bank; 1D4B), rat LAMP1 (1:3000 (IB); Enzo Life Sciences; Ly1C6), human LAMP2 (1:3000 (IB); 1:100 (IF); Developmental Studies Hybridoma Bank; H4B4), LAMP-2B (1:5000 (IB); generated in our laboratory), Hsc70 (1:5000 (IB); 1:100 (IF); Novus Biologicals; 13D3), GAPDH (1:3000 (IB); Abcam; 6C5), hexokinase (1:1000 (IB); Cell Signaling; C35C4), tGFP (1:2000 (IB); Origene; 2H8), ATGL (1:1000 (IB); 1:100 (IF); Cell Signaling; 30A4), CGI58 (1:3000 (IB); ProSci Inc; 45186), ATG5 (1:2000 (IB); 1:100 (IF); Novus Biologicals; 53818), Beclin1 (1:2000 (IB); 1:100 (IF); Novus Biologicals; 500249), LC3 (1:1000 (IB); 1:100 (IF); Cell Signaling; 2775), Atg14 (1:100 (IF); MBL; PD026), Atg7 (1:100 (IF); Cell Signaling; 2631), vps15 (1:100 (IF); Novus Biologicals; NBP130463), vps34 (1:2000 (IB); 1:100 (IF); Life Technologies; 382100), p62 (1:2000 (IB); Enzo Life Sciences; 9860), NBR1 (1:3000 (IB); 1:100 (IF); Abnova; A01), Rab7 (1:100 (IF); Abcam; EPR7589), ULK1-P (1:100 (IF); Cell Signaling; 5869), Atg9 (1:100 (IF); Novus Biologicals; 56893), UVRAG (1:100 (IF); Abcam; 174550), K48-specific ubiquitin (1:2000 (IB); Millipore; Apu2), IkB (1:5000 (IB); Santa Cruz Biotechnology; C21), and Actin (1:10,000 (IB); Abcam; AC15).

The dilutions and sources of antibodies used for immunoblot (IB) and immunofluorescence (IF) in this study are: PLIN2 (1:2000 (IB); 1:100 (IF); Progen Biotechnik; GP40), PLIN3 (1:2000 (IB); 1:100 (IF); ProSci Inc; 3881), L2A (1:5000 (1B); Life Technologies; 512200), mouse LAMP1 (1:3000 (IB); 1:100 (IF); Developmental Studies Hybridoma Bank; 1D4B), rat LAMP1 (1:3000 (IB); Enzo Life Sciences; Ly1C6), human LAMP2 (1:3000 (IB); 1:100 (IF); Developmental Studies Hybridoma Bank; H4B4), LAMP-2B (1:5000 (IB); generated in our laboratory), Hsc70 (1:5000 (IB); 1:100 (IF); Novus Biologicals; 13D3), GAPDH (1:3000 (IB); Abcam; 6C5), hexokinase (1:1000 (IB); Cell Signaling; C35C4), tGFP (1:2000 (IB); Origene; 2H8), ATGL (1:1000 (IB); 1:100 (IF); Cell Signaling; 30A4), CGI58 (1:3000 (IB); ProSci Inc; 45186), ATG5 (1:2000 (IB); 1:100 (IF); Novus Biologicals; 53818), Beclin1 (1:2000 (IB); 1:100 (IF); Novus Biologicals; 500249), LC3 (1:1000 (IB); 1:100 (IF); Cell Signaling; 2775), Atg14 (1:100 (IF); MBL; PD026), Atg7 (1:100 (IF); Cell Signaling; 2631), vps15 (1:100 (IF); Novus Biologicals; NBP130463), vps34 (1:2000 (IB); 1:100 (IF); Life Technologies; 382100), p62 (1:2000 (IB); Enzo Life Sciences; 9860), NBR1 (1:3000 (IB); 1:100 (IF); Abnova; A01), Rab7 (1:100 (IF); Abcam; EPR7589), ULK1-P (1:100 (IF); Cell Signaling; 5869), Atg9 (1:100 (IF); Novus Biologicals; 56893), UVRAG (1:100 (IF); Abcam; 174550), K48-specific ubiquitin (1:2000 (IB); Millipore; Apu2), IkB (1:5000 (IB); Santa Cruz Biotechnology; C21), and Actin (1:10,000 (IB); Abcam; AC15).

Animals and TreatmentsAdult Wistar male rats (25) and C57BL/6 male mice (12) (Charles River Laboratories) were used. L2A knockout mice were generated as described22. Where indicated, rats or mice were starved for 48 h or 24 h before organelle isolation by removing food but maintaining water ad libitum. Animals were injected intraperitoneally with leupeptin (2 mg/100 g body weight) 2 h prior to organelle isolation. All animal studies were approved by the Institutional Animal Care and Use Committee. Randomization between control and L2AKO mice was not possible as animal group is based on genotype. In the case of starvation and treatment with vehicle or leupeptin, animals were randomly attributed to each group. The investigators were not blinded to allocation during experiments and outcome assessment. No statistical method was used to predetermine sample size.

Animals and Treatments

Adult Wistar male rats (25) and C57BL/6 male mice (12) (Charles River Laboratories) were used. L2A knockout mice were generated as described22. Where indicated, rats or mice were starved for 48 h or 24 h before organelle isolation by removing food but maintaining water ad libitum. Animals were injected intraperitoneally with leupeptin (2 mg/100 g body weight) 2 h prior to organelle isolation. All animal studies were approved by the Institutional Animal Care and Use Committee. Randomization between control and L2AKO mice was not possible as animal group is based on genotype. In the case of starvation and treatment with vehicle or leupeptin, animals were randomly attributed to each group. The investigators were not blinded to allocation during experiments and outcome assessment. No statistical method was used to predetermine sample size.

TG MeasurementTriglycerides (TG) were measured by a commercial kit (Sigma-Aldrich) according to the manufacturer’s instructions.

TG Measurement

Triglycerides (TG) were measured by a commercial kit (Sigma-Aldrich) according to the manufacturer’s instructions.

TG SynthesisTG synthesis was measured as described before18. Briefly, synthesis of triglycerides derived from [14C]-oleate (PerkinElmer) complexed with BSA was measured by incubation of untreated or OL-treated cells with [14C]-oleate-BSA (0.2 μCi, 4 h), followed by extraction of cellular lipids and analysis by thin-layer chromatography. Incorporation of radioactivity into TG was determined using a Storm Gel and Blot Imaging System (GE Healthcare) followed by densitometry using ImageJ. The rate of TG synthesis was calculated as the amount of incorporated [14C]-oleate in relative units per mg protein.

TG Synthesis

TG synthesis was measured as described before18. Briefly, synthesis of triglycerides derived from [14C]-oleate (PerkinElmer) complexed with BSA was measured by incubation of untreated or OL-treated cells with [14C]-oleate-BSA (0.2 μCi, 4 h), followed by extraction of cellular lipids and analysis by thin-layer chromatography. Incorporation of radioactivity into TG was determined using a Storm Gel and Blot Imaging System (GE Healthcare) followed by densitometry using ImageJ. The rate of TG synthesis was calculated as the amount of incorporated [14C]-oleate in relative units per mg protein.

Beta-oxidationRates of fatty acid beta-oxidation were determined by metabolic radioactive labeling as described earlier18 and by respiratory measurements. Briefly, the rate of carbon dioxide production resulting from the oxidation of [14C]-oleate was measured. Cells pretreated with or without OL (0.06 mM) were treated with [14C]-oleate-BSA (0.8 μCi, 4 h). Where indicated inhibitors, ammonium chloride (20 mM) and leupeptin (100 μM), and diethylumbelliferyl phosphate (Deup) (100 μM) were added with the radiolabeled oleate for 4 h. The released [14C]-carbon dioxide was trapped at 37°C for 1 h onto filter paper pre-soaked in 100 mM sodium hydroxide. The rate of beta-oxidation was calculated as the amount of trapped [14C]-carbon dioxide in relative units produced per mg protein per hour. Equal numbers of cells were plated in XF96 plates (Seahorse Bioscience) in low-glucose DMEM (GIBCO) supplemented with serum. Cells were treated with OL (0.12 mM) for 16 h following which cell respiration was assayed as time-resolved measurements of oxygen consumption rate (OCR) in a respirometer (Seahorse Bioscience). Fatty acid beta-oxidation was calculated as specified by the manufacturer by injecting etomoxir (50 μM) in the buffered assay medium.

Beta-oxidation

Rates of fatty acid beta-oxidation were determined by metabolic radioactive labeling as described earlier18 and by respiratory measurements. Briefly, the rate of carbon dioxide production resulting from the oxidation of [14C]-oleate was measured. Cells pretreated with or without OL (0.06 mM) were treated with [14C]-oleate-BSA (0.8 μCi, 4 h). Where indicated inhibitors, ammonium chloride (20 mM) and leupeptin (100 μM), and diethylumbelliferyl phosphate (Deup) (100 μM) were added with the radiolabeled oleate for 4 h. The released [14C]-carbon dioxide was trapped at 37°C for 1 h onto filter paper pre-soaked in 100 mM sodium hydroxide. The rate of beta-oxidation was calculated as the amount of trapped [14C]-carbon dioxide in relative units produced per mg protein per hour. Equal numbers of cells were plated in XF96 plates (Seahorse Bioscience) in low-glucose DMEM (GIBCO) supplemented with serum. Cells were treated with OL (0.12 mM) for 16 h following which cell respiration was assayed as time-resolved measurements of oxygen consumption rate (OCR) in a respirometer (Seahorse Bioscience). Fatty acid beta-oxidation was calculated as specified by the manufacturer by injecting etomoxir (50 μM) in the buffered assay medium.

Cell ImagingImmunofluorescence microscopyCells grown on coverslips were fixed with 4% paraformaldehyde (or pre-chilled methanol in case of staining for Hsc70, in order to highlight the membrane-associated protein) for 30 min, permeabilized and blocked with 1% BSA, 0.01% Triton X-100 in phosphate buffer saline (PBS). Incubation with primary and secondary antibody conjugated to Alexa Fluor 488 or Cy5 (Invitrogen) in 0.1% BSA in PBS was performed at room temperature for 1 h each. For LD staining, cells were incubated with BODIPY 493/503 (Invitrogen, 20 μg/ml) or 1,6-diphenyl-1,3,5-hexatriene (DPH, Sigma-Aldrich, 5 μM; pseudo-colored to red or green) for 20 min at 37°C prior to fixation. For mitochondria staining, cells were incubated with MitoTracker (50 mM) and MitoTracker CMXRos (50 nM) (Invitrogen) for 20 min 37°C prior to fixation. Coverslips were mounted in DAPI-Fluoromount-G (Southern Biotechnology) or Draq5 (Invitrogen; pseudo-colored to blue) to stain the nucleus. All images were acquired with Axiovert 200 fluorescence microscope (Carl Zeiss Microscopy) with ×63 objective and 1.4 numerical aperture, mounted with an ApoTome.2 slider, and prepared using Adobe Photoshop CS3 (Adobe Systems) and ImageJ (NIH). Where indicated, cells plated in glass-bottom 96-well plates were treated, stained with BODIPY493/503, fixed and images were acquired using a high-content microscope (Operetta, Perkin Elmer). Images of 9 different fields per well were captured, resulting in an average of 1500–2000 cells. Nuclei and puncta were identified using the manufacturer’s software. The number of particles/puncta per cell was quantified using ‘analyze particles’ function of ImageJ after thresholding in non-saturated images. The percentage of colocalization was determined by ‘JACoP’ plugin in ImageJ after thresholding of individual frames56. Where indicated, the colocalized pixels were highlighted using ‘colocalization’ plugin of ImageJ. 3D reconstruction images were modeled from Z-sections as mixed renderings using the Inside4D module for AxioVision Rel. 4.8. after applying the Nyquist-Shannon sampling theorem. Immunofluorescence and direct fluorescence quantifications of number of lipid droplets, number of puncta and percentage colocalization, were performed blinded.Live-cell imagingLive-cell imaging was performed in cells grown in glass-bottom dishes (MatTek) 24 h post-transfection (with 8 h OL treatment) at 37°C in a CO2-enriched atmosphere in recording medium (phenol red-free DMEM (GIBCO), 3.7 mg/ml NaHCO3, and 25 mM HEPES, pH 7.4) using a 4D spinning-disk confocal microscope (PerkinElmer, TE2000S) with 60x NA 1.4 oil objective and ORCA-ERA camera (Hamamatsu). Images were acquired as a stack of 50–50 Z optical planes (0.4 μm) encompassing the entire cell width at a fixed rate of maximum speed for ~20 min. Image sequences were processed with Volocity and ImageJ. Using Volocity, small and large LD clusters were identified as <1 μm3 and >1 μm3, respectively and were tracked using the software to calculate the displacement (straight line distance between the start and the end point of the track), displacement rate (displacement/(time of last time point in the track – time of the first time point of the track)) and tracking velocity (average speed over the whole track).Electron microscopyCells were fixed with 2.5% glutaraldehyde and 2% paraformaldehyde in 0.1 M sodium cacodylate buffer pH 7.4 at 4°C for 3 h. Post-fixation with 1% osmium tetroxide was followed by dehydration with graded series of ethanol, infiltration and embedding in EMbed 812 (Electron Microscopy Sciences). 80 nm sections were cut with ultramicrotome (Leica), stained with uranyl acetate followed by lead citrate and viewed in JEM-1200EX transmission electron microscope (JEOL) at 80 kV. Morphometric analysis was done using ImageJ with LD identified as electron translucent structures lacking surrounding membrane. Micrographs were independently annotated by two people and the average of their scoring was used for each micrograph.

Cell Imaging

Immunofluorescence microscopyCells grown on coverslips were fixed with 4% paraformaldehyde (or pre-chilled methanol in case of staining for Hsc70, in order to highlight the membrane-associated protein) for 30 min, permeabilized and blocked with 1% BSA, 0.01% Triton X-100 in phosphate buffer saline (PBS). Incubation with primary and secondary antibody conjugated to Alexa Fluor 488 or Cy5 (Invitrogen) in 0.1% BSA in PBS was performed at room temperature for 1 h each. For LD staining, cells were incubated with BODIPY 493/503 (Invitrogen, 20 μg/ml) or 1,6-diphenyl-1,3,5-hexatriene (DPH, Sigma-Aldrich, 5 μM; pseudo-colored to red or green) for 20 min at 37°C prior to fixation. For mitochondria staining, cells were incubated with MitoTracker (50 mM) and MitoTracker CMXRos (50 nM) (Invitrogen) for 20 min 37°C prior to fixation. Coverslips were mounted in DAPI-Fluoromount-G (Southern Biotechnology) or Draq5 (Invitrogen; pseudo-colored to blue) to stain the nucleus. All images were acquired with Axiovert 200 fluorescence microscope (Carl Zeiss Microscopy) with ×63 objective and 1.4 numerical aperture, mounted with an ApoTome.2 slider, and prepared using Adobe Photoshop CS3 (Adobe Systems) and ImageJ (NIH). Where indicated, cells plated in glass-bottom 96-well plates were treated, stained with BODIPY493/503, fixed and images were acquired using a high-content microscope (Operetta, Perkin Elmer). Images of 9 different fields per well were captured, resulting in an average of 1500–2000 cells. Nuclei and puncta were identified using the manufacturer’s software. The number of particles/puncta per cell was quantified using ‘analyze particles’ function of ImageJ after thresholding in non-saturated images. The percentage of colocalization was determined by ‘JACoP’ plugin in ImageJ after thresholding of individual frames56. Where indicated, the colocalized pixels were highlighted using ‘colocalization’ plugin of ImageJ. 3D reconstruction images were modeled from Z-sections as mixed renderings using the Inside4D module for AxioVision Rel. 4.8. after applying the Nyquist-Shannon sampling theorem. Immunofluorescence and direct fluorescence quantifications of number of lipid droplets, number of puncta and percentage colocalization, were performed blinded.

Immunofluorescence microscopy

Cells grown on coverslips were fixed with 4% paraformaldehyde (or pre-chilled methanol in case of staining for Hsc70, in order to highlight the membrane-associated protein) for 30 min, permeabilized and blocked with 1% BSA, 0.01% Triton X-100 in phosphate buffer saline (PBS). Incubation with primary and secondary antibody conjugated to Alexa Fluor 488 or Cy5 (Invitrogen) in 0.1% BSA in PBS was performed at room temperature for 1 h each. For LD staining, cells were incubated with BODIPY 493/503 (Invitrogen, 20 μg/ml) or 1,6-diphenyl-1,3,5-hexatriene (DPH, Sigma-Aldrich, 5 μM; pseudo-colored to red or green) for 20 min at 37°C prior to fixation. For mitochondria staining, cells were incubated with MitoTracker (50 mM) and MitoTracker CMXRos (50 nM) (Invitrogen) for 20 min 37°C prior to fixation. Coverslips were mounted in DAPI-Fluoromount-G (Southern Biotechnology) or Draq5 (Invitrogen; pseudo-colored to blue) to stain the nucleus. All images were acquired with Axiovert 200 fluorescence microscope (Carl Zeiss Microscopy) with ×63 objective and 1.4 numerical aperture, mounted with an ApoTome.2 slider, and prepared using Adobe Photoshop CS3 (Adobe Systems) and ImageJ (NIH). Where indicated, cells plated in glass-bottom 96-well plates were treated, stained with BODIPY493/503, fixed and images were acquired using a high-content microscope (Operetta, Perkin Elmer). Images of 9 different fields per well were captured, resulting in an average of 1500–2000 cells. Nuclei and puncta were identified using the manufacturer’s software. The number of particles/puncta per cell was quantified using ‘analyze particles’ function of ImageJ after thresholding in non-saturated images. The percentage of colocalization was determined by ‘JACoP’ plugin in ImageJ after thresholding of individual frames56. Where indicated, the colocalized pixels were highlighted using ‘colocalization’ plugin of ImageJ. 3D reconstruction images were modeled from Z-sections as mixed renderings using the Inside4D module for AxioVision Rel. 4.8. after applying the Nyquist-Shannon sampling theorem. Immunofluorescence and direct fluorescence quantifications of number of lipid droplets, number of puncta and percentage colocalization, were performed blinded.

Live-cell imagingLive-cell imaging was performed in cells grown in glass-bottom dishes (MatTek) 24 h post-transfection (with 8 h OL treatment) at 37°C in a CO2-enriched atmosphere in recording medium (phenol red-free DMEM (GIBCO), 3.7 mg/ml NaHCO3, and 25 mM HEPES, pH 7.4) using a 4D spinning-disk confocal microscope (PerkinElmer, TE2000S) with 60x NA 1.4 oil objective and ORCA-ERA camera (Hamamatsu). Images were acquired as a stack of 50–50 Z optical planes (0.4 μm) encompassing the entire cell width at a fixed rate of maximum speed for ~20 min. Image sequences were processed with Volocity and ImageJ. Using Volocity, small and large LD clusters were identified as <1 μm3 and >1 μm3, respectively and were tracked using the software to calculate the displacement (straight line distance between the start and the end point of the track), displacement rate (displacement/(time of last time point in the track – time of the first time point of the track)) and tracking velocity (average speed over the whole track).

Live-cell imaging

Live-cell imaging was performed in cells grown in glass-bottom dishes (MatTek) 24 h post-transfection (with 8 h OL treatment) at 37°C in a CO2-enriched atmosphere in recording medium (phenol red-free DMEM (GIBCO), 3.7 mg/ml NaHCO3, and 25 mM HEPES, pH 7.4) using a 4D spinning-disk confocal microscope (PerkinElmer, TE2000S) with 60x NA 1.4 oil objective and ORCA-ERA camera (Hamamatsu). Images were acquired as a stack of 50–50 Z optical planes (0.4 μm) encompassing the entire cell width at a fixed rate of maximum speed for ~20 min. Image sequences were processed with Volocity and ImageJ. Using Volocity, small and large LD clusters were identified as <1 μm3 and >1 μm3, respectively and were tracked using the software to calculate the displacement (straight line distance between the start and the end point of the track), displacement rate (displacement/(time of last time point in the track – time of the first time point of the track)) and tracking velocity (average speed over the whole track).

Electron microscopyCells were fixed with 2.5% glutaraldehyde and 2% paraformaldehyde in 0.1 M sodium cacodylate buffer pH 7.4 at 4°C for 3 h. Post-fixation with 1% osmium tetroxide was followed by dehydration with graded series of ethanol, infiltration and embedding in EMbed 812 (Electron Microscopy Sciences). 80 nm sections were cut with ultramicrotome (Leica), stained with uranyl acetate followed by lead citrate and viewed in JEM-1200EX transmission electron microscope (JEOL) at 80 kV. Morphometric analysis was done using ImageJ with LD identified as electron translucent structures lacking surrounding membrane. Micrographs were independently annotated by two people and the average of their scoring was used for each micrograph.

Electron microscopy

Cells were fixed with 2.5% glutaraldehyde and 2% paraformaldehyde in 0.1 M sodium cacodylate buffer pH 7.4 at 4°C for 3 h. Post-fixation with 1% osmium tetroxide was followed by dehydration with graded series of ethanol, infiltration and embedding in EMbed 812 (Electron Microscopy Sciences). 80 nm sections were cut with ultramicrotome (Leica), stained with uranyl acetate followed by lead citrate and viewed in JEM-1200EX transmission electron microscope (JEOL) at 80 kV. Morphometric analysis was done using ImageJ with LD identified as electron translucent structures lacking surrounding membrane. Micrographs were independently annotated by two people and the average of their scoring was used for each micrograph.

Cellular subfractionationLysosome isolationRat liver lysosomes were isolated from a light mitochondrial-lysosomal fraction by centrifugation in a discontinuous metrizamide density gradient57.LD isolationRat and mouse liver LD were isolated as described previously18. Livers homogenized in 0.25 M sucrose were centrifuged at 6,800 g for 5 min at 4°C. The supernatant and the fatty layer were centrifuged at 17,000 g for 10 min at 4°C to pellet lysosomes. The supernatant was adjusted to 20% sucrose (LD-enriched fraction) and centrifuged in a discontinuous sucrose density gradient at 28,000 g for 60 min at 4°C. The LD fraction collected was delipidated, solubilized in SDS, and equal protein amount was analyzed by immunoblot.

Cellular subfractionation

Lysosome isolationRat liver lysosomes were isolated from a light mitochondrial-lysosomal fraction by centrifugation in a discontinuous metrizamide density gradient57.

Lysosome isolation

Rat liver lysosomes were isolated from a light mitochondrial-lysosomal fraction by centrifugation in a discontinuous metrizamide density gradient57.

LD isolationRat and mouse liver LD were isolated as described previously18. Livers homogenized in 0.25 M sucrose were centrifuged at 6,800 g for 5 min at 4°C. The supernatant and the fatty layer were centrifuged at 17,000 g for 10 min at 4°C to pellet lysosomes. The supernatant was adjusted to 20% sucrose (LD-enriched fraction) and centrifuged in a discontinuous sucrose density gradient at 28,000 g for 60 min at 4°C. The LD fraction collected was delipidated, solubilized in SDS, and equal protein amount was analyzed by immunoblot.

LD isolation

Rat and mouse liver LD were isolated as described previously18. Livers homogenized in 0.25 M sucrose were centrifuged at 6,800 g for 5 min at 4°C. The supernatant and the fatty layer were centrifuged at 17,000 g for 10 min at 4°C to pellet lysosomes. The supernatant was adjusted to 20% sucrose (LD-enriched fraction) and centrifuged in a discontinuous sucrose density gradient at 28,000 g for 60 min at 4°C. The LD fraction collected was delipidated, solubilized in SDS, and equal protein amount was analyzed by immunoblot.

CoimmunoprecipitationCoimmunoprecipitation was performed in total cell lysates or LD-enriched fractions prepared in 25 mM Tris (pH 7.2), 150 mM NaCl, 5 mM MgCl2, 0.5% NP-40, 1 mM DTT, 5% glycerol and protease inhibitors using standard procedures. Equal protein amount of each fraction was incubated with primary antibody (1 μg) overnight followed by incubation with protein A-conjugated sepharose for 1 h, centrifugation and washes.

Coimmunoprecipitation

Coimmunoprecipitation was performed in total cell lysates or LD-enriched fractions prepared in 25 mM Tris (pH 7.2), 150 mM NaCl, 5 mM MgCl2, 0.5% NP-40, 1 mM DTT, 5% glycerol and protease inhibitors using standard procedures. Equal protein amount of each fraction was incubated with primary antibody (1 μg) overnight followed by incubation with protein A-conjugated sepharose for 1 h, centrifugation and washes.

General MethodsTotal cell lysates were prepared in RIPA buffer (150 mM NaCl, 1% NP-40, 0.5% NaDoc, 0.1% SDS, 50 mM Tris pH 8) containing protease and phosphatase inhibitors. Protein concentration was measured using bovine serum albumin as a standard58. Isoelectric focusing (IEF) was done using the Protean IEF Cell (Bio-Rad) at 20°C with rapid ramping to voltage 10,000 V at a current limit of 50 μA using IPG Strips with 4–7pH range (Bio-Rad). Where indicated total cellular fractions were treated with λ-phosphatase according to manufacturer’s instructions (New England BioLabs) before isoelectric focusing. Phos-tag PAGE was performed according to manufacturer’s instructions (Wako) using 100 μM Phos-tag. Samples were subjected to SDS-PAGE, transferred to nitrocellulose membrane, blocked with low-fat milk and incubated with primary antibody overnight. The proteins were visualized by using peroxidase-conjugated secondary antibodies and chemiluminescent reagent (PerkinElmer) in LAS-3000 Imaging System (Fujifilm). Densitometric quantification was performed on unsaturated images using ImageJ.

General Methods

Total cell lysates were prepared in RIPA buffer (150 mM NaCl, 1% NP-40, 0.5% NaDoc, 0.1% SDS, 50 mM Tris pH 8) containing protease and phosphatase inhibitors. Protein concentration was measured using bovine serum albumin as a standard58. Isoelectric focusing (IEF) was done using the Protean IEF Cell (Bio-Rad) at 20°C with rapid ramping to voltage 10,000 V at a current limit of 50 μA using IPG Strips with 4–7pH range (Bio-Rad). Where indicated total cellular fractions were treated with λ-phosphatase according to manufacturer’s instructions (New England BioLabs) before isoelectric focusing. Phos-tag PAGE was performed according to manufacturer’s instructions (Wako) using 100 μM Phos-tag. Samples were subjected to SDS-PAGE, transferred to nitrocellulose membrane, blocked with low-fat milk and incubated with primary antibody overnight. The proteins were visualized by using peroxidase-conjugated secondary antibodies and chemiluminescent reagent (PerkinElmer) in LAS-3000 Imaging System (Fujifilm). Densitometric quantification was performed on unsaturated images using ImageJ.

Sample size, Randomization and Statistical AnalysesIn the studies performed in cell lines in culture, all experiments were repeated at least 3 times and often with duplicate blots in the same experiment. All numerical results are reported as mean ± standard error of the mean (SEM). Statistical significance of the difference between experimental groups was analyzed by two-tailed unpaired Student’s t-test using MS Excel. Differences were considered statistically significant for P<0.05. For those experiments in which an estimate of variation was possible based on previous studies or published work by others, we have confirmed that the variance is similar between the control and experimental groups compared. For those instances in which previous information was not available, we have compared with other conditions in which changes in cellular components resulted in changes in CMA.

Sample size, Randomization and Statistical Analyses

In the studies performed in cell lines in culture, all experiments were repeated at least 3 times and often with duplicate blots in the same experiment. All numerical results are reported as mean ± standard error of the mean (SEM). Statistical significance of the difference between experimental groups was analyzed by two-tailed unpaired Student’s t-test using MS Excel. Differences were considered statistically significant for P<0.05. For those experiments in which an estimate of variation was possible based on previous studies or published work by others, we have confirmed that the variance is similar between the control and experimental groups compared. For those instances in which previous information was not available, we have compared with other conditions in which changes in cellular components resulted in changes in CMA.

Supplementary Material123456789

Supplementary Material
