---
pmid: '20075868'
title: Nucleolar retention of a translational C/EBPalpha isoform stimulates rDNA transcription
  and cell size.
authors:
- Müller C
- Bremer A
- Schreiber S
- Eichwald S
- Calkhoven CF
journal: EMBO J
year: '2010'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2810377
doi: 10.1038/emboj.2009.404
---

# Nucleolar retention of a translational C/EBPalpha isoform stimulates rDNA transcription and cell size.
**Authors:** Müller C, Bremer A, Schreiber S, Eichwald S, Calkhoven CF
**Journal:** EMBO J (2010)
**DOI:** [10.1038/emboj.2009.404](https://doi.org/10.1038/emboj.2009.404)
**PMC:** [PMC2810377](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2810377/)

## Abstract

1. EMBO J. 2010 Mar 3;29(5):897-909. doi: 10.1038/emboj.2009.404. Epub 2010 Jan
14.

Nucleolar retention of a translational C/EBPalpha isoform stimulates rDNA 
transcription and cell size.

Müller C(1), Bremer A, Schreiber S, Eichwald S, Calkhoven CF.

Author information:
(1)Leibniz Institute for Age Research-Fritz Lipmann Institute, Jena, Germany.

The messenger RNA of the intronless CEBPA gene is translated into distinct 
protein isoforms through the usage of consecutive translation initiation sites. 
These translational isoforms have distinct functions in the regulation of 
differentiation and proliferation due to the presence of different N-terminal 
sequences. Here, we describe the function of an N-terminally extended protein 
isoform of CCAAT enhancer-binding protein alpha (C/EBPalpha) that is translated 
from an alternative non-AUG initiation codon. We show that a basic amino-acid 
motif within its N-terminus is required for nucleolar retention and for 
interaction with nucleophosmin (NPM). In the nucleoli, extended-C/EBPalpha 
occupies the ribosomal DNA (rDNA) promoter and associates with the Pol 
I-specific factors upstream-binding factor 1 (UBF-1) and SL1 to stimulate rRNA 
synthesis. Furthermore, during differentiation of HL-60 cells, endogenous 
expression of extended-C/EBPalpha is lost concomitantly with nucleolar 
C/EBPalpha immunostaining probably reflecting the reduced requirement for 
ribosome biogenesis in differentiated cells. Finally, overexpression of 
extended-C/EBPalpha induces an increase in cell size. Altogether, our results 
suggest that control of rRNA synthesis is a novel function of C/EBPalpha adding 
to its role as key regulator of cell growth and proliferation.

DOI: 10.1038/emboj.2009.404
PMCID: PMC2810377
PMID: 20075868 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare that they have no conflict 
of interest.

## Full Text

IntroductionIncreased cell proliferation, for example in cancerous growth, requires an increase in protein synthesis to fulfill the need for extra cell mass. Stimulation of RNA polymerase I (Pol I)-dependent ribosomal DNA (rDNA) transcription is a crucial event in ribosome biogenesis, which is prerequisite for enhanced protein synthesis (Ruggero and Pandolfi, 2003). In recent years, a growing number of RNA polymerase II (Pol II)-dependent transcription factors have been found to regulate RNA Pol I-dependent rDNA transcription in addition. These transcription factors either stimulate rRNA synthesis during proliferation like c-Myc (Arabi et al, 2005; Grandori et al, 2005) or inhibit rRNA synthesis as has been shown for CTCF (Torrano et al, 2006) and the lineage-specific transcription factors MyoD, myogenin and CCAAT enhancer-binding protein β (C/EBPβ) that selectively interact with rRNA genes during differentiation (Ali et al, 2008).C/EBPα controls cell proliferation as well as transcription of genes involved in differentiation and energy homeostasis in various cell types through redundant as well as specific functions compared with C/EBPβ and other C/EBP family members (Nerlov, 2007). In the haematopoietic system, C/EBPα is required for the differentiation of myelomonocytic cells, and mutations in the CEBPA gene are considered to be critically involved in the etiology of about 10% of human acute myeloid leukemia (AML) (Nerlov, 2004; Kirstetter et al, 2008). A pivotal mechanism that controls C/EBPα expression in the cell is the regulated translation of its intronless messenger RNA into three functionally different protein isoforms from three successive translation initiation sites (Ossipow et al, 1993; Calkhoven et al, 2000) (Figure 1A). The full-length-C/EBPα isoform (p42) that is translated from the first AUG codon in the C/EBPα reading frame is a complete transcription factor possessing all sequences for transactivation of genes and for inhibition of proliferation (Ossipow et al, 1993). In contrast, the truncated-C/EBPα isoform (p30), which is translated from an internal AUG codon, lacks the amino-terminal sequences for transactivation and is unable to restrict proliferation probably due to the absence of sequences required for the inhibition of E2F-dependent target genes (Lin et al, 1993; Porse et al, 2001). An evolutionary conserved upstream open reading frame (uORF) in the mRNA leader sequence controls the ratio between the full-length (p42) and truncated (p30) C/EBPα isoforms. Because of the cis-regulatory uORF, translation of the C/EBPα isoforms is under the control of the mTOR kinase-mediated regulation of eukaryotic initiation factor (eIF) 4E as well as the stress-induced eIF2α-kinase family (Calkhoven et al, 2000). Differential expression of the translational isoforms of C/EBPα provides a versatile mechanism to deliver functions on demand. However, it seems also to provide a critical target for deregulation and mutation during cancer development as is illustrated by a mouse model for the frequent mutation in human AML (Pabst et al, 2001) that results in the synthesis of only the truncated-C/EBPα isoform and causes AML with complete penetrance (Kirstetter et al, 2008).The function of the extended-C/EBPα isoform, which is translated from an upstream non-AUG codon has never been experimentally addressed. In this study, we reveal a novel function of C/EBPα that is specific for the extended isoform. We show that the extended-C/EBPα isoform can localize to nucleoli where it binds to the rDNA promoter. The extended-C/EBPα isoform interacts with and facilitates the recruitment of the Pol I-specific transacting factor upstream-binding factor 1 (UBF-1) and components of the TATA-binding protein (TBP)–TBP-associated factor (TAFI) complex SL1, and it enhances acetylation of histone H3 and H4 at the rDNA promoter. Furthermore, its nucleolar localization results in an increase in cell size. We identified a motif within the additional amino-terminal domain of the extended isoform that serves as a nucleolar localization signal (NoLS) and is required for interaction with nucleophosmin (NPM). In addition to the NoLS, a phosphorylation-mimicking mutation of a specific serine residue (S299D) strongly stimulates nucleolar retention, indicating that nucleolar retention of extended-C/EBPα may be further regulated through phosphorylation. We also show that downregulation of endogenous expression of extended-C/EBPα on phorbolester-induced differentiation of HL-60 cells correlates with loss of C/EBPα-specific nucleolar staining. Our data further strengthens the concept that regulated translation of the C/EBPα mRNA into protein isoforms with distinct functions is a key-regulatory mechanism in the control of proliferation and differentiation in C/EBPα expressing cells.

Introduction

Increased cell proliferation, for example in cancerous growth, requires an increase in protein synthesis to fulfill the need for extra cell mass. Stimulation of RNA polymerase I (Pol I)-dependent ribosomal DNA (rDNA) transcription is a crucial event in ribosome biogenesis, which is prerequisite for enhanced protein synthesis (Ruggero and Pandolfi, 2003). In recent years, a growing number of RNA polymerase II (Pol II)-dependent transcription factors have been found to regulate RNA Pol I-dependent rDNA transcription in addition. These transcription factors either stimulate rRNA synthesis during proliferation like c-Myc (Arabi et al, 2005; Grandori et al, 2005) or inhibit rRNA synthesis as has been shown for CTCF (Torrano et al, 2006) and the lineage-specific transcription factors MyoD, myogenin and CCAAT enhancer-binding protein β (C/EBPβ) that selectively interact with rRNA genes during differentiation (Ali et al, 2008).

C/EBPα controls cell proliferation as well as transcription of genes involved in differentiation and energy homeostasis in various cell types through redundant as well as specific functions compared with C/EBPβ and other C/EBP family members (Nerlov, 2007). In the haematopoietic system, C/EBPα is required for the differentiation of myelomonocytic cells, and mutations in the CEBPA gene are considered to be critically involved in the etiology of about 10% of human acute myeloid leukemia (AML) (Nerlov, 2004; Kirstetter et al, 2008). A pivotal mechanism that controls C/EBPα expression in the cell is the regulated translation of its intronless messenger RNA into three functionally different protein isoforms from three successive translation initiation sites (Ossipow et al, 1993; Calkhoven et al, 2000) (Figure 1A). The full-length-C/EBPα isoform (p42) that is translated from the first AUG codon in the C/EBPα reading frame is a complete transcription factor possessing all sequences for transactivation of genes and for inhibition of proliferation (Ossipow et al, 1993). In contrast, the truncated-C/EBPα isoform (p30), which is translated from an internal AUG codon, lacks the amino-terminal sequences for transactivation and is unable to restrict proliferation probably due to the absence of sequences required for the inhibition of E2F-dependent target genes (Lin et al, 1993; Porse et al, 2001). An evolutionary conserved upstream open reading frame (uORF) in the mRNA leader sequence controls the ratio between the full-length (p42) and truncated (p30) C/EBPα isoforms. Because of the cis-regulatory uORF, translation of the C/EBPα isoforms is under the control of the mTOR kinase-mediated regulation of eukaryotic initiation factor (eIF) 4E as well as the stress-induced eIF2α-kinase family (Calkhoven et al, 2000). Differential expression of the translational isoforms of C/EBPα provides a versatile mechanism to deliver functions on demand. However, it seems also to provide a critical target for deregulation and mutation during cancer development as is illustrated by a mouse model for the frequent mutation in human AML (Pabst et al, 2001) that results in the synthesis of only the truncated-C/EBPα isoform and causes AML with complete penetrance (Kirstetter et al, 2008).

The function of the extended-C/EBPα isoform, which is translated from an upstream non-AUG codon has never been experimentally addressed. In this study, we reveal a novel function of C/EBPα that is specific for the extended isoform. We show that the extended-C/EBPα isoform can localize to nucleoli where it binds to the rDNA promoter. The extended-C/EBPα isoform interacts with and facilitates the recruitment of the Pol I-specific transacting factor upstream-binding factor 1 (UBF-1) and components of the TATA-binding protein (TBP)–TBP-associated factor (TAFI) complex SL1, and it enhances acetylation of histone H3 and H4 at the rDNA promoter. Furthermore, its nucleolar localization results in an increase in cell size. We identified a motif within the additional amino-terminal domain of the extended isoform that serves as a nucleolar localization signal (NoLS) and is required for interaction with nucleophosmin (NPM). In addition to the NoLS, a phosphorylation-mimicking mutation of a specific serine residue (S299D) strongly stimulates nucleolar retention, indicating that nucleolar retention of extended-C/EBPα may be further regulated through phosphorylation. We also show that downregulation of endogenous expression of extended-C/EBPα on phorbolester-induced differentiation of HL-60 cells correlates with loss of C/EBPα-specific nucleolar staining. Our data further strengthens the concept that regulated translation of the C/EBPα mRNA into protein isoforms with distinct functions is a key-regulatory mechanism in the control of proliferation and differentiation in C/EBPα expressing cells.

ResultsExtended-C/EBPα is translated from a non-AUG alternative initiation codonAmino-acid sequence comparison reveals a high degree of conservation among mammals of the region upstream of the first AUG-codon in the C/EBPα coding frame as exemplified in Figure 1B for the murine and human C/EBPα cDNAs. For murine C/EBPα, these N-terminally extended sequences can be incorporated into the C/EBPα proteins through initiation from one of the two successive CUG alternative initiation codons (A-sites in Figure 1A) that lie −28 and −37 codons upstream of the first AUG (site B in Figure 1A) of the C/EBPα coding frame (Calkhoven et al, 2000) (and data not shown). In case of the human sequence, a GUG alternative initiation codon is found at position −35. Mutation of the most prominently used CUG codon (ΔA2) in the rat C/EBPα cDNA or the GUG codon (ΔA1) in the human C/EBPα cDNA as well as removal of the complete sequence upstream of the first AUG (Δ5′) result in complete loss of extended-C/EBPα expression in COS-1 cells (Figure 1C). Conversely, converting the CUG or GUG codons into a proper AUG codon (Aopt) results in predominant expression of the extended isoform (Figure 1C). Hence, alternative CUG or GUG codons in the 5′-leader sequence of C/EBPα mRNAs can be used for initiation of translation.The extended-C/EBPα isoform is retained in the nucleolusC/EBPα is a transcription factor that predominantly localizes to the nucleus. It contains a bipartite nuclear localization signal (NLS) within its carboxy-terminal basic region that is present in all three translational isoforms (Williams et al, 1997). To compare the subcellular distribution between the three translational isoforms of C/EBPα, we examined their cellular localization by fluorescence microscopic analysis using the Zeiss ApoTome system. For this purpose, we transfected C33A cells with expression constructs containing sequences for the separate C/EBPα isoforms or constructs with the C/EBPα isoforms fused at their carboxy-termini to enhanced green fluorescent protein (EGFP). We found that in a consistent percentage (10%) of transfected cells, the extended-C/EBPα isoform localized to the nucleoli as was detected by either direct (Figure 2A) or indirect fluorescence (see Supplementary Figure S1). Nucleolar localization was verified by co-immunostaining of the nucleolar marker protein Fibrillarin, which is a component of the rRNA processing machinery. In contrast, nucleolar localization was never observed with transient expression of the full-length- or truncated-C/EBPα isoforms, indicating that nucleolar localization is a specific feature of the extended-C/EBPα isoform. Interestingly, during the establishment of C33A cell lines that stably express extended-C/EBPα, the percentage of cells with nucleolar retention continuously raises up to 100%, in long-term cultures, suggesting that these cells have a selective proliferation or survival advantage (data not shown).NoLSs are often characterized by stretches of basic amino acids (Carmo-Fonseca et al, 2000; Emmott and Hiscox, 2009). The extended-C/EBPα domain contains a conserved RRXR motif that is also found in proven NoLSs of other proteins like Parp2, p14/19Arf, ING1b, Rpp29 or HIV Tat (Kubota et al, 1989; Jarrous et al, 1999; Weber et al, 1999; Scott et al, 2001; Meder et al, 2005) (Figure 2B). To examine whether the extended domain is required and sufficient for nucleolar retention, we fused the 28 amino acids of the extended domain to EGFP and determined cellular localization by microscopic analysis. Figure 2C shows that the sequences of the extended C/EBPα domain alone are very efficient in retaining the attached EGFP in the nucleolus in almost 100% of transfected cells. In contrast, the mutation of the RRQR motif into AAAA abolished nucleolar localization of the extended domain EGFP fusion construct and resulted in even distribution over the cell similar to the EGFP control. Hence, nucleolar localization is a unique feature of the extended-C/EBPα isoform and the RRXR NoLS-motif within the extended domain of C/EBPα is required for its nucleolar retention.Pseudo-phosphorylation of serine 299 of extended- C/EBPα stimulates nucleolar retentionThe observation that transiently expressed extended-C/EBPα is retained in the nucleoli in a portion of cells, yet unrestrained nucleolar retention can be achieved with the sole EGFP-fused extended domain indicates that nucleolar retention of the whole protein may be subject of further regulation. It has been shown for other proteins with partial nucleolar localization like Parp2 or ING1b that in addition to the RRXR motif also their NLSs contributed to the efficiency of nucleolar retention independent from their function for nuclear localization (Scott et al, 2001; Meder et al, 2005). We, therefore, examined the effect of different mutations within the bipartite NLS of C/EBPα and identified a serine residue within the NLS (S299 in full-length-C/EBPα) whose mutation into a phosphorylation-mimicking aspartate causes efficient nucleolar retention. Table 1 presents data from triplicate transfection experiments in four cell lines (C33A, HEK293, COS-1, HeLa) showing that in all cell lines, the S299D mutation rendered nucleolar retention to virtually 100% of transfected cells, compared with around 10% nucleolar versus 90% nucleoplasmic staining for the wt extended-C/EBPα-EGFP. Mutation of the RRQR NoLS-motif in the extended-C/EBPαS299D-EGFP construct completely abolished nucleolar retention in all cases. Notably, we did not score a single cell with nucleolar localization of full-length- or truncated-C/EBPαS299D-EGFP fusion proteins either with or without the S299D mutation during the examination of all the cell lines. Figure 3A shows a typical example for all C/EBPαS299D-EGFP isoforms in C33A cells and Figure 3B shows nucleolar retention for extended-C/EBPαS299D-EGFP in five additional cell lines. Mutation of the serine 299 into alanine did not stimulate nucleolar retention of the extended-C/EBPα-EGFP fusion protein or the extended-C/EBPα isoform (data not shown). Similar results were obtained by immunostaining of cells transfected with expression constructs for native C/EBPα isoforms without EGFP (Supplementary Figure S2A). Although, a similar extended isoform of the related C/EBPβ exists that is translated from an upstream AUG-codon, it does not have a NoLS, and we did not observe localization of extended-C/EBPβ in the nucleolus whether it harbours an S239D mutation that corresponds to the S299D mutation in C/EBPα or not (data not shown). Hence, our data suggest that nucleolar retention of extended-C/EBPα is cell-type independent, requires the RRQR NoLS-motif in the extended domain and is enhanced by phosphorylation of serine 299 within the NLS.Endogenous C/EBPα localizes to the nucleolus in myelomonocytic cellsThe promyelocytic human leukemia cell line HL-60 has retained the capacity to differentiate in cell culture and is one of the few cell lines that express endogenous C/EBPα. Immunoblot analysis confirmed that proliferating and undifferentiated HL-60 cells express all three C/EBPα translational isoforms, depicted extended-, full-length- and truncated-C/EBPα (Figure 4A). Intriguingly, indirect fluorescent immunostaining of C/EBPα in HL-60 cells using four different anti-C/EBPα antibodies revealed a prominent staining of the nucleoli in addition to the expected nuclear staining (Figure 4B; Supplementary Figure S3A). Induction of macrophage differentiation by phorbolester (TPA) treatment resulted in complete disappearance of the C/EBPα-specific staining of the nucleoli although staining of the remaining part of the nucleus remained unchanged (Figure 4B). Immunoblotting revealed that the expression of the extended- and truncated-C/EBPα isoforms were specifically downregulated in the differentiated HL-60 cells (Figure 4A). Hence, loss of expression of the extended-C/EBPα isoform correlates with loss of nucleolar staining. C/EBPα-specific nucleolar staining was also found in another human leukemia cell line, the promonocytic U937 cell line (Supplementary Figure S3B and C).Extended-C/EBPα interacts with Pol I-specific factors and its nucleolar retention depends on ongoing rDNA transcriptionThe nucleolus is the place where rDNA transcription is performed through the activity of RNA Pol I. The nucleolar transcription factor UBF-1 is a crucial component of the RNA Pol I preinitiation complex (Grummt, 2003). Immunofluorescence microscopy revealed that ectopically expressed extended-C/EBPαS299D but not the corresponding ΔNoLS mutant localizes with endogenous UBF-1 in the nucleoli (Supplementary Figure S4A). To examine whether C/EBPα interacts with UBF-1, HEK293 cells were transfected with expression plasmids for extended-, full-length or truncated-C/EBPα, respectively, followed by co-immunoprecipitation (Co-IP) experiments. We found that both wt and the S299D mutant of extended-C/EBPα (Figure 5A) as well as full-length-C/EBPα (Figure 5B) can be co-precipitated together with endogenous UBF-1 using anti-UBF-1 antibodies. In contrast, we could not detect a clear co-precipitation of truncated-C/EBPα.RNA Pol I transcription requires the SL1 complex, which is composed of the TBP and three Pol I-specific TAFs, TAFI110, TAFI63 and TAFI48. SL1 was shown to be essential for UBF-1 recruitment and pre-initiation complex formation (Friedrich et al, 2005). As full-length C/EBPα is known to bind to and recruit TBP to Pol II promoters (Nerlov and Ziff, 1995), we examined whether extended-C/EBPα interacts with TBP and TAFI48. Co-IP using anti-C/EBPα antibodies resulted in precipitation of TBP by extended-C/EBPα, as well as the S299D and the S299D-ΔNoLS mutant thereof (Figure 5C). Hence, interaction with TBP is a genuine property of both full-length and extended-C/EBPα. In addition, we could co-precipitate extended-C/EBPαS299D with anti-TAFI48 antibodies; however, co-precipitation was not seen with the ΔNoLS mutant (Figure 5D, upper panel). Furthermore, TAFI48 can be precipitated with anti-TBP antibodies more efficiently in cells that express extended-C/EBPα, in particular the S299D mutant, compared with ΔNoLS or empty vector controls (Figure 5D, lower panel). Overall these results indicate that extended-C/EBPα, UBF-1, and TBP and TAFI48 as part of the SL1 complex reside as a complex in the nucleolus.It has been shown that the RRQR motif in Parp2 is required for nucleolar retention and for interaction with NPM (also known as B23) (Meder et al, 2005), a ubiquitously expressed phosphoprotein with multiple functions that constantly shuttles between the nucleolus, nucleoplasm and cytoplasm (Grisendi et al, 2006). To examine whether a similar interaction between NPM and C/EBPα exists, Co-IP experiments were set up and as presented in Figure 5E, NPM could only be co-immunoprecipitated with wt and the S299D mutant of extended-C/EBPα while co-imunoprecipitation failed with the ΔNoLS mutant or full-length isoform that both lack RRQR motif.Typically, nucleolar retention of regulatory factors depends on ongoing rDNA transcription. To examine whether nucleolar localization of the extended-C/EBPαS299D mutant depends on ongoing rDNA transcription, we analysed its localization in the presence of a low concentration of actinomycin D (50 ng/ml), which specifically inhibits RNA Pol I transcription (Perry and Kelley, 1970). As shown in Figure 6 and Supplementary Figure S4B, nucleolar localization of extended-C/EBPαS299D was lost in the presence of actinomycin D, whereas inhibition of RNA Pol II driven transcription with α-amanitin had no effect on the localization of the extended isoform. Taken together, these results show that extended-C/EBPα interacts with functional nucleolar proteins, UBF-1 and NPM, and the SL1 components TBP and TAFI48, and that its nucleolar retention depends on ongoing rDNA transcription.Extended-C/EBPα stimulates RNA Pol I-dependent rRNA synthesisAs C/EBPα is a transcription factor, we examined whether extended-C/EBPα was able to regulate RNA Pol I-mediated rRNA synthesis as has been shown also for other RNA Pol II-dependent transcription factors, including c-Myc (Arabi et al, 2005; Grandori et al, 2005). To address this question, we co-transfected a human rRNA minigene reporter construct (pMr1930-BH) (Mayer et al, 2005) together with expression vectors for extended-C/EBPα in HEK293T cells. We observed that overexpression of both extended-C/EBPα and the S299D mutant thereof strongly induced RNA Pol I-mediated transcription of the reporter RNA to a similar extent as c-Myc, which was used as a positive control (Figure 7A). To examine whether extended-C/EBPα is able to stimulate endogenous rDNA transcription, we analysed the expression levels of 45S pre-rRNA by Northern blotting in long-term C33A cell lines that stably express extended-, full-length- or truncated-C/EBPα wt proteins, the S299D mutants thereof and ΔNoLS mutants. Figure 7B shows that expression of extended-C/EBPα and extended-C/EBPαS299D results in increased levels of endogenous 45S pre-rRNA compared with the empty vector control. In contrast, expression of the full-length- or truncated-C/EBPα isoforms either with or without the S299D mutation was not able to enhance 45S pre-rRNA levels. The ΔNoLS mutation prevents induction of 45S pr-RNA (Figure 7C). To examine a possible indirect Pol II-dependent regulation of rDNA transcription by extended-C/EBPα, 45S pre-rRNA levels were analysed under inhibition of Pol II by α-amanitin (50 μg/ml). The results in Figure 7D show that inhibition of Pol II only slightly affected induction of 45S pre-rRNA, suggesting a direct regulation of Pol I by extended-C/EBPα. To examine whether selective elimination of extended-C/EBPα impedes rRNA synthesis, we knocked-down expression of endogenous C/EBPα in HL-60 cells by introduction of lentiviral human C/EBPα-specific shRNA (Figure 8A). Subsequently, in the C/EBPα knock-down cells, expression was restored from retrovirally transduced constructs coding for all three C/EBPα-FLAG isoforms, or from a construct with a ΔCUG mutation avoiding expression of the extended-C/EBPα isoform (Figure 8B). As demonstrated by Northern analysis, 45S pre-rRNA levels were only significantly stimulated in the presence of extended-C/EBPα and not when full-length- and truncated-C/EBPα are expressed alone from the ΔCUGα construct. Once more, the requirement of the N-terminal NoLS was demonstrated as stimulation of 45S pre-rRNA levels by overexpression of extended-C/EBPαS299D alone is abrogated when NoLS is mutated (Figure 8C). Taken together, extended-C/EBPα directly and specifically enhances rDNA transcription, which depends on nucleolar retention mediated by the N-terminally located NoLS.Bioinformatic analysis revealed two C/EBP-binding motifs in the intergenic spacer of the human rDNA repeats (Figure 9A). One of the C/EBP-binding motifs lies 25 bp upstream of the transcription start, in the same proximal promoter sequences that are crucial for Pol I regulation by c-Myc (Figure 9A; region Cp, same as region 42.9 in Grandori et al, 2005). Chromatin immunoprecipitation (ChIP) experiments using anti-C/EBPα antibodies revealed that extended-C/EBPα occupies this rDNA promoter segment (Figure 9B; upper chart, region Cp) in C33A cells. Notably, the more efficient nucleolar retention of extended-C/EBPαS299D results in recovery of more Cp DNA fragments whereas the ΔNoLS mutation abolishes DNA recovery (Figure 9B and C). In addition, extended-C/EBPα binds to a more distal region containing the second C/EBP-binding motif (Figure 9A and B; upper chart, region Cd). Extended-C/EBPα does not interact with region M that does not contain a C/EBP-binding motif, which, however, was shown to be implicated in c-Myc-dependent regulation (Figure 9A and B; region M, same as region 42 in Grandori et al, 2005).Region Cp covers the Pol I promoter and is also heavily occupied by UBF-1, as was also observed by others (Figure 9B, lower chart, region Cp) (Grandori et al, 2005; here region 42.9). Interestingly, the enhanced interaction of extended-C/EBPαS299D with the regions Cp and Cd improves recruitment of UBF-1 to these regions (Figure 9B and C). To investigate whether extended-C/EBPα also facilitates the recruitment of components of the SL1 complex to the rDNA promoter, we performed a ChIP assay using an anti-TBP antibody. Figure 9D shows that indeed recruitment of TBP is enhanced on extended-C/EBPα binding to region Cp. c-Myc like extended-C/EBPα binds most strongly to region Cp and stimulates histone H3 and H4 acetylation especially at the neighbouring region M (Grandori et al, 2005). C/EBPα is known to activate transcription from Pol II promoters through recruitment of histone acetylases (Erickson et al, 2001; Bararia et al, 2008). ChIP analysis using anti-histone H3 or H4 antibodies revealed that histone acetylation is enhanced particularly at site M and to a lesser extend to Cp when extended-C/EBPα binds to the rDNA promoter (Figure 9E). Hence, our results suggest that the occupation of the rDNA promoter region by extended-C/EBPα facilitates recruitment of UBF-1 and SL1, and induces histone acetylation.Overexpression of extended-C/EBPα results in an increase in cell sizeIncrease in cell size is a well-known consequence of enhanced ribosome biogenesis (Baserga, 2007), and the stimulation of rRNA synthesis by c-Myc has been associated with increased cell size in different systems (Oskarsson and Trumpp, 2005). Cell volume analysis of the stably transfected cells revealed that expression of extended-C/EBPαwt- and in particular the extended-C/EBPαS299D isoform results in a profound increase in the mean cell volume, whereas the other isoforms show no effect (Figure 10). C33A cells lack a functional Brahma protein as part of the SWI/SNF chromatin-remodelling complex, which is required for C/EBPα-induced proliferation arrest (Müller et al, 2004). To study whether the absence of Brahma is instrumental in the pronounced increase in cell volume, we analysed C33A cells in which Brahma function was restored by expression from a retroviral construct compared with dominant negative mutant of Brahma and empty vector. In none of the situations, however, cell size was affected (Supplementary Figure S5A). Hence, Brahma function has no effect on cell size. In addition, over-expression of extended-C/EBPαwt- and the extended-C/EBPαS299D isoform in HEK293 cells that have a functional SWI/SNF complex resulted in a clear although less pronounced increase in cell size (Supplementary Figure S5B).

Extended-C/EBPα is translated from a non-AUG alternative initiation codonAmino-acid sequence comparison reveals a high degree of conservation among mammals of the region upstream of the first AUG-codon in the C/EBPα coding frame as exemplified in Figure 1B for the murine and human C/EBPα cDNAs. For murine C/EBPα, these N-terminally extended sequences can be incorporated into the C/EBPα proteins through initiation from one of the two successive CUG alternative initiation codons (A-sites in Figure 1A) that lie −28 and −37 codons upstream of the first AUG (site B in Figure 1A) of the C/EBPα coding frame (Calkhoven et al, 2000) (and data not shown). In case of the human sequence, a GUG alternative initiation codon is found at position −35. Mutation of the most prominently used CUG codon (ΔA2) in the rat C/EBPα cDNA or the GUG codon (ΔA1) in the human C/EBPα cDNA as well as removal of the complete sequence upstream of the first AUG (Δ5′) result in complete loss of extended-C/EBPα expression in COS-1 cells (Figure 1C). Conversely, converting the CUG or GUG codons into a proper AUG codon (Aopt) results in predominant expression of the extended isoform (Figure 1C). Hence, alternative CUG or GUG codons in the 5′-leader sequence of C/EBPα mRNAs can be used for initiation of translation.

Extended-C/EBPα is translated from a non-AUG alternative initiation codon

Amino-acid sequence comparison reveals a high degree of conservation among mammals of the region upstream of the first AUG-codon in the C/EBPα coding frame as exemplified in Figure 1B for the murine and human C/EBPα cDNAs. For murine C/EBPα, these N-terminally extended sequences can be incorporated into the C/EBPα proteins through initiation from one of the two successive CUG alternative initiation codons (A-sites in Figure 1A) that lie −28 and −37 codons upstream of the first AUG (site B in Figure 1A) of the C/EBPα coding frame (Calkhoven et al, 2000) (and data not shown). In case of the human sequence, a GUG alternative initiation codon is found at position −35. Mutation of the most prominently used CUG codon (ΔA2) in the rat C/EBPα cDNA or the GUG codon (ΔA1) in the human C/EBPα cDNA as well as removal of the complete sequence upstream of the first AUG (Δ5′) result in complete loss of extended-C/EBPα expression in COS-1 cells (Figure 1C). Conversely, converting the CUG or GUG codons into a proper AUG codon (Aopt) results in predominant expression of the extended isoform (Figure 1C). Hence, alternative CUG or GUG codons in the 5′-leader sequence of C/EBPα mRNAs can be used for initiation of translation.

The extended-C/EBPα isoform is retained in the nucleolusC/EBPα is a transcription factor that predominantly localizes to the nucleus. It contains a bipartite nuclear localization signal (NLS) within its carboxy-terminal basic region that is present in all three translational isoforms (Williams et al, 1997). To compare the subcellular distribution between the three translational isoforms of C/EBPα, we examined their cellular localization by fluorescence microscopic analysis using the Zeiss ApoTome system. For this purpose, we transfected C33A cells with expression constructs containing sequences for the separate C/EBPα isoforms or constructs with the C/EBPα isoforms fused at their carboxy-termini to enhanced green fluorescent protein (EGFP). We found that in a consistent percentage (10%) of transfected cells, the extended-C/EBPα isoform localized to the nucleoli as was detected by either direct (Figure 2A) or indirect fluorescence (see Supplementary Figure S1). Nucleolar localization was verified by co-immunostaining of the nucleolar marker protein Fibrillarin, which is a component of the rRNA processing machinery. In contrast, nucleolar localization was never observed with transient expression of the full-length- or truncated-C/EBPα isoforms, indicating that nucleolar localization is a specific feature of the extended-C/EBPα isoform. Interestingly, during the establishment of C33A cell lines that stably express extended-C/EBPα, the percentage of cells with nucleolar retention continuously raises up to 100%, in long-term cultures, suggesting that these cells have a selective proliferation or survival advantage (data not shown).NoLSs are often characterized by stretches of basic amino acids (Carmo-Fonseca et al, 2000; Emmott and Hiscox, 2009). The extended-C/EBPα domain contains a conserved RRXR motif that is also found in proven NoLSs of other proteins like Parp2, p14/19Arf, ING1b, Rpp29 or HIV Tat (Kubota et al, 1989; Jarrous et al, 1999; Weber et al, 1999; Scott et al, 2001; Meder et al, 2005) (Figure 2B). To examine whether the extended domain is required and sufficient for nucleolar retention, we fused the 28 amino acids of the extended domain to EGFP and determined cellular localization by microscopic analysis. Figure 2C shows that the sequences of the extended C/EBPα domain alone are very efficient in retaining the attached EGFP in the nucleolus in almost 100% of transfected cells. In contrast, the mutation of the RRQR motif into AAAA abolished nucleolar localization of the extended domain EGFP fusion construct and resulted in even distribution over the cell similar to the EGFP control. Hence, nucleolar localization is a unique feature of the extended-C/EBPα isoform and the RRXR NoLS-motif within the extended domain of C/EBPα is required for its nucleolar retention.

The extended-C/EBPα isoform is retained in the nucleolus

C/EBPα is a transcription factor that predominantly localizes to the nucleus. It contains a bipartite nuclear localization signal (NLS) within its carboxy-terminal basic region that is present in all three translational isoforms (Williams et al, 1997). To compare the subcellular distribution between the three translational isoforms of C/EBPα, we examined their cellular localization by fluorescence microscopic analysis using the Zeiss ApoTome system. For this purpose, we transfected C33A cells with expression constructs containing sequences for the separate C/EBPα isoforms or constructs with the C/EBPα isoforms fused at their carboxy-termini to enhanced green fluorescent protein (EGFP). We found that in a consistent percentage (10%) of transfected cells, the extended-C/EBPα isoform localized to the nucleoli as was detected by either direct (Figure 2A) or indirect fluorescence (see Supplementary Figure S1). Nucleolar localization was verified by co-immunostaining of the nucleolar marker protein Fibrillarin, which is a component of the rRNA processing machinery. In contrast, nucleolar localization was never observed with transient expression of the full-length- or truncated-C/EBPα isoforms, indicating that nucleolar localization is a specific feature of the extended-C/EBPα isoform. Interestingly, during the establishment of C33A cell lines that stably express extended-C/EBPα, the percentage of cells with nucleolar retention continuously raises up to 100%, in long-term cultures, suggesting that these cells have a selective proliferation or survival advantage (data not shown).

NoLSs are often characterized by stretches of basic amino acids (Carmo-Fonseca et al, 2000; Emmott and Hiscox, 2009). The extended-C/EBPα domain contains a conserved RRXR motif that is also found in proven NoLSs of other proteins like Parp2, p14/19Arf, ING1b, Rpp29 or HIV Tat (Kubota et al, 1989; Jarrous et al, 1999; Weber et al, 1999; Scott et al, 2001; Meder et al, 2005) (Figure 2B). To examine whether the extended domain is required and sufficient for nucleolar retention, we fused the 28 amino acids of the extended domain to EGFP and determined cellular localization by microscopic analysis. Figure 2C shows that the sequences of the extended C/EBPα domain alone are very efficient in retaining the attached EGFP in the nucleolus in almost 100% of transfected cells. In contrast, the mutation of the RRQR motif into AAAA abolished nucleolar localization of the extended domain EGFP fusion construct and resulted in even distribution over the cell similar to the EGFP control. Hence, nucleolar localization is a unique feature of the extended-C/EBPα isoform and the RRXR NoLS-motif within the extended domain of C/EBPα is required for its nucleolar retention.

Pseudo-phosphorylation of serine 299 of extended- C/EBPα stimulates nucleolar retentionThe observation that transiently expressed extended-C/EBPα is retained in the nucleoli in a portion of cells, yet unrestrained nucleolar retention can be achieved with the sole EGFP-fused extended domain indicates that nucleolar retention of the whole protein may be subject of further regulation. It has been shown for other proteins with partial nucleolar localization like Parp2 or ING1b that in addition to the RRXR motif also their NLSs contributed to the efficiency of nucleolar retention independent from their function for nuclear localization (Scott et al, 2001; Meder et al, 2005). We, therefore, examined the effect of different mutations within the bipartite NLS of C/EBPα and identified a serine residue within the NLS (S299 in full-length-C/EBPα) whose mutation into a phosphorylation-mimicking aspartate causes efficient nucleolar retention. Table 1 presents data from triplicate transfection experiments in four cell lines (C33A, HEK293, COS-1, HeLa) showing that in all cell lines, the S299D mutation rendered nucleolar retention to virtually 100% of transfected cells, compared with around 10% nucleolar versus 90% nucleoplasmic staining for the wt extended-C/EBPα-EGFP. Mutation of the RRQR NoLS-motif in the extended-C/EBPαS299D-EGFP construct completely abolished nucleolar retention in all cases. Notably, we did not score a single cell with nucleolar localization of full-length- or truncated-C/EBPαS299D-EGFP fusion proteins either with or without the S299D mutation during the examination of all the cell lines. Figure 3A shows a typical example for all C/EBPαS299D-EGFP isoforms in C33A cells and Figure 3B shows nucleolar retention for extended-C/EBPαS299D-EGFP in five additional cell lines. Mutation of the serine 299 into alanine did not stimulate nucleolar retention of the extended-C/EBPα-EGFP fusion protein or the extended-C/EBPα isoform (data not shown). Similar results were obtained by immunostaining of cells transfected with expression constructs for native C/EBPα isoforms without EGFP (Supplementary Figure S2A). Although, a similar extended isoform of the related C/EBPβ exists that is translated from an upstream AUG-codon, it does not have a NoLS, and we did not observe localization of extended-C/EBPβ in the nucleolus whether it harbours an S239D mutation that corresponds to the S299D mutation in C/EBPα or not (data not shown). Hence, our data suggest that nucleolar retention of extended-C/EBPα is cell-type independent, requires the RRQR NoLS-motif in the extended domain and is enhanced by phosphorylation of serine 299 within the NLS.

Pseudo-phosphorylation of serine 299 of extended- C/EBPα stimulates nucleolar retention

The observation that transiently expressed extended-C/EBPα is retained in the nucleoli in a portion of cells, yet unrestrained nucleolar retention can be achieved with the sole EGFP-fused extended domain indicates that nucleolar retention of the whole protein may be subject of further regulation. It has been shown for other proteins with partial nucleolar localization like Parp2 or ING1b that in addition to the RRXR motif also their NLSs contributed to the efficiency of nucleolar retention independent from their function for nuclear localization (Scott et al, 2001; Meder et al, 2005). We, therefore, examined the effect of different mutations within the bipartite NLS of C/EBPα and identified a serine residue within the NLS (S299 in full-length-C/EBPα) whose mutation into a phosphorylation-mimicking aspartate causes efficient nucleolar retention. Table 1 presents data from triplicate transfection experiments in four cell lines (C33A, HEK293, COS-1, HeLa) showing that in all cell lines, the S299D mutation rendered nucleolar retention to virtually 100% of transfected cells, compared with around 10% nucleolar versus 90% nucleoplasmic staining for the wt extended-C/EBPα-EGFP. Mutation of the RRQR NoLS-motif in the extended-C/EBPαS299D-EGFP construct completely abolished nucleolar retention in all cases. Notably, we did not score a single cell with nucleolar localization of full-length- or truncated-C/EBPαS299D-EGFP fusion proteins either with or without the S299D mutation during the examination of all the cell lines. Figure 3A shows a typical example for all C/EBPαS299D-EGFP isoforms in C33A cells and Figure 3B shows nucleolar retention for extended-C/EBPαS299D-EGFP in five additional cell lines. Mutation of the serine 299 into alanine did not stimulate nucleolar retention of the extended-C/EBPα-EGFP fusion protein or the extended-C/EBPα isoform (data not shown). Similar results were obtained by immunostaining of cells transfected with expression constructs for native C/EBPα isoforms without EGFP (Supplementary Figure S2A). Although, a similar extended isoform of the related C/EBPβ exists that is translated from an upstream AUG-codon, it does not have a NoLS, and we did not observe localization of extended-C/EBPβ in the nucleolus whether it harbours an S239D mutation that corresponds to the S299D mutation in C/EBPα or not (data not shown). Hence, our data suggest that nucleolar retention of extended-C/EBPα is cell-type independent, requires the RRQR NoLS-motif in the extended domain and is enhanced by phosphorylation of serine 299 within the NLS.

Endogenous C/EBPα localizes to the nucleolus in myelomonocytic cellsThe promyelocytic human leukemia cell line HL-60 has retained the capacity to differentiate in cell culture and is one of the few cell lines that express endogenous C/EBPα. Immunoblot analysis confirmed that proliferating and undifferentiated HL-60 cells express all three C/EBPα translational isoforms, depicted extended-, full-length- and truncated-C/EBPα (Figure 4A). Intriguingly, indirect fluorescent immunostaining of C/EBPα in HL-60 cells using four different anti-C/EBPα antibodies revealed a prominent staining of the nucleoli in addition to the expected nuclear staining (Figure 4B; Supplementary Figure S3A). Induction of macrophage differentiation by phorbolester (TPA) treatment resulted in complete disappearance of the C/EBPα-specific staining of the nucleoli although staining of the remaining part of the nucleus remained unchanged (Figure 4B). Immunoblotting revealed that the expression of the extended- and truncated-C/EBPα isoforms were specifically downregulated in the differentiated HL-60 cells (Figure 4A). Hence, loss of expression of the extended-C/EBPα isoform correlates with loss of nucleolar staining. C/EBPα-specific nucleolar staining was also found in another human leukemia cell line, the promonocytic U937 cell line (Supplementary Figure S3B and C).

Endogenous C/EBPα localizes to the nucleolus in myelomonocytic cells

The promyelocytic human leukemia cell line HL-60 has retained the capacity to differentiate in cell culture and is one of the few cell lines that express endogenous C/EBPα. Immunoblot analysis confirmed that proliferating and undifferentiated HL-60 cells express all three C/EBPα translational isoforms, depicted extended-, full-length- and truncated-C/EBPα (Figure 4A). Intriguingly, indirect fluorescent immunostaining of C/EBPα in HL-60 cells using four different anti-C/EBPα antibodies revealed a prominent staining of the nucleoli in addition to the expected nuclear staining (Figure 4B; Supplementary Figure S3A). Induction of macrophage differentiation by phorbolester (TPA) treatment resulted in complete disappearance of the C/EBPα-specific staining of the nucleoli although staining of the remaining part of the nucleus remained unchanged (Figure 4B). Immunoblotting revealed that the expression of the extended- and truncated-C/EBPα isoforms were specifically downregulated in the differentiated HL-60 cells (Figure 4A). Hence, loss of expression of the extended-C/EBPα isoform correlates with loss of nucleolar staining. C/EBPα-specific nucleolar staining was also found in another human leukemia cell line, the promonocytic U937 cell line (Supplementary Figure S3B and C).

Extended-C/EBPα interacts with Pol I-specific factors and its nucleolar retention depends on ongoing rDNA transcriptionThe nucleolus is the place where rDNA transcription is performed through the activity of RNA Pol I. The nucleolar transcription factor UBF-1 is a crucial component of the RNA Pol I preinitiation complex (Grummt, 2003). Immunofluorescence microscopy revealed that ectopically expressed extended-C/EBPαS299D but not the corresponding ΔNoLS mutant localizes with endogenous UBF-1 in the nucleoli (Supplementary Figure S4A). To examine whether C/EBPα interacts with UBF-1, HEK293 cells were transfected with expression plasmids for extended-, full-length or truncated-C/EBPα, respectively, followed by co-immunoprecipitation (Co-IP) experiments. We found that both wt and the S299D mutant of extended-C/EBPα (Figure 5A) as well as full-length-C/EBPα (Figure 5B) can be co-precipitated together with endogenous UBF-1 using anti-UBF-1 antibodies. In contrast, we could not detect a clear co-precipitation of truncated-C/EBPα.RNA Pol I transcription requires the SL1 complex, which is composed of the TBP and three Pol I-specific TAFs, TAFI110, TAFI63 and TAFI48. SL1 was shown to be essential for UBF-1 recruitment and pre-initiation complex formation (Friedrich et al, 2005). As full-length C/EBPα is known to bind to and recruit TBP to Pol II promoters (Nerlov and Ziff, 1995), we examined whether extended-C/EBPα interacts with TBP and TAFI48. Co-IP using anti-C/EBPα antibodies resulted in precipitation of TBP by extended-C/EBPα, as well as the S299D and the S299D-ΔNoLS mutant thereof (Figure 5C). Hence, interaction with TBP is a genuine property of both full-length and extended-C/EBPα. In addition, we could co-precipitate extended-C/EBPαS299D with anti-TAFI48 antibodies; however, co-precipitation was not seen with the ΔNoLS mutant (Figure 5D, upper panel). Furthermore, TAFI48 can be precipitated with anti-TBP antibodies more efficiently in cells that express extended-C/EBPα, in particular the S299D mutant, compared with ΔNoLS or empty vector controls (Figure 5D, lower panel). Overall these results indicate that extended-C/EBPα, UBF-1, and TBP and TAFI48 as part of the SL1 complex reside as a complex in the nucleolus.It has been shown that the RRQR motif in Parp2 is required for nucleolar retention and for interaction with NPM (also known as B23) (Meder et al, 2005), a ubiquitously expressed phosphoprotein with multiple functions that constantly shuttles between the nucleolus, nucleoplasm and cytoplasm (Grisendi et al, 2006). To examine whether a similar interaction between NPM and C/EBPα exists, Co-IP experiments were set up and as presented in Figure 5E, NPM could only be co-immunoprecipitated with wt and the S299D mutant of extended-C/EBPα while co-imunoprecipitation failed with the ΔNoLS mutant or full-length isoform that both lack RRQR motif.Typically, nucleolar retention of regulatory factors depends on ongoing rDNA transcription. To examine whether nucleolar localization of the extended-C/EBPαS299D mutant depends on ongoing rDNA transcription, we analysed its localization in the presence of a low concentration of actinomycin D (50 ng/ml), which specifically inhibits RNA Pol I transcription (Perry and Kelley, 1970). As shown in Figure 6 and Supplementary Figure S4B, nucleolar localization of extended-C/EBPαS299D was lost in the presence of actinomycin D, whereas inhibition of RNA Pol II driven transcription with α-amanitin had no effect on the localization of the extended isoform. Taken together, these results show that extended-C/EBPα interacts with functional nucleolar proteins, UBF-1 and NPM, and the SL1 components TBP and TAFI48, and that its nucleolar retention depends on ongoing rDNA transcription.

Extended-C/EBPα interacts with Pol I-specific factors and its nucleolar retention depends on ongoing rDNA transcription

The nucleolus is the place where rDNA transcription is performed through the activity of RNA Pol I. The nucleolar transcription factor UBF-1 is a crucial component of the RNA Pol I preinitiation complex (Grummt, 2003). Immunofluorescence microscopy revealed that ectopically expressed extended-C/EBPαS299D but not the corresponding ΔNoLS mutant localizes with endogenous UBF-1 in the nucleoli (Supplementary Figure S4A). To examine whether C/EBPα interacts with UBF-1, HEK293 cells were transfected with expression plasmids for extended-, full-length or truncated-C/EBPα, respectively, followed by co-immunoprecipitation (Co-IP) experiments. We found that both wt and the S299D mutant of extended-C/EBPα (Figure 5A) as well as full-length-C/EBPα (Figure 5B) can be co-precipitated together with endogenous UBF-1 using anti-UBF-1 antibodies. In contrast, we could not detect a clear co-precipitation of truncated-C/EBPα.

RNA Pol I transcription requires the SL1 complex, which is composed of the TBP and three Pol I-specific TAFs, TAFI110, TAFI63 and TAFI48. SL1 was shown to be essential for UBF-1 recruitment and pre-initiation complex formation (Friedrich et al, 2005). As full-length C/EBPα is known to bind to and recruit TBP to Pol II promoters (Nerlov and Ziff, 1995), we examined whether extended-C/EBPα interacts with TBP and TAFI48. Co-IP using anti-C/EBPα antibodies resulted in precipitation of TBP by extended-C/EBPα, as well as the S299D and the S299D-ΔNoLS mutant thereof (Figure 5C). Hence, interaction with TBP is a genuine property of both full-length and extended-C/EBPα. In addition, we could co-precipitate extended-C/EBPαS299D with anti-TAFI48 antibodies; however, co-precipitation was not seen with the ΔNoLS mutant (Figure 5D, upper panel). Furthermore, TAFI48 can be precipitated with anti-TBP antibodies more efficiently in cells that express extended-C/EBPα, in particular the S299D mutant, compared with ΔNoLS or empty vector controls (Figure 5D, lower panel). Overall these results indicate that extended-C/EBPα, UBF-1, and TBP and TAFI48 as part of the SL1 complex reside as a complex in the nucleolus.

It has been shown that the RRQR motif in Parp2 is required for nucleolar retention and for interaction with NPM (also known as B23) (Meder et al, 2005), a ubiquitously expressed phosphoprotein with multiple functions that constantly shuttles between the nucleolus, nucleoplasm and cytoplasm (Grisendi et al, 2006). To examine whether a similar interaction between NPM and C/EBPα exists, Co-IP experiments were set up and as presented in Figure 5E, NPM could only be co-immunoprecipitated with wt and the S299D mutant of extended-C/EBPα while co-imunoprecipitation failed with the ΔNoLS mutant or full-length isoform that both lack RRQR motif.

Typically, nucleolar retention of regulatory factors depends on ongoing rDNA transcription. To examine whether nucleolar localization of the extended-C/EBPαS299D mutant depends on ongoing rDNA transcription, we analysed its localization in the presence of a low concentration of actinomycin D (50 ng/ml), which specifically inhibits RNA Pol I transcription (Perry and Kelley, 1970). As shown in Figure 6 and Supplementary Figure S4B, nucleolar localization of extended-C/EBPαS299D was lost in the presence of actinomycin D, whereas inhibition of RNA Pol II driven transcription with α-amanitin had no effect on the localization of the extended isoform. Taken together, these results show that extended-C/EBPα interacts with functional nucleolar proteins, UBF-1 and NPM, and the SL1 components TBP and TAFI48, and that its nucleolar retention depends on ongoing rDNA transcription.

Extended-C/EBPα stimulates RNA Pol I-dependent rRNA synthesisAs C/EBPα is a transcription factor, we examined whether extended-C/EBPα was able to regulate RNA Pol I-mediated rRNA synthesis as has been shown also for other RNA Pol II-dependent transcription factors, including c-Myc (Arabi et al, 2005; Grandori et al, 2005). To address this question, we co-transfected a human rRNA minigene reporter construct (pMr1930-BH) (Mayer et al, 2005) together with expression vectors for extended-C/EBPα in HEK293T cells. We observed that overexpression of both extended-C/EBPα and the S299D mutant thereof strongly induced RNA Pol I-mediated transcription of the reporter RNA to a similar extent as c-Myc, which was used as a positive control (Figure 7A). To examine whether extended-C/EBPα is able to stimulate endogenous rDNA transcription, we analysed the expression levels of 45S pre-rRNA by Northern blotting in long-term C33A cell lines that stably express extended-, full-length- or truncated-C/EBPα wt proteins, the S299D mutants thereof and ΔNoLS mutants. Figure 7B shows that expression of extended-C/EBPα and extended-C/EBPαS299D results in increased levels of endogenous 45S pre-rRNA compared with the empty vector control. In contrast, expression of the full-length- or truncated-C/EBPα isoforms either with or without the S299D mutation was not able to enhance 45S pre-rRNA levels. The ΔNoLS mutation prevents induction of 45S pr-RNA (Figure 7C). To examine a possible indirect Pol II-dependent regulation of rDNA transcription by extended-C/EBPα, 45S pre-rRNA levels were analysed under inhibition of Pol II by α-amanitin (50 μg/ml). The results in Figure 7D show that inhibition of Pol II only slightly affected induction of 45S pre-rRNA, suggesting a direct regulation of Pol I by extended-C/EBPα. To examine whether selective elimination of extended-C/EBPα impedes rRNA synthesis, we knocked-down expression of endogenous C/EBPα in HL-60 cells by introduction of lentiviral human C/EBPα-specific shRNA (Figure 8A). Subsequently, in the C/EBPα knock-down cells, expression was restored from retrovirally transduced constructs coding for all three C/EBPα-FLAG isoforms, or from a construct with a ΔCUG mutation avoiding expression of the extended-C/EBPα isoform (Figure 8B). As demonstrated by Northern analysis, 45S pre-rRNA levels were only significantly stimulated in the presence of extended-C/EBPα and not when full-length- and truncated-C/EBPα are expressed alone from the ΔCUGα construct. Once more, the requirement of the N-terminal NoLS was demonstrated as stimulation of 45S pre-rRNA levels by overexpression of extended-C/EBPαS299D alone is abrogated when NoLS is mutated (Figure 8C). Taken together, extended-C/EBPα directly and specifically enhances rDNA transcription, which depends on nucleolar retention mediated by the N-terminally located NoLS.Bioinformatic analysis revealed two C/EBP-binding motifs in the intergenic spacer of the human rDNA repeats (Figure 9A). One of the C/EBP-binding motifs lies 25 bp upstream of the transcription start, in the same proximal promoter sequences that are crucial for Pol I regulation by c-Myc (Figure 9A; region Cp, same as region 42.9 in Grandori et al, 2005). Chromatin immunoprecipitation (ChIP) experiments using anti-C/EBPα antibodies revealed that extended-C/EBPα occupies this rDNA promoter segment (Figure 9B; upper chart, region Cp) in C33A cells. Notably, the more efficient nucleolar retention of extended-C/EBPαS299D results in recovery of more Cp DNA fragments whereas the ΔNoLS mutation abolishes DNA recovery (Figure 9B and C). In addition, extended-C/EBPα binds to a more distal region containing the second C/EBP-binding motif (Figure 9A and B; upper chart, region Cd). Extended-C/EBPα does not interact with region M that does not contain a C/EBP-binding motif, which, however, was shown to be implicated in c-Myc-dependent regulation (Figure 9A and B; region M, same as region 42 in Grandori et al, 2005).Region Cp covers the Pol I promoter and is also heavily occupied by UBF-1, as was also observed by others (Figure 9B, lower chart, region Cp) (Grandori et al, 2005; here region 42.9). Interestingly, the enhanced interaction of extended-C/EBPαS299D with the regions Cp and Cd improves recruitment of UBF-1 to these regions (Figure 9B and C). To investigate whether extended-C/EBPα also facilitates the recruitment of components of the SL1 complex to the rDNA promoter, we performed a ChIP assay using an anti-TBP antibody. Figure 9D shows that indeed recruitment of TBP is enhanced on extended-C/EBPα binding to region Cp. c-Myc like extended-C/EBPα binds most strongly to region Cp and stimulates histone H3 and H4 acetylation especially at the neighbouring region M (Grandori et al, 2005). C/EBPα is known to activate transcription from Pol II promoters through recruitment of histone acetylases (Erickson et al, 2001; Bararia et al, 2008). ChIP analysis using anti-histone H3 or H4 antibodies revealed that histone acetylation is enhanced particularly at site M and to a lesser extend to Cp when extended-C/EBPα binds to the rDNA promoter (Figure 9E). Hence, our results suggest that the occupation of the rDNA promoter region by extended-C/EBPα facilitates recruitment of UBF-1 and SL1, and induces histone acetylation.

Extended-C/EBPα stimulates RNA Pol I-dependent rRNA synthesis

As C/EBPα is a transcription factor, we examined whether extended-C/EBPα was able to regulate RNA Pol I-mediated rRNA synthesis as has been shown also for other RNA Pol II-dependent transcription factors, including c-Myc (Arabi et al, 2005; Grandori et al, 2005). To address this question, we co-transfected a human rRNA minigene reporter construct (pMr1930-BH) (Mayer et al, 2005) together with expression vectors for extended-C/EBPα in HEK293T cells. We observed that overexpression of both extended-C/EBPα and the S299D mutant thereof strongly induced RNA Pol I-mediated transcription of the reporter RNA to a similar extent as c-Myc, which was used as a positive control (Figure 7A). To examine whether extended-C/EBPα is able to stimulate endogenous rDNA transcription, we analysed the expression levels of 45S pre-rRNA by Northern blotting in long-term C33A cell lines that stably express extended-, full-length- or truncated-C/EBPα wt proteins, the S299D mutants thereof and ΔNoLS mutants. Figure 7B shows that expression of extended-C/EBPα and extended-C/EBPαS299D results in increased levels of endogenous 45S pre-rRNA compared with the empty vector control. In contrast, expression of the full-length- or truncated-C/EBPα isoforms either with or without the S299D mutation was not able to enhance 45S pre-rRNA levels. The ΔNoLS mutation prevents induction of 45S pr-RNA (Figure 7C). To examine a possible indirect Pol II-dependent regulation of rDNA transcription by extended-C/EBPα, 45S pre-rRNA levels were analysed under inhibition of Pol II by α-amanitin (50 μg/ml). The results in Figure 7D show that inhibition of Pol II only slightly affected induction of 45S pre-rRNA, suggesting a direct regulation of Pol I by extended-C/EBPα. To examine whether selective elimination of extended-C/EBPα impedes rRNA synthesis, we knocked-down expression of endogenous C/EBPα in HL-60 cells by introduction of lentiviral human C/EBPα-specific shRNA (Figure 8A). Subsequently, in the C/EBPα knock-down cells, expression was restored from retrovirally transduced constructs coding for all three C/EBPα-FLAG isoforms, or from a construct with a ΔCUG mutation avoiding expression of the extended-C/EBPα isoform (Figure 8B). As demonstrated by Northern analysis, 45S pre-rRNA levels were only significantly stimulated in the presence of extended-C/EBPα and not when full-length- and truncated-C/EBPα are expressed alone from the ΔCUGα construct. Once more, the requirement of the N-terminal NoLS was demonstrated as stimulation of 45S pre-rRNA levels by overexpression of extended-C/EBPαS299D alone is abrogated when NoLS is mutated (Figure 8C). Taken together, extended-C/EBPα directly and specifically enhances rDNA transcription, which depends on nucleolar retention mediated by the N-terminally located NoLS.

Bioinformatic analysis revealed two C/EBP-binding motifs in the intergenic spacer of the human rDNA repeats (Figure 9A). One of the C/EBP-binding motifs lies 25 bp upstream of the transcription start, in the same proximal promoter sequences that are crucial for Pol I regulation by c-Myc (Figure 9A; region Cp, same as region 42.9 in Grandori et al, 2005). Chromatin immunoprecipitation (ChIP) experiments using anti-C/EBPα antibodies revealed that extended-C/EBPα occupies this rDNA promoter segment (Figure 9B; upper chart, region Cp) in C33A cells. Notably, the more efficient nucleolar retention of extended-C/EBPαS299D results in recovery of more Cp DNA fragments whereas the ΔNoLS mutation abolishes DNA recovery (Figure 9B and C). In addition, extended-C/EBPα binds to a more distal region containing the second C/EBP-binding motif (Figure 9A and B; upper chart, region Cd). Extended-C/EBPα does not interact with region M that does not contain a C/EBP-binding motif, which, however, was shown to be implicated in c-Myc-dependent regulation (Figure 9A and B; region M, same as region 42 in Grandori et al, 2005).

Region Cp covers the Pol I promoter and is also heavily occupied by UBF-1, as was also observed by others (Figure 9B, lower chart, region Cp) (Grandori et al, 2005; here region 42.9). Interestingly, the enhanced interaction of extended-C/EBPαS299D with the regions Cp and Cd improves recruitment of UBF-1 to these regions (Figure 9B and C). To investigate whether extended-C/EBPα also facilitates the recruitment of components of the SL1 complex to the rDNA promoter, we performed a ChIP assay using an anti-TBP antibody. Figure 9D shows that indeed recruitment of TBP is enhanced on extended-C/EBPα binding to region Cp. c-Myc like extended-C/EBPα binds most strongly to region Cp and stimulates histone H3 and H4 acetylation especially at the neighbouring region M (Grandori et al, 2005). C/EBPα is known to activate transcription from Pol II promoters through recruitment of histone acetylases (Erickson et al, 2001; Bararia et al, 2008). ChIP analysis using anti-histone H3 or H4 antibodies revealed that histone acetylation is enhanced particularly at site M and to a lesser extend to Cp when extended-C/EBPα binds to the rDNA promoter (Figure 9E). Hence, our results suggest that the occupation of the rDNA promoter region by extended-C/EBPα facilitates recruitment of UBF-1 and SL1, and induces histone acetylation.

Overexpression of extended-C/EBPα results in an increase in cell sizeIncrease in cell size is a well-known consequence of enhanced ribosome biogenesis (Baserga, 2007), and the stimulation of rRNA synthesis by c-Myc has been associated with increased cell size in different systems (Oskarsson and Trumpp, 2005). Cell volume analysis of the stably transfected cells revealed that expression of extended-C/EBPαwt- and in particular the extended-C/EBPαS299D isoform results in a profound increase in the mean cell volume, whereas the other isoforms show no effect (Figure 10). C33A cells lack a functional Brahma protein as part of the SWI/SNF chromatin-remodelling complex, which is required for C/EBPα-induced proliferation arrest (Müller et al, 2004). To study whether the absence of Brahma is instrumental in the pronounced increase in cell volume, we analysed C33A cells in which Brahma function was restored by expression from a retroviral construct compared with dominant negative mutant of Brahma and empty vector. In none of the situations, however, cell size was affected (Supplementary Figure S5A). Hence, Brahma function has no effect on cell size. In addition, over-expression of extended-C/EBPαwt- and the extended-C/EBPαS299D isoform in HEK293 cells that have a functional SWI/SNF complex resulted in a clear although less pronounced increase in cell size (Supplementary Figure S5B).

Overexpression of extended-C/EBPα results in an increase in cell size

Increase in cell size is a well-known consequence of enhanced ribosome biogenesis (Baserga, 2007), and the stimulation of rRNA synthesis by c-Myc has been associated with increased cell size in different systems (Oskarsson and Trumpp, 2005). Cell volume analysis of the stably transfected cells revealed that expression of extended-C/EBPαwt- and in particular the extended-C/EBPαS299D isoform results in a profound increase in the mean cell volume, whereas the other isoforms show no effect (Figure 10). C33A cells lack a functional Brahma protein as part of the SWI/SNF chromatin-remodelling complex, which is required for C/EBPα-induced proliferation arrest (Müller et al, 2004). To study whether the absence of Brahma is instrumental in the pronounced increase in cell volume, we analysed C33A cells in which Brahma function was restored by expression from a retroviral construct compared with dominant negative mutant of Brahma and empty vector. In none of the situations, however, cell size was affected (Supplementary Figure S5A). Hence, Brahma function has no effect on cell size. In addition, over-expression of extended-C/EBPαwt- and the extended-C/EBPαS299D isoform in HEK293 cells that have a functional SWI/SNF complex resulted in a clear although less pronounced increase in cell size (Supplementary Figure S5B).

DiscussionIn this study, we reveal a novel function of the transcription factor C/EBPα that is specific for the translational protein isoform called extended-C/EBPα. Results presented in this manuscript provide evidence that extended-C/EBPα localizes in the nucleoli and stimulates RNA Pol I-dependent rDNA transcription, and concomitantly increases cell size.Regulation of Pol I activity and rRNA synthesis by C/EBPαWe show an increase in rRNA levels from endogenous loci and increased Pol I transcription from an rRNA minigene reporter in response to extended-C/EBPα expression. These findings are supported by ChIP assays that revealed interaction of extended-C/EBPα with rDNA promoter sequences. Furthermore, expression of extended-C/EBPα results in increased acetylation of histones H3 and H4 within the rDNA promoter, reflecting enhanced promoter activation through recruitment of acetyltransferases. Finally, our results suggest that extended-C/EBPα stimulates Pol I pre-initiation complex formation as extended-C/EBPα interacts with UBF-1 and components of the SL1 complex, and facilitates their recruitment to rDNA promoter sites. Remarkably, similar functions have also been described for c-Myc (Arabi et al, 2005; Grandori et al, 2005). Indeed, both proteins can bind to the proximal promoter region (region Cp), recruit the same set of factors and induce histone acetylation in the same region M. Other RNA Pol II transcription factors like Runx2, MyoD, Myogenin and C/EBPβ interact with rRNA genes, yet repress rRNA synthesis during differentiation (Grandori et al, 2005; Young et al, 2007; Ali et al, 2008). These inhibitory factors contact the rRNA promoter during mitosis when the nucleolar structures are dissolved (Young et al, 2007; Ali et al, 2008), making nucleolar retention signals dispensable. This stands in contrast to the activating function of extended-C/EBPα that requires the NoLS for rDNA promoter binding and stimulation of rRNA synthesis.We show that besides extended- also full-length-C/EBPα interacts with UBF-1. In contrast to the extended isoform, full-length-C/EBPα is not retained in the nucleoli and rather seems to inhibit rRNA synthesis (Figures 2A and 7B; data not shown). Interestingly, the Pol I inhibitory factors Runx2, MyoD and Myogenin also interact with UBF-1 (Young et al, 2007; Ali et al, 2008). Although the observed repression by full-length-C/EBPα could be caused indirectly by its anti-proliferative activities, or through sequestration of UBF-1 within the nucleoplasm, it may inhibit rRNA synthesis involving direct rDNA promoter interaction similarly to Runx2, MyoD or Myogenin. Such a direct interaction of full-length C/EBPα with the rDNA promoter is likely as we recovered promoter rDNA (region Cp) in ChIP analysis using both anti-UBF-1 as well as anti-C/EBPα antibodies from proliferating HL-60 cells expressing all translational C/EBPα isoforms as well as from differentiating HL-60 cells, which express only the full-length C/EBPα isoform (data not shown).Extended-C/EBPα expression results in an increased cell sizeOverexpression of extended-C/EBPα, in particular of the S299D mutant that shows augmented nucleolar retention and Pol I activation, results in a pronounced increase in cell volume. Obviously, cell growth (increase in cell size) requires an enhanced protein biosynthesis rate, which directly depends on ribosome biogenesis with rRNA synthesis as the rate-limiting process. Accordingly, other positive regulators of Pol1-dependent rRNA transcription like c-Myc (Oskarsson and Trumpp, 2005) and IRS-1 (Valentinis et al, 2000; Sun et al, 2003) are able to increase cell size in specific systems.Specific nucleolar retention of the extended-C/EBPα isoformExtended-C/EBPα is the only translational C/EBPα isoform that can be retained in the nucleoli. This can be explained by the presence of a conserved RRXR nucleolar localization (NoLS) motif in the N-terminal part of the protein that is also found in other nucleolar-retained factors (Kubota et al, 1989; Jarrous et al, 1999; Weber et al, 1999; Zhang and Xiong, 1999; Scott et al, 2001; Meder et al, 2005). For both Parp2 (Li, 1997; Meder et al, 2005) and extended-C/EBPα (this study), the RRXR motif is required for interaction with NPM (also known as B23), which could mean that either these factors need to be nucleolar to interact with NPM or that they are recruited to the nucleolus through binding to NPM. NPM is implicated in cancer pathogenesis as it is over-expressed in a diversity of solid tumours, and in several haematologic malignancies the NPM1 gene is found mutated or rearranged (Falini et al, 2007). Therefore, it would be interesting to examine whether cancerous mutations of NPM have an effect on functions of C/EBPα in the control of cell proliferation and growth.Apart from the RRXR motif, we observed that a phosphorylation-mimicking mutation of a Serine residue within the NLS (S299 as numbered for the full-length isoform) strongly enhances nucleolar retention of extended-C/EBPα. This observation is reminiscent to the contribution of NLSs of Parp2 or ING1b that contribute to the efficiency of nucleolar retention (Scott et al, 2001; Meder et al, 2005). Serine 299 lies in a region that is conserved between all C/EBPs and its phosphorylation has been associated with opposite effects on DNA binding and/or subcellular distribution (Mahoney et al, 1992; Yin et al, 1996; Buck et al, 2001). As we observed that in long-term cultures of cells that stably express the wild-type extended-C/EBPα isoform, the percentage of cells with nucleolar localization of extended-C/EBPα continuously increases, a physiological mechanism must exist that stimulates nucleolar retention and thereby provides a proliferation or survival advantage for the cells. To adequately address which signalling pathways could be involved, a more comprehensive experimental setup is required.Translation from non-AUG initiation codonsExpression of the extended translational isoform of C/EBPα depends on initiation from non-AUG codons, CUGs in murine and GUG in human C/EBPα transcripts, respectively (Figure 1). The molecular mechanisms of selection of alternative non-AUG codons for initiation are still largely unknown; however, a study by Schwab et al (2004) indicates that a subset of ribosomes is scanning specifically for CUG initiation codons. Several studies point to a physiological role of selection of non-AUG codons in transcripts of other regulatory proteins, including c-Myc (Hann et al, 1992; Kim et al, 2003), JunD (Short and Pfarr, 2002), FGF2 (Kevil et al, 1995) and VEGF (Huez et al, 2001) (reviewed in (Touriol et al, 2003)). The underlying mechanism of translational upregulation of the extended C/EBPα isoform is currently under investigation.Regulation of extended-C/EBPα expression and function in vivoThe lack of antibodies that specifically recognize extended-C/EBPα and the impossibility to specifically target the endogenous protein by knockdown strategies (all protein isoforms can be translated from a single mRNA molecule) makes it technically difficult to address its physiological functions and sites of action in depth. Nevertheless, we could detect C/EBPα-specific nucleolar localization and expression of extended-C/EBPα in the leukemia cell line HL-60. The concomitant loss of both, C/EBPα-specific nucleolar localization and extended isoform expression during HL-60 differentiation provides a good indication that extended-C/EBPα is responsible for the nucleolar staining in the undifferentiated cells. We presume that expression of extended-C/EBPα is likely to be timely and spatially restricted only to those few cells that proliferate in a tissue. Preliminary analysis of primary bone marrow cells revealed weak but consistent staining of C/EBPα in the nucleoli as well as detection of an extended isoform by immunoblotting (data not shown). To uncover extended-C/EBPα physiology in vivo in more detail, antibodies have to be generated that specifically recognize this isoform. In addition, a mouse model in which either expression of extended-C/EBPα is abrogated by mutation of its initiation codon by a knock-in approach, or in which extended-C/EBPα can be induced from a transgene would be instrumental.A hypothetical model of a C/EBPα-based translational switchDuring our study, we did not observe that overexpression of extended-C/EBPα stimulates cell proliferation in the cell lines tested. The truncated-C/EBPα isoform that is co-regulated with the extended isoform in HL-60 cells (Figure 4), however, drives proliferation in several cell types (Lin et al, 1993; Calkhoven et al, 2000). We would like to propose a hypothetical translational switch model explaining how controlled translation into extended- and truncated-C/EBPα contributes to coordination of cell growth and proliferation: the truncated isoform blocks the anti-proliferative activity of the full-length isoform in the nucleus; the extended isoform escapes from this inhibition by relocating to the nucleolus where it stimulates rRNA synthesis as prerequisite for cell growth and proliferation (Supplementary Figure S6).By revealing a function of the extended-C/EBPα isoform in ribosome biogenesis, a new and fascinating aspect of C/EBPα-mediated control of cell growth and proliferation emerges. Whether increased extended-C/EBPα expression or its nucleolar retention might be involved in cancer development has to be investigated.

In this study, we reveal a novel function of the transcription factor C/EBPα that is specific for the translational protein isoform called extended-C/EBPα. Results presented in this manuscript provide evidence that extended-C/EBPα localizes in the nucleoli and stimulates RNA Pol I-dependent rDNA transcription, and concomitantly increases cell size.

Regulation of Pol I activity and rRNA synthesis by C/EBPαWe show an increase in rRNA levels from endogenous loci and increased Pol I transcription from an rRNA minigene reporter in response to extended-C/EBPα expression. These findings are supported by ChIP assays that revealed interaction of extended-C/EBPα with rDNA promoter sequences. Furthermore, expression of extended-C/EBPα results in increased acetylation of histones H3 and H4 within the rDNA promoter, reflecting enhanced promoter activation through recruitment of acetyltransferases. Finally, our results suggest that extended-C/EBPα stimulates Pol I pre-initiation complex formation as extended-C/EBPα interacts with UBF-1 and components of the SL1 complex, and facilitates their recruitment to rDNA promoter sites. Remarkably, similar functions have also been described for c-Myc (Arabi et al, 2005; Grandori et al, 2005). Indeed, both proteins can bind to the proximal promoter region (region Cp), recruit the same set of factors and induce histone acetylation in the same region M. Other RNA Pol II transcription factors like Runx2, MyoD, Myogenin and C/EBPβ interact with rRNA genes, yet repress rRNA synthesis during differentiation (Grandori et al, 2005; Young et al, 2007; Ali et al, 2008). These inhibitory factors contact the rRNA promoter during mitosis when the nucleolar structures are dissolved (Young et al, 2007; Ali et al, 2008), making nucleolar retention signals dispensable. This stands in contrast to the activating function of extended-C/EBPα that requires the NoLS for rDNA promoter binding and stimulation of rRNA synthesis.We show that besides extended- also full-length-C/EBPα interacts with UBF-1. In contrast to the extended isoform, full-length-C/EBPα is not retained in the nucleoli and rather seems to inhibit rRNA synthesis (Figures 2A and 7B; data not shown). Interestingly, the Pol I inhibitory factors Runx2, MyoD and Myogenin also interact with UBF-1 (Young et al, 2007; Ali et al, 2008). Although the observed repression by full-length-C/EBPα could be caused indirectly by its anti-proliferative activities, or through sequestration of UBF-1 within the nucleoplasm, it may inhibit rRNA synthesis involving direct rDNA promoter interaction similarly to Runx2, MyoD or Myogenin. Such a direct interaction of full-length C/EBPα with the rDNA promoter is likely as we recovered promoter rDNA (region Cp) in ChIP analysis using both anti-UBF-1 as well as anti-C/EBPα antibodies from proliferating HL-60 cells expressing all translational C/EBPα isoforms as well as from differentiating HL-60 cells, which express only the full-length C/EBPα isoform (data not shown).

Regulation of Pol I activity and rRNA synthesis by C/EBPα

We show an increase in rRNA levels from endogenous loci and increased Pol I transcription from an rRNA minigene reporter in response to extended-C/EBPα expression. These findings are supported by ChIP assays that revealed interaction of extended-C/EBPα with rDNA promoter sequences. Furthermore, expression of extended-C/EBPα results in increased acetylation of histones H3 and H4 within the rDNA promoter, reflecting enhanced promoter activation through recruitment of acetyltransferases. Finally, our results suggest that extended-C/EBPα stimulates Pol I pre-initiation complex formation as extended-C/EBPα interacts with UBF-1 and components of the SL1 complex, and facilitates their recruitment to rDNA promoter sites. Remarkably, similar functions have also been described for c-Myc (Arabi et al, 2005; Grandori et al, 2005). Indeed, both proteins can bind to the proximal promoter region (region Cp), recruit the same set of factors and induce histone acetylation in the same region M. Other RNA Pol II transcription factors like Runx2, MyoD, Myogenin and C/EBPβ interact with rRNA genes, yet repress rRNA synthesis during differentiation (Grandori et al, 2005; Young et al, 2007; Ali et al, 2008). These inhibitory factors contact the rRNA promoter during mitosis when the nucleolar structures are dissolved (Young et al, 2007; Ali et al, 2008), making nucleolar retention signals dispensable. This stands in contrast to the activating function of extended-C/EBPα that requires the NoLS for rDNA promoter binding and stimulation of rRNA synthesis.

We show that besides extended- also full-length-C/EBPα interacts with UBF-1. In contrast to the extended isoform, full-length-C/EBPα is not retained in the nucleoli and rather seems to inhibit rRNA synthesis (Figures 2A and 7B; data not shown). Interestingly, the Pol I inhibitory factors Runx2, MyoD and Myogenin also interact with UBF-1 (Young et al, 2007; Ali et al, 2008). Although the observed repression by full-length-C/EBPα could be caused indirectly by its anti-proliferative activities, or through sequestration of UBF-1 within the nucleoplasm, it may inhibit rRNA synthesis involving direct rDNA promoter interaction similarly to Runx2, MyoD or Myogenin. Such a direct interaction of full-length C/EBPα with the rDNA promoter is likely as we recovered promoter rDNA (region Cp) in ChIP analysis using both anti-UBF-1 as well as anti-C/EBPα antibodies from proliferating HL-60 cells expressing all translational C/EBPα isoforms as well as from differentiating HL-60 cells, which express only the full-length C/EBPα isoform (data not shown).

Extended-C/EBPα expression results in an increased cell sizeOverexpression of extended-C/EBPα, in particular of the S299D mutant that shows augmented nucleolar retention and Pol I activation, results in a pronounced increase in cell volume. Obviously, cell growth (increase in cell size) requires an enhanced protein biosynthesis rate, which directly depends on ribosome biogenesis with rRNA synthesis as the rate-limiting process. Accordingly, other positive regulators of Pol1-dependent rRNA transcription like c-Myc (Oskarsson and Trumpp, 2005) and IRS-1 (Valentinis et al, 2000; Sun et al, 2003) are able to increase cell size in specific systems.

Extended-C/EBPα expression results in an increased cell size

Overexpression of extended-C/EBPα, in particular of the S299D mutant that shows augmented nucleolar retention and Pol I activation, results in a pronounced increase in cell volume. Obviously, cell growth (increase in cell size) requires an enhanced protein biosynthesis rate, which directly depends on ribosome biogenesis with rRNA synthesis as the rate-limiting process. Accordingly, other positive regulators of Pol1-dependent rRNA transcription like c-Myc (Oskarsson and Trumpp, 2005) and IRS-1 (Valentinis et al, 2000; Sun et al, 2003) are able to increase cell size in specific systems.

Specific nucleolar retention of the extended-C/EBPα isoformExtended-C/EBPα is the only translational C/EBPα isoform that can be retained in the nucleoli. This can be explained by the presence of a conserved RRXR nucleolar localization (NoLS) motif in the N-terminal part of the protein that is also found in other nucleolar-retained factors (Kubota et al, 1989; Jarrous et al, 1999; Weber et al, 1999; Zhang and Xiong, 1999; Scott et al, 2001; Meder et al, 2005). For both Parp2 (Li, 1997; Meder et al, 2005) and extended-C/EBPα (this study), the RRXR motif is required for interaction with NPM (also known as B23), which could mean that either these factors need to be nucleolar to interact with NPM or that they are recruited to the nucleolus through binding to NPM. NPM is implicated in cancer pathogenesis as it is over-expressed in a diversity of solid tumours, and in several haematologic malignancies the NPM1 gene is found mutated or rearranged (Falini et al, 2007). Therefore, it would be interesting to examine whether cancerous mutations of NPM have an effect on functions of C/EBPα in the control of cell proliferation and growth.Apart from the RRXR motif, we observed that a phosphorylation-mimicking mutation of a Serine residue within the NLS (S299 as numbered for the full-length isoform) strongly enhances nucleolar retention of extended-C/EBPα. This observation is reminiscent to the contribution of NLSs of Parp2 or ING1b that contribute to the efficiency of nucleolar retention (Scott et al, 2001; Meder et al, 2005). Serine 299 lies in a region that is conserved between all C/EBPs and its phosphorylation has been associated with opposite effects on DNA binding and/or subcellular distribution (Mahoney et al, 1992; Yin et al, 1996; Buck et al, 2001). As we observed that in long-term cultures of cells that stably express the wild-type extended-C/EBPα isoform, the percentage of cells with nucleolar localization of extended-C/EBPα continuously increases, a physiological mechanism must exist that stimulates nucleolar retention and thereby provides a proliferation or survival advantage for the cells. To adequately address which signalling pathways could be involved, a more comprehensive experimental setup is required.

Specific nucleolar retention of the extended-C/EBPα isoform

Extended-C/EBPα is the only translational C/EBPα isoform that can be retained in the nucleoli. This can be explained by the presence of a conserved RRXR nucleolar localization (NoLS) motif in the N-terminal part of the protein that is also found in other nucleolar-retained factors (Kubota et al, 1989; Jarrous et al, 1999; Weber et al, 1999; Zhang and Xiong, 1999; Scott et al, 2001; Meder et al, 2005). For both Parp2 (Li, 1997; Meder et al, 2005) and extended-C/EBPα (this study), the RRXR motif is required for interaction with NPM (also known as B23), which could mean that either these factors need to be nucleolar to interact with NPM or that they are recruited to the nucleolus through binding to NPM. NPM is implicated in cancer pathogenesis as it is over-expressed in a diversity of solid tumours, and in several haematologic malignancies the NPM1 gene is found mutated or rearranged (Falini et al, 2007). Therefore, it would be interesting to examine whether cancerous mutations of NPM have an effect on functions of C/EBPα in the control of cell proliferation and growth.

Apart from the RRXR motif, we observed that a phosphorylation-mimicking mutation of a Serine residue within the NLS (S299 as numbered for the full-length isoform) strongly enhances nucleolar retention of extended-C/EBPα. This observation is reminiscent to the contribution of NLSs of Parp2 or ING1b that contribute to the efficiency of nucleolar retention (Scott et al, 2001; Meder et al, 2005). Serine 299 lies in a region that is conserved between all C/EBPs and its phosphorylation has been associated with opposite effects on DNA binding and/or subcellular distribution (Mahoney et al, 1992; Yin et al, 1996; Buck et al, 2001). As we observed that in long-term cultures of cells that stably express the wild-type extended-C/EBPα isoform, the percentage of cells with nucleolar localization of extended-C/EBPα continuously increases, a physiological mechanism must exist that stimulates nucleolar retention and thereby provides a proliferation or survival advantage for the cells. To adequately address which signalling pathways could be involved, a more comprehensive experimental setup is required.

Translation from non-AUG initiation codonsExpression of the extended translational isoform of C/EBPα depends on initiation from non-AUG codons, CUGs in murine and GUG in human C/EBPα transcripts, respectively (Figure 1). The molecular mechanisms of selection of alternative non-AUG codons for initiation are still largely unknown; however, a study by Schwab et al (2004) indicates that a subset of ribosomes is scanning specifically for CUG initiation codons. Several studies point to a physiological role of selection of non-AUG codons in transcripts of other regulatory proteins, including c-Myc (Hann et al, 1992; Kim et al, 2003), JunD (Short and Pfarr, 2002), FGF2 (Kevil et al, 1995) and VEGF (Huez et al, 2001) (reviewed in (Touriol et al, 2003)). The underlying mechanism of translational upregulation of the extended C/EBPα isoform is currently under investigation.

Translation from non-AUG initiation codons

Expression of the extended translational isoform of C/EBPα depends on initiation from non-AUG codons, CUGs in murine and GUG in human C/EBPα transcripts, respectively (Figure 1). The molecular mechanisms of selection of alternative non-AUG codons for initiation are still largely unknown; however, a study by Schwab et al (2004) indicates that a subset of ribosomes is scanning specifically for CUG initiation codons. Several studies point to a physiological role of selection of non-AUG codons in transcripts of other regulatory proteins, including c-Myc (Hann et al, 1992; Kim et al, 2003), JunD (Short and Pfarr, 2002), FGF2 (Kevil et al, 1995) and VEGF (Huez et al, 2001) (reviewed in (Touriol et al, 2003)). The underlying mechanism of translational upregulation of the extended C/EBPα isoform is currently under investigation.

Regulation of extended-C/EBPα expression and function in vivoThe lack of antibodies that specifically recognize extended-C/EBPα and the impossibility to specifically target the endogenous protein by knockdown strategies (all protein isoforms can be translated from a single mRNA molecule) makes it technically difficult to address its physiological functions and sites of action in depth. Nevertheless, we could detect C/EBPα-specific nucleolar localization and expression of extended-C/EBPα in the leukemia cell line HL-60. The concomitant loss of both, C/EBPα-specific nucleolar localization and extended isoform expression during HL-60 differentiation provides a good indication that extended-C/EBPα is responsible for the nucleolar staining in the undifferentiated cells. We presume that expression of extended-C/EBPα is likely to be timely and spatially restricted only to those few cells that proliferate in a tissue. Preliminary analysis of primary bone marrow cells revealed weak but consistent staining of C/EBPα in the nucleoli as well as detection of an extended isoform by immunoblotting (data not shown). To uncover extended-C/EBPα physiology in vivo in more detail, antibodies have to be generated that specifically recognize this isoform. In addition, a mouse model in which either expression of extended-C/EBPα is abrogated by mutation of its initiation codon by a knock-in approach, or in which extended-C/EBPα can be induced from a transgene would be instrumental.

Regulation of extended-C/EBPα expression and function in vivo

The lack of antibodies that specifically recognize extended-C/EBPα and the impossibility to specifically target the endogenous protein by knockdown strategies (all protein isoforms can be translated from a single mRNA molecule) makes it technically difficult to address its physiological functions and sites of action in depth. Nevertheless, we could detect C/EBPα-specific nucleolar localization and expression of extended-C/EBPα in the leukemia cell line HL-60. The concomitant loss of both, C/EBPα-specific nucleolar localization and extended isoform expression during HL-60 differentiation provides a good indication that extended-C/EBPα is responsible for the nucleolar staining in the undifferentiated cells. We presume that expression of extended-C/EBPα is likely to be timely and spatially restricted only to those few cells that proliferate in a tissue. Preliminary analysis of primary bone marrow cells revealed weak but consistent staining of C/EBPα in the nucleoli as well as detection of an extended isoform by immunoblotting (data not shown). To uncover extended-C/EBPα physiology in vivo in more detail, antibodies have to be generated that specifically recognize this isoform. In addition, a mouse model in which either expression of extended-C/EBPα is abrogated by mutation of its initiation codon by a knock-in approach, or in which extended-C/EBPα can be induced from a transgene would be instrumental.

A hypothetical model of a C/EBPα-based translational switchDuring our study, we did not observe that overexpression of extended-C/EBPα stimulates cell proliferation in the cell lines tested. The truncated-C/EBPα isoform that is co-regulated with the extended isoform in HL-60 cells (Figure 4), however, drives proliferation in several cell types (Lin et al, 1993; Calkhoven et al, 2000). We would like to propose a hypothetical translational switch model explaining how controlled translation into extended- and truncated-C/EBPα contributes to coordination of cell growth and proliferation: the truncated isoform blocks the anti-proliferative activity of the full-length isoform in the nucleus; the extended isoform escapes from this inhibition by relocating to the nucleolus where it stimulates rRNA synthesis as prerequisite for cell growth and proliferation (Supplementary Figure S6).By revealing a function of the extended-C/EBPα isoform in ribosome biogenesis, a new and fascinating aspect of C/EBPα-mediated control of cell growth and proliferation emerges. Whether increased extended-C/EBPα expression or its nucleolar retention might be involved in cancer development has to be investigated.

A hypothetical model of a C/EBPα-based translational switch

During our study, we did not observe that overexpression of extended-C/EBPα stimulates cell proliferation in the cell lines tested. The truncated-C/EBPα isoform that is co-regulated with the extended isoform in HL-60 cells (Figure 4), however, drives proliferation in several cell types (Lin et al, 1993; Calkhoven et al, 2000). We would like to propose a hypothetical translational switch model explaining how controlled translation into extended- and truncated-C/EBPα contributes to coordination of cell growth and proliferation: the truncated isoform blocks the anti-proliferative activity of the full-length isoform in the nucleus; the extended isoform escapes from this inhibition by relocating to the nucleolus where it stimulates rRNA synthesis as prerequisite for cell growth and proliferation (Supplementary Figure S6).

By revealing a function of the extended-C/EBPα isoform in ribosome biogenesis, a new and fascinating aspect of C/EBPα-mediated control of cell growth and proliferation emerges. Whether increased extended-C/EBPα expression or its nucleolar retention might be involved in cancer development has to be investigated.

Materials and methodsDNA constructsThe pSG5- and pcDNA3-based rC/EBPα wt and mutant constructs have been described earlier (Calkhoven et al, 2000). The ΔNoLS, S299D and S299A mutations were introduced by site-directed mutagenesis (Kunkel et al, 1991). Mutations of the hC/EBPα-cDNA-pcDNA3 were generated by PCR: GTG into ATG (Aopt) or CCC (ΔA1) and removal of the 5′UTR (Δ5′). All EGFP constructs were generated using EGFP sequences from the pEGFP-N3 plasmid (BD Clontech). The extended-C/EBPα domain and the corresponding ΔNoLS–EGFP-constructs were generated by PCR. The CMV-Myc vector was kindly provided by Martin Eilers (University of Würzburg). The pHrP2-BH, pTβuc- and pGem3HR plasmids (Mayer et al, 2005) were kindly provided by Ingrid Grummt (DKFZ, Heidelberg).Cell culture, transfection, immunofluorescence and cell volume analysisHL-60 and U937 cells were propagated in RPMI, all other cells in DMEM, plus 10% FCS (Invitrogen) at 5% CO2 and 37°C, and G-418 (0.8 mg/ml) whenever required. HL-60 differentiation was induced by 200 nM phorbol-12-myristate-13-acetate (TPA) for 12 h. Cells were transfected in 5-cm dishes with 5 μg plasmid using FUGENE (Roche) for MEFs, DEAE-dextran for COS-1 (Calkhoven et al, 2000) or calcium phosphate. Treatments with actinomycin D (50 ng/ml; Sigma) or α-amanitin (50 μg/ml; Sigma) were done for 2 h before fixation. For immunostaining, cells were washed twice with PBS, fixed with 3.7% paraformaldehyde for 10 min at 37°C, washed three times with PBS for 5 min at RT, permeabilized with 0.3% triton X-100 for 5 min at RT, blocked in 3% BSA in PBS and incubated with primary antibody (anti-C/EBPα 14AA, C18, N19 or anti-UBF-1 F-9: 1 μg/ml, Santa Cruz Biotechnology Inc.); anti-C/EBPα MA1-825 (1 μg/ml, ABR Affinity BioReagents); anti-Fibrillarin AFB01 (2.5 μg/ml, Cytoskeleton Inc.) for 45 min at RT. After triple washing for 10 min with PBS, cells were incubated with Alexa Fluor 488 or 568 conjugated secondary antibodies (Invitrogen) for 45 min at RT, then triple washed again and incubated with DAPI (1 mg/ml MeOH) for 15 min at RT. After triple washing for 10 min with PBS and single rinsing with H2O at RT, coverslips were mounted with Moviol (Sigma) onto glass slides. Microscopic analysis was performed with the ApoTome deconvolution system (Zeiss). Cell volume in femtoliter was determined using the CASY Model TTC (Innovatis).Lentiviral and retroviral transductionHL-60 cells were infected with pLKO.1-puro retroviral constructs containing shRNAs against human C/EBPα (5′-CCG GGC TGG AGC TGA CCA GTG ACA ACT CGA GTT GTC ACT GGT CAG CTC CAG CTT TTT -3′) or non-target shRNA control (Sigma-Aldrich) and propagated under puromycin selection (1.5 μg/ml). Subsequently, HL-60 cells containing the human C/EBPα-shRNA were retrovirally transduced using pMSCV-neo-based vectors containing either wt rat C/EBPα-FLAG cDNA, cDNA containing the ΔCUG mutation, extended-C/EBPαS299D-FLAG, extended-C/EBPαS299D-ΔNoLS-FLAG or empty vector control, and propagated on G418 (1.8 mg/ml) in addition.Co-immunoprecipitationHek293T cells (7 × 105) were transfected with 5 μg of expression vectors as indicated. Cells were lysed in 50 mM Tris/HCl pH 7.4, 300 mM NaCl, 5 mM EDTA, 1% Triton X-100 supplemented with 1 mM PMSF and protease inhibitor cocktail (Roche). Lysates were pre-cleared for 30 min with Protein G-Sepharose beads (Amersham) and incubated for 4 h with 1 μg of primary antibody (anti-UBF-1 (F-9), anti-C/EBPα (14AA), anti-TBP (N-12), anti-TAFI48 (M19) or normal mouse, rabbit or goat IgG, respectively, as negative control (all Santa Cruz Biotechnology), which had been coupled to Protein G-Sepharose beads before (1 h incubation in 1 × PBS). Beads were then washed five times for 5 min with 50 mM Tris/HCl pH 7.4, 300 mM NaCl, 5 mM EDTA 0.1% TritonX-100 supplemented with 1 mM PMSF and protease inhibitor cocktail and once with 1 × PBS. Proteins were eluted by boiling the beads in SDS sample buffer and analysed by western blotting.Western blottingWestern blotting was performed as described earlier (Calkhoven et al, 2000) with the following antibodies: anti-C/EBPα antibodies (14AA, C-18), anti-Myc antibody (9E10), anti-UBF-1 (F-9), antibody anti-TBP (N-12), anti-TAFI48 (M19), anti-α-Tubulin (TU-02) all from Santa Cruz Biotechnology, anti-NPM (clone FC-61991) from Zymed Laboratories), HRP-conjugated seconary antibodies (Amersham Life Technologies or Santa Cruz Biotechnology). Bands were visualized by chemiluminescence (ECL, Amersham Life Technologies).Chromatin immunoprecipitationC33A cells expressing extended-C/EBPα, extended-C/EBPαS299D, extended-C/EBPαS299DΔNoLS or control cells containing the empty pcDNA3 vector were cultured in DMEM with 10% FBS. ChIP assay was performed with ∼5 × 106 cells essentially as described in Puig et al (2003) using a Bioruptor (Diagenode, Inc.) for sonication (details on request). ChIP antibodies were against C/EBPα (14AA), UBF-1 (F-9), TBP (N-12) and non-specific rabbit IgG from Santa Cruz Biotechnology, acetylated histone H3 (#06-599) and acetylated histone H4 (#06-866) from Upstate Biotechnologies. The fold enrichment was calculated relative to the background detected with non-specific rabbit IgG. For the semi-quantitative PCR, 1/50 (1 μl) of DNA obtained from the ChIP assay was used as template in a PCR reaction with 28 cycles. Primer pairs were for Cp (89 bp) 5′-ccc ggg gga ggt ata tct tt-3′ and 5′-cca acc tct ccg acg aca-3′, for Cd (114 bp) 5′-aaa tag gac gga gaa cgt agc-3′ and 5′-gtt aca ttg ccc gaa aga tgg-3′ for M (155 bp) 5′-aga ggg gct gcg ttt tcg gcc-3′ and 5′-cga gac aga tcc ggc tgg cag-3′.rRNA minigene reporter assay and Northern blottingThe rRNA minigene reporter assay was performed as described in Mayer et al (2005) using transfection of Hek293T cells with 8 μg reporter plasmid (pHrP2-BH) and 1 or 2.5 μg of pcDNA3-based C/EBPα expression vectors, supplemented with empty pcDNA3 plasmid to a total amount of 10.5 μg. For analysis by Northern blotting, 10 μg of total RNA was isolated 40 h later using the guanidine isothiocyanate method. For detection of the 45S rRNA transcript, cells were serum starved for 24 h, replenished with serum for 4 h in the absence or presence of 50 μg α-amanitin following total RNA isolation. A measure of 30 μg of total RNA were analysed by Northern blotting using a probe from the pGem3HR plasmid (Mayer et al, 2005) and Northern with a β-actin probe or ethidium bromide staining was used as loading control.

Materials and methods

DNA constructsThe pSG5- and pcDNA3-based rC/EBPα wt and mutant constructs have been described earlier (Calkhoven et al, 2000). The ΔNoLS, S299D and S299A mutations were introduced by site-directed mutagenesis (Kunkel et al, 1991). Mutations of the hC/EBPα-cDNA-pcDNA3 were generated by PCR: GTG into ATG (Aopt) or CCC (ΔA1) and removal of the 5′UTR (Δ5′). All EGFP constructs were generated using EGFP sequences from the pEGFP-N3 plasmid (BD Clontech). The extended-C/EBPα domain and the corresponding ΔNoLS–EGFP-constructs were generated by PCR. The CMV-Myc vector was kindly provided by Martin Eilers (University of Würzburg). The pHrP2-BH, pTβuc- and pGem3HR plasmids (Mayer et al, 2005) were kindly provided by Ingrid Grummt (DKFZ, Heidelberg).

DNA constructs

The pSG5- and pcDNA3-based rC/EBPα wt and mutant constructs have been described earlier (Calkhoven et al, 2000). The ΔNoLS, S299D and S299A mutations were introduced by site-directed mutagenesis (Kunkel et al, 1991). Mutations of the hC/EBPα-cDNA-pcDNA3 were generated by PCR: GTG into ATG (Aopt) or CCC (ΔA1) and removal of the 5′UTR (Δ5′). All EGFP constructs were generated using EGFP sequences from the pEGFP-N3 plasmid (BD Clontech). The extended-C/EBPα domain and the corresponding ΔNoLS–EGFP-constructs were generated by PCR. The CMV-Myc vector was kindly provided by Martin Eilers (University of Würzburg). The pHrP2-BH, pTβuc- and pGem3HR plasmids (Mayer et al, 2005) were kindly provided by Ingrid Grummt (DKFZ, Heidelberg).

Cell culture, transfection, immunofluorescence and cell volume analysisHL-60 and U937 cells were propagated in RPMI, all other cells in DMEM, plus 10% FCS (Invitrogen) at 5% CO2 and 37°C, and G-418 (0.8 mg/ml) whenever required. HL-60 differentiation was induced by 200 nM phorbol-12-myristate-13-acetate (TPA) for 12 h. Cells were transfected in 5-cm dishes with 5 μg plasmid using FUGENE (Roche) for MEFs, DEAE-dextran for COS-1 (Calkhoven et al, 2000) or calcium phosphate. Treatments with actinomycin D (50 ng/ml; Sigma) or α-amanitin (50 μg/ml; Sigma) were done for 2 h before fixation. For immunostaining, cells were washed twice with PBS, fixed with 3.7% paraformaldehyde for 10 min at 37°C, washed three times with PBS for 5 min at RT, permeabilized with 0.3% triton X-100 for 5 min at RT, blocked in 3% BSA in PBS and incubated with primary antibody (anti-C/EBPα 14AA, C18, N19 or anti-UBF-1 F-9: 1 μg/ml, Santa Cruz Biotechnology Inc.); anti-C/EBPα MA1-825 (1 μg/ml, ABR Affinity BioReagents); anti-Fibrillarin AFB01 (2.5 μg/ml, Cytoskeleton Inc.) for 45 min at RT. After triple washing for 10 min with PBS, cells were incubated with Alexa Fluor 488 or 568 conjugated secondary antibodies (Invitrogen) for 45 min at RT, then triple washed again and incubated with DAPI (1 mg/ml MeOH) for 15 min at RT. After triple washing for 10 min with PBS and single rinsing with H2O at RT, coverslips were mounted with Moviol (Sigma) onto glass slides. Microscopic analysis was performed with the ApoTome deconvolution system (Zeiss). Cell volume in femtoliter was determined using the CASY Model TTC (Innovatis).

Cell culture, transfection, immunofluorescence and cell volume analysis

HL-60 and U937 cells were propagated in RPMI, all other cells in DMEM, plus 10% FCS (Invitrogen) at 5% CO2 and 37°C, and G-418 (0.8 mg/ml) whenever required. HL-60 differentiation was induced by 200 nM phorbol-12-myristate-13-acetate (TPA) for 12 h. Cells were transfected in 5-cm dishes with 5 μg plasmid using FUGENE (Roche) for MEFs, DEAE-dextran for COS-1 (Calkhoven et al, 2000) or calcium phosphate. Treatments with actinomycin D (50 ng/ml; Sigma) or α-amanitin (50 μg/ml; Sigma) were done for 2 h before fixation. For immunostaining, cells were washed twice with PBS, fixed with 3.7% paraformaldehyde for 10 min at 37°C, washed three times with PBS for 5 min at RT, permeabilized with 0.3% triton X-100 for 5 min at RT, blocked in 3% BSA in PBS and incubated with primary antibody (anti-C/EBPα 14AA, C18, N19 or anti-UBF-1 F-9: 1 μg/ml, Santa Cruz Biotechnology Inc.); anti-C/EBPα MA1-825 (1 μg/ml, ABR Affinity BioReagents); anti-Fibrillarin AFB01 (2.5 μg/ml, Cytoskeleton Inc.) for 45 min at RT. After triple washing for 10 min with PBS, cells were incubated with Alexa Fluor 488 or 568 conjugated secondary antibodies (Invitrogen) for 45 min at RT, then triple washed again and incubated with DAPI (1 mg/ml MeOH) for 15 min at RT. After triple washing for 10 min with PBS and single rinsing with H2O at RT, coverslips were mounted with Moviol (Sigma) onto glass slides. Microscopic analysis was performed with the ApoTome deconvolution system (Zeiss). Cell volume in femtoliter was determined using the CASY Model TTC (Innovatis).

Lentiviral and retroviral transductionHL-60 cells were infected with pLKO.1-puro retroviral constructs containing shRNAs against human C/EBPα (5′-CCG GGC TGG AGC TGA CCA GTG ACA ACT CGA GTT GTC ACT GGT CAG CTC CAG CTT TTT -3′) or non-target shRNA control (Sigma-Aldrich) and propagated under puromycin selection (1.5 μg/ml). Subsequently, HL-60 cells containing the human C/EBPα-shRNA were retrovirally transduced using pMSCV-neo-based vectors containing either wt rat C/EBPα-FLAG cDNA, cDNA containing the ΔCUG mutation, extended-C/EBPαS299D-FLAG, extended-C/EBPαS299D-ΔNoLS-FLAG or empty vector control, and propagated on G418 (1.8 mg/ml) in addition.

Lentiviral and retroviral transduction

HL-60 cells were infected with pLKO.1-puro retroviral constructs containing shRNAs against human C/EBPα (5′-CCG GGC TGG AGC TGA CCA GTG ACA ACT CGA GTT GTC ACT GGT CAG CTC CAG CTT TTT -3′) or non-target shRNA control (Sigma-Aldrich) and propagated under puromycin selection (1.5 μg/ml). Subsequently, HL-60 cells containing the human C/EBPα-shRNA were retrovirally transduced using pMSCV-neo-based vectors containing either wt rat C/EBPα-FLAG cDNA, cDNA containing the ΔCUG mutation, extended-C/EBPαS299D-FLAG, extended-C/EBPαS299D-ΔNoLS-FLAG or empty vector control, and propagated on G418 (1.8 mg/ml) in addition.

Co-immunoprecipitationHek293T cells (7 × 105) were transfected with 5 μg of expression vectors as indicated. Cells were lysed in 50 mM Tris/HCl pH 7.4, 300 mM NaCl, 5 mM EDTA, 1% Triton X-100 supplemented with 1 mM PMSF and protease inhibitor cocktail (Roche). Lysates were pre-cleared for 30 min with Protein G-Sepharose beads (Amersham) and incubated for 4 h with 1 μg of primary antibody (anti-UBF-1 (F-9), anti-C/EBPα (14AA), anti-TBP (N-12), anti-TAFI48 (M19) or normal mouse, rabbit or goat IgG, respectively, as negative control (all Santa Cruz Biotechnology), which had been coupled to Protein G-Sepharose beads before (1 h incubation in 1 × PBS). Beads were then washed five times for 5 min with 50 mM Tris/HCl pH 7.4, 300 mM NaCl, 5 mM EDTA 0.1% TritonX-100 supplemented with 1 mM PMSF and protease inhibitor cocktail and once with 1 × PBS. Proteins were eluted by boiling the beads in SDS sample buffer and analysed by western blotting.

Co-immunoprecipitation

Hek293T cells (7 × 105) were transfected with 5 μg of expression vectors as indicated. Cells were lysed in 50 mM Tris/HCl pH 7.4, 300 mM NaCl, 5 mM EDTA, 1% Triton X-100 supplemented with 1 mM PMSF and protease inhibitor cocktail (Roche). Lysates were pre-cleared for 30 min with Protein G-Sepharose beads (Amersham) and incubated for 4 h with 1 μg of primary antibody (anti-UBF-1 (F-9), anti-C/EBPα (14AA), anti-TBP (N-12), anti-TAFI48 (M19) or normal mouse, rabbit or goat IgG, respectively, as negative control (all Santa Cruz Biotechnology), which had been coupled to Protein G-Sepharose beads before (1 h incubation in 1 × PBS). Beads were then washed five times for 5 min with 50 mM Tris/HCl pH 7.4, 300 mM NaCl, 5 mM EDTA 0.1% TritonX-100 supplemented with 1 mM PMSF and protease inhibitor cocktail and once with 1 × PBS. Proteins were eluted by boiling the beads in SDS sample buffer and analysed by western blotting.

Western blottingWestern blotting was performed as described earlier (Calkhoven et al, 2000) with the following antibodies: anti-C/EBPα antibodies (14AA, C-18), anti-Myc antibody (9E10), anti-UBF-1 (F-9), antibody anti-TBP (N-12), anti-TAFI48 (M19), anti-α-Tubulin (TU-02) all from Santa Cruz Biotechnology, anti-NPM (clone FC-61991) from Zymed Laboratories), HRP-conjugated seconary antibodies (Amersham Life Technologies or Santa Cruz Biotechnology). Bands were visualized by chemiluminescence (ECL, Amersham Life Technologies).

Western blotting

Western blotting was performed as described earlier (Calkhoven et al, 2000) with the following antibodies: anti-C/EBPα antibodies (14AA, C-18), anti-Myc antibody (9E10), anti-UBF-1 (F-9), antibody anti-TBP (N-12), anti-TAFI48 (M19), anti-α-Tubulin (TU-02) all from Santa Cruz Biotechnology, anti-NPM (clone FC-61991) from Zymed Laboratories), HRP-conjugated seconary antibodies (Amersham Life Technologies or Santa Cruz Biotechnology). Bands were visualized by chemiluminescence (ECL, Amersham Life Technologies).

Chromatin immunoprecipitationC33A cells expressing extended-C/EBPα, extended-C/EBPαS299D, extended-C/EBPαS299DΔNoLS or control cells containing the empty pcDNA3 vector were cultured in DMEM with 10% FBS. ChIP assay was performed with ∼5 × 106 cells essentially as described in Puig et al (2003) using a Bioruptor (Diagenode, Inc.) for sonication (details on request). ChIP antibodies were against C/EBPα (14AA), UBF-1 (F-9), TBP (N-12) and non-specific rabbit IgG from Santa Cruz Biotechnology, acetylated histone H3 (#06-599) and acetylated histone H4 (#06-866) from Upstate Biotechnologies. The fold enrichment was calculated relative to the background detected with non-specific rabbit IgG. For the semi-quantitative PCR, 1/50 (1 μl) of DNA obtained from the ChIP assay was used as template in a PCR reaction with 28 cycles. Primer pairs were for Cp (89 bp) 5′-ccc ggg gga ggt ata tct tt-3′ and 5′-cca acc tct ccg acg aca-3′, for Cd (114 bp) 5′-aaa tag gac gga gaa cgt agc-3′ and 5′-gtt aca ttg ccc gaa aga tgg-3′ for M (155 bp) 5′-aga ggg gct gcg ttt tcg gcc-3′ and 5′-cga gac aga tcc ggc tgg cag-3′.

Chromatin immunoprecipitation

C33A cells expressing extended-C/EBPα, extended-C/EBPαS299D, extended-C/EBPαS299DΔNoLS or control cells containing the empty pcDNA3 vector were cultured in DMEM with 10% FBS. ChIP assay was performed with ∼5 × 106 cells essentially as described in Puig et al (2003) using a Bioruptor (Diagenode, Inc.) for sonication (details on request). ChIP antibodies were against C/EBPα (14AA), UBF-1 (F-9), TBP (N-12) and non-specific rabbit IgG from Santa Cruz Biotechnology, acetylated histone H3 (#06-599) and acetylated histone H4 (#06-866) from Upstate Biotechnologies. The fold enrichment was calculated relative to the background detected with non-specific rabbit IgG. For the semi-quantitative PCR, 1/50 (1 μl) of DNA obtained from the ChIP assay was used as template in a PCR reaction with 28 cycles. Primer pairs were for Cp (89 bp) 5′-ccc ggg gga ggt ata tct tt-3′ and 5′-cca acc tct ccg acg aca-3′, for Cd (114 bp) 5′-aaa tag gac gga gaa cgt agc-3′ and 5′-gtt aca ttg ccc gaa aga tgg-3′ for M (155 bp) 5′-aga ggg gct gcg ttt tcg gcc-3′ and 5′-cga gac aga tcc ggc tgg cag-3′.

rRNA minigene reporter assay and Northern blottingThe rRNA minigene reporter assay was performed as described in Mayer et al (2005) using transfection of Hek293T cells with 8 μg reporter plasmid (pHrP2-BH) and 1 or 2.5 μg of pcDNA3-based C/EBPα expression vectors, supplemented with empty pcDNA3 plasmid to a total amount of 10.5 μg. For analysis by Northern blotting, 10 μg of total RNA was isolated 40 h later using the guanidine isothiocyanate method. For detection of the 45S rRNA transcript, cells were serum starved for 24 h, replenished with serum for 4 h in the absence or presence of 50 μg α-amanitin following total RNA isolation. A measure of 30 μg of total RNA were analysed by Northern blotting using a probe from the pGem3HR plasmid (Mayer et al, 2005) and Northern with a β-actin probe or ethidium bromide staining was used as loading control.

rRNA minigene reporter assay and Northern blotting

The rRNA minigene reporter assay was performed as described in Mayer et al (2005) using transfection of Hek293T cells with 8 μg reporter plasmid (pHrP2-BH) and 1 or 2.5 μg of pcDNA3-based C/EBPα expression vectors, supplemented with empty pcDNA3 plasmid to a total amount of 10.5 μg. For analysis by Northern blotting, 10 μg of total RNA was isolated 40 h later using the guanidine isothiocyanate method. For detection of the 45S rRNA transcript, cells were serum starved for 24 h, replenished with serum for 4 h in the absence or presence of 50 μg α-amanitin following total RNA isolation. A measure of 30 μg of total RNA were analysed by Northern blotting using a probe from the pGem3HR plasmid (Mayer et al, 2005) and Northern with a β-actin probe or ethidium bromide staining was used as loading control.

Supplementary MaterialSupplementary Figures S1–S6Review Process File

Supplementary Material

Supplementary Figures S1–S6

Review Process File
