---
pmid: '29100089'
title: De Novo Mutations in Protein Kinase Genes CAMK2A and CAMK2B Cause Intellectual
  Disability.
authors:
- Küry S
- van Woerden GM
- Besnard T
- Proietti Onori M
- Latypova X
- Towne MC
- Cho MT
- Prescott TE
- Ploeg MA
- Sanders S
- Stessman HAF
- Pujol A
- Distel B
- Robak LA
- Bernstein JA
- Denommé-Pichon AS
- Lesca G
- Sellars EA
- Berg J
- Carré W
- Busk ØL
- van Bon BWM
- Waugh JL
- Deardorff M
- Hoganson GE
- Bosanko KB
- Johnson DS
- Dabir T
- Holla ØL
- Sarkar A
- Tveten K
- de Bellescize J
- Braathen GJ
- Terhal PA
- Grange DK
- van Haeringen A
- Lam C
- Mirzaa G
- Burton J
- Bhoj EJ
- Douglas J
- Santani AB
- Nesbitt AI
- Helbig KL
- Andrews MV
- Begtrup A
- Tang S
- van Gassen KLI
- Juusola J
- Foss K
- Enns GM
- Moog U
- Hinderhofer K
- Paramasivam N
- Lincoln S
- Kusako BH
- Lindenbaum P
- Charpentier E
- Nowak CB
- Cherot E
- Simonet T
- Ruivenkamp CAL
- Hahn S
- Brownstein CA
- Xia F
- Schmitt S
- Deb W
- Bonneau D
- Nizon M
- Quinquis D
- Chelly J
- Rudolf G
- Sanlaville D
- Parent P
- Gilbert-Dussardier B
- Toutain A
- Sutton VR
- Thies J
- Peart-Vissers LELM
- Boisseau P
- Vincent M
- Grabrucker AM
- Dubourg C
- Undiagnosed Diseases Network
- Tan WH
- Verbeek NE
- Granzow M
- Santen GWE
- Shendure J
- Isidor B
- Pasquier L
- Redon R
- Yang Y
- State MW
- Kleefstra T
- Cogné B
- GEM HUGO
- Deciphering Developmental Disorders Study
- Petrovski S
- Retterer K
- Eichler EE
- Rosenfeld JA
- Agrawal PB
- Bézieau S
- Odent S
- Elgersma Y
- Mercier S
journal: Am J Hum Genet
year: '2017'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC5673671
doi: 10.1016/j.ajhg.2017.10.003
---

# De Novo Mutations in Protein Kinase Genes CAMK2A and CAMK2B Cause Intellectual Disability.
**Authors:** Küry S, van Woerden GM, Besnard T, Proietti Onori M, Latypova X, Towne MC, Cho MT, Prescott TE, Ploeg MA, Sanders S, Stessman HAF, Pujol A, Distel B, Robak LA, Bernstein JA, Denommé-Pichon AS, Lesca G, Sellars EA, Berg J, Carré W, Busk ØL, van Bon BWM, Waugh JL, Deardorff M, Hoganson GE, Bosanko KB, Johnson DS, Dabir T, Holla ØL, Sarkar A, Tveten K, de Bellescize J, Braathen GJ, Terhal PA, Grange DK, van Haeringen A, Lam C, Mirzaa G, Burton J, Bhoj EJ, Douglas J, Santani AB, Nesbitt AI, Helbig KL, Andrews MV, Begtrup A, Tang S, van Gassen KLI, Juusola J, Foss K, Enns GM, Moog U, Hinderhofer K, Paramasivam N, Lincoln S, Kusako BH, Lindenbaum P, Charpentier E, Nowak CB, Cherot E, Simonet T, Ruivenkamp CAL, Hahn S, Brownstein CA, Xia F, Schmitt S, Deb W, Bonneau D, Nizon M, Quinquis D, Chelly J, Rudolf G, Sanlaville D, Parent P, Gilbert-Dussardier B, Toutain A, Sutton VR, Thies J, Peart-Vissers LELM, Boisseau P, Vincent M, Grabrucker AM, Dubourg C, Undiagnosed Diseases Network, Tan WH, Verbeek NE, Granzow M, Santen GWE, Shendure J, Isidor B, Pasquier L, Redon R, Yang Y, State MW, Kleefstra T, Cogné B, GEM HUGO, Deciphering Developmental Disorders Study, Petrovski S, Retterer K, Eichler EE, Rosenfeld JA, Agrawal PB, Bézieau S, Odent S, Elgersma Y, Mercier S
**Journal:** Am J Hum Genet (2017)
**DOI:** [10.1016/j.ajhg.2017.10.003](https://doi.org/10.1016/j.ajhg.2017.10.003)
**PMC:** [PMC5673671](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5673671/)

## Abstract

1. Am J Hum Genet. 2017 Nov 2;101(5):768-788. doi: 10.1016/j.ajhg.2017.10.003.

De Novo Mutations in Protein Kinase Genes CAMK2A and CAMK2B Cause Intellectual 
Disability.

Küry S(1), van Woerden GM(2), Besnard T(3), Proietti Onori M(2), Latypova X(3), 
Towne MC(4), Cho MT(5), Prescott TE(6), Ploeg MA(2), Sanders S(7), Stessman 
HAF(8), Pujol A(9), Distel B(10), Robak LA(11), Bernstein JA(12), Denommé-Pichon 
AS(13), Lesca G(14), Sellars EA(15), Berg J(16), Carré W(17), Busk ØL(6), van 
Bon BWM(18), Waugh JL(19), Deardorff M(20), Hoganson GE(21), Bosanko KB(15), 
Johnson DS(22), Dabir T(23), Holla ØL(6), Sarkar A(24), Tveten K(6), de 
Bellescize J(25), Braathen GJ(6), Terhal PA(26), Grange DK(27), van Haeringen 
A(28), Lam C(29), Mirzaa G(30), Burton J(21), Bhoj EJ(31), Douglas J(32), 
Santani AB(33), Nesbitt AI(34), Helbig KL(35), Andrews MV(27), Begtrup A(5), 
Tang S(36), van Gassen KLI(26), Juusola J(5), Foss K(37), Enns GM(12), Moog 
U(38), Hinderhofer K(38), Paramasivam N(39), Lincoln S(32), Kusako BH(32), 
Lindenbaum P(40), Charpentier E(40), Nowak CB(32), Cherot E(17), Simonet T(25), 
Ruivenkamp CAL(28), Hahn S(29), Brownstein CA(4), Xia F(41), Schmitt S(3), Deb 
W(3), Bonneau D(13), Nizon M(3), Quinquis D(3), Chelly J(42), Rudolf G(43), 
Sanlaville D(14), Parent P(44), Gilbert-Dussardier B(45), Toutain A(46), Sutton 
VR(47), Thies J(48), Peart-Vissers LELM(18), Boisseau P(3), Vincent M(3), 
Grabrucker AM(49), Dubourg C(17); Undiagnosed Diseases Network; Tan WH(32), 
Verbeek NE(26), Granzow M(38), Santen GWE(28), Shendure J(50), Isidor B(3), 
Pasquier L(51), Redon R(40), Yang Y(41), State MW(7), Kleefstra T(18), Cogné 
B(3); GEM HUGO(52); Deciphering Developmental Disorders Study(53); Petrovski 
S(54), Retterer K(5), Eichler EE(50), Rosenfeld JA(11), Agrawal PB(55), Bézieau 
S(56), Odent S(51), Elgersma Y(57), Mercier S(3).

Author information:
(1)CHU Nantes, Service de Génétique Médicale, 9 quai Moncousu, 44093 Nantes 
Cedex 1, France. Electronic address: sebastien.kury@chu-nantes.fr.
(2)Department of Neuroscience, Erasmus University Medical Center, 3015 CN 
Rotterdam, the Netherlands; ENCORE Expertise Center for Neurodevelopmental 
Disorders, Erasmus University Medical Center, 3015 CN Rotterdam, the 
Netherlands.
(3)CHU Nantes, Service de Génétique Médicale, 9 quai Moncousu, 44093 Nantes 
Cedex 1, France.
(4)Division of Genetics and Genomics, Boston Children's Hospital and Harvard 
Medical School, Boston, MA 02115, USA; Gene Discovery Core, The Manton Center 
for Orphan Disease Research, Boston Children's Hospital and Harvard Medical 
School, Boston, MA 02115, USA.
(5)GeneDx, Gaithersburg, MD 20877, USA.
(6)Department of Medical Genetics, Telemark Hospital Trust, 3710 Skien, Norway.
(7)Department of Psychiatry, UCSF Weill Institute for Neurosciences, University 
of California, San Francisco, San Francisco, CA 94158, USA.
(8)Department of Genome Sciences, University of Washington School of Medicine, 
Seattle, WA 98195, USA; Department of Pharmacology, Creighton University Medical 
School, Omaha, NE 68178, USA.
(9)Neurometabolic Diseases Laboratory, IDIBELL, Gran Via, 199, L'Hospitalet de 
Llobregat, 08908 Barcelona, and CIBERER U759, Center for Biomedical Research on 
Rare Diseases, 08908 Barcelona, Spain; Catalan Institution of Research and 
Advanced Studies (ICREA), 08010 Barcelona, Spain.
(10)Department of Neuroscience, Erasmus University Medical Center, 3015 CN 
Rotterdam, the Netherlands; ENCORE Expertise Center for Neurodevelopmental 
Disorders, Erasmus University Medical Center, 3015 CN Rotterdam, the 
Netherlands; Department of Medical Biochemistry, Academic Medical Center, 
University of Amsterdam, 1105AZ Amsterdam, the Netherlands.
(11)Department of Molecular and Human Genetics, Baylor College of Medicine, 
Houston, TX 77030, USA.
(12)Department of Pediatrics, Stanford University School of Medicine, Stanford, 
CA 94305, USA.
(13)CHU Angers, Département de Biochimie et Génétique, 49933 Angers Cedex 9, 
France; UMR INSERM 1083 - CNRS 6015, 49933 Angers Cedex 9, France.
(14)Service de génétique, Centre de Référence des Anomalies du Développement, 
Hospices Civils de Lyon, 69288 Lyon, France; INSERM U1028, CNRS UMR5292, Centre 
de Recherche en Neurosciences de Lyon, 69675 Bron, France.
(15)Section of Genetics and Metabolism, Arkansas Children's Hospital, Little 
Rock, AR 72202, USA.
(16)Molecular and Clinical Medicine, School of Medicine, University of Dundee, 
Ninewells Hospital & Medical School, Dundee DD1 9SY, UK.
(17)Laboratoire de Génétique Moléculaire & Génomique, CHU de Rennes, 35033 
Rennes, France.
(18)Department of Human Genetics, Nijmegen Center for Molecular Life Sciences, 
Institute for Genetic and Metabolic Disease, Radboud University Nijmegen Medical 
Center, 6525 GA Nijmegen, the Netherlands.
(19)Department of Neurology, Boston Children's Hospital and Harvard Medical 
School, Boston, MA 02115, USA.
(20)Department of Pediatrics, Division of Genetics, Children's Hospital of 
Philadelphia, Philadelphia, PA 19104, USA.
(21)Department of Pediatrics, University of Illinois at Chicago, College of 
Medicine, Chicago, IL 60612, USA.
(22)Sheffield Children's Hospital, Western Bank, Sheffield S10 2TH, UK.
(23)Northern Ireland Regional Genetics Centre, Belfast Health and Social Care 
Trust, Belfast City Hospital, Lisburn Road, Belfast BT9 7AB, UK.
(24)Nottingham Regional Genetics Service, City Hospital Campus, Nottingham 
University Hospitals NHS Trust, The Gables, Hucknall Road, Nottingham NG5 1PB, 
UK.
(25)Epilepsy, Sleep and Pediatric Neurophysiology Department, Hospices Civils, 
Lyon, 69677 Bron, France.
(26)Department of Genetics, University Medical Center Utrecht, Utrecht 3584 EA, 
the Netherlands.
(27)Division of Genetics and Genomic Medicine, Department of Pediatrics, 
Washington University School of Medicine, Saint Louis, MO 63110, USA.
(28)Department of Clinical Genetics, Leiden University Medical Center (LUMC), 
2333 ZA Leiden, the Netherlands.
(29)Division of Genetic Medicine, Department of Pediatrics, University of 
Washington School of Medicine and Seattle Children's Hospital, Seattle, WA 
98105, USA.
(30)Division of Genetic Medicine, Department of Pediatrics, University of 
Washington School of Medicine and Seattle Children's Hospital, Seattle, WA 
98105, USA; Center for Integrative Brain Research, Seattle Children's Research 
Institute, Seattle, WA 98101, USA.
(31)Center for Applied Genomics, The Children's Hospital of Philadelphia, 
Philadelphia, PA 19104, USA; Division of Human Genetics, The Children's Hospital 
of Philadelphia, Philadelphia, PA 19104, USA.
(32)Division of Genetics and Genomics, Boston Children's Hospital and Harvard 
Medical School, Boston, MA 02115, USA.
(33)Division of Genomic Diagnostics, Department of Path and Lab Medicine, 
Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA; Department of 
Path and Lab Medicine, Perelman School of Medicine, University of Pennsylvania, 
Philadelphia, PA 19104-4238, USA.
(34)Division of Genomic Diagnostics, Department of Path and Lab Medicine, 
Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA.
(35)Division of Clinical Genomics, Ambry Genetics, 15 Argonaut, Aliso Viejo, CA 
92656, USA; Division of Neurology, The Children's Hospital of Philadelphia, 
Philadelphia, PA 19104, USA.
(36)Division of Clinical Genomics, Ambry Genetics, 15 Argonaut, Aliso Viejo, CA 
92656, USA.
(37)Center for Integrative Brain Research, Seattle Children's Research 
Institute, Seattle, WA 98101, USA.
(38)Institute of Human Genetics, University Heidelberg, Im Neuenheimer Feld 366, 
69120 Heidelberg, Germany.
(39)Medical Faculty Heidelberg, Heidelberg University, 69120 Heidelberg, Germany 
and Division of Theoretical Bioinformatics, German Cancer Research Center 
(DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany.
(40)INSERM, CNRS, UNIV Nantes, l'institut du thorax, 44007 Nantes, France; CHU 
Nantes, l'institut du thorax, 44093 Nantes, France.
(41)Department of Molecular and Human Genetics, Baylor College of Medicine, 
Houston, TX 77030, USA; Baylor Genetics, Houston, TX 77030, USA.
(42)Laboratoire de Diagnostic Génétique, Hôpitaux Universitaires de Strasbourg, 
Nouvel Hôpital Civil, 67091 Strasbourg, France; Fédération de Médecine 
Translationnelle de Strasbourg (FMTS), Université de Strasbourg, 67000 
Strasbourg, France; Institut de Génétique et de Biologie Moléculaire et 
Cellulaire (IGBMC), INSERM-U964/CNRS-UMR7104/Université de Strasbourg, 67404 
Illkirch, France.
(43)Fédération de Médecine Translationnelle de Strasbourg (FMTS), Université de 
Strasbourg, 67000 Strasbourg, France; Institut de Génétique et de Biologie 
Moléculaire et Cellulaire (IGBMC), INSERM-U964/CNRS-UMR7104/Université de 
Strasbourg, 67404 Illkirch, France; Service of Neurology, University Hospital of 
Strasbourg, Hospital of Hautepierre, 1 avenue Molière, 67098 Strasbourg Cedex, 
France.
(44)CHRU Brest, Génétique médicale, 29609 Brest, France.
(45)CHU Poitiers, Service de Génétique, BP577, 86021 Poitiers, France; EA 3808 
Université Poitiers, France.
(46)CHU Tours, Service de Génétique, 2 Boulevard Tonnellé, 37044 Tours, France.
(47)Baylor Genetics, Houston, TX 77030, USA.
(48)Division of Genetic Medicine, Department of Pediatrics, Seattle Children's 
Hospital, Seattle, WA 98105, USA.
(49)Department of Biological Sciences, University of Limerick, Limerick V94 
T9PX, Ireland; Bernal Institute, University of Limerick, Limerick V94 T9PX, 
Ireland.
(50)Department of Genome Sciences, University of Washington School of Medicine, 
Seattle, WA 98195, USA; Howard Hughes Medical Institute, Seattle, WA 98195, USA.
(51)CHU Rennes, Service de Génétique Clinique, CNRS UMR6290, Université Rennes1, 
35203 Rennes, France.
(52)Réseau de génétique et génomique médicale - Hôpitaux Universitaires du Grand 
Ouest, CHU Rennes, Service de Génétique Clinique, 35203 Rennes, France.
(53)Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 
1SA, UK.
(54)Department of Medicine, The University of Melbourne, Austin Health and Royal 
Melbourne Hospital, Melbourne, VIC 3010, Australia.
(55)Division of Genetics and Genomics, Boston Children's Hospital and Harvard 
Medical School, Boston, MA 02115, USA; Gene Discovery Core, The Manton Center 
for Orphan Disease Research, Boston Children's Hospital and Harvard Medical 
School, Boston, MA 02115, USA; Division of Newborn Medicine, Boston Children's 
Hospital and Harvard Medical School, Boston, MA 02115, USA.
(56)CHU Nantes, Service de Génétique Médicale, 9 quai Moncousu, 44093 Nantes 
Cedex 1, France; CRCINA, Inserm, Université d'Angers, Université de Nantes, 
44000 Nantes, France.
(57)Department of Neuroscience, Erasmus University Medical Center, 3015 CN 
Rotterdam, the Netherlands; ENCORE Expertise Center for Neurodevelopmental 
Disorders, Erasmus University Medical Center, 3015 CN Rotterdam, the 
Netherlands. Electronic address: y.elgersma@erasmusmc.nl.

Calcium/calmodulin-dependent protein kinase II (CAMK2) is one of the first 
proteins shown to be essential for normal learning and synaptic plasticity in 
mice, but its requirement for human brain development has not yet been 
established. Through a multi-center collaborative study based on a whole-exome 
sequencing approach, we identified 19 exceedingly rare de novo CAMK2A or CAMK2B 
variants in 24 unrelated individuals with intellectual disability. Variants were 
assessed for their effect on CAMK2 function and on neuronal migration. For both 
CAMK2A and CAMK2B, we identified mutations that decreased or increased CAMK2 
auto-phosphorylation at Thr286/Thr287. We further found that all mutations 
affecting auto-phosphorylation also affected neuronal migration, highlighting 
the importance of tightly regulated CAMK2 auto-phosphorylation in neuronal 
function and neurodevelopment. Our data establish the importance of CAMK2A and 
CAMK2B and their auto-phosphorylation in human brain function and expand the 
phenotypic spectrum of the disorders caused by variants in key players of the 
glutamatergic signaling pathway.

Copyright © 2017 American Society of Human Genetics. Published by Elsevier Inc. 
All rights reserved.

DOI: 10.1016/j.ajhg.2017.10.003
PMCID: PMC5673671
PMID: 29100089 [Indexed for MEDLINE]

## Full Text

Abstract

Calcium/calmodulin-dependent protein kinase II (CAMK2) is one of the first proteins shown to be essential for normal learning and synaptic plasticity in mice, but its requirement for human brain development has not yet been established. Through a multi-center collaborative study based on a whole-exome sequencing approach, we identified 19 exceedingly rare de novo CAMK2A or CAMK2B variants in 24 unrelated individuals with intellectual disability. Variants were assessed for their effect on CAMK2 function and on neuronal migration. For both CAMK2A and CAMK2B , we identified mutations that decreased or increased CAMK2 auto-phosphorylation at Thr286/Thr287. We further found that all mutations affecting auto-phosphorylation also affected neuronal migration, highlighting the importance of tightly regulated CAMK2 auto-phosphorylation in neuronal function and neurodevelopment. Our data establish the importance of CAMK2A and CAMK2B and their auto-phosphorylation in human brain function and expand the phenotypic spectrum of the disorders caused by variants in key players of the glutamatergic signaling pathway.

Introduction

Modification of synaptic strength, i.e., synaptic plasticity, is a cornerstone in human capacity to adapt to environmental change. The ability of synapses to modulate their strength is critical for learning and memory processes. 1 Both strengthening (known as long-term potentiation [LTP]) and weakening (known as long-term depression [LTD]) of synaptic transmission have been shown to contribute to distinct types of learning and long-term memory. 2 , 3 , 4 , 5

Abnormal synaptic plasticity is a well-recognized cause of numerous neurological and psychiatric disorders, 6 as exemplified by dysfunctional ionotropic glutamate receptor signaling, notably α-amino-3-hydroxy-5-methyl-4-isoxazole propionic acid receptors (AMPAR) and N-methyl-D-aspartate receptor (NMDAR) signaling. 7 , 8 , 9 , 10 , 11 For example, pathogenic variants in AMPAR genes GRIA3 (MIM: 305915 ) (GluA3, associated with mental retardation, X-linked 94 [MIM: 305915 ]) 12 , 13 and CACNG2 (MIM: 602911 ) (stargazin, associated with mental retardation, autosomal-dominant 10 [MIM: 614256 ]), 14 and in genes of NMDAR subunits GRIN1 (MIM: 138249 ) (GluN1, associated with mental retardation, autosomal-dominant 8 [MIM: 614254 ]), GRIN2A (MIM: 138253 ) (GluN2A, associated with focal epilepsy, with speech disorder, and with or without mental retardation [MIM: 245570 ]), GRIN2B (MIM: 138252 ) (GluN2B, associated with early infantile epileptic encephalopathy, 27 [MIM: 616139 ] and mental retardation, autosomal-dominant 6 [MIM: 613970 ]), and GRIN2D (MIM: 602717 ) (GluN2D, associated with epileptic encephalopathy, early infantile, 46 [MIM: 617162 ]) 14 , 15 , 16 , 17 , 18 , 19 , 20 are a cause of neurodevelopmental disorders.

Another major regulator of synaptic plasticity is the Ca 2+ /calmodulin-dependent serine/threonine protein kinase CAMK2, whose activation is necessary and sufficient for hippocampal LTP induction. 21 In the hippocampus, CAMK2 forms a dodecameric holoenzyme that is mainly composed of its two predominant subunits, alpha (CAMK2A) and beta (CAMK2B), which together represent 2% of the total hippocampal protein content. 22 The CAMK2A and CAMK2B subunits share a very high degree of homology and consist of four distinct domains: a catalytic domain containing the active site required for CAMK2 kinase activity, a regulatory domain comprising the calcium-calmodulin binding site (including the autoinhibitory sub-domain of the Thr286/Thr287 phosphorylation site required for autonomous [calcium-independent] activity), a variable domain, i.e., the domain in which CAMK2A and CAMK2B show the largest difference, and an association domain necessary for assembly of the (mixed) holoenzyme with 10–12 CAMK2 subunits. 21 , 23 , 24 Subunits combine mainly to form CAMK2A homomers or CAMK2A-CAMK2B heteromers. 2

CAMK2 contributes in various ways to synaptic plasticity, but most critical is its regulation of ionotropic glutamate receptors. In particular, CAMK2 binding to the NMDAR and its ability to regulate the membrane insertion and activity of AMPAR are crucial for regulating synaptic strength. 2 , 24 , 25 , 26 This process is triggered by Ca 2+ entry through the NMDAR, resulting in Ca 2+ /calmodulin binding to the CAMK2 regulatory domain. 24 Subsequent auto-phosphorylation of amino acid residue Thr286 in the inhibitory domain of the α-subunit isoform (or Thr287 in the β-subunit isoform) of CAMK2 allows the enzyme to maintain its activated conformation, thereby effectively converting a very brief Ca 2+ signal into a long-lasting enzymatic change. Activated CAMK2 translocates from the dendritic shaft to post synaptic densities (PSDs) of dendritic spines, where it binds to NMDAR (GluN1, 2A and 2B) and enhances the insertion and function of AMPAR GluA1 ( GRIA1 [MIM: 138248 ]), stargazin, and SAP97 ( DLG1 [MIM: 601014 ]) 24 in the PSD. The importance of these events is highlighted by the observation that Camk2 knock-in mice with mutations interfering with calmodulin binding, or silencing the autonomous or kinetic activity of the protein, result in profound learning and plasticity deficits. 27 , 28 , 29

Paradoxically, although the Camk2a knock-out was the first murine knock-out in the field of neuroscience to establish a critical role for NMDA receptor-mediated calcium signaling in learning and synaptic plasticity, 30 , 31 the importance of CAMK2A (MIM: 114078 ) for cognitive function has not yet been established. Similarly, Camk2b has been shown to be important for learning and synaptic plasticity in mice, 32 , 33 but mutations in CAMK2B (MIM: 607707 ) have not yet been described. In the present study, we report 19 rare variants in CAMK2A or CAMK2B in 24 unrelated individuals with intellectual disability (ID), with 23 individuals shown to have de novo occurrence of the variants. We also provide evidence that most of these alterations of CAMK2 are likely to affect protein and neuronal function.

Discussion

The first mouse mutant with impaired learning and memory was reported 25 years ago when the Camk2a knock-out was shown to have impaired hippocampus-dependent learning and impaired hippocampal synaptic plasticity. 30 , 31 Subsequently, an overwhelming amount of data has highlighted the importance of CAMK2A and CAMK2B for many brain areas. 2 , 21 , 63 The regulation of kinase activity by auto-phosphorylation has been demonstrated to be essential for the role of CAMK2 in murine neuronal function. 2 , 28 , 62 We now provide evidence that proper CAMK2A and CAMK2B functioning is also important for the human brain. We report 24 individuals with a neurodevelopmental disorder who carry a rare heterozygous nonsense, missense, or splice site variant in CAMK2A or CAMK2B —the majority (23/24) were proven de novo . The core clinical findings in affected heterozygous individuals are neurological and consistent with the knockout mouse models for Camk2a and Camk2b 30 , 32 , 64 , 65 (the majority of which are homozygous knock-out). Variants in CAMK2A or CAMK2B result in a non-dysmorphic neurodevelopmental phenotype characterized by a variable degree of ID. Language development is particularly impaired and behavioral problems are frequent. In our cohort, motor delay was more pronounced in the subgroup of individuals with CAMK2A variants, while ID appeared to be more severe and more frequently accompanied by hypotonia with CAMK2B variants. The differential diagnosis of the CAMK2A/B-associated disorder includes many individually rare neurodevelopmental disorders and is virtually impossible to diagnose without molecular testing.

A number of molecular arguments helped us to draw an outline of the clinical entity caused by CAMK2A or CAMK2B variants. The 23 proven de novo events reported here occur in two genes bioinformatically predicted to be intolerant to functional variations ( Table S4 ). They are the only such events in the CAMK2 genes noted in 19,980 trios. The extremely rare variants were identified in individuals who shared the same main clinical phenotype, making it unlikely that they represent incidental findings. The probability of observing 13 or more CAMK2A and 10 or more CAMK2B non-synonymous de novo variants by chance among 19,980 trios is extremely low ( CAMK2A p = 1.7 × 10 −11 and CAMK2B p = 3.9 × 10 −8 ), implicating CAMK2A and CAMK2B as genome-wide significant developmental delay associated genes (exome-wide multiplicity adjusted p < 2.6 × 10 −6 ).

In order to test this assumption further, we employed a number of functional assays to investigate the pathogenicity of the CAMK2A/B missense variants. We analyzed their effect on protein stability, Thr286/Thr287 (auto)phosphorylation, and neuronal function by using a neuronal migration assay. Whereas overexpression or knockdown of CAMK2A does not lead to aberrant migration, we found that five out of eight tested missense variants (CAMK2A p.(Phe98Ser) , CAMK2A p.(Glu109Asp) , CAMK2A p.(Glu183Val) , CAMK2A p.(His282Arg) , and CAMK2A p.(Thr286Pro) ) affect neuronal migration when expressed in vivo . Notably, all these mutants showed either increased or decreased Thr286 phosphorylation in our cellular assays. Auto-phosphorylation of the Thr286 residue has been shown to be essential for CAMK2A function: it renders the protein autonomously active and facilitates interaction with the carboxy-terminal region of GRIN2B (GluN2B) NMDAR subunit. 2 , 3 , 66 Based on our results we further conclude that the CAMK2A p.(Phe98Ser) and CAMK2A p.(Glu183Val) variants result in a dominant acting loss of function (LoF) of the protein, whereas CAMK2A p.(Glu109Asp) and CAMK2A p.(His282Arg) induce a dominant acting gain of function (GoF). Surprisingly, the CAMK2A p.(Thr286Pro) variant, which cannot be auto-phosphorylated at Thr286, behaves as a dominant acting GoF mutation. For CAMK2A p.(Pro138Ala) , CAMK2A p.(Pro212Leu) , or CAMK2A p.(Pro235Leu) , we found minimal effect on stability, autophosphorylation, and neuronal migration; thus, the pathogenicity of these variants by functional assay cannot be definitively confirmed. However, they likely affect other aspects of CAMK2 and neuronal function that were not examined in this study. For instance, the CAMK2A p.(Pro212Leu) variant recurs in three individuals with similar phenotypes, is located in the most missense depleted sequence of CAMK2A ( Figure S10 ), and does not appear in large variant databases like ExAC or gnomAD, which is consistent with pathogenicity. This example underscores the necessity of designing additional tests to assess the effects of variants that could not be confidently classified by our screening.

We found that increased as well as decreased levels of CAMK2B during neuronal development caused migration defects, indicating that cortical development requires an optimal amount of CAMK2B protein. Of the four tested missense variants in CAMK2B, we found that CAMK2B p.(Lys301Glu) acts as a LoF mutation, showing reduced phosphorylation of Thr287 with less impaired migration compared to increased levels of CAMK2B WT . In contrast, increased Thr287 phosphorylation severely affected neuronal migration (compared to CAMK2B WT ), suggesting that CAMK2B p.(Glu110Lys) , CAMK2B p.(Pro139Leu) , and CAMK2B p.(Glu237Lys) act as GoF mutations. Taken together, these results indicate that not only LoF but also GoF mutations in CAMK2B can disturb neurodevelopment, consistent with the notion that cortical development is highly sensitive to dosage of CAMK2B protein.

Considering that three of the variants, p.His282Arg, p.Thr286Pro, and p.Lys301Glu, are positioned in the regulatory domain, their effect on CAMK2 function could have potentially been predicted a priori . The CAMK2A His residue at 282 has been shown to be one of the important amino acid residues for potent inhibition of CAMK2A. Indeed, substitution of the histidine with an alanine at this position reduced the inhibitory potency about 25-fold. 67 This is consistent with our findings of increased Thr286 autophosphorylation found in the CAMK2A p.(His282Arg) mutant. Similarly, the CAMK2A p.Thr286Pro variant blocks the critical Thr286 auto-phosphorylation of CAMK2A and hence is likely to be highly pathogenic. Surprisingly, however, we found that instead of acting as a LoF mutant, the variant renders the CAMK2A protein constitutively active and that silencing the kinase activity of this mutant form of CAMK2A normalizes migration. This is a rather unexpected finding, but one possible explanation could be that the change from a threonine into a proline causes a conformational change that is similar to undergoing phosphorylation at this site or to introducing a negatively charged amino acid. 61 , 62 Whether this is indeed the case will be subject of future research. Lastly, the CAMK2B p.Lys301Glu variant is located in the Ca 2+ /Calmodulin (CaM) binding domain of CAMK2B and is therefore likely to interfere with Ca 2+ /CaM binding and hence its activity. Indeed, our in vitro data show that phosphorylation of CAMK2B at Thr287 is largely absent. A knock-in mutation in CAMK2A (CAMK2 p.(Thr305Asp) ) that interferes with Ca/CaM binding has a dramatic effect on synaptic plasticity and learning. 27 However, the Camk2b p.(Ala303Arg) mouse mutant, in which CaM can no longer bind to Camk2b, 68 has severe motor deficits 69 but no clear spatial learning or plasticity phenotype. 33 Individual 23, in whom the CAMK2B p.(Lys301Gly) variant was identified, also presents the hemizygous variant NLGN3 (GenBank: NM_018977 ; c.214dupG; p.Val72Glyfs ∗ 17), inherited from his mother who has microcephaly but no other cognitive or developmental impairments. Hence, it is still possible that the combination of both variants resulted in the clinical features.

The remainder of the missense variants found in our cohort all lie in the catalytic domain of the kinase and their effect is harder to predict a priori . The CAMK2A p.Glu183Val (CAMK2A p.(Glu183Val) ) variant was recently published by the Simons Simplex Collection as part of a large cohort study on autism spectrum disorders 38 and studied in a mouse model. 55 In this study, the authors show that the CAMK2A p.(Glu183Val) variant results in reduced expression and activity of CAMK2A and causes behavioral deficits. 55 These results are in line with our finding that the variant CAMK2A p.(Glu183Val) renders the protein unstable, reduces autonomous activity, and affects neuronal migration. The parallel findings emphasize the usefulness of the assays we employed to assess pathogenicity.

We observed that some variants resulted in increased and others in decreased CAMK2 activity and that all such variants also affected neuronal migration. However, the precise underlying pathogenic mechanisms may nevertheless be more complex and require further functional studies. An unresolved issue is whether haploinsufficiency could also lead to the same phenotype, as suggested by frameshift CAMK2A c.65del (p.Gly22Glufs ∗ 10) and nonsense CAMK2B c.85C>T (p.Arg29 ∗ ) variants and by CAMK2A c.1204+1G>A which leads to out-of-frame skipping of exon 17. According to variant databases, CAMK2A and CAMK2B are predicted to be haploinsufficient ( Table S4 ). Indeed, loss-of-function variants are exceedingly rare, in particular for the most abundant transcripts in brain, which are GenBank: NM_171825.2 for CAMK2A and GenBank: NM_172079.2 for CAMK2B ( Figures S7 and S8 ). For these two transcripts combined, only three LoF variants were identified in 62,000 exomes sequenced by ExAC, and three in more than 69,500 exomes sequenced by centers participating in the study. A small deletion in 5q32 encompassing CAMK2A and four other genes was reported in two case subjects with Treacher Collins syndrome in which ID was suspected to be due to the deletion of CAMK2A . 70 The haploinsufficiency hypothesis is consistent with our knockdown experiments that showed that reduced CAMK2B disrupts migration. Yet, given that the amount of protein was reduced to 20%, this experiment may not be an appropriate test for haploinsufficiency. Our migration assay was insensitive to reducing CAMK2A levels, but this is not surprising given that CAMK2A is hardly expressed at this time point. Even if haploinsufficiency accounts for some of the case subjects, it is noteworthy that, CAMK2A and CAMK2B taken together, the ID seen in individuals included in our study seems more severe when associated with GoF variants (4/5) than when associated with LoF variants (1/6) ( Table S2 ). This is consistent with the rather weak phenotypes of heterozygous Camk2a and Camk2b knock-out mice, 27 , 28 , 30 , 31 , 65 , 71 in contrast with the more severe phenotypes of mice with variants affecting auto-phosphorylation.

Of note, the case of variant CAMK2A c.1204+1G>A contrasts with the other variants in the series, as this is the only variant to interrupt the association domain. A similar observation was made in a recessive form of ID (B. Reversade, personal communication).

Our functional studies highlight the variety of mechanisms leading to ID related to CAMK2A/B mutations. Most certainly, this variety reflects the multiple ways CAMK2 acts on synaptic plasticity. 24 We can therefore postulate that the spectrum of neurological disorders associated with CAMK2A/B variants could extend to other rare or common neurological or psychiatric disorders, as already suggested by others. 72 In our series, we stressed the frequency of abnormal behaviors, which is in line with a previous study suggesting a relationship between CAMK2A mutations and autism spectrum disorder. 55 CAMK2A involvement was also recently shown in bipolar disorders. 73 Given their tight link in NMDA-dependent LTP process, it is very tempting to draw a comparison between CAMK2 and GRIN2B or GRIN2A . Variants in the latter genes may cause ID 17 , 74 , 75 and infantile epileptic encephalopathy, 18 epileptic aphasia, 20 or be associated with Tourette syndrome, 76 schizophrenia, 77 or autism. 78

In conclusion, our observations highlight a rare genetic cause of developmental delay/ID due to alterations in genes encoding subunits of CAMK2, a key holoenzyme in learning and memory processes in the AMPAR- and NMDAR-dependent signaling pathways. Our findings expand the phenotypic spectrum of the disorders caused by variants in other key actors of this pathway. We have identified mutations in two CAMK2 paralogous genes as the cause of a neurodevelopmental phenotype characterized by ID. Functional studies performed in this study suggest that CAMK2A and CAMK2B may be involved in a broad range of neurologic and psychiatric disorders.
