---
pmid: '19299493'
title: Biochemical and genetic evidence for a role of IGHMBP2 in the translational
  machinery.
authors:
- de Planell-Saguer M
- Schroeder DG
- Rodicio MC
- Cox GA
- Mourelatos Z
journal: Hum Mol Genet
year: '2009'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2685751
doi: 10.1093/hmg/ddp134
---

# Biochemical and genetic evidence for a role of IGHMBP2 in the translational machinery.
**Authors:** de Planell-Saguer M, Schroeder DG, Rodicio MC, Cox GA, Mourelatos Z
**Journal:** Hum Mol Genet (2009)
**DOI:** [10.1093/hmg/ddp134](https://doi.org/10.1093/hmg/ddp134)
**PMC:** [PMC2685751](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2685751/)

## Abstract

1. Hum Mol Genet. 2009 Jun 15;18(12):2115-26. doi: 10.1093/hmg/ddp134. Epub 2009 
Mar 19.

Biochemical and genetic evidence for a role of IGHMBP2 in the translational 
machinery.

de Planell-Saguer M(1), Schroeder DG, Rodicio MC, Cox GA, Mourelatos Z.

Author information:
(1)Department of Pathology and Laboratory Medicine, Division of Neuropathology, 
University of Pennsylvania School of Medicine, Philadelphia, PA 19104-6100, USA.

The human motor neuron degenerative disease spinal muscular atrophy with 
respiratory distress type 1 (SMARD1) is caused by loss of function mutations of 
immunoglobulin mu-binding protein 2 (IGHMBP2), a protein of unknown function 
that contains DNA/RNA helicase and nucleic acid-binding domains. Reduced IGHMBP2 
protein levels in neuromuscular degeneration (nmd) mice, the mouse model of 
SMARD1, lead to motor neuron degeneration. We report the biochemical 
characterization of IGHMBP2 and the isolation of a modifier locus that rescues 
the phenotype and motor neuron degeneration of nmd mice. We find that a 166 kb 
BAC transgene derived from CAST/EiJ mice and containing tRNA genes and activator 
of basal transcription 1 (Abt1), a protein-coding gene that is required for 
ribosome biogenesis, contains the genetic modifier responsible for motor neuron 
rescue. Our biochemical investigations show that IGHMBP2 associates physically 
with tRNAs and in particular with tRNA(Tyr), which are present in the modifier 
and with the ABT1 protein. We find that transcription factor IIIC-220 kDa 
(TFIIIC220), an essential factor required for tRNA transcription, and the 
helicases Reptin and Pontin, which function in transcription and in ribosome 
biogenesis, are also part of IGHMBP2-containing complexes. Our findings strongly 
suggest that IGHMBP2 is a component of the translational machinery and that 
these components can be manipulated genetically to suppress motor neuron 
degeneration.

DOI: 10.1093/hmg/ddp134
PMCID: PMC2685751
PMID: 19299493 [Indexed for MEDLINE]

## Full Text

Abstract

The human motor neuron degenerative disease spinal muscular atrophy with respiratory distress type 1 (SMARD1) is caused by loss of function mutations of immunoglobulin µ-binding protein 2 (IGHMBP2), a protein of unknown function that contains DNA/RNA helicase and nucleic acid-binding domains. Reduced IGHMBP2 protein levels in neuromuscular degeneration ( nmd ) mice, the mouse model of SMARD1, lead to motor neuron degeneration. We report the biochemical characterization of IGHMBP2 and the isolation of a modifier locus that rescues the phenotype and motor neuron degeneration of nmd mice. We find that a 166 kb BAC transgene derived from CAST/EiJ mice and containing tRNA genes and activator of basal transcription 1 ( Abt1 ), a protein-coding gene that is required for ribosome biogenesis, contains the genetic modifier responsible for motor neuron rescue. Our biochemical investigations show that IGHMBP2 associates physically with tRNAs and in particular with tRNA Tyr , which are present in the modifier and with the ABT1 protein. We find that transcription factor IIIC-220 kDa (TFIIIC220), an essential factor required for tRNA transcription, and the helicases Reptin and Pontin, which function in transcription and in ribosome biogenesis, are also part of IGHMBP2-containing complexes. Our findings strongly suggest that IGHMBP2 is a component of the translational machinery and that these components can be manipulated genetically to suppress motor neuron degeneration.

INTRODUCTION

Spinal muscular atrophy with respiratory distress type 1 (SMARD1) is an autosomal recessive neurodegenerative disease ( 1 , 2 ). The disease is also known as distal spinal muscular atrophy 1 (DSMA1, MIM no. 604320). The primary pathology of SMARD1 consists of degeneration of spinal cord motor neurons and axonal loss leading to muscular atrophy ( 1 , 2 ). Diaphragmatic paralysis is one of the earliest and most severe manifestations of the disease and typically leads to early infant death ( 1 , 2 ). SMARD1 results from mutations in the immunoglobulin µ-binding protein 2 (IGHMBP2) ( 1 , 2 ). The human IGHMBP2 is a 993 amino acid protein that contains an RNA/DNA helicase domain (found in the superfamily I helicases), an R3H, single-stranded nucleic acid-binding domain and a zinc finger domain ( 1 – 3 ). IGHMBP2 mRNA is ubiquitously expressed ( 4 ). On the basis of in vitro binding assays, IGHMBP2 has been proposed to bind DNA and was given several names in the literature: glial factor 1 [ GF1 ( 5 )], immunoglobulin S-µ-binding protein-2 [ Smbp2 or IGHMBP2 ( 4 , 6 )], rat insulin-enhancer-binding protein [ Rip1 ( 7 )] and cardiac transcription factor 1 [ Catf1 ( 8 )]. Proposed functions, based on in vitro data, include transcriptional activation, immunoglobulin class switching and pre-mRNA splicing ( 4 – 9 ). In neurons, IGHMBP2 is predominantly found in the cytoplasm, including dendrites and axons ( 3 ), arguing that IGHMBP2 has a major function outside the nucleus. The presence of an RNA helicase domain and a single-stranded nucleic acid-binding domain suggests that IGHMBP2 may function in RNA processing, regulation or metabolism.

Neuromuscular degeneration ( nmd ) mice that are homozygous for mutations of the Ighmbp2 gene develop a phenotype that is very similar to SMARD1 ( 10 ). The pathology of nmd mice consists of motor neuron degeneration with axonal loss leading to neurogenic muscle atrophy and death by ∼8–12 weeks ( 10 – 12 ). We previously reported that the nmd mutation is a single A-to-G transition 23 bp into intron 4 of the mouse Ighmbp2 gene that creates a new favorable splice donor, resulting in a frameshift and premature stop codon ( 10 ). In nmd homozygous mice, low levels (∼20%) of correctly spliced mRNA are evident in all nmd tissues, indicating that this is a hypomorphic allele and not a complete null mutation ( 10 ). We reported that the motor neuron degeneration of nmd mice was completely corrected by transgenic expression of wild-type IGHMBP2 driven by a neuron-specific promoter ( 11 ). This finding demonstrates that motor neuron degeneration in nmd mice is cell autonomous and is a direct consequence of reduced IGHMBP2 protein levels in motor neurons. Interestingly, these transgenic mice later developed a primary myopathy affecting both skeletal muscles and the myocardium, suggesting that IGHMBP2 is also essential for myocyte maintenance ( 11 ). Such primary myopathy has not been described in human patients with SMARD1, presumably because it is superceded by the severe motor neuronopathy that leads to early death, before manifestation of the primary myopathy ( 11 ).

Mice contain 418 tRNA genes and humans contain 505 tRNA genes, corresponding to tRNAs that decode the standard 20 amino acids ( 13 ). Approximately 6% of these genes contain introns ( 13 ). tRNAs are transcribed in the nucleus by RNA polymerase III as longer pre-tRNA transcripts ( 14 , 15 ). tRNA genes contain intragenic RNA polymerase III promoters whose core elements consist of the A and B boxes (whose sequences roughly correspond to the D- and T-loops of the mature tRNA) ( 16 , 17 ). The A and B boxes are recognized by a multisubunit complex known as general transcription factor IIIC (TFIIIC). Mammalian TFIIIC220, one of the subunits of TFIIIC, binds directly to box B of RNA polymerase III promoters and is essential for tRNA transcription ( 16 , 17 ). Pre-tRNA transcripts contain 5′-leader and 3′-trailer sequences ( 14 , 15 ). Several tRNAs also contain a single, short (14–60 nt) intron which is found in the anticodon loop, typically one nucleotide 3′ to the anticodon ( 13 – 15 ). Processing of pre-tRNAs involves removal of the 5′-leader by RNAse P, removal of the 3′-trailer by various endo- and exo-nucleases, splicing of the intron, addition of the CCA trinucleotide to the 3′-end and numerous tRNA modifications ( 14 , 15 , 18 , 19 ). Addition of the CCA trinucleotide typically occurs prior to tRNA splicing and is followed by aminoacylation and export of the mature tRNA to the cytoplasm ( 15 ).

The biogenesis of ribosomes in eukaryotes is a complicated, step-wise process that involves more than 200 proteins and an equal number of small nucleolar RNAs (snoRNAs) ( 20 ). RNA polymerase I generates a large pre-rRNA precursor (termed 47S/45S) in mammals that contains the mature 18S, 28S and 5.8S rRNAs. A series of endonucleolytic cleavages and exonucleolytic processing steps leads to generation of the mature rRNAs ( 21 ). snoRNAs are involved in the processing and modification of ribosomal RNA (rRNA) and are found in cells complexed with proteins (snoRNPs). U3 snoRNP is a large ribonucloprotein composed of a 217 nucleotide U3 snoRNA and multiple proteins ( 22 ) and directs the first cleavage of the pre-rRNA precursor ( 22 , 23 ).

Spinal muscular atrophy (SMA), although distinct from SMARD1, shares many clinical and pathological features with SMARD1. SMA is caused by deletions or loss of function mutations of the survival of motor neurons (SMN) gene ( 24 ). The pathology of SMA consists of degeneration of spinal cord motor neurons, leading to muscular atrophy ( 25 ). The SMN protein associates with seven additional proteins, known as Gemins2–8, into a large complex termed the SMN complex ( 26 – 28 ). The SMN complex plays critical roles in the biogenesis of spliceosomal small nuclear RNPs (snRNPs) and possibly other RNPs ( 26 , 29 – 32 ).

Here, we show that IGHMBP2 is not a component of the SMN complex, but rather it interacts with a unique set of small RNAs that includes tRNA Tyr , and with proteins, including helicases, TFIIIC220 and activator of basal transcription 1 (ABT1), that are required for tRNA transcription and ribosome biogenesis and function. These biochemical observations are corroborated in mutant nmd mice by the analysis of a genetic suppressor of motor neuron disease encoding multiple tRNA Tyr genes and ABT1 that we have shown to associate physically with IGHMBP2. Our findings strongly suggest that IGHMBP2 is a component of the translational machinery and that these components can be manipulated genetically to suppress motor neuron degeneration.

DISCUSSION

Dysregulation of pre-mRNA, rRNA or tRNA pathways is emerging as an important feature of neurodegenerative disorders and in particular in degeneration of motor neurons or their axons.

SMA is caused by reduced SMN protein levels, and in the SMA mouse, this reduction results in generalized pre-mRNA splicing defects ( 42 ). Most cells, except motor neurons, apparently tolerate such splicing defects caused by reduced SMN levels. It is also possible that in motor neurons, SMN may have additional, as yet unidentified functions that promote axonal and neuronal survival.

Charcot–Marie–Tooth (CMT) neuropathies are common diseases of the peripheral nervous system that are caused by degeneration of axons (either axonal or demyelinating) of motor neurons or sensory neurons. Mutations of the tyrosyl-tRNA synthetase (also known as YARS), the enzyme that aminoacylates tRNA Tyr , cause dominant intermediate CMT neuropathy type C (DI-CMTC) ( 43 ). Mutations of glycyl-tRNA synthetase (also known as GARS), the enzyme that aminoacylates tRNA Gly , cause CMT neuropathy type 2D (CMTD2) and distal spinal muscular atrophy type V (dSMA-V) ( 44 , 45 ). Recent studies indicate that dSMA-V and CMTD2 are best characterized as primary motor neuronopathies ( 45 ). These findings strongly indicate that disruption of tRNA function may underlie neurodegeneration.

Amyotrophic lateral sclerosis type 4 (ALS4) is caused by mutations of senataxin ( 46 ). Senataxin has an RNA helicase domain that is similar to that of IGHMBP2 and shows significant similarity to the yeast helicase Sen1p, which is involved in the processing and maturation of tRNAs and other small noncoding RNAs ( 47 ). Senataxin is localized to the cytoplasm and in the nucleolus of neurons, suggesting a function in ribosomal RNA biogenesis and/or translation ( 48 ).

Two RNA-binding proteins are also major players in the etiopathogenesis of sporadic and familial ALS. Cytoplasmic aggregation of the TARDBP RNA-binding protein [TAR-DNA-binding protein 43 (TDP 43)] is found in ALS motor neurons ( 49 ), and mutations of TARDBP cause familial ALS ( 50 – 53 ). Mutations of FUS (fused in sarcoma), another RNA-binding protein, cause familial ALS, and like TARDBP, FUS aggregates in the cytoplasm of affected motor neurons ( 54 , 55 ).

Our study adds SMARD1 as another motor neuron degenerative disease that is likely caused by dysregulation of tRNA and rRNA pathways. Mice contain 10 tRNA Tyr genes, all of which contain introns ( 13 ). Humans contain 15 tRNA Tyr genes, 14 of which contain introns ( 13 ). The tRNA Tyr genes are unique among tRNA genes because in virtually all organisms, from yeast to humans, they contain introns ( 13 ). Another exceptional feature of tRNA Tyr is the presence of a pseudo-uridine modification of the uridine that is found in the center of the GUA anticodon in all tRNA Tyr from yeast to humans (Fig. 3 C) and which is required for proper codon recognition ( 56 ). Furthermore, this modification is dependent on the presence of the intron and leads to the hypothesis that the strong evolutionary pressure to maintain the intron in virtually all tRNA Tyr genes from yeast to humans may be to direct this pseudo-uridine modification ( 56 , 57 ). In mice, half of all tRNA Tyr genes are located in an ∼8 kb region of chromosome 13 ( 13 ). Intriguingly, we found that this cluster of the five tRNA Tyr genes is present in the BAC-27k3 transgene that rescues motor neuron degeneration of nmd mice, and tRNA Tyr physically associates with IGHMBP2. The association of IGHMBP2 with TFIIIC220 is also very interesting because TFIIIC220 binds to tRNA promoters and is an essential factor for tRNA transcription.

The finding that the ABT1, Reptin and Pontin proteins associate with IGHMBP2 is also interesting. Both Reptin and Pontin are evolutionary conserved and ubiquitously expressed ∼50 kDa proteins that contain ATPase/helicase domains and are found in multiple complexes involved in chromatin remodeling and gene transcription ( 58 – 67 ). Reptin and Pontin also associate with U3 snoRNA and are required for U3 snoRNP biogenesis. ABT1 is an ∼31 kDa protein that is evolutionary conserved from yeast ( Saccharomyces cerevisiae ) to humans. Mouse Abt1 was initially implicated in general transcription on the basis of its ability to associate with TATA-binding protein and to stimulate weakly the activity of reporter genes ( 68 ). However, subsequent protein–protein interaction studies and localization studies indicated that ABT1, like its yeast homolog, functions in ribosome biogenesis ( 41 ). The yeast homolog of ABT1 is called Esf2p (for eighteen S factor 2 protein), is an essential protein and biochemical and genetic studies have unequivocally demonstrated that it functions in ribosome biogenesis ( 41 ). Specifically, Esf2p associates with U3 snoRNP (small nucleolar ribonucleoprotein) and the 5′ ETS (external transcribed spacer) of pre-ribosomal RNA (pre-rRNA) and is essential for early pre-rRNA processing ( 41 ). U3 snoRNP contains four proteins that are common to most snoRNPs and also many additional proteins, which are required for its assembly and function. Reptin and Pontin are two of the proteins that associate with U3 snoRNP and are required for its assembly in human cells ( 22 ). The yeast homolog of Reptin is also required for the assembly of yeast snoRNPs including yeast U3 snoRNP ( 69 ). Our finding that IGHMBP2 also associates with Reptin and Pontin and the fact that the Abt1 gene, whose protein product is required for ribosome biogenesis, is contained in the 27k3 genetic modifier suggest that IGHMBP2 plays a role in ribosome biogenesis or function.

While our manuscript was under revision, Guenther et al . ( 70 ) reported that IGHMBP2 associates with ribosomes. Our findings are in support of a role of IGHMBP2 in mRNA translation, and our biochemical and genetic investigations have led to a remarkable convergence of the pathways that IGHMBP2 is likely to function, which include ribosome and tRNA function, both of which are essential for mRNA translation. Future work will focus on delineating the molecular mechanism(s) of IGHMBP2 function and its specific requirements for motor neuron maintenance.
