---
pmid: '23793891'
title: The nuclear cap-binding complex interacts with the U4/U6·U5 tri-snRNP and promotes
  spliceosome assembly in mammalian cells.
authors:
- Pabis M
- Neufeld N
- Steiner MC
- Bojic T
- Shav-Tal Y
- Neugebauer KM
journal: RNA
year: '2013'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3708526
doi: 10.1261/rna.037069.112
---

# The nuclear cap-binding complex interacts with the U4/U6·U5 tri-snRNP and promotes spliceosome assembly in mammalian cells.
**Authors:** Pabis M, Neufeld N, Steiner MC, Bojic T, Shav-Tal Y, Neugebauer KM
**Journal:** RNA (2013)
**DOI:** [10.1261/rna.037069.112](https://doi.org/10.1261/rna.037069.112)
**PMC:** [PMC3708526](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3708526/)

## Abstract

1. RNA. 2013 Aug;19(8):1054-63. doi: 10.1261/rna.037069.112. Epub 2013 Jun 21.

The nuclear cap-binding complex interacts with the U4/U6·U5 tri-snRNP and 
promotes spliceosome assembly in mammalian cells.

Pabis M(1), Neufeld N, Steiner MC, Bojic T, Shav-Tal Y, Neugebauer KM.

Author information:
(1)Max Planck Institute of Molecular Cell Biology and Genetics, 01307 Dresden, 
Germany.

The nuclear cap-binding complex (CBC) binds to the 7-methyl guanosine cap 
present on every RNA polymerase II transcript. CBC has been implicated in many 
aspects of RNA biogenesis; in addition to roles in miRNA biogenesis, 
nonsense-mediated decay, 3'-end formation, and snRNA export from the nucleus, 
CBC promotes pre-mRNA splicing. An unresolved question is how CBC participates 
in splicing. To investigate CBC's role in splicing, we used mass spectrometry to 
identify proteins that copurify with mammalian CBC. Numerous components of 
spliceosomal snRNPs were specifically detected. Among these, three U4/U6·U5 
snRNP proteins (hBrr2, hPrp4, and hPrp31) copurified with CBC in an 
RNA-independent fashion, suggesting that a significant fraction of CBC forms a 
complex with the U4/U6·U5 snRNP and that the activity of CBC might be associated 
with snRNP recruitment to pre-mRNA. To test this possibility, CBC was depleted 
from HeLa cells by RNAi. Chromatin immunoprecipitation and live-cell imaging 
assays revealed decreased cotranscriptional accumulation of U4/U6·U5 snRNPs on 
active transcription units, consistent with a requirement for CBC in 
cotranscriptional spliceosome assembly. Surprisingly, recruitment of U1 and U2 
snRNPs was also affected, indicating that RNA-mediated interactions between CBC 
and snRNPs contribute to splicing. On the other hand, CBC depletion did not 
impair snRNP biogenesis, ruling out the possibility that decreased snRNP 
recruitment was due to changes in nuclear snRNP concentration. Taken together, 
the data support a model whereby CBC promotes pre-mRNA splicing through a 
network of interactions with and among spliceosomal snRNPs during 
cotranscriptional spliceosome assembly.

DOI: 10.1261/rna.037069.112
PMCID: PMC3708526
PMID: 23793891 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONThe nuclear cap-binding complex (CBC) is a heterodimer composed of two subunits, CBP20 and CBP80. CBC binds tightly to the 7-methyl guanosine (m7G) cap, a modification added to the 5′ end of every RNA polymerase II (Pol II) transcript (Izaurralde et al. 1994, 1995). Thus, CBC binds a variety of RNA species: messenger RNAs (mRNAs), small nuclear RNAs (snRNAs), most microRNA (miRNA) precursors, and other small RNAs (Muller-McNicoll and Neugebauer 2013). CBP20 is highly conserved and interacts with the m7G cap through an RNA recognition motif. CBP80 ensures high-affinity binding of the cap by CBP20 and provides a platform for interactions with other factors (Mazza et al. 2001; Calero et al. 2002). The m7G cap is added cotranscriptionally when the nascent RNA is 20–30 nucleotides (nt) long (Rasmussen and Lis 1993). Because CBC binds to the m7G cap immediately (Visa et al. 1996; Listerman et al. 2006), it can influence the nuclear life of all RNAs transcribed by Pol II.Several aspects of mRNA biogenesis require or are stimulated by CBC: pre-mRNA splicing, mRNA 3′-end formation, and mRNA export (Izaurralde et al. 1994; Flaherty et al. 1997; Cheng et al. 2006; Narita et al. 2007). Furthermore, CBC plays roles in snRNA and miRNA biogenesis (Izaurralde et al. 1995; Laubinger et al. 2008; Gruber et al. 2009; Sabin et al. 2009). This multiplicity of CBC functions can, in many cases, be explained by interactions between CBC and regulatory partners (Muller-McNicoll and Neugebauer 2013). First, CBC promotes mRNA 3′-end formation in two ways: interactions with the mRNA polyadenylation machinery (Flaherty et al. 1997) and interactions with negative elongation factor (NELF, a regulator of histone mRNA 3′-end processing) through direct binding to its component NELF-E (Narita et al. 2007). Second, CBC interacts with the TREX complex that mediates mRNA export, by direct binding to the TREX component Aly (Cheng et al. 2006). Third, CBC binds the phosphorylated adapter for RNA export (PHAX) to export snRNAs from the nucleus during snRNP biogenesis (Ohno et al. 2000). Fourth, arsenite-resistance protein 2 (ARS2), found to promote processing of pri-miRNAs to miRNA, copurifies with both CBC and Microprocessor (Laubinger et al. 2008; Gruber et al. 2009; Sabin et al. 2009). These four examples illustrate how CBC helps to recruit factors that accomplish diverse yet specific steps in gene expression.It is clear that CBC plays a major role in splicing, yet no specific interactors have been identified to explain how. Splicing is catalyzed by the spliceosome, a dynamic complex composed of the spliceosomal small nuclear ribonucleoprotein particles (U1, U2, and U4/U6·U5 snRNPs) and a large number of additional non-snRNP splicing factors (Wahl et al. 2009). The spliceosome assembles on each intron after 5′ and 3′ splice sites (5′ and 3′ ss) have been recognized by U1 and U2 snRNPs, respectively. Replacement of the U1 snRNP by the U4/U6·U5 snRNP triggers spliceosome maturation and catalysis. Work in yeast implicated CBC in early steps of spliceosome assembly; genetic deletion of CBP20/80 genes selectively compromised the splicing of introns with nonconsensus splice sites, suggesting that CBC promotes splice site recognition (Colot et al. 1996; Lewis et al. 1996a; Fortes et al. 1999). It was proposed that CBC could facilitate definition of first exons in mammalian cells by stimulating the recruitment of U1 snRNP to the proximal 5′ ss (Lewis et al. 1996b). However, a subsequent study concluded that CBC was not required for U1 snRNP recruitment; rather, CBC promoted the release of U1 and recruitment of U4/U6·U5 snRNP during spliceosome assembly (O’Mullane and Eperon 1998). Work in yeast has implicated CBC in U1, U2, and U4/U6·U5 snRNP recruitment to nascent transcripts, lengthening the list of potential interactions with splicing components (Gornemann et al. 2005; Bragulat et al. 2010). Thus, it is currently unclear whether CBC acts on one or more snRNPs and whether observed effects are direct or indirect.We sought to address the role of CBC in splicing, starting with CBC immunopurification from cell extracts and mass spectrometry. We considered two scenarios likely: first, CBC might interact with a single protein component of one or more snRNP(s), directly recruiting it to the pre-mRNA; second, CBC might bind to a non-snRNP splicing factor capable of recruiting spliceosomal snRNPs. Instead, numerous snRNP components were detected; validation experiments revealed RNA-independent copurification of CBC with three U4/U6·U5 snRNP proteins, indicative of U4/U6·U5 snRNP association with CBC through direct or indirect protein–protein interactions. We then focused on how CBC mediates snRNP interactions with pre-mRNA in vivo, combining spliceosome assembly assays and live-cell fluorescence imaging in the context of CBC knockdown. Our results suggest that CBC promotes cotranscriptional spliceosome assembly and splicing through links to the U4/U6·U5 tri-snRNP.

INTRODUCTION

The nuclear cap-binding complex (CBC) is a heterodimer composed of two subunits, CBP20 and CBP80. CBC binds tightly to the 7-methyl guanosine (m7G) cap, a modification added to the 5′ end of every RNA polymerase II (Pol II) transcript (Izaurralde et al. 1994, 1995). Thus, CBC binds a variety of RNA species: messenger RNAs (mRNAs), small nuclear RNAs (snRNAs), most microRNA (miRNA) precursors, and other small RNAs (Muller-McNicoll and Neugebauer 2013). CBP20 is highly conserved and interacts with the m7G cap through an RNA recognition motif. CBP80 ensures high-affinity binding of the cap by CBP20 and provides a platform for interactions with other factors (Mazza et al. 2001; Calero et al. 2002). The m7G cap is added cotranscriptionally when the nascent RNA is 20–30 nucleotides (nt) long (Rasmussen and Lis 1993). Because CBC binds to the m7G cap immediately (Visa et al. 1996; Listerman et al. 2006), it can influence the nuclear life of all RNAs transcribed by Pol II.

Several aspects of mRNA biogenesis require or are stimulated by CBC: pre-mRNA splicing, mRNA 3′-end formation, and mRNA export (Izaurralde et al. 1994; Flaherty et al. 1997; Cheng et al. 2006; Narita et al. 2007). Furthermore, CBC plays roles in snRNA and miRNA biogenesis (Izaurralde et al. 1995; Laubinger et al. 2008; Gruber et al. 2009; Sabin et al. 2009). This multiplicity of CBC functions can, in many cases, be explained by interactions between CBC and regulatory partners (Muller-McNicoll and Neugebauer 2013). First, CBC promotes mRNA 3′-end formation in two ways: interactions with the mRNA polyadenylation machinery (Flaherty et al. 1997) and interactions with negative elongation factor (NELF, a regulator of histone mRNA 3′-end processing) through direct binding to its component NELF-E (Narita et al. 2007). Second, CBC interacts with the TREX complex that mediates mRNA export, by direct binding to the TREX component Aly (Cheng et al. 2006). Third, CBC binds the phosphorylated adapter for RNA export (PHAX) to export snRNAs from the nucleus during snRNP biogenesis (Ohno et al. 2000). Fourth, arsenite-resistance protein 2 (ARS2), found to promote processing of pri-miRNAs to miRNA, copurifies with both CBC and Microprocessor (Laubinger et al. 2008; Gruber et al. 2009; Sabin et al. 2009). These four examples illustrate how CBC helps to recruit factors that accomplish diverse yet specific steps in gene expression.

It is clear that CBC plays a major role in splicing, yet no specific interactors have been identified to explain how. Splicing is catalyzed by the spliceosome, a dynamic complex composed of the spliceosomal small nuclear ribonucleoprotein particles (U1, U2, and U4/U6·U5 snRNPs) and a large number of additional non-snRNP splicing factors (Wahl et al. 2009). The spliceosome assembles on each intron after 5′ and 3′ splice sites (5′ and 3′ ss) have been recognized by U1 and U2 snRNPs, respectively. Replacement of the U1 snRNP by the U4/U6·U5 snRNP triggers spliceosome maturation and catalysis. Work in yeast implicated CBC in early steps of spliceosome assembly; genetic deletion of CBP20/80 genes selectively compromised the splicing of introns with nonconsensus splice sites, suggesting that CBC promotes splice site recognition (Colot et al. 1996; Lewis et al. 1996a; Fortes et al. 1999). It was proposed that CBC could facilitate definition of first exons in mammalian cells by stimulating the recruitment of U1 snRNP to the proximal 5′ ss (Lewis et al. 1996b). However, a subsequent study concluded that CBC was not required for U1 snRNP recruitment; rather, CBC promoted the release of U1 and recruitment of U4/U6·U5 snRNP during spliceosome assembly (O’Mullane and Eperon 1998). Work in yeast has implicated CBC in U1, U2, and U4/U6·U5 snRNP recruitment to nascent transcripts, lengthening the list of potential interactions with splicing components (Gornemann et al. 2005; Bragulat et al. 2010). Thus, it is currently unclear whether CBC acts on one or more snRNPs and whether observed effects are direct or indirect.

We sought to address the role of CBC in splicing, starting with CBC immunopurification from cell extracts and mass spectrometry. We considered two scenarios likely: first, CBC might interact with a single protein component of one or more snRNP(s), directly recruiting it to the pre-mRNA; second, CBC might bind to a non-snRNP splicing factor capable of recruiting spliceosomal snRNPs. Instead, numerous snRNP components were detected; validation experiments revealed RNA-independent copurification of CBC with three U4/U6·U5 snRNP proteins, indicative of U4/U6·U5 snRNP association with CBC through direct or indirect protein–protein interactions. We then focused on how CBC mediates snRNP interactions with pre-mRNA in vivo, combining spliceosome assembly assays and live-cell fluorescence imaging in the context of CBC knockdown. Our results suggest that CBC promotes cotranscriptional spliceosome assembly and splicing through links to the U4/U6·U5 tri-snRNP.

RESULTS AND DISCUSSIONIdentification of proteins that copurify with CBCTo identify protein interaction partners of CBC, a transgenic HeLa cell line stably expressing CBP20 with a C-terminal GFP-tag was established, using bacterial artificial chromosome (BAC) recombineering. BAC transgenesis offers the advantage that the “third allele” encoding the tagged protein of interest is expressed at physiological levels, due to usage of endogenous regulatory elements, and the GFP tag is effective for immunopurification and mass spectrometry (Poser et al. 2008). CBP20-GFP correctly localizes to the nucleus and binds both m7G caps and CBP80 (Supplemental Fig. 1A; Pabis et al. 2010). Cell lysates were treated with benzonase to degrade DNA and RNA before immunopurification, thereby minimizing detection of proteins linked to CBC through nucleic acids (Cheeseman and Desai 2005; Pabis et al. 2010). Table 1 lists proteins specifically identified from both gel slice and shotgun preparations (see Supplemental Tables 1 and 2 for entire data sets). As expected, robustly detected proteins copurifying with CBP20 were CBP80, ARS2, PHAX, importins, and components of the recently characterized nuclear exosome targeting (NEXT) complex (Lubas et al. 2011).TABLE 1.Protein interactors of CBP20-GFPThe largest group of factors copurifying with CBP20 included 15 U1, U2, U5, U4/U6, and U4/U6·U5 snRNP-specific proteins. Identification of these proteins was highly specific, because many of the approximately 200 proteins present in the mammalian spliceosome (Wahl et al. 2009) were not detected; moreover, highly abundant non-snRNP splicing factors (e.g., SR proteins) were absent. Despite the fact that the U4-, U5-, and U6-containing snRNPs are 10 times less abundant than U1 and U2 snRNPs (Yu et al. 1999), the number of peptides representing U4/U6·U5 snRNP components was remarkably high (Table 1). Thus, the data suggest that CBC associates with all spliceosomal snRNPs, among them U4/U6·U5 snRNP in particular.We focused on CBC interactions with snRNPs for validation by coimmunoprecipitation, using transgenic cell lines harboring GFP-tagged candidates (see Supplemental Fig. 1A–C; Sapra et al. 2009). It is possible that many proteins that coimmunoprecipitate with CBC are linked indirectly through either (pre-)mRNA or through the snRNAs contained in snRNPs. RNase digestion is able to remove both species of RNA and thereby abolish RNA-mediated interactions; for example, we show that U1 snRNP is disrupted by RNase (Supplemental Fig. 1E; Sapra et al. 2009). Therefore, cell lysates were treated with or without RNase A, and copurified material was probed for untagged CBP20 and CBP80 by Western blotting (Fig. 1, upper panel; Supplemental Fig. 1D). As expected, CBP20 associations with CBP80 and ARS2 were RNase insensitive (Supplemental Fig. 1D). Similarly, CBC coimmunoprecipitated with the NEXT complex component ZC3H18/NHN1 from both mock and RNase-treated cell extracts (Fig. 1). This observation, together with the report that NHN1 associates with mRNPs in a CBC-dependent manner (Merz et al. 2007), points to a potential role of CBC in recruiting NHN1 to the mRNP in vivo.FIGURE 1.Validation of splicing factor binding to CBC. (Upper panel) Extracts from transgenic HeLa cells harboring GFP-tagged splicing factors, indicated left of the panel, were incubated with or without RNase A and subjected to immunoprecipitation with α-GFP. 0.6% of input (Inp) and 33% of the immunoprecipitate were analyzed by Western blot, using α-CBP20 and α-CBP80. Nonspecific background was assessed by immunoprecipitation with nonimmune IgG (IgG). See Supplemental Figure 1, D and E, for additional data and evidence that RNase treatment disrupted snRNPs. (Lower panel) Extracts from the same transgenic HeLa cells treated with and without α-amanitin to inhibit Pol II transcription, as indicated. All experiments were performed two to four times each, and representative gels are shown.It was possible to validate the interactions of U1, U2, and U4/U6·U5 snRNPs with CBC, although many snRNP proteins appeared to be indirectly linked by RNA (Table 1). Specifically, GFP-tagged U1 snRNP (U1-70K, U1A), U2 snRNP (SF3A1), and U4/U6·U5 snRNP (U5-200K, U4/U6-60K, U4/U6-61K, U5-116K, and Prp8) were all able to coimmunoprecipitate CBP20 and CBP80 in the absence of RNase (Fig. 1, upper panel; Supplemental Fig. 1D,E). However, coimmunoprecipitation of U2AF65, U1-70K, U1-A, and SF3A1 was strongly reduced by RNase treatment, suggesting linkage through snRNA or pre-mRNA. In contrast, U5-200K (the human homolog of yeast Brr2, an RNA helicase), U4/U6-60K (hPrp4, a WD-repeat protein), and U4/U6-61K (hPrp31, a U4 snRNA-binding and Nop domain-containing protein) remained bound after RNase digestion, suggesting RNA-independent association of the U4/U6·U5 snRNP with CBC. To determine whether active transcription and splicing was strictly required for snRNP interactions with CBC, we treated cells with α-amanitin before cell lysis. Transcription inhibition did not disrupt interactions with these proteins. The insensitivity of U1-70K to α-amanitin (Fig. 1, lower panel) indicates that the loss of U1-70K upon RNase treatment (Fig. 1, upper panel) is indeed due to disruption of the U1 snRNP (Supplemental Fig. 1E). The continued detection of both CBP20 and CBP80 in immunoprecipitates of U1 and U4/U6·U5 snRNP proteins after transcription inhibition suggests that spliceosomal snRNPs may be constitutively associated with CBC. Alternatively, these interactions may be maintained by remaining unspliced RNA in the extract. The transcription- and RNA-independent interactions between CBC and snRNPs may explain why CBC is detected in purified spliceosomes assembled on pre-mRNAs that lack m7G caps (Fabrizio et al. 2009).Taken together, this series of validation experiments indicates that CBC interacts in an RNA-independent manner with the U4/U6·U5 snRNP (Table 1). There are no amino acid sequences or protein motifs shared among these partners, making it difficult to speculate on whether CBC association with these proteins is due to direct binding. Moreover, we cannot distinguish between binding to CBP20 and CBP80 in these experiments. In this study, we sought to understand the role of CBC in splicing, which requires spliceosome assembly from mature snRNPs. Therefore, we aimed to pursue the role of CBC in snRNP recruitment in vivo, rather than focus on CBC interactions with individual proteins. To do so, we coupled CBC depletion by RNAi with specific assays that interrogate the interactions of whole spliceosomal snRNPs with pre-mRNA.SnRNP levels are unaffected by CBC depletionTreatment of HeLa cells with CBP80-specific interfering RNAs by two independent methods was effective, leading to an ∼80% reduction in CBP80 mRNA and protein levels (Fig. 2A; Supplemental Fig. 2B,C). Importantly, CBP80 depletion also led to an ∼50% reduction in CBP20 protein levels, resulting in an overall efficient knockdown of the whole CBC and indicating that CBP20 stability requires CBP80. This finding and the observed impairment of cell proliferation (Fig. 2A; Supplemental Fig. 2A) are consistent with a prior study (Narita et al. 2007). Furthermore, expression levels of seven distinct splicing factors—including proteins copurifying with CBC, U1-70K, SF3A1, Prp8, and U5-116K—were unchanged upon CBC knockdown (Supplemental Fig. 2D), indicating that CBP80 depletion does not destabilize splicing factors in general.FIGURE 2.Efficient CBC knockdown does not affect snRNP levels. HeLa cells transduced with retroviral vectors expressing CBP80 shRNA or without shRNA (Control) were assayed on day 6. (A) Assessment of CBC depletion through semiquantitative Western blotting (left panel) of CBP80, CBP20, and GAPDH after CBP80 knockdown (KD). Decreasing amounts of the same lysate were loaded. Comparison of changes in CBP80 and CBP20 RNA and protein levels (right panel). Band intensities were measured as integrated densities from Western blots and normalized to GAPDH. RNA levels were determined by RT-qPCR and normalized to 18S rRNA. N ≥ 8 different knockdowns. Error bars are the SEM. The changes are statistically significant (P < 0.005), as determined by the Student’s t-test. (B) Immunoprecipitation of snRNPs with α-TMG (K121), α-Sm proteins (Y12), and α-SART3. RNA was extracted, resolved on a 10% urea gel, and detected by Northern blot. A longer exposure of α-TMG IP lanes is shown.Because these CBC depletion conditions were designed to test spliceosome assembly and snRNP dynamics in downstream assays, we were concerned that snRNP levels might be affected. CBC and PHAX binding to nuclear m7G-capped pre-snRNAs mediates their translocation to the cytoplasm, where Sm ring assembly and snRNA cap tri-methylation occur (Izaurralde et al. 1995; Ohno et al. 2000; Muller-McNicoll and Neugebauer 2013). If CBC knockdown were to impair snRNA export, reductions in cytoplasmic snRNP maturation steps and consequently cellular snRNP levels could occur. However, Northern blotting of total RNA revealed that levels of U1, U2, U4, and U6 snRNAs were similar among control and CBC-depleted cells (Fig. 2B). Importantly, levels of assembled snRNPs containing Sm proteins and tri-methyl guanosine (TMG) caps were unaffected. Quantitation revealed no significant differences in U1 and U2 levels between knockdown and control cells (Supplemental Fig. 3A). Because U4 and U6 snRNAs were less well detected by Northern blot, U4 and U6 snRNAs were immunoprecipitated with α-SART3, specific for the U4/U6 snRNP (Stanek et al. 2003). Detection of U4 and U6 snRNAs was robust and unchanged by CBC depletion (Fig. 2B; Supplemental Fig. 3A). U5 snRNA levels were also not affected (Supplemental Fig. 3B). This surprising lack of effect on snRNP levels led us to verify CBC and PHAX binding to pre-snRNAs; both CBC and PHAX pulled down pre-snRNAs, as expected (Supplemental Fig. 3C). Finally, metabolic labeling was performed to interrogate the degree of snRNA 5′-cap tri-methylation and showed no change upon CBC depletion (Supplemental Fig. 3D). We conclude that residual levels of CBC must be sufficient to support snRNA biogenesis in depleted cells. Because total spliceosomal snRNP levels were unaffected, it was possible to investigate a distinct role for CBC in splicing.CBC depletion reduces cotranscriptional splicingWe selected the FOS gene to investigate the role of CBC in cotranscriptional splicing and spliceosome assembly for the following reasons. First, FOS transcripts are efficiently spliced cotranscriptionally (Listerman et al. 2006; Vargas et al. 2011). Second, FOS is strongly inducible (Piechaczyk and Blanchard 1994), permitting measurement of spliced and unspliced transcripts during the period of CBC depletion with minimal background from previously synthesized mRNA. Third, robust FOS transcription upon induction permits detection of splicing factor recruitment to the transcription unit by chromatin immunoprecipitation (ChIP) (Listerman et al. 2006; Sapra et al. 2009). Fourth, FOS is characterized by a relatively simple exon–intron structure (Fig. 3A). These features make FOS an ideal gene to study the effects of CBC depletion on splicing.FIGURE 3.CBC depletion reduces cotranscriptional spliceosome assembly and splicing. HeLa cells transduced with control and α-CBP80 shRNA retroviral vectors were subjected to RNA isolation, ChRIP, or ChIP on day 6. (A) Schematic representation of FOS pre-mRNA and mRNA, showing the location of qPCR amplicons detecting RNA with either spliced or unspliced intron 1 (blue) and intron 3 (red). Total RNA (left panel) was reverse-transcribed with an oligo(dT) primer and the relative abundance of spliced to unspliced poly(A)+ RNA assessed by qPCR. Nascent RNA (right panel) was isolated by ChRIP from an α-AcH4 IP and reverse-transcribed, using a primer in intron 2 (for intron 1 splicing) or in exon 4 (for intron 3 splicing). qPCR was conducted to assess the relative abundance of spliced to unspliced RNA. n ≥ 4; error bars are the SEM. (B) Schematic representation of the FOS gene. (Gray lines) The location of qPCR amplicons used in ChIP. The following antibodies were used: α-CBP80, α-Pol II-CTD (4H8), α-U2AF-65 (MC3), α-U5-116K, and α-GFP in case of transgenic cell lines expressing U1-70K-GFP or SF3A1-GFP. All values were calculated as percent input, normalized to nonimmune ChIP, and presented as fold enrichment over an intergenic region. In Pol II ChIP, normalization to intergenic was omitted due to absence of background. N ≥ 3; error bars are the SEM. Significant differences between knockdown and control determined by the Student’s t-test: (*) P < 0.05, (**) P < 0.01.As a first step, we used ChIP to quantitate the impact of CBP80 RNAi on CBC levels at the FOS transcription unit. CBP80 ChIP confirmed that the ∼80% decrease in cellular CBP80 protein corresponded to the reduction in CBP80 levels along the FOS gene (Fig. 3B). Acetylated histone H4 (AcH4) and Pol II levels were unchanged by CBC depletion (Fig. 3B; Supplemental Fig. 4B), suggesting that FOS transcription is not CBC dependent. Next, we examined steady-state intron removal in FOS RNA by RT-qPCR, which revealed that the levels of unspliced FOS introns 1 and 3 increased by twofold upon CBC knockdown (Fig. 3A). In parallel, a decrease in total FOS mRNA was observed (Supplemental Fig. 4A). The levels of another short-lived, spliced mRNA, MYC, were also significantly reduced, and splicing of the first intron was inhibited by ∼30% (data not shown). At the same time, mRNAs with long half-lives, such as ACTB, were unaffected by CBC depletion (Supplemental Fig. 4A), indicating that CBC depletion inhibits but does not abolish splicing activity. Another indication of cellular splicing activity is morphological: Nuclear speckles are normally interconnected, irregularly shaped nuclear compartments where splicing factors concentrate. Upon splicing inhibition with spliceostatin A, speckles become “rounded up” (Kaida et al. 2007). Supplemental Figure 5 shows that nuclear speckles also displayed a rounded-up morphology when cells were depleted of CBC, consistent with the expected overall splicing inhibition (see the Introduction). To determine the effect of CBC depletion on cotranscriptional splicing, nascent FOS RNA was retrieved by chromatin RNA immunoprecipitation (ChRIP), in which cross-linked active chromatin is immunoprecipitated with antibodies to AcH4 (Listerman et al. 2006). Figure 3A shows that cotranscriptional splicing of FOS introns 1 and 3 was impaired upon CBC depletion, similar to total pre-mRNA splicing. This finding indicates that splicing inhibition is not likely due to effects on NEXT complex activity, because NEXT would not be expected to destabilize nascent RNA. We conclude that the absence of CBC leads to a decreased efficiency of cotranscriptional intron removal.CBC depletion impairs cotranscriptional spliceosome assemblyKnowing that CBC is required for efficient nascent FOS RNA splicing, we sought to determine the step at which spliceosome assembly is altered in the absence of CBC. We therefore investigated the recruitment profiles of snRNPs, using a cotranscriptional spliceosome assembly assay based on ChIP (Gornemann et al. 2005; Listerman et al. 2006; Sapra et al. 2009). SnRNP-specific proteins were used as proxies for each snRNP: U1 (U1-70K), U2 (SF3A1), and U4/U6·U5 (U5-116K). GFP-tagged snRNP proteins expressed stably from BACs were incorporated into snRNPs and correctly localized (Supplemental Fig. 1A–C).The levels and profiles of the U1, U2, and U4/U6·U5 snRNPs as well as U2AF65 were determined at four positions within exons and introns along the length of FOS (Fig. 3B). Upon CBC knockdown, accumulation of all three snRNPs was decreased (Fig. 3B). These effects are specific, because U2AF65 accumulation was unaffected. U5-116K accumulation was reduced significantly at the beginning as well as the end of FOS (Fig. 3B). The decrease in U4/U6·U5 snRNP recruitment is consistent with the notion that CBC associates with protein components of the tri-snRNP, namely, U5-200K, U4/U6-60K, and U4/U6-61K (see Fig. 1; Table 1) to assist spliceosome assembly. CBC depletion also caused marked reductions in U1 and U2 snRNP recruitment with accumulation of U1-70K-GFP 50% reduced at all positions (Fig. 3B). Because our results point to CBC interactions with U4/U6·U5 and not U1 snRNP proteins (Fig. 1; Table 1), these data suggest that U4/U6·U5 snRNP association with CBC could assist U1 snRNP recruitment. Prior studies reported U5 snRNP interactions with the 5′ ss early in spliceosome assembly as well as U1 snRNP recruitment in the context of a multi-snRNP-containing complex (Wyatt et al. 1992; Malca et al. 2003). Thus, our data support the notion that CBC helps recruit U1 snRNP to the first 5′ ss (Lewis et al. 1996b), not by direct interactions with U1 but instead indirectly through the U4/U6·U5 snRNP. Moreover, the U1 snRNP is retained on multiple intron-containing pre-mRNAs after the first splicing event, rendering subsequent splicing more efficient (Crabb et al. 2010). Indeed, splicing of both FOS introns 1 and 3 was CBC dependent, suggesting—together with the observed snRNP profiles—that CBC may help retain the U1 snRNP on nascent RNA until all introns are removed. We conclude that proper cotranscriptional accumulation of U1, U2, and U4/U6·U5 snRNPs is dependent on CBC in vivo.CBC promotes U1 and U5 snRNP interactions with pre-mRNAThe above data suggest that CBC’s overall role in FOS splicing is to enhance spliceosomal snRNP interactions with nascent RNA through its association with U4/U6·U5 snRNPs. To test this proposal more broadly, we used a live-cell imaging strategy to analyze the dynamic interaction of snRNPs with pre-mRNA both globally and at a different test gene. We addressed total cellular pre-mRNA by monitoring snRNP dynamics in the nucleoplasm; complementary to this, we used an integrated, inducible transcription unit (E3) containing three exons and two introns of the β-globin gene (Fig. 4A; Huranova et al. 2010; Pabis et al. 2010; Brody et al. 2011). E3 mRNA contains the MS2 binding site in the 3′ UTR, such that active transcription sites can be monitored in living cells with a fluorescently tagged MS2-binding protein. BACs harboring U1-70K-GFP, U2AF65-GFP, and Prp8-GFP were stably integrated into E3 cells, and fluorescence imaging revealed that U2AF65, U1, and U5 snRNPs were detectable in speckles and nucleoplasm, as expected (Fig. 4B). Importantly, each of these splicing factors accumulated at the E3 gene during active transcription, as seen by colocalization with E3 nascent RNAs at the transcription site (Fig. 4B).FIGURE 4.CBC depletion reduces snRNP interactions with active transcription sites. (A) Diagram of the E3 construct stably integrated into U2OS cells. Expression is driven by a minimal CMV promoter (P) under control of the tetracycline response element and induced in the presence of doxycycline (dox) by the transactivator rtTA. The gene contains three β-globin exons (E1–E3) and a CFP coding sequence, 18 MS2 repeats, a polyadenylation cleavage signal, and a transcription terminator (T). (B) Imaging of nascent RNA at the E3 transcription unit in fixed cells by RNA FISH (Cy3-labeled probe to the MS2 sequence repeats; red) together with the stably integrated U1 snRNP (U1-70K-GFP; green), U2AF65-GFP, or U5 snRNP proteins (Prp8-GFP). Scale bars, 5 μm. (C) FRAP curves show recovery of U1-70K-GFP, U2AF65-GFP, and Prp8-GFP fluorescence at bleached spots placed in nucleoplasm, speckles, or at the E3 transcription site visualized by MS2BP-mCherry. Three independent experiments were performed; in each 10–20 cells were tested. Error bars represent the SEM. For U1-70K-GFP and Prp8-GFP, the differences between recovery curves in control and CBC knockdown cells are statistically significant, as established using the Mann–Whitney test.FRAP experiments revealed CBC-dependent dynamics among U1 and U4/U6·U5 snRNPs at both the E3 transcription site and in the nucleoplasm, where endogenous splicing occurs (Fig. 4C). In both locations, significantly faster recovery of both U1 and U5 snRNPs was observed upon CBC depletion, indicating that each snRNP was retained at sites of transcription for a shorter period of time. In contrast, mobility of U1 and U5 snRNPs in nuclear speckles was unchanged, suggesting that snRNP accumulation in speckles does not depend on interactions with CBC. No effects on U2AF65 recovery or mobility were observed at any location (Fig. 4C), in agreement with the ChIP data (Fig. 3B). Because the mobility of nucleoplasmic snRNPs is determined by association with pre-mRNA (Huranova et al. 2010), these data indicate that CBC knockdown weakens U1 and U4/U6·U5 snRNP interactions with endogenous pre-mRNAs in living cells. This finding is consistent with morphological evidence that splicing is globally inhibited (Supplemental Fig. 5).

RESULTS AND DISCUSSION

Identification of proteins that copurify with CBCTo identify protein interaction partners of CBC, a transgenic HeLa cell line stably expressing CBP20 with a C-terminal GFP-tag was established, using bacterial artificial chromosome (BAC) recombineering. BAC transgenesis offers the advantage that the “third allele” encoding the tagged protein of interest is expressed at physiological levels, due to usage of endogenous regulatory elements, and the GFP tag is effective for immunopurification and mass spectrometry (Poser et al. 2008). CBP20-GFP correctly localizes to the nucleus and binds both m7G caps and CBP80 (Supplemental Fig. 1A; Pabis et al. 2010). Cell lysates were treated with benzonase to degrade DNA and RNA before immunopurification, thereby minimizing detection of proteins linked to CBC through nucleic acids (Cheeseman and Desai 2005; Pabis et al. 2010). Table 1 lists proteins specifically identified from both gel slice and shotgun preparations (see Supplemental Tables 1 and 2 for entire data sets). As expected, robustly detected proteins copurifying with CBP20 were CBP80, ARS2, PHAX, importins, and components of the recently characterized nuclear exosome targeting (NEXT) complex (Lubas et al. 2011).TABLE 1.Protein interactors of CBP20-GFPThe largest group of factors copurifying with CBP20 included 15 U1, U2, U5, U4/U6, and U4/U6·U5 snRNP-specific proteins. Identification of these proteins was highly specific, because many of the approximately 200 proteins present in the mammalian spliceosome (Wahl et al. 2009) were not detected; moreover, highly abundant non-snRNP splicing factors (e.g., SR proteins) were absent. Despite the fact that the U4-, U5-, and U6-containing snRNPs are 10 times less abundant than U1 and U2 snRNPs (Yu et al. 1999), the number of peptides representing U4/U6·U5 snRNP components was remarkably high (Table 1). Thus, the data suggest that CBC associates with all spliceosomal snRNPs, among them U4/U6·U5 snRNP in particular.We focused on CBC interactions with snRNPs for validation by coimmunoprecipitation, using transgenic cell lines harboring GFP-tagged candidates (see Supplemental Fig. 1A–C; Sapra et al. 2009). It is possible that many proteins that coimmunoprecipitate with CBC are linked indirectly through either (pre-)mRNA or through the snRNAs contained in snRNPs. RNase digestion is able to remove both species of RNA and thereby abolish RNA-mediated interactions; for example, we show that U1 snRNP is disrupted by RNase (Supplemental Fig. 1E; Sapra et al. 2009). Therefore, cell lysates were treated with or without RNase A, and copurified material was probed for untagged CBP20 and CBP80 by Western blotting (Fig. 1, upper panel; Supplemental Fig. 1D). As expected, CBP20 associations with CBP80 and ARS2 were RNase insensitive (Supplemental Fig. 1D). Similarly, CBC coimmunoprecipitated with the NEXT complex component ZC3H18/NHN1 from both mock and RNase-treated cell extracts (Fig. 1). This observation, together with the report that NHN1 associates with mRNPs in a CBC-dependent manner (Merz et al. 2007), points to a potential role of CBC in recruiting NHN1 to the mRNP in vivo.FIGURE 1.Validation of splicing factor binding to CBC. (Upper panel) Extracts from transgenic HeLa cells harboring GFP-tagged splicing factors, indicated left of the panel, were incubated with or without RNase A and subjected to immunoprecipitation with α-GFP. 0.6% of input (Inp) and 33% of the immunoprecipitate were analyzed by Western blot, using α-CBP20 and α-CBP80. Nonspecific background was assessed by immunoprecipitation with nonimmune IgG (IgG). See Supplemental Figure 1, D and E, for additional data and evidence that RNase treatment disrupted snRNPs. (Lower panel) Extracts from the same transgenic HeLa cells treated with and without α-amanitin to inhibit Pol II transcription, as indicated. All experiments were performed two to four times each, and representative gels are shown.It was possible to validate the interactions of U1, U2, and U4/U6·U5 snRNPs with CBC, although many snRNP proteins appeared to be indirectly linked by RNA (Table 1). Specifically, GFP-tagged U1 snRNP (U1-70K, U1A), U2 snRNP (SF3A1), and U4/U6·U5 snRNP (U5-200K, U4/U6-60K, U4/U6-61K, U5-116K, and Prp8) were all able to coimmunoprecipitate CBP20 and CBP80 in the absence of RNase (Fig. 1, upper panel; Supplemental Fig. 1D,E). However, coimmunoprecipitation of U2AF65, U1-70K, U1-A, and SF3A1 was strongly reduced by RNase treatment, suggesting linkage through snRNA or pre-mRNA. In contrast, U5-200K (the human homolog of yeast Brr2, an RNA helicase), U4/U6-60K (hPrp4, a WD-repeat protein), and U4/U6-61K (hPrp31, a U4 snRNA-binding and Nop domain-containing protein) remained bound after RNase digestion, suggesting RNA-independent association of the U4/U6·U5 snRNP with CBC. To determine whether active transcription and splicing was strictly required for snRNP interactions with CBC, we treated cells with α-amanitin before cell lysis. Transcription inhibition did not disrupt interactions with these proteins. The insensitivity of U1-70K to α-amanitin (Fig. 1, lower panel) indicates that the loss of U1-70K upon RNase treatment (Fig. 1, upper panel) is indeed due to disruption of the U1 snRNP (Supplemental Fig. 1E). The continued detection of both CBP20 and CBP80 in immunoprecipitates of U1 and U4/U6·U5 snRNP proteins after transcription inhibition suggests that spliceosomal snRNPs may be constitutively associated with CBC. Alternatively, these interactions may be maintained by remaining unspliced RNA in the extract. The transcription- and RNA-independent interactions between CBC and snRNPs may explain why CBC is detected in purified spliceosomes assembled on pre-mRNAs that lack m7G caps (Fabrizio et al. 2009).Taken together, this series of validation experiments indicates that CBC interacts in an RNA-independent manner with the U4/U6·U5 snRNP (Table 1). There are no amino acid sequences or protein motifs shared among these partners, making it difficult to speculate on whether CBC association with these proteins is due to direct binding. Moreover, we cannot distinguish between binding to CBP20 and CBP80 in these experiments. In this study, we sought to understand the role of CBC in splicing, which requires spliceosome assembly from mature snRNPs. Therefore, we aimed to pursue the role of CBC in snRNP recruitment in vivo, rather than focus on CBC interactions with individual proteins. To do so, we coupled CBC depletion by RNAi with specific assays that interrogate the interactions of whole spliceosomal snRNPs with pre-mRNA.

Identification of proteins that copurify with CBC

To identify protein interaction partners of CBC, a transgenic HeLa cell line stably expressing CBP20 with a C-terminal GFP-tag was established, using bacterial artificial chromosome (BAC) recombineering. BAC transgenesis offers the advantage that the “third allele” encoding the tagged protein of interest is expressed at physiological levels, due to usage of endogenous regulatory elements, and the GFP tag is effective for immunopurification and mass spectrometry (Poser et al. 2008). CBP20-GFP correctly localizes to the nucleus and binds both m7G caps and CBP80 (Supplemental Fig. 1A; Pabis et al. 2010). Cell lysates were treated with benzonase to degrade DNA and RNA before immunopurification, thereby minimizing detection of proteins linked to CBC through nucleic acids (Cheeseman and Desai 2005; Pabis et al. 2010). Table 1 lists proteins specifically identified from both gel slice and shotgun preparations (see Supplemental Tables 1 and 2 for entire data sets). As expected, robustly detected proteins copurifying with CBP20 were CBP80, ARS2, PHAX, importins, and components of the recently characterized nuclear exosome targeting (NEXT) complex (Lubas et al. 2011).

Protein interactors of CBP20-GFP

The largest group of factors copurifying with CBP20 included 15 U1, U2, U5, U4/U6, and U4/U6·U5 snRNP-specific proteins. Identification of these proteins was highly specific, because many of the approximately 200 proteins present in the mammalian spliceosome (Wahl et al. 2009) were not detected; moreover, highly abundant non-snRNP splicing factors (e.g., SR proteins) were absent. Despite the fact that the U4-, U5-, and U6-containing snRNPs are 10 times less abundant than U1 and U2 snRNPs (Yu et al. 1999), the number of peptides representing U4/U6·U5 snRNP components was remarkably high (Table 1). Thus, the data suggest that CBC associates with all spliceosomal snRNPs, among them U4/U6·U5 snRNP in particular.

We focused on CBC interactions with snRNPs for validation by coimmunoprecipitation, using transgenic cell lines harboring GFP-tagged candidates (see Supplemental Fig. 1A–C; Sapra et al. 2009). It is possible that many proteins that coimmunoprecipitate with CBC are linked indirectly through either (pre-)mRNA or through the snRNAs contained in snRNPs. RNase digestion is able to remove both species of RNA and thereby abolish RNA-mediated interactions; for example, we show that U1 snRNP is disrupted by RNase (Supplemental Fig. 1E; Sapra et al. 2009). Therefore, cell lysates were treated with or without RNase A, and copurified material was probed for untagged CBP20 and CBP80 by Western blotting (Fig. 1, upper panel; Supplemental Fig. 1D). As expected, CBP20 associations with CBP80 and ARS2 were RNase insensitive (Supplemental Fig. 1D). Similarly, CBC coimmunoprecipitated with the NEXT complex component ZC3H18/NHN1 from both mock and RNase-treated cell extracts (Fig. 1). This observation, together with the report that NHN1 associates with mRNPs in a CBC-dependent manner (Merz et al. 2007), points to a potential role of CBC in recruiting NHN1 to the mRNP in vivo.

Validation of splicing factor binding to CBC. (Upper panel) Extracts from transgenic HeLa cells harboring GFP-tagged splicing factors, indicated left of the panel, were incubated with or without RNase A and subjected to immunoprecipitation with α-GFP. 0.6% of input (Inp) and 33% of the immunoprecipitate were analyzed by Western blot, using α-CBP20 and α-CBP80. Nonspecific background was assessed by immunoprecipitation with nonimmune IgG (IgG). See Supplemental Figure 1, D and E, for additional data and evidence that RNase treatment disrupted snRNPs. (Lower panel) Extracts from the same transgenic HeLa cells treated with and without α-amanitin to inhibit Pol II transcription, as indicated. All experiments were performed two to four times each, and representative gels are shown.

It was possible to validate the interactions of U1, U2, and U4/U6·U5 snRNPs with CBC, although many snRNP proteins appeared to be indirectly linked by RNA (Table 1). Specifically, GFP-tagged U1 snRNP (U1-70K, U1A), U2 snRNP (SF3A1), and U4/U6·U5 snRNP (U5-200K, U4/U6-60K, U4/U6-61K, U5-116K, and Prp8) were all able to coimmunoprecipitate CBP20 and CBP80 in the absence of RNase (Fig. 1, upper panel; Supplemental Fig. 1D,E). However, coimmunoprecipitation of U2AF65, U1-70K, U1-A, and SF3A1 was strongly reduced by RNase treatment, suggesting linkage through snRNA or pre-mRNA. In contrast, U5-200K (the human homolog of yeast Brr2, an RNA helicase), U4/U6-60K (hPrp4, a WD-repeat protein), and U4/U6-61K (hPrp31, a U4 snRNA-binding and Nop domain-containing protein) remained bound after RNase digestion, suggesting RNA-independent association of the U4/U6·U5 snRNP with CBC. To determine whether active transcription and splicing was strictly required for snRNP interactions with CBC, we treated cells with α-amanitin before cell lysis. Transcription inhibition did not disrupt interactions with these proteins. The insensitivity of U1-70K to α-amanitin (Fig. 1, lower panel) indicates that the loss of U1-70K upon RNase treatment (Fig. 1, upper panel) is indeed due to disruption of the U1 snRNP (Supplemental Fig. 1E). The continued detection of both CBP20 and CBP80 in immunoprecipitates of U1 and U4/U6·U5 snRNP proteins after transcription inhibition suggests that spliceosomal snRNPs may be constitutively associated with CBC. Alternatively, these interactions may be maintained by remaining unspliced RNA in the extract. The transcription- and RNA-independent interactions between CBC and snRNPs may explain why CBC is detected in purified spliceosomes assembled on pre-mRNAs that lack m7G caps (Fabrizio et al. 2009).

Taken together, this series of validation experiments indicates that CBC interacts in an RNA-independent manner with the U4/U6·U5 snRNP (Table 1). There are no amino acid sequences or protein motifs shared among these partners, making it difficult to speculate on whether CBC association with these proteins is due to direct binding. Moreover, we cannot distinguish between binding to CBP20 and CBP80 in these experiments. In this study, we sought to understand the role of CBC in splicing, which requires spliceosome assembly from mature snRNPs. Therefore, we aimed to pursue the role of CBC in snRNP recruitment in vivo, rather than focus on CBC interactions with individual proteins. To do so, we coupled CBC depletion by RNAi with specific assays that interrogate the interactions of whole spliceosomal snRNPs with pre-mRNA.

SnRNP levels are unaffected by CBC depletionTreatment of HeLa cells with CBP80-specific interfering RNAs by two independent methods was effective, leading to an ∼80% reduction in CBP80 mRNA and protein levels (Fig. 2A; Supplemental Fig. 2B,C). Importantly, CBP80 depletion also led to an ∼50% reduction in CBP20 protein levels, resulting in an overall efficient knockdown of the whole CBC and indicating that CBP20 stability requires CBP80. This finding and the observed impairment of cell proliferation (Fig. 2A; Supplemental Fig. 2A) are consistent with a prior study (Narita et al. 2007). Furthermore, expression levels of seven distinct splicing factors—including proteins copurifying with CBC, U1-70K, SF3A1, Prp8, and U5-116K—were unchanged upon CBC knockdown (Supplemental Fig. 2D), indicating that CBP80 depletion does not destabilize splicing factors in general.FIGURE 2.Efficient CBC knockdown does not affect snRNP levels. HeLa cells transduced with retroviral vectors expressing CBP80 shRNA or without shRNA (Control) were assayed on day 6. (A) Assessment of CBC depletion through semiquantitative Western blotting (left panel) of CBP80, CBP20, and GAPDH after CBP80 knockdown (KD). Decreasing amounts of the same lysate were loaded. Comparison of changes in CBP80 and CBP20 RNA and protein levels (right panel). Band intensities were measured as integrated densities from Western blots and normalized to GAPDH. RNA levels were determined by RT-qPCR and normalized to 18S rRNA. N ≥ 8 different knockdowns. Error bars are the SEM. The changes are statistically significant (P < 0.005), as determined by the Student’s t-test. (B) Immunoprecipitation of snRNPs with α-TMG (K121), α-Sm proteins (Y12), and α-SART3. RNA was extracted, resolved on a 10% urea gel, and detected by Northern blot. A longer exposure of α-TMG IP lanes is shown.Because these CBC depletion conditions were designed to test spliceosome assembly and snRNP dynamics in downstream assays, we were concerned that snRNP levels might be affected. CBC and PHAX binding to nuclear m7G-capped pre-snRNAs mediates their translocation to the cytoplasm, where Sm ring assembly and snRNA cap tri-methylation occur (Izaurralde et al. 1995; Ohno et al. 2000; Muller-McNicoll and Neugebauer 2013). If CBC knockdown were to impair snRNA export, reductions in cytoplasmic snRNP maturation steps and consequently cellular snRNP levels could occur. However, Northern blotting of total RNA revealed that levels of U1, U2, U4, and U6 snRNAs were similar among control and CBC-depleted cells (Fig. 2B). Importantly, levels of assembled snRNPs containing Sm proteins and tri-methyl guanosine (TMG) caps were unaffected. Quantitation revealed no significant differences in U1 and U2 levels between knockdown and control cells (Supplemental Fig. 3A). Because U4 and U6 snRNAs were less well detected by Northern blot, U4 and U6 snRNAs were immunoprecipitated with α-SART3, specific for the U4/U6 snRNP (Stanek et al. 2003). Detection of U4 and U6 snRNAs was robust and unchanged by CBC depletion (Fig. 2B; Supplemental Fig. 3A). U5 snRNA levels were also not affected (Supplemental Fig. 3B). This surprising lack of effect on snRNP levels led us to verify CBC and PHAX binding to pre-snRNAs; both CBC and PHAX pulled down pre-snRNAs, as expected (Supplemental Fig. 3C). Finally, metabolic labeling was performed to interrogate the degree of snRNA 5′-cap tri-methylation and showed no change upon CBC depletion (Supplemental Fig. 3D). We conclude that residual levels of CBC must be sufficient to support snRNA biogenesis in depleted cells. Because total spliceosomal snRNP levels were unaffected, it was possible to investigate a distinct role for CBC in splicing.

SnRNP levels are unaffected by CBC depletion

Treatment of HeLa cells with CBP80-specific interfering RNAs by two independent methods was effective, leading to an ∼80% reduction in CBP80 mRNA and protein levels (Fig. 2A; Supplemental Fig. 2B,C). Importantly, CBP80 depletion also led to an ∼50% reduction in CBP20 protein levels, resulting in an overall efficient knockdown of the whole CBC and indicating that CBP20 stability requires CBP80. This finding and the observed impairment of cell proliferation (Fig. 2A; Supplemental Fig. 2A) are consistent with a prior study (Narita et al. 2007). Furthermore, expression levels of seven distinct splicing factors—including proteins copurifying with CBC, U1-70K, SF3A1, Prp8, and U5-116K—were unchanged upon CBC knockdown (Supplemental Fig. 2D), indicating that CBP80 depletion does not destabilize splicing factors in general.

Efficient CBC knockdown does not affect snRNP levels. HeLa cells transduced with retroviral vectors expressing CBP80 shRNA or without shRNA (Control) were assayed on day 6. (A) Assessment of CBC depletion through semiquantitative Western blotting (left panel) of CBP80, CBP20, and GAPDH after CBP80 knockdown (KD). Decreasing amounts of the same lysate were loaded. Comparison of changes in CBP80 and CBP20 RNA and protein levels (right panel). Band intensities were measured as integrated densities from Western blots and normalized to GAPDH. RNA levels were determined by RT-qPCR and normalized to 18S rRNA. N ≥ 8 different knockdowns. Error bars are the SEM. The changes are statistically significant (P < 0.005), as determined by the Student’s t-test. (B) Immunoprecipitation of snRNPs with α-TMG (K121), α-Sm proteins (Y12), and α-SART3. RNA was extracted, resolved on a 10% urea gel, and detected by Northern blot. A longer exposure of α-TMG IP lanes is shown.

Because these CBC depletion conditions were designed to test spliceosome assembly and snRNP dynamics in downstream assays, we were concerned that snRNP levels might be affected. CBC and PHAX binding to nuclear m7G-capped pre-snRNAs mediates their translocation to the cytoplasm, where Sm ring assembly and snRNA cap tri-methylation occur (Izaurralde et al. 1995; Ohno et al. 2000; Muller-McNicoll and Neugebauer 2013). If CBC knockdown were to impair snRNA export, reductions in cytoplasmic snRNP maturation steps and consequently cellular snRNP levels could occur. However, Northern blotting of total RNA revealed that levels of U1, U2, U4, and U6 snRNAs were similar among control and CBC-depleted cells (Fig. 2B). Importantly, levels of assembled snRNPs containing Sm proteins and tri-methyl guanosine (TMG) caps were unaffected. Quantitation revealed no significant differences in U1 and U2 levels between knockdown and control cells (Supplemental Fig. 3A). Because U4 and U6 snRNAs were less well detected by Northern blot, U4 and U6 snRNAs were immunoprecipitated with α-SART3, specific for the U4/U6 snRNP (Stanek et al. 2003). Detection of U4 and U6 snRNAs was robust and unchanged by CBC depletion (Fig. 2B; Supplemental Fig. 3A). U5 snRNA levels were also not affected (Supplemental Fig. 3B). This surprising lack of effect on snRNP levels led us to verify CBC and PHAX binding to pre-snRNAs; both CBC and PHAX pulled down pre-snRNAs, as expected (Supplemental Fig. 3C). Finally, metabolic labeling was performed to interrogate the degree of snRNA 5′-cap tri-methylation and showed no change upon CBC depletion (Supplemental Fig. 3D). We conclude that residual levels of CBC must be sufficient to support snRNA biogenesis in depleted cells. Because total spliceosomal snRNP levels were unaffected, it was possible to investigate a distinct role for CBC in splicing.

CBC depletion reduces cotranscriptional splicingWe selected the FOS gene to investigate the role of CBC in cotranscriptional splicing and spliceosome assembly for the following reasons. First, FOS transcripts are efficiently spliced cotranscriptionally (Listerman et al. 2006; Vargas et al. 2011). Second, FOS is strongly inducible (Piechaczyk and Blanchard 1994), permitting measurement of spliced and unspliced transcripts during the period of CBC depletion with minimal background from previously synthesized mRNA. Third, robust FOS transcription upon induction permits detection of splicing factor recruitment to the transcription unit by chromatin immunoprecipitation (ChIP) (Listerman et al. 2006; Sapra et al. 2009). Fourth, FOS is characterized by a relatively simple exon–intron structure (Fig. 3A). These features make FOS an ideal gene to study the effects of CBC depletion on splicing.FIGURE 3.CBC depletion reduces cotranscriptional spliceosome assembly and splicing. HeLa cells transduced with control and α-CBP80 shRNA retroviral vectors were subjected to RNA isolation, ChRIP, or ChIP on day 6. (A) Schematic representation of FOS pre-mRNA and mRNA, showing the location of qPCR amplicons detecting RNA with either spliced or unspliced intron 1 (blue) and intron 3 (red). Total RNA (left panel) was reverse-transcribed with an oligo(dT) primer and the relative abundance of spliced to unspliced poly(A)+ RNA assessed by qPCR. Nascent RNA (right panel) was isolated by ChRIP from an α-AcH4 IP and reverse-transcribed, using a primer in intron 2 (for intron 1 splicing) or in exon 4 (for intron 3 splicing). qPCR was conducted to assess the relative abundance of spliced to unspliced RNA. n ≥ 4; error bars are the SEM. (B) Schematic representation of the FOS gene. (Gray lines) The location of qPCR amplicons used in ChIP. The following antibodies were used: α-CBP80, α-Pol II-CTD (4H8), α-U2AF-65 (MC3), α-U5-116K, and α-GFP in case of transgenic cell lines expressing U1-70K-GFP or SF3A1-GFP. All values were calculated as percent input, normalized to nonimmune ChIP, and presented as fold enrichment over an intergenic region. In Pol II ChIP, normalization to intergenic was omitted due to absence of background. N ≥ 3; error bars are the SEM. Significant differences between knockdown and control determined by the Student’s t-test: (*) P < 0.05, (**) P < 0.01.As a first step, we used ChIP to quantitate the impact of CBP80 RNAi on CBC levels at the FOS transcription unit. CBP80 ChIP confirmed that the ∼80% decrease in cellular CBP80 protein corresponded to the reduction in CBP80 levels along the FOS gene (Fig. 3B). Acetylated histone H4 (AcH4) and Pol II levels were unchanged by CBC depletion (Fig. 3B; Supplemental Fig. 4B), suggesting that FOS transcription is not CBC dependent. Next, we examined steady-state intron removal in FOS RNA by RT-qPCR, which revealed that the levels of unspliced FOS introns 1 and 3 increased by twofold upon CBC knockdown (Fig. 3A). In parallel, a decrease in total FOS mRNA was observed (Supplemental Fig. 4A). The levels of another short-lived, spliced mRNA, MYC, were also significantly reduced, and splicing of the first intron was inhibited by ∼30% (data not shown). At the same time, mRNAs with long half-lives, such as ACTB, were unaffected by CBC depletion (Supplemental Fig. 4A), indicating that CBC depletion inhibits but does not abolish splicing activity. Another indication of cellular splicing activity is morphological: Nuclear speckles are normally interconnected, irregularly shaped nuclear compartments where splicing factors concentrate. Upon splicing inhibition with spliceostatin A, speckles become “rounded up” (Kaida et al. 2007). Supplemental Figure 5 shows that nuclear speckles also displayed a rounded-up morphology when cells were depleted of CBC, consistent with the expected overall splicing inhibition (see the Introduction). To determine the effect of CBC depletion on cotranscriptional splicing, nascent FOS RNA was retrieved by chromatin RNA immunoprecipitation (ChRIP), in which cross-linked active chromatin is immunoprecipitated with antibodies to AcH4 (Listerman et al. 2006). Figure 3A shows that cotranscriptional splicing of FOS introns 1 and 3 was impaired upon CBC depletion, similar to total pre-mRNA splicing. This finding indicates that splicing inhibition is not likely due to effects on NEXT complex activity, because NEXT would not be expected to destabilize nascent RNA. We conclude that the absence of CBC leads to a decreased efficiency of cotranscriptional intron removal.

CBC depletion reduces cotranscriptional splicing

We selected the FOS gene to investigate the role of CBC in cotranscriptional splicing and spliceosome assembly for the following reasons. First, FOS transcripts are efficiently spliced cotranscriptionally (Listerman et al. 2006; Vargas et al. 2011). Second, FOS is strongly inducible (Piechaczyk and Blanchard 1994), permitting measurement of spliced and unspliced transcripts during the period of CBC depletion with minimal background from previously synthesized mRNA. Third, robust FOS transcription upon induction permits detection of splicing factor recruitment to the transcription unit by chromatin immunoprecipitation (ChIP) (Listerman et al. 2006; Sapra et al. 2009). Fourth, FOS is characterized by a relatively simple exon–intron structure (Fig. 3A). These features make FOS an ideal gene to study the effects of CBC depletion on splicing.

CBC depletion reduces cotranscriptional spliceosome assembly and splicing. HeLa cells transduced with control and α-CBP80 shRNA retroviral vectors were subjected to RNA isolation, ChRIP, or ChIP on day 6. (A) Schematic representation of FOS pre-mRNA and mRNA, showing the location of qPCR amplicons detecting RNA with either spliced or unspliced intron 1 (blue) and intron 3 (red). Total RNA (left panel) was reverse-transcribed with an oligo(dT) primer and the relative abundance of spliced to unspliced poly(A)+ RNA assessed by qPCR. Nascent RNA (right panel) was isolated by ChRIP from an α-AcH4 IP and reverse-transcribed, using a primer in intron 2 (for intron 1 splicing) or in exon 4 (for intron 3 splicing). qPCR was conducted to assess the relative abundance of spliced to unspliced RNA. n ≥ 4; error bars are the SEM. (B) Schematic representation of the FOS gene. (Gray lines) The location of qPCR amplicons used in ChIP. The following antibodies were used: α-CBP80, α-Pol II-CTD (4H8), α-U2AF-65 (MC3), α-U5-116K, and α-GFP in case of transgenic cell lines expressing U1-70K-GFP or SF3A1-GFP. All values were calculated as percent input, normalized to nonimmune ChIP, and presented as fold enrichment over an intergenic region. In Pol II ChIP, normalization to intergenic was omitted due to absence of background. N ≥ 3; error bars are the SEM. Significant differences between knockdown and control determined by the Student’s t-test: (*) P < 0.05, (**) P < 0.01.

As a first step, we used ChIP to quantitate the impact of CBP80 RNAi on CBC levels at the FOS transcription unit. CBP80 ChIP confirmed that the ∼80% decrease in cellular CBP80 protein corresponded to the reduction in CBP80 levels along the FOS gene (Fig. 3B). Acetylated histone H4 (AcH4) and Pol II levels were unchanged by CBC depletion (Fig. 3B; Supplemental Fig. 4B), suggesting that FOS transcription is not CBC dependent. Next, we examined steady-state intron removal in FOS RNA by RT-qPCR, which revealed that the levels of unspliced FOS introns 1 and 3 increased by twofold upon CBC knockdown (Fig. 3A). In parallel, a decrease in total FOS mRNA was observed (Supplemental Fig. 4A). The levels of another short-lived, spliced mRNA, MYC, were also significantly reduced, and splicing of the first intron was inhibited by ∼30% (data not shown). At the same time, mRNAs with long half-lives, such as ACTB, were unaffected by CBC depletion (Supplemental Fig. 4A), indicating that CBC depletion inhibits but does not abolish splicing activity. Another indication of cellular splicing activity is morphological: Nuclear speckles are normally interconnected, irregularly shaped nuclear compartments where splicing factors concentrate. Upon splicing inhibition with spliceostatin A, speckles become “rounded up” (Kaida et al. 2007). Supplemental Figure 5 shows that nuclear speckles also displayed a rounded-up morphology when cells were depleted of CBC, consistent with the expected overall splicing inhibition (see the Introduction). To determine the effect of CBC depletion on cotranscriptional splicing, nascent FOS RNA was retrieved by chromatin RNA immunoprecipitation (ChRIP), in which cross-linked active chromatin is immunoprecipitated with antibodies to AcH4 (Listerman et al. 2006). Figure 3A shows that cotranscriptional splicing of FOS introns 1 and 3 was impaired upon CBC depletion, similar to total pre-mRNA splicing. This finding indicates that splicing inhibition is not likely due to effects on NEXT complex activity, because NEXT would not be expected to destabilize nascent RNA. We conclude that the absence of CBC leads to a decreased efficiency of cotranscriptional intron removal.

CBC depletion impairs cotranscriptional spliceosome assemblyKnowing that CBC is required for efficient nascent FOS RNA splicing, we sought to determine the step at which spliceosome assembly is altered in the absence of CBC. We therefore investigated the recruitment profiles of snRNPs, using a cotranscriptional spliceosome assembly assay based on ChIP (Gornemann et al. 2005; Listerman et al. 2006; Sapra et al. 2009). SnRNP-specific proteins were used as proxies for each snRNP: U1 (U1-70K), U2 (SF3A1), and U4/U6·U5 (U5-116K). GFP-tagged snRNP proteins expressed stably from BACs were incorporated into snRNPs and correctly localized (Supplemental Fig. 1A–C).The levels and profiles of the U1, U2, and U4/U6·U5 snRNPs as well as U2AF65 were determined at four positions within exons and introns along the length of FOS (Fig. 3B). Upon CBC knockdown, accumulation of all three snRNPs was decreased (Fig. 3B). These effects are specific, because U2AF65 accumulation was unaffected. U5-116K accumulation was reduced significantly at the beginning as well as the end of FOS (Fig. 3B). The decrease in U4/U6·U5 snRNP recruitment is consistent with the notion that CBC associates with protein components of the tri-snRNP, namely, U5-200K, U4/U6-60K, and U4/U6-61K (see Fig. 1; Table 1) to assist spliceosome assembly. CBC depletion also caused marked reductions in U1 and U2 snRNP recruitment with accumulation of U1-70K-GFP 50% reduced at all positions (Fig. 3B). Because our results point to CBC interactions with U4/U6·U5 and not U1 snRNP proteins (Fig. 1; Table 1), these data suggest that U4/U6·U5 snRNP association with CBC could assist U1 snRNP recruitment. Prior studies reported U5 snRNP interactions with the 5′ ss early in spliceosome assembly as well as U1 snRNP recruitment in the context of a multi-snRNP-containing complex (Wyatt et al. 1992; Malca et al. 2003). Thus, our data support the notion that CBC helps recruit U1 snRNP to the first 5′ ss (Lewis et al. 1996b), not by direct interactions with U1 but instead indirectly through the U4/U6·U5 snRNP. Moreover, the U1 snRNP is retained on multiple intron-containing pre-mRNAs after the first splicing event, rendering subsequent splicing more efficient (Crabb et al. 2010). Indeed, splicing of both FOS introns 1 and 3 was CBC dependent, suggesting—together with the observed snRNP profiles—that CBC may help retain the U1 snRNP on nascent RNA until all introns are removed. We conclude that proper cotranscriptional accumulation of U1, U2, and U4/U6·U5 snRNPs is dependent on CBC in vivo.

CBC depletion impairs cotranscriptional spliceosome assembly

Knowing that CBC is required for efficient nascent FOS RNA splicing, we sought to determine the step at which spliceosome assembly is altered in the absence of CBC. We therefore investigated the recruitment profiles of snRNPs, using a cotranscriptional spliceosome assembly assay based on ChIP (Gornemann et al. 2005; Listerman et al. 2006; Sapra et al. 2009). SnRNP-specific proteins were used as proxies for each snRNP: U1 (U1-70K), U2 (SF3A1), and U4/U6·U5 (U5-116K). GFP-tagged snRNP proteins expressed stably from BACs were incorporated into snRNPs and correctly localized (Supplemental Fig. 1A–C).

The levels and profiles of the U1, U2, and U4/U6·U5 snRNPs as well as U2AF65 were determined at four positions within exons and introns along the length of FOS (Fig. 3B). Upon CBC knockdown, accumulation of all three snRNPs was decreased (Fig. 3B). These effects are specific, because U2AF65 accumulation was unaffected. U5-116K accumulation was reduced significantly at the beginning as well as the end of FOS (Fig. 3B). The decrease in U4/U6·U5 snRNP recruitment is consistent with the notion that CBC associates with protein components of the tri-snRNP, namely, U5-200K, U4/U6-60K, and U4/U6-61K (see Fig. 1; Table 1) to assist spliceosome assembly. CBC depletion also caused marked reductions in U1 and U2 snRNP recruitment with accumulation of U1-70K-GFP 50% reduced at all positions (Fig. 3B). Because our results point to CBC interactions with U4/U6·U5 and not U1 snRNP proteins (Fig. 1; Table 1), these data suggest that U4/U6·U5 snRNP association with CBC could assist U1 snRNP recruitment. Prior studies reported U5 snRNP interactions with the 5′ ss early in spliceosome assembly as well as U1 snRNP recruitment in the context of a multi-snRNP-containing complex (Wyatt et al. 1992; Malca et al. 2003). Thus, our data support the notion that CBC helps recruit U1 snRNP to the first 5′ ss (Lewis et al. 1996b), not by direct interactions with U1 but instead indirectly through the U4/U6·U5 snRNP. Moreover, the U1 snRNP is retained on multiple intron-containing pre-mRNAs after the first splicing event, rendering subsequent splicing more efficient (Crabb et al. 2010). Indeed, splicing of both FOS introns 1 and 3 was CBC dependent, suggesting—together with the observed snRNP profiles—that CBC may help retain the U1 snRNP on nascent RNA until all introns are removed. We conclude that proper cotranscriptional accumulation of U1, U2, and U4/U6·U5 snRNPs is dependent on CBC in vivo.

CBC promotes U1 and U5 snRNP interactions with pre-mRNAThe above data suggest that CBC’s overall role in FOS splicing is to enhance spliceosomal snRNP interactions with nascent RNA through its association with U4/U6·U5 snRNPs. To test this proposal more broadly, we used a live-cell imaging strategy to analyze the dynamic interaction of snRNPs with pre-mRNA both globally and at a different test gene. We addressed total cellular pre-mRNA by monitoring snRNP dynamics in the nucleoplasm; complementary to this, we used an integrated, inducible transcription unit (E3) containing three exons and two introns of the β-globin gene (Fig. 4A; Huranova et al. 2010; Pabis et al. 2010; Brody et al. 2011). E3 mRNA contains the MS2 binding site in the 3′ UTR, such that active transcription sites can be monitored in living cells with a fluorescently tagged MS2-binding protein. BACs harboring U1-70K-GFP, U2AF65-GFP, and Prp8-GFP were stably integrated into E3 cells, and fluorescence imaging revealed that U2AF65, U1, and U5 snRNPs were detectable in speckles and nucleoplasm, as expected (Fig. 4B). Importantly, each of these splicing factors accumulated at the E3 gene during active transcription, as seen by colocalization with E3 nascent RNAs at the transcription site (Fig. 4B).FIGURE 4.CBC depletion reduces snRNP interactions with active transcription sites. (A) Diagram of the E3 construct stably integrated into U2OS cells. Expression is driven by a minimal CMV promoter (P) under control of the tetracycline response element and induced in the presence of doxycycline (dox) by the transactivator rtTA. The gene contains three β-globin exons (E1–E3) and a CFP coding sequence, 18 MS2 repeats, a polyadenylation cleavage signal, and a transcription terminator (T). (B) Imaging of nascent RNA at the E3 transcription unit in fixed cells by RNA FISH (Cy3-labeled probe to the MS2 sequence repeats; red) together with the stably integrated U1 snRNP (U1-70K-GFP; green), U2AF65-GFP, or U5 snRNP proteins (Prp8-GFP). Scale bars, 5 μm. (C) FRAP curves show recovery of U1-70K-GFP, U2AF65-GFP, and Prp8-GFP fluorescence at bleached spots placed in nucleoplasm, speckles, or at the E3 transcription site visualized by MS2BP-mCherry. Three independent experiments were performed; in each 10–20 cells were tested. Error bars represent the SEM. For U1-70K-GFP and Prp8-GFP, the differences between recovery curves in control and CBC knockdown cells are statistically significant, as established using the Mann–Whitney test.FRAP experiments revealed CBC-dependent dynamics among U1 and U4/U6·U5 snRNPs at both the E3 transcription site and in the nucleoplasm, where endogenous splicing occurs (Fig. 4C). In both locations, significantly faster recovery of both U1 and U5 snRNPs was observed upon CBC depletion, indicating that each snRNP was retained at sites of transcription for a shorter period of time. In contrast, mobility of U1 and U5 snRNPs in nuclear speckles was unchanged, suggesting that snRNP accumulation in speckles does not depend on interactions with CBC. No effects on U2AF65 recovery or mobility were observed at any location (Fig. 4C), in agreement with the ChIP data (Fig. 3B). Because the mobility of nucleoplasmic snRNPs is determined by association with pre-mRNA (Huranova et al. 2010), these data indicate that CBC knockdown weakens U1 and U4/U6·U5 snRNP interactions with endogenous pre-mRNAs in living cells. This finding is consistent with morphological evidence that splicing is globally inhibited (Supplemental Fig. 5).

CBC promotes U1 and U5 snRNP interactions with pre-mRNA

The above data suggest that CBC’s overall role in FOS splicing is to enhance spliceosomal snRNP interactions with nascent RNA through its association with U4/U6·U5 snRNPs. To test this proposal more broadly, we used a live-cell imaging strategy to analyze the dynamic interaction of snRNPs with pre-mRNA both globally and at a different test gene. We addressed total cellular pre-mRNA by monitoring snRNP dynamics in the nucleoplasm; complementary to this, we used an integrated, inducible transcription unit (E3) containing three exons and two introns of the β-globin gene (Fig. 4A; Huranova et al. 2010; Pabis et al. 2010; Brody et al. 2011). E3 mRNA contains the MS2 binding site in the 3′ UTR, such that active transcription sites can be monitored in living cells with a fluorescently tagged MS2-binding protein. BACs harboring U1-70K-GFP, U2AF65-GFP, and Prp8-GFP were stably integrated into E3 cells, and fluorescence imaging revealed that U2AF65, U1, and U5 snRNPs were detectable in speckles and nucleoplasm, as expected (Fig. 4B). Importantly, each of these splicing factors accumulated at the E3 gene during active transcription, as seen by colocalization with E3 nascent RNAs at the transcription site (Fig. 4B).

CBC depletion reduces snRNP interactions with active transcription sites. (A) Diagram of the E3 construct stably integrated into U2OS cells. Expression is driven by a minimal CMV promoter (P) under control of the tetracycline response element and induced in the presence of doxycycline (dox) by the transactivator rtTA. The gene contains three β-globin exons (E1–E3) and a CFP coding sequence, 18 MS2 repeats, a polyadenylation cleavage signal, and a transcription terminator (T). (B) Imaging of nascent RNA at the E3 transcription unit in fixed cells by RNA FISH (Cy3-labeled probe to the MS2 sequence repeats; red) together with the stably integrated U1 snRNP (U1-70K-GFP; green), U2AF65-GFP, or U5 snRNP proteins (Prp8-GFP). Scale bars, 5 μm. (C) FRAP curves show recovery of U1-70K-GFP, U2AF65-GFP, and Prp8-GFP fluorescence at bleached spots placed in nucleoplasm, speckles, or at the E3 transcription site visualized by MS2BP-mCherry. Three independent experiments were performed; in each 10–20 cells were tested. Error bars represent the SEM. For U1-70K-GFP and Prp8-GFP, the differences between recovery curves in control and CBC knockdown cells are statistically significant, as established using the Mann–Whitney test.

FRAP experiments revealed CBC-dependent dynamics among U1 and U4/U6·U5 snRNPs at both the E3 transcription site and in the nucleoplasm, where endogenous splicing occurs (Fig. 4C). In both locations, significantly faster recovery of both U1 and U5 snRNPs was observed upon CBC depletion, indicating that each snRNP was retained at sites of transcription for a shorter period of time. In contrast, mobility of U1 and U5 snRNPs in nuclear speckles was unchanged, suggesting that snRNP accumulation in speckles does not depend on interactions with CBC. No effects on U2AF65 recovery or mobility were observed at any location (Fig. 4C), in agreement with the ChIP data (Fig. 3B). Because the mobility of nucleoplasmic snRNPs is determined by association with pre-mRNA (Huranova et al. 2010), these data indicate that CBC knockdown weakens U1 and U4/U6·U5 snRNP interactions with endogenous pre-mRNAs in living cells. This finding is consistent with morphological evidence that splicing is globally inhibited (Supplemental Fig. 5).

CONCLUSIONSHere we have addressed the function of mammalian CBC in pre-mRNA splicing in vivo, using mass spectrometry followed by quantitative analyses of splicing activities upon CBC knockdown. We report the following major observations. First, several U1, U2, and U4/U6·U5 snRNP proteins specifically copurified with CBC from nuclease-treated cell lysates. Protein components of U4/U6·U5 snRNPs were most robustly detected, and three—U5-200K (hBrr2), U4/U6-60K (hPrp31), and U4/U6-60K (hPrp4)—associated with CBC in an RNase-insensitive manner. Second, cotranscriptional spliceosome assembly was reduced by CBC depletion, with major defects in U1, U2, and U4/U6·U5 snRNP accumulation on FOS. Third, FRAP experiments showed that U1 and U4/U6·U5 snRNPs exhibit higher nucleoplasmic mobility and associate less stably with an intron-containing transcription site upon CBC depletion, indicative of global as well as specific defects in snRNP association with pre-mRNA. Finally, total snRNP levels were unaffected, ruling out the possibility that splicing defects were caused by an impairment of snRNP biogenesis. Taken together, the data indicate that the primary role of CBC in splicing is to promote snRNP interactions with nascent transcripts, via RNA-independent protein–protein interactions with U4/U6·U5 snRNP components. Because CBC interactions with U1 and U2 snRNP components were RNA dependent, we suggest that snRNP association with pre-mRNA and CBC may be cooperative and depend on inter-snRNP interactions during spliceosome assembly.We propose that splicing inhibition may explain the cell proliferation defect that we and others have observed upon CBC knockdown (Narita et al. 2007). We document a 40% reduction in cellular levels of FOS and MYC mRNAs, with roles in cell cycle progression (Piechaczyk and Blanchard 1994). A prior global study reported down-regulation of 250 transcripts upon CBC knockdown; 249 of these were intron-containing (Narita et al. 2007), suggesting that their reduction could be due to transcript instability caused by decreased splicing. Genetic deletion of CBC subunits in budding yeast also leads to growth impairment (Fortes et al. 1999). Other explanations for reduced cell proliferation include effects on nonsense-mediated decay, transcription elongation, or alternative splicing (Hosoda et al. 2005; Raczynska et al. 2010; Lenasi et al. 2011). The clear impact of CBC depletion on spliceosome assembly and constitutive splicing detected here, without changes in snRNP levels, underscores the physiological importance of CBC in modulating snRNP dynamics in living cells.

CONCLUSIONS

Here we have addressed the function of mammalian CBC in pre-mRNA splicing in vivo, using mass spectrometry followed by quantitative analyses of splicing activities upon CBC knockdown. We report the following major observations. First, several U1, U2, and U4/U6·U5 snRNP proteins specifically copurified with CBC from nuclease-treated cell lysates. Protein components of U4/U6·U5 snRNPs were most robustly detected, and three—U5-200K (hBrr2), U4/U6-60K (hPrp31), and U4/U6-60K (hPrp4)—associated with CBC in an RNase-insensitive manner. Second, cotranscriptional spliceosome assembly was reduced by CBC depletion, with major defects in U1, U2, and U4/U6·U5 snRNP accumulation on FOS. Third, FRAP experiments showed that U1 and U4/U6·U5 snRNPs exhibit higher nucleoplasmic mobility and associate less stably with an intron-containing transcription site upon CBC depletion, indicative of global as well as specific defects in snRNP association with pre-mRNA. Finally, total snRNP levels were unaffected, ruling out the possibility that splicing defects were caused by an impairment of snRNP biogenesis. Taken together, the data indicate that the primary role of CBC in splicing is to promote snRNP interactions with nascent transcripts, via RNA-independent protein–protein interactions with U4/U6·U5 snRNP components. Because CBC interactions with U1 and U2 snRNP components were RNA dependent, we suggest that snRNP association with pre-mRNA and CBC may be cooperative and depend on inter-snRNP interactions during spliceosome assembly.

We propose that splicing inhibition may explain the cell proliferation defect that we and others have observed upon CBC knockdown (Narita et al. 2007). We document a 40% reduction in cellular levels of FOS and MYC mRNAs, with roles in cell cycle progression (Piechaczyk and Blanchard 1994). A prior global study reported down-regulation of 250 transcripts upon CBC knockdown; 249 of these were intron-containing (Narita et al. 2007), suggesting that their reduction could be due to transcript instability caused by decreased splicing. Genetic deletion of CBC subunits in budding yeast also leads to growth impairment (Fortes et al. 1999). Other explanations for reduced cell proliferation include effects on nonsense-mediated decay, transcription elongation, or alternative splicing (Hosoda et al. 2005; Raczynska et al. 2010; Lenasi et al. 2011). The clear impact of CBC depletion on spliceosome assembly and constitutive splicing detected here, without changes in snRNP levels, underscores the physiological importance of CBC in modulating snRNP dynamics in living cells.

MATERIALS AND METHODSCell lines and RNAiTransgenic HeLa cell lines with stable integration of BACs carrying C-terminally tagged SF3A1, CBP20, U1-70K, U5-116K, U5-200K, U4/U6-60K, U4/U6-61K, U2AF65, NHN1, and CDC2 were generated as described (Poser et al. 2008). HeLa and Phoenix gag-pol packaging cells were cultured in high-glucose DMEM with 10% FCS. Transcription was inhibited by adding α-amanitin (Sigma-Aldrich) to a final concentration of 10 μg/mL to the cell culture medium for 10 h (Listerman et al. 2007); the treatment was effective, because the cells died the following day. FOS induction was with 5 µM calcium ionophore A23187 for 15 min after 2 h of serum starvation (Listerman et al. 2006). Human U2OS E3 cells (Brody et al. 2011) were stably integrated with BACs expressing U1-70K-GFP, U2AF65, or Prp8-GFP and maintained in low-glucose DMEM with 10% FCS. The RNAi target sequence in CBP80, AAGACTCATGTACCCATGTTA, was selected with the Ambion siRNA target finder and the shRNA expression cassette inserted into pRVH1-hygro (Schuck et al. 2004). For viral particle production, packaging cells were cotransfected with pMD.G and pRVH1 plasmids (empty control or shRNA). HeLa cells were infected with virus-containing media plus 4 µg/mL polybrene, selected with 800 µg/mL hygromycin B, and harvested 6 d after infection.Mass spectrometryImmunopurification of CBP20-GFP was carried out from ∼8 × 107 HeLa transgenic cells using goat polyclonal α-GFP antibody (Cheeseman and Desai 2005). Ninety units per milliliter of benzonase was added to the initial lysate to degrade DNA and RNA. Eluted polypeptides were resolved by SDS-PAGE and subjected to in-gel digestion with trypsin (Shevchenko et al. 2006); LC-MS/MS analysis of peptide mixtures was performed on an Ultimate nanoLC system interfaced online with a linear ion trap LTQ. Data validation and organization were carried out with Scaffold 2, based on the PeptideProphet algorithm (Keller et al. 2002). For shotgun MS, see the Supplemental Methods.Immunoprecipitation (IP), Western and Northern blotHeLa extracts were prepared in NET-2 buffer (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 0.05% NP-40) plus protease inhibitors. IP was at 4°C with GammaBind G Sepharose beads coupled to specific (12 µg of α-GFP, 1 µg of α-TMG, 5 µL of α-SART-3, or 250 µL of Y12 hybridoma supernatant) and control (nonimmune mouse or goat) antibodies. RNase A treatment was at 100 µg/mL for 20 min at room temperature. For Western blotting, total HeLa extract was prepared in NEST-2 buffer (50 mM Tris-HCl at pH 6.8, 20 mM EDTA, 5% SDS) plus protease inhibitors; proteins were resolved on 4%–20% or 4%–12% Tris-glycine gels.Northern blotting was performed as described (Bell and Bindereif 1999). RNA—0.75% and 0.25% of input samples and 3.75% and 1.25% of IP samples—was separated on 10% TBE-urea gels, transferred to a nylon membrane, and UV-cross-linked. Hybridization was performed with a mixture of probes (5S rRNA, U3 snoRNA, U1, U2, U4, and U6 snRNA) in Church-Buffer. Probe sequences were amplified by RT-PCR from total HeLa cell RNA and digoxigenin (DIG)–labeled by PCR; see Supplemental Table 3 for primers. α-DIG chemiluminescence was detected using the CHEMI-SMART 5100 documentation and analysis system; quantification was with ImageJ.RT-qPCR, ChIP, and ChRIPTotal RNA was isolated using TRIzol, DNase-treated, and reverse-transcribed from an oligo(dT) or specific reverse primer for 18S rRNA. See Supplemental Table 3 for primers. To calculate fold changes in mRNA levels, the following equation was used: 2ΔCt(target)/2ΔCt(norm), where ΔCt = Ct (control sample) – Ct (experimental sample). Ct stands for threshold cycle value, “target” indicates the gene of interest, and “norm” indicates RNA for normalization. ChIP and ChRIP were performed as described (Listerman et al. 2006). Approximately 2 × 107 cells were used per IP. The antibodies used were 10 µL of α-CBP80 serum, 4 µg of α-RNA Pol II, 12 µg of α-GFP, 30 µg of α-U2AF-65, 3 µL of α-U5-116K serum, and 9 µL of α-AcH4 serum and control (nonimmune mouse or goat IgGs).FRAPE3 cells were maintained in Phenol Red–free Leibovitz’s L-15 with 10% FCS at 37°C. For CBP80 knockdown, cells were transfected with CBP80 or negative Stealth RNAi in Optimem Plus Glutamax medium, using Lipofectamine 2000. After 48 h, cells were transfected with MS2-mCherry using FuGENE 6 in order to mark active transcription sites, and then transcriptionally induced with doxycycline (1 μg/mL) for several hours. FRAP experiments were performed 1 d later. FRAP image sequences were obtained on an Olympus FV1000 inverted scanning confocal microscope with a heated chamber and objective heater (37°C) and a 60×, 1.35 NA oil objective. Cells were scanned using a 488-nm laser for detection of GFP-labeled U1-70K, U2AF65, or Prp8 and a 561-nm laser for the detection of MS2-mcherry. GFP-tagged U1-70K, U2AF65, and Prp8 were bleached using the 488-nm laser in the nucleoplasm, speckles, and at the active transcription site. Five pre-bleach images were acquired. Post-bleach images were acquired at a frequency of four images per 2 sec. For analysis of fluorescence recovery, FRAP data were normalized and calculated as previously described (Shav-Tal et al. 2005). Statistical significance was assessed with the Mann–Whitney test. Detection of active transcription sites in fixed cells was by RNA FISH to nascent mRNAs, using a Cy3-labeled probe to the MS2 sequence repeats in the 3′ UTR of the mRNA (Brody et al. 2011).AntibodiesThe following mouse monoclonal antibodies were used: α-GAPDH (Novus Biologicals); α-RNA Pol II CTD, 4H8 (Abcam); α-TMG, K121 (Calbiochem); α-U2AF-65, MC3 (Gama-Carvalho et al. 1997); and α-Sm proteins, Y12 (Lerner et al. 1981). The following rabbit polyclonal antibodies were used: α-AcH4 (Upstate); α-CBP80 and α-CBP20 (Izaurralde et al. 1994, 1995); α-SART-3 (Stanek et al. 2003); and α-U5-116K (Fabrizio et al. 1997). Goat polyclonal α-GFP was a gift of D. Drechsel (MPI-CBG). Nonimmune IgGs and HRP-conjugated secondary antibodies were from Sigma-Aldrich or GE Healthcare.

MATERIALS AND METHODS

Cell lines and RNAiTransgenic HeLa cell lines with stable integration of BACs carrying C-terminally tagged SF3A1, CBP20, U1-70K, U5-116K, U5-200K, U4/U6-60K, U4/U6-61K, U2AF65, NHN1, and CDC2 were generated as described (Poser et al. 2008). HeLa and Phoenix gag-pol packaging cells were cultured in high-glucose DMEM with 10% FCS. Transcription was inhibited by adding α-amanitin (Sigma-Aldrich) to a final concentration of 10 μg/mL to the cell culture medium for 10 h (Listerman et al. 2007); the treatment was effective, because the cells died the following day. FOS induction was with 5 µM calcium ionophore A23187 for 15 min after 2 h of serum starvation (Listerman et al. 2006). Human U2OS E3 cells (Brody et al. 2011) were stably integrated with BACs expressing U1-70K-GFP, U2AF65, or Prp8-GFP and maintained in low-glucose DMEM with 10% FCS. The RNAi target sequence in CBP80, AAGACTCATGTACCCATGTTA, was selected with the Ambion siRNA target finder and the shRNA expression cassette inserted into pRVH1-hygro (Schuck et al. 2004). For viral particle production, packaging cells were cotransfected with pMD.G and pRVH1 plasmids (empty control or shRNA). HeLa cells were infected with virus-containing media plus 4 µg/mL polybrene, selected with 800 µg/mL hygromycin B, and harvested 6 d after infection.

Cell lines and RNAi

Transgenic HeLa cell lines with stable integration of BACs carrying C-terminally tagged SF3A1, CBP20, U1-70K, U5-116K, U5-200K, U4/U6-60K, U4/U6-61K, U2AF65, NHN1, and CDC2 were generated as described (Poser et al. 2008). HeLa and Phoenix gag-pol packaging cells were cultured in high-glucose DMEM with 10% FCS. Transcription was inhibited by adding α-amanitin (Sigma-Aldrich) to a final concentration of 10 μg/mL to the cell culture medium for 10 h (Listerman et al. 2007); the treatment was effective, because the cells died the following day. FOS induction was with 5 µM calcium ionophore A23187 for 15 min after 2 h of serum starvation (Listerman et al. 2006). Human U2OS E3 cells (Brody et al. 2011) were stably integrated with BACs expressing U1-70K-GFP, U2AF65, or Prp8-GFP and maintained in low-glucose DMEM with 10% FCS. The RNAi target sequence in CBP80, AAGACTCATGTACCCATGTTA, was selected with the Ambion siRNA target finder and the shRNA expression cassette inserted into pRVH1-hygro (Schuck et al. 2004). For viral particle production, packaging cells were cotransfected with pMD.G and pRVH1 plasmids (empty control or shRNA). HeLa cells were infected with virus-containing media plus 4 µg/mL polybrene, selected with 800 µg/mL hygromycin B, and harvested 6 d after infection.

Mass spectrometryImmunopurification of CBP20-GFP was carried out from ∼8 × 107 HeLa transgenic cells using goat polyclonal α-GFP antibody (Cheeseman and Desai 2005). Ninety units per milliliter of benzonase was added to the initial lysate to degrade DNA and RNA. Eluted polypeptides were resolved by SDS-PAGE and subjected to in-gel digestion with trypsin (Shevchenko et al. 2006); LC-MS/MS analysis of peptide mixtures was performed on an Ultimate nanoLC system interfaced online with a linear ion trap LTQ. Data validation and organization were carried out with Scaffold 2, based on the PeptideProphet algorithm (Keller et al. 2002). For shotgun MS, see the Supplemental Methods.

Mass spectrometry

Immunopurification of CBP20-GFP was carried out from ∼8 × 107 HeLa transgenic cells using goat polyclonal α-GFP antibody (Cheeseman and Desai 2005). Ninety units per milliliter of benzonase was added to the initial lysate to degrade DNA and RNA. Eluted polypeptides were resolved by SDS-PAGE and subjected to in-gel digestion with trypsin (Shevchenko et al. 2006); LC-MS/MS analysis of peptide mixtures was performed on an Ultimate nanoLC system interfaced online with a linear ion trap LTQ. Data validation and organization were carried out with Scaffold 2, based on the PeptideProphet algorithm (Keller et al. 2002). For shotgun MS, see the Supplemental Methods.

Immunoprecipitation (IP), Western and Northern blotHeLa extracts were prepared in NET-2 buffer (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 0.05% NP-40) plus protease inhibitors. IP was at 4°C with GammaBind G Sepharose beads coupled to specific (12 µg of α-GFP, 1 µg of α-TMG, 5 µL of α-SART-3, or 250 µL of Y12 hybridoma supernatant) and control (nonimmune mouse or goat) antibodies. RNase A treatment was at 100 µg/mL for 20 min at room temperature. For Western blotting, total HeLa extract was prepared in NEST-2 buffer (50 mM Tris-HCl at pH 6.8, 20 mM EDTA, 5% SDS) plus protease inhibitors; proteins were resolved on 4%–20% or 4%–12% Tris-glycine gels.Northern blotting was performed as described (Bell and Bindereif 1999). RNA—0.75% and 0.25% of input samples and 3.75% and 1.25% of IP samples—was separated on 10% TBE-urea gels, transferred to a nylon membrane, and UV-cross-linked. Hybridization was performed with a mixture of probes (5S rRNA, U3 snoRNA, U1, U2, U4, and U6 snRNA) in Church-Buffer. Probe sequences were amplified by RT-PCR from total HeLa cell RNA and digoxigenin (DIG)–labeled by PCR; see Supplemental Table 3 for primers. α-DIG chemiluminescence was detected using the CHEMI-SMART 5100 documentation and analysis system; quantification was with ImageJ.

Immunoprecipitation (IP), Western and Northern blot

HeLa extracts were prepared in NET-2 buffer (50 mM Tris-HCl at pH 7.5, 150 mM NaCl, 0.05% NP-40) plus protease inhibitors. IP was at 4°C with GammaBind G Sepharose beads coupled to specific (12 µg of α-GFP, 1 µg of α-TMG, 5 µL of α-SART-3, or 250 µL of Y12 hybridoma supernatant) and control (nonimmune mouse or goat) antibodies. RNase A treatment was at 100 µg/mL for 20 min at room temperature. For Western blotting, total HeLa extract was prepared in NEST-2 buffer (50 mM Tris-HCl at pH 6.8, 20 mM EDTA, 5% SDS) plus protease inhibitors; proteins were resolved on 4%–20% or 4%–12% Tris-glycine gels.

Northern blotting was performed as described (Bell and Bindereif 1999). RNA—0.75% and 0.25% of input samples and 3.75% and 1.25% of IP samples—was separated on 10% TBE-urea gels, transferred to a nylon membrane, and UV-cross-linked. Hybridization was performed with a mixture of probes (5S rRNA, U3 snoRNA, U1, U2, U4, and U6 snRNA) in Church-Buffer. Probe sequences were amplified by RT-PCR from total HeLa cell RNA and digoxigenin (DIG)–labeled by PCR; see Supplemental Table 3 for primers. α-DIG chemiluminescence was detected using the CHEMI-SMART 5100 documentation and analysis system; quantification was with ImageJ.

RT-qPCR, ChIP, and ChRIPTotal RNA was isolated using TRIzol, DNase-treated, and reverse-transcribed from an oligo(dT) or specific reverse primer for 18S rRNA. See Supplemental Table 3 for primers. To calculate fold changes in mRNA levels, the following equation was used: 2ΔCt(target)/2ΔCt(norm), where ΔCt = Ct (control sample) – Ct (experimental sample). Ct stands for threshold cycle value, “target” indicates the gene of interest, and “norm” indicates RNA for normalization. ChIP and ChRIP were performed as described (Listerman et al. 2006). Approximately 2 × 107 cells were used per IP. The antibodies used were 10 µL of α-CBP80 serum, 4 µg of α-RNA Pol II, 12 µg of α-GFP, 30 µg of α-U2AF-65, 3 µL of α-U5-116K serum, and 9 µL of α-AcH4 serum and control (nonimmune mouse or goat IgGs).

RT-qPCR, ChIP, and ChRIP

Total RNA was isolated using TRIzol, DNase-treated, and reverse-transcribed from an oligo(dT) or specific reverse primer for 18S rRNA. See Supplemental Table 3 for primers. To calculate fold changes in mRNA levels, the following equation was used: 2ΔCt(target)/2ΔCt(norm), where ΔCt = Ct (control sample) – Ct (experimental sample). Ct stands for threshold cycle value, “target” indicates the gene of interest, and “norm” indicates RNA for normalization. ChIP and ChRIP were performed as described (Listerman et al. 2006). Approximately 2 × 107 cells were used per IP. The antibodies used were 10 µL of α-CBP80 serum, 4 µg of α-RNA Pol II, 12 µg of α-GFP, 30 µg of α-U2AF-65, 3 µL of α-U5-116K serum, and 9 µL of α-AcH4 serum and control (nonimmune mouse or goat IgGs).

FRAPE3 cells were maintained in Phenol Red–free Leibovitz’s L-15 with 10% FCS at 37°C. For CBP80 knockdown, cells were transfected with CBP80 or negative Stealth RNAi in Optimem Plus Glutamax medium, using Lipofectamine 2000. After 48 h, cells were transfected with MS2-mCherry using FuGENE 6 in order to mark active transcription sites, and then transcriptionally induced with doxycycline (1 μg/mL) for several hours. FRAP experiments were performed 1 d later. FRAP image sequences were obtained on an Olympus FV1000 inverted scanning confocal microscope with a heated chamber and objective heater (37°C) and a 60×, 1.35 NA oil objective. Cells were scanned using a 488-nm laser for detection of GFP-labeled U1-70K, U2AF65, or Prp8 and a 561-nm laser for the detection of MS2-mcherry. GFP-tagged U1-70K, U2AF65, and Prp8 were bleached using the 488-nm laser in the nucleoplasm, speckles, and at the active transcription site. Five pre-bleach images were acquired. Post-bleach images were acquired at a frequency of four images per 2 sec. For analysis of fluorescence recovery, FRAP data were normalized and calculated as previously described (Shav-Tal et al. 2005). Statistical significance was assessed with the Mann–Whitney test. Detection of active transcription sites in fixed cells was by RNA FISH to nascent mRNAs, using a Cy3-labeled probe to the MS2 sequence repeats in the 3′ UTR of the mRNA (Brody et al. 2011).

E3 cells were maintained in Phenol Red–free Leibovitz’s L-15 with 10% FCS at 37°C. For CBP80 knockdown, cells were transfected with CBP80 or negative Stealth RNAi in Optimem Plus Glutamax medium, using Lipofectamine 2000. After 48 h, cells were transfected with MS2-mCherry using FuGENE 6 in order to mark active transcription sites, and then transcriptionally induced with doxycycline (1 μg/mL) for several hours. FRAP experiments were performed 1 d later. FRAP image sequences were obtained on an Olympus FV1000 inverted scanning confocal microscope with a heated chamber and objective heater (37°C) and a 60×, 1.35 NA oil objective. Cells were scanned using a 488-nm laser for detection of GFP-labeled U1-70K, U2AF65, or Prp8 and a 561-nm laser for the detection of MS2-mcherry. GFP-tagged U1-70K, U2AF65, and Prp8 were bleached using the 488-nm laser in the nucleoplasm, speckles, and at the active transcription site. Five pre-bleach images were acquired. Post-bleach images were acquired at a frequency of four images per 2 sec. For analysis of fluorescence recovery, FRAP data were normalized and calculated as previously described (Shav-Tal et al. 2005). Statistical significance was assessed with the Mann–Whitney test. Detection of active transcription sites in fixed cells was by RNA FISH to nascent mRNAs, using a Cy3-labeled probe to the MS2 sequence repeats in the 3′ UTR of the mRNA (Brody et al. 2011).

AntibodiesThe following mouse monoclonal antibodies were used: α-GAPDH (Novus Biologicals); α-RNA Pol II CTD, 4H8 (Abcam); α-TMG, K121 (Calbiochem); α-U2AF-65, MC3 (Gama-Carvalho et al. 1997); and α-Sm proteins, Y12 (Lerner et al. 1981). The following rabbit polyclonal antibodies were used: α-AcH4 (Upstate); α-CBP80 and α-CBP20 (Izaurralde et al. 1994, 1995); α-SART-3 (Stanek et al. 2003); and α-U5-116K (Fabrizio et al. 1997). Goat polyclonal α-GFP was a gift of D. Drechsel (MPI-CBG). Nonimmune IgGs and HRP-conjugated secondary antibodies were from Sigma-Aldrich or GE Healthcare.

The following mouse monoclonal antibodies were used: α-GAPDH (Novus Biologicals); α-RNA Pol II CTD, 4H8 (Abcam); α-TMG, K121 (Calbiochem); α-U2AF-65, MC3 (Gama-Carvalho et al. 1997); and α-Sm proteins, Y12 (Lerner et al. 1981). The following rabbit polyclonal antibodies were used: α-AcH4 (Upstate); α-CBP80 and α-CBP20 (Izaurralde et al. 1994, 1995); α-SART-3 (Stanek et al. 2003); and α-U5-116K (Fabrizio et al. 1997). Goat polyclonal α-GFP was a gift of D. Drechsel (MPI-CBG). Nonimmune IgGs and HRP-conjugated secondary antibodies were from Sigma-Aldrich or GE Healthcare.

SUPPLEMENTAL MATERIALSupplemental material is available for this article.

SUPPLEMENTAL MATERIAL

Supplemental material is available for this article.
