---
pmid: '39001778'
title: Penicillin-binding protein 3 sequence variations reduce susceptibility of Pseudomonas
  aeruginosa to β-lactams but inhibit cell division.
authors:
- Glen KA
- Lamont IL
journal: J Antimicrob Chemother
year: '2024'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC11368433
doi: 10.1093/jac/dkae203
---

# Penicillin-binding protein 3 sequence variations reduce susceptibility of Pseudomonas aeruginosa to β-lactams but inhibit cell division.
**Authors:** Glen KA, Lamont IL
**Journal:** J Antimicrob Chemother (2024)
**DOI:** [10.1093/jac/dkae203](https://doi.org/10.1093/jac/dkae203)
**PMC:** [PMC11368433](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11368433/)

## Abstract

1. J Antimicrob Chemother. 2024 Sep 3;79(9):2170-2178. doi: 10.1093/jac/dkae203.

Penicillin-binding protein 3 sequence variations reduce susceptibility of 
Pseudomonas aeruginosa to β-lactams but inhibit cell division.

Glen KA(1), Lamont IL(1).

Author information:
(1)Department of Biochemistry, University of Otago, PO Box 56, Dunedin 9054, New 
Zealand.

BACKGROUND: β-lactam antibiotics, which inhibit penicillin-binding protein 3 
(PBP3) that is required for cell division, play a key role in treating P. 
aeruginosa infections. Some sequence variations in PBP3 have been associated 
with β-lactam resistance but the effects of variations on antibiotic 
susceptibility and on cell division have not been quantified. Antibiotic efflux 
can also reduce susceptibility.
OBJECTIVES: To quantify the effects of PBP3 variations on β-lactam 
susceptibility and cell morphology in P. aeruginosa.
METHODS: Nineteen PBP3 variants were expressed from a plasmid in the reference 
strain P. aeruginosa PAO1 and genome engineering was used to construct five 
mutants expressing PBP3 variants from the chromosome. The effects of the 
variations on β-lactam minimum inhibitory concentration (MIC) and cell 
morphology were measured.
RESULTS: Some PBP3 variations reduced susceptibility to a variety of β-lactam 
antibiotics including meropenem, ceftazidime, cefepime and ticarcillin with 
different variations affecting different antibiotics. None of the tested 
variations reduced susceptibility to imipenem or piperacillin. Antibiotic 
susceptibility was further reduced when PBP3 variants were expressed in mutant 
bacteria overexpressing the MexAB-OprM efflux pump, with some variations 
conferring clinical levels of resistance. Some PBP3 variations, and sub-MIC 
levels of β-lactams, reduced bacterial growth rates and inhibited cell division, 
causing elongated cells.
CONCLUSIONS: PBP3 variations in P. aeruginosa can increase the MIC of multiple 
β-lactam antibiotics, although not imipenem or piperacillin. PBP3 variations, or 
the presence of sub-lethal levels of β-lactams, result in elongated cells 
indicating that variations reduce the activity of PBP3 and may reduce bacterial 
fitness.

© The Author(s) 2024. Published by Oxford University Press on behalf of British 
Society for Antimicrobial Chemotherapy.

DOI: 10.1093/jac/dkae203
PMCID: PMC11368433
PMID: 39001778 [Indexed for MEDLINE]

## Full Text

Introduction
Pseudomonas aeruginosa is an opportunistic pathogen that is a common cause of acute hospital-acquired infections, septicaemia, urinary tract infections, pneumonia and bacteraemia.1–4 It can also chronically infect the lungs of people with cystic fibrosis and other forms of lung disease.5,6 Infections are associated with high levels of mortality and morbidity.7,8 β-lactam antibiotics are commonly used in the treatment of P. aeruginosa infections, with carbapenems such as meropenem and imipenem being especially important as they represent one of the last lines of defence against antibiotic-resistant P. aeruginosa.9 Cephalosporins (ceftazidime and cefepime), penicillins (piperacillin and ticarcillin) and aztreonam (a monobactam) are also widely used in treating infections.10 β-lactams covalently bind the catalytic site of penicillin-binding proteins (PBPs) causing permanent inactivation of these enzymes.11 PBP3, encoded by the ftsI gene, is required for cell division and as the only essential PBP in P. aeruginosa is an important target for β-lactam antibiotics, with inhibition of PBP3 causing filamentation and cell lysis.12 PBP3 is part of a multienzyme complex termed the divisome and is composed of a catalytic domain and a domain that is involved in protein–protein interactions.13A significant proportion of P. aeruginosa isolates are resistant to β-lactams and such bacteria cause thousands of deaths each year worldwide.1,14,15 Carbapenem-resistant P. aeruginosa have been classified by the WHO as a critical target for development of new treatments16 because infections with these organisms are particularly difficult to manage and infected patients have high mortality rates.17,18 Resistance to β-lactams is multifactorial. It is associated with mutations that inactivate the porin OprD that enables entry of some β-lactams into the bacteria, that alter the activity of the intrinsic AmpC β-lactamase19 or that upregulate the multi-drug efflux pump MexAB-OprM.20–22 Resistance can also arise through the acquisition of β-lactamases that degrade β-lactams.23,24 Recently, mutations in ftsI that result in PBP3 sequence variations have also been associated with β-lactam resistance in P. aeruginosa. Mutations have been identified by comparing the genomes of antibiotic-resistant and antibiotic-susceptible isolates25–37 and by experimentally evolving antibiotic-resistant mutants from antibiotic-susceptible laboratory reference strains in P. aeruginosa.26,27,38–41 For example, three of 13 P. aeruginosa mutants that had been experimentally evolved to be resistant to meropenem had acquired PBP3 variations.38 A sequence variation, F533L, reduced the affinity of PBP3 protein for meropenem providing a mechanistic link between this variation and reduced susceptibility to β-lactams.39 Mutations resulting in amino acid substitutions in PBPs can also reduce the affinity of these antibiotics for their target proteins in E. coli.42 However, although PBP3 variations are often present in β-lactam-resistant isolates of P. aeruginosa, their impact on antibiotic susceptibility in the absence of mutations in other genes is unknown. The primary aim of this research was to quantify the effects of commonly occurring PBP3 variations on the susceptibility of P. aeruginosa to a wide range of β-lactam antibiotics. In addition, due to the critical role of PBP3 in cell division, the effects of PBP3 variations on cell division were investigated.

Introduction

Pseudomonas aeruginosa is an opportunistic pathogen that is a common cause of acute hospital-acquired infections, septicaemia, urinary tract infections, pneumonia and bacteraemia.1–4 It can also chronically infect the lungs of people with cystic fibrosis and other forms of lung disease.5,6 Infections are associated with high levels of mortality and morbidity.7,8 β-lactam antibiotics are commonly used in the treatment of P. aeruginosa infections, with carbapenems such as meropenem and imipenem being especially important as they represent one of the last lines of defence against antibiotic-resistant P. aeruginosa.9 Cephalosporins (ceftazidime and cefepime), penicillins (piperacillin and ticarcillin) and aztreonam (a monobactam) are also widely used in treating infections.10 β-lactams covalently bind the catalytic site of penicillin-binding proteins (PBPs) causing permanent inactivation of these enzymes.11 PBP3, encoded by the ftsI gene, is required for cell division and as the only essential PBP in P. aeruginosa is an important target for β-lactam antibiotics, with inhibition of PBP3 causing filamentation and cell lysis.12 PBP3 is part of a multienzyme complex termed the divisome and is composed of a catalytic domain and a domain that is involved in protein–protein interactions.13

A significant proportion of P. aeruginosa isolates are resistant to β-lactams and such bacteria cause thousands of deaths each year worldwide.1,14,15 Carbapenem-resistant P. aeruginosa have been classified by the WHO as a critical target for development of new treatments16 because infections with these organisms are particularly difficult to manage and infected patients have high mortality rates.17,18 Resistance to β-lactams is multifactorial. It is associated with mutations that inactivate the porin OprD that enables entry of some β-lactams into the bacteria, that alter the activity of the intrinsic AmpC β-lactamase19 or that upregulate the multi-drug efflux pump MexAB-OprM.20–22 Resistance can also arise through the acquisition of β-lactamases that degrade β-lactams.23,24 Recently, mutations in ftsI that result in PBP3 sequence variations have also been associated with β-lactam resistance in P. aeruginosa. Mutations have been identified by comparing the genomes of antibiotic-resistant and antibiotic-susceptible isolates25–37 and by experimentally evolving antibiotic-resistant mutants from antibiotic-susceptible laboratory reference strains in P. aeruginosa.26,27,38–41 For example, three of 13 P. aeruginosa mutants that had been experimentally evolved to be resistant to meropenem had acquired PBP3 variations.38 A sequence variation, F533L, reduced the affinity of PBP3 protein for meropenem providing a mechanistic link between this variation and reduced susceptibility to β-lactams.39 Mutations resulting in amino acid substitutions in PBPs can also reduce the affinity of these antibiotics for their target proteins in E. coli.42 However, although PBP3 variations are often present in β-lactam-resistant isolates of P. aeruginosa, their impact on antibiotic susceptibility in the absence of mutations in other genes is unknown. The primary aim of this research was to quantify the effects of commonly occurring PBP3 variations on the susceptibility of P. aeruginosa to a wide range of β-lactam antibiotics. In addition, due to the critical role of PBP3 in cell division, the effects of PBP3 variations on cell division were investigated.

Materials and methodsBacteria and growth conditionsBacterial strains and plasmids that were used in this study are listed in Table S1 (available as Supplementary data at JAC Online). Bacteria were grown at 37°C in Luria–Bertani (LB) medium with shaking at 200 rpm or on lysogeny agar (LA), except that for conjugation, P. aeruginosa PAO1 was grown at 42°C (standing culture) in LB containing 0.4% potassium nitrate as an alternative electron acceptor for microaerobic growth. Media were supplemented with gentamicin (E. coli, 20 mg/L; P. aeruginosa 30 mg/L) tetracycline (E. coli, 12.5 mg/L; P. aeruginosa, 25 mg/L) and δ-aminolaevulinic acid (ALA) (50 mg/L) as required.Molecular biology methodsGenomic DNA was purified using the UltraClean Microbial Kit (Qiagen, Hilden, Germany) plasmids were purified using the NucleoSpin® Plasmid kit (Macherey Nagel, Dueren, Germany) and PCR products were purified and extracted from agarose gels using the NucleoSpin® Gel and PCR Clean-up kit (Macherey Nagel, Dueren, Germany) according to the manufacturer’s instructions.For DNA cloning, PCR amplification was carried out with genomic DNA as template using high-fidelity Q5 polymerase (New England Biolabs, Ipswich, MA, USA) with appropriate primers (Table S2). Amplicons for expression of the PBP3-encoding ftsI gene included the predicted ribosome binding site and those for engineering ftsI mutations into the chromosome extended approximately 1 kb either side of the targeted mutation. Additional ftsI alleles were purchased as synthetic DNA from GenScript Biotech (Singapore). Amplicons and synthetic alleles were cloned into pJN10543 for expression, or pEX18Tc44 for allele exchange, using appropriate restriction enzymes (New England Biolabs). All plasmid constructs were sequenced to confirm that the intended sequence was present. pJN105 constructs were transformed into P. aeruginosa made competent by sucrose washing.45 pEX18Tc constructs were transformed into E. coli JM83 and then transferred into the P. aeruginosa PAO1 chromosome by conjugation followed by homologous recombination, with E. coli ST18 serving as the donor strain, as previously described.46–48 Mutants in which the wild-type allele had been replaced by the introduced allele were identified by sucrose selection followed by PCR amplification and sequencing using primers specific to the mutation-containing DNA region.Minimum inhibitory concentration (MIC) testingOvernight cultures of P. aeruginosa were diluted to 1.5 × 106 cfu/mL and two 5 µL aliquots were spotted onto LA plates supplemented with doubling concentrations of antibiotic.38,49 The plates were incubated for 24 h. The MIC of each antibiotic was defined as the lowest concentration that inhibited growth. CLSI breakpoints were used to classify resistant and sensitive phenotypes.50 LA, rather than Mueller–Hinton agar, was used for MIC testing to be consistent with the media used for other experiments. In our hands, MIC values for P. aeruginosa are the same on Mueller–Hinton agar and LA.Bacterial growth kineticsGrowth of bacteria was analysed as described previously.38 In brief, overnight cultures were diluted to 1.5 × 106 cfu/mL in LB and portions (200 µL) dispensed into wells of a Corning® 96-well Clear Round Bottom TC-treated Microplate (Corning, Durham, NC, USA). The plates were incubated at 37°C at 200 rpm for 24 h in a BMG FLUOstar Omega microplate reader, and the optical density at 600 nm (OD600) was recorded every 30 min for 24 h. The OD600 at each time point was then blank-corrected before calculating the area under the curve (AUC). AUC analysis was performed using GraphPad prism v.9. The logistic AUC was used as a metric of growth. This metric provides a measure of growth that includes lag phase, log phase growth rate and final cell density. Growth analysis was performed with three biological replicates (three technical replicates each) in all cases.Cell length measurementOvernight cultures were diluted to 1.5 × 106 cfu/mL in LB, supplemented with antibiotics as required, and portions (200 µL) dispensed into Corning® 96-well Clear Round Bottom TC-treated Microplate. The plates were incubated at 37°C at 200 rpm for 7 or 24 hours in a BMG FLUOstar Omega microplate reader. Portions of culture (10–30 µL) were placed onto a glass microscope slide and allowed to dry at room temperature. Gram staining was carried out using the Remel Gram Stain Kit (Remel, Kansas, USA) according to the manufacturer’s instructions. Slides were imaged under ×1000 oil immersion using an ECLIPSE Ci-L microscope fitted with a TS-HD Lite camera (Nikon). The lengths of separated cells were measured using ImageJ.51 Lengths of all cells on a minimum of two separate microscope images for each of three biological repeats were counted for mutants expressing PBP3 variants. For bacteria grown in the presence of antibiotics, 101 cells were counted for each treatment. Cell length statistics were calculated with GraphPad prism v.9. Mann–Whitney U tests were performed on the cell length data.Cell biomassOvernight cultures were diluted to 1.5 × 106 cfu/mL in 115 mL of LB. Cultures were incubated at 37°C until the OD600 of 1.25 was reached. Then 100 grams of cell culture was centrifuged at 9000 g and the pellets were washed twice with 5 mL of phosphate-buffered saline (pH 7.2). The weights of the cell pellets were then measured. Finally, t-tests were performed on data from three biological replicates of each strain.Bioinformatic analysis
P. aeruginosa genomes in our collection (Table S3) and in the NCBI database (https://www.ncbi.nlm.nih.gov/using nt/nr database) were screened in a tBLASTn search with PBP3 from P. aeruginosa PAO1 as the query sequence to identify genomes that encoded PBP3 with different sequences (sequence variations). The effects of sequence variations on protein function were predicted using PROVEAN (protein variation effect analyser) v.1.1.5, with values below −2.5 considered to be significant.52 Protein structures were visualized using PyMOl (Schrödinger, LLC). For phylogenetic analysis, ParSNP v.1.2 from the Harvest suite v.1.1.2 was used to construct a core genome alignment of isolates with P. aeruginosa PAO1 as the reference strain.53 A phylogenetic tree was generated from the alignment using Parsnp54 and visualized on TreeGraph v.2.55

Materials and methods

Bacteria and growth conditionsBacterial strains and plasmids that were used in this study are listed in Table S1 (available as Supplementary data at JAC Online). Bacteria were grown at 37°C in Luria–Bertani (LB) medium with shaking at 200 rpm or on lysogeny agar (LA), except that for conjugation, P. aeruginosa PAO1 was grown at 42°C (standing culture) in LB containing 0.4% potassium nitrate as an alternative electron acceptor for microaerobic growth. Media were supplemented with gentamicin (E. coli, 20 mg/L; P. aeruginosa 30 mg/L) tetracycline (E. coli, 12.5 mg/L; P. aeruginosa, 25 mg/L) and δ-aminolaevulinic acid (ALA) (50 mg/L) as required.

Bacteria and growth conditions

Bacterial strains and plasmids that were used in this study are listed in Table S1 (available as Supplementary data at JAC Online). Bacteria were grown at 37°C in Luria–Bertani (LB) medium with shaking at 200 rpm or on lysogeny agar (LA), except that for conjugation, P. aeruginosa PAO1 was grown at 42°C (standing culture) in LB containing 0.4% potassium nitrate as an alternative electron acceptor for microaerobic growth. Media were supplemented with gentamicin (E. coli, 20 mg/L; P. aeruginosa 30 mg/L) tetracycline (E. coli, 12.5 mg/L; P. aeruginosa, 25 mg/L) and δ-aminolaevulinic acid (ALA) (50 mg/L) as required.

Molecular biology methodsGenomic DNA was purified using the UltraClean Microbial Kit (Qiagen, Hilden, Germany) plasmids were purified using the NucleoSpin® Plasmid kit (Macherey Nagel, Dueren, Germany) and PCR products were purified and extracted from agarose gels using the NucleoSpin® Gel and PCR Clean-up kit (Macherey Nagel, Dueren, Germany) according to the manufacturer’s instructions.For DNA cloning, PCR amplification was carried out with genomic DNA as template using high-fidelity Q5 polymerase (New England Biolabs, Ipswich, MA, USA) with appropriate primers (Table S2). Amplicons for expression of the PBP3-encoding ftsI gene included the predicted ribosome binding site and those for engineering ftsI mutations into the chromosome extended approximately 1 kb either side of the targeted mutation. Additional ftsI alleles were purchased as synthetic DNA from GenScript Biotech (Singapore). Amplicons and synthetic alleles were cloned into pJN10543 for expression, or pEX18Tc44 for allele exchange, using appropriate restriction enzymes (New England Biolabs). All plasmid constructs were sequenced to confirm that the intended sequence was present. pJN105 constructs were transformed into P. aeruginosa made competent by sucrose washing.45 pEX18Tc constructs were transformed into E. coli JM83 and then transferred into the P. aeruginosa PAO1 chromosome by conjugation followed by homologous recombination, with E. coli ST18 serving as the donor strain, as previously described.46–48 Mutants in which the wild-type allele had been replaced by the introduced allele were identified by sucrose selection followed by PCR amplification and sequencing using primers specific to the mutation-containing DNA region.

Molecular biology methods

Genomic DNA was purified using the UltraClean Microbial Kit (Qiagen, Hilden, Germany) plasmids were purified using the NucleoSpin® Plasmid kit (Macherey Nagel, Dueren, Germany) and PCR products were purified and extracted from agarose gels using the NucleoSpin® Gel and PCR Clean-up kit (Macherey Nagel, Dueren, Germany) according to the manufacturer’s instructions.

For DNA cloning, PCR amplification was carried out with genomic DNA as template using high-fidelity Q5 polymerase (New England Biolabs, Ipswich, MA, USA) with appropriate primers (Table S2). Amplicons for expression of the PBP3-encoding ftsI gene included the predicted ribosome binding site and those for engineering ftsI mutations into the chromosome extended approximately 1 kb either side of the targeted mutation. Additional ftsI alleles were purchased as synthetic DNA from GenScript Biotech (Singapore). Amplicons and synthetic alleles were cloned into pJN10543 for expression, or pEX18Tc44 for allele exchange, using appropriate restriction enzymes (New England Biolabs). All plasmid constructs were sequenced to confirm that the intended sequence was present. pJN105 constructs were transformed into P. aeruginosa made competent by sucrose washing.45 pEX18Tc constructs were transformed into E. coli JM83 and then transferred into the P. aeruginosa PAO1 chromosome by conjugation followed by homologous recombination, with E. coli ST18 serving as the donor strain, as previously described.46–48 Mutants in which the wild-type allele had been replaced by the introduced allele were identified by sucrose selection followed by PCR amplification and sequencing using primers specific to the mutation-containing DNA region.

Minimum inhibitory concentration (MIC) testingOvernight cultures of P. aeruginosa were diluted to 1.5 × 106 cfu/mL and two 5 µL aliquots were spotted onto LA plates supplemented with doubling concentrations of antibiotic.38,49 The plates were incubated for 24 h. The MIC of each antibiotic was defined as the lowest concentration that inhibited growth. CLSI breakpoints were used to classify resistant and sensitive phenotypes.50 LA, rather than Mueller–Hinton agar, was used for MIC testing to be consistent with the media used for other experiments. In our hands, MIC values for P. aeruginosa are the same on Mueller–Hinton agar and LA.

Minimum inhibitory concentration (MIC) testing

Overnight cultures of P. aeruginosa were diluted to 1.5 × 106 cfu/mL and two 5 µL aliquots were spotted onto LA plates supplemented with doubling concentrations of antibiotic.38,49 The plates were incubated for 24 h. The MIC of each antibiotic was defined as the lowest concentration that inhibited growth. CLSI breakpoints were used to classify resistant and sensitive phenotypes.50 LA, rather than Mueller–Hinton agar, was used for MIC testing to be consistent with the media used for other experiments. In our hands, MIC values for P. aeruginosa are the same on Mueller–Hinton agar and LA.

Bacterial growth kineticsGrowth of bacteria was analysed as described previously.38 In brief, overnight cultures were diluted to 1.5 × 106 cfu/mL in LB and portions (200 µL) dispensed into wells of a Corning® 96-well Clear Round Bottom TC-treated Microplate (Corning, Durham, NC, USA). The plates were incubated at 37°C at 200 rpm for 24 h in a BMG FLUOstar Omega microplate reader, and the optical density at 600 nm (OD600) was recorded every 30 min for 24 h. The OD600 at each time point was then blank-corrected before calculating the area under the curve (AUC). AUC analysis was performed using GraphPad prism v.9. The logistic AUC was used as a metric of growth. This metric provides a measure of growth that includes lag phase, log phase growth rate and final cell density. Growth analysis was performed with three biological replicates (three technical replicates each) in all cases.

Bacterial growth kinetics

Growth of bacteria was analysed as described previously.38 In brief, overnight cultures were diluted to 1.5 × 106 cfu/mL in LB and portions (200 µL) dispensed into wells of a Corning® 96-well Clear Round Bottom TC-treated Microplate (Corning, Durham, NC, USA). The plates were incubated at 37°C at 200 rpm for 24 h in a BMG FLUOstar Omega microplate reader, and the optical density at 600 nm (OD600) was recorded every 30 min for 24 h. The OD600 at each time point was then blank-corrected before calculating the area under the curve (AUC). AUC analysis was performed using GraphPad prism v.9. The logistic AUC was used as a metric of growth. This metric provides a measure of growth that includes lag phase, log phase growth rate and final cell density. Growth analysis was performed with three biological replicates (three technical replicates each) in all cases.

Cell length measurementOvernight cultures were diluted to 1.5 × 106 cfu/mL in LB, supplemented with antibiotics as required, and portions (200 µL) dispensed into Corning® 96-well Clear Round Bottom TC-treated Microplate. The plates were incubated at 37°C at 200 rpm for 7 or 24 hours in a BMG FLUOstar Omega microplate reader. Portions of culture (10–30 µL) were placed onto a glass microscope slide and allowed to dry at room temperature. Gram staining was carried out using the Remel Gram Stain Kit (Remel, Kansas, USA) according to the manufacturer’s instructions. Slides were imaged under ×1000 oil immersion using an ECLIPSE Ci-L microscope fitted with a TS-HD Lite camera (Nikon). The lengths of separated cells were measured using ImageJ.51 Lengths of all cells on a minimum of two separate microscope images for each of three biological repeats were counted for mutants expressing PBP3 variants. For bacteria grown in the presence of antibiotics, 101 cells were counted for each treatment. Cell length statistics were calculated with GraphPad prism v.9. Mann–Whitney U tests were performed on the cell length data.

Cell length measurement

Overnight cultures were diluted to 1.5 × 106 cfu/mL in LB, supplemented with antibiotics as required, and portions (200 µL) dispensed into Corning® 96-well Clear Round Bottom TC-treated Microplate. The plates were incubated at 37°C at 200 rpm for 7 or 24 hours in a BMG FLUOstar Omega microplate reader. Portions of culture (10–30 µL) were placed onto a glass microscope slide and allowed to dry at room temperature. Gram staining was carried out using the Remel Gram Stain Kit (Remel, Kansas, USA) according to the manufacturer’s instructions. Slides were imaged under ×1000 oil immersion using an ECLIPSE Ci-L microscope fitted with a TS-HD Lite camera (Nikon). The lengths of separated cells were measured using ImageJ.51 Lengths of all cells on a minimum of two separate microscope images for each of three biological repeats were counted for mutants expressing PBP3 variants. For bacteria grown in the presence of antibiotics, 101 cells were counted for each treatment. Cell length statistics were calculated with GraphPad prism v.9. Mann–Whitney U tests were performed on the cell length data.

Cell biomassOvernight cultures were diluted to 1.5 × 106 cfu/mL in 115 mL of LB. Cultures were incubated at 37°C until the OD600 of 1.25 was reached. Then 100 grams of cell culture was centrifuged at 9000 g and the pellets were washed twice with 5 mL of phosphate-buffered saline (pH 7.2). The weights of the cell pellets were then measured. Finally, t-tests were performed on data from three biological replicates of each strain.

Cell biomass

Overnight cultures were diluted to 1.5 × 106 cfu/mL in 115 mL of LB. Cultures were incubated at 37°C until the OD600 of 1.25 was reached. Then 100 grams of cell culture was centrifuged at 9000 g and the pellets were washed twice with 5 mL of phosphate-buffered saline (pH 7.2). The weights of the cell pellets were then measured. Finally, t-tests were performed on data from three biological replicates of each strain.

Bioinformatic analysis
P. aeruginosa genomes in our collection (Table S3) and in the NCBI database (https://www.ncbi.nlm.nih.gov/using nt/nr database) were screened in a tBLASTn search with PBP3 from P. aeruginosa PAO1 as the query sequence to identify genomes that encoded PBP3 with different sequences (sequence variations). The effects of sequence variations on protein function were predicted using PROVEAN (protein variation effect analyser) v.1.1.5, with values below −2.5 considered to be significant.52 Protein structures were visualized using PyMOl (Schrödinger, LLC). For phylogenetic analysis, ParSNP v.1.2 from the Harvest suite v.1.1.2 was used to construct a core genome alignment of isolates with P. aeruginosa PAO1 as the reference strain.53 A phylogenetic tree was generated from the alignment using Parsnp54 and visualized on TreeGraph v.2.55

Bioinformatic analysis

P. aeruginosa genomes in our collection (Table S3) and in the NCBI database (https://www.ncbi.nlm.nih.gov/using nt/nr database) were screened in a tBLASTn search with PBP3 from P. aeruginosa PAO1 as the query sequence to identify genomes that encoded PBP3 with different sequences (sequence variations). The effects of sequence variations on protein function were predicted using PROVEAN (protein variation effect analyser) v.1.1.5, with values below −2.5 considered to be significant.52 Protein structures were visualized using PyMOl (Schrödinger, LLC). For phylogenetic analysis, ParSNP v.1.2 from the Harvest suite v.1.1.2 was used to construct a core genome alignment of isolates with P. aeruginosa PAO1 as the reference strain.53 A phylogenetic tree was generated from the alignment using Parsnp54 and visualized on TreeGraph v.2.55

ResultsIdentifying PBP3 variations that reduce susceptibility to β-lactamsThe genomes of 259 P. aeruginosa isolates in our collection were screened by tBLASTn searches for PBP3 with different sequences from that of the β-lactam-sensitive reference strain PAO1 (sequence variations). Combined with variations identified previously in clinical isolates and in experimentally evolved β-lactam resistant mutants, a total of 91 PBP3 variations were identified (Figure S1). P. aeruginosa isolates that express PBP3 variants were widely dispersed on a phylogenetic tree of P. aeruginosa (Figure S2).Seventeen PBP3 variations were chosen for initial study and to allow selection of key variations for more detailed investigation. PBP3 variations were chosen on the basis of their prevalence in different isolates, the likelihood that the amino acid differences would alter protein function, whether or not they arose in β-lactam experimental evolution studies and whether they were located near the catalytic site of PBP3 (Table S4, Figure S1). In addition, the PBP3 variations T91A and L240V that were present infrequently in isolates and did not arise in experimental evolution studies were included for comparison.As an initial screen to determine which differences alter antibiotic MIC, ftsI genes encoding the 19 PBP3 variants were cloned into plasmid expression vector pJN105 and transformed into the antibiotic-sensitive P. aeruginosa laboratory reference strain PAO1. This approach was used in preference to engineering of sequence variations into the chromosome because it is faster and less complex, allowing high throughput screening of variations of interest.MIC testing was carried out on bacteria expressing the PBP3 variants for representative carbapenem, cephalosporin and monobactam antibiotics (Table S4). None of the PBP3 variations increased imipenem or aztreonam MIC and nine of the variations did not alter the MIC of any of the tested antibiotics. The other 10 variations increased the MIC of meropenem and/or ceftazidime. The amino acid differences that increased the MICs of β-lactams were all located in the catalytic domain of PBP3.The effects of chromosomally encoded PBP3 variants on β-lactam susceptibilityThe approach described previously provides a good initial screen for the effects of PBP3 variations. However, expression from the plasmid-encoded gene was likely to be higher than that of chromosomally expressed PBP3. In addition, the presence of the chromosomal wild-type allele as well as the plasmid may dampen the effects of plasmid-encoded PBP3 variants. To better represent the normal situation, five ftsI alleles encoding PBP3 variants that reduced susceptibility to at least one β-lactam when expressed from pJN105 were engineered into the chromosome of P. aeruginosa PAO1, replacing the wild-type gene. MIC testing was performed on the resulting mutants with a range of antibiotics covering all classes of clinically used β-lactams. The introduced variations increased the MIC of between three and six of the tested antibiotics (Table 1). However, none of the tested variations increased the MIC of imipenem or piperacillin and indeed, three variations slightly reduced the piperacillin MIC.Table 1.MIC values for P. aeruginosa with genome-encoded PBP3 variantsAntibiotic MIC (mg/L)aPBP3 variationMerImiCeftAztCarbPipTicCefpWild-type0.540.5232280.5A244T140.254321162V471G240.521281320.5R504C84182562642F533L840.52321161V537L440.581282322Values are medians of three biological replicates.Mer, Meropenem; Imi, Imipenem; Ceft, Ceftazidime; Azt, Aztreonam; Carb, carbenicillin; Pip, piperacillin; Tic, Ticaracillin; Cefp, Cefipeme.Effects of PBP3 variations in conjunction with increased effluxMutations altering the MexR repressor protein increase expression of the MexAB-OprM efflux pump and are associated with reduced susceptibility to β-lactam antibiotics.20,56 To investigate the effects of PBP3 variations in combination with increased mexAB-oprM expression, pJN105 expressing PBP3 variants with changes in the catalytic domain were transformed into a PAO1 mexR mutant strain and MICs were determined (Table 2). The PBP3 variations were chosen because they increased β-lactam MIC in wild-type strain PAO1 (Table S4) or were present in multiple clinical isolates (Figure S1).Table 2.MIC values for a P. aeruginosa mexR mutant expressing PBP3 variants from an expression vectorAntibiotic MIC (mg/L) aPBP3 variationMeropenemImipenemCeftazidimeAztreonamWild-type84216L240V84416A244T164232Y367C84216V465G84216V471G164216R504C324432F507L84216I524T164232P527S164232F533L324216V537L164232Values are medians of three biological replicates. P. aeruginosa are classified as being resistant if the MIC is ≥8 mg/L (meropenem and imipenem) or ≥32 mg/L (ceftazidime and aztreonam).50In bacteria expressing only the chromosomal (wild-type) PBP3, the mexR mutation increased ß-lactam MICs of meropenem by 16-fold, of ceftazidime by 4-fold, and of aztreonam by 8-fold but did not increase the MIC of imipenem (Table 1, Table S4) consistent with previous findings that MexAB-OprM does not export imipenem.10,56,57 Eight of the 11 PBP3 variations tested further increased the MIC of at least one antibiotic. Notably, the V537L, A244T, I524T and P527S PBP3 variations all increased aztreonam MIC in the mexR mutant but not when expressed in wild-type PAO1. None of the tested PBP3 variations increased the MIC of imipenem in the mexR mutant.Effects of PBP3 variations on growth and morphologyAs PBP3 is an essential protein required for cell division, we tested the hypothesis that mutations altering PBP3 affect bacterial growth. Three of the five mutants expressing a PBP3 variant had a significant reduction in growth, as measured by absorbance, relative to wild-type (Figure 1). The PBP3 variation A244T caused the greatest reduction in rate of change of absorbance. A reduced rate of absorbance increase indicates that cells were generated at a lower rate. A reduced rate of cell formation may be due to a reduced rate of incorporation of nutrients into cells (i.e. slower formation of biomass and generation of normal sized cells). Alternatively, it could occur due to reduced rates of cell division and consequent increases in cell size with no change in the rate of increase of biomass.58 The biomass of the bacteria in liquid culture was measured to distinguish between these possibilities.Figure 1.Growth of P. aeruginosa PAO1 with chromosomally expressed PBP3 variants. Bacteria were grown and growth monitored for 24 h. Values are AUC for three biological replicates (each the mean of three technical replicates). *P < 0.05; **P < 0.01; ns, not significant (unpaired t-tests).The biomass of wild-type bacteria from 100 mL of culture, at OD600 = 1.25, was 0.2328 ± 0.0144 g and that of bacteria with the PBP3 variation A244T was 0.2689 ± 0.0246, which was significantly higher (P = 0.0129) showing that the variation did not result in reduced biomass.The lengths of cells were measured to determine the effects of PBP3 variations on morphology. All tested variations caused cells to be significantly longer than those of wild-type bacteria (Figure 2, Table S5, Figure S3). The mutant expressing the PBP3 variation A244T had the most elongated cells.Figure 2.Cell length of P. aeruginosa with chromosomally expressed PBP3 variants in exponential phase. (a) Representative images of P. aeruginosa with wild-type PBP3 (top) or PBP3 variation A244T (bottom). (b) Lengths of P. aeruginosa cells in exponential growth phase. All cells from 15 images were measured (between 717 and 1913 cells for each strain). The red line is the median length of wild-type cells. All of the PBP3 variations caused average cell length to be significantly (P < 0.001) longer than wild-type (Mann–Whitney U-test). Median and interquartile values are shown in Table S5.Effects of sub-inhibitory concentrations of antibiotics on cell morphologyβ-lactams inhibit PBP3 and this can inhibit cell division and result in filamentation in E. coli.59 As changes to PBP3 result in filamentation of P. aeruginosa, we tested the hypothesis that β-lactam antibiotics would also increase the filamentation of bacteria. P. aeruginosa PAO1, as well as strains expressing the PBP3 variants R504C and F533L that caused the greatest increase in meropenem MIC, were grown with sub-inhibitory concentrations of meropenem and cell length measured.Sub-MIC levels of meropenem caused significant elongation of strain PAO1 even at the lowest concentration tested (1/16 of the MIC) (Figure 3a, Figure S4, Table S6). The average cell length in the presence of 0.5 mg/L meropenem, the maximum concentration that permitted growth, was 20 times longer than that of cells grown in the absence of antibiotics. The two isolates with PBP3 variations that were tested also exhibited cell elongation in the presence of meropenem. However, the effect of the antibiotic on cell length was much less than for strain PAO1.Figure 3.Effects of meropenem and piperacillin on cell length of P. aeruginosa with chromosomally expressed PBP3 variants. (a) Cell length in the presence of varying concentrations of meropenem. (b) Cell length in the presence of varying concentrations of piperacillin: 101 cells were measured under each growth condition. Asterisks indicate significant differences in pairwise comparisons (Mann–Whitney U-test). *P < 0.5; ****P < 0.0001. Median and interquartile values are shown in Table S6.The effects of sub-MIC levels of piperacillin were also tested on strain PAO1 and on the R504C mutant, which has the same MIC, and the V471G mutant, which has a lower MIC than wild-type. The average length of PAO1 cells grown in the presence of 1 mg/L piperacillin increased 30-fold compared to cells grown in the absence of antibiotic, and even the lowest level of piperacillin tested (32 times less than MIC) caused an increase of 1.7 times in average cell length (Figure 3b, Table S6). Piperacillin also increased the average lengths of cells of the mutant strains, by 8.8-fold for the PBP3 variation V471G in the presence of 0.25 mg/L piperacillin and by 15.5-fold for the R504C PBP3 variation in the presence of 0.5 mg/L piperacillin.

Identifying PBP3 variations that reduce susceptibility to β-lactamsThe genomes of 259 P. aeruginosa isolates in our collection were screened by tBLASTn searches for PBP3 with different sequences from that of the β-lactam-sensitive reference strain PAO1 (sequence variations). Combined with variations identified previously in clinical isolates and in experimentally evolved β-lactam resistant mutants, a total of 91 PBP3 variations were identified (Figure S1). P. aeruginosa isolates that express PBP3 variants were widely dispersed on a phylogenetic tree of P. aeruginosa (Figure S2).Seventeen PBP3 variations were chosen for initial study and to allow selection of key variations for more detailed investigation. PBP3 variations were chosen on the basis of their prevalence in different isolates, the likelihood that the amino acid differences would alter protein function, whether or not they arose in β-lactam experimental evolution studies and whether they were located near the catalytic site of PBP3 (Table S4, Figure S1). In addition, the PBP3 variations T91A and L240V that were present infrequently in isolates and did not arise in experimental evolution studies were included for comparison.As an initial screen to determine which differences alter antibiotic MIC, ftsI genes encoding the 19 PBP3 variants were cloned into plasmid expression vector pJN105 and transformed into the antibiotic-sensitive P. aeruginosa laboratory reference strain PAO1. This approach was used in preference to engineering of sequence variations into the chromosome because it is faster and less complex, allowing high throughput screening of variations of interest.MIC testing was carried out on bacteria expressing the PBP3 variants for representative carbapenem, cephalosporin and monobactam antibiotics (Table S4). None of the PBP3 variations increased imipenem or aztreonam MIC and nine of the variations did not alter the MIC of any of the tested antibiotics. The other 10 variations increased the MIC of meropenem and/or ceftazidime. The amino acid differences that increased the MICs of β-lactams were all located in the catalytic domain of PBP3.

Identifying PBP3 variations that reduce susceptibility to β-lactams

The genomes of 259 P. aeruginosa isolates in our collection were screened by tBLASTn searches for PBP3 with different sequences from that of the β-lactam-sensitive reference strain PAO1 (sequence variations). Combined with variations identified previously in clinical isolates and in experimentally evolved β-lactam resistant mutants, a total of 91 PBP3 variations were identified (Figure S1). P. aeruginosa isolates that express PBP3 variants were widely dispersed on a phylogenetic tree of P. aeruginosa (Figure S2).

Seventeen PBP3 variations were chosen for initial study and to allow selection of key variations for more detailed investigation. PBP3 variations were chosen on the basis of their prevalence in different isolates, the likelihood that the amino acid differences would alter protein function, whether or not they arose in β-lactam experimental evolution studies and whether they were located near the catalytic site of PBP3 (Table S4, Figure S1). In addition, the PBP3 variations T91A and L240V that were present infrequently in isolates and did not arise in experimental evolution studies were included for comparison.

As an initial screen to determine which differences alter antibiotic MIC, ftsI genes encoding the 19 PBP3 variants were cloned into plasmid expression vector pJN105 and transformed into the antibiotic-sensitive P. aeruginosa laboratory reference strain PAO1. This approach was used in preference to engineering of sequence variations into the chromosome because it is faster and less complex, allowing high throughput screening of variations of interest.

MIC testing was carried out on bacteria expressing the PBP3 variants for representative carbapenem, cephalosporin and monobactam antibiotics (Table S4). None of the PBP3 variations increased imipenem or aztreonam MIC and nine of the variations did not alter the MIC of any of the tested antibiotics. The other 10 variations increased the MIC of meropenem and/or ceftazidime. The amino acid differences that increased the MICs of β-lactams were all located in the catalytic domain of PBP3.

The effects of chromosomally encoded PBP3 variants on β-lactam susceptibilityThe approach described previously provides a good initial screen for the effects of PBP3 variations. However, expression from the plasmid-encoded gene was likely to be higher than that of chromosomally expressed PBP3. In addition, the presence of the chromosomal wild-type allele as well as the plasmid may dampen the effects of plasmid-encoded PBP3 variants. To better represent the normal situation, five ftsI alleles encoding PBP3 variants that reduced susceptibility to at least one β-lactam when expressed from pJN105 were engineered into the chromosome of P. aeruginosa PAO1, replacing the wild-type gene. MIC testing was performed on the resulting mutants with a range of antibiotics covering all classes of clinically used β-lactams. The introduced variations increased the MIC of between three and six of the tested antibiotics (Table 1). However, none of the tested variations increased the MIC of imipenem or piperacillin and indeed, three variations slightly reduced the piperacillin MIC.Table 1.MIC values for P. aeruginosa with genome-encoded PBP3 variantsAntibiotic MIC (mg/L)aPBP3 variationMerImiCeftAztCarbPipTicCefpWild-type0.540.5232280.5A244T140.254321162V471G240.521281320.5R504C84182562642F533L840.52321161V537L440.581282322Values are medians of three biological replicates.Mer, Meropenem; Imi, Imipenem; Ceft, Ceftazidime; Azt, Aztreonam; Carb, carbenicillin; Pip, piperacillin; Tic, Ticaracillin; Cefp, Cefipeme.

The effects of chromosomally encoded PBP3 variants on β-lactam susceptibility

The approach described previously provides a good initial screen for the effects of PBP3 variations. However, expression from the plasmid-encoded gene was likely to be higher than that of chromosomally expressed PBP3. In addition, the presence of the chromosomal wild-type allele as well as the plasmid may dampen the effects of plasmid-encoded PBP3 variants. To better represent the normal situation, five ftsI alleles encoding PBP3 variants that reduced susceptibility to at least one β-lactam when expressed from pJN105 were engineered into the chromosome of P. aeruginosa PAO1, replacing the wild-type gene. MIC testing was performed on the resulting mutants with a range of antibiotics covering all classes of clinically used β-lactams. The introduced variations increased the MIC of between three and six of the tested antibiotics (Table 1). However, none of the tested variations increased the MIC of imipenem or piperacillin and indeed, three variations slightly reduced the piperacillin MIC.

MIC values for P. aeruginosa with genome-encoded PBP3 variants

Values are medians of three biological replicates.

Mer, Meropenem; Imi, Imipenem; Ceft, Ceftazidime; Azt, Aztreonam; Carb, carbenicillin; Pip, piperacillin; Tic, Ticaracillin; Cefp, Cefipeme.

Effects of PBP3 variations in conjunction with increased effluxMutations altering the MexR repressor protein increase expression of the MexAB-OprM efflux pump and are associated with reduced susceptibility to β-lactam antibiotics.20,56 To investigate the effects of PBP3 variations in combination with increased mexAB-oprM expression, pJN105 expressing PBP3 variants with changes in the catalytic domain were transformed into a PAO1 mexR mutant strain and MICs were determined (Table 2). The PBP3 variations were chosen because they increased β-lactam MIC in wild-type strain PAO1 (Table S4) or were present in multiple clinical isolates (Figure S1).Table 2.MIC values for a P. aeruginosa mexR mutant expressing PBP3 variants from an expression vectorAntibiotic MIC (mg/L) aPBP3 variationMeropenemImipenemCeftazidimeAztreonamWild-type84216L240V84416A244T164232Y367C84216V465G84216V471G164216R504C324432F507L84216I524T164232P527S164232F533L324216V537L164232Values are medians of three biological replicates. P. aeruginosa are classified as being resistant if the MIC is ≥8 mg/L (meropenem and imipenem) or ≥32 mg/L (ceftazidime and aztreonam).50In bacteria expressing only the chromosomal (wild-type) PBP3, the mexR mutation increased ß-lactam MICs of meropenem by 16-fold, of ceftazidime by 4-fold, and of aztreonam by 8-fold but did not increase the MIC of imipenem (Table 1, Table S4) consistent with previous findings that MexAB-OprM does not export imipenem.10,56,57 Eight of the 11 PBP3 variations tested further increased the MIC of at least one antibiotic. Notably, the V537L, A244T, I524T and P527S PBP3 variations all increased aztreonam MIC in the mexR mutant but not when expressed in wild-type PAO1. None of the tested PBP3 variations increased the MIC of imipenem in the mexR mutant.

Effects of PBP3 variations in conjunction with increased efflux

Mutations altering the MexR repressor protein increase expression of the MexAB-OprM efflux pump and are associated with reduced susceptibility to β-lactam antibiotics.20,56 To investigate the effects of PBP3 variations in combination with increased mexAB-oprM expression, pJN105 expressing PBP3 variants with changes in the catalytic domain were transformed into a PAO1 mexR mutant strain and MICs were determined (Table 2). The PBP3 variations were chosen because they increased β-lactam MIC in wild-type strain PAO1 (Table S4) or were present in multiple clinical isolates (Figure S1).

MIC values for a P. aeruginosa mexR mutant expressing PBP3 variants from an expression vector

Values are medians of three biological replicates. P. aeruginosa are classified as being resistant if the MIC is ≥8 mg/L (meropenem and imipenem) or ≥32 mg/L (ceftazidime and aztreonam).50

In bacteria expressing only the chromosomal (wild-type) PBP3, the mexR mutation increased ß-lactam MICs of meropenem by 16-fold, of ceftazidime by 4-fold, and of aztreonam by 8-fold but did not increase the MIC of imipenem (Table 1, Table S4) consistent with previous findings that MexAB-OprM does not export imipenem.10,56,57 Eight of the 11 PBP3 variations tested further increased the MIC of at least one antibiotic. Notably, the V537L, A244T, I524T and P527S PBP3 variations all increased aztreonam MIC in the mexR mutant but not when expressed in wild-type PAO1. None of the tested PBP3 variations increased the MIC of imipenem in the mexR mutant.

Effects of PBP3 variations on growth and morphologyAs PBP3 is an essential protein required for cell division, we tested the hypothesis that mutations altering PBP3 affect bacterial growth. Three of the five mutants expressing a PBP3 variant had a significant reduction in growth, as measured by absorbance, relative to wild-type (Figure 1). The PBP3 variation A244T caused the greatest reduction in rate of change of absorbance. A reduced rate of absorbance increase indicates that cells were generated at a lower rate. A reduced rate of cell formation may be due to a reduced rate of incorporation of nutrients into cells (i.e. slower formation of biomass and generation of normal sized cells). Alternatively, it could occur due to reduced rates of cell division and consequent increases in cell size with no change in the rate of increase of biomass.58 The biomass of the bacteria in liquid culture was measured to distinguish between these possibilities.Figure 1.Growth of P. aeruginosa PAO1 with chromosomally expressed PBP3 variants. Bacteria were grown and growth monitored for 24 h. Values are AUC for three biological replicates (each the mean of three technical replicates). *P < 0.05; **P < 0.01; ns, not significant (unpaired t-tests).The biomass of wild-type bacteria from 100 mL of culture, at OD600 = 1.25, was 0.2328 ± 0.0144 g and that of bacteria with the PBP3 variation A244T was 0.2689 ± 0.0246, which was significantly higher (P = 0.0129) showing that the variation did not result in reduced biomass.The lengths of cells were measured to determine the effects of PBP3 variations on morphology. All tested variations caused cells to be significantly longer than those of wild-type bacteria (Figure 2, Table S5, Figure S3). The mutant expressing the PBP3 variation A244T had the most elongated cells.Figure 2.Cell length of P. aeruginosa with chromosomally expressed PBP3 variants in exponential phase. (a) Representative images of P. aeruginosa with wild-type PBP3 (top) or PBP3 variation A244T (bottom). (b) Lengths of P. aeruginosa cells in exponential growth phase. All cells from 15 images were measured (between 717 and 1913 cells for each strain). The red line is the median length of wild-type cells. All of the PBP3 variations caused average cell length to be significantly (P < 0.001) longer than wild-type (Mann–Whitney U-test). Median and interquartile values are shown in Table S5.

Effects of PBP3 variations on growth and morphology

As PBP3 is an essential protein required for cell division, we tested the hypothesis that mutations altering PBP3 affect bacterial growth. Three of the five mutants expressing a PBP3 variant had a significant reduction in growth, as measured by absorbance, relative to wild-type (Figure 1). The PBP3 variation A244T caused the greatest reduction in rate of change of absorbance. A reduced rate of absorbance increase indicates that cells were generated at a lower rate. A reduced rate of cell formation may be due to a reduced rate of incorporation of nutrients into cells (i.e. slower formation of biomass and generation of normal sized cells). Alternatively, it could occur due to reduced rates of cell division and consequent increases in cell size with no change in the rate of increase of biomass.58 The biomass of the bacteria in liquid culture was measured to distinguish between these possibilities.

Growth of P. aeruginosa PAO1 with chromosomally expressed PBP3 variants. Bacteria were grown and growth monitored for 24 h. Values are AUC for three biological replicates (each the mean of three technical replicates). *P < 0.05; **P < 0.01; ns, not significant (unpaired t-tests).

The biomass of wild-type bacteria from 100 mL of culture, at OD600 = 1.25, was 0.2328 ± 0.0144 g and that of bacteria with the PBP3 variation A244T was 0.2689 ± 0.0246, which was significantly higher (P = 0.0129) showing that the variation did not result in reduced biomass.

The lengths of cells were measured to determine the effects of PBP3 variations on morphology. All tested variations caused cells to be significantly longer than those of wild-type bacteria (Figure 2, Table S5, Figure S3). The mutant expressing the PBP3 variation A244T had the most elongated cells.

Cell length of P. aeruginosa with chromosomally expressed PBP3 variants in exponential phase. (a) Representative images of P. aeruginosa with wild-type PBP3 (top) or PBP3 variation A244T (bottom). (b) Lengths of P. aeruginosa cells in exponential growth phase. All cells from 15 images were measured (between 717 and 1913 cells for each strain). The red line is the median length of wild-type cells. All of the PBP3 variations caused average cell length to be significantly (P < 0.001) longer than wild-type (Mann–Whitney U-test). Median and interquartile values are shown in Table S5.

Effects of sub-inhibitory concentrations of antibiotics on cell morphologyβ-lactams inhibit PBP3 and this can inhibit cell division and result in filamentation in E. coli.59 As changes to PBP3 result in filamentation of P. aeruginosa, we tested the hypothesis that β-lactam antibiotics would also increase the filamentation of bacteria. P. aeruginosa PAO1, as well as strains expressing the PBP3 variants R504C and F533L that caused the greatest increase in meropenem MIC, were grown with sub-inhibitory concentrations of meropenem and cell length measured.Sub-MIC levels of meropenem caused significant elongation of strain PAO1 even at the lowest concentration tested (1/16 of the MIC) (Figure 3a, Figure S4, Table S6). The average cell length in the presence of 0.5 mg/L meropenem, the maximum concentration that permitted growth, was 20 times longer than that of cells grown in the absence of antibiotics. The two isolates with PBP3 variations that were tested also exhibited cell elongation in the presence of meropenem. However, the effect of the antibiotic on cell length was much less than for strain PAO1.Figure 3.Effects of meropenem and piperacillin on cell length of P. aeruginosa with chromosomally expressed PBP3 variants. (a) Cell length in the presence of varying concentrations of meropenem. (b) Cell length in the presence of varying concentrations of piperacillin: 101 cells were measured under each growth condition. Asterisks indicate significant differences in pairwise comparisons (Mann–Whitney U-test). *P < 0.5; ****P < 0.0001. Median and interquartile values are shown in Table S6.The effects of sub-MIC levels of piperacillin were also tested on strain PAO1 and on the R504C mutant, which has the same MIC, and the V471G mutant, which has a lower MIC than wild-type. The average length of PAO1 cells grown in the presence of 1 mg/L piperacillin increased 30-fold compared to cells grown in the absence of antibiotic, and even the lowest level of piperacillin tested (32 times less than MIC) caused an increase of 1.7 times in average cell length (Figure 3b, Table S6). Piperacillin also increased the average lengths of cells of the mutant strains, by 8.8-fold for the PBP3 variation V471G in the presence of 0.25 mg/L piperacillin and by 15.5-fold for the R504C PBP3 variation in the presence of 0.5 mg/L piperacillin.

Effects of sub-inhibitory concentrations of antibiotics on cell morphology

β-lactams inhibit PBP3 and this can inhibit cell division and result in filamentation in E. coli.59 As changes to PBP3 result in filamentation of P. aeruginosa, we tested the hypothesis that β-lactam antibiotics would also increase the filamentation of bacteria. P. aeruginosa PAO1, as well as strains expressing the PBP3 variants R504C and F533L that caused the greatest increase in meropenem MIC, were grown with sub-inhibitory concentrations of meropenem and cell length measured.

Sub-MIC levels of meropenem caused significant elongation of strain PAO1 even at the lowest concentration tested (1/16 of the MIC) (Figure 3a, Figure S4, Table S6). The average cell length in the presence of 0.5 mg/L meropenem, the maximum concentration that permitted growth, was 20 times longer than that of cells grown in the absence of antibiotics. The two isolates with PBP3 variations that were tested also exhibited cell elongation in the presence of meropenem. However, the effect of the antibiotic on cell length was much less than for strain PAO1.

Effects of meropenem and piperacillin on cell length of P. aeruginosa with chromosomally expressed PBP3 variants. (a) Cell length in the presence of varying concentrations of meropenem. (b) Cell length in the presence of varying concentrations of piperacillin: 101 cells were measured under each growth condition. Asterisks indicate significant differences in pairwise comparisons (Mann–Whitney U-test). *P < 0.5; ****P < 0.0001. Median and interquartile values are shown in Table S6.

The effects of sub-MIC levels of piperacillin were also tested on strain PAO1 and on the R504C mutant, which has the same MIC, and the V471G mutant, which has a lower MIC than wild-type. The average length of PAO1 cells grown in the presence of 1 mg/L piperacillin increased 30-fold compared to cells grown in the absence of antibiotic, and even the lowest level of piperacillin tested (32 times less than MIC) caused an increase of 1.7 times in average cell length (Figure 3b, Table S6). Piperacillin also increased the average lengths of cells of the mutant strains, by 8.8-fold for the PBP3 variation V471G in the presence of 0.25 mg/L piperacillin and by 15.5-fold for the R504C PBP3 variation in the presence of 0.5 mg/L piperacillin.

DiscussionThe aim of this research was to quantify the effects of PBP3 variations on β-lactam susceptibility and cell morphology of P. aeruginosa. The results show, to the best of our knowledge for the first time, that some PBP3 variations do indeed reduce susceptibility of P. aeruginosa to a range of β-lactams. However, none of the tested variations reduced susceptibility to imipenem or piperacillin.
P. aeruginosa isolates with PBP3 variations were distributed throughout the phylogenetic tree (Figure S2) suggesting that PBP3 variations have arisen independently on multiple occasions. The most common PBP3 variation in clinical isolates, R504C, was sufficient to increase the MIC of six out of eight β-lactams tested (Table 1). The R504C and F533L variations that were sufficient to make P. aeruginosa PAO1 resistant to meropenem both reduce the affinity of PBP3 for β-lactams,39,60 providing an explanation for the effects of these variations on meropenem MIC. All the PBP3 variants expressed from the chromosome increased meropenem and ticarcillin MICs showing that the effectiveness of these antibiotics can be reduced by the presence of PBP3 variations in clinical isolates. Conversely, none of the tested variations increased the MIC of piperacillin or imipenem (Tables 1 and 2, Table S4) and indeed some variations made the bacteria more susceptible to piperacillin (Table 1) indicating that these antibiotics are good options when treating infections of P. aeruginosa with PBP3 variations. The mechanistic basis for increased piperacillin susceptibility due to PBP3 variations is not clear.β-lactam resistance in P. aeruginosa is multifactorial and combinations of different resistance mechanisms can increase the MIC more than one alone. Several combinations of PBP3 variations with a mexR mutation increased the MIC to the point that the bacteria were classified as being resistant to at least one of the antibiotics tested, whereas neither the mexR mutation or the PBP3 variation alone was sufficient to confer resistance. Other differences in the genetic backgrounds of different isolates, such as the presence of variations in AmpC or OprD,10,21 can also be expected to affect the contribution of PBP3 variations to resistance. Nonetheless, the presence of PBP3 variations could serve as a marker to identify β-lactams that will be less effective in treating infections, contributing to ongoing efforts to predict antibiotic susceptibility of infecting bacteria from their genome sequences.61,62Sequence changes that contribute to antibiotic resistance have the potential to affect bacterial fitness63 and the PBP3 variations provide a very clear example of this. Variations that increased the MIC of β-lactams caused a reduction in growth and an increase in filamentation (Figures 1 and 2). This indicates that sequence variation can reduce the activity of PBP3, leading to a reduction in cell division rate, consistent with the known role of PBP3 in cell division.10,12,64 It seems likely that the occurrence of PBP3 variants in clinical isolates of P. aeruginosa represents a trade-off between reduced enzyme activity and cell division, and the need to reduce the affinity of PBP3 for β-lactam antibiotics. The extent to which the presence of PBP3 variations reduces bacterial fitness during infection in the absence of β-lactam treatment remains to be determined. Sub-MIC levels of meropenem or piperacillin also caused increases in cell length, even at 1/16 of the MIC, suggesting that these antibiotics impact cell division at levels well below the MIC. These antibiotics also increased filamentation of bacteria with PBP3 variations, although the effect was less pronounced. These findings suggest that β-lactams retain some activity even against bacteria that are classified as resistant. Curiously, low concentrations of meropenem caused a reduction in cell length of P. aeruginosa expressing the PBP3 variants R504C or F533L. The reduction in length may be due to inhibition of PBP1a, PBP1b or PBP2 as inhibition of these non-essential PBPs causes altered cell morphology.12PBP3 variations that resulted in increased β-lactam MIC are all located near the catalytic site in close topological proximity to each other (Figure 4). Differences outside this location did not increase the MIC towards the tested β-lactams. Differences in the catalytic domain could potentially interfere with access of β-lactams to the active site or covalent attachment to the PBP. Imipenem is smaller than many β-lactams65,66 and the PBP3 variations probably do not affect its access to the active site, explaining why these variations did not alter the MIC of imipenem.Figure 4.Locations of PBP3 variations that reduce susceptibility to β-lactams. The structures of PBP3 with antibiotics bound at the active site are shown in blue for the protein–protein interactions domain and in red for the catalytic domain. Amino acid residues identified here as reducing antibiotic susceptibility are shown in magenta. (a) meropenem (cyan) (PDB ID 3PBR); (b) aztreonam (yellow) (PDB ID 3PBS) and (c) ceftazidime (green) (PDB ID 3PBO).In conclusion, our findings show that different variations in PBP3 affect susceptibility to different β-lactams. Furthermore, reducing the activity of PBP3 through a sequence variation or by the presence of sub-MIC levels of antibiotic leads to filamentation that is likely to affect bacterial fitness. None of the PBP3 variations increased the MIC of imipenem or piperacillin indicating that the presence of PBP3 variations in infecting bacteria will not reduce the effectiveness of these antibiotics.

The aim of this research was to quantify the effects of PBP3 variations on β-lactam susceptibility and cell morphology of P. aeruginosa. The results show, to the best of our knowledge for the first time, that some PBP3 variations do indeed reduce susceptibility of P. aeruginosa to a range of β-lactams. However, none of the tested variations reduced susceptibility to imipenem or piperacillin.

P. aeruginosa isolates with PBP3 variations were distributed throughout the phylogenetic tree (Figure S2) suggesting that PBP3 variations have arisen independently on multiple occasions. The most common PBP3 variation in clinical isolates, R504C, was sufficient to increase the MIC of six out of eight β-lactams tested (Table 1). The R504C and F533L variations that were sufficient to make P. aeruginosa PAO1 resistant to meropenem both reduce the affinity of PBP3 for β-lactams,39,60 providing an explanation for the effects of these variations on meropenem MIC. All the PBP3 variants expressed from the chromosome increased meropenem and ticarcillin MICs showing that the effectiveness of these antibiotics can be reduced by the presence of PBP3 variations in clinical isolates. Conversely, none of the tested variations increased the MIC of piperacillin or imipenem (Tables 1 and 2, Table S4) and indeed some variations made the bacteria more susceptible to piperacillin (Table 1) indicating that these antibiotics are good options when treating infections of P. aeruginosa with PBP3 variations. The mechanistic basis for increased piperacillin susceptibility due to PBP3 variations is not clear.

β-lactam resistance in P. aeruginosa is multifactorial and combinations of different resistance mechanisms can increase the MIC more than one alone. Several combinations of PBP3 variations with a mexR mutation increased the MIC to the point that the bacteria were classified as being resistant to at least one of the antibiotics tested, whereas neither the mexR mutation or the PBP3 variation alone was sufficient to confer resistance. Other differences in the genetic backgrounds of different isolates, such as the presence of variations in AmpC or OprD,10,21 can also be expected to affect the contribution of PBP3 variations to resistance. Nonetheless, the presence of PBP3 variations could serve as a marker to identify β-lactams that will be less effective in treating infections, contributing to ongoing efforts to predict antibiotic susceptibility of infecting bacteria from their genome sequences.61,62

Sequence changes that contribute to antibiotic resistance have the potential to affect bacterial fitness63 and the PBP3 variations provide a very clear example of this. Variations that increased the MIC of β-lactams caused a reduction in growth and an increase in filamentation (Figures 1 and 2). This indicates that sequence variation can reduce the activity of PBP3, leading to a reduction in cell division rate, consistent with the known role of PBP3 in cell division.10,12,64 It seems likely that the occurrence of PBP3 variants in clinical isolates of P. aeruginosa represents a trade-off between reduced enzyme activity and cell division, and the need to reduce the affinity of PBP3 for β-lactam antibiotics. The extent to which the presence of PBP3 variations reduces bacterial fitness during infection in the absence of β-lactam treatment remains to be determined. Sub-MIC levels of meropenem or piperacillin also caused increases in cell length, even at 1/16 of the MIC, suggesting that these antibiotics impact cell division at levels well below the MIC. These antibiotics also increased filamentation of bacteria with PBP3 variations, although the effect was less pronounced. These findings suggest that β-lactams retain some activity even against bacteria that are classified as resistant. Curiously, low concentrations of meropenem caused a reduction in cell length of P. aeruginosa expressing the PBP3 variants R504C or F533L. The reduction in length may be due to inhibition of PBP1a, PBP1b or PBP2 as inhibition of these non-essential PBPs causes altered cell morphology.12

PBP3 variations that resulted in increased β-lactam MIC are all located near the catalytic site in close topological proximity to each other (Figure 4). Differences outside this location did not increase the MIC towards the tested β-lactams. Differences in the catalytic domain could potentially interfere with access of β-lactams to the active site or covalent attachment to the PBP. Imipenem is smaller than many β-lactams65,66 and the PBP3 variations probably do not affect its access to the active site, explaining why these variations did not alter the MIC of imipenem.

Locations of PBP3 variations that reduce susceptibility to β-lactams. The structures of PBP3 with antibiotics bound at the active site are shown in blue for the protein–protein interactions domain and in red for the catalytic domain. Amino acid residues identified here as reducing antibiotic susceptibility are shown in magenta. (a) meropenem (cyan) (PDB ID 3PBR); (b) aztreonam (yellow) (PDB ID 3PBS) and (c) ceftazidime (green) (PDB ID 3PBO).

In conclusion, our findings show that different variations in PBP3 affect susceptibility to different β-lactams. Furthermore, reducing the activity of PBP3 through a sequence variation or by the presence of sub-MIC levels of antibiotic leads to filamentation that is likely to affect bacterial fitness. None of the PBP3 variations increased the MIC of imipenem or piperacillin indicating that the presence of PBP3 variations in infecting bacteria will not reduce the effectiveness of these antibiotics.

Supplementary Materialdkae203_Supplementary_Data

Supplementary Material
