---
pmid: '31327635'
title: Unique Structural Features of the Mitochondrial AAA+ Protease AFG3L2 Reveal
  the Molecular Basis for Activity in Health and Disease.
authors:
- Puchades C
- Ding B
- Song A
- Wiseman RL
- Lander GC
- Glynn SE
journal: Mol Cell
year: '2019'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6731152
doi: 10.1016/j.molcel.2019.06.016
---

# Unique Structural Features of the Mitochondrial AAA+ Protease AFG3L2 Reveal the Molecular Basis for Activity in Health and Disease.
**Authors:** Puchades C, Ding B, Song A, Wiseman RL, Lander GC, Glynn SE
**Journal:** Mol Cell (2019)
**DOI:** [10.1016/j.molcel.2019.06.016](https://doi.org/10.1016/j.molcel.2019.06.016)
**PMC:** [PMC6731152](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6731152/)

## Abstract

1. Mol Cell. 2019 Sep 5;75(5):1073-1085.e6. doi: 10.1016/j.molcel.2019.06.016.
Epub  2019 Jul 18.

Unique Structural Features of the Mitochondrial AAA+ Protease AFG3L2 Reveal the 
Molecular Basis for Activity in Health and Disease.

Puchades C(1), Ding B(2), Song A(1), Wiseman RL(3), Lander GC(4), Glynn SE(5).

Author information:
(1)Department of Integrative Structural and Computational Biology, The Scripps 
Research Institute, La Jolla, CA 92037, USA; Department of Molecular Medicine, 
The Scripps Research Institute, La Jolla, CA 92037, USA; Skaggs Graduate School 
of Chemical and Biological Sciences, The Scripps Research Institute, La Jolla, 
CA 92037, USA.
(2)Department of Biochemistry and Cell Biology, Stony Brook University, Stony 
Brook, NY 11794, USA.
(3)Department of Molecular Medicine, The Scripps Research Institute, La Jolla, 
CA 92037, USA.
(4)Department of Integrative Structural and Computational Biology, The Scripps 
Research Institute, La Jolla, CA 92037, USA. Electronic address: 
glander@scripps.edu.
(5)Department of Biochemistry and Cell Biology, Stony Brook University, Stony 
Brook, NY 11794, USA. Electronic address: steven.glynn@stonybrook.edu.

Mitochondrial AAA+ quality-control proteases regulate diverse aspects of 
mitochondrial biology through specialized protein degradation, but the 
underlying mechanisms of these enzymes remain poorly defined. The mitochondrial 
AAA+ protease AFG3L2 is of particular interest, as genetic mutations localized 
throughout AFG3L2 are linked to diverse neurodegenerative disorders. However, a 
lack of structural data has limited our understanding of how mutations impact 
enzymatic function. Here, we used cryoelectron microscopy (cryo-EM) to determine 
a substrate-bound structure of the catalytic core of human AFG3L2. This 
structure identifies multiple specialized structural features that integrate 
with conserved motifs required for ATP-dependent translocation to unfold and 
degrade targeted proteins. Many disease-relevant mutations localize to these 
unique structural features of AFG3L2 and distinctly influence its activity and 
stability. Our results provide a molecular basis for neurological phenotypes 
associated with different AFG3L2 mutations and establish a structural framework 
to understand how different members of the AAA+ superfamily achieve specialized 
biological functions.

Copyright © 2019 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.molcel.2019.06.016
PMCID: PMC6731152
PMID: 31327635 [Indexed for MEDLINE]

Conflict of interest statement: Declaration of interests The authors declare no 
competing interests.

## Full Text

INTRODUCTIONProtein quality control is central to cellular function and survival in all living organisms (Chen et al., 2011). Maintaining protein quality control relies on a complex network of ATP-dependent proteases that regulate protein homeostasis (Bohovych et al., 2015). The membrane-anchored AAA+ protease FtsH is essential for growth in E. coli, and is conserved across all kingdoms of life (Langklotz et al., 2012). These FtsH-related proteins specialize in protein quality control in membranous environments and are found in eukaryotic mitochondrial and chloroplastic membranes, functioning as hexameric, ATP-driven protein quality control systems that process membrane-embedded and associated protein substrates (Leonhard et al., 1996; Wagner et al., 2012). All FtsH-related AAA+ proteases are characterized by a shared topology comprising an N-terminal domain, a transmembrane region, an AAA+ ATPase domain, and a zinc-metalloprotease domain (Janska et al., 2013; Sauer and Baker, 2011). Notably, mitochondria contain two types of FtsH-related AAA+ proteases, referred to as m- and i-AAA proteases, which are tethered to the mitochondrial inner membrane (IM), but expose their enzymatic domains to the matrix and intermembrane spaces (IMS), respectively (Figure 1A) (Leonhard et al., 1996).We previously solved a cryo-electron microscopy (cryo-EM) structure of the yeast i-AAA YME1, which revealed how a sequential ATP hydrolysis cycle drives hand-over-hand translocation of substrate through the central pore of the hexamer (Puchades et al., 2017). Given the strong structural commonalities shared by all FtsH-related AAA+ proteases, the basic translocation mechanism we described in YME1 is likely to be conserved from bacteria to humans. However, despite this conserved mechanism, FtsH-related proteins have each evolved the ability to recognize and degrade distinct substrates within unique cellular environments, giving rise to this family’s modulation of diverse biological pathways (Bittner et al., 2017; Glynn, 2017; Nishimura et al., 2016). While their individual substrate specificities likely involve the integration of specialized mechanisms to recognize, recruit, engage, and degrade protein targets, the molecular basis for this specialization remains poorly understood.The human m-AAA protease assembles as homohexamers of AFG3L2 subunits or heterohexamers comprising AFG3L2 subunits and subunits of the closely related homolog paraplegin (SPG7) (Banfi et al., 1999; Koppen et al., 2007). AFG3L2-containing complexes regulate important biological functions, including mitochondrial ribosome assembly, the expression, maturation, and degradation of electron transport chain complexes, supervision of mitochondrial dynamics, and the regulation of calcium homeostasis (Arlt et al., 1998; Consolato et al., 2018; Konig et al., 2016; Nolden et al., 2005). Furthermore, m-AAA proteases are essential for axonal development in mammals, and loss or reduction of wild-type AFG3L2 lead to pleiotropic phenotypes, such as mitochondrial transport defects, mitochondrial fragmentation, and reductions in both mitochondrial membrane potential and respiration (Almajan et al., 2012; Kondadi et al., 2014; Levytskyy et al., 2016; Maltecca et al., 2008; Maltecca et al., 2009; Mancini et al., 2018). In fact, point mutations localized throughout AFG3L2 are linked to multiple neurodegenerative disorders in humans that present with diverse pathologies and severity. The majority of identified AFG3L2 mutations are implicated in the autosomal dominant disease spinocerebellar ataxia type 28 (SCA28) (Cagnoli et al., 2006; Di Bella et al., 2010; Edener et al., 2010; Lobbe et al., 2014; Mariotti et al., 2008; Svenstrup et al., 2017; Szpisjak et al., 2017; Zuhlke et al., 2015). An alternative heterozygous AFG3L2 point mutation has been causatively linked to dominant optic atrophy (DOA), whereas homozygous individuals carrying rare mutations in AFG3L2 present neurodegenerative phenotypes distinct from either DOA or SCA28 (Table S1) (Charif et al., 2015; Colavito et al., 2017; Eskandrani et al., 2017; Pierson et al., 2011). The genetic relationship between diverse AFG3L2 mutations and distinct disease severity and neurodegenerative pathologies indicates that these mutations differentially impact m-AAA protease activity. Despite this relationship, a lack of structural information has limited our understanding of the molecular mechanisms linking specific mutations to altered enzymatic activity and ultimately pathology.Here, we present an atomic model of the human AFG3L2 homohexameric m-AAA protease trapped in the act of processing substrate, revealing the network of molecular interactions responsible for substrate engagement, unfolding, transfer across the proteolytic chamber, and proteolysis. While this structure confirms that the fundamental mechanism relating ATP hydrolysis to substrate translocation is conserved between i-AAA and m-AAA, this study also reveals how AFG3L2 has evolved unique structural features for processing substrates located within or in proximity to the matrix-facing surface of the IM. Further, our structure defines how disease-relevant mutations implicated in AFG3L2-associated neurodegenerative disorders distinctly impact the mechanism of this AAA+ protease. Thus, our structure-function studies not only provide a much-needed structural framework to understand how different AAA+ proteases build upon a common mechanistic core to process specific substrates, but also reveal a molecular basis to define the distinct neuropathologies associated with different AFG3L2 mutations.

INTRODUCTION

Protein quality control is central to cellular function and survival in all living organisms (Chen et al., 2011). Maintaining protein quality control relies on a complex network of ATP-dependent proteases that regulate protein homeostasis (Bohovych et al., 2015). The membrane-anchored AAA+ protease FtsH is essential for growth in E. coli, and is conserved across all kingdoms of life (Langklotz et al., 2012). These FtsH-related proteins specialize in protein quality control in membranous environments and are found in eukaryotic mitochondrial and chloroplastic membranes, functioning as hexameric, ATP-driven protein quality control systems that process membrane-embedded and associated protein substrates (Leonhard et al., 1996; Wagner et al., 2012). All FtsH-related AAA+ proteases are characterized by a shared topology comprising an N-terminal domain, a transmembrane region, an AAA+ ATPase domain, and a zinc-metalloprotease domain (Janska et al., 2013; Sauer and Baker, 2011). Notably, mitochondria contain two types of FtsH-related AAA+ proteases, referred to as m- and i-AAA proteases, which are tethered to the mitochondrial inner membrane (IM), but expose their enzymatic domains to the matrix and intermembrane spaces (IMS), respectively (Figure 1A) (Leonhard et al., 1996).

We previously solved a cryo-electron microscopy (cryo-EM) structure of the yeast i-AAA YME1, which revealed how a sequential ATP hydrolysis cycle drives hand-over-hand translocation of substrate through the central pore of the hexamer (Puchades et al., 2017). Given the strong structural commonalities shared by all FtsH-related AAA+ proteases, the basic translocation mechanism we described in YME1 is likely to be conserved from bacteria to humans. However, despite this conserved mechanism, FtsH-related proteins have each evolved the ability to recognize and degrade distinct substrates within unique cellular environments, giving rise to this family’s modulation of diverse biological pathways (Bittner et al., 2017; Glynn, 2017; Nishimura et al., 2016). While their individual substrate specificities likely involve the integration of specialized mechanisms to recognize, recruit, engage, and degrade protein targets, the molecular basis for this specialization remains poorly understood.

The human m-AAA protease assembles as homohexamers of AFG3L2 subunits or heterohexamers comprising AFG3L2 subunits and subunits of the closely related homolog paraplegin (SPG7) (Banfi et al., 1999; Koppen et al., 2007). AFG3L2-containing complexes regulate important biological functions, including mitochondrial ribosome assembly, the expression, maturation, and degradation of electron transport chain complexes, supervision of mitochondrial dynamics, and the regulation of calcium homeostasis (Arlt et al., 1998; Consolato et al., 2018; Konig et al., 2016; Nolden et al., 2005). Furthermore, m-AAA proteases are essential for axonal development in mammals, and loss or reduction of wild-type AFG3L2 lead to pleiotropic phenotypes, such as mitochondrial transport defects, mitochondrial fragmentation, and reductions in both mitochondrial membrane potential and respiration (Almajan et al., 2012; Kondadi et al., 2014; Levytskyy et al., 2016; Maltecca et al., 2008; Maltecca et al., 2009; Mancini et al., 2018). In fact, point mutations localized throughout AFG3L2 are linked to multiple neurodegenerative disorders in humans that present with diverse pathologies and severity. The majority of identified AFG3L2 mutations are implicated in the autosomal dominant disease spinocerebellar ataxia type 28 (SCA28) (Cagnoli et al., 2006; Di Bella et al., 2010; Edener et al., 2010; Lobbe et al., 2014; Mariotti et al., 2008; Svenstrup et al., 2017; Szpisjak et al., 2017; Zuhlke et al., 2015). An alternative heterozygous AFG3L2 point mutation has been causatively linked to dominant optic atrophy (DOA), whereas homozygous individuals carrying rare mutations in AFG3L2 present neurodegenerative phenotypes distinct from either DOA or SCA28 (Table S1) (Charif et al., 2015; Colavito et al., 2017; Eskandrani et al., 2017; Pierson et al., 2011). The genetic relationship between diverse AFG3L2 mutations and distinct disease severity and neurodegenerative pathologies indicates that these mutations differentially impact m-AAA protease activity. Despite this relationship, a lack of structural information has limited our understanding of the molecular mechanisms linking specific mutations to altered enzymatic activity and ultimately pathology.

Here, we present an atomic model of the human AFG3L2 homohexameric m-AAA protease trapped in the act of processing substrate, revealing the network of molecular interactions responsible for substrate engagement, unfolding, transfer across the proteolytic chamber, and proteolysis. While this structure confirms that the fundamental mechanism relating ATP hydrolysis to substrate translocation is conserved between i-AAA and m-AAA, this study also reveals how AFG3L2 has evolved unique structural features for processing substrates located within or in proximity to the matrix-facing surface of the IM. Further, our structure defines how disease-relevant mutations implicated in AFG3L2-associated neurodegenerative disorders distinctly impact the mechanism of this AAA+ protease. Thus, our structure-function studies not only provide a much-needed structural framework to understand how different AAA+ proteases build upon a common mechanistic core to process specific substrates, but also reveal a molecular basis to define the distinct neuropathologies associated with different AFG3L2 mutations.

RESULTSStructure of the human substrate-bound AFG3L2 catalytic coreWe solved a ~3.1 Å resolution cryo-EM structure of a truncated construct of AFG3L2 comprising the ATPase and protease domains (residues 272–797; coreAFG3L2) (Figures 1 and S1). The construct was stabilized for structural analysis by incorporating ATPase-inactivating Walker B (WB, E408Q) and protease-inactivating (PI, E575Q) mutations, and vitrified in the presence of saturating concentrations of the non-hydrolyzable ATP analog AMP-PNP. The resulting reconstruction was of sufficient quality to build an atomic model of the six protease domains and four of the six ATPase domains (Figures 1 and S1). The atomic models of the large and small ATPase subdomains of one of the well-resolved subunits were independently rigid-body fit into the low resolution cryo-EM density of the remaining two ATPase domains to generate a backbone trace in these regions, completing an atomic model of the entire homohexameric AFG3L2 complex (Figures 1, S2, and Table S2).Our reconstruction shows that AFG3L2 assembles into two stacked rings with a planar C6 symmetric protease base topped by an asymmetric ATPase spiral (Figures 1D, 1E, and 1F) – a quaternary structure similar to that observed for substrate-bound YME1 (Figure S2A) (Puchades et al., 2017). The three upper subunits of the AFG3L2 staircase are bound to AMP-PNP (termed ATP2-ATP4 for simplicity), while the lowermost subunit is bound to ADP (Figure S2). The limited resolution of the remaining two subunits did not allow for unambiguous identification of nucleotide state, although the positioning of these subunits within the staircase correspond to the nucleotide-free (apo) ‘step’-subunit and the uppermost ATP-bound subunit (ATP1) within YME1 (Figure S2A). While we could not experimentally verify the identity of the nucleotide in these subunits, we previously showed that nucleotide state allosterically determines the rigid body domain rotations of the ATPases and consequently their position within the staircase. We therefore refer to these subunits as apo and ATP1 (Figure S2A). Importantly, we observe density corresponding to substrate peptide threaded through the center of the ATPase spiral of AFG3L2, as has been observed for other AAA+ proteins (de la Pena et al., 2018; Gates et al., 2017; Monroe et al., 2017; Puchades et al., 2017; Ripstein et al., 2017; Yu et al., 2018). Unexpectedly, this translocating substrate density directly contacts the center of the protease ring. Furthermore, use of the protease inactivating E575Q mutation resulted in observable density corresponding to substrate trapped in the proteolytic active sites of the ATP2–4 subunits (Figure 1C and 1F).Modulation of a conserved translocation mechanism by AFG3L2-specific pore-loop 1 residuesNumerous structures of substrate-bound AAA+ unfoldases have been solved by cryo-EM, and suggest a conserved substrate translocation mechanism wherein a sequential ATP hydrolysis cycle powers a hand-over-hand conveyance of unfolded polypeptide through the central channel of the hexamer (de la Pena et al., 2018; Gates et al., 2017; Monroe et al., 2017; Puchades et al., 2017; Ripstein et al., 2017; Yu et al., 2018). Like in other AAA+ ATPases, an aromatic residue within the ATPase domain pore-loop 1 of AFG3L2 (F381) forms a spiral staircase around the translocating substrate, and intercalates into its backbone (Figures 2A, 2C, S2B, and S2C). To verify the key role of F381 in substrate translocation, we monitored ATP hydrolysis and substrate degradation for an engineered hexameric AFG3L2 (cchexAFG3L2), where the transmembrane domains were substituted by a coiled coil that ensures hexamerization of active subunits in the absence of stabilizing mutations (Figure 1B) (Ding et al., 2018; Rampello and Glynn, 2017; Shi et al., 2016). Incorporation of an F381A substitution abolished substrate degradation, while only mildly impacting ATP hydrolysis (Figure 2B), confirming a conserved role for the aromatic pore-loop 1 residue in substrate handling.A requirement for the hand-over-hand translocation mechanism is allosteric transmission of nucleotide state to the pore-loops. This allostery was described in YME1, wherein an Asp-Gly-Phe motif located at the interface between neighboring ATPases alternates conformations between a compact α-helix and an extended loop that spans the nucleotide-binding pocket at the inter-protomer interface (Puchades et al., 2017). Conformational switching of this region, termed the inter-subunit signaling (ISS) motif, is directly controlled by nucleotide state, which in turn defines the positions of pore-loops 1 and 2 (Augustin et al., 2009; Puchades et al., 2017). Together, this gives rise to a mechanism wherein the pore loops of ATP-bound subunits engage the substrate, ATP hydrolysis in the lowermost subunit causes its pore loops to detach from the substrate, and nucleotide exchange enables re-engagement of substrate at the top of the staircase. The correlation between nucleotide state, conformation of the ISS and positioning of the pore loops is conserved in our AFG3L2 structure, indicating that allosteric regulation of pore-loop conformation in response to nucleotide state, and thereby the fundamental mechanism of substrate translocation, is conserved between i- and m-AAA proteases (Figure S2B and S2C).Despite conservation of the overall mechanism, the pore-loops of AFG3L2 contain structural features distinct from those observed in YME1 or other AAA+ ATPases, which modulate substrate translocation. The methionine residue (M380) immediately preceding the pore-loop 1 aromatic (F381) contacts F381 of the counterclockwise adjacent subunit to form a continuous chain of residues that wrap around the substrate in the central channel (Figure 2C). This methionine is largely conserved in FtsH-related enzymes, with the notable exception of YME1 (Figure S3A). An M380V mutation, which converts this methionine to the corresponding valine in YME1, increased ATP hydrolysis while moderately decreasing substrate degradation indicating reduced enzymatic efficiency (Figure 2B). Importantly, in most classical AAA+ proteins, including Vps4, the NDB1 domain of HSP104, as well as five of the six ATPase subunits of the 26S proteasome, this residue corresponds to a lysine that forms π-cation interactions with the aromatic residues of both cis and trans loops (de la Pena et al., 2018; Gates et al., 2017; Monroe et al., 2017). In AFG3L2, the methionine cannot engage in π-cation interactions with the aromatic staircase and instead is limited to steric hindrance and weak van der Waals forces. Replacing this methionine with a lysine (M380K) completely abolished AFG3L2 activity (Figure 2B), consistent with corresponding Met to Lys mutations in the AFG3L2 homologs Yta10 and Yta12 that obliterated substrate processing in yeast (Tatsuta et al., 2007). Together, these data show that the residue immediately prior to the pore-loop 1 aromatic gives rise to a distinct configuration of the central channel that specifically influences substrate translocation in different AAA+ proteins.Pore-loop 2 coordinates substrate transfer to a central protrusion within the protease domainPore-loop 2 is the least conserved region of the ATPase domain across FtsH-related proteases. Surprisingly, we observe that a four-residue insertion in pore-loop 2 of AFG3L2 positions the central aromatic residue F421 closer to the substrate than observed for YME1 (Puchades et al., 2017), creating a second spiral staircase that engages the substrate – an interaction not observed in any other substrate-bound AAA+ ATPase solved to date. Incorporation of an F421A substitution in cchexAFG3L2 impaired substrate degradation with no impact on ATPase activity (Figure 2B). These additional contacts with the translocating substrate likely increase the ATPase’s “grip” on substrate for optimal processing of AFG3L2-specific substrates.Importantly, the extended AFG3L2 pore-loop 2 positions F421 of the two lowest spiral subunits (ATP4 and ADP; Figure 2E and 2F) deep in the interior of the proteolytic cavity. In this configuration, F421 of the ADP subunit’s pore-loop 2 contacts a ‘central protrusion’ within the protease domain. This protrusion is formed by residues 673–695 of each subunit, which form a hexamer of six upward-projecting loops at the central base of the protease ring, directed toward the incoming translocating substrate (Figure 2D and 2E). Six M683 residues crown the top of the central protrusion, and F421 from the lowermost pore-loop 2 directly contacts the nearest M683 in a manner reminiscent of the Phe-Met interactions observed in the pore-loop 1 spiral (Figure 2D–2F). Moreover, at low sigma values the density corresponding to translocating substrate extends beyond the lowermost F421 and in close proximity to the M683 crown (Figure 2D). Incorporation of an M683A substitution into cchexAFG3L2 moderately reduced substrate degradation while minimally affecting ATP hydrolysis, mirroring the impact of the F421A mutation (Figure 2B). We thus conclude that the organization of the extended pore-loop 2 and the central protease loops in AFG3L2 likely plays an important role in facilitating substrate transfer from the ATPase spiral to the proteolytic ring.The N-terminus of the AFG3L2 ATPase domain mediates membrane-proximal interactions with the substrateOur structure also reveals a previously undescribed organization of the ATPase domain’s N-terminus. This N-terminus connects the catalytic core of AFG3L2 to the transmembrane domains and begins with a flexible glycine-rich region (residues 272–287). Despite this region being present in our coreAFG3L2 construct, it was not observed in our cryo-EM reconstruction (Figure S3A). However, the residues immediately following this region (288–295) form an additional spiral staircase above the ATPase domains that surrounds and contacts the translocating substrate (blue in Figure 3A). Deleting the ordered, substrate-interacting ATPase N-terminus in cchexAFG3L2 (residues 272–295) significantly impaired substrate degradation without impacting hydrolysis, whereas deletion of just the flexible Gly-rich linker region (residues 272–281) did not influence either activity. While the resolution of our reconstruction in this region is insufficient to determine the precise nature of the interactions with the substrate, the position of the backbone chain indicates that L288 and F289 are in the immediate vicinity of the substrate. Accordingly, an F289A substitution reduced the rate of protein degradation by ~30 % with no significant alteration in ATPase rate (Figure 3D). Based on these findings, we speculate that the N-termini might form an additional, substrate-intercalating staircase, closely mirroring the organization of the pore loops 1 and 2.After leaving the site of substrate interaction near the central channel, each N-terminus snakes across the interface between neighboring ATPases, directly above the intersubunit interface where the ISS motif engages the clockwise adjacent subunit (Figure 3B and 3C). Interestingly, L299 of each N-terminus inserts into a hydrophobic groove formed by the trans packing of the ISS motif against each ATP-bound subunit (Figure 3C). In the ADP-bound subunit, for which there are no ISS interactions and thus the hydrophobic groove is absent, L299 is repositioned closer to the ISS of the cis subunit. The maintenance of these interactions regardless of nucleotide state suggested that the packing of L299 against the ATPase body may stabilize the N-terminus and promote interaction with substrate. As predicted, cchexAFG3L2 containing L299A shows 50% reduction in substrate degradation without impacting ATP hydrolysis (Figure 3D). Taken together, this reveals an unanticipated role for the AFG3L2 ATPase N-terminus in mediating contacts with the substrate that are important for engagement and translocation.The AFG3L2 C-terminus encircles the hexamer to adopt a membrane proximal positionOur construct contained the complete AFG3L2 C-terminus, comprising residues 750–797 – a region that is highly variable among FtsH-related AAA+ proteases and, in AFG3L2, contains a number of charged residues at the far C-terminus (Figure S3B). While we do not observe density for the charged C-terminal tail in the structure (residues 780–797), well-defined density is present for the residues immediately preceding the tail (residues 750–779) in the three AMP-PNP bound subunits (ATP2–4) (Figure 3A). Unexpectedly, this ordered C-terminus extends upwards from the base of the protease domain along the exterior surface of the complex to interact with both the protease and the ATPase domains of the cis subunit, and the interdomain linker of the counterclockwise adjacent subunit. This arrangement of the C-terminus would position the highly charged tail at the membrane-proximal face of AFG3L2. Truncation of the C-terminus at residue 750 in the coreAFG3L2WB/PI construct, which is not stabilized by the hexamerizing coiled coil, decreased recovery of AFG3L2 hexamers as measured by size exclusion chromatography (SEC; Figure S4A). In contrast, removal of the unobserved charged tail alone (residues 781–797) did not impact complex recovery. This indicates that the interaction of the C-terminal residues 750–779 across neighboring ATPase domains is important for stabilization and assembly of the active hexamer. Consistent with this, deletion of residues 750–797 in cchexAFG3L2 reduced both ATPase activity and substrate degradation, whereas deletion of residues 781–797 had no significant effect on either activity. The final C-terminal residue for which we observe well-ordered density is W779, which is sandwiched between M321 and K328 at the membrane-proximal surface of the ATPase domain (Figure 3C). These stabilizing sulfur-aromatic and cation-π interactions likely secure W779 in position, anchoring the base of the charged C-terminal tail for potential substrate interaction. In support of this, incorporating a W779R substitution into cchexAFG3L2 impaired substrate degradation without impacting ATP hydrolysis (Figure 3D). This reveals an important role for the C-terminus in dictating complex stability, and shows that interactions driven by the organization of C-terminal residues influence important aspects of AFG3L2 substrate processing.Disease-relevant AFG3L2 mutations localize to four ‘hotspots’ on the AFG3L2 structureWe next sought to evaluate how specific disease-relevant AFG3L2 mutations impact its activity. Mapping disease-relevant AFG3L2 mutations onto our structure reveals that all mutations localize to four ‘hotspots’ within the hexameric complex (Figure 1C–1F). Intriguingly, all of the autosomal dominant mutations localize to intersubunit interfaces of the hexamer, including 1) the lateral interface between adjacent ATPase subunits where the nucleotide binding pocket is formed, 2) the tightly interconnected helices that dominate the lateral interface of the protease domain, and 3) the central protrusion of the AFG3L2 protease ring that is involved in substrate transfer to the protease domains. In contrast, recessive mutations localize near the protease active sites in the periphery of the protease ring. The clustering of these mutations into distinct regions in the structure suggests distinct effects on AFG3L2 activity.Disease mutations in the AAA+ domains disrupt nucleotide-dependent substrate translocationTwo disease-relevant mutations, N432T (linked to SCA28) and R468C (DOA), localize to the vicinity of the nucleotide-binding pocket at the ATPase intersubunit interface (Charif et al., 2015; Colavito et al., 2017; Di Bella et al., 2010). Our structure shows that R468 is one of two Arg-finger residues that projects from the adjacent large ATPase subdomain to coordinate interactions with the γ-phosphate (Figure 3C and S2B). These Arg-finger residues are highly conserved across classical AAA+ enzymes and are suggested to play roles in ATP binding and/or intersubunit interactions (Wendler et al., 2012). In fact, mutation of analogous Arg-fingers in bacterial FtsH obliterated activity (Karata et al., 1999). Accordingly, the R468C patient mutation abolishes ATP hydrolysis and substrate degradation when incorporated into cchexAFG3L2 (Figure 2B). Consistent with impaired ATP binding at the intersubunit interface, R468C decreased recovery of AFG3L2 hexamers by SEC, mirroring the behavior of an ATP-binding incompetent Walker A control (K354A, denoted AFG3L2WA) (Figure S4B).Previously, homology models of FtsH were used to predict that N432T might impair substrate engagement in the central pore (Di Bella et al., 2010). Our structure shows that N432 interacts with R416 within pore-loop 2, positioning this loop to facilitate substrate engagement (Figure 2A). Strict conservation of both N432 and R416 within the FtsH family implies that this interaction exists in other family members (Figure S3A). Both R416A and disease-relevant N432T substitutions abolished substrate degradation in cchexAFG3L2, indicating a key functional role for this interaction. Surprisingly, these two mutations also significantly reduced ATPase activity. N432 is located within helix α5 immediately adjacent to the ISS motif, suggesting that this mutation might disrupt allosteric coordination of ATP hydrolysis. In agreement, N432T displayed a significantly reduced maximal steady-state ATPase rate but retained high affinity for ATP, indicating that this mutation impairs catalysis without impacting nucleotide binding (Figure S4F). Thus, our structure-function studies indicate that the disease-relevant N432T mutation allosterically affects both the ATP hydrolysis cycle and organization of pore-loop 2.Disease mutations in the protease domain concentrate near the central protrusionWith the exception of N432T, all SCA28-associated mutations identified to date localize to two hotspots within, or at the base of, the central protease loops. These loops form a channel-like protrusion at the center of the proteolytic ring that interacts with pore-loop 2 and the translocating substrate (Figure 4A–4D). In each subunit, the base of the central protease loop is stabilized by π-stacking intrasubunit interactions between Y689 and F675 (Figure 4C). Intriguingly, layered rings alternating between acidic (E686 and E691) and basic (K687 and R695) residues create a highly charged channel surface (Figure 4A and 4B).All of the mutations in this region line the channel of the central protrusion and likely impact the integrity of its structure. E691K swaps the charge of one of the acidic rings, which would disrupt the charged ring system within the channel (Figure 4B). Similarly, A694E positions a negatively charged residue adjacent to E691, introducing charge repulsion between these residues (Figure 4B). At the base of the central protease loop, substitutions of Y689 (Y689N and Y689H) would eliminate the π-stacking interactions that maintain loop integrity (Figure 4C). Finally, P688 introduces a sharp turn at the base of the loop that correctly positions Y689 and K687 for interaction with other residues in this region (Figure 4C). The P688T substitution will likely add flexibility at this position and decrease the stability of the central protrusion.In contrast to the central protease loop, residues at the base of the central protrusion are highly conserved across FtsH-related proteases, and our structure indicates that the majority of these residues are involved in intersubunit interactions that stabilize the proteolytic ring. Notably, all remaining SCA28-associated residues are located in this region, including one of the most highly mutated residues in disease, M666, which packs tightly into a hydrophobic pocket at the intersubunit interface (Figure 4D). Mutation of M666 to Thr or Val would perturb this hydrophobic pocket. M666R, which would disrupt this pocket by introducing a positive charge, results in one of the most severe disease phenotypes in patients. R702Q and G671R/E would destabilize a charged cluster at the base of the central protrusion (Figure 4D). Similarly, E700K would disrupt ionic interactions with K669 at the base of the central protrusion in the same subunit (Figure 4D). T654I and S674L would introduce bulky hydrophobic residues into a closely packed intersubunit interface (Figure 4D). Based on these structural observations, we predict that all SCA28-associated mutations localized within or at the base of the central protrusion destabilize the protease, thereby impacting AFG3L2 function. A collection of residues previously identified as important for assembly of the heterohexameric yeast m-AAA protease also locate to the central protrusion, further highlighting the importance of this feature in complex stability (Lee et al., 2011).Incorporating these protease mutations into coreAFG3L32WB/PI significantly impaired recovery of the hexameric protein by SEC, albeit to different extents (Figure S4C and S4D). For example, M666R, which disrupts the hydrophobic interface between protease subunits, completely abolished hexamer recovery, while P688T, which increases flexibility at the base of the protrusion, only modestly affected stability (Figure S4C and S4D). This global destabilization of the catalytic core was further evident when we incorporated protease mutations into cchexAFG3L2. Despite hexamerization, all protease mutations show reductions in both ATP hydrolysis and substrate degradation that reflect the degree of destabilization observed by SEC (Figure 5A). Moreover, reductions of activity in vitro appear to correlate with disease severity in patients. For instance, P688T presents only a modest reduction in activity, and is associated with the mildest condition in SCA28 patients, whereas M666R shows a near complete loss of activity and is associated with a very severe, early onset phenotype in patients (Di Bella et al., 2010; Svenstrup et al., 2017).To further define the impact of these mutations on AFG3L2 stability, we incorporated the mild mutation P688T and the severe mutations M666R and E691K into C-terminally Flag-tagged AFG3L2 (AFG3L2FT) and monitored assembly and stability by Blue-Native (BN-PAGE) and SDS-PAGE. By BN-PAGE, P688T showed a small increase in AFG3L2 oligomer migration and a small increase in AFG3L2 monomers, indicating modest destabilization (Figure 5B). In contrast, M666R and E691K show significant streaking of the AFG3L2 oligomer band (Figure 5B), consistent with the more severe destabilization of AFG3L2 observed in vitro (Figure S4C and S4D). The varying levels of AFG3L2 destabilization afforded by these mutations appears to be in part mediated through proteolytic cleavage of the protomer. Using antibodies that recognize the N- or C-terminus of AFG3L2FT, we observed the accumulation of an N-terminal cleavage product of ~50 kDa in SDS-PAGE that corresponds to loss of the C-terminal protease domain for all three of these mutants (Figure 5C). P688T leads to a more modest accumulation of this cleavage product, consistent with the modest effect on oligomer stability (Figures 5C and S4C) and activity (Figure 5A). However, AFG3L2 harboring the M666R or E691K mutation showed significant accumulation of this cleavage product (Figure 5C), reflecting marked destabilization (Figures 5B, S4C, and S4D) and severe reduction in activity (Figure 5A). These results further support a model whereby mutations in the protease domain destabilize the protease ring, leading to SCA28, with the extent of destabilization being an important factor for disease severity and age of onset in patients (Table S1).Analyzing the ATPase mutants N432T and R468C using the same approach produced starkly contrasting results (Figure 5B and 5C). To determine whether their distinct profiles are related to impairment of the ATP hydrolysis cycle, we expressed control mutants Walker A (prevents ATP binding) and Walker B (prevents ATP hydrolysis). As in our SEC experiments, the Walker B mutant increased stability of the AFG3L2 hexamer, whereas appearance of AFG3L2 monomers in the Walker A mutant indicates disassembly of the complex. In agreement with our in vitro data, R468C closely mirrors the behavior of the Walker A mutant, potentially reflecting the unique DOA pathology observed in patients (Figure 5B and 5C). While the SDS-PAGE profile of N432T is similar to that of the Walker B mutant, N432T causes a mild destabilization of AFG3L2 oligomers by BN-PAGE (Figure 5B). Our biochemical and structural data indicate that N432T may allosterically impact both ATP hydrolysis and pore-loop organization (Figure 2). Thus, the decreased stability observed in cells might be due to impairment of substrate binding, providing a potential explanation for the dominant negative effect seen for this mutation (Di Bella et al., 2010).Recessively inherited mutations are involved in substrate cleavage at the proteolytic active siteIn contrast to the SCA28 and DOA mutations, two single-point mutations in the protease domain of AFG3L2 are linked to severe neurodegenerative disorders with distinct phenotypes only in homozygotes (Y616C, SPAX5; A572T, as yet unnamed neurodegenerative disease) (Eskandrani et al., 2017; Pierson et al., 2011). We found that both mutations are in close proximity to the protease active site (Figure 6A). In AFG3L2, six zinc-metalloprotease active sites are sequestered to the interior periphery of the proteolytic ring (Figure 6A). At each site, a zinc ion is coordinated for catalysis by H574 and H578 from helix α10 and D649 in helix α12 (Figure 6A). The His residues are part of a strictly conserved HEXXH motif that is required for proteolytic activity. Next to this motif is the conserved residue A572, which is directed towards a hydrophobic pocket on the opposite side of α10 from the active site (Figures 6A and S5A). The disease mutation A572T would introduce a polar side chain into this pocket, perturbing the local structure of the active site. In agreement, cchexAFG3L2 containing the A572T mutation shows significantly decreased protein degradation activity and a moderate defect in ATP hydrolysis (Figure 6B). Moreover, introduction of A572T to coreAFG3L2WB/PI did not hamper hexamer recovery by SEC, indicating local structural defects rather than global destabilization (Figure S4E).To confirm that A572T directly affects proteolytic activity, we used an ATPase-inactive/peptidase-active variant of the coreAFG3L2 protein (coreAFG3L2WB) to measure the ATP-independent cleavage of a small fluorogenic peptide (Ding et al., 2018). Mutations in the N-terminus (coreAFG3L2WB/L299A) and central protrusion (coreAFG3L2WB/M683A), both of which lower substrate degradation rates in the ATPase active context (Figures 2 and 3), did not alter the peptide cleavage rate, indicating that this assay reports solely on proteolytic activity independent of upstream mechanistic defects (Figure 6C). A572T reduced peptide cleavage rates by 50%, demonstrating that this mutation directly impairs proteolysis at the active site (Figure 6C).Our structure of catalytically inactive AFG3L2 contained density extending across three of the six protease active sites, which we attribute to substrate peptide that is positioned for cleavage (Figure 6A). This substrate appears to form an additional β-strand next to β7, positioning the polypeptide above the catalytic triad. AFG3L2 preferentially cleaves sequences containing Phe residues in the P1’ position, immediately C-terminal of the scissile bond (Ding et al., 2018). The position of the substrate in our structure locates the putative specificity site (S1’) to a hydrophobic pocket created by V571, L603 and G645 from a single subunit, and L615 from the adjacent subunit (Figure S5B). Although no density is observed for the side chain of the substrate residue closest to this position, a Phe side chain can be easily accommodated in this pocket. L615 is part of a non-conserved loop (Y614, L615 and Y616) at the intersubunit interface, and density for the substrate backbone is seen positioned between the side chains of Y614 and the SPAX5-related residue Y616 (Figures 6A and S5B). Strikingly, the disease mutation Y616C displayed increased ATPase and protein degradation rates, with a modest reduction in complex stability (Figure 6B and S4E). Moreover, ATP-independent peptidase activity was two-fold higher for Y616C compared to controls (Figure 6C). These results suggest that substitution of Y616 with a smaller Cys may reduce steric hindrance of incoming polypeptides, increasing accessibility to the S1’ pocket. The distinct gain-of-function effect of Y616C provides a plausible explanation for the unique phenotype of this mutation in patients.

Structure of the human substrate-bound AFG3L2 catalytic coreWe solved a ~3.1 Å resolution cryo-EM structure of a truncated construct of AFG3L2 comprising the ATPase and protease domains (residues 272–797; coreAFG3L2) (Figures 1 and S1). The construct was stabilized for structural analysis by incorporating ATPase-inactivating Walker B (WB, E408Q) and protease-inactivating (PI, E575Q) mutations, and vitrified in the presence of saturating concentrations of the non-hydrolyzable ATP analog AMP-PNP. The resulting reconstruction was of sufficient quality to build an atomic model of the six protease domains and four of the six ATPase domains (Figures 1 and S1). The atomic models of the large and small ATPase subdomains of one of the well-resolved subunits were independently rigid-body fit into the low resolution cryo-EM density of the remaining two ATPase domains to generate a backbone trace in these regions, completing an atomic model of the entire homohexameric AFG3L2 complex (Figures 1, S2, and Table S2).Our reconstruction shows that AFG3L2 assembles into two stacked rings with a planar C6 symmetric protease base topped by an asymmetric ATPase spiral (Figures 1D, 1E, and 1F) – a quaternary structure similar to that observed for substrate-bound YME1 (Figure S2A) (Puchades et al., 2017). The three upper subunits of the AFG3L2 staircase are bound to AMP-PNP (termed ATP2-ATP4 for simplicity), while the lowermost subunit is bound to ADP (Figure S2). The limited resolution of the remaining two subunits did not allow for unambiguous identification of nucleotide state, although the positioning of these subunits within the staircase correspond to the nucleotide-free (apo) ‘step’-subunit and the uppermost ATP-bound subunit (ATP1) within YME1 (Figure S2A). While we could not experimentally verify the identity of the nucleotide in these subunits, we previously showed that nucleotide state allosterically determines the rigid body domain rotations of the ATPases and consequently their position within the staircase. We therefore refer to these subunits as apo and ATP1 (Figure S2A). Importantly, we observe density corresponding to substrate peptide threaded through the center of the ATPase spiral of AFG3L2, as has been observed for other AAA+ proteins (de la Pena et al., 2018; Gates et al., 2017; Monroe et al., 2017; Puchades et al., 2017; Ripstein et al., 2017; Yu et al., 2018). Unexpectedly, this translocating substrate density directly contacts the center of the protease ring. Furthermore, use of the protease inactivating E575Q mutation resulted in observable density corresponding to substrate trapped in the proteolytic active sites of the ATP2–4 subunits (Figure 1C and 1F).

Structure of the human substrate-bound AFG3L2 catalytic core

We solved a ~3.1 Å resolution cryo-EM structure of a truncated construct of AFG3L2 comprising the ATPase and protease domains (residues 272–797; coreAFG3L2) (Figures 1 and S1). The construct was stabilized for structural analysis by incorporating ATPase-inactivating Walker B (WB, E408Q) and protease-inactivating (PI, E575Q) mutations, and vitrified in the presence of saturating concentrations of the non-hydrolyzable ATP analog AMP-PNP. The resulting reconstruction was of sufficient quality to build an atomic model of the six protease domains and four of the six ATPase domains (Figures 1 and S1). The atomic models of the large and small ATPase subdomains of one of the well-resolved subunits were independently rigid-body fit into the low resolution cryo-EM density of the remaining two ATPase domains to generate a backbone trace in these regions, completing an atomic model of the entire homohexameric AFG3L2 complex (Figures 1, S2, and Table S2).

Our reconstruction shows that AFG3L2 assembles into two stacked rings with a planar C6 symmetric protease base topped by an asymmetric ATPase spiral (Figures 1D, 1E, and 1F) – a quaternary structure similar to that observed for substrate-bound YME1 (Figure S2A) (Puchades et al., 2017). The three upper subunits of the AFG3L2 staircase are bound to AMP-PNP (termed ATP2-ATP4 for simplicity), while the lowermost subunit is bound to ADP (Figure S2). The limited resolution of the remaining two subunits did not allow for unambiguous identification of nucleotide state, although the positioning of these subunits within the staircase correspond to the nucleotide-free (apo) ‘step’-subunit and the uppermost ATP-bound subunit (ATP1) within YME1 (Figure S2A). While we could not experimentally verify the identity of the nucleotide in these subunits, we previously showed that nucleotide state allosterically determines the rigid body domain rotations of the ATPases and consequently their position within the staircase. We therefore refer to these subunits as apo and ATP1 (Figure S2A). Importantly, we observe density corresponding to substrate peptide threaded through the center of the ATPase spiral of AFG3L2, as has been observed for other AAA+ proteins (de la Pena et al., 2018; Gates et al., 2017; Monroe et al., 2017; Puchades et al., 2017; Ripstein et al., 2017; Yu et al., 2018). Unexpectedly, this translocating substrate density directly contacts the center of the protease ring. Furthermore, use of the protease inactivating E575Q mutation resulted in observable density corresponding to substrate trapped in the proteolytic active sites of the ATP2–4 subunits (Figure 1C and 1F).

Modulation of a conserved translocation mechanism by AFG3L2-specific pore-loop 1 residuesNumerous structures of substrate-bound AAA+ unfoldases have been solved by cryo-EM, and suggest a conserved substrate translocation mechanism wherein a sequential ATP hydrolysis cycle powers a hand-over-hand conveyance of unfolded polypeptide through the central channel of the hexamer (de la Pena et al., 2018; Gates et al., 2017; Monroe et al., 2017; Puchades et al., 2017; Ripstein et al., 2017; Yu et al., 2018). Like in other AAA+ ATPases, an aromatic residue within the ATPase domain pore-loop 1 of AFG3L2 (F381) forms a spiral staircase around the translocating substrate, and intercalates into its backbone (Figures 2A, 2C, S2B, and S2C). To verify the key role of F381 in substrate translocation, we monitored ATP hydrolysis and substrate degradation for an engineered hexameric AFG3L2 (cchexAFG3L2), where the transmembrane domains were substituted by a coiled coil that ensures hexamerization of active subunits in the absence of stabilizing mutations (Figure 1B) (Ding et al., 2018; Rampello and Glynn, 2017; Shi et al., 2016). Incorporation of an F381A substitution abolished substrate degradation, while only mildly impacting ATP hydrolysis (Figure 2B), confirming a conserved role for the aromatic pore-loop 1 residue in substrate handling.A requirement for the hand-over-hand translocation mechanism is allosteric transmission of nucleotide state to the pore-loops. This allostery was described in YME1, wherein an Asp-Gly-Phe motif located at the interface between neighboring ATPases alternates conformations between a compact α-helix and an extended loop that spans the nucleotide-binding pocket at the inter-protomer interface (Puchades et al., 2017). Conformational switching of this region, termed the inter-subunit signaling (ISS) motif, is directly controlled by nucleotide state, which in turn defines the positions of pore-loops 1 and 2 (Augustin et al., 2009; Puchades et al., 2017). Together, this gives rise to a mechanism wherein the pore loops of ATP-bound subunits engage the substrate, ATP hydrolysis in the lowermost subunit causes its pore loops to detach from the substrate, and nucleotide exchange enables re-engagement of substrate at the top of the staircase. The correlation between nucleotide state, conformation of the ISS and positioning of the pore loops is conserved in our AFG3L2 structure, indicating that allosteric regulation of pore-loop conformation in response to nucleotide state, and thereby the fundamental mechanism of substrate translocation, is conserved between i- and m-AAA proteases (Figure S2B and S2C).Despite conservation of the overall mechanism, the pore-loops of AFG3L2 contain structural features distinct from those observed in YME1 or other AAA+ ATPases, which modulate substrate translocation. The methionine residue (M380) immediately preceding the pore-loop 1 aromatic (F381) contacts F381 of the counterclockwise adjacent subunit to form a continuous chain of residues that wrap around the substrate in the central channel (Figure 2C). This methionine is largely conserved in FtsH-related enzymes, with the notable exception of YME1 (Figure S3A). An M380V mutation, which converts this methionine to the corresponding valine in YME1, increased ATP hydrolysis while moderately decreasing substrate degradation indicating reduced enzymatic efficiency (Figure 2B). Importantly, in most classical AAA+ proteins, including Vps4, the NDB1 domain of HSP104, as well as five of the six ATPase subunits of the 26S proteasome, this residue corresponds to a lysine that forms π-cation interactions with the aromatic residues of both cis and trans loops (de la Pena et al., 2018; Gates et al., 2017; Monroe et al., 2017). In AFG3L2, the methionine cannot engage in π-cation interactions with the aromatic staircase and instead is limited to steric hindrance and weak van der Waals forces. Replacing this methionine with a lysine (M380K) completely abolished AFG3L2 activity (Figure 2B), consistent with corresponding Met to Lys mutations in the AFG3L2 homologs Yta10 and Yta12 that obliterated substrate processing in yeast (Tatsuta et al., 2007). Together, these data show that the residue immediately prior to the pore-loop 1 aromatic gives rise to a distinct configuration of the central channel that specifically influences substrate translocation in different AAA+ proteins.

Modulation of a conserved translocation mechanism by AFG3L2-specific pore-loop 1 residues

Numerous structures of substrate-bound AAA+ unfoldases have been solved by cryo-EM, and suggest a conserved substrate translocation mechanism wherein a sequential ATP hydrolysis cycle powers a hand-over-hand conveyance of unfolded polypeptide through the central channel of the hexamer (de la Pena et al., 2018; Gates et al., 2017; Monroe et al., 2017; Puchades et al., 2017; Ripstein et al., 2017; Yu et al., 2018). Like in other AAA+ ATPases, an aromatic residue within the ATPase domain pore-loop 1 of AFG3L2 (F381) forms a spiral staircase around the translocating substrate, and intercalates into its backbone (Figures 2A, 2C, S2B, and S2C). To verify the key role of F381 in substrate translocation, we monitored ATP hydrolysis and substrate degradation for an engineered hexameric AFG3L2 (cchexAFG3L2), where the transmembrane domains were substituted by a coiled coil that ensures hexamerization of active subunits in the absence of stabilizing mutations (Figure 1B) (Ding et al., 2018; Rampello and Glynn, 2017; Shi et al., 2016). Incorporation of an F381A substitution abolished substrate degradation, while only mildly impacting ATP hydrolysis (Figure 2B), confirming a conserved role for the aromatic pore-loop 1 residue in substrate handling.

A requirement for the hand-over-hand translocation mechanism is allosteric transmission of nucleotide state to the pore-loops. This allostery was described in YME1, wherein an Asp-Gly-Phe motif located at the interface between neighboring ATPases alternates conformations between a compact α-helix and an extended loop that spans the nucleotide-binding pocket at the inter-protomer interface (Puchades et al., 2017). Conformational switching of this region, termed the inter-subunit signaling (ISS) motif, is directly controlled by nucleotide state, which in turn defines the positions of pore-loops 1 and 2 (Augustin et al., 2009; Puchades et al., 2017). Together, this gives rise to a mechanism wherein the pore loops of ATP-bound subunits engage the substrate, ATP hydrolysis in the lowermost subunit causes its pore loops to detach from the substrate, and nucleotide exchange enables re-engagement of substrate at the top of the staircase. The correlation between nucleotide state, conformation of the ISS and positioning of the pore loops is conserved in our AFG3L2 structure, indicating that allosteric regulation of pore-loop conformation in response to nucleotide state, and thereby the fundamental mechanism of substrate translocation, is conserved between i- and m-AAA proteases (Figure S2B and S2C).

Despite conservation of the overall mechanism, the pore-loops of AFG3L2 contain structural features distinct from those observed in YME1 or other AAA+ ATPases, which modulate substrate translocation. The methionine residue (M380) immediately preceding the pore-loop 1 aromatic (F381) contacts F381 of the counterclockwise adjacent subunit to form a continuous chain of residues that wrap around the substrate in the central channel (Figure 2C). This methionine is largely conserved in FtsH-related enzymes, with the notable exception of YME1 (Figure S3A). An M380V mutation, which converts this methionine to the corresponding valine in YME1, increased ATP hydrolysis while moderately decreasing substrate degradation indicating reduced enzymatic efficiency (Figure 2B). Importantly, in most classical AAA+ proteins, including Vps4, the NDB1 domain of HSP104, as well as five of the six ATPase subunits of the 26S proteasome, this residue corresponds to a lysine that forms π-cation interactions with the aromatic residues of both cis and trans loops (de la Pena et al., 2018; Gates et al., 2017; Monroe et al., 2017). In AFG3L2, the methionine cannot engage in π-cation interactions with the aromatic staircase and instead is limited to steric hindrance and weak van der Waals forces. Replacing this methionine with a lysine (M380K) completely abolished AFG3L2 activity (Figure 2B), consistent with corresponding Met to Lys mutations in the AFG3L2 homologs Yta10 and Yta12 that obliterated substrate processing in yeast (Tatsuta et al., 2007). Together, these data show that the residue immediately prior to the pore-loop 1 aromatic gives rise to a distinct configuration of the central channel that specifically influences substrate translocation in different AAA+ proteins.

Pore-loop 2 coordinates substrate transfer to a central protrusion within the protease domainPore-loop 2 is the least conserved region of the ATPase domain across FtsH-related proteases. Surprisingly, we observe that a four-residue insertion in pore-loop 2 of AFG3L2 positions the central aromatic residue F421 closer to the substrate than observed for YME1 (Puchades et al., 2017), creating a second spiral staircase that engages the substrate – an interaction not observed in any other substrate-bound AAA+ ATPase solved to date. Incorporation of an F421A substitution in cchexAFG3L2 impaired substrate degradation with no impact on ATPase activity (Figure 2B). These additional contacts with the translocating substrate likely increase the ATPase’s “grip” on substrate for optimal processing of AFG3L2-specific substrates.Importantly, the extended AFG3L2 pore-loop 2 positions F421 of the two lowest spiral subunits (ATP4 and ADP; Figure 2E and 2F) deep in the interior of the proteolytic cavity. In this configuration, F421 of the ADP subunit’s pore-loop 2 contacts a ‘central protrusion’ within the protease domain. This protrusion is formed by residues 673–695 of each subunit, which form a hexamer of six upward-projecting loops at the central base of the protease ring, directed toward the incoming translocating substrate (Figure 2D and 2E). Six M683 residues crown the top of the central protrusion, and F421 from the lowermost pore-loop 2 directly contacts the nearest M683 in a manner reminiscent of the Phe-Met interactions observed in the pore-loop 1 spiral (Figure 2D–2F). Moreover, at low sigma values the density corresponding to translocating substrate extends beyond the lowermost F421 and in close proximity to the M683 crown (Figure 2D). Incorporation of an M683A substitution into cchexAFG3L2 moderately reduced substrate degradation while minimally affecting ATP hydrolysis, mirroring the impact of the F421A mutation (Figure 2B). We thus conclude that the organization of the extended pore-loop 2 and the central protease loops in AFG3L2 likely plays an important role in facilitating substrate transfer from the ATPase spiral to the proteolytic ring.

Pore-loop 2 coordinates substrate transfer to a central protrusion within the protease domain

Pore-loop 2 is the least conserved region of the ATPase domain across FtsH-related proteases. Surprisingly, we observe that a four-residue insertion in pore-loop 2 of AFG3L2 positions the central aromatic residue F421 closer to the substrate than observed for YME1 (Puchades et al., 2017), creating a second spiral staircase that engages the substrate – an interaction not observed in any other substrate-bound AAA+ ATPase solved to date. Incorporation of an F421A substitution in cchexAFG3L2 impaired substrate degradation with no impact on ATPase activity (Figure 2B). These additional contacts with the translocating substrate likely increase the ATPase’s “grip” on substrate for optimal processing of AFG3L2-specific substrates.

Importantly, the extended AFG3L2 pore-loop 2 positions F421 of the two lowest spiral subunits (ATP4 and ADP; Figure 2E and 2F) deep in the interior of the proteolytic cavity. In this configuration, F421 of the ADP subunit’s pore-loop 2 contacts a ‘central protrusion’ within the protease domain. This protrusion is formed by residues 673–695 of each subunit, which form a hexamer of six upward-projecting loops at the central base of the protease ring, directed toward the incoming translocating substrate (Figure 2D and 2E). Six M683 residues crown the top of the central protrusion, and F421 from the lowermost pore-loop 2 directly contacts the nearest M683 in a manner reminiscent of the Phe-Met interactions observed in the pore-loop 1 spiral (Figure 2D–2F). Moreover, at low sigma values the density corresponding to translocating substrate extends beyond the lowermost F421 and in close proximity to the M683 crown (Figure 2D). Incorporation of an M683A substitution into cchexAFG3L2 moderately reduced substrate degradation while minimally affecting ATP hydrolysis, mirroring the impact of the F421A mutation (Figure 2B). We thus conclude that the organization of the extended pore-loop 2 and the central protease loops in AFG3L2 likely plays an important role in facilitating substrate transfer from the ATPase spiral to the proteolytic ring.

The N-terminus of the AFG3L2 ATPase domain mediates membrane-proximal interactions with the substrateOur structure also reveals a previously undescribed organization of the ATPase domain’s N-terminus. This N-terminus connects the catalytic core of AFG3L2 to the transmembrane domains and begins with a flexible glycine-rich region (residues 272–287). Despite this region being present in our coreAFG3L2 construct, it was not observed in our cryo-EM reconstruction (Figure S3A). However, the residues immediately following this region (288–295) form an additional spiral staircase above the ATPase domains that surrounds and contacts the translocating substrate (blue in Figure 3A). Deleting the ordered, substrate-interacting ATPase N-terminus in cchexAFG3L2 (residues 272–295) significantly impaired substrate degradation without impacting hydrolysis, whereas deletion of just the flexible Gly-rich linker region (residues 272–281) did not influence either activity. While the resolution of our reconstruction in this region is insufficient to determine the precise nature of the interactions with the substrate, the position of the backbone chain indicates that L288 and F289 are in the immediate vicinity of the substrate. Accordingly, an F289A substitution reduced the rate of protein degradation by ~30 % with no significant alteration in ATPase rate (Figure 3D). Based on these findings, we speculate that the N-termini might form an additional, substrate-intercalating staircase, closely mirroring the organization of the pore loops 1 and 2.After leaving the site of substrate interaction near the central channel, each N-terminus snakes across the interface between neighboring ATPases, directly above the intersubunit interface where the ISS motif engages the clockwise adjacent subunit (Figure 3B and 3C). Interestingly, L299 of each N-terminus inserts into a hydrophobic groove formed by the trans packing of the ISS motif against each ATP-bound subunit (Figure 3C). In the ADP-bound subunit, for which there are no ISS interactions and thus the hydrophobic groove is absent, L299 is repositioned closer to the ISS of the cis subunit. The maintenance of these interactions regardless of nucleotide state suggested that the packing of L299 against the ATPase body may stabilize the N-terminus and promote interaction with substrate. As predicted, cchexAFG3L2 containing L299A shows 50% reduction in substrate degradation without impacting ATP hydrolysis (Figure 3D). Taken together, this reveals an unanticipated role for the AFG3L2 ATPase N-terminus in mediating contacts with the substrate that are important for engagement and translocation.

The N-terminus of the AFG3L2 ATPase domain mediates membrane-proximal interactions with the substrate

Our structure also reveals a previously undescribed organization of the ATPase domain’s N-terminus. This N-terminus connects the catalytic core of AFG3L2 to the transmembrane domains and begins with a flexible glycine-rich region (residues 272–287). Despite this region being present in our coreAFG3L2 construct, it was not observed in our cryo-EM reconstruction (Figure S3A). However, the residues immediately following this region (288–295) form an additional spiral staircase above the ATPase domains that surrounds and contacts the translocating substrate (blue in Figure 3A). Deleting the ordered, substrate-interacting ATPase N-terminus in cchexAFG3L2 (residues 272–295) significantly impaired substrate degradation without impacting hydrolysis, whereas deletion of just the flexible Gly-rich linker region (residues 272–281) did not influence either activity. While the resolution of our reconstruction in this region is insufficient to determine the precise nature of the interactions with the substrate, the position of the backbone chain indicates that L288 and F289 are in the immediate vicinity of the substrate. Accordingly, an F289A substitution reduced the rate of protein degradation by ~30 % with no significant alteration in ATPase rate (Figure 3D). Based on these findings, we speculate that the N-termini might form an additional, substrate-intercalating staircase, closely mirroring the organization of the pore loops 1 and 2.

After leaving the site of substrate interaction near the central channel, each N-terminus snakes across the interface between neighboring ATPases, directly above the intersubunit interface where the ISS motif engages the clockwise adjacent subunit (Figure 3B and 3C). Interestingly, L299 of each N-terminus inserts into a hydrophobic groove formed by the trans packing of the ISS motif against each ATP-bound subunit (Figure 3C). In the ADP-bound subunit, for which there are no ISS interactions and thus the hydrophobic groove is absent, L299 is repositioned closer to the ISS of the cis subunit. The maintenance of these interactions regardless of nucleotide state suggested that the packing of L299 against the ATPase body may stabilize the N-terminus and promote interaction with substrate. As predicted, cchexAFG3L2 containing L299A shows 50% reduction in substrate degradation without impacting ATP hydrolysis (Figure 3D). Taken together, this reveals an unanticipated role for the AFG3L2 ATPase N-terminus in mediating contacts with the substrate that are important for engagement and translocation.

The AFG3L2 C-terminus encircles the hexamer to adopt a membrane proximal positionOur construct contained the complete AFG3L2 C-terminus, comprising residues 750–797 – a region that is highly variable among FtsH-related AAA+ proteases and, in AFG3L2, contains a number of charged residues at the far C-terminus (Figure S3B). While we do not observe density for the charged C-terminal tail in the structure (residues 780–797), well-defined density is present for the residues immediately preceding the tail (residues 750–779) in the three AMP-PNP bound subunits (ATP2–4) (Figure 3A). Unexpectedly, this ordered C-terminus extends upwards from the base of the protease domain along the exterior surface of the complex to interact with both the protease and the ATPase domains of the cis subunit, and the interdomain linker of the counterclockwise adjacent subunit. This arrangement of the C-terminus would position the highly charged tail at the membrane-proximal face of AFG3L2. Truncation of the C-terminus at residue 750 in the coreAFG3L2WB/PI construct, which is not stabilized by the hexamerizing coiled coil, decreased recovery of AFG3L2 hexamers as measured by size exclusion chromatography (SEC; Figure S4A). In contrast, removal of the unobserved charged tail alone (residues 781–797) did not impact complex recovery. This indicates that the interaction of the C-terminal residues 750–779 across neighboring ATPase domains is important for stabilization and assembly of the active hexamer. Consistent with this, deletion of residues 750–797 in cchexAFG3L2 reduced both ATPase activity and substrate degradation, whereas deletion of residues 781–797 had no significant effect on either activity. The final C-terminal residue for which we observe well-ordered density is W779, which is sandwiched between M321 and K328 at the membrane-proximal surface of the ATPase domain (Figure 3C). These stabilizing sulfur-aromatic and cation-π interactions likely secure W779 in position, anchoring the base of the charged C-terminal tail for potential substrate interaction. In support of this, incorporating a W779R substitution into cchexAFG3L2 impaired substrate degradation without impacting ATP hydrolysis (Figure 3D). This reveals an important role for the C-terminus in dictating complex stability, and shows that interactions driven by the organization of C-terminal residues influence important aspects of AFG3L2 substrate processing.

The AFG3L2 C-terminus encircles the hexamer to adopt a membrane proximal position

Our construct contained the complete AFG3L2 C-terminus, comprising residues 750–797 – a region that is highly variable among FtsH-related AAA+ proteases and, in AFG3L2, contains a number of charged residues at the far C-terminus (Figure S3B). While we do not observe density for the charged C-terminal tail in the structure (residues 780–797), well-defined density is present for the residues immediately preceding the tail (residues 750–779) in the three AMP-PNP bound subunits (ATP2–4) (Figure 3A). Unexpectedly, this ordered C-terminus extends upwards from the base of the protease domain along the exterior surface of the complex to interact with both the protease and the ATPase domains of the cis subunit, and the interdomain linker of the counterclockwise adjacent subunit. This arrangement of the C-terminus would position the highly charged tail at the membrane-proximal face of AFG3L2. Truncation of the C-terminus at residue 750 in the coreAFG3L2WB/PI construct, which is not stabilized by the hexamerizing coiled coil, decreased recovery of AFG3L2 hexamers as measured by size exclusion chromatography (SEC; Figure S4A). In contrast, removal of the unobserved charged tail alone (residues 781–797) did not impact complex recovery. This indicates that the interaction of the C-terminal residues 750–779 across neighboring ATPase domains is important for stabilization and assembly of the active hexamer. Consistent with this, deletion of residues 750–797 in cchexAFG3L2 reduced both ATPase activity and substrate degradation, whereas deletion of residues 781–797 had no significant effect on either activity. The final C-terminal residue for which we observe well-ordered density is W779, which is sandwiched between M321 and K328 at the membrane-proximal surface of the ATPase domain (Figure 3C). These stabilizing sulfur-aromatic and cation-π interactions likely secure W779 in position, anchoring the base of the charged C-terminal tail for potential substrate interaction. In support of this, incorporating a W779R substitution into cchexAFG3L2 impaired substrate degradation without impacting ATP hydrolysis (Figure 3D). This reveals an important role for the C-terminus in dictating complex stability, and shows that interactions driven by the organization of C-terminal residues influence important aspects of AFG3L2 substrate processing.

Disease-relevant AFG3L2 mutations localize to four ‘hotspots’ on the AFG3L2 structureWe next sought to evaluate how specific disease-relevant AFG3L2 mutations impact its activity. Mapping disease-relevant AFG3L2 mutations onto our structure reveals that all mutations localize to four ‘hotspots’ within the hexameric complex (Figure 1C–1F). Intriguingly, all of the autosomal dominant mutations localize to intersubunit interfaces of the hexamer, including 1) the lateral interface between adjacent ATPase subunits where the nucleotide binding pocket is formed, 2) the tightly interconnected helices that dominate the lateral interface of the protease domain, and 3) the central protrusion of the AFG3L2 protease ring that is involved in substrate transfer to the protease domains. In contrast, recessive mutations localize near the protease active sites in the periphery of the protease ring. The clustering of these mutations into distinct regions in the structure suggests distinct effects on AFG3L2 activity.

Disease-relevant AFG3L2 mutations localize to four ‘hotspots’ on the AFG3L2 structure

We next sought to evaluate how specific disease-relevant AFG3L2 mutations impact its activity. Mapping disease-relevant AFG3L2 mutations onto our structure reveals that all mutations localize to four ‘hotspots’ within the hexameric complex (Figure 1C–1F). Intriguingly, all of the autosomal dominant mutations localize to intersubunit interfaces of the hexamer, including 1) the lateral interface between adjacent ATPase subunits where the nucleotide binding pocket is formed, 2) the tightly interconnected helices that dominate the lateral interface of the protease domain, and 3) the central protrusion of the AFG3L2 protease ring that is involved in substrate transfer to the protease domains. In contrast, recessive mutations localize near the protease active sites in the periphery of the protease ring. The clustering of these mutations into distinct regions in the structure suggests distinct effects on AFG3L2 activity.

Disease mutations in the AAA+ domains disrupt nucleotide-dependent substrate translocationTwo disease-relevant mutations, N432T (linked to SCA28) and R468C (DOA), localize to the vicinity of the nucleotide-binding pocket at the ATPase intersubunit interface (Charif et al., 2015; Colavito et al., 2017; Di Bella et al., 2010). Our structure shows that R468 is one of two Arg-finger residues that projects from the adjacent large ATPase subdomain to coordinate interactions with the γ-phosphate (Figure 3C and S2B). These Arg-finger residues are highly conserved across classical AAA+ enzymes and are suggested to play roles in ATP binding and/or intersubunit interactions (Wendler et al., 2012). In fact, mutation of analogous Arg-fingers in bacterial FtsH obliterated activity (Karata et al., 1999). Accordingly, the R468C patient mutation abolishes ATP hydrolysis and substrate degradation when incorporated into cchexAFG3L2 (Figure 2B). Consistent with impaired ATP binding at the intersubunit interface, R468C decreased recovery of AFG3L2 hexamers by SEC, mirroring the behavior of an ATP-binding incompetent Walker A control (K354A, denoted AFG3L2WA) (Figure S4B).Previously, homology models of FtsH were used to predict that N432T might impair substrate engagement in the central pore (Di Bella et al., 2010). Our structure shows that N432 interacts with R416 within pore-loop 2, positioning this loop to facilitate substrate engagement (Figure 2A). Strict conservation of both N432 and R416 within the FtsH family implies that this interaction exists in other family members (Figure S3A). Both R416A and disease-relevant N432T substitutions abolished substrate degradation in cchexAFG3L2, indicating a key functional role for this interaction. Surprisingly, these two mutations also significantly reduced ATPase activity. N432 is located within helix α5 immediately adjacent to the ISS motif, suggesting that this mutation might disrupt allosteric coordination of ATP hydrolysis. In agreement, N432T displayed a significantly reduced maximal steady-state ATPase rate but retained high affinity for ATP, indicating that this mutation impairs catalysis without impacting nucleotide binding (Figure S4F). Thus, our structure-function studies indicate that the disease-relevant N432T mutation allosterically affects both the ATP hydrolysis cycle and organization of pore-loop 2.

Disease mutations in the AAA+ domains disrupt nucleotide-dependent substrate translocation

Two disease-relevant mutations, N432T (linked to SCA28) and R468C (DOA), localize to the vicinity of the nucleotide-binding pocket at the ATPase intersubunit interface (Charif et al., 2015; Colavito et al., 2017; Di Bella et al., 2010). Our structure shows that R468 is one of two Arg-finger residues that projects from the adjacent large ATPase subdomain to coordinate interactions with the γ-phosphate (Figure 3C and S2B). These Arg-finger residues are highly conserved across classical AAA+ enzymes and are suggested to play roles in ATP binding and/or intersubunit interactions (Wendler et al., 2012). In fact, mutation of analogous Arg-fingers in bacterial FtsH obliterated activity (Karata et al., 1999). Accordingly, the R468C patient mutation abolishes ATP hydrolysis and substrate degradation when incorporated into cchexAFG3L2 (Figure 2B). Consistent with impaired ATP binding at the intersubunit interface, R468C decreased recovery of AFG3L2 hexamers by SEC, mirroring the behavior of an ATP-binding incompetent Walker A control (K354A, denoted AFG3L2WA) (Figure S4B).

Previously, homology models of FtsH were used to predict that N432T might impair substrate engagement in the central pore (Di Bella et al., 2010). Our structure shows that N432 interacts with R416 within pore-loop 2, positioning this loop to facilitate substrate engagement (Figure 2A). Strict conservation of both N432 and R416 within the FtsH family implies that this interaction exists in other family members (Figure S3A). Both R416A and disease-relevant N432T substitutions abolished substrate degradation in cchexAFG3L2, indicating a key functional role for this interaction. Surprisingly, these two mutations also significantly reduced ATPase activity. N432 is located within helix α5 immediately adjacent to the ISS motif, suggesting that this mutation might disrupt allosteric coordination of ATP hydrolysis. In agreement, N432T displayed a significantly reduced maximal steady-state ATPase rate but retained high affinity for ATP, indicating that this mutation impairs catalysis without impacting nucleotide binding (Figure S4F). Thus, our structure-function studies indicate that the disease-relevant N432T mutation allosterically affects both the ATP hydrolysis cycle and organization of pore-loop 2.

Disease mutations in the protease domain concentrate near the central protrusionWith the exception of N432T, all SCA28-associated mutations identified to date localize to two hotspots within, or at the base of, the central protease loops. These loops form a channel-like protrusion at the center of the proteolytic ring that interacts with pore-loop 2 and the translocating substrate (Figure 4A–4D). In each subunit, the base of the central protease loop is stabilized by π-stacking intrasubunit interactions between Y689 and F675 (Figure 4C). Intriguingly, layered rings alternating between acidic (E686 and E691) and basic (K687 and R695) residues create a highly charged channel surface (Figure 4A and 4B).All of the mutations in this region line the channel of the central protrusion and likely impact the integrity of its structure. E691K swaps the charge of one of the acidic rings, which would disrupt the charged ring system within the channel (Figure 4B). Similarly, A694E positions a negatively charged residue adjacent to E691, introducing charge repulsion between these residues (Figure 4B). At the base of the central protease loop, substitutions of Y689 (Y689N and Y689H) would eliminate the π-stacking interactions that maintain loop integrity (Figure 4C). Finally, P688 introduces a sharp turn at the base of the loop that correctly positions Y689 and K687 for interaction with other residues in this region (Figure 4C). The P688T substitution will likely add flexibility at this position and decrease the stability of the central protrusion.In contrast to the central protease loop, residues at the base of the central protrusion are highly conserved across FtsH-related proteases, and our structure indicates that the majority of these residues are involved in intersubunit interactions that stabilize the proteolytic ring. Notably, all remaining SCA28-associated residues are located in this region, including one of the most highly mutated residues in disease, M666, which packs tightly into a hydrophobic pocket at the intersubunit interface (Figure 4D). Mutation of M666 to Thr or Val would perturb this hydrophobic pocket. M666R, which would disrupt this pocket by introducing a positive charge, results in one of the most severe disease phenotypes in patients. R702Q and G671R/E would destabilize a charged cluster at the base of the central protrusion (Figure 4D). Similarly, E700K would disrupt ionic interactions with K669 at the base of the central protrusion in the same subunit (Figure 4D). T654I and S674L would introduce bulky hydrophobic residues into a closely packed intersubunit interface (Figure 4D). Based on these structural observations, we predict that all SCA28-associated mutations localized within or at the base of the central protrusion destabilize the protease, thereby impacting AFG3L2 function. A collection of residues previously identified as important for assembly of the heterohexameric yeast m-AAA protease also locate to the central protrusion, further highlighting the importance of this feature in complex stability (Lee et al., 2011).Incorporating these protease mutations into coreAFG3L32WB/PI significantly impaired recovery of the hexameric protein by SEC, albeit to different extents (Figure S4C and S4D). For example, M666R, which disrupts the hydrophobic interface between protease subunits, completely abolished hexamer recovery, while P688T, which increases flexibility at the base of the protrusion, only modestly affected stability (Figure S4C and S4D). This global destabilization of the catalytic core was further evident when we incorporated protease mutations into cchexAFG3L2. Despite hexamerization, all protease mutations show reductions in both ATP hydrolysis and substrate degradation that reflect the degree of destabilization observed by SEC (Figure 5A). Moreover, reductions of activity in vitro appear to correlate with disease severity in patients. For instance, P688T presents only a modest reduction in activity, and is associated with the mildest condition in SCA28 patients, whereas M666R shows a near complete loss of activity and is associated with a very severe, early onset phenotype in patients (Di Bella et al., 2010; Svenstrup et al., 2017).To further define the impact of these mutations on AFG3L2 stability, we incorporated the mild mutation P688T and the severe mutations M666R and E691K into C-terminally Flag-tagged AFG3L2 (AFG3L2FT) and monitored assembly and stability by Blue-Native (BN-PAGE) and SDS-PAGE. By BN-PAGE, P688T showed a small increase in AFG3L2 oligomer migration and a small increase in AFG3L2 monomers, indicating modest destabilization (Figure 5B). In contrast, M666R and E691K show significant streaking of the AFG3L2 oligomer band (Figure 5B), consistent with the more severe destabilization of AFG3L2 observed in vitro (Figure S4C and S4D). The varying levels of AFG3L2 destabilization afforded by these mutations appears to be in part mediated through proteolytic cleavage of the protomer. Using antibodies that recognize the N- or C-terminus of AFG3L2FT, we observed the accumulation of an N-terminal cleavage product of ~50 kDa in SDS-PAGE that corresponds to loss of the C-terminal protease domain for all three of these mutants (Figure 5C). P688T leads to a more modest accumulation of this cleavage product, consistent with the modest effect on oligomer stability (Figures 5C and S4C) and activity (Figure 5A). However, AFG3L2 harboring the M666R or E691K mutation showed significant accumulation of this cleavage product (Figure 5C), reflecting marked destabilization (Figures 5B, S4C, and S4D) and severe reduction in activity (Figure 5A). These results further support a model whereby mutations in the protease domain destabilize the protease ring, leading to SCA28, with the extent of destabilization being an important factor for disease severity and age of onset in patients (Table S1).Analyzing the ATPase mutants N432T and R468C using the same approach produced starkly contrasting results (Figure 5B and 5C). To determine whether their distinct profiles are related to impairment of the ATP hydrolysis cycle, we expressed control mutants Walker A (prevents ATP binding) and Walker B (prevents ATP hydrolysis). As in our SEC experiments, the Walker B mutant increased stability of the AFG3L2 hexamer, whereas appearance of AFG3L2 monomers in the Walker A mutant indicates disassembly of the complex. In agreement with our in vitro data, R468C closely mirrors the behavior of the Walker A mutant, potentially reflecting the unique DOA pathology observed in patients (Figure 5B and 5C). While the SDS-PAGE profile of N432T is similar to that of the Walker B mutant, N432T causes a mild destabilization of AFG3L2 oligomers by BN-PAGE (Figure 5B). Our biochemical and structural data indicate that N432T may allosterically impact both ATP hydrolysis and pore-loop organization (Figure 2). Thus, the decreased stability observed in cells might be due to impairment of substrate binding, providing a potential explanation for the dominant negative effect seen for this mutation (Di Bella et al., 2010).

Disease mutations in the protease domain concentrate near the central protrusion

With the exception of N432T, all SCA28-associated mutations identified to date localize to two hotspots within, or at the base of, the central protease loops. These loops form a channel-like protrusion at the center of the proteolytic ring that interacts with pore-loop 2 and the translocating substrate (Figure 4A–4D). In each subunit, the base of the central protease loop is stabilized by π-stacking intrasubunit interactions between Y689 and F675 (Figure 4C). Intriguingly, layered rings alternating between acidic (E686 and E691) and basic (K687 and R695) residues create a highly charged channel surface (Figure 4A and 4B).

All of the mutations in this region line the channel of the central protrusion and likely impact the integrity of its structure. E691K swaps the charge of one of the acidic rings, which would disrupt the charged ring system within the channel (Figure 4B). Similarly, A694E positions a negatively charged residue adjacent to E691, introducing charge repulsion between these residues (Figure 4B). At the base of the central protease loop, substitutions of Y689 (Y689N and Y689H) would eliminate the π-stacking interactions that maintain loop integrity (Figure 4C). Finally, P688 introduces a sharp turn at the base of the loop that correctly positions Y689 and K687 for interaction with other residues in this region (Figure 4C). The P688T substitution will likely add flexibility at this position and decrease the stability of the central protrusion.

In contrast to the central protease loop, residues at the base of the central protrusion are highly conserved across FtsH-related proteases, and our structure indicates that the majority of these residues are involved in intersubunit interactions that stabilize the proteolytic ring. Notably, all remaining SCA28-associated residues are located in this region, including one of the most highly mutated residues in disease, M666, which packs tightly into a hydrophobic pocket at the intersubunit interface (Figure 4D). Mutation of M666 to Thr or Val would perturb this hydrophobic pocket. M666R, which would disrupt this pocket by introducing a positive charge, results in one of the most severe disease phenotypes in patients. R702Q and G671R/E would destabilize a charged cluster at the base of the central protrusion (Figure 4D). Similarly, E700K would disrupt ionic interactions with K669 at the base of the central protrusion in the same subunit (Figure 4D). T654I and S674L would introduce bulky hydrophobic residues into a closely packed intersubunit interface (Figure 4D). Based on these structural observations, we predict that all SCA28-associated mutations localized within or at the base of the central protrusion destabilize the protease, thereby impacting AFG3L2 function. A collection of residues previously identified as important for assembly of the heterohexameric yeast m-AAA protease also locate to the central protrusion, further highlighting the importance of this feature in complex stability (Lee et al., 2011).

Incorporating these protease mutations into coreAFG3L32WB/PI significantly impaired recovery of the hexameric protein by SEC, albeit to different extents (Figure S4C and S4D). For example, M666R, which disrupts the hydrophobic interface between protease subunits, completely abolished hexamer recovery, while P688T, which increases flexibility at the base of the protrusion, only modestly affected stability (Figure S4C and S4D). This global destabilization of the catalytic core was further evident when we incorporated protease mutations into cchexAFG3L2. Despite hexamerization, all protease mutations show reductions in both ATP hydrolysis and substrate degradation that reflect the degree of destabilization observed by SEC (Figure 5A). Moreover, reductions of activity in vitro appear to correlate with disease severity in patients. For instance, P688T presents only a modest reduction in activity, and is associated with the mildest condition in SCA28 patients, whereas M666R shows a near complete loss of activity and is associated with a very severe, early onset phenotype in patients (Di Bella et al., 2010; Svenstrup et al., 2017).

To further define the impact of these mutations on AFG3L2 stability, we incorporated the mild mutation P688T and the severe mutations M666R and E691K into C-terminally Flag-tagged AFG3L2 (AFG3L2FT) and monitored assembly and stability by Blue-Native (BN-PAGE) and SDS-PAGE. By BN-PAGE, P688T showed a small increase in AFG3L2 oligomer migration and a small increase in AFG3L2 monomers, indicating modest destabilization (Figure 5B). In contrast, M666R and E691K show significant streaking of the AFG3L2 oligomer band (Figure 5B), consistent with the more severe destabilization of AFG3L2 observed in vitro (Figure S4C and S4D). The varying levels of AFG3L2 destabilization afforded by these mutations appears to be in part mediated through proteolytic cleavage of the protomer. Using antibodies that recognize the N- or C-terminus of AFG3L2FT, we observed the accumulation of an N-terminal cleavage product of ~50 kDa in SDS-PAGE that corresponds to loss of the C-terminal protease domain for all three of these mutants (Figure 5C). P688T leads to a more modest accumulation of this cleavage product, consistent with the modest effect on oligomer stability (Figures 5C and S4C) and activity (Figure 5A). However, AFG3L2 harboring the M666R or E691K mutation showed significant accumulation of this cleavage product (Figure 5C), reflecting marked destabilization (Figures 5B, S4C, and S4D) and severe reduction in activity (Figure 5A). These results further support a model whereby mutations in the protease domain destabilize the protease ring, leading to SCA28, with the extent of destabilization being an important factor for disease severity and age of onset in patients (Table S1).

Analyzing the ATPase mutants N432T and R468C using the same approach produced starkly contrasting results (Figure 5B and 5C). To determine whether their distinct profiles are related to impairment of the ATP hydrolysis cycle, we expressed control mutants Walker A (prevents ATP binding) and Walker B (prevents ATP hydrolysis). As in our SEC experiments, the Walker B mutant increased stability of the AFG3L2 hexamer, whereas appearance of AFG3L2 monomers in the Walker A mutant indicates disassembly of the complex. In agreement with our in vitro data, R468C closely mirrors the behavior of the Walker A mutant, potentially reflecting the unique DOA pathology observed in patients (Figure 5B and 5C). While the SDS-PAGE profile of N432T is similar to that of the Walker B mutant, N432T causes a mild destabilization of AFG3L2 oligomers by BN-PAGE (Figure 5B). Our biochemical and structural data indicate that N432T may allosterically impact both ATP hydrolysis and pore-loop organization (Figure 2). Thus, the decreased stability observed in cells might be due to impairment of substrate binding, providing a potential explanation for the dominant negative effect seen for this mutation (Di Bella et al., 2010).

Recessively inherited mutations are involved in substrate cleavage at the proteolytic active siteIn contrast to the SCA28 and DOA mutations, two single-point mutations in the protease domain of AFG3L2 are linked to severe neurodegenerative disorders with distinct phenotypes only in homozygotes (Y616C, SPAX5; A572T, as yet unnamed neurodegenerative disease) (Eskandrani et al., 2017; Pierson et al., 2011). We found that both mutations are in close proximity to the protease active site (Figure 6A). In AFG3L2, six zinc-metalloprotease active sites are sequestered to the interior periphery of the proteolytic ring (Figure 6A). At each site, a zinc ion is coordinated for catalysis by H574 and H578 from helix α10 and D649 in helix α12 (Figure 6A). The His residues are part of a strictly conserved HEXXH motif that is required for proteolytic activity. Next to this motif is the conserved residue A572, which is directed towards a hydrophobic pocket on the opposite side of α10 from the active site (Figures 6A and S5A). The disease mutation A572T would introduce a polar side chain into this pocket, perturbing the local structure of the active site. In agreement, cchexAFG3L2 containing the A572T mutation shows significantly decreased protein degradation activity and a moderate defect in ATP hydrolysis (Figure 6B). Moreover, introduction of A572T to coreAFG3L2WB/PI did not hamper hexamer recovery by SEC, indicating local structural defects rather than global destabilization (Figure S4E).To confirm that A572T directly affects proteolytic activity, we used an ATPase-inactive/peptidase-active variant of the coreAFG3L2 protein (coreAFG3L2WB) to measure the ATP-independent cleavage of a small fluorogenic peptide (Ding et al., 2018). Mutations in the N-terminus (coreAFG3L2WB/L299A) and central protrusion (coreAFG3L2WB/M683A), both of which lower substrate degradation rates in the ATPase active context (Figures 2 and 3), did not alter the peptide cleavage rate, indicating that this assay reports solely on proteolytic activity independent of upstream mechanistic defects (Figure 6C). A572T reduced peptide cleavage rates by 50%, demonstrating that this mutation directly impairs proteolysis at the active site (Figure 6C).Our structure of catalytically inactive AFG3L2 contained density extending across three of the six protease active sites, which we attribute to substrate peptide that is positioned for cleavage (Figure 6A). This substrate appears to form an additional β-strand next to β7, positioning the polypeptide above the catalytic triad. AFG3L2 preferentially cleaves sequences containing Phe residues in the P1’ position, immediately C-terminal of the scissile bond (Ding et al., 2018). The position of the substrate in our structure locates the putative specificity site (S1’) to a hydrophobic pocket created by V571, L603 and G645 from a single subunit, and L615 from the adjacent subunit (Figure S5B). Although no density is observed for the side chain of the substrate residue closest to this position, a Phe side chain can be easily accommodated in this pocket. L615 is part of a non-conserved loop (Y614, L615 and Y616) at the intersubunit interface, and density for the substrate backbone is seen positioned between the side chains of Y614 and the SPAX5-related residue Y616 (Figures 6A and S5B). Strikingly, the disease mutation Y616C displayed increased ATPase and protein degradation rates, with a modest reduction in complex stability (Figure 6B and S4E). Moreover, ATP-independent peptidase activity was two-fold higher for Y616C compared to controls (Figure 6C). These results suggest that substitution of Y616 with a smaller Cys may reduce steric hindrance of incoming polypeptides, increasing accessibility to the S1’ pocket. The distinct gain-of-function effect of Y616C provides a plausible explanation for the unique phenotype of this mutation in patients.

Recessively inherited mutations are involved in substrate cleavage at the proteolytic active site

In contrast to the SCA28 and DOA mutations, two single-point mutations in the protease domain of AFG3L2 are linked to severe neurodegenerative disorders with distinct phenotypes only in homozygotes (Y616C, SPAX5; A572T, as yet unnamed neurodegenerative disease) (Eskandrani et al., 2017; Pierson et al., 2011). We found that both mutations are in close proximity to the protease active site (Figure 6A). In AFG3L2, six zinc-metalloprotease active sites are sequestered to the interior periphery of the proteolytic ring (Figure 6A). At each site, a zinc ion is coordinated for catalysis by H574 and H578 from helix α10 and D649 in helix α12 (Figure 6A). The His residues are part of a strictly conserved HEXXH motif that is required for proteolytic activity. Next to this motif is the conserved residue A572, which is directed towards a hydrophobic pocket on the opposite side of α10 from the active site (Figures 6A and S5A). The disease mutation A572T would introduce a polar side chain into this pocket, perturbing the local structure of the active site. In agreement, cchexAFG3L2 containing the A572T mutation shows significantly decreased protein degradation activity and a moderate defect in ATP hydrolysis (Figure 6B). Moreover, introduction of A572T to coreAFG3L2WB/PI did not hamper hexamer recovery by SEC, indicating local structural defects rather than global destabilization (Figure S4E).

To confirm that A572T directly affects proteolytic activity, we used an ATPase-inactive/peptidase-active variant of the coreAFG3L2 protein (coreAFG3L2WB) to measure the ATP-independent cleavage of a small fluorogenic peptide (Ding et al., 2018). Mutations in the N-terminus (coreAFG3L2WB/L299A) and central protrusion (coreAFG3L2WB/M683A), both of which lower substrate degradation rates in the ATPase active context (Figures 2 and 3), did not alter the peptide cleavage rate, indicating that this assay reports solely on proteolytic activity independent of upstream mechanistic defects (Figure 6C). A572T reduced peptide cleavage rates by 50%, demonstrating that this mutation directly impairs proteolysis at the active site (Figure 6C).

Our structure of catalytically inactive AFG3L2 contained density extending across three of the six protease active sites, which we attribute to substrate peptide that is positioned for cleavage (Figure 6A). This substrate appears to form an additional β-strand next to β7, positioning the polypeptide above the catalytic triad. AFG3L2 preferentially cleaves sequences containing Phe residues in the P1’ position, immediately C-terminal of the scissile bond (Ding et al., 2018). The position of the substrate in our structure locates the putative specificity site (S1’) to a hydrophobic pocket created by V571, L603 and G645 from a single subunit, and L615 from the adjacent subunit (Figure S5B). Although no density is observed for the side chain of the substrate residue closest to this position, a Phe side chain can be easily accommodated in this pocket. L615 is part of a non-conserved loop (Y614, L615 and Y616) at the intersubunit interface, and density for the substrate backbone is seen positioned between the side chains of Y614 and the SPAX5-related residue Y616 (Figures 6A and S5B). Strikingly, the disease mutation Y616C displayed increased ATPase and protein degradation rates, with a modest reduction in complex stability (Figure 6B and S4E). Moreover, ATP-independent peptidase activity was two-fold higher for Y616C compared to controls (Figure 6C). These results suggest that substitution of Y616 with a smaller Cys may reduce steric hindrance of incoming polypeptides, increasing accessibility to the S1’ pocket. The distinct gain-of-function effect of Y616C provides a plausible explanation for the unique phenotype of this mutation in patients.

DiscussionA hand-over-hand conveyance of substrate along the central pore of a spiraling ATPase ring is emerging as the conserved mechanism that enables ATP-driven substrate translocation across much of the AAA+ superfamily. However, a fundamental question remains unanswered: “What are the unique evolutionary adaptations that enable different AAA+ proteins to process their distinct substrates?” Our structure of AFG3L2 with substrate bound in both the ATPase and protease domains reveals how an intricate network within the AFG3L2 hexamer coordinates ATP-driven substrate engagement, translocation, and transfer across the degradation chamber for proteolysis. We show that unique structural features integrate with a core nucleotide-dependent mechanism to drive substrate processing, and the functional relevance of these distinctive features is underscored by our finding that disease-linked mutations are concentrated at these substrate-interacting, non-conserved regions. We further show how disease-relevant substitutions differentially impact AFG3L2 activity and stability, providing a molecular basis for AFG3L2-associated neurodegenerative conditions.An important first step for AAA+ activity is the recognition of the appropriate substrates for processing. While the C-termini of FtsH-related AAA+ proteases have been shown to modulate substrate specificity, the mechanisms responsible for this modulation are unknown (Akiyama, 1999). Our structure reveals that the C-terminus of AFG3L2 establishes interdomain interactions between neighboring subunits that stabilize the hexamer, while simultaneously positioning a highly charged tail for interactions at the membrane surface (Figure 7A). This organization is required for AFG3L2 activity, and is in contrast to the organization of the C-terminus in YME1, which we previously observed to be an unstructured loop extending away from the protease ring into the IMS (Figure 7B). Thus, we speculate that distinct C-termini evolved in m- and i-AAA proteases, enabling charge-driven interactions for recruitment of membrane-embedded substrates in AFG3L2, and mediating recognition of soluble IMS substrates in YME1. In line with this hypothesis, alteration of the C-terminus of YME1 affects processing of soluble substrates of YME1, but not membrane-embedded substrates (Graef et al., 2007).In the central pore of the ATPase spiral observed in many AAA+ ATPases, the conserved aromatic residue in pore-loop 1 forms a staircase around the substrate that drives translocation. However, in AFG3L2 the residue immediately adjacent to the pore-loop 1 aromatic links consecutive pore loops to form a tightly packed and continuous chain of residues that surround the substrate. Similarly, both the N-termini of the ATPases and pore-loop 2 are not conserved, and form additional spiral staircases that contact the substrate. Integration of these unique features with the core machinery for nucleotide-driven allosteric changes allows all elements to cooperate in pulling on substrate as ATP is hydrolyzed (Figure 7A). Additionally, the extended pore-loop 2 of AFG3L2 mediates transfer of the substrate to the central protrusion of the protease. These ancillary substrate-interacting elements are not present in YME1, where the translocating substrate enters a large, unoccupied degradation chamber (Figure 7B). Thus, our data demonstrate how the cumulative effect of substitutions and/or small insertions around the core elements alters how substrate is handled in different AAA+ proteins (Figure 7B). These changes are likely to affect not only substrate specificity, but also fundamental enzymatic characteristics such as grip on substrate and unfoldase efficacy.AAA+ proteases are required to degrade substrates with highly diverse sequences. However, evidence exists for site-specific cleavage for some substrates. The presence of substrate in the protease active site in our AFG3L2 structure offers insights into this enzymatic “decision”. We observe that the substrate is stabilized in the active site by backbone-mediated interactions compatible with sequence-independent cleavage for broad house-keeping activity. However, our structure also reveals a putative specificity pocket (S1’) that is capable of accommodating an aromatic residue, consistent with the moderate degree of preference for Phe immediately C-terminal of the scissile bond (P1’) (Ding et al., 2018). Moreover, the specificity pocket involves residues from the adjacent subunit, suggesting that heterohexameric m-AAA proteases may display altered substrate specificity. These results will guide future studies to determine how substrate interactions in the proteolytic active site might influence sequence preference in different AAA+ proteases, and promote site-specific processing of substrates.Importantly, how different mutations in AFG3L2 lead to distinct neurological phenotypes is an important, open question in the field. Here, we show that mutations associated with different diseases influence distinct aspects of AFG3L2 stability and activity. Interestingly, the distinct biochemical impact of different mutations appears to correlate with unique phenotypes and inheritance patterns, suggesting that these differences might constitute the molecular basis for the different neurodegenerative conditions associated with AFG3L2 in patients (Table S1). While the functional impact of these different AFG3L2 mutations on mitochondrial proteostasis and function remain to be established, our work provides a molecular basis to begin to understand the distinct structure-phenotype relationship observed in patients harboring different mutations in this critical AAA+ protease.

A hand-over-hand conveyance of substrate along the central pore of a spiraling ATPase ring is emerging as the conserved mechanism that enables ATP-driven substrate translocation across much of the AAA+ superfamily. However, a fundamental question remains unanswered: “What are the unique evolutionary adaptations that enable different AAA+ proteins to process their distinct substrates?” Our structure of AFG3L2 with substrate bound in both the ATPase and protease domains reveals how an intricate network within the AFG3L2 hexamer coordinates ATP-driven substrate engagement, translocation, and transfer across the degradation chamber for proteolysis. We show that unique structural features integrate with a core nucleotide-dependent mechanism to drive substrate processing, and the functional relevance of these distinctive features is underscored by our finding that disease-linked mutations are concentrated at these substrate-interacting, non-conserved regions. We further show how disease-relevant substitutions differentially impact AFG3L2 activity and stability, providing a molecular basis for AFG3L2-associated neurodegenerative conditions.

An important first step for AAA+ activity is the recognition of the appropriate substrates for processing. While the C-termini of FtsH-related AAA+ proteases have been shown to modulate substrate specificity, the mechanisms responsible for this modulation are unknown (Akiyama, 1999). Our structure reveals that the C-terminus of AFG3L2 establishes interdomain interactions between neighboring subunits that stabilize the hexamer, while simultaneously positioning a highly charged tail for interactions at the membrane surface (Figure 7A). This organization is required for AFG3L2 activity, and is in contrast to the organization of the C-terminus in YME1, which we previously observed to be an unstructured loop extending away from the protease ring into the IMS (Figure 7B). Thus, we speculate that distinct C-termini evolved in m- and i-AAA proteases, enabling charge-driven interactions for recruitment of membrane-embedded substrates in AFG3L2, and mediating recognition of soluble IMS substrates in YME1. In line with this hypothesis, alteration of the C-terminus of YME1 affects processing of soluble substrates of YME1, but not membrane-embedded substrates (Graef et al., 2007).

In the central pore of the ATPase spiral observed in many AAA+ ATPases, the conserved aromatic residue in pore-loop 1 forms a staircase around the substrate that drives translocation. However, in AFG3L2 the residue immediately adjacent to the pore-loop 1 aromatic links consecutive pore loops to form a tightly packed and continuous chain of residues that surround the substrate. Similarly, both the N-termini of the ATPases and pore-loop 2 are not conserved, and form additional spiral staircases that contact the substrate. Integration of these unique features with the core machinery for nucleotide-driven allosteric changes allows all elements to cooperate in pulling on substrate as ATP is hydrolyzed (Figure 7A). Additionally, the extended pore-loop 2 of AFG3L2 mediates transfer of the substrate to the central protrusion of the protease. These ancillary substrate-interacting elements are not present in YME1, where the translocating substrate enters a large, unoccupied degradation chamber (Figure 7B). Thus, our data demonstrate how the cumulative effect of substitutions and/or small insertions around the core elements alters how substrate is handled in different AAA+ proteins (Figure 7B). These changes are likely to affect not only substrate specificity, but also fundamental enzymatic characteristics such as grip on substrate and unfoldase efficacy.

AAA+ proteases are required to degrade substrates with highly diverse sequences. However, evidence exists for site-specific cleavage for some substrates. The presence of substrate in the protease active site in our AFG3L2 structure offers insights into this enzymatic “decision”. We observe that the substrate is stabilized in the active site by backbone-mediated interactions compatible with sequence-independent cleavage for broad house-keeping activity. However, our structure also reveals a putative specificity pocket (S1’) that is capable of accommodating an aromatic residue, consistent with the moderate degree of preference for Phe immediately C-terminal of the scissile bond (P1’) (Ding et al., 2018). Moreover, the specificity pocket involves residues from the adjacent subunit, suggesting that heterohexameric m-AAA proteases may display altered substrate specificity. These results will guide future studies to determine how substrate interactions in the proteolytic active site might influence sequence preference in different AAA+ proteases, and promote site-specific processing of substrates.

Importantly, how different mutations in AFG3L2 lead to distinct neurological phenotypes is an important, open question in the field. Here, we show that mutations associated with different diseases influence distinct aspects of AFG3L2 stability and activity. Interestingly, the distinct biochemical impact of different mutations appears to correlate with unique phenotypes and inheritance patterns, suggesting that these differences might constitute the molecular basis for the different neurodegenerative conditions associated with AFG3L2 in patients (Table S1). While the functional impact of these different AFG3L2 mutations on mitochondrial proteostasis and function remain to be established, our work provides a molecular basis to begin to understand the distinct structure-phenotype relationship observed in patients harboring different mutations in this critical AAA+ protease.

STAR METHODSCONTACT FOR REAGENT AND RESOURCE SHARINGFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Steven E. Glynn (steven.glynn@stonybrook.edu).EXPERIMENTAL MODEL AND SUBJECT DETAILSMicrobe StrainsEscherichia Coli DH5α were grown in LB broth supplemented with 100 µg/ml ampicillin at 37 °C. Escherichia Coli BL21-CodonPlus (DE3)-RIL were grown in LB broth supplemented with 100 µg/ml ampicillin and 34 µg/ml chloramphenicol at 37 °C.Cell LinesHuman embryonic kidney (HEK) 293T are female and cultured in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum (Gemini), 1% penicillin-streptomycin, 1% glutamine at 37°C and 5% CO 2.METHOD DETAILSCloning, Expression and PurificationAll molecular cloning was performed using E. Coli DH5-α Competent Cells (NEB) (Key Resources Table). Plasmids containing sequences encoding cchexAFG3L2 and coreAFG3L2 were constructed by subcloning a sequence encoding residues 272–797 of human AFG3L2 into a 2G-T vector and a modified 2G-T vector containing a cc-hex sequence respectively (Ding et al., 2018). Truncated AFG3L2 variants were constructed similarly using the primers listed in the Table S3. Mutant AFG3L2 constructs were created by site-directed mutagenesis using the primers listed in the Table S3. All AFG3L2 enzymes were expressed in E. coli BL21-CodonPlus (Key Resources Table) cells by induction at OD600~0.6 with 0.2 mM IPTG at 16 °C for 14 hours. For biochemical and biophysical characterization assays, cells containing AFG3L2 constructs were centrifuged at 4000 rpm for 30 minutes. Cell pellets containing cchexAFG3L2 or variants were resuspended in lysis buffer (20 mM Tris-HCl, pH 7.8, 300 mM NaCl, 10% Glycerol, 0.1 mM EDTA and 10 mM β-mercaptoethanol) supplemented with 10 mM PMSF, and lysed by sonication. Cell lysates were centrifuged at 14000 rpm for 30 minutes and the supernatants loaded to a Glutathione Superflow Agarose column (Pierce). Unbound proteins were washed off by addition of 10 column volumes of lysis buffer and bound proteins eluted by addition of lysis buffer supplemented with 10 mM reduced glutathione (Pierce). To remove the His6-GST tag, 0.5 mg TEV protease was added to the eluted proteins and the mixtures incubated at 4 °C for 16 hours. Cleaved cchexAFG3L2 proteins were then applied to a Ni-NTA column (Thermo Scientific) and collected in the flow-through. Proteins were concentrated and loaded onto a Superose 6 10/300 GL Increase size exclusion column (GE Healthcare) equilibrated with SEC buffer (20 mM Tris-HCl, pH 7.8, 100 mM NaCl, 10% Glycerol, 0.1 mM EDTA and 1 mM DTT). coreAFG3L2WB, coreAFG3L2WB/PI and their variants were purified using a similar protocol with the following modifications. Lysis buffer contained 50 mM HEPES-HCl, pH 7.5, 300 mM KCl, 0.1 mM EDTA, 10% glycerol and 10 mM β-mercaptoethanol, and SEC buffer contained 25 mM HEPES-HCl, pH 7.5, 100 mM KCl, 10% glycerol, and 1 mM DTT. Size exclusion chromatography was performed using a Superdex 200 Increase 10/300 GL Increase column (GE Healthcare). For cryo-EM structure determination, coreAFG3L2WB/PI was further purified by the following modifications to the protocol. Prior to size exclusion chromatography, coreAFG3L2WB/PI was exchanged into buffer containing 50 mM HEPES-HCl, pH 7.5, 150 mM KCl, 2 mM EDTA, 10% glycerol, and 10 mM β-mercaptoethanol. After buffer exchange, 15 mM MgCl2 and 1 mM AMP-PNP were added to the protein sample, and the mixture was incubated for 4 hours on ice. The concentrated protein sample was then applied to a Superdex 200 Increase 10/300 GL Increase column (GE Healthcare) equilibrated with buffer containing 25 mM HEPES-HCl, pH 7.5, 100 mM KCl, 20 µM AMP-PNP, 10 mM MgCl2, 10% glycerol and 1 mM DTT. Fractions corresponding to the calculated elution volume of related AFG3L2 hexamers were pooled, concentrated, flash-frozen, and stored at −80 °C.The SFGFP-10/11A226G-β20 model substrate was produced by inserting a sequence encoding the β20 degron into a plasmid encoding SFGFP-10/11A226G, which was a gift from Prof. Robert Sauer (MIT) (Iosefson et al., 2015). Proteins were expressed in E. coli BL21-CodonPlus cells by growth at 37 °C followed by addition of 1 mM IPTG and growth at 16 °C for 14 hours. Cells were centrifuged at 14,000 r pm for 30 minutes and resuspended in lysis buffer containing 20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, 10 mM imidazole, and 10 mM β-mercaptoethanol. The supernatants were applied to a Ni-NTA column (Thermo Scientific) followed by washing with buffer containing 20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, 50 mM imidazole, and 10 mM β-mercaptoethanol. Proteins were eluted by addition of elution buffer (20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, 250 mM imidazole, and 10 mM β-mercaptoethanol). Size exclusion chromatography was performed using a Superdex 200 Increase 10/300 GL Increase column (GE Healthcare) equilibrated with buffer containing 20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, and 1 mM DTT. The fractions corresponding to the correct protein were collected, concentrated, and flash frozen with liquid nitrogen for storage at −80 °C.Biochemical AssaysATPase activity, fluorescence-based protein degradation, and fluorogenic peptide cleavage assays were performed as previously described with some modifications (Ding et al., 2018; Rampello and Glynn, 2017). ATPase assays, as a coupled-enzyme assay monitoring the loss of NADH at 340 nm, were carried out at 37 °C in a 384-well clear bottom plate (Corning) using a SpectraMax M5 plate reader (Molecular Devices) with 1 µM cchexAFG3L2 or its variants. Reaction buffer contains 25 mM HEPES-KOH, pH 7.5, 5 mM MgCl2, 10% glycerol, 1 mM NADH, 21.5 U ml−1 lactate dehydrogenase and an ATP regeneration system (2 mM ATP, 7.5 mM phosphoenolpyruvate and 18.8 U ml−1 pyruvate kinase) (Key Resources Table). For fluorescence-based protein degradation assay, the reactions were carried out in a 384-well black plate (ex = 467 nm; em = 511 nm) at 37 ºC with 1 µM cchexAFG3L2 or its variants and 20 µM SFGFP-10/11A226G-β20. The initial time points with linear decrease in 340 nm signal were used to calculate the ATPase rates. Steady-state ATPase data were fit to the Hill version of the Michaelis−Menten equation [v = kATPase/(1 + K0.5/[ATP]n]. Fluorescence based protein degradation assays were performed at 37 °C in a 384-well black plate (Corning) using a SpectraMax M5 plate reader (ex = 467 nm; em = 511 nm) with 1 µM cchexAFG3L2 or its variants and 20 µM SFGFP-10/11A226G-β20 in the buffer containing 25 mM HEPES-KOH, pH 7.5, 100 mM KCl, 5 mM MgCl2, 10% glycerol, 1 mM DTT, 2 mM ATP, 7.5 mM phosphoenolpyruvate, and 18.8 U ml−1 pyruvate kinase (Key Resources Table). Initial cleavage rates were determined by measuring loss of 511 nm fluorescence over early linear time points. Fluorogenic peptide cleavage assays were performed at 37 °C with 1 µM coreAFG3L2WB or its variants and 50 µM peptide (Leu-(3-NO2-Tyr)-Phe-Gly-(Lys-Abz)) (GenScript) (Key Resources Table) in a 384-well black plate using SpectraMax M5 plate reader (ex = 320 nm; em = 420 nm). The reactions were performed in the buffer containing 25 mM HEPES-KOH, pH 7.5, 100 mM KCl, 5 mM MgCl2, 1 mM DTT, and 10% glycerol. Initial peptide cleavage rates were calculated from the loss of 420 nm fluorescence over early linear time points. Kaleidagraph (Synergy Software; Key Resources Table) was used to generate the ATPase Michaelis-Menten plot and related kinetic data shown in Figure S4F. Prism (GraphPad Software) was used for data presentation for all the other biochemical data (Key Resources Table).Size Exclusion Chromatography (SEC) assayFor determination of SEC profiles, one liter of E.coli culture containing coreAFG3L2WB/PI or its variants were expressed and purified following the procedure described above for use in biochemical characterization prior to SEC. Proteins were applied to a Superdex 200 10/300 GL Increase column equilibrated with buffer containing 25 mM HEPES-HCl (pH 7.5),100 mM KCl, 10% glycerol, and 1 mM DTT. Elution profiles compared against molecular weight standards to determine the ratio of assembled hexameric species to disassembled species for each coreAFG3L2WB/PI variant. MATLAB (MathWorks) was used to present the data (Key Resources Table).Sample preparation for electron microscopyFor cryo-EM structure determination, the coreAFG3L2WB/PI construct was incubated on ice for 5 minutes in 20mM Tris pH 8, 100mM NaCl, 5mM MgCl2, 1mM DTT, 1mM AMPPNP, 1% glycerol and 0.05% Lauryl Maltose Neopentyl Glycol (LMNG, Anatrace). Glycerol and LMNG were added to the buffer to help reduce the preferential orientation of the AFG3L2 particles in vitrified ice. 3 µl of the sample at 2 mg/ml were applied to 300 mesh UltrAuFoil Holey Gold Films R1.2/1.3 (Quantifoil) that had been previously plasma cleaned in a 75% argon / 25% oxygen atmosphere at 15 Watts for 6 seconds using a Solarus plasma cleaner (Gatan, Inc). The grids were loaded into a Vitrobot (ThermoFisher) with an environment chamber at a temperature of 4 °C and 100% humidity. Grids were blotted for 3 seconds with Whatman No.1 filter paper and plunged into a liquid ethane slurry.Electron microscopy data acquisitionCryo-EM data were collected on a ThermoFisher Talos Arctica transmission electron microscope (TEM) operating at 200keV, which was aligned as previously described (Herzik et al., 2017). Dose-fractionated movies were collected using a Gatan K2 Summit direct electron detector operating in electron counting mode, saving 30 frames during a 7s exposure. At an exposure rate of 4.3 e−/pixel, this resulted in a cumulative exposure of 40 e−/Å2. The Leginon data collection software (Suloway et al., 2005) (Key Resources Table) was used to collect 5,707 micrographs by moving to the center of four holes and image shifting to acquire four exposures at 36,000x nominal magnification (1.15 Å/pixel at the specimen level), with a nominal defocus range of −0.8 to −1.8 µm.Image processingThe Appion image processing workflow (Lander et al., 2009) was used to perform real-time image pre-processing during cryo-EM data collection (Key Resources Table). Micrograph frames were aligned using MotionCorr2 (Zheng et al., 2017) and CTF parameters were estimated with CTFFind4 (Rohou and Grigorieff, 2015) (Key Resources Table). Only micrographs with Appion confidence values above 95% were further processed. A small subset of ~20,000 particles were selected with the FindEM template-based particle picker (Roseman, 2004) (Key Resources Table), using the negative stain 2D classes as templates. This subset of particles was extracted using an unbinned box size of 256 pixels, and binned by a factor of four for 2D classification using RELION 1.4 (Scheres, 2012) (Key Resources Table). The resulting class averages were used as templates to select particles using FindEM, yielding 4,541,491 coordinates for putative particle.Relion 3.0 (Zivanov et al., 2018) (Key Resources Table) was used for all the remaining processing steps (Figure S1B). Particles were extracted using an unbinned box size of 160 pixels and binned by a factor of four for 2D classification. The resulting class averages showed a preferred hexameric view of the complex, so only the particles contributing to one of these hexameric classes were retained for subsequent processing steps. All other particles contributing to 2D classes containing well-defined secondary structural elements were also selected for 3D analysis. A total of 2,901,805 particles were extracted (binned by two, box size 80 pixels) and classified into five classes using the scaled and low-pass filtered cryo-EM structure of yeast YME1 (EMDB-7023) (Puchades et al., 2017) as an initial model and a regularization parameter of eight. The 1,129,437 particles contributing to the three best 3D class averages containing well-defined structural features were extracted with a box size of 240 pixels using centered coordinates. These particles were refined to a resolution of ~3.2 Å resolution, although the observable structural features were not consistent with this reported resolution, and were not sufficiently resolved for atomic modeling.We next performed CTF refinement and beam tilt estimation (estimated beam tilt x=−0.25, beam tilt y=0.07), and used these estimates to re-refine the particle stack, which yielded a structure with a reported resolution of ~3.0 Å. However, this reconstruction still appeared to suffer from issues with flexibility and misalignment, and did not contain structural features that are consistent with 3.0 Å cryo-EM density. Suspecting that these apparent misalignments may be due to subtle motions of the ATPase and protease rings relative to one another, we next performed a multi-body refinement of the AFG3L2 hexamer using three masked regions: the c6-symmetric protease ring, the four most stable ATPase domains (subunits B-E), and the two remaining ATPase domains for which the cryo-EM density was poorly resolved (subunits A,F). While these two flexible domains were not improved by the multi-body refinement approach and are reported at ~5 Å resolution, the observable details of the protease and four stable ATPase domains improved substantially. The reported resolutions for the ATPase and protease domains are ~3.0 Å and ~2.9 Å, respectively, by FSC 0.143.Local resolution estimation using the “blocres” function in the Bsoft package (Heymann, 2018) indicates that there are regions within the reconstructions that are resolved to better than the FSC-reported resolutions. However, we do not observe EM density corresponding to ordered water molecules, which should be visible at better than 2.9 Å resolution so we suspect that the reported FSC-based resolutions are slightly inflated, possibly due to the presence of preferred views in the final reconstruction (Figure S1D). The individual reconstructions output from the multi-body refinement were individually sharpened and low-pass filtered (Table S2), and a composite map was generated to facilitate atomic model building using the “vop max” function in UCSF Chimera (Goddard et al., 2007).Atomic model building and refinementA homology model was generated using the structure of a subunit of YME1 (PDB ID: 6AZ0; Key Resources Table) as a starting point. Six copies of this atomic model were generated and each split into the three domains and rigid body fit into each subunit using the “fit in map” function in UCSF Chimera (Goddard et al., 2007). The ATPase domains of the apo and ATP1 subunits (subunits F and A, respectively) were not further processed as the quality of the density for these domains did not allow for confident atomic modeling, and are included in the deposited model only as a Cα trace. The homology model for the rest of the hexamer, including nucleotides and coordinating metals, was adjusted using the COOT software package (Emsley and Cowtan, 2004), and then real-space refined with rigid body fitting and simulated annealing using the PHENIX package (Afonine et al., 2012) (Key Resources Table). This refined model was used as input for a multi-model-generating pipeline (Herzik et al., 2018) that aided in assessing the quality of the model and the map for mechanistic interpretation (Figure S1F). Briefly, the refined model and the map were used to generate 200 models in Rosetta, and the top ten scoring models were selected for further refinement in Phenix. The per-residue Cα root mean square deviation (RMSD) of top ten models was calculated and used to identify regions with poor model convergence. These regions were inspected and re-modeled and re-refined. Regions of the map with poor convergence either had the side-chains were truncated to the Cβ or removed entirely from the model.UCSF Chimera (Goddard et al., 2007) was used to generate the figures (Key Resources Table).Cell Culture and Mitochondrial isolationHEK293T cells (Key Resources Table) were cultured in DMEM (Cellgro) supplemented with 10% fetal bovine serum (Cellgro), 1% penicillin/streptavidin (Gibco) and 1% L-glutamine (Cellgro) at 37°C (5% CO 2). Cells were transiently transfected with pcDNA3.1 vector expressing C-terminally Flag-tagged AFG3L2 (GenScript) with the indicated single-point mutations (Key Resources Table). After 36h, mitochondria were isolated from cells as previously reported (Haynes et al., 2010).SDS- and BN-PAGE analysisFreshly isolated mitochondria were lysed in SDS (20 mM Hepes pH 7.5, 100 mM NaCl, 1 mM EDTA, 1% Triton supplemented with EDTA-free protease inhibitors (Roche)) or BN-Lysis buffer (20 mM Tris pH 7.5, 10% w/v glycerol, 50 mM NaCl, 0.1 mM EDTA, and 1mM PMSF supplemented with 1% n-dodecyl-β-D-maltoside). Samples were normalized by total protein concentration using Bio-Rad protein quantification. For SDS-PAGE, lysates were separated on 10 % acrylamide gels and transferred onto nitrocellulose membranes (Bio-Rad) for immunoblotting. Following incubation with primary antibodies, membranes were incubated with IR-labeled secondary antibodies (Li-COR Biosciences) and analyzed using the Odyssey Infrared Imaging System (Li-COR Biosciences). BN-PAGE samples were separated on 4–16% polyacrylamide gels, and western blotted as described (Geissler et al., 2002). Primary antibodies: anti-AFG3L2 (Genetex, 1:1000 dilution), anti-Flag (M2 Flag, Sigma, 1:1000 dilution), anti-Tubulin (Sigma, 1:10000 dilution) (Key Resources Table).QUANTIFICATION AND STATISTICAL ANALYSISStatistical details of all experiments can be found in the relevant figure legends. The biochemical data reflect the average of at least three independent experiments, and the error bars are the standard deviations generated from each group of data. Student’s two-tailed t test were performed to indicate the statistical significance of our data. Statistical details of all experiments can be found in the relevant figure legends.DATA AND MATERIALS AVAILABILITYAll the cryo-EM maps were deposited in the Electron Microscopy Data Bank under accession code EMD-0552. The associated atomic coordinates were deposited into the Protein Data Bank with accession code 6NYY.

STAR METHODS

CONTACT FOR REAGENT AND RESOURCE SHARINGFurther information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Steven E. Glynn (steven.glynn@stonybrook.edu).

CONTACT FOR REAGENT AND RESOURCE SHARING

Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Steven E. Glynn (steven.glynn@stonybrook.edu).

EXPERIMENTAL MODEL AND SUBJECT DETAILSMicrobe StrainsEscherichia Coli DH5α were grown in LB broth supplemented with 100 µg/ml ampicillin at 37 °C. Escherichia Coli BL21-CodonPlus (DE3)-RIL were grown in LB broth supplemented with 100 µg/ml ampicillin and 34 µg/ml chloramphenicol at 37 °C.Cell LinesHuman embryonic kidney (HEK) 293T are female and cultured in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum (Gemini), 1% penicillin-streptomycin, 1% glutamine at 37°C and 5% CO 2.

EXPERIMENTAL MODEL AND SUBJECT DETAILS

Microbe StrainsEscherichia Coli DH5α were grown in LB broth supplemented with 100 µg/ml ampicillin at 37 °C. Escherichia Coli BL21-CodonPlus (DE3)-RIL were grown in LB broth supplemented with 100 µg/ml ampicillin and 34 µg/ml chloramphenicol at 37 °C.

Microbe Strains

Escherichia Coli DH5α were grown in LB broth supplemented with 100 µg/ml ampicillin at 37 °C. Escherichia Coli BL21-CodonPlus (DE3)-RIL were grown in LB broth supplemented with 100 µg/ml ampicillin and 34 µg/ml chloramphenicol at 37 °C.

Cell LinesHuman embryonic kidney (HEK) 293T are female and cultured in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum (Gemini), 1% penicillin-streptomycin, 1% glutamine at 37°C and 5% CO 2.

Human embryonic kidney (HEK) 293T are female and cultured in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum (Gemini), 1% penicillin-streptomycin, 1% glutamine at 37°C and 5% CO 2.

METHOD DETAILSCloning, Expression and PurificationAll molecular cloning was performed using E. Coli DH5-α Competent Cells (NEB) (Key Resources Table). Plasmids containing sequences encoding cchexAFG3L2 and coreAFG3L2 were constructed by subcloning a sequence encoding residues 272–797 of human AFG3L2 into a 2G-T vector and a modified 2G-T vector containing a cc-hex sequence respectively (Ding et al., 2018). Truncated AFG3L2 variants were constructed similarly using the primers listed in the Table S3. Mutant AFG3L2 constructs were created by site-directed mutagenesis using the primers listed in the Table S3. All AFG3L2 enzymes were expressed in E. coli BL21-CodonPlus (Key Resources Table) cells by induction at OD600~0.6 with 0.2 mM IPTG at 16 °C for 14 hours. For biochemical and biophysical characterization assays, cells containing AFG3L2 constructs were centrifuged at 4000 rpm for 30 minutes. Cell pellets containing cchexAFG3L2 or variants were resuspended in lysis buffer (20 mM Tris-HCl, pH 7.8, 300 mM NaCl, 10% Glycerol, 0.1 mM EDTA and 10 mM β-mercaptoethanol) supplemented with 10 mM PMSF, and lysed by sonication. Cell lysates were centrifuged at 14000 rpm for 30 minutes and the supernatants loaded to a Glutathione Superflow Agarose column (Pierce). Unbound proteins were washed off by addition of 10 column volumes of lysis buffer and bound proteins eluted by addition of lysis buffer supplemented with 10 mM reduced glutathione (Pierce). To remove the His6-GST tag, 0.5 mg TEV protease was added to the eluted proteins and the mixtures incubated at 4 °C for 16 hours. Cleaved cchexAFG3L2 proteins were then applied to a Ni-NTA column (Thermo Scientific) and collected in the flow-through. Proteins were concentrated and loaded onto a Superose 6 10/300 GL Increase size exclusion column (GE Healthcare) equilibrated with SEC buffer (20 mM Tris-HCl, pH 7.8, 100 mM NaCl, 10% Glycerol, 0.1 mM EDTA and 1 mM DTT). coreAFG3L2WB, coreAFG3L2WB/PI and their variants were purified using a similar protocol with the following modifications. Lysis buffer contained 50 mM HEPES-HCl, pH 7.5, 300 mM KCl, 0.1 mM EDTA, 10% glycerol and 10 mM β-mercaptoethanol, and SEC buffer contained 25 mM HEPES-HCl, pH 7.5, 100 mM KCl, 10% glycerol, and 1 mM DTT. Size exclusion chromatography was performed using a Superdex 200 Increase 10/300 GL Increase column (GE Healthcare). For cryo-EM structure determination, coreAFG3L2WB/PI was further purified by the following modifications to the protocol. Prior to size exclusion chromatography, coreAFG3L2WB/PI was exchanged into buffer containing 50 mM HEPES-HCl, pH 7.5, 150 mM KCl, 2 mM EDTA, 10% glycerol, and 10 mM β-mercaptoethanol. After buffer exchange, 15 mM MgCl2 and 1 mM AMP-PNP were added to the protein sample, and the mixture was incubated for 4 hours on ice. The concentrated protein sample was then applied to a Superdex 200 Increase 10/300 GL Increase column (GE Healthcare) equilibrated with buffer containing 25 mM HEPES-HCl, pH 7.5, 100 mM KCl, 20 µM AMP-PNP, 10 mM MgCl2, 10% glycerol and 1 mM DTT. Fractions corresponding to the calculated elution volume of related AFG3L2 hexamers were pooled, concentrated, flash-frozen, and stored at −80 °C.The SFGFP-10/11A226G-β20 model substrate was produced by inserting a sequence encoding the β20 degron into a plasmid encoding SFGFP-10/11A226G, which was a gift from Prof. Robert Sauer (MIT) (Iosefson et al., 2015). Proteins were expressed in E. coli BL21-CodonPlus cells by growth at 37 °C followed by addition of 1 mM IPTG and growth at 16 °C for 14 hours. Cells were centrifuged at 14,000 r pm for 30 minutes and resuspended in lysis buffer containing 20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, 10 mM imidazole, and 10 mM β-mercaptoethanol. The supernatants were applied to a Ni-NTA column (Thermo Scientific) followed by washing with buffer containing 20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, 50 mM imidazole, and 10 mM β-mercaptoethanol. Proteins were eluted by addition of elution buffer (20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, 250 mM imidazole, and 10 mM β-mercaptoethanol). Size exclusion chromatography was performed using a Superdex 200 Increase 10/300 GL Increase column (GE Healthcare) equilibrated with buffer containing 20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, and 1 mM DTT. The fractions corresponding to the correct protein were collected, concentrated, and flash frozen with liquid nitrogen for storage at −80 °C.Biochemical AssaysATPase activity, fluorescence-based protein degradation, and fluorogenic peptide cleavage assays were performed as previously described with some modifications (Ding et al., 2018; Rampello and Glynn, 2017). ATPase assays, as a coupled-enzyme assay monitoring the loss of NADH at 340 nm, were carried out at 37 °C in a 384-well clear bottom plate (Corning) using a SpectraMax M5 plate reader (Molecular Devices) with 1 µM cchexAFG3L2 or its variants. Reaction buffer contains 25 mM HEPES-KOH, pH 7.5, 5 mM MgCl2, 10% glycerol, 1 mM NADH, 21.5 U ml−1 lactate dehydrogenase and an ATP regeneration system (2 mM ATP, 7.5 mM phosphoenolpyruvate and 18.8 U ml−1 pyruvate kinase) (Key Resources Table). For fluorescence-based protein degradation assay, the reactions were carried out in a 384-well black plate (ex = 467 nm; em = 511 nm) at 37 ºC with 1 µM cchexAFG3L2 or its variants and 20 µM SFGFP-10/11A226G-β20. The initial time points with linear decrease in 340 nm signal were used to calculate the ATPase rates. Steady-state ATPase data were fit to the Hill version of the Michaelis−Menten equation [v = kATPase/(1 + K0.5/[ATP]n]. Fluorescence based protein degradation assays were performed at 37 °C in a 384-well black plate (Corning) using a SpectraMax M5 plate reader (ex = 467 nm; em = 511 nm) with 1 µM cchexAFG3L2 or its variants and 20 µM SFGFP-10/11A226G-β20 in the buffer containing 25 mM HEPES-KOH, pH 7.5, 100 mM KCl, 5 mM MgCl2, 10% glycerol, 1 mM DTT, 2 mM ATP, 7.5 mM phosphoenolpyruvate, and 18.8 U ml−1 pyruvate kinase (Key Resources Table). Initial cleavage rates were determined by measuring loss of 511 nm fluorescence over early linear time points. Fluorogenic peptide cleavage assays were performed at 37 °C with 1 µM coreAFG3L2WB or its variants and 50 µM peptide (Leu-(3-NO2-Tyr)-Phe-Gly-(Lys-Abz)) (GenScript) (Key Resources Table) in a 384-well black plate using SpectraMax M5 plate reader (ex = 320 nm; em = 420 nm). The reactions were performed in the buffer containing 25 mM HEPES-KOH, pH 7.5, 100 mM KCl, 5 mM MgCl2, 1 mM DTT, and 10% glycerol. Initial peptide cleavage rates were calculated from the loss of 420 nm fluorescence over early linear time points. Kaleidagraph (Synergy Software; Key Resources Table) was used to generate the ATPase Michaelis-Menten plot and related kinetic data shown in Figure S4F. Prism (GraphPad Software) was used for data presentation for all the other biochemical data (Key Resources Table).Size Exclusion Chromatography (SEC) assayFor determination of SEC profiles, one liter of E.coli culture containing coreAFG3L2WB/PI or its variants were expressed and purified following the procedure described above for use in biochemical characterization prior to SEC. Proteins were applied to a Superdex 200 10/300 GL Increase column equilibrated with buffer containing 25 mM HEPES-HCl (pH 7.5),100 mM KCl, 10% glycerol, and 1 mM DTT. Elution profiles compared against molecular weight standards to determine the ratio of assembled hexameric species to disassembled species for each coreAFG3L2WB/PI variant. MATLAB (MathWorks) was used to present the data (Key Resources Table).Sample preparation for electron microscopyFor cryo-EM structure determination, the coreAFG3L2WB/PI construct was incubated on ice for 5 minutes in 20mM Tris pH 8, 100mM NaCl, 5mM MgCl2, 1mM DTT, 1mM AMPPNP, 1% glycerol and 0.05% Lauryl Maltose Neopentyl Glycol (LMNG, Anatrace). Glycerol and LMNG were added to the buffer to help reduce the preferential orientation of the AFG3L2 particles in vitrified ice. 3 µl of the sample at 2 mg/ml were applied to 300 mesh UltrAuFoil Holey Gold Films R1.2/1.3 (Quantifoil) that had been previously plasma cleaned in a 75% argon / 25% oxygen atmosphere at 15 Watts for 6 seconds using a Solarus plasma cleaner (Gatan, Inc). The grids were loaded into a Vitrobot (ThermoFisher) with an environment chamber at a temperature of 4 °C and 100% humidity. Grids were blotted for 3 seconds with Whatman No.1 filter paper and plunged into a liquid ethane slurry.Electron microscopy data acquisitionCryo-EM data were collected on a ThermoFisher Talos Arctica transmission electron microscope (TEM) operating at 200keV, which was aligned as previously described (Herzik et al., 2017). Dose-fractionated movies were collected using a Gatan K2 Summit direct electron detector operating in electron counting mode, saving 30 frames during a 7s exposure. At an exposure rate of 4.3 e−/pixel, this resulted in a cumulative exposure of 40 e−/Å2. The Leginon data collection software (Suloway et al., 2005) (Key Resources Table) was used to collect 5,707 micrographs by moving to the center of four holes and image shifting to acquire four exposures at 36,000x nominal magnification (1.15 Å/pixel at the specimen level), with a nominal defocus range of −0.8 to −1.8 µm.Image processingThe Appion image processing workflow (Lander et al., 2009) was used to perform real-time image pre-processing during cryo-EM data collection (Key Resources Table). Micrograph frames were aligned using MotionCorr2 (Zheng et al., 2017) and CTF parameters were estimated with CTFFind4 (Rohou and Grigorieff, 2015) (Key Resources Table). Only micrographs with Appion confidence values above 95% were further processed. A small subset of ~20,000 particles were selected with the FindEM template-based particle picker (Roseman, 2004) (Key Resources Table), using the negative stain 2D classes as templates. This subset of particles was extracted using an unbinned box size of 256 pixels, and binned by a factor of four for 2D classification using RELION 1.4 (Scheres, 2012) (Key Resources Table). The resulting class averages were used as templates to select particles using FindEM, yielding 4,541,491 coordinates for putative particle.Relion 3.0 (Zivanov et al., 2018) (Key Resources Table) was used for all the remaining processing steps (Figure S1B). Particles were extracted using an unbinned box size of 160 pixels and binned by a factor of four for 2D classification. The resulting class averages showed a preferred hexameric view of the complex, so only the particles contributing to one of these hexameric classes were retained for subsequent processing steps. All other particles contributing to 2D classes containing well-defined secondary structural elements were also selected for 3D analysis. A total of 2,901,805 particles were extracted (binned by two, box size 80 pixels) and classified into five classes using the scaled and low-pass filtered cryo-EM structure of yeast YME1 (EMDB-7023) (Puchades et al., 2017) as an initial model and a regularization parameter of eight. The 1,129,437 particles contributing to the three best 3D class averages containing well-defined structural features were extracted with a box size of 240 pixels using centered coordinates. These particles were refined to a resolution of ~3.2 Å resolution, although the observable structural features were not consistent with this reported resolution, and were not sufficiently resolved for atomic modeling.We next performed CTF refinement and beam tilt estimation (estimated beam tilt x=−0.25, beam tilt y=0.07), and used these estimates to re-refine the particle stack, which yielded a structure with a reported resolution of ~3.0 Å. However, this reconstruction still appeared to suffer from issues with flexibility and misalignment, and did not contain structural features that are consistent with 3.0 Å cryo-EM density. Suspecting that these apparent misalignments may be due to subtle motions of the ATPase and protease rings relative to one another, we next performed a multi-body refinement of the AFG3L2 hexamer using three masked regions: the c6-symmetric protease ring, the four most stable ATPase domains (subunits B-E), and the two remaining ATPase domains for which the cryo-EM density was poorly resolved (subunits A,F). While these two flexible domains were not improved by the multi-body refinement approach and are reported at ~5 Å resolution, the observable details of the protease and four stable ATPase domains improved substantially. The reported resolutions for the ATPase and protease domains are ~3.0 Å and ~2.9 Å, respectively, by FSC 0.143.Local resolution estimation using the “blocres” function in the Bsoft package (Heymann, 2018) indicates that there are regions within the reconstructions that are resolved to better than the FSC-reported resolutions. However, we do not observe EM density corresponding to ordered water molecules, which should be visible at better than 2.9 Å resolution so we suspect that the reported FSC-based resolutions are slightly inflated, possibly due to the presence of preferred views in the final reconstruction (Figure S1D). The individual reconstructions output from the multi-body refinement were individually sharpened and low-pass filtered (Table S2), and a composite map was generated to facilitate atomic model building using the “vop max” function in UCSF Chimera (Goddard et al., 2007).Atomic model building and refinementA homology model was generated using the structure of a subunit of YME1 (PDB ID: 6AZ0; Key Resources Table) as a starting point. Six copies of this atomic model were generated and each split into the three domains and rigid body fit into each subunit using the “fit in map” function in UCSF Chimera (Goddard et al., 2007). The ATPase domains of the apo and ATP1 subunits (subunits F and A, respectively) were not further processed as the quality of the density for these domains did not allow for confident atomic modeling, and are included in the deposited model only as a Cα trace. The homology model for the rest of the hexamer, including nucleotides and coordinating metals, was adjusted using the COOT software package (Emsley and Cowtan, 2004), and then real-space refined with rigid body fitting and simulated annealing using the PHENIX package (Afonine et al., 2012) (Key Resources Table). This refined model was used as input for a multi-model-generating pipeline (Herzik et al., 2018) that aided in assessing the quality of the model and the map for mechanistic interpretation (Figure S1F). Briefly, the refined model and the map were used to generate 200 models in Rosetta, and the top ten scoring models were selected for further refinement in Phenix. The per-residue Cα root mean square deviation (RMSD) of top ten models was calculated and used to identify regions with poor model convergence. These regions were inspected and re-modeled and re-refined. Regions of the map with poor convergence either had the side-chains were truncated to the Cβ or removed entirely from the model.UCSF Chimera (Goddard et al., 2007) was used to generate the figures (Key Resources Table).Cell Culture and Mitochondrial isolationHEK293T cells (Key Resources Table) were cultured in DMEM (Cellgro) supplemented with 10% fetal bovine serum (Cellgro), 1% penicillin/streptavidin (Gibco) and 1% L-glutamine (Cellgro) at 37°C (5% CO 2). Cells were transiently transfected with pcDNA3.1 vector expressing C-terminally Flag-tagged AFG3L2 (GenScript) with the indicated single-point mutations (Key Resources Table). After 36h, mitochondria were isolated from cells as previously reported (Haynes et al., 2010).SDS- and BN-PAGE analysisFreshly isolated mitochondria were lysed in SDS (20 mM Hepes pH 7.5, 100 mM NaCl, 1 mM EDTA, 1% Triton supplemented with EDTA-free protease inhibitors (Roche)) or BN-Lysis buffer (20 mM Tris pH 7.5, 10% w/v glycerol, 50 mM NaCl, 0.1 mM EDTA, and 1mM PMSF supplemented with 1% n-dodecyl-β-D-maltoside). Samples were normalized by total protein concentration using Bio-Rad protein quantification. For SDS-PAGE, lysates were separated on 10 % acrylamide gels and transferred onto nitrocellulose membranes (Bio-Rad) for immunoblotting. Following incubation with primary antibodies, membranes were incubated with IR-labeled secondary antibodies (Li-COR Biosciences) and analyzed using the Odyssey Infrared Imaging System (Li-COR Biosciences). BN-PAGE samples were separated on 4–16% polyacrylamide gels, and western blotted as described (Geissler et al., 2002). Primary antibodies: anti-AFG3L2 (Genetex, 1:1000 dilution), anti-Flag (M2 Flag, Sigma, 1:1000 dilution), anti-Tubulin (Sigma, 1:10000 dilution) (Key Resources Table).

METHOD DETAILS

Cloning, Expression and PurificationAll molecular cloning was performed using E. Coli DH5-α Competent Cells (NEB) (Key Resources Table). Plasmids containing sequences encoding cchexAFG3L2 and coreAFG3L2 were constructed by subcloning a sequence encoding residues 272–797 of human AFG3L2 into a 2G-T vector and a modified 2G-T vector containing a cc-hex sequence respectively (Ding et al., 2018). Truncated AFG3L2 variants were constructed similarly using the primers listed in the Table S3. Mutant AFG3L2 constructs were created by site-directed mutagenesis using the primers listed in the Table S3. All AFG3L2 enzymes were expressed in E. coli BL21-CodonPlus (Key Resources Table) cells by induction at OD600~0.6 with 0.2 mM IPTG at 16 °C for 14 hours. For biochemical and biophysical characterization assays, cells containing AFG3L2 constructs were centrifuged at 4000 rpm for 30 minutes. Cell pellets containing cchexAFG3L2 or variants were resuspended in lysis buffer (20 mM Tris-HCl, pH 7.8, 300 mM NaCl, 10% Glycerol, 0.1 mM EDTA and 10 mM β-mercaptoethanol) supplemented with 10 mM PMSF, and lysed by sonication. Cell lysates were centrifuged at 14000 rpm for 30 minutes and the supernatants loaded to a Glutathione Superflow Agarose column (Pierce). Unbound proteins were washed off by addition of 10 column volumes of lysis buffer and bound proteins eluted by addition of lysis buffer supplemented with 10 mM reduced glutathione (Pierce). To remove the His6-GST tag, 0.5 mg TEV protease was added to the eluted proteins and the mixtures incubated at 4 °C for 16 hours. Cleaved cchexAFG3L2 proteins were then applied to a Ni-NTA column (Thermo Scientific) and collected in the flow-through. Proteins were concentrated and loaded onto a Superose 6 10/300 GL Increase size exclusion column (GE Healthcare) equilibrated with SEC buffer (20 mM Tris-HCl, pH 7.8, 100 mM NaCl, 10% Glycerol, 0.1 mM EDTA and 1 mM DTT). coreAFG3L2WB, coreAFG3L2WB/PI and their variants were purified using a similar protocol with the following modifications. Lysis buffer contained 50 mM HEPES-HCl, pH 7.5, 300 mM KCl, 0.1 mM EDTA, 10% glycerol and 10 mM β-mercaptoethanol, and SEC buffer contained 25 mM HEPES-HCl, pH 7.5, 100 mM KCl, 10% glycerol, and 1 mM DTT. Size exclusion chromatography was performed using a Superdex 200 Increase 10/300 GL Increase column (GE Healthcare). For cryo-EM structure determination, coreAFG3L2WB/PI was further purified by the following modifications to the protocol. Prior to size exclusion chromatography, coreAFG3L2WB/PI was exchanged into buffer containing 50 mM HEPES-HCl, pH 7.5, 150 mM KCl, 2 mM EDTA, 10% glycerol, and 10 mM β-mercaptoethanol. After buffer exchange, 15 mM MgCl2 and 1 mM AMP-PNP were added to the protein sample, and the mixture was incubated for 4 hours on ice. The concentrated protein sample was then applied to a Superdex 200 Increase 10/300 GL Increase column (GE Healthcare) equilibrated with buffer containing 25 mM HEPES-HCl, pH 7.5, 100 mM KCl, 20 µM AMP-PNP, 10 mM MgCl2, 10% glycerol and 1 mM DTT. Fractions corresponding to the calculated elution volume of related AFG3L2 hexamers were pooled, concentrated, flash-frozen, and stored at −80 °C.The SFGFP-10/11A226G-β20 model substrate was produced by inserting a sequence encoding the β20 degron into a plasmid encoding SFGFP-10/11A226G, which was a gift from Prof. Robert Sauer (MIT) (Iosefson et al., 2015). Proteins were expressed in E. coli BL21-CodonPlus cells by growth at 37 °C followed by addition of 1 mM IPTG and growth at 16 °C for 14 hours. Cells were centrifuged at 14,000 r pm for 30 minutes and resuspended in lysis buffer containing 20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, 10 mM imidazole, and 10 mM β-mercaptoethanol. The supernatants were applied to a Ni-NTA column (Thermo Scientific) followed by washing with buffer containing 20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, 50 mM imidazole, and 10 mM β-mercaptoethanol. Proteins were eluted by addition of elution buffer (20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, 250 mM imidazole, and 10 mM β-mercaptoethanol). Size exclusion chromatography was performed using a Superdex 200 Increase 10/300 GL Increase column (GE Healthcare) equilibrated with buffer containing 20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, and 1 mM DTT. The fractions corresponding to the correct protein were collected, concentrated, and flash frozen with liquid nitrogen for storage at −80 °C.

Cloning, Expression and Purification

All molecular cloning was performed using E. Coli DH5-α Competent Cells (NEB) (Key Resources Table). Plasmids containing sequences encoding cchexAFG3L2 and coreAFG3L2 were constructed by subcloning a sequence encoding residues 272–797 of human AFG3L2 into a 2G-T vector and a modified 2G-T vector containing a cc-hex sequence respectively (Ding et al., 2018). Truncated AFG3L2 variants were constructed similarly using the primers listed in the Table S3. Mutant AFG3L2 constructs were created by site-directed mutagenesis using the primers listed in the Table S3. All AFG3L2 enzymes were expressed in E. coli BL21-CodonPlus (Key Resources Table) cells by induction at OD600~0.6 with 0.2 mM IPTG at 16 °C for 14 hours. For biochemical and biophysical characterization assays, cells containing AFG3L2 constructs were centrifuged at 4000 rpm for 30 minutes. Cell pellets containing cchexAFG3L2 or variants were resuspended in lysis buffer (20 mM Tris-HCl, pH 7.8, 300 mM NaCl, 10% Glycerol, 0.1 mM EDTA and 10 mM β-mercaptoethanol) supplemented with 10 mM PMSF, and lysed by sonication. Cell lysates were centrifuged at 14000 rpm for 30 minutes and the supernatants loaded to a Glutathione Superflow Agarose column (Pierce). Unbound proteins were washed off by addition of 10 column volumes of lysis buffer and bound proteins eluted by addition of lysis buffer supplemented with 10 mM reduced glutathione (Pierce). To remove the His6-GST tag, 0.5 mg TEV protease was added to the eluted proteins and the mixtures incubated at 4 °C for 16 hours. Cleaved cchexAFG3L2 proteins were then applied to a Ni-NTA column (Thermo Scientific) and collected in the flow-through. Proteins were concentrated and loaded onto a Superose 6 10/300 GL Increase size exclusion column (GE Healthcare) equilibrated with SEC buffer (20 mM Tris-HCl, pH 7.8, 100 mM NaCl, 10% Glycerol, 0.1 mM EDTA and 1 mM DTT). coreAFG3L2WB, coreAFG3L2WB/PI and their variants were purified using a similar protocol with the following modifications. Lysis buffer contained 50 mM HEPES-HCl, pH 7.5, 300 mM KCl, 0.1 mM EDTA, 10% glycerol and 10 mM β-mercaptoethanol, and SEC buffer contained 25 mM HEPES-HCl, pH 7.5, 100 mM KCl, 10% glycerol, and 1 mM DTT. Size exclusion chromatography was performed using a Superdex 200 Increase 10/300 GL Increase column (GE Healthcare). For cryo-EM structure determination, coreAFG3L2WB/PI was further purified by the following modifications to the protocol. Prior to size exclusion chromatography, coreAFG3L2WB/PI was exchanged into buffer containing 50 mM HEPES-HCl, pH 7.5, 150 mM KCl, 2 mM EDTA, 10% glycerol, and 10 mM β-mercaptoethanol. After buffer exchange, 15 mM MgCl2 and 1 mM AMP-PNP were added to the protein sample, and the mixture was incubated for 4 hours on ice. The concentrated protein sample was then applied to a Superdex 200 Increase 10/300 GL Increase column (GE Healthcare) equilibrated with buffer containing 25 mM HEPES-HCl, pH 7.5, 100 mM KCl, 20 µM AMP-PNP, 10 mM MgCl2, 10% glycerol and 1 mM DTT. Fractions corresponding to the calculated elution volume of related AFG3L2 hexamers were pooled, concentrated, flash-frozen, and stored at −80 °C.

The SFGFP-10/11A226G-β20 model substrate was produced by inserting a sequence encoding the β20 degron into a plasmid encoding SFGFP-10/11A226G, which was a gift from Prof. Robert Sauer (MIT) (Iosefson et al., 2015). Proteins were expressed in E. coli BL21-CodonPlus cells by growth at 37 °C followed by addition of 1 mM IPTG and growth at 16 °C for 14 hours. Cells were centrifuged at 14,000 r pm for 30 minutes and resuspended in lysis buffer containing 20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, 10 mM imidazole, and 10 mM β-mercaptoethanol. The supernatants were applied to a Ni-NTA column (Thermo Scientific) followed by washing with buffer containing 20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, 50 mM imidazole, and 10 mM β-mercaptoethanol. Proteins were eluted by addition of elution buffer (20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, 250 mM imidazole, and 10 mM β-mercaptoethanol). Size exclusion chromatography was performed using a Superdex 200 Increase 10/300 GL Increase column (GE Healthcare) equilibrated with buffer containing 20 mM Tris-HCl, pH 7.5, 300 mM NaCl, 10 % glycerol, and 1 mM DTT. The fractions corresponding to the correct protein were collected, concentrated, and flash frozen with liquid nitrogen for storage at −80 °C.

Biochemical AssaysATPase activity, fluorescence-based protein degradation, and fluorogenic peptide cleavage assays were performed as previously described with some modifications (Ding et al., 2018; Rampello and Glynn, 2017). ATPase assays, as a coupled-enzyme assay monitoring the loss of NADH at 340 nm, were carried out at 37 °C in a 384-well clear bottom plate (Corning) using a SpectraMax M5 plate reader (Molecular Devices) with 1 µM cchexAFG3L2 or its variants. Reaction buffer contains 25 mM HEPES-KOH, pH 7.5, 5 mM MgCl2, 10% glycerol, 1 mM NADH, 21.5 U ml−1 lactate dehydrogenase and an ATP regeneration system (2 mM ATP, 7.5 mM phosphoenolpyruvate and 18.8 U ml−1 pyruvate kinase) (Key Resources Table). For fluorescence-based protein degradation assay, the reactions were carried out in a 384-well black plate (ex = 467 nm; em = 511 nm) at 37 ºC with 1 µM cchexAFG3L2 or its variants and 20 µM SFGFP-10/11A226G-β20. The initial time points with linear decrease in 340 nm signal were used to calculate the ATPase rates. Steady-state ATPase data were fit to the Hill version of the Michaelis−Menten equation [v = kATPase/(1 + K0.5/[ATP]n]. Fluorescence based protein degradation assays were performed at 37 °C in a 384-well black plate (Corning) using a SpectraMax M5 plate reader (ex = 467 nm; em = 511 nm) with 1 µM cchexAFG3L2 or its variants and 20 µM SFGFP-10/11A226G-β20 in the buffer containing 25 mM HEPES-KOH, pH 7.5, 100 mM KCl, 5 mM MgCl2, 10% glycerol, 1 mM DTT, 2 mM ATP, 7.5 mM phosphoenolpyruvate, and 18.8 U ml−1 pyruvate kinase (Key Resources Table). Initial cleavage rates were determined by measuring loss of 511 nm fluorescence over early linear time points. Fluorogenic peptide cleavage assays were performed at 37 °C with 1 µM coreAFG3L2WB or its variants and 50 µM peptide (Leu-(3-NO2-Tyr)-Phe-Gly-(Lys-Abz)) (GenScript) (Key Resources Table) in a 384-well black plate using SpectraMax M5 plate reader (ex = 320 nm; em = 420 nm). The reactions were performed in the buffer containing 25 mM HEPES-KOH, pH 7.5, 100 mM KCl, 5 mM MgCl2, 1 mM DTT, and 10% glycerol. Initial peptide cleavage rates were calculated from the loss of 420 nm fluorescence over early linear time points. Kaleidagraph (Synergy Software; Key Resources Table) was used to generate the ATPase Michaelis-Menten plot and related kinetic data shown in Figure S4F. Prism (GraphPad Software) was used for data presentation for all the other biochemical data (Key Resources Table).

Biochemical Assays

ATPase activity, fluorescence-based protein degradation, and fluorogenic peptide cleavage assays were performed as previously described with some modifications (Ding et al., 2018; Rampello and Glynn, 2017). ATPase assays, as a coupled-enzyme assay monitoring the loss of NADH at 340 nm, were carried out at 37 °C in a 384-well clear bottom plate (Corning) using a SpectraMax M5 plate reader (Molecular Devices) with 1 µM cchexAFG3L2 or its variants. Reaction buffer contains 25 mM HEPES-KOH, pH 7.5, 5 mM MgCl2, 10% glycerol, 1 mM NADH, 21.5 U ml−1 lactate dehydrogenase and an ATP regeneration system (2 mM ATP, 7.5 mM phosphoenolpyruvate and 18.8 U ml−1 pyruvate kinase) (Key Resources Table). For fluorescence-based protein degradation assay, the reactions were carried out in a 384-well black plate (ex = 467 nm; em = 511 nm) at 37 ºC with 1 µM cchexAFG3L2 or its variants and 20 µM SFGFP-10/11A226G-β20. The initial time points with linear decrease in 340 nm signal were used to calculate the ATPase rates. Steady-state ATPase data were fit to the Hill version of the Michaelis−Menten equation [v = kATPase/(1 + K0.5/[ATP]n]. Fluorescence based protein degradation assays were performed at 37 °C in a 384-well black plate (Corning) using a SpectraMax M5 plate reader (ex = 467 nm; em = 511 nm) with 1 µM cchexAFG3L2 or its variants and 20 µM SFGFP-10/11A226G-β20 in the buffer containing 25 mM HEPES-KOH, pH 7.5, 100 mM KCl, 5 mM MgCl2, 10% glycerol, 1 mM DTT, 2 mM ATP, 7.5 mM phosphoenolpyruvate, and 18.8 U ml−1 pyruvate kinase (Key Resources Table). Initial cleavage rates were determined by measuring loss of 511 nm fluorescence over early linear time points. Fluorogenic peptide cleavage assays were performed at 37 °C with 1 µM coreAFG3L2WB or its variants and 50 µM peptide (Leu-(3-NO2-Tyr)-Phe-Gly-(Lys-Abz)) (GenScript) (Key Resources Table) in a 384-well black plate using SpectraMax M5 plate reader (ex = 320 nm; em = 420 nm). The reactions were performed in the buffer containing 25 mM HEPES-KOH, pH 7.5, 100 mM KCl, 5 mM MgCl2, 1 mM DTT, and 10% glycerol. Initial peptide cleavage rates were calculated from the loss of 420 nm fluorescence over early linear time points. Kaleidagraph (Synergy Software; Key Resources Table) was used to generate the ATPase Michaelis-Menten plot and related kinetic data shown in Figure S4F. Prism (GraphPad Software) was used for data presentation for all the other biochemical data (Key Resources Table).

Size Exclusion Chromatography (SEC) assayFor determination of SEC profiles, one liter of E.coli culture containing coreAFG3L2WB/PI or its variants were expressed and purified following the procedure described above for use in biochemical characterization prior to SEC. Proteins were applied to a Superdex 200 10/300 GL Increase column equilibrated with buffer containing 25 mM HEPES-HCl (pH 7.5),100 mM KCl, 10% glycerol, and 1 mM DTT. Elution profiles compared against molecular weight standards to determine the ratio of assembled hexameric species to disassembled species for each coreAFG3L2WB/PI variant. MATLAB (MathWorks) was used to present the data (Key Resources Table).

Size Exclusion Chromatography (SEC) assay

For determination of SEC profiles, one liter of E.coli culture containing coreAFG3L2WB/PI or its variants were expressed and purified following the procedure described above for use in biochemical characterization prior to SEC. Proteins were applied to a Superdex 200 10/300 GL Increase column equilibrated with buffer containing 25 mM HEPES-HCl (pH 7.5),100 mM KCl, 10% glycerol, and 1 mM DTT. Elution profiles compared against molecular weight standards to determine the ratio of assembled hexameric species to disassembled species for each coreAFG3L2WB/PI variant. MATLAB (MathWorks) was used to present the data (Key Resources Table).

Sample preparation for electron microscopyFor cryo-EM structure determination, the coreAFG3L2WB/PI construct was incubated on ice for 5 minutes in 20mM Tris pH 8, 100mM NaCl, 5mM MgCl2, 1mM DTT, 1mM AMPPNP, 1% glycerol and 0.05% Lauryl Maltose Neopentyl Glycol (LMNG, Anatrace). Glycerol and LMNG were added to the buffer to help reduce the preferential orientation of the AFG3L2 particles in vitrified ice. 3 µl of the sample at 2 mg/ml were applied to 300 mesh UltrAuFoil Holey Gold Films R1.2/1.3 (Quantifoil) that had been previously plasma cleaned in a 75% argon / 25% oxygen atmosphere at 15 Watts for 6 seconds using a Solarus plasma cleaner (Gatan, Inc). The grids were loaded into a Vitrobot (ThermoFisher) with an environment chamber at a temperature of 4 °C and 100% humidity. Grids were blotted for 3 seconds with Whatman No.1 filter paper and plunged into a liquid ethane slurry.

Sample preparation for electron microscopy

For cryo-EM structure determination, the coreAFG3L2WB/PI construct was incubated on ice for 5 minutes in 20mM Tris pH 8, 100mM NaCl, 5mM MgCl2, 1mM DTT, 1mM AMPPNP, 1% glycerol and 0.05% Lauryl Maltose Neopentyl Glycol (LMNG, Anatrace). Glycerol and LMNG were added to the buffer to help reduce the preferential orientation of the AFG3L2 particles in vitrified ice. 3 µl of the sample at 2 mg/ml were applied to 300 mesh UltrAuFoil Holey Gold Films R1.2/1.3 (Quantifoil) that had been previously plasma cleaned in a 75% argon / 25% oxygen atmosphere at 15 Watts for 6 seconds using a Solarus plasma cleaner (Gatan, Inc). The grids were loaded into a Vitrobot (ThermoFisher) with an environment chamber at a temperature of 4 °C and 100% humidity. Grids were blotted for 3 seconds with Whatman No.1 filter paper and plunged into a liquid ethane slurry.

Electron microscopy data acquisitionCryo-EM data were collected on a ThermoFisher Talos Arctica transmission electron microscope (TEM) operating at 200keV, which was aligned as previously described (Herzik et al., 2017). Dose-fractionated movies were collected using a Gatan K2 Summit direct electron detector operating in electron counting mode, saving 30 frames during a 7s exposure. At an exposure rate of 4.3 e−/pixel, this resulted in a cumulative exposure of 40 e−/Å2. The Leginon data collection software (Suloway et al., 2005) (Key Resources Table) was used to collect 5,707 micrographs by moving to the center of four holes and image shifting to acquire four exposures at 36,000x nominal magnification (1.15 Å/pixel at the specimen level), with a nominal defocus range of −0.8 to −1.8 µm.

Electron microscopy data acquisition

Cryo-EM data were collected on a ThermoFisher Talos Arctica transmission electron microscope (TEM) operating at 200keV, which was aligned as previously described (Herzik et al., 2017). Dose-fractionated movies were collected using a Gatan K2 Summit direct electron detector operating in electron counting mode, saving 30 frames during a 7s exposure. At an exposure rate of 4.3 e−/pixel, this resulted in a cumulative exposure of 40 e−/Å2. The Leginon data collection software (Suloway et al., 2005) (Key Resources Table) was used to collect 5,707 micrographs by moving to the center of four holes and image shifting to acquire four exposures at 36,000x nominal magnification (1.15 Å/pixel at the specimen level), with a nominal defocus range of −0.8 to −1.8 µm.

Image processingThe Appion image processing workflow (Lander et al., 2009) was used to perform real-time image pre-processing during cryo-EM data collection (Key Resources Table). Micrograph frames were aligned using MotionCorr2 (Zheng et al., 2017) and CTF parameters were estimated with CTFFind4 (Rohou and Grigorieff, 2015) (Key Resources Table). Only micrographs with Appion confidence values above 95% were further processed. A small subset of ~20,000 particles were selected with the FindEM template-based particle picker (Roseman, 2004) (Key Resources Table), using the negative stain 2D classes as templates. This subset of particles was extracted using an unbinned box size of 256 pixels, and binned by a factor of four for 2D classification using RELION 1.4 (Scheres, 2012) (Key Resources Table). The resulting class averages were used as templates to select particles using FindEM, yielding 4,541,491 coordinates for putative particle.Relion 3.0 (Zivanov et al., 2018) (Key Resources Table) was used for all the remaining processing steps (Figure S1B). Particles were extracted using an unbinned box size of 160 pixels and binned by a factor of four for 2D classification. The resulting class averages showed a preferred hexameric view of the complex, so only the particles contributing to one of these hexameric classes were retained for subsequent processing steps. All other particles contributing to 2D classes containing well-defined secondary structural elements were also selected for 3D analysis. A total of 2,901,805 particles were extracted (binned by two, box size 80 pixels) and classified into five classes using the scaled and low-pass filtered cryo-EM structure of yeast YME1 (EMDB-7023) (Puchades et al., 2017) as an initial model and a regularization parameter of eight. The 1,129,437 particles contributing to the three best 3D class averages containing well-defined structural features were extracted with a box size of 240 pixels using centered coordinates. These particles were refined to a resolution of ~3.2 Å resolution, although the observable structural features were not consistent with this reported resolution, and were not sufficiently resolved for atomic modeling.We next performed CTF refinement and beam tilt estimation (estimated beam tilt x=−0.25, beam tilt y=0.07), and used these estimates to re-refine the particle stack, which yielded a structure with a reported resolution of ~3.0 Å. However, this reconstruction still appeared to suffer from issues with flexibility and misalignment, and did not contain structural features that are consistent with 3.0 Å cryo-EM density. Suspecting that these apparent misalignments may be due to subtle motions of the ATPase and protease rings relative to one another, we next performed a multi-body refinement of the AFG3L2 hexamer using three masked regions: the c6-symmetric protease ring, the four most stable ATPase domains (subunits B-E), and the two remaining ATPase domains for which the cryo-EM density was poorly resolved (subunits A,F). While these two flexible domains were not improved by the multi-body refinement approach and are reported at ~5 Å resolution, the observable details of the protease and four stable ATPase domains improved substantially. The reported resolutions for the ATPase and protease domains are ~3.0 Å and ~2.9 Å, respectively, by FSC 0.143.Local resolution estimation using the “blocres” function in the Bsoft package (Heymann, 2018) indicates that there are regions within the reconstructions that are resolved to better than the FSC-reported resolutions. However, we do not observe EM density corresponding to ordered water molecules, which should be visible at better than 2.9 Å resolution so we suspect that the reported FSC-based resolutions are slightly inflated, possibly due to the presence of preferred views in the final reconstruction (Figure S1D). The individual reconstructions output from the multi-body refinement were individually sharpened and low-pass filtered (Table S2), and a composite map was generated to facilitate atomic model building using the “vop max” function in UCSF Chimera (Goddard et al., 2007).

Image processing

The Appion image processing workflow (Lander et al., 2009) was used to perform real-time image pre-processing during cryo-EM data collection (Key Resources Table). Micrograph frames were aligned using MotionCorr2 (Zheng et al., 2017) and CTF parameters were estimated with CTFFind4 (Rohou and Grigorieff, 2015) (Key Resources Table). Only micrographs with Appion confidence values above 95% were further processed. A small subset of ~20,000 particles were selected with the FindEM template-based particle picker (Roseman, 2004) (Key Resources Table), using the negative stain 2D classes as templates. This subset of particles was extracted using an unbinned box size of 256 pixels, and binned by a factor of four for 2D classification using RELION 1.4 (Scheres, 2012) (Key Resources Table). The resulting class averages were used as templates to select particles using FindEM, yielding 4,541,491 coordinates for putative particle.

Relion 3.0 (Zivanov et al., 2018) (Key Resources Table) was used for all the remaining processing steps (Figure S1B). Particles were extracted using an unbinned box size of 160 pixels and binned by a factor of four for 2D classification. The resulting class averages showed a preferred hexameric view of the complex, so only the particles contributing to one of these hexameric classes were retained for subsequent processing steps. All other particles contributing to 2D classes containing well-defined secondary structural elements were also selected for 3D analysis. A total of 2,901,805 particles were extracted (binned by two, box size 80 pixels) and classified into five classes using the scaled and low-pass filtered cryo-EM structure of yeast YME1 (EMDB-7023) (Puchades et al., 2017) as an initial model and a regularization parameter of eight. The 1,129,437 particles contributing to the three best 3D class averages containing well-defined structural features were extracted with a box size of 240 pixels using centered coordinates. These particles were refined to a resolution of ~3.2 Å resolution, although the observable structural features were not consistent with this reported resolution, and were not sufficiently resolved for atomic modeling.

We next performed CTF refinement and beam tilt estimation (estimated beam tilt x=−0.25, beam tilt y=0.07), and used these estimates to re-refine the particle stack, which yielded a structure with a reported resolution of ~3.0 Å. However, this reconstruction still appeared to suffer from issues with flexibility and misalignment, and did not contain structural features that are consistent with 3.0 Å cryo-EM density. Suspecting that these apparent misalignments may be due to subtle motions of the ATPase and protease rings relative to one another, we next performed a multi-body refinement of the AFG3L2 hexamer using three masked regions: the c6-symmetric protease ring, the four most stable ATPase domains (subunits B-E), and the two remaining ATPase domains for which the cryo-EM density was poorly resolved (subunits A,F). While these two flexible domains were not improved by the multi-body refinement approach and are reported at ~5 Å resolution, the observable details of the protease and four stable ATPase domains improved substantially. The reported resolutions for the ATPase and protease domains are ~3.0 Å and ~2.9 Å, respectively, by FSC 0.143.

Local resolution estimation using the “blocres” function in the Bsoft package (Heymann, 2018) indicates that there are regions within the reconstructions that are resolved to better than the FSC-reported resolutions. However, we do not observe EM density corresponding to ordered water molecules, which should be visible at better than 2.9 Å resolution so we suspect that the reported FSC-based resolutions are slightly inflated, possibly due to the presence of preferred views in the final reconstruction (Figure S1D). The individual reconstructions output from the multi-body refinement were individually sharpened and low-pass filtered (Table S2), and a composite map was generated to facilitate atomic model building using the “vop max” function in UCSF Chimera (Goddard et al., 2007).

Atomic model building and refinementA homology model was generated using the structure of a subunit of YME1 (PDB ID: 6AZ0; Key Resources Table) as a starting point. Six copies of this atomic model were generated and each split into the three domains and rigid body fit into each subunit using the “fit in map” function in UCSF Chimera (Goddard et al., 2007). The ATPase domains of the apo and ATP1 subunits (subunits F and A, respectively) were not further processed as the quality of the density for these domains did not allow for confident atomic modeling, and are included in the deposited model only as a Cα trace. The homology model for the rest of the hexamer, including nucleotides and coordinating metals, was adjusted using the COOT software package (Emsley and Cowtan, 2004), and then real-space refined with rigid body fitting and simulated annealing using the PHENIX package (Afonine et al., 2012) (Key Resources Table). This refined model was used as input for a multi-model-generating pipeline (Herzik et al., 2018) that aided in assessing the quality of the model and the map for mechanistic interpretation (Figure S1F). Briefly, the refined model and the map were used to generate 200 models in Rosetta, and the top ten scoring models were selected for further refinement in Phenix. The per-residue Cα root mean square deviation (RMSD) of top ten models was calculated and used to identify regions with poor model convergence. These regions were inspected and re-modeled and re-refined. Regions of the map with poor convergence either had the side-chains were truncated to the Cβ or removed entirely from the model.UCSF Chimera (Goddard et al., 2007) was used to generate the figures (Key Resources Table).

Atomic model building and refinement

A homology model was generated using the structure of a subunit of YME1 (PDB ID: 6AZ0; Key Resources Table) as a starting point. Six copies of this atomic model were generated and each split into the three domains and rigid body fit into each subunit using the “fit in map” function in UCSF Chimera (Goddard et al., 2007). The ATPase domains of the apo and ATP1 subunits (subunits F and A, respectively) were not further processed as the quality of the density for these domains did not allow for confident atomic modeling, and are included in the deposited model only as a Cα trace. The homology model for the rest of the hexamer, including nucleotides and coordinating metals, was adjusted using the COOT software package (Emsley and Cowtan, 2004), and then real-space refined with rigid body fitting and simulated annealing using the PHENIX package (Afonine et al., 2012) (Key Resources Table). This refined model was used as input for a multi-model-generating pipeline (Herzik et al., 2018) that aided in assessing the quality of the model and the map for mechanistic interpretation (Figure S1F). Briefly, the refined model and the map were used to generate 200 models in Rosetta, and the top ten scoring models were selected for further refinement in Phenix. The per-residue Cα root mean square deviation (RMSD) of top ten models was calculated and used to identify regions with poor model convergence. These regions were inspected and re-modeled and re-refined. Regions of the map with poor convergence either had the side-chains were truncated to the Cβ or removed entirely from the model.

UCSF Chimera (Goddard et al., 2007) was used to generate the figures (Key Resources Table).

Cell Culture and Mitochondrial isolationHEK293T cells (Key Resources Table) were cultured in DMEM (Cellgro) supplemented with 10% fetal bovine serum (Cellgro), 1% penicillin/streptavidin (Gibco) and 1% L-glutamine (Cellgro) at 37°C (5% CO 2). Cells were transiently transfected with pcDNA3.1 vector expressing C-terminally Flag-tagged AFG3L2 (GenScript) with the indicated single-point mutations (Key Resources Table). After 36h, mitochondria were isolated from cells as previously reported (Haynes et al., 2010).

Cell Culture and Mitochondrial isolation

HEK293T cells (Key Resources Table) were cultured in DMEM (Cellgro) supplemented with 10% fetal bovine serum (Cellgro), 1% penicillin/streptavidin (Gibco) and 1% L-glutamine (Cellgro) at 37°C (5% CO 2). Cells were transiently transfected with pcDNA3.1 vector expressing C-terminally Flag-tagged AFG3L2 (GenScript) with the indicated single-point mutations (Key Resources Table). After 36h, mitochondria were isolated from cells as previously reported (Haynes et al., 2010).

SDS- and BN-PAGE analysisFreshly isolated mitochondria were lysed in SDS (20 mM Hepes pH 7.5, 100 mM NaCl, 1 mM EDTA, 1% Triton supplemented with EDTA-free protease inhibitors (Roche)) or BN-Lysis buffer (20 mM Tris pH 7.5, 10% w/v glycerol, 50 mM NaCl, 0.1 mM EDTA, and 1mM PMSF supplemented with 1% n-dodecyl-β-D-maltoside). Samples were normalized by total protein concentration using Bio-Rad protein quantification. For SDS-PAGE, lysates were separated on 10 % acrylamide gels and transferred onto nitrocellulose membranes (Bio-Rad) for immunoblotting. Following incubation with primary antibodies, membranes were incubated with IR-labeled secondary antibodies (Li-COR Biosciences) and analyzed using the Odyssey Infrared Imaging System (Li-COR Biosciences). BN-PAGE samples were separated on 4–16% polyacrylamide gels, and western blotted as described (Geissler et al., 2002). Primary antibodies: anti-AFG3L2 (Genetex, 1:1000 dilution), anti-Flag (M2 Flag, Sigma, 1:1000 dilution), anti-Tubulin (Sigma, 1:10000 dilution) (Key Resources Table).

SDS- and BN-PAGE analysis

Freshly isolated mitochondria were lysed in SDS (20 mM Hepes pH 7.5, 100 mM NaCl, 1 mM EDTA, 1% Triton supplemented with EDTA-free protease inhibitors (Roche)) or BN-Lysis buffer (20 mM Tris pH 7.5, 10% w/v glycerol, 50 mM NaCl, 0.1 mM EDTA, and 1mM PMSF supplemented with 1% n-dodecyl-β-D-maltoside). Samples were normalized by total protein concentration using Bio-Rad protein quantification. For SDS-PAGE, lysates were separated on 10 % acrylamide gels and transferred onto nitrocellulose membranes (Bio-Rad) for immunoblotting. Following incubation with primary antibodies, membranes were incubated with IR-labeled secondary antibodies (Li-COR Biosciences) and analyzed using the Odyssey Infrared Imaging System (Li-COR Biosciences). BN-PAGE samples were separated on 4–16% polyacrylamide gels, and western blotted as described (Geissler et al., 2002). Primary antibodies: anti-AFG3L2 (Genetex, 1:1000 dilution), anti-Flag (M2 Flag, Sigma, 1:1000 dilution), anti-Tubulin (Sigma, 1:10000 dilution) (Key Resources Table).

QUANTIFICATION AND STATISTICAL ANALYSISStatistical details of all experiments can be found in the relevant figure legends. The biochemical data reflect the average of at least three independent experiments, and the error bars are the standard deviations generated from each group of data. Student’s two-tailed t test were performed to indicate the statistical significance of our data. Statistical details of all experiments can be found in the relevant figure legends.

QUANTIFICATION AND STATISTICAL ANALYSIS

Statistical details of all experiments can be found in the relevant figure legends. The biochemical data reflect the average of at least three independent experiments, and the error bars are the standard deviations generated from each group of data. Student’s two-tailed t test were performed to indicate the statistical significance of our data. Statistical details of all experiments can be found in the relevant figure legends.

DATA AND MATERIALS AVAILABILITYAll the cryo-EM maps were deposited in the Electron Microscopy Data Bank under accession code EMD-0552. The associated atomic coordinates were deposited into the Protein Data Bank with accession code 6NYY.

DATA AND MATERIALS AVAILABILITY

All the cryo-EM maps were deposited in the Electron Microscopy Data Bank under accession code EMD-0552. The associated atomic coordinates were deposited into the Protein Data Bank with accession code 6NYY.

Supplementary Material1

Supplementary Material
