---
pmid: '20534433'
title: Synaptonemal complex formation and meiotic checkpoint signaling are linked
  to the lateral element protein Red1.
authors:
- Eichinger CS
- Jentsch S
journal: Proc Natl Acad Sci U S A
year: '2010'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2895081
doi: 10.1073/pnas.1004248107
---

# Synaptonemal complex formation and meiotic checkpoint signaling are linked to the lateral element protein Red1.
**Authors:** Eichinger CS, Jentsch S
**Journal:** Proc Natl Acad Sci U S A (2010)
**DOI:** [10.1073/pnas.1004248107](https://doi.org/10.1073/pnas.1004248107)
**PMC:** [PMC2895081](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2895081/)

## Abstract

1. Proc Natl Acad Sci U S A. 2010 Jun 22;107(25):11370-5. doi: 
10.1073/pnas.1004248107. Epub 2010 Jun 3.

Synaptonemal complex formation and meiotic checkpoint signaling are linked to 
the lateral element protein Red1.

Eichinger CS(1), Jentsch S.

Author information:
(1)Department of Molecular Cell Biology, Max Planck Institute of Biochemistry, 
82152 Martinsried, Germany.

Meiosis generates four haploid daughters from a diploid parental cell. Central 
steps of meiosis are the pairing and recombination of homologous chromosomes 
followed by their segregation in two rounds of cell division. Meiotic 
recombination is monitored by a specialized DNA damage checkpoint pathway and is 
guided by a unique chromosomal structure called synaptonemal complex (SC), but 
how these events are coordinated is unclear. Here, we identify the SC protein 
Red1 as a crucial regulator of early meiosis. Red1 interacts with two subunits 
of the 9-1-1 checkpoint complex via two distinct 9-1-1 subunit-specific motifs. 
Association of 9-1-1 with Red1 is essential not only for meiotic checkpoint 
activation but for SC formation. Moreover, Red1 becomes SUMO-modified, which 
fosters interaction of Red1 with the central SC element Zip1, thereby securing 
timely SC formation. Thus, Red1, in addition to its structural role in the SC, 
is a crucial coordinator of meiosis by coupling checkpoint signaling to SC 
formation.

DOI: 10.1073/pnas.1004248107
PMCID: PMC2895081
PMID: 20534433 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no conflict of interest.

## Full Text

Abstract

Meiosis generates four haploid daughters from a diploid parental cell. Central steps of meiosis are the pairing and recombination of homologous chromosomes followed by their segregation in two rounds of cell division. Meiotic recombination is monitored by a specialized DNA damage checkpoint pathway and is guided by a unique chromosomal structure called synaptonemal complex (SC), but how these events are coordinated is unclear. Here, we identify the SC protein Red1 as a crucial regulator of early meiosis. Red1 interacts with two subunits of the 9-1-1 checkpoint complex via two distinct 9-1-1 subunit-specific motifs. Association of 9-1-1 with Red1 is essential not only for meiotic checkpoint activation but for SC formation. Moreover, Red1 becomes SUMO-modified, which fosters interaction of Red1 with the central SC element Zip1, thereby securing timely SC formation. Thus, Red1, in addition to its structural role in the SC, is a crucial coordinator of meiosis by coupling checkpoint signaling to SC formation.

Discussion

In this study, we uncovered detailed molecular mechanisms for two major functions of meiosis: meiotic checkpoint signaling and SC formation. Both processes are directly functionally linked to the lateral element protein Red1 through covalent and noncovalent protein-protein interactions. Whereas the bulk of Red1 appears to play a structural role within the SC, only a small subfraction of Red1 is apparently involved in signaling. A central finding of this work is that Red1 acts as a downstream effector of 9-1-1 in meiotic checkpoint signaling. Intriguingly, Red1 appears to associate with two subunits of the ring-shaped heterotrimeric complex, by which means undesired interactions of 9-1-1 (e.g., with Dpb11/TopBP1) are perhaps effectively blocked. Also interesting is the finding that 9-1-1 binding involves sequences reminiscent of PIP boxes, which are usually found in PIPs. Importantly, however, although PCNA and 9-1-1 are structurally highly related ( 3 ), these regions of Red1 bind 9-1-1 but not PCNA, thereby confirming that the two related clamps operate in distinct pathways. Moreover, the two binding sites in Red1 are 9-1-1 subunit-specific, suggesting that different 9-1-1 subunits may perhaps play distinct roles in DNA damage pathways. Although we identified the binding of Red1 with 9-1-1 by two-hybrid assays, we assume that this interaction is direct and occurs in vivo, given the subunit-specific phenotypes of the 9-1-1–deficient Red1 mutants and the presence of the two 9-1-1 subunit-specific PIP box-like sequences in Red1.

Because we found that intact 9-1-1 binding sites in Red1 are essential for 9-1-1–dependent meiotic checkpoint signaling, we propose that the SC protein Red1 may take the position of TopBP1/Dpb11 in the meiotic checkpoint pathway ( Fig. S6 ). Interestingly, the interaction essential for checkpoint signaling for both Red1 and TopBP1/Dpb11 is the Ddc1 subunit of 9-1-1, suggesting that the two proteins might indeed be alternative 9-1-1 binding partners. This model has important implications. Linking 9-1-1 to Red1 may further trigger downstream signaling events that take place at the SC, such as the phosphorylation of Red1 and Hop1, which, in turn, may recruit the kinase Mek1, which might recognize the phosphorylated forms of the proteins by means of its forkhead-associated domain. Notably, Mek1 plays a major role in ensuring interhomolog vs. intersister recombination ( 36 ), perhaps partially through its coupling to the SC. This phenomenon, often called “barrier-to-sister chromatid recombination,” leads to high crossover rates and high spore viabilities. Our finding that experimental disruption of 9-1-1–Red1 interaction leads to a high number of inviable spores supports this model.

Another advantage of a 9-1-1–Red1 interaction is the possibility of connecting checkpoint signaling to SC formation. Mutants defective in checkpoint signaling are known to be at least partially defective in SC formation ( 17 , 37 , 38 ). Our data now show that this functional interconnection is brought about through the physical 9-1-1–Red1 interaction. In fact, the finding that the very same mutant ( red1 −Ddc1 ) is completely defective in both processes shows how exquisitely these two essential meiotic events are coupled.

We also discovered that Red1 is subject to another layer of control. We found that yeast Red1 is SUMOylated during early meiosis concomitantly with the appearance of SCs. Interestingly, when we looked in human 293T cells that express tagged versions of either SUMO1 or SUMO2 (humans possess three SUMO variants), we found that Sycp3 (alias Scp3, Cor1), a possible functional analogue of yeast Red1 ( 39 ), which is known to bind human Ubc9 ( 40 ), is modified specifically by SUMO2 ( Fig. S7 ). Thus, SUMOylation of lateral element SC proteins (Red1 and Sycp3) appears to be a conserved feature. Interestingly, a SIM-containing C-terminal fragment of the central element protein Zip1 from yeast specifically interacts with the SUMO-modified form of Red1, suggesting that SC zipping is stimulated by Red1 SUMOylation. This finding draws interesting parallels to other SUMO-stimulated interactions, for example, the binding of PCNA to the SIM-containing Srs2 helicase, an interaction that is greatly stimulated by PCNA SUMOylation ( 41 ). Notably, strongly reduced Red1 SUMOylation (in red1 KR cells) causes a delay in complete SC formation by several hours (without the occurrence of any polycomplexes), which, in turn, causes reduced spore viability. Interestingly, this effect on SC assembly by the red1 KR mutant is much milder than the effect observed in zip3 deletion strains or in zip1 mutants defective in SUMO binding ( 29 ). This suggests that the putative SUMO ligase Zip3 and the SIM motif of Zip1 may have additional roles in meiosis.

An interesting open question is whether SUMO-modified Red1 recruits Zip1 along the entire chromosome. The staining of whole chromosomes with anti-SUMO antibodies indeed suggests a broad role for SUMO in these complexes ( 28 ), perhaps as a “zipping glue.” However, the low steady-state level of modified Red1 and our finding that Red1 SUMOylation triggers SC formation timing (rather than SC formation itself) argue for a regulatory role instead. In our model, Red1-SUMO initiates zipping by helping to recruit the initial Red1-Zip1 associations, perhaps analogous to the role of the ends of zippers in clothing. Firm interaction between the SC elements may then be completed by poly-SUMO chains that seem to bind both Zip1 and Red1 ( 34 ). Given the recent finding that Red1 binds SUMO chains via two SIMs in the C-terminal domain between residues 710 and 770 ( 34 ), SC assembly or disassembly might involve an intramolecular interaction between SUMO chains added to Red1’s lysine-rich region and the two SIMs at its C terminus. Notably, Red1 SUMOylation is stimulated by the putative meiotic ligase Zip3, which is known to associate with the MRX complex and concentrates at the first meiotic DSBs ( 42 ). Zip3 also binds the central element SC protein Zip1 ( 42 ), and these proteins, together with SUMO, first localize to recombination sites before Zip1 and SUMO spread along the entire chromosome ( 28 ). Thus, Zip3 may SUMOylate only Red1 molecules that are close to the DSBs, thereby perhaps stimulating the first specific Red1-Zip1 interactions. Because Zip3 also associates with the recombinase Rad51 ( 42 ), the SUMO ligase might be guided by homology search to the homologous chromosome and initiate SUMOylation of Red1 also at the recipient homologous chromosome. This model seems particularly attractive because it suggests a mechanism of how the information for timely zipping is transmitted from one chromosome to its homolog.

Materials and Methods

Yeast strains used are listed in Table S2 . Strains are isogenic to strain SK1 (Y1083, for all meiotic assays), DF5 ( Fig. 4 D ), or PJ69-7A (for all yeast two-hybrid assays). Detailed methods for the construction of mutants and strains, synchronous meiotic time course experiments, spore viability, germination and meiotic nuclear division assays, live cell microscopy, and protein techniques can be found in SI Text .
