---
pmid: '21299499'
title: 'IQ-motif selectivity in human IQGAP2 and IQGAP3: binding of calmodulin and
  myosin essential light chain.'
authors:
- Atcheson E
- Hamilton E
- Pathmanathan S
- Greer B
- Harriott P
- Timson DJ
journal: Biosci Rep
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3263943
doi: 10.1042/BSR20100123
---

# IQ-motif selectivity in human IQGAP2 and IQGAP3: binding of calmodulin and myosin essential light chain.
**Authors:** Atcheson E, Hamilton E, Pathmanathan S, Greer B, Harriott P, Timson DJ
**Journal:** Biosci Rep (2011)
**DOI:** [10.1042/BSR20100123](https://doi.org/10.1042/BSR20100123)
**PMC:** [PMC3263943](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3263943/)

## Abstract

1. Biosci Rep. 2011 Oct;31(5):371-9. doi: 10.1042/BSR20100123.

IQ-motif selectivity in human IQGAP2 and IQGAP3: binding of calmodulin and 
myosin essential light chain.

Atcheson E(1), Hamilton E(1), Pathmanathan S(1), Greer B(1), Harriott P(1), 
Timson DJ(1).

Author information:
(1)School of Biological Sciences, Queen's University Belfast, Medical Biology 
Centre, 97 Lisburn Road, Belfast BT9 7BL, U.K.

The IQGAP [IQ-motif-containing GAP (GTPase-activating protein)] family members 
are eukaryotic proteins that act at the interface between cellular signalling 
and the cytoskeleton. As such they collect numerous inputs from a variety of 
signalling pathways. A key binding partner is the calcium-sensing protein CaM 
(calmodulin). This protein binds mainly through a series of IQ-motifs which are 
located towards the middle of the primary sequence of the IQGAPs. In some 
IQGAPs, these motifs also provide binding sites for CaM-like proteins such as 
myosin essential light chain and S100B. Using synthetic peptides and native gel 
electrophoresis, the binding properties of the IQ-motifs from human IQGAP2 and 
IQGAP3 have been mapped. The second and third IQ-motifs in IQGAP2 and all four 
of the IQ-motifs of IQGAP3 interacted with CaM in the presence of calcium ions. 
However, there were differences in the type of interaction: while some IQ-motifs 
were able to form complexes with CaM which were stable under the conditions of 
the experiment, others formed more transient interactions. The first IQ-motifs 
from IQGAP2 and IQGAP3 formed transient interactions with CaM in the absence of 
calcium and the first motif from IQGAP3 formed a transient interaction with the 
myosin essential light chain Mlc1sa. None of these IQ-motifs interacted with 
S100B. Molecular modelling suggested that all of the IQ-motifs, except the first 
one from IQGAP2 formed α-helices in solution. These results extend our knowledge 
of the selectivity of IQ-motifs for CaM and related proteins.

DOI: 10.1042/BSR20100123
PMCID: PMC3263943
PMID: 21299499 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONThe IQ-motif is present in a wide variety of proteins that bind to CaM (calmodulin) and structurally related proteins [1–3]. The consensus sequence for the motif is generally accepted to be IQXXXRGXXXR, where X is any amino acid [2]. Nevertheless the motif shows considerable variation, even in the five consensus residues. Sequences containing the motif are usually α-helical in conformation and CaM (and similar proteins) bind to this motif by partially wrapping around this α-helix. Multiple IQ-motifs are present in most members of the myosin superfamily where they mediate interactions between CaM and various types of myosin light chain [4,5].CaM is a widely distributed intracellular sensor of calcium ions. It binds to a wide variety of cellular targets in order to transduce calcium-based signals. A key group of targets in signalling to the cytoskeleton are the IQGAP [IQ-motif-containing GAP (GTPase-activating protein)] family of proteins [6]. These relatively large proteins (150–180 kDa) sit at the interface between cellular signalling and cytoskeletal reorganization [7–12]. They bind to a variety of signalling molecules (including kinases, growth factor receptors and small GTPases [11]), integrate the information received from them and pass it on to the cytoskeleton. This is achieved by direct interaction with actin [13], by interaction with actin-organizing proteins such as N-WASP (neuronal Wiskott–Aldrich syndrome protein) [14] and by regulation of microtubule-organizing proteins such as CLIP-170 (cytoplasmic linker protein-170) and the APC (adenomatous polyposis coli) protein [15,16].Members of the IQGAP family are multi-domain proteins (Figure 1a). At the N-terminus is the calponin homology domain that interacts with actin (and may also bind to CaM) [13,17–20]. The C-terminal GAP-related domain interacts with, and inhibits, the small GTPases CDC42 (cell division cycle 42) and Rac1 [6,19,21–26]. The C-terminal region also interacts with CLIP-170 and APC [15,16]. A number of IQ-motifs are located approximately in the middle of the primary sequence. The exact number varies according to isoform and species. In the three human isoforms, IQGAP1, IQGAP2 and IQGAP3, there are four, except in an alternatively spliced testicular form of IQGAP2 which has only three [6,12,22,27,28]. The situation in the budding yeast (Saccharomyces cerevisiae) IQGAP protein, Iqg1p, is more complicated. Here, difficulty in identifying IQ-motifs has lead to different numbers of IQ-motifs being proposed. There appear to be at least eight, but some authors suggest that there may be as few as five or as many as 11 [29–31].Figure 1The general primary structure of IQGAP family members and the peptides used in the present paper(a) A schematic diagram of the motifs found in IQGAP family proteins (not to scale). CHD, calponin homology domain; WW, WW domain; IQ, IQ-motif containing region; GRD, GAP-related domain; RGC, Ras-GAP C-terminal domain. (b) The peptide sequences used in the present study. Sequences derived from IQGAP2 are designated 2IQ1, 2IQ2, 2IQ3 and 2IQ4 in the order they appear in the primary sequence. Similarly, those derived from IQGAP3 are designated 3IQ1, 3IQ2, 3IQ3 and 3IQ4. The diagnostic, commonly conserved residues of IQ-motifs are highlighted in bold. Note that, as seen here, there is considerable variation in many IQ-motifs, including at these key residues.The IQ-motifs of the IQGAP family interact with CaM [22,32]. However, it appears that not all of them interact under the same conditions. For example, some IQ-motifs in human IQGAP1 are occupied with CaM only in the presence of calcium ions, whereas others are able to interact regardless of the calcium concentration [32,33]. Some IQGAPs, including human IQGAP1, S. cerevisiae Iqg1p and Schizosaccharomyces pombe Rng2p interact with myosin essential light chains [24,30,34–36]. The physiological significance of this interaction in humans is unknown, but in yeasts it is required for the successful completion of cytokinesis [20,30,31,37–43]. Human IQGAP1 also interacts, through its IQ-motifs, with S100B [35,44]. This interaction is important in the cytoskeleton-mediated rearrangement of cell membranes [44].In order to understand the IQGAPs' overall function as signal integrators, it is necessary to understand what binds, where it binds and under what conditions it binds. Although considerable work has been done to understand the interactions of human IQGAP1 with CaM (e.g. [9,32,33]), relatively little work has been done on IQGAP2 and IQGAP3. Here, the interaction of IQ-motifs in these two proteins with CaM and myosin essential light chain was mapped using synthetic peptides corresponding to the IQ-motifs (Figure 1b) and native gel electrophoresis.

INTRODUCTION

The IQ-motif is present in a wide variety of proteins that bind to CaM (calmodulin) and structurally related proteins [1–3]. The consensus sequence for the motif is generally accepted to be IQXXXRGXXXR, where X is any amino acid [2]. Nevertheless the motif shows considerable variation, even in the five consensus residues. Sequences containing the motif are usually α-helical in conformation and CaM (and similar proteins) bind to this motif by partially wrapping around this α-helix. Multiple IQ-motifs are present in most members of the myosin superfamily where they mediate interactions between CaM and various types of myosin light chain [4,5].

CaM is a widely distributed intracellular sensor of calcium ions. It binds to a wide variety of cellular targets in order to transduce calcium-based signals. A key group of targets in signalling to the cytoskeleton are the IQGAP [IQ-motif-containing GAP (GTPase-activating protein)] family of proteins [6]. These relatively large proteins (150–180 kDa) sit at the interface between cellular signalling and cytoskeletal reorganization [7–12]. They bind to a variety of signalling molecules (including kinases, growth factor receptors and small GTPases [11]), integrate the information received from them and pass it on to the cytoskeleton. This is achieved by direct interaction with actin [13], by interaction with actin-organizing proteins such as N-WASP (neuronal Wiskott–Aldrich syndrome protein) [14] and by regulation of microtubule-organizing proteins such as CLIP-170 (cytoplasmic linker protein-170) and the APC (adenomatous polyposis coli) protein [15,16].

Members of the IQGAP family are multi-domain proteins (Figure 1a). At the N-terminus is the calponin homology domain that interacts with actin (and may also bind to CaM) [13,17–20]. The C-terminal GAP-related domain interacts with, and inhibits, the small GTPases CDC42 (cell division cycle 42) and Rac1 [6,19,21–26]. The C-terminal region also interacts with CLIP-170 and APC [15,16]. A number of IQ-motifs are located approximately in the middle of the primary sequence. The exact number varies according to isoform and species. In the three human isoforms, IQGAP1, IQGAP2 and IQGAP3, there are four, except in an alternatively spliced testicular form of IQGAP2 which has only three [6,12,22,27,28]. The situation in the budding yeast (Saccharomyces cerevisiae) IQGAP protein, Iqg1p, is more complicated. Here, difficulty in identifying IQ-motifs has lead to different numbers of IQ-motifs being proposed. There appear to be at least eight, but some authors suggest that there may be as few as five or as many as 11 [29–31].

The general primary structure of IQGAP family members and the peptides used in the present paper

(a) A schematic diagram of the motifs found in IQGAP family proteins (not to scale). CHD, calponin homology domain; WW, WW domain; IQ, IQ-motif containing region; GRD, GAP-related domain; RGC, Ras-GAP C-terminal domain. (b) The peptide sequences used in the present study. Sequences derived from IQGAP2 are designated 2IQ1, 2IQ2, 2IQ3 and 2IQ4 in the order they appear in the primary sequence. Similarly, those derived from IQGAP3 are designated 3IQ1, 3IQ2, 3IQ3 and 3IQ4. The diagnostic, commonly conserved residues of IQ-motifs are highlighted in bold. Note that, as seen here, there is considerable variation in many IQ-motifs, including at these key residues.

The IQ-motifs of the IQGAP family interact with CaM [22,32]. However, it appears that not all of them interact under the same conditions. For example, some IQ-motifs in human IQGAP1 are occupied with CaM only in the presence of calcium ions, whereas others are able to interact regardless of the calcium concentration [32,33]. Some IQGAPs, including human IQGAP1, S. cerevisiae Iqg1p and Schizosaccharomyces pombe Rng2p interact with myosin essential light chains [24,30,34–36]. The physiological significance of this interaction in humans is unknown, but in yeasts it is required for the successful completion of cytokinesis [20,30,31,37–43]. Human IQGAP1 also interacts, through its IQ-motifs, with S100B [35,44]. This interaction is important in the cytoskeleton-mediated rearrangement of cell membranes [44].

In order to understand the IQGAPs' overall function as signal integrators, it is necessary to understand what binds, where it binds and under what conditions it binds. Although considerable work has been done to understand the interactions of human IQGAP1 with CaM (e.g. [9,32,33]), relatively little work has been done on IQGAP2 and IQGAP3. Here, the interaction of IQ-motifs in these two proteins with CaM and myosin essential light chain was mapped using synthetic peptides corresponding to the IQ-motifs (Figure 1b) and native gel electrophoresis.

MATERIALS AND METHODSProtein expression and purificationThe myosin essential light chain, Mlc1sa and S100B were prepared as previously described [35]. The coding sequence for human CaM was amplified by PCR from the IMAGE clone (ID 2821489) [45] using primers that incorporated NcoI and XhoI restriction enzyme sites at the 5′ and 3′ ends respectively. This sequence was cloned into the respective sites in pET-21d (Novagen) such that the coding sequence was in frame with the His6 tag-encoding sequence in the vector. The sequence of the coding sequence was verified (MWG Biotech). Thus the expressed protein incorporated the amino acid residues LEHHHHHH at the C-terminus in addition to the full-length sequence of human CaM. Escherichia coli Rosetta (DE3) cells were transformed with this expression vector. A single colony was picked and grown, overnight, shaking at 30°C in LB (Luria–Bertani) broth supplemented with 100 μg/ml ampicillin and 100 μg/ml chloramphenicol. This culture was diluted into 1 litre of LB broth (supplemented with 100 μg/ml ampicillin and 100 μg/ml chloramphenicol) and grown, shaking, at 30°C until the D600 reached 0.6–1.0 (approx. 3–4 h). The culture was then induced by addition of IPTG (isopropyl β-D-thiogalactoside) to a final concentration of 2 mM and grown for a further 3 h. The cells were harvested by centrifugation (4000 g for 15 min), resuspended in buffer R [50 mM Hepes-OH, pH 7.5, 150 mM sodium chloride and 10% (v/v) glycerol] and stored at −80°C until required.Cell resuspensions were thawed and then sonicated on ice (three pulses of 100 W for 30 s with 30–60 s gaps in between for cooling) to release the proteins. The insoluble matter was pelleted by centrifugation (22000 g for 15 min). The supernatant was made of 0.5 mM in calcium chloride and then immediately applied to a phenyl-Sepharose column (3 ml; Sigma), which had been previously equilibrated in 50 mM Hepes-OH, pH 7.5, 5 mM calcium chloride, 100 mM sodium chloride and 0.5 mM DTT (dithiothreitol). The protein was allowed to flow through under gravity. The column was washed with 20 ml of 50 mM Hepes-OH, pH 7.5, 0.1 mM calcium chloride, 100 mM sodium chloride followed by 50 mM Hepes-OH, pH 7.5, 0.1 mM calcium chloride and 500 mM sodium chloride. CaM was eluted with three 2 ml washes with 50 mM Hepes-OH, pH 7.5, 1 mM EGTA. CaM-containing fractions were identified by SDS/PAGE (15% gel) and dialysed against buffer R supplemented with 2 mM DTT. Protein concentrations were determined using the method of Bradford [46] with BSA as a standard. Proteins were stored in 50–100 μl aliquots at −80°C.Peptide synthesisPeptides corresponding to the IQ-motifs from IQGAP2 were designated 2IQ1, 2IQ2, 2IQ3 and 2IQ4 in the order in which they occur in the primary sequence. Similar nomenclature was used for those corresponding to IQGAP3 (Figure 1b). Peptides were synthesized by solid-phase peptide synthesis using standard Fmoc (fluoren-9-ylmethoxycarbonyl) chemistry [47] at a scale of 20 μmol using a 433A Peptide Synthesizer (Applera). Each peptide was synthesized on a Rink Amide MBHA resin (Merck Biosciences) to produce a free amide upon cleavage and was then capped at the N-terminus using acetic anhydride. Peptides were elongated using 2-(1H-7-azabenzotriazol-1-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate methanaminium (HATU) and DIEA (N,N-di-isopropylethylamine) as coupling agents and 5-fold molar excesses of each N-α-Fmoc-protected amino acid (Merck). The synthesized peptides were cleaved from the resin by stirring for 2.5 h at 20°C in TFA (trifluoroacetic acid)/ethanedithiol/tri-isopropylsilane/water (97:1:1:1, by vol.), then separated from the resin by filtration under pressure using a glass-sintered funnel and precipitated with diethyl ether several times. This pellet was then resuspended in water and lyophilized. Peptide purity was then analysed by reverse-phase HPLC using a Gemini-C18 column of dimensions 250 mm×4.6 mm (Phenomenex) and eluted with a linear gradient of 0.05% TFA in acetonitrile and 0.05% TFA in water from a ratio of 1:49 to 3:2, at a flow rate of 1 ml/min with UV detection at a wavelength of 214 nm. The molecular masses of the peptides were verified using a Finnigan LCQ Ion Trap Mass Spectrometer (Thermo Scientific).Native gel electrophoresisProtein–peptide interactions were detected by native gel electrophoresis. Reaction mixes (35 μl) contained protein at a fixed concentration of 35 μM and peptide concentrations varying from 18 to 350 μM. These mixes (together with protein-only and peptide-only controls) were incubated at 22°C for 1 h, mixed with 15 μl of loading buffer [125 mM Tris/HCl, pH 6.8, 20% (v/v) glycerol, 1% (w/v) DTT and 0.002% (w/v) Bromophenol Blue] and then loaded on to native gels (20 cm×20 cm; 1 mm thick). The gels comprised a stacking gel [5% (w/v) polyacrylamide and 125 mM Tris/HCl, pH 6.8] and a separating gel of 15% (w/v) polyacrylamide in 375 mM Tris/HCl (pH 8.8). The gels were electrophoresed at 15–30 mA (constant current) for 4.5–8 h with 25 mM Tris/HCl (pH 8.8) and 250 mM glycine as the running buffer. On completion of electrophoresis, the gels were stained in 0.25% (w/v) Coomassie Brilliant Blue R250 (Sigma) dissolved in 10% (v/v) acetic acid/45% (v/v) ethanol and destained fully in 5% (v/v) ethanol; 7.5% (v/v) acetic acid. Gels were imaged using a ChemiDoc XRS gel documentation system (Bio-Rad).Where discrete shifts were detected, the fraction bound was estimated by densitometry using Quantity One software (Bio-Rad). Care was taken to ensure that no saturated pixels were present in the images used. The background density per unit area was estimated by selecting three to five random areas away from the bands. Since the bands were not of equal sizes, areas containing all of each of the bands were selected by eye and the density estimated. These densities were then normalized by dividing the area of the box and corrected by subtracting the mean background density per unit area. The fraction bound (Y) was then estimated by dividing the corrected, normalized intensity of the upper band (the protein–peptide complex) by the sum of the corrected, normalized intensities of the upper and lower (free protein) bands. The mean values of Y from three independent experiments were plotted against peptide concentration. The apparent dissociation constant (Kd,app) was estimated by non-linear curve fitting [48] (as implemented in GraphPad Prism 3.0, GraphPad Software, CA, U.S.A.) to the equation Y=Ymax[peptide]/Kd,app+[peptide], where Ymax is the maximum (limiting) value of Y.Peptide modellingThe peptide sequences were submitted to the Phyre online molecular modelling server (http://www.sbg.bio.ic.ac.uk/~phyre/) [49]. Since the peptides were shorter than the minimum length required by this program, they were extended with polyalanine. None of these additional alanine residues appeared in the final models. The highest ranked model was then computationally solvated and the energy of the resulting complex minimized using YASARA (http://www.yasara.org/minimizationserver.htm) [50]. The PDB files of the final, minimized and solvated models are provided as supplementary data (available at http://www.bioscirep.org/bsr/031/bsr0310371add.htm).

MATERIALS AND METHODS

Protein expression and purificationThe myosin essential light chain, Mlc1sa and S100B were prepared as previously described [35]. The coding sequence for human CaM was amplified by PCR from the IMAGE clone (ID 2821489) [45] using primers that incorporated NcoI and XhoI restriction enzyme sites at the 5′ and 3′ ends respectively. This sequence was cloned into the respective sites in pET-21d (Novagen) such that the coding sequence was in frame with the His6 tag-encoding sequence in the vector. The sequence of the coding sequence was verified (MWG Biotech). Thus the expressed protein incorporated the amino acid residues LEHHHHHH at the C-terminus in addition to the full-length sequence of human CaM. Escherichia coli Rosetta (DE3) cells were transformed with this expression vector. A single colony was picked and grown, overnight, shaking at 30°C in LB (Luria–Bertani) broth supplemented with 100 μg/ml ampicillin and 100 μg/ml chloramphenicol. This culture was diluted into 1 litre of LB broth (supplemented with 100 μg/ml ampicillin and 100 μg/ml chloramphenicol) and grown, shaking, at 30°C until the D600 reached 0.6–1.0 (approx. 3–4 h). The culture was then induced by addition of IPTG (isopropyl β-D-thiogalactoside) to a final concentration of 2 mM and grown for a further 3 h. The cells were harvested by centrifugation (4000 g for 15 min), resuspended in buffer R [50 mM Hepes-OH, pH 7.5, 150 mM sodium chloride and 10% (v/v) glycerol] and stored at −80°C until required.Cell resuspensions were thawed and then sonicated on ice (three pulses of 100 W for 30 s with 30–60 s gaps in between for cooling) to release the proteins. The insoluble matter was pelleted by centrifugation (22000 g for 15 min). The supernatant was made of 0.5 mM in calcium chloride and then immediately applied to a phenyl-Sepharose column (3 ml; Sigma), which had been previously equilibrated in 50 mM Hepes-OH, pH 7.5, 5 mM calcium chloride, 100 mM sodium chloride and 0.5 mM DTT (dithiothreitol). The protein was allowed to flow through under gravity. The column was washed with 20 ml of 50 mM Hepes-OH, pH 7.5, 0.1 mM calcium chloride, 100 mM sodium chloride followed by 50 mM Hepes-OH, pH 7.5, 0.1 mM calcium chloride and 500 mM sodium chloride. CaM was eluted with three 2 ml washes with 50 mM Hepes-OH, pH 7.5, 1 mM EGTA. CaM-containing fractions were identified by SDS/PAGE (15% gel) and dialysed against buffer R supplemented with 2 mM DTT. Protein concentrations were determined using the method of Bradford [46] with BSA as a standard. Proteins were stored in 50–100 μl aliquots at −80°C.

Protein expression and purification

The myosin essential light chain, Mlc1sa and S100B were prepared as previously described [35]. The coding sequence for human CaM was amplified by PCR from the IMAGE clone (ID 2821489) [45] using primers that incorporated NcoI and XhoI restriction enzyme sites at the 5′ and 3′ ends respectively. This sequence was cloned into the respective sites in pET-21d (Novagen) such that the coding sequence was in frame with the His6 tag-encoding sequence in the vector. The sequence of the coding sequence was verified (MWG Biotech). Thus the expressed protein incorporated the amino acid residues LEHHHHHH at the C-terminus in addition to the full-length sequence of human CaM. Escherichia coli Rosetta (DE3) cells were transformed with this expression vector. A single colony was picked and grown, overnight, shaking at 30°C in LB (Luria–Bertani) broth supplemented with 100 μg/ml ampicillin and 100 μg/ml chloramphenicol. This culture was diluted into 1 litre of LB broth (supplemented with 100 μg/ml ampicillin and 100 μg/ml chloramphenicol) and grown, shaking, at 30°C until the D600 reached 0.6–1.0 (approx. 3–4 h). The culture was then induced by addition of IPTG (isopropyl β-D-thiogalactoside) to a final concentration of 2 mM and grown for a further 3 h. The cells were harvested by centrifugation (4000 g for 15 min), resuspended in buffer R [50 mM Hepes-OH, pH 7.5, 150 mM sodium chloride and 10% (v/v) glycerol] and stored at −80°C until required.

Cell resuspensions were thawed and then sonicated on ice (three pulses of 100 W for 30 s with 30–60 s gaps in between for cooling) to release the proteins. The insoluble matter was pelleted by centrifugation (22000 g for 15 min). The supernatant was made of 0.5 mM in calcium chloride and then immediately applied to a phenyl-Sepharose column (3 ml; Sigma), which had been previously equilibrated in 50 mM Hepes-OH, pH 7.5, 5 mM calcium chloride, 100 mM sodium chloride and 0.5 mM DTT (dithiothreitol). The protein was allowed to flow through under gravity. The column was washed with 20 ml of 50 mM Hepes-OH, pH 7.5, 0.1 mM calcium chloride, 100 mM sodium chloride followed by 50 mM Hepes-OH, pH 7.5, 0.1 mM calcium chloride and 500 mM sodium chloride. CaM was eluted with three 2 ml washes with 50 mM Hepes-OH, pH 7.5, 1 mM EGTA. CaM-containing fractions were identified by SDS/PAGE (15% gel) and dialysed against buffer R supplemented with 2 mM DTT. Protein concentrations were determined using the method of Bradford [46] with BSA as a standard. Proteins were stored in 50–100 μl aliquots at −80°C.

Peptide synthesisPeptides corresponding to the IQ-motifs from IQGAP2 were designated 2IQ1, 2IQ2, 2IQ3 and 2IQ4 in the order in which they occur in the primary sequence. Similar nomenclature was used for those corresponding to IQGAP3 (Figure 1b). Peptides were synthesized by solid-phase peptide synthesis using standard Fmoc (fluoren-9-ylmethoxycarbonyl) chemistry [47] at a scale of 20 μmol using a 433A Peptide Synthesizer (Applera). Each peptide was synthesized on a Rink Amide MBHA resin (Merck Biosciences) to produce a free amide upon cleavage and was then capped at the N-terminus using acetic anhydride. Peptides were elongated using 2-(1H-7-azabenzotriazol-1-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate methanaminium (HATU) and DIEA (N,N-di-isopropylethylamine) as coupling agents and 5-fold molar excesses of each N-α-Fmoc-protected amino acid (Merck). The synthesized peptides were cleaved from the resin by stirring for 2.5 h at 20°C in TFA (trifluoroacetic acid)/ethanedithiol/tri-isopropylsilane/water (97:1:1:1, by vol.), then separated from the resin by filtration under pressure using a glass-sintered funnel and precipitated with diethyl ether several times. This pellet was then resuspended in water and lyophilized. Peptide purity was then analysed by reverse-phase HPLC using a Gemini-C18 column of dimensions 250 mm×4.6 mm (Phenomenex) and eluted with a linear gradient of 0.05% TFA in acetonitrile and 0.05% TFA in water from a ratio of 1:49 to 3:2, at a flow rate of 1 ml/min with UV detection at a wavelength of 214 nm. The molecular masses of the peptides were verified using a Finnigan LCQ Ion Trap Mass Spectrometer (Thermo Scientific).

Peptide synthesis

Peptides corresponding to the IQ-motifs from IQGAP2 were designated 2IQ1, 2IQ2, 2IQ3 and 2IQ4 in the order in which they occur in the primary sequence. Similar nomenclature was used for those corresponding to IQGAP3 (Figure 1b). Peptides were synthesized by solid-phase peptide synthesis using standard Fmoc (fluoren-9-ylmethoxycarbonyl) chemistry [47] at a scale of 20 μmol using a 433A Peptide Synthesizer (Applera). Each peptide was synthesized on a Rink Amide MBHA resin (Merck Biosciences) to produce a free amide upon cleavage and was then capped at the N-terminus using acetic anhydride. Peptides were elongated using 2-(1H-7-azabenzotriazol-1-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate methanaminium (HATU) and DIEA (N,N-di-isopropylethylamine) as coupling agents and 5-fold molar excesses of each N-α-Fmoc-protected amino acid (Merck). The synthesized peptides were cleaved from the resin by stirring for 2.5 h at 20°C in TFA (trifluoroacetic acid)/ethanedithiol/tri-isopropylsilane/water (97:1:1:1, by vol.), then separated from the resin by filtration under pressure using a glass-sintered funnel and precipitated with diethyl ether several times. This pellet was then resuspended in water and lyophilized. Peptide purity was then analysed by reverse-phase HPLC using a Gemini-C18 column of dimensions 250 mm×4.6 mm (Phenomenex) and eluted with a linear gradient of 0.05% TFA in acetonitrile and 0.05% TFA in water from a ratio of 1:49 to 3:2, at a flow rate of 1 ml/min with UV detection at a wavelength of 214 nm. The molecular masses of the peptides were verified using a Finnigan LCQ Ion Trap Mass Spectrometer (Thermo Scientific).

Native gel electrophoresisProtein–peptide interactions were detected by native gel electrophoresis. Reaction mixes (35 μl) contained protein at a fixed concentration of 35 μM and peptide concentrations varying from 18 to 350 μM. These mixes (together with protein-only and peptide-only controls) were incubated at 22°C for 1 h, mixed with 15 μl of loading buffer [125 mM Tris/HCl, pH 6.8, 20% (v/v) glycerol, 1% (w/v) DTT and 0.002% (w/v) Bromophenol Blue] and then loaded on to native gels (20 cm×20 cm; 1 mm thick). The gels comprised a stacking gel [5% (w/v) polyacrylamide and 125 mM Tris/HCl, pH 6.8] and a separating gel of 15% (w/v) polyacrylamide in 375 mM Tris/HCl (pH 8.8). The gels were electrophoresed at 15–30 mA (constant current) for 4.5–8 h with 25 mM Tris/HCl (pH 8.8) and 250 mM glycine as the running buffer. On completion of electrophoresis, the gels were stained in 0.25% (w/v) Coomassie Brilliant Blue R250 (Sigma) dissolved in 10% (v/v) acetic acid/45% (v/v) ethanol and destained fully in 5% (v/v) ethanol; 7.5% (v/v) acetic acid. Gels were imaged using a ChemiDoc XRS gel documentation system (Bio-Rad).Where discrete shifts were detected, the fraction bound was estimated by densitometry using Quantity One software (Bio-Rad). Care was taken to ensure that no saturated pixels were present in the images used. The background density per unit area was estimated by selecting three to five random areas away from the bands. Since the bands were not of equal sizes, areas containing all of each of the bands were selected by eye and the density estimated. These densities were then normalized by dividing the area of the box and corrected by subtracting the mean background density per unit area. The fraction bound (Y) was then estimated by dividing the corrected, normalized intensity of the upper band (the protein–peptide complex) by the sum of the corrected, normalized intensities of the upper and lower (free protein) bands. The mean values of Y from three independent experiments were plotted against peptide concentration. The apparent dissociation constant (Kd,app) was estimated by non-linear curve fitting [48] (as implemented in GraphPad Prism 3.0, GraphPad Software, CA, U.S.A.) to the equation Y=Ymax[peptide]/Kd,app+[peptide], where Ymax is the maximum (limiting) value of Y.

Native gel electrophoresis

Protein–peptide interactions were detected by native gel electrophoresis. Reaction mixes (35 μl) contained protein at a fixed concentration of 35 μM and peptide concentrations varying from 18 to 350 μM. These mixes (together with protein-only and peptide-only controls) were incubated at 22°C for 1 h, mixed with 15 μl of loading buffer [125 mM Tris/HCl, pH 6.8, 20% (v/v) glycerol, 1% (w/v) DTT and 0.002% (w/v) Bromophenol Blue] and then loaded on to native gels (20 cm×20 cm; 1 mm thick). The gels comprised a stacking gel [5% (w/v) polyacrylamide and 125 mM Tris/HCl, pH 6.8] and a separating gel of 15% (w/v) polyacrylamide in 375 mM Tris/HCl (pH 8.8). The gels were electrophoresed at 15–30 mA (constant current) for 4.5–8 h with 25 mM Tris/HCl (pH 8.8) and 250 mM glycine as the running buffer. On completion of electrophoresis, the gels were stained in 0.25% (w/v) Coomassie Brilliant Blue R250 (Sigma) dissolved in 10% (v/v) acetic acid/45% (v/v) ethanol and destained fully in 5% (v/v) ethanol; 7.5% (v/v) acetic acid. Gels were imaged using a ChemiDoc XRS gel documentation system (Bio-Rad).

Where discrete shifts were detected, the fraction bound was estimated by densitometry using Quantity One software (Bio-Rad). Care was taken to ensure that no saturated pixels were present in the images used. The background density per unit area was estimated by selecting three to five random areas away from the bands. Since the bands were not of equal sizes, areas containing all of each of the bands were selected by eye and the density estimated. These densities were then normalized by dividing the area of the box and corrected by subtracting the mean background density per unit area. The fraction bound (Y) was then estimated by dividing the corrected, normalized intensity of the upper band (the protein–peptide complex) by the sum of the corrected, normalized intensities of the upper and lower (free protein) bands. The mean values of Y from three independent experiments were plotted against peptide concentration. The apparent dissociation constant (Kd,app) was estimated by non-linear curve fitting [48] (as implemented in GraphPad Prism 3.0, GraphPad Software, CA, U.S.A.) to the equation Y=Ymax[peptide]/Kd,app+[peptide], where Ymax is the maximum (limiting) value of Y.

Peptide modellingThe peptide sequences were submitted to the Phyre online molecular modelling server (http://www.sbg.bio.ic.ac.uk/~phyre/) [49]. Since the peptides were shorter than the minimum length required by this program, they were extended with polyalanine. None of these additional alanine residues appeared in the final models. The highest ranked model was then computationally solvated and the energy of the resulting complex minimized using YASARA (http://www.yasara.org/minimizationserver.htm) [50]. The PDB files of the final, minimized and solvated models are provided as supplementary data (available at http://www.bioscirep.org/bsr/031/bsr0310371add.htm).

Peptide modelling

The peptide sequences were submitted to the Phyre online molecular modelling server (http://www.sbg.bio.ic.ac.uk/~phyre/) [49]. Since the peptides were shorter than the minimum length required by this program, they were extended with polyalanine. None of these additional alanine residues appeared in the final models. The highest ranked model was then computationally solvated and the energy of the resulting complex minimized using YASARA (http://www.yasara.org/minimizationserver.htm) [50]. The PDB files of the final, minimized and solvated models are provided as supplementary data (available at http://www.bioscirep.org/bsr/031/bsr0310371add.htm).

RESULTSExpression and purification of human CaM in E. coliInitial attempts to purify recombinant human CaM from extracts of E. coli cells using nickel–agarose resin to capture the C-terminal His6 tag were not successful. Typically the purifications resulted in two to three other proteins at comparable concentrations to CaM (results not shown). However, purification based on using phenyl-Sepharose, based on the method of Gopalakrishna and Anderson [51] was successful. This resulted in a single major band (>90% of eluted material) on SDS/PAGE (Supplementary Figure S1 available at http://www.bioscirep.org/bsr/031/bsr0310371add.htm) and yields of approx. 5 mg of CaM per litre of bacterial culture.Interaction of IQ-motifs from human IQGAP2 and IQGAP3 with CaMInitial screens for interaction with CaM (35 μM) were carried out with all eight peptides (350 μM) in the presence (5 mM calcium chloride) and absence (5 mM EGTA) of calcium ions. These revealed that 2IQ2, 2IQ3, 3IQ1, 3IQ2, 3IQ3 and 3IQ4 interacted in the presence of calcium ions under these conditions (Figure 2). The nature of these interactions was not identical. In some cases (2IQ3, 3IQ2, 3IQ3 and 3IQ4) discrete shifts were observed, suggesting a relatively long-lived stable complex is formed. Other IQ-motifs (2IQ2, 3IQ1) caused a blurring of the CaM band, suggesting that a more transient interaction has occurred. Fewer interactions were observed in the absence of calcium ions. Under these conditions, only 2IQ1 and perhaps 3IQ1 appeared to interact (Figure 2).Figure 2Interactions of IQ-motif peptides (350 μM) from IQGAP2 and IQGAP3 with CaM (35 μM) in the presence and absence of calcium ions under the conditions of native gel electrophoresis (see the Materials and methods section)These results are representative of three independent experiments.Where discrete, stable interactions were observed, these were quantified by titration of the peptide concentration and densitometric estimation of the fraction bound. It should be noted that the apparent dissociation constants obtained by this method will be highly influenced on the conditions of the experiment and are unlikely to be accurate estimates of the in vivo, cellular affinities. Nevertheless, they do enable the ranking of the affinities of interaction. It was not possible to quantify the interaction of 2IQ3 with CaM in the presence of calcium ions. Although this peptide caused a clear, dose-dependent shift in the migration of CaM this shift was a mixture of discrete bands and blurring (Figure 3). This suggests that this IQ-motif's behaviour is intermediate between those which bind stably and those which are in more rapid exchange. However, the interactions between CaM and 3IQ2, 3IQ3 and 3IQ4 in the presence of calcium could be quantified, revealing that the affinity is greatest for 3IQ2, followed by 3IQ4 and then 3IQ3 (Figure 4 and Table 1).Figure 3Interaction of the third IQ-motif from IQGAP2 with CaM (35 μM) in the presence of calcium ionsThe interaction shows a clear dependence of shift on peptide concentration. The numbers above the gel indicate the peptide concentration in μM. CaM represents the calmodulin-only (35 μM) control and Pep the peptide-only (350 μM) control. These results are representative of three independent experiments.Figure 4Interaction of IQ-motif peptides from IQGAP3 with CaM (35 μM)The left-hand panels show the gel shifts with the peptide concentrations given above in μM. CaM represents the calmodulin-only (35 μM) control and Pep the peptide-only (350 μM) control. The right-hand panels show graphs of the fraction bound (Y) against peptide concentration fitted to the binding equation Y=Ymax[peptide]/Kd,app+[peptide]. Each point represents the mean of three independent determinations (as described in the Materials and methods section) and the error bars the S.D. of these means. For each experiment, the goodness of fit was ascertained by comparing the fit to this equation with a fit to a straight line (F-test). In all cases, the better fit was to the binding equation. The quality of data was assessed by calculation of the residuals. In all cases, these were small and random.Table 1Apparent binding constants (Kd,app) for the interactions between IQ-motifs from human IQGAP3 and CaM in the presence of calcium ionsThese values were determined from native gel electrophoresis as described in the Materials and methods section and are shown±S.E.M. derived from the curve fitting. nd, not determined.IQ-motifKd,app (μM)3IQ1nd3IQ227±43IQ376±183IQ465±12Interaction of IQ-motifs from human IQGAP2 and IQGAP3 with myosin essential light chain and S100BPrevious work has demonstrated that IQGAP1 interacts, through its first IQ-motif, with the myosin essential light chain Mlc1sa and, through its first and second IQ-motifs, with S100B [35]. Increasing concentrations of 3IQ1 caused blurring of the Mlc1sa band in a native gel shift assay (Figure 5). The peptides 3IQ2 and 3IQ3 caused some loss of intensity of the Mlc1sa band at high concentrations without the appearance of a detectable additional band. No interactions were detected between Mlc1sa and 3IQ4 or with any of the IQ-motifs from IQGAP2 (Figure 5). Furthermore, no interactions between S100B and any of the IQ-motifs were detected (results not shown).Figure 5Interaction between the IQ-motifs of IQGAP2 and IQGAP3 and the myosin essential light chain, Mlc1sa(a) No interactions were detected between the IQ-motifs of IQGAP2 and Mlc1sa (concentrations indicated above the gel). (b) The first IQ-motif from IQGAP3 interacts with Mlc1sa. This interaction shows a clear dependence of shift on peptide concentration, but does not result in a discrete band corresponding to the peptide–Mlc1sa complex. For the second and third IQ-motifs, some blurring of the protein band was observed. No interaction between the fourth IQ-motif and Mlc1sa was detected. The numbers above the gels indicate the peptide concentration in μM. Mlc1sa represents the myosin essential light chain-only (35 μM) control and Pep the peptide-only (35 μM) control. These results are representative of three independent experiments.Structural models of IQ-motifs from human IQGAP2 and IQGAP3To help understand the differences in affinity between the various IQ-motifs and CaM, structural models were constructed. As expected, the majority of the motifs are predicted to adopt an α-helical conformation in solution. The one exception is 2IQ1, which adopts a disordered, random coil conformation (Figure 6). In addition, the model for 3IQ3 only contains approximately half the residues in an α-helical conformation. The remainder were not modelled by Phyre. Therefore, it is possible that this motif also has some disordered residues in solution. These are modelled structures and so should be treated with some caution. In particular, it is likely that the conformation of the motifs alters on interaction with targets. Nevertheless, these models suggest that the majority of the IQ-motifs that interact with CaM will do so in a manner similar to that observed in myosins (e.g. [4]).Figure 6Modelled structures of the IQ-motifs from human IQGAP2 and IQGAP3The peptide backbone is shown as a cartoon and the side chains in ball and stick format. In each case, the N-terminus of the peptide is in the lower left-hand corner of the structure. All except 2IQ2 display an α-helical conformation. 2IQ2 has a random coil conformation with no recognizable elements of secondary structure. For clarity, the models are shown in the absence of solvating water molecules. 3IQ3 appears shorter than the remainder as only part of this sequence could modelled by the Phyre server. This may indicate that this motif has a more flexible or random coil structure than the other IQ-motifs from IQGAP3.

Expression and purification of human CaM in E. coliInitial attempts to purify recombinant human CaM from extracts of E. coli cells using nickel–agarose resin to capture the C-terminal His6 tag were not successful. Typically the purifications resulted in two to three other proteins at comparable concentrations to CaM (results not shown). However, purification based on using phenyl-Sepharose, based on the method of Gopalakrishna and Anderson [51] was successful. This resulted in a single major band (>90% of eluted material) on SDS/PAGE (Supplementary Figure S1 available at http://www.bioscirep.org/bsr/031/bsr0310371add.htm) and yields of approx. 5 mg of CaM per litre of bacterial culture.

Expression and purification of human CaM in E. coli

Initial attempts to purify recombinant human CaM from extracts of E. coli cells using nickel–agarose resin to capture the C-terminal His6 tag were not successful. Typically the purifications resulted in two to three other proteins at comparable concentrations to CaM (results not shown). However, purification based on using phenyl-Sepharose, based on the method of Gopalakrishna and Anderson [51] was successful. This resulted in a single major band (>90% of eluted material) on SDS/PAGE (Supplementary Figure S1 available at http://www.bioscirep.org/bsr/031/bsr0310371add.htm) and yields of approx. 5 mg of CaM per litre of bacterial culture.

Interaction of IQ-motifs from human IQGAP2 and IQGAP3 with CaMInitial screens for interaction with CaM (35 μM) were carried out with all eight peptides (350 μM) in the presence (5 mM calcium chloride) and absence (5 mM EGTA) of calcium ions. These revealed that 2IQ2, 2IQ3, 3IQ1, 3IQ2, 3IQ3 and 3IQ4 interacted in the presence of calcium ions under these conditions (Figure 2). The nature of these interactions was not identical. In some cases (2IQ3, 3IQ2, 3IQ3 and 3IQ4) discrete shifts were observed, suggesting a relatively long-lived stable complex is formed. Other IQ-motifs (2IQ2, 3IQ1) caused a blurring of the CaM band, suggesting that a more transient interaction has occurred. Fewer interactions were observed in the absence of calcium ions. Under these conditions, only 2IQ1 and perhaps 3IQ1 appeared to interact (Figure 2).Figure 2Interactions of IQ-motif peptides (350 μM) from IQGAP2 and IQGAP3 with CaM (35 μM) in the presence and absence of calcium ions under the conditions of native gel electrophoresis (see the Materials and methods section)These results are representative of three independent experiments.Where discrete, stable interactions were observed, these were quantified by titration of the peptide concentration and densitometric estimation of the fraction bound. It should be noted that the apparent dissociation constants obtained by this method will be highly influenced on the conditions of the experiment and are unlikely to be accurate estimates of the in vivo, cellular affinities. Nevertheless, they do enable the ranking of the affinities of interaction. It was not possible to quantify the interaction of 2IQ3 with CaM in the presence of calcium ions. Although this peptide caused a clear, dose-dependent shift in the migration of CaM this shift was a mixture of discrete bands and blurring (Figure 3). This suggests that this IQ-motif's behaviour is intermediate between those which bind stably and those which are in more rapid exchange. However, the interactions between CaM and 3IQ2, 3IQ3 and 3IQ4 in the presence of calcium could be quantified, revealing that the affinity is greatest for 3IQ2, followed by 3IQ4 and then 3IQ3 (Figure 4 and Table 1).Figure 3Interaction of the third IQ-motif from IQGAP2 with CaM (35 μM) in the presence of calcium ionsThe interaction shows a clear dependence of shift on peptide concentration. The numbers above the gel indicate the peptide concentration in μM. CaM represents the calmodulin-only (35 μM) control and Pep the peptide-only (350 μM) control. These results are representative of three independent experiments.Figure 4Interaction of IQ-motif peptides from IQGAP3 with CaM (35 μM)The left-hand panels show the gel shifts with the peptide concentrations given above in μM. CaM represents the calmodulin-only (35 μM) control and Pep the peptide-only (350 μM) control. The right-hand panels show graphs of the fraction bound (Y) against peptide concentration fitted to the binding equation Y=Ymax[peptide]/Kd,app+[peptide]. Each point represents the mean of three independent determinations (as described in the Materials and methods section) and the error bars the S.D. of these means. For each experiment, the goodness of fit was ascertained by comparing the fit to this equation with a fit to a straight line (F-test). In all cases, the better fit was to the binding equation. The quality of data was assessed by calculation of the residuals. In all cases, these were small and random.Table 1Apparent binding constants (Kd,app) for the interactions between IQ-motifs from human IQGAP3 and CaM in the presence of calcium ionsThese values were determined from native gel electrophoresis as described in the Materials and methods section and are shown±S.E.M. derived from the curve fitting. nd, not determined.IQ-motifKd,app (μM)3IQ1nd3IQ227±43IQ376±183IQ465±12

Interaction of IQ-motifs from human IQGAP2 and IQGAP3 with CaM

Initial screens for interaction with CaM (35 μM) were carried out with all eight peptides (350 μM) in the presence (5 mM calcium chloride) and absence (5 mM EGTA) of calcium ions. These revealed that 2IQ2, 2IQ3, 3IQ1, 3IQ2, 3IQ3 and 3IQ4 interacted in the presence of calcium ions under these conditions (Figure 2). The nature of these interactions was not identical. In some cases (2IQ3, 3IQ2, 3IQ3 and 3IQ4) discrete shifts were observed, suggesting a relatively long-lived stable complex is formed. Other IQ-motifs (2IQ2, 3IQ1) caused a blurring of the CaM band, suggesting that a more transient interaction has occurred. Fewer interactions were observed in the absence of calcium ions. Under these conditions, only 2IQ1 and perhaps 3IQ1 appeared to interact (Figure 2).

Interactions of IQ-motif peptides (350 μM) from IQGAP2 and IQGAP3 with CaM (35 μM) in the presence and absence of calcium ions under the conditions of native gel electrophoresis (see the Materials and methods section)

These results are representative of three independent experiments.

Where discrete, stable interactions were observed, these were quantified by titration of the peptide concentration and densitometric estimation of the fraction bound. It should be noted that the apparent dissociation constants obtained by this method will be highly influenced on the conditions of the experiment and are unlikely to be accurate estimates of the in vivo, cellular affinities. Nevertheless, they do enable the ranking of the affinities of interaction. It was not possible to quantify the interaction of 2IQ3 with CaM in the presence of calcium ions. Although this peptide caused a clear, dose-dependent shift in the migration of CaM this shift was a mixture of discrete bands and blurring (Figure 3). This suggests that this IQ-motif's behaviour is intermediate between those which bind stably and those which are in more rapid exchange. However, the interactions between CaM and 3IQ2, 3IQ3 and 3IQ4 in the presence of calcium could be quantified, revealing that the affinity is greatest for 3IQ2, followed by 3IQ4 and then 3IQ3 (Figure 4 and Table 1).

Interaction of the third IQ-motif from IQGAP2 with CaM (35 μM) in the presence of calcium ions

The interaction shows a clear dependence of shift on peptide concentration. The numbers above the gel indicate the peptide concentration in μM. CaM represents the calmodulin-only (35 μM) control and Pep the peptide-only (350 μM) control. These results are representative of three independent experiments.

Interaction of IQ-motif peptides from IQGAP3 with CaM (35 μM)

The left-hand panels show the gel shifts with the peptide concentrations given above in μM. CaM represents the calmodulin-only (35 μM) control and Pep the peptide-only (350 μM) control. The right-hand panels show graphs of the fraction bound (Y) against peptide concentration fitted to the binding equation Y=Ymax[peptide]/Kd,app+[peptide]. Each point represents the mean of three independent determinations (as described in the Materials and methods section) and the error bars the S.D. of these means. For each experiment, the goodness of fit was ascertained by comparing the fit to this equation with a fit to a straight line (F-test). In all cases, the better fit was to the binding equation. The quality of data was assessed by calculation of the residuals. In all cases, these were small and random.

Apparent binding constants (Kd,app) for the interactions between IQ-motifs from human IQGAP3 and CaM in the presence of calcium ions

These values were determined from native gel electrophoresis as described in the Materials and methods section and are shown±S.E.M. derived from the curve fitting. nd, not determined.

Interaction of IQ-motifs from human IQGAP2 and IQGAP3 with myosin essential light chain and S100BPrevious work has demonstrated that IQGAP1 interacts, through its first IQ-motif, with the myosin essential light chain Mlc1sa and, through its first and second IQ-motifs, with S100B [35]. Increasing concentrations of 3IQ1 caused blurring of the Mlc1sa band in a native gel shift assay (Figure 5). The peptides 3IQ2 and 3IQ3 caused some loss of intensity of the Mlc1sa band at high concentrations without the appearance of a detectable additional band. No interactions were detected between Mlc1sa and 3IQ4 or with any of the IQ-motifs from IQGAP2 (Figure 5). Furthermore, no interactions between S100B and any of the IQ-motifs were detected (results not shown).Figure 5Interaction between the IQ-motifs of IQGAP2 and IQGAP3 and the myosin essential light chain, Mlc1sa(a) No interactions were detected between the IQ-motifs of IQGAP2 and Mlc1sa (concentrations indicated above the gel). (b) The first IQ-motif from IQGAP3 interacts with Mlc1sa. This interaction shows a clear dependence of shift on peptide concentration, but does not result in a discrete band corresponding to the peptide–Mlc1sa complex. For the second and third IQ-motifs, some blurring of the protein band was observed. No interaction between the fourth IQ-motif and Mlc1sa was detected. The numbers above the gels indicate the peptide concentration in μM. Mlc1sa represents the myosin essential light chain-only (35 μM) control and Pep the peptide-only (35 μM) control. These results are representative of three independent experiments.

Interaction of IQ-motifs from human IQGAP2 and IQGAP3 with myosin essential light chain and S100B

Previous work has demonstrated that IQGAP1 interacts, through its first IQ-motif, with the myosin essential light chain Mlc1sa and, through its first and second IQ-motifs, with S100B [35]. Increasing concentrations of 3IQ1 caused blurring of the Mlc1sa band in a native gel shift assay (Figure 5). The peptides 3IQ2 and 3IQ3 caused some loss of intensity of the Mlc1sa band at high concentrations without the appearance of a detectable additional band. No interactions were detected between Mlc1sa and 3IQ4 or with any of the IQ-motifs from IQGAP2 (Figure 5). Furthermore, no interactions between S100B and any of the IQ-motifs were detected (results not shown).

Interaction between the IQ-motifs of IQGAP2 and IQGAP3 and the myosin essential light chain, Mlc1sa

(a) No interactions were detected between the IQ-motifs of IQGAP2 and Mlc1sa (concentrations indicated above the gel). (b) The first IQ-motif from IQGAP3 interacts with Mlc1sa. This interaction shows a clear dependence of shift on peptide concentration, but does not result in a discrete band corresponding to the peptide–Mlc1sa complex. For the second and third IQ-motifs, some blurring of the protein band was observed. No interaction between the fourth IQ-motif and Mlc1sa was detected. The numbers above the gels indicate the peptide concentration in μM. Mlc1sa represents the myosin essential light chain-only (35 μM) control and Pep the peptide-only (35 μM) control. These results are representative of three independent experiments.

Structural models of IQ-motifs from human IQGAP2 and IQGAP3To help understand the differences in affinity between the various IQ-motifs and CaM, structural models were constructed. As expected, the majority of the motifs are predicted to adopt an α-helical conformation in solution. The one exception is 2IQ1, which adopts a disordered, random coil conformation (Figure 6). In addition, the model for 3IQ3 only contains approximately half the residues in an α-helical conformation. The remainder were not modelled by Phyre. Therefore, it is possible that this motif also has some disordered residues in solution. These are modelled structures and so should be treated with some caution. In particular, it is likely that the conformation of the motifs alters on interaction with targets. Nevertheless, these models suggest that the majority of the IQ-motifs that interact with CaM will do so in a manner similar to that observed in myosins (e.g. [4]).Figure 6Modelled structures of the IQ-motifs from human IQGAP2 and IQGAP3The peptide backbone is shown as a cartoon and the side chains in ball and stick format. In each case, the N-terminus of the peptide is in the lower left-hand corner of the structure. All except 2IQ2 display an α-helical conformation. 2IQ2 has a random coil conformation with no recognizable elements of secondary structure. For clarity, the models are shown in the absence of solvating water molecules. 3IQ3 appears shorter than the remainder as only part of this sequence could modelled by the Phyre server. This may indicate that this motif has a more flexible or random coil structure than the other IQ-motifs from IQGAP3.

Structural models of IQ-motifs from human IQGAP2 and IQGAP3

To help understand the differences in affinity between the various IQ-motifs and CaM, structural models were constructed. As expected, the majority of the motifs are predicted to adopt an α-helical conformation in solution. The one exception is 2IQ1, which adopts a disordered, random coil conformation (Figure 6). In addition, the model for 3IQ3 only contains approximately half the residues in an α-helical conformation. The remainder were not modelled by Phyre. Therefore, it is possible that this motif also has some disordered residues in solution. These are modelled structures and so should be treated with some caution. In particular, it is likely that the conformation of the motifs alters on interaction with targets. Nevertheless, these models suggest that the majority of the IQ-motifs that interact with CaM will do so in a manner similar to that observed in myosins (e.g. [4]).

Modelled structures of the IQ-motifs from human IQGAP2 and IQGAP3

The peptide backbone is shown as a cartoon and the side chains in ball and stick format. In each case, the N-terminus of the peptide is in the lower left-hand corner of the structure. All except 2IQ2 display an α-helical conformation. 2IQ2 has a random coil conformation with no recognizable elements of secondary structure. For clarity, the models are shown in the absence of solvating water molecules. 3IQ3 appears shorter than the remainder as only part of this sequence could modelled by the Phyre server. This may indicate that this motif has a more flexible or random coil structure than the other IQ-motifs from IQGAP3.

DISCUSSIONThe results presented here show that there are clear differences in the affinity of the various IQ-motifs for CaM. Furthermore, they also demonstrate that there are (at least) two modes of interaction. Some IQ-motifs form complexes that are stable under the conditions of native gel electrophoresis. This method is a non-equilibrium technique. Prior to electrophoresis, there is an equilibrium between the bound and unbound material. Once the current is applied the bound and unbound begin to separate (since they have different masses and overall charges). This disturbs the equilibrium which will tend to reassert itself (Le Chatelier's Principle). The speed at which this occurs determines whether or not a discrete band will be observed. If dissociation of the complex is slow, with respect to the time of electrophoresis, then a discrete band will be observed representing a stable complex. If the dissociation time is comparable with the electrophoresis time, then blurring may be observed as the complex has partially dissociated during the experiment. These more dynamic complexes may be important in transducing transient changes in the prevailing calcium ion concentration, whereas the longer-lived complexes will remain bound if there are temporary reductions in the calcium concentration.The interaction of 3IQ1 with Mlc1sa was not unexpected since, of the eight IQ-motifs studied here, it is the most similar to the first IQ-motif from IQGAP1 (1IQ1). This motif, which shows similarity to myosin essential light chain binding regions in myosin heavy chains, has already been shown to interact with Mlc1sa [34,35]. However, the mode of interaction with the two IQ-motifs is different. Discrete shifts were observed with 1IQ1 [35], whereas less discrete, more blurred shifts were seen here (Figure 5). This may mean that IQGAP1 has long-lived, stable interactions with Mlc1sa while IQGAP3 makes more transient ones. However, in the absence of in vivo data on the significance of this interaction, the possibility of artefactual binding due to the high similarity to 1IQ1 cannot be ruled out completely. The physiological significance of the interaction between mammalian IQGAPs and myosin light chains is not clear. It has been suggested by analogy with yeasts that the interaction may be important in cytokinesis [52]. Although mammalian and sea urchin IQGAPs have been implicated in cytokinesis [26,53], thus far only the IQGAP–Cdc42 interaction has been shown to be important in this process.Taking the data presented here together with previously published work, we can begin to understand the selectivity of the various IQ-motifs towards CaM and related proteins (Table 2). This suggests considerable differences between the three human isoforms. Interestingly, very little binding to the IQ-motifs from IQGAP2 was detected. It is possible that these motifs only interact with CaM very transiently (i.e. with half-lives of seconds or less) as such interactions would not be detected by this gel electrophoresis method. Previous work has suggested that the arginine residues of the motifs are important for the binding of CaM in the absence of calcium ions [32]. This is only partly supported by the data presented here. While 3IQ1 has both arginine residues, 2IQ1 only has the second one (Figure 1). Unlike in IQGAP1, very little interaction between the IQ-motifs and CaM was detected in the absence of calcium ions with only 2IQ1 and, perhaps, 3IQ1 interacting under these conditions. Again this suggests differences between the three isoforms. Although the IQ-motifs from IQGAP2 and IQGAP3 may interact transiently in the absence of calcium (and thus not be detected by this method), interactions with IQGAP1 were also detected by non-equilibrium methods and must, therefore, be relatively long-lived [32,33].Table 2Interactions of IQ motifs from the three human isoforms of IQGAP1 with CaM and the structurally related proteins myosin essential light chain and S100BThe IQ motifs for IQGAP1 are named in a similar manner to that used in the present paper for IQGAP2 and IQGAP3 (i.e. 1IQ1, etc., in the order they appear in the primary sequence). The lifetime of the complex has been assessed from whether or not discrete bands were observed on native gel electrophoresis. In [32], the main method for detecting interaction with CaM was CaM–Sepharose pull-downs. Since this is also a non-equilibrium technique, it can be assumed that these interactions are also relatively long-lived. It is likely that future experiments will expand the list of known binders.IQ-motifKnown bindingReference1IQ1Mlc1sa (long-lived)[35]S100B+Ca2+ (long-lived)[35]CaM+Ca2+[32]1IQ2S100B+Ca2+ (long-lived)[35]CaM+Ca2+[32]1IQ3CaM-Ca2+[32]CaM+Ca2+[32]Rap1*[56]1IQ4CaM-Ca2+[32]CaM+Ca2+[32]Rap1[56]2IQ1CaM-Ca2+ (transient)This work2IQ2CaM+Ca2+ (transient)This work2IQ3CaM+Ca2+ (intermediate lifetime)This work2IQ4None detected3IQ1CaM+Ca2+ (transient)This workCaM-Ca2+ (transient)?This workMlc1sa (transient)This work3IQ2CaM+Ca2+ (long-lived)This work3IQ3CaM+Ca2+ (long-lived)This work3IQ4CaM+Ca2+ (long-lived)This work*The binding of the GTPase Rap1 has been localized to both 1IQ3 and 1IQ4. It is possible that only one is required or that further sequences are needed for recognition and interaction.The various IQ-motifs show some similarity and when the IQ-motifs from all three human IQGAP family proteins were compared, it was found that they tend to cluster according to position in sequence and not according to the protein from which they were derived – i.e. 1IQ1, 2IQ1 and 3IQ1 clustered together rather than all the motifs from IQGAP1, etc. clustering (Figure 7). Consistent with the molecular modelling data (Figure 6), 2IQ1 is the motif showing least similarity to the other 11. This lends support to the hypothesis that this motif has a different structure and, thus, may explain its ability to interact with CaM in the absence of calcium ions despite lacking one of the arginine residues. 2IQ1 is also the motif that is completely absent from the testicular splice variant of human IQGAP2 [28], suggesting that the signals transduced by this motif are not present in this tissue.Figure 7The similarity of all 12 IQ-motifs from human IQGAP1, IQGAP2 and IQGAP3An alignment was generated using ClustalW (as implemented at http://align.genome.jp/) [55] applying the BLOSUM matrix for both pairwise and multiple alignments. The gap opening penalty was 10 and the gap extension penalty 0.1. The unrooted dendrogram based on that alignment is shown. Interestingly, the motifs group according to position in the primary sequence, not the isoform they are derived from. Consistent with its predicted random coil structure, 2IQ1 is the most distantly related motif.The interaction with CaM in the presence of calcium ions is believed to rely largely on non-polar interactions, with the first isoleucine residue being particularly important [32]. Again the results presented here partly support this conclusion. The three IQ-motifs that interact most stably with CaM in the presence of calcium ions (3IQ2, 3IQ3 and 3IQ4), all contain several non-polar residues including isoleucine. However, this is also the case for 2IQ2, 2IQ3 and 2IQ4 (which have no or transient interactions with CaM) suggesting that additional factors are important in determining which motifs can interact. Structural work suggests that substitution of the glycine residue of the IQ-motif with larger side chains results in a more extended conformation of the bound myosin light chain or CaM in myosin V [4]. If this logic is extended to IQGAP2 and IQGAP3, then Mlc1sa is bound in a compact formation to 3IQ1 and CaM binds in a compact formation, except to 2IQ3 and 3IQ3. These two IQ-motifs may interact with a more extended form of CaM. However, the cardiac Cav1.2 calcium channel the IQ-motif, which contains a bulky lysine residue in place of the glycine, binds to a compact form of calcium-CaM [54]. Overall, the work presented here extends our knowledge of how CaM interacts with the IQ-motifs of IQGAP family proteins and also highlights the need for high-resolution structural studies on these interactions.

The results presented here show that there are clear differences in the affinity of the various IQ-motifs for CaM. Furthermore, they also demonstrate that there are (at least) two modes of interaction. Some IQ-motifs form complexes that are stable under the conditions of native gel electrophoresis. This method is a non-equilibrium technique. Prior to electrophoresis, there is an equilibrium between the bound and unbound material. Once the current is applied the bound and unbound begin to separate (since they have different masses and overall charges). This disturbs the equilibrium which will tend to reassert itself (Le Chatelier's Principle). The speed at which this occurs determines whether or not a discrete band will be observed. If dissociation of the complex is slow, with respect to the time of electrophoresis, then a discrete band will be observed representing a stable complex. If the dissociation time is comparable with the electrophoresis time, then blurring may be observed as the complex has partially dissociated during the experiment. These more dynamic complexes may be important in transducing transient changes in the prevailing calcium ion concentration, whereas the longer-lived complexes will remain bound if there are temporary reductions in the calcium concentration.

The interaction of 3IQ1 with Mlc1sa was not unexpected since, of the eight IQ-motifs studied here, it is the most similar to the first IQ-motif from IQGAP1 (1IQ1). This motif, which shows similarity to myosin essential light chain binding regions in myosin heavy chains, has already been shown to interact with Mlc1sa [34,35]. However, the mode of interaction with the two IQ-motifs is different. Discrete shifts were observed with 1IQ1 [35], whereas less discrete, more blurred shifts were seen here (Figure 5). This may mean that IQGAP1 has long-lived, stable interactions with Mlc1sa while IQGAP3 makes more transient ones. However, in the absence of in vivo data on the significance of this interaction, the possibility of artefactual binding due to the high similarity to 1IQ1 cannot be ruled out completely. The physiological significance of the interaction between mammalian IQGAPs and myosin light chains is not clear. It has been suggested by analogy with yeasts that the interaction may be important in cytokinesis [52]. Although mammalian and sea urchin IQGAPs have been implicated in cytokinesis [26,53], thus far only the IQGAP–Cdc42 interaction has been shown to be important in this process.

Taking the data presented here together with previously published work, we can begin to understand the selectivity of the various IQ-motifs towards CaM and related proteins (Table 2). This suggests considerable differences between the three human isoforms. Interestingly, very little binding to the IQ-motifs from IQGAP2 was detected. It is possible that these motifs only interact with CaM very transiently (i.e. with half-lives of seconds or less) as such interactions would not be detected by this gel electrophoresis method. Previous work has suggested that the arginine residues of the motifs are important for the binding of CaM in the absence of calcium ions [32]. This is only partly supported by the data presented here. While 3IQ1 has both arginine residues, 2IQ1 only has the second one (Figure 1). Unlike in IQGAP1, very little interaction between the IQ-motifs and CaM was detected in the absence of calcium ions with only 2IQ1 and, perhaps, 3IQ1 interacting under these conditions. Again this suggests differences between the three isoforms. Although the IQ-motifs from IQGAP2 and IQGAP3 may interact transiently in the absence of calcium (and thus not be detected by this method), interactions with IQGAP1 were also detected by non-equilibrium methods and must, therefore, be relatively long-lived [32,33].

Interactions of IQ motifs from the three human isoforms of IQGAP1 with CaM and the structurally related proteins myosin essential light chain and S100B

The IQ motifs for IQGAP1 are named in a similar manner to that used in the present paper for IQGAP2 and IQGAP3 (i.e. 1IQ1, etc., in the order they appear in the primary sequence). The lifetime of the complex has been assessed from whether or not discrete bands were observed on native gel electrophoresis. In [32], the main method for detecting interaction with CaM was CaM–Sepharose pull-downs. Since this is also a non-equilibrium technique, it can be assumed that these interactions are also relatively long-lived. It is likely that future experiments will expand the list of known binders.

*The binding of the GTPase Rap1 has been localized to both 1IQ3 and 1IQ4. It is possible that only one is required or that further sequences are needed for recognition and interaction.

The various IQ-motifs show some similarity and when the IQ-motifs from all three human IQGAP family proteins were compared, it was found that they tend to cluster according to position in sequence and not according to the protein from which they were derived – i.e. 1IQ1, 2IQ1 and 3IQ1 clustered together rather than all the motifs from IQGAP1, etc. clustering (Figure 7). Consistent with the molecular modelling data (Figure 6), 2IQ1 is the motif showing least similarity to the other 11. This lends support to the hypothesis that this motif has a different structure and, thus, may explain its ability to interact with CaM in the absence of calcium ions despite lacking one of the arginine residues. 2IQ1 is also the motif that is completely absent from the testicular splice variant of human IQGAP2 [28], suggesting that the signals transduced by this motif are not present in this tissue.

The similarity of all 12 IQ-motifs from human IQGAP1, IQGAP2 and IQGAP3

An alignment was generated using ClustalW (as implemented at http://align.genome.jp/) [55] applying the BLOSUM matrix for both pairwise and multiple alignments. The gap opening penalty was 10 and the gap extension penalty 0.1. The unrooted dendrogram based on that alignment is shown. Interestingly, the motifs group according to position in the primary sequence, not the isoform they are derived from. Consistent with its predicted random coil structure, 2IQ1 is the most distantly related motif.

The interaction with CaM in the presence of calcium ions is believed to rely largely on non-polar interactions, with the first isoleucine residue being particularly important [32]. Again the results presented here partly support this conclusion. The three IQ-motifs that interact most stably with CaM in the presence of calcium ions (3IQ2, 3IQ3 and 3IQ4), all contain several non-polar residues including isoleucine. However, this is also the case for 2IQ2, 2IQ3 and 2IQ4 (which have no or transient interactions with CaM) suggesting that additional factors are important in determining which motifs can interact. Structural work suggests that substitution of the glycine residue of the IQ-motif with larger side chains results in a more extended conformation of the bound myosin light chain or CaM in myosin V [4]. If this logic is extended to IQGAP2 and IQGAP3, then Mlc1sa is bound in a compact formation to 3IQ1 and CaM binds in a compact formation, except to 2IQ3 and 3IQ3. These two IQ-motifs may interact with a more extended form of CaM. However, the cardiac Cav1.2 calcium channel the IQ-motif, which contains a bulky lysine residue in place of the glycine, binds to a compact form of calcium-CaM [54]. Overall, the work presented here extends our knowledge of how CaM interacts with the IQ-motifs of IQGAP family proteins and also highlights the need for high-resolution structural studies on these interactions.

Online dataSupplementary data

Online data

Supplementary data
