---
pmid: '21390126'
title: Inactivating mutations of acetyltransferase genes in B-cell lymphoma.
authors:
- Pasqualucci L
- Dominguez-Sola D
- Chiarenza A
- Fabbri G
- Grunn A
- Trifonov V
- Kasper LH
- Lerach S
- Tang H
- Ma J
- Rossi D
- Chadburn A
- Murty VV
- Mullighan CG
- Gaidano G
- Rabadan R
- Brindle PK
- Dalla-Favera R
journal: Nature
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3271441
doi: 10.1038/nature09730
---

# Inactivating mutations of acetyltransferase genes in B-cell lymphoma.
**Authors:** Pasqualucci L, Dominguez-Sola D, Chiarenza A, Fabbri G, Grunn A, Trifonov V, Kasper LH, Lerach S, Tang H, Ma J, Rossi D, Chadburn A, Murty VV, Mullighan CG, Gaidano G, Rabadan R, Brindle PK, Dalla-Favera R
**Journal:** Nature (2011)
**DOI:** [10.1038/nature09730](https://doi.org/10.1038/nature09730)
**PMC:** [PMC3271441](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3271441/)

## Abstract

1. Nature. 2011 Mar 10;471(7337):189-95. doi: 10.1038/nature09730.

Inactivating mutations of acetyltransferase genes in B-cell lymphoma.

Pasqualucci L(1), Dominguez-Sola D, Chiarenza A, Fabbri G, Grunn A, Trifonov V, 
Kasper LH, Lerach S, Tang H, Ma J, Rossi D, Chadburn A, Murty VV, Mullighan CG, 
Gaidano G, Rabadan R, Brindle PK, Dalla-Favera R.

Author information:
(1)Institute for Cancer Genetics, Herbert Irving Comprehensive Cancer Center, 
Columbia University, New York, New York 10032, USA. lp171@columbia.edu

B-cell non-Hodgkin's lymphoma comprises biologically and clinically distinct 
diseases the pathogenesis of which is associated with genetic lesions affecting 
oncogenes and tumour-suppressor genes. We report here that the two most common 
types--follicular lymphoma and diffuse large B-cell lymphoma--harbour frequent 
structural alterations inactivating CREBBP and, more rarely, EP300, two highly 
related histone and non-histone acetyltransferases (HATs) that act as 
transcriptional co-activators in multiple signalling pathways. Overall, about 
39% of diffuse large B-cell lymphoma and 41% of follicular lymphoma cases 
display genomic deletions and/or somatic mutations that remove or inactivate the 
HAT coding domain of these two genes. These lesions usually affect one allele, 
suggesting that reduction in HAT dosage is important for lymphomagenesis. We 
demonstrate specific defects in acetylation-mediated inactivation of the BCL6 
oncoprotein and activation of the p53 tumour suppressor. These results identify 
CREBBP/EP300 mutations as a major pathogenetic mechanism shared by common forms 
of B-cell non-Hodgkin's lymphoma, with direct implications for the use of drugs 
targeting acetylation/deacetylation mechanisms.

DOI: 10.1038/nature09730
PMCID: PMC3271441
PMID: 21390126 [Indexed for MEDLINE]

## Full Text

Diffuse Large B-cell Lymphoma (DLBCL) represents the most common form of B-cell non-Hodgkin Lymphoma (B-NHL), accounting for ~30% of the de-novo diagnoses and also arising as a frequent clinical evolution of Follicular Lymphoma (FL)1. The molecular pathogenesis of DLBCL is associated with multiple genetic lesions that segregate in part with individual phenotypic subtypes, namely germinal-center B-cell-like (GCB) and activated B-cell-like (ABC) DLBCL, suggesting the involvement of distinct oncogenic pathways2–8. However, the full spectrum of lesions that contribute to malignant transformation remains unknown. Genome-wide efforts toward the identification and functional characterization of the entire set of structural alterations present in the DLBCL genome are required for a complete understanding of its pathogenesis9.

Toward this end, we have integrated next generation whole-exome sequencing analysis of 7 DLBCL cases and genome-wide high-density single nucleotide polymorphism (SNP) array analysis of 72 DLBCL cases. This combined approach led to the identification of >450 loci that are affected by somatic point mutations and/or by recurrent, focal gene copy number (CN) aberrations. Among those that have been so far independently validated, the most commonly involved regions were those harboring the acetyltransferase genes CREBBP (CBP) and EP300 (p300). CREBBP encodes a highly conserved and ubiquitously expressed nuclear phosphoprotein that, together with the closely related protein EP300, belongs to the KAT3 family of histone/protein lysine acetyltransferases10,11. CREBBP and EP300 function as transcriptional coactivators for a large number of DNA-binding transcription factors involved in multiple signaling and developmental pathways, by modifying lysine residues on both histone and non-histone nuclear proteins12,13. CREBBP and EP300 enhance transcription by multiple mechanisms, including: i) targeted acetylation of chromatin12,13; ii) acetylation of transcriptional activators (e.g., the tumor suppressors p53 and GATA1)14–17; and iii) acetylation-mediated inactivation of transcriptional repressors (e.g. the DLBCL-associated oncogene BCL6)18. Additionally, both molecules were found to exhibit ubiquitin ligase activity19,20. Consistent with the involvement in critical cellular functions, homozygous null mice for either Crebbp or Ep300 are early embryonic lethal21,22, and the same is true for the compound Crebbp/Ep300 double heterozygous mice21,22. Haploinsufficiency of CREBBP (and, rarely, EP300) is responsible for the Rubinstein-Taybi syndrome (RTS), an autosomal congenital disorder characterized by mental and growth retardation, skeletal abnormalities and predisposition to tumor development23–25. Somatic mutations of these two genes are exceedingly rare in epithelial cancers26–28, and only three cases were reported to carry EP300, but not CREBBP mutations in hematologic malignancies29,30. Additionally, chromosomal translocations of CREBBP are associated with an infrequent type of acute myeloid leukemia (AML) and with therapy-related AML and myelodysplastic syndrome, although the precise consequences of these lesions have not been fully elucidated31–33.

Monoallelic lesions of CREBBP in DLBCLFollowing initial observations from whole-exome sequencing analysis of 7 DLBCL cases and paired normal DNAs, we performed targeted re-sequencing of the entire CREBBP coding exons in 134 DLBCL samples representative of major phenotypic subtypes. This analysis revealed a total of 34 sequence variants distributed in 30 samples and whose somatic origin was documented by analysis of paired normal DNAs (available in 8 cases)(Fig. 1a,b and Table S1). Of these variants, 17 (50%) were inactivating events, including nonsense mutations (n=9), frameshift insertions/deletions (n=7) and mutations at consensus splice donor/splice acceptor sites (n=1), which generate aberrant transcripts carrying premature stop codons. Based on their distribution along the CREBBP protein, these mutations are all predicted to cause the elimination or truncation of the HAT domain (Fig. 1a and Table S1). The remaining variants included 3 in-frame deletions and 14 missense mutations, primarily within the HAT domain (Fig. 1a and Table S1), suggesting that they may be functionally important (see below). While CREBBP mutations were identified in both DLBCL phenotypic subtypes, their frequency was significantly higher in GCB-DLBCL, where they account for ~32% of the cases (n=21/65) as compared to 13% in ABC/non-GC DLBCL (n=9/69; p<0.01)(Fig. 1c).High density SNP array analysis, available for 72 samples from the same panel, and FISH analysis revealed the presence of monoallelic deletions encompassing or internal to the CREBBP locus in 8 additional cases (5 GCB-DLBCL and 3 ABC/non-GC-DLBCL), and a homozygous deletion in one patient (Fig. 2a,b and Fig. S1). Notably, the loss of genetic material was smaller than 240Kb in 4 cases and, in two (2147 and 2043), only involved a limited subset of CREBBP exons, thereby identifying this gene as the specific target (Fig. 2a and Table S2). In two additional samples for which CN data were not available, direct sequencing analysis revealed a hemizygous missense mutation, reflecting either the loss of the second allele or copy neutral loss of heterozygosity (Table S1). When combining the sequencing data with the CN data, CREBBP mutations and deletions were found to be mutually exclusive in most samples, revealing a predominantly monoallelic distribution (n=33/39 cases) (Fig. 2c). Rare instances of biallelic genetic lesions include the homozygous loss, a missense mutation with loss of heterozygosity (2 primary biopsies), biallelic nucleotide substitutions (2 cases), and a frameshift deletion with missense mutation of the second allele in the OCI-Ly8 cell line. Thus, 29% of all DLBCL patients (n=39/134), corresponding to 41.5% of GCB- and 17% of ABC-DLBCL, harbor genomic alterations affecting the CREBBP gene (Fig. 2d).

Monoallelic lesions of CREBBP in DLBCL

Following initial observations from whole-exome sequencing analysis of 7 DLBCL cases and paired normal DNAs, we performed targeted re-sequencing of the entire CREBBP coding exons in 134 DLBCL samples representative of major phenotypic subtypes. This analysis revealed a total of 34 sequence variants distributed in 30 samples and whose somatic origin was documented by analysis of paired normal DNAs (available in 8 cases)(Fig. 1a,b and Table S1). Of these variants, 17 (50%) were inactivating events, including nonsense mutations (n=9), frameshift insertions/deletions (n=7) and mutations at consensus splice donor/splice acceptor sites (n=1), which generate aberrant transcripts carrying premature stop codons. Based on their distribution along the CREBBP protein, these mutations are all predicted to cause the elimination or truncation of the HAT domain (Fig. 1a and Table S1). The remaining variants included 3 in-frame deletions and 14 missense mutations, primarily within the HAT domain (Fig. 1a and Table S1), suggesting that they may be functionally important (see below). While CREBBP mutations were identified in both DLBCL phenotypic subtypes, their frequency was significantly higher in GCB-DLBCL, where they account for ~32% of the cases (n=21/65) as compared to 13% in ABC/non-GC DLBCL (n=9/69; p<0.01)(Fig. 1c).

High density SNP array analysis, available for 72 samples from the same panel, and FISH analysis revealed the presence of monoallelic deletions encompassing or internal to the CREBBP locus in 8 additional cases (5 GCB-DLBCL and 3 ABC/non-GC-DLBCL), and a homozygous deletion in one patient (Fig. 2a,b and Fig. S1). Notably, the loss of genetic material was smaller than 240Kb in 4 cases and, in two (2147 and 2043), only involved a limited subset of CREBBP exons, thereby identifying this gene as the specific target (Fig. 2a and Table S2). In two additional samples for which CN data were not available, direct sequencing analysis revealed a hemizygous missense mutation, reflecting either the loss of the second allele or copy neutral loss of heterozygosity (Table S1). When combining the sequencing data with the CN data, CREBBP mutations and deletions were found to be mutually exclusive in most samples, revealing a predominantly monoallelic distribution (n=33/39 cases) (Fig. 2c). Rare instances of biallelic genetic lesions include the homozygous loss, a missense mutation with loss of heterozygosity (2 primary biopsies), biallelic nucleotide substitutions (2 cases), and a frameshift deletion with missense mutation of the second allele in the OCI-Ly8 cell line. Thus, 29% of all DLBCL patients (n=39/134), corresponding to 41.5% of GCB- and 17% of ABC-DLBCL, harbor genomic alterations affecting the CREBBP gene (Fig. 2d).

Frequent mutations of CREBBP in FLWe next analyzed various types of mature B-NHL, including FL, Burkitt Lymphoma (BL), Marginal-Zone Lymphoma (MZL) and Chronic Lymphocytic Leukemia (CLL). Mutations analogous to those found in DLBCL were frequent in FL (32.6%, with 16 events distributed in 15/46 cases), but not in other lymphoma types, suggesting a specific role in the pathogenesis of these two diseases (Fig. S2). This analysis also revealed the existence of several mutational hotspots at specific codons within the HAT domain, including R1446, also mutated in B-cell acute lymphoblastic leukemia (B-ALL)34, Y1503 and D1435; in addition, a 3bp in-frame deletion causing the loss of S1680 was observed in 5 cases, suggesting a functional role for this presently uncharacterized serine (Table S1). None of these recurrent changes were detected in paired normal DNA, thus excluding that they represent germline polymorphisms. While CN data were not available for the same FL panel, array-CGH analysis performed on a distinct dataset showed deletions spanning the CREBBP locus in only 1/68 cases (not shown). Collectively, these findings identify somatic mutations of CREBBP as a common event in FL.

Frequent mutations of CREBBP in FL

We next analyzed various types of mature B-NHL, including FL, Burkitt Lymphoma (BL), Marginal-Zone Lymphoma (MZL) and Chronic Lymphocytic Leukemia (CLL). Mutations analogous to those found in DLBCL were frequent in FL (32.6%, with 16 events distributed in 15/46 cases), but not in other lymphoma types, suggesting a specific role in the pathogenesis of these two diseases (Fig. S2). This analysis also revealed the existence of several mutational hotspots at specific codons within the HAT domain, including R1446, also mutated in B-cell acute lymphoblastic leukemia (B-ALL)34, Y1503 and D1435; in addition, a 3bp in-frame deletion causing the loss of S1680 was observed in 5 cases, suggesting a functional role for this presently uncharacterized serine (Table S1). None of these recurrent changes were detected in paired normal DNA, thus excluding that they represent germline polymorphisms. While CN data were not available for the same FL panel, array-CGH analysis performed on a distinct dataset showed deletions spanning the CREBBP locus in only 1/68 cases (not shown). Collectively, these findings identify somatic mutations of CREBBP as a common event in FL.

EP300 mutations in DLBCL and FLGiven the significant structural and functional similarities between CREBBP and EP300, we investigated whether this second member of the KAT3 acetyltransferase family is also targeted by structural alterations in B-NHL. Mutational analysis of the same panel identified 19 sequence variants leading to amino acid changes (n=11), in-frame deletions (n=2), and premature stop codons due to frameshift deletions, aberrant splicing or nonsense mutations (n=6) (Fig. S3a and Table S3). These lesions were found in 10% of DLBCL (n=13/134) and 8.7% of FL samples (n=4/46), but were virtually absent in other B-NHLs (Fig. S3b). Seven additional DLBCLs harbored monoallelic deletions spanning, although not limited to, the EP300 locus (Fig. S3c, Table S4 and data not shown). Notably, structural alterations of CREBBP and EP300 co-exist in only a minority of the affected cases (n=6/53 DLBCL and 0/19 FL) (Fig. S3d), suggesting that inactivation of these loci is at least in part functionally equivalent. Thus, overall ~39% of all DLBCL and at least 41% of FL cases (based on mutations only) display structural alterations of KAT3 family genes.

EP300 mutations in DLBCL and FL

Given the significant structural and functional similarities between CREBBP and EP300, we investigated whether this second member of the KAT3 acetyltransferase family is also targeted by structural alterations in B-NHL. Mutational analysis of the same panel identified 19 sequence variants leading to amino acid changes (n=11), in-frame deletions (n=2), and premature stop codons due to frameshift deletions, aberrant splicing or nonsense mutations (n=6) (Fig. S3a and Table S3). These lesions were found in 10% of DLBCL (n=13/134) and 8.7% of FL samples (n=4/46), but were virtually absent in other B-NHLs (Fig. S3b). Seven additional DLBCLs harbored monoallelic deletions spanning, although not limited to, the EP300 locus (Fig. S3c, Table S4 and data not shown). Notably, structural alterations of CREBBP and EP300 co-exist in only a minority of the affected cases (n=6/53 DLBCL and 0/19 FL) (Fig. S3d), suggesting that inactivation of these loci is at least in part functionally equivalent. Thus, overall ~39% of all DLBCL and at least 41% of FL cases (based on mutations only) display structural alterations of KAT3 family genes.

CREBBP and EP300 protein expressionIn order to compare the CREBBP and EP300 protein levels in normal and transformed B-cells, and to investigate the expression of the retained normal allele in cases carrying monoallelic genomic alterations, we examined the expression pattern of these two proteins in reactive human tonsils and in 78 DLBCL primary cases, of which 49 harbored both genes in wild-type configuration, by immunofluorescence (IF) and immunohistochemical (IHC) analysis. Consistent with their reported ubiquitous expression, CREBBP and EP300 were detectable in naïve B-cells within the mantle zone and, at higher levels, in germinal center (GC) B-cells (Fig 3a). Notably, most of the monoallelically-deleted DLBCL cases were positive for the two proteins, although at reduced levels, demonstrating that the residual wild-type allele is expressed (see Fig.3b,c for representative examples). Furthermore, RT-PCR amplification and direct sequencing of the ten CREBBP-mutated cell lines invariably showed the presence of the wild-type allele (Fig. S4), while western blot analysis using antibodies directed against the N-terminal portion of CREBBP revealed the expression of a full-length protein, corresponding to the wild-type allele, in most of the cell lines carrying truncating gene mutations, as well as in the monoallelically-deleted SUDHL5 cells (Fig. 3d, top panel). Slightly different findings were observed for EP300 where, in contrast with the primary biopsies, the presence of truncating mutations or deletions was associated with the absence of protein expression in four affected cell lines (Fig. 3d, second panel from top). With the exception of SUDHL2, both CREBBP and EP300 mRNAs were readily detected by northern blot analysis in all lines examined (Fig 3d, panel 4 and 5 from top).Interestingly, a few additional lines were found to express very low to undetectable protein levels, despite the presence of mRNA and the absence of structural alterations in the corresponding gene (see CREBBP in BJAB and SUHDL2, or EP300 in SUDHL7) (Fig. 3d). Similarly, 6/78 (8%) DLBCL biopsies from patients with intact alleles appeared to lack expression of the two proteins, either simultaneously (3 cases) or individually (2 CREBBP+/EP300− and 1 CREBBP−/EP300+ cases)(Fig. 3c,e). Thus, the fraction of DLBCL patients with defective CREBBP and/or EP300 may be higher than that determined based on genetic lesions alone, suggesting that alternative mechanisms of KAT3 gene family inactivation may play a role in this disease.

CREBBP and EP300 protein expression

In order to compare the CREBBP and EP300 protein levels in normal and transformed B-cells, and to investigate the expression of the retained normal allele in cases carrying monoallelic genomic alterations, we examined the expression pattern of these two proteins in reactive human tonsils and in 78 DLBCL primary cases, of which 49 harbored both genes in wild-type configuration, by immunofluorescence (IF) and immunohistochemical (IHC) analysis. Consistent with their reported ubiquitous expression, CREBBP and EP300 were detectable in naïve B-cells within the mantle zone and, at higher levels, in germinal center (GC) B-cells (Fig 3a). Notably, most of the monoallelically-deleted DLBCL cases were positive for the two proteins, although at reduced levels, demonstrating that the residual wild-type allele is expressed (see Fig.3b,c for representative examples). Furthermore, RT-PCR amplification and direct sequencing of the ten CREBBP-mutated cell lines invariably showed the presence of the wild-type allele (Fig. S4), while western blot analysis using antibodies directed against the N-terminal portion of CREBBP revealed the expression of a full-length protein, corresponding to the wild-type allele, in most of the cell lines carrying truncating gene mutations, as well as in the monoallelically-deleted SUDHL5 cells (Fig. 3d, top panel). Slightly different findings were observed for EP300 where, in contrast with the primary biopsies, the presence of truncating mutations or deletions was associated with the absence of protein expression in four affected cell lines (Fig. 3d, second panel from top). With the exception of SUDHL2, both CREBBP and EP300 mRNAs were readily detected by northern blot analysis in all lines examined (Fig 3d, panel 4 and 5 from top).

Interestingly, a few additional lines were found to express very low to undetectable protein levels, despite the presence of mRNA and the absence of structural alterations in the corresponding gene (see CREBBP in BJAB and SUHDL2, or EP300 in SUDHL7) (Fig. 3d). Similarly, 6/78 (8%) DLBCL biopsies from patients with intact alleles appeared to lack expression of the two proteins, either simultaneously (3 cases) or individually (2 CREBBP+/EP300− and 1 CREBBP−/EP300+ cases)(Fig. 3c,e). Thus, the fraction of DLBCL patients with defective CREBBP and/or EP300 may be higher than that determined based on genetic lesions alone, suggesting that alternative mechanisms of KAT3 gene family inactivation may play a role in this disease.

CREBBP missense mutants fail to acetylate BCL6 and p53While the presence of gross gene deletions and the distribution of truncating mutations clearly predict a complete gene inactivation or the loss of multiple key functional domains, including the HAT, the functional consequences of the numerous CREBBP missense mutations required direct experimental analysis. Notably, of the 30 total events identified, 27 cluster within HAT coding exons, suggesting a selective pressure to alter the CREBBP enzymatic activity (Table S1). In particular, 19 mutations were located within a 68 amino acids stretch that is 96% identical to EP300 and includes the contact surface for coenzyme A (CoA)35 (Fig. S5). We therefore examined the effect of these mutations on the ability of CREBBP to acetylate known physiologic substrates. We selected BCL6 and p53 because of their biological relevance for GC development and lymphomagenesis4,36. In fact, acetylation of the tumor suppressor p53 is indispensable for its transcriptional activity14,15,37, while EP300-mediated acetylation of the proto-oncoprotein BCL6 leads to inactivation of its transcriptional repressor function18.Transient transfection/co-immunoprecipitation assays confirmed that, analogous to EP30018, CREBBP binds to and acetylates BCL6, leading to a dose dependent impairment in its ability to repress a BCL6-responsive reporter gene (Fig. S6a,b). We then generated HA-tagged constructs for expression of 9 representative CREBBP alleles harboring missense mutations within (n=6) or immediately outside (n=2) the core HAT domain, as well as a premature stop codon (R1360X) (Fig. 4a). Notably, all of the HAT domain mutant proteins had lost their ability to acetylate BCL6 (Fig. 4b) and to interfere with its transrepression activity, consistent with the reported role of acetylation in inactivating BCL6 (Fig 4c). Conversely, no significant effects were observed from the C1240R and K1320R proteins or from two additional alleles (P1053L, Q1079H) harboring mutations outside the HAT domain (Table S1 and data not shown), although C1240R appeared to have lost its activity in the reporter assay, suggesting that alternative mechanisms may be involved.When tested on the tumor suppressor p53, the same core HAT mutants were either impaired or severely attenuated (H1487Y and D1435E) in their acetylation activity, as compared to wild-type CREBBP or to C1240R and K1320R (Fig. 4d). It should be noted that C1240R and K1320R represent a second mutational event in cases carrying an additional truncating CREBBP mutation (see Table S1), suggesting the possibility that they represent passenger events or they confer more subtle functional consequences. Taken together, these findings demonstrate that CREBBP missense mutant alleles have been selected for their defective HAT activity. In particular, the impairment on BCL6 and p53 acetylation is consistent with specific effects of CREBBP genetic lesions in favoring the constitutive activity of the BCL6 oncogene over the function of the p53 tumor suppressor.

CREBBP missense mutants fail to acetylate BCL6 and p53

While the presence of gross gene deletions and the distribution of truncating mutations clearly predict a complete gene inactivation or the loss of multiple key functional domains, including the HAT, the functional consequences of the numerous CREBBP missense mutations required direct experimental analysis. Notably, of the 30 total events identified, 27 cluster within HAT coding exons, suggesting a selective pressure to alter the CREBBP enzymatic activity (Table S1). In particular, 19 mutations were located within a 68 amino acids stretch that is 96% identical to EP300 and includes the contact surface for coenzyme A (CoA)35 (Fig. S5). We therefore examined the effect of these mutations on the ability of CREBBP to acetylate known physiologic substrates. We selected BCL6 and p53 because of their biological relevance for GC development and lymphomagenesis4,36. In fact, acetylation of the tumor suppressor p53 is indispensable for its transcriptional activity14,15,37, while EP300-mediated acetylation of the proto-oncoprotein BCL6 leads to inactivation of its transcriptional repressor function18.

Transient transfection/co-immunoprecipitation assays confirmed that, analogous to EP30018, CREBBP binds to and acetylates BCL6, leading to a dose dependent impairment in its ability to repress a BCL6-responsive reporter gene (Fig. S6a,b). We then generated HA-tagged constructs for expression of 9 representative CREBBP alleles harboring missense mutations within (n=6) or immediately outside (n=2) the core HAT domain, as well as a premature stop codon (R1360X) (Fig. 4a). Notably, all of the HAT domain mutant proteins had lost their ability to acetylate BCL6 (Fig. 4b) and to interfere with its transrepression activity, consistent with the reported role of acetylation in inactivating BCL6 (Fig 4c). Conversely, no significant effects were observed from the C1240R and K1320R proteins or from two additional alleles (P1053L, Q1079H) harboring mutations outside the HAT domain (Table S1 and data not shown), although C1240R appeared to have lost its activity in the reporter assay, suggesting that alternative mechanisms may be involved.

When tested on the tumor suppressor p53, the same core HAT mutants were either impaired or severely attenuated (H1487Y and D1435E) in their acetylation activity, as compared to wild-type CREBBP or to C1240R and K1320R (Fig. 4d). It should be noted that C1240R and K1320R represent a second mutational event in cases carrying an additional truncating CREBBP mutation (see Table S1), suggesting the possibility that they represent passenger events or they confer more subtle functional consequences. Taken together, these findings demonstrate that CREBBP missense mutant alleles have been selected for their defective HAT activity. In particular, the impairment on BCL6 and p53 acetylation is consistent with specific effects of CREBBP genetic lesions in favoring the constitutive activity of the BCL6 oncogene over the function of the p53 tumor suppressor.

CREBBP mutations reduce affinity for Acetyl-CoA bindingTo investigate the mechanisms responsible for the observed loss of function on BCL6 and p53, we examined the effect of CREBBP mutations on subcellular localization, enzyme-substrate complex formation, and enzymatic activity. All mutants tested were correctly localized in the nuclear compartment and could be efficiently co-immunoprecipitated with BCL6 or p53 (Fig. S7 and data not shown), indicating that the inability to acetylate these substrates in vivo was not due to mislocalization or to impaired physical interaction. Instead, in vitro acetylation assays using recombinant GST-p53 and highly purified, wild-type or mutated CREBBP proteins demonstrated a marked, Acetyl-CoA dose-dependent defect in the enzymatic activity of the core-HAT mutants (Fig. 5 and Fig. S8), suggesting that these changes impair the affinity for Acetyl-CoA (see also Fig. S5).

CREBBP mutations reduce affinity for Acetyl-CoA binding

To investigate the mechanisms responsible for the observed loss of function on BCL6 and p53, we examined the effect of CREBBP mutations on subcellular localization, enzyme-substrate complex formation, and enzymatic activity. All mutants tested were correctly localized in the nuclear compartment and could be efficiently co-immunoprecipitated with BCL6 or p53 (Fig. S7 and data not shown), indicating that the inability to acetylate these substrates in vivo was not due to mislocalization or to impaired physical interaction. Instead, in vitro acetylation assays using recombinant GST-p53 and highly purified, wild-type or mutated CREBBP proteins demonstrated a marked, Acetyl-CoA dose-dependent defect in the enzymatic activity of the core-HAT mutants (Fig. 5 and Fig. S8), suggesting that these changes impair the affinity for Acetyl-CoA (see also Fig. S5).

CREBBP missense mutants fail to rescue the phenotype of Crebbp/Ep300 null cellsTo further investigate the consequences of CREBBP missense mutations on its function as a transcriptional co-activator, we examined the response of endogenous cAMP response element-binding protein (CREB) target genes in mouse embryonic fibroblasts carrying conditional Crebbp and Ep300 knock-out alleles (dKO MEFs), reconstituted with retroviral vectors expressing either wild-type or DLBCL-associated mutated Crebbp cDNAs (Fig. S9a), and treated with forskolin and IBMX (FI), two inducers of PKA activity. In normal cells, PKA-mediated phosphorylation of CREB is required for the recruitment of CREBBP, which in turn leads to transcriptional activation of cAMP-responsive genes38. As previously observed39, quantitative RT-PCR analysis of CREBBP/EP300-dependent CREB target genes confirmed their upregulation in dKO cells reconstituted with wild-type CREBBP expression constructs (Fig. S9b, column 2 from left). In contrast, all four DLBCL-derived CREBBP mutants tested were generally deficient for cAMP-responsive transcription, as was the HAT-dead control W1502A/Y1503S40 (Fig. S9b and Fig. S10). These mutants were also associated with reduced endogenous histone H3K18 acetylation (Fig. S11). Finally, we tested whether the mutant CREBBP polypeptides were proficient in rescuing the proliferative defect of the dKO cells, as this system provides a specific readout for the biological activity of the CREBBP point mutants in a physiologic setting. Notably, cells transduced with the four DLBCL-associated alleles displayed significantly impaired cell growth compared to wild-type reconstituted cells, as measured in the YFP+ (dKO) population (Fig. S9c). Collectively, the data presented above provide direct experimental evidence for a role of HAT missense mutations in impairing CREBBP acetyltransferase activity and its function as a transcriptional coactivator.

CREBBP missense mutants fail to rescue the phenotype of Crebbp/Ep300 null cells

To further investigate the consequences of CREBBP missense mutations on its function as a transcriptional co-activator, we examined the response of endogenous cAMP response element-binding protein (CREB) target genes in mouse embryonic fibroblasts carrying conditional Crebbp and Ep300 knock-out alleles (dKO MEFs), reconstituted with retroviral vectors expressing either wild-type or DLBCL-associated mutated Crebbp cDNAs (Fig. S9a), and treated with forskolin and IBMX (FI), two inducers of PKA activity. In normal cells, PKA-mediated phosphorylation of CREB is required for the recruitment of CREBBP, which in turn leads to transcriptional activation of cAMP-responsive genes38. As previously observed39, quantitative RT-PCR analysis of CREBBP/EP300-dependent CREB target genes confirmed their upregulation in dKO cells reconstituted with wild-type CREBBP expression constructs (Fig. S9b, column 2 from left). In contrast, all four DLBCL-derived CREBBP mutants tested were generally deficient for cAMP-responsive transcription, as was the HAT-dead control W1502A/Y1503S40 (Fig. S9b and Fig. S10). These mutants were also associated with reduced endogenous histone H3K18 acetylation (Fig. S11). Finally, we tested whether the mutant CREBBP polypeptides were proficient in rescuing the proliferative defect of the dKO cells, as this system provides a specific readout for the biological activity of the CREBBP point mutants in a physiologic setting. Notably, cells transduced with the four DLBCL-associated alleles displayed significantly impaired cell growth compared to wild-type reconstituted cells, as measured in the YFP+ (dKO) population (Fig. S9c). Collectively, the data presented above provide direct experimental evidence for a role of HAT missense mutations in impairing CREBBP acetyltransferase activity and its function as a transcriptional coactivator.

DiscussionThe results herein indicate that inactivation of CREBBP/EP300 represents a common event in the two most frequent forms of B-NHL, namely FL and DLBCL. Previous extensive surveys in malignancies of epithelial origin have reported inactivating mutations of EP300 and CREBBP in exceedingly rare cases (<2% of primary biopsies)26–28. Considering their virtual absence in solid tumors, and the finding of recurrent mutations in B-ALL34, our results point to a specific role for CREBBP/EP300 inactivation in the pathogenesis of malignancies derived from B-lymphocytes. Overall, CREBBP/EP300 lesions are among the most frequent structural alterations yet detected in FL and DLBCL, thus representing an important feature of the pathogenesis of these common diseases. Moreover, the observation of significantly reduced CREBBP and EP300 expression levels in a sizeable fraction of DLBCL samples, independent of genetic lesions (Fig. 3e and Fig. S12), suggests that additional epigenetic mechanisms may cause reduction of HAT dosage in a larger number of tumors.One key observation of this study is that CREBBP/EP300 lesions are mostly detected in heterozygosity, suggesting a haploinsufficent role in tumor suppression. This notion is supported by several observations. First, congenital heterozygous mutations of CREBBP/EP300 are sufficient to cause significant pathologic and developmental phenotypes, including tumorigenesis, in patients with RTS23–25, thereby confirming the deleterious effect of reduced HAT activity. Second, a fraction of mice with conditional deletion of Crebbp in mature B-cells showed reduced survival past 12-months of age41. Hematologic malignancies were also reported in 22% of constitutive 10–21 month-old Crebbp heterozygous mice and ~30% of chimeric animals after bone marrow or spleen cell transplantation from Crebbp heterozygotes42. Third, in contrast with the abundantly expressed histone deacetylases (HDACs), HATs are limiting in the cell, suggesting that small dosage variations can have severe biological consequences43. Overall, these data provide direct evidence for CREBBP (and, more rarely, EP300) as haploinsufficient tumor suppressors, whose specific role in lymphomagenesis will have to be tested by conditional deletion of these alleles in GC B-cells.Given the global involvement of CREBBP/EP300 on gene transcriptional regulation, it is difficult to predict which cellular targets/pathways may be critically affected by HAT reduction in lymphomagenesis. At this stage, our results demonstrate that mutant CREBBP and EP300 proteins are deficient in acetylating BCL6 and p53, leading to constitutive activation of the oncoprotein and to decreased p53 tumor suppressor activity. The balance between the activities of these two genes is critical for the regulation of DNA damage responses in mature GC cells during immunoglobulin gene remodeling36,44. Thus, the consequences of BCL6 activity overriding p53 would be an increased tolerance for DNA damage in the context of diminished apoptotic and cell cycle arrest responses.These results have important therapeutic implications in view of current attempts to utilize a variety of HDAC inhibitors as anti-cancer drugs. While the benefits of these compounds have been proven in certain malignancies, such as mature T-cell lymphoproliferative disorders, their efficacy in other common cancers, including B-NHL, is uncertain at this stage45. The findings of this study suggest that the use of HDAC inhibitors has a rational basis in B-NHL, as it may contribute to re-establishing physiologic acetylation levels. On the other hand, their efficacy should be re-evaluated by stratifying patients based on the presence of HAT defects as well as by testing the numerous HDAC and sirtuin inhibitors with target specificity.

The results herein indicate that inactivation of CREBBP/EP300 represents a common event in the two most frequent forms of B-NHL, namely FL and DLBCL. Previous extensive surveys in malignancies of epithelial origin have reported inactivating mutations of EP300 and CREBBP in exceedingly rare cases (<2% of primary biopsies)26–28. Considering their virtual absence in solid tumors, and the finding of recurrent mutations in B-ALL34, our results point to a specific role for CREBBP/EP300 inactivation in the pathogenesis of malignancies derived from B-lymphocytes. Overall, CREBBP/EP300 lesions are among the most frequent structural alterations yet detected in FL and DLBCL, thus representing an important feature of the pathogenesis of these common diseases. Moreover, the observation of significantly reduced CREBBP and EP300 expression levels in a sizeable fraction of DLBCL samples, independent of genetic lesions (Fig. 3e and Fig. S12), suggests that additional epigenetic mechanisms may cause reduction of HAT dosage in a larger number of tumors.

One key observation of this study is that CREBBP/EP300 lesions are mostly detected in heterozygosity, suggesting a haploinsufficent role in tumor suppression. This notion is supported by several observations. First, congenital heterozygous mutations of CREBBP/EP300 are sufficient to cause significant pathologic and developmental phenotypes, including tumorigenesis, in patients with RTS23–25, thereby confirming the deleterious effect of reduced HAT activity. Second, a fraction of mice with conditional deletion of Crebbp in mature B-cells showed reduced survival past 12-months of age41. Hematologic malignancies were also reported in 22% of constitutive 10–21 month-old Crebbp heterozygous mice and ~30% of chimeric animals after bone marrow or spleen cell transplantation from Crebbp heterozygotes42. Third, in contrast with the abundantly expressed histone deacetylases (HDACs), HATs are limiting in the cell, suggesting that small dosage variations can have severe biological consequences43. Overall, these data provide direct evidence for CREBBP (and, more rarely, EP300) as haploinsufficient tumor suppressors, whose specific role in lymphomagenesis will have to be tested by conditional deletion of these alleles in GC B-cells.

Given the global involvement of CREBBP/EP300 on gene transcriptional regulation, it is difficult to predict which cellular targets/pathways may be critically affected by HAT reduction in lymphomagenesis. At this stage, our results demonstrate that mutant CREBBP and EP300 proteins are deficient in acetylating BCL6 and p53, leading to constitutive activation of the oncoprotein and to decreased p53 tumor suppressor activity. The balance between the activities of these two genes is critical for the regulation of DNA damage responses in mature GC cells during immunoglobulin gene remodeling36,44. Thus, the consequences of BCL6 activity overriding p53 would be an increased tolerance for DNA damage in the context of diminished apoptotic and cell cycle arrest responses.

These results have important therapeutic implications in view of current attempts to utilize a variety of HDAC inhibitors as anti-cancer drugs. While the benefits of these compounds have been proven in certain malignancies, such as mature T-cell lymphoproliferative disorders, their efficacy in other common cancers, including B-NHL, is uncertain at this stage45. The findings of this study suggest that the use of HDAC inhibitors has a rational basis in B-NHL, as it may contribute to re-establishing physiologic acetylation levels. On the other hand, their efficacy should be re-evaluated by stratifying patients based on the presence of HAT defects as well as by testing the numerous HDAC and sirtuin inhibitors with target specificity.

Methods SummaryMutation analysisThe complete coding sequences and exon/intron junctions of CREBBP and EP300 were analyzed by PCR amplification and direct sequencing of whole genome amplified DNA using the oligonucleotides reported in Tables S5 and S6. Mutations were confirmed from both strands on independent PCR products amplified from genomic DNA, and their somatic origin was documented by analysis of matched normal DNA in available cases.SNP array analysiswas performed using Affymetrix Genome-Wide 6.0 Arrays and a computational workflow described in detail in the Supplementary Information.In vivo and in vitro characterization of HAT activityThe ability of CREBBP mutants to acetylate BCL6 and p53 was assessed in HEK293T cells after co-transfection of pCMV-Flag-BCL6 (or pCIN4-Flag-p53) with plasmids encoding wild-type vs mutant HA-tagged mouse Crebbp. BCL6 acetylation was evaluated on Flag/M2 immunoprecipitates using antibodies directed against acetyl lysines; for p53, western blot analysis was performed on whole cell extracts using a specific anti-acetylated p53 antibody. The amounts of exogenous CREBBP were monitored using anti-HA and anti-CREBBP (A22, Santa Cruz Biotechnology) antibodies. In vitro acetylation assays were performed using recombinant GST-p53 and purified CREBBP-HA proteins, in the presence of the indicated amounts of acetyl-CoA.Transient Transfection/Reporter gene assaysThe effect of CREBBP on BCL6-dependent transcription was assessed in HEK293T cells co-transfected with a luciferase reporter vector containing 5 BCL6 consensus binding sites upstream of the SV40 promoter (5XBCL6)46 and the pCMV-Flag-BCL6 construct, in the absence or presence of wild-type vs mutant CREBBP-HA expression vectors (see Full Methods).Reconstitution of Crebbp/Ep300 null MEFsCrebbpflox/flox;Ep300flox/flox;YFP+ conditional (dKO) MEFs have been described39. CREBBP expression was restored by retroviral infection with constructs encoding for HA-tagged wild-type or mutant Crebbp, and cells were analyzed for H3K18 acetylation, cAMP-dependent transcriptional responses and cell proliferation (see Full Methods).Full MethodsDNA extraction, amplification and sequencingGenomic DNA was extracted by standard methods, and whole genome amplification was performed using the RepliG kit (QIAGEN) according to the manufacturer’s instructions. Sequences for all annotated exons and flanking introns of CREBBP and EP300 were obtained from the UCSC Human Genome database, using the corresponding mRNA accession number as a reference (NM_004380.2 and NM_001429.3, respectively). PCR primers, located ≥50 bp upstream or downstream to target exon boundaries, were designed in the Primer 3 program (http://frodo.wi.mit.edu/primer3/) and filtered using UCSC In Silico PCR to exclude pairs yielding more than a single product (Table S5 and Table S6). Purified amplicons were sequenced directly from both strands (Genewiz, Inc, South Plainfield, NJ) as described, and compared to the corresponding germline sequences, using the Mutation Surveyor Version 2.41 software package (Softgenetics, State College, PA; http://www.softgenetics.com). Somatic mutations were confirmed on independent PCR products obtained from high molecular weight genomic DNA. Synonymous mutations, previously reported polymorphisms (Human dbSNP Database at NCBI, Build 130, and Ensembl Database) and changes present in the matched normal DNA, when available, were excluded. In cases carrying multiple events within the same gene, the allelic distribution of the mutations was determined by cloning and sequencing PCR products obtained from cDNA and spanning both events (N=10 clones each).Northern Blot analysis of CREBBP and EP300 expressionTotal RNA (12µg) was extracted from exponentially growing cell lines by TRIzol (Invitrogen) and Northern blot analysis was performed according to standard procedures, with radiolabeled probes corresponding to a 0.9Kb fragment of the human CREBBP cDNA (region 204–1143, GenBank accession No. NM_004380.2) or a 1.2Kb fragment of the human EP300 cDNA (region 6277–7524, GenBank accession No. NM_001429.3), and GAPDH as a control for loading. CREBBP and EP300 expression levels were then quantitated by Phosphorimager analysis and normalized with the GAPDH levels.Tissue microarrays, immunohistochemistry (IHC) and immunofluorescence (IF) analysisDLBCL tissue microarrays were constructed according to standard procedures and analyzed by IHC, using rabbit polyclonal antibodies directed against the N-terminus of CREBBP (A22) or EP300 (N15)(Santa Cruz Biotechnology). Cases were scored as positive if ≥20% tumor cells were stained by the antibody. IF analysis of CREBBP expression in conditional Crebbp/Ep300 dKO MEFs was performed 3 days after deletion of the endogenous loci and 5 days after infection with HA-tagged CREBBP retrovirus, using the CREBBP (A22) antibody. The specificity of both the A22 and the N15 antibodies had been previously validated on paraffin-embedded cell pellets from HEK293T cells transfected with control and CREBBP-specific shRNAs as well as by IF staining of Crebbp/Ep300 dKO MEFs (data not shown).Protein extraction and western blot analysisWhole cell extracts were prepared in RIPA buffer containing protease inhibitors as described18 and were analyzed by western blotting according to standard methods, using the following primary antibodies: anti-FLAG/M2 (Sigma), anti-HA (3F10)(Roche), anti-BCL6 (Cell Marque, GI191E/A8), anti-acetyl lysines (Cell Signaling, rabbit), anti-acetylated p53 (kind gift of W. Gu), anti p53 (DO1), anti-CREBBP (A22), anti-EP300 (N15)(all from Santa Cruz Biotechnology), anti-GFP (JL-8)(Clontech), anti-β-actin (clone AC-15) and anti-β-tubulin (clone B-5-1-2)(both from Sigma). Proteins were resolved by SDS-PAGE in 3–8% NuPAGE Tris-Acetate gels (Invitrogen), and visualized using a chemiluminescence detection kit (Pierce) as recommended by the manufacturer.Transient transfections/reporter gene assaysTransient transfections were performed in HEK293T cells using polyethylenimine (PEI), as described47. For reporter assays, cells were seeded on a 24-well plate and transfected using 100 ng of a Luciferase reporter construct driven by a PGL3-SV40-based backbone with 5 BCL6 binding sites46, 2.5 ng of pRL-SV40 (Promega), 1 ng of BCL6-encoding plasmid and the indicated doses of wild-type and mutant CREBBP expression vectors. The total amount of transfected DNA was kept constant in each experiment by adding pCMV-HA or pcDNA3 vector sequences to a final amount of 600ng/well. All experiments were performed in duplicate and luciferase activities were measured thirty-six hours after transfection using the Dual-Luciferase Reporter Assay System (Promega), according to the manufacturer’s instructions. Given the known effects of CREBBP on the SV40 promoter and on other regulatory sequences, the response of the BCL6 reporter was normalized first to that of a SV40-Renilla reporter construct, and then to the basal activity of CREBBP on the 5X-BCL6 reporter, in the absence of BCL6. Data are expressed as differences relative to the basal activity of the reporter construct (set as 1) after the above-mentioned normalization.Co-immunoprecipitation assaysTo examine the interaction between CREBBP and BCL6 or p53, HEK293T cells were co-transfected with plasmids encoding for the two proteins, together with wild-type or mutant CREBBP-HA. Thirty-six hours after transfection (or twenty-four hours for p53), cells were lysed in IP buffer (50 mM Tris, pH 7.0, 250 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.05% NP40, 10 mM sodium fluoride, 0.1mM sodium orthovanadate and protease inhibitor cocktail [Sigma]), and the cleared lysates were incubated overnight at 4°C with anti-HA or anti-Flag/M2 beads (Sigma). Immunocomplexes were eluted by incubating the beads in IP buffer containing HA or 3X-Flag peptide, respectively (Sigma). A fraction of the final eluates was resolved by SDS-PAGE and analyzed by western blot.In vitro acetyl-transferase assaysRecombinant GST-p53 and CREBBP-HA proteins were obtained as described in detail in the Supplementary Information and used for in vitro acteylation assays according to published protocols, with minor modifications14,48,49. Briefly, 10–24 ng of recombinant CREBBP-HA and 100 ng of GST-p53 were combined in 40 uL reactions containing 50 mM Tris-Cl (pH 8.0), 10% Glycerol, 1mM DTT, 1mM PMSF, 0.1 mM EDTA, 10 mM Butyric Acid (#B103500, Sigma), and variable amounts of Acetyl-CoA (Sigma)(2mM to 25 nM). As shown in Fig. 5, Acetyl-CoA concentrations as low as 25 nM were sufficient to obtain efficient acetylation of the substrate by CREBBP. Reactions were performed for 60 minutes at 30°C, and stopped by addition of an equal volume of 2X Laemmli buffer followed by heating for 10 minutes at 70°C. A fraction of the final product was resolved by SDS-PAGE in 3–8% Tris-Acetate gradient gels (Invitrogen) and analyzed by western blotting.Reconstitution of conditional Crebbp/Ep300 dKO MEFs for analysis of CREB target genes transcription, H3K18 acetylation and cell growthThe protocols for MEF isolation, cell culture and retroviral transduction, as well as the murine Crebbp (CBP)-HA retroviral construct used as template for the generation of various CREBBP mutants have been described previously50. In this system, Cre mediated recombination induces expression of YFP, allowing for specific identification of the deleted cells. Crebbpflox/flox; Ep300flox/flox; YFP+ MEFs were first infected with retroviruses encoding either wild-type or selected CREBBP point mutants, and the endogenous Crebbpflox and Ep300flox loci were deleted after 48 hours by infection with Cre-expressing adenovirus (Ad-Cre). The W1502A/Y1503S HAT-dead mutation was used as negative control40. In all experiments, transduction efficiencies were 70% or higher, as assessed by IF analysis of HA positive cells 3 days after deletion of endogenous Crebbp/Ep300 and 5 days after retroviral infection. To examine cAMP-dependent gene expression, MEFs were cultured for 16 hrs in DMEM containing 0.1% FBS and treated for 90 min with 10µM forskolin + 100µM IBMX (or ethanol vehicle) before harvesting in TRIzol (Invitrogen); qRT-PCR assays were performed as reported50. Expression of Crebbp and Ep300 was verified by IF analysis using rabbit polyclonal anti-CREBBP (A-22) and anti-EP300 (N-20) antibodies (Santa Cruz Biotechnology). Analysis of H3K18 acetylation was performed as described, using the anti-H3K18Ac antibody (ab1191) (Abcam) and the HA-11 monoclonal antibody against the HA epitope (Boehringer Mannheim). Confocal images were taken using the same settings for all mutants and mean intensity ratios for individual nuclei were collected using SlideBook 5 software. Nuclei where the Crebbp (HA) signal was at least 2.5 fold above background were used to calculate the ratio of the H3K18Ac mean signal intensity to the Crebbp (HA) mean signal intensity. For the growth assays shown in Fig. S11, Crebbpflox/flox;Ep300flox/flox;YFP+ MEFs were infected with CREBBP retroviruses 48 hours before Ad-Cre mediated deletion of endogenous Crebbp and Ep300. From each reconstituted population, equivalent numbers of YFP+ (dKO) MEFs were seeded at day 1 (i.e., the day following overnight Ad-Cre treatment), and the total number of YFP+ cells was calculated on Day 11 from the total cell number, based on the percentage of YFP+ cells as assessed by flow cytometric analysis.

Methods Summary

Mutation analysisThe complete coding sequences and exon/intron junctions of CREBBP and EP300 were analyzed by PCR amplification and direct sequencing of whole genome amplified DNA using the oligonucleotides reported in Tables S5 and S6. Mutations were confirmed from both strands on independent PCR products amplified from genomic DNA, and their somatic origin was documented by analysis of matched normal DNA in available cases.

Mutation analysis

The complete coding sequences and exon/intron junctions of CREBBP and EP300 were analyzed by PCR amplification and direct sequencing of whole genome amplified DNA using the oligonucleotides reported in Tables S5 and S6. Mutations were confirmed from both strands on independent PCR products amplified from genomic DNA, and their somatic origin was documented by analysis of matched normal DNA in available cases.

SNP array analysiswas performed using Affymetrix Genome-Wide 6.0 Arrays and a computational workflow described in detail in the Supplementary Information.

SNP array analysis

was performed using Affymetrix Genome-Wide 6.0 Arrays and a computational workflow described in detail in the Supplementary Information.

In vivo and in vitro characterization of HAT activityThe ability of CREBBP mutants to acetylate BCL6 and p53 was assessed in HEK293T cells after co-transfection of pCMV-Flag-BCL6 (or pCIN4-Flag-p53) with plasmids encoding wild-type vs mutant HA-tagged mouse Crebbp. BCL6 acetylation was evaluated on Flag/M2 immunoprecipitates using antibodies directed against acetyl lysines; for p53, western blot analysis was performed on whole cell extracts using a specific anti-acetylated p53 antibody. The amounts of exogenous CREBBP were monitored using anti-HA and anti-CREBBP (A22, Santa Cruz Biotechnology) antibodies. In vitro acetylation assays were performed using recombinant GST-p53 and purified CREBBP-HA proteins, in the presence of the indicated amounts of acetyl-CoA.

In vivo and in vitro characterization of HAT activity

The ability of CREBBP mutants to acetylate BCL6 and p53 was assessed in HEK293T cells after co-transfection of pCMV-Flag-BCL6 (or pCIN4-Flag-p53) with plasmids encoding wild-type vs mutant HA-tagged mouse Crebbp. BCL6 acetylation was evaluated on Flag/M2 immunoprecipitates using antibodies directed against acetyl lysines; for p53, western blot analysis was performed on whole cell extracts using a specific anti-acetylated p53 antibody. The amounts of exogenous CREBBP were monitored using anti-HA and anti-CREBBP (A22, Santa Cruz Biotechnology) antibodies. In vitro acetylation assays were performed using recombinant GST-p53 and purified CREBBP-HA proteins, in the presence of the indicated amounts of acetyl-CoA.

Transient Transfection/Reporter gene assaysThe effect of CREBBP on BCL6-dependent transcription was assessed in HEK293T cells co-transfected with a luciferase reporter vector containing 5 BCL6 consensus binding sites upstream of the SV40 promoter (5XBCL6)46 and the pCMV-Flag-BCL6 construct, in the absence or presence of wild-type vs mutant CREBBP-HA expression vectors (see Full Methods).

Transient Transfection/Reporter gene assays

The effect of CREBBP on BCL6-dependent transcription was assessed in HEK293T cells co-transfected with a luciferase reporter vector containing 5 BCL6 consensus binding sites upstream of the SV40 promoter (5XBCL6)46 and the pCMV-Flag-BCL6 construct, in the absence or presence of wild-type vs mutant CREBBP-HA expression vectors (see Full Methods).

Reconstitution of Crebbp/Ep300 null MEFsCrebbpflox/flox;Ep300flox/flox;YFP+ conditional (dKO) MEFs have been described39. CREBBP expression was restored by retroviral infection with constructs encoding for HA-tagged wild-type or mutant Crebbp, and cells were analyzed for H3K18 acetylation, cAMP-dependent transcriptional responses and cell proliferation (see Full Methods).

Reconstitution of Crebbp/Ep300 null MEFs

Crebbpflox/flox;Ep300flox/flox;YFP+ conditional (dKO) MEFs have been described39. CREBBP expression was restored by retroviral infection with constructs encoding for HA-tagged wild-type or mutant Crebbp, and cells were analyzed for H3K18 acetylation, cAMP-dependent transcriptional responses and cell proliferation (see Full Methods).

Full MethodsDNA extraction, amplification and sequencingGenomic DNA was extracted by standard methods, and whole genome amplification was performed using the RepliG kit (QIAGEN) according to the manufacturer’s instructions. Sequences for all annotated exons and flanking introns of CREBBP and EP300 were obtained from the UCSC Human Genome database, using the corresponding mRNA accession number as a reference (NM_004380.2 and NM_001429.3, respectively). PCR primers, located ≥50 bp upstream or downstream to target exon boundaries, were designed in the Primer 3 program (http://frodo.wi.mit.edu/primer3/) and filtered using UCSC In Silico PCR to exclude pairs yielding more than a single product (Table S5 and Table S6). Purified amplicons were sequenced directly from both strands (Genewiz, Inc, South Plainfield, NJ) as described, and compared to the corresponding germline sequences, using the Mutation Surveyor Version 2.41 software package (Softgenetics, State College, PA; http://www.softgenetics.com). Somatic mutations were confirmed on independent PCR products obtained from high molecular weight genomic DNA. Synonymous mutations, previously reported polymorphisms (Human dbSNP Database at NCBI, Build 130, and Ensembl Database) and changes present in the matched normal DNA, when available, were excluded. In cases carrying multiple events within the same gene, the allelic distribution of the mutations was determined by cloning and sequencing PCR products obtained from cDNA and spanning both events (N=10 clones each).Northern Blot analysis of CREBBP and EP300 expressionTotal RNA (12µg) was extracted from exponentially growing cell lines by TRIzol (Invitrogen) and Northern blot analysis was performed according to standard procedures, with radiolabeled probes corresponding to a 0.9Kb fragment of the human CREBBP cDNA (region 204–1143, GenBank accession No. NM_004380.2) or a 1.2Kb fragment of the human EP300 cDNA (region 6277–7524, GenBank accession No. NM_001429.3), and GAPDH as a control for loading. CREBBP and EP300 expression levels were then quantitated by Phosphorimager analysis and normalized with the GAPDH levels.Tissue microarrays, immunohistochemistry (IHC) and immunofluorescence (IF) analysisDLBCL tissue microarrays were constructed according to standard procedures and analyzed by IHC, using rabbit polyclonal antibodies directed against the N-terminus of CREBBP (A22) or EP300 (N15)(Santa Cruz Biotechnology). Cases were scored as positive if ≥20% tumor cells were stained by the antibody. IF analysis of CREBBP expression in conditional Crebbp/Ep300 dKO MEFs was performed 3 days after deletion of the endogenous loci and 5 days after infection with HA-tagged CREBBP retrovirus, using the CREBBP (A22) antibody. The specificity of both the A22 and the N15 antibodies had been previously validated on paraffin-embedded cell pellets from HEK293T cells transfected with control and CREBBP-specific shRNAs as well as by IF staining of Crebbp/Ep300 dKO MEFs (data not shown).Protein extraction and western blot analysisWhole cell extracts were prepared in RIPA buffer containing protease inhibitors as described18 and were analyzed by western blotting according to standard methods, using the following primary antibodies: anti-FLAG/M2 (Sigma), anti-HA (3F10)(Roche), anti-BCL6 (Cell Marque, GI191E/A8), anti-acetyl lysines (Cell Signaling, rabbit), anti-acetylated p53 (kind gift of W. Gu), anti p53 (DO1), anti-CREBBP (A22), anti-EP300 (N15)(all from Santa Cruz Biotechnology), anti-GFP (JL-8)(Clontech), anti-β-actin (clone AC-15) and anti-β-tubulin (clone B-5-1-2)(both from Sigma). Proteins were resolved by SDS-PAGE in 3–8% NuPAGE Tris-Acetate gels (Invitrogen), and visualized using a chemiluminescence detection kit (Pierce) as recommended by the manufacturer.Transient transfections/reporter gene assaysTransient transfections were performed in HEK293T cells using polyethylenimine (PEI), as described47. For reporter assays, cells were seeded on a 24-well plate and transfected using 100 ng of a Luciferase reporter construct driven by a PGL3-SV40-based backbone with 5 BCL6 binding sites46, 2.5 ng of pRL-SV40 (Promega), 1 ng of BCL6-encoding plasmid and the indicated doses of wild-type and mutant CREBBP expression vectors. The total amount of transfected DNA was kept constant in each experiment by adding pCMV-HA or pcDNA3 vector sequences to a final amount of 600ng/well. All experiments were performed in duplicate and luciferase activities were measured thirty-six hours after transfection using the Dual-Luciferase Reporter Assay System (Promega), according to the manufacturer’s instructions. Given the known effects of CREBBP on the SV40 promoter and on other regulatory sequences, the response of the BCL6 reporter was normalized first to that of a SV40-Renilla reporter construct, and then to the basal activity of CREBBP on the 5X-BCL6 reporter, in the absence of BCL6. Data are expressed as differences relative to the basal activity of the reporter construct (set as 1) after the above-mentioned normalization.Co-immunoprecipitation assaysTo examine the interaction between CREBBP and BCL6 or p53, HEK293T cells were co-transfected with plasmids encoding for the two proteins, together with wild-type or mutant CREBBP-HA. Thirty-six hours after transfection (or twenty-four hours for p53), cells were lysed in IP buffer (50 mM Tris, pH 7.0, 250 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.05% NP40, 10 mM sodium fluoride, 0.1mM sodium orthovanadate and protease inhibitor cocktail [Sigma]), and the cleared lysates were incubated overnight at 4°C with anti-HA or anti-Flag/M2 beads (Sigma). Immunocomplexes were eluted by incubating the beads in IP buffer containing HA or 3X-Flag peptide, respectively (Sigma). A fraction of the final eluates was resolved by SDS-PAGE and analyzed by western blot.In vitro acetyl-transferase assaysRecombinant GST-p53 and CREBBP-HA proteins were obtained as described in detail in the Supplementary Information and used for in vitro acteylation assays according to published protocols, with minor modifications14,48,49. Briefly, 10–24 ng of recombinant CREBBP-HA and 100 ng of GST-p53 were combined in 40 uL reactions containing 50 mM Tris-Cl (pH 8.0), 10% Glycerol, 1mM DTT, 1mM PMSF, 0.1 mM EDTA, 10 mM Butyric Acid (#B103500, Sigma), and variable amounts of Acetyl-CoA (Sigma)(2mM to 25 nM). As shown in Fig. 5, Acetyl-CoA concentrations as low as 25 nM were sufficient to obtain efficient acetylation of the substrate by CREBBP. Reactions were performed for 60 minutes at 30°C, and stopped by addition of an equal volume of 2X Laemmli buffer followed by heating for 10 minutes at 70°C. A fraction of the final product was resolved by SDS-PAGE in 3–8% Tris-Acetate gradient gels (Invitrogen) and analyzed by western blotting.Reconstitution of conditional Crebbp/Ep300 dKO MEFs for analysis of CREB target genes transcription, H3K18 acetylation and cell growthThe protocols for MEF isolation, cell culture and retroviral transduction, as well as the murine Crebbp (CBP)-HA retroviral construct used as template for the generation of various CREBBP mutants have been described previously50. In this system, Cre mediated recombination induces expression of YFP, allowing for specific identification of the deleted cells. Crebbpflox/flox; Ep300flox/flox; YFP+ MEFs were first infected with retroviruses encoding either wild-type or selected CREBBP point mutants, and the endogenous Crebbpflox and Ep300flox loci were deleted after 48 hours by infection with Cre-expressing adenovirus (Ad-Cre). The W1502A/Y1503S HAT-dead mutation was used as negative control40. In all experiments, transduction efficiencies were 70% or higher, as assessed by IF analysis of HA positive cells 3 days after deletion of endogenous Crebbp/Ep300 and 5 days after retroviral infection. To examine cAMP-dependent gene expression, MEFs were cultured for 16 hrs in DMEM containing 0.1% FBS and treated for 90 min with 10µM forskolin + 100µM IBMX (or ethanol vehicle) before harvesting in TRIzol (Invitrogen); qRT-PCR assays were performed as reported50. Expression of Crebbp and Ep300 was verified by IF analysis using rabbit polyclonal anti-CREBBP (A-22) and anti-EP300 (N-20) antibodies (Santa Cruz Biotechnology). Analysis of H3K18 acetylation was performed as described, using the anti-H3K18Ac antibody (ab1191) (Abcam) and the HA-11 monoclonal antibody against the HA epitope (Boehringer Mannheim). Confocal images were taken using the same settings for all mutants and mean intensity ratios for individual nuclei were collected using SlideBook 5 software. Nuclei where the Crebbp (HA) signal was at least 2.5 fold above background were used to calculate the ratio of the H3K18Ac mean signal intensity to the Crebbp (HA) mean signal intensity. For the growth assays shown in Fig. S11, Crebbpflox/flox;Ep300flox/flox;YFP+ MEFs were infected with CREBBP retroviruses 48 hours before Ad-Cre mediated deletion of endogenous Crebbp and Ep300. From each reconstituted population, equivalent numbers of YFP+ (dKO) MEFs were seeded at day 1 (i.e., the day following overnight Ad-Cre treatment), and the total number of YFP+ cells was calculated on Day 11 from the total cell number, based on the percentage of YFP+ cells as assessed by flow cytometric analysis.

Full Methods

DNA extraction, amplification and sequencingGenomic DNA was extracted by standard methods, and whole genome amplification was performed using the RepliG kit (QIAGEN) according to the manufacturer’s instructions. Sequences for all annotated exons and flanking introns of CREBBP and EP300 were obtained from the UCSC Human Genome database, using the corresponding mRNA accession number as a reference (NM_004380.2 and NM_001429.3, respectively). PCR primers, located ≥50 bp upstream or downstream to target exon boundaries, were designed in the Primer 3 program (http://frodo.wi.mit.edu/primer3/) and filtered using UCSC In Silico PCR to exclude pairs yielding more than a single product (Table S5 and Table S6). Purified amplicons were sequenced directly from both strands (Genewiz, Inc, South Plainfield, NJ) as described, and compared to the corresponding germline sequences, using the Mutation Surveyor Version 2.41 software package (Softgenetics, State College, PA; http://www.softgenetics.com). Somatic mutations were confirmed on independent PCR products obtained from high molecular weight genomic DNA. Synonymous mutations, previously reported polymorphisms (Human dbSNP Database at NCBI, Build 130, and Ensembl Database) and changes present in the matched normal DNA, when available, were excluded. In cases carrying multiple events within the same gene, the allelic distribution of the mutations was determined by cloning and sequencing PCR products obtained from cDNA and spanning both events (N=10 clones each).

DNA extraction, amplification and sequencing

Genomic DNA was extracted by standard methods, and whole genome amplification was performed using the RepliG kit (QIAGEN) according to the manufacturer’s instructions. Sequences for all annotated exons and flanking introns of CREBBP and EP300 were obtained from the UCSC Human Genome database, using the corresponding mRNA accession number as a reference (NM_004380.2 and NM_001429.3, respectively). PCR primers, located ≥50 bp upstream or downstream to target exon boundaries, were designed in the Primer 3 program (http://frodo.wi.mit.edu/primer3/) and filtered using UCSC In Silico PCR to exclude pairs yielding more than a single product (Table S5 and Table S6). Purified amplicons were sequenced directly from both strands (Genewiz, Inc, South Plainfield, NJ) as described, and compared to the corresponding germline sequences, using the Mutation Surveyor Version 2.41 software package (Softgenetics, State College, PA; http://www.softgenetics.com). Somatic mutations were confirmed on independent PCR products obtained from high molecular weight genomic DNA. Synonymous mutations, previously reported polymorphisms (Human dbSNP Database at NCBI, Build 130, and Ensembl Database) and changes present in the matched normal DNA, when available, were excluded. In cases carrying multiple events within the same gene, the allelic distribution of the mutations was determined by cloning and sequencing PCR products obtained from cDNA and spanning both events (N=10 clones each).

Northern Blot analysis of CREBBP and EP300 expressionTotal RNA (12µg) was extracted from exponentially growing cell lines by TRIzol (Invitrogen) and Northern blot analysis was performed according to standard procedures, with radiolabeled probes corresponding to a 0.9Kb fragment of the human CREBBP cDNA (region 204–1143, GenBank accession No. NM_004380.2) or a 1.2Kb fragment of the human EP300 cDNA (region 6277–7524, GenBank accession No. NM_001429.3), and GAPDH as a control for loading. CREBBP and EP300 expression levels were then quantitated by Phosphorimager analysis and normalized with the GAPDH levels.

Northern Blot analysis of CREBBP and EP300 expression

Total RNA (12µg) was extracted from exponentially growing cell lines by TRIzol (Invitrogen) and Northern blot analysis was performed according to standard procedures, with radiolabeled probes corresponding to a 0.9Kb fragment of the human CREBBP cDNA (region 204–1143, GenBank accession No. NM_004380.2) or a 1.2Kb fragment of the human EP300 cDNA (region 6277–7524, GenBank accession No. NM_001429.3), and GAPDH as a control for loading. CREBBP and EP300 expression levels were then quantitated by Phosphorimager analysis and normalized with the GAPDH levels.

Tissue microarrays, immunohistochemistry (IHC) and immunofluorescence (IF) analysisDLBCL tissue microarrays were constructed according to standard procedures and analyzed by IHC, using rabbit polyclonal antibodies directed against the N-terminus of CREBBP (A22) or EP300 (N15)(Santa Cruz Biotechnology). Cases were scored as positive if ≥20% tumor cells were stained by the antibody. IF analysis of CREBBP expression in conditional Crebbp/Ep300 dKO MEFs was performed 3 days after deletion of the endogenous loci and 5 days after infection with HA-tagged CREBBP retrovirus, using the CREBBP (A22) antibody. The specificity of both the A22 and the N15 antibodies had been previously validated on paraffin-embedded cell pellets from HEK293T cells transfected with control and CREBBP-specific shRNAs as well as by IF staining of Crebbp/Ep300 dKO MEFs (data not shown).

Tissue microarrays, immunohistochemistry (IHC) and immunofluorescence (IF) analysis

DLBCL tissue microarrays were constructed according to standard procedures and analyzed by IHC, using rabbit polyclonal antibodies directed against the N-terminus of CREBBP (A22) or EP300 (N15)(Santa Cruz Biotechnology). Cases were scored as positive if ≥20% tumor cells were stained by the antibody. IF analysis of CREBBP expression in conditional Crebbp/Ep300 dKO MEFs was performed 3 days after deletion of the endogenous loci and 5 days after infection with HA-tagged CREBBP retrovirus, using the CREBBP (A22) antibody. The specificity of both the A22 and the N15 antibodies had been previously validated on paraffin-embedded cell pellets from HEK293T cells transfected with control and CREBBP-specific shRNAs as well as by IF staining of Crebbp/Ep300 dKO MEFs (data not shown).

Protein extraction and western blot analysisWhole cell extracts were prepared in RIPA buffer containing protease inhibitors as described18 and were analyzed by western blotting according to standard methods, using the following primary antibodies: anti-FLAG/M2 (Sigma), anti-HA (3F10)(Roche), anti-BCL6 (Cell Marque, GI191E/A8), anti-acetyl lysines (Cell Signaling, rabbit), anti-acetylated p53 (kind gift of W. Gu), anti p53 (DO1), anti-CREBBP (A22), anti-EP300 (N15)(all from Santa Cruz Biotechnology), anti-GFP (JL-8)(Clontech), anti-β-actin (clone AC-15) and anti-β-tubulin (clone B-5-1-2)(both from Sigma). Proteins were resolved by SDS-PAGE in 3–8% NuPAGE Tris-Acetate gels (Invitrogen), and visualized using a chemiluminescence detection kit (Pierce) as recommended by the manufacturer.

Protein extraction and western blot analysis

Whole cell extracts were prepared in RIPA buffer containing protease inhibitors as described18 and were analyzed by western blotting according to standard methods, using the following primary antibodies: anti-FLAG/M2 (Sigma), anti-HA (3F10)(Roche), anti-BCL6 (Cell Marque, GI191E/A8), anti-acetyl lysines (Cell Signaling, rabbit), anti-acetylated p53 (kind gift of W. Gu), anti p53 (DO1), anti-CREBBP (A22), anti-EP300 (N15)(all from Santa Cruz Biotechnology), anti-GFP (JL-8)(Clontech), anti-β-actin (clone AC-15) and anti-β-tubulin (clone B-5-1-2)(both from Sigma). Proteins were resolved by SDS-PAGE in 3–8% NuPAGE Tris-Acetate gels (Invitrogen), and visualized using a chemiluminescence detection kit (Pierce) as recommended by the manufacturer.

Transient transfections/reporter gene assaysTransient transfections were performed in HEK293T cells using polyethylenimine (PEI), as described47. For reporter assays, cells were seeded on a 24-well plate and transfected using 100 ng of a Luciferase reporter construct driven by a PGL3-SV40-based backbone with 5 BCL6 binding sites46, 2.5 ng of pRL-SV40 (Promega), 1 ng of BCL6-encoding plasmid and the indicated doses of wild-type and mutant CREBBP expression vectors. The total amount of transfected DNA was kept constant in each experiment by adding pCMV-HA or pcDNA3 vector sequences to a final amount of 600ng/well. All experiments were performed in duplicate and luciferase activities were measured thirty-six hours after transfection using the Dual-Luciferase Reporter Assay System (Promega), according to the manufacturer’s instructions. Given the known effects of CREBBP on the SV40 promoter and on other regulatory sequences, the response of the BCL6 reporter was normalized first to that of a SV40-Renilla reporter construct, and then to the basal activity of CREBBP on the 5X-BCL6 reporter, in the absence of BCL6. Data are expressed as differences relative to the basal activity of the reporter construct (set as 1) after the above-mentioned normalization.

Transient transfections/reporter gene assays

Transient transfections were performed in HEK293T cells using polyethylenimine (PEI), as described47. For reporter assays, cells were seeded on a 24-well plate and transfected using 100 ng of a Luciferase reporter construct driven by a PGL3-SV40-based backbone with 5 BCL6 binding sites46, 2.5 ng of pRL-SV40 (Promega), 1 ng of BCL6-encoding plasmid and the indicated doses of wild-type and mutant CREBBP expression vectors. The total amount of transfected DNA was kept constant in each experiment by adding pCMV-HA or pcDNA3 vector sequences to a final amount of 600ng/well. All experiments were performed in duplicate and luciferase activities were measured thirty-six hours after transfection using the Dual-Luciferase Reporter Assay System (Promega), according to the manufacturer’s instructions. Given the known effects of CREBBP on the SV40 promoter and on other regulatory sequences, the response of the BCL6 reporter was normalized first to that of a SV40-Renilla reporter construct, and then to the basal activity of CREBBP on the 5X-BCL6 reporter, in the absence of BCL6. Data are expressed as differences relative to the basal activity of the reporter construct (set as 1) after the above-mentioned normalization.

Co-immunoprecipitation assaysTo examine the interaction between CREBBP and BCL6 or p53, HEK293T cells were co-transfected with plasmids encoding for the two proteins, together with wild-type or mutant CREBBP-HA. Thirty-six hours after transfection (or twenty-four hours for p53), cells were lysed in IP buffer (50 mM Tris, pH 7.0, 250 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.05% NP40, 10 mM sodium fluoride, 0.1mM sodium orthovanadate and protease inhibitor cocktail [Sigma]), and the cleared lysates were incubated overnight at 4°C with anti-HA or anti-Flag/M2 beads (Sigma). Immunocomplexes were eluted by incubating the beads in IP buffer containing HA or 3X-Flag peptide, respectively (Sigma). A fraction of the final eluates was resolved by SDS-PAGE and analyzed by western blot.

Co-immunoprecipitation assays

To examine the interaction between CREBBP and BCL6 or p53, HEK293T cells were co-transfected with plasmids encoding for the two proteins, together with wild-type or mutant CREBBP-HA. Thirty-six hours after transfection (or twenty-four hours for p53), cells were lysed in IP buffer (50 mM Tris, pH 7.0, 250 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.05% NP40, 10 mM sodium fluoride, 0.1mM sodium orthovanadate and protease inhibitor cocktail [Sigma]), and the cleared lysates were incubated overnight at 4°C with anti-HA or anti-Flag/M2 beads (Sigma). Immunocomplexes were eluted by incubating the beads in IP buffer containing HA or 3X-Flag peptide, respectively (Sigma). A fraction of the final eluates was resolved by SDS-PAGE and analyzed by western blot.

In vitro acetyl-transferase assaysRecombinant GST-p53 and CREBBP-HA proteins were obtained as described in detail in the Supplementary Information and used for in vitro acteylation assays according to published protocols, with minor modifications14,48,49. Briefly, 10–24 ng of recombinant CREBBP-HA and 100 ng of GST-p53 were combined in 40 uL reactions containing 50 mM Tris-Cl (pH 8.0), 10% Glycerol, 1mM DTT, 1mM PMSF, 0.1 mM EDTA, 10 mM Butyric Acid (#B103500, Sigma), and variable amounts of Acetyl-CoA (Sigma)(2mM to 25 nM). As shown in Fig. 5, Acetyl-CoA concentrations as low as 25 nM were sufficient to obtain efficient acetylation of the substrate by CREBBP. Reactions were performed for 60 minutes at 30°C, and stopped by addition of an equal volume of 2X Laemmli buffer followed by heating for 10 minutes at 70°C. A fraction of the final product was resolved by SDS-PAGE in 3–8% Tris-Acetate gradient gels (Invitrogen) and analyzed by western blotting.

In vitro acetyl-transferase assays

Recombinant GST-p53 and CREBBP-HA proteins were obtained as described in detail in the Supplementary Information and used for in vitro acteylation assays according to published protocols, with minor modifications14,48,49. Briefly, 10–24 ng of recombinant CREBBP-HA and 100 ng of GST-p53 were combined in 40 uL reactions containing 50 mM Tris-Cl (pH 8.0), 10% Glycerol, 1mM DTT, 1mM PMSF, 0.1 mM EDTA, 10 mM Butyric Acid (#B103500, Sigma), and variable amounts of Acetyl-CoA (Sigma)(2mM to 25 nM). As shown in Fig. 5, Acetyl-CoA concentrations as low as 25 nM were sufficient to obtain efficient acetylation of the substrate by CREBBP. Reactions were performed for 60 minutes at 30°C, and stopped by addition of an equal volume of 2X Laemmli buffer followed by heating for 10 minutes at 70°C. A fraction of the final product was resolved by SDS-PAGE in 3–8% Tris-Acetate gradient gels (Invitrogen) and analyzed by western blotting.

Reconstitution of conditional Crebbp/Ep300 dKO MEFs for analysis of CREB target genes transcription, H3K18 acetylation and cell growthThe protocols for MEF isolation, cell culture and retroviral transduction, as well as the murine Crebbp (CBP)-HA retroviral construct used as template for the generation of various CREBBP mutants have been described previously50. In this system, Cre mediated recombination induces expression of YFP, allowing for specific identification of the deleted cells. Crebbpflox/flox; Ep300flox/flox; YFP+ MEFs were first infected with retroviruses encoding either wild-type or selected CREBBP point mutants, and the endogenous Crebbpflox and Ep300flox loci were deleted after 48 hours by infection with Cre-expressing adenovirus (Ad-Cre). The W1502A/Y1503S HAT-dead mutation was used as negative control40. In all experiments, transduction efficiencies were 70% or higher, as assessed by IF analysis of HA positive cells 3 days after deletion of endogenous Crebbp/Ep300 and 5 days after retroviral infection. To examine cAMP-dependent gene expression, MEFs were cultured for 16 hrs in DMEM containing 0.1% FBS and treated for 90 min with 10µM forskolin + 100µM IBMX (or ethanol vehicle) before harvesting in TRIzol (Invitrogen); qRT-PCR assays were performed as reported50. Expression of Crebbp and Ep300 was verified by IF analysis using rabbit polyclonal anti-CREBBP (A-22) and anti-EP300 (N-20) antibodies (Santa Cruz Biotechnology). Analysis of H3K18 acetylation was performed as described, using the anti-H3K18Ac antibody (ab1191) (Abcam) and the HA-11 monoclonal antibody against the HA epitope (Boehringer Mannheim). Confocal images were taken using the same settings for all mutants and mean intensity ratios for individual nuclei were collected using SlideBook 5 software. Nuclei where the Crebbp (HA) signal was at least 2.5 fold above background were used to calculate the ratio of the H3K18Ac mean signal intensity to the Crebbp (HA) mean signal intensity. For the growth assays shown in Fig. S11, Crebbpflox/flox;Ep300flox/flox;YFP+ MEFs were infected with CREBBP retroviruses 48 hours before Ad-Cre mediated deletion of endogenous Crebbp and Ep300. From each reconstituted population, equivalent numbers of YFP+ (dKO) MEFs were seeded at day 1 (i.e., the day following overnight Ad-Cre treatment), and the total number of YFP+ cells was calculated on Day 11 from the total cell number, based on the percentage of YFP+ cells as assessed by flow cytometric analysis.

Reconstitution of conditional Crebbp/Ep300 dKO MEFs for analysis of CREB target genes transcription, H3K18 acetylation and cell growth

The protocols for MEF isolation, cell culture and retroviral transduction, as well as the murine Crebbp (CBP)-HA retroviral construct used as template for the generation of various CREBBP mutants have been described previously50. In this system, Cre mediated recombination induces expression of YFP, allowing for specific identification of the deleted cells. Crebbpflox/flox; Ep300flox/flox; YFP+ MEFs were first infected with retroviruses encoding either wild-type or selected CREBBP point mutants, and the endogenous Crebbpflox and Ep300flox loci were deleted after 48 hours by infection with Cre-expressing adenovirus (Ad-Cre). The W1502A/Y1503S HAT-dead mutation was used as negative control40. In all experiments, transduction efficiencies were 70% or higher, as assessed by IF analysis of HA positive cells 3 days after deletion of endogenous Crebbp/Ep300 and 5 days after retroviral infection. To examine cAMP-dependent gene expression, MEFs were cultured for 16 hrs in DMEM containing 0.1% FBS and treated for 90 min with 10µM forskolin + 100µM IBMX (or ethanol vehicle) before harvesting in TRIzol (Invitrogen); qRT-PCR assays were performed as reported50. Expression of Crebbp and Ep300 was verified by IF analysis using rabbit polyclonal anti-CREBBP (A-22) and anti-EP300 (N-20) antibodies (Santa Cruz Biotechnology). Analysis of H3K18 acetylation was performed as described, using the anti-H3K18Ac antibody (ab1191) (Abcam) and the HA-11 monoclonal antibody against the HA epitope (Boehringer Mannheim). Confocal images were taken using the same settings for all mutants and mean intensity ratios for individual nuclei were collected using SlideBook 5 software. Nuclei where the Crebbp (HA) signal was at least 2.5 fold above background were used to calculate the ratio of the H3K18Ac mean signal intensity to the Crebbp (HA) mean signal intensity. For the growth assays shown in Fig. S11, Crebbpflox/flox;Ep300flox/flox;YFP+ MEFs were infected with CREBBP retroviruses 48 hours before Ad-Cre mediated deletion of endogenous Crebbp and Ep300. From each reconstituted population, equivalent numbers of YFP+ (dKO) MEFs were seeded at day 1 (i.e., the day following overnight Ad-Cre treatment), and the total number of YFP+ cells was calculated on Day 11 from the total cell number, based on the percentage of YFP+ cells as assessed by flow cytometric analysis.

Supplementary Material12

Supplementary Material
