---
pmid: '19490895'
title: Proteasome-mediated turnover of the transcription coactivator NPR1 plays dual
  roles in regulating plant immunity.
authors:
- Spoel SH
- Mou Z
- Tada Y
- Spivey NW
- Genschik P
- Dong X
journal: Cell
year: '2009'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2704463
doi: 10.1016/j.cell.2009.03.038
---

# Proteasome-mediated turnover of the transcription coactivator NPR1 plays dual roles in regulating plant immunity.
**Authors:** Spoel SH, Mou Z, Tada Y, Spivey NW, Genschik P, Dong X
**Journal:** Cell (2009)
**DOI:** [10.1016/j.cell.2009.03.038](https://doi.org/10.1016/j.cell.2009.03.038)
**PMC:** [PMC2704463](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2704463/)

## Abstract

1. Cell. 2009 May 29;137(5):860-72. doi: 10.1016/j.cell.2009.03.038.

Proteasome-mediated turnover of the transcription coactivator NPR1 plays dual 
roles in regulating plant immunity.

Spoel SH(1), Mou Z, Tada Y, Spivey NW, Genschik P, Dong X.

Author information:
(1)Department of Biology, Duke University, Durham, NC 27708, USA.

Comment in
    Cell. 2009 May 29;137(5):804-6. doi: 10.1016/j.cell.2009.05.010.

Systemic acquired resistance (SAR) is a broad-spectrum plant immune response 
involving profound transcriptional changes that are regulated by the coactivator 
NPR1. Nuclear translocation of NPR1 is a critical regulatory step, but how the 
protein is regulated in the nucleus is unknown. Here, we show that turnover of 
nuclear NPR1 protein plays an important role in modulating transcription of its 
target genes. In the absence of pathogen challenge, NPR1 is continuously cleared 
from the nucleus by the proteasome, which restricts its coactivator activity to 
prevent untimely activation of SAR. Surprisingly, inducers of SAR promote NPR1 
phosphorylation at residues Ser11/Ser15, and then facilitate its recruitment to 
a Cullin3-based ubiquitin ligase. Turnover of phosphorylated NPR1 is required 
for full induction of target genes and establishment of SAR. These in vivo data 
demonstrate dual roles for coactivator turnover in both preventing and 
stimulating gene transcription to regulate plant immunity.

DOI: 10.1016/j.cell.2009.03.038
PMCID: PMC2704463
PMID: 19490895 [Indexed for MEDLINE]

## Full Text

INTRODUCTION

Immune responses are tightly regulated in all eukaryotes to ensure that they are effective only against invading pathogens but not harmful to selves. In contrast to animals, plants lack a specialized immune system and instead rely on each individual cell for defense. In response to pathogen challenge, plant cells undergo dramatic transcription reprogramming to favor immune responses over normal cellular functions. Failure to do so results in infection. On the other hand, suppressing immune responses in the absence of a pathogen threat is equally important for maintaining plant growth and development. Thus, plants have sophisticated regulatory mechanisms to control defense-related transcription.

An important signal molecule for defense-related transcription in plants is salicylic acid (SA) . Pathogen-induced increases in cellular SA levels or exogenous application of SA leads to profound changes in gene transcription (reviewed in Durrant and Dong, 2004 ). These changes occur through the activity of the transcription co-activator NPR1 ( n onexpressor of p athogenesis- r elated ( PR ) genes), a master regulator of plant immunity. Mutations in the NPR1 gene in Arabidopsis block this SA-mediated transcriptional reprogramming and renders the plant completely defective in systemic acquired resistance (SAR), an inducible immune response against a broad-spectrum of pathogens ( Cao et al., 1994 ; Delaney et al., 1995 ; Wang et al., 2006 ).

The activity of NPR1 is regulated in part by its subcellular localization ( Kinkema et al., 2000 ). In unchallenged cells NPR1 is predominantly sequestered in the cytoplasm as a high molecular weight oligomeric complex ( Mou et al., 2003 ). The oligomeric complex is formed through redox-sensitive intermolecular disulfide bonds between conserved cysteine residues. Upon pathogen infection, accumulation of SA triggers a change in cellular reduction potential, resulting in partial reduction of NPR1 oligomer to monomer. A bipartite nuclear localization sequence targets the released NPR1 monomer to the nucleus where it functions as a co-activator of gene transcription ( Kinkema et al., 2000 ). Furthermore, NPR1 was found to interact with TGA transcription factors ( Després et al., 2000 ; Zhang et al., 1999 ; Zhou et al., 2000 ) whose binding motif has been shown to be essential for SA-responsiveness of the PR-1 gene ( Lebel et al., 1998 ). NPR1 may affect both the DNA binding capacity and the activity of TGA factors ( Després et al., 2003 ; Després et al., 2000 ; Fan and Dong, 2002 ; Johnson et al., 2003 ; Rochon et al., 2006 ). Besides the PR genes, which encode antimicrobial effectors, NPR1 also directly activates the expression of several WRKY transcription factors with both activator and suppressor activities ( Wang et al., 2006 ). Thus, NPR1 regulates plant immunity through a transcription cascade involving multiple transcription factors.

A major challenge in understanding the function of NPR1 is to uncover the nuclear regulation of this co-activator. Phosphorylation and ubiquitin-mediated proteolysis are prominent post-translational mechanisms that control transcription regulators. In mammalian immunity, the co-factor IκB, which shares structural features with NPR1 ( Cao et al., 1997 ; Ryals et al., 1997 ), functions to sequester the transcription factor NF-κB in the cytoplasm and prevents it from activating gene expression. In response to pathogen attack, IκB is rapidly phosphorylated and targeted for ubiquitin-mediated proteolysis, allowing NF-κB to localize to the nucleus and activate target genes ( Hayden and Ghosh, 2004 ). Furthermore, transcription factors are often unstable and a significant overlap has been found between transcriptional activation domains and domains that regulate ubiquitin-mediated proteolysis ( Salghetti et al., 2000 ). Recent findings indicate that proteasome-mediated turnover of activators may be essential for their ability to activate transcription ( Collins and Tansey, 2006 ). Whereas activator turnover is thought to stimulate gene transcription, it remains largely unknown if proteolysis plays a role in the regulation of transcription co-activators.

In this study we investigated if the co-activator NPR1 is regulated by post-translational mechanisms. Our findings revealed opposing roles for co-activator proteolysis in the regulation of gene transcription and demonstrate for the first time that multi-cellular organisms employ proteolysis-coupled transcription as a mechanism to control their responses to external stimuli.

DISCUSSION

Proteasome-mediated protein degradation plays a pivotal role in many plant growth and developmental processes, including photomorphogenesis and plant hormone signaling ( Smalle and Vierstra, 2004 ). In these instances the proteasome either degrades activators to suppress transcription or degrades repressor proteins to activate gene expression. Our study discovered that proteasome-mediated degradation of NPR1 not only prevents untimely gene activation, but also plays an essential role in stimulating gene expression during plant immune responses.

NPR1 monomer is constitutively cleared from the nucleus by a CUL3-based ubiquitin ligase to restrict its transcription co-activator activity ( Figures 1 and 2 ). A similar regulatory mechanism was observed recently for the yeast transcription factor GAL4, a regulator of galactose metabolism ( Muratani et al., 2005 ). In the absence of galactose, GAL4 activity is limited by proteasome-mediated destruction, preventing wasteful activation of metabolic genes. Proteasome-mediated degradation of transcription factors was also found to prevent inappropriate transcription at tissue specific gene loci in embryonic stem cells ( Szutorisz et al., 2006 ). Importantly, our data suggests that the proteasome may restrict gene transcription not only by destruction of transcription factors, but also by proteolysis of co-activators to prevent assembly of active transcriptional complexes ( Figure 7 ). In the case of NPR1, this mechanism renders SAR inactive to avoid detrimental fitness costs associated with constitutive defense ( Heidel et al., 2004 ; van Hulten et al., 2006 ).

To our surprise, activation of SAR did not prevent CUL3-mediated degradation of NPR1. Instead, blocking NPR1 turnover by inhibition of proteasome activity, genetically knocking down CUL3 activity, and mutating the IκB-like phosphodegron, all compromised transcription of the NPR1 target genes WRKY18 , WRKY38 , and WRKY62 ( Figures 3 , 4 , and 6 ), indicating that NPR1 turnover also stimulates transcription. Compared to the WRKY genes, expression of PR-1 was less dependent on the proteasome ( Figure S7 ), suggesting that NPR1 target genes may require different rates of NPR1 turnover. Recent reports have demonstrated that in yeast and human cells, transcription factors are often unstable and their instability correlates with their ability to induce target gene transcription ( Kim et al., 2003 ; Lipford et al., 2005 ; Muratani et al., 2005 ; Reid et al., 2003 ; von der Lehr et al., 2003 ). It is thought that turnover of transcription factors promotes gene expression by continuously delivering ‘fresh’ activator to gene promoters. This may be necessary to sustain a high ratio of transcriptional active over inactive activator ( Collins and Tansey, 2006 ). Although mono-ubiquitinylation of co-activators has been reported to stimulate their activity ( Wu et al., 2007 ), a role for co-activator proteolysis in gene transcription has not been established. In fact, it was previously reported that the turnover of co-activators associated with the human estrogen receptor-α did not stimulate estrogen receptor-mediated transcription ( Lonard et al., 2000 ). Our findings suggest, however, that gene activation by proteolysis is not limited to transcription factors, but also includes transcription co-activators like NPR1 ( Figure 7 ).

Turnover of NPR1 is limited by its nuclear translocation ( Figure 1 ). We recently reported that S -nitrosylation of Cys156 in NPR1 facilitates oligomer formation in the cytoplasm. Mutating Cys156 to alanine (C156A) inhibited this regulatory step and caused depletion of the npr1C156A protein in response to SA treatment ( Tada et al., 2008 ). Here, we found that the C156A mutation does not affect the protein turnover rate in our cell-free degradation assay ( Figure S11 ). Instead, the SA-induced instability of this mutant protein in planta was reversed by inhibition of the proteasome activity ( Figure S12 ). Thus, nuclear turnover of NPR1 presented in this report underlined the importance of S -nitrosylation-mediated oligomerization in the cytoplasm to maintain NPR1 homeostasis. Upon SAR induction, a large amount of NPR1 monomer is released from the cytoplasmic oligomer, translocated into the nucleus, and subsequently turned over. To maintain protein homeostasis, oligomerization of NPR1 in the cytoplasm is facilitated by a pathogen-induced increase in cellular GSNO levels ( Tada et al., 2008 ). NPR1 oligomerization and monomer release occur sequentially according to the SA-induced biphasic redox changes ( Tada et al., 2008 ), which may be responsible for the observed fluctuations in NPR1-GFP levels after avirulent pathogen inoculation ( Figure 6 ). Because a constitutive promoter was used to drive the expression of NPR1-GFP, we were able to detect these dynamic changes in protein stability. Such fluctuations are much harder to detect for the endogenous NPR1 protein as NPR1 gene transcription also fluctuates with the redox changes (data not shown).

NPR1 contains a conserved N-terminal phosphodegron motif ( Figure 5C ) that is found in many unstable transcriptional regulators, including IκB, β-Catenin, c-Myc, c-Jun, and SRC-3 ( Hayden and Ghosh, 2004 ; Karin and Ben-Neriah, 2000 ; Wu et al., 2007 ). Phosphorylation of the phosphodegron motif signals the destruction of these regulators by recruiting CUL1-based ubiquitin ligases. We showed that SA-induced phosphorylation of Ser11 and Ser15 in the phosphodegron motif of NPR1 also promotes its poly-ubiquitinylation and degradation ( Figures 5 and 6 ). Although site-specific (de)phosphorylation has been found to regulate substrate recruitment to CUL1-based ubiquitin ligases ( Petroski and Deshaies, 2005 ), the mechanisms that control substrate delivery to CUL3 complexes are not yet well understood. We demonstrated that the NPR1 phosphodegron regulates interaction with a CUL3-based ubiquitin ligase and promotes NPR1 turnover ( Figures 6A-6D ), suggesting that phosphorylation may also be a common mechanism to target substrates for CUL3-mediated proteolysis. Similar to CUL1-substrate interaction, phosphorylation of Ser11/15 may create or stabilize a binding site for the CUL3-based ubiquitin ligase ( Figure 7 ). The role of Ser11/15 phosphorylation-mediated degradation as a key regulatory switch for SAR was clearly revealed by the diminished transcription of NPR1 target genes and the failure to induce resistance in 35S::npr1S11/15A-GFP mutant plants ( Figures 6E and 6F ). This conclusion was further supported by the phenotype of the 35S::npr1S11/15D-GFP mutant, which has a higher basal level of resistance but is also insensitive to SAR induction ( Figure 6F ).

Turnover of NPR1 plays dual roles in regulating the transcription of target genes. Whereas CUL3-mediated degradation prevented NPR1 from initiating transcription in uninduced cells, it was necessary for full-scale activation of transcription in SAR-induced cells. Thus, NPR1 may be targeted for degradation by the CUL3-based ligase in distinct ways. Indeed, we identified phosphorylation of NPR1 as an important functional switch for its CUL3-mediated transcription activity: phosphorylation was required for SAR-induced turnover of NPR1, but was dispensable for basal turnover ( Figure 6 ). The mechanism by which phosphorylation may promote the activity of proteasome-regulated activators was investigated for the yeast activator GCN4, an essential regulator of amino acid biosynthesis genes. Inhibition of the proteasome reduced the ability of GCN4 to recruit RNA polymerase II to its target promoters ( Lipford et al., 2005 ). Interestingly, degradation of GCN4 is signaled by SRB10, a cyclin-dependent protein kinase that is intimately associated with RNA polymerase II ( Chi et al., 2001 ; Liao et al., 1995 ), suggesting that GCN4 is degraded after it has initiated transcription. This supports a model in which phosphorylation labels an activator as “exhausted” once it has initiated transcription. Subsequent removal of the activator by proteasome-mediated degradation may allow “fresh” activator to bind the target promoter and re-initiate transcription ( Collins and Tansey, 2006 ; Kodadek et al., 2006 ; Lipford et al., 2005 ). As shown in the working model in Figure 7 , we hypothesize that a similar mechanism may regulate the SA-induced activity of NPR1 as its phosphorylation is inducible ( Figure 5 ), specifically occurs in the nucleus ( Figure 5E ), and stimulates turnover-mediated gene transcription ( Figure 6 ). Whereas SAR-induced phosphorylated NPR1 is probably turned over after interaction with the target promoter, the uninduced non-phosphorylated NPR1 may be degraded before binding to the target gene promoter ( Figure 7 ). Where exactly in the nucleus these events occur will be the subject of future investigation. Moreover, the identity of the substrate adaptor for the NPR1-CUL3 interaction has yet to be revealed and whether the same adaptor binds to non-phosphorylated NPR1 and phosphorylated NPR1 needs to be determined. Regardless of these specifics, this study clearly demonstrates that phosphorylation-mediated turnover of distinct components of transcriptional complexes (e.g. transcription factors, co-activators) may be a common mechanism by which single-cellular as well as multi-cellular organisms regulate gene transcription.
