---
pmid: '15525680'
title: Mycobacterium tuberculosis functional network analysis by global subcellular
  protein profiling.
authors:
- Mawuenyega KG
- Forst CV
- Dobos KM
- Belisle JT
- Chen J
- Bradbury EM
- Bradbury AR
- Chen X
journal: Mol Biol Cell
year: '2005'
full_text_available: false
pmcid: PMC539182
doi: 10.1091/mbc.e04-04-0329
---

# Mycobacterium tuberculosis functional network analysis by global subcellular protein profiling.
**Authors:** Mawuenyega KG, Forst CV, Dobos KM, Belisle JT, Chen J, Bradbury EM, Bradbury AR, Chen X
**Journal:** Mol Biol Cell (2005)
**DOI:** [10.1091/mbc.e04-04-0329](https://doi.org/10.1091/mbc.e04-04-0329)
**PMC:** [PMC539182](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC539182/)

## Abstract

1. Mol Biol Cell. 2005 Jan;16(1):396-404. doi: 10.1091/mbc.e04-04-0329. Epub 2004
 Nov 3.

Mycobacterium tuberculosis functional network analysis by global subcellular 
protein profiling.

Mawuenyega KG(1), Forst CV, Dobos KM, Belisle JT, Chen J, Bradbury EM, Bradbury 
AR, Chen X.

Author information:
(1)Cell Biology, Structural Biology, and Flow Cytometry, Bioscience Division, 
Los Alamos National Laboratory, Los Alamos, NM 87545, USA.

Trends in increased tuberculosis infection and a fatality rate of approximately 
23% have necessitated the search for alternative biomarkers using newly 
developed postgenomic approaches. Here we provide a systematic analysis of 
Mycobacterium tuberculosis (Mtb) by directly profiling its gene products. This 
analysis combines high-throughput proteomics and computational approaches to 
elucidate the globally expressed complements of the three subcellular 
compartments (the cell wall, membrane, and cytosol) of Mtb. We report the 
identifications of 1044 proteins and their corresponding localizations in these 
compartments. Genome-based computational and metabolic pathways analyses were 
performed and integrated with proteomics data to reconstruct response networks. 
From the reconstructed response networks for fatty acid degradation and lipid 
biosynthesis pathways in Mtb, we identified proteins whose involvements in these 
pathways were not previously suspected. Furthermore, the subcellular 
localizations of these expressed proteins provide interesting insights into the 
compartmentalization of these pathways, which appear to traverse from cell wall 
to cytoplasm. Results of this large-scale subcellular proteome profile of Mtb 
have confirmed and validated the computational network hypothesis that 
functionally related proteins work together in larger organizational structures.

DOI: 10.1091/mbc.e04-04-0329
PMCID: PMC539182
PMID: 15525680 [Indexed for MEDLINE]
