---
pmid: '32750333'
title: Complex Autoinflammatory Syndrome Unveils Fundamental Principles of JAK1 Kinase
  Transcriptional and Biochemical Function.
authors:
- Gruber CN
- Calis JJA
- Buta S
- Evrony G
- Martin JC
- Uhl SA
- Caron R
- Jarchin L
- Dunkin D
- Phelps R
- Webb BD
- Saland JM
- Merad M
- Orange JS
- Mace EM
- Rosenberg BR
- Gelb BD
- Bogunovic D
journal: Immunity
year: '2020'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC7398039
doi: 10.1016/j.immuni.2020.07.006
---

# Complex Autoinflammatory Syndrome Unveils Fundamental Principles of JAK1 Kinase Transcriptional and Biochemical Function.
**Authors:** Gruber CN, Calis JJA, Buta S, Evrony G, Martin JC, Uhl SA, Caron R, Jarchin L, Dunkin D, Phelps R, Webb BD, Saland JM, Merad M, Orange JS, Mace EM, Rosenberg BR, Gelb BD, Bogunovic D
**Journal:** Immunity (2020)
**DOI:** [10.1016/j.immuni.2020.07.006](https://doi.org/10.1016/j.immuni.2020.07.006)
**PMC:** [PMC7398039](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7398039/)

## Abstract

1. Immunity. 2020 Sep 15;53(3):672-684.e11. doi: 10.1016/j.immuni.2020.07.006.
Epub  2020 Aug 3.

Complex Autoinflammatory Syndrome Unveils Fundamental Principles of JAK1 Kinase 
Transcriptional and Biochemical Function.

Gruber CN(1), Calis JJA(2), Buta S(1), Evrony G(3), Martin JC(4), Uhl SA(1), 
Caron R(1), Jarchin L(5), Dunkin D(6), Phelps R(7), Webb BD(8), Saland JM(9), 
Merad M(10), Orange JS(11), Mace EM(11), Rosenberg BR(1), Gelb BD(8), Bogunovic 
D(12).

Author information:
(1)Department of Microbiology, Icahn School of Medicine at Mount Sinai, New 
York, NY 10029, USA.
(2)Program in Immunogenomics, The Rockefeller University, New York, NY 10065, 
USA; Laboratory of Virology and Infectious Disease, The Rockefeller University, 
New York, NY 10065, USA; Center for Translational Immunology, Department of 
Pediatric Immunology & Rheumatology, University Medical Center Utrecht, 
University of Utrecht, Utrecht, the Netherlands.
(3)Center for Genetics and Genomics, New York University Grossman School of 
Medicine, New York, NY, USA.
(4)Precision Immunology Institute, Icahn School of Medicine at Mount Sinai, New 
York, NY 10029, USA; Department of Oncological Sciences, Icahn School of 
Medicine at Mount Sinai, New York, NY 10029, USA; Université de Nantes, Inserm, 
CHU Nantes, Centre de Recherche en Transplantation et Immunologie, UMR 1064, 
ITUN, 44000 Nantes, France; CHU Nantes, Laboratoire d'Immunologie, Center for 
Immuno Monitoring Nantes-Atlantique (CIMNA), 44000 Nantes, France.
(5)Department of Pediatrics, Icahn School of Medicine at Mount Sinai, New York, 
NY 10029, USA; Department of Gastroenterology, Icahn School of Medicine at Mount 
Sinai, New York, NY 10029, USA.
(6)Department of Pediatrics, Icahn School of Medicine at Mount Sinai, New York, 
NY 10029, USA; Department of Gastroenterology, Icahn School of Medicine at Mount 
Sinai, New York, NY 10029, USA; Mindich Child Health and Development Institute, 
Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA.
(7)Department of Dermatology, Icahn School of Medicine at Mount Sinai, New York, 
NY 10029, USA; Department of Pathology, Icahn School of Medicine at Mount Sinai, 
New York, NY 10029, USA.
(8)Department of Pediatrics, Icahn School of Medicine at Mount Sinai, New York, 
NY 10029, USA; Department of Genetics and Genomic Sciences, Icahn School of 
Medicine at Mount Sinai, New York, NY 10029, USA; Mindich Child Health and 
Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY 
10029, USA.
(9)Department of Pediatrics, Icahn School of Medicine at Mount Sinai, New York, 
NY 10029, USA.
(10)Precision Immunology Institute, Icahn School of Medicine at Mount Sinai, New 
York, NY 10029, USA; Department of Oncological Sciences, Icahn School of 
Medicine at Mount Sinai, New York, NY 10029, USA.
(11)Department of Pediatrics, Vagelos College of Physicians and Surgeons, 
Columbia University Irving Medical Center, New York, NY 10032, USA.
(12)Department of Microbiology, Icahn School of Medicine at Mount Sinai, New 
York, NY 10029, USA; Precision Immunology Institute, Icahn School of Medicine at 
Mount Sinai, New York, NY 10029, USA; Department of Pediatrics, Icahn School of 
Medicine at Mount Sinai, New York, NY 10029, USA; Mindich Child Health and 
Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY 
10029, USA. Electronic address: dusan.bogunovic@mssm.edu.

Comment in
    Immunity. 2020 Sep 15;53(3):481-484. doi: 10.1016/j.immuni.2020.08.010.

Autoinflammatory disease can result from monogenic errors of immunity. We 
describe a patient with early-onset multi-organ immune dysregulation resulting 
from a mosaic, gain-of-function mutation (S703I) in JAK1, encoding a kinase 
essential for signaling downstream of >25 cytokines. By custom single-cell RNA 
sequencing, we examine mosaicism with single-cell resolution. We find that JAK1 
transcription was predominantly restricted to a single allele across different 
cells, introducing the concept of a mutational "transcriptotype" that differs 
from the genotype. Functionally, the mutation increases JAK1 activity and 
transactivates partnering JAKs, independent of its catalytic domain. S703I JAK1 
is not only hypermorphic for cytokine signaling but also neomorphic, as it 
enables signaling cascades not canonically mediated by JAK1. Given these 
results, the patient was treated with tofacitinib, a JAK inhibitor, leading to 
the rapid resolution of clinical disease. These findings offer a platform for 
personalized medicine with the concurrent discovery of fundamental biological 
principles.

Copyright © 2020 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.immuni.2020.07.006
PMCID: PMC7398039
PMID: 32750333 [Indexed for MEDLINE]

Conflict of interest statement: Declaration of Interests The authors declare no 
competing interests.

## Full Text

IntroductionMonogenic disease mutations afford the opportunity to study the bona fide function of human genes in vivo, which have guided our understanding of biology and medicine for decades. Undiagnosed disease programs, by means of next-generation sequencing, have recently provided a platform to identify, diagnose, and study these rare patients with unusual clinical presentations (Lee et al., 2014; Splinter et al., 2018; Yang et al., 2014). In turn, clinical management can, in some cases, be highly personalized.To date, studies of rare immunologic diseases have identified germline gain-of-function (GoF) and loss-of-function (LoF) mutations throughout the Janus kinase-signal transducer and activator of transcription (JAK-STAT) signaling axis (Dupuis et al., 2001; Etheridge et al., 2014; Flanagan et al., 2014; Hambleton et al., 2013; Holland et al., 2007; Kofoed et al., 2003; Macchi et al., 1995; Mead et al., 2012; Minegishi et al., 2006, 2007; Russell et al., 1995; van de Veerdonk et al., 2011), the primary signal transduction pathway for cytokines. The Janus kinase (JAK) family contains four tyrosine kinases (JAK1, JAK2, JAK3, TYK2) constitutively associated with cytokine receptors. Upon cytokine binding, JAKs act in partnership to phosphorylate themselves, the receptors, and then STATs, which can then act directly as transcription factors or activate other signaling pathways further downstream (O’Shea et al., 2015). JAK1 is activated by a broad range of cytokines (γc, gp130, interferon [IFN], and interleukin-10 [IL-10] family cytokines). It can phosphorylate any signal transducer and activator of transcription (STAT) protein (STAT1–6) and is universally expressed in all tissues (O’Shea et al., 2015). Through the formation of specific combinations of cytokine receptors, JAK partners, and STAT dimers, JAK1 orchestrates unique downstream signals for each cytokine.The need to better understand JAK regulation has deepened with the expanding clinical use of JAK inhibitors (O’Shea and Gadina, 2019). The breadth of successfully treated inflammatory conditions signifies the central pathophysiological role of JAK hyperactivity across immune diseases. However, the complete list of disorders resulting from JAK-STAT dysregulation remains unknown. Furthermore, it is unclear which specific JAK-mediated pathways drive disease, a key issue for the design of inhibitors with greater selectivity for individual members of the JAK family.Herein, we identify a mutation (S703I) of JAK1 in a patient with a severe, early-onset immunodysregulatory syndrome identified in our undiagnosed disease program. Using extensive next-generation genomic, molecular, and multi-parametric immunological tools, we probe the effects of S703I JAK1 in vitro and ex vivo to investigate clinical dysfunction in vivo.

Introduction

Monogenic disease mutations afford the opportunity to study the bona fide function of human genes in vivo, which have guided our understanding of biology and medicine for decades. Undiagnosed disease programs, by means of next-generation sequencing, have recently provided a platform to identify, diagnose, and study these rare patients with unusual clinical presentations (Lee et al., 2014; Splinter et al., 2018; Yang et al., 2014). In turn, clinical management can, in some cases, be highly personalized.

To date, studies of rare immunologic diseases have identified germline gain-of-function (GoF) and loss-of-function (LoF) mutations throughout the Janus kinase-signal transducer and activator of transcription (JAK-STAT) signaling axis (Dupuis et al., 2001; Etheridge et al., 2014; Flanagan et al., 2014; Hambleton et al., 2013; Holland et al., 2007; Kofoed et al., 2003; Macchi et al., 1995; Mead et al., 2012; Minegishi et al., 2006, 2007; Russell et al., 1995; van de Veerdonk et al., 2011), the primary signal transduction pathway for cytokines. The Janus kinase (JAK) family contains four tyrosine kinases (JAK1, JAK2, JAK3, TYK2) constitutively associated with cytokine receptors. Upon cytokine binding, JAKs act in partnership to phosphorylate themselves, the receptors, and then STATs, which can then act directly as transcription factors or activate other signaling pathways further downstream (O’Shea et al., 2015). JAK1 is activated by a broad range of cytokines (γc, gp130, interferon [IFN], and interleukin-10 [IL-10] family cytokines). It can phosphorylate any signal transducer and activator of transcription (STAT) protein (STAT1–6) and is universally expressed in all tissues (O’Shea et al., 2015). Through the formation of specific combinations of cytokine receptors, JAK partners, and STAT dimers, JAK1 orchestrates unique downstream signals for each cytokine.

The need to better understand JAK regulation has deepened with the expanding clinical use of JAK inhibitors (O’Shea and Gadina, 2019). The breadth of successfully treated inflammatory conditions signifies the central pathophysiological role of JAK hyperactivity across immune diseases. However, the complete list of disorders resulting from JAK-STAT dysregulation remains unknown. Furthermore, it is unclear which specific JAK-mediated pathways drive disease, a key issue for the design of inhibitors with greater selectivity for individual members of the JAK family.

Herein, we identify a mutation (S703I) of JAK1 in a patient with a severe, early-onset immunodysregulatory syndrome identified in our undiagnosed disease program. Using extensive next-generation genomic, molecular, and multi-parametric immunological tools, we probe the effects of S703I JAK1 in vitro and ex vivo to investigate clinical dysfunction in vivo.

ResultsComplex Immunodysregulatory SyndromeWe studied an 18-year-old female who was referred to our undiagnosed disease program with a complex primary autoinflammatory and atopic syndrome (Figure 1
A). The patient was born to a non-consanguineous family with no history of immunologic disease. At birth, the patient was noted to have a widespread pustular rash in a linear pattern that predominantly affected the left side of the body (Figure 1B). The lesions continually progressed, and a biopsy later revealed inflammatory linear verrucous epidermal nevus (Figure S1A). At ~1 year of age, she developed recurrent emesis and diarrhea. Repeat endoscopic biopsies demonstrated chronic, unspecified inflammation at various sites (most frequently colonic, but also gastric, duodenal, ileal, and esophageal regions). Eosinophilic infiltration of the colon was consistently noted (Figures 1C and S1B). Likewise, peripheral eosinophilia with a fluctuating pattern was frequently observed (Figure S1C). At 3 years of age, she developed rapid weight gain, edema, and proteinuria. Renal biopsy demonstrated membranous nephropathy (MN) (Figures 1D and S1D), which was refractory to treatment with corticosteroids, and later, cyclosporine and tacrolimus. According to her history, the nephrotic syndrome was ameliorated via an elemental diet, but this was not able to be consistently maintained. Of note, serology was negative for known MN autoantibodies or other autoreactive antibodies (Figure S1E). Kidney transplantation was performed at age 11, but MN recurred within 1 year, followed by antibody-mediated rejection, requiring subsequent hemodialysis. Over this time, she also experienced asthma, food and environmental allergies, severely stunted growth with leg length discrepancy, and poor weight gain (for an extended clinical report, see STAR Methods).Figure 1De Novo Mutation in JAK1 Identified in a Patient with Immunodysregulatory Syndrome(A) Schematic representing clinical history of the patient, with gray bars representing the kinetics of each disease feature.(B) Photograph of the dermatologic lesions on the arm.(C) Histology of the cecal mucosa showing expansion of the lamina propria secondary to increased inflammatory cell infiltrate, with eosinophils in the lamina propria and crypt epithelium (arrows).(D) Electron microscopy of a renal biopsy obtained during disease recurrence that demonstrates irregular glomerular basement membranes and subepithelial and intramembranous immune type dense deposits.(E) Patient’s family pedigree.(F) Whole-exome sequencing reads mapping to JAK1 locus c.2108, with variant nucleotides displayed in green.(G) Representative chromatograms from 3 independent experiments of Sanger sequencing of peripheral blood DNA to confirm c.2108 G > T JAK1.(H) Proportion of cells carrying the heterozygous JAK1 mutation, as estimated by digital droplet PCR with WT- and mutation-specific probes. DNA was obtained from bilateral cheek swabs, Ficoll-fractionated whole blood, and epithelial tissue isolated from a colonic biopsy (n = 1).(I) Model for the development of the de novo mutation and its distribution into all 3 germ layers.See also Figure S1.Whole-Exome Sequencing Reveals a De Novo JAK1 MutationGiven the overall healthy state of the parents and the early onset of disease in the patient, we hypothesized that either a recessive or de novo genetic mutation was the cause of the clinical syndrome (Figure 1E). We performed whole-exome sequencing on peripheral blood cells obtained from the patient and her parents. Subsequent variant analysis failed to produce any likely variants by a recessive model of inheritance (Table S1). Because of the asymmetric manifestations of disease, including limb length discrepancy and irregularly distributed dermatitis, we then considered the possibility of lower-read-frequency de novo mosaic mutations, which are typically excluded from common analysis pipelines. One candidate de novo variant, JAK1 c.2108G > T, which constituted 27% of the reads mapping to the region, was identified (Figures 1E and 1F). The presence of the c.2108G > T variant was confirmed by Sanger sequencing (Figure 1G), and this variant was absent from all of the publicly available genome sequences from healthy individuals. This mutation results in the substitution of serine to isoleucine at position 703 (S703I) in a highly conserved region (Figure S1F) and is predicted to be highly damaging (combined annotation-dependent depletion [CADD] score of 27.6). We then investigated the presence of c.2108G > T in non-hematopoietic tissues. We performed digital droplet PCR (ddPCR) with mutation-specific probes to estimate the fraction of cells carrying the mutation in different tissues. We identified the mutation at various frequencies in DNA from buccal swabs, granulocytes, peripheral blood mononuclear cells (PBMCs), and endoscopic biopsy samples fractionated into epithelia and associated immune cells (Figures 1H and S1G). These tissues represent all three germ layers, signifying that the mutation must have arisen in the first ~12 cell divisions between fertilization and gastrulation (Figure 1I) (Moore et al., 2015).Allele Characterization Indicates that S703I Confers a GoF on JAK1The S703I mutation localizes to the pseudokinase domain of JAK1, a putative regulatory domain (Figure 2
A). Although S703I is located between the germline JAK1 mutations identified to date, these other mutations diverged in their downstream consequences (LoF and GoF), making functional predictions for S703I difficult (Del Bel et al., 2017, Eletto et al., 2016). To assess the possible pathogenicity of the mutation and its impact on JAK1 function, we transduced WT JAK1, S703I JAK1, and empty vector lentiviruses into U4C cells, a fibrosarcoma cell line previously selected to lack endogenous JAK1 (Pellegrini et al., 1989). Transduction with S703I JAK1, but not WT JAK1 or Luciferase, led to basal phosphorylation of STAT proteins and active target gene transcription in the absence of cytokine stimulation (Figures 2B–2E). S703I-transduced cells hyperresponded to IFN-α, in terms of both the proximal phosphorylation of STAT1 and STAT2 and the induction of IFN-stimulated genes (ISGs) (Figures 2B and 2C). Similarly, these cells hyperphosphorylated downstream STATs in response to IFN-γ or IL-6 (Figures 2D, 2E, and S2A).Figure 2S703I JAK1 Confers Constitutively Active and Hyperresponsive STAT Signaling(A) Localization of the S703I (red) mutation to the pseudokinase domain (blue) of JAK1, as represented in the linear sequence and the predicted structure, as modeled on TYK2.(B) Immunoblotting for STAT phosphorylation in U4C Cells (JAK1−/−) reconstituted with lentiviruses for vector control (luciferase), WT JAK1, or S703I JAK1 and stimulated for 15 min with indicated doses of IFN-α (IU/mL).(C) qPCR from transduced U4C cells for IFN-stimulated gene expression (MX1 and RSAD2) at baseline and following 100 IU/mL of IFN-α for 8 h (n = 3). Columns represent means and error bars represent standard deviations. ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.001, 2-tailed Student’s t test with Welch’s correction.(D and E) Stimulation of transduced U4C cells with IFN-γ (0.1 ng/mL) (D) or IL-6 (25 ng/mL) (E) for 15 min.(F and G) Derivation of B-EBV cells and isolation of JAK1 WT/WT and JAK1 S703I/WT cells from patient blood, followed by stimulation with IFN-α (100 IU/mL) (F) or IFN-γ (0.1 ng/mL) (G) for 15 min. All of the data are representative of at least 3 independent experiments.See also Figure S2.For direct confirmation of the pathogenicity of the mutation in cells from the patient, we derived an EBV-immortalized B cell (B-EBV) line from the patient’s PBMCs. Given the mosaicism for JAK1 in the patient’s cells, individual lines were cloned from single cells to derive purely wild-type (WT) or S703I heterozygous mutant cells (Figure S2B). A comparison of STAT phosphorylation in patient WT and mutant B-EBV cells supported the GoF role of S703I JAK1, both at baseline and in response to cytokines (Figures 2F and 2G). The isogenic control derived from the same patient pinpointed S703I JAK1 as the probable pathogenic mutation in the patient’s genome. These results indicate that S703I is GoF for basal- and cytokine-induced STAT signaling.S703I JAK1 Transactivates Partnering JAKs Independently of Its Own Kinase ActivityTo dissect the mechanisms underlying upregulated STAT signaling, we assessed the impact of S703I on JAK auto-phosphorylation. Consistent with the increase in STAT phosphorylation, S703I JAK1 was itself hyperphosphorylated (Figure 3
A). In addition, JAK2, TYK2, and JAK3 phosphorylation were upregulated (Figures 3B–3D and S2C), suggesting that the interacting JAK partners may also play a role in the GoF. We reasoned that partnering JAK activity could be overactivated by JAK1 S703I from two mechanisms: increased formation of the receptor complex or direct crosstalk between JAK proteins. We assessed whether JAK1 S703I allowed for more cytokine receptor at the surface, as the structural domains of JAK1 scaffold the receptor complex (Li et al., 2013). However, surface staining for the type I IFN receptor subunit (IFNAR2) demonstrated equivalent receptor expression in WT JAK1 and S703I JAK1 cells, indicating that JAK1 scaffolding of the receptor complex was unaffected by the patient’s mutation (Figures 3E and S2D).Figure 3S703I JAK1 Mediates Hyperactive STAT Phosphorylation by Transactivation of Partnering JAKs(A) Immunoprecipitation of JAK1 from JAK1-transduced U4C cells and immunoblotting for total phosphorylation (4G10) after stimulation with IFN-α (100 IU/mL) for 15 min.(B) Immunoprecipitation of JAK2 from patient-derived B-EBV cells and western blotting for phosphorylation at the activation loop after stimulation with IFN-γ (0.1 ng/mL) for 15 min.(C) Immunoprecipitation of TYK2 from JAK1-transduced U4C cells and western blotting for phosphorylation at the activation loop after stimulation with IFN-α (100 IU/mL) for 15 min.(D) Immunoprecipitation of JAK3 from patient-derived B-EBV cells and western blotting for total phosphorylation stimulation with IL-21 (50 ng/mL) for 15 min.(E) Surface staining of the IFNAR2 subunit by flow cytometry in U4C cells transduced with the indicated constructs. Lines indicate means and error bars represent SD. ∗∗∗p < 0.001, ****p < 0.0001, analysis of variance (ANOVA) with Tukey's post-hoc analysis.(F and G) Transduction of U4C cells with catalytically inactivated JAK1 (K908A), WT JAK1, S703I JAK1, or double-mutant JAK1 (K908A S703I), followed by stimulation with IFN-α (1,000 IU/mL) (F) or IFN-γ (1.0 ng/mL) (G).(H) Transduction of U4C cells with catalytically inactivated JAK1 (K908A), WT JAK1, A634D JAK1, or double-mutant JAK1 (K908A A634D), followed by stimulation with IFN-α (1000 IU/mL) or IFN-γ (1.0 ng/mL).(I) Inhibition of STAT phosphorylation by a 4-h treatment with saturating doses (1 and 10 μM) of JAK inhibitors followed by 15-min stimulation with IFN-α (1,000 IU/mL).All of the data are representative of at least 3 independent experiments.See also Figure S2.Next, we hypothesized that the mutant JAK1 pseudokinase domain transactivated the kinase activity of JAK2 and TYK2. To investigate this mechanism, we mutated the ATP-binding site (K908A) of JAK1 to render it catalytically inactive. This well-characterized mutation retains the signaling capability of the receptor complex, making it possible to study signaling by the partnering JAKs in isolation (Eletto et al., 2016; Li et al., 2013). As expected, STAT phosphorylation was largely reduced in the absence of JAK1 activity (Figures 3F and 3G). However, following inactivation of the kinase domain of S703I JAK1 (S703I/K908A), an aberrant increase in STAT phosphorylation relative to kinase-inactivated JAK1 without the S703I mutation was observed upon cytokine stimulation (Figures 3F and 3G). This result indicates that S703I JAK1 transactivated JAK2 and TYK2, revealing that pseudokinase domains can regulate partnering JAKs in trans, in addition to traditionally understood cis-regulation (Babon et al., 2014). This mechanism was conserved with the other reported JAK1 GoF mutation (A634D, Del Bel e al 2017) (Figure 3H), but not to the JAK2 GoF mutation (V617F) that is common in hematologic malignancy (Figure S2E).Finally, to clinically substantiate the importance of partnering JAK activity in JAK1 S703I (and other JAK-mediated diseases), we compared the potency of selective and non-selective JAK inhibitors, filgotinib and tofacitinib, respectively (Changelian et al., 2003, Rompaey et al., 2013). Unlike a pan-JAK inhibitor, a JAK1-specific inhibitor was unable to completely abrogate JAK1 S703I signaling owing to the continued signaling of partnering JAKs transactivated by the JAK1 pseudokinase (Figures 3I and S2F, schematic). These findings underscore the biochemical and clinical importance of trans-regulation and indicates that JAK selectivity need not mean clinical efficacy. This notion not only challenges the current dogma in drug development but it also proves that precision medicine must be used on a mutation-specific basis.Ex Vivo Analysis Reveals a Cell-Intrinsic GoFTo more robustly investigate the consequences of S703I on immune cells, we performed mass cytometry (CyTOF) immunophenotyping on whole blood from the patient. Despite the central role of JAK1 in immune cell differentiation and proliferation, the patient’s immune cell distribution was largely within the normal range, barring a few exceptions (Figures 4A and 4B; Table S2). As detected clinically, eosinophils fluctuated to high counts (Figure S1C). B cells, however, trended toward lower frequencies than in healthy donors, but largely retained a normal distribution of naive, memory, and class-switched cells (Figures S3A–S3C). Natural killer (NK) cells exhibited an increase in CD56hi cells (>12-fold over healthy controls) (Figures 4B, S3E, and S3F), a subset understood to be less mature but highly proliferative and rapid in cytokine production as compared to the majority CD56lo subset with high cytotoxicity (Figures S3G and S3H) (Poli et al., 2009). Extensive phenotyping of the patient’s NK cells was performed concurrently with STAT1 GoF NK cells, which also demonstrated functional immaturity (Vargas-Hernandez et al., 2017). This analysis revealed that JAK1 GoF NK cells aligned phenotypically with prototypical immature “CD56bright” NK cells, unlike those from STAT1 GoF patients (Figure S3I). However, whether this phenotype results from the primary pathology or clinical intervention remains to be determined in this case.Figure 4CyTOF Analysis Reveals Cytokine-, STAT-, and Cell-Type-Specific Gain of Function(A) Representative (tSNE) plots generated from immunophenotyping CyTOF data of whole blood from 2 independent experiments.(B) Manually gated CyTOF populations from the whole blood of 5 separate healthy controls and the patient on 2 separate occasions (2X) were quantified as the percentage of single cells and expressed as relative frequency (patient/controls). The gray bars indicate means with standard deviations of healthy donors, and colored bars indicate means with standard deviations of patient. Multiple t tests performed correcting for multiple comparisons using the Holm-Sidak method. ∗∗∗p < 0.001.(C) Relative MSI of phospho-STAT staining from intracellular phospho-CyTOF of whole blood from 4 healthy donors (n = 4) and the patient on 3 separate occasions (3X). The columns represent means and the error bars represent standard deviations.(D) Ex vivo stimulation with IFN-α (100 IU/mL), IL-2 (50 ng/mL), and IL-4 (50 ng/mL) for 15 min. The color intensity indicates the log2 fold-change in MSI over the unstimulated healthy control for each cell type (n = 1).Cent Mem, central memory; C Mono, classical monocyte; DN, double negative; DP, double positive; Eff Mem, effector memory; FC, fold change; HC, healthy control; ILC, innate lymphoid cells; I Mono, intermediate monocytes; MAIT, mucosal associate invariant T cells; mDC, myeloid dendritic cell; MSI, mean signal intensity; NC Mono, non-classical monocyte; P, patient; pDC, plasmacytoid dendritic cell; Th, T helper; . tSNE, t-distributed stochastic neighbor embedding.See also Figures S3 and S4.We then performed phospho-CyTOF to analyze the phosphorylation of all STATs downstream of JAK1 in all of the major immune cells of whole blood. Given the ubiquitous expression and diverse signaling capabilities of JAK1, as well as the basal activity of S703I observed in vitro, we hypothesized that all STATs within all immune cells would be hyperphosphorylated at baseline. Heightened phosphorylation of STAT1, STAT3, STAT4, STAT5, and STAT6 was observed, but not universally. Certain immune subsets, but not others, exhibited high basal phosphorylation of specific STATs (Figure 4C). For example, granulocytes from the patient displayed baseline STAT1 phosphorylation, but not STAT3 phosphorylation, whereas T cells displayed basal STAT3 phosphorylation but not STAT1 phosphorylation. STAT6 phosphorylation, however, was not upregulated in any immune subset, except B cells.To evaluate the functional consequences of the observed basal phosphorylation, we assessed the expression of downstream genes from bulk PBMCs. We detected the elevated expression of downstream pSTAT target genes, including IFI27, IFIT1, MX1, and SIGLEC1 (Figure S4A). In addition, we tested non-hematopoietic tissues for baseline STAT phosphorylation by immunohistochemistry. In skin and gastrointestinal biopsies, but not renal biopsy, we detected highly phosphorylated STAT1 and STAT3 as compared to healthy samples (Figure S4B). This basal phosphorylation was observed in the apparent absence of any overt increase in circulating JAK-STAT cytokine concentrations (Figure S4C), suggesting that intrinsic S703I JAK1 activity drove this process, consistent with our in vitro results (Figures 2B–2F).Ex Vivo Cytokine Stimulation Generates Non-canonical Signaling Pathways in Patient CellsWhole blood was then stimulated ex vivo with a series of cytokines that engage JAK1 with various cytokine receptors, JAK partners, and downstream STAT targets. In response to IFN-α or IL-2, patient leukocytes hyperphosphorylated STAT1 and STAT3 or STAT5, respectively (Figures 4D and S4D and S4E, represented plots). By contrast, STAT6 phosphorylation in response to IL-4 was similar to that in healthy control cells, as seen in the baseline STAT6 data (Figure 4D). Likewise, IL-5 stimulation led to normal STAT5 phosphorylation (Figure S5F). These differential responses were also noted in patient B-EBV cells (Figures S5G–S5I). However, stimulation with either IL-2 or IL-4 induced the phosphorylation of STAT1 exclusively in patient cells, contrasting the canonical signaling cascade induced by these cytokines (Figure 4D). This non-canonical response suggests that S703I confers promiscuity onto JAK1, allowing it to transverse traditional signaling axes and establish non-canonical pathways. These results indicate that S703I JAK1 is a GoF mutation ex vivo, displaying both unexpected pathway promiscuity and specificity in the activation of STAT signaling.Custom scRNA-Seq Maps Cellular Distribution and Transcriptomic Signatures of S703I JAK1To follow up on the differences across immune cell types, we performed single-cell RNA sequencing (scRNA-seq) of the patient’s PBMCs. We aimed to specifically map and evaluate the impact of S703I JAK1 across immune cell subsets by relating single-cell resolution gene expression patterns to cell type and, given the mosaicism, JAK1 genotype. However, sequence data from droplet-based scRNA-seq platforms are typically restricted to 5′ or 3′ transcript regions and therefore do not include coverage of the S703I site (c.2108) in JAK1 mRNA (Figure S5A). Therefore, using custom barcode microbeads and a modified library preparation procedure, we adapted the inDrops scRNA-seq methodology (Klein et al., 2015; Zilionis et al., 2017) to target exon 16 (containing the S703I site) of JAK1 in addition to the standard mRNA 3′ regions (Figure S5B; Method Details). We used this custom inDrops scRNA-seq to analyze the patient’s PBMCs, and concurrently processed the same sample on the 10X Genomics Chromium platform (Figures S5C and S5D). Clustering and manual annotation of cell types based on gene expression patterns distinguished the expected PBMC populations, with relative frequencies consistent with the corresponding CyTOF analyses (Figures 5
A and S5D; Table S3). Again, we observed a relative increase in CD56hi NK cell frequency. Using sequence data from JAK1-targeted libraries, we assigned putative per-cell JAK1 genotypes (based on transcript sequences) to a subset of PBMCs for which transcript and read depths were sufficient in the JAK1-targeted inDrops dataset (Figures S5E and S5F). Within this subset, we found that the mutant allele was not evenly distributed across immune cell-type clusters (Figures 5B and 5C). For example, B cells and monocytes mostly carried WT transcrip, whereas 69% of CD56hi NK cells contained JAK1 mRNA with the S703I mutation. This variable distribution may reflect differences in the intrinsic tolerance to the mutation in these cell types, which is consistent with the expansion of the CD56hi NK cell subset and the higher expression of JAK1 in NK cells than in other cell types (Figure S5G).Figure 5Custom scRNA-Seq Maps JAK1 Allele Distribution, Transcriptomic Impact, and Expression Patterns(A) tSNE plots and cell-type assignments from scRNA-seq of patient PBMCs, with an inDrops platform adapted to target the mutant JAK1 transcript. n = 4,763 cells.(B) tSNE plots representing the subset of cells with sufficient JAK1 counts to be assigned putative JAK1 genotypes (based on transcript sequences). Cells in which any mutant transcript was detected above empirically determined thresholds (> 5 JAK1 transcripts) were assigned “S703I JAK1” (purple), while cells with only WT transcript detected were assigned “WT JAK1” (orange).(C) Doughnut charts quantifying allele distribution in cells meeting genotyping criteria (cell count in the center), as in (B).(D) Expression of the ISG IFI44L, a statistically significant differentially expressed gene in the comparison of WT JAK1 and S703I JAK1 genotyped cells.(E) Gene set scores for IFN-α signaling in CD14+ monocytes.(F) Number of unique transcripts detected per cell for the WT or S703I JAK1 allele (left) or a control variant GNLY rs12845 (right). The bubble size indicates the number of cells. The color coding indicates cells containing S703I JAK1 (purple), WT JAK1 without S703I JAK1 (orange), WT JAK1 with S703I JAK1 detected below threshold (yellow), or insufficient transcripts counts (gray).(G) Transcript genotyping of JAK1 rs2230587 from healthy control PBMCs (n = 96) by single-cell qPCR with allele-specific probes. The histogram represents the relative frequency of cells expressing binned allele ratios as quantified by oligonucleotide standards.(I) Single-cell qPCR transcript genotyping of control gene NACA (rs4902).See also Figure S5.We next performed differential gene expression analysis, comparing patient cells expressing the WT and mutant alleles of JAK1. Although this analysis was constrained by a limited number of cells for which transcript-level genotype information was available, the intrasample comparison can be conducted on an inherently isogenic background with identical exposure history. We detected statistically significant differences in ISG expression for IFI44L and for an ISG set between WT and mutant monocytes (Figures 5D and 5E). These data suggest that tolerance to S703I may differ between cell types and confirm the cell-intrinsic nature of the GoF described above.JAK1 Expression Is Restricted to a Single Allele across Immune CellsGiven the mosaicism of a heterozygous mutation, we expected to observe some cells (i.e., homozygous WT JAK1) containing only WT JAK1 transcripts and others (i.e., heterozygous S703I JAK1) containing both WT and S703I JAK1 transcripts. However, we found that expression of the 2 alleles seemed almost mutually exclusive, as very few cells expressed both transcripts, as opposed to the ~50% that was expected given our genomic estimates (Figures 1D and 5F, left panel). By contrast, other genes from our dataset containing heterozygous variants exhibited the expected biallelic distribution (Figure 5F, right panel). This result suggests that JAK1 may be subject to monoallelic bias, a pattern that has only recently been recognized in a fraction of the autosomal transcriptome (Borel et al., 2015; Deng et al., 2014; Gimelbrant et al., 2007; Jeffries et al., 2012). We further tested this hypothesis of biased allele expression by sorting single cells from healthy donor PBMCs heterozygous for a synonymous SNP (rs2230587) of JAK1, adjacent to S703I. qPCR of isolated RNA with allele-specific probes revealed that relative expression of the two alleles was not normally distributed, but rather biased to one allele or the other (Figure 5G), unlike in a control gene (Figure 5H). A query of publicly available murine expression data revealed a similar allele restriction for JAK1, which, at least in mice, remains fixed over time (Savova et al., 2016). Whether the observed phenomenon in this patient represents transcriptional bursting or mitotically stable monoallelic expression and the potential impact on immune dysfunction remains to be fully determined. In either sense, these data perhaps indicate a departure from the classic genetic interpretation of heterozygosity and allow for a shift in understanding of the genetic penetrance of disease.Clinical and Biological Immune Dysfunction Can Be Rescued with TofacitinibHaving identified JAK1 hyperactivity as the putative driver of clinical disease in the patient, we then considered the use of JAK inhibitors for the clinical treatment of this patient. We compared the ability of the two US Food and Drug Administration (FDA)-approved pan-JAK inhibitors available at the time to reduce basal STAT phosphorylation in S703I-transduced U4C cells. Despite its lower relative potency against JAK1, tofacitinib inhibited STAT phosphorylation in a comparable dose response to ruxolitinib (Figure 6
A), again reflecting the independence of JAK1 catalysis and STAT hyperphosphorylation. Similar results were obtained with patient-derived B-EBV cells, which, unlike the transduced fibrosarcoma U4C cells, are hematopoietic in origin and therefore express JAK3 (Figure 6B). Next, we treated patient blood ex vivo with the two compounds at equimolar doses that mimic physiological dosing (Chen et al., 2014; Krishnaswami et al., 2014; Lamba et al., 2016), and we further assessed the inhibition of IFN-α stimulation. Analysis of phospho-STAT inhibition across whole-blood immune subsets by phospho-CyTOF revealed that tofacitinib attenuated the response more potently than ruxolitinib in nearly all cell types (Figure 6C).Figure 6Treatment with Tofacitinib Rescues STAT Hyperphosphorylation and Resolves Clinical Disease(A) In vitro assessment of JAK-inhibitor efficacy after 4-h drug treatment for resolution of basal STAT phosphorylation in transduced U4Cs.(B) Similar analysis in patient-derived B-EBVs, with doses indicated.(C) Ex vivo inhibition with equimolar doses (500 nM) of ruxolitinib and tofacitinib for 4 h followed by IFN-α stimulation (1,000 IU/mL) for 15 min (n = 1).(D) C-Reactive Protein monitoring from patient sera over the course of treatment. Dotted line represents the upper limit of normal (ULN).(E) Erythrocyte sedimentation rate in patient blood over the course of treatment, with a dotted line indicating the ULN.(F) Gross appearance of skin before and 5 months after tofacitinib treatment (Tx). (Image of the arm also in Figure 1.)(G) Colonic biopsies before and 6 months post-therapy.(H) ISG expression in RNA from bulk PBMCs isolated throughout treatment (n = 1). The dotted line represents the average expression across 3 healthy controls.(I) Phospho-CyTOF analysis comparing relative changes in pSTAT MSI from before and 10 months post-therapy (n = 1).See also Figure S6.Following these extensive functional studies, we treated the patient with low-dose tofacitinib (5 mg daily). Within 8 weeks, circulating inflammatory markers (erythrocyte sedimentation rate [ESR] and C-reactive protein) normalized (Figures 6D and 6E). Near-complete improvement in dermatitis followed, both grossly and histologically (Figure 6F). By 6 months, the patient reported complete resolution of gastrointestinal symptoms (decrease in modified Pediatric Ulcerative Colitis Activity Index [PUCAI] from 35–50 initially to 0). Biopsy of colonic tissue revealed the restoration of crypt architecture and the complete resolution of eosinophilic infiltrates (Figure 6G). The patient remained stable for 2 years after initiation of therapy until, unfortunately, the patient succumbed to acute respiratory failure due to coronavirus disease 2019 (COVID-19).Finally, we confirmed the pharmacological rescue of JAK hyperactivity in the patient’s cells after tofacitinib treatment. RNA isolated from PBMCs revealed that the expression of ISGs, which was elevated before treatment, progressively declined to normal amounts (Figure 6H). CyTOF analysis was then performed to confirm the decrease in basal STAT phosphorylation (Figure S6). Reductions were observed across cell types in STAT3, STAT4, STAT5, and STAT6 phosphorylation, whereas STAT1 phosphorylation was reduced in some, but not all cell types (Figure 6I). Overall, these results validate S703I JAK1 as the etiology in vivo of the widespread immune dysregulation (summarized in Table S4), and they illustrate the power of precision medicine both as a treatment approach for patients with rare diseases and as a means of discovering fundamental physiological mechanisms.

Complex Immunodysregulatory SyndromeWe studied an 18-year-old female who was referred to our undiagnosed disease program with a complex primary autoinflammatory and atopic syndrome (Figure 1
A). The patient was born to a non-consanguineous family with no history of immunologic disease. At birth, the patient was noted to have a widespread pustular rash in a linear pattern that predominantly affected the left side of the body (Figure 1B). The lesions continually progressed, and a biopsy later revealed inflammatory linear verrucous epidermal nevus (Figure S1A). At ~1 year of age, she developed recurrent emesis and diarrhea. Repeat endoscopic biopsies demonstrated chronic, unspecified inflammation at various sites (most frequently colonic, but also gastric, duodenal, ileal, and esophageal regions). Eosinophilic infiltration of the colon was consistently noted (Figures 1C and S1B). Likewise, peripheral eosinophilia with a fluctuating pattern was frequently observed (Figure S1C). At 3 years of age, she developed rapid weight gain, edema, and proteinuria. Renal biopsy demonstrated membranous nephropathy (MN) (Figures 1D and S1D), which was refractory to treatment with corticosteroids, and later, cyclosporine and tacrolimus. According to her history, the nephrotic syndrome was ameliorated via an elemental diet, but this was not able to be consistently maintained. Of note, serology was negative for known MN autoantibodies or other autoreactive antibodies (Figure S1E). Kidney transplantation was performed at age 11, but MN recurred within 1 year, followed by antibody-mediated rejection, requiring subsequent hemodialysis. Over this time, she also experienced asthma, food and environmental allergies, severely stunted growth with leg length discrepancy, and poor weight gain (for an extended clinical report, see STAR Methods).Figure 1De Novo Mutation in JAK1 Identified in a Patient with Immunodysregulatory Syndrome(A) Schematic representing clinical history of the patient, with gray bars representing the kinetics of each disease feature.(B) Photograph of the dermatologic lesions on the arm.(C) Histology of the cecal mucosa showing expansion of the lamina propria secondary to increased inflammatory cell infiltrate, with eosinophils in the lamina propria and crypt epithelium (arrows).(D) Electron microscopy of a renal biopsy obtained during disease recurrence that demonstrates irregular glomerular basement membranes and subepithelial and intramembranous immune type dense deposits.(E) Patient’s family pedigree.(F) Whole-exome sequencing reads mapping to JAK1 locus c.2108, with variant nucleotides displayed in green.(G) Representative chromatograms from 3 independent experiments of Sanger sequencing of peripheral blood DNA to confirm c.2108 G > T JAK1.(H) Proportion of cells carrying the heterozygous JAK1 mutation, as estimated by digital droplet PCR with WT- and mutation-specific probes. DNA was obtained from bilateral cheek swabs, Ficoll-fractionated whole blood, and epithelial tissue isolated from a colonic biopsy (n = 1).(I) Model for the development of the de novo mutation and its distribution into all 3 germ layers.See also Figure S1.

Complex Immunodysregulatory Syndrome

We studied an 18-year-old female who was referred to our undiagnosed disease program with a complex primary autoinflammatory and atopic syndrome (Figure 1
A). The patient was born to a non-consanguineous family with no history of immunologic disease. At birth, the patient was noted to have a widespread pustular rash in a linear pattern that predominantly affected the left side of the body (Figure 1B). The lesions continually progressed, and a biopsy later revealed inflammatory linear verrucous epidermal nevus (Figure S1A). At ~1 year of age, she developed recurrent emesis and diarrhea. Repeat endoscopic biopsies demonstrated chronic, unspecified inflammation at various sites (most frequently colonic, but also gastric, duodenal, ileal, and esophageal regions). Eosinophilic infiltration of the colon was consistently noted (Figures 1C and S1B). Likewise, peripheral eosinophilia with a fluctuating pattern was frequently observed (Figure S1C). At 3 years of age, she developed rapid weight gain, edema, and proteinuria. Renal biopsy demonstrated membranous nephropathy (MN) (Figures 1D and S1D), which was refractory to treatment with corticosteroids, and later, cyclosporine and tacrolimus. According to her history, the nephrotic syndrome was ameliorated via an elemental diet, but this was not able to be consistently maintained. Of note, serology was negative for known MN autoantibodies or other autoreactive antibodies (Figure S1E). Kidney transplantation was performed at age 11, but MN recurred within 1 year, followed by antibody-mediated rejection, requiring subsequent hemodialysis. Over this time, she also experienced asthma, food and environmental allergies, severely stunted growth with leg length discrepancy, and poor weight gain (for an extended clinical report, see STAR Methods).Figure 1De Novo Mutation in JAK1 Identified in a Patient with Immunodysregulatory Syndrome(A) Schematic representing clinical history of the patient, with gray bars representing the kinetics of each disease feature.(B) Photograph of the dermatologic lesions on the arm.(C) Histology of the cecal mucosa showing expansion of the lamina propria secondary to increased inflammatory cell infiltrate, with eosinophils in the lamina propria and crypt epithelium (arrows).(D) Electron microscopy of a renal biopsy obtained during disease recurrence that demonstrates irregular glomerular basement membranes and subepithelial and intramembranous immune type dense deposits.(E) Patient’s family pedigree.(F) Whole-exome sequencing reads mapping to JAK1 locus c.2108, with variant nucleotides displayed in green.(G) Representative chromatograms from 3 independent experiments of Sanger sequencing of peripheral blood DNA to confirm c.2108 G > T JAK1.(H) Proportion of cells carrying the heterozygous JAK1 mutation, as estimated by digital droplet PCR with WT- and mutation-specific probes. DNA was obtained from bilateral cheek swabs, Ficoll-fractionated whole blood, and epithelial tissue isolated from a colonic biopsy (n = 1).(I) Model for the development of the de novo mutation and its distribution into all 3 germ layers.See also Figure S1.

De Novo Mutation in JAK1 Identified in a Patient with Immunodysregulatory Syndrome

(A) Schematic representing clinical history of the patient, with gray bars representing the kinetics of each disease feature.

(B) Photograph of the dermatologic lesions on the arm.

(C) Histology of the cecal mucosa showing expansion of the lamina propria secondary to increased inflammatory cell infiltrate, with eosinophils in the lamina propria and crypt epithelium (arrows).

(D) Electron microscopy of a renal biopsy obtained during disease recurrence that demonstrates irregular glomerular basement membranes and subepithelial and intramembranous immune type dense deposits.

(E) Patient’s family pedigree.

(F) Whole-exome sequencing reads mapping to JAK1 locus c.2108, with variant nucleotides displayed in green.

(G) Representative chromatograms from 3 independent experiments of Sanger sequencing of peripheral blood DNA to confirm c.2108 G > T JAK1.

(H) Proportion of cells carrying the heterozygous JAK1 mutation, as estimated by digital droplet PCR with WT- and mutation-specific probes. DNA was obtained from bilateral cheek swabs, Ficoll-fractionated whole blood, and epithelial tissue isolated from a colonic biopsy (n = 1).

(I) Model for the development of the de novo mutation and its distribution into all 3 germ layers.

See also Figure S1.

Whole-Exome Sequencing Reveals a De Novo JAK1 MutationGiven the overall healthy state of the parents and the early onset of disease in the patient, we hypothesized that either a recessive or de novo genetic mutation was the cause of the clinical syndrome (Figure 1E). We performed whole-exome sequencing on peripheral blood cells obtained from the patient and her parents. Subsequent variant analysis failed to produce any likely variants by a recessive model of inheritance (Table S1). Because of the asymmetric manifestations of disease, including limb length discrepancy and irregularly distributed dermatitis, we then considered the possibility of lower-read-frequency de novo mosaic mutations, which are typically excluded from common analysis pipelines. One candidate de novo variant, JAK1 c.2108G > T, which constituted 27% of the reads mapping to the region, was identified (Figures 1E and 1F). The presence of the c.2108G > T variant was confirmed by Sanger sequencing (Figure 1G), and this variant was absent from all of the publicly available genome sequences from healthy individuals. This mutation results in the substitution of serine to isoleucine at position 703 (S703I) in a highly conserved region (Figure S1F) and is predicted to be highly damaging (combined annotation-dependent depletion [CADD] score of 27.6). We then investigated the presence of c.2108G > T in non-hematopoietic tissues. We performed digital droplet PCR (ddPCR) with mutation-specific probes to estimate the fraction of cells carrying the mutation in different tissues. We identified the mutation at various frequencies in DNA from buccal swabs, granulocytes, peripheral blood mononuclear cells (PBMCs), and endoscopic biopsy samples fractionated into epithelia and associated immune cells (Figures 1H and S1G). These tissues represent all three germ layers, signifying that the mutation must have arisen in the first ~12 cell divisions between fertilization and gastrulation (Figure 1I) (Moore et al., 2015).

Whole-Exome Sequencing Reveals a De Novo JAK1 Mutation

Given the overall healthy state of the parents and the early onset of disease in the patient, we hypothesized that either a recessive or de novo genetic mutation was the cause of the clinical syndrome (Figure 1E). We performed whole-exome sequencing on peripheral blood cells obtained from the patient and her parents. Subsequent variant analysis failed to produce any likely variants by a recessive model of inheritance (Table S1). Because of the asymmetric manifestations of disease, including limb length discrepancy and irregularly distributed dermatitis, we then considered the possibility of lower-read-frequency de novo mosaic mutations, which are typically excluded from common analysis pipelines. One candidate de novo variant, JAK1 c.2108G > T, which constituted 27% of the reads mapping to the region, was identified (Figures 1E and 1F). The presence of the c.2108G > T variant was confirmed by Sanger sequencing (Figure 1G), and this variant was absent from all of the publicly available genome sequences from healthy individuals. This mutation results in the substitution of serine to isoleucine at position 703 (S703I) in a highly conserved region (Figure S1F) and is predicted to be highly damaging (combined annotation-dependent depletion [CADD] score of 27.6). We then investigated the presence of c.2108G > T in non-hematopoietic tissues. We performed digital droplet PCR (ddPCR) with mutation-specific probes to estimate the fraction of cells carrying the mutation in different tissues. We identified the mutation at various frequencies in DNA from buccal swabs, granulocytes, peripheral blood mononuclear cells (PBMCs), and endoscopic biopsy samples fractionated into epithelia and associated immune cells (Figures 1H and S1G). These tissues represent all three germ layers, signifying that the mutation must have arisen in the first ~12 cell divisions between fertilization and gastrulation (Figure 1I) (Moore et al., 2015).

Allele Characterization Indicates that S703I Confers a GoF on JAK1The S703I mutation localizes to the pseudokinase domain of JAK1, a putative regulatory domain (Figure 2
A). Although S703I is located between the germline JAK1 mutations identified to date, these other mutations diverged in their downstream consequences (LoF and GoF), making functional predictions for S703I difficult (Del Bel et al., 2017, Eletto et al., 2016). To assess the possible pathogenicity of the mutation and its impact on JAK1 function, we transduced WT JAK1, S703I JAK1, and empty vector lentiviruses into U4C cells, a fibrosarcoma cell line previously selected to lack endogenous JAK1 (Pellegrini et al., 1989). Transduction with S703I JAK1, but not WT JAK1 or Luciferase, led to basal phosphorylation of STAT proteins and active target gene transcription in the absence of cytokine stimulation (Figures 2B–2E). S703I-transduced cells hyperresponded to IFN-α, in terms of both the proximal phosphorylation of STAT1 and STAT2 and the induction of IFN-stimulated genes (ISGs) (Figures 2B and 2C). Similarly, these cells hyperphosphorylated downstream STATs in response to IFN-γ or IL-6 (Figures 2D, 2E, and S2A).Figure 2S703I JAK1 Confers Constitutively Active and Hyperresponsive STAT Signaling(A) Localization of the S703I (red) mutation to the pseudokinase domain (blue) of JAK1, as represented in the linear sequence and the predicted structure, as modeled on TYK2.(B) Immunoblotting for STAT phosphorylation in U4C Cells (JAK1−/−) reconstituted with lentiviruses for vector control (luciferase), WT JAK1, or S703I JAK1 and stimulated for 15 min with indicated doses of IFN-α (IU/mL).(C) qPCR from transduced U4C cells for IFN-stimulated gene expression (MX1 and RSAD2) at baseline and following 100 IU/mL of IFN-α for 8 h (n = 3). Columns represent means and error bars represent standard deviations. ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.001, 2-tailed Student’s t test with Welch’s correction.(D and E) Stimulation of transduced U4C cells with IFN-γ (0.1 ng/mL) (D) or IL-6 (25 ng/mL) (E) for 15 min.(F and G) Derivation of B-EBV cells and isolation of JAK1 WT/WT and JAK1 S703I/WT cells from patient blood, followed by stimulation with IFN-α (100 IU/mL) (F) or IFN-γ (0.1 ng/mL) (G) for 15 min. All of the data are representative of at least 3 independent experiments.See also Figure S2.For direct confirmation of the pathogenicity of the mutation in cells from the patient, we derived an EBV-immortalized B cell (B-EBV) line from the patient’s PBMCs. Given the mosaicism for JAK1 in the patient’s cells, individual lines were cloned from single cells to derive purely wild-type (WT) or S703I heterozygous mutant cells (Figure S2B). A comparison of STAT phosphorylation in patient WT and mutant B-EBV cells supported the GoF role of S703I JAK1, both at baseline and in response to cytokines (Figures 2F and 2G). The isogenic control derived from the same patient pinpointed S703I JAK1 as the probable pathogenic mutation in the patient’s genome. These results indicate that S703I is GoF for basal- and cytokine-induced STAT signaling.

Allele Characterization Indicates that S703I Confers a GoF on JAK1

The S703I mutation localizes to the pseudokinase domain of JAK1, a putative regulatory domain (Figure 2
A). Although S703I is located between the germline JAK1 mutations identified to date, these other mutations diverged in their downstream consequences (LoF and GoF), making functional predictions for S703I difficult (Del Bel et al., 2017, Eletto et al., 2016). To assess the possible pathogenicity of the mutation and its impact on JAK1 function, we transduced WT JAK1, S703I JAK1, and empty vector lentiviruses into U4C cells, a fibrosarcoma cell line previously selected to lack endogenous JAK1 (Pellegrini et al., 1989). Transduction with S703I JAK1, but not WT JAK1 or Luciferase, led to basal phosphorylation of STAT proteins and active target gene transcription in the absence of cytokine stimulation (Figures 2B–2E). S703I-transduced cells hyperresponded to IFN-α, in terms of both the proximal phosphorylation of STAT1 and STAT2 and the induction of IFN-stimulated genes (ISGs) (Figures 2B and 2C). Similarly, these cells hyperphosphorylated downstream STATs in response to IFN-γ or IL-6 (Figures 2D, 2E, and S2A).Figure 2S703I JAK1 Confers Constitutively Active and Hyperresponsive STAT Signaling(A) Localization of the S703I (red) mutation to the pseudokinase domain (blue) of JAK1, as represented in the linear sequence and the predicted structure, as modeled on TYK2.(B) Immunoblotting for STAT phosphorylation in U4C Cells (JAK1−/−) reconstituted with lentiviruses for vector control (luciferase), WT JAK1, or S703I JAK1 and stimulated for 15 min with indicated doses of IFN-α (IU/mL).(C) qPCR from transduced U4C cells for IFN-stimulated gene expression (MX1 and RSAD2) at baseline and following 100 IU/mL of IFN-α for 8 h (n = 3). Columns represent means and error bars represent standard deviations. ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.001, 2-tailed Student’s t test with Welch’s correction.(D and E) Stimulation of transduced U4C cells with IFN-γ (0.1 ng/mL) (D) or IL-6 (25 ng/mL) (E) for 15 min.(F and G) Derivation of B-EBV cells and isolation of JAK1 WT/WT and JAK1 S703I/WT cells from patient blood, followed by stimulation with IFN-α (100 IU/mL) (F) or IFN-γ (0.1 ng/mL) (G) for 15 min. All of the data are representative of at least 3 independent experiments.See also Figure S2.

S703I JAK1 Confers Constitutively Active and Hyperresponsive STAT Signaling

(A) Localization of the S703I (red) mutation to the pseudokinase domain (blue) of JAK1, as represented in the linear sequence and the predicted structure, as modeled on TYK2.

(B) Immunoblotting for STAT phosphorylation in U4C Cells (JAK1−/−) reconstituted with lentiviruses for vector control (luciferase), WT JAK1, or S703I JAK1 and stimulated for 15 min with indicated doses of IFN-α (IU/mL).

(C) qPCR from transduced U4C cells for IFN-stimulated gene expression (MX1 and RSAD2) at baseline and following 100 IU/mL of IFN-α for 8 h (n = 3). Columns represent means and error bars represent standard deviations. ∗p < 0.05, ∗∗p < 0.01, ∗∗∗p < 0.001, 2-tailed Student’s t test with Welch’s correction.

(D and E) Stimulation of transduced U4C cells with IFN-γ (0.1 ng/mL) (D) or IL-6 (25 ng/mL) (E) for 15 min.

(F and G) Derivation of B-EBV cells and isolation of JAK1 WT/WT and JAK1 S703I/WT cells from patient blood, followed by stimulation with IFN-α (100 IU/mL) (F) or IFN-γ (0.1 ng/mL) (G) for 15 min. All of the data are representative of at least 3 independent experiments.

See also Figure S2.

For direct confirmation of the pathogenicity of the mutation in cells from the patient, we derived an EBV-immortalized B cell (B-EBV) line from the patient’s PBMCs. Given the mosaicism for JAK1 in the patient’s cells, individual lines were cloned from single cells to derive purely wild-type (WT) or S703I heterozygous mutant cells (Figure S2B). A comparison of STAT phosphorylation in patient WT and mutant B-EBV cells supported the GoF role of S703I JAK1, both at baseline and in response to cytokines (Figures 2F and 2G). The isogenic control derived from the same patient pinpointed S703I JAK1 as the probable pathogenic mutation in the patient’s genome. These results indicate that S703I is GoF for basal- and cytokine-induced STAT signaling.

S703I JAK1 Transactivates Partnering JAKs Independently of Its Own Kinase ActivityTo dissect the mechanisms underlying upregulated STAT signaling, we assessed the impact of S703I on JAK auto-phosphorylation. Consistent with the increase in STAT phosphorylation, S703I JAK1 was itself hyperphosphorylated (Figure 3
A). In addition, JAK2, TYK2, and JAK3 phosphorylation were upregulated (Figures 3B–3D and S2C), suggesting that the interacting JAK partners may also play a role in the GoF. We reasoned that partnering JAK activity could be overactivated by JAK1 S703I from two mechanisms: increased formation of the receptor complex or direct crosstalk between JAK proteins. We assessed whether JAK1 S703I allowed for more cytokine receptor at the surface, as the structural domains of JAK1 scaffold the receptor complex (Li et al., 2013). However, surface staining for the type I IFN receptor subunit (IFNAR2) demonstrated equivalent receptor expression in WT JAK1 and S703I JAK1 cells, indicating that JAK1 scaffolding of the receptor complex was unaffected by the patient’s mutation (Figures 3E and S2D).Figure 3S703I JAK1 Mediates Hyperactive STAT Phosphorylation by Transactivation of Partnering JAKs(A) Immunoprecipitation of JAK1 from JAK1-transduced U4C cells and immunoblotting for total phosphorylation (4G10) after stimulation with IFN-α (100 IU/mL) for 15 min.(B) Immunoprecipitation of JAK2 from patient-derived B-EBV cells and western blotting for phosphorylation at the activation loop after stimulation with IFN-γ (0.1 ng/mL) for 15 min.(C) Immunoprecipitation of TYK2 from JAK1-transduced U4C cells and western blotting for phosphorylation at the activation loop after stimulation with IFN-α (100 IU/mL) for 15 min.(D) Immunoprecipitation of JAK3 from patient-derived B-EBV cells and western blotting for total phosphorylation stimulation with IL-21 (50 ng/mL) for 15 min.(E) Surface staining of the IFNAR2 subunit by flow cytometry in U4C cells transduced with the indicated constructs. Lines indicate means and error bars represent SD. ∗∗∗p < 0.001, ****p < 0.0001, analysis of variance (ANOVA) with Tukey's post-hoc analysis.(F and G) Transduction of U4C cells with catalytically inactivated JAK1 (K908A), WT JAK1, S703I JAK1, or double-mutant JAK1 (K908A S703I), followed by stimulation with IFN-α (1,000 IU/mL) (F) or IFN-γ (1.0 ng/mL) (G).(H) Transduction of U4C cells with catalytically inactivated JAK1 (K908A), WT JAK1, A634D JAK1, or double-mutant JAK1 (K908A A634D), followed by stimulation with IFN-α (1000 IU/mL) or IFN-γ (1.0 ng/mL).(I) Inhibition of STAT phosphorylation by a 4-h treatment with saturating doses (1 and 10 μM) of JAK inhibitors followed by 15-min stimulation with IFN-α (1,000 IU/mL).All of the data are representative of at least 3 independent experiments.See also Figure S2.Next, we hypothesized that the mutant JAK1 pseudokinase domain transactivated the kinase activity of JAK2 and TYK2. To investigate this mechanism, we mutated the ATP-binding site (K908A) of JAK1 to render it catalytically inactive. This well-characterized mutation retains the signaling capability of the receptor complex, making it possible to study signaling by the partnering JAKs in isolation (Eletto et al., 2016; Li et al., 2013). As expected, STAT phosphorylation was largely reduced in the absence of JAK1 activity (Figures 3F and 3G). However, following inactivation of the kinase domain of S703I JAK1 (S703I/K908A), an aberrant increase in STAT phosphorylation relative to kinase-inactivated JAK1 without the S703I mutation was observed upon cytokine stimulation (Figures 3F and 3G). This result indicates that S703I JAK1 transactivated JAK2 and TYK2, revealing that pseudokinase domains can regulate partnering JAKs in trans, in addition to traditionally understood cis-regulation (Babon et al., 2014). This mechanism was conserved with the other reported JAK1 GoF mutation (A634D, Del Bel e al 2017) (Figure 3H), but not to the JAK2 GoF mutation (V617F) that is common in hematologic malignancy (Figure S2E).Finally, to clinically substantiate the importance of partnering JAK activity in JAK1 S703I (and other JAK-mediated diseases), we compared the potency of selective and non-selective JAK inhibitors, filgotinib and tofacitinib, respectively (Changelian et al., 2003, Rompaey et al., 2013). Unlike a pan-JAK inhibitor, a JAK1-specific inhibitor was unable to completely abrogate JAK1 S703I signaling owing to the continued signaling of partnering JAKs transactivated by the JAK1 pseudokinase (Figures 3I and S2F, schematic). These findings underscore the biochemical and clinical importance of trans-regulation and indicates that JAK selectivity need not mean clinical efficacy. This notion not only challenges the current dogma in drug development but it also proves that precision medicine must be used on a mutation-specific basis.

S703I JAK1 Transactivates Partnering JAKs Independently of Its Own Kinase Activity

To dissect the mechanisms underlying upregulated STAT signaling, we assessed the impact of S703I on JAK auto-phosphorylation. Consistent with the increase in STAT phosphorylation, S703I JAK1 was itself hyperphosphorylated (Figure 3
A). In addition, JAK2, TYK2, and JAK3 phosphorylation were upregulated (Figures 3B–3D and S2C), suggesting that the interacting JAK partners may also play a role in the GoF. We reasoned that partnering JAK activity could be overactivated by JAK1 S703I from two mechanisms: increased formation of the receptor complex or direct crosstalk between JAK proteins. We assessed whether JAK1 S703I allowed for more cytokine receptor at the surface, as the structural domains of JAK1 scaffold the receptor complex (Li et al., 2013). However, surface staining for the type I IFN receptor subunit (IFNAR2) demonstrated equivalent receptor expression in WT JAK1 and S703I JAK1 cells, indicating that JAK1 scaffolding of the receptor complex was unaffected by the patient’s mutation (Figures 3E and S2D).Figure 3S703I JAK1 Mediates Hyperactive STAT Phosphorylation by Transactivation of Partnering JAKs(A) Immunoprecipitation of JAK1 from JAK1-transduced U4C cells and immunoblotting for total phosphorylation (4G10) after stimulation with IFN-α (100 IU/mL) for 15 min.(B) Immunoprecipitation of JAK2 from patient-derived B-EBV cells and western blotting for phosphorylation at the activation loop after stimulation with IFN-γ (0.1 ng/mL) for 15 min.(C) Immunoprecipitation of TYK2 from JAK1-transduced U4C cells and western blotting for phosphorylation at the activation loop after stimulation with IFN-α (100 IU/mL) for 15 min.(D) Immunoprecipitation of JAK3 from patient-derived B-EBV cells and western blotting for total phosphorylation stimulation with IL-21 (50 ng/mL) for 15 min.(E) Surface staining of the IFNAR2 subunit by flow cytometry in U4C cells transduced with the indicated constructs. Lines indicate means and error bars represent SD. ∗∗∗p < 0.001, ****p < 0.0001, analysis of variance (ANOVA) with Tukey's post-hoc analysis.(F and G) Transduction of U4C cells with catalytically inactivated JAK1 (K908A), WT JAK1, S703I JAK1, or double-mutant JAK1 (K908A S703I), followed by stimulation with IFN-α (1,000 IU/mL) (F) or IFN-γ (1.0 ng/mL) (G).(H) Transduction of U4C cells with catalytically inactivated JAK1 (K908A), WT JAK1, A634D JAK1, or double-mutant JAK1 (K908A A634D), followed by stimulation with IFN-α (1000 IU/mL) or IFN-γ (1.0 ng/mL).(I) Inhibition of STAT phosphorylation by a 4-h treatment with saturating doses (1 and 10 μM) of JAK inhibitors followed by 15-min stimulation with IFN-α (1,000 IU/mL).All of the data are representative of at least 3 independent experiments.See also Figure S2.

S703I JAK1 Mediates Hyperactive STAT Phosphorylation by Transactivation of Partnering JAKs

(A) Immunoprecipitation of JAK1 from JAK1-transduced U4C cells and immunoblotting for total phosphorylation (4G10) after stimulation with IFN-α (100 IU/mL) for 15 min.

(B) Immunoprecipitation of JAK2 from patient-derived B-EBV cells and western blotting for phosphorylation at the activation loop after stimulation with IFN-γ (0.1 ng/mL) for 15 min.

(C) Immunoprecipitation of TYK2 from JAK1-transduced U4C cells and western blotting for phosphorylation at the activation loop after stimulation with IFN-α (100 IU/mL) for 15 min.

(D) Immunoprecipitation of JAK3 from patient-derived B-EBV cells and western blotting for total phosphorylation stimulation with IL-21 (50 ng/mL) for 15 min.

(E) Surface staining of the IFNAR2 subunit by flow cytometry in U4C cells transduced with the indicated constructs. Lines indicate means and error bars represent SD. ∗∗∗p < 0.001, ****p < 0.0001, analysis of variance (ANOVA) with Tukey's post-hoc analysis.

(F and G) Transduction of U4C cells with catalytically inactivated JAK1 (K908A), WT JAK1, S703I JAK1, or double-mutant JAK1 (K908A S703I), followed by stimulation with IFN-α (1,000 IU/mL) (F) or IFN-γ (1.0 ng/mL) (G).

(H) Transduction of U4C cells with catalytically inactivated JAK1 (K908A), WT JAK1, A634D JAK1, or double-mutant JAK1 (K908A A634D), followed by stimulation with IFN-α (1000 IU/mL) or IFN-γ (1.0 ng/mL).

(I) Inhibition of STAT phosphorylation by a 4-h treatment with saturating doses (1 and 10 μM) of JAK inhibitors followed by 15-min stimulation with IFN-α (1,000 IU/mL).

All of the data are representative of at least 3 independent experiments.

See also Figure S2.

Next, we hypothesized that the mutant JAK1 pseudokinase domain transactivated the kinase activity of JAK2 and TYK2. To investigate this mechanism, we mutated the ATP-binding site (K908A) of JAK1 to render it catalytically inactive. This well-characterized mutation retains the signaling capability of the receptor complex, making it possible to study signaling by the partnering JAKs in isolation (Eletto et al., 2016; Li et al., 2013). As expected, STAT phosphorylation was largely reduced in the absence of JAK1 activity (Figures 3F and 3G). However, following inactivation of the kinase domain of S703I JAK1 (S703I/K908A), an aberrant increase in STAT phosphorylation relative to kinase-inactivated JAK1 without the S703I mutation was observed upon cytokine stimulation (Figures 3F and 3G). This result indicates that S703I JAK1 transactivated JAK2 and TYK2, revealing that pseudokinase domains can regulate partnering JAKs in trans, in addition to traditionally understood cis-regulation (Babon et al., 2014). This mechanism was conserved with the other reported JAK1 GoF mutation (A634D, Del Bel e al 2017) (Figure 3H), but not to the JAK2 GoF mutation (V617F) that is common in hematologic malignancy (Figure S2E).

Finally, to clinically substantiate the importance of partnering JAK activity in JAK1 S703I (and other JAK-mediated diseases), we compared the potency of selective and non-selective JAK inhibitors, filgotinib and tofacitinib, respectively (Changelian et al., 2003, Rompaey et al., 2013). Unlike a pan-JAK inhibitor, a JAK1-specific inhibitor was unable to completely abrogate JAK1 S703I signaling owing to the continued signaling of partnering JAKs transactivated by the JAK1 pseudokinase (Figures 3I and S2F, schematic). These findings underscore the biochemical and clinical importance of trans-regulation and indicates that JAK selectivity need not mean clinical efficacy. This notion not only challenges the current dogma in drug development but it also proves that precision medicine must be used on a mutation-specific basis.

Ex Vivo Analysis Reveals a Cell-Intrinsic GoFTo more robustly investigate the consequences of S703I on immune cells, we performed mass cytometry (CyTOF) immunophenotyping on whole blood from the patient. Despite the central role of JAK1 in immune cell differentiation and proliferation, the patient’s immune cell distribution was largely within the normal range, barring a few exceptions (Figures 4A and 4B; Table S2). As detected clinically, eosinophils fluctuated to high counts (Figure S1C). B cells, however, trended toward lower frequencies than in healthy donors, but largely retained a normal distribution of naive, memory, and class-switched cells (Figures S3A–S3C). Natural killer (NK) cells exhibited an increase in CD56hi cells (>12-fold over healthy controls) (Figures 4B, S3E, and S3F), a subset understood to be less mature but highly proliferative and rapid in cytokine production as compared to the majority CD56lo subset with high cytotoxicity (Figures S3G and S3H) (Poli et al., 2009). Extensive phenotyping of the patient’s NK cells was performed concurrently with STAT1 GoF NK cells, which also demonstrated functional immaturity (Vargas-Hernandez et al., 2017). This analysis revealed that JAK1 GoF NK cells aligned phenotypically with prototypical immature “CD56bright” NK cells, unlike those from STAT1 GoF patients (Figure S3I). However, whether this phenotype results from the primary pathology or clinical intervention remains to be determined in this case.Figure 4CyTOF Analysis Reveals Cytokine-, STAT-, and Cell-Type-Specific Gain of Function(A) Representative (tSNE) plots generated from immunophenotyping CyTOF data of whole blood from 2 independent experiments.(B) Manually gated CyTOF populations from the whole blood of 5 separate healthy controls and the patient on 2 separate occasions (2X) were quantified as the percentage of single cells and expressed as relative frequency (patient/controls). The gray bars indicate means with standard deviations of healthy donors, and colored bars indicate means with standard deviations of patient. Multiple t tests performed correcting for multiple comparisons using the Holm-Sidak method. ∗∗∗p < 0.001.(C) Relative MSI of phospho-STAT staining from intracellular phospho-CyTOF of whole blood from 4 healthy donors (n = 4) and the patient on 3 separate occasions (3X). The columns represent means and the error bars represent standard deviations.(D) Ex vivo stimulation with IFN-α (100 IU/mL), IL-2 (50 ng/mL), and IL-4 (50 ng/mL) for 15 min. The color intensity indicates the log2 fold-change in MSI over the unstimulated healthy control for each cell type (n = 1).Cent Mem, central memory; C Mono, classical monocyte; DN, double negative; DP, double positive; Eff Mem, effector memory; FC, fold change; HC, healthy control; ILC, innate lymphoid cells; I Mono, intermediate monocytes; MAIT, mucosal associate invariant T cells; mDC, myeloid dendritic cell; MSI, mean signal intensity; NC Mono, non-classical monocyte; P, patient; pDC, plasmacytoid dendritic cell; Th, T helper; . tSNE, t-distributed stochastic neighbor embedding.See also Figures S3 and S4.We then performed phospho-CyTOF to analyze the phosphorylation of all STATs downstream of JAK1 in all of the major immune cells of whole blood. Given the ubiquitous expression and diverse signaling capabilities of JAK1, as well as the basal activity of S703I observed in vitro, we hypothesized that all STATs within all immune cells would be hyperphosphorylated at baseline. Heightened phosphorylation of STAT1, STAT3, STAT4, STAT5, and STAT6 was observed, but not universally. Certain immune subsets, but not others, exhibited high basal phosphorylation of specific STATs (Figure 4C). For example, granulocytes from the patient displayed baseline STAT1 phosphorylation, but not STAT3 phosphorylation, whereas T cells displayed basal STAT3 phosphorylation but not STAT1 phosphorylation. STAT6 phosphorylation, however, was not upregulated in any immune subset, except B cells.To evaluate the functional consequences of the observed basal phosphorylation, we assessed the expression of downstream genes from bulk PBMCs. We detected the elevated expression of downstream pSTAT target genes, including IFI27, IFIT1, MX1, and SIGLEC1 (Figure S4A). In addition, we tested non-hematopoietic tissues for baseline STAT phosphorylation by immunohistochemistry. In skin and gastrointestinal biopsies, but not renal biopsy, we detected highly phosphorylated STAT1 and STAT3 as compared to healthy samples (Figure S4B). This basal phosphorylation was observed in the apparent absence of any overt increase in circulating JAK-STAT cytokine concentrations (Figure S4C), suggesting that intrinsic S703I JAK1 activity drove this process, consistent with our in vitro results (Figures 2B–2F).

Ex Vivo Analysis Reveals a Cell-Intrinsic GoF

To more robustly investigate the consequences of S703I on immune cells, we performed mass cytometry (CyTOF) immunophenotyping on whole blood from the patient. Despite the central role of JAK1 in immune cell differentiation and proliferation, the patient’s immune cell distribution was largely within the normal range, barring a few exceptions (Figures 4A and 4B; Table S2). As detected clinically, eosinophils fluctuated to high counts (Figure S1C). B cells, however, trended toward lower frequencies than in healthy donors, but largely retained a normal distribution of naive, memory, and class-switched cells (Figures S3A–S3C). Natural killer (NK) cells exhibited an increase in CD56hi cells (>12-fold over healthy controls) (Figures 4B, S3E, and S3F), a subset understood to be less mature but highly proliferative and rapid in cytokine production as compared to the majority CD56lo subset with high cytotoxicity (Figures S3G and S3H) (Poli et al., 2009). Extensive phenotyping of the patient’s NK cells was performed concurrently with STAT1 GoF NK cells, which also demonstrated functional immaturity (Vargas-Hernandez et al., 2017). This analysis revealed that JAK1 GoF NK cells aligned phenotypically with prototypical immature “CD56bright” NK cells, unlike those from STAT1 GoF patients (Figure S3I). However, whether this phenotype results from the primary pathology or clinical intervention remains to be determined in this case.Figure 4CyTOF Analysis Reveals Cytokine-, STAT-, and Cell-Type-Specific Gain of Function(A) Representative (tSNE) plots generated from immunophenotyping CyTOF data of whole blood from 2 independent experiments.(B) Manually gated CyTOF populations from the whole blood of 5 separate healthy controls and the patient on 2 separate occasions (2X) were quantified as the percentage of single cells and expressed as relative frequency (patient/controls). The gray bars indicate means with standard deviations of healthy donors, and colored bars indicate means with standard deviations of patient. Multiple t tests performed correcting for multiple comparisons using the Holm-Sidak method. ∗∗∗p < 0.001.(C) Relative MSI of phospho-STAT staining from intracellular phospho-CyTOF of whole blood from 4 healthy donors (n = 4) and the patient on 3 separate occasions (3X). The columns represent means and the error bars represent standard deviations.(D) Ex vivo stimulation with IFN-α (100 IU/mL), IL-2 (50 ng/mL), and IL-4 (50 ng/mL) for 15 min. The color intensity indicates the log2 fold-change in MSI over the unstimulated healthy control for each cell type (n = 1).Cent Mem, central memory; C Mono, classical monocyte; DN, double negative; DP, double positive; Eff Mem, effector memory; FC, fold change; HC, healthy control; ILC, innate lymphoid cells; I Mono, intermediate monocytes; MAIT, mucosal associate invariant T cells; mDC, myeloid dendritic cell; MSI, mean signal intensity; NC Mono, non-classical monocyte; P, patient; pDC, plasmacytoid dendritic cell; Th, T helper; . tSNE, t-distributed stochastic neighbor embedding.See also Figures S3 and S4.

CyTOF Analysis Reveals Cytokine-, STAT-, and Cell-Type-Specific Gain of Function

(A) Representative (tSNE) plots generated from immunophenotyping CyTOF data of whole blood from 2 independent experiments.

(B) Manually gated CyTOF populations from the whole blood of 5 separate healthy controls and the patient on 2 separate occasions (2X) were quantified as the percentage of single cells and expressed as relative frequency (patient/controls). The gray bars indicate means with standard deviations of healthy donors, and colored bars indicate means with standard deviations of patient. Multiple t tests performed correcting for multiple comparisons using the Holm-Sidak method. ∗∗∗p < 0.001.

(C) Relative MSI of phospho-STAT staining from intracellular phospho-CyTOF of whole blood from 4 healthy donors (n = 4) and the patient on 3 separate occasions (3X). The columns represent means and the error bars represent standard deviations.

(D) Ex vivo stimulation with IFN-α (100 IU/mL), IL-2 (50 ng/mL), and IL-4 (50 ng/mL) for 15 min. The color intensity indicates the log2 fold-change in MSI over the unstimulated healthy control for each cell type (n = 1).

Cent Mem, central memory; C Mono, classical monocyte; DN, double negative; DP, double positive; Eff Mem, effector memory; FC, fold change; HC, healthy control; ILC, innate lymphoid cells; I Mono, intermediate monocytes; MAIT, mucosal associate invariant T cells; mDC, myeloid dendritic cell; MSI, mean signal intensity; NC Mono, non-classical monocyte; P, patient; pDC, plasmacytoid dendritic cell; Th, T helper; . tSNE, t-distributed stochastic neighbor embedding.

See also Figures S3 and S4.

We then performed phospho-CyTOF to analyze the phosphorylation of all STATs downstream of JAK1 in all of the major immune cells of whole blood. Given the ubiquitous expression and diverse signaling capabilities of JAK1, as well as the basal activity of S703I observed in vitro, we hypothesized that all STATs within all immune cells would be hyperphosphorylated at baseline. Heightened phosphorylation of STAT1, STAT3, STAT4, STAT5, and STAT6 was observed, but not universally. Certain immune subsets, but not others, exhibited high basal phosphorylation of specific STATs (Figure 4C). For example, granulocytes from the patient displayed baseline STAT1 phosphorylation, but not STAT3 phosphorylation, whereas T cells displayed basal STAT3 phosphorylation but not STAT1 phosphorylation. STAT6 phosphorylation, however, was not upregulated in any immune subset, except B cells.

To evaluate the functional consequences of the observed basal phosphorylation, we assessed the expression of downstream genes from bulk PBMCs. We detected the elevated expression of downstream pSTAT target genes, including IFI27, IFIT1, MX1, and SIGLEC1 (Figure S4A). In addition, we tested non-hematopoietic tissues for baseline STAT phosphorylation by immunohistochemistry. In skin and gastrointestinal biopsies, but not renal biopsy, we detected highly phosphorylated STAT1 and STAT3 as compared to healthy samples (Figure S4B). This basal phosphorylation was observed in the apparent absence of any overt increase in circulating JAK-STAT cytokine concentrations (Figure S4C), suggesting that intrinsic S703I JAK1 activity drove this process, consistent with our in vitro results (Figures 2B–2F).

Ex Vivo Cytokine Stimulation Generates Non-canonical Signaling Pathways in Patient CellsWhole blood was then stimulated ex vivo with a series of cytokines that engage JAK1 with various cytokine receptors, JAK partners, and downstream STAT targets. In response to IFN-α or IL-2, patient leukocytes hyperphosphorylated STAT1 and STAT3 or STAT5, respectively (Figures 4D and S4D and S4E, represented plots). By contrast, STAT6 phosphorylation in response to IL-4 was similar to that in healthy control cells, as seen in the baseline STAT6 data (Figure 4D). Likewise, IL-5 stimulation led to normal STAT5 phosphorylation (Figure S5F). These differential responses were also noted in patient B-EBV cells (Figures S5G–S5I). However, stimulation with either IL-2 or IL-4 induced the phosphorylation of STAT1 exclusively in patient cells, contrasting the canonical signaling cascade induced by these cytokines (Figure 4D). This non-canonical response suggests that S703I confers promiscuity onto JAK1, allowing it to transverse traditional signaling axes and establish non-canonical pathways. These results indicate that S703I JAK1 is a GoF mutation ex vivo, displaying both unexpected pathway promiscuity and specificity in the activation of STAT signaling.

Ex Vivo Cytokine Stimulation Generates Non-canonical Signaling Pathways in Patient Cells

Whole blood was then stimulated ex vivo with a series of cytokines that engage JAK1 with various cytokine receptors, JAK partners, and downstream STAT targets. In response to IFN-α or IL-2, patient leukocytes hyperphosphorylated STAT1 and STAT3 or STAT5, respectively (Figures 4D and S4D and S4E, represented plots). By contrast, STAT6 phosphorylation in response to IL-4 was similar to that in healthy control cells, as seen in the baseline STAT6 data (Figure 4D). Likewise, IL-5 stimulation led to normal STAT5 phosphorylation (Figure S5F). These differential responses were also noted in patient B-EBV cells (Figures S5G–S5I). However, stimulation with either IL-2 or IL-4 induced the phosphorylation of STAT1 exclusively in patient cells, contrasting the canonical signaling cascade induced by these cytokines (Figure 4D). This non-canonical response suggests that S703I confers promiscuity onto JAK1, allowing it to transverse traditional signaling axes and establish non-canonical pathways. These results indicate that S703I JAK1 is a GoF mutation ex vivo, displaying both unexpected pathway promiscuity and specificity in the activation of STAT signaling.

Custom scRNA-Seq Maps Cellular Distribution and Transcriptomic Signatures of S703I JAK1To follow up on the differences across immune cell types, we performed single-cell RNA sequencing (scRNA-seq) of the patient’s PBMCs. We aimed to specifically map and evaluate the impact of S703I JAK1 across immune cell subsets by relating single-cell resolution gene expression patterns to cell type and, given the mosaicism, JAK1 genotype. However, sequence data from droplet-based scRNA-seq platforms are typically restricted to 5′ or 3′ transcript regions and therefore do not include coverage of the S703I site (c.2108) in JAK1 mRNA (Figure S5A). Therefore, using custom barcode microbeads and a modified library preparation procedure, we adapted the inDrops scRNA-seq methodology (Klein et al., 2015; Zilionis et al., 2017) to target exon 16 (containing the S703I site) of JAK1 in addition to the standard mRNA 3′ regions (Figure S5B; Method Details). We used this custom inDrops scRNA-seq to analyze the patient’s PBMCs, and concurrently processed the same sample on the 10X Genomics Chromium platform (Figures S5C and S5D). Clustering and manual annotation of cell types based on gene expression patterns distinguished the expected PBMC populations, with relative frequencies consistent with the corresponding CyTOF analyses (Figures 5
A and S5D; Table S3). Again, we observed a relative increase in CD56hi NK cell frequency. Using sequence data from JAK1-targeted libraries, we assigned putative per-cell JAK1 genotypes (based on transcript sequences) to a subset of PBMCs for which transcript and read depths were sufficient in the JAK1-targeted inDrops dataset (Figures S5E and S5F). Within this subset, we found that the mutant allele was not evenly distributed across immune cell-type clusters (Figures 5B and 5C). For example, B cells and monocytes mostly carried WT transcrip, whereas 69% of CD56hi NK cells contained JAK1 mRNA with the S703I mutation. This variable distribution may reflect differences in the intrinsic tolerance to the mutation in these cell types, which is consistent with the expansion of the CD56hi NK cell subset and the higher expression of JAK1 in NK cells than in other cell types (Figure S5G).Figure 5Custom scRNA-Seq Maps JAK1 Allele Distribution, Transcriptomic Impact, and Expression Patterns(A) tSNE plots and cell-type assignments from scRNA-seq of patient PBMCs, with an inDrops platform adapted to target the mutant JAK1 transcript. n = 4,763 cells.(B) tSNE plots representing the subset of cells with sufficient JAK1 counts to be assigned putative JAK1 genotypes (based on transcript sequences). Cells in which any mutant transcript was detected above empirically determined thresholds (> 5 JAK1 transcripts) were assigned “S703I JAK1” (purple), while cells with only WT transcript detected were assigned “WT JAK1” (orange).(C) Doughnut charts quantifying allele distribution in cells meeting genotyping criteria (cell count in the center), as in (B).(D) Expression of the ISG IFI44L, a statistically significant differentially expressed gene in the comparison of WT JAK1 and S703I JAK1 genotyped cells.(E) Gene set scores for IFN-α signaling in CD14+ monocytes.(F) Number of unique transcripts detected per cell for the WT or S703I JAK1 allele (left) or a control variant GNLY rs12845 (right). The bubble size indicates the number of cells. The color coding indicates cells containing S703I JAK1 (purple), WT JAK1 without S703I JAK1 (orange), WT JAK1 with S703I JAK1 detected below threshold (yellow), or insufficient transcripts counts (gray).(G) Transcript genotyping of JAK1 rs2230587 from healthy control PBMCs (n = 96) by single-cell qPCR with allele-specific probes. The histogram represents the relative frequency of cells expressing binned allele ratios as quantified by oligonucleotide standards.(I) Single-cell qPCR transcript genotyping of control gene NACA (rs4902).See also Figure S5.We next performed differential gene expression analysis, comparing patient cells expressing the WT and mutant alleles of JAK1. Although this analysis was constrained by a limited number of cells for which transcript-level genotype information was available, the intrasample comparison can be conducted on an inherently isogenic background with identical exposure history. We detected statistically significant differences in ISG expression for IFI44L and for an ISG set between WT and mutant monocytes (Figures 5D and 5E). These data suggest that tolerance to S703I may differ between cell types and confirm the cell-intrinsic nature of the GoF described above.

Custom scRNA-Seq Maps Cellular Distribution and Transcriptomic Signatures of S703I JAK1

To follow up on the differences across immune cell types, we performed single-cell RNA sequencing (scRNA-seq) of the patient’s PBMCs. We aimed to specifically map and evaluate the impact of S703I JAK1 across immune cell subsets by relating single-cell resolution gene expression patterns to cell type and, given the mosaicism, JAK1 genotype. However, sequence data from droplet-based scRNA-seq platforms are typically restricted to 5′ or 3′ transcript regions and therefore do not include coverage of the S703I site (c.2108) in JAK1 mRNA (Figure S5A). Therefore, using custom barcode microbeads and a modified library preparation procedure, we adapted the inDrops scRNA-seq methodology (Klein et al., 2015; Zilionis et al., 2017) to target exon 16 (containing the S703I site) of JAK1 in addition to the standard mRNA 3′ regions (Figure S5B; Method Details). We used this custom inDrops scRNA-seq to analyze the patient’s PBMCs, and concurrently processed the same sample on the 10X Genomics Chromium platform (Figures S5C and S5D). Clustering and manual annotation of cell types based on gene expression patterns distinguished the expected PBMC populations, with relative frequencies consistent with the corresponding CyTOF analyses (Figures 5
A and S5D; Table S3). Again, we observed a relative increase in CD56hi NK cell frequency. Using sequence data from JAK1-targeted libraries, we assigned putative per-cell JAK1 genotypes (based on transcript sequences) to a subset of PBMCs for which transcript and read depths were sufficient in the JAK1-targeted inDrops dataset (Figures S5E and S5F). Within this subset, we found that the mutant allele was not evenly distributed across immune cell-type clusters (Figures 5B and 5C). For example, B cells and monocytes mostly carried WT transcrip, whereas 69% of CD56hi NK cells contained JAK1 mRNA with the S703I mutation. This variable distribution may reflect differences in the intrinsic tolerance to the mutation in these cell types, which is consistent with the expansion of the CD56hi NK cell subset and the higher expression of JAK1 in NK cells than in other cell types (Figure S5G).Figure 5Custom scRNA-Seq Maps JAK1 Allele Distribution, Transcriptomic Impact, and Expression Patterns(A) tSNE plots and cell-type assignments from scRNA-seq of patient PBMCs, with an inDrops platform adapted to target the mutant JAK1 transcript. n = 4,763 cells.(B) tSNE plots representing the subset of cells with sufficient JAK1 counts to be assigned putative JAK1 genotypes (based on transcript sequences). Cells in which any mutant transcript was detected above empirically determined thresholds (> 5 JAK1 transcripts) were assigned “S703I JAK1” (purple), while cells with only WT transcript detected were assigned “WT JAK1” (orange).(C) Doughnut charts quantifying allele distribution in cells meeting genotyping criteria (cell count in the center), as in (B).(D) Expression of the ISG IFI44L, a statistically significant differentially expressed gene in the comparison of WT JAK1 and S703I JAK1 genotyped cells.(E) Gene set scores for IFN-α signaling in CD14+ monocytes.(F) Number of unique transcripts detected per cell for the WT or S703I JAK1 allele (left) or a control variant GNLY rs12845 (right). The bubble size indicates the number of cells. The color coding indicates cells containing S703I JAK1 (purple), WT JAK1 without S703I JAK1 (orange), WT JAK1 with S703I JAK1 detected below threshold (yellow), or insufficient transcripts counts (gray).(G) Transcript genotyping of JAK1 rs2230587 from healthy control PBMCs (n = 96) by single-cell qPCR with allele-specific probes. The histogram represents the relative frequency of cells expressing binned allele ratios as quantified by oligonucleotide standards.(I) Single-cell qPCR transcript genotyping of control gene NACA (rs4902).See also Figure S5.

Custom scRNA-Seq Maps JAK1 Allele Distribution, Transcriptomic Impact, and Expression Patterns

(A) tSNE plots and cell-type assignments from scRNA-seq of patient PBMCs, with an inDrops platform adapted to target the mutant JAK1 transcript. n = 4,763 cells.

(B) tSNE plots representing the subset of cells with sufficient JAK1 counts to be assigned putative JAK1 genotypes (based on transcript sequences). Cells in which any mutant transcript was detected above empirically determined thresholds (> 5 JAK1 transcripts) were assigned “S703I JAK1” (purple), while cells with only WT transcript detected were assigned “WT JAK1” (orange).

(C) Doughnut charts quantifying allele distribution in cells meeting genotyping criteria (cell count in the center), as in (B).

(D) Expression of the ISG IFI44L, a statistically significant differentially expressed gene in the comparison of WT JAK1 and S703I JAK1 genotyped cells.

(E) Gene set scores for IFN-α signaling in CD14+ monocytes.

(F) Number of unique transcripts detected per cell for the WT or S703I JAK1 allele (left) or a control variant GNLY rs12845 (right). The bubble size indicates the number of cells. The color coding indicates cells containing S703I JAK1 (purple), WT JAK1 without S703I JAK1 (orange), WT JAK1 with S703I JAK1 detected below threshold (yellow), or insufficient transcripts counts (gray).

(G) Transcript genotyping of JAK1 rs2230587 from healthy control PBMCs (n = 96) by single-cell qPCR with allele-specific probes. The histogram represents the relative frequency of cells expressing binned allele ratios as quantified by oligonucleotide standards.

(I) Single-cell qPCR transcript genotyping of control gene NACA (rs4902).

See also Figure S5.

We next performed differential gene expression analysis, comparing patient cells expressing the WT and mutant alleles of JAK1. Although this analysis was constrained by a limited number of cells for which transcript-level genotype information was available, the intrasample comparison can be conducted on an inherently isogenic background with identical exposure history. We detected statistically significant differences in ISG expression for IFI44L and for an ISG set between WT and mutant monocytes (Figures 5D and 5E). These data suggest that tolerance to S703I may differ between cell types and confirm the cell-intrinsic nature of the GoF described above.

JAK1 Expression Is Restricted to a Single Allele across Immune CellsGiven the mosaicism of a heterozygous mutation, we expected to observe some cells (i.e., homozygous WT JAK1) containing only WT JAK1 transcripts and others (i.e., heterozygous S703I JAK1) containing both WT and S703I JAK1 transcripts. However, we found that expression of the 2 alleles seemed almost mutually exclusive, as very few cells expressed both transcripts, as opposed to the ~50% that was expected given our genomic estimates (Figures 1D and 5F, left panel). By contrast, other genes from our dataset containing heterozygous variants exhibited the expected biallelic distribution (Figure 5F, right panel). This result suggests that JAK1 may be subject to monoallelic bias, a pattern that has only recently been recognized in a fraction of the autosomal transcriptome (Borel et al., 2015; Deng et al., 2014; Gimelbrant et al., 2007; Jeffries et al., 2012). We further tested this hypothesis of biased allele expression by sorting single cells from healthy donor PBMCs heterozygous for a synonymous SNP (rs2230587) of JAK1, adjacent to S703I. qPCR of isolated RNA with allele-specific probes revealed that relative expression of the two alleles was not normally distributed, but rather biased to one allele or the other (Figure 5G), unlike in a control gene (Figure 5H). A query of publicly available murine expression data revealed a similar allele restriction for JAK1, which, at least in mice, remains fixed over time (Savova et al., 2016). Whether the observed phenomenon in this patient represents transcriptional bursting or mitotically stable monoallelic expression and the potential impact on immune dysfunction remains to be fully determined. In either sense, these data perhaps indicate a departure from the classic genetic interpretation of heterozygosity and allow for a shift in understanding of the genetic penetrance of disease.

JAK1 Expression Is Restricted to a Single Allele across Immune Cells

Given the mosaicism of a heterozygous mutation, we expected to observe some cells (i.e., homozygous WT JAK1) containing only WT JAK1 transcripts and others (i.e., heterozygous S703I JAK1) containing both WT and S703I JAK1 transcripts. However, we found that expression of the 2 alleles seemed almost mutually exclusive, as very few cells expressed both transcripts, as opposed to the ~50% that was expected given our genomic estimates (Figures 1D and 5F, left panel). By contrast, other genes from our dataset containing heterozygous variants exhibited the expected biallelic distribution (Figure 5F, right panel). This result suggests that JAK1 may be subject to monoallelic bias, a pattern that has only recently been recognized in a fraction of the autosomal transcriptome (Borel et al., 2015; Deng et al., 2014; Gimelbrant et al., 2007; Jeffries et al., 2012). We further tested this hypothesis of biased allele expression by sorting single cells from healthy donor PBMCs heterozygous for a synonymous SNP (rs2230587) of JAK1, adjacent to S703I. qPCR of isolated RNA with allele-specific probes revealed that relative expression of the two alleles was not normally distributed, but rather biased to one allele or the other (Figure 5G), unlike in a control gene (Figure 5H). A query of publicly available murine expression data revealed a similar allele restriction for JAK1, which, at least in mice, remains fixed over time (Savova et al., 2016). Whether the observed phenomenon in this patient represents transcriptional bursting or mitotically stable monoallelic expression and the potential impact on immune dysfunction remains to be fully determined. In either sense, these data perhaps indicate a departure from the classic genetic interpretation of heterozygosity and allow for a shift in understanding of the genetic penetrance of disease.

Clinical and Biological Immune Dysfunction Can Be Rescued with TofacitinibHaving identified JAK1 hyperactivity as the putative driver of clinical disease in the patient, we then considered the use of JAK inhibitors for the clinical treatment of this patient. We compared the ability of the two US Food and Drug Administration (FDA)-approved pan-JAK inhibitors available at the time to reduce basal STAT phosphorylation in S703I-transduced U4C cells. Despite its lower relative potency against JAK1, tofacitinib inhibited STAT phosphorylation in a comparable dose response to ruxolitinib (Figure 6
A), again reflecting the independence of JAK1 catalysis and STAT hyperphosphorylation. Similar results were obtained with patient-derived B-EBV cells, which, unlike the transduced fibrosarcoma U4C cells, are hematopoietic in origin and therefore express JAK3 (Figure 6B). Next, we treated patient blood ex vivo with the two compounds at equimolar doses that mimic physiological dosing (Chen et al., 2014; Krishnaswami et al., 2014; Lamba et al., 2016), and we further assessed the inhibition of IFN-α stimulation. Analysis of phospho-STAT inhibition across whole-blood immune subsets by phospho-CyTOF revealed that tofacitinib attenuated the response more potently than ruxolitinib in nearly all cell types (Figure 6C).Figure 6Treatment with Tofacitinib Rescues STAT Hyperphosphorylation and Resolves Clinical Disease(A) In vitro assessment of JAK-inhibitor efficacy after 4-h drug treatment for resolution of basal STAT phosphorylation in transduced U4Cs.(B) Similar analysis in patient-derived B-EBVs, with doses indicated.(C) Ex vivo inhibition with equimolar doses (500 nM) of ruxolitinib and tofacitinib for 4 h followed by IFN-α stimulation (1,000 IU/mL) for 15 min (n = 1).(D) C-Reactive Protein monitoring from patient sera over the course of treatment. Dotted line represents the upper limit of normal (ULN).(E) Erythrocyte sedimentation rate in patient blood over the course of treatment, with a dotted line indicating the ULN.(F) Gross appearance of skin before and 5 months after tofacitinib treatment (Tx). (Image of the arm also in Figure 1.)(G) Colonic biopsies before and 6 months post-therapy.(H) ISG expression in RNA from bulk PBMCs isolated throughout treatment (n = 1). The dotted line represents the average expression across 3 healthy controls.(I) Phospho-CyTOF analysis comparing relative changes in pSTAT MSI from before and 10 months post-therapy (n = 1).See also Figure S6.Following these extensive functional studies, we treated the patient with low-dose tofacitinib (5 mg daily). Within 8 weeks, circulating inflammatory markers (erythrocyte sedimentation rate [ESR] and C-reactive protein) normalized (Figures 6D and 6E). Near-complete improvement in dermatitis followed, both grossly and histologically (Figure 6F). By 6 months, the patient reported complete resolution of gastrointestinal symptoms (decrease in modified Pediatric Ulcerative Colitis Activity Index [PUCAI] from 35–50 initially to 0). Biopsy of colonic tissue revealed the restoration of crypt architecture and the complete resolution of eosinophilic infiltrates (Figure 6G). The patient remained stable for 2 years after initiation of therapy until, unfortunately, the patient succumbed to acute respiratory failure due to coronavirus disease 2019 (COVID-19).Finally, we confirmed the pharmacological rescue of JAK hyperactivity in the patient’s cells after tofacitinib treatment. RNA isolated from PBMCs revealed that the expression of ISGs, which was elevated before treatment, progressively declined to normal amounts (Figure 6H). CyTOF analysis was then performed to confirm the decrease in basal STAT phosphorylation (Figure S6). Reductions were observed across cell types in STAT3, STAT4, STAT5, and STAT6 phosphorylation, whereas STAT1 phosphorylation was reduced in some, but not all cell types (Figure 6I). Overall, these results validate S703I JAK1 as the etiology in vivo of the widespread immune dysregulation (summarized in Table S4), and they illustrate the power of precision medicine both as a treatment approach for patients with rare diseases and as a means of discovering fundamental physiological mechanisms.

Clinical and Biological Immune Dysfunction Can Be Rescued with Tofacitinib

Having identified JAK1 hyperactivity as the putative driver of clinical disease in the patient, we then considered the use of JAK inhibitors for the clinical treatment of this patient. We compared the ability of the two US Food and Drug Administration (FDA)-approved pan-JAK inhibitors available at the time to reduce basal STAT phosphorylation in S703I-transduced U4C cells. Despite its lower relative potency against JAK1, tofacitinib inhibited STAT phosphorylation in a comparable dose response to ruxolitinib (Figure 6
A), again reflecting the independence of JAK1 catalysis and STAT hyperphosphorylation. Similar results were obtained with patient-derived B-EBV cells, which, unlike the transduced fibrosarcoma U4C cells, are hematopoietic in origin and therefore express JAK3 (Figure 6B). Next, we treated patient blood ex vivo with the two compounds at equimolar doses that mimic physiological dosing (Chen et al., 2014; Krishnaswami et al., 2014; Lamba et al., 2016), and we further assessed the inhibition of IFN-α stimulation. Analysis of phospho-STAT inhibition across whole-blood immune subsets by phospho-CyTOF revealed that tofacitinib attenuated the response more potently than ruxolitinib in nearly all cell types (Figure 6C).Figure 6Treatment with Tofacitinib Rescues STAT Hyperphosphorylation and Resolves Clinical Disease(A) In vitro assessment of JAK-inhibitor efficacy after 4-h drug treatment for resolution of basal STAT phosphorylation in transduced U4Cs.(B) Similar analysis in patient-derived B-EBVs, with doses indicated.(C) Ex vivo inhibition with equimolar doses (500 nM) of ruxolitinib and tofacitinib for 4 h followed by IFN-α stimulation (1,000 IU/mL) for 15 min (n = 1).(D) C-Reactive Protein monitoring from patient sera over the course of treatment. Dotted line represents the upper limit of normal (ULN).(E) Erythrocyte sedimentation rate in patient blood over the course of treatment, with a dotted line indicating the ULN.(F) Gross appearance of skin before and 5 months after tofacitinib treatment (Tx). (Image of the arm also in Figure 1.)(G) Colonic biopsies before and 6 months post-therapy.(H) ISG expression in RNA from bulk PBMCs isolated throughout treatment (n = 1). The dotted line represents the average expression across 3 healthy controls.(I) Phospho-CyTOF analysis comparing relative changes in pSTAT MSI from before and 10 months post-therapy (n = 1).See also Figure S6.

Treatment with Tofacitinib Rescues STAT Hyperphosphorylation and Resolves Clinical Disease

(A) In vitro assessment of JAK-inhibitor efficacy after 4-h drug treatment for resolution of basal STAT phosphorylation in transduced U4Cs.

(B) Similar analysis in patient-derived B-EBVs, with doses indicated.

(C) Ex vivo inhibition with equimolar doses (500 nM) of ruxolitinib and tofacitinib for 4 h followed by IFN-α stimulation (1,000 IU/mL) for 15 min (n = 1).

(D) C-Reactive Protein monitoring from patient sera over the course of treatment. Dotted line represents the upper limit of normal (ULN).

(E) Erythrocyte sedimentation rate in patient blood over the course of treatment, with a dotted line indicating the ULN.

(F) Gross appearance of skin before and 5 months after tofacitinib treatment (Tx). (Image of the arm also in Figure 1.)

(G) Colonic biopsies before and 6 months post-therapy.

(H) ISG expression in RNA from bulk PBMCs isolated throughout treatment (n = 1). The dotted line represents the average expression across 3 healthy controls.

(I) Phospho-CyTOF analysis comparing relative changes in pSTAT MSI from before and 10 months post-therapy (n = 1).

See also Figure S6.

Following these extensive functional studies, we treated the patient with low-dose tofacitinib (5 mg daily). Within 8 weeks, circulating inflammatory markers (erythrocyte sedimentation rate [ESR] and C-reactive protein) normalized (Figures 6D and 6E). Near-complete improvement in dermatitis followed, both grossly and histologically (Figure 6F). By 6 months, the patient reported complete resolution of gastrointestinal symptoms (decrease in modified Pediatric Ulcerative Colitis Activity Index [PUCAI] from 35–50 initially to 0). Biopsy of colonic tissue revealed the restoration of crypt architecture and the complete resolution of eosinophilic infiltrates (Figure 6G). The patient remained stable for 2 years after initiation of therapy until, unfortunately, the patient succumbed to acute respiratory failure due to coronavirus disease 2019 (COVID-19).

Finally, we confirmed the pharmacological rescue of JAK hyperactivity in the patient’s cells after tofacitinib treatment. RNA isolated from PBMCs revealed that the expression of ISGs, which was elevated before treatment, progressively declined to normal amounts (Figure 6H). CyTOF analysis was then performed to confirm the decrease in basal STAT phosphorylation (Figure S6). Reductions were observed across cell types in STAT3, STAT4, STAT5, and STAT6 phosphorylation, whereas STAT1 phosphorylation was reduced in some, but not all cell types (Figure 6I). Overall, these results validate S703I JAK1 as the etiology in vivo of the widespread immune dysregulation (summarized in Table S4), and they illustrate the power of precision medicine both as a treatment approach for patients with rare diseases and as a means of discovering fundamental physiological mechanisms.

DiscussionUndiagnosed disease programs have shown promise for the detection of potential causative variants of disease (Lee et al., 2014; Splinter et al., 2018; Yang et al., 2014). This report demonstrates the value of the in-depth study of select patients identified in these programs. Most immediately, the findings described herein directed the successful treatment of a complex immunodysregulatory disease in a personalized molecular fashion. More broadly, this case implicates JAK1 dysfunction in common forms of multifactorial diseases, including dermatitis, enteritis, colitis, and eosinophilic disorders. These features align with the other reported JAK1 GoF mutation recently published (Del Bel et al., 2017), and together provide strong justification for the expanding use of JAK inhibitors in these disorders (O’Shea and Gadina, 2019; O’Shea et al., 2015). However, to date, MN has not been recognized to involve JAK-STAT dysregulation. Identifying more patients with JAK1 GoF mutations will be critical to determine whether the link between JAK1 GoF and MN is causal. If true, then MN, the most common cause of nephrotic syndrome, may be amenable to early treatment with JAK inhibitors. In fact, baricitinib, a JAK1 and JAK2 inhibitor, demonstrated recent success in a clinical trial for diabetic nephropathy, a distinct but related nephrotic syndrome (Tuttle et al., 2018).The absence of MN in the other reported JAK1 GoF mutation (A634D) may represent important distinctions in the behavior of different mutated forms of JAK1, rather than variable penetrance of the same genetic etiology. Disruptions of the pseudokinase domain may have vastly divergent functional consequences, which can already be gleaned by comparing the JAK1 mutations identified to date: P733L and P832S (LoF) versus A634D and S703I (GoF). Moreover, our findings indicate that GoF may not necessarily lead to the universal activation of downstream pathways, as S703I caused the hyperactivation of some pathways, but not others. Furthermore, disruption of the pseudokinase domain by S703I enabled cells to respond promiscuously via non-canonical signaling pathways. These findings suggest that the pseudokinase domain is not a simple “on/off” switch. This regulatory complexity may stem from the ability of the JAK1 pseudokinase to modulate the activity of JAK2 and TYK2, as demonstrated here. Consequently, careful study of each mutation is warranted, each yielding valuable information on fundamental JAK1 function. The complex regulation becomes especially important, given the high incidence of oncogenic JAK mutations, as well as the expanding therapeutic use of JAK inhibitors. Regarding the latter, the evidence presented here and elsewhere (Eletto et al., 2016; Haan et al., 2011; Li et al., 2013) of the highly cooperative action of JAKs challenges the strategic wisdom of increasing the selectivity of JAK inhibitors.Following transactivation of partnering JAKs, S703I JAK1 constitutively upregulated STAT1, STAT2, STAT3, STAT4, and STAT5 phosphorylation and their downstream target genes, all in the apparent absence of circulating cytokine. These STATs mediate signaling for >25 cytokine pathways, and thus the clinical disease that results is likely driven by a complex mixture of these pathways. Most predominantly, the pathophysiology in this patient appeared to be a combined (1) autoinflammatory and (2) atopic disease process. The presence of severe, early-onset unspecified inflammation throughout the gastrointestinal tract and skin with drastically high acute phase reactants is evocative of primary autoinflammatory disorders. In this regard, the JAK1 GoF here represents a partial phenocopy of other systemic autoinflammatory diseases that overactivate the IFN and IL-6 axes. These include STAT3 GoF mutations (Fabre et al., 2019) and the type I interferonopathies, disorders of overactive STAT1 and STAT2 activity (Rodero and Crow, 2016)—pathways that were highly upregulated by JAK1 S703I. Other disease features suggested an atopic syndrome. Eosinophils were elevated in peripheral circulation and infiltrating into the gastrointestinal lamina propria and crypt epithelium. Asthma, food and environmental allergies, and a skin disease similar to atopic dermatitis were severe and early in onset. Lastly, her renal disease (membranous nephropathy) could only be ameliorated by an elemental diet of strictly liquid nutrients, suggesting a dietary hypersensitivity trigger. Considering the high basal phosphorylation of STAT5 in this patient, these features seem to phenocopy GoF STAT5b (Ma et al., 2017) mutations, which result in early-onset eosinophilia, urticaria, dermatitis, and diarrhea. While it is possible the pathological inflammatory stage of COVID-19 pathogenesis was exaggerated in this patient, other risk factors must be considered, including the comorbidities (e.g., asthma, renal failure) and the combined immunosuppression (i.e., tofacitinib, tacrolimus, and oral glucocorticoids) in this patient. In particular, there is an urgent need to understand how JAK inhibition modulates both the protective and pathological inflammatory immune responses to SARS-CoV-2, as JAK inhibitors are in clinical trials for COVID-19.Finally, this study advances our understanding of mosaicism—at both a genomic and transcriptomic level—in personalized medicine. Genomically, this JAK1 mutation was uncovered, mapped across tissues to trace its embryological origin, and directly linked to phenotypic consequences by deriving JAK1-WT and JAK1-S703I cell lines from the patient. These findings underscore how asymmetric clinical manifestations like those observed here (leg-length discrepancy and dermatitis along lines of embryological migration) should prompt suspicions of mosaicism and guide genetic analysis with carefully chosen techniques. In particular, the analyses described here have only recently become possible with technological advances in single-cell assays. Here, by adapting the inDrops scRNA-seq platform, we implement an approach for the detection and analysis of a specific mutation within a transcript region not readily accessible by standard methods. The resulting single-cell resolution data allowed us to determine mutant allele frequency in different cell populations, identify mutation-associated gene expression patterns, and, unexpectedly, to observe allelic bias in the transcription of JAK1. This bias is consistent with recent evidence of widespread transcriptional bursting and monoallelic expression of autosomal genes (Borel et al., 2015; Gimelbrant et al., 2007; Jeffries et al., 2012; Reinius and Sandberg, 2015). This report demonstrates monoallelic expression of a mutated gene. Biased allelic expression, in conjunction with mosaicism, may prove an important point of focus for future genetic studies of variable penetrance, affected carriers, and undiagnosed disease.In conclusion, intense basic mechanistic investigations of a single mutation identified pan-JAK inhibition, as opposed to highly selective JAK1 inhibition, as the optimal therapy for personalized medicine, leading to biological and clinical rescue. This approach constitutes a workflow for alike monogenic syndromes.

Undiagnosed disease programs have shown promise for the detection of potential causative variants of disease (Lee et al., 2014; Splinter et al., 2018; Yang et al., 2014). This report demonstrates the value of the in-depth study of select patients identified in these programs. Most immediately, the findings described herein directed the successful treatment of a complex immunodysregulatory disease in a personalized molecular fashion. More broadly, this case implicates JAK1 dysfunction in common forms of multifactorial diseases, including dermatitis, enteritis, colitis, and eosinophilic disorders. These features align with the other reported JAK1 GoF mutation recently published (Del Bel et al., 2017), and together provide strong justification for the expanding use of JAK inhibitors in these disorders (O’Shea and Gadina, 2019; O’Shea et al., 2015). However, to date, MN has not been recognized to involve JAK-STAT dysregulation. Identifying more patients with JAK1 GoF mutations will be critical to determine whether the link between JAK1 GoF and MN is causal. If true, then MN, the most common cause of nephrotic syndrome, may be amenable to early treatment with JAK inhibitors. In fact, baricitinib, a JAK1 and JAK2 inhibitor, demonstrated recent success in a clinical trial for diabetic nephropathy, a distinct but related nephrotic syndrome (Tuttle et al., 2018).

The absence of MN in the other reported JAK1 GoF mutation (A634D) may represent important distinctions in the behavior of different mutated forms of JAK1, rather than variable penetrance of the same genetic etiology. Disruptions of the pseudokinase domain may have vastly divergent functional consequences, which can already be gleaned by comparing the JAK1 mutations identified to date: P733L and P832S (LoF) versus A634D and S703I (GoF). Moreover, our findings indicate that GoF may not necessarily lead to the universal activation of downstream pathways, as S703I caused the hyperactivation of some pathways, but not others. Furthermore, disruption of the pseudokinase domain by S703I enabled cells to respond promiscuously via non-canonical signaling pathways. These findings suggest that the pseudokinase domain is not a simple “on/off” switch. This regulatory complexity may stem from the ability of the JAK1 pseudokinase to modulate the activity of JAK2 and TYK2, as demonstrated here. Consequently, careful study of each mutation is warranted, each yielding valuable information on fundamental JAK1 function. The complex regulation becomes especially important, given the high incidence of oncogenic JAK mutations, as well as the expanding therapeutic use of JAK inhibitors. Regarding the latter, the evidence presented here and elsewhere (Eletto et al., 2016; Haan et al., 2011; Li et al., 2013) of the highly cooperative action of JAKs challenges the strategic wisdom of increasing the selectivity of JAK inhibitors.

Following transactivation of partnering JAKs, S703I JAK1 constitutively upregulated STAT1, STAT2, STAT3, STAT4, and STAT5 phosphorylation and their downstream target genes, all in the apparent absence of circulating cytokine. These STATs mediate signaling for >25 cytokine pathways, and thus the clinical disease that results is likely driven by a complex mixture of these pathways. Most predominantly, the pathophysiology in this patient appeared to be a combined (1) autoinflammatory and (2) atopic disease process. The presence of severe, early-onset unspecified inflammation throughout the gastrointestinal tract and skin with drastically high acute phase reactants is evocative of primary autoinflammatory disorders. In this regard, the JAK1 GoF here represents a partial phenocopy of other systemic autoinflammatory diseases that overactivate the IFN and IL-6 axes. These include STAT3 GoF mutations (Fabre et al., 2019) and the type I interferonopathies, disorders of overactive STAT1 and STAT2 activity (Rodero and Crow, 2016)—pathways that were highly upregulated by JAK1 S703I. Other disease features suggested an atopic syndrome. Eosinophils were elevated in peripheral circulation and infiltrating into the gastrointestinal lamina propria and crypt epithelium. Asthma, food and environmental allergies, and a skin disease similar to atopic dermatitis were severe and early in onset. Lastly, her renal disease (membranous nephropathy) could only be ameliorated by an elemental diet of strictly liquid nutrients, suggesting a dietary hypersensitivity trigger. Considering the high basal phosphorylation of STAT5 in this patient, these features seem to phenocopy GoF STAT5b (Ma et al., 2017) mutations, which result in early-onset eosinophilia, urticaria, dermatitis, and diarrhea. While it is possible the pathological inflammatory stage of COVID-19 pathogenesis was exaggerated in this patient, other risk factors must be considered, including the comorbidities (e.g., asthma, renal failure) and the combined immunosuppression (i.e., tofacitinib, tacrolimus, and oral glucocorticoids) in this patient. In particular, there is an urgent need to understand how JAK inhibition modulates both the protective and pathological inflammatory immune responses to SARS-CoV-2, as JAK inhibitors are in clinical trials for COVID-19.

Finally, this study advances our understanding of mosaicism—at both a genomic and transcriptomic level—in personalized medicine. Genomically, this JAK1 mutation was uncovered, mapped across tissues to trace its embryological origin, and directly linked to phenotypic consequences by deriving JAK1-WT and JAK1-S703I cell lines from the patient. These findings underscore how asymmetric clinical manifestations like those observed here (leg-length discrepancy and dermatitis along lines of embryological migration) should prompt suspicions of mosaicism and guide genetic analysis with carefully chosen techniques. In particular, the analyses described here have only recently become possible with technological advances in single-cell assays. Here, by adapting the inDrops scRNA-seq platform, we implement an approach for the detection and analysis of a specific mutation within a transcript region not readily accessible by standard methods. The resulting single-cell resolution data allowed us to determine mutant allele frequency in different cell populations, identify mutation-associated gene expression patterns, and, unexpectedly, to observe allelic bias in the transcription of JAK1. This bias is consistent with recent evidence of widespread transcriptional bursting and monoallelic expression of autosomal genes (Borel et al., 2015; Gimelbrant et al., 2007; Jeffries et al., 2012; Reinius and Sandberg, 2015). This report demonstrates monoallelic expression of a mutated gene. Biased allelic expression, in conjunction with mosaicism, may prove an important point of focus for future genetic studies of variable penetrance, affected carriers, and undiagnosed disease.

In conclusion, intense basic mechanistic investigations of a single mutation identified pan-JAK inhibition, as opposed to highly selective JAK1 inhibition, as the optimal therapy for personalized medicine, leading to biological and clinical rescue. This approach constitutes a workflow for alike monogenic syndromes.

STAR★MethodsKey Resources Table
REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesMouse anti-JAK1 Clone B-3Santa Cruz BiotechnologyCat No. sc-376996; AB_2687564Rabbit anti-JAK2 Clone D2E12Cell Signaling TechnologyCat No. 3230T; AB_10691469Rabbit anti-phospho-JAK2 Tyr1007/1008Cell Signaling TechnologyCat No 3771S; RRID:AB_330403Rabbit anti-JAK3 Clone D1H3Cell Signaling TechnologyCat No. 8827S; RRID:AB_10999548Rabbit anti-TYK2 Clone D415TCell Signaling TechnologyCat No. 14193S; RRID:AB_2798419Rabbit anti-phospho-TYK2 Tyr1054/1055Cell Signaling TechnologyCat No. 9321; RRID:AB_2303972Mouse anti-phospho-tyrosine Clone 4G10Millipore SigmaCat No. 05-321XMouse anti-STAT1 Clone C-111Santa Cruz BiotechnologyCat No. sc417; RRID:AB_675902Rabbit anti-STAT2Millipore SigmaCat No. 06502; RRID:AB_31014Rabbit anti-phospho-Tyr 701-STAT1 Clone 58D6Cell Signaling TechnologyCat No. 9167; RRID:AB_561284Rabbit anti-phospho-Tyr-689-STAT2 Clone D3P2PCell Signaling TechnologyCat No. 88410; RRID:AB_2800123Rabbit anti-phospho-STAT3 Tyr705 Clone D3A7Cell Signaling TechnologyCat No. 9145; RRID:AB_2491009Rabbit anti-phospho-STAT5 Tyr694 Clone C11C5Cell Signaling TechnologyCat No. 9359S; RRID:AB_823649Rabbit anti-phospho-STAT6 Tyr64Cell Signaling TechnologyCat No. 9361T; RRID:AB_331595Rabbit anti-β-actin Clone 13E5Cell Signaling TechnologyCat No. 4970; RRID:AB_2223172Mouse anti-GAPDH Clone D16H11Cell Signaling TechnologyCat No. 5174; RRID:AB_10622025Goat anti-mouse IgG HRP-conjugatedSouthern BiotechCat No. 101005; RRID:AB_2687483Goat anti-rabbit IgG HRP-conjugatedSouthern BiotechCat No. 403005; RRID:AB_2687483Rabbit anti-IFNAR2 MMHAR-2PBLCat No. 21385-1; RRID:AB_387828Discovery OmniMap anti-rabbit HRP (RUO)RocheCat No. 760-4311; RRID:AB_2811043anti-IgD 141Pr-conjugated Clone IA6-02BiolegendCat No.348202; RRID:AB_10550095anti-CD19 142Nd-conjugated Clone HIB19BiolegendCat No.302202; RRID:AB_266181anti-CD45RA 143Nd-conjugated Clone HI100BiolegendCat No.304102; RRID:AB_314406anti-CD141 144Nd-conjugated Clone M80BiolegendCat No.344102; RRID:AB_2661788anti-CD4 145Nd-conjugated Clone RPA-T4BiolegendCat No.300502; RRID:AB_314070anti-CD8 146Nd-conjugated Clone RPA-T8BiolegendCat No.301002; RRID:AB_2661818anti-CD20 147Sm-conjugated Clone 2H7BiolegendCat No.302302; RRID:AB_314250anti-CD16 148Nd-conjugated Clone 3G8BiolegendCat No.302014; RRID:AB_314214anti-CD127 149Sm-conjugated Clone A019D5FluidigmCat No.3149011B; RRID:AB_2661792anti-CD1c 150Nd-conjugated Clone L161BiolegendCat No.331502; RRID:AB_2661820anti-CD123 151Eu-conjugated Clone 6H6BiolegendCat No.306002; RRID:AB_2661822anti-CD66b 152Sm-conjugated Clone G10F5BiolegendCat No.305102; RRID:AB_2661823anti-PD1 153Eu-conjugated Clone EH12.2H7BiolegendCat No.329926; RRID:AB_11147365anti-CD86 154Sm-conjugated Clone IT2.2BiolegendCat No.305410; RRID:AB_314530anti-CD27 155Gd-conjugated Clone O323BiolegendCat No.302802; RRID:AB_2661825anti-PDL1 156Gd-conjugated Clone 29E.2A3BiolegendCat No.329710; RRID:AB_2275581anti-CD33 158Gd-conjugated Clone WM53BiolegendCat No.303402; RRID:AB_314346anti-CD24 159Tb-conjugated Clone ML5BiolegendCat No.311102; RRID:AB_314851anti-CD14 160Gd-conjugated Clone M5E2BiolegendCat No.301810; RRID:AB_31419anti-CD56 161Dy-conjugated Clone B159BD BiosciencesCat No.555513; RRID:AB_395903anti-CD169 162Dy-conjugated Clone 7-239BiolegendCat No. 346002; RRID:AB_2189031anti-CXCR5 163Dy-conjugated Clone REA103MiltenyiCat No.130-122-325; RRID:AB_2801905anti-CD69 164Dy-conjugated Clone FN50BiolegendCat No.310902; RRID:AB_314837anti-CCR6 165Ho-conjugated Clone G034E3BiolegendCat No.353402; RRID:AB_10918625anti-CD25 166Er-conjugated Clone M-A251BiolegendCat No.356102; RRID:AB_2661833anti-CCR7 167Er-conjugated Clone G043H7BiolegendCat No.353256; RRID:AB_2814291anti-CD3 168Er-conjugated Clone UCHT1BiolegendCat No.300402; RRID:AB_2661835anti-CX3CR1 169Tm-conjugated Clone 2A9-1BiolegendCat No.341602; RRID:AB_1595422anti-CD38 170Er-conjugated Clone HB-7BiolegendCat No.356602; RRID:AB_2661836anti-CD161 171Yb-conjugated Clone HP-3G10BiolegendCat No.339902; RRID:AB_2661837anti-CD209 172Yb-conjugated Clone 9E9A8BiolegendCat No.330102; RRID:AB_1134253anti-CXCR3 173Yb-conjugated Clone REA232MiltenyiCat No.130-108-022; RRID:AB_2655743anti-HLADR 174Yb-conjugated Clone L243BiolegendCat No.307602; RRID:AB_314680anti-Axl 175Lu-conjugated Clone 108724R&D SystemsCat No.MAB154; RRID:AB_2062558anti-CCR4 176Yb-conjugated Clone 205410R&D SystemsCat No.MAB1567; RRID:AB_2074395anti-pSTAT5 147 Sm-conjugated Clone 47FluidigmCat No.3147012A; RRID:AB_2827887anti-pSTAT6 149 Sm-conjugated Clone 18FluidigmCat No.3149004Aanti-pSTAT1 153 Eu-conjugated Clone 4aFluidigmCat No.3153005A; RRID:AB_2744689anti-pp38 156 Gd-conjugated Clone D3F9FluidigmCat No.3156002A; RRID:AB_2661826anti-pSTAT3 158 Gd-conjugated Clone 4FluidigmCat No.3158005A; RRID:AB_2811100anti-pMAPKAP2 159 Tb-conjugated Clone 27B7FluidigmCat No.3159010A; RRID:AB_2661828anti-STAT3 165 Ho-conjugated Clone 124H6FluidigmCat No.3173003Aanti-STAT1 169 Tm-conjugated Clone 10C4B40BiolegendCat No.661002; RRID:AB_2563664anti-pERK 171 Yb-conjugated Clone D13.14.4EFluidigmCat No.3171010A; RRID:AB_2811250anti-pS6 175 Lu-conjugated Clone N7-548FluidigmCat No.3175009A; RRID:AB_2811251Mouse anti-NKp46 Pacific Blue Clone 900BioLegendCat No. 331912; RRID:AB_2149280Mouse anti-CD56 Brilliant Violet 605 Clone HCD56BioLegendCat No. 318334; RRID:AB_2561912Mouse anti-CD16 Brilliant Violet 650 Clone 3G8BioLegendCat No. 302041; RRID:AB_11125578Mouse anti-CD3 Brilliant Violet 711 Clone OKT3BioLegendCat No. 317328; RRID:AB_2562907Mouse anti-CD8-alpha Brilliant Violet 785 Clone RPA-T8BioLegendCat No. 301046; RRID:AB_2563264Mouse anti-CD107a FITC Clone eBioH4A3eBioscienceCat No.53-1079-42; RRID:AB_2016657Mouse anti-NKp44 PE Clone Z231Beckman CoulterCat No. IM3710; RRID:AB_ 2857937Mouse anti-NKG2D PE-Cy7 Clone 1D11BioLegendCat No. 320811; RRID:AB_2133275Mouse anti-CD69 PE-CF594 Clone FN50BD BiosciencesCat No. 562617; RRID:AB_2737680Mouse anti-NKp30 APC Clone P30-15BioLegendCat No. 325209; RRID:AB_2149450Mouse anti-CD16 APC Cy7 Clone B73.1BD BiosciencesCat No. 561306; RRID:AB_10643005Mouse anti-CD25 APC-Alexa Fluor 700 Clone B1.49.9Beckman CoulterCat No. A86356;Rat anti-CD2 Pacific Blue Clone 39C1.5Beckman CoulterCat No. B09685; RRID:AB_2847880Mouse anti-CD244 FITC Clone 25235BD BiosciencesCat No. 550815; RRID:AB_393900Mouse anti-CD11c PerCP Cy5.5 Clone BU15Beckman CoulterCat No. B19719;Mouse anti-CD28 PE Clone L293BD BiosciencesCat No. 348047; RRID:AB_400368Mouse anti-CD11a PE-Cy7 Clone HI111BD BiosciencesCat No. 561387; RRID:AB_10611572Mouse anti-CD54 PE-Cy5 Clone HA58BD BiosciencesCat No. 555512; RRID:AB_395902Mouse anti-CD11b PE-CF594 Clone ICRF44BD BiosciencesCat No. 562399; RRID:AB_2737613Mouse anti-CD18 APC Clone 6.7BD BiosciencesCat No. 551060; RRID:AB_398485Mouse anti-CD158e Brilliant Violet 421 Clone DX9BioLegendCat No. 312714; RRID:AB_312714Mouse anti-CD158b FITC Clone DX27BioLegendCat No. 312604; RRID:AB_2296486Mouse anti-CD94 PerCP Cy5.5 Clone HP-3D9BD BiosciencesCat No. 562361; RRID:AB_11152081Mouse anti-NKG2C PE Clone 134591R&D SystemsCat No. FAB138P ; RRID:AB_2132983Mouse anti-CD158a/h/g PE-Cy7 Clone HP-MA4Affymetrix eBioscienceCat No. 25-1589-42; RRID:AB_10854424Mouse anti-KIR2DS4 APC Clone 179315R&D SystemsCat No. FAB1847A; RRID:AB_2130821Recombinant anti-KLRG1 APC-Vio770 Clone REA226Miltenyi BiotecCat No. 130103642; RRID:AB_2652580Mouse anti-NKG2A Alexa Fluor 700 Clone 131411R&D SystemsCat No. FAB1059N; RRID:AB_10972129Mouse anti-CD57 Pacific Blue Clone NC1Beckman CoulterCat No. A74779 ; RRID:AB_131600Mouse anti-CD62L Brilliant Violet 650 Clone DREG-56BioLegendCat No. 304831; RRID:AB_2561461Mouse anti-CD127 Brilliant Violet 785 Clone A019D5BioLegendCat No. 351330; RRID:AB_2563605Mouse anti-IL-15R-alpha Alexa Fluor 488 Clone 151303R&D SystemsCat No. FAB1471G; RRID:AB_10891330Mouse anti-CD117 PE-Cy7 Clone 104D2D1Beckman CoulterCat No. IM3698; RRID:AB_131184Mouse anti-CD16 PE-CF594 Clone 3G8BD BiosciencesCat No. 562293; RRID:AB_11151916Mouse anti-CD94 APC Clone DX22BioLegendCat No. 305508; RRID:AB_2133129Mouse anti-Perforin Briliant Violet 421 Clone B-D48BioLegendCat No. 353307; RRID:AB_ 11149688Mouse anti-IFN-gamma Alexa Fluor 700 Clone 4S.B3BioLegendCat No. 502520; RRID:AB_528921Mouse anti-Eomes eFluor 660 Clone WD1928eBioscienceCat No. 50-4877-42; RRID:AB_2574229Mouse anti-CD34 Alexa Fluor 700 Clone 581BioLegendCat No. 343526; RRID:AB_2561495Mouse anti-Perforin APC Cy7 Clone dG9BioLegendCat No. 308128; RRID:AB_2572051Mouse anti-Helios Brilliant Violet 421 Clone 22F6BioLegendCat No. 137233; RRID:AB_2565798Mouse anti-T-bet FITC Clone 4B10BioLegendCat No. 644812; RRID:AB_2200540Mouse anti-IRF8 PercP- eFluor 710 Clone V3GYWCHeBioscienceCat No. 46-9852-80; RRID:AB_2573903Mouse anti-GATA3 PE Cy7 Clone L50-823BD BiosciencesCat No. 560405; RRID:AB_1645544Mouse anti-NKp44 PE Clone P44-8BioLegendCat No.325107; RRID:AB_756099Goat anti-mouse IgG Alexa Fluor 647Thermo FisherCat No. A-21235; RRID:AB_2535804Bacterial and Virus StrainsDH5-Alpha Competent E. ColiMolecular Cloning LaboratoriesCat No. DA-196Biological SamplesHuman whole blood samplesVarious institutionsN/AChemicals, Peptides, and Recombinant ProteinsIntron-A Recombinant Interferon Alpha-2bMerck PharmaceuticalsCat No. NDC0085057102Recombinant Human IL-2BioLegendCat No. 589102Recombinant Human IL-4BioLegendCat No. 574002Recombinant Human IL-5BioLegendCat No. 560701Recombinant Human IL-6BioLegendCat No. 570802Recombinant Human IL-13BioLegendCat No. 571102Recombinant Human IL-21BioLegendCat No. 571202Recombinant Human Interferon GammaBioLegendCat No. 570206phorbol 12-myristate 13-acetateSigma-AldrichCat No. P1585IonomycinSigma-AldrichCat No. I0634Brefeldin ASigma-AldrichCat No. B7651RuxolitinibSelleckchemCat No. S1378TofacitinibSelleckchemCat No. S5001FilgotinibSelleckchemCat No. S7605Proteomic Stabilizer Prot1SMART TUBE IncCat No. 501351691HeparinSigmaCat No. 201060Osmium tetroxide (99.9%)ACROS organicsCat No. 191180010Cell-ID 20-plex Pd Barcoding KitFluidigmCat No. 201060Human TruStain FcX (Fc Receptor Blocking Solution)BioLegendCat No. 422301X8 MaxPar conjugation kitsFluidigmCat No. 201300125nM Ir IntercalatorFluidigmCat No. 201192AOsO4ACROS OrganicsCat No. AC319010050Maxpar Cell Acquisition BufferFluidigmCat No. 201241Cytofix/Cytoperm Fixation/Permeabilization SolutionBDCat No. 554714FoxP3 Transcription Factor Staining BufferTonboCat No. TNB0607Live/Dead AquaThermo FisherCat No. L34957Discovery Ultra antibody blockRocheCat No. 760-4204Pierce ECL Western Blotting SubstrateThermo Fisher ScientificCat No. 32106RIPA Lysis and Extraction BufferThermo Fisher ScientificCat No. 89900Protease/Phosphatase Inhibitor CocktailCell Signaling TechnologiesCat No. 5872Nupage Western Blot Sample BufferThermo Fisher ScientificCat No. NP0007Dynabeads Protein GThermo Fisher ScientificCat No. 10007DHistopaque 1077Millipore SigmaCat No. 10771-500CyclosporinSigma AldrichCat No. C3662OptiPrep Density Gradient MediumSigma AldrichCat Nol. D1556-250mLNovec 7500 Engineered Fluid3MCat No. Novec 75001H,1H, 2H, 2H – Perfluorooctanol, 97%Alfa Aesar, Thermo Fisher ScientificCAS No. 647-42-750 g of 10 weight % 008-Flurorosurfactant in HFE7500Ran Biotechnologies008-FlurorSurfactant-10wtH-50GExonuclease I (E. Coli)New England BiolabsM0293LFast Digest HinfIThermo Fisher ScientificFD0804AMPure XPBeckman CoulterA63881RNAClean XPBeckman CoulterA66514RNaseHNew England BiolabsM0297LCritical Commercial AssaysQuikChange II XL site-directed mutagenesis kitAgilent TechnologiesCat No. 200522Gateway LR Clonase II Enyme MixThermo Fisher ScientificCat No. 11791100Gateway BP Clonase Enzyme MixThermo Fisher ScientificCat No. 11789020QIAamp DNA Mini KitQIAGENCat No. 51304RNeasy RNA Isolation KitQIAGENCat No. 74106Applied Biosystems High-Capacity cDNA Reverse Transcription KitThermo Fisher ScientificCat No. 4368814TaqMan Universal Master Mix II with UNGThermo Fisher ScientificCat No. 4440039Human Magnetic Luminex Assay CustomR&DCat No. LXSAHMBio-Plex Pro Human Inflammation PanelBioRadCat No. 171al001mCell-ID 20-Plex Pd Barcoding KitFluidigmCat No. 201060Chromium single cell Chip Kit V210X GenomicsCat No. 120236Chromium single cell 3′ Library and Gel Bead Kit10X GenomicsCat No.120237Discovery ChromoMap DAB KitRocheCat No. 760-159MycoAlert PLUS Mycoplasma Detection KitLonzaCat No. LT07-703Lipofectamine 3000 Transfection ReagentThermo FisherCat No. L3000001Invitrogen Taq polymeraseThermo FisherCat No. 10342020Digital droplet PCR SupermixBioradCat No. 1863026IgG Autoantibody ArrayRayBiotechCat No. PAH-AIDG-G1-16)Single Cell Lysis BufferAmbionCat No. 4458235SuperScript VILO RT kitThermo FisherCat No. 11754050SuperScript™ III Reverse TranscriptaseThermo Fisher ScientificCat No. 18080093HiScribe T7 High Yield RNA Synthesis KitNew England BiolabsCat No. E2040SKapa HiFi HotStart ReadyMix PCRocheKR0370NextSeq 500/550 High Output Kit v2.5 (75 cycles)Illumina20024906Deposited DatascRNA-seq Data and AnalysisThis paperhttps://github.com/jorgcalis/JAK1-alleles-pipelineExperimental Models: Cell LinesHEK293TATCCCRL-3216OP9ATCCCRL-2749U4C (JAK1−/−)Sandra PellegriniN/Aγ2A (JAK2−/−)Sandra PellegriniN/AEBV-immortalized B cellsIcahn School of Medicine at Mount SinaiN/AOligonucleotidesMX1 mRNA TaqMan FAM Hs200895608_m1Thermo FisherCat No. 4331182RSAD2 mRNA TaqMan FAM Hs00369813_m1Thermo FisherCat No. 4351370scRNaseq polyT oligonucleotide template 5′-BAAAAAAAAAAAAAAAAAAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG – 3′Integrated DNA TechnologiesN/AscRNaseq JAK1 oligonucleotide template 5′ - GAGTGTGGCCCATTCATCAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG −3′Integrated DNA TechnologiesN/AscRNaseq JAK1 blocking oligonucleotide 5′-GAGTGTGGCCCATTCATCAA-3′Integrated DNA TechnologiesN/AscRNaseq second reverse transcription JAK1 primer: 5′- TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGCCATGGAAATTCAAAGTTGCCAAACAG-3′Integrated DNA TechnologiesN/AJAK1 single cell qPCR amplification (F: GTCCTCTGGATCTCTTCATGCA, R: GCTGTTTGGCAACTTTGAATTTCC)Thermo FisherN/AJAK1 single cell genotyping probes (AAGGACATC[G/A]CTTTTC)Thermo FisherN/ARecombinant DNApTRP IRES RFP PuroRThis paperN/ApDONR223 JAK1AddGeneCat No. 23932pDONR223 JAK2AddGeneCat No. 23915pCAGGS-VSV-GThis paperN/ApCMV-Gag/PolThis paperN/ASoftware and AlgorithmsCytobankBeckman Coulterhttps://www.cytobank.orgFlowJoBecton Dickinson Companyhttps://www.flowjo.comR for Statistical Computinghttps://www.r-project.orgRStudioThe R Foundationhttps://rstudio.comCellRanger10X Genomicshttps://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-rangerPython v2.7Pythonhttps://www.python.org/Seurat v2.3.4 R toolkitSatija Lab, New York Genome Centerhttps://satijalab.org/seurat/EdgeR v3.12.1Bioconductorhttp://bioinf.wehi.edu.au/edgeR/inDrops.py v0.3GitHubhttps://github.com/indrops/indropsGenome Analysis Toolkit (GATK)Broad Institutehttps://gatk.broadinstitute.org/hc/en-usIngenuity Variant AnalysisQIAGENhttps://variants.ingenuity.comGraphPad Prism 7GraphPad Softwarehttps://www.graphpad.com/scientific-software/prism/QX100 softwareBioRadN/AGenepix Pro 7.0 softwareMolecular DevicesN/A
Resource AvailabilityLead ContactFurther information and requests for reagents may be directed to the Lead Contact, Dusan Bogunovic (dusan.bogunovic@mssm.edu).Materials AvailabilityAll unique/stable reagents generated in this study are available from the Lead Contact with a completed Materials Transfer Agreement.Data and Code AvailabilityThe raw data for experiments performed, including the PCR, multiplex ELISA, mass cytometry, flow cytometry, scRNaseq data, are available upon request from the lead contact. For privacy concerns of the study participants, raw scRNA-Seq data are not available, and the complete data files from whole exome sequencing will be restricted to the variants in Table S1. scRNA-Seq gene x cell matrices, python scripts and associated Seurat code (R-based) used for the tailored analysis of JAK1-specific scRNA-Seq are available at https://github.com/jorgcalis/JAK1-alleles-pipeline.Experimental Model and Subject DetailsPatientsThis study reports an 18-year-old female who presented during early childhood with persistent, recurrent cutaneous and gastrointestinal inflammatory disease with eosinophilic infiltration and peripheral eosinophilia. Notable physical features include a leg length discrepancy, short stature and low body weight. In parallel, she developed refractory membranous nephropathy leading to end stage renal disease. A kidney transplant was complicated by disease recurrence in the graft, progressing over several years, as well as acute rejection ultimately rendering the patient dialysis-dependent. Specifically, her past medical history by organ involvement is further detailed, as follows:Renal: At 3 years of age, she developed rapid weight gain, edema and proteinuria. Renal biopsy at age 7 demonstrated membranous nephropathy, which was refractory to treatment with corticosteroids, and later cyclosporine and tacrolimus. By history the nephrotic syndrome was ameliorated by use of an elemental diet, but this was not able to be consistently maintained. Serological testing for anti-PLA-2R receptor, anti- thrombospondin and anti–bovine serum albumin was all negative. A gradual decline in renal function was observed and at age 11 a living-donor kidney transplantation was performed. One year later, nephrotic range proteinuria recurred, and biopsy confirmed relapsing membranous nephropathy. The graft function further declined, and an episode of acute antibody-mediated rejection resulted in transplant failure at age 16. She has required long-term hemodialysis since that time, delivered via an AV fistula, and is currently being evaluated for a second transplant.Dermatologic: At birth, a pustular rash involving face and extremities was noted. The rash persisted and worsened after discharge. Skin biopsy at 3 months suggested Inflammatory Linear Verrucous Nevus. Skin involvement spread and worsened in intensity with age, manifesting as a diffuse, erythematous rash involving the face, trunk and extremities, with prominence on the left side. Biopsies later demonstrated a subacute or chronic spongiotic dermatitis. The epidermis was acanthotic and showed varying degrees of intercellular edema (spongiosis) with widening of the intercellular spaces; the stratum corneum was thickened and focally compact; the dermis contained a perivascular lymphohistiocytic infiltrate which extended around the superficial and deep vascular plexus. Of note, the biopsies did not show the changes commonly associated with epidermal nevi: alternating ortho and parakeratosis, epidermolytic hyperkeratosis, or acantholytic dyskeratosis. Rather, these clinical and histologic changes in the skin likely represent a form of blaschkitis, an inflammatory skin condition, presenting as papules or vesicles, occurring along the lines of Blaschko (which represent somatically distinct bands of ectodermal migration).Gastrointestinal: In infancy, she experienced recurrent emesis and diarrhea unresponsive to formula changes. Bloody stools were noted at 10 months of age and watery diarrhea and abdominal pain became persistent. Repeat endoscopic biopsies demonstrated chronic, unspecified inflammation at various sites (most frequently colonic, but also gastric, duodenal, ileal and esophageal regions). Eosinophilic infiltration of the colon was consistently noted. Symptoms were only marginally responsive to treatment with corticosteroids, chronic antibiotics and a severely restricted diet.Growth disturbances: Growth impairment was reflected by short stature (Z score < −3) and low body weight (Z score −2 to −8), currently 138 cm and 31 kg. Nutritional etiologies were addressed by placement of a G-tube at age 10 with some improvement in growth. Growth hormone was administered for 5 years with moderate benefit. Leg length discrepancy was identified at birth, with left extremity smaller than right in girth and length.Immunologic: Allergic reactions were observed to enalapril (anaphylaxis), milk (rash), soy (rash) and wheat (rash). She also experienced occasional episodes of dyspnea along with lip and leg angioedema, without an identifiable inciting allergen. Asthma was diagnosed and managed with bronchodilators. Acute phase reactants were noted to be consistently elevated, including ESR and C-reactive protein. Complement (C3 and C4) levels were within reference range. Likewise, quantitative immunoglobulin testing for IgG, IgA, IgE and IgM was within normal limits. Seroconversion after immunization was observed for all vaccinations except varicella virus, hepatitis A virus and hepatitis B virus.Family history was largely unremarkable, with no family history of consanguinity or gastrointestinal, renal, immunologic or dermatologic disease. Mother, father and older brother are alive and well.Response to tofacitinib: After 8 weeks of tofacitinib there was complete normalization of acute phase reactants, ESR and C-reactive protein upon laboratory assessment. The patient was previously unable to tolerate dairy, soy and gluten due to severe abdominal pain and diarrhea. After initiation of tofacitinib, she liberalized her diet without restriction and remained asymptomatic, with formed stools and without abdominal pain.At baseline the endoscopic findings included altered vascularity and friable mucosa from rectum to descending colon with microscopic patchy, active colitis with eosinophilic infiltration noted in the ascending colon. After 6 months of treatment on tofacitinib 5mg daily, the colon was grossly normal, with microscopic active colitis and complete resolution of eosinophilia. The dose was further increased to 7.5 mg daily thereafter. The dermatitis significantly improved, as seen in the images.Tofacitinib is 30% renally-excreted and the dose administered was limited due to chronic kidney disease. Presumably, after re-transplantation the dose may be escalated with a potentially greater effect. To date, the drug has been very well tolerated with no evidence of adverse effects with close monitoring.Coronavirus Disease 2019 (COVID-19): Two years after initiation of tofacitinib treatment, the patient developed an acute respiratory infection that was later diagnosed as COVID-19. She was admitted to the hospital where she was treated with azithromycin and hydroxychloroquine and subsequently intubated. Her immunosuppressive treatment regiment, including tofacitinib, was held constant over this time. Unfortunately, despite intensive supportive care, the patient expired 7 days after admission.Cell linesU4C cells (JAK1
−/−) and γ2A cells (JAK2
−/−) were obtained from S. Pellegrini and cultured in DMEM supplemented with 10% fetal bovine serum (FBS) (Invitrogen), GlutaMAX (350 ng/ml; GIBCO), and penicillin/streptomycin (GIBCO). All cell lines expressing ectopic JAK1 and JAK2 variants were generated by lentiviral transduction and were subsequently selected with puromycin and FACS for matched RFP expression. EBV-transformed lymphoblastoid cell lines (EBV-B cells) were generated by infecting PBMCs from healthy controls or the patient with EBV supernatants. EBV-B cells were cultured in RPMI with 10% FBS, 1% glutamine and 1% penicillin/streptomycin. Single-cell clones were isolated by limiting dilution analysis on OP9 feeder cells and expanded in conditioned media. After genotyping by Sanger sequencing, WT/WT and WT/S703I clones were selected.Method DetailsVariant analysisAll samples were collected with informed consent in accordance with IRB-approved protocols (Study ID# IF2349568). For whole-exome sequencing, DNA was isolated (QIAGEN Cat No 69504) from Ficoll-isolated granulocytes from whole blood of the proband and her healthy parents. Library preparation, sequencing (150 bp paired-end reads) and alignment for whole-exome sequencing were performed with the Genewiz exome- sequencing package. Potential disease-causing variants were investigated by Ingenuity Variant Analysis (QIAGEN). Hight-quality variants were identified by filtering as follows: exclusion of common variants (> 0.1% allele frequencies in public databases); retention of variants in coding regions resulting in substitutions, premature stops, frameshifts or altered splicing; exclusion of variants with CADD scores below MSC thresholds. For recessive models of inheritance, only homozygous or compound heterozygous mutations were assessed. For de novo inheritance, all variants in the parents were excluded, and alleles with known haploinsufficiency, hemizygous, or dominant-negative effects were included. For the validation of WES results, JAK1 was then amplified from PBMC DNA by PCR and Sanger sequenced.Digital droplet PCRFor the determination of mosaic allele fractions, DNA was isolated from bilateral buccal swabs, fractionated blood and a gastrointestinal endoscopic biopsy specimen in which the epithelial layer was isolated by chemical dissociation. Digital droplet PCR was performed with JAK1 amplification primers, mutation-specific probes (IDT), and with ddPCR Supermix (Biorad 1863026). Amplification and quantification were performed on a QX100 Droplet Digital PCR system (BioRad). Cellular genotypes were estimated with QX100 software (BioRad), assuming heterozygosity.Autoantibody ArrayWe analyzed the presence of circulating autoantibodies using IgG autoantibody arrays (RayBiotech PAH-AIDG-G1-16). In addition to samples from the patient in this study, plasma from five healthy donors, two patients with systemic lupus erythematosus and Immune dysregulation, polyendocrinopathy, enteropathy, X-linked (IPEX) were included. All samples were run in duplicate. Before processing, samples were was first clarified and diluted at a ratio of 1:200. Genepix Pro 7.0 software was used to analyze the images.Cloning and mutagenesisThe JAK1 plasmid was obtained from Addgene (Plasmid #23932) in a Gateway-compatible backbone, pDONR 223. Site-directed mutagenesis with specific primers (Quikchange II, Agilent 200521) was performed to obtain the WT coding sequence in the same plasmid, and plasmids encoding the S703I, K908A or S703I/K908A forms were then generated. A control vector containing the luciferase gene was also cloned. Plasmids were then subcloned into a lentivirus-compatible pTRIP-X-IRES-RFP backbone with puromycin resistance. Pseudotyped lentiviral particles were produced by the transfection of HEK293T cells with pCAGGS-VSV-G, pCMV-Gag/Pol and genes of interest.In vitro stimulationsFor the analysis of STAT phosphorylation, cells were stimulated with the indicated doses of recombinant IFNα-2b (Intron-A, Merck), IFNγ, IL-2, IL-4, and IL-6, IL-13 (Biolegend) for 15 minutes and then lysed for western blotting. For the analysis of gene induction, cells were stimulated for 8 hours, after which cells were lysed for RNA isolation. For JAK inhibitor treatment, cells were incubated with ruxolitinib (Selleckchem S1378), filgotinib (SelleckchemS7605) or tofacitinib (Selleckchem S5001) at the indicated doses for 4 hours.ImmunoblottingCells were lysed in RIPA buffer (Thermo Fisher 89900) supplemented with protease/phosphatase inhibitor cocktail (Cell Signaling #5872). Lysates were sonicated, centrifuged to remove insoluble complexes, then boiled with NuPage sample buffer (Thermo Fisher NP0007) containing 20 mM DTT. The samples were subjected to gel electrophoresis and semi-dry transfer, and the resulting immunoblots were blocked in 5% BSA, then incubated overnight with primary antibody followed by HRP-conjugated secondary antibodies. Primary antibodies against the following targets were used: GAPDH (Cell Signaling D16H11), STAT1 (Santa Cruz C-111), phospho-STAT1 (Cell Signaling 58D6), phospho-STAT2 (Cell Signaling D3P2P), phospho-STAT3 (Cell Signaling D3A7), phospho-STAT5 (Cell Signaling C11C5), phospho-STAT6 (Cell Signaling 9361), JAK1 (Santa Cruz B3), JAK2 (Cell Signaling D2E12), TYK2 (Cell Signaling D4I5T), phospho-JAK1 (Cell Signaling 3331), phospho-JAK2 (Cell Signaling 3771), and phospho-TYK2 (Cell Signaling 9321). For the analysis of JAK phosphorylation, lysates were first incubated overnight with antibodies against total JAK protein conjugated to Protein G Dynabeads (Thermo Fisher 10007D). Immunoblotting was then performed as above.RT-qPCRCell lines or isolated PBMCs were lysed and RNA was isolated with RNeasy spin columns (QIAGEN 74104). Reverse transcription was performed with the High-Capacity RT Kit (Applied Biosystems 4368814). The resulting cDNA was then subjected to qPCR with the TaqMan Master Mix II with UNG (Thermo Fisher 4440038), on a Roche LightCycler 480, with the following primers/probes: 18S (4318839), MX1 (hs00895608), RSAD2 (hs00369813), SIGLEC1 (hs00988063) IFIT1 (hs01911452). The relative expression of each transcript was normalized relative to 18S by the ΔΔCt method.Mass cytometryWhole blood was collected, by venipuncture, into sodium heparin vacutainer tubes. For immunophenotyping, blood was immediately stained and processed for mass cytometry. For intracellular staining, whole blood was stimulated by incubation for 15 minutes with the cytokines indicated and was then immediately stabilized with Proteomic Stabilizer PROT1 (SmartTube) and frozen at −80°C. Similarly, for inhibition with ruxolitinib and tofacitinib, blood was treated for 4 hours with 500 nM inhibitor and was then stimulated by incubation for 15 minutes with 1000 IU/mL IFNα, before stabilization and freezing.Frozen samples were thawed according to the manufacturer’s recommended protocol. The thawed samples were washed with barcode permeabilization buffer (Fluidigm) and barcoded with Fluidigm’s Cell-ID 20-Plex Pd Barcoding Kit. Samples were then washed and pooled into a single tube, Fc-blocked (FcX, Biolegend) and heparin-blocked to prevent non-specific binding. Cells were then stained with a cocktail of markers to identify major immune populations. All antibodies in the panel were either conjugated in-house with X8 MaxPar conjugation kits (Fluidigm) or purchased from Fluidigm. The antibody cocktail was filtered through an Amicon filter with 0.1 μm pores before staining.After surface staining, the samples were permeabilized with methanol and stored for at least 12 hours in methanol at −80°C. Samples were then washed, heparin-blocked and stained with a cocktail of phosphorylation and signaling antibodies. The stained samples were washed and incubated in freshly diluted 2.4% formaldehyde containing 125nM Ir Intercalator (Fluidigm), 0.02% saponin and 30 nM OsO4 (ACROS Organics) for 30 minutes at room temperature. Samples were then washed and acquired immediately after staining.Samples were washed once with PBS+0.2% BSA, once in PBS, and once in CAS buffer (Fluidigm). They were and resuspended at a concentration of 1 million cells per mL in CAS buffer containing a 1/20 dilution of EQ beads (Fluidigm). Following routine instrument tuning and optimization, samples were run at an acquisition rate of < 300 events per second on a Helios mass cytometer (Fluidigm) with a modified wide-bore injector (Fluidigm). FCS files were then normalized and concatenated with Fluidigm acquisition software, and the barcoded samples were deconvoluted with a MATLAB-based debarcoding application (“Palladium-based mass tag cell barcoding with a doublet-filtering scheme and single-cell deconvolution algorithm”). The FCS files were then uploaded to Cytobank for analysis. Cell events were identified as Ir191/193-positive and Ce140-negative events. Doublets were excluded on the basis of Mahalanobis distance and barcode separation and with the Gaussian parameters acquired with Helios CyTOF software. Downstream data analysis was performed on Cytobank, by both tSNE analysis and biaxial gating of immune populations, as shown in Figure S3. Mean signal intensities were calculated, and relative induction was determined by normalization relative to the mean for healthy control samples.Flow cytometryCryopreserved PBMCs from the patient of this study (collected before and after the initiation of tofacitinib treatment), a patient harboring a c.1154C > T p.T385M STAT1 GoF mutation (Vargas-Hernandez et al., 2017), and three unrelated healthy donors were thawed and allowed to rest briefly in complete RPMI medium supplemented with 10% FCS. Cells were immunostained with antibodies in 2% FBS in PBS for 45 minutes. For the panel assessing effector function and activation, cells were stimulated with phorbol 12-myristate 13-acetate (10 ng/ml, Sigma-Aldrich) and ionomycin (1 μg/ml, Sigma-Aldrich) for 4 hours at 37°C in the presence of brefeldin A (10 μg/ml, Sigma-Aldrich) and anti-CD107a antibody. For the panels evaluating effector function and transcription factors, cells were then permeabilized with BD Cytofix/Cytoperm (BD Biosciences) or FoxP3 buffer (Tonbo), and were stained by incubation with antibody for 45-60 minutes. Activated cells were stained for surface markers for 20-25 minutes after the four hours of incubation. Data were acquired on a FACSAria machine (BD Biosciences) with the capacity to detect 18 fluorescent parameters and exported to FlowJo 10.5.3 (BD Biosciences) for analysis. The frequency of cells positive for each parameter was compared with the mean and standard deviation for three healthy donors analyzed in parallel with the samples from the patient.IFNAR2 staining was performed on transduced U4C cells surface stained with anti-IFNAR2 antibody (PBL 21385-1) at a 1:100 dilution for 1 hour on ice. Secondary antibody staining was performed with an anti-murine IgG antibody with an Alexa Fluor 647 tag (Thermo Fischer A-21235). Subsequently the cells were stained with a Live/Dead Aqua stain (Thermo L34957) and then fixed in 4% PFA. Data were acquired on a BD Canto II machine (BD Biosciences) and analyzed using FlowJo 10.5.3.Multiplex ELISAPlasma was collected by Ficoll isolation from heparinized whole blood and clarified by centrifugation. Circulating cytokine levels were determined in magnetic Luminex assays with the Bio-Plex Pro Human Inflammation (BioRad 171al001m) and custom Human Cytokine Panel (R&D LXSAHM), according to the manufacturer’s protocol. Samples were quantified on a MAGPIX xMAP Instrument (Luminex). Cytokine concentrations were quantified by comparison with standard curves and were subsequently translated into Z-scores.ImmunohistochemistryImmunohistochemistry staining was performed with a Discovery Ultra instrument (Roche), with the staining module of RUO Discovery Multimer V2 (V0.00.0083). Slides were first incubated in blocking agent containing 2% BSA PBS, and Discovery Ultra antibody block (Cat # 760-4204) (Roche). Slides were incubated with primary antibodies for 60 minutes at 37°C. The following primary antibodies were used: anti-phospho-STAT1 (Tyr701) 58D6 (Cell Signaling) and anti-phospho-STAT3 (Tyr 705) D3A7 XP (Cell Signaling) at 1:100 dilution in blocking agent. The slides were then incubated with Omni Map anti-rabbit HRP-conjugated secondary antibody (Multimer HRP) (Cat # 760-4311) (Roche) for 32 minutes. Positive signals were detected with the Discovery ChromoMap DAB Kit (Cat #760-159) (Roche).Modified inDrops single cell RNA-Seq targeting JAK1 S703I siteinDrops and related droplet microfluidics single cell RNA-Seq strategies coencapsulate single “barcode microbeads” and individual cells in droplets. Reverse transcription of polyadenylated (polyA) mRNA incorporates a cellular barcode sequence (different for each bead) into nascent cDNA. After downstream high throughput sequencing, reads can be assigned to individual cells by barcode sequences, thereby enabling expression quantification at single cell resolution. As the cellular barcode is introduced downstream of transcript polyA tails, sequence data is typically restricted to transcript regions immediately proximal to 3′ termini. Because the S703I site is located at position 2402 from 5′ transcript start and 2690 from the 3′ transcript polyA tail, it is not accessible by standard droplet microfluidics single cell RNA-Seq platforms. Therefore, we adapted the inDrops method by generating custom barcode microbeads containing JAK1-specific primers (flanking the S703I site) in addition to polyT primer sequences, enabling more efficient JAK1 target capture and access to the S703I site. Following within-droplet reverse transcription, second strand synthesis and in vitro transcription amplification, samples are split into two parallel library preparations: one for standard polyT-primed libraries, and one for JAK1-targeted libraries. During data processing of resulting high throughput sequencing data, sequence reads from both libraries are assigned to individual cells based on shared barcode sequences. A detailed description of this approach appears below.JAK1-targeted hydrogel microbeadsBarcoding hydrogel beads were prepared according to established inDrops protocol (Zilionis et al., 2017) with modified primers (Zilionis et al., 2019) with the following modifications. For the second round of split-and-pool primer extension for barcode synthesis, hydrogel beads were added to microplate wells (n = 384) containing oligonucleotide templates for both standard polyT primers and for an additional primer complementary to an S703I-adjacent region of the JAK1 transcript (11.53uM polyT oligonucleotide template, 2.3uM JAK1 oligonucleotide template) Within a given well, both polyT and JAK1 oligonucleotide templates carried identical “barcode 2” sequences, ensuring that extended primers on hydrogels contained matching barcodes. Subsequent exonuclease processing steps included corresponding JAK1 primer complementary blocking oligonucleotides.JAK1-targeted library preparationFreshly isolated PBMC were co-encapsulated with JAK1-targeted hydrogel beads. The standard inDrops protocol was followed for reverse transcription, droplet breakage, second strand synthesis and in vitro transcription (IVT) amplification. IVT reactions (20 ul) were then split to two parallel library preparations: 10 ul of IVT product was prepped for polyT-primed gene expression libraries according to the standard inDrops protocol, and 10 ul of IVT product (typically reserved as a “backup” aliquot) was prepped according to a JAK1-targeted protocol as follows. IVT products were reverse transcribed by SuperScript III with a JAK1-specific primer (with 5′ extension containing Illumina adaptor sequence) flanking the S703I site (55C for 1 hr, 70C for 15 min). RT reactions were treated with RNase H (37C for 30 min, 65C for 20 min) to remove RNA template from cDNA heteroduplexes. Following purification on 1.5X Ampure XP beads (Beckman Coulter), JAK1-enriched cDNA was amplified by PCR with Illumina-adapted inDrops primers (KAPA HiFi Master Mix; 2 cycles 98C x 20 s, 55C x 30 s, 72C for 40 s; 16-18 cycles 98C x 20 s, 65C x 30 s, 72C for 40 s; final 72C extension x 5 min). Final JAK1-targeted libraries were purified on 0.8X Ampure XP beads.Oligonucleotide sequencespolyT oligonucleotide template 5′-BAAAAAAAAAAAAAAAAAAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG – 3′JAK1 oligonucleotide template 5′ - GAGTGTGGCCCATTCATCAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG −3′JAK1 blocking oligonucleotide 5′-GAGTGTGGCCCATTCATCAA-3′Final “on bead” polyT primer sequence 5′- CGATGACGTAATACGACTCACTATAGGGTGTCGGGTGCAG[bc1,8nt]GTCTCGTGGGCTCGGAGATGTGTA TAAGAGACAG[bc2,8nt]NNNNNNTTTTTTTTTTTTTTTTTTTV-3′Final “on bead” JAK1 primer sequence 5′- CGATGACGTAATACGACTCACTATAGGGTGTCGGGTGCAG[bc1,8nt]GTCTCGTGGGCTCGGAGATGTGTA TAAGAGACAG[bc2,8nt]NNNNNNTTGATGAATGGGCCACACTC-3;Second reverse transcription JAK1 primer: 5′- TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGCCATGGAAATTCAAAGTTGCCAAACAG-3′High Throughput SequencingBoth standard polyT-primed libraries and JAK1-targeted libraries were pooled together and sequenced in multiplex on the Illumina NextSeq 500 platform with 75-cycle reagent kits in paired-end, dual index configuration:Read 1 containing transcript/JAK1 data: 61 cyclesi7 read containing cell barcode data: 8 cyclesi5 read containing sample index data: 8 cyclesRead 2 containing cell barcode and unique molecular identified (UMI) data: 14 cycles
Data ProcessingFASTQ sequence files were processed with the inDrops.py workflow script (v0.3, https://github.com/inDrops/inDrops). For polyT-primed gene expression libraries, resulting gene x cell matrices and per cell read counts were used for downstream analyses in Seurat (details below), using only “cell-containing droplets” defined by a read count higher than 8000-10000 depending on read count evaluation from droplets in the corresponding library batch.For JAK1-targeted libraries, BAM files (appended with cellular barcode and UMI data) were used to evaluate genotypes at single cell resolution as follows. Reads covering the JAK1 S703I site (chr1:64845519 – 64845521 on the minus strand; human genome reference GRCh38) containing the wild-type sequence (GAT) or S703I (GCT) sequence were quantified per UMI per cellular barcode. If at least 2 and at least 90% of the reads for a given cellular barcode/UMI combination contained the same sequence (wild-type or S703I), then this combination was designated a JAK1 transcript of the given genotype and assigned to the appropriate cellular barcode.We next used per cell JAK1 WT and S703I transcript counts to assign putative genotypes to individual cells. As cell free RNA in suspension can co-encapsulate with cells thereby generating unwanted background signal in droplet microfluidics scRNA-Seq methods, we aimed to apply a stringent, data driven threshold for genotyping assignment. To evaluate the potential influence of cell free RNA, we quantified the frequency of JAK1 transcripts detected in empty (i.e., cell free) droplets (defined as barcodes with 1000 – 2000 reads in polyT-primed libraries) using the same barcode/UMI strategy described above. We found that more than 99% of empty droplets had less than 3 JAK1 transcripts. Guided by these data, cells were only considered for JAK1 genotype assignment if at least 5 JAK1 transcripts were detected. For JAK1 genotype assignment, individual cells were classified in one of the following four categories: cells that carried the S703I allele were classified as “MUT+” regardless of carrying the wild-type allele; the remainder (S703I allele negative) cells that carried the wild-type allele were classified as “WT+MUTneg” if no S703I transcripts were detected, and as “WT+MUTnonZero” if 1-to-4 S703I transcripts were detected; cells that carried neither the wild-type nor the S703I allele were classified as “WTnegMUTneg.”Single cell RNA-Seq Gene Expression AnalysisData from JAK1-targeted inDrops single cell RNA-Seq data were analyzed in conjunction with corresponding single cell RNA-Seq data acquired from the same PBMC specimen on the 10X Genomics Chromium platform. Sequence reads from 10X libraries were processed with the CellRanger software package (10X Genomics) using default parameters. Gene x cell matrices from both methods were further analyzed with Seurat (v2.3.4) (Butler et al., 2018) in the R statistical framework as follows. For applicable cells in the JAK1-targeted inDrops dataset, JAK1 genotyping information was imported as per cell metadata entries. Genes with detectable expression in fewer than 5 cells per sample were excluded. Cells with detectable expression of fewer than 250 genes or fewer than 1000 UMI counts or greater than 20% UMIs from mitochondrial gene transcripts (measure of cell viability) were removed from further analysis. Gene expression data were log normalized and scaled (regressing out effects based on total UMI counts and mitochondrial gene expression). All genes with detectable expression in both inDrops and 10X datasets were used for CCA dimensionality reduction (20 dimensions). CCA dimension scores were aligned between datasets using the AlignSubspace function. Aligned data were log-normalized and scaled as above. Graph- based clustering and tSNE visualization was performed with the FindClusters and RunTSNE functions with default settings (using the first 19 CCA dimensions, based on examination of distinct expression patterns with the DimHeatmap function). Marker genes for individual clusters were identified with the FindMarkers() function in Seurat (paramenters: min.pct = 0.1, only.pos = TRUE, logfc.threshold = log(1.5), min.diff.pct = 0.1, pseudocount.use = 0.1). Briefly, up to 100 cells were randomly selected from each cluster, and each cluster was contrasted against all others for differential gene expression. Significant genes (p < 0.05) were ranked by log2 fold-change; the top 25 genes for each cluster, as well as a select list of additional genes of interest, were selected as marker genes for supervised cluster annotation (Table S3). Expression patterns were further evaluated on tSNE plots. Select clusters were merged based on shared expression patterns as indicated.Gene expression differences were assessed for cells assigned JAK1 genotype classifications of either MUT+ (S703I) or WT+MUTneg (WT). Plasmablasts and dendritic cells were excluded from analysis due to insufficient cells assigned to both genotype groups. Apparent cellular doublets and cells in the platelet cluster were also excluded. Differential gene expression testing was performed on genes detected in at least 20% of cells in any cell type cluster (of either JAK1 genoytpe group) with edgeR (v.3.12.1) (Robinson et al., 2010), including modifications included for single cell RNA-Seq data as described (Soneson and Robinson, 2018). A linear model including factors for cellular gene detection rate, cell type, JAK1 genotype group and an interaction term (cell type cluster and JAK1 genotype group), was fit with the glmQLFit function. Differential gene expression testing (WT versus S703I) was tested across all cell types with the glmQLFtest function, with significance thresholds set at FDR 0.05. In addition, gene set enrichment testing for Molecular Signatures Database Collections H, C2- CPC, C3-TFT and C7 collections (http://software.broadinstitute.org/gsea/msigdb), supplemented with two additional interferon stimulated gene sets (Rosenberg et al., 2018) was performed with CAMERA (Wu and Smyth, 2012) and the linear model described above. Only gene sets with 5 or more genes in the fitted model were included in the analysis.Single Cell qPCR Transcript GenotypingPBMCs from a healthy donor carrying a heterozygous SNP in JAK1 (rs2230587) were isolated by Ficoll gradient. Cells were stained with antibodies against CD3, CD19, CD14 and CD56 (Biolegend), and 100 single cells were FACS-sorted into single cell lysis buffer (Ambion 4458235). Following DNase treatment, cDNA was generated using SuperScript VILO RT kit (ThermoFisher 11754050). A linear preamplification was then performed using JAK1 primers (gtcctctggatctcttcatgca, gctgtttggcaactttgaatttcc) and primers to a negative control gene NACA (cccaggcaaccacacaac, ccgactctgttttgctttactgact). Using qPCR with custom TaqMan genotyping primers (above) and allele-specific probes (JAK1 aaggacatc[g/a]cttttc; agcagctgaaat[T/C]gatgaa) that were individually fluorescently tagged (VIC and FAM), allelic ratios were determined by endpoint genotyping. Quantification was carried out by interpolation from a standard curve of oligonucleotide standards.Quantification and Statistical AnalysisData are presented as the mean ± standard deviation (SD) or standard error of the mean (SEM), as indicated in the legend for each figure. Statistical parameters including the methods implemented, corrections for multiple comparisons, exact values of n, identity of replicates, definitions of center and dispersion and statistical significance are reported in the Figure Legends when necessary. Statistical testing for the analysis of single cell RNA sequencing is discussed in the corresponding section of the Method Details. All statistical tests were calculated in the R statistical framework or GraphPad PRISM.

STAR★Methods

Key Resources Table
REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesMouse anti-JAK1 Clone B-3Santa Cruz BiotechnologyCat No. sc-376996; AB_2687564Rabbit anti-JAK2 Clone D2E12Cell Signaling TechnologyCat No. 3230T; AB_10691469Rabbit anti-phospho-JAK2 Tyr1007/1008Cell Signaling TechnologyCat No 3771S; RRID:AB_330403Rabbit anti-JAK3 Clone D1H3Cell Signaling TechnologyCat No. 8827S; RRID:AB_10999548Rabbit anti-TYK2 Clone D415TCell Signaling TechnologyCat No. 14193S; RRID:AB_2798419Rabbit anti-phospho-TYK2 Tyr1054/1055Cell Signaling TechnologyCat No. 9321; RRID:AB_2303972Mouse anti-phospho-tyrosine Clone 4G10Millipore SigmaCat No. 05-321XMouse anti-STAT1 Clone C-111Santa Cruz BiotechnologyCat No. sc417; RRID:AB_675902Rabbit anti-STAT2Millipore SigmaCat No. 06502; RRID:AB_31014Rabbit anti-phospho-Tyr 701-STAT1 Clone 58D6Cell Signaling TechnologyCat No. 9167; RRID:AB_561284Rabbit anti-phospho-Tyr-689-STAT2 Clone D3P2PCell Signaling TechnologyCat No. 88410; RRID:AB_2800123Rabbit anti-phospho-STAT3 Tyr705 Clone D3A7Cell Signaling TechnologyCat No. 9145; RRID:AB_2491009Rabbit anti-phospho-STAT5 Tyr694 Clone C11C5Cell Signaling TechnologyCat No. 9359S; RRID:AB_823649Rabbit anti-phospho-STAT6 Tyr64Cell Signaling TechnologyCat No. 9361T; RRID:AB_331595Rabbit anti-β-actin Clone 13E5Cell Signaling TechnologyCat No. 4970; RRID:AB_2223172Mouse anti-GAPDH Clone D16H11Cell Signaling TechnologyCat No. 5174; RRID:AB_10622025Goat anti-mouse IgG HRP-conjugatedSouthern BiotechCat No. 101005; RRID:AB_2687483Goat anti-rabbit IgG HRP-conjugatedSouthern BiotechCat No. 403005; RRID:AB_2687483Rabbit anti-IFNAR2 MMHAR-2PBLCat No. 21385-1; RRID:AB_387828Discovery OmniMap anti-rabbit HRP (RUO)RocheCat No. 760-4311; RRID:AB_2811043anti-IgD 141Pr-conjugated Clone IA6-02BiolegendCat No.348202; RRID:AB_10550095anti-CD19 142Nd-conjugated Clone HIB19BiolegendCat No.302202; RRID:AB_266181anti-CD45RA 143Nd-conjugated Clone HI100BiolegendCat No.304102; RRID:AB_314406anti-CD141 144Nd-conjugated Clone M80BiolegendCat No.344102; RRID:AB_2661788anti-CD4 145Nd-conjugated Clone RPA-T4BiolegendCat No.300502; RRID:AB_314070anti-CD8 146Nd-conjugated Clone RPA-T8BiolegendCat No.301002; RRID:AB_2661818anti-CD20 147Sm-conjugated Clone 2H7BiolegendCat No.302302; RRID:AB_314250anti-CD16 148Nd-conjugated Clone 3G8BiolegendCat No.302014; RRID:AB_314214anti-CD127 149Sm-conjugated Clone A019D5FluidigmCat No.3149011B; RRID:AB_2661792anti-CD1c 150Nd-conjugated Clone L161BiolegendCat No.331502; RRID:AB_2661820anti-CD123 151Eu-conjugated Clone 6H6BiolegendCat No.306002; RRID:AB_2661822anti-CD66b 152Sm-conjugated Clone G10F5BiolegendCat No.305102; RRID:AB_2661823anti-PD1 153Eu-conjugated Clone EH12.2H7BiolegendCat No.329926; RRID:AB_11147365anti-CD86 154Sm-conjugated Clone IT2.2BiolegendCat No.305410; RRID:AB_314530anti-CD27 155Gd-conjugated Clone O323BiolegendCat No.302802; RRID:AB_2661825anti-PDL1 156Gd-conjugated Clone 29E.2A3BiolegendCat No.329710; RRID:AB_2275581anti-CD33 158Gd-conjugated Clone WM53BiolegendCat No.303402; RRID:AB_314346anti-CD24 159Tb-conjugated Clone ML5BiolegendCat No.311102; RRID:AB_314851anti-CD14 160Gd-conjugated Clone M5E2BiolegendCat No.301810; RRID:AB_31419anti-CD56 161Dy-conjugated Clone B159BD BiosciencesCat No.555513; RRID:AB_395903anti-CD169 162Dy-conjugated Clone 7-239BiolegendCat No. 346002; RRID:AB_2189031anti-CXCR5 163Dy-conjugated Clone REA103MiltenyiCat No.130-122-325; RRID:AB_2801905anti-CD69 164Dy-conjugated Clone FN50BiolegendCat No.310902; RRID:AB_314837anti-CCR6 165Ho-conjugated Clone G034E3BiolegendCat No.353402; RRID:AB_10918625anti-CD25 166Er-conjugated Clone M-A251BiolegendCat No.356102; RRID:AB_2661833anti-CCR7 167Er-conjugated Clone G043H7BiolegendCat No.353256; RRID:AB_2814291anti-CD3 168Er-conjugated Clone UCHT1BiolegendCat No.300402; RRID:AB_2661835anti-CX3CR1 169Tm-conjugated Clone 2A9-1BiolegendCat No.341602; RRID:AB_1595422anti-CD38 170Er-conjugated Clone HB-7BiolegendCat No.356602; RRID:AB_2661836anti-CD161 171Yb-conjugated Clone HP-3G10BiolegendCat No.339902; RRID:AB_2661837anti-CD209 172Yb-conjugated Clone 9E9A8BiolegendCat No.330102; RRID:AB_1134253anti-CXCR3 173Yb-conjugated Clone REA232MiltenyiCat No.130-108-022; RRID:AB_2655743anti-HLADR 174Yb-conjugated Clone L243BiolegendCat No.307602; RRID:AB_314680anti-Axl 175Lu-conjugated Clone 108724R&D SystemsCat No.MAB154; RRID:AB_2062558anti-CCR4 176Yb-conjugated Clone 205410R&D SystemsCat No.MAB1567; RRID:AB_2074395anti-pSTAT5 147 Sm-conjugated Clone 47FluidigmCat No.3147012A; RRID:AB_2827887anti-pSTAT6 149 Sm-conjugated Clone 18FluidigmCat No.3149004Aanti-pSTAT1 153 Eu-conjugated Clone 4aFluidigmCat No.3153005A; RRID:AB_2744689anti-pp38 156 Gd-conjugated Clone D3F9FluidigmCat No.3156002A; RRID:AB_2661826anti-pSTAT3 158 Gd-conjugated Clone 4FluidigmCat No.3158005A; RRID:AB_2811100anti-pMAPKAP2 159 Tb-conjugated Clone 27B7FluidigmCat No.3159010A; RRID:AB_2661828anti-STAT3 165 Ho-conjugated Clone 124H6FluidigmCat No.3173003Aanti-STAT1 169 Tm-conjugated Clone 10C4B40BiolegendCat No.661002; RRID:AB_2563664anti-pERK 171 Yb-conjugated Clone D13.14.4EFluidigmCat No.3171010A; RRID:AB_2811250anti-pS6 175 Lu-conjugated Clone N7-548FluidigmCat No.3175009A; RRID:AB_2811251Mouse anti-NKp46 Pacific Blue Clone 900BioLegendCat No. 331912; RRID:AB_2149280Mouse anti-CD56 Brilliant Violet 605 Clone HCD56BioLegendCat No. 318334; RRID:AB_2561912Mouse anti-CD16 Brilliant Violet 650 Clone 3G8BioLegendCat No. 302041; RRID:AB_11125578Mouse anti-CD3 Brilliant Violet 711 Clone OKT3BioLegendCat No. 317328; RRID:AB_2562907Mouse anti-CD8-alpha Brilliant Violet 785 Clone RPA-T8BioLegendCat No. 301046; RRID:AB_2563264Mouse anti-CD107a FITC Clone eBioH4A3eBioscienceCat No.53-1079-42; RRID:AB_2016657Mouse anti-NKp44 PE Clone Z231Beckman CoulterCat No. IM3710; RRID:AB_ 2857937Mouse anti-NKG2D PE-Cy7 Clone 1D11BioLegendCat No. 320811; RRID:AB_2133275Mouse anti-CD69 PE-CF594 Clone FN50BD BiosciencesCat No. 562617; RRID:AB_2737680Mouse anti-NKp30 APC Clone P30-15BioLegendCat No. 325209; RRID:AB_2149450Mouse anti-CD16 APC Cy7 Clone B73.1BD BiosciencesCat No. 561306; RRID:AB_10643005Mouse anti-CD25 APC-Alexa Fluor 700 Clone B1.49.9Beckman CoulterCat No. A86356;Rat anti-CD2 Pacific Blue Clone 39C1.5Beckman CoulterCat No. B09685; RRID:AB_2847880Mouse anti-CD244 FITC Clone 25235BD BiosciencesCat No. 550815; RRID:AB_393900Mouse anti-CD11c PerCP Cy5.5 Clone BU15Beckman CoulterCat No. B19719;Mouse anti-CD28 PE Clone L293BD BiosciencesCat No. 348047; RRID:AB_400368Mouse anti-CD11a PE-Cy7 Clone HI111BD BiosciencesCat No. 561387; RRID:AB_10611572Mouse anti-CD54 PE-Cy5 Clone HA58BD BiosciencesCat No. 555512; RRID:AB_395902Mouse anti-CD11b PE-CF594 Clone ICRF44BD BiosciencesCat No. 562399; RRID:AB_2737613Mouse anti-CD18 APC Clone 6.7BD BiosciencesCat No. 551060; RRID:AB_398485Mouse anti-CD158e Brilliant Violet 421 Clone DX9BioLegendCat No. 312714; RRID:AB_312714Mouse anti-CD158b FITC Clone DX27BioLegendCat No. 312604; RRID:AB_2296486Mouse anti-CD94 PerCP Cy5.5 Clone HP-3D9BD BiosciencesCat No. 562361; RRID:AB_11152081Mouse anti-NKG2C PE Clone 134591R&D SystemsCat No. FAB138P ; RRID:AB_2132983Mouse anti-CD158a/h/g PE-Cy7 Clone HP-MA4Affymetrix eBioscienceCat No. 25-1589-42; RRID:AB_10854424Mouse anti-KIR2DS4 APC Clone 179315R&D SystemsCat No. FAB1847A; RRID:AB_2130821Recombinant anti-KLRG1 APC-Vio770 Clone REA226Miltenyi BiotecCat No. 130103642; RRID:AB_2652580Mouse anti-NKG2A Alexa Fluor 700 Clone 131411R&D SystemsCat No. FAB1059N; RRID:AB_10972129Mouse anti-CD57 Pacific Blue Clone NC1Beckman CoulterCat No. A74779 ; RRID:AB_131600Mouse anti-CD62L Brilliant Violet 650 Clone DREG-56BioLegendCat No. 304831; RRID:AB_2561461Mouse anti-CD127 Brilliant Violet 785 Clone A019D5BioLegendCat No. 351330; RRID:AB_2563605Mouse anti-IL-15R-alpha Alexa Fluor 488 Clone 151303R&D SystemsCat No. FAB1471G; RRID:AB_10891330Mouse anti-CD117 PE-Cy7 Clone 104D2D1Beckman CoulterCat No. IM3698; RRID:AB_131184Mouse anti-CD16 PE-CF594 Clone 3G8BD BiosciencesCat No. 562293; RRID:AB_11151916Mouse anti-CD94 APC Clone DX22BioLegendCat No. 305508; RRID:AB_2133129Mouse anti-Perforin Briliant Violet 421 Clone B-D48BioLegendCat No. 353307; RRID:AB_ 11149688Mouse anti-IFN-gamma Alexa Fluor 700 Clone 4S.B3BioLegendCat No. 502520; RRID:AB_528921Mouse anti-Eomes eFluor 660 Clone WD1928eBioscienceCat No. 50-4877-42; RRID:AB_2574229Mouse anti-CD34 Alexa Fluor 700 Clone 581BioLegendCat No. 343526; RRID:AB_2561495Mouse anti-Perforin APC Cy7 Clone dG9BioLegendCat No. 308128; RRID:AB_2572051Mouse anti-Helios Brilliant Violet 421 Clone 22F6BioLegendCat No. 137233; RRID:AB_2565798Mouse anti-T-bet FITC Clone 4B10BioLegendCat No. 644812; RRID:AB_2200540Mouse anti-IRF8 PercP- eFluor 710 Clone V3GYWCHeBioscienceCat No. 46-9852-80; RRID:AB_2573903Mouse anti-GATA3 PE Cy7 Clone L50-823BD BiosciencesCat No. 560405; RRID:AB_1645544Mouse anti-NKp44 PE Clone P44-8BioLegendCat No.325107; RRID:AB_756099Goat anti-mouse IgG Alexa Fluor 647Thermo FisherCat No. A-21235; RRID:AB_2535804Bacterial and Virus StrainsDH5-Alpha Competent E. ColiMolecular Cloning LaboratoriesCat No. DA-196Biological SamplesHuman whole blood samplesVarious institutionsN/AChemicals, Peptides, and Recombinant ProteinsIntron-A Recombinant Interferon Alpha-2bMerck PharmaceuticalsCat No. NDC0085057102Recombinant Human IL-2BioLegendCat No. 589102Recombinant Human IL-4BioLegendCat No. 574002Recombinant Human IL-5BioLegendCat No. 560701Recombinant Human IL-6BioLegendCat No. 570802Recombinant Human IL-13BioLegendCat No. 571102Recombinant Human IL-21BioLegendCat No. 571202Recombinant Human Interferon GammaBioLegendCat No. 570206phorbol 12-myristate 13-acetateSigma-AldrichCat No. P1585IonomycinSigma-AldrichCat No. I0634Brefeldin ASigma-AldrichCat No. B7651RuxolitinibSelleckchemCat No. S1378TofacitinibSelleckchemCat No. S5001FilgotinibSelleckchemCat No. S7605Proteomic Stabilizer Prot1SMART TUBE IncCat No. 501351691HeparinSigmaCat No. 201060Osmium tetroxide (99.9%)ACROS organicsCat No. 191180010Cell-ID 20-plex Pd Barcoding KitFluidigmCat No. 201060Human TruStain FcX (Fc Receptor Blocking Solution)BioLegendCat No. 422301X8 MaxPar conjugation kitsFluidigmCat No. 201300125nM Ir IntercalatorFluidigmCat No. 201192AOsO4ACROS OrganicsCat No. AC319010050Maxpar Cell Acquisition BufferFluidigmCat No. 201241Cytofix/Cytoperm Fixation/Permeabilization SolutionBDCat No. 554714FoxP3 Transcription Factor Staining BufferTonboCat No. TNB0607Live/Dead AquaThermo FisherCat No. L34957Discovery Ultra antibody blockRocheCat No. 760-4204Pierce ECL Western Blotting SubstrateThermo Fisher ScientificCat No. 32106RIPA Lysis and Extraction BufferThermo Fisher ScientificCat No. 89900Protease/Phosphatase Inhibitor CocktailCell Signaling TechnologiesCat No. 5872Nupage Western Blot Sample BufferThermo Fisher ScientificCat No. NP0007Dynabeads Protein GThermo Fisher ScientificCat No. 10007DHistopaque 1077Millipore SigmaCat No. 10771-500CyclosporinSigma AldrichCat No. C3662OptiPrep Density Gradient MediumSigma AldrichCat Nol. D1556-250mLNovec 7500 Engineered Fluid3MCat No. Novec 75001H,1H, 2H, 2H – Perfluorooctanol, 97%Alfa Aesar, Thermo Fisher ScientificCAS No. 647-42-750 g of 10 weight % 008-Flurorosurfactant in HFE7500Ran Biotechnologies008-FlurorSurfactant-10wtH-50GExonuclease I (E. Coli)New England BiolabsM0293LFast Digest HinfIThermo Fisher ScientificFD0804AMPure XPBeckman CoulterA63881RNAClean XPBeckman CoulterA66514RNaseHNew England BiolabsM0297LCritical Commercial AssaysQuikChange II XL site-directed mutagenesis kitAgilent TechnologiesCat No. 200522Gateway LR Clonase II Enyme MixThermo Fisher ScientificCat No. 11791100Gateway BP Clonase Enzyme MixThermo Fisher ScientificCat No. 11789020QIAamp DNA Mini KitQIAGENCat No. 51304RNeasy RNA Isolation KitQIAGENCat No. 74106Applied Biosystems High-Capacity cDNA Reverse Transcription KitThermo Fisher ScientificCat No. 4368814TaqMan Universal Master Mix II with UNGThermo Fisher ScientificCat No. 4440039Human Magnetic Luminex Assay CustomR&DCat No. LXSAHMBio-Plex Pro Human Inflammation PanelBioRadCat No. 171al001mCell-ID 20-Plex Pd Barcoding KitFluidigmCat No. 201060Chromium single cell Chip Kit V210X GenomicsCat No. 120236Chromium single cell 3′ Library and Gel Bead Kit10X GenomicsCat No.120237Discovery ChromoMap DAB KitRocheCat No. 760-159MycoAlert PLUS Mycoplasma Detection KitLonzaCat No. LT07-703Lipofectamine 3000 Transfection ReagentThermo FisherCat No. L3000001Invitrogen Taq polymeraseThermo FisherCat No. 10342020Digital droplet PCR SupermixBioradCat No. 1863026IgG Autoantibody ArrayRayBiotechCat No. PAH-AIDG-G1-16)Single Cell Lysis BufferAmbionCat No. 4458235SuperScript VILO RT kitThermo FisherCat No. 11754050SuperScript™ III Reverse TranscriptaseThermo Fisher ScientificCat No. 18080093HiScribe T7 High Yield RNA Synthesis KitNew England BiolabsCat No. E2040SKapa HiFi HotStart ReadyMix PCRocheKR0370NextSeq 500/550 High Output Kit v2.5 (75 cycles)Illumina20024906Deposited DatascRNA-seq Data and AnalysisThis paperhttps://github.com/jorgcalis/JAK1-alleles-pipelineExperimental Models: Cell LinesHEK293TATCCCRL-3216OP9ATCCCRL-2749U4C (JAK1−/−)Sandra PellegriniN/Aγ2A (JAK2−/−)Sandra PellegriniN/AEBV-immortalized B cellsIcahn School of Medicine at Mount SinaiN/AOligonucleotidesMX1 mRNA TaqMan FAM Hs200895608_m1Thermo FisherCat No. 4331182RSAD2 mRNA TaqMan FAM Hs00369813_m1Thermo FisherCat No. 4351370scRNaseq polyT oligonucleotide template 5′-BAAAAAAAAAAAAAAAAAAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG – 3′Integrated DNA TechnologiesN/AscRNaseq JAK1 oligonucleotide template 5′ - GAGTGTGGCCCATTCATCAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG −3′Integrated DNA TechnologiesN/AscRNaseq JAK1 blocking oligonucleotide 5′-GAGTGTGGCCCATTCATCAA-3′Integrated DNA TechnologiesN/AscRNaseq second reverse transcription JAK1 primer: 5′- TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGCCATGGAAATTCAAAGTTGCCAAACAG-3′Integrated DNA TechnologiesN/AJAK1 single cell qPCR amplification (F: GTCCTCTGGATCTCTTCATGCA, R: GCTGTTTGGCAACTTTGAATTTCC)Thermo FisherN/AJAK1 single cell genotyping probes (AAGGACATC[G/A]CTTTTC)Thermo FisherN/ARecombinant DNApTRP IRES RFP PuroRThis paperN/ApDONR223 JAK1AddGeneCat No. 23932pDONR223 JAK2AddGeneCat No. 23915pCAGGS-VSV-GThis paperN/ApCMV-Gag/PolThis paperN/ASoftware and AlgorithmsCytobankBeckman Coulterhttps://www.cytobank.orgFlowJoBecton Dickinson Companyhttps://www.flowjo.comR for Statistical Computinghttps://www.r-project.orgRStudioThe R Foundationhttps://rstudio.comCellRanger10X Genomicshttps://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-rangerPython v2.7Pythonhttps://www.python.org/Seurat v2.3.4 R toolkitSatija Lab, New York Genome Centerhttps://satijalab.org/seurat/EdgeR v3.12.1Bioconductorhttp://bioinf.wehi.edu.au/edgeR/inDrops.py v0.3GitHubhttps://github.com/indrops/indropsGenome Analysis Toolkit (GATK)Broad Institutehttps://gatk.broadinstitute.org/hc/en-usIngenuity Variant AnalysisQIAGENhttps://variants.ingenuity.comGraphPad Prism 7GraphPad Softwarehttps://www.graphpad.com/scientific-software/prism/QX100 softwareBioRadN/AGenepix Pro 7.0 softwareMolecular DevicesN/A

Key Resources Table

REAGENT or RESOURCESOURCEIDENTIFIERAntibodiesMouse anti-JAK1 Clone B-3Santa Cruz BiotechnologyCat No. sc-376996; AB_2687564Rabbit anti-JAK2 Clone D2E12Cell Signaling TechnologyCat No. 3230T; AB_10691469Rabbit anti-phospho-JAK2 Tyr1007/1008Cell Signaling TechnologyCat No 3771S; RRID:AB_330403Rabbit anti-JAK3 Clone D1H3Cell Signaling TechnologyCat No. 8827S; RRID:AB_10999548Rabbit anti-TYK2 Clone D415TCell Signaling TechnologyCat No. 14193S; RRID:AB_2798419Rabbit anti-phospho-TYK2 Tyr1054/1055Cell Signaling TechnologyCat No. 9321; RRID:AB_2303972Mouse anti-phospho-tyrosine Clone 4G10Millipore SigmaCat No. 05-321XMouse anti-STAT1 Clone C-111Santa Cruz BiotechnologyCat No. sc417; RRID:AB_675902Rabbit anti-STAT2Millipore SigmaCat No. 06502; RRID:AB_31014Rabbit anti-phospho-Tyr 701-STAT1 Clone 58D6Cell Signaling TechnologyCat No. 9167; RRID:AB_561284Rabbit anti-phospho-Tyr-689-STAT2 Clone D3P2PCell Signaling TechnologyCat No. 88410; RRID:AB_2800123Rabbit anti-phospho-STAT3 Tyr705 Clone D3A7Cell Signaling TechnologyCat No. 9145; RRID:AB_2491009Rabbit anti-phospho-STAT5 Tyr694 Clone C11C5Cell Signaling TechnologyCat No. 9359S; RRID:AB_823649Rabbit anti-phospho-STAT6 Tyr64Cell Signaling TechnologyCat No. 9361T; RRID:AB_331595Rabbit anti-β-actin Clone 13E5Cell Signaling TechnologyCat No. 4970; RRID:AB_2223172Mouse anti-GAPDH Clone D16H11Cell Signaling TechnologyCat No. 5174; RRID:AB_10622025Goat anti-mouse IgG HRP-conjugatedSouthern BiotechCat No. 101005; RRID:AB_2687483Goat anti-rabbit IgG HRP-conjugatedSouthern BiotechCat No. 403005; RRID:AB_2687483Rabbit anti-IFNAR2 MMHAR-2PBLCat No. 21385-1; RRID:AB_387828Discovery OmniMap anti-rabbit HRP (RUO)RocheCat No. 760-4311; RRID:AB_2811043anti-IgD 141Pr-conjugated Clone IA6-02BiolegendCat No.348202; RRID:AB_10550095anti-CD19 142Nd-conjugated Clone HIB19BiolegendCat No.302202; RRID:AB_266181anti-CD45RA 143Nd-conjugated Clone HI100BiolegendCat No.304102; RRID:AB_314406anti-CD141 144Nd-conjugated Clone M80BiolegendCat No.344102; RRID:AB_2661788anti-CD4 145Nd-conjugated Clone RPA-T4BiolegendCat No.300502; RRID:AB_314070anti-CD8 146Nd-conjugated Clone RPA-T8BiolegendCat No.301002; RRID:AB_2661818anti-CD20 147Sm-conjugated Clone 2H7BiolegendCat No.302302; RRID:AB_314250anti-CD16 148Nd-conjugated Clone 3G8BiolegendCat No.302014; RRID:AB_314214anti-CD127 149Sm-conjugated Clone A019D5FluidigmCat No.3149011B; RRID:AB_2661792anti-CD1c 150Nd-conjugated Clone L161BiolegendCat No.331502; RRID:AB_2661820anti-CD123 151Eu-conjugated Clone 6H6BiolegendCat No.306002; RRID:AB_2661822anti-CD66b 152Sm-conjugated Clone G10F5BiolegendCat No.305102; RRID:AB_2661823anti-PD1 153Eu-conjugated Clone EH12.2H7BiolegendCat No.329926; RRID:AB_11147365anti-CD86 154Sm-conjugated Clone IT2.2BiolegendCat No.305410; RRID:AB_314530anti-CD27 155Gd-conjugated Clone O323BiolegendCat No.302802; RRID:AB_2661825anti-PDL1 156Gd-conjugated Clone 29E.2A3BiolegendCat No.329710; RRID:AB_2275581anti-CD33 158Gd-conjugated Clone WM53BiolegendCat No.303402; RRID:AB_314346anti-CD24 159Tb-conjugated Clone ML5BiolegendCat No.311102; RRID:AB_314851anti-CD14 160Gd-conjugated Clone M5E2BiolegendCat No.301810; RRID:AB_31419anti-CD56 161Dy-conjugated Clone B159BD BiosciencesCat No.555513; RRID:AB_395903anti-CD169 162Dy-conjugated Clone 7-239BiolegendCat No. 346002; RRID:AB_2189031anti-CXCR5 163Dy-conjugated Clone REA103MiltenyiCat No.130-122-325; RRID:AB_2801905anti-CD69 164Dy-conjugated Clone FN50BiolegendCat No.310902; RRID:AB_314837anti-CCR6 165Ho-conjugated Clone G034E3BiolegendCat No.353402; RRID:AB_10918625anti-CD25 166Er-conjugated Clone M-A251BiolegendCat No.356102; RRID:AB_2661833anti-CCR7 167Er-conjugated Clone G043H7BiolegendCat No.353256; RRID:AB_2814291anti-CD3 168Er-conjugated Clone UCHT1BiolegendCat No.300402; RRID:AB_2661835anti-CX3CR1 169Tm-conjugated Clone 2A9-1BiolegendCat No.341602; RRID:AB_1595422anti-CD38 170Er-conjugated Clone HB-7BiolegendCat No.356602; RRID:AB_2661836anti-CD161 171Yb-conjugated Clone HP-3G10BiolegendCat No.339902; RRID:AB_2661837anti-CD209 172Yb-conjugated Clone 9E9A8BiolegendCat No.330102; RRID:AB_1134253anti-CXCR3 173Yb-conjugated Clone REA232MiltenyiCat No.130-108-022; RRID:AB_2655743anti-HLADR 174Yb-conjugated Clone L243BiolegendCat No.307602; RRID:AB_314680anti-Axl 175Lu-conjugated Clone 108724R&D SystemsCat No.MAB154; RRID:AB_2062558anti-CCR4 176Yb-conjugated Clone 205410R&D SystemsCat No.MAB1567; RRID:AB_2074395anti-pSTAT5 147 Sm-conjugated Clone 47FluidigmCat No.3147012A; RRID:AB_2827887anti-pSTAT6 149 Sm-conjugated Clone 18FluidigmCat No.3149004Aanti-pSTAT1 153 Eu-conjugated Clone 4aFluidigmCat No.3153005A; RRID:AB_2744689anti-pp38 156 Gd-conjugated Clone D3F9FluidigmCat No.3156002A; RRID:AB_2661826anti-pSTAT3 158 Gd-conjugated Clone 4FluidigmCat No.3158005A; RRID:AB_2811100anti-pMAPKAP2 159 Tb-conjugated Clone 27B7FluidigmCat No.3159010A; RRID:AB_2661828anti-STAT3 165 Ho-conjugated Clone 124H6FluidigmCat No.3173003Aanti-STAT1 169 Tm-conjugated Clone 10C4B40BiolegendCat No.661002; RRID:AB_2563664anti-pERK 171 Yb-conjugated Clone D13.14.4EFluidigmCat No.3171010A; RRID:AB_2811250anti-pS6 175 Lu-conjugated Clone N7-548FluidigmCat No.3175009A; RRID:AB_2811251Mouse anti-NKp46 Pacific Blue Clone 900BioLegendCat No. 331912; RRID:AB_2149280Mouse anti-CD56 Brilliant Violet 605 Clone HCD56BioLegendCat No. 318334; RRID:AB_2561912Mouse anti-CD16 Brilliant Violet 650 Clone 3G8BioLegendCat No. 302041; RRID:AB_11125578Mouse anti-CD3 Brilliant Violet 711 Clone OKT3BioLegendCat No. 317328; RRID:AB_2562907Mouse anti-CD8-alpha Brilliant Violet 785 Clone RPA-T8BioLegendCat No. 301046; RRID:AB_2563264Mouse anti-CD107a FITC Clone eBioH4A3eBioscienceCat No.53-1079-42; RRID:AB_2016657Mouse anti-NKp44 PE Clone Z231Beckman CoulterCat No. IM3710; RRID:AB_ 2857937Mouse anti-NKG2D PE-Cy7 Clone 1D11BioLegendCat No. 320811; RRID:AB_2133275Mouse anti-CD69 PE-CF594 Clone FN50BD BiosciencesCat No. 562617; RRID:AB_2737680Mouse anti-NKp30 APC Clone P30-15BioLegendCat No. 325209; RRID:AB_2149450Mouse anti-CD16 APC Cy7 Clone B73.1BD BiosciencesCat No. 561306; RRID:AB_10643005Mouse anti-CD25 APC-Alexa Fluor 700 Clone B1.49.9Beckman CoulterCat No. A86356;Rat anti-CD2 Pacific Blue Clone 39C1.5Beckman CoulterCat No. B09685; RRID:AB_2847880Mouse anti-CD244 FITC Clone 25235BD BiosciencesCat No. 550815; RRID:AB_393900Mouse anti-CD11c PerCP Cy5.5 Clone BU15Beckman CoulterCat No. B19719;Mouse anti-CD28 PE Clone L293BD BiosciencesCat No. 348047; RRID:AB_400368Mouse anti-CD11a PE-Cy7 Clone HI111BD BiosciencesCat No. 561387; RRID:AB_10611572Mouse anti-CD54 PE-Cy5 Clone HA58BD BiosciencesCat No. 555512; RRID:AB_395902Mouse anti-CD11b PE-CF594 Clone ICRF44BD BiosciencesCat No. 562399; RRID:AB_2737613Mouse anti-CD18 APC Clone 6.7BD BiosciencesCat No. 551060; RRID:AB_398485Mouse anti-CD158e Brilliant Violet 421 Clone DX9BioLegendCat No. 312714; RRID:AB_312714Mouse anti-CD158b FITC Clone DX27BioLegendCat No. 312604; RRID:AB_2296486Mouse anti-CD94 PerCP Cy5.5 Clone HP-3D9BD BiosciencesCat No. 562361; RRID:AB_11152081Mouse anti-NKG2C PE Clone 134591R&D SystemsCat No. FAB138P ; RRID:AB_2132983Mouse anti-CD158a/h/g PE-Cy7 Clone HP-MA4Affymetrix eBioscienceCat No. 25-1589-42; RRID:AB_10854424Mouse anti-KIR2DS4 APC Clone 179315R&D SystemsCat No. FAB1847A; RRID:AB_2130821Recombinant anti-KLRG1 APC-Vio770 Clone REA226Miltenyi BiotecCat No. 130103642; RRID:AB_2652580Mouse anti-NKG2A Alexa Fluor 700 Clone 131411R&D SystemsCat No. FAB1059N; RRID:AB_10972129Mouse anti-CD57 Pacific Blue Clone NC1Beckman CoulterCat No. A74779 ; RRID:AB_131600Mouse anti-CD62L Brilliant Violet 650 Clone DREG-56BioLegendCat No. 304831; RRID:AB_2561461Mouse anti-CD127 Brilliant Violet 785 Clone A019D5BioLegendCat No. 351330; RRID:AB_2563605Mouse anti-IL-15R-alpha Alexa Fluor 488 Clone 151303R&D SystemsCat No. FAB1471G; RRID:AB_10891330Mouse anti-CD117 PE-Cy7 Clone 104D2D1Beckman CoulterCat No. IM3698; RRID:AB_131184Mouse anti-CD16 PE-CF594 Clone 3G8BD BiosciencesCat No. 562293; RRID:AB_11151916Mouse anti-CD94 APC Clone DX22BioLegendCat No. 305508; RRID:AB_2133129Mouse anti-Perforin Briliant Violet 421 Clone B-D48BioLegendCat No. 353307; RRID:AB_ 11149688Mouse anti-IFN-gamma Alexa Fluor 700 Clone 4S.B3BioLegendCat No. 502520; RRID:AB_528921Mouse anti-Eomes eFluor 660 Clone WD1928eBioscienceCat No. 50-4877-42; RRID:AB_2574229Mouse anti-CD34 Alexa Fluor 700 Clone 581BioLegendCat No. 343526; RRID:AB_2561495Mouse anti-Perforin APC Cy7 Clone dG9BioLegendCat No. 308128; RRID:AB_2572051Mouse anti-Helios Brilliant Violet 421 Clone 22F6BioLegendCat No. 137233; RRID:AB_2565798Mouse anti-T-bet FITC Clone 4B10BioLegendCat No. 644812; RRID:AB_2200540Mouse anti-IRF8 PercP- eFluor 710 Clone V3GYWCHeBioscienceCat No. 46-9852-80; RRID:AB_2573903Mouse anti-GATA3 PE Cy7 Clone L50-823BD BiosciencesCat No. 560405; RRID:AB_1645544Mouse anti-NKp44 PE Clone P44-8BioLegendCat No.325107; RRID:AB_756099Goat anti-mouse IgG Alexa Fluor 647Thermo FisherCat No. A-21235; RRID:AB_2535804Bacterial and Virus StrainsDH5-Alpha Competent E. ColiMolecular Cloning LaboratoriesCat No. DA-196Biological SamplesHuman whole blood samplesVarious institutionsN/AChemicals, Peptides, and Recombinant ProteinsIntron-A Recombinant Interferon Alpha-2bMerck PharmaceuticalsCat No. NDC0085057102Recombinant Human IL-2BioLegendCat No. 589102Recombinant Human IL-4BioLegendCat No. 574002Recombinant Human IL-5BioLegendCat No. 560701Recombinant Human IL-6BioLegendCat No. 570802Recombinant Human IL-13BioLegendCat No. 571102Recombinant Human IL-21BioLegendCat No. 571202Recombinant Human Interferon GammaBioLegendCat No. 570206phorbol 12-myristate 13-acetateSigma-AldrichCat No. P1585IonomycinSigma-AldrichCat No. I0634Brefeldin ASigma-AldrichCat No. B7651RuxolitinibSelleckchemCat No. S1378TofacitinibSelleckchemCat No. S5001FilgotinibSelleckchemCat No. S7605Proteomic Stabilizer Prot1SMART TUBE IncCat No. 501351691HeparinSigmaCat No. 201060Osmium tetroxide (99.9%)ACROS organicsCat No. 191180010Cell-ID 20-plex Pd Barcoding KitFluidigmCat No. 201060Human TruStain FcX (Fc Receptor Blocking Solution)BioLegendCat No. 422301X8 MaxPar conjugation kitsFluidigmCat No. 201300125nM Ir IntercalatorFluidigmCat No. 201192AOsO4ACROS OrganicsCat No. AC319010050Maxpar Cell Acquisition BufferFluidigmCat No. 201241Cytofix/Cytoperm Fixation/Permeabilization SolutionBDCat No. 554714FoxP3 Transcription Factor Staining BufferTonboCat No. TNB0607Live/Dead AquaThermo FisherCat No. L34957Discovery Ultra antibody blockRocheCat No. 760-4204Pierce ECL Western Blotting SubstrateThermo Fisher ScientificCat No. 32106RIPA Lysis and Extraction BufferThermo Fisher ScientificCat No. 89900Protease/Phosphatase Inhibitor CocktailCell Signaling TechnologiesCat No. 5872Nupage Western Blot Sample BufferThermo Fisher ScientificCat No. NP0007Dynabeads Protein GThermo Fisher ScientificCat No. 10007DHistopaque 1077Millipore SigmaCat No. 10771-500CyclosporinSigma AldrichCat No. C3662OptiPrep Density Gradient MediumSigma AldrichCat Nol. D1556-250mLNovec 7500 Engineered Fluid3MCat No. Novec 75001H,1H, 2H, 2H – Perfluorooctanol, 97%Alfa Aesar, Thermo Fisher ScientificCAS No. 647-42-750 g of 10 weight % 008-Flurorosurfactant in HFE7500Ran Biotechnologies008-FlurorSurfactant-10wtH-50GExonuclease I (E. Coli)New England BiolabsM0293LFast Digest HinfIThermo Fisher ScientificFD0804AMPure XPBeckman CoulterA63881RNAClean XPBeckman CoulterA66514RNaseHNew England BiolabsM0297LCritical Commercial AssaysQuikChange II XL site-directed mutagenesis kitAgilent TechnologiesCat No. 200522Gateway LR Clonase II Enyme MixThermo Fisher ScientificCat No. 11791100Gateway BP Clonase Enzyme MixThermo Fisher ScientificCat No. 11789020QIAamp DNA Mini KitQIAGENCat No. 51304RNeasy RNA Isolation KitQIAGENCat No. 74106Applied Biosystems High-Capacity cDNA Reverse Transcription KitThermo Fisher ScientificCat No. 4368814TaqMan Universal Master Mix II with UNGThermo Fisher ScientificCat No. 4440039Human Magnetic Luminex Assay CustomR&DCat No. LXSAHMBio-Plex Pro Human Inflammation PanelBioRadCat No. 171al001mCell-ID 20-Plex Pd Barcoding KitFluidigmCat No. 201060Chromium single cell Chip Kit V210X GenomicsCat No. 120236Chromium single cell 3′ Library and Gel Bead Kit10X GenomicsCat No.120237Discovery ChromoMap DAB KitRocheCat No. 760-159MycoAlert PLUS Mycoplasma Detection KitLonzaCat No. LT07-703Lipofectamine 3000 Transfection ReagentThermo FisherCat No. L3000001Invitrogen Taq polymeraseThermo FisherCat No. 10342020Digital droplet PCR SupermixBioradCat No. 1863026IgG Autoantibody ArrayRayBiotechCat No. PAH-AIDG-G1-16)Single Cell Lysis BufferAmbionCat No. 4458235SuperScript VILO RT kitThermo FisherCat No. 11754050SuperScript™ III Reverse TranscriptaseThermo Fisher ScientificCat No. 18080093HiScribe T7 High Yield RNA Synthesis KitNew England BiolabsCat No. E2040SKapa HiFi HotStart ReadyMix PCRocheKR0370NextSeq 500/550 High Output Kit v2.5 (75 cycles)Illumina20024906Deposited DatascRNA-seq Data and AnalysisThis paperhttps://github.com/jorgcalis/JAK1-alleles-pipelineExperimental Models: Cell LinesHEK293TATCCCRL-3216OP9ATCCCRL-2749U4C (JAK1−/−)Sandra PellegriniN/Aγ2A (JAK2−/−)Sandra PellegriniN/AEBV-immortalized B cellsIcahn School of Medicine at Mount SinaiN/AOligonucleotidesMX1 mRNA TaqMan FAM Hs200895608_m1Thermo FisherCat No. 4331182RSAD2 mRNA TaqMan FAM Hs00369813_m1Thermo FisherCat No. 4351370scRNaseq polyT oligonucleotide template 5′-BAAAAAAAAAAAAAAAAAAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG – 3′Integrated DNA TechnologiesN/AscRNaseq JAK1 oligonucleotide template 5′ - GAGTGTGGCCCATTCATCAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG −3′Integrated DNA TechnologiesN/AscRNaseq JAK1 blocking oligonucleotide 5′-GAGTGTGGCCCATTCATCAA-3′Integrated DNA TechnologiesN/AscRNaseq second reverse transcription JAK1 primer: 5′- TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGCCATGGAAATTCAAAGTTGCCAAACAG-3′Integrated DNA TechnologiesN/AJAK1 single cell qPCR amplification (F: GTCCTCTGGATCTCTTCATGCA, R: GCTGTTTGGCAACTTTGAATTTCC)Thermo FisherN/AJAK1 single cell genotyping probes (AAGGACATC[G/A]CTTTTC)Thermo FisherN/ARecombinant DNApTRP IRES RFP PuroRThis paperN/ApDONR223 JAK1AddGeneCat No. 23932pDONR223 JAK2AddGeneCat No. 23915pCAGGS-VSV-GThis paperN/ApCMV-Gag/PolThis paperN/ASoftware and AlgorithmsCytobankBeckman Coulterhttps://www.cytobank.orgFlowJoBecton Dickinson Companyhttps://www.flowjo.comR for Statistical Computinghttps://www.r-project.orgRStudioThe R Foundationhttps://rstudio.comCellRanger10X Genomicshttps://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-rangerPython v2.7Pythonhttps://www.python.org/Seurat v2.3.4 R toolkitSatija Lab, New York Genome Centerhttps://satijalab.org/seurat/EdgeR v3.12.1Bioconductorhttp://bioinf.wehi.edu.au/edgeR/inDrops.py v0.3GitHubhttps://github.com/indrops/indropsGenome Analysis Toolkit (GATK)Broad Institutehttps://gatk.broadinstitute.org/hc/en-usIngenuity Variant AnalysisQIAGENhttps://variants.ingenuity.comGraphPad Prism 7GraphPad Softwarehttps://www.graphpad.com/scientific-software/prism/QX100 softwareBioRadN/AGenepix Pro 7.0 softwareMolecular DevicesN/A

Resource AvailabilityLead ContactFurther information and requests for reagents may be directed to the Lead Contact, Dusan Bogunovic (dusan.bogunovic@mssm.edu).Materials AvailabilityAll unique/stable reagents generated in this study are available from the Lead Contact with a completed Materials Transfer Agreement.Data and Code AvailabilityThe raw data for experiments performed, including the PCR, multiplex ELISA, mass cytometry, flow cytometry, scRNaseq data, are available upon request from the lead contact. For privacy concerns of the study participants, raw scRNA-Seq data are not available, and the complete data files from whole exome sequencing will be restricted to the variants in Table S1. scRNA-Seq gene x cell matrices, python scripts and associated Seurat code (R-based) used for the tailored analysis of JAK1-specific scRNA-Seq are available at https://github.com/jorgcalis/JAK1-alleles-pipeline.

Resource Availability

Lead ContactFurther information and requests for reagents may be directed to the Lead Contact, Dusan Bogunovic (dusan.bogunovic@mssm.edu).

Lead Contact

Further information and requests for reagents may be directed to the Lead Contact, Dusan Bogunovic (dusan.bogunovic@mssm.edu).

Materials AvailabilityAll unique/stable reagents generated in this study are available from the Lead Contact with a completed Materials Transfer Agreement.

Materials Availability

All unique/stable reagents generated in this study are available from the Lead Contact with a completed Materials Transfer Agreement.

Data and Code AvailabilityThe raw data for experiments performed, including the PCR, multiplex ELISA, mass cytometry, flow cytometry, scRNaseq data, are available upon request from the lead contact. For privacy concerns of the study participants, raw scRNA-Seq data are not available, and the complete data files from whole exome sequencing will be restricted to the variants in Table S1. scRNA-Seq gene x cell matrices, python scripts and associated Seurat code (R-based) used for the tailored analysis of JAK1-specific scRNA-Seq are available at https://github.com/jorgcalis/JAK1-alleles-pipeline.

Data and Code Availability

The raw data for experiments performed, including the PCR, multiplex ELISA, mass cytometry, flow cytometry, scRNaseq data, are available upon request from the lead contact. For privacy concerns of the study participants, raw scRNA-Seq data are not available, and the complete data files from whole exome sequencing will be restricted to the variants in Table S1. scRNA-Seq gene x cell matrices, python scripts and associated Seurat code (R-based) used for the tailored analysis of JAK1-specific scRNA-Seq are available at https://github.com/jorgcalis/JAK1-alleles-pipeline.

Experimental Model and Subject DetailsPatientsThis study reports an 18-year-old female who presented during early childhood with persistent, recurrent cutaneous and gastrointestinal inflammatory disease with eosinophilic infiltration and peripheral eosinophilia. Notable physical features include a leg length discrepancy, short stature and low body weight. In parallel, she developed refractory membranous nephropathy leading to end stage renal disease. A kidney transplant was complicated by disease recurrence in the graft, progressing over several years, as well as acute rejection ultimately rendering the patient dialysis-dependent. Specifically, her past medical history by organ involvement is further detailed, as follows:Renal: At 3 years of age, she developed rapid weight gain, edema and proteinuria. Renal biopsy at age 7 demonstrated membranous nephropathy, which was refractory to treatment with corticosteroids, and later cyclosporine and tacrolimus. By history the nephrotic syndrome was ameliorated by use of an elemental diet, but this was not able to be consistently maintained. Serological testing for anti-PLA-2R receptor, anti- thrombospondin and anti–bovine serum albumin was all negative. A gradual decline in renal function was observed and at age 11 a living-donor kidney transplantation was performed. One year later, nephrotic range proteinuria recurred, and biopsy confirmed relapsing membranous nephropathy. The graft function further declined, and an episode of acute antibody-mediated rejection resulted in transplant failure at age 16. She has required long-term hemodialysis since that time, delivered via an AV fistula, and is currently being evaluated for a second transplant.Dermatologic: At birth, a pustular rash involving face and extremities was noted. The rash persisted and worsened after discharge. Skin biopsy at 3 months suggested Inflammatory Linear Verrucous Nevus. Skin involvement spread and worsened in intensity with age, manifesting as a diffuse, erythematous rash involving the face, trunk and extremities, with prominence on the left side. Biopsies later demonstrated a subacute or chronic spongiotic dermatitis. The epidermis was acanthotic and showed varying degrees of intercellular edema (spongiosis) with widening of the intercellular spaces; the stratum corneum was thickened and focally compact; the dermis contained a perivascular lymphohistiocytic infiltrate which extended around the superficial and deep vascular plexus. Of note, the biopsies did not show the changes commonly associated with epidermal nevi: alternating ortho and parakeratosis, epidermolytic hyperkeratosis, or acantholytic dyskeratosis. Rather, these clinical and histologic changes in the skin likely represent a form of blaschkitis, an inflammatory skin condition, presenting as papules or vesicles, occurring along the lines of Blaschko (which represent somatically distinct bands of ectodermal migration).Gastrointestinal: In infancy, she experienced recurrent emesis and diarrhea unresponsive to formula changes. Bloody stools were noted at 10 months of age and watery diarrhea and abdominal pain became persistent. Repeat endoscopic biopsies demonstrated chronic, unspecified inflammation at various sites (most frequently colonic, but also gastric, duodenal, ileal and esophageal regions). Eosinophilic infiltration of the colon was consistently noted. Symptoms were only marginally responsive to treatment with corticosteroids, chronic antibiotics and a severely restricted diet.Growth disturbances: Growth impairment was reflected by short stature (Z score < −3) and low body weight (Z score −2 to −8), currently 138 cm and 31 kg. Nutritional etiologies were addressed by placement of a G-tube at age 10 with some improvement in growth. Growth hormone was administered for 5 years with moderate benefit. Leg length discrepancy was identified at birth, with left extremity smaller than right in girth and length.Immunologic: Allergic reactions were observed to enalapril (anaphylaxis), milk (rash), soy (rash) and wheat (rash). She also experienced occasional episodes of dyspnea along with lip and leg angioedema, without an identifiable inciting allergen. Asthma was diagnosed and managed with bronchodilators. Acute phase reactants were noted to be consistently elevated, including ESR and C-reactive protein. Complement (C3 and C4) levels were within reference range. Likewise, quantitative immunoglobulin testing for IgG, IgA, IgE and IgM was within normal limits. Seroconversion after immunization was observed for all vaccinations except varicella virus, hepatitis A virus and hepatitis B virus.Family history was largely unremarkable, with no family history of consanguinity or gastrointestinal, renal, immunologic or dermatologic disease. Mother, father and older brother are alive and well.Response to tofacitinib: After 8 weeks of tofacitinib there was complete normalization of acute phase reactants, ESR and C-reactive protein upon laboratory assessment. The patient was previously unable to tolerate dairy, soy and gluten due to severe abdominal pain and diarrhea. After initiation of tofacitinib, she liberalized her diet without restriction and remained asymptomatic, with formed stools and without abdominal pain.At baseline the endoscopic findings included altered vascularity and friable mucosa from rectum to descending colon with microscopic patchy, active colitis with eosinophilic infiltration noted in the ascending colon. After 6 months of treatment on tofacitinib 5mg daily, the colon was grossly normal, with microscopic active colitis and complete resolution of eosinophilia. The dose was further increased to 7.5 mg daily thereafter. The dermatitis significantly improved, as seen in the images.Tofacitinib is 30% renally-excreted and the dose administered was limited due to chronic kidney disease. Presumably, after re-transplantation the dose may be escalated with a potentially greater effect. To date, the drug has been very well tolerated with no evidence of adverse effects with close monitoring.Coronavirus Disease 2019 (COVID-19): Two years after initiation of tofacitinib treatment, the patient developed an acute respiratory infection that was later diagnosed as COVID-19. She was admitted to the hospital where she was treated with azithromycin and hydroxychloroquine and subsequently intubated. Her immunosuppressive treatment regiment, including tofacitinib, was held constant over this time. Unfortunately, despite intensive supportive care, the patient expired 7 days after admission.Cell linesU4C cells (JAK1
−/−) and γ2A cells (JAK2
−/−) were obtained from S. Pellegrini and cultured in DMEM supplemented with 10% fetal bovine serum (FBS) (Invitrogen), GlutaMAX (350 ng/ml; GIBCO), and penicillin/streptomycin (GIBCO). All cell lines expressing ectopic JAK1 and JAK2 variants were generated by lentiviral transduction and were subsequently selected with puromycin and FACS for matched RFP expression. EBV-transformed lymphoblastoid cell lines (EBV-B cells) were generated by infecting PBMCs from healthy controls or the patient with EBV supernatants. EBV-B cells were cultured in RPMI with 10% FBS, 1% glutamine and 1% penicillin/streptomycin. Single-cell clones were isolated by limiting dilution analysis on OP9 feeder cells and expanded in conditioned media. After genotyping by Sanger sequencing, WT/WT and WT/S703I clones were selected.

Experimental Model and Subject Details

PatientsThis study reports an 18-year-old female who presented during early childhood with persistent, recurrent cutaneous and gastrointestinal inflammatory disease with eosinophilic infiltration and peripheral eosinophilia. Notable physical features include a leg length discrepancy, short stature and low body weight. In parallel, she developed refractory membranous nephropathy leading to end stage renal disease. A kidney transplant was complicated by disease recurrence in the graft, progressing over several years, as well as acute rejection ultimately rendering the patient dialysis-dependent. Specifically, her past medical history by organ involvement is further detailed, as follows:Renal: At 3 years of age, she developed rapid weight gain, edema and proteinuria. Renal biopsy at age 7 demonstrated membranous nephropathy, which was refractory to treatment with corticosteroids, and later cyclosporine and tacrolimus. By history the nephrotic syndrome was ameliorated by use of an elemental diet, but this was not able to be consistently maintained. Serological testing for anti-PLA-2R receptor, anti- thrombospondin and anti–bovine serum albumin was all negative. A gradual decline in renal function was observed and at age 11 a living-donor kidney transplantation was performed. One year later, nephrotic range proteinuria recurred, and biopsy confirmed relapsing membranous nephropathy. The graft function further declined, and an episode of acute antibody-mediated rejection resulted in transplant failure at age 16. She has required long-term hemodialysis since that time, delivered via an AV fistula, and is currently being evaluated for a second transplant.Dermatologic: At birth, a pustular rash involving face and extremities was noted. The rash persisted and worsened after discharge. Skin biopsy at 3 months suggested Inflammatory Linear Verrucous Nevus. Skin involvement spread and worsened in intensity with age, manifesting as a diffuse, erythematous rash involving the face, trunk and extremities, with prominence on the left side. Biopsies later demonstrated a subacute or chronic spongiotic dermatitis. The epidermis was acanthotic and showed varying degrees of intercellular edema (spongiosis) with widening of the intercellular spaces; the stratum corneum was thickened and focally compact; the dermis contained a perivascular lymphohistiocytic infiltrate which extended around the superficial and deep vascular plexus. Of note, the biopsies did not show the changes commonly associated with epidermal nevi: alternating ortho and parakeratosis, epidermolytic hyperkeratosis, or acantholytic dyskeratosis. Rather, these clinical and histologic changes in the skin likely represent a form of blaschkitis, an inflammatory skin condition, presenting as papules or vesicles, occurring along the lines of Blaschko (which represent somatically distinct bands of ectodermal migration).Gastrointestinal: In infancy, she experienced recurrent emesis and diarrhea unresponsive to formula changes. Bloody stools were noted at 10 months of age and watery diarrhea and abdominal pain became persistent. Repeat endoscopic biopsies demonstrated chronic, unspecified inflammation at various sites (most frequently colonic, but also gastric, duodenal, ileal and esophageal regions). Eosinophilic infiltration of the colon was consistently noted. Symptoms were only marginally responsive to treatment with corticosteroids, chronic antibiotics and a severely restricted diet.Growth disturbances: Growth impairment was reflected by short stature (Z score < −3) and low body weight (Z score −2 to −8), currently 138 cm and 31 kg. Nutritional etiologies were addressed by placement of a G-tube at age 10 with some improvement in growth. Growth hormone was administered for 5 years with moderate benefit. Leg length discrepancy was identified at birth, with left extremity smaller than right in girth and length.Immunologic: Allergic reactions were observed to enalapril (anaphylaxis), milk (rash), soy (rash) and wheat (rash). She also experienced occasional episodes of dyspnea along with lip and leg angioedema, without an identifiable inciting allergen. Asthma was diagnosed and managed with bronchodilators. Acute phase reactants were noted to be consistently elevated, including ESR and C-reactive protein. Complement (C3 and C4) levels were within reference range. Likewise, quantitative immunoglobulin testing for IgG, IgA, IgE and IgM was within normal limits. Seroconversion after immunization was observed for all vaccinations except varicella virus, hepatitis A virus and hepatitis B virus.Family history was largely unremarkable, with no family history of consanguinity or gastrointestinal, renal, immunologic or dermatologic disease. Mother, father and older brother are alive and well.Response to tofacitinib: After 8 weeks of tofacitinib there was complete normalization of acute phase reactants, ESR and C-reactive protein upon laboratory assessment. The patient was previously unable to tolerate dairy, soy and gluten due to severe abdominal pain and diarrhea. After initiation of tofacitinib, she liberalized her diet without restriction and remained asymptomatic, with formed stools and without abdominal pain.At baseline the endoscopic findings included altered vascularity and friable mucosa from rectum to descending colon with microscopic patchy, active colitis with eosinophilic infiltration noted in the ascending colon. After 6 months of treatment on tofacitinib 5mg daily, the colon was grossly normal, with microscopic active colitis and complete resolution of eosinophilia. The dose was further increased to 7.5 mg daily thereafter. The dermatitis significantly improved, as seen in the images.Tofacitinib is 30% renally-excreted and the dose administered was limited due to chronic kidney disease. Presumably, after re-transplantation the dose may be escalated with a potentially greater effect. To date, the drug has been very well tolerated with no evidence of adverse effects with close monitoring.Coronavirus Disease 2019 (COVID-19): Two years after initiation of tofacitinib treatment, the patient developed an acute respiratory infection that was later diagnosed as COVID-19. She was admitted to the hospital where she was treated with azithromycin and hydroxychloroquine and subsequently intubated. Her immunosuppressive treatment regiment, including tofacitinib, was held constant over this time. Unfortunately, despite intensive supportive care, the patient expired 7 days after admission.

This study reports an 18-year-old female who presented during early childhood with persistent, recurrent cutaneous and gastrointestinal inflammatory disease with eosinophilic infiltration and peripheral eosinophilia. Notable physical features include a leg length discrepancy, short stature and low body weight. In parallel, she developed refractory membranous nephropathy leading to end stage renal disease. A kidney transplant was complicated by disease recurrence in the graft, progressing over several years, as well as acute rejection ultimately rendering the patient dialysis-dependent. Specifically, her past medical history by organ involvement is further detailed, as follows:

Renal: At 3 years of age, she developed rapid weight gain, edema and proteinuria. Renal biopsy at age 7 demonstrated membranous nephropathy, which was refractory to treatment with corticosteroids, and later cyclosporine and tacrolimus. By history the nephrotic syndrome was ameliorated by use of an elemental diet, but this was not able to be consistently maintained. Serological testing for anti-PLA-2R receptor, anti- thrombospondin and anti–bovine serum albumin was all negative. A gradual decline in renal function was observed and at age 11 a living-donor kidney transplantation was performed. One year later, nephrotic range proteinuria recurred, and biopsy confirmed relapsing membranous nephropathy. The graft function further declined, and an episode of acute antibody-mediated rejection resulted in transplant failure at age 16. She has required long-term hemodialysis since that time, delivered via an AV fistula, and is currently being evaluated for a second transplant.

Dermatologic: At birth, a pustular rash involving face and extremities was noted. The rash persisted and worsened after discharge. Skin biopsy at 3 months suggested Inflammatory Linear Verrucous Nevus. Skin involvement spread and worsened in intensity with age, manifesting as a diffuse, erythematous rash involving the face, trunk and extremities, with prominence on the left side. Biopsies later demonstrated a subacute or chronic spongiotic dermatitis. The epidermis was acanthotic and showed varying degrees of intercellular edema (spongiosis) with widening of the intercellular spaces; the stratum corneum was thickened and focally compact; the dermis contained a perivascular lymphohistiocytic infiltrate which extended around the superficial and deep vascular plexus. Of note, the biopsies did not show the changes commonly associated with epidermal nevi: alternating ortho and parakeratosis, epidermolytic hyperkeratosis, or acantholytic dyskeratosis. Rather, these clinical and histologic changes in the skin likely represent a form of blaschkitis, an inflammatory skin condition, presenting as papules or vesicles, occurring along the lines of Blaschko (which represent somatically distinct bands of ectodermal migration).

Gastrointestinal: In infancy, she experienced recurrent emesis and diarrhea unresponsive to formula changes. Bloody stools were noted at 10 months of age and watery diarrhea and abdominal pain became persistent. Repeat endoscopic biopsies demonstrated chronic, unspecified inflammation at various sites (most frequently colonic, but also gastric, duodenal, ileal and esophageal regions). Eosinophilic infiltration of the colon was consistently noted. Symptoms were only marginally responsive to treatment with corticosteroids, chronic antibiotics and a severely restricted diet.

Growth disturbances: Growth impairment was reflected by short stature (Z score < −3) and low body weight (Z score −2 to −8), currently 138 cm and 31 kg. Nutritional etiologies were addressed by placement of a G-tube at age 10 with some improvement in growth. Growth hormone was administered for 5 years with moderate benefit. Leg length discrepancy was identified at birth, with left extremity smaller than right in girth and length.

Immunologic: Allergic reactions were observed to enalapril (anaphylaxis), milk (rash), soy (rash) and wheat (rash). She also experienced occasional episodes of dyspnea along with lip and leg angioedema, without an identifiable inciting allergen. Asthma was diagnosed and managed with bronchodilators. Acute phase reactants were noted to be consistently elevated, including ESR and C-reactive protein. Complement (C3 and C4) levels were within reference range. Likewise, quantitative immunoglobulin testing for IgG, IgA, IgE and IgM was within normal limits. Seroconversion after immunization was observed for all vaccinations except varicella virus, hepatitis A virus and hepatitis B virus.

Family history was largely unremarkable, with no family history of consanguinity or gastrointestinal, renal, immunologic or dermatologic disease. Mother, father and older brother are alive and well.

Response to tofacitinib: After 8 weeks of tofacitinib there was complete normalization of acute phase reactants, ESR and C-reactive protein upon laboratory assessment. The patient was previously unable to tolerate dairy, soy and gluten due to severe abdominal pain and diarrhea. After initiation of tofacitinib, she liberalized her diet without restriction and remained asymptomatic, with formed stools and without abdominal pain.

At baseline the endoscopic findings included altered vascularity and friable mucosa from rectum to descending colon with microscopic patchy, active colitis with eosinophilic infiltration noted in the ascending colon. After 6 months of treatment on tofacitinib 5mg daily, the colon was grossly normal, with microscopic active colitis and complete resolution of eosinophilia. The dose was further increased to 7.5 mg daily thereafter. The dermatitis significantly improved, as seen in the images.

Tofacitinib is 30% renally-excreted and the dose administered was limited due to chronic kidney disease. Presumably, after re-transplantation the dose may be escalated with a potentially greater effect. To date, the drug has been very well tolerated with no evidence of adverse effects with close monitoring.

Coronavirus Disease 2019 (COVID-19): Two years after initiation of tofacitinib treatment, the patient developed an acute respiratory infection that was later diagnosed as COVID-19. She was admitted to the hospital where she was treated with azithromycin and hydroxychloroquine and subsequently intubated. Her immunosuppressive treatment regiment, including tofacitinib, was held constant over this time. Unfortunately, despite intensive supportive care, the patient expired 7 days after admission.

Cell linesU4C cells (JAK1
−/−) and γ2A cells (JAK2
−/−) were obtained from S. Pellegrini and cultured in DMEM supplemented with 10% fetal bovine serum (FBS) (Invitrogen), GlutaMAX (350 ng/ml; GIBCO), and penicillin/streptomycin (GIBCO). All cell lines expressing ectopic JAK1 and JAK2 variants were generated by lentiviral transduction and were subsequently selected with puromycin and FACS for matched RFP expression. EBV-transformed lymphoblastoid cell lines (EBV-B cells) were generated by infecting PBMCs from healthy controls or the patient with EBV supernatants. EBV-B cells were cultured in RPMI with 10% FBS, 1% glutamine and 1% penicillin/streptomycin. Single-cell clones were isolated by limiting dilution analysis on OP9 feeder cells and expanded in conditioned media. After genotyping by Sanger sequencing, WT/WT and WT/S703I clones were selected.

U4C cells (JAK1
−/−) and γ2A cells (JAK2
−/−) were obtained from S. Pellegrini and cultured in DMEM supplemented with 10% fetal bovine serum (FBS) (Invitrogen), GlutaMAX (350 ng/ml; GIBCO), and penicillin/streptomycin (GIBCO). All cell lines expressing ectopic JAK1 and JAK2 variants were generated by lentiviral transduction and were subsequently selected with puromycin and FACS for matched RFP expression. EBV-transformed lymphoblastoid cell lines (EBV-B cells) were generated by infecting PBMCs from healthy controls or the patient with EBV supernatants. EBV-B cells were cultured in RPMI with 10% FBS, 1% glutamine and 1% penicillin/streptomycin. Single-cell clones were isolated by limiting dilution analysis on OP9 feeder cells and expanded in conditioned media. After genotyping by Sanger sequencing, WT/WT and WT/S703I clones were selected.

Method DetailsVariant analysisAll samples were collected with informed consent in accordance with IRB-approved protocols (Study ID# IF2349568). For whole-exome sequencing, DNA was isolated (QIAGEN Cat No 69504) from Ficoll-isolated granulocytes from whole blood of the proband and her healthy parents. Library preparation, sequencing (150 bp paired-end reads) and alignment for whole-exome sequencing were performed with the Genewiz exome- sequencing package. Potential disease-causing variants were investigated by Ingenuity Variant Analysis (QIAGEN). Hight-quality variants were identified by filtering as follows: exclusion of common variants (> 0.1% allele frequencies in public databases); retention of variants in coding regions resulting in substitutions, premature stops, frameshifts or altered splicing; exclusion of variants with CADD scores below MSC thresholds. For recessive models of inheritance, only homozygous or compound heterozygous mutations were assessed. For de novo inheritance, all variants in the parents were excluded, and alleles with known haploinsufficiency, hemizygous, or dominant-negative effects were included. For the validation of WES results, JAK1 was then amplified from PBMC DNA by PCR and Sanger sequenced.Digital droplet PCRFor the determination of mosaic allele fractions, DNA was isolated from bilateral buccal swabs, fractionated blood and a gastrointestinal endoscopic biopsy specimen in which the epithelial layer was isolated by chemical dissociation. Digital droplet PCR was performed with JAK1 amplification primers, mutation-specific probes (IDT), and with ddPCR Supermix (Biorad 1863026). Amplification and quantification were performed on a QX100 Droplet Digital PCR system (BioRad). Cellular genotypes were estimated with QX100 software (BioRad), assuming heterozygosity.Autoantibody ArrayWe analyzed the presence of circulating autoantibodies using IgG autoantibody arrays (RayBiotech PAH-AIDG-G1-16). In addition to samples from the patient in this study, plasma from five healthy donors, two patients with systemic lupus erythematosus and Immune dysregulation, polyendocrinopathy, enteropathy, X-linked (IPEX) were included. All samples were run in duplicate. Before processing, samples were was first clarified and diluted at a ratio of 1:200. Genepix Pro 7.0 software was used to analyze the images.Cloning and mutagenesisThe JAK1 plasmid was obtained from Addgene (Plasmid #23932) in a Gateway-compatible backbone, pDONR 223. Site-directed mutagenesis with specific primers (Quikchange II, Agilent 200521) was performed to obtain the WT coding sequence in the same plasmid, and plasmids encoding the S703I, K908A or S703I/K908A forms were then generated. A control vector containing the luciferase gene was also cloned. Plasmids were then subcloned into a lentivirus-compatible pTRIP-X-IRES-RFP backbone with puromycin resistance. Pseudotyped lentiviral particles were produced by the transfection of HEK293T cells with pCAGGS-VSV-G, pCMV-Gag/Pol and genes of interest.In vitro stimulationsFor the analysis of STAT phosphorylation, cells were stimulated with the indicated doses of recombinant IFNα-2b (Intron-A, Merck), IFNγ, IL-2, IL-4, and IL-6, IL-13 (Biolegend) for 15 minutes and then lysed for western blotting. For the analysis of gene induction, cells were stimulated for 8 hours, after which cells were lysed for RNA isolation. For JAK inhibitor treatment, cells were incubated with ruxolitinib (Selleckchem S1378), filgotinib (SelleckchemS7605) or tofacitinib (Selleckchem S5001) at the indicated doses for 4 hours.ImmunoblottingCells were lysed in RIPA buffer (Thermo Fisher 89900) supplemented with protease/phosphatase inhibitor cocktail (Cell Signaling #5872). Lysates were sonicated, centrifuged to remove insoluble complexes, then boiled with NuPage sample buffer (Thermo Fisher NP0007) containing 20 mM DTT. The samples were subjected to gel electrophoresis and semi-dry transfer, and the resulting immunoblots were blocked in 5% BSA, then incubated overnight with primary antibody followed by HRP-conjugated secondary antibodies. Primary antibodies against the following targets were used: GAPDH (Cell Signaling D16H11), STAT1 (Santa Cruz C-111), phospho-STAT1 (Cell Signaling 58D6), phospho-STAT2 (Cell Signaling D3P2P), phospho-STAT3 (Cell Signaling D3A7), phospho-STAT5 (Cell Signaling C11C5), phospho-STAT6 (Cell Signaling 9361), JAK1 (Santa Cruz B3), JAK2 (Cell Signaling D2E12), TYK2 (Cell Signaling D4I5T), phospho-JAK1 (Cell Signaling 3331), phospho-JAK2 (Cell Signaling 3771), and phospho-TYK2 (Cell Signaling 9321). For the analysis of JAK phosphorylation, lysates were first incubated overnight with antibodies against total JAK protein conjugated to Protein G Dynabeads (Thermo Fisher 10007D). Immunoblotting was then performed as above.RT-qPCRCell lines or isolated PBMCs were lysed and RNA was isolated with RNeasy spin columns (QIAGEN 74104). Reverse transcription was performed with the High-Capacity RT Kit (Applied Biosystems 4368814). The resulting cDNA was then subjected to qPCR with the TaqMan Master Mix II with UNG (Thermo Fisher 4440038), on a Roche LightCycler 480, with the following primers/probes: 18S (4318839), MX1 (hs00895608), RSAD2 (hs00369813), SIGLEC1 (hs00988063) IFIT1 (hs01911452). The relative expression of each transcript was normalized relative to 18S by the ΔΔCt method.Mass cytometryWhole blood was collected, by venipuncture, into sodium heparin vacutainer tubes. For immunophenotyping, blood was immediately stained and processed for mass cytometry. For intracellular staining, whole blood was stimulated by incubation for 15 minutes with the cytokines indicated and was then immediately stabilized with Proteomic Stabilizer PROT1 (SmartTube) and frozen at −80°C. Similarly, for inhibition with ruxolitinib and tofacitinib, blood was treated for 4 hours with 500 nM inhibitor and was then stimulated by incubation for 15 minutes with 1000 IU/mL IFNα, before stabilization and freezing.Frozen samples were thawed according to the manufacturer’s recommended protocol. The thawed samples were washed with barcode permeabilization buffer (Fluidigm) and barcoded with Fluidigm’s Cell-ID 20-Plex Pd Barcoding Kit. Samples were then washed and pooled into a single tube, Fc-blocked (FcX, Biolegend) and heparin-blocked to prevent non-specific binding. Cells were then stained with a cocktail of markers to identify major immune populations. All antibodies in the panel were either conjugated in-house with X8 MaxPar conjugation kits (Fluidigm) or purchased from Fluidigm. The antibody cocktail was filtered through an Amicon filter with 0.1 μm pores before staining.After surface staining, the samples were permeabilized with methanol and stored for at least 12 hours in methanol at −80°C. Samples were then washed, heparin-blocked and stained with a cocktail of phosphorylation and signaling antibodies. The stained samples were washed and incubated in freshly diluted 2.4% formaldehyde containing 125nM Ir Intercalator (Fluidigm), 0.02% saponin and 30 nM OsO4 (ACROS Organics) for 30 minutes at room temperature. Samples were then washed and acquired immediately after staining.Samples were washed once with PBS+0.2% BSA, once in PBS, and once in CAS buffer (Fluidigm). They were and resuspended at a concentration of 1 million cells per mL in CAS buffer containing a 1/20 dilution of EQ beads (Fluidigm). Following routine instrument tuning and optimization, samples were run at an acquisition rate of < 300 events per second on a Helios mass cytometer (Fluidigm) with a modified wide-bore injector (Fluidigm). FCS files were then normalized and concatenated with Fluidigm acquisition software, and the barcoded samples were deconvoluted with a MATLAB-based debarcoding application (“Palladium-based mass tag cell barcoding with a doublet-filtering scheme and single-cell deconvolution algorithm”). The FCS files were then uploaded to Cytobank for analysis. Cell events were identified as Ir191/193-positive and Ce140-negative events. Doublets were excluded on the basis of Mahalanobis distance and barcode separation and with the Gaussian parameters acquired with Helios CyTOF software. Downstream data analysis was performed on Cytobank, by both tSNE analysis and biaxial gating of immune populations, as shown in Figure S3. Mean signal intensities were calculated, and relative induction was determined by normalization relative to the mean for healthy control samples.Flow cytometryCryopreserved PBMCs from the patient of this study (collected before and after the initiation of tofacitinib treatment), a patient harboring a c.1154C > T p.T385M STAT1 GoF mutation (Vargas-Hernandez et al., 2017), and three unrelated healthy donors were thawed and allowed to rest briefly in complete RPMI medium supplemented with 10% FCS. Cells were immunostained with antibodies in 2% FBS in PBS for 45 minutes. For the panel assessing effector function and activation, cells were stimulated with phorbol 12-myristate 13-acetate (10 ng/ml, Sigma-Aldrich) and ionomycin (1 μg/ml, Sigma-Aldrich) for 4 hours at 37°C in the presence of brefeldin A (10 μg/ml, Sigma-Aldrich) and anti-CD107a antibody. For the panels evaluating effector function and transcription factors, cells were then permeabilized with BD Cytofix/Cytoperm (BD Biosciences) or FoxP3 buffer (Tonbo), and were stained by incubation with antibody for 45-60 minutes. Activated cells were stained for surface markers for 20-25 minutes after the four hours of incubation. Data were acquired on a FACSAria machine (BD Biosciences) with the capacity to detect 18 fluorescent parameters and exported to FlowJo 10.5.3 (BD Biosciences) for analysis. The frequency of cells positive for each parameter was compared with the mean and standard deviation for three healthy donors analyzed in parallel with the samples from the patient.IFNAR2 staining was performed on transduced U4C cells surface stained with anti-IFNAR2 antibody (PBL 21385-1) at a 1:100 dilution for 1 hour on ice. Secondary antibody staining was performed with an anti-murine IgG antibody with an Alexa Fluor 647 tag (Thermo Fischer A-21235). Subsequently the cells were stained with a Live/Dead Aqua stain (Thermo L34957) and then fixed in 4% PFA. Data were acquired on a BD Canto II machine (BD Biosciences) and analyzed using FlowJo 10.5.3.Multiplex ELISAPlasma was collected by Ficoll isolation from heparinized whole blood and clarified by centrifugation. Circulating cytokine levels were determined in magnetic Luminex assays with the Bio-Plex Pro Human Inflammation (BioRad 171al001m) and custom Human Cytokine Panel (R&D LXSAHM), according to the manufacturer’s protocol. Samples were quantified on a MAGPIX xMAP Instrument (Luminex). Cytokine concentrations were quantified by comparison with standard curves and were subsequently translated into Z-scores.ImmunohistochemistryImmunohistochemistry staining was performed with a Discovery Ultra instrument (Roche), with the staining module of RUO Discovery Multimer V2 (V0.00.0083). Slides were first incubated in blocking agent containing 2% BSA PBS, and Discovery Ultra antibody block (Cat # 760-4204) (Roche). Slides were incubated with primary antibodies for 60 minutes at 37°C. The following primary antibodies were used: anti-phospho-STAT1 (Tyr701) 58D6 (Cell Signaling) and anti-phospho-STAT3 (Tyr 705) D3A7 XP (Cell Signaling) at 1:100 dilution in blocking agent. The slides were then incubated with Omni Map anti-rabbit HRP-conjugated secondary antibody (Multimer HRP) (Cat # 760-4311) (Roche) for 32 minutes. Positive signals were detected with the Discovery ChromoMap DAB Kit (Cat #760-159) (Roche).Modified inDrops single cell RNA-Seq targeting JAK1 S703I siteinDrops and related droplet microfluidics single cell RNA-Seq strategies coencapsulate single “barcode microbeads” and individual cells in droplets. Reverse transcription of polyadenylated (polyA) mRNA incorporates a cellular barcode sequence (different for each bead) into nascent cDNA. After downstream high throughput sequencing, reads can be assigned to individual cells by barcode sequences, thereby enabling expression quantification at single cell resolution. As the cellular barcode is introduced downstream of transcript polyA tails, sequence data is typically restricted to transcript regions immediately proximal to 3′ termini. Because the S703I site is located at position 2402 from 5′ transcript start and 2690 from the 3′ transcript polyA tail, it is not accessible by standard droplet microfluidics single cell RNA-Seq platforms. Therefore, we adapted the inDrops method by generating custom barcode microbeads containing JAK1-specific primers (flanking the S703I site) in addition to polyT primer sequences, enabling more efficient JAK1 target capture and access to the S703I site. Following within-droplet reverse transcription, second strand synthesis and in vitro transcription amplification, samples are split into two parallel library preparations: one for standard polyT-primed libraries, and one for JAK1-targeted libraries. During data processing of resulting high throughput sequencing data, sequence reads from both libraries are assigned to individual cells based on shared barcode sequences. A detailed description of this approach appears below.JAK1-targeted hydrogel microbeadsBarcoding hydrogel beads were prepared according to established inDrops protocol (Zilionis et al., 2017) with modified primers (Zilionis et al., 2019) with the following modifications. For the second round of split-and-pool primer extension for barcode synthesis, hydrogel beads were added to microplate wells (n = 384) containing oligonucleotide templates for both standard polyT primers and for an additional primer complementary to an S703I-adjacent region of the JAK1 transcript (11.53uM polyT oligonucleotide template, 2.3uM JAK1 oligonucleotide template) Within a given well, both polyT and JAK1 oligonucleotide templates carried identical “barcode 2” sequences, ensuring that extended primers on hydrogels contained matching barcodes. Subsequent exonuclease processing steps included corresponding JAK1 primer complementary blocking oligonucleotides.JAK1-targeted library preparationFreshly isolated PBMC were co-encapsulated with JAK1-targeted hydrogel beads. The standard inDrops protocol was followed for reverse transcription, droplet breakage, second strand synthesis and in vitro transcription (IVT) amplification. IVT reactions (20 ul) were then split to two parallel library preparations: 10 ul of IVT product was prepped for polyT-primed gene expression libraries according to the standard inDrops protocol, and 10 ul of IVT product (typically reserved as a “backup” aliquot) was prepped according to a JAK1-targeted protocol as follows. IVT products were reverse transcribed by SuperScript III with a JAK1-specific primer (with 5′ extension containing Illumina adaptor sequence) flanking the S703I site (55C for 1 hr, 70C for 15 min). RT reactions were treated with RNase H (37C for 30 min, 65C for 20 min) to remove RNA template from cDNA heteroduplexes. Following purification on 1.5X Ampure XP beads (Beckman Coulter), JAK1-enriched cDNA was amplified by PCR with Illumina-adapted inDrops primers (KAPA HiFi Master Mix; 2 cycles 98C x 20 s, 55C x 30 s, 72C for 40 s; 16-18 cycles 98C x 20 s, 65C x 30 s, 72C for 40 s; final 72C extension x 5 min). Final JAK1-targeted libraries were purified on 0.8X Ampure XP beads.Oligonucleotide sequencespolyT oligonucleotide template 5′-BAAAAAAAAAAAAAAAAAAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG – 3′JAK1 oligonucleotide template 5′ - GAGTGTGGCCCATTCATCAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG −3′JAK1 blocking oligonucleotide 5′-GAGTGTGGCCCATTCATCAA-3′Final “on bead” polyT primer sequence 5′- CGATGACGTAATACGACTCACTATAGGGTGTCGGGTGCAG[bc1,8nt]GTCTCGTGGGCTCGGAGATGTGTA TAAGAGACAG[bc2,8nt]NNNNNNTTTTTTTTTTTTTTTTTTTV-3′Final “on bead” JAK1 primer sequence 5′- CGATGACGTAATACGACTCACTATAGGGTGTCGGGTGCAG[bc1,8nt]GTCTCGTGGGCTCGGAGATGTGTA TAAGAGACAG[bc2,8nt]NNNNNNTTGATGAATGGGCCACACTC-3;Second reverse transcription JAK1 primer: 5′- TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGCCATGGAAATTCAAAGTTGCCAAACAG-3′High Throughput SequencingBoth standard polyT-primed libraries and JAK1-targeted libraries were pooled together and sequenced in multiplex on the Illumina NextSeq 500 platform with 75-cycle reagent kits in paired-end, dual index configuration:Read 1 containing transcript/JAK1 data: 61 cyclesi7 read containing cell barcode data: 8 cyclesi5 read containing sample index data: 8 cyclesRead 2 containing cell barcode and unique molecular identified (UMI) data: 14 cycles
Data ProcessingFASTQ sequence files were processed with the inDrops.py workflow script (v0.3, https://github.com/inDrops/inDrops). For polyT-primed gene expression libraries, resulting gene x cell matrices and per cell read counts were used for downstream analyses in Seurat (details below), using only “cell-containing droplets” defined by a read count higher than 8000-10000 depending on read count evaluation from droplets in the corresponding library batch.For JAK1-targeted libraries, BAM files (appended with cellular barcode and UMI data) were used to evaluate genotypes at single cell resolution as follows. Reads covering the JAK1 S703I site (chr1:64845519 – 64845521 on the minus strand; human genome reference GRCh38) containing the wild-type sequence (GAT) or S703I (GCT) sequence were quantified per UMI per cellular barcode. If at least 2 and at least 90% of the reads for a given cellular barcode/UMI combination contained the same sequence (wild-type or S703I), then this combination was designated a JAK1 transcript of the given genotype and assigned to the appropriate cellular barcode.We next used per cell JAK1 WT and S703I transcript counts to assign putative genotypes to individual cells. As cell free RNA in suspension can co-encapsulate with cells thereby generating unwanted background signal in droplet microfluidics scRNA-Seq methods, we aimed to apply a stringent, data driven threshold for genotyping assignment. To evaluate the potential influence of cell free RNA, we quantified the frequency of JAK1 transcripts detected in empty (i.e., cell free) droplets (defined as barcodes with 1000 – 2000 reads in polyT-primed libraries) using the same barcode/UMI strategy described above. We found that more than 99% of empty droplets had less than 3 JAK1 transcripts. Guided by these data, cells were only considered for JAK1 genotype assignment if at least 5 JAK1 transcripts were detected. For JAK1 genotype assignment, individual cells were classified in one of the following four categories: cells that carried the S703I allele were classified as “MUT+” regardless of carrying the wild-type allele; the remainder (S703I allele negative) cells that carried the wild-type allele were classified as “WT+MUTneg” if no S703I transcripts were detected, and as “WT+MUTnonZero” if 1-to-4 S703I transcripts were detected; cells that carried neither the wild-type nor the S703I allele were classified as “WTnegMUTneg.”Single cell RNA-Seq Gene Expression AnalysisData from JAK1-targeted inDrops single cell RNA-Seq data were analyzed in conjunction with corresponding single cell RNA-Seq data acquired from the same PBMC specimen on the 10X Genomics Chromium platform. Sequence reads from 10X libraries were processed with the CellRanger software package (10X Genomics) using default parameters. Gene x cell matrices from both methods were further analyzed with Seurat (v2.3.4) (Butler et al., 2018) in the R statistical framework as follows. For applicable cells in the JAK1-targeted inDrops dataset, JAK1 genotyping information was imported as per cell metadata entries. Genes with detectable expression in fewer than 5 cells per sample were excluded. Cells with detectable expression of fewer than 250 genes or fewer than 1000 UMI counts or greater than 20% UMIs from mitochondrial gene transcripts (measure of cell viability) were removed from further analysis. Gene expression data were log normalized and scaled (regressing out effects based on total UMI counts and mitochondrial gene expression). All genes with detectable expression in both inDrops and 10X datasets were used for CCA dimensionality reduction (20 dimensions). CCA dimension scores were aligned between datasets using the AlignSubspace function. Aligned data were log-normalized and scaled as above. Graph- based clustering and tSNE visualization was performed with the FindClusters and RunTSNE functions with default settings (using the first 19 CCA dimensions, based on examination of distinct expression patterns with the DimHeatmap function). Marker genes for individual clusters were identified with the FindMarkers() function in Seurat (paramenters: min.pct = 0.1, only.pos = TRUE, logfc.threshold = log(1.5), min.diff.pct = 0.1, pseudocount.use = 0.1). Briefly, up to 100 cells were randomly selected from each cluster, and each cluster was contrasted against all others for differential gene expression. Significant genes (p < 0.05) were ranked by log2 fold-change; the top 25 genes for each cluster, as well as a select list of additional genes of interest, were selected as marker genes for supervised cluster annotation (Table S3). Expression patterns were further evaluated on tSNE plots. Select clusters were merged based on shared expression patterns as indicated.Gene expression differences were assessed for cells assigned JAK1 genotype classifications of either MUT+ (S703I) or WT+MUTneg (WT). Plasmablasts and dendritic cells were excluded from analysis due to insufficient cells assigned to both genotype groups. Apparent cellular doublets and cells in the platelet cluster were also excluded. Differential gene expression testing was performed on genes detected in at least 20% of cells in any cell type cluster (of either JAK1 genoytpe group) with edgeR (v.3.12.1) (Robinson et al., 2010), including modifications included for single cell RNA-Seq data as described (Soneson and Robinson, 2018). A linear model including factors for cellular gene detection rate, cell type, JAK1 genotype group and an interaction term (cell type cluster and JAK1 genotype group), was fit with the glmQLFit function. Differential gene expression testing (WT versus S703I) was tested across all cell types with the glmQLFtest function, with significance thresholds set at FDR 0.05. In addition, gene set enrichment testing for Molecular Signatures Database Collections H, C2- CPC, C3-TFT and C7 collections (http://software.broadinstitute.org/gsea/msigdb), supplemented with two additional interferon stimulated gene sets (Rosenberg et al., 2018) was performed with CAMERA (Wu and Smyth, 2012) and the linear model described above. Only gene sets with 5 or more genes in the fitted model were included in the analysis.Single Cell qPCR Transcript GenotypingPBMCs from a healthy donor carrying a heterozygous SNP in JAK1 (rs2230587) were isolated by Ficoll gradient. Cells were stained with antibodies against CD3, CD19, CD14 and CD56 (Biolegend), and 100 single cells were FACS-sorted into single cell lysis buffer (Ambion 4458235). Following DNase treatment, cDNA was generated using SuperScript VILO RT kit (ThermoFisher 11754050). A linear preamplification was then performed using JAK1 primers (gtcctctggatctcttcatgca, gctgtttggcaactttgaatttcc) and primers to a negative control gene NACA (cccaggcaaccacacaac, ccgactctgttttgctttactgact). Using qPCR with custom TaqMan genotyping primers (above) and allele-specific probes (JAK1 aaggacatc[g/a]cttttc; agcagctgaaat[T/C]gatgaa) that were individually fluorescently tagged (VIC and FAM), allelic ratios were determined by endpoint genotyping. Quantification was carried out by interpolation from a standard curve of oligonucleotide standards.

Method Details

Variant analysisAll samples were collected with informed consent in accordance with IRB-approved protocols (Study ID# IF2349568). For whole-exome sequencing, DNA was isolated (QIAGEN Cat No 69504) from Ficoll-isolated granulocytes from whole blood of the proband and her healthy parents. Library preparation, sequencing (150 bp paired-end reads) and alignment for whole-exome sequencing were performed with the Genewiz exome- sequencing package. Potential disease-causing variants were investigated by Ingenuity Variant Analysis (QIAGEN). Hight-quality variants were identified by filtering as follows: exclusion of common variants (> 0.1% allele frequencies in public databases); retention of variants in coding regions resulting in substitutions, premature stops, frameshifts or altered splicing; exclusion of variants with CADD scores below MSC thresholds. For recessive models of inheritance, only homozygous or compound heterozygous mutations were assessed. For de novo inheritance, all variants in the parents were excluded, and alleles with known haploinsufficiency, hemizygous, or dominant-negative effects were included. For the validation of WES results, JAK1 was then amplified from PBMC DNA by PCR and Sanger sequenced.

Variant analysis

All samples were collected with informed consent in accordance with IRB-approved protocols (Study ID# IF2349568). For whole-exome sequencing, DNA was isolated (QIAGEN Cat No 69504) from Ficoll-isolated granulocytes from whole blood of the proband and her healthy parents. Library preparation, sequencing (150 bp paired-end reads) and alignment for whole-exome sequencing were performed with the Genewiz exome- sequencing package. Potential disease-causing variants were investigated by Ingenuity Variant Analysis (QIAGEN). Hight-quality variants were identified by filtering as follows: exclusion of common variants (> 0.1% allele frequencies in public databases); retention of variants in coding regions resulting in substitutions, premature stops, frameshifts or altered splicing; exclusion of variants with CADD scores below MSC thresholds. For recessive models of inheritance, only homozygous or compound heterozygous mutations were assessed. For de novo inheritance, all variants in the parents were excluded, and alleles with known haploinsufficiency, hemizygous, or dominant-negative effects were included. For the validation of WES results, JAK1 was then amplified from PBMC DNA by PCR and Sanger sequenced.

Digital droplet PCRFor the determination of mosaic allele fractions, DNA was isolated from bilateral buccal swabs, fractionated blood and a gastrointestinal endoscopic biopsy specimen in which the epithelial layer was isolated by chemical dissociation. Digital droplet PCR was performed with JAK1 amplification primers, mutation-specific probes (IDT), and with ddPCR Supermix (Biorad 1863026). Amplification and quantification were performed on a QX100 Droplet Digital PCR system (BioRad). Cellular genotypes were estimated with QX100 software (BioRad), assuming heterozygosity.

Digital droplet PCR

For the determination of mosaic allele fractions, DNA was isolated from bilateral buccal swabs, fractionated blood and a gastrointestinal endoscopic biopsy specimen in which the epithelial layer was isolated by chemical dissociation. Digital droplet PCR was performed with JAK1 amplification primers, mutation-specific probes (IDT), and with ddPCR Supermix (Biorad 1863026). Amplification and quantification were performed on a QX100 Droplet Digital PCR system (BioRad). Cellular genotypes were estimated with QX100 software (BioRad), assuming heterozygosity.

Autoantibody ArrayWe analyzed the presence of circulating autoantibodies using IgG autoantibody arrays (RayBiotech PAH-AIDG-G1-16). In addition to samples from the patient in this study, plasma from five healthy donors, two patients with systemic lupus erythematosus and Immune dysregulation, polyendocrinopathy, enteropathy, X-linked (IPEX) were included. All samples were run in duplicate. Before processing, samples were was first clarified and diluted at a ratio of 1:200. Genepix Pro 7.0 software was used to analyze the images.

Autoantibody Array

We analyzed the presence of circulating autoantibodies using IgG autoantibody arrays (RayBiotech PAH-AIDG-G1-16). In addition to samples from the patient in this study, plasma from five healthy donors, two patients with systemic lupus erythematosus and Immune dysregulation, polyendocrinopathy, enteropathy, X-linked (IPEX) were included. All samples were run in duplicate. Before processing, samples were was first clarified and diluted at a ratio of 1:200. Genepix Pro 7.0 software was used to analyze the images.

Cloning and mutagenesisThe JAK1 plasmid was obtained from Addgene (Plasmid #23932) in a Gateway-compatible backbone, pDONR 223. Site-directed mutagenesis with specific primers (Quikchange II, Agilent 200521) was performed to obtain the WT coding sequence in the same plasmid, and plasmids encoding the S703I, K908A or S703I/K908A forms were then generated. A control vector containing the luciferase gene was also cloned. Plasmids were then subcloned into a lentivirus-compatible pTRIP-X-IRES-RFP backbone with puromycin resistance. Pseudotyped lentiviral particles were produced by the transfection of HEK293T cells with pCAGGS-VSV-G, pCMV-Gag/Pol and genes of interest.

Cloning and mutagenesis

The JAK1 plasmid was obtained from Addgene (Plasmid #23932) in a Gateway-compatible backbone, pDONR 223. Site-directed mutagenesis with specific primers (Quikchange II, Agilent 200521) was performed to obtain the WT coding sequence in the same plasmid, and plasmids encoding the S703I, K908A or S703I/K908A forms were then generated. A control vector containing the luciferase gene was also cloned. Plasmids were then subcloned into a lentivirus-compatible pTRIP-X-IRES-RFP backbone with puromycin resistance. Pseudotyped lentiviral particles were produced by the transfection of HEK293T cells with pCAGGS-VSV-G, pCMV-Gag/Pol and genes of interest.

In vitro stimulationsFor the analysis of STAT phosphorylation, cells were stimulated with the indicated doses of recombinant IFNα-2b (Intron-A, Merck), IFNγ, IL-2, IL-4, and IL-6, IL-13 (Biolegend) for 15 minutes and then lysed for western blotting. For the analysis of gene induction, cells were stimulated for 8 hours, after which cells were lysed for RNA isolation. For JAK inhibitor treatment, cells were incubated with ruxolitinib (Selleckchem S1378), filgotinib (SelleckchemS7605) or tofacitinib (Selleckchem S5001) at the indicated doses for 4 hours.

In vitro stimulations

For the analysis of STAT phosphorylation, cells were stimulated with the indicated doses of recombinant IFNα-2b (Intron-A, Merck), IFNγ, IL-2, IL-4, and IL-6, IL-13 (Biolegend) for 15 minutes and then lysed for western blotting. For the analysis of gene induction, cells were stimulated for 8 hours, after which cells were lysed for RNA isolation. For JAK inhibitor treatment, cells were incubated with ruxolitinib (Selleckchem S1378), filgotinib (SelleckchemS7605) or tofacitinib (Selleckchem S5001) at the indicated doses for 4 hours.

ImmunoblottingCells were lysed in RIPA buffer (Thermo Fisher 89900) supplemented with protease/phosphatase inhibitor cocktail (Cell Signaling #5872). Lysates were sonicated, centrifuged to remove insoluble complexes, then boiled with NuPage sample buffer (Thermo Fisher NP0007) containing 20 mM DTT. The samples were subjected to gel electrophoresis and semi-dry transfer, and the resulting immunoblots were blocked in 5% BSA, then incubated overnight with primary antibody followed by HRP-conjugated secondary antibodies. Primary antibodies against the following targets were used: GAPDH (Cell Signaling D16H11), STAT1 (Santa Cruz C-111), phospho-STAT1 (Cell Signaling 58D6), phospho-STAT2 (Cell Signaling D3P2P), phospho-STAT3 (Cell Signaling D3A7), phospho-STAT5 (Cell Signaling C11C5), phospho-STAT6 (Cell Signaling 9361), JAK1 (Santa Cruz B3), JAK2 (Cell Signaling D2E12), TYK2 (Cell Signaling D4I5T), phospho-JAK1 (Cell Signaling 3331), phospho-JAK2 (Cell Signaling 3771), and phospho-TYK2 (Cell Signaling 9321). For the analysis of JAK phosphorylation, lysates were first incubated overnight with antibodies against total JAK protein conjugated to Protein G Dynabeads (Thermo Fisher 10007D). Immunoblotting was then performed as above.

Immunoblotting

Cells were lysed in RIPA buffer (Thermo Fisher 89900) supplemented with protease/phosphatase inhibitor cocktail (Cell Signaling #5872). Lysates were sonicated, centrifuged to remove insoluble complexes, then boiled with NuPage sample buffer (Thermo Fisher NP0007) containing 20 mM DTT. The samples were subjected to gel electrophoresis and semi-dry transfer, and the resulting immunoblots were blocked in 5% BSA, then incubated overnight with primary antibody followed by HRP-conjugated secondary antibodies. Primary antibodies against the following targets were used: GAPDH (Cell Signaling D16H11), STAT1 (Santa Cruz C-111), phospho-STAT1 (Cell Signaling 58D6), phospho-STAT2 (Cell Signaling D3P2P), phospho-STAT3 (Cell Signaling D3A7), phospho-STAT5 (Cell Signaling C11C5), phospho-STAT6 (Cell Signaling 9361), JAK1 (Santa Cruz B3), JAK2 (Cell Signaling D2E12), TYK2 (Cell Signaling D4I5T), phospho-JAK1 (Cell Signaling 3331), phospho-JAK2 (Cell Signaling 3771), and phospho-TYK2 (Cell Signaling 9321). For the analysis of JAK phosphorylation, lysates were first incubated overnight with antibodies against total JAK protein conjugated to Protein G Dynabeads (Thermo Fisher 10007D). Immunoblotting was then performed as above.

RT-qPCRCell lines or isolated PBMCs were lysed and RNA was isolated with RNeasy spin columns (QIAGEN 74104). Reverse transcription was performed with the High-Capacity RT Kit (Applied Biosystems 4368814). The resulting cDNA was then subjected to qPCR with the TaqMan Master Mix II with UNG (Thermo Fisher 4440038), on a Roche LightCycler 480, with the following primers/probes: 18S (4318839), MX1 (hs00895608), RSAD2 (hs00369813), SIGLEC1 (hs00988063) IFIT1 (hs01911452). The relative expression of each transcript was normalized relative to 18S by the ΔΔCt method.

Cell lines or isolated PBMCs were lysed and RNA was isolated with RNeasy spin columns (QIAGEN 74104). Reverse transcription was performed with the High-Capacity RT Kit (Applied Biosystems 4368814). The resulting cDNA was then subjected to qPCR with the TaqMan Master Mix II with UNG (Thermo Fisher 4440038), on a Roche LightCycler 480, with the following primers/probes: 18S (4318839), MX1 (hs00895608), RSAD2 (hs00369813), SIGLEC1 (hs00988063) IFIT1 (hs01911452). The relative expression of each transcript was normalized relative to 18S by the ΔΔCt method.

Mass cytometryWhole blood was collected, by venipuncture, into sodium heparin vacutainer tubes. For immunophenotyping, blood was immediately stained and processed for mass cytometry. For intracellular staining, whole blood was stimulated by incubation for 15 minutes with the cytokines indicated and was then immediately stabilized with Proteomic Stabilizer PROT1 (SmartTube) and frozen at −80°C. Similarly, for inhibition with ruxolitinib and tofacitinib, blood was treated for 4 hours with 500 nM inhibitor and was then stimulated by incubation for 15 minutes with 1000 IU/mL IFNα, before stabilization and freezing.Frozen samples were thawed according to the manufacturer’s recommended protocol. The thawed samples were washed with barcode permeabilization buffer (Fluidigm) and barcoded with Fluidigm’s Cell-ID 20-Plex Pd Barcoding Kit. Samples were then washed and pooled into a single tube, Fc-blocked (FcX, Biolegend) and heparin-blocked to prevent non-specific binding. Cells were then stained with a cocktail of markers to identify major immune populations. All antibodies in the panel were either conjugated in-house with X8 MaxPar conjugation kits (Fluidigm) or purchased from Fluidigm. The antibody cocktail was filtered through an Amicon filter with 0.1 μm pores before staining.After surface staining, the samples were permeabilized with methanol and stored for at least 12 hours in methanol at −80°C. Samples were then washed, heparin-blocked and stained with a cocktail of phosphorylation and signaling antibodies. The stained samples were washed and incubated in freshly diluted 2.4% formaldehyde containing 125nM Ir Intercalator (Fluidigm), 0.02% saponin and 30 nM OsO4 (ACROS Organics) for 30 minutes at room temperature. Samples were then washed and acquired immediately after staining.Samples were washed once with PBS+0.2% BSA, once in PBS, and once in CAS buffer (Fluidigm). They were and resuspended at a concentration of 1 million cells per mL in CAS buffer containing a 1/20 dilution of EQ beads (Fluidigm). Following routine instrument tuning and optimization, samples were run at an acquisition rate of < 300 events per second on a Helios mass cytometer (Fluidigm) with a modified wide-bore injector (Fluidigm). FCS files were then normalized and concatenated with Fluidigm acquisition software, and the barcoded samples were deconvoluted with a MATLAB-based debarcoding application (“Palladium-based mass tag cell barcoding with a doublet-filtering scheme and single-cell deconvolution algorithm”). The FCS files were then uploaded to Cytobank for analysis. Cell events were identified as Ir191/193-positive and Ce140-negative events. Doublets were excluded on the basis of Mahalanobis distance and barcode separation and with the Gaussian parameters acquired with Helios CyTOF software. Downstream data analysis was performed on Cytobank, by both tSNE analysis and biaxial gating of immune populations, as shown in Figure S3. Mean signal intensities were calculated, and relative induction was determined by normalization relative to the mean for healthy control samples.

Mass cytometry

Whole blood was collected, by venipuncture, into sodium heparin vacutainer tubes. For immunophenotyping, blood was immediately stained and processed for mass cytometry. For intracellular staining, whole blood was stimulated by incubation for 15 minutes with the cytokines indicated and was then immediately stabilized with Proteomic Stabilizer PROT1 (SmartTube) and frozen at −80°C. Similarly, for inhibition with ruxolitinib and tofacitinib, blood was treated for 4 hours with 500 nM inhibitor and was then stimulated by incubation for 15 minutes with 1000 IU/mL IFNα, before stabilization and freezing.

Frozen samples were thawed according to the manufacturer’s recommended protocol. The thawed samples were washed with barcode permeabilization buffer (Fluidigm) and barcoded with Fluidigm’s Cell-ID 20-Plex Pd Barcoding Kit. Samples were then washed and pooled into a single tube, Fc-blocked (FcX, Biolegend) and heparin-blocked to prevent non-specific binding. Cells were then stained with a cocktail of markers to identify major immune populations. All antibodies in the panel were either conjugated in-house with X8 MaxPar conjugation kits (Fluidigm) or purchased from Fluidigm. The antibody cocktail was filtered through an Amicon filter with 0.1 μm pores before staining.

After surface staining, the samples were permeabilized with methanol and stored for at least 12 hours in methanol at −80°C. Samples were then washed, heparin-blocked and stained with a cocktail of phosphorylation and signaling antibodies. The stained samples were washed and incubated in freshly diluted 2.4% formaldehyde containing 125nM Ir Intercalator (Fluidigm), 0.02% saponin and 30 nM OsO4 (ACROS Organics) for 30 minutes at room temperature. Samples were then washed and acquired immediately after staining.

Samples were washed once with PBS+0.2% BSA, once in PBS, and once in CAS buffer (Fluidigm). They were and resuspended at a concentration of 1 million cells per mL in CAS buffer containing a 1/20 dilution of EQ beads (Fluidigm). Following routine instrument tuning and optimization, samples were run at an acquisition rate of < 300 events per second on a Helios mass cytometer (Fluidigm) with a modified wide-bore injector (Fluidigm). FCS files were then normalized and concatenated with Fluidigm acquisition software, and the barcoded samples were deconvoluted with a MATLAB-based debarcoding application (“Palladium-based mass tag cell barcoding with a doublet-filtering scheme and single-cell deconvolution algorithm”). The FCS files were then uploaded to Cytobank for analysis. Cell events were identified as Ir191/193-positive and Ce140-negative events. Doublets were excluded on the basis of Mahalanobis distance and barcode separation and with the Gaussian parameters acquired with Helios CyTOF software. Downstream data analysis was performed on Cytobank, by both tSNE analysis and biaxial gating of immune populations, as shown in Figure S3. Mean signal intensities were calculated, and relative induction was determined by normalization relative to the mean for healthy control samples.

Flow cytometryCryopreserved PBMCs from the patient of this study (collected before and after the initiation of tofacitinib treatment), a patient harboring a c.1154C > T p.T385M STAT1 GoF mutation (Vargas-Hernandez et al., 2017), and three unrelated healthy donors were thawed and allowed to rest briefly in complete RPMI medium supplemented with 10% FCS. Cells were immunostained with antibodies in 2% FBS in PBS for 45 minutes. For the panel assessing effector function and activation, cells were stimulated with phorbol 12-myristate 13-acetate (10 ng/ml, Sigma-Aldrich) and ionomycin (1 μg/ml, Sigma-Aldrich) for 4 hours at 37°C in the presence of brefeldin A (10 μg/ml, Sigma-Aldrich) and anti-CD107a antibody. For the panels evaluating effector function and transcription factors, cells were then permeabilized with BD Cytofix/Cytoperm (BD Biosciences) or FoxP3 buffer (Tonbo), and were stained by incubation with antibody for 45-60 minutes. Activated cells were stained for surface markers for 20-25 minutes after the four hours of incubation. Data were acquired on a FACSAria machine (BD Biosciences) with the capacity to detect 18 fluorescent parameters and exported to FlowJo 10.5.3 (BD Biosciences) for analysis. The frequency of cells positive for each parameter was compared with the mean and standard deviation for three healthy donors analyzed in parallel with the samples from the patient.IFNAR2 staining was performed on transduced U4C cells surface stained with anti-IFNAR2 antibody (PBL 21385-1) at a 1:100 dilution for 1 hour on ice. Secondary antibody staining was performed with an anti-murine IgG antibody with an Alexa Fluor 647 tag (Thermo Fischer A-21235). Subsequently the cells were stained with a Live/Dead Aqua stain (Thermo L34957) and then fixed in 4% PFA. Data were acquired on a BD Canto II machine (BD Biosciences) and analyzed using FlowJo 10.5.3.

Flow cytometry

Cryopreserved PBMCs from the patient of this study (collected before and after the initiation of tofacitinib treatment), a patient harboring a c.1154C > T p.T385M STAT1 GoF mutation (Vargas-Hernandez et al., 2017), and three unrelated healthy donors were thawed and allowed to rest briefly in complete RPMI medium supplemented with 10% FCS. Cells were immunostained with antibodies in 2% FBS in PBS for 45 minutes. For the panel assessing effector function and activation, cells were stimulated with phorbol 12-myristate 13-acetate (10 ng/ml, Sigma-Aldrich) and ionomycin (1 μg/ml, Sigma-Aldrich) for 4 hours at 37°C in the presence of brefeldin A (10 μg/ml, Sigma-Aldrich) and anti-CD107a antibody. For the panels evaluating effector function and transcription factors, cells were then permeabilized with BD Cytofix/Cytoperm (BD Biosciences) or FoxP3 buffer (Tonbo), and were stained by incubation with antibody for 45-60 minutes. Activated cells were stained for surface markers for 20-25 minutes after the four hours of incubation. Data were acquired on a FACSAria machine (BD Biosciences) with the capacity to detect 18 fluorescent parameters and exported to FlowJo 10.5.3 (BD Biosciences) for analysis. The frequency of cells positive for each parameter was compared with the mean and standard deviation for three healthy donors analyzed in parallel with the samples from the patient.

IFNAR2 staining was performed on transduced U4C cells surface stained with anti-IFNAR2 antibody (PBL 21385-1) at a 1:100 dilution for 1 hour on ice. Secondary antibody staining was performed with an anti-murine IgG antibody with an Alexa Fluor 647 tag (Thermo Fischer A-21235). Subsequently the cells were stained with a Live/Dead Aqua stain (Thermo L34957) and then fixed in 4% PFA. Data were acquired on a BD Canto II machine (BD Biosciences) and analyzed using FlowJo 10.5.3.

Multiplex ELISAPlasma was collected by Ficoll isolation from heparinized whole blood and clarified by centrifugation. Circulating cytokine levels were determined in magnetic Luminex assays with the Bio-Plex Pro Human Inflammation (BioRad 171al001m) and custom Human Cytokine Panel (R&D LXSAHM), according to the manufacturer’s protocol. Samples were quantified on a MAGPIX xMAP Instrument (Luminex). Cytokine concentrations were quantified by comparison with standard curves and were subsequently translated into Z-scores.

Multiplex ELISA

Plasma was collected by Ficoll isolation from heparinized whole blood and clarified by centrifugation. Circulating cytokine levels were determined in magnetic Luminex assays with the Bio-Plex Pro Human Inflammation (BioRad 171al001m) and custom Human Cytokine Panel (R&D LXSAHM), according to the manufacturer’s protocol. Samples were quantified on a MAGPIX xMAP Instrument (Luminex). Cytokine concentrations were quantified by comparison with standard curves and were subsequently translated into Z-scores.

ImmunohistochemistryImmunohistochemistry staining was performed with a Discovery Ultra instrument (Roche), with the staining module of RUO Discovery Multimer V2 (V0.00.0083). Slides were first incubated in blocking agent containing 2% BSA PBS, and Discovery Ultra antibody block (Cat # 760-4204) (Roche). Slides were incubated with primary antibodies for 60 minutes at 37°C. The following primary antibodies were used: anti-phospho-STAT1 (Tyr701) 58D6 (Cell Signaling) and anti-phospho-STAT3 (Tyr 705) D3A7 XP (Cell Signaling) at 1:100 dilution in blocking agent. The slides were then incubated with Omni Map anti-rabbit HRP-conjugated secondary antibody (Multimer HRP) (Cat # 760-4311) (Roche) for 32 minutes. Positive signals were detected with the Discovery ChromoMap DAB Kit (Cat #760-159) (Roche).

Immunohistochemistry

Immunohistochemistry staining was performed with a Discovery Ultra instrument (Roche), with the staining module of RUO Discovery Multimer V2 (V0.00.0083). Slides were first incubated in blocking agent containing 2% BSA PBS, and Discovery Ultra antibody block (Cat # 760-4204) (Roche). Slides were incubated with primary antibodies for 60 minutes at 37°C. The following primary antibodies were used: anti-phospho-STAT1 (Tyr701) 58D6 (Cell Signaling) and anti-phospho-STAT3 (Tyr 705) D3A7 XP (Cell Signaling) at 1:100 dilution in blocking agent. The slides were then incubated with Omni Map anti-rabbit HRP-conjugated secondary antibody (Multimer HRP) (Cat # 760-4311) (Roche) for 32 minutes. Positive signals were detected with the Discovery ChromoMap DAB Kit (Cat #760-159) (Roche).

Modified inDrops single cell RNA-Seq targeting JAK1 S703I siteinDrops and related droplet microfluidics single cell RNA-Seq strategies coencapsulate single “barcode microbeads” and individual cells in droplets. Reverse transcription of polyadenylated (polyA) mRNA incorporates a cellular barcode sequence (different for each bead) into nascent cDNA. After downstream high throughput sequencing, reads can be assigned to individual cells by barcode sequences, thereby enabling expression quantification at single cell resolution. As the cellular barcode is introduced downstream of transcript polyA tails, sequence data is typically restricted to transcript regions immediately proximal to 3′ termini. Because the S703I site is located at position 2402 from 5′ transcript start and 2690 from the 3′ transcript polyA tail, it is not accessible by standard droplet microfluidics single cell RNA-Seq platforms. Therefore, we adapted the inDrops method by generating custom barcode microbeads containing JAK1-specific primers (flanking the S703I site) in addition to polyT primer sequences, enabling more efficient JAK1 target capture and access to the S703I site. Following within-droplet reverse transcription, second strand synthesis and in vitro transcription amplification, samples are split into two parallel library preparations: one for standard polyT-primed libraries, and one for JAK1-targeted libraries. During data processing of resulting high throughput sequencing data, sequence reads from both libraries are assigned to individual cells based on shared barcode sequences. A detailed description of this approach appears below.

Modified inDrops single cell RNA-Seq targeting JAK1 S703I site

inDrops and related droplet microfluidics single cell RNA-Seq strategies coencapsulate single “barcode microbeads” and individual cells in droplets. Reverse transcription of polyadenylated (polyA) mRNA incorporates a cellular barcode sequence (different for each bead) into nascent cDNA. After downstream high throughput sequencing, reads can be assigned to individual cells by barcode sequences, thereby enabling expression quantification at single cell resolution. As the cellular barcode is introduced downstream of transcript polyA tails, sequence data is typically restricted to transcript regions immediately proximal to 3′ termini. Because the S703I site is located at position 2402 from 5′ transcript start and 2690 from the 3′ transcript polyA tail, it is not accessible by standard droplet microfluidics single cell RNA-Seq platforms. Therefore, we adapted the inDrops method by generating custom barcode microbeads containing JAK1-specific primers (flanking the S703I site) in addition to polyT primer sequences, enabling more efficient JAK1 target capture and access to the S703I site. Following within-droplet reverse transcription, second strand synthesis and in vitro transcription amplification, samples are split into two parallel library preparations: one for standard polyT-primed libraries, and one for JAK1-targeted libraries. During data processing of resulting high throughput sequencing data, sequence reads from both libraries are assigned to individual cells based on shared barcode sequences. A detailed description of this approach appears below.

JAK1-targeted hydrogel microbeadsBarcoding hydrogel beads were prepared according to established inDrops protocol (Zilionis et al., 2017) with modified primers (Zilionis et al., 2019) with the following modifications. For the second round of split-and-pool primer extension for barcode synthesis, hydrogel beads were added to microplate wells (n = 384) containing oligonucleotide templates for both standard polyT primers and for an additional primer complementary to an S703I-adjacent region of the JAK1 transcript (11.53uM polyT oligonucleotide template, 2.3uM JAK1 oligonucleotide template) Within a given well, both polyT and JAK1 oligonucleotide templates carried identical “barcode 2” sequences, ensuring that extended primers on hydrogels contained matching barcodes. Subsequent exonuclease processing steps included corresponding JAK1 primer complementary blocking oligonucleotides.

JAK1-targeted hydrogel microbeads

Barcoding hydrogel beads were prepared according to established inDrops protocol (Zilionis et al., 2017) with modified primers (Zilionis et al., 2019) with the following modifications. For the second round of split-and-pool primer extension for barcode synthesis, hydrogel beads were added to microplate wells (n = 384) containing oligonucleotide templates for both standard polyT primers and for an additional primer complementary to an S703I-adjacent region of the JAK1 transcript (11.53uM polyT oligonucleotide template, 2.3uM JAK1 oligonucleotide template) Within a given well, both polyT and JAK1 oligonucleotide templates carried identical “barcode 2” sequences, ensuring that extended primers on hydrogels contained matching barcodes. Subsequent exonuclease processing steps included corresponding JAK1 primer complementary blocking oligonucleotides.

JAK1-targeted library preparationFreshly isolated PBMC were co-encapsulated with JAK1-targeted hydrogel beads. The standard inDrops protocol was followed for reverse transcription, droplet breakage, second strand synthesis and in vitro transcription (IVT) amplification. IVT reactions (20 ul) were then split to two parallel library preparations: 10 ul of IVT product was prepped for polyT-primed gene expression libraries according to the standard inDrops protocol, and 10 ul of IVT product (typically reserved as a “backup” aliquot) was prepped according to a JAK1-targeted protocol as follows. IVT products were reverse transcribed by SuperScript III with a JAK1-specific primer (with 5′ extension containing Illumina adaptor sequence) flanking the S703I site (55C for 1 hr, 70C for 15 min). RT reactions were treated with RNase H (37C for 30 min, 65C for 20 min) to remove RNA template from cDNA heteroduplexes. Following purification on 1.5X Ampure XP beads (Beckman Coulter), JAK1-enriched cDNA was amplified by PCR with Illumina-adapted inDrops primers (KAPA HiFi Master Mix; 2 cycles 98C x 20 s, 55C x 30 s, 72C for 40 s; 16-18 cycles 98C x 20 s, 65C x 30 s, 72C for 40 s; final 72C extension x 5 min). Final JAK1-targeted libraries were purified on 0.8X Ampure XP beads.

JAK1-targeted library preparation

Freshly isolated PBMC were co-encapsulated with JAK1-targeted hydrogel beads. The standard inDrops protocol was followed for reverse transcription, droplet breakage, second strand synthesis and in vitro transcription (IVT) amplification. IVT reactions (20 ul) were then split to two parallel library preparations: 10 ul of IVT product was prepped for polyT-primed gene expression libraries according to the standard inDrops protocol, and 10 ul of IVT product (typically reserved as a “backup” aliquot) was prepped according to a JAK1-targeted protocol as follows. IVT products were reverse transcribed by SuperScript III with a JAK1-specific primer (with 5′ extension containing Illumina adaptor sequence) flanking the S703I site (55C for 1 hr, 70C for 15 min). RT reactions were treated with RNase H (37C for 30 min, 65C for 20 min) to remove RNA template from cDNA heteroduplexes. Following purification on 1.5X Ampure XP beads (Beckman Coulter), JAK1-enriched cDNA was amplified by PCR with Illumina-adapted inDrops primers (KAPA HiFi Master Mix; 2 cycles 98C x 20 s, 55C x 30 s, 72C for 40 s; 16-18 cycles 98C x 20 s, 65C x 30 s, 72C for 40 s; final 72C extension x 5 min). Final JAK1-targeted libraries were purified on 0.8X Ampure XP beads.

Oligonucleotide sequencespolyT oligonucleotide template 5′-BAAAAAAAAAAAAAAAAAAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG – 3′JAK1 oligonucleotide template 5′ - GAGTGTGGCCCATTCATCAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG −3′JAK1 blocking oligonucleotide 5′-GAGTGTGGCCCATTCATCAA-3′Final “on bead” polyT primer sequence 5′- CGATGACGTAATACGACTCACTATAGGGTGTCGGGTGCAG[bc1,8nt]GTCTCGTGGGCTCGGAGATGTGTA TAAGAGACAG[bc2,8nt]NNNNNNTTTTTTTTTTTTTTTTTTTV-3′Final “on bead” JAK1 primer sequence 5′- CGATGACGTAATACGACTCACTATAGGGTGTCGGGTGCAG[bc1,8nt]GTCTCGTGGGCTCGGAGATGTGTA TAAGAGACAG[bc2,8nt]NNNNNNTTGATGAATGGGCCACACTC-3;Second reverse transcription JAK1 primer: 5′- TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGCCATGGAAATTCAAAGTTGCCAAACAG-3′

Oligonucleotide sequences

polyT oligonucleotide template 5′-BAAAAAAAAAAAAAAAAAAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG – 3′

JAK1 oligonucleotide template 5′ - GAGTGTGGCCCATTCATCAANNNNNN [bc2, 8nt] CTGTCTCTTATACACATCTCCGAGCCCACG −3′

JAK1 blocking oligonucleotide 5′-GAGTGTGGCCCATTCATCAA-3′

Final “on bead” polyT primer sequence 5′- CGATGACGTAATACGACTCACTATAGGGTGTCGGGTGCAG[bc1,8nt]GTCTCGTGGGCTCGGAGATGTGTA TAAGAGACAG[bc2,8nt]NNNNNNTTTTTTTTTTTTTTTTTTTV-3′

Final “on bead” JAK1 primer sequence 5′- CGATGACGTAATACGACTCACTATAGGGTGTCGGGTGCAG[bc1,8nt]GTCTCGTGGGCTCGGAGATGTGTA TAAGAGACAG[bc2,8nt]NNNNNNTTGATGAATGGGCCACACTC-3;

Second reverse transcription JAK1 primer: 5′- TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGCCATGGAAATTCAAAGTTGCCAAACAG-3′

High Throughput SequencingBoth standard polyT-primed libraries and JAK1-targeted libraries were pooled together and sequenced in multiplex on the Illumina NextSeq 500 platform with 75-cycle reagent kits in paired-end, dual index configuration:Read 1 containing transcript/JAK1 data: 61 cyclesi7 read containing cell barcode data: 8 cyclesi5 read containing sample index data: 8 cyclesRead 2 containing cell barcode and unique molecular identified (UMI) data: 14 cycles

High Throughput Sequencing

Both standard polyT-primed libraries and JAK1-targeted libraries were pooled together and sequenced in multiplex on the Illumina NextSeq 500 platform with 75-cycle reagent kits in paired-end, dual index configuration:Read 1 containing transcript/JAK1 data: 61 cyclesi7 read containing cell barcode data: 8 cyclesi5 read containing sample index data: 8 cyclesRead 2 containing cell barcode and unique molecular identified (UMI) data: 14 cycles

Read 1 containing transcript/JAK1 data: 61 cycles

i7 read containing cell barcode data: 8 cycles

i5 read containing sample index data: 8 cycles

Read 2 containing cell barcode and unique molecular identified (UMI) data: 14 cycles

Data ProcessingFASTQ sequence files were processed with the inDrops.py workflow script (v0.3, https://github.com/inDrops/inDrops). For polyT-primed gene expression libraries, resulting gene x cell matrices and per cell read counts were used for downstream analyses in Seurat (details below), using only “cell-containing droplets” defined by a read count higher than 8000-10000 depending on read count evaluation from droplets in the corresponding library batch.For JAK1-targeted libraries, BAM files (appended with cellular barcode and UMI data) were used to evaluate genotypes at single cell resolution as follows. Reads covering the JAK1 S703I site (chr1:64845519 – 64845521 on the minus strand; human genome reference GRCh38) containing the wild-type sequence (GAT) or S703I (GCT) sequence were quantified per UMI per cellular barcode. If at least 2 and at least 90% of the reads for a given cellular barcode/UMI combination contained the same sequence (wild-type or S703I), then this combination was designated a JAK1 transcript of the given genotype and assigned to the appropriate cellular barcode.We next used per cell JAK1 WT and S703I transcript counts to assign putative genotypes to individual cells. As cell free RNA in suspension can co-encapsulate with cells thereby generating unwanted background signal in droplet microfluidics scRNA-Seq methods, we aimed to apply a stringent, data driven threshold for genotyping assignment. To evaluate the potential influence of cell free RNA, we quantified the frequency of JAK1 transcripts detected in empty (i.e., cell free) droplets (defined as barcodes with 1000 – 2000 reads in polyT-primed libraries) using the same barcode/UMI strategy described above. We found that more than 99% of empty droplets had less than 3 JAK1 transcripts. Guided by these data, cells were only considered for JAK1 genotype assignment if at least 5 JAK1 transcripts were detected. For JAK1 genotype assignment, individual cells were classified in one of the following four categories: cells that carried the S703I allele were classified as “MUT+” regardless of carrying the wild-type allele; the remainder (S703I allele negative) cells that carried the wild-type allele were classified as “WT+MUTneg” if no S703I transcripts were detected, and as “WT+MUTnonZero” if 1-to-4 S703I transcripts were detected; cells that carried neither the wild-type nor the S703I allele were classified as “WTnegMUTneg.”

Data Processing

FASTQ sequence files were processed with the inDrops.py workflow script (v0.3, https://github.com/inDrops/inDrops). For polyT-primed gene expression libraries, resulting gene x cell matrices and per cell read counts were used for downstream analyses in Seurat (details below), using only “cell-containing droplets” defined by a read count higher than 8000-10000 depending on read count evaluation from droplets in the corresponding library batch.

For JAK1-targeted libraries, BAM files (appended with cellular barcode and UMI data) were used to evaluate genotypes at single cell resolution as follows. Reads covering the JAK1 S703I site (chr1:64845519 – 64845521 on the minus strand; human genome reference GRCh38) containing the wild-type sequence (GAT) or S703I (GCT) sequence were quantified per UMI per cellular barcode. If at least 2 and at least 90% of the reads for a given cellular barcode/UMI combination contained the same sequence (wild-type or S703I), then this combination was designated a JAK1 transcript of the given genotype and assigned to the appropriate cellular barcode.

We next used per cell JAK1 WT and S703I transcript counts to assign putative genotypes to individual cells. As cell free RNA in suspension can co-encapsulate with cells thereby generating unwanted background signal in droplet microfluidics scRNA-Seq methods, we aimed to apply a stringent, data driven threshold for genotyping assignment. To evaluate the potential influence of cell free RNA, we quantified the frequency of JAK1 transcripts detected in empty (i.e., cell free) droplets (defined as barcodes with 1000 – 2000 reads in polyT-primed libraries) using the same barcode/UMI strategy described above. We found that more than 99% of empty droplets had less than 3 JAK1 transcripts. Guided by these data, cells were only considered for JAK1 genotype assignment if at least 5 JAK1 transcripts were detected. For JAK1 genotype assignment, individual cells were classified in one of the following four categories: cells that carried the S703I allele were classified as “MUT+” regardless of carrying the wild-type allele; the remainder (S703I allele negative) cells that carried the wild-type allele were classified as “WT+MUTneg” if no S703I transcripts were detected, and as “WT+MUTnonZero” if 1-to-4 S703I transcripts were detected; cells that carried neither the wild-type nor the S703I allele were classified as “WTnegMUTneg.”

Single cell RNA-Seq Gene Expression AnalysisData from JAK1-targeted inDrops single cell RNA-Seq data were analyzed in conjunction with corresponding single cell RNA-Seq data acquired from the same PBMC specimen on the 10X Genomics Chromium platform. Sequence reads from 10X libraries were processed with the CellRanger software package (10X Genomics) using default parameters. Gene x cell matrices from both methods were further analyzed with Seurat (v2.3.4) (Butler et al., 2018) in the R statistical framework as follows. For applicable cells in the JAK1-targeted inDrops dataset, JAK1 genotyping information was imported as per cell metadata entries. Genes with detectable expression in fewer than 5 cells per sample were excluded. Cells with detectable expression of fewer than 250 genes or fewer than 1000 UMI counts or greater than 20% UMIs from mitochondrial gene transcripts (measure of cell viability) were removed from further analysis. Gene expression data were log normalized and scaled (regressing out effects based on total UMI counts and mitochondrial gene expression). All genes with detectable expression in both inDrops and 10X datasets were used for CCA dimensionality reduction (20 dimensions). CCA dimension scores were aligned between datasets using the AlignSubspace function. Aligned data were log-normalized and scaled as above. Graph- based clustering and tSNE visualization was performed with the FindClusters and RunTSNE functions with default settings (using the first 19 CCA dimensions, based on examination of distinct expression patterns with the DimHeatmap function). Marker genes for individual clusters were identified with the FindMarkers() function in Seurat (paramenters: min.pct = 0.1, only.pos = TRUE, logfc.threshold = log(1.5), min.diff.pct = 0.1, pseudocount.use = 0.1). Briefly, up to 100 cells were randomly selected from each cluster, and each cluster was contrasted against all others for differential gene expression. Significant genes (p < 0.05) were ranked by log2 fold-change; the top 25 genes for each cluster, as well as a select list of additional genes of interest, were selected as marker genes for supervised cluster annotation (Table S3). Expression patterns were further evaluated on tSNE plots. Select clusters were merged based on shared expression patterns as indicated.Gene expression differences were assessed for cells assigned JAK1 genotype classifications of either MUT+ (S703I) or WT+MUTneg (WT). Plasmablasts and dendritic cells were excluded from analysis due to insufficient cells assigned to both genotype groups. Apparent cellular doublets and cells in the platelet cluster were also excluded. Differential gene expression testing was performed on genes detected in at least 20% of cells in any cell type cluster (of either JAK1 genoytpe group) with edgeR (v.3.12.1) (Robinson et al., 2010), including modifications included for single cell RNA-Seq data as described (Soneson and Robinson, 2018). A linear model including factors for cellular gene detection rate, cell type, JAK1 genotype group and an interaction term (cell type cluster and JAK1 genotype group), was fit with the glmQLFit function. Differential gene expression testing (WT versus S703I) was tested across all cell types with the glmQLFtest function, with significance thresholds set at FDR 0.05. In addition, gene set enrichment testing for Molecular Signatures Database Collections H, C2- CPC, C3-TFT and C7 collections (http://software.broadinstitute.org/gsea/msigdb), supplemented with two additional interferon stimulated gene sets (Rosenberg et al., 2018) was performed with CAMERA (Wu and Smyth, 2012) and the linear model described above. Only gene sets with 5 or more genes in the fitted model were included in the analysis.

Single cell RNA-Seq Gene Expression Analysis

Data from JAK1-targeted inDrops single cell RNA-Seq data were analyzed in conjunction with corresponding single cell RNA-Seq data acquired from the same PBMC specimen on the 10X Genomics Chromium platform. Sequence reads from 10X libraries were processed with the CellRanger software package (10X Genomics) using default parameters. Gene x cell matrices from both methods were further analyzed with Seurat (v2.3.4) (Butler et al., 2018) in the R statistical framework as follows. For applicable cells in the JAK1-targeted inDrops dataset, JAK1 genotyping information was imported as per cell metadata entries. Genes with detectable expression in fewer than 5 cells per sample were excluded. Cells with detectable expression of fewer than 250 genes or fewer than 1000 UMI counts or greater than 20% UMIs from mitochondrial gene transcripts (measure of cell viability) were removed from further analysis. Gene expression data were log normalized and scaled (regressing out effects based on total UMI counts and mitochondrial gene expression). All genes with detectable expression in both inDrops and 10X datasets were used for CCA dimensionality reduction (20 dimensions). CCA dimension scores were aligned between datasets using the AlignSubspace function. Aligned data were log-normalized and scaled as above. Graph- based clustering and tSNE visualization was performed with the FindClusters and RunTSNE functions with default settings (using the first 19 CCA dimensions, based on examination of distinct expression patterns with the DimHeatmap function). Marker genes for individual clusters were identified with the FindMarkers() function in Seurat (paramenters: min.pct = 0.1, only.pos = TRUE, logfc.threshold = log(1.5), min.diff.pct = 0.1, pseudocount.use = 0.1). Briefly, up to 100 cells were randomly selected from each cluster, and each cluster was contrasted against all others for differential gene expression. Significant genes (p < 0.05) were ranked by log2 fold-change; the top 25 genes for each cluster, as well as a select list of additional genes of interest, were selected as marker genes for supervised cluster annotation (Table S3). Expression patterns were further evaluated on tSNE plots. Select clusters were merged based on shared expression patterns as indicated.

Gene expression differences were assessed for cells assigned JAK1 genotype classifications of either MUT+ (S703I) or WT+MUTneg (WT). Plasmablasts and dendritic cells were excluded from analysis due to insufficient cells assigned to both genotype groups. Apparent cellular doublets and cells in the platelet cluster were also excluded. Differential gene expression testing was performed on genes detected in at least 20% of cells in any cell type cluster (of either JAK1 genoytpe group) with edgeR (v.3.12.1) (Robinson et al., 2010), including modifications included for single cell RNA-Seq data as described (Soneson and Robinson, 2018). A linear model including factors for cellular gene detection rate, cell type, JAK1 genotype group and an interaction term (cell type cluster and JAK1 genotype group), was fit with the glmQLFit function. Differential gene expression testing (WT versus S703I) was tested across all cell types with the glmQLFtest function, with significance thresholds set at FDR 0.05. In addition, gene set enrichment testing for Molecular Signatures Database Collections H, C2- CPC, C3-TFT and C7 collections (http://software.broadinstitute.org/gsea/msigdb), supplemented with two additional interferon stimulated gene sets (Rosenberg et al., 2018) was performed with CAMERA (Wu and Smyth, 2012) and the linear model described above. Only gene sets with 5 or more genes in the fitted model were included in the analysis.

Single Cell qPCR Transcript GenotypingPBMCs from a healthy donor carrying a heterozygous SNP in JAK1 (rs2230587) were isolated by Ficoll gradient. Cells were stained with antibodies against CD3, CD19, CD14 and CD56 (Biolegend), and 100 single cells were FACS-sorted into single cell lysis buffer (Ambion 4458235). Following DNase treatment, cDNA was generated using SuperScript VILO RT kit (ThermoFisher 11754050). A linear preamplification was then performed using JAK1 primers (gtcctctggatctcttcatgca, gctgtttggcaactttgaatttcc) and primers to a negative control gene NACA (cccaggcaaccacacaac, ccgactctgttttgctttactgact). Using qPCR with custom TaqMan genotyping primers (above) and allele-specific probes (JAK1 aaggacatc[g/a]cttttc; agcagctgaaat[T/C]gatgaa) that were individually fluorescently tagged (VIC and FAM), allelic ratios were determined by endpoint genotyping. Quantification was carried out by interpolation from a standard curve of oligonucleotide standards.

Single Cell qPCR Transcript Genotyping

PBMCs from a healthy donor carrying a heterozygous SNP in JAK1 (rs2230587) were isolated by Ficoll gradient. Cells were stained with antibodies against CD3, CD19, CD14 and CD56 (Biolegend), and 100 single cells were FACS-sorted into single cell lysis buffer (Ambion 4458235). Following DNase treatment, cDNA was generated using SuperScript VILO RT kit (ThermoFisher 11754050). A linear preamplification was then performed using JAK1 primers (gtcctctggatctcttcatgca, gctgtttggcaactttgaatttcc) and primers to a negative control gene NACA (cccaggcaaccacacaac, ccgactctgttttgctttactgact). Using qPCR with custom TaqMan genotyping primers (above) and allele-specific probes (JAK1 aaggacatc[g/a]cttttc; agcagctgaaat[T/C]gatgaa) that were individually fluorescently tagged (VIC and FAM), allelic ratios were determined by endpoint genotyping. Quantification was carried out by interpolation from a standard curve of oligonucleotide standards.

Quantification and Statistical AnalysisData are presented as the mean ± standard deviation (SD) or standard error of the mean (SEM), as indicated in the legend for each figure. Statistical parameters including the methods implemented, corrections for multiple comparisons, exact values of n, identity of replicates, definitions of center and dispersion and statistical significance are reported in the Figure Legends when necessary. Statistical testing for the analysis of single cell RNA sequencing is discussed in the corresponding section of the Method Details. All statistical tests were calculated in the R statistical framework or GraphPad PRISM.

Quantification and Statistical Analysis

Data are presented as the mean ± standard deviation (SD) or standard error of the mean (SEM), as indicated in the legend for each figure. Statistical parameters including the methods implemented, corrections for multiple comparisons, exact values of n, identity of replicates, definitions of center and dispersion and statistical significance are reported in the Figure Legends when necessary. Statistical testing for the analysis of single cell RNA sequencing is discussed in the corresponding section of the Method Details. All statistical tests were calculated in the R statistical framework or GraphPad PRISM.
