---
pmid: '15358143'
title: Nucleocapsid protein of SARS coronavirus tightly binds to human cyclophilin
  A.
authors:
- Luo C
- Luo H
- Zheng S
- Gui C
- Yue L
- Yu C
- Sun T
- He P
- Chen J
- Shen J
- Luo X
- Li Y
- Liu H
- Bai D
- Shen J
- Yang Y
- Li F
- Zuo J
- Hilgenfeld R
- Pei G
- Chen K
- Shen X
- Jiang H
journal: Biochem Biophys Res Commun
year: '2004'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC7092810
doi: 10.1016/j.bbrc.2004.07.003
---

# Nucleocapsid protein of SARS coronavirus tightly binds to human cyclophilin A.
**Authors:** Luo C, Luo H, Zheng S, Gui C, Yue L, Yu C, Sun T, He P, Chen J, Shen J, Luo X, Li Y, Liu H, Bai D, Shen J, Yang Y, Li F, Zuo J, Hilgenfeld R, Pei G, Chen K, Shen X, Jiang H
**Journal:** Biochem Biophys Res Commun (2004)
**DOI:** [10.1016/j.bbrc.2004.07.003](https://doi.org/10.1016/j.bbrc.2004.07.003)
**PMC:** [PMC7092810](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7092810/)

## Abstract

1. Biochem Biophys Res Commun. 2004 Aug 27;321(3):557-65. doi: 
10.1016/j.bbrc.2004.07.003.

Nucleocapsid protein of SARS coronavirus tightly binds to human cyclophilin A.

Luo C(1), Luo H, Zheng S, Gui C, Yue L, Yu C, Sun T, He P, Chen J, Shen J, Luo 
X, Li Y, Liu H, Bai D, Shen J, Yang Y, Li F, Zuo J, Hilgenfeld R, Pei G, Chen K, 
Shen X, Jiang H.

Author information:
(1)Institute of Materia Medica, Shanghai Institutes for Biological Sciences, 
Chinese Academy of Sciences, Shanghai 201203, China.

Severe acute respiratory syndrome coronavirus (SARS-CoV) is responsible for SARS 
infection. Nucleocapsid protein (NP) of SARS-CoV (SARS_NP) functions in 
enveloping the entire genomic RNA and interacts with viron structural proteins, 
thus playing important roles in the process of virus particle assembly and 
release. Protein-protein interaction analysis using bioinformatics tools 
indicated that SARS_NP may bind to human cyclophilin A (hCypA), and surface 
plasmon resonance (SPR) technology revealed this binding with the equilibrium 
dissociation constant ranging from 6 to 160nM. The probable binding sites of 
these two proteins were detected by modeling the three-dimensional structure of 
the SARS_NP-hCypA complex, from which the important interaction residue pairs 
between the proteins were deduced. Mutagenesis experiments were carried out for 
validating the binding model, whose correctness was assessed by the observed 
effects on the binding affinities between the proteins. The reliability of the 
binding sites derived by the molecular modeling was confirmed by the fact that 
the computationally predicted values of the relative free energies of the 
binding for SARS_NP (or hCypA) mutants to the wild-type hCypA (or SARS_NP) are 
in good agreement with the data determined by SPR. Such presently observed 
SARS_NP-hCypA interaction model might provide a new hint for facilitating the 
understanding of another possible SARS-CoV infection pathway against human cell.

DOI: 10.1016/j.bbrc.2004.07.003
PMCID: PMC7092810
PMID: 15358143 [Indexed for MEDLINE]

## Full Text

Between the end of 2002 and July 2003, a new epidemic disease called severe acute respiratory syndrome (SARS) affected more than 30 countries [1], [2], [3]. A new coronavirus, SARS coronavirus (SARS-CoV), was then identified to be responsible for the infection [4], [5], [6]. Following this, remarkable achievements have been made in genome sequencing of SARS-CoV [7], [8], SARS protein functional studies [9], [10], [11], three-dimensional structure elucidation of SARS proteins, by both modeling [12], [13], [14] and X-ray crystallography [15], clinical studies [16], virus entrance mechanisms [17], [18], and anti-SARS drug discovery [12], [19], [20], [21]. Although the SARS pandemic has been controlled by now, many scientific questions remain to be answered, such as which specific features of the virus are responsible for its pathogenicity, and which mechanisms are at work during infection. Furthermore, no vaccines or drugs are presently available for the prevention or therapy of SARS. Therefore, probing the infection mechanism at the molecular level is of utmost significance, since it is the foundation for discovering and developing anti-SARS drugs or vaccines.

Mapping virus–host protein interactions can provide important clues on the initial stages of infection. Genomic sequence analyses revealed the genome organization and phylogeny of the SARS coronavirus [7], [8]. The SARS-CoV genome contains 11 major open reading frames (ORFs) that encode, among others, the replicase polyprotein, the spike (S) protein, the small envelope (E) protein, the membrane (M) protein, and the nucleocapsid (N) protein [7], [8]. For coronaviruses, the N protein (NP) plays an important role during host cell entry and virus particle assembly and release [22], [23], [24]. In particular, NP binds to a defined packaging signal on viral RNA, leading to the formation of the helical nucleocapsid [23]. A similar function of NP has been found in other viruses, in addition to coronaviruses [25]. However, the mechanism by which NP facilitates virus replication is not established.

To gain insight into the function of SARS_NP in infection, we have used a bioinformatics approach to analyze potential interactions between SARS_NP and the proteins encoded in the human genome. We find strong evidence for an interaction between the SARS_NP and human cyclophilin A. Cyclophilins were discovered originally for their high affinity against cyclosporin A (CsA), an immunosuppressive drug used to prevent allograft rejection [26]. More recently, cyclophilins have been shown to play an important role in HIV infection. The Gag polyprotein of human immunodeficiency virus type 1 (HIV-1) binds most members of the cyclophilin family of peptidyl-peptide-prolyl isomerases [27]. However, of the 15 known human cyclophilins, only human cyclophilin A (CypA) is integrated inside the viral core of HIV-1 by interacting specifically with the capsid domain (CA) of the Gag polyprotein [28]. CypA subsequently performs an essential function in HIV-1 replication. The X-ray crystal structure of the complex between human CypA (hCypA) and the N-terminal domain of CA (residues 1–151) revealed the binding mode of these two proteins and provided a clear picture of the role that CypA plays in disassembly of the viral core [29]. Moreover, it has been shown that maturation in the presence of CsA during virion budding causes a quantitative reduction in viral infectivity [30].

Although no capsid protein is encoded in SARS-CoV genome [7], [8], sequence alignment indicates that the segment Val235-Pro369 of SARS_NP is homologous to HIV-1 CA with a sequence identity of 25.1% and a similarity of 36.7%. Encouraged by the evidence from the bioinformatics analysis and the knowledge of the role of CypA in HIV-1 infection, we presumed that SARS_NP may function just like the HIV-1 viral capsid domain, i.e., bind to human CypA (hCypA) during virus replication.

We have verified the suspected SARS_NP–hCypA interaction by using surface plasmon resonance (SPR) biosensor technology, and the probable binding sites of these two proteins were detected by modeling the three-dimensional (3D) structure of the SARS_NP–hCypA complex, from which the important interaction residue pairs between the proteins were deduced. To validate the binding model, which involves the peptidyl-prolyl isomerase active site of hCypA, mutagenesis experiments were carried out with the two proteins, and the observed effects on the binding affinities were used to assess the correctness of the interaction model. The computationally predicted values of the relative free energies of binding of SARS_NP (or hCypA) mutants to the wild-type hCypA (or SARS_NP) are in good agreement with the data determined by SPR, indicating that the binding sites derived by molecular modeling are reliable. This finding may provide a new approach for discovering anti-SARS-CoV agents.

Materials and methods
Protein–protein interaction analysis by bioinformatics. A new method developed in our laboratory was applied to map the binding proteins of SARS_NP. In brief, the procedure of this method comprised the following steps: (1) homology search was performed to identify the homologous proteins of SARS_NP. These proteins were selected as queries for searching the protein–protein interaction database DIP (http://www.DIP.org). (2) Proteins in the DIP database that possibly interact with SARS_NP homologous proteins were selected as the candidates of the SARS_NP binding proteins. (3) PFAM database (http://www.sanger.ac.uk/Software/Pfam/) was searched to obtain the domain distributions of the binding protein candidates. (4) Keywords search was performed on the PFAM and Interpro databases (http://www.ebi.ac.uk/interpro/) to annotate the biological functions of the SARS_NP binding protein candidates. Proteins with functions related to immunosuppression, immunoregulation, and virus infection were picked out for further analysis and biological assay.
Chemicals and enzymes. The restriction and modifying enzymes in this work were purchased from TaKaRa. The bacterial strains M15 and DH5α, and the vector pQE30 were from Qiagen. TRIzol and Superscript II reverse transcriptase were purchased from Gibco. The chelating affinity column and lower molecular weight (LMW) marker were purchased from Amersham–Pharmacia Biotech. Isopropyl β-d-thiogalactoside (IPTG) was purchased from Promega. All other chemicals were from Sigma in their analytical grade.
Plasmids and site-directed mutagenesis. All the recombinant DNA methods including PCR, restriction digestion, ligation, Escherichia coli transformation, and plasmid DNA preparation were performed according to the standard methods [31].SARS-CoV (isolate BJ01) RNA was extracted with TRIzol reagent according to the manufacturer’s instruction (www.genehub.net/trizol.htm). The reverse transcription was performed with the random priming method by the Superscript II reverse transcriptase. The SARS_NP cDNA was subsequently amplified by PCR, using the following primers: Fw, 5′ATTAGGATCCTCTGATAATGGACCCCAATCA3′; and Rv, 5′TTAAGTCGACTGCCTGAGTTGAATCAGCAGA 3′. After digestion with BamHI and SalI, the PCR product was inserted into the BamHI and SalI sites of the vector pQE30 to construct the plasmid pQE30-SARS_NP. The SARS_NP insert was verified by sequencing.The site-directed mutational plasmids pQE30-SARS_NP-Q307A, pQE30-SARS_NP-Q307D, pQE30-SARS_NP-W302A, and pQE30-SARS_NP-I305A were obtained according to the Quick-change Mutagensis (Stratagene).The plasmid pQE30-CypA was kindly provided by the Center of Medical Laboratory Sciences, Nanjing PLA Hospital, Nanjing 210002, China. The site-directed mutational plasmids pQE30-CypA-R55A, pQE30-CypA-R55D, pQE30-CypA-Q63A, pQE30-CypA-W121F, pQE30-CypA-R69A, and pQE30-CypA-R69K were obtained according to the Quick-change Mutagensis (Stratagene).
Expression and purification of SARS_NP, hCypA, and mutants. SARS_NP wild-type protein was expressed and purified based on the following procedure. The recombinant plasmid pQE30-SARS_NP was transfected into M15 (DE3) bacterial strain. Clones were grown overnight in LB medium containing 100 mg/L ampicillin and 25 mg/L kanamycin. Expression of the His-tagged recombinant protein was induced at an OD600 of 0.7–0.9 with the addition of isopropyl-β-d-thiogalactopyranoside (IPTG) to a final concentration of 1 mM. After induction for 12 h at 37 °C, the cells were harvested by centrifugation for 30 min at 4000 r/m, 4 °C and stored at −70 °C. During the protein purification, cells were resuspended in 20 ml buffer A (20 mM Tris–HCl pH 8.0, 500 mM NaCl, 5 mM imidazole, and 1 mM PMSF) and then lysed by sonication for 15 min in icy bath. The lysate was cleared by centrifugation at 4 °C, 14,000 r/m for 60 min. To the supernatant were added DNAase and RNAase to the final concentration of 2 μg/ml for either of them in order to get rid of the possible bound DNA and RNA scraps to the protein of interest. This mixture was incubated at 20 °C for 3 h before further treatments.The nuclease treated supernatant was loaded on a 4 ml Sepharose Ni–NTA column (Amersham–Pharmacia) equilibrated with 30 ml buffer A. The column was eluted with buffer B (20 mM Tris–HCl, pH 8.0, 500 mM NaCl, and 500 mM imidazole) after being washed by buffer C (20 mM Tris–HCl, pH 8.0, 500 mM NaCl, and 120 mM imidazole). The elution fraction was further applied to a gel filtration column (Hiprep 16/60 Sephacyl S100) on a FPLC (Amersham–Pharmacia) system.The expression and purification of the SARS_NP mutants (SARS_NP-Q307A, SARS_NP-Q307D, SARS_NP-W302A, and SARS_NP-I305A) in this work were carried out using the similar procedures for the SARS_NP wild-type protein.The procedures of expression and purification for hCypA wild-type and its mutants hCypA-R55A, hCypA-R55D, hCypA-Q63A, hCypA-W121F, hCypA-R69A, and hCypA-R69K are similar to those for NP wild-type and its mutants except that for hCypA wild-type and its mutants: (1) the expression temperature is 25 °C; (2) the concentration of IPTG is 0.5 mM; and (3) the expression time is 7 h.
Protein–protein binding assay. The binding affinities of protein–protein interactions (hCypA to SARS_NP and SARS_NP mutant proteins, and SARS_NP to hCypA and hCypA mutant proteins) were determined emplying SPR technology based Biacore 3000 instrument (Biacore AB, Rapsgatan 7, S-754 50 Uppsala, Sweden). Immobilization of the protein to the CM5 sensor chip (Biacore) was carried out by the standard primary amine coupling reaction. The protein to be covalently bound to the matrix was diluted in 10 mM sodium acetate buffer (pH 4.3) to a final concentration of 0.35 mg/ml. Equilibration of the baseline was completed by a continuous flow of HBS-EP running buffer (10 mM Hepes, 150 mM NaCl, 3.4 mM EDTA, and 0.005% (v/v) surfactant P20, pH 7.4) through the chip for 1–2 h. All the Biacore data were collected at 25 °C with HBS-EP as running buffer at a constant flow of 20 μl/min. All the sensorgrams were processed by using automatic correction for non-specific bulk refractive index effects. Langmuir binding fitting model was used for estimating the K
D values, in which the association rate constant (k
on) and dissociation rate constant (k
off) are fitted simultaneously by rate Eq. (1),(1)dRdt=kon×C×(Rmax-R)-koff×R,where R represents the response unit, C is the concentration of the ligands, and(2)KD=koff/kon.

Molecular modeling and free energy calculation. Based on the sequence alignment (Fig. 2A), the 3D model of the segment Val235-Pro369 of SARS_NP was constructed by using Homology module of Insight II (Insight II, 2000) and the MODELLER program [32] taking the 3D structure of capsid isolated from the X-ray crystal structure of CA151–hCypA complex (PDB entry 1AK4) as a template. After structural optimization, the 3D model of segment Val235-Pro369 was manually docked into the active site groove of the X-ray crystal structure of hCypA by taking loop Trp302-Pro310 as the SARS_NP binding loop. The entire complex structure was then optimized by using molecular mechanics method with AMBER force field and Kollman-all-Atom charges [33].Complexes of wild-type hCypA (or SARS_NP) with mutated SARS_NP (or hCypA) were constructed based on the 3D model of SARS_NP–hCypA complex (Fig. 2B) by using Biopolymer module encoded in Sybyl 6.8 [34]. These complex models were solvated by a 28 Å TIP3P [35] water sphere. Then the solvated models were equilibrated for 300 ps at 300 K by using molecular dynamics (MD) simulation, and a 100 ps MD simulation was run on each model system, sampling the conformations for binding free energy calculation. All MD simulations were carried out by using the AMBER 7.0 program with AMBER force field (parm98) [33]. The MM/PBSA method [36] was used to evaluate the binding free energies of SARS_NP–hCypA complexes. In general, the average binding free energy of protein–protein interaction (averaged over 50 samples isolated from the MD trajectory), ΔG
bind, is calculated by Eq. (3):(3)ΔGbind=Gcomplex-Gprotein1-Gprotein2=ΔEMM+ΔGPB+ΔGNP,where ΔE
MM is the interaction energy of the two proteins calculated by the AMBER force field, including internal, electrostatic, and van der Waals components; ΔG
PB is the polar solvation contribution calculated by using a finite-difference Poisson–Boltzmann (PB) model [37], and ΔG
NP is the non-polar solvation energy, which is obtained from the solvent accessible surface area (SASA). The relative binding free energy, ΔΔG
bind, is the difference between the binding free energy of a mutated SARS_NP (or hCypA) to wild-type hCypA (or SARS_NP) and that of wild-type SARS_NP to wild-type hCypA.

Materials and methods

Protein–protein interaction analysis by bioinformatics. A new method developed in our laboratory was applied to map the binding proteins of SARS_NP. In brief, the procedure of this method comprised the following steps: (1) homology search was performed to identify the homologous proteins of SARS_NP. These proteins were selected as queries for searching the protein–protein interaction database DIP (http://www.DIP.org). (2) Proteins in the DIP database that possibly interact with SARS_NP homologous proteins were selected as the candidates of the SARS_NP binding proteins. (3) PFAM database (http://www.sanger.ac.uk/Software/Pfam/) was searched to obtain the domain distributions of the binding protein candidates. (4) Keywords search was performed on the PFAM and Interpro databases (http://www.ebi.ac.uk/interpro/) to annotate the biological functions of the SARS_NP binding protein candidates. Proteins with functions related to immunosuppression, immunoregulation, and virus infection were picked out for further analysis and biological assay.

Chemicals and enzymes. The restriction and modifying enzymes in this work were purchased from TaKaRa. The bacterial strains M15 and DH5α, and the vector pQE30 were from Qiagen. TRIzol and Superscript II reverse transcriptase were purchased from Gibco. The chelating affinity column and lower molecular weight (LMW) marker were purchased from Amersham–Pharmacia Biotech. Isopropyl β-d-thiogalactoside (IPTG) was purchased from Promega. All other chemicals were from Sigma in their analytical grade.

Plasmids and site-directed mutagenesis. All the recombinant DNA methods including PCR, restriction digestion, ligation, Escherichia coli transformation, and plasmid DNA preparation were performed according to the standard methods [31].

SARS-CoV (isolate BJ01) RNA was extracted with TRIzol reagent according to the manufacturer’s instruction (www.genehub.net/trizol.htm). The reverse transcription was performed with the random priming method by the Superscript II reverse transcriptase. The SARS_NP cDNA was subsequently amplified by PCR, using the following primers: Fw, 5′ATTAGGATCCTCTGATAATGGACCCCAATCA3′; and Rv, 5′TTAAGTCGACTGCCTGAGTTGAATCAGCAGA 3′. After digestion with BamHI and SalI, the PCR product was inserted into the BamHI and SalI sites of the vector pQE30 to construct the plasmid pQE30-SARS_NP. The SARS_NP insert was verified by sequencing.

The site-directed mutational plasmids pQE30-SARS_NP-Q307A, pQE30-SARS_NP-Q307D, pQE30-SARS_NP-W302A, and pQE30-SARS_NP-I305A were obtained according to the Quick-change Mutagensis (Stratagene).

The plasmid pQE30-CypA was kindly provided by the Center of Medical Laboratory Sciences, Nanjing PLA Hospital, Nanjing 210002, China. The site-directed mutational plasmids pQE30-CypA-R55A, pQE30-CypA-R55D, pQE30-CypA-Q63A, pQE30-CypA-W121F, pQE30-CypA-R69A, and pQE30-CypA-R69K were obtained according to the Quick-change Mutagensis (Stratagene).

Expression and purification of SARS_NP, hCypA, and mutants. SARS_NP wild-type protein was expressed and purified based on the following procedure. The recombinant plasmid pQE30-SARS_NP was transfected into M15 (DE3) bacterial strain. Clones were grown overnight in LB medium containing 100 mg/L ampicillin and 25 mg/L kanamycin. Expression of the His-tagged recombinant protein was induced at an OD600 of 0.7–0.9 with the addition of isopropyl-β-d-thiogalactopyranoside (IPTG) to a final concentration of 1 mM. After induction for 12 h at 37 °C, the cells were harvested by centrifugation for 30 min at 4000 r/m, 4 °C and stored at −70 °C. During the protein purification, cells were resuspended in 20 ml buffer A (20 mM Tris–HCl pH 8.0, 500 mM NaCl, 5 mM imidazole, and 1 mM PMSF) and then lysed by sonication for 15 min in icy bath. The lysate was cleared by centrifugation at 4 °C, 14,000 r/m for 60 min. To the supernatant were added DNAase and RNAase to the final concentration of 2 μg/ml for either of them in order to get rid of the possible bound DNA and RNA scraps to the protein of interest. This mixture was incubated at 20 °C for 3 h before further treatments.

The nuclease treated supernatant was loaded on a 4 ml Sepharose Ni–NTA column (Amersham–Pharmacia) equilibrated with 30 ml buffer A. The column was eluted with buffer B (20 mM Tris–HCl, pH 8.0, 500 mM NaCl, and 500 mM imidazole) after being washed by buffer C (20 mM Tris–HCl, pH 8.0, 500 mM NaCl, and 120 mM imidazole). The elution fraction was further applied to a gel filtration column (Hiprep 16/60 Sephacyl S100) on a FPLC (Amersham–Pharmacia) system.

The expression and purification of the SARS_NP mutants (SARS_NP-Q307A, SARS_NP-Q307D, SARS_NP-W302A, and SARS_NP-I305A) in this work were carried out using the similar procedures for the SARS_NP wild-type protein.

The procedures of expression and purification for hCypA wild-type and its mutants hCypA-R55A, hCypA-R55D, hCypA-Q63A, hCypA-W121F, hCypA-R69A, and hCypA-R69K are similar to those for NP wild-type and its mutants except that for hCypA wild-type and its mutants: (1) the expression temperature is 25 °C; (2) the concentration of IPTG is 0.5 mM; and (3) the expression time is 7 h.

Protein–protein binding assay. The binding affinities of protein–protein interactions (hCypA to SARS_NP and SARS_NP mutant proteins, and SARS_NP to hCypA and hCypA mutant proteins) were determined emplying SPR technology based Biacore 3000 instrument (Biacore AB, Rapsgatan 7, S-754 50 Uppsala, Sweden). Immobilization of the protein to the CM5 sensor chip (Biacore) was carried out by the standard primary amine coupling reaction. The protein to be covalently bound to the matrix was diluted in 10 mM sodium acetate buffer (pH 4.3) to a final concentration of 0.35 mg/ml. Equilibration of the baseline was completed by a continuous flow of HBS-EP running buffer (10 mM Hepes, 150 mM NaCl, 3.4 mM EDTA, and 0.005% (v/v) surfactant P20, pH 7.4) through the chip for 1–2 h. All the Biacore data were collected at 25 °C with HBS-EP as running buffer at a constant flow of 20 μl/min. All the sensorgrams were processed by using automatic correction for non-specific bulk refractive index effects. Langmuir binding fitting model was used for estimating the K
D values, in which the association rate constant (k
on) and dissociation rate constant (k
off) are fitted simultaneously by rate Eq. (1),(1)dRdt=kon×C×(Rmax-R)-koff×R,where R represents the response unit, C is the concentration of the ligands, and(2)KD=koff/kon.

Molecular modeling and free energy calculation. Based on the sequence alignment (Fig. 2A), the 3D model of the segment Val235-Pro369 of SARS_NP was constructed by using Homology module of Insight II (Insight II, 2000) and the MODELLER program [32] taking the 3D structure of capsid isolated from the X-ray crystal structure of CA151–hCypA complex (PDB entry 1AK4) as a template. After structural optimization, the 3D model of segment Val235-Pro369 was manually docked into the active site groove of the X-ray crystal structure of hCypA by taking loop Trp302-Pro310 as the SARS_NP binding loop. The entire complex structure was then optimized by using molecular mechanics method with AMBER force field and Kollman-all-Atom charges [33].

Complexes of wild-type hCypA (or SARS_NP) with mutated SARS_NP (or hCypA) were constructed based on the 3D model of SARS_NP–hCypA complex (Fig. 2B) by using Biopolymer module encoded in Sybyl 6.8 [34]. These complex models were solvated by a 28 Å TIP3P [35] water sphere. Then the solvated models were equilibrated for 300 ps at 300 K by using molecular dynamics (MD) simulation, and a 100 ps MD simulation was run on each model system, sampling the conformations for binding free energy calculation. All MD simulations were carried out by using the AMBER 7.0 program with AMBER force field (parm98) [33]. The MM/PBSA method [36] was used to evaluate the binding free energies of SARS_NP–hCypA complexes. In general, the average binding free energy of protein–protein interaction (averaged over 50 samples isolated from the MD trajectory), ΔG
bind, is calculated by Eq. (3):(3)ΔGbind=Gcomplex-Gprotein1-Gprotein2=ΔEMM+ΔGPB+ΔGNP,where ΔE
MM is the interaction energy of the two proteins calculated by the AMBER force field, including internal, electrostatic, and van der Waals components; ΔG
PB is the polar solvation contribution calculated by using a finite-difference Poisson–Boltzmann (PB) model [37], and ΔG
NP is the non-polar solvation energy, which is obtained from the solvent accessible surface area (SASA). The relative binding free energy, ΔΔG
bind, is the difference between the binding free energy of a mutated SARS_NP (or hCypA) to wild-type hCypA (or SARS_NP) and that of wild-type SARS_NP to wild-type hCypA.

Results and discussionMapping interactions between host proteins and SARS_NPA homology search was performed between the SARS_CoV nucleocapsid protein (SARS_NP) and the yeast proteome, and 12 proteins homologous to SARS_NP were found. Proteins binding to these SARS_NP homologous proteins possibly bind to SARS_NP. Therefore, taking these homologous proteins as queries, the protein–protein interaction database DIP [38] (http://www.DIP.org) was searched to identify potential SARS_NP binding proteins, and more than 100 possible candidates were addressed. The biological functions of the binding protein candidates were checked using the annotation information in the PFAM and Interpro databases (http://www.ebi.ac.uk/interpro/). Keywords used in the protein functional check were “immunosuppression,” “immunoregulation,” and “virus infection.” In this way, two potential SARS_NP binding domains related to cyclophilins [26] (PFAM ID PF00160) and human HIV-1 Rev-interacting protein (hRIP) [39] (PFAM ID PF01412) were identified. It is intriguing that both cyclophilins and hRIP are associated with HIV-1 infection [27], [28], [29], [39].Validation of SARS_NP–hCypA interaction by SPR technologySurface plasmon resonance (SPR) measurements were used to determine the binding affinity of SARS_NP to hCypA. In separate experiments, the dissociation constant (K
D) of SARS_NP from hCypA immobilized on the CM5 sensor chip and the dissociation constant (KD′) of hCypA from immobilized SARS_NP were determined. The binding responses in resonance units (RUs) were continuously recorded and presented graphically as a function of time. The association could be described by a simple equilibrium of the A + B ⇔ AB type (A, analyte; B, ligand; and AB, complex). To determine the equilibrium dissociation constant for the interaction, the equilibrium response (R
eq) data were fit to an independent binding-site model,(4)Req=∑iRmax,i×C×KA,i1+C×KA,i,where R
max stands for the maximal response, C is the concentration of injection sample, and K
A is the association constant. For a single-site interaction, i
 = 1, for two–site binding, i
 = 2, and so on. The results are shown in Fig. 1
and listed in Table 1
. For the first determination (with hCypA immobilized on the chip), the rate association constant (k
on) was 1.17 × 104
 M−1
 s−1, the rate dissociation constant (k
off) was 7.10 × 10−5
 s−1, and the equilibrium dissociation constant (K
D) was estimated (k
off/k
on) as 6.04 × 10−9
 M. For the second determination (with SARS_NP immobilized on the chip), the rate association constant (kon′) and the rate dissociation constant (koff′) between the two proteins were 854 M−1
 s−1 and 1.36 × 10−4
 s−1, respectively, and the equilibrium dissociation constant (KD′) was 1.59 × 10−7
 M. This indicates that SARS_NP binds hCypA with a high affinity, much higher than the binding affinity between HIV-1 CA and hCypA, ∼16 μM [40]. Furthermore, the sensorgrams (Fig. 1) and the k
on and k
off values (Table 1) revealed that both the association and dissociation between SARS_NP (or hCypA) and hCypA (or SARS_NP) are slow processes.Fig. 1Surface plasmon resonance analysis of SARS_NP–hCypA binding. (A) Sensorgram for SARS_NP binding to an hCypA surface on the CM5 sensor chip. Binding responses are shown for SARS_NP injected at concentrations of 31.25, 62.5, 125, 250, 500, and 1000 nM. (B) Sensorgram for hCypA binding to a SARS_NP surface on the CM5 sensor chip. Binding responses are shown for hCypA injected at concentrations of 0.312, 1.25, 2.5, 5.0, 10.0, 12.0, and 20.0 μM.Table 1Binding affinities of wild-type hCypA to SARS_NP and its mutants and wild-type SARS_NP to hCypA and its mutantsNP mutationkon (M−1 s−1)koff (s−1)KD (nM)ΔΔGbind (kcal/mol)ExperimentalaPredictedbhCypA binding affinityWild type1.17 ± 0.005 × 1047.10 ± 0.032 × 10−56.040.00.0Trp302Ala1.43 ± 0.053 × 1043.41 ± 0.088 × 10−32382.182.00Ile305Ala7.67 ± 0.094 × 1039.86 ± 0.075 × 10−41291.810.50Gln307Ala1.59 ± 0.032 × 1041.99 ± 0.093 × 10−412.60.443.95Gln307Asp1.12 ± 0.088 × 1042.25 ± 0.026 × 10−420.10.7111.75  hCypA mutationkon′ (M−1 s−1)koff′ (s−1)KD′ (nM)ΔΔGbind (kcal/mol)ExperimentalcPredicteddSARS_NP binding affinityWild type854 ± 24.51.36 ± 0.066 × 10−41590.00.0Arg55Ala715 ± 4.335.14 ± 0.018 × 10−47190.901.13Arg55Asp4.74 ± 0.019 × 1031.02 ± 0.094 × 10−32150.184.02Gln63Ala7.31 ± 0.0872.72 ± 0.018 × 10−437,2003.234.95Trp121Phe7.99 ± 0.0962.11 ± 0.012 × 10−426,3003.030.78Arg69Ala5.10 ± 0.028 × 1033.80 ± 0.076 × 10−474.6−0.45−0.89Arg69Lys123 ± 2.533.89 ± 0.039 × 10−431601.770.48aRelative binding free energies of hCypA–SARS_NP mutants binding relative to the hCypA–SARS_NP binding calculated from the SPR dissociation constants (KDs).bPredicted relative binding free energies of hCypA–SARS_NP mutants binding relative to the hCypA–SARS_NP binding calculated by the MM/PBSA method.cRelative binding free energies of SARS_NP-hCypA mutants binding relative to the SARS_NP–hCypA binding calculated from the SPR dissociation constants (KD′s).dPredicted relative binding free energies of SARS_NP–hCypA mutants binding relative to the SARS_NP–hCypA binding calculated by the MM/PBSA method.Identification of binding sitesEncouraged by the experimental verification of the binding between SARS_NP and hCypA with high affinity, we made an attempt to identify the responsible binding sites on both proteins. This was achieved by constructing a three-dimensional (3D) model for the SARS_NP–hCypA complex. At present, the X-ray crystal structure of the complex between human cyclophilin A (hCypA) and the N-terminal domain (residues 1–151) of HIV-1 capsid protein (CA151) is one of the experimental structures for hCypA in complex with another protein [29], [41]. Sequence alignment suggested that the segment Val235-Pro369 of SARS_NP is homologous to HIV-1 CA (Fig. 2A
), with an identity of 25.1% and a similarity of 36.7%. Accordingly, we deduced that the binding site of SARS_NP to hCypA is located within this segment. We constructed a 3D model for the SARS_NP–hCypA complex by taking the crystal structure of the hCypA–CA151 complex [29] (PDB entry 1AK4) as a template. First, a 3D model of the segment Val235-Pro369 of SARS_NP was generated by homology modeling based on the 3D structure of HIV-1 CA151 in complex with hCypA–CA [29]. Further domain analysis indicated that the SARS_NP segment Val235-Pro369 is a special domain, which belongs to the SCP (sterol carrier protein) protein family. The modeled 3D structure of SARS_NP segment Val235-Pro369 was docked manually into the active site of hCypA, allowing loop Trp302-Pro310 of NP, which aligns well with HIV-CA Pro85-Pro93, to lie in the active-site groove of hCypA in an extended conformation. The overall structural model of the SARS_NP(235–369)–hCypA complex is shown in Fig. 2B.Fig. 2Structure model for SARS_NP–hCypA complex. (A) The sequence alignments between HIV-1 capsid protein (1–231) and SARS_NP. Asterisks label the identical or conserved residues, colons designate the conserved substitutions, and dots point out the semi-conserved substitutions. Alignment was carried out by the program CLUSTALW 1.81 [47]. (B) Schematic representation of the 3D model of SARS_NP(235–369)–hCypA complex. The binding loop Trp302-Pro310 was highlighted in red color. Residues shown in ball-and-stick model are essential in the SARS_NP–hCypA binding, and site-directed mutagenesis were performed on these residues to validate this binding model. This picture was generated using the MolScript program [48]. (C) Binding of SARS_NP loop Trp302-Pro310 (shown as CPK model) to the hCypA active-site groove. The hCypA is rendered by electrostatic surface calculated by MOLCAD program encoded in Sybyl 6.8 [34]. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this paper.)Structurally, SARS_NP loop Trp302-Pro310 fits well into the active-site groove of hCypA in a way similar to CA loop Pro85-Pro93 (Fig. 2C). Several important interactions can be deduced from the binding model (Fig. 3
). Briefly, two hydrogen bonds form between the side-chain of hCypA Arg55 (Nη1 and N) and SARS_NP Gln307 (Oε1 and O); the N atom of hCypA Gln63 hydrogen bonds to the carbonyl O atom of SARS_NP Ile305; the Nε1 atom of hCypA Trp121 forms a hydrogen bond to the carbonyl oxygen of SARS_NP Phe308; and the carbonyl oxygens of Gly72 and Asn102 of hCypA accept hydrogen bonds from the main-chain amides of SARS_NP Ile305 and Ala306, respectively (Fig. 3). Interestingly, the 3D model demonstrates that a cation–π interaction [42] exists between hCypA Arg69 and SARS_NP Trp302, and a CH⋯π hydrogen bond [43], [44] between the γ-CH2 group of Gln307 and hCypA Phe133 (Fig. 3). In addition, extensive hydrophobic contacts exist across the SARS_NP–hCypA interface.Fig. 3Hydrogen bonds (green dot lines), cation–π (blue dot line), and -CH⋯π hydrogen bond (red dot line) interactions between SARS_NP and hCypA. Residues of SARS_NP are represented as ball-and-stick model, and residues of hCypA are represented as stick model. Atoms are rendered by colors of atom types: carbon gray, nitrogen blue, and oxygen red. The two green balls represent the centers of benzene rings of Phe113 and Trp121, respectively. Distances are in Angstrom. This picture was generated using the MolScript program [48]. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this paper.)Validation of the binding model by site-directed mutagenesisThe modeled structure described above indicated that SARS_NP–hCypA complex formation occurs mainly between the SARS_NP loop Trp302-Pro310 and the active site of hCypA (Fig. 2C). To validate this binding model, each residue of both proteins that, according to our model, makes an essential contribution to complex formation (Fig. 3), was mutated and the effect of the mutation on the binding energy was predicted computationally as well as measured experimentally. Binding affinity measurements were performed by SPR technology, and the free energy changes relative to the wild-type protein–protein interaction of the mutations were calculated using MM/PBSA [36]. The results are listed in Table 1. During the SPR determination, wild-type hCypA was immobilized on the chip in the assay for hCypA binding to mutated SARS_NP proteins; while in the assay for SARS_NP binding to the mutated hCypA proteins, wild-type SARS_NP was immobilized on the chip. Overall, the mutational analyses revealed that SARS_NP loop Trp302-Pro310 is the key determinant for hCypA recognition, and that this loop binds to the active-site groove of hCypA, probably exactly as predicted by our model (Fig. 2, Fig. 3).For the SARS_NP protein, the mutational analyses revealed that Trp302 is a major determinant of hCypA recognition, since mutation of Trp302 to Ala reduced the hCypA binding by 2.18 kcal/mol, which agrees well with the computational predicted value of relative free energy, 2.0 kcal/mol. The mutation Trp302Ala abolishes the cation–π interaction between the indole ring of the tryptophane and the side-chain of hCypA Arg69 (Fig. 3). The SARS_NP–hCypA binding model predicts a contact between the side-chain of Ile305 and hCypA Asn102. Indeed, replacement of Ile305 by Ala decreases the binding affinity by 1.81 kcal/mol (Table 1). This is also in agreement with the computational value of 0.55 kcal/mol. Mutation of Gln307 to Ala abolishes the hydrogen bond formed between this residue and hCypA Arg55 (Nη1) and also the C–H⋯π hydrogen bond with hCypA Phe113 (Fig. 3), thereby reducing the hCypA binding affinity by about 0.44 kcal/mol. A decrease of the latter, albeit of larger magnitude (3.95 kcal/mol), had also been predicted by the theoretical calculations. To confirm the C–H⋯π hydrogen bond, we performed a mutation of Gln307 to Asp. The negatively charged side-chain of Asp creates a repulsive interaction with the π electrons of the benzene ring of hCypA Phe133, thus decreasing the hCypA binding affinity by about 0.71 kcal/mol.In addition to SARS_NP residues predicted to be important by the binding model, we also mutated relevant hCypA residues. Mutation of Arg55 to Ala disrupts the two hydrogen bonds formed by the side-chain with the carbonyl O and side-chain Oε1 atoms of Gln307 (Fig. 3), respectively, which reduces the binding affinity (as determined by SPR) by about 0.90 kcal/mol. Mutation of Gln63 to Ala abolishes the hydrogen bond between its side-chain and the carbonyl O atom of Ile305, thus decreasing the SARS_NP binding affinity by about 3.23 kcal/mol. Trp121 contributes to SARS_NP binding by forming an intermolecular hydrogen bond between its indole NH and SARS_NP Phe308 O atom. Mutation of this tryptophane to Phe decreases the SARS_NP binding affinity by 3.03 kcal/mol. Additionally, mutation of Arg69 to Lys was also performed, which decreases the SARS_NP binding affinity by 1.77 kcal/mol. This result indicated again that a cation–π interaction exists between hCypA Arg69 and SARS_NP Trp302 (Fig. 3). For the above mutations, the free energy changes for SARS_NP binding to hCypA mutant proteins, relative to SARS_NP binding to wild-type hCypA, were also calculated by using the computational method, and all of these were found to be well in agreement with the experimental results (Table 1).Mutation of hCypA Arg69 to Ala was performed to abolish the cation–π interaction with SARS_NP (Fig. 3). However, SPR determination indicated that this mutation increased the binding affinity by about 0.45 kcal/mol, and free energy prediction basically agreed with this (ΔΔG
bind
 = 0.89 kcal/mol). The SARS_NP–hCypA binding model indicated that this cation–π interaction is located at the rim of the protein–protein interface and may be surrounded by solvent molecules (Fig. 2B). Molecular dynamics simulations (unpublished result) indicated that there are many water molecules in the SARS_NP–hCypA interface due to the attraction of the cationic side-chain of Arg69. Substitution of Arg69 by Ala tends to prevent water molecules from entering the protein–protein interface due to the hydrophobicity of Ala and Trp302, which is beneficial to the SARS_NP–hCypA binding.Binding affinity changes caused by the described mutations can be rationalized energetically by the 3D model of SARS_NP–hCypA complex, and the experimentally determined relative binding free energies (ΔΔG
bind) are largely in agreement with the predicted values resulting from the calculations based on the 3D model (Table 1). The SARS_NP–hCypA binding model therefore clearly rationalizes why changing the side-chains of some residues is energetically unfavorable or favorable. This demonstrated further the reliability of the interaction model.Significance of SARS_NP–hCypA interaction for SARS-CoV infectionAlthough cyclophilins, especially CypA, have been studied extensively, the functions of this family of proteins are not known with certainty [27]. Recently, the significance of the Gag–hCypA interaction for the retroviral life cycle has been demonstrated [27], [28], [29], [30]. Evidence for Gag–hCypA interaction during HIV-1 virus infection comes from the observation that cyclosporin (CsA), an inhibitor of cyclophilin, blocks viral infection if present at the time of virus entry into host cells, and leads to a decrease of the yield of infectious particles released if added to infected cells [30], [45], [46].In conclusion, we have found in the present study that the NP of SARS-CoV binds hCypA with high affinity; the equilibrium dissociation constant (K
D) ranges from 6 to 160 nM, which is 100–2700 times higher than the binding affinity of HIV-1 CA to hCypA. Sequence alignment and molecular modeling revealed that SARS_NP segment Val235-Pro369 may interact with hCypA (Fig. 2); more precisely, SARS_NP loop Trp302-Pro310 fits into the active-site groove of hCypA through hydrogen bonding, cation–π and -CH⋯π hydrogen bonding interactions (Fig. 2, Fig. 3). The putative binding sites have been verified by site-directed mutagenesis and determination of the binding affinity (Table 1). Hopefully, this observed SARS_NP–hCypA interaction might provide a new hint for the understanding of a possible SARS-CoV infection pathway against human cell, and further supply a feasible approach for anti-SARS agent screening.

Results and discussion

Mapping interactions between host proteins and SARS_NPA homology search was performed between the SARS_CoV nucleocapsid protein (SARS_NP) and the yeast proteome, and 12 proteins homologous to SARS_NP were found. Proteins binding to these SARS_NP homologous proteins possibly bind to SARS_NP. Therefore, taking these homologous proteins as queries, the protein–protein interaction database DIP [38] (http://www.DIP.org) was searched to identify potential SARS_NP binding proteins, and more than 100 possible candidates were addressed. The biological functions of the binding protein candidates were checked using the annotation information in the PFAM and Interpro databases (http://www.ebi.ac.uk/interpro/). Keywords used in the protein functional check were “immunosuppression,” “immunoregulation,” and “virus infection.” In this way, two potential SARS_NP binding domains related to cyclophilins [26] (PFAM ID PF00160) and human HIV-1 Rev-interacting protein (hRIP) [39] (PFAM ID PF01412) were identified. It is intriguing that both cyclophilins and hRIP are associated with HIV-1 infection [27], [28], [29], [39].

Mapping interactions between host proteins and SARS_NP

A homology search was performed between the SARS_CoV nucleocapsid protein (SARS_NP) and the yeast proteome, and 12 proteins homologous to SARS_NP were found. Proteins binding to these SARS_NP homologous proteins possibly bind to SARS_NP. Therefore, taking these homologous proteins as queries, the protein–protein interaction database DIP [38] (http://www.DIP.org) was searched to identify potential SARS_NP binding proteins, and more than 100 possible candidates were addressed. The biological functions of the binding protein candidates were checked using the annotation information in the PFAM and Interpro databases (http://www.ebi.ac.uk/interpro/). Keywords used in the protein functional check were “immunosuppression,” “immunoregulation,” and “virus infection.” In this way, two potential SARS_NP binding domains related to cyclophilins [26] (PFAM ID PF00160) and human HIV-1 Rev-interacting protein (hRIP) [39] (PFAM ID PF01412) were identified. It is intriguing that both cyclophilins and hRIP are associated with HIV-1 infection [27], [28], [29], [39].

Validation of SARS_NP–hCypA interaction by SPR technologySurface plasmon resonance (SPR) measurements were used to determine the binding affinity of SARS_NP to hCypA. In separate experiments, the dissociation constant (K
D) of SARS_NP from hCypA immobilized on the CM5 sensor chip and the dissociation constant (KD′) of hCypA from immobilized SARS_NP were determined. The binding responses in resonance units (RUs) were continuously recorded and presented graphically as a function of time. The association could be described by a simple equilibrium of the A + B ⇔ AB type (A, analyte; B, ligand; and AB, complex). To determine the equilibrium dissociation constant for the interaction, the equilibrium response (R
eq) data were fit to an independent binding-site model,(4)Req=∑iRmax,i×C×KA,i1+C×KA,i,where R
max stands for the maximal response, C is the concentration of injection sample, and K
A is the association constant. For a single-site interaction, i
 = 1, for two–site binding, i
 = 2, and so on. The results are shown in Fig. 1
and listed in Table 1
. For the first determination (with hCypA immobilized on the chip), the rate association constant (k
on) was 1.17 × 104
 M−1
 s−1, the rate dissociation constant (k
off) was 7.10 × 10−5
 s−1, and the equilibrium dissociation constant (K
D) was estimated (k
off/k
on) as 6.04 × 10−9
 M. For the second determination (with SARS_NP immobilized on the chip), the rate association constant (kon′) and the rate dissociation constant (koff′) between the two proteins were 854 M−1
 s−1 and 1.36 × 10−4
 s−1, respectively, and the equilibrium dissociation constant (KD′) was 1.59 × 10−7
 M. This indicates that SARS_NP binds hCypA with a high affinity, much higher than the binding affinity between HIV-1 CA and hCypA, ∼16 μM [40]. Furthermore, the sensorgrams (Fig. 1) and the k
on and k
off values (Table 1) revealed that both the association and dissociation between SARS_NP (or hCypA) and hCypA (or SARS_NP) are slow processes.Fig. 1Surface plasmon resonance analysis of SARS_NP–hCypA binding. (A) Sensorgram for SARS_NP binding to an hCypA surface on the CM5 sensor chip. Binding responses are shown for SARS_NP injected at concentrations of 31.25, 62.5, 125, 250, 500, and 1000 nM. (B) Sensorgram for hCypA binding to a SARS_NP surface on the CM5 sensor chip. Binding responses are shown for hCypA injected at concentrations of 0.312, 1.25, 2.5, 5.0, 10.0, 12.0, and 20.0 μM.Table 1Binding affinities of wild-type hCypA to SARS_NP and its mutants and wild-type SARS_NP to hCypA and its mutantsNP mutationkon (M−1 s−1)koff (s−1)KD (nM)ΔΔGbind (kcal/mol)ExperimentalaPredictedbhCypA binding affinityWild type1.17 ± 0.005 × 1047.10 ± 0.032 × 10−56.040.00.0Trp302Ala1.43 ± 0.053 × 1043.41 ± 0.088 × 10−32382.182.00Ile305Ala7.67 ± 0.094 × 1039.86 ± 0.075 × 10−41291.810.50Gln307Ala1.59 ± 0.032 × 1041.99 ± 0.093 × 10−412.60.443.95Gln307Asp1.12 ± 0.088 × 1042.25 ± 0.026 × 10−420.10.7111.75  hCypA mutationkon′ (M−1 s−1)koff′ (s−1)KD′ (nM)ΔΔGbind (kcal/mol)ExperimentalcPredicteddSARS_NP binding affinityWild type854 ± 24.51.36 ± 0.066 × 10−41590.00.0Arg55Ala715 ± 4.335.14 ± 0.018 × 10−47190.901.13Arg55Asp4.74 ± 0.019 × 1031.02 ± 0.094 × 10−32150.184.02Gln63Ala7.31 ± 0.0872.72 ± 0.018 × 10−437,2003.234.95Trp121Phe7.99 ± 0.0962.11 ± 0.012 × 10−426,3003.030.78Arg69Ala5.10 ± 0.028 × 1033.80 ± 0.076 × 10−474.6−0.45−0.89Arg69Lys123 ± 2.533.89 ± 0.039 × 10−431601.770.48aRelative binding free energies of hCypA–SARS_NP mutants binding relative to the hCypA–SARS_NP binding calculated from the SPR dissociation constants (KDs).bPredicted relative binding free energies of hCypA–SARS_NP mutants binding relative to the hCypA–SARS_NP binding calculated by the MM/PBSA method.cRelative binding free energies of SARS_NP-hCypA mutants binding relative to the SARS_NP–hCypA binding calculated from the SPR dissociation constants (KD′s).dPredicted relative binding free energies of SARS_NP–hCypA mutants binding relative to the SARS_NP–hCypA binding calculated by the MM/PBSA method.

Validation of SARS_NP–hCypA interaction by SPR technology

Surface plasmon resonance (SPR) measurements were used to determine the binding affinity of SARS_NP to hCypA. In separate experiments, the dissociation constant (K
D) of SARS_NP from hCypA immobilized on the CM5 sensor chip and the dissociation constant (KD′) of hCypA from immobilized SARS_NP were determined. The binding responses in resonance units (RUs) were continuously recorded and presented graphically as a function of time. The association could be described by a simple equilibrium of the A + B ⇔ AB type (A, analyte; B, ligand; and AB, complex). To determine the equilibrium dissociation constant for the interaction, the equilibrium response (R
eq) data were fit to an independent binding-site model,(4)Req=∑iRmax,i×C×KA,i1+C×KA,i,where R
max stands for the maximal response, C is the concentration of injection sample, and K
A is the association constant. For a single-site interaction, i
 = 1, for two–site binding, i
 = 2, and so on. The results are shown in Fig. 1
and listed in Table 1
. For the first determination (with hCypA immobilized on the chip), the rate association constant (k
on) was 1.17 × 104
 M−1
 s−1, the rate dissociation constant (k
off) was 7.10 × 10−5
 s−1, and the equilibrium dissociation constant (K
D) was estimated (k
off/k
on) as 6.04 × 10−9
 M. For the second determination (with SARS_NP immobilized on the chip), the rate association constant (kon′) and the rate dissociation constant (koff′) between the two proteins were 854 M−1
 s−1 and 1.36 × 10−4
 s−1, respectively, and the equilibrium dissociation constant (KD′) was 1.59 × 10−7
 M. This indicates that SARS_NP binds hCypA with a high affinity, much higher than the binding affinity between HIV-1 CA and hCypA, ∼16 μM [40]. Furthermore, the sensorgrams (Fig. 1) and the k
on and k
off values (Table 1) revealed that both the association and dissociation between SARS_NP (or hCypA) and hCypA (or SARS_NP) are slow processes.Fig. 1Surface plasmon resonance analysis of SARS_NP–hCypA binding. (A) Sensorgram for SARS_NP binding to an hCypA surface on the CM5 sensor chip. Binding responses are shown for SARS_NP injected at concentrations of 31.25, 62.5, 125, 250, 500, and 1000 nM. (B) Sensorgram for hCypA binding to a SARS_NP surface on the CM5 sensor chip. Binding responses are shown for hCypA injected at concentrations of 0.312, 1.25, 2.5, 5.0, 10.0, 12.0, and 20.0 μM.Table 1Binding affinities of wild-type hCypA to SARS_NP and its mutants and wild-type SARS_NP to hCypA and its mutantsNP mutationkon (M−1 s−1)koff (s−1)KD (nM)ΔΔGbind (kcal/mol)ExperimentalaPredictedbhCypA binding affinityWild type1.17 ± 0.005 × 1047.10 ± 0.032 × 10−56.040.00.0Trp302Ala1.43 ± 0.053 × 1043.41 ± 0.088 × 10−32382.182.00Ile305Ala7.67 ± 0.094 × 1039.86 ± 0.075 × 10−41291.810.50Gln307Ala1.59 ± 0.032 × 1041.99 ± 0.093 × 10−412.60.443.95Gln307Asp1.12 ± 0.088 × 1042.25 ± 0.026 × 10−420.10.7111.75  hCypA mutationkon′ (M−1 s−1)koff′ (s−1)KD′ (nM)ΔΔGbind (kcal/mol)ExperimentalcPredicteddSARS_NP binding affinityWild type854 ± 24.51.36 ± 0.066 × 10−41590.00.0Arg55Ala715 ± 4.335.14 ± 0.018 × 10−47190.901.13Arg55Asp4.74 ± 0.019 × 1031.02 ± 0.094 × 10−32150.184.02Gln63Ala7.31 ± 0.0872.72 ± 0.018 × 10−437,2003.234.95Trp121Phe7.99 ± 0.0962.11 ± 0.012 × 10−426,3003.030.78Arg69Ala5.10 ± 0.028 × 1033.80 ± 0.076 × 10−474.6−0.45−0.89Arg69Lys123 ± 2.533.89 ± 0.039 × 10−431601.770.48aRelative binding free energies of hCypA–SARS_NP mutants binding relative to the hCypA–SARS_NP binding calculated from the SPR dissociation constants (KDs).bPredicted relative binding free energies of hCypA–SARS_NP mutants binding relative to the hCypA–SARS_NP binding calculated by the MM/PBSA method.cRelative binding free energies of SARS_NP-hCypA mutants binding relative to the SARS_NP–hCypA binding calculated from the SPR dissociation constants (KD′s).dPredicted relative binding free energies of SARS_NP–hCypA mutants binding relative to the SARS_NP–hCypA binding calculated by the MM/PBSA method.

Surface plasmon resonance analysis of SARS_NP–hCypA binding. (A) Sensorgram for SARS_NP binding to an hCypA surface on the CM5 sensor chip. Binding responses are shown for SARS_NP injected at concentrations of 31.25, 62.5, 125, 250, 500, and 1000 nM. (B) Sensorgram for hCypA binding to a SARS_NP surface on the CM5 sensor chip. Binding responses are shown for hCypA injected at concentrations of 0.312, 1.25, 2.5, 5.0, 10.0, 12.0, and 20.0 μM.

Binding affinities of wild-type hCypA to SARS_NP and its mutants and wild-type SARS_NP to hCypA and its mutants

Relative binding free energies of hCypA–SARS_NP mutants binding relative to the hCypA–SARS_NP binding calculated from the SPR dissociation constants (KDs).

Predicted relative binding free energies of hCypA–SARS_NP mutants binding relative to the hCypA–SARS_NP binding calculated by the MM/PBSA method.

Relative binding free energies of SARS_NP-hCypA mutants binding relative to the SARS_NP–hCypA binding calculated from the SPR dissociation constants (KD′s).

Predicted relative binding free energies of SARS_NP–hCypA mutants binding relative to the SARS_NP–hCypA binding calculated by the MM/PBSA method.

Identification of binding sitesEncouraged by the experimental verification of the binding between SARS_NP and hCypA with high affinity, we made an attempt to identify the responsible binding sites on both proteins. This was achieved by constructing a three-dimensional (3D) model for the SARS_NP–hCypA complex. At present, the X-ray crystal structure of the complex between human cyclophilin A (hCypA) and the N-terminal domain (residues 1–151) of HIV-1 capsid protein (CA151) is one of the experimental structures for hCypA in complex with another protein [29], [41]. Sequence alignment suggested that the segment Val235-Pro369 of SARS_NP is homologous to HIV-1 CA (Fig. 2A
), with an identity of 25.1% and a similarity of 36.7%. Accordingly, we deduced that the binding site of SARS_NP to hCypA is located within this segment. We constructed a 3D model for the SARS_NP–hCypA complex by taking the crystal structure of the hCypA–CA151 complex [29] (PDB entry 1AK4) as a template. First, a 3D model of the segment Val235-Pro369 of SARS_NP was generated by homology modeling based on the 3D structure of HIV-1 CA151 in complex with hCypA–CA [29]. Further domain analysis indicated that the SARS_NP segment Val235-Pro369 is a special domain, which belongs to the SCP (sterol carrier protein) protein family. The modeled 3D structure of SARS_NP segment Val235-Pro369 was docked manually into the active site of hCypA, allowing loop Trp302-Pro310 of NP, which aligns well with HIV-CA Pro85-Pro93, to lie in the active-site groove of hCypA in an extended conformation. The overall structural model of the SARS_NP(235–369)–hCypA complex is shown in Fig. 2B.Fig. 2Structure model for SARS_NP–hCypA complex. (A) The sequence alignments between HIV-1 capsid protein (1–231) and SARS_NP. Asterisks label the identical or conserved residues, colons designate the conserved substitutions, and dots point out the semi-conserved substitutions. Alignment was carried out by the program CLUSTALW 1.81 [47]. (B) Schematic representation of the 3D model of SARS_NP(235–369)–hCypA complex. The binding loop Trp302-Pro310 was highlighted in red color. Residues shown in ball-and-stick model are essential in the SARS_NP–hCypA binding, and site-directed mutagenesis were performed on these residues to validate this binding model. This picture was generated using the MolScript program [48]. (C) Binding of SARS_NP loop Trp302-Pro310 (shown as CPK model) to the hCypA active-site groove. The hCypA is rendered by electrostatic surface calculated by MOLCAD program encoded in Sybyl 6.8 [34]. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this paper.)Structurally, SARS_NP loop Trp302-Pro310 fits well into the active-site groove of hCypA in a way similar to CA loop Pro85-Pro93 (Fig. 2C). Several important interactions can be deduced from the binding model (Fig. 3
). Briefly, two hydrogen bonds form between the side-chain of hCypA Arg55 (Nη1 and N) and SARS_NP Gln307 (Oε1 and O); the N atom of hCypA Gln63 hydrogen bonds to the carbonyl O atom of SARS_NP Ile305; the Nε1 atom of hCypA Trp121 forms a hydrogen bond to the carbonyl oxygen of SARS_NP Phe308; and the carbonyl oxygens of Gly72 and Asn102 of hCypA accept hydrogen bonds from the main-chain amides of SARS_NP Ile305 and Ala306, respectively (Fig. 3). Interestingly, the 3D model demonstrates that a cation–π interaction [42] exists between hCypA Arg69 and SARS_NP Trp302, and a CH⋯π hydrogen bond [43], [44] between the γ-CH2 group of Gln307 and hCypA Phe133 (Fig. 3). In addition, extensive hydrophobic contacts exist across the SARS_NP–hCypA interface.Fig. 3Hydrogen bonds (green dot lines), cation–π (blue dot line), and -CH⋯π hydrogen bond (red dot line) interactions between SARS_NP and hCypA. Residues of SARS_NP are represented as ball-and-stick model, and residues of hCypA are represented as stick model. Atoms are rendered by colors of atom types: carbon gray, nitrogen blue, and oxygen red. The two green balls represent the centers of benzene rings of Phe113 and Trp121, respectively. Distances are in Angstrom. This picture was generated using the MolScript program [48]. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this paper.)

Identification of binding sites

Encouraged by the experimental verification of the binding between SARS_NP and hCypA with high affinity, we made an attempt to identify the responsible binding sites on both proteins. This was achieved by constructing a three-dimensional (3D) model for the SARS_NP–hCypA complex. At present, the X-ray crystal structure of the complex between human cyclophilin A (hCypA) and the N-terminal domain (residues 1–151) of HIV-1 capsid protein (CA151) is one of the experimental structures for hCypA in complex with another protein [29], [41]. Sequence alignment suggested that the segment Val235-Pro369 of SARS_NP is homologous to HIV-1 CA (Fig. 2A
), with an identity of 25.1% and a similarity of 36.7%. Accordingly, we deduced that the binding site of SARS_NP to hCypA is located within this segment. We constructed a 3D model for the SARS_NP–hCypA complex by taking the crystal structure of the hCypA–CA151 complex [29] (PDB entry 1AK4) as a template. First, a 3D model of the segment Val235-Pro369 of SARS_NP was generated by homology modeling based on the 3D structure of HIV-1 CA151 in complex with hCypA–CA [29]. Further domain analysis indicated that the SARS_NP segment Val235-Pro369 is a special domain, which belongs to the SCP (sterol carrier protein) protein family. The modeled 3D structure of SARS_NP segment Val235-Pro369 was docked manually into the active site of hCypA, allowing loop Trp302-Pro310 of NP, which aligns well with HIV-CA Pro85-Pro93, to lie in the active-site groove of hCypA in an extended conformation. The overall structural model of the SARS_NP(235–369)–hCypA complex is shown in Fig. 2B.Fig. 2Structure model for SARS_NP–hCypA complex. (A) The sequence alignments between HIV-1 capsid protein (1–231) and SARS_NP. Asterisks label the identical or conserved residues, colons designate the conserved substitutions, and dots point out the semi-conserved substitutions. Alignment was carried out by the program CLUSTALW 1.81 [47]. (B) Schematic representation of the 3D model of SARS_NP(235–369)–hCypA complex. The binding loop Trp302-Pro310 was highlighted in red color. Residues shown in ball-and-stick model are essential in the SARS_NP–hCypA binding, and site-directed mutagenesis were performed on these residues to validate this binding model. This picture was generated using the MolScript program [48]. (C) Binding of SARS_NP loop Trp302-Pro310 (shown as CPK model) to the hCypA active-site groove. The hCypA is rendered by electrostatic surface calculated by MOLCAD program encoded in Sybyl 6.8 [34]. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this paper.)

Structure model for SARS_NP–hCypA complex. (A) The sequence alignments between HIV-1 capsid protein (1–231) and SARS_NP. Asterisks label the identical or conserved residues, colons designate the conserved substitutions, and dots point out the semi-conserved substitutions. Alignment was carried out by the program CLUSTALW 1.81 [47]. (B) Schematic representation of the 3D model of SARS_NP(235–369)–hCypA complex. The binding loop Trp302-Pro310 was highlighted in red color. Residues shown in ball-and-stick model are essential in the SARS_NP–hCypA binding, and site-directed mutagenesis were performed on these residues to validate this binding model. This picture was generated using the MolScript program [48]. (C) Binding of SARS_NP loop Trp302-Pro310 (shown as CPK model) to the hCypA active-site groove. The hCypA is rendered by electrostatic surface calculated by MOLCAD program encoded in Sybyl 6.8 [34]. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this paper.)

Structurally, SARS_NP loop Trp302-Pro310 fits well into the active-site groove of hCypA in a way similar to CA loop Pro85-Pro93 (Fig. 2C). Several important interactions can be deduced from the binding model (Fig. 3
). Briefly, two hydrogen bonds form between the side-chain of hCypA Arg55 (Nη1 and N) and SARS_NP Gln307 (Oε1 and O); the N atom of hCypA Gln63 hydrogen bonds to the carbonyl O atom of SARS_NP Ile305; the Nε1 atom of hCypA Trp121 forms a hydrogen bond to the carbonyl oxygen of SARS_NP Phe308; and the carbonyl oxygens of Gly72 and Asn102 of hCypA accept hydrogen bonds from the main-chain amides of SARS_NP Ile305 and Ala306, respectively (Fig. 3). Interestingly, the 3D model demonstrates that a cation–π interaction [42] exists between hCypA Arg69 and SARS_NP Trp302, and a CH⋯π hydrogen bond [43], [44] between the γ-CH2 group of Gln307 and hCypA Phe133 (Fig. 3). In addition, extensive hydrophobic contacts exist across the SARS_NP–hCypA interface.Fig. 3Hydrogen bonds (green dot lines), cation–π (blue dot line), and -CH⋯π hydrogen bond (red dot line) interactions between SARS_NP and hCypA. Residues of SARS_NP are represented as ball-and-stick model, and residues of hCypA are represented as stick model. Atoms are rendered by colors of atom types: carbon gray, nitrogen blue, and oxygen red. The two green balls represent the centers of benzene rings of Phe113 and Trp121, respectively. Distances are in Angstrom. This picture was generated using the MolScript program [48]. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this paper.)

Hydrogen bonds (green dot lines), cation–π (blue dot line), and -CH⋯π hydrogen bond (red dot line) interactions between SARS_NP and hCypA. Residues of SARS_NP are represented as ball-and-stick model, and residues of hCypA are represented as stick model. Atoms are rendered by colors of atom types: carbon gray, nitrogen blue, and oxygen red. The two green balls represent the centers of benzene rings of Phe113 and Trp121, respectively. Distances are in Angstrom. This picture was generated using the MolScript program [48]. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this paper.)

Validation of the binding model by site-directed mutagenesisThe modeled structure described above indicated that SARS_NP–hCypA complex formation occurs mainly between the SARS_NP loop Trp302-Pro310 and the active site of hCypA (Fig. 2C). To validate this binding model, each residue of both proteins that, according to our model, makes an essential contribution to complex formation (Fig. 3), was mutated and the effect of the mutation on the binding energy was predicted computationally as well as measured experimentally. Binding affinity measurements were performed by SPR technology, and the free energy changes relative to the wild-type protein–protein interaction of the mutations were calculated using MM/PBSA [36]. The results are listed in Table 1. During the SPR determination, wild-type hCypA was immobilized on the chip in the assay for hCypA binding to mutated SARS_NP proteins; while in the assay for SARS_NP binding to the mutated hCypA proteins, wild-type SARS_NP was immobilized on the chip. Overall, the mutational analyses revealed that SARS_NP loop Trp302-Pro310 is the key determinant for hCypA recognition, and that this loop binds to the active-site groove of hCypA, probably exactly as predicted by our model (Fig. 2, Fig. 3).For the SARS_NP protein, the mutational analyses revealed that Trp302 is a major determinant of hCypA recognition, since mutation of Trp302 to Ala reduced the hCypA binding by 2.18 kcal/mol, which agrees well with the computational predicted value of relative free energy, 2.0 kcal/mol. The mutation Trp302Ala abolishes the cation–π interaction between the indole ring of the tryptophane and the side-chain of hCypA Arg69 (Fig. 3). The SARS_NP–hCypA binding model predicts a contact between the side-chain of Ile305 and hCypA Asn102. Indeed, replacement of Ile305 by Ala decreases the binding affinity by 1.81 kcal/mol (Table 1). This is also in agreement with the computational value of 0.55 kcal/mol. Mutation of Gln307 to Ala abolishes the hydrogen bond formed between this residue and hCypA Arg55 (Nη1) and also the C–H⋯π hydrogen bond with hCypA Phe113 (Fig. 3), thereby reducing the hCypA binding affinity by about 0.44 kcal/mol. A decrease of the latter, albeit of larger magnitude (3.95 kcal/mol), had also been predicted by the theoretical calculations. To confirm the C–H⋯π hydrogen bond, we performed a mutation of Gln307 to Asp. The negatively charged side-chain of Asp creates a repulsive interaction with the π electrons of the benzene ring of hCypA Phe133, thus decreasing the hCypA binding affinity by about 0.71 kcal/mol.In addition to SARS_NP residues predicted to be important by the binding model, we also mutated relevant hCypA residues. Mutation of Arg55 to Ala disrupts the two hydrogen bonds formed by the side-chain with the carbonyl O and side-chain Oε1 atoms of Gln307 (Fig. 3), respectively, which reduces the binding affinity (as determined by SPR) by about 0.90 kcal/mol. Mutation of Gln63 to Ala abolishes the hydrogen bond between its side-chain and the carbonyl O atom of Ile305, thus decreasing the SARS_NP binding affinity by about 3.23 kcal/mol. Trp121 contributes to SARS_NP binding by forming an intermolecular hydrogen bond between its indole NH and SARS_NP Phe308 O atom. Mutation of this tryptophane to Phe decreases the SARS_NP binding affinity by 3.03 kcal/mol. Additionally, mutation of Arg69 to Lys was also performed, which decreases the SARS_NP binding affinity by 1.77 kcal/mol. This result indicated again that a cation–π interaction exists between hCypA Arg69 and SARS_NP Trp302 (Fig. 3). For the above mutations, the free energy changes for SARS_NP binding to hCypA mutant proteins, relative to SARS_NP binding to wild-type hCypA, were also calculated by using the computational method, and all of these were found to be well in agreement with the experimental results (Table 1).Mutation of hCypA Arg69 to Ala was performed to abolish the cation–π interaction with SARS_NP (Fig. 3). However, SPR determination indicated that this mutation increased the binding affinity by about 0.45 kcal/mol, and free energy prediction basically agreed with this (ΔΔG
bind
 = 0.89 kcal/mol). The SARS_NP–hCypA binding model indicated that this cation–π interaction is located at the rim of the protein–protein interface and may be surrounded by solvent molecules (Fig. 2B). Molecular dynamics simulations (unpublished result) indicated that there are many water molecules in the SARS_NP–hCypA interface due to the attraction of the cationic side-chain of Arg69. Substitution of Arg69 by Ala tends to prevent water molecules from entering the protein–protein interface due to the hydrophobicity of Ala and Trp302, which is beneficial to the SARS_NP–hCypA binding.Binding affinity changes caused by the described mutations can be rationalized energetically by the 3D model of SARS_NP–hCypA complex, and the experimentally determined relative binding free energies (ΔΔG
bind) are largely in agreement with the predicted values resulting from the calculations based on the 3D model (Table 1). The SARS_NP–hCypA binding model therefore clearly rationalizes why changing the side-chains of some residues is energetically unfavorable or favorable. This demonstrated further the reliability of the interaction model.

Validation of the binding model by site-directed mutagenesis

The modeled structure described above indicated that SARS_NP–hCypA complex formation occurs mainly between the SARS_NP loop Trp302-Pro310 and the active site of hCypA (Fig. 2C). To validate this binding model, each residue of both proteins that, according to our model, makes an essential contribution to complex formation (Fig. 3), was mutated and the effect of the mutation on the binding energy was predicted computationally as well as measured experimentally. Binding affinity measurements were performed by SPR technology, and the free energy changes relative to the wild-type protein–protein interaction of the mutations were calculated using MM/PBSA [36]. The results are listed in Table 1. During the SPR determination, wild-type hCypA was immobilized on the chip in the assay for hCypA binding to mutated SARS_NP proteins; while in the assay for SARS_NP binding to the mutated hCypA proteins, wild-type SARS_NP was immobilized on the chip. Overall, the mutational analyses revealed that SARS_NP loop Trp302-Pro310 is the key determinant for hCypA recognition, and that this loop binds to the active-site groove of hCypA, probably exactly as predicted by our model (Fig. 2, Fig. 3).

For the SARS_NP protein, the mutational analyses revealed that Trp302 is a major determinant of hCypA recognition, since mutation of Trp302 to Ala reduced the hCypA binding by 2.18 kcal/mol, which agrees well with the computational predicted value of relative free energy, 2.0 kcal/mol. The mutation Trp302Ala abolishes the cation–π interaction between the indole ring of the tryptophane and the side-chain of hCypA Arg69 (Fig. 3). The SARS_NP–hCypA binding model predicts a contact between the side-chain of Ile305 and hCypA Asn102. Indeed, replacement of Ile305 by Ala decreases the binding affinity by 1.81 kcal/mol (Table 1). This is also in agreement with the computational value of 0.55 kcal/mol. Mutation of Gln307 to Ala abolishes the hydrogen bond formed between this residue and hCypA Arg55 (Nη1) and also the C–H⋯π hydrogen bond with hCypA Phe113 (Fig. 3), thereby reducing the hCypA binding affinity by about 0.44 kcal/mol. A decrease of the latter, albeit of larger magnitude (3.95 kcal/mol), had also been predicted by the theoretical calculations. To confirm the C–H⋯π hydrogen bond, we performed a mutation of Gln307 to Asp. The negatively charged side-chain of Asp creates a repulsive interaction with the π electrons of the benzene ring of hCypA Phe133, thus decreasing the hCypA binding affinity by about 0.71 kcal/mol.

In addition to SARS_NP residues predicted to be important by the binding model, we also mutated relevant hCypA residues. Mutation of Arg55 to Ala disrupts the two hydrogen bonds formed by the side-chain with the carbonyl O and side-chain Oε1 atoms of Gln307 (Fig. 3), respectively, which reduces the binding affinity (as determined by SPR) by about 0.90 kcal/mol. Mutation of Gln63 to Ala abolishes the hydrogen bond between its side-chain and the carbonyl O atom of Ile305, thus decreasing the SARS_NP binding affinity by about 3.23 kcal/mol. Trp121 contributes to SARS_NP binding by forming an intermolecular hydrogen bond between its indole NH and SARS_NP Phe308 O atom. Mutation of this tryptophane to Phe decreases the SARS_NP binding affinity by 3.03 kcal/mol. Additionally, mutation of Arg69 to Lys was also performed, which decreases the SARS_NP binding affinity by 1.77 kcal/mol. This result indicated again that a cation–π interaction exists between hCypA Arg69 and SARS_NP Trp302 (Fig. 3). For the above mutations, the free energy changes for SARS_NP binding to hCypA mutant proteins, relative to SARS_NP binding to wild-type hCypA, were also calculated by using the computational method, and all of these were found to be well in agreement with the experimental results (Table 1).

Mutation of hCypA Arg69 to Ala was performed to abolish the cation–π interaction with SARS_NP (Fig. 3). However, SPR determination indicated that this mutation increased the binding affinity by about 0.45 kcal/mol, and free energy prediction basically agreed with this (ΔΔG
bind
 = 0.89 kcal/mol). The SARS_NP–hCypA binding model indicated that this cation–π interaction is located at the rim of the protein–protein interface and may be surrounded by solvent molecules (Fig. 2B). Molecular dynamics simulations (unpublished result) indicated that there are many water molecules in the SARS_NP–hCypA interface due to the attraction of the cationic side-chain of Arg69. Substitution of Arg69 by Ala tends to prevent water molecules from entering the protein–protein interface due to the hydrophobicity of Ala and Trp302, which is beneficial to the SARS_NP–hCypA binding.

Binding affinity changes caused by the described mutations can be rationalized energetically by the 3D model of SARS_NP–hCypA complex, and the experimentally determined relative binding free energies (ΔΔG
bind) are largely in agreement with the predicted values resulting from the calculations based on the 3D model (Table 1). The SARS_NP–hCypA binding model therefore clearly rationalizes why changing the side-chains of some residues is energetically unfavorable or favorable. This demonstrated further the reliability of the interaction model.

Significance of SARS_NP–hCypA interaction for SARS-CoV infectionAlthough cyclophilins, especially CypA, have been studied extensively, the functions of this family of proteins are not known with certainty [27]. Recently, the significance of the Gag–hCypA interaction for the retroviral life cycle has been demonstrated [27], [28], [29], [30]. Evidence for Gag–hCypA interaction during HIV-1 virus infection comes from the observation that cyclosporin (CsA), an inhibitor of cyclophilin, blocks viral infection if present at the time of virus entry into host cells, and leads to a decrease of the yield of infectious particles released if added to infected cells [30], [45], [46].In conclusion, we have found in the present study that the NP of SARS-CoV binds hCypA with high affinity; the equilibrium dissociation constant (K
D) ranges from 6 to 160 nM, which is 100–2700 times higher than the binding affinity of HIV-1 CA to hCypA. Sequence alignment and molecular modeling revealed that SARS_NP segment Val235-Pro369 may interact with hCypA (Fig. 2); more precisely, SARS_NP loop Trp302-Pro310 fits into the active-site groove of hCypA through hydrogen bonding, cation–π and -CH⋯π hydrogen bonding interactions (Fig. 2, Fig. 3). The putative binding sites have been verified by site-directed mutagenesis and determination of the binding affinity (Table 1). Hopefully, this observed SARS_NP–hCypA interaction might provide a new hint for the understanding of a possible SARS-CoV infection pathway against human cell, and further supply a feasible approach for anti-SARS agent screening.

Significance of SARS_NP–hCypA interaction for SARS-CoV infection

Although cyclophilins, especially CypA, have been studied extensively, the functions of this family of proteins are not known with certainty [27]. Recently, the significance of the Gag–hCypA interaction for the retroviral life cycle has been demonstrated [27], [28], [29], [30]. Evidence for Gag–hCypA interaction during HIV-1 virus infection comes from the observation that cyclosporin (CsA), an inhibitor of cyclophilin, blocks viral infection if present at the time of virus entry into host cells, and leads to a decrease of the yield of infectious particles released if added to infected cells [30], [45], [46].

In conclusion, we have found in the present study that the NP of SARS-CoV binds hCypA with high affinity; the equilibrium dissociation constant (K
D) ranges from 6 to 160 nM, which is 100–2700 times higher than the binding affinity of HIV-1 CA to hCypA. Sequence alignment and molecular modeling revealed that SARS_NP segment Val235-Pro369 may interact with hCypA (Fig. 2); more precisely, SARS_NP loop Trp302-Pro310 fits into the active-site groove of hCypA through hydrogen bonding, cation–π and -CH⋯π hydrogen bonding interactions (Fig. 2, Fig. 3). The putative binding sites have been verified by site-directed mutagenesis and determination of the binding affinity (Table 1). Hopefully, this observed SARS_NP–hCypA interaction might provide a new hint for the understanding of a possible SARS-CoV infection pathway against human cell, and further supply a feasible approach for anti-SARS agent screening.
