---
pmid: '20123734'
title: NIR, an inhibitor of histone acetyltransferases, regulates transcription factor
  TAp63 and is controlled by the cell cycle.
authors:
- Heyne K
- Willnecker V
- Schneider J
- Conrad M
- Raulf N
- Schüle R
- Roemer K
journal: Nucleic Acids Res
year: '2010'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC2879503
doi: 10.1093/nar/gkq016
---

# NIR, an inhibitor of histone acetyltransferases, regulates transcription factor TAp63 and is controlled by the cell cycle.
**Authors:** Heyne K, Willnecker V, Schneider J, Conrad M, Raulf N, Schüle R, Roemer K
**Journal:** Nucleic Acids Res (2010)
**DOI:** [10.1093/nar/gkq016](https://doi.org/10.1093/nar/gkq016)
**PMC:** [PMC2879503](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2879503/)

## Abstract

1. Nucleic Acids Res. 2010 Jun;38(10):3159-71. doi: 10.1093/nar/gkq016. Epub 2010
 Jan 31.

NIR, an inhibitor of histone acetyltransferases, regulates transcription factor 
TAp63 and is controlled by the cell cycle.

Heyne K(1), Willnecker V, Schneider J, Conrad M, Raulf N, Schüle R, Roemer K.

Author information:
(1)José Carreras Research Center, Bldg 45.3, University of Saarland Medical 
School, 66421 Homburg/Saar, Germany.

p63 Is a sequence-specific transcription factor that regulates epithelial stem 
cell maintenance and epithelial differentiation. In addition, the TAp63 isoform 
with an N-terminal transactivation domain functions as an inducer of apoptosis 
during the development of sympathetic neurons. Previous work has indicated that 
the co-activator and histone acetyltransferase (HAT), p300, can bind to TAp63 
and stimulate TAp63-dependent transcription of the p21Cip1 gene. Novel INHAT 
Repressor (NIR) is an inhibitor of HAT. Here, we report that the central portion 
of NIR binds to the transactivation domain and the C-terminal oligomerization 
domain of TAp63. NIR is highly expressed in G2/M phase of the cell cycle and 
only weakly expressed in G1/S. Furthermore, except during mitosis, NIR is 
predominantly localized in the nucleolus; only a small portion co-localizes with 
TAp63 in the nucleoplasm and at the p21 gene promoter. Consistent with NIR 
acting as a repressor, the induced translocation of NIR from the nucleolus into 
the nucleoplasm resulted in the inhibition of TAp63-dependent transactivation of 
p21. Conversely, knockdown of NIR by RNAi stimulated p21 transcription in the 
presence of TAp63. Thus, NIR is a cell-cycle-controlled, novel negative 
regulator of TAp63. The low levels of nucleoplasmic NIR might act as a buffer 
toward potentially toxic TAp63.

DOI: 10.1093/nar/gkq016
PMCID: PMC2879503
PMID: 20123734 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONp63 is a close relative of the p53 tumor suppressor and transcription factor. p53 stimulates, upon potentially tumor-inducing damage, the transcription of cell cycle arrest and pro-apoptotic genes among numerous others (1). Both family members can target a common subset of genes in response to damage, and p63 is also involved in tumor suppression and progression, although more subtly (2,3). In contrast to p53, however, p63 exerts an important function during development—in the morphogenesis of stratified epithelia (4). Point mutations in the p63 gene have been associated with ectodermal dysplasias in humans (5). It is therefore no surprise that many target genes are unique to each transcription factor. Tetrameric p63 seems to preferentially bind to two consecutive 10-mer sequence motifs with the consensus (rrrCGTGyyy), (t/a,a/t,a,C,A/T,T,G,t,t/a,t), or (rrrC,A/G,T/A,Gyyy), whereas tetrameric p53 preferentially recognizes (rrrC,A/T,A/T,Gyyy) (r = purines; y = pyrimidines). Both proteins apparently prefer spacing between the 10-mers not exceeding 0–2 bp (1,6–8).p63 exhibits a high degree of molecular complexity. The presence in the p63 gene of an internal promoter in intron 3 can give rise to proteins with the major transactivation domain deleted (ΔNp63). These DNA binding-active, transactivation-impaired isoforms seem to act primarily as transcriptional repressors, although the presence of two cryptic minor transactivation domains may indeed allow them to transactivate a distinct subset of genes (7,9,10). Adding to the complexity, at least three variants (α, β and γ) are generated by alternative splicing within the 3′ part of the gene, coding for the C-terminus, and all can occur in combination with the N-terminus-complete (TAp63) and N-terminus-deleted (ΔNp63) variants, thus adding up to at least six isoforms (11). A recent comprehensive study has revealed that almost 2000 genes are bound by p63 in a human cervical carcinoma line, and that ∼800 of these are differentially expressed in response to active p63 (12). Among the sequences regulated by both p63 and p53 are the genes for the cyclin-dependent kinase inhibitor p21Waf/Cip1 (CDKN1A), the ubiquitin ligase MDM2, and the pro-apoptotic proteins Bax, Noxa, PERP and PUMA. Many other targets, including the gene for the SMARCD3 regulator of chromatin, are specifically regulated by p63 (6,8,13). Physiologically, p63 function is critical, for instance, for epithelial stem cell maintenance (4,14), squamous epithelial differentiation and skin renewal (15–17), and for the efficient apoptosis of developing sympathetic neurons (18).p300/CBP is a transcriptional co-activator that acts as a bridging factor between various transactivators and the basal transcription machinery in response to specific signals. It is also a histone acetyltransferase (HAT) that transfers an acetyl group to the ε-amino group of a lysine residue (19–21). Gene transcription by the p53 family of proteins (comprising p53, p63 and p73) is regulated by p300/CBP (22–26). Besides histones, p300 can acetylate non-histone substrates, including p53, p63 and p73 themselves. Acetylation of p53 (27) and TAp63 (22), but not ΔNp63 (22) and p73 (28), facilitates the transcriptional activation of a major target, the p21Waf/Cip1 gene, whereas acetylation of p73 seems to specifically influence the expression of some pro-apoptotic genes upon DNA damage (29).NIR was recently discovered by us as being a new member of the INHAT (inhibitors of HATs) family of co-repressors (30). INHATs were first described as part of a multiprotein complex capable of inhibiting the p300/CBP and pCAF effects on histones (31). The active moiety Set/TAFß1 was reported to sterically hinder HAT activity by direct binding to histone substrates. Set/TAFb1 as well as pp32, another member of the INHAT family (32), and NIR (30) preferentially associate with hypoacetylated histones and are inhibited by H3 and H4 acetylation (30,33,34). However, in contrast to the previously identified INHATs, NIR does not seem to coexist in complexes with histone deacetylases (HDACs) (30). NIR binds to the p53 tumor suppressor and is recruited by it to p53-regulated genes. Here, we show that NIR is controlled by the cell cycle and associates with and regulates the TAp63 regulator of differentiation and stem cells.

INTRODUCTION

p63 is a close relative of the p53 tumor suppressor and transcription factor. p53 stimulates, upon potentially tumor-inducing damage, the transcription of cell cycle arrest and pro-apoptotic genes among numerous others (1). Both family members can target a common subset of genes in response to damage, and p63 is also involved in tumor suppression and progression, although more subtly (2,3). In contrast to p53, however, p63 exerts an important function during development—in the morphogenesis of stratified epithelia (4). Point mutations in the p63 gene have been associated with ectodermal dysplasias in humans (5). It is therefore no surprise that many target genes are unique to each transcription factor. Tetrameric p63 seems to preferentially bind to two consecutive 10-mer sequence motifs with the consensus (rrrCGTGyyy), (t/a,a/t,a,C,A/T,T,G,t,t/a,t), or (rrrC,A/G,T/A,Gyyy), whereas tetrameric p53 preferentially recognizes (rrrC,A/T,A/T,Gyyy) (r = purines; y = pyrimidines). Both proteins apparently prefer spacing between the 10-mers not exceeding 0–2 bp (1,6–8).

p63 exhibits a high degree of molecular complexity. The presence in the p63 gene of an internal promoter in intron 3 can give rise to proteins with the major transactivation domain deleted (ΔNp63). These DNA binding-active, transactivation-impaired isoforms seem to act primarily as transcriptional repressors, although the presence of two cryptic minor transactivation domains may indeed allow them to transactivate a distinct subset of genes (7,9,10). Adding to the complexity, at least three variants (α, β and γ) are generated by alternative splicing within the 3′ part of the gene, coding for the C-terminus, and all can occur in combination with the N-terminus-complete (TAp63) and N-terminus-deleted (ΔNp63) variants, thus adding up to at least six isoforms (11). A recent comprehensive study has revealed that almost 2000 genes are bound by p63 in a human cervical carcinoma line, and that ∼800 of these are differentially expressed in response to active p63 (12). Among the sequences regulated by both p63 and p53 are the genes for the cyclin-dependent kinase inhibitor p21Waf/Cip1 (CDKN1A), the ubiquitin ligase MDM2, and the pro-apoptotic proteins Bax, Noxa, PERP and PUMA. Many other targets, including the gene for the SMARCD3 regulator of chromatin, are specifically regulated by p63 (6,8,13). Physiologically, p63 function is critical, for instance, for epithelial stem cell maintenance (4,14), squamous epithelial differentiation and skin renewal (15–17), and for the efficient apoptosis of developing sympathetic neurons (18).

p300/CBP is a transcriptional co-activator that acts as a bridging factor between various transactivators and the basal transcription machinery in response to specific signals. It is also a histone acetyltransferase (HAT) that transfers an acetyl group to the ε-amino group of a lysine residue (19–21). Gene transcription by the p53 family of proteins (comprising p53, p63 and p73) is regulated by p300/CBP (22–26). Besides histones, p300 can acetylate non-histone substrates, including p53, p63 and p73 themselves. Acetylation of p53 (27) and TAp63 (22), but not ΔNp63 (22) and p73 (28), facilitates the transcriptional activation of a major target, the p21Waf/Cip1 gene, whereas acetylation of p73 seems to specifically influence the expression of some pro-apoptotic genes upon DNA damage (29).

NIR was recently discovered by us as being a new member of the INHAT (inhibitors of HATs) family of co-repressors (30). INHATs were first described as part of a multiprotein complex capable of inhibiting the p300/CBP and pCAF effects on histones (31). The active moiety Set/TAFß1 was reported to sterically hinder HAT activity by direct binding to histone substrates. Set/TAFb1 as well as pp32, another member of the INHAT family (32), and NIR (30) preferentially associate with hypoacetylated histones and are inhibited by H3 and H4 acetylation (30,33,34). However, in contrast to the previously identified INHATs, NIR does not seem to coexist in complexes with histone deacetylases (HDACs) (30). NIR binds to the p53 tumor suppressor and is recruited by it to p53-regulated genes. Here, we show that NIR is controlled by the cell cycle and associates with and regulates the TAp63 regulator of differentiation and stem cells.

MATERIALS AND METHODSPlasmids, chemicals and antibodiespGEX-p53 was kindly provided by Lienhart Schmitz (Department of Chemistry, University of Bern, Bern, Switzerland). All GST-p63 deletion mutants, GST-TAp63γ full-length as well as GST-ΔNp63α were generated by polymerase chain reaction (PCR) and cloned into pGEX-4T1 (Amersham Biosciences). pRTS-1 was kindly provided by Georg Bornkamm (GSF Institute of Clinical Molecular Biology, Muinich, Germany). pRTS-TAp63γ and pRTS-p53 were generated by PCR and cloning into pRTS-1. Cloning details are available upon request. All constructs were verified by sequencing. pCMX-myc-NIR full-length and pCMX-Flag-NIR full-length were constructed as reported previously (30). pRcCMV-p51A (corresponds to Homo sapiens TAp63γ) was kindly provided by Matthias Dobbelstein (Molecular Oncology, University of Goettingen, Goettingen, Germany). The drugs doxycycline, actinomycin D, adriamycin (ADR), cycloheximide, 5-fluorouracil, MG132, methyl methanesulfonate and mycophenolic acid were from Sigma (St Louis, USA), as were the protease inhibitor cocktail, 4′,6-diamidin-2′-phenylindoldihydrochlorid (DAPI), the ß-actin monoclonal antibody, the TRITC-conjugated secondary anti-mouse and the peroxidase-conjugated secondary anti-mouse and anti-rabbit antibodies. Hygromycin B and the transfection reagent Nanofectin I were from PAA (Pasching, Austria). RNAifect was from Qiagen (Duesseldorf, Germany). The p63 monoclonal antibody 4A4 was purchased from Chemicon/Millipore (Billerica, USA). The p53 monoclonal antibody DO-1 was purchased from Calbiochem (San Diego, USA). The p21 monoclonal antibody SX118 was from BD Pharmingen (Franklin Lakes, USA). The secondary Alexa Fluor 488-labeled rabbit antibody was from Invitrogen/Molecular Probes (Carlsbad, USA).Cell culture, transfection and synchronizationAll cells were maintained at 37°C in a 7% CO2 atmosphere. H1299 and HaCaT cells were maintained in DMEM (PAA, Pasching, Austria) with 10% fetal calf serum. HCT116 cells were maintained in McCoy’s 5A Medium (Sigma, St Louis, USA) with 10% fetal calf serum. Human primary fibroblasts were maintained in MEM with Earle’s Salts (PAA, Pasching, Austria) supplemented with 10% fetal calf serum, 1% non-essential amino acid solution (Sigma, St Louis, USA) and 1% sodium pyruvate (PAA, Pasching, Austria). For transient transfection, cells were seeded to a confluency of 60–70 % and were transfected with Nanofectin I (PAA) following the manufacturer’s recommendation. For stable transfection, H1299 cells were seeded to a confluency of 60–70% and were transfected with the doxycycline-inducible pRTS-TAp63γ. 24 h after transfection cells were selected in 400 µg/ml hygromycin B (PAA, Pasching, Austria). Within the next 2–3 weeks, hygromycin B-resistant cells grew out and were expanded. The stable cell line was maintained in culture under hygromycin B selection in the absence of doxycycline. For synchronization of human diploid fibroblast cultures, dishes with contact-inhibited cells were split 1 : 2 and incubated for 24 h in medium supplemented with 0.1% FCS. Medium was then replaced by full medium plus 10 ng/ml FGF (Biomol, Hamburg, Germany) for 6 h, and supplemented with 0.25 mM mimosine (Sigma, St. Louis, USA) for 17 h. Cultures were then washed several times in phosphate-buffered saline (PBS; to remove mimosine) and fed with full medium (time point: 0 h). The cells were harvested after the indicated times for FACS, RNA and protein analysis.GST pull-down assayThe GST-fusion proteins were expressed in Escherichia coli by transformation with the appropriate plasmids (available upon request). Expression was induced by treating exponentially growing cultures with isopropyl-1-thio-β-d-galactopyranoside (100 nM) for 4 h, and the cell pellets were then resuspended in GST-low salt buffer [20 mM Tris/HCl pH 7.5, 100 mM NaCl, 1 mM EDTA, 1% NP40, 2 mM DTT, protease inhibitor cocktail (Sigma)], supplemented with lysozyme to a final concentration of 50 μg/ml. The extracts were sonicated for 1 min, cleared by centrifugation and were then incubated for 30 min with Glutathione-Sepharose beads (50% slurry; GE-Healthcare) and were washed four times with GST-0.4 M salt buffer (50 mM Tris/HCl, 400 mM NaCl, 2 mM EDTA, 0.5% NP40). Quantities of bead suspension loaded with approximately equal amounts of the fusion proteins [tested by sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE)] were then incubated over night at 4°C with equal amounts of in vitro translated radiolabeled NIR proteins (2 μCi/reaction). The in vitro translation was performed with a TNT-T7 Coupled Reticulocyte Lysate System, according to the manufacturer’s protocol (Promega), using 2–4 µg plasmid and 35S radiolabeled cysteine and methionine (20 μCi). After incubation, all probes were washed four times with GST-0.4 M salt buffer. Then GST protein complexes were eluted from the Sepharose beads by adding SDS-sample buffer and boiling samples for 10 min. The proteins were separated by SDS–PAGE and visualized by autoradiography.Quantitative RNA analyses and statisticsCells were seeded in 10 cm-, 6 cm- or 35-mm dishes and treated with the indicated reagents 24 h later. At the indicated time points, medium was removed and solution D (236.4 g guanidium thiocyanate in 293 ml water, 17.6 ml 0.75 M sodium citrate pH 7.0 and 26.4 ml 10% sarcosyl, 0.72% 2-mercaptoethanol) was added. Cells were scraped off and 0.1 ml of 2 M sodium acetate pH 4.0, 1 ml of water-saturated phenol and 0.2 ml of chloroform–isoamylalcohol (49 : 1) were added, mixed and cooled on ice for 15 min. After centrifugation (10 000g, 20 min, 4°C), the aqueous phase was collected and precipitated overnight with isopropanol at –20°C. After a further centrifugation (10 000g, 20 min, 4°C), RNA was re-dissolved in Solution D and precipitated with isopropanol at –20°C for 1 h. The pellet was washed in 70% ethanol and dissolved in DEPC-water. The RNA was digested with RNase-free DNase I (Roche, Germany) for 60 min at 37°C, and 1.5–5 µg was used for the first-strand cDNA synthesis with SuperScriptTMIII (Invitrogen, USA) as specified by the manufacturer. Semiquantitative real-time (RT)-PCR analysis was performed with AmpliTaqR Gold DNA polymerase (Applied Biosystems, USA), using the following primers: p21 (for: ggcggcagaccagcatgacagatt; rev: atgaagccggcccacccaacctc; TA: 64°C), NIR (for: cagctggtgtcctgtctgtc; rev: gcagtgcacatactgccagt; TA: 58°C), p63γ (for: ttagcatggactgtatccgc; rev: aagctcattcctgaagcagg; TA: 60°C), gapdh (for: tggtatcgtggaaggactcatgac; rev: agtccagtgagcttcccgttcagc; TA: 64°C). Quantitative RT-PCR for p21, NIR and gapdh was performed with LightCycler® FastStart DNA Master SYBR Green I (Roche, Mannheim, Germany); the primers shown above were used at a final concentration of 0.5 µM (TA: 64°C, MgCl2 2 mM). Statistical analyses were performed with GraphPad QuickCalcs and Sigma Plot; P-values <0.05 were considered statistically significant.NIR knockdownShort interfering RNAs [siRNA; NIR1- sense: r(GACCUGAACUUCCCAGAGA)dTdT; antisense: r(UCUCUGGGAAGUUCAGGUC)dTdT; NIR2- sense: r(GACAGGAAGGAUGAAGACA)dTdT; antisense: r(UGUCUUCAUCCUUCCUGUC)dTdT] were used to silence gene expression in the indicated cell lines. As a control, an unrelated siRNA [control-sense: r(UUCUCCGAACGUGUCACGU)dTdT; antisense: r(ACGUGACACGUUCGGAGAA)dTdT] was used. Exponentially growing cells were transfected with siRNA (40 nM) by RNAifect (Qiagen, Germany). With a routine transfection efficiency for siRNA of >90 %, a knockdown of NIR could be observed as early as 24 h after transfection.Protein extraction and western blot analysisCells were lysed in standard SDS-lysis buffer heated to 100°C, containing 100 mM Tris–HCl (pH 6.8), 100 mM DTT, 4% SDS and 20% glycerol. Fifteen micrograms of protein were subjected to 8–13 % SDS-PAGE and transferred to a polyvinylidene fluoride (PVDF) membrane (Immobilon-P; Millipore, USA). Signals were detected upon overnight incubation of the membranes with one of the indicated antibodies (α-NIR 1 : 1000; α-p63 1 : 1000, α-actin 1 : 10 000, α-p21 1 : 1000), followed by a final incubation with a peroxidase-conjugated secondary anti-mouse (1 : 2000) or anti-rabbit (1 : 2000) antibody and Pierce ECL Western Blotting substrate (Rockford, USA), performed as specified by the supplier.Co-immunoprecipitationFor co-immunoprecipitation analysis, H1299 cells containing an inducible pRTS1 vector with a HA-tagged TAp63γ transgene were induced with 1 μg/ml doxycyclin for 12 h. H1299 control cells harbored TAp63γ in reverse orientation. All cells were harvested and washed twice in sterile PBS. Subsequently, 9/10 of the cells were lysed for 1 h in BC100 buffer (50 mM Tris–HCl pH 7.4, 100 mM NaCl, 10% glycerol and 0.2% Triton-X100) with freshly added protease inhibitors. The remaining 1/10 of the cells was lysed in SDS-sample buffer and served as an input control. The BC100 extracts were sonicated for 5 min, divided into two equal parts and incubated with 4 µg anti-HA antibody (Sigma, H6908) or 4 µg of irrelevant mouse antibody. The extracts were then incubated over night on a rotating wheel. A mixture of protein A, G agarose/sepharose beads in a 1:1 ratio was added to each tube. The samples were incubated on a rotating wheel for another 4 h. Then extracts were washed three times with a buffer containing 50 mM Tris/HCl pH 7.4, 100 mM NaCl, 2 mM EDTA, 0.5% NP-40 and freshly added protease inhibitors. Finally, SDS-sample buffer was added to the Sepharose beads. All probes were boiled for 10 min and subjected to a 10% SDS-PAGE for western blot analysis.Chromatin immunoprecipitationChromatin immunoprecipitation (ChIP) on HaCaT and H1299 cells was performed essentially as suggested by the Upstate ChIP protocol, with few variations. H1299-pRTS-TAp63γ cells were transfected for 24 h with 4 μg of plasmid producing NIR per 2 × 106 cells, and were induced to express TAp63γ for 12 h. Expression was monitored by immunoblotting. For ChIP analysis, an additional quenching step with glycine (final concentration 0.125 M) was inserted after formaldehyde fixation of the cells. Extracts were diluted 5-fold in ChIP dilution buffer. For immunoprecipitation, a mixture of salmon sperm DNA protein A agarose (Upstate), gammaBindG Sepharose and protein G Sepharose (both GE Healthcare) were used in a 1 : 1 : 1 ratio. To block unspecific binding of proteins, BSA (final concentration 1%; NEB) was added to all agarose/Sepharose mixtures for 1 h prior to IP. For ChIP on HaCaT cells, washing was done with ChIP-dilution buffer (3×; Upstate), low-salt immune complex wash buffer (3×), high-salt immune complex wash buffer (3×), LiCl immune complex wash buffer (3×) and, finally, TE buffer (5×). The sequential washing for ChIP on H1299 cells involved washing in dilution buffer (1×), low-salt buffer (2×), high-salt buffer (3×), LiCl buffer (2×) and TE buffer (2×). Reverse cross-linking was done over night. Immunoprecipitations were performed with 2 µg anti-p63 antibody 4A4, anti-p53 antibody DO-1 or anti NIR antibody 2910, or with mouse and rabbit irrelevant antibodies. PCR conditions and primers were described before (30). Primer pair one (PP1) for the detection of p21 promoter sequences was: (for: acctttcaccattcccctac); (rev: gcccaaggacaaaatagcca). Primer pair two (PP2) was: (for: ccttagcctgttactctgaac); (rev: tgcaaccatgcacttgaatgtg). The primer pair to detect the U6 gene promoter was: (for: ggcctatttcccatgattcc); (rev: atttgcgtgtcatccttgc).Immunofluorescence analysesCells were grown on glass coverslips or 4 Chamber Polystyrene Vessel Tissue Culture Treated Glass Slides (BD Falcon, Franklin Lakes, USA), and 24 h after seeding they were either treated with the indicated drugs or transfected with TAp63γ- or NIR-expression plasmid. Cells were fixed with paraformaldehyde (4% in PBS), permeabilized on ice with 0.2% Triton-X/PBS, blocked with 1 µg/ml BSA (Merck, Darmstadt, Germany) for 15 min at 37°C and then incubated with the primary antibody (α-NIR 1 : 100; α-p63 1 : 300) for 45 min. After several washings with PBS and 0.03% Triton-X/PBS the cells were incubated with the secondary antibody (TRITC-conjugated anti-mouse 1 : 100; Alexa488-conjugated anti-rabbit 1 : 7000) for 30 min. DNA was stained with 0.2 µg/ml DAPI (4′,6-diamidin-2′-phenylindoldihydrochlorid, Sigma, USA) in methanol to visualize the nuclei. Intracellular localization was studied with a Leica DM IRB/E fluorescence microscope equipped with an Axio Cam color camera (Zeiss) and was analyzed with the Axio Vision 3.0 software.

MATERIALS AND METHODS

Plasmids, chemicals and antibodiespGEX-p53 was kindly provided by Lienhart Schmitz (Department of Chemistry, University of Bern, Bern, Switzerland). All GST-p63 deletion mutants, GST-TAp63γ full-length as well as GST-ΔNp63α were generated by polymerase chain reaction (PCR) and cloned into pGEX-4T1 (Amersham Biosciences). pRTS-1 was kindly provided by Georg Bornkamm (GSF Institute of Clinical Molecular Biology, Muinich, Germany). pRTS-TAp63γ and pRTS-p53 were generated by PCR and cloning into pRTS-1. Cloning details are available upon request. All constructs were verified by sequencing. pCMX-myc-NIR full-length and pCMX-Flag-NIR full-length were constructed as reported previously (30). pRcCMV-p51A (corresponds to Homo sapiens TAp63γ) was kindly provided by Matthias Dobbelstein (Molecular Oncology, University of Goettingen, Goettingen, Germany). The drugs doxycycline, actinomycin D, adriamycin (ADR), cycloheximide, 5-fluorouracil, MG132, methyl methanesulfonate and mycophenolic acid were from Sigma (St Louis, USA), as were the protease inhibitor cocktail, 4′,6-diamidin-2′-phenylindoldihydrochlorid (DAPI), the ß-actin monoclonal antibody, the TRITC-conjugated secondary anti-mouse and the peroxidase-conjugated secondary anti-mouse and anti-rabbit antibodies. Hygromycin B and the transfection reagent Nanofectin I were from PAA (Pasching, Austria). RNAifect was from Qiagen (Duesseldorf, Germany). The p63 monoclonal antibody 4A4 was purchased from Chemicon/Millipore (Billerica, USA). The p53 monoclonal antibody DO-1 was purchased from Calbiochem (San Diego, USA). The p21 monoclonal antibody SX118 was from BD Pharmingen (Franklin Lakes, USA). The secondary Alexa Fluor 488-labeled rabbit antibody was from Invitrogen/Molecular Probes (Carlsbad, USA).

Plasmids, chemicals and antibodies

pGEX-p53 was kindly provided by Lienhart Schmitz (Department of Chemistry, University of Bern, Bern, Switzerland). All GST-p63 deletion mutants, GST-TAp63γ full-length as well as GST-ΔNp63α were generated by polymerase chain reaction (PCR) and cloned into pGEX-4T1 (Amersham Biosciences). pRTS-1 was kindly provided by Georg Bornkamm (GSF Institute of Clinical Molecular Biology, Muinich, Germany). pRTS-TAp63γ and pRTS-p53 were generated by PCR and cloning into pRTS-1. Cloning details are available upon request. All constructs were verified by sequencing. pCMX-myc-NIR full-length and pCMX-Flag-NIR full-length were constructed as reported previously (30). pRcCMV-p51A (corresponds to Homo sapiens TAp63γ) was kindly provided by Matthias Dobbelstein (Molecular Oncology, University of Goettingen, Goettingen, Germany). The drugs doxycycline, actinomycin D, adriamycin (ADR), cycloheximide, 5-fluorouracil, MG132, methyl methanesulfonate and mycophenolic acid were from Sigma (St Louis, USA), as were the protease inhibitor cocktail, 4′,6-diamidin-2′-phenylindoldihydrochlorid (DAPI), the ß-actin monoclonal antibody, the TRITC-conjugated secondary anti-mouse and the peroxidase-conjugated secondary anti-mouse and anti-rabbit antibodies. Hygromycin B and the transfection reagent Nanofectin I were from PAA (Pasching, Austria). RNAifect was from Qiagen (Duesseldorf, Germany). The p63 monoclonal antibody 4A4 was purchased from Chemicon/Millipore (Billerica, USA). The p53 monoclonal antibody DO-1 was purchased from Calbiochem (San Diego, USA). The p21 monoclonal antibody SX118 was from BD Pharmingen (Franklin Lakes, USA). The secondary Alexa Fluor 488-labeled rabbit antibody was from Invitrogen/Molecular Probes (Carlsbad, USA).

Cell culture, transfection and synchronizationAll cells were maintained at 37°C in a 7% CO2 atmosphere. H1299 and HaCaT cells were maintained in DMEM (PAA, Pasching, Austria) with 10% fetal calf serum. HCT116 cells were maintained in McCoy’s 5A Medium (Sigma, St Louis, USA) with 10% fetal calf serum. Human primary fibroblasts were maintained in MEM with Earle’s Salts (PAA, Pasching, Austria) supplemented with 10% fetal calf serum, 1% non-essential amino acid solution (Sigma, St Louis, USA) and 1% sodium pyruvate (PAA, Pasching, Austria). For transient transfection, cells were seeded to a confluency of 60–70 % and were transfected with Nanofectin I (PAA) following the manufacturer’s recommendation. For stable transfection, H1299 cells were seeded to a confluency of 60–70% and were transfected with the doxycycline-inducible pRTS-TAp63γ. 24 h after transfection cells were selected in 400 µg/ml hygromycin B (PAA, Pasching, Austria). Within the next 2–3 weeks, hygromycin B-resistant cells grew out and were expanded. The stable cell line was maintained in culture under hygromycin B selection in the absence of doxycycline. For synchronization of human diploid fibroblast cultures, dishes with contact-inhibited cells were split 1 : 2 and incubated for 24 h in medium supplemented with 0.1% FCS. Medium was then replaced by full medium plus 10 ng/ml FGF (Biomol, Hamburg, Germany) for 6 h, and supplemented with 0.25 mM mimosine (Sigma, St. Louis, USA) for 17 h. Cultures were then washed several times in phosphate-buffered saline (PBS; to remove mimosine) and fed with full medium (time point: 0 h). The cells were harvested after the indicated times for FACS, RNA and protein analysis.

Cell culture, transfection and synchronization

All cells were maintained at 37°C in a 7% CO2 atmosphere. H1299 and HaCaT cells were maintained in DMEM (PAA, Pasching, Austria) with 10% fetal calf serum. HCT116 cells were maintained in McCoy’s 5A Medium (Sigma, St Louis, USA) with 10% fetal calf serum. Human primary fibroblasts were maintained in MEM with Earle’s Salts (PAA, Pasching, Austria) supplemented with 10% fetal calf serum, 1% non-essential amino acid solution (Sigma, St Louis, USA) and 1% sodium pyruvate (PAA, Pasching, Austria). For transient transfection, cells were seeded to a confluency of 60–70 % and were transfected with Nanofectin I (PAA) following the manufacturer’s recommendation. For stable transfection, H1299 cells were seeded to a confluency of 60–70% and were transfected with the doxycycline-inducible pRTS-TAp63γ. 24 h after transfection cells were selected in 400 µg/ml hygromycin B (PAA, Pasching, Austria). Within the next 2–3 weeks, hygromycin B-resistant cells grew out and were expanded. The stable cell line was maintained in culture under hygromycin B selection in the absence of doxycycline. For synchronization of human diploid fibroblast cultures, dishes with contact-inhibited cells were split 1 : 2 and incubated for 24 h in medium supplemented with 0.1% FCS. Medium was then replaced by full medium plus 10 ng/ml FGF (Biomol, Hamburg, Germany) for 6 h, and supplemented with 0.25 mM mimosine (Sigma, St. Louis, USA) for 17 h. Cultures were then washed several times in phosphate-buffered saline (PBS; to remove mimosine) and fed with full medium (time point: 0 h). The cells were harvested after the indicated times for FACS, RNA and protein analysis.

GST pull-down assayThe GST-fusion proteins were expressed in Escherichia coli by transformation with the appropriate plasmids (available upon request). Expression was induced by treating exponentially growing cultures with isopropyl-1-thio-β-d-galactopyranoside (100 nM) for 4 h, and the cell pellets were then resuspended in GST-low salt buffer [20 mM Tris/HCl pH 7.5, 100 mM NaCl, 1 mM EDTA, 1% NP40, 2 mM DTT, protease inhibitor cocktail (Sigma)], supplemented with lysozyme to a final concentration of 50 μg/ml. The extracts were sonicated for 1 min, cleared by centrifugation and were then incubated for 30 min with Glutathione-Sepharose beads (50% slurry; GE-Healthcare) and were washed four times with GST-0.4 M salt buffer (50 mM Tris/HCl, 400 mM NaCl, 2 mM EDTA, 0.5% NP40). Quantities of bead suspension loaded with approximately equal amounts of the fusion proteins [tested by sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE)] were then incubated over night at 4°C with equal amounts of in vitro translated radiolabeled NIR proteins (2 μCi/reaction). The in vitro translation was performed with a TNT-T7 Coupled Reticulocyte Lysate System, according to the manufacturer’s protocol (Promega), using 2–4 µg plasmid and 35S radiolabeled cysteine and methionine (20 μCi). After incubation, all probes were washed four times with GST-0.4 M salt buffer. Then GST protein complexes were eluted from the Sepharose beads by adding SDS-sample buffer and boiling samples for 10 min. The proteins were separated by SDS–PAGE and visualized by autoradiography.

GST pull-down assay

The GST-fusion proteins were expressed in Escherichia coli by transformation with the appropriate plasmids (available upon request). Expression was induced by treating exponentially growing cultures with isopropyl-1-thio-β-d-galactopyranoside (100 nM) for 4 h, and the cell pellets were then resuspended in GST-low salt buffer [20 mM Tris/HCl pH 7.5, 100 mM NaCl, 1 mM EDTA, 1% NP40, 2 mM DTT, protease inhibitor cocktail (Sigma)], supplemented with lysozyme to a final concentration of 50 μg/ml. The extracts were sonicated for 1 min, cleared by centrifugation and were then incubated for 30 min with Glutathione-Sepharose beads (50% slurry; GE-Healthcare) and were washed four times with GST-0.4 M salt buffer (50 mM Tris/HCl, 400 mM NaCl, 2 mM EDTA, 0.5% NP40). Quantities of bead suspension loaded with approximately equal amounts of the fusion proteins [tested by sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE)] were then incubated over night at 4°C with equal amounts of in vitro translated radiolabeled NIR proteins (2 μCi/reaction). The in vitro translation was performed with a TNT-T7 Coupled Reticulocyte Lysate System, according to the manufacturer’s protocol (Promega), using 2–4 µg plasmid and 35S radiolabeled cysteine and methionine (20 μCi). After incubation, all probes were washed four times with GST-0.4 M salt buffer. Then GST protein complexes were eluted from the Sepharose beads by adding SDS-sample buffer and boiling samples for 10 min. The proteins were separated by SDS–PAGE and visualized by autoradiography.

Quantitative RNA analyses and statisticsCells were seeded in 10 cm-, 6 cm- or 35-mm dishes and treated with the indicated reagents 24 h later. At the indicated time points, medium was removed and solution D (236.4 g guanidium thiocyanate in 293 ml water, 17.6 ml 0.75 M sodium citrate pH 7.0 and 26.4 ml 10% sarcosyl, 0.72% 2-mercaptoethanol) was added. Cells were scraped off and 0.1 ml of 2 M sodium acetate pH 4.0, 1 ml of water-saturated phenol and 0.2 ml of chloroform–isoamylalcohol (49 : 1) were added, mixed and cooled on ice for 15 min. After centrifugation (10 000g, 20 min, 4°C), the aqueous phase was collected and precipitated overnight with isopropanol at –20°C. After a further centrifugation (10 000g, 20 min, 4°C), RNA was re-dissolved in Solution D and precipitated with isopropanol at –20°C for 1 h. The pellet was washed in 70% ethanol and dissolved in DEPC-water. The RNA was digested with RNase-free DNase I (Roche, Germany) for 60 min at 37°C, and 1.5–5 µg was used for the first-strand cDNA synthesis with SuperScriptTMIII (Invitrogen, USA) as specified by the manufacturer. Semiquantitative real-time (RT)-PCR analysis was performed with AmpliTaqR Gold DNA polymerase (Applied Biosystems, USA), using the following primers: p21 (for: ggcggcagaccagcatgacagatt; rev: atgaagccggcccacccaacctc; TA: 64°C), NIR (for: cagctggtgtcctgtctgtc; rev: gcagtgcacatactgccagt; TA: 58°C), p63γ (for: ttagcatggactgtatccgc; rev: aagctcattcctgaagcagg; TA: 60°C), gapdh (for: tggtatcgtggaaggactcatgac; rev: agtccagtgagcttcccgttcagc; TA: 64°C). Quantitative RT-PCR for p21, NIR and gapdh was performed with LightCycler® FastStart DNA Master SYBR Green I (Roche, Mannheim, Germany); the primers shown above were used at a final concentration of 0.5 µM (TA: 64°C, MgCl2 2 mM). Statistical analyses were performed with GraphPad QuickCalcs and Sigma Plot; P-values <0.05 were considered statistically significant.

Quantitative RNA analyses and statistics

Cells were seeded in 10 cm-, 6 cm- or 35-mm dishes and treated with the indicated reagents 24 h later. At the indicated time points, medium was removed and solution D (236.4 g guanidium thiocyanate in 293 ml water, 17.6 ml 0.75 M sodium citrate pH 7.0 and 26.4 ml 10% sarcosyl, 0.72% 2-mercaptoethanol) was added. Cells were scraped off and 0.1 ml of 2 M sodium acetate pH 4.0, 1 ml of water-saturated phenol and 0.2 ml of chloroform–isoamylalcohol (49 : 1) were added, mixed and cooled on ice for 15 min. After centrifugation (10 000g, 20 min, 4°C), the aqueous phase was collected and precipitated overnight with isopropanol at –20°C. After a further centrifugation (10 000g, 20 min, 4°C), RNA was re-dissolved in Solution D and precipitated with isopropanol at –20°C for 1 h. The pellet was washed in 70% ethanol and dissolved in DEPC-water. The RNA was digested with RNase-free DNase I (Roche, Germany) for 60 min at 37°C, and 1.5–5 µg was used for the first-strand cDNA synthesis with SuperScriptTMIII (Invitrogen, USA) as specified by the manufacturer. Semiquantitative real-time (RT)-PCR analysis was performed with AmpliTaqR Gold DNA polymerase (Applied Biosystems, USA), using the following primers: p21 (for: ggcggcagaccagcatgacagatt; rev: atgaagccggcccacccaacctc; TA: 64°C), NIR (for: cagctggtgtcctgtctgtc; rev: gcagtgcacatactgccagt; TA: 58°C), p63γ (for: ttagcatggactgtatccgc; rev: aagctcattcctgaagcagg; TA: 60°C), gapdh (for: tggtatcgtggaaggactcatgac; rev: agtccagtgagcttcccgttcagc; TA: 64°C). Quantitative RT-PCR for p21, NIR and gapdh was performed with LightCycler® FastStart DNA Master SYBR Green I (Roche, Mannheim, Germany); the primers shown above were used at a final concentration of 0.5 µM (TA: 64°C, MgCl2 2 mM). Statistical analyses were performed with GraphPad QuickCalcs and Sigma Plot; P-values <0.05 were considered statistically significant.

NIR knockdownShort interfering RNAs [siRNA; NIR1- sense: r(GACCUGAACUUCCCAGAGA)dTdT; antisense: r(UCUCUGGGAAGUUCAGGUC)dTdT; NIR2- sense: r(GACAGGAAGGAUGAAGACA)dTdT; antisense: r(UGUCUUCAUCCUUCCUGUC)dTdT] were used to silence gene expression in the indicated cell lines. As a control, an unrelated siRNA [control-sense: r(UUCUCCGAACGUGUCACGU)dTdT; antisense: r(ACGUGACACGUUCGGAGAA)dTdT] was used. Exponentially growing cells were transfected with siRNA (40 nM) by RNAifect (Qiagen, Germany). With a routine transfection efficiency for siRNA of >90 %, a knockdown of NIR could be observed as early as 24 h after transfection.

NIR knockdown

Short interfering RNAs [siRNA; NIR1- sense: r(GACCUGAACUUCCCAGAGA)dTdT; antisense: r(UCUCUGGGAAGUUCAGGUC)dTdT; NIR2- sense: r(GACAGGAAGGAUGAAGACA)dTdT; antisense: r(UGUCUUCAUCCUUCCUGUC)dTdT] were used to silence gene expression in the indicated cell lines. As a control, an unrelated siRNA [control-sense: r(UUCUCCGAACGUGUCACGU)dTdT; antisense: r(ACGUGACACGUUCGGAGAA)dTdT] was used. Exponentially growing cells were transfected with siRNA (40 nM) by RNAifect (Qiagen, Germany). With a routine transfection efficiency for siRNA of >90 %, a knockdown of NIR could be observed as early as 24 h after transfection.

Protein extraction and western blot analysisCells were lysed in standard SDS-lysis buffer heated to 100°C, containing 100 mM Tris–HCl (pH 6.8), 100 mM DTT, 4% SDS and 20% glycerol. Fifteen micrograms of protein were subjected to 8–13 % SDS-PAGE and transferred to a polyvinylidene fluoride (PVDF) membrane (Immobilon-P; Millipore, USA). Signals were detected upon overnight incubation of the membranes with one of the indicated antibodies (α-NIR 1 : 1000; α-p63 1 : 1000, α-actin 1 : 10 000, α-p21 1 : 1000), followed by a final incubation with a peroxidase-conjugated secondary anti-mouse (1 : 2000) or anti-rabbit (1 : 2000) antibody and Pierce ECL Western Blotting substrate (Rockford, USA), performed as specified by the supplier.

Protein extraction and western blot analysis

Cells were lysed in standard SDS-lysis buffer heated to 100°C, containing 100 mM Tris–HCl (pH 6.8), 100 mM DTT, 4% SDS and 20% glycerol. Fifteen micrograms of protein were subjected to 8–13 % SDS-PAGE and transferred to a polyvinylidene fluoride (PVDF) membrane (Immobilon-P; Millipore, USA). Signals were detected upon overnight incubation of the membranes with one of the indicated antibodies (α-NIR 1 : 1000; α-p63 1 : 1000, α-actin 1 : 10 000, α-p21 1 : 1000), followed by a final incubation with a peroxidase-conjugated secondary anti-mouse (1 : 2000) or anti-rabbit (1 : 2000) antibody and Pierce ECL Western Blotting substrate (Rockford, USA), performed as specified by the supplier.

Co-immunoprecipitationFor co-immunoprecipitation analysis, H1299 cells containing an inducible pRTS1 vector with a HA-tagged TAp63γ transgene were induced with 1 μg/ml doxycyclin for 12 h. H1299 control cells harbored TAp63γ in reverse orientation. All cells were harvested and washed twice in sterile PBS. Subsequently, 9/10 of the cells were lysed for 1 h in BC100 buffer (50 mM Tris–HCl pH 7.4, 100 mM NaCl, 10% glycerol and 0.2% Triton-X100) with freshly added protease inhibitors. The remaining 1/10 of the cells was lysed in SDS-sample buffer and served as an input control. The BC100 extracts were sonicated for 5 min, divided into two equal parts and incubated with 4 µg anti-HA antibody (Sigma, H6908) or 4 µg of irrelevant mouse antibody. The extracts were then incubated over night on a rotating wheel. A mixture of protein A, G agarose/sepharose beads in a 1:1 ratio was added to each tube. The samples were incubated on a rotating wheel for another 4 h. Then extracts were washed three times with a buffer containing 50 mM Tris/HCl pH 7.4, 100 mM NaCl, 2 mM EDTA, 0.5% NP-40 and freshly added protease inhibitors. Finally, SDS-sample buffer was added to the Sepharose beads. All probes were boiled for 10 min and subjected to a 10% SDS-PAGE for western blot analysis.

Co-immunoprecipitation

For co-immunoprecipitation analysis, H1299 cells containing an inducible pRTS1 vector with a HA-tagged TAp63γ transgene were induced with 1 μg/ml doxycyclin for 12 h. H1299 control cells harbored TAp63γ in reverse orientation. All cells were harvested and washed twice in sterile PBS. Subsequently, 9/10 of the cells were lysed for 1 h in BC100 buffer (50 mM Tris–HCl pH 7.4, 100 mM NaCl, 10% glycerol and 0.2% Triton-X100) with freshly added protease inhibitors. The remaining 1/10 of the cells was lysed in SDS-sample buffer and served as an input control. The BC100 extracts were sonicated for 5 min, divided into two equal parts and incubated with 4 µg anti-HA antibody (Sigma, H6908) or 4 µg of irrelevant mouse antibody. The extracts were then incubated over night on a rotating wheel. A mixture of protein A, G agarose/sepharose beads in a 1:1 ratio was added to each tube. The samples were incubated on a rotating wheel for another 4 h. Then extracts were washed three times with a buffer containing 50 mM Tris/HCl pH 7.4, 100 mM NaCl, 2 mM EDTA, 0.5% NP-40 and freshly added protease inhibitors. Finally, SDS-sample buffer was added to the Sepharose beads. All probes were boiled for 10 min and subjected to a 10% SDS-PAGE for western blot analysis.

Chromatin immunoprecipitationChromatin immunoprecipitation (ChIP) on HaCaT and H1299 cells was performed essentially as suggested by the Upstate ChIP protocol, with few variations. H1299-pRTS-TAp63γ cells were transfected for 24 h with 4 μg of plasmid producing NIR per 2 × 106 cells, and were induced to express TAp63γ for 12 h. Expression was monitored by immunoblotting. For ChIP analysis, an additional quenching step with glycine (final concentration 0.125 M) was inserted after formaldehyde fixation of the cells. Extracts were diluted 5-fold in ChIP dilution buffer. For immunoprecipitation, a mixture of salmon sperm DNA protein A agarose (Upstate), gammaBindG Sepharose and protein G Sepharose (both GE Healthcare) were used in a 1 : 1 : 1 ratio. To block unspecific binding of proteins, BSA (final concentration 1%; NEB) was added to all agarose/Sepharose mixtures for 1 h prior to IP. For ChIP on HaCaT cells, washing was done with ChIP-dilution buffer (3×; Upstate), low-salt immune complex wash buffer (3×), high-salt immune complex wash buffer (3×), LiCl immune complex wash buffer (3×) and, finally, TE buffer (5×). The sequential washing for ChIP on H1299 cells involved washing in dilution buffer (1×), low-salt buffer (2×), high-salt buffer (3×), LiCl buffer (2×) and TE buffer (2×). Reverse cross-linking was done over night. Immunoprecipitations were performed with 2 µg anti-p63 antibody 4A4, anti-p53 antibody DO-1 or anti NIR antibody 2910, or with mouse and rabbit irrelevant antibodies. PCR conditions and primers were described before (30). Primer pair one (PP1) for the detection of p21 promoter sequences was: (for: acctttcaccattcccctac); (rev: gcccaaggacaaaatagcca). Primer pair two (PP2) was: (for: ccttagcctgttactctgaac); (rev: tgcaaccatgcacttgaatgtg). The primer pair to detect the U6 gene promoter was: (for: ggcctatttcccatgattcc); (rev: atttgcgtgtcatccttgc).

Chromatin immunoprecipitation

Chromatin immunoprecipitation (ChIP) on HaCaT and H1299 cells was performed essentially as suggested by the Upstate ChIP protocol, with few variations. H1299-pRTS-TAp63γ cells were transfected for 24 h with 4 μg of plasmid producing NIR per 2 × 106 cells, and were induced to express TAp63γ for 12 h. Expression was monitored by immunoblotting. For ChIP analysis, an additional quenching step with glycine (final concentration 0.125 M) was inserted after formaldehyde fixation of the cells. Extracts were diluted 5-fold in ChIP dilution buffer. For immunoprecipitation, a mixture of salmon sperm DNA protein A agarose (Upstate), gammaBindG Sepharose and protein G Sepharose (both GE Healthcare) were used in a 1 : 1 : 1 ratio. To block unspecific binding of proteins, BSA (final concentration 1%; NEB) was added to all agarose/Sepharose mixtures for 1 h prior to IP. For ChIP on HaCaT cells, washing was done with ChIP-dilution buffer (3×; Upstate), low-salt immune complex wash buffer (3×), high-salt immune complex wash buffer (3×), LiCl immune complex wash buffer (3×) and, finally, TE buffer (5×). The sequential washing for ChIP on H1299 cells involved washing in dilution buffer (1×), low-salt buffer (2×), high-salt buffer (3×), LiCl buffer (2×) and TE buffer (2×). Reverse cross-linking was done over night. Immunoprecipitations were performed with 2 µg anti-p63 antibody 4A4, anti-p53 antibody DO-1 or anti NIR antibody 2910, or with mouse and rabbit irrelevant antibodies. PCR conditions and primers were described before (30). Primer pair one (PP1) for the detection of p21 promoter sequences was: (for: acctttcaccattcccctac); (rev: gcccaaggacaaaatagcca). Primer pair two (PP2) was: (for: ccttagcctgttactctgaac); (rev: tgcaaccatgcacttgaatgtg). The primer pair to detect the U6 gene promoter was: (for: ggcctatttcccatgattcc); (rev: atttgcgtgtcatccttgc).

Immunofluorescence analysesCells were grown on glass coverslips or 4 Chamber Polystyrene Vessel Tissue Culture Treated Glass Slides (BD Falcon, Franklin Lakes, USA), and 24 h after seeding they were either treated with the indicated drugs or transfected with TAp63γ- or NIR-expression plasmid. Cells were fixed with paraformaldehyde (4% in PBS), permeabilized on ice with 0.2% Triton-X/PBS, blocked with 1 µg/ml BSA (Merck, Darmstadt, Germany) for 15 min at 37°C and then incubated with the primary antibody (α-NIR 1 : 100; α-p63 1 : 300) for 45 min. After several washings with PBS and 0.03% Triton-X/PBS the cells were incubated with the secondary antibody (TRITC-conjugated anti-mouse 1 : 100; Alexa488-conjugated anti-rabbit 1 : 7000) for 30 min. DNA was stained with 0.2 µg/ml DAPI (4′,6-diamidin-2′-phenylindoldihydrochlorid, Sigma, USA) in methanol to visualize the nuclei. Intracellular localization was studied with a Leica DM IRB/E fluorescence microscope equipped with an Axio Cam color camera (Zeiss) and was analyzed with the Axio Vision 3.0 software.

Immunofluorescence analyses

Cells were grown on glass coverslips or 4 Chamber Polystyrene Vessel Tissue Culture Treated Glass Slides (BD Falcon, Franklin Lakes, USA), and 24 h after seeding they were either treated with the indicated drugs or transfected with TAp63γ- or NIR-expression plasmid. Cells were fixed with paraformaldehyde (4% in PBS), permeabilized on ice with 0.2% Triton-X/PBS, blocked with 1 µg/ml BSA (Merck, Darmstadt, Germany) for 15 min at 37°C and then incubated with the primary antibody (α-NIR 1 : 100; α-p63 1 : 300) for 45 min. After several washings with PBS and 0.03% Triton-X/PBS the cells were incubated with the secondary antibody (TRITC-conjugated anti-mouse 1 : 100; Alexa488-conjugated anti-rabbit 1 : 7000) for 30 min. DNA was stained with 0.2 µg/ml DAPI (4′,6-diamidin-2′-phenylindoldihydrochlorid, Sigma, USA) in methanol to visualize the nuclei. Intracellular localization was studied with a Leica DM IRB/E fluorescence microscope equipped with an Axio Cam color camera (Zeiss) and was analyzed with the Axio Vision 3.0 software.

RESULTSNIR is a novel interaction partner of transcription factor TAp63Our initial objective was to identify new protein interactions involved in transcriptional regulation by the p53 protein family. Since this family recruits to promoters, and is regulated by HATs, low-stringency in silico analyses for novel proteins harboring putative INHAT (inhibitor of HAT) domains were performed. This resulted in the cloning of NIR (Novel INHAT Repressor), and subsequent work showed that NIR can indeed associate with, and regulate, p53 (30). Here, we asked whether NIR would also be to regulate p63, a transcription factor with the highest homology to p53 in its central DNA-binding domain (63%) and important for the development of stratified epithelia as well as for epithelial stem cell maintenance.First, in vitro synthesized, 35S-labeled full-length NIR or NIR (147–609) consisting of the central portion of the protein lacking both INHAT domains, were employed in GST pull-down assays with either GST alone, p53 or TAp63γ (the major transactivating form of p63 containing the N-terminal transactivation domain, central DNA-binding domain and C-terminal oligomerization domain). We observed that both NIR and NIR (147–609) bound to either transcription factor but not GST alone (Figure 1A). Next, we sought to map the binding of NIR to TAp63γ. For this purpose, full-length 35S-labeled NIR was used in GST pull-down assays with either GST alone or one of the TAp63γ fragments depicted in Figure 1B. The assay revealed that NIR binds to two domains on TAp63γ; the transactivation domain and the oligomerization domain. However, NIR failed to bind to the ΔNp63 isoform in this assay, suggesting that robust binding of NIR and TAp63γ requires the transactivation domain. NIR also failed to associate with the core DNA binding domain and, as expected, with GST alone. To confirm the interaction between NIR and TAp63 in vivo, co-immunoprecipitations were carried out. Of note in this context, endogenous NIR is predominantly localized in the nucleoli, and mostly in the outer granular region where ribosome subunit assembly takes place. A small portion of NIR can be visualized in dot-like structures in the nucleoplasm. In contrast, TAp63γ is exclusively localized in the nucleoplasm (Figure 1C). This subcellular distribution, combined with the fact that stable expression of TAp63γ provoked high levels of apoptotic death in our cell system, led us to expect that endogenous NIR (90 kDa; detected as a single band by polyclonal anti-NIR antibody 2910) would be difficult to co-immunoprecipitate with transfected TAp63γ. We therefore generated H1299 cultures in which an HA-tagged TAp63γ was conditionally expressed (see below). In this system, endogenous NIR could be co-precipitated with TAp63γ, whereas NIR failed to come down in the absence of TAp63γ (Figure 1D). Combined, these findings show that NIR and TAp63γ can physically associate.
Figure 1.NIR associates with TAp63γ and p53. (A) GST pull-down assay. (Upper panels) Equal load of GST-p53 (p53) and GST-TAp63γ(TAp63γ); M, marker. (Lower panels) In vitro translated 35S-labeled full-length NIR or NIR (147–609) central portion is retained by GST- TAp63γ (TAp63γ) and by GST-p53 (p53) but not GST alone (IN, Input). The scheme shows NIR with the N- and C-terminal INHAT domains and the nuclear localization signal (NLS). (B) 35S-labeled full-length NIR binds to the transactivation domain (TA) and the oligomerization domain (OD) of TAp63γ in GST pull-down assays. PR, proline-rich domain; DBD, DNA binding domain. Radiolabeled NIR fails to bind efficiently to ΔNp63. (C) Fluorescence microscopy showing subcellular localization of endogenous NIR (green), detected with polyclonal antibody 2910, and of transfected TAp63γ (red) in human H1299 lung adenocarcinoma cells deficient for p63 and p53. Note the predominant localization of NIR in the nucleoli (single arrow) and occasional cells with detectable staining for NIR in the nucleoplasm (double arrow). By contrast, TAp63γ was mostly nucleoplasmic. Cells were stained with antibody 2910 for NIR and antibody 4A4 for TAp63γ. (D) Co-immunoprecipitation of endogenous NIR and transfected HA-tagged TAp63γ with anti-HA antibody.Nucleoplasmic NIR acts as a repressor of TAp63NIR has been documented to act as a repressor of coactivator-stimulated transactivation (30). We therefore wished to test whether ectopic NIR would influence transcriptional activation by ectopic TAp63γ of the best-studied and most responsive endogenous p63 target, the gene for the p21Cip1 (CDKN1A) inhibitor of cyclin-dependent kinases (p21 hereafter). Transient transfections of NIR had documented that ectopic NIR, in contrast to the endogenous protein which is mostly present in the nucleoli (see above), is primarily localized in the nucleoplasm, as is TAp63γ (Figure 2A). In accord with its function as a transactivator of the p21 gene, transfected TAp63γ stimulated the transcription of the endogenous p21 gene ∼5-fold in RT-qPCR assays (Figure 2B). Simultaneous transfection of NIR, while not measurably affecting TAp63γ expression, blocked the transactivation of p21 almost completely, as predicted if NIR acted as a repressor of TAp63γ-mediated transactivation.
Figure 2.NIR can inhibit TAp63γ-induced transactivation and change subcellular localization under actinomycin D-induced stress. (A) In contrast to endogenous NIR, which was mostly present in the nucleoli (Figure 2B), transfected (Flag-tagged) NIR was predominantly nucleoplasmic. (B) When TAp63γ- and p53-deficient H1299 cells were transfected to produce TAp63γ, expression of the endogenous TAp63γ-target gene p21 was stimulated. Co-expression of TAp63γ and ectopic NIR (in addition to the endogenous) resulted in the inhibition of p21 expression. p21 transcript levels were determined by RT-qPCR. Error bars denote standard deviations derived from at least three experiments. (C) Fluorescence microscopy documenting the presence of NIR in the nucleoli in mock-treated human diploid fibroblasts (HDF) and H1299 cells. Treatment of the cultures with 1.0 μg/ml ActD for 6 h triggered NIR translocation to the nucleoplasm in both cell types.In a further set of experiments, we asked whether conditions can be found under which endogenous NIR would leave the nucleoli to localize to the nucleoplasm, and whether this would affect TAp63γ-regulated gene expression. For this purpose, human H1299 lung adenocarcinoma, human HCT116 colon adenocarcinoma, human immortal HaCaT keratinocytes and human diploid fibroblasts were either left untreated or were treated for 6 h with one of several stressors (Table 1). NIR was mostly nucleolar in all (100%) interphase (i.e. non-mitotic) cells when mock-treated (NIR localization in mitotic cells: see below). Among the stressors employed, only the nucleolar stressor and inhibitor of rRNA synthesis actinomcyin D (ActD) induced translocation of NIR from the nucleoli to the nucleoplasm in all interphase cells (100%), at a dose (0.5–1.0 μg/ml) well below the one needed to cause global transcription inhibition (>5–10 μg/ml) (Figure 2C). Nucleoplasmic localization of a subset of nucleolar proteins upon Act D treatment has been reported before (35); these, however, are not known to interact with TAp63. Some increase in nucleoplasmic NIR was also observed following ultraviolet (UV) irradiation (100 J/m2) and exposure to mycophenolic acid (25 μg/ml) in all four cell types. As ActD induced NIR translocation most consistently, this compound was used in the further studies.
Table 1.Subcellular localization of NIR after a 6 -h exposure to various stressorsStressorCell lineNucleoliNucleoplasmNoneH1299++++–HCT116++++–HaCaT++++–HDF++++–Actinomycin DH1299–++++(1 μg/ml)HCT116–++++HaCaT–++++HDF–++++Adriamycin (0.34 μM)H1299++++–HCT116++++–HaCaT++++–HDF++++–Cycloheximide (10 μg/ml)H1299++++–HCT116++++–HaCaT++++–HDF++++–5-Fluorouracil (0.375 mM)H1299++++HCT116++++HaCaT++++–HDF++++–H2O2 (0.4 mM)H1299++++HCT116ndndHaCaT++++–HDF++++MG132 (10 μM)H1299++++–HCT116++++–HaCaT++++–HDF++++MMS (0.1 mg/ml)H1299++++–HCT116++++–HaCaT++++–HDF++++Mycophenolic acid (25 μg/ml)H1299++++HCT116++++HaCaT++++HDF++++–UV (100 J/m2)H1299++++HCT116–++++HaCaT++++HDF++++–: less than 10% of total NIR stain, in the majority of cells (>90%), by visual inspection.++: approximately half of total NIR stain, in the majority of cells (>90%), by visual inspection.++++: more than 90% of total NIR stain, in the majority of cells (>90%), by visual inspection.nd: not determined.To test whether endogenous NIR, upon translocation from the nucleoli to the nucleoplasm, can regulate nucleoplasmic TAp63γ, and since TAp63γ overproduced in transient transfections was a powerful inducer of cell death, we established doxycyline (doxy)-inducible cell lines on the basis of vector pRTS1 (36). The advantage of this system, which employs a trans-repressor active in the absence of doxy and a transactivator active in its presence, was that transgene expression was very tightly controlled, with absolutely no detectable expression in doxy-free medium versus expression that could be finely tuned in the presence of defined doses of the drug. In these cells, doxy induced the production of TAp63γ with a HA-tag at the N-terminus, which in turn stimulated the expression of endogenous p21, indicative of HA-TAp63γ being functional as a transcription factor (Figure 3A). The levels of endogenous NIR remained unchanged.
Figure 3.Endogenous NIR translocated to the nucleoplasm upon ActD treatment was able to inhibit TAp63γ-induced gene expression. (A) TAp63γ- and p53-deficient H1299 cells were stably bulk-transfected with either vector alone or pRTS-TAp63γ. The HA-tagged TAp63γ-transgene was tightly controlled by doxycycline (doxy): in the absence of drug, no transgene expression was detectable whereas in the presence of 1.0 μg/ml doxy for at least 8 h, maximal transgene expression became visible in western blots with anti-HA antibody (1 : 1000). Furthermore, a p21 signal detected by anti-p21 antibody (1 : 1000) was found only in the presence of TAp63γ. The levels of NIR always remained constant; NIR was detected with the polyclonal antibody 2910. (B) RT-qPCR quantitation of p21 transcript levels in cultures with or without TAp63γ, and in the presence or absence of stress. ActD (0.5 μg/ml) strongly inhibited TAp63γ-induced transactivation while ADR (0.34 μM) inhibited only weakly. (C) RT-qPCR quantitation of gapdh transcript levels. ActD had no significant effects. Error bars show standard deviations derived from at least three experiments.In the following series of experiments, we examined the effect on TAp63γ-induced p21 expression of a low dose of ActD (0.5 µg/ml) previously determined to suffice to translocate nucleolar NIR to the nucleoplasm (see Figure 3C and Table 1). Notably, transactivation of p21 by TAp63γ was completely inhibited in the presence of nucleoplasmic NIR (P < 0.0001; Figure 3B). In contrast to ActD, the drug ADR, used at a dose known to cause significant DNA damage (0.3 μM) but unable to trigger NIR translocation (Table 1), inhibited p21 transactivation significantly (P < 0.001) but much weaker. To exclude that the effect of ActD on TAp63γ-induced p21 expression was the result of a general toxicity or impairment of gene transcription, gapdh expression was monitored in parallel. No inhibitory effect of ActD was discernible (Figure 3C). Combined, these data thus suggest that endogenous nucleoplasmic NIR can act as an inhibitor of TAp63γ.NIR knockdown can activate TAp63Cells tolerate neither permanent NIR knockdown nor overproduction (data not shown). To test whether temporary knockdown of NIR would affect transactivation of p21 by TAp63γ, we resorted to NIR knockdown by transient siRNA transfection. However, as shown in Figure 1C, normally only a small portion of endogenous NIR co-localizes with TAp63γ in the nucleoplasm of a cell. Therefore, a first study was designed to determine, with two sets of PCR primer pairs and in HaCaT cells that harbor endogenous TAp63γ as well as in H1299 cells with doxycycline-inducible TAp63γ, whether TAp63γ and NIR can be detected at the p21 gene promoter region that contains the p63 (and p53) recognition motif. The ChIPs were performed with monoclonal antibody 4A4 to precipitate TAp63γ and polyclonal antibody 2910 (30) to precipitate NIR. Clearly, 4A4 and 2910 precipitated the p21 promoter fragment in HaCaT cells, while irrelevant antibodies or an antibody directed against p53 failed to do so (HaCaT cells express mutant p53 defective for specific DNA binding; Figure 4A, PP1). Furthermore, 4A4 and 2910 also precipitated the p21 promoter in H1299 cells doxycycline-treated and transfected to express TAp63γ and exogenous NIR (Figure 4A, PP2). This suggests that TAp63γ and NIR are associated with the p21 promoter in cells that naturally or forcibly produce these proteins. The U6 gene promoter that is not responsive to the p53 protein family was not precipitated by any of the antibodies, excluding that NIR is co-precipitating promoters simply by associating with histones.
Figure 4.NIR occupies the p21 gene promoter, and its knockdown relieves repression in the presence of TAp63γ. (A) ChIP of the p21 promoter containing the TAp63γ -binding site and of the U6 promoter as a control. The p21 promoter fragment was amplified with two different primer pairs, PP1 and PP2 (‘Materials and methods’ section). PP1: Primer pair 1 was used to analyze cultures of HaCaT cells expressing endogenous TAp63γ and NIR (both present at low levels in the nucleoplasm). ChIP assays were performed either with irrelevant mouse IgG (mIgG), anti-p63 antibody 4A4, anti-p53 antibody DO-1 (HaCaT cells produce mutant p53 incapable of specific DNA binding), irrelevant rabbit IgG (rIgG), or the rabbit polyclonal anti-NIR antibody 2910. PP2: Primer pair 2 was employed to analyze cultures of H1299 cells with doxycycline-inducible TAp63γ and exogenous NIR. (B) Western blot showing that H1299 cells with doxycycline-inducible TAp63γ could be stimulated by defined low levels of doxy (50 ng/ml) to produce low quantities of TAp63γ (compare with β-actin levels). TAp63γ was detected with anti-HA antibody at 1 : 1000; β-actin by a monoclonal antibody from Sigma (1 : 3000). (C) Effect of NIR knockdown on p21 transcript level the presence of TAp63γ. Control H1299 cultures and H1299-pRTS-TAp63γ cultures with TAp63γ expressed at low levels [see (B)] were RNAi-fected with control siRNA or NIR siRNA; NIR knockdown and TAp63γ expression were monitored by RT-PCR. p21 transcript levels were determined by RT-qPCR. Error bars denote standard deviations of the means of at least three experiments.Since only small amounts of NIR were normally present in the nucleoplasm, we reasoned that this NIR might be able to effectively inhibit only low levels of TAp63γ. Knockdown of NIR might then activate p21 expression in the presence of the now de-repressed low levels of TAp63γ. HaCaT cells with endogenous TAp63γ and NIR, however, proved not to be useful for these studies, probably because they also produce high levels of ΔNp63α and mutant p53 known to act dominant-negatively toward TAp63γ. For the following experiments we therefore returned to the doxy-inducible system described above. This allowed us to induce only ∼1/10th of the steady-state levels of TAp63γ induced by 1.0 μg/ml doxy in H1299 cells by adding only 50 ng/ml doxy to the culture medium (Figure 4B). This level was not sufficient to detectably stimulate p21 expression. However, and notably, in this low-level expression system, simultaneous knockdown of NIR (by 60–70 %) through NIR siRNAs, but not transfection with irrelevant siRNA, led to stimulation of p21 expression by TAp63γ (Figure 4C). These findings suggest that the low levels of nucleoplasmic NIR normally present in cells can inhibit p21 transactivation by TAp63γ. Combined, our observations show that NIR binds to TAp63 and acts as a repressor of TAp63-mediated transactivation.NIR is controlled by the cell cycleInspection of immunofluorescence images revealed that in non-synchronous cell cultures, NIR is most prominently expressed in cells undergoing mitosis. In mitotic metaphase cells, the majority of the overproduced NIR was present in the vicinity of the condensed chromatin, within structures probably identical with the perichromosomal region first described by Gautier and colleagues (37). In contrast, NIR was predominantly present in the nucleoli in non-mitotic cells (Figure 5A; see also Figure 1C). To examine whether NIR transcript and protein expression are varying in the different phases of cell cycle, we synchronized human diploid fibroblasts (HDFs) by serum starvation/mimosine blocking and the subsequent induction of proliferation by feeding of the cultures with full medium. Figure 5B summarizes a typical result of an analysis of NIR expression in synchronously proliferating HDF at different time points during cell cycle progression. Flow cytometry revealed that cells that were arrested in G1 phase enter S-phase at ∼6 h after release from G1 arrest and approach G2/M phase at the 12–18 h time points. In accord with the immunofluorescence results obtained with non-synchronous cultures, NIR transcript levels in HDF, quantified by RT-qPCR, were lowest in G1 phase cells and gradually increased as the cultures approached G2/M. Concomitantly, but with some delay, NIR protein levels were lowest during S-phase and increased towards G2/M. In contrast, the levels of the gapdh transcript and β-actin protein controls remained stable throughout the time course. Thus, NIR transcript and protein levels are controlled by the cell cycle in HDF.
Figure 5.NIR expression is controlled by the cell cycle. (A) Fluorescence microscope image of H1299 cells after staining of DNA with DAPI and of NIR with polyclonal anti-NIR antibody 2910 (1 : 100) plus Alexa488-conjugated anti-rabbit secondary antibody (1 : 7000). NIR shows highest expression levels in mitotic cells (here: mitotic metaphase) and is primarily found in the vicinity of the chromatin; in non-mitotic cells, it is expressed at lower levels and mostly localized in the nucleoli. (B) Synchronized human diploid fibroblasts move through the cell cycle after release from G1 arrest for 24 h. Upper diagram shows flow cytometry profiles of cultures that had been DNA-stained with propidium iodide. The NIR transcript levels relative to the levels of gapdh were determined by reverse transcription quantitative PCR (RT-qPCR). Error bars denote SD from three RT-qPCRs. For western blot analysis of NIR protein expression, 15 μg of total protein were subjected to SDS-PAGE and were incubated with anti-NIR antibody 2910 (1 : 1000) and anti-β-actin antibody (1 : 10 000). (C) G1/S-phase cell cycle arrest induced by activation of transcription factor p53 and the subsequent transactivation of the p21 gene results in the reduction of NIR transcript levels. Human wild-type p53 proficient MCF7 breast carcinoma cells and human diploid fibroblasts (HDFs) were either mock-treated (–; mock) or treated with 10 μM of the small interfering molecule-activator of p53, nutlin-3, for 48 h (MCF7) to 72 h (HDF). Western immunoblotting was performed on 15 μg of total protein run on an SDS-PAGE and incubated with anti-p53 antibody DO-1 (1 : 2000) and anti-β-actin antibody (1 : 10 000). The expression of the p53 target gene p21 and of NIR relative to gapdh was measured by RT-qPCR. The error bars denote standard deviations from three experiments. (D) Expression of p21 alone recapitulates the effect of p53-expression. p53-deficient H1299 cells were either transfected with an irrelevant plasmid (mock) or with plasmid producing p21. Again, the expression of the p21 and NIR transcripts relative to gapdh was determined by RT-qPCR, and the error bars denote standard deviations from three experiments.Since both TAp63 (Figure 3A) and p53 (30) can transactivate p21, and p21 as inhibitor of cyclin-dependent kinases can arrest the cell cycle at the G1/S boundary, we next asked whether activation of p53 in wild-type p53 proficient human MCF7 breast carcinoma cells and HDF would affect p21 and NIR transcript levels. As expected, non-stress-induced stabilization and activation of p53 by the compound nutlin-3 that disrupts the association of p53 with its major inhibitor MDM2, led to the transactivation of p21 in both MCF7 and HDF (Figure 5C). Simultaneously, and in agreement with the results on the synchronized cultures, significantly less NIR transcript was observed when p21 transcript levels were high, while gapdh transcript levels remained constant. Finally, we sought to determine whether expression of p21 alone, in H1299 cells that lack p53 as well as TAp63, would suffice to reduce NIR expression. Figure 5D shows that in H1299 cultures transfected either with a plasmid producing EGFP as control or p21, the levels of p21 transcript rose significantly in the p21-transfected cultures as expected while simultaneously, the NIR transcript levels fell off. Combined, these data thus indicate that the expression of the inhibitor of TAp63 and p53, NIR, at the transcript and protein level is lowest during G1/S phase of cell cycle and suggest, since both TAp63 and p53 can induce a G1/S arrest by transactivating p21, that TAp63/p53 and NIR may be interconnected through a feedback loop in cycling cells (Figure 6).
Figure 6.Scheme outlining the potential interactions between NIR, p53/TAp63 and the cell cycle.Since it was known that p53 can repress genes by sequence-specific DNA binding (1), we also tested whether TAp63/p53 might regulate the NIR gene directly. Computer-aided and visual inspection of the genomic NIR sequences had revealed three putative TAp63/p53 recognition motifs; one positioned at –10 252 relative to the transcription start site; one at 10 090 in intron 12; and one at 10 847 in intron 13. The most distal motif seems to be almost perfect, with no spacers between the half-sites and a total of only four deviations from consensus at non-conserved positions. The remaining two motifs have either a spacer or several mutations at non-conserved sites. However, none of these sites proved functional—neither activation nor repression was observed—in the context of a luciferase reporter assay (data not shown). Furthermore, a computer-selected, bona fide NIR gene regulatory region (position –130 to 5000) also failed to confer TAp63/p53-responsiveness. Together, these findings suggest that NIR may not be directly regulated by TAp63 and p53.

NIR is a novel interaction partner of transcription factor TAp63Our initial objective was to identify new protein interactions involved in transcriptional regulation by the p53 protein family. Since this family recruits to promoters, and is regulated by HATs, low-stringency in silico analyses for novel proteins harboring putative INHAT (inhibitor of HAT) domains were performed. This resulted in the cloning of NIR (Novel INHAT Repressor), and subsequent work showed that NIR can indeed associate with, and regulate, p53 (30). Here, we asked whether NIR would also be to regulate p63, a transcription factor with the highest homology to p53 in its central DNA-binding domain (63%) and important for the development of stratified epithelia as well as for epithelial stem cell maintenance.First, in vitro synthesized, 35S-labeled full-length NIR or NIR (147–609) consisting of the central portion of the protein lacking both INHAT domains, were employed in GST pull-down assays with either GST alone, p53 or TAp63γ (the major transactivating form of p63 containing the N-terminal transactivation domain, central DNA-binding domain and C-terminal oligomerization domain). We observed that both NIR and NIR (147–609) bound to either transcription factor but not GST alone (Figure 1A). Next, we sought to map the binding of NIR to TAp63γ. For this purpose, full-length 35S-labeled NIR was used in GST pull-down assays with either GST alone or one of the TAp63γ fragments depicted in Figure 1B. The assay revealed that NIR binds to two domains on TAp63γ; the transactivation domain and the oligomerization domain. However, NIR failed to bind to the ΔNp63 isoform in this assay, suggesting that robust binding of NIR and TAp63γ requires the transactivation domain. NIR also failed to associate with the core DNA binding domain and, as expected, with GST alone. To confirm the interaction between NIR and TAp63 in vivo, co-immunoprecipitations were carried out. Of note in this context, endogenous NIR is predominantly localized in the nucleoli, and mostly in the outer granular region where ribosome subunit assembly takes place. A small portion of NIR can be visualized in dot-like structures in the nucleoplasm. In contrast, TAp63γ is exclusively localized in the nucleoplasm (Figure 1C). This subcellular distribution, combined with the fact that stable expression of TAp63γ provoked high levels of apoptotic death in our cell system, led us to expect that endogenous NIR (90 kDa; detected as a single band by polyclonal anti-NIR antibody 2910) would be difficult to co-immunoprecipitate with transfected TAp63γ. We therefore generated H1299 cultures in which an HA-tagged TAp63γ was conditionally expressed (see below). In this system, endogenous NIR could be co-precipitated with TAp63γ, whereas NIR failed to come down in the absence of TAp63γ (Figure 1D). Combined, these findings show that NIR and TAp63γ can physically associate.
Figure 1.NIR associates with TAp63γ and p53. (A) GST pull-down assay. (Upper panels) Equal load of GST-p53 (p53) and GST-TAp63γ(TAp63γ); M, marker. (Lower panels) In vitro translated 35S-labeled full-length NIR or NIR (147–609) central portion is retained by GST- TAp63γ (TAp63γ) and by GST-p53 (p53) but not GST alone (IN, Input). The scheme shows NIR with the N- and C-terminal INHAT domains and the nuclear localization signal (NLS). (B) 35S-labeled full-length NIR binds to the transactivation domain (TA) and the oligomerization domain (OD) of TAp63γ in GST pull-down assays. PR, proline-rich domain; DBD, DNA binding domain. Radiolabeled NIR fails to bind efficiently to ΔNp63. (C) Fluorescence microscopy showing subcellular localization of endogenous NIR (green), detected with polyclonal antibody 2910, and of transfected TAp63γ (red) in human H1299 lung adenocarcinoma cells deficient for p63 and p53. Note the predominant localization of NIR in the nucleoli (single arrow) and occasional cells with detectable staining for NIR in the nucleoplasm (double arrow). By contrast, TAp63γ was mostly nucleoplasmic. Cells were stained with antibody 2910 for NIR and antibody 4A4 for TAp63γ. (D) Co-immunoprecipitation of endogenous NIR and transfected HA-tagged TAp63γ with anti-HA antibody.

NIR is a novel interaction partner of transcription factor TAp63

Our initial objective was to identify new protein interactions involved in transcriptional regulation by the p53 protein family. Since this family recruits to promoters, and is regulated by HATs, low-stringency in silico analyses for novel proteins harboring putative INHAT (inhibitor of HAT) domains were performed. This resulted in the cloning of NIR (Novel INHAT Repressor), and subsequent work showed that NIR can indeed associate with, and regulate, p53 (30). Here, we asked whether NIR would also be to regulate p63, a transcription factor with the highest homology to p53 in its central DNA-binding domain (63%) and important for the development of stratified epithelia as well as for epithelial stem cell maintenance.

First, in vitro synthesized, 35S-labeled full-length NIR or NIR (147–609) consisting of the central portion of the protein lacking both INHAT domains, were employed in GST pull-down assays with either GST alone, p53 or TAp63γ (the major transactivating form of p63 containing the N-terminal transactivation domain, central DNA-binding domain and C-terminal oligomerization domain). We observed that both NIR and NIR (147–609) bound to either transcription factor but not GST alone (Figure 1A). Next, we sought to map the binding of NIR to TAp63γ. For this purpose, full-length 35S-labeled NIR was used in GST pull-down assays with either GST alone or one of the TAp63γ fragments depicted in Figure 1B. The assay revealed that NIR binds to two domains on TAp63γ; the transactivation domain and the oligomerization domain. However, NIR failed to bind to the ΔNp63 isoform in this assay, suggesting that robust binding of NIR and TAp63γ requires the transactivation domain. NIR also failed to associate with the core DNA binding domain and, as expected, with GST alone. To confirm the interaction between NIR and TAp63 in vivo, co-immunoprecipitations were carried out. Of note in this context, endogenous NIR is predominantly localized in the nucleoli, and mostly in the outer granular region where ribosome subunit assembly takes place. A small portion of NIR can be visualized in dot-like structures in the nucleoplasm. In contrast, TAp63γ is exclusively localized in the nucleoplasm (Figure 1C). This subcellular distribution, combined with the fact that stable expression of TAp63γ provoked high levels of apoptotic death in our cell system, led us to expect that endogenous NIR (90 kDa; detected as a single band by polyclonal anti-NIR antibody 2910) would be difficult to co-immunoprecipitate with transfected TAp63γ. We therefore generated H1299 cultures in which an HA-tagged TAp63γ was conditionally expressed (see below). In this system, endogenous NIR could be co-precipitated with TAp63γ, whereas NIR failed to come down in the absence of TAp63γ (Figure 1D). Combined, these findings show that NIR and TAp63γ can physically associate.
Figure 1.NIR associates with TAp63γ and p53. (A) GST pull-down assay. (Upper panels) Equal load of GST-p53 (p53) and GST-TAp63γ(TAp63γ); M, marker. (Lower panels) In vitro translated 35S-labeled full-length NIR or NIR (147–609) central portion is retained by GST- TAp63γ (TAp63γ) and by GST-p53 (p53) but not GST alone (IN, Input). The scheme shows NIR with the N- and C-terminal INHAT domains and the nuclear localization signal (NLS). (B) 35S-labeled full-length NIR binds to the transactivation domain (TA) and the oligomerization domain (OD) of TAp63γ in GST pull-down assays. PR, proline-rich domain; DBD, DNA binding domain. Radiolabeled NIR fails to bind efficiently to ΔNp63. (C) Fluorescence microscopy showing subcellular localization of endogenous NIR (green), detected with polyclonal antibody 2910, and of transfected TAp63γ (red) in human H1299 lung adenocarcinoma cells deficient for p63 and p53. Note the predominant localization of NIR in the nucleoli (single arrow) and occasional cells with detectable staining for NIR in the nucleoplasm (double arrow). By contrast, TAp63γ was mostly nucleoplasmic. Cells were stained with antibody 2910 for NIR and antibody 4A4 for TAp63γ. (D) Co-immunoprecipitation of endogenous NIR and transfected HA-tagged TAp63γ with anti-HA antibody.

NIR associates with TAp63γ and p53. (A) GST pull-down assay. (Upper panels) Equal load of GST-p53 (p53) and GST-TAp63γ(TAp63γ); M, marker. (Lower panels) In vitro translated 35S-labeled full-length NIR or NIR (147–609) central portion is retained by GST- TAp63γ (TAp63γ) and by GST-p53 (p53) but not GST alone (IN, Input). The scheme shows NIR with the N- and C-terminal INHAT domains and the nuclear localization signal (NLS). (B) 35S-labeled full-length NIR binds to the transactivation domain (TA) and the oligomerization domain (OD) of TAp63γ in GST pull-down assays. PR, proline-rich domain; DBD, DNA binding domain. Radiolabeled NIR fails to bind efficiently to ΔNp63. (C) Fluorescence microscopy showing subcellular localization of endogenous NIR (green), detected with polyclonal antibody 2910, and of transfected TAp63γ (red) in human H1299 lung adenocarcinoma cells deficient for p63 and p53. Note the predominant localization of NIR in the nucleoli (single arrow) and occasional cells with detectable staining for NIR in the nucleoplasm (double arrow). By contrast, TAp63γ was mostly nucleoplasmic. Cells were stained with antibody 2910 for NIR and antibody 4A4 for TAp63γ. (D) Co-immunoprecipitation of endogenous NIR and transfected HA-tagged TAp63γ with anti-HA antibody.

Nucleoplasmic NIR acts as a repressor of TAp63NIR has been documented to act as a repressor of coactivator-stimulated transactivation (30). We therefore wished to test whether ectopic NIR would influence transcriptional activation by ectopic TAp63γ of the best-studied and most responsive endogenous p63 target, the gene for the p21Cip1 (CDKN1A) inhibitor of cyclin-dependent kinases (p21 hereafter). Transient transfections of NIR had documented that ectopic NIR, in contrast to the endogenous protein which is mostly present in the nucleoli (see above), is primarily localized in the nucleoplasm, as is TAp63γ (Figure 2A). In accord with its function as a transactivator of the p21 gene, transfected TAp63γ stimulated the transcription of the endogenous p21 gene ∼5-fold in RT-qPCR assays (Figure 2B). Simultaneous transfection of NIR, while not measurably affecting TAp63γ expression, blocked the transactivation of p21 almost completely, as predicted if NIR acted as a repressor of TAp63γ-mediated transactivation.
Figure 2.NIR can inhibit TAp63γ-induced transactivation and change subcellular localization under actinomycin D-induced stress. (A) In contrast to endogenous NIR, which was mostly present in the nucleoli (Figure 2B), transfected (Flag-tagged) NIR was predominantly nucleoplasmic. (B) When TAp63γ- and p53-deficient H1299 cells were transfected to produce TAp63γ, expression of the endogenous TAp63γ-target gene p21 was stimulated. Co-expression of TAp63γ and ectopic NIR (in addition to the endogenous) resulted in the inhibition of p21 expression. p21 transcript levels were determined by RT-qPCR. Error bars denote standard deviations derived from at least three experiments. (C) Fluorescence microscopy documenting the presence of NIR in the nucleoli in mock-treated human diploid fibroblasts (HDF) and H1299 cells. Treatment of the cultures with 1.0 μg/ml ActD for 6 h triggered NIR translocation to the nucleoplasm in both cell types.In a further set of experiments, we asked whether conditions can be found under which endogenous NIR would leave the nucleoli to localize to the nucleoplasm, and whether this would affect TAp63γ-regulated gene expression. For this purpose, human H1299 lung adenocarcinoma, human HCT116 colon adenocarcinoma, human immortal HaCaT keratinocytes and human diploid fibroblasts were either left untreated or were treated for 6 h with one of several stressors (Table 1). NIR was mostly nucleolar in all (100%) interphase (i.e. non-mitotic) cells when mock-treated (NIR localization in mitotic cells: see below). Among the stressors employed, only the nucleolar stressor and inhibitor of rRNA synthesis actinomcyin D (ActD) induced translocation of NIR from the nucleoli to the nucleoplasm in all interphase cells (100%), at a dose (0.5–1.0 μg/ml) well below the one needed to cause global transcription inhibition (>5–10 μg/ml) (Figure 2C). Nucleoplasmic localization of a subset of nucleolar proteins upon Act D treatment has been reported before (35); these, however, are not known to interact with TAp63. Some increase in nucleoplasmic NIR was also observed following ultraviolet (UV) irradiation (100 J/m2) and exposure to mycophenolic acid (25 μg/ml) in all four cell types. As ActD induced NIR translocation most consistently, this compound was used in the further studies.
Table 1.Subcellular localization of NIR after a 6 -h exposure to various stressorsStressorCell lineNucleoliNucleoplasmNoneH1299++++–HCT116++++–HaCaT++++–HDF++++–Actinomycin DH1299–++++(1 μg/ml)HCT116–++++HaCaT–++++HDF–++++Adriamycin (0.34 μM)H1299++++–HCT116++++–HaCaT++++–HDF++++–Cycloheximide (10 μg/ml)H1299++++–HCT116++++–HaCaT++++–HDF++++–5-Fluorouracil (0.375 mM)H1299++++HCT116++++HaCaT++++–HDF++++–H2O2 (0.4 mM)H1299++++HCT116ndndHaCaT++++–HDF++++MG132 (10 μM)H1299++++–HCT116++++–HaCaT++++–HDF++++MMS (0.1 mg/ml)H1299++++–HCT116++++–HaCaT++++–HDF++++Mycophenolic acid (25 μg/ml)H1299++++HCT116++++HaCaT++++HDF++++–UV (100 J/m2)H1299++++HCT116–++++HaCaT++++HDF++++–: less than 10% of total NIR stain, in the majority of cells (>90%), by visual inspection.++: approximately half of total NIR stain, in the majority of cells (>90%), by visual inspection.++++: more than 90% of total NIR stain, in the majority of cells (>90%), by visual inspection.nd: not determined.To test whether endogenous NIR, upon translocation from the nucleoli to the nucleoplasm, can regulate nucleoplasmic TAp63γ, and since TAp63γ overproduced in transient transfections was a powerful inducer of cell death, we established doxycyline (doxy)-inducible cell lines on the basis of vector pRTS1 (36). The advantage of this system, which employs a trans-repressor active in the absence of doxy and a transactivator active in its presence, was that transgene expression was very tightly controlled, with absolutely no detectable expression in doxy-free medium versus expression that could be finely tuned in the presence of defined doses of the drug. In these cells, doxy induced the production of TAp63γ with a HA-tag at the N-terminus, which in turn stimulated the expression of endogenous p21, indicative of HA-TAp63γ being functional as a transcription factor (Figure 3A). The levels of endogenous NIR remained unchanged.
Figure 3.Endogenous NIR translocated to the nucleoplasm upon ActD treatment was able to inhibit TAp63γ-induced gene expression. (A) TAp63γ- and p53-deficient H1299 cells were stably bulk-transfected with either vector alone or pRTS-TAp63γ. The HA-tagged TAp63γ-transgene was tightly controlled by doxycycline (doxy): in the absence of drug, no transgene expression was detectable whereas in the presence of 1.0 μg/ml doxy for at least 8 h, maximal transgene expression became visible in western blots with anti-HA antibody (1 : 1000). Furthermore, a p21 signal detected by anti-p21 antibody (1 : 1000) was found only in the presence of TAp63γ. The levels of NIR always remained constant; NIR was detected with the polyclonal antibody 2910. (B) RT-qPCR quantitation of p21 transcript levels in cultures with or without TAp63γ, and in the presence or absence of stress. ActD (0.5 μg/ml) strongly inhibited TAp63γ-induced transactivation while ADR (0.34 μM) inhibited only weakly. (C) RT-qPCR quantitation of gapdh transcript levels. ActD had no significant effects. Error bars show standard deviations derived from at least three experiments.In the following series of experiments, we examined the effect on TAp63γ-induced p21 expression of a low dose of ActD (0.5 µg/ml) previously determined to suffice to translocate nucleolar NIR to the nucleoplasm (see Figure 3C and Table 1). Notably, transactivation of p21 by TAp63γ was completely inhibited in the presence of nucleoplasmic NIR (P < 0.0001; Figure 3B). In contrast to ActD, the drug ADR, used at a dose known to cause significant DNA damage (0.3 μM) but unable to trigger NIR translocation (Table 1), inhibited p21 transactivation significantly (P < 0.001) but much weaker. To exclude that the effect of ActD on TAp63γ-induced p21 expression was the result of a general toxicity or impairment of gene transcription, gapdh expression was monitored in parallel. No inhibitory effect of ActD was discernible (Figure 3C). Combined, these data thus suggest that endogenous nucleoplasmic NIR can act as an inhibitor of TAp63γ.

Nucleoplasmic NIR acts as a repressor of TAp63

NIR has been documented to act as a repressor of coactivator-stimulated transactivation (30). We therefore wished to test whether ectopic NIR would influence transcriptional activation by ectopic TAp63γ of the best-studied and most responsive endogenous p63 target, the gene for the p21Cip1 (CDKN1A) inhibitor of cyclin-dependent kinases (p21 hereafter). Transient transfections of NIR had documented that ectopic NIR, in contrast to the endogenous protein which is mostly present in the nucleoli (see above), is primarily localized in the nucleoplasm, as is TAp63γ (Figure 2A). In accord with its function as a transactivator of the p21 gene, transfected TAp63γ stimulated the transcription of the endogenous p21 gene ∼5-fold in RT-qPCR assays (Figure 2B). Simultaneous transfection of NIR, while not measurably affecting TAp63γ expression, blocked the transactivation of p21 almost completely, as predicted if NIR acted as a repressor of TAp63γ-mediated transactivation.
Figure 2.NIR can inhibit TAp63γ-induced transactivation and change subcellular localization under actinomycin D-induced stress. (A) In contrast to endogenous NIR, which was mostly present in the nucleoli (Figure 2B), transfected (Flag-tagged) NIR was predominantly nucleoplasmic. (B) When TAp63γ- and p53-deficient H1299 cells were transfected to produce TAp63γ, expression of the endogenous TAp63γ-target gene p21 was stimulated. Co-expression of TAp63γ and ectopic NIR (in addition to the endogenous) resulted in the inhibition of p21 expression. p21 transcript levels were determined by RT-qPCR. Error bars denote standard deviations derived from at least three experiments. (C) Fluorescence microscopy documenting the presence of NIR in the nucleoli in mock-treated human diploid fibroblasts (HDF) and H1299 cells. Treatment of the cultures with 1.0 μg/ml ActD for 6 h triggered NIR translocation to the nucleoplasm in both cell types.

NIR can inhibit TAp63γ-induced transactivation and change subcellular localization under actinomycin D-induced stress. (A) In contrast to endogenous NIR, which was mostly present in the nucleoli (Figure 2B), transfected (Flag-tagged) NIR was predominantly nucleoplasmic. (B) When TAp63γ- and p53-deficient H1299 cells were transfected to produce TAp63γ, expression of the endogenous TAp63γ-target gene p21 was stimulated. Co-expression of TAp63γ and ectopic NIR (in addition to the endogenous) resulted in the inhibition of p21 expression. p21 transcript levels were determined by RT-qPCR. Error bars denote standard deviations derived from at least three experiments. (C) Fluorescence microscopy documenting the presence of NIR in the nucleoli in mock-treated human diploid fibroblasts (HDF) and H1299 cells. Treatment of the cultures with 1.0 μg/ml ActD for 6 h triggered NIR translocation to the nucleoplasm in both cell types.

In a further set of experiments, we asked whether conditions can be found under which endogenous NIR would leave the nucleoli to localize to the nucleoplasm, and whether this would affect TAp63γ-regulated gene expression. For this purpose, human H1299 lung adenocarcinoma, human HCT116 colon adenocarcinoma, human immortal HaCaT keratinocytes and human diploid fibroblasts were either left untreated or were treated for 6 h with one of several stressors (Table 1). NIR was mostly nucleolar in all (100%) interphase (i.e. non-mitotic) cells when mock-treated (NIR localization in mitotic cells: see below). Among the stressors employed, only the nucleolar stressor and inhibitor of rRNA synthesis actinomcyin D (ActD) induced translocation of NIR from the nucleoli to the nucleoplasm in all interphase cells (100%), at a dose (0.5–1.0 μg/ml) well below the one needed to cause global transcription inhibition (>5–10 μg/ml) (Figure 2C). Nucleoplasmic localization of a subset of nucleolar proteins upon Act D treatment has been reported before (35); these, however, are not known to interact with TAp63. Some increase in nucleoplasmic NIR was also observed following ultraviolet (UV) irradiation (100 J/m2) and exposure to mycophenolic acid (25 μg/ml) in all four cell types. As ActD induced NIR translocation most consistently, this compound was used in the further studies.
Table 1.Subcellular localization of NIR after a 6 -h exposure to various stressorsStressorCell lineNucleoliNucleoplasmNoneH1299++++–HCT116++++–HaCaT++++–HDF++++–Actinomycin DH1299–++++(1 μg/ml)HCT116–++++HaCaT–++++HDF–++++Adriamycin (0.34 μM)H1299++++–HCT116++++–HaCaT++++–HDF++++–Cycloheximide (10 μg/ml)H1299++++–HCT116++++–HaCaT++++–HDF++++–5-Fluorouracil (0.375 mM)H1299++++HCT116++++HaCaT++++–HDF++++–H2O2 (0.4 mM)H1299++++HCT116ndndHaCaT++++–HDF++++MG132 (10 μM)H1299++++–HCT116++++–HaCaT++++–HDF++++MMS (0.1 mg/ml)H1299++++–HCT116++++–HaCaT++++–HDF++++Mycophenolic acid (25 μg/ml)H1299++++HCT116++++HaCaT++++HDF++++–UV (100 J/m2)H1299++++HCT116–++++HaCaT++++HDF++++–: less than 10% of total NIR stain, in the majority of cells (>90%), by visual inspection.++: approximately half of total NIR stain, in the majority of cells (>90%), by visual inspection.++++: more than 90% of total NIR stain, in the majority of cells (>90%), by visual inspection.nd: not determined.

Subcellular localization of NIR after a 6 -h exposure to various stressors

–: less than 10% of total NIR stain, in the majority of cells (>90%), by visual inspection.

++: approximately half of total NIR stain, in the majority of cells (>90%), by visual inspection.

++++: more than 90% of total NIR stain, in the majority of cells (>90%), by visual inspection.

nd: not determined.

To test whether endogenous NIR, upon translocation from the nucleoli to the nucleoplasm, can regulate nucleoplasmic TAp63γ, and since TAp63γ overproduced in transient transfections was a powerful inducer of cell death, we established doxycyline (doxy)-inducible cell lines on the basis of vector pRTS1 (36). The advantage of this system, which employs a trans-repressor active in the absence of doxy and a transactivator active in its presence, was that transgene expression was very tightly controlled, with absolutely no detectable expression in doxy-free medium versus expression that could be finely tuned in the presence of defined doses of the drug. In these cells, doxy induced the production of TAp63γ with a HA-tag at the N-terminus, which in turn stimulated the expression of endogenous p21, indicative of HA-TAp63γ being functional as a transcription factor (Figure 3A). The levels of endogenous NIR remained unchanged.
Figure 3.Endogenous NIR translocated to the nucleoplasm upon ActD treatment was able to inhibit TAp63γ-induced gene expression. (A) TAp63γ- and p53-deficient H1299 cells were stably bulk-transfected with either vector alone or pRTS-TAp63γ. The HA-tagged TAp63γ-transgene was tightly controlled by doxycycline (doxy): in the absence of drug, no transgene expression was detectable whereas in the presence of 1.0 μg/ml doxy for at least 8 h, maximal transgene expression became visible in western blots with anti-HA antibody (1 : 1000). Furthermore, a p21 signal detected by anti-p21 antibody (1 : 1000) was found only in the presence of TAp63γ. The levels of NIR always remained constant; NIR was detected with the polyclonal antibody 2910. (B) RT-qPCR quantitation of p21 transcript levels in cultures with or without TAp63γ, and in the presence or absence of stress. ActD (0.5 μg/ml) strongly inhibited TAp63γ-induced transactivation while ADR (0.34 μM) inhibited only weakly. (C) RT-qPCR quantitation of gapdh transcript levels. ActD had no significant effects. Error bars show standard deviations derived from at least three experiments.

Endogenous NIR translocated to the nucleoplasm upon ActD treatment was able to inhibit TAp63γ-induced gene expression. (A) TAp63γ- and p53-deficient H1299 cells were stably bulk-transfected with either vector alone or pRTS-TAp63γ. The HA-tagged TAp63γ-transgene was tightly controlled by doxycycline (doxy): in the absence of drug, no transgene expression was detectable whereas in the presence of 1.0 μg/ml doxy for at least 8 h, maximal transgene expression became visible in western blots with anti-HA antibody (1 : 1000). Furthermore, a p21 signal detected by anti-p21 antibody (1 : 1000) was found only in the presence of TAp63γ. The levels of NIR always remained constant; NIR was detected with the polyclonal antibody 2910. (B) RT-qPCR quantitation of p21 transcript levels in cultures with or without TAp63γ, and in the presence or absence of stress. ActD (0.5 μg/ml) strongly inhibited TAp63γ-induced transactivation while ADR (0.34 μM) inhibited only weakly. (C) RT-qPCR quantitation of gapdh transcript levels. ActD had no significant effects. Error bars show standard deviations derived from at least three experiments.

In the following series of experiments, we examined the effect on TAp63γ-induced p21 expression of a low dose of ActD (0.5 µg/ml) previously determined to suffice to translocate nucleolar NIR to the nucleoplasm (see Figure 3C and Table 1). Notably, transactivation of p21 by TAp63γ was completely inhibited in the presence of nucleoplasmic NIR (P < 0.0001; Figure 3B). In contrast to ActD, the drug ADR, used at a dose known to cause significant DNA damage (0.3 μM) but unable to trigger NIR translocation (Table 1), inhibited p21 transactivation significantly (P < 0.001) but much weaker. To exclude that the effect of ActD on TAp63γ-induced p21 expression was the result of a general toxicity or impairment of gene transcription, gapdh expression was monitored in parallel. No inhibitory effect of ActD was discernible (Figure 3C). Combined, these data thus suggest that endogenous nucleoplasmic NIR can act as an inhibitor of TAp63γ.

NIR knockdown can activate TAp63Cells tolerate neither permanent NIR knockdown nor overproduction (data not shown). To test whether temporary knockdown of NIR would affect transactivation of p21 by TAp63γ, we resorted to NIR knockdown by transient siRNA transfection. However, as shown in Figure 1C, normally only a small portion of endogenous NIR co-localizes with TAp63γ in the nucleoplasm of a cell. Therefore, a first study was designed to determine, with two sets of PCR primer pairs and in HaCaT cells that harbor endogenous TAp63γ as well as in H1299 cells with doxycycline-inducible TAp63γ, whether TAp63γ and NIR can be detected at the p21 gene promoter region that contains the p63 (and p53) recognition motif. The ChIPs were performed with monoclonal antibody 4A4 to precipitate TAp63γ and polyclonal antibody 2910 (30) to precipitate NIR. Clearly, 4A4 and 2910 precipitated the p21 promoter fragment in HaCaT cells, while irrelevant antibodies or an antibody directed against p53 failed to do so (HaCaT cells express mutant p53 defective for specific DNA binding; Figure 4A, PP1). Furthermore, 4A4 and 2910 also precipitated the p21 promoter in H1299 cells doxycycline-treated and transfected to express TAp63γ and exogenous NIR (Figure 4A, PP2). This suggests that TAp63γ and NIR are associated with the p21 promoter in cells that naturally or forcibly produce these proteins. The U6 gene promoter that is not responsive to the p53 protein family was not precipitated by any of the antibodies, excluding that NIR is co-precipitating promoters simply by associating with histones.
Figure 4.NIR occupies the p21 gene promoter, and its knockdown relieves repression in the presence of TAp63γ. (A) ChIP of the p21 promoter containing the TAp63γ -binding site and of the U6 promoter as a control. The p21 promoter fragment was amplified with two different primer pairs, PP1 and PP2 (‘Materials and methods’ section). PP1: Primer pair 1 was used to analyze cultures of HaCaT cells expressing endogenous TAp63γ and NIR (both present at low levels in the nucleoplasm). ChIP assays were performed either with irrelevant mouse IgG (mIgG), anti-p63 antibody 4A4, anti-p53 antibody DO-1 (HaCaT cells produce mutant p53 incapable of specific DNA binding), irrelevant rabbit IgG (rIgG), or the rabbit polyclonal anti-NIR antibody 2910. PP2: Primer pair 2 was employed to analyze cultures of H1299 cells with doxycycline-inducible TAp63γ and exogenous NIR. (B) Western blot showing that H1299 cells with doxycycline-inducible TAp63γ could be stimulated by defined low levels of doxy (50 ng/ml) to produce low quantities of TAp63γ (compare with β-actin levels). TAp63γ was detected with anti-HA antibody at 1 : 1000; β-actin by a monoclonal antibody from Sigma (1 : 3000). (C) Effect of NIR knockdown on p21 transcript level the presence of TAp63γ. Control H1299 cultures and H1299-pRTS-TAp63γ cultures with TAp63γ expressed at low levels [see (B)] were RNAi-fected with control siRNA or NIR siRNA; NIR knockdown and TAp63γ expression were monitored by RT-PCR. p21 transcript levels were determined by RT-qPCR. Error bars denote standard deviations of the means of at least three experiments.Since only small amounts of NIR were normally present in the nucleoplasm, we reasoned that this NIR might be able to effectively inhibit only low levels of TAp63γ. Knockdown of NIR might then activate p21 expression in the presence of the now de-repressed low levels of TAp63γ. HaCaT cells with endogenous TAp63γ and NIR, however, proved not to be useful for these studies, probably because they also produce high levels of ΔNp63α and mutant p53 known to act dominant-negatively toward TAp63γ. For the following experiments we therefore returned to the doxy-inducible system described above. This allowed us to induce only ∼1/10th of the steady-state levels of TAp63γ induced by 1.0 μg/ml doxy in H1299 cells by adding only 50 ng/ml doxy to the culture medium (Figure 4B). This level was not sufficient to detectably stimulate p21 expression. However, and notably, in this low-level expression system, simultaneous knockdown of NIR (by 60–70 %) through NIR siRNAs, but not transfection with irrelevant siRNA, led to stimulation of p21 expression by TAp63γ (Figure 4C). These findings suggest that the low levels of nucleoplasmic NIR normally present in cells can inhibit p21 transactivation by TAp63γ. Combined, our observations show that NIR binds to TAp63 and acts as a repressor of TAp63-mediated transactivation.

NIR knockdown can activate TAp63

Cells tolerate neither permanent NIR knockdown nor overproduction (data not shown). To test whether temporary knockdown of NIR would affect transactivation of p21 by TAp63γ, we resorted to NIR knockdown by transient siRNA transfection. However, as shown in Figure 1C, normally only a small portion of endogenous NIR co-localizes with TAp63γ in the nucleoplasm of a cell. Therefore, a first study was designed to determine, with two sets of PCR primer pairs and in HaCaT cells that harbor endogenous TAp63γ as well as in H1299 cells with doxycycline-inducible TAp63γ, whether TAp63γ and NIR can be detected at the p21 gene promoter region that contains the p63 (and p53) recognition motif. The ChIPs were performed with monoclonal antibody 4A4 to precipitate TAp63γ and polyclonal antibody 2910 (30) to precipitate NIR. Clearly, 4A4 and 2910 precipitated the p21 promoter fragment in HaCaT cells, while irrelevant antibodies or an antibody directed against p53 failed to do so (HaCaT cells express mutant p53 defective for specific DNA binding; Figure 4A, PP1). Furthermore, 4A4 and 2910 also precipitated the p21 promoter in H1299 cells doxycycline-treated and transfected to express TAp63γ and exogenous NIR (Figure 4A, PP2). This suggests that TAp63γ and NIR are associated with the p21 promoter in cells that naturally or forcibly produce these proteins. The U6 gene promoter that is not responsive to the p53 protein family was not precipitated by any of the antibodies, excluding that NIR is co-precipitating promoters simply by associating with histones.
Figure 4.NIR occupies the p21 gene promoter, and its knockdown relieves repression in the presence of TAp63γ. (A) ChIP of the p21 promoter containing the TAp63γ -binding site and of the U6 promoter as a control. The p21 promoter fragment was amplified with two different primer pairs, PP1 and PP2 (‘Materials and methods’ section). PP1: Primer pair 1 was used to analyze cultures of HaCaT cells expressing endogenous TAp63γ and NIR (both present at low levels in the nucleoplasm). ChIP assays were performed either with irrelevant mouse IgG (mIgG), anti-p63 antibody 4A4, anti-p53 antibody DO-1 (HaCaT cells produce mutant p53 incapable of specific DNA binding), irrelevant rabbit IgG (rIgG), or the rabbit polyclonal anti-NIR antibody 2910. PP2: Primer pair 2 was employed to analyze cultures of H1299 cells with doxycycline-inducible TAp63γ and exogenous NIR. (B) Western blot showing that H1299 cells with doxycycline-inducible TAp63γ could be stimulated by defined low levels of doxy (50 ng/ml) to produce low quantities of TAp63γ (compare with β-actin levels). TAp63γ was detected with anti-HA antibody at 1 : 1000; β-actin by a monoclonal antibody from Sigma (1 : 3000). (C) Effect of NIR knockdown on p21 transcript level the presence of TAp63γ. Control H1299 cultures and H1299-pRTS-TAp63γ cultures with TAp63γ expressed at low levels [see (B)] were RNAi-fected with control siRNA or NIR siRNA; NIR knockdown and TAp63γ expression were monitored by RT-PCR. p21 transcript levels were determined by RT-qPCR. Error bars denote standard deviations of the means of at least three experiments.

NIR occupies the p21 gene promoter, and its knockdown relieves repression in the presence of TAp63γ. (A) ChIP of the p21 promoter containing the TAp63γ -binding site and of the U6 promoter as a control. The p21 promoter fragment was amplified with two different primer pairs, PP1 and PP2 (‘Materials and methods’ section). PP1: Primer pair 1 was used to analyze cultures of HaCaT cells expressing endogenous TAp63γ and NIR (both present at low levels in the nucleoplasm). ChIP assays were performed either with irrelevant mouse IgG (mIgG), anti-p63 antibody 4A4, anti-p53 antibody DO-1 (HaCaT cells produce mutant p53 incapable of specific DNA binding), irrelevant rabbit IgG (rIgG), or the rabbit polyclonal anti-NIR antibody 2910. PP2: Primer pair 2 was employed to analyze cultures of H1299 cells with doxycycline-inducible TAp63γ and exogenous NIR. (B) Western blot showing that H1299 cells with doxycycline-inducible TAp63γ could be stimulated by defined low levels of doxy (50 ng/ml) to produce low quantities of TAp63γ (compare with β-actin levels). TAp63γ was detected with anti-HA antibody at 1 : 1000; β-actin by a monoclonal antibody from Sigma (1 : 3000). (C) Effect of NIR knockdown on p21 transcript level the presence of TAp63γ. Control H1299 cultures and H1299-pRTS-TAp63γ cultures with TAp63γ expressed at low levels [see (B)] were RNAi-fected with control siRNA or NIR siRNA; NIR knockdown and TAp63γ expression were monitored by RT-PCR. p21 transcript levels were determined by RT-qPCR. Error bars denote standard deviations of the means of at least three experiments.

Since only small amounts of NIR were normally present in the nucleoplasm, we reasoned that this NIR might be able to effectively inhibit only low levels of TAp63γ. Knockdown of NIR might then activate p21 expression in the presence of the now de-repressed low levels of TAp63γ. HaCaT cells with endogenous TAp63γ and NIR, however, proved not to be useful for these studies, probably because they also produce high levels of ΔNp63α and mutant p53 known to act dominant-negatively toward TAp63γ. For the following experiments we therefore returned to the doxy-inducible system described above. This allowed us to induce only ∼1/10th of the steady-state levels of TAp63γ induced by 1.0 μg/ml doxy in H1299 cells by adding only 50 ng/ml doxy to the culture medium (Figure 4B). This level was not sufficient to detectably stimulate p21 expression. However, and notably, in this low-level expression system, simultaneous knockdown of NIR (by 60–70 %) through NIR siRNAs, but not transfection with irrelevant siRNA, led to stimulation of p21 expression by TAp63γ (Figure 4C). These findings suggest that the low levels of nucleoplasmic NIR normally present in cells can inhibit p21 transactivation by TAp63γ. Combined, our observations show that NIR binds to TAp63 and acts as a repressor of TAp63-mediated transactivation.

NIR is controlled by the cell cycleInspection of immunofluorescence images revealed that in non-synchronous cell cultures, NIR is most prominently expressed in cells undergoing mitosis. In mitotic metaphase cells, the majority of the overproduced NIR was present in the vicinity of the condensed chromatin, within structures probably identical with the perichromosomal region first described by Gautier and colleagues (37). In contrast, NIR was predominantly present in the nucleoli in non-mitotic cells (Figure 5A; see also Figure 1C). To examine whether NIR transcript and protein expression are varying in the different phases of cell cycle, we synchronized human diploid fibroblasts (HDFs) by serum starvation/mimosine blocking and the subsequent induction of proliferation by feeding of the cultures with full medium. Figure 5B summarizes a typical result of an analysis of NIR expression in synchronously proliferating HDF at different time points during cell cycle progression. Flow cytometry revealed that cells that were arrested in G1 phase enter S-phase at ∼6 h after release from G1 arrest and approach G2/M phase at the 12–18 h time points. In accord with the immunofluorescence results obtained with non-synchronous cultures, NIR transcript levels in HDF, quantified by RT-qPCR, were lowest in G1 phase cells and gradually increased as the cultures approached G2/M. Concomitantly, but with some delay, NIR protein levels were lowest during S-phase and increased towards G2/M. In contrast, the levels of the gapdh transcript and β-actin protein controls remained stable throughout the time course. Thus, NIR transcript and protein levels are controlled by the cell cycle in HDF.
Figure 5.NIR expression is controlled by the cell cycle. (A) Fluorescence microscope image of H1299 cells after staining of DNA with DAPI and of NIR with polyclonal anti-NIR antibody 2910 (1 : 100) plus Alexa488-conjugated anti-rabbit secondary antibody (1 : 7000). NIR shows highest expression levels in mitotic cells (here: mitotic metaphase) and is primarily found in the vicinity of the chromatin; in non-mitotic cells, it is expressed at lower levels and mostly localized in the nucleoli. (B) Synchronized human diploid fibroblasts move through the cell cycle after release from G1 arrest for 24 h. Upper diagram shows flow cytometry profiles of cultures that had been DNA-stained with propidium iodide. The NIR transcript levels relative to the levels of gapdh were determined by reverse transcription quantitative PCR (RT-qPCR). Error bars denote SD from three RT-qPCRs. For western blot analysis of NIR protein expression, 15 μg of total protein were subjected to SDS-PAGE and were incubated with anti-NIR antibody 2910 (1 : 1000) and anti-β-actin antibody (1 : 10 000). (C) G1/S-phase cell cycle arrest induced by activation of transcription factor p53 and the subsequent transactivation of the p21 gene results in the reduction of NIR transcript levels. Human wild-type p53 proficient MCF7 breast carcinoma cells and human diploid fibroblasts (HDFs) were either mock-treated (–; mock) or treated with 10 μM of the small interfering molecule-activator of p53, nutlin-3, for 48 h (MCF7) to 72 h (HDF). Western immunoblotting was performed on 15 μg of total protein run on an SDS-PAGE and incubated with anti-p53 antibody DO-1 (1 : 2000) and anti-β-actin antibody (1 : 10 000). The expression of the p53 target gene p21 and of NIR relative to gapdh was measured by RT-qPCR. The error bars denote standard deviations from three experiments. (D) Expression of p21 alone recapitulates the effect of p53-expression. p53-deficient H1299 cells were either transfected with an irrelevant plasmid (mock) or with plasmid producing p21. Again, the expression of the p21 and NIR transcripts relative to gapdh was determined by RT-qPCR, and the error bars denote standard deviations from three experiments.Since both TAp63 (Figure 3A) and p53 (30) can transactivate p21, and p21 as inhibitor of cyclin-dependent kinases can arrest the cell cycle at the G1/S boundary, we next asked whether activation of p53 in wild-type p53 proficient human MCF7 breast carcinoma cells and HDF would affect p21 and NIR transcript levels. As expected, non-stress-induced stabilization and activation of p53 by the compound nutlin-3 that disrupts the association of p53 with its major inhibitor MDM2, led to the transactivation of p21 in both MCF7 and HDF (Figure 5C). Simultaneously, and in agreement with the results on the synchronized cultures, significantly less NIR transcript was observed when p21 transcript levels were high, while gapdh transcript levels remained constant. Finally, we sought to determine whether expression of p21 alone, in H1299 cells that lack p53 as well as TAp63, would suffice to reduce NIR expression. Figure 5D shows that in H1299 cultures transfected either with a plasmid producing EGFP as control or p21, the levels of p21 transcript rose significantly in the p21-transfected cultures as expected while simultaneously, the NIR transcript levels fell off. Combined, these data thus indicate that the expression of the inhibitor of TAp63 and p53, NIR, at the transcript and protein level is lowest during G1/S phase of cell cycle and suggest, since both TAp63 and p53 can induce a G1/S arrest by transactivating p21, that TAp63/p53 and NIR may be interconnected through a feedback loop in cycling cells (Figure 6).
Figure 6.Scheme outlining the potential interactions between NIR, p53/TAp63 and the cell cycle.Since it was known that p53 can repress genes by sequence-specific DNA binding (1), we also tested whether TAp63/p53 might regulate the NIR gene directly. Computer-aided and visual inspection of the genomic NIR sequences had revealed three putative TAp63/p53 recognition motifs; one positioned at –10 252 relative to the transcription start site; one at 10 090 in intron 12; and one at 10 847 in intron 13. The most distal motif seems to be almost perfect, with no spacers between the half-sites and a total of only four deviations from consensus at non-conserved positions. The remaining two motifs have either a spacer or several mutations at non-conserved sites. However, none of these sites proved functional—neither activation nor repression was observed—in the context of a luciferase reporter assay (data not shown). Furthermore, a computer-selected, bona fide NIR gene regulatory region (position –130 to 5000) also failed to confer TAp63/p53-responsiveness. Together, these findings suggest that NIR may not be directly regulated by TAp63 and p53.

NIR is controlled by the cell cycle

Inspection of immunofluorescence images revealed that in non-synchronous cell cultures, NIR is most prominently expressed in cells undergoing mitosis. In mitotic metaphase cells, the majority of the overproduced NIR was present in the vicinity of the condensed chromatin, within structures probably identical with the perichromosomal region first described by Gautier and colleagues (37). In contrast, NIR was predominantly present in the nucleoli in non-mitotic cells (Figure 5A; see also Figure 1C). To examine whether NIR transcript and protein expression are varying in the different phases of cell cycle, we synchronized human diploid fibroblasts (HDFs) by serum starvation/mimosine blocking and the subsequent induction of proliferation by feeding of the cultures with full medium. Figure 5B summarizes a typical result of an analysis of NIR expression in synchronously proliferating HDF at different time points during cell cycle progression. Flow cytometry revealed that cells that were arrested in G1 phase enter S-phase at ∼6 h after release from G1 arrest and approach G2/M phase at the 12–18 h time points. In accord with the immunofluorescence results obtained with non-synchronous cultures, NIR transcript levels in HDF, quantified by RT-qPCR, were lowest in G1 phase cells and gradually increased as the cultures approached G2/M. Concomitantly, but with some delay, NIR protein levels were lowest during S-phase and increased towards G2/M. In contrast, the levels of the gapdh transcript and β-actin protein controls remained stable throughout the time course. Thus, NIR transcript and protein levels are controlled by the cell cycle in HDF.
Figure 5.NIR expression is controlled by the cell cycle. (A) Fluorescence microscope image of H1299 cells after staining of DNA with DAPI and of NIR with polyclonal anti-NIR antibody 2910 (1 : 100) plus Alexa488-conjugated anti-rabbit secondary antibody (1 : 7000). NIR shows highest expression levels in mitotic cells (here: mitotic metaphase) and is primarily found in the vicinity of the chromatin; in non-mitotic cells, it is expressed at lower levels and mostly localized in the nucleoli. (B) Synchronized human diploid fibroblasts move through the cell cycle after release from G1 arrest for 24 h. Upper diagram shows flow cytometry profiles of cultures that had been DNA-stained with propidium iodide. The NIR transcript levels relative to the levels of gapdh were determined by reverse transcription quantitative PCR (RT-qPCR). Error bars denote SD from three RT-qPCRs. For western blot analysis of NIR protein expression, 15 μg of total protein were subjected to SDS-PAGE and were incubated with anti-NIR antibody 2910 (1 : 1000) and anti-β-actin antibody (1 : 10 000). (C) G1/S-phase cell cycle arrest induced by activation of transcription factor p53 and the subsequent transactivation of the p21 gene results in the reduction of NIR transcript levels. Human wild-type p53 proficient MCF7 breast carcinoma cells and human diploid fibroblasts (HDFs) were either mock-treated (–; mock) or treated with 10 μM of the small interfering molecule-activator of p53, nutlin-3, for 48 h (MCF7) to 72 h (HDF). Western immunoblotting was performed on 15 μg of total protein run on an SDS-PAGE and incubated with anti-p53 antibody DO-1 (1 : 2000) and anti-β-actin antibody (1 : 10 000). The expression of the p53 target gene p21 and of NIR relative to gapdh was measured by RT-qPCR. The error bars denote standard deviations from three experiments. (D) Expression of p21 alone recapitulates the effect of p53-expression. p53-deficient H1299 cells were either transfected with an irrelevant plasmid (mock) or with plasmid producing p21. Again, the expression of the p21 and NIR transcripts relative to gapdh was determined by RT-qPCR, and the error bars denote standard deviations from three experiments.

NIR expression is controlled by the cell cycle. (A) Fluorescence microscope image of H1299 cells after staining of DNA with DAPI and of NIR with polyclonal anti-NIR antibody 2910 (1 : 100) plus Alexa488-conjugated anti-rabbit secondary antibody (1 : 7000). NIR shows highest expression levels in mitotic cells (here: mitotic metaphase) and is primarily found in the vicinity of the chromatin; in non-mitotic cells, it is expressed at lower levels and mostly localized in the nucleoli. (B) Synchronized human diploid fibroblasts move through the cell cycle after release from G1 arrest for 24 h. Upper diagram shows flow cytometry profiles of cultures that had been DNA-stained with propidium iodide. The NIR transcript levels relative to the levels of gapdh were determined by reverse transcription quantitative PCR (RT-qPCR). Error bars denote SD from three RT-qPCRs. For western blot analysis of NIR protein expression, 15 μg of total protein were subjected to SDS-PAGE and were incubated with anti-NIR antibody 2910 (1 : 1000) and anti-β-actin antibody (1 : 10 000). (C) G1/S-phase cell cycle arrest induced by activation of transcription factor p53 and the subsequent transactivation of the p21 gene results in the reduction of NIR transcript levels. Human wild-type p53 proficient MCF7 breast carcinoma cells and human diploid fibroblasts (HDFs) were either mock-treated (–; mock) or treated with 10 μM of the small interfering molecule-activator of p53, nutlin-3, for 48 h (MCF7) to 72 h (HDF). Western immunoblotting was performed on 15 μg of total protein run on an SDS-PAGE and incubated with anti-p53 antibody DO-1 (1 : 2000) and anti-β-actin antibody (1 : 10 000). The expression of the p53 target gene p21 and of NIR relative to gapdh was measured by RT-qPCR. The error bars denote standard deviations from three experiments. (D) Expression of p21 alone recapitulates the effect of p53-expression. p53-deficient H1299 cells were either transfected with an irrelevant plasmid (mock) or with plasmid producing p21. Again, the expression of the p21 and NIR transcripts relative to gapdh was determined by RT-qPCR, and the error bars denote standard deviations from three experiments.

Since both TAp63 (Figure 3A) and p53 (30) can transactivate p21, and p21 as inhibitor of cyclin-dependent kinases can arrest the cell cycle at the G1/S boundary, we next asked whether activation of p53 in wild-type p53 proficient human MCF7 breast carcinoma cells and HDF would affect p21 and NIR transcript levels. As expected, non-stress-induced stabilization and activation of p53 by the compound nutlin-3 that disrupts the association of p53 with its major inhibitor MDM2, led to the transactivation of p21 in both MCF7 and HDF (Figure 5C). Simultaneously, and in agreement with the results on the synchronized cultures, significantly less NIR transcript was observed when p21 transcript levels were high, while gapdh transcript levels remained constant. Finally, we sought to determine whether expression of p21 alone, in H1299 cells that lack p53 as well as TAp63, would suffice to reduce NIR expression. Figure 5D shows that in H1299 cultures transfected either with a plasmid producing EGFP as control or p21, the levels of p21 transcript rose significantly in the p21-transfected cultures as expected while simultaneously, the NIR transcript levels fell off. Combined, these data thus indicate that the expression of the inhibitor of TAp63 and p53, NIR, at the transcript and protein level is lowest during G1/S phase of cell cycle and suggest, since both TAp63 and p53 can induce a G1/S arrest by transactivating p21, that TAp63/p53 and NIR may be interconnected through a feedback loop in cycling cells (Figure 6).
Figure 6.Scheme outlining the potential interactions between NIR, p53/TAp63 and the cell cycle.

Scheme outlining the potential interactions between NIR, p53/TAp63 and the cell cycle.

Since it was known that p53 can repress genes by sequence-specific DNA binding (1), we also tested whether TAp63/p53 might regulate the NIR gene directly. Computer-aided and visual inspection of the genomic NIR sequences had revealed three putative TAp63/p53 recognition motifs; one positioned at –10 252 relative to the transcription start site; one at 10 090 in intron 12; and one at 10 847 in intron 13. The most distal motif seems to be almost perfect, with no spacers between the half-sites and a total of only four deviations from consensus at non-conserved positions. The remaining two motifs have either a spacer or several mutations at non-conserved sites. However, none of these sites proved functional—neither activation nor repression was observed—in the context of a luciferase reporter assay (data not shown). Furthermore, a computer-selected, bona fide NIR gene regulatory region (position –130 to 5000) also failed to confer TAp63/p53-responsiveness. Together, these findings suggest that NIR may not be directly regulated by TAp63 and p53.

DISCUSSIONThe transcription of genes packaged within a complex consisting of histone proteins is regulated by specific chemical modifications to the N-termini of these proteins. Among the numerous modifying enzymes identified to date, p300 of the metazoan-specific p300/CBP family of HATs is perhaps the most extensively studied (19–21). It functions as a global transcriptional co-activator through the acetylation of all four core histones plus the recruitment of other transcription factors (21). p300 itself is recruited to gene promoters by physical association with DNA-binding proteins such as the p53/p63/p73 family of tumor suppressors/developmental regulators (22–27), and can use these—in addition to neighboring histones—as substrates for acetylation (22,27). At least two further concepts of chromatin regulation through acetyl groups have been realized: deacetylation by HDACs and ‘substrate masking’ by the binding to substrates of INHATs (31,38). INHATs, such as the Set/TAF1β oncoprotein (31) and pp32 (32), seem to be part of larger protein complexes that include HDACs (34). We have recently identified NIR, a novel INHAT that functions independently of HDACs, that can be recruited to promoters by p53 and that can repress neighboring genes (30). Our current studies have indicated that the p53-relative TAp63 may bind to NIR at least as well as p53, and since p63 is an important regulator in embryonic development, we here examined the effects of NIR on TAp63.p63 can occur in at least six isoforms (11). All share the DNA-binding domain but may carry different C-termini (α, β and γ) as the result of alternative splicing, and in addition, can vary at their N-termini as the result of alternative promoter usage in having either a major transactivation domain (TAp63) or a deletion of the TA and only a minor transactivation domain in their C-terminus (ΔNp63). We observed that robust binding of NIR requires the TA domain, suggesting that the TAp63 isoforms of p63 are subject to regulation by this INHAT. NIR was associated with the p63- responsive p21 promoter in vivo and acted as a negative regulator of transactivation by TAp63. Knockdown of NIR by RNAi technology relieved p21 gene repression in the presence of TAp63. Thus, NIR is a negative regulator of TAp63. However, in proliferating tumor cells that were not in the mitosis phase of cell cycle, as well as in proliferating and arrested human diploid fibroblasts, the majority of cellular NIR was sequestered at or within the nucleoli; only a small portion co-localized with TAp63 in the nucleoplasm. Accordingly, nucleoplasmic NIR was able to inhibit only small quantities of nucleoplasmic TAp63γ (Figure 5), whereas the larger levels of nucleoplasmic NIR induced upon actinomycin D-provoked nucleolus-nucleoplasma translocation of NIR was very well able to repress the p21 gene transcription induced by higher TAp63γ levels (Figure 4). Thus, in cells not experiencing nucleolar stress, NIR might act as a buffer toward the low levels of active TAp63 (and p53), while in cells with nucleolar stress, NIR which is usually expressed to high total levels in most cell types (30), might act as a powerful repressor of TAp63/p53. However, why would repression of TAp63, and in particular of the stress-responsive p53, in the face of nucleolar stress be favorable? NIR might help limit the cellular response to stress in cases of reversible nucleolar damage. However, NIR could also function to specifically inhibit target genes such as p21 that are known to suppress apoptosis (39,40), and thereby support an apoptotic response if necessary.Numerous studies have identified p63 as acting mainly during embryonic development—in the commitment of ectodermal cells to the epidermal lineage; the maintenance of epidermal stem cells; and the development of stratified epithelia (4,41,42). Although not yet fully understood, proper function of p63 in these processes appears to depend upon a subtle balance between the ΔNp63- and TAp63-isoforms (16), with ΔNp63 being the dominant isoform at both the transcript and protein level (2). Although ΔNp63 can act as a transcriptional activator of certain genes, it is thought to mainly function as a repressor, through the occupation of promoters regulated by p53 and p63 (43,44). By contrast, the precise function of TAp63 is less clear. While ΔNp63 is strongly expressed during stratification and differentiation of epithelia, TAp63 is weaker expressed and seems to play a minor role in late differentiation (16). However, other work has identified TAp63 as being expressed earlier than ΔNp63, at a time point immediately upstream of the commitment to stratification, and has suggested that TAp63 may constitute the ‘switch’ for stratification, with ΔNp63 counteracting this function and thereby allowing cell differentiation (17). In this context, NIR might thus act as a buffer of TAp63 activity and might help prevent the differentiation block observed with overactive TAp63. Alternatively or in addition, NIR might buffer TAp63 activity during the late stages of epidermal differentiation. One should also consider a role of NIR in tumor development or progression since not only p53 but also p63 has been implicated in cancers, and notably, it is not only the inhibitory ΔNp63 isoform that has been observed to be overproduced in many tumor types. A subpopulation of lymphoid cells and most malignant lymphomas over-express exclusively TAp63 (45,46). Ongoing work will examine whether NIR is of importance here.The nucleolus is a dynamic structure that is regulated by the cell cycle, and with it many nucleolar proteins. In particular components of the rRNA processing machinery, such as the multifunctional chaperone nucleophosmin (NPM; B23), leave the nucleolus organizer region during mitotic prophase and metaphase. Some of these factors are redistributed to become attached to the surface of the condensed chromatin, the perichromosomal region (PR), in a way reminiscent of the distribution of NIR in Figure 6A (37). The PR has been suggested to function as a chromosome-protecting insulator, and NIR in this layer might shield the chromatin from the action of HATs. There are several notable similarities between NIR and NPM. For example, both proteins are lowly expressed during G0/G1 phase of cell cycle and are more strongly produced in response to mitogenic stimuli (Figure 6 and ref. 47). We also observed that NIR like NPM is generally produced to higher levels in highly proliferating, transformed cells compared to non-transformed cells (data not shown). Finally, both NIR and NPM can bind to, and functionally inhibit, p53 (30,48,49). Future studies will address whether NIR and NPM can interact directly, and whether NIR might share other functions of NPM such as the regulation of tumor suppressor p14ARF.

The transcription of genes packaged within a complex consisting of histone proteins is regulated by specific chemical modifications to the N-termini of these proteins. Among the numerous modifying enzymes identified to date, p300 of the metazoan-specific p300/CBP family of HATs is perhaps the most extensively studied (19–21). It functions as a global transcriptional co-activator through the acetylation of all four core histones plus the recruitment of other transcription factors (21). p300 itself is recruited to gene promoters by physical association with DNA-binding proteins such as the p53/p63/p73 family of tumor suppressors/developmental regulators (22–27), and can use these—in addition to neighboring histones—as substrates for acetylation (22,27). At least two further concepts of chromatin regulation through acetyl groups have been realized: deacetylation by HDACs and ‘substrate masking’ by the binding to substrates of INHATs (31,38). INHATs, such as the Set/TAF1β oncoprotein (31) and pp32 (32), seem to be part of larger protein complexes that include HDACs (34). We have recently identified NIR, a novel INHAT that functions independently of HDACs, that can be recruited to promoters by p53 and that can repress neighboring genes (30). Our current studies have indicated that the p53-relative TAp63 may bind to NIR at least as well as p53, and since p63 is an important regulator in embryonic development, we here examined the effects of NIR on TAp63.

p63 can occur in at least six isoforms (11). All share the DNA-binding domain but may carry different C-termini (α, β and γ) as the result of alternative splicing, and in addition, can vary at their N-termini as the result of alternative promoter usage in having either a major transactivation domain (TAp63) or a deletion of the TA and only a minor transactivation domain in their C-terminus (ΔNp63). We observed that robust binding of NIR requires the TA domain, suggesting that the TAp63 isoforms of p63 are subject to regulation by this INHAT. NIR was associated with the p63- responsive p21 promoter in vivo and acted as a negative regulator of transactivation by TAp63. Knockdown of NIR by RNAi technology relieved p21 gene repression in the presence of TAp63. Thus, NIR is a negative regulator of TAp63. However, in proliferating tumor cells that were not in the mitosis phase of cell cycle, as well as in proliferating and arrested human diploid fibroblasts, the majority of cellular NIR was sequestered at or within the nucleoli; only a small portion co-localized with TAp63 in the nucleoplasm. Accordingly, nucleoplasmic NIR was able to inhibit only small quantities of nucleoplasmic TAp63γ (Figure 5), whereas the larger levels of nucleoplasmic NIR induced upon actinomycin D-provoked nucleolus-nucleoplasma translocation of NIR was very well able to repress the p21 gene transcription induced by higher TAp63γ levels (Figure 4). Thus, in cells not experiencing nucleolar stress, NIR might act as a buffer toward the low levels of active TAp63 (and p53), while in cells with nucleolar stress, NIR which is usually expressed to high total levels in most cell types (30), might act as a powerful repressor of TAp63/p53. However, why would repression of TAp63, and in particular of the stress-responsive p53, in the face of nucleolar stress be favorable? NIR might help limit the cellular response to stress in cases of reversible nucleolar damage. However, NIR could also function to specifically inhibit target genes such as p21 that are known to suppress apoptosis (39,40), and thereby support an apoptotic response if necessary.

Numerous studies have identified p63 as acting mainly during embryonic development—in the commitment of ectodermal cells to the epidermal lineage; the maintenance of epidermal stem cells; and the development of stratified epithelia (4,41,42). Although not yet fully understood, proper function of p63 in these processes appears to depend upon a subtle balance between the ΔNp63- and TAp63-isoforms (16), with ΔNp63 being the dominant isoform at both the transcript and protein level (2). Although ΔNp63 can act as a transcriptional activator of certain genes, it is thought to mainly function as a repressor, through the occupation of promoters regulated by p53 and p63 (43,44). By contrast, the precise function of TAp63 is less clear. While ΔNp63 is strongly expressed during stratification and differentiation of epithelia, TAp63 is weaker expressed and seems to play a minor role in late differentiation (16). However, other work has identified TAp63 as being expressed earlier than ΔNp63, at a time point immediately upstream of the commitment to stratification, and has suggested that TAp63 may constitute the ‘switch’ for stratification, with ΔNp63 counteracting this function and thereby allowing cell differentiation (17). In this context, NIR might thus act as a buffer of TAp63 activity and might help prevent the differentiation block observed with overactive TAp63. Alternatively or in addition, NIR might buffer TAp63 activity during the late stages of epidermal differentiation. One should also consider a role of NIR in tumor development or progression since not only p53 but also p63 has been implicated in cancers, and notably, it is not only the inhibitory ΔNp63 isoform that has been observed to be overproduced in many tumor types. A subpopulation of lymphoid cells and most malignant lymphomas over-express exclusively TAp63 (45,46). Ongoing work will examine whether NIR is of importance here.

The nucleolus is a dynamic structure that is regulated by the cell cycle, and with it many nucleolar proteins. In particular components of the rRNA processing machinery, such as the multifunctional chaperone nucleophosmin (NPM; B23), leave the nucleolus organizer region during mitotic prophase and metaphase. Some of these factors are redistributed to become attached to the surface of the condensed chromatin, the perichromosomal region (PR), in a way reminiscent of the distribution of NIR in Figure 6A (37). The PR has been suggested to function as a chromosome-protecting insulator, and NIR in this layer might shield the chromatin from the action of HATs. There are several notable similarities between NIR and NPM. For example, both proteins are lowly expressed during G0/G1 phase of cell cycle and are more strongly produced in response to mitogenic stimuli (Figure 6 and ref. 47). We also observed that NIR like NPM is generally produced to higher levels in highly proliferating, transformed cells compared to non-transformed cells (data not shown). Finally, both NIR and NPM can bind to, and functionally inhibit, p53 (30,48,49). Future studies will address whether NIR and NPM can interact directly, and whether NIR might share other functions of NPM such as the regulation of tumor suppressor p14ARF.

Conflict of interest statement. None declared.
