---
pmid: '15908602'
title: Brassinosteroid homeostasis in Arabidopsis is ensured by feedback expressions
  of multiple genes involved in its metabolism.
authors:
- Tanaka K
- Asami T
- Yoshida S
- Nakamura Y
- Matsuo T
- Okamoto S
journal: Plant Physiol
year: '2005'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC1150425
doi: 10.1104/pp.104.058040
---

# Brassinosteroid homeostasis in Arabidopsis is ensured by feedback expressions of multiple genes involved in its metabolism.
**Authors:** Tanaka K, Asami T, Yoshida S, Nakamura Y, Matsuo T, Okamoto S
**Journal:** Plant Physiol (2005)
**DOI:** [10.1104/pp.104.058040](https://doi.org/10.1104/pp.104.058040)
**PMC:** [PMC1150425](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1150425/)

## Abstract

1. Plant Physiol. 2005 Jun;138(2):1117-25. doi: 10.1104/pp.104.058040. Epub 2005 
May 20.

Brassinosteroid homeostasis in Arabidopsis is ensured by feedback expressions of 
multiple genes involved in its metabolism.

Tanaka K(1), Asami T, Yoshida S, Nakamura Y, Matsuo T, Okamoto S.

Author information:
(1)Department of Agricultural Sciences and Natural Resources , Kagoshima 
University, Kagoshima 890-0065, Japan.

Homeostasis of brassinosteroids (BRs) is essential for normal growth and 
development in higher plants. We examined responsiveness of 11 BR metabolic gene 
expressions to the decrease or increase of endogenous BR contents in Arabidopsis 
(Arabidopsis thaliana) to expand our knowledge of molecular mechanisms 
underlying BR homeostasis. Five BR-specific biosynthesis genes (DET2, DWF4, CPD, 
BR6ox1, and ROT3) and two sterol biosynthesis genes (FK and DWF5) were 
up-regulated in BR-depleted wild-type plants grown under brassinazole, a BR 
biosynthesis inhibitor. On the other hand, in BR-excessive wild-type plants that 
were fed with brassinolide, four BR-specific synthesis genes (DWF4, CPD, BR6ox1, 
and ROT3) and a sterol synthesis gene (DWF7) were down-regulated and a BR 
inactivation gene (BAS1) was up-regulated. However, their response to 
fluctuation of BR levels was highly reduced (DWF4) or nullified (the other eight 
genes) in a bri1 mutant. Taken together, our results imply that BR homeostasis 
is maintained through feedback expressions of multiple genes, each of which is 
involved not only in BR-specific biosynthesis and inactivation, but also in 
sterol biosynthesis. Our results also indicate that their feedback expressions 
are under the control of a BRI1-mediated signaling pathway. Moreover, a weak 
response in the mutant suggests that DWF4 alone is likely to be regulated in 
other way(s) in addition to BRI1 mediation.

DOI: 10.1104/pp.104.058040
PMCID: PMC1150425
PMID: 15908602 [Indexed for MEDLINE]

## Full Text

Abstract

Homeostasis of brassinosteroids (BRs) is essential for normal growth and development in higher plants. We examined responsiveness of 11 BR metabolic gene expressions to the decrease or increase of endogenous BR contents in Arabidopsis ( Arabidopsis thaliana ) to expand our knowledge of molecular mechanisms underlying BR homeostasis. Five BR-specific biosynthesis genes ( DET2 , DWF4 , CPD , BR6ox1 , and ROT3 ) and two sterol biosynthesis genes ( FK and DWF5 ) were up-regulated in BR-depleted wild-type plants grown under brassinazole, a BR biosynthesis inhibitor. On the other hand, in BR-excessive wild-type plants that were fed with brassinolide, four BR-specific synthesis genes ( DWF4 , CPD , BR6ox1 , and ROT3 ) and a sterol synthesis gene ( DWF7 ) were down-regulated and a BR inactivation gene ( BAS1 ) was up-regulated. However, their response to fluctuation of BR levels was highly reduced ( DWF4 ) or nullified (the other eight genes) in a bri1 mutant. Taken together, our results imply that BR homeostasis is maintained through feedback expressions of multiple genes, each of which is involved not only in BR-specific biosynthesis and inactivation, but also in sterol biosynthesis. Our results also indicate that their feedback expressions are under the control of a BRI1-mediated signaling pathway. Moreover, a weak response in the mutant suggests that DWF4 alone is likely to be regulated in other way(s) in addition to BRI1 mediation.

DISCUSSION

Phytohormone homeostasis is believed to be crucial for normal growth and development in higher plants. For that reason, it is readily inferred that BR biosynthesis and inactivation can be modulated in a feedback manner to reduce elevated BR concentrations when bioactive BRs are given excessively to the plants. We observed BL-induced down-regulation of four BR-specific biosynthesis genes, DWF4 , CPD , BR6ox1 , and ROT3 , as well as BL-induced up-regulation of a BR inactivation gene, BAS1 , in our experimental system ( Fig. 4 ; Table I ), which are consistent with data in previous reports ( Mathur et al., 1998 ; Bancos et al., 2002 ; Goda et al., 2002 ). Taken together, these results indicate that the increase of endogenous BR contents results in a feedback regulation at mRNA levels of these BR metabolic genes to maintain BR homeostasis. Furthermore, we found that a sterol biosynthesis gene, DWF7 , was markedly down-regulated by BL addition ( Fig. 4 ). Mussig et al. (2002) also observed BL-induced down-regulation of DWF7 in DNA microarray analysis. Nevertheless, that study did not refer to this phenomenon. Our time-course experiment showed that the kinetic behavior of DWF7 differed greatly from those of the four BR-specific biosynthesis genes described above ( Fig. 4 ); DWF7 mRNA level was decreased rapidly upon BL addition, but it never reached as low as the levels of BR-specific biosynthesis genes. Kinetic differences between DWF7 and other BR-specific biosynthesis genes probably reflect that the former affects not only BR synthesis, but also sterol synthesis, whereas the latter influences only BR production.

Little information is available regarding responses of BR metabolic genes when endogenous BR concentrations are reduced. A few studies have shown that some BR-specific biosynthesis genes, DET2 , DWF4 , and CPD , are up-regulated in BR-deficient mutants such as dwf1 , dwf4 , and cpd ( Noguchi et al., 2000 ; Choe et al., 2001 ). Our study found that BR6ox1 and ROT3 , in addition to the three genes described above, were up-regulated when endogenous BR concentrations were temporarily reduced by Brz ( Fig. 2 ; Table I ). All five BR-specific biosynthesis genes we tested responded to a decrease of endogenous BR contents; four of them, aside from DET2 , also respond to their increase. These results suggest that the BR-specific biosynthesis genes play important roles in maintaining BR homeostasis. Although Choe et al. (2001) reported lower expression of BAS1 in a dwf4 mutant than that in wild type, we did not detect Brz-induced down-regulation of BAS1 in our system ( Fig. 2 ). Currently, causes of the discrepancy in BAS1 response to the depletion of endogenous BR contents between the two laboratories remain unknown. Those different results might reflect different experimental conditions with respect to the use of either a BR-deficient mutant or a Brz-fed wild-type Arabidopsis. We observed that Brz caused up-regulation of sterol biosynthesis genes FK and DWF5 ( Fig. 2 ). FK expression in Brz-fed plants is consistent with results of He et al. (2003) . Their study demonstrated that FK :: GUS expression is higher in BR-deficient det2 mutants than in wild type. Taken together, our results indicate that BR homeostasis is finely modulated by the feedback expressions of multiple (at least nine) BR metabolic genes, each of which is involved not only in BR-specific biosynthesis and inactivation, but also in sterol biosynthesis.

Why are sterol biosynthesis genes FK , DWF5 , and DWF7 up-regulated or down-regulated by respective administration of Brz or BL ( Figs. 2 and 4 ; Table I )? Two plant sterols, campesterol and its epimer, are precursor compounds for BR biosynthesis ( Schaller, 2003 ). For that reason, endogenous BR concentrations, whether excessive or depleted, may affect expression of these sterol biosynthesis genes to decrease or increase precursor contents and thereby control the BR biosynthesis rate. However, their altered expressions should accessorily influence contents of sterols other than two precursor sterols for BR production because the three genes are also proven to function in synthesis of two major sterols: sitosterol and stigmasterol ( Catterou et al., 2001 ; Schaller, 2003 ). Otherwise, both the rise and fall of endogenous BR concentrations may affect sterol contents and compositions positively. In animal systems, specialized areas called lipid rafts of cellular membranes, which are rich in both cholesterol and sphingolipids, are known to function as membrane scaffolds in signal transduction and protein sorting ( Simons and Toomre, 2000 ). Furthermore, Mongrand et al. (2004) reported recently that tobacco plasma membranes possess such lipid rafts that are rich in sitosterol, stigmasterol, and 24-methylcholesterol as well as in cholesterol and sphingolipids. Therefore, changes of sterol contents and compositions that depend on endogenous BR concentrations may alter the properties and number of lipid rafts and then indirectly modulate BR perception on membrane-bound BR receptors. It also cannot be ruled out that endogenous BR-dependent expressions of the sterol biosynthesis genes may merely reflect crosstalk between BRs and signaling plant sterols, as in the case of interactions between BRs and other phytohormones ( Tanaka et al., 2003 ). Increasingly numerous reports indicate that plant sterols function as signal molecules, but not as membrane constituents for plant growth and development, especially in embryogenesis ( Diener et al., 2000 ; Jang et al., 2000 ; Schrick et al., 2000 , 2002 ; Souter et al., 2002 ). Therefore, BRs may crosstalk with signaling plant sterols by means of the control of sterol biosynthesis. In any case, further studies are necessary to elucidate these conjectures relating to BR content-dependent expression of sterol biosynthesis genes.

Feedback regulation of endogenous BR concentrations is probably achieved through successful perception and transduction of BR signals because bioactive BRs, BL, and castasterone accumulate in BR-insensitive bri1/cbb2 and bin2/dwf12 mutants ( Noguchi et al., 1999 ; Choe et al., 2002 ). Furthermore, a BR-specific biosynthesis gene, CPD , fails to be down-regulated in the same mutants fed with BL ( Li et al., 2001 ; Bancos et al., 2002 ). This inference is supported strongly by our observation showing that expression of nine BR metabolic genes had a reduced or no response to fluctuation of endogenous BR contents in a bri1-401 mutant, whereas all of them clearly responded in wild type ( Table I ). It is interesting to speculate why DWF4 expression was regulated by fluctuation of endogenous BR contents in this mutant even though its responsiveness was much weaker than that in wild-type plants. One possible explanation for this phenomenon is that feedback expression of DWF4 is mediated by other receptor(s) in addition to a BRI1 receptor. Hu et al. (2000 , 2004 ) reported that BR-responsive genes CycD3 and RAV1 are regulated by epi-BL in bri1 mutants. Furthermore, BRI1-like receptor genes have been shown to form a small gene family in the respective genomes of Arabidopsis, rice, and tomato ( Yin et al., 2002 ; Nomura et al., 2003 ; Cano-Delgado et al., 2004 ; Zhou et al., 2004 ). These reports together suggest that feedback expression of DWF4 is also under the control of either BRI1 homologs or unknown receptors without homology to BRI1. Alternatively, DWF4 expression may be regulated directly in a feedback manner by end-products (BL and castasterone) or immediate products of DWF4-catalyzing reactions.

What molecular mechanisms are hidden under BR signaling mediated feedback expressions of its metabolic genes? Our results shed some light on those mechanisms. First, we found that seven BR metabolic genes that were up-regulated by the decrease of endogenous BR contents were classifiable into two groups with regard to their responsiveness: early response genes ( FK , DWF5 , DET2 , DWF4 , and BR6ox1 ) and late response genes ( CPD and ROT3 ; Figs. 2 and 3 ). Results suggest that two independent feedback pathways modulate these two classes of genes separately when BR contents return to appropriate levels from its depletion state. Alternatively, a single pathway regulates both classes sequentially. Secondly, we clarified that nine BR metabolic genes responded to either excessive ( DWF7 and BAS1 ), depleted ( FK , DWF5 , and DET2 ), or both levels ( CPD , DWF4 , BR6ox1 , and ROT3 ) of endogenous BRs. This result gives rise to several questions concerning the feedback expressions of BR metabolic genes as described below. How are the rise and fall of BR concentrations sensed in plant cells? Are they monitored separately by different steps in BR signaling cascade or together detected by a single step, for example, perception of BR signals? How are the differences of BR levels distinguished and transmitted as distinct messages to each BR metabolic gene if the latter is the case? What signaling components are responsible for this process? Wang et al. (2002) reported that a single component downstream of BRI1, BZR1, is involved in feedback expression of CPD . More recently, BZR1 has been shown to act as a novel transcriptional repressor that binds directly to the promoters of BR-specific biosynthesis genes including CPD and DWF4 in BR homeostasis ( He et al., 2005 ). Therefore, BZR1-interacting proteins and target genes of BZR1 may be good candidates for components that function in BR-signaling-mediated feedback expressions of BR metabolic genes. Based on our results, we are currently searching for missing links, including the above proteins (genes), that connect BR signaling and BR metabolism.
