---
pmid: '31582854'
title: FACT mediates cohesin function on chromatin.
authors:
- Garcia-Luis J
- Lazar-Stefanita L
- Gutierrez-Escribano P
- Thierry A
- Cournac A
- García A
- González S
- Sánchez M
- Jarmuz A
- Montoya A
- Dore M
- Kramer H
- Karimi MM
- Antequera F
- Koszul R
- Aragon L
journal: Nat Struct Mol Biol
year: '2019'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6779571
doi: 10.1038/s41594-019-0307-x
---

# FACT mediates cohesin function on chromatin.
**Authors:** Garcia-Luis J, Lazar-Stefanita L, Gutierrez-Escribano P, Thierry A, Cournac A, García A, González S, Sánchez M, Jarmuz A, Montoya A, Dore M, Kramer H, Karimi MM, Antequera F, Koszul R, Aragon L
**Journal:** Nat Struct Mol Biol (2019)
**DOI:** [10.1038/s41594-019-0307-x](https://doi.org/10.1038/s41594-019-0307-x)
**PMC:** [PMC6779571](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6779571/)

## Abstract

1. Nat Struct Mol Biol. 2019 Oct;26(10):970-979. doi: 10.1038/s41594-019-0307-x. 
Epub 2019 Oct 3.

FACT mediates cohesin function on chromatin.

Garcia-Luis J(1), Lazar-Stefanita L(2), Gutierrez-Escribano P(1), Thierry A(2), 
Cournac A(2), García A(3), González S(3), Sánchez M(3), Jarmuz A(1), Montoya 
A(4), Dore M(5), Kramer H(4), Karimi MM(5), Antequera F(3), Koszul R(6), Aragon 
L(7).

Author information:
(1)Cell Cycle Group, MRC London Institute of Medical Sciences (LMS), London, UK.
(2)Institut Pasteur, Department of Genomes and Genetics, Unité Régulation 
Spatiale des Génomes, Paris, France.
(3)Instituto de Biología Funcional y Genómica (IBFG), CSIC/Universidad de 
Salamanca, Salamanca, Spain.
(4)Biological Mass Spectrometry and Proteomics Facility, MRC London Institute of 
Medical Sciences (LMS), London, UK.
(5)Bioinformatics Facility, MRC London Institute of Medical Sciences (LMS), 
London, UK.
(6)Institut Pasteur, Department of Genomes and Genetics, Unité Régulation 
Spatiale des Génomes, Paris, France. romain.koszul@pasteur.fr.
(7)Cell Cycle Group, MRC London Institute of Medical Sciences (LMS), London, UK. 
luis.aragon@csc.mrc.ac.uk.

Cohesin is a regulator of genome architecture with roles in sister chromatid 
cohesion and chromosome compaction. The recruitment and mobility of cohesin 
complexes on DNA is restricted by nucleosomes. Here, we show that the role of 
cohesin in chromosome organization requires the histone chaperone FACT 
('facilitates chromatin transcription') in Saccharomyces cerevisiae. We find 
that FACT interacts directly with cohesin, and is dynamically required for its 
localization on chromatin. Depletion of FACT in metaphase cells prevents cohesin 
accumulation at pericentric regions and causes reduced binding on chromosome 
arms. Using the Hi-C technique, we show that cohesin-dependent TAD (topological 
associated domain)-like structures in G1 and metaphase chromosomes are reduced 
in the absence of FACT. Sister chromatid cohesion is intact in FACT-depleted 
cells, although chromosome segregation failure is observed. Our data show that 
FACT contributes to the formation of cohesin-dependent TADs, thus uncovering a 
new role for this complex in nuclear organization during interphase and mitotic 
chromosome folding.

DOI: 10.1038/s41594-019-0307-x
PMCID: PMC6779571
PMID: 31582854 [Indexed for MEDLINE]

Conflict of interest statement: Competing interests The authors declare no 
competing interests.

## Full Text

Cohesin complexes not only hold sister chromatids but spatially organise chromosomes into loop domains 1–3. Cohesin has been suggested to perform these functions by co-entrapment of DNAs within a tripartite ring 4. Cohesin contains a pair of SMC proteins with ABC-like ATPase domains that are formed as the N and C terminal domains come into contact when the proteins fold onto themselves 5. In addition, the interaction of the SMC pair through a folded hinge shapes them into a V structure that is subsequently bound by the N- and C-terminal domains of a kleisin subunit, generating a ring 4 thought to embrace DNAs. Until recently, the role of cohesin was thought to be in the maintenance of sister chromatid cohesion 5, however recent studies have demonstrated that the complex also modulates the organisation of interphase nuclei and mitotic chromosomes. 2,3,6. Analysis of vertebrate cohesin has shown that it is required to fold individual chromatids into distinct loops by maintaining contacts between different loci in cis
6. The current favoured model for how these cohesin-dependent loops might be formed on chromosomes involves capture and bending of a DNA segment by cohesin rings to form a loop that progressively becomes larger 7. This activity is usually referred to as loop extrusion 7. Direct evidence for cohesin loop extrusion activity has not yet been obtained. In contrast, single molecule analyses of a related SMC complex, condensin, has shown an ability to form DNA loops in a manner consistent with extrusion 8. The activity of cohesin in loop extrusion was proposed to explain two main observations; that cohesin is required to maintain cis looping 6 and, that cohesin is blocked at TAD borders by CTCF on sites exhibiting convergent CTCF DNA motifs 7,9. It is currently not clear whether cohesin can indeed extrude loops or it requires an associated DNA translocase to do so.

Single molecule analysis of purified cohesin complexes has shown a capacity to diffuse along naked λ-DNA molecules in an ATP-independent manner 10. Interestingly, the presence of nucleosomes on these λ-DNA molecules was shown to restrict cohesin mobility 10. In addition to blocking cohesin diffusion 10 nucleosomes have an inhibitory effect on cohesin binding. The ATP-dependent chromatin remodeler RSC (Remodels the Structure of Chromatin) and the Irc5 complex in yeast, as well as the SNF2 complex in humans are required for cohesin association to chromosomes 11–13. Collectively, these findings suggest that cohesin functions on chromatinised eukaryotic genomes that involve binding to specific sites or translocation, require accessory factors to overcome nucleosomes. Here we set out to identify chromatin-remodelers that are important for cohesin function. We found that in addition to RSC, the histone chaperone FACT (Facilitates chromatin transcription) is a key regulator of cohesin’s role in genome folding, but dispensable for its function in sister chromatid cohesion.

ResultsCohesin binding leads to changes in nucleosome positions at pericentric regionsWe first sought to analyse if nucleosomal occupancy is dynamic at sites of cohesin binding. In Saccharomyces cerevisiae CEN sequences are cohesin loading sites where complexes initially access chromosomes before spreading towards pericentric regions 14–17. Genome-wide MNase-seq maps were generated 18 in G1 and G2/M arrests to analyse nucleosome occupancy around centromeric regions (Fig. 1a). Although the yeast genome is characterised by having well-defined nucleosomal positions throughout 19, we detected differences in the positioning and occupancy of individual nucleosomes around pericentromeric regions between the G1 and G2/M MNase-seq maps (Fig. 1a). The general trend involved the loss of defined positioning in G2/M arrests (increased nucleosome fuzziness- Fig 1b). The differences were confirmed using DANPOS 2 20 (Fig. 1b) and showed a general correlation with regions enriched in pericentric cohesin (Fig. 1c). Moreover, auxin-mediated degradation of cohesin’s kleisin Mcd1 in G2/M arrests shifted the nucleosomal profiles closer to what we had observed for G1 arrests (Fig. 1a), i.e. reverting nucleosome fuzziness (Fig. 1b). Together the results are consistent with the idea that nucleosomes are dynamic around CEN regions where cohesins are loaded and spread to pericentromeric sites.The Cohesin complex physically interacts with chromatin remodellersNext, we sought to identify potential chromatin remodellers involved in cohesin function. To this aim, cells expressing Mcd1 tagged with 6xHA were arrested in metaphase and Mcd1 immunoprecipitates were analysed by mass spectrometry (Table 1). Importantly, cell extracts were treated with benzonase to eliminate DNA prior to Mcd1 immunoprecipitation, to rule out indirect interactions mediated by DNA (Fig. 1d). We found that some chromatin remodellers were highly enriched in our pulldowns (Table 1). The most abundant was the Spt16 subunit of the FACT complex (facilitates chromatin transcription) (Table 1). Consistent with previous reports 21, several subunits of RSC were also amongst the hits (Table 1).We were particularly intrigued by the identification of FACT because chromosome assembly in vitro requires FACT alongside the SMC complex condensin 22. FACT is a heterodimeric complex containing Spt16 and Pob3 that acts as a histone chaperone regulating the assembly and disassembly of nucleosomes during transcription elongation, DNA replication and DNA repair 23. The ability of FACT to mobilise nucleosomes makes it an ideal candidate to facilitate cohesin functions on chromatin or the extrusion of chromatin loops through cohesin rings. We confirmed the interaction between cohesin and FACT in metaphase arrests using cells that expressed SMC1-6HA and SPT16-9MYC to validate that cohesin physically interacts with this histone chaperone (Fig. 1e).FACT is necessary for the localization of cohesin on yeast chromosomesTo further explore a possible functional relationship between FACT and cohesin, we sought to test whether inactivation of FACT in G2/M arrests caused any alterations in cohesin binding. An auxin (IAA)-inducible degron of Spt16 (SPT16-AID) 24 was used to deplete FACT from cells in metaphase (Suppl. Fig. 1). We performed calibrated ChIP-seq as described in the literature 25 for Smc1, a subunit of the cohesin complex, to analyse cohesin localisation before and after Spt16 depletion along yeast chromosomes (Fig. 2a). Centromeric regions are ideal sites to study cohesin translocation because cohesin complexes are initially loaded at core CEN sequences and then move away towards pericentromeric regions 17. We observed a small effect on cohesin localisation to core CEN sequence upon Spt16 degradation, however its accumulation at pericentric regions was severely reduced (Fig. 2a-b). This suggest that FACT might not be required for cohesin initial loading at CEN sites 25, but it is necessary for its accumulation on peri-centric regions (Fig. 2b). Mcd1 is expressed at low levels in G1-arrested cells, thus cohesin association to pericentric regions is very modest at this stage of the cell cycle 17. However, cohesin loading can still occur in G1
25 when cohesin expression is artificially induced. Indeed, expression of MCD1 from the GAL1-10 promoter caused a significant enrichment of Smc1 both at core CEN sequences and the surrounding pericentric regions (Fig. 2c). In contrast, MCD1 expression in G1-arrests depleted in Spt16 showed no enrichment of cohesin at pericentric regions (Fig. 2c), despite the fact that Smc1 still was present at core CEN sequences (Fig. 2c). Therefore, these results demonstrate that in the absence of the histone chaperone FACT, cohesins can be loaded at core CEN sites but fail to accumulate at pericentric regions. To confirm that the loss of cohesin signal in FACT-depleted cells was not due to a defect in the ability of cohesin to load, we first tested whether cohesin loading occurs in G2/M arrested cells (the stage where we depleted FACT in our experiments). To do this, we used an auxin (IAA)-inducible degron of Sth1 (STH1-AID), a core subunit of RSC, because RSC is required for cohesin loading26. Cohesin binding around centromeres and chromosome arms was similar to wildtype cells in the Sth1-depletion (Fig. 2d & Suppl. Fig. 2). In addition, FACT depletion in the absence of the cohesin release factor Wapl also did not affect cohesin binding on chromosomes (Fig. 2d), demonstrating that cohesin can be loaded normally in the absence of FACT. Therefore, the loss of cohesin signal in FACT-depleted cells cannot be caused by an impairment in cohesin loading but rather the stability of cohesin on chromatin.FACT contribution to cohesin localization is independent of transcriptionIn yeast, cohesin complexes can be divided into two classes with respect to their chromosomal association mechanisms, those loaded at CEN sequences, which move and accumulate in pericentric regions 27,28, and those that load at chromosomal arm sites and translocate to regions of convergent transcription 29. Importantly, the translocation of cohesins from CEN sites to pericentromeric regions requires Scc2 17 and is independent of transcription 17, while the movement of arm loaded cohesins to sites between convergent genes involves relocation dependent on transcription 29–32. FACT plays important roles during transcription initiation and elongation33,34. Therefore, we decided to investigate to what extent changes in transcription upon FACT-depletion are responsible for the disruption in the cohesin ChIP profiles. First, we focused on centromeric regions where transcription should not affect movement of CEN loaded cohesins to pericentric regions 17. To confirm that accumulation of cohesin at pericentromeric regions does not require active transcription, we treated wildtype cells with the transcription repressor thiolutin and used calibrated ChiP-seq to Smc1 to compare cohesin enrichment in the presence and absence of active transcription (Fig. 3a). Thiolutin treatment had little effect on Smc1 binding at core CEN sequences or pericentric sites (Fig. 3a). Therefore, as reported previously17, accumulation of cohesin at these sites is independent of transcription. In contrast, Spt16 degradation in thiolutin-treated cells caused a reduction in cohesin presence on pericentric regions (Fig. 3a). Therefore, at pericentric regions FACT facilitates cohesin accumulation independently of active transcription.The analysis of cohesin translocation on chromosome arms is more challenging that at centromeres because cohesin loading sites at chromosome arms are not well defined. ChIP profiles reflect the final position of the complexes. Our results demonstrate that FACT inactivation causes a decrease in cohesin binding at chromosomal arm sites (Fig. 2a). We therefore decided to test whether the effect of FACT on cohesin deposition at chromosome arms was independent on transcription, as observed for pericentric regions (Fig. 3a). Smc1 binding across chromosome arm sites was reduced in wildtype cells treated with thiolutin (Fig. 3b) confirming that active transcription is important for cohesin enrichment at these sites 29–32. Thiolutin treatment in FACT-depleted cells caused a further reduction in cohesin binding across chromosome sites (Fig. 3b).To further dissect the contribution of FACT to cohesin stability, we analysed how Smc1 enrichment at convergent transcription sites correlates with the changes in transcription at these sites when Spt16 is depleted (Fig. 3c). Analysis of cohesin binding sites confirmed their enrichment in positions in the genome where transcription units are oriented in a convergent manner 29–31. Interestingly, cohesin enrichment was observed as a double peak at both 3’ ends of the genes (Fig. 3c). We classified convergent sites into categories depending on whether the transcription of the flanking genes had change by 2-fold up or down or had remain the same (Fig. 3c) upon Spt16 depletion. We then compared cohesin enrichment changes in each transcription-pair category (Fig. 3c). Smc1 was reduced similarly in all categories (Fig. 3c). Therefore, we conclude that FACT has a direct function in cohesin stability on chromosome arm sites and that changes in cohesin binding cannot be explained solely by the altered transcription profiles upon depletion of Spt16.FACT is not required for sister chromatid cohesionHaving demonstrated that FACT has a role in maintaining cohesin on chromosomes (Fig. 3a-c), we sought to investigate the functional consequences for chromosome architecture. Cohesin holds sister chromatid together 35,36 and organises intra-chromatid loops that provide an integral level of structure to chromosomes 2,3. First, we tested whether cohesion between sister chromatids is affected in G2/M arrested cells when Spt16 is depleted. We scored cohesion using tetO/tetR-based tags at sub-telomeric regions of chromosome IV (Fig. 3d). Cells depleted for Spt16 showed no defects in cohesion (Fig. 3d).FACT activity affects the role of cohesin in genome foldingNext, we tested whether Spt16 is necessary to maintain mitotic chromosome architecture. Recent studies on mitotic yeast chromosomes using Hi-C have shown that cohesin compacts chromosome arms through cis interactions, independently of sister chromatid cohesion 2,3. We therefore considered the possibility that the reduction in cohesins observed upon Spt16-depletion (Fig. 2a) might impact on cis contacts along yeast metaphase chromosomes. To explore this, we built Hi-C libraries from G2/M arrested cells depleted for Spt16 and compared it to control arrests (Fig. 4a and Suppl. Fig. 3). The resulting contact maps show that depletion of Spt16 led to a reduction in short-distance intra-chromosomal contacts while long-range intra-chromosomal contacts were increased (> 100kb), as suggested by i) a larger width of the main diagonal in the contact maps of cells depleted for Spt16 (Suppl. Fig. 3), and as confirmed by computing ii) the contact probability p as a function of genomic distance s of all chromosome arms (Fig. 4b) and iii) the log-ratio between the normalized contact maps of depleted and non-depleted cells (Fig. 4a). The ratio map also shows that Spt16 depletion cause an increase of inter-chromosomal contacts (Fig. 4a). These results suggest that chromatin fibres loose internal organization in the absence of Spt16.To further characterize mitotic chromosomes, 2 kb contact maps were generated. At this resolution, triangular contact patterns reminiscent of the topological associating domain (TADs) described in metazoans appear along mitotic chromosomes in G2/M arrested cells but not G1 arrests (Fig. 4c). The boundaries of these domains involve regions enriched in cohesin (red diamonds, Fig. 4c), and stronger contacts between pairs of boundaries are often visible, consistent with the establishment of cohesin loops (black triangles, Fig. 4c) (previous Hi-C studies on cohesin failed to directly characterize these loops, as the experiments were not analysed at this resolution 2,3). We tested whether the triangular patterns of the 2 kb contact maps were dependent on cohesin. To this aim, we built Hi-C libraries from cells depleted for Mcd1 while arrested in G2/M (Fig. 2c). The resulting contact maps showed a significant reduction in TAD-like structures (Fig. 4c), as expected from previous reports demonstrating a key role for cohesin in loop formation 2,3,6. The depletion of Spt16 results in a strong reduction of cohesin deposition (Fig. 4c), as well as a fading of the cohesin-dependent TADs-like domains signal (Fig. 4c).Our Hi-C analysis is consistent with a role for FACT in affecting cohesin-dependent loops, however we cannot differentiate whether FACT is important to establish these loops or to maintain them. To test a potential role for FACT in de novo loop formation by cohesin, we first investigated whether overexpressing MCD1 in G1, which causes cohesin loading (Fig. 2c), leads to de novo formation of loop, which are normally absent in G1 cells 2,3 (Fig. 4c). Hi-C libraries from G1 arrested cells overexpressing Mcd1 from the GAL1-10 promoter were thus generated. TADs-like domain signals were observed in 2 kb contact maps of G1 overexpressing Mcd1 (Fig. 4d) demonstrating de novo loop formation by cohesin under this experimental condition (Fig. 4d). Next, we depleted Spt16 during the overexpression of Mcd1 in G1 cells, cohesin dependent loops were now strongly reduced, and hardly visible, in the absence of FACT (Suppl. Fig. 4). The log-ratio between the normalized contact maps of G1 depleted and non-depleted cells (Fig. 4e) and the contact probabilities as a function of genomic distance s of all chromosome arms (Fig. 4f) confirms that de novo loop formation by cohesins in G1 requires FACT.FACT-dependent chromosome organisation in metaphase is required for chromosome segregationOur results demonstrate that while cohesin’s role in sister chromatid cohesion is maintained in the absence of FACT (Fig. 3d), its function in organising chromatin loops in metaphase chromosomes is impaired (Fig. 4a-c). We therefore hypothesized that depletion of FACT in metaphase arrested cells might lead to defects in chromosome structure that are likely to compromise the ability of chromosomes to segregate faithfully if allowed to proceed into anaphase. To test this, we arrested SPT16-AID cells in metaphase, depleted FACT, and released cells from the arrest. To ensure that cells did not pass beyond telophase, so that we could accurately score chromosome segregation, we performed these experiments in a CDC15-AID genetic background. Therefore, upon release from metaphase, the addition of auxin to degrade Spt16, also induced the degradation of the mitotic exit kinase Cdc15, causing a tight telophase arrest. Similar to our analysis of sister chromatid cohesion, we used tetO/tetR-based tags on chromosome IV to evaluate chromosome segregation in the Cdc15 block. As predicted, we observed high levels of chromosome missegregation in cells depleted for Spt16 and Cdc15, but not Cdc15 only (Fig. 5).

Cohesin binding leads to changes in nucleosome positions at pericentric regionsWe first sought to analyse if nucleosomal occupancy is dynamic at sites of cohesin binding. In Saccharomyces cerevisiae CEN sequences are cohesin loading sites where complexes initially access chromosomes before spreading towards pericentric regions 14–17. Genome-wide MNase-seq maps were generated 18 in G1 and G2/M arrests to analyse nucleosome occupancy around centromeric regions (Fig. 1a). Although the yeast genome is characterised by having well-defined nucleosomal positions throughout 19, we detected differences in the positioning and occupancy of individual nucleosomes around pericentromeric regions between the G1 and G2/M MNase-seq maps (Fig. 1a). The general trend involved the loss of defined positioning in G2/M arrests (increased nucleosome fuzziness- Fig 1b). The differences were confirmed using DANPOS 2 20 (Fig. 1b) and showed a general correlation with regions enriched in pericentric cohesin (Fig. 1c). Moreover, auxin-mediated degradation of cohesin’s kleisin Mcd1 in G2/M arrests shifted the nucleosomal profiles closer to what we had observed for G1 arrests (Fig. 1a), i.e. reverting nucleosome fuzziness (Fig. 1b). Together the results are consistent with the idea that nucleosomes are dynamic around CEN regions where cohesins are loaded and spread to pericentromeric sites.

Cohesin binding leads to changes in nucleosome positions at pericentric regions

We first sought to analyse if nucleosomal occupancy is dynamic at sites of cohesin binding. In Saccharomyces cerevisiae CEN sequences are cohesin loading sites where complexes initially access chromosomes before spreading towards pericentric regions 14–17. Genome-wide MNase-seq maps were generated 18 in G1 and G2/M arrests to analyse nucleosome occupancy around centromeric regions (Fig. 1a). Although the yeast genome is characterised by having well-defined nucleosomal positions throughout 19, we detected differences in the positioning and occupancy of individual nucleosomes around pericentromeric regions between the G1 and G2/M MNase-seq maps (Fig. 1a). The general trend involved the loss of defined positioning in G2/M arrests (increased nucleosome fuzziness- Fig 1b). The differences were confirmed using DANPOS 2 20 (Fig. 1b) and showed a general correlation with regions enriched in pericentric cohesin (Fig. 1c). Moreover, auxin-mediated degradation of cohesin’s kleisin Mcd1 in G2/M arrests shifted the nucleosomal profiles closer to what we had observed for G1 arrests (Fig. 1a), i.e. reverting nucleosome fuzziness (Fig. 1b). Together the results are consistent with the idea that nucleosomes are dynamic around CEN regions where cohesins are loaded and spread to pericentromeric sites.

The Cohesin complex physically interacts with chromatin remodellersNext, we sought to identify potential chromatin remodellers involved in cohesin function. To this aim, cells expressing Mcd1 tagged with 6xHA were arrested in metaphase and Mcd1 immunoprecipitates were analysed by mass spectrometry (Table 1). Importantly, cell extracts were treated with benzonase to eliminate DNA prior to Mcd1 immunoprecipitation, to rule out indirect interactions mediated by DNA (Fig. 1d). We found that some chromatin remodellers were highly enriched in our pulldowns (Table 1). The most abundant was the Spt16 subunit of the FACT complex (facilitates chromatin transcription) (Table 1). Consistent with previous reports 21, several subunits of RSC were also amongst the hits (Table 1).We were particularly intrigued by the identification of FACT because chromosome assembly in vitro requires FACT alongside the SMC complex condensin 22. FACT is a heterodimeric complex containing Spt16 and Pob3 that acts as a histone chaperone regulating the assembly and disassembly of nucleosomes during transcription elongation, DNA replication and DNA repair 23. The ability of FACT to mobilise nucleosomes makes it an ideal candidate to facilitate cohesin functions on chromatin or the extrusion of chromatin loops through cohesin rings. We confirmed the interaction between cohesin and FACT in metaphase arrests using cells that expressed SMC1-6HA and SPT16-9MYC to validate that cohesin physically interacts with this histone chaperone (Fig. 1e).

The Cohesin complex physically interacts with chromatin remodellers

Next, we sought to identify potential chromatin remodellers involved in cohesin function. To this aim, cells expressing Mcd1 tagged with 6xHA were arrested in metaphase and Mcd1 immunoprecipitates were analysed by mass spectrometry (Table 1). Importantly, cell extracts were treated with benzonase to eliminate DNA prior to Mcd1 immunoprecipitation, to rule out indirect interactions mediated by DNA (Fig. 1d). We found that some chromatin remodellers were highly enriched in our pulldowns (Table 1). The most abundant was the Spt16 subunit of the FACT complex (facilitates chromatin transcription) (Table 1). Consistent with previous reports 21, several subunits of RSC were also amongst the hits (Table 1).

We were particularly intrigued by the identification of FACT because chromosome assembly in vitro requires FACT alongside the SMC complex condensin 22. FACT is a heterodimeric complex containing Spt16 and Pob3 that acts as a histone chaperone regulating the assembly and disassembly of nucleosomes during transcription elongation, DNA replication and DNA repair 23. The ability of FACT to mobilise nucleosomes makes it an ideal candidate to facilitate cohesin functions on chromatin or the extrusion of chromatin loops through cohesin rings. We confirmed the interaction between cohesin and FACT in metaphase arrests using cells that expressed SMC1-6HA and SPT16-9MYC to validate that cohesin physically interacts with this histone chaperone (Fig. 1e).

FACT is necessary for the localization of cohesin on yeast chromosomesTo further explore a possible functional relationship between FACT and cohesin, we sought to test whether inactivation of FACT in G2/M arrests caused any alterations in cohesin binding. An auxin (IAA)-inducible degron of Spt16 (SPT16-AID) 24 was used to deplete FACT from cells in metaphase (Suppl. Fig. 1). We performed calibrated ChIP-seq as described in the literature 25 for Smc1, a subunit of the cohesin complex, to analyse cohesin localisation before and after Spt16 depletion along yeast chromosomes (Fig. 2a). Centromeric regions are ideal sites to study cohesin translocation because cohesin complexes are initially loaded at core CEN sequences and then move away towards pericentromeric regions 17. We observed a small effect on cohesin localisation to core CEN sequence upon Spt16 degradation, however its accumulation at pericentric regions was severely reduced (Fig. 2a-b). This suggest that FACT might not be required for cohesin initial loading at CEN sites 25, but it is necessary for its accumulation on peri-centric regions (Fig. 2b). Mcd1 is expressed at low levels in G1-arrested cells, thus cohesin association to pericentric regions is very modest at this stage of the cell cycle 17. However, cohesin loading can still occur in G1
25 when cohesin expression is artificially induced. Indeed, expression of MCD1 from the GAL1-10 promoter caused a significant enrichment of Smc1 both at core CEN sequences and the surrounding pericentric regions (Fig. 2c). In contrast, MCD1 expression in G1-arrests depleted in Spt16 showed no enrichment of cohesin at pericentric regions (Fig. 2c), despite the fact that Smc1 still was present at core CEN sequences (Fig. 2c). Therefore, these results demonstrate that in the absence of the histone chaperone FACT, cohesins can be loaded at core CEN sites but fail to accumulate at pericentric regions. To confirm that the loss of cohesin signal in FACT-depleted cells was not due to a defect in the ability of cohesin to load, we first tested whether cohesin loading occurs in G2/M arrested cells (the stage where we depleted FACT in our experiments). To do this, we used an auxin (IAA)-inducible degron of Sth1 (STH1-AID), a core subunit of RSC, because RSC is required for cohesin loading26. Cohesin binding around centromeres and chromosome arms was similar to wildtype cells in the Sth1-depletion (Fig. 2d & Suppl. Fig. 2). In addition, FACT depletion in the absence of the cohesin release factor Wapl also did not affect cohesin binding on chromosomes (Fig. 2d), demonstrating that cohesin can be loaded normally in the absence of FACT. Therefore, the loss of cohesin signal in FACT-depleted cells cannot be caused by an impairment in cohesin loading but rather the stability of cohesin on chromatin.

FACT is necessary for the localization of cohesin on yeast chromosomes

To further explore a possible functional relationship between FACT and cohesin, we sought to test whether inactivation of FACT in G2/M arrests caused any alterations in cohesin binding. An auxin (IAA)-inducible degron of Spt16 (SPT16-AID) 24 was used to deplete FACT from cells in metaphase (Suppl. Fig. 1). We performed calibrated ChIP-seq as described in the literature 25 for Smc1, a subunit of the cohesin complex, to analyse cohesin localisation before and after Spt16 depletion along yeast chromosomes (Fig. 2a). Centromeric regions are ideal sites to study cohesin translocation because cohesin complexes are initially loaded at core CEN sequences and then move away towards pericentromeric regions 17. We observed a small effect on cohesin localisation to core CEN sequence upon Spt16 degradation, however its accumulation at pericentric regions was severely reduced (Fig. 2a-b). This suggest that FACT might not be required for cohesin initial loading at CEN sites 25, but it is necessary for its accumulation on peri-centric regions (Fig. 2b). Mcd1 is expressed at low levels in G1-arrested cells, thus cohesin association to pericentric regions is very modest at this stage of the cell cycle 17. However, cohesin loading can still occur in G1
25 when cohesin expression is artificially induced. Indeed, expression of MCD1 from the GAL1-10 promoter caused a significant enrichment of Smc1 both at core CEN sequences and the surrounding pericentric regions (Fig. 2c). In contrast, MCD1 expression in G1-arrests depleted in Spt16 showed no enrichment of cohesin at pericentric regions (Fig. 2c), despite the fact that Smc1 still was present at core CEN sequences (Fig. 2c). Therefore, these results demonstrate that in the absence of the histone chaperone FACT, cohesins can be loaded at core CEN sites but fail to accumulate at pericentric regions. To confirm that the loss of cohesin signal in FACT-depleted cells was not due to a defect in the ability of cohesin to load, we first tested whether cohesin loading occurs in G2/M arrested cells (the stage where we depleted FACT in our experiments). To do this, we used an auxin (IAA)-inducible degron of Sth1 (STH1-AID), a core subunit of RSC, because RSC is required for cohesin loading26. Cohesin binding around centromeres and chromosome arms was similar to wildtype cells in the Sth1-depletion (Fig. 2d & Suppl. Fig. 2). In addition, FACT depletion in the absence of the cohesin release factor Wapl also did not affect cohesin binding on chromosomes (Fig. 2d), demonstrating that cohesin can be loaded normally in the absence of FACT. Therefore, the loss of cohesin signal in FACT-depleted cells cannot be caused by an impairment in cohesin loading but rather the stability of cohesin on chromatin.

FACT contribution to cohesin localization is independent of transcriptionIn yeast, cohesin complexes can be divided into two classes with respect to their chromosomal association mechanisms, those loaded at CEN sequences, which move and accumulate in pericentric regions 27,28, and those that load at chromosomal arm sites and translocate to regions of convergent transcription 29. Importantly, the translocation of cohesins from CEN sites to pericentromeric regions requires Scc2 17 and is independent of transcription 17, while the movement of arm loaded cohesins to sites between convergent genes involves relocation dependent on transcription 29–32. FACT plays important roles during transcription initiation and elongation33,34. Therefore, we decided to investigate to what extent changes in transcription upon FACT-depletion are responsible for the disruption in the cohesin ChIP profiles. First, we focused on centromeric regions where transcription should not affect movement of CEN loaded cohesins to pericentric regions 17. To confirm that accumulation of cohesin at pericentromeric regions does not require active transcription, we treated wildtype cells with the transcription repressor thiolutin and used calibrated ChiP-seq to Smc1 to compare cohesin enrichment in the presence and absence of active transcription (Fig. 3a). Thiolutin treatment had little effect on Smc1 binding at core CEN sequences or pericentric sites (Fig. 3a). Therefore, as reported previously17, accumulation of cohesin at these sites is independent of transcription. In contrast, Spt16 degradation in thiolutin-treated cells caused a reduction in cohesin presence on pericentric regions (Fig. 3a). Therefore, at pericentric regions FACT facilitates cohesin accumulation independently of active transcription.The analysis of cohesin translocation on chromosome arms is more challenging that at centromeres because cohesin loading sites at chromosome arms are not well defined. ChIP profiles reflect the final position of the complexes. Our results demonstrate that FACT inactivation causes a decrease in cohesin binding at chromosomal arm sites (Fig. 2a). We therefore decided to test whether the effect of FACT on cohesin deposition at chromosome arms was independent on transcription, as observed for pericentric regions (Fig. 3a). Smc1 binding across chromosome arm sites was reduced in wildtype cells treated with thiolutin (Fig. 3b) confirming that active transcription is important for cohesin enrichment at these sites 29–32. Thiolutin treatment in FACT-depleted cells caused a further reduction in cohesin binding across chromosome sites (Fig. 3b).To further dissect the contribution of FACT to cohesin stability, we analysed how Smc1 enrichment at convergent transcription sites correlates with the changes in transcription at these sites when Spt16 is depleted (Fig. 3c). Analysis of cohesin binding sites confirmed their enrichment in positions in the genome where transcription units are oriented in a convergent manner 29–31. Interestingly, cohesin enrichment was observed as a double peak at both 3’ ends of the genes (Fig. 3c). We classified convergent sites into categories depending on whether the transcription of the flanking genes had change by 2-fold up or down or had remain the same (Fig. 3c) upon Spt16 depletion. We then compared cohesin enrichment changes in each transcription-pair category (Fig. 3c). Smc1 was reduced similarly in all categories (Fig. 3c). Therefore, we conclude that FACT has a direct function in cohesin stability on chromosome arm sites and that changes in cohesin binding cannot be explained solely by the altered transcription profiles upon depletion of Spt16.

FACT contribution to cohesin localization is independent of transcription

In yeast, cohesin complexes can be divided into two classes with respect to their chromosomal association mechanisms, those loaded at CEN sequences, which move and accumulate in pericentric regions 27,28, and those that load at chromosomal arm sites and translocate to regions of convergent transcription 29. Importantly, the translocation of cohesins from CEN sites to pericentromeric regions requires Scc2 17 and is independent of transcription 17, while the movement of arm loaded cohesins to sites between convergent genes involves relocation dependent on transcription 29–32. FACT plays important roles during transcription initiation and elongation33,34. Therefore, we decided to investigate to what extent changes in transcription upon FACT-depletion are responsible for the disruption in the cohesin ChIP profiles. First, we focused on centromeric regions where transcription should not affect movement of CEN loaded cohesins to pericentric regions 17. To confirm that accumulation of cohesin at pericentromeric regions does not require active transcription, we treated wildtype cells with the transcription repressor thiolutin and used calibrated ChiP-seq to Smc1 to compare cohesin enrichment in the presence and absence of active transcription (Fig. 3a). Thiolutin treatment had little effect on Smc1 binding at core CEN sequences or pericentric sites (Fig. 3a). Therefore, as reported previously17, accumulation of cohesin at these sites is independent of transcription. In contrast, Spt16 degradation in thiolutin-treated cells caused a reduction in cohesin presence on pericentric regions (Fig. 3a). Therefore, at pericentric regions FACT facilitates cohesin accumulation independently of active transcription.

The analysis of cohesin translocation on chromosome arms is more challenging that at centromeres because cohesin loading sites at chromosome arms are not well defined. ChIP profiles reflect the final position of the complexes. Our results demonstrate that FACT inactivation causes a decrease in cohesin binding at chromosomal arm sites (Fig. 2a). We therefore decided to test whether the effect of FACT on cohesin deposition at chromosome arms was independent on transcription, as observed for pericentric regions (Fig. 3a). Smc1 binding across chromosome arm sites was reduced in wildtype cells treated with thiolutin (Fig. 3b) confirming that active transcription is important for cohesin enrichment at these sites 29–32. Thiolutin treatment in FACT-depleted cells caused a further reduction in cohesin binding across chromosome sites (Fig. 3b).

To further dissect the contribution of FACT to cohesin stability, we analysed how Smc1 enrichment at convergent transcription sites correlates with the changes in transcription at these sites when Spt16 is depleted (Fig. 3c). Analysis of cohesin binding sites confirmed their enrichment in positions in the genome where transcription units are oriented in a convergent manner 29–31. Interestingly, cohesin enrichment was observed as a double peak at both 3’ ends of the genes (Fig. 3c). We classified convergent sites into categories depending on whether the transcription of the flanking genes had change by 2-fold up or down or had remain the same (Fig. 3c) upon Spt16 depletion. We then compared cohesin enrichment changes in each transcription-pair category (Fig. 3c). Smc1 was reduced similarly in all categories (Fig. 3c). Therefore, we conclude that FACT has a direct function in cohesin stability on chromosome arm sites and that changes in cohesin binding cannot be explained solely by the altered transcription profiles upon depletion of Spt16.

FACT is not required for sister chromatid cohesionHaving demonstrated that FACT has a role in maintaining cohesin on chromosomes (Fig. 3a-c), we sought to investigate the functional consequences for chromosome architecture. Cohesin holds sister chromatid together 35,36 and organises intra-chromatid loops that provide an integral level of structure to chromosomes 2,3. First, we tested whether cohesion between sister chromatids is affected in G2/M arrested cells when Spt16 is depleted. We scored cohesion using tetO/tetR-based tags at sub-telomeric regions of chromosome IV (Fig. 3d). Cells depleted for Spt16 showed no defects in cohesion (Fig. 3d).

FACT is not required for sister chromatid cohesion

Having demonstrated that FACT has a role in maintaining cohesin on chromosomes (Fig. 3a-c), we sought to investigate the functional consequences for chromosome architecture. Cohesin holds sister chromatid together 35,36 and organises intra-chromatid loops that provide an integral level of structure to chromosomes 2,3. First, we tested whether cohesion between sister chromatids is affected in G2/M arrested cells when Spt16 is depleted. We scored cohesion using tetO/tetR-based tags at sub-telomeric regions of chromosome IV (Fig. 3d). Cells depleted for Spt16 showed no defects in cohesion (Fig. 3d).

FACT activity affects the role of cohesin in genome foldingNext, we tested whether Spt16 is necessary to maintain mitotic chromosome architecture. Recent studies on mitotic yeast chromosomes using Hi-C have shown that cohesin compacts chromosome arms through cis interactions, independently of sister chromatid cohesion 2,3. We therefore considered the possibility that the reduction in cohesins observed upon Spt16-depletion (Fig. 2a) might impact on cis contacts along yeast metaphase chromosomes. To explore this, we built Hi-C libraries from G2/M arrested cells depleted for Spt16 and compared it to control arrests (Fig. 4a and Suppl. Fig. 3). The resulting contact maps show that depletion of Spt16 led to a reduction in short-distance intra-chromosomal contacts while long-range intra-chromosomal contacts were increased (> 100kb), as suggested by i) a larger width of the main diagonal in the contact maps of cells depleted for Spt16 (Suppl. Fig. 3), and as confirmed by computing ii) the contact probability p as a function of genomic distance s of all chromosome arms (Fig. 4b) and iii) the log-ratio between the normalized contact maps of depleted and non-depleted cells (Fig. 4a). The ratio map also shows that Spt16 depletion cause an increase of inter-chromosomal contacts (Fig. 4a). These results suggest that chromatin fibres loose internal organization in the absence of Spt16.To further characterize mitotic chromosomes, 2 kb contact maps were generated. At this resolution, triangular contact patterns reminiscent of the topological associating domain (TADs) described in metazoans appear along mitotic chromosomes in G2/M arrested cells but not G1 arrests (Fig. 4c). The boundaries of these domains involve regions enriched in cohesin (red diamonds, Fig. 4c), and stronger contacts between pairs of boundaries are often visible, consistent with the establishment of cohesin loops (black triangles, Fig. 4c) (previous Hi-C studies on cohesin failed to directly characterize these loops, as the experiments were not analysed at this resolution 2,3). We tested whether the triangular patterns of the 2 kb contact maps were dependent on cohesin. To this aim, we built Hi-C libraries from cells depleted for Mcd1 while arrested in G2/M (Fig. 2c). The resulting contact maps showed a significant reduction in TAD-like structures (Fig. 4c), as expected from previous reports demonstrating a key role for cohesin in loop formation 2,3,6. The depletion of Spt16 results in a strong reduction of cohesin deposition (Fig. 4c), as well as a fading of the cohesin-dependent TADs-like domains signal (Fig. 4c).Our Hi-C analysis is consistent with a role for FACT in affecting cohesin-dependent loops, however we cannot differentiate whether FACT is important to establish these loops or to maintain them. To test a potential role for FACT in de novo loop formation by cohesin, we first investigated whether overexpressing MCD1 in G1, which causes cohesin loading (Fig. 2c), leads to de novo formation of loop, which are normally absent in G1 cells 2,3 (Fig. 4c). Hi-C libraries from G1 arrested cells overexpressing Mcd1 from the GAL1-10 promoter were thus generated. TADs-like domain signals were observed in 2 kb contact maps of G1 overexpressing Mcd1 (Fig. 4d) demonstrating de novo loop formation by cohesin under this experimental condition (Fig. 4d). Next, we depleted Spt16 during the overexpression of Mcd1 in G1 cells, cohesin dependent loops were now strongly reduced, and hardly visible, in the absence of FACT (Suppl. Fig. 4). The log-ratio between the normalized contact maps of G1 depleted and non-depleted cells (Fig. 4e) and the contact probabilities as a function of genomic distance s of all chromosome arms (Fig. 4f) confirms that de novo loop formation by cohesins in G1 requires FACT.

FACT activity affects the role of cohesin in genome folding

Next, we tested whether Spt16 is necessary to maintain mitotic chromosome architecture. Recent studies on mitotic yeast chromosomes using Hi-C have shown that cohesin compacts chromosome arms through cis interactions, independently of sister chromatid cohesion 2,3. We therefore considered the possibility that the reduction in cohesins observed upon Spt16-depletion (Fig. 2a) might impact on cis contacts along yeast metaphase chromosomes. To explore this, we built Hi-C libraries from G2/M arrested cells depleted for Spt16 and compared it to control arrests (Fig. 4a and Suppl. Fig. 3). The resulting contact maps show that depletion of Spt16 led to a reduction in short-distance intra-chromosomal contacts while long-range intra-chromosomal contacts were increased (> 100kb), as suggested by i) a larger width of the main diagonal in the contact maps of cells depleted for Spt16 (Suppl. Fig. 3), and as confirmed by computing ii) the contact probability p as a function of genomic distance s of all chromosome arms (Fig. 4b) and iii) the log-ratio between the normalized contact maps of depleted and non-depleted cells (Fig. 4a). The ratio map also shows that Spt16 depletion cause an increase of inter-chromosomal contacts (Fig. 4a). These results suggest that chromatin fibres loose internal organization in the absence of Spt16.

To further characterize mitotic chromosomes, 2 kb contact maps were generated. At this resolution, triangular contact patterns reminiscent of the topological associating domain (TADs) described in metazoans appear along mitotic chromosomes in G2/M arrested cells but not G1 arrests (Fig. 4c). The boundaries of these domains involve regions enriched in cohesin (red diamonds, Fig. 4c), and stronger contacts between pairs of boundaries are often visible, consistent with the establishment of cohesin loops (black triangles, Fig. 4c) (previous Hi-C studies on cohesin failed to directly characterize these loops, as the experiments were not analysed at this resolution 2,3). We tested whether the triangular patterns of the 2 kb contact maps were dependent on cohesin. To this aim, we built Hi-C libraries from cells depleted for Mcd1 while arrested in G2/M (Fig. 2c). The resulting contact maps showed a significant reduction in TAD-like structures (Fig. 4c), as expected from previous reports demonstrating a key role for cohesin in loop formation 2,3,6. The depletion of Spt16 results in a strong reduction of cohesin deposition (Fig. 4c), as well as a fading of the cohesin-dependent TADs-like domains signal (Fig. 4c).

Our Hi-C analysis is consistent with a role for FACT in affecting cohesin-dependent loops, however we cannot differentiate whether FACT is important to establish these loops or to maintain them. To test a potential role for FACT in de novo loop formation by cohesin, we first investigated whether overexpressing MCD1 in G1, which causes cohesin loading (Fig. 2c), leads to de novo formation of loop, which are normally absent in G1 cells 2,3 (Fig. 4c). Hi-C libraries from G1 arrested cells overexpressing Mcd1 from the GAL1-10 promoter were thus generated. TADs-like domain signals were observed in 2 kb contact maps of G1 overexpressing Mcd1 (Fig. 4d) demonstrating de novo loop formation by cohesin under this experimental condition (Fig. 4d). Next, we depleted Spt16 during the overexpression of Mcd1 in G1 cells, cohesin dependent loops were now strongly reduced, and hardly visible, in the absence of FACT (Suppl. Fig. 4). The log-ratio between the normalized contact maps of G1 depleted and non-depleted cells (Fig. 4e) and the contact probabilities as a function of genomic distance s of all chromosome arms (Fig. 4f) confirms that de novo loop formation by cohesins in G1 requires FACT.

FACT-dependent chromosome organisation in metaphase is required for chromosome segregationOur results demonstrate that while cohesin’s role in sister chromatid cohesion is maintained in the absence of FACT (Fig. 3d), its function in organising chromatin loops in metaphase chromosomes is impaired (Fig. 4a-c). We therefore hypothesized that depletion of FACT in metaphase arrested cells might lead to defects in chromosome structure that are likely to compromise the ability of chromosomes to segregate faithfully if allowed to proceed into anaphase. To test this, we arrested SPT16-AID cells in metaphase, depleted FACT, and released cells from the arrest. To ensure that cells did not pass beyond telophase, so that we could accurately score chromosome segregation, we performed these experiments in a CDC15-AID genetic background. Therefore, upon release from metaphase, the addition of auxin to degrade Spt16, also induced the degradation of the mitotic exit kinase Cdc15, causing a tight telophase arrest. Similar to our analysis of sister chromatid cohesion, we used tetO/tetR-based tags on chromosome IV to evaluate chromosome segregation in the Cdc15 block. As predicted, we observed high levels of chromosome missegregation in cells depleted for Spt16 and Cdc15, but not Cdc15 only (Fig. 5).

FACT-dependent chromosome organisation in metaphase is required for chromosome segregation

Our results demonstrate that while cohesin’s role in sister chromatid cohesion is maintained in the absence of FACT (Fig. 3d), its function in organising chromatin loops in metaphase chromosomes is impaired (Fig. 4a-c). We therefore hypothesized that depletion of FACT in metaphase arrested cells might lead to defects in chromosome structure that are likely to compromise the ability of chromosomes to segregate faithfully if allowed to proceed into anaphase. To test this, we arrested SPT16-AID cells in metaphase, depleted FACT, and released cells from the arrest. To ensure that cells did not pass beyond telophase, so that we could accurately score chromosome segregation, we performed these experiments in a CDC15-AID genetic background. Therefore, upon release from metaphase, the addition of auxin to degrade Spt16, also induced the degradation of the mitotic exit kinase Cdc15, causing a tight telophase arrest. Similar to our analysis of sister chromatid cohesion, we used tetO/tetR-based tags on chromosome IV to evaluate chromosome segregation in the Cdc15 block. As predicted, we observed high levels of chromosome missegregation in cells depleted for Spt16 and Cdc15, but not Cdc15 only (Fig. 5).

DiscussionIn this study we identified FACT as a new regulator of cohesin in budding yeast. Our findings shed new light on the relationship between cohesin and chromatin factors and suggests a novel role of this histone chaperone in genome and chromosome organisation. FACT is a conserved histone chaperone, involved in both assembly and disassembly of core nucleosome particles by modulation of H2A-H2B histone dimers 37. In yeast, FACT is a heterodimeric complex composed of Spt16 and Pob3 38. In addition, the HMG protein, Nhp6, is also required for FACT roles during transcription, replication and DNA repair 38. Interestingly the ATP-dependent chromatin remodeling enzyme Chd1 (chromodomain-helicase-DNA binding protein 1) functions alongside FACT during transcription elongation 39. Chd1 is a regulator of cohesin’s roles in sister chromatid cohesion and chromosome condensation 40. Interestingly, Chd1 was identified in our mass spectrometry screen as one of the chromatin factors physically interacting with Mcd1 (Table 1), raising the possibility that Chd1 regulation of cohesin is through FACT.One of the prominent roles of FACT is in facilitating DNA replication38, because nucleosome represent barriers for the unwinding of the MCM helicase. A direct interaction between MCM and FACT is thought to recruit the histone chaperone during replication and unfold DNA from histone octamers so that MCM movement can proceed through the chromatinised genome 41. Our findings demonstrate that the movement of cohesin from its loading site at the core CEN sequence to the flanking pericentromeric regions is severely disrupted when FACT is depleted from yeast cells (Fig. 2b-c). This strongly suggests a role for FACT in facilitating cohesin movement through chromatin, at least at centromeric regions. It is also possible that FACT impacts on the loading of cohesin by binding to nucleosomes (particularly H2A-H2B dimers) and releasing free DNA that could then promote cohesin binding. Such a role would be similar to that described for RSC, which generates nucleosome-free regions allowing cohesin and its loader to access DNA 26. It is however important to note that depleting FACT in the absence of Wapl supresses the loss of cohesin binding (Fig. 2d) observed when FACT is inactivated in wildtype cells (Fig. 2a). Therefore FACT is not strictly required for the loading of cohesin. Supporting this view is the observation that depletion FACT leads to very different effects on the localisation of bound cohesin compared to the inactivation of RSC; severe disruption in cohesin localisation is observed when FACT is inactivated in G2/M cells (Fig. 2a), while RSC depletion does not affect the localisation of already bound cohesin (Fig. 2d). Therefore, it is likely that RSC plays a prominent role at the loading stage while FACT is required to maintain the stability of loaded cohesin on chromatin, either by facilitating its movement through nucleosomal arrays and/or counteracting its removal by anti-establishment factors, such as Wapl.The translocation of cohesins from CEN sites to pericentromeric regions is independent of transcription 17. However, cohesin is found on intergenic regions (IGRs) that lie between convergent genes29–32 suggesting that its movement to these sites might be facilitated by transcription 29–32. Since FACT is important during transcription initiation and elongation33,34, an alteration of the transcription profile in FACT-depleted cells could impact on the changes observed in cohesin stability (Fig. 2a). Several lines of evidence rule out this possibility. First, when FACT is depleted in the absence of transcription, cohesin localisation is still disrupted (Fig. 3b). Second, like FACT-depletion, inactivation of RSC also alters transcription profiles, however stability of cohesin is not affected by RSC-depletion (Fig. 2d). These findings strongly suggest that the stability of bound cohesin in G2/M cells is not affected by transcription changes. We propose that transcription might play a more important role in the initial localisation of cohesin when they are loaded in S phase, but not in metaphase arrested cells.Abrogating cohesin function leads to a loss of cis-loops required for TADs in interphase nuclei of human cells 6 as well as correct folding of mitotic chromosome arms in budding yeast 3. Previous studies in yeast had not found structures resembling topologically associated domains (TADs) found in metazoans. However, when we increased the resolution of normalized contact maps to 2kb (Fig. 4a) we detected triangular contact patterns in mitotic chromosomes (G2/M arrests) (Fig. 4a) which are comparable to the TAD structures described in metazoans. These structures were not observed in G1 (Fig. 4a), however overexpression of the kleisin subunit of cohesin in these G1 arrests led to the formation of de novo cis-looping and TAD-like structures (Fig. 4d) with similar characteristics to the structures observed in metaphase (Fig. 4a). This confirms that cohesin dependent loops can be formed in the absence of sister-chromatid cohesion 3, and demonstrates that cohesin role in genome folding is functionally distinct from its role in sister chromatid cohesion. Importantly, formation of G1 loops also required Spt16 (Fig. 4d), demonstrating that the cohesin-dependent process that leads to the formation or maintenance of these genomic structures heavily relies on the histone chaperone FACT.In summary, our work demonstrates that the FACT complex is a mediator of cohesin stability on chromatin (Fig. 2b-c) and through this function facilitates the formation of cohesin-dependent loops (Fig. 4). Recently, an involvement of FACT in chromatin architecture independent of transcription has been proposed 42, our data suggests that this function might be related to cohesin’s role in genome architecture. We speculate that similar to its roles in transcription and replication, FACT function as a histone chaperone is likely to occur through destabilization of core nucleosomal particles, to allow movement of cohesin on chromatin. We demonstrate that FACT contributes to cohesin’s role in organizing mitotic chromosome folding but not sister chromatid cohesion. Finally, our results support the original claim that cohesin complexes have two distinct roles, in chromosome organisation and sister chromatid cohesion 3543 and demonstrate that FACT is exclusively necessary for functions in genome folding.

In this study we identified FACT as a new regulator of cohesin in budding yeast. Our findings shed new light on the relationship between cohesin and chromatin factors and suggests a novel role of this histone chaperone in genome and chromosome organisation. FACT is a conserved histone chaperone, involved in both assembly and disassembly of core nucleosome particles by modulation of H2A-H2B histone dimers 37. In yeast, FACT is a heterodimeric complex composed of Spt16 and Pob3 38. In addition, the HMG protein, Nhp6, is also required for FACT roles during transcription, replication and DNA repair 38. Interestingly the ATP-dependent chromatin remodeling enzyme Chd1 (chromodomain-helicase-DNA binding protein 1) functions alongside FACT during transcription elongation 39. Chd1 is a regulator of cohesin’s roles in sister chromatid cohesion and chromosome condensation 40. Interestingly, Chd1 was identified in our mass spectrometry screen as one of the chromatin factors physically interacting with Mcd1 (Table 1), raising the possibility that Chd1 regulation of cohesin is through FACT.

One of the prominent roles of FACT is in facilitating DNA replication38, because nucleosome represent barriers for the unwinding of the MCM helicase. A direct interaction between MCM and FACT is thought to recruit the histone chaperone during replication and unfold DNA from histone octamers so that MCM movement can proceed through the chromatinised genome 41. Our findings demonstrate that the movement of cohesin from its loading site at the core CEN sequence to the flanking pericentromeric regions is severely disrupted when FACT is depleted from yeast cells (Fig. 2b-c). This strongly suggests a role for FACT in facilitating cohesin movement through chromatin, at least at centromeric regions. It is also possible that FACT impacts on the loading of cohesin by binding to nucleosomes (particularly H2A-H2B dimers) and releasing free DNA that could then promote cohesin binding. Such a role would be similar to that described for RSC, which generates nucleosome-free regions allowing cohesin and its loader to access DNA 26. It is however important to note that depleting FACT in the absence of Wapl supresses the loss of cohesin binding (Fig. 2d) observed when FACT is inactivated in wildtype cells (Fig. 2a). Therefore FACT is not strictly required for the loading of cohesin. Supporting this view is the observation that depletion FACT leads to very different effects on the localisation of bound cohesin compared to the inactivation of RSC; severe disruption in cohesin localisation is observed when FACT is inactivated in G2/M cells (Fig. 2a), while RSC depletion does not affect the localisation of already bound cohesin (Fig. 2d). Therefore, it is likely that RSC plays a prominent role at the loading stage while FACT is required to maintain the stability of loaded cohesin on chromatin, either by facilitating its movement through nucleosomal arrays and/or counteracting its removal by anti-establishment factors, such as Wapl.

The translocation of cohesins from CEN sites to pericentromeric regions is independent of transcription 17. However, cohesin is found on intergenic regions (IGRs) that lie between convergent genes29–32 suggesting that its movement to these sites might be facilitated by transcription 29–32. Since FACT is important during transcription initiation and elongation33,34, an alteration of the transcription profile in FACT-depleted cells could impact on the changes observed in cohesin stability (Fig. 2a). Several lines of evidence rule out this possibility. First, when FACT is depleted in the absence of transcription, cohesin localisation is still disrupted (Fig. 3b). Second, like FACT-depletion, inactivation of RSC also alters transcription profiles, however stability of cohesin is not affected by RSC-depletion (Fig. 2d). These findings strongly suggest that the stability of bound cohesin in G2/M cells is not affected by transcription changes. We propose that transcription might play a more important role in the initial localisation of cohesin when they are loaded in S phase, but not in metaphase arrested cells.

Abrogating cohesin function leads to a loss of cis-loops required for TADs in interphase nuclei of human cells 6 as well as correct folding of mitotic chromosome arms in budding yeast 3. Previous studies in yeast had not found structures resembling topologically associated domains (TADs) found in metazoans. However, when we increased the resolution of normalized contact maps to 2kb (Fig. 4a) we detected triangular contact patterns in mitotic chromosomes (G2/M arrests) (Fig. 4a) which are comparable to the TAD structures described in metazoans. These structures were not observed in G1 (Fig. 4a), however overexpression of the kleisin subunit of cohesin in these G1 arrests led to the formation of de novo cis-looping and TAD-like structures (Fig. 4d) with similar characteristics to the structures observed in metaphase (Fig. 4a). This confirms that cohesin dependent loops can be formed in the absence of sister-chromatid cohesion 3, and demonstrates that cohesin role in genome folding is functionally distinct from its role in sister chromatid cohesion. Importantly, formation of G1 loops also required Spt16 (Fig. 4d), demonstrating that the cohesin-dependent process that leads to the formation or maintenance of these genomic structures heavily relies on the histone chaperone FACT.

In summary, our work demonstrates that the FACT complex is a mediator of cohesin stability on chromatin (Fig. 2b-c) and through this function facilitates the formation of cohesin-dependent loops (Fig. 4). Recently, an involvement of FACT in chromatin architecture independent of transcription has been proposed 42, our data suggests that this function might be related to cohesin’s role in genome architecture. We speculate that similar to its roles in transcription and replication, FACT function as a histone chaperone is likely to occur through destabilization of core nucleosomal particles, to allow movement of cohesin on chromatin. We demonstrate that FACT contributes to cohesin’s role in organizing mitotic chromosome folding but not sister chromatid cohesion. Finally, our results support the original claim that cohesin complexes have two distinct roles, in chromosome organisation and sister chromatid cohesion 3543 and demonstrate that FACT is exclusively necessary for functions in genome folding.

MethodsMedia, culture conditions and synchronisationsS. cerevisiae strains used in this study are shown in Supplementary Table 1. To arrest the cells in G1, α-factor was added to exponentially growing MATa cultures (OD600=0.5) to a final concentration of 3 x 10-8 M. To arrest cells in G2/M, Nocodazole (1.5 mg/ml stock in DMSO 100%) was added to exponentially growing cultures (OD600=0.5) to a final concentration of 0.015 mg/ml. When arrests in Nocodazole are maintained for more than 2 hours cells are spun and resuspended in YPD containing fresh nocodazole (0.015 mg/ml). Cultures were monitored by microscopy until ≥90% of cells were arrested, typically 2 hours at 25 °C. To release cells from G1, the culture was spun (4,000 r.p.m, 1 min) and washed in YPD 3 times. The pellet was then resuspended in YPD containing 0.1 mg/ml pronase. To release cells from Nocodazole, the culture was spun (4,000 r.p.m, 1 min) and washed in YPD containing 1% DMSO 5 times. The pellet was then resuspended in YPD. To degrade proteins tagged with AID* epitope, a stock of IAA of 0.6M in ethanol 100% was used for a final concentration of 6mM. To inhibit transcription a stock of thiolutin at 1mg/ml in DMSO was added for 30 minutes to the culture at a final concentration of 10ug/ml.MicroscopyA Leica IRB epifluorescence microscope with a Hamamatsu 742-95 digital camera, a 63X/1.4 lens and OpenLab software (Improvision) was used for single cell visualization. 1 ml of cell culture was collected from each time point and mixed with glycerol (20% final concentration) to preserve TetO/TetR signal. Cells were then frozen at -80 °C. For visualization, cells where centrifuged at 3,000 r.p.m. for 2 minute and 1μl of the pellet was mixed with 1μl of DAPI solution (DAPI 4 μg/ml Triton 1 %) on the microscope slide. For each field 20 z-focal planes images were captured (0.3 μm depth between each consecutive image). At least 100 cells were score for GFP dots.Hi-C librariesHi-C experiments were performed according to a protocol adapted from 44. Aliquots of 1-3 x 109 cells in 150 ml culture medium were fixed in 3% formaldehyde (Sigma-Aldrich, Ref. F8775) and quenched in 0.4 M glycine for 20 min at room temperature and 4°C, respectively. Cells were harvested, washed with culture medium and pellets were stored at -80°C. Pellets were thawed on ice and resuspended in chilled TE buffer supplemented with protease inhibitors. Cells were lysed using Precellys VK05 Lysing KIT (Ozyme, Ref. 0042) at 6,700 rpm. Lysates were treated with 0.5 % SDS for 20 min at 65°C and digested overnight with DpnII (CFinal=500 U/pellet; NEB R0543M) at 37°C. Digestions were centrifuged for 20 min at 16,000 g and the pellets suspended in cold water. 5’ DpnII overhangs were filled-in and labelled with biotin using 30 μM dNTP (dATP, dGTP, dTTP and Biotin-14-dCTP Invitrogen Ref. 19518018). Biotinylated DNA fragments were ligated using T4 DNA ligase (CFinal=250 Weiss U/pellet; Thermo Scientific 10621441) for 4 h at 16°C. Cross-link was reverted through an overnight treatment in presence of 250 μg/ml proteinase K at 65°C. Total DNA was extracted using phenol/chloroform, precipitated and treated with RNAse A 500 μg/ml. Non-ligated DNA fragments were removed by T4 DNA polymerase (CFinal=5 U/pellet; NEB M0203L) treatment; whereas ligated fragments were 500 bp sheared using Covaris S220 and pulled-down using Dynabeads Myone Streptavidin C1 (Invitrogen 65001). Resulting libraries were amplified using custom-made primers for paired-end deep-sequencing on NextSeq500 Illumina platform (2x75 bp cycles).Generation and normalization of contact mapsThe reads from each Hi-C library were processed as follows. First, PCR duplicates were removed using the custom-made tags present on the primers and the resulting reads were mapped independently using Bowtie 2 (mode: --very-sensitive --rdg 500,3 --rfg 500,3) against the reference genome of S. cerevisiae (S288C) and indexed on DpnII restriction fragments. An iterative alignment procedure (20 bp window) was used to maximize the yield of uniquely mapped reads (mapping quality > 30). Reads were classified as either valid Hi-C products or unwanted events to be filtered out (i.e. loops, non-digested fragments, etc.; for details see {Cournac, 2012 #40}) based on their assignment and orientation on the DpnII restriction fragments in the reference genome. Valid Hi-C reads were used to generate contact frequency maps. In these maps, each vector consists in a fixed size genomic bin made of successive RF (either 5, 25, or 50 kb). Bins with a high variance in contact frequency (< 1.5 Standard Deviation or 1.5 - 2 S.D.) were filtered out. Filtered contact maps were normalized as described 45. Approximately 15 million of valid reads were used to build each contact map. Similarity between pairs of normalized contact maps was assessed by computing the log2 ratio for each point.Contact probability as a function of the genomic distance p(s)The contact probability p(s) as a function of the genomic distance s separating two DNA positions in cis was computed as follow. First, self-circularization events were discarded by removing pairs of read oriented in opposite directions along the reference genome and separated by less than 1.5 kb. The remaining reads were log-binned as a function of their genomic distance (in kb) on each chromosomal arm: bin = [log1.1(s)]. The p(s) curve is the histogram computed on the sum of read pairs for each bin. This sum is weighted by both bin-size 1.1(1+bin) (log binning), as well as by the length of the chromosomes.Chromatin immunoprecipitationFor ChIP analysis cells were grown to OD600 = 0.5 and arrested at the required cell cycle stage. 87 OD600 units of S. cerevisiae were mixed with 43 OD600 units of asynchronous C. glabrata. Cells were fixed for 15 minutes at 25 °C, and quenched with glycine (final concentration 125 mM) for 7 minutes before cells were harvested by centrifugation at 4,000 r.p.m. for 1 minute. The cell pellets were washed in PBS and transferred to a screw cap tube and frozen on dry ice. The pellets were stored at -80 °C. Pellets were resuspended in 300 μl of IP buffer (150 mM NaCl, 50 mM Tris-HCl (pH 7.5), 5 mM EDTA, NP-40 (0.05% v/v), Triton® X-100 (1% v/v)) containing phenylmethanesulfonyl fluoride (PMSF, final concentration 1 mM) and Complete protease inhibitor cocktail (without EDTA, from Roche). 500 μl of glass beads were added to the tubes. Cells were broken in a FastPrep® FP120 (BIO101) by 3 repetitions of a 20 seconds cycle, power setting 5.5. Cells were maintained on ice for 2 minutes after each cycle. The tubes were pierced with a hot needle and placed into new eppendorf and spun at 2000 r.p.m. for 2 minutes to collect the lysate. 100 μl of IP buffer containing PMSF and protease inhibitors were then added and tubes spun again at 2000 r.p.m. for 1 minute. The cell lysate was spun down for 10 minutes at 15,000 r.p.m at 4 °C. This pellet was resuspended in 1ml of IP buffer containing PMSF and protease inhibitors, and sonicated for 1 hour (30 seconds on, 30 seconds off) at high power at 4 °C in a Diagenode Bioruptor. After sonication samples were spun down for 10 minutes at 15,000 r.p.m. and the supernatant taken. 200 μl of the sonicated chromatin were taken as “input” and 400 μl was incubated with 40 μg of HA antibody (anti-HA 12CA5 from Roche) in a sonicator at low power for 30 minutes (30 seconds on, 30 seconds off). The “input” DNA was precipitated with 0.3 M sodium acetate and 2.5 volumes of cold ethanol and spun down at 15,000 r.p.m. for 30 minutes, and the supernatant removed. The pellet was washed with 70 % ethanol and air dried. After antibody binding the IP sample was spun down at 13,000 r.p.m for 5 minutes and the supernatant added to 60 μl of Dynabeads protein G (Invitrogen) previously equilibrated with IP buffer. Then the samples were incubated for 2 hours at 4 °C in a rotating wheel and washed 5 times with IP buffer using a magnetic separator rack. Finally “input” samples and IP samples were resuspended in de-crosslinking buffer (TE 1X, 1 % SDS, 10 μg/ml RNase A, 1 mg/ml proteinase K) and incubated at 65 °C overnight. Samples were purified using ChIP DNA Clean & Concentrator kit (Zymoresearch) according to the manufacturer instructions.Protein coimmunoprecipitationCells were grown to OD600 = 1 wash in cold water and resuspended in ice-cold buffer (50 mM HEPES, 150 mM KCl, 1.5 mM MgCl2, 0.5 mM DTT, and 0.5% Triton X-100 (pH 7.5) supplemented with Complete protease inhibitor cocktail tablets (from Roche)). Cells were broken in a FastPrep® FP120 (BIO101) by 3 repetitions of a 20 seconds cycle, power setting 5.5. Extracts were maintained on ice for 2 minutes after each cycle. To the cell extract 0.5mM CaCl2, 20 U DnaseI (NEB), 0.2 mg RNaseA (Qiagen) and 500 U Benzonase (Millipore) were added in order to degrade DNA and RNA from the sample. Cell extracts were incubated with protein G Dynabeads (Invitrogen) bound to anti-Myc antibody (Roche, 9E10) for 2 h at 4°C. Finally, beads were washed 5 time in washing buffer (10mM Tris-Cl pH 7.5, 150mM NaCl, 0.5 % Triton) and unbound from the antibody by incubating at °C for 4 minutes in SR Buffer (2% SDS, 0.125 M Tris-Cl pH 6.8). Immunoprecipitated proteins were mixed with SS buffer (5% sacarose, 0.0125 % Bromophenol blue) and run in SDS-PAGE gel.Digestion with MNaseMononucleosomal DNA was prepared as described in 18. For MNase digestions 160 ml of cells (OD600=1) where fixed with 1 % formaldehyde for 30 minutes at 25 °C with 100 r.p.m. agitation and quenched with 125 mM glycine for 10 minutes at 25 °C with 100 r.p.m. agitation. Cells were spun and washed twice with PBS. Pellet was resuspended in 20 ml sorbitol/ tris buffer (1M sorbitol, 50 mM Tris-HCl pH 7.5, 10 mM β-mercaptoethanol) containing 10 mg of Zymolyase 20T and incubated at 30 °C for 5 minutes or until spheroplast were formed. Spheroplasts were spun at 3,000 r.p.m. for 5 minutes and washed with sorbitol/tris buffer without β-mercaptoethanol. Spheroplasts were resuspended in NP buffer (1 M sorbitol, 50 mM NaCl, 10mM Tris pH 7.5, 5 mM MgCl2, 1mM CaCl2, 1mM β-mercaptoethanol, 500 μM spermidine, 0.075 % NP40) and digested with 600, 1200 or 1800 U/ ml of MNase (NEB) for 20 minutes at 37 °C. MNase digestion was stopped adding 200 μl of STOP solution (5% SDS, 50 mM EDTA). DNA was decrosslinked by adding 1 mg/ml proteinase K and incubating at 65 °C overnight. DNA was purified by adding 0.6M potassium acetate pH 5.5 and incubating on ice 5 minutes and centrifuging 10 minutes at 3,500 r.p.m followed by a phenol chloroform extraction. Finally DNA was precipitated by adding 25 μg/ml glycogen, 1/25 columes NaCl, 0.7 volumes Isopropanol, incubating 30 minutes at -20 °C and centrifuging 30 minutes at 13,200 r.p.m. The pellet was washed in 70 % ethanol and resuspended in TE 1X. Samples were run in parallel in a 1.5 % agarose gel and the mononucleosomes of a digestion with a ratio of 80:20 mononucleosomes to dinucleosomes were purified and used in Next Generation Sequencing.Next-generation sequencing and DANPOS analysisLibraries of mononucleosomal and ChIP DNA were constructed following the Illumina compatible NEXTflexTM ChIP-Seq kit protocol and sequenced in an Illumina NextSeq500 platform using the paired-end protocol. Reads were aligned to the S.cerevisiae (SacCer3) genome using Bowtie 46. Nucleosome occupancy maps were generated using the NUCwave algorithm 47. Nucleosome fuzziness was analyzed using the dpos utility of DANPOS2 application 20. To make it compatible with NUCwave maps, we used the following parameters: a span of 1 bp and a read extension of 50 bp. ChIP-seq data representation was performed using the Model-based Analysis of ChIP-Seq (MACS) 48 with a read extension of 147 bp.Calibrated ChIP-Seq analysisCalibrated Chip-seq analysis as done as in 25. The quality of the raw sequence data was assessed using FastQC (version 0.11.5). The reads were mapped to the S. cerevisiae genome (sacCer3) as the experimental genome or C. glabrata genome (CBS138) as calibration genome using Bowtie2 (version 2.3.4) with the default parameters. To generate an alignment in which all the sequences exclusively map to the sacCer3 genome, the whole reads were first mapped to CBS138 genome and the unmapped reads were retrieved as a separate Fastq file. Subsequently, these unmapped reads were re-aligned to sacCer3 genome and the generated aligned BAM file therefore contained reads that were unique to sacCer3. The same method was used to generate alignments unique to CBS138.Applying a recent published method 26 for calibrating ChIP-seq data and adding cells with the calibration genome (CBS138) to different samples of cells with the experimental genome (sacCer3), we calculated occupancy ratio (ORi) for experimental sample i from the number of reads assigned to calibration (WCi) and experimental (WXi) genomes among sequences derived from input DNA and the number of reads assigned to calibration (IPCi) and experimental (IPXi) genomes among sequences derived from ChIP-seq sample. Then ORi was calculated as: ORi=WciWxi×IPxiIPci For each sample, the alignment BAM file was converted to Wig format using bedtools (version 2.27.1). To obtain a calibrated version, the Wig file with the conventional ChIP-seq profile was multiplied by the occupancy ratio (OR) corresponding to each sample. Subsequently, these calibrated BigWig files were used to generate average profile plots for regions in +/- 20KB of centromeres for 16 chromosomes in sacCer3 genomeAll sequencing data that support the findings of this study have been deposited in GEO and are accessible through the following accession number: GSE118534.MNase-seq data analysisPaired-end Sequencing reads were aligned to Yeast genome sacCer3 using bowtie version 2.2.9 with default parameters. DANPOS2 was applied for two separate pair-wise comparisons at each nucleosome between G1 arrest and G2/M arrest, and G2/M arrest and G2/M +IAA. DANPOS output for pair-wise comparison contains a set of reference positions (with fixed length of 200bp) for nucleosomes defined by integrating all the nucleosomal positions called in each of the two conditions under study. The DANPOS-generated reference positions in two pair-wise comparisons were very similar but there was a small fraction of positions that did not overlap well in two comparisons. DANPOS also calculated Fuzziness scores for each condition and FDR and log10 p-value for each pair-wise comparison. We reported the nucleosomes with significant fuzziness change as reference nucleosomes in each comparison with FDR < 0.01 and p-value<-2, recommended in DANPOS manual. The fuzziness of these nucleosomes may be increased or decreased from one condition to the other. We also used DNAPOS-generated wig files for visualization of Fuzziness on IGV genome browser.Mcd1 PulldownsMCD1-HA and MCD1 untagged strains expressing CDC20 under the galactose promoter were grown at 25ºC in YEP medium containing 2% galactose to mid-logarithmic phase and arrested in metaphase by the addition of 2% glucose. 800 ODs of cells were harvested at 4ºC, washed once with ice-cold water and frozen at -80ºC. Cell pellets were mechanically broken in Lysis Buffer containing 150 mM KCl, 1mM MgCl2, 0.12mM Cacl2 50 mM Tris pH 7, 1 mM EDTA, 1% Triton 100-X, 5% glycerol and 1 mM PMSF supplemented with EDTA-free Complete Protease Inhibitor and Phosphatase Inhibitor Cocktail tablets (Roche). Protein extracts were treated with 600μg of RNase A (Qiagen), 900U of Benzonase (Millipore) and 600U of DnaseI (NEB) and immunoprecipitated using anti-HA Epitope Tag Protein Isolation kit (Miltenyi Biotec). After 1h incubation at 4ºC, anti-HA magnetic beads were washed with 750mM NaCl-Lysis Buffer and subjected to 2 consecutive rounds of elution in 50mM Tris-HCl pH6.8, 5mM EDTA, 1%SDS and 50mM DTT elution buffer.Protein identification and quantification by LC-MS/MSSample processingProtein samples eluted from anti-HA magnetic beads were buffer exchanged, reduced and alkylated using a FASP protocol. Briefly, samples were loaded onto 30 kDa centrifugal concentrators (Millipore, MRCF0R030) and buffer exchange was carried out by centrifugation on a bench top centrifuge (15min, 12,000g). Multiple buffer exchanges were performed sequentially with UA buffer (8M urea in 100mM Tris pH 8.5, 3x200μl), reduction with 10mM DTT in UA buffer (30min, 40°C) and alkylation with 50mM chloroacetamide in UA buffer (20min, 25°C). This was followed by buffer exchange into UA buffer (3x100μl) and 50mM ammonium bicarbonate (3x100μl). Digestion was carried out with mass spectrometry grade trypsin (Promega, V5280) using 1μg protease per digest (16h, 37°C). Tryptic peptides were collected by centrifugation into a fresh collection tube (10min, 12,000g) and washing of the concentrator with 0.5M sodium chloride (50μl, 10min, 12,000g) for maximal recovery. Following acidification with 1% trifluoroacetic acid (TFA) to a final concentration of 0.2%, collected protein digests were desalted using Glygen C18 spin tips (Glygen Corp, TT2C18.96) and peptides eluted with 60% acetonitrile, 0.1% formic acid (FA). Eluents were then dried using vacuum centrifugation.Liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysisDried tryptic digests were redissolved in 0.1% TFA by shaking (1200rpm) for 30min and sonication on an ultrasonic water bath for 10min, followed by centrifugation (20,000g, 5°C) for 10min. LC-MS/MS analysis was carried out in technical duplicates and separation was performed using an Ultimate 3000 RSLC nano liquid chromatography system (Thermo Scientific) coupled to a coupled to a Q-Exactive mass spectrometer (Thermo Scientific) via an EASY spray source (Thermo Scientific). For LC-MS/MS analysis protein digest solutions were injected and loaded onto a trap column (Acclaim PepMap 100 C18, 100μm × 2cm) for desalting and concentration at 8μL/min in 2% acetonitrile, 0.1% TFA. Peptides were then eluted on-line to an analytical column (Acclaim Pepmap RSLC C18, 75μm × 50cm) at a flow rate of 250nL/min. Peptides were separated using a 120 minute gradient, 4-25% of buffer B for 90 minutes followed by 25-45% buffer B for another 30 minutes (composition of buffer B – 80% acetonitrile, 0.1% FA) and subsequent column conditioning and equilibration. Eluted peptides were analysed by the mass spectrometer operating in positive polarity using a data-dependent acquisition mode. Ions for fragmentation were determined from an initial MS1 survey scan at 70,000 resolution, followed by HCD (Higher Energy Collision Induced Dissociation) of the top 12 most abundant ions at 17,500 resolution. MS1 and MS2 scan AGC targets were set to 3e6 and 5e4 for maximum injection times of 50ms and 50ms respectively. A survey scan m/z range of 400 – 1800 was used, normalised collision energy set to 27%, charge exclusion enabled with unassigned and +1 charge states rejected and a minimal AGC target of 1e3.Raw data processingData was processed using the MaxQuant software platform (v1.5.8.3), with database searches carried out by the in-built Andromeda search engine against the Uniprot S.cerevisiae database (version 20160815, number of entries: 6,729). A reverse decoy search approach was used at a 1% false discovery rate (FDR) for both peptide spectrum matches and protein groups. Search parameters included: maximum missed cleavages set to 2, fixed modification of cysteine carbamidomethylation and variable modifications of methionine oxidation, protein N-terminal acetylation and serine, threonine, tyrosine phosphorylation. Label-free quantification was enabled with an LFQ minimum ratio count of 2. ‘Match between runs’ function was used with match and alignment time limits of 1 and 20 minutes respectively.

Media, culture conditions and synchronisationsS. cerevisiae strains used in this study are shown in Supplementary Table 1. To arrest the cells in G1, α-factor was added to exponentially growing MATa cultures (OD600=0.5) to a final concentration of 3 x 10-8 M. To arrest cells in G2/M, Nocodazole (1.5 mg/ml stock in DMSO 100%) was added to exponentially growing cultures (OD600=0.5) to a final concentration of 0.015 mg/ml. When arrests in Nocodazole are maintained for more than 2 hours cells are spun and resuspended in YPD containing fresh nocodazole (0.015 mg/ml). Cultures were monitored by microscopy until ≥90% of cells were arrested, typically 2 hours at 25 °C. To release cells from G1, the culture was spun (4,000 r.p.m, 1 min) and washed in YPD 3 times. The pellet was then resuspended in YPD containing 0.1 mg/ml pronase. To release cells from Nocodazole, the culture was spun (4,000 r.p.m, 1 min) and washed in YPD containing 1% DMSO 5 times. The pellet was then resuspended in YPD. To degrade proteins tagged with AID* epitope, a stock of IAA of 0.6M in ethanol 100% was used for a final concentration of 6mM. To inhibit transcription a stock of thiolutin at 1mg/ml in DMSO was added for 30 minutes to the culture at a final concentration of 10ug/ml.

Media, culture conditions and synchronisations

S. cerevisiae strains used in this study are shown in Supplementary Table 1. To arrest the cells in G1, α-factor was added to exponentially growing MATa cultures (OD600=0.5) to a final concentration of 3 x 10-8 M. To arrest cells in G2/M, Nocodazole (1.5 mg/ml stock in DMSO 100%) was added to exponentially growing cultures (OD600=0.5) to a final concentration of 0.015 mg/ml. When arrests in Nocodazole are maintained for more than 2 hours cells are spun and resuspended in YPD containing fresh nocodazole (0.015 mg/ml). Cultures were monitored by microscopy until ≥90% of cells were arrested, typically 2 hours at 25 °C. To release cells from G1, the culture was spun (4,000 r.p.m, 1 min) and washed in YPD 3 times. The pellet was then resuspended in YPD containing 0.1 mg/ml pronase. To release cells from Nocodazole, the culture was spun (4,000 r.p.m, 1 min) and washed in YPD containing 1% DMSO 5 times. The pellet was then resuspended in YPD. To degrade proteins tagged with AID* epitope, a stock of IAA of 0.6M in ethanol 100% was used for a final concentration of 6mM. To inhibit transcription a stock of thiolutin at 1mg/ml in DMSO was added for 30 minutes to the culture at a final concentration of 10ug/ml.

MicroscopyA Leica IRB epifluorescence microscope with a Hamamatsu 742-95 digital camera, a 63X/1.4 lens and OpenLab software (Improvision) was used for single cell visualization. 1 ml of cell culture was collected from each time point and mixed with glycerol (20% final concentration) to preserve TetO/TetR signal. Cells were then frozen at -80 °C. For visualization, cells where centrifuged at 3,000 r.p.m. for 2 minute and 1μl of the pellet was mixed with 1μl of DAPI solution (DAPI 4 μg/ml Triton 1 %) on the microscope slide. For each field 20 z-focal planes images were captured (0.3 μm depth between each consecutive image). At least 100 cells were score for GFP dots.

A Leica IRB epifluorescence microscope with a Hamamatsu 742-95 digital camera, a 63X/1.4 lens and OpenLab software (Improvision) was used for single cell visualization. 1 ml of cell culture was collected from each time point and mixed with glycerol (20% final concentration) to preserve TetO/TetR signal. Cells were then frozen at -80 °C. For visualization, cells where centrifuged at 3,000 r.p.m. for 2 minute and 1μl of the pellet was mixed with 1μl of DAPI solution (DAPI 4 μg/ml Triton 1 %) on the microscope slide. For each field 20 z-focal planes images were captured (0.3 μm depth between each consecutive image). At least 100 cells were score for GFP dots.

Hi-C librariesHi-C experiments were performed according to a protocol adapted from 44. Aliquots of 1-3 x 109 cells in 150 ml culture medium were fixed in 3% formaldehyde (Sigma-Aldrich, Ref. F8775) and quenched in 0.4 M glycine for 20 min at room temperature and 4°C, respectively. Cells were harvested, washed with culture medium and pellets were stored at -80°C. Pellets were thawed on ice and resuspended in chilled TE buffer supplemented with protease inhibitors. Cells were lysed using Precellys VK05 Lysing KIT (Ozyme, Ref. 0042) at 6,700 rpm. Lysates were treated with 0.5 % SDS for 20 min at 65°C and digested overnight with DpnII (CFinal=500 U/pellet; NEB R0543M) at 37°C. Digestions were centrifuged for 20 min at 16,000 g and the pellets suspended in cold water. 5’ DpnII overhangs were filled-in and labelled with biotin using 30 μM dNTP (dATP, dGTP, dTTP and Biotin-14-dCTP Invitrogen Ref. 19518018). Biotinylated DNA fragments were ligated using T4 DNA ligase (CFinal=250 Weiss U/pellet; Thermo Scientific 10621441) for 4 h at 16°C. Cross-link was reverted through an overnight treatment in presence of 250 μg/ml proteinase K at 65°C. Total DNA was extracted using phenol/chloroform, precipitated and treated with RNAse A 500 μg/ml. Non-ligated DNA fragments were removed by T4 DNA polymerase (CFinal=5 U/pellet; NEB M0203L) treatment; whereas ligated fragments were 500 bp sheared using Covaris S220 and pulled-down using Dynabeads Myone Streptavidin C1 (Invitrogen 65001). Resulting libraries were amplified using custom-made primers for paired-end deep-sequencing on NextSeq500 Illumina platform (2x75 bp cycles).

Hi-C libraries

Hi-C experiments were performed according to a protocol adapted from 44. Aliquots of 1-3 x 109 cells in 150 ml culture medium were fixed in 3% formaldehyde (Sigma-Aldrich, Ref. F8775) and quenched in 0.4 M glycine for 20 min at room temperature and 4°C, respectively. Cells were harvested, washed with culture medium and pellets were stored at -80°C. Pellets were thawed on ice and resuspended in chilled TE buffer supplemented with protease inhibitors. Cells were lysed using Precellys VK05 Lysing KIT (Ozyme, Ref. 0042) at 6,700 rpm. Lysates were treated with 0.5 % SDS for 20 min at 65°C and digested overnight with DpnII (CFinal=500 U/pellet; NEB R0543M) at 37°C. Digestions were centrifuged for 20 min at 16,000 g and the pellets suspended in cold water. 5’ DpnII overhangs were filled-in and labelled with biotin using 30 μM dNTP (dATP, dGTP, dTTP and Biotin-14-dCTP Invitrogen Ref. 19518018). Biotinylated DNA fragments were ligated using T4 DNA ligase (CFinal=250 Weiss U/pellet; Thermo Scientific 10621441) for 4 h at 16°C. Cross-link was reverted through an overnight treatment in presence of 250 μg/ml proteinase K at 65°C. Total DNA was extracted using phenol/chloroform, precipitated and treated with RNAse A 500 μg/ml. Non-ligated DNA fragments were removed by T4 DNA polymerase (CFinal=5 U/pellet; NEB M0203L) treatment; whereas ligated fragments were 500 bp sheared using Covaris S220 and pulled-down using Dynabeads Myone Streptavidin C1 (Invitrogen 65001). Resulting libraries were amplified using custom-made primers for paired-end deep-sequencing on NextSeq500 Illumina platform (2x75 bp cycles).

Generation and normalization of contact mapsThe reads from each Hi-C library were processed as follows. First, PCR duplicates were removed using the custom-made tags present on the primers and the resulting reads were mapped independently using Bowtie 2 (mode: --very-sensitive --rdg 500,3 --rfg 500,3) against the reference genome of S. cerevisiae (S288C) and indexed on DpnII restriction fragments. An iterative alignment procedure (20 bp window) was used to maximize the yield of uniquely mapped reads (mapping quality > 30). Reads were classified as either valid Hi-C products or unwanted events to be filtered out (i.e. loops, non-digested fragments, etc.; for details see {Cournac, 2012 #40}) based on their assignment and orientation on the DpnII restriction fragments in the reference genome. Valid Hi-C reads were used to generate contact frequency maps. In these maps, each vector consists in a fixed size genomic bin made of successive RF (either 5, 25, or 50 kb). Bins with a high variance in contact frequency (< 1.5 Standard Deviation or 1.5 - 2 S.D.) were filtered out. Filtered contact maps were normalized as described 45. Approximately 15 million of valid reads were used to build each contact map. Similarity between pairs of normalized contact maps was assessed by computing the log2 ratio for each point.

Generation and normalization of contact maps

The reads from each Hi-C library were processed as follows. First, PCR duplicates were removed using the custom-made tags present on the primers and the resulting reads were mapped independently using Bowtie 2 (mode: --very-sensitive --rdg 500,3 --rfg 500,3) against the reference genome of S. cerevisiae (S288C) and indexed on DpnII restriction fragments. An iterative alignment procedure (20 bp window) was used to maximize the yield of uniquely mapped reads (mapping quality > 30). Reads were classified as either valid Hi-C products or unwanted events to be filtered out (i.e. loops, non-digested fragments, etc.; for details see {Cournac, 2012 #40}) based on their assignment and orientation on the DpnII restriction fragments in the reference genome. Valid Hi-C reads were used to generate contact frequency maps. In these maps, each vector consists in a fixed size genomic bin made of successive RF (either 5, 25, or 50 kb). Bins with a high variance in contact frequency (< 1.5 Standard Deviation or 1.5 - 2 S.D.) were filtered out. Filtered contact maps were normalized as described 45. Approximately 15 million of valid reads were used to build each contact map. Similarity between pairs of normalized contact maps was assessed by computing the log2 ratio for each point.

Contact probability as a function of the genomic distance p(s)The contact probability p(s) as a function of the genomic distance s separating two DNA positions in cis was computed as follow. First, self-circularization events were discarded by removing pairs of read oriented in opposite directions along the reference genome and separated by less than 1.5 kb. The remaining reads were log-binned as a function of their genomic distance (in kb) on each chromosomal arm: bin = [log1.1(s)]. The p(s) curve is the histogram computed on the sum of read pairs for each bin. This sum is weighted by both bin-size 1.1(1+bin) (log binning), as well as by the length of the chromosomes.

Contact probability as a function of the genomic distance p(s)

The contact probability p(s) as a function of the genomic distance s separating two DNA positions in cis was computed as follow. First, self-circularization events were discarded by removing pairs of read oriented in opposite directions along the reference genome and separated by less than 1.5 kb. The remaining reads were log-binned as a function of their genomic distance (in kb) on each chromosomal arm: bin = [log1.1(s)]. The p(s) curve is the histogram computed on the sum of read pairs for each bin. This sum is weighted by both bin-size 1.1(1+bin) (log binning), as well as by the length of the chromosomes.

Chromatin immunoprecipitationFor ChIP analysis cells were grown to OD600 = 0.5 and arrested at the required cell cycle stage. 87 OD600 units of S. cerevisiae were mixed with 43 OD600 units of asynchronous C. glabrata. Cells were fixed for 15 minutes at 25 °C, and quenched with glycine (final concentration 125 mM) for 7 minutes before cells were harvested by centrifugation at 4,000 r.p.m. for 1 minute. The cell pellets were washed in PBS and transferred to a screw cap tube and frozen on dry ice. The pellets were stored at -80 °C. Pellets were resuspended in 300 μl of IP buffer (150 mM NaCl, 50 mM Tris-HCl (pH 7.5), 5 mM EDTA, NP-40 (0.05% v/v), Triton® X-100 (1% v/v)) containing phenylmethanesulfonyl fluoride (PMSF, final concentration 1 mM) and Complete protease inhibitor cocktail (without EDTA, from Roche). 500 μl of glass beads were added to the tubes. Cells were broken in a FastPrep® FP120 (BIO101) by 3 repetitions of a 20 seconds cycle, power setting 5.5. Cells were maintained on ice for 2 minutes after each cycle. The tubes were pierced with a hot needle and placed into new eppendorf and spun at 2000 r.p.m. for 2 minutes to collect the lysate. 100 μl of IP buffer containing PMSF and protease inhibitors were then added and tubes spun again at 2000 r.p.m. for 1 minute. The cell lysate was spun down for 10 minutes at 15,000 r.p.m at 4 °C. This pellet was resuspended in 1ml of IP buffer containing PMSF and protease inhibitors, and sonicated for 1 hour (30 seconds on, 30 seconds off) at high power at 4 °C in a Diagenode Bioruptor. After sonication samples were spun down for 10 minutes at 15,000 r.p.m. and the supernatant taken. 200 μl of the sonicated chromatin were taken as “input” and 400 μl was incubated with 40 μg of HA antibody (anti-HA 12CA5 from Roche) in a sonicator at low power for 30 minutes (30 seconds on, 30 seconds off). The “input” DNA was precipitated with 0.3 M sodium acetate and 2.5 volumes of cold ethanol and spun down at 15,000 r.p.m. for 30 minutes, and the supernatant removed. The pellet was washed with 70 % ethanol and air dried. After antibody binding the IP sample was spun down at 13,000 r.p.m for 5 minutes and the supernatant added to 60 μl of Dynabeads protein G (Invitrogen) previously equilibrated with IP buffer. Then the samples were incubated for 2 hours at 4 °C in a rotating wheel and washed 5 times with IP buffer using a magnetic separator rack. Finally “input” samples and IP samples were resuspended in de-crosslinking buffer (TE 1X, 1 % SDS, 10 μg/ml RNase A, 1 mg/ml proteinase K) and incubated at 65 °C overnight. Samples were purified using ChIP DNA Clean & Concentrator kit (Zymoresearch) according to the manufacturer instructions.

Chromatin immunoprecipitation

For ChIP analysis cells were grown to OD600 = 0.5 and arrested at the required cell cycle stage. 87 OD600 units of S. cerevisiae were mixed with 43 OD600 units of asynchronous C. glabrata. Cells were fixed for 15 minutes at 25 °C, and quenched with glycine (final concentration 125 mM) for 7 minutes before cells were harvested by centrifugation at 4,000 r.p.m. for 1 minute. The cell pellets were washed in PBS and transferred to a screw cap tube and frozen on dry ice. The pellets were stored at -80 °C. Pellets were resuspended in 300 μl of IP buffer (150 mM NaCl, 50 mM Tris-HCl (pH 7.5), 5 mM EDTA, NP-40 (0.05% v/v), Triton® X-100 (1% v/v)) containing phenylmethanesulfonyl fluoride (PMSF, final concentration 1 mM) and Complete protease inhibitor cocktail (without EDTA, from Roche). 500 μl of glass beads were added to the tubes. Cells were broken in a FastPrep® FP120 (BIO101) by 3 repetitions of a 20 seconds cycle, power setting 5.5. Cells were maintained on ice for 2 minutes after each cycle. The tubes were pierced with a hot needle and placed into new eppendorf and spun at 2000 r.p.m. for 2 minutes to collect the lysate. 100 μl of IP buffer containing PMSF and protease inhibitors were then added and tubes spun again at 2000 r.p.m. for 1 minute. The cell lysate was spun down for 10 minutes at 15,000 r.p.m at 4 °C. This pellet was resuspended in 1ml of IP buffer containing PMSF and protease inhibitors, and sonicated for 1 hour (30 seconds on, 30 seconds off) at high power at 4 °C in a Diagenode Bioruptor. After sonication samples were spun down for 10 minutes at 15,000 r.p.m. and the supernatant taken. 200 μl of the sonicated chromatin were taken as “input” and 400 μl was incubated with 40 μg of HA antibody (anti-HA 12CA5 from Roche) in a sonicator at low power for 30 minutes (30 seconds on, 30 seconds off). The “input” DNA was precipitated with 0.3 M sodium acetate and 2.5 volumes of cold ethanol and spun down at 15,000 r.p.m. for 30 minutes, and the supernatant removed. The pellet was washed with 70 % ethanol and air dried. After antibody binding the IP sample was spun down at 13,000 r.p.m for 5 minutes and the supernatant added to 60 μl of Dynabeads protein G (Invitrogen) previously equilibrated with IP buffer. Then the samples were incubated for 2 hours at 4 °C in a rotating wheel and washed 5 times with IP buffer using a magnetic separator rack. Finally “input” samples and IP samples were resuspended in de-crosslinking buffer (TE 1X, 1 % SDS, 10 μg/ml RNase A, 1 mg/ml proteinase K) and incubated at 65 °C overnight. Samples were purified using ChIP DNA Clean & Concentrator kit (Zymoresearch) according to the manufacturer instructions.

Protein coimmunoprecipitationCells were grown to OD600 = 1 wash in cold water and resuspended in ice-cold buffer (50 mM HEPES, 150 mM KCl, 1.5 mM MgCl2, 0.5 mM DTT, and 0.5% Triton X-100 (pH 7.5) supplemented with Complete protease inhibitor cocktail tablets (from Roche)). Cells were broken in a FastPrep® FP120 (BIO101) by 3 repetitions of a 20 seconds cycle, power setting 5.5. Extracts were maintained on ice for 2 minutes after each cycle. To the cell extract 0.5mM CaCl2, 20 U DnaseI (NEB), 0.2 mg RNaseA (Qiagen) and 500 U Benzonase (Millipore) were added in order to degrade DNA and RNA from the sample. Cell extracts were incubated with protein G Dynabeads (Invitrogen) bound to anti-Myc antibody (Roche, 9E10) for 2 h at 4°C. Finally, beads were washed 5 time in washing buffer (10mM Tris-Cl pH 7.5, 150mM NaCl, 0.5 % Triton) and unbound from the antibody by incubating at °C for 4 minutes in SR Buffer (2% SDS, 0.125 M Tris-Cl pH 6.8). Immunoprecipitated proteins were mixed with SS buffer (5% sacarose, 0.0125 % Bromophenol blue) and run in SDS-PAGE gel.

Protein coimmunoprecipitation

Cells were grown to OD600 = 1 wash in cold water and resuspended in ice-cold buffer (50 mM HEPES, 150 mM KCl, 1.5 mM MgCl2, 0.5 mM DTT, and 0.5% Triton X-100 (pH 7.5) supplemented with Complete protease inhibitor cocktail tablets (from Roche)). Cells were broken in a FastPrep® FP120 (BIO101) by 3 repetitions of a 20 seconds cycle, power setting 5.5. Extracts were maintained on ice for 2 minutes after each cycle. To the cell extract 0.5mM CaCl2, 20 U DnaseI (NEB), 0.2 mg RNaseA (Qiagen) and 500 U Benzonase (Millipore) were added in order to degrade DNA and RNA from the sample. Cell extracts were incubated with protein G Dynabeads (Invitrogen) bound to anti-Myc antibody (Roche, 9E10) for 2 h at 4°C. Finally, beads were washed 5 time in washing buffer (10mM Tris-Cl pH 7.5, 150mM NaCl, 0.5 % Triton) and unbound from the antibody by incubating at °C for 4 minutes in SR Buffer (2% SDS, 0.125 M Tris-Cl pH 6.8). Immunoprecipitated proteins were mixed with SS buffer (5% sacarose, 0.0125 % Bromophenol blue) and run in SDS-PAGE gel.

Digestion with MNaseMononucleosomal DNA was prepared as described in 18. For MNase digestions 160 ml of cells (OD600=1) where fixed with 1 % formaldehyde for 30 minutes at 25 °C with 100 r.p.m. agitation and quenched with 125 mM glycine for 10 minutes at 25 °C with 100 r.p.m. agitation. Cells were spun and washed twice with PBS. Pellet was resuspended in 20 ml sorbitol/ tris buffer (1M sorbitol, 50 mM Tris-HCl pH 7.5, 10 mM β-mercaptoethanol) containing 10 mg of Zymolyase 20T and incubated at 30 °C for 5 minutes or until spheroplast were formed. Spheroplasts were spun at 3,000 r.p.m. for 5 minutes and washed with sorbitol/tris buffer without β-mercaptoethanol. Spheroplasts were resuspended in NP buffer (1 M sorbitol, 50 mM NaCl, 10mM Tris pH 7.5, 5 mM MgCl2, 1mM CaCl2, 1mM β-mercaptoethanol, 500 μM spermidine, 0.075 % NP40) and digested with 600, 1200 or 1800 U/ ml of MNase (NEB) for 20 minutes at 37 °C. MNase digestion was stopped adding 200 μl of STOP solution (5% SDS, 50 mM EDTA). DNA was decrosslinked by adding 1 mg/ml proteinase K and incubating at 65 °C overnight. DNA was purified by adding 0.6M potassium acetate pH 5.5 and incubating on ice 5 minutes and centrifuging 10 minutes at 3,500 r.p.m followed by a phenol chloroform extraction. Finally DNA was precipitated by adding 25 μg/ml glycogen, 1/25 columes NaCl, 0.7 volumes Isopropanol, incubating 30 minutes at -20 °C and centrifuging 30 minutes at 13,200 r.p.m. The pellet was washed in 70 % ethanol and resuspended in TE 1X. Samples were run in parallel in a 1.5 % agarose gel and the mononucleosomes of a digestion with a ratio of 80:20 mononucleosomes to dinucleosomes were purified and used in Next Generation Sequencing.

Digestion with MNase

Mononucleosomal DNA was prepared as described in 18. For MNase digestions 160 ml of cells (OD600=1) where fixed with 1 % formaldehyde for 30 minutes at 25 °C with 100 r.p.m. agitation and quenched with 125 mM glycine for 10 minutes at 25 °C with 100 r.p.m. agitation. Cells were spun and washed twice with PBS. Pellet was resuspended in 20 ml sorbitol/ tris buffer (1M sorbitol, 50 mM Tris-HCl pH 7.5, 10 mM β-mercaptoethanol) containing 10 mg of Zymolyase 20T and incubated at 30 °C for 5 minutes or until spheroplast were formed. Spheroplasts were spun at 3,000 r.p.m. for 5 minutes and washed with sorbitol/tris buffer without β-mercaptoethanol. Spheroplasts were resuspended in NP buffer (1 M sorbitol, 50 mM NaCl, 10mM Tris pH 7.5, 5 mM MgCl2, 1mM CaCl2, 1mM β-mercaptoethanol, 500 μM spermidine, 0.075 % NP40) and digested with 600, 1200 or 1800 U/ ml of MNase (NEB) for 20 minutes at 37 °C. MNase digestion was stopped adding 200 μl of STOP solution (5% SDS, 50 mM EDTA). DNA was decrosslinked by adding 1 mg/ml proteinase K and incubating at 65 °C overnight. DNA was purified by adding 0.6M potassium acetate pH 5.5 and incubating on ice 5 minutes and centrifuging 10 minutes at 3,500 r.p.m followed by a phenol chloroform extraction. Finally DNA was precipitated by adding 25 μg/ml glycogen, 1/25 columes NaCl, 0.7 volumes Isopropanol, incubating 30 minutes at -20 °C and centrifuging 30 minutes at 13,200 r.p.m. The pellet was washed in 70 % ethanol and resuspended in TE 1X. Samples were run in parallel in a 1.5 % agarose gel and the mononucleosomes of a digestion with a ratio of 80:20 mononucleosomes to dinucleosomes were purified and used in Next Generation Sequencing.

Next-generation sequencing and DANPOS analysisLibraries of mononucleosomal and ChIP DNA were constructed following the Illumina compatible NEXTflexTM ChIP-Seq kit protocol and sequenced in an Illumina NextSeq500 platform using the paired-end protocol. Reads were aligned to the S.cerevisiae (SacCer3) genome using Bowtie 46. Nucleosome occupancy maps were generated using the NUCwave algorithm 47. Nucleosome fuzziness was analyzed using the dpos utility of DANPOS2 application 20. To make it compatible with NUCwave maps, we used the following parameters: a span of 1 bp and a read extension of 50 bp. ChIP-seq data representation was performed using the Model-based Analysis of ChIP-Seq (MACS) 48 with a read extension of 147 bp.

Next-generation sequencing and DANPOS analysis

Libraries of mononucleosomal and ChIP DNA were constructed following the Illumina compatible NEXTflexTM ChIP-Seq kit protocol and sequenced in an Illumina NextSeq500 platform using the paired-end protocol. Reads were aligned to the S.cerevisiae (SacCer3) genome using Bowtie 46. Nucleosome occupancy maps were generated using the NUCwave algorithm 47. Nucleosome fuzziness was analyzed using the dpos utility of DANPOS2 application 20. To make it compatible with NUCwave maps, we used the following parameters: a span of 1 bp and a read extension of 50 bp. ChIP-seq data representation was performed using the Model-based Analysis of ChIP-Seq (MACS) 48 with a read extension of 147 bp.

Calibrated ChIP-Seq analysisCalibrated Chip-seq analysis as done as in 25. The quality of the raw sequence data was assessed using FastQC (version 0.11.5). The reads were mapped to the S. cerevisiae genome (sacCer3) as the experimental genome or C. glabrata genome (CBS138) as calibration genome using Bowtie2 (version 2.3.4) with the default parameters. To generate an alignment in which all the sequences exclusively map to the sacCer3 genome, the whole reads were first mapped to CBS138 genome and the unmapped reads were retrieved as a separate Fastq file. Subsequently, these unmapped reads were re-aligned to sacCer3 genome and the generated aligned BAM file therefore contained reads that were unique to sacCer3. The same method was used to generate alignments unique to CBS138.Applying a recent published method 26 for calibrating ChIP-seq data and adding cells with the calibration genome (CBS138) to different samples of cells with the experimental genome (sacCer3), we calculated occupancy ratio (ORi) for experimental sample i from the number of reads assigned to calibration (WCi) and experimental (WXi) genomes among sequences derived from input DNA and the number of reads assigned to calibration (IPCi) and experimental (IPXi) genomes among sequences derived from ChIP-seq sample. Then ORi was calculated as: ORi=WciWxi×IPxiIPci For each sample, the alignment BAM file was converted to Wig format using bedtools (version 2.27.1). To obtain a calibrated version, the Wig file with the conventional ChIP-seq profile was multiplied by the occupancy ratio (OR) corresponding to each sample. Subsequently, these calibrated BigWig files were used to generate average profile plots for regions in +/- 20KB of centromeres for 16 chromosomes in sacCer3 genomeAll sequencing data that support the findings of this study have been deposited in GEO and are accessible through the following accession number: GSE118534.

Calibrated ChIP-Seq analysis

Calibrated Chip-seq analysis as done as in 25. The quality of the raw sequence data was assessed using FastQC (version 0.11.5). The reads were mapped to the S. cerevisiae genome (sacCer3) as the experimental genome or C. glabrata genome (CBS138) as calibration genome using Bowtie2 (version 2.3.4) with the default parameters. To generate an alignment in which all the sequences exclusively map to the sacCer3 genome, the whole reads were first mapped to CBS138 genome and the unmapped reads were retrieved as a separate Fastq file. Subsequently, these unmapped reads were re-aligned to sacCer3 genome and the generated aligned BAM file therefore contained reads that were unique to sacCer3. The same method was used to generate alignments unique to CBS138.

Applying a recent published method 26 for calibrating ChIP-seq data and adding cells with the calibration genome (CBS138) to different samples of cells with the experimental genome (sacCer3), we calculated occupancy ratio (ORi) for experimental sample i from the number of reads assigned to calibration (WCi) and experimental (WXi) genomes among sequences derived from input DNA and the number of reads assigned to calibration (IPCi) and experimental (IPXi) genomes among sequences derived from ChIP-seq sample. Then ORi was calculated as: ORi=WciWxi×IPxiIPci For each sample, the alignment BAM file was converted to Wig format using bedtools (version 2.27.1). To obtain a calibrated version, the Wig file with the conventional ChIP-seq profile was multiplied by the occupancy ratio (OR) corresponding to each sample. Subsequently, these calibrated BigWig files were used to generate average profile plots for regions in +/- 20KB of centromeres for 16 chromosomes in sacCer3 genome

All sequencing data that support the findings of this study have been deposited in GEO and are accessible through the following accession number: GSE118534.

MNase-seq data analysisPaired-end Sequencing reads were aligned to Yeast genome sacCer3 using bowtie version 2.2.9 with default parameters. DANPOS2 was applied for two separate pair-wise comparisons at each nucleosome between G1 arrest and G2/M arrest, and G2/M arrest and G2/M +IAA. DANPOS output for pair-wise comparison contains a set of reference positions (with fixed length of 200bp) for nucleosomes defined by integrating all the nucleosomal positions called in each of the two conditions under study. The DANPOS-generated reference positions in two pair-wise comparisons were very similar but there was a small fraction of positions that did not overlap well in two comparisons. DANPOS also calculated Fuzziness scores for each condition and FDR and log10 p-value for each pair-wise comparison. We reported the nucleosomes with significant fuzziness change as reference nucleosomes in each comparison with FDR < 0.01 and p-value<-2, recommended in DANPOS manual. The fuzziness of these nucleosomes may be increased or decreased from one condition to the other. We also used DNAPOS-generated wig files for visualization of Fuzziness on IGV genome browser.

MNase-seq data analysis

Paired-end Sequencing reads were aligned to Yeast genome sacCer3 using bowtie version 2.2.9 with default parameters. DANPOS2 was applied for two separate pair-wise comparisons at each nucleosome between G1 arrest and G2/M arrest, and G2/M arrest and G2/M +IAA. DANPOS output for pair-wise comparison contains a set of reference positions (with fixed length of 200bp) for nucleosomes defined by integrating all the nucleosomal positions called in each of the two conditions under study. The DANPOS-generated reference positions in two pair-wise comparisons were very similar but there was a small fraction of positions that did not overlap well in two comparisons. DANPOS also calculated Fuzziness scores for each condition and FDR and log10 p-value for each pair-wise comparison. We reported the nucleosomes with significant fuzziness change as reference nucleosomes in each comparison with FDR < 0.01 and p-value<-2, recommended in DANPOS manual. The fuzziness of these nucleosomes may be increased or decreased from one condition to the other. We also used DNAPOS-generated wig files for visualization of Fuzziness on IGV genome browser.

Mcd1 PulldownsMCD1-HA and MCD1 untagged strains expressing CDC20 under the galactose promoter were grown at 25ºC in YEP medium containing 2% galactose to mid-logarithmic phase and arrested in metaphase by the addition of 2% glucose. 800 ODs of cells were harvested at 4ºC, washed once with ice-cold water and frozen at -80ºC. Cell pellets were mechanically broken in Lysis Buffer containing 150 mM KCl, 1mM MgCl2, 0.12mM Cacl2 50 mM Tris pH 7, 1 mM EDTA, 1% Triton 100-X, 5% glycerol and 1 mM PMSF supplemented with EDTA-free Complete Protease Inhibitor and Phosphatase Inhibitor Cocktail tablets (Roche). Protein extracts were treated with 600μg of RNase A (Qiagen), 900U of Benzonase (Millipore) and 600U of DnaseI (NEB) and immunoprecipitated using anti-HA Epitope Tag Protein Isolation kit (Miltenyi Biotec). After 1h incubation at 4ºC, anti-HA magnetic beads were washed with 750mM NaCl-Lysis Buffer and subjected to 2 consecutive rounds of elution in 50mM Tris-HCl pH6.8, 5mM EDTA, 1%SDS and 50mM DTT elution buffer.

Mcd1 Pulldowns

MCD1-HA and MCD1 untagged strains expressing CDC20 under the galactose promoter were grown at 25ºC in YEP medium containing 2% galactose to mid-logarithmic phase and arrested in metaphase by the addition of 2% glucose. 800 ODs of cells were harvested at 4ºC, washed once with ice-cold water and frozen at -80ºC. Cell pellets were mechanically broken in Lysis Buffer containing 150 mM KCl, 1mM MgCl2, 0.12mM Cacl2 50 mM Tris pH 7, 1 mM EDTA, 1% Triton 100-X, 5% glycerol and 1 mM PMSF supplemented with EDTA-free Complete Protease Inhibitor and Phosphatase Inhibitor Cocktail tablets (Roche). Protein extracts were treated with 600μg of RNase A (Qiagen), 900U of Benzonase (Millipore) and 600U of DnaseI (NEB) and immunoprecipitated using anti-HA Epitope Tag Protein Isolation kit (Miltenyi Biotec). After 1h incubation at 4ºC, anti-HA magnetic beads were washed with 750mM NaCl-Lysis Buffer and subjected to 2 consecutive rounds of elution in 50mM Tris-HCl pH6.8, 5mM EDTA, 1%SDS and 50mM DTT elution buffer.

Protein identification and quantification by LC-MS/MSSample processingProtein samples eluted from anti-HA magnetic beads were buffer exchanged, reduced and alkylated using a FASP protocol. Briefly, samples were loaded onto 30 kDa centrifugal concentrators (Millipore, MRCF0R030) and buffer exchange was carried out by centrifugation on a bench top centrifuge (15min, 12,000g). Multiple buffer exchanges were performed sequentially with UA buffer (8M urea in 100mM Tris pH 8.5, 3x200μl), reduction with 10mM DTT in UA buffer (30min, 40°C) and alkylation with 50mM chloroacetamide in UA buffer (20min, 25°C). This was followed by buffer exchange into UA buffer (3x100μl) and 50mM ammonium bicarbonate (3x100μl). Digestion was carried out with mass spectrometry grade trypsin (Promega, V5280) using 1μg protease per digest (16h, 37°C). Tryptic peptides were collected by centrifugation into a fresh collection tube (10min, 12,000g) and washing of the concentrator with 0.5M sodium chloride (50μl, 10min, 12,000g) for maximal recovery. Following acidification with 1% trifluoroacetic acid (TFA) to a final concentration of 0.2%, collected protein digests were desalted using Glygen C18 spin tips (Glygen Corp, TT2C18.96) and peptides eluted with 60% acetonitrile, 0.1% formic acid (FA). Eluents were then dried using vacuum centrifugation.Liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysisDried tryptic digests were redissolved in 0.1% TFA by shaking (1200rpm) for 30min and sonication on an ultrasonic water bath for 10min, followed by centrifugation (20,000g, 5°C) for 10min. LC-MS/MS analysis was carried out in technical duplicates and separation was performed using an Ultimate 3000 RSLC nano liquid chromatography system (Thermo Scientific) coupled to a coupled to a Q-Exactive mass spectrometer (Thermo Scientific) via an EASY spray source (Thermo Scientific). For LC-MS/MS analysis protein digest solutions were injected and loaded onto a trap column (Acclaim PepMap 100 C18, 100μm × 2cm) for desalting and concentration at 8μL/min in 2% acetonitrile, 0.1% TFA. Peptides were then eluted on-line to an analytical column (Acclaim Pepmap RSLC C18, 75μm × 50cm) at a flow rate of 250nL/min. Peptides were separated using a 120 minute gradient, 4-25% of buffer B for 90 minutes followed by 25-45% buffer B for another 30 minutes (composition of buffer B – 80% acetonitrile, 0.1% FA) and subsequent column conditioning and equilibration. Eluted peptides were analysed by the mass spectrometer operating in positive polarity using a data-dependent acquisition mode. Ions for fragmentation were determined from an initial MS1 survey scan at 70,000 resolution, followed by HCD (Higher Energy Collision Induced Dissociation) of the top 12 most abundant ions at 17,500 resolution. MS1 and MS2 scan AGC targets were set to 3e6 and 5e4 for maximum injection times of 50ms and 50ms respectively. A survey scan m/z range of 400 – 1800 was used, normalised collision energy set to 27%, charge exclusion enabled with unassigned and +1 charge states rejected and a minimal AGC target of 1e3.Raw data processingData was processed using the MaxQuant software platform (v1.5.8.3), with database searches carried out by the in-built Andromeda search engine against the Uniprot S.cerevisiae database (version 20160815, number of entries: 6,729). A reverse decoy search approach was used at a 1% false discovery rate (FDR) for both peptide spectrum matches and protein groups. Search parameters included: maximum missed cleavages set to 2, fixed modification of cysteine carbamidomethylation and variable modifications of methionine oxidation, protein N-terminal acetylation and serine, threonine, tyrosine phosphorylation. Label-free quantification was enabled with an LFQ minimum ratio count of 2. ‘Match between runs’ function was used with match and alignment time limits of 1 and 20 minutes respectively.

Protein identification and quantification by LC-MS/MS

Sample processingProtein samples eluted from anti-HA magnetic beads were buffer exchanged, reduced and alkylated using a FASP protocol. Briefly, samples were loaded onto 30 kDa centrifugal concentrators (Millipore, MRCF0R030) and buffer exchange was carried out by centrifugation on a bench top centrifuge (15min, 12,000g). Multiple buffer exchanges were performed sequentially with UA buffer (8M urea in 100mM Tris pH 8.5, 3x200μl), reduction with 10mM DTT in UA buffer (30min, 40°C) and alkylation with 50mM chloroacetamide in UA buffer (20min, 25°C). This was followed by buffer exchange into UA buffer (3x100μl) and 50mM ammonium bicarbonate (3x100μl). Digestion was carried out with mass spectrometry grade trypsin (Promega, V5280) using 1μg protease per digest (16h, 37°C). Tryptic peptides were collected by centrifugation into a fresh collection tube (10min, 12,000g) and washing of the concentrator with 0.5M sodium chloride (50μl, 10min, 12,000g) for maximal recovery. Following acidification with 1% trifluoroacetic acid (TFA) to a final concentration of 0.2%, collected protein digests were desalted using Glygen C18 spin tips (Glygen Corp, TT2C18.96) and peptides eluted with 60% acetonitrile, 0.1% formic acid (FA). Eluents were then dried using vacuum centrifugation.

Sample processing

Protein samples eluted from anti-HA magnetic beads were buffer exchanged, reduced and alkylated using a FASP protocol. Briefly, samples were loaded onto 30 kDa centrifugal concentrators (Millipore, MRCF0R030) and buffer exchange was carried out by centrifugation on a bench top centrifuge (15min, 12,000g). Multiple buffer exchanges were performed sequentially with UA buffer (8M urea in 100mM Tris pH 8.5, 3x200μl), reduction with 10mM DTT in UA buffer (30min, 40°C) and alkylation with 50mM chloroacetamide in UA buffer (20min, 25°C). This was followed by buffer exchange into UA buffer (3x100μl) and 50mM ammonium bicarbonate (3x100μl). Digestion was carried out with mass spectrometry grade trypsin (Promega, V5280) using 1μg protease per digest (16h, 37°C). Tryptic peptides were collected by centrifugation into a fresh collection tube (10min, 12,000g) and washing of the concentrator with 0.5M sodium chloride (50μl, 10min, 12,000g) for maximal recovery. Following acidification with 1% trifluoroacetic acid (TFA) to a final concentration of 0.2%, collected protein digests were desalted using Glygen C18 spin tips (Glygen Corp, TT2C18.96) and peptides eluted with 60% acetonitrile, 0.1% formic acid (FA). Eluents were then dried using vacuum centrifugation.

Liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysisDried tryptic digests were redissolved in 0.1% TFA by shaking (1200rpm) for 30min and sonication on an ultrasonic water bath for 10min, followed by centrifugation (20,000g, 5°C) for 10min. LC-MS/MS analysis was carried out in technical duplicates and separation was performed using an Ultimate 3000 RSLC nano liquid chromatography system (Thermo Scientific) coupled to a coupled to a Q-Exactive mass spectrometer (Thermo Scientific) via an EASY spray source (Thermo Scientific). For LC-MS/MS analysis protein digest solutions were injected and loaded onto a trap column (Acclaim PepMap 100 C18, 100μm × 2cm) for desalting and concentration at 8μL/min in 2% acetonitrile, 0.1% TFA. Peptides were then eluted on-line to an analytical column (Acclaim Pepmap RSLC C18, 75μm × 50cm) at a flow rate of 250nL/min. Peptides were separated using a 120 minute gradient, 4-25% of buffer B for 90 minutes followed by 25-45% buffer B for another 30 minutes (composition of buffer B – 80% acetonitrile, 0.1% FA) and subsequent column conditioning and equilibration. Eluted peptides were analysed by the mass spectrometer operating in positive polarity using a data-dependent acquisition mode. Ions for fragmentation were determined from an initial MS1 survey scan at 70,000 resolution, followed by HCD (Higher Energy Collision Induced Dissociation) of the top 12 most abundant ions at 17,500 resolution. MS1 and MS2 scan AGC targets were set to 3e6 and 5e4 for maximum injection times of 50ms and 50ms respectively. A survey scan m/z range of 400 – 1800 was used, normalised collision energy set to 27%, charge exclusion enabled with unassigned and +1 charge states rejected and a minimal AGC target of 1e3.

Liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysis

Dried tryptic digests were redissolved in 0.1% TFA by shaking (1200rpm) for 30min and sonication on an ultrasonic water bath for 10min, followed by centrifugation (20,000g, 5°C) for 10min. LC-MS/MS analysis was carried out in technical duplicates and separation was performed using an Ultimate 3000 RSLC nano liquid chromatography system (Thermo Scientific) coupled to a coupled to a Q-Exactive mass spectrometer (Thermo Scientific) via an EASY spray source (Thermo Scientific). For LC-MS/MS analysis protein digest solutions were injected and loaded onto a trap column (Acclaim PepMap 100 C18, 100μm × 2cm) for desalting and concentration at 8μL/min in 2% acetonitrile, 0.1% TFA. Peptides were then eluted on-line to an analytical column (Acclaim Pepmap RSLC C18, 75μm × 50cm) at a flow rate of 250nL/min. Peptides were separated using a 120 minute gradient, 4-25% of buffer B for 90 minutes followed by 25-45% buffer B for another 30 minutes (composition of buffer B – 80% acetonitrile, 0.1% FA) and subsequent column conditioning and equilibration. Eluted peptides were analysed by the mass spectrometer operating in positive polarity using a data-dependent acquisition mode. Ions for fragmentation were determined from an initial MS1 survey scan at 70,000 resolution, followed by HCD (Higher Energy Collision Induced Dissociation) of the top 12 most abundant ions at 17,500 resolution. MS1 and MS2 scan AGC targets were set to 3e6 and 5e4 for maximum injection times of 50ms and 50ms respectively. A survey scan m/z range of 400 – 1800 was used, normalised collision energy set to 27%, charge exclusion enabled with unassigned and +1 charge states rejected and a minimal AGC target of 1e3.

Raw data processingData was processed using the MaxQuant software platform (v1.5.8.3), with database searches carried out by the in-built Andromeda search engine against the Uniprot S.cerevisiae database (version 20160815, number of entries: 6,729). A reverse decoy search approach was used at a 1% false discovery rate (FDR) for both peptide spectrum matches and protein groups. Search parameters included: maximum missed cleavages set to 2, fixed modification of cysteine carbamidomethylation and variable modifications of methionine oxidation, protein N-terminal acetylation and serine, threonine, tyrosine phosphorylation. Label-free quantification was enabled with an LFQ minimum ratio count of 2. ‘Match between runs’ function was used with match and alignment time limits of 1 and 20 minutes respectively.

Raw data processing

Data was processed using the MaxQuant software platform (v1.5.8.3), with database searches carried out by the in-built Andromeda search engine against the Uniprot S.cerevisiae database (version 20160815, number of entries: 6,729). A reverse decoy search approach was used at a 1% false discovery rate (FDR) for both peptide spectrum matches and protein groups. Search parameters included: maximum missed cleavages set to 2, fixed modification of cysteine carbamidomethylation and variable modifications of methionine oxidation, protein N-terminal acetylation and serine, threonine, tyrosine phosphorylation. Label-free quantification was enabled with an LFQ minimum ratio count of 2. ‘Match between runs’ function was used with match and alignment time limits of 1 and 20 minutes respectively.

Supplementary Material12

Supplementary Material
