---
pmid: '19656744'
title: 'Ribosomal protein S3: A multi-functional protein that interacts with both
  p53 and MDM2 through its KH domain.'
authors:
- Yadavilli S
- Mayo LD
- Higgins M
- Lain S
- Hegde V
- Deutsch WA
journal: DNA Repair (Amst)
year: '2009'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2748156
doi: 10.1016/j.dnarep.2009.07.003
---

# Ribosomal protein S3: A multi-functional protein that interacts with both p53 and MDM2 through its KH domain.
**Authors:** Yadavilli S, Mayo LD, Higgins M, Lain S, Hegde V, Deutsch WA
**Journal:** DNA Repair (Amst) (2009)
**DOI:** [10.1016/j.dnarep.2009.07.003](https://doi.org/10.1016/j.dnarep.2009.07.003)
**PMC:** [PMC2748156](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2748156/)

## Abstract

1. DNA Repair (Amst). 2009 Oct 2;8(10):1215-24. doi:
10.1016/j.dnarep.2009.07.003.  Epub 2009 Aug 4.

Ribosomal protein S3: A multi-functional protein that interacts with both p53 
and MDM2 through its KH domain.

Yadavilli S(1), Mayo LD, Higgins M, Lain S, Hegde V, Deutsch WA.

Author information:
(1)Pennington Biomedical Research Center, Louisiana State University System, 
Baton Rouge, 70808, USA.

The p53 protein responds to cellular stress and regulates genes involved in cell 
cycle, apoptosis, and DNA repair. Under normal conditions, p53 levels are kept 
low through MDM2-mediated ubiquitination and proteosomal degradation. In search 
for novel proteins that participate in this regulatory loop, we performed an 
MDM2 peptide pull-down assay and mass spectrometry to screen for potential 
interacting partners of MDM2. We identified ribosomal protein S3 (RPS3), whose 
interaction with MDM2, and notably p53, was further established by His and GST 
pull-down assays, fluorescence resonance energy transfer and an in situ 
proximity ligation assay. Additionally, in cells exposed to oxidative stress, 
p53 levels increased slightly over 24h, whereas MDM2 levels declined after 6h 
exposure, but rose over the next 18h of exposure. Conversely, in cells exposed 
to oxidative stress and harboring siRNA to knockdown RPS3 expression, decreased 
p53 levels and loss of the E3 ubiquitin ligase domain possessed by MDM2 were 
observed. DNA pull-down assays using a 7,8-dihydro-8-oxoguanine duplex 
oligonucleotide as a substrate found that RPS3 acted as a scaffold for the 
additional binding of MDM2 and p53, suggesting that RPS3 interacts with 
important proteins involved in maintaining genomic integrity.

DOI: 10.1016/j.dnarep.2009.07.003
PMCID: PMC2748156
PMID: 19656744 [Indexed for MEDLINE]

Conflict of interest statement: Conflict of interest The authors declare that 
there are no conflicts of interest.

## Full Text

Abstract

The p53 protein responds to cellular stress and regulates genes involved in cell cycle, apoptosis, and DNA repair. Under normal conditions, p53 levels are kept low through MDM2-mediated ubiquitination and proteosomal degradation. In search for novel proteins that participate in this regulatory loop, we performed an MDM2 peptide pull-down assay and mass spectrometry to screen for potential interacting partners of MDM2. We identified ribosomal protein S3 (RPS3), whose interaction with MDM2, and notably p53, was further established by His and GST pull-down assays, fluorescence resonance energy transfer and an in situ proximity ligation assay. Additionally, in cells exposed to oxidative stress, p53 levels increased slightly over 24 hrs, whereas MDM2 levels declined after 6 hrs exposure, but rose over the next 18 hrs of exposure. Conversely, in cells exposed to oxidative stress and harboring siRNA to knockdown RPS3 expression, decreased p53 levels and loss of the E3 ubiquitin ligase domain possessed by MDM2 was observed. DNA pull-down assays using a 7, 8-dihydro-8-oxoguanine duplex oligonucleotide as a substrate found that RPS3 acted as a scaffold for the additional binding of MDM2 and p53, suggesting that RPS3 interacts with important proteins involved in maintaining genomic integrity.

1. Introduction

P53 is a transcription factor that can regulate numerous downstream targets to induce cell cycle arrest, apoptosis and DNA repair [ 1 , 2 ]. It is activated by a variety of circumstances including DNA damage, oxidative stress and hypoxia. Cells that encounter such conditions either arrest cell division or undergo apoptosis if they have a functional p53 pathway.

In the absence of stress signals, the p53 protein is kept in check by MDM2 through different mechanisms, one of which includes acting as an E3 ubiquitin ligase to target p53 for proteosomal degradation [ 3 , 4 ]. Also, low levels of MDM2 induce monoubiquitination and nuclear export of p53, whereas high levels promote its polyubiquitination and nuclear degradation [ 3 – 6 ]. This pathway also involves AKT1, as it phosphorylates MDM2 and results in the translocation of MDM2 to the nucleus where it binds to p53 and targets it for degradation by the proteosome [ 7 , 8 ].

When cells are exposed to stress, DNA damage inducible kinases are rapidly activated to phosphorylate MDM2 at an N-terminal Serine [ 9 ], leading to destabilization of this protein through MDM2-mediated autoubiquitination. In regards to p53, DNA damage activates ATM kinase to phosphorylate S15 of p53, with a subsequent phosphorylation occurring at T18 and S20. These amino acids ordinarily interact with a hydrophobic pocket in the MDM2 N-terminus, but upon phosphorylation of T18 of p53, a destabilization results with subsequent dissociation of the p53/MDM2 complex [ 10 ]. Furthermore, MDM2 is a substrate for DNA-PK that can modify S17 which causes a structural change that prevents its association with p53 [ 11 ].

In an attempt to identify other novel proteins that interact with MDM2, we performed a co-precipitation of proteins bound to a biotinylated MDM2 peptide attached to streptavidin beads in which ribosomal protein S3 (RPS3) was one of the two proteins captured, and subsequently led to the discovery that RPS3 also interacts with p53 to protect it from MDM2-mediated ubiquitination.

3.6. RPS3 knockdown results in decreased p53 and MDM2 levels

Since the presence of RPS3 appeared in vitro to protect p53 from MDM2 ubiquitination, we next determined the effect of RNAi-mediated knockdown of RPS3 on the protein levels of both MDM2 and p53. We used a siRNA that was previously reported by us to decrease the total amount of RPS3 protein by about 50% [ 14 ]. This level of knockdown has been shown to have no adverse affects on protein synthesis that might contribute to ribosomal instability. Furthermore, RNAi knockdown appeared to only affect cytosolic, ribosome-free RPS3 levels, whereas ribosomal-bound RPS3 was not affected [ 14 ].

Cells were transiently transfected with a siRNA construct specific for RPS3, or with a scrambled, non-specific siRNA control construct (NS), and exposed to oxidative stress for various periods of time for up to 24 h. Protein extracts were separated by SDS-PAGE and analyzed by western analysis. In control cells (NS), p53 levels increased roughly 60% when cells were exposed to oxidative stress ( Fig. 7A ). On the other hand, in cells transiently transfected with RPS3 siRNA, there was roughly 65% as much p53 that did not substantially change in the first 12 h after exposure to oxidative stress. The levels of MDM2 in control (NS) cells clearly changed in response to oxidative stress, in which there was a substantial 80% decrease within the first 6 h of exposure to hydrogen peroxide, and then an incremental increase over the next 18 h for cells transiently transfected with a scrambled siRNA ( Fig. 7B ). The changes in MDM2 levels in cells transiently transfected with RPS3 siRNA also showed a temporary 80% drop in MDM2 levels that incrementally increased after 24 h exposure to oxidative stress. However, western analysis revealed that the faster migrating band identified by our antibody cocktail to MDM2 was much fainter in cells expressing siRNA to knockdown RPS3 ( Fig. 7B ). Taken together, our results show decreases in both p53 and MDM2 levels (oxidative-stress dependant) in cells expressing siRNA for the knockdown of RPS3.

4. Discussion

Even though the existence of the p53/MDM2 regulatory loop has been known for some time, the system most likely contains as yet identified components as our studies suggest. The unexpected finding that RPS3 was trapped using a biotinylated peptide of MDM2 as bait led us to examine whether RPS3 also interacted with p53, which in fact was observed using several different in vitro and in vivo assays.

RPS3 was originally identified as a component of the small ribosomal subunit where it is involved in protein synthesis [ 20 , 21 ]. We subsequently found that RPS3 is involved in an oxidative-stress signaling pathway that involves its translocation to the nucleus and co-localization with foci of 8-oxodG [ 12 ]. RPS3’s ability to also enhance the catalytic activity of BER proteins [ 22 ] pointed to the multifunctional roles RPS3 participates in. More recently, that view was underscored by the surprising discovery that RPS3 is a subunit of nuclear NF-κB complexes [ 14 ]. The translocation of RPS3 to the nucleus has been shown to occur by CD3/CD28 (TCR) and TNFα stimulation [ 14 ], as well as ERK1/2 mediated phosphorylation in response to oxidative stress [ 12 ].

Our findings show that RPS3 interacts with both p53 and MDM2. Furthermore, the interaction depends upon the KH domain possessed by RPS3. This domain was originally defined as an important ingredient for binding to single-stranded RNA and DNA [ 23 ]. As an example, hnRNP K has been found to be associated with RNA in RNP particles [ 24 ], as well DNA recognition [ 25 , 26 ]. Notably, hnRNP K has also been found to be an important co-activator of p53 in response to DNA damage [ 27 ]. More recently, the KH domain of RPS3 was found to be essential for the interaction between it and p65 in NF-κB complexes [ 14 ]. Thus, KH domains offer an important ingredient in DNA damage response and transcriptional activities.

Using the highly sensitive FRET assay for demonstrating protein:protein interactions at single-cell resolution, we observed a robust interaction between RPS3 and p53 that rose over two-fold when cells were exposed to oxidative stress. Similar observations were made using the equally sensitive in situ proximity ligation assay. In regard to the interaction between RPS3 and MDM2, it is clear using FRET that these proteins interact, but whether the association is enhanced upon exposure to oxidative stress is not as clear as that observed for the interaction between RPS3 and p53. The FRET results appear to suggest a moderate increase in the interaction between RPS3 and MDM2 under oxidative stress conditions, whereas the in situ proximity ligation assay clearly shows a robust increase in the intensity of interaction between RPS3 and MDM2 when cells were exposed to oxidative stress.

Our data for experiments performed in the presence of Nutlin-3 imply that the interaction mechanism between RPS3 and p53 is different than that between RPS3 and MDM2. For example, the presence of Nutlin-3 reduced over two-fold the nuclear interaction between p53 and RPS3 under oxidative-stress conditions. On the other hand, Nutlin-3 had little if any affect on the interaction between RPS3 and MDM2, regardless of oxidative conditions or subcellular location. These results therefore imply that the optimal nuclear interaction for RPS3 with p53 is likely more efficient in the added presence of MDM2.

Our in vitro and in vivo data suggest that at least one biological endpoint of the interaction of RPS3 with p53 and MDM2 is to protect p53 from MDM2-mediated ubiquitination. This was suggested in cells expressing a siRNA duplex to knockdown RPS3 that resulted in a 30% to 40% decrease for p53 and the loss of one of the domains possessed by MDM2. It is known that ribosomal instability can give rise to the binding of ribosomal proteins L5, L11, and L23 to MDM2 [ 28 – 30 ]. A question therefore arises as to whether the conditions chosen for the experiments presented in Fig. 7 was due to ribosomal stress brought on by the iRNA-generated partial decrease of endogenous RPS3 protein. This seems extremely unlikely based upon our previous results [ 14 ] using the same iRNA construct as was used here, in which we found using several methods that the partial knockdown (40–60%) of RPS3 had no effect on protein synthesis.

The results from our DNA pull-down assay offers the intriguing possibility that the presence of 8-oxodG in damaged DNA could act as staging position for the interaction of RPS3 with p53 and MDM2. What remains to be determined is how the interactions at 8-oxodG between RPS3, MDM2, and p53 are coordinated and controlled. Clearly, post-translational modifications of RPS3 could dictate its involvement in its interaction with both p53 and MDM2. It is known that p53 is post-translationally modified at roughly 30 different sites by phosphorylation, acetylation, methylation, O-glycosylation, neddylation, sumoylation and ubiquitination [ 31 ]. Similar post-translatioanal modifications for RPS3 have been identified, including ERK1/2 phosphorylation [ 12 ], Hrr25 phosphorylation [ 21 ], and neddylation [ 32 ]. We have also found that RPS3 is subject to ubiquitination and AKT phosphorylation (unpublished observations). Clearly, any or all of the above modifications could play an important role in the in vivo interactions established between RPS3, p53, and MDM2. What remains to be seen is how the interaction between RPS3, p53 or MDM2 affects genomic stability, especially at sites of 8-oxodG
