---
pmid: '25910212'
title: Widespread macromolecular interaction perturbations in human genetic disorders.
authors:
- Sahni N
- Yi S
- Taipale M
- Fuxman Bass JI
- Coulombe-Huntington J
- Yang F
- Peng J
- Weile J
- Karras GI
- Wang Y
- Kovács IA
- Kamburov A
- Krykbaeva I
- Lam MH
- Tucker G
- Khurana V
- Sharma A
- Liu YY
- Yachie N
- Zhong Q
- Shen Y
- Palagi A
- San-Miguel A
- Fan C
- Balcha D
- Dricot A
- Jordan DM
- Walsh JM
- Shah AA
- Yang X
- Stoyanova AK
- Leighton A
- Calderwood MA
- Jacob Y
- Cusick ME
- Salehi-Ashtiani K
- Whitesell LJ
- Sunyaev S
- Berger B
- Barabási AL
- Charloteaux B
- Hill DE
- Hao T
- Roth FP
- Xia Y
- Walhout AJM
- Lindquist S
- Vidal M
journal: Cell
year: '2015'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC4441215
doi: 10.1016/j.cell.2015.04.013
---

# Widespread macromolecular interaction perturbations in human genetic disorders.
**Authors:** Sahni N, Yi S, Taipale M, Fuxman Bass JI, Coulombe-Huntington J, Yang F, Peng J, Weile J, Karras GI, Wang Y, Kovács IA, Kamburov A, Krykbaeva I, Lam MH, Tucker G, Khurana V, Sharma A, Liu YY, Yachie N, Zhong Q, Shen Y, Palagi A, San-Miguel A, Fan C, Balcha D, Dricot A, Jordan DM, Walsh JM, Shah AA, Yang X, Stoyanova AK, Leighton A, Calderwood MA, Jacob Y, Cusick ME, Salehi-Ashtiani K, Whitesell LJ, Sunyaev S, Berger B, Barabási AL, Charloteaux B, Hill DE, Hao T, Roth FP, Xia Y, Walhout AJM, Lindquist S, Vidal M
**Journal:** Cell (2015)
**DOI:** [10.1016/j.cell.2015.04.013](https://doi.org/10.1016/j.cell.2015.04.013)
**PMC:** [PMC4441215](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4441215/)

## Abstract

1. Cell. 2015 Apr 23;161(3):647-660. doi: 10.1016/j.cell.2015.04.013.

Widespread macromolecular interaction perturbations in human genetic disorders.

Sahni N(1), Yi S(1), Taipale M(2), Fuxman Bass JI(3), Coulombe-Huntington J(4), 
Yang F(5), Peng J(6), Weile J(5), Karras GI(2), Wang Y(1), Kovács IA(7), 
Kamburov A(8), Krykbaeva I(2), Lam MH(9), Tucker G(6), Khurana V(2), Sharma 
A(7), Liu YY(10), Yachie N(5), Zhong Q(8), Shen Y(1), Palagi A(8), San-Miguel 
A(8), Fan C(1), Balcha D(1), Dricot A(1), Jordan DM(11), Walsh JM(8), Shah 
AA(8), Yang X(8), Stoyanova AK(8), Leighton A(6), Calderwood MA(1), Jacob Y(12), 
Cusick ME(1), Salehi-Ashtiani K(8), Whitesell LJ(13), Sunyaev S(14), Berger 
B(15), Barabási AL(16), Charloteaux B(1), Hill DE(1), Hao T(1), Roth FP(17), Xia 
Y(18), Walhout AJM(19), Lindquist S(20), Vidal M(21).

Author information:
(1)Genomic Analysis of Network Perturbations Center of Excellence in Genomic 
Science (CEGS), Dana-Farber Cancer Institute, Boston, MA 02215, USA; Center for 
Cancer Systems Biology (CCSB) and Department of Cancer Biology, Dana-Farber 
Cancer Institute, Boston, MA 02215, USA; Department of Genetics, Harvard Medical 
School, Boston, MA 02115, USA.
(2)Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA.
(3)Program in Systems Biology, Program in Molecular Medicine, University of 
Massachusetts Medical School, Worcester, MA 01605, USA.
(4)Department of Bioengineering, Faculty of Engineering, McGill University, 
Montreal, QC H3A 0C3, Canada.
(5)Genomic Analysis of Network Perturbations Center of Excellence in Genomic 
Science (CEGS), Dana-Farber Cancer Institute, Boston, MA 02215, USA; Center for 
Cancer Systems Biology (CCSB) and Department of Cancer Biology, Dana-Farber 
Cancer Institute, Boston, MA 02215, USA; Departments of Molecular Genetics and 
Computer Science, University of Toronto, Toronto, ON M5S 3E1, Canada; Donnelly 
Centre, University of Toronto, Toronto, ON M5S 3E1, Canada; Lunenfeld-Tanenbaum 
Research Institute, Mt. Sinai Hospital, Toronto, ON M5G 1X5, Canada.
(6)Computer Science and Artificial Intelligence Laboratory, Massachusetts 
Institute of Technology, Cambridge, MA 02139, USA.
(7)Genomic Analysis of Network Perturbations Center of Excellence in Genomic 
Science (CEGS), Dana-Farber Cancer Institute, Boston, MA 02215, USA; Center for 
Cancer Systems Biology (CCSB) and Department of Cancer Biology, Dana-Farber 
Cancer Institute, Boston, MA 02215, USA; Center for Complex Network Research 
(CCNR) and Departments of Physics, Biology and Computer Science, Northeastern 
University, Boston, MA 02115, USA.
(8)Center for Cancer Systems Biology (CCSB) and Department of Cancer Biology, 
Dana-Farber Cancer Institute, Boston, MA 02215, USA; Department of Genetics, 
Harvard Medical School, Boston, MA 02115, USA.
(9)Donnelly Centre, University of Toronto, Toronto, ON M5S 3E1, Canada.
(10)Center for Cancer Systems Biology (CCSB) and Department of Cancer Biology, 
Dana-Farber Cancer Institute, Boston, MA 02215, USA; Center for Complex Network 
Research (CCNR) and Departments of Physics, Biology and Computer Science, 
Northeastern University, Boston, MA 02115, USA.
(11)Division of Genetics, Department of Medicine, Brigham and Women's Hospital, 
Harvard Medical School, Boston, MA 02115, USA; Program in Biophysics, Harvard 
University, Cambridge, MA 02139, USA.
(12)Center for Cancer Systems Biology (CCSB) and Department of Cancer Biology, 
Dana-Farber Cancer Institute, Boston, MA 02215, USA; Department of Genetics, 
Harvard Medical School, Boston, MA 02115, USA; Département de Virologie, Unité 
de Génétique Moléculaire des Virus ARN (GMVR), Institut Pasteur, UMR3569, Centre 
National de la Recherche Scientifique, and Université Paris Diderot, Paris, 
France.
(13)Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA; 
Department of Biology and Howard Hughes Medical Institute, Massachusetts 
Institute of Technology, Cambridge, MA 02139, USA.
(14)Division of Genetics, Department of Medicine, Brigham and Women's Hospital, 
Harvard Medical School, Boston, MA 02115, USA; The Broad Institute of MIT and 
Harvard, Cambridge, MA 02142, USA.
(15)Computer Science and Artificial Intelligence Laboratory, Massachusetts 
Institute of Technology, Cambridge, MA 02139, USA; Department of Mathematics and 
Electrical Engineering and Computer Science, Massachusetts Institute of 
Technology, Cambridge, MA 02139, USA.
(16)Genomic Analysis of Network Perturbations Center of Excellence in Genomic 
Science (CEGS), Dana-Farber Cancer Institute, Boston, MA 02215, USA; Center for 
Cancer Systems Biology (CCSB) and Department of Cancer Biology, Dana-Farber 
Cancer Institute, Boston, MA 02215, USA; Center for Complex Network Research 
(CCNR) and Departments of Physics, Biology and Computer Science, Northeastern 
University, Boston, MA 02115, USA; Department of Medicine, Brigham and Women's 
Hospital, Harvard Medical School, Boston, MA 02115, USA.
(17)Genomic Analysis of Network Perturbations Center of Excellence in Genomic 
Science (CEGS), Dana-Farber Cancer Institute, Boston, MA 02215, USA; Center for 
Cancer Systems Biology (CCSB) and Department of Cancer Biology, Dana-Farber 
Cancer Institute, Boston, MA 02215, USA; Departments of Molecular Genetics and 
Computer Science, University of Toronto, Toronto, ON M5S 3E1, Canada; Donnelly 
Centre, University of Toronto, Toronto, ON M5S 3E1, Canada; Lunenfeld-Tanenbaum 
Research Institute, Mt. Sinai Hospital, Toronto, ON M5G 1X5, Canada; Canadian 
Institute for Advanced Research, Toronto, ON M5G 1Z8, Canada.
(18)Center for Cancer Systems Biology (CCSB) and Department of Cancer Biology, 
Dana-Farber Cancer Institute, Boston, MA 02215, USA; Department of 
Bioengineering, Faculty of Engineering, McGill University, Montreal, QC H3A 0C3, 
Canada.
(19)Center for Cancer Systems Biology (CCSB) and Department of Cancer Biology, 
Dana-Farber Cancer Institute, Boston, MA 02215, USA; Program in Systems Biology, 
Program in Molecular Medicine, University of Massachusetts Medical School, 
Worcester, MA 01605, USA.
(20)Whitehead Institute for Biomedical Research, Cambridge, MA 02142, USA; 
Department of Biology and Howard Hughes Medical Institute, Massachusetts 
Institute of Technology, Cambridge, MA 02139, USA; Howard Hughes Medical 
Institute, Cambridge, MA 02139, USA. Electronic address: lindquist@wi.mit.edu.
(21)Genomic Analysis of Network Perturbations Center of Excellence in Genomic 
Science (CEGS), Dana-Farber Cancer Institute, Boston, MA 02215, USA; Center for 
Cancer Systems Biology (CCSB) and Department of Cancer Biology, Dana-Farber 
Cancer Institute, Boston, MA 02215, USA; Department of Genetics, Harvard Medical 
School, Boston, MA 02115, USA. Electronic address: marc_vidal@dfci.harvard.edu.

Comment in
    Nat Rev Genet. 2015 Jun;16(6):317. doi: 10.1038/nrg3957.

How disease-associated mutations impair protein activities in the context of 
biological networks remains mostly undetermined. Although a few renowned alleles 
are well characterized, functional information is missing for over 100,000 
disease-associated variants. Here we functionally profile several thousand 
missense mutations across a spectrum of Mendelian disorders using various 
interaction assays. The majority of disease-associated alleles exhibit wild-type 
chaperone binding profiles, suggesting they preserve protein folding or 
stability. While common variants from healthy individuals rarely affect 
interactions, two-thirds of disease-associated alleles perturb protein-protein 
interactions, with half corresponding to "edgetic" alleles affecting only a 
subset of interactions while leaving most other interactions unperturbed. With 
transcription factors, many alleles that leave protein-protein interactions 
intact affect DNA binding. Different mutations in the same gene leading to 
different interaction profiles often result in distinct disease phenotypes. Thus 
disease-associated alleles that perturb distinct protein activities rather than 
grossly affecting folding and stability are relatively widespread.

Copyright © 2015 Elsevier Inc. All rights reserved.

DOI: 10.1016/j.cell.2015.04.013
PMCID: PMC4441215
PMID: 25910212 [Indexed for MEDLINE]

## Full Text

INTRODUCTION

Over a hundred thousand genetic variants have been identified across a large number of Mendelian disorders ( Amberger et al., 2011 ), complex traits ( Hindorff et al., 2009 ), and cancer types ( Chin et al., 2011 ). However, many fundamental questions regarding genotype-phenotype relationships remain unresolved ( Vidal et al., 2011 ). One critical challenge is to distinguish causal disease mutations from non-pathogenic polymorphisms. Even when causal mutations are identified, the functional consequence of such mutations is often elusive ( Sahni et al., 2013 ).

Genotypic information alone rarely elucidates the mechanistic insights pertaining to disease pathogenesis. Although genotype-phenotype relationships can be modeled under the assumption that most disease-associated mutations lead to complete loss of protein function, e.g. through radical changes such as protein misfolding and instability ( Subramanian and Kumar, 2006 ) ( Figure 1A ), the reality is often more complex, as in the case of mutations affecting the same gene but giving rise to clinically distinguishable diseases ( Zhong et al., 2009 ). In addition, since genes and gene products do not function in isolation but interact with each other in the context of interactome networks ( Vidal et al., 2011 ), it is likely that many diseases result from perturbations of such complex networks ( Goh et al., 2007 ).

Missense mutations are among the most common sequence alterations in Mendelian disorders, accounting for more than half of all reported mutations in the Human Gene Mutation Database (HGMD) ( Stenson et al., 2014 ). In principle, missense mutations may have no functional consequences, disrupt the three-dimensional structure of the corresponding protein, or exert specific effects on particular molecular or biochemical interactions ( Figure 1A ), such as protein-protein interactions (PPIs), protein-DNA interactions (PDIs), or enzyme-substrate interactions, while leaving all other functional properties unperturbed. We previously reported that a considerable portion of Mendelian disease mutations could indeed be predicted computationally to cause interaction-specific, or “edgetic”, perturbations ( Zhong et al., 2009 ). However, only a small number of genes and associated mutations were experimentally tested in that study, and the extent to which disease mutations globally lead to interaction perturbations remains to be determined.

Here we describe a multi-pronged approach to systematically decipher molecular interaction perturbations associated with missense mutations. Since chaperones and associated quality control factors (QCFs) can salvage unstable proteins by assisting with folding, and an increase in protein-chaperone interactions (PCIs) has been observed for a number of disease mutants ( Whitesell and Lindquist, 2005 ), our systematic approach begins with characterizing PCIs for large numbers of disease-associated alleles, followed by systematic measurements of PPI and PDI profile changes caused by mutations, a strategy referred to as “edgotyping” ( Figure 1B ).

We provide evidence for widespread interaction perturbations across a broad spectrum of human Mendelian disorders. Our results suggest that interaction profiling helps distinguish disease-causing mutations from common variants. Furthermore, the integration of different types of molecular interactions expands our ability to understand complex genotype-phenotype relationships.

DISCUSSION

In this systematic characterization of mutations across various human Mendelian disorders, we have found surprisingly widespread disease-specific perturbations of macromolecular interactions. Approximately 60% of disease-associated missense mutations perturb PPIs, among which half result in complete loss of interactions, generally caused by protein misfolding and impaired expression, and the other half lead to edgetic perturbations. Importantly, different mutations in the same gene frequently result in different interaction perturbation profiles. This strongly suggests that the “edgotype” of a mutation represents a fundamental link between genotype and phenotype.

Our systematic edgotyping strategy provides a practical approach to classifying candidate disease alleles emerging from genome-wide association studies and from sporadic and somatic mutation sequencing approaches. Edgotyping achieves a high precision in identifying candidate disease-causing mutations based on the interaction perturbations relative to WT alleles ( Figure S6A ). However, the overall sensitivity of an edgotyping approach is compromised due to the false negative rate inherent to the assays used. We expect that a significant fraction of variants currently viewed as non-interaction-perturbing (quasi-WT) will eventually be proven to cause disease. This circumstance likely arises from the incomplete nature of current human interactome network maps ( Rolland et al., 2014 ). Nevertheless, because edgetic mutations cannot become quasi-WT or quasi-null even as interactome maps improve, our estimate of edgetic mutations already provides a reliable minimum lower bound for their frequency.

An alternative possibility is that quasi-WT mutations affect disease phenotypes through perturbation of different types of molecular interactions. Biological signaling is regulated at multiple levels, and various types of molecular interactions are involved ( Sahni et al., 2013 ) as we have shown for PPI and PDI networks. In addition, protein-RNA ( Lee et al., 2006 ) and protein-metabolite ( Carpten et al., 2007 ) interactions have also been shown to be involved in disease. Perturbations of these alternative interaction networks will undoubtedly result in distinct disease consequences. One can envision that integration of additional types of interaction perturbation information with computational predictions will be necessary for a complete understanding of the cellular networks governing a particular disease state ( Figure S7D ). As a major benefit, perturbed interactions spotlight specific targets and pathways that are altered in a patient-specific context. This type of information could provide a much-needed guide in efforts to developing better diagnostic tools and more personalized medical treatments.
