---
pmid: '9298646'
title: Comparing the predicted and observed properties of proteins encoded in the
  genome of Escherichia coli K-12.
authors:
- Link AJ
- Robison K
- Church GM
journal: Electrophoresis
year: '1997'
full_text_available: false
doi: 10.1002/elps.1150180807
---

# Comparing the predicted and observed properties of proteins encoded in the genome of Escherichia coli K-12.
**Authors:** Link AJ, Robison K, Church GM
**Journal:** Electrophoresis (1997)
**DOI:** [10.1002/elps.1150180807](https://doi.org/10.1002/elps.1150180807)

## Abstract

1. Electrophoresis. 1997 Aug;18(8):1259-313. doi: 10.1002/elps.1150180807.

Comparing the predicted and observed properties of proteins encoded in the 
genome of Escherichia coli K-12.

Link AJ(1), Robison K, Church GM.

Author information:
(1)Department of Genetics, Harvard Medical School, Boston, MA 02115, USA.

Mining the emerging abundance of microbial genome sequences for hypotheses is an 
exciting prospect of "functional genomics". At the forefront of this effort, we 
compared the predictions of the complete Escherichia coli genomic sequence with 
the observed gene products by assessing 381 proteins for their mature N-termini, 
in vivo abundances, isoelectric points, molecular masses, and cellular 
locations. Two-dimensional gel electrophoresis (2-DE) and Edman sequencing were 
combined to sequence Coomassie-stained 2-DE spots representing the abundant 
proteins of wild-type E. coli K-12 strains. Greater than 90% of the abundant 
proteins in the E. coli proteome lie in a small isoelectric point and molecular 
mass window of 4-7 and 10-100 kDa, respectively. We identified several highly 
abundant proteins, YjbJ, YjbP, YggX, HdeA, and AhpC, which would not have been 
predicted from the genomic sequence alone. Of the 223 uniquely identified loci, 
60% of the encoded proteins are proteolytically processed. As previously 
reported, the initiator methionine was efficiently cleaved when the penultimate 
amino acid was serine or alanine. In contrast, when the penultimate amino acid 
was threonine, glycine, or proline, cleavage was variable, and valine did not 
signal cleavage. Although signal peptide cleavage sites tended to follow 
predicted rules, the length of the putative signal sequence was occassionally 
greater than the consensus. For proteins predicted to be in the cytoplasm or 
inner membrane, the N-terminal amino acids were highly constrained compared to 
proteins localized to the periplasm or outer membrane. Although cytoplasmic 
proteins follow the N-end rule for protein stability, proteins in the periplasm 
or outer membrane do not follow this rule; several have N-terminal amino acids 
predicted to destabilize the proteins. Surprisingly, 18% of the identified 2-DE 
spots represent isoforms in which protein products of the same gene have 
different observed pI and M(r), suggesting they are post-translationally 
processed. Although most of the predicted and observed values for isoelectric 
point and molecular mass show reasonable concordance, for several proteins the 
observed values significantly deviate from the expected values. Such 
discrepancies may represent either highly processed proteins or 
misinterpretations of the genomic sequence. Our data suggest that AhpC, CspC, 
and HdeA exist as covalent homomultimers, and that IcdA exists as at least three 
isoforms even under conditions in which covalent modification is not predicted. 
We enriched for proteins based on subcellular location and found several 
proteins in unexpected subcellular locations.

DOI: 10.1002/elps.1150180807
PMID: 9298646 [Indexed for MEDLINE]
