---
pmid: '20462497'
title: Structure and mechanism of receptor sharing by the IL-10R2 common chain.
authors:
- Yoon SI
- Jones BC
- Logsdon NJ
- Harris BD
- Deshpande A
- Radaeva S
- Halloran BA
- Gao B
- Walter MR
journal: Structure
year: '2010'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC2879597
doi: 10.1016/j.str.2010.02.009
---

# Structure and mechanism of receptor sharing by the IL-10R2 common chain.
**Authors:** Yoon SI, Jones BC, Logsdon NJ, Harris BD, Deshpande A, Radaeva S, Halloran BA, Gao B, Walter MR
**Journal:** Structure (2010)
**DOI:** [10.1016/j.str.2010.02.009](https://doi.org/10.1016/j.str.2010.02.009)
**PMC:** [PMC2879597](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2879597/)

## Abstract

1. Structure. 2010 May 12;18(5):638-48. doi: 10.1016/j.str.2010.02.009.

Structure and mechanism of receptor sharing by the IL-10R2 common chain.

Yoon SI(1), Jones BC, Logsdon NJ, Harris BD, Deshpande A, Radaeva S, Halloran 
BA, Gao B, Walter MR.

Author information:
(1)Department of Microbiology, University of Alabama at Birmingham, Birmingham, 
AL 35294, USA.

Comment in
    Structure. 2010 May 12;18(5):551-2. doi: 10.1016/j.str.2010.04.003.

IL-10R2 is a shared cell surface receptor required for the activation of five 
class 2 cytokines (IL-10, IL-22, IL-26, IL-28, and IL-29) that play critical 
roles in host defense. To define the molecular mechanisms that regulate its 
promiscuous binding, we have determined the crystal structure of the IL-10R2 
ectodomain at 2.14 A resolution. IL-10R2 residues required for binding were 
identified by alanine scanning and used to derive computational models of 
IL-10/IL-10R1/IL-10R2 and IL-22/IL-22R1/IL-10R2 ternary complexes. The models 
reveal a conserved binding epitope that is surrounded by two clefts that 
accommodate the structural and chemical diversity of the cytokines. These 
results provide a structural framework for interpreting IL-10R2 single 
nucleotide polymorphisms associated with human disease.

Copyright 2010 Elsevier Ltd. All rights reserved.

DOI: 10.1016/j.str.2010.02.009
PMCID: PMC2879597
PMID: 20462497 [Indexed for MEDLINE]

## Full Text

Introduction

IL-10R2 is a ubiquitously expressed and essential receptor chain for at least five IL-10 family cytokines that share ~10–20% sequence identity and distinct 3D structures ( Donnelly et al., 2004 ; Walter, 2004 ). IL-10R2, first called CRF2–4, was originally discovered as an essential component of the IL-10 hetero-dimeric signaling complex formed between the IL-10R1 chain and IL-10R2 ( Kotenko et al., 1997 ). Solution and cell binding studies have shown IL-10R1 exhibits high affinity (nM) for IL-10, whereas IL-10R2 chain affinity is very low (~mM) ( Ding et al., 2000 ; Logsdon et al., 2002 ; Yoon et al., 2005 ; Yoon et al., 2006 ). IL-10 mediated assembly of the IL-10R1/IL-10R2 complex activates intracellular kinases, JAK1 and TYK2, which phosphorylate the intracellular domains of the receptor as well as STAT3 ( Moore et al., 2001 ). Subsequent studies have revealed IL-10R2 also forms IL-22R1/IL-10R2 ( Kotenko et al., 2001 ; Xie et al., 2000 ), IL-20R1/IL-10R2 ( Sheikh et al., 2004 ) and IL-28R1/IL-10R2 ( Kotenko et al., 2003 ; Sheppard et al., 2003 ) heterodimers that are activated by IL-22, IL-26, and IL-28/ IL-29, respectively ( Fig S1 ). Thus, IL-10R2 functions as a common signaling chain in the class 2 cytokine family as observed for gp130, IL-2γc, and GM-CSF βc chains in the class 1 cytokine family ( Wang et al., 2009 ).

Cytokines that signal via IL-10R2 induce pleotropic activities that protect the host from over-exuberant immune responses and activate innate immunity programs in epithelial cells. IL-10 inhibits macrophage and dendritic cell function, prevents pro-inflammatory cytokine synthesis (e.g. TNF-α, IFN-γ, IL-12, IL-l), and blocks antigen presentation ( Moore et al., 2001 ). As a result of its potent immunosuppressive functions, numerous pathogens, including HIV, evade host immune responses by increasing the production of cellular IL-10 ( Blackburn and Wherry, 2007 ; Redpath et al., 2001 ). In addition, cytomegalovirus (CMV) and Epstein Barr virus (EBV) express their own virally encoded IL-10s (cmvIL-10 and ebvIL-10) that disrupt immune function ( Slobedman et al., 2009 ). IL-22, produced by TH17 cells, induces epithelial cells to produce anti-microbial peptides ( Liang et al., 2006 ) but also exhibits protective functions in the gut, lung, and liver ( Radaeva et al., 2004 ; Zenewicz et al., 2007 ). IL-26 is also produced by TH17 cells, up-regulates pro-inflammatory genes in intestinal epithelial cells, and its expression is increased in active Chron’s disease ( Dambacher et al., 2009 ). Finally, IL-28 and IL-29 form the recently discovered type III interferon family ( Kotenko et al., 2003 ; Sheppard et al., 2003 ). IL-28 and IL-29 exhibit potent antiviral activities in keratinocytes and dendritic cells and may play an important role in resolving hepatitis C infection ( Ge et al., 2009 ).

The molecular basis, whereby IL-10R2 is able to form promiscuous receptor-ligand interactions is unknown. In addition, single nucleotide polymorphisms (SNPs) in the IL-10R2 extracellular domain are associated with hepatitis virus subtype B (HBV) persistence ( Frodsham et al., 2006 ), early-onset colitis ( Glocker et al., 2009 ), graft-versus-host disease related to transplant rejection ( Lin et al., 2005 ; Sivula et al., 2009 ), and systemic sclerosis ( Hikami et al., 2008 ); an autoimmune disease characterized by fibrosis of skin and other organs. Thus, structure-function studies on IL-10R2 may be useful in understanding SNPs that alter immune function and result in human disease.

To define the molecular basis for promiscuous binding to diverse cytokine complexes, the crystal structure of the soluble extracellular IL-10R2 chain (sIL-10R2) has been determined at 2.14Ǻ resolution. A series of sIL-10R2 mutants were evaluated for binding to three different binary complexes (BCs, IL-22/sIL-22R1, cmvIL-10/sIL-10R1, hIL-10/sIL-10R1) using tri-molecular surface plasmon resonance (SPR) assays. The binding data provided input for computational docking studies to generate IL-10 and IL-22 ternary complex models. These studies reveal IL-10R2 uses a central common binding epitope, centered on Y82 IL-10R2 , which is surrounded by two clefts that allow the IL-10R2 chain to scan the unique structures and chemistries of helices A and D on the cytokines. The studies reveal IL-10R2 shares a conserved recognition motif with gp130 and IL-2γc suggesting a common origin for the promiscuous class 1 and class 2 cytokine receptors. Despite these common features, sIL-10R2 forms structurally and energetically distinct contacts with IL-10/sIL-10R1 and IL-22/sIL-22R1 BCs, which could assist in the prediction of IL-10R2 SNP function.

Discussion

This study was initiated to understand the molecular mechanisms that allow the sIL-10R2 common chain to promiscuously bind to at least five different class 2 cytokines, which induce diverse cellular activities. The questions we sought to address were: 1) what are the structural features of sIL-10R2 that facilitate promiscuous binding? 2) Does sIL-10R2 form energetically and structurally identical, or distinct, interactions when engaging different binding partners? In addition to determining the molecular basis for promiscuous low affinity protein-protein interactions (14–250µM), for which there is a paucity of literature, these studies also have implications for interpreting IL-10R2 SNPs which cause human disease. In particular, SNPs in sIL-10R2 result in numerous pathologies including early-onset colitis ( Glocker et al., 2009 ) and persistent HBV infection ( Frodsham et al., 2006 ). Several more IL-10R2 SNPs have been identified in patients with inflammatory bowel disease (personal communication, Christoph Klein, Hannover, de), which further argues for the need to understand the structure and mechanism of IL-10R2 binding.

High affinity interactions between hIL-10/sIL-10R1, cmvIL-10/sIL-10R1, and IL-22/sIL-22R1 binary complexes depend on a conserved L2 loop tyrosine, which inserts itself into a cleft between helices A and B on the respective cytokines ( Fig 1D ). This paradigm is reversed for sIL-10R2, which forms L2/L3 and L3/L5 clefts ( Fig 2 ) that selectively recognize protruding surfaces on helix D cmvIL-10 , hIL-10 and helix A IL-22 , respectively ( Figs 4 , 5 ). Thus, sIL-10R2 site 2 recognition is largely defined by the distinct interhelix angles, between helices A and D, found in the different cytokines ( Presnell and Cohen, 1989 ; Walter, 2004 ), Figs 4A , 5A ). Additional contacts are made by tyrosine, arginine, and glutamate residues located at the tips of L2 sIL-10R2 and L5 sIL-10R2 .

While the sIL-10R2 clefts provide the ability to recognize distinct cytokine features, Y82 sIL-10R2 appears to be the common sIL-10R2 epitope required for interaction with each binary receptor complex, much like the conserved L2 tyrosine of sIL-10R1 and sIL-22R1 is highly conserved and required for high affinity IL-22 and IL-10 interactions ( Fig 1D ). Interestingly, prior structure-function studies on the shared receptors, gp130 and γc, has shown aromatic residues (F169 gp130 and Y103 γc ) form conserved contacts in the center of the IL-6/gp130, IL-2/γc, and IL-4/γc interfaces ( Wang et al., 2009 ). Superposition of the D1 domain of sIL-10R2, with gp130, and γc reveals L3 Y82 sIL-10R2 is structurally conserved with F169 gp130 and Y103 γc ( Fig 6A ). Furthermore, superposition of IL-6 from the IL-6/IL-6R/gp130 structure and IL-22 from the IL-22/sIL-22R1/sIL-10R2 model, reveals Y82 sIL-10R2 and F169 gp130 are separated by 4Ǻ (Cα atoms) in their respective complexes ( Fig 6B ). This result provides additional independent data to confirm the IL-22 TC model ( Fig 4 ), and suggests the structurally conserved L3 aromatic residues (Tyr or Phe) form the critical common binding epitope used by the shared cytokine receptors, gp130, IL-2γc, and IL-10R2, to recognize diverse cytokine surfaces. Mutation of F169 gp130 and Y103 γc functionally disrupts receptor-ligand interactions and subsequent cell signaling ( Kurth et al., 1999 ; Middleton et al., 1996 ). Together, these data suggest Y82 sIL-10R2 is critical for sIL-10R2 function and hints at a common origin of promiscuous shared cytokine receptors.

In addition, Olosz and Malek ( Olosz and Malek, 2000 ) pointed out Y103 γc is structurally conserved with the erythropoietin receptor (EPOR) residue, F93. However, the position of F93 EPOR has diverged significantly from the tight cluster of common chain aromatic residues shown in Figure 6A ( Fig S7 ). Nonetheless, it is notable that F93 EPOR is found in the interface of the EPO/EPOR complex ( Syed et al., 1998 ) and in the interface of the EPOR dimer structure, which presumably mimics the unbound EPOR cell surface complex ( Livnah et al., 1999 ). In contrast to IL-10R2, gp130, and γc, EPOR, as well as the growth hormone receptor (GHR), form homodimeric receptor complexes (e.g. 1:2 EPO/EPOR or GH/GHR, ( de Vos et al., 1992 ; Syed et al., 1998 )). GHR also contains a functionally critical aromatic residue in its L3 chain (W104 GHR , ( Clackson and Wells, 1995 )), but its position has diverged even further away from Y82 sIL-10R2 than F93 EPOR ( Fig S7 ). This suggests L3 aromatic residues are generally critical for recognition of diverse class 1 and class 2 cytokines. However, EPOR and GHR, which must form high and low affinity binding interfaces, presumably require greater structural diversity in the positions of their aromatic residues (F93 EPOR and W104 GHR ) than the specialized common chain receptors (IL-10R2, gp130, γc), which only form promiscuous low affinity contacts.

Since IL-10R2 activates multiple cytokine complexes, the cellular phenotype of an IL-10R2 SNP can be caused by disrupting the function of one or more cytokines. Thus, we sought to determine if sIL-10R2 mutants will have the same binding phenotype is each complex. Our studies found select sIL-10R2 residues (Y56 sIL-10R2 , Y140 sIL-10R2 , and W143 sIL-10R2 ) have distinct energetic functions in the IL-22 and cmvIL-10/hIL-10 TCs ( Figs 3 , S5 ). This result suggests specific sIL-10R2 SNPs could selectively disrupt IL-10 or IL-22 signaling and suggests caution should be applied in attributing biological outcomes to specific cytokines. Additional studies will be required to characterize how IL-26 and IL-28/IL-29 are recognized by IL-10R2.

The studies reported here have focused on sIL-10R2 residues directly involved in cytokine and R1 chain recognition. However, other mechanisms that disrupt IL-10R2 function are possible. For example, the K47E sIL-10R2 SNP, which is associated with failure to clear HBV, has been shown to increase IL-10R2 expression levels on cells ( Frodsham et al., 2006 ). Consistent with this interpretation, we found E47 sIL-10R2 and K47 sIL-10R2 had no impact on sIL-10R2 binding to IL-22/sIL-22R1 (data not shown). From a structural perspective, K47 sIL-10R2 is considered the preferred residue since it forms a salt bridge with E96 sIL-10R2 , which would be disrupted when replaced with a glutamate residue ( Fig 1E ). How this mutation increases IL-10R2 surface expression remains unknown.

A key feature of IL-10R2 promiscuity is its very low affinity for the cellular cytokine complexes that it activates. This presents technical challenges that have, to date, prevented crystal structure analysis of the class 2 cytokine ternary complexes. To overcome this problem, we used computational docking studies to generate IL-22 and cmvIL-10 ternary complex models. Although not crystal structures, these models provide clear molecular explanations for how sIL-10R2 distinctly recognizes IL-22 and IL-10 binary complexes, which are consistent with experimental data. However, with the current docking parameters, we failed to generate an independent hIL-10 TC model. Interestingly, the success, or failure, of the docking experiments correlates with sIL-10R2 affinity for IL-22/sIL-22R1 (~14µM), cmvIL-10/sIL-10R1 (~80µM), and hIL-10/sIL-10R1 (~250µM) binary complexes ( Logsdon et al., 2004 ; Yoon et al., 2005 ). This suggests successful sIL-10R2 docking requires a K d of at least ~100µM using the current docking parameters and restraints.

In summary, our structural, biochemical, modeling, and activity data provide novel insights into the promiscuous binding of the IL-10R2 common chain. In particular, the studies demonstrate that despite being a shared weak-binding receptor, sIL-10R2 residues make distinct energetic and structural contributions to binding. This provides an molecular frame work to characterize additional IL-10R2 SNPs and might be used to design protein therapeutics that restore one or more cytokine activities in patients with defective IL-10R2 signaling complexes.
