---
pmid: '25732823'
title: DBC1 functions as a tumor suppressor by regulating p53 stability.
authors:
- Qin B
- Minter-Dykhouse K
- Yu J
- Zhang J
- Liu T
- Zhang H
- Lee S
- Kim J
- Wang L
- Lou Z
journal: Cell Rep
year: '2015'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4351187
doi: 10.1016/j.celrep.2015.01.066
---

# DBC1 functions as a tumor suppressor by regulating p53 stability.
**Authors:** Qin B, Minter-Dykhouse K, Yu J, Zhang J, Liu T, Zhang H, Lee S, Kim J, Wang L, Lou Z
**Journal:** Cell Rep (2015)
**DOI:** [10.1016/j.celrep.2015.01.066](https://doi.org/10.1016/j.celrep.2015.01.066)
**PMC:** [PMC4351187](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4351187/)

## Abstract

1. Cell Rep. 2015 Mar 3;10(8):1324-34. doi: 10.1016/j.celrep.2015.01.066. Epub
2015  Feb 26.

DBC1 functions as a tumor suppressor by regulating p53 stability.

Qin B(1), Minter-Dykhouse K(2), Yu J(2), Zhang J(3), Liu T(4), Zhang H(4), Lee 
S(4), Kim J(4), Wang L(2), Lou Z(5).

Author information:
(1)State Key Laboratory of Genetic Engineering, School of Life Sciences, Fudan 
University, Shanghai, 200433, China; Division of Oncology Research, Mayo Clinic, 
Rochester, MN 55905, USA.
(2)Department of Molecular Pharmacology and Experimental Therapeutics, Mayo 
Clinic, Rochester, MN 55905, USA.
(3)Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, MN 
55905, USA.
(4)Division of Oncology Research, Mayo Clinic, Rochester, MN 55905, USA.
(5)Division of Oncology Research, Mayo Clinic, Rochester, MN 55905, USA. 
Electronic address: lou.zhenkun@mayo.edu.

DBC1 (deleted in breast cancer 1), also known as CCAR2 or KIAA1967, is an 
important negative regulator of SIRT1 and cellular stress response. Although the 
Dbc1 gene localizes at a region that is homozygously deleted in breast cancer, 
its role in tumorigenesis remains unclear. It has been suggested to be either a 
tumor suppressor or an oncogene. Therefore, the function of DBC1 in cancer needs 
to be further explored. Here, we report that Dbc1 knockout mice are tumor prone, 
suggesting that DBC1 functions as a tumor suppressor in vivo. Our data suggest 
that the increased tumor incidence in Dbc1 knockout mice is independent of 
Sirt1. Instead, we found that DBC1 loss results in less p53 protein in vitro and 
in vivo. DBC1 directly binds p53 and stabilizes it through competition with 
MDM2. These studies reveal that DBC1 plays an important role in tumor 
suppression through p53 regulation.

Copyright © 2015 The Authors. Published by Elsevier Inc. All rights reserved.

DOI: 10.1016/j.celrep.2015.01.066
PMCID: PMC4351187
PMID: 25732823 [Indexed for MEDLINE]

Conflict of interest statement: Conflict of interest: None declared.

## Full Text

INTRODUCTIONp53 is a pivotal tumor suppressor and one of the most mutated genes in human cancer, with p53 mutations observed in over half of all human cancers (Brady and Attardi, 2010; Vousden and Prives, 2009). p53 regulates multiple cellular functions, including apoptosis, cell cycle arrest, cell metabolism, and senescence. One of the major regulatory mechanisms of p53 is its ubiquitination by a ubiquitin E3 ligase MDM2 (murine double minute 2), which leads to p53 degradation under normal conditions (Haupt et al., 1997). Under stress conditions, such as DNA damage, viral infection and oncogene activation, p53 is quickly accumulated, resulting in its activation. Several mechanisms are responsible for p53 induction, including MDM2 inactivation, interrupted interaction between p53 and MDM2, and activation of ubiquitin proteases (Dai and Gu, 2010).The Dbc1 gene was initially discovered as a gene deleted in human chromosome 8p21 in breast cancer (Hamaguchi et al., 2002). DBC1 is composed of a leucine zipper motif at the amino-terminus, coiled coil domain at the carboxy-terminus, a nuclear localization signal, an EF hand domain and a Nudix domain (Anantharaman and Aravind, 2008; Kim et al., 2008). DBC1 is processed into C-terminal p120 and p66 fragments, which relocate from nucleus to mitochondria and enhance apoptotic signaling with tumor necrosis factor-α (TNFα) treatment in HeLa cells (Sundararajan et al., 2005). Several studies also suggest that DBC1 regulates hormone receptor activity. For instance, DBC1 activates retinoic acid receptor α and androgen receptor; and represses transcription activity of estrogen receptor β (Fu et al., 2009; Garapaty et al., 2009; Koyama et al., 2010). Furthermore, we and others have found that DBC1 negatively regulates SIRT1 activity through binding to its active site (Kim et al., 2008; Zhao et al., 2008). DNA damage and oxidative stress increase the DBC1-SIRT1 interaction, whereas PKA and AMPK induce dissociation of SIRT1 from DBC1 (Yuan et al., 2012; Nin et al., 2012). DBC1 also binds to methyltransferase SUV39H1 and inhibits cellular H3K9 methylation (Li et al., 2009).The role of DBC1 in tumorigenesis is more puzzling. Dbc1 is deleted in several types of cancer, and has been suggested to suppress tumor development (Hamaguchi et al., 2002; Kim et al., 2009; Di Marcotullio et al., 2011). DBC1 is also associated with good outcome in gastric cancer (Noguchi et al., 2014). But other studies have shown that DBC1 is overexpressed in breast cancer, gastric cancer and other tumor type, and is correlated with poor prognosis (Cha et al., 2009; Hiraike et al., 2010; Kang et al., 2012; Zhang et al., 2014). Downregulation of DBC1 inhibits the proliferation and invasive potential of gastric cancer cells (Bae et al., 2014). Because of these conflicting findings, DBC1 function in tumorigenesis remains unclear.Here we show that p53 level is decreased in DBC1-deficient cells and tissues. DBC1 binds to the N-terminus and DNA binding domain of p53, competing with MDM2 and stabilizing p53. Depletion of Dbc1 promotes tumorigenesis in mice.

INTRODUCTION

p53 is a pivotal tumor suppressor and one of the most mutated genes in human cancer, with p53 mutations observed in over half of all human cancers (Brady and Attardi, 2010; Vousden and Prives, 2009). p53 regulates multiple cellular functions, including apoptosis, cell cycle arrest, cell metabolism, and senescence. One of the major regulatory mechanisms of p53 is its ubiquitination by a ubiquitin E3 ligase MDM2 (murine double minute 2), which leads to p53 degradation under normal conditions (Haupt et al., 1997). Under stress conditions, such as DNA damage, viral infection and oncogene activation, p53 is quickly accumulated, resulting in its activation. Several mechanisms are responsible for p53 induction, including MDM2 inactivation, interrupted interaction between p53 and MDM2, and activation of ubiquitin proteases (Dai and Gu, 2010).

The Dbc1 gene was initially discovered as a gene deleted in human chromosome 8p21 in breast cancer (Hamaguchi et al., 2002). DBC1 is composed of a leucine zipper motif at the amino-terminus, coiled coil domain at the carboxy-terminus, a nuclear localization signal, an EF hand domain and a Nudix domain (Anantharaman and Aravind, 2008; Kim et al., 2008). DBC1 is processed into C-terminal p120 and p66 fragments, which relocate from nucleus to mitochondria and enhance apoptotic signaling with tumor necrosis factor-α (TNFα) treatment in HeLa cells (Sundararajan et al., 2005). Several studies also suggest that DBC1 regulates hormone receptor activity. For instance, DBC1 activates retinoic acid receptor α and androgen receptor; and represses transcription activity of estrogen receptor β (Fu et al., 2009; Garapaty et al., 2009; Koyama et al., 2010). Furthermore, we and others have found that DBC1 negatively regulates SIRT1 activity through binding to its active site (Kim et al., 2008; Zhao et al., 2008). DNA damage and oxidative stress increase the DBC1-SIRT1 interaction, whereas PKA and AMPK induce dissociation of SIRT1 from DBC1 (Yuan et al., 2012; Nin et al., 2012). DBC1 also binds to methyltransferase SUV39H1 and inhibits cellular H3K9 methylation (Li et al., 2009).

The role of DBC1 in tumorigenesis is more puzzling. Dbc1 is deleted in several types of cancer, and has been suggested to suppress tumor development (Hamaguchi et al., 2002; Kim et al., 2009; Di Marcotullio et al., 2011). DBC1 is also associated with good outcome in gastric cancer (Noguchi et al., 2014). But other studies have shown that DBC1 is overexpressed in breast cancer, gastric cancer and other tumor type, and is correlated with poor prognosis (Cha et al., 2009; Hiraike et al., 2010; Kang et al., 2012; Zhang et al., 2014). Downregulation of DBC1 inhibits the proliferation and invasive potential of gastric cancer cells (Bae et al., 2014). Because of these conflicting findings, DBC1 function in tumorigenesis remains unclear.

Here we show that p53 level is decreased in DBC1-deficient cells and tissues. DBC1 binds to the N-terminus and DNA binding domain of p53, competing with MDM2 and stabilizing p53. Depletion of Dbc1 promotes tumorigenesis in mice.

RESULTSDBC1 loss promotes tumorigenesisTo test that DBC1 is a bona fide tumor suppressor in vivo, we generated Dbc1 knockout (KO) mice. Dbc1+/+, Dbc1+/− and Dbc1−/− mice were monitored for 24 months. Dbc1 KO mice were born in expected Mendelian ratios (Table S1). But compared with Dbc1+/+ mice, the disease free survival of Dbc1+/− and Dbc1−/− mice was dramatically decreased (Fig. 1A). Dbc1+/− and Dbc1−/− mice developed more tumors than Dbc1+/+ mice, including lymphomas, liver tumors, lung tumors, and teratomas (Fig. 1B–1C, and Fig. S1). Further analysis of these tumors showed that lymphomas and liver tumors are the primary tumors. In Dbc1−/− mice, liver tumors and lymphomas took up to16.95% and 16.95%, respectively. Lung tumors and teratomas took 8.47% and 3.39 % (Table S2). These results indicate that DBC1 is a tumor suppressor.Stabilization of p53 by DBC1In our investigation of DBC1 tumor suppressor function in vitro using Dbc1 KO cells, we unexpectedly found that p53 level decreased in Dbc1 KO cells under unstressed condition (Fig. 2A). In addition, p53 induction following DNA damage was compromised (Fig. 2A–2B, and Fig. S2A). To test whether depletion of DBC1 affect p53 mRNA level, we performed RT-PCR assay. We didn’t detect any significant difference of p53 mRNA level between wild-type (WT) and KO cells (Fig. 2C). To further confirm this result in human cells, we knocked down DBC1 in human primary cells and found that p53 levels also decreased when DBC1 was depleted (Fig. 2D). However, knockdown of DBC1 in cancer cell lines generated variable results, and some cell lines showed decreased p53 levels upon DBC1 depletion (e.g. A549, Fig. S2B) while some did not (e.g. U2OS, Fig. S2C). This is probably due to complicated genetic background in different cancer cell lines. We also tested p53 levels in vivo using Dbc1 KO mice, and found less p53 protein in the tissues of Dbc1−/− mice (Fig. 2E). These results suggest that DBC1 regulates p53 levels in vitro and in vivo.DBC1 prevents p53 degradation from proteasomeTo determine whether the half-life of p53 protein was affected by DBC1, we treated cells with cycloheximide and found that p53 was degraded much faster in Dbc1 KO cells (Fig. 3A and 3B). These results suggest that DBC1 regulates p53 stability. To explore the mechanism of stabilization of p53 by DBC1, we treated Dbc1+/+ and Dbc1−/− cells with MG132. There was no significant difference in p53 level after MG132 treatment in both cells (Fig. 3C), suggesting that DBC1 regulates p53 in a proteasome-dependent manner. We further found that ubiquitination signals from p53 immunoprecipitates were enhanced in the absence of DBC1 (Fig. 3D). To exclude the possibility that increased ubiquitination signals are caused by ubiquitination of p53 associated proteins rather than p53 itself, we transfected His-ub into cells and examined p53 ubiquitination under denatured conditions. This experiment further confirmed increased ubiquitination of p53 in Dbc1 KO cells (Fig. 3E).DBC1 competes with MDM2 in p53 bindingWe next studied how DBC1 regulated p53 stability and ubiquitination. DBC1 was reported to promote acetylation of p53 through inhibiting SIRT1 (Kim et al., 2008; Zhao et al., 2008). If DBC1 regulates p53 level through SIRT1, we would expect that inhibiting SIRT1 would rescue p53 loss in Dbc1−/− cells. To test this hypothesis, we knocked down SIRT1 in Dbc1+/+ and Dbc1−/− cells and found that depletion of SIRT1 did not result in increased p53 protein level (Fig. 4A and Fig. S3A). Instead, SIRT1 downregulation resulted a slightly decreased p53 levels. This result suggests that DBC1 stabilizes p53 through a SIRT1 independent manner.To study how DBC1 regulates p53, we next examined whether DBC1 itself interactes with p53. We found that DBC1 coimmunoprecipitated with p53 and vice versa (Fig. 4B and 4C). We failed to detect an interaction between DBC1 and MDM2 (Fig. 4C). To test whether the interaction between DBC1 and p53 is direct or not, we purified p53 and DBC1 protein and performed an in vitro binding assay. As shown in Figure 4D, p53 could directly pull down DBC1 under cell-free conditions, suggesting a direct interaction between DBC1 and p53. Next we mapped the DBC1-p53 interaction with a series of p53 deletion mutants, and found that DBC1 bound to N terminus of p53 (AA1–75) and DNA binding domain (DBD, AA76–320) (Fig. 4E and 4F). Interestingly, previous studies showed that MDM2 bound to the same region of p53 (Coutts et al., 2009). We also mapped the DBC1 interaction region with p53, and found that N terminus of DBC1 protein was responsible for the interaction with p53 (Fig. 4G), not the leucine zipper domain, which mediated the DBC1 interaction with SIRT1(Kim et al., 2008). These results led us to hypothesize that DBC1 stabilizes p53 by competing with MDM2 for p53 binding. We carried out an in vitro competition assay, and found that increased doses of DBC1 protein were able to compete off MDM2 in p53 binding (Fig. 4H). DBC1 competed with MDM2 for binding to both p53 N terminus and DBD domain (Fig. S3B and S3C). We also found enhanced MDM2-p53 interaction in cells in the absence of DBC1 (Fig. 4I). Furthermore, treating cells with the MDM2 inhibitor Nutlin, or depleting MDM2 by shRNA could reverse the reduction of p53 protein in DBC1-depleted cells (Fig. 4J and 4K). These results suggest that DBC1 regulates p53 by affecting the MDM2-p53 interaction. This function of DBC1 is independent of its role in SIRT1 regulation.Depletion of DBC1 promotes cell proliferation and cell transformation and favors multinucleationIt is reported that depletion of p53 induces multinucleated cells (Armit et al., 2002). We also noticed binucleated cells in DBC1 KO cells (Fig. S4A–S4C). To further explore the physiological function of the DBC1-p53 interaction, we utilized cell growth assay. We found that deletion of Dbc1 promoted cell proliferation (Fig. 5A, and 5B, and Fig. S4D). To test whether SIRT1 or p53 involved in DBC1 regulated cell proliferation, we reintroduced full length DBC1 and deletion mutants of DBC1 into Dbc1 KO cells. Reconstitution of full-length DBC1 and DBC1ΔLZ truncation reversed the increase of cell proliferation, but not DBC1Δ1–230 truncation (Fig. 5C and 5D). These results suggest that the p53 binding region of DBC1, rather than the SIRT1 binding region, is important for suppression of cell proliferation. Depletion of p53 causes spontaneous transformation (Hermannstadter et al., 2009; Lehnertz et al., 2011). To evaluate whether Dbc1-deficiency affects cell transformation, we performed colony formation assays. Loss of Dbc1 dramatically increased colony number (Fig. 5E and Fig. S4E). Reconstitution of full length DBC1 and DBC1ΔLZ, but not DBC1Δ1–230, blocked cell transformation (Fig. 5F and Fig. S4F). These results indicate that DBC1 inhibits cell proliferation and transformation, and maintains genome stability through the p53 interaction region.DBC1 suppresses tumorigenesis through p53, independent of SIRT1 In VivoBecause DBC1 suppresses SIRT1 activity, we further tested whether DBC1 suppresses tumorigenesis through its regulation of SIRT1 in vivo by crossing Sirt1−/− mice and Dbc1−/− mice. We had few Dbc1−/−;Sirt1−/− mice because of breeding difficulties. However, we found that Dbc1−/−;Sirt1+/− mice showed similar disease-free survival rate and tumor incidence as Dbc1−/−;Sirt1+/+ (Fig. 6A–6C and Table S3). These results, together with our in vitro cell transformation results, suggest that spontaneous tumor development in Dbc1−/− is unlikely through SIRT1. We also generated Dbc1×Trp53 double KO mice. Trp53−/− mice showed low survival rate and high frequency of tumorigenesis, and loss of Dbc1 did not further affect tumor incidence and mouse survival in Trp53−/− background (Fig. 6D–6E and Table S4). These results suggested that DBC1 inhibited tumorigenesis mainly through p53 but not SIRT1 in vivo.Stabilization of p53 mutant by DBC1The reported role of DBC1 in human cancer has been confusing. DBC1 is found to be downregulated or overexpressed and acts as a good or bad prognosis marker in human cancers, sometimes in the same type of cancer (Hamaguchi et al., 2002; Kim et al., 2009 ; Cha et al., 2009; Hiraike et al., 2010; Di Marcotullio et al., 2011; Noguchi et al., 2014; Bae et al., 2014; Zhang et al., 2014). To understand the discrepancy, we tested whether DBC1 affects mutant p53 function. MDA-MB-231 cells was reported to harbor a p53 R280K mutation (Olivier et al., 2002). Immunoprecipitation assay showed that DBC1 still interacted with mutant p53 (Fig. 7A), and mutant p53 levels were also decreased with loss of DBC1 (Fig. 7B) and MDM2 protein levels were comparable to those in cell lines with WT p53 (Fig. S5). To study whether DBC1 affects mutant p53 stability, we carried out ubiquitination assay. Similar to WT p53, increased ubiquitination of mutant p53 was observed after depletion of DBC1 (Fig. 7C). To explore whether DBC1 affects mutant p53 function, we performed cell proliferation assay and cell viability assays. Loss of DBC1 in cell lines with mutant p53 suppressed tumor cell proliferation (Fig. 7D) and impaired cell viability under stress (Fig. 7E and 7F). These phenotypes were opposite to what we obtained using cells with WT p53. These results suggest that DBC1 also stabilizes mutant p53 and promotes its function, and DBC1 could have tumor suppression or tumor promoting activity depending on genetic background of p53.

DBC1 loss promotes tumorigenesisTo test that DBC1 is a bona fide tumor suppressor in vivo, we generated Dbc1 knockout (KO) mice. Dbc1+/+, Dbc1+/− and Dbc1−/− mice were monitored for 24 months. Dbc1 KO mice were born in expected Mendelian ratios (Table S1). But compared with Dbc1+/+ mice, the disease free survival of Dbc1+/− and Dbc1−/− mice was dramatically decreased (Fig. 1A). Dbc1+/− and Dbc1−/− mice developed more tumors than Dbc1+/+ mice, including lymphomas, liver tumors, lung tumors, and teratomas (Fig. 1B–1C, and Fig. S1). Further analysis of these tumors showed that lymphomas and liver tumors are the primary tumors. In Dbc1−/− mice, liver tumors and lymphomas took up to16.95% and 16.95%, respectively. Lung tumors and teratomas took 8.47% and 3.39 % (Table S2). These results indicate that DBC1 is a tumor suppressor.

DBC1 loss promotes tumorigenesis

To test that DBC1 is a bona fide tumor suppressor in vivo, we generated Dbc1 knockout (KO) mice. Dbc1+/+, Dbc1+/− and Dbc1−/− mice were monitored for 24 months. Dbc1 KO mice were born in expected Mendelian ratios (Table S1). But compared with Dbc1+/+ mice, the disease free survival of Dbc1+/− and Dbc1−/− mice was dramatically decreased (Fig. 1A). Dbc1+/− and Dbc1−/− mice developed more tumors than Dbc1+/+ mice, including lymphomas, liver tumors, lung tumors, and teratomas (Fig. 1B–1C, and Fig. S1). Further analysis of these tumors showed that lymphomas and liver tumors are the primary tumors. In Dbc1−/− mice, liver tumors and lymphomas took up to16.95% and 16.95%, respectively. Lung tumors and teratomas took 8.47% and 3.39 % (Table S2). These results indicate that DBC1 is a tumor suppressor.

Stabilization of p53 by DBC1In our investigation of DBC1 tumor suppressor function in vitro using Dbc1 KO cells, we unexpectedly found that p53 level decreased in Dbc1 KO cells under unstressed condition (Fig. 2A). In addition, p53 induction following DNA damage was compromised (Fig. 2A–2B, and Fig. S2A). To test whether depletion of DBC1 affect p53 mRNA level, we performed RT-PCR assay. We didn’t detect any significant difference of p53 mRNA level between wild-type (WT) and KO cells (Fig. 2C). To further confirm this result in human cells, we knocked down DBC1 in human primary cells and found that p53 levels also decreased when DBC1 was depleted (Fig. 2D). However, knockdown of DBC1 in cancer cell lines generated variable results, and some cell lines showed decreased p53 levels upon DBC1 depletion (e.g. A549, Fig. S2B) while some did not (e.g. U2OS, Fig. S2C). This is probably due to complicated genetic background in different cancer cell lines. We also tested p53 levels in vivo using Dbc1 KO mice, and found less p53 protein in the tissues of Dbc1−/− mice (Fig. 2E). These results suggest that DBC1 regulates p53 levels in vitro and in vivo.

Stabilization of p53 by DBC1

In our investigation of DBC1 tumor suppressor function in vitro using Dbc1 KO cells, we unexpectedly found that p53 level decreased in Dbc1 KO cells under unstressed condition (Fig. 2A). In addition, p53 induction following DNA damage was compromised (Fig. 2A–2B, and Fig. S2A). To test whether depletion of DBC1 affect p53 mRNA level, we performed RT-PCR assay. We didn’t detect any significant difference of p53 mRNA level between wild-type (WT) and KO cells (Fig. 2C). To further confirm this result in human cells, we knocked down DBC1 in human primary cells and found that p53 levels also decreased when DBC1 was depleted (Fig. 2D). However, knockdown of DBC1 in cancer cell lines generated variable results, and some cell lines showed decreased p53 levels upon DBC1 depletion (e.g. A549, Fig. S2B) while some did not (e.g. U2OS, Fig. S2C). This is probably due to complicated genetic background in different cancer cell lines. We also tested p53 levels in vivo using Dbc1 KO mice, and found less p53 protein in the tissues of Dbc1−/− mice (Fig. 2E). These results suggest that DBC1 regulates p53 levels in vitro and in vivo.

DBC1 prevents p53 degradation from proteasomeTo determine whether the half-life of p53 protein was affected by DBC1, we treated cells with cycloheximide and found that p53 was degraded much faster in Dbc1 KO cells (Fig. 3A and 3B). These results suggest that DBC1 regulates p53 stability. To explore the mechanism of stabilization of p53 by DBC1, we treated Dbc1+/+ and Dbc1−/− cells with MG132. There was no significant difference in p53 level after MG132 treatment in both cells (Fig. 3C), suggesting that DBC1 regulates p53 in a proteasome-dependent manner. We further found that ubiquitination signals from p53 immunoprecipitates were enhanced in the absence of DBC1 (Fig. 3D). To exclude the possibility that increased ubiquitination signals are caused by ubiquitination of p53 associated proteins rather than p53 itself, we transfected His-ub into cells and examined p53 ubiquitination under denatured conditions. This experiment further confirmed increased ubiquitination of p53 in Dbc1 KO cells (Fig. 3E).

DBC1 prevents p53 degradation from proteasome

To determine whether the half-life of p53 protein was affected by DBC1, we treated cells with cycloheximide and found that p53 was degraded much faster in Dbc1 KO cells (Fig. 3A and 3B). These results suggest that DBC1 regulates p53 stability. To explore the mechanism of stabilization of p53 by DBC1, we treated Dbc1+/+ and Dbc1−/− cells with MG132. There was no significant difference in p53 level after MG132 treatment in both cells (Fig. 3C), suggesting that DBC1 regulates p53 in a proteasome-dependent manner. We further found that ubiquitination signals from p53 immunoprecipitates were enhanced in the absence of DBC1 (Fig. 3D). To exclude the possibility that increased ubiquitination signals are caused by ubiquitination of p53 associated proteins rather than p53 itself, we transfected His-ub into cells and examined p53 ubiquitination under denatured conditions. This experiment further confirmed increased ubiquitination of p53 in Dbc1 KO cells (Fig. 3E).

DBC1 competes with MDM2 in p53 bindingWe next studied how DBC1 regulated p53 stability and ubiquitination. DBC1 was reported to promote acetylation of p53 through inhibiting SIRT1 (Kim et al., 2008; Zhao et al., 2008). If DBC1 regulates p53 level through SIRT1, we would expect that inhibiting SIRT1 would rescue p53 loss in Dbc1−/− cells. To test this hypothesis, we knocked down SIRT1 in Dbc1+/+ and Dbc1−/− cells and found that depletion of SIRT1 did not result in increased p53 protein level (Fig. 4A and Fig. S3A). Instead, SIRT1 downregulation resulted a slightly decreased p53 levels. This result suggests that DBC1 stabilizes p53 through a SIRT1 independent manner.To study how DBC1 regulates p53, we next examined whether DBC1 itself interactes with p53. We found that DBC1 coimmunoprecipitated with p53 and vice versa (Fig. 4B and 4C). We failed to detect an interaction between DBC1 and MDM2 (Fig. 4C). To test whether the interaction between DBC1 and p53 is direct or not, we purified p53 and DBC1 protein and performed an in vitro binding assay. As shown in Figure 4D, p53 could directly pull down DBC1 under cell-free conditions, suggesting a direct interaction between DBC1 and p53. Next we mapped the DBC1-p53 interaction with a series of p53 deletion mutants, and found that DBC1 bound to N terminus of p53 (AA1–75) and DNA binding domain (DBD, AA76–320) (Fig. 4E and 4F). Interestingly, previous studies showed that MDM2 bound to the same region of p53 (Coutts et al., 2009). We also mapped the DBC1 interaction region with p53, and found that N terminus of DBC1 protein was responsible for the interaction with p53 (Fig. 4G), not the leucine zipper domain, which mediated the DBC1 interaction with SIRT1(Kim et al., 2008). These results led us to hypothesize that DBC1 stabilizes p53 by competing with MDM2 for p53 binding. We carried out an in vitro competition assay, and found that increased doses of DBC1 protein were able to compete off MDM2 in p53 binding (Fig. 4H). DBC1 competed with MDM2 for binding to both p53 N terminus and DBD domain (Fig. S3B and S3C). We also found enhanced MDM2-p53 interaction in cells in the absence of DBC1 (Fig. 4I). Furthermore, treating cells with the MDM2 inhibitor Nutlin, or depleting MDM2 by shRNA could reverse the reduction of p53 protein in DBC1-depleted cells (Fig. 4J and 4K). These results suggest that DBC1 regulates p53 by affecting the MDM2-p53 interaction. This function of DBC1 is independent of its role in SIRT1 regulation.

DBC1 competes with MDM2 in p53 binding

We next studied how DBC1 regulated p53 stability and ubiquitination. DBC1 was reported to promote acetylation of p53 through inhibiting SIRT1 (Kim et al., 2008; Zhao et al., 2008). If DBC1 regulates p53 level through SIRT1, we would expect that inhibiting SIRT1 would rescue p53 loss in Dbc1−/− cells. To test this hypothesis, we knocked down SIRT1 in Dbc1+/+ and Dbc1−/− cells and found that depletion of SIRT1 did not result in increased p53 protein level (Fig. 4A and Fig. S3A). Instead, SIRT1 downregulation resulted a slightly decreased p53 levels. This result suggests that DBC1 stabilizes p53 through a SIRT1 independent manner.

To study how DBC1 regulates p53, we next examined whether DBC1 itself interactes with p53. We found that DBC1 coimmunoprecipitated with p53 and vice versa (Fig. 4B and 4C). We failed to detect an interaction between DBC1 and MDM2 (Fig. 4C). To test whether the interaction between DBC1 and p53 is direct or not, we purified p53 and DBC1 protein and performed an in vitro binding assay. As shown in Figure 4D, p53 could directly pull down DBC1 under cell-free conditions, suggesting a direct interaction between DBC1 and p53. Next we mapped the DBC1-p53 interaction with a series of p53 deletion mutants, and found that DBC1 bound to N terminus of p53 (AA1–75) and DNA binding domain (DBD, AA76–320) (Fig. 4E and 4F). Interestingly, previous studies showed that MDM2 bound to the same region of p53 (Coutts et al., 2009). We also mapped the DBC1 interaction region with p53, and found that N terminus of DBC1 protein was responsible for the interaction with p53 (Fig. 4G), not the leucine zipper domain, which mediated the DBC1 interaction with SIRT1(Kim et al., 2008). These results led us to hypothesize that DBC1 stabilizes p53 by competing with MDM2 for p53 binding. We carried out an in vitro competition assay, and found that increased doses of DBC1 protein were able to compete off MDM2 in p53 binding (Fig. 4H). DBC1 competed with MDM2 for binding to both p53 N terminus and DBD domain (Fig. S3B and S3C). We also found enhanced MDM2-p53 interaction in cells in the absence of DBC1 (Fig. 4I). Furthermore, treating cells with the MDM2 inhibitor Nutlin, or depleting MDM2 by shRNA could reverse the reduction of p53 protein in DBC1-depleted cells (Fig. 4J and 4K). These results suggest that DBC1 regulates p53 by affecting the MDM2-p53 interaction. This function of DBC1 is independent of its role in SIRT1 regulation.

Depletion of DBC1 promotes cell proliferation and cell transformation and favors multinucleationIt is reported that depletion of p53 induces multinucleated cells (Armit et al., 2002). We also noticed binucleated cells in DBC1 KO cells (Fig. S4A–S4C). To further explore the physiological function of the DBC1-p53 interaction, we utilized cell growth assay. We found that deletion of Dbc1 promoted cell proliferation (Fig. 5A, and 5B, and Fig. S4D). To test whether SIRT1 or p53 involved in DBC1 regulated cell proliferation, we reintroduced full length DBC1 and deletion mutants of DBC1 into Dbc1 KO cells. Reconstitution of full-length DBC1 and DBC1ΔLZ truncation reversed the increase of cell proliferation, but not DBC1Δ1–230 truncation (Fig. 5C and 5D). These results suggest that the p53 binding region of DBC1, rather than the SIRT1 binding region, is important for suppression of cell proliferation. Depletion of p53 causes spontaneous transformation (Hermannstadter et al., 2009; Lehnertz et al., 2011). To evaluate whether Dbc1-deficiency affects cell transformation, we performed colony formation assays. Loss of Dbc1 dramatically increased colony number (Fig. 5E and Fig. S4E). Reconstitution of full length DBC1 and DBC1ΔLZ, but not DBC1Δ1–230, blocked cell transformation (Fig. 5F and Fig. S4F). These results indicate that DBC1 inhibits cell proliferation and transformation, and maintains genome stability through the p53 interaction region.

Depletion of DBC1 promotes cell proliferation and cell transformation and favors multinucleation

It is reported that depletion of p53 induces multinucleated cells (Armit et al., 2002). We also noticed binucleated cells in DBC1 KO cells (Fig. S4A–S4C). To further explore the physiological function of the DBC1-p53 interaction, we utilized cell growth assay. We found that deletion of Dbc1 promoted cell proliferation (Fig. 5A, and 5B, and Fig. S4D). To test whether SIRT1 or p53 involved in DBC1 regulated cell proliferation, we reintroduced full length DBC1 and deletion mutants of DBC1 into Dbc1 KO cells. Reconstitution of full-length DBC1 and DBC1ΔLZ truncation reversed the increase of cell proliferation, but not DBC1Δ1–230 truncation (Fig. 5C and 5D). These results suggest that the p53 binding region of DBC1, rather than the SIRT1 binding region, is important for suppression of cell proliferation. Depletion of p53 causes spontaneous transformation (Hermannstadter et al., 2009; Lehnertz et al., 2011). To evaluate whether Dbc1-deficiency affects cell transformation, we performed colony formation assays. Loss of Dbc1 dramatically increased colony number (Fig. 5E and Fig. S4E). Reconstitution of full length DBC1 and DBC1ΔLZ, but not DBC1Δ1–230, blocked cell transformation (Fig. 5F and Fig. S4F). These results indicate that DBC1 inhibits cell proliferation and transformation, and maintains genome stability through the p53 interaction region.

DBC1 suppresses tumorigenesis through p53, independent of SIRT1 In VivoBecause DBC1 suppresses SIRT1 activity, we further tested whether DBC1 suppresses tumorigenesis through its regulation of SIRT1 in vivo by crossing Sirt1−/− mice and Dbc1−/− mice. We had few Dbc1−/−;Sirt1−/− mice because of breeding difficulties. However, we found that Dbc1−/−;Sirt1+/− mice showed similar disease-free survival rate and tumor incidence as Dbc1−/−;Sirt1+/+ (Fig. 6A–6C and Table S3). These results, together with our in vitro cell transformation results, suggest that spontaneous tumor development in Dbc1−/− is unlikely through SIRT1. We also generated Dbc1×Trp53 double KO mice. Trp53−/− mice showed low survival rate and high frequency of tumorigenesis, and loss of Dbc1 did not further affect tumor incidence and mouse survival in Trp53−/− background (Fig. 6D–6E and Table S4). These results suggested that DBC1 inhibited tumorigenesis mainly through p53 but not SIRT1 in vivo.

DBC1 suppresses tumorigenesis through p53, independent of SIRT1 In Vivo

Because DBC1 suppresses SIRT1 activity, we further tested whether DBC1 suppresses tumorigenesis through its regulation of SIRT1 in vivo by crossing Sirt1−/− mice and Dbc1−/− mice. We had few Dbc1−/−;Sirt1−/− mice because of breeding difficulties. However, we found that Dbc1−/−;Sirt1+/− mice showed similar disease-free survival rate and tumor incidence as Dbc1−/−;Sirt1+/+ (Fig. 6A–6C and Table S3). These results, together with our in vitro cell transformation results, suggest that spontaneous tumor development in Dbc1−/− is unlikely through SIRT1. We also generated Dbc1×Trp53 double KO mice. Trp53−/− mice showed low survival rate and high frequency of tumorigenesis, and loss of Dbc1 did not further affect tumor incidence and mouse survival in Trp53−/− background (Fig. 6D–6E and Table S4). These results suggested that DBC1 inhibited tumorigenesis mainly through p53 but not SIRT1 in vivo.

Stabilization of p53 mutant by DBC1The reported role of DBC1 in human cancer has been confusing. DBC1 is found to be downregulated or overexpressed and acts as a good or bad prognosis marker in human cancers, sometimes in the same type of cancer (Hamaguchi et al., 2002; Kim et al., 2009 ; Cha et al., 2009; Hiraike et al., 2010; Di Marcotullio et al., 2011; Noguchi et al., 2014; Bae et al., 2014; Zhang et al., 2014). To understand the discrepancy, we tested whether DBC1 affects mutant p53 function. MDA-MB-231 cells was reported to harbor a p53 R280K mutation (Olivier et al., 2002). Immunoprecipitation assay showed that DBC1 still interacted with mutant p53 (Fig. 7A), and mutant p53 levels were also decreased with loss of DBC1 (Fig. 7B) and MDM2 protein levels were comparable to those in cell lines with WT p53 (Fig. S5). To study whether DBC1 affects mutant p53 stability, we carried out ubiquitination assay. Similar to WT p53, increased ubiquitination of mutant p53 was observed after depletion of DBC1 (Fig. 7C). To explore whether DBC1 affects mutant p53 function, we performed cell proliferation assay and cell viability assays. Loss of DBC1 in cell lines with mutant p53 suppressed tumor cell proliferation (Fig. 7D) and impaired cell viability under stress (Fig. 7E and 7F). These phenotypes were opposite to what we obtained using cells with WT p53. These results suggest that DBC1 also stabilizes mutant p53 and promotes its function, and DBC1 could have tumor suppression or tumor promoting activity depending on genetic background of p53.

Stabilization of p53 mutant by DBC1

The reported role of DBC1 in human cancer has been confusing. DBC1 is found to be downregulated or overexpressed and acts as a good or bad prognosis marker in human cancers, sometimes in the same type of cancer (Hamaguchi et al., 2002; Kim et al., 2009 ; Cha et al., 2009; Hiraike et al., 2010; Di Marcotullio et al., 2011; Noguchi et al., 2014; Bae et al., 2014; Zhang et al., 2014). To understand the discrepancy, we tested whether DBC1 affects mutant p53 function. MDA-MB-231 cells was reported to harbor a p53 R280K mutation (Olivier et al., 2002). Immunoprecipitation assay showed that DBC1 still interacted with mutant p53 (Fig. 7A), and mutant p53 levels were also decreased with loss of DBC1 (Fig. 7B) and MDM2 protein levels were comparable to those in cell lines with WT p53 (Fig. S5). To study whether DBC1 affects mutant p53 stability, we carried out ubiquitination assay. Similar to WT p53, increased ubiquitination of mutant p53 was observed after depletion of DBC1 (Fig. 7C). To explore whether DBC1 affects mutant p53 function, we performed cell proliferation assay and cell viability assays. Loss of DBC1 in cell lines with mutant p53 suppressed tumor cell proliferation (Fig. 7D) and impaired cell viability under stress (Fig. 7E and 7F). These phenotypes were opposite to what we obtained using cells with WT p53. These results suggest that DBC1 also stabilizes mutant p53 and promotes its function, and DBC1 could have tumor suppression or tumor promoting activity depending on genetic background of p53.

DISCUSSIONBecause of the crucial role of p53 in inhibiting cancer, it has been intensively studied, including the mechanism of its stabilization and activation. p53 stability is predominantly regulated in a MDM2 dependent manner. The posttranslational modification of p53, such as phosphorylation, acetylation, methylation and deubiquination, affects the interaction between p53 and MDM2, and thus regulates p53 protein stability (Lavin and Gueven, 2006). p53 consists of several different domain: transactivation domain, DNA-binding domain, tetramerization domain and C-terminal regulatory domain (Bode and Dong, 2004). Some proteins compete with MDM2 binding to p53 transactivation domain and DBD domain, and thereby prevent p53 from ubiquitination. These proteins include Ing1b (Leung et al., 2002) and DBC1 identified in the current study. DBC1 binds to the p53 N terminus and DBD domain (Fig.4E and 4F). MDM2, the E3 ligase of p53 protein, also binds to these same regions (Coutts et al., 2009), suggesting the potential competition between DBC1 and MDM2, which is confirmed by the in vitro competition assay (Fig. 4H and Fig. S3B–C).p53 is the guardian of genome and Trp53 KO mice have high tumor incidence, with lymphomas accounting for up to 70% of the tumors, while the remaining tumors are sarcomas (Jacks et al., 1994). The tumor spectrum of Dbc1−/− mice is different from that of Trp53 KO mice, but shares similarity with that of Trp53+/− mice. Like Dbc1−/− mice, lymphomas, liver tumors, lung tumors were developed in Trp53+/− mice. It is possible that loss of DBC1 caused partial loss of p53 protein in Dbc1−/− mice, similar to Trp53+/− mice (Jacks et al., 1994). It is also possible that p53 is not the only protein to interact with DBC1; other binding partners may too affect the tumor incidence and spectrum. Similar differences in tumor spectrum have also been observed in other established p53 regulators, such as p19ARF. ARF−/− mice have lower incidence of lymphomas (25%) and primarily have poorly differentiated sarcomas (50%), with carcinomas and gliomas appearing in other mice (Kamijo et al., 1997; Kamijo et al., 1999).The function of DBC1 in human cancer has been controversial. It has been suggested to promote or suppress cancer cell growth in different studies. p53 status might be the cause of this complexity. The TP53 gene is the most commonly mutated gene in different types of cancer (Kandoth et al., 2013). Mutant p53 gains new functions, and promotes cancer cell survival, proliferation and migration (Muller and Vousden, 2014). In our study, we found that DBC1 suppressed cell proliferation and tumorigenesis in wild-type p53 background in vitro and in vivo. Interestingly, DBC1 also stabilized p53 R280K mutant in cancer cell, and enhanced cell proliferation and cell viability (Fig. 7D, 7E and 7F). These results may help shed light on the puzzle of conflicting DBC1 function in tumors. In the tumor development process, cancer cells accumulate mutations, such as p53, and these mutants acquire new functions to help tumor progression. A tumor suppression signaling pathway, such as the DBC1-p53 pathway, is thus hijacked, to promote tumor development. Therefore DBC1 could be a double edged sword through its role in regulating p53.Taken together, our studies establish for the first time that DBC1 is a tumor suppressor, identify a SIRT1 independent function of DBC1, and reveal a new regulatory mechanism of p53. These studies provide important insights into cancer etiology.

Because of the crucial role of p53 in inhibiting cancer, it has been intensively studied, including the mechanism of its stabilization and activation. p53 stability is predominantly regulated in a MDM2 dependent manner. The posttranslational modification of p53, such as phosphorylation, acetylation, methylation and deubiquination, affects the interaction between p53 and MDM2, and thus regulates p53 protein stability (Lavin and Gueven, 2006). p53 consists of several different domain: transactivation domain, DNA-binding domain, tetramerization domain and C-terminal regulatory domain (Bode and Dong, 2004). Some proteins compete with MDM2 binding to p53 transactivation domain and DBD domain, and thereby prevent p53 from ubiquitination. These proteins include Ing1b (Leung et al., 2002) and DBC1 identified in the current study. DBC1 binds to the p53 N terminus and DBD domain (Fig.4E and 4F). MDM2, the E3 ligase of p53 protein, also binds to these same regions (Coutts et al., 2009), suggesting the potential competition between DBC1 and MDM2, which is confirmed by the in vitro competition assay (Fig. 4H and Fig. S3B–C).

p53 is the guardian of genome and Trp53 KO mice have high tumor incidence, with lymphomas accounting for up to 70% of the tumors, while the remaining tumors are sarcomas (Jacks et al., 1994). The tumor spectrum of Dbc1−/− mice is different from that of Trp53 KO mice, but shares similarity with that of Trp53+/− mice. Like Dbc1−/− mice, lymphomas, liver tumors, lung tumors were developed in Trp53+/− mice. It is possible that loss of DBC1 caused partial loss of p53 protein in Dbc1−/− mice, similar to Trp53+/− mice (Jacks et al., 1994). It is also possible that p53 is not the only protein to interact with DBC1; other binding partners may too affect the tumor incidence and spectrum. Similar differences in tumor spectrum have also been observed in other established p53 regulators, such as p19ARF. ARF−/− mice have lower incidence of lymphomas (25%) and primarily have poorly differentiated sarcomas (50%), with carcinomas and gliomas appearing in other mice (Kamijo et al., 1997; Kamijo et al., 1999).

The function of DBC1 in human cancer has been controversial. It has been suggested to promote or suppress cancer cell growth in different studies. p53 status might be the cause of this complexity. The TP53 gene is the most commonly mutated gene in different types of cancer (Kandoth et al., 2013). Mutant p53 gains new functions, and promotes cancer cell survival, proliferation and migration (Muller and Vousden, 2014). In our study, we found that DBC1 suppressed cell proliferation and tumorigenesis in wild-type p53 background in vitro and in vivo. Interestingly, DBC1 also stabilized p53 R280K mutant in cancer cell, and enhanced cell proliferation and cell viability (Fig. 7D, 7E and 7F). These results may help shed light on the puzzle of conflicting DBC1 function in tumors. In the tumor development process, cancer cells accumulate mutations, such as p53, and these mutants acquire new functions to help tumor progression. A tumor suppression signaling pathway, such as the DBC1-p53 pathway, is thus hijacked, to promote tumor development. Therefore DBC1 could be a double edged sword through its role in regulating p53.

Taken together, our studies establish for the first time that DBC1 is a tumor suppressor, identify a SIRT1 independent function of DBC1, and reveal a new regulatory mechanism of p53. These studies provide important insights into cancer etiology.

EXPERIMENTAL PROCEDURESAnimal studiesAll mice in this study were housed in the Mayo Clinic animal breeding facility. All experimental protocols were approved by the Institutional Animal Care and Use Committee at Mayo Clinic, and all studies were performed according to the methods approved in the protocol A19812. Mice were maintained in a 12hr light/dark cycle, and fed ad libitum normal food. Dbc1−/− mice were generated as described in (Escande et al., 2010). Dbc1+/−mice crossed with Trp53+/− mice to breed Dbc1×Trp53 double KO mice. Dbc1+/−mice crossed with Sirt1+/− mice to breed Dbc1×Sirt1 double KO mice.Cell culture, plasmids, antibodies and transfection reagentsDbc1+/+ and Dbc1−/− MEF cells were cultured in DMEM supplemented with 15% FBS. WI38 and IMR90 cells were cultured in EMEM supplemented with 10% FBS. All these cell lines were kept in a humidified 37°C 5% CO2 5%O2 incubator. MDA-MB-231 cells were cultured in L-15 medium with 10% FBS and kept in a humidified 37°C incubator without CO2.DBC1-His plasmid and DBC1-HA deletion constructs were described in(Kim et al., 2008). p53 GST deletions were kindly provided by Dr. Sengupta (De et al., 2012). DBC1 antibody was described in (Kim et al., 2008). p53 antibodies (DO-1 and R19) were purchased from Santa Cruz and p53 antibody (1C12) was purchased from Cell Signaling Technology and. p21 antibody (C-19) and ubquitin antibody (P4D1) were purchased from Santa Cruz. BrdU antibody was purchased from BD Bioscience. MDM2 antibody were purchase from Calbiochem.Lipofectamine 2000 and 3000 from Invitrogen were used for transfection.RNA interferenceDBC1 shRNA sh1 (target sequence CCCATCTGTGACTTCCTAGAA), sh2 (target sequence CGGGTCTTCACTGGTATTGTT) and SIRT1 shRNA (target sequence GCCATGAAGTATGACAAAGAT) were purchased from Sigma. Lentivirus were made according to manufacturer’s protocol. DBC1 SiRNA (5′-CAGCUUGCAUGACUACUUUUU-3′) was purchased from Dharmacon.Western and Immunoprecipitation AssayCells were lysed with NETN buffer (20mM Tris-HCl, pH 8.0, 100mM NaCl, 1mM EDTA, 0.5% Nonidet P-40 with 50mM b-glycerophosphate,10mM NaF, and 1 mg/ml each of pepstatin A and aprotinin. After centrifugation, the supernatant was removed and incubated with antibody and protein A or protein G Sepharose beads (Amersham Biosciences) for 2 hours or overnight at 4°C. The samples were separated by SDS–PAGE following three washes with NETN buffer. Western blot were carried out following standard procedures.GST Pulldown AssayGST and p53 deletion GST fusion proteins were purified in a standard way and incubated with cell lysates. After three washes, the bound proteins were separated with SDS-PAGE and blotted with indicated antibodies. His-DBC1 protein was purified from 293T cells transfected with His-DBC1 plasmids by Ni-NTA Agarose. For in vitro binding assay, GST fusion proteins were incubated with His-DBC1, which was purified from transected 293Tcells. After washes, the complex were separated by SDS-PAGE and blotted with indicated antibodies.p53 Ubiquitination AssayCells were treated with 10 µM MG132 for 4 hours, and then cell were lysed with NETN buffer and immunoprecipitated with polyclonal p53 antibody. The immunocomplexes were analyzed by western blot.in vitro Competition AssayMDM2-His protein was purchase from Abcam. GST-p53 and p53 deletion fusion proteins were purified from E. coli BL-21 (DE3) (Promega). Protein amount was quantified by BCA kit (Promega). 1µg/ml MDM2 and 1µg/ml GST-p53 protein and increasing amounts of DBC1 (0, 2×, 10× relative ratio to MDM2) were incubated in binding buffer (25 mM Tris, pH 7.8, 200 mM NaCl, 1 mM EDTA, 1 mM DTT, 10% glycerol, 0.2% Nonidet P-40, fresh 1 mM PMSF) for 6 h at 4 °C. After washing, the bound proteins were eluted with SDS sample buffer and were separated by SDS-PAGE.Cell proliferation AssayCells were seeded in 96-well plates and then formazan production was determined by the CellTiter 96 AQueous Non- Radioactive Cell Proliferation Assay kit (Promega, Madison, WI, USA). Briefly, 20 µL of MTS is added to each well, 3 hours later, The absorbance at 490 nm was measured with a microplate reader.BrdU Incorporation AssayBrdU labeling assay was carried out as described in (Chen et al., 2010). Cells were kept in medium containing 10 µM BrdU (Calbiochem) for 30 min and then fixed in 4% paraformaldehyde. Cells were permeabilized with 0.5% Triton X-100 (Sigma) and DNA was denatured in 2 M HCl and then blocked with 1% goat serum in PBS. Alexa-488 labelled anti-BrdU (Invitorgen) was added. Then cells were stained with 4,6-diamidino-2-phenylindole (DAPI) to show the nuclei.RT-PCRTotal RNA was extracted with PARIS kit (Applied Biosystems) and was reverse transcribed with the Superscript First-Strand Synthesis System for RT-PCR (Invitrogen). Primers for mosue GAPDH and mouse p53 were described in (Heyer et al., 2000)Immunofluorescent StainingCells were seeded on coverslips and then fixed with 4% paraformaldehyde on ice, washed with PBS, and permeabilized for 10 min with 0.1% Triton X-100. Cells were blocked with 5% goat serum and then incubated with alpha tubulin for 1 hour at room temperature. After washing with PBS, Alexa-488-conjugated anti-mouse antibody (Jackson ImmunoResearch) was added and incubated for 30 min at room temperature. At last cells were counterstained with DAPI.HistologyTissues were removed and then fixed in formalin. Paraffin tissue sections were stained with hematoxylin and esosin (H&E) to evaluate histology following the standard protocol.Statistical AnalysisFor survival curve, log-rank test was used for statistical analysis.For the rest Comparisons were carried out with a two-tailed unpaired Student's t test (*indicates p < 0.05, ** indicates p < 0.01). Results are presented as mean ± standard error of mean (SEM).

EXPERIMENTAL PROCEDURES

Animal studiesAll mice in this study were housed in the Mayo Clinic animal breeding facility. All experimental protocols were approved by the Institutional Animal Care and Use Committee at Mayo Clinic, and all studies were performed according to the methods approved in the protocol A19812. Mice were maintained in a 12hr light/dark cycle, and fed ad libitum normal food. Dbc1−/− mice were generated as described in (Escande et al., 2010). Dbc1+/−mice crossed with Trp53+/− mice to breed Dbc1×Trp53 double KO mice. Dbc1+/−mice crossed with Sirt1+/− mice to breed Dbc1×Sirt1 double KO mice.

Animal studies

All mice in this study were housed in the Mayo Clinic animal breeding facility. All experimental protocols were approved by the Institutional Animal Care and Use Committee at Mayo Clinic, and all studies were performed according to the methods approved in the protocol A19812. Mice were maintained in a 12hr light/dark cycle, and fed ad libitum normal food. Dbc1−/− mice were generated as described in (Escande et al., 2010). Dbc1+/−mice crossed with Trp53+/− mice to breed Dbc1×Trp53 double KO mice. Dbc1+/−mice crossed with Sirt1+/− mice to breed Dbc1×Sirt1 double KO mice.

Cell culture, plasmids, antibodies and transfection reagentsDbc1+/+ and Dbc1−/− MEF cells were cultured in DMEM supplemented with 15% FBS. WI38 and IMR90 cells were cultured in EMEM supplemented with 10% FBS. All these cell lines were kept in a humidified 37°C 5% CO2 5%O2 incubator. MDA-MB-231 cells were cultured in L-15 medium with 10% FBS and kept in a humidified 37°C incubator without CO2.DBC1-His plasmid and DBC1-HA deletion constructs were described in(Kim et al., 2008). p53 GST deletions were kindly provided by Dr. Sengupta (De et al., 2012). DBC1 antibody was described in (Kim et al., 2008). p53 antibodies (DO-1 and R19) were purchased from Santa Cruz and p53 antibody (1C12) was purchased from Cell Signaling Technology and. p21 antibody (C-19) and ubquitin antibody (P4D1) were purchased from Santa Cruz. BrdU antibody was purchased from BD Bioscience. MDM2 antibody were purchase from Calbiochem.Lipofectamine 2000 and 3000 from Invitrogen were used for transfection.

Cell culture, plasmids, antibodies and transfection reagents

Dbc1+/+ and Dbc1−/− MEF cells were cultured in DMEM supplemented with 15% FBS. WI38 and IMR90 cells were cultured in EMEM supplemented with 10% FBS. All these cell lines were kept in a humidified 37°C 5% CO2 5%O2 incubator. MDA-MB-231 cells were cultured in L-15 medium with 10% FBS and kept in a humidified 37°C incubator without CO2.

DBC1-His plasmid and DBC1-HA deletion constructs were described in(Kim et al., 2008). p53 GST deletions were kindly provided by Dr. Sengupta (De et al., 2012). DBC1 antibody was described in (Kim et al., 2008). p53 antibodies (DO-1 and R19) were purchased from Santa Cruz and p53 antibody (1C12) was purchased from Cell Signaling Technology and. p21 antibody (C-19) and ubquitin antibody (P4D1) were purchased from Santa Cruz. BrdU antibody was purchased from BD Bioscience. MDM2 antibody were purchase from Calbiochem.

Lipofectamine 2000 and 3000 from Invitrogen were used for transfection.

RNA interferenceDBC1 shRNA sh1 (target sequence CCCATCTGTGACTTCCTAGAA), sh2 (target sequence CGGGTCTTCACTGGTATTGTT) and SIRT1 shRNA (target sequence GCCATGAAGTATGACAAAGAT) were purchased from Sigma. Lentivirus were made according to manufacturer’s protocol. DBC1 SiRNA (5′-CAGCUUGCAUGACUACUUUUU-3′) was purchased from Dharmacon.

RNA interference

DBC1 shRNA sh1 (target sequence CCCATCTGTGACTTCCTAGAA), sh2 (target sequence CGGGTCTTCACTGGTATTGTT) and SIRT1 shRNA (target sequence GCCATGAAGTATGACAAAGAT) were purchased from Sigma. Lentivirus were made according to manufacturer’s protocol. DBC1 SiRNA (5′-CAGCUUGCAUGACUACUUUUU-3′) was purchased from Dharmacon.

Western and Immunoprecipitation AssayCells were lysed with NETN buffer (20mM Tris-HCl, pH 8.0, 100mM NaCl, 1mM EDTA, 0.5% Nonidet P-40 with 50mM b-glycerophosphate,10mM NaF, and 1 mg/ml each of pepstatin A and aprotinin. After centrifugation, the supernatant was removed and incubated with antibody and protein A or protein G Sepharose beads (Amersham Biosciences) for 2 hours or overnight at 4°C. The samples were separated by SDS–PAGE following three washes with NETN buffer. Western blot were carried out following standard procedures.

Western and Immunoprecipitation Assay

Cells were lysed with NETN buffer (20mM Tris-HCl, pH 8.0, 100mM NaCl, 1mM EDTA, 0.5% Nonidet P-40 with 50mM b-glycerophosphate,10mM NaF, and 1 mg/ml each of pepstatin A and aprotinin. After centrifugation, the supernatant was removed and incubated with antibody and protein A or protein G Sepharose beads (Amersham Biosciences) for 2 hours or overnight at 4°C. The samples were separated by SDS–PAGE following three washes with NETN buffer. Western blot were carried out following standard procedures.

GST Pulldown AssayGST and p53 deletion GST fusion proteins were purified in a standard way and incubated with cell lysates. After three washes, the bound proteins were separated with SDS-PAGE and blotted with indicated antibodies. His-DBC1 protein was purified from 293T cells transfected with His-DBC1 plasmids by Ni-NTA Agarose. For in vitro binding assay, GST fusion proteins were incubated with His-DBC1, which was purified from transected 293Tcells. After washes, the complex were separated by SDS-PAGE and blotted with indicated antibodies.

GST Pulldown Assay

GST and p53 deletion GST fusion proteins were purified in a standard way and incubated with cell lysates. After three washes, the bound proteins were separated with SDS-PAGE and blotted with indicated antibodies. His-DBC1 protein was purified from 293T cells transfected with His-DBC1 plasmids by Ni-NTA Agarose. For in vitro binding assay, GST fusion proteins were incubated with His-DBC1, which was purified from transected 293Tcells. After washes, the complex were separated by SDS-PAGE and blotted with indicated antibodies.

p53 Ubiquitination AssayCells were treated with 10 µM MG132 for 4 hours, and then cell were lysed with NETN buffer and immunoprecipitated with polyclonal p53 antibody. The immunocomplexes were analyzed by western blot.

p53 Ubiquitination Assay

Cells were treated with 10 µM MG132 for 4 hours, and then cell were lysed with NETN buffer and immunoprecipitated with polyclonal p53 antibody. The immunocomplexes were analyzed by western blot.

in vitro Competition AssayMDM2-His protein was purchase from Abcam. GST-p53 and p53 deletion fusion proteins were purified from E. coli BL-21 (DE3) (Promega). Protein amount was quantified by BCA kit (Promega). 1µg/ml MDM2 and 1µg/ml GST-p53 protein and increasing amounts of DBC1 (0, 2×, 10× relative ratio to MDM2) were incubated in binding buffer (25 mM Tris, pH 7.8, 200 mM NaCl, 1 mM EDTA, 1 mM DTT, 10% glycerol, 0.2% Nonidet P-40, fresh 1 mM PMSF) for 6 h at 4 °C. After washing, the bound proteins were eluted with SDS sample buffer and were separated by SDS-PAGE.

in vitro Competition Assay

MDM2-His protein was purchase from Abcam. GST-p53 and p53 deletion fusion proteins were purified from E. coli BL-21 (DE3) (Promega). Protein amount was quantified by BCA kit (Promega). 1µg/ml MDM2 and 1µg/ml GST-p53 protein and increasing amounts of DBC1 (0, 2×, 10× relative ratio to MDM2) were incubated in binding buffer (25 mM Tris, pH 7.8, 200 mM NaCl, 1 mM EDTA, 1 mM DTT, 10% glycerol, 0.2% Nonidet P-40, fresh 1 mM PMSF) for 6 h at 4 °C. After washing, the bound proteins were eluted with SDS sample buffer and were separated by SDS-PAGE.

Cell proliferation AssayCells were seeded in 96-well plates and then formazan production was determined by the CellTiter 96 AQueous Non- Radioactive Cell Proliferation Assay kit (Promega, Madison, WI, USA). Briefly, 20 µL of MTS is added to each well, 3 hours later, The absorbance at 490 nm was measured with a microplate reader.

Cell proliferation Assay

Cells were seeded in 96-well plates and then formazan production was determined by the CellTiter 96 AQueous Non- Radioactive Cell Proliferation Assay kit (Promega, Madison, WI, USA). Briefly, 20 µL of MTS is added to each well, 3 hours later, The absorbance at 490 nm was measured with a microplate reader.

BrdU Incorporation AssayBrdU labeling assay was carried out as described in (Chen et al., 2010). Cells were kept in medium containing 10 µM BrdU (Calbiochem) for 30 min and then fixed in 4% paraformaldehyde. Cells were permeabilized with 0.5% Triton X-100 (Sigma) and DNA was denatured in 2 M HCl and then blocked with 1% goat serum in PBS. Alexa-488 labelled anti-BrdU (Invitorgen) was added. Then cells were stained with 4,6-diamidino-2-phenylindole (DAPI) to show the nuclei.

BrdU Incorporation Assay

BrdU labeling assay was carried out as described in (Chen et al., 2010). Cells were kept in medium containing 10 µM BrdU (Calbiochem) for 30 min and then fixed in 4% paraformaldehyde. Cells were permeabilized with 0.5% Triton X-100 (Sigma) and DNA was denatured in 2 M HCl and then blocked with 1% goat serum in PBS. Alexa-488 labelled anti-BrdU (Invitorgen) was added. Then cells were stained with 4,6-diamidino-2-phenylindole (DAPI) to show the nuclei.

RT-PCRTotal RNA was extracted with PARIS kit (Applied Biosystems) and was reverse transcribed with the Superscript First-Strand Synthesis System for RT-PCR (Invitrogen). Primers for mosue GAPDH and mouse p53 were described in (Heyer et al., 2000)

Total RNA was extracted with PARIS kit (Applied Biosystems) and was reverse transcribed with the Superscript First-Strand Synthesis System for RT-PCR (Invitrogen). Primers for mosue GAPDH and mouse p53 were described in (Heyer et al., 2000)

Immunofluorescent StainingCells were seeded on coverslips and then fixed with 4% paraformaldehyde on ice, washed with PBS, and permeabilized for 10 min with 0.1% Triton X-100. Cells were blocked with 5% goat serum and then incubated with alpha tubulin for 1 hour at room temperature. After washing with PBS, Alexa-488-conjugated anti-mouse antibody (Jackson ImmunoResearch) was added and incubated for 30 min at room temperature. At last cells were counterstained with DAPI.

Immunofluorescent Staining

Cells were seeded on coverslips and then fixed with 4% paraformaldehyde on ice, washed with PBS, and permeabilized for 10 min with 0.1% Triton X-100. Cells were blocked with 5% goat serum and then incubated with alpha tubulin for 1 hour at room temperature. After washing with PBS, Alexa-488-conjugated anti-mouse antibody (Jackson ImmunoResearch) was added and incubated for 30 min at room temperature. At last cells were counterstained with DAPI.

HistologyTissues were removed and then fixed in formalin. Paraffin tissue sections were stained with hematoxylin and esosin (H&E) to evaluate histology following the standard protocol.

Tissues were removed and then fixed in formalin. Paraffin tissue sections were stained with hematoxylin and esosin (H&E) to evaluate histology following the standard protocol.

Statistical AnalysisFor survival curve, log-rank test was used for statistical analysis.For the rest Comparisons were carried out with a two-tailed unpaired Student's t test (*indicates p < 0.05, ** indicates p < 0.01). Results are presented as mean ± standard error of mean (SEM).

Statistical Analysis

For survival curve, log-rank test was used for statistical analysis.

For the rest Comparisons were carried out with a two-tailed unpaired Student's t test (*indicates p < 0.05, ** indicates p < 0.01). Results are presented as mean ± standard error of mean (SEM).

Supplementary Material12

Supplementary Material
