---
pmid: '24965446'
title: Host factors that interact with the pestivirus N-terminal protease, Npro, are
  components of the ribonucleoprotein complex.
authors:
- Jefferson M
- Donaszi-Ivanov A
- Pollen S
- Dalmay T
- Saalbach G
- Powell PP
journal: J Virol
year: '2014'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC4178888
doi: 10.1128/JVI.00984-14
---

# Host factors that interact with the pestivirus N-terminal protease, Npro, are components of the ribonucleoprotein complex.
**Authors:** Jefferson M, Donaszi-Ivanov A, Pollen S, Dalmay T, Saalbach G, Powell PP
**Journal:** J Virol (2014)
**DOI:** [10.1128/JVI.00984-14](https://doi.org/10.1128/JVI.00984-14)
**PMC:** [PMC4178888](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4178888/)

## Abstract

1. J Virol. 2014 Sep;88(18):10340-53. doi: 10.1128/JVI.00984-14. Epub 2014 Jun
25.

Host factors that interact with the pestivirus N-terminal protease, Npro, are 
components of the ribonucleoprotein complex.

Jefferson M(1), Donaszi-Ivanov A(1), Pollen S(2), Dalmay T(2), Saalbach G(3), 
Powell PP(4).

Author information:
(1)Biomedical Research Centre, Norwich Medical School, University of East 
Anglia, Norwich, Norfolk, United Kingdom.
(2)Biological Sciences, University of East Anglia, Norwich, United Kingdom.
(3)John Innes Centre, Norwich Research Park, Colney, Norwich, United Kingdom.
(4)Biomedical Research Centre, Norwich Medical School, University of East 
Anglia, Norwich, Norfolk, United Kingdom p.powell@uea.ac.uk.

The viral N-terminal protease N(pro) of pestiviruses counteracts cellular 
antiviral defenses through inhibition of IRF3. Here we used mass spectrometry to 
identify a new role for N(pro) through its interaction with over 55 associated 
proteins, mainly ribosomal proteins and ribonucleoproteins, including RNA 
helicase A (DHX9), Y-box binding protein (YBX1), DDX3, DDX5, eIF3, IGF2BP1, 
multiple myeloma tumor protein 2, interleukin enhancer binding factor 3 (IEBP3), 
guanine nucleotide binding protein 3, and polyadenylate-binding protein 1 
(PABP-1). These are components of the translation machinery, ribonucleoprotein 
particles (RNPs), and stress granules. Significantly, we found that stress 
granule formation was inhibited in MDBK cells infected with a noncytopathic 
bovine viral diarrhea virus (BVDV) strain, Kyle. However, ribonucleoproteins 
binding to N(pro) did not inhibit these proteins from aggregating into stress 
granules. N(pro) interacted with YBX1 though its TRASH domain, since the mutant 
C112R protein with an inactive TRASH domain no longer redistributed to stress 
granules. Interestingly, RNA helicase A and La autoantigen relocated from a 
nuclear location to form cytoplasmic granules with N(pro). To address a proviral 
role for N(pro) in RNP granules, we investigated whether N(pro) affected RNA 
interference (RNAi), since interacting proteins are involved in RISC function 
during RNA silencing. Using glyceraldehyde-3-phosphate dehydrogenase (GAPDH) 
silencing with small interfering RNAs (siRNAs) followed by Northern blotting of 
GAPDH, expression of N(pro) had no effect on RNAi silencing activity, 
contrasting with other viral suppressors of interferon. We propose that N(pro) 
is involved with virus RNA translation in the cytoplasm for virus particle 
production, and when translation is inhibited following stress, it redistributes 
to the replication complex.
IMPORTANCE: Although the pestivirus N-terminal protease, N(pro), has been shown 
to have an important role in degrading IRF3 to prevent apoptosis and interferon 
production during infection, the function of this unique viral protease in the 
pestivirus life cycle remains to be elucidated. We used proteomic mass 
spectrometry to identify novel interacting proteins and have shown that N(pro) 
is present in ribosomal and ribonucleoprotein particles (RNPs), indicating a 
translational role in virus particle production. The virus itself can prevent 
stress granule assembly from these complexes, but this inhibition is not due to 
N(pro). A proviral role to subvert RNA silencing through binding of these host 
RNP proteins was not identified for this viral suppressor of interferon.

Copyright © 2014 Jefferson et al.

DOI: 10.1128/JVI.00984-14
PMCID: PMC4178888
PMID: 24965446 [Indexed for MEDLINE]

## Full Text

INTRODUCTIONPestiviruses are a group of small positive-stranded RNA viruses in the Flavivirus family that cause economically important diseases of farm animals and include bovine viral diarrhea virus (BVDV), classical swine fever virus (CSFV), and border disease virus (BDV) (1). The genome, of approximately 12 kb, encodes only 12 proteins, which are co- and posttranslationally processed from a single RNA into the N-terminal protease (Npro), capsid, E1, E2, p7, NS2, NS3, NS4A, NS4B, NS5A, and NS5B (2). Pestiviruses replicate in the cytoplasm with no nuclear component and mature in intracellular vesicles that are thought to arise from the endoplasmic reticulum (ER) and are released by exocytosis (3). Translation is initiated by an internal ribosome entry site (IRES), which binds the 40S ribosomal subunit and eIF3 to direct cap-independent translation, but translation does not require the translation initiation factors eIF4B and eIF4F (4). The N-terminal protease, Npro, is a 168-amino-acid autoprotease, unique to pestiviruses, with cysteine protease activity in a Glu22-His49-Cys69 triad that acts to cleave itself from the nascent polypeptide (5). Deletion of Npro has no effect on viral replication or virulence (6); however, interestingly, expression of Npro alone in cells has been shown to suppress the innate immune response by inhibiting apoptosis and interferon production (7, 8).The innate immune response to infection takes place through cellular recognition of double-stranded RNA replication intermediates by a family of RNA helicases (9, 10), including protein kinase R, which phosphorylates initiation factor eIF2 to arrest translation and promotes stress granule formation (11), RNA helicase A, which can pair with MAVS/IPS-1 (12) and also activate NF-κB (13), and importantly, RIG-I and MDA-5, which transduce the signal for interferon production (14, 15). Most if not all viruses encode proteins that block these innate responses to double-stranded RNA (dsRNA) (16). In the case of pestiviruses, we have previously found that in infected cells, IRF3, a central regulator of interferon transcription and initiator of apoptosis, is lost, which inhibits dsRNA-induced interferon alpha and beta expression (7, 17). The loss of IRF3 is due to a single viral protein, Npro, and direct interaction of Npro with IRF3 has been demonstrated by coimmunoprecipitation (17), although it is not known whether this involves recruitment of other cellular proteins. Unique to pestiviruses, the structure of Npro has been solved to 1.25-Å resolution (18) and contains two compact units, a protease domain and an interaction domain. The interaction domain contains a novel metal-binding TRASH motif at the C-terminal end, consisting of Cys-X21-Cys-X3-Cys at its C terminus, which coordinates a single zinc atom, and mutations at this site attenuate its properties. TRASH domains are abundant in prokaryotes but are also found in a few vertebrate proteins (19).To learn more about the function of Npro, in this study we have taken a proteomic approach to investigate cellular proteins that interact with it, using mass spectrometry and pull-down analysis. We found that this viral protein interacts directly with 40S and 60S ribosomal proteins and initiation factor eIF3a, dependent on the presence of zinc. Npro also bound to several RNA helicases and dsRNA binding proteins, including RNA helicase A (DHX9), YBX1, DDX3X, and DDX5. In fact, many of the interacting cellular proteins are components of cytoplasmic ribonucleoprotein particles (RNPs), responsible for controlling mRNA translation and which can be assembled into stress granules to control the rate of translational initiation or mRNA decay (20). Stress granule formation may also regulate virus replication, and some viruses may inhibit the formation of or disassemble these granules during infection (21). To investigate whether the interaction of Npro with ribonucleoproteins influences their function and localization, we examined whether cells expressing Npro or infected with BVDV and then subjected to oxidative stress were protected from stress granule formation. BVDV-infected cells did not form stress granules when exposed to oxidative stress; however, this was not due to Npro binding to ribonucleoproteins, since cells expressing Npro alone formed stress granules and Npro codistributed with stress granule proteins. The RISC complex is an important component of RNPs, and we asked if Npro had a functional role in inhibiting RNA interference, as has been shown for other viral inhibitors of interferon synthesis (22, 23).

INTRODUCTION

Pestiviruses are a group of small positive-stranded RNA viruses in the Flavivirus family that cause economically important diseases of farm animals and include bovine viral diarrhea virus (BVDV), classical swine fever virus (CSFV), and border disease virus (BDV) (1). The genome, of approximately 12 kb, encodes only 12 proteins, which are co- and posttranslationally processed from a single RNA into the N-terminal protease (Npro), capsid, E1, E2, p7, NS2, NS3, NS4A, NS4B, NS5A, and NS5B (2). Pestiviruses replicate in the cytoplasm with no nuclear component and mature in intracellular vesicles that are thought to arise from the endoplasmic reticulum (ER) and are released by exocytosis (3). Translation is initiated by an internal ribosome entry site (IRES), which binds the 40S ribosomal subunit and eIF3 to direct cap-independent translation, but translation does not require the translation initiation factors eIF4B and eIF4F (4). The N-terminal protease, Npro, is a 168-amino-acid autoprotease, unique to pestiviruses, with cysteine protease activity in a Glu22-His49-Cys69 triad that acts to cleave itself from the nascent polypeptide (5). Deletion of Npro has no effect on viral replication or virulence (6); however, interestingly, expression of Npro alone in cells has been shown to suppress the innate immune response by inhibiting apoptosis and interferon production (7, 8).

The innate immune response to infection takes place through cellular recognition of double-stranded RNA replication intermediates by a family of RNA helicases (9, 10), including protein kinase R, which phosphorylates initiation factor eIF2 to arrest translation and promotes stress granule formation (11), RNA helicase A, which can pair with MAVS/IPS-1 (12) and also activate NF-κB (13), and importantly, RIG-I and MDA-5, which transduce the signal for interferon production (14, 15). Most if not all viruses encode proteins that block these innate responses to double-stranded RNA (dsRNA) (16). In the case of pestiviruses, we have previously found that in infected cells, IRF3, a central regulator of interferon transcription and initiator of apoptosis, is lost, which inhibits dsRNA-induced interferon alpha and beta expression (7, 17). The loss of IRF3 is due to a single viral protein, Npro, and direct interaction of Npro with IRF3 has been demonstrated by coimmunoprecipitation (17), although it is not known whether this involves recruitment of other cellular proteins. Unique to pestiviruses, the structure of Npro has been solved to 1.25-Å resolution (18) and contains two compact units, a protease domain and an interaction domain. The interaction domain contains a novel metal-binding TRASH motif at the C-terminal end, consisting of Cys-X21-Cys-X3-Cys at its C terminus, which coordinates a single zinc atom, and mutations at this site attenuate its properties. TRASH domains are abundant in prokaryotes but are also found in a few vertebrate proteins (19).

To learn more about the function of Npro, in this study we have taken a proteomic approach to investigate cellular proteins that interact with it, using mass spectrometry and pull-down analysis. We found that this viral protein interacts directly with 40S and 60S ribosomal proteins and initiation factor eIF3a, dependent on the presence of zinc. Npro also bound to several RNA helicases and dsRNA binding proteins, including RNA helicase A (DHX9), YBX1, DDX3X, and DDX5. In fact, many of the interacting cellular proteins are components of cytoplasmic ribonucleoprotein particles (RNPs), responsible for controlling mRNA translation and which can be assembled into stress granules to control the rate of translational initiation or mRNA decay (20). Stress granule formation may also regulate virus replication, and some viruses may inhibit the formation of or disassemble these granules during infection (21). To investigate whether the interaction of Npro with ribonucleoproteins influences their function and localization, we examined whether cells expressing Npro or infected with BVDV and then subjected to oxidative stress were protected from stress granule formation. BVDV-infected cells did not form stress granules when exposed to oxidative stress; however, this was not due to Npro binding to ribonucleoproteins, since cells expressing Npro alone formed stress granules and Npro codistributed with stress granule proteins. The RISC complex is an important component of RNPs, and we asked if Npro had a functional role in inhibiting RNA interference, as has been shown for other viral inhibitors of interferon synthesis (22, 23).

MATERIALS AND METHODSReagents and virus infection.HEK-293 (human embryonic kidney) cells were maintained in Dulbecco's modified Eagle medium (DMEM)-Glutamax (Invitrogen) supplemented with 10% heat-inactivated fetal calf serum. Npro-mCherry was generated from pcDNA3 Npro (7) and cloning in frame into pcDNA3mCherry (Invitrogen). Npro-mCherry C112R and D136N point mutants were created using a QuikChange kit according to the manufacturer's instructions (Stratagene). mCherry-tagged TIA1, pEGFP-tagged RNA helicase A (RHA), V5-tagged YBX-1 V5, DDX1-V5, and DCP1a, and Myc-tagged DDX3 were variously transfected into cells with Fugene HD (Roche). Transfected cells were allowed 24 h to recover at 37°C before stimulation with 100 μM sodium arsenate (NaA). Plasmids used were obtained from the following sources: pDDX3-myc was from Andrew Bowie and Martina Schoder (24), pEGFP- RHA full length (amino acids [aa] 1 to 1270) and pEGFP-RHA R1163A with a mutation at aa 1163 were a gift from T. Nakajima, Department of Genome Science, Institute of Medical Science, St. Marianna University School of Medicine, Kanagawa, Japan (25). pDDX1-V5, pDCP1a-V5, and YBX-V5 were from S. Ishura, University of Tokyo (26). The mCherry-tagged construct TIA1 was from T. Eisinger-Mathason, University of Virginia, as described in reference 27. La-V5 was generated by T. Csorba (University of East Anglia [UEA], Norwich, United Kingdom). Npro antibody raised to peptide KTNKQKPMGVEEPVYDATGKPLFGDPS, which corresponds to amino acids 11 to 37 of the Alfort strain of CSFV, linked to keyhold limpet hemocyanin (KLH) and ovalbumin (OVA), was raised in rabbits as described in reference 7. Mouse anti-RHA antibody (ab54593) and rabbit anti-YBX1 (ab12148) were from Abcam. Alpha-actin antibodies were from Sigma. For virus infections, bovine MDBK cells were grown in minimal essential medium (MEM) and 10% BVDV-free medium with Glutamax (Invitrogen). Cells were seeded on coverslips in 24-well plates and infected with the BVDV Kyle noncytopathic strain of BVDV (from J. Brownlie, Royal Vet College, South Mimms, United Kingdom). After absorption of virus for 1 h, cells were fed with fresh medium and incubated overnight at 37°C before treatment or transfection with DNA.GST Pulldown analysis.Bacteria (Escherichia coli BL21) were transformed with plasmids encoding glutathione S-transferase (GST) or GST-Npro, cultured for 4 h, and then induced with 100 mM isopropyl-b-d-thiogalactopyranoside (IPTG) for 4 h. Bacteria were lysed with the Bug Buster Protein extraction reagent (Merck). Lysates were clarified by centrifugation, resuspended in low-salt pulldown buffer (LPD) (20 mM Tris-HCl [pH 7.6], 200 mM NaCl, 0.5% NP-40, 0.5 mM dithiothreitol [DTT], and 0.4 mM phenylmethylsulfonyl fluoride [PMSF]) and 1 mg/ml protease inhibitors, and added to 3 ml of 50% glutathione–Sepharose bead slurry (Amersham Biosciences) overnight at 4°C. Beads were washed 7 times in a high-salt pulldown buffer (HPD) (500 mM NaCl and 1% Triton) to remove nonspecific proteins and reequilibrated with LPD with or without 1 mM zinc chloride to obtain beads binding either GST alone or GST Npro.HEK 293 cell were lysed in MPER buffer (Pierce), and supernatants were clarified by centrifugation. Lysates were precleared by adding glutathione beads overnight at 4°C, and beads were separated by centrifugation. Precleared supernatants were added to GST-Sepharose beads or GST-Npro-Sepharose beads as appropriate. Beads were washed seven times in LPD buffer, and proteins were analyzed by SDS-PAGE. For mass spectrometry analysis, proteins were eluted from the beads with 10 mM or 30 mM reduced glutathione.For V5 antibody pulldowns, HEK 293 cells were transfected with plasmids expressing protein with a V5 tag for 24 to 48 h, and cells were lysed in MPER with 1 mM ZnCl and incubated on ice for 30 min. In some experiments, a zinc chelator, N,N,N′,N′-Tetrakis (2-pyridylmethyl)ethylenediamine (TPEN), was used. Lysates were precleared with 50% slurry protein G-Sepharose for 1 h, before addition of anti-V5 antibody followed by protein G Sepharose beads. Beads were washed seven times in in LPD, resuspened in sample preparation buffer, and analyzed by SDS-PAGE followed by Western blotting.Identification of virus-host protein partners by mass spectrometry.Proteins were eluted from washed beads by adding 2-fold-concentrated SDS-gel sample loading buffer and heating to 80°C for 10 min. The supernatant was loaded onto a Novex gel (10% Bis-Tris SDS gel; Life Technologies/Invitrogen, Carlsbad, CA) and run about 1/3 of the length of the lane. The lanes were cut out and stained with InstantBlue (Expedeon Ltd., Harston, United Kingdom) in separate trays until bands were just visible. The area covering the bands was cut into 5 to 6 slices, which were washed, reduced, and alkylated and treated with trypsin according to standard procedures. Peptides were extracted with 5% formic acid–50% acetonitrile, dried down, and redissolved in 0.1% trifluoroacetic acid (TFA). For liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysis, a sample aliquot was applied via a nanoAcquity (Waters, Manchester, United Kingdom) ultraperformance liquid chromatography (UPLC) system running at a flow rate of 250 nl min−1 to an LTQ-Orbitrap mass spectrometer (Thermo Fisher, Waltham, MA). Peptides were trapped using a precolumn (Symmetry C18, 5 μm, 180 μm by 20 mm; Waters), which was then switched in-line to an analytical column (BEH C18, 1.7 μm, 75 μm by 250 mm; Waters) for separation. Peptides were eluted with a gradient of 3 to 40% acetonitrile in water–0.1% formic acid at a rate of 0.67% min−1. The column was connected to a 10-μm SilicaTip nanospray emitter (New Objective, Woburn, MA, USA) attached to a nanospray interface (Proxeon, Odense, Denmark) for infusion into the mass spectrometer. The mass spectrometer was operated in positive ion mode at a capillary temperature of 200°C. The source voltage and focusing voltages were tuned for the transmission of Met-Arg-Phe-Ala (MRFA) peptide (m/z 524) (Sigma-Aldrich, St. Louis, MO). Data-dependent analysis was carried out in orbitrap-ion trap parallel mode using collision-induced fragmentation (CID) on the 6 most abundant ions in each cycle. The orbitrap was run with a resolution of 30,000 over the MS range from m/z 350 to m/z 1,800 and an MS target of 106 and 1-s maximum scan time. Collision energy was 35, and an isolation width of 2 was used. Only monoisotopic 2+ and 3+ charged precursors were selected for MS2 fragmentation stage. The MS2 was triggered by a minimal signal of 1,000 with an automatic gain control target of 3 × 104 ions and 150-ms scan time using the chromatography function for peak apex detection. Dynamic exclusion was set to 1 count and 60-s exclusion with an exclusion mass window of ±20 ppm. MS scans were saved in profile mode, while MS/MS scans were saved in centroid mode.Raw files were processed using the software program MaxQuant, version 1.3.0.5 (28) (http://maxquant.org), to generate recalibrated peak list files, which were used for a database search using an in-house Mascot 2.4 server (Matrix Science Limited, London, United Kingdom). Mascot mgf files were generated from MaxQuant apl files using a suitable Perl script. Mascot searches were performed on the Sprot_sptrembl20121031.fasta database with taxonomy set to human using trypsin/P with 2 missed cleavages, 6-ppm precursor tolerance, 0.6-Da fragment tolerance, carbamidomethylation (C) as fixed, and oxidation (M) and acetylation (protein N terminus) as variable modifications. Mascot search results were imported and evaluated in the program Scaffold 3.6.1 (Proteome Software, Portland, OR, USA), resulting in a false-discovery rate of 0% for both peptides and proteins. For the detection of GST and CSFV Npro, the protein sequences of those proteins were added to a custom database with 1,000 random E. coli sequences (downloaded from uniprot.org) as a background, and the searches were performed in the same way as described above.The mass spectrometry proteomics data have been deposited in the ProteomeXchange Consortium (http://proteomecentral.proteomexchange.org) via the PRIDE partner repository (29) with the data set identifier PXD000115 and doi number 10.6019/PXD000115 (PRIDE accession numbers 28015 to 28026).Immunoblotting.Proteins were separated by SDS-PAGE (10 to 15% acrylamide) and transferred to polyvinylidene difluoride (PVDF) membranes (0.45-um transfer membrane; Thermo Scientific). Membranes were blocked with 5% (wt/vol) dried skimmed milk in Tris-buffered saline (TBS) containing 0.5% Tween 20. Membranes were probed with antibodies to YBX, V5, Npro, and RNA helicase A. Proteins were detected with IRDye-labeled secondary antibodies (926-322 and 926-680; Li-Cor Biosciences) at a 1:10,000 dilution or with horseradish peroxidase (HRP)-conjugated antibodies (111-035-003; Jackson Laboratory). Proteins detected by the labeled secondary antibodies were visualized on the Odyssey infrared system or enhanced chemiluminescence detection of horseradish peroxidase activity.Immunocytochemistry and fluorescence microscopy.HEK293 cells were blocked in goat serum gelatin quench and permeabilized with 0.2% Triton X-100. Primary antibodies were diluted into 0.2% Triton X-100 in goat gelatin quench and incubated with cells for 1 h at room temperature, and cells were washed three times in 0.1% Tween. Antibodies used were anti-RHA monoclonal antibody (MAb) (ab54593; Abcam), anti-V5 MAb (R960; Invitrogen), anti-Myc 9E10 MAb, rabbit anti-YB1 (ab12148; Abcam), and anti-ubiquitin FK2 MAb (BML-PW8810; Enzo).Caspase 3/7 apoptosis assay.Following cell treatment, caspase activity was measured in cell lysates using the caspase 3/7 Glo assay (Promega) in 96-well plates with the addition of the luminescent substrate Z-DEVD. Luciferase activity was measured using an Envision plate reader (PerkinElmer). Fold induction of caspase activity was obtained by dividing the activity in the treated samples by the activity obtained in the untreated cells for either uninfected cells or cells infected with BVDV.GAPDH silencing and Northern blot analysis.Glyceraldehyde-3-phosphate dehydrogenase (GAPDH)- or RHA-specific small interfering RNA (siRNA) (Dharmacon) at a concentration of 50 mM was transfected into cells for 24 h using the JetPrime reagent (Polyplus). Total RNA was isolated (GeneElute; Sigma), separated on formaldehyde gels, and blotted onto nitrocellulose membranes. The membrane was hybridized with 32P-labeled primer against GAPDH (GGCATGGACTGTGGTCATGAG) and 18S RNA (TTACAGGGCCTCGAAAGAGT).

MATERIALS AND METHODS

Reagents and virus infection.HEK-293 (human embryonic kidney) cells were maintained in Dulbecco's modified Eagle medium (DMEM)-Glutamax (Invitrogen) supplemented with 10% heat-inactivated fetal calf serum. Npro-mCherry was generated from pcDNA3 Npro (7) and cloning in frame into pcDNA3mCherry (Invitrogen). Npro-mCherry C112R and D136N point mutants were created using a QuikChange kit according to the manufacturer's instructions (Stratagene). mCherry-tagged TIA1, pEGFP-tagged RNA helicase A (RHA), V5-tagged YBX-1 V5, DDX1-V5, and DCP1a, and Myc-tagged DDX3 were variously transfected into cells with Fugene HD (Roche). Transfected cells were allowed 24 h to recover at 37°C before stimulation with 100 μM sodium arsenate (NaA). Plasmids used were obtained from the following sources: pDDX3-myc was from Andrew Bowie and Martina Schoder (24), pEGFP- RHA full length (amino acids [aa] 1 to 1270) and pEGFP-RHA R1163A with a mutation at aa 1163 were a gift from T. Nakajima, Department of Genome Science, Institute of Medical Science, St. Marianna University School of Medicine, Kanagawa, Japan (25). pDDX1-V5, pDCP1a-V5, and YBX-V5 were from S. Ishura, University of Tokyo (26). The mCherry-tagged construct TIA1 was from T. Eisinger-Mathason, University of Virginia, as described in reference 27. La-V5 was generated by T. Csorba (University of East Anglia [UEA], Norwich, United Kingdom). Npro antibody raised to peptide KTNKQKPMGVEEPVYDATGKPLFGDPS, which corresponds to amino acids 11 to 37 of the Alfort strain of CSFV, linked to keyhold limpet hemocyanin (KLH) and ovalbumin (OVA), was raised in rabbits as described in reference 7. Mouse anti-RHA antibody (ab54593) and rabbit anti-YBX1 (ab12148) were from Abcam. Alpha-actin antibodies were from Sigma. For virus infections, bovine MDBK cells were grown in minimal essential medium (MEM) and 10% BVDV-free medium with Glutamax (Invitrogen). Cells were seeded on coverslips in 24-well plates and infected with the BVDV Kyle noncytopathic strain of BVDV (from J. Brownlie, Royal Vet College, South Mimms, United Kingdom). After absorption of virus for 1 h, cells were fed with fresh medium and incubated overnight at 37°C before treatment or transfection with DNA.

Reagents and virus infection.

HEK-293 (human embryonic kidney) cells were maintained in Dulbecco's modified Eagle medium (DMEM)-Glutamax (Invitrogen) supplemented with 10% heat-inactivated fetal calf serum. Npro-mCherry was generated from pcDNA3 Npro (7) and cloning in frame into pcDNA3mCherry (Invitrogen). Npro-mCherry C112R and D136N point mutants were created using a QuikChange kit according to the manufacturer's instructions (Stratagene). mCherry-tagged TIA1, pEGFP-tagged RNA helicase A (RHA), V5-tagged YBX-1 V5, DDX1-V5, and DCP1a, and Myc-tagged DDX3 were variously transfected into cells with Fugene HD (Roche). Transfected cells were allowed 24 h to recover at 37°C before stimulation with 100 μM sodium arsenate (NaA). Plasmids used were obtained from the following sources: pDDX3-myc was from Andrew Bowie and Martina Schoder (24), pEGFP- RHA full length (amino acids [aa] 1 to 1270) and pEGFP-RHA R1163A with a mutation at aa 1163 were a gift from T. Nakajima, Department of Genome Science, Institute of Medical Science, St. Marianna University School of Medicine, Kanagawa, Japan (25). pDDX1-V5, pDCP1a-V5, and YBX-V5 were from S. Ishura, University of Tokyo (26). The mCherry-tagged construct TIA1 was from T. Eisinger-Mathason, University of Virginia, as described in reference 27. La-V5 was generated by T. Csorba (University of East Anglia [UEA], Norwich, United Kingdom). Npro antibody raised to peptide KTNKQKPMGVEEPVYDATGKPLFGDPS, which corresponds to amino acids 11 to 37 of the Alfort strain of CSFV, linked to keyhold limpet hemocyanin (KLH) and ovalbumin (OVA), was raised in rabbits as described in reference 7. Mouse anti-RHA antibody (ab54593) and rabbit anti-YBX1 (ab12148) were from Abcam. Alpha-actin antibodies were from Sigma. For virus infections, bovine MDBK cells were grown in minimal essential medium (MEM) and 10% BVDV-free medium with Glutamax (Invitrogen). Cells were seeded on coverslips in 24-well plates and infected with the BVDV Kyle noncytopathic strain of BVDV (from J. Brownlie, Royal Vet College, South Mimms, United Kingdom). After absorption of virus for 1 h, cells were fed with fresh medium and incubated overnight at 37°C before treatment or transfection with DNA.

GST Pulldown analysis.Bacteria (Escherichia coli BL21) were transformed with plasmids encoding glutathione S-transferase (GST) or GST-Npro, cultured for 4 h, and then induced with 100 mM isopropyl-b-d-thiogalactopyranoside (IPTG) for 4 h. Bacteria were lysed with the Bug Buster Protein extraction reagent (Merck). Lysates were clarified by centrifugation, resuspended in low-salt pulldown buffer (LPD) (20 mM Tris-HCl [pH 7.6], 200 mM NaCl, 0.5% NP-40, 0.5 mM dithiothreitol [DTT], and 0.4 mM phenylmethylsulfonyl fluoride [PMSF]) and 1 mg/ml protease inhibitors, and added to 3 ml of 50% glutathione–Sepharose bead slurry (Amersham Biosciences) overnight at 4°C. Beads were washed 7 times in a high-salt pulldown buffer (HPD) (500 mM NaCl and 1% Triton) to remove nonspecific proteins and reequilibrated with LPD with or without 1 mM zinc chloride to obtain beads binding either GST alone or GST Npro.HEK 293 cell were lysed in MPER buffer (Pierce), and supernatants were clarified by centrifugation. Lysates were precleared by adding glutathione beads overnight at 4°C, and beads were separated by centrifugation. Precleared supernatants were added to GST-Sepharose beads or GST-Npro-Sepharose beads as appropriate. Beads were washed seven times in LPD buffer, and proteins were analyzed by SDS-PAGE. For mass spectrometry analysis, proteins were eluted from the beads with 10 mM or 30 mM reduced glutathione.For V5 antibody pulldowns, HEK 293 cells were transfected with plasmids expressing protein with a V5 tag for 24 to 48 h, and cells were lysed in MPER with 1 mM ZnCl and incubated on ice for 30 min. In some experiments, a zinc chelator, N,N,N′,N′-Tetrakis (2-pyridylmethyl)ethylenediamine (TPEN), was used. Lysates were precleared with 50% slurry protein G-Sepharose for 1 h, before addition of anti-V5 antibody followed by protein G Sepharose beads. Beads were washed seven times in in LPD, resuspened in sample preparation buffer, and analyzed by SDS-PAGE followed by Western blotting.

GST Pulldown analysis.

Bacteria (Escherichia coli BL21) were transformed with plasmids encoding glutathione S-transferase (GST) or GST-Npro, cultured for 4 h, and then induced with 100 mM isopropyl-b-d-thiogalactopyranoside (IPTG) for 4 h. Bacteria were lysed with the Bug Buster Protein extraction reagent (Merck). Lysates were clarified by centrifugation, resuspended in low-salt pulldown buffer (LPD) (20 mM Tris-HCl [pH 7.6], 200 mM NaCl, 0.5% NP-40, 0.5 mM dithiothreitol [DTT], and 0.4 mM phenylmethylsulfonyl fluoride [PMSF]) and 1 mg/ml protease inhibitors, and added to 3 ml of 50% glutathione–Sepharose bead slurry (Amersham Biosciences) overnight at 4°C. Beads were washed 7 times in a high-salt pulldown buffer (HPD) (500 mM NaCl and 1% Triton) to remove nonspecific proteins and reequilibrated with LPD with or without 1 mM zinc chloride to obtain beads binding either GST alone or GST Npro.

HEK 293 cell were lysed in MPER buffer (Pierce), and supernatants were clarified by centrifugation. Lysates were precleared by adding glutathione beads overnight at 4°C, and beads were separated by centrifugation. Precleared supernatants were added to GST-Sepharose beads or GST-Npro-Sepharose beads as appropriate. Beads were washed seven times in LPD buffer, and proteins were analyzed by SDS-PAGE. For mass spectrometry analysis, proteins were eluted from the beads with 10 mM or 30 mM reduced glutathione.

For V5 antibody pulldowns, HEK 293 cells were transfected with plasmids expressing protein with a V5 tag for 24 to 48 h, and cells were lysed in MPER with 1 mM ZnCl and incubated on ice for 30 min. In some experiments, a zinc chelator, N,N,N′,N′-Tetrakis (2-pyridylmethyl)ethylenediamine (TPEN), was used. Lysates were precleared with 50% slurry protein G-Sepharose for 1 h, before addition of anti-V5 antibody followed by protein G Sepharose beads. Beads were washed seven times in in LPD, resuspened in sample preparation buffer, and analyzed by SDS-PAGE followed by Western blotting.

Identification of virus-host protein partners by mass spectrometry.Proteins were eluted from washed beads by adding 2-fold-concentrated SDS-gel sample loading buffer and heating to 80°C for 10 min. The supernatant was loaded onto a Novex gel (10% Bis-Tris SDS gel; Life Technologies/Invitrogen, Carlsbad, CA) and run about 1/3 of the length of the lane. The lanes were cut out and stained with InstantBlue (Expedeon Ltd., Harston, United Kingdom) in separate trays until bands were just visible. The area covering the bands was cut into 5 to 6 slices, which were washed, reduced, and alkylated and treated with trypsin according to standard procedures. Peptides were extracted with 5% formic acid–50% acetonitrile, dried down, and redissolved in 0.1% trifluoroacetic acid (TFA). For liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysis, a sample aliquot was applied via a nanoAcquity (Waters, Manchester, United Kingdom) ultraperformance liquid chromatography (UPLC) system running at a flow rate of 250 nl min−1 to an LTQ-Orbitrap mass spectrometer (Thermo Fisher, Waltham, MA). Peptides were trapped using a precolumn (Symmetry C18, 5 μm, 180 μm by 20 mm; Waters), which was then switched in-line to an analytical column (BEH C18, 1.7 μm, 75 μm by 250 mm; Waters) for separation. Peptides were eluted with a gradient of 3 to 40% acetonitrile in water–0.1% formic acid at a rate of 0.67% min−1. The column was connected to a 10-μm SilicaTip nanospray emitter (New Objective, Woburn, MA, USA) attached to a nanospray interface (Proxeon, Odense, Denmark) for infusion into the mass spectrometer. The mass spectrometer was operated in positive ion mode at a capillary temperature of 200°C. The source voltage and focusing voltages were tuned for the transmission of Met-Arg-Phe-Ala (MRFA) peptide (m/z 524) (Sigma-Aldrich, St. Louis, MO). Data-dependent analysis was carried out in orbitrap-ion trap parallel mode using collision-induced fragmentation (CID) on the 6 most abundant ions in each cycle. The orbitrap was run with a resolution of 30,000 over the MS range from m/z 350 to m/z 1,800 and an MS target of 106 and 1-s maximum scan time. Collision energy was 35, and an isolation width of 2 was used. Only monoisotopic 2+ and 3+ charged precursors were selected for MS2 fragmentation stage. The MS2 was triggered by a minimal signal of 1,000 with an automatic gain control target of 3 × 104 ions and 150-ms scan time using the chromatography function for peak apex detection. Dynamic exclusion was set to 1 count and 60-s exclusion with an exclusion mass window of ±20 ppm. MS scans were saved in profile mode, while MS/MS scans were saved in centroid mode.Raw files were processed using the software program MaxQuant, version 1.3.0.5 (28) (http://maxquant.org), to generate recalibrated peak list files, which were used for a database search using an in-house Mascot 2.4 server (Matrix Science Limited, London, United Kingdom). Mascot mgf files were generated from MaxQuant apl files using a suitable Perl script. Mascot searches were performed on the Sprot_sptrembl20121031.fasta database with taxonomy set to human using trypsin/P with 2 missed cleavages, 6-ppm precursor tolerance, 0.6-Da fragment tolerance, carbamidomethylation (C) as fixed, and oxidation (M) and acetylation (protein N terminus) as variable modifications. Mascot search results were imported and evaluated in the program Scaffold 3.6.1 (Proteome Software, Portland, OR, USA), resulting in a false-discovery rate of 0% for both peptides and proteins. For the detection of GST and CSFV Npro, the protein sequences of those proteins were added to a custom database with 1,000 random E. coli sequences (downloaded from uniprot.org) as a background, and the searches were performed in the same way as described above.The mass spectrometry proteomics data have been deposited in the ProteomeXchange Consortium (http://proteomecentral.proteomexchange.org) via the PRIDE partner repository (29) with the data set identifier PXD000115 and doi number 10.6019/PXD000115 (PRIDE accession numbers 28015 to 28026).

Identification of virus-host protein partners by mass spectrometry.

Proteins were eluted from washed beads by adding 2-fold-concentrated SDS-gel sample loading buffer and heating to 80°C for 10 min. The supernatant was loaded onto a Novex gel (10% Bis-Tris SDS gel; Life Technologies/Invitrogen, Carlsbad, CA) and run about 1/3 of the length of the lane. The lanes were cut out and stained with InstantBlue (Expedeon Ltd., Harston, United Kingdom) in separate trays until bands were just visible. The area covering the bands was cut into 5 to 6 slices, which were washed, reduced, and alkylated and treated with trypsin according to standard procedures. Peptides were extracted with 5% formic acid–50% acetonitrile, dried down, and redissolved in 0.1% trifluoroacetic acid (TFA). For liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysis, a sample aliquot was applied via a nanoAcquity (Waters, Manchester, United Kingdom) ultraperformance liquid chromatography (UPLC) system running at a flow rate of 250 nl min−1 to an LTQ-Orbitrap mass spectrometer (Thermo Fisher, Waltham, MA). Peptides were trapped using a precolumn (Symmetry C18, 5 μm, 180 μm by 20 mm; Waters), which was then switched in-line to an analytical column (BEH C18, 1.7 μm, 75 μm by 250 mm; Waters) for separation. Peptides were eluted with a gradient of 3 to 40% acetonitrile in water–0.1% formic acid at a rate of 0.67% min−1. The column was connected to a 10-μm SilicaTip nanospray emitter (New Objective, Woburn, MA, USA) attached to a nanospray interface (Proxeon, Odense, Denmark) for infusion into the mass spectrometer. The mass spectrometer was operated in positive ion mode at a capillary temperature of 200°C. The source voltage and focusing voltages were tuned for the transmission of Met-Arg-Phe-Ala (MRFA) peptide (m/z 524) (Sigma-Aldrich, St. Louis, MO). Data-dependent analysis was carried out in orbitrap-ion trap parallel mode using collision-induced fragmentation (CID) on the 6 most abundant ions in each cycle. The orbitrap was run with a resolution of 30,000 over the MS range from m/z 350 to m/z 1,800 and an MS target of 106 and 1-s maximum scan time. Collision energy was 35, and an isolation width of 2 was used. Only monoisotopic 2+ and 3+ charged precursors were selected for MS2 fragmentation stage. The MS2 was triggered by a minimal signal of 1,000 with an automatic gain control target of 3 × 104 ions and 150-ms scan time using the chromatography function for peak apex detection. Dynamic exclusion was set to 1 count and 60-s exclusion with an exclusion mass window of ±20 ppm. MS scans were saved in profile mode, while MS/MS scans were saved in centroid mode.

Raw files were processed using the software program MaxQuant, version 1.3.0.5 (28) (http://maxquant.org), to generate recalibrated peak list files, which were used for a database search using an in-house Mascot 2.4 server (Matrix Science Limited, London, United Kingdom). Mascot mgf files were generated from MaxQuant apl files using a suitable Perl script. Mascot searches were performed on the Sprot_sptrembl20121031.fasta database with taxonomy set to human using trypsin/P with 2 missed cleavages, 6-ppm precursor tolerance, 0.6-Da fragment tolerance, carbamidomethylation (C) as fixed, and oxidation (M) and acetylation (protein N terminus) as variable modifications. Mascot search results were imported and evaluated in the program Scaffold 3.6.1 (Proteome Software, Portland, OR, USA), resulting in a false-discovery rate of 0% for both peptides and proteins. For the detection of GST and CSFV Npro, the protein sequences of those proteins were added to a custom database with 1,000 random E. coli sequences (downloaded from uniprot.org) as a background, and the searches were performed in the same way as described above.

The mass spectrometry proteomics data have been deposited in the ProteomeXchange Consortium (http://proteomecentral.proteomexchange.org) via the PRIDE partner repository (29) with the data set identifier PXD000115 and doi number 10.6019/PXD000115 (PRIDE accession numbers 28015 to 28026).

Immunoblotting.Proteins were separated by SDS-PAGE (10 to 15% acrylamide) and transferred to polyvinylidene difluoride (PVDF) membranes (0.45-um transfer membrane; Thermo Scientific). Membranes were blocked with 5% (wt/vol) dried skimmed milk in Tris-buffered saline (TBS) containing 0.5% Tween 20. Membranes were probed with antibodies to YBX, V5, Npro, and RNA helicase A. Proteins were detected with IRDye-labeled secondary antibodies (926-322 and 926-680; Li-Cor Biosciences) at a 1:10,000 dilution or with horseradish peroxidase (HRP)-conjugated antibodies (111-035-003; Jackson Laboratory). Proteins detected by the labeled secondary antibodies were visualized on the Odyssey infrared system or enhanced chemiluminescence detection of horseradish peroxidase activity.

Immunoblotting.

Proteins were separated by SDS-PAGE (10 to 15% acrylamide) and transferred to polyvinylidene difluoride (PVDF) membranes (0.45-um transfer membrane; Thermo Scientific). Membranes were blocked with 5% (wt/vol) dried skimmed milk in Tris-buffered saline (TBS) containing 0.5% Tween 20. Membranes were probed with antibodies to YBX, V5, Npro, and RNA helicase A. Proteins were detected with IRDye-labeled secondary antibodies (926-322 and 926-680; Li-Cor Biosciences) at a 1:10,000 dilution or with horseradish peroxidase (HRP)-conjugated antibodies (111-035-003; Jackson Laboratory). Proteins detected by the labeled secondary antibodies were visualized on the Odyssey infrared system or enhanced chemiluminescence detection of horseradish peroxidase activity.

Immunocytochemistry and fluorescence microscopy.HEK293 cells were blocked in goat serum gelatin quench and permeabilized with 0.2% Triton X-100. Primary antibodies were diluted into 0.2% Triton X-100 in goat gelatin quench and incubated with cells for 1 h at room temperature, and cells were washed three times in 0.1% Tween. Antibodies used were anti-RHA monoclonal antibody (MAb) (ab54593; Abcam), anti-V5 MAb (R960; Invitrogen), anti-Myc 9E10 MAb, rabbit anti-YB1 (ab12148; Abcam), and anti-ubiquitin FK2 MAb (BML-PW8810; Enzo).

Immunocytochemistry and fluorescence microscopy.

HEK293 cells were blocked in goat serum gelatin quench and permeabilized with 0.2% Triton X-100. Primary antibodies were diluted into 0.2% Triton X-100 in goat gelatin quench and incubated with cells for 1 h at room temperature, and cells were washed three times in 0.1% Tween. Antibodies used were anti-RHA monoclonal antibody (MAb) (ab54593; Abcam), anti-V5 MAb (R960; Invitrogen), anti-Myc 9E10 MAb, rabbit anti-YB1 (ab12148; Abcam), and anti-ubiquitin FK2 MAb (BML-PW8810; Enzo).

Caspase 3/7 apoptosis assay.Following cell treatment, caspase activity was measured in cell lysates using the caspase 3/7 Glo assay (Promega) in 96-well plates with the addition of the luminescent substrate Z-DEVD. Luciferase activity was measured using an Envision plate reader (PerkinElmer). Fold induction of caspase activity was obtained by dividing the activity in the treated samples by the activity obtained in the untreated cells for either uninfected cells or cells infected with BVDV.

Caspase 3/7 apoptosis assay.

Following cell treatment, caspase activity was measured in cell lysates using the caspase 3/7 Glo assay (Promega) in 96-well plates with the addition of the luminescent substrate Z-DEVD. Luciferase activity was measured using an Envision plate reader (PerkinElmer). Fold induction of caspase activity was obtained by dividing the activity in the treated samples by the activity obtained in the untreated cells for either uninfected cells or cells infected with BVDV.

GAPDH silencing and Northern blot analysis.Glyceraldehyde-3-phosphate dehydrogenase (GAPDH)- or RHA-specific small interfering RNA (siRNA) (Dharmacon) at a concentration of 50 mM was transfected into cells for 24 h using the JetPrime reagent (Polyplus). Total RNA was isolated (GeneElute; Sigma), separated on formaldehyde gels, and blotted onto nitrocellulose membranes. The membrane was hybridized with 32P-labeled primer against GAPDH (GGCATGGACTGTGGTCATGAG) and 18S RNA (TTACAGGGCCTCGAAAGAGT).

GAPDH silencing and Northern blot analysis.

Glyceraldehyde-3-phosphate dehydrogenase (GAPDH)- or RHA-specific small interfering RNA (siRNA) (Dharmacon) at a concentration of 50 mM was transfected into cells for 24 h using the JetPrime reagent (Polyplus). Total RNA was isolated (GeneElute; Sigma), separated on formaldehyde gels, and blotted onto nitrocellulose membranes. The membrane was hybridized with 32P-labeled primer against GAPDH (GGCATGGACTGTGGTCATGAG) and 18S RNA (TTACAGGGCCTCGAAAGAGT).

RESULTSHost proteins interacting with Npro are components of ribosomes and ribonucleoprotein particles.Orbitrap mass spectrometry was used to identify novel cellular proteins interacting with Npro from the classical swine fever virus strain Alfort. Purified recombinant GST-Npro protein was incubated with HEK 293 cell lysates that had either been untreated or transfected with synthetic dsRNA, poly(I·C), for 4 h both to induce expression of early-response genes that may be important following viral infection and to investigate the requirement of dsRNA for proteins to bind to Npro. Table 1 shows 45 proteins that were specifically eluted from Npro-GST compared to results with GST alone, with either a low (10 mM) or high (30 mM) glutathione concentration and with or without poly(I·C). The list is in order of the number of peptides found for each protein. (A full list is given in the PRIDE partner repository, with the data set identifier PXD000095 and doi number 10.6019/PXD000095 (PRIDE accession no. 28008). Interestingly, many of the hits were ribosomal proteins and dsRNA binding proteins, which form ribonucleoprotein particles in the cytoplasm and which are involved in RNA metabolism. Transfection of dsRNA poly(I·C) into cells for 4 h before lysis made very little difference in the proteins detected, indicating that binding to Npro was not dependent on dsRNA, normally present in cells only during viral infection and replication. In addition to ribosomal proteins, interacting proteins included Y-box binding protein (YBX-1), La autoantigen, RNA helicase A (DHX9), multiple myeloma tumor protein 2, interleukin enhancer binding factor 3 (IEBP3), guanine nucleotide binding protein 3, and polyadenylate binding protein 1 (PABP-1). These proteins are involved in RNA metabolism, including translation and mRNA degradation, and they are normally found distributed through the cytoplasm of cells (30). It may be that Npro interacts with one or two of these RNA binding proteins, and the other proteins are pulled down in the same complex. Surprisingly, we found no direct binding of Npro to the immune modulators IRF3, IRF7, Hax-1, or I-κBα, which have previously been described by yeast 2-hybrid interaction (17, 31–34) in cells of immune lineage. This may reflect differences between yeast 2-hybrid and protein pulldown mass spectrometry, that expression levels of these proteins are low in HEK293 cells compared to those in immune cells, or the fact that additional proteins are required. Our previous work has shown that Npro is able to bind and degrade IRF3 in HEK 293 cells (8).TABLE 1Cellular proteins associated with GST-tagged NproaPeptide abundance rankAccession no.Protein identifiedMol massb (kDa)Association with Npro at GSH concn (mM) of:1030No pI·CpI·CNo pI·CpI·C% IDcNo. of peptides% IDNo. of peptides% IDNo. of peptides% IDNo. of peptides1P09211Glutathione S-transferase23100710010100910092P6124740S ribosomal protein S3a3010061004100310043P1588040S ribosomal protein S23110031001100310034Q00839Heterogeneous nuclear ribonucleoprotein U9110061002100410045P67809Y-box-binding protein 1 (YBX-1)3610041004100510046P6242460S ribosomal protein L7a301003851100310057P19338Nucleolin7710041002100310048P6275060S ribosomal protein L23a1810041004100210039P3902360S ribosomal protein L3461003991100283110P6226340S ribosomal protein S1416100210011002100211P05455La autoantigen, Sjoegren syndrome type B antigen (SS-B)47100310021003100312Q5JR9440S ribosomal protein S82410029911001100313Q59FI9Ribosomal protein L1221100210021002100214P2637360S ribosomal protein L132410029911001100315P6224440S ribosomal protein S15a1510021002100299116P4678240S ribosomal protein S523100210021002100317P6227740S ribosomal protein S131710031002100299118Q59ES8Heterogeneous nuclear ribonucleoprotein M6510039301003100219P6208140S ribosomal protein S7221001460100199120P8373160S ribosomal protein L24181002990100321Q08211ATP-dependent RNA helicase A (RHA, DEAH box protein 9)1411002100483122P1862160S ribosomal protein L1721100210021002100223P6125460S ribosomal protein L26171001981990100124Q53HK9Ribosomal protein P03410018511001100225P1812460S ribosomal protein L7?851991991100226P6289960S ribosomal protein L3114100210021002100227P3901940S ribosomal protein S19161002871100228P38159Heterogeneous nuclear ribonucleoprotein G (hnRNP G)421002100283129Q9BU76Multiple myeloma tumor-associated protein 2291002851991100230Q12906Interleukin enhancer-binding factor 3 (TCP80)9510021002100231P8409860S ribosomal protein L19231002100160032P8388160S ribosomal protein L36a12981840100183133P6131360S ribosomal protein L152498133087199134P3657860S ribosomal protein L448851991100135P31943Heterogeneous nuclear ribonucleoprotein H (hnRNP H)49100385136048036Q9BVP2Guanine nucleotide-binding protein-like 3621001851100237P61978Heterogeneous nuclear ribonucleoprotein K (hnRNP K)518511002100238Q8NC51Plasminogen activator inhibitor 1 RNA-binding protein45851851100239P08579U2 small nuclear ribonucleoprotein B″ (SU2 snRNP B)25991100270040Q9Y3D928S ribosomal protein S23 (mitochondrial)22100185148041P11940Polyadenylate-binding protein 1 (PABP-1)71991100242P2763560S ribosomal protein L1025100283143O60832H/ACA ribonucleoprotein complex subunit 45860099144P22626Heterogeneous nuclear ribonucleoproteins A2/B13756099145Q59GL1Synaptotagmin binding, cytoplasmic RNA interacting protein variant601001aNpro interacts with ribosomal and ribonucleoproteins, components of ribonuclear particles (RNPs). Interaction of Npro with cellular proteins was identified by mass spectrometry analysis. Recombinant GST-tagged Npro was used to pull down cellular binding partners from HEK293 cells. Cells were either control (“no. pI·C”) or transfected with poly(I·C) (pI·C) (dsRNA) for 4 h before lysis. Beads were eluted with low (10 mM) or high (30 mM) glutathione (GSH). Proteins binding to GST alone were used for a control. The accession number and molecular mass for each protein are shown in order of abundance of peptides found.bMolecular mass.cID, identity.Npro is a metalloprotein coordinating a single zinc ion through a putative metal binding domain called a TRASH motif at Cys112-X21-Cys134-X3-Cys138 (19). This domain is thought to be required for IRF3 interaction and degradation (34). In order to investigate which proteins were interacting with the zinc binding TRASH domain in the C-terminal end of Npro, in a second experiment, the recombinant protein was bound to lysates either in the presence of zinc or in the presence of a chelator of zinc, TPEN. The interacting proteins are shown in Table 2 (deposited in PRIDE, accession no. 28009). In this experiment, a total of 55 cellular proteins were found to bind Npro (Table 2), of which 9 were also detectedin the group shown in Table 1 (numbers in parentheses in column 1 of Table 2). There were 28 additional proteins in the presence of zinc and only 2 extra proteins bound when zinc was absent (desmoglein and desmoplakin); 25 proteins bound irrespective of the presence of zinc. Interestingly, the proteins that required zinc were involved in RNA translation, and included signal recognition particle, RNA helicases DDX 5 and DHX 15, eukaryotic initiation factor eIF3i, and tRNA ligases. These are ribonucleoproteins, also present in polysomes, stress granules, and P bodies. For some proteins that bound Npro both in the presence and absence of zinc, the number of peptides increased in the presence of zinc, showing increased affinity and suggesting some interaction with the TRASH motif in Npro. Heterogeneous ribonucleoproteins hnRNP M, U, A/B/C, and H and G were found as cellular binding partners, and these proteins are involved in mRNA nuclear export, localization, translation, and stability, important in both the nucleus and the cytoplasm. In addition, many of these factors are also involved in posttranscriptional gene regulation.TABLE 2Mass spectrometry identification of cellular proteins binding to Npro with or without zincaPeptide abundance rankbAccession no.Protein identifiedMol massc (kDa)Binding to Npro with:No ZnZn present% bindingNo. of peptides% IDdNo. of peptides1 (21)Q08211ATP-dependent RNA helicase A; DEAH box protein 914110030100502P28161Glutathione S-transferase261001110093C7DJS2Glutathione S-transferase17100210024P52272Heterogeneous nuclear ribonucleoprotein M78100610065 (2)P6124740S ribosomal protein S3a30100110076P2339640S ribosomal protein S327100210077P54577Tyrosine-tRNA ligase5910098B5BTY4ATP-dependent RNA helicase DDX3X73100210069B1AHM1DEAD (Asp-Glu-Ala-Asp) box polypeptide 1773991100510Q9NZI8Insulin-like growth factor 2 mRNA-binding protein 1631002100511P12956Lupus Ku autoantigen protein p7070390100512 (38)Q8NC51Plasminogen activator inhibitor 1 RNA-binding protein45620100213P4678240S ribosomal protein S523781100514P6270140S ribosomal protein S430781100315Q9Y262Eukaryotic translation initiation factor 3 subunit L (eIF3l)67350100316Q92900ATP-dependent helicase RENT1124991100317 (41)P11940Polyadenylate-binding protein 1 (PABP-1)71250100318P61011Signal recognition particle 54-kDa protein; (SRP54)56180100219O76094Signal recognition particle 72-kDa protein (SRP72)75100320Q9Y285Phenylalanine-tRNA ligase alpha subunit58100321 (45)Q59GL1Synaptotagmin binding, cytoplasmic RNA interacting protein60100322Q0VAC0Heterogeneous nuclear ribonucleoprotein A134100299123B3KNR3Glutathione S-transferase1999199124 (19)P6208140S ribosomal protein S722981100225 (30)Q12906Interleukin enhancer-binding factor 3; TCP8095781100326Q13435Splicing factor 3B subunit 2100781100227Q9UHB9Signal recognition particle 68-kDa protein71100328P04844Ribophorin II (RPN-II)69100229P49458Signal recognition particle 9-kDa protein; SRP910100230Q9H6T3RNA polymerase II-associated protein 376100331 (11)P05455La autoantigen; Sjoegren syndrome type B antigen (SS-B)47100332Q0254360S ribosomal protein L18a21100333P37108Signal recognition particle 14-kDa protein15100234P07910Heterogeneous nuclear ribonucleoproteins C1/C234781100235P6291360S ribosomal protein L1120781100236P6226640S ribosomal protein S2316410100237P6227740S ribosomal protein S1317100338Q02413Desmoglein 1114100339Q6NVW7Importin subunit alpha58100240Q9BUJ2Heterogeneous nuclear ribonucleoprotein U-like protein 196100241P14923Desmoplakin 382100142P4677860S ribosomal protein L2119100143Q96EY7Pentatricopeptide repeat-containing protein 3 (mitochondrial)79100144P26640Valine-tRNA ligase14099145F8W7C660S ribosomal protein L1019100246O43143ATP-dependent RNA helicase DHX15 (DEAH box protein 15)91100247P17844ATP-dependent RNA helicase DDX5; DEAD box protein 569100248 (44)P22626Heterogeneous nuclear ribonucleoproteins A2/B137100249Q13347Eukaryotic translation initiation factor 3 (eIF-3-beta)37100250Q53GX7Threonyl-tRNA synthetase variant83100251Q99729Heterogeneous nuclear ribonucleoprotein A/B36100152Q53F35Acidic (leucine-rich) nuclear phosphoprotein 3229100153P6285140S ribosomal protein S251499154Q9Y5A9YTH domain family protein 2; high-glucose-regulated protein 86299155P1807760S ribosomal protein L35a13991aNpro binding to cellular proteins through TRASH domain by zinc chelation. Cellular proteins associated with Npro in the presence or absence or zinc, detected by Orbitrap mass spectrometry analysis.bThe accession number and molecular mass for each protein are shown in order of abundance of peptides found. Proteins also identified in Table 1 are indicated by numbers in parentheses.cMolecular mass.dID, identity.BVDV infection inhibits stress granule formation.Virus infection may trigger stress in cells. The ribonucleoproteins shown to interact with Npro described above are known to aggregate, after exposure of cells to stress, into RNA granules known as stress granules, which regulate initiation, termination, and decay of mRNAs (20). Since several viruses have been shown to actively inhibit stress granule assembly at certain times postinfection, possibly to allow synthesis of their own proteins (35), we asked whether pestivirus infection affects stress granule formation. Arsenate treatment is widely used for oxidative stress to induce stress granule formation (20), and in the next experiments, cells were treated with arsenate in the presence and absence of BVDV (Fig. 1A). TIA1, an RNA binding protein which nucleates stress granule formation, was used as a stress granule marker (36). TIA1 is not only a translational silencer but also a key protein in the prion-like aggregation of further mRNPs to stress granules. YBX1, shown above to interact with Npro, is a major component of cellular RNPs which accumulates in granules following cellular stress. MDBK cells expressing YBX1 or TIA1 alone and treated with sodium arsenate for 4 h showed YBX1 and TIA1 accumulation in stress granules, and cells died by apoptosis, seen as DNA fragmentation (Fig. 1Ai and iii). In cells infected with BVDV for 48 h, YBX1 and TIA1 remained cytoplasmic following sodium arsenate treatment for 4 h (Fig. 1Aii and iv), and they did not accumulate in stress granules or in virus replication centers. Furthermore, BVDV-infected cells were protected from apoptosis following sodium arsenate treatment. This experiment is shown using a quantitative caspase 3/7 assay in Fig. 1B, where there is a decrease in caspase activity after sodium arsenate treatment of BVDV-infected cells compared to that for uninfected cells.FIG 1(A) MDBK cells infected with BVDV are protected from stress-induced apoptosis. (i) Control MDBK cells transfected with YBX1-V5 and treated with NaA for 4 h. (ii) Cells infected with BVDV for 48 h postinfection (hpi), transfected with YBX1-V5 overnight, and treated with NaA for 4 h. YBX-V5 was detected with anti-V5 MAb and anti-mouse Alexa 488 secondary antibody. Virus was detected with V182 hyperimmune serum and anti-bovine cy5 secondary antibody. (iii) MDBK cells transfected with TIA1-mCherry plasmid and treated with NaA for 4 h. (TIA was pseudocolored green for consistency). (iv) MDBK cells infected with BVDV for 48 hpi, transfected with TIA1-mCherry, and treated with NaA for 4 h. Virus was detected with V182 hyperimmune serum and anti-bovine cy5 secondary antibody, and TIA1 was pseudocolored green. (B) BVDV infection decreases stress-induced caspase 3/7 activation. MDBK cells, either uninfected (−) or infected with (+) BVDV noncytopathic (ncp) Kyle for 48 h, were stressed with NaA for 4 h. The graph shows the fold increase in caspase activity normalized to that of control cells and significance with one -sided t test value, P = 0.00004.Npro interaction with YBX1 does not prevent formation of stress granules.We have previously shown that ectopic expression of the single viral protein Npro can inhibit apoptosis and interferon production through binding and degradation of IRF3 (8). Since we show here that Npro is multifunctional and also binds to ribonucleoproteins, such as YBX1, a major component of stress granules, and that virus infection inhibits stress granule formation, we thought that Npro was a good candidate for the individual protein responsible for stress granule inhibition. Fluorescently tagged Npro (Npro-GFP or Npro-mCherry) was expressed in cells. Npro distributed to the nucleus and cytoplasm (Fig. 2Ai) and subsequently redistributed to cytoplasmic granules following stress with sodium arsenate (Fig. 2Aii). These cells were protected from apoptosis, as we have described in detail in our previous work (8), indicating stress granule assembly in the absence of apoptosis activation. However, when Npro-mCherry was expressed in BVDV-infected cells, Npro relocated to an area associated with the viral replication complex (Fig. 2Aiii). In the next experiments, we focused on possible functional roles of Npro by investigating its binding to, localization with, and effect on host proteins identified in Tables 1 and 2, outside the context of viral infection and in the absence of the other viral proteins. First, we investigated whether Npro redistributed with YBX-1 in stressed cells. When coexpressed together, Npro and YBX1 were both located to the cytoplasm in unstressed cells (Fig. 2Bi). Cells expressing Npro-mCherry and YBX1 treated with sodium arsenate for 4 h showed redistribution of Npro to cytoplasmic granules containing YBX1 (Fig. 2Bii), indicating that expression of Npro itself does not inhibit the formation of stress granule complexes in cells. In cells expressing the mutant Npro C112R protein, perinuclear stress granules containing YBX1 were formed, but these granules no longer colocalized with Npro C112R, which was found in smaller cytoplasmic foci (Fig. 2Biii, insert). Taken together, these results indicate that wild-type Npro binds to YBX1 through its TRASH domain, since Npro with the C112R mutation, which disrupts the zinc binding TRASH domain, no longer codistributes with YBX1 to stress granules. It demonstrates that Npro relocation to the stress granule complex is attributable to its binding to host proteins, such as YBX1. During infection, however, we show that stress granule formation is inhibited and YBX1 remains cytoplasmic (Fig. 1A), whereas Npro redistributes to the replication complex after infected cells are stressed (Fig. 2A). This suggests that since YBX1 is not recruited to the replication complex during infection, it is not required for viral replication, but it may have an important function bound to Npro at sites of viral genome translation in the cytoplasm.FIG 2Interaction and colocalization of Npro with Y-box protein 1 (YBX-1) requires the zinc-binding TRASH motif. (A) Npro localization in BVDV-infected MDBK cells. (i) Control MDBK cells transfected with Npro-GFP. (ii) MDBK cells transfected with Npro-GFP and treated with NaA for 4 h. (iii) Cells infected with BVDV for 48 hpi, transfected with Npro-GFP for 24 h, and treated with NaA for 4 h. Virus was detected with V182 bovine hyperimmune serum and anti-bovine cy5 secondary antibody. (B) Npro codistributes with YBX1 to large cytoplasmic stress granules. Cells stably expressing Npro-mCherry and transfected with YBX-1-V5 were either untreated or treated with NaA for 4 h. YBX1 was detected using anti-V5 MAb and anti-mouse Alexa 488 secondary antibody, and DNA was stained with 4′,6-diamidino-2-phenylindole (DAPI) (blue). (i) Control untreated HeLa cells, (ii) Cells stably expressing Npro-mCherry, transfected with YBX1-V5 and treated with NaA for 4 h. (iii) Cells stably expressing mutant Npro C112R-mCherry and transfected with YBX1-V5 and treated with NaA for 4 h. (C) Protein pulldown of HeLa cell lysates with recombinant Npro shows interaction with endogenous cellular YBX1 requiring zinc. The GST and GST-Npro recombinant proteins were washed in zinc buffer or zinc chelator, TPEN. Glutathione-Sepharose beads were used to pull down GST-Npro and its cobinding proteins. Top panel, endogenous YBX1 detected in cell lysates at 36 kDa with an anti-YBX1 antibody; middle panel, GST and GST-Npro recombinant proteins detected using anti-GST antibody and anti-Npro, respectively; bottom panel, HeLa cell lysates incubated with GST and GST-Npro proteins were pulled down with glutathione-Sepharose beads, washed extensively, and separated by SDS-PAGE. Proteins were transferred to membranes and probed with rabbit YBX1 antibody, which was detected with anti-rabbit-HRP by enhanced chemiluminescence (ECL). (D) Interaction of Npro with YBX1 increases after stress. Cells stably expressing Npro-mCherry were transfected with YBX1-V5, and cells were untreated (−) or treated (+) with sodium arsenate for 4 h (NaA). Lysates were pulled down with anti-V5 antibody bound to Sepharose beads. Top panels, Western blots of cell lysates either untreated (−) or treated (+) with NaA to detect YBX-V5 with anti-V5 monoclonal antibody (left) or Npro with rabbit anti-Npro antibody (right) using anti-mouse or anti-rabbit HRP antibodies, respectively. Bottom panels, immunoprecipitation of proteins using an anti-V5 antibody bound to Sepharose beads, followed by transfer of proteins to membranes and Western blotting with anti-V5 (left panel) or anti-Npro (right panel) antibodies. (E) Npro and YBX accumulate following stress. (i) Lysates from control cells (−) or cells expressing Npro-mCherry (+ Npro), either untreated (−NaA) or stressed for 4 h (+NaA), were blotted with antibodies against Npro and actin. (ii) Lysates from control cells (−) or cells expressing Npro (+Npro) either untreated (−) or treated with NaA (+) for 4 h were blotted with anti-YBX1 and antiactin antibodies.The binding of Npro to YBX1 was confirmed biochemically by a coimmunoprecipitation assay using either recombinant Npro GST (Fig. 2C) or lysates from HEK 293 cells expressing Npro-mCherry (Fig. 2D). Cell lysates were passed over glutathione beads bound to GST or Npro GST that had been washed in zinc buffer or washed with a zinc chelator, TPEN (Fig. 2C). Proteins pulled down were separated by gel electrophoresis and blotted with an anti-YBX 1 antibody. YBX1 was pulled down with the Npro recombinant protein specifically in the presence of zinc and not when the zinc had been chelated. This confirmed that Npro interacts with YBX1 through its zinc-binding TRASH motif.In the next experiment, cells stably expressing Npro-mCherry were cotransfected with a plasmid encoding YBX1 tagged with V5 to investigate binding in cells (Fig. 2D). Cells were treated with sodium arsenate for 4 h to promote the formation of stress granules. Proteins pulled down with YBX1 using anti-V5 antibody were separated on a gel and probed with anti-Npro antibodies. Npro was detected bound to YBX1 in cells that had been stressed but not in unstressed cells (Fig. 2D). This may reflect the increased affinity or abundance of the complex bound together in stress granules in cells.In cells treated with sodium arsenate for 4 h, there was accumulation of the Npro protein (Fig. 2Ei). Oxidative stress also increased the levels of YBX1 (Fig. 2Eii, lanes 1 and 2). In the presence of Npro, increased levels of YBX1 were precipitated both with and without arsenate treatment (Fig. 2Eii, lanes 3 and 4). These results show that both induction of stress and expression of Npro led to the accumulation of YBX1. The requirement for YBX1 during BVDV infection was attempted by silencing of YBX1 in infected MDBK cells, but because of the extremely low rates of transfection of these cells and the high level of expression of YBX1, it was not possible. (Pestiviruses have a narrow tropism to their host species, and therefore gene knockdown experiments are not possible using knocked-out mouse embryonic fibroblasts [MEFs]).Colocalization of interacting host factors with Npro after stress.As an assay for Npro interaction with factors identified by mass spectrometry, oxidative stress was used to determine their codistribution to granules. We coexpressed Npro-mCherry with TIA1, DDX1, DDX3, and DCP1a. Figure 3A shows codistribution of these proteins in the cytoplasm and their redistribution with Npro to stress granules as an indication of cobinding. DDX1 was also found in smaller cytoplasmic foci. DDX1 and DDX3 are RNA helicases with roles in transcriptional and posttranscriptional RNA metabolism, including RNA splicing, translation, DNA repair, and transcription (10). The decapping enzyme DCP1a, a component of P bodies, which are related to stress granules but contain other proteins involved in mRNA decay (20), codistributed with Npro to stress granules (Fig. 3Aiv). Stress granules are complex structures made up of RNA-binding proteins, 40S ribosomal subunits, but the composition can vary (20). We have shown that DDX1 and DDX3 aggregate into granules which also comprise YBX1 and TIA1 (Fig. 3Bi to iii), whereas DCP1a formed a ring-like structure around the TIA1-containing granules (Fig. 3Biv), demonstrating a separate domain for this protein. The imaging experiments, together with the proteomic data, show that Npro interacts with important antiviral RNA binding proteins in the cytoplasm, required for translation. Pestivirus infection prevents stress granule aggregation by an unidentified mechanism, but expression of Npro alone does not. During infection, the interaction may have an effect on viral or host translation.FIG 3Pestivirus Npro forms a complex with ribonucleoproteins and redistributes to stress granules. (A) Npro redistribution with TIA1, DDX1, DDX3, and DCP1a following stress. Cells were either untreated (CON) or subjected to oxidative stress for 4 h (+NaA). Cell expressing Npro-GFP were cotransfected with TIA1-mCherry (i). Cells expressing Npro-mCherry were cotransfected with DDX1-V5 (ii), DDX3-Myc (iii), or DCP1a-V5 (iv), which were detected with anti-V5 or anti-Myc MAbs and secondary anti-mouse Alexa 488 antibodies. (B) Codistribution of YBX1 and TIA1-containing particles with RNA helicases DDX1 and DDX3. HeLa cells were transfected with plasmids encoding DDX1-V5 (i) or DDX3-Myc (ii) and treated with NaA for 4 h. Endogenous YBX1 was detected with rabbit anti-YBX1 and an anti-rabbit Alexa 594 secondary antibody, and DDX1 and DDX3 were detected with V5 and Myc MAbs, respectively, and an anti-mouse Alexa 488 secondary antibody. HeLa cells were cotransfected with a plasmid encoding TIA1-mCherry and YBX1-V5 (iii) or DCP1a-V5 (iv), detected using anti-V5 with anti-mouse Alexa 488 secondary antibody.RNA helicase A and La autoantigen redistribute with Npro from the nucleus to cytoplasmic stress granules.Another important protein shown to interact with Npro is RNA helicase A (RHA). RHA (also known as DEAH box protein 9 [DHX9]) binds dsRNA; it is a DExD/H box helicase with both DNA and RNA helicase activity. Its antiviral properties include binding to the RNA-induced silencing complex (RISC) to help load dsRNA onto Argonaute for microRNA (miRNA) production (37). To confirm interaction between RHA and Npro, cells were transfected with a plasmid encoding RHA tagged to a hemagglutinin (HA) epitope. Cell lysates were incubated with beads bound to GST or GST-Npro recombinant protein and washed as described. RHA was pulled down with beads containing GST Npro but not GST beads alone (Fig. 4A), showing specific binding of RHA by recombinant Npro. When HEK 293 cells stably expressing cDNA for GST Npro were cotransfected with a plasmid encoding RHA-HA, GST-Npro was pulled down by GST in cells overexpressing RHA but not in cells without RHA (Fig. 4B). This indicates an increased efficiency of GST binding and Npro pulldown in the presence of RHA, possibly due to conformational change. Next we investigated the localization of RHA in cells expressing Npro. HeLa cells were cotransfected with RHA-GFP and Npro-mCherry. In untreated cells, RHA localized to the nucleus (Fig. 4Ci, top left panels), and there was little staining in the cytoplasm, whereas Npro expression was both nuclear and cytoplasmic. When these cells were subjected to oxidative stress, RHA relocated from the nucleus to cytoplasmic stress granules (Fig. 4Ci, top right panels, +NaA), and these granules colocalized with Npro. Since pestiviruses replicate in the cytoplasm with no nuclear component, we used RHA-GFP carrying a mutation (R1163A) which blocks its nuclear translocation (25) to investigate the effect on RHA, which is expressed exclusively in the cytoplasm. In unstressed cells, the mutant RHA-GFP R1163A was seen only in the cytoplasm (Fig. 4Ci, bottom panels). Treatment with sodium arsenate led to large cytoplasmic stress granules that colocalized with Npro-mCherry, clearly indicating RHA binding in a complex to Npro in cytoplasmic stress granules (Fig. 4Ci, bottom panels).FIG 4RNA helicase A (DHX9) binds to Npro and redistributes from the nucleus to cytoplasmic stress granules following stress. (A) RHA tagged with HA was transfected into HEK293 cells, and lysates were incubated with GST or GST-Npro recombinant proteins. Cobinding proteins were pulled down with glutathione-Sepharose beads. Top panel, Western blot of recombinant protein with anti-GST antibody (RP); middle panel, Western blot of cell lysates expressing RHA with anti-HA antibody detected with anti-mouse HRP secondary antibody; bottom panel, Western blot of proteins pulled down with glutathione-Sepharose beads and detected with anti-HA antibody. (B) GST-Npro expressed alone (−) or with RHA-HA (+) in HEK293 cells was pulled down using glutathione-Sepharose beads from cells lysates. Top panel, Western blot of cell lysates with anti-Npro antibody; middle panel, Western blot of cell lysates with anti-HA antibody; bottom panel, Western blot of proteins pulled down with glutathione beads using anti-Npro antibody. (C) RNA helicase A (DHX9) redistributes from the nucleus to cytoplasmic stress granules with Npro following stress. (i) HeLa cells were cotransfected with Npro-mCherry and either RHA-GFP (RHA +Npro) or mutant RHA R1163A-GFP (R1163A +Npro). Cells were either untreated (CON) or subjected to oxidative stress (+NaA). (ii) MBDK cells were infected with BVDV for 48 h and transfected with either RHA-GFP or mutant RHA R1163A GFP for 24 h. Cells were either untreated (CON) or subjected to oxidative stress (+NaA). (iii) HeLa cells were cotransfected with Npro-mCherry and La autoantigen-GFP. Cells were either untreated (CON) or subjected to oxidative stress (+NaA).In order to investigate RHA relocalization in BVDV-infected cells, MDBK cells were infected with virus and transfected with RHA-GFP (Fig. 4Cii). In control BVDV-infected cells, RHA-GFP distributed to the nucleus. When cells were stressed, there was no change in the distribution of RHA-GFP, which remained in the nucleus (Fig. 4Cii, top right panel.) This confirms the results shown in Fig. 1, that stress granules are not formed when BVDV-infected cells are stressed and RHA, TIA1, and YBX1 do not redistribute into stress granules. Similarly, the RHA R1163A -GFP mutant remained cytoplasmic both before and after treatment with sodium arsenate and did not relocate to stress granules (Fig. 4Cii, lower panels).We have shown by mass spectrometry that La autoantigen interacted with Npro (Table 1). La autoantigen is an RNA-binding protein which binds several viral RNAs, including those of poliovirus and human immunodeficiency virus (HIV), to enhance their translation in vitro (38). In the next experiment, we showed that the La autoantigen also redistributes from the nucleus to cytoplasmic granules with Npro following induction of stress (Fig. 4Ciii). La-GFP was predominantly nuclear before stress, but after treatment it was seen in small cytoplasmic dots colocalizing with Npro (Fig. 4Ciii). Thus, we have identified components of ribonucleoparticles made up from both cytoplasmic and nuclear proteins that bind to Npro in cells and which are redistributed to a cytoplasmic complex together after stress.Pestivirus Npro does not suppress RNA interference in mammalian cells.RNA granules aggregating with Npro are involved in several different RNA metabolic processes, and some of the components, for example RNA helicase A, interact directly with the RNA interference (RNAi) pathway (37). Silencing of RHA has been shown to block the RNAi pathway, and RHA is important for loading short RNAs, including siRNAs and miRNAs, onto RISC (37). The interacting protein La autoantigen is an RNAi activator that promotes the release of cleaved mRNA from Ago2 and efficient RNAi, an antiviral defense (39). Since Npro is a viral suppressor of interferon synthesis and apoptosis, and other viral suppressors of interferon, such as Ebola vp35, vaccinia virus E3L, and influenza virus NS1, also block the RNAi pathway, we investigated whether Npro was a viral suppressor of RNA silencing (22, 23). In the next series of experiments, the GAPDH housekeeping gene was silenced using GAPDH-specific siRNAs in control or Npro-expressing cells or cells where RNA helicase A had been silenced (RHA siRNA). GAPDH RNA levels were analyzed by Northern blotting with a 32P-labeled GAPDH-specific primer (Fig. 5ii). GAPDH was efficiently knocked down by siRNAs in control cells (Fig. 5ii, left panel, lane 4). There was no difference in GAPDH knockdown between control and Npro mCherry-expressing cells (Fig. 5ii, middle panel, lane 4), showing that Npro does not suppress RNAi from exogenously added siRNAs. It has been previously reported that RNA helicase A inhibits RISC loading (37) and that it might be involved in RNAi, so we investigated whether silencing of RHA had an effect on the RNAi assay of GAPDH knockdown with GAPDH-specific siRNAs, as described above. We silenced RHA with RHA-specific siRNAs (Fig. 5i, right panel). RHA depletion shows some suppression of RNAi from externally added siRNAs (Fig. 5ii, right panel, lane 4), as has been reported previously (37).FIG 5Npro does not suppress RNA silencing.(i) Western blot of control HeLa cells (−), cells stably expressing Npro-mCherry (+Npro) (left panel), or HeLa cells silenced for RHA expression (RHA siRNA) (right panel). Npro was detected with a rabbit anti-Npro antibody and visualized with IRDye-labeled secondary antibodies. RHA was detected with mouse anti-RHA and visualized with HRP-labeled secondary antibody. (ii) Northern blot analysis of GAPDH mRNA levels following GAPDH silencing using a 32P-labeled GAPDH-specific primer. 1, control cells; 2, mock-treated cells; 3, nontargeting siRNA-treated cells; 4, GAPDH siRNA-treated cells. Control cells (left panel), cells expressing Npro (middle panel), and cells silenced for RHA (right panel) are shown. An 18S RNA blot is shown for equal loading.

Host proteins interacting with Npro are components of ribosomes and ribonucleoprotein particles.Orbitrap mass spectrometry was used to identify novel cellular proteins interacting with Npro from the classical swine fever virus strain Alfort. Purified recombinant GST-Npro protein was incubated with HEK 293 cell lysates that had either been untreated or transfected with synthetic dsRNA, poly(I·C), for 4 h both to induce expression of early-response genes that may be important following viral infection and to investigate the requirement of dsRNA for proteins to bind to Npro. Table 1 shows 45 proteins that were specifically eluted from Npro-GST compared to results with GST alone, with either a low (10 mM) or high (30 mM) glutathione concentration and with or without poly(I·C). The list is in order of the number of peptides found for each protein. (A full list is given in the PRIDE partner repository, with the data set identifier PXD000095 and doi number 10.6019/PXD000095 (PRIDE accession no. 28008). Interestingly, many of the hits were ribosomal proteins and dsRNA binding proteins, which form ribonucleoprotein particles in the cytoplasm and which are involved in RNA metabolism. Transfection of dsRNA poly(I·C) into cells for 4 h before lysis made very little difference in the proteins detected, indicating that binding to Npro was not dependent on dsRNA, normally present in cells only during viral infection and replication. In addition to ribosomal proteins, interacting proteins included Y-box binding protein (YBX-1), La autoantigen, RNA helicase A (DHX9), multiple myeloma tumor protein 2, interleukin enhancer binding factor 3 (IEBP3), guanine nucleotide binding protein 3, and polyadenylate binding protein 1 (PABP-1). These proteins are involved in RNA metabolism, including translation and mRNA degradation, and they are normally found distributed through the cytoplasm of cells (30). It may be that Npro interacts with one or two of these RNA binding proteins, and the other proteins are pulled down in the same complex. Surprisingly, we found no direct binding of Npro to the immune modulators IRF3, IRF7, Hax-1, or I-κBα, which have previously been described by yeast 2-hybrid interaction (17, 31–34) in cells of immune lineage. This may reflect differences between yeast 2-hybrid and protein pulldown mass spectrometry, that expression levels of these proteins are low in HEK293 cells compared to those in immune cells, or the fact that additional proteins are required. Our previous work has shown that Npro is able to bind and degrade IRF3 in HEK 293 cells (8).TABLE 1Cellular proteins associated with GST-tagged NproaPeptide abundance rankAccession no.Protein identifiedMol massb (kDa)Association with Npro at GSH concn (mM) of:1030No pI·CpI·CNo pI·CpI·C% IDcNo. of peptides% IDNo. of peptides% IDNo. of peptides% IDNo. of peptides1P09211Glutathione S-transferase23100710010100910092P6124740S ribosomal protein S3a3010061004100310043P1588040S ribosomal protein S23110031001100310034Q00839Heterogeneous nuclear ribonucleoprotein U9110061002100410045P67809Y-box-binding protein 1 (YBX-1)3610041004100510046P6242460S ribosomal protein L7a301003851100310057P19338Nucleolin7710041002100310048P6275060S ribosomal protein L23a1810041004100210039P3902360S ribosomal protein L3461003991100283110P6226340S ribosomal protein S1416100210011002100211P05455La autoantigen, Sjoegren syndrome type B antigen (SS-B)47100310021003100312Q5JR9440S ribosomal protein S82410029911001100313Q59FI9Ribosomal protein L1221100210021002100214P2637360S ribosomal protein L132410029911001100315P6224440S ribosomal protein S15a1510021002100299116P4678240S ribosomal protein S523100210021002100317P6227740S ribosomal protein S131710031002100299118Q59ES8Heterogeneous nuclear ribonucleoprotein M6510039301003100219P6208140S ribosomal protein S7221001460100199120P8373160S ribosomal protein L24181002990100321Q08211ATP-dependent RNA helicase A (RHA, DEAH box protein 9)1411002100483122P1862160S ribosomal protein L1721100210021002100223P6125460S ribosomal protein L26171001981990100124Q53HK9Ribosomal protein P03410018511001100225P1812460S ribosomal protein L7?851991991100226P6289960S ribosomal protein L3114100210021002100227P3901940S ribosomal protein S19161002871100228P38159Heterogeneous nuclear ribonucleoprotein G (hnRNP G)421002100283129Q9BU76Multiple myeloma tumor-associated protein 2291002851991100230Q12906Interleukin enhancer-binding factor 3 (TCP80)9510021002100231P8409860S ribosomal protein L19231002100160032P8388160S ribosomal protein L36a12981840100183133P6131360S ribosomal protein L152498133087199134P3657860S ribosomal protein L448851991100135P31943Heterogeneous nuclear ribonucleoprotein H (hnRNP H)49100385136048036Q9BVP2Guanine nucleotide-binding protein-like 3621001851100237P61978Heterogeneous nuclear ribonucleoprotein K (hnRNP K)518511002100238Q8NC51Plasminogen activator inhibitor 1 RNA-binding protein45851851100239P08579U2 small nuclear ribonucleoprotein B″ (SU2 snRNP B)25991100270040Q9Y3D928S ribosomal protein S23 (mitochondrial)22100185148041P11940Polyadenylate-binding protein 1 (PABP-1)71991100242P2763560S ribosomal protein L1025100283143O60832H/ACA ribonucleoprotein complex subunit 45860099144P22626Heterogeneous nuclear ribonucleoproteins A2/B13756099145Q59GL1Synaptotagmin binding, cytoplasmic RNA interacting protein variant601001aNpro interacts with ribosomal and ribonucleoproteins, components of ribonuclear particles (RNPs). Interaction of Npro with cellular proteins was identified by mass spectrometry analysis. Recombinant GST-tagged Npro was used to pull down cellular binding partners from HEK293 cells. Cells were either control (“no. pI·C”) or transfected with poly(I·C) (pI·C) (dsRNA) for 4 h before lysis. Beads were eluted with low (10 mM) or high (30 mM) glutathione (GSH). Proteins binding to GST alone were used for a control. The accession number and molecular mass for each protein are shown in order of abundance of peptides found.bMolecular mass.cID, identity.Npro is a metalloprotein coordinating a single zinc ion through a putative metal binding domain called a TRASH motif at Cys112-X21-Cys134-X3-Cys138 (19). This domain is thought to be required for IRF3 interaction and degradation (34). In order to investigate which proteins were interacting with the zinc binding TRASH domain in the C-terminal end of Npro, in a second experiment, the recombinant protein was bound to lysates either in the presence of zinc or in the presence of a chelator of zinc, TPEN. The interacting proteins are shown in Table 2 (deposited in PRIDE, accession no. 28009). In this experiment, a total of 55 cellular proteins were found to bind Npro (Table 2), of which 9 were also detectedin the group shown in Table 1 (numbers in parentheses in column 1 of Table 2). There were 28 additional proteins in the presence of zinc and only 2 extra proteins bound when zinc was absent (desmoglein and desmoplakin); 25 proteins bound irrespective of the presence of zinc. Interestingly, the proteins that required zinc were involved in RNA translation, and included signal recognition particle, RNA helicases DDX 5 and DHX 15, eukaryotic initiation factor eIF3i, and tRNA ligases. These are ribonucleoproteins, also present in polysomes, stress granules, and P bodies. For some proteins that bound Npro both in the presence and absence of zinc, the number of peptides increased in the presence of zinc, showing increased affinity and suggesting some interaction with the TRASH motif in Npro. Heterogeneous ribonucleoproteins hnRNP M, U, A/B/C, and H and G were found as cellular binding partners, and these proteins are involved in mRNA nuclear export, localization, translation, and stability, important in both the nucleus and the cytoplasm. In addition, many of these factors are also involved in posttranscriptional gene regulation.TABLE 2Mass spectrometry identification of cellular proteins binding to Npro with or without zincaPeptide abundance rankbAccession no.Protein identifiedMol massc (kDa)Binding to Npro with:No ZnZn present% bindingNo. of peptides% IDdNo. of peptides1 (21)Q08211ATP-dependent RNA helicase A; DEAH box protein 914110030100502P28161Glutathione S-transferase261001110093C7DJS2Glutathione S-transferase17100210024P52272Heterogeneous nuclear ribonucleoprotein M78100610065 (2)P6124740S ribosomal protein S3a30100110076P2339640S ribosomal protein S327100210077P54577Tyrosine-tRNA ligase5910098B5BTY4ATP-dependent RNA helicase DDX3X73100210069B1AHM1DEAD (Asp-Glu-Ala-Asp) box polypeptide 1773991100510Q9NZI8Insulin-like growth factor 2 mRNA-binding protein 1631002100511P12956Lupus Ku autoantigen protein p7070390100512 (38)Q8NC51Plasminogen activator inhibitor 1 RNA-binding protein45620100213P4678240S ribosomal protein S523781100514P6270140S ribosomal protein S430781100315Q9Y262Eukaryotic translation initiation factor 3 subunit L (eIF3l)67350100316Q92900ATP-dependent helicase RENT1124991100317 (41)P11940Polyadenylate-binding protein 1 (PABP-1)71250100318P61011Signal recognition particle 54-kDa protein; (SRP54)56180100219O76094Signal recognition particle 72-kDa protein (SRP72)75100320Q9Y285Phenylalanine-tRNA ligase alpha subunit58100321 (45)Q59GL1Synaptotagmin binding, cytoplasmic RNA interacting protein60100322Q0VAC0Heterogeneous nuclear ribonucleoprotein A134100299123B3KNR3Glutathione S-transferase1999199124 (19)P6208140S ribosomal protein S722981100225 (30)Q12906Interleukin enhancer-binding factor 3; TCP8095781100326Q13435Splicing factor 3B subunit 2100781100227Q9UHB9Signal recognition particle 68-kDa protein71100328P04844Ribophorin II (RPN-II)69100229P49458Signal recognition particle 9-kDa protein; SRP910100230Q9H6T3RNA polymerase II-associated protein 376100331 (11)P05455La autoantigen; Sjoegren syndrome type B antigen (SS-B)47100332Q0254360S ribosomal protein L18a21100333P37108Signal recognition particle 14-kDa protein15100234P07910Heterogeneous nuclear ribonucleoproteins C1/C234781100235P6291360S ribosomal protein L1120781100236P6226640S ribosomal protein S2316410100237P6227740S ribosomal protein S1317100338Q02413Desmoglein 1114100339Q6NVW7Importin subunit alpha58100240Q9BUJ2Heterogeneous nuclear ribonucleoprotein U-like protein 196100241P14923Desmoplakin 382100142P4677860S ribosomal protein L2119100143Q96EY7Pentatricopeptide repeat-containing protein 3 (mitochondrial)79100144P26640Valine-tRNA ligase14099145F8W7C660S ribosomal protein L1019100246O43143ATP-dependent RNA helicase DHX15 (DEAH box protein 15)91100247P17844ATP-dependent RNA helicase DDX5; DEAD box protein 569100248 (44)P22626Heterogeneous nuclear ribonucleoproteins A2/B137100249Q13347Eukaryotic translation initiation factor 3 (eIF-3-beta)37100250Q53GX7Threonyl-tRNA synthetase variant83100251Q99729Heterogeneous nuclear ribonucleoprotein A/B36100152Q53F35Acidic (leucine-rich) nuclear phosphoprotein 3229100153P6285140S ribosomal protein S251499154Q9Y5A9YTH domain family protein 2; high-glucose-regulated protein 86299155P1807760S ribosomal protein L35a13991aNpro binding to cellular proteins through TRASH domain by zinc chelation. Cellular proteins associated with Npro in the presence or absence or zinc, detected by Orbitrap mass spectrometry analysis.bThe accession number and molecular mass for each protein are shown in order of abundance of peptides found. Proteins also identified in Table 1 are indicated by numbers in parentheses.cMolecular mass.dID, identity.

Host proteins interacting with Npro are components of ribosomes and ribonucleoprotein particles.

Orbitrap mass spectrometry was used to identify novel cellular proteins interacting with Npro from the classical swine fever virus strain Alfort. Purified recombinant GST-Npro protein was incubated with HEK 293 cell lysates that had either been untreated or transfected with synthetic dsRNA, poly(I·C), for 4 h both to induce expression of early-response genes that may be important following viral infection and to investigate the requirement of dsRNA for proteins to bind to Npro. Table 1 shows 45 proteins that were specifically eluted from Npro-GST compared to results with GST alone, with either a low (10 mM) or high (30 mM) glutathione concentration and with or without poly(I·C). The list is in order of the number of peptides found for each protein. (A full list is given in the PRIDE partner repository, with the data set identifier PXD000095 and doi number 10.6019/PXD000095 (PRIDE accession no. 28008). Interestingly, many of the hits were ribosomal proteins and dsRNA binding proteins, which form ribonucleoprotein particles in the cytoplasm and which are involved in RNA metabolism. Transfection of dsRNA poly(I·C) into cells for 4 h before lysis made very little difference in the proteins detected, indicating that binding to Npro was not dependent on dsRNA, normally present in cells only during viral infection and replication. In addition to ribosomal proteins, interacting proteins included Y-box binding protein (YBX-1), La autoantigen, RNA helicase A (DHX9), multiple myeloma tumor protein 2, interleukin enhancer binding factor 3 (IEBP3), guanine nucleotide binding protein 3, and polyadenylate binding protein 1 (PABP-1). These proteins are involved in RNA metabolism, including translation and mRNA degradation, and they are normally found distributed through the cytoplasm of cells (30). It may be that Npro interacts with one or two of these RNA binding proteins, and the other proteins are pulled down in the same complex. Surprisingly, we found no direct binding of Npro to the immune modulators IRF3, IRF7, Hax-1, or I-κBα, which have previously been described by yeast 2-hybrid interaction (17, 31–34) in cells of immune lineage. This may reflect differences between yeast 2-hybrid and protein pulldown mass spectrometry, that expression levels of these proteins are low in HEK293 cells compared to those in immune cells, or the fact that additional proteins are required. Our previous work has shown that Npro is able to bind and degrade IRF3 in HEK 293 cells (8).

Cellular proteins associated with GST-tagged Nproa

Npro interacts with ribosomal and ribonucleoproteins, components of ribonuclear particles (RNPs). Interaction of Npro with cellular proteins was identified by mass spectrometry analysis. Recombinant GST-tagged Npro was used to pull down cellular binding partners from HEK293 cells. Cells were either control (“no. pI·C”) or transfected with poly(I·C) (pI·C) (dsRNA) for 4 h before lysis. Beads were eluted with low (10 mM) or high (30 mM) glutathione (GSH). Proteins binding to GST alone were used for a control. The accession number and molecular mass for each protein are shown in order of abundance of peptides found.

Molecular mass.

ID, identity.

Npro is a metalloprotein coordinating a single zinc ion through a putative metal binding domain called a TRASH motif at Cys112-X21-Cys134-X3-Cys138 (19). This domain is thought to be required for IRF3 interaction and degradation (34). In order to investigate which proteins were interacting with the zinc binding TRASH domain in the C-terminal end of Npro, in a second experiment, the recombinant protein was bound to lysates either in the presence of zinc or in the presence of a chelator of zinc, TPEN. The interacting proteins are shown in Table 2 (deposited in PRIDE, accession no. 28009). In this experiment, a total of 55 cellular proteins were found to bind Npro (Table 2), of which 9 were also detectedin the group shown in Table 1 (numbers in parentheses in column 1 of Table 2). There were 28 additional proteins in the presence of zinc and only 2 extra proteins bound when zinc was absent (desmoglein and desmoplakin); 25 proteins bound irrespective of the presence of zinc. Interestingly, the proteins that required zinc were involved in RNA translation, and included signal recognition particle, RNA helicases DDX 5 and DHX 15, eukaryotic initiation factor eIF3i, and tRNA ligases. These are ribonucleoproteins, also present in polysomes, stress granules, and P bodies. For some proteins that bound Npro both in the presence and absence of zinc, the number of peptides increased in the presence of zinc, showing increased affinity and suggesting some interaction with the TRASH motif in Npro. Heterogeneous ribonucleoproteins hnRNP M, U, A/B/C, and H and G were found as cellular binding partners, and these proteins are involved in mRNA nuclear export, localization, translation, and stability, important in both the nucleus and the cytoplasm. In addition, many of these factors are also involved in posttranscriptional gene regulation.

Mass spectrometry identification of cellular proteins binding to Npro with or without zinca

Npro binding to cellular proteins through TRASH domain by zinc chelation. Cellular proteins associated with Npro in the presence or absence or zinc, detected by Orbitrap mass spectrometry analysis.

The accession number and molecular mass for each protein are shown in order of abundance of peptides found. Proteins also identified in Table 1 are indicated by numbers in parentheses.

Molecular mass.

ID, identity.

BVDV infection inhibits stress granule formation.Virus infection may trigger stress in cells. The ribonucleoproteins shown to interact with Npro described above are known to aggregate, after exposure of cells to stress, into RNA granules known as stress granules, which regulate initiation, termination, and decay of mRNAs (20). Since several viruses have been shown to actively inhibit stress granule assembly at certain times postinfection, possibly to allow synthesis of their own proteins (35), we asked whether pestivirus infection affects stress granule formation. Arsenate treatment is widely used for oxidative stress to induce stress granule formation (20), and in the next experiments, cells were treated with arsenate in the presence and absence of BVDV (Fig. 1A). TIA1, an RNA binding protein which nucleates stress granule formation, was used as a stress granule marker (36). TIA1 is not only a translational silencer but also a key protein in the prion-like aggregation of further mRNPs to stress granules. YBX1, shown above to interact with Npro, is a major component of cellular RNPs which accumulates in granules following cellular stress. MDBK cells expressing YBX1 or TIA1 alone and treated with sodium arsenate for 4 h showed YBX1 and TIA1 accumulation in stress granules, and cells died by apoptosis, seen as DNA fragmentation (Fig. 1Ai and iii). In cells infected with BVDV for 48 h, YBX1 and TIA1 remained cytoplasmic following sodium arsenate treatment for 4 h (Fig. 1Aii and iv), and they did not accumulate in stress granules or in virus replication centers. Furthermore, BVDV-infected cells were protected from apoptosis following sodium arsenate treatment. This experiment is shown using a quantitative caspase 3/7 assay in Fig. 1B, where there is a decrease in caspase activity after sodium arsenate treatment of BVDV-infected cells compared to that for uninfected cells.FIG 1(A) MDBK cells infected with BVDV are protected from stress-induced apoptosis. (i) Control MDBK cells transfected with YBX1-V5 and treated with NaA for 4 h. (ii) Cells infected with BVDV for 48 h postinfection (hpi), transfected with YBX1-V5 overnight, and treated with NaA for 4 h. YBX-V5 was detected with anti-V5 MAb and anti-mouse Alexa 488 secondary antibody. Virus was detected with V182 hyperimmune serum and anti-bovine cy5 secondary antibody. (iii) MDBK cells transfected with TIA1-mCherry plasmid and treated with NaA for 4 h. (TIA was pseudocolored green for consistency). (iv) MDBK cells infected with BVDV for 48 hpi, transfected with TIA1-mCherry, and treated with NaA for 4 h. Virus was detected with V182 hyperimmune serum and anti-bovine cy5 secondary antibody, and TIA1 was pseudocolored green. (B) BVDV infection decreases stress-induced caspase 3/7 activation. MDBK cells, either uninfected (−) or infected with (+) BVDV noncytopathic (ncp) Kyle for 48 h, were stressed with NaA for 4 h. The graph shows the fold increase in caspase activity normalized to that of control cells and significance with one -sided t test value, P = 0.00004.

BVDV infection inhibits stress granule formation.

Virus infection may trigger stress in cells. The ribonucleoproteins shown to interact with Npro described above are known to aggregate, after exposure of cells to stress, into RNA granules known as stress granules, which regulate initiation, termination, and decay of mRNAs (20). Since several viruses have been shown to actively inhibit stress granule assembly at certain times postinfection, possibly to allow synthesis of their own proteins (35), we asked whether pestivirus infection affects stress granule formation. Arsenate treatment is widely used for oxidative stress to induce stress granule formation (20), and in the next experiments, cells were treated with arsenate in the presence and absence of BVDV (Fig. 1A). TIA1, an RNA binding protein which nucleates stress granule formation, was used as a stress granule marker (36). TIA1 is not only a translational silencer but also a key protein in the prion-like aggregation of further mRNPs to stress granules. YBX1, shown above to interact with Npro, is a major component of cellular RNPs which accumulates in granules following cellular stress. MDBK cells expressing YBX1 or TIA1 alone and treated with sodium arsenate for 4 h showed YBX1 and TIA1 accumulation in stress granules, and cells died by apoptosis, seen as DNA fragmentation (Fig. 1Ai and iii). In cells infected with BVDV for 48 h, YBX1 and TIA1 remained cytoplasmic following sodium arsenate treatment for 4 h (Fig. 1Aii and iv), and they did not accumulate in stress granules or in virus replication centers. Furthermore, BVDV-infected cells were protected from apoptosis following sodium arsenate treatment. This experiment is shown using a quantitative caspase 3/7 assay in Fig. 1B, where there is a decrease in caspase activity after sodium arsenate treatment of BVDV-infected cells compared to that for uninfected cells.

(A) MDBK cells infected with BVDV are protected from stress-induced apoptosis. (i) Control MDBK cells transfected with YBX1-V5 and treated with NaA for 4 h. (ii) Cells infected with BVDV for 48 h postinfection (hpi), transfected with YBX1-V5 overnight, and treated with NaA for 4 h. YBX-V5 was detected with anti-V5 MAb and anti-mouse Alexa 488 secondary antibody. Virus was detected with V182 hyperimmune serum and anti-bovine cy5 secondary antibody. (iii) MDBK cells transfected with TIA1-mCherry plasmid and treated with NaA for 4 h. (TIA was pseudocolored green for consistency). (iv) MDBK cells infected with BVDV for 48 hpi, transfected with TIA1-mCherry, and treated with NaA for 4 h. Virus was detected with V182 hyperimmune serum and anti-bovine cy5 secondary antibody, and TIA1 was pseudocolored green. (B) BVDV infection decreases stress-induced caspase 3/7 activation. MDBK cells, either uninfected (−) or infected with (+) BVDV noncytopathic (ncp) Kyle for 48 h, were stressed with NaA for 4 h. The graph shows the fold increase in caspase activity normalized to that of control cells and significance with one -sided t test value, P = 0.00004.

Npro interaction with YBX1 does not prevent formation of stress granules.We have previously shown that ectopic expression of the single viral protein Npro can inhibit apoptosis and interferon production through binding and degradation of IRF3 (8). Since we show here that Npro is multifunctional and also binds to ribonucleoproteins, such as YBX1, a major component of stress granules, and that virus infection inhibits stress granule formation, we thought that Npro was a good candidate for the individual protein responsible for stress granule inhibition. Fluorescently tagged Npro (Npro-GFP or Npro-mCherry) was expressed in cells. Npro distributed to the nucleus and cytoplasm (Fig. 2Ai) and subsequently redistributed to cytoplasmic granules following stress with sodium arsenate (Fig. 2Aii). These cells were protected from apoptosis, as we have described in detail in our previous work (8), indicating stress granule assembly in the absence of apoptosis activation. However, when Npro-mCherry was expressed in BVDV-infected cells, Npro relocated to an area associated with the viral replication complex (Fig. 2Aiii). In the next experiments, we focused on possible functional roles of Npro by investigating its binding to, localization with, and effect on host proteins identified in Tables 1 and 2, outside the context of viral infection and in the absence of the other viral proteins. First, we investigated whether Npro redistributed with YBX-1 in stressed cells. When coexpressed together, Npro and YBX1 were both located to the cytoplasm in unstressed cells (Fig. 2Bi). Cells expressing Npro-mCherry and YBX1 treated with sodium arsenate for 4 h showed redistribution of Npro to cytoplasmic granules containing YBX1 (Fig. 2Bii), indicating that expression of Npro itself does not inhibit the formation of stress granule complexes in cells. In cells expressing the mutant Npro C112R protein, perinuclear stress granules containing YBX1 were formed, but these granules no longer colocalized with Npro C112R, which was found in smaller cytoplasmic foci (Fig. 2Biii, insert). Taken together, these results indicate that wild-type Npro binds to YBX1 through its TRASH domain, since Npro with the C112R mutation, which disrupts the zinc binding TRASH domain, no longer codistributes with YBX1 to stress granules. It demonstrates that Npro relocation to the stress granule complex is attributable to its binding to host proteins, such as YBX1. During infection, however, we show that stress granule formation is inhibited and YBX1 remains cytoplasmic (Fig. 1A), whereas Npro redistributes to the replication complex after infected cells are stressed (Fig. 2A). This suggests that since YBX1 is not recruited to the replication complex during infection, it is not required for viral replication, but it may have an important function bound to Npro at sites of viral genome translation in the cytoplasm.FIG 2Interaction and colocalization of Npro with Y-box protein 1 (YBX-1) requires the zinc-binding TRASH motif. (A) Npro localization in BVDV-infected MDBK cells. (i) Control MDBK cells transfected with Npro-GFP. (ii) MDBK cells transfected with Npro-GFP and treated with NaA for 4 h. (iii) Cells infected with BVDV for 48 hpi, transfected with Npro-GFP for 24 h, and treated with NaA for 4 h. Virus was detected with V182 bovine hyperimmune serum and anti-bovine cy5 secondary antibody. (B) Npro codistributes with YBX1 to large cytoplasmic stress granules. Cells stably expressing Npro-mCherry and transfected with YBX-1-V5 were either untreated or treated with NaA for 4 h. YBX1 was detected using anti-V5 MAb and anti-mouse Alexa 488 secondary antibody, and DNA was stained with 4′,6-diamidino-2-phenylindole (DAPI) (blue). (i) Control untreated HeLa cells, (ii) Cells stably expressing Npro-mCherry, transfected with YBX1-V5 and treated with NaA for 4 h. (iii) Cells stably expressing mutant Npro C112R-mCherry and transfected with YBX1-V5 and treated with NaA for 4 h. (C) Protein pulldown of HeLa cell lysates with recombinant Npro shows interaction with endogenous cellular YBX1 requiring zinc. The GST and GST-Npro recombinant proteins were washed in zinc buffer or zinc chelator, TPEN. Glutathione-Sepharose beads were used to pull down GST-Npro and its cobinding proteins. Top panel, endogenous YBX1 detected in cell lysates at 36 kDa with an anti-YBX1 antibody; middle panel, GST and GST-Npro recombinant proteins detected using anti-GST antibody and anti-Npro, respectively; bottom panel, HeLa cell lysates incubated with GST and GST-Npro proteins were pulled down with glutathione-Sepharose beads, washed extensively, and separated by SDS-PAGE. Proteins were transferred to membranes and probed with rabbit YBX1 antibody, which was detected with anti-rabbit-HRP by enhanced chemiluminescence (ECL). (D) Interaction of Npro with YBX1 increases after stress. Cells stably expressing Npro-mCherry were transfected with YBX1-V5, and cells were untreated (−) or treated (+) with sodium arsenate for 4 h (NaA). Lysates were pulled down with anti-V5 antibody bound to Sepharose beads. Top panels, Western blots of cell lysates either untreated (−) or treated (+) with NaA to detect YBX-V5 with anti-V5 monoclonal antibody (left) or Npro with rabbit anti-Npro antibody (right) using anti-mouse or anti-rabbit HRP antibodies, respectively. Bottom panels, immunoprecipitation of proteins using an anti-V5 antibody bound to Sepharose beads, followed by transfer of proteins to membranes and Western blotting with anti-V5 (left panel) or anti-Npro (right panel) antibodies. (E) Npro and YBX accumulate following stress. (i) Lysates from control cells (−) or cells expressing Npro-mCherry (+ Npro), either untreated (−NaA) or stressed for 4 h (+NaA), were blotted with antibodies against Npro and actin. (ii) Lysates from control cells (−) or cells expressing Npro (+Npro) either untreated (−) or treated with NaA (+) for 4 h were blotted with anti-YBX1 and antiactin antibodies.The binding of Npro to YBX1 was confirmed biochemically by a coimmunoprecipitation assay using either recombinant Npro GST (Fig. 2C) or lysates from HEK 293 cells expressing Npro-mCherry (Fig. 2D). Cell lysates were passed over glutathione beads bound to GST or Npro GST that had been washed in zinc buffer or washed with a zinc chelator, TPEN (Fig. 2C). Proteins pulled down were separated by gel electrophoresis and blotted with an anti-YBX 1 antibody. YBX1 was pulled down with the Npro recombinant protein specifically in the presence of zinc and not when the zinc had been chelated. This confirmed that Npro interacts with YBX1 through its zinc-binding TRASH motif.In the next experiment, cells stably expressing Npro-mCherry were cotransfected with a plasmid encoding YBX1 tagged with V5 to investigate binding in cells (Fig. 2D). Cells were treated with sodium arsenate for 4 h to promote the formation of stress granules. Proteins pulled down with YBX1 using anti-V5 antibody were separated on a gel and probed with anti-Npro antibodies. Npro was detected bound to YBX1 in cells that had been stressed but not in unstressed cells (Fig. 2D). This may reflect the increased affinity or abundance of the complex bound together in stress granules in cells.In cells treated with sodium arsenate for 4 h, there was accumulation of the Npro protein (Fig. 2Ei). Oxidative stress also increased the levels of YBX1 (Fig. 2Eii, lanes 1 and 2). In the presence of Npro, increased levels of YBX1 were precipitated both with and without arsenate treatment (Fig. 2Eii, lanes 3 and 4). These results show that both induction of stress and expression of Npro led to the accumulation of YBX1. The requirement for YBX1 during BVDV infection was attempted by silencing of YBX1 in infected MDBK cells, but because of the extremely low rates of transfection of these cells and the high level of expression of YBX1, it was not possible. (Pestiviruses have a narrow tropism to their host species, and therefore gene knockdown experiments are not possible using knocked-out mouse embryonic fibroblasts [MEFs]).

Npro interaction with YBX1 does not prevent formation of stress granules.

We have previously shown that ectopic expression of the single viral protein Npro can inhibit apoptosis and interferon production through binding and degradation of IRF3 (8). Since we show here that Npro is multifunctional and also binds to ribonucleoproteins, such as YBX1, a major component of stress granules, and that virus infection inhibits stress granule formation, we thought that Npro was a good candidate for the individual protein responsible for stress granule inhibition. Fluorescently tagged Npro (Npro-GFP or Npro-mCherry) was expressed in cells. Npro distributed to the nucleus and cytoplasm (Fig. 2Ai) and subsequently redistributed to cytoplasmic granules following stress with sodium arsenate (Fig. 2Aii). These cells were protected from apoptosis, as we have described in detail in our previous work (8), indicating stress granule assembly in the absence of apoptosis activation. However, when Npro-mCherry was expressed in BVDV-infected cells, Npro relocated to an area associated with the viral replication complex (Fig. 2Aiii). In the next experiments, we focused on possible functional roles of Npro by investigating its binding to, localization with, and effect on host proteins identified in Tables 1 and 2, outside the context of viral infection and in the absence of the other viral proteins. First, we investigated whether Npro redistributed with YBX-1 in stressed cells. When coexpressed together, Npro and YBX1 were both located to the cytoplasm in unstressed cells (Fig. 2Bi). Cells expressing Npro-mCherry and YBX1 treated with sodium arsenate for 4 h showed redistribution of Npro to cytoplasmic granules containing YBX1 (Fig. 2Bii), indicating that expression of Npro itself does not inhibit the formation of stress granule complexes in cells. In cells expressing the mutant Npro C112R protein, perinuclear stress granules containing YBX1 were formed, but these granules no longer colocalized with Npro C112R, which was found in smaller cytoplasmic foci (Fig. 2Biii, insert). Taken together, these results indicate that wild-type Npro binds to YBX1 through its TRASH domain, since Npro with the C112R mutation, which disrupts the zinc binding TRASH domain, no longer codistributes with YBX1 to stress granules. It demonstrates that Npro relocation to the stress granule complex is attributable to its binding to host proteins, such as YBX1. During infection, however, we show that stress granule formation is inhibited and YBX1 remains cytoplasmic (Fig. 1A), whereas Npro redistributes to the replication complex after infected cells are stressed (Fig. 2A). This suggests that since YBX1 is not recruited to the replication complex during infection, it is not required for viral replication, but it may have an important function bound to Npro at sites of viral genome translation in the cytoplasm.

Interaction and colocalization of Npro with Y-box protein 1 (YBX-1) requires the zinc-binding TRASH motif. (A) Npro localization in BVDV-infected MDBK cells. (i) Control MDBK cells transfected with Npro-GFP. (ii) MDBK cells transfected with Npro-GFP and treated with NaA for 4 h. (iii) Cells infected with BVDV for 48 hpi, transfected with Npro-GFP for 24 h, and treated with NaA for 4 h. Virus was detected with V182 bovine hyperimmune serum and anti-bovine cy5 secondary antibody. (B) Npro codistributes with YBX1 to large cytoplasmic stress granules. Cells stably expressing Npro-mCherry and transfected with YBX-1-V5 were either untreated or treated with NaA for 4 h. YBX1 was detected using anti-V5 MAb and anti-mouse Alexa 488 secondary antibody, and DNA was stained with 4′,6-diamidino-2-phenylindole (DAPI) (blue). (i) Control untreated HeLa cells, (ii) Cells stably expressing Npro-mCherry, transfected with YBX1-V5 and treated with NaA for 4 h. (iii) Cells stably expressing mutant Npro C112R-mCherry and transfected with YBX1-V5 and treated with NaA for 4 h. (C) Protein pulldown of HeLa cell lysates with recombinant Npro shows interaction with endogenous cellular YBX1 requiring zinc. The GST and GST-Npro recombinant proteins were washed in zinc buffer or zinc chelator, TPEN. Glutathione-Sepharose beads were used to pull down GST-Npro and its cobinding proteins. Top panel, endogenous YBX1 detected in cell lysates at 36 kDa with an anti-YBX1 antibody; middle panel, GST and GST-Npro recombinant proteins detected using anti-GST antibody and anti-Npro, respectively; bottom panel, HeLa cell lysates incubated with GST and GST-Npro proteins were pulled down with glutathione-Sepharose beads, washed extensively, and separated by SDS-PAGE. Proteins were transferred to membranes and probed with rabbit YBX1 antibody, which was detected with anti-rabbit-HRP by enhanced chemiluminescence (ECL). (D) Interaction of Npro with YBX1 increases after stress. Cells stably expressing Npro-mCherry were transfected with YBX1-V5, and cells were untreated (−) or treated (+) with sodium arsenate for 4 h (NaA). Lysates were pulled down with anti-V5 antibody bound to Sepharose beads. Top panels, Western blots of cell lysates either untreated (−) or treated (+) with NaA to detect YBX-V5 with anti-V5 monoclonal antibody (left) or Npro with rabbit anti-Npro antibody (right) using anti-mouse or anti-rabbit HRP antibodies, respectively. Bottom panels, immunoprecipitation of proteins using an anti-V5 antibody bound to Sepharose beads, followed by transfer of proteins to membranes and Western blotting with anti-V5 (left panel) or anti-Npro (right panel) antibodies. (E) Npro and YBX accumulate following stress. (i) Lysates from control cells (−) or cells expressing Npro-mCherry (+ Npro), either untreated (−NaA) or stressed for 4 h (+NaA), were blotted with antibodies against Npro and actin. (ii) Lysates from control cells (−) or cells expressing Npro (+Npro) either untreated (−) or treated with NaA (+) for 4 h were blotted with anti-YBX1 and antiactin antibodies.

The binding of Npro to YBX1 was confirmed biochemically by a coimmunoprecipitation assay using either recombinant Npro GST (Fig. 2C) or lysates from HEK 293 cells expressing Npro-mCherry (Fig. 2D). Cell lysates were passed over glutathione beads bound to GST or Npro GST that had been washed in zinc buffer or washed with a zinc chelator, TPEN (Fig. 2C). Proteins pulled down were separated by gel electrophoresis and blotted with an anti-YBX 1 antibody. YBX1 was pulled down with the Npro recombinant protein specifically in the presence of zinc and not when the zinc had been chelated. This confirmed that Npro interacts with YBX1 through its zinc-binding TRASH motif.

In the next experiment, cells stably expressing Npro-mCherry were cotransfected with a plasmid encoding YBX1 tagged with V5 to investigate binding in cells (Fig. 2D). Cells were treated with sodium arsenate for 4 h to promote the formation of stress granules. Proteins pulled down with YBX1 using anti-V5 antibody were separated on a gel and probed with anti-Npro antibodies. Npro was detected bound to YBX1 in cells that had been stressed but not in unstressed cells (Fig. 2D). This may reflect the increased affinity or abundance of the complex bound together in stress granules in cells.

In cells treated with sodium arsenate for 4 h, there was accumulation of the Npro protein (Fig. 2Ei). Oxidative stress also increased the levels of YBX1 (Fig. 2Eii, lanes 1 and 2). In the presence of Npro, increased levels of YBX1 were precipitated both with and without arsenate treatment (Fig. 2Eii, lanes 3 and 4). These results show that both induction of stress and expression of Npro led to the accumulation of YBX1. The requirement for YBX1 during BVDV infection was attempted by silencing of YBX1 in infected MDBK cells, but because of the extremely low rates of transfection of these cells and the high level of expression of YBX1, it was not possible. (Pestiviruses have a narrow tropism to their host species, and therefore gene knockdown experiments are not possible using knocked-out mouse embryonic fibroblasts [MEFs]).

Colocalization of interacting host factors with Npro after stress.As an assay for Npro interaction with factors identified by mass spectrometry, oxidative stress was used to determine their codistribution to granules. We coexpressed Npro-mCherry with TIA1, DDX1, DDX3, and DCP1a. Figure 3A shows codistribution of these proteins in the cytoplasm and their redistribution with Npro to stress granules as an indication of cobinding. DDX1 was also found in smaller cytoplasmic foci. DDX1 and DDX3 are RNA helicases with roles in transcriptional and posttranscriptional RNA metabolism, including RNA splicing, translation, DNA repair, and transcription (10). The decapping enzyme DCP1a, a component of P bodies, which are related to stress granules but contain other proteins involved in mRNA decay (20), codistributed with Npro to stress granules (Fig. 3Aiv). Stress granules are complex structures made up of RNA-binding proteins, 40S ribosomal subunits, but the composition can vary (20). We have shown that DDX1 and DDX3 aggregate into granules which also comprise YBX1 and TIA1 (Fig. 3Bi to iii), whereas DCP1a formed a ring-like structure around the TIA1-containing granules (Fig. 3Biv), demonstrating a separate domain for this protein. The imaging experiments, together with the proteomic data, show that Npro interacts with important antiviral RNA binding proteins in the cytoplasm, required for translation. Pestivirus infection prevents stress granule aggregation by an unidentified mechanism, but expression of Npro alone does not. During infection, the interaction may have an effect on viral or host translation.FIG 3Pestivirus Npro forms a complex with ribonucleoproteins and redistributes to stress granules. (A) Npro redistribution with TIA1, DDX1, DDX3, and DCP1a following stress. Cells were either untreated (CON) or subjected to oxidative stress for 4 h (+NaA). Cell expressing Npro-GFP were cotransfected with TIA1-mCherry (i). Cells expressing Npro-mCherry were cotransfected with DDX1-V5 (ii), DDX3-Myc (iii), or DCP1a-V5 (iv), which were detected with anti-V5 or anti-Myc MAbs and secondary anti-mouse Alexa 488 antibodies. (B) Codistribution of YBX1 and TIA1-containing particles with RNA helicases DDX1 and DDX3. HeLa cells were transfected with plasmids encoding DDX1-V5 (i) or DDX3-Myc (ii) and treated with NaA for 4 h. Endogenous YBX1 was detected with rabbit anti-YBX1 and an anti-rabbit Alexa 594 secondary antibody, and DDX1 and DDX3 were detected with V5 and Myc MAbs, respectively, and an anti-mouse Alexa 488 secondary antibody. HeLa cells were cotransfected with a plasmid encoding TIA1-mCherry and YBX1-V5 (iii) or DCP1a-V5 (iv), detected using anti-V5 with anti-mouse Alexa 488 secondary antibody.

Colocalization of interacting host factors with Npro after stress.

As an assay for Npro interaction with factors identified by mass spectrometry, oxidative stress was used to determine their codistribution to granules. We coexpressed Npro-mCherry with TIA1, DDX1, DDX3, and DCP1a. Figure 3A shows codistribution of these proteins in the cytoplasm and their redistribution with Npro to stress granules as an indication of cobinding. DDX1 was also found in smaller cytoplasmic foci. DDX1 and DDX3 are RNA helicases with roles in transcriptional and posttranscriptional RNA metabolism, including RNA splicing, translation, DNA repair, and transcription (10). The decapping enzyme DCP1a, a component of P bodies, which are related to stress granules but contain other proteins involved in mRNA decay (20), codistributed with Npro to stress granules (Fig. 3Aiv). Stress granules are complex structures made up of RNA-binding proteins, 40S ribosomal subunits, but the composition can vary (20). We have shown that DDX1 and DDX3 aggregate into granules which also comprise YBX1 and TIA1 (Fig. 3Bi to iii), whereas DCP1a formed a ring-like structure around the TIA1-containing granules (Fig. 3Biv), demonstrating a separate domain for this protein. The imaging experiments, together with the proteomic data, show that Npro interacts with important antiviral RNA binding proteins in the cytoplasm, required for translation. Pestivirus infection prevents stress granule aggregation by an unidentified mechanism, but expression of Npro alone does not. During infection, the interaction may have an effect on viral or host translation.

Pestivirus Npro forms a complex with ribonucleoproteins and redistributes to stress granules. (A) Npro redistribution with TIA1, DDX1, DDX3, and DCP1a following stress. Cells were either untreated (CON) or subjected to oxidative stress for 4 h (+NaA). Cell expressing Npro-GFP were cotransfected with TIA1-mCherry (i). Cells expressing Npro-mCherry were cotransfected with DDX1-V5 (ii), DDX3-Myc (iii), or DCP1a-V5 (iv), which were detected with anti-V5 or anti-Myc MAbs and secondary anti-mouse Alexa 488 antibodies. (B) Codistribution of YBX1 and TIA1-containing particles with RNA helicases DDX1 and DDX3. HeLa cells were transfected with plasmids encoding DDX1-V5 (i) or DDX3-Myc (ii) and treated with NaA for 4 h. Endogenous YBX1 was detected with rabbit anti-YBX1 and an anti-rabbit Alexa 594 secondary antibody, and DDX1 and DDX3 were detected with V5 and Myc MAbs, respectively, and an anti-mouse Alexa 488 secondary antibody. HeLa cells were cotransfected with a plasmid encoding TIA1-mCherry and YBX1-V5 (iii) or DCP1a-V5 (iv), detected using anti-V5 with anti-mouse Alexa 488 secondary antibody.

RNA helicase A and La autoantigen redistribute with Npro from the nucleus to cytoplasmic stress granules.Another important protein shown to interact with Npro is RNA helicase A (RHA). RHA (also known as DEAH box protein 9 [DHX9]) binds dsRNA; it is a DExD/H box helicase with both DNA and RNA helicase activity. Its antiviral properties include binding to the RNA-induced silencing complex (RISC) to help load dsRNA onto Argonaute for microRNA (miRNA) production (37). To confirm interaction between RHA and Npro, cells were transfected with a plasmid encoding RHA tagged to a hemagglutinin (HA) epitope. Cell lysates were incubated with beads bound to GST or GST-Npro recombinant protein and washed as described. RHA was pulled down with beads containing GST Npro but not GST beads alone (Fig. 4A), showing specific binding of RHA by recombinant Npro. When HEK 293 cells stably expressing cDNA for GST Npro were cotransfected with a plasmid encoding RHA-HA, GST-Npro was pulled down by GST in cells overexpressing RHA but not in cells without RHA (Fig. 4B). This indicates an increased efficiency of GST binding and Npro pulldown in the presence of RHA, possibly due to conformational change. Next we investigated the localization of RHA in cells expressing Npro. HeLa cells were cotransfected with RHA-GFP and Npro-mCherry. In untreated cells, RHA localized to the nucleus (Fig. 4Ci, top left panels), and there was little staining in the cytoplasm, whereas Npro expression was both nuclear and cytoplasmic. When these cells were subjected to oxidative stress, RHA relocated from the nucleus to cytoplasmic stress granules (Fig. 4Ci, top right panels, +NaA), and these granules colocalized with Npro. Since pestiviruses replicate in the cytoplasm with no nuclear component, we used RHA-GFP carrying a mutation (R1163A) which blocks its nuclear translocation (25) to investigate the effect on RHA, which is expressed exclusively in the cytoplasm. In unstressed cells, the mutant RHA-GFP R1163A was seen only in the cytoplasm (Fig. 4Ci, bottom panels). Treatment with sodium arsenate led to large cytoplasmic stress granules that colocalized with Npro-mCherry, clearly indicating RHA binding in a complex to Npro in cytoplasmic stress granules (Fig. 4Ci, bottom panels).FIG 4RNA helicase A (DHX9) binds to Npro and redistributes from the nucleus to cytoplasmic stress granules following stress. (A) RHA tagged with HA was transfected into HEK293 cells, and lysates were incubated with GST or GST-Npro recombinant proteins. Cobinding proteins were pulled down with glutathione-Sepharose beads. Top panel, Western blot of recombinant protein with anti-GST antibody (RP); middle panel, Western blot of cell lysates expressing RHA with anti-HA antibody detected with anti-mouse HRP secondary antibody; bottom panel, Western blot of proteins pulled down with glutathione-Sepharose beads and detected with anti-HA antibody. (B) GST-Npro expressed alone (−) or with RHA-HA (+) in HEK293 cells was pulled down using glutathione-Sepharose beads from cells lysates. Top panel, Western blot of cell lysates with anti-Npro antibody; middle panel, Western blot of cell lysates with anti-HA antibody; bottom panel, Western blot of proteins pulled down with glutathione beads using anti-Npro antibody. (C) RNA helicase A (DHX9) redistributes from the nucleus to cytoplasmic stress granules with Npro following stress. (i) HeLa cells were cotransfected with Npro-mCherry and either RHA-GFP (RHA +Npro) or mutant RHA R1163A-GFP (R1163A +Npro). Cells were either untreated (CON) or subjected to oxidative stress (+NaA). (ii) MBDK cells were infected with BVDV for 48 h and transfected with either RHA-GFP or mutant RHA R1163A GFP for 24 h. Cells were either untreated (CON) or subjected to oxidative stress (+NaA). (iii) HeLa cells were cotransfected with Npro-mCherry and La autoantigen-GFP. Cells were either untreated (CON) or subjected to oxidative stress (+NaA).In order to investigate RHA relocalization in BVDV-infected cells, MDBK cells were infected with virus and transfected with RHA-GFP (Fig. 4Cii). In control BVDV-infected cells, RHA-GFP distributed to the nucleus. When cells were stressed, there was no change in the distribution of RHA-GFP, which remained in the nucleus (Fig. 4Cii, top right panel.) This confirms the results shown in Fig. 1, that stress granules are not formed when BVDV-infected cells are stressed and RHA, TIA1, and YBX1 do not redistribute into stress granules. Similarly, the RHA R1163A -GFP mutant remained cytoplasmic both before and after treatment with sodium arsenate and did not relocate to stress granules (Fig. 4Cii, lower panels).We have shown by mass spectrometry that La autoantigen interacted with Npro (Table 1). La autoantigen is an RNA-binding protein which binds several viral RNAs, including those of poliovirus and human immunodeficiency virus (HIV), to enhance their translation in vitro (38). In the next experiment, we showed that the La autoantigen also redistributes from the nucleus to cytoplasmic granules with Npro following induction of stress (Fig. 4Ciii). La-GFP was predominantly nuclear before stress, but after treatment it was seen in small cytoplasmic dots colocalizing with Npro (Fig. 4Ciii). Thus, we have identified components of ribonucleoparticles made up from both cytoplasmic and nuclear proteins that bind to Npro in cells and which are redistributed to a cytoplasmic complex together after stress.

RNA helicase A and La autoantigen redistribute with Npro from the nucleus to cytoplasmic stress granules.

Another important protein shown to interact with Npro is RNA helicase A (RHA). RHA (also known as DEAH box protein 9 [DHX9]) binds dsRNA; it is a DExD/H box helicase with both DNA and RNA helicase activity. Its antiviral properties include binding to the RNA-induced silencing complex (RISC) to help load dsRNA onto Argonaute for microRNA (miRNA) production (37). To confirm interaction between RHA and Npro, cells were transfected with a plasmid encoding RHA tagged to a hemagglutinin (HA) epitope. Cell lysates were incubated with beads bound to GST or GST-Npro recombinant protein and washed as described. RHA was pulled down with beads containing GST Npro but not GST beads alone (Fig. 4A), showing specific binding of RHA by recombinant Npro. When HEK 293 cells stably expressing cDNA for GST Npro were cotransfected with a plasmid encoding RHA-HA, GST-Npro was pulled down by GST in cells overexpressing RHA but not in cells without RHA (Fig. 4B). This indicates an increased efficiency of GST binding and Npro pulldown in the presence of RHA, possibly due to conformational change. Next we investigated the localization of RHA in cells expressing Npro. HeLa cells were cotransfected with RHA-GFP and Npro-mCherry. In untreated cells, RHA localized to the nucleus (Fig. 4Ci, top left panels), and there was little staining in the cytoplasm, whereas Npro expression was both nuclear and cytoplasmic. When these cells were subjected to oxidative stress, RHA relocated from the nucleus to cytoplasmic stress granules (Fig. 4Ci, top right panels, +NaA), and these granules colocalized with Npro. Since pestiviruses replicate in the cytoplasm with no nuclear component, we used RHA-GFP carrying a mutation (R1163A) which blocks its nuclear translocation (25) to investigate the effect on RHA, which is expressed exclusively in the cytoplasm. In unstressed cells, the mutant RHA-GFP R1163A was seen only in the cytoplasm (Fig. 4Ci, bottom panels). Treatment with sodium arsenate led to large cytoplasmic stress granules that colocalized with Npro-mCherry, clearly indicating RHA binding in a complex to Npro in cytoplasmic stress granules (Fig. 4Ci, bottom panels).

RNA helicase A (DHX9) binds to Npro and redistributes from the nucleus to cytoplasmic stress granules following stress. (A) RHA tagged with HA was transfected into HEK293 cells, and lysates were incubated with GST or GST-Npro recombinant proteins. Cobinding proteins were pulled down with glutathione-Sepharose beads. Top panel, Western blot of recombinant protein with anti-GST antibody (RP); middle panel, Western blot of cell lysates expressing RHA with anti-HA antibody detected with anti-mouse HRP secondary antibody; bottom panel, Western blot of proteins pulled down with glutathione-Sepharose beads and detected with anti-HA antibody. (B) GST-Npro expressed alone (−) or with RHA-HA (+) in HEK293 cells was pulled down using glutathione-Sepharose beads from cells lysates. Top panel, Western blot of cell lysates with anti-Npro antibody; middle panel, Western blot of cell lysates with anti-HA antibody; bottom panel, Western blot of proteins pulled down with glutathione beads using anti-Npro antibody. (C) RNA helicase A (DHX9) redistributes from the nucleus to cytoplasmic stress granules with Npro following stress. (i) HeLa cells were cotransfected with Npro-mCherry and either RHA-GFP (RHA +Npro) or mutant RHA R1163A-GFP (R1163A +Npro). Cells were either untreated (CON) or subjected to oxidative stress (+NaA). (ii) MBDK cells were infected with BVDV for 48 h and transfected with either RHA-GFP or mutant RHA R1163A GFP for 24 h. Cells were either untreated (CON) or subjected to oxidative stress (+NaA). (iii) HeLa cells were cotransfected with Npro-mCherry and La autoantigen-GFP. Cells were either untreated (CON) or subjected to oxidative stress (+NaA).

In order to investigate RHA relocalization in BVDV-infected cells, MDBK cells were infected with virus and transfected with RHA-GFP (Fig. 4Cii). In control BVDV-infected cells, RHA-GFP distributed to the nucleus. When cells were stressed, there was no change in the distribution of RHA-GFP, which remained in the nucleus (Fig. 4Cii, top right panel.) This confirms the results shown in Fig. 1, that stress granules are not formed when BVDV-infected cells are stressed and RHA, TIA1, and YBX1 do not redistribute into stress granules. Similarly, the RHA R1163A -GFP mutant remained cytoplasmic both before and after treatment with sodium arsenate and did not relocate to stress granules (Fig. 4Cii, lower panels).

We have shown by mass spectrometry that La autoantigen interacted with Npro (Table 1). La autoantigen is an RNA-binding protein which binds several viral RNAs, including those of poliovirus and human immunodeficiency virus (HIV), to enhance their translation in vitro (38). In the next experiment, we showed that the La autoantigen also redistributes from the nucleus to cytoplasmic granules with Npro following induction of stress (Fig. 4Ciii). La-GFP was predominantly nuclear before stress, but after treatment it was seen in small cytoplasmic dots colocalizing with Npro (Fig. 4Ciii). Thus, we have identified components of ribonucleoparticles made up from both cytoplasmic and nuclear proteins that bind to Npro in cells and which are redistributed to a cytoplasmic complex together after stress.

Pestivirus Npro does not suppress RNA interference in mammalian cells.RNA granules aggregating with Npro are involved in several different RNA metabolic processes, and some of the components, for example RNA helicase A, interact directly with the RNA interference (RNAi) pathway (37). Silencing of RHA has been shown to block the RNAi pathway, and RHA is important for loading short RNAs, including siRNAs and miRNAs, onto RISC (37). The interacting protein La autoantigen is an RNAi activator that promotes the release of cleaved mRNA from Ago2 and efficient RNAi, an antiviral defense (39). Since Npro is a viral suppressor of interferon synthesis and apoptosis, and other viral suppressors of interferon, such as Ebola vp35, vaccinia virus E3L, and influenza virus NS1, also block the RNAi pathway, we investigated whether Npro was a viral suppressor of RNA silencing (22, 23). In the next series of experiments, the GAPDH housekeeping gene was silenced using GAPDH-specific siRNAs in control or Npro-expressing cells or cells where RNA helicase A had been silenced (RHA siRNA). GAPDH RNA levels were analyzed by Northern blotting with a 32P-labeled GAPDH-specific primer (Fig. 5ii). GAPDH was efficiently knocked down by siRNAs in control cells (Fig. 5ii, left panel, lane 4). There was no difference in GAPDH knockdown between control and Npro mCherry-expressing cells (Fig. 5ii, middle panel, lane 4), showing that Npro does not suppress RNAi from exogenously added siRNAs. It has been previously reported that RNA helicase A inhibits RISC loading (37) and that it might be involved in RNAi, so we investigated whether silencing of RHA had an effect on the RNAi assay of GAPDH knockdown with GAPDH-specific siRNAs, as described above. We silenced RHA with RHA-specific siRNAs (Fig. 5i, right panel). RHA depletion shows some suppression of RNAi from externally added siRNAs (Fig. 5ii, right panel, lane 4), as has been reported previously (37).FIG 5Npro does not suppress RNA silencing.(i) Western blot of control HeLa cells (−), cells stably expressing Npro-mCherry (+Npro) (left panel), or HeLa cells silenced for RHA expression (RHA siRNA) (right panel). Npro was detected with a rabbit anti-Npro antibody and visualized with IRDye-labeled secondary antibodies. RHA was detected with mouse anti-RHA and visualized with HRP-labeled secondary antibody. (ii) Northern blot analysis of GAPDH mRNA levels following GAPDH silencing using a 32P-labeled GAPDH-specific primer. 1, control cells; 2, mock-treated cells; 3, nontargeting siRNA-treated cells; 4, GAPDH siRNA-treated cells. Control cells (left panel), cells expressing Npro (middle panel), and cells silenced for RHA (right panel) are shown. An 18S RNA blot is shown for equal loading.

Pestivirus Npro does not suppress RNA interference in mammalian cells.

RNA granules aggregating with Npro are involved in several different RNA metabolic processes, and some of the components, for example RNA helicase A, interact directly with the RNA interference (RNAi) pathway (37). Silencing of RHA has been shown to block the RNAi pathway, and RHA is important for loading short RNAs, including siRNAs and miRNAs, onto RISC (37). The interacting protein La autoantigen is an RNAi activator that promotes the release of cleaved mRNA from Ago2 and efficient RNAi, an antiviral defense (39). Since Npro is a viral suppressor of interferon synthesis and apoptosis, and other viral suppressors of interferon, such as Ebola vp35, vaccinia virus E3L, and influenza virus NS1, also block the RNAi pathway, we investigated whether Npro was a viral suppressor of RNA silencing (22, 23). In the next series of experiments, the GAPDH housekeeping gene was silenced using GAPDH-specific siRNAs in control or Npro-expressing cells or cells where RNA helicase A had been silenced (RHA siRNA). GAPDH RNA levels were analyzed by Northern blotting with a 32P-labeled GAPDH-specific primer (Fig. 5ii). GAPDH was efficiently knocked down by siRNAs in control cells (Fig. 5ii, left panel, lane 4). There was no difference in GAPDH knockdown between control and Npro mCherry-expressing cells (Fig. 5ii, middle panel, lane 4), showing that Npro does not suppress RNAi from exogenously added siRNAs. It has been previously reported that RNA helicase A inhibits RISC loading (37) and that it might be involved in RNAi, so we investigated whether silencing of RHA had an effect on the RNAi assay of GAPDH knockdown with GAPDH-specific siRNAs, as described above. We silenced RHA with RHA-specific siRNAs (Fig. 5i, right panel). RHA depletion shows some suppression of RNAi from externally added siRNAs (Fig. 5ii, right panel, lane 4), as has been reported previously (37).

Npro does not suppress RNA silencing.(i) Western blot of control HeLa cells (−), cells stably expressing Npro-mCherry (+Npro) (left panel), or HeLa cells silenced for RHA expression (RHA siRNA) (right panel). Npro was detected with a rabbit anti-Npro antibody and visualized with IRDye-labeled secondary antibodies. RHA was detected with mouse anti-RHA and visualized with HRP-labeled secondary antibody. (ii) Northern blot analysis of GAPDH mRNA levels following GAPDH silencing using a 32P-labeled GAPDH-specific primer. 1, control cells; 2, mock-treated cells; 3, nontargeting siRNA-treated cells; 4, GAPDH siRNA-treated cells. Control cells (left panel), cells expressing Npro (middle panel), and cells silenced for RHA (right panel) are shown. An 18S RNA blot is shown for equal loading.

DISCUSSIONPestiviruses are cytoplasmic RNA viruses that must evade host innate immune responses, which include stress, apoptosis, and interferon induction. In this study, we have shown that BVDV can inhibit stress granule formation, adding this to the list of evasion strategies of this virus (7, 8, 40). Viruses must prevent translational silencing in their host cell in order to survive. Many RNA viruses inhibit stress granule formation during infection, whereas others induce them to then inhibit at later times in infection (41). In some cases, virus infection itself can trigger stress granules as an antiviral pathway through the activation of kinases, such as protein kinase R (PKR), that phosphorylate eIF2 to shut off host translation and limit viral replication (42). In the absence of stress and other viral proteins, our proteomics analysis found that Npro bound to many RNA binding proteins, such as the 40S and 60S ribosomal proteins, ribonucleoproteins, and numerous other components of translational machinery in the cytoplasm, including YBX-1, RHA (DHX9), IGF2BP1, DEAD box helicases, eIF3, PABP-1, and heterogeneous nuclear ribonucleoproteins (hnRNPs) (30). We found that the binding was not dependent on the presence of viral RNA or dsRNA [poly(I·C)]. In BVDV-infected cells which had been stressed, Npro redistributed from the cytoplasm to the replication complex, whereas RNP proteins remained cytoplasmic and stress granules were not seen. In uninfected cells, Npro bound to ribonucleoproteins and ribosomal proteins in the cytoplasm, and it was assembled by them into stress granule complexes, since Npro mutated in the C-terminal TRASH motif was no longer assembled into stress granules. Therefore, it is unlikely that Npro is the viral protein responsible for preventing the redistribution of RNPs to stress granules in BVDV-infected cells. This is in contrast to picornaviruses, where expression of the leader (L) protein was sufficient to inhibit stress granule assembly (43). It is possible that Npro recruits these proteins at some point during infection for viral RNA translation. Another possible effect is to stall the translation of host proteins, but only at early stages of infection, since cells persistently infected with pestiviruses do not show translation arrest, and cells are protected from apoptosis (38). We describe here the proteomic content of the RNPs to which Npro binds, but how the proteomic composition is linked to function or changes in different cellular compartments is largely unknown (30). Some RNA viral proteases, such as poliovirus proteases, actively bind and cleave stress granule proteins, including G3BP, PABP, and eIF4G, early during infection (44); nevertheless, stable stress granules containing TIA1 and positive-sense mRNAs form, promoting host cell shutoff (45). The pro- or antiviral functions of stress granule proteins are still unclear, since they act as sites of viral and mRNA sequestration and translational initiation and repression. They may act to reprogram protein expression by silencing mRNAs selectively during infection and also to sequester viral transcripts.We have shown that Npro interacted with YBX1. Interestingly, this is a novel partner of NS3 in hepatitis C virus infection, which redistributes to lipid droplets for virus assembly to control the balance between infectious particle production and viral RNA replication (46). In alphavirus infections, the Sindbis virus protein nsP3 is also found in YBX1 protein-RNA complexes in endosome-like vesicular organelles (47), where they have a function in viral RNA synthesis. In addition, Npro interacts with many of the same proteins seen to interact with the YBX1 interactome, including IGFBP2, DDX3, ILF2, and RHA (DXH9). These are multifunctional proteins, and it has been suggested that the YBX1 macromolecular complex has a panviral role in modulating virus production (48).Both interacting partners RHA and La autoantigen are part of ribonucleoprotein particles, and we have shown here that they can be transported rapidly from the nucleus to the cytoplasm following stress. Nuclear pre-mRNA is packaged into hnRNPs and transported out of the nucleus. Several hnRNP proteins were identified as binding to Npro in our mass spectrometry analysis. During infection, RHA may be recruited to viral RNA (vRNA), since other studies have shown RHA binding to 5′ and 3′ untranslated regions (UTRs) of pestivirus RNA to increase replication, which would indicate a proviral effect (49), whereas binding to the 3′ untranscribed region (UTR) of Dengue virus mediates an antiviral effect (50). The cellular reorganization of RHA with Npro seen in this study is similar to that seen during picornavirus infection, where RHA plays an important part in its replication (51). RHA is phosphorylated by dsRNA binding kinase PKR, normally activated upon viral infection (52), and also binds to nuclear factor kappa B p65, a central regulator of the inflammatory response (53). Other Npro-interacting proteins may have proviral roles; for example, DDX3 binding to vaccinia virus K7 (24) and hepatitis C virus (HCV) core promotes replication (54). Interesting corroboration of the interactions described in this paper comes from a broad study using 70 viral immune modulators from 30 viral species, where they found many of the same interactors we show in Tables 1 and 2 (55). This shows that many other viruses also target this cellular pathway.The localization of RNAi machinery in stress granules highlights their antiviral role in targeting viral RNA, and viruses must subvert RNAi activity, which would otherwise degrade their genomes. Another important protein found to interact with Npro is poly(ADP-ribose) (PARP), which modifies Argonaute 2 function and has a role in miRNA generation (56). Our experiments here show that Npro expression does not suppress the exogenous siRNA pathway, although it is possible that binding of Npro to PARP may repress miRNA production. The host factors RHA and La autoantigen are also important in RNA interference, being involved in RISC loading and unloading (37, 38). We demonstrated that RHA binds to Npro in stress granules, and we asked if Npro inhibited RNAi by using siRNAs to silence the GAPDH housekeeping gene. Npro did not inhibit exogenous siRNA silencing of GAPDH. These results suggest that Npro is not a viral suppressor of silencing (VSS) in mammalian cells or at least is not involved in the siRNA pathway. In some systems, viral antagonists of interferon have been shown to inhibit RNAi but not in some mammalian cells (57).In summary, the N-terminal protease, Npro, of pestiviruses forms a complex with ribosomal and ribonucleoproteins in the cytoplasm which serves to recuit the machinery for translation and production of viral particles. Pestiviruses themselves inhibit oxidative stress-induced stress granule formation, which would prevent inhibition of cellular protein synthesis and promote viral protein synthesis. The modulation of viral replication by these interacting protein remains to be determined.

Pestiviruses are cytoplasmic RNA viruses that must evade host innate immune responses, which include stress, apoptosis, and interferon induction. In this study, we have shown that BVDV can inhibit stress granule formation, adding this to the list of evasion strategies of this virus (7, 8, 40). Viruses must prevent translational silencing in their host cell in order to survive. Many RNA viruses inhibit stress granule formation during infection, whereas others induce them to then inhibit at later times in infection (41). In some cases, virus infection itself can trigger stress granules as an antiviral pathway through the activation of kinases, such as protein kinase R (PKR), that phosphorylate eIF2 to shut off host translation and limit viral replication (42). In the absence of stress and other viral proteins, our proteomics analysis found that Npro bound to many RNA binding proteins, such as the 40S and 60S ribosomal proteins, ribonucleoproteins, and numerous other components of translational machinery in the cytoplasm, including YBX-1, RHA (DHX9), IGF2BP1, DEAD box helicases, eIF3, PABP-1, and heterogeneous nuclear ribonucleoproteins (hnRNPs) (30). We found that the binding was not dependent on the presence of viral RNA or dsRNA [poly(I·C)]. In BVDV-infected cells which had been stressed, Npro redistributed from the cytoplasm to the replication complex, whereas RNP proteins remained cytoplasmic and stress granules were not seen. In uninfected cells, Npro bound to ribonucleoproteins and ribosomal proteins in the cytoplasm, and it was assembled by them into stress granule complexes, since Npro mutated in the C-terminal TRASH motif was no longer assembled into stress granules. Therefore, it is unlikely that Npro is the viral protein responsible for preventing the redistribution of RNPs to stress granules in BVDV-infected cells. This is in contrast to picornaviruses, where expression of the leader (L) protein was sufficient to inhibit stress granule assembly (43). It is possible that Npro recruits these proteins at some point during infection for viral RNA translation. Another possible effect is to stall the translation of host proteins, but only at early stages of infection, since cells persistently infected with pestiviruses do not show translation arrest, and cells are protected from apoptosis (38). We describe here the proteomic content of the RNPs to which Npro binds, but how the proteomic composition is linked to function or changes in different cellular compartments is largely unknown (30). Some RNA viral proteases, such as poliovirus proteases, actively bind and cleave stress granule proteins, including G3BP, PABP, and eIF4G, early during infection (44); nevertheless, stable stress granules containing TIA1 and positive-sense mRNAs form, promoting host cell shutoff (45). The pro- or antiviral functions of stress granule proteins are still unclear, since they act as sites of viral and mRNA sequestration and translational initiation and repression. They may act to reprogram protein expression by silencing mRNAs selectively during infection and also to sequester viral transcripts.

We have shown that Npro interacted with YBX1. Interestingly, this is a novel partner of NS3 in hepatitis C virus infection, which redistributes to lipid droplets for virus assembly to control the balance between infectious particle production and viral RNA replication (46). In alphavirus infections, the Sindbis virus protein nsP3 is also found in YBX1 protein-RNA complexes in endosome-like vesicular organelles (47), where they have a function in viral RNA synthesis. In addition, Npro interacts with many of the same proteins seen to interact with the YBX1 interactome, including IGFBP2, DDX3, ILF2, and RHA (DXH9). These are multifunctional proteins, and it has been suggested that the YBX1 macromolecular complex has a panviral role in modulating virus production (48).

Both interacting partners RHA and La autoantigen are part of ribonucleoprotein particles, and we have shown here that they can be transported rapidly from the nucleus to the cytoplasm following stress. Nuclear pre-mRNA is packaged into hnRNPs and transported out of the nucleus. Several hnRNP proteins were identified as binding to Npro in our mass spectrometry analysis. During infection, RHA may be recruited to viral RNA (vRNA), since other studies have shown RHA binding to 5′ and 3′ untranslated regions (UTRs) of pestivirus RNA to increase replication, which would indicate a proviral effect (49), whereas binding to the 3′ untranscribed region (UTR) of Dengue virus mediates an antiviral effect (50). The cellular reorganization of RHA with Npro seen in this study is similar to that seen during picornavirus infection, where RHA plays an important part in its replication (51). RHA is phosphorylated by dsRNA binding kinase PKR, normally activated upon viral infection (52), and also binds to nuclear factor kappa B p65, a central regulator of the inflammatory response (53). Other Npro-interacting proteins may have proviral roles; for example, DDX3 binding to vaccinia virus K7 (24) and hepatitis C virus (HCV) core promotes replication (54). Interesting corroboration of the interactions described in this paper comes from a broad study using 70 viral immune modulators from 30 viral species, where they found many of the same interactors we show in Tables 1 and 2 (55). This shows that many other viruses also target this cellular pathway.

The localization of RNAi machinery in stress granules highlights their antiviral role in targeting viral RNA, and viruses must subvert RNAi activity, which would otherwise degrade their genomes. Another important protein found to interact with Npro is poly(ADP-ribose) (PARP), which modifies Argonaute 2 function and has a role in miRNA generation (56). Our experiments here show that Npro expression does not suppress the exogenous siRNA pathway, although it is possible that binding of Npro to PARP may repress miRNA production. The host factors RHA and La autoantigen are also important in RNA interference, being involved in RISC loading and unloading (37, 38). We demonstrated that RHA binds to Npro in stress granules, and we asked if Npro inhibited RNAi by using siRNAs to silence the GAPDH housekeeping gene. Npro did not inhibit exogenous siRNA silencing of GAPDH. These results suggest that Npro is not a viral suppressor of silencing (VSS) in mammalian cells or at least is not involved in the siRNA pathway. In some systems, viral antagonists of interferon have been shown to inhibit RNAi but not in some mammalian cells (57).

In summary, the N-terminal protease, Npro, of pestiviruses forms a complex with ribosomal and ribonucleoproteins in the cytoplasm which serves to recuit the machinery for translation and production of viral particles. Pestiviruses themselves inhibit oxidative stress-induced stress granule formation, which would prevent inhibition of cellular protein synthesis and promote viral protein synthesis. The modulation of viral replication by these interacting protein remains to be determined.
