---
pmid: '9271120'
title: 'Repression of p53-mediated transcription by MDM2: a dual mechanism.'
authors:
- Thut CJ
- Goodrich JA
- Tjian R
journal: Genes Dev
year: '1997'
full_text_available: true
full_text_extraction_method: html
pmcid: PMC316412
doi: 10.1101/gad.11.15.1974
---

# Repression of p53-mediated transcription by MDM2: a dual mechanism.
**Authors:** Thut CJ, Goodrich JA, Tjian R
**Journal:** Genes Dev (1997)
**DOI:** [10.1101/gad.11.15.1974](https://doi.org/10.1101/gad.11.15.1974)
**PMC:** [PMC316412](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC316412/)

## Abstract

1. Genes Dev. 1997 Aug 1;11(15):1974-86. doi: 10.1101/gad.11.15.1974.

Repression of p53-mediated transcription by MDM2: a dual mechanism.

Thut CJ(1), Goodrich JA, Tjian R.

Author information:
(1)Howard Hughes Medical Institute, Department of Molecular and Cell Biology, 
University of California, Berkeley 94720-3204, USA.

The oncoprotein MDM2 binds to the activation domain of the tumor suppressor p53 
and inhibits its ability to stimulate transcription. This same region of p53 is 
able to bind several basal transcription factors that appear to be important for 
the transactivation function of p53. It has therefore been suggested that MDM2 
acts to inhibit p53 by concealing its activation domain from the basal 
machinery. Here we present data suggesting that MDM2 possesses an additional 
inhibitory function. Our experiments reveal that in addition to a p53-binding 
domain, MDM2 also contains an inhibitory domain that can directly repress basal 
transcription in the absence of p53. By fusing portions of MDM2 to a 
heterologous DNA-binding domain to allow p53-independent promoter recruitment, 
we have localized this inhibitory domain to a region encompassing amino acids 
50-222 of MDM2. Furthermore, the function of this inhibitory domain does not 
require the presence of either TFIIA or the TAFs. Of the remaining basal 
factors, both the small subunit of TFIIE and monomeric TBP are bound by the MDM2 
inhibitory domain. It is possible that MDM2 inhibits the ability of the 
preinitiation complex to synthesize RNA through one of these interactions. Our 
results are consistent with a model in which MDM2 represses p53-dependent 
transcription by a dual mechanism: a masking of the activation domain of p53 
through a protein-protein interaction that additionally serves to recruit MDM2 
to the promoter where it directly interferes with the basal transcription 
machinery.

DOI: 10.1101/gad.11.15.1974
PMCID: PMC316412
PMID: 9271120 [Indexed for MEDLINE]

## Full Text

Abstract

The oncoprotein MDM2 binds to the activation domain of the tumor suppressor p53 and inhibits its ability to stimulate transcription. This same region of p53 is able to bind several basal transcription factors that appear to be important for the transactivation function of p53. It has therefore been suggested that MDM2 acts to inhibit p53 by concealing its activation domain from the basal machinery. Here we present data suggesting that MDM2 possesses an additional inhibitory function. Our experiments reveal that in addition to a p53-binding domain, MDM2 also contains an inhibitory domain that can directly repress basal transcription in the absence of p53. By fusing portions of MDM2 to a heterologous DNA-binding domain to allow p53-independent promoter recruitment, we have localized this inhibitory domain to a region encompassing amino acids 50–222 of MDM2. Furthermore, the function of this inhibitory domain does not require the presence of either TFIIA or the TAFs. Of the remaining basal factors, both the small subunit of TFIIE and monomeric TBP are bound by the MDM2 inhibitory domain. It is possible that MDM2 inhibits the ability of the preinitiation complex to synthesize RNA through one of these interactions. Our results are consistent with a model in which MDM2 represses p53-dependent transcription by a dual mechanism: a masking of the activation domain of p53 through a protein–protein interaction that additionally serves to recruit MDM2 to the promoter where it directly interferes with the basal transcription machinery.

Discussion

In this report we provide evidence that MDM2, an oncoprotein that inhibits p53-dependent transcription, contains a previously undetected repression domain between amino acids 50 and 222 that can inhibit basal transcription in a p53-independent manner. Several mechanisms have been proposed to explain the ability of repressors to inhibit basal transcription, including direct inhibition of the activities of the general transcription machinery by the repressor ( Sauer et al. 1995 ), modification of chromatin structure near the promoter ( Bunker and Kingston 1994 ), and recruitment of additional factors or “corepressors” that themselves mediate transcriptional inhibition ( Keleher et al. 1992 ; Paroush et al. 1994 ). We have demonstrated that MDM2 is able to inhibit basal transcription in a highly purified system devoid of chromatin. Furthermore, this system is not likely to contain potential corepressors, unless they are tightly associated with either Pol II or TFIIH. Our studies of the repression domain of MDM2 thus suggest that this domain directly inhibits one or more functions of the basal machinery. In an attempt to identify components of the basal machinery that might be targeted by MDM2, we used a combination of in vitro transcription assays and in vitro binding assays to define a subset of the basal factors sufficient to support MDM2-dependent basal inhibition and to identify potential contacts between MDM2 and the general transcription machinery. These studies led to the finding that neither TFIIA nor the TAFs are necessary for the ability of MDM2 to inhibit transcription. Of the remaining basal factors, MDM2 was found to interact with both the 34K subunit of TFIIE and monomeric TBP, suggesting that through these contacts MDM2 might inhibit their ability to function in the preinitiation complex. However, the additional finding that the inhibitory domain of MDM2 was not able to interact with TBP when in complex with TAFs leads us to speculate that the interaction between MDM2 and TBP may not be functionally relevant.

Our finding that the 34K subunit of TFIIE may be the target of the repressive activities of MDM2 was intriguing because it has been shown previously that a Drosophila repressor, Krüppel, also directly interacts with the 34K subunit of TFIIE ( Sauer et al. 1995 ). During transcription initiation, TFIIE is thought to play a key role in the ability of the general transcription machinery to progress from the promoter-bound complex to the actively elongating complex, and repressors that target TFIIE might disrupt this transition by inhibiting one or more of the known functions of TFIIE. TFIIE appears to work in concert with the basal transcription factor TFIIH to facilitate DNA strand separation at the promoter, leading to promoter clearance and elongation ( Schaeffer et al. 1993 ; Goodrich and Tjian 1994b ; Holstege et al. 1995 , 1996 ). TFIIE is also believed to be required for the recruitment of TFIIH ( Flores et al. 1992 ; Maxon et al. 1994 ), a multisubunit complex that contains several enzymatic activities itself, including helicase, kinase, and ATPase ( Sopta et al. 1989 ; Flores et al. 1992 ; Schaeffer et al. 1993 ; Serizawa et al. 1993 ; Roy et al. 1994 ). These enzymatic properties of TFIIH have been shown to be regulated in vitro by TFIIE ( Lu et al. 1992 ; Ohkuma and Roeder 1994 ; Serizawa et al. 1994 ; Ohkuma et al. 1995 ). By binding to the small subunit of TFIIE, it is possible that MDM2 affects the ability of TFIIE to recruit TFIIH or to influence its kinase, helicase, or ATPase activities. Alternatively, TFIIE may play a more direct role in DNA strand separation or recruitment and stability of the preinitiation complex, offering other potential functions to be targeted by MDM2 ( Maxon et al. 1994 ; Holstege et al. 1995 ). Future studies will address the ability of MDM2 to inhibit these specific functions of TFIIE in an attempt to determine the mechanism by which MDM2 can directly repress basal transcription.

Although the precise mechanisms by which specific repressors exert their inhibitory activities are not well known, at least three types of repressors have been proposed to exist: those that compete with activators for promoter binding sites, “quenchers” that directly inhibit the action of specific transactivators, and inhibitors that directly repress the functioning of the basal transcription machinery ( Levine and Manley 1989 ; Johnson 1995 ). Much circumstantial evidence has been accumulated to suggest that MDM2 represses the transactivating functions of p53 by masking or interfering with its activation domain. This domain of p53 has been shown to interact with several components of the transcription machinery, including TAF II 32 ( Lu and Levine 1995 ; Thut et al. 1995 ), TAF II 70 ( Thut et al. 1995 ), TBP ( Seto et al. 1992 ; Chen et al. 1993 ; Liu et al. 1993 ; Truant et al. 1993 ), and TFIIH ( Xiao et al. 1994 ). A functional importance for these interactions has been shown in the case of TAF II 32 and TAF II 70 ( Thut et al. 1995 ). Interestingly, the same mutation in the activation domain of p53 that impairs its ability to bind TAF II 32 ( Lu and Levine 1995 ; Thut et al. 1995 ), TAF II 70 ( Thut et al. 1995 ), and TBP (amino acids 22 and 23) also disrupts its ability to contact MDM2 ( Lin et al. 1994 ), suggesting that MDM2 and the basal factors may interact with overlapping portions of p53. Furthermore, the recent crystal structure of the p53/MDM2 interaction interface showed that residues 22 and 23 are contained within this interface. These correlations lend further support to the hypothesis that MDM2 inhibits p53-dependent activation by disrupting interactions with components of the general transcription machinery.

Our results suggest that MDM2 can also directly inhibit the ability of the general transcription factors to potentiate mRNA synthesis. However, we do not believe that our findings exclude the possibility that MDM2 may also inhibit p53-dependent transcription by masking its activation domain from important basal factor contacts. On the contrary, the experiments described here, in conjuction with the correlation between the MDM2 and basal factor binding sites in p53’s activation domain, suggest that it is likely that MDM2 functions as both a quencher of p53 and a direct inhibitor of the basal machinery (Fig. 6 ). In addition, it is possible that these two mechanisms of inhibition by MDM2 complement each other and enhance the ability of MDM2 to regulate p53-responsive promoters. In support of this hypothesis, a similar dual mechanism of inhibition involving both potential masking of an activator and recruitment of a general repression domain has also been proposed to explain the inhibition of E2F-dependent transcription by Rb ( Bremner et al. 1995 ; Sellers et al. 1995 ; Weintraub et al. 1995 ). Much like the MDM2/p53 case, an interaction between the activation domain of E2F and Rb has been shown to be important for the ability of Rb to inhibit transcription ( Flemington et al. 1993 ; Hagemeier et al. 1993 ; Helin et al. 1993 ). In addition, Rb is capable of repressing transcription in an E2F-independent manner when fused to a heterologous DNA-binding domain, although it is not clear whether Rb targets the basal machinery or whether it serves to quench nearby activators ( Bremner et al. 1995 ; Sellers et al. 1995 ; Weintraub et al. 1995 ). These results suggest that transcriptional inhibitors that repress transcription via multiple, distinct mechanisms are not so uncommon.

If MDM2 can inhibit the transactivating ability of p53 by directly interacting with this protein, why then should MDM2 contain an additional inhibitory domain? We believe that the answer to this question is best addressed in the context of complex eukaryotic promoters. Differential gene expression, especially in metazoans, is regulated by integrating a wide variety of positive and negative growth signals that dictate the level of transcripts arising from a particular promoter. This integration is likely to occur by the cooperative action of multiple transcriptional regulators bound at a promoter, each of which can influence the functions of the basal transcription machinery. Because most complex regulatory elements bear multiple activator binding sites, it is expected that efficient inhibition of transcription from such a promoter would require either multiple repressors affecting different sets of activators or a single repressor or silencer capable of impeding transactivation of multiple enhancer factors. The current masking model for MDM2 inhibition of p53-dependent transcription would predict that MDM2 could repress only activating signals from p53 itself but would leave the activity of other activators bound to the same promoter region largely unaffected. We suggest that the basal inhibitory domain of MDM2 functions to repress the effects of other activators bound at p53-responsive promoters (Fig. 6 ). In this way, MDM2 may serve to cause a more stringent regulation of p53-responsive promoters by decreasing both p53-dependent and -independent activation. If this model is correct, one would predict that MDM2 might be able to inhibit transcription stimulated by a subset of activators other than p53 if MDM2 could be recruited to the promoter in a p53-independent manner. In an attempt to test this hypothesis, we asked whether G4–MDM2 could inhibit transcription enhanced by several additional activators including Sp1, CTF ( C CAAT box-binding t ranscription f actor), and Lex–VP16. However, in none of these cases was transcription repressed by G4–MDM2 (data not shown). Although these results do not support our model, they also do not rule it out. First, transcriptional activators are likely to stimulate the activities of the basal machinery by diverse mechanisms. Some transactivators may be able to overcome the repressive effects of MDM2, perhaps by stimulating a step required for initiation that occurs downstream of the step inhibited by MDM2. For this reason, we believe that it is quite possible that MDM2 might only repress a subset of transcriptional activators and that we have not yet tested one that is appropriate. Second, basal promoters themselves are not equivalent. To alter the number of transcripts arising from a particular promoter, a transcriptional modulator must presumably affect the rate-limiting step at that promoter (e.g., basal factor recruitment, DNA strand separation, promoter clearance, etc.). This rate-limiting step will vary because of differences in the core promoter sequences and may cause some promoters to be insensitive to the effects of the inhibitory domain of MDM2. Finally, it is conceivable that in a bona fide p53-regulated promoter/enhancer, repression of p53 by MDM2 alone is sufficient to weaken the levels of transcription because of the cooperative or synergistic interplay of multiple activators that may be disrupted by repressors. Future studies using cellular, p53-responsive promoters will help to address the potential in vivo role of the inhibitory domain of MDM2 that we have described here. In addition, these studies may lead to a better understanding of the capacity of MDM2 to antagonize the biological effects of p53 and will extend our understanding of general mechanisms of transcriptional repression.
