---
pmid: '30814734'
title: Structural basis for KCTD-mediated rapid desensitization of GABA(B) signalling.
authors:
- Zheng S
- Abreu N
- Levitz J
- Kruse AC
journal: Nature
year: '2019'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC6405316
doi: 10.1038/s41586-019-0990-0
---

# Structural basis for KCTD-mediated rapid desensitization of GABA(B) signalling.
**Authors:** Zheng S, Abreu N, Levitz J, Kruse AC
**Journal:** Nature (2019)
**DOI:** [10.1038/s41586-019-0990-0](https://doi.org/10.1038/s41586-019-0990-0)
**PMC:** [PMC6405316](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6405316/)

## Abstract

1. Nature. 2019 Mar;567(7746):127-131. doi: 10.1038/s41586-019-0990-0. Epub 2019 
Feb 27.

Structural basis for KCTD-mediated rapid desensitization of GABA(B) signalling.

Zheng S(1), Abreu N(2), Levitz J(2), Kruse AC(3).

Author information:
(1)Department of Biological Chemistry and Molecular Pharmacology, Blavatnik 
Institute, Harvard Medical School, Boston, MA, USA.
(2)Department of Biochemistry, Weill Cornell Medicine, New York, NY, USA.
(3)Department of Biological Chemistry and Molecular Pharmacology, Blavatnik 
Institute, Harvard Medical School, Boston, MA, USA. 
andrew_kruse@hms.harvard.edu.

The GABAB (γ-aminobutyric acid type B) receptor is one of the principal 
inhibitory neurotransmitter receptors in the brain, and it signals through 
heterotrimeric G proteins to activate a variety of effectors, including 
G-protein-coupled inwardly rectifying potassium channels (GIRKs)1,2. 
GABAB-receptor signalling is tightly regulated by auxiliary subunits called 
KCTDs, which control the kinetics of GIRK activation and desensitization3-5. 
However, the mechanistic basis for KCTD modulation of GABAB signalling remains 
incompletely understood. Here, using a combination of X-ray crystallography, 
electron microscopy, and functional and biochemical experiments, we reveal the 
molecular details of KCTD binding to both GABAB receptors and G-protein βγ 
subunits. KCTDs associate with the receptor by forming an asymmetric pentameric 
ring around a region of the receptor carboxy-terminal tail, while a second KCTD 
domain, H1, engages in a symmetric interaction with five copies of Gβγ in which 
the G-protein subunits also interact directly with one another. We further show 
that KCTD binding to Gβγ is highly cooperative, defining a model in which KCTD 
proteins cooperatively strip G proteins from GIRK channels to induce rapid 
desensitization following receptor activation. These results provide a framework 
for understanding the molecular basis for the precise temporal control of GABAB 
signalling by KCTD proteins.

DOI: 10.1038/s41586-019-0990-0
PMCID: PMC6405316
PMID: 30814734 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing interests.

## Full Text

The GABAB receptor (GABABR) is expressed in excitatory and inhibitory synapses throughout the brain, and it is an important target for anxiolytic and antispastic drugs1,2. The receptor is a heterodimer composed of GABAB1 and GABAB2 subunits, both of which are essential for the formation and trafficking of a functional receptor6. Upon activation, the receptor catalyzes dissociation of heterotrimeric G proteins into Gαi/o and Gβγ subunits to initiate downstream signaling7,8. Following activation of presynaptic GABABRs, the Gβγ heterodimer typically inhibits voltage-gated calcium channels (VGCCs) to suppress neurotransmitter release, while activation of postsynaptic GABABRs usually activates outward potassium conductance via G protein-coupled inwardly rectifying potassium channels (GIRKs)9–11 to generate slow inhibitory postsynaptic currents (sIPSCs)11,12.

GABAB signaling is controlled by the proteins KCTD8, KCTD12, and KCTD16, which serve as auxiliary receptor subunits that regulate the rise time and duration of GIRK currents and enhance receptor expression levels3–5. These effects are mediated by KCTD interaction with the GABAB2 receptor C-terminus and with Gβγ proteins4,13. KCTDs are genetically associated with mood disorders including bipolar disorder14, and their heterogeneous subtype expression patterns serve to create pharmacologically distinct populations of GABAB receptors in different brain regions15. Despite their importance, the mechanisms of KCTD receptor binding and G protein sequestration are ill-defined. Here, we sought to tackle these problems to define the structural and mechanistic basis for KCTD function.

KCTDs 8, 12, and 16 consist of an N-terminal BTB domain followed by a region of unknown structure termed the “H1” domain. The BTB domain mediates binding to GABAB receptors at the distal receptor tail centered around GABAB2 Tyr903 (we use human sequence numbering throughout; this residue is equivalent to Tyr902 in mouse)13,16. We sought to identify a GABAB2 peptide fragment capable of binding to a KCTD protein. We found that a peptide consisting of GABAB2 residues 876 – 913 was capable of binding to KCTD16BTB (Fig. 1a and Extended Data Fig. 1a and 1b), and we determined the crystal structure of this peptide in complex with KCTD16BTB to 3.2 Å resolution (Fig. 1b and Extended Data Table 1 and Extended Data Fig. 2).

The structure shows an unusual overall architecture, with KCTD16BTB adopting an asymmetric pentameric arrangement and wrapping around the GABAB2 C-terminus (Fig. 1b). While previous reports suggested that KCTD16 functions as a tetramer, both our structure and a recently reported peptide-free KCTD16BTB structure showed a pentameric arrangement17 (Extended Data Fig 1c). Over a stretch of roughly 25 amino acids, nearly every side chain in the GABAB2 tail interacts with KCTD16BTB (Fig. 1b). Within the interface, KCTD16BTB residue Phe80 makes particularly extensive contacts with the peptide in each of the five subunits (Fig. 1c). Confirming the importance of these interactions, we found that an F80A mutation caused a complete loss of GABAB2 peptide binding activity in vitro (Extended Data Fig. 1d), and an equivalent substitution in KCTD12 resulted in loss of GABAB-mediated KCTD12 membrane localization in cells (Fig. 1d). Almost all residues in the interaction interface are conserved in all GABAB-regulating KCTD subtypes (KCTD8, KCTD12 and KCTD16), accounting for the specific binding of these KCTDs for GABAB receptor (Extended Data Fig. 1e
).

In addition to binding the GABAB2 receptor tail, KCTDs also interact with Gβγ subunits4. In an in vitro pull-down assay with GST-tagged KCTD12 we found that a sub-stochiometric amount of GABAB2 receptor tail but near stochiometric amounts of Gβ1γ2 heterodimer could be pulled down by KCTD12. In contrast, KCTD12 failed to co-purify Gαβγ heterotrimer irrespective of Gα subtype (Fig. 2a and Extended Data Fig. 3a and 3b). While the KCTD12 BTB domain mediates interaction with the receptor, its C-terminal H1 domain is responsible for desensitization18. Consistent with this, we observed that the KCTD12 H1 domain alone was sufficient to bind Gβγ with an affinity of 185 nM (Fig. 2b; Extended Data Figure 3c). Negative stain electron microscopy (EM) 2D class averages of KCTD12H1 in complex with Gβγ revealed a symmetric pentameric assembly with the H1 domain pentamer engaging in interaction with five copies of Gβγ heterodimer (Fig. 2b and Extended Data Fig. 3d–f).

To investigate the details of H1 domain interaction with Gβγ in more detail, we determined the crystal structure of the KCTD12H1/Gβγ complex to 3.7 Å resolution (Extended Data Table 1 and Extended Data Fig. 2b). In parallel, we also determined a negative-stain EM envelope for full-length KCTD12 in complex with Gβγ (Extended Data 4). These data showed that the KCTD12H1/Gβγ complex is pentameric and shows near-perfect C5 symmetry. KCTD12H1 possesses a β-propeller-like fold, with each H1 domain comprising a separate blade of the propeller. The propeller is surrounded by a tightly packed ring of Gβγ heterodimers in direct contact with one another. Interactions between KCTD12 and the G protein are confined to the Gβ subunit, with Gγ subunits peripherally arrayed around the edge of the pentameric ring (Fig. 2c). Previous reports had suggested that the motif NFLEQ in KCTD12 is important for receptor desensitization18. However, in the crystal structure this region does not interact with Gβγ subunits, but instead is involved in oligomeric contacts between adjacent KCTD12 subunits (Extended Data Fig. 5a).

The KCTD12 H1 domain itself consists of a five-stranded antiparallel β-sheet interrupted by two α helices (Extended Data Fig. 5b). The five Gβγ subunits each interact with two KCTD12H1 subunits, which together occlude a large surface on Gβ (Fig. 3a and Extended Data Fig. 5c). The loop between the first and second β strands of KCTD12H1 includes basic residues which interact with an acidic patch on the Gβ propeller (Fig. 3b), and a longer loop in KCTD12H1 after the first α helix sits in the groove between the N-terminal helix of Gβγ and the β propeller domain (Fig. 3c). Mutating residues in the KCTD12 H1 domain in either of these interfaces (R232D or R257D) entirely abolished G protein binding (Fig. 3d). Although both KCTD mutants can localize to the cell membrane in the presence of GABAB receptors (Extended Data Fig. 5d), they fail to desensitize GABAB-mediated GIRK channel currents (Fig. 3e and 3f). Interestingly, the KCTD binding site on the surface of Gβγ would partially occlude Gα binding (Fig. 3g), but is entirely non-overlapping with the GIRK channel binding site (Fig. 3g and 3h)19. Hence, binding of a KCTD12 pentamer to a single Gβγ subunit in complex with GIRK2 could be accommodated with minimal structural rearrangement, although the full 5:5 complex is incompatible due to extensive clashes between the GIRK2 intracellular domain and other KCTD12-bound Gβγ dimers (Fig. 3h and Extended Data Fig. 6a and 6b).

One of the more remarkable features of the complex is the fact that Gβγ subunits pack so closely to one another that they interact directly, burying a 351 Å2 surface. In contrast, the GIRK channel is bound by four spatially isolated Gβγ subunits (Extended Data Fig. 6a–c). Residues in the Gβ-Gβ interface observed in our structure are highly conserved across species, raising the possibility that this interaction is functionally important. To test this, we incubated KCTD12H1 with a substoichiometric amount of Gβγ. Under these conditions, only full 5:5 complexes and free KCTD12 were seen in size exclusion chromatography, with no evidence of partial oligomers, suggesting that KCTD12 binding to Gβγ is highly cooperative (Extended Data Fig. 6d). When a single Gβ residue involved in oligomeric Gβ-Gβ contacts was mutated (R42D), we observed a broad 5:5 complex peak and free Gβγ in size exclusion. Introduction of a second mutation (R46D) resulted in only partial complex formation (Extended Data Fig. 6e), further supporting a role of cooperative Gβ-Gβ contacts in KCTD binding.

Gβγ is the key signaling intermediate in GABAB activation of GIRK currents, and in the course of a signaling event a Gβγ subunit sequentially interacts with a series of progressively higher affinity binding partners. The KD’s for Gβγ binding to GTP-bound Gαi, GIRK channels, KCTD12, and GDP-bound Gαi are (in order): undetectable, ~250 μM20,21, 185 nM (Extended Data 3c), and 3 nM22. If Gβγ affinity for KCTDs is so much higher than for GIRK channels, how can KCTDs desensitize the channels without preventing their activation in the first place? Clearly the on rate for Gβγ association with KCTDs must be lower than that for association with GIRK channels, otherwise Gβγ subunits would be sequestered prematurely. We propose one possible explanation for how this might be achieved.

KCTDs bind at the distal C-terminus of the GABA receptor B2 subunit, roughly 350 Å from the last transmembrane domain with an intervening structurally rigid coiled-coil domain. This means that KCTDs can diffuse far from the membrane plane (Fig. 4). When the GABAB receptor is activated, Gβγ subunits are liberated from Gα but remain tethered to the membrane due to their lipid anchor, resulting in GIRK channel activation while KCTDs are at a relatively low local concentration. When a KCTD molecule happens to encounter a GIRK-bound Gβγ subunit it can bind simultaneously, but formation of the oligomeric complex is sterically incompatible with Gβγ binding to GIRK channels. The higher affinity and cooperativity of binding to KCTD may then drive stripping of subsequent Gβγ subunits from GIRKs following an initial encounter, resulting in rapid channel closure. Finally, reassociation of Gβγ with GDP-bound Gα subunits could restore equilibrium and reset the cycle (Extended Data 7).

The regulation of GABAB signaling by KCTD proteins is a unique and important means of achieving functional diversity in these critical receptors. Here we have shown how the KCTD BTB domain recognizes the GABAB2 C-terminus to tether a Gβγ-sequestering H1 domain on the receptor’s distal tail, providing a means of efficiently deactivating GIRK channels. In this way KCTDs afford GABAB receptors with tight temporal control of signaling not achievable with the slower β-arrestin pathway used to desensitize most GPCRs.

METHODSProtein Expression and PurificationHuman KCTD12 and KCTD16 genes were synthesized (Integrated DNA Technologies). KCTD12 (residues 32–131) and KCTD16 (residues 23–124) BTB domains were cloned into a pET28a expression vector (EMD Millipore) for the expression of an N-terminal His6-SUMO fusion protein or an untagged version in Escherichia coli strain BL21(DE3). KCTD12 (residues 200–325) and KCTD16 (residues 158–279) H1 domain were cloned into the multiple cloning site 1 of pETduet-1 vector (EMD Millipore) and expressed with an N-terminal His6-SUMO tag followed by 3C protease site. All human GABAB2 C-terminal truncations were cloned into the same vector with an N-terminal His6-GST or His6-SUMO fusion protein followed by 3C protease site and were expressed in BL21(DE3) E. coli. KCTD12 (residues 32–325) and KCTD16 (residues 21–279) constructs containing both BTB and H1 domains were cloned into the vector pVL1393 vector (Expression Systems) and protein was expressed using baculovirus infection of Sf9 insect cells (Expression Systems) according the manufacturer’s protocols. These constructs were expressed with an N-terminal His6-GST fusion protein followed by a 3C protease site. Human Gαq was cloned into pVL1393 without tags. Human Gβ1γ2 heterodimer was modified to include an N-terminal His6 tag on the Gβ subunit and a C68S point mutation in Gγ, which eliminates the lipid modification site. A bicistronic vector based on pVL1392 was used to prepare baculovirus encoding both subunits. All point mutations and deletions were introduced by the QuikChange method.Plasmids encoding untagged KCTD16 BTB and His6-GST-tagged GABAB2 fragment were co-transformed into E. coli BL21(DE3) strain. Bacteria were cultured in LB medium supplemented with 50 μg ml−1 ampicillin and 50 μg ml−1 kanamycin at 37 °C to an OD600 value of 0.8, and protein expression was induced by the addition of 0.5 mM isopropyl 1-thio-β-D-glucopyranoside followed by shaking overnight at 18 °C. Cells were harvested by centrifugation, resuspended in buffer A (50 mM Tris-HCl, pH 8.0, 250 mM NaCl and 25 mM Imidazole) and lysed by sonication. His-GST-GABAB2 fragment in complex with KCTD16BTB was purified from clarified lysate by Ni-NTA affinity chromatography. Resin was washed with buffer A and protein then eluted with buffer B (50 mM Tris-HCl, pH 8.0, 250 mM NaCl and 250 mM Imidazole). The eluted fraction was incubated with 3C protease overnight to cleave His6-GST tag. Protein solution was diluted three-fold by 20 mM HEPES-NaOH, pH 7.6 and loaded onto a Q sepharose column (GE Healthcare). The flow-through was collected, concentrated and further purified by size exclusion chromatography (SEC) Superdex S200 10/300 equilibrated in storage buffer (10 mM HEPES-NaOH pH 7.6, and 150 mM NaCl). Protein expression and nickel affinity purification for His6-SUMO-tagged KCTD12 H1 was performed as described above. After tag cleavage by 3C protease, the protein sample was concentrated and purified on a Superdex S200 10/300 size exclusion column equilibrated in storage buffer.His6-GST-tagged KCTD12 32–325 or KCTD16 21–279 was expressed in Sf9 insect cell cultures using the BestBac baculovirus system (Expression Systems). Infection was performed when cells reached a density of 3 × 106 cell per mL and flasks were shaken at 27 °C for 36 hours before harvest. Cells were harvested by centrifugation and lysed in buffer A using a glass dounce tissue grinder. Protein was purified by Ni-NTA gravity flow chromatography followed by glutathione agarose bead affinity purification (GE Healthcare). The protein was further purified by Superdex S200 10/300 equilibrated in storage buffer. His6-tagged Gβ1 and Gγ2 (C68S mutant) were expressed in Sf9 insect cells using the BestBac system (Expression Systems) as described above. Purification was performed as previously reported23.For KCTD12H1/Gβ1γ2 complex assembly and purification, individually purified KCTD12 H1 and Gβ1γ2 proteins were mixed in a 1:1.2 molar ratio and then incubated at room temperature for 30 min. Excess Gβ1γ2 was separated from KCTD12 H1/Gβ1γ2 complex on a Superdex S200 10/300 column in binding buffer (20 mM HEPES-NaCl pH 7.6 and 50 mM NaCl). For KCTD16 21–279/GABAB2 876–913 complex, purified His6-SUMO-tagged GABAB2 876–913 and His6-GST-tagged KCTD16 21–279 were mixed in a 2:1 molar ratio. The His6-SUMO tag of GABAB2 and His6-GST tag of KCTD16 21–279 were cleaved by 3C protease at 4°C overnight. The protein complex was further purified by SEC S200 10/300 with storage buffer. For KCTD12 32–325/GABAB2 876–913/ Gβ1γ2 complex, individually purified His6-GST-tagged KCTD12 32–325, His6-SUMO-tagged GABAB2 876–913, and Gβ1γ2 were mixed in a 1:2:1.2 ratio. After tag cleavage by 3C protease the protein sample was loaded onto Superdex S200 10/300 equilibrated in binding buffer.GST pull-down assayTo identify a minimal portion of GABAB2 suitable for binding KCTDs, a series of GABAB2 C-terminal truncations were cloned. His6-SMT3 tagged KCTD12 or KCTD16 and His6-GST tagged GABAB2 truncation were co-transformed and co-expressed in E. coli BL21(DE3) strain. GST pull-down experiment was performed as previously described24. For interaction analysis, individually purified His6-SUMO-tagged GABAB2 876–913 or Gβ1γ2 complex or Gαqβ1γ2 complex was added to His6-GST-tagged KCTD12 32–325 in an excessive amount in 200 μl of binding buffer. The mixtures were incubated with 20 μL of glutathione-Sepharose beads and gently rotated at 4 °C for 1 hour. After the beads were washed three times with binding buffer, the bound protein was eluted with 20 μL of binding buffer supplemented with 20 mM glutathione. Input and eluate samples were resolved using SDS-PAGE and stained with Coomassie Blue.Crystallization and structure determinationThe KCTD16 BTB domain (23–124) and GABAB2 fragment (876–913) were crystallized at 20 °C using the hanging-drop vapor diffusion method by mixing 0.5 μL of protein (19 mg mL−1) and 0.5 μL of reservoir solution containing 100 mM MgCl2, 100 mM Tris-HCl pH 7.5 and 12% (w/v) PEG8000. Initial efforts to crystallize the KCTD12 H1 domain in complex with Gβ1γ2 were only successful in wells with fungal contamination. Although these crystals did not diffract, we reasoned that trimming by fungal proteases might be facilitating crystallogenesis. Indeed, strongly diffracting crystals were obtained when purified KCTD12 H1 (200–325) and Gβ1γ2 were mixed in a 1:1.2 molar ratio with a small amount of trypsin (1:1000 by mass) for 30 min at room temperature prior to setting trays. Crystals were obtained by the sitting drop vapor diffusion method in drops mixed with 100 nL of protein (9 mg/ml−1) and 100 nL of reservoir solution containing 0.2 M NaCl, 0.1 M sodium cacodylate and 8% (w/v) PEG8000 after overnight growth at 20 °C. The crystals were flash-frozen in liquid nitrogen without additional cryoprotection.Diffraction data were collected at Advanced Photon Source GM/CA beamlines 23ID-B and 23ID-D with a 20 μm collimated beam dimension. Data were processed in HKL200025. Crystals for KCTD16BTB in complex with the GABAB2 fragment complex belonged to the P21 space group, and contained two copies of KCTD16 BTB-GABAB2 fragment complex per asymmetric unit. Phase was determined with molecular replacement using Phaser in Phenix26 with the KCTD16BTB structure (PDB ID: 5A15)17 as a search model. The final refined structure includes two copies of KCTD16BTB pentamer with residues 23–124 resolved, and two GABAB2 C-terminal peptides with residues 884–913 resolved. Ramachandran analysis showed that 95.7% of residues are in the favored regions, and 4.3% in the allowed regions. Crystals for KCTD12H1/Gβ1γ2 complex belonged to space group P212121. The structure was solved by molecular replacement using Phaser in Phenix with the structure of bovine Gβ1γ2 heterodimer (PDB ID: 1OMW)27 as a search model. The final model includes one copy of KCTD12 H1 pentamer (residues, 206–324) and five copies of Gβ1γ2 complex. The model was refined through iterative model building in Coot28 and reciprocal space refinement in phenix.refine29. The final structure has 97.36% and 2.64% of residues in the favored and allowed regions of Ramachandran plot respectively.Isothermal titration calorimetryITC experiments were performed with an ITC200 microcalorimeter (Microcal Inc.) at 30 °C. Purified KCTD12H1 and Gβ1γ2 were exchanged to the same buffer (20 mM HEPES-NaCl, pH 7.6 and 100 mM NaCl) with a HiTrap Desalting column (GE healthcare) before titration. 160 μM of Gβ1γ2 was injected via syringe into sample cell containing 16 μM of KCTD12H1. The integrated heat data were analyzed using one-set-of-sites model in Origin according to the manufacturer’s instructions. The dissociation constant KD was calculated as 1/KA.Electron microscopy and 3D reconstructionFor negative staining, 2.5 μL of protein solution consisting of a complex of GABAB2 C-terminal peptide, full-length KCTD12, and Gβ1γ2 at a concentration of ~0.01 mg/mL was added to a glow-discharged carbon-coated copper grid (Electron Microscopy Science) and allowed to adsorb for 20 seconds. Grids were then washed twice with deionized water and then stained twice with freshly prepared 0.75% (w/v) uranyl formate. Filter paper was applied to absorb residual liquid between each step.Images were collected at room temperature using a Philips Tecnai T12 electron microscope equipped with an LaB6 filament and operated at 120 kV. Images were collected at a magnification of 52,000-fold and a defocus value of 1.5 μm on a Gatan 4K CCD using low-dose collection procedure. Particles were manually picked in Eman230 and 2D class averages were calculated with Relion31. 3D initial model building, 3D classification, and refinement were carried out in CisTEM32.HEK 293T cell imagingKCTD12 or mutants were cloned into pCDNA3.1 vector for the expression of an N-terminal protein C (PC-KCTD12) or eGFP fusion protein (eGFP-KCTD12). HEK 293T cells were cultured in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 5% Fetal Bovine Serum (FBS) and maintained at 37°C in a 5% CO2 humidified incubator. Cells were seeded on poly-L-lysine coated 18 mm coverslips 18 hr before transfection. Transfections were done using Lipofectamine 3000 (Invitrogen) with 0.7 μg per well of eGFP-KCTD12, eGFP-KCTD12-F87A, eGFP-KCTD12-R232D or eGFP-KCTD12-R257D with and without 0.5 μg each of GABAB1A and GABAB2.Living cells were imaged at room temperature (~25°C) 42 hr after transfection in extracellular solution composed of 135 mM NaCl, 5.4 mM KCl, 10 mM HEPES, 2 mM CaCl2, and 1 mM MgCl2, pH 7.4. Imaging was performed on an Olympus IX-73 microscope using a 60× 1.49 NA APO N TIRFM objective (Olympus) and snapshots were taken with a sCMOS ORCA-Flash4 v3.0 camera (Hamamatsu). GFP was excited with a 488 nm laser diode. Images were saved as 16-bit TIFF files. An ROI was drawn around a blank area lacking cells to perform background-subtraction using a macro on Fiji (ImageJ) that subtracts the mean intensity value of the ROI from the image. Line-scan analysis was performed on background-subtracted images.Whole-cell patch clamp electrophysiologyChinese Hamster Ovary (CHO) cells were cultured in Nutrient Mixture F-12 Ham (Sigma-Aldrich) supplemented with 10% FBS, 14 mM Sodium Bicarbonate, and 11 mM D-Glucose (pH 7.2) and maintained at 37 °C in a 5% CO2 humidified incubator. Cells were seeded on poly-L-ornithine coated 18 mm coverslips and transfected ~24 hr later. Transfections were done using Lipofectamine 2000 (Invitrogen) with 0.7 μg of either KCTD12, EGFP-KCTD12, EGFP-KCTD12-F80A, EGFP-KCTD12-R232D or EGFP-KCTD12-R257D, 0.7 μg GIRK1-F137S (homotetramerization mutant), 0.3 μg tdTomato, and 0.35 μg each of rat GABAB1A and GABAB2. Cells were maintained in 2 μM CGP 54626 post-transfection to reduce basal activity of GABAB and maintain cell health.Whole-cell patch clamp experiments were performed 42–72 hr after transfection in an extracellular solution containing 120 mM KCl, 25 mM NaCl, 10 mM HEPES, 2 mM CaCl2, and 1 mM MgCl2 (pH 7.4). GABAB responses were induced with a 30 s application of 100 μM baclofen. Solutions were delivered to a recording chamber using a gravity-driven perfusion system with exchange times of ~1 s. Cells were voltage-clamped at −60 mV using an Axopatch 200B amplifier (Molecular Devices). Patch pipettes with resistances of 3–7 MΩ were filled with an intracellular solution composed of 107.5 mM potassium gluconate, 32.5 mM KCl, 10 mM HEPES, 10 mM Tris phosphocreatine, 5 mM EGTA, 4 mM Na2ATP, and 0.6 mM Na2GTP (pH 7.2). Currents were sampled at 10 kHz and filtered using a low-pass Bessel (8-pole) at 2 kHz. Recordings were analyzed using Clampfit (Molecular Devices) and Prism (GraphPad). The number of experimental replicates per condition is as follows: 4 -KCTD, 2 +KCTD, 3 +EGFP-KCTD, 2 +EGFP-KCTD12-R257D, 2 +EGFP-KCTD12-R232D. The number of cells measured per condition is as follows: 13 -KCTD, 11 +KCTD, 13 +EGFP-KCTD, 9 +EGFP-KCTD12-R257D, 6 +EGFP-KCTD12-R232D. Relative desensitization of the baclofen-induced response was calculated as follows: 1 – (amplitude prior to baclofen washout)/(peak amplitude following baclofen application). Error bars indicate SEM, and statistical significance was assessed using a two-tailed parametric Student’s t-test with a threshold of p < 0.05.Data Availability StatementThe refined coordinates and structure factors for the KCTD16BTB/GABAB2 peptide complex and KCTD12H1/Gβ1γ2 complex have been deposited in the protein data bank under the accession codes 6M8R and 6M8S respectively.

Protein Expression and PurificationHuman KCTD12 and KCTD16 genes were synthesized (Integrated DNA Technologies). KCTD12 (residues 32–131) and KCTD16 (residues 23–124) BTB domains were cloned into a pET28a expression vector (EMD Millipore) for the expression of an N-terminal His6-SUMO fusion protein or an untagged version in Escherichia coli strain BL21(DE3). KCTD12 (residues 200–325) and KCTD16 (residues 158–279) H1 domain were cloned into the multiple cloning site 1 of pETduet-1 vector (EMD Millipore) and expressed with an N-terminal His6-SUMO tag followed by 3C protease site. All human GABAB2 C-terminal truncations were cloned into the same vector with an N-terminal His6-GST or His6-SUMO fusion protein followed by 3C protease site and were expressed in BL21(DE3) E. coli. KCTD12 (residues 32–325) and KCTD16 (residues 21–279) constructs containing both BTB and H1 domains were cloned into the vector pVL1393 vector (Expression Systems) and protein was expressed using baculovirus infection of Sf9 insect cells (Expression Systems) according the manufacturer’s protocols. These constructs were expressed with an N-terminal His6-GST fusion protein followed by a 3C protease site. Human Gαq was cloned into pVL1393 without tags. Human Gβ1γ2 heterodimer was modified to include an N-terminal His6 tag on the Gβ subunit and a C68S point mutation in Gγ, which eliminates the lipid modification site. A bicistronic vector based on pVL1392 was used to prepare baculovirus encoding both subunits. All point mutations and deletions were introduced by the QuikChange method.Plasmids encoding untagged KCTD16 BTB and His6-GST-tagged GABAB2 fragment were co-transformed into E. coli BL21(DE3) strain. Bacteria were cultured in LB medium supplemented with 50 μg ml−1 ampicillin and 50 μg ml−1 kanamycin at 37 °C to an OD600 value of 0.8, and protein expression was induced by the addition of 0.5 mM isopropyl 1-thio-β-D-glucopyranoside followed by shaking overnight at 18 °C. Cells were harvested by centrifugation, resuspended in buffer A (50 mM Tris-HCl, pH 8.0, 250 mM NaCl and 25 mM Imidazole) and lysed by sonication. His-GST-GABAB2 fragment in complex with KCTD16BTB was purified from clarified lysate by Ni-NTA affinity chromatography. Resin was washed with buffer A and protein then eluted with buffer B (50 mM Tris-HCl, pH 8.0, 250 mM NaCl and 250 mM Imidazole). The eluted fraction was incubated with 3C protease overnight to cleave His6-GST tag. Protein solution was diluted three-fold by 20 mM HEPES-NaOH, pH 7.6 and loaded onto a Q sepharose column (GE Healthcare). The flow-through was collected, concentrated and further purified by size exclusion chromatography (SEC) Superdex S200 10/300 equilibrated in storage buffer (10 mM HEPES-NaOH pH 7.6, and 150 mM NaCl). Protein expression and nickel affinity purification for His6-SUMO-tagged KCTD12 H1 was performed as described above. After tag cleavage by 3C protease, the protein sample was concentrated and purified on a Superdex S200 10/300 size exclusion column equilibrated in storage buffer.His6-GST-tagged KCTD12 32–325 or KCTD16 21–279 was expressed in Sf9 insect cell cultures using the BestBac baculovirus system (Expression Systems). Infection was performed when cells reached a density of 3 × 106 cell per mL and flasks were shaken at 27 °C for 36 hours before harvest. Cells were harvested by centrifugation and lysed in buffer A using a glass dounce tissue grinder. Protein was purified by Ni-NTA gravity flow chromatography followed by glutathione agarose bead affinity purification (GE Healthcare). The protein was further purified by Superdex S200 10/300 equilibrated in storage buffer. His6-tagged Gβ1 and Gγ2 (C68S mutant) were expressed in Sf9 insect cells using the BestBac system (Expression Systems) as described above. Purification was performed as previously reported23.For KCTD12H1/Gβ1γ2 complex assembly and purification, individually purified KCTD12 H1 and Gβ1γ2 proteins were mixed in a 1:1.2 molar ratio and then incubated at room temperature for 30 min. Excess Gβ1γ2 was separated from KCTD12 H1/Gβ1γ2 complex on a Superdex S200 10/300 column in binding buffer (20 mM HEPES-NaCl pH 7.6 and 50 mM NaCl). For KCTD16 21–279/GABAB2 876–913 complex, purified His6-SUMO-tagged GABAB2 876–913 and His6-GST-tagged KCTD16 21–279 were mixed in a 2:1 molar ratio. The His6-SUMO tag of GABAB2 and His6-GST tag of KCTD16 21–279 were cleaved by 3C protease at 4°C overnight. The protein complex was further purified by SEC S200 10/300 with storage buffer. For KCTD12 32–325/GABAB2 876–913/ Gβ1γ2 complex, individually purified His6-GST-tagged KCTD12 32–325, His6-SUMO-tagged GABAB2 876–913, and Gβ1γ2 were mixed in a 1:2:1.2 ratio. After tag cleavage by 3C protease the protein sample was loaded onto Superdex S200 10/300 equilibrated in binding buffer.

Protein Expression and Purification

Human KCTD12 and KCTD16 genes were synthesized (Integrated DNA Technologies). KCTD12 (residues 32–131) and KCTD16 (residues 23–124) BTB domains were cloned into a pET28a expression vector (EMD Millipore) for the expression of an N-terminal His6-SUMO fusion protein or an untagged version in Escherichia coli strain BL21(DE3). KCTD12 (residues 200–325) and KCTD16 (residues 158–279) H1 domain were cloned into the multiple cloning site 1 of pETduet-1 vector (EMD Millipore) and expressed with an N-terminal His6-SUMO tag followed by 3C protease site. All human GABAB2 C-terminal truncations were cloned into the same vector with an N-terminal His6-GST or His6-SUMO fusion protein followed by 3C protease site and were expressed in BL21(DE3) E. coli. KCTD12 (residues 32–325) and KCTD16 (residues 21–279) constructs containing both BTB and H1 domains were cloned into the vector pVL1393 vector (Expression Systems) and protein was expressed using baculovirus infection of Sf9 insect cells (Expression Systems) according the manufacturer’s protocols. These constructs were expressed with an N-terminal His6-GST fusion protein followed by a 3C protease site. Human Gαq was cloned into pVL1393 without tags. Human Gβ1γ2 heterodimer was modified to include an N-terminal His6 tag on the Gβ subunit and a C68S point mutation in Gγ, which eliminates the lipid modification site. A bicistronic vector based on pVL1392 was used to prepare baculovirus encoding both subunits. All point mutations and deletions were introduced by the QuikChange method.

Plasmids encoding untagged KCTD16 BTB and His6-GST-tagged GABAB2 fragment were co-transformed into E. coli BL21(DE3) strain. Bacteria were cultured in LB medium supplemented with 50 μg ml−1 ampicillin and 50 μg ml−1 kanamycin at 37 °C to an OD600 value of 0.8, and protein expression was induced by the addition of 0.5 mM isopropyl 1-thio-β-D-glucopyranoside followed by shaking overnight at 18 °C. Cells were harvested by centrifugation, resuspended in buffer A (50 mM Tris-HCl, pH 8.0, 250 mM NaCl and 25 mM Imidazole) and lysed by sonication. His-GST-GABAB2 fragment in complex with KCTD16BTB was purified from clarified lysate by Ni-NTA affinity chromatography. Resin was washed with buffer A and protein then eluted with buffer B (50 mM Tris-HCl, pH 8.0, 250 mM NaCl and 250 mM Imidazole). The eluted fraction was incubated with 3C protease overnight to cleave His6-GST tag. Protein solution was diluted three-fold by 20 mM HEPES-NaOH, pH 7.6 and loaded onto a Q sepharose column (GE Healthcare). The flow-through was collected, concentrated and further purified by size exclusion chromatography (SEC) Superdex S200 10/300 equilibrated in storage buffer (10 mM HEPES-NaOH pH 7.6, and 150 mM NaCl). Protein expression and nickel affinity purification for His6-SUMO-tagged KCTD12 H1 was performed as described above. After tag cleavage by 3C protease, the protein sample was concentrated and purified on a Superdex S200 10/300 size exclusion column equilibrated in storage buffer.

His6-GST-tagged KCTD12 32–325 or KCTD16 21–279 was expressed in Sf9 insect cell cultures using the BestBac baculovirus system (Expression Systems). Infection was performed when cells reached a density of 3 × 106 cell per mL and flasks were shaken at 27 °C for 36 hours before harvest. Cells were harvested by centrifugation and lysed in buffer A using a glass dounce tissue grinder. Protein was purified by Ni-NTA gravity flow chromatography followed by glutathione agarose bead affinity purification (GE Healthcare). The protein was further purified by Superdex S200 10/300 equilibrated in storage buffer. His6-tagged Gβ1 and Gγ2 (C68S mutant) were expressed in Sf9 insect cells using the BestBac system (Expression Systems) as described above. Purification was performed as previously reported23.

For KCTD12H1/Gβ1γ2 complex assembly and purification, individually purified KCTD12 H1 and Gβ1γ2 proteins were mixed in a 1:1.2 molar ratio and then incubated at room temperature for 30 min. Excess Gβ1γ2 was separated from KCTD12 H1/Gβ1γ2 complex on a Superdex S200 10/300 column in binding buffer (20 mM HEPES-NaCl pH 7.6 and 50 mM NaCl). For KCTD16 21–279/GABAB2 876–913 complex, purified His6-SUMO-tagged GABAB2 876–913 and His6-GST-tagged KCTD16 21–279 were mixed in a 2:1 molar ratio. The His6-SUMO tag of GABAB2 and His6-GST tag of KCTD16 21–279 were cleaved by 3C protease at 4°C overnight. The protein complex was further purified by SEC S200 10/300 with storage buffer. For KCTD12 32–325/GABAB2 876–913/ Gβ1γ2 complex, individually purified His6-GST-tagged KCTD12 32–325, His6-SUMO-tagged GABAB2 876–913, and Gβ1γ2 were mixed in a 1:2:1.2 ratio. After tag cleavage by 3C protease the protein sample was loaded onto Superdex S200 10/300 equilibrated in binding buffer.

GST pull-down assayTo identify a minimal portion of GABAB2 suitable for binding KCTDs, a series of GABAB2 C-terminal truncations were cloned. His6-SMT3 tagged KCTD12 or KCTD16 and His6-GST tagged GABAB2 truncation were co-transformed and co-expressed in E. coli BL21(DE3) strain. GST pull-down experiment was performed as previously described24. For interaction analysis, individually purified His6-SUMO-tagged GABAB2 876–913 or Gβ1γ2 complex or Gαqβ1γ2 complex was added to His6-GST-tagged KCTD12 32–325 in an excessive amount in 200 μl of binding buffer. The mixtures were incubated with 20 μL of glutathione-Sepharose beads and gently rotated at 4 °C for 1 hour. After the beads were washed three times with binding buffer, the bound protein was eluted with 20 μL of binding buffer supplemented with 20 mM glutathione. Input and eluate samples were resolved using SDS-PAGE and stained with Coomassie Blue.

GST pull-down assay

To identify a minimal portion of GABAB2 suitable for binding KCTDs, a series of GABAB2 C-terminal truncations were cloned. His6-SMT3 tagged KCTD12 or KCTD16 and His6-GST tagged GABAB2 truncation were co-transformed and co-expressed in E. coli BL21(DE3) strain. GST pull-down experiment was performed as previously described24. For interaction analysis, individually purified His6-SUMO-tagged GABAB2 876–913 or Gβ1γ2 complex or Gαqβ1γ2 complex was added to His6-GST-tagged KCTD12 32–325 in an excessive amount in 200 μl of binding buffer. The mixtures were incubated with 20 μL of glutathione-Sepharose beads and gently rotated at 4 °C for 1 hour. After the beads were washed three times with binding buffer, the bound protein was eluted with 20 μL of binding buffer supplemented with 20 mM glutathione. Input and eluate samples were resolved using SDS-PAGE and stained with Coomassie Blue.

Crystallization and structure determinationThe KCTD16 BTB domain (23–124) and GABAB2 fragment (876–913) were crystallized at 20 °C using the hanging-drop vapor diffusion method by mixing 0.5 μL of protein (19 mg mL−1) and 0.5 μL of reservoir solution containing 100 mM MgCl2, 100 mM Tris-HCl pH 7.5 and 12% (w/v) PEG8000. Initial efforts to crystallize the KCTD12 H1 domain in complex with Gβ1γ2 were only successful in wells with fungal contamination. Although these crystals did not diffract, we reasoned that trimming by fungal proteases might be facilitating crystallogenesis. Indeed, strongly diffracting crystals were obtained when purified KCTD12 H1 (200–325) and Gβ1γ2 were mixed in a 1:1.2 molar ratio with a small amount of trypsin (1:1000 by mass) for 30 min at room temperature prior to setting trays. Crystals were obtained by the sitting drop vapor diffusion method in drops mixed with 100 nL of protein (9 mg/ml−1) and 100 nL of reservoir solution containing 0.2 M NaCl, 0.1 M sodium cacodylate and 8% (w/v) PEG8000 after overnight growth at 20 °C. The crystals were flash-frozen in liquid nitrogen without additional cryoprotection.Diffraction data were collected at Advanced Photon Source GM/CA beamlines 23ID-B and 23ID-D with a 20 μm collimated beam dimension. Data were processed in HKL200025. Crystals for KCTD16BTB in complex with the GABAB2 fragment complex belonged to the P21 space group, and contained two copies of KCTD16 BTB-GABAB2 fragment complex per asymmetric unit. Phase was determined with molecular replacement using Phaser in Phenix26 with the KCTD16BTB structure (PDB ID: 5A15)17 as a search model. The final refined structure includes two copies of KCTD16BTB pentamer with residues 23–124 resolved, and two GABAB2 C-terminal peptides with residues 884–913 resolved. Ramachandran analysis showed that 95.7% of residues are in the favored regions, and 4.3% in the allowed regions. Crystals for KCTD12H1/Gβ1γ2 complex belonged to space group P212121. The structure was solved by molecular replacement using Phaser in Phenix with the structure of bovine Gβ1γ2 heterodimer (PDB ID: 1OMW)27 as a search model. The final model includes one copy of KCTD12 H1 pentamer (residues, 206–324) and five copies of Gβ1γ2 complex. The model was refined through iterative model building in Coot28 and reciprocal space refinement in phenix.refine29. The final structure has 97.36% and 2.64% of residues in the favored and allowed regions of Ramachandran plot respectively.

Crystallization and structure determination

The KCTD16 BTB domain (23–124) and GABAB2 fragment (876–913) were crystallized at 20 °C using the hanging-drop vapor diffusion method by mixing 0.5 μL of protein (19 mg mL−1) and 0.5 μL of reservoir solution containing 100 mM MgCl2, 100 mM Tris-HCl pH 7.5 and 12% (w/v) PEG8000. Initial efforts to crystallize the KCTD12 H1 domain in complex with Gβ1γ2 were only successful in wells with fungal contamination. Although these crystals did not diffract, we reasoned that trimming by fungal proteases might be facilitating crystallogenesis. Indeed, strongly diffracting crystals were obtained when purified KCTD12 H1 (200–325) and Gβ1γ2 were mixed in a 1:1.2 molar ratio with a small amount of trypsin (1:1000 by mass) for 30 min at room temperature prior to setting trays. Crystals were obtained by the sitting drop vapor diffusion method in drops mixed with 100 nL of protein (9 mg/ml−1) and 100 nL of reservoir solution containing 0.2 M NaCl, 0.1 M sodium cacodylate and 8% (w/v) PEG8000 after overnight growth at 20 °C. The crystals were flash-frozen in liquid nitrogen without additional cryoprotection.

Diffraction data were collected at Advanced Photon Source GM/CA beamlines 23ID-B and 23ID-D with a 20 μm collimated beam dimension. Data were processed in HKL200025. Crystals for KCTD16BTB in complex with the GABAB2 fragment complex belonged to the P21 space group, and contained two copies of KCTD16 BTB-GABAB2 fragment complex per asymmetric unit. Phase was determined with molecular replacement using Phaser in Phenix26 with the KCTD16BTB structure (PDB ID: 5A15)17 as a search model. The final refined structure includes two copies of KCTD16BTB pentamer with residues 23–124 resolved, and two GABAB2 C-terminal peptides with residues 884–913 resolved. Ramachandran analysis showed that 95.7% of residues are in the favored regions, and 4.3% in the allowed regions. Crystals for KCTD12H1/Gβ1γ2 complex belonged to space group P212121. The structure was solved by molecular replacement using Phaser in Phenix with the structure of bovine Gβ1γ2 heterodimer (PDB ID: 1OMW)27 as a search model. The final model includes one copy of KCTD12 H1 pentamer (residues, 206–324) and five copies of Gβ1γ2 complex. The model was refined through iterative model building in Coot28 and reciprocal space refinement in phenix.refine29. The final structure has 97.36% and 2.64% of residues in the favored and allowed regions of Ramachandran plot respectively.

Isothermal titration calorimetryITC experiments were performed with an ITC200 microcalorimeter (Microcal Inc.) at 30 °C. Purified KCTD12H1 and Gβ1γ2 were exchanged to the same buffer (20 mM HEPES-NaCl, pH 7.6 and 100 mM NaCl) with a HiTrap Desalting column (GE healthcare) before titration. 160 μM of Gβ1γ2 was injected via syringe into sample cell containing 16 μM of KCTD12H1. The integrated heat data were analyzed using one-set-of-sites model in Origin according to the manufacturer’s instructions. The dissociation constant KD was calculated as 1/KA.

Isothermal titration calorimetry

ITC experiments were performed with an ITC200 microcalorimeter (Microcal Inc.) at 30 °C. Purified KCTD12H1 and Gβ1γ2 were exchanged to the same buffer (20 mM HEPES-NaCl, pH 7.6 and 100 mM NaCl) with a HiTrap Desalting column (GE healthcare) before titration. 160 μM of Gβ1γ2 was injected via syringe into sample cell containing 16 μM of KCTD12H1. The integrated heat data were analyzed using one-set-of-sites model in Origin according to the manufacturer’s instructions. The dissociation constant KD was calculated as 1/KA.

Electron microscopy and 3D reconstructionFor negative staining, 2.5 μL of protein solution consisting of a complex of GABAB2 C-terminal peptide, full-length KCTD12, and Gβ1γ2 at a concentration of ~0.01 mg/mL was added to a glow-discharged carbon-coated copper grid (Electron Microscopy Science) and allowed to adsorb for 20 seconds. Grids were then washed twice with deionized water and then stained twice with freshly prepared 0.75% (w/v) uranyl formate. Filter paper was applied to absorb residual liquid between each step.Images were collected at room temperature using a Philips Tecnai T12 electron microscope equipped with an LaB6 filament and operated at 120 kV. Images were collected at a magnification of 52,000-fold and a defocus value of 1.5 μm on a Gatan 4K CCD using low-dose collection procedure. Particles were manually picked in Eman230 and 2D class averages were calculated with Relion31. 3D initial model building, 3D classification, and refinement were carried out in CisTEM32.

Electron microscopy and 3D reconstruction

For negative staining, 2.5 μL of protein solution consisting of a complex of GABAB2 C-terminal peptide, full-length KCTD12, and Gβ1γ2 at a concentration of ~0.01 mg/mL was added to a glow-discharged carbon-coated copper grid (Electron Microscopy Science) and allowed to adsorb for 20 seconds. Grids were then washed twice with deionized water and then stained twice with freshly prepared 0.75% (w/v) uranyl formate. Filter paper was applied to absorb residual liquid between each step.

Images were collected at room temperature using a Philips Tecnai T12 electron microscope equipped with an LaB6 filament and operated at 120 kV. Images were collected at a magnification of 52,000-fold and a defocus value of 1.5 μm on a Gatan 4K CCD using low-dose collection procedure. Particles were manually picked in Eman230 and 2D class averages were calculated with Relion31. 3D initial model building, 3D classification, and refinement were carried out in CisTEM32.

HEK 293T cell imagingKCTD12 or mutants were cloned into pCDNA3.1 vector for the expression of an N-terminal protein C (PC-KCTD12) or eGFP fusion protein (eGFP-KCTD12). HEK 293T cells were cultured in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 5% Fetal Bovine Serum (FBS) and maintained at 37°C in a 5% CO2 humidified incubator. Cells were seeded on poly-L-lysine coated 18 mm coverslips 18 hr before transfection. Transfections were done using Lipofectamine 3000 (Invitrogen) with 0.7 μg per well of eGFP-KCTD12, eGFP-KCTD12-F87A, eGFP-KCTD12-R232D or eGFP-KCTD12-R257D with and without 0.5 μg each of GABAB1A and GABAB2.Living cells were imaged at room temperature (~25°C) 42 hr after transfection in extracellular solution composed of 135 mM NaCl, 5.4 mM KCl, 10 mM HEPES, 2 mM CaCl2, and 1 mM MgCl2, pH 7.4. Imaging was performed on an Olympus IX-73 microscope using a 60× 1.49 NA APO N TIRFM objective (Olympus) and snapshots were taken with a sCMOS ORCA-Flash4 v3.0 camera (Hamamatsu). GFP was excited with a 488 nm laser diode. Images were saved as 16-bit TIFF files. An ROI was drawn around a blank area lacking cells to perform background-subtraction using a macro on Fiji (ImageJ) that subtracts the mean intensity value of the ROI from the image. Line-scan analysis was performed on background-subtracted images.

HEK 293T cell imaging

KCTD12 or mutants were cloned into pCDNA3.1 vector for the expression of an N-terminal protein C (PC-KCTD12) or eGFP fusion protein (eGFP-KCTD12). HEK 293T cells were cultured in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 5% Fetal Bovine Serum (FBS) and maintained at 37°C in a 5% CO2 humidified incubator. Cells were seeded on poly-L-lysine coated 18 mm coverslips 18 hr before transfection. Transfections were done using Lipofectamine 3000 (Invitrogen) with 0.7 μg per well of eGFP-KCTD12, eGFP-KCTD12-F87A, eGFP-KCTD12-R232D or eGFP-KCTD12-R257D with and without 0.5 μg each of GABAB1A and GABAB2.

Living cells were imaged at room temperature (~25°C) 42 hr after transfection in extracellular solution composed of 135 mM NaCl, 5.4 mM KCl, 10 mM HEPES, 2 mM CaCl2, and 1 mM MgCl2, pH 7.4. Imaging was performed on an Olympus IX-73 microscope using a 60× 1.49 NA APO N TIRFM objective (Olympus) and snapshots were taken with a sCMOS ORCA-Flash4 v3.0 camera (Hamamatsu). GFP was excited with a 488 nm laser diode. Images were saved as 16-bit TIFF files. An ROI was drawn around a blank area lacking cells to perform background-subtraction using a macro on Fiji (ImageJ) that subtracts the mean intensity value of the ROI from the image. Line-scan analysis was performed on background-subtracted images.

Whole-cell patch clamp electrophysiologyChinese Hamster Ovary (CHO) cells were cultured in Nutrient Mixture F-12 Ham (Sigma-Aldrich) supplemented with 10% FBS, 14 mM Sodium Bicarbonate, and 11 mM D-Glucose (pH 7.2) and maintained at 37 °C in a 5% CO2 humidified incubator. Cells were seeded on poly-L-ornithine coated 18 mm coverslips and transfected ~24 hr later. Transfections were done using Lipofectamine 2000 (Invitrogen) with 0.7 μg of either KCTD12, EGFP-KCTD12, EGFP-KCTD12-F80A, EGFP-KCTD12-R232D or EGFP-KCTD12-R257D, 0.7 μg GIRK1-F137S (homotetramerization mutant), 0.3 μg tdTomato, and 0.35 μg each of rat GABAB1A and GABAB2. Cells were maintained in 2 μM CGP 54626 post-transfection to reduce basal activity of GABAB and maintain cell health.Whole-cell patch clamp experiments were performed 42–72 hr after transfection in an extracellular solution containing 120 mM KCl, 25 mM NaCl, 10 mM HEPES, 2 mM CaCl2, and 1 mM MgCl2 (pH 7.4). GABAB responses were induced with a 30 s application of 100 μM baclofen. Solutions were delivered to a recording chamber using a gravity-driven perfusion system with exchange times of ~1 s. Cells were voltage-clamped at −60 mV using an Axopatch 200B amplifier (Molecular Devices). Patch pipettes with resistances of 3–7 MΩ were filled with an intracellular solution composed of 107.5 mM potassium gluconate, 32.5 mM KCl, 10 mM HEPES, 10 mM Tris phosphocreatine, 5 mM EGTA, 4 mM Na2ATP, and 0.6 mM Na2GTP (pH 7.2). Currents were sampled at 10 kHz and filtered using a low-pass Bessel (8-pole) at 2 kHz. Recordings were analyzed using Clampfit (Molecular Devices) and Prism (GraphPad). The number of experimental replicates per condition is as follows: 4 -KCTD, 2 +KCTD, 3 +EGFP-KCTD, 2 +EGFP-KCTD12-R257D, 2 +EGFP-KCTD12-R232D. The number of cells measured per condition is as follows: 13 -KCTD, 11 +KCTD, 13 +EGFP-KCTD, 9 +EGFP-KCTD12-R257D, 6 +EGFP-KCTD12-R232D. Relative desensitization of the baclofen-induced response was calculated as follows: 1 – (amplitude prior to baclofen washout)/(peak amplitude following baclofen application). Error bars indicate SEM, and statistical significance was assessed using a two-tailed parametric Student’s t-test with a threshold of p < 0.05.

Whole-cell patch clamp electrophysiology

Chinese Hamster Ovary (CHO) cells were cultured in Nutrient Mixture F-12 Ham (Sigma-Aldrich) supplemented with 10% FBS, 14 mM Sodium Bicarbonate, and 11 mM D-Glucose (pH 7.2) and maintained at 37 °C in a 5% CO2 humidified incubator. Cells were seeded on poly-L-ornithine coated 18 mm coverslips and transfected ~24 hr later. Transfections were done using Lipofectamine 2000 (Invitrogen) with 0.7 μg of either KCTD12, EGFP-KCTD12, EGFP-KCTD12-F80A, EGFP-KCTD12-R232D or EGFP-KCTD12-R257D, 0.7 μg GIRK1-F137S (homotetramerization mutant), 0.3 μg tdTomato, and 0.35 μg each of rat GABAB1A and GABAB2. Cells were maintained in 2 μM CGP 54626 post-transfection to reduce basal activity of GABAB and maintain cell health.

Whole-cell patch clamp experiments were performed 42–72 hr after transfection in an extracellular solution containing 120 mM KCl, 25 mM NaCl, 10 mM HEPES, 2 mM CaCl2, and 1 mM MgCl2 (pH 7.4). GABAB responses were induced with a 30 s application of 100 μM baclofen. Solutions were delivered to a recording chamber using a gravity-driven perfusion system with exchange times of ~1 s. Cells were voltage-clamped at −60 mV using an Axopatch 200B amplifier (Molecular Devices). Patch pipettes with resistances of 3–7 MΩ were filled with an intracellular solution composed of 107.5 mM potassium gluconate, 32.5 mM KCl, 10 mM HEPES, 10 mM Tris phosphocreatine, 5 mM EGTA, 4 mM Na2ATP, and 0.6 mM Na2GTP (pH 7.2). Currents were sampled at 10 kHz and filtered using a low-pass Bessel (8-pole) at 2 kHz. Recordings were analyzed using Clampfit (Molecular Devices) and Prism (GraphPad). The number of experimental replicates per condition is as follows: 4 -KCTD, 2 +KCTD, 3 +EGFP-KCTD, 2 +EGFP-KCTD12-R257D, 2 +EGFP-KCTD12-R232D. The number of cells measured per condition is as follows: 13 -KCTD, 11 +KCTD, 13 +EGFP-KCTD, 9 +EGFP-KCTD12-R257D, 6 +EGFP-KCTD12-R232D. Relative desensitization of the baclofen-induced response was calculated as follows: 1 – (amplitude prior to baclofen washout)/(peak amplitude following baclofen application). Error bars indicate SEM, and statistical significance was assessed using a two-tailed parametric Student’s t-test with a threshold of p < 0.05.

Data Availability StatementThe refined coordinates and structure factors for the KCTD16BTB/GABAB2 peptide complex and KCTD12H1/Gβ1γ2 complex have been deposited in the protein data bank under the accession codes 6M8R and 6M8S respectively.

Data Availability Statement

The refined coordinates and structure factors for the KCTD16BTB/GABAB2 peptide complex and KCTD12H1/Gβ1γ2 complex have been deposited in the protein data bank under the accession codes 6M8R and 6M8S respectively.

Extended DataExtended Data Figure 1 |Mapping the GABAB2 binding region of BTB domaina, His6-SMT3-tagged KCTD16BTB or KCTD12BTB and His6-GST-tagged GABAB2 peptides were co-expressed in E. coli and purified with nickel affinity chromatography. Purified protein were loaded as input for pull-down with glutathione sepharose beads. Glutathione input (IN) and pull-down (PD) fractions were analyzed by SDS-PAGE and Coomassie blue staining. b, His6-GST-tagged GABAB2 and untagged KCTD16BTB were co-expressed in E. coli. Clarified lysate was pulled down with glutathione Sepharose beads and eluate was analyzed by SDS-PAGE and Coomassie blue staining. c, Structural superposition of the KCTD16BTB structure in GABAB2 peptide-free (grey) and bound state (red; PDB ID 5A15). d, His6-SMT3-tagged KCTD16BTB wild-type (WT) or F80A mutant and His6-GST-tagged GABAB2 peptides were co-expressed and purified with nickel affinity. The eluate was treated with ULP1 to cleave SMT3 tag (IN) prior to glutathione Sepharose beads pull-down. e, Sequence alignment of the BTB domain of KCTD family members from Homo sapiens. Residues with 98%, 80% and 60% similarity are shown in black, grey, and light grey, respectively.Extended Data Figure 2 |Electron density mapsa, Composite omit 2Fo-Fc electron density map contoured at 1.0 σ for GABAB2 peptide in KCTD16BTB/GABAB2 peptide complex structure. b, Composite omit 2Fo-Fc electron density map contoured at 1.0 σ for KCTD12H1 domain in KCTD12H1 - Gβ1γ2 complex structure.Extended Data Figure 3 |Representative negative stain electron micrographsa, The full SDS-PAGE gel of pulldown experiment shown in Figure 2a. b, Gαi competes with KCTD12 for binding Gβ1γ2. Pull-down experiments were carried out in the same way as Figure 2a except that Gαi subtype instead of Gαq was used. c, Isothermal titration calorimetry affinity measurement for KCTD12H1 binding to Gβ1γ2. d, Representative image of KCTD16BTB+H1/GABAB2Cter. e, Representative image of KCTD12H1/Gβ1γ2. f, Representative image of KCTD12BTB+H1/ GABAB2Cter/Gβ1γ2.Extended Data Figure 4 |3D negative stain EM reconstruction of the KCTD12 full-length complex including KCTD12 full length, GABAB2Cter and Gβ1γ2a, Three different views of the three-dimensional reconstruction of full length KCTD12-Gβ1γ2 complex with KCTD16BTB-GABAB2 peptide complex (red) and KCTD12H1- Gβ1γ2 complex (blue-magenta) crystal structures docked into the negative stain electron microscopy envelope. Negative stain 2D class averages are reproduced from Figure 2b for reference. b, The Fourier shell correlation (FSC) curve as a function of spatial frequency for negative staining EM map. Resolution is indicated below. c, Structure of full-length KCTD12 complex. The BTB domain is separated from H1 domain by 35 Å. Dashed lines represent the linker sequence between BTB and H1 domains.Extended Data Figure 5 |Structural and functional analysis of H1 domain.a, Ribbon representation of KCTD12 H1 domain. The NFLEQ motif important for desensitization is colored orange. b, Ribbon representation and topology map of the KCTD12 H1 monomer. Secondary structure elements are labeled c, Ribbon representation of two KCTD12H1 subunits (green and cyan) and Gβ1γ2. d, Subcellular localization of eGFP-KCTD12-R232D and R257D mutants with or without GABAB receptorExtended Data Figure 6 |Gβ1γ2 binding to KCTD12 is highly cooperative.a, Ribbon representation of GIRK and Gβγ complex from cytosolic side. Four isolated Gβγ heterodimers associate with a single GIRK channel tetramer. b, Ribbon representation of KCTD12H1 and Gβγ complex from cytosolic side. Gβγ subunits interact directly with one another. c, Interactions between two adjacent Gβ subunits (labeled 1 and 2). d, 0.5:5, 1:5 or 5:5 molar ratios of Gβ1γ2 to KCTD12H1 were mixed and subjected to size exclusion separation. The peaks correspond to 5:5 complexes and H1 domain alone. e, A stoichiometric amount of Gβ1γ2 single mutant R42D (in the Gβ1 subunit) or double mutant R42D/R46D was incubated with KCTD12H1 and analyzed by size exclusion chromatography. Fractions from peak 1, peak 2, and peak 3 were analyzed by SDS-PAGE, corresponding to 5:5 full complex, partial complex and free Gβ1γ2, respectively.Extended Data Figure 7 |A model for GABAB signaling and desensitization.a, Agonist binding to GABAB1 receptor causes the dissociation of Gβγ heterodimer from Gα. Four copies of Gβγ bind to a GIRK channel tetramer, resulting in channel activation. Afterwards, KCTD bound to the GABAB2 C-terminus strips four copies of Gβγ from the GIRK channel and deactivates it. Following nucleotide hydrolysis, GDP-bound Gα binds to Gβγ, sequestering it from KCTD and priming the system for another signaling cycle. b, Calculated energies for the series of progressively tighter binding events are commensurate with the approximate energy released by GTP hydrolysis.Extended Data Table 1 |Crystallographic statisticsKCTD16BTB/GABAB2CterKCTD12H1/ Gβ1γ2Data collectionSpace groupP21P212121Cell dimensions a, b, c (Å)92.5, 65.1, 114.1109.1, 122.0, 206.4 α, β,γ (°)90.0, 99.6, 90.090.0, 90.0, 90.0Resolution (Å)50 – 3.2 (3.28 – 3.20)50 – 3.71 (3.80 – 3.71)Rmerge (%)33.2 (99.7)31.2(105.9)CC1/2 (%)92.6 (56.1)96.2 (52.0)I / σI3.4 (0.6)3.56 (1.09)Completeness (%)99.2 (96.7)99.0 (97.6)Redundancy3.3 (2.7)3.9 (4.0)RefinementResolution (Å)38.6–3.2 (3.29–3.20)48.4–3.71 (3.76–3.71)No. reflections2200229614Rwork / Rfree(%)23.6 / 27.625.6 / 28.1No. atoms846218990 Protein846218990 Ligand/ion00 Water00B-factors57.393.0 Protein57.393.0 Ligand/ionN/AN/A WaterN/AN/AR.m.s. deviations Bond lengths (Å)0.0050.003 Bond angles (°)1.0950.764*Values in parentheses are for highest shell.

Extended Data

Mapping the GABAB2 binding region of BTB domain

a, His6-SMT3-tagged KCTD16BTB or KCTD12BTB and His6-GST-tagged GABAB2 peptides were co-expressed in E. coli and purified with nickel affinity chromatography. Purified protein were loaded as input for pull-down with glutathione sepharose beads. Glutathione input (IN) and pull-down (PD) fractions were analyzed by SDS-PAGE and Coomassie blue staining. b, His6-GST-tagged GABAB2 and untagged KCTD16BTB were co-expressed in E. coli. Clarified lysate was pulled down with glutathione Sepharose beads and eluate was analyzed by SDS-PAGE and Coomassie blue staining. c, Structural superposition of the KCTD16BTB structure in GABAB2 peptide-free (grey) and bound state (red; PDB ID 5A15). d, His6-SMT3-tagged KCTD16BTB wild-type (WT) or F80A mutant and His6-GST-tagged GABAB2 peptides were co-expressed and purified with nickel affinity. The eluate was treated with ULP1 to cleave SMT3 tag (IN) prior to glutathione Sepharose beads pull-down. e, Sequence alignment of the BTB domain of KCTD family members from Homo sapiens. Residues with 98%, 80% and 60% similarity are shown in black, grey, and light grey, respectively.

Electron density maps

a, Composite omit 2Fo-Fc electron density map contoured at 1.0 σ for GABAB2 peptide in KCTD16BTB/GABAB2 peptide complex structure. b, Composite omit 2Fo-Fc electron density map contoured at 1.0 σ for KCTD12H1 domain in KCTD12H1 - Gβ1γ2 complex structure.

Representative negative stain electron micrographs

a, The full SDS-PAGE gel of pulldown experiment shown in Figure 2a. b, Gαi competes with KCTD12 for binding Gβ1γ2. Pull-down experiments were carried out in the same way as Figure 2a except that Gαi subtype instead of Gαq was used. c, Isothermal titration calorimetry affinity measurement for KCTD12H1 binding to Gβ1γ2. d, Representative image of KCTD16BTB+H1/GABAB2Cter. e, Representative image of KCTD12H1/Gβ1γ2. f, Representative image of KCTD12BTB+H1/ GABAB2Cter/Gβ1γ2.

3D negative stain EM reconstruction of the KCTD12 full-length complex including KCTD12 full length, GABAB2Cter and Gβ1γ2

a, Three different views of the three-dimensional reconstruction of full length KCTD12-Gβ1γ2 complex with KCTD16BTB-GABAB2 peptide complex (red) and KCTD12H1- Gβ1γ2 complex (blue-magenta) crystal structures docked into the negative stain electron microscopy envelope. Negative stain 2D class averages are reproduced from Figure 2b for reference. b, The Fourier shell correlation (FSC) curve as a function of spatial frequency for negative staining EM map. Resolution is indicated below. c, Structure of full-length KCTD12 complex. The BTB domain is separated from H1 domain by 35 Å. Dashed lines represent the linker sequence between BTB and H1 domains.

Structural and functional analysis of H1 domain.

a, Ribbon representation of KCTD12 H1 domain. The NFLEQ motif important for desensitization is colored orange. b, Ribbon representation and topology map of the KCTD12 H1 monomer. Secondary structure elements are labeled c, Ribbon representation of two KCTD12H1 subunits (green and cyan) and Gβ1γ2. d, Subcellular localization of eGFP-KCTD12-R232D and R257D mutants with or without GABAB receptor

Gβ1γ2 binding to KCTD12 is highly cooperative.

a, Ribbon representation of GIRK and Gβγ complex from cytosolic side. Four isolated Gβγ heterodimers associate with a single GIRK channel tetramer. b, Ribbon representation of KCTD12H1 and Gβγ complex from cytosolic side. Gβγ subunits interact directly with one another. c, Interactions between two adjacent Gβ subunits (labeled 1 and 2). d, 0.5:5, 1:5 or 5:5 molar ratios of Gβ1γ2 to KCTD12H1 were mixed and subjected to size exclusion separation. The peaks correspond to 5:5 complexes and H1 domain alone. e, A stoichiometric amount of Gβ1γ2 single mutant R42D (in the Gβ1 subunit) or double mutant R42D/R46D was incubated with KCTD12H1 and analyzed by size exclusion chromatography. Fractions from peak 1, peak 2, and peak 3 were analyzed by SDS-PAGE, corresponding to 5:5 full complex, partial complex and free Gβ1γ2, respectively.

A model for GABAB signaling and desensitization.

a, Agonist binding to GABAB1 receptor causes the dissociation of Gβγ heterodimer from Gα. Four copies of Gβγ bind to a GIRK channel tetramer, resulting in channel activation. Afterwards, KCTD bound to the GABAB2 C-terminus strips four copies of Gβγ from the GIRK channel and deactivates it. Following nucleotide hydrolysis, GDP-bound Gα binds to Gβγ, sequestering it from KCTD and priming the system for another signaling cycle. b, Calculated energies for the series of progressively tighter binding events are commensurate with the approximate energy released by GTP hydrolysis.

Crystallographic statistics

*Values in parentheses are for highest shell.

Supplementary Material1

Supplementary Material
