---
pmid: '35733257'
title: Structural basis of Tom20 and Tom22 cytosolic domains as the human TOM complex
  receptors.
authors:
- Su J
- Liu D
- Yang F
- Zuo MQ
- Li C
- Dong MQ
- Sun S
- Sui SF
journal: Proc Natl Acad Sci U S A
year: '2022'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC9245660
doi: 10.1073/pnas.2200158119
---

# Structural basis of Tom20 and Tom22 cytosolic domains as the human TOM complex receptors.
**Authors:** Su J, Liu D, Yang F, Zuo MQ, Li C, Dong MQ, Sun S, Sui SF
**Journal:** Proc Natl Acad Sci U S A (2022)
**DOI:** [10.1073/pnas.2200158119](https://doi.org/10.1073/pnas.2200158119)
**PMC:** [PMC9245660](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9245660/)

## Abstract

1. Proc Natl Acad Sci U S A. 2022 Jun 28;119(26):e2200158119. doi: 
10.1073/pnas.2200158119. Epub 2022 Jun 22.

Structural basis of Tom20 and Tom22 cytosolic domains as the human TOM complex 
receptors.

Su J(1), Liu D(1), Yang F(1), Zuo MQ(2), Li C(1), Dong MQ(2), Sun S(1), Sui 
SF(1)(3).

Author information:
(1)State Key Laboratory of Membrane Biology, Beijing Advanced Innovation Center 
for Structural Biology, Beijing Frontier Research Center for Biological 
Structure, School of Life Sciences, Tsinghua University, 100084, Beijing, China.
(2)National Institute of Biological Sciences, 102206 Beijing, China.
(3)Department of Biology, Southern University of Science and Technology, 
Shenzhen, Guangdong 518055, China.

Mitochondrial preproteins synthesized in cytosol are imported into mitochondria 
by a multisubunit translocase of the outer membrane (TOM) complex. Functioned as 
the receptor, the TOM complex components, Tom 20, Tom22, and Tom70, recognize 
the presequence and further guide the protein translocation. Their deficiency 
has been linked with neurodegenerative diseases and cardiac pathology. Although 
several structures of the TOM complex have been reported by cryoelectron 
microscopy (cryo-EM), how Tom22 and Tom20 function as TOM receptors remains 
elusive. Here we determined the structure of TOM core complex at 2.53 Å and 
captured the structure of the TOM complex containing Tom22 and Tom20 cytosolic 
domains at 3.74 Å. Structural analysis indicates that Tom20 and Tom22 share a 
similar three-helix bundle structural feature in the cytosolic domain. Further 
structure-guided biochemical analysis reveals that the Tom22 cytosolic domain is 
responsible for binding to the presequence, and the helix H1 is critical for 
this binding. Altogether, our results provide insights into the functional 
mechanism of the TOM complex recognizing and transferring preproteins across the 
mitochondrial membrane.

DOI: 10.1073/pnas.2200158119
PMCID: PMC9245660
PMID: 35733257 [Indexed for MEDLINE]

Conflict of interest statement: The authors declare no competing interest.

## Full Text

As an essential organelle in eukaryotic cells, mitochondria play roles in various important cellular activities, including energy regulation, metabolic signaling, apoptosis, and autophagy (1–4). About 1,500 different proteins are involved in carrying out mitochondrial functions. Most of these proteins are encoded by nuclear genes and synthesized in the cytosol (4–6). Therefore, they must be translocated into the mitochondria to play roles, which is mediated mainly by the translocase of the outer mitochondrial membrane (TOM) complex (1). With the help of the TOM complex, the precursor proteins can be recognized and transferred from the cytosol to four different locations of mitochondria: the outer membrane (OM), the inner membrane (IM), the intermembrane space (IMS), and the matrix (7–9).

The TOM complex consists of seven different components, which can be divided into α-helical and β-barrel membrane proteins (10, 11). The channel-forming protein Tom40 is a β-barrel subunit, which is assembled by the sorting and assembly machinery (SAM) complex (12). As the main gate for preproteins transfer (13), Tom40 is released in the outer membrane (14). Recent research has shown that most translocation proteins have short sequences in N-terminal and are recognized by receptor proteins Tom70, Tom20, and Tom22 (15, 16, 27). Tom70, Tom22, Tom20, and three small proteins Tom5, Tom6, and Tom7 are α-helical subunits (17–20). Two Tom22 receptors connect two Tom40s at the dimer interface (21). Tom5, Tom6, and Tom7 surround Tom40 (22–24). Tom6 can interact with Tom22 to make the TOM complex more stable (25, 26), but Tom7 seems like an inhibitor to reduce stability (6).

The relationship between the TOM complex and human diseases has been found in recent years (28–31). Mutations in Tom40 and Tom22 lead to neurodegenerative diseases such as late-onset Alzheimer’s disease (LOAD) (32) and Huntington’s disease (HD) (33). Tom70 plays a critical role in cardiac pathology through its properties that impact oxidation and apoptosis (34). Moreover, in the study of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), Tom70 was found to interact with SARS-CoV-2 ORF9b, suggesting the virus may modulate the host immune response through this interaction (35).

Although the structure of the human TOM complex has been resolved recently (36–38), the structure of receptor proteins Tom20, Tom70, and full-length Tom22 are still unresolved, which restricts the understanding of the mechanism of preproteins transport into mitochondria. Here, we determined the structures of the human mitochondrial TOM core complex without and with cross-linking treatment at 2.53-Å and 3.74-Å resolutions, respectively. The Tom22 and Tom20 cytosolic domains were resolved. Combination of the two determined structures of TOM complex with the biochemical binding assay, we proposed possible ways for Tom20 and Tom22 to participate in presequence translocation.

ResultsOverall Structure of Human TOM Core Complex at 2.53 Å.In order to prepare high-quality samples of the TOM complex for single-particle analysis, the plasmids expressing the C-terminal Flag-tagged human Tom22 were transformed into H293F cells. Mitochondria were first isolated from the transformed cells and then solubilized by digitonin. The TOM complexes were extracted using anti-Flag beads and further purified by the gel filtration chromatography. Using single-particle cryoelectron microscopy (cryo-EM), we resolved the structure of the human TOM core complex at an overall resolution of 2.53 Å (Fig. 1A and SI Appendix, Fig. S1). The quality of the density map is sufficient for assignment of main chains and many bulky side chains of each subunit (SI Appendix, Fig. S2 and Table S1). One lipid molecule phosphatidylcholine (PC) 01 (PC01) located between two Tom40 subunits was well defined in the density map (Fig. 2). The final atomic model shows that the human TOM complex forms a centrosymmetric dimer and each monomer contains subunits of Tom40, Tom22, Tom5, Tom6, and Tom7 with 1:1:1:1:1 stoichiometry, agreeing with previous studies (36, 38) (Fig. 1B and SI Appendix, Fig. S3A). The dimension of the dimeric TOM complex is 113 Å by 75 Å by 91 Å (Fig. 1B). When we superposed our structure to the yeast TOM complexes (37), differences were observed, especially for Tom7 and Tom22 (SI Appendix, Fig. S3B), suggesting the existence of structural deviations of the TOM complex between different species.Fig. 1.Overall structure of the human TOM core complex at 2.53 Å. (A) Cryo-EM density map and (B) the atomic model of the human TOM core complex in two different views. Tom5, Tom6, Tom7, Tom22, and Tom40 are colored by orange, green, cyan, magenta, and yellow, respectively. (C to F) Interactions between Tom5 (C), Tom22 (D), Tom6 (F), Tom7 (E), and Tom40. H-bonds and salt bridges observed in our structure and reported before are the red lines and black dotted lines, respectively. Residues from Tom5, Tom6, Tom7, and Tom22 are colored and labeled by orange, green, cyan, and magenta, respectively. Residues from Tom40 are colored by yellow and are labeled in black. Approximate IMS, cytosol space and OM boundaries are separated by black dashed lines.Fig. 2.Interactions between the human TOM core complex and lipid molecules. (A) Positions and interactions of PC lipids with corresponding Tom subunits. Lipids are colored by red. H-bonds and π-cation are indicated as black dashed lines. (B) PC01 forms hydrophobic interactions with Tom40 subunits. Residues from Tom40 are colored yellow and labeled in black. Lipids are colored red.The overall structure of the human TOM complex has a binocular-like architecture with the Tom40 portion corresponding to the lens cone (Fig. 1A). Tom40, as the core subunit of the TOM complex, forms the main conducting channels for precursor protein translocation (Fig. 1 A and B). Consistent with previous report (38, 41), Tom40 generates a β-barrel composed of 19 β-strands and 18 loop-like connections (SI Appendix, Fig. S4A). Strands β2 to β18 show an antiparallel pattern, and on the contrary, strands 1 and 19 associate in a parallel manner to close the barrel (SI Appendix, Fig. S4A). The unequal number of strands in the barrel requires the first strand to form parallel and antiparallel interactions with 2 adjacent strands at the same time. The distorted N-terminal α-helix of Tom40 nestles within the barrel, tethering the first β-strand (Fig. 1B and SI Appendix, Fig. S4B). Our structure shows the N-terminal α-helix of Tom40 forming a hydrogen bond (H-bond) network that facilitates its orientation against the interior wall of the pore adjacent to β-strands β8 to β19 (SI Appendix, Fig. S4B). Electrostatic potential distribution analysis showed that Tom40 possesses notable electrostatic features. The inner face of the Tom40 β-barrel is negatively charged, which plays a role in facilitating the translocation of mitochondrial charged preproteins. The outer face of the Tom40 β-barrel is hydrophobic, which helps the Tom40 β-barrel to insert into the lipid membrane (SI Appendix, Fig. S1E).Tom22, Tom5, Tom6, and Tom7, as receptors or cofactors, decorate around the Tom40 β-barrel (Fig. 1B). Due to the high quality of our structure, H-bonds and salt bridges formed between Tom40 and Tom22, Tom5, Tom6, and Tom7 are well defined. R21 and R39 of Tom5 form salt bridges with E244 and D198 of Tom40, and Y34/L38/R39 of Tom5 forms H-bonds with G232/Q223/Y221 of Tom40 (Fig. 1C). These interactions are important for Tom5 attaching to the whole complex (Fig. 1C). Meanwhile, T90/L95/E102 from Tom22 forms two H-bonds and one salt bridge with H347/Y303/K330 of Tom40 (Fig. 1D). R38 of Tom7 interacts with Y135/V136/G137 of Tom40 to form a H-bond network, and R24, L52, and D41 of Tom7 make H-bonds with G158, H117, and N111 of Tom40, respectively (Fig. 1E). Also, N48/R60/D64/D66/L67 of Tom6 interacts with V299/Y274/S279/Q281 of Tom40 via five H-bonds (Fig. 1F). Some of the interactions mentioned above (marked in red lines in Fig. 1) were not observed in the previously reported TOM complex structures (33).Previous studies have indicated that the TOM complex can exist in different oligomeric states from dimer, trimer, to tetramer and even higher oligomers, depending on the cellular environment (37, 38). The dimerization of the TOM complex is mainly mediated by Tom40. If we do not consider the lipid effects, the hydrophobic interface between two Tom40 protomers, formed by four β-strands (β17, β18, β19, and β1), is only 555 Å2. But after considering the lipid effects, things will be changed. In our structure, the lipid PC01 in the junction between two β-barrels contributes extra 401.9-Å2 and 395.8-Å2 interface areas to the two Tom40 protomers, respectively, enhancing the hydrophobic interaction (36) (SI Appendix, Fig. S5). This lipid is very stable as all high-resolution TOM complex structures reported so far contain this lipid (36–40). It functions as a sticker, stabilizing the contact between two Tom40 subunits through forming extra 37 hydrophobic protein–lipid interactions (Fig. 2B). This suggests that oligomerization may be influenced by the lipid environment, and the lipid composition of the outer mitochondrial membrane may play an important role in regulating the oligomeric state of the TOM complex (36, 38).Another 13 lipids (PC2–PC14) were resolved in each monomer in the higher resolution TOM core complex, among which PC03, PC06, PC10, PC11, and PC13 were also observed in a previous study (38) (SI Appendix, Fig. S6). However, detailed analysis indicated that the interactions between lipids and residues are different. The phosphate heads of PC02 and PC03 form H-bonds with R60 and R43 of Tom6, respectively, as well as Y254/W259 and W332/R348 of Tom40, respectively (Fig. 2A, 1 and 2). PC05, PC11, and PC13 form H-bonds with R29 of Tom6, W78/R82 of Tom22, and E46/P47 of Tom7, respectively (Fig. 2A, 3, 7, and 8). PC06 links K17 and Q20 of Tom7 to V210 of Tom40 (Fig. 2A, 4) via H-bonds. PC09 forms H-bonds with R24 of Tom7 and S183 of Tom40 (Fig. 2A, 6). PC10 links R82 of Tom22 to N156 of Tom40 via H-bonds (Fig. 2A, 9). All the above lipids functioned as bridges connecting Tom22, Tom6, and Tom7 to Tom40 to make the TOM core complex stable. Besides these H-bonds, we additionally observed four π-cation interactions between the choline moieties and the aromatic amino acid residues. W25 of Tom7 forms one π-cation interaction with the choline moiety of PC09 (Fig. 2A, 6). F72, F80, and W86 of Tom22 form three π-cation interactions with the choline moieties of PC05, PC14, and PC10, respectively (Fig. 2A, 3, 5, and 9). Thus, the choline moieties are specifically recognized by Tom subunits Tom7 and Tom22 through π-cation interactions, which suggests that the mitochondria-specific PCs may facilitate the localization of the TOM complex in the mitochondrial outer membrane.Fig. 3.Overall structure of the human Tom complex containing Tom22 and Tom20 cytosolic domains at 3.74 Å. (A) Cryo-EM density map and (B) the atomic model of the human TOM core complex containing Tom20 and Tom22 cytosolic domains in two different views. The 3.74-Å map is contoured at the 0.0109 σ-level to show the cytosolic domains. Tom5, Tom6, Tom7, Tom20, Tom22, and Tom40 are colored by orange, green, cyan, blue, magenta, and yellow, respectively. (C) Comparation of 2D classification of the cross-linked TOM complex and the uncross-linked TOM complex. White arrows indicate the position of Tom20 subunits. (D) Schematic of the secondary structural elements (Upper) of Tom22 and fitting of α-helices H1 to H4 of Tom22 (cartoon representation) into the EM density map (transparent white) in three different views. CD, cytosolic domain; TMH, transmembrane helix; IMSD, intermembrane space domain.Fig. 4.Cytosolic domain of Tom22 is indispensable for binding with the presequence. (A) Structural comparison of Tom20 and Tom22 cytosolic domains. (B) Interaction of Tom20 and Tom22 with Su9-DHFR-EGFP protein analyzed by in vitro pull-down assay. (C) Schematic of the constructions of Tom22-T1∼Tom22-T3′. (D) Interaction of the cytosolic domain of Tom22 with Su9-DHFR-EGFP protein analyzed by in vitro pull-down assay. (E) Surface electrostatic potentials of Tom22 and presequence from PDB: 3AWR. (F) Hypothetical models for Tom20 and Tom22 presequence binding and translocating. Top: Tom20 and Tom22 bind and translocate presequence to Tom40, respectively. Bottom: Tom20 binds and translocates presequence to Tom40 cooperating with Tom22.Structure of the TOM Complex Containing Tom22 and Tom20 Cytosolic Domains at 3.74 Å.By optimizing the purification method, Tom20, which was not determined in the previously published structures (38), and our TOM core complex, appears to be a stoichiometric component in our sample (SI Appendix, Fig. S1A). To stabilize Tom20 in the TOM complex, cross-linking was performed with glutaraldehyde. As a consequence, we resolved the human TOM complex containing Tom22 and Tom20 cytosolic domains at 3.74 Å (Fig. 3 A and B and SI Appendix, Figs. S7 and S8 and Table S1).Tom20, as the major import receptor, together with Tom70 and Tom22 target mitochondrial precursor proteins in the cytosol precisely (42). Previous studies concluded that Tom20 is anchored to the outer membrane by its N-terminal transmembrane segment and exposes its C-terminal receptor domain to the cytosol (45, 46). The structure of the cytosolic domain of Tom20 has been determined by X-ray diffraction and NMR (47, 49). However, the position of Tom20 in the TOM complex has not been defined, probably due to its dynamic feature. After cross-linking of the TOM complex, Tom20 was undoubtedly observed in the images of two-dimensional (2D) class averages compared with those of the noncross-linking TOM complex (Fig. 3C). Corresponding to the 2D results, the final reconstruction exhibited an extra density (Fig. 3A and SI Appendix, Fig. S8A), which we attributed to Tom20. Chemical cross-linking of proteins coupled with mass spectrometry (CXMS) was also carried out to help the assignment of Tom20. The result showed that K56 and K61 of Tom20 were cross-linked to both K184 of Tom40 and K76 of Tom22 (SI Appendix, Fig. S8B and Table S2), suggesting that the loop following α-helix3 of the Tom20 cytosolic domain should locate in the groove formed by Tom40 and Tom22 (SI Appendix, Fig. S8B). Combining the structure and the results of CXMS, we assigned the extra density as the Tom20 cytosolic domain unambiguously. However, we cannot build the atomic model de novo due to poor density, so we docked the crystal structure (Protein Data Bank [PDB]: 3AWR) into the map and manually adjusted it in order to match the map (Fig. 3A and SI Appendix, Fig. S8A). In our structure, the Tom20 cytosolic domain is located on the top of Tom40 and close to Tom22’s side, blocking the entrance of preproteins into the Tom40 β-barrel (Fig. 3 A and B).Tom22 is another important receptor of the TOM complex, but the structural basis of binding between Tom22 and preproteins is not clear (48, 50–52). In our structure, the almost full-length Tom22 structure was resolved (Fig. 3D). In addition to the C-terminal long bended α-helix4 (H4) that is consistent with the published structure, the complete N-terminal cytosolic domain was also resolved (Fig. 3D). The structure indicates that the Tom22 cytosolic domain is a three-helix bundle, which is very similar to the Tom20 cytosolic domain (Fig. 4A and SI Appendix, Fig. S9A). Additionally, the sequence alignment of Tom20 and Tom22 across multiple organisms also showed some structural similarity within the N terminus of Tom22 and C terminus of Tom20, especially for the α-helix H2 of Tom22 and the α-helix H5 of Tom20 (SI Appendix, Fig. S9B). Previous studies have shown that both Tom20 and Tom22 are involved in targeting signal recognition during protein import (42–44). Our pull-down experiments using purified Tom20 and Tom22 also supported this. As shown in Fig. 4B, both Tom20 and Tom22 were pulled down by the DHFR-EGFP protein fused with the presequence of subunit 9 of F0-ATPase (Su9) (42), but not by DHFR-EGFP protein without Su9, suggesting that Tom22 indeed forms a complex with the presequence as Tom20.As the cytosolic domain of Tom20 is the main site for presequence binding, we assumed that Tom22 might also bind to the presequence via its cytosolic domain. To verify this idea, we constructed three plasmids Tom22-T1, Tom22-T2, and Tom22-T3, by adding tobacco etch virus (TEV) protease sites to the joints of helices H1-H2, H2-H3, and H3-H4, respectively (Fig. 4C and SI Appendix, Fig. S10). Tom22 proteins without helix H1 (Tom22-T1′), helices H1 to H2 (Tom22-T2′) and helices H1 to H3 (Tom22-T3′) were obtained by using TEV protease (Fig. 4C). Pull-down experiments showed that Tom22-T1, Tom22-T2, and Tom22-T3 have the ability to bind to the presequence-fused DHFR-EGFP protein similar to wild-type Tom22 (Fig. 4D). However, deleting the first helix H1 led to a significant decrease (∼75%) in the binding ability (Fig. 4D). Electrostatic potential distribution analysis also supports the interaction between helix H1 and the presequence as they are all hydrophobic (Fig. 4E). Further deleting helix H2 only slightly affected the binding (Fig. 4D). After deleting all three helices, H1 to H3, Tom22 almost lost its ability to bind to the presequence (Fig. 4D). These results indicated that all of the three helices of the cytosolic domain of Tom22 collectively are indispensable for the binding of Tom22 with the presequence.Possible Presequence Binding Modes.Combining cryo-EM and cross-linking, we captured the structure of the TOM complex containing Tom22 and Tom20 cytosolic domains (Fig. 3). Further structure-guided biochemical analysis indicated that the N-terminal cytosolic domain of Tom22 is responsible for binding to the presequence, mainly via hydrophobic interactions (Fig. 4 D and E). As Tom20 and Tom22 are involved in the same pathway of targeting signal recognition (42), there may be two possible modes for presequence binding (Fig. 4F).One mode is that Tom20 and Tom22 may bind to presequences and guide the protein translocation independently. The other one is that the preprotein with a presequence may be recognized first by Tom20 and subsequently by Tom22. In this mode, the presequence-loaded cytosolic domain of Tom20 may move to the top of Tom40 and close to Tom22’s side through the swing of the α-helix2 (H2) of Tom20. Then Tom22 may accept the presequence (donated from Tom20) and guide it to the entrance of Tom40. The high-resolution structure of the presequence bound to the TOM complex would be required to test these hypotheses.

Overall Structure of Human TOM Core Complex at 2.53 Å.In order to prepare high-quality samples of the TOM complex for single-particle analysis, the plasmids expressing the C-terminal Flag-tagged human Tom22 were transformed into H293F cells. Mitochondria were first isolated from the transformed cells and then solubilized by digitonin. The TOM complexes were extracted using anti-Flag beads and further purified by the gel filtration chromatography. Using single-particle cryoelectron microscopy (cryo-EM), we resolved the structure of the human TOM core complex at an overall resolution of 2.53 Å (Fig. 1A and SI Appendix, Fig. S1). The quality of the density map is sufficient for assignment of main chains and many bulky side chains of each subunit (SI Appendix, Fig. S2 and Table S1). One lipid molecule phosphatidylcholine (PC) 01 (PC01) located between two Tom40 subunits was well defined in the density map (Fig. 2). The final atomic model shows that the human TOM complex forms a centrosymmetric dimer and each monomer contains subunits of Tom40, Tom22, Tom5, Tom6, and Tom7 with 1:1:1:1:1 stoichiometry, agreeing with previous studies (36, 38) (Fig. 1B and SI Appendix, Fig. S3A). The dimension of the dimeric TOM complex is 113 Å by 75 Å by 91 Å (Fig. 1B). When we superposed our structure to the yeast TOM complexes (37), differences were observed, especially for Tom7 and Tom22 (SI Appendix, Fig. S3B), suggesting the existence of structural deviations of the TOM complex between different species.Fig. 1.Overall structure of the human TOM core complex at 2.53 Å. (A) Cryo-EM density map and (B) the atomic model of the human TOM core complex in two different views. Tom5, Tom6, Tom7, Tom22, and Tom40 are colored by orange, green, cyan, magenta, and yellow, respectively. (C to F) Interactions between Tom5 (C), Tom22 (D), Tom6 (F), Tom7 (E), and Tom40. H-bonds and salt bridges observed in our structure and reported before are the red lines and black dotted lines, respectively. Residues from Tom5, Tom6, Tom7, and Tom22 are colored and labeled by orange, green, cyan, and magenta, respectively. Residues from Tom40 are colored by yellow and are labeled in black. Approximate IMS, cytosol space and OM boundaries are separated by black dashed lines.Fig. 2.Interactions between the human TOM core complex and lipid molecules. (A) Positions and interactions of PC lipids with corresponding Tom subunits. Lipids are colored by red. H-bonds and π-cation are indicated as black dashed lines. (B) PC01 forms hydrophobic interactions with Tom40 subunits. Residues from Tom40 are colored yellow and labeled in black. Lipids are colored red.The overall structure of the human TOM complex has a binocular-like architecture with the Tom40 portion corresponding to the lens cone (Fig. 1A). Tom40, as the core subunit of the TOM complex, forms the main conducting channels for precursor protein translocation (Fig. 1 A and B). Consistent with previous report (38, 41), Tom40 generates a β-barrel composed of 19 β-strands and 18 loop-like connections (SI Appendix, Fig. S4A). Strands β2 to β18 show an antiparallel pattern, and on the contrary, strands 1 and 19 associate in a parallel manner to close the barrel (SI Appendix, Fig. S4A). The unequal number of strands in the barrel requires the first strand to form parallel and antiparallel interactions with 2 adjacent strands at the same time. The distorted N-terminal α-helix of Tom40 nestles within the barrel, tethering the first β-strand (Fig. 1B and SI Appendix, Fig. S4B). Our structure shows the N-terminal α-helix of Tom40 forming a hydrogen bond (H-bond) network that facilitates its orientation against the interior wall of the pore adjacent to β-strands β8 to β19 (SI Appendix, Fig. S4B). Electrostatic potential distribution analysis showed that Tom40 possesses notable electrostatic features. The inner face of the Tom40 β-barrel is negatively charged, which plays a role in facilitating the translocation of mitochondrial charged preproteins. The outer face of the Tom40 β-barrel is hydrophobic, which helps the Tom40 β-barrel to insert into the lipid membrane (SI Appendix, Fig. S1E).Tom22, Tom5, Tom6, and Tom7, as receptors or cofactors, decorate around the Tom40 β-barrel (Fig. 1B). Due to the high quality of our structure, H-bonds and salt bridges formed between Tom40 and Tom22, Tom5, Tom6, and Tom7 are well defined. R21 and R39 of Tom5 form salt bridges with E244 and D198 of Tom40, and Y34/L38/R39 of Tom5 forms H-bonds with G232/Q223/Y221 of Tom40 (Fig. 1C). These interactions are important for Tom5 attaching to the whole complex (Fig. 1C). Meanwhile, T90/L95/E102 from Tom22 forms two H-bonds and one salt bridge with H347/Y303/K330 of Tom40 (Fig. 1D). R38 of Tom7 interacts with Y135/V136/G137 of Tom40 to form a H-bond network, and R24, L52, and D41 of Tom7 make H-bonds with G158, H117, and N111 of Tom40, respectively (Fig. 1E). Also, N48/R60/D64/D66/L67 of Tom6 interacts with V299/Y274/S279/Q281 of Tom40 via five H-bonds (Fig. 1F). Some of the interactions mentioned above (marked in red lines in Fig. 1) were not observed in the previously reported TOM complex structures (33).Previous studies have indicated that the TOM complex can exist in different oligomeric states from dimer, trimer, to tetramer and even higher oligomers, depending on the cellular environment (37, 38). The dimerization of the TOM complex is mainly mediated by Tom40. If we do not consider the lipid effects, the hydrophobic interface between two Tom40 protomers, formed by four β-strands (β17, β18, β19, and β1), is only 555 Å2. But after considering the lipid effects, things will be changed. In our structure, the lipid PC01 in the junction between two β-barrels contributes extra 401.9-Å2 and 395.8-Å2 interface areas to the two Tom40 protomers, respectively, enhancing the hydrophobic interaction (36) (SI Appendix, Fig. S5). This lipid is very stable as all high-resolution TOM complex structures reported so far contain this lipid (36–40). It functions as a sticker, stabilizing the contact between two Tom40 subunits through forming extra 37 hydrophobic protein–lipid interactions (Fig. 2B). This suggests that oligomerization may be influenced by the lipid environment, and the lipid composition of the outer mitochondrial membrane may play an important role in regulating the oligomeric state of the TOM complex (36, 38).Another 13 lipids (PC2–PC14) were resolved in each monomer in the higher resolution TOM core complex, among which PC03, PC06, PC10, PC11, and PC13 were also observed in a previous study (38) (SI Appendix, Fig. S6). However, detailed analysis indicated that the interactions between lipids and residues are different. The phosphate heads of PC02 and PC03 form H-bonds with R60 and R43 of Tom6, respectively, as well as Y254/W259 and W332/R348 of Tom40, respectively (Fig. 2A, 1 and 2). PC05, PC11, and PC13 form H-bonds with R29 of Tom6, W78/R82 of Tom22, and E46/P47 of Tom7, respectively (Fig. 2A, 3, 7, and 8). PC06 links K17 and Q20 of Tom7 to V210 of Tom40 (Fig. 2A, 4) via H-bonds. PC09 forms H-bonds with R24 of Tom7 and S183 of Tom40 (Fig. 2A, 6). PC10 links R82 of Tom22 to N156 of Tom40 via H-bonds (Fig. 2A, 9). All the above lipids functioned as bridges connecting Tom22, Tom6, and Tom7 to Tom40 to make the TOM core complex stable. Besides these H-bonds, we additionally observed four π-cation interactions between the choline moieties and the aromatic amino acid residues. W25 of Tom7 forms one π-cation interaction with the choline moiety of PC09 (Fig. 2A, 6). F72, F80, and W86 of Tom22 form three π-cation interactions with the choline moieties of PC05, PC14, and PC10, respectively (Fig. 2A, 3, 5, and 9). Thus, the choline moieties are specifically recognized by Tom subunits Tom7 and Tom22 through π-cation interactions, which suggests that the mitochondria-specific PCs may facilitate the localization of the TOM complex in the mitochondrial outer membrane.Fig. 3.Overall structure of the human Tom complex containing Tom22 and Tom20 cytosolic domains at 3.74 Å. (A) Cryo-EM density map and (B) the atomic model of the human TOM core complex containing Tom20 and Tom22 cytosolic domains in two different views. The 3.74-Å map is contoured at the 0.0109 σ-level to show the cytosolic domains. Tom5, Tom6, Tom7, Tom20, Tom22, and Tom40 are colored by orange, green, cyan, blue, magenta, and yellow, respectively. (C) Comparation of 2D classification of the cross-linked TOM complex and the uncross-linked TOM complex. White arrows indicate the position of Tom20 subunits. (D) Schematic of the secondary structural elements (Upper) of Tom22 and fitting of α-helices H1 to H4 of Tom22 (cartoon representation) into the EM density map (transparent white) in three different views. CD, cytosolic domain; TMH, transmembrane helix; IMSD, intermembrane space domain.Fig. 4.Cytosolic domain of Tom22 is indispensable for binding with the presequence. (A) Structural comparison of Tom20 and Tom22 cytosolic domains. (B) Interaction of Tom20 and Tom22 with Su9-DHFR-EGFP protein analyzed by in vitro pull-down assay. (C) Schematic of the constructions of Tom22-T1∼Tom22-T3′. (D) Interaction of the cytosolic domain of Tom22 with Su9-DHFR-EGFP protein analyzed by in vitro pull-down assay. (E) Surface electrostatic potentials of Tom22 and presequence from PDB: 3AWR. (F) Hypothetical models for Tom20 and Tom22 presequence binding and translocating. Top: Tom20 and Tom22 bind and translocate presequence to Tom40, respectively. Bottom: Tom20 binds and translocates presequence to Tom40 cooperating with Tom22.

Overall Structure of Human TOM Core Complex at 2.53 Å.

In order to prepare high-quality samples of the TOM complex for single-particle analysis, the plasmids expressing the C-terminal Flag-tagged human Tom22 were transformed into H293F cells. Mitochondria were first isolated from the transformed cells and then solubilized by digitonin. The TOM complexes were extracted using anti-Flag beads and further purified by the gel filtration chromatography. Using single-particle cryoelectron microscopy (cryo-EM), we resolved the structure of the human TOM core complex at an overall resolution of 2.53 Å (Fig. 1A and SI Appendix, Fig. S1). The quality of the density map is sufficient for assignment of main chains and many bulky side chains of each subunit (SI Appendix, Fig. S2 and Table S1). One lipid molecule phosphatidylcholine (PC) 01 (PC01) located between two Tom40 subunits was well defined in the density map (Fig. 2). The final atomic model shows that the human TOM complex forms a centrosymmetric dimer and each monomer contains subunits of Tom40, Tom22, Tom5, Tom6, and Tom7 with 1:1:1:1:1 stoichiometry, agreeing with previous studies (36, 38) (Fig. 1B and SI Appendix, Fig. S3A). The dimension of the dimeric TOM complex is 113 Å by 75 Å by 91 Å (Fig. 1B). When we superposed our structure to the yeast TOM complexes (37), differences were observed, especially for Tom7 and Tom22 (SI Appendix, Fig. S3B), suggesting the existence of structural deviations of the TOM complex between different species.

Overall structure of the human TOM core complex at 2.53 Å. (A) Cryo-EM density map and (B) the atomic model of the human TOM core complex in two different views. Tom5, Tom6, Tom7, Tom22, and Tom40 are colored by orange, green, cyan, magenta, and yellow, respectively. (C to F) Interactions between Tom5 (C), Tom22 (D), Tom6 (F), Tom7 (E), and Tom40. H-bonds and salt bridges observed in our structure and reported before are the red lines and black dotted lines, respectively. Residues from Tom5, Tom6, Tom7, and Tom22 are colored and labeled by orange, green, cyan, and magenta, respectively. Residues from Tom40 are colored by yellow and are labeled in black. Approximate IMS, cytosol space and OM boundaries are separated by black dashed lines.

Interactions between the human TOM core complex and lipid molecules. (A) Positions and interactions of PC lipids with corresponding Tom subunits. Lipids are colored by red. H-bonds and π-cation are indicated as black dashed lines. (B) PC01 forms hydrophobic interactions with Tom40 subunits. Residues from Tom40 are colored yellow and labeled in black. Lipids are colored red.

The overall structure of the human TOM complex has a binocular-like architecture with the Tom40 portion corresponding to the lens cone (Fig. 1A). Tom40, as the core subunit of the TOM complex, forms the main conducting channels for precursor protein translocation (Fig. 1 A and B). Consistent with previous report (38, 41), Tom40 generates a β-barrel composed of 19 β-strands and 18 loop-like connections (SI Appendix, Fig. S4A). Strands β2 to β18 show an antiparallel pattern, and on the contrary, strands 1 and 19 associate in a parallel manner to close the barrel (SI Appendix, Fig. S4A). The unequal number of strands in the barrel requires the first strand to form parallel and antiparallel interactions with 2 adjacent strands at the same time. The distorted N-terminal α-helix of Tom40 nestles within the barrel, tethering the first β-strand (Fig. 1B and SI Appendix, Fig. S4B). Our structure shows the N-terminal α-helix of Tom40 forming a hydrogen bond (H-bond) network that facilitates its orientation against the interior wall of the pore adjacent to β-strands β8 to β19 (SI Appendix, Fig. S4B). Electrostatic potential distribution analysis showed that Tom40 possesses notable electrostatic features. The inner face of the Tom40 β-barrel is negatively charged, which plays a role in facilitating the translocation of mitochondrial charged preproteins. The outer face of the Tom40 β-barrel is hydrophobic, which helps the Tom40 β-barrel to insert into the lipid membrane (SI Appendix, Fig. S1E).

Tom22, Tom5, Tom6, and Tom7, as receptors or cofactors, decorate around the Tom40 β-barrel (Fig. 1B). Due to the high quality of our structure, H-bonds and salt bridges formed between Tom40 and Tom22, Tom5, Tom6, and Tom7 are well defined. R21 and R39 of Tom5 form salt bridges with E244 and D198 of Tom40, and Y34/L38/R39 of Tom5 forms H-bonds with G232/Q223/Y221 of Tom40 (Fig. 1C). These interactions are important for Tom5 attaching to the whole complex (Fig. 1C). Meanwhile, T90/L95/E102 from Tom22 forms two H-bonds and one salt bridge with H347/Y303/K330 of Tom40 (Fig. 1D). R38 of Tom7 interacts with Y135/V136/G137 of Tom40 to form a H-bond network, and R24, L52, and D41 of Tom7 make H-bonds with G158, H117, and N111 of Tom40, respectively (Fig. 1E). Also, N48/R60/D64/D66/L67 of Tom6 interacts with V299/Y274/S279/Q281 of Tom40 via five H-bonds (Fig. 1F). Some of the interactions mentioned above (marked in red lines in Fig. 1) were not observed in the previously reported TOM complex structures (33).

Previous studies have indicated that the TOM complex can exist in different oligomeric states from dimer, trimer, to tetramer and even higher oligomers, depending on the cellular environment (37, 38). The dimerization of the TOM complex is mainly mediated by Tom40. If we do not consider the lipid effects, the hydrophobic interface between two Tom40 protomers, formed by four β-strands (β17, β18, β19, and β1), is only 555 Å2. But after considering the lipid effects, things will be changed. In our structure, the lipid PC01 in the junction between two β-barrels contributes extra 401.9-Å2 and 395.8-Å2 interface areas to the two Tom40 protomers, respectively, enhancing the hydrophobic interaction (36) (SI Appendix, Fig. S5). This lipid is very stable as all high-resolution TOM complex structures reported so far contain this lipid (36–40). It functions as a sticker, stabilizing the contact between two Tom40 subunits through forming extra 37 hydrophobic protein–lipid interactions (Fig. 2B). This suggests that oligomerization may be influenced by the lipid environment, and the lipid composition of the outer mitochondrial membrane may play an important role in regulating the oligomeric state of the TOM complex (36, 38).

Another 13 lipids (PC2–PC14) were resolved in each monomer in the higher resolution TOM core complex, among which PC03, PC06, PC10, PC11, and PC13 were also observed in a previous study (38) (SI Appendix, Fig. S6). However, detailed analysis indicated that the interactions between lipids and residues are different. The phosphate heads of PC02 and PC03 form H-bonds with R60 and R43 of Tom6, respectively, as well as Y254/W259 and W332/R348 of Tom40, respectively (Fig. 2A, 1 and 2). PC05, PC11, and PC13 form H-bonds with R29 of Tom6, W78/R82 of Tom22, and E46/P47 of Tom7, respectively (Fig. 2A, 3, 7, and 8). PC06 links K17 and Q20 of Tom7 to V210 of Tom40 (Fig. 2A, 4) via H-bonds. PC09 forms H-bonds with R24 of Tom7 and S183 of Tom40 (Fig. 2A, 6). PC10 links R82 of Tom22 to N156 of Tom40 via H-bonds (Fig. 2A, 9). All the above lipids functioned as bridges connecting Tom22, Tom6, and Tom7 to Tom40 to make the TOM core complex stable. Besides these H-bonds, we additionally observed four π-cation interactions between the choline moieties and the aromatic amino acid residues. W25 of Tom7 forms one π-cation interaction with the choline moiety of PC09 (Fig. 2A, 6). F72, F80, and W86 of Tom22 form three π-cation interactions with the choline moieties of PC05, PC14, and PC10, respectively (Fig. 2A, 3, 5, and 9). Thus, the choline moieties are specifically recognized by Tom subunits Tom7 and Tom22 through π-cation interactions, which suggests that the mitochondria-specific PCs may facilitate the localization of the TOM complex in the mitochondrial outer membrane.

Overall structure of the human Tom complex containing Tom22 and Tom20 cytosolic domains at 3.74 Å. (A) Cryo-EM density map and (B) the atomic model of the human TOM core complex containing Tom20 and Tom22 cytosolic domains in two different views. The 3.74-Å map is contoured at the 0.0109 σ-level to show the cytosolic domains. Tom5, Tom6, Tom7, Tom20, Tom22, and Tom40 are colored by orange, green, cyan, blue, magenta, and yellow, respectively. (C) Comparation of 2D classification of the cross-linked TOM complex and the uncross-linked TOM complex. White arrows indicate the position of Tom20 subunits. (D) Schematic of the secondary structural elements (Upper) of Tom22 and fitting of α-helices H1 to H4 of Tom22 (cartoon representation) into the EM density map (transparent white) in three different views. CD, cytosolic domain; TMH, transmembrane helix; IMSD, intermembrane space domain.

Cytosolic domain of Tom22 is indispensable for binding with the presequence. (A) Structural comparison of Tom20 and Tom22 cytosolic domains. (B) Interaction of Tom20 and Tom22 with Su9-DHFR-EGFP protein analyzed by in vitro pull-down assay. (C) Schematic of the constructions of Tom22-T1∼Tom22-T3′. (D) Interaction of the cytosolic domain of Tom22 with Su9-DHFR-EGFP protein analyzed by in vitro pull-down assay. (E) Surface electrostatic potentials of Tom22 and presequence from PDB: 3AWR. (F) Hypothetical models for Tom20 and Tom22 presequence binding and translocating. Top: Tom20 and Tom22 bind and translocate presequence to Tom40, respectively. Bottom: Tom20 binds and translocates presequence to Tom40 cooperating with Tom22.

Structure of the TOM Complex Containing Tom22 and Tom20 Cytosolic Domains at 3.74 Å.By optimizing the purification method, Tom20, which was not determined in the previously published structures (38), and our TOM core complex, appears to be a stoichiometric component in our sample (SI Appendix, Fig. S1A). To stabilize Tom20 in the TOM complex, cross-linking was performed with glutaraldehyde. As a consequence, we resolved the human TOM complex containing Tom22 and Tom20 cytosolic domains at 3.74 Å (Fig. 3 A and B and SI Appendix, Figs. S7 and S8 and Table S1).Tom20, as the major import receptor, together with Tom70 and Tom22 target mitochondrial precursor proteins in the cytosol precisely (42). Previous studies concluded that Tom20 is anchored to the outer membrane by its N-terminal transmembrane segment and exposes its C-terminal receptor domain to the cytosol (45, 46). The structure of the cytosolic domain of Tom20 has been determined by X-ray diffraction and NMR (47, 49). However, the position of Tom20 in the TOM complex has not been defined, probably due to its dynamic feature. After cross-linking of the TOM complex, Tom20 was undoubtedly observed in the images of two-dimensional (2D) class averages compared with those of the noncross-linking TOM complex (Fig. 3C). Corresponding to the 2D results, the final reconstruction exhibited an extra density (Fig. 3A and SI Appendix, Fig. S8A), which we attributed to Tom20. Chemical cross-linking of proteins coupled with mass spectrometry (CXMS) was also carried out to help the assignment of Tom20. The result showed that K56 and K61 of Tom20 were cross-linked to both K184 of Tom40 and K76 of Tom22 (SI Appendix, Fig. S8B and Table S2), suggesting that the loop following α-helix3 of the Tom20 cytosolic domain should locate in the groove formed by Tom40 and Tom22 (SI Appendix, Fig. S8B). Combining the structure and the results of CXMS, we assigned the extra density as the Tom20 cytosolic domain unambiguously. However, we cannot build the atomic model de novo due to poor density, so we docked the crystal structure (Protein Data Bank [PDB]: 3AWR) into the map and manually adjusted it in order to match the map (Fig. 3A and SI Appendix, Fig. S8A). In our structure, the Tom20 cytosolic domain is located on the top of Tom40 and close to Tom22’s side, blocking the entrance of preproteins into the Tom40 β-barrel (Fig. 3 A and B).Tom22 is another important receptor of the TOM complex, but the structural basis of binding between Tom22 and preproteins is not clear (48, 50–52). In our structure, the almost full-length Tom22 structure was resolved (Fig. 3D). In addition to the C-terminal long bended α-helix4 (H4) that is consistent with the published structure, the complete N-terminal cytosolic domain was also resolved (Fig. 3D). The structure indicates that the Tom22 cytosolic domain is a three-helix bundle, which is very similar to the Tom20 cytosolic domain (Fig. 4A and SI Appendix, Fig. S9A). Additionally, the sequence alignment of Tom20 and Tom22 across multiple organisms also showed some structural similarity within the N terminus of Tom22 and C terminus of Tom20, especially for the α-helix H2 of Tom22 and the α-helix H5 of Tom20 (SI Appendix, Fig. S9B). Previous studies have shown that both Tom20 and Tom22 are involved in targeting signal recognition during protein import (42–44). Our pull-down experiments using purified Tom20 and Tom22 also supported this. As shown in Fig. 4B, both Tom20 and Tom22 were pulled down by the DHFR-EGFP protein fused with the presequence of subunit 9 of F0-ATPase (Su9) (42), but not by DHFR-EGFP protein without Su9, suggesting that Tom22 indeed forms a complex with the presequence as Tom20.As the cytosolic domain of Tom20 is the main site for presequence binding, we assumed that Tom22 might also bind to the presequence via its cytosolic domain. To verify this idea, we constructed three plasmids Tom22-T1, Tom22-T2, and Tom22-T3, by adding tobacco etch virus (TEV) protease sites to the joints of helices H1-H2, H2-H3, and H3-H4, respectively (Fig. 4C and SI Appendix, Fig. S10). Tom22 proteins without helix H1 (Tom22-T1′), helices H1 to H2 (Tom22-T2′) and helices H1 to H3 (Tom22-T3′) were obtained by using TEV protease (Fig. 4C). Pull-down experiments showed that Tom22-T1, Tom22-T2, and Tom22-T3 have the ability to bind to the presequence-fused DHFR-EGFP protein similar to wild-type Tom22 (Fig. 4D). However, deleting the first helix H1 led to a significant decrease (∼75%) in the binding ability (Fig. 4D). Electrostatic potential distribution analysis also supports the interaction between helix H1 and the presequence as they are all hydrophobic (Fig. 4E). Further deleting helix H2 only slightly affected the binding (Fig. 4D). After deleting all three helices, H1 to H3, Tom22 almost lost its ability to bind to the presequence (Fig. 4D). These results indicated that all of the three helices of the cytosolic domain of Tom22 collectively are indispensable for the binding of Tom22 with the presequence.

Structure of the TOM Complex Containing Tom22 and Tom20 Cytosolic Domains at 3.74 Å.

By optimizing the purification method, Tom20, which was not determined in the previously published structures (38), and our TOM core complex, appears to be a stoichiometric component in our sample (SI Appendix, Fig. S1A). To stabilize Tom20 in the TOM complex, cross-linking was performed with glutaraldehyde. As a consequence, we resolved the human TOM complex containing Tom22 and Tom20 cytosolic domains at 3.74 Å (Fig. 3 A and B and SI Appendix, Figs. S7 and S8 and Table S1).

Tom20, as the major import receptor, together with Tom70 and Tom22 target mitochondrial precursor proteins in the cytosol precisely (42). Previous studies concluded that Tom20 is anchored to the outer membrane by its N-terminal transmembrane segment and exposes its C-terminal receptor domain to the cytosol (45, 46). The structure of the cytosolic domain of Tom20 has been determined by X-ray diffraction and NMR (47, 49). However, the position of Tom20 in the TOM complex has not been defined, probably due to its dynamic feature. After cross-linking of the TOM complex, Tom20 was undoubtedly observed in the images of two-dimensional (2D) class averages compared with those of the noncross-linking TOM complex (Fig. 3C). Corresponding to the 2D results, the final reconstruction exhibited an extra density (Fig. 3A and SI Appendix, Fig. S8A), which we attributed to Tom20. Chemical cross-linking of proteins coupled with mass spectrometry (CXMS) was also carried out to help the assignment of Tom20. The result showed that K56 and K61 of Tom20 were cross-linked to both K184 of Tom40 and K76 of Tom22 (SI Appendix, Fig. S8B and Table S2), suggesting that the loop following α-helix3 of the Tom20 cytosolic domain should locate in the groove formed by Tom40 and Tom22 (SI Appendix, Fig. S8B). Combining the structure and the results of CXMS, we assigned the extra density as the Tom20 cytosolic domain unambiguously. However, we cannot build the atomic model de novo due to poor density, so we docked the crystal structure (Protein Data Bank [PDB]: 3AWR) into the map and manually adjusted it in order to match the map (Fig. 3A and SI Appendix, Fig. S8A). In our structure, the Tom20 cytosolic domain is located on the top of Tom40 and close to Tom22’s side, blocking the entrance of preproteins into the Tom40 β-barrel (Fig. 3 A and B).

Tom22 is another important receptor of the TOM complex, but the structural basis of binding between Tom22 and preproteins is not clear (48, 50–52). In our structure, the almost full-length Tom22 structure was resolved (Fig. 3D). In addition to the C-terminal long bended α-helix4 (H4) that is consistent with the published structure, the complete N-terminal cytosolic domain was also resolved (Fig. 3D). The structure indicates that the Tom22 cytosolic domain is a three-helix bundle, which is very similar to the Tom20 cytosolic domain (Fig. 4A and SI Appendix, Fig. S9A). Additionally, the sequence alignment of Tom20 and Tom22 across multiple organisms also showed some structural similarity within the N terminus of Tom22 and C terminus of Tom20, especially for the α-helix H2 of Tom22 and the α-helix H5 of Tom20 (SI Appendix, Fig. S9B). Previous studies have shown that both Tom20 and Tom22 are involved in targeting signal recognition during protein import (42–44). Our pull-down experiments using purified Tom20 and Tom22 also supported this. As shown in Fig. 4B, both Tom20 and Tom22 were pulled down by the DHFR-EGFP protein fused with the presequence of subunit 9 of F0-ATPase (Su9) (42), but not by DHFR-EGFP protein without Su9, suggesting that Tom22 indeed forms a complex with the presequence as Tom20.

As the cytosolic domain of Tom20 is the main site for presequence binding, we assumed that Tom22 might also bind to the presequence via its cytosolic domain. To verify this idea, we constructed three plasmids Tom22-T1, Tom22-T2, and Tom22-T3, by adding tobacco etch virus (TEV) protease sites to the joints of helices H1-H2, H2-H3, and H3-H4, respectively (Fig. 4C and SI Appendix, Fig. S10). Tom22 proteins without helix H1 (Tom22-T1′), helices H1 to H2 (Tom22-T2′) and helices H1 to H3 (Tom22-T3′) were obtained by using TEV protease (Fig. 4C). Pull-down experiments showed that Tom22-T1, Tom22-T2, and Tom22-T3 have the ability to bind to the presequence-fused DHFR-EGFP protein similar to wild-type Tom22 (Fig. 4D). However, deleting the first helix H1 led to a significant decrease (∼75%) in the binding ability (Fig. 4D). Electrostatic potential distribution analysis also supports the interaction between helix H1 and the presequence as they are all hydrophobic (Fig. 4E). Further deleting helix H2 only slightly affected the binding (Fig. 4D). After deleting all three helices, H1 to H3, Tom22 almost lost its ability to bind to the presequence (Fig. 4D). These results indicated that all of the three helices of the cytosolic domain of Tom22 collectively are indispensable for the binding of Tom22 with the presequence.

Possible Presequence Binding Modes.Combining cryo-EM and cross-linking, we captured the structure of the TOM complex containing Tom22 and Tom20 cytosolic domains (Fig. 3). Further structure-guided biochemical analysis indicated that the N-terminal cytosolic domain of Tom22 is responsible for binding to the presequence, mainly via hydrophobic interactions (Fig. 4 D and E). As Tom20 and Tom22 are involved in the same pathway of targeting signal recognition (42), there may be two possible modes for presequence binding (Fig. 4F).One mode is that Tom20 and Tom22 may bind to presequences and guide the protein translocation independently. The other one is that the preprotein with a presequence may be recognized first by Tom20 and subsequently by Tom22. In this mode, the presequence-loaded cytosolic domain of Tom20 may move to the top of Tom40 and close to Tom22’s side through the swing of the α-helix2 (H2) of Tom20. Then Tom22 may accept the presequence (donated from Tom20) and guide it to the entrance of Tom40. The high-resolution structure of the presequence bound to the TOM complex would be required to test these hypotheses.

Possible Presequence Binding Modes.

Combining cryo-EM and cross-linking, we captured the structure of the TOM complex containing Tom22 and Tom20 cytosolic domains (Fig. 3). Further structure-guided biochemical analysis indicated that the N-terminal cytosolic domain of Tom22 is responsible for binding to the presequence, mainly via hydrophobic interactions (Fig. 4 D and E). As Tom20 and Tom22 are involved in the same pathway of targeting signal recognition (42), there may be two possible modes for presequence binding (Fig. 4F).

One mode is that Tom20 and Tom22 may bind to presequences and guide the protein translocation independently. The other one is that the preprotein with a presequence may be recognized first by Tom20 and subsequently by Tom22. In this mode, the presequence-loaded cytosolic domain of Tom20 may move to the top of Tom40 and close to Tom22’s side through the swing of the α-helix2 (H2) of Tom20. Then Tom22 may accept the presequence (donated from Tom20) and guide it to the entrance of Tom40. The high-resolution structure of the presequence bound to the TOM complex would be required to test these hypotheses.

Materials and MethodsCell Culture and Protein Expression.The cDNA of human Tom20 and Tom22 was cloned into the CMV-3 vector with C-terminal Flag-tag. The DNA of Su9-TEV-DHFR-EGFP-Strep was synthesized by General Biosystems Co. and cloned into the CMV-3 vector. The 293F (Invitrogen) cells were cultured in SMM 293-TII (Sino Biological) at 37 °C with 5% (vol/vol) CO2 in a ZCZY-CS8 shaker (Shanghai Zhichu Instrument Co.). When cell density reached 2 × 106 cells/mL, the plasmid was transfected into cells. After cultivation for 48 h, cells were collected and washed once with phosphate buffered saline (PBS) (137 mM NaCl, 2.7 mM KCl, 4.3 mM Na2HPO4, 1.4 mM KH2PO4, pH 7.2). Cell pellets were frozen in liquid nitrogen and stored at −80 °C.Mitochondria Isolation.Cell pellets were resuspended in buffer A (20 mM MOPS pH 7.4, 200 mM sorbitol, 70 mM sucrose, 0.5 mM ethylene diamine tetraacetic acid, 2 mg/mL bovine serum albumin, 0.5 mM phenylmethanesulfonyl fluoride (PMSF)) and incubated for 1 h. The suspension was transferred to homogenizer (Sigma) for 30 cycles on ice. Cells debris was removed by centrifugation at 2,000 × g for 15 min. The supernatant was collected and centrifuged again at 12,000 × g for 30 min to isolate the mitochondria. The pellet was collected as the pure mitochondria.TOM Complex Purification.Mitochondria were suspended with buffer B (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 1 mM PMSF) and mixed with 1% (vol/vol) digitonin (BIOSYNTH, No. D3200). After incubation for 3 h, the mixture was centrifuged at 150,000 × g for 30 min at 4 °C. Flag beads (Sigma) were added into the supernatant and incubated for 1 h at 4 °C. The beads were washed in a gravity column with 10 column volumes of washing buffer (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 0.01% digitonin). The TOM complex was eluted with washing buffer containing Flag peptides (Genscript) and concentrated to 100 μL using a 50-kDa cutoff centrifugal filter (Millipore). The complex was further purified by gel filtration with a Superose 6 Increase 3.2/300 GL column (GE Healthcare). Fractions were collected for sodium dodecyl sulfate polyAcrylamide gel electrophoresis (SDS-PAGE) analysis and cryosample preparation.Chemical Cross-Linking of Proteins for Cryo-EM.After purification with Flag beads (Sigma), the TOM complex was cross-linked by 0.1% glutaraldehyde (Sigma) for 1 h at 0 °C and the cross-linking reaction was quenched by 50 mM Tris⋅HCl (pH 8.0). The cross-linked complex was further purified by gel filtration with a Superose 6 Increase 3.2/300 GL column (GE Healthcare). Fractions were collected for SDS-PAGE analysis and concentrated to 20 μL using a 50-kDa cutoff centrifugal filter (Millipore) for the cryosample preparation.CXMS.The purified TOM protein complex was cross-linked by 1 mM disuccinimidyl suberate (DSS) (Thermo Fisher Scientific, 21655) for 1 h at room temperature and the cross-linking reaction was quenched by 50 mM Tris⋅HCl (pH 8.0). The cross-linked complex was purified by gel filtration with a Superose 6 Increase 3.2/300 GL column (GE Healthcare) and precipitated with ice-cold acetone of fourfold volume at −20 °C (53). The protein precipitation was resuspended in 8 M urea, 100 mM Tris, pH 8.5 by sonication for 10 min. After overnight trypsin digestion, the peptides were desalted by C18 spin columns (Pierce, 89870) according to the manufacturer’s instructions. The liquid chromotography with mass spectrometry)/mass spectrometry analysis was performed on an Easy-nLC 1000 ultra high performance liquid chromatography (Thermo Fisher Scientific) coupled to a Q Exactive HF Orbitrap mass spectrometer (Thermo Fisher Scientific). Peptides were loaded on a precolumn (75-μm inner diameter, 4 cm long, packed with ODS-AQ 12 nm to 10 mm beads from YMC Co.) and separated on an analytical column (75-μm inner diameter, 13 cm long, packed with ReproSil-Pur C18-AQ 1.9 μm 120 Å resin from Dr. Maisch GmbH) using a linear gradient. It was made with buffer A (0.1% formic acid) and buffer B (100% acetonitrile and 0.1% formic acid) as follows: 0 to 2 min, 0 to 4% B; 2 to 52 min, 4 to 36% B; 52 to 55 min, 36 to 95% B; and 55 to 60 min, 95% B. The flow rate was 350 nL/min. The top 20 most intense precursor ions from each full scan (resolution 60,000) were isolated for HCD MS2 (resolution 15,000; NCE 27) with a dynamic exclusion time of 30 s. Precursors with 1+, 2+, 8+, and above 8+ or unassigned charge states were excluded for fragmentation. pLink 2 was used to identify cross-linked peptides pairs with a false discovery rate <5% at the spectrum level, E value <1E-5, number of spectrum >2, which resulted in the identification of 14 cross-linked peptide pairs (54).Cryo-EM Sample Preparation and Data Collection.A total of 4 μL of sample with or without linking was applied to a glow-discharged grid (R 0.6/1 Au, 400 mesh, Quantifoil). Grids were bolted by FEI Mark IV Vitrobot for 3.5 s and plunged into liquid ethane. Images were collected on a Titan Krios G3i TEM (Thermo Fisher Scientific) operated at 300 kV with a Gatan K3 Summit direct electron detector and GIF Quantum imaging energy filter. Two datasets were acquired with AutoEMation (written by J. Lei) (55). The first dataset (without chemical cross-linking) was collected at a nominal magnification of ×105,000. The pixel size was 0.84 Å/pixel with defocus between −1.3 μm and −1.8 μm. The total dose rate on the detector was about 50 electrons/Å2 with a total exposure time of 1.28 s. Each micrograph stack contained 32 frames. Each micrograph was corrected for subregion motion correction and dose weighted using University of California San Francisco (UCSF) MotionCor2 (56). The second dataset (chemical cross-linking) was collected at a nominal magnification of ×81,000. The pixel size was 1.08 Å/pixel with defocus between −1.3 μm and −1.8 μm. The total dose rate on the detector was about 50 electrons/Å2 with the total exposure time of 2.56 s. Each micrograph stack contained 32 frames. Each micrograph was corrected for subregion motion correction and dose weighted using UCSF MotionCor2.Single-Particle Image Processing.For the first dataset of samples without chemical cross-linking, 3,297 micrographs were collected and imported into cryoSPARC (57) for Patch Contrast Transfer Function (CTF) estimation (Mulit). A total of 200 particles were manually picked by a manual picker and applied to 2D classification. Good classes were selected and used as templates to run the template picker. A total of 2,280,709 particles were picked with a particle diameter of 220 Å. Then ice contamination and bad particles were removed manually. Finally, 2,196,398 particles were extracted, binned fourfold, and applied to 2D classification. After 2D classification, 599,803 good particles were kept to do ab initio reconstruction with C1 symmetry. Four different classes were obtained and the best class accounted for 48.6% of the total particles. The particles of the best class were reextracted and applied to nonuniform refinement with C2 symmetry. The resolution was 2.53 Å based on the gold-standard Fourier shell correlation (FSC) 0.143 criteria.The second dataset of samples with chemical cross-linking had 2,421 micrographs. Using the result of the first dataset as a template, 2,206,847 particles were picked by template picker in cryoSPARC with particle diameter of 220 Å. After removing the ice contamination and bad particles, 1,563,576 particles were extracted and binned fourfold. Two rounds of 2D classification were used to remove bad particles. A total of 717,656 particles were kept and applied to ab initio reconstruction with C1 symmetry. The particles of the best class accounted for 45.6% of all particles and reextracted without binning. A total of 401,033 particles were applied to 2D classification. After that, 347,601 particles were selected and processed by cryoSPARC and RELION (58–60) separately. In cryoSPARC, the selected particles were applied to nonuniform refinement with C2 symmetry, which resulted in the final resolution of 2.74 Å. In RELION, particles were applied to three-dimensional (3D) classification with only one class, and then refinement was performed with the result of the 3D classification as the reference. The final resolution was 3.74 Å. To improve the resolution of Tom20, a local mask surrounding the region of Tom20 was created based on the final density map using Chimera (61). Refinement was rerun with the local mask and the final resolution was 4.47 Å.Model Building and Refinement.The β-barrel structure of Tom40 was clearly identified from the 2.53-Å resolution map. The homology model of Tom40 was derived from human VDAC1 channel (PDB: 6G73) (62). The unrefined Tom40 model was first rigid-body fitted into the sharpened map using UCSF Chimera, showing good agreement with the density data. Additional adjustments to the backbone and side chains for this model were performed manually in COOT (63), residue by residue. Due to the high resolution of the map, Tom22, Tom5, Tom6, and Tom7 subunits can be traced easily compared to the yeast Tom complex structure (PDB: 6UCU). Tom22, Tom5, Tom6, and Tom7 density maps were segmented out with Chimera, separately. The initial models of Tom22, Tom5, Tom6, and Tom7 were generated separately with PHENIX (64) (map to model function) for supplying segmented maps and sequences. Then Tom22, Tom5, Tom6, and Tom7 initial models were also rigid-body fitted into the map using UCSF Chimera and refined residue by residue using COOT. A very clear lipid density was identified between two Tom40 subunits. For simplicity, the lipid in the structure was modeled as PCs. The length of the acyl chain of the lipid was adjusted according to specific density. Finally, Tom40, Tom22, Tom5, Tom6, and Tom7 were combined into a single full complex PDB file. The full model of the TOM complex with ligands was subjected to real-space refinement in PHENIX. The geometries of the final model were validated with the comprehensive model validation section in PHENIX and detailed information is listed in SI Appendix, Table S1. For Tom20, we docked the crystal structure of Tom20 (PDB: 3AWR) into the map and manually adjusted it in order to match the map in COOT, and the final model was further refined by PHENIX.Constructions of Plasmid.We constructed three plasmids Tom22-T1, Tom22-T2, and Tom22-T3 by adding TEV protease site (GAAAACCTGTATTTTCAGAGC) to the joints of helices H1-H2, H2-H3, and H3-H4, respectively, by PCR. Plasmid was conducted by the PerfectStart Green PCR SuperMix (Trans). The primer sequences were as follows:Tom22-H1, F: GAAAACCTGTATTTTCAGAGCGGCGACGCGGAGAAGCCTTom22-H1, R: GCTCTGAAAATACAGGTTTTCTTTCGGGAGCAATTCGTCTom22-H2, F: GAAAACCTGTATTTTCAGAGCACCCTGTCGGAGAGACTATom22-H2, R: GCTCTGAAAATACAGGTTTTCTCATCTAGCTCCTCATCTom22-H3, F: GAAAACCTGTATTTTCAGAGCXXGGAGAGGGTCCGGTCCTom22-H3, R: GCTCTGAAAATACAGGTTTTAAACATCTCCGTCAGGCC.Tom22, Tom20, and Su9-TEV-DHFR-EGFP Purification.For purification of Tom20 and Tom22, wild-type or indicated variants, cell debris was collected during mitochondria isolation (described above) and 1% digitonin was added to extract protein. The suspension was centrifuged at 150,000 × g for 30 min at 4 °C and then the supernatant was incubated with Flag-affinity beads (Sigma) for 1 h at 4 °C. The resin was washed in a gravity column by washing buffer (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 0.01% digitonin) with 10 column volumes. Proteins were eluted with Flag peptides (Genscript) and concentrated to 100 μL using a 20-kDa cutoff centrifugal filter (Millipore) and analyzed by SDS-PAGE. Truncated Tom22 proteins were obtained by treating with TEV protease to remove different parts. All proteins were further purified by gel filtration with a Superose 200 Increase 3.2/300 GL column (GE Healthcare) and the fractions were collected.For Su9-TEV-DHFR-EGFP, the 293F cells were collected and extracted by 1% digitonin. After centrifugation, supernatant was incubated with Strep-affinity beads (Sigma). Su9-TEV-DHFR-EGFP was eluted with buffer (20 mM MOPS pH 7.4, 150 mM KCI) containing 5 mM desthiobiotin (Sigma). Part of the protein was treated with TEV protease to remove Su9. All proteins were further purified by gel filtration with a Superose 200 Increase 3.2/300 GL column (GE Healthcare) and the fractions were collected.Pull-Down.Su9-TEV-DHFR-EGFP was incubated with Tom20 and Tom22 for 1 h at room temperature, respectively. Then Strep-affinity beads (Sigma) were added to the mixture for 1 h at 4 °C. After incubation, the resin was washed in a gravity column by washing buffer (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 0.01% digitonin) with 10 column volumes. All proteins were eluted with washing buffer containing 5 mM desthiobiotin (Sigma). The elution was collected and analyzed by Western blot. DHFR-EGFP was used as a negative control to perform the same experiment described above.Tom22, wild-type or indicated variants, were incubated with Su9-TEV-DHFR-EGFP for 1 h at room temperature, respectively. Then Flag-affinity beads (Sigma) were added to the mixture for 1 h at 4 °C. After incubation, the resin was washed in a gravity column by washing buffer (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 0.01% digitonin) with 10 column volumes. All proteins were eluted with washing buffer containing Flag peptides (Genscript). The elution was collected and analyzed by Western blot.Western Blotting.The samples were electrotransferred to polyvinylidene fluoride membranes (Millipore) at 90 V for 1 h at 4 °C after SDS-PAGE. The membranes were incubated with methyl alcohol for 3 min and washed for 3 min with the transfer buffer (25 mM Tris, 192 mM glycine). Then, the membranes were blocked by 5% nonfat milk at room temperature for 1 h. After blocking, the membranes were incubated with primary antibodies, anti-Flag antibodies (Proteintech Company) for Tom20 and Tom22, or anti-Strep antibodies (Proteintech) for Su9-TEV-DHFR-EGFP and DHFR-EGFP, at room temperature for 1 h with a 2,000× dilution. Then the membranes were washed by TBS-T buffer (150 mM NaCl, 10 mM Tris, 10% Tween-20). The secondary antibody, anti-mouse antibody (Proteintech), was incubated with the membranes at room temperature for 1 h with a 10,000× dilution. After thorough washing, the protein bands were visualized by Western Quick Block Kit (GenScript) on Chemidoc MP (BIO-RAD).

Materials and Methods

Cell Culture and Protein Expression.The cDNA of human Tom20 and Tom22 was cloned into the CMV-3 vector with C-terminal Flag-tag. The DNA of Su9-TEV-DHFR-EGFP-Strep was synthesized by General Biosystems Co. and cloned into the CMV-3 vector. The 293F (Invitrogen) cells were cultured in SMM 293-TII (Sino Biological) at 37 °C with 5% (vol/vol) CO2 in a ZCZY-CS8 shaker (Shanghai Zhichu Instrument Co.). When cell density reached 2 × 106 cells/mL, the plasmid was transfected into cells. After cultivation for 48 h, cells were collected and washed once with phosphate buffered saline (PBS) (137 mM NaCl, 2.7 mM KCl, 4.3 mM Na2HPO4, 1.4 mM KH2PO4, pH 7.2). Cell pellets were frozen in liquid nitrogen and stored at −80 °C.

Cell Culture and Protein Expression.

The cDNA of human Tom20 and Tom22 was cloned into the CMV-3 vector with C-terminal Flag-tag. The DNA of Su9-TEV-DHFR-EGFP-Strep was synthesized by General Biosystems Co. and cloned into the CMV-3 vector. The 293F (Invitrogen) cells were cultured in SMM 293-TII (Sino Biological) at 37 °C with 5% (vol/vol) CO2 in a ZCZY-CS8 shaker (Shanghai Zhichu Instrument Co.). When cell density reached 2 × 106 cells/mL, the plasmid was transfected into cells. After cultivation for 48 h, cells were collected and washed once with phosphate buffered saline (PBS) (137 mM NaCl, 2.7 mM KCl, 4.3 mM Na2HPO4, 1.4 mM KH2PO4, pH 7.2). Cell pellets were frozen in liquid nitrogen and stored at −80 °C.

Mitochondria Isolation.Cell pellets were resuspended in buffer A (20 mM MOPS pH 7.4, 200 mM sorbitol, 70 mM sucrose, 0.5 mM ethylene diamine tetraacetic acid, 2 mg/mL bovine serum albumin, 0.5 mM phenylmethanesulfonyl fluoride (PMSF)) and incubated for 1 h. The suspension was transferred to homogenizer (Sigma) for 30 cycles on ice. Cells debris was removed by centrifugation at 2,000 × g for 15 min. The supernatant was collected and centrifuged again at 12,000 × g for 30 min to isolate the mitochondria. The pellet was collected as the pure mitochondria.

Mitochondria Isolation.

Cell pellets were resuspended in buffer A (20 mM MOPS pH 7.4, 200 mM sorbitol, 70 mM sucrose, 0.5 mM ethylene diamine tetraacetic acid, 2 mg/mL bovine serum albumin, 0.5 mM phenylmethanesulfonyl fluoride (PMSF)) and incubated for 1 h. The suspension was transferred to homogenizer (Sigma) for 30 cycles on ice. Cells debris was removed by centrifugation at 2,000 × g for 15 min. The supernatant was collected and centrifuged again at 12,000 × g for 30 min to isolate the mitochondria. The pellet was collected as the pure mitochondria.

TOM Complex Purification.Mitochondria were suspended with buffer B (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 1 mM PMSF) and mixed with 1% (vol/vol) digitonin (BIOSYNTH, No. D3200). After incubation for 3 h, the mixture was centrifuged at 150,000 × g for 30 min at 4 °C. Flag beads (Sigma) were added into the supernatant and incubated for 1 h at 4 °C. The beads were washed in a gravity column with 10 column volumes of washing buffer (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 0.01% digitonin). The TOM complex was eluted with washing buffer containing Flag peptides (Genscript) and concentrated to 100 μL using a 50-kDa cutoff centrifugal filter (Millipore). The complex was further purified by gel filtration with a Superose 6 Increase 3.2/300 GL column (GE Healthcare). Fractions were collected for sodium dodecyl sulfate polyAcrylamide gel electrophoresis (SDS-PAGE) analysis and cryosample preparation.

TOM Complex Purification.

Mitochondria were suspended with buffer B (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 1 mM PMSF) and mixed with 1% (vol/vol) digitonin (BIOSYNTH, No. D3200). After incubation for 3 h, the mixture was centrifuged at 150,000 × g for 30 min at 4 °C. Flag beads (Sigma) were added into the supernatant and incubated for 1 h at 4 °C. The beads were washed in a gravity column with 10 column volumes of washing buffer (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 0.01% digitonin). The TOM complex was eluted with washing buffer containing Flag peptides (Genscript) and concentrated to 100 μL using a 50-kDa cutoff centrifugal filter (Millipore). The complex was further purified by gel filtration with a Superose 6 Increase 3.2/300 GL column (GE Healthcare). Fractions were collected for sodium dodecyl sulfate polyAcrylamide gel electrophoresis (SDS-PAGE) analysis and cryosample preparation.

Chemical Cross-Linking of Proteins for Cryo-EM.After purification with Flag beads (Sigma), the TOM complex was cross-linked by 0.1% glutaraldehyde (Sigma) for 1 h at 0 °C and the cross-linking reaction was quenched by 50 mM Tris⋅HCl (pH 8.0). The cross-linked complex was further purified by gel filtration with a Superose 6 Increase 3.2/300 GL column (GE Healthcare). Fractions were collected for SDS-PAGE analysis and concentrated to 20 μL using a 50-kDa cutoff centrifugal filter (Millipore) for the cryosample preparation.

Chemical Cross-Linking of Proteins for Cryo-EM.

After purification with Flag beads (Sigma), the TOM complex was cross-linked by 0.1% glutaraldehyde (Sigma) for 1 h at 0 °C and the cross-linking reaction was quenched by 50 mM Tris⋅HCl (pH 8.0). The cross-linked complex was further purified by gel filtration with a Superose 6 Increase 3.2/300 GL column (GE Healthcare). Fractions were collected for SDS-PAGE analysis and concentrated to 20 μL using a 50-kDa cutoff centrifugal filter (Millipore) for the cryosample preparation.

CXMS.The purified TOM protein complex was cross-linked by 1 mM disuccinimidyl suberate (DSS) (Thermo Fisher Scientific, 21655) for 1 h at room temperature and the cross-linking reaction was quenched by 50 mM Tris⋅HCl (pH 8.0). The cross-linked complex was purified by gel filtration with a Superose 6 Increase 3.2/300 GL column (GE Healthcare) and precipitated with ice-cold acetone of fourfold volume at −20 °C (53). The protein precipitation was resuspended in 8 M urea, 100 mM Tris, pH 8.5 by sonication for 10 min. After overnight trypsin digestion, the peptides were desalted by C18 spin columns (Pierce, 89870) according to the manufacturer’s instructions. The liquid chromotography with mass spectrometry)/mass spectrometry analysis was performed on an Easy-nLC 1000 ultra high performance liquid chromatography (Thermo Fisher Scientific) coupled to a Q Exactive HF Orbitrap mass spectrometer (Thermo Fisher Scientific). Peptides were loaded on a precolumn (75-μm inner diameter, 4 cm long, packed with ODS-AQ 12 nm to 10 mm beads from YMC Co.) and separated on an analytical column (75-μm inner diameter, 13 cm long, packed with ReproSil-Pur C18-AQ 1.9 μm 120 Å resin from Dr. Maisch GmbH) using a linear gradient. It was made with buffer A (0.1% formic acid) and buffer B (100% acetonitrile and 0.1% formic acid) as follows: 0 to 2 min, 0 to 4% B; 2 to 52 min, 4 to 36% B; 52 to 55 min, 36 to 95% B; and 55 to 60 min, 95% B. The flow rate was 350 nL/min. The top 20 most intense precursor ions from each full scan (resolution 60,000) were isolated for HCD MS2 (resolution 15,000; NCE 27) with a dynamic exclusion time of 30 s. Precursors with 1+, 2+, 8+, and above 8+ or unassigned charge states were excluded for fragmentation. pLink 2 was used to identify cross-linked peptides pairs with a false discovery rate <5% at the spectrum level, E value <1E-5, number of spectrum >2, which resulted in the identification of 14 cross-linked peptide pairs (54).

The purified TOM protein complex was cross-linked by 1 mM disuccinimidyl suberate (DSS) (Thermo Fisher Scientific, 21655) for 1 h at room temperature and the cross-linking reaction was quenched by 50 mM Tris⋅HCl (pH 8.0). The cross-linked complex was purified by gel filtration with a Superose 6 Increase 3.2/300 GL column (GE Healthcare) and precipitated with ice-cold acetone of fourfold volume at −20 °C (53). The protein precipitation was resuspended in 8 M urea, 100 mM Tris, pH 8.5 by sonication for 10 min. After overnight trypsin digestion, the peptides were desalted by C18 spin columns (Pierce, 89870) according to the manufacturer’s instructions. The liquid chromotography with mass spectrometry)/mass spectrometry analysis was performed on an Easy-nLC 1000 ultra high performance liquid chromatography (Thermo Fisher Scientific) coupled to a Q Exactive HF Orbitrap mass spectrometer (Thermo Fisher Scientific). Peptides were loaded on a precolumn (75-μm inner diameter, 4 cm long, packed with ODS-AQ 12 nm to 10 mm beads from YMC Co.) and separated on an analytical column (75-μm inner diameter, 13 cm long, packed with ReproSil-Pur C18-AQ 1.9 μm 120 Å resin from Dr. Maisch GmbH) using a linear gradient. It was made with buffer A (0.1% formic acid) and buffer B (100% acetonitrile and 0.1% formic acid) as follows: 0 to 2 min, 0 to 4% B; 2 to 52 min, 4 to 36% B; 52 to 55 min, 36 to 95% B; and 55 to 60 min, 95% B. The flow rate was 350 nL/min. The top 20 most intense precursor ions from each full scan (resolution 60,000) were isolated for HCD MS2 (resolution 15,000; NCE 27) with a dynamic exclusion time of 30 s. Precursors with 1+, 2+, 8+, and above 8+ or unassigned charge states were excluded for fragmentation. pLink 2 was used to identify cross-linked peptides pairs with a false discovery rate <5% at the spectrum level, E value <1E-5, number of spectrum >2, which resulted in the identification of 14 cross-linked peptide pairs (54).

Cryo-EM Sample Preparation and Data Collection.A total of 4 μL of sample with or without linking was applied to a glow-discharged grid (R 0.6/1 Au, 400 mesh, Quantifoil). Grids were bolted by FEI Mark IV Vitrobot for 3.5 s and plunged into liquid ethane. Images were collected on a Titan Krios G3i TEM (Thermo Fisher Scientific) operated at 300 kV with a Gatan K3 Summit direct electron detector and GIF Quantum imaging energy filter. Two datasets were acquired with AutoEMation (written by J. Lei) (55). The first dataset (without chemical cross-linking) was collected at a nominal magnification of ×105,000. The pixel size was 0.84 Å/pixel with defocus between −1.3 μm and −1.8 μm. The total dose rate on the detector was about 50 electrons/Å2 with a total exposure time of 1.28 s. Each micrograph stack contained 32 frames. Each micrograph was corrected for subregion motion correction and dose weighted using University of California San Francisco (UCSF) MotionCor2 (56). The second dataset (chemical cross-linking) was collected at a nominal magnification of ×81,000. The pixel size was 1.08 Å/pixel with defocus between −1.3 μm and −1.8 μm. The total dose rate on the detector was about 50 electrons/Å2 with the total exposure time of 2.56 s. Each micrograph stack contained 32 frames. Each micrograph was corrected for subregion motion correction and dose weighted using UCSF MotionCor2.

Cryo-EM Sample Preparation and Data Collection.

A total of 4 μL of sample with or without linking was applied to a glow-discharged grid (R 0.6/1 Au, 400 mesh, Quantifoil). Grids were bolted by FEI Mark IV Vitrobot for 3.5 s and plunged into liquid ethane. Images were collected on a Titan Krios G3i TEM (Thermo Fisher Scientific) operated at 300 kV with a Gatan K3 Summit direct electron detector and GIF Quantum imaging energy filter. Two datasets were acquired with AutoEMation (written by J. Lei) (55). The first dataset (without chemical cross-linking) was collected at a nominal magnification of ×105,000. The pixel size was 0.84 Å/pixel with defocus between −1.3 μm and −1.8 μm. The total dose rate on the detector was about 50 electrons/Å2 with a total exposure time of 1.28 s. Each micrograph stack contained 32 frames. Each micrograph was corrected for subregion motion correction and dose weighted using University of California San Francisco (UCSF) MotionCor2 (56). The second dataset (chemical cross-linking) was collected at a nominal magnification of ×81,000. The pixel size was 1.08 Å/pixel with defocus between −1.3 μm and −1.8 μm. The total dose rate on the detector was about 50 electrons/Å2 with the total exposure time of 2.56 s. Each micrograph stack contained 32 frames. Each micrograph was corrected for subregion motion correction and dose weighted using UCSF MotionCor2.

Single-Particle Image Processing.For the first dataset of samples without chemical cross-linking, 3,297 micrographs were collected and imported into cryoSPARC (57) for Patch Contrast Transfer Function (CTF) estimation (Mulit). A total of 200 particles were manually picked by a manual picker and applied to 2D classification. Good classes were selected and used as templates to run the template picker. A total of 2,280,709 particles were picked with a particle diameter of 220 Å. Then ice contamination and bad particles were removed manually. Finally, 2,196,398 particles were extracted, binned fourfold, and applied to 2D classification. After 2D classification, 599,803 good particles were kept to do ab initio reconstruction with C1 symmetry. Four different classes were obtained and the best class accounted for 48.6% of the total particles. The particles of the best class were reextracted and applied to nonuniform refinement with C2 symmetry. The resolution was 2.53 Å based on the gold-standard Fourier shell correlation (FSC) 0.143 criteria.The second dataset of samples with chemical cross-linking had 2,421 micrographs. Using the result of the first dataset as a template, 2,206,847 particles were picked by template picker in cryoSPARC with particle diameter of 220 Å. After removing the ice contamination and bad particles, 1,563,576 particles were extracted and binned fourfold. Two rounds of 2D classification were used to remove bad particles. A total of 717,656 particles were kept and applied to ab initio reconstruction with C1 symmetry. The particles of the best class accounted for 45.6% of all particles and reextracted without binning. A total of 401,033 particles were applied to 2D classification. After that, 347,601 particles were selected and processed by cryoSPARC and RELION (58–60) separately. In cryoSPARC, the selected particles were applied to nonuniform refinement with C2 symmetry, which resulted in the final resolution of 2.74 Å. In RELION, particles were applied to three-dimensional (3D) classification with only one class, and then refinement was performed with the result of the 3D classification as the reference. The final resolution was 3.74 Å. To improve the resolution of Tom20, a local mask surrounding the region of Tom20 was created based on the final density map using Chimera (61). Refinement was rerun with the local mask and the final resolution was 4.47 Å.

Single-Particle Image Processing.

For the first dataset of samples without chemical cross-linking, 3,297 micrographs were collected and imported into cryoSPARC (57) for Patch Contrast Transfer Function (CTF) estimation (Mulit). A total of 200 particles were manually picked by a manual picker and applied to 2D classification. Good classes were selected and used as templates to run the template picker. A total of 2,280,709 particles were picked with a particle diameter of 220 Å. Then ice contamination and bad particles were removed manually. Finally, 2,196,398 particles were extracted, binned fourfold, and applied to 2D classification. After 2D classification, 599,803 good particles were kept to do ab initio reconstruction with C1 symmetry. Four different classes were obtained and the best class accounted for 48.6% of the total particles. The particles of the best class were reextracted and applied to nonuniform refinement with C2 symmetry. The resolution was 2.53 Å based on the gold-standard Fourier shell correlation (FSC) 0.143 criteria.

The second dataset of samples with chemical cross-linking had 2,421 micrographs. Using the result of the first dataset as a template, 2,206,847 particles were picked by template picker in cryoSPARC with particle diameter of 220 Å. After removing the ice contamination and bad particles, 1,563,576 particles were extracted and binned fourfold. Two rounds of 2D classification were used to remove bad particles. A total of 717,656 particles were kept and applied to ab initio reconstruction with C1 symmetry. The particles of the best class accounted for 45.6% of all particles and reextracted without binning. A total of 401,033 particles were applied to 2D classification. After that, 347,601 particles were selected and processed by cryoSPARC and RELION (58–60) separately. In cryoSPARC, the selected particles were applied to nonuniform refinement with C2 symmetry, which resulted in the final resolution of 2.74 Å. In RELION, particles were applied to three-dimensional (3D) classification with only one class, and then refinement was performed with the result of the 3D classification as the reference. The final resolution was 3.74 Å. To improve the resolution of Tom20, a local mask surrounding the region of Tom20 was created based on the final density map using Chimera (61). Refinement was rerun with the local mask and the final resolution was 4.47 Å.

Model Building and Refinement.The β-barrel structure of Tom40 was clearly identified from the 2.53-Å resolution map. The homology model of Tom40 was derived from human VDAC1 channel (PDB: 6G73) (62). The unrefined Tom40 model was first rigid-body fitted into the sharpened map using UCSF Chimera, showing good agreement with the density data. Additional adjustments to the backbone and side chains for this model were performed manually in COOT (63), residue by residue. Due to the high resolution of the map, Tom22, Tom5, Tom6, and Tom7 subunits can be traced easily compared to the yeast Tom complex structure (PDB: 6UCU). Tom22, Tom5, Tom6, and Tom7 density maps were segmented out with Chimera, separately. The initial models of Tom22, Tom5, Tom6, and Tom7 were generated separately with PHENIX (64) (map to model function) for supplying segmented maps and sequences. Then Tom22, Tom5, Tom6, and Tom7 initial models were also rigid-body fitted into the map using UCSF Chimera and refined residue by residue using COOT. A very clear lipid density was identified between two Tom40 subunits. For simplicity, the lipid in the structure was modeled as PCs. The length of the acyl chain of the lipid was adjusted according to specific density. Finally, Tom40, Tom22, Tom5, Tom6, and Tom7 were combined into a single full complex PDB file. The full model of the TOM complex with ligands was subjected to real-space refinement in PHENIX. The geometries of the final model were validated with the comprehensive model validation section in PHENIX and detailed information is listed in SI Appendix, Table S1. For Tom20, we docked the crystal structure of Tom20 (PDB: 3AWR) into the map and manually adjusted it in order to match the map in COOT, and the final model was further refined by PHENIX.

Model Building and Refinement.

The β-barrel structure of Tom40 was clearly identified from the 2.53-Å resolution map. The homology model of Tom40 was derived from human VDAC1 channel (PDB: 6G73) (62). The unrefined Tom40 model was first rigid-body fitted into the sharpened map using UCSF Chimera, showing good agreement with the density data. Additional adjustments to the backbone and side chains for this model were performed manually in COOT (63), residue by residue. Due to the high resolution of the map, Tom22, Tom5, Tom6, and Tom7 subunits can be traced easily compared to the yeast Tom complex structure (PDB: 6UCU). Tom22, Tom5, Tom6, and Tom7 density maps were segmented out with Chimera, separately. The initial models of Tom22, Tom5, Tom6, and Tom7 were generated separately with PHENIX (64) (map to model function) for supplying segmented maps and sequences. Then Tom22, Tom5, Tom6, and Tom7 initial models were also rigid-body fitted into the map using UCSF Chimera and refined residue by residue using COOT. A very clear lipid density was identified between two Tom40 subunits. For simplicity, the lipid in the structure was modeled as PCs. The length of the acyl chain of the lipid was adjusted according to specific density. Finally, Tom40, Tom22, Tom5, Tom6, and Tom7 were combined into a single full complex PDB file. The full model of the TOM complex with ligands was subjected to real-space refinement in PHENIX. The geometries of the final model were validated with the comprehensive model validation section in PHENIX and detailed information is listed in SI Appendix, Table S1. For Tom20, we docked the crystal structure of Tom20 (PDB: 3AWR) into the map and manually adjusted it in order to match the map in COOT, and the final model was further refined by PHENIX.

Constructions of Plasmid.We constructed three plasmids Tom22-T1, Tom22-T2, and Tom22-T3 by adding TEV protease site (GAAAACCTGTATTTTCAGAGC) to the joints of helices H1-H2, H2-H3, and H3-H4, respectively, by PCR. Plasmid was conducted by the PerfectStart Green PCR SuperMix (Trans). The primer sequences were as follows:Tom22-H1, F: GAAAACCTGTATTTTCAGAGCGGCGACGCGGAGAAGCCTTom22-H1, R: GCTCTGAAAATACAGGTTTTCTTTCGGGAGCAATTCGTCTom22-H2, F: GAAAACCTGTATTTTCAGAGCACCCTGTCGGAGAGACTATom22-H2, R: GCTCTGAAAATACAGGTTTTCTCATCTAGCTCCTCATCTom22-H3, F: GAAAACCTGTATTTTCAGAGCXXGGAGAGGGTCCGGTCCTom22-H3, R: GCTCTGAAAATACAGGTTTTAAACATCTCCGTCAGGCC.

Constructions of Plasmid.

We constructed three plasmids Tom22-T1, Tom22-T2, and Tom22-T3 by adding TEV protease site (GAAAACCTGTATTTTCAGAGC) to the joints of helices H1-H2, H2-H3, and H3-H4, respectively, by PCR. Plasmid was conducted by the PerfectStart Green PCR SuperMix (Trans). The primer sequences were as follows:Tom22-H1, F: GAAAACCTGTATTTTCAGAGCGGCGACGCGGAGAAGCCTTom22-H1, R: GCTCTGAAAATACAGGTTTTCTTTCGGGAGCAATTCGTCTom22-H2, F: GAAAACCTGTATTTTCAGAGCACCCTGTCGGAGAGACTATom22-H2, R: GCTCTGAAAATACAGGTTTTCTCATCTAGCTCCTCATCTom22-H3, F: GAAAACCTGTATTTTCAGAGCXXGGAGAGGGTCCGGTCCTom22-H3, R: GCTCTGAAAATACAGGTTTTAAACATCTCCGTCAGGCC.

Tom22-H1, F: GAAAACCTGTATTTTCAGAGCGGCGACGCGGAGAAGCCT

Tom22-H1, R: GCTCTGAAAATACAGGTTTTCTTTCGGGAGCAATTCGTC

Tom22-H2, F: GAAAACCTGTATTTTCAGAGCACCCTGTCGGAGAGACTA

Tom22-H2, R: GCTCTGAAAATACAGGTTTTCTCATCTAGCTCCTCATC

Tom22-H3, F: GAAAACCTGTATTTTCAGAGCXXGGAGAGGGTCCGGTCC

Tom22-H3, R: GCTCTGAAAATACAGGTTTTAAACATCTCCGTCAGGCC.

Tom22, Tom20, and Su9-TEV-DHFR-EGFP Purification.For purification of Tom20 and Tom22, wild-type or indicated variants, cell debris was collected during mitochondria isolation (described above) and 1% digitonin was added to extract protein. The suspension was centrifuged at 150,000 × g for 30 min at 4 °C and then the supernatant was incubated with Flag-affinity beads (Sigma) for 1 h at 4 °C. The resin was washed in a gravity column by washing buffer (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 0.01% digitonin) with 10 column volumes. Proteins were eluted with Flag peptides (Genscript) and concentrated to 100 μL using a 20-kDa cutoff centrifugal filter (Millipore) and analyzed by SDS-PAGE. Truncated Tom22 proteins were obtained by treating with TEV protease to remove different parts. All proteins were further purified by gel filtration with a Superose 200 Increase 3.2/300 GL column (GE Healthcare) and the fractions were collected.For Su9-TEV-DHFR-EGFP, the 293F cells were collected and extracted by 1% digitonin. After centrifugation, supernatant was incubated with Strep-affinity beads (Sigma). Su9-TEV-DHFR-EGFP was eluted with buffer (20 mM MOPS pH 7.4, 150 mM KCI) containing 5 mM desthiobiotin (Sigma). Part of the protein was treated with TEV protease to remove Su9. All proteins were further purified by gel filtration with a Superose 200 Increase 3.2/300 GL column (GE Healthcare) and the fractions were collected.

Tom22, Tom20, and Su9-TEV-DHFR-EGFP Purification.

For purification of Tom20 and Tom22, wild-type or indicated variants, cell debris was collected during mitochondria isolation (described above) and 1% digitonin was added to extract protein. The suspension was centrifuged at 150,000 × g for 30 min at 4 °C and then the supernatant was incubated with Flag-affinity beads (Sigma) for 1 h at 4 °C. The resin was washed in a gravity column by washing buffer (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 0.01% digitonin) with 10 column volumes. Proteins were eluted with Flag peptides (Genscript) and concentrated to 100 μL using a 20-kDa cutoff centrifugal filter (Millipore) and analyzed by SDS-PAGE. Truncated Tom22 proteins were obtained by treating with TEV protease to remove different parts. All proteins were further purified by gel filtration with a Superose 200 Increase 3.2/300 GL column (GE Healthcare) and the fractions were collected.

For Su9-TEV-DHFR-EGFP, the 293F cells were collected and extracted by 1% digitonin. After centrifugation, supernatant was incubated with Strep-affinity beads (Sigma). Su9-TEV-DHFR-EGFP was eluted with buffer (20 mM MOPS pH 7.4, 150 mM KCI) containing 5 mM desthiobiotin (Sigma). Part of the protein was treated with TEV protease to remove Su9. All proteins were further purified by gel filtration with a Superose 200 Increase 3.2/300 GL column (GE Healthcare) and the fractions were collected.

Pull-Down.Su9-TEV-DHFR-EGFP was incubated with Tom20 and Tom22 for 1 h at room temperature, respectively. Then Strep-affinity beads (Sigma) were added to the mixture for 1 h at 4 °C. After incubation, the resin was washed in a gravity column by washing buffer (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 0.01% digitonin) with 10 column volumes. All proteins were eluted with washing buffer containing 5 mM desthiobiotin (Sigma). The elution was collected and analyzed by Western blot. DHFR-EGFP was used as a negative control to perform the same experiment described above.Tom22, wild-type or indicated variants, were incubated with Su9-TEV-DHFR-EGFP for 1 h at room temperature, respectively. Then Flag-affinity beads (Sigma) were added to the mixture for 1 h at 4 °C. After incubation, the resin was washed in a gravity column by washing buffer (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 0.01% digitonin) with 10 column volumes. All proteins were eluted with washing buffer containing Flag peptides (Genscript). The elution was collected and analyzed by Western blot.

Su9-TEV-DHFR-EGFP was incubated with Tom20 and Tom22 for 1 h at room temperature, respectively. Then Strep-affinity beads (Sigma) were added to the mixture for 1 h at 4 °C. After incubation, the resin was washed in a gravity column by washing buffer (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 0.01% digitonin) with 10 column volumes. All proteins were eluted with washing buffer containing 5 mM desthiobiotin (Sigma). The elution was collected and analyzed by Western blot. DHFR-EGFP was used as a negative control to perform the same experiment described above.

Tom22, wild-type or indicated variants, were incubated with Su9-TEV-DHFR-EGFP for 1 h at room temperature, respectively. Then Flag-affinity beads (Sigma) were added to the mixture for 1 h at 4 °C. After incubation, the resin was washed in a gravity column by washing buffer (20 mM MOPS pH 7.4, 150 mM KCI, 10% [vol/vol] glycerol, 0.01% digitonin) with 10 column volumes. All proteins were eluted with washing buffer containing Flag peptides (Genscript). The elution was collected and analyzed by Western blot.

Western Blotting.The samples were electrotransferred to polyvinylidene fluoride membranes (Millipore) at 90 V for 1 h at 4 °C after SDS-PAGE. The membranes were incubated with methyl alcohol for 3 min and washed for 3 min with the transfer buffer (25 mM Tris, 192 mM glycine). Then, the membranes were blocked by 5% nonfat milk at room temperature for 1 h. After blocking, the membranes were incubated with primary antibodies, anti-Flag antibodies (Proteintech Company) for Tom20 and Tom22, or anti-Strep antibodies (Proteintech) for Su9-TEV-DHFR-EGFP and DHFR-EGFP, at room temperature for 1 h with a 2,000× dilution. Then the membranes were washed by TBS-T buffer (150 mM NaCl, 10 mM Tris, 10% Tween-20). The secondary antibody, anti-mouse antibody (Proteintech), was incubated with the membranes at room temperature for 1 h with a 10,000× dilution. After thorough washing, the protein bands were visualized by Western Quick Block Kit (GenScript) on Chemidoc MP (BIO-RAD).

Western Blotting.

The samples were electrotransferred to polyvinylidene fluoride membranes (Millipore) at 90 V for 1 h at 4 °C after SDS-PAGE. The membranes were incubated with methyl alcohol for 3 min and washed for 3 min with the transfer buffer (25 mM Tris, 192 mM glycine). Then, the membranes were blocked by 5% nonfat milk at room temperature for 1 h. After blocking, the membranes were incubated with primary antibodies, anti-Flag antibodies (Proteintech Company) for Tom20 and Tom22, or anti-Strep antibodies (Proteintech) for Su9-TEV-DHFR-EGFP and DHFR-EGFP, at room temperature for 1 h with a 2,000× dilution. Then the membranes were washed by TBS-T buffer (150 mM NaCl, 10 mM Tris, 10% Tween-20). The secondary antibody, anti-mouse antibody (Proteintech), was incubated with the membranes at room temperature for 1 h with a 10,000× dilution. After thorough washing, the protein bands were visualized by Western Quick Block Kit (GenScript) on Chemidoc MP (BIO-RAD).

Supplementary MaterialSupplementary File

Supplementary Material
