---
pmid: '21478870'
title: A diverse range of gene products are effectors of the type I interferon antiviral
  response.
authors:
- Schoggins JW
- Wilson SJ
- Panis M
- Murphy MY
- Jones CT
- Bieniasz P
- Rice CM
journal: Nature
year: '2011'
full_text_available: true
full_text_extraction_method: xml
pmcid: PMC3409588
doi: 10.1038/nature09907
---

# A diverse range of gene products are effectors of the type I interferon antiviral response.
**Authors:** Schoggins JW, Wilson SJ, Panis M, Murphy MY, Jones CT, Bieniasz P, Rice CM
**Journal:** Nature (2011)
**DOI:** [10.1038/nature09907](https://doi.org/10.1038/nature09907)
**PMC:** [PMC3409588](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3409588/)

## Abstract

1. Nature. 2011 Apr 28;472(7344):481-5. doi: 10.1038/nature09907. Epub 2011 Apr
10.

A diverse range of gene products are effectors of the type I interferon 
antiviral response.

Schoggins JW(1), Wilson SJ, Panis M, Murphy MY, Jones CT, Bieniasz P, Rice CM.

Author information:
(1)Laboratory of Virology and Infectious Disease, Center for the Study of 
Hepatitis C, The Rockefeller University, New York, New York 10065, USA.

Erratum in
    Nature. 2015 Sep 3;525(7567):144. doi: 10.1038/nature14554.

Comment in
    J Hepatol. 2012 Mar;56(3):726-8. doi: 10.1016/j.jhep.2011.07.009.
    Gastroenterology. 2012 Jan;142(1):178-80. doi: 10.1053/j.gastro.2011.11.008.

The type I interferon response protects cells against invading viral pathogens. 
The cellular factors that mediate this defence are the products of 
interferon-stimulated genes (ISGs). Although hundreds of ISGs have been 
identified since their discovery more than 25 years ago, only a few have been 
characterized with respect to antiviral activity. For most ISG products, little 
is known about their antiviral potential, their target specificity and their 
mechanisms of action. Using an overexpression screening approach, here we show 
that different viruses are targeted by unique sets of ISGs. We find that each 
viral species is susceptible to multiple antiviral genes, which together 
encompass a range of inhibitory activities. To conduct the screen, more than 380 
human ISGs were tested for their ability to inhibit the replication of several 
important human and animal viruses, including hepatitis C virus, yellow fever 
virus, West Nile virus, chikungunya virus, Venezuelan equine encephalitis virus 
and human immunodeficiency virus type-1. Broadly acting effectors included IRF1, 
C6orf150 (also known as MB21D1), HPSE, RIG-I (also known as DDX58), MDA5 (also 
known as IFIH1) and IFITM3, whereas more targeted antiviral specificity was 
observed with DDX60, IFI44L, IFI6, IFITM2, MAP3K14, MOV10, NAMPT (also known as 
PBEF1), OASL, RTP4, TREX1 and UNC84B (also known as SUN2). Combined expression 
of pairs of ISGs showed additive antiviral effects similar to those of moderate 
type I interferon doses. Mechanistic studies uncovered a common theme of 
translational inhibition for numerous effectors. Several ISGs, including ADAR, 
FAM46C, LY6E and MCOLN2, enhanced the replication of certain viruses, 
highlighting another layer of complexity in the highly pleiotropic type I 
interferon system.

DOI: 10.1038/nature09907
PMCID: PMC3409588
PMID: 21478870 [Indexed for MEDLINE]

## Full Text

To identify novel antiviral IFN effectors, we developed a cell-based assay to determine the effects of hundreds of ISG products on viral replication. The assay relied on a bicistronic lentiviral vector co-expressing an ISG and the red fluorescent protein, TagRFP (Fig. 1a and Supplementary Fig. 1). ISG/TagRFP-expressing cells were challenged with a green fluorescent protein (GFP)-expressing virus and replication within the TagRFP-positive population was quantified by flow cytometry (FACS) (Fig. 1b). In pilot experiments, eleven ISGs (PSMB8, PSMB9, IFI6, GBP1, IFITM3, MX1, PLSCR1, RTP4, CCDC75, IFIT3, IFI27) and two control genes, Gaussia (Gluc) and Firefly (Fluc) luciferase, were tested for inhibition of vesicular stomatitis virus (VSV) and YFV in STAT 1-deficient human fibroblasts (STAT1−/−Fib) or in human hepatoma cells (Huh-7). STAT1−/−Fib produce but do not respond to IFN4, while Huh-7 respond to but are not robust producers of IFN5. Inhibition was assessed quantitatively by FACS (Fig. 1c) and qualitatively by fluorescence microscopy (Supplementary Fig. 1c). IFI6 and MX1 showed specificity for YFV and VSV, respectively. (Supplementary Fig. 1d). Both viruses were inhibited by IFITM3, which also suppresses influenza A virus, WNV, and dengue virus6. These results demonstrate the utility of this screening platform and suggest its potential to reveal antiviral ISGs targeting divergent virus families.

To conduct a large-scale antiviral ISG screen, we analyzed published microarray datasets from IFN-treated cells or tissues and established inclusion criteria1,7,8,9,10,11,12,13,14,15 (Supplementary Table 1). We compiled a list of 389 ISGs, inserted each into the bicistronic lentiviral vector, and produced high-titer lentiviral stocks (Supplementary Methods). Approximately 50 of these lentiviral stocks failed to mediate high-level transduction in target cells (Supplementary Fig. 2a), perhaps resulting from ISG-mediated toxicity, auto-downregulation of IRES-driven TagRFP expression, poor packaging of the vector genome, or activity against the HIV-1 based vector. We consider the latter possibility likely since several known antiretroviral genes (EIF2AK2/PKR, APOBEC3G, BST2/tetherin, MOV10) were included in this subset16,17,18. The potential cytotoxicity of ISG overexpression was inferred from FACS-based cell counts (Supplementary Fig. 3a) and validated for a number of genes by luminescence-based cell viability assay (Supplementary Fig. 3b). While two genes were toxic (CDKN1A and TNFSFR10A), the vast majority of ISGs did not strongly impact cell viability.

The lentivirus-expressed ISG collection was screened for inhibition of HCV expressing Ypet GFP (Supplementary Fig. 4) in Huh-7.5, a RIG-I defective derivative of Huh-7 cells, at 48 h and 72 h post-infection. Data are represented as a normal distribution (Fig. 1d) or dot plots (Fig. 1e). Most genes affected HCV infection to some extent, with the mean (Z-score = 0) level of inhibition at 20%. Four genes (MDA5, RIG-I, IRF1, IRF7) had Z-scores below −3.0, while 53 genes had Z-scores between −1.0 and −3.0 (Supplementary Tables 2 and 3). At 72 h post-infection, HCV replication approaches a maximum (Supplementary Fig. 4d) and the overall ISG profile constricted (Fig. 1e), likely reflecting the ability of the virus to overcome inhibition. Transient transfection was used to screen the subset of ISGs that were poorly expressed via lentiviral transduction, and several genes (including DDX60, MOV10, and MS4A4A) were found to have anti-HCV activity (Supplementary Fig. 5). To investigate the impact of the host cell environment on ISG activity, we rescreened HCV in a less-permissive cell type (Huh-7)19. The strongest effectors were consistently active in the two cell lines (Fig. 1e and Fig. 2a), while several additional genes (including MAP3K14 and SLC1A1) inhibited HCV replication in Huh-7 cells (Z-score below −2.0, Supplementary Table 4). Taken together, these data suggest at least two categories of anti-HCV ISGs: i) strong inhibitors that likely feed back into IFN or other signaling pathways and ii) modest inhibitors that may have targeted effector functions.

We also screened the ISG collection with other medically important viruses from three different families, Flaviviridae (YFV, WNV), Togaviridae (CHIKV, VEEV), and Retroviridae (HIV-1) (Fig. 2a). We assayed YFV, WNV, VEEV, and CHIKV in STAT1−/−Fib and HIV-1 in MT-4 cells (an HTLV-1-transformed T cell line). Global ISG inhibition profiles were similar for all viruses, with 10-25 genes exhibiting Z scores of −1.5 or less (Fig. 2a and Supplementary Tables 7-10). YFV was also screened in Huh-7 cells (Supplementary Fig. 6a) and this gave fewer hits, with five genes showing a Z-score below −1.5 (Supplementary Tables 5 and 6). The strongest hits in Huh-7 cells (HPSE, IRF1, IRF7, IFITM3, IFI6) were also among the top inhibitors in STAT1−/−Fib, suggesting these effectors function independently of cell type. Differences in data spread between the two cell lines may be due to higher levels of YFV replication in Huh-7 cells (Supplementary Fig. 6b) and/or increased ISG expression in STAT1−/−Fib (Supplementary Fig. 2b), and indicate that STAT1−/−Fib provide a more sensitive environment for detecting ISG-mediated inhibition.

The strongest antiviral ISGs from each primary screen were validated by generating independent lentiviral stocks and measuring inhibition from at least eight replicates (Fig. 3b). Forty-five hits were confirmed to significantly reduce virus replication compared to a control (Fluc) (p<0.001 in 44 assays and p<0.05 in 1 assay), and five genes were false positives. Further validation was conducted for a subset of ISGs. Inhibition of YFV by IFI6, IFITM3, and RTP4 was dose-dependent (Supplementary Fig. 7a), and both IFITM3 and IFI6 inhibited production of the parental, non-GFP virus (YFV-17D) (Supplementary Fig. 7b, c). We also measured the effects of anti-HCV ISGs when they were introduced after HCV infection. In contrast to the 20-70% range of inhibition observed when ISGs were expressed prior to infection (Fig 2b), ISG expression after infection impacted replication by either less than 20% (DDIT4, NT5C3, IFI44L, MAP3K14, IRF2) or greater than 80% (IRF7, IRF1, MDA5, RIG-I). Only one gene, OASL, showed an intermediate phentoype (Supplementary Fig 8). These data demonstrate that the overexpression platform is highly reliable for identifying antiviral ISGs and sensitive enough to detect even modest levels of inhibition in both pre- and post-infection scenarios.

In the primary screens, several genes enhanced viral replication (Fig 2a). Six of these (ADAR, APOBEC3A, FAM46C, IDO1, LY6E, and MCOLN2) were confirmed with YFV, WNV, VEEV, and CHIKV in STAT1−/−Fib (Fig 2d). For most genes, we found an increase in the number of YFV or VEEV-infected cells, whereas WNV and CHIKV were largely unaffected. Consistent with the primary YFV screen, LY6E conferred a striking 75% enhancement in infection frequency (Fig 2d and Supplementary Fig 9), a phenotype that may correlate with studies implicating LY6E in susceptibility of chickens to Marek’s disease virus20 and mice to adenovirus21. Analysis of mean GFP intensity in infected cells revealed that ADAR significantly enhanced replication of all viruses (Supplementary Fig 9). Thus, in contrast to LY6E, which caused an increase in the number of infected cells, ADAR affected the level of replication within the cell. ADAR encodes an RNA-specific adenosine deaminase that has been reported to stimulate HIV-1 infection by RNA editing22. ADAR also promotes measles virus infection23, supporting a pan-viral mechanism of enhancement.

The screening and validation data revealed both broad-acting and specific effectors. IRF1, a transcription factor in the IFN pathway, inhibited all viruses tested (Fig. 2c), even in a STAT1−/− background. Given that over 200 IRF1 binding sites reside in the human genome24, our data suggest that IRF1 activates a unique antiviral program. Indeed, microarray analysis of IRF1-transduced Huh-7 or STAT1−/−Fib revealed induction of numerous ISGs (Supplementary Fig 10), but no IFN genes were induced (data not shown). The majority of genes induced greater than 3-fold (p<0.05) in Huh-7 were similarly upregulated in STAT1−/−Fib, and both cells lines had a subset of uniquely regulated transcripts. Several of our antiviral hits were among the IRF1-induced transcripts, supporting the observation that IRF1 effector mechanisms partially overlap with IFN, but do not require its action. Genes C6orf150, HPSE, RIG-I, MDA5, IFITM3, IRF7, and NAMPT/PBEF1 were also active against more than one virus species, while other genes showed more restricted antiviral specificities. Gene ontology analysis was performed to classify confirmed antiviral ISGs. Compared to the entire ISG collection, the validated list showed a statistically significant overrepresentation of three molecular functions (nucleic acid binding, hydrolase, and helicase activity) and three biological processes (signal transduction, transcription initiation, and small molecule transport) (Supplementary Fig 11 and Supplementary Table 11).

The identification of modest to strong inhibitors suggests that an effective IFN response requires the combinatorial action of numerous ISGs. This hypothesis has been inferred from microarray1 and knockout studies25, but experimental data showing combinatorial ISG function is limited. To detemine whether ISG effects were greater when expressed in combination, selected hits from the HCV, HIV-1, and YFV screens were tested in all two-gene combinations (Fig 3a and Supplementary Fig 12). ISG effects were generally additive, with some combinations reducing replication greater than 90%, an efficacy similar to moderate doses of IFN (Fig 3b, c, d, e). For YFV, when one inhibitory and one enhancing ISG were tested together (e.g. C6orf150+MCOLN2), the magnitude of viral replication was more strongly influenced by the inhibitory gene.

To probe antiviral mechanisms of action, select ISGs were tested for inhibition at various life cycle stages. Using HCV pseudoparticles (HCVpp), none of the eight anti-HCV ISGs significantly impaired viral entry (Supplementary Fig 13a). In contrast, analysis of subgenomic HCV replicons expressing Gluc revealed that all ISGs tested inhibited primary translation 25-70 % at 4 h post-transfection (Fig 4a, b and Supplementary Fig 13b, c). Notably, the strongest translational inhibitors were also the most potent suppressors of replication, as measured by Gluc production at 72 h (Fig 4c) and confirmed by quanitfying HCV genome copy number (Supplementary Fig 13d). The finding that eight genes with a range of predicted molecular functions (Supplementary Table 11) each block initial translation suggests the importance of this host strategy for HCV suppression. IRF1 and RIG-I were also associated with a kinetic delay in the onset of replication, suggesting multiple blocks in the HCV life cycle.

We also investigated the mechanism of C6orf150, which encodes a putative nucleotidyltransferase26 and contains a mab-21 domain, but is otherwise uncharacterized. Since C6orf150 inhibited alphaviruses CHIKV and VEEV, we tested it against the related Sindbis virus (SINV). C6orf150 inhibited SINV-GFP replication (Fig 4d) and SINV (non-GFP) infectious virus production (Fig 4e). We next assessed the activity of C6orf150 against a Fluc-expressing temperature-sensitive virus, SINV(ts6), which is competent for primary translation but does not replicate at the nonpermissive temperature (see Supporting Methods). SINV(ts6) was susceptible to C6orf150 inhibition (Fig 4f), indicating an early viral translation block. Interestingly, the pan-viral inhibitor IRF1 also blocked SINV(ts6) translation, suggesting this effector mechanism is reinforced by IFN-induced transcriptional factors. Similar to the HCV life cycle studies, the importance of translational inhibition is highlighted by a corresponding reduction in replication of wild type SINV-Luc (Fig 4g).

The antiviral effects of type I IFN were first described over 50 years ago27. Since then, great strides into the mechanisms governing virus-mediated IFN production, IFN gene regulation, and IFN signaling have been made28,29. Insight into downstream IFN effector mechanisms, however, have largely been limited to a select group of ISGs, likely due to difficulties involved in systematically overexpressing hundreds of genes. With the strategy presented here, we have overcome these technical barriers and identified multiple novel antiviral ISGs, many of which target early steps in the viral life cycle. Combined with well-known effectors such as PKR and IFIT proteins, life-cycle characterization of these newly identified ISGs suggests that the IFN system may use multiple strategies to engender a single outcome such as translational inhibition. Clinically, IFN is used to treat several viral infections and is a principle component of HCV treatment. Given the clinical side effects of IFN, and the natural ability of viruses to thwart its activity30, exploiting the actions of individual ISGs may be a preferable therapeutic strategy. Further insight into the biochemical mechanisms of these effectors may provide a platform for future studies on the development of alternatives to IFN-based therapies.

MethodsViruses, replicons, and cellsHuh-7, Huh-7.5, and 293T cells were maintained in DMEM (Invitrogen) with 10% FBS and 0.1mM non-essential amino acids. STAT1−/−Fib (an SV40 large T antigen immortalized skin fibroblast line) and MT-4 (an HTLV-1 immortalized T cell line) were grown in RPMI (Invitrogen) with 10% FBS. BHK-J cells were grown in MEM (Invitrogen) with 7.5% FBS. Infectious HCV-Ypet31 (derived from Bi-Ypet-Jc1FLAG2), YFV-Venus31 (derived from YF17D-5′C25Venus2AUbi), WNV-GFP32 (derived from pBELO-WNV-GFP-RZ ic) and CHIKV-GFP33 (derived from pCHIKV-LR 5′GFP), SINV/SINV-GFP34 (derived from pS300/pS300-GFP), SINV-Fluc and the temperature sensitive variant SINV(ts6)Fluc were derived from pToto1101/Luc and pToto11-1/Luc:ts6. SINV(ts6)Fluc carries mutations that prevent replication at the nonpermissive temperature, but the virus retains the ability to translate incoming genomes35. VEEV-GFP is a double subgenomic EGFP reporter virus derived from the TC83 vaccine strain of Venezuelan equine encephalitis (kindly provided by I. Frolov).HCV subgenomic replicon RNAs were derived from Bi-Gluc-JFH(SG) and Bi-Gluc-JFH-GNN(SG), which are bicistronic constructs expressing Gluc from the HCV IRES and genotype 2a JFH-1 nonstructural proteins from the EMCV IRES. The GNN variant encodes polymerase mutations and is not competent for replication. Viral stocks were generated by electroporation of in vitro transcribed RNAs into Huh-7.5 (HCV) or BHK-J (YFV, WNV, VEEV, CHIKV, SINV) as previously described32,33,35, 36,37,38. Single cycle HIV-1-GFP was generated by transfection of HEK 293T cells with a modified env- proviral plasmid encoding GFP in place of Nef (pR7/Δenv/GFP)39. HIV particles were pseudotyped with VSV-G glycoprotein. Lentiviral pseudoparticles with no envelope or carrying HCV glycoprotein (HCVpp) were generated and assayed for HCV entry as previously described40. Experiments with WNV, CHIKV, and HIV-1 were carried out in biosafety level 3 containment in compliance with institutional and federal guidelines.DNA constructsTo generate a lentiviral-based, Gateway-compatible destination vector for ISG expression, a Gateway expression cassette (containing the tetracyline-inducible-hybrid CMV promoter, chloramphenicol resistance gene, and ccdB suicide gene) from pLenti4.TO.V5-DEST (Invitrogen) was subcloned into the XhoI-NdeI sites of pTRIP-EGFP41. Overlap extension PCR followed by a three-fragment ligation was used to insert the EMCV IRES and TagRFP (Evrogen) downstream of the Gateway module(EMCV IRES 5′ oligo:5′-GATATCTCGAGGCCCCTCTCCCTCCCCCCCCCCTAA-3′,EMCV IRES 3′ oligo:5′-CACGATGATAATATGGCCACAACCCCGCGGATATG-3′,TagRFP 5′ oligo:5′-CATAGCTAGCATGGTGTCTAAGGGCGAAGAGCTG,TagRFP 3′ oligo:5′-CTAGCAAACTGGGGCACAAACTTAATTGACCGCGGGGTACCTGCG-3′.To enhance gene expression, the β-globin intron (IVSβ) was subcloned from pLenti6-TR (Invitrogen) into the SpeI site between the CMV promoter and the Gateway cassette(IVSβ 5′ oligo:5′-GACCCACTAGTGTGAGTTTGGGGACCCTTGATTG-3′,IVSβ 3′ oligo:5′-CATGCCTTCTTCTTTTTCCTACAGACTAGTCCCAG-3′).All DEST vector variants were grown in DB3.1 cells (Invitrogen) under ampicillin-chloramphenicol double selection. The final vector was named pTRIP.CMV.IVSb.ires.TagRFP-DEST.Sequence-validated, Gateway-compatible ORFEXPRESS shuttle clones corresponding to 387 ISGs were obtained from Genecopoeia. Two additional ISG entry clones encoding RSAD2 and ZC3HAV1 and two control clones encoding Firefly luciferase (Fluc) or Gaussia luciferase (Gluc) were generated as follows. Genes encoding RSAD2, ZC3HAV1/ZAP, Fluc, and Gluc were PCR-amplified with oligos containing attB sites flanking gene-specific sequences(RSAD2 5′ oligo:5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACCATGTGGGTGCTTACACCTGC-3′,RSAD2 3′ oligo:5′-GGGGACCACTTTGTACAAGAAAGCTGGGTCTACCAATCCAGCTTCAGAT-3′,ZCHAV1 5′ oligo:5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACCATGGCGGACCCGGAG GTGTG-5′,ZCHAV1 3′ oligo:5′-GGGGACCACTTTGTACAAGAAAGCTGGGTTTACTCTGGCCCTCTCTTCATCT-3′,Fluc 5′ oligo:5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACCATGGAAGATGCCAAAAACATTAAGAA-3′,Fluc 3′ oligo:5′-GGGGACCACTTTGTACAAGAAAGCTGGGTTTACACGGCGATCTTGCCGCCCTTC-3′,Gluc 5′ oligo:5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACCATGGGAGTCAAAGTTCTGTTTGCCC-3′,Gluc 3′ oligo:5′-GGGGACCACTTTGTACAAGAAAGCTGGGTTTAGTCACCACCGGCCCCCTTGATC-3′.PCR products were purified over GFX columns (GE Healthcare) and cloned into pDONR (Invitrogen) with BP Clonase. BP reactions were transformed into OMNIMax competent E. coli (Invitrogen) and colonies were screened by restriction digest and sequencing.The ISG-encoding sequences from pENTR clones were moved into pTRIP.CMV.IVSb.ires.TagRFP-DEST using LR Clonase II (Invitrogen) according to manufacturer’s instructions. LR reaction products were transformed into MDS42Rec reduced genome E. coli. (Scarab Genomics). One or two colonies for each clone were grown in 3 ml Luria broth (LB) plus ampicillin and transfection-quality plasmid DNA was purified over anion-exchange columns (Qiagen). All pTRIP.CMV.IVSb.ISG.ires.TagRFP constructs were sequenced at the 5′ end of the expression cassette to verify gene insertion (sequencing oligo: 5′-CCTGCCTTTCTCTTTATGG-3′).Generation of ISG-expressing lentiviral pseudoparticlesLentiviral pseudoparticles were generated by co-transfecting 4×105 293T cells in 6 well plates or 1.2×104 cells in 96 well plates with plasmids expressing: 1) the pTRIP.CMV.IVSb.ISG.ires.TagRFP proviral DNA, 2) HIV gag-pol, and 3) the vesicular stomatitis virus glycoprotein (VSV-G) in a ratio of 1:0.8:0.2, respectively. For each transfection, 6 μl Fugene (Roche) was combined with 2.0 μg total DNA in 100 μl Optimem (Gibco). Transfections were carried out for 6 h followed by a media change to DMEM with 3%FBS. Supernatants were harvested at 48 h and 72 h, pooled, cleared by centrifugation, and stored at −80°C.Lentiviral transduction and viral replication assaysTransduction assaysHuh-7, Huh-7.5, STAT1−/−Fib were seeded into 24 well plates at a density of 7 ×104 cells/well and transduced with lentiviral pseudoparticles by spinoculation at 1000-1500 x g for 1 h at 37°C in media containing 3% FBS, 20 mM Hepes, and 4 μg/ml polybrene. For HIV studies, suspension MT-4 cells were transduced in a 96 well format. For monolayer experiments with Huh-7, Huh-7.5, STAT1−/−Fib, cells were split 1:2 or 1:3 at 48 h post-transduction.Replication assaysTansduced cells were infected with the indicated virus at a dose that gave approximately 50% infected (GFP+) cells, which had been previously determined by FACS-based infectivity assays. Infected cells were harvested at various time points, within the first viral replication cycle when possible: HCV (48 h or 72 h), HIV (48 h), YFV (24 h), WNV (6 h), VEEV (5.5 h), CHIKV (10 h). Adherent cells were harvested into 200 μl Accumax cell dissociation medium (eBioscience) and transferred to a 96 well plate. Cells were pelleted at 2,000 RPM for 5 min at 4°C and resuspended in 1% paraformaldehyde fixation solution for at least 1 h. Cells were pelleted by centrifugation at 2,000 RPM for 5 min at 4°C, resuspended in cold 1X PBS+3% FBS, and stored at 4°C until FACS analysis. Samples were analyzed in a 96 well-based high throughput manner using the LSRII/HTS flow cytometer (BD Biosciences) equipped with a 561nm laser for detection of TagRFP. Data was analyzed using Flowjo software (Treestar) with a 0.1% compensation matrix.For interferon titrations, Huh-7 or MT-4 T cells were treated with varying doses of IFNβ (PBL InterferonSource) or 1000 U/mL IFNα (Sigma), respectively, for 24 h prior to infection. Cells were infected with HCV (Huh-7) or HIV-1 (MT-4) and replication levels were monitored by FACS as described above.For SINV-Fluc and SINV(ts6)-Fluc infection, cells were infected at 28°C (ts6) or 37°C (wild type) and harvested 4 h post-infection. Intracellular Fluc levels were assessed by the Luciferase Assay System reporter assay (Promega) according to the manufacturer’s instructions.For HCV life cycle studies, subgenomic RNAs were generated by in vitro transcription using the RiboMax T7 transcription kit (Promega). 175ng RNA were transfected into 3.5 × 104 ISG-expressing Huh-7 cells with Mirus Reagent (Mirus Bio). Translation and replication were monitored by sampling cell supernatants and assaying for Gluc production over time (at 2, 4, 6, 8, 13.5, 24, 48, and 72 h post-transfection) with the Renilla Luciferase Assay System (Promega), or by quantifying HCV genome copy number by reverse transcription (RT)-PCR as previously described42.DNA transfection-based ISG screen in of Huh-7 cellsHuh-7 cells were seeded into 24 well plates at a density of 7×104 cells/well. The next day, growth media was changed to DMEM with 1% FBS and 0.1mM non-essential amino acids. Lentiviral plasmid DNAs were transfected at 400 ng DNA/well using Lipofectamine 2000 (Invitrogen) in a total volume of 1 ml/well. Plates were centrifuged at 1000 x g for 30 min at 37°C. Five hours later, media was changed to DMEM with 10% FBS and two days post-transfection, cells were challenged with HCV-Ypet and assayed for replication by FACS as described above.ImmunofluorescenceSTAT1−/−Fib expressing IRF1.ires.TagRFP, C6orf150.ires.TagRFP, or MAP3K14.ires.TagRFP were stained for protein levels using Cellomics Whole Cell Stain Kit (Thermo Scientific) according to the manufacturer’s instructions. Affinity-purified polyclonal rabbit antisera detecting human IRF1, MAP3K14 (alias: NIK), and C6orf150 were obtained from Abcam or Sigma and used at manufacturer’s recommended concentrations. An AlexaFluor-488 conjugated goat anti-rabbit antibody (Invitrogen) was used at a concentration of 1:1000 for detection, and samples were visualized by epifluorescence microscopy on an Eclipse TE300 (Nikon) inverted microscope.Microarray analysisHuh-7 and STAT1−/−Fib were transduced with lentiviruses expressing Fluc.ires.TagRFP or IRF1.ires.TagRFP. Total RNA was harvested from transduced cells 48 h later using Qiagen RNeasy Mini Kit. RNA amplification was carried out with MessageAmp™ Premier RNA Amplification Kit (Applied Biosystems). 200 ng of total RNA was used to synthesize the first strand of cDNA using ArrayScript reverse transcriptase and an oligo(dT) primer bearing a T7 promoter. The single-stranded cDNA was then converted into a double-stranded DNA (dsDNA) by DNA polymerase I in the presence of E. coli RNase H and DNA ligase. The dsDNA served as a template for in vitro transcription in a reaction containing biotin-labeled UTP, unlabeled NTPs and T7 RNA Polymerase. The amplified, biotin-labeled antisense RNA (aRNA) was purified and quality was assessed using the Agilent 2100 Bioanalyzer and the RNA 6000 Nano kit. 750 ng of aRNA in 5 μl was mixed with 10 μl of hybridization reagents and heated at 65°C for 10min. After cooling to room temperature, 15 μl of the hybridization solution was applied to an Illumina HumanHT-12 v4 chip. The chip was incubated for about 18 hours at 58°C. After washing and staining with streptavidin-Cy3, the chip was scanned using Illumina BeadArray Reader. The scanning was done using standard DirectHyb Gene Expression protocol with the following settings: Factor=1, PMT=587, Filter=100%. The raw data was extracted using Illumina BeadStudio software without normalization. Data was analyzed using GeneSpring Software with global normalization. Students unpaired t-test was used to determine statistical significance.

Viruses, replicons, and cellsHuh-7, Huh-7.5, and 293T cells were maintained in DMEM (Invitrogen) with 10% FBS and 0.1mM non-essential amino acids. STAT1−/−Fib (an SV40 large T antigen immortalized skin fibroblast line) and MT-4 (an HTLV-1 immortalized T cell line) were grown in RPMI (Invitrogen) with 10% FBS. BHK-J cells were grown in MEM (Invitrogen) with 7.5% FBS. Infectious HCV-Ypet31 (derived from Bi-Ypet-Jc1FLAG2), YFV-Venus31 (derived from YF17D-5′C25Venus2AUbi), WNV-GFP32 (derived from pBELO-WNV-GFP-RZ ic) and CHIKV-GFP33 (derived from pCHIKV-LR 5′GFP), SINV/SINV-GFP34 (derived from pS300/pS300-GFP), SINV-Fluc and the temperature sensitive variant SINV(ts6)Fluc were derived from pToto1101/Luc and pToto11-1/Luc:ts6. SINV(ts6)Fluc carries mutations that prevent replication at the nonpermissive temperature, but the virus retains the ability to translate incoming genomes35. VEEV-GFP is a double subgenomic EGFP reporter virus derived from the TC83 vaccine strain of Venezuelan equine encephalitis (kindly provided by I. Frolov).HCV subgenomic replicon RNAs were derived from Bi-Gluc-JFH(SG) and Bi-Gluc-JFH-GNN(SG), which are bicistronic constructs expressing Gluc from the HCV IRES and genotype 2a JFH-1 nonstructural proteins from the EMCV IRES. The GNN variant encodes polymerase mutations and is not competent for replication. Viral stocks were generated by electroporation of in vitro transcribed RNAs into Huh-7.5 (HCV) or BHK-J (YFV, WNV, VEEV, CHIKV, SINV) as previously described32,33,35, 36,37,38. Single cycle HIV-1-GFP was generated by transfection of HEK 293T cells with a modified env- proviral plasmid encoding GFP in place of Nef (pR7/Δenv/GFP)39. HIV particles were pseudotyped with VSV-G glycoprotein. Lentiviral pseudoparticles with no envelope or carrying HCV glycoprotein (HCVpp) were generated and assayed for HCV entry as previously described40. Experiments with WNV, CHIKV, and HIV-1 were carried out in biosafety level 3 containment in compliance with institutional and federal guidelines.

Viruses, replicons, and cells

Huh-7, Huh-7.5, and 293T cells were maintained in DMEM (Invitrogen) with 10% FBS and 0.1mM non-essential amino acids. STAT1−/−Fib (an SV40 large T antigen immortalized skin fibroblast line) and MT-4 (an HTLV-1 immortalized T cell line) were grown in RPMI (Invitrogen) with 10% FBS. BHK-J cells were grown in MEM (Invitrogen) with 7.5% FBS. Infectious HCV-Ypet31 (derived from Bi-Ypet-Jc1FLAG2), YFV-Venus31 (derived from YF17D-5′C25Venus2AUbi), WNV-GFP32 (derived from pBELO-WNV-GFP-RZ ic) and CHIKV-GFP33 (derived from pCHIKV-LR 5′GFP), SINV/SINV-GFP34 (derived from pS300/pS300-GFP), SINV-Fluc and the temperature sensitive variant SINV(ts6)Fluc were derived from pToto1101/Luc and pToto11-1/Luc:ts6. SINV(ts6)Fluc carries mutations that prevent replication at the nonpermissive temperature, but the virus retains the ability to translate incoming genomes35. VEEV-GFP is a double subgenomic EGFP reporter virus derived from the TC83 vaccine strain of Venezuelan equine encephalitis (kindly provided by I. Frolov).

HCV subgenomic replicon RNAs were derived from Bi-Gluc-JFH(SG) and Bi-Gluc-JFH-GNN(SG), which are bicistronic constructs expressing Gluc from the HCV IRES and genotype 2a JFH-1 nonstructural proteins from the EMCV IRES. The GNN variant encodes polymerase mutations and is not competent for replication. Viral stocks were generated by electroporation of in vitro transcribed RNAs into Huh-7.5 (HCV) or BHK-J (YFV, WNV, VEEV, CHIKV, SINV) as previously described32,33,35, 36,37,38. Single cycle HIV-1-GFP was generated by transfection of HEK 293T cells with a modified env- proviral plasmid encoding GFP in place of Nef (pR7/Δenv/GFP)39. HIV particles were pseudotyped with VSV-G glycoprotein. Lentiviral pseudoparticles with no envelope or carrying HCV glycoprotein (HCVpp) were generated and assayed for HCV entry as previously described40. Experiments with WNV, CHIKV, and HIV-1 were carried out in biosafety level 3 containment in compliance with institutional and federal guidelines.

DNA constructsTo generate a lentiviral-based, Gateway-compatible destination vector for ISG expression, a Gateway expression cassette (containing the tetracyline-inducible-hybrid CMV promoter, chloramphenicol resistance gene, and ccdB suicide gene) from pLenti4.TO.V5-DEST (Invitrogen) was subcloned into the XhoI-NdeI sites of pTRIP-EGFP41. Overlap extension PCR followed by a three-fragment ligation was used to insert the EMCV IRES and TagRFP (Evrogen) downstream of the Gateway module(EMCV IRES 5′ oligo:5′-GATATCTCGAGGCCCCTCTCCCTCCCCCCCCCCTAA-3′,EMCV IRES 3′ oligo:5′-CACGATGATAATATGGCCACAACCCCGCGGATATG-3′,TagRFP 5′ oligo:5′-CATAGCTAGCATGGTGTCTAAGGGCGAAGAGCTG,TagRFP 3′ oligo:5′-CTAGCAAACTGGGGCACAAACTTAATTGACCGCGGGGTACCTGCG-3′.To enhance gene expression, the β-globin intron (IVSβ) was subcloned from pLenti6-TR (Invitrogen) into the SpeI site between the CMV promoter and the Gateway cassette(IVSβ 5′ oligo:5′-GACCCACTAGTGTGAGTTTGGGGACCCTTGATTG-3′,IVSβ 3′ oligo:5′-CATGCCTTCTTCTTTTTCCTACAGACTAGTCCCAG-3′).All DEST vector variants were grown in DB3.1 cells (Invitrogen) under ampicillin-chloramphenicol double selection. The final vector was named pTRIP.CMV.IVSb.ires.TagRFP-DEST.Sequence-validated, Gateway-compatible ORFEXPRESS shuttle clones corresponding to 387 ISGs were obtained from Genecopoeia. Two additional ISG entry clones encoding RSAD2 and ZC3HAV1 and two control clones encoding Firefly luciferase (Fluc) or Gaussia luciferase (Gluc) were generated as follows. Genes encoding RSAD2, ZC3HAV1/ZAP, Fluc, and Gluc were PCR-amplified with oligos containing attB sites flanking gene-specific sequences(RSAD2 5′ oligo:5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACCATGTGGGTGCTTACACCTGC-3′,RSAD2 3′ oligo:5′-GGGGACCACTTTGTACAAGAAAGCTGGGTCTACCAATCCAGCTTCAGAT-3′,ZCHAV1 5′ oligo:5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACCATGGCGGACCCGGAG GTGTG-5′,ZCHAV1 3′ oligo:5′-GGGGACCACTTTGTACAAGAAAGCTGGGTTTACTCTGGCCCTCTCTTCATCT-3′,Fluc 5′ oligo:5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACCATGGAAGATGCCAAAAACATTAAGAA-3′,Fluc 3′ oligo:5′-GGGGACCACTTTGTACAAGAAAGCTGGGTTTACACGGCGATCTTGCCGCCCTTC-3′,Gluc 5′ oligo:5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACCATGGGAGTCAAAGTTCTGTTTGCCC-3′,Gluc 3′ oligo:5′-GGGGACCACTTTGTACAAGAAAGCTGGGTTTAGTCACCACCGGCCCCCTTGATC-3′.PCR products were purified over GFX columns (GE Healthcare) and cloned into pDONR (Invitrogen) with BP Clonase. BP reactions were transformed into OMNIMax competent E. coli (Invitrogen) and colonies were screened by restriction digest and sequencing.The ISG-encoding sequences from pENTR clones were moved into pTRIP.CMV.IVSb.ires.TagRFP-DEST using LR Clonase II (Invitrogen) according to manufacturer’s instructions. LR reaction products were transformed into MDS42Rec reduced genome E. coli. (Scarab Genomics). One or two colonies for each clone were grown in 3 ml Luria broth (LB) plus ampicillin and transfection-quality plasmid DNA was purified over anion-exchange columns (Qiagen). All pTRIP.CMV.IVSb.ISG.ires.TagRFP constructs were sequenced at the 5′ end of the expression cassette to verify gene insertion (sequencing oligo: 5′-CCTGCCTTTCTCTTTATGG-3′).

DNA constructs

To generate a lentiviral-based, Gateway-compatible destination vector for ISG expression, a Gateway expression cassette (containing the tetracyline-inducible-hybrid CMV promoter, chloramphenicol resistance gene, and ccdB suicide gene) from pLenti4.TO.V5-DEST (Invitrogen) was subcloned into the XhoI-NdeI sites of pTRIP-EGFP41. Overlap extension PCR followed by a three-fragment ligation was used to insert the EMCV IRES and TagRFP (Evrogen) downstream of the Gateway module

(EMCV IRES 5′ oligo:

5′-GATATCTCGAGGCCCCTCTCCCTCCCCCCCCCCTAA-3′,

EMCV IRES 3′ oligo:

5′-CACGATGATAATATGGCCACAACCCCGCGGATATG-3′,

TagRFP 5′ oligo:

5′-CATAGCTAGCATGGTGTCTAAGGGCGAAGAGCTG,

TagRFP 3′ oligo:

5′-CTAGCAAACTGGGGCACAAACTTAATTGACCGCGGGGTACCTGCG-3′.

To enhance gene expression, the β-globin intron (IVSβ) was subcloned from pLenti6-TR (Invitrogen) into the SpeI site between the CMV promoter and the Gateway cassette

(IVSβ 5′ oligo:

5′-GACCCACTAGTGTGAGTTTGGGGACCCTTGATTG-3′,

IVSβ 3′ oligo:

5′-CATGCCTTCTTCTTTTTCCTACAGACTAGTCCCAG-3′).

All DEST vector variants were grown in DB3.1 cells (Invitrogen) under ampicillin-chloramphenicol double selection. The final vector was named pTRIP.CMV.IVSb.ires.TagRFP-DEST.

Sequence-validated, Gateway-compatible ORFEXPRESS shuttle clones corresponding to 387 ISGs were obtained from Genecopoeia. Two additional ISG entry clones encoding RSAD2 and ZC3HAV1 and two control clones encoding Firefly luciferase (Fluc) or Gaussia luciferase (Gluc) were generated as follows. Genes encoding RSAD2, ZC3HAV1/ZAP, Fluc, and Gluc were PCR-amplified with oligos containing attB sites flanking gene-specific sequences

(RSAD2 5′ oligo:

5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACCATGTGGGTGCTTACACCTGC-3′,

RSAD2 3′ oligo:

5′-GGGGACCACTTTGTACAAGAAAGCTGGGTCTACCAATCCAGCTTCAGAT-3′,

ZCHAV1 5′ oligo:

5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACCATGGCGGACCCGGAG GTGTG-5′,

ZCHAV1 3′ oligo:

5′-GGGGACCACTTTGTACAAGAAAGCTGGGTTTACTCTGGCCCTCTCTTCATCT-3′,

Fluc 5′ oligo:

5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACCATGGAAGATGCCAAAAACATTAAGAA-3′,

Fluc 3′ oligo:

5′-GGGGACCACTTTGTACAAGAAAGCTGGGTTTACACGGCGATCTTGCCGCCCTTC-3′,

Gluc 5′ oligo:

5′-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACCATGGGAGTCAAAGTTCTGTTTGCCC-3′,

Gluc 3′ oligo:

5′-GGGGACCACTTTGTACAAGAAAGCTGGGTTTAGTCACCACCGGCCCCCTTGATC-3′.

PCR products were purified over GFX columns (GE Healthcare) and cloned into pDONR (Invitrogen) with BP Clonase. BP reactions were transformed into OMNIMax competent E. coli (Invitrogen) and colonies were screened by restriction digest and sequencing.

The ISG-encoding sequences from pENTR clones were moved into pTRIP.CMV.IVSb.ires.TagRFP-DEST using LR Clonase II (Invitrogen) according to manufacturer’s instructions. LR reaction products were transformed into MDS42Rec reduced genome E. coli. (Scarab Genomics). One or two colonies for each clone were grown in 3 ml Luria broth (LB) plus ampicillin and transfection-quality plasmid DNA was purified over anion-exchange columns (Qiagen). All pTRIP.CMV.IVSb.ISG.ires.TagRFP constructs were sequenced at the 5′ end of the expression cassette to verify gene insertion (sequencing oligo: 5′-CCTGCCTTTCTCTTTATGG-3′).

Generation of ISG-expressing lentiviral pseudoparticlesLentiviral pseudoparticles were generated by co-transfecting 4×105 293T cells in 6 well plates or 1.2×104 cells in 96 well plates with plasmids expressing: 1) the pTRIP.CMV.IVSb.ISG.ires.TagRFP proviral DNA, 2) HIV gag-pol, and 3) the vesicular stomatitis virus glycoprotein (VSV-G) in a ratio of 1:0.8:0.2, respectively. For each transfection, 6 μl Fugene (Roche) was combined with 2.0 μg total DNA in 100 μl Optimem (Gibco). Transfections were carried out for 6 h followed by a media change to DMEM with 3%FBS. Supernatants were harvested at 48 h and 72 h, pooled, cleared by centrifugation, and stored at −80°C.

Generation of ISG-expressing lentiviral pseudoparticles

Lentiviral pseudoparticles were generated by co-transfecting 4×105 293T cells in 6 well plates or 1.2×104 cells in 96 well plates with plasmids expressing: 1) the pTRIP.CMV.IVSb.ISG.ires.TagRFP proviral DNA, 2) HIV gag-pol, and 3) the vesicular stomatitis virus glycoprotein (VSV-G) in a ratio of 1:0.8:0.2, respectively. For each transfection, 6 μl Fugene (Roche) was combined with 2.0 μg total DNA in 100 μl Optimem (Gibco). Transfections were carried out for 6 h followed by a media change to DMEM with 3%FBS. Supernatants were harvested at 48 h and 72 h, pooled, cleared by centrifugation, and stored at −80°C.

Lentiviral transduction and viral replication assaysTransduction assaysHuh-7, Huh-7.5, STAT1−/−Fib were seeded into 24 well plates at a density of 7 ×104 cells/well and transduced with lentiviral pseudoparticles by spinoculation at 1000-1500 x g for 1 h at 37°C in media containing 3% FBS, 20 mM Hepes, and 4 μg/ml polybrene. For HIV studies, suspension MT-4 cells were transduced in a 96 well format. For monolayer experiments with Huh-7, Huh-7.5, STAT1−/−Fib, cells were split 1:2 or 1:3 at 48 h post-transduction.Replication assaysTansduced cells were infected with the indicated virus at a dose that gave approximately 50% infected (GFP+) cells, which had been previously determined by FACS-based infectivity assays. Infected cells were harvested at various time points, within the first viral replication cycle when possible: HCV (48 h or 72 h), HIV (48 h), YFV (24 h), WNV (6 h), VEEV (5.5 h), CHIKV (10 h). Adherent cells were harvested into 200 μl Accumax cell dissociation medium (eBioscience) and transferred to a 96 well plate. Cells were pelleted at 2,000 RPM for 5 min at 4°C and resuspended in 1% paraformaldehyde fixation solution for at least 1 h. Cells were pelleted by centrifugation at 2,000 RPM for 5 min at 4°C, resuspended in cold 1X PBS+3% FBS, and stored at 4°C until FACS analysis. Samples were analyzed in a 96 well-based high throughput manner using the LSRII/HTS flow cytometer (BD Biosciences) equipped with a 561nm laser for detection of TagRFP. Data was analyzed using Flowjo software (Treestar) with a 0.1% compensation matrix.For interferon titrations, Huh-7 or MT-4 T cells were treated with varying doses of IFNβ (PBL InterferonSource) or 1000 U/mL IFNα (Sigma), respectively, for 24 h prior to infection. Cells were infected with HCV (Huh-7) or HIV-1 (MT-4) and replication levels were monitored by FACS as described above.For SINV-Fluc and SINV(ts6)-Fluc infection, cells were infected at 28°C (ts6) or 37°C (wild type) and harvested 4 h post-infection. Intracellular Fluc levels were assessed by the Luciferase Assay System reporter assay (Promega) according to the manufacturer’s instructions.For HCV life cycle studies, subgenomic RNAs were generated by in vitro transcription using the RiboMax T7 transcription kit (Promega). 175ng RNA were transfected into 3.5 × 104 ISG-expressing Huh-7 cells with Mirus Reagent (Mirus Bio). Translation and replication were monitored by sampling cell supernatants and assaying for Gluc production over time (at 2, 4, 6, 8, 13.5, 24, 48, and 72 h post-transfection) with the Renilla Luciferase Assay System (Promega), or by quantifying HCV genome copy number by reverse transcription (RT)-PCR as previously described42.

Lentiviral transduction and viral replication assays

Transduction assaysHuh-7, Huh-7.5, STAT1−/−Fib were seeded into 24 well plates at a density of 7 ×104 cells/well and transduced with lentiviral pseudoparticles by spinoculation at 1000-1500 x g for 1 h at 37°C in media containing 3% FBS, 20 mM Hepes, and 4 μg/ml polybrene. For HIV studies, suspension MT-4 cells were transduced in a 96 well format. For monolayer experiments with Huh-7, Huh-7.5, STAT1−/−Fib, cells were split 1:2 or 1:3 at 48 h post-transduction.

Transduction assays

Huh-7, Huh-7.5, STAT1−/−Fib were seeded into 24 well plates at a density of 7 ×104 cells/well and transduced with lentiviral pseudoparticles by spinoculation at 1000-1500 x g for 1 h at 37°C in media containing 3% FBS, 20 mM Hepes, and 4 μg/ml polybrene. For HIV studies, suspension MT-4 cells were transduced in a 96 well format. For monolayer experiments with Huh-7, Huh-7.5, STAT1−/−Fib, cells were split 1:2 or 1:3 at 48 h post-transduction.

Replication assaysTansduced cells were infected with the indicated virus at a dose that gave approximately 50% infected (GFP+) cells, which had been previously determined by FACS-based infectivity assays. Infected cells were harvested at various time points, within the first viral replication cycle when possible: HCV (48 h or 72 h), HIV (48 h), YFV (24 h), WNV (6 h), VEEV (5.5 h), CHIKV (10 h). Adherent cells were harvested into 200 μl Accumax cell dissociation medium (eBioscience) and transferred to a 96 well plate. Cells were pelleted at 2,000 RPM for 5 min at 4°C and resuspended in 1% paraformaldehyde fixation solution for at least 1 h. Cells were pelleted by centrifugation at 2,000 RPM for 5 min at 4°C, resuspended in cold 1X PBS+3% FBS, and stored at 4°C until FACS analysis. Samples were analyzed in a 96 well-based high throughput manner using the LSRII/HTS flow cytometer (BD Biosciences) equipped with a 561nm laser for detection of TagRFP. Data was analyzed using Flowjo software (Treestar) with a 0.1% compensation matrix.For interferon titrations, Huh-7 or MT-4 T cells were treated with varying doses of IFNβ (PBL InterferonSource) or 1000 U/mL IFNα (Sigma), respectively, for 24 h prior to infection. Cells were infected with HCV (Huh-7) or HIV-1 (MT-4) and replication levels were monitored by FACS as described above.For SINV-Fluc and SINV(ts6)-Fluc infection, cells were infected at 28°C (ts6) or 37°C (wild type) and harvested 4 h post-infection. Intracellular Fluc levels were assessed by the Luciferase Assay System reporter assay (Promega) according to the manufacturer’s instructions.For HCV life cycle studies, subgenomic RNAs were generated by in vitro transcription using the RiboMax T7 transcription kit (Promega). 175ng RNA were transfected into 3.5 × 104 ISG-expressing Huh-7 cells with Mirus Reagent (Mirus Bio). Translation and replication were monitored by sampling cell supernatants and assaying for Gluc production over time (at 2, 4, 6, 8, 13.5, 24, 48, and 72 h post-transfection) with the Renilla Luciferase Assay System (Promega), or by quantifying HCV genome copy number by reverse transcription (RT)-PCR as previously described42.

Replication assays

Tansduced cells were infected with the indicated virus at a dose that gave approximately 50% infected (GFP+) cells, which had been previously determined by FACS-based infectivity assays. Infected cells were harvested at various time points, within the first viral replication cycle when possible: HCV (48 h or 72 h), HIV (48 h), YFV (24 h), WNV (6 h), VEEV (5.5 h), CHIKV (10 h). Adherent cells were harvested into 200 μl Accumax cell dissociation medium (eBioscience) and transferred to a 96 well plate. Cells were pelleted at 2,000 RPM for 5 min at 4°C and resuspended in 1% paraformaldehyde fixation solution for at least 1 h. Cells were pelleted by centrifugation at 2,000 RPM for 5 min at 4°C, resuspended in cold 1X PBS+3% FBS, and stored at 4°C until FACS analysis. Samples were analyzed in a 96 well-based high throughput manner using the LSRII/HTS flow cytometer (BD Biosciences) equipped with a 561nm laser for detection of TagRFP. Data was analyzed using Flowjo software (Treestar) with a 0.1% compensation matrix.

For interferon titrations, Huh-7 or MT-4 T cells were treated with varying doses of IFNβ (PBL InterferonSource) or 1000 U/mL IFNα (Sigma), respectively, for 24 h prior to infection. Cells were infected with HCV (Huh-7) or HIV-1 (MT-4) and replication levels were monitored by FACS as described above.

For SINV-Fluc and SINV(ts6)-Fluc infection, cells were infected at 28°C (ts6) or 37°C (wild type) and harvested 4 h post-infection. Intracellular Fluc levels were assessed by the Luciferase Assay System reporter assay (Promega) according to the manufacturer’s instructions.

For HCV life cycle studies, subgenomic RNAs were generated by in vitro transcription using the RiboMax T7 transcription kit (Promega). 175ng RNA were transfected into 3.5 × 104 ISG-expressing Huh-7 cells with Mirus Reagent (Mirus Bio). Translation and replication were monitored by sampling cell supernatants and assaying for Gluc production over time (at 2, 4, 6, 8, 13.5, 24, 48, and 72 h post-transfection) with the Renilla Luciferase Assay System (Promega), or by quantifying HCV genome copy number by reverse transcription (RT)-PCR as previously described42.

DNA transfection-based ISG screen in of Huh-7 cellsHuh-7 cells were seeded into 24 well plates at a density of 7×104 cells/well. The next day, growth media was changed to DMEM with 1% FBS and 0.1mM non-essential amino acids. Lentiviral plasmid DNAs were transfected at 400 ng DNA/well using Lipofectamine 2000 (Invitrogen) in a total volume of 1 ml/well. Plates were centrifuged at 1000 x g for 30 min at 37°C. Five hours later, media was changed to DMEM with 10% FBS and two days post-transfection, cells were challenged with HCV-Ypet and assayed for replication by FACS as described above.

DNA transfection-based ISG screen in of Huh-7 cells

Huh-7 cells were seeded into 24 well plates at a density of 7×104 cells/well. The next day, growth media was changed to DMEM with 1% FBS and 0.1mM non-essential amino acids. Lentiviral plasmid DNAs were transfected at 400 ng DNA/well using Lipofectamine 2000 (Invitrogen) in a total volume of 1 ml/well. Plates were centrifuged at 1000 x g for 30 min at 37°C. Five hours later, media was changed to DMEM with 10% FBS and two days post-transfection, cells were challenged with HCV-Ypet and assayed for replication by FACS as described above.

ImmunofluorescenceSTAT1−/−Fib expressing IRF1.ires.TagRFP, C6orf150.ires.TagRFP, or MAP3K14.ires.TagRFP were stained for protein levels using Cellomics Whole Cell Stain Kit (Thermo Scientific) according to the manufacturer’s instructions. Affinity-purified polyclonal rabbit antisera detecting human IRF1, MAP3K14 (alias: NIK), and C6orf150 were obtained from Abcam or Sigma and used at manufacturer’s recommended concentrations. An AlexaFluor-488 conjugated goat anti-rabbit antibody (Invitrogen) was used at a concentration of 1:1000 for detection, and samples were visualized by epifluorescence microscopy on an Eclipse TE300 (Nikon) inverted microscope.

Immunofluorescence

STAT1−/−Fib expressing IRF1.ires.TagRFP, C6orf150.ires.TagRFP, or MAP3K14.ires.TagRFP were stained for protein levels using Cellomics Whole Cell Stain Kit (Thermo Scientific) according to the manufacturer’s instructions. Affinity-purified polyclonal rabbit antisera detecting human IRF1, MAP3K14 (alias: NIK), and C6orf150 were obtained from Abcam or Sigma and used at manufacturer’s recommended concentrations. An AlexaFluor-488 conjugated goat anti-rabbit antibody (Invitrogen) was used at a concentration of 1:1000 for detection, and samples were visualized by epifluorescence microscopy on an Eclipse TE300 (Nikon) inverted microscope.

Microarray analysisHuh-7 and STAT1−/−Fib were transduced with lentiviruses expressing Fluc.ires.TagRFP or IRF1.ires.TagRFP. Total RNA was harvested from transduced cells 48 h later using Qiagen RNeasy Mini Kit. RNA amplification was carried out with MessageAmp™ Premier RNA Amplification Kit (Applied Biosystems). 200 ng of total RNA was used to synthesize the first strand of cDNA using ArrayScript reverse transcriptase and an oligo(dT) primer bearing a T7 promoter. The single-stranded cDNA was then converted into a double-stranded DNA (dsDNA) by DNA polymerase I in the presence of E. coli RNase H and DNA ligase. The dsDNA served as a template for in vitro transcription in a reaction containing biotin-labeled UTP, unlabeled NTPs and T7 RNA Polymerase. The amplified, biotin-labeled antisense RNA (aRNA) was purified and quality was assessed using the Agilent 2100 Bioanalyzer and the RNA 6000 Nano kit. 750 ng of aRNA in 5 μl was mixed with 10 μl of hybridization reagents and heated at 65°C for 10min. After cooling to room temperature, 15 μl of the hybridization solution was applied to an Illumina HumanHT-12 v4 chip. The chip was incubated for about 18 hours at 58°C. After washing and staining with streptavidin-Cy3, the chip was scanned using Illumina BeadArray Reader. The scanning was done using standard DirectHyb Gene Expression protocol with the following settings: Factor=1, PMT=587, Filter=100%. The raw data was extracted using Illumina BeadStudio software without normalization. Data was analyzed using GeneSpring Software with global normalization. Students unpaired t-test was used to determine statistical significance.

Microarray analysis

Huh-7 and STAT1−/−Fib were transduced with lentiviruses expressing Fluc.ires.TagRFP or IRF1.ires.TagRFP. Total RNA was harvested from transduced cells 48 h later using Qiagen RNeasy Mini Kit. RNA amplification was carried out with MessageAmp™ Premier RNA Amplification Kit (Applied Biosystems). 200 ng of total RNA was used to synthesize the first strand of cDNA using ArrayScript reverse transcriptase and an oligo(dT) primer bearing a T7 promoter. The single-stranded cDNA was then converted into a double-stranded DNA (dsDNA) by DNA polymerase I in the presence of E. coli RNase H and DNA ligase. The dsDNA served as a template for in vitro transcription in a reaction containing biotin-labeled UTP, unlabeled NTPs and T7 RNA Polymerase. The amplified, biotin-labeled antisense RNA (aRNA) was purified and quality was assessed using the Agilent 2100 Bioanalyzer and the RNA 6000 Nano kit. 750 ng of aRNA in 5 μl was mixed with 10 μl of hybridization reagents and heated at 65°C for 10min. After cooling to room temperature, 15 μl of the hybridization solution was applied to an Illumina HumanHT-12 v4 chip. The chip was incubated for about 18 hours at 58°C. After washing and staining with streptavidin-Cy3, the chip was scanned using Illumina BeadArray Reader. The scanning was done using standard DirectHyb Gene Expression protocol with the following settings: Factor=1, PMT=587, Filter=100%. The raw data was extracted using Illumina BeadStudio software without normalization. Data was analyzed using GeneSpring Software with global normalization. Students unpaired t-test was used to determine statistical significance.

Supplementary Material14

Supplementary Material
