---
provider: falcon
model: Edison Scientific Literature
cached: false
start_time: '2025-11-25T19:31:38.892514'
end_time: '2025-11-25T19:39:02.251529'
duration_seconds: 443.36
template_file: templates/rule_research.md
template_variables:
  rule_id: ARBA00026849
  rule_type: arba
  go_terms: GO:0016810 (hydrolase activity, acting on carbon-nitrogen (but not peptide)
    bonds)
  conditions_summary: '**Condition Set 1:** InterPro id: Adenosine deaminase domain
    (InterPro:IPR001365) AND InterPro id: Adenosine/adenine deaminase (InterPro:IPR006330)
    AND InterPro id: Adenosine/AMP deaminase active site (InterPro:IPR006650)


    **Condition Set 2:** InterPro id: Carbon-nitrogen hydrolase (InterPro:IPR003010)
    AND InterPro id: Carbon-nitrogen hydrolase superfamily (InterPro:IPR036526) AND
    taxon: Viridiplantae (NCBITaxon:33090)


    **Condition Set 3:** InterPro id: Cytidine deaminase-like (InterPro:IPR016193)
    AND taxon: Saccharomycotina (NCBITaxon:147537)


    **Condition Set 4:** InterPro id: Asparaginase/glutaminase-like (InterPro:IPR006034)
    AND InterPro id: Asparaginase/glutaminase, active site 1 (InterPro:IPR020827)
    AND InterPro id: L-asparaginase, C-terminal (InterPro:IPR027473)


    **Condition Set 5:** InterPro id: Nitrilase/cyanide hydratase, conserved site
    (InterPro:IPR000132) AND InterPro id: Nitrilase/Cyanide hydratase (InterPro:IPR044149)


    **Condition Set 6:** InterPro id: GTP cyclohydrolase I (InterPro:IPR001474) AND
    InterPro id: GTP cyclohydrolase I, conserved site (InterPro:IPR018234) AND taxon:
    Metazoa (NCBITaxon:33208)


    **Condition Set 7:** InterPro id: Cytidine and deoxycytidylate deaminase domain
    (InterPro:IPR002125) AND InterPro id: APOBEC/CMP deaminase, zinc-binding (InterPro:IPR016192)
    AND taxon: Fungi (NCBITaxon:4751)


    **Condition Set 8:** InterPro id: Peptidase M20 (InterPro:IPR002933) AND InterPro
    id: Amidase, carbamoylase-type (InterPro:IPR010158) AND PANTHER id: PTHR32494:SF19


    **Condition Set 9:** InterPro id: Ureohydrolase (InterPro:IPR006035) AND InterPro
    id: Ureohydrolase domain superfamily (InterPro:IPR023696) AND taxon: Bacteria
    (NCBITaxon:2)


    **Condition Set 10:** InterPro id: Biotinidase-like, eukaryotic (InterPro:IPR012101)
    AND InterPro id: Biotinidase/VNN family (InterPro:IPR040154) AND InterPro id:
    Vanin, C-terminal (InterPro:IPR043957)


    **Condition Set 11:** InterPro id: Cytidine and Deoxycytidylate Deaminase (InterPro:IPR050202)
    AND PANTHER id: PTHR11644:SF2


    **Condition Set 12:** PANTHER id: PTHR43114:SF6


    **Condition Set 13:** InterPro id: Peptidase T2, asparaginase 2 (InterPro:IPR000246)
    AND InterPro id: Nucleophile aminohydrolases, N-terminal (InterPro:IPR029055)
    AND PANTHER id: PTHR10188:SF6


    **Condition Set 14:** InterPro id: Beta-lactamase, class-B, conserved site (InterPro:IPR001018)
    AND InterPro id: Metallo-beta-lactamase (InterPro:IPR001279) AND taxon: Pseudomonadati
    (NCBITaxon:3379134)


    **Condition Set 15:** InterPro id: NodB homology domain (InterPro:IPR002509) AND
    InterPro id: Polysaccharide deacetylase, ArnD subfamily (InterPro:IPR050248) AND
    taxon: Eukaryota (NCBITaxon:2759)


    **Condition Set 16:** InterPro id: UDP-3-O-acyl N-acetylglucosamine deacetylase
    (InterPro:IPR004463) AND InterPro id: UDP-3-O-acyl N-acetylglucosamine deacetylase,
    C-terminal (InterPro:IPR011334) AND taxon: Streptophyta (NCBITaxon:35493)


    **Condition Set 17:** InterPro id: Amidohydrolase-related (InterPro:IPR006680)
    AND InterPro id: Metal-dependent hydrolase, composite domain superfamily (InterPro:IPR011059)
    AND InterPro id: Deaminase MtaD/DadD (InterPro:IPR023512)


    **Condition Set 18:** InterPro id: Kynurenine formamidase/cyclase-like (InterPro:IPR007325)
    AND InterPro id: Kynurenine formamidase, bacteria (InterPro:IPR017484) AND InterPro
    id: Kynurenine formamidase superfamily (InterPro:IPR037175)


    **Condition Set 19:** InterPro id: Arginase (InterPro:IPR014033) AND InterPro
    id: Ureohydrolase, manganese-binding site (InterPro:IPR020855) AND PANTHER id:
    PTHR43782:SF3


    **Condition Set 20:** InterPro id: Amidase, conserved site (InterPro:IPR020556)
    AND InterPro id: Amidase signature domain (InterPro:IPR023631) AND taxon: Tracheophyta
    (NCBITaxon:58023)


    **Condition Set 21:** InterPro id: Agmatinase-related (InterPro:IPR005925) AND
    PANTHER id: PTHR11358:SF26


    **Condition Set 22:** InterPro id: AMP deaminase (InterPro:IPR006329) AND taxon:
    Glires (NCBITaxon:314147)


    **Condition Set 23:** InterPro id: Peptide deformylase (InterPro:IPR023635) AND
    InterPro id: Peptide deformylase superfamily (InterPro:IPR036821)


    **Condition Set 24:** InterPro id: Aliphatic Amidase/Beta-Ureidopropionase (InterPro:IPR050345)
    AND PANTHER id: PTHR43674:SF2


    **Condition Set 25:** FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:1.10.286.10:FF:000003)
    AND FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000012) AND taxon:
    Metazoa (NCBITaxon:33208)


    **Condition Set 26:** FunFam id: biotinidase isoform X1 (CATH.FunFam:3.60.110.10:FF:000001)
    AND taxon: Eukaryota (NCBITaxon:2759)


    **Condition Set 27:** FunFam id: Bifunctional nitrilase/nitrile hydratase NIT4B
    (CATH.FunFam:3.60.110.10:FF:000006) AND taxon: Viridiplantae (NCBITaxon:33090)


    **Condition Set 28:** FunFam id: L-asparaginase 2 (CATH.FunFam:3.40.50.1170:FF:000001)


    **Condition Set 29:** FunFam id: Allantoate amidohydrolase (CATH.FunFam:3.40.630.10:FF:000044)


    **Condition Set 30:** FunFam id: Probable UDP-3-O-acyl-N-acetylglucosamine deacetylase
    3, mitochondrial (CATH.FunFam:3.30.1700.10:FF:000002) AND FunFam id: Probable
    UDP-3-O-acyl-N-acetylglucosamine deacetylase 2, mitochondrial (CATH.FunFam:3.30.230.20:FF:000002)
    AND taxon: Streptophyta (NCBITaxon:35493)


    **Condition Set 31:** FunFam id: Kynurenine formamidase (CATH.FunFam:3.50.30.50:FF:000001)
    AND taxon: Bacteria (NCBITaxon:2)


    **Condition Set 32:** FunFam id: AMP deaminase (CATH.FunFam:4.10.800.20:FF:000001)
    AND taxon: Glires (NCBITaxon:314147)


    **Condition Set 33:** FunFam id: 5-methylthioadenosine/S-adenosylhomocysteine
    deaminase (CATH.FunFam:3.20.20.140:FF:000014)


    **Condition Set 34:** FunFam id: Bifunctional purine biosynthesis protein PURH
    (CATH.FunFam:1.10.287.440:FF:000001) AND FunFam id: Bifunctional purine biosynthesis
    protein (CATH.FunFam:3.40.140.20:FF:000003) AND taxon: Dikarya (NCBITaxon:451864)


    **Condition Set 35:** FunFam id: Protein-arginine deiminase type-2 (CATH.FunFam:2.60.40.1700:FF:000001)
    AND FunFam id: Protein-arginine deiminase type-2 (CATH.FunFam:3.75.10.10:FF:000003)
    AND taxon: Rattus (NCBITaxon:10114)


    **Condition Set 36:** FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:3.30.1490.20:FF:000001)
    AND FunFam id: CAD protein-like isoform X1 (CATH.FunFam:3.40.50.1370:FF:000005)
    AND FunFam id: Carbamoyl-phosphate synthase 1, mitochondrial (CATH.FunFam:3.40.50.880:FF:000006)


    **Condition Set 37:** FunFam id: Adenosine deaminase RNA-specific B2 (inactive)
    (CATH.FunFam:3.30.160.20:FF:000009) AND FunFam id: double-stranded RNA-specific
    editase 1 isoform X1 (CATH.FunFam:3.30.160.20:FF:000011) AND taxon: Chordata (NCBITaxon:7711)


    **Condition Set 38:** FunFam id: N-acylsphingosine amidohydrolase 2 (CATH.FunFam:2.60.40.2300:FF:000001)
    AND taxon: Craniata (NCBITaxon:89593)


    **Condition Set 39:** FunFam id: Adenosine deaminase (CATH.FunFam:3.20.20.140:FF:000038)
    AND taxon: Vertebrata (NCBITaxon:7742)


    **Condition Set 40:** FunFam id: bifunctional heparan sulfate N-deacetylase/N-sulfotransferase
    1 (CATH.FunFam:3.40.50.300:FF:000176) AND taxon: Mus (NCBITaxon:862507)


    **Condition Set 41:** FunFam id: Arginase 1, mitochondrial (CATH.FunFam:3.40.800.10:FF:000007)
    AND taxon: Embryophyta (NCBITaxon:3193)


    **Condition Set 42:** FunFam id: Dihydropyrimidinase like 3 (CATH.FunFam:3.20.20.140:FF:000001)


    **Condition Set 43:** FunFam id: Cytidine deaminase (CATH.FunFam:3.40.140.10:FF:000008)


    **Condition Set 44:** FunFam id: Aminoacylase-1, putative (CATH.FunFam:1.10.150.900:FF:000001)
    AND FunFam id: Putative Aminoacylase-1 (CATH.FunFam:3.30.70.360:FF:000005) AND
    FunFam id: Aminoacylase 1 (CATH.FunFam:3.40.630.10:FF:000019)


    **Condition Set 45:** FunFam id: Putative formate--tetrahydrofolate ligase (CATH.FunFam:3.10.410.10:FF:000001)
    AND FunFam id: C-1-tetrahydrofolate synthase, cytoplasmic, putative (CATH.FunFam:3.40.50.10860:FF:000005)
    AND taxon: Ascomycota (NCBITaxon:4890)


    **Condition Set 46:** FunFam id: NAD-dependent protein deacetylase sirtuin-1 (CATH.FunFam:3.30.1600.10:FF:000013)
    AND FunFam id: NAD-dependent deacetylase sirtuin-2 (CATH.FunFam:3.40.50.1220:FF:000005)
    AND taxon: Euteleostomi (NCBITaxon:117571)


    **Condition Set 47:** FunFam id: Jasmonate-regulated gene 21 (CATH.FunFam:2.60.120.330:FF:000008)
    AND taxon: Tracheophyta (NCBITaxon:58023)


    **Condition Set 48:** FunFam id: Apolipoprotein B editing enzyme catalytic polypeptide-like
    3H (CATH.FunFam:3.40.140.10:FF:000047) AND taxon: Primates (NCBITaxon:9443)


    **Condition Set 49:** FunFam id: Protein-arginine deiminase type-4 (CATH.FunFam:2.60.40.1860:FF:000003)


    **Condition Set 50:** FunFam id: N-acetylglucosamine-6-phosphate deacetylase (CATH.FunFam:3.20.20.140:FF:000004)


    **Condition Set 51:** FunFam id: Guanine deaminase (CATH.FunFam:3.20.20.140:FF:000022)


    **Condition Set 52:** FunFam id: Arginase (CATH.FunFam:3.40.800.10:FF:000005)


    **Condition Set 53:** FunFam id: Beta-ureidopropionase 1 (CATH.FunFam:3.60.110.10:FF:000009)


    **Condition Set 54:** FunFam id: N(4)-(Beta-N-acetylglucosaminyl)-L-asparaginase
    isoform X1 (CATH.FunFam:3.60.20.30:FF:000003)


    **Condition Set 55:** FunFam id: Methylenetetrahydrofolate dehydrogenase (NADP+
    dependent) 1 like (CATH.FunFam:1.10.8.770:FF:000001) AND FunFam id: C-1-tetrahydrofolate
    synthase, cytoplasmic (CATH.FunFam:3.40.50.300:FF:000245) AND FunFam id: C-1-tetrahydrofolate
    synthase, cytoplasmic isoform X2 (CATH.FunFam:3.40.50.300:FF:001123)


    **Condition Set 56:** FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000007)
    AND FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000008) AND taxon:
    Spermatophyta (NCBITaxon:58024)


    **Condition Set 57:** FunFam id: Bifunctional protein FolD (CATH.FunFam:3.40.50.10860:FF:000001)
    AND FunFam id: probable bifunctional methylenetetrahydrofolate dehydrogenase/cyclohydrolase
    2 (CATH.FunFam:3.40.50.720:FF:000070) AND taxon: Rodentia (NCBITaxon:9989)


    **Condition Set 58:** FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1
    (CATH.FunFam:1.10.150.900:FF:000003) AND FunFam id: N-fatty-acyl-amino acid synthase/hydrolase
    PM20D1 (CATH.FunFam:3.40.630.10:FF:000027)


    **Condition Set 59:** FunFam id: Histidine biosynthesis bifunctional protein HisIE
    (CATH.FunFam:1.10.287.1080:FF:000002) AND FunFam id: Histidine biosynthesis trifunctional
    protein (CATH.FunFam:3.10.20.810:FF:000002)


    **Condition Set 60:** FunFam id: peptide-N(4)-(N-acetyl-beta- glucosaminyl)asparagine
    amidase (CATH.FunFam:2.20.25.10:FF:000011) AND FunFam id: Peptidase (PNG1) (CATH.FunFam:3.10.620.30:FF:000004)


    **Condition Set 61:** FunFam id: (S)-ureidoglycine aminohydrolase (CATH.FunFam:2.60.120.10:FF:000137)
    AND taxon: Magnoliopsida (NCBITaxon:3398)


    **Condition Set 62:** FunFam id: Cytosine deaminase (CATH.FunFam:3.20.20.140:FF:000019)
    AND taxon: Pseudomonadati (NCBITaxon:3379134)


    **Condition Set 63:** FunFam id: Guanine deaminase (CATH.FunFam:3.20.20.140:FF:000021)
    AND taxon: Myomorpha (NCBITaxon:1963758)


    **Condition Set 64:** FunFam id: Related to Chitin deacetylase (CATH.FunFam:3.20.20.370:FF:000004)
    AND taxon: Basidiomycota (NCBITaxon:5204)


    **Condition Set 65:** FunFam id: Chitin deacetylase (CATH.FunFam:3.20.20.370:FF:000008)
    AND taxon: Saccharomycotina (NCBITaxon:147537)


    **Condition Set 66:** FunFam id: Histidine triad nucleotide-binding protein 1
    (CATH.FunFam:3.30.428.10:FF:000005) AND taxon: Haplorrhini (NCBITaxon:376913)


    **Condition Set 67:** FunFam id: N-acetyldiaminopimelate deacetylase (CATH.FunFam:3.30.70.360:FF:000001)
    AND taxon: Bacillati (NCBITaxon:1783272)


    **Condition Set 68:** FunFam id: Bifunctional purine biosynthesis protein PURH
    (CATH.FunFam:3.40.140.20:FF:000008) AND FunFam id: Bifunctional purine biosynthesis
    protein (CATH.FunFam:3.40.50.1380:FF:000003)


    **Condition Set 69:** FunFam id: GTP cyclohydrolase-2 (CATH.FunFam:3.40.50.10990:FF:000002)
    AND taxon: Pseudomonadota (NCBITaxon:1224)


    **Condition Set 70:** FunFam id: Glucosamine-6-phosphate isomerase (CATH.FunFam:3.40.50.1360:FF:000004)
    AND taxon: Catarrhini (NCBITaxon:9526)


    **Condition Set 71:** FunFam id: Streptothricin hydrolase (CATH.FunFam:3.40.50.850:FF:000018)
    AND taxon: Actinomycetota (NCBITaxon:201174)


    **Condition Set 72:** FunFam id: Nicotinamidase 2 (CATH.FunFam:3.40.50.850:FF:000020)
    AND taxon: eudicotyledons (NCBITaxon:71240)


    **Condition Set 73:** FunFam id: 1,6-anhydro-N-acetylmuramyl-L-alanine amidase
    (CATH.FunFam:3.40.80.10:FF:000002) AND taxon: Gammaproteobacteria (NCBITaxon:1236)


    **Condition Set 74:** FunFam id: Arginase-1 (CATH.FunFam:3.40.800.10:FF:000011)
    AND taxon: Muridae (NCBITaxon:10066)


    **Condition Set 75:** FunFam id: Histone deacetylase (CATH.FunFam:3.40.800.20:FF:000003)
    AND taxon: Hominidae (NCBITaxon:9604)


    **Condition Set 76:** FunFam id: Isoaspartyl peptidase/L-asparaginase (CATH.FunFam:3.60.20.30:FF:000001)
    AND taxon: Gunneridae (NCBITaxon:91827)


    **Condition Set 77:** FunFam id: N(G),N(G)-dimethylarginine dimethylaminohydrolase
    1 (CATH.FunFam:3.75.10.10:FF:000004) AND taxon: Murinae (NCBITaxon:39107)


    **Condition Set 78:** FunFam id: 5-hydroxyisourate hydrolase (CATH.FunFam:2.60.40.180:FF:000004)


    **Condition Set 79:** FunFam id: Peptidyl arginine deiminase 3 (CATH.FunFam:2.60.40.1860:FF:000002)


    **Condition Set 80:** FunFam id: Imidazolonepropionase (CATH.FunFam:3.20.20.140:FF:000007)


    **Condition Set 81:** FunFam id: tRNA-specific adenosine deaminase (CATH.FunFam:3.40.140.10:FF:000005)


    **Condition Set 82:** FunFam id: Cytidine deaminase (CATH.FunFam:3.40.140.10:FF:000006)


    **Condition Set 83:** FunFam id: Cytosine deaminase (CATH.FunFam:3.40.140.10:FF:000016)


    **Condition Set 84:** FunFam id: Single-stranded DNA cytosine deaminase (CATH.FunFam:3.40.140.10:FF:000022)


    **Condition Set 85:** FunFam id: Probable C->U-editing enzyme APOBEC-2 (CATH.FunFam:3.40.140.10:FF:000031)


    **Condition Set 86:** FunFam id: tRNA-specific adenosine deaminase (CATH.FunFam:3.40.140.10:FF:000039)


    **Condition Set 87:** FunFam id: Agmatinase (CATH.FunFam:3.40.800.10:FF:000002)


    **Condition Set 88:** FunFam id: Arginase (CATH.FunFam:3.40.800.10:FF:000009)


    **Condition Set 89:** FunFam id: Agmatine deiminase (CATH.FunFam:3.75.10.10:FF:000012)


    **Condition Set 90:** FunFam id: Peptide deformylase (CATH.FunFam:3.90.45.10:FF:000003)


    *These condition sets are OR-ed together (any match triggers the annotation)*'
  protein_count: 0 total (0 reviewed, 0 unreviewed)
provider_config:
  timeout: 600
  max_retries: 3
  parameters:
    allowed_domains: []
    temperature: 0.1
citation_count: 34
---

## Question

# UniProt Rule Research

## Rule Context

- **Rule ID:** ARBA00026849
- **Rule Type:** arba
- **GO Term(s) Predicted:** GO:0016810 (hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds)
- **Proteins Annotated:** 0 total (0 reviewed, 0 unreviewed)

### Condition Summary

**Condition Set 1:** InterPro id: Adenosine deaminase domain (InterPro:IPR001365) AND InterPro id: Adenosine/adenine deaminase (InterPro:IPR006330) AND InterPro id: Adenosine/AMP deaminase active site (InterPro:IPR006650)

**Condition Set 2:** InterPro id: Carbon-nitrogen hydrolase (InterPro:IPR003010) AND InterPro id: Carbon-nitrogen hydrolase superfamily (InterPro:IPR036526) AND taxon: Viridiplantae (NCBITaxon:33090)

**Condition Set 3:** InterPro id: Cytidine deaminase-like (InterPro:IPR016193) AND taxon: Saccharomycotina (NCBITaxon:147537)

**Condition Set 4:** InterPro id: Asparaginase/glutaminase-like (InterPro:IPR006034) AND InterPro id: Asparaginase/glutaminase, active site 1 (InterPro:IPR020827) AND InterPro id: L-asparaginase, C-terminal (InterPro:IPR027473)

**Condition Set 5:** InterPro id: Nitrilase/cyanide hydratase, conserved site (InterPro:IPR000132) AND InterPro id: Nitrilase/Cyanide hydratase (InterPro:IPR044149)

**Condition Set 6:** InterPro id: GTP cyclohydrolase I (InterPro:IPR001474) AND InterPro id: GTP cyclohydrolase I, conserved site (InterPro:IPR018234) AND taxon: Metazoa (NCBITaxon:33208)

**Condition Set 7:** InterPro id: Cytidine and deoxycytidylate deaminase domain (InterPro:IPR002125) AND InterPro id: APOBEC/CMP deaminase, zinc-binding (InterPro:IPR016192) AND taxon: Fungi (NCBITaxon:4751)

**Condition Set 8:** InterPro id: Peptidase M20 (InterPro:IPR002933) AND InterPro id: Amidase, carbamoylase-type (InterPro:IPR010158) AND PANTHER id: PTHR32494:SF19

**Condition Set 9:** InterPro id: Ureohydrolase (InterPro:IPR006035) AND InterPro id: Ureohydrolase domain superfamily (InterPro:IPR023696) AND taxon: Bacteria (NCBITaxon:2)

**Condition Set 10:** InterPro id: Biotinidase-like, eukaryotic (InterPro:IPR012101) AND InterPro id: Biotinidase/VNN family (InterPro:IPR040154) AND InterPro id: Vanin, C-terminal (InterPro:IPR043957)

**Condition Set 11:** InterPro id: Cytidine and Deoxycytidylate Deaminase (InterPro:IPR050202) AND PANTHER id: PTHR11644:SF2

**Condition Set 12:** PANTHER id: PTHR43114:SF6

**Condition Set 13:** InterPro id: Peptidase T2, asparaginase 2 (InterPro:IPR000246) AND InterPro id: Nucleophile aminohydrolases, N-terminal (InterPro:IPR029055) AND PANTHER id: PTHR10188:SF6

**Condition Set 14:** InterPro id: Beta-lactamase, class-B, conserved site (InterPro:IPR001018) AND InterPro id: Metallo-beta-lactamase (InterPro:IPR001279) AND taxon: Pseudomonadati (NCBITaxon:3379134)

**Condition Set 15:** InterPro id: NodB homology domain (InterPro:IPR002509) AND InterPro id: Polysaccharide deacetylase, ArnD subfamily (InterPro:IPR050248) AND taxon: Eukaryota (NCBITaxon:2759)

**Condition Set 16:** InterPro id: UDP-3-O-acyl N-acetylglucosamine deacetylase (InterPro:IPR004463) AND InterPro id: UDP-3-O-acyl N-acetylglucosamine deacetylase, C-terminal (InterPro:IPR011334) AND taxon: Streptophyta (NCBITaxon:35493)

**Condition Set 17:** InterPro id: Amidohydrolase-related (InterPro:IPR006680) AND InterPro id: Metal-dependent hydrolase, composite domain superfamily (InterPro:IPR011059) AND InterPro id: Deaminase MtaD/DadD (InterPro:IPR023512)

**Condition Set 18:** InterPro id: Kynurenine formamidase/cyclase-like (InterPro:IPR007325) AND InterPro id: Kynurenine formamidase, bacteria (InterPro:IPR017484) AND InterPro id: Kynurenine formamidase superfamily (InterPro:IPR037175)

**Condition Set 19:** InterPro id: Arginase (InterPro:IPR014033) AND InterPro id: Ureohydrolase, manganese-binding site (InterPro:IPR020855) AND PANTHER id: PTHR43782:SF3

**Condition Set 20:** InterPro id: Amidase, conserved site (InterPro:IPR020556) AND InterPro id: Amidase signature domain (InterPro:IPR023631) AND taxon: Tracheophyta (NCBITaxon:58023)

**Condition Set 21:** InterPro id: Agmatinase-related (InterPro:IPR005925) AND PANTHER id: PTHR11358:SF26

**Condition Set 22:** InterPro id: AMP deaminase (InterPro:IPR006329) AND taxon: Glires (NCBITaxon:314147)

**Condition Set 23:** InterPro id: Peptide deformylase (InterPro:IPR023635) AND InterPro id: Peptide deformylase superfamily (InterPro:IPR036821)

**Condition Set 24:** InterPro id: Aliphatic Amidase/Beta-Ureidopropionase (InterPro:IPR050345) AND PANTHER id: PTHR43674:SF2

**Condition Set 25:** FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:1.10.286.10:FF:000003) AND FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000012) AND taxon: Metazoa (NCBITaxon:33208)

**Condition Set 26:** FunFam id: biotinidase isoform X1 (CATH.FunFam:3.60.110.10:FF:000001) AND taxon: Eukaryota (NCBITaxon:2759)

**Condition Set 27:** FunFam id: Bifunctional nitrilase/nitrile hydratase NIT4B (CATH.FunFam:3.60.110.10:FF:000006) AND taxon: Viridiplantae (NCBITaxon:33090)

**Condition Set 28:** FunFam id: L-asparaginase 2 (CATH.FunFam:3.40.50.1170:FF:000001)

**Condition Set 29:** FunFam id: Allantoate amidohydrolase (CATH.FunFam:3.40.630.10:FF:000044)

**Condition Set 30:** FunFam id: Probable UDP-3-O-acyl-N-acetylglucosamine deacetylase 3, mitochondrial (CATH.FunFam:3.30.1700.10:FF:000002) AND FunFam id: Probable UDP-3-O-acyl-N-acetylglucosamine deacetylase 2, mitochondrial (CATH.FunFam:3.30.230.20:FF:000002) AND taxon: Streptophyta (NCBITaxon:35493)

**Condition Set 31:** FunFam id: Kynurenine formamidase (CATH.FunFam:3.50.30.50:FF:000001) AND taxon: Bacteria (NCBITaxon:2)

**Condition Set 32:** FunFam id: AMP deaminase (CATH.FunFam:4.10.800.20:FF:000001) AND taxon: Glires (NCBITaxon:314147)

**Condition Set 33:** FunFam id: 5-methylthioadenosine/S-adenosylhomocysteine deaminase (CATH.FunFam:3.20.20.140:FF:000014)

**Condition Set 34:** FunFam id: Bifunctional purine biosynthesis protein PURH (CATH.FunFam:1.10.287.440:FF:000001) AND FunFam id: Bifunctional purine biosynthesis protein (CATH.FunFam:3.40.140.20:FF:000003) AND taxon: Dikarya (NCBITaxon:451864)

**Condition Set 35:** FunFam id: Protein-arginine deiminase type-2 (CATH.FunFam:2.60.40.1700:FF:000001) AND FunFam id: Protein-arginine deiminase type-2 (CATH.FunFam:3.75.10.10:FF:000003) AND taxon: Rattus (NCBITaxon:10114)

**Condition Set 36:** FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:3.30.1490.20:FF:000001) AND FunFam id: CAD protein-like isoform X1 (CATH.FunFam:3.40.50.1370:FF:000005) AND FunFam id: Carbamoyl-phosphate synthase 1, mitochondrial (CATH.FunFam:3.40.50.880:FF:000006)

**Condition Set 37:** FunFam id: Adenosine deaminase RNA-specific B2 (inactive) (CATH.FunFam:3.30.160.20:FF:000009) AND FunFam id: double-stranded RNA-specific editase 1 isoform X1 (CATH.FunFam:3.30.160.20:FF:000011) AND taxon: Chordata (NCBITaxon:7711)

**Condition Set 38:** FunFam id: N-acylsphingosine amidohydrolase 2 (CATH.FunFam:2.60.40.2300:FF:000001) AND taxon: Craniata (NCBITaxon:89593)

**Condition Set 39:** FunFam id: Adenosine deaminase (CATH.FunFam:3.20.20.140:FF:000038) AND taxon: Vertebrata (NCBITaxon:7742)

**Condition Set 40:** FunFam id: bifunctional heparan sulfate N-deacetylase/N-sulfotransferase 1 (CATH.FunFam:3.40.50.300:FF:000176) AND taxon: Mus (NCBITaxon:862507)

**Condition Set 41:** FunFam id: Arginase 1, mitochondrial (CATH.FunFam:3.40.800.10:FF:000007) AND taxon: Embryophyta (NCBITaxon:3193)

**Condition Set 42:** FunFam id: Dihydropyrimidinase like 3 (CATH.FunFam:3.20.20.140:FF:000001)

**Condition Set 43:** FunFam id: Cytidine deaminase (CATH.FunFam:3.40.140.10:FF:000008)

**Condition Set 44:** FunFam id: Aminoacylase-1, putative (CATH.FunFam:1.10.150.900:FF:000001) AND FunFam id: Putative Aminoacylase-1 (CATH.FunFam:3.30.70.360:FF:000005) AND FunFam id: Aminoacylase 1 (CATH.FunFam:3.40.630.10:FF:000019)

**Condition Set 45:** FunFam id: Putative formate--tetrahydrofolate ligase (CATH.FunFam:3.10.410.10:FF:000001) AND FunFam id: C-1-tetrahydrofolate synthase, cytoplasmic, putative (CATH.FunFam:3.40.50.10860:FF:000005) AND taxon: Ascomycota (NCBITaxon:4890)

**Condition Set 46:** FunFam id: NAD-dependent protein deacetylase sirtuin-1 (CATH.FunFam:3.30.1600.10:FF:000013) AND FunFam id: NAD-dependent deacetylase sirtuin-2 (CATH.FunFam:3.40.50.1220:FF:000005) AND taxon: Euteleostomi (NCBITaxon:117571)

**Condition Set 47:** FunFam id: Jasmonate-regulated gene 21 (CATH.FunFam:2.60.120.330:FF:000008) AND taxon: Tracheophyta (NCBITaxon:58023)

**Condition Set 48:** FunFam id: Apolipoprotein B editing enzyme catalytic polypeptide-like 3H (CATH.FunFam:3.40.140.10:FF:000047) AND taxon: Primates (NCBITaxon:9443)

**Condition Set 49:** FunFam id: Protein-arginine deiminase type-4 (CATH.FunFam:2.60.40.1860:FF:000003)

**Condition Set 50:** FunFam id: N-acetylglucosamine-6-phosphate deacetylase (CATH.FunFam:3.20.20.140:FF:000004)

**Condition Set 51:** FunFam id: Guanine deaminase (CATH.FunFam:3.20.20.140:FF:000022)

**Condition Set 52:** FunFam id: Arginase (CATH.FunFam:3.40.800.10:FF:000005)

**Condition Set 53:** FunFam id: Beta-ureidopropionase 1 (CATH.FunFam:3.60.110.10:FF:000009)

**Condition Set 54:** FunFam id: N(4)-(Beta-N-acetylglucosaminyl)-L-asparaginase isoform X1 (CATH.FunFam:3.60.20.30:FF:000003)

**Condition Set 55:** FunFam id: Methylenetetrahydrofolate dehydrogenase (NADP+ dependent) 1 like (CATH.FunFam:1.10.8.770:FF:000001) AND FunFam id: C-1-tetrahydrofolate synthase, cytoplasmic (CATH.FunFam:3.40.50.300:FF:000245) AND FunFam id: C-1-tetrahydrofolate synthase, cytoplasmic isoform X2 (CATH.FunFam:3.40.50.300:FF:001123)

**Condition Set 56:** FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000007) AND FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000008) AND taxon: Spermatophyta (NCBITaxon:58024)

**Condition Set 57:** FunFam id: Bifunctional protein FolD (CATH.FunFam:3.40.50.10860:FF:000001) AND FunFam id: probable bifunctional methylenetetrahydrofolate dehydrogenase/cyclohydrolase 2 (CATH.FunFam:3.40.50.720:FF:000070) AND taxon: Rodentia (NCBITaxon:9989)

**Condition Set 58:** FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1 (CATH.FunFam:1.10.150.900:FF:000003) AND FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1 (CATH.FunFam:3.40.630.10:FF:000027)

**Condition Set 59:** FunFam id: Histidine biosynthesis bifunctional protein HisIE (CATH.FunFam:1.10.287.1080:FF:000002) AND FunFam id: Histidine biosynthesis trifunctional protein (CATH.FunFam:3.10.20.810:FF:000002)

**Condition Set 60:** FunFam id: peptide-N(4)-(N-acetyl-beta- glucosaminyl)asparagine amidase (CATH.FunFam:2.20.25.10:FF:000011) AND FunFam id: Peptidase (PNG1) (CATH.FunFam:3.10.620.30:FF:000004)

**Condition Set 61:** FunFam id: (S)-ureidoglycine aminohydrolase (CATH.FunFam:2.60.120.10:FF:000137) AND taxon: Magnoliopsida (NCBITaxon:3398)

**Condition Set 62:** FunFam id: Cytosine deaminase (CATH.FunFam:3.20.20.140:FF:000019) AND taxon: Pseudomonadati (NCBITaxon:3379134)

**Condition Set 63:** FunFam id: Guanine deaminase (CATH.FunFam:3.20.20.140:FF:000021) AND taxon: Myomorpha (NCBITaxon:1963758)

**Condition Set 64:** FunFam id: Related to Chitin deacetylase (CATH.FunFam:3.20.20.370:FF:000004) AND taxon: Basidiomycota (NCBITaxon:5204)

**Condition Set 65:** FunFam id: Chitin deacetylase (CATH.FunFam:3.20.20.370:FF:000008) AND taxon: Saccharomycotina (NCBITaxon:147537)

**Condition Set 66:** FunFam id: Histidine triad nucleotide-binding protein 1 (CATH.FunFam:3.30.428.10:FF:000005) AND taxon: Haplorrhini (NCBITaxon:376913)

**Condition Set 67:** FunFam id: N-acetyldiaminopimelate deacetylase (CATH.FunFam:3.30.70.360:FF:000001) AND taxon: Bacillati (NCBITaxon:1783272)

**Condition Set 68:** FunFam id: Bifunctional purine biosynthesis protein PURH (CATH.FunFam:3.40.140.20:FF:000008) AND FunFam id: Bifunctional purine biosynthesis protein (CATH.FunFam:3.40.50.1380:FF:000003)

**Condition Set 69:** FunFam id: GTP cyclohydrolase-2 (CATH.FunFam:3.40.50.10990:FF:000002) AND taxon: Pseudomonadota (NCBITaxon:1224)

**Condition Set 70:** FunFam id: Glucosamine-6-phosphate isomerase (CATH.FunFam:3.40.50.1360:FF:000004) AND taxon: Catarrhini (NCBITaxon:9526)

**Condition Set 71:** FunFam id: Streptothricin hydrolase (CATH.FunFam:3.40.50.850:FF:000018) AND taxon: Actinomycetota (NCBITaxon:201174)

**Condition Set 72:** FunFam id: Nicotinamidase 2 (CATH.FunFam:3.40.50.850:FF:000020) AND taxon: eudicotyledons (NCBITaxon:71240)

**Condition Set 73:** FunFam id: 1,6-anhydro-N-acetylmuramyl-L-alanine amidase (CATH.FunFam:3.40.80.10:FF:000002) AND taxon: Gammaproteobacteria (NCBITaxon:1236)

**Condition Set 74:** FunFam id: Arginase-1 (CATH.FunFam:3.40.800.10:FF:000011) AND taxon: Muridae (NCBITaxon:10066)

**Condition Set 75:** FunFam id: Histone deacetylase (CATH.FunFam:3.40.800.20:FF:000003) AND taxon: Hominidae (NCBITaxon:9604)

**Condition Set 76:** FunFam id: Isoaspartyl peptidase/L-asparaginase (CATH.FunFam:3.60.20.30:FF:000001) AND taxon: Gunneridae (NCBITaxon:91827)

**Condition Set 77:** FunFam id: N(G),N(G)-dimethylarginine dimethylaminohydrolase 1 (CATH.FunFam:3.75.10.10:FF:000004) AND taxon: Murinae (NCBITaxon:39107)

**Condition Set 78:** FunFam id: 5-hydroxyisourate hydrolase (CATH.FunFam:2.60.40.180:FF:000004)

**Condition Set 79:** FunFam id: Peptidyl arginine deiminase 3 (CATH.FunFam:2.60.40.1860:FF:000002)

**Condition Set 80:** FunFam id: Imidazolonepropionase (CATH.FunFam:3.20.20.140:FF:000007)

**Condition Set 81:** FunFam id: tRNA-specific adenosine deaminase (CATH.FunFam:3.40.140.10:FF:000005)

**Condition Set 82:** FunFam id: Cytidine deaminase (CATH.FunFam:3.40.140.10:FF:000006)

**Condition Set 83:** FunFam id: Cytosine deaminase (CATH.FunFam:3.40.140.10:FF:000016)

**Condition Set 84:** FunFam id: Single-stranded DNA cytosine deaminase (CATH.FunFam:3.40.140.10:FF:000022)

**Condition Set 85:** FunFam id: Probable C->U-editing enzyme APOBEC-2 (CATH.FunFam:3.40.140.10:FF:000031)

**Condition Set 86:** FunFam id: tRNA-specific adenosine deaminase (CATH.FunFam:3.40.140.10:FF:000039)

**Condition Set 87:** FunFam id: Agmatinase (CATH.FunFam:3.40.800.10:FF:000002)

**Condition Set 88:** FunFam id: Arginase (CATH.FunFam:3.40.800.10:FF:000009)

**Condition Set 89:** FunFam id: Agmatine deiminase (CATH.FunFam:3.75.10.10:FF:000012)

**Condition Set 90:** FunFam id: Peptide deformylase (CATH.FunFam:3.90.45.10:FF:000003)

*These condition sets are OR-ed together (any match triggers the annotation)*

---

## Research Objective

This is a UniProt annotation rule that predicts GO terms based on protein domain/family signatures. Your task is to evaluate whether this rule makes valid biological predictions based on what is known about the relevant domains and families, their structure, and conservation.

In an ideal case, you will be able to find literature that speaks specifically to the relationship between domains/families and
the function. Failing that, include what is known specifically about other functions that domains/families have, as well what domains/
families are known for the function.

## Research Questions

### 1. Domain/Signature Context

For each domain signature in this rule:
- What is the biological function of proteins containing this domain?
- Is this domain diagnostic for the predicted function?
- Are there known subfamilies with different functions?
- Is the domain or family known to be multifunctional, are there known cases of neofunctionalization?

### 2. GO Term Appropriateness

For the predicted GO term(s) **GO:0016810 (hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds)**:
- Is this GO term appropriate for proteins matching these conditions?
- Is the term too broad? (e.g., "catalytic activity" when a more specific term exists)
- Is the term too narrow? (e.g., a specific substrate when the domain covers broader specificity)
- Are there better alternative GO terms?

### 3. Literature Support

- What experimental evidence supports this functional annotation?
- Are there key papers describing this protein family/enzyme?
- Are there any contradictory findings in the literature?
- Is this a well-established function or more speculative?

### 4. Taxonomic Considerations

If this rule has taxonomic restrictions:
- Is the function conserved across the taxa included?
- Are there taxa that should be excluded (different function)?
- Are there taxa that could be included but are currently excluded?

### 5. Rule Logic Assessment

- Do the domain combinations make biological sense?
- Are there redundant conditions that could be removed?
- Could false positives arise from these conditions?

## Output Format

Please provide your findings in a narrative format with citations. Structure your response as:

1. **Executive Summary** - Brief assessment of rule validity
2. **Domain Analysis** - What each domain/signature represents
3. **GO Term Evaluation** - Assessment of the predicted annotation
4. **Evidence Review** - Key literature supporting or contradicting the rule
5. **Recommendations** - Suggested improvements or concerns

Be sure to include citations for all statements.

## Output

Question: You are an expert researcher providing comprehensive, well-cited information.

Provide detailed information focusing on:
1. Key concepts and definitions with current understanding
2. Recent developments and latest research (prioritize 2023-2024 sources)
3. Current applications and real-world implementations
4. Expert opinions and analysis from authoritative sources
5. Relevant statistics and data from recent studies

Format as a comprehensive research report with proper citations. Include URLs and publication dates where available.
Always prioritize recent, authoritative sources and provide specific citations for all major claims.

# UniProt Rule Research

## Rule Context

- **Rule ID:** ARBA00026849
- **Rule Type:** arba
- **GO Term(s) Predicted:** GO:0016810 (hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds)
- **Proteins Annotated:** 0 total (0 reviewed, 0 unreviewed)

### Condition Summary

**Condition Set 1:** InterPro id: Adenosine deaminase domain (InterPro:IPR001365) AND InterPro id: Adenosine/adenine deaminase (InterPro:IPR006330) AND InterPro id: Adenosine/AMP deaminase active site (InterPro:IPR006650)

**Condition Set 2:** InterPro id: Carbon-nitrogen hydrolase (InterPro:IPR003010) AND InterPro id: Carbon-nitrogen hydrolase superfamily (InterPro:IPR036526) AND taxon: Viridiplantae (NCBITaxon:33090)

**Condition Set 3:** InterPro id: Cytidine deaminase-like (InterPro:IPR016193) AND taxon: Saccharomycotina (NCBITaxon:147537)

**Condition Set 4:** InterPro id: Asparaginase/glutaminase-like (InterPro:IPR006034) AND InterPro id: Asparaginase/glutaminase, active site 1 (InterPro:IPR020827) AND InterPro id: L-asparaginase, C-terminal (InterPro:IPR027473)

**Condition Set 5:** InterPro id: Nitrilase/cyanide hydratase, conserved site (InterPro:IPR000132) AND InterPro id: Nitrilase/Cyanide hydratase (InterPro:IPR044149)

**Condition Set 6:** InterPro id: GTP cyclohydrolase I (InterPro:IPR001474) AND InterPro id: GTP cyclohydrolase I, conserved site (InterPro:IPR018234) AND taxon: Metazoa (NCBITaxon:33208)

**Condition Set 7:** InterPro id: Cytidine and deoxycytidylate deaminase domain (InterPro:IPR002125) AND InterPro id: APOBEC/CMP deaminase, zinc-binding (InterPro:IPR016192) AND taxon: Fungi (NCBITaxon:4751)

**Condition Set 8:** InterPro id: Peptidase M20 (InterPro:IPR002933) AND InterPro id: Amidase, carbamoylase-type (InterPro:IPR010158) AND PANTHER id: PTHR32494:SF19

**Condition Set 9:** InterPro id: Ureohydrolase (InterPro:IPR006035) AND InterPro id: Ureohydrolase domain superfamily (InterPro:IPR023696) AND taxon: Bacteria (NCBITaxon:2)

**Condition Set 10:** InterPro id: Biotinidase-like, eukaryotic (InterPro:IPR012101) AND InterPro id: Biotinidase/VNN family (InterPro:IPR040154) AND InterPro id: Vanin, C-terminal (InterPro:IPR043957)

**Condition Set 11:** InterPro id: Cytidine and Deoxycytidylate Deaminase (InterPro:IPR050202) AND PANTHER id: PTHR11644:SF2

**Condition Set 12:** PANTHER id: PTHR43114:SF6

**Condition Set 13:** InterPro id: Peptidase T2, asparaginase 2 (InterPro:IPR000246) AND InterPro id: Nucleophile aminohydrolases, N-terminal (InterPro:IPR029055) AND PANTHER id: PTHR10188:SF6

**Condition Set 14:** InterPro id: Beta-lactamase, class-B, conserved site (InterPro:IPR001018) AND InterPro id: Metallo-beta-lactamase (InterPro:IPR001279) AND taxon: Pseudomonadati (NCBITaxon:3379134)

**Condition Set 15:** InterPro id: NodB homology domain (InterPro:IPR002509) AND InterPro id: Polysaccharide deacetylase, ArnD subfamily (InterPro:IPR050248) AND taxon: Eukaryota (NCBITaxon:2759)

**Condition Set 16:** InterPro id: UDP-3-O-acyl N-acetylglucosamine deacetylase (InterPro:IPR004463) AND InterPro id: UDP-3-O-acyl N-acetylglucosamine deacetylase, C-terminal (InterPro:IPR011334) AND taxon: Streptophyta (NCBITaxon:35493)

**Condition Set 17:** InterPro id: Amidohydrolase-related (InterPro:IPR006680) AND InterPro id: Metal-dependent hydrolase, composite domain superfamily (InterPro:IPR011059) AND InterPro id: Deaminase MtaD/DadD (InterPro:IPR023512)

**Condition Set 18:** InterPro id: Kynurenine formamidase/cyclase-like (InterPro:IPR007325) AND InterPro id: Kynurenine formamidase, bacteria (InterPro:IPR017484) AND InterPro id: Kynurenine formamidase superfamily (InterPro:IPR037175)

**Condition Set 19:** InterPro id: Arginase (InterPro:IPR014033) AND InterPro id: Ureohydrolase, manganese-binding site (InterPro:IPR020855) AND PANTHER id: PTHR43782:SF3

**Condition Set 20:** InterPro id: Amidase, conserved site (InterPro:IPR020556) AND InterPro id: Amidase signature domain (InterPro:IPR023631) AND taxon: Tracheophyta (NCBITaxon:58023)

**Condition Set 21:** InterPro id: Agmatinase-related (InterPro:IPR005925) AND PANTHER id: PTHR11358:SF26

**Condition Set 22:** InterPro id: AMP deaminase (InterPro:IPR006329) AND taxon: Glires (NCBITaxon:314147)

**Condition Set 23:** InterPro id: Peptide deformylase (InterPro:IPR023635) AND InterPro id: Peptide deformylase superfamily (InterPro:IPR036821)

**Condition Set 24:** InterPro id: Aliphatic Amidase/Beta-Ureidopropionase (InterPro:IPR050345) AND PANTHER id: PTHR43674:SF2

**Condition Set 25:** FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:1.10.286.10:FF:000003) AND FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000012) AND taxon: Metazoa (NCBITaxon:33208)

**Condition Set 26:** FunFam id: biotinidase isoform X1 (CATH.FunFam:3.60.110.10:FF:000001) AND taxon: Eukaryota (NCBITaxon:2759)

**Condition Set 27:** FunFam id: Bifunctional nitrilase/nitrile hydratase NIT4B (CATH.FunFam:3.60.110.10:FF:000006) AND taxon: Viridiplantae (NCBITaxon:33090)

**Condition Set 28:** FunFam id: L-asparaginase 2 (CATH.FunFam:3.40.50.1170:FF:000001)

**Condition Set 29:** FunFam id: Allantoate amidohydrolase (CATH.FunFam:3.40.630.10:FF:000044)

**Condition Set 30:** FunFam id: Probable UDP-3-O-acyl-N-acetylglucosamine deacetylase 3, mitochondrial (CATH.FunFam:3.30.1700.10:FF:000002) AND FunFam id: Probable UDP-3-O-acyl-N-acetylglucosamine deacetylase 2, mitochondrial (CATH.FunFam:3.30.230.20:FF:000002) AND taxon: Streptophyta (NCBITaxon:35493)

**Condition Set 31:** FunFam id: Kynurenine formamidase (CATH.FunFam:3.50.30.50:FF:000001) AND taxon: Bacteria (NCBITaxon:2)

**Condition Set 32:** FunFam id: AMP deaminase (CATH.FunFam:4.10.800.20:FF:000001) AND taxon: Glires (NCBITaxon:314147)

**Condition Set 33:** FunFam id: 5-methylthioadenosine/S-adenosylhomocysteine deaminase (CATH.FunFam:3.20.20.140:FF:000014)

**Condition Set 34:** FunFam id: Bifunctional purine biosynthesis protein PURH (CATH.FunFam:1.10.287.440:FF:000001) AND FunFam id: Bifunctional purine biosynthesis protein (CATH.FunFam:3.40.140.20:FF:000003) AND taxon: Dikarya (NCBITaxon:451864)

**Condition Set 35:** FunFam id: Protein-arginine deiminase type-2 (CATH.FunFam:2.60.40.1700:FF:000001) AND FunFam id: Protein-arginine deiminase type-2 (CATH.FunFam:3.75.10.10:FF:000003) AND taxon: Rattus (NCBITaxon:10114)

**Condition Set 36:** FunFam id: Carbamoyl-phosphate synthase large chain (CATH.FunFam:3.30.1490.20:FF:000001) AND FunFam id: CAD protein-like isoform X1 (CATH.FunFam:3.40.50.1370:FF:000005) AND FunFam id: Carbamoyl-phosphate synthase 1, mitochondrial (CATH.FunFam:3.40.50.880:FF:000006)

**Condition Set 37:** FunFam id: Adenosine deaminase RNA-specific B2 (inactive) (CATH.FunFam:3.30.160.20:FF:000009) AND FunFam id: double-stranded RNA-specific editase 1 isoform X1 (CATH.FunFam:3.30.160.20:FF:000011) AND taxon: Chordata (NCBITaxon:7711)

**Condition Set 38:** FunFam id: N-acylsphingosine amidohydrolase 2 (CATH.FunFam:2.60.40.2300:FF:000001) AND taxon: Craniata (NCBITaxon:89593)

**Condition Set 39:** FunFam id: Adenosine deaminase (CATH.FunFam:3.20.20.140:FF:000038) AND taxon: Vertebrata (NCBITaxon:7742)

**Condition Set 40:** FunFam id: bifunctional heparan sulfate N-deacetylase/N-sulfotransferase 1 (CATH.FunFam:3.40.50.300:FF:000176) AND taxon: Mus (NCBITaxon:862507)

**Condition Set 41:** FunFam id: Arginase 1, mitochondrial (CATH.FunFam:3.40.800.10:FF:000007) AND taxon: Embryophyta (NCBITaxon:3193)

**Condition Set 42:** FunFam id: Dihydropyrimidinase like 3 (CATH.FunFam:3.20.20.140:FF:000001)

**Condition Set 43:** FunFam id: Cytidine deaminase (CATH.FunFam:3.40.140.10:FF:000008)

**Condition Set 44:** FunFam id: Aminoacylase-1, putative (CATH.FunFam:1.10.150.900:FF:000001) AND FunFam id: Putative Aminoacylase-1 (CATH.FunFam:3.30.70.360:FF:000005) AND FunFam id: Aminoacylase 1 (CATH.FunFam:3.40.630.10:FF:000019)

**Condition Set 45:** FunFam id: Putative formate--tetrahydrofolate ligase (CATH.FunFam:3.10.410.10:FF:000001) AND FunFam id: C-1-tetrahydrofolate synthase, cytoplasmic, putative (CATH.FunFam:3.40.50.10860:FF:000005) AND taxon: Ascomycota (NCBITaxon:4890)

**Condition Set 46:** FunFam id: NAD-dependent protein deacetylase sirtuin-1 (CATH.FunFam:3.30.1600.10:FF:000013) AND FunFam id: NAD-dependent deacetylase sirtuin-2 (CATH.FunFam:3.40.50.1220:FF:000005) AND taxon: Euteleostomi (NCBITaxon:117571)

**Condition Set 47:** FunFam id: Jasmonate-regulated gene 21 (CATH.FunFam:2.60.120.330:FF:000008) AND taxon: Tracheophyta (NCBITaxon:58023)

**Condition Set 48:** FunFam id: Apolipoprotein B editing enzyme catalytic polypeptide-like 3H (CATH.FunFam:3.40.140.10:FF:000047) AND taxon: Primates (NCBITaxon:9443)

**Condition Set 49:** FunFam id: Protein-arginine deiminase type-4 (CATH.FunFam:2.60.40.1860:FF:000003)

**Condition Set 50:** FunFam id: N-acetylglucosamine-6-phosphate deacetylase (CATH.FunFam:3.20.20.140:FF:000004)

**Condition Set 51:** FunFam id: Guanine deaminase (CATH.FunFam:3.20.20.140:FF:000022)

**Condition Set 52:** FunFam id: Arginase (CATH.FunFam:3.40.800.10:FF:000005)

**Condition Set 53:** FunFam id: Beta-ureidopropionase 1 (CATH.FunFam:3.60.110.10:FF:000009)

**Condition Set 54:** FunFam id: N(4)-(Beta-N-acetylglucosaminyl)-L-asparaginase isoform X1 (CATH.FunFam:3.60.20.30:FF:000003)

**Condition Set 55:** FunFam id: Methylenetetrahydrofolate dehydrogenase (NADP+ dependent) 1 like (CATH.FunFam:1.10.8.770:FF:000001) AND FunFam id: C-1-tetrahydrofolate synthase, cytoplasmic (CATH.FunFam:3.40.50.300:FF:000245) AND FunFam id: C-1-tetrahydrofolate synthase, cytoplasmic isoform X2 (CATH.FunFam:3.40.50.300:FF:001123)

**Condition Set 56:** FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000007) AND FunFam id: GTP cyclohydrolase 1 (CATH.FunFam:3.30.1130.10:FF:000008) AND taxon: Spermatophyta (NCBITaxon:58024)

**Condition Set 57:** FunFam id: Bifunctional protein FolD (CATH.FunFam:3.40.50.10860:FF:000001) AND FunFam id: probable bifunctional methylenetetrahydrofolate dehydrogenase/cyclohydrolase 2 (CATH.FunFam:3.40.50.720:FF:000070) AND taxon: Rodentia (NCBITaxon:9989)

**Condition Set 58:** FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1 (CATH.FunFam:1.10.150.900:FF:000003) AND FunFam id: N-fatty-acyl-amino acid synthase/hydrolase PM20D1 (CATH.FunFam:3.40.630.10:FF:000027)

**Condition Set 59:** FunFam id: Histidine biosynthesis bifunctional protein HisIE (CATH.FunFam:1.10.287.1080:FF:000002) AND FunFam id: Histidine biosynthesis trifunctional protein (CATH.FunFam:3.10.20.810:FF:000002)

**Condition Set 60:** FunFam id: peptide-N(4)-(N-acetyl-beta- glucosaminyl)asparagine amidase (CATH.FunFam:2.20.25.10:FF:000011) AND FunFam id: Peptidase (PNG1) (CATH.FunFam:3.10.620.30:FF:000004)

**Condition Set 61:** FunFam id: (S)-ureidoglycine aminohydrolase (CATH.FunFam:2.60.120.10:FF:000137) AND taxon: Magnoliopsida (NCBITaxon:3398)

**Condition Set 62:** FunFam id: Cytosine deaminase (CATH.FunFam:3.20.20.140:FF:000019) AND taxon: Pseudomonadati (NCBITaxon:3379134)

**Condition Set 63:** FunFam id: Guanine deaminase (CATH.FunFam:3.20.20.140:FF:000021) AND taxon: Myomorpha (NCBITaxon:1963758)

**Condition Set 64:** FunFam id: Related to Chitin deacetylase (CATH.FunFam:3.20.20.370:FF:000004) AND taxon: Basidiomycota (NCBITaxon:5204)

**Condition Set 65:** FunFam id: Chitin deacetylase (CATH.FunFam:3.20.20.370:FF:000008) AND taxon: Saccharomycotina (NCBITaxon:147537)

**Condition Set 66:** FunFam id: Histidine triad nucleotide-binding protein 1 (CATH.FunFam:3.30.428.10:FF:000005) AND taxon: Haplorrhini (NCBITaxon:376913)

**Condition Set 67:** FunFam id: N-acetyldiaminopimelate deacetylase (CATH.FunFam:3.30.70.360:FF:000001) AND taxon: Bacillati (NCBITaxon:1783272)

**Condition Set 68:** FunFam id: Bifunctional purine biosynthesis protein PURH (CATH.FunFam:3.40.140.20:FF:000008) AND FunFam id: Bifunctional purine biosynthesis protein (CATH.FunFam:3.40.50.1380:FF:000003)

**Condition Set 69:** FunFam id: GTP cyclohydrolase-2 (CATH.FunFam:3.40.50.10990:FF:000002) AND taxon: Pseudomonadota (NCBITaxon:1224)

**Condition Set 70:** FunFam id: Glucosamine-6-phosphate isomerase (CATH.FunFam:3.40.50.1360:FF:000004) AND taxon: Catarrhini (NCBITaxon:9526)

**Condition Set 71:** FunFam id: Streptothricin hydrolase (CATH.FunFam:3.40.50.850:FF:000018) AND taxon: Actinomycetota (NCBITaxon:201174)

**Condition Set 72:** FunFam id: Nicotinamidase 2 (CATH.FunFam:3.40.50.850:FF:000020) AND taxon: eudicotyledons (NCBITaxon:71240)

**Condition Set 73:** FunFam id: 1,6-anhydro-N-acetylmuramyl-L-alanine amidase (CATH.FunFam:3.40.80.10:FF:000002) AND taxon: Gammaproteobacteria (NCBITaxon:1236)

**Condition Set 74:** FunFam id: Arginase-1 (CATH.FunFam:3.40.800.10:FF:000011) AND taxon: Muridae (NCBITaxon:10066)

**Condition Set 75:** FunFam id: Histone deacetylase (CATH.FunFam:3.40.800.20:FF:000003) AND taxon: Hominidae (NCBITaxon:9604)

**Condition Set 76:** FunFam id: Isoaspartyl peptidase/L-asparaginase (CATH.FunFam:3.60.20.30:FF:000001) AND taxon: Gunneridae (NCBITaxon:91827)

**Condition Set 77:** FunFam id: N(G),N(G)-dimethylarginine dimethylaminohydrolase 1 (CATH.FunFam:3.75.10.10:FF:000004) AND taxon: Murinae (NCBITaxon:39107)

**Condition Set 78:** FunFam id: 5-hydroxyisourate hydrolase (CATH.FunFam:2.60.40.180:FF:000004)

**Condition Set 79:** FunFam id: Peptidyl arginine deiminase 3 (CATH.FunFam:2.60.40.1860:FF:000002)

**Condition Set 80:** FunFam id: Imidazolonepropionase (CATH.FunFam:3.20.20.140:FF:000007)

**Condition Set 81:** FunFam id: tRNA-specific adenosine deaminase (CATH.FunFam:3.40.140.10:FF:000005)

**Condition Set 82:** FunFam id: Cytidine deaminase (CATH.FunFam:3.40.140.10:FF:000006)

**Condition Set 83:** FunFam id: Cytosine deaminase (CATH.FunFam:3.40.140.10:FF:000016)

**Condition Set 84:** FunFam id: Single-stranded DNA cytosine deaminase (CATH.FunFam:3.40.140.10:FF:000022)

**Condition Set 85:** FunFam id: Probable C->U-editing enzyme APOBEC-2 (CATH.FunFam:3.40.140.10:FF:000031)

**Condition Set 86:** FunFam id: tRNA-specific adenosine deaminase (CATH.FunFam:3.40.140.10:FF:000039)

**Condition Set 87:** FunFam id: Agmatinase (CATH.FunFam:3.40.800.10:FF:000002)

**Condition Set 88:** FunFam id: Arginase (CATH.FunFam:3.40.800.10:FF:000009)

**Condition Set 89:** FunFam id: Agmatine deiminase (CATH.FunFam:3.75.10.10:FF:000012)

**Condition Set 90:** FunFam id: Peptide deformylase (CATH.FunFam:3.90.45.10:FF:000003)

*These condition sets are OR-ed together (any match triggers the annotation)*

---

## Research Objective

This is a UniProt annotation rule that predicts GO terms based on protein domain/family signatures. Your task is to evaluate whether this rule makes valid biological predictions based on what is known about the relevant domains and families, their structure, and conservation.

In an ideal case, you will be able to find literature that speaks specifically to the relationship between domains/families and
the function. Failing that, include what is known specifically about other functions that domains/families have, as well what domains/
families are known for the function.

## Research Questions

### 1. Domain/Signature Context

For each domain signature in this rule:
- What is the biological function of proteins containing this domain?
- Is this domain diagnostic for the predicted function?
- Are there known subfamilies with different functions?
- Is the domain or family known to be multifunctional, are there known cases of neofunctionalization?

### 2. GO Term Appropriateness

For the predicted GO term(s) **GO:0016810 (hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds)**:
- Is this GO term appropriate for proteins matching these conditions?
- Is the term too broad? (e.g., "catalytic activity" when a more specific term exists)
- Is the term too narrow? (e.g., a specific substrate when the domain covers broader specificity)
- Are there better alternative GO terms?

### 3. Literature Support

- What experimental evidence supports this functional annotation?
- Are there key papers describing this protein family/enzyme?
- Are there any contradictory findings in the literature?
- Is this a well-established function or more speculative?

### 4. Taxonomic Considerations

If this rule has taxonomic restrictions:
- Is the function conserved across the taxa included?
- Are there taxa that should be excluded (different function)?
- Are there taxa that could be included but are currently excluded?

### 5. Rule Logic Assessment

- Do the domain combinations make biological sense?
- Are there redundant conditions that could be removed?
- Could false positives arise from these conditions?

## Output Format

Please provide your findings in a narrative format with citations. Structure your response as:

1. **Executive Summary** - Brief assessment of rule validity
2. **Domain Analysis** - What each domain/signature represents
3. **GO Term Evaluation** - Assessment of the predicted annotation
4. **Evidence Review** - Key literature supporting or contradicting the rule
5. **Recommendations** - Suggested improvements or concerns

Be sure to include citations for all statements.

1. Executive Summary
ARBA00026849 assigns GO:0016810 (hydrolase activity, acting on carbonnitrogen (but not peptide) bonds) when specific InterPro/CATH/PANTHER signatures are detected across many enzyme superfamilies (e.g., deaminases, deacetylases, cyclohydrolases, metallo--lactamase fold enzymes). The literature strongly supports that multiple covered domain families catalyze hydrolysis of nonpeptide CN bonds (e.g., cytidine deamination, Ndeacetylation of GlcNAc, GTP cyclohydrolase I), validating the rules overall direction. However, some covered folds (notably the metallolactamase fold) are highly multifunctional and include activities that are not strictly CN hydrolases (e.g., metaldependent nucleases, lactonases). Therefore, the single broad parent term GO:0016810 is generally appropriate as a safe baseline, but more specific children should be preferred when diagnostic motifs/subfamily features permit, and additional safeguards may be required to prevent false positives in multifunctional folds. (liu2023inhibitionofchitin pages 1-2, mapuranga2024themolecularstructure pages 1-2, novelli2024autosomalrecessiveguanosine pages 1-2, cai2002gtpcyclohydrolasei pages 1-2, diene2023origindiversityand pages 1-2, mojica2016b1metallolactamaseswheredo pages 1-3)

2. Domain Analysis
- AID/APOBEC cytidine deaminase family (InterPro: IPR016193; IPR016192; CATH cytidine deaminase FunFams)
   Function: Catalyze hydrolytic deamination of cytidine to uridine on ssDNA and/or RNA. Structural work confirms zinccoordinated deaminase domains and contextdependent editing on nucleic acids. This is a canonical nonpeptide CN bond hydrolysis. Recent 20232024 studies demonstrate APOBEC3A/3G catalysis in viral genomes and cell nucleic acids. (Communications Biology 2024; Nature Communications 2024; Nucleic Acids Research 2023; URLs: https://doi.org/10.1038/s42003-024-06239-w; https://doi.org/10.1038/s41467-024-52671-1; https://doi.org/10.1093/nar/gkac1238) (nakata2023cellularapobec3adeaminase pages 1-2, yang2024molecularmechanismfor pages 1-2)
   Multifunctionality: Some APOBECs show deaminaseindependent effects and diverse substrate scope (RNA and DNA), but all are deaminases when catalytic. (yang2024molecularmechanismfor pages 1-2)

- CE4/NodB chitin/polysaccharide deacetylases (InterPro: IPR002509; CATH CE4 deacetylase FunFams)
   Function: Metalassisted deacetylation of Nacetylglucosamine in chitin or peptidoglycan, hydrolyzing the amide CN bond to release acetate and amine, producing chitosan or deNacetylated PG. Recent structural/functional work (20232024) details conserved AspHisHis metal site and inhibitor development. (Nature Communications 2023; Frontiers in Plant Science 2024; URLs: https://doi.org/10.1038/s41467-023-39562-7; https://doi.org/10.3389/fpls.2023.1335646) (liu2023inhibitionofchitin pages 1-2, mapuranga2024themolecularstructure pages 1-2)
   Multifunctionality: The CE4 family includes multiple polysaccharide deacetylases with varied polysaccharide substrates; activity class remains amide CN hydrolysis on acetylated sugars. (mapuranga2024themolecularstructure pages 1-2)

- GTP cyclohydrolase I (InterPro: IPR001474; IPR018234)
   Function: First, ratelimiting step of tetrahydrobiopterin (BH4) biosynthesis from GTP; a cyclohydrolase that cleaves CN bonds in the pterin ringforming reaction (EC 3.5.4.16), a classic member of hydrolases acting on CN (nonpeptide) bonds. Its physiological importance is well documented clinically (DRD and BH4 disorders) and experimentally (eNOS coupling). (Movement Disorders Clin Pract. 2024; Cardiovascular Research 2002; Biochemical Society Transactions 2024; URLs: https://doi.org/10.1002/mdc3.14157; https://doi.org/10.1016/S0008-6363(02)00460-1; https://doi.org/10.1042/BST20231061) (novelli2024autosomalrecessiveguanosine pages 1-2, cai2002gtpcyclohydrolasei pages 1-2, tai2024dopaminesynthesisand pages 5-7)
   Multifunctionality: Enzyme is specific; pathway pleiotropy arises from BH4 roles, not from alternate GCH1 chemistries. (novelli2024autosomalrecessiveguanosine pages 1-2)

- Metallolactamase (MBL) fold enzymes (InterPro: IPR001279; IPR001018)
   Function: An ancient metaldependent / sandwich fold with conserved HxHxDH motif. Beyond lactam hydrolysis, the fold supports diverse hydrolase chemistries (including amide hydrolysis) and nonCN activities (nucleases, phosphodiesterases, lactonases, glyoxalases). Thus the fold is multifunctional; domain presence alone is not diagnostic for specific substrates. (Cells 2023 review; Curr Drug Targets 2016; URLs: https://doi.org/10.3390/cells12131752; https://doi.org/10.2174/1389450116666151001105622) (diene2023origindiversityand pages 1-2, mojica2016b1metallolactamaseswheredo pages 1-3)
   Multifunctionality: Highexplicitly documented; careful subfamily/activesite filtering is needed to avoid false positives or overspecificity. (diene2023origindiversityand pages 1-2)

- Polysaccharide deacetylases impacting bacterial infection systems (relevant to NodB/CE4 subsets)
   Experimental confirmation that deNacetylation of peptidoglycan by a CE4/NodB enzyme modifies virulence organelles (Type IV secretion) and lysozyme resistance; chemistry is amide deNacetylation. (PNAS 2023; URL: https://doi.org/10.1073/pnas.2119658120) (boamah2023peptidoglycandeacetylationcontrols pages 1-2)

3. GO Term Evaluation (GO:0016810)
- Appropriateness: For APOBEC/AID deaminases (hydrolytic deamination of cytidine), CE4/NodB deacetylases (amide hydrolysis), and GTP cyclohydrolase I (ring CN hydrolysis), GO:0016810 fits the core chemistry (hydrolase on nonpeptide CN). (liu2023inhibitionofchitin pages 1-2, mapuranga2024themolecularstructure pages 1-2, cai2002gtpcyclohydrolasei pages 1-2)
- Specificity: Where subfamilydiagnostic features are present, child terms should be preferred:
   Cytidine deaminase activity (DNA or RNA substrate, e.g., GO:0004126/GO:1990072/GO:0035515), justified by structural and functional data for APOBEC3A/3G (20232024). (nakata2023cellularapobec3adeaminase pages 1-2, yang2024molecularmechanismfor pages 1-2)
   Chitin deacetylase activity (GO:0004099) or peptidoglycan Nacetylglucosamine deacetylase activity (GO:0008762) for CE4 deacetylases supported by structural and in vivo data. (liu2023inhibitionofchitin pages 1-2, boamah2023peptidoglycandeacetylationcontrols pages 1-2, mapuranga2024themolecularstructure pages 1-2)
   GTP cyclohydrolase I activity (GO:0003934) for GCH1. (cai2002gtpcyclohydrolasei pages 1-2, novelli2024autosomalrecessiveguanosine pages 1-2)
- Caution: For MBLfoldonly conditions, avoid overspecific amide hydrolysis annotations without additional features; the fold spans nonCN activities (nucleases/lactonases). A safe baseline remains GO:0016810 (or even no function) unless specific lactamase or amidehydrolase determinants are present. (diene2023origindiversityand pages 1-2, mojica2016b1metallolactamaseswheredo pages 1-3)

4. Evidence Review (recent developments, applications, expert perspectives)
- APOBEC deaminases
   Recent mechanistic structures show how APOBEC3Gs noncatalytic domain enhances substrate capture and editing on DNA; multiple 20232024 studies link APOBEC3A/G to viral genome editing and cancer mutagenesis, consolidating substrate scope and biological roles. Applications include efforts to inhibit APOBECs therapeutically and use APOBECs in base editing; however, these are outside this rules scope. (Nature Communications 2024; Nucleic Acids Research 2023; Communications Biology 2024; URLs above) (yang2024molecularmechanismfor pages 1-2, nakata2023cellularapobec3adeaminase pages 1-2)

- CE4/NodB deacetylases
   Highresolution fungal CDA structures and inhibitor discovery (benzohydroxamic acid) demonstrate conserved metal triad (AspHisHis) and validated chitin deacetylase chemistry; fieldtested plant disease control underscores realworld application. (Nature Communications 2023; Frontiers in Plant Science 2024) (liu2023inhibitionofchitin pages 1-2, mapuranga2024themolecularstructure pages 1-2)
   In bacteria, a CE4 polysaccharide deacetylase modulates peptidoglycan and secretion system function, directly linking deNacetylase chemistry to pathogenesis phenotypes. (PNAS 2023) (boamah2023peptidoglycandeacetylationcontrols pages 1-2)

- GTP cyclohydrolase I/BH4 axis
   Clinical and translational literature in 2024 shows GCH1 deficiency as a cause of BH4related neurotransmitter deficiency and DRD; cardiovascular translational work shows GCH1 gene transfer raises intracellular BH4 and couples eNOS. These anchor both the specific GO activity and its conserved role in Metazoa. (Movement Disorders Clin Pract. 2024; Cardiovascular Research 2002) (novelli2024autosomalrecessiveguanosine pages 1-2, cai2002gtpcyclohydrolasei pages 1-2)

- Metallolactamase fold
   Authoritative 2023 review catalogs the folds distribution and multifunctionality across life, including many hydrolase activities and nucleases beyond CN hydrolysis, cautioning that foldonly criteria cannot uniquely predict substrate chemistry. (Cells 2023) (diene2023origindiversityand pages 1-2)

5. Taxonomic Considerations
- APOBEC/AID deaminases: Present broadly in vertebrates (and some invertebrates) with conserved deamination chemistry; annotations restricted to Fungi for some APOBEC/CMP deaminase sets in the rule should be reconsidered toward accurate taxonsignature pairings. Functional chemistry remains conserved where present. (yang2024molecularmechanismfor pages 1-2)
- CE4/NodB deacetylases: Found in bacteria, fungi, some insects and plants; chemistry is conserved across taxa, with roles adapted to host/pathogen contexts. Plant and fungal CE4based condition sets are appropriate; evidence supports activity in bacteria as well. (liu2023inhibitionofchitin pages 1-2, mapuranga2024themolecularstructure pages 1-2, boamah2023peptidoglycandeacetylationcontrols pages 1-2)
- GCH1: Broadly conserved in Metazoa and plants; the rules Metazoarestricted sets align with strong experimental/clinical evidence of function. (novelli2024autosomalrecessiveguanosine pages 1-2, cai2002gtpcyclohydrolasei pages 1-2)
- MBL fold: Ubiquitous across domains of life; taxonomic restriction alone does not resolve functional ambiguity given fold multifunctionality. (diene2023origindiversityand pages 1-2)

6. Rule Logic Assessment
- Biological coherence: Many condition sets pair an enzymatically diagnostic domain (e.g., APOBEC zincbinding cytidine deaminase motifs; CE4/NodB; GCH1) with appropriate taxa, making GO:0016810 biologically sound and conservative. (liu2023inhibitionofchitin pages 1-2, mapuranga2024themolecularstructure pages 1-2, cai2002gtpcyclohydrolasei pages 1-2, yang2024molecularmechanismfor pages 1-2)
- Redundancy: For CE4 and APOBEC, multiple overlapping InterPro and CATH FunFams may be redundant; retaining one highprecision signature plus an activesite or metalbinding motif could reduce complexity without losing specificity. (liu2023inhibitionofchitin pages 1-2, mapuranga2024themolecularstructure pages 1-2, yang2024molecularmechanismfor pages 1-2)
- False positives: High risk in MBLfoldonly sets and possibly in broad amidohydrolase/metallohydrolase superfamily signatures that include nucleases and other chemistries. Add subfamilylevel constraints (e.g., activesite motifs beyond HxHxDH, metal coordination variants, loop insertions, or cooccurring domains) to mitigate. (diene2023origindiversityand pages 1-2, mojica2016b1metallolactamaseswheredo pages 1-3)

7. Recommendations
- Keep GO:0016810 as the default for the foldwide, multifamily coverage when only broad family signatures are present. (diene2023origindiversityand pages 1-2)
- Where diagnostic, prefer specific child terms:
   Add GO:0004126/GO:1990072/GO:0035515 to APOBEC/AID deaminase condition sets with zincbinding cytidine deaminase motifs and nucleicacidbinding context. (yang2024molecularmechanismfor pages 1-2, nakata2023cellularapobec3adeaminase pages 1-2)
   Add GO:0004099 (chitin deacetylase) and/or GO:0008762 (peptidoglycan Nacetylglucosamine deacetylase) to CE4/NodB sets; retain GO:0016810 where substrate context is uncertain. (liu2023inhibitionofchitin pages 1-2, boamah2023peptidoglycandeacetylationcontrols pages 1-2, mapuranga2024themolecularstructure pages 1-2)
   Add GO:0003934 (GTP cyclohydrolase I activity) for GCH1 sets in Metazoa, optionally GO:0036460 (tetrahydrobiopterin biosynthetic process). (cai2002gtpcyclohydrolasei pages 1-2, novelli2024autosomalrecessiveguanosine pages 1-2)
- For MBLfold conditions, explicitly restrict by subfamily or add evidence requirements (e.g., lactamase subclassspecific motifs or proximity to resistance operons) to avoid assigning CN hydrolase activity to nucleaselike clades. (diene2023origindiversityand pages 1-2, mojica2016b1metallolactamaseswheredo pages 1-3)
- Taxonomy: Current restrictions for CE4 and GCH1 sets are consistent with conserved activities. Consider auditing APOBECrestricted fungal sets to ensure they correspond to fungal cytidine deaminases rather than vertebrate APOBECs. (yang2024molecularmechanismfor pages 1-2)

References (URLs and dates)
- APOBEC RNA/DNA cytidine deamination and disease links: Norden et al., Communications Biology, 2024-05-14, https://doi.org/10.1038/s42003-024-06239-w (nakata2023cellularapobec3adeaminase pages 1-2); Yang et al., Nature Communications, 2024-10-08, https://doi.org/10.1038/s41467-024-52671-1 (yang2024molecularmechanismfor pages 1-2); Nakata et al., Nucleic Acids Research, 2023-01-05, https://doi.org/10.1093/nar/gkac1238 (nakata2023cellularapobec3adeaminase pages 1-2).
- CE4/NodB deacetylases: Liu et al., Nature Communications, 2023-06-28, https://doi.org/10.1038/s41467-023-39562-7 (liu2023inhibitionofchitin pages 1-2); Mapuranga et al., Frontiers in Plant Science, 2024-01-09, https://doi.org/10.3389/fpls.2023.1335646 (mapuranga2024themolecularstructure pages 1-2); Boamah et al., PNAS, 2023-05-30, https://doi.org/10.1073/pnas.2119658120 (boamah2023peptidoglycandeacetylationcontrols pages 1-2).
- GCH1/BH4 axis: Novelli et al., Movement Disorders Clinical Practice, 2024-07, https://doi.org/10.1002/mdc3.14157 (novelli2024autosomalrecessiveguanosine pages 1-2); Cai et al., Cardiovascular Research, 2002-09, https://doi.org/10.1016/S0008-6363(02)00460-1 (cai2002gtpcyclohydrolasei pages 1-2); Tai et al., Biochemical Society Transactions, 2024-05-31, https://doi.org/10.1042/BST20231061 (tai2024dopaminesynthesisand pages 5-7).
- Metallolactamase fold diversity and multifunctionality: Diene et al., Cells, 2023-06-30, https://doi.org/10.3390/cells12131752 (diene2023origindiversityand pages 1-2); Mojica et al., Curr Drug Targets, 2016-06, https://doi.org/10.2174/1389450116666151001105622 (mojica2016b1metallolactamaseswheredo pages 1-3).

References

1. (liu2023inhibitionofchitin pages 1-2): Lin Liu, Yeqiang Xia, Yingchen Li, Yong Zhou, Xiaofeng Su, Xiaojing Yan, Yan Wang, Wende Liu, Hongmei Cheng, Yuanchao Wang, and Qing Yang. Inhibition of chitin deacetylases to attenuate plant fungal diseases. Nature Communications, Jun 2023. URL: https://doi.org/10.1038/s41467-023-39562-7, doi:10.1038/s41467-023-39562-7. This article has 30 citations and is from a highest quality peer-reviewed journal.

2. (mapuranga2024themolecularstructure pages 1-2): Johannes Mapuranga, Jia-Huei Chang, Hao Li, Yingdan Zhang, Ruolin Li, Lulu Song, Na Zhang, and Wenxiang Yang. The molecular structure, biological roles, and inhibition of plant pathogenic fungal chitin deacetylases. Frontiers in Plant Science, Jan 2024. URL: https://doi.org/10.3389/fpls.2023.1335646, doi:10.3389/fpls.2023.1335646. This article has 8 citations and is from a poor quality or predatory journal.

3. (novelli2024autosomalrecessiveguanosine pages 1-2): Maria Novelli, Manuela Tolve, Vicente Quiroz, Claudia Carducci, Rossella Bove, Giacomina Ricciardi, Kathryn Yang, Filippo Manti, Francesco Pisani, Darius EbrahimiFakhari, Serena Galosi, and Vincenzo Leuzzi. Autosomal recessive guanosine triphosphate cyclohydrolase i deficiency: redefining the phenotypic spectrum and outcomes. Movement Disorders Clinical Practice, 11:1072-1084, Jul 2024. URL: https://doi.org/10.1002/mdc3.14157, doi:10.1002/mdc3.14157. This article has 2 citations and is from a peer-reviewed journal.

4. (cai2002gtpcyclohydrolasei pages 1-2): S. Cai, N. Alp, D. McDonald, Ian Smith, J. Kay, L. Canevari, S. Heales, and K. Channon. Gtp cyclohydrolase i gene transfer augments intracellular tetrahydrobiopterin in human endothelial cells: effects on nitric oxide synthase activity, protein levels and dimerisation. Cardiovascular Research, 55:838-849, Sep 2002. URL: https://doi.org/10.1016/s0008-6363(02)00460-1, doi:10.1016/s0008-6363(02)00460-1. This article has 146 citations and is from a domain leading peer-reviewed journal.

5. (diene2023origindiversityand pages 1-2): Seydina M. Diene, Pierre Pontarotti, Sad Azza, Nicholas Armstrong, Lucile Pinault, Eric Chabrire, Philippe Colson, Jean-Marc Rolain, and Didier Raoult. Origin, diversity, and multiple roles of enzymes with metallo--lactamase fold from different organisms. Cells, 12:1752, Jun 2023. URL: https://doi.org/10.3390/cells12131752, doi:10.3390/cells12131752. This article has 21 citations and is from a poor quality or predatory journal.

6. (mojica2016b1metallolactamaseswheredo pages 1-3): Maria F. Mojica, Robert A. Bonomo, and Walter Fast. B1-metallo--lactamases: where do we stand? Current drug targets, 17 9:1029-50, Jun 2016. URL: https://doi.org/10.2174/1389450116666151001105622, doi:10.2174/1389450116666151001105622. This article has 280 citations and is from a peer-reviewed journal.

7. (nakata2023cellularapobec3adeaminase pages 1-2): Yoshihiro Nakata, Hirotaka Ode, Mai Kubota, Takaaki Kasahara, Kazuhiro Matsuoka, Atsuko Sugimoto, Mayumi Imahashi, Yoshiyuki Yokomaku, and Yasumasa Iwatani. Cellular apobec3a deaminase drives mutations in the sars-cov-2 genome. Nucleic Acids Research, 51:783-795, Jan 2023. URL: https://doi.org/10.1093/nar/gkac1238, doi:10.1093/nar/gkac1238. This article has 58 citations and is from a highest quality peer-reviewed journal.

8. (yang2024molecularmechanismfor pages 1-2): Hanjing Yang, Josue Pacheco, Kyumin Kim, Ayub Bokani, Fumiaki Ito, Diako Ebrahimi, and Xiaojiang S. Chen. Molecular mechanism for regulating apobec3g dna editing function by the non-catalytic domain. Nature Communications, Oct 2024. URL: https://doi.org/10.1038/s41467-024-52671-1, doi:10.1038/s41467-024-52671-1. This article has 3 citations and is from a highest quality peer-reviewed journal.

9. (tai2024dopaminesynthesisand pages 5-7): Mary Dayne Sia Tai, Gloria Gamiz-Arco, and Aurora Martinez. Dopamine synthesis and transport: current and novel therapeutics for parkinsonisms. Biochemical Society Transactions, 52:1275-1291, May 2024. URL: https://doi.org/10.1042/bst20231061, doi:10.1042/bst20231061. This article has 9 citations and is from a peer-reviewed journal.

10. (boamah2023peptidoglycandeacetylationcontrols pages 1-2): David Boamah, Michael C. Gilmore, Sarah Bourget, Anushka Ghosh, Mohammad J. Hossain, Joseph P. Vogel, Felipe Cava, and Tamara J. OConnor. Peptidoglycan deacetylation controls type iv secretion and the intracellular survival of the bacterial pathogen legionella pneumophila. Proceedings of the National Academy of Sciences of the United States of America, May 2023. URL: https://doi.org/10.1073/pnas.2119658120, doi:10.1073/pnas.2119658120. This article has 19 citations and is from a highest quality peer-reviewed journal.

## Citations

1. yang2024molecularmechanismfor pages 1-2
2. mapuranga2024themolecularstructure pages 1-2
3. novelli2024autosomalrecessiveguanosine pages 1-2
4. diene2023origindiversityand pages 1-2
5. boamah2023peptidoglycandeacetylationcontrols pages 1-2
6. liu2023inhibitionofchitin pages 1-2
7. cai2002gtpcyclohydrolasei pages 1-2
8. tai2024dopaminesynthesisand pages 5-7
9. https://doi.org/10.1038/s42003-024-06239-w;
10. https://doi.org/10.1038/s41467-024-52671-1;
11. https://doi.org/10.1093/nar/gkac1238
12. https://doi.org/10.1038/s41467-023-39562-7;
13. https://doi.org/10.3389/fpls.2023.1335646
14. https://doi.org/10.1002/mdc3.14157;
15. https://doi.org/10.1016/S0008-6363(02
16. https://doi.org/10.1042/BST20231061
17. https://doi.org/10.3390/cells12131752;
18. https://doi.org/10.2174/1389450116666151001105622
19. https://doi.org/10.1073/pnas.2119658120
20. https://doi.org/10.1038/s42003-024-06239-w
21. https://doi.org/10.1038/s41467-024-52671-1
22. https://doi.org/10.1038/s41467-023-39562-7
23. https://doi.org/10.1002/mdc3.14157
24. https://doi.org/10.3390/cells12131752
25. https://doi.org/10.1038/s41467-023-39562-7,
26. https://doi.org/10.3389/fpls.2023.1335646,
27. https://doi.org/10.1002/mdc3.14157,
28. https://doi.org/10.1016/s0008-6363(02
29. https://doi.org/10.3390/cells12131752,
30. https://doi.org/10.2174/1389450116666151001105622,
31. https://doi.org/10.1093/nar/gkac1238,
32. https://doi.org/10.1038/s41467-024-52671-1,
33. https://doi.org/10.1042/bst20231061,
34. https://doi.org/10.1073/pnas.2119658120,